
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-02</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09234-1'>WSTF nuclear autophagy regulates chronic but not acute inflammation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'nature.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 15:50:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Acute inflammation is an essential response that our bodies use to combat infections1. However, in the absence of infections, chronic inflammation can have a pivotal role in the onset and progression of chronic diseases, such as arthritis, cancer, autoimmune disorders, metabolic-dysfunction-associated steatohepatitis (MASH), and most ageing-associated pathologies2,3. The underlying mechanisms that distinguish chronic inflammation from its acute counterpart remain unclear, posing challenges to the development of targeted therapies for these major diseases. Here we identify a mechanism that separates the two responses: during chronic but not acute inflammation, chromatin remodelling is influenced by nuclear autophagy, in which the WSTF protein of the ISWI chromatin-remodelling complex interacts with the ATG8 autophagy protein family in the nucleus. This interaction leads to WSTF nuclear export and subsequent degradation by autophagosomes and lysosomes in the cytoplasm. Loss of WSTF leads to chromatin opening over inflammatory genes, amplifying inflammation. Cell-penetrating peptides that block the WSTF–ATG8 interaction do not affect acute inflammation but suppress chronic inflammation in senescence as well as in MASH and osteoarthritis in mouse models and patient samples. The ability to specifically target chronic inflammation without blunting acute inflammation offers an approach for treating common chronic inflammatory diseases. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout RNA-seq and ATAC–seq data have been deposited at the NCBI Gene Expression Omnibus (GEO) database under accession number GSE214410. Other original data are included in the Supplementary Information or are available on reasonable request. Source data are provided with this paper. Medzhitov, R. Origin and physiological roles of inflammation. Franceschi, C., Garagnani, P., Vitale, G., Capri, M. & Salvioli, S. Inflammaging and ‘Garb-aging'. Hotamisligil, G. S. Inflammation and metabolic disorders. Chronic inflammation in the etiology of disease across the life span. Ferrucci, L. & Fabbri, E. Inflammageing: chronic inflammation in ageing, cardiovascular disease, and frailty. Deretic, V. Autophagy in inflammation, infection, and immunometabolism. Cadwell, K. Crosstalk between autophagy and inflammatory signalling pathways: balancing defence and homeostasis. Levine, B., Mizushima, N. & Virgin, H. W. Autophagy in immunity and inflammation. Deretic, V. Autophagy as an innate immunity paradigm: expanding the scope and repertoire of pattern recognition receptors. Mochida, K. et al. Receptor-mediated selective autophagy degrades the endoplasmic reticulum and the nucleus. Dou, Z. et al. Autophagy mediates degradation of nuclear lamina. Liu, H. et al. VAMP2 controls murine epidermal differentiation and carcinogenesis by regulation of nucleophagy. Xu, C. et al. SIRT1 is downregulated by autophagy in senescence and ageing. Papandreou, M.-E., Konstantinidis, G. & Tavernarakis, N. Nucleophagy delays aging and preserves germline immortality. Zhao, H. et al. Destabilizing heterochromatin by APOE mediates senescence. Lamark, T. & Johansen, T. Mechanisms of selective autophagy. Behrends, C., Sowa, M. E., Gygi, S. P. & Harper, J. W. Network organization of the human autophagy system. Mizushima, N., Yamamoto, A., Matsui, M., Yoshimori, T. & Ohsumi, Y. In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker. Clapier, C. R., Iwasa, J., Cairns, B. R. & Peterson, C. L. Mechanisms of action and regulation of ATP-dependent chromatin-remodelling complexes. Cellular senescence: putting the paradoxes in perspective. Dou, Z. et al. Cytoplasmic chromatin triggers inflammation in senescence and cancer. & Varga-Weisz, P. WSTF-ISWI chromatin remodeling complex targets heterochromatic replication foci. Corona, D. F. et al. ISWI is an ATP-dependent nucleosome remodeling factor. Yadon, A. N. & Tsukiyama, T. SnapShot: chromatin remodeling: ISWI. Persistent DNA damage signalling triggers senescence-associated inflammatory cytokine secretion. Paull, T. T. Mechanisms of ATM activation. Xiao, A. et al. WSTF regulates the H2A.X DNA damage response via a novel tyrosine kinase activity. & Greenleaf, W. J. ATAC-seq: a method for assaying chromatin accessibility genome-wide. Chien, Y. et al. Control of the senescence-associated secretory phenotype by NF-κB promotes senescence and enhances chemosensitivity. Kang, T.-W. et al. Senescence surveillance of pre-malignant hepatocytes limits liver cancer development. Nusinow, D. P. et al. Quantitative proteomics of the Cancer Cell Line Encyclopedia. Pezone, A. et al. Inflammation and DNA damage: cause, effect or both. Creation of a novel peptide with enhanced nuclear localization in prostate and pancreatic cancer cell lines. Brunt, E. M. Pathology of nonalcoholic fatty liver disease. Brown, G. T. & Kleiner, D. E. Histopathology of nonalcoholic fatty liver disease and nonalcoholic steatohepatitis. Michelotti, G. A. et al. Smoothened is a master regulator of adult liver repair. Syn, W.-K. et al. Hedgehog-mediated epithelial-to-mesenchymal transition and fibrogenic repair in nonalcoholic fatty liver disease. Increased glutaminolysis marks active scarring in nonalcoholic steatohepatitis progression. Hansen, H. H. et al. Human translatability of the GAN diet-induced obese mouse model of non-alcoholic steatohepatitis. Woodell-May, J. E. & Sommerfeld, S. D. Role of inflammation and the immune system in the progression of osteoarthritis. Robinson, W. H. et al. Low-grade inflammation as a key mediator of the pathogenesis of osteoarthritis. Glasson, S. S., Chambers, M. G., Van Den Berg, W. B. The OARSI histopathology initiative—recommendations for histological assessments of osteoarthritis in the mouse. Sherwood, J. C., Bertrand, J., Eldridge, S. E. & Dell'Accio, F. Cellular and molecular mechanisms of cartilage damage and repair. Wang, Y. et al. Nuclear autophagy interactome unveils WSTF as a constitutive nuclear inhibitor of inflammation. Chen Lf, Fischle, W., Verdin, E. & Greene, W. C. Duration of nuclear NF-kappaB action regulated by reversible acetylation. & Dillin, A. Macroautophagy in quiescent and senescent cells: a pathway to longevity? & Debnath, J. Autophagy at the crossroads of catabolism and anabolism. Vizioli, M. G. et al. Mitochondria-to-nucleus retrograde signaling drives formation of cytoplasmic chromatin and inflammation in senescence. Practical guidelines for high-resolution epigenomic profiling of nucleosomal histones in postmortem human brain tissue. Deep profiling of proteome and phosphoproteome by isobaric labeling, extensive liquid chromatography, and mass spectrometry. Xu, P., Duong, D. M. & Peng, J. Systematical optimization of reverse-phase chromatography for shotgun proteomics. Pagala, V. R. et al. Quantitative protein analysis by mass spectrometry. Niu, M. et al. Extensive peptide fractionation and y ion-based interference detection method for enabling accurate quantification by isobaric labeling and mass spectrometry. Systematic optimization of long gradient chromatography mass spectrometry for deep analysis of brain proteome. Wang, X. et al. JUMP: a tag-based database search tool for peptide identification with high sensitivity and accuracy. Li, Y. et al. JUMPg: an integrative proteogenomics pipeline identifying unannotated proteins in human brain and cancer cells. Shi, H. et al. Amino acids license kinase mTORC1 activity and Treg cell function via small G proteins Rag and Rheb. Peng, J., Elias, J. E., Thoreen, C. C., Licklider, L. J. & Gygi, S. P. Evaluation of multidimensional chromatography coupled with tandem mass spectrometry (LC/LC-MS/MS) for large-scale protein analysis: the yeast proteome. & Aebersold, R. Interpretation of shotgun proteomic data: the protein inference problem. Kleiner, D. E. et al. Design and validation of a histological scoring system for nonalcoholic fatty liver disease. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Carpenter, A. E. et al. CellProfiler: image analysis software for identifying and quantifying cell phenotypes. McQuin, C. et al. CellProfiler 3.0: next-generation image processing for biology. Kamekura, S. et al. Osteoarthritis development in novel experimental mouse models induced by knee joint instability. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Inflammatory networks during cellular senescence: causes and consequences. Babicki, S. et al. Heatmapper: web-enabled heat mapping for all. Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. & Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. John, S. et al. Chromatin accessibility pre-determines glucocorticoid receptor binding patterns. Differential oestrogen receptor binding is associated with clinical outcome in breast cancer. An, H. et al. TEX264 is an endoplasmic reticulum-resident ATG8-interacting protein critical for ER remodeling during nutrient stress. A., Mancias, J. D. & Gygi, S. P. Proteome-wide protein expression profiling across five pancreatic cell lines. McAlister, G. C. et al. MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes. Huttlin, E. L. et al. A tissue-specific atlas of mouse protein phosphorylation and expression. & Gygi, S. P. Global analysis of protein expression and phosphorylation levels in nicotine-treated pancreatic stellate cells. Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Chen, H. et al. cGAS suppresses genomic instability as a decelerator of replication forks. Kirkin, V. et al. A role for NBR1 in autophagosomal degradation of ubiquitinated substrates. Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. & Sternberg, M. J. E. The Phyre2 web portal for protein modeling, prediction and analysis. Structures of CaV2 Ca2+/CaM-IQ domain complexes reveal binding modes that underlie calcium-dependent inactivation and facilitation. Highly accurate protein structure prediction with AlphaFold. Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. The ClusPro web server for protein-protein docking. The proteomic studies in human fibroblasts were conducted in collaboration with J. W. Harper and J. D. Mancias. We thank Y. Tan for generating the sgRNA constructs targeting SMARCA5 and the next-generation sequencing samples in control cells; J. Xia, T. Kumazawa, J. Zhu, E. Wang, K. Xiang, C. Pan, T. Yin, L. Tan and B. J. W. Lim for technical support and discussions; the members of the microscopy core facility of Center for Regenerative Medicine at Massachusetts General Hospital for assistance with confocal microscopy; the members of the next-generation sequencing core at Massachusetts General Hospital for assistance with RNA-seq; R. Graham for advice on ATAC–seq; and P. B. Alexander, J. Rajagopal, Y.-C. Hsu, D. A. Sinclair, D. B. Sykes, A. Sahay, J. Zhu, K. Hochedlinger, R. T. Lee and R. Mostoslavsky for reading our manuscript. is supported by National Institutes of Health (NIH) awards R35GM137889, R01AG082785, R00AG053406, UG3CA268117, UH3CA268117 and R21AG073894, Glenn Foundation for Medical Research and AFAR Grant for Junior Faculty, and the Hevolution/AFAR New Investigator award; T.J. by a strategic thematic grant from UiT and by TOPPFORSK (grant 249884) program of the Research Council of Norway; X.-F.W. is supported in part by the Harvard Stem Cell Institute and Harvard Catalyst NIH UL1TR002541. is supported by Glenn Foundation for Medical Research Postdoctoral Fellowships in Aging Research from American Federation for Aging Research (AFAR); M.S.S. Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA, USA Yu Wang, Yaosi Liang, Yanxin Xu, Angelique Onorati, Ana Maria Cabral Burkard, Chia-Kang Ho, Ulrike Rieprecht, Tara O'Brien & Zhixun Dou Yu Wang, Yaosi Liang, Yanxin Xu, Angelique Onorati, Ana Maria Cabral Burkard, Chia-Kang Ho, Ulrike Rieprecht, Tara O'Brien & Zhixun Dou Department of Medicine, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Yu Wang, Yaosi Liang, Yanxin Xu, Angelique Onorati, Ana Maria Cabral Burkard, Chia-Kang Ho, Ulrike Rieprecht, Tara O'Brien & Zhixun Dou Yu Wang, Athanasios Kournoutis & Terje Johansen Department of Cell Biology, Harvard Medical School, Boston, MA, USA Department of Pharmacology and Cancer Biology, Duke University Medical Center, Durham, NC, USA Yaosi Liang, Xinyi Ma & Xiao-Fan Wang Division of Gastroenterology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Marc Samuel Sherman, Kathleen E. Corey & Wolfram Goessling Department of Neurology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Xianting Li, Xiaoting Zhou & Zhenyu Yue Department of Neuroscience, Icahn School of Medicine at Mount Sinai, New York, NY, USA Xianting Li, Xiaoting Zhou & Zhenyu Yue Center for Parkinson's Disease Neurobiology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Xianting Li, Xiaoting Zhou & Zhenyu Yue Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Xianting Li, Xiaoting Zhou & Zhenyu Yue Department of Medicine, Duke University, Durham, NC, USA Kuo Du, Raquel Maeso-Díaz & Anna Mae Diehl Department of Plastic and Reconstructive Surgery, Shanghai Ninth People's Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, China Department of Orthopaedic Surgery, Duke University, Durham, NC, USA Jing Xie, Hui Zhang, Xinyi Ma & Benjamin Alman Department of Molecular Biology, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Department of Pathology, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA Department of Biochemistry and Structural Biology, Sam and Ann Barshop Institute for Longevity and Aging Studies, UT Health San Antonio, San Antonio, TX, USA Bioinformatics Core, Department of Biostatistics, Harvard T.H. Chan School of Public Health, Boston, MA, USA Jihe Liu & Shannan J. Ho Sui Aging, Cancer and Immuno-oncology Program, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, CA, USA Corey Bretz, Aaron P. Havas & Peter D. Adams State Key Laboratory of Common Mechanism Research for Major Diseases, Suzhou Institute of Systems Medicine, Chinese Academy of Medical Sciences & Peking Union Medical College, Suzhou, China Department of Cell and Cancer Biology, University of Toledo, College of Medicine and Life Sciences, Toledo, OH, USA Department of Molecular Biology and MGH Research Institute, Massachusetts General Hospital, Boston, MA, USA Department of Genetics, Harvard Medical School, Boston, MA, USA Krantz Family Center for Cancer Research, Massachusetts General Hospital, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and conducted most of the experiments. contributed to the MS results of ATG8 nuclear interactomes and proteomes in human fibroblasts, performed in the J. W. Harper and J. D. Mancias laboratories. contributed to part of the GABARAP–WSTF interaction results. contributed to part of the data in MASH and OA experiments. contributed to cohort 2 data of patients with MASH. performed cancer cell WSTF co-IP with p65 and GABARAP experiments. contributed to part of the autophagy and senescence data. contributed to mouse brain LC3B interactome data. conducted part of the mouse MASH experiments. contributed to part of the IHC staining and RT–qPCR for mouse MASH experiments. contributed to part of the osteoarthritis experiments. contributed to part of the shATG13 and shFIP200 data. contributed to computational analyses of RNA-seq and ATAC–seq data. contributed to experimental design and part of the next-generation sequencing data. contributed to part of the protein interaction maps. contributed to part of the mouse NRAS liver tissue samples. generated samples for next-generation sequencing in senescent cells and contributed to part of the autophagy, senescence and inflammation results. conceived and supervised inflammation experiments. contributed to reagents of ISWI complex and supervised epigenetic experiments. supervised the study and provided funding support. Correspondence to Xiao-Fan Wang, Terje Johansen or Zhixun Dou. The authors declare no competing interests. Nature thanks Katja Simon, Anja Zeigerer, Charles Roberts, and the other, anonymous, reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, IMR90 cells were subjected to protein extraction using different approaches. Supernatants were analysed by immunoblotting. b, IMR90 cells were subjected to subcellular fractionation, generating nuclear and membrane fractions, then analysed by immunoblotting. c, IMR90 cells with indicated stable expression were fractionated, and the nuclear and chromatin fractions (collectively referred to as “nucleus”) were subjected to HA co-IP and immunoblotting. d, Overlap of autophagy interaction network (AIN) from a previous study versus the ATG8 nuclear interaction network from this study. e, Immunoblotting showing the cytoplasmic and nuclear fractions of mouse brains. f, The cytoplasmic and nuclear fractions of the mouse brains were subjected to GFP co-IP. Silver-stained protein gels are shown. g, Interaction maps showing chromatin remodelling proteins in binding to human ATG8s and mouse LC3B. h-j, IMR90 cells were induced to senescence by HRasV12 and analysed by immunoblotting (h). Scheme depicting the workflow for TMT-proteomic analysis (i). The altered proteins in senescence were subjected to GO Cellular Component analysis, identifying nuclear proteins, which were further analysed by GO Biological Processes (j); P values are presented as -log10 (P). k-l, IMR90 cells stably expressing control or sh-ATG7 hairpin were treated with etoposide to induce senescence, followed by immunoblotting analyses for whole cell extract (k) or nuclear fractions (l). m, Binding intensities of human ATG8 (n = 3 biologically independent samples) and mouse LC3B to WSTF (n = 1; lysates pooled from 3 animals), and the protein levels of WSTF upon HRasV12 (n = 3 biologically independent samples) or etoposide-induced senescence (n = 2 biologically independent samples) from proteomic analyses. Results were presented as mean values with s.d. ; one-way ANOVA coupled with Dunnett's post hoc test for. For HRasV12 comparison, P values were calculated using an unpaired two-tailed Student's t-test; exact P values are shown in the figure; n.s. n, mRNA levels of WSTF from RNA-Seq upon HRasV12 (n = 3 biologically independent samples) or etoposide-induced senescence (n = 2 biologically independent samples) from proteomic analyses. Results were presented as mean values with s.d. ; P values were calculated using an unpaired two-tailed Student's t-test. No statistical analysis was performed for groups with n < 3. a, Primary human and mouse fibroblasts were induced to senescence and analysed by immunoblotting. b, A549 senescence showed WSTF loss. c-d, IMR90 quiescence does not show WSTF loss. e-f, Senescent IMR90 induced by HRasV12 or replication exhaustion showed WSTF and not SNF2H loss. g, RT-qPCR analyses of BAZ1B/WSTF and CDKN2A/p16 under various senescent conditions. Results were normalized to Lamin A/C and presented as mean values with s.d. ; n = 3 biologically independent samples; P values were calculated from unpaired two-tailed Student's t test. : non-significant; exact P values are shown in the figure. h-i, IMR90 cells stably expressing sh-NTC or sh-ATG7 were treated by IR (h) or etoposide (i) and harvested at indicated days, and analysed by immunoblotting. j-l, ATG13 or FIP200 knockdown inhibited WSTF loss in senescent IMR90 (j-k) and ARPE19 (l). m and n, Senescent IMR90 cells were treated with bafilomycin A1 (m) or MG132 (n) for 2 days, and analysed by immunoblotting. o, IMR90 cells were left untreated or treated with IR to induce senescence (harvested on day 14) and then stained with WSTF and LAMP1 antibodies, followed by imaging under a confocal microscopy. p, ARPE19 cells stably expressing mCherry-GFP-tagged WSTF were induced to senescence; the lysosomal inhibitor Lys05 was added for the last 2 days. Confocal microscopy images are shown. q, Percentage of cells with cytoplasmic GFP or mCherry. Data presented are mean values from four randomly selected fields, with over 200 cells analysed. **** P value < 0.0001; one-way ANOVA coupled with Dunnett's post hoc test. r, SNF2H is unchanged in senescent IMR90 cells. a, In vitro translated 35S-methionine HA-WSTF was subjected to GST pull-down using GST-ATG8 proteins purified from bacteria. b-c, HEK293T cells were transfected with HA-tagged constructs, followed by HA co-IP and immunoblotting (b). Quantification of the relative binding to WSTF is shown (c) from 3 independent experiments. ; exact P values are shown in the figure; unpaired two-tailed Student's t-test. d-f, Endogenous co-IP in HEK293T (d), HeLa (e), and MDA-MB-231(f), showing GABARAP-WSTF interaction. g-h, GST-GABARAP pulldown of in vitro translated WSTF (g) or SNF2H (h). are shown; four randomly selected fields with over 200 cells were used; exact P values are shown in the figure; one-way ANOVA coupled with Dunnett's post hoc test. k, BiFC assay in senescent IMR90, stained with indicated antibodies, and imaged under a confocal microscopy. l, In vitro pull-down showing the importance of F77 residue of GABARAP in binding WSTF. are shown (n = 3 biological replicates); exact P values are shown in the figure; one-way ANOVA coupled with Tukey's post hoc test. n, Scheme of WSTF mutants in binding to GABARAP, summarizing the key findings. p, GST-GABARAP pulldown of in vitro translated WSTF. s, IMR90 stably expressing HA-WSTF constructs were induced to senescence and analysed by immunoblotting. t-v, IMR90 expressing HA-WSTF constructs were stained and imaged. are shown; over 400 cells in 4 different fields were randomly selected and scored; exact P values are shown in the figure; n.s. : non-significant; one-way ANOVA coupled with Dunnett's post hoc test. a and b, Proliferating (Untr.) or senescent IMR90 cell lysates were subjected to GST-tagged GABARAP pulldown (a) with or without lambda protein phosphatase treatment of the cell lysates (b), then followed by immunoblotting with indicated antibodies. c, IMR90 cells expressing HA-GFP or HA-GABARAP were treated with IR to induce senescence with or without ATM inhibitor KU-55933 (ATMi), then subjected to HA immunoprecipitation followed by immunoblotting with indicated antibodies. d, Untreated or senescent IMR90 cells treated with or without ATMi were subjected to GST-tagged GABARAP pulldown, then evaluated by immunoblotting with the indicated antibodies. e, mCherry-GFP-tagged WSTF stably expressing ARPE19 cells and IMR90 cells were either left untreated, or treated with etoposide to induce senescence. ATMi was added after 5 days of etoposide treatment and the cells were analysed after 10 days of etoposide treatment. Cells were imaged using a confocal microscopy. Representative images and scale bars are shown. f, Bar graphs showing the quantification of the percentage of cells with cytoplasmic GFP or mCherry. Data presented are mean values from four randomly selected fields, with over 200 cells analysed. ****P < 0.0001; one-way ANOVA coupled with Dunnett's post hoc test. g, Untreated or senescent IMR90 cells stably expressing non-targeting control (sh-NTC) or sh-ATM were subjected to GST-tagged GABARAP pulldown followed by immunoblotting with indicated antibodies. The western blot experiments shown in a-d and g were repeated independently at least three times with similar results. a, Scheme of experimental design. WSTF overexpression did not affect SA-β-gal activity. are shown; 4 randomly selected fields; n.s. Results were normalized to Lamin A/C and presented as mean values with s.d. non-significant; exact P values are shown in the figure; one-way ANOVA coupled with Dunnett's post hoc test. c, IMR90 cells stably expressing vector control or WSTF were treated by IR and analysed by immunoblotting. d-e, The conditioned media from IR-induced senescent cells were analysed by cytokine arrays, showing different exposures. are shown; n = 3 biological replicates; n.s. non-significant; **** P values < 0.0001; unpaired two-tailed Student's t-test. f-g, IMR90 were induced to senescence by HRasV12, then analysed by immunoblotting (f) or RT-qPCR (g). ; n = 3 biologically independent samples; **** P values < 0.0001; one-way ANOVA coupled with Dunnett's post hoc test. h-k, IMR90 with WSTF knockdown were analysed by immunoblotting (h-j) or RT-qPCR (k). ; n = 3 biological repeats; exact P values are shown in the figure; two-way ANOVA coupled with Tukey's post hoc test. l, Stably expressed vector or WSTF IMR90 cells were infected by HRasV12 retrovirus and were harvested 7 days post infection. The cells were analysed by RNA-Seq. Differentially expressed genes (DEGs) were plotted in the HRasV12 condition. Representative genes down-regulated by WSTF in HRasV12-induced senescence were annotated. m, GO analysis using DEGs downregulated by WSTF in senescence. GO terms, P values, and the number of genes are shown. Statistical significance was assessed using a two-sided Fisher exact test with Benjamini–Hochberg correction for multiple comparisons; P values are presented as -log10 (P). n, IMR90 cells were treated with IR or HRasV12 to establish senescence first, and then were infected with lentivirus encoding vector or WSTF, followed by immunoblotting. The western blot experiments shown in c, f, h-j and n were repeated independently at least three times with similar results. a, IMR90 cells stably expressing vector control or WSTF were left untreated or treated with IR to induce senescence, followed by immunoblotting analyses. b, IMR90 cells were stained with γH2AX and H3K27me3 antibodies, followed by imaging under a confocal microscopy. Quantification was performed across four biologically independent experiments. For each replicate, results presented are the mean values from four randomly selected imaging fields, with over 200 cells analysed per field. Statistical analysis was conducted using unpaired two-tailed Student's t-test. c, Vector and WSTF stably expressing IMR90 cells were left untreated or induced to senescence with IR, then stained with a p65 antibody, followed by imaging under a confocal microscopy. Bar graphs showing the quantification of the percentage of nuclear p65 positive cells in vector and WSTF cells with or without IR treatment. Results shown are the mean values from four randomly selected fields with over 200 cells. Statistical analysis was conducted using two-way ANOVA coupled with Tukey's post hoc test. d, Schematic illustrations of WSTF full-length and truncations. e, HEK293T cells were transfected with HA-tagged wild-type or mutant WSTF constructs, and then subjected to HA immunoprecipitation and immunoblotting with indicated antibodies. f-i, full-length WSTF or WSTF truncations were stably expressed in IMR90 cells and were left untreated or induced to senescence by IR. Cells were harvested on day 14 post IR and analysed by immunoblotting (f and h) or RT-qPCR (g and i). Results of RT-qPCR were normalized to Lamin A/C and presented as mean values with s.d. Different letters (a, b, c, and d) represent significant differences (P < 0.05), whereas the same letter indicates no statistical significance, calculated by one-way ANOVA coupled with Dunnett's post hoc test. a and b, IMR90 cells were engineered to express control sgRNA or sgRNA against SNF2H, combined with vector or WSTF overexpression. The cells were left untreated or induced to senescence by IR, harvested on day 14, and analysed by immunoblotting (a) or RT-qPCR (b). are shown; n = 3 biologically independent samples; exact P values are shown in the figure; one-way ANOVA coupled with Dunnett's post hoc test. c, Scheme of experimental design for ATAC-Seq. d, The representative track views of a housekeeper gene Vinculin (VCL). e, GO analysis showing the top downregulated categories of differential accessible regions (DARs) by WSTF in senescent cells. Statistical significance was assessed using a two-sided Fisher exact test with Benjamini–Hochberg correction for multiple comparisons.P values are displayed as -log10 (P). f, The genes from DEGs downregulated by WSTF from RNA-Seq were overlapped with genes from DARs downregulated by WSTF from ATAC-Seq. The overlapping genes were subjected to GO analysis, and the GO terms with the numbers of genes and P values are shown. Statistical significance was assessed using a two-sided Fisher exact test with Benjamini–Hochberg correction for multiple comparisons. g and h, Transcription factor prediction from RNA-Seq using DEGs (g) and ATAC-Seq using DARs (h) that are downregulated by WSTF in HRasV12-induced senescence. i, IMR90 cells stably expressing HA-WSTF were treated with etoposide to induce senescence, and fractionated into cytoplasm and nucleus, then subjected to p65 immunoprecipitation and immunoblotting with indicated antibodies. Because endogenous WSTF is lost in senescence, HA-WSTF overexpressed senescent cells were used for co-IP. j, In vitro translated Flag-p65, HA-WSTF, and HA-SNF2H were subjected to Flag immunoprecipitation and immunoblotting with indicated antibodies. Note that p65 binds quantitatively more to WSTF than SNF2H. k and l, HeLa and MDA-MB-231 cells were subjected to p65 and WSTF IP at the endogenous level, followed by immunoblotting with indicated antibodies. The western blot experiments shown in i-l were repeated independently at least three times with similar results. a, Liver sections 6 days post injection were stained with NRasV12 and WSTF antibodies, followed by imaging. Note the expression level of WSTF was reduced in the GFP group and increased in the WSTF group. b, Bar graphs showing the percentage of WSTF-positive cells based on NRas positivity. Data presented are mean values from four randomly selected fields, with over 200 cells analysed. **** P value < 0.0001; unpaired two-tailed Student's t-test. c and e, Additional representative images of liver sections stained with NRasV12 and CD45 antibodies, imaged under a confocal microscopy. Scale bars are shown as indicated. d, RT-qPCR analyses of liver from the NRas/GFP and NRas/WSTF groups. The relative expression levels of pro-inflammatory genes and immune cell genes were measured. Data are presented as mean values with s.e.m. ; 10 mice in each group were used; P values were calculated by unpaired two-tailed Student's t-test and shown in the figure. f, Additional images of liver from NRasV12/GFP and NRasV12/WSTF groups 6 months post injection. Note while the GFP group did not develop liver tumour, all mice in the WSTF group developed liver tumours. g, CCLE analysis of WSTF and pro-inflammatory cytokine expression. Cell lines in the lowest 50% (WSTF-low, n = 171) and highest 50% (WSTF-high, n = 170) of WSTF expression were grouped, and the expression levels of selected inflammatory genes were compared. Box plots display the median (centre line), the 25th and 75th percentiles (box bounds), and the 10th and 90th percentiles (whiskers). Data points outside the whiskers are shown as individual dots. Statistical significance was assessed using an unpaired two-tailed Welch's t-test (assuming unequal variances). See Materials and Methods for additional details. h and i, RT-qPCR analyses of IL6, IL8 and WSTF in sh-NTC, sh-WSTF #1 (h) and sh-WSTF #2 (i) in cancer cells. Results were normalized to Lamin A/C and presented as mean values with s.d. ; n = 3 biologically independent samples; exact P values are shown; n.s. non-significant; unpaired two-tailed Student's t-test. j, HeLa and MDA-MB-231 were stably expressed with non-targeting control (sh-NTC) or sh-WSTF. The cells were harvested and analysed by immunoblotting. The western blot experiments shown in j were repeated independently at least three times with similar results. a, IMR90 were treated with TNFα (20 ng/mL) and THP-1 were treated with LPS (1 μg/mL) for 24 h. b, IMR90 treated with IFNβ were analysed by RT-qPCR. ; n = 3 biologically independent samples; exact P values are shown; n.s. c-d, cGAS knockout BJ fibroblasts were chronically transfected by ISD (5 μg/mL) (c); IMR90 with MAVS knockdown were chronically transfected by poly I:C (0.25 μg/mL). e-f, BiFC assay in acute and chronic inflammation. g, mCherry-GFP-WSTF ARPE19 were transfected with poly I:C (0.25 μg/mL) chronically. h, IMR90 with FIP200 knockdown were transfected by ISD chronically. l, ATMi rescued WSTF loss in IMR90 under chronic poly I:C or ISD transfection. n, Sequences of synthesized peptides. o-p, GST-GABARAP pulldown of in vitro translated WSTF with 20 μg/mL competing peptides. q, PHYRE2 prediction of the hydrophobic helix of WSTF aligned with Cav2.2. GABARAP-binding region is highlighted, forming a hydrophobic helix with high confidence (top). ClusPro modelling of GABARAP UDS binding to WSTF helix (bottom). t, Co-IP of IMR90 whole cell extracts treated with NLS-cell penetrating peptides in cultured media for 48 h. u, Fractionation control for Fig. v-w, IMR90 undergoing chronic poly I:C or ISD transfection were treated with NLS-CPP-peptides. ; n = 3 biologically independent samples; exact P values are shown; n.s. x, NLS-CPP-WSTF rescued WSTF in senescent IMR90. y, Sequences of mouse peptides. z-aa, Ms NLS-CPP-WSTF rescued WSTF in senescent AML12 and reduced the SASP gene expression. P values were calculated using one-way ANOVA coupled with Dunnett's post hoc test. The experiments shown in a, c, d-m, o, p, t, u, x, z were repeated independently at least three times with similar results. a and b, Related to Fig. 4a and b, additional images of WSTF staining of liver samples from control or MASH patients in cohort 1 and 2. c, Immunoblotting analysis of liver samples from mice fed on a chow diet or on the MCDE diet. d, Liver sections from chow diet or MCDE-treated mice were stained with WSTF and LAMP1 antibodies and analysed by a confocal microscopy. The colocalization of WSTF and LAMP1 is highlighted by arrows. WSTF signals in the MCDE group were deliberately overexposed to show the colocalization with LAMP1. e, Experimental design for establishing MASH in mice using MCDE model and scheme for administering peptides. f, Representative immunoblotting analysis of liver samples from mice fed on a chow diet or on the MCDE diet with NLS-CPP treatments. g, Quantifications of f (n = 8 animal samples). Mean and P values are shown; statistical analysis was conducted using unpaired two-tailed Student's t-test. h, Representative images of IHC staining for WSTF, F4/80, α-SMA, and Sirius Red. i, Quantifications of IHC staining for h. n = 8 animal samples; data were from four randomly selected fields for each mouse. Results shown are the mean values; P values are shown as indicated, calculated by one-way ANOVA coupled with Dunnett's post hoc test. j, RT-qPCR analyses of liver samples. The relative expression levels of pro-inflammatory genes and macrophage genes were measured. Results shown are the mean values; 8 mice in each group were used; P values are shown as indicated, calculated by one-way ANOVA coupled with Dunnett's post hoc test. k, Liver sections were stained with an IL6 antibody then imaged using a confocal microscopy. a, Scheme of synthesized fragments derived from WSTF sequences. b, Flag-tagged GABARAP and 3NLS-tagged human WSTF fragment or vector control were transfected in HEK293T, and were subjected to Flag co-IP. d, co-IP in mouse BMK. e, Senescent primary MEFs were analysed by immunoblotting. f, Liver sections 1 month post retro-orbital injection with AAV2/8 were stained with HA antibody. g, Experimental design for establishing MASH in mice using the CDAA-HFD model and scheme for administering WSTF fragments. i, Quantification of h. Mean and P values are shown. n (chow)=4, n (CDAA-HFD) = 5 for both 3NLS-mWSTF 407-553-HA and 3NLS-GFP-HA; Data are presented as mean values with s.d. ; P values were calculated by one-way ANOVA coupled with Dunnett's post hoc test; * P < 0.05; ** P < 0.01. j, Representative images of IHC staining for F4/80, α-SMA, and Sirius Red. Quantification is shown in k, n = 8; data were from four randomly selected fields for each mouse. Results are presented as mean with P values. l, RT-qPCR analyses of liver samples. The relative expression levels of pro-inflammatory genes and macrophage genes were measured. Results shown are the mean values; 7 mice in the chow diet group and 8 mice each in AAV2/8 3NLS-mPP6-HA and 3NLS-GFP-HA groups. Mean and P values are shown. n (chow)=4, n (GAN) = 5 for both 3NLS-mWSTF 407-553-HA and 3NLS-GFP-HA; Data are presented as mean values with s.d. ; P values were calculated by one-way ANOVA coupled with Dunnett's post hoc test; exact P values are shown. n, Liver samples from mice fed on a GAN diet with AAV2/8 3NLS-GFP-HA or 3NLS-mWSTF 407-553-HA treatments. Samples were stained with HA and IL6 antibodies, then imaged using a confocal microscopy. a and b, IHC analyses for IL6, IL8, and MMP13 using ex vivo cultured cartilage explants. Quantification is shown in b. n = 3; data were from four randomly selected fields for each human sample. Results are presented as mean values with s.e.m. Statistical analysis was conducted using two-tailed Student's t-test. c, Scheme of experimental design to test NLS-CPPs in chondrocytes of the damaged articular cartilage explants from OA patients. d, Immunoblotting analysis of cytokine levels in the culture media of chondrocytes using indicated antibodies. e and f, MMP13 staining of the cultured chondrocytes. Quantification is shown in f. Results are presented as mean values with s.d. and exact P values; n = 3 biologically independent samples; data were from four randomly selected fields for each sample. Statistical analysis was conducted using unpaired two-tailed Student's t-test. g, Mice cartilages undergone sham or OA surgery were stained with WSTF and LAMP1 antibodies and analysed by a confocal microscopy. WSTF signals in the OA group were deliberately overexposed to show the colocalization with LAMP1, highlighted by arrows. 5d, additional representative images of IHC for cytokine staining in cartilages using indicated antibodies. 5f, representative images of safranin O staining to evaluate cartilage damage. Uncropped immunoblot images for the main figures with molecular mass markers indicated. Uncropped immunoblot images for the extended data figures with molecular mass markers indicated. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Wang, Y., Eapen, V.V., Liang, Y. et al. WSTF nuclear autophagy regulates chronic but not acute inflammation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09189-3'>Ancient DNA reveals the prehistory of the Uralic and Yeniseian peoples</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'nature.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 15:41:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples, but much of their history is poorly understood. In particular, the genomic formation of populations that speak Uralic and Yeniseian languages today is unknown. Here, by generating genome-wide data for 180 ancient individuals spanning this region, we show that the Early-to-Mid-Holocene hunter-gatherers harboured a continuous gradient of ancestry from fully European-related in the Baltic, to fully East Asian-related in the Transbaikal. Contemporaneous groups in Northeast Siberia were off-gradient and descended from a population that was the primary source for Native Americans, which then mixed with populations of Inland East Asia and the Amur River Basin to produce two populations whose expansion coincided with the collapse of pre-Bronze Age population structure. Ancestry from the first population, Cis-Baikal Late Neolithic–Bronze Age (Cisbaikal_LNBA), is associated with Yeniseian-speaking groups and those that admixed with them, and ancestry from the second, Yakutia Late Neolithic–Bronze Age (Yakutia_LNBA), is associated with migrations of prehistoric Uralic speakers. We show that Yakutia_LNBA first dispersed westwards from the Lena River Basin around 4,000 years ago into the Altai-Sayan region and into West Siberian communities associated with Seima-Turbino metallurgy—a suite of advanced bronze casting techniques that expanded explosively from the Altai1. The 16 Seima-Turbino period individuals were diverse in their ancestry, also harbouring DNA from Indo-Iranian-associated pastoralists and from a range of hunter-gatherer groups. Thus, both cultural transmission and migration were key to the Seima-Turbino phenomenon, which was involved in the initial spread of early Uralic-speaking communities. This is a preview of subscription content, access via your institution Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout The newly reported data in this study can be obtained from the European Nucleotide Archive under accession number PRJEB86428. Bam files of aligned reads for the 180 newly published ancient individuals and 15 newly reported whole-genome sequences from a subset of these individuals can be found at secondary accession ERP169776, and the genotypes that we used for analysis can be found at secondary accession ERZ25719453. Genotype files in PLINK format for the 229 modern individuals for whom we newly report SNP array can be found at secondary accession ERZ26790638. All maps in the main text and in the Supplementary Information were created using ArcGIS 10.6.1 and QGIS 3.40.6. Figures presenting genetic data were created using Rstudio running R version 4.4.1, and further edited in Adobe Illustrator version 28. Archaeological images in Supplementary Information, section 3 were edited in Adobe Photoshop 25.12.2 and Adobe Acrobat 2025.001.20458. Genes reveal traces of common recent demographic history for most of the Uralic-speaking populations. Ancient Fennoscandian genomes reveal origin and spread of Siberian ancestry in Europe. The arrival of Siberian ancestry connecting the Eastern Baltic to Uralic speakers further east. Reich, D. et al. Reconstructing Native American population history. Flegontov, P. et al. Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America. Nielsen, S. V. et al. Bayesian inference of admixture graphs on Native American and Arctic populations. Flegontov, P. et al. Genomic study of the Ket: a Paleo-Eskimo-related ethnic group with significant ancient North Eurasian ancestry. The genetic history of admixture across inner Eurasia. Kidd, K. K. et al. North Asian population relationships in a global context. Svyatko, S. V. et al. Freshwater reservoir effects in archaeological contexts of Siberia and the Eurasian Steppe. Kılınç, G. M. et al. Human population dynamics and Yersinia pestis in ancient northeast Asia. Yu, H. et al. Paleolithic to Bronze Age Siberians reveal connections with first Americans and across Eurasia. Harney, É., Patterson, N., Reich, D. & Wakeley, J. Assessing the performance of qpAdm: a statistical tool for studying population admixture. Performance of qpAdm-based screens for genetic admixture on graph-shaped histories and stepping-stone landscapes. Davidson, R. et al. Allelic bias when performing in-solution enrichment of ancient human DNA. Grebenyuk, P. S., Fedorchenko, A. Y., Dyakonov, V. M., Lebedintsev, A. I. A. in Humans in the Siberian Landscapes: Ethnocultural Dynamics and Interaction with Nature and Space (eds Bocharnikov, V. N. & Steblyanskaya, A. N.) 89–133 (Springer, 2022). Ancient DNA indicates human population shifts and admixture in northern and southern China. Moreno-Mayar, J. V. et al. Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans. The genomic history of southeastern Europe. Massive migration from the steppe was a source for Indo-European languages in Europe. Raghavan, M. et al. Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. de Barros Damgaard, P. et al. The first horse herders and the impact of early Bronze Age steppe expansions into Asia. Saag, L. et al. Genetic ancestry changes in Stone to Bronze Age transition in the East European plain. The formation of human populations in South and Central Asia. Posth, C. et al. Palaeogenomics of Upper Palaeolithic to Neolithic European hunter-gatherers. Allentoft, M. E. et al. Population genomics of Bronze Age Eurasia. de Barros Damgaard, P. et al. 137 ancient human genomes from across the Eurasian steppes. Ancient genomes suggest the eastern Pontic-Caspian steppe as the source of western Iron Age nomads. Wei, L.-H. et al. Paternal origin of Paleo-Indians in Siberia: insights from Y-chromosome sequences. Karmin, M. et al. A recent bottleneck of Y chromosome diversity coincides with a global change in culture. Investigating the effects of prehistoric migrations in Siberia: genetic variation and the origins of Yakuts. Chernykh, E. N. & Kuz'minykh, S. V. Drevnyaya metallurgiya Severnoy Evrazii (Seiminsko-Turbinskiy fenomen) (Nauka, 1989). Marchenko, Z. V., Svyatko, S. V., Molodin, V. I., Grishin, A. E. & Rykun, M. P. Radiocarbon chronology of complexes with Seima-Turbino type objects (Bronze Age) in Southwestern Siberia. The origins of metallurgy in China. Chernykh, E. N. in Nomadic Cultures in the Mega-Structure of the Eurasian World (eds Savinetskaya, I & Hommel, P. N.) 234–249 (Academic Studies, 2017). Ilumäe, A.-M. et al. Human Y chromosome haplogroup N: a non-trivial time-resolved phylogeography that cuts across language families. Kuzminykh, S. V. Seima-Turbino transcultural phenomenon: migration or diffusion of technology. In Mobility and Migration: Concepts, Methods, Results: Programme and Abstracts of the V International Scientific Symposium (eds Molodin, V. I. Makarov, N. P. Khronologiya i periodizatsiya epokhi Neolita i Bronzy Krasnoyarskoy lesostepi [The chronology and periodization of the Neolithic and Bronze Krasnoyarsk forest]. Bronze age Northern Eurasian genetics in the context of development of metallurgy and Siberian ancestry. Kristiansen, K. The Rise of Bronze Age Peripheries and the Expansion of International Trade 1950–1100 bc. Powell, W. et al. Tin from Uluburun shipwreck shows small-scale commodity exchange fueled continental tin supply across Late Bronze Age Eurasia. Singh, M. & Glowacki, L. Human social organization during the Late Pleistocene: Beyond the nomadic-egalitarian model. & Nordqvist, K. Made for exchange: the Russian Karelian lithic industry and hunter-fisher-gatherer exchange networks in prehistoric north-eastern Europe. The world's oldest-known promontory fort: Amnya and the acceleration of hunter-gatherer diversity in Siberia 8000 years ago. Holopainen, S. Indo-Iranian Borrowings in Uralic: Critical Overview of Sound Substitutions and Distribution Criterion. Drastic demographic events triggered the Uralic spread. Kumar, V. et al. Genetic continuity of Bronze Age ancestry with increased Steppe-related ancestry in Late Iron Age Uzbekistan. Guarino-Vignon, P., Marchi, N., Bendezu-Sarmiento, J., Heyer, E. & Bon, C. Genetic continuity of Indo-Iranian speakers since the Iron Age in southern Central Asia. Kovtun, I. V. Predystoriya Indoariyskoy Mifologii (Aziya-Print, 2013). Häkkinen, J. in Iter Polyphonicum Multilinguae (eds Hyytiäinen, T. et al.) 91–101 (2012). Buchhorn, M. et al. Copernicus Global Land Service: land cover 100m: collection 3: epoch 2019: Globe (V3.0.1) [Data set]. & Meyer, M. Extraction of highly degraded DNA from ancient bones, teeth and sediments for high-throughput sequencing. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. Briggs, A. W. & Heyn, P. in Ancient DNA. Rohland, N., Harney, E., Mallick, S., Nordenfelt, S. & Reich, D. Partial uracil–DNA–glycosylase treatment for screening of ancient DNA. Gansauge, M.-T., Aximu-Petri, A., Nagel, S. & M MEYER, Manual and automated preparation of single-stranded DNA libraries for the sequencing of DNA from ancient biological remains and other sources of highly degraded DNA. Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. Maricic, T., Whitten, M. & Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. The UCSC Genome Browser Database: 2016 update. Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. Behar, D. M. et al. A “Copernican” reassessment of the human mitochondrial DNA tree from its root. Fu, Q. et al. A revised timescale for human evolution based on ancient mitochondrial genomes. & Nielsen, R. ANGSD: analysis of next generation sequencing data. Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. The genetic history of the Southern Arc: a bridge between West Asia and Europe. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Patterson, N., Price, A. L. & Reich, D. Population structure and eigenanalysis. Ancient admixture in human history. Maier, R., Flegontov, P., Flegontova, O., Işıldak, U., Changmai, P. & Reich, D. On the limits of fitting complex models of population history to f-statistics. Kennett, D. J. et al. Archaeogenomic evidence reveals prehistoric matrilineal dynasty. Van de Loosdrecht, M. et al. Pleistocene North African genomes link near Eastern and sub-Saharan African human populations. Monroy Kuhn, J. M., Jakobsson, M. & Günther, T. Estimating genetic kin relationships in prehistoric populations. We dedicate this paper to Oleg Balanovsky, who had a leading role in the collection of present-day samples newly reported in this study, and who would have been an author had he not died in 2021. The authors thank N. Adamski, R. Bernardos, N. Bradman, A. Chizhevsky, M. Ferry, E. Idrisov, J. Kidd, E. Kostyleva, S. Kuz'minykh, K. Mandl, P. Nymadawa, O. Poshekhonova, H. Ringbauer, L. Saroyants, K. Stewardson, S. Tur, Y. Yusupov and Z. Zhang for wet laboratory or bioinformatic support, providing permission to analyse samples that they shared or critical comments. We acknowledge E. Besprozvanny, T. Chikisheva, A. Chizhevskiy, O. Goryunova, E. Kostyleva, N. Kungurova, D. Maslyuzhenko, A. Polevodov, A. Shalapinin, G. Sinitsyna, Z. Trufanova and V. Zakh for providing permissions to use their previously published figures in the supplementary information. acknowledges support from the Russian Science Foundation (project 22-18-00470). acknowledges support from the Collaborative Research Grants Program 091019CRP2119 to Nazarbayev University. was conducted within the framework of the scientific research programme of the Diamond and Precious Metals Geology Institute, Siberian Branch of the Russian Academy of Science (project FUFG-2024-0005). was carried out within the State Assignment FWRZ-2021–0006. was supported by the IHMC RAS research programme (FMZF-2025-0008). is supported by ERC Horizon 2020 research and innovation programme grant agreements: 951385 (COREX), 865515 (SUSTAIN), 324202 (NeoMilk), 788616 (YMPACT), and by Wellcome Senior Research Fellowship Grant 100719/Z/12/Z. was supported by BBSRC (BB/L009382/1), Wellcome Trust and Royal Society (098386/Z/12/Z, 224575/Z/21/Z). We thank the Museum of the Institute of Plant and Animal Ecology UB RAS for sharing samples. was supported by the Czech Science Foundation (project 21-27624S) and the EU Operational Program Just Transition (‘LERCO—Life Environment Research Center Ostrava', project CZ.10.03.01/00/22_003/0000003). was supported by the Czech Ministry of Education, Youth and Sports (programme ERC CZ, project LL2103). were supported by John Templeton Foundation grant 61220. were supported by gifts from Jean-Francois Clin. was supported by National Institutes of Health grant HG012287 and by the Allen Discovery Center programme, a Paul G. Allen Frontiers Group advised programme of the Paul G. Allen Family Foundation, and is an Investigator of the Howard Hughes Medical Institute. These authors contributed equally: Tian Chen Zeng, Leonid A. Vyazov, Alexander Kim Department of Human Evolutionary Biology, Harvard University, Cambridge, MA, USA Tian Chen Zeng, Leonid A. Vyazov, Pavel Flegontov, Kendra Sirak, Robert Maier, Iosif Lazaridis, Ali Akbari, Aisling Kearns, Megan Michel, Iñigo Olalde, Swapan Mallick, Nadin Rohland & David Reich Department of Genetics, Harvard Medical School, Boston, MA, USA Alexander Kim, Kendra Sirak, Robert Maier, Iosif Lazaridis, Ali Akbari, Kim Callan, Lora Iliev, Aisling Kearns, Matthew Mah, Adam Micco, Megan Michel, Fatma Zalzala, Swapan Mallick, Nadin Rohland & David Reich Department of Anthropology, Harvard University, Cambridge, MA, USA Institute of Parasitology, Biology Centre CAS, České Budějovice, Czech Republic Department of Anthropology, Washington University in St Louis, St Louis, MO, USA Spatial Analysis, Interpretation, and Exploration Laboratory, Washington University in St. Louis, St. Louis, MO, USA Department of Archaeology, Ethnography and Museology, Altai State University, Barnaul, Russia Alexey A. Tishkin, Yaroslav V. Frolov, Sergey P. Grushin & Yurii F. Kiryushin Historical, Ecological and Cultural Association ‘Povolzhye', Samara, Russia Diamond and Precious Metals Geology Institute, Siberian Branch of the Russian Academy of Sciences, Yakutsk, Russia Anatoly P. Derevianko, Viktor M. Dyakonov, Elena N. Solovyova, Aleksandr D. Stepanov, Aleksei A. Timoshchenko & Anton V. Vybornov Dmitry N. Enshin, Svetlana N. Skochina, Sergey M. Slepchenko & Konstantin N. Solodovnikov Department of Recreational Geography, Service, Tourism and Hospitality, Institute of Geography, Altai State University, Barnaul, Russia Ural Federal University, Yekaterinburg, Russia LLC ‘Archaeology of the East European Plain', Moscow, Russia Institute of Natural Sciences, M. K. Ammosov North-Eastern Federal University, Yakutsk, Russia Siberian State University of Physical Education and Sport, Omsk, Russia Arctic Research Center of Sakha Republic, Yakutia, Russia Stanislav Dryomov, Elena Starikovskaya & Rem Sukernik Department of Genetics, Yale Medical School, New Haven, CT, USA Department of Archaeogenetics, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany Laboratory of Archaeogenetics, Nazarbayev University, Astana, Kazakhstan Laboratory of Human Genetics, National Center for Biotechnology, Astana, Kazakhstan Kim Callan, Lora Iliev, Matthew Mah, Adam Micco, Megan Michel, Fatma Zalzala, Swapan Mallick & David Reich Olivia Cheronet, Daniel Fernandes & Ron Pinhasi Olivia Cheronet, Kadir Toykan Özdoğan & Ron Pinhasi Servizio di Bioarcheologia, Museo delle Civiltà, Rome, Italy Broad Institute of MIT and Harvard, Cambridge, MA, USA Matthew Mah, Swapan Mallick, Nadin Rohland & David Reich BIOMICs Research Group, Department of Zoology and Animal Cell Biology, University of the Basque Country UPV/EHU, Vitoria-Gasteiz, Spain Ikerbasque-Basque Foundation of Science, Bilbao, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the manuscript and supplementary materials with input from all co-authors. supervised different aspects of the study. contributed to the framing and interpretation of results. edited archaeological information with input from D.R. contributed extensively to sample procurement. contributed genetic data from present-day people. carried out bioinformatic data processing. and M. Michel carried out wet laboratory work. Correspondence to Tian Chen Zeng, Leonid A. Vyazov, Alexander Kim, Ron Pinhasi, Vagheesh M. Narasimhan or David Reich. The authors declare no competing interests. Nature thanks Henny Piezonka, Edward Vajda and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. These include all sites 1) whose samples fall on the NEAHG cline, 2) whose samples fall in the Cisbaikal_LNBA cluster or are admixed with it, 3) whose samples fall in the Yakutia_LNBA cluster or are admixed with it, 4) whose samples are a part of the ten-population East Siberian transect described in our qpAdm modelling, and 5) whose samples are from Seima-Turbino period individuals. Each site is represented by a pie chart, whose size is proportional to the number of individuals from that site; the white fraction represents previously-published samples, and the black newly-published samples. Our sampling fills geographic and temporal lacunae. Temporal and geographic disposition of cultures from the Mesolithic to the Late Bronze and Iron Ages across Northern Eurasia. Sites whose samples are analyzed in our paper are highlighted in darker boxes, within containing boxes indicating archaeological cultures. Sites whose colors are darker are those that we believe are most securely dated (based on radiocarbon, isotopic, and archaeological evidence). To illuminate the role that levels of ANE ancestry plays in generating variation among the populations we analyze, we use as a basis for another projection 71 shotgun-sequenced ancient individuals from across Eurasia, of which a large proportion are enriched in ANE ancestry and fall outside the range of present-day variation (e.g. individuals from populations like Tyumen_HG.SG or Kazakhstan_Botai.SG; for full list, see Supplementary Information section 4). The North Eurasian Hunter-Gatherer cline forms a curved arc stretching from EHG populations to present-day East Asians; the center of the arc dominated by populations rich in ANE ancestry is moved toward the positive direction in PC2. Clines formed by later Inner Asian populations, such as present-day Uralic, Turkic, and Mongolic speakers, as well as Late Bronze Age and Iron Age steppe populations such as Scythians and Sarmatians, are distinguished from the NEAHG cline by their much lower values along PC2, suggesting a much lower level of ANE ancestry. This PCA shows that populations along the NEAHG cline, remaining stable for many millennia, were substantially outside the range of present-day genetic variation in Northern Eurasia. To further uncover possible structure among the East Asian ancestries within the populations that we analyze, we constructed a third PCA, using as a basis 37 East Asian present-day populations that have minimal West Eurasian admixture, and a single West Eurasian population (Norwegian), all genotyped on the Affymetrix Human Origins array (for a full list of populations analyzed, refer to Supplementary Information section 4). We projected all other shotgun-sequenced and hybridization-captured ancient and present-day individuals onto this basis. Once again, the North Eurasian Hunter-Gatherer cline forms a curved arc stretching from West Eurasian populations to present-day East Asians, with the center of the arc deflected toward the AG3 individual. East Asian populations are now differentiated along PC2, with Southeast Asians and East Asian agriculturalists taking on especially negative values along that dimension; populations from the Amur River Basin taking on intermediate values; then populations on the Mongolian Plateau and surrounding areas. A large gap separates these populations from Yakutia_LNBA and Russia_Tatarka_BA, which take on very positive values along PC2, close to present-day Nganasans and a genetically very similar Iron-Age individual from Yakutia who clusters with Nganasans in the previous two PCAs (Yakutia_IA.SG; also see Extended Data Fig. As one moves East along the NEAHG cline, their positions along PC2 tend to converge to the values found among populations of the Mongolian Plateau. In contrast, the Dzhilinda1_M_N_8.4 kya and Kolyma_M_10.1 kya individuals, and the Syalakh_Belkachi, Yakutia_LNBA and Russia_Tatarka_BA populations do not fall on the NEAHG cline and are shifted in the positive direction on PC2, toward the positions occupied by Nganasans, Beringian populations, and Native Americans. Lastly, Uralic populations possess the most positive values among PC2 when compared to Turkic, Mongolic and Tungusic populations. To understand structure among NEAHG populations and non-NEAHG Siberians, we constructed two PCAs with ancient individuals including all individuals from the NEAHG cline, ancient non-NEAHG Siberians, and a selection of ancient Beringians and Native Americans. Notably, all these populations possess combinations of only WHG, EHG, ANE and East Asian ancestries. No individuals were projected in these PCAs. The first PCA (Extended Data Fig. 6b) includes only individuals East of the Altai mountains. (A) In the first PCA we highlight several patterns. 1) the North Eurasian Hunter-Gatherer cline forms a curved arc stretching from West Eurasian populations to East Asian populations along PC1 and PC2. Populations rich in East Asian ancestry are differentiated along PC3, with individuals and populations within or closely related to the Cisbaikal_LNBA cluster having the most positive values, followed by those in the Transbaikal_EMN cluster and populations of the Mongolian Plateau, followed by individuals and populations in the Yakutia_LNBA cluster, followed by those from the Amur River Basin, followed by populations from the Bering Straits and the Americas. Notably, all individuals along the NEAHG cline, including individuals rich in East Asian ancestry (e.g. Cisbaikal_EN, Transbaikal_EMN, and all NEAHG individuals from the Krasnoyarsk region) form a straight line in PC3, suggesting a constant source of East Asian ancestry at the East Asian terminus of the NEAHG cline. 2) Khaiyrgas_16.7 kya occupies a central position among the other groups rich in East Asian ancestry in East Siberia, Beringia and the Americas, suggesting a lack of shared drift with later populations of the Bering region or the Americas. The situation is different for later populations: Kolyma_M_10.1 kya falls among ancient Beringian populations, while the more East Asian-admixed Ust-Kyakhta_14 kya and Dzhilinda1_M_N_8.4kya occupy a position in between Syalakh-Belkachi and ancient Bering Straits populations, with the even more East Asian-admixed Syalakh-Belkachi population showing even less of this displacement towards ancient Bering Straits populations. (B) We find a similar pattern in the second PCA, except with an opposite ordering of the clusters along PC3. Our results suggest that the distinctions we discover between groupings produced by the clustering analyses in Supplementary Information Section 6 can be recovered in PCA analyses aimed at recovering fine-scale structure, despite underlying similarities in deep ancestry in populations in East Siberia, Beringia, and the Americas—all the products of admixture between ANE and East Asian ancestry. Panel A shows the widespread distribution of individuals with Ancient Paleosiberian (APS) ancestry in Siberia before the Holocene, >10 kya. Panel B shows the formation of the NEAHG cline by ~10 kya, and the formation of the population on its eastern terminus (Transbaikal_EMN) through admixture between Amur River and Inland East Asian ancestries. Panel C shows the emergence of Cisbaikal_LNBA and Yakutia_LNBA in genetic turnovers in the Cis-Baikal and Northeastern Siberian regions in the Mid-Holocene, and the genetic diversity of Seima-Turbino period individuals ~4.0 kya. Panel D shows the genetic gradient between West Eurasian ancestry and Yakutia_LNBA formed by present-day Uralic populations, along with all locations from which present-day populations with Cisbaikal_LNBA ancestry were sampled (grey dots ringed with black), alongside the geographic locations of two late Bronze Age/early Iron Age individuals (grey dots ringed with yellow) with >90% Cisbaikal_LNBA ancestry. Details of populations created by the grouping procedure applied to individuals in Northeastern Siberia. Central Siberian populations from the Yenisei Basin (including Kets and South Siberian Turks) are highlighted in brown, while Arctic North American and Asian populations on either side of the Bering Straits populations are highlighted in blue. Bering Straits populations that are heavily European-admixed (Aleut and Yukagir_forest) are colored dark blue, while Samoyedic populations (Enets, Selkup, and Nganasan) are colored violet. Despite the similarity of the APS-rich populations in this comparison (all being admixtures between APS ancestry and East Asian ancestry), present-day groups of the Bering Straits are always closer to groups with “Route 2” APS ancestry (i.e., Kolyma_M_10.1 kya → Dzhilinda1_8.4 kya → Syalakh-Belkachi → Yakutia_LNBA), while Central Siberian populations of the Yenisei Basin are always closer to Cisbaikal_LNBA. For the version including a comparison with Ust-Kyakhta, refer to Supplementary Information Section 8; Figs. For details, refer to Supplementary Information Section 5. 1, containing the group/population labels of Fig. The codes are: ATN, Altaian; ATN_C, Altaian_Chelkan; BSK, Bashkir; BSM, Besermyan; BRY, Buryat; XNB_AR, China_AR_Xianbei_IA; CVS, Chuvash; DUR, Daur; DGN, Dolgan; DGX, Dongxiang; ENT, Enets; EST, Estonian; EVN, Even; EVN_E, Evenk_FarEast; EVN_T, Evenk_Transbaikal; FIN.SG, FIN.SG; LVL, Finland_Levanluhta; SAM, Finland_Saami_Modern.SG; FIN, Finnish; HZN, Hezhen; KLM, Kalmyk; KKP, Karakalpak; KRL, Karelian; KZK, Kazakh; KZK_C, Kazakh_China; BRL, Kazakhstan_Berel_IA; SARM_C, Kazakhstan_CaspianSteppe_Sarmatian; SARM_C.SG, Kazakhstan_CaspianSteppe_Sarmatian.SG; SAKA_K, Kazakhstan_CentralKazakhSteppe_Saka; SARM_K, Kazakhstan_CentralKazakhSteppe_Sarmatian.SG; KRK, Kazakhstan_Karakhanid.SG; KLK_1, Kazakhstan_Karluk_1.SG; KLK_2, Kazakhstan_Karluk_2.SG; KMK, Kazakhstan_Kimak.SG; KPC_1, Kazakhstan_Kipchak1.SG; KPC_2, Kazakhstan_Kipchak2.SG; SAKA_TS, Kazakhstan_Kyrgystan_TianShan_Saka; BRL_P, Kazakhstan_Pazyryk_Berel; TSM, Kazakhstan_Tasmola; SARM_W, Kazakhstan_WesternKazakhSteppe_Sarmatian; KET, Ket; KKS, Khakass; KKS_K, Khakass_Kachin; KMG, Khamnegan; KHT, Khanty; KOM, Komi_Zyrian; KRG_C, Kyrgyz_China; KRG_K, Kyrgyz_Kyrgyzstan; KRG_T, Kyrgyz_Tajikistan; TUR, Kyrgyzstan_Turk.SG; MNS, Mansi; MRI, Mari.SG; SCY, Moldova_Scythian; MGL, Mongol; MGA, Mongola; XNB_M, Mongolia_IA_Xianbei; MDV, Mordovian; NNI, Nanai; NGD, Negidal; NGS, Nganasan; NVH, Nivh; NGI_A, Nogai_Astrakhan; NGI_K, Nogai_Karachay_Cherkessia; NGI_S, Nogai_Stavropol; ORQ, Oroqen; ADB, Russia_Aldy_Bel; BLS, Russia_Bolshoy; MHE_1, Russia_EarlyMedieval_Heshui_Mohe_1; MHE_2, Russia_EarlyMedieval_Heshui_Mohe_2; SARM_S, Russia_EarlySarmatian_SouthernUrals.SG; KRS_o1, Russia_Karasuk_o1.SG; KRS_o, Russia_Karasuk_oRISE.SG; KRS, Russia_Karasuk.SG; SARM_L, Russia_LateSarmatian.SG; SARM_S.SG, Russia_MiddleSarmatian_SouthernUrals.SG; SARM, Russia_Sarmatian; SARM.SG, Russia_Sarmatian.SG; TGR, Russia_Tagar.SG; SAM.DG, Saami.DG; SKP, Selkup; SHR_K, Shor_Khakassia; SHR_M, Shor_Mountain; TTR_A, Tatar_Astrakhan; TTR_I, Tatar_Irtysh_Barabinsk.SG; TTR_K, Tatar_Kazan; TTR_M, Tatar_Mishar; TTR_S, Tatar_Siberian; TTR_Z, Tatar_Siberian_Zabolotniye; TTR_T, Tatar_Tomsk.SG; TTR_V, Tatar_Volga.SG; TDZ, Todzin; TFL, Tofalar; TBL, Tubalar; TKM, Turkmen; TVN, Tuvinian; UDM, Udmurt; SCY_U, Ukraine_Scythian; ULC, Ulchi; UYG, Uyghur; UZB, Uzbek; VPS, Veps; XIB, Xibo; YKT, Yakut; YKG_F, Yukagir_Forest; YKG_T, Yukagir_Tundra; KNY.SG, Russia_Yenisei_Krasnoyarsk_LBA.SG; KNY_o1.SG, Russia_Yenisei_Krasnoyarsk_LBA_o1.SG. China_Paleolithic includes the Tianyuan and Amur_River_33K genomes, “East Eurasian Population” is some population grouping in Siberia or Northeast Asia other than Yakutia_LNBA, and X are Admixed Inner Eurasian populations (AIEA populations) including ancient Central Asian nomads from the Late Bronze to Iron Age down to the Scytho-Sarmatian period, as well as modern or ancient populations that speak languages from the Yukaghiric, Yeniseian (Kets), Uralic, Turkic, Mongolic, Tungusic, and Nivkh language families. Modern Uralic-speaking populations, and ancient putatively Uralic-speaking populations uniformly prefer Yakutia_LNBA to other East Asian ancestries no matter the other population used in the comparison. Furthermore, at any level of admixture between East and West Eurasian ancestries, the population with the greatest affinity to Yakutia_LNBA is always a Uralic-speaking population. f4-statistics therefore highlight the connection between Uralic populations and Yakutia_LNBA ancestry over other sources of East Asian ancestry. Double-bladed dagger with a ring-shaped pommel, robbery find, unknown provenance (probable Omsk region or Rostovka). Double-bladed dagger with a horse figurine on the pommel, an accidental find near Shemonaikha, East Kazakhstan. Scapula-shaped celt with goat image, Rostovka, cluster of finds near burial 21. 6. double-bladed plate dagger with a double elk-head figurine pommel, an accidental find near Perm' (probably associated with the Turbino site). Top of staff with a horse figurine, an accidental find near Omsk. Single-ear long spearhead with a relief figurine of a Felidae predator (tiger or mountain leopard) on the socket (9a. This file includes sections 1–11, Supplementary Figs. It includes discussion of archaeological context, details of sample preparation, details of population genetic analysis using PCA, ADMIXTURE, qpAdm and other formal methods such as f4-statistics, relatedness analysis, uniparental markers, and also linguistic discussion and archaeological interpretation. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Zeng, T.C., Vyazov, L.A., Kim, A. et al. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09219-0'>The mutagenic forces shaping the genomes of lung cancer in never smokers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'nature.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 15:03:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Lung cancer in never smokers (LCINS) accounts for around 25% of all lung cancers1,2 and has been associated with exposure to second-hand tobacco smoke and air pollution in observational studies3,4,5. Here we use data from the Sherlock-Lung study to evaluate mutagenic exposures in LCINS by examining the cancer genomes of 871 treatment-naive individuals with lung cancer who had never smoked, from 28 geographical locations. KRAS mutations were 3.8 times more common in adenocarcinomas of never smokers from North America and Europe than in those from East Asia, whereas a higher prevalence of EGFR and TP53 mutations was observed in adenocarcinomas of never smokers from East Asia. Signature SBS40a, with unknown cause6, contributed the largest proportion of single base substitutions in adenocarcinomas, and was enriched in cases with EGFR mutations. Signature SBS22a, which is associated with exposure to aristolochic acid7,8, was observed almost exclusively in patients from Taiwan. Exposure to secondhand smoke was not associated with individual driver mutations or mutational signatures. By contrast, patients from regions with high levels of air pollution were more likely to have TP53 mutations and shorter telomeres. They also exhibited an increase in most types of mutations, including a 3.9-fold increase in signature SBS4, which has previously been linked with tobacco smoking9, and a 76% increase in the clock-like10 signature SBS5. A positive dose–response effect was observed with air-pollution levels, correlating with both a decrease in telomere length and an increase in somatic mutations, mainly attributed to signatures SBS4 and SBS5. Our results elucidate the diversity of mutational processes shaping the genomic landscape of lung cancer in never smokers. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout Detailed access information for the publicly available datasets is available in Supplementary Table 13. Data from the rnaturalearthdata v.1.0.0 (https://github.com/ropensci/rnaturalearthdata) were used to generate maps. Data on passive smoking and PM2.5 estimates of outdoor air pollution are available in Supplementary Table 14. Human reference genome GRCh38 was downloaded from the GATK resources at https://github.com/broadinstitute/gatk/blob/master/src/test/resources/large/Homo_sapiens_assembly38.fasta.gz. The WGS bioinformatics pipelines are available at https://github.com/xtmgah/Sherlock-Lung. The Battenberg SCNA calling algorithm is available at https://github.com/Wedge-lab/battenberg. Sun, S., Schiller, J. H. & Gazdar, A. F. Lung cancer in never smokers—a different disease. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. World Health Organization & International Agency for Research on Cancer. Tobacco Smoke and Involuntary Smoking: IARC Monographs on the Evaluation of Carcinogenic Risks to Humans Vol. Outdoor air pollution and cancer: an overview of the current evidence and public health recommendations. Ciabattini, M., Rizzello, E., Lucaroni, F., Palombi, L. & Boffetta, P. Systematic review and meta-analysis of recent high-quality studies on exposure to particulate matter and risk of lung cancer. Geographic variation of mutagenic exposures in kidney cancer genomes. Poon, S. L. et al. Genome-wide mutational signatures of aristolochic acid and its application as a screening tool. Hoang, M. L. et al. Mutational signature of aristolochic acid exposure as revealed by whole-exome sequencing. Alexandrov, L. B. et al. Signatures of mutational processes in human cancer. Alexandrov, L. B. et al. Clock-like mutational processes in human somatic cells. Proctor, R. N. Tobacco and the global lung cancer epidemic. Siegel, D. A., Fedewa, S. A., Henley, S. J., Pollack, L. A. Proportion of never smokers among men and women with lung cancer in 7 US states. Lui, N. S. et al. Sub-solid lung adenocarcinoma in Asian versus Caucasian patients: different biology but similar outcomes. Gaughan, E. M., Cryer, S. K., Yeap, B. Y., Jackman, D. M. & Costa, D. B. Family history of lung cancer in never smokers with non-small-cell lung cancer and its association with tumors harboring EGFR mutations. Toh, C. K. et al. Never-smokers with lung cancer: epidemiologic evidence of a distinct disease entity. Yano, T. et al. Never-smoking nonsmall cell lung cancer as a separate entity: clinicopathologic features and survival. Brennan, P. et al. High cumulative risk of lung cancer death among smokers and nonsmokers in Central and Eastern Europe. Wang, P., Sun, S., Lam, S. & Lockwood, W. W. New insights into the biology and development of lung cancer in never smokers—implications for early detection and treatment. Koh, G., Degasperi, A., Zou, X., Momen, S. & Nik-Zainal, S. Mutational signatures: emerging concepts, caveats and clinical applications. The repertoire of mutational signatures in human cancer. The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium. Pan-cancer analysis of whole genomes. Wang, X. et al. Association between smoking history and tumor mutation burden in advanced non-small cell lung cancer. Lee, J. J. et al. Tracing oncogene rearrangements in the mutational history of lung adenocarcinoma. Zhang, T. et al. Genomic and evolutionary classification of lung cancer in never smokers. Landi, M. T. et al. Tracing lung cancer risk factors through mutational signatures in never-smokers: the Sherlock-Lung study. Auton, A. et al. A global reference for human genetic variation. Islam, S. M. A. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. Sondka, Z. et al. COSMIC: a curated database of somatic variants and clinical data for cancer. Zou, X. et al. A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage. Steele, C. D. et al. Signatures of copy number alterations in human cancer. Everall, A. et al. Comprehensive repertoire of the chromosomal alteration and mutational signatures across 16 cancer types from 10,983 cancer patients. Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. Degasperi, A. et al. A practical framework and online tool for mutational signature analyses show inter-tissue variation and driver dependencies. Huang, K. L. et al. Pathogenic germline variants in 10,389 adult cancers. Nguyen, L., Martens, J. W. M., Van Hoeck, A. & Cuppen, E. Pan-cancer landscape of homologous recombination deficiency. Davies, H. et al. HRDetect is a predictor of BRCA1 and BRCA2 deficiency based on mutational signatures. Zhang, T. et al. Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition. Letouze, E. et al. Mutational signatures reveal the dynamic interplay of risk factors and cellular processes during liver tumorigenesis. Fujimoto, A. et al. Whole-genome mutational landscape and characterization of noncoding and structural mutations in liver cancer. & Harris, R. S. APOBEC enzymes: mutagenic fuel for cancer evolution and heterogeneity. et al. Proteogenomics of non-smoking lung cancer in East Asia delineates molecular signatures of pathogenesis and progression. Zhang, T. et al. APOBEC affects tumor evolution and age at onset of lung cancer in smokers. Morton, L. M. et al. Radiation-related genomic profile of papillary thyroid carcinoma after the Chernobyl accident. Lawson, A. R. J. et al. Extensive heterogeneity in somatic mutation and selection in the human bladder. Otlu, B. et al. Topography of mutational signatures in human cancer. Jamal-Hanjani, M. et al. Tracking the evolution of non-small-cell lung cancer. Distinct genomic landscape of lung adenocarcinoma from household use of smoky coal. Lung adenocarcinoma promotion by air pollutants. Monthly global estimates of fine particulate matter and their uncertainty. Passive smoking-induced mutagenesis as a promoter of lung carcinogenesis. Characterization of somatic mutations in air pollution-related lung cancer. Verifying the accuracy of self-reported smoking behavior in female volunteer soldiers. Environment And Genetics in Lung cancer Etiology (EAGLE) study: an integrative population-based case–control study of lung cancer. Bergmann, E. A., Chen, B. J., Arora, K., Vacic, V. & Zody, M. C. Conpair: concordance and contamination estimator for matched tumor–normal pairs. Pedersen, B. S. et al. Somalier: rapid relatedness estimation for cancer and germline studies using efficient genome sketches. The life history of 21 breast cancers. Boot, A. et al. In-depth characterization of the cisplatin mutational signature in human cell lines and in esophageal and liver tumors. Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Imielinski, M. et al. Mapping the hallmarks of lung adenocarcinoma with massively parallel sequencing. Lee, J. K. et al. Clonal history and genetic predictors of transformation into small-cell carcinomas from lung adenocarcinomas. The Cancer Genome Atlas Research Network. Comprehensive molecular profiling of lung adenocarcinoma. Carrot-Zhang, J. et al. Whole-genome characterization of lung adenocarcinomas lacking the RTK/RAS/RAF pathway. Pedersen, B. S. & Quinlan, A. R. Mosdepth: quick coverage calculation for genomes and exomes. Sadedin, S. P. & Oshlack, A. Bazam: a rapid method for read extraction and realignment of high-throughput sequencing data. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Freed, D., Pan, R. & Aldana, R. TNscope: accurate detection of somatic mutations with haplotype-based variant candidate detection and machine learning filtering. The mutational constraint spectrum quantified from variation in 141,456 humans. Ramos, A. H. et al. Oncotator: cancer variant annotation tool. Hasan, M. S., Wu, X., Watson, L. T. & Zhang, L. UPS-indel: a universal positioning system for indels. Mayakonda, A., Lin, D. C., Assenov, Y., Plass, C. & Koeffler, H. P. Maftools: efficient and comprehensive analysis of somatic variants in cancer. Jun, G. et al. Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data. Martinez-Jimenez, F. et al. A compendium of mutational cancer driver genes. Martincorena, I. et al. Universal patterns of selection in cancer and somatic tissues. The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers. Muiños, F., Martinez-Jimenez, F., Pich, O., Gonzalez-Perez, A. & Lopez-Bigas, N. In silico saturation mutagenesis of cancer genes. Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Yuan, K., Macintyre, G., Liu, W., PCAWG-11 working group & Markowetz, F. Ccube: a fast and robust method for estimating cancer cell fractions. Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Diverse mechanisms of somatic structural variations in human cancer genomes. Chen, X. et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. & Yang, L. Somatic structural variation signatures in pediatric brain tumors. Candidate cancer driver mutations in distal regulatory elements and long-range chromatin interaction networks. Ding, Z. et al. Estimating telomere length from whole genome sequence data. Alexandrov, L. B., Nik-Zainal, S., Wedge, D. C., Campbell, P. J. & Stratton, M. R. Deciphering signatures of mutational processes operative in human cancer. Bergstrom, E. N. et al. SigProfilerMatrixGenerator: a tool for visualizing and exploring patterns of small mutational events. Díaz-Gay, M. et al. Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment. Evaluating topography of mutational signatures with SigProfilerTopography. Controlling the false discovery rate—a practical and powerful approach to multiple testing. This work was supported by the Intramural Research Program of the National Cancer Institute, US NIH (project ZIACP101231 to M.T.L. ); by the NIH grants R01ES032547-01, R01CA269919-01 and 1U01CA290479-01 to L.B.A. laboratory was also supported by the Sanford Stem Cell Institute at the University of California San Diego. M.D.-G. and P.G.-G. were awarded fellowships within the Generación D initiative, Red.es, Ministerio para la Transformación Digital y de la Función Pública, for talent attraction (C005/24-ED CV1), funded by the European Union NextGenerationEU funds, through the Plan de Recuperación, Transformación y Resiliencia (PRTR). The funders had no roles in study design, data collection and analysis, decision to publish or preparation of the manuscript. We thank the study participants; P. Kraft for reading and commenting on the manuscript; and the staff at Westat for their assistance with collecting samples and corresponding clinical data. This work used the computational resources of the NIH HPC Biowulf cluster (http://hpc.nih.gov). These authors contributed equally: Marcos Díaz-Gay, Tongwu Zhang Department of Cellular and Molecular Medicine, University of California San Diego, La Jolla, CA, USA Marcos Díaz-Gay, Azhar Khandekar, Christopher D. Steele, Burçak Otlu, Shuvro P. Nandi, Raviteja Vangara, Erik N. Bergstrom, Mariya Kazachkova & Ludmil B. Alexandrov Department of Bioengineering, University of California San Diego, La Jolla, CA, USA Marcos Díaz-Gay, Azhar Khandekar, Christopher D. Steele, Burçak Otlu, Shuvro P. Nandi, Raviteja Vangara, Erik N. Bergstrom, Mariya Kazachkova & Ludmil B. Alexandrov Moores Cancer Center, University of California San Diego, La Jolla, CA, USA Marcos Díaz-Gay, Azhar Khandekar, Christopher D. Steele, Burçak Otlu, Shuvro P. Nandi, Raviteja Vangara, Erik N. Bergstrom, Mariya Kazachkova & Ludmil B. Alexandrov Digital Genomics Group, Cancer Genomics Program, Spanish National Cancer Research Center (CNIO), Madrid, Spain Tongwu Zhang, Phuc H. Hoang, Azhar Khandekar, Wei Zhao, Jian Sang, John P. McElderry, Caleb Hartman, Frank J. Colón-Matos, Mona Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, Jianxin Shi, Nathaniel Rothman, Qing Lan, Bin Zhu, Stephen J. Chanock, Jiyeon Choi & Maria Teresa Landi Department of Pathology and Laboratory Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA Marina K. Baine, William D. Travis & Soo-Ryum Yang Department of Pathology, Brigham and Women's Hospital, Boston, MA, USA Department of Health Informatics, Graduate School of Informatics, Middle East Technical University, Ankara, Turkey Cancer Evolution and Genome Instability Laboratory, Francis Crick Institute, London, UK Division of Pulmonary and Critical Care Medicine, Mayo Clinic, Rochester, MN, USA Biobanco IBSP-CV FISABIO, Valencia, Spain Red Valenciana de Biobancos FISABIO, Valencia, Spain Department of Cancer Epidemiology, H. Lee Moffitt Cancer Center and Research Institute, Tampa, FL, USA Thoracic Surgery, Roswell Park Comprehensive Cancer Center, Buffalo, NY, USA Department of Cancer Epidemiology and Primary Prevention, Maria Skłodowska-Curie National Research Institute of Oncology, Warsaw, Poland Blokhin National Medical Research Centre of Oncology, Moscow, Russia Anush Mukeria, Oxana Shangina & David Zaridze Institute of Public Health and Preventive Medicine, Second Faculty of Medicine, Charles University, Prague, Czech Republic Department of Oncology, Second Faculty of Medicine, Charles University and Motol University Hospital, Prague, Czech Republic Department of Occupational Health and Toxicology, National Center for Environmental Risk Monitoring, National Institute of Public Health, Bucharest, Romania International Organization for Cancer Prevention and Research (IOCPR), Belgrade, Serbia Department of Environmental Health, Harvard T.H. Chan School of Public Health, Boston, MA, USA Department of Medicine, Massachusetts General Hospital, Boston, MA, USA Genomic Epidemiology Branch, International Agency for Research on Cancer (IARC/WHO), Lyon, France IHU RespirERA, Biobank-BB-0033-0025, Côte d'Azur University, Nice, France Department of Mathematics, Harvard University, Cambridge, MA, USA Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, MA, USA Department of Clinical Sciences and Community Health, University of Milan, Milan, Italy Fondazione IRCCS Ca' Granda Ospedale Maggiore Policlinico, Milan, Italy Sanford Stem Cell Institute, University of California San Diego, La Jolla, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Ludmil B. Alexandrov or Maria Teresa Landi. is a co-founder, CSO, scientific advisory member and consultant for io9, has equity and receives income. The terms of this arrangement have been reviewed and approved by the University of California San Diego in accordance with its conflict-of-interest policies. is also a compensated member of the scientific advisory board of Inocras. 's spouse is an employee of Biotheranostics. declare a US provisional patent application filed with the University of California San Diego with serial number 63/269,033. also declares US provisional applications filed with the University of California San Diego with serial numbers 63/366,392, 63/289,601, 63/483,237, 63/412,835 and 63/492,348. is also an inventor of US patent 10,776,718 for source identification by non-negative matrix factorization. S.-R.Y has received consulting fees from AstraZeneca, Sanofi, Amgen, AbbVie and Sanofi, and speaking fees from AstraZeneca, Medscape, PRIME Education and Medical Learning Institute. The remaining authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, DBS, ID, CN and SV mutational signatures enrichment analysis with geographical regions. Horizontal lines marking statistically significant thresholds were included at 0.05 (dashed orange line) and 0.01 FDR value levels (dashed red line). Blue-coloured signatures were enriched in North American and European patients, whereas red-coloured signatures were enriched in East Asian patients. Statistical significance was evaluated using multivariable logistic regression models for geographical regions and adjusted by age, sex, and tumour purity. b, SBS, DBS, ID CN, and SV mutational signatures enrichment analysis with biological sexes. Blue-coloured signatures were enriched in males, whereas red-coloured signatures were enriched in females. Statistical significance was evaluated using multivariable logistic regression models for biological sex and adjusted by age, genetic ancestry, and tumour purity. c–e, Detail of the enrichment of EGFR (c), TP53 (d) and KRAS (e) driver mutations in North American and European versus East Asian LCINS adenocarcinoma cases. f, Driver mutations enrichment analysis with biological sexes. Blue-coloured genes were enriched in males, whereas red-coloured genes were enriched in females. Statistical significance was evaluated using multivariable logistic regression models for biological sex and adjusted by age, genetic ancestry, and tumour purity. g, Quantification of the tumour mutational burden for TP53 wild-type and mutant tumours across EGFR mutation status (n = 271 TP53 wild-type EGFR wild-type, n = 241 TP53 wild-type EGFR mutant, n = 81 TP53 mutant EGFR wild-type, n = 144 TP53 mutant EGFR mutant). Statistical significance was evaluated using a multivariable linear regression model for EGFR mutation status and adjusted by age, sex, ancestry, and tumour purity. h, SBS, DBS, ID, CN and SV mutational signatures enrichment analysis with EGFR mutation status. Blue-coloured signatures were enriched in EGFR mutant tumours, whereas red-coloured signatures were enriched in EGFR wild-type tumours. Statistical significance was evaluated using multivariable logistic regression models for EGFR mutation status and adjusted by age, sex, genetic ancestry, and tumour purity. a, Differences between smoker and never-smoker lung cancer cases across SBS signatures. Volcano plot (top) indicating the enrichment of SBS signature prevalence in never-smokers (left) and smokers (right) with lung cancer. Statistically significant enrichments were evaluated using multivariable logistic regression models for smoking status and adjusted by age, sex, histology, genetic ancestry, and tumour purity. Firth's bias-reduced logistic regressions were used for regression presenting complete or quasi-complete separation. P-values were adjusted for multiple comparisons based on the total number of mutational signatures considered, and adjusted p-values were reported as FDR values. Horizontal lines marking statistically significant thresholds were included at 0.05 (dashed orange line) and 0.01 FDR levels (dashed red line). Bar plot (bottom) indicating prevalence by smoking history. b, Tumour mutational burden differences between SBS4-positive (n = 56) and negative (n = 815) LCINS tumours for SBS, DBS, ID, CN segments, and SV events. Statistical significance was evaluated using two-sided Wilcoxon rank sum tests. c–e, Mutational signature landscape for SBS (c), DBS (d) and ID (e) mutation types, including absolute and relative number of mutations assigned to each mutational signature, unsupervised clustering based on the signature contributions, and sample-level annotations of sex, genetic ancestry, passive smoking, and accuracy of signature reconstruction based on cosine similarity. f, Driver mutations landscape, including different types of genomic alterations, as well as sample-level annotations of sex, genetic ancestry, histology, and tumour purity. g, Enrichment of EGFR p.L858R hotspot driver mutations in SBS4-positive tumours from never-smokers compared to smokers using multivariable logistic regressions considering clinical and epidemiological covariates, including age, sex, genetic ancestry, histology, and tumour purity (n = 5 mutated non-smoker cases, n = 1 mutated smoker case, n = 51 non-smoker wild-type cases, n = 301 smoker wild-type tumours). a,b, Distribution of SBS4 mutations with replication timing in our cohort of never-smokers (a) and in the smokers from the PCAWG cohort (b). Data are separated into deciles, with each segment harbouring 10% of the observed replication time signal in the x-axis, and the normalized mutational density displayed in the y-axis. Black dashed lines represent the behaviour of simulated mutations. c,d, Association of SBS4 mutations with nucleosome occupancy in never-smokers (c) and smokers (d). The solid blue line represents real somatic mutations, whereas the dashed grey line indicates the distribution of simulated mutations. a–e, Differences in DBS, ID, CN, and SV burden using univariate comparisons based on two-sided Wilcoxon rank sum tests (a) as well as multivariable linear regressions considering clinical and epidemiological covariates (b–e), including age, sex, genetic ancestry, and tumour purity (n = 250 passive smokers, n = 208 non-passive smokers). f, Enrichment of mutational signatures derived from DBS, ID, CN, and SV alterations. Horizontal lines marking statistically significant thresholds were included at 0.05 (dashed orange line) and 0.01 FDR value levels (dashed red line). Statistical significance was evaluated using multivariable logistic regression models for passive smoking history and adjusted by age, sex, genetic ancestry, histology and tumour purity. a,b, Differences in the number of CN segments and SV events using univariate comparisons based on two-sided Wilcoxon rank sum tests (a) as well as multivariable linear regressions, considering clinical and epidemiological covariates (b), including age, sex, genetic ancestry, histology, and tumour purity, for patients diagnosed in geographical regions with high and low PM2.5 exposure levels (threshold defined at 20 μg m−3; n = 440 high-pollution group, n = 413 low-pollution group; only samples for which the country of origin was known are included). Error bars indicate 95% CIs (b). c, Volcano plots indicating enrichment of mutational signatures derived from CN and SV alterations. Horizontal lines marking statistically significant thresholds were included at 0.05 (dashed orange line) and 0.01 FDR value levels (dashed red line). Statistical significance was evaluated using multivariable logistic regression models for PM2.5 exposure levels and adjusted by age, sex, genetic ancestry, histology, and tumour purity. Two-sided Student's t-tests were used to calculate statistical significance. a, Quantification of SBS burden excluding SBS4 mutations for patients living in geographical regions with high and low PM2.5 exposure levels (threshold defined at 20 μg m−3; n = 440 high-pollution group, n = 413 low-pollution group; only samples for which the country of origin was known are included). Statistical significance was evaluated using two-sided Wilcoxon rank sum tests. b, Forest plot corresponding to a multivariable linear regression considering high or low PM2.5 exposure group, age, sex, genetic ancestry, histology and tumour sample purity as covariates and SBS burden as independent variable (threshold defined at 20 μg m−3; n = 440 high-pollution group, n = 413 low-pollution group; only samples for which the country of origin was known are included). c, Scatter plot showing a significant correlation between individual sample estimates of PM2.5 exposure and SBS burden. Statistical significance was evaluated using a multivariable linear regression of the individual PM2.5 estimates per sample and mutation burden (log10 scale), and adjusted by age, sex, genetic ancestry, histology and tumour purity. Blue lines and bands indicate univariate linear regressions and 95% CIs for average mutation burden versus average PM2.5 estimates. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09203-8'>PPP2R1A mutations portend improved survival after cancer immunotherapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'nature.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 15:03:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Immune checkpoint blockade (ICB) therapy is effective against many cancers, although resistance remains a major issue and new strategies are needed to improve clinical outcomes1,2,3,4,5. Here we studied ICB response in a cohort of patients with ovarian clear cell carcinoma—a cancer type that poses considerable clinical challenges and lacks effective therapies6,7,8. We observed significantly prolonged overall survival and progression-free survival in patients with tumours with PPP2R1A mutations. Importantly, our findings were validated in additional ICB-treated patient cohorts across multiple cancer types. Translational analyses from tumour biopsies demonstrated enhanced IFNγ signalling, and the presence of tertiary lymphoid structures at the baseline, as well as enhanced immune infiltration and expansion of CD45RO+CD8+ T cells in the tumour neighbourhood after ICB treatment in PPP2R1A-mutated tumours. Parallel preclinical investigations showed that targeting PPP2R1A (by pharmacological inhibition or genetic modifications) in in vitro and in vivo models was associated with improved survival in the setting of treatment with several forms of immunotherapy, including chimeric antigen receptor (CAR)-T cell therapy and ICB. The results from these studies suggest that therapeutic targeting of PPP2R1A may represent an effective strategy to improve patient outcomes after ICB or other forms of immunotherapy, although additional mechanistic and therapeutic insights are needed. Immune checkpoint inhibitors have revolutionized the management of many solid tumours but, disappointingly, have shown minimal efficacy in both advanced and recurrent ovarian cancer. Response rates to single agent programmed cell death protein 1 (PD-1) and programmed death-ligand 1 (PD-L1) inhibitors in recurrent ovarian cancer are low1,2,3, ranging from 5% to 15%. Unfortunately, biomarkers, such as PD-L1 expression, tumour mutation burden, BRCA1/2 mutation status and homologous recombination deficiency, have not been significantly associated with response or clinical benefit in patients with ovarian cancer treated with ICB therapies3,4,5. Some data suggest that combination ICB targeting both PD-1/PD-L1 and CTLA-4 may be more effective in recurrent ovarian cancer. A randomized trial demonstrated a response rate of 31.4% in patients with recurrent epithelial ovarian cancer who received nivolumab (anti-PD1) plus ipilimumab (anti-CTLA-4), compared with 12.2% in those who received nivolumab alone4. Furthermore, patients in this trial with ovarian clear cell carcinoma (OCCC; a rare histological subtype associated with a worse prognosis compared with the more prevalent high-grade serous ovarian cancer, HGSOC6,7,8) demonstrated a fivefold greater odds of response compared with those with HGSOC4. Thus, patients with recurrent OCCC may especially benefit from ICB; however, the response rate in this group remains around 15%, suggesting that only a subset of patients with OCCC derive clinical benefit. On the basis of these observations, we prospectively enrolled patients with OCCC into a trial investigating the efficacy of dual ICB in the platinum-resistant setting (NCT03026062). During the course of this trial, we observed that patients whose tumours had inactivating somatic mutations in PPP2R1A—a gene encoding the most common scaffold component of the protein phosphatase 2A (PP2A) complex—demonstrated significantly prolonged overall survival (OS) and progression-free survival (PFS). Critically, this finding was validated in external cohorts of patients treated with ICB. We also performed translational investigation of paired pretreatment and on-treatment tumour biopsies, revealing enhanced antitumour immune responses in those with PPP2R1A mutations. Beyond this, we conducted concurrent studies in several different preclinical models, which suggested that targeted inhibition of PPP2R1A enhanced anti-tumour immunity and treatment response in the setting of immunotherapy. To better understand the biological correlates of the enhanced response to ICB, we investigated a cohort of 34 patients with platinum-resistant OCCC who were treated with a combination of anti-CTLA-4 and anti-PD-1/PD-L1 antibodies (Fig. All except one were enrolled on an investigator-initiated clinical trial of tremelimumab (anti-CTLA-4) and durvalumab (anti-PD-L1) treatment in patients with platinum-resistant ovarian cancer (NCT03026062). The remaining patient (patient 0) was treated on a separate clinical trial of ipilimumab (anti-CTLA-4) and nivolumab (anti-PD-1; NCT01928394; Extended Data Fig. A summary of demographic and clinicopathological information for all patients is shown in Table 1. The patients received a median of two previous lines of therapy (range, 1–9), with no previous ICB. a, Schematic of the clinical trial design, sampling and subsequent analyses. Part of the figure was created using BioRender. Response assessments shown are from only during the trial. CR, complete response (defined according to RECIST v.1.1, modified for immunotherapy); mut, mutated; PD, progressive disease; PR, partial response; SD, stable disease; WT, wild-type. c, Kaplan–Meier survival analysis of OS based on the PPP2R1A mutation status. d, Kaplan–Meier survival analysis of OS based on the ARID1A mutation status in the subgroup of patients with wild-type PPP2R1A. e, Kaplan–Meier survival analysis of OS based on the PPP2R1A mutation status in the subgroup of patients with ARID1A mutations. f, Representative patient computed tomography (CT) scan images (patient (Pt) 0) demonstrating the disease burden over time, with initial progression then response. CT scan images are shown at the baseline, and at 12, 24 and 36 weeks of therapy; the arrowheads mark representative tumour lesions. g, The changes in target lesion responses, as revealed by the sum of tumour diameters. For patient 31 (asterisk), follow-up was continued until 70.4 months from the baseline. For c–e, P values were calculated using one-sided log-rank tests. Next-generation sequencing analysis of tumour biopsies identified ARID1A and PIK3CA somatic mutations or alterations in the majority of patients (73.5% and 55.9%, respectively) (Extended Data Fig. The remaining two patients had S256F and W257G missense mutations, respectively (Extended Data Fig. All of the observed mutations are known to impair the binding of the major scaffold A subunit (encoded by PPP2R1A) to the regulatory B subunits of PP2A, therefore conferring a loss of function9. Only one case of mismatch repair deficiency/microsatellite instability (MSI) was identified among the evaluable tumours, and it was present in a patient with wild-type PPP2R1A, consistent with published literature demonstrating a low likelihood of mismatch repair deficiency or MSIhigh status in patients with OCCC10. Previous studies have found that E17K and C77F are activating oncogenic AKT1 mutations in solid tumours, and AKT2 amplification is a known driver of ovarian cancer pathogenesis11,12. We sought to specifically evaluate the impact of ARID1A, PPP2R1A and AKT mutations on the outcomes in this cohort based on observations in three ‘exceptional responder' patients with platinum-resistant OCCC who were treated with combination ICB and lived for over 5 years from the start of treatment. A brief clinical course of these patients is provided below. The first patient (patient 0), who had a somatic PPP2R1A mutation (R183Q), received ipilimumab and nivolumab for 15 months and lived for 5.5 years (with disease) after her first dose of ICB. To put these outcomes in perspective, the median OS in patients with platinum-resistant OCCC after progression on frontline therapy is only 10.9 months13. All three of these patients also had somatic ARID1A mutations, which are associated with improved response to immunotherapy in several cancer types, yet such associations have not been reported for loss-of-function mutations in PPP2R1A or activating alterations in AKT14,15,16,17. These observations led us to hypothesize that somatic mutations in one or both of these genes may correlate with improved survival in other patients with OCCC treated with ICB. To test this hypothesis, we compared the outcomes of patients with OCCC based on ARID1A and PPP2R1A mutation status to see whether these alterations are correlated with OS in the setting of ICB treatment. The small number of patients with AKT alterations (n = 4) limited specific analysis of this cohort individually. We observed a significantly longer OS in patients whose tumours had a PPP2R1A mutation compared with those without a PPP2R1A mutation (median OS 66.9 versus 9.2 months; hazard ratio, 0.40; 95% CI 0.15–1.08; one-sided log-rank P = 0.031; Fig. There was also a significant difference in PFS, with PPP2R1A-mutation carriers having a median PFS of 3.0 months compared with 1.8 months for non-mutation carriers (one-sided log-rank P = 0.034; Extended Data Fig. As 9 of the 11 patients with PPP2R1A mutations had concurrent ARID1A mutations, to discriminate between the prognostic effects of ARID1A and PPP2R1A mutations, we first evaluated the prognostic effects of ARID1A mutations in patients whose tumours were wild-type for PPP2R1A (n = 23; Fig. The wild-type and mutated ARID1A groups did not show a statistically significant difference in survival, with a median OS of 9.2 versus 5.1 months (one-sided log-rank P = 0.055; Fig. Next, we examined the OS in patients with PPP2R1A mutations within the subset of patients with mutated ARID1A (n = 25; Fig. In these cases, we still observed a trend toward longer survival after ICB in patients with OCCC with concurrent PPP2R1A mutations (OS, 66.9 versus 9.2 months; one-sided log-rank P = 0.047; Fig. In summary, these data suggest that PPP2R1A mutations may have a dominant effect in association with longer survival of patients with OCCC treated with ICB, although, due to the limited sample size in each subgroup, we cannot rule out the possibility that ARID1A mutations may also have independent prognostic value. Examination of clinical course and responses in our cohort of patients with PPP2R1A mutations provided a key insight that the observed survival benefit was associated with delayed responses that sometimes followed a period of initial progression or stable disease (Fig. Note that, to guard against pseudoprogression, all of the patients in this study were enrolled on clinical trials that allowed for continuation of immunotherapy beyond apparent progression of disease as determined by Response Evaluation Criteria in Solid Tumours (RECIST) v.1.1, which may have contributed to durable disease control in patients with PPP2R1A-mutated tumours. Moreover, higher rates of grade 3 or higher immune-related adverse events were noted in PPP2R1A mutation carriers (45.5% versus 13.0% for non-carriers; P = 0.079; Table 1), which is consistent with previous studies demonstrating a positive correlation between immune-related adverse events and favourable outcomes in patients treated with ICB18. The pretreatment tumour samples from 29 patients in the clinical cohort were obtained and profiled using RNA sequencing (RNA-seq); among these, 14 paired on-treatment samples were also analysed (Supplementary Table 1). In comparing the transcriptional profiles of pretreatment samples from patients with PPP2R1A mutations to those with wild-type PPP2R1A, we observed significant enrichment of the IFNγ response pathway in PPP2R1A-mutant samples, indicating activation of adaptative immune responses (Fig. The enhanced antitumoural immune responses in PPP2R1A-mutant cases were even more accentuated after ICB treatment, as revealed by the enrichment of multiple additional immune-response-related pathways, including inflammatory response, complement pathway, allograft rejection, IFNα response and IL-2 signalling (Fig. Notably, these differences in antitumoural immune responses remained significant even after accounting for potential confounding from AKT alterations (Extended Data Fig. a,b, Gene set enrichment analysis (GSEA) of the pretreatment (a) and on-treatment (b) OCCC samples, showing upregulated immune signatures in the PPP2R1A-mutant group. FDR, false discovery rate; NES, normalized enrichment score. c–f, Longitudinal changes in the relative abundance of CD8+ T cells (c), activated NK cells (d), resting NK cells (e) and total NK cells (f) in paired samples from the PPP2R1A-mutant and wild-type groups. Longitudinal changes in TCR (g) and BCR (h) richness in paired samples from the PPP2R1A-mutant and wild-type groups. For c–h, P values were calculated using two-sided Wilcoxon signed-rank tests. In longitudinal comparisons within the PPP2R1A-mutant group, significant upregulation of the IFNγ and IFNα response pathways was observed in the on-treatment samples compared with the baseline, indicating further immune activation with combined ICB therapies (Extended Data Fig. Consistently, when comparing the composition of tumour-infiltrating immune cells based on immune cell deconvolution, increases in the relative and absolute proportions of tumour-infiltrating CD8+ T cells and activated natural killer (NK) cells were observed, along with a trend of upregulation in total T cell receptor (TCR) and B cell receptor (BCR) richness. This was also accompanied by a decrease in the proportion of resting NK cells (Fig. Conversely, these changes were not significant in the wild-type PPP2R1A group in longitudinal comparisons (Fig. Given the inherent limitations of bulk RNA-seq data in revealing detailed immune cell phenotypes and the lack of spatial information, we further performed highly multiplexed spatial phenotyping of pretreatment and on-treatment samples using CODEX with a 26-antibody panel (Fig. A total of 378,868 high-quality cells from 28 samples was obtained for downstream analysis (Supplementary Table 1 and Methods). In the baseline tumours, PPP2R1A-mutant cases exhibited higher levels of intratumoural infiltration of MHC-II+ immune cells compared with the wild-type cases (Extended Data Fig. Furthermore, B cells (CD20+), particularly proliferating B cells (CD20+Ki-67+) and germinal centre B cells (CD20+CD21+Ki-67+), showed a trend of higher densities in PPP2R1A-mutant samples, despite the small cohort size (Fig. Consistent with this finding, peritumoural secondary follicle-like tertiary lymphoid structures (TLSs) were observed in three of these samples (Fig. As TLSs have recently been linked to favourable response to ICB19,20,21, our preliminary observations shed light on the importance of further evaluating the potential association between TLSs and PPP2R1A mutations in future studies based on larger cohorts. Finally, at the on-treatment timepoint, PPP2R1A-mutant samples also exhibited significantly higher densities of tumour-infiltrating immune cells, including CD45+CD56+ NK cells (Extended Data Fig. a, The antibody panel applied in CODEX assay. The diagram was created using BioRender. d, A representative CODEX image showing TLSs from the pretreatment sample of a patient with mutant PPP2R1A (patient 160). e. Schematic of the multicellular neighbourhood analysis. The diagram was created using BioRender. In f, the numbers of cells in each group are as follows: 7,043 (mutant PPP2R1A, pretreatment), 12,758 (mutant PPP2R1A, on treatment), 17,346 (WT PPP2R1A, pretreatment) and 66,177 (WT PPP2R1A, on treatment). In g, the numbers of cells in each group are as follows: 523 (mutant PPP2R1A, pretreatment), 1,532 (mutant PPP2R1A, on treatment), 212 (WT PPP2R1A, pretreatment), 2,536 (WT PPP2R1A, on treatment). h,i, Representative CODEX images showing PD-1+CD8+ T cells and CD45RO+PD-1−CD8+ T cells in the neighbourhood of tumour cells in PPP2R1A-mutant (patient 157; h) and wild-type (patient 137; i) on-treatment samples. P values were calculated using two-sided Wilcoxon rank-sum tests. To further examine the spatial relationships between tumour and immune cells, we conducted a multicellular neighbourhood analysis. Notably, we observed a significant upregulation of PD-1+CD8+ T cells in the neighbourhoods of tumour cells after ICB treatment in both PPP2R1A-mutant and wild-type cases (Fig. PD-1 expression is known to be induced on T cells after activation and can persist until they reach a terminally exhausted state25. These findings suggest that ICB drugs were delivered to the tumour sites in both groups and elicited effective T cell responses. Nevertheless, CD45RO+PD-1−CD8+ memory T cells were uniquely upregulated in the tumour neighbourhoods of PPP2R1A-mutant cases after treatment, particularly around MHC-I+ tumour cells (Fig. CD45RO+ memory T cells, which could serve as a replenishing pool for effector T cells, have been shown to mediate sustained responses to ICB therapy across multiple cohorts28,29,30. Thus, the lack of this population in wild-type PPP2R1A cases may account for the shorter duration of disease control and inferior outcomes. Finally, we observed an enrichment of NK cells in tumour neighbourhoods in PPP2R1A-mutant cases at both the pretreatment and on-treatment timepoints (Extended Data Fig. These results further complemented the RNA-seq analysis showing that activated NK cells were upregulated after ICB treatment in PPP2R1A-mutant cases. Together, these findings provided further evidence supporting the superior outcomes after ICB therapy in PPP2R1A-mutant OCCC. When further examining the differences between long-term and short-term survivors within the PPP2R1A-mutant group, as dichotomized by median OS (Extended Data Fig. 6), enhanced antitumoural immunity was more evident in long-term survivors after ICB treatment. This was indicated by the enrichment of multiple immune signalling pathways, as well as upregulation of T cell and B cell lineage markers (such as CD3E, CD4 and CD79A/B), MHC class II molecules (such as HLA-DRA, HLA-DQA2 and CD74), lymphocyte activation or function markers (TNFRSF13B and GZMB), co-stimulatory molecule (CD86), immune checkpoint (PDCD1) and chemotaxis chemokines (CCL2, CCL3 and CCL4) (Extended Data Fig. Finally, we sought to further understand the role of AKT alterations, as several patients with tumours containing AKT alterations (wild-type PPP2R1A) in the study cohort exhibited responses to ICB. Notably, our preclinical data indicate that phosphorylated AKT (pAKT) was downregulated in the HEC50B cell line after CRISPR correction of PPP2R1A from mutant (R183W) to wild-type and upregulated in OVCAR429 cells after CRISPR editing of PPP2R1A from wild-type to mutant (R183Q) (Extended Data Fig. These findings suggest that AKT activation is closely associated with PPP2R1A mutational status. Consistently, as described above, the AKT-altered cases in our clinical cohort exhibited multiple known genetic alterations that would lead to AKT activation. Next, we characterized the transcriptional profiles of these cases. RNA-seq data from pretreatment samples of three AKT-altered cases revealed significant enrichment of pathways related to IFNγ response, IFNα response and allograft rejection compared with those with wild-type AKT and PPP2R1A, providing a potential explanation for the efficacy of ICB treatment in this subpopulation (Extended Data Fig. To further validate the immune-related phenotype of PPP2R1A-mutant tumours identified in our discovery clinical cohort, we used a series of in vitro and in vivo models. First, by reviewing previously published data31,32,33, we observed that PPP2R1A consistently ranked among the top hits in genome-wide CRISPR immune screens comparing T cell-treated and control groups, indicating that PPP2R1A knockout increased tumour cell sensitivity to T cell-mediated killing (Fig. Subsequently, we constructed PPP2R1A-knockdown SKOV3 cell lines (Extended Data Fig. When exposed to tumour-reactive B7H3 CAR-T cells, the apoptotic rate of PPP2R1A-knockdown cells was significantly higher than the negative control (Fig. Similarly, treatment of the wild-type SKOV3 cell line with the PP2A-specific inhibitor LB-100 enhanced cancer cell killing by B7H3 CAR-T cells (Fig. However, these observations primarily reflect the influence of a reduction in normal PPP2R1A protein levels. To directly assess the impact of PPP2R1A mutation and align with our clinical observations, we established SKOV3 cell lines with stable overexpression of wild-type PPP2R1A and the P179R mutant (Extended Data Fig. Consistent with the results above, cancer cells with the PPP2R1AP179R mutant showed increased sensitivity to CAR-T-cell-mediated killing (Fig. We next engineered the parental and CRISPR-edited cell lines to overexpress human CD19 (hCD19) (Extended Data Fig. 8d), and exposed them to hCD19 CAR-T cells at varying effector:target ratios. Consistently, hCD19 CAR-T cells exhibited higher killing efficacy against PPP2R1A-mutant cell lines in a dose-dependent manner (Fig. b,c, Expression of activated caspase-3 in genetically modified (b) or LB100-treated (c) SKOV3 cell lines after exposure to B7H3 CAR-T cells. Representative results from three independent experiments are shown. d, Overexpression of the PPP2R1AP179R mutant increased SKOV3 sensitivity to CAR-T-cell-mediated killing. Representative results from four independent replications are shown. For each group, four independent replicates were performed. h, Schematic of the study design for PDX models. The diagram was created using BioRender. Hu-BLT, humanized-bone marrow, liver and thymus mice; Hu-EC, human endometrial cancer. i, Tumours from PPP2R1A-mutated PDXs at the end of treatment. k, Schematic of the study design for syngeneic mouse OCCC models. The diagram was created using BioRender. l,m, The tumour size measured during treatment in the Arid1a−/−Pik3caH1047R (l) or Ppp2r1aR183QArid1a−/−Pik3caH1047R (m) syngeneic mouse models. In l–n, each group included five independent replicates. P values were calculated using two-sided Student's t-tests (j) and repeated-measures analysis of variance (ANOVA) with Tukey's test for pairwise comparisons (b–d and l–n). Moreover, endometrial cancers have the highest rates of PPP2R1A mutations among all cancer types. Thus, to further validate the role of PPP2R1A mutations in ICB response in vivo, we collected human endometrial cancer samples with and without PPP2R1A mutations and established patient-derived xenograft (PDX) models on humanized BLT mice. The PPP2R1A mutational status of the PDX tumours was confirmed by Sanger sequencing (Extended Data Fig. Mice were randomized into two groups, receiving either anti-PD-L1 therapy or human IgG isotype control (Fig. After 3 weeks of treatment, significant reductions in tumour size and weight were observed in the anti-PD-L1 treatment group compared with the control group in PPP2R1A-mutant PDX models (Fig. By contrast, no therapeutic effect of anti-PD-L1 treatment was observed in wild-type PPP2R1A PDX models (Extended Data Fig. 8f,g), suggesting that PPP2R1A mutations specifically confer sensitivity to ICB treatment in this setting. Notably, PPP2R1A-mutant OCCC cases often had concurrent ARID1A mutations (Extended Data Fig. To further rule out the potential confounding effect of ARID1A mutations on immunotherapy response and validate our clinical findings, we generated an isogenic Ppp2r1aR183Q mutant in an OCCC cell line derived from a conditional Arid1aflox/flox(Gt)Rosa26Pik3ca*H1047R genetic OCCC mouse model in the C57BL/6 background34 (Extended Data Fig. This set of mutations was based on the co-occurring mutations in PPP2R1A, ARID1A and PIK3CA seen in 15% of the overall cohort and 45% of PPP2R1A-mutant OCCCs in the trial (Extended Data Fig. These isogenic cells enabled us to establish syngeneic immunocompetent mouse models. After establishment of tumours to around 100 mm3, mice were randomized and treated with anti-PD-L1 or IgG control antibodies (Fig. As ARID1A inactivation also sensitizes tumours to ICB15,35, we reduced the dose of anti-PD-L1 to 2.5 mg per kg weekly based on the response in syngeneic tumours formed by Arid1a−/−Pik3ca*H1047R cells (Fig. This enabled us to show that tumour burdens, as measured by tumour size during treatment and tumour weight at the end point, were reduced by the anti-PD-L1 treatment in tumours formed by Arid1a−/−Pik3ca*H1047RPpp2r1aR183W cells compared with the IgG controls (Fig. Furthermore, in both models, anti-PD-L1 treatment was not associated with significant changes in mouse body weight, indicating that the treatment was well tolerated (Extended Data Fig. On the basis of our findings in OCCC, we hypothesized that PPP2R1A loss-of-function mutations in other tumour types may also be associated with improved outcomes in patients treated with ICB. To test this hypothesis, we used cBioPortal36,37 to analyse two independent datasets: one cohort of 1,661 patients treated with ICB, and another cohort of 7,564 patients undergoing various treatments for advanced cancer. Patients with PPP2R1A mutations treated with ICB showed significantly longer OS (median OS not reached versus 18.0 months (95% CI 16.0–21.0 months), P = 0.033; Extended Data Fig. By contrast, no significant differences in OS were observed in patients with PPP2R1A mutations who received other therapies (Extended Data Fig. Notably, the group with PPP2R1A mutations in the immunotherapy cohort encompassed a variety of cancer types and heterogenous concurrent mutations (Supplementary Tables 3 and 4), therefore making it unlikely that the observed survival difference was driven by cancer-type-specific factors or by other mutations. Furthermore, the survival benefit associated with PPP2R1A mutations was also revealed in two additional cohorts including a total of 286 patients with non-small cell lung cancer and melanoma who were treated with ICB (Extended Data Fig. However, the limited number of patients with PPP2R1A mutations in these cohorts precluded robust statistical analyses. Nonetheless, these aggregated data suggest that the correlation between PPP2R1A mutations and survival is not merely prognostic of better cancer treatment outcomes but, rather, is specific to patients treated with ICB, with potential relevance in cancers beyond OCCC. Moreover, given the high prevalence of PPP2R1A mutations in endometrial cancer, and the fact that immunotherapy with the combination of lenvatinib and pembrolizumab (len–pem) is an approved standard second-line therapy, we investigated the relationship between PPP2R1A-inactivating mutations and outcomes in a cohort of these patients. Among this cohort, 24 patients (23.8%) had endometrial cancers with PPP2R1A mutations. However, the median PFS was not significantly different between the two groups (7.5 versus 5.5 months; P = 0.246; Extended Data Fig. Analysis of only the TP53-mutated uterine tumours treated with len–pem revealed similar trends: the median OS was not reached in PPP2R1A-mutated patients versus 16.0 months in those without somatic PPP2R1A mutations (P = 0.027), and the median PFS was 7.5 versus 4.8 months, respectively (P = 0.199; Extended Data Fig. Finally, we also performed an analysis of wild-type ARID1A cases, and again observed a better OS, despite a similar PFS, in the PPP2R1A-mutant group (Extended Data Fig. Reminiscent of the results in OCCC cohort (Fig. 1e), these data further corroborated that PPP2R1A mutations could confer prognostic values independent of ARID1A status. We identified a significant correlation between loss-of-function PPP2R1A mutations and improved OS in patients with OCCC, and possibly other cancers, who were treated with ICB. Although the exact mechanisms through which these somatic genetic changes improve the outcomes after ICB await further investigation, our data suggest that the presence of these mutations is associated with enhanced immune activation in the tumour microenvironment, and these effects are further boosted after treatment with ICB. Importantly, PPP2R1A mutations are not associated with improved survival of patients with OCCC in the setting of treatment with standard chemotherapy38; thus, this effect appears to be ICB specific, suggesting that these mutations may serve as predictive biomarkers for ICB response in this cancer type. With the use of in vitro and in vivo models, publicly available datasets and a retrospectively analysed endometrial cancer cohort, we were able to extend the positive correlation between PPP2R1A mutations and ICB sensitivity as well as the survival benefits brought by ICB to other cancer types. In addition to serving as a potential biomarker for improved survival after ICB in OCCC and other cancers, the association between PPP2R1A mutations specifically and anti-cancer immunity probably has broader therapeutic significance for improving ICB outcomes. In this regard, there is compelling preclinical evidence that pharmacological inhibition of PP2A (the multimeric phosphatase of which the major scaffold component is encoded by PPP2R1A) improved the efficacy of ICB in syngeneic mouse models39. This study showed induction of synergistic anti-cancer immunity (including immunologic memory) by the combination of a PP2A inhibitor (LB-100) and anti-PD-1 ICB. The therapeutic role of LB-100 was mostly attributed to its direct influences on T cells, with enhanced CD8+ T cell activation, proliferation and trafficking to the tumour after treatment. Notably, these immunological effects were independent of changes in expression of TIM-3, OX40, CTLA-4 and LAG3, suggesting that there may be therapeutic opportunities for combining LB-100 with other immunomodulatory drugs beyond anti-PD-1 antibodies. Indeed, PPP2R1A genetic alterations in T cells were also proven to directly influence their functionalities. In a previous study, a short-hairpin RNA (shRNA) screen in tumour-infiltrating lymphocytes demonstrated that downregulation of a PP2A regulatory subunit inhibited T cell apoptosis and enhanced T cell proliferation and cytokine production40. Lastly, knockout of PPP2R1A in regulatory T cells in mice leads to a severe and widespread lymphoproliferative autoimmune disorder, due to inability to suppress effector immune responses41. Consistent with previously reported results39, our data also revealed that LB-100 could sensitize SKOV3 cells to CAR-T cell killing. Furthermore, as a complement, we also extended the cell killing assays to PPP2R1A knockdown and PPP2R1A-mutant cancer cell lines, so that the effects of altered PPP2R1A downstream pathways were fully attributable to cancer cells. Our results suggest that alterations in the cancer cell PPP2R1A pathway may also indirectly enhance T cell function. On the basis of previous studies, potential mechanisms include increased tumour DNA damage and enhanced neoantigen expression42. For example, PP2A inactivation has been shown to induce MSI in tumour cells or, through the production of cytosolic double-stranded DNA (dsDNA), activate tumour-intrinsic type I IFN signalling in preclinical models of other tumour types43,44. Importantly, here we provide direct evidence of enhanced sensitivity of PPP2R1A-mutant human cancer to ICB by using humanized PDX models. The humanized immune system in the model could better mimic the potential effect of ICB in patients with somatic PPP2R1A mutations, and these data aligned well with our clinical findings. Taken together, these preclinical studies support a functional link between PP2A loss/inhibition and enhanced anti-tumour immunity and suggest that pharmacological inhibition of PP2A may represent a robust strategy for activating antitumour immunity due to its impact on both the tumour and microenvironmental cell compartments and the potential to provide benefit in patients without PPP2R1A mutations. In the aforementioned preclinical studies, PP2A inhibition in tumour-infiltrating lymphocytes and loss of function in regulatory T cells were demonstrated to have important roles in immune activation; thus, systemic administration of a pharmacological inhibitor of PP2A may be associated with more rapid tumour regression by accelerating immune responses in both the tumour and immune cell compartments. Consistent with these observations, we have recently activated a phase Ib/II clinical trial of ICB and LB-100 in patients with OCCC who do not have somatic PPP2R1A mutations (NCT06065462). Finally, in this study we also observed an enrichment of NK cells, as well as their phenotypic transition to an activated state, in PPP2R1A-mutated OCCC after ICB treatment. Indeed, the vital roles of NK cells in mediating ICB efficacy and predicting ICB responses have been reported in recent preclinical and clinical studies45,46,47. Our findings further supported these observations and highlighted the potential for NK-cell-based immunotherapies in treating PPP2R1A-mutated tumours. However, further studies are warranted to provide deeper mechanistic insights and assess therapeutic efficacy. Although these findings are provocative, limitations exist with regard to sample size and to the potential contribution of other mutations (such as those of ARID1A) on response to ICB in this cohort. Nonetheless, this study examined a rare cancer with translational analyses performed on unique clinical samples, and these data therefore represent a valuable contribution to the literature. Furthermore, this investigation provides evidence in support of the potential use of PPP2R1A as a predictive biomarker for prognosis after ICB and as a target to improve clinical outcomes in the setting of ICB therapy. Confirmatory studies in larger cohorts and deeper mechanistic studies with additional preclinical models are warranted. The patients included in this analysis were identified through a retrospective review of all patients with recurrent, platinum-resistant or refractory OCCC (including clear cell carcinomas arising from peritoneal and fallopian tube origins), defined as progression within 180 days of a previous line of platinum-containing therapy, treated with ICB (MD Anderson Cancer Center Institutional Review Board (IRB) 2020-1098). The patients included in this analysis were treated on two different clinical trial protocols, both IRB approved by MD Anderson, in which combination CTLA-4 and PD-1/L1-targeting ICB was administered (Clinicaltrials.gov: NCT03026062 (n = 33) and NCT01928394 (n = 1)). Informed consent was obtained from all of the patients enrolled in the two trials before starting treatment. NCT03026062 was an open-label, adaptively randomized phase II trial of tremelimumab and durvalumab administered in combination or sequentially. Sequential therapy included tremelimumab 3 mg per kg every 4 weeks for up to 4 doses, followed by durvalumab 1.5 g every 4 weeks for up to 9 doses after progression. Combination therapy consisted of tremelimumab 1 mg per kg plus durvalumab 1.5 g every 4 weeks for up to 4 doses followed by durvalumab monotherapy (1.5 g for up to 9 doses or until progression). Notably, enrolment on this trial was amended, initially limiting it to patients with OCCC, and later further restricting participation to those with OCCC with inactivating PPP2R1A or activating AKT alterations. An additional patient included in this study received treatment on NCT01928394, an open-label phase II trial in which the patients were stratified by platinum-free interval and then randomized to different dosing schedules of nivolumab and ipilimumab. The participant included in this study received ipilimumab 1 mg per kg every 6 weeks plus nivolumab 3 mg per kg every 2 weeks until progression. Demographic and clinicopathological information, including tumour somatic mutation testing and mismatch repair/microsatellite stability status, was collected from the medical record. Tumour mutation testing using next-generation sequencing was obtained as part of the patients' standard care either at MD Anderson Cancer Center or through a commercial lab (typically Caris or FoundationOne). Clinical DNA mutation testing was not available for two participants; in these cases, T200 next-generation sequencing was performed on their pretreatment biopsies as previously described49. RECIST v.1.1, modified for immunotherapy50, was used according to the respective clinical trial protocols to assess the response. Independent radiology assessment was performed by experienced radiologists at The University of Texas MD Anderson Cancer Center, and progressive disease was determined on the basis of both the target lesion size as well as non-target lesion equivocal progression. The patients in the study were allowed to continue treatment beyond progression based on risk–benefit considerations. A swimmer plot was used to describe patient overall responses and outcomes. Spider plots and waterfall plots were used to describe target lesion responses, as revealed by the sum of tumour diameters, during the on-trial period. OS was calculated from the time of study registration to the earliest date of death or data cut-off (29 January 2024). PFS was calculated from the time of study registration to the date of disease progression or death or the last time of clinical evaluation. For patients 44, 115 and 126, owing to rapid disease progression leading to the patients' death, no tumour measurements were performed based on CT scan, and PFS was determined based on clinical criteria. OS and PFS were estimated using the Kaplan–Meier method and compared by variables of interest using one-sided log-rank rest. Clinical data analysis was performed in R statistical environment (v.4.3.1; https://www.r-project.org/). Pretreatment image-guided core biopsies, stored as formalin-fixed paraffin-embedded (FFPE) samples, were collected before the first dose of the study drug. On-treatment biopsies, also stored as FFPE samples, were performed when there was a lesion amenable to biopsy as assessed by the cycle 2 imaging study by an MD Anderson Cancer Center radiologist. When possible, on-treatment biopsies were obtained from the same area of tumour as the pretreatment biopsies. Samples were collected, labelled, processed and stored by the MD Anderson Cancer Center Gynecologic Oncology Tumour Bank for translational studies. Sample identification and associated patient-level data were entered into a secure database. In brief, poly(A) RNA was captured from 10–250 ng of total RNA using magnetic oligo-dT beads. After bead elution and clean-up, the resultant poly(A) RNA was fragmented using heat and magnesium. First-strand synthesis was performed using random priming followed by second-strand synthesis with the incorporation of dUTP into the second strand. The ends of the resulting double-stranded cDNA fragments were repaired, 5′-phosphorylated, 3′-A-tailed and Illumina specific indexed adapters were ligated. The products were purified and enriched for a full-length library with 9–16 cycles of PCR. The strand labelled with dUTP is not amplified, resulting in a strand-specific library. The library pool was quantified by quantitative PCR, then sequenced on one lane of the NovaSeq 6000 (Illumina) S4 flow cell using the 100-nucleotide paired-end run format. RNA-seq FASTQ files were processed through FastQC (v.0.11.5), a quality-control tool to evaluate the quality of sequencing reads at both the base and read levels, and RNA-SeQC (v.1.1.8) to generate a series of RNA-seq-related quality-control metrics51. STAR 2-pass alignment (v.2.7.0 f)52 was performed with the default parameters to generate RNA-seq BAM files. HTSeq-count (v.0.11.0)53 was applied to aligned RNA-seq BAM files to count for each gene how many aligned reads overlap with its exons. In the initial assessment of transcriptomic-level similarity and distinctions of all of the samples, two samples shown as significant outliers were removed from further analysis. Differential gene expression analyses were then performed using the R package DESeq2 (v.1.42.1)54, with raw count data as input. A ranked list of coding genes was generated based on the Wald statistic in DESeq2, and subsequently processed by GSEA55 using the R package clusterProfiler (v.4.6.2)56 against the Hallmark gene sets from Molecular Signature Database (MSigDB)57 to identify significantly enriched pathways. The false-discovery-rate-corrected q values were used for multiple-hypothesis-testing adjustment58. The HTSeq raw count data were then normalized to transcripts per kilobase million (TPM), and the TPM expression values were used as an input to the software CIBERSORTx59 (https://cibersortx.stanford.edu/) for immune cell deconvolution in each sample, using the default signature matrix LM22. Next, intergroup comparisons of the abundance of certain immune cell populations were performed using Wilcoxon rank-sum and signed-rank tests. The T cell and B cell receptor repertoires were reconstructed using TRUST4 (v.1.0.5.1)60 on the aligned RNA-seq BAM files. Repertoire richness was calculated as the total number of unique receptor sequences with >1 mapped reads, to account for singletons. Downstream data analysis was performed with the R package immunarch (v.1.0.0; https://immunarch.com/). Only samples with detectable TCR/BCR were used for comparisons. Wilcoxon signed-rank and rank-sum tests were performed for comparing paired samples and the entire cohort, respectively. A multiplexed barcoding image analysis platform was performed using methods and reagents similar to those previously described61. Using the PhenoCycler Fusion platform (Akoya Biosciences) staining kit (Akoya, 7000008), the antibody cocktail was prepared by combining staining buffer, N blocker, G blocker, J blocker, S blocker and all barcoded antibodies, including pan-cytokeratin, CD45, CD3e, CD8, CD4, CD45RO, granzyme B, CD56, CD20, CD19, CD21, FOXP3, PD-1, PD-L1, CTLA-4, OX40, CD73, CD39, CD68, ARG1, CD206, CD31, HLA-DR, Ki-67, CD33, MHC-I and 4′,6-diamidino-2-phenylindole (DAPI) (detailed information of the antibody panel is provided in Supplementary Table 5). The slides were placed in a plastic humidity chamber, and 190 µl of antibody cocktail, including all of the markers above, was dropped onto the sample and incubated at 4 °C overnight. Next, a 96-well reporter plate (Akoya, 7000006) was prepared containing barcoded fluorophores according to custom-designed multiplex barcoding experiments. Each well was filled with 245 µl of a solution containing the reporter and specific barcode fluorophores for that cycle and placed into the PhenoCycler Fusion instrument (Akoya Biosciences). Each cycle will have 2 or 3 antibodies plus their specific barcode fluorophores and DAPI. At the end of each cycle, each image from the tissue sample was captured at a high magnification of ×20 using the PhenoCycler Fusion fluorescence microscope (Akoya Biosciences). Then, the signal was erased to start a new cycle until the desired antibody image panel was complete. Experienced pathologist analysed each tissue sample using image analysis software (QuPath v.0.4.4)62. Marker co-expression was used to identify specific cell phenotypes in the tumour and stroma compartments. The data were preprocessed to be consolidated using the R package phenoptr (v.0.2.2, Akoya Biosciences). The densities of cell phenotypes (marker co-expression) were expressed as the number of cells per mm2. The cell densities were then compared between PPP2R1A-mutant and wild-type PPP2R1A samples using two-sided Wilcoxon rank-sum tests. Multicellular neighbourhood analysis was performed, with each multicellular neighbourhood defined as a collection of cells of which the centroids were within 80 μm of the centre cell. Each cell in the images was taken as the centre iteratively to obtain a neighbourhood composition vector. In such a way, a neighbourhood composition matrix was constructed for the whole dataset. The neighbourhood compositions of certain cell types were then compared between different timepoints or sample groups using two-sided Wilcoxon rank-sum tests. To generate PPP2R1A knockdown cell lines, SKOV3 cells were transduced with retrovirally expressed shRNAs. Fully synthesized dsDNA fragments (Twist Bioscience) encoding gene-specific shRNAs were inserted into the pSuper.retro.puro (VEC-PRT-0002, OligoEngine) according to the manufacturer's protocol. To generate retroviral supernatants, HEK293T cells were seeded 16 h before transfection and transfected with the retroviral vectors encoding different shRNAs (Supplementary Table 6), along with lentiviral packaging plasmids, pCMV-VSV-G and pBS-CMV-GagPol (8454 and 35614, Addgene) by the jetPRIME transfection reagent (101000046, VWR) according to the manufacturer's protocol. Viral supernatants were collected at 48 h after transfection and filtered by 0.45 μm PVDF Syringe Filter Unit (SLHV033NK, Millipore-Sigma) to remove cell debris. Designated titres of retrovirus were used to infect cells in the presence of 8 μg ml−1 hexadimethrine bromide (107689, Sigma-Aldrich). A series of genetic modified cell lines was generated by sorting GFP+ cells 72 h after retrovirus transduction. Subsequent immunoblot assays were performed to assess the knockdown efficacy. To ectopically express wild-type PPP2R1A and the P179R mutant in SKOV3 cells, dsDNA fragments encoding human open reading frames (ORFs) of wild-type PPP2R1A (NM_014225.6) and the P179R mutant were inserted into the lentiviral vector, pLVX-IRES-ZsGreen1 (PT4064-5, Takara Bio). To generate lentiviral supernatants, HEK293T cells were seeded 16 h before transfection and transfected with the lentiviral vectors encoding wild-type PPP2R1A and P179R mutant, along with the lentiviral packaging plasmids pCMV-VSV-G and psPAX2 (8454 and 12260, Addgene), using the jetPRIME transfection reagent according to the manufacturer's protocol. The viral supernatants were collected 48 h after transfection and filtered as previously described. Designated titres of lentivirus were used to infect cells in the presence of 8 μg ml−1 hexadimethrine bromide (Sigma-Aldrich). Stable cell lines were generated by sorting GFP+ cells 72 h after lentivirus transduction. To perform adenine base editing (ABE) targeting the c.547C>T mutation of the PPP2R1A gene in HEC50B cells, the primer for the single guide RNA (sgRNA) was designed as TGCGGCCCGCCGCACCATGG and was synthesized by IDT. The sgRNA was cloned into pLentiCRISPR V2 (Addgene, 83480). To remove Cas9 from the vector, the pLentiCRISPR V2 vector was digested with XbaI and BamHI. A PCR fragment for ABE8e containing an overlapping cloning site with the digested vector was inserted into the vector using the ClonExpress Ultra One Step Cloning kit (Cellagen C115-01). The following oligonucleotides were used for PCR: forward, 5′-GAACACAGGACCGGTTCTAGAGCCACCATGAAACGGACAG-3′; reverse, 5′-AAGTTTGTTGCGCCGGATCCGACTTTCCTCTTCTTCTTGGGCTCGA-3′. The resulting plasmid was used for lentivirus packaging and infection. Single clones were then selected and validated for editing efficiency by Sanger sequencing. To introduce R183Q (c.548G>A mutation) targeting the PPP2R1A gene in OVCAR429 cells, cytosine base editing (CBE) was used. The primer for the sgRNA was designed as CTGCGGCCCGCCGCACCATG and was synthesized by IDT. The sgRNA was cloned into lentiGuide-puro (Addgene, 52963). For CBE, hyBE4max (Addgene, 157942)65 was used and cloned into pLentiCas9-blast (Addgene, 52962) using the same cloning protocol mentioned above. The resulting plasmids were used for lentivirus packaging and infection. Single clones were then selected and validated for editing efficiency by Sanger sequencing. Notably, the CBE induced two mutations, c.548G>A and c.549G>A, ultimately leading to the amino acid change from Arg (CGG) to Gln (CAA). Furthermore, we also generated HEC50B and OVCAR429 overexpressing hCD19 for downstream cytotoxicity assays using a similar approach to that described previously66. In brief, the hCD19 ORF was obtained from the Wistar Institute Molecular Screening and Protein Expression Facility, and then PCR-amplified and cloned into pLVX-M-puro vector (Addgene, 125839). HEK293T cells were transfected by Lipofectamine 2000 for lentivirus packaging. Lentivirus was collected and filtered with 0.45 mm filter 48 h after transfection. Cells infected with lentivirus were selected in 1 μg ml−1 puromycin or 1 μg ml−1 blasticidin 48 h after infection. The cells were then analysed using flow cytometry. Cells were washed with PBS containing 0.5% (w/v) BSA (PBS/BSA). APC anti-human CD19 antibody (BioLegend, 302212) was used at a 1:20 dilution. After staining, cells were then washed twice with cold PBS/BSA buffer, resuspended in this buffer and analysed using flow cytometry. To generate R183Q (c.548G>A mutation) targeting the Ppp2r1a gene in Arid1a−/−Pik3caH1047R mouse ovarian clear cell cancer cells, CBE was performed using the sgRNA 5′-CAGCGGCCCGCCGCACCATG-3′ synthesized by IDT. In brief, the sgRNA was cloned into sgRNA-expressing vector containing a mCherry marker (Addgene, 210212). hyBE4max-eGFP (Addgene, 157942) and sgRNA expressing plasmid (10 μg) were co-transfected into 6 cm dish using polyethylenimine (MedChemExpress, HY-K2014). After 72 h, transfected cells were enriched by fluorescence-activated cell sorting through the selection of positive eGFP and mCherry expression. Single clones were then selected and validated for editing efficiency by Sanger sequencing. Cells were lysed with RIPA buffer (Thermo Fisher Scientific, 89901) with protease/phosphatase inhibitor (Thermo Fisher Scientific, A32959) on ice for 30 min. Proteins were denatured using SDS loading buffer (Bio-Rad, 1610747) at 95 °C for 5 min. Proteins were then separated by SDS–PAGE gel and transferred to a nitrocellulose membrane (Bio-Rad, 1620115). The membranes were blocked with 5% non-fat milk and incubated with primary antibodies at 4 °C overnight. Objective signals were amplified with HRP-conjugated secondary antibodies (Cell Signaling, 77074) and detected by chemiluminescent substrate (Bio-Rad, 34094). Human T cells were activated by priming human CD3+ T cells isolated from PBMCs of the healthy donor with human T-Activator CD3/CD28 Dynabeads (11161D, Gibco) and 200 U ml−1 IL-2 (rhIL-2; Prometheus Laboratories, NDC Code 65483-116-07) for 24 h. During the activation, T cells were transduced with human B7H3-(SS1)-hBBZ-CAR encoding g-retrovirus to make T cells targeting human B7H3 in the presence of 10 μg ml−1 protamine sulfate (P3369, Millipore-Sigma) by centrifugation for 2 h at 1,800 rpm at room temperature. Transduction rates of CAR vectors were >85%. The construction of hCD19 CAR-T was performed similarly to as described previously66. In brief, the FMC63 mouse-derived anti-human CD19 single chain variable region67 was synthesized by Genscript and cloned into the pTRPE lentiviral expression plasmid backbone68. SKOV3 cells were prestained with CFSE (565082, BD Bioscience) according to the manual and seeded into the 96-well plate with round-bottom wells before the LB-100 (S7537, Selleck Chemicals) treatment. Tumour cells were mixed with paired B7H3 CAR-T cells at an effector:target ratio of 1:1 at 37 °C for 3 h. The percentage of activated caspase-3+ cells in tumour cells, stained with active caspase-3 antibody (C92-605, 560626, BD Bioscience), was assessed using LSRFortessa X-20 Cell Analyzer (BD Biosciences) and used to determine the tumour apoptosis rate induced by tumour-reactive T cells. A set of genetically modified SKOV3 cell lines was plated into six-well plates (2 × 105 cells per well in completed medium). The next day, the 6-day cultured B7H3 CAR-T cells were co-cultured with SKOV3-derived cells at an effector:target ratio of 1:1 overnight. Adherent cells were collected using 0.25% trypsin-EDTA and stained with Trypan Blue for manual cell counting. The killing efficiency was calculated as (1 − number of live cells in T cell treated group/number of live cells in control group) × 100%. For the cytotoxicity assay of the HEC50B and OVCAR429 cell lines, 10,000 target cells were seeded into 24-well tissue culture plates. hCD19 CAR-T cells were co-cultured with target cells at each effector:target ratio in RPMI 1640 fully supplemented in the absence of cytokines. For treatment, the cell medium was changed every 2 days with appropriate CAR-T cells for a total of 4 days. Next, colonies were stained with 0.05% crystal violet and the integrated density was measured using NIH ImageJ (v.1.54g). The generation of humanized-bone marrow, liver and thymus (Hu-BLT) mice69,70 was performed at The Wistar institute in accordance with Institutional Animal Care and Use Committee (IACUC)-approved protocols (201360 and 201533). In brief, 6-to-8-week-old female NSG mice (NOD.Cg-PrkdcscidIl2rgtm1Wjl/SzJ, Jackson Laboratory) were housed and maintained in individual microisolator cages in a rack system capable of managing air exchange with filters. The room temperature, humidity and pressure were controlled. Mice were maintained at 22–23 °C with 40–60% humidity and under a 12 h–12 h light–dark cycle. All of the mice were pretreated by busulfan at 30 mg per kg and were then implanted with human fetal thymic tissue fragments and fetal liver tissue fragments under the mouse renal capsule. After surgery, the mice were injected through the tail vein with CD34+ haematopoietic stem cells isolated from human fetal liver tissues. Human fetal liver and thymus tissues were procured from Advanced Bioscience Resources with The Wistar Institute IRB 21809310. PDXs were generated using human endometrial tumour samples obtained from Christiana Care under IRB CCC-41045. HLA-A genotyping was performed with fetal liver and PDX tumour. In brief, genomic DNA of human tumour and fetal tissues was extracted using the Monarch Genomic DNA Purification Kit (NEB, T3010S). The HLA-A locus sequence was determined using the Olerup SSP HLA-A genotype kit (CareDx, 101.401-48). Tumour samples were prepared by cutting endometrial PDX tumour chunks into small pieces of around 1 × 2 × 2 mm. Then, 2 weeks after PDX implantation, the mice were randomly assigned into two treatment groups: anti-PD-L1 antibody (Selleck, A2013) and human IgG1 isotype control (Syd Lab, PA007125) (4 mg per kg body weight in 100 μl PBS intraperitoneally, twice a week). After 3 weeks of treatment, the mice were euthanized 1 week later, and the tumours were surgically dissected. The study protocol was approved by The University of Texas MD Anderson Cancer Center (protocol number, 00002399-RN00). For syngeneic models, 2 × 106 mouse OCCC cells were subcutaneously injected into 6-to-8-week-old female immunocompetent C57BL/6 mice (027, 2159769, Charles River Laboratories). Mice were maintained under the same housing conditions as the PDX models, as described above. Then, 9 days after injection, the mice were randomly assigned into two treatment groups: anti-mouse PD-L1 antibody (BioXcell, BE0101) or an IgG2b isotype control (BioXCell, BE0090) (2.5 mg per kg body weight in 100 μl PBS intraperitoneally, once a week). The body weight and tumour size were measured twice a week. After 2 weeks of treatment, the mice were euthanized and the tumours were surgically dissected. Pharmacy records were queried in November 2023 to identify patients with uterine or endometrial cancer who received len–pem combination therapy over the past four years (from September 2019 to November 2023). Demographic and clinicopathological data, including cancer histology, mismatch repair/microsatellite stability status and tumour somatic mutation testing, were abstracted from the medical record under MD Anderson IRB approved protocol 2020-1098. The following histologies were included: serous, clear cell, carcinosarcoma, mesonephric-like adenocarcinoma and mixed high-grade carcinoma, given the relatively higher prevalence of PPP2R1A mutations in these high-risk histologies. The patients were also stratified by TP53 status, given that most OCCCs are wild-type TP53, while most high-grade endometrial carcinomas are TP53-mutated. Patients who never actually received len–pem, those with MSI-H/dMMR tumours and those who had no available imaging after starting treatment were excluded. PFS was calculated from the treatment start date to date of radiographic progression or last imaging study (if no progression). OS was estimated using the Kaplan–Meier method and compared by PPP2R1A mutation status using two-sided log-rank tests. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The raw data and processed gene count matrix and are available for download at the NCBI Gene Expression Omnibus (GEO) under accession number GSE276284. The MSigDB hallmark gene sets are available online (https://www.gsea-msigdb.org/gsea/msigdb). Source data are provided with this paper. All data analysis was performed using publicly available software, packages and tools, as described in the Methods. The computational code used in the analysis is available at GitHub (https://github.com/Coolgenome/OCCC_PPP2R1A), where a detailed description of the analysis flow, including specific code and functions used in the study, is provided to ensure transparency and reproducibility of the results. Disis, M. L. et al. Efficacy and safety of avelumab for patients with recurrent or refractory ovarian cancer: phase 1b results from the JAVELIN solid tumor trial. Hamanishi, J. et al. Nivolumab versus gemcitabine or pegylated liposomal doxorubicin for patients with platinum-resistant ovarian cancer: open-label, randomized trial in japan (NINJA). A. et al. Antitumor activity and safety of pembrolizumab in patients with advanced recurrent ovarian cancer: results from the phase II KEYNOTE-100 study. Zamarin, D. et al. Randomized phase II trial of nivolumab versus nivolumab and ipilimumab for recurrent or persistent ovarian cancer: an NRG oncology study. Liu Y. L. et al. BRCA mutations, homologous DNA repair deficiency, tumor mutational burden, and response to immune checkpoint inhibition in recurrent ovarian cancer. Sugiyama, T. et al. Clinical characteristics of clear cell carcinoma of the ovary: a distinct histologic type with poor prognosis and resistance to platinum-based chemotherapy. Oliver, K. E. et al. An evaluation of progression free survival and overall survival of ovarian cancer patients with clear cell carcinoma versus serous carcinoma treated with platinum therapy: an NRG Oncology/Gynecologic Oncology Group experience. Patterns of clinicopathological features and outcome in epithelial ovarian cancer patients: 35 years of prospectively collected data. Remmerie, M. & Janssens, V. PP2A: a promising biomarker and therapeutic target in endometrial cancer. Leskela, S. et al. Mismatch repair deficiency in ovarian carcinoma: frequency, causes, and consequences. & Lauring, J. Functional analysis of non-hotspot AKT1 mutants found in human breast cancers identifies novel driver mutations: implications for personalized medicine. Foster, K. I. et al. Clinical implications of tumor-based next-generation sequencing in high-grade epithelial ovarian cancer. The post-progression survival of patients with recurrent or persistent ovarian clear cell carcinoma: results from a randomized phase III study in JGOG3017/GCIG. Maxwell, M. B. et al. ARID1A suppresses R-loop-mediated STING-type I interferon pathway activation of anti-tumor immunity. Shen, J. et al. ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade. Goswami, S. et al. ARID1A mutation plus CXCL13 expression act as combinatorial biomarkers to predict responses to immune checkpoint therapy in mUCC. Jiang, T., Chen, X., Su, C., Ren, S. & Zhou, C. Pan-cancer analysis of ARID1A alterations as biomarkers for immunotherapy outcomes. Das, S. & Johnson, D. B. Immune-related adverse events and anti-tumor efficacy of immune checkpoint inhibitors. Tertiary lymphoid structures generate and propagate anti-tumor antibody-producing plasma cells in renal cell cancer. A. et al. B cells and tertiary lymphoid structures promote immunotherapy response. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. He, S. et al. High-plex imaging of RNA and proteins at subcellular resolution in fixed tissue by spatial molecular imaging. Response to tumor-infiltrating lymphocyte adoptive therapy is associated with preexisting CD8+ T-myeloid cell networks in melanoma. Gulati, G. S., D'Silva, J. P., Liu, Y., Wang, L. & Newman, A. M. Profiling cell identity and tissue architecture with single-cell and spatial transcriptomics. Huang, A. C. et al. T-cell invigoration to tumour burden ratio associated with anti-PD-1 response. Defining CD8+ T cells that provide the proliferative burst after PD-1 therapy. Ribas, A. et al. PD-1 blockade expands intratumoral memory T cells. Distinct immune cell populations define response to anti-PD-1 monotherapy and anti-PD-1/anti-CTLA-4 combined therapy. The proportion of circulating CD45RO+CD8+ memory T cells is correlated with clinical response in melanoma patients treated with ipilimumab. Lawson, K. A. et al. Functional genomic landscape of cancer-intrinsic evasion of killing by T cells. Pan, D. et al. A major chromatin regulator determines resistance of tumor cells to T cell-mediated killing. Integrating genome-wide CRISPR immune screen with multi-omic clinical data reveals distinct classes of tumor intrinsic immune regulators. Bitler, B. G. et al. ARID1A-mutated ovarian cancers depend on HDAC6 activity. Fukumoto, T. et al. HDAC6 inhibition synergizes with anti-PD-L1 therapy in ARID1A-inactivated ovarian cancer. Gao J. et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Survival following chemotherapy in ovarian clear cell carcinoma is not associated with pathological misclassification of tumor histotype. Ho, W. S. et al. Pharmacologic inhibition of protein phosphatase-2A achieves durable immune-mediated antitumor activity when combined with PD-1 blockade. Apostolidis, S. A. et al. Phosphatase PP2A is requisite for the function of regulatory T cells. Clark, M. C. et al. A combination of protein phosphatase 2A inhibition and checkpoint immunotherapy: a perfect storm. Protein phosphatase 2A inactivation induces microsatellite instability, neoantigen production and immune response. Mondal, I. et al. PP2Ac deficiency enhances tumor immunogenicity by activating STING-type i interferon signaling in glioblastoma. Contribution of NK cells to immunotherapy mediated by PD-1/PD-L1 blockade. Huuhtanen, J. et al. Single-cell characterization of anti-LAG-3 and anti-PD-1 combination treatment in patients with melanoma. Distinct mechanisms of mismatch-repair deficiency delineate two modes of response to anti-PD-1 immunotherapy in endometrial carcinoma. Hinchcliff, E. M. et al. Randomized phase 2 trial of tremelimumab and durvalumab in combination versus sequentially in recurrent platinum-resistant ovarian cancer. Chen, K. et al. Clinical actionability enhanced through deep targeted sequencing of solid tumors. Guidelines for the evaluation of immune therapy activity in solid tumors: immune-related response criteria. DeLuca, D. S. et al. RNA-SeQC: RNA-seq metrics for quality control and process optimization. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. The Molecular Signatures Database Hallmark Gene Set Collection. Benjamini, Y., Drai, D., Elmer, G., Kafkafi, N. & Golani, I. Controlling the false discovery rate in behavior genetics research. Determining cell type abundance and expression from bulk tissues with digital cytometry. Song, L. et al. TRUST4: immune repertoire reconstruction from bulk and single-cell RNA-seq data. Rodriguez, S., Sun, B., McAllen, S., Jiang, M. & Parra, E. R. Multiplexed barcoding image analysis for immunoprofiling and spatial mapping characterization in the single-cell analysis of paraffin tissue samples. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Schmidt U., Weigert M., Broaddus C., Myers G. Cell detection with star-convex polygons. Medical Image Computing and Computer Assisted Intervention—MICCAI 2018 (eds Frangi, A. F. et al.) Vol. Richter, M. F. et al. Phage-assisted evolution of an adenine base editor with improved Cas domain compatibility and activity. Zhang, X. et al. Increasing the efficiency and targeting range of cytidine base editors through fusion of a single-stranded DNA-binding protein domain. Targeting branched N-glycans and fucosylation sensitizes ovarian tumors to immune checkpoint blockade. Nicholson, I. C. et al. Construction and characterisation of a functional CD19 specific single chain Fv fragment for immunotherapy of B lineage leukaemia and lymphoma. Leibman, R. S. et al. Supraphysiologic control over HIV-1 replication mediated by CD8 T cells expressing a re-engineered CD4-based chimeric antigen receptor. Yuan, Z. et al. Recapitulating cross-species transmission of simian immunodeficiency virus SIVcpz to humans by using humanized BLT mice. Targeting the mevalonate pathway suppresses ARID1A-inactivated cancers by promoting pyroptosis. This work was supported by the National Institutes of Health T32 training grant (T32 CA101642 to A.K. ), Dunwoody-Edwards-Reese Philanthropic support, the MD Anderson Cancer Center Ovarian Cancer Moon Shots Program and Support Grant (CA016672; to A.A.J. ), which supports the Biostatistics Resource Group and Tissue Biospecimen and Pathology Resource Group, NCI K07CA201013 (to L.A.M. ), US National Institutes of Health grants (R01CA202919, R01CA239128, R01CA243142, R01CA260661, R01CA276569, P50CA281701 and P50CA272218 (to R.Z.) )), US Department of Defense (OC190181 and OC210124 to R.Z. and the Frank McGraw Memorial Chair in Cancer Research (to A.K.S.). is a CPRIT Scholar in Cancer Research. was in part supported by National Cancer Institute (NCI) grants U01CA294518 and R01CA266280, and research funding provided by the James P. Allison Institute and the Institute for Data Science in Oncology at the University of Texas MD Anderson Cancer Center. is an associate member of the James P. Allison Institute and Focus Area Co-Lead of the Institute for Data Science in Oncology at MD Anderson. This project was also supported by C. Haymaker and The Translational Molecular Pathology Immunoprofiling Laboratory (TMP-IL) at the Department Translational Molecular Pathology, the University of Texas MD Anderson Cancer Center. MD Anderson's Advanced Technology Genomics Core (ATGC) and Institutional Tissue Bank were supported by the Cancer Center Support grant P30CA016672 and the ATGC core was also supported by the National Institutes of Health instrument grant 1S10OD024977. Support of Core Facilities at The Wistar Institute was provided by Cancer Centre Support Grant (CCSG) CA010815. The clinical trials on which the included patients were enrolled were supported by AstraZeneca (NCT03026062) and Bristol-Meyers-Squibb (NCT01928394). These authors contributed equally: Yibo Dai, Anne Knisely, Mitsutake Yano, Minghao Dang These authors jointly supervised this work: Rugang Zhang, Linghua Wang, Amir A. Jazaeri Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Yibo Dai, Minghao Dang, Jennifer A. Wargo & Linghua Wang The University of Texas MD Anderson Cancer Center UTHealth Houston Graduate School of Biomedical Sciences, Houston, TX, USA Yibo Dai, Jennifer A. Wargo, Rugang Zhang, Linghua Wang & Amir A. Jazaeri Department of Gynecologic Oncology and Reproductive Medicine, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Anne Knisely, Sanghoon Lee, Ami Patel, Shrina Patel, Joseph Celestino, R. Tyler Hillman, Shannon N. Westin, Anil K. Sood, Pamela T. Soliman, Aaron Shafer, Larissa A. Meyer, David M. Gershenson, Karen Lu & Amir A. Jazaeri Mitsutake Yano, Chen Wang, Hao Nie & Rugang Zhang Northwestern University, Feinberg School of Medicine, Robert H Lurie Comprehensive Cancer Center, Chicago, IL, USA Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Manoj Chelvanambi, Matthew Lastrapes & Jennifer A. Wargo Immunology, Metastasis and Microenvironment Program, The Wistar Institute, Philadelphia, PA, USA Heng Liu, Zhe Yuan & Luis J. Montaner Helen F. Graham Cancer Center & Research Institute, Newark, DE, USA Department of Biostatistics, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Baohua Sun, Renganayaki Krishna Pandurengan & Edwin Roger Parra Cuentas Department of Anatomic Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Abdominal Imaging, The University of Texas MD Anderson Cancer Center, Houston, TX, USA The James P. Allison Institute, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Institute for Data Science in Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar supervised clinical data analysis and interpretation. supervised bioinformatic data analysis and interpretation. performed the multiplexed protein imaging assay. were involved in initial protocol development and patient enrolment. performed pathological review of clinical samples. performed radiological review of the clinical cases and participated in clinical response evaluation in the trials. participated in drafting and revising the manuscript. Correspondence to Rugang Zhang, Linghua Wang or Amir A. Jazaeri. reports a consulting role at Merck, Onxeo, ImmunoGen, GSK, Astra Zeneca and Iylon. The other authors declare no competing interests. Nature thanks Dmitriy Zamarin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Flow chart showing overview of patients included in the study cohort and samples analysed by translational assays. b. Oncoplot showing genetic alterations reported in the study cohort. P value was calculated using one-sided log-rank test. GSEA of PPP2R1Amut AKTWT versus PPP2R1AWT AKTWT samples in pretreatment (a) and on-treatment (b) timepoints, showing the enrichment of immune-related signatures in the former group. c. GSEA based on longitudinal comparisons of on-treatment versus pretreatment samples in the PPP2R1A-mutant group, showing the upregulated immune signatures upon ICB treatment. d-g. Longitudinal changes of the relative abundance of CD8+ T cells (d), activated NK cells (e), resting NK cells (f), and total NK cells (g) in all samples from the PPP2R1A-mutant and WT groups. Longitudinal changes of TCR (h) and BCR (i) richness in all samples from the PPP2R1A-mutant and WT groups. For panels d-i, P values were calculated using two-sided Wilcoxon rank sum tests. For panels a-d, P values were calculated using two-sided Wilcoxon rank sum tests. For panels e-h, P values were calculated using two-sided Wilcoxon signed rank tests. d. Representative images from multiplex protein imaging, showing TLSs from the pretreatment samples of PPP2R1A-mutant patients (patient 162 and patient 124. f-g. The cellular densities of all immune cells (f), and CD45+ CD56+ NK cells (g) in on-treatment samples grouped by PPP2R1A mutation status. The amount of NK cells (CD45+ CD4− CD8− CD56+) in the neighbourhood of all, MHC I+, and MHC I− tumour cells. 5h for cell counts of corresponding groups. A representative image from multiplex protein imaging, showing NK cells in the neighbourhood of tumour cells in a PPP2R1A-mutant on-treatment sample. For panels a-c and f-i, P values were calculated using two-sided Wilcoxon rank sum tests. a. Pathway analysis based on the transcriptomic-level comparison of samples from long-term versus short-term survivors in the pretreatment timepoint. b-c. Enriched pathways (b) and representative DEGs (c) based on the transcriptomic-level comparison of samples from long-term versus short-term survivors in the on-treatment timepoint. a. Validation of PPP2R1A mutation status in the parental and CRISPR-edited HEC50B cell lines by Sanger-sequencing. c. Validation of PPP2R1A mutation status in the parental and CRISPR-edited OVCAR429 cell lines by Sanger-sequencing. e. Different immune signatures in pretreatment samples with and without AKT alterations. For b and d, three independent experiments were performed, and representative results were shown. For gel source data, see Supplementary Fig. Three independent pairs of PPP2R1A-specific shRNAs (sh#1-3) were used to generate a set of SKOV3 cell lines with PPP2R1A-knockdown (KD). Two independent experiments were performed, with the representative results shown. For gel source data, see Supplementary Fig. SKOV3 transduced with lentiviral vectors encoding PPP2R1A WT and P179R mutant were selected by sorting GFP positive cells. SKOV3 transduced with empty vector was used as the negative control. Enriched CD19+ cancer cells were subsequently used for cytotoxicity assays. e. Validation of PPP2R1A mutational status in PDX by Sanger-sequencing. f-g. Humanized-BLT mice bearing PPP2R1A-WT endometrial cancer PDXs were randomized into the indicated treatment groups. Images of tumours from the indicated groups are shown at the end of treatment (f). For panel g, P values were calculated using two-sided Student's t-test. Kaplan-Meier survival analysis of the impact of PPP2R1A mutation status on the OS of patients receiving immunotherapy (a) or other therapies (b) from large pan-cancer cohorts (Samstein et al., 2019; Zehir et al., 2017). For panels a-d, P values were calculated using two-sided log-rank tests. Kaplan-Meier survival analysis of the impact of PPP2R1A mutation status on the OS (a) and PFS (b) of uterine cancer patients with high-risk histology who received combined len-pem therapies. c-d. Kaplan-Meier survival analysis of the impact of PPP2R1A mutation status on the OS (c) and PFS (d) of uterine cancer patients with TP53 mutations who received combined len-pem therapies. Same as a-d, but for uterine cancer patients in the same cohort with WT ARID1A. For panels a-h, P values were calculated using two-sided log-rank test. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Dai, Y., Knisely, A., Yano, M. et al. PPP2R1A mutations portend improved survival after cancer immunotherapy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01448-y'>Cancer biomarkers discovered using pan-cancer plasma proteomic profiling</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'nature.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 13:48:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Biomedical Engineering Mass-spectrometry-based proteomic data of tumour patient plasma samples present opportunities for improving cancer detection. Here we generate plasma proteomic profiles from 2,251 pan-cancer patient samples and investigate potential diagnostic biomarkers. Proteomic subtyping with different dominant tumour types links proteomic features and clinical indicators such as tumour stage. The highly immune-activated subtype, consisting of renal and bladder cancers, shows elevated glucose–insulin metabolism and reduced lipid metabolism. Comparison of the plasma proteome before and after surgery indicates that proteome patterns could be used to monitor post-surgery therapeutic effects. We also develop a binary classified model that distinguishes between tumour types and healthy controls, as well as a multicancer model for pan-cancer classification of proteins that could be useful biomarkers, and validate their performance in an independent cohort. In addition, we find that the plasma proteome, along with clinical indicators, whole blood cells and so on, can distinguish the pathological subtypes of specific tumour types. This study portrays a pan-cancer plasma proteomic landscape, providing information on plasma biomarkers that could help in discovering diagnostic opportunities. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 digital issues and online access to articles Prices may be subject to local taxes which are calculated during checkout Proteome raw data have been deposited in the ProteomeXchange Consortium (dataset identifier: PXD051137) via the iProX partner repository (https://www.iprox.cn/) under Project ID: IPX0003227000 (ref. The main data supporting the results in this study are available within the paper and its Supplementary Information. Source data are provided with this paper. All code for computational analyses were derived from publicly available websites and previous publications, and are cited in the corresponding Methods sections. The code used for this study is deposited in GitHub at https://doi.org/10.5281/zenodo.15179267 (ref. Siegel, R. L., Miller, K. D. & Jemal, A. Gao, Q. et al. Integrated proteogenomic characterization of HBV-related hepatocellular carcinoma. Xu, J. Y. et al. Integrative proteomic characterization of human lung adenocarcinoma. Cao, L. et al. Proteogenomic characterization of pancreatic ductal adenocarcinoma. Dong, L. et al. Proteogenomic characterization identifies clinically relevant subgroups of intrahepatic cholangiocarcinoma. Ying, W. Phenomic studies on diseases: potential and challenges. Crowley, E., Di Nicolantonio, F., Loupakis, F. & Bardelli, A. Liquid biopsy: monitoring cancer-genetics in the blood. Marrugo-Ramírez, J., Mir, M. & Samitier, J. Blood-based cancer biomarkers in liquid biopsy: a promising non-invasive alternative to tissue biopsy. Bettegowda, C. et al. Detection of circulating tumor DNA in early- and late-stage human malignancies. Thierry, A. R., El Messaoudi, S., Gahan, P. B., Anker, P. & Stroun, M. Origins, structures, and functions of circulating DNA in oncology. Kilgour, E., Rothwell, D. G., Brady, G. & Dive, C. Liquid biopsy-based biomarkers of treatment response and resistance. Ren, L., Shi, L. & Zheng, Y. Reference materials for improving reliability of multiomics profiling. Bronkhorst, A. J., Ungerer, V. & Holdenrieder, S. The emerging role of cell-free DNA as a molecular marker for cancer management. Hammarström, S. The carcinoembryonic antigen (CEA) family: structures, suggested functions and expression in normal and malignant tissues. Tempero, M. A. et al. Relationship of carbohydrate antigen 19-9 and Lewis antigens in pancreatic cancer. Taketa, K. Alpha-fetoprotein: reevaluation in hepatology. Cohen, J. D. et al. Detection and localization of surgically resectable cancers with a multi-analyte blood test. Nedelkov, D. Mass spectrometry-based immunoassays for the next phase of clinical applications. Blood molecular markers associated with COVID-19 immunopathology and multi-organ damage. Torsetnes, S. B. et al. Multiplexing determination of small cell lung cancer biomarkers and their isovariants in serum by immunocapture LC–MS/MS. Martínez-Jiménez, F. et al. A compendium of mutational cancer driver genes. Gillette, M. A. et al. Proteogenomic characterization reveals therapeutic vulnerabilities in lung adenocarcinoma. Zhong, Z. et al. Cyclin D1/cyclin-dependent kinase 4 interacts with filamin A and affects the migration and invasion potential of breast cancer cells. Sheng, F. et al. Chromium (VI) promotes EMT by regulating FLNA in BLCA. Ge, S. et al. A proteomic landscape of diffuse-type gastric cancer. Integrated omics of metastatic colorectal cancer. Li, Y. et al. Exosomal circPABPC1 promotes colorectal cancer liver metastases by regulating HMGA2 in the nucleus and BMP4/ADAM19 in the cytoplasm. Kim, J. K. et al. Identifying diagnostic microRNAs and investigating their biological implications in rectal cancer. Tang, N. et al. Correlation analysis between four serum biomarkers of liver fibrosis and liver function in infants with cholestasis. Nicholson, A. G. et al. The 2021 WHO classification of lung tumors: impact of advances since 2015. Kargl, J. et al. Neutrophils dominate the immune cell composition in non-small cell lung cancer. Chen, C. H. et al. Upregulation of MARCKS in kidney cancer and its potential as a therapeutic target. Felder, M. et al. MUC16 (CA125): tumor biomarker to cancer therapy, a work in progress. Dochez, V. et al. Biomarkers and algorithms for diagnosis of ovarian cancer: CA125, HE4, RMI and ROMA, a review. Uhlén, M. et al. Proteomics. Tissue-based map of the human proteome. Vilà, M. R., Nicolás, A., Morote, J., de, I. Increased glyceraldehyde-3-phosphate dehydrogenase expression in renal cell carcinoma identified by RNA-based, arbitrarily primed polymerase chain reaction. & Butte, A. J. xCell: digitally portraying the tissue cellular heterogeneity landscape. Yoshihara, K. et al. Inferring tumour purity and stromal and immune cell admixture from expression data. Role of lipids in the metabolism and activation of immune cells. Vitale, C. et al. Venous thromboembolism and lung cancer: a review. Kattula, S., Byrnes, J. R. & Wolberg, A. S. Fibrinogen and fibrin in hemostasis and thrombosis. Jiang, W., Pan, X., Yan, H. & Wang, G. Prognostic significance of the Hsp70 gene family in colorectal cancer. Lee, S. L. et al. in Heat Shock Protein-Based Therapies (eds Asea, A. Szymanski, J. J. et al. Cell-free DNA ultra-low-pass whole genome sequencing to distinguish malignant peripheral nerve sheath tumor (MPNST) from its benign precursor lesion: a cross-sectional study. Sharma, P. et al. The next decade of immune checkpoint therapy. Robinson, J. L., Feizi, A., Uhlén, M. & Nielsen, J. A systematic investigation of the malignant functions and diagnostic potential of the cancer secretome. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Ge, S. et al. Author Correction: A proteomic landscape of diffuse-type gastric cancer. Wilkerson, M. D. & Hayes, D. N. ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. Yu, G., Wang, L. G., Han, Y. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Fabregat, A. et al. The Reactome Pathway Knowledgebase. Kanehisa, M., Furumichi, M., Tanabe, M., Sato, Y. & Morishima, K. KEGG: new perspectives on genomes, pathways, diseases and drugs. Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Ashburner, M. et al. Gene Ontology: tool for the unification of biology. The Gene Ontology Consortium. Hänzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. Bai, L. et al. Cancer biomarkers discovered using pan-cancer plasma proteomic profiling. Bai, L. et al. Cancer biomarkers discovered using pan-cancer plasma proteomic profiling. This work was supported by the National Key Research and Development Program of China (2022YFA1303200 [C.D. ]), the National Natural Science Foundation of China (32330062 [C.D. ]), sponsored by the Program of Shanghai Academic/Technology Research Leader (22XD1420100 [C.D. ]), the Major Project of Special Development Funds of Zhangjiang National Independent Innovation Demonstration Zone (ZJ2019-ZD-004 [C.D. ]), the Shanghai Municipal Science and Technology Major Project (2023SHZDZX02 [C.D. ]), the Fudan Original Research Personalized Support Project [C.D. ], Cultivation Project of Precision Medicine Joint Fund of Hebei Natural Science Foundation (H2022201044 [Y.J. ]), Government-funded clinical talents Training Project of Hebei Province, Hebei Province ‘Three three Three Talent Project' supported project (C20221002 [Y.J. ]), the National Natural Science Foundation of China (82273792 [Y.J. ]), Innovative team for precise care and rehabilitation of patients with cancer (IT2023C07 [Y.J. ]), the Young Scientists Fund of the National Natural Science Foundation of China (32201215 [J.F. ]), the National Ten Thousand Plan Young Top Talents [Y.Q. ], the Natural Science Foundation of China (82172817 [Y.Q]), and the Shanghai ‘Science and Technology Innovation Action Plan' medical innovation research project (22Y11905100 [Y.Q]). This work is supported by the Shanghai Municipal Science and Technology Major Project, the Human Phenome Data Center of Fudan university, and the Shanghai Phenomic precision measurement professional technical service platform (23DZ2290800). These authors contributed equally: Lin Bai, Jiacheng Lyu, Jinwen Feng, Xiaoqiang Qiao, Yuanyuan Qu, Guojian Yang, Yuanxue Zhu, Lingxiao Liao. Clinical Research Center for Cell-based Immunotherapy of Shanghai Pudong Hospital, Fudan University Pudong Medical Center, State Key Laboratory of Genetics and Development of Complex Phenotypes, School of Life Sciences, Human Phenome Institute, Department of Urology, Fudan University Shanghai Cancer Center, Center for Cell and Gene Therapy, Fudan University, Shanghai, China Lin Bai, Jiacheng Lyu, Jinwen Feng, Yuanyuan Qu, Guojian Yang, Zeya Xu, Tao Ji, Peng Ran, Hailiang Zhang, Lingli Zhu, Sha Tian, Subei Tan, Yan Pu, Lingling Li, Dingwei Ye & Chen Ding Department of Medical Oncology, Affiliated Hospital of Hebei University, Hebei Key Laboratory of Cancer Radiotherapy and Chemotherapy, Baoding, Hebei, China Xiaoqiang Qiao, Yuanxue Zhu, Aimin Zang, Xiaofang Wang, Yumiao Li, Guofa Zhao, Zizheng Song, Jin Song & Youchao Jia College of Pharmaceutical Sciences, Key Laboratory of Medicinal Chemistry and Molecular Diagnosis, Ministry of Education, Hebei University, Baoding, China Department of Oncology, Shanghai Medical College, Shanghai Genitourinary Cancer Institute, Shanghai, China Yuanyuan Qu, Hailiang Zhang & Dingwei Ye Department of Neurosurgery, The Affiliated Nanhua Hospital, Hengyang Medical School, University of South China, Hengyang, China Hebei Normal University of Science and Technology, Qinhuangdao, China Clinical Lab, Affiliated Hospital of Hebei University, Baoding, Hebei, China Department of Hematology, Beijing Tongren Hospital, Capital Medical University, Beijing, China Department of Hepatological Surgery, Affiliated Hospital of Hebei University, Baoding, Hebei, China CT/MRI room, Affiliated Hospital of Hebei University, Baoding, Hebei, China College of Clinical Medicine, Hebei University, Baoding, China Dan Liu & Xiangpeng Gao Department of Medical Oncology, Affiliated Hospital of Hebei University, Baoding, China Dan Liu & Xiangpeng Gao Departments of Cancer Research Institute, Affiliated Cancer Hospital of Xinjiang Medical University, Xinjiang Key Laboratory of Translational Biomedical Engineering, Urumqi, China Wenjia Guo & Chen Ding Department of Pediatric Orthopedics, Spine Center, Xin Hua Hospital Affiliated to, Shanghai Jiao Tong University School of Medicine, Shanghai, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and planned the project. were responsible for samples and clinical information collection. contributed to sample preparation. All authors provided critical feedback and helped shape the research, analysis and paper. Correspondence to Yongshi Liao, Dingwei Ye, Wenjun Yang, Youchao Jia or Chen Ding. The authors declare no competing interests. Nature Biomedical Engineering thanks Jens Nielsen, Karin D. Rodland and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Clinical characteristics of pan-cancer patients, and proteome data matrix used in the paper. Plasma proteomic subtyping revealing heterogeneity of pan-cancer. Differential plasma proteomes of different physiological system groups. Specific plasma protein patterns distinguish different tumour types and subtypes. Integration of tissue and plasma proteomes to nominate core tumour biomarkers. Immune infiltration in pan-cancer tumours. Short-term changes of the plasma proteome after surgery. Machine learning-based model construction for binary and multitumour classification. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Bai, L., Lyu, J., Feng, J. et al. Cancer biomarkers discovered using pan-cancer plasma proteomic profiling. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biomedical Engineering © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250701020711.htm'>Antarctica's ocean flip: Satellites catch sudden salt surge melting ice from below</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 12:56:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have discovered a dramatic and unexpected shift in the Southern Ocean, with surface water salinity rising and sea ice in steep decline. For decades, the ocean's surface freshened (becoming less salty), helping sea ice grow. Now, scientists say that trend has sharply reversed. Using European satellite data, research led by the University of Southampton has discovered a sudden rise in surface salinity south of 50° latitude. Dr Alessandro Silvano from the University of Southampton who led the research said: "Saltier surface water allows deep ocean heat to rise more easily, melting sea ice from below. It's a dangerous feedback loop: less ice leads to more heat, which leads to even less ice. The effects are already global: stronger storms, warmer oceans, and shrinking habitats for penguins and other iconic Antarctic wildlife." Now, new satellite technology, combined with information from floating robotic devices which travel up and down the water column, shows this trend has reversed; surface salinity is increasing, stratification is weakening, and sea ice has reached multiple record lows -- with large openings of open ocean in the sea ice (polynyas) returning. Contrary to the new findings, man-made climate change was generally expected to sustain Antarctic Sea ice cover over the coming years. Aditya Narayanan, a postdoctoral research fellow at the University of Southampton and co-author on the paper, explains: "While scientists expected that human-driven climate change would eventually lead to Antarctic Sea ice decline, the timing and nature of this shift remained uncertain. "Previous projections emphasized enhanced surface freshening and stronger ocean stratification, which could have supported sustained sea ice cover. Instead, a rapid reduction in sea ice -- an important reflector of solar radiation -- has occurred, potentially accelerating global warming." Professor Alberto Naveira Garabato, co-author of the study and Regius Professor of Ocean Sciences at the University of Southampton added: "The new findings suggest that our current understanding may be insufficient to accurately predict future changes." The paper Rising surface salinity and declining sea ice: a new Southern Ocean state revealed by satellites is published in Proceedings of the National Academy of Sciences and is available online. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/chinas-tianwen-3-mission-could-beat-the-u-s-in-the-race-to-grab-mars-rocks/'>This Is How China Plans to Win the Race to Return Rocks from Mars</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>How China Could Win the Race to Return Rocks from Mars China's first Mars rover, Zhurong, with its Tianwen-1 landing platform, which touched down on the Red Planet in May 2021. On May 14, 2021, China's Tianwen-1 lander plummeted from space to streak through the skies above Mars's vast plain of Utopia Planitia, with an aeroshell protecting it from the heat and plasma of its high-speed atmospheric entry. This engineering feat was hugely significant, confirming China as a major player in planetary exploration. That project, Tianwen-3, is set to launch via two Long March 5 rockets from Wenchang spaceport on the Chinese island of Hainan in late 2028. The mission aims to collect samples of Martian rock and soil for delivery back to Earth, where subsequent studies could, potentially, redefine our understanding of life itself and our place in the cosmos. (Tianwen-2, another Chinese sample return mission, launched in May 2025 but is bound for a near-Earth asteroid as well as a comet; China has also pulled off two successful lunar sample return missions, Chang'e 5 and Chang'e 6.) If you're enjoying this article, consider supporting our award-winning journalism by subscribing. Tianwen-3 will use the same approach as Tianwen-1 to make planetfall in a yet-to-be-selected landing area. The lander will use proven tech from China's lunar explorations, drilling as deep as two meters for subsurface samples and scooping up material from the surface. Additionally, a helicopter drone—following the lead of NASA's pioneering Ingenuity flights—will collect selected additional rock and loose particle samples from within around 100 meters of Tianwen-3's landing site. The lander will operate for around two months on the surface, in which it will use scientific payloads such as a ground penetrating radar and a Raman spectrometer to gather more data on the broader geological context for collected samples. When the time comes, it will fire off a solid rocket booster to send a canister containing at least 500 grams of material into Mars orbit to dock with the waiting orbiter-returner spacecraft. The eventual return trip should bring Tianwen-3's samples home sometime in 2031. Once on Earth, the samples will be swiftly secured and transported to a custom-built Mars sample laboratory, where they'll be extensively analyzed while carefully following “planetary protection” policies meant to prevent any potential otherworldly cross-contamination. Although Tianwen-3's objectives range from investigating Mars's interior to studying its wispy atmosphere, scientists working on the mission are most eager for what it might reveal about the planet's possible former or even extant life. If, in fact, anything ever dwelled on the Red Planet, then Tianwen-3's samples could conceivably contain various telltale signs of its presence—so-called biosignatures. A convincing biosignature could come in different types, Li says. Another biosignature is biogenic isotope fractionation—the distinctive way that living organisms alter the natural ratios of stable isotopes in their ecosystems; on Earth, for instance, biochemical processes such as photosynthesis prefer the lighter carbon-12 rather than heavier carbon isotopes, leading to detectable shifts in the proportions of these isotopes with respect to the surrounding environment. A third approach, Li adds, would be to look for fossil evidence, such as the imprints that microbes may leave behind in mudstones and other fine-grained sedimentary rocks. China's astrobiological focus for its Martian explorations is “a laudable, ambitious goal,” says Mahesh Anand, a professor of planetary science and exploration at the Open University in England. “This is exactly what we have been recommending over the years: to look for any signs of biogenic activity or even just to understand that there was a habitable environment. China's approach is simpler in many ways than the U.S.-led Mars Sample Return (MSR) mission, which is a joint project between NASA and the European Space Agency (ESA). MSR's workhorse, NASA's Perseverance rover, is already on Mars, where it has spent more than four years collecting dozens of carefully selected samples from Jezero Crater, a diverse site harboring an ancient river delta and other complex geological features that may preserve evidence of past life. And the spacecraft's landing ellipse—the area within which Tianwen-3 is most likely to touch down—spans some 50 by 20 kilometers, meaning a precision touchdown to visit any especially alluring targets is highly unlikely. “The reason why NASA went with Perseverance as this first step was so that you would have this curated, intentionally selected and well-recorded process and contextual process of where these rocks came from,” says Casey Dreier, chief of space policy at the Planetary Society, a U.S. space science advocacy group. Dreier says that this moment, in which the U.S. appears to be ceding leadership in this area to China, will have implications for global space exploration. If the White House has its way, he adds, then Perseverance's samples might only find their way back to Earth after an even more complex, expensive and distant human spaceflight program led by SpaceX lands astronauts on Mars. Which means, for now, China will get its shot at a major first in space exploration. The richer science on offer from a complex MSR-style plan is enticing, he says, but sticking to simplicity and clearly achievable near-term results “probably has a higher chance of returning science than planning on something that might take decades.” For China, retrieving Martian material fits into the strategic framework of its broader, solar system-encompassing Tianwen program, the name of which translates to “heavenly questions.” Beyond Tianwen-3 and its already-launched asteroid-and-comet-bound sibling Tianwen-2, there is also Tianwen-4, slated for liftoff around 2029, which will target the Jupiter system and its intriguing Galilean moon Callisto. For Li, Tianwen-3 remains the Tianwen program's most compelling project, in part because its path to Mars and back is so straightforward; although lofty, its objectives still appear eminently within reach. “It is important for humanity to understand its position in the solar system and the universe,” he says, because this would mark a profound milestone in human history. He also covers topics such as planetary exploration, commercial developments and emerging space powers.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/for-dolphins-echolocation-may-be-more-like-touching-than-seeing/'>For Dolphins, Echolocation May Be More Like ‘Touching' Than ‘Seeing'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For Dolphins, Echolocation May Be More Like ‘Touching' Than ‘Seeing' Dolphins seem to “feel” their way across the sea with narrow, sweeping beams of sonar Trying to unlock your car, you fumble and drop the keys. You squat down and run your hand across the invisible pavement. To the left, you feel a firm, rubbery tire. Reversing course, you pass over jagged pebbles and papery leaves. Finally your fingers discover—and instantly close around—a notched piece of metal. This kind of tactile exploration may be the closest we can get to imagining the experience of dolphin echolocation, say the authors of a study on dolphin brains that was recently published in PLOS ONE. We typically imagine echolocation as “seeing” with sound—experiencing auditory signals as a world of images like the ones our brains typically create with light from our eyes. Like sonar, which turns sonic waves into visual representations, echolocators emit sounds and then decode spatial and textural information in the echoes that bounce back. “It made a neat little story because you have visual and auditory [brain regions] right next to each other,” says Lori Marino, a neuroscientist and president of the Whale Sanctuary Project, who was not an author of the new study but is mentioned in its acknowledgments section. She adds, however, that thanks to today's more precise technology, “the whole [research] landscape is changing.” Although we still can't translate echolocation perfectly into human terms, the new findings suggest a better metaphor: “touching” with sound. They measured the diffusion of the movement of water molecules along nerve fibers—like cars along a highway, as Marino puts it—to better understand which parts of the brain interact in living dolphins and in sei whales. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “Anytime you need to move quickly, decisively, and without consciously deliberating, your cerebellum comes alive,” says Peter Cook, a comparative neuroscientist at the New College of Florida and senior author of the new study. He and his colleagues found a strong connection between these two brain structures in the dolphins but not in the sei whale. This makes sense to lead author Sophie Flem, a master's student at the New College of Florida. If you need to constantly fine-tune your movements to home in on prey, Flem says, “it does seem intuitive that something like a cerebellum would really help.” And there's another way in which echolocation seems more like touch than vision: a dolphin's sonar beam is far narrower than our visual field. Whereas we take in 180 degrees at a glance, dolphins move their beam around and build spatial understanding gradually—like a human groping for dropped keys in the dark. Still, it would be hubris to presume we know for certain what an animal's echolocation actually feels like. Cody Cottier is a freelance journalist based in Fort Collins, Colo.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250701020657.htm'>Banned in Europe, sprayed in America: The fungicide threatening our pollinators</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 08:11:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>"This can have a big knock-on population impact over time because it affects both male and female fertility." Even at the lowest dose, flies showed a 37 percent drop in egg production compared with unexposed individuals. Chlorothalonil is a widely applied broad-spectrum fungicide in American agriculture, used on crops like: Supervising author Associate Professor Fleur Ponton says the dramatic decline was unexpected. "We expected the effect to increase far more gradually with higher amounts. Although banned in the European Union, chlorothalonil is extensively applied to Australian crops including orchards and vineyards, often preventatively when no disease is present. The findings add to mounting evidence of global insect population decline, with some regions reporting drops exceeding 75 percent in recent decades. The research highlights a critical knowledge gap in pesticide regulation, with fewer than 25 scientific papers examining chlorothalonil's effects on insects despite its widespread use. The researchers recommend more sustainable practices, including reduced application frequency to allow insect population recovery between treatments. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250701234739.htm'>Researchers tested 200 toddlers — 96 chemicals were lurking in their bodies</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 04:56:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A national study published in Environmental Science & Technology finds children aged 2 to 4 years in the United States are routinely exposed to a broad range of potentially harmful chemicals. The researchers analyzed urine samples from 201 children aged 2 to 4 years. "Our study shows that childhood exposure to potentially harmful chemicals is widespread. This is alarming because we know early childhood is a critical window for brain and body development," said Deborah H. Bennett, lead author and UC Davis professor in the Department of Public Health Sciences. "Many of these chemicals are known or suspected to interfere with hormones, brain development and immune function." The NIH-funded ECHO Cohort combines data from pregnancy and pediatric cohorts to examine the impacts of early environmental exposures on child health and development. This study looked at samples of 201 children from four states (California, Georgia, New York and Washington). The researchers looked for childhood exposure to common environmental chemicals, including: Most of the children's mothers had provided urine samples during pregnancy. They found the children had higher levels of several chemicals than their mothers did during pregnancy. "Exposure to certain chemicals in early childhood -- such as pesticides, plasticizers and flame retardants -- has been linked to developmental delays, hormone disruption and other long-term health issues," said Jiwon Oh, first author of the study and a postdoctoral scholar in the UC Davis Department of Public Health Sciences. "This new study highlights the urgent need for expanded biomonitoring and stronger regulations to protect children from harmful exposures." A complete list of authors and funders appears in the paper. Yet, there are many simple steps parents can take to help reduce their children's contact with harmful chemicals. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250701234735.htm'>Ultrafast 12-minute MRI maps brain chemistry to spot disease before symptoms</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-02 04:30:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new technology that uses clinical MRI machines to image metabolic activity in the brain could give researchers and clinicians unique insight into brain function and disease, researchers at the University of Illinois Urbana-Champaign report. The non-invasive, high-resolution metabolic imaging of the whole brain revealed differences in metabolic activity and neurotransmitter levels among brain regions; found metabolic alterations in brain tumors; and mapped and characterized multiple sclerosis lesions -- with patients only spending minutes in an MRI scanner. "Understanding the brain, how it works and what goes wrong when it is injured or diseased is considered one of the most exciting and challenging scientific endeavors of our time," Liang said. Conventional MRI provides high-resolution, detailed imaging of brain structures. Functional MRI maps brain activity by detecting changes in blood flow and blood oxygenation level, which are closely linked to neural activity. "Metabolic imaging, therefore, can lead to early diagnosis and intervention of brain diseases." Both MRI and fMRI techniques are based on magnetic resonance signals from water molecules. "Our technology overcomes several long-standing technical barriers to fast high-resolution metabolic imaging by synergistically integrating ultrafast data acquisition with physics-based machine learning methods for data processing," Liang said. In healthy subjects, the researchers found and mapped varying metabolic and neurotransmitter activity across different brain regions, indicating that such activity is not universal. In subjects with multiple sclerosis, the technique detected molecular changes associated with neuroinflammatory response and reduced neuronal activity up to 70 days before changes become visible on clinical MRI images, the researchers report. The researchers foresee potential for broad clinical use of their technique: By tracking metabolic changes over time, clinicians can assess the effectiveness of treatments for neurological conditions, Liang said. "Paul envisioned this exciting possibility and the general approach, but it has been very difficult to achieve his dream of fast high-resolution metabolic imaging in the clinical setting. "As healthcare is moving towards personalized, predictive and precision medicine, this high-speed, high-resolution technology can provide a timely and effective tool to address an urgent unmet need for noninvasive metabolic imaging in clinical applications." This work was supported by the Arnold and Mabel Beckman Foundation. Materials provided by University of Illinois at Urbana-Champaign, News Bureau. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            