
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09634-3'>A long-necked early dinosaur from a newly discovered Upper Triassic basin in the Andes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:09:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article During the Late Triassic epoch (237–201 million years ago), the terrestrial ecosystems of Pangaea underwent drastic changes1,2 that led to the rise and diversification of mammaliaforms3, crocodylomorphs4 and dinosaurs5. Although the Carnian sedimentary rocks of South America provided much of the available evidence for understanding the early evolution of these clades, key discoveries have remained restricted to the Ischigualasto-Villa Unión6,7 and Paraná basins8 in Argentina and Brazil, respectively. Here we report a Carnian tetrapod assemblage from the previously unrecognized Northern Precordillera Basin in northwestern Argentina. Discoveries at this basin, in the Quebrada Santo Domingo site, include a nearly complete skeleton of the early sauropodomorph Huayracursor jaguensis gen. et sp. nov., and typical components of Late Carnian faunas, such as hyperodapedontine rhynchosaurs, gomphodontosuchine traversodontid cynodonts, and aetosaurs. Compared to its generally small and short-necked Carnian counterparts9,10, Huayracursor is larger and exhibits an incipient elongation of its cervical vertebrae, representing an intermediate condition for size and cervical elongation between known Carnian and Norian sauropodomorphs11. This discovery provides one of the oldest pieces of evidence of increased body mass and neck elongation in early Sauropodomorpha. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access $199.00 per year only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The published article and its Supplementary Information files include all data generated or analysed during this study. The data matrix used in the phylogenetic analysis, as well as the phylogenetic results, are available at Figshare (https://doi.org/10.6084/m9.figshare.29136656)67. Dal Corso, J. et al. Extinction and dawn of the modern world in the Carnian (Late Triassic). Singh, S. A. et al. Niche partitioning shaped herbivore macroevolution through the early Mesozoic. Rougier, G. W., Martinelli, A. G. & Forasiepi, A. M. Mesozoic Mammals from South America and their Forerunners (Springer, 2021). Leardi, J. M., Yáñez, I. & Pol, D. South American crocodylomorphs (Archosauria; Crocodylomorpha): a review of the early fossil record in the continent and its relevance on understanding the origins of the clade. South Am. Earth Sci. Bernardi, M., Gianolla, P., Petti, F. M., Mietto, P. & Benton, M. J. Dinosaur diversification linked with the Carnian Pluvial Episode. Martínez, R. N. et al. Vertebrate succession in the Ischigualasto Formation. Desojo, J. B. et al. The Late Triassic Ischigualasto Formation at Cerro Las Lajas (La Rioja, Argentina): fossil tetrapods, high-resolution chronostratigraphy, and faunal correlations. Schultz, C. L. et al. Triassic faunal successions of the Paraná Basin, southern Brazil. South Am. Earth Sci. Langer, M. C. et al. in South American Sauropodomorph Dinosaurs (eds Otero, A. et al.) 1–49 (Springer, 2022). Pol, D., Otero, A., Apaldetti, C. & Martínez, R. N. Triassic sauropodomorph dinosaurs from South America: The origin and diversification of dinosaur dominated herbivorous faunas. South Am. Earth Sci. Müller, R. T., Langer, M. C. & Dias-da-Silva, S. An exceptionally preserved association of complete dinosaur skeletons reveals the oldest long-necked sauropodomorphs. Novas, F. E. et al. Review of the fossil record of early dinosaurs from South America, and its phylogenetic implications. South Am. Earth Sci. Pretto, F. A., Langer, M. C. & Schultz, C. L. A new dinosaur (Saurischia: Sauropodomorpha) from the Late Triassic of Brazil provides insights on the evolution of sauropodomorph body plan. Bronzati, M., Müller, R. T. & Langer, M. C. Skull remains of the dinosaur Saturnalia tupiniquim (Late Triassic, Brazil): with comments on the early evolution of sauropodomorph feeding behaviour. PLoS ONE 14, e0221387 (2019). Müller, R. T., Ferreira, J. D., Pretto, F. A., Bronzati, M. & Kerber, L. The endocranial anatomy of Buriolestes schultzi (Dinosauria: Saurischia) and the early evolution of brain tissues in sauropodomorph dinosaurs. Martínez, R. N. & Alcober, O. A basal sauropodomorph (Dinosauria: Saurischia) from the Ischigualasto Formation (Triassic, Carnian) and the early evolution of Sauropodomorpha. PLoS ONE 4, e4397 (2009). Apaldetti, C., Martínez, R. N., Pol, D. & Souter, T. Redescription of the skull of Coloradisaurus brevis (Dinosauria, Sauropodomorpha) from the Late Triassic Los Colorados Formation of the Ischigualasto-Villa Unión Basin, northwestern Argentina. Müller, R. T. et al. Early evolution of sauropodomorphs: anatomy and phylogenetic relationships of a remarkably well-preserved dinosaur from the Upper Triassic of southern Brazil. Sereno, P. C., Martínez, R. N. & Alcober, O. A. Osteology of Eoraptor lunensis (Dinosauria, Sauropodomorpha). Damke, L. V. S., Bem, F. P., Doering, M., Piovesan, T. R. & Müller, R. T. The elongated neck of sauropodomorph dinosaurs evolved gradually. Langer, M. C., Franca, M. A. G. & Gabriel, S. The pectoral girdle and forelimb anatomy of the stem-sauropodomorph Saturnalia tupiniquim (Upper Triassic, Brazil). Griffin, C. T. et al. Africa's oldest dinosaurs reveal early suppression of dinosaur distribution. Langer, M. C., Bittencourt, J. S. & Schultz, C. L. A reassessment of the basal dinosaur Guaibasaurus candelariensis, from the Late Triassic Caturrita Formation of south Brazil. Langer, M. C., McPhee, B. W., Marsola, J. C. A., Roberto-da-Silva, L. & Cabreira, S. F. Anatomy of the dinosaur Pampadromaeus barberenai (Saurischia—Sauropodomorpha) from the Late Triassic Santa Maria Formation of southern Brazil. PLoS ONE 14, e0212543 (2019). McPhee, B. W., Bittencourt, J. S., Langer, M. C., Apaldetti, C. & Da Rosa, Á. A. S. Reassessment of Unaysaurus tolentinoi (Dinosauria: Sauropodomorpha) from the Late Triassic (early Norian) of Brazil, with a consideration of the evidence for monophyly within non-sauropodan sauropodomorphs. Martínez, R. N. Adeopapposaurus mognai, gen. et sp. nov. (Dinosauria: Sauropodomorpha), with comments on adaptations of basal Sauropodomorpha. Otero, A. & Pol, D. Postcranial anatomy and phylogenetic relationships of Mussaurus patagonicus (Dinosauria, Sauropodomorpha). Ballell, A., Rayfield, E. J. & Benton, M. J. Osteological redescription of the Late Triassic sauropodomorph dinosaur Thecodontosaurus antiquus based on new material from Tytherington, southwestern England. Ezcurra, M. D. A new early dinosaur (Saurischia: Sauropodomorpha) from the Late Triassic of Argentina: a reassessment of dinosaur origin and phylogeny. Damke, L. V. S., Langer, M. C., Da-Rosa, Á. A. S. & Müller, R. T. New specimens of Saturnalia tupiniquim (Dinosauria: Sauropodomorpha): insights into intraspecific variation, rostral anatomy, and skull size. Langer, M. C. The pelvic and hind limb anatomy of the stem-sauropodomorph Saturnalia tupiniquim (Late Triassic, Brazil). Marsola, J. C. A. et al. A new dinosaur with theropod affinities from the Late Triassic Santa Maria Formation, South Brazil. Cabreira, S. F. et al. A unique Late Triassic dinosauromorph assemblage reveals dinosaur ancestral anatomy and diet. Novas, F. E. New information on the systematics and postcranial skeleton of Herrerasaurus ischigualastensis (Theropoda: Herrerasauridae) from the Ischigualasto Formation (Upper Triassic) of Argentina. Novas, F. E., Ezcurra, M. D., Chatterjee, S. & Kutty, T. S. New dinosaur species from the Upper Triassic Upper Maleri and Lower Dharmaram formations of Central India. Otero, A. & Hutchinson, J. R. in South American Sauropodomorph Dinosaurs (eds Otero, A. et al.) 443–472 (Springer, 2022). Müller, R. T. & Garcia, M. S. A sauropodomorph (Dinosauria, Saurischia) specimen from the Upper Triassic of southern Brazil and the early increase in size in Sauropodomorpha. Ezcurra, M. D., Müller, R. T., Novas, F. E. & Chatterjee, S. Osteology of the sauropodomorph dinosaur Jaklapallisaurus asymmetricus from the Late Triassic of central India. Martínez, R. N., Apaldetti, C. & Abelin, D. Basal sauropodomorphs from the Ischigualasto Formation. Colombi, C. et al. A high-precision U–Pb zircon age constraints the timing of the faunistic and palynofloristic events of the Carnian Ischigualasto Formation, San Juan, Argentina. South Am. Earth Sci. Coughlin, T. J. Linked orogen-oblique fault zones in the Central Andes: the basis of a new model for Andean orogenesis and metallogenesis (Univ. Kent, D. V., Santi Malnis, P., Colombi, C. E., Alcober, O. & Martínez, R. N. Age constraints on the dispersal of dinosaurs in the Late Triassic from magnetochronology of the Los Colorados Formation (Argentina). Milana, J. P. & Alcober, O. A. Modelo tectonosedimentario de la cuenca triásica de Ischigualasto (San Juan, Argentina). Fauqué, L. E. et al. Hoja Geológica 2969-IV, Villa Unión, provincias de La Rioja y San Juan (Servicio Geológico Minero Argentino, 2020). Limarino, C. O., Ciccioli, P. L., Krapovickas, V. & Benedito, L. D. Estratigrafía de las sucesiones mesozoicas, paleógenas y neógenas de las quebradas Santo Domingo y el Peñón (Precordillera Septentrional riojana). López Gamundí, O. R. et al. in Cuencas Sedimentarias Argentinas (eds Chebli, G. & Spalletti L. Gulbranson, E. L. et al. Paleoenvironments and age of the Talampaya Formation: the Permo-Triassic boundary in northwestern Argentina. South Am. Earth Sci. Colombi, C. E., Limarino, C. O. & Alcober, O. A. Allogenic controls on the fluvial architecture and fossil preservation of the Upper Triassic Ischigualasto Formation, NW Argentina. Espinoza, M. et al. The synrift phase of the early Domeyko Basin (Triassic, northern Chile): sedimentary, volcanic, and tectonic interplay in the evolution of an ancient subduction-related rift basin. Basin Res. Desojo, J. B. Redescription of the aetosaur Chilenosuchus forttae Casamiquela (Diapsida: Archosauria): presence of continental Triassic in northern Chile. Geológica Chile 30, 53–63 (2003). Cao, W. et al. Improving global paleogeography since the late Paleozoic using paleobiology. Melchor, R. N., de Valais, S. & Genise, J. F. Bird-like fossil footprints from the Late Triassic. Melchor, R. N., Buchwaldt, R. & Bowring, S. A Late Eocene date for Late Triassic bird tracks. Vizán, H. et al. Geological setting and paleomagnetism of the Eocene red beds of Laguna Brava Formation (Quebrada Santo Domingo, northwestern Argentina). Hechenleitner, E. M. et al. Two Late Cretaceous sauropods reveal titanosaurian dispersal across South America. Caminos, R., Zamuner, A. B., Limarino, C. & Fauqué, L. Triásico Superior fosilífero en la Precordillera riojana. Espina, J. & Bodnar, J. in Reunión de Comunicaciones de la Asociación Paleontológica Argentina (eds Forasiepi, A. M. et al.) 52–53 (Asociación Paleontológica Argentina, 2024). Ramos, V. A. The tectonics of the Central Andes; 30 to 33 S latitude. Jordan, T. E. et al. Andean tectonics related to geometry of subducted Nazca plate. Bracaccini, O. Contribución al conocimiento geológico de la Precordillera Sanjuanina-Mendocina. Fauqué, L. E. & Caminos, R. L. Hoja geológica 2969-II, Tinogasta, Provincias de La Rioja, Catamarca y San Juan (Servicio Geológico Minero Argentino, 2006). Müller, R. D. et al. GPlates: building a virtual Earth through deep time. Goloboff, P. A. & Morales, M. E. TNT version 1.6, with a graphical interface for MacOS and Linux, including new routines in parallel. Ezcurra, M. D. Exploring the effects of weighting against homoplasy in genealogies of palaeontological phylogenetic matrices. Campione, N. E., Evans, D. C., Brown, C. M. & Carrano, M. T. Body mass estimation in non‐avian bipeds using a theoretical conversion to quadruped stylopodial proportions. Campione, N. E. & Evans, D. C. A universal scaling relationship between body mass and proximal limb bone dimensions in quadrupedal terrestrial tetrapods. Hechenleitner, E. M. et al. Phylogenetic results. Download references The authors thank the Gobierno de La Rioja, G. Luna and the whole crew of Ministerio de Turismo y Culturas de La Rioja for contributing to developing the Quebrada Santo Domingo Project; the Subsecretaría de Patrimonio Cultural y Museos for granting the field permits; S. de la Vega, C. Bustamante, L. Leuzinger, L. A. Burllaile, M. V. Fernández-Blanco, J. Troteyn, M. Miñana, T. Melo, M. Larrovere, M. Macchioli-Grande and H. Aciar for their contributions to the expeditions at QSD; E. Rayfield, A. Vargas, S. Soto-Acuña and J. Botelho for insightful comments on an early draft and support; R. González, P. González and A. Varas for help during the expeditions; A. Sangiorgio for assistance with computed tomography scans; R. Martínez, R. Müller, F. Pretto, L. Kerber, H. Francischini and C. Schultz for access to the specimens under their care. The Willi Hennig Society provided TNT software. This work was supported by the Paleontological Society International Research Program (Sepkoski Grants), the Agencia Nacional de Promoción Científica y Tecnológica (PICT 2020-1951 and PICT 2018-0717), CONICET PIBAA 2022-2023, ANID – MILENIO – NCN2023_025 and the Ministerio de Turismo y Culturas of La Rioja. Consejo Nacional de Investigaciones Científicas y Técnicas (CONICET), Ciudad Autónoma de Buenos Aires, Argentina E. Martín Hechenleitner, Agustín G. Martinelli, Sebastián Rocher, Lucas E. Fiorelli, Malena Juarez, Jeremías R. A. Taborda & Julia B. Desojo Centro Regional de Investigaciones Científicas y Transferencia Tecnológica de La Rioja (CRILAR), Provincia de La Rioja, UNLaR, SEGEMAR, UNCa, CONICET, Anillaco, Argentina E. Martín Hechenleitner, Lucas E. Fiorelli & Malena Juarez Sección Paleontología de Vertebrados, Museo Argentino de Ciencias Naturales “Bernardino Rivadavia”, Ciudad Autónoma de Buenos Aires, Argentina Agustín G. Martinelli Early Evolutionary Transitions of Mammals, Millennium Nucleus, Santiago, Chile Agustín G. Martinelli Instituto de Geología y Recursos Naturales, Centro de Investigación e Innovación Tecnológica, Universidad Nacional de La Rioja (INGeReN-CENIIT-UNLaR), La Rioja, Argentina Sebastián Rocher Centro de Investigaciones en Ciencias de la Tierra (CICTERRA), UNC CONICET, Córdoba, Argentina Jeremías R. A. Taborda División Paleontología Vertebrados, Museo de La Plata, Facultad de Ciencias Naturales y Museo, Universidad Nacional de La Plata, La Plata, Argentina Julia B. Desojo Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar coordinated fieldwork and research. and M.J. contributed to preparing specimens and curation. conducted the phylogenetic analyses. conceptualized and conducted the geological analyses. provided funding. conducted fieldwork. All authors described the specimens, collected data and contributed to writing the manuscript and the Supplementary Information. Correspondence to E. Martín Hechenleitner, Agustín G. Martinelli or Sebastián Rocher. The authors declare no competing interests. Nature thanks Rodrigo Müller, Fernando Novas and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b, Exaeretodon. c, Huayracursor. d,e, Hyperodapedon. Duplicate specimens of Exaeretodon and Hyperodapedon highlight the size variations observed in the figured skull materials. Scale bars: 50 mm (a,c,d) and 10 mm (e). a, Huayracursor. b, Bagualosaurus. c, Saturnalia. d, Chromogisaurus. e, Buriolestes (reversed). f, Panphagia (reversed). g, Macrocollum (reversed). h, Adeopapposaurus (reversed). i, Mbiresaurus (reversed). j, Pampadromaeus (reversed). k, Herrerasaurus. Scale bars: 20 mm. a, Huayracursor. c, Saturnalia (reversed). d, Pampadromaeus (reversed). e, Mbiresaurus (reversed). f, Bagualosaurus. g, Coloradisaurus. Scale bars: 10 mm. a, Huayracursor. b, Mbiresaurus. c, Eoraptor. e, Macrocollum. f, Saturnalia. h, Coelophysis (reversed). Scale bars: 10 mm. a, Huayracursor. b, Eoraptor. c, Panphagia. d, Mbiresaurus. e, Macrocollum. f, Saturnalia. g, Coelophysis (reversed). h, Herrerasaurus (reversed). Scale bars: 10 mm. Supplementary sections 1–10, including Supplementary Figs. 1–7, Supplementary Tables 1–10 and additional references. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Hechenleitner, E.M., Martinelli, A.G., Rocher, S. et al. A long-necked early dinosaur from a newly discovered Upper Triassic basin in the Andes. Download citation Received: 24 May 2025 Accepted: 16 September 2025 Published: 15 October 2025 DOI: https://doi.org/10.1038/s41586-025-09634-3 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1476-4687 (online) ISSN 0028-0836 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09619-2'>The astrocytic ensemble acts as a multiday trace to stabilize memory</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:05:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Recalled memories become transiently labile and require stabilization1,2,3. The mechanism for stabilizing memories of survival-critical experiences, which are often emotionally salient and repeated, remains unclear4. Here we identify an astrocytic ensemble that is transcriptionally primed by emotional experience and functionally triggered by repeated experience to stabilize labile memory. Using a novel brain-wide Fos tagging and imaging method, we found that astrocytic Fos ensembles were preferentially recruited in regions with neuronal engrams5 and were more widespread during fear recall than during conditioning. We established the induction mechanism of the astrocytic ensemble, which involves two steps: (1) an initial fear experience that induces day-long, slow astrocytic state changes with noradrenaline receptor upregulation; and (2) enhanced noradrenaline responses during recall, a repeated experience, enabling astrocytes to integrate coincident signals from local engrams and long-range noradrenergic projections, which induce secondary astrocytic state changes, including the upregulation of Fos and the neuromodulatory molecule IGFBP2. The astrocytic ensemble thus acts as a multiday trace in a subset of astrocytes after experience-dependent neural activity, which are eligible to capture future repeated experiences for stabilizing memories. Recall transiently destabilizes memories, which require re-stabilization to become long-lasting1,2,3. Despite its importance in human cognition and neuropsychiatric disorders6,7, the mechanisms that specifically stabilize memories of critical experiences—those that are essential for survival and frequently marked by emotional salience and repetition—remain incompletely understood. Memory traces are linked to specialized neuronal ensembles (neuronal engrams), which include neuronal populations that become Fos+ during both initial and repeated experiences8. However, neuronal Fos alone, a plasticity marker that indicates circuit remodelling9, is insufficient for memory stabilization, implying the need for additional cellular substrates. By tiling the entire mammalian central nervous system (CNS), astrocytes structurally and functionally interact with neurons and other glia. Their functions contribute to information processing and animal behaviour, and their dysfunctions are widely implicated in CNS disorders10,11. Astrocytes are known to be developmentally diverse12,13,14 and can adaptively change their molecular programmes in response to physiology11,15,16,17,18 and pathology19,20,21—hereafter referred to as state—in specific ways that suggest flexibly altered functions in a context-dependent manner. This adaptability is akin to experience-dependent neuronal responses (such as Fos upregulation)22 and the subsequent formation of inter-regional neuronal engram network8. However, the manifold nature of astrocyte responses through an extensive array of cell-surface receptors for neuroactive biomolecules19,23,24 have made it challenging to determine whether, when and how multiple inputs are integrated by a subset of astrocytes to generate specific states that form astrocyte ensembles that are responsible for particular cellular outputs15,18,24,25. This is because the field has lacked adequate tools to explore behaviourally relevant astrocyte ensembles (BAEs) in an unbiased manner at the single-cell level as well as in a brain-wide manner, which would enable ensemble-specific molecular dissection and functional perturbation. To agnostically and holistically identify BAEs, we developed a brain-wide astrocyte Fos tagging tool that enables permanent genetic access to BAEs that express Fos due to a specific experience. Although behaviourally relevant stimuli-induced astrocyte Fos mRNA22 and c-Fos protein18,26 have been observed in selected brain regions, its brain-wide profiling remains largely unexplored. We systematically validated a novel genetic and imaging strategy for whole-brain BAE tagging, demonstrating astrocyte selectivity, Fos sensitivity and temporal regulation (Extended Data Figs. First, using a CNS-wide-transducing adenovirus (AAV) vector (AAV-PHP.eB) with an astrocyte-specific GfaABC1D promoter27, we achieved widespread, astrocyte-selective (97.3–99.5%) expression (Extended Data Fig. Second, we tested whether using AAV-PHP.eB to express Cre-dependent mNeonGreen (mNG) can serve as a marker of transient astrocytic Fos induction in Fos-iCreERT2 (TRAP2) mice28. This intersectional approach aimed to drive astrocytic mNG expression only when the Fos promoter was active during fast-acting 4-hydroxytamoxifen (4-OHT) administration, enabling behaviour epoch-specific astrocyte Fos tagging across the CNS. As a positive control, we virally expressed the astrocytic Gαq-coupled designer receptor exclusively activated by designer drugs (DREADD) hM3Dq, a strong inducer of astrocyte c-Fos26,29,30, combined with 4-OHT and the hM3Dq agonist clozapine N-oxide (CNO) intraperitoneal injection. hM3Dq stimulation yielded approximately 282 mNG+ astrocytes per mm2, and significantly lower tagging in controls (Extended Data Fig. Third, to probe the temporal dynamics of Fos-mNG tagging, dark-adapted mice were exposed to light at various intervals relative to 4-OHT administration28. Tagging peaked within a 6-h window around 4-OHT administration, with an approximately 22-fold increase in astrocytes in visual cortex compared with those in somatosensory cortex, indicating regional specificity and dependence on local neuronal activity (Extended Data Fig. Collectively, these experiments validated our behaviour epoch-specific, brain-wide BAE tagging strategy. To broadly explore BAEs in a learning paradigm, we used a contextual fear conditioning and recall task5,28 (Fig. TRAP2::AAV-PHP.eB-GfaABC1D-DIO-mNG mice underwent three days of habituation to minimize contextual novelty and isolate associative components of fear learning and recall28,31. On day 4, one group received 3 foot shocks with immediate 4-OHT (FC), and another received 4-OHT immediately after recall 24 h post-FC (FR). Controls were non-shocked mice (NoFC) or fear-conditioned mice kept in the home cage for 24 h and given 4-OHT without recall (NoFR). Shocked groups exhibited robust freezing in the conditioned chamber, indicating learning and recall (Fig. One week later, whole-brain serial two-photon tomography (STPT)32 enabled the detection of individual mNG+ astrocytes (BAEs) (Supplementary Videos 1 and 2). After excluding ventricular niches for bona fide astrocyte counts (Methods), BAE counts across 677 regions covered approximately 80.7% of the brain (Supplementary Table 2). a, Fos+ astrocytes were tagged with Cre-dependent mNG AAV in four cohorts (NoFC, FC, NoFR and FR), followed by STPT, 3D reconstruction, segmentation and single-cell counting. c, 3D rendering of whole brain, the hippocampus (HPC), amygdala and primary visual cortex (VISp) with Fos-mNG astrocytes. A, anterior; D, dorsal; L, lateral; M medial; P. posterior; V, ventral. d,e, Heat map of fold changes in BAE count (versus NoFC (d) and FR versus FC (e); NoFC: n = 6; FC: n = 4; NoFR: n = 4; FR: n = 4 mice). Abbreviations for brain regions are listed in Extended Data Fig. Levene's test followed by one-way analysis of variance (ANOVA) or Kruskal–Wallis; post hoc Tukey or Dunn test with Bonferroni correction. Full data in Supplementary Table 2. f–h, FC- and FR-induced Fos counts in neurons versus astrocytes across engram areas for FC versus NoFC (f), FR versus NoFC (g) and FR versus NoFC with log scales, showing correlation Pearson's correlation coefficient (r) (h). h, Amygdalar subregions are indicated with yellow dots. j, Images of tagged Fos+ neurons and BAEs (representative of 10–22 sections from 3–5 mice per group). k, Quantification of Fos+ neurons (NoFC: n = 11 slices from 4 mice; FC: n = 10 slices from 3 mice; NoFR: n = 12 slices from 4 mice; FR: n = 13 slices from 4 mice) and BAE density (NoFC: n = 22 slices from 4 mice; FC: n = 18 slices from 3 mice; NoFR: n = 17 slices from 3 mice; FR: n = 15 slices from 5 mice). Sample sizes and replicates in Supplementary Table 1. Whereas the number of Fos+ neurons increased during both FC and FR, forming networks causally linked to fear memory traces—commonly referred to as engram neurons5,8,28, the density of Fos+ astrocytes during FC (FC-BAE; Fig. 1c) was seldom distinguishable from the NoFC control (NoFC-BAE). However, we found that the number of BAEs during recall (FR-BAE) was augmented across multiple regions (Fig. This was not simply reflecting delayed Fos upregulation after FC, because Fos induction was significantly higher in FR than in NoFR. Normalized comparisons across engram regions5 (Fig. 1d) showed a significantly greater number of FR-BAEs compared with FC-BAEs, particularly in the amygdala (Fig. 1e), the region long implicated in fear memory across species3,4,33,34,35. Consistently, in the lateral and basolateral amygdala (LA/B), astrocytic c-Fos protein was unchanged after FC or in NoFR versus NoFC, but FR induced a marked increase at 1.5 h with partial decay by 6 h (Extended Data Fig. Consistently, the number of of FR-BAEs in the LA/B (FR-BAELA/B) was robust whether 4-OHT was given immediately or 3 h post-FR (Extended Data Fig. These data indicate that astrocyte Fos promoter activity, and the associated surge in Fos mRNA and c-Fos protein expression, peaks within a biologically plausible window of 0–3 h following FR, but not following FC. A recent study18 reported an increase in hippocampal astrocytic Fos after FC without habituation, contrasting with our findings (Fig. We hypothesized that this reflects contextual novelty: without prior habituation, novelty strongly drives astrocytic Fos. Supporting this, our ‘FC + novelty' group (FC in novel context with no habituation) showed widespread astrocytic Fos, including in CA1 (Extended Data Fig. Thus, the CA1 astrocytic Fos is likely to signal novelty, whereas our habituated FC protocol reveals astrocytic ensembles that are more selectively engaged in learning and recall. To holistically compare the anatomical positioning of BAEs with previously reported brain-wide Fos+ neuron distribution during FC and FR5, we analysed subregions from previously defined regions in which Fos+ neuron counts of FC and FR were significantly higher than in non-shocked controls, referred to as neuronal ‘engram significant brain regions'5 (Fig. The number of Fos+ neurons increased during FC versus non-shocked controls, whereas there was no apparent change in astrocytic Fos counts (Fig. By contrast, FR elicited a strong correlation between neuronal and astrocytic Fos induction (r = 0.56; Fig. 1g,h), suggesting that local astrocyte–neuron interactions may underlie FR-BAE formation22. Notably, amygdala subregions that displayed selective astrocytic Fos increases during FR (Fig. 1e) also showed parallel neuronal Fos increases (Fig. To test whether astrocytic and neuronal ensembles are co-engaged within the same regions, we performed dual-colour Fos tagging in the amygdala of TRAP2 mice (Fig. Neuronal Fos increased after FC or FR, consistent with past reports5,8,22. By contrast, astrocytic tagging was minimal following FC and NoFR but was robustly induced after FR (Fig. These results confirm that astrocyte Fos induction in the amygdala is recall-specific and distinct from neuronal ensemble dynamics. To identify mechanisms underlying astrocyte Fos induction, we combined pharmacological screening, circuit genetics, in vivo imaging and transcriptomics (Figs. Growth factors that activate c-Fos in neurons36 did not trigger c-Fos signals in primary astrocytes (Extended Data Fig. 7), indicating that the induction mechanisms of c-Fos in neurons and astrocytes are distinct. Given the accumulating evidence that astrocyte G-protein-coupled receptor (GPCR) activation upregulates c-Fos expression26,29,30, we then stimulated primary astrocytes with a variety of neurotransmitters that bind to GPCRs expressed in astrocytes11,22,23, observing that noradrenaline (NA) increased c-Fos signals (Fig. The astrocyte c-Fos signals induced by NA at 0.1 µM and 1 µM were both abolished by propranolol (Fig. 2d), which antagonizes Gαs-coupled β-adrenergic receptors (β-receptors), but not by prazosin or atipamezole, which antagonize Gαq-coupled α1-adrenoreceptors (α1-receptors) or Gαi-coupled α2-adrenoreceptors, respectively (Fig. The astrocyte c-Fos induction was resistant to the Ca2+ chelator 1,2-bis(2-aminophenoxy)ethane-N,N,N′,N′-tetraacetic acid (BAPTA-AM)37. The membrane-permeable Ca2+ ionophore ionomycin did not increase astrocyte c-Fos signals (Fig. However, the inhibition of cAMP production by the adenylate cyclase inhibitor SQ22536 attenuated astrocyte c-Fos signals induced by the adenylate cyclase–cAMP activator forskolin, the β-receptor agonist isoproterenol or NA (Fig. Together, these data show that NA induces c-Fos signals in primary astrocytes mainly through Gαs-coupled β-receptors and subsequent increases in cAMP. b, NA increased c-Fos intensity in GFAP+ astrocytes (n = 27–50 cells per condition). One-way ANOVA, Dunnett's test versus vehicle (Veh). 5-HT, serotonin; a.u., arbitrary units; ACh, acetylcholine; DA, dopamine; GABA, γ-aminobutyric acid. c, Image showing NA (1 µM)-induced c-Fos increase (vehicle: n = 40; NA 1 μM: n = 41 cells). d, NA-induced c-Fos was blocked by inhibition of β-receptors or cAMP but not by Ca2+ mobilization (n = 28–55 cells per condition). One-way ANOVA, Dunnett's or Tukey's test. AC, adenylate cyclase; Atip, atipamezole; Frsk, forskolin; Isop, isoproterenol; Praz, prazosin; Prop, propranolol. e, Schematic of AAV-mediated Grm3 (encoding mGluR3) expression. f,g, Convergent NA and glutamate stimulation enhanced c-Fos in mGluR3+ astrocytes, but not in mGluR3− astrocytes (n = 98–131 cells per condition). i, Ionomycin (Iono)-induced increase in Ca2+ increased c-Fos when combined with NA, isoproterenol (β-receptor agonist) or forskolin (n = 41–59 cells per condition). j, Schematic of AAV injection into amygdala (LA/B) and LC in TRAP2 mice. m, Quantification of Fos-tagged astrocytes (mNG+) localized near hM3Dq+ LA/B neurons and NA fibres in l (representative of five biological replicates from the LC+/LA+ group). n–q, FR-BAE tagging with β-receptor blockade (propranolol, 10 mg kg−1) (n), LC silencing (o), amygdala (Amy) silencing (p) or RAM-based engram silencing (q) reduced BAE density. Scale bars (n,o–q, top and middle), 1 mm. o–q, Bottom image, magnified view showing local hM4Di-mCherry in neurons. Sample sizes and replicates in Supplementary Table 1. a, AAV injection of GRABNE2h with optical fibre above LA/B to monitor bulk NA response (representative image from 6 mice; scale bar, 200 µm). b, ΔF/F traces showing NA signals in home cage (HC) and CtxA on fear conditioning (FC) day. NA signal increased with CtxA exposure and foot shocks, and showed larger, longer responses during recall (FR). Right, summary plots (n = 5–6 mice). c, Top, AAV injection of astrocyte-selective cAMPinG1 and RCaMP3 with optical fibre above LA/B to monitor astrocyte cAMP and Ca2+. Bottom, co-expression of cAMPinG1 and RCaMP3 in LA/B astrocytes (representative image from 8 mice; scale bars: 200 µm (main image), 20 µm (magnified views, bottom)). d, Astrocyte cAMP signals increased with context exposure and shocks, and during FR they increased gradually from HC baseline and were further enhanced in CtxA. Propranolol (10 mg kg−1, β-receptor antagonist) abolished these effects. Right, summary plot (n = 8 mice). e, Astrocyte Ca2+ signals also increased with context exposure and shocks. Right, summary plots (n = 7 mice). Sample sizes and number of replicates in Supplementary Table 1. a, t-Distributed stochastic neighbour embedding (tSNE) clustering of scRNA-seq data from amygdalar astrocytes across NoFC, FC, NoFR and FR conditions (n = 4 mice per group). b, Dot plots showing Fos expression. Dot colour indicates z-score (grey, depletion; green, enrichment). c, Left, dot plots showing adrenoreceptor gene expression. Undetected receptors (Adra1d, Adra2c and Adrb3) are not shown. Right, Adra1a and Adrb1 upregulation in NoFR versus FC. e, RNAscope images and violin plots of Adra1a and Adrb1 in Slc1a3+ astrocytes (18–28 cells from 5 mice per group). One-way ANOVA, Dunnett's test versus NoFC. f, Co-expression index for Adra1a and Adrb1 (Methods) peaks at 1 day post-FC. One-way ANOVA, Dunnett's test versus NoFC. h, Representative images and quantification of tagged FR-BAE, peaking at 1 day post-FC. One-way ANOVA, Dunnett's test versus NoFC. i, RNAscope of Adrb1 and Fos in Slc1a3+ astrocytes (FR). Arrows indicate a Fos and Adrb1-high cell (representative images from 18 sections, 6 mice). j, Pearson's correlation coefficient (r) for Fos and Adrb1 expression (z-scores). l, Schematic of Adra1a or Adrb1 conditional knockout (cKO). m, RNAscope of Adra1a and Adrb1 in Slc1a3+ astrocytes. n, Violin plots showing reduced targeted receptor after conditional knockout (77–117 cells, 3 mice per group). o,p, Images (o) and quantification (p) of FR-BAEs, showing decreased density following Adra1a or Adrb1 conditional knockout. n = 5–9 mice, One-way ANOVA, Dunnett's test versus FR controls. Sample sizes and replicates in Supplementary Table 1. Astrocytes in vivo respond to both local circuits and long-range neuromodulators15. 3d) suggest that local activity also contributes to Fos induction in vivo. In mGluR3-expressing astrocytes, glutamate alone was ineffective, but in combination with NA it strongly enhanced c-Fos intensity (Fig. No such synergy was seen in mGluR3-negative astrocytes. Because Gαi-coupled GPCRs such as mGluR3 can enhance Gβγ–Ca2+ signalling when Gαq–phospholipase Cβ is co-activated29,40, NA may recruit α1-receptors (Gαq) and mGluR3 (Gαi) to cooperate with β-receptor–cAMP pathways (Fig. Supporting this, ionomycin-induced Ca2+ signals enhanced c-Fos when combined with NA, isoproterenol or forskolin (Fig. Thus, astrocytic c-Fos is driven primarily by β-receptor–cAMP and is further potentiated by convergent glutamate and NA signalling. To test this convergence in vivo, we expressed neuron-specific hM3Dq in LA/B and locus coeruleus (LC), which mainly consists of noradrenergic neurons to stimulate them with CNO (Fig. The number of Fos-mNG+ astrocytes remained low when only LA/B or LC neurons were activated, but increased markedly when both were stimulated (Fig. This provides in vivo evidence that coincident local and noradrenergic inputs induce astrocytic Fos. We next tested the necessity of local and far-reaching NA neural signals during FR-BAE tagging. TRAP2::AAV-PHP.eB-GfaABC1D-DIO-mNG mice received one of four perturbations at the time of tagging: β-receptor antagonism with propranolol (Fig. 2n) or hM4Di-mediated silencing of LC neurons (Fig. 2p) or amygdalar engram neurons tagged with the robust activity marking (RAM) system41,42 whose hM4Di-based silencing (Extended Data Fig. 8) is known to reduce fear memory recall41 (Fig. In all conditions, FR-BAE density decreased significantly (Fig. 2r), demonstrating dependence on both LC-NA and local engram activity. To further test whether FR-BAEs reside in anatomical contexts where they could receive convergent inputs from both noradrenergic projections and local engram neurons, we performed dual-colour Fos tagging with neuron-specific DIO-mCherry and astrocyte-specific DIO-mNG AAVs in TRAP2 mice, combined with noradrenaline transporter (NET) immunostaining to visualize noradrenergic projections (Extended Data Fig. Territories of FR-BAEs showed greater overlap with noradrenergic projections and FR engram neurons than those of non-FR-BAEs, with no change in astrocyte territory size. These findings indicate that FR-BAEs are positioned to integrate convergent inputs from noradrenergic projections and local engram neurons, supporting the model that their coincidence drives FR-BAE induction in the amygdala. 2) support a convergence model in which NA and engram activity induce astrocyte Fos. However, since both activities occur during FC and FR8,43, the stronger astrocyte Fos induction during FR than FC (Fig. 1) raises the question of how astrocytes generate this selective response. To address this, we examined NA release and astrocyte cAMP–Ca2+ signalling dynamics across fear learning and recall using in vivo fibre photometry (Fig. We deployed AAV2/9-hSyn1-GRABNE2h44 to monitor bulk NA signals in the LA/B of freely behaving mice (Fig. During habituation 24 h prior to FC, NA signals were observed when the mice were placed into context A (CtxA; foot shock chamber) (peak ΔF/F = 0.142 ± 0.017), probably reflecting mild stressors and/or locomotion44 (NoFC; Fig. On top of the contextual NA signals, single foot shocks evoked enhanced and relatively fast (peak ΔF/F = 0.206 ± 0.04; half-decay = 4.3 min) NA responses in LA/B (FC; Fig. In addition, further enhanced (peak ΔF/F = 0.211 ± 0.022) and prolonged (half-decay = 13.5 min) NA signals were observed during the contextual FR (FR; Fig. The prolonged NA signals were absent in context B (CtxB) (Extended Data Fig. 10c,d), a neutral, non-conditioned context where mice show no freezing (Extended Data Fig. 10b), indicating that recall of contextual fear drives sustained NA signals. We next used dual-colour fibre photometry (AAV2/5-GfaABC1D-cAMPinG1 and AAV2/5-GfaABC1D-RCaMP345) to measure astrocyte cAMP and Ca2+ signals (Fig. Consistent with the NA signals (Fig. 3a,b), cAMPinG1 signals increased during habituation and foot shocks, which fully decayed within about 2 min (NoFC and FC; Fig. On the FR day, however, astrocytes displayed a progressive increase in cAMP (FR; Fig. 3d), which was absent in CtxB (Extended Data Fig. β-receptor antagonism (propranolol) in vivo abolished the astrocytic cAMP signalling during FR (Fig. 2n,r) and freezing (Extended Data Fig. Foot shock elicited astrocyte Ca2+ signals as reported previously46, and Ca2+ event frequency increased during FR (Fig. These data collectively indicate that fear experiences induce the enhancement of NA signals within the amygdala, which strongly trigger astrocyte cAMP signals through β-receptors during FR. The increased astrocyte cAMP–Ca2+ signalling during FR raised the question of how astrocytes mount stronger responses during recall than during conditioning, despite NA release and engram neural activity occurring in both fear experiences. To address this, we examined experience-dependent molecular changes in astrocytes under NoFC, FC, NoFR and FR conditions. We dissected amygdala tissue 90 min after behavioural testing and isolated astrocytes by fluorescence-activated cell sorting (FACS) using the ATPase Na+/K+ transporting subunit β2 (ATP1B2)47 antibody ACSA2 (Extended Data Fig. 11a) and performed single-cell RNA sequencing (scRNA-seq). After quality control, we obtained profiles of 1,903 astrocytes (Fig. 11b,c) and identified 4 transcriptionally distinct astrocyte clusters (AST1–4; Fig. We did not observe any noticeable shifts in the subcluster composition across conditions, suggesting that FC and FR do not alter cluster identity. Of note, FR astrocytes showed the highest Fos levels (Fig. 4b), in line with BAE tagging (Fig. Whereas scRNA-seq provides a snapshot at a single time point (approximately twofold increase in FR versus FC; Fig. 4b), the Fos-TRAP system integrates promoter activity over hours, which is likely to account for the greater fold differences observed with tagging (more than 30-fold in FR versus FC; Fig. Subsequent analyses revealed that Adra1a (α1-receptor) and Adrb1 (β1-adrenoreceptor) as the two dominant adrenoreceptor genes expressed in amygdalar astrocytes. Notably, both receptors were upregulated in the FC group (1.5 h post-FC) and further elevated in the NoFR group (24 h post-FC without recall), suggesting continued priming of astrocytes over time (Fig. This data suggests a model in which FC induces a molecular state change in astrocytes, with a latency, increasing adrenoreceptor expression to potentiate NA signalling and enable future repeated fear-evoked Fos induction during FR. The enhancement in astrocytic adrenoreceptor expression is likely to underlie the NA–β-receptor-driven cAMP signals during FR (Fig. 3c,d) and Fos induction in response to recall of strong fear memory a day after conditioning (Fig. To test this model, we explored temporal changes in adrenoreceptor expression, their link to FR-BAE induction, and causal roles in Fos induction. We examined the time course of Adra1a and Adrb1 expression following FC, using single-cell RNAscope on amygdalar sections collected at 1.5 h, 1 day, 2 days, 3 days and 14 days post-conditioning (Fig. Adrb1 expression was significantly increased from 1.5 h to 3 days post-conditioning (approximately twofold versus NoFC; Fig. 4e), but returned to baseline by 14 days. By contrast, Adra1a expression was increased at 1 day post-conditioning (Fig. We calculated a co-expression index for each cell (relative Adrb1 expression × relative Adra1a expression; Methods) and found a peak at 1 day post-FC (Fig. To test whether this molecular profile predicts the degree of astrocyte Fos induction during recall, we performed FR-BAE tagging at multiple time points post-FC (1 day, 2 days, 3 days and 14 days; Fig. Consistent with the adrenoreceptor co-expression time course (Fig. 4f), FR-BAE density peaked at 1 day and declined thereafter (Fig. 4h), suggesting a defined window for FR-BAE induction. Moreover, RNAscope further revealed strong Fos–Adrb1 correlation in FR astrocytes (Fig. To determine whether these adrenoreceptors are causally required for FR-BAE induction, we performed astrocyte-specific knockdown of Adra1a or Adrb1 in the amygdala (Fig. Knockdown of either receptor (Fig. 4m,n) significantly reduced FR-BAE density (Fig. 4o,p), indicating that both Gαq-coupled α1-receptors and Gαs-coupled β1-receptors are necessary for FR-BAE induction in vivo. Our earlier finding that the α1-receptor antagonist prazosin did not block NA-induced c-Fos induction in cultured astrocytes (Fig. 2d) probably reflects the downregulation of Adra1a in vitro17, highlighting the importance of in vivo validation. Collectively, our data support a model in which initial fear experience primes astrocytes to upregulate adrenoreceptor expression over time, creating a multiday molecular trace that allows future NA release upon repeated experiences (that is, recall) to selectively recruit the astrocyte ensemble. Transcriptomics enabled us to probe molecular changes of FR-BAEs linked to functional outputs. We mined differentially expressed genes between Adrb1-positive and Adrb1-negative astrocytes (Extended Data Fig. 12a) and identified Igfbp2 as highly upregulated in Adrb1-positive astrocytes (Extended Data Fig. Our single-cell RNAscope revealed highly correlated expression of Adrb1 and Igfbp2 (Extended Data Fig. 12c–e), suggesting that astrocytic Adrb1 upregulation could promote Igfbp2 induction. Igfbp2 encodes the secreted protein IGFBP2, which was recently reported to be enriched in ‘peri-neuronal engram' Fos+ astrocytes of the amygdala after fear recall22. Given that IGFBP2 and its binding partners, insulin-like growth factors (IGFs), are crucial for synaptic plasticity52,53 and that chronic deletion of IGFBP2 disrupts synaptic transmission, circuit maturation and memory22,52,54, we hypothesized that IGFBP2 expression in FR-BAELA/B contributes to circuit de-stabilization and re-stabilization upon recall2,3. We tested this hypothesis with a spatiotemporally specific in vivo pharmacological perturbation of IGFBP2 (Extended Data Fig. We bilaterally infused an IGFBP2-neutralizing antibody52 (IGFBP2-Ab; 1 mg ml−1) immediately after FR in CtxA—that is, within the reconsolidation window during which memories become labile to be weakened or strengthened2. Mice received repeated daily infusions and FR sessions for five days (Extended Data Fig. IGFBP2-Ab treatment reduced freezing compared with controls (Extended Data Fig. 12g,h), indicating impaired memory stabilization; controls infused with IgG showed no change. Combined, our transcriptomics, RNAscope and pharmacological perturbation data indicate that a subset of astrocytes increases adrenoreceptor expression upon fear experiences, enhancing NA responsiveness and recall-driven induction of Fos and Igfbp2 to stabilize fear memory. To directly examine the causal role of FR-BAE signalling in memory stabilization, we performed a loss-of-function experiment selectively targeting FR-BAEs in LA/B. We used iβARK, a genetically encoded 122-amino acid peptide that binds Gαq–GTP to suppress GPCR signalling26 (Extended Data Fig. In ex vivo Ca2+ imaging in amygdalar astrocytes, iβARK reduced α1-receptor-evoked Ca2+ responses relative to its mutant control, although some residual oscillatory activity persisted (Extended Data Fig. 13c,f,g), consistent with our prior findings in the striatum26. To improve efficacy, we developed iβARK2 by fusing the original peptide to a membrane-anchoring Lck domain, enabling more effective proximity-based inhibition (Extended Data Fig. iβARK2 showed robust membrane localization across astrocytic compartments, including fine processes (Extended Data Fig. Astrocytes expressing iβARK2 displayed significantly attenuated Ca2+ responses relative to both iβARK and the control iβARK2mut (Extended Data Fig. We next generated Cre-dependent AAV versions of iβARK2 and iβARK2mut for use in Fos-iCreERT2 TRAP mice to selectively target FR-BAEs (Fig. Remarkably, silencing FR-BAEs with iβARK2 impaired memory stabilization upon repeated recall (Fig. 5c), consistent with the behavioural effects observed upon neutralization of astrocyte-derived IGFBP2 (Extended Data Fig. Moreover, astrocytic IGFBP2 levels were significantly reduced in iβARK2-expressing mice compared with iβARK2mut controls (Fig. 5d), suggesting that suppression of GPCR signalling in FR-BAEs disrupts the recall-dependent transcriptional induction of IGFBP2 and compromises memory stabilization. a, Schematic of behavioural and RNAscope analysis. c, Reduced freezing at second recall in iβARK2 versus sustained freezing in iβARK2mut control (iβARK2mut: n = 13; iβARK2: n = 11 mice). Wilcoxon matched-pairs signed rank test. d, RNAscope images of Igfbp2 with mCherry–iβARK2 or mCherry–iβARK2mut and violin plots showing lower Igfbp2 expression in the presence of iβARK2 (iβARK2mut: n = 23; iβARK2: n = 24 cells; 4 mice per group). e, FR-BAE tagging with astrocytic Adrb1 overexpression. f, RNAscope images of Adrb1 and Igfbp2 in Slc1a3+ astrocytes with or without Adrb1 overexpression. Graphs showing increased expression with Adrb1-OE (ctrl: 501; Adrb1-OE: 1,810 cells; 18 sections from 6 mice per group). g,h, Images (g) and quantification (h) showing higher FR-BAE density with Adrb1 overexpression. i, Memory tests under weak (one-shock) versus strong (three-shock) conditioning. j, Adrb1 overexpression has no effect on freezing behaviour under strong conditioning (ctrl: n = 5; Adrb1-OE: n = 7 mice). Wilcoxon matched-pairs signed rank test. k, Under weak conditioning, Adrb1 overexpression enhanced freezing at second recall (n = 13 mice per group). Wilcoxon matched-pairs signed rank test. l, After strong conditioning, Adrb1 overexpression caused greater freezing in CtxB versus control (ctrl: n = 13; Adrb1-OE: n = 14 mice). m, FR engram labelling with mCherry and c-Fos immunohistochemistry 90 min after freezing test with or without Adrb1 overexpression. n,o, FR engram neurons exhibit similar density but higher reactivation with Adrb1 overexpression (ctrl: n = 15; Adrb1-OE: n = 12 slices; 3 mice per group). Sample sizes and replicates in Supplementary Table 1. To further explore the causal impact of BAE signalling on neuronal engram and memory formation, we performed a gain-of-function experiment targeting astrocytic β1-receptors in LA/B. Astrocytes transduced with AAV2/5-GfaABC1D-Adrb1 for Adrb1 overexpression (Adrb1-OE) showed significantly increased Igfbp2 expression compared with controls (Fig. Adrb1 overexpression also increased the density of FR-BAEs (Fos-mNG+ astrocytes) by around threefold relative to controls (Fig. We next functionally assessed the behavioural effect of the augmented astrocyte ensembles (Fig. We detected no differences in ambulation in an open arena or time spent in the centre zone during habituation between control and Adrb1-OE mice (Extended Data Fig. We then tested whether Adrb1 overexpression affected memory stabilization. With three-shock conditioning, both groups showed equivalent freezing during initial recall and second recall after reconsolidation (Fig. 5j), potentially owing to a ceiling effect. Adrb1-OE mice did not show altered freezing during FR one day after FC; however, Adrb1-OE mice exhibited significantly increased freezing during the second recall (Fig. 5k), indicating enhanced memory stabilization following the reconsolidation after initial recall. In another cohort with three-shock conditioning, initial recall showed no significant difference in freezing between the groups (Extended Data Fig. However, we observed significantly enhanced freezing in an altered context (CtxB) in mice with Adrb1 overexpression compared with controls (Fig. These data suggest that overstabilization of the initial fear memory leads to generalization or a loss of memory precision41,55,56,57. Finally, we explored whether fear generalization is accompanied by activity of engram neurons. Mice were euthanized 1.5 h after the CtxB freezing test and the reactivation of FR engram neurons was assessed by c-Fos immunoreactivity (Fig. The overall number of mCherry+ neurons (FR engram neurons) was not significantly different between groups, but the c-Fos levels in FR engram neurons in the Adrb1-OE mice, which showed fear generalization, were significantly higher than those in control mice (Fig. These findings suggest that the larger astrocyte ensembles contribute to enhance subsequent reactivation of engram neurons, consistent with the view that memory retrievability is not determined during memory formation, but may be regulated after memory retrieval8, and demonstrating that astrocytes are involved in that crucial process. These data demonstrate that astrocytes can link discrete fear experiences across at least a day-long timescale. As postulated more than a century ago58 and recently implicated in brain computation11,25,59,60, astrocytes may integrate information on slower timescales than neurons. Whereas neurons and astrocytes have been traditionally compared over milliseconds versus seconds-to-minutes, our findings reveal a functionally significant mismatch on the scale of days. Unlike neurons that express the immediate early genes during both FC and FR5,8,28, astrocytes respond to fear conditioning (FC) with rapid Ca2+ and cAMP increases61, but undergo slower transcriptional state changes62. Our single-cell transcriptomics and RNAscope revealed that multiple days post-FC, without recall, astrocytes upregulated Adra1a and Adrb1, marking a learning-dependent state that facilitates heightened responses during repeated experiences (FR). This in turn drives expression of Fos and Igfbp2, supporting memory stabilization. The astrocyte ensembles identified in this study may partially meet the criteria for engram cells8 that undergo enduring biochemical changes upon experiences. However, our findings suggest that the astrocytic ensemble may not serve as a direct memory trace encoding experience-specific information like neurons because its perturbation did not alter initial fear recall. Instead, it may function as an eligibility trace for memory stabilization over days, creating a permissive microenvironment within circuits that supports the stabilization of memories related to repeated experiences. From a computational and bioenergetic perspective, astrocytes may underlie retrospective linking63,64, which benefits from spaced intervals over days between learning sessions65 by providing a slow modulatory trace that complements fast neuronal activity64, thereby offloading the energetically costly task of maintaining firing patterns in neurons across days and enabling more stable circuit transformations60. Mounting evidence from animal3,4,57 and human imaging6 studies has highlighted the amygdala and NA in modulating long-term memory. Early pharmacological studies targeting protein synthesis66, β-receptors67, PKA35 or cAMP–CREB34 in the amygdala disrupted memory re-stabilization, which our findings indicate could partially be mediated by astrocytes. We show that astrocytes undergo learning-dependent increases in adrenoreceptor expressions, and manipulating these receptors alters memory stability, precision and neuronal engram representations. Notably, both NA input and reactivated neuronal engram activity are required to recruit astrocytic ensembles that stabilize labile memories. These findings imply that astrocyte ensembles may not simply be passive bulk substrates of NA signals, but instead exert behaviourally relevant, circuit-specific regulations through astrocyte-defined microcircuitry15,68. Disrupting the precise NA–astrocyte signalling through β1R overexpression led to fear generalization, suggesting physiological astrocytic roles in selective memory stability that constrain maladaptive updating. In this regard, targeting these astrocyte–neuron interaction mechanisms could be beneficial in treating cognitive and neuropsychiatric dysfunctions, including post-traumatic stress disorder, in which hyperactivity of the NA system is a prominent feature4,69. Deciphering how diverse astrocyte responses link to multiple circuit outputs has long been a challenge15. Considerable effort has been devoted to probe24 or perturb11 astrocyte Ca2+ signals since they were first reported70, yet causal relationships are usually inferred from separate subjects, leaving input–output links unclear. Astrocyte perturbation in a bulk manner is also incongruent with the heterogeneity of astrocyte identity49,71, state62,72 and response59,73. Our approaches bridge this gap: by exploiting context-specific astrocyte state changes18,20,22, the brain-wide BAE tagging method enables broad exploration of ensembles causal for neural processing and behaviour. This framework facilitates research into astrocytic ensembles across diverse behaviours and disease conditions. Expanding such efforts will accelerate discoveries in computation and pathology that require understanding of multicellular interactions as a core attribute of brain biology. Young adult (8- to 12-week-old) wild-type C57BL/6NCrSlc mice obtained from the Japan SLC, Fos2A-iCreERT2 (TRAP228, JAX 030323) mice obtained from the Jackson Laboratory, Adra1a flox mice49,74 obtained from RIKEN (RBRC11837) and Adrb1 flox mice50 (accession number CDB1068K, generated by RIKEN Center for Biosystems Dynamics Research) were used for in vivo experiments. TRAP2 mice were bred and maintained in our animal facility, backcrossed with C57BL/6NCrSlc at least eight times before use for experiments. Wild-type C57BL/6NCrSlc female mice were obtained from the Japan SLC to obtain postnatal day 0–1 (P0–P1) pups. Experiments were done using both male and female mice. Number of mice used in each experiment is listed in the main text and/or figure legends accordingly. Mice were housed in groups of 2–5 in a temperature- and humidity-controlled room (23 ± 3 °C, 45 ± 5% humidity) under a 12-h light:dark cycle (lights on from 07:00 to 19:00) and given ad libitum access to water and laboratory mouse diet at all times. After fibre implantation, the mice were single housed under a 12-h light:dark cycle (lights on from 07:00 to 19:00) and given ad libitum access to water and laboratory mouse diet at all times. After fibre implantation, the mice were singly housed. All experiments were approved by the RIKEN Animal Care and Use Committee. AAVpro 293T cells (Clontech, 632273) were maintained in Dulbecco's Modified Eagle Medium (DMEM; Gibco, 11995073) with fetal bovine serum (FBS; Cosmo Bio, CCP-FBS-BR-500) and 0.1% antibiotic-antimycotic (Gibco, 15240062). Cells were grown in a humidified cell culture incubation with 95% air, 5% CO2 at 37 °C. All plasmid constructs were generated using standard molecular biology techniques and the In-Fusion HD Cloning Kit (Clontech). The pZac2.1 GfaABC1D-Rpl22-HA plasmids (Addgene plasmid #111811) were digested with XhoI and NotI, which removed the Rpl22-HA sequence. Multiple cloning sequences (MCS) comprised of restriction enzyme sites that were absent from the backbone pZac2.1 plasmid were generated via gene synthesis (Eurofins Genomics) and incorporated to produce pZac2.1-GfaABC1D-MCS plasmids. The PCR-amplified DIO-GFP sequence from Addgene plasmid #28304 was cloned into the mammalian expression vector pCAG-tdTomato (Addgene #83029) between EcoRI sites to generate pCAG-DIO-EGFP-tdTomato. The PCR-amplified WPRE sequence from pFBAAV-CAG-XCaMP-R-WPRE (a gift from H. Bito12) was inserted into the NotI site of the resulting plasmid to generate pZac2.1-GfaABC1D-DIO-EGFP-tdTomato-WPRE plasmids. The plasmids were digested with NotI and AgeI, which removed the tdTomato sequence, and a PCR-amplified mScarlet-I sequence (Addgene #98816) was incorporated, and the resulting plasmids were digested with XhoI and KpnI, which removed the EGFP sequence, and the mNG sequence from Addgene #99135 was ligated using DNA ligation kit (Takara) to generate pZac2.1-GfaABC1D-DIO-mNG-mScarlet-I-WPRE plasmids. The mScarlet-I sequence was removed with restriction enzymatic digestion with PspOMI and NotI to generate pZac2.1-GfaABC1D-DIO-mNG-WPRE plasmids. The PCR-amplified hM4Di-mCherry sequence from Addgene #92286 was cloned into the PmeI site of pAAV-RAM-d2TTA::TRE-MCS-WPRE (Addgene #63931) to generate pAAV-RAM-d2TTA::TRE-hM4Di-mCherry-WPRE plasmids. The PCR-amplified XCaMP-Gf-WPRE from the pFBAAV-CAG-XCaMP-Gf-WPRE plasmid (a gift from H. Bito75) was cloned into the NcoI and HindIII site of pAAV-hSyn1-cyto-mRuby3-iATPSnFR1.0 (Addgene #102557) to generate pAAV-hSyn1-XCaMP-Gf-WPRE plasmids. All constructs were sequenced before use. All our constructs have been deposited to Addgene in the Nagai Lab repository for unrestricted distribution (http://www.addgene.org/Jun_Nagai, Supplementary Table 4). Recombinant AAVs were generated by triple transfection of AAVpro 293T cells (Clontech, 632273) using polyethylenimine (PEI; Polysciences inc., 24765-100). Cells were cultured up to 50–70% confluency in DMEM (Gibco, 11995073) with FBS (Cosmo Bio, CCP-FBS-BR-500) and 0.1% antibiotic-antimycotic (Gibco, 15240062). The plasmids and PEI were mixed with Opti-MEM (31985-088) and kept for 5 min at room temperature. The mixture was combined with a DNA mix which contained pAAV Capsids, pHelper (Agilent Technologies, 240071-54) and pAAV transgenes. All these plasmids had a concentration of 4.2 μg per dish. The DNA mix was then applied with PEI/Opti-MEM, and incubated for 20 min at room temperature. Then, this solution was added to the cells and incubated for 24 h. After incubation, the medium was changed to DMEM with 2% FBS and 0.1% antibiotic-antimycotic. Viral particles were collected from the medium at 120 h post-transfection. Cell pellets were centrifuged (3,500 rpm, 30 min, 4 °C). The supernatant was filtered by Setricup (Merc, S2HVU02RE) and pellets were resuspended in FBS-free DMEM and freeze-thawed three times. In the case of PHP.eB capsid virus production, the supernatant was resuspended in DMEM and fractured by harshly pipetting. Thawed pellets were centrifuged (3,500 rpm, 10 min, 4 °C) again and the supernatant was filtered by Setricup (Merc, S2HVU02RE). Media containing viral particles were centrifuged (22,000 rpm, 2 h, 4 °C) by Optima XE-100 (Beckman Coulter, A94516). The pellets were collected with 10–50 μl PBS and clarified by centrifugation (4,000 rpm, 10 min, 4 °C). The supernatant-containing viral particles were stored at 4 °C. AAV titre was analysed by quantitative PCR (QuantStudio 12 K; Applied Biosystems) using primers designed to selectively bind AAV2 inverted terminal repeats (forward: 5′-AACATGCTACGCAGAGAGGGAGTGG-3′, reverse: 5′-CATGAGACAAGGAACCCCTAGTGATGGAG-3′). All surgical procedures were conducted under general anaesthesia using continuous isoflurane (induction at 5%, maintenance at 1.5% to 2% vol/vol). Depth of anaesthesia was monitored continuously and adjusted when necessary. Intravenous administration of AAV-PHP.eB (50 µl to deliver 2.5 × 1012 genome copies (GCs) per mouse) was performed by injection into the retro-orbital sinus of TRAP2 mice. Two weeks after AAV-PHP.eB GfaABC1D-DIO-mNG-WPRE or AAV-PHP.eBGfaABC1D-DIO-mNG-mScaret-I-WPRE injections, mice were administered 4-OHT for inducing Cre recombination under the Fos promoter (Fos tagging) and then returned to their home cage until they were euthanized 7 days after tagging. In some cases, Fos tagging was performed using the FLP–FRT system (fDIO), and mice were euthanized one day after tagging. For stereotaxic microinjection of AAVs, mice were fitted into a stereotaxic frame (David Kopf Instruments) with their heads secured by blunt ear bars and their noses placed into a veterinary grade anaesthesia and ventilation system (VetEquip 911103). The surgical incision site was then cleaned with 70% ethanol (vol/vol). Craniotomies (1–2 mm in diameter) were made powered by a high-speed drill at specific coordinates based on the bregma position. Bilateral viral injections were conducted using a stereotaxic apparatus (David Kopf Instruments) to guide the placement of beveled glass pipettes (World Precision Instruments, 1B100-4) into the regions of interest: the basolateral amygdala (anterior–posterior (AP): −1.48 mm, medial–lateral (ML): 3.3 mm, dorsal–ventral (DV): 5.00 mm to bregma) and LC (AP: −5.4 mm, ML: 0.85 mm, DV: 3.7 mm to bregma). Total of 0.3 μl of AAV was delivered at 0.1 μl min−1 using a syringe pump (SmartTouch Pump, World Precision Instruments). Mice were then sutured, returned to their home cage, and allowed to rest for at least two to three weeks before any assessments were conducted. AAV titres (GCs per ml) were adjusted with sterile saline to deliver the indicated genome copies into the regions of interest: AAV2/5-GfaABC1D-DIO-mNG-WPRE (3.0 × 1012), AAV2/5-GfaABC1D-hM3Dq-mCherry (3.0 × 1012), AAV2/DJ8-RAM-d2TTA::TRE-hM4Di-WPRE (3.0 × 1013), AAV2/5-GfaABC1D-Flag-mAdrb1-WPRE (3.0 × 1012), AAV2/9-rTH-Cre (3.0 × 1012), AAV2/8-hSyn1-DIO-hM3Dq (1.0 × 1013), AAV2/8-hSyn1-DIO-hM4Di (1.0 × 1013), AAV2/8-hSyn1-DIO-mCherry (3.0 × 1012), AAV2/DJ8-Ef1a-hM3Dq (4.8 × 1011), AAV2/DJ8-Ef1a-hM4Di (4.0 × 1012), AAV2/5-hSyn1-mCherry-CAXX (3.0 × 1012) AAV2/9-hSyn1-GRABNE2h-WPRE (2.7 × 1012), AAV2/1 GfaABC1D-cAMPinG1-NE (1.3 × 1013), and AAV2/1 GfaABC1D-RCaMP3 (1.5 × 1013), GfaABC1D-GCaMP6f (1.1 × 1013), AAV2/5-GfaABC1D-iβARK-mCherry (1.3 × 1012), AAV2/5-GfaABC1D-iβARK(D110A)-mCherry (1.3 × 1012), AAV2/5-GfaABC1D-Lck-mCherry-iβARK (1.3 × 1012), AAV2/5-GfaABC1D-Lck-mCherry-iβARK(D110A) (1.1 × 1013), AAV2/5-GfaABC1D-DIO-Lck-mCherry-iβARK-4x6T (3.0 × 1013), AAV2/5-GfaABC1D-Lck-mCherry-iβARK(D110A) (3.0 × 1013), AAV2/5-Fos-FlpoERT2 (3.0 × 1012), AAV2/5-GfaABC1D-fDIO-mNG (1.0 × 1012), AAV2/5-GfaABC1D-Cre-4x6T (1.2 × 1013). 4-OHT (Sigma, H6278) was dissolved at 25 mg ml−1 in 100% ethanol with agitation at 37 °C for 15 min and was then aliquoted and stored at −20 °C for up to several weeks until use. Before use, 4-OHT was redissolved in ethanol by shaking at 37 °C for 15 min and corn oil (Sigma, 259853 and S5007) was added to give a final concentration of 3 mg ml−1 4-OHT. The ethanol was evaporated by incubation at 50 °C. The final 3 mg ml−1 4-OHT solutions were always used on the day they were prepared. All 4-OHT injections were delivered intraperitoneally at 50 mg kg−1 2–4 weeks after AAV injection into TRAP2 mice for inducing Cre recombination under the Fos promoter. CNO (Tocris, 4936) was administered intraperitoneally 0.5 h before the initiation of behavioural assessments. CNO was administered intraperitoneally 20 min before 4-OHT intraperitoneal injection. The final solutions were always used on the day they were prepared. Mice were placed on doxycycline (40 mg kg−1, Bio-Serv, F4159) chow 24 h prior to and for at least 1 week after viral injection, switched to doxycycline-free chow 48 h before recall, and euthanized 24 h later. IGFBP2 antibody was infused bilaterally through implanted guide cannulas targeting basolateral amygdala. Infusions were performed using an internal cannula connected to a 10-µl Hamiltion syringe. One microgram of IGFBP2 antibody (1 mg ml−1, 1 µl per site) was infused into mice immediately after a fear recall paradigm at 400 nl min−1 using a syringe pump (SmartTouch Pump, World Precision Instruments). Mice were deeply anaesthetized with isoflurane and transcardially perfused with PBS, followed by 4% paraformaldehyde (PFA) in PBS for fixation. The brains were embedded with OCT compound (Sakura Finetek Japan, 4583) and sliced into coronal sections with a thickness of 50 μm at −20 °C using cryostat (CryoStar NX70, Thermo Fisher Scientific, 956960). The sections were washed with PBS 3 times for 5 min each and stored in 50% glycerol (Fujifilm, 075-00616) in PBS at −20 °C. For staining with c-Fos antibody, the sections were permeabilized with PBS containing 1% Triton X-100 (Nacalai tesque, 35501), incubated at room temperature for 3 h, and then incubated with 0.3% Triton X-100 in PBS containing 1% normal donkey serum (NDS, Sigma-Aldrich, D9663) at room temperature for 1 h for blocking, and then incubated at 4 °C overnight in blocking buffer. With the following primary antibodies, Triton X-100 was used at 0.1% for permeabilization and PBS containing 1% NDS was used for blocking and antibody reactions: rat anti-GFAP (1:500, Thermo Fisher Scientific, 13-0300), mouse anti-S100β (1:500, Sigma-Aldrich, S2532), goat-Sox9 (1:100, R&D systems, AF3075) mouse anti-RFP (1:500, MBL, M155-3), rabbit anti-RFP (1:500, Rockland, 600-401-379), mouse anti-NET (1:2,000, MAb technologies, NET05-2) and rabbit anti-c-Fos (1:5,000, Synaptic Systems, 226003). After washing with PBS 3 times for 10 min each, the sections were incubated at room temperature for 2 h in blocking buffer with the following secondary antibodies: Alexa Fluor 488 donkey anti-rat, Alexa Fluor 488 donkey anti-chicken, Alexa Fluor 568 donkey anti-mouse, Alexa Fluor 568 donkey anti-rabbit, Alexa Fluor 568 donkey anti-goat, Alexa Fluor 647 donkey anti-rabbit, Alexa Fluor 647 donkey anti-mouse, Alexa Fluor 647 donkey anti-chicken (1:1,000, Abcam, ThermoFisher and Jackson ImmunoResearch). The sections were incubated at 4 °C overnight in blocking buffer with secondary antibodies when staining with anti-c-Fos antibody. After washing with PBS 3 times for 10 min each, the sections were mounted on slides (Fluoromount-G, Southern bio, 0100-35) and visualized under a confocal laser microscope (Olympus, FV3000) and a fluorescent microscope (Olympus, VS200). Immunoreactivities were analysed using ImageJ software (NIH). 4b) were quantified based on a consistent c-Fos intensity threshold for each experiment. This threshold was defined such that the number of c-Fos+ neurons or astrocytes corresponded to the values previously reported18,42,77. Number of cells, field of views, sections and mice used in each analysis is listed in the main text and/or figure legends accordingly. Mice were returned to the dark chamber for an additional two days and then returned to a regular light-dark cycle until the time of euthanasia seven days after tagging. Behavioural tests were performed during the light cycle between 07:00 and 19:00. All the experimental mice were transferred to the behaviour testing room at least 1 h before the tests to acclimatize to the environment and to reduce stress. Mice were habituated by freely exploring the conditioning chamber (CtxA) that consisted of a square partition (30 × 30 × 15 cm; 100 lux and no odour) with a grid floor wired to a shock generator (Ohara) for 15 min per day for 3 days. On the fourth day, they were either fear conditioned (1 time or 3 times shock; 2 s, 0.3 mA) (FC group) or presented with no shocks (NoFC group) in the conditioning chamber. One day after the conditioning, mice were either returned to CtxA for 3 min for fear memory recall (FR group) or kept in their home cages (NoFR group). Freezing durations exceeding 70% in the conditioned chamber 1 day post-conditioning with 3 foot shocks or exceeding 50% 1 day post-conditioning with a single foot shock, are considered indicative of sufficient conditioning. All the experimental mice received intraperitoneal injections of 4-OHT (50 mg kg−1) immediately after the task for Fos tagging. Freezing was automatically measured throughout the testing trial by the TimeFZ4 software (version 2021_7_29, Ohara) and subsequently processed by a blinded investigator. The chambers were cleaned with 70% ethanol between tests. In some cases, 10 mg kg−1 Propranolol (MedChemExpress, HY-B0573) was administered by intraperitoneal injection to mice 0.5 h before, and 2.5 h and 5.5 h after 4-OHT-based Fos tagging in order to block β-adrenergic signalling throughout the tagging timeframe. In some experiments using DREADDs, mice were injected with CNO (1 mg kg−1) 10 min before the task in CtxA. 2n in which CNO was used for silencing neuronal activity or propranolol was used for antagonizing β-adrenoreceptor signalling to test their roles in astrocyte Fos induction during fear memory recall, both were administered with consideration of their pharmacokinetics. Propranolol was given both before and after the behavioural session to ensure effective β-receptor blockade throughout the several-hour Fos-tagging window (Extended Data Fig. 3), given its short half-life78 (~1.5–2 h), whereas CNO was given once before the session owing to its prolonged effect79 (>6 h). Freezing was measured for 3 min in the CtxB chamber, which consisted of a rhombic partition (30 cm × 60 cm × 15 cm, 10 lux and 1% acetic acid odour) with a flat floor (Ohara). Mice were deeply anaesthetized with isoflurane and transcardially perfused with PBS, followed by 4% PFA in PBS for fixation of brain tissues. To oxidize agarose, 2.25 g agarose (A6013, Sigma-Aldrich) was added to 100 ml of 10 mM sodium periodate (S1878, Sigma-Aldrich)/50 mM phosphate buffer and stirred gently for 2.5 h at room temperature in the dark. The oxidized agarose was filtered on a bottle-top vacuum filter (0.2 μm, 569-0020, Thermo Scientific) washed 3 times with phosphate buffer and resuspended in 50 ml of phosphate buffer to make 4.5% oxidized agarose solution. The oxidized agarose solution was boiled with a microwave oven to melt the agarose and kept at 60–65 °C. The brains were pat-dried and embedded in oxidized agarose solution with a plastic mould (18646A, Polysciences). For covalent crosslinking between oxidized agarose and the surface of the brain, borate/borohydride solution was prepared the day before use by adding 0.2 g sodium borohydride (452882, Sigma-Aldrich) in 100 ml of prewarmed (40 °C) borate buffer (50 mM borax, 221732, Sigma-Aldrich; 50 mM boric acid, B6768, Sigma-Aldrich; pH 9.0–9.5) in the chemical fume hood. The agarose block was removed from the mould, soaked in borate/borohydride solution to initiate crosslinking and incubated at 4 °C overnight. The block was washed and kept in phosphate buffer at 4 °C until imaging. The agarose block was mounted on a glass slide using Loctite 401 glue immediately before imaging. STPT was performed with the TissueCyte 1400FC system (TissueVision) equipped with Coherent Chameleon Ultra II laser (Coherent) or Spectra-Physics Insight X3+ laser (MKS Spectra-Physics) and a Nikon 16× water-immersion objective (MRP07220, NA: 0.8). The laser wavelength was tuned at 1,000 nm. Brains were coronally sectioned with a built-in vibrating microtome at 50-μm thickness with a frequency of 60 Hz and a speed of 0.5 mm s−1. To scan an entire coronal plane, individual tile images were acquired (832 × 832 pixels per frame, resolution at 1.382 μm per pixel) by moving the stage between each tile. Following the full scanning of a coronal plane, a 50-μm section was cut away by vibrating microtome and imaging of a new cut surface repeated to acquire optical sections covering the entire brain from the caudal to the rostral side (~320 coronal sections). The raw image tiles were first trimmed and subjected to flat field correction, then stitched into 2D mosaic coronal section images using the Autostitcher software (TissueVision). Each region of the whole brain was aligned and segmented according to the Mouse Brain Atlas (NeuroInfo, MBL Bioscience, BN-200). Owing to the high quality preservation of tissue architecture during imaging, we semi-automatically performed alignment to a standard mouse brain atlas, segmentation of 839 brain regions, and detection of single Fos-tagged mNG astrocytes (BAEs), defined by their territories that typically displayed diameters of approximately 40 to 60 μm with ~10-µm-wide cell bodies68 (Fig. Consistent with our intersectional genetic approach, we rarely observed neuron-like mNG-positive cells, with larger cell bodies (>~20-μm diameter) and elongated processes (>~50-μm). We only found these cells in regions nearby ventricles and the midline where well-characterized GFAP-positive neural progenitor cells reside48,80, and they were excluded in subsequent analyses. As a result, using this brain-wide analysis approach we successfully quantified the density of BAEs in 677 brain regions that accounted for 80.7% of the whole brain. To observe correlation between Fos+ astrocytes and neurons, we used the same TRAP2 (Fos-iCreERT2) system28 used by Roy et al.5, but instead with a Cre-dependent mNG reporter expressed in Fos+ astrocytes, allowing comparison of Fos+ cell counts from anatomically matched regions between studies. The regions of interest were first narrowed to the 117 engram-signifiant regions defined by Roy et al. Then, to ensure analysis of bona fide astrocyte-tagged regions, areas with possible neuronal leakage (mNG+ neuron-like cells), particularly near ventricles were excluded (see above). In addition, high-level hierarchical structure areas (such as midbrain or thalamus) with extremely high total counts that skew correlation analysis were also excluded. In the end, this selection yielded 75 regions for subsequent correlational analyses. Changes in Fos+ astrocyte numbers (FC–NoFC and FR–NoFC) were calculated using the non-foot shocked control group as the baseline, consistent with Roy et al. Supplementary table 1 from Roy et al. provided region-by-region counts of Fos+ neurons in non-foot shocked controls, FC and FR groups, enabling us to extract the corresponding changes in Fos+ neuron numbers for each of the 75 regions. Pearson correlation analyses between these neuronal and astrocytic Fos+ cell count changes were performed. For image visualization in 3D, low-resolution image stacks were created across all four channels. These stacks were downsampled to 25 μm per pixel resolution in the x and y axes, while the inter-slice distance was maintained at 50 μm, resulting in a voxel resolution of 25 × 25 × 50 μm3. This template, generated from STPT averaged over 1,675 specimens, was combined with corresponding atlas annotations (filename: P56_Annotation.nii.gz). Image alignment was performed using the Advanced Normalization Tools (ANTs)82. This was followed by deformable (SyN) registration at four scales (downscaling factors of 8, 4, 2 and 1) with Gaussian smoothing of 4, 2, 1 and 1 voxels and 1,000, 500, 100 and 50 iterations, respectively, using mutual information as the metric. After computing the transformation, individual image stacks and atlas labels were mapped to and from the CCFv3 image space. After aligning with the NIfTI world coordinates, cell positions were mapped to the Allen template using the antsApplyTransformsToPoints function from ANTs. Brain regions from the Allen atlas annotations were then assigned to individual cells based on their locations in the CCFv3 image space. Astrocyte primary cultures were obtained as previously described with some modifications83. In brief, cerebral cortical tissues were isolated from P0–P1 neonatal mice and dissociated with a scalpel, gently agitated in 0.05% Trypsin-EDTA (Gibco, 25300-054), and rapidly strained with a Falcon 100 µm cell strainer (352360). Dissociated cells were seeded at a rate of four brains per bottle on poly-d-lysine-coated bottles. The cells were cultured in DMEM (Gibco, 11995073) supplemented with FBS (Cosmo Bio, CCP-FBS-BR-500) and penicillin-streptomycin. When the cells reached confluence after several weeks, the culture bottles were gently shaken at 120 rpm overnight at 37 °C to remove non-astrocytic cells. Astrocytes were then detached using Trypsin-EDTA, replated onto poly-d-lysine-coated bottles, and after 1–2 weeks, cultures were shaken again at 120 rpm for 4 h. Astrocytes were then re-seeded on coverslips in 12-well plates at a density of 1.0 × 105 cells per well, which were maintained in 5% CO2 at 37 °C, with the medium being changed every 2 to 3 days. Five days after re-seeding, primary astrocytes were incubated with tetrodotoxin (TTX at 1 µM, Fujifilm, 207-15901) to eliminate the potential effect of neuronal firing, for at least 5 min to allow adequate equilibration, together with the following inhibitors, blockers or chelators indicated for each experiment unless otherwise described: prazosin hydrochloride (MedChemExpress, HY-B0193A), atipamezole hydrochloride (Tokyo Chemical Industry, A2956), propranolol hydrochloride (MedChemExpress, HY-B0573), BAPTA-AM (SIGMA, A1076), and SQ 22536 (Tokyo chemical industry, Q0105). The cultures were then incubated with the following growth factors, neurotransmitters, neuromodulators or neuropeptides in 5% CO2 at 37 °C for 1.5 h: l-glutamic acid monosodium salt hydrate (Sigma-Aldrich, G5889), GABA (Sigma-Aldrich, A2129), ATP disodium salt hydrate from yeast (Nacalai tesque, 01072-24), l-noradrenaline hydrochloride (Sigma-Aldrich, 74480), dopamine hydrochloride (Sigma-Aldrich, H8502), acetylcholine chloride (Sigma-Aldrich, A6625), serotonin hydrochloride (Alfa Aesar, B21263.03), brain derived neurotrophic factor (BDNF, Fujifilm, 028-16451), neurotrophin-3 (Fujifilm, 146-09231, 50 ng ml−1), neurotrophin-4 (Fujifilm, 142-06634, 100 ng ml−1), forskolin (Fujifilm, 067-02191, 10 μM), nerve growth factor-β (NGFβ, Fujifilm, 141-07601, 100 ng ml−1), epidermal growth factor (EGF, Fujifilm, 059-07873, 100 ng ml−1), PDGF-AA (Fujifilm, 163–19731, 100 ng ml−1), PDGF-BB (Fujifilm, 164–24031, 100 ng ml−1), Ciliary Neurotrophic Factor (CNTF, Fujifilm, 032-23501, 100 ng ml−1), IGF1 (Fujifilm, 099-04511, 100 ng ml−1). The following agonist and ionophore were used: isoproterenol (MedChemExpress, HY-B0468) and ionomycin (LKT laboratories, I5752). Ninety minutes after incubation, primary astrocytes were fixed in 4% PFA (Nacalai Tesque) for 10 min, washed with PBS 3 times, permeabilized with 0.1% Triton X-100 (Nacalai Tesque) for 10 min and blocked with 1% NDS (Sigma-Aldrich, D9663) for 30 min. Primary astrocytes were then incubated overnight at 4 °C with the following primary antibodies: rabbit anti-c-Fos (1:1,000, Synaptic Systems, 226003), rat anti-GFAP (1:1,000, Thermo Fisher Scientific, 13-0300), rabbit anti-IBA1 (1:1,000, Fujifilm, 019-19741), chicken anti-TUJ1 (1:1,000 Novus Biologicals, NB100-1612), mouse anti-OLIG2 (1:100, Millipore, MABN50). The cultures were washed 3 times in PBS for 10 min each, then incubated with DAPI (1:10,000, Sigma-Aldrich, D9542) and the following secondary antibodies (1:1,000) in PBS containing 1% NDS for 1 h at room temperature: Alexa Fluor 488 donkey anti-rat (abcam, ab150153), Alexa Fluor 568 donkey anti-mouse (abcam, ab175700), Alexa Fluor 568 donkey anti-rabbit (Abcam, ab175692), Alexa Fluor 647 donkey anti-rabbit (abcam, ab150067), Alexa Fluor 647 donkey anti-chicken (Jackson ImmunoResearch, 703-605-155). Fluorescent images were taken using a 20× objective lens (UPlanXApo 20×, Olympus) with a confocal laser-scanning microscope (Olympus, FV3000 with Fluoview). Immunoreactivity quantification of c-Fos was performed using ImageJ. In some experiments involving a heterologous expression of mGluR3, we applied AAV2/5-GfaABC1D-Grm3 (800,000 multiplicity of infection) 5 h after re-seeding for expressing mGluR3 in primary astrocytes and incubated for 5 days before pharmacological experiments. Ninety minutes after stimulation with 0.1 µM NA and/or 100 µM glutamate in the presence of 1 µM TTX, primary astrocytes were fixed and stained as described above using primary antibodies: rabbit anti-mGluR3 (1:500, abcam, ab166608) and mouse anti-c-Fos (1:50, Santa Cruz Biotechnology, sc166940). It is noteworthy that the mouse anti-c-Fos antibody, different from the rabbit anti-c-Fos antibody used in other experiments, exhibited overall low immunoreactivity. However, this did not affect the quantification of c-Fos mobilization. Fluorescent imaging and quantification of c-Fos intensity were performed as described above. AAVs for expressing the noradrenaline biosensor GRABNE2h on the neuronal cell surface, or calcium indicator RCaMP3 in astrocytes with cAMP sensor cAMPinG1 in astrocytes were unilaterally injected into the basolateral amygdala. Optical fibres were implanted above the injection site two weeks post-injection to capture emission of fluorescence signals, with recordings conducted one week after fibre implantation. For fluorescence signal collection, mice were tethered to a patch cable, and a 465 nm LED light (Doric, 40–50 µW at the tip of patch cable) or a 560 nm LED light (Doric, 40–50 µW at the tip of patch cable) was used to excite the signal, with an isosbestic 405 nm LED light used to correct for movement artifacts. Mice tethered to the patch cable were habituated in a testing chamber for 5 min on the day of recording. A time-division multiplexing scheme was used, where the 465 nm, 560 nm and 405 nm LED lights were emitted alternately at 20 Hz (turned on for 24 ms and off for 26 ms), with the timing precisely controlled by a programmable pulse generator (Master-9, AMPI). Each excitation light was reflected by a dichroic mirror and coupled into an optical fibre cable (Doric, 200 μm in diameter, 2 m in length, NA 0.57) through a pinhole (200 μm in diameter). The fluorescence signals were detected by a photomultiplier tube with a GaAsP photocathode (H16722-40; Hamamatsu Photonics) and digitized by a data acquisition module (USB-6341, National Instruments). A custom-written LabVIEW program extracted signals at a sampling frequency of 1 kHz in real time. Fluorescence signals collected during each light pulse were averaged, after dropping the first and last 2 samples (1 ms each) to eliminate potential noise caused during the alternation of LED lights. The data were smoothed using moving average and corrected for bleaching correction by fitting to least-squares linear. The baselines of GRABNE2h and astrocyte cAMPinG1 signals were identified during periods of no spontaneous signals in home cages for 1–2 min before the context exposure. Signal analyses were performed in Python (v.3.0.0) using JupyterLab (v.3.6.7, Project Jupyter). The area under the curve of fluorescence traces was calculated with scikit-learn (sklearn, v.1.2.2, RRID:SCR_002577), and half-decay times of signal responses were extracted. Astrocytic RCaMP3 signals were bandpass filtered between 0.0003 and 1 Hz with SciPy85 (v.1.11.4). Peaks of Ca2+ signals were defined as data points exceeding 4× s.d. above the baseline mean, with baseline noise estimated from ≥20 s intervals lacking spontaneous activity in home cage recordings before context exposure. Coronal amygdalar slices were prepared from adult mice with AAV injection26,87. In brief, animals were deeply anaesthetized with isoflurane and decapitated with sharp shears. The brains were placed and sliced in ice-cold modified artificial cerebrospinal fluid (ACSF) containing the following (in mM): 194 sucrose, 30 NaCl, 4.5 KCl, 1 MgCl2, 26 NaHCO3, 1.2 NaH2PO4 and 10 d-glucose, saturated with 95% O2 and 5% CO2. A vibratome (DSK NLS-MT) was used to cut 300-µm brain sections. The slices were allowed to equilibrate for 30 min at 32–34 °C in normal ACSF containing (in mM); 124 NaCl, 4.5 KCl, 2 CaCl2, 1 MgCl2, 26 NaHCO3, 1.2 NaH2PO4 and 10 d-glucose continuously bubbled with 95% O2 and 5% CO2. Slices were then stored at 21–23 °C in the same buffer until use. All slices were used within 2–6 h of slicing. Cells for all the experiments were imaged with a 16× water-immersion objective lens (Nikon, N16XLWD-PF, MRP07220, NA: 0.8) with Nikon two-photon laser-scanning microscope (A1R MP) equipped with a 920 nm wavelength laser (Alcor, Spark Lasers). To image GCaMP6f signals, astrocytes located in the LA/B and typically ∼20 to ∼30 µm below the slice surface were selected for imaging. Images were acquired at 1 frame per second. Phenylephrine (PE, Tocris Bioscience 2838) was applied in the bath in the presence of 300 nM tetrodotoxin (Cayman Chemical 14964) were applied in the bath at least 5 min prior to recording to allow adequate equilibration. A constant flow of fresh buffer perfused the imaging chamber at all times. Spontaneous Ca2+ activity was recorded for 3 min (0 µM PE), followed by a 2-minute bath application of 1 µM PE, a 5-min washout, a 2-min application of 3 µM PE, and a 5-min washout. Ca2+ signals were processed in ImageJ (NIH) and presented as the relative change in fluorescence (ΔF/F). Mice were anaesthetized with isoflurane 90 min after the initiation of behavioural paradigms and transcardially perfused with PBS (30 ml for 2 min). The amygdala was freshly dissected out using a Brain Matrix (Ted Pella, 15067) and was placed into cold Hanks' balanced salt solution (HBSS) (14025092, Gibco) containing 10 mM HEPES (17514-15, Naclai Tesque), 0.54% glucose (16806-25, Naclai Tesque), 5.0 µg ml−1 actinomycin D (A1410, Sigma-Aldrich), 10 µM triptolide (T3652, Sigma-Aldrich) and 27.1 µg ml−1 anisomycin (A9789, Sigma-Aldrich), as previously described88,89. Then, the tissue block was incubated for 15 min at 37 °C in 3 ml enzyme solution (HBSS, 10 mM HEPES, 0.54% glucose, 0.5 mM EDTA (15575020, Invitrogen), 1.0 mM l-cysteine (10309-41, Naclai Tesque), 5.0 µg ml−1 actinomycin D, 10 µM triptolide and 27.1 µg ml−1 anisomycin) containing 20 U papain per mouse (LS003127, Worthington Biochemical). The tissue was then homogenized by passing through a 18 G needle and was further incubated for 15 min at 37 °C. Following homogenization using a 20 G needle, the resulting cell suspension was centrifuged at 500g for 5 min at 4 °C. The homogenate was separated by gradient centrifugation with 30% Percoll (P1644, Sigma-Aldrich) in PBS at 500g for 25 min at 4 °C (no brake). The pellet containing astrocytes at the bottom of the tube was then collected and washed once with PBS containing 2% FBS and 10 mM EDTA before staining. Fc receptors were blocked with Fc block (2.4G2, BD Bioscience) for 10 min at 4 °C before incubation with primary antibodies. Cells were stained with antibodies directed against CD11b-BV786 (1:200, M1/70, BD Bioscience), CD45-APC-Cy7 (1:200, 30-F11, BioLegend), O1-eFluor660 (1:200, 50-6506-82, eBioscience) and ASCA2-PE-Cy7 (1:200, 130-116-246, Miltenyi Biotec) for 40 min at 4 °C. Additionally, cells were treated with hashtag antibodies to label source samples (TotalSeqB0301-B0310, B0312 and B0314, BioLegend). After washing, cells were sorted using a CytoFLEX SRT (Beckman Coulter). Data were acquired with CytExpert software (Beckman Coulter). Post-acquisition analysis was performed using FlowJo software, version 10.9.0. Single-cell libraries were generated and sequenced on the Illumina NovaSeq 6000 sequencer. Sequence reads were processed and aligned to the mouse genome (mm10, GENCODE vM23/Ensembl98; https://cf.10xgenomics.com/supp/cell-exp/refdata-gex-mm10-2020-A.tar.gz) using CellRanger 3.0. Initially, genes expressed in <3 cells and cells with <100 genes and mitochondrial reads <6% were excluded from the analysis. Pericytes were removed from amygdalar cells based on Pdgfrb. Hashtag oligos (HTOs) were demultiplexed using HTODemux and singlet cells were used for subsequent analysis. Principal component analysis was performed on the expression data matrix using the Seurat. Thirty principal components were used for generating tSNE plots. Transcriptomic clusters were identified from a total 1,903 amygdalar astrocytes with resolution set to 0.4. Differentially expressed gene candidates from a comparison of Adrb1-positive and Adrb1-negative astrocytes were estimated using Mann–Whitney–Wilcoxon test. Cryosections were prepared as described above and stored at −80 °C. RNAscope in situ hybridization was performed using RNAscope Multiplex Fluorescent Reagent Kit v.2 with TSA Vivid Dyes (ACDBio, 323270), according to the manufacturer's instructions. The sections were washed for 5 min with PBS, and then heated at 60 °C for 30 min before post-fix. After dehydration in an ethanol series, hydrogen peroxide was applied for 10 min at room temperature. The sections were washed with ddH2O twice, and then incubated in Target Retrieval Reagents for 5 min at 90–100 °C. A wash with ddH2O for 15 s was followed by a dehydration with 100% ethanol for 3 min and dried at room temperature. Then, the sections were incubated with Protease III for 30 min at 40 °C. Thereafter, the sections were washed twice with wash buffer for 2 min between incubations at 40 °C. In the staining of mCherry, slices were washed in 0.1 M PBS 3 times for 10 min each, followed by immunohistochemistry that was performed as described above. Following primary and secondary antibodies used: rabbit anti-RFP (1:250, Rockland, 600-401-379), Alexa Fluor 568 donkey anti-rabbit (1:1,000, Abcam, ab175692). Finally, the sections were incubated with DAPI at room temperature, removed DAPI, and mounted in Prolong Gold Antifade Reagent (Invitrogen, P36934). Images were obtained in the same way as immunohistochemistry described above except with a step size of 0.5 µm. Astrocyte somata were delineated using Slc1a3 signals. To account for differences in somata sizes, the signals were normalized using Slc1a3. Subsequently, the normalized signals were z-scored within each sample to evaluate gene expression across a heterogeneous cell population. This approach facilitated the robust identification of cells exhibiting high gene expression relative to the population mean. 4e,f, adrenoreceptor-positive cells were identified based on scRNA-seq data, selecting the top 50% of cells ranked by Adra1a expression and the top 20% of cells ranked by Adrb1 expression. Co-expression index of Adra1a and Adrb1, calculated by relative expression level of Adra1a to NoFC median multiplied by relative expression level of Adrb1 for each cell. 4j, the correlation analysis of Fos and Adrb1, expression levels were evaluated using z-scores of RNAscope signal intensity to assess gene expression across a heterogeneous population of cells, enabling robust identification of cells with high gene expression relative to the population mean. Specifically, we identified astrocytes with z-scores greater than 2 in RNAscope Fos signal intensity as Fos-high astrocytes, capturing 3% of the astrocyte population with significantly elevated Fos expression, consistent with scRNA-seq data in Fig. Similarly, Adrb1-high astrocytes were identified using z-scores greater than 0.5, capturing 26% of astrocytes, also in line with scRNA-seq data in Fig. In addition, Igfbp2-high astrocytes were identified using z-scores greater than 0.5, capturing 24% of all astrocytes in Extended Data Fig. 12d,e, which is consistent with scRNA-seq data indicating that Igfbp2-positive astrocytes occupy 24% of the total astrocytes analysed. Most statistical tests were run in Prism 10.1.2 (GraphPad). The specific tests used are indicated in the figure legends and Supplementary Table 1. All tests were two-sided unless otherwise noted. Astrocyte Fos count analyses across groups for each brain region were run with R v.4.3.2 on RStudio v.2023.12.1.402 and relied on the tidyverse, car, dunn.test and dplyr packages and base R functions aov(), TukeyHSD() and p.adjust(). The data were first analysed with Levene's test for homogeneity of variance. Brain regions that passed Levene's test (P > 0.05) were followed by a one-way ANOVA test with Tukey's honest significant difference test and regions that failed Levene's test (P < 0.05; no homogeneity of variances) were followed by the Kruskal–Wallis test with Dunn's post hoc test and Bonferroni adjustment. False discovery rates were calculated across groups and brain regions from post hoc P values using the Benjamini–Hochberg method. Mice were randomly assigned to experimental groups, and mice assigned to different experimental conditions were run in parallel. Mouse experiments and analyses were done blinded to group allocation. Tissue samples that yielded insufficient cell counts (less than 30 cells) during scRNA-seq quality control were excluded from the study. The results of statistical comparisons, n numbers and P values are shown in the figure panels or figure legends along with averages. n is defined as the numbers of cells, sections or mice throughout on a case-by-case basis; the unit of analysis is stated in the text or in each figure legend. In the figures, summarized data are shown as mean ± s.e.m. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All the single-cell transcriptomics data are available at Gene Expression Omnibus with accession identifier GSE272414. All raw replicate data used to generate the figures and the associated statistical tests are provided in Supplementary Table 1. All raw data supporting the findings of this study are available on the RIKEN CBS data sharing platform (https://neurodata.riken.jp/id/20250831-001; https://doi.org/10.60178/cbs.20250831-001). Custom code for 3D rendering of the brain-wide Fos-mNG astrocyte map is available on GitHub (https://github.com/BrainImageAnalysis/Astrocyte3DMapping/). Custom code for statistical analysis brain-wide Fos-mNG astrocyte counts, fibre photometry data analysis and single-cell transcriptomic analysis is available on GitHub (https://github.com/Jun-Nagai-Lab/Dewa-et-al). Misanin, J. R., Miller, R. R. & Lewis, D. J. Retrograde amnesia produced by electroconvulsive shock after reactivation of a consolidated memory trace. & Sara, S. J. Reconsolidation of memory after its reactivation. The restless engram: consolidations never end. Lee, J. L. C., Nader, K. & Schiller, D. An update on memory reconsolidation updating. Roy, D. S. et al. Brain-wide mapping reveals that engrams for a single memory are distributed across multiple brain regions. Agren, T. et al. Disruption of reconsolidation erases a fear memory trace in the human amygdala. & Nader, K. Amyloid beta secreted during consolidation prevents memory malleability. Flavell, S. W. & Greenberg, M. E. Signaling mechanisms linking neuronal activity to gene expression and plasticity of the nervous system. & Lyons, D. A. Glia as architects of central nervous system formation and function. Nagai, J. et al. Behaviorally consequential astrocytic regulation of neural circuits. Endo, F. et al. Molecular basis of astrocyte diversity and morphology across the CNS in health and disease. Molofsky, A. V. et al. Astrocyte-encoded positional cues maintain sensorimotor circuit integrity. Siletti, K. et al. Transcriptomic diversity of cell types across the adult human brain. Murphy-Royal, C., Ching, S. & Papouin, T. A conceptual framework for astrocyte function. Induction of astrocytic Slc22a3 regulates sensory processing through histone serotonylation. Sardar, D. et al. Mapping astrocyte transcriptional signatures in response to neuroactive compounds. Williamson, M. R. et al. Learning-associated astrocyte ensembles regulate memory recall. Yu, X. et al. Context-specific striatal astrocyte molecular responses are phenotypically exploitable. Lee, H.-G. et al. Disease-associated astrocyte epigenetic memory promotes CNS pathology. Escartin, C. et al. Reactive astrocyte nomenclature, definitions, and future directions. Sun, W. et al. Spatial transcriptomics reveal neuron-astrocyte synergy in long-term memory. Making sense of astrocytic calcium signals - from acquisition to interpretation. Mu, Y. et al. Glia accumulate evidence that actions are futile and suppress unsuccessful behavior. Specific and behaviorally consequential astrocyte Gq GPCR signaling attenuation in vivo with iβARK. Engineered AAVs for efficient noninvasive gene delivery to the central and peripheral nervous systems. DeNardo, L. A. et al. Temporal evolution of cortical ensembles promoting remote memory retrieval. Nagai, J. et al. Hyperactivity with disrupted attention by activation of an astrocyte synaptogenic cue. Adamsky, A. et al. Astrocytic activation generates de novo neuronal potentiation and memory enhancement. Fanselow, M. S. Factors governing one-trial contextual conditioning. Serial two-photon tomography for automated ex vivo mouse brain imaging. Piantadosi, S. C. et al. Holographic stimulation of opposing amygdala ensembles bidirectionally modulates valence-specific behavior via mutual inhibition. Long-term memory is facilitated by cAMP response element-binding protein overexpression in the amygdala. Tronson, N. C., Wiseman, S. L., Olausson, P. & Taylor, J. R. Bidirectional behavioral plasticity of memory reconsolidation depends on amygdalar protein kinase A. Nat. Lin, Y. et al. Activity-dependent regulation of inhibitory synapse development by Npas4. Smith, N. A. et al. Fluorescent Ca2+ indicators directly inhibit the Na,K-ATPase and disrupt cellular functions. Rapid manifestation of reactive astrogliosis in acute hippocampal brain slices. Sun, W. et al. Glutamate-dependent neuroglial calcium signaling differs between young and adult brain. Pfeil, E. M. et al. Heterotrimeric G protein subunit Gαq is a master switch for Gβγ-mediated calcium mobilization by Gi-coupled GPCRs. Sun, X. et al. Functionally distinct neuronal ensembles within the memory engram. Sørensen, A. T. et al. A robust activity marking system for exploring active neuronal ensembles. Uematsu, A. et al. Modular organization of the brainstem noradrenaline system coordinates opposing learning states. Monitoring norepinephrine release in vivo using next-generation GRABNE sensors. Yokoyama, T. et al. A multicolor suite for deciphering population coding of calcium and cAMP in vivo. Suthard, R. L. et al. Basolateral amygdala astrocytes are engaged by the acquisition and expression of a contextual fear memory. Batiuk, M. Y. et al. An immunoaffinity-based method for isolating ultrapure adult astrocytes based on ATP1B2 targeting by the ACSA-2 antibody. Furube, E. et al. Neural stem cell phenotype of tanycyte-like ependymal cells in the circumventricular organs and central canal of adult mouse brain. Kohro, Y. et al. Spinal astrocytes in superficial laminae gate brainstem descending control of mechanosensory hypersensitivity. In vivo effects of cardiomyocyte-specific β-1 blockade on afterload- and frequency-dependent cardiac performance. Gleichman, A. J., Kawaguchi, R., Sofroniew, M. V. & Carmichael, S. T. A toolbox of astrocyte-specific, serotype-independent adeno-associated viral vectors using microRNA targeting sequences. Caldwell, A. L. M. et al. Aberrant astrocyte protein secretion contributes to altered neuronal development in multiple models of neurodevelopmental disorders. Tu, X. et al. Local autocrine plasticity signaling in single dendritic spines by insulin-like growth factors. Khan, S. et al. IGFBP2 plays an essential role in cognitive development during early life. Han, J.-H. et al. Increasing CREB in the auditory thalamus enhances memory and generalization of auditory conditioned fear. Ramsaran, A. I. et al. A shift in the mechanisms controlling hippocampal engram formation during brain maturation. McCall, J. G. et al. Locus coeruleus to basolateral amygdala noradrenergic projections promote anxiety-like behavior. Ramón y Cajal, S. Algunas conjeturas sobre el mecanismo anatómico de la ideación, asociación y atención. Cahill, M. K. et al. Network-level encoding of local neurotransmitters in cortical astrocytes. Gong, L., Pasqualetti, F., Papouin, T. & Ching, S. Astrocytes as a mechanism for contextually-guided network dynamics and function. Distinct temporal integration of noradrenaline signaling by astrocytic second messengers during vigilance. Sapkota, D. et al. Activity-dependent translation dynamically alters the proteome of the perisynaptic astrocyte process. Choucry, A., Nomoto, M. & Inokuchi, K. Engram mechanisms of memory linking and identity. Zaki, Y. et al. Offline ensemble co-reactivation links memories across days. & Byrne, J. H. The right time to learn: mechanisms and optimization of spaced learning. Nader, K., Schafe, G. E. & Le Doux, J. E. Fear memories require protein synthesis in the amygdala for reconsolidation after retrieval. & Ledoux, J. E. Disruption of reconsolidation but not consolidation of auditory fear conditioning by noradrenergic blockade in the amygdala. Baldwin, K. T., Murai, K. K. & Khakh, B. S. Astrocyte morphology. Naegeli, C. et al. Locus coeruleus activity mediates hyperresponsiveness in posttraumatic stress disorder. Cornell-Bell, A. H., Finkbeiner, S. M., Cooper, M. S. & Smith, S. J. Glutamate induces calcium waves in cultured astrocytes: long-range glial signaling. Ollivier, M. et al. Crym-positive striatal astrocytes gate perseverative behaviour. Rusakov, D. A., Bard, L., Stewart, M. G. & Henneberger, C. Diversity of astroglial functions alludes to subcellular specialisation. King, C. M. et al. Local resting Ca2+ controls the scale of astroglial Ca2+ signals. Rational engineering of XCaMPs, a multicolor GECI suite for in vivo imaging of complex brain circuit dynamics. Stress disrupts engram ensembles in lateral amygdala to generalize threat memory in mice. Levy, A., Ngai, S. H., Finck, A. D., Kawashima, K. & Spector, S. Disposition of propranolol isomers in mice. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. The Allen Mouse Brain Common Coordinate Framework: a 3D reference atlas. B., Epstein, C. L., Grossman, M. & Gee, J. C. Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain. Schildge, S., Bohrer, C., Beck, K. & Schachtrup, C. Isolation and culture of mouse cortical astrocytes. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Yu, X. et al. Reducing astrocyte calcium signaling in vivo alters striatal microcircuits and causes repetitive behavior. Yamasaki, A. et al. IRF8 and MAFB drive distinct transcriptional machineries in different resident macrophages of the central nervous system. Marsh, S. E. et al. Dissection of artifactual and confounding glial signatures by single-cell sequencing of mouse and human brain. We thank N. Fujimori, R. Ando, N. Miyasaka and N. Mataga for technical assistance; M. Motoyoshi for assisting with imaging; Y. Li for sharing GRAB sensors; and T. J. McHugh, J. P. Johansen, T. Kitamura, K. Tanaka, T. Kitazawa, T. Kanaya, B. Wulaer, E. Shigetomi, S. Okabe, B. Deneen and B. S. Khakh for valuable comments and discussions. STPT, cryosectioning and image acquisition were performed with common use equipment at the Support Unit for Bio-Material Analysis, RRD of RIKEN CBS. We thank the Division of Gene Expression Dynamics, the MEXT Cooperative Research Project Program, the Medical Research Center Initiative for High Depth Omics, and acknowledge support from CURE:JPMXP1323015486 for the Medical Institute of Bioregulation, Kyushu University for sequencing. groups are supported by Japan Agency for Medical Research and Development (AMED) grant JP24gm1910004. This work was supported in part by JSPS KAKENHI grant numbers 24K02128 (to J.N. by Japan Science and Technology Agency (JST) under grant numbers JPMJFR2249 (to J.N. ), by AMED under grant numbers JP21km0908001 (to J.N. ), Multidisciplinary Frontier Brain and Neuroscience Discoveries (Brain/MINDS 2.0) by AMED JP23wm0625001 (to H.S. ), by RIKEN-Osaka University Science and Technology Hub Collaborative Research Program (to J.N. ), and by the Takeda Science Foundation (to J.N. ), Daiichi Sankyo Foundation of Life Science (to T.M. ), Mochida Memorial Foundation for Medical and Pharmaceutical Research (to T.M. ), Astellas Foundation for Research on Metabolic Disorders (to T.M. ), Ono Pharmaceutical Foundation for Oncology (to T.M. and the Uehara Memorial Foundation (to T.M.). were partly supported by JSPS Research Fellowships (24KJ2074 and 22KJ3141, respectively). These authors contributed equally: Ken-ichi Dewa, Kodai Kaseda Ken-ichi Dewa, Kodai Kaseda, Aoi Kuwahara, Hideaki Kubotera, Atsuko Komori, Mika A. Holtz & Jun Nagai Department of Life Science and Medical Bioscience, Graduate School of Advanced Science and Engineering, TWIns, Waseda University, Tokyo, Japan Kodai Kaseda, Aoi Kuwahara & Jun Nagai Division of Allergy and Immunology, Medical Institute of Bioregulation, Kyushu University, Fukuoka, Japan Laboratory of Molecular Neuropharmacology, Graduate School of Pharamceutical Sciences, Osaka University, Suita, Japan Japan Science and Technology Agency (JST), Fusion Oriented REsearch for disruptive Science and Technology (FOREST), Kawaguchi, Japan Atsushi Kasai, Minako Ito & Jun Nagai Division of Brain Sciences, Institute for Advanced Medical Research, Keio University School of Medicine, Tokyo, Japan Department of Optical Neural and Molecular Physiology, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Center for Living Systems Information Science, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Department of Brain Development and Regeneration, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Laboratory of Deconstruction of Stem Cells, Institute for Frontier Life and Medical Sciences, Kyoto University, Kyoto, Japan Department of Molecular and System Pharmacology, Graduate School of Pharmaceutical Sciences, Kyushu University, Fukuoka, Japan Kyushu University Institute for Advanced Study, Fukuoka, Japan Division of Cardiology, Department of Medicine, The Johns Hopkins Medical Institutions, Baltimore, MD, USA Precursory Research for Embryonic Science and Technology (PRESTO), Japan Science and Technology Agency, Kawaguchi, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out vector constructions with help from A. Komori, performed mouse surgeries and behaviour tests, and with M.A.H. A. Kasai provided guidance on the analysis of brain-wide data. performed ex vivo astrocyte calcium imaging and analysis. A. Kuwahara performed primary astrocyte experiments with help from K.D., who established the protocol. conducted in vivo fibre photometry and animal behaviour experiments, including associated surgeries with guidance from N.T. on coding for data analysis. provided guidance on astrocyte sensors. provided key mouse lines and contributed intellectually to related analyses. Tissue preparation for single-cell transcriptomics was performed by K.D., K.K., H.K. and analysis of sequencing data by K.K. performed in vivo infusion of antibodies, for which K.K. performed mouse surgeries and histological analyses. conceptualized and directed the project, guided analyses and wrote the paper with help from K.D., K.K., A.K. All authors commented on and approved the manuscript. The authors declare no competing interests. Nature thanks Christian Henneberger, Gertrudis Perea and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic illustrating an AAV PHP.eB vector intravenous (i.v.) injection into retro orbital sinus of adult mice for expressing red fluorescence reporter mScarlet-I in astrocytes in whole brain. b, Representative images showing that whole-brain expression of mScarlet-I was observed in AAV injected mice, but not in saline injected mice. AAV PHP.eB-mediated mScarlet-I delivery resulted in expression within S100β/GFAP positive astrocytes. c, Bar plots showing that 97.3%–99.5% of astrocytes were positive for mScarlet-I, whose signals were enhanced with anti-RFP antibody, in the indicated 7 brain regions (left) and that those regions had few mScarlet-I-positive neurons (right). Data are mean ± SEM (error bars smaller than symbols in some cases). Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. AAV2/5-GfaABC1D-hM3Dq-mCherry was stereotaxically delivered into the orbitofrontal cortex (ORB) to induce Fos in astrocytes by 1 mg/kg CNO i.p. injection and tag those Fos+ astrocytes with mNG by 50 mg/kg 4-hydroxytamoxifen (4-OHT) i.p. b, hM3Dq activation with CNO administration increased Fos expression in ORB astrocytes. Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. b, Timeline of visual stimulation experiment to determine effective time window of astrocyte Fos tagging. c, Example images of Fos-tagged (mNG-positive) astrocytes in the primary visual cortex (VISp) of TRAP2 mice that underwent the visual stimulation experiment (representative of 9–12 slices from 3-4 mice/group). d, Quantification of mNG astrocyte density in TRAP2 mice VISp and primary somatosensory cortex (SSp): NS: n = 12 slices from 4 mice, −24 h: n = 12 slices from 4 mice, −12 h: n = 12 slices from 3 mice, −6 h: n = 12 slices from 4 mice, −3 h: n = 12 slides from 4 mice, 0 h: n = 12 slices from 4 mice, 3 h: n = 9 slices from 3 mice, 6 h: n = 9 slices from 3 mice, 12 h: n = 9 slices from 3 mice, 24 h: n = 9 slices from 3 mice. Two-way ANOVA, Šídák's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. a, Representative immunostaining of c-Fos expression in a subset of amygdalar astrocytes after fear conditioning (FC) and fear recall (FR). Arrowheads denote c-Fos+ Sox9+ cells (representative of 15 slices from 3 mice/group). b, Quantification of c-Fos+ astrocytes in the lateral and basolateral amygdala (LA/B) after FC and FR. Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. b, Timeline of fear recall experiment to determine effective time window of astrocyte Fos tagging. c, Example images of Fos-tagged (mNG-positive) astrocytes in the lateral and basolateral amygdala (LA/B) of TRAP2 mice that underwent the fear recall experiment (representative of 17–48 slices from 3–8 mice/group). Two-way ANOVA, Šídák's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. Alongside NoFC and FC groups (Fig. 1a), an additional cohort received 4-OHT immediately after CtxA exposure with shocks but no habituation (FC+Novelty). Brains were fixed and processed for STPT, 3D reconstruction, regional segmentation, and single-cell counting as in Fig. b, 3D rendering of the whole brain and three brain regions - the hippocampus (HPC; CA1, CA2, CA3 and dentate gyrus), the amygdala (lateral, basolateral and basomedial amygdalar subregions), primary visual cortex (VISp) - showing that Fos-mNG astrocytes (white dots). D dorsal, V ventral, A anterior, P posterior, M medial, L lateral. c-d, Heatmap of BAE count fold changes (vs NoFC in c, vs FC in d, n = 4 mice/group). *FDR < 0.05; full stats in Supplementary Table 2. Levene's test followed by one-way ANOVA or Kruskal-Wallis; post-hoc Tukey or Dunn with Bonferroni. Primary visual area (VISp), primary somatosensory area barrel field (SSp-bfd), orbital cortex (ORB), prelimbic cortex (PL), infralimbic cortex (ILA), entorhinal cortex (ENT), anterior cingulate cortex (ACA), olfactory cortex (OLF), retrosplenial cortex (RSP), lateral amygdala (LA), basal amygdala (BA), central amygdala (CEA), mediodorsal thalamus (MD), dorsal geniculate thalamus (GENd), anterodorsal thalamus (AD), anteroventral thalamus (AV), ventroposterior medial nucleus of thalamus (VPM), caudoputamen (CP), nucleus accumbens (ACB), dentate gyrus (DG), hippocampal CA1 (CA1), hippocampal CA3 (CA3), para-subiculum (PAR), and pre-subiculum (PRE). Sample sizes and replicates in Supplementary Table 1. a, Schematics illustrating the design of pharmacological screening. b, Representative images from 9 biological replicates showing high purity of primary astrocyte culture. The immunoreactivities of Iba1, a microglia marker, Tuj1, a neuronal marker and Olig2, an oligodendrocyte marker, were not evident, and most of the DAPI signal overlapped with GFAP-positive primary astrocytes. c, Graph showing that c-Fos intensity in GFAP-positive astrocytes was increased with noradrenaline (NA), but not with growth factors that are known to induce c-Fos expression in primary neurons. TTX (1 µM) was used throughout the culture experiments to eliminate the potential effect of neuronal firing. One-way ANOVA, Dunnett's test (vs Veh). Sample sizes and replicates in Supplementary Table 1. Mice underwent fear conditioning two weeks later followed by i.p. Brain samples were intracardially fixed for immunohistochemistry analysis. b, Graphs showing the percentage of RAM-expressed neurons in LA/B (left), co-expression of c-Fos and RAM (center), and overall c-Fos+ neurons (right). ****p < 0.0001. c, Representative images (from mKate2: n = 15 slices, hM4Di: n = 14 slices, 5 mice/group) showing expression of neuronal marker NeuN and RAM-tagged engram neurons (indicated by mCherry signals) in b. Overlapping c-Fos-expressing engram neurons were fewer in the RAM-hM4Di group. Sample sizes and replicates in Supplementary Table 1. a, Images (representative of n = 16 slices from 4 mice) showing FR-BAE (green) in the amygdala overlapping with astrocyte markers GFAP and S100β (cyan) and proximal to NA projections (white) and engram neurons (red). b, Graphs showing that engram neurons (left) and NA projections (center) are statistically more colocalized with FR-BAE than non-FR-BAE astrocytes. Astrocyte territory size is not different between BAE and non-FR-BAE astrocytes (right). Sample sizes and replicates in Supplementary Table 1. Termination sites were mapped onto a standard atlas (adapted from Paxinos, G. & Franklin, K.B.J. The Mouse Brain in Stereotaxic Coordinates, 2nd ed. Circles show fiber tips from individual mice, color-coded by experimental group. b, Freezing before, during, after FC, during FR, and in CtxB (distinct from CtxA). Freezing was lower in CtxB and in CtxA with Prop vs FR (FR: n = 14; CtxB: n = 13; Prop: n = 8 mice). c, dF/F traces: NA signals evoked by CtxA or CtxB (early) persisted longer in CtxA (late). d, Scatter plots: integrated areas of early and late NA signals in CtxA vs CtxB. e, Astrocyte cAMP traces: gradual buildup in HC, strong early responses to context exposure. FR further enhanced late cAMP in CtxA, not CtxB. f, Scatter plots: late cAMP signal areas higher in CtxA vs CtxB, consistent with GRABNE2h. g, Astrocyte Ca2+ traces: evoked by context exposure (early). At 7–15 min, signal frequency remained elevated in CtxA, not CtxB (late). h, Scatter plots: late Ca2+ frequency higher in CtxA vs CtxB. Wilcoxon signed-rank test or paired t test was used based on normality of the data distribution. Data are mean ± SEM (error bars smaller than symbols in some cases). Sample sizes and replicates in Supplementary Table 1. a, Representative gating strategy of FACS-sorting of ACSA2+ astrocytes from the amygdala for scRNA-seq. b, Violin plot showing relative expression levels of cell type marker genes in scRNA-seq. Aldoc, an astrocyte marker, was the only gene showing high expression. c, tSNE map showing astrocyte genes Aldoc, Atp1a2 (coding astrocyte cell surface antigen-2, ACSA2 with which we aimed to FACS-sort astrocytes from the amygdala tissue), and Slc1a3 with which we identified high expression in astrocytes across amygdalar astrocytes in our RNAscope experiments. d, Left: heatmap showing the top 20 genes that were upregulated in each astrocyte cluster relative to the others. Right: Example cluster marker genes identified were plotted onto the tSNE map. See also Supplementary Table 3. e, Left: tSNE plot of astrocyte clusters to illustrate that the 4 clusters are found in every single mouse examined. Right: Bar graph showing the proportion of each astrocyte cluster in the four experimental groups. a, Top 10 differentially expressed gene candidates, ranked by p value comparing Adrb1-positive vs. -negative astrocytes. Heatmap shows decreased (blue) or increased (red) expression. b, Violin plot showing Igfbp2 enrichment in Adrb1-positive astrocytes. c, RNAscope images (representative of 588 cells from 6 mice) of Adrb1 and Igfbp2 in Slc1a3-expressing astrocytes in LA/B from FR. e, Adrb1 expression enriched in Igfbp2-high astrocytes (n = 451 low, 137 high cells). Igfbp2 expression enriched in Adrb1-high astrocytes (n = 426 low, 162 high cells). f, Schematic of in vivo Igfbp2 perturbation. Mice implanted with bilateral cannulae into LA/B for Igfbp2-neutralizing antibody or control IgG infusion immediately after FR. g, Scatter graph: IgG infusion (n = 9 mice) showed no change in freezing between 1st and 2nd FR, whereas Igfbp2 antibody infusion (n = 12 mice) reduced freezing in 2nd FR, indicating destabilized memory. h, Freezing behaviour during 5-day Igfbp2 antibody infusions into LA/B within reconsolidation windows. Two-way ANOVA with Šídák's test. Sites mapped on a standard atlas (adapted from Paxinos, G. & Franklin, K.B.J. The Mouse Brain in Stereotaxic Coordinates, 2nd ed. Each circle indicates cannula termination, coloured by group. Data are mean and SEM (error bars smaller than symbols in some cases). a, Schematics showing the Gαq GPCR signalling cascade and the blocking of which with iβARK. Left: iβARK, a 122-residue inhibitory peptide from β-adrenergic receptor kinase26, directly binds to Gαq subunits and thus sequesters them to block subsequent Gαq-GPCR-induced signalling cascades. Right: iβARK2 with Lck allows it to localize to the plasma membrane, thus being more proximal to and binding Gαq subunits more efficiently. b, Top: AAV2/5 constructs for astrocytic iβARK or iβARKmut. c, Schematic for Ca2+ imaging of iβARK-positive LA/B astrocytes. d, Representative images showing expression of iβARK and iβARK2 in LA/B. e, Representative pseudo-coloured images showing the high fluorescence of GCaMP6f in LA/B astrocytes (representative of n = 28 iβARK2mut and 13 iβARK2 astrocytes). f, dF/F traces of Ca2+ responses in a iβARK(D110A)- or iβARK-expressing astrocyte evoked by PE (phenylephrine), an agonist of Gαq-coupled α1A-adrenoceptor (coded by Adra1a) analyzed for soma and processes separately. g, Scatter graph showing the average Ca2+ responses to 0, 1, and 3 µM PE, where iβARK significantly reduced the PE-induced Ca2+ responses. Two-way ANOVA, Tukey's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. a, Graphs showing no alteration in locomotion of Adrb1-OE mice during the open field test. Two-way ANOVA and Šídák's multiple comparisons test were used. b, Graphs showing no alteration in time spent in center zone of Adrb1-OE mice during the open field test. c-k, Minute-by-minute freezing behaviour of Ctrl and Adrb1-OE mice across conditioning and recall sessions. After three-shock FC (c), freezing was measured during the 1st (d) and 2nd FR (e). After one-shock FC (f), freezing was measured during the 1st (g) and 2nd FR (h), with Adrb1-OE mice showing significantly more freezing. In a separate three-shock FC cohort (i), freezing was measured during the 1st FR (j) and CtxB exposure (k), where Adrb1-OE mice also showed increased freezing. Two-way ANOVA with Šídák's test. Sample sizes and replicates in Supplementary Table 1. Details of raw replicates, statistical analyses, n numbers and relevant parameters, related to all figures. Fos-mNG astrocyte counts in 677 brain regions in NoFC, FC, FC + novelty, NoFR and FR groups. Single-cell transcriptomic data including the list of top 20 genes that differentiate astrocyte clusters and the differentially expressed genes estimated between Adrb1-positive and negative astrocytes. AAV constructs and Addgene IDs. STPT images, astrocyte marking, and brain region segmentation in the NoFC group. STPT images, astrocyte marking, and brain region segmentation in the FR group. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The astrocytic ensemble acts as a multiday trace to stabilize memory. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/mens-brains-shrink-faster-than-womens-what-that-means-for-alzheimers/'>Men's Brains Shrink Faster than Women's. What That Means for Alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Women's brains age more slowly than men's, but they still have higher rates of Alzheimer's disease During ageing, men experience a greater reduction in volume across more regions of the brain than women do, according to a longitudinal study published today in the Proceedings of the National Academy of Sciences. The authors suggest this means that age-related brain changes do not explain why women are more frequently diagnosed with Alzheimer's disease than men are. “It's really important that we understand what happens in the healthy brain so that we can better understand what happens when people get these neurodegenerative conditions,” says Fiona Kumfor, a clinical neuropsychologist at the University of Sydney, Australia. Nearly twice as many women are diagnosed with Alzheimer's disease as men, and ageing is the biggest risk factor for the disease. This has prompted research into age-related sex differences in the brain. “If women's brains declined more, that could have helped explain their higher Alzheimer's prevalence,” says co-author Anne Ravndal, a PhD student at the University of Oslo. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Previous research investigating sex differences in brain ageing has shown mixed results, Ravndal adds. Several studies have found that men experience greater loss of total grey matter and hippocampus size compared with women, whereas other work has reported a sharper decline of grey matter in women. The latest study included more than 12,500 magnetic resonance imaging (MRI) brain scans from 4,726 people — at least two scans per person, taken an average of three years apart — who did not have Alzheimer's disease or any cognitive impairments and were control participants in 14 larger data sets. Overall, men experienced a greater reduction in volume across more regions of the brain than women did. For example, the postcentral cortex, which is responsible for processing sensations of touch, pain and temperature, as well as the body's own position and movements, declined by 2.0% per year in men and by 1.2% annually in women. Kumfor says the results suggest that men age faster than women do. That sex differences in brain volume don't seem to play a part in the condition is not entirely surprising, says Kumfor, because neurodegenerative conditions are complex. Understanding the disorders will require longitudinal studies involving people with Alzheimer's disease, to compare how their brains change over time. “Just looking at age-related changes in brain atrophy is unlikely to explain the complexities behind it,” she adds. Although the study is robust, further investigations with more diverse data sets are needed, says Brodtmann. For example, people included in the study had high levels of education — a protective factor against Alzheimer's disease — meaning that they were not fully representative of the general population. The data sets also lacked information about other factors that influence a person's risk of Alzheimer's disease, such as ethnicity and the age at which women experience menopause. The authors say that when they adjusted their analysis for education level, some parts of men's brains no longer showed a steeper decline than occurred in women. This article is reproduced with permission and was first published on October 13, 2025. First published in 1869, Nature is the world's leading multidisciplinary science journal. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251015032316.htm'>This European treatment for joint pain just passed a major scientific test</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 14:47:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Participants with mild to moderate knee osteoarthritis experienced notable pain reduction and improved mobility within four months of receiving the treatment. The radiation exposure was only a tiny fraction of what is used in cancer therapy. Because the study included a control group that underwent a simulated treatment, researchers were able to clearly separate genuine treatment effects from placebo responses, which are common in osteoarthritis trials. Early results from this Korean study were presented at the American Society for Radiation Oncology (ASTRO) Annual Meeting. "People with painful knee osteoarthritis often face a difficult choice between the risks of side effects from pain medications and the risks of joint replacement surgery," said Byoung Hyuck Kim, MD, PhD, principal investigator on the trial and an assistant professor of radiation oncology at Seoul National University College of Medicine, Boramae Medical Center. It develops as the cartilage that cushions bone ends wears away over time. Low-dose radiation therapy has long been used in European countries such as Germany and Spain to treat joint pain, where it is widely accepted. However, Dr. Kim noted that before this study, rigorous placebo-controlled evidence was scarce, and awareness of the treatment remained low among clinicians in other parts of the world. In this study, he explained, the radiation dose was less than 5% of what is typically used in cancer therapy, and no radiation-related side effects were observed. The multicenter study included 114 participants with mild to moderate knee osteoarthritis, recruited from three academic hospitals in Korea. Each participant underwent six treatment sessions without knowing which group they were assigned to. The effectiveness of treatment was evaluated using internationally recognized standards, identifying a participant as a "responder" if they achieved meaningful improvement in at least two of three measures: pain, physical function, and overall assessment of their condition. Participants also completed separate questionnaires addressing pain, stiffness, and mobility. For other secondary outcomes, including the amount of pain medication needed, there were no significant differences. Dr. Kim said this trial differed from previous research in two critical ways. "The sham-controlled design helped rule out placebo effects, and we limited stronger analgesics, which made differences between groups more clearly attributable to the radiation itself," he said. But using these pain relievers could mask the effects of radiation therapy," he said. Responses in the placebo arms were substantial -- about 40% met the criteria for treatment response without true radiation -- but are consistent with rates reported in prior osteoarthritis trials of injections or medications and in at least one similar European study, Dr. Kim said. We need to examine this more closely in future studies." Radiation therapy may be best suited for patients with underlying inflammation and preserved joint structure, he explained. He emphasized that low-dose radiation should be considered as part of shared decision-making alongside standard measures such as weight loss, physiotherapy, and medications. "In clinical practice, responses could be even stronger when radiation is properly combined with other treatments, and patient satisfaction may be higher than with current options alone." Planned studies also include larger, pragmatic trials to evaluate outcomes in specific subgroups and health-economic analyses comparing low-dose radiation with injections and medication regimens. Scientists Extend Lifespan by over 70% in Elderly Male Mice with New Treatment The “Really Big One” Might Trigger California's Next Catastrophe, Scientists Warn Promising New Treatment Could Calm Anxiety for Millions of Americans Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69001262/fault-lines-synched-together/'>Two of the Biggest Fault Lines in the World Are Synched Together. That Could Be Disastrous.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers hoping to unravel the dance of earthquake movement in North America's West Coast discovered a disturbing fact: Two of the world's largest fault lines sometimes work in conjunction. In a new study published in the journal Geosphere, researchers from Oregon State University—led by Chris Goldfinger, a marine geologist and geophysicist—show evidence of what's called “partial synchronization” of the northern San Andreas Fault and Cascadia Subduction Zone. That likely implies those unique turbidite formations were stacked by two earthquakes, one from each zone, in quick succession—a matter of years or even minutes apart. “When you tune an old radio, you're essentially causing one oscillator to vibrate at the same frequency as the other one,” he said. “When these faults synchronize, one fault could tune up the other and cause earthquakes in pairs.” “If they both went off together, then you've got potentially San Francisco, Portland, Seattle, and Vancouver [B.C.] “But it's pretty clear that if something like this happened—and we think the evidence for it is strong—we need to be prepared.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Go Deeper Into Mantle Than Ever Before Experts Found an Ancient  Tomb Full of Treasure An Elusive Rat Finally Showed Its Face to Science Chinese Tomb Mural from 8th Century Has Blonde Man Scientists Just Solved a Major Mystery About Hail</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69022085/stevia-hair-loss/'>Scientists Found a Chemical That Could Help Regrow Your Hair. It Might Be In Your Pantry.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>One of the most widespread medications for baldness, which is the active ingredient in products like Rogaine, is minoxidil (originally developed to treat high blood pressure). While this treatment has helped many people grow hair, it does come with a few downsides. Standard treatments typically have poor water solubility and skin permeability, so some hair growth experts resort to applying minoxidil with microneedling, a technique that punctures small holes in the top layer of the skin to stimulate growth factors and increase the medication's absorption. As described in a new study, scientists from China and Australia have developed a dissolving patch that can perform this microneedling work while also delivering a dose of minoxidil to the scalp, and it's derived from a somewhat unlikely ingredient: Stevioside (STV), the natural sweetener derived from the Stevia plant. “This method not only enhances drug penetration into skin but also eliminates other issues associated with traditional topical formulations, such as slow onset of action and/or inaccurate dosing.” Minoxidil, however, suppresses certain enzymes and inhibits androgen receptors, which has the net effect of decreasing the formation and binding of dihydrotestosterone. It also boosts aromatase, which converts testosterone into estradiol—a kind of estrogen that further stimulates hair growth. In 35 days, scientists saw hair regrowth in 67 percent of the target area, compared to only 25 percent from the control group—a promising start for this Stevia-based hair loss treatment. “While the mouse model results are promising, human hair growth cycles differ, and androgenetic alopecia is influenced by multiple factors,” Lifeng Kang, a co-author of the study from the University of Sydney, told Gizmodo. “Therefore, clinical trials are essential to confirm efficacy and safety in humans.” Unwanted hair loss isn't a thing of the past (at least, not yet), but science has it in its crosshairs—no pun intended. A Scientist Says We Can Slow Time With Our Minds Scientists Found a Surprising Way To Target Deadly Tumors</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64603-8'>Spatial biology using single-cell mass spectrometry imaging and integrated microscopy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 10:01:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Spatial biology pursues the analysis of cells in their native microenvironment, often with regard to morphology and gene- or protein expression. The spatial analysis of lipid and metabolic profiles by matrix-assisted laser desorption/ionization mass spectrometry imaging (MALDI-MSI) can add another layer of molecular information. A seamless integration of MSI data with other (multimodal) methods at a single-cell level is often hampered by insufficient co-registration and resolution. Here, we introduce a MALDI-MSI based method that integrates in-source bright-field and fluorescence microscopy, allowing for a coupled (sub-)cellular investigation of the same sample in both modalities. Presented examples from cell culture and tissue analysis include the visualization of intracellular lipid distributions in macrophages during phagocytosis, and the heterogeneity of lipid profiles of tumor infiltrating neutrophils correlated to their individual microenvironments. The achieved combination of lipid profiling with morphological features and protein expression on the single-cell level constitutes a powerful method for cell biology. Two cells in a tissue or a cell culture are never exactly identical, even if they share the same genetic heritage and microenvironment. This heterogeneity can be a consequence of differences in age, ontogeny, or tissue imprinting, resulting in alterations of morphology, gene expression, molecular composition, and ultimately function. In living systems, individual cells are part of interdependent multicellular structures, i.e., tissues, and undergo a constant exchange within local regulatory networks as part of the almost inestimable complexity that is life. Thus, to characterize the identity and state of a cell, the description should follow a holistic approach and include information from a wide range of analytical techniques1. In recent years, single-cell RNAseq and spectral flow cytometry have emerged as valuable tools to profile cellular heterogeneity. Similarly, advances in single-cell proteomics provide information on the heterogeneity on the protein level, albeit with much lower throughput2,3. Under the umbrella of spatial biology, the emergence of spatial-omics technologies including deep visual proteomics, multiplexed immunohistochemistry and spatial RNAseq have added information on morphology and the spatial relationship of cells4,5,6,7. A complementary addition to these techniques is the analysis by high spatial resolution mass spectrometry imaging (MSI). Importantly, this method can augment information on active transcripts and protein expression with the resulting downstream production of lipids and further metabolites, as well as glycans and peptides8. The most widely used MSI approach is based on matrix-assisted laser desorption ionization (MALDI). Herein, finely focused laser pulses are applied to pre-defined pixel positions on the sample surface, resulting in spatially defined mass spectra that can be processed to reveal signal intensity maps for up to hundreds of analytes differentiated based on their molecular weight. To increase signal intensity for a wide range of analyte classes, the technique can be amended with laser postionization (MALDI-2)9. Typically, eukaryotic cells stretch over distances between 10 and 100 µm and display a wide variety in shape and form. Prominent sub-cellular structures like the nucleus or other large organelles, but also morphological features, show sizes between 3 and 10 µm. Their spatially resolved analysis, therefore, requires pixel sizes of no more than 1 × 1 µm². The current bench-mark figure on the pixel size of MALDI-MSI, however, is at about 5 × 5 µm². Less established experimental techniques dedicated to high resolving power, such as transmission-mode or inverse geometry (t-)MALDI-2-MSI and others, enables pixel sizes ≤ 1 × 1 µm² and can thus fulfill this criterion10,11,12,13,14,15,16. Recent applications of MALDI-2 and t-MALDI-2-MSI for single-cell mass spectrometry have revealed a strong molecular heterogeneity for lipids and metabolites within clonal populations of cells and seemingly homogeneous tissue regions, demonstrating its high potential for cell biology research17,18,19,20. To contextualize these metabolic phenotypes with existing knowledge on the single-cell level, however, MALDI-MSI has to be connected to prevalent microscopy techniques conducted on the same slide. To date, the general application of this contextualization on the same slide is strongly limited by three main factors: The spatial resolving power of MALDI-MSI, precise co-registration with optical modalities, and automation of data analysis. Especially in the context of tightly packed cellular systems, such as tissue, shortcomings of the first two factors lead to erroneous or ambiguous assignment of mass spectrometric information to individual cells. To date, this restricts the analysis on the single cell level directly from tissue to laborious manual annotation of comparably low numbers of cells, while automated analysis is limited to cellular neighborhoods rather than single cells17,18,21. To address these challenges, we here present the development of hard and software solutions on a prototype ion source coupled to an orthogonal time-of-flight (oTOF) mass analyzer. Our approach enables the generation of unambiguous cell-specific molecular and morphometric information of statistically relevant numbers of cells directly from tissue. It utilizes transmission mode for ultra-high spatial resolution and is equipped with laser postionization (MALDI-2) to increase sensitivity for the analysis of lipids and metabolites9,15,22,23. While previous implementations of t-MALDI ion sources have only hinted at the potential to utilize the employed optical components for microscopy of the sample inside the ion source15,24, we here make full use of this option by including in-source bright field (BF) as well as fluorescence microscopy (FM). A full technical description of the optical setup and preparation protocols is presented in the methods. In short, the ion source of a Bruker timsTOF-fleX MALDI-2 was amended for t-MALDI analysis. Dedicated lighting strategies enable the use of BF as well as FM inside the ion source (Fig. Importantly, laser delivery and in-source microscopy share essential components of the optical beam path and for stage movement. In the xy-plane (sample surface), the site of laser ablation is therefore directly linked to a specific position on the optical microscope image. Different from other instrumentation that combines MALDI-MSI with microscopy in one instrument25,26, this design feature inherently co-registers both modalities by utilizing the same coordinate system for in-source microscopy and MALDI-MSI (Fig. Next to providing highly resolved orientation to set up imaging runs, FM-images acquired inside the ion source constitute a unique and essential link for precise co-registration to other optical modalities. In the z-direction, the inherent coupling of the focal plane of optical microscopy and laser delivery enables a fast and easy setup of experiments based on autofocusing routines. a A schematic drawing of the t-MALDI-2 setup illustrates the general layout. The sample is illuminated homogeneously by an LED ring. Light is collected by the microscope objective and focused onto the sensor of a digital camera. The MALDI laser-beam is co-aligned using a dichroic mirror and focused onto the sample by the same objective. Ions generated by the MALDI process with optional MALDI-2 are extracted to a mass analyzer. b A horizontal cut through the schematic displays the fluorescence microscope components. Fluorescence is excited by a fiber-coupled LED and filtered by an excitation filter, a dichroic mirror and an emission filter. ((a) and (b) are renderings created by the authors in Autodesk Fusion 360.) c To enable fluorescence microscopy and t-MALDI-2-MSI on the same sample, different staining steps are performed following a brief fixation. Samples are imaged with an external slide scanning fluorescence microscope, and are then prepared for MALDI-MSI by homogeneous coating with MALDI matrix (Supplementary Fig. To enable high fidelity co-registration of the fluorescence microscopy image, a second fluorescence image is recorded inside the ion source prior to t-MALDI-2-MSI (Supplementary Fig. To enable FM analysis prior to MSI analysis, we have developed dedicated staining protocols optimized to omit or strongly reduce chemical alterations and spatial delocalization of the analyte content of interest. In parts these are based on protocols developed for the analysis of cell culture17,20. To control for the adverse effect of immunofluorescence (IF) staining on overall performance reported in the literature27, the speficially developed methods and pipelines were benchmarked with regard to microscopic image quality, conservation of chemical and morphological integrity, as well as mass spectrometric depth of information. For this, we conducted combined FM and t-MALDI-2-MSI on coronal cryo-sections of fresh-frozen mouse cerebellum. This tissue exhibits histological features across different lateral scales, each presenting distinct lipid profiles. It has been intensely used as a model system across a wide range of microscopic and MSI methods28,29,30,31 and is well characterized with regard to its lipidome based on the analysis of lipid extracts32. Details on sample preparation and data acquisition are described in the methods. To inspect and demonstrate the quality of our dedicated staining-based FM analysis on the same tissue section prior to t-MALDI-MSI analysis, we employed small molecule stains, targeting the nuclei and F-actin, as well as IF to target calbindin expressed in Purkinje cell bodies (Fig. Slide scanning FM recorded prior to matrix application in an external microscope (VS200, Evident) at 50X displays the expected structures and morphology for all three staining methods in high quality and corroborates the feasibility of the employed protocols (Fig. a An external fluorescence image of an 8 µm thick section of a mouse cerebellum recorded prior to matrix application using Hoechst 33342 (nuclei, DAPI channel, blue), CellMask Green Actin Tracking Stain (actin, FITC channel, green) and Alexa Fluor 594 Anti-calbindin antibody (calbindin, TxRed channel, red). The different layers of the brain (molecular layer, granular layer, and arbor vitae), as well as the Purkinje cells are well distinguishable. b The external FM image is overlayed with the ion signal intensity distribution of [PC(40:6) + H]+ at m/z 834.60 (gray) revealing a successful co-registration of both modalities. c The overlay of three ion signal intensities of the molecular ion of Hoechst 33342 at m/z 452.23 (cyan), [PC(38:6) + H]+ at m/z 806.57 (magenta) and [PC(40:6) + H]+ at m/z 834.60 (yellow), affirms the ability to produce images of high contrast while preserving and reproducing the biological structure of the sample. After matrix application by resublimation, the sample was transferred into the MALDI ion source, followed by in-source slide scanning FM (50X, DAPI channel). Utilizing the FM image of the DAPI channel generated inside the ion source as an intermediate enables a straightforward co-registration of external FM images with MALDI-MSI (Supplementary Fig. In this, our approach omits the use of fiducial markers across modalities, relieving an inherent limitation of conventional methods. Difficulties in the identification of these “cross-modality” markers that need to represent the exact same spatial location in conventional methods have been described to limit the precision and spatial fidelity of co-registration of FM and MSI on a cellular or sub-cellular level, especially for large areas of interest21,33,34. Subsequently, t-MALDI-2-MSI data at a pixel size of 1 × 1 µm² were acquired in positive ion mode. Equivalent data in the negative ion mode was recorded on a neighboring section. Facilitated by the choice of matrix and the use of MALDI-2, mass spectra in the positive mode produce more depth of information, and this ion mode was used for all further experiments (Supplementary Table 1). Overview and zoomed in regions of the resulting MS images display distinct intensity distributions demonstrated on the example of ion signals of two different phosphatidylcholines (PC) ([PC(38:6) + H]+ at m/z 806.57 and [PC(40:6) + H]+ at m/z 834.60) and the molecular ion signal of the Hoechst stain of the nuclei at m/z 452.23 (Fig. Direct comparison of t-MALDI-2-MSI with microscopy confirms that morphological features at the different lateral scales are largely retained during the matrix application step and subsequent MSI analysis (Fig. The overlay of both modalities reveals a strong correlation with histological features of the white matter and granular layer, discernible based on the F-actin staining and the nuclei. Similarly, signal intensity distribution of PC(40:6), associated with the Purkinje cell layer35 shows good correlation with the IF-based specific staining of these structures (Fig. Overall, deviations for co-registration were found to be less than 1 µm (Supplementary Fig. 1b, c and Supplementary Note 1), corroborating the precision and fidelity of our approach. To explore the depth of information for lipid analysis available from pre-stained t-MALDI-2-MSI, mass spectral information was matched against data from an unstained tissue section of the same brain. Overall, the staining protocol does not introduce a decrease in depth of lipid information with regard to the tentatively assigned lipids species (Supplementary Table 2, Supplementary Fig. While alkali metal cations of lipids dominate the spectra of the naïve tissue sections staining, washing steps that are part of the staining protocols shift the ionization towards protonation (Supplementary Fig. Interestingly, only the naïve tissue produces sizeable signal intensities of phosphatidic acids. Their appearance has been associated with in-source fragmentation of alkali metal cations of other phospholipids36. In addition, lipid information from t-MALDI-2-MSI was compared to a previously published full lipidomics profile for murine cerebellum derived from bulk32. This analysis reveals that ~82% of the reported molecular lipid content of cerebellum is detected in the t-MALDI-2 measurement (Supplementary Note 2). This amounts to 66 annotated ion species that produce contrast-rich intensity distributions in t-MALDI-2-MSI in positive and/or negative ion mode measurements (Supplementary Table 1). Comparison of the detected signal intensities with the molar content of the different lipoforms of each of the detected lipid classes shows a good agreement with the literature32 (Supplementary Fig. Notably, this includes lipid classes, such as phosphatidylethanolamine (PE) and phosphatidylserine (PS) that have been described to be affected if more extensive chemical fixation is used during sample preparation37, corroborating the aptitude of our speficially developed protocols. To demonstrate the potential of histology guided t-MALDI-2-MSI for the analysis of sub-cellular structures, we investigated THP-1-derived macrophages. Cells were cultured and differentiated directly in chamber slides. Freshly differentiated cells were incubated with pHrodo red E. coli bioparticles for 2 h. Upon phagocytosis, these particles become fluorescent and pinpoint acidified phagolysosomes inside the cells. For analysis, cells were briefly fixed and their actin filament and nuclei stained following MALDI compatible protocols previously reported17. Slide scanning FM on an external slide scanner prior to coating with MALDI matrix reveals heterogeneously pronounced phagocytosis with some cells showing a large number of internalized bioparticles (Fig. a Fluorescence image of THP-1 derived macrophages recorded before MALDI matrix application with a slide scanning microscope using Hoechst 33342 (nuclei, DAPI channel, blue), CellMask Green Actin Tracking Stain (actin, FITC channel, green) and pHrodo (phagolysosomes, Cy3 channel, red). b An overlay of three different ion signal intensity distributions measured using t-MALDI-2-MSI with 1.5 × 1.5 µm² pixel size in positive ion mode of the molecular ion of Hoechst 33342 (m/z 452.23, blue), [PE(33:1) + H]+ (m/z 704.52, red) and [PC(34:1) + H]+ (m/z 760.58, green) reveals a strong cell-to-cell heterogeneity. c and d Zoom in from (a) and (b), marked in each as a blue and red square, respectively, demonstrates intracellular differences in lipid distribution. To investigate the intracellular molecular distributions related to phagocytosis, masks for the regions with pHrodo+ (red) and remaining cell (green) are created based on the FM image. After coating with MALDI matrix using resublimation, in-source FM (50X) and t-MALDI-2-MSI data at 1.5 × 1.5 µm² pixel size in positive ion mode were collected for a few hundred cells (Fig. Image co-registration between the externally generated FM image and MSI data was performed by using the in-source FM data as an intermediate (Supplementary Fig. The overlay of both modalities corroborates a high-fidelity co-registration on the level of the employed MALDI pixel size. Areas visualized by the pHrodo red stain in FM accurately co-align with the distribution of specific lipid signal intensities in MSI, revealing little to no leakage or diffusion of lipid content or other preparation artifacts that may compromise (sub-)cellular integrity (Fig. To investigate the molecular distributions specifically related to phagolysosomes, we used subcellular spatial information provided by the fluorescence signal produced by the uptake of pHrodo bioparticles. Utilizing the high-fidelity co-registration, we overlayed this histological information to t-MALDI-2-MSI data and identified those pixels that correlate with phagolysosomes to generate phagolysosome-specific mass spectra. Lipid annotation was based on previously published results for THP-1 derived macrophages and E. coli, respectively20,38. Comparative analysis of the resulting phagolysosome-specific molecular information revealed an increase of signal intensity for numerous lipid species (Fig. Confirmed by the respective MS images, lipids, such as PE(33:1) and PE(34:2) spatially correlate with the phagolysosomes identified by fluorescence (Fig. Both PE species are abundantly expressed in the membrane of E. coli cells38. Notably, PE(33:1) is not detected in non-phagocytosing THP-1 cells at all (Fig. Its presence, confirmed by additional on-cell MS/MS experiments (described in Methods), provides a direct link to the phagocytized pathogen material and corroborates the highly accurate co-registration of both modalities. Comparative analysis of the resulting region-specific mass spectral data from masks as shown in Fig. 3c,d reveal an increase of signal intensity for a number of lipid species inside the pHrodo+ regions. Signals with a log2 fold change above 2.8 are marked in red. Identified m/z-values are denoted in a table with tentative assignments, where applicable. Other tentatively assigned lipid species linked to the phagolysosomes can be attributed to degradation of E. coli-derived phospholipids during phagocytosis39,40. These degradation products are most likely the result of enzymatic activity and reactions with reactive oxygen species inside the phagolysosomes. They include lipid metabolites, such as the diacylglycerol DG(33:1), which can be directly linked to the enzymatic cleavage of PE(33:1) and lipid species tentatively identified as lysophosphatidylglycerols (LPG). The latter can be linked to the enzymatic breakdown of phosphatidylglycerols (PG) and cardiolipins (CL) both highly expressed in the membrane of E.coli 38,40 (Fig. In a second exemplary study, we applied our methods to the single-cell molecular analysis of tumor tissue. This tissue type typically contains a large number of different cell types in a wide range of states, translating to a wide variety of metabolic microenvironments (tumor microenvironment (TME)) of different size and shape and a highly heterogeneous lipid profile across the tissue41. The employed 4T1 mouse mammary carcinoma model is highly malignant with locally aggressive growth and exhibits metastasis to regional lymph nodes and distant organs, particularly lung, liver, and bone42,43. It has diverse inflammatory TMEs with a large number of infiltrating immune cells, such as T-cells, macrophages, neutrophils, and natural killer cells present44,45. The site and cell-specific molecular profile of these immune cells is of great importance for their fate and function and is believed to be in constant interdependence with their microenvironment46. Consequently, the investigation of this crosstalk not only requires a molecular analysis at the cellular level but also an assessment of the immediate vicinity. For analysis, a heterogeneous area of interest that includes a broad range of TMEs was selected based on H&E staining and histological annotation (Fig. A consecutive section was prepared using MALDI-compatible staining that targets cell nuclei, F-actin, and three IF stains that selectively target immune cells, (CD45), neutrophils (Ly6G), and T3 neutrophils (DcTRAIL-R1)46,47. All channels were recorded on the complete section using external slide scanning FM at 50X before matrix application (Fig. Co-registration with t-MALDI-2-MSI was based on the DAPI channel recorded inside the ion source after matrix application as an intermediate (Supplementary Fig. After microscopy, t-MALDI-2-MSI data was recorded in positive ion mode in seven separate measurements across the region of interest at a pixel size of 1 × 1 µm² resulting in a total of approx. For data analysis, we selected 63 molecular ion species annotated to individual lipid species based on exact mass ( < 3 ppm) (see Supplementary Fig. 4 for selected images and Supplementary Table 3). a Annotation of an H&E stain of a 4T1 tumor section; ROI of the t-MALDI-2-MSI conducted on a consecutive section marked in blue. b FM using Hoechst 33342 (nuclei, DAPI channel, blue), CellMask Green Actin Tracking Stain (actin, FITC channel, gray), IF stains for neutrophil cells (Ly6G, TxRed channel, red), immune cells (CD45, Cy5 channel, yellow), and T3 neutrophils (DcTRAIL-R1, Cy7 channel, not pictured in (b)) prior to MALDI matrix application. c t-MALDI-2-MSI overlays in positive ion mode of acylcarnitine [ACAR 18:0 + H]+ (m/z 428.38, yellow), [PC(O 32:0) + H]+ (m/z 720.59, magenta), [PC(34:1) + H]+ (m/z 760.58, cyan) and [TG 52:4 + H]+ (m/z 855.75, blue) corroborating histological annotations from (a), (b). d Zoom-in of blue squares (b) and (c). FM shows Hoechst (blue), CellMask (gray), neutrophils (red) and immune cells (yellow). e Zoom-in of red squares in (b) and (c). FM shows neutrophils (red), immune cells (yellow) and T3 neutrophils (cyan). f Zoom-in of green squares in (b) and (c). FM shows Hoechst (blue), CellMask (gray), neutrophils (red) and immune cells (yellow). MSI overlays: Molecular ion of Heme B (m/z 616.18, yellow), [PC(O-32:0) + H]+ (m/z 720.59, magenta) and [SM(34:1;O2) + H]+ (m/z 703.57, cyan). g 63,408 cell masks based on the FM image inside the complete ROI. Zooms show cell masks overlaid with FM and MSI. For MSI ion signal intensity of [PC(O 32:0) + H]+ at m/z 720.59 selected exemplarily. Spatial distribution for a number of glycero- and glycerophospholipids such as [PC(34:1) + H]+ at m/z 760.58 reveal a sizeable heterogeneity across the tumor sample (Fig. Signal intensity of the ether lipid [PC(O-32:0) + H]+, detected at m/z 720.59, correlates with the neutrophil-specific Ly6G-stain (Fig. This is consistent with a significant overexpression of this lipid species in mouse neutrophils reported previously48,49. Other specific signal intensity distributions also directly correlate with structures visible in the microscopy data. As expected50, signal intensity for triacylglycerides (TG) such as [TG(52:5) + H]+ correlates with the adipose tissue (Fig. Acylcarnitine, detected at m/z 428.38 as [ACAR(18:0) + H]+ is an important intermediate in lipid metabolism and a reported marker for hypoxia51. Its signal intensity distribution correlates with the detection of tumor-associated T3 neutrophils identified by the DcTRAIL-R1 staining that have been described to be found in hypoxic areas46,47 (Fig. Heme B, detected at m/z 616.18, is directly associated with erythrocytes and, hence, serves as marker for hemorrhage or blood vessels52 (Fig. While general assignment of lipid profiles to TMEs has been reported previously, the approach described here allows for the analysis of single cells within their tissue environment in statistically relevant numbers. For this we adapted our previously reported workflow for the analysis of single cells in culture to the analysis of tissue sections17,20. Robust and reliable cell segmentation, necessary for automated analysis21, was provided based on the external FM images using Mesmer by DeepCell (Fig. This resulted in segmentation masks for each cell and enabled the calculation of a variety of morphometric parameters using CellProfiler54. To produce single-cell mass spectra, cell segmentation masks were projected on the t-MALDI-2-MSI data, again taking advantage of the inherent link between in-source FM and the acquired MSI data (Supplementary Fig. Of note, already small inaccuracies of the co-registration on the order of a few micrometer as well as increased pixel size can lead to larger ambiguities in the assignment of MS data to specific single cells17,21. An overlay of the projected masks with the lipid neutrophil marker [PC(O-32:0) + H]+ and comparison to the specific Ly6G staining demonstrates the high-fidelity co-registration and corroborates the ability of our combined method to deduce single-cell mass spectra (Fig. Single-cell molecular data is generated for all 63 annotated lipid species by summing the respective signal intensities over all pixels allocated to a specific cell using the respective mask. Subsequently, all values are normalized to the sum of all signal intensities for the respective cell. Cell segmentation and subsequent data analysis resulted in combined morphometric and molecular data for 63,408 individual cells. Notably, this analysis retains the histological context of all cells and includes parameters describing location within the tissue and fluorescence intensity for all employed channels. This enables the identification of CD45+, Ly6G+, and DcTRAIL-R1+ cells based on intensity thresholding for each specific channel. For visualization, this information can be projected onto the segmentation canvas, thereby identifying the location and local density of different sub-groups of immune cells in the tissue section (Fig. a Cell masks colored corresponding to their FM-based classification are projected onto their original position of the t-MALDI-2-MSI measurement region. All other cells are marked in dark gray. b The 59,998 Ly6G- cells were separated into eight different clusters (tissue microenvironments) using k-means clustering on the standardized single-cell MS data. Cell masks for Ly6G- cells are colored according to their cluster and projected onto their original position of the t-MALDI-2-MSI measurement region. All other cells are marked in dark gray. c 3410 Ly6G+ cells were identified based on FM and selected for analysis. They are plotted in a UMAP based on a list of 63 annotated ion signals. Each dot represents an individual cell, while the distance between dots visualizes the similarity of the standardized lipid profiles of the cells. Using DBSCAN, the UMAP reveals eight clearly discernable clusters (neutrophil subtypes) which are color coded. d Cell masks for Ly6G+ cells are projected onto their original position of the t-MALDI-2-MSI measurement region. All other cells are marked dark gray. The colors of cell masks correlates with the clusters found in (c). e Zoom-in of the region annotated with lymphocyte accumulation (red squares in a, b, and d). f Zoom-in of the region annotated with intratumoral hemorrhage (blue squares in a, b, and d). Processing of the combined IF and MSI single-cell data enabled spatial biology analysis on the level of lipids and metabolites, by classifying cells based on molecular similarity. In this context, analysis on the single-cell basis is especially important for scarce or isolated cells like specific immune cells because their molecular profile may substantially differ from the surrounding TME. This type of analysis is demonstrated on the example of infiltrating neutrophils inside the 4T1 tumor section. To classify neutrophils based on their molecular profile independent of their microenvironment and vice versa, single-cell data were split into two groups based on Ly6G staining. The resulting subgroups contained 3410 Ly6G+ neutrophils and 59,998 Ly6G- cells, respectively. Both groups were independently analyzed using Uniform Manifold Approximation and Projection (UMAP) and appropriate clustering algorithms. To identify molecularly similar cell types within the tissue, all Ly6G- cells were subjected to k-means clustering, resulting in eight discernible clusters. By assigning a specific color to each cluster, the outlines of each cell can be filled in with the respective color and projected back onto their original location (Fig. The resulting map of classified cells displays clearly discernible tissue micro-environments where molecularly similar cells cluster in specific areas of the tissue. Clusters IV, V, and VIII contain cells of the skin, the adipose tissue, and the wider tumor periphery, respectively. Inside the viable tumor tissue, a clear separation into TMEs reveals a layer-like structure with frayed edges (clusters II, III, and VI). Cluster VII contains cells directly adjacent to the tumor edges and away from the adipose tissue, but also cells inside the tumor close to its necrotic core. Cluster I contains cells allocated to hypoxic regions of the tumor. The direct connection to the morphometric data allows to characterize each cluster with regard to immune infiltration. Figure 7, summarizes the total amount of cells as well as the numbers of CD45+, Ly6G+, and DcTRAIL-R1+ cells that are found within the boundaries of the respective cluster. Data matrix tabulating the number of neutrophil subtypes found in each tissue region. For each subtype, the percentage of DcTRAIL-R1+ cells is calculated. In a neighborhood analysis for all neutrophils, the mean number of cells categorized as CD45+ cells, Ly6G+ cells and DcTRAIL-R1+ cells found in a 15 µm radius are presented. Additionally, for each microenvironment, the total number of contained cells, CD45+, Ly6G+ and DcTRAIL-R1+ is noted. Next to TMEs, we investigated the molecular heterogeneity of neutrophils. For this, the Ly6G+ cells were analyzed using a UMAP and classified using Density-Based Spatial Clustering of Applications with Noise (DBSCAN) (Fig. This results in a clear separation of eight clusters. To provide linguistic distinction from clusters described for the tissue microenvironments, results for neutrophils will be referred to as subtypes. Colored respectively and projected onto the cell map, these different subtypes can be found in distinct areas of the tumor (Fig. Apparent from the comparison of the images depicted in Fig. 6, some subtypes show a clear spatial correlation with a specific microenvironment and histological features (Fig. Subtype 6 is almost exclusively located in the skin. Neutrophils are found in skin in comparably high number and have important and specific barrier functions55. Our data suggests a significantly altered lipid profile for these neutrophils. For example, in contrast to all other subtypes, subtype 6 shows a low expression of PC(O-32:0) (Supplementary Fig. Neutrophils of subtypes 5 and 7 are located in specific tissue microenvironments that correlate with the adipose tissue (clusters IV and V) in the periphery of the tumor. Their lipid profiles show an increase of TG and DG lipid species (Fig. Subtype 4 is predominantly found in non-adipose tissue close to the tumor (cluster VII). Similarly, subtype 2 also populates this peripheral region but is additionally found inside the tumor close to its necrotic core. Lipid profiles for both subtypes are low in DG and TG. While their overall cell density in this area is relatively high, they do not form agglomerates. A heat map of standardized intensities of 63 ion species for the eight neutrophil subtypes visualizes differences in lipid profiles. Interestingly, neutrophils in this periphery of the tumor show strong molecular similarities to their respective microenvironment (Supplementary Fig. This may be explained by an uptake of extracellular vesicles or cell debris produced by the surrounding tissue and “recycling” of the lipid content by the neutrophils. Notably, the expression of lipid species that require specific enzymatic machinery for their synthesis such as PC ether lipids (PC-O) do not show similarities to the surrounding tissue but are upregulated only in the respective neutrophil populations (Supplementary Fig. The exclusive co-localization of these ion species to Ly6G+-cells also precludes the involvement of measurement artefacts, such as analyte diffusion into this convergence of lipid profiles between neutrophils and their respective TME (Fig. With the exception of subtype 4, all neutrophils found in the tumor periphery and the adipose tissue show a relatively high density of immune cells in their direct vicinity (Fig. This observation is corroborated by a neighborhood analysis that counts the average number of immune cells within a distance of 15 µm to the cell edge of each individual neutrophil (Fig. Inside the tumor, neutrophils of the subtype 1 are found mostly in the viable tumor tissue (clusters II, III, and V) (Fig. Notably, their lipid profile shows little similarity to their respective surrounding TMEs (Supplementary Fig. This may indicate a decreased up-take and turnover of lipids or a reduced residence time within the tumor tissue. While the overall density of subtype 1 neutrophils in the tumor increases towards the adjacent adipose tissue, they are mostly isolated with relatively few other immune cells around them (Fig. In contrast, neutrophils of subtypes 3 and 8 form strong agglomerations (Fig. These subtypes are located within TME I with subtype 8 being confined to the center of the region. Their neighborhood analysis results in high numbers of overall immune cells in their direct vicinity (Fig. About 90% of subtype 8 neutrophils are identified as DcTRAIL+. The lipid profile of this subtype strongly differs from that of the surrounding hypoxic TME I (Supplementary Fig. It presents an increased signal for ACAR 18:0 and an increase in signal intensities for lipid metabolites, such as LPCs (Fig. 8) and DGs, while TGs are mostly absent. This may indicate an upregulation of specific lipid metabolism in this subtype. Similar to subtype 8, neutrophils of the subtype 3 are almost exclusively found in agglomerates inside TME I but are located further away from its center. In contrast to subtype 8, however, only 17% are DcTRAIL-R1+. The lipid profile of subtype 3 also contains high signal intensities for ACAR 18:0 but reduced levels of LPC and DG. Again, in contrast to subtype 8, the lipid profile of subtype 3 neutrophils shows great similarities to its surrounding TME I. This increased molecular crosstalk may indicate a longer residence time of neutrophils within this hypoxic TME as opposed to the viable TMEs II, III, and VI. In its own right, MSI analysis can provide specific molecular information of single cells, thereby delivering an important puzzle piece towards the understanding of individual cellular behavior. For a more holistic approach, however, the resulting lipidomic or metabolomic information has to be put into the context of the cell type, differentiation status, and state, but also has to regard its current tissue macro- and microenvironment¹. Here, a tight connection to optical microscopy analysis has long since been identified as a key feature in the integration of MALDI-MSI into the “biomedical toolbox”56,57. In this context, high-quality microscopy analysis, including IF, is typically carried out on adjacent tissue sections, introducing well-known challenges in the accuracy of co-registration. Because of their small size in the range of section thickness, this particularly limits the analysis of specifically targeted single cells33,56,58,59. For the microscopic analysis of the same section, staining-based microscopy is conventionally carried out after MALDI-MSI. While impressive progress has been reported very recently in this field, damage or distortion of the tissue introduced during the measurement and the removal of the matrix can impede accurate co-registration on the cellular level, especially for large areas of interest. In addition, cross-reactions with the matrix and laser irradiation that precedes microscopy, can impede binding properties for antibody based analysis12,60,61,62,63. Successful microscopy of the same sample prior to MALDI-MSI has largely been restricted to the limited analytical capabilities offered by unstained tissue, e.g. by autofluorescence or phase-contrast microscopy27,33,64,65. Alternatively, other non-destructive optical methods, such as IR-imaging and micro-Raman imaging, have been employed prior to MSI analysis but have not been utilized for automated cell segmentation66,67. Exceeding these current limitations, the technique presented here includes slide scanning microscopy outside and inside the ion source of the same sample recorded prior to ultra-high resolving power t-MALDI-2-MSI without compromising performance in either modality. Of note this includes the successful implementation of pre-MALDI IF analysis. Dedicated staining techniques allow for a robust and automated cell segmentation and retain the full spectral information in MSI. Uniquely, the high fidelity co-registration of both modalities for a large field of view facilitated by in-source FM that omits the need of “cross-modality” fiducial markers, allows for the automated generation of single-cell mass spectra from tissue in large numbers. While previous efforts that relied on manual steps in co-registration and definition of cellular ROIs21,68, this automation lays the groundwork for the reliable and large-scale, generation of information-rich highly resolved data for both modalities on the same tissue sample. Two exploratory studies highlight the advantages of the described method for the molecular analysis with cellular and even sub-cellular resolution. Using the phagocytosis of E. coli bioparticles by macrophages as an example, we demonstrate how the technique can be used to investigate metabolic processes inside sub-cellular compartments like the phagolysosomes. Assignment of bacterial PE 33:1 and PE 34:2 confirmed the uptake of E.coli bioparticles. The additional identification of increased amounts of the respective DG species suggests their partial degradation. In addition, the tentative identification of several LPG species provides further useful leads to the metabolic breakdown of E. coli derived CL and PG by enzymatic degradation and/or oxidation of lipids inside the phagolysosome69. Overall, this study illustrates a dedicated approach to analyzing molecular profiles of subcellular compartments. In future applications, this largely untargeted technique may be used to investigate and monitor metabolic turnover on the organelle level of specific single cells. In our study of the 4T1 tumor model, we demonstrate spatial biology analysis on the lipid level for highly heterogeneous tissue. While previous work on single-cell MSI from tissue sections has been limited by comparatively large pixel size and by the manual annotation of small numbers of cells17,18,19,70, automated segmentation based on slide scanning (immuno-)FM resulted in the identification of more than 63 K cells and allowed for the identification and classification of common immune cells (CD45+, ~ 23.7 K cells; 38% of all cells), relatively rare neutrophils (Ly6G+, ~ 3.4 K cells; 5.4%), and very rare tumor associated T3-neutrophils (DcTRAIL-R1+, 276 cells; 0.4%) within the tissue. Importantly, the segmentation, in combination with highly accurate co-registration of the microscopy with t-MALDI-2-MSI results, allowed for the generation of single-cell mass spectra for each cell. Using statistical analysis, cells were clustered based on their lipid profile, revealing eight distinct tissue microenvironments inside the tumor and around its periphery. Likewise, an independent clustering of just the Ly6G+ cells revealed eight subtypes of neutrophils inside the investigated tissue region. Comparison of lipid profiles between neutrophils and their direct tissue environment revealed striking similarities in some cases. These results allow for a new perspective on the uptake and recycling of lipids by specific neutrophil subtypes. They also reveal distinctive differences between neutrophils and their direct environment, such as the increased levels of ether lipid species, also described in the literature49. In addition to a general upregulation of these lipid species, we are able to identify subtype and TME-specific differences in their expression. Neighborhood analysis of all investigated immune cells in the direct vicinity of the neutrophils provides a first hint at their swarming behavior. While some of the subtypes represent relatively isolated cells, others are found in agglomeration with neutrophils and other immune cells. This TME shows a signal intensity increase for ACAR 18:0, as well as a high density of immune cells, both described as markers for hypoxia51. The inner core of this TME is populated by densely packed T3-neutrophils (most of them of subtype 8), concurring with the literature46,47. A second group of agglomerated mostly DcTRAIL-R1- neutrophils (subtype 3) is also located inside the same hypoxic area but further away from its center. The subtyping now enables an in-depth analysis of underlying lipid profiles and integration with the state of research. While both subtypes, for example, express increased levels of ACAR 18:0, only subtype 8 (mainly DcTRAIL-R1+ cells) shows increased levels of fully saturated LPCs. These mediator are known to be important in metastasis formation and in the recruitment of phagocytes during apoptosis71,72 and our data points to a previously unknown tentative connection with T3-neutrophils. While a full investigation of the underlying biological mechanisms is not within the focus of this publication, the presented proof-of-concept studies showcase the great potential of the technique. The seamless integration of morphometric data with (sub-)cellular lipid and possibly metabolite analysis significantly expands the informational space available for each individual cell inside the tissue. Exceeding the current state-of-the art that is largely limited to the analysis of cellular neighborhoods, our approach enables a direct in situ contextualization of single individual cells with their direct tissue microenvironment on the combined molecular and morphometric level. The presented techniques introduces a versatile set of tools for an improved integration of single-cell MSI into existing workflows. Embedded within a full suite of spatial biology methods, in future applications, it will allow for the combination of IF-based information that may describe the regulation of specific genes and the expression of individual proteins with the expression of lipids and possibly also metabolites in the same cell and its direct surrounding73. This addition of “spatial biology of lipids and metabolites” on the single-cell level provides an important additional layer of information to complete pathway analysis from the transcript and proteome level down to the regulation of the molecular products. It can be a valuable asset to review and validate downstream effects of genetic or transcriptomic variation, as well as protein inhibition, but also help to investigate regulatory functions of small molecules on gene and protein expression. Future developments in t-MALDI-MSI will need to focus on further increasing the sensitivity and analytical depth of the method. E.g., innovative approaches to sample preparation combined with plasma-based ionization approaches, as described by Young et al. in the same issue of this journal, can help to increase sensitivity and assist in the analysis of additional classes of metabolites74. Overall, the methods described here have great potential to help untangle the influence of endogenous and exogenous factors such as immune cell infiltration, infection, or drug delivery on cellular behavior in a specially resolved manner. They are, however, by no means limited to these applications but can be applied to decipher open biological questions in a wide range of tissues and cell types. All reagents, resources, and software used are listed in the table of resources (Supplementary Table 4) THP-1 ACC-16 cells (DSMZ) were cultured in T25 cell culture flasks at 37 °C and 5% CO2 in RPMI 1640 Medium (Lonza), which was supplemented with 2 mM L-glutamine, 10% fetal bovine serum and 1 mM sodium pyruvate. About 10,000 cells per chamber were sown in Millicell EZ Slide 8-well chamber slides (Sigma-Aldrich). Differentiation to M0 macrophages was induced by exposure to 20 ng/ml phorbol 12-myristate 13-acetate (PMA, Sigma-Aldrich) for 24 h followed by a 3-day rest phase in PMA-free medium. Two hours before harvest, pHrodo™ Red E. coli BioParticles (Thermo Fisher) were added to the culture medium resulting in a concentration of 95 µg/mL. Cells were washed with PBS and fixed for 5 min using 4% formaldehyde (low-methanol, Roth). F-Actin and nuclei were stained according to a protocol described by Bien et al. using 1X CellMask Green Actin Tracking Stain (Thermo Fisher) diluted in PBS for 30 min and 1 µg/mL Hoechst 33342 (Sigma-Aldrich) dissolved in PBS for 5 min17. Cells were washed with PBS after every staining step. After staining, cells were washed using 150 mM ammonium acetate (Sigma-Aldrich) and dried at room temperature for 2 h. All animal experiments were carried out in accordance with local animal welfare guidelines approved by the responsible authorities (North Rhine-Westphalia State Agency for Nature, Environment and Consumer Protection, LANUV, Protocol No. Female BALB/c mice (Charles River Laboratories) were used at the age of 8–12 weeks. Mice were housed under a 12 h light/dark cycle with ad libitum access to food and water. For tumor implantation, 106 4T1 cells were resuspended in 25 µl of DMEM cell culture medium (Thermo Fisher) and then injected orthotopically into the lower left mammary fat pad of the mice. Tumors were allowed to grow for 9 days and tumor size was monitored daily with a digital caliper. The local ethics committee approved a maximum tumor diameter of 8 mm. This limit was strictly adhered to and not exceeded in all experiments conducted within this study. On day 9 after tumor implantation, the animals were sacrificed. Brain tissue was harvested from untreated mice used as the control group for the mentioned study. Mouse brain and tumor tissue were embedded in Epredia M-1 Embedding Matrix (Thermo Fisher) and snap-frozen in liquid nitrogen. 10 µm (cerebellum) or 8 µm (tumor) thick tissue sections were produced with a cryostat (CM 3050 S, Leica Biosystems) at −20 °C. Sections were thaw-mounted onto SuperFrost glass slides (Thermo Fisher) for H&E-staining or IntelliSlides (Bruker) for t-MALDI-2 MSI measurements. Samples for t-MALDI-2 MSI were vacuum-sealed and stored at −80 °C before further use. Frozen tissue sections were slowly brought to room temperature in the vacuum-sealed package. After opening the seal, sections were further dried under a gentle nitrogen stream for 10 min. For unstained control tissue the section was directly transferred to the sublimation chamber. For staining, a contour was drawn around each section using a Super PAP Pen (Science Services), followed by another 10 min drying under a gentle nitrogen stream, to create a hydrophobic barrier and thus enabling pipetting aqueous solutions on and off the sample. The following washing, fixation, and staining steps were all performed by pipetting ~100 µl of the accordant solution onto the sample and removing it after the specified time. Slides with tissue sections were placed on a metal plate standing in ice water. Samples were quickly fixed in 4 % formaldehyde (low-methanol, Sigma-Aldrich) in PBS for 5 min and subsequently washed with PBS for 30 s. After tissue fixation, the primary antibody, Ly6G (ab307167, Abcam), DcTRAIL-R1 (130-110-873, Miltenyi Biotec), CD45 (130-110-798, Miltenyi Biotec) or Anti-Calbindin (ab229915, Abcam), diluted 1:500 (v/v) in 1% BSA (Serva) in PBS was applied to the sample and incubated for 60 min. The sample was then washed three times for 30 s in PBS. In case of a dual antibody staining, the secondary antibody, Alexa Fluor 594 (ab307167, Abcam), diluted 1:1000 (v/v) in 1% BSA in PBS was applied and incubated for 60 min, followed by three times 30 s washing steps in PBS. 5 µg/ml Hoechst 33342 (Sigma-Aldrich) and 1X CellMask Green Actin Tracking Stain (Thermo Fisher) diluted in PBS were applied and incubated for 5 min. After staining, the slide was lifted from the metal plate and parts of the hydrophobic barrier were removed at the bottom of the slide with a pipette tip. Samples were then washed with 2 ml of ammonium acetate (Sigma-Aldrich) each, by letting it flow carefully over the section. For final drying, samples were placed under a gentle N2 stream for 20 min. Hematoxylin and eosin (H&E) stains of tissue sections were performed on untreated consecutive sections directly after sectioning. Sections were stained for 30 s with hematoxylin (Sigma-Aldrich) with subsequent bluing in tap water for 2 min. Afterwards, the tissue was stained for 30 s with eosin (Sigma-Aldrich) and then rinsed with water. Fluorescence images of cell cultures and tissue sections were recorded at 50X using the DAPI, FITC, TxRed, Cy3, and Cy5 channels of a slide scanning microscope (VS200, Evident) equipped with a SpectraSplit 7 filterset (Kromnigon). For each channel individually, the exposure time was set by performing the auto exposure procedure of the microscope on a region with high signal intensity for the respective fluorophore. This procedure was conducted prior to the imaging run, and the entire slide was subsequently imaged with a constant exposure time, with a maximum exposure of 500 ms. H&E-stained tissues were recorded at 20X in BF mode. Image processing was performed with OlyVIA (Evident). Channel histograms were linearly adjusted for display. Channels are shown with linear LUTs (red, green, blue, cyan, magenta, yellow). For further information please refer to the light microscopy reporting table (Supplementary Table 5). A schematic of the optical set-up used for material ejection for the t-MALDI-2-MSI analysis and in-source brightfield and FM image generation is depicted in Fig. The sample is mounted on a three-axis piezo stage (SmarAct) and is illuminated homogeneously by 15 green LEDs (LED528EHP, Thorlabs) mounted on top of a 50X objective (M Plan Apo NUV HR 50X, Mitutoyo) with a custom-designed 3D printed LED ring. Light from the sample is collected by the objective and focused by a tube lens (TTL100-A, Thorlabs) onto the sensor of a digital camera (XCG-CG160, Sony). A simple top-view observation of the sample is in addition possible by the pre-installed camera of the prototype timsTOF fleX MALDI-2 mass spectrometer. The beam path of the MALDI laser (smartbeam, wavelength 355 nm, pulse duration 7 ns) of the timsTOF fleX MALDI-2 mass spectrometer is re-routed and condensed through a 6:1 telescope optic and co-aligned to the microscopy path with a dichroic mirror (L-07246, Laseroptik Garbsen). For recording FM images, a filter cube (DFM2/M, Thorlabs) with a filter set (DFM2T1, Thorlabs) for the suitable fluorescence channel (MDF-BFP, DAPI; MDF-FITC, FITC; MDF-MCHC, mCherry; all Thorlabs) consisting of excitation and emission filters and a dichroic mirror is mounted in front of the tube lens. In fluorescence mode, the LED ring is turned off and fluorescence is excited by the respective fiber-coupled LED (M385FP1, DAPI; M455F3, FITC; MINTF4, mCherry; all Thorlabs) mounted with a collimating optic directly to the filter cube. The trigger signal for image acquisition of the camera and fluorescence excitation is provided as a “position reached” event by the controller of the piezo stage and is handled by a digital delay pulse generator (9200 Sapphire Series, Quantum Composers). With this pulse generator, the fluorescence excitation time was set from 10 ms to 2 s per image depending on the sample and fluorescence channel. To achieve a reproducible and homogeneous coating of the sample slides with MALDI-matrix, a custom-designed resublimation chamber was developed. A schematic of this setup is shown in Supplementary Fig. Under atmospheric pressure conditions, the hot-side is preheated to 80 °C and 0.5 ml of 10 mg/ml CHCA MALDI-matrix (Sigma-Aldrich) in acetone solution is pipetted into the matrix-reservoir. The sample is then clamped onto the cold-side at room temperature and the whole device is sealed and evacuated. After reaching a final pressure of 10-2 mbar, the cold-side is first cooled to -18 °C, then the hot side is heated to 150 °C to start the sublimation and resublimation. After 60 min, the heating of the hot-side is turned off and the cold-side is heated to room temperature before venting the system to prevent condensation of ambient humidity on the sample. The sample is then put onto a room temperature metal block in a saturated atmosphere of 0.5% ethanol in water at 70 °C for 1.5 min for recrystallization. To demonstrate differences in contrast for in-source BF microscopy induced by the choice of matrix, cerebellum samples were coated with DHB and HABA MALDI-matrix (both Sigma-Aldrich) using the same settings as described above for CHCA (Supplementary Fig. The MALDI laser and microscope camera utilize the same objective for focusing and observation in the transmission-MALDI setup employed in this work. A schematic of the setup is depicted in Fig. After aligning the focal planes, all optics are kept at a constant position. The objective (M Plan Apo NUV HR 50X, Mitutoyo) has a small focal depth of 1.4 µm and samples have to be in focus during optical microscopy and MALDI-MSI. This can be achieved by methodically adjusting the z-position of the sample with regard to the objective while recording sequential optical microscope images at z-increments of 1 µm. For each of these images a Laplacian filter is applied and the variance of the result is calculated, which serves as a measure of image sharpness. The stage is then moved to the z-position corresponding the maximum sharpness image and a fine-scan with 0.1 µm z-distance is performed in the range of +/- 1 µm in the same way as before to find the z-position of highest sharpness at a submicron precision. This autofocus routine can be executed both BF and FM mode. The Python code of this program AutoFocusClass.py is provided in Data S1 in the supplementary information. The scale factor to convert pixel on the camera sensor to position of the stage was calculated by using a 1951 USAF Resolution Test Target (Thorlabs). For selection of a region of interest (ROI) for an optical microscope scan in the ion source, autofocusing points are placed manually on the sample. The ROI is defined as the smallest enveloping rectangle that contains all points. The stage is then moved to each of the autofocusing points and an autofocus routine as described in “Autofocus of optics” is performed, which yields the focal z-offset of that xy-coordinate. Using a radial basis function interpolation, the z-offset is calculated for every coordinate in a 10 x 10 µm mesh inside the ROI and saved to a z-interpolation file. For this an up to 100 µm x 100 µm wide x/y-position mesh is created in the ROI. An optical microscope image is then taken at each mesh point at the z-offset of the closest point to the xy-position from the z-interpolation file. To create a complete image of the ROI, an empty canvas of the correspondent size is created. Using the scale factor of the setup, images are cropped from the edges to a size of 100 x 100 µm and are placed onto the empty canvas using the center coordinates from the xy-position mesh. After creating a z-interpolation file for a ROI once, multiple modalities, such as brightfield or different fluorescence channels can be recorded without creating a new interpolation. ImageJ was used to create overlay images of different modalities. The Python code of this workflow can be found in the files RecordPositions.py, ZforRecordPositions.py, Scanning.py and stitching.py in Data S1 in the supplementary information. All experiments were carried out in positive ion mode with the exception of one measurement from a brain tissue section. The MALDI-MSI run is set up partly using the proprietary flexImaging software (Bruker). Instead of using an external scan of the sample and teaching that to the stage coordinate system, the (multimodal) image of the ROI (see “In-source microscopy workflow”) is used. In the classic flexImaging workflow, three teaching points would be manually selected on the optical image. The corresponding stage coordinates would then be recorded by driving the sample-stage manually to the respective positions. Since center coordinates and pixel positions on the ROI image are already established from recording the ROI image, three x/y-positions from the 100 x 100 µm mesh (top left, bottom left, bottom right) and their corresponding stage coordinates are automatically deployed as teaching points. With these points, the setup file for the t-MALDI-2 MSI run can be automatically created, defeating any positioning errors introduced by manually selecting teaching points in the classic workflow. When the MALDI-laser is not directed at the exact middle pixel of the in-source optical microscope, fine adjustments of a few pixels offset can be applied at this stage. The automatically created setup file for the t-MALDI-2 MSI run can be opened directly in flexImaging to create measurement regions, select settings like pixel size, and laser shots per pixel. The previously acquired z-interpolation is recognized by timsControl software during t-MALDI-2 MSI measurements by providing the interpolation mesh as a geometry file. The Python code for creating the flexImaging setup files can be found in Createmis.py Data S1 in the supplementary information. All t-MALDI-2 MSI experiments were performed with 25 laser shots per pixel and a pixel size of 1 × 1 µm2 for the cerebellum and tumor sample and 1.5 × 1.5 µm2 for cell culture samples. Signal intensity distributions are visualized using SCiLS Lab software (Bruker) using weak de-noising. For the THP-1 cells lipid signals were identified as described by Schwenzfeier et al. using shotgun lipidomics by electrospray ionization (ESI) mass spectrometry on a Q-Exactive Plus orbitrap mass spectrometer (Thermo Fisher) equipped with a dual ion funnel20,75. Lipid extracts of 72 h PMA stimulated THP-1 macrophages were produced by an adapted protocol of Folch et al. 76. Approximately 106 cells were centrifuged at 1000 g to separate them from the medium. Cells were resuspended in 1 ml CHCl3/MeOH (2:1, v/v) and lyzed for 10 min in an ultrasonic bath at room temperature followed by an incubation of 50 min at 4 °C in a cryoshaker at 500 RPM. Phase separation was induced by addition of 240 μl H2O and centrifugation at 9880 g for 10 min. The organic phase was transferred to a glass vial and the solvent was removed using a nitrogen evaporator. Lipid extracts were vacuum sealed and stored at -80 °C. For ESI-MS analysis, extracts were re-dissolved in 500 μl MeOH and measured both in positive and negative ion modes using a flow rate of 5 µl/min. A data dependent analysis workflow (DDA) was applied using tandem MS based on collision-induced dissociation (CID) with stepped normalization collision energies (NCE) of 20, 25, 30 and an exclusion time of 300 s. The mass resolving power was set to 280,000 (at m/z 200) with an AGC target of 1E6 and a maximum injection time of 250 ms. The resulting data was merged with a t-MALDI-2-MSI dataset and imported into LipostarMSI (Molecular Horizons) for annotation. The resulting feature list was hand-curated, which involved removing of isotope peaks and addition of known species from E. coli. For mouse cerebellum and the 4T1 murine tumor section, annotations were based on accurate mass (< 3 ppm). For this, signals with S/N > 3 were extracted from the mean spectrum and annotated using the LipidMaps “Bulk Structure Search”77. In positive ion-mode sphingolipids, glycerolipids, sterol lipids and glycerophospholipids were selected as possible lipid classes either as [M + H]+- or [M-H2O + H]+-ions. The comparative analysis of untreated tissue also included [M+Na]+ and [M + K]+ ion species. In negative ion-mode only [M-H]--ions were searched for. The resulting annotation lists were manually curated to remove biologically unlikely annotation (e.g., lipids with odd side chains) or lipid classes that are usually not detected in the respective ion-mode (e.g., glycerophosphoinositols in positive ion-mode). Using this method, a total of 66 ion signals for the mouse cerebellum and 63 ion signals for the 4T1 tumor sample were annotated, respectively. The lipid annotation of mouse cerebellum was subsequently matched with data from a full lipidomics analysis reported in the literature32. Because t-MALDI-2-MSI without the use of additional techniques like OzID or Paternò-Büchi-reactions78,79 is not able to differentiate isomers within a lipid class (e.g., PC(16:0/18:1) and PC(16:1/18:0)) or between lipid classes (e.g., PE(16:0/18:0) and PC(15:0/16:0)) the literature data was reduced to 255 unique lipid species according to the here employed level of specificity. Of the 66 annotated m/z-values, 63 were also detected by Fitzner et al. 32. The remaining unmatched values primarily corresponded to sulfoglycosphingolipids and fatty acids, lipid classes that were not analyzed in the original publication. Overall, the annotation accounted for approximately 82% of the total lipid mass identified by Fitzner et al., calculated by summing all lipids found in both datasets and using the quantitative amounts reported. Lipid nomenclature and shorthand notation is reported according to Liebisch et al. 80. The external and in-source fluorescence images were co-registered using SimpleITK 2.3.1 (NumFOCUS)81,82. For initialization a landmark registration was used. Based on the resulting translation matrix an affine image registration was performed using Mattes mutual information as metric and the gradient descent algorithm as optimizer83. The number of bins for the mutual information was set to 50, the parameters for the optimizer were set as follows: learning rate (1), number of iterations (50), convergence minimum value (1E-6) and convergence window (20). The sampling strategy was set to random and 1% of the pixels were sampled. The resulting transformation matrix was combined with the initial translation matrix to result in a global transformation matrix. The precision of the registration was determined by placing pairs of landmarks at distinct features, which are visible in the external fluorescence image as well as the MALDI image of the cerebellum dataset (Supplementary Fig. While the precision of the placement of each individual landmark is limited by the manual selection process and differences between modalities, overall accuracy is ensured by determining the average over a total of eleven pairs of landmarks. Cell masks for tumor tissue were created by using DeepCell Mesmer 0.12.9 (Van Valen Lab)53 based on the DAPI channel (Hoechst 33342, nuclei) and the FITC channel (CellMask Green Actin Tracking Stain, actin). Outlines for subcellular features were determined by thresholding fluorescence signal intensity. Using FISCAS20, mass spectra were calculated for each cell or subcellular compartment by compiling MSI pixels based on their co-localized masks. MSI pixels assigned to multiple cell masks were scaled according to the hit coverage of the segmentation masks. Morphometric parameters were acquired using CellProfilers 4.2.1 (Cimini Lab)54 measurement modules. For cell classification, immune cells were identified and divided into three groups (CD45+, LyG6+, DcTRAIL-R1+) based on thresholding the mean intensities of the corresponding antibody stains in their respective fluorescence channels (Cy5, TxRed, Cy7). Ly6G+ cells showed high intensities on the TxRed and Cy5 channel, while DcTRAIL-R1+ cells showed high intensities on all three channels. The clustering of the tumor microenvironment (TME) was done using k-means clustering on the standardized MS data of all Ly6G- cells. The number of clusters was determined by visual inspection of the clustering results for values of k between 6 and 10. The Ly6G+ cells were also clustered using Density-Based Spatial Clustering of Applications with Noise (DBSCAN) with an epsilon of 0.7 and a minimum number of seven neighbors, based on a uniform manifold approximation and projection (UMAP) of their standardized mass spectra. The clusters are visualized both directly on the cell masks as well as in a UMAP of the standardized mass spectra using default parameters. Ly6G+ cells were assigned to a specific tumor region, by identifying the most frequent TME label of all neighbors within 40 µm. The presented work describes method development and can be considered proof-of-concept. No statistical method was used to predetermine sample size. For cerebellum and tumor tissues, multiple technical replicates with varying staining and sample preparation conditions were performed, and representative datasets were selected for display. For both tissue types, two additional MSI experiments with optimized sample preparation but varying fluorescence stains yielded comparable results. For the tumor sample, the full staining panel was applied once. For cell experiments, two independent replicates with consistent outcomes were performed, and representative data are shown. No data were excluded from the analyses. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment. The research presented here complies with all ethical regulations and was approved by the local ethics committee of the University of Münster and the North Rhine-Westphalia State Agency for Nature, Environment and Consumer Protection, LANUV, where applicable. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. MALDI MSI data, fluorescence and brightfield microscopy images as well as processed data have been deposited at OMERO and are publicly available including an interactive data viewer at https://doi.org/10.57860/min_prj_00001284. Any additional information required to open or reanalyze the data reported in this paper is available from the lead contact upon request. Source data are provided with this paper. All original code has been deposited at OMERO and is publicly available at https://doi.org/10.57860/min_prj_000012. The code is included in the main folder (Data S1.zip) and within the tumor project (Data_tumor.zip) and additional information is available from the respective readme files. Any additional information required to use code reported in this paper is available from the lead contact upon request. Establishing a conceptual framework for holistic cell states and state transitions. Bennett, H. M., Stephenson, W., Rose, C. M. & Darmanis, S. Single-cell proteomics enabled by next-generation sequencing or mass spectrometry. Boekweg, H. & Payne, S. H. Challenges and opportunities for single-cell computational proteomics. Bressan, D., Battistoni, G. & Hannon, G. J. The dawn of spatial omics. Highly multiplexed imaging in the omics era: understanding tissue structures in health and disease. Deep Visual Proteomics defines single-cell identity and heterogeneity. Novel technical developments in mass spectrometry imaging in 2020: a mini review. Soltwisch, J. et al. Mass spectrometry imaging with laser-induced postionization. Cuypers, E. et al. ‘On the spot' digital pathology of breast cancer based on single-cell mass spectrometry imaging. Taylor, M. J., Lukowski, J. K. & Anderton, C. R. Spatially resolved mass spectrometry at the single cell: recent innovations in proteomics and metabolomics. Bourceau, P. et al. Visualization of metabolites and microbes at high spatial resolution using MALDI mass spectrometry imaging and in situ fluorescence labeling. Muller, M. A., Zweig, N., Spengler, B., Weinert, M. & Heiles, S. Lipid signatures and inter-cellular heterogeneity of naive and lipopolysaccharide-stimulated human microglia-like cells. Global and spatial metabolomics of individual cells using a tapered pneumatically assisted nano-DESI probe. Niehaus, M., Soltwisch, J., Belov, M. E. & Dreisewerd, K. Transmission-mode MALDI-2 mass spectrometry imaging of cells and tissues at subcellular resolution. Kompauer, M., Heiles, S. & Spengler, B. Atmospheric pressure MALDI mass spectrometry imaging of tissues and cells at 1.4-μm lateral resolution. Bien, T., Koerfer, K., Schwenzfeier, J., Dreisewerd, K. & Soltwisch, J. Mass spectrometry imaging to explore molecular heterogeneity in cell culture. Rappez, L. et al. SpaceM reveals metabolic states of single cells. Capolupo, L. et al. Sphingolipids control dermal fibroblast heterogeneity. Schwenzfeier, J., Weischer, S., Bessler, S. & Soltwisch, J. Introducing FISCAS, a tool for the effective generation of single cell MALDI-MSI data. Ščupáková, K., Dewez, F., Walch, A. K., Heeren, R. M. A. & Balluff, B. Morphometric cell classification for single-cell MALDI-mass spectrometry imaging. Bien, T., Bessler, S., Dreisewerd, K. & Soltwisch, J. Transmission-mode MALDI mass spectrometry imaging of single cells: optimizing sample preparation protocols. Soltwisch, J. et al. MALDI-2 on a trapped ion mobility quadrupole time-of-flight instrument for rapid mass spectrometry imaging and ion mobility separation of complex lipid profiles. Enhanced MALDI-2 sensitivity with reflecting post-ionization laser for high-resolution MS imaging combined with real-time microscope imaging. Shimma, S. & Sagawa, T. Microscopy and mass spectrometry imaging reveals the distributions of curcumin species in dried turmeric root. Harada, T. et al. Visualization of volatile substances in different organelles with an atmospheric-pressure mass microscope. Shafer, C. C. & Neumann, E. K. Optimized combination of MALDI MSI and immunofluorescence for neuroimaging of lipids within cellular microenvironments. Direct visualization of neurotransmitters in rat brain slices by desorption electrospray ionization mass spectrometry imaging (DESI - MS). Shariatgorji, M., Svenningsson, P. & Andrén, P. E. Mass spectrometry imaging, an emerging technology in. SIMS imaging in neurobiology and cell biology. Schröder, H., Moser, N. & Huggenberger, S. The Mouse Cerebellum. in Neuroanatomy of the Mouse: An Introduction (eds. Schröder, H., Moser, N. & Huggenberger, S.) 153–170 (Springer, 2020). Fitzner, D. et al. Cell-type- and brain-region-resolved mouse brain lipidome. Patterson, N. H., Tuck, M., Van de Plas, R. & Caprioli, R. M. Advanced registration and analysis of MALDI imaging mass spectrometry measurements through autofluorescence microscopy. Integrating multiplex immunofluorescent and mass spectrometry imaging to map myeloid heterogeneity in its metabolic and cellular context. Sugiura, Y. et al. Visualization of the cell-selective distribution of PUFA-containing phosphatidylcholines in mouse brain by imaging mass spectrometry. Vandergrift, G. W. et al. Are phosphatidic acids ubiquitous in mammalian tissues or overemphasized in mass spectrometry imaging applications? Vos, D. R. N., Bowman, A. P., Heeren, R. M. A., Balluff, B. & Ellis, S. R. Class-specific depletion of lipid ion signals in tissues upon formalin fixation. Kralj, T. et al. Multi-omic analysis to characterize metabolic adaptation of the E. coli lipidome in response to environmental stress. Elsbach, P. & Weiss, J. Phagocytosis of bacteria and phospholipid degradation. Biophysica Acta (BBA) - Rev. Hambrey, P. N. & Mellors, A. Cardiolipin degradation by rat liver lysosomes. & Locasale, J. W. Metabolic landscape of the tumor microenvironment at single cell resolution. DuPré, S. A., Redelman, D. & Hunter, K. W. The mouse mammary carcinoma 4T1: characterization of the cellular landscape of primary tumours and metastatic tumour foci. Schrörs, B. et al. Multi-omics characterization of the 4T1 murine mammary gland tumor model. Hoffmann, E. et al. Profiling specific cell populations within the inflammatory tumor microenvironment by oscillating-gradient diffusion-weighted MRI. Li, P. et al. Dual roles of neutrophils in metastatic colonization are governed by the host NK cell status. Ng, M. S. F. et al. Deterministic reprogramming of neutrophils within tumors. Silvestre-Roig, C., Kalafati, L. & Chavakis, T. Neutrophils are shaped by the tumor microenvironment: novel possibilities for targeting neutrophils in cancer. Spangenberg, P. et al. msiFlow: automated workflows for reproducible and scalable multimodal mass spectrometry imaging and microscopy data analysis. Morgan, P. K. et al. A lipid atlas of human and mouse immune cells provides insights into ferroptosis susceptibility. Ahmadian, M., Duncan, R. E., Jaworski, K., Sarkadi-Nagy, E. & Sook Sul, H. Triacylglycerol metabolism in adipose tissue. Chughtai, K., Jiang, L., Greenwood, T. R., Glunde, K. & Heeren, R. M. A. Mass spectrometry images acylcarnitines, phosphatidylcholines, and sphingomyelin in MDA-MB-231 breast tumor models[S]. Anderson, D. M. G., Spraggins, J. M., Rose, K. L. & Schey, K. L. High spatial resolution imaging mass spectrometry of human optic nerve lipids and proteins. Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. Vicanolo, T. et al. Matrix-producing neutrophils populate and shield the skin. & Race, A. M. An overview of image registration for aligning mass spectrometry imaging with clinically relevant imaging modalities. Fan, R. Integrative spatial protein profiling with multi-omics. et al. MSIr: automatic registration service for mass spectrometry imaging and histology. Yang, E. et al. FluoMALDI microscopy: matrix co-crystallization simultaneously enhances fluorescence and MALDI imaging. Integration of mass cytometry and mass spectrometry imaging for spatially resolved single-cell metabolic profiling. Dufresne, M. et al. Aminated cinnamic acid analogs as dual polarity matrices for high spatial resolution MALDI imaging mass spectrometry. B. et al. A streamlined workflow for microscopy-driven MALDI imaging mass spectrometry data collection. Kaya, I. et al. Histology-compatible MALDI mass spectrometry based imaging of neuronal lipids for subsequent immunofluorescent staining. Connecting structure and function from organisms to molecules in small-animal symbioses through chemo-histo-tomography. Geier, B. et al. Spatial metabolomics of in situ host–microbe interactions at the micrometre scale. Yang, E. et al. RaMALDI: enabling simultaneous Raman and MALDI imaging of the same tissue section. Deep MALDI-MS spatial omics guided by quantum cascade laser mid-infrared imaging microscopy. Samarah, L. Z. et al. Single-cell metabolic profiling: metabolite formulas from isotopic fine structures in heterogeneous plant cell populations. Yeung, T., Ozdamar, B., Paroutis, P. & Grinstein, S. Lipid metabolism and dynamics during phagocytosis. Nikitina, A. et al. A co-registration pipeline for multimodal MALDI and confocal imaging analysis of stem cell colonies. Schneider, G., Sellers, Z. P., Abdel-Latif, A., Morris, A. J. & Ratajczak, M. Z. Bioactive lipids, LPC and LPA, are novel prometastatic factors and their tissue levels increase in response to radio/chemotherapy. Lauber, K. et al. Apoptotic cells induce migration of phagocytes via caspase-3-mediated release of a lipid attraction signal. Volpi, E. V. & Bridger, J. M. FISH glossary: an overview of the fluorescence in situ hybridization technique. Young, R. et al. Subcellular mass spectrometry imaging of lipids and nucleotides using transmission geometry ambient laser desorption and plasma ionisation. Belov, M. E. et al. Design and performance of a novel interface for combined matrix-assisted laser desorption ionization at elevated pressure and electrospray ionization with orbitrap mass spectrometry. Folch, J., Lees, M. & Sloane Stanley, G. H. A simple method for the isolation and purification of total lipides from animal tissues. Fahy, E., Sud, M., Cotter, D. & Subramaniam, S. LIPID MAPS online tools for lipid research. Claes, B. S. R. et al. Mass spectrometry imaging of lipids with isomer resolution using high-pressure ozone-induced dissociation. Bednařík, A., Bölsker, S., Soltwisch, J. & Dreisewerd, K. An on-tissue paternò-büchi reaction for localization of carbon-carbon double bonds in phospholipids and glycolipids by matrix-assisted laser-desorption-ionization mass-spectrometry imaging. Liebisch, G. et al. Update on LIPID MAPS classification, nomenclature, and shorthand notation for MS-derived lipid structures. Yaniv, Z., Lowekamp, B. C., Johnson, H. J. & Beare, R. SimpleITK image-analysis notebooks: a collaborative environment for education and reproducible research. Lowekamp, B. C., Chen, D. T., Ibanez, L. & Blezek, D. The Design of SimpleITK. Mattes, D., Haynor, D. R., Vesselle, H., Lewellyn, T. K. & Eubank, W. Nonrigid multimodality image registration. in Medical Imaging 2001: Image Processing vol. Potthoff et al. Data Share for ‘Spatial biology using single-cell mass spectrometry imaging with integrated fluorescence microscopy'. We thank Dagmar Mense for help with cell culture, Mathis Richter for helpful discussions on immunohistochemistry, Alexander Fengler and Vincent Suerdieck (all University of Münster) for help with mouse tissue sectioning, Benedikt Geier (Stanford), Robert Ernst and Hyojik Yang (both University of Maryland) for helpful discussions, Henning Peise, Arne Fütterer, Andreas Wöste, Raik Milautzki (all Bruker) for hard- and software support, and Sarah Weischer, Jens Wendt, and Thomas Zobel (Münster Imaging Network Cells in Motion Interfaculty Center) for support with image analysis. Figure 1 was created in part using biorender (agreement number ZR28JUV8U0). Financial support from the German Research Foundation to KD and OS (Z1 project of the TRR332; project no. 449437943) and to JS (project no. Open Access funding enabled and organized by Projekt DEAL. Alexander Potthoff, Jan Schwenzfeier, Sebastian Bessler, Klaus Dreisewerd & Jens Soltwisch Bruker Daltonics GmbH & Co. KG, Bremen, Germany Marcel Niehaus & Jens Höhndorf Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors declare the following competing interest(s): MN and JH are employees of Bruker Daltonik GmbH & Co KG (Bremen). KD, MN, AP, and JS have patents pending regarding parts of the hardware development (WO/2024/041681, WO/2025/067576). receives funds from Novo Nordisk and consults Novo Nordisk and Roche. Parts of this research have been funded by Bruker Daltonik GmbH & Co KG by supplying hardware, software, and salaries for AP and JSch. The remaining authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Potthoff, A., Schwenzfeier, J., Niehaus, M. et al. Spatial biology using single-cell mass spectrometry imaging and integrated microscopy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03365-1'>Scientists lose jobs and grants as US government shutdown takes a toll</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 01:44:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Traci Watson a senior editor for news and research highlights at Nature. Lawmakers from the opposition Democratic party say that they will only pass the spending bill if it extends popular health-care subsidies, a condition that Republicans do not want to negotiate. The Trump administration said in a court filing Friday that it will lay off 4,100- 4,200 federal employees, an action officially termed a reduction in force (RIF). Trump team guts AIDS-eradication programme and slashes HIV research grants Starting Friday night, some 1,300 staff members of the US Centers for Disease Control and Prevention (CDC) received RIF notices, although the notices for 700 were quickly rescinded, according to Local 2883 of the American Federation of Government Employees, a union representing CDC employees. Word of layoffs at the CDC's influential National Health and Nutrition Examination Survey (NHANES) sparked particular concern among epidemiologists. “This gold standard survey may no longer be able to operate in the future,” he says. Some CDC employees have now been laid off twice in the span of half a year, says mathematical statistician Isaac Michael. Several court orders have preserved their employment status for now, though they are still not allowed to work even when the government reopens. But some of his colleagues received a second layoff notice within the last few days, making it unlikely they will ever be reinstated. Trump links autism and Tylenol: is there any truth to it? Trump links autism and Tylenol: is there any truth to it? If a state experiences a future uptick in maternal or infant deaths, “we won't even know there's a problem, because we're not collecting any reliable data, and we won't be able to do anything to help”, Michael says. Andrew Nixon, communications director at the US Department of Health and Human Services (HHS), which oversees the CDC, said that all HHS employees receiving reduction-in-force notices were designated as non-essential by their respective divisions, and that the department will continue to close “wasteful and duplicative entities”. Coinciding with the shutdown, the administration has also announced a fresh round of cuts to research projects, adding to billions of dollars in federal research grants revoked since Trump took office in January. An analysis by Nature found that the list includes grants to 33 academic institutions, which have a combined value of $620 million. Five key climate and space projects on Trump's chopping block Five key climate and space projects on Trump's chopping block Science and the shutdown: 5 things to watch as US impasse drags on Trump moves to slash NSF: why are the proposed budget cuts so big? The EU's Horizon research fund needs to stay a defence-free zone The EU's Horizon research fund needs to stay a defence-free zone More than 30% of this century's science Nobel prizewinners immigrated: see their journeys Volunteer scientists work ‘nights and weekends' to guide vaccine advice in US Science and the shutdown: 5 things to watch as US impasse drags on Trump moves to slash NSF: why are the proposed budget cuts so big? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            