
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09614-7'>Isolation, engineering and ecology of temperate phages from the human gut</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:54:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Large-scale metagenomic and data-mining efforts have revealed an expansive diversity of bacteriophages (phages) within the human gut1,2,3. However, functional understanding of phage–host interactions within this complex environment is limited, largely due to a lack of cultured isolates available for experimental validation. Here we characterize 134 inducible prophages originating from 252 human gut bacterial isolates using 10 different induction conditions to expand the experimentally validated temperate phage–host pairs originating from the human gut. Importantly, only 18% of computationally predicted prophages could be induced in pure cultures. Moreover, we construct a 78-member synthetic microbiome that, when co-cultured in the presence of human colonic cells (Caco2), led to the induction of 35% phage species. Using cultured isolates, we demonstrate that human host-associated cellular products may act as induction agents, providing a possible link between gastrointestinal cell lysis and temperate phage populations4,5. We provide key insights into prophage diversity and genetics, including a genetic pathway for domestication, finding that polylysogeny was common and resulted in coordinated prophage induction, and that differential induction can be influenced by divergent prophage integration sites. More broadly, our study highlights the importance of culture-based techniques, alongside experimental validation, genomics and computational prediction, to understand the biology and function of temperate phages in the human gut microbiome. These culture-based approaches will enable applications across synthetic biology, biotechnology and microbiome fields. The human gut microbiota consists of a plethora of microorganisms along with the viruses that infect them, including phages. These viruses are thought to shape the gut microbial community through predation, horizontal gene transfer and lysogenic conversion6,7. Recent advances in computational mining of gut metagenomes have revealed an expansive collection of viral metagenome-assembled genomes and efforts cataloguing this diversity have led to the discovery of several important viral families1,2,3,8,9,10. Moreover, lysogeny is common within the gut, with up to 90% of bacteria predicted to harbour prophages11,12. However, the extent to which these prophages re-enter lytic replication remains unclear. For example, the inactivation of resident prophages represents a common strategy whereby the bacterial population can escape lysis while maintaining beneficial phage genes13,14. Furthermore, initiation of lytic replication by resident prophages is complex, involving both host- and phage-specific cues15,16. Within the gut, little is known about temperate phages and how they interact with our commensals. Advances in cultivation of the microbiota have enabled the isolation and archiving of previously ‘unculturable' gut bacterial species17,18, along with their phages. Here we use a collection of 252 human gut bacterial isolates (50 Actinomycetota, 1 Fusobacteriota, 51 Bacillota, 57 Pseudomonadota and 93 Bacteroidota) to computationally identify and experimentally validate inducible prophages (Fig. We began by exposing our bacterial isolate cultures to eight different induction agents and conditions, which included a standard medium control, well-known inducing agents such as mitomycin C (0.3 and 3 µg ml−1) and hydrogen peroxide (0.5 mM), along with lesser-known induction conditions with potential relevance to the gut, including the sugar substitute Stevia (3.7 and 37 mg ml−1) and two starvation conditions (50% carbon depletion and 100% short-chain fatty acid (SCFA) depletion)19,20,21,22,23. After induction, the samples were processed for DNA extraction and 433 viral induction samples that passed our inclusion criteria were sequenced (Extended Data Fig. This resulted in the detection of 125 inducible gut prophages, representing 63 (23%) phage species (95% average nucleotide identity (ANI), over 85% alignment fraction (AF))24 from 73 (29%) bacterial isolates (5 Actinomycetota, 1 Fusobacteriota, 10 Bacillota, 17 Pseudomonadota and 40 Bacteroidota) (Fig. a, Schematic of the methods used. (2) Bacterial synthetic community co-cultured with and without Caco2 cell monolayer. (3) Induction of bacterial isolate pure cultures with cell culture medium, Caco2 cells or cell products. The green lines connect isolates sharing induced prophage species and the black lines connect isolates of different genera with the same prophage species. c, The number of induced prophage species within the community (left; n = 29) and community co-cultured with Caco2 cell monolayer (right; n = 57); the grey shading indicates prophage species that were previously identified in standard conditions and pink shading shows prophage species induced only within the synthetic community. d, The number of induced prophage species as in c, coloured by phyla. e, Host abundance within the synthetic microbiome community at the phyla level, shown as the average of timepoints (n = 3) and replicates (n = 5). f, Phylogenetic tree of the bacterial synthetic community (n = 78), and heat map depicting induced prophages within the community (left; n = 22) and the community co-cultured with Caco2 cell monolayer (right; n = 42) identified using KrakenUnique. g, Isolates (n = 32) picked for pure culture induction in DMEM cell medium, Caco2 lysates and Caco2 cell monolayer. The bars represent the number of high-quality predictions (146); white shows total predictions, pink shows newly induced prophages (n = 9), grey squares show prophages also induced under standard conditions (n = 20). h, The percentage of induced isolates and the total number of isolates. i, The percentage of induced and the total number of high-quality prophage predictions. Fusobacteriota was excluded from h and i due to a single isolate. To further expand on the prophage induction triggers tested above, we constructed a synthetic bacterial community based on a subset of our isolates (n = 78, 4 Actinomycetota, 1 Fusobacteriota, 4 Bacillota, 28 Pseudomonadota and 41 Bacteroidota; Supplementary Table 1). Using this community, we investigated the effects of bacterial co-culture, in which the competition for resources, production of microbial byproducts and quorum sensing may affect prophage induction, as well as community co-culture with a monolayer of human colonic epithelial cells (Caco2), to investigate human host-associated factors5,16 (Extended Data Fig. In total, 29 phage species out of 162 (17%) were identified as induced in community co-culture using read mapping, yet, notably, 57 phage species (35%) were induced within the Caco2 co-culture, with a total of 22 phage species being newly identified as inducible across both experiments (Fig. However, there was a shift in the bacterial community composition within the Caco2 co-culture, dominated by Pseudomonadota and Bacteroidota, potentially leading to detection of prophages induced from isolates extinct in the community only co-culture25 (Fig. In a complementary approach, we detected the induction of distinct prophage–host pairs by identifying unique k-mers within each prophage genome. Considering the increased prophage induction within the Caco2 co-cultured community, we wanted to investigate whether human cells or cell lysis products (independent of community effects) act as prophage induction triggers. To this end, we selected 32 bacterial isolates from the community (3 Actinomycetota, 1 Fusobacteriota, 2 Bacillota, 14 Pseudomonadota and 12 Bacteroidota; Supplementary Table 1) for pure culture induction assays using Caco2 cell monolayers, Caco2 cellular lysates and DMEM cell culture medium alone (Extended Data Fig. These conditions induced 29 prophages, with 25 observed within the Caco2 cell lysate condition and 14 in the Caco2 monolayer and DMEM cell medium (Fig. Importantly, nine of the induced prophages had not been previously detected in our bacterial isolate cultures using standard induction agents, indicating that human host-associated cellular products act as induction triggers. Taken together, 35 out of 146 (24%) prophages were found to be inducible across all conditions within these 32 bacterial isolates. Consistent with previous reports of substantial lysogeny within the human gut11, 237 out of 252 of bacterial isolates (94%) were computationally predicted to contain high-quality prophage regions. The highest concordance between inducible and predicted prophage regions was observed within Bacteroidota isolates, in which 80 predictions (27%) from 41 isolates (44%) were inducible. Comparatively, in Pseudomonadota, which had the highest number of predicted prophages (4.5 per isolate), just 12% of prophages were found to be inducible. Moreover, combining the synthetic communities, a total of 36% of prophage species was detected as induced, coinciding with recent reports from human gut metagenomes (8–36%)26,27. Although our experimental approach does not provide comprehensive identification of all gut prophages due to factors including detection limits and potential unidentified induction conditions, it is likely that a substantial portion of predicted prophages within our dataset rarely undergo induction. Given the inherent challenges in assigning taxonomy to phages28, we applied both a gene voting-based search and a gene sharing network method using vContact2 (refs. 3a and Supplementary Table 4) with taxonomy assigned to the highest taxonomic resolution shared between methods. In total, 26% (35 out of 134) of phages could be assigned to ICTV (International Committee on Taxonomy of Viruses) accepted taxa at the family level or lower. These belonged to previously reported phage taxa infecting Pseudomonadota (Bcepmuvirus, Punavirus, Uetakevirus and Peduoviridae), one Spbetavirus infecting Bacillota and 16 prophages belonging to the Winoviridae family infecting Bacteroidota. Although lacking ICTV classification, 30 genomes could be grouped into viral clusters (genus-subfamily level) together with previously described phages. Notably, 19 of these clustered with Hankyphage, a recently described virus thought to lysogenize several Bacteroides species30. Further taxonomic classification grouped ten prophages at the species level with Hankyphage, whereas the remaining nine clustered into seven potential novel species, forming a putative novel genus that we name Hankyvirus after the original phage characterized (Extended Data Fig. Comparing the Hankyvirus species to bacterial genomes in NCBI RefSeq database (95% ANI over 85% AF), we identified 52 host species originating from 9 genera and 5 families, indicating a broad host range of this genus (Extended Data Fig. Correspondingly, we find two Hankyvirus species induced within both Bacteroides and Phocaeicola isolates within our collection, providing experimental validation of these phages as actively replicating across these two host genera. a, Gene sharing network of inducible prophage species (solid circles; n = 68) coloured by host. Database representatives (9,920) are translucent and coloured by host phyla when applicable, and are otherwise coloured grey. The portion of the network connecting to induced prophages is shown; the full network is shown in Extended Data Fig. 3. b, The mean fractional abundance and detection frequency (prevalence) of Caudoviricetes phages within 1,241 viromes originating from the human gut. A minimum of 70% coverage over the phage was required to be counted as present within a virome. The bacterial host phyla of inducible prophage species coloured as in a, and database reference genomes (n = 875) are shown in grey. Approximately half of our inducible prophage species (28 out of 68) could be detected in gut viromes9 (n = 1,241; Fig. LoVEphage, a recently discovered Bacteroidota phage9,10, was the most common, being detected in around 8% (97 out of 1,241) of the viromes and representing up to 64% of reads within one virome (Supplementary Table 3). Three phages in our collection were species-level members of LoVEphage, induced from Bacteroides thetaiotaomicron, Phocaeicola dorei and Phocaeicola vulgatus hosts (Extended Data Fig. An additional eight phage species were detected in 2–5% of gut viromes (Supplementary Table 3). These included the four species within the Hankyvirus genus, one Uetakevirus infecting Escherichia coli and three previously uncharacterized Bacteroidota phages (Wilby, Saffi and Shia; Extended Data Fig. Discernible integrase or site-specific recombination genes, both of which are used as hallmark genes for a temperate lifestyle32,33, were absent in 28% (19 out of 68) of our inducible phage species, including Hankyviruses. We found transposases in ten of these viruses, while the remaining nine lacked any discernible integration genes, illustrating the difficulty in assigning phage lifestyle based on genomic data alone. Diversity-generating retroelements (DGRs) are prevalent within the gut virome, and tail-targeting DGRs are known to enable rapid host switching in a Bordetella phage34,35. We found DGRs in 19% (13 out of 68) of our inducible prophage species, the majority of which were seen in Bacteroidota phages, in which 43% (12 out of 28) of species encoded DGRs targeting known and genomically predicted tail proteins (Supplementary Table 3). Concordantly, we found eight Bacteroidota prophage species actively replicating across different bacterial species, three of which replicating across different bacterial genera (Fig. 1b (connecting lines)), highlighting the involvement of DGRs in phage host range expansion through diversification of tail proteins34. More recently, bacterial DGRs were implicated in anti-phage defence mechanisms, and targeted engineering using DGRs accelerated evolution within both host receptor and the reciprocal phage binding domain36,37. Notably, we found four prophage species containing DGRs that encoded a second variable region (VR) targeting genes distal from the reverse transcriptase cassette9 (Extended Data Fig. The second VR was found in proximity to counter defence genes, such as DNA methyltransferase, indicating further involvement of DGRs in the phage–host arms race34. The retention of cryptic prophages is known to provide the host with adaptive fitness advantages and has been shown to result in a bimodal size distribution of prophage genomes13,14. Concordantly, we find bimodal length distributions of prophages across all host phyla within our collection (Extended Data Fig. 4a), with the early peak corresponding to sequences with low completeness scores (<50% complete, n = 1,236) and later peaks corresponding to high-quality predictions (>50% complete, n = 736) and experimentally inducible prophages (n = 134; Fig. To investigate differences in gene content between these groups, we performed gene enrichment analysis of annotated PHROG gene categories and found that small prophage genomes lacked essential phage genes (such as structural, head and packaging, and lysis genes) but were enriched in accessory genes and genes of unknown function38 (Extended Data Fig. Similarly, when limiting the comparison to high-quality predictions, we found an enrichment of structural genes (head and packaging, connector, lysis and tail) within induced prophages, whereas non-induced predictions were enriched for accessory genes and genes of unknown function, indicating that a subset of high-quality predictions might be cryptic prophage-like elements or poor predictions (Fig. a, The length distribution of induced, high-quality (>50% complete) and low-quality (<50% complete) predicted prophage genomes. b, The percentage frequency change in PHROG gene categories between induced and high-quality (>50% complete) predicted prophage genomes, counting the total number of genes (blue) or the presence/absence count of each gene category within prophage genomes (yellow). Significant P values (shown above the bars) were calculated using two-sided Fisher's exact tests and adjusted using the Hochberg method. c, The dN/dS ratio between induced–induced or induced–non-induced prophage pairs (left; n = 313) and between their hosts (right; n = 389). No significant difference was found based on two-sided Wilcoxon rank-sum tests. d, The dN/dS ratio of PHROG gene categories (n = 7,893) between induced–induced and induced–non-induced prophage pairs. Significant P values (shown above the brackets) were calculated using two-sided Wilcoxon rank-sum tests and adjusted using the Hochberg method. e, Absolute qPCR quantification of ΦPomma in Stevia (left, n = 3 biological replicates) and hydrogen peroxide (right, n = 3 biological replicates). The ΦPomma wild type is shown in red and ΦPomma ∆tran is shown in blue. f, DNA-sequencing reads from CC01414 ΦPomma wild type and ΦPomma ∆tran mapped to the Bacteroides faecis CC01414 genome; colours are as in e. The Stevia condition is shown by the solid lines and the hydrogen peroxide condition is shown by the dotted lines. For the ΦPomma genome map, genes are coloured by PHROG categories; unknown genes are shown in grey. The deleted ‘DNA transposition protein' gene is highlighted by a black dashed box. We next sought to investigate potential genetic mechanisms leading to trapping of prophages within the host genome by comparing experimentally induced prophages to highly similar, non-induced prophages (95% ANI, 85% AF; Supplementary Table 7). To classify these non-induced prophages as putatively cryptic, we restricted the analysis to prophages that had been sequenced (but not induced) in the same condition(s) as their inducible counterparts, with the rationale that highly similar prophages should respond to the same induction triggers. This resulted in a total of 231 prophage pairs between 65 induced and 58 non-induced prophages. No significant changes were found in gene frequency (P > 0.05, Fisher's exact test), indicating that, although gene loss may be characteristic of cryptic prophages, it is unlikely to be the initial cause of inactivation. Moreover, although we detected 201 homologous gene transfer (HGT) and 65 insertion–deletion events, there was no significant difference in the number of total events when compared to a set of high sequence similarity induced prophage pairs (222 pairs, P = 0.46, Wilcoxon rank-sum test). Comparing host ANI between the induced and non-induced prophage pairs (Extended Data Fig. 4c,d), we found no association with induction (P = 0.6, Pearson's correlation), suggesting that prophage inactivation was not driven by diversification of the host or integration into divergent non-permissive hosts. To investigate whether non-induced prophages have an elevated number of mutations, we measured the ratio of non-synonymous to synonymous substitution rates (dN/dS) within the set of induced to non-induced prophage pairs, and their associated hosts. We found an overall elevated mutation rate in prophages (mean = 0.89, median = 0.18) compared with the host genome (mean = 0.16, median = 0.095, P < 2 × 10−16, Wilcoxon rank-sum test), but no significant difference between induced to induced or induced to non-induced prophage pairs or their host genomes (P = 0.99 and P = 0.62 respectively, Wilcoxon rank-sum test; Fig. Comparing gene substitution rates, we found 143 genes with elevated dN/dS rates (>1), indicating diversifying selection (Fig. Notably, the majority (110 out of 143) of these genes lacked a known function and 40% (57 out of 143) were associated with DGRs, highlighting an active and not yet deciphered role of DGRs within gut prophages. Importantly, within the non-induced prophages we found a significant increase in the dN/dS substitution rates in genes involved in integration and excision (P = 0.002, Wilcoxon rank-sum test), suggesting that non-functional mutations in these genes provides a pathway for the inactivation of prophages. To test whether the inactivation of integration and excision genes could trap a prophage inside its host genome, we constructed a gene deletion mutant of the inducible prophage Pomma by knocking out its DNA transposition protein (ΦPomma ∆tran) within the Bacteroides faecis host strain CC01414 (Extended Data Fig. From our previous inductions, we found that prophage Pomma was selectively induced by hydrogen peroxide and Stevia (37 mg ml−1). Using these two inducers, we compared the induction of wild-type ΦPomma versus the ∆tran mutant using quantitative PCR (qPCR) and sequencing of chloroform and DNase-treated supernatants. qPCR analysis showed a 3.5 and 2.6 log increase in ΦPomma concentration within the wild type versus ∆tran deletion mutant in samples treated with Stevia and hydrogen peroxide, respectively (Fig. Through sequencing, we observed clear induction in the wild-type strain, with 35- and 17-fold increased coverage over the bacterial background in the Stevia-treated and hydrogen-peroxide-treated samples, respectively, whereas no increase over the bacterial background was detected in the ∆tran mutant strain (Fig. To determine whether prophage induction was linked to phylogeny, we examined the induction response across our ten conditions and standard growth control (Fig. Combined, the two concentrations of mitomycin C induced the largest number of prophages (n = 70) and the most Pseudomonadota prophages (n = 17). The Caco2 human cell induction conditions induced 29 prophages from 32 tested hosts, with Bacteroidota (n = 16) followed by Pseudomonadota (n = 9) showing the largest numbers of induced prophages. Comparing induction conditions across each phylum, the only significant difference observed was within the Pseudomonadota phyla in response to 3 µg ml−1 mitomycin C (P = 0.024, Fisher's exact test). a, The number of induced prophages per sample (condition in rows and isolates in columns). b, The distribution of induced single and polylysogens per bacterial phyla. Fusobacteriota were excluded due to a single isolate. c, Kendall's rank correlation between number of inducible prophages within lysogens and the number of conditions in which each phage was detected as induced (the size is based on the number of observations). d, The fold change in induced prophages over background in isolates grown in standard medium (left, n = 3 biological replicates) or with the addition of hydrogen peroxide (right, n = 3 biological replicates). Normality was tested using the Shapiro–Wilk test, and significant P values (shown above brackets) of normal data were calculated using two-sided paired t-tests. e, The genome location of prophage ΦWilby (red) and ΦPomma (grey). Shaded lines connect genes with 100% amino acid identity (AAI). We next investigated polylysogeny across our collection and its influence on induction (Fig. Polylysogeny was most prevalent within the Bacteroidota isolates, in which 28 out of 41 (68%) of lysogens had more than one inducible prophage compared with 11 out of 38 isolates (29%) across the other phyla (P = 0.002, Fisher's exact test). We then compared whether polylysogeny influenced induction, observing a positive correlation between the number of co-inhabiting inducible prophages and conditions leading to induction of each prophage (τ = 0.22, P = 0.002, Kendall's rank correlation; Fig. Prophages residing in polylysogens (n = 90) were induced on average in 2.7 conditions compared with 2.1 conditions in single lysogens (n = 35, P = 0.03, Wilcoxon rank-sum test), suggesting that polylysogeny may promote simultaneous prophage induction and reduce stability within lysogens. Finally, we investigated differential induction within polylysogens by measuring the abundance of phage DNA in the supernatants of five highly similar (99% ANI) Bacteroidota caccae isolates harbouring the same two prophages (ΦWilby and ΦPomma; Supplementary Table 11). We identified an overall preferential induction of ΦWilby within standard medium (P = 0.006), but not in hydrogen-peroxide-treated samples (P = 0.9, Wilcoxon rank-sum test), with isolate CC01407 demonstrating the most marked difference between the two phages (P = 0.026, paired t-test; Fig. Calculating the ratio of ΦWilby over ΦPomma within each isolate, we found a significant variance of means between the isolates in both standard medium (P = 0.012) and hydrogen peroxide (P = 0.0008, analysis of variance (ANOVA)). These results implied that the host genetic background, even within highly similar isolates, may affect prophage induction. We previously identified phage ΦPomma as a transposable prophage that does not use site-specific integration, but randomly inserts into the host genome39. To investigate the prophage integration sites within our isolates, we used long-read sequencing on the five B. caccae strains. Genomic analysis identified ΦWilby integrated into the same tRNA gene location, which is characteristic of site-specific integration; however, the transposable prophage ΦPomma was found in four different genomic locations within the five isolates (Fig. 4e), implicating the integration site as a possible driver for the observed differential induction within these isolates. The high microbial load within the human gut represents an optimal environment for temperate phages, as frequent interactions with their hosts provide ample opportunity for lysogeny12. Concordantly, the majority of bacteria within the gut are predicted to be lysogens, with up to 90% of bacteria harbouring at least one prophage11,12. However, the degree to which these prophages engage in lytic replication is poorly understood. Using our defined culture collection of 252 gut bacterial isolates, we predict that the majority harbour prophage-like elements (94%), but find that only a fraction of predicted prophages could be experimentally induced in pure culture (18%). Caveats to our approach include experimental cut-offs for detection and minimum amounts of DNA required for sequencing, which could exclude detection of low-level-inducing prophages. Moreover, there are probably biases towards induction and detection of Caudoviricetes prophages and, indeed, all but one prophage (from the Inoviridae family) belonged to this class. Moreover, considering little is known about prophage induction triggers within the gut, it is plausible that some of our isolates carry prophages that were not induced due to a lack of appropriate induction triggers. To address this, we constructed a synthetic microbiome community and co-cultured it with or without human cells to simulate the biologically relevant conditions within the human gut. Within the community co-culture, we estimate that around 29% of prophages were induced, with around half of these induced only in co-culture with human epithelial Caco2 cells. However, whether this induction was triggered by human cell factors or was the result of spontaneous induction mirroring the shift in host taxa was unclear36,40. We therefore investigated human host factors in the absence of community effects, using 32 pure culture isolates exposed to Caco2 cell monolayers, Caco2 cellular lysates and DMEM cell culture medium alone. We observed a modest increase in induction with lysed Caco2 cellular products compared with cell culture medium or intact cells, suggesting that human cellular lysis products act as prophage induction triggers. This is in accordance with previous observations of temperate virion expansion found in patients with inflammatory bowel disease that are associated with increased inflammation and cell death4. Recent advances have highlighted the complexities governing prophage induction within natural environments, ranging from SOS-independent induction triggers, interprophage competition and interference by defence mechanisms41,42,43,44,45,46,47,48. Across our pure culture and community inductions, only a minority of predicted prophages was detected to undergo lytic replication. We therefore propose that, although the genetic pool of integrated prophages within the gut is large, only a fraction of these will readily undergo lytic replication. This is consistent with previous studies estimating low induction rates within the human gut and reduced lytic infection rates of temperate gut phages25,49. Furthermore, we detect distinct gene enrichment patterns where non-induced prophage predictions encoded fewer structural and lysis-associated genes, indicating that a portion of high-quality predictions might consist of prophage remnants or poor prophage predictions. Moreover, non-induced predictions with high sequence similarity to experimentally induced prophages exhibited increased non-synonymous substitution rates in integrase and excision-related genes. Deletion of one of these genes in an active prophage led to complete abolishment of induction, providing evidence for a genetic pathway towards prophage domestication. A considerable portion of our isolates (52%) with inducible prophages were polylysogens, harbouring more than one replicating prophage48. We found a positive correlation between polylysogeny and successful prophage induction conditions, which is consistent with previous reports of phage anti-repressor proteins targeting non-cognate prophages leading to synchronized prophage induction50. Finally, we show that induction of polylysogenic prophages varied between near identical isolates, which correlated with divergent prophage integration sites within the host genome. Thus, prophage induction is complex and influenced by growth condition, polylysogeny and prophage integration site. In summary, we demonstrate the feasibility of culture-based approaches to provide insights into temperate phage biology and their interactions within human-associated commensals, and provide a validated collection of phage–host pairs for future use in synthetic biology, microbiome and biotechnological advances. A culture collection of 252 bacterial isolates previously isolated and sequenced from human gut samples was used for prophage induction51. All bacterial culture work was performed in yeast-extract casitone fatty acid (YCFA) medium at 37 °C under anaerobic conditions (Whitely A95 anaerobic workstation) containing 10% carbon dioxide, 10% hydrogen and 80% nitrogen52. Each isolate was streaked onto YCFA agar plates and incubated for 24 h before a single colony was inoculated in 1 ml YCFA medium in a 96-well plate and incubated for 24 h. Frozen stocks of the 96-well master plates were maintained in glycerol suspension (25%, v/v) at −80 °C. Before each induction, 96-well plates containing 1 ml YCFA were inoculated from the frozen master plate and grown overnight. A set of 40 single-copy marker genes were extracted from the 252 bacterial isolates using progenome-classifier53 and translated into amino acid sequences using SeqKit54 (v.2). Maximum-likelihood trees were constructed using RAxML57 (v.8.2.12) PROTGAMMALGF model with 100 bootstraps replicates and visualized in iTOL58. Bacterial clusters sharing 99% ANI were identified using dRep59 (v.3.0.0) with the ‘-pa 0.9 --sa 0.99' flags. Prophage regions were predicted using Virsorter60, Vibrant61 (default settings) and VirFinder62 (minimum length 5 kb; 0.7 score; and P value 0.05). Completeness was predicted using CheckV63 and contaminating bacterial regions were removed. Trimmed predictions were located within their cognate bacterial genome and overlapping predictions were merged using R IRanges64 (v.2.28.0). Prophages were induced by one of two methods. (1) Overnight starter cultures were diluted 1:50 in 1.5 ml standard YCFA medium and grown for 5 h before the addition of mitomycin C19 (0.3 or 3 μg ml−1, M4287, Sigma-Aldrich) or hydrogen peroxide20 (0.5 mM, H1009, Sigma-Aldrich). All cultures were then grown for 20–25 h followed by centrifugation at 4,000g for 30 min and 1 ml supernatants were collected. The supernatants were treated with 10 μg ml−1 DNase I (DN25, Sigma-Aldrich) and 0.01 volume RNase A (R6148, Sigma-Aldrich) for 1 h at 37 °C. Viral particles were precipitated in 7% PEG 8000 0.3 M NaCl overnight at 4 °C, followed by centrifugation at 14,000g for 30 min, after which the pellets were dissolved in 50 μl TE buffer at 4 °C. Next, 20 μl of each sample was mixed with 5 μl loading dye containing 0.8% SDS and 60 mM EDTA and heated at 65 °C for 10 min. The samples were loaded onto 0.4% agarose gels and run in TAE for 1.5 h, followed by visualization of phage sized (about 50 kb) DNA bands using Image Studio Lite (LI-COR Biosciences) with a sample to control well signal ratio cut off of 0.03 (refs. Samples with suspected viral DNA were treated with 0.5% SDS and 100 μg ml−1 proteinase K at 55 °C for 1 h followed by a 10 min inactivation at 65 °C. Phenol–chloroform–isoamyl alcohol (25:24:1) extraction was performed followed by sodium acetate (0.3 M final) and 70% ethanol precipitation with 0.4 mg ml−1 glycogen overnight at 4 °C. The DNA quantity was validated using Qubit (Thermo Fisher Scientific), with a minimum of 2 ng µl−1 required for sequencing. From these, a subset of samples was selected for sequencing as follows. First, all of the samples grown in standard YCFA medium or induced with mitomycin C were selected. Second, samples from at least one isolate within a bacterial cluster (99% ANI) grown in the remaining five induction conditions were selected, except for the Fusobacteria isolate, which was sequenced only in standard medium and mitomycin C. For 17 out of 84 clusters, more than one isolate was sequenced in all conditions. Nextera-XT libraries were constructed and sequenced on either the Illumina NextSeq 2000 or Illumina NextSeq 550 system. Human colonic epithelial immortalized cells (Caco2 TC766; genotype verified by the AGRF Human Cell Line Identification Service) were maintained in Dulbecco's modified Eagle medium (DMEM, low glucose, GlutaMAX supplement, pyruvate) (Thermo Fisher Scientific), containing 10% FCS (Bovogen) in 5% CO2 at 37 °C in T75 cm2 flasks (Nunc, Thermo Fisher Scientific). Cells were routinely tested for the presence of mycoplasma contamination (MycoStrip, Invivogen) and were confirmed to be mycoplasma negative. For sonicated cell inductions, 4 × 107 cells in 5 ml DMEM (without FCS) were sonicated on ice (5 cycles, 30 s on and off at 40% frequency). Sonicated Caco2 cells were confirmed by observing efficient lysis of cells by bright-field microscopy and stored at −20 °C. Before the addition of bacterial cells, sonicated cells were thawed on ice and pre-reduced DMEM was resuspended with the cells for a final cell density of 5 × 105 cells per ml in the Whitley A95 Anaerobic Workstation (Don Whitley Scientific) at 37 °C. Sonicated cells in DMEM (2 ml) were added to 6-well tissue culture plates to achieve a total of 1 × 106 cells per well. Two days before induction, 1 × 106 Caco2 cells were seeded in 6-well tissue culture plates (Nunc, Thermo Fisher Scientific) in a final volume of 2 ml per well with DMEM containing 10% FCS and incubated for 48 h at 37 °C under 5% CO2. Caco2 confluent cell layers were serum starved under anaerobic conditions for 2 h by replacing the cell medium with 2 ml pre-reduced DMEM (no FCS) per well in the Whitley A95 anaerobic workstation at 37 °C. Individual working stocks of bacterial isolates were prepared from overnight bacterial cultures that were centrifuged at 4,000g for 10 min, resuspended to an optical density at 600 nm (OD600) of 5.5, combined 1:1 with 150 μl 50% glycerol and stored as frozen glycerol stocks (25% final glycerol). Glycerol stocks of individual bacterial isolates were thawed and 8.5 µl of each isolate was added to 6-well tissue culture plates containing 2 ml pre-reduced DMEM medium only, 2 ml sonicated Caco2 cells (total cell number, 1 × 106 cells per well) or confluent Caco2 cell layer. All cultures were grown for 24 h followed by centrifugation at 4,000g for 30 min after which 1.8 ml supernatants were collected, and viral DNA was extracted and sequenced on the Illumina NextSeq 2000 system as previously described. Reads were trimmed using Trimmomatic67 (v.0.38) (SLIDINGWINDOW:4:25 MINLEN:100) and used to identify induced prophages using two approaches. First, high-quality prophage predictions (>50% completeness) were validated for induction as follows. Read coverage for each library were obtained on their corresponding genome using Bowtie2 (ref. Genome coverage in 100 bp increments was obtained using Samtools69 (v.1.9) and Deeptools70 (v.3.1.3) and the average modified z score, coverage fold increase and Cohen's D of prophage regions was calculated as follows: where z-scoreave is the average z score of the predicted region, xp is 100 bp coverage increments of the phage region, xh is 100 bp coverage increments of the host and \(\tilde{x}\) is median coverage of the host and where Cohen's D is the effect size of prophage versus host coverage, Sh and Sp is the s.d. of the host and phage coverage, respectively71. Regions with a minimum average modified z score of 3.5 or an average twofold coverage and Cohen's D larger than 0.7 were retained. A custom Python script was then used to refine the start stop positions of the prophage regions within each genome, removing flanking 100 bp increments with coverage less than 25% of the mean prophage coverage (code is available at https://doi.org/10.26180/29946902.v1). In a second approach, regions of increased coverage were identified without previous prophage predictions using hafeZ72 (v.1.0.2) (default settings with the -N -S flags). Some of the identified regions of interest were found to be split across several host contigs. To resolve these into full-length phage contigs, de novo assembly using MetaViral SPade73 (default setting) was performed and contigs overlapping with hafeZ prediction were retained. The resulting contigs were dereplicated at 99% ANI over 85% of the AF using scripts from the CheckV repository and the longest representative of each prophage were retained for further analysis. Proteins from the resulting contigs were predicted and annotated using PROKKA74 (v.1.14.6) (default settings, --hmms) with the PHROGS38 database. Furthermore, all proteins were scanned against the hmm databases provided by Cenote-Taker2 (ref. To remove potential fragmented protein hits against hallmark genes, a custom phage database was constructed of genomes from refs. 9,31 and the INphared database (December 2021 version) dereplicated at 95% ANI over 85% AF using CheckV scripts77. The same HMM searches were performed on proteins from the database and half the average length of the middle 80% percentile was calculated as a cut-off for each Caudoviricetes hallmark gene (265 amino acids for terminase large subunit, 245 amino acids for portal protein and 186 amino acids for major head protein). To identify any non-Caudoviricetes genomes, HMM searches for Microviridae, Tectiviridae and Inoviridae were performed as follows. 78 was made into HMM profiles using MAFFT v.7.310 with the standard settings followed by TABAJARA (-t 0.5 -p 50 -w 15 -b 15 -mb 15 -m c -gs 20 -md 3 -cs yes -mb 20)79. Multiple-sequence alignments of the double jelly-roll hallmark protein of Tectiviridae were obtained from a previous study80 and turned into HMM profiles using HMMER v.3.3.1 hmmbuild. These and the Inoviridae morphogenesis protein family HMMs provided by ref. 81 were searched against all proteins using hmmscan (-E 1e-9 and score ≥ 30). Contigs containing at least one viral hallmark gene were retained. Working stocks of the synthetic microbiome community were prepared by combining 1.5 ml of overnight bacterial culture diluted to OD600 0.7, after which the community was centrifuged at 8,000g for 10 min, resuspended in 10 ml fresh YCFA and stored as frozen glycerol stocks (25% final glycerol). Human Caco2 confluent cell layers grown for 48 h in 175 cm2 tissue culture flasks (NUNC) in DMEM containing 10% FCS with 5% CO2 at 37 °C were transferred to anaerobic conditions (Whitley A95 anaerobic workstation) and the medium in each flask was replaced with 70 ml pre-warmed and pre-reduced (overnight) DMEM (without FCS). Next, 200 μl of the frozen community stock was added to each 70 ml tissue culture flasks containing Caco2 cell layers as well as to culture bottles containing 200 ml pre-reduced YCFA medium, both in five replicates. All cultures were grown anaerobically with samples (14 ml) taken at 24, 48 and 72 h. Total metagenome DNA extraction was performed using the FastDNA SPIN Kit for Soil (MP Biomedicals) on 1 ml of each sample. The remainder of the sample was centrifuged at 3,000g for 30 min and the supernatant was collected and filtered through 0.45 μM syringe filters (Acrodisc, Pall) followed by incubation for 15 min with 0.1 volumes of chloroform at room temperature. The samples were then centrifuged at 4,000g for 15 min and 9 ml of the aqueous phase was collected and treated with 10 μg ml−1 DNase I (DN25, Sigma-Aldrich) and 120 μl RNase A (R6148, Sigma-Aldrich) for 1 h at room temperature. Viral particles were precipitated using 7% PEG 8000 incubated overnight at 4 °C, centrifuged at 12,000g for 1 h and resuspended in 100 μl TE buffer. Viral DNA was extracted and sequenced on the Illumina NextSeq 2000 system as previously described. A database of community prophage genomes (high-quality predictions n = 338) and bacterial host genomes (n = 78, masked for prophage regions using bedtools (v.2.26.0)) was constructed70. Decontaminated community reads from samples were aligned against the database using Bowtie2 (default flags) and coverage was obtained using Samtools ‘coverage'. Host abundance was calculated from the Samtools outputs of total DNA extracted sequence libraries. In viral-enriched samples, prophage species were regarded to be induced if at least one representative genome was covered with reads ≥85% of the length with a twofold increase in coverage (depth) over the mean host genome coverage (sum coverage of host contigs normalized by length) in a minimum three out of five replicates. For detection of individual prophage genomes, a custom metagenomic read classification database was built using KrakenUniq (v.1.0.4)82 containing prophage and prophage-masked host genomes. For database construction purposes, phage sequences were assigned the NCBI taxonomy IDs of their host bacteria. NCBI taxonomic data were downloaded using the ‘krakenuniq-download --db taxonomy' command on 7 July 2024. The KrakenUniq database was constructed with the krakenuniq-build command using Jellyfish (v.1.1.12) for k-mer extraction, a k-mer size of 21 and the --taxids-for-genomes option. Paired-end reads were merged using the read_merger.pl script within KrakenUniq and subsequently classified using the krakenuniq command with default parameters. On the basis of data from pure isolate inductions, a cut-off of 0.25 k-mer coverage, 10 reads and 100 unique k-mers was selected for calling detection of phage and a cut-off of 10 reads and 18,000 unique k-mers for calling detection of bacterial host genome was used. Prophage genomes were regarded as induced if they had a twofold increase in kmerDuplicity over mean host genome kmerDuplicity (sum duplicity of host contigs normalized to length) in a minimum of three out of five replicates. Prophages from undetected isolates were regarded as induced if detected in a minimum of three replicates. To estimate the number of detectable prophages within the community (prophages with at least 100 unique k-mers), pairwise distances between all prophage and prophage-masked host genomes were calculated using Mash (v.2.2.2) with a k-mer size 21 and sketch size 5000 (ref. Neighbour-joining was applied to the resulting distance matrix as implemented in RapidNJ (v.2.3.2) with default parameters84. Using the resulting tree, a metagenomic classification database was constructed using Expam (v.1.2.2.5) with a k-mer size 21 and number of unique k-mers per genome was obtained using the CountUniqueKmers.py script (https://github.com/seansolari/expam/scripts/database/CountUniqueKmers.py)85. Viral taxonomy was assigned based on a combination of the protein alignment method previously described2 against the INphared database and genus level clustering using vContact229 against phage genomes in the custom made database used for hallmark gene searches. In cases in which the taxonomic assignments from the protein voting and genus-level clustering method differed, the lowest common classification was assigned. Species level dereplication was performed at 95% ANI over 85% AF using scripts from the CheckV repository. DGRs were identified using DGRscan86 with the default settings, and remote VRs were identified querying the template repeat using BLASTn (v.2.7.1+) (-dust no -perc_identity 75 -qcov_hsp_perc 50 -ungapped -word_size 4). DGR-positive genomes were defined as genomes encoding both a reverse transcriptase gene and containing repeat regions. The fractional abundance and prevalence of induced prophages within gut viromes were performed as described previously9 using the 1,241 human gut viromes described therein. Reads from each virome were competitively aligned to the temperate phage species genomes together with the custom database previously described. The fractional abundance of a genomes was calculated as follows: and the sum fractional abundance was normalized to 1 as previously described87. A genome was counted as present within a virome if at least 70% of the genome length was covered by reads. Proteins of predicted prophages were predicted using PROKKA v.1.14.6 (default setting, --hmms) and annotated using the PHROG database. The total gene counts per PHROG category and the presence–absence of PHROG categories within each genome were obtained for induced, high-quality predictions (>50% completeness) and low-quality predictions (<50% completeness). The percentage gene frequency change of PHROG categories between induced, high-quality and low-quality predictions was calculated for total genes and presence–absence counts as follows: where fcry and fin are the gene frequencies in the high completeness prediction and induced prophage set, respectively. High-quality predictions were aligned to induced prophages using BLASTn (v.2.15.0+) and pairs with a minimum of 95% ANI over 85% AF (Checkv anicalc.py63 script) were further filtered to only include hosts that had been sequenced in the same condition(s) as the induced prophage. The same search was performed to identify induced–induced prophage pairs. The number of HGT and insertion–deletion events between the pairs was calculated using R IRanges (v.2.28.0) and splicejam (v. 0.0.77) packages, where an HGT event was defined as a gap of a at least 50 bp within the alignment present in both pairs and insertion–deletion events was defined as gap (minimum 50 bp) present in one of the pairs but not the other88. Gaps involving the ends of either prophage were excluded. Host ANI of prophage pairs was calculated using fastANI89 (v.1.33). dN/dS ratios between prophage pairs was calculated using dRep (compare --SkipMash --S_algorithm goANI) and the dnds_from_drep.py90 script. Gene deletion of the DNA transposition protein gene within ΦPomma in B. faecis isolate CC01414 was achieved using the CRISPR–Cas-based system described previously91. First, we redesigned pB025, which contains the FnCas12a system, with a sgRNA targeting the DNA transposition protein gene (gene location, base pairs 5610–6524) along with a repair template containing 1,000 bp of homologous DNA up and downstream of this gene (Supplemental Table 10). This plasmid (pB025_09) was transformed into competent E. coli S17 and grown aerobically in LB medium supplemented with 100 µg ml−1 ampicillin. B. faecis was grown in brain–heart infusion (BHI) liquid medium supplemented with haemin, resazurin and vitamin K3 (menadione) under anaerobic conditions. Conjugation was performed under anaerobic conditions and B. faecis transconjugants were selected for with 200 µg ml−1 gentamicin and 25 µg ml−1 erythromycin. A deletion mutant was identified before anhydrotetracycline (aTc) induction, presumably due to leaky expression. This deletion mutant was verified by PCR and sanger sequencing confirming the clean deletion of the DNA transposition protein gene (CC01414 ∆tran). Induction of ΦPomma in CC01414 wild type and deletion mutant (∆tran) was performed as described previously using hydrogen peroxide (0.5 mM) and stevia (37 mg ml−1) using three separate induction reactions for each condition and isolate. Lysates were treated with 2% chloroform, centrifuged for 20 min at 4,000g at 4 °C, DNase treated and phage precipitation with PEG, DNA extraction and sequencing was performed as described previously. qPCR of DNA extracted phage lysates was performed in technical triplicates using SYBR Green I Master Mix (Roche Diagnostics) with the Roche Lightcycler 480 system containing 1 μM of each primer, 2 μl of DNA template and 1× SYBR Green I Master Kit, in a final reaction volume of 20 μl. Cycle parameters were as follows: initial denaturation at 95 °C for 10 min; followed by 45 cycles of 95 °C for 20 s, 62 °C for 20 s, and 72 °C for 30 s. Primers were designed using Primer blast and no cross reactivity to bacterial background was detected (https://www.ncbi.nlm.nih.gov/tools/primer-blast). The standard curve was produced with gBlock sequence from IDT containing the sequence targeted by ΦPomma primers. Isolates were streaked onto YCFA plates and grown for 24 h. Three sperate colonies from each isolate were inoculated into 1 ml YCFA broth and grown overnight. Overnight cultures were diluted 1:50 into 1.5 ml standard YCFA medium and hydrogen peroxide was added after 5 h of growth. All cultures were grown for an additional 20 h and lysates were treated with 2% chloroform and centrifuged for 20 min at 4,000g at 4 °C and frozen at −80 °C until analysis was performed. qPCR was performed as previously described using 5 μl of DNA template and annealing temperature of 60 °C for 30 s and elongation of 30 s. qPCR primer pairs were custom designed using Primer 3 (https://primer3.org/). In silico PCR amplification (http://insilico.ehu.eus/user_seqs/PCR/) did not show cross reactivity of primers to the rest of the bacterial genome. Standard curves for primer efficiency analysis were generated by tenfold dilution in PCR-grade H2O. Samples were diluted tenfold and qPCRs was performed in triplicates. The efficiency of each primer calculated as in equation (5) and corrected ΔCt values calculated as in equation (6): Isolates were streaked onto YCFA plates and grown for 24 h. Single colonies were grown overnight in 40 ml of YCFA medium, pelleted by centrifugation at 4,000g for 10 min and washed four times in 1 ml of PBS. DNA was extracted using the Monarch HMW DNA extraction kit (New England Biolabs) according to the Gram-positive bacteria protocol, with modifications. Cells were lysed in 300 μl of STET buffer (8% sucrose 5% Triton X-100 50 mM EDTA, 50 mM Tris pH 8) containing 10 mg ml−1 lysozyme, 300 μl of HMW gDNA tissue lysis buffer and 20 μl of proteinase K and incubated at 56 °C for 10 min. The lysates were treated with 10 μl of RNase A at 56 °C for 5 min followed by 300 μl of protein separation solution. The samples were mixed by inversion for 2 min then centrifuged at 4 °C for 20 min at 16,000g. The supernatants were collected and 550 μl of isopropanol was added to 800 μl of supernatant. The samples were inverted for 5 min, or until DNA was precipitated, and DNA was pelleted by centrifugation at 4 °C for 10 min at 12,000g. The resulting pellet was washed twice with 500 μl of gDNA wash buffer and resuspended in nuclease free water. Library preparation and Oxford Nanopore MinION sequencing was performed using either the Oxford Nanopore ligation sequencing kit (SQK-LSK109) with native barcoding expansion kit (EXP-NBD114) (CC01407, CC01390, CC01401 and CC01405) or the rapid barcoding kit 96 (SQK-RBK110.96, CC01404). Resulting long reads were hybrid assembled with Illumina short reads into closed genomes using dragonfly (v.1.0.14) (CC01407, CC01390, CC01401 and CC01405) or into near complete genome using unicycler92 (v.0.4.7) (CC01404) with subsequent scaffolding using RagTag93 with CC01407 genome as reference. Significance of PHROG gene category was calculated with Fisher's exact test (two sided) and P values were adjusted with the Hochberg method using R base stats (R v.4.1.3) and rstatix (v.0.7.0) packages. Pearson's correlation test between host ANI and phage pair inducibility as well as Kendall's rank correlation between the number prophages within lysogens and prophage inducibility was calculated and plotted using the R ggpubr (v.0.4.0) package. Significance of horizontal gene transfers and dN/dS data was calculated with Wilcoxon rank-sum test (two sided) and adjusted by Hochberg method using R ggpubr (v.0.4.0) and rstatix (v.0.7.0) packages. Normality of qPCR fold change between induced prophage in polylysogens calculated with Shapiro–Wilk test and significance tested with paired t-test (two sided) using rstatix (v.0.7.0), preferential induction calculated with Wilcoxon rank-sum test (two sided) and variance of means between isolates was calculated with ANOVA using the R base stats (R v.4.1.3) package. Genome maps were visualized using the R gggenomes (v.0.9.9.9000) package, and genome read coverage was visualized using R ggplot2 (v.3.5.1). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data for this study have been deposited in the European Nucleotide Archive (ENA) at EMBL-EBI under project number PRJEB64565 and accession numbers are provided in Supplementary Table 1, 3 and 12. Bacterial isolates are available through the Australian Microbiome Culture Collection (AusMiCC). Metadata of viromes used in this study along with accession numbers are provided in Supplementary Table 5. Bioinformatic scripts and figure data are available at Figshare94 (https://doi.org/10.26180/29946902.v1). Illustrations were made using Inkscape (https://inkscape.org). This study did not generate any new code. Analysis scripts are available at Figshare94 (https://doi.org/10.26180/29946902.v1). Camarillo-Guerrero, L. F., Almeida, A., Rangel-Pineros, G., Finn, R. D. & Lawley, T. D. Massive expansion of human gut bacteriophage diversity. Nayfach, S. et al. Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome. The Gut Virome Database reveals age-dependent patterns of virome diversity in the human gut. Clooney, A. G. et al. Whole-virome analysis sheds light on viral dark matter in inflammatory bowel disease. Expansion of bacteriophages is linked to aggravated intestinal inflammation and colitis. Explaining microbial population genomics through phage predation. Chevallereau, A., Pons, B. J., van Houte, S. & Westra, E. R. Interactions between bacterial and phage communities in natural environments. Zuo, T. et al. Human-gut-DNA virome variations across geography, ethnicity, and urbanization. Benler, S. et al. Thousands of previously unknown phages discovered in whole-community human gut metagenomes. Van Espen, L. et al. A previously undescribed highly prevalent phage identified in a Danish enteric virome catalog. Govier, T. & Verwoerd, W. The promise and pitfalls of prophages. Anthenelli, M. et al. Phage and bacteria diversification through a prophage acquisition ratchet. Bobay, L. M., Touchon, M. & Rocha, E. P. C. Pervasive domestication of defective prophages by bacteria. Wang, X. et al. Cryptic prophages help bacteria cope with adverse environments. Erez, Z. et al. Communication between viruses guides lysis-lysogeny decisions. Silpe, J. E., Duddy, O. P. & Bassler, B. L. Natural and synthetic inhibitors of a phage-encoded quorum-sensing receptor affect phage–host dynamics in mixed bacterial communities. Browne, H. P. et al. Culturing of ‘unculturable' human microbiota reveals novel taxa and extensive sporulation. Forster, S. C. et al. A human gut bacterial genome and culture collection for improved metagenomic analyses. Induction of phage formation in the lysogenic Escherichia coli K-12 by mitomycin C. Nature 184, 1079–1080 (1959). & Wȩgrzyn, G. Hydrogen peroxide-mediated induction of the Shiga toxin-converting lambdoid prophage ST2-8624 in Escherichia coli O157:H7. Oh, J.-H. et al. Dietary fructose and microbiota-derived short-chain fatty acids promote bacteriophage production in the gut symbiont Lactobacillus reuteri. Morris, R. M., Cain, K. R., Hvorecny, K. L. & Kollman, J. M. Lysogenic host–virus interactions in SAR11 marine bacteria. Boling, L. et al. Dietary prophage inducers and antimicrobials: toward landscaping the human gut microbiome. Minimum information about an uncultivated virus genome (MIUVIG). A. et al. Abundance measurements reveal the balance between lysis and lysogeny in the human gut microbiome. & Maurice, C. F. Bacteriophages playing nice: lysogenic bacteriophage replication stable in the human gut microbiota. Shalon, D. et al. Profiling the human intestinal environment under physiological conditions. Adriaenssens, E. M. Phage diversity in the human gut microbiome: a taxonomist's perspective. Bin Jang, H. et al. Taxonomic assignment of uncultivated prokaryotic virus genomes is enabled by gene-sharing networks. Benler, S. et al. A diversity-generating retroelement encoded by a globally ubiquitous Bacteroides phage. Analysis of metagenome-assembled viral genomes from the human gut reveals diverse putative CrAss-like phages with unique genomic features. Viruses in the faecal microbiota of monozygotic twins and their mothers. The human gut virome is highly diverse, stable, and individual specific. Reverse transcriptase-mediated tropism switching in Bordetella bacteriophage. Roux, S. et al. Ecology and molecular targets of hypermutation in the global microbiome. Harnessing diversity generating retroelements for in vivo targeted hyper-mutagenesis. Targeted hypermutation of putative antigen sensors in multicellular bacteria. Terzian, P. et al. PHROG: families of prokaryotic virus proteins clustered using remote homology. O'Brien, S., Kümmerli, R., Paterson, S., Winstanley, C. & Brockhurst, M. A. Transposable temperate phages promote the evolution of divergent social strategies in Pseudomonas aeruginosa populations. Moreno-Gallego, J. L. et al. Virome diversity correlates with intestinal microbiome diversity in adult monozygotic twins. Silpe, J. E., Duddy, O. P. & Bassler, B. L. Induction mechanisms and strategies underlying interprophage competition during polylysogeny. Refardt, D. Within-host competition determines reproductive success of temperate bacteriophages. Azulay, G. et al. A dual-function phage regulator controls the response of cohabiting phage elements via regulation of the bacterial SOS response. Guo, Y. et al. Control of lysogeny and antiphage defense by a prophage-encoded kinase-phosphatase module. Song, S. et al. CRISPR-Cas controls cryptic prophages. The Escherichia coli CRISPR system protects from λ lysogenization, lysogens, and prophage induction. Silpe, J. E. et al. Small protein modules dictate prophage fates during polylysogeny. Mathieu, A. et al. Virulent coliphages in 1-year-old children fecal samples are fewer, but more infectious than temperate coliphages. Lemire, S., Figueroa-Bossi, N. & Bossi, L. Bacteriophage crosstalk: coordination of prophage induction by trans-acting antirepressors. D'Adamo, G. L. et al. Bacterial clade-specific analysis identifies distinct epithelial responses in inflammatory bowel disease. & Harmsen, H. J. M. Growth requirements and fermentation products of Fusobacterium prausnitzii, and a proposal to reclassify it as Faecalibacterium prausnitzii gen. nov., comb. Mende, D. R. et al. ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes. & Hu, F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Capella-Gutiérrez, S., Silla-Martínez, J. M. & Gabaldón, T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. & Bork, P. Interactive tree of life (iTOL) v5: an online tool for phylogenetic tree display and annotation. & Banfield, J. F. DRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. Roux, S., Enault, F., Hurwitz, B. L. & Sullivan, M. B. VirSorter: mining viral signal from microbial genomic data. & Anantharaman, K. VIBRANT: automated recovery, annotation and curation of microbial viruses, and evaluation of viral community function from genomic sequences. & Sun, F. VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data. Nayfach, S. et al. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. Lawrence, M. et al. Software for computing and annotating genomic ranges. Spontaneously induced prophages are abundant in a naturally evolved bacterial starter culture and deliver competitive advantage to the host. Differential expression of sucrase-isomaltase in clones isolated from early and late passages of the cell line caco-2: evidence for glucose-dependent negative regulation. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Kieft, K. & Anantharaman, K. Deciphering active prophages from metagenomes. A. hafeZ: active prophage identification through read mapping. & Pevzner, P. A. Metaviral SPAdes: assembly of viruses from metagenomic data. Seemann, T. Prokka: rapid prokaryotic genome annotation. & Buck, C. B. Cenote-Taker 2 democratizes virus discovery and sequence annotation. Cook, R. et al. INfrastructure for a PHAge REference Database: identification of large-scale biases in the current collection of cultured phage genomes. Gut virome of mammals and birds reveals high genetic diversity of the family Microviridae. Ibrahim, B. et al. Bioinformatics meets virology: the European Virus Bioinformatics Center's second annual meeting. Yutin, N., Bäckström, D., Ettema, T. J. G., Krupovic, M. & Koonin, E. V. Vast diversity of prokaryotic virus genomes encoding double jelly-roll major capsid proteins uncovered by genomic and metagenomic sequence analysis. Roux, S., Krupovic, M., Daly, R.A. et al. Cryptic inoviruses revealed as pervasive in bacteria and archaea across Earth's biomes. Breitwieser, F. P., Baker, D. N. & Salzberg, S. L. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Solari, S. M., Young, R. B., Marcelino, V. R. & Forster, S. C. Expam—high-resolution analysis of metagenomes using distance trees. Identification of diversity-generating retroelements in human microbiomes. Viruses as winners in the game of life. O'Donnell, S. & Fischer, G. MUM&Co: accurate detection of all SV types through whole-genome alignment. Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. & Aluru, S. High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries. Consistent metagenome-derived metrics verify and delineate bacterial species boundaries. Zheng, L. et al. CRISPR/Cas-based genome editing for human gut commensal Bacteroides species. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Unicycler: resolving bacterial genome assemblies from short and long sequencing reads. Automated assembly scaffolding using RagTag elevates a new tomato system for high-throughput genome editing. Dahlman, S. et al. Data and code for ‘Isolation, engineering and ecological dynamics of temperate phages from the human gut'. This work was supported by the Australian Research Council (ARC) Discovery Project grant (DP210103296). and L.A.-F. were supported by Monash University Postgraduate Research Scholarship funding their doctoral studies; R.A.E. by an award from the National Institute of Health (NIH NIDDK; RC2DK116713) and awards from the ARC (DP220102915, DP250103825, and FL250100019); S.C.F. by a CSL Centenary Fellowship; and J.J.B by a National Health and Medical Research Council (NHMRC) Investigator Grant Leadership Level 1 (NHMRC; 2026130) and Monash University Research Talent Accelerator (RTA) 2023 program. We acknowledge the staff at the Hudson Genomics Facility, Monash eResearch Team and the Victorian State Government Operational Infrastructure Scheme for support. Open access funding provided by Monash University. These authors contributed equally: Samuel C. Forster, Jeremy J. Barr School of Biological Sciences, Monash University, Melbourne, Victoria, Australia Sofia Dahlman, Laura Avellaneda-Franco, Ciaren Kett, Dinesh Subedi, Nathan Campbell & Jeremy J. Barr Centre for Innate Immunity and Infectious Disease, Hudson Institute of Medical Research, Melbourne, Victoria, Australia Young, Jodee A. Gould, Jasmine D. Bell, Callum A. H. Docherty & Samuel C. Forster Department of Molecular and Translational Sciences, Monash University, Melbourne, Victoria, Australia Young, Jodee A. Gould, Jasmine D. Bell, Callum A. H. Docherty & Samuel C. Forster Centre to Impact AMR, Monash University, Melbourne, Victoria, Australia School of Optometry and Vision Science, UNSW Medicine, University of New South Wales, Sydney, New South Wales, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed in vitro inductions, molecular work, sequencing, informatic analyses and most of the data preparation. L.A. assisted with informatic analyses, data interpretation, molecular work and sequencing. assisted with molecular work and sequencing. isolated and assisted with the cultured bacterial isolates. assisted with identification of induced prophages. performed synthetic microbiome co-culture and pure-culture inductions. S.S. assisted in design of analysis methods for synthetic microbiome inductions. are advisors to or employees of Biomebank. The other authors declare no competing interests. Nature thanks Benjamin Adler, Martha Clokie, Mart Krupovic and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic of bacterial single isolate inductions. Isolates were grown 20–25 h in standard media, Mitomycin C (0.3 and 3 μg/mL), hydrogen peroxide (0.5 mM), Stevia (3.7 and 37 μg/mL), carbon depleted or SCFA depleted media (n = 2,016). Samples were centrifugated, DNAse treated, PEG concentrated, and phage sized DNA bands were detected on agarose gels. DNA was extracted from samples with phage sized bands (n = 859) and DNA concentrations >2 ng/µL were chosen for sequencing based on condition and bacterial cluster (n = 452), 19 of which failed library preparation. Second, a subset of isolates (n = 32) were induced in DMEM media with or without sonicated or intact human colonic epithelial cells (Caco2) and DNA was extracted (n = 96), 3 of which failed library preparation. Prophages were predicted using Virsorter, Vibrant and VirFinder (n = 1972) and predictions >50% complete were trimmed for bacterial flanking regions (n = 736). Prophage reads (n = 526) were aligned to host genome and predictions with >2-fold coverage and Cohen's D > 0.7 or mean zscore >3.5 were retained. Further, reads were investigated for induction using HafeZ. De novo assembly using MetaViralSpades was used to resolve predictions spanning host contigs. Predictions were dereplicated at 99% ANI and 85% AF retaining the longest representative with at least one viral hallmark gene. b, Phylogenetic tree of bacterial isolates with heatmap depicting sequenced phage-enriched samples (shown in blue); Actinomycetota yellow (20/409), Fusobacteriota black (6/8), Bacillota blue (37/414), Pseudomonadota red (216/498) and Bacteroidota teal (247/786). c, Schematic of the synthetic community inductions. Isolates (n = 78) were co-cultured with or without Caco2 cell monolayer for 24, 48 and 72 h. Host abundance was calculated on total metagenome DNA using read mapping. Prophage induction was detected in phage enriched DNA at species level using read mapping as well as on individual prophage using KrakenUnique. a, Annotated genome maps of the eight Hankyvirus genus phages species. DGR template and variable repeats denoted with black lines. b, Host range of Hankyvirus and LoVEphage species induced in this study. Light green denotes hosts found in NCBI RefSeq bacterial genome database; dark green denotes hosts in which the phage was actively induced in this study. a, Gene sharing network between induced prophage species (solid circles, n = 68) and database representatives (9920). Representatives in translucence and coloured by host phyla when applicable otherwise in grey, except Crassvirales genomes from Yutin et al. 2021 which are highlighted with solid black boarded grey circles31. b, Mean fractional abundance and prevalence of Caudoviricetes phages within gut viromes (1241) shown as in Fig. 2b, but highlighting Crassvirales genomes in large solid grey circles. c, Annotated genome maps of four phage species encoding a second variable repeat (VR) distal from the RT cassette. DGR template and variable repeats denoted with black lines. a, Length distribution of all predicted prophages in this study separated by host phyla. Fusobacteriota excluded as only single isolate. A bimodal length distribution was observed for all phyla, with an initial peak at around 8 kb followed by a second peak at around 37 kb. b, Frequency of PHROG gene categories across induced (green, n = 134), high quality (> 50% completeness, orange, n = 736) and low quality (< 50% completeness, blue, n = 1236) predictions. Significant p values calculated using Fisher's exact test (two sided) and adjusted by Hochberg method shown above bars. Comparisons coloured by genus and shape based on NCBI taxon. Host pairs with less than 80% similarity grouped separately as too divergent for reliable ANI score. d, Density of ANI comparisons of all isolates within the dataset separated by phyla, coloured as in panel a. e, PCR amplification with phage specific primers surrounding the targeted gene (DNA transposition protein), visualized by agarose gel electrophoresis (single experiment) alongside GeneRuler 1 kb DNA ladder, showing deletion of a ~ 900 bp fragment in øPomma ∆tran mutant (gene product size, 915 bp). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Isolation, engineering and ecology of temperate phages from the human gut. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41565-025-02010-2'>Wearable biomolecular sensing nanotechnologies in chronic disease management</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:43:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Over the past decade, consumer wearable sensors have become increasingly ubiquitous in health monitoring, enabling the widespread tracking of key biophysical parameters. The transition towards next-generation body-interfaced biomolecular sensing technologies, fuelled by the integration of reagentless sensing strategies with advanced nanomaterials, marks the next substantial leap forward. These innovations enable unobtrusive, multimodal monitoring of both physiological parameters and biochemical disease markers in real time. This Review examines the current generation of body-interfaced biomolecular sensing technologies, with a particular emphasis on materials innovation and nanotechnological advancements, and discusses their pivotal role in chronic disease monitoring. The discussion extends to the challenges and prospects in this rapidly evolving field, highlighting the potential for materials-focused approaches to transform the landscape of chronic disease monitoring and management with body-interfaced bioelectronics. By harnessing the power of materials and nanotechnological innovations, these biomolecular sensing technologies promise to enhance diagnostic capabilities and foster a more proactive, personalized approach to combating these diseases. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The top 10 causes of death. WHO https://www.who.int/news-room/fact-sheets/detail/the-top-10-causes-of-death (2020). Zhao, C., Park, J., Root, S. E. & Bao, Z. Skin-inspired soft bioelectronic materials, devices and systems. Google Scholar Lee, G.-H. et al. Multifunctional materials for implantable and wearable photonic healthcare devices. Google Scholar Flynn, C. D. et al. Biomolecular sensors for advanced physiological monitoring. Google Scholar Townsend, N. et al. Epidemiology of cardiovascular disease in Europe. Article PubMed Google Scholar Mahmud, A. et al. Monitoring cardiac biomarkers with aptamer-based molecular pendulum sensors. Google Scholar Saenger, A. K. A tale of two biomarkers: the use of troponin and CK-MB in contemporary practice. Google Scholar Libby, P. et al. Atherosclerosis. Article PubMed Google Scholar Wang, H., Rosendaal, F. R., Cushman, M. & Van Hylckama Vlieg, A. D‐dimer, thrombin generation, and risk of a first venous thrombosis in the elderly. Google Scholar Pan, X. et al. Associations of circulating choline and its related metabolites with cardiometabolic biomarkers: an international pooled analysis. Google Scholar Tousoulis, D. et al. Serum osteoprotegerin and osteopontin levels are associated with arterial stiffness and the presence and severity of coronary artery disease. Article PubMed Google Scholar Tomic, D., Shaw, J. E. & Magliano, D. J. The burden and risks of emerging complications of diabetes mellitus. Google Scholar Lee, M. et al. β-hydroxybutyrate as a biomarker of β-cell function in new-onset type 2 diabetes and its association with treatment response at 6 months. Article PubMed Google Scholar Zheng, X. et al. Hyocholic acid species as novel biomarkers for metabolic disorders. Google Scholar Katsiki, N., Mikhailidis, D. P. & Banach, M. Leptin, cardiovascular diseases and type 2 diabetes mellitus. Acta Pharmacol. Google Scholar Theofilopoulos, A. N., Kono, D. H. & Baccala, R. The multiple pathways to autoimmunity. Google Scholar Kolarz, B., Podgorska, D. & Podgorski, R. Insights of rheumatoid arthritis biomarkers. Article PubMed Google Scholar Pisetsky, D. S. Anti-DNA antibodies—quintessential biomarkers of SLE. Article PubMed Google Scholar Shirzaei Sani, E. et al. A stretchable wireless wearable bioelectronic system for multiplexed monitoring and combination treatment of infected chronic wounds. Google Scholar Colhoun, H. M. & Marcovecchio, M. L. Biomarkers of diabetic kidney disease. Google Scholar Khalil, M. et al. Neurofilaments as biomarkers in neurological disorders. Article PubMed Google Scholar Hagberg, L., Edén, A., Zetterberg, H., Price, R. W. & Gisslén, M. Blood biomarkers for HIV infection with focus on neurologic complications—a review. Acta Neurol. Google Scholar Busche, M. A. & Hyman, B. T. Synergy between amyloid-β and tau in Alzheimer's disease. Google Scholar Emin, D. et al. Small soluble α-synuclein aggregates are the toxic species in Parkinson's disease. Google Scholar Sarkar, S. et al. Expression of microRNA-34a in Alzheimer's disease brain targets genes linked to synaptic plasticity, energy metabolism, and resting state network activity. Brain Res. Google Scholar Martinez, B. & Peplow, P. MicroRNAs in Parkinson's disease and emerging therapeutic targets. Neural Regen. Google Scholar Otani, N., Hoshiyama, E., Ouchi, M., Takekawa, H. & Suzuki, K. Uric acid and neurological disease: a narrative review. Google Scholar Moustafa, A. A., Hewedi, D. H., Eissa, A. M., Frydecka, D. & Misiak, B. in Diet and Exercise in Cognitive Function and Neurological Diseases (eds. Wang, L., Hu, Y., Jiang, N. & Yetisen, A. K. Biosensors for psychiatric biomarkers in mental health monitoring. Article PubMed Google Scholar Zamani, M., Wilhelm, T. & Furst, A. L. Perspective—electrochemical sensors for neurotransmitters and psychiatrics: steps toward physiological mental health monitoring. Google Scholar Schumann, G. et al. Stratified medicine for mental disorders. Article PubMed Google Scholar Rutsch, A., Kantsjö, J. & Ronchi, F. The gut–brain axis: how microbiota and host inflammasome influence brain physiology and pathology. Google Scholar Roomruangwong, C. et al. Menstruation distress is strongly associated with hormone–immune–metabolic biomarkers. Article PubMed Google Scholar Ciebiera, M. et al. Nutrition in gynecological diseases: current perspectives. Google Scholar Mavreli, D., Theodora, M. & Kolialexi, A. Known biomarkers for monitoring pregnancy complications. Expert Rev. Article PubMed Google Scholar Eastell, R. & Hannon, R. A. Biomarkers of bone health and osteoporosis risk. Article PubMed Google Scholar Amin, M. N. et al. Inflammatory cytokines in the pathogenesis of cardiovascular disease and cancer. SAGE Open Med. Google Scholar Jain, K. K. in The Handbook of Biomarkers 27–238 (Springer, 2017). Liu, C. H. et al. Biomarkers of chronic inflammation in disease development and prevention: challenges and opportunities. Article PubMed Google Scholar Tu, J. et al. A wireless patch for the monitoring of C-reactive protein in sweat. This paper introduces a wearable, wireless patch that enables real-time, non-invasive monitoring of the inflammatory biomarker C-reactive protein in sweat, correlating with serum levels and demonstrating high translation potential for the point-of-care management of chronic diseases. Google Scholar Liu, Y. et al. Revolutionizing precision medicine: exploring wearable sensors for therapeutic drug monitoring and personalized therapy. Google Scholar Sempionatto, J. R., Montiel, V. R.-V., Vargas, E., Teymourian, H. & Wang, J. Wearable and mobile sensors for personalized nutrition. ACS Sens. Article PubMed Google Scholar Wang, M. et al. A wearable electrochemical biosensor for the monitoring of metabolites and nutrients. This study reports on a wearable electrochemical biosensor capable of continuously monitoring multiple non-electroactive metabolites and nutrients, including all essential amino acids and vitamins, in sweat during both exercise and rest, utilizing innovative MIPs, redox-active nanoparticles and integrated sweat induction and sampling technologies. Google Scholar Güttler, N. et al. Omega-3 fatty acids and vitamin D in cardiology. Google Scholar Kevadiya, B. D. et al. Diagnostics for SARS-CoV-2 infections. Google Scholar Wang, C. et al. Wound management materials and technologies from bench to bedside and beyond. Google Scholar Cappon, G., Vettoretti, M., Sparacino, G. & Facchinetti, A. Continuous glucose monitoring sensors for diabetes management: a review of technologies and applications. Google Scholar Arlett, J. L., Myers, E. B. & Roukes, M. L. Comparative advantages of mechanical biosensors. Article PubMed Google Scholar Zhang, Y. et al. Nanozymes for nanohealthcare. Methods Prim. Google Scholar Wu, X., Ge, J., Yang, C., Hou, M. & Liu, Z. Facile synthesis of multiple enzyme-containing metal–organic frameworks in a biomolecule-friendly environment. Google Scholar Wang, M. et al. Printable molecule-selective core–shell nanoparticles for wearable and implantable sensing. This study reports on a skin-interfaced, printable wearable sensor leveraging core–shell nanoparticles with MIP shells for customizable and selective sweat analysis, featuring electrochemical regeneration via electrical pulses for repeated use without performance loss. Google Scholar Yao, J., Yang, M. & Duan, Y. Chemistry, biology, and medicine of fluorescent nanomaterials and related systems: new insights into biosensing, bioimaging, genomics, diagnostics, and therapy. Article PubMed Google Scholar Biswas, A. et al. Advances in top-down and bottom-up surface nanofabrication: techniques, applications & future prospects. Colloid Interface Sci. Article PubMed Google Scholar Wang, P. et al. DNA origami guided self-assembly of plasmonic polymers with robust long-range plasmonic resonance. Article PubMed Google Scholar Fu, K. et al. Accelerated electron transfer in nanostructured electrodes improves the sensitivity of electrochemical biosensors. Google Scholar Altug, H., Oh, S.-H., Maier, S. A. & Homola, J. Advances and applications of nanophotonic biosensors. Article PubMed Google Scholar Bauch, M., Toma, K., Toma, M., Zhang, Q. & Dostalek, J. Plasmon-enhanced fluorescence biosensors: a review. Article PubMed Google Scholar Fu, W. et al. Efficient optical plasmonic tweezer-controlled single-molecule SERS characterization of pH-dependent amylin species in aqueous milieus. Google Scholar Min, J. et al. Skin-interfaced wearable sweat sensors for precision medicine. Google Scholar Xu, Y. et al. In-ear integrated sensor array for the continuous monitoring of brain activity and of lactate in sweat. Google Scholar Arwani, R. T. et al. Stretchable ionic–electronic bilayer hydrogel electronics enable in situ detection of solid-state epidermal biomarkers. This article reports on a stretchable ionic–electronic bilayer hydrogel sensor capable of detecting solid-state epidermal biomarkers, including cholesterol and lactate, directly on the skin surface without the need for fluid sampling, combining innovative materials science with electrochemical sensing for non-invasive, continuous health monitoring. Article PubMed Google Scholar Ye, C. et al. A wearable aptamer nanobiosensor for non-invasive female hormone monitoring. Article PubMed Google Scholar Wu, Z. et al. Interstitial fluid-based wearable biosensors for minimally invasive healthcare and biomedical applications. Google Scholar Friedel, M. et al. Opportunities and challenges in the diagnostic utility of dermal interstitial fluid. Article PubMed Google Scholar Tehrani, F. et al. An integrated wearable microneedle array for the continuous monitoring of multiple biomarkers in interstitial fluid. In this study, multiple biomarkers were continuously monitored using a fully integrated wearable microneedle array in ISF during daily activities. Article PubMed Google Scholar Lipani, L. et al. Non-invasive, transdermal, path-selective and specific glucose monitoring via a graphene-based platform. Article PubMed Google Scholar Gao, Y. et al. A flexible multiplexed immunosensor for point-of-care in situ wound monitoring. Google Scholar Xiong, Z. et al. A wireless and battery-free wound infection sensor based on DNA hydrogel. Google Scholar Lee, Y. et al. Wireless, intraoral hybrid electronics for real-time quantification of sodium intake toward hypertension management. Natl Acad. Google Scholar Mannoor, M. S. et al. Graphene-based wireless bacteria detection on tooth enamel. Article PubMed Google Scholar Kim, J. et al. Wearable salivary uric acid mouthguard biosensor with integrated wireless electronics. Google Scholar Güntner, A. T. et al. Breath sensors for health monitoring. ACS Sens. Article PubMed Google Scholar Ates, H. C. & Dincer, C. Wearable breath analysis. Google Scholar Nguyen, P. Q. et al. Wearable materials with embedded synthetic biology sensors for biomolecule detection. In this study, freeze-dried, cell-free synthetic biology circuits were successfully integrated into wearable materials, enabling the non-invasive detection of various biomolecules, chemicals and pathogens, including SARS-CoV-2, without the need for living cells or complex instrumentation. Article PubMed Google Scholar Heng, W. et al. A smart mask for exhaled breath condensate harvesting and analysis. This work demonstrates continuous breath condensate sampling and accurate electrochemical monitoring of metabolites and inflammatory biomarkers in exhaled breath, offering practical applications in both daily life and clinical settings. Google Scholar Ye, Y. et al. Smart contact lens with dual-sensing platform for monitoring intraocular pressure and matrix metalloproteinase-9. Google Scholar Kim, J. et al. Wearable smart sensor systems integrated on soft contact lenses for wireless ocular diagnostics. Google Scholar Keum, D. H. et al. Wireless smart contact lens for diabetic diagnosis and therapy. Google Scholar Sempionatto, J. R. et al. Eyeglasses-based tear biosensing system: non-invasive detection of alcohol, vitamins and glucose. Google Scholar Seo, H. et al. Smart contact lenses as wearable ophthalmic devices for disease monitoring and health management. Google Scholar Park, W. et al. In-depth correlation analysis between tear glucose and blood glucose using a wireless smart contact lens. In this study, a wireless smart contact lens was successfully developed and validated for continuous, real-time monitoring of tear glucose, establishing a strong correlation with blood glucose through the concept of personalized lag time across multiple species and diabetic conditions. Google Scholar Kalantar-Zadeh, K. et al. A human pilot trial of ingestible electronic capsules capable of sensing different gases in the gut. This article demonstrates a human pilot trial of ingestible electronic capsules capable of sensing oxygen, hydrogen and carbon dioxide in the gut, providing real-time data on gastrointestinal gas profiles and transit times. Google Scholar Belknap, R. et al. Feasibility of an ingestible sensor-based system for monitoring adherence to tuberculosis therapy. PLoS ONE 8, e53373 (2013). Google Scholar Siddiqui, I., Majid, H. & Abid, S. Update on clinical and research application of fecal biomarkers for gastrointestinal diseases. World J. Gastrointest. Google Scholar Steiger, C. et al. Ingestible electronics for diagnostics and therapy. Article PubMed Google Scholar De la Paz, E. et al. A self-powered ingestible wireless biosensing system for real-time in situ monitoring of gastrointestinal tract metabolites. Google Scholar Mimee, M. et al. An ingestible bacterial–electronic system to monitor gastrointestinal health. This study reports on the development of an ingestible micro-bio-electronic device that combines engineered probiotic bacteria with miniaturized electronics for the in situ detection of gastrointestinal biomarkers, as validated in both in vitro and in vivo porcine models. Google Scholar Inda-Webb, M. E. et al. Sub-1.4 cm3 capsule for detecting labile inflammatory biomarkers in situ. Article PubMed Google Scholar Li, J. et al. A tissue-like neurotransmitter sensor for the brain and gut. Google Scholar Wang, L. et al. Functionalized helical fibre bundles of carbon nanotubes as electrochemical sensors for long-term in vivo monitoring of multiple disease biomarkers. Article PubMed Google Scholar Li, R. et al. A flexible and physically transient electrochemical sensor for real-time wireless nitric oxide monitoring. Google Scholar Ferguson, B. S. et al. Real-time, aptamer-based tracking of circulating therapeutic agents in living animals. Google Scholar Arroyo-Currás, N. et al. Real-time measurement of small molecules directly in awake, ambulatory animals. Natl Acad. Google Scholar Holmström, N., Nilsson, P., Carlsten, J. & Bowald, S. Long-term in vivo experience of an electrochemical sensor using the potential step technique for measurement of mixed venous oxygen pressure. Article PubMed Google Scholar Theuns, D. A. M. J. et al. Prognostic role of high‐sensitivity C‐reactive protein and B‐type natriuretic peptide in implantable cardioverter‐defibrillator patients. Pacing Clin. Article PubMed Google Scholar Abbassy, M. et al. Biosensors with left ventricular assist devices. Heart Fail. Google Scholar Neubeck, L. et al. The mobile revolution—using smartphone apps to prevent cardiovascular disease. Article PubMed Google Scholar Xu, C. et al. A physicochemical-sensing electronic skin for stress response monitoring. Google Scholar Torrente-Rodríguez, R. M. et al. Investigation of cortisol dynamics in human sweat using a graphene-based wireless mHealth system. Google Scholar Song, Y. et al. 3D-printed epifluidic electronic skin for machine learning-powered multimodal health surveillance. Google Scholar Sundhoro, M. et al. Rapid and accurate electrochemical sensor for food allergen detection in complex foods. Google Scholar Criscuolo, F., Cantù, F., Taurino, I., Carrara, S. & De Micheli, G. A wearable electrochemical sensing system for non-invasive monitoring of lithium drug in bipolar disorder. IEEE Sens. Google Scholar Yang, Y. et al. A laser-engraved wearable sensor for sensitive detection of uric acid and tyrosine in sweat. Article PubMed Google Scholar Alipour, A., Gabrielson, S. & Patel, P. B. Ingestible sensors and medication adherence: focus on use in serious mental illness. Google Scholar Kane, J. M. et al. First experience with a wireless system incorporating physiologic assessments and direct confirmation of digital tablet ingestions in ambulatory patients with schizophrenia or bipolar disorder. Article PubMed Google Scholar Sempionatto, J. R., Lasalde-Ramírez, J. A., Mahato, K., Wang, J. & Gao, W. Wearable chemical sensors for biomarker discovery in the omics era. Google Scholar Xu, C., Solomon, S. A. & Gao, W. Artificial intelligence-powered electronic skin. Google Scholar Flynn, C. D. & Chang, D. Artificial intelligence in point-of-care biosensing: challenges and opportunities. Google Scholar Rodbard, D. Continuous glucose monitoring: a review of recent studies demonstrating improved glycemic outcomes. Diabetes Technol. Article PubMed Google Scholar Soto, R. J., Hall, J. R., Brown, M. D., Taylor, J. & Schoenfisch, M. H. In vivo chemical sensors: role of biocompatibility on performance and utility. Google Scholar Chang, D. et al. A high-dimensional microfluidic approach for selection of aptamers with programmable binding affinities. Article PubMed Google Scholar Zhu, Y. et al. Lab-on-a-contact lens: recent advances and future opportunities in diagnostics and therapeutics. Google Scholar Smith, J. L. & Rice, M. J. Why have so many intravascular glucose monitoring devices failed? Sideri, K. et al. Digital pills for the remote monitoring of medication intake: a stakeholder analysis and assessment of marketing approval and patent granting policies. Law Biosci. De Miguel Beriain, I. & Morla González, M. ‘Digital pills' for mental diseases: an ethical and social analysis of the issues behind the concept. Law Biosci. Tadikonda, S. HealthVerity: Real World Data and Evidence. Case 824-019 (Harvard Business School, 2023). Galindo, R. J. et al. Continuous glucose monitors and automated insulin dosing systems in the hospital consensus guideline. Tu, J. & Gao, W. Ethical considerations of wearable technologies in human research. Zheng, H. et al. Reverse iontophoresis with the development of flexible electronics: a review. Article PubMed Wang, C. et al. A microfluidic wearable device for wound exudate management and analysis in human chronic wounds. Article PubMed Shibasaki, K., Kimura, M., Ikarashi, R., Yamaguchi, A. & Watanabe, T. Uric acid concentration in saliva and its changes with the patients receiving treatment for hyperuricemia. Daum, K. M. & Hill, R. M. Human tear glucose. Min, J. et al. Continuous biochemical profiling of the gastrointestinal tract using an integrated smart capsule. Barton, M. & Yanagisawa, M. Endothelin: 30 years from discovery to therapy. Article PubMed Tektonidou, M. G. & Ward, M. M. Validation of new biomarkers in systemic autoimmune diseases. PRECISION study: evaluating the accuracy of the LabPatch continuous glucose monitor. A study of non-invasive measurement of blood glucose and blood pressure. Epicore Biosystems launches Discovery Patch® Sweat Collection System. Epicore Biosystems https://www.prnewswire.com/news-releases/epicore-biosystems-launches-discovery-patch-sweat-collection-system-301392407.html (2021). Law, R. Biolinq granted de novo classification for needle-free glucose monitor. Medical Device Network https://www.medicaldevice-network.com/news/biolinq-granted-de-novo-classification-for-needle-free-glucose-monitor (2025). Eversense E3 Continuous Glucose Monitoring (CGM) System – P160048/S021 FDA https://www.fda.gov/medical-devices/recently-approved-devices/eversense-e3-continuous-glucose-monitoring-cgm-system-p160048s021 (2023). Gao, W. et al. Fully integrated wearable sensor arrays for multiplexed in situ perspiration analysis. Das, J. et al. Reagentless biomolecular analysis using a molecular pendulum. This article introduces a novel molecular pendulum sensing mechanism capable of the reagentless detection of diverse protein biomarkers in multiple biofluids, enabling continuous real-time monitoring and in vivo measurements. Article PubMed Zargartalebi, H. et al. Capillary-assisted molecular pendulum bioanalysis. Article PubMed Rivnay, J. et al. Organic electrochemical transistors. Lu, Z. et al. Biomolecule sensors based on organic electrochemical transistors. Koh, A. et al. A soft, wearable microfluidic device for the capture, storage, and colorimetric sensing of sweat. Lee, K. H. et al. Synergistic SERS enhancement in GaN–Ag hybrid system toward label-free and multiplexed detection of antibiotics in aqueous solutions. Xu, K., Zhou, R., Takei, K. & Hong, M. Toward flexible surface-enhanced Raman scattering (SERS) sensors for point-of-care diagnostics. Wang, Y. et al. Wearable plasmonic-metasurface sensor for noninvasive and universal molecular fingerprint detection on biointerfaces. Lin, L. & Wang, L. V. The emerging role of photoacoustic imaging in clinical oncology. Article PubMed Zargartalebi, H. et al. Active-reset protein sensors enable continuous in vivo monitoring of inflammation. This article introduces an active-reset methodology that enables receptor regeneration through the application of an alternating electric potential and facilitates continuous protein monitoring. Article PubMed Sun, N. et al. Aptamer melting biosensors for thousands of signaling and regenerating cycles. Article PubMed This project was supported by the National Science Foundation (grants 2145802 and 2444815), National Institutes of Health (grants R01HL155815, R01HL165002 and R21DK13266), Army Research Office (grant W911NF-23-1-0041), American Cancer Society Research Scholar (grant RSG-21-181-01-CTPS), US Army Medical Research Acquisition Activity (grant HT9425-24-1-0249), Advanced Research Projects Agency for Health Sprint for Women's Health (award number ARPA-H-ICHUB-24-101-504), Heritage Medical Research Institute, Natural Sciences and Engineering Research Council of Canada and Chan Zuckerberg Biohub Chicago. These authors contributed equally: Jiaobing Tu, Connor D. Flynn. Andrew and Peggy Cherng Department of Medical Engineering, Division of Engineering and Applied Science, California Institute of Technology, Pasadena, CA, USA Jiaobing Tu, Jeonghee Yeom & Wei Gao Department of Chemistry, Weinberg College of Arts and Sciences, Northwestern University, Evanston, IL, USA Connor D. Flynn & Shana O. Kelley Chan Zuckerberg Biohub Chicago, Chicago, IL, USA Connor D. Flynn, Zhenwei Wu & Shana O. Kelley Department of Biomedical Engineering, McCormick School of Engineering, Northwestern University, Evanston, IL, USA Zhenwei Wu & Shana O. Kelley Department of Pharmaceutical Sciences, Leslie Dan Faculty of Pharmacy, University of Toronto, Toronto, Ontario, Canada Department of Biochemistry and Molecular Genetics, Feinberg School of Medicine, Northwestern University, Evanston, IL, USA International Institute for Nanotechnology, Northwestern University, Evanston, IL, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Shana O. Kelley or Wei Gao. The authors declare no competing interests. Nature Nanotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Tu, J., Flynn, C.D., Yeom, J. et al. Wearable biomolecular sensing nanotechnologies in chronic disease management. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09350-y'>Population-specific polygenic risk scores for people of Han Chinese ancestry</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:28:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Predicting complex disease risks on the basis of individual genomic profiles is an advancing field in human genetics1,2. However, most genetic studies have focused on populations of European ancestry, creating a global imbalance in precision medicine and underscoring the need for genomic research in non-European groups3,4. The Taiwan Precision Medicine Initiative recruited more than half a million Taiwanese residents, providing a large dataset of genetic profiles and electronic medical record data for people with Han Chinese ancestry. Using extensive phenotypic data, we conducted comprehensive genomic analyses across the medical phenome with individuals genetically similar to Han Chinese reference populations. These analyses identified population-specific genetic risk variants and new findings for various complex traits. We developed polygenic risk scores, demonstrating strong predictive performance for conditions such as cardiometabolic diseases, autoimmune disorders, cancers and infectious diseases. We observed consistent findings in an independent dataset, Taiwan Biobank, and among people of East Asian ancestry in the UK Biobank and the All of Us Project. Our approach of characterizing the phenome-wide genomic landscape, developing population-specific risk-prediction models, assessing their performance and identifying the genetic effect on health serves as a model for similar studies in other diverse study populations. A principal promise of modern genetics is the ability to predict complex disease risk on the basis of a person's genetic profile. If successful, health management strategies can be developed to mitigate risk (disease prevention) and to optimize care (early diagnosis and effective treatment). Large-scale studies by the UK Biobank (UKB) and the Electronic Medical Records and Genomics Network show that risk prediction on the basis of genetics holds promise, and several countries are exploring ways to implement risk-based management in clinical practice1,2. Using polygenic risk scores (PRS) to predict disease risk and identify individuals at high risk is an emerging ‘precision medicine' approach to leverage genetic findings in clinical practice. However, a substantial limitation is that current PRS models are predominantly based on genome-wide association studies (GWAS) with participants of European ancestry (EUR)4,5, often leading to reduced predictive performance in groups of other ancestry6,7. To fully realize the potential of precision medicine for diverse global populations, population-specific phenome-wide genomic discovery must be performed at scale and clinically applicable polygenic risk models must be optimized in and across populations. To fill this research gap in a population of East Asian ancestry (EAS), we characterized the complex genetic architecture of the population of Han Chinese ancestry phenome wide, developed population-specific PRS and assessed the external validity of the models across populations with varying degrees of genetic similarity. Although several biobanks have been built to recruit subjects from East Asia, they have moderate sample size (72,000–212,000), and many focus on specific conditions8,9,10,11,12. By contrast, biobanks with predominantly EUR participants13,14,15,16 have significantly larger sample sizes (224,000–635,000) and access to more comprehensive clinical data. The moderate sample size and limited phenotypes in existing EAS biobanks hamper discovery of unique genetic effects and preclude the development of robust and clinically useful PRS models for EAS. We assembled a large non-EUR cohort, the Taiwan Precision Medicine Initiative (TPMI), and genotyped more than half a million participants across 16 medical centres in Taiwan from 2019 to 2023. All the participants, who are overwhelmingly of Han Chinese ancestry, contributed DNA samples for genetic profiling with a custom-designed genotyping array and consented to provide their longitudinal electronic medical records (EMRs) from 5 years before enrolment and into the future. Here we present the results of comprehensive genomic analyses with extensive genetic and medical data derived from the TPMI cohort, including phenome-wide GWAS and PRS model development. We identified numerous population-specific risk variants/genes, observed evidence of genetic pleiotropy and pinpointed clusters of traits that shared similar genetic aetiology. Then, we developed and validated PRS prediction models for numerous conditions against external datasets including those from the Taiwan Biobank (TWB), the UKB and the All of Us Project. Our results show the benefits of leveraging a large cohort from an understudied population to identify unique genetic underpinnings of the human phenome, interpret causal effects by means of fine mapping and colocalization and improve the performance of population-specific PRS models, which together better illuminate the clinical implications of genetic risk. We performed comprehensive genomic analyses, including GWAS, heritability estimation and PRS model building and evaluation, across a wide range of diseases and quantitative traits using 463,447 individuals genetically similar to Han Chinese reference populations from TPMI. We examined 695 dichotomized phenotypes (phecodes; case n > 2,000) and 24 quantitative traits (sample size > 100,000), spanning numerous disease categories (defined by phecode groupings18,19), such as neoplasms, metabolic disorders, circulatory conditions, autoimmune diseases and more (Fig. The phecodes, derived from International Classification of Diseases codes18,19, alongside quantitative traits such as blood pressure, body mass index (BMI), liver enzymes and lipid levels, provide a robust dataset for exploring genetic contributions to human health (Supplementary Tables 1 and 2). The log-transformed case proportion identified from EMRs showed a moderate but significant correlation with the log-transformed 5-year disease prevalence from the National Health Insurance Research Database (NHIRD) in Taiwan20 (r = 0.656, P = 2.69 × 10−84) (Fig. 1), indicating that the TPMI's hospital-based design may not fully capture mild and common illnesses, which are primarily observed in local and primary care clinics. Our GWAS identified at least one significant locus (P < 5 × 10−8) for 265 phecodes of the 695 tested and all 24 quantitative traits. Highlighting the robustness of the TPMI data, we observed a high replication rate of reported disease loci from EAS GWAS on the GWAS catalogue (actual/expected ratio (AER) = 78.17%, considering the statistical power with the published tool, PGRM21), particularly for endocrine and metabolic/hematopoietic diseases (AER = 88.68% and 84.62%, respectively; Extended Data Fig. Lower replication rates for respiratory disease (AER = 23.53%) may reflect limited case numbers; untyped genetic variants, such as rare variants, copy number variation and structural variants; or recruitment bias such as age distribution. We applied the sum-of-single-effects model for fine mapping to identify the independent variant–trait associations and reported the genetic variant with highest posterior inclusion probability of identified credible sets as well as the single lead variant for the failed fine-mapping regions and major histocompatibility complex region (MHC region; chromosome 6: 25,391,792–33,424,245). Our analyses showed a total of 2,656 fine-mapping-identified independent association signals, including 1,309 from phecodes GWAS and 1,347 from quantitative traits. Notably, 95 new associations, defined as having no previously reported results within 1 Mb in the NHGRI-EBI GWAS Catalogue21 of relevant GWAS, were identified across 50 phecodes and seven quantitative traits. In addition, we identified 217 new hits from previously reported regions, defined as having low linkage disequilibrium (r2 < 0.1) with any variant observed in the NHGRI-EBI GWAS Catalogue within 1 Mb for the same phenotype (Supplementary Tables 4 and 5). After applying the multiple testing correction, 1,502 fine-mapped associations passed a Bonferroni-adjusted threshold (5 × 10−8/(695 + 24) = 6.95 × 10−11), as well as 21 previously unreported variants and 115 new hits. Of the 95 new genetic associations, 30 variants are rare (minor allele frequency (MAF) < 0.05) in populations with other ancestry (African or African American (AFR), Admixed American (AMR), South Asian (SAS) and non-Finnish European) in the Genome Aggregation Database (gnomAD)) and 33 variants less than 0.01 in EUR, the most extensively studied population, which explains why they were not reported in previous GWAS. For example, single nucleotide polymorphism (SNP) rs17089782, a missense variant in PIBF1 (p.R405Q) on chromosome 13 is significantly associated with thyroid cancer (P = 2.8 × 10−9) in the TPMI cohort. However, PIBF1 is essential for immune regulation, especially during pregnancy, and is relevant to autoimmune diseases and cancer22. This gene, highly expressed in the nervous system, had previously been linked to obesity hypoventilation syndrome in a small study (n = 30)23 and associated with bone mineral density24. In addition, when we compare the effect size in TPMI and UKB for the rest of the new findings, 25 exhibit a significant different effect size (P < 0.05). For instance, a TPMI-identified platelet-count-associated variant, rs12955741, located in the intergenic region between TGIF1 and DLGAP1, exhibits a different effect size (β) compared to that in UKB (βTPMI = 0.044, βUKB = −0.005, P = 1.7 × 10−9). Moreover, the high hepatitis B virus carrier rate in Taiwan25 contrasts sharply with its rarity in European cohorts, enabling TPMI to identify new loci associated with viral hepatitis B (case number in TPMI = 23,618 versus UKB = 132). Among the 26 independent loci identified in our analysis of hepatitis B, 19 fine-mapped loci are new (Extended Data Fig. Notably, 18 of these 19 loci were found to be associated with liver function or diseases (Supplementary Table 5). These new associations highlight the uniqueness of certain disease loci in the TPMI cohort, presenting opportunities for developing population-specific therapeutic interventions and advancing precision medicine. All identified independent associations are summarized in Fig. The identification of the MHC region as a significant hotspot on chromosome 6 emphasizes its extensive involvement in immune-related diseases across several categories. Similarly, the short arm of chromosome 11 (INS-KCNQ1 region) also affect various traits, including metabolic, endocrine and genitourinary diseases. These hotspots of trait-relevant variants implied the shared genetic mechanism among diseases and potential of pleiotropic effects. Vertical bars show the accumulated number of independent variant–trait associations for dichotomized phecodes (top) and quantitative traits (bottom). Each category of diseases and traits is represented by a corresponding colour. Linkage disequilibrium score regression analysis (LDSC)26 showed strong liability-scaled SNP heritability (h2) for conditions such as alcoholism (h2 = 0.213), retention of urine (h2 = 0.163) and open-angle glaucoma (h2 = 0.160). Among quantitative traits, body height (h2 = 0.323), BMI (h2 = 0.218) and high-density lipoprotein cholesterol (h2 = 0.191) exhibited the highest heritability estimates (Supplementary Table 6 and Supplementary Information), highlighting the significant role of genetics in these traits. These results have far-reaching implications for precision medicine, as higher heritability signals indicate the potential for more accurate genetic risk-prediction models that could improve personalized disease risk assessments. We then partitioned heritability at the gene level and identified 329 unique genes contributing significantly to phenotypic variation (h2 > 0.1% and Z-score > 1.64). Of these, 45 affected more than one phecode and/or quantitative category, including key genes such as APOE, APOC1, TOMM40, ABCG2 and KCNQ1 (Fig. We also conducted a colocalization analysis to elucidate the potential molecular function of identified GWAS signals with three expression quantitative traits locus (eQTL) datasets, including the Genotype-Tissue Expression Project (GTEx)27, Multi-ancestry Analysis of Gene Expression (MAGE)28 and the Japan COVID-19 Task Force (JCTF)29 (Fig. Our results identified 391 unique genes that potentially mediate the outcome through their expression level (posterior probability > 0.9), including GBAP1, which colocalized with five different traits (uric acid, serum creatinine, hematocrit, hypertension and gout). Among the colocalized genes, 75 of them can be identified only in the multi-ancestry lymphoblastoid cell lines eQTL (MAGE; 20 genes) and/or Japanese whole blood eQTL (JCTF; 59 genes). Our findings demonstrate the effect of these genes (such as APOE, ABCG2 and KCNQ1) on several traits and disorders. By elucidating shared genetic effects, these results offer opportunities to develop precision medicine approaches that address comorbidities, such as treating hyperlipidaemia and reducing dementia risk through a single intervention targeting APOE, which influences both lipid metabolism and Alzheimer's disease risk. This gene-level understanding emphasizes the potential to optimize therapeutic strategies by leveraging genetic pleiotropy in disease management. a,b, Circle plots showing gene-level heritability and colocalization (coloc) with gene expression for phecodes (a), summarized in parent (integer) phecodes, and quantitative traits (b). Dots represent gene-level h2 > 10−3; squares indicate colocalization posterior probability (PP) for the hypothesis that both traits are associated and share a single causal variant (H4) > 0.9; and triangles show both. Inner circle indicates the number of associated traits for each identified gene. The bar chart shows the number of identified genes by category and grouped by type of pleiotropy. The cluster of autoimmune and infectious diseases, which includes viral hepatitis B, psoriasis and systemic lupus erythematosus, illuminates shared immune system pathways and potential gene–pathogen interaction. The kidney-related cluster involved gout, chronic kidney disease, calculus of kidney and ureter, ankylosing spondylitis and measures of urea nitrogen, creatinine and uric acid. Therefore, these findings demonstrate the importance of population-specific genetic studies, as differences in genetic architectures between populations can significantly affect the accuracy of PRS models. Building on these insights, we developed and validated PRS models that demonstrated strong predictive performance for a wide range of diseases. Although we used five PRS tools, including LDpred231, Lassosum232, PRS-CS33, SBayesR34 and MegaPRS35 (Supplementary Tables 9–13), we found that LDpred2 outperformed the others for most traits (Extended Data Fig. Therefore, we took the results of LDpred2 for further comparisons. Of the 265 PRS models for phecodes, area under the receiver operating characteristic curve (AUC) values exceeded 0.55 for 105 dichotomized phecodes with a significant P value (P < 0.05). Additionally, the explained variance of models for 24 quantitative traits ranged from 0.028 (aspartate aminotransferase) to 0.227 (height). The most predictive PRS models included highly heritable traits such as ankylosing spondylitis (AUC = 0.812 ± 0.016), psoriasis (0.709 ± 0.016), atrial fibrillation (0.702 ± 0.014), prostate cancer (0.696 ± 0.018), systemic lupus erythematosus (0.696 ± 0.015), rheumatoid arthritis (0.646 ± 0.011), type 2 diabetes (0.640 ± 0.005), female breast cancer (0.611 ± 0.010) and hypertension (0.610 ± 0.004). Interestingly, the PRS for hepatitis B also demonstrated high genetic predictability (0.654 ± 0.008). Because h² represents the upper bound of variance that can be explained by PRS, we examined the proportion of heritability captured by our models (r2/h2). A total of 36 traits, including prostate cancer (r2/h2 = 0.054/0.070), type 2 diabetes (0.066/0.126) and high-density lipoprotein cholesterol (0.136/0.191), reached more than 50% of their SNP heritability, indicating that PRS can achieve near-optimal predictive accuracy for highly heritable traits. However, for complex diseases influenced by both genetic and environmental factors, PRS performance is inherently constrained by the fraction of heritability attributable to common variants. These findings reinforce the importance of SNP heritability as a reference point for evaluating PRS utility and highlight the need for larger, ancestrally diverse datasets to further enhance genetic prediction models (Extended Data Fig. Leveraging the identified clusters, we performed a multitrait PRS training, PRSmix+36, for the traits in each cluster (Fig. Notably, multitrait PRS models improved prediction accuracy for the cardiometabolic disease cluster with a 0.040 increase in AUC (from 0.608 to 0.648) and a 1.770-fold improvement in phenotypic variance explained (r2). The significant enhancement of multitrait PRS prediction (comparing r2 of LDpred2 and PRSmix+ with a paired t-test, P = 1.07 × 10−13) highlights the potential of leveraging shared genetic architecture to enhance disease risk prediction. Figure 5 demonstrates the performance of single-trait and multitrait PRS across three disease clusters, as well as the differing effectiveness of PRS in predicting genetic risk across various disease categories. Grey bars indicate SNP heritability (estimated from TPMI GWAS unrelated set (n = 248,754) with LDSC) and the coloured bars represent the r2 values, indicating the proportion of variance explained by the PRS among TPMI validation set (n = 20,000) from single-trait PRS (LDpred2, red bar) or multitrait PRS (PRSmix+, blue bar). The dot-and-whisker plots showcase predictive accuracy using AUC with 95% confidence intervals for dichotomized traits. Asterisks indicate h2 estimates that consider the MHC region. To evaluate the robustness and generalizability of our PRS models, we performed an external validation of the models (hypertension, type 2 diabetes, viral hepatitis B, gout, calculus of kidney from PRSmix+ and others from LDpred2) in TWB, unrelated individuals genetically similar to Han Chinese reference populations, n = 88,628), UKB (self-reported EAS, n = 9,893) and All of Us (genetically inferred EAS, n = 6,895). We found that the prediction accuracy, AUC, of our models ranged from 0.548 (glaucoma) to 0.712 (prostate cancer) in TWB, 0.557 (female breast cancer) to 0.634 (hypertension) in UKB and 0.520 (migraine) to 0.709 (gout) in All of Us (Extended Data Fig. Although the TWB questionnaire did not contain specific details on hepatitis B status, we used antihepatitis B core total antibodies (Anti-HBc) as an indicator of infection or past infection and hepatitis B surface-antigen (HBsAg) as a marker of acute/chronic infection. TPMI-derived PRS models perform better than the UKB EUR-derived models when applied to EAS for viral hepatitis B, type 2 diabetes, hypertension, gout and migraine. However, the overlapping confidence intervals in UKB and All of Us may be due to their limited sample size of EAS. These results indicate that population-specific PRS models allow for more accurate risk stratification and enable personalized healthcare interventions for EAS. Additionally, we assessed the performance of TPMI-derived PRS and TPMI-included cross-population PRS across various ancestry groups, including populations of EUR, AFR, AMR and SAS ancestry from the UKB and All of Us cohorts (Extended Data Fig. Performance varied by diseases, but consistent results were observed for female breast cancer and glaucoma across populations, and TPMI-included cross-population PRS slightly but not significantly improved in the populations other than EAS and EUR. Although overall health is hard to define with a few metrics, herein, we used the count of clinical visits and duration of hospitalization to roughly describe individuals' overall health. We found that 131 of the top-performing PRS models (LDpred2 models with AUC > 0.55 and all PRSmix+ models for phecodes and all models for quantitative traits) are significantly associated with overall health indices, explaining 8.47% of the variation in clinical visit frequency (P = 2.69 × 10−14) and 10.29% of the variation in hospitalization duration (P = 5.62 × 10−27; Extended Data Table 1 and Supplementary Table 15) in the comparison between top and bottom 5% groups after adjusting for sex, age and recruiting hospital. In short, quantification of the effect of PRS for various diseases and traits on human health opens up opportunities for developing precision health management strategies. This study represents a large-scale GWAS in the population of Han Chinese ancestry, using data of around 500,000 individuals recruited from 16 medical centres across Taiwan. We investigated the genetic architecture of 695 dichotomized phecodes and 24 quantitative traits, identifying 2,656 independent variant–trait associations and showed that population-specific genetic risk-prediction PRS models for a wide range of diseases performed well in the population. Indeed, for the traits with sufficient sample size in the cohort, PRS performance rivals those developed for EUR using UKB data. These findings show that population-specific PRS models can be developed successfully for populations of non-EUR, and our project serves as a model for large-scale genetic studies in other populations. Recent large-scale projects that emphasize ancestral diversity in human genetic studies have discovered new findings with the inclusion of subjects of non-EUR. MVP conducted multi-ancestry GWAS on 635,000 participants, identifying more than 2,000 signals unique to populations with non-EUR16. With the TPMI dataset, we performed larger GWAS in subjects genetically similar to Han Chinese reference populations for several traits than published studies. For instance, the previous largest meta-analysis for type 2 diabetes included 20,573 cases who were of Han Chinese ancestry37. By contrast, our GWAS included 59,289 cases of type 2 diabetes, almost tripling the number of cases ever tested and identified five unreported type 2 diabetes associated loci from known regions, demonstrating the power of TPMI sample size. Identification of new and population-specific risk variants may lead to further understanding of their molecular mechanism and underline the need for population-specific weightings in PRS models. Moreover, population-specific findings also better explain the performance of population-specific PRS models in the population in question. In short, our population-specific genomic profiles for comprehensive phenotypes provide a solid foundation for PRS development. Our understanding of how the genetic factors influencing hepatitis B, an endemic infectious disease in Taiwan with an estimated hepatitis B virus carrier rate of 9.78% among the unvaccinated cohort (born before 1984)25, also benefited from the large dataset. With 23,618 cases, a significant increase from previous studies of only a few thousand cases38,39,40, we identified 26 fine-mapped signals, including 19 new loci, and showed a significant negative correlation between hepatitis B and other autoimmune diseases, such as Sicca syndrome, psoriasis and systemic lupus erythematosus. Our well-performed and validated PRS model for hepatitis B demonstrated that the host genome may determine the severity and symptoms of this infectious disease. This is similar to that previously reported in COVID-19 and pneumonia, where genetic factors have been shown to influence disease outcomes41,42,43,44. Our unexpected success of GWAS and PRS for hepatitis B not only demonstrates the power of the large sample size of TPMI but also shows the necessity of population-specific genetic study for population-enriched diseases. The benefits extend beyond differences in ancestry to include environmental factors such as pathogen exposure, food intake and lifestyle influences. Exploring how the human genome interacts with these diverse external and environmental factors can greatly enhance our understanding of how genetic variants contribute to disease susceptibility or severity. In addition, the comprehensive phenotypic data allow us to investigate the genetic correlation among several traits that have substantial implications for clinical applications and leverage them to improve the performance of PRS models. By identifying shared genetic risks across diseases, at-risk individuals can be alerted to pursue early detection of comorbidities and targeted prevention strategies. For example, the clustering of cardiometabolic traits, such as type 2 diabetes, hypertension and BMI, highlights their interconnected genetic basis and indicates that individuals with a high genetic risk for one condition may benefit from early screening and intervention for related conditions45. Additionally, the shared genetic architecture allows the development of multitrait PRS models that integrate genetic risks across correlated traits, improving prediction accuracy and enabling precision medicine approaches that address several health outcomes simultaneously46. Although previous studies have proven the utility of multitraits on target diseases36,47,48, we have extended the use this approach on a phenome-wide level and demonstrated the improvement across different types of traits. We evaluated our PRS models across several large cohorts, including the TWB, UKB and All of Us. The TPMI-derived PRS models consistently outperformed those developed from EUR when applied to diseases in people with Han Chinese or EAS ancestry from the three large cohorts. When comparing with EUR-derived PRS models, we also observed better performance across several traits in EAS, particularly for cardiometabolic and autoimmune diseases. Similarly, the TPMI-included cross-population model slightly improves performance in populations of other ancestries. These results highlight the need for population-specific models and emphasize the importance of genetic data from diverse populations to advance cross-population models. By integrating these well-developed PRS models, we estimate that genetics account for 10.3% of variation of hospitalization duration in TPMI. Although the estimates of genetic contributions to health measure may be influenced by disease prevalences and ascertainment biases, our result indicate integrating genetic risk-based health management strategies with traditional risk factors, such as age, sex, smoking and BMI, may enhance prediction models and refine personalized risk stratification. As with other large-scale epidemiological studies, ascertainment bias is also observed in our study. TPMI's case proportion shows a significant but moderate correlation with the prevalence from NHIRD, implying potential ascertainment bias of TPMI's hospital-based design. Compared to the general population in Taiwan (NHIRD), TPMI participants are overrepresented in the middle-aged group (year of birth 1940–1970, 54.3% versus 38.3%), include slightly more females (55.1% versus 50.6%) and have a higher proportion of participants from northern Taiwan (59.5% versus 47.3%). These demographic differences, along with the volunteer-based recruitment process, probably contribute to the lower case proportions observed in TPMI (Fig. Importantly, a significant portion of disease records in the NHIRD originate from local clinics and primary care settings, which are not covered in TPMI. Notably, the EMRs of the participants are incomplete, as some participants receive care from several health providers, but the TPMI only has access to EMRs from their enrolment hospitals. We acknowledge that ascertainment biases may influence disease prevalence and heritability estimates. Thus, we accounted for case-control ascertainment by applying liability-scale transformations using population prevalence data from NHIRD and used independent validation for PRS (TWB, UKB and All of Us) to mitigate the effect of ascertainment biases, while acknowledging that residual biases may persist. Methods like inverse probability weighting could mitigate such biases49, but these require detailed external reference data that are unavailable at present. Additionally, we observed a relatively low estimated heritability for body height and BMI in TPMI, compared to values reported in the literature50,51. These estimates may be affected by factors such as inconsistencies in assessment across EMRs, variations in statistical approaches and reduced bias of assortative mating in TPMI population52,53,54. These limitations also emphasize the need for future adjustments to enhance generalizability. In addition to ascertainment bias, our study has other commonly found limitations. First, the TPMI cohort size is not sufficiently large to study some of the severe subtypes of many diseases, such as diabetes insipidus and neurofibromatosis. Second, we attempted to use eQTLs to elucidate the molecular mechanism of diseases, but the underrepresentation of EAS in current eQTL datasets, such as GTEx, poses challenges27. Gene expression regulation varies across ancestries55,56, and differences in LD structures further complicate colocalization analyses. Therefore, ancestral diversity is an urgent need not only in genomic data but also in transcriptomic, proteomic, metabolomic and epigenomic datasets. Third, the current project retrieved EMRs from an average of 5 years before enrolment, so some important data such as age of disease onset for the older participants are not available. Incomplete EMRs lead to less precise case definition of some participants. Fourth, some of the younger participants have high-risk genetic profiles but are disease free for those diseases. The duration of the project is too short to determine whether they will eventually develop those diseases. An effort is underway to gain access to the complete EMRs of the TPMI participants and to recruit more participants with severe subtypes of common diseases. Future studies are being planned to study the high-risk individuals who escape disease development to identify genetic and non-genetic factors that mitigate their disease risk. Furthermore, the meta-analysis integrating TPMI with other large-scale EAS biobanks, such as TWB11,12, Korean Genome and Epidemiology Study9, China Kadoorie Biobank10 and Biobank Japan8, may further enhance our understanding of the genetic aetiology in EAS and improve prediction models. This study demonstrates that population-specific risk-prediction models, such as those developed for EAS in this work, can achieve strong predictive performance for traits with high relevance in that population. The PRS we developed for EAS performed well for several traits, including diseases with significant public health implications, such as type 2 diabetes and systemic lupus erythematosus. However, for certain traits, such as female breast cancer and glaucoma, PRS derived from both UKB and TPMI performed comparably, indicating that the genetic architecture of some traits allows for generalizable models. These findings emphasize the importance of developing and validating PRS models in diverse global populations to maximize their utility and equity in genetic risk prediction. Although our results emphasize the utility of developing population-specific PRS, further research is needed to directly compare their performance with multipopulation models and assess their generalizability and to assess their effect on disease prevention and management. In particular, longitudinal studies and real-world implementations will be critical to determine the extent to which PRS-guided interventions can delay disease onset or improve health outcomes. Furthermore, it is hoped that if all can obtain their genetic profiles and determine their risk for major diseases, many diseases can be prevented or their onset can be delayed significantly, thereby fulfilling the promise of modern genetics. In conclusion, we used a large-scale dataset of individuals genetically similar to Han Chinese reference populations produced by the TPMI to conduct phenome-wide genetic analyses and leverage these genetic findings to train risk-prediction models for several diseases and traits. The developed models are validated in EAS of different biobanks and demonstrate a consistent performance that bodes well for their use in populations of Han Chinese and EAS ancestry. Our approach can serve as a template for developing PRS models in populations that are currently without such resources, anticipating the time when all populations around the world can benefit from risk-based health management as part of the precision health movement. We used the TPMI dataset, which links extensive EMRs with genotypic data for 486,956 individuals. Dichotomized disease status was defined by phecodes, which were based on information extracted from the EMR using International Classification of Diseases codes18,19. To ensure robustness, cases were defined by having the diagnosis of the relevant condition on two or more clinical visits. In this study, we focused on 695 phecodes that had at least 2,000 cases and 24 quantitative traits that were measured in at least 100,000 individuals. These phecodes spanned 16 disease categories, including but not limited to infectious diseases, neoplasms, endocrine/metabolic disorders and circulatory system diseases. The 24 quantitative traits were categorized into anthropometric, circulatory, hematological, kidney-related, liver-related and metabolic measurements. We performed genotyping using two customized high-density Axiom SNP arrays produced by Thermo Fisher, TPMv1 and TPMv2. The raw genotypic data underwent quality control measures, and the genetic variants were excluded when they had a call rate less than 0.98, MAF < 0.01, or Hardy–Weinberg equilibrium test P < 1 × 10−6. We also excluded individuals with overall call rate less than 0.95, failed heterozygosity check, or inconsistent documented versus genetically determined sex. For this study, we only included the genetic variants found on both genotyping arrays and excluded variants with a significant batch effect in GWAS. The proportion of genetic ancestry was determined by ADMIXTURE60, and the projected principal component scores with 1000 Genomes as a reference panel were applied to determine individuals' ancestry61. As a result, 401,710 genetic variants and 463,447 Han Chinese participants passed all quality control measures and were used in the subsequent studies. Details are found in Supplementary Information and on GitHub (https://github.com/TPMI-Taiwan/tpmi-qc). Phasing was conducted on quality-control-passed genotype data with SHAPEIT562. Genome imputation was carried out with IMPUTE5 using a reference panel of 1,498 whole-genome-sequenced TWB subjects12,63. We also conducted postimputation quality control with exclusion criteria INFO score ≤ 0.7 and MAF ≤ 0.01. In addition, we also performed a chip-GWAS for minimizing the bias from different chips, resulting in a dataset of 8,046,864 well-imputed common genetic variants. We performed a principal component analysis (PCA) on the basis of genotyped variants to capture the effect of population structure. To diminish the effect of close relatives, the main PCA was conducted in a genetically unrelated subset, and other subjects were projected with the calculated PC weightings. Then these PCA scores were leveraged to accurately quantify the proportion of identity by descent and degree of relatedness. The maximum unrelated set was determined on the basis of these estimated degrees of relatedness. PC-AiR and PC-Relate were used for PCA and relatedness estimation, and PRIMUS was used for identifying the maximum unrelated set with the third degree as threshold64,65,66. To maximize the statistical power, we used a mixed-effect regression model to examine the association between genotype and outcome of interest, logistic regression for dichotomized phecode and linear regression for quantitative traits. The mixed-effect model accounted for relatedness among individuals by including a random effect for pairwise kinship. The model was also adjusted for key covariates, including age, sex, age2, interactions between age/age2 and sex, genotyping chip, enrolment hospital and ten genetic principal components to control for population stratification. SAIGE was applied for the mixed-effect model GWAS67. In the GWAS set, we selected an unrelated subset (n = 248,754) to perform GWAS using a generalized linear model with PLINK2, and we conducted 1:10 age, sex-matching for the traits with imbalanced case/control ratio (less than 1/20). These PLINK2 GWAS statistics were then used for heritability and genetic correlation estimation68. To systematically evaluate the performance of our GWAS, we leveraged a presummarized phenotype–genotype reference map69, which collected 5,879 genetic associations for 149 unique phecodes from 523 published GWAS, including 1,215 associations from EAS. We performed fine mapping to identify the independent GWAS signals in all genomic regions containing any variant with a P value less than 5 × 10−8 and plus or minus 1.5 Mb of the regional lead variant14, except the MHC region (chromosome 6: 25,391,792–33,424,245) because of its complex linkage disequilibrium structure. We used the reported 95% credible set to determine the independent signals, and up to ten signals were allowed for each region. The genome-wide significant threshold was applied for defining a credible set as an independent hit, and a further requirement of log Bayes factor > 2 was applied for the second hit. SuSiE was conducted for this summary statistics-based fine mapping with linkage disequilibrium derived from our imputation reference panel70, which reflects our study population's genetic architecture. We comprehensively compared our GWAS results with reported significant signals on the NHGRI-EBI GWAS Catalogue21, downloaded on 11 March 2024. The mapping of phecodes and quantitative traits to GWAS catalogue phenotypes is summarized in Supplementary Table 16. We classified a variant as new if the fine-mapped independent signal was not located within 1 Mb of any reported genome-wide significant association (P < 5 × 10−8) for the corresponding phenotype. Additionally, a variant was considered a new hit if the highest linkage disequilibrium r² was less than 0.1, with any reported significant association within 1 Mb. Associations derived from uncertain and umbrella phecodes were excluded, and for duplicated genetic variants or regions, we only reported the association with the smaller P value or from the phecode with the more specific definition. Finally, we used ANNOVAR to annotate the new variants with data from the RefSeqGene database (updated 17 August 2020)71,72. For the new variants, we explored their allele frequencies in non-Finnish European, AFR, SAS and AMR from gnomAD73. We also compared the effect size between TPMI and UKB with a t-test for investigating the ancestry-specific effect. For traits with a higher prevalence in our dataset (TPMI) than the population (NHIRD), we applied the equation from ref. For other traits, we used the adapted equation from ref. Additionally, we conducted LDSC to obtain pairwise genetic correlations to assess the similarity of genetic mechanisms between traits76. On the basis of the genetic correlation matrix, we used a hierarchical cluster analysis to identify groups of traits that share genetic mechanisms. We used the weighted pair group method with arithmetic mean for clustering, and the resulting cluster tree was used for group identification. Moreover, we estimate the genetic correlation across populations, TPMI and UKB, to demonstrate varied genetic architecture in different ancestry populations. For the UKB GWAS, we applied a generalized linear model from PLINK2 with the predefined phecode (https://github.com/umich-cphds/createUKBphenome) and corresponding baseline quantitative measures among the identified unrelated set (n = 378,544). We used both gene-level heritability estimation and colocalization analysis to map our GWAS findings to functional units, specifically genes. We estimated heritability for genes that overlapped with fine-mapped regions, where gene regions were defined as the gene body plus or minus 10 kb for gene-level heritability. Additionally, to illustrate the molecular functions of genes of interest, we used colocalization analysis to examine whether there are shared common genetic causal variants between tissue-specific gene expression and traits of interest. The posterior probabilities were used to evaluate colocalization between gene expression and the trait of interest. The preserved dataset of 100,000 unrelated TPMI subjects was split into two subsets, training (n = 80,000) and validation (n = 20,000), for PRS model building. Five popular PRS tools were used—LDpred231, Lassosum232, PRS-CS33, SBayesR34 and MegaPRS35—and the training subset was applied for parameter selection and model optimization if needed. LDpred2, PRS-CS and SBayesR assumed the effect of genetic variants following a mixture distribution with different predefined parameters and applied a Bayesian framework for distribution estimation. Lassosum2 used a penalized regression (LASSO) for weight estimating, and MegaPRS leveraged MAF and linkage disequilibrium for model building. The explained variance (r2) was used to evaluate the performance of PRS for quantitative traits75,79, and two indices, AUC and liability-scaled r2, were used for PRS of phecodes. Additionally, we include partial r² estimates, calculated using the R package rsq. To account for population stratification in cross-cohort predictions, we also report r² with PCs as covariates in Supplementary Tables 9–14. For AUC comparisons, we include a baseline model incorporating standard covariates (sex, age and PCs) to better assess the added predictive power of PRS. To further leverage the gene's pleiotropy and shared genetic mechanism among traits, we conducted multitrait PRS model building for the traits in the same genetic cluster based on pairwise genetic correlation identified in the previous step. We pooled all PRS models from the five tools for those identified traits and applied an elastic net regression to combine their weighting and find the most optimized model for the target trait. The cross-population PRS models were based on both TPMI and UKB European GWAS (https://pheweb.org/UKB-TOPMed/), and PRS-CSx was applied80. We conducted an external validation of our developed PRS using data from the TWB, EAS from UKB and All of Us. TWB is a community-based biobank, and it has recruited over 200,000 participants in Taiwan. Herein, we used 88,628 unrelated subjects (greater than third degree; we removed 5,242 overlapped individuals), who were genotyped with the Axiom customized chip TWB2 (equivalent to TPMv1); their genotyping quality control, phasing and imputation followed the same protocol as described above. The self-reported disease condition was queried from their baseline questionnaire, except for cancer. Because the study design of TWB excluded cancer patients at recruitment, we used both baseline and follow-up self-reporting data to define cancer cases and controls. UKB has enroled approximately 500,000 participants since 2006 and linked their genetic data with enriched phenotypic data. Their ancestral population was determined by self-reported ethnic background, such as self-reported Chinese as EAS (n = 1,572); white, British, Irish and any other white background as EUR (n = 472,869); Black or Black British, Caribbean, African and any other Black background as AFR (n = 8,074); and Asian or Asian British, Indian, Pakistani, Bangladeshi and any other Asian background as SAS (n = 9,893). All of Us intends to enrol more than 1 million participants in the United States and has released whole-genome genotyping data for approximately 312,000 participants as of the first quarter of 2024. The genetically confirmed EAS as well as other superpopulations and their linked EMR were used for validating our PRS models. Moreover, we compared the TPMI-derived PRS model with UKB-derived models to investigate the performance of population-specific PRS. The UKB-derived models were based on published UKB European GWAS (https://pheweb.org/UKB-TOPMed/), and LDpred2-auto was applied for model building. We evaluated the genetic effect on overall health measures. We used the number of clinical visits and the aggregate duration of hospitalization as overall health indices. Owing to collinearity among PRS for different traits, we used a partial least square-generalized linear model to extract components from the PRS of qualified traits with the R package plsRglm81. We then estimated the covariate-adjusted proportion of genetic contribution (r2) by comparing the full model with the null model, which included only covariates such as sex, age and hospital. We used a likelihood ratio test to obtain the significances of regression models. We selected covariate-matched controls from subjects without hospitalization records as the bottom group for hospitalization models. Written informed consent was obtained from the subjects in accordance with institutional requirements and the Declaration of Helsinki principles. All collected information was de-identified before statistical data analysis. The analysis with TWB was approved by Institutional Review Boards of Academia Sinica (AS-IRB-BM-19014), and the NHIRD analysis with the Health and Welfare Data Science Center (HWDC) was approved by Institutional Review Boards of Academia Sinica (AS-IRB-BM-23056). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All PRS models and GWAS results (summary statistics from SAIGE and PLINK) are available from the TPMI website (https://pheweb.ibms.sinica.edu.tw/). The UKB phecode GWAS was obtained from UK Biobank TOPMed-imputed PheWeb (https://pheweb.org/UKB-TOPMed/). A detailed description of data availability and the application process of TWB, NHIRD, UKB and All of Us can be found on their websites (TWB, https://www.biobank.org.tw/english.php; NHIRD, https://dep.mohw.gov.tw/DOS/cp-5119-59201-113.html; UKB, https://www.ukbiobank.ac.uk/; All of Us: https://www.researchallofus.org/). Code for genotyping quality control process and analysis is available at GitHub (https://github.com/TPMI-Taiwan/). Selection, optimization and validation of ten chronic disease polygenic risk scores for clinical implementation in diverse US populations. Thompson, D. J. et al. A systematic evaluation of the performance and properties of the UK Biobank Polygenic Risk Score (PRS) Release. Analysis of polygenic risk score usage and performance in diverse human populations. Martin, A. R. et al. Human demographic history impacts genetic risk prediction across diverse populations. Bentley, A. R., Callier, S. & Rotimi, C. N. Diversity and inclusion in genomic research: why the uneven progress? Smith, J. L. et al. Multi-ancestry polygenic risk score for coronary heart disease based on an ancestrally diverse genome-wide association study and population-specific optimization. Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases. & Lee, S. Genome-wide study on 72,298 individuals in Korean biobank data for 76 traits. Walters, R. G. et al. Genotyping and population characteristics of the China Kadoorie Biobank. A. et al. Taiwan Biobank: a rich biomedical research database of the Taiwanese population. Wei, C. Y. et al. Genetic profiles of 103,106 individuals in the Taiwan Biobank provide insights into the health and history of Han Chinese. Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. All of Us Research Program Genomics Investigators. Genomic data in the All of Us Research Program. Verma, A. et al. Diversity and scale: genetic architecture of 2068 traits in the VA Million Veteran Program. The Taiwan Precision Medicine Initiative provides a cohort for large-scale studies. Bastarache, L. Using phecodes for research with the electronic health record: from PheWAS to PheRS. Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data. Lin, L. Y., Warren-Gash, C., Smeeth, L. & Chen, P. C. Data resource profile: the National Health Insurance Research Database (NHIRD). The NHGRI-EBI GWAS catalog: knowledgebase and deposition resource. Performance of multigene testing in cytologically indeterminate thyroid nodules and molecular risk stratification. He, D. et al. A longitudinal genome-wide association study of bone mineral density mean and variability in the UK Biobank. Chang, K. C. et al. Survey of hepatitis B virus infection status after 35 years of universal vaccination implementation in Taiwan. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Taylor, D. J. et al. Sources of gene expression variation in a globally diverse human cohort. The whole blood transcriptional regulation landscape in 465 COVID-19 infected samples from Japan COVID-19 Task Force. Brown, B. C., Asian Genetic Epidemiology Network Type 2 Diabetes Consortium, Ye, C. J., Price, A. L. & Zaitlen, N. Transethnic genetic-correlation estimates from summary statistics. Prive, F., Arbel, J., Aschard, H. & Vilhjalmsson, B. J. Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores. & Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Improved polygenic prediction by Bayesian multiple regression on summary statistics. & Speed, D. Improved genetic prediction of complex traits from individual-level data or summary statistics. Truong, B. et al. Integrative polygenic risk score improves the prediction accuracy of complex traits and diseases. Identification of type 2 diabetes loci in 433,540 East Asian individuals. Chang, S. W. et al. A genome-wide association study on chronic HBV infection and its clinical progression in male Han-Taiwanese. Zeng, Z. et al. Genome-wide association study identifies new loci associated with risk of HBV infection and disease progression. Li, Y. et al. Genome-wide association study identifies 8p21.3 associated with persistent hepatitis B virus infection among Chinese. A second update on mapping the human genetic architecture of COVID-19. A first update on mapping the human genetic architecture of COVID-19. Human genetic variants identified that affect COVID susceptibility and severity. Kember, R. L. et al. Polygenic risk scores for cardiometabolic traits demonstrate importance of ancestry for predictive precision medicine. Khunsriraksakul, C. et al. Multi-ancestry and multi-trait genome-wide association meta-analyses inform clinical risk prediction for systemic lupus erythematosus. & Wallace, C. shaPRS: leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores. Zhai, S., Guo, B., Wu, B., Mehrotra, D. V. & Shen, J. Integrating multiple traits for improving polygenic risk prediction in disease and pharmacogenomics GWAS. Participation bias in the UK Biobank distorts genetic associations and downstream analyses. Wainschtein, P. et al. Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data. Yengo, L. et al. Imprint of assortative mating on the human genome. Border, R. et al. Assortative mating biases marker-based heritability estimators. Li, M. X. et al. A major gene model of adult height is suggested in Chinese. & Gamazon, E. R. On using local ancestry to characterize the genetic architecture of human traits: genetic regulation of gene expression in multiethnic or admixed populations. Gay, N. R. et al. Impact of admixture and ancestry on eQTL analysis and GWAS colocalization in GTEx. Burch, K. S. et al. Partitioning gene-level contributions to complex-trait heritability by allele frequency identifies disease-relevant genes. Kirby, J. C. et al. PheKB: a catalog and workflow for creating electronic phenotype algorithms for transportability. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genomes Project Consortium et al. A global reference for human genetic variation. Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. & Marchini, J. Genotype imputation using the positional Burrows Wheeler transform. Conomos, M. P., Reiner, A. P., Weir, B. S. & Thornton, T. A. Model-free estimation of recent genetic relatedness. Staples, J. et al. PRIMUS: rapid reconstruction of pedigrees from genome-wide estimates of identity by descent. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. The phenotype-genotype reference map: improving biobank data science through replication. Zou, Y., Carbonetto, P., Wang, G. & Stephens, M. Fine-mapping from summary data with the “Sum of Single Effects” model. Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Frankish, A. et al. GENCODE: reference annotation for the human and mouse genomes in 2023. Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. & Robinson, M. R. Liability-scale heritability estimation for biobank studies of low-prevalence disease. Lee, S. H., Goddard, M. E., Wray, N. R. & Visscher, P. M. A better coefficient of determination for genetic profile analysis. Bulik-Sullivan, B. et al. An atlas of genetic correlations across human diseases and traits. Wallace, C. Statistical testing of shared genetic control for potentially related traits. Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. Ni, G. et al. A comparison of ten polygenic score methods for psychiatric disorders applied across multiple cohorts. Improving polygenic prediction in ancestrally diverse populations. Bertrand, F. & Maumy-Bertrand, M. plsRglm: partial least squares linear and generalized linear regression for processing incomplete datasets by cross-validation and bootstrap techniques with R. Preprint at https://arxiv.org/abs/1810.01005 (2018). We acknowledge the use of data from the National Health Insurance Research Database, provided by the Ministry of Health and Welfare of Taiwan. This study was funded in part by the Academia Sinica (grant nos. and the National Development Fund, Executive Yuan (grant no. This work used ASGC (Academia Sinica Grid-computing Center) Distributed Cloud resources, which is supported by Academia Sinica. Analysis using UK Biobank data used computational resources hosted by the Michigan State University High-Performance Computing Center. Data from the UK Biobank include data provided by patients and collected by the National Health Service (NHS) England as part of their care and support. UK Biobank data also include data assets made available by National Safe Haven as part of the Data and Connectivity National Core Study, led by Health Data Research UK in partnership with the Office for National Statistics and funded by UK Research and Innovation (research that commenced between 1 October 2020 and 31 March 2021, grant no. These authors jointly supervised this work: Wayne Huey-Herng Sheu, Shun-Fa Yang, Jyh-Ming Liou, Jaw-Yuan Wang, Jeng-Fong Chiou, Jer-Yuarn Wu, Cathy S.-J. Institute of Biomedical Sciences, Academia Sinica, Taipei, Taiwan Hung-Hsin Chen, Chien-Hsiun Chen, Ling-Hui Li, Che-Yu Chou, Erh-Chan Yeh, Ming-Fang Tsai, Yi-Min Liu, Chun-yu Wei, Jen-Ping Su, Wan-Jia Lin, Elin H. F. Wang, Cheng-Shin Yang, Ru-Hui Weng, Yu-Chi Chen, Chun-Ping Chang, Tai-Hsun Wu, Yu-Chang Lin, Sye-Pu Chen, Feng-Jen Hsieh, Yuan-Tsong Chen, Pui-Yan Kwok, Jer-Yuarn Wu & Cathy S. J. Fann Institute of Statistical Science, Academia Sinica, Taipei, Taiwan Biomedical Translation Research Center, Academia Sinica, Taipei, Taiwan Core Laboratory of Neoantigen Analysis for Personalized Cancer Vaccine, Office of R&D, Taipei Medical University, Taipei, Taiwan School of Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan Chi-Lu Chiang, Jeng-Kai Jiang, I-Hui Lee, Shih-Yao Lin, Yi-Chen Yeh & Ling-Ming Tseng Division of Colon and Rectal Surgery, Department of Surgery, Taipei Veterans General Hospital, Taipei, Taiwan Institute of Brain Science, National Yang Ming Chiao Tung University, Taipei, Taiwan Institute of Food Safety and Health Risk Assessment, National Yang Ming Chiao Tung University, Taipei, Taiwan Institute of Biomedical Informatics, School of Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan Shih-Yao Lin, Fu-Pang Chang, Hsiang-Ling Ho, Yi-Chen Yeh, Wen-Yih Liang & Paul Chih-Hsueh Chen Department of Biotechnology and Laboratory Science in Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan Yu-Cheng Hsieh, Yi-Ming Chen, Wei-Ju Lee, Hsin Tung & Yi-Jing Sheen Department of Medical Research, Institute of Clinical Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan Division of Pediatric Genetics and Metabolism, Children's Medical Center, Taichung Veterans General Hospital, Taichung, Taiwan Division of Gastroenterology, Department of Internal Medicine, Shuang Ho Hospital, Taipei Medical University, New Taipei City, Taiwan Division of Cardiology, Department of Internal Medicine, Shuang Ho Hospital, Taipei Medical University, New Taipei City, Taiwan Division of Cardiovascular Medicine, Department of Internal Medicine, Wan Fang Hospital, Taipei Medical University, Taipei, Taiwan Center for Medical Education and Humanizing Health Professional Education, Kaohsiung Medical University, Kaohsiung, Taiwan Division of Endocrinology and Metabolism, Department of Internal Medicine, Taichung Veterans General Hospital, Taichung, Taiwan Michigan State University, East Lansing, MI, USA Khoury College of Computer Sciences, Northeastern University, Boston, MA, USA School of Medicine, National Tsing Hua University, Hsinchu, Taiwan Department of Dermatology, Chang Gung Memorial Hopistal, Linkou, Taiwan Division of Rheumatology, Allergy and Immunology, Department of Internal Medicine, Chang Gung Memorial Hospital at Linkou, Taoyuan, Taiwan Department of Psychiatry and Sleep center, Chang Gung Memorial Hospital, Taoyuan, Taiwan College of Medicine, Chang Gung University, Taoyuan, Taiwan Department of Obstetrics and Gynecology, Chang Gung Memorial Hospital, Linkou Medical Center and Chang Gung University College of Medicine, Taoyuan, Taiwan Division of Rheumatology, Allergy and Immunology, Department of Internal Medicine, New Taipei City Municipal TuCheng Hospital, New Taipei City, Taiwan Doctoral Program of Clinical and Experimental Medicine, National Sun Yat-sen University, Kaohsiung, Taiwan Yi-Lwun Ho, Ming-Shiang Wu, Yen-Bin Liu, Jyh-Ming Jimmy Juang, Mao-Hsin Lin, Yen-Hung Lin & Jyh-Ming Liou Cardiovascular Center and Heart Failure Center, Department of Internal Medicine, National Taiwan University Hospital, Taipei, Taiwan Division of Hematology and Oncology, Department of Internal Medicine, Chung Shan Medical University Hospital, Taichung, Taiwan School of Medicine, Chung Shan Medical University, Taichung, Taiwan Endocrinology and Metabolism, Tri-Service General Hospital, National Defense Medical Center, Taipei, Taiwan Department of Dermatology, Tri-Service General Hospital, National Defense Medical Center, Taipei, Taiwan Department and Graduate Institute of Biochemistry, National Defense Medical Center, Taipei, Taiwan Division of Gastroenterology, Department of Internal Medicine, Tri-Service General Hospital, National Defense Medical Center, Taipei, Taiwan Psychiatry, Tri-Service General Hospital, National Defense Medical Center, Taipei, Taiwan Urology, Tri-Service General Hospital, National Defense Medical Center, Taipei, Taiwan Cardiovascular and Mitochondria Related Disease Research Center, Hualien Tzu Chi Hospital, Buddhist Tzu Chi Medical Foundation, Hualien, Taiwan Center of General Education, Buddhist Tzu Chi Medical Foundation, Tzu Chi University of Science and Technology, Hualien, Taiwan Center of Stem Cell and Precision Medicine, Hualien Tzu Chi Hospital, Buddhist Tzu Chi Medical Foundation, Hualien, Taiwan School of Medicine, Tzu Chi University, Hualien, Taiwan Department of Hematology and Oncology, Hualien Tzu Chi Hospital, Buddhist Tzu Chi Medical Foundation, Hualien, Taiwan Division of Allergy, Immunology and Rheumatology, Hualien Tzu Chi Hospital, Buddhist Tzu Chi Medical Foundation, Hualien, Taiwan Department of Ophthalmology, Far Eastern Memorial Hospital, New Taipei City, Taiwan Department of Electrical Engineering, Yuan Ze University, Taoyuan, Taiwan Department of Internal Medicine, Far Eastern Memorial Hospital, New Taipei City, Taiwan Department of Biomedical Engineering, Ming Chuan University, Taoyuan, Taiwan Department of Public Health, College of Medicine, Fu Jen Catholic University, New Taipei City, Taiwan Department of Cardiology, Fu Jen Catholic University Hospital, Fu Jen Catholic University, New Taipei City, Taiwan Precision Medicine Center, Fu Jen Catholic University Hospital, Fu Jen Catholic University, New Taipei City, Taiwan Graduate Institute of Business Administration, College of Management, Fu Jen Catholic University, New Taipei City, Taiwan Koo Foundation Sun Yat-Sen Cancer Center, Taipei, Taiwan Institute of Public Health, National Yang Ming Chiao Tung University, Taipei, Taiwan Division of Gastroenterology and Hepatology, Department of Internal Medicine, Ditmanson Medical Foundation Chia-Yi Christian Hospital, Chiayi, Taiwan Clinical Trial Center, Department of Medical Research, Ditmanson Medical Foundation Chia-Yi Christian Hospital, Chiayi City, Taiwan Fu-Jen Catholic University School of Medicine, New Taipei City, Taiwan Department of Internal Medicine, Cathay General Hospital, Taipei, Taiwan Department of Clinical Pathology, Cathay General Hospital, Taipei, Taiwan Cardiovascular Research Institute, University of California, San Francisco, CA, USA Institute for Human Genetics, University of California, San Francisco, CA, USA Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan Division of Endocrinology and Metabolism, Department of Internal Medicine, Taipei Veterans General Hospital, Taipei, Taiwan Institute of Medicine, Chung Shan Medical University, Taichung, Taiwan Division of Gastroenterology and Hepatology, Department of Internal Medicine, National Taiwan University Hospital, Taipei, Taiwan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors reviewed and approved the final manuscript. Correspondence to Hung-Hsin Chen or Cathy S. J. Fann. is a founder, shareholder and serves on the Board of Directors of Genomic Prediction, Inc. (GP) and E.W. is an employee and shareholder of GP. The other authors declare no competing interests. Nature thanks Bogdan Pasaniuc and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The significantly positive correlations are observed across disease categories between the case proportion in TPMI (x-axis) and the 5-year prevalence in NHIRD (y-axis), except congenital anomalies. The replication rates of TPMI GWAS-identified loci when compared to previously reported loci from the GWAS catalog are presented in this bar chart. Red bars indicate the comparison of TPMI findings to that of all ancestries and blue bars represent the comparison to East Asian ancestries. Heatmap showing genetic correlations among heritable traits. Genetic correlations were estimated using LDSC, with colors representing the correlation coefficients between traits. The weighted pair group method with arithmetic mean (WPGMA) was used for clustering with the correlation coefficient as distance between traits. Each phecode is positioned at a unique x-coordinate, with the tool that has the highest AUC highlighted. Gray bars indicate SNP-heritability (estimated from TPMI GWAS unrelated set [n = 248,754] with LDSC), and the colored bar chart presents the r² values, indicating the proportion of variance explained by the PRS among TPMI validation set (n = 20,000), and dots and error bars show Area Under the receiver operating characteristic Curve (AUC) with 95% confidence interval. An asterisk (*) indicates estimates considering the MHC region. PRS performance is presented as Area Under the receiver operating characteristic Curve (AUC) with 95% confidence inverval in TPMI (orange, TPMI validation set, n = 20,000), Taiwan Biobank (green, n = 88,628), East Asians in UK Biobank (blue, n = 1,572), and East Asians in All of Us (purple, n = 6,895). Detailed methods of genotyping and phenotyping quality control and discussion of SNP-based heritability for quantitative traits, including Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Population-specific polygenic risk scores for people of Han Chinese ancestry. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09497-8'>Aboveground biomass in Australian tropical forests now a net carbon source</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:13:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 646, pages 611–618 (2025)Cite this article Tropical forests act as important global carbon sinks1, and Earth System Models predict increasing near-term carbon sink capacity for these forests, with elevated atmospheric carbon dioxide concentration thought to stimulate tree growth2,3. However, current forest inventory data analyses suggest that the carbon sink capacity of intact tropical forests may be in decline, portending a possible future switch from carbon sinks to carbon sources3,4,5,6,7. Here we use long-term forest inventory data (1971–2019) from Australian moist tropical forests and a causal inference framework8,9,10 to assess the carbon balance of woody aboveground standing biomass over time, the demographic processes accounting for it, and its climatic drivers, including cyclones. We find that a transition from sink (0.62 ± 0.04 Mg C ha−1 yr−1: 1971–2000) to source (−0.93 ± 0.11 Mg C ha−1 yr−1: 2010–2019) has occurred for the aboveground woody biomass of these forests, with sink capacity declining at a rate of 0.041 ± 0.032 Mg C ha−1 yr−1. The transition was driven by increasingly extreme temperature and other climate anomalies, which have increased tree mortality and associated biomass losses4, with no evidence of the carbon fertilization (stimulation) of woody tree growth. Forest dynamics underlying carbon sink capacity were also punctuated by cyclones, with impacts of a similar magnitude to long-term climate-induced changes. Our findings suggest the potential for a similar response to climate change by woody aboveground biomass in moist tropical forests globally, which could culminate in a long-term switch from carbon sinks to carbon sources. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Plot geographic locations, cyclone records, climate data (summarized to census interval lengths) and plot-level aggregated census data are available at Figshare (https://doi.org/10.6084/m9.figshare.c.7922084). Stem-level inventory data (1971–2013) are available at the CSIRO data portal (https://doi.org/10.4225/08/59475c67be7a4), and monthly climate data are available at the TERN data portal (https://doi.org/10.25901/rxtc-th28). All analyses were run using the R statistical environment87; the R code to reproduce the analyses, in-text numbers and figures is available in the Supplementary Information (Code S1). Pan, Y. et al. The enduring world forest carbon sink. Google Scholar Cox, P. M. et al. Sensitivity of tropical carbon to climate change constrained by carbon dioxide variability. Google Scholar Koch, A., Hubau, W. & Lewis, S. L. Earth System Models are not capturing present‐day tropical forest carbon dynamics. Google Scholar Bauman, D. et al. Tropical tree mortality has increased with rising atmospheric water stress. Google Scholar Brienen, R. J. W. et al. Long-term decline of the Amazon carbon sink. Google Scholar Gatti, L. V. et al. Amazonia as a carbon source linked to deforestation and climate change. Google Scholar Hubau, W. et al. Asynchronous carbon sink saturation in African and Amazonian tropical forests. Google Scholar Arif, S. & MacNeil, M. A. Applying the structural causal model framework for observational causal inference in ecology. Google Scholar Segura-Garcia, C., Bauman, D., Arruda, V. L. S., Alencar, A. Human land occupation regulates the effect of the climate on the burned area of the Brazilian Cerrado. Google Scholar Pearl, J. Causal inference in statistics: an overview. Google Scholar Murphy, H. T., Bradford, M. G., Dalongeville, A., Ford, A. J. & Metcalfe, D. J. No evidence for long-term increases in biomass and stem density in the tropical rain forests of Australia. Google Scholar Cavaleri, M. A. et al. Tropical rainforest carbon sink declines during El Niño as a result of reduced photosynthesis and increased respiration rates. Google Scholar Rifai, S. W. et al. ENSO drives interannual variation of forest woody growth across the tropics. Google Scholar Grossiord, C. et al. Plant responses to rising vapor pressure deficit. Google Scholar Feldpausch, T. R. et al. Amazon forest response to repeated droughts. Google Scholar Sanchez-Martinez, P. et al. Amazon rainforest adjusts to long-term experimental drought. Parker, C. L., Bruyère, C. L., Mooney, P. A. & Lynch, A. H. The response of land-falling tropical cyclone characteristics to projected climate change in northeast Australia. Google Scholar Lavendar, S. L. & Walsh, K. J. E. Dynamically downscaled simulations of Australian region tropical cyclones in current and future climates. Google Scholar Ibanez, T. et al. Globally consistent impact of tropical cyclones on the structure of tropical and subtropical forests. Google Scholar Murphy, H. T., Metcalfe, D. J., Bradford, M. G. & Ford, A. J. Community divergence in a tropical forest following a severe cyclone. Google Scholar Murphy, H. T. & Metcalfe, D. J. The perfect storm: Weed invasion and intense storms in tropical forests. Google Scholar Brienen, R. J. W. et al. Forest carbon sink neutralized by pervasive growth-lifespan trade-offs. Google Scholar Montreal Process Implementation Group for Australia and National Forest Inventory Steering Committee. Australia's State of the Forests Report 2018 (ABARES, 2018). Feldpausch, T. R. et al. Tree height integrated into pantropical forest biomass estimates. Google Scholar Bradford, M. G., Murphy, H. T., Ford, A. J., Hogan, D. L. & J., M. D. in Ecology Vol. Bauman, D. et al. Tropical tree growth sensitivity to climate is driven by species intrinsic growth rate and leaf traits. Google Scholar Bradford, M. & Murphy, H. T. The importance of large-diameter trees in the wet tropical rainforests of Australia. Google Scholar Rowland, L. et al. Shock and stabilisation following long-term drought in tropical forest from 15 years of litterfall dynamics. Google Scholar Staal, A. et al. Hysteresis of tropical forests in the 21st century. Google Scholar Rifai, S. W. et al. Thirty-eight years of CO2 fertilization has outpaced growing aridity to drive greening of Australian woody ecosystems. Google Scholar Jiang, M. et al. The fate of carbon in a mature forest under carbon dioxide enrichment. Google Scholar Cunha, H. F. V. et al. Direct evidence for phosphorus limitation on Amazon forest productivity. Google Scholar Jiang, M. et al. Microbial competition for phosphorus limits the CO2 response of a mature forest. Google Scholar Crous, K. Y. et al. Leaf warming in the canopy of mature tropical trees reduced photosynthesis due to downregulation of photosynthetic capacity and reduced stomatal conductance. Chambers, J. Q., Higuchi, N., Schimel, J. P., Ferreira, L. V. & Melack, J. M. Decomposition and carbon cycling of dead trees in tropical forests of the central Amazon. Google Scholar Walker, W. S. et al. The global potential for increased storage of carbon on land. Google Scholar Raich, J. W. & Schlesinger, W. H. The global carbon dioxide flux in soil respiration and its relationship to vegetation and climate. Viscarra Rossel, R. A., Webster, R., Bui, E. N. & Baldock, J. A. Baseline map of organic carbon in Australian soil to support national carbon accounting and monitoring under climate change. Nottingham, A. T. et al. Microbial diversity declines in warmed tropical soil and respiration rise exceed predictions as communities adapt. Sotta, E. D. et al. Effects of an induced drought on soil carbon dioxide (CO2) efflux and soil CO2 production in an Eastern Amazonian rainforest, Brazil. Luo, Z., Feng, W., Luo, Y., Baldock, J. & Wang, E. Soil organic carbon dynamics jointly controlled by climate, carbon inputs, soil properties and soil carbon fractions. Smith, M. N. et al. Empirical evidence for resilience of tropical forest photosynthesis in a warmer world. Allen, C. D., Breshears, D. D. & McDowell, N. G. On underestimation of global vulnerability to tree mortality and forest die‐off from hotter drought in the Anthropocene. Hammond, W. M. et al. Global field observations of tree die-off reveal hotter-drought fingerprint for Earth's forests. Middleby, K. B., Cheesman, A. W. & Cernusak, L. A. Impacts of elevated temperature and vapour pressure deficit on leaf gas exchange and plant growth across six tropical rainforest tree species. Slot, M., Rifai, S. W., Eze, C. E. & Winter, K. The stomatal response to vapor pressure deficit drives the apparent temperature response of photosynthesis in tropical forests. Duan, H. et al. CO2 and temperature effects on morphological and physiological traits affecting risk of drought-induced mortality. Breshears, D. D. et al. Underappreciated plant vulnerabilities to heat waves. Doughty, C. E. et al. Tropical forests are approaching critical temperature thresholds. Cunningham, S. C. & Read, J. Foliar temperature tolerance of temperate and tropical evergreen rain forest trees of Australia. Menezes-Silva, P. E. et al. Different ways to die in a changing world: Consequences of climate change for tree species performance and survival through an ecophysiological perspective. Allen, C. D. et al. A global overview of drought and heat-induced tree mortality reveals emerging climate change risks for forests. Teskey, R. et al. Responses of tree species to heat waves and extreme heat events. Plant Cell Environ. Turton, S. M. Landscape-scale impacts of Cyclone Larry on the forests of northeast Australia, including comparisons with previous cyclones impacting the region between 1858 and 2006. Qie, L. et al. Long-term carbon sink in Borneo's forests halted by drought and vulnerable to edge effects. Ibanez, T. et al. Damage to tropical forests caused by cyclones is driven by wind speed but mediated by topographical exposure and tree characteristics. Metcalfe, D. J., Bradford, M. G. & Ford, A. J. Cyclone damage to tropical rain forests: Species- and community-level impacts. Connell, J. H. & Green, P. T. Seedling dynamics over thirty-two years in a tropical rain forest tree. Berenguer, E. et al. A large-scale field assessment of carbon stocks in human-modified tropical forests. Cummings, D. L., Boone Kauffman, J., Perry, D. A. & Flint Hughes, R. Aboveground biomass and structure of rainforests in the southwestern Brazilian Amazon. Malhi, Y. et al. The regional variation of aboveground live biomass in old-growth Amazonian forests. Djuikouo, M. N. K., Doucet, J.-L., Nguembou, C. K., Lewis, S. L. & Sonké, B. Diversity and aboveground biomass in three tropical forest types in the Dja Biosphere Reserve, Cameroon. Carbon storage and emissions offset potential in an East African tropical rainforest. Lewis, S. L. et al. Above-ground biomass and structure of 260 African tropical forests. Kotowska, M. M., Leuschner, C., Triadiati, T., Meriem, S. & Hertel, D. Quantifying above- and belowground biomass carbon loss with forest conversion in tropical lowlands of Sumatra (Indonesia). Katayama, A. et al. Carbon allocation in a Bornean tropical rainforest without dry seasons. Slik, J. W. F. et al. Environmental correlates of tree biomass, basal area, wood specific gravity and stem density gradients in Borneo's tropical forests. Martin, A. R., Doraisami, M. & Thomas, S. C. Global patterns in wood carbon concentration across the world's trees and forests. The CSIRO Rainforest Permanent Plots of North Queensland: Site, Structural, Floristic and Edaphic Descriptions (Rainforest CRC, Cairns, 2006). Talbot, J. et al. Methods to estimate aboveground wood productivity from long-term forest inventory plots. Chave, J. et al. Tree allometry and improved estimation of carbon stocks and balance in tropical forests. Ledo, A. et al. Re-evaluation of individual diameter:height allometric models to improve biomass estimation of tropical trees. Hutchinson, M. F., Xu, T., Kesteven, J. L., Marang, I. J. & Evans, B. J. ANUClimate v2.0. NCI Data Catalogue https://doi.org/10.25914/60a10aa56dd1b (2022). Abatzoglou, J. T., Dobrowski, S. Z., Parks, S. A. & Hegewisch, K. C. TerraClimate, a high-resolution global dataset of monthly climate and climatic water balance from 1958-2015. Dlugokencky, E. J., Mund, J. W., Crotwell, A. M., Crotwell, M. J. & Thoning, K. W. NOAA Global Greenhouse Gas Reference Network Flask-Air Sample Measurements of CO2, CH4, CO, N2O, H2, SF6 and isotopic ratios at Global and Regional Background Sites, 1997-2020, Version: 2021-07-30 (ed. NOAA GML Carbon Cycle Cooperative Global Air Sampling Network) (GML, 2021). Aragão, L. E. O. C. et al. Spatial patterns and fire response of recent Amazonian droughts. Wood, S. N. Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models. Cinelli, C., Forney, A. A crash course in good and bad controls. Adams, H. D. et al. A multi-species synthesis of physiological mechanisms in drought-induced tree mortality. Dray, S., Legendre, P. & Peres-Neto, P. R. Spatial modelling: a comprehensive framework for principal coordinate analysis of neighbour matrices (PCNM). Bauman, D., Drouet, T., Dray, S. & Vleminckx, J. Disentangling good from bad practices in the selection of spatial or phylogenetic eigenvectors. Dray, S. et al. R package version 0.3-24: adespatial: multivariate multiscale spatial analysis. Bauman, D., Drouet, T., Fortin, M. J. & Dray, S. Optimizing the choice of a spatial weighting matrix in eigenvector‐based methods. & Legendre, P. An empirical comparison of permutation methods for tests of partial regression coefficients in a linear model. McElreath, R. Statistical Rethinking: A Bayesian Course with Examples in R and STAN 2nd edn (Chapman and Hall, 2020). Lenth, R. R package v.1.10: emmeans: estimated marginal means, aka least-squares means. R Core Team. R: a language and environment for statistical computing. https://www.R-project.org/ (R Foundation for Statistical Computing, 2025). We acknowledge the early work of G. Stocker, who established the QPRP-CSIRO plots used in this study, with funding administered by CSIRO. We also acknowledge the CSIRO staff and volunteers who have maintained and censused the plots since 1971. H.C. was supported by seed grant funding from the Research School of Biology, the Australian National University and an AGRTP scholarship. was funded by the Horizon 2020 research and innovation program of the European Union under the Marie Skłodowska–Curie grant agreement no. P.M. was funded by a Royal Society of London Wolfson Fellowship RSWF\211008, ARC grant DP17010409 and NERC grant NE/W006308/1. was funded by the University of Maryland and a Royal Society Wolfson Visiting Fellowship (RSWVF\R1\221018). Unaffiliated: Andrew Ford, Matthew Bradford Hawkesbury Institute for the Environment, Western Sydney University, Richmond, New South Wales, Australia AMAP, Université de Montpellier, IRD, CIRAD, CNRS, INRAE, Montpellier, France Environmental Change Institute, School of Geography and the Environment, University of Oxford, Oxford, UK Laboratoire d'Écologie Végétale et Biogéochimie, Université Libre de Bruxelles, Brussels, Belgium Department of Geology and Earth System Science Interdisciplinary Center, University of Maryland, College Park MD, USA School of Geosciences, University of Edinburgh, Edinburgh, UK Michael N. Evans & Patrick Meir Australian Plant Phenomics Network, School of Agriculture, Food and Wine, University of Adelaide, Adelaide, South Australia, Australia CREAF, Barcelona, Spain Research School of Biology, Australian National University, Canberra, Australian Capital Territory, Australia Adrienne Nicotra & Patrick Meir CSIRO, Townsville, Queensland, Australia Australian Tropical Sciences Precinct, James Cook University, Townsville, Queensland, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and H.M. contributed to conceptualization; H.C., D.B., M.N.E. devised the methodology; M.B., H.M., D.B. and H.C. curated the data; H.C., A.N., P.M., I.C., O.B., A.F., M.B. and H.M. conducted the investigation; H.C. helped with visualization; H.C., A.N., P.M., H.M. and M.B. provided funding acquisition; H.C., A.N. and P.M. handled the project administration; and A.N. and P.M. supervised the study. The original manuscript was drafted by H.C., D.B., H.M., P.M. and A.N. All authors contributed to reviewing and editing the paper. Correspondence to Hannah Carle. The authors declare no competing interests. Nature thanks Flávia Costa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. CRT is calculated by dividing a, total standing aboveground biomass (AGB) by b, biomass loss rate. c, CRTs were calculated per decade for each plot (grey lines), then averaged to give regional means. Panels (a-b) show regional means for standing biomass and biomass loss, plotted using a loess smooth on all plot-by-census data. Grey shading is the 95th percentile confidence interval for error on the loess-smoothed annual means. a, Wood density, b, stem density, c, basal area, and d, mean AGB of dead stems (biomass loss divided by number of dead stems) for n = 279 timepoints (n = 20 plots). This arises because the statistical infrequency of cyclones results in smaller sample size for this group. Net AGB rate is the annualized change in per hectare AGB gains minus AGB losses. Boxplots show the interquartile range of the distribution, with census records for individual plots (open circles) and statistical outliers based on the inter-quartile range of the Net AGB distribution (closed circles). Bars represent each census interval and are shaded by interval length (lighter bars for longer intervals) to aid visual interpretation. Red triangles represent the cyclone dates for each plot affected. Code S1 Carbon sink Aus analysis. Replicates the analyses and reproduces figures, tables and in-text numbers. Install R and RStudio. Create a new project and copy Code S1 and data (https://doi.org/10.6084/m9.figshare.c.7922084) into that directory. Install required packages and dependencies. Outputs 11 tiff (main text and Extended Data Figs. and component panels) and five csv files. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Carle, H., Bauman, D., Evans, M.N. et al. Aboveground biomass in Australian tropical forests now a net carbon source. Issue date: 16 October 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09619-2'>The astrocytic ensemble acts as a multiday trace to stabilize memory</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:05:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Recalled memories become transiently labile and require stabilization1,2,3. The mechanism for stabilizing memories of survival-critical experiences, which are often emotionally salient and repeated, remains unclear4. Here we identify an astrocytic ensemble that is transcriptionally primed by emotional experience and functionally triggered by repeated experience to stabilize labile memory. Using a novel brain-wide Fos tagging and imaging method, we found that astrocytic Fos ensembles were preferentially recruited in regions with neuronal engrams5 and were more widespread during fear recall than during conditioning. We established the induction mechanism of the astrocytic ensemble, which involves two steps: (1) an initial fear experience that induces day-long, slow astrocytic state changes with noradrenaline receptor upregulation; and (2) enhanced noradrenaline responses during recall, a repeated experience, enabling astrocytes to integrate coincident signals from local engrams and long-range noradrenergic projections, which induce secondary astrocytic state changes, including the upregulation of Fos and the neuromodulatory molecule IGFBP2. The astrocytic ensemble thus acts as a multiday trace in a subset of astrocytes after experience-dependent neural activity, which are eligible to capture future repeated experiences for stabilizing memories. Recall transiently destabilizes memories, which require re-stabilization to become long-lasting1,2,3. Despite its importance in human cognition and neuropsychiatric disorders6,7, the mechanisms that specifically stabilize memories of critical experiences—those that are essential for survival and frequently marked by emotional salience and repetition—remain incompletely understood. Memory traces are linked to specialized neuronal ensembles (neuronal engrams), which include neuronal populations that become Fos+ during both initial and repeated experiences8. However, neuronal Fos alone, a plasticity marker that indicates circuit remodelling9, is insufficient for memory stabilization, implying the need for additional cellular substrates. By tiling the entire mammalian central nervous system (CNS), astrocytes structurally and functionally interact with neurons and other glia. Their functions contribute to information processing and animal behaviour, and their dysfunctions are widely implicated in CNS disorders10,11. Astrocytes are known to be developmentally diverse12,13,14 and can adaptively change their molecular programmes in response to physiology11,15,16,17,18 and pathology19,20,21—hereafter referred to as state—in specific ways that suggest flexibly altered functions in a context-dependent manner. This adaptability is akin to experience-dependent neuronal responses (such as Fos upregulation)22 and the subsequent formation of inter-regional neuronal engram network8. However, the manifold nature of astrocyte responses through an extensive array of cell-surface receptors for neuroactive biomolecules19,23,24 have made it challenging to determine whether, when and how multiple inputs are integrated by a subset of astrocytes to generate specific states that form astrocyte ensembles that are responsible for particular cellular outputs15,18,24,25. This is because the field has lacked adequate tools to explore behaviourally relevant astrocyte ensembles (BAEs) in an unbiased manner at the single-cell level as well as in a brain-wide manner, which would enable ensemble-specific molecular dissection and functional perturbation. To agnostically and holistically identify BAEs, we developed a brain-wide astrocyte Fos tagging tool that enables permanent genetic access to BAEs that express Fos due to a specific experience. Although behaviourally relevant stimuli-induced astrocyte Fos mRNA22 and c-Fos protein18,26 have been observed in selected brain regions, its brain-wide profiling remains largely unexplored. We systematically validated a novel genetic and imaging strategy for whole-brain BAE tagging, demonstrating astrocyte selectivity, Fos sensitivity and temporal regulation (Extended Data Figs. First, using a CNS-wide-transducing adenovirus (AAV) vector (AAV-PHP.eB) with an astrocyte-specific GfaABC1D promoter27, we achieved widespread, astrocyte-selective (97.3–99.5%) expression (Extended Data Fig. Second, we tested whether using AAV-PHP.eB to express Cre-dependent mNeonGreen (mNG) can serve as a marker of transient astrocytic Fos induction in Fos-iCreERT2 (TRAP2) mice28. This intersectional approach aimed to drive astrocytic mNG expression only when the Fos promoter was active during fast-acting 4-hydroxytamoxifen (4-OHT) administration, enabling behaviour epoch-specific astrocyte Fos tagging across the CNS. As a positive control, we virally expressed the astrocytic Gαq-coupled designer receptor exclusively activated by designer drugs (DREADD) hM3Dq, a strong inducer of astrocyte c-Fos26,29,30, combined with 4-OHT and the hM3Dq agonist clozapine N-oxide (CNO) intraperitoneal injection. hM3Dq stimulation yielded approximately 282 mNG+ astrocytes per mm2, and significantly lower tagging in controls (Extended Data Fig. Third, to probe the temporal dynamics of Fos-mNG tagging, dark-adapted mice were exposed to light at various intervals relative to 4-OHT administration28. Tagging peaked within a 6-h window around 4-OHT administration, with an approximately 22-fold increase in astrocytes in visual cortex compared with those in somatosensory cortex, indicating regional specificity and dependence on local neuronal activity (Extended Data Fig. Collectively, these experiments validated our behaviour epoch-specific, brain-wide BAE tagging strategy. To broadly explore BAEs in a learning paradigm, we used a contextual fear conditioning and recall task5,28 (Fig. TRAP2::AAV-PHP.eB-GfaABC1D-DIO-mNG mice underwent three days of habituation to minimize contextual novelty and isolate associative components of fear learning and recall28,31. On day 4, one group received 3 foot shocks with immediate 4-OHT (FC), and another received 4-OHT immediately after recall 24 h post-FC (FR). Controls were non-shocked mice (NoFC) or fear-conditioned mice kept in the home cage for 24 h and given 4-OHT without recall (NoFR). Shocked groups exhibited robust freezing in the conditioned chamber, indicating learning and recall (Fig. One week later, whole-brain serial two-photon tomography (STPT)32 enabled the detection of individual mNG+ astrocytes (BAEs) (Supplementary Videos 1 and 2). After excluding ventricular niches for bona fide astrocyte counts (Methods), BAE counts across 677 regions covered approximately 80.7% of the brain (Supplementary Table 2). a, Fos+ astrocytes were tagged with Cre-dependent mNG AAV in four cohorts (NoFC, FC, NoFR and FR), followed by STPT, 3D reconstruction, segmentation and single-cell counting. c, 3D rendering of whole brain, the hippocampus (HPC), amygdala and primary visual cortex (VISp) with Fos-mNG astrocytes. A, anterior; D, dorsal; L, lateral; M medial; P. posterior; V, ventral. d,e, Heat map of fold changes in BAE count (versus NoFC (d) and FR versus FC (e); NoFC: n = 6; FC: n = 4; NoFR: n = 4; FR: n = 4 mice). Abbreviations for brain regions are listed in Extended Data Fig. Levene's test followed by one-way analysis of variance (ANOVA) or Kruskal–Wallis; post hoc Tukey or Dunn test with Bonferroni correction. Full data in Supplementary Table 2. f–h, FC- and FR-induced Fos counts in neurons versus astrocytes across engram areas for FC versus NoFC (f), FR versus NoFC (g) and FR versus NoFC with log scales, showing correlation Pearson's correlation coefficient (r) (h). h, Amygdalar subregions are indicated with yellow dots. j, Images of tagged Fos+ neurons and BAEs (representative of 10–22 sections from 3–5 mice per group). k, Quantification of Fos+ neurons (NoFC: n = 11 slices from 4 mice; FC: n = 10 slices from 3 mice; NoFR: n = 12 slices from 4 mice; FR: n = 13 slices from 4 mice) and BAE density (NoFC: n = 22 slices from 4 mice; FC: n = 18 slices from 3 mice; NoFR: n = 17 slices from 3 mice; FR: n = 15 slices from 5 mice). Sample sizes and replicates in Supplementary Table 1. Whereas the number of Fos+ neurons increased during both FC and FR, forming networks causally linked to fear memory traces—commonly referred to as engram neurons5,8,28, the density of Fos+ astrocytes during FC (FC-BAE; Fig. 1c) was seldom distinguishable from the NoFC control (NoFC-BAE). However, we found that the number of BAEs during recall (FR-BAE) was augmented across multiple regions (Fig. This was not simply reflecting delayed Fos upregulation after FC, because Fos induction was significantly higher in FR than in NoFR. Normalized comparisons across engram regions5 (Fig. 1d) showed a significantly greater number of FR-BAEs compared with FC-BAEs, particularly in the amygdala (Fig. 1e), the region long implicated in fear memory across species3,4,33,34,35. Consistently, in the lateral and basolateral amygdala (LA/B), astrocytic c-Fos protein was unchanged after FC or in NoFR versus NoFC, but FR induced a marked increase at 1.5 h with partial decay by 6 h (Extended Data Fig. Consistently, the number of of FR-BAEs in the LA/B (FR-BAELA/B) was robust whether 4-OHT was given immediately or 3 h post-FR (Extended Data Fig. These data indicate that astrocyte Fos promoter activity, and the associated surge in Fos mRNA and c-Fos protein expression, peaks within a biologically plausible window of 0–3 h following FR, but not following FC. A recent study18 reported an increase in hippocampal astrocytic Fos after FC without habituation, contrasting with our findings (Fig. We hypothesized that this reflects contextual novelty: without prior habituation, novelty strongly drives astrocytic Fos. Supporting this, our ‘FC + novelty' group (FC in novel context with no habituation) showed widespread astrocytic Fos, including in CA1 (Extended Data Fig. Thus, the CA1 astrocytic Fos is likely to signal novelty, whereas our habituated FC protocol reveals astrocytic ensembles that are more selectively engaged in learning and recall. To holistically compare the anatomical positioning of BAEs with previously reported brain-wide Fos+ neuron distribution during FC and FR5, we analysed subregions from previously defined regions in which Fos+ neuron counts of FC and FR were significantly higher than in non-shocked controls, referred to as neuronal ‘engram significant brain regions'5 (Fig. The number of Fos+ neurons increased during FC versus non-shocked controls, whereas there was no apparent change in astrocytic Fos counts (Fig. By contrast, FR elicited a strong correlation between neuronal and astrocytic Fos induction (r = 0.56; Fig. 1g,h), suggesting that local astrocyte–neuron interactions may underlie FR-BAE formation22. Notably, amygdala subregions that displayed selective astrocytic Fos increases during FR (Fig. 1e) also showed parallel neuronal Fos increases (Fig. To test whether astrocytic and neuronal ensembles are co-engaged within the same regions, we performed dual-colour Fos tagging in the amygdala of TRAP2 mice (Fig. Neuronal Fos increased after FC or FR, consistent with past reports5,8,22. By contrast, astrocytic tagging was minimal following FC and NoFR but was robustly induced after FR (Fig. These results confirm that astrocyte Fos induction in the amygdala is recall-specific and distinct from neuronal ensemble dynamics. To identify mechanisms underlying astrocyte Fos induction, we combined pharmacological screening, circuit genetics, in vivo imaging and transcriptomics (Figs. Growth factors that activate c-Fos in neurons36 did not trigger c-Fos signals in primary astrocytes (Extended Data Fig. 7), indicating that the induction mechanisms of c-Fos in neurons and astrocytes are distinct. Given the accumulating evidence that astrocyte G-protein-coupled receptor (GPCR) activation upregulates c-Fos expression26,29,30, we then stimulated primary astrocytes with a variety of neurotransmitters that bind to GPCRs expressed in astrocytes11,22,23, observing that noradrenaline (NA) increased c-Fos signals (Fig. The astrocyte c-Fos signals induced by NA at 0.1 µM and 1 µM were both abolished by propranolol (Fig. 2d), which antagonizes Gαs-coupled β-adrenergic receptors (β-receptors), but not by prazosin or atipamezole, which antagonize Gαq-coupled α1-adrenoreceptors (α1-receptors) or Gαi-coupled α2-adrenoreceptors, respectively (Fig. The astrocyte c-Fos induction was resistant to the Ca2+ chelator 1,2-bis(2-aminophenoxy)ethane-N,N,N′,N′-tetraacetic acid (BAPTA-AM)37. The membrane-permeable Ca2+ ionophore ionomycin did not increase astrocyte c-Fos signals (Fig. However, the inhibition of cAMP production by the adenylate cyclase inhibitor SQ22536 attenuated astrocyte c-Fos signals induced by the adenylate cyclase–cAMP activator forskolin, the β-receptor agonist isoproterenol or NA (Fig. Together, these data show that NA induces c-Fos signals in primary astrocytes mainly through Gαs-coupled β-receptors and subsequent increases in cAMP. b, NA increased c-Fos intensity in GFAP+ astrocytes (n = 27–50 cells per condition). One-way ANOVA, Dunnett's test versus vehicle (Veh). 5-HT, serotonin; a.u., arbitrary units; ACh, acetylcholine; DA, dopamine; GABA, γ-aminobutyric acid. c, Image showing NA (1 µM)-induced c-Fos increase (vehicle: n = 40; NA 1 μM: n = 41 cells). d, NA-induced c-Fos was blocked by inhibition of β-receptors or cAMP but not by Ca2+ mobilization (n = 28–55 cells per condition). One-way ANOVA, Dunnett's or Tukey's test. AC, adenylate cyclase; Atip, atipamezole; Frsk, forskolin; Isop, isoproterenol; Praz, prazosin; Prop, propranolol. e, Schematic of AAV-mediated Grm3 (encoding mGluR3) expression. f,g, Convergent NA and glutamate stimulation enhanced c-Fos in mGluR3+ astrocytes, but not in mGluR3− astrocytes (n = 98–131 cells per condition). i, Ionomycin (Iono)-induced increase in Ca2+ increased c-Fos when combined with NA, isoproterenol (β-receptor agonist) or forskolin (n = 41–59 cells per condition). j, Schematic of AAV injection into amygdala (LA/B) and LC in TRAP2 mice. m, Quantification of Fos-tagged astrocytes (mNG+) localized near hM3Dq+ LA/B neurons and NA fibres in l (representative of five biological replicates from the LC+/LA+ group). n–q, FR-BAE tagging with β-receptor blockade (propranolol, 10 mg kg−1) (n), LC silencing (o), amygdala (Amy) silencing (p) or RAM-based engram silencing (q) reduced BAE density. Scale bars (n,o–q, top and middle), 1 mm. o–q, Bottom image, magnified view showing local hM4Di-mCherry in neurons. Sample sizes and replicates in Supplementary Table 1. a, AAV injection of GRABNE2h with optical fibre above LA/B to monitor bulk NA response (representative image from 6 mice; scale bar, 200 µm). b, ΔF/F traces showing NA signals in home cage (HC) and CtxA on fear conditioning (FC) day. NA signal increased with CtxA exposure and foot shocks, and showed larger, longer responses during recall (FR). Right, summary plots (n = 5–6 mice). c, Top, AAV injection of astrocyte-selective cAMPinG1 and RCaMP3 with optical fibre above LA/B to monitor astrocyte cAMP and Ca2+. Bottom, co-expression of cAMPinG1 and RCaMP3 in LA/B astrocytes (representative image from 8 mice; scale bars: 200 µm (main image), 20 µm (magnified views, bottom)). d, Astrocyte cAMP signals increased with context exposure and shocks, and during FR they increased gradually from HC baseline and were further enhanced in CtxA. Propranolol (10 mg kg−1, β-receptor antagonist) abolished these effects. Right, summary plot (n = 8 mice). e, Astrocyte Ca2+ signals also increased with context exposure and shocks. Right, summary plots (n = 7 mice). Sample sizes and number of replicates in Supplementary Table 1. a, t-Distributed stochastic neighbour embedding (tSNE) clustering of scRNA-seq data from amygdalar astrocytes across NoFC, FC, NoFR and FR conditions (n = 4 mice per group). b, Dot plots showing Fos expression. Dot colour indicates z-score (grey, depletion; green, enrichment). c, Left, dot plots showing adrenoreceptor gene expression. Undetected receptors (Adra1d, Adra2c and Adrb3) are not shown. Right, Adra1a and Adrb1 upregulation in NoFR versus FC. e, RNAscope images and violin plots of Adra1a and Adrb1 in Slc1a3+ astrocytes (18–28 cells from 5 mice per group). One-way ANOVA, Dunnett's test versus NoFC. f, Co-expression index for Adra1a and Adrb1 (Methods) peaks at 1 day post-FC. One-way ANOVA, Dunnett's test versus NoFC. h, Representative images and quantification of tagged FR-BAE, peaking at 1 day post-FC. One-way ANOVA, Dunnett's test versus NoFC. i, RNAscope of Adrb1 and Fos in Slc1a3+ astrocytes (FR). Arrows indicate a Fos and Adrb1-high cell (representative images from 18 sections, 6 mice). j, Pearson's correlation coefficient (r) for Fos and Adrb1 expression (z-scores). l, Schematic of Adra1a or Adrb1 conditional knockout (cKO). m, RNAscope of Adra1a and Adrb1 in Slc1a3+ astrocytes. n, Violin plots showing reduced targeted receptor after conditional knockout (77–117 cells, 3 mice per group). o,p, Images (o) and quantification (p) of FR-BAEs, showing decreased density following Adra1a or Adrb1 conditional knockout. n = 5–9 mice, One-way ANOVA, Dunnett's test versus FR controls. Sample sizes and replicates in Supplementary Table 1. Astrocytes in vivo respond to both local circuits and long-range neuromodulators15. 3d) suggest that local activity also contributes to Fos induction in vivo. In mGluR3-expressing astrocytes, glutamate alone was ineffective, but in combination with NA it strongly enhanced c-Fos intensity (Fig. No such synergy was seen in mGluR3-negative astrocytes. Because Gαi-coupled GPCRs such as mGluR3 can enhance Gβγ–Ca2+ signalling when Gαq–phospholipase Cβ is co-activated29,40, NA may recruit α1-receptors (Gαq) and mGluR3 (Gαi) to cooperate with β-receptor–cAMP pathways (Fig. Supporting this, ionomycin-induced Ca2+ signals enhanced c-Fos when combined with NA, isoproterenol or forskolin (Fig. Thus, astrocytic c-Fos is driven primarily by β-receptor–cAMP and is further potentiated by convergent glutamate and NA signalling. To test this convergence in vivo, we expressed neuron-specific hM3Dq in LA/B and locus coeruleus (LC), which mainly consists of noradrenergic neurons to stimulate them with CNO (Fig. The number of Fos-mNG+ astrocytes remained low when only LA/B or LC neurons were activated, but increased markedly when both were stimulated (Fig. This provides in vivo evidence that coincident local and noradrenergic inputs induce astrocytic Fos. We next tested the necessity of local and far-reaching NA neural signals during FR-BAE tagging. TRAP2::AAV-PHP.eB-GfaABC1D-DIO-mNG mice received one of four perturbations at the time of tagging: β-receptor antagonism with propranolol (Fig. 2n) or hM4Di-mediated silencing of LC neurons (Fig. 2p) or amygdalar engram neurons tagged with the robust activity marking (RAM) system41,42 whose hM4Di-based silencing (Extended Data Fig. 8) is known to reduce fear memory recall41 (Fig. In all conditions, FR-BAE density decreased significantly (Fig. 2r), demonstrating dependence on both LC-NA and local engram activity. To further test whether FR-BAEs reside in anatomical contexts where they could receive convergent inputs from both noradrenergic projections and local engram neurons, we performed dual-colour Fos tagging with neuron-specific DIO-mCherry and astrocyte-specific DIO-mNG AAVs in TRAP2 mice, combined with noradrenaline transporter (NET) immunostaining to visualize noradrenergic projections (Extended Data Fig. Territories of FR-BAEs showed greater overlap with noradrenergic projections and FR engram neurons than those of non-FR-BAEs, with no change in astrocyte territory size. These findings indicate that FR-BAEs are positioned to integrate convergent inputs from noradrenergic projections and local engram neurons, supporting the model that their coincidence drives FR-BAE induction in the amygdala. 2) support a convergence model in which NA and engram activity induce astrocyte Fos. However, since both activities occur during FC and FR8,43, the stronger astrocyte Fos induction during FR than FC (Fig. 1) raises the question of how astrocytes generate this selective response. To address this, we examined NA release and astrocyte cAMP–Ca2+ signalling dynamics across fear learning and recall using in vivo fibre photometry (Fig. We deployed AAV2/9-hSyn1-GRABNE2h44 to monitor bulk NA signals in the LA/B of freely behaving mice (Fig. During habituation 24 h prior to FC, NA signals were observed when the mice were placed into context A (CtxA; foot shock chamber) (peak ΔF/F = 0.142 ± 0.017), probably reflecting mild stressors and/or locomotion44 (NoFC; Fig. On top of the contextual NA signals, single foot shocks evoked enhanced and relatively fast (peak ΔF/F = 0.206 ± 0.04; half-decay = 4.3 min) NA responses in LA/B (FC; Fig. In addition, further enhanced (peak ΔF/F = 0.211 ± 0.022) and prolonged (half-decay = 13.5 min) NA signals were observed during the contextual FR (FR; Fig. The prolonged NA signals were absent in context B (CtxB) (Extended Data Fig. 10c,d), a neutral, non-conditioned context where mice show no freezing (Extended Data Fig. 10b), indicating that recall of contextual fear drives sustained NA signals. We next used dual-colour fibre photometry (AAV2/5-GfaABC1D-cAMPinG1 and AAV2/5-GfaABC1D-RCaMP345) to measure astrocyte cAMP and Ca2+ signals (Fig. Consistent with the NA signals (Fig. 3a,b), cAMPinG1 signals increased during habituation and foot shocks, which fully decayed within about 2 min (NoFC and FC; Fig. On the FR day, however, astrocytes displayed a progressive increase in cAMP (FR; Fig. 3d), which was absent in CtxB (Extended Data Fig. β-receptor antagonism (propranolol) in vivo abolished the astrocytic cAMP signalling during FR (Fig. 3d), which suppressed FR-BAE induction (Fig. 2n,r) and freezing (Extended Data Fig. Foot shock elicited astrocyte Ca2+ signals as reported previously46, and Ca2+ event frequency increased during FR (Fig. These data collectively indicate that fear experiences induce the enhancement of NA signals within the amygdala, which strongly trigger astrocyte cAMP signals through β-receptors during FR. The increased astrocyte cAMP–Ca2+ signalling during FR raised the question of how astrocytes mount stronger responses during recall than during conditioning, despite NA release and engram neural activity occurring in both fear experiences. To address this, we examined experience-dependent molecular changes in astrocytes under NoFC, FC, NoFR and FR conditions. We dissected amygdala tissue 90 min after behavioural testing and isolated astrocytes by fluorescence-activated cell sorting (FACS) using the ATPase Na+/K+ transporting subunit β2 (ATP1B2)47 antibody ACSA2 (Extended Data Fig. 11a) and performed single-cell RNA sequencing (scRNA-seq). After quality control, we obtained profiles of 1,903 astrocytes (Fig. 11b,c) and identified 4 transcriptionally distinct astrocyte clusters (AST1–4; Fig. We did not observe any noticeable shifts in the subcluster composition across conditions, suggesting that FC and FR do not alter cluster identity. Of note, FR astrocytes showed the highest Fos levels (Fig. 4b), in line with BAE tagging (Fig. Whereas scRNA-seq provides a snapshot at a single time point (approximately twofold increase in FR versus FC; Fig. 4b), the Fos-TRAP system integrates promoter activity over hours, which is likely to account for the greater fold differences observed with tagging (more than 30-fold in FR versus FC; Fig. Subsequent analyses revealed that Adra1a (α1-receptor) and Adrb1 (β1-adrenoreceptor) as the two dominant adrenoreceptor genes expressed in amygdalar astrocytes. Notably, both receptors were upregulated in the FC group (1.5 h post-FC) and further elevated in the NoFR group (24 h post-FC without recall), suggesting continued priming of astrocytes over time (Fig. This data suggests a model in which FC induces a molecular state change in astrocytes, with a latency, increasing adrenoreceptor expression to potentiate NA signalling and enable future repeated fear-evoked Fos induction during FR. The enhancement in astrocytic adrenoreceptor expression is likely to underlie the NA–β-receptor-driven cAMP signals during FR (Fig. 3c,d) and Fos induction in response to recall of strong fear memory a day after conditioning (Fig. To test this model, we explored temporal changes in adrenoreceptor expression, their link to FR-BAE induction, and causal roles in Fos induction. We examined the time course of Adra1a and Adrb1 expression following FC, using single-cell RNAscope on amygdalar sections collected at 1.5 h, 1 day, 2 days, 3 days and 14 days post-conditioning (Fig. Adrb1 expression was significantly increased from 1.5 h to 3 days post-conditioning (approximately twofold versus NoFC; Fig. 4e), but returned to baseline by 14 days. By contrast, Adra1a expression was increased at 1 day post-conditioning (Fig. We calculated a co-expression index for each cell (relative Adrb1 expression × relative Adra1a expression; Methods) and found a peak at 1 day post-FC (Fig. To test whether this molecular profile predicts the degree of astrocyte Fos induction during recall, we performed FR-BAE tagging at multiple time points post-FC (1 day, 2 days, 3 days and 14 days; Fig. Consistent with the adrenoreceptor co-expression time course (Fig. 4f), FR-BAE density peaked at 1 day and declined thereafter (Fig. 4h), suggesting a defined window for FR-BAE induction. Moreover, RNAscope further revealed strong Fos–Adrb1 correlation in FR astrocytes (Fig. To determine whether these adrenoreceptors are causally required for FR-BAE induction, we performed astrocyte-specific knockdown of Adra1a or Adrb1 in the amygdala (Fig. Knockdown of either receptor (Fig. 4m,n) significantly reduced FR-BAE density (Fig. 4o,p), indicating that both Gαq-coupled α1-receptors and Gαs-coupled β1-receptors are necessary for FR-BAE induction in vivo. Our earlier finding that the α1-receptor antagonist prazosin did not block NA-induced c-Fos induction in cultured astrocytes (Fig. 2d) probably reflects the downregulation of Adra1a in vitro17, highlighting the importance of in vivo validation. Collectively, our data support a model in which initial fear experience primes astrocytes to upregulate adrenoreceptor expression over time, creating a multiday molecular trace that allows future NA release upon repeated experiences (that is, recall) to selectively recruit the astrocyte ensemble. Transcriptomics enabled us to probe molecular changes of FR-BAEs linked to functional outputs. We mined differentially expressed genes between Adrb1-positive and Adrb1-negative astrocytes (Extended Data Fig. 12a) and identified Igfbp2 as highly upregulated in Adrb1-positive astrocytes (Extended Data Fig. Our single-cell RNAscope revealed highly correlated expression of Adrb1 and Igfbp2 (Extended Data Fig. 12c–e), suggesting that astrocytic Adrb1 upregulation could promote Igfbp2 induction. Igfbp2 encodes the secreted protein IGFBP2, which was recently reported to be enriched in ‘peri-neuronal engram' Fos+ astrocytes of the amygdala after fear recall22. Given that IGFBP2 and its binding partners, insulin-like growth factors (IGFs), are crucial for synaptic plasticity52,53 and that chronic deletion of IGFBP2 disrupts synaptic transmission, circuit maturation and memory22,52,54, we hypothesized that IGFBP2 expression in FR-BAELA/B contributes to circuit de-stabilization and re-stabilization upon recall2,3. We tested this hypothesis with a spatiotemporally specific in vivo pharmacological perturbation of IGFBP2 (Extended Data Fig. We bilaterally infused an IGFBP2-neutralizing antibody52 (IGFBP2-Ab; 1 mg ml−1) immediately after FR in CtxA—that is, within the reconsolidation window during which memories become labile to be weakened or strengthened2. Mice received repeated daily infusions and FR sessions for five days (Extended Data Fig. IGFBP2-Ab treatment reduced freezing compared with controls (Extended Data Fig. 12g,h), indicating impaired memory stabilization; controls infused with IgG showed no change. Combined, our transcriptomics, RNAscope and pharmacological perturbation data indicate that a subset of astrocytes increases adrenoreceptor expression upon fear experiences, enhancing NA responsiveness and recall-driven induction of Fos and Igfbp2 to stabilize fear memory. To directly examine the causal role of FR-BAE signalling in memory stabilization, we performed a loss-of-function experiment selectively targeting FR-BAEs in LA/B. We used iβARK, a genetically encoded 122-amino acid peptide that binds Gαq–GTP to suppress GPCR signalling26 (Extended Data Fig. In ex vivo Ca2+ imaging in amygdalar astrocytes, iβARK reduced α1-receptor-evoked Ca2+ responses relative to its mutant control, although some residual oscillatory activity persisted (Extended Data Fig. 13c,f,g), consistent with our prior findings in the striatum26. To improve efficacy, we developed iβARK2 by fusing the original peptide to a membrane-anchoring Lck domain, enabling more effective proximity-based inhibition (Extended Data Fig. iβARK2 showed robust membrane localization across astrocytic compartments, including fine processes (Extended Data Fig. Astrocytes expressing iβARK2 displayed significantly attenuated Ca2+ responses relative to both iβARK and the control iβARK2mut (Extended Data Fig. We next generated Cre-dependent AAV versions of iβARK2 and iβARK2mut for use in Fos-iCreERT2 TRAP mice to selectively target FR-BAEs (Fig. Remarkably, silencing FR-BAEs with iβARK2 impaired memory stabilization upon repeated recall (Fig. 5c), consistent with the behavioural effects observed upon neutralization of astrocyte-derived IGFBP2 (Extended Data Fig. Moreover, astrocytic IGFBP2 levels were significantly reduced in iβARK2-expressing mice compared with iβARK2mut controls (Fig. 5d), suggesting that suppression of GPCR signalling in FR-BAEs disrupts the recall-dependent transcriptional induction of IGFBP2 and compromises memory stabilization. a, Schematic of behavioural and RNAscope analysis. c, Reduced freezing at second recall in iβARK2 versus sustained freezing in iβARK2mut control (iβARK2mut: n = 13; iβARK2: n = 11 mice). Wilcoxon matched-pairs signed rank test. d, RNAscope images of Igfbp2 with mCherry–iβARK2 or mCherry–iβARK2mut and violin plots showing lower Igfbp2 expression in the presence of iβARK2 (iβARK2mut: n = 23; iβARK2: n = 24 cells; 4 mice per group). e, FR-BAE tagging with astrocytic Adrb1 overexpression. f, RNAscope images of Adrb1 and Igfbp2 in Slc1a3+ astrocytes with or without Adrb1 overexpression. Graphs showing increased expression with Adrb1-OE (ctrl: 501; Adrb1-OE: 1,810 cells; 18 sections from 6 mice per group). g,h, Images (g) and quantification (h) showing higher FR-BAE density with Adrb1 overexpression. i, Memory tests under weak (one-shock) versus strong (three-shock) conditioning. j, Adrb1 overexpression has no effect on freezing behaviour under strong conditioning (ctrl: n = 5; Adrb1-OE: n = 7 mice). Wilcoxon matched-pairs signed rank test. k, Under weak conditioning, Adrb1 overexpression enhanced freezing at second recall (n = 13 mice per group). Wilcoxon matched-pairs signed rank test. l, After strong conditioning, Adrb1 overexpression caused greater freezing in CtxB versus control (ctrl: n = 13; Adrb1-OE: n = 14 mice). m, FR engram labelling with mCherry and c-Fos immunohistochemistry 90 min after freezing test with or without Adrb1 overexpression. n,o, FR engram neurons exhibit similar density but higher reactivation with Adrb1 overexpression (ctrl: n = 15; Adrb1-OE: n = 12 slices; 3 mice per group). Sample sizes and replicates in Supplementary Table 1. To further explore the causal impact of BAE signalling on neuronal engram and memory formation, we performed a gain-of-function experiment targeting astrocytic β1-receptors in LA/B. Astrocytes transduced with AAV2/5-GfaABC1D-Adrb1 for Adrb1 overexpression (Adrb1-OE) showed significantly increased Igfbp2 expression compared with controls (Fig. Adrb1 overexpression also increased the density of FR-BAEs (Fos-mNG+ astrocytes) by around threefold relative to controls (Fig. We next functionally assessed the behavioural effect of the augmented astrocyte ensembles (Fig. We detected no differences in ambulation in an open arena or time spent in the centre zone during habituation between control and Adrb1-OE mice (Extended Data Fig. We then tested whether Adrb1 overexpression affected memory stabilization. With three-shock conditioning, both groups showed equivalent freezing during initial recall and second recall after reconsolidation (Fig. 5j), potentially owing to a ceiling effect. Adrb1-OE mice did not show altered freezing during FR one day after FC; however, Adrb1-OE mice exhibited significantly increased freezing during the second recall (Fig. 5k), indicating enhanced memory stabilization following the reconsolidation after initial recall. In another cohort with three-shock conditioning, initial recall showed no significant difference in freezing between the groups (Extended Data Fig. However, we observed significantly enhanced freezing in an altered context (CtxB) in mice with Adrb1 overexpression compared with controls (Fig. These data suggest that overstabilization of the initial fear memory leads to generalization or a loss of memory precision41,55,56,57. Finally, we explored whether fear generalization is accompanied by activity of engram neurons. Mice were euthanized 1.5 h after the CtxB freezing test and the reactivation of FR engram neurons was assessed by c-Fos immunoreactivity (Fig. The overall number of mCherry+ neurons (FR engram neurons) was not significantly different between groups, but the c-Fos levels in FR engram neurons in the Adrb1-OE mice, which showed fear generalization, were significantly higher than those in control mice (Fig. These findings suggest that the larger astrocyte ensembles contribute to enhance subsequent reactivation of engram neurons, consistent with the view that memory retrievability is not determined during memory formation, but may be regulated after memory retrieval8, and demonstrating that astrocytes are involved in that crucial process. These data demonstrate that astrocytes can link discrete fear experiences across at least a day-long timescale. As postulated more than a century ago58 and recently implicated in brain computation11,25,59,60, astrocytes may integrate information on slower timescales than neurons. Whereas neurons and astrocytes have been traditionally compared over milliseconds versus seconds-to-minutes, our findings reveal a functionally significant mismatch on the scale of days. Unlike neurons that express the immediate early genes during both FC and FR5,8,28, astrocytes respond to fear conditioning (FC) with rapid Ca2+ and cAMP increases61, but undergo slower transcriptional state changes62. Our single-cell transcriptomics and RNAscope revealed that multiple days post-FC, without recall, astrocytes upregulated Adra1a and Adrb1, marking a learning-dependent state that facilitates heightened responses during repeated experiences (FR). This in turn drives expression of Fos and Igfbp2, supporting memory stabilization. The astrocyte ensembles identified in this study may partially meet the criteria for engram cells8 that undergo enduring biochemical changes upon experiences. However, our findings suggest that the astrocytic ensemble may not serve as a direct memory trace encoding experience-specific information like neurons because its perturbation did not alter initial fear recall. Instead, it may function as an eligibility trace for memory stabilization over days, creating a permissive microenvironment within circuits that supports the stabilization of memories related to repeated experiences. From a computational and bioenergetic perspective, astrocytes may underlie retrospective linking63,64, which benefits from spaced intervals over days between learning sessions65 by providing a slow modulatory trace that complements fast neuronal activity64, thereby offloading the energetically costly task of maintaining firing patterns in neurons across days and enabling more stable circuit transformations60. Mounting evidence from animal3,4,57 and human imaging6 studies has highlighted the amygdala and NA in modulating long-term memory. Early pharmacological studies targeting protein synthesis66, β-receptors67, PKA35 or cAMP–CREB34 in the amygdala disrupted memory re-stabilization, which our findings indicate could partially be mediated by astrocytes. We show that astrocytes undergo learning-dependent increases in adrenoreceptor expressions, and manipulating these receptors alters memory stability, precision and neuronal engram representations. Notably, both NA input and reactivated neuronal engram activity are required to recruit astrocytic ensembles that stabilize labile memories. These findings imply that astrocyte ensembles may not simply be passive bulk substrates of NA signals, but instead exert behaviourally relevant, circuit-specific regulations through astrocyte-defined microcircuitry15,68. Disrupting the precise NA–astrocyte signalling through β1R overexpression led to fear generalization, suggesting physiological astrocytic roles in selective memory stability that constrain maladaptive updating. In this regard, targeting these astrocyte–neuron interaction mechanisms could be beneficial in treating cognitive and neuropsychiatric dysfunctions, including post-traumatic stress disorder, in which hyperactivity of the NA system is a prominent feature4,69. Deciphering how diverse astrocyte responses link to multiple circuit outputs has long been a challenge15. Considerable effort has been devoted to probe24 or perturb11 astrocyte Ca2+ signals since they were first reported70, yet causal relationships are usually inferred from separate subjects, leaving input–output links unclear. Astrocyte perturbation in a bulk manner is also incongruent with the heterogeneity of astrocyte identity49,71, state62,72 and response59,73. Our approaches bridge this gap: by exploiting context-specific astrocyte state changes18,20,22, the brain-wide BAE tagging method enables broad exploration of ensembles causal for neural processing and behaviour. This framework facilitates research into astrocytic ensembles across diverse behaviours and disease conditions. Expanding such efforts will accelerate discoveries in computation and pathology that require understanding of multicellular interactions as a core attribute of brain biology. Young adult (8- to 12-week-old) wild-type C57BL/6NCrSlc mice obtained from the Japan SLC, Fos2A-iCreERT2 (TRAP228, JAX 030323) mice obtained from the Jackson Laboratory, Adra1a flox mice49,74 obtained from RIKEN (RBRC11837) and Adrb1 flox mice50 (accession number CDB1068K, generated by RIKEN Center for Biosystems Dynamics Research) were used for in vivo experiments. TRAP2 mice were bred and maintained in our animal facility, backcrossed with C57BL/6NCrSlc at least eight times before use for experiments. Wild-type C57BL/6NCrSlc female mice were obtained from the Japan SLC to obtain postnatal day 0–1 (P0–P1) pups. Experiments were done using both male and female mice. Number of mice used in each experiment is listed in the main text and/or figure legends accordingly. Mice were housed in groups of 2–5 in a temperature- and humidity-controlled room (23 ± 3 °C, 45 ± 5% humidity) under a 12-h light:dark cycle (lights on from 07:00 to 19:00) and given ad libitum access to water and laboratory mouse diet at all times. After fibre implantation, the mice were single housed under a 12-h light:dark cycle (lights on from 07:00 to 19:00) and given ad libitum access to water and laboratory mouse diet at all times. After fibre implantation, the mice were singly housed. All experiments were approved by the RIKEN Animal Care and Use Committee. AAVpro 293T cells (Clontech, 632273) were maintained in Dulbecco's Modified Eagle Medium (DMEM; Gibco, 11995073) with fetal bovine serum (FBS; Cosmo Bio, CCP-FBS-BR-500) and 0.1% antibiotic-antimycotic (Gibco, 15240062). Cells were grown in a humidified cell culture incubation with 95% air, 5% CO2 at 37 °C. All plasmid constructs were generated using standard molecular biology techniques and the In-Fusion HD Cloning Kit (Clontech). The pZac2.1 GfaABC1D-Rpl22-HA plasmids (Addgene plasmid #111811) were digested with XhoI and NotI, which removed the Rpl22-HA sequence. Multiple cloning sequences (MCS) comprised of restriction enzyme sites that were absent from the backbone pZac2.1 plasmid were generated via gene synthesis (Eurofins Genomics) and incorporated to produce pZac2.1-GfaABC1D-MCS plasmids. The PCR-amplified DIO-GFP sequence from Addgene plasmid #28304 was cloned into the mammalian expression vector pCAG-tdTomato (Addgene #83029) between EcoRI sites to generate pCAG-DIO-EGFP-tdTomato. The PCR-amplified WPRE sequence from pFBAAV-CAG-XCaMP-R-WPRE (a gift from H. Bito12) was inserted into the NotI site of the resulting plasmid to generate pZac2.1-GfaABC1D-DIO-EGFP-tdTomato-WPRE plasmids. The plasmids were digested with NotI and AgeI, which removed the tdTomato sequence, and a PCR-amplified mScarlet-I sequence (Addgene #98816) was incorporated, and the resulting plasmids were digested with XhoI and KpnI, which removed the EGFP sequence, and the mNG sequence from Addgene #99135 was ligated using DNA ligation kit (Takara) to generate pZac2.1-GfaABC1D-DIO-mNG-mScarlet-I-WPRE plasmids. The mScarlet-I sequence was removed with restriction enzymatic digestion with PspOMI and NotI to generate pZac2.1-GfaABC1D-DIO-mNG-WPRE plasmids. The PCR-amplified hM4Di-mCherry sequence from Addgene #92286 was cloned into the PmeI site of pAAV-RAM-d2TTA::TRE-MCS-WPRE (Addgene #63931) to generate pAAV-RAM-d2TTA::TRE-hM4Di-mCherry-WPRE plasmids. The PCR-amplified XCaMP-Gf-WPRE from the pFBAAV-CAG-XCaMP-Gf-WPRE plasmid (a gift from H. Bito75) was cloned into the NcoI and HindIII site of pAAV-hSyn1-cyto-mRuby3-iATPSnFR1.0 (Addgene #102557) to generate pAAV-hSyn1-XCaMP-Gf-WPRE plasmids. All constructs were sequenced before use. All our constructs have been deposited to Addgene in the Nagai Lab repository for unrestricted distribution (http://www.addgene.org/Jun_Nagai, Supplementary Table 4). Recombinant AAVs were generated by triple transfection of AAVpro 293T cells (Clontech, 632273) using polyethylenimine (PEI; Polysciences inc., 24765-100). Cells were cultured up to 50–70% confluency in DMEM (Gibco, 11995073) with FBS (Cosmo Bio, CCP-FBS-BR-500) and 0.1% antibiotic-antimycotic (Gibco, 15240062). The plasmids and PEI were mixed with Opti-MEM (31985-088) and kept for 5 min at room temperature. The mixture was combined with a DNA mix which contained pAAV Capsids, pHelper (Agilent Technologies, 240071-54) and pAAV transgenes. All these plasmids had a concentration of 4.2 μg per dish. The DNA mix was then applied with PEI/Opti-MEM, and incubated for 20 min at room temperature. Then, this solution was added to the cells and incubated for 24 h. After incubation, the medium was changed to DMEM with 2% FBS and 0.1% antibiotic-antimycotic. Viral particles were collected from the medium at 120 h post-transfection. Cell pellets were centrifuged (3,500 rpm, 30 min, 4 °C). The supernatant was filtered by Setricup (Merc, S2HVU02RE) and pellets were resuspended in FBS-free DMEM and freeze-thawed three times. In the case of PHP.eB capsid virus production, the supernatant was resuspended in DMEM and fractured by harshly pipetting. Thawed pellets were centrifuged (3,500 rpm, 10 min, 4 °C) again and the supernatant was filtered by Setricup (Merc, S2HVU02RE). Media containing viral particles were centrifuged (22,000 rpm, 2 h, 4 °C) by Optima XE-100 (Beckman Coulter, A94516). The pellets were collected with 10–50 μl PBS and clarified by centrifugation (4,000 rpm, 10 min, 4 °C). The supernatant-containing viral particles were stored at 4 °C. AAV titre was analysed by quantitative PCR (QuantStudio 12 K; Applied Biosystems) using primers designed to selectively bind AAV2 inverted terminal repeats (forward: 5′-AACATGCTACGCAGAGAGGGAGTGG-3′, reverse: 5′-CATGAGACAAGGAACCCCTAGTGATGGAG-3′). All surgical procedures were conducted under general anaesthesia using continuous isoflurane (induction at 5%, maintenance at 1.5% to 2% vol/vol). Depth of anaesthesia was monitored continuously and adjusted when necessary. Intravenous administration of AAV-PHP.eB (50 µl to deliver 2.5 × 1012 genome copies (GCs) per mouse) was performed by injection into the retro-orbital sinus of TRAP2 mice. Two weeks after AAV-PHP.eB GfaABC1D-DIO-mNG-WPRE or AAV-PHP.eBGfaABC1D-DIO-mNG-mScaret-I-WPRE injections, mice were administered 4-OHT for inducing Cre recombination under the Fos promoter (Fos tagging) and then returned to their home cage until they were euthanized 7 days after tagging. In some cases, Fos tagging was performed using the FLP–FRT system (fDIO), and mice were euthanized one day after tagging. For stereotaxic microinjection of AAVs, mice were fitted into a stereotaxic frame (David Kopf Instruments) with their heads secured by blunt ear bars and their noses placed into a veterinary grade anaesthesia and ventilation system (VetEquip 911103). The surgical incision site was then cleaned with 70% ethanol (vol/vol). Craniotomies (1–2 mm in diameter) were made powered by a high-speed drill at specific coordinates based on the bregma position. Bilateral viral injections were conducted using a stereotaxic apparatus (David Kopf Instruments) to guide the placement of beveled glass pipettes (World Precision Instruments, 1B100-4) into the regions of interest: the basolateral amygdala (anterior–posterior (AP): −1.48 mm, medial–lateral (ML): 3.3 mm, dorsal–ventral (DV): 5.00 mm to bregma) and LC (AP: −5.4 mm, ML: 0.85 mm, DV: 3.7 mm to bregma). Total of 0.3 μl of AAV was delivered at 0.1 μl min−1 using a syringe pump (SmartTouch Pump, World Precision Instruments). Mice were then sutured, returned to their home cage, and allowed to rest for at least two to three weeks before any assessments were conducted. AAV titres (GCs per ml) were adjusted with sterile saline to deliver the indicated genome copies into the regions of interest: AAV2/5-GfaABC1D-DIO-mNG-WPRE (3.0 × 1012), AAV2/5-GfaABC1D-hM3Dq-mCherry (3.0 × 1012), AAV2/DJ8-RAM-d2TTA::TRE-hM4Di-WPRE (3.0 × 1013), AAV2/5-GfaABC1D-Flag-mAdrb1-WPRE (3.0 × 1012), AAV2/9-rTH-Cre (3.0 × 1012), AAV2/8-hSyn1-DIO-hM3Dq (1.0 × 1013), AAV2/8-hSyn1-DIO-hM4Di (1.0 × 1013), AAV2/8-hSyn1-DIO-mCherry (3.0 × 1012), AAV2/DJ8-Ef1a-hM3Dq (4.8 × 1011), AAV2/DJ8-Ef1a-hM4Di (4.0 × 1012), AAV2/5-hSyn1-mCherry-CAXX (3.0 × 1012) AAV2/9-hSyn1-GRABNE2h-WPRE (2.7 × 1012), AAV2/1 GfaABC1D-cAMPinG1-NE (1.3 × 1013), and AAV2/1 GfaABC1D-RCaMP3 (1.5 × 1013), GfaABC1D-GCaMP6f (1.1 × 1013), AAV2/5-GfaABC1D-iβARK-mCherry (1.3 × 1012), AAV2/5-GfaABC1D-iβARK(D110A)-mCherry (1.3 × 1012), AAV2/5-GfaABC1D-Lck-mCherry-iβARK (1.3 × 1012), AAV2/5-GfaABC1D-Lck-mCherry-iβARK(D110A) (1.1 × 1013), AAV2/5-GfaABC1D-DIO-Lck-mCherry-iβARK-4x6T (3.0 × 1013), AAV2/5-GfaABC1D-Lck-mCherry-iβARK(D110A) (3.0 × 1013), AAV2/5-Fos-FlpoERT2 (3.0 × 1012), AAV2/5-GfaABC1D-fDIO-mNG (1.0 × 1012), AAV2/5-GfaABC1D-Cre-4x6T (1.2 × 1013). 4-OHT (Sigma, H6278) was dissolved at 25 mg ml−1 in 100% ethanol with agitation at 37 °C for 15 min and was then aliquoted and stored at −20 °C for up to several weeks until use. Before use, 4-OHT was redissolved in ethanol by shaking at 37 °C for 15 min and corn oil (Sigma, 259853 and S5007) was added to give a final concentration of 3 mg ml−1 4-OHT. The ethanol was evaporated by incubation at 50 °C. The final 3 mg ml−1 4-OHT solutions were always used on the day they were prepared. All 4-OHT injections were delivered intraperitoneally at 50 mg kg−1 2–4 weeks after AAV injection into TRAP2 mice for inducing Cre recombination under the Fos promoter. CNO (Tocris, 4936) was administered intraperitoneally 0.5 h before the initiation of behavioural assessments. CNO was administered intraperitoneally 20 min before 4-OHT intraperitoneal injection. The final solutions were always used on the day they were prepared. Mice were placed on doxycycline (40 mg kg−1, Bio-Serv, F4159) chow 24 h prior to and for at least 1 week after viral injection, switched to doxycycline-free chow 48 h before recall, and euthanized 24 h later. IGFBP2 antibody was infused bilaterally through implanted guide cannulas targeting basolateral amygdala. Infusions were performed using an internal cannula connected to a 10-µl Hamiltion syringe. One microgram of IGFBP2 antibody (1 mg ml−1, 1 µl per site) was infused into mice immediately after a fear recall paradigm at 400 nl min−1 using a syringe pump (SmartTouch Pump, World Precision Instruments). Mice were deeply anaesthetized with isoflurane and transcardially perfused with PBS, followed by 4% paraformaldehyde (PFA) in PBS for fixation. The brains were embedded with OCT compound (Sakura Finetek Japan, 4583) and sliced into coronal sections with a thickness of 50 μm at −20 °C using cryostat (CryoStar NX70, Thermo Fisher Scientific, 956960). The sections were washed with PBS 3 times for 5 min each and stored in 50% glycerol (Fujifilm, 075-00616) in PBS at −20 °C. For staining with c-Fos antibody, the sections were permeabilized with PBS containing 1% Triton X-100 (Nacalai tesque, 35501), incubated at room temperature for 3 h, and then incubated with 0.3% Triton X-100 in PBS containing 1% normal donkey serum (NDS, Sigma-Aldrich, D9663) at room temperature for 1 h for blocking, and then incubated at 4 °C overnight in blocking buffer. With the following primary antibodies, Triton X-100 was used at 0.1% for permeabilization and PBS containing 1% NDS was used for blocking and antibody reactions: rat anti-GFAP (1:500, Thermo Fisher Scientific, 13-0300), mouse anti-S100β (1:500, Sigma-Aldrich, S2532), goat-Sox9 (1:100, R&D systems, AF3075) mouse anti-RFP (1:500, MBL, M155-3), rabbit anti-RFP (1:500, Rockland, 600-401-379), mouse anti-NET (1:2,000, MAb technologies, NET05-2) and rabbit anti-c-Fos (1:5,000, Synaptic Systems, 226003). After washing with PBS 3 times for 10 min each, the sections were incubated at room temperature for 2 h in blocking buffer with the following secondary antibodies: Alexa Fluor 488 donkey anti-rat, Alexa Fluor 488 donkey anti-chicken, Alexa Fluor 568 donkey anti-mouse, Alexa Fluor 568 donkey anti-rabbit, Alexa Fluor 568 donkey anti-goat, Alexa Fluor 647 donkey anti-rabbit, Alexa Fluor 647 donkey anti-mouse, Alexa Fluor 647 donkey anti-chicken (1:1,000, Abcam, ThermoFisher and Jackson ImmunoResearch). The sections were incubated at 4 °C overnight in blocking buffer with secondary antibodies when staining with anti-c-Fos antibody. After washing with PBS 3 times for 10 min each, the sections were mounted on slides (Fluoromount-G, Southern bio, 0100-35) and visualized under a confocal laser microscope (Olympus, FV3000) and a fluorescent microscope (Olympus, VS200). Immunoreactivities were analysed using ImageJ software (NIH). 8b) and astrocytes (Extended Data Fig. 4b) were quantified based on a consistent c-Fos intensity threshold for each experiment. This threshold was defined such that the number of c-Fos+ neurons or astrocytes corresponded to the values previously reported18,42,77. Number of cells, field of views, sections and mice used in each analysis is listed in the main text and/or figure legends accordingly. Mice were returned to the dark chamber for an additional two days and then returned to a regular light-dark cycle until the time of euthanasia seven days after tagging. Behavioural tests were performed during the light cycle between 07:00 and 19:00. All the experimental mice were transferred to the behaviour testing room at least 1 h before the tests to acclimatize to the environment and to reduce stress. Mice were habituated by freely exploring the conditioning chamber (CtxA) that consisted of a square partition (30 × 30 × 15 cm; 100 lux and no odour) with a grid floor wired to a shock generator (Ohara) for 15 min per day for 3 days. On the fourth day, they were either fear conditioned (1 time or 3 times shock; 2 s, 0.3 mA) (FC group) or presented with no shocks (NoFC group) in the conditioning chamber. One day after the conditioning, mice were either returned to CtxA for 3 min for fear memory recall (FR group) or kept in their home cages (NoFR group). Freezing durations exceeding 70% in the conditioned chamber 1 day post-conditioning with 3 foot shocks or exceeding 50% 1 day post-conditioning with a single foot shock, are considered indicative of sufficient conditioning. All the experimental mice received intraperitoneal injections of 4-OHT (50 mg kg−1) immediately after the task for Fos tagging. Freezing was automatically measured throughout the testing trial by the TimeFZ4 software (version 2021_7_29, Ohara) and subsequently processed by a blinded investigator. The chambers were cleaned with 70% ethanol between tests. In some cases, 10 mg kg−1 Propranolol (MedChemExpress, HY-B0573) was administered by intraperitoneal injection to mice 0.5 h before, and 2.5 h and 5.5 h after 4-OHT-based Fos tagging in order to block β-adrenergic signalling throughout the tagging timeframe. In some experiments using DREADDs, mice were injected with CNO (1 mg kg−1) 10 min before the task in CtxA. 2n in which CNO was used for silencing neuronal activity or propranolol was used for antagonizing β-adrenoreceptor signalling to test their roles in astrocyte Fos induction during fear memory recall, both were administered with consideration of their pharmacokinetics. Propranolol was given both before and after the behavioural session to ensure effective β-receptor blockade throughout the several-hour Fos-tagging window (Extended Data Fig. 3), given its short half-life78 (~1.5–2 h), whereas CNO was given once before the session owing to its prolonged effect79 (>6 h). Freezing was measured for 3 min in the CtxB chamber, which consisted of a rhombic partition (30 cm × 60 cm × 15 cm, 10 lux and 1% acetic acid odour) with a flat floor (Ohara). Mice were deeply anaesthetized with isoflurane and transcardially perfused with PBS, followed by 4% PFA in PBS for fixation of brain tissues. To oxidize agarose, 2.25 g agarose (A6013, Sigma-Aldrich) was added to 100 ml of 10 mM sodium periodate (S1878, Sigma-Aldrich)/50 mM phosphate buffer and stirred gently for 2.5 h at room temperature in the dark. The oxidized agarose was filtered on a bottle-top vacuum filter (0.2 μm, 569-0020, Thermo Scientific) washed 3 times with phosphate buffer and resuspended in 50 ml of phosphate buffer to make 4.5% oxidized agarose solution. The oxidized agarose solution was boiled with a microwave oven to melt the agarose and kept at 60–65 °C. The brains were pat-dried and embedded in oxidized agarose solution with a plastic mould (18646A, Polysciences). For covalent crosslinking between oxidized agarose and the surface of the brain, borate/borohydride solution was prepared the day before use by adding 0.2 g sodium borohydride (452882, Sigma-Aldrich) in 100 ml of prewarmed (40 °C) borate buffer (50 mM borax, 221732, Sigma-Aldrich; 50 mM boric acid, B6768, Sigma-Aldrich; pH 9.0–9.5) in the chemical fume hood. The agarose block was removed from the mould, soaked in borate/borohydride solution to initiate crosslinking and incubated at 4 °C overnight. The block was washed and kept in phosphate buffer at 4 °C until imaging. The agarose block was mounted on a glass slide using Loctite 401 glue immediately before imaging. STPT was performed with the TissueCyte 1400FC system (TissueVision) equipped with Coherent Chameleon Ultra II laser (Coherent) or Spectra-Physics Insight X3+ laser (MKS Spectra-Physics) and a Nikon 16× water-immersion objective (MRP07220, NA: 0.8). The laser wavelength was tuned at 1,000 nm. Brains were coronally sectioned with a built-in vibrating microtome at 50-μm thickness with a frequency of 60 Hz and a speed of 0.5 mm s−1. To scan an entire coronal plane, individual tile images were acquired (832 × 832 pixels per frame, resolution at 1.382 μm per pixel) by moving the stage between each tile. Following the full scanning of a coronal plane, a 50-μm section was cut away by vibrating microtome and imaging of a new cut surface repeated to acquire optical sections covering the entire brain from the caudal to the rostral side (~320 coronal sections). The raw image tiles were first trimmed and subjected to flat field correction, then stitched into 2D mosaic coronal section images using the Autostitcher software (TissueVision). Each region of the whole brain was aligned and segmented according to the Mouse Brain Atlas (NeuroInfo, MBL Bioscience, BN-200). Owing to the high quality preservation of tissue architecture during imaging, we semi-automatically performed alignment to a standard mouse brain atlas, segmentation of 839 brain regions, and detection of single Fos-tagged mNG astrocytes (BAEs), defined by their territories that typically displayed diameters of approximately 40 to 60 μm with ~10-µm-wide cell bodies68 (Fig. Consistent with our intersectional genetic approach, we rarely observed neuron-like mNG-positive cells, with larger cell bodies (>~20-μm diameter) and elongated processes (>~50-μm). We only found these cells in regions nearby ventricles and the midline where well-characterized GFAP-positive neural progenitor cells reside48,80, and they were excluded in subsequent analyses. As a result, using this brain-wide analysis approach we successfully quantified the density of BAEs in 677 brain regions that accounted for 80.7% of the whole brain. To observe correlation between Fos+ astrocytes and neurons, we used the same TRAP2 (Fos-iCreERT2) system28 used by Roy et al.5, but instead with a Cre-dependent mNG reporter expressed in Fos+ astrocytes, allowing comparison of Fos+ cell counts from anatomically matched regions between studies. The regions of interest were first narrowed to the 117 engram-signifiant regions defined by Roy et al. Then, to ensure analysis of bona fide astrocyte-tagged regions, areas with possible neuronal leakage (mNG+ neuron-like cells), particularly near ventricles were excluded (see above). In addition, high-level hierarchical structure areas (such as midbrain or thalamus) with extremely high total counts that skew correlation analysis were also excluded. In the end, this selection yielded 75 regions for subsequent correlational analyses. Changes in Fos+ astrocyte numbers (FC–NoFC and FR–NoFC) were calculated using the non-foot shocked control group as the baseline, consistent with Roy et al. Supplementary table 1 from Roy et al. provided region-by-region counts of Fos+ neurons in non-foot shocked controls, FC and FR groups, enabling us to extract the corresponding changes in Fos+ neuron numbers for each of the 75 regions. Pearson correlation analyses between these neuronal and astrocytic Fos+ cell count changes were performed. For image visualization in 3D, low-resolution image stacks were created across all four channels. These stacks were downsampled to 25 μm per pixel resolution in the x and y axes, while the inter-slice distance was maintained at 50 μm, resulting in a voxel resolution of 25 × 25 × 50 μm3. This template, generated from STPT averaged over 1,675 specimens, was combined with corresponding atlas annotations (filename: P56_Annotation.nii.gz). Image alignment was performed using the Advanced Normalization Tools (ANTs)82. This was followed by deformable (SyN) registration at four scales (downscaling factors of 8, 4, 2 and 1) with Gaussian smoothing of 4, 2, 1 and 1 voxels and 1,000, 500, 100 and 50 iterations, respectively, using mutual information as the metric. After computing the transformation, individual image stacks and atlas labels were mapped to and from the CCFv3 image space. After aligning with the NIfTI world coordinates, cell positions were mapped to the Allen template using the antsApplyTransformsToPoints function from ANTs. Brain regions from the Allen atlas annotations were then assigned to individual cells based on their locations in the CCFv3 image space. Astrocyte primary cultures were obtained as previously described with some modifications83. In brief, cerebral cortical tissues were isolated from P0–P1 neonatal mice and dissociated with a scalpel, gently agitated in 0.05% Trypsin-EDTA (Gibco, 25300-054), and rapidly strained with a Falcon 100 µm cell strainer (352360). Dissociated cells were seeded at a rate of four brains per bottle on poly-d-lysine-coated bottles. The cells were cultured in DMEM (Gibco, 11995073) supplemented with FBS (Cosmo Bio, CCP-FBS-BR-500) and penicillin-streptomycin. When the cells reached confluence after several weeks, the culture bottles were gently shaken at 120 rpm overnight at 37 °C to remove non-astrocytic cells. Astrocytes were then detached using Trypsin-EDTA, replated onto poly-d-lysine-coated bottles, and after 1–2 weeks, cultures were shaken again at 120 rpm for 4 h. Astrocytes were then re-seeded on coverslips in 12-well plates at a density of 1.0 × 105 cells per well, which were maintained in 5% CO2 at 37 °C, with the medium being changed every 2 to 3 days. Five days after re-seeding, primary astrocytes were incubated with tetrodotoxin (TTX at 1 µM, Fujifilm, 207-15901) to eliminate the potential effect of neuronal firing, for at least 5 min to allow adequate equilibration, together with the following inhibitors, blockers or chelators indicated for each experiment unless otherwise described: prazosin hydrochloride (MedChemExpress, HY-B0193A), atipamezole hydrochloride (Tokyo Chemical Industry, A2956), propranolol hydrochloride (MedChemExpress, HY-B0573), BAPTA-AM (SIGMA, A1076), and SQ 22536 (Tokyo chemical industry, Q0105). The cultures were then incubated with the following growth factors, neurotransmitters, neuromodulators or neuropeptides in 5% CO2 at 37 °C for 1.5 h: l-glutamic acid monosodium salt hydrate (Sigma-Aldrich, G5889), GABA (Sigma-Aldrich, A2129), ATP disodium salt hydrate from yeast (Nacalai tesque, 01072-24), l-noradrenaline hydrochloride (Sigma-Aldrich, 74480), dopamine hydrochloride (Sigma-Aldrich, H8502), acetylcholine chloride (Sigma-Aldrich, A6625), serotonin hydrochloride (Alfa Aesar, B21263.03), brain derived neurotrophic factor (BDNF, Fujifilm, 028-16451), neurotrophin-3 (Fujifilm, 146-09231, 50 ng ml−1), neurotrophin-4 (Fujifilm, 142-06634, 100 ng ml−1), forskolin (Fujifilm, 067-02191, 10 μM), nerve growth factor-β (NGFβ, Fujifilm, 141-07601, 100 ng ml−1), epidermal growth factor (EGF, Fujifilm, 059-07873, 100 ng ml−1), PDGF-AA (Fujifilm, 163–19731, 100 ng ml−1), PDGF-BB (Fujifilm, 164–24031, 100 ng ml−1), Ciliary Neurotrophic Factor (CNTF, Fujifilm, 032-23501, 100 ng ml−1), IGF1 (Fujifilm, 099-04511, 100 ng ml−1). The following agonist and ionophore were used: isoproterenol (MedChemExpress, HY-B0468) and ionomycin (LKT laboratories, I5752). Ninety minutes after incubation, primary astrocytes were fixed in 4% PFA (Nacalai Tesque) for 10 min, washed with PBS 3 times, permeabilized with 0.1% Triton X-100 (Nacalai Tesque) for 10 min and blocked with 1% NDS (Sigma-Aldrich, D9663) for 30 min. Primary astrocytes were then incubated overnight at 4 °C with the following primary antibodies: rabbit anti-c-Fos (1:1,000, Synaptic Systems, 226003), rat anti-GFAP (1:1,000, Thermo Fisher Scientific, 13-0300), rabbit anti-IBA1 (1:1,000, Fujifilm, 019-19741), chicken anti-TUJ1 (1:1,000 Novus Biologicals, NB100-1612), mouse anti-OLIG2 (1:100, Millipore, MABN50). The cultures were washed 3 times in PBS for 10 min each, then incubated with DAPI (1:10,000, Sigma-Aldrich, D9542) and the following secondary antibodies (1:1,000) in PBS containing 1% NDS for 1 h at room temperature: Alexa Fluor 488 donkey anti-rat (abcam, ab150153), Alexa Fluor 568 donkey anti-mouse (abcam, ab175700), Alexa Fluor 568 donkey anti-rabbit (Abcam, ab175692), Alexa Fluor 647 donkey anti-rabbit (abcam, ab150067), Alexa Fluor 647 donkey anti-chicken (Jackson ImmunoResearch, 703-605-155). Fluorescent images were taken using a 20× objective lens (UPlanXApo 20×, Olympus) with a confocal laser-scanning microscope (Olympus, FV3000 with Fluoview). Immunoreactivity quantification of c-Fos was performed using ImageJ. In some experiments involving a heterologous expression of mGluR3, we applied AAV2/5-GfaABC1D-Grm3 (800,000 multiplicity of infection) 5 h after re-seeding for expressing mGluR3 in primary astrocytes and incubated for 5 days before pharmacological experiments. Ninety minutes after stimulation with 0.1 µM NA and/or 100 µM glutamate in the presence of 1 µM TTX, primary astrocytes were fixed and stained as described above using primary antibodies: rabbit anti-mGluR3 (1:500, abcam, ab166608) and mouse anti-c-Fos (1:50, Santa Cruz Biotechnology, sc166940). It is noteworthy that the mouse anti-c-Fos antibody, different from the rabbit anti-c-Fos antibody used in other experiments, exhibited overall low immunoreactivity. However, this did not affect the quantification of c-Fos mobilization. Fluorescent imaging and quantification of c-Fos intensity were performed as described above. AAVs for expressing the noradrenaline biosensor GRABNE2h on the neuronal cell surface, or calcium indicator RCaMP3 in astrocytes with cAMP sensor cAMPinG1 in astrocytes were unilaterally injected into the basolateral amygdala. Optical fibres were implanted above the injection site two weeks post-injection to capture emission of fluorescence signals, with recordings conducted one week after fibre implantation. For fluorescence signal collection, mice were tethered to a patch cable, and a 465 nm LED light (Doric, 40–50 µW at the tip of patch cable) or a 560 nm LED light (Doric, 40–50 µW at the tip of patch cable) was used to excite the signal, with an isosbestic 405 nm LED light used to correct for movement artifacts. Mice tethered to the patch cable were habituated in a testing chamber for 5 min on the day of recording. A time-division multiplexing scheme was used, where the 465 nm, 560 nm and 405 nm LED lights were emitted alternately at 20 Hz (turned on for 24 ms and off for 26 ms), with the timing precisely controlled by a programmable pulse generator (Master-9, AMPI). Each excitation light was reflected by a dichroic mirror and coupled into an optical fibre cable (Doric, 200 μm in diameter, 2 m in length, NA 0.57) through a pinhole (200 μm in diameter). The fluorescence signals were detected by a photomultiplier tube with a GaAsP photocathode (H16722-40; Hamamatsu Photonics) and digitized by a data acquisition module (USB-6341, National Instruments). A custom-written LabVIEW program extracted signals at a sampling frequency of 1 kHz in real time. Fluorescence signals collected during each light pulse were averaged, after dropping the first and last 2 samples (1 ms each) to eliminate potential noise caused during the alternation of LED lights. The data were smoothed using moving average and corrected for bleaching correction by fitting to least-squares linear. The baselines of GRABNE2h and astrocyte cAMPinG1 signals were identified during periods of no spontaneous signals in home cages for 1–2 min before the context exposure. Signal analyses were performed in Python (v.3.0.0) using JupyterLab (v.3.6.7, Project Jupyter). The area under the curve of fluorescence traces was calculated with scikit-learn (sklearn, v.1.2.2, RRID:SCR_002577), and half-decay times of signal responses were extracted. Astrocytic RCaMP3 signals were bandpass filtered between 0.0003 and 1 Hz with SciPy85 (v.1.11.4). Peaks of Ca2+ signals were defined as data points exceeding 4× s.d. above the baseline mean, with baseline noise estimated from ≥20 s intervals lacking spontaneous activity in home cage recordings before context exposure. Signal traces were visualized with Matplotlib86 (v.3.8.0). Coronal amygdalar slices were prepared from adult mice with AAV injection26,87. In brief, animals were deeply anaesthetized with isoflurane and decapitated with sharp shears. The brains were placed and sliced in ice-cold modified artificial cerebrospinal fluid (ACSF) containing the following (in mM): 194 sucrose, 30 NaCl, 4.5 KCl, 1 MgCl2, 26 NaHCO3, 1.2 NaH2PO4 and 10 d-glucose, saturated with 95% O2 and 5% CO2. A vibratome (DSK NLS-MT) was used to cut 300-µm brain sections. The slices were allowed to equilibrate for 30 min at 32–34 °C in normal ACSF containing (in mM); 124 NaCl, 4.5 KCl, 2 CaCl2, 1 MgCl2, 26 NaHCO3, 1.2 NaH2PO4 and 10 d-glucose continuously bubbled with 95% O2 and 5% CO2. Slices were then stored at 21–23 °C in the same buffer until use. All slices were used within 2–6 h of slicing. Cells for all the experiments were imaged with a 16× water-immersion objective lens (Nikon, N16XLWD-PF, MRP07220, NA: 0.8) with Nikon two-photon laser-scanning microscope (A1R MP) equipped with a 920 nm wavelength laser (Alcor, Spark Lasers). To image GCaMP6f signals, astrocytes located in the LA/B and typically ∼20 to ∼30 µm below the slice surface were selected for imaging. Images were acquired at 1 frame per second. Phenylephrine (PE, Tocris Bioscience 2838) was applied in the bath in the presence of 300 nM tetrodotoxin (Cayman Chemical 14964) were applied in the bath at least 5 min prior to recording to allow adequate equilibration. A constant flow of fresh buffer perfused the imaging chamber at all times. Spontaneous Ca2+ activity was recorded for 3 min (0 µM PE), followed by a 2-minute bath application of 1 µM PE, a 5-min washout, a 2-min application of 3 µM PE, and a 5-min washout. Ca2+ signals were processed in ImageJ (NIH) and presented as the relative change in fluorescence (ΔF/F). Mice were anaesthetized with isoflurane 90 min after the initiation of behavioural paradigms and transcardially perfused with PBS (30 ml for 2 min). The amygdala was freshly dissected out using a Brain Matrix (Ted Pella, 15067) and was placed into cold Hanks' balanced salt solution (HBSS) (14025092, Gibco) containing 10 mM HEPES (17514-15, Naclai Tesque), 0.54% glucose (16806-25, Naclai Tesque), 5.0 µg ml−1 actinomycin D (A1410, Sigma-Aldrich), 10 µM triptolide (T3652, Sigma-Aldrich) and 27.1 µg ml−1 anisomycin (A9789, Sigma-Aldrich), as previously described88,89. Then, the tissue block was incubated for 15 min at 37 °C in 3 ml enzyme solution (HBSS, 10 mM HEPES, 0.54% glucose, 0.5 mM EDTA (15575020, Invitrogen), 1.0 mM l-cysteine (10309-41, Naclai Tesque), 5.0 µg ml−1 actinomycin D, 10 µM triptolide and 27.1 µg ml−1 anisomycin) containing 20 U papain per mouse (LS003127, Worthington Biochemical). The tissue was then homogenized by passing through a 18 G needle and was further incubated for 15 min at 37 °C. Following homogenization using a 20 G needle, the resulting cell suspension was centrifuged at 500g for 5 min at 4 °C. The homogenate was separated by gradient centrifugation with 30% Percoll (P1644, Sigma-Aldrich) in PBS at 500g for 25 min at 4 °C (no brake). The pellet containing astrocytes at the bottom of the tube was then collected and washed once with PBS containing 2% FBS and 10 mM EDTA before staining. Fc receptors were blocked with Fc block (2.4G2, BD Bioscience) for 10 min at 4 °C before incubation with primary antibodies. Cells were stained with antibodies directed against CD11b-BV786 (1:200, M1/70, BD Bioscience), CD45-APC-Cy7 (1:200, 30-F11, BioLegend), O1-eFluor660 (1:200, 50-6506-82, eBioscience) and ASCA2-PE-Cy7 (1:200, 130-116-246, Miltenyi Biotec) for 40 min at 4 °C. Additionally, cells were treated with hashtag antibodies to label source samples (TotalSeqB0301-B0310, B0312 and B0314, BioLegend). After washing, cells were sorted using a CytoFLEX SRT (Beckman Coulter). Data were acquired with CytExpert software (Beckman Coulter). Post-acquisition analysis was performed using FlowJo software, version 10.9.0. Single-cell libraries were generated and sequenced on the Illumina NovaSeq 6000 sequencer. Sequence reads were processed and aligned to the mouse genome (mm10, GENCODE vM23/Ensembl98; https://cf.10xgenomics.com/supp/cell-exp/refdata-gex-mm10-2020-A.tar.gz) using CellRanger 3.0. Initially, genes expressed in <3 cells and cells with <100 genes and mitochondrial reads <6% were excluded from the analysis. Pericytes were removed from amygdalar cells based on Pdgfrb. Hashtag oligos (HTOs) were demultiplexed using HTODemux and singlet cells were used for subsequent analysis. Principal component analysis was performed on the expression data matrix using the Seurat. Thirty principal components were used for generating tSNE plots. Transcriptomic clusters were identified from a total 1,903 amygdalar astrocytes with resolution set to 0.4. Differentially expressed gene candidates from a comparison of Adrb1-positive and Adrb1-negative astrocytes were estimated using Mann–Whitney–Wilcoxon test. Cryosections were prepared as described above and stored at −80 °C. RNAscope in situ hybridization was performed using RNAscope Multiplex Fluorescent Reagent Kit v.2 with TSA Vivid Dyes (ACDBio, 323270), according to the manufacturer's instructions. The sections were washed for 5 min with PBS, and then heated at 60 °C for 30 min before post-fix. After dehydration in an ethanol series, hydrogen peroxide was applied for 10 min at room temperature. The sections were washed with ddH2O twice, and then incubated in Target Retrieval Reagents for 5 min at 90–100 °C. A wash with ddH2O for 15 s was followed by a dehydration with 100% ethanol for 3 min and dried at room temperature. Then, the sections were incubated with Protease III for 30 min at 40 °C. Thereafter, the sections were washed twice with wash buffer for 2 min between incubations at 40 °C. In the staining of mCherry, slices were washed in 0.1 M PBS 3 times for 10 min each, followed by immunohistochemistry that was performed as described above. Following primary and secondary antibodies used: rabbit anti-RFP (1:250, Rockland, 600-401-379), Alexa Fluor 568 donkey anti-rabbit (1:1,000, Abcam, ab175692). Finally, the sections were incubated with DAPI at room temperature, removed DAPI, and mounted in Prolong Gold Antifade Reagent (Invitrogen, P36934). Images were obtained in the same way as immunohistochemistry described above except with a step size of 0.5 µm. Astrocyte somata were delineated using Slc1a3 signals. To account for differences in somata sizes, the signals were normalized using Slc1a3. Subsequently, the normalized signals were z-scored within each sample to evaluate gene expression across a heterogeneous cell population. This approach facilitated the robust identification of cells exhibiting high gene expression relative to the population mean. 4e,f, adrenoreceptor-positive cells were identified based on scRNA-seq data, selecting the top 50% of cells ranked by Adra1a expression and the top 20% of cells ranked by Adrb1 expression. Co-expression index of Adra1a and Adrb1, calculated by relative expression level of Adra1a to NoFC median multiplied by relative expression level of Adrb1 for each cell. 4j, the correlation analysis of Fos and Adrb1, expression levels were evaluated using z-scores of RNAscope signal intensity to assess gene expression across a heterogeneous population of cells, enabling robust identification of cells with high gene expression relative to the population mean. Specifically, we identified astrocytes with z-scores greater than 2 in RNAscope Fos signal intensity as Fos-high astrocytes, capturing 3% of the astrocyte population with significantly elevated Fos expression, consistent with scRNA-seq data in Fig. Similarly, Adrb1-high astrocytes were identified using z-scores greater than 0.5, capturing 26% of astrocytes, also in line with scRNA-seq data in Fig. In addition, Igfbp2-high astrocytes were identified using z-scores greater than 0.5, capturing 24% of all astrocytes in Extended Data Fig. 12d,e, which is consistent with scRNA-seq data indicating that Igfbp2-positive astrocytes occupy 24% of the total astrocytes analysed. Most statistical tests were run in Prism 10.1.2 (GraphPad). The specific tests used are indicated in the figure legends and Supplementary Table 1. All tests were two-sided unless otherwise noted. Astrocyte Fos count analyses across groups for each brain region were run with R v.4.3.2 on RStudio v.2023.12.1.402 and relied on the tidyverse, car, dunn.test and dplyr packages and base R functions aov(), TukeyHSD() and p.adjust(). The data were first analysed with Levene's test for homogeneity of variance. Brain regions that passed Levene's test (P > 0.05) were followed by a one-way ANOVA test with Tukey's honest significant difference test and regions that failed Levene's test (P < 0.05; no homogeneity of variances) were followed by the Kruskal–Wallis test with Dunn's post hoc test and Bonferroni adjustment. False discovery rates were calculated across groups and brain regions from post hoc P values using the Benjamini–Hochberg method. Mice were randomly assigned to experimental groups, and mice assigned to different experimental conditions were run in parallel. Mouse experiments and analyses were done blinded to group allocation. Tissue samples that yielded insufficient cell counts (less than 30 cells) during scRNA-seq quality control were excluded from the study. The results of statistical comparisons, n numbers and P values are shown in the figure panels or figure legends along with averages. n is defined as the numbers of cells, sections or mice throughout on a case-by-case basis; the unit of analysis is stated in the text or in each figure legend. In the figures, summarized data are shown as mean ± s.e.m. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All the single-cell transcriptomics data are available at Gene Expression Omnibus with accession identifier GSE272414. All raw replicate data used to generate the figures and the associated statistical tests are provided in Supplementary Table 1. All raw data supporting the findings of this study are available on the RIKEN CBS data sharing platform (https://neurodata.riken.jp/id/20250831-001; https://doi.org/10.60178/cbs.20250831-001). Custom code for 3D rendering of the brain-wide Fos-mNG astrocyte map is available on GitHub (https://github.com/BrainImageAnalysis/Astrocyte3DMapping/). Custom code for statistical analysis brain-wide Fos-mNG astrocyte counts, fibre photometry data analysis and single-cell transcriptomic analysis is available on GitHub (https://github.com/Jun-Nagai-Lab/Dewa-et-al). Misanin, J. R., Miller, R. R. & Lewis, D. J. Retrograde amnesia produced by electroconvulsive shock after reactivation of a consolidated memory trace. & Sara, S. J. Reconsolidation of memory after its reactivation. The restless engram: consolidations never end. Lee, J. L. C., Nader, K. & Schiller, D. An update on memory reconsolidation updating. Roy, D. S. et al. Brain-wide mapping reveals that engrams for a single memory are distributed across multiple brain regions. Agren, T. et al. Disruption of reconsolidation erases a fear memory trace in the human amygdala. & Nader, K. Amyloid beta secreted during consolidation prevents memory malleability. Flavell, S. W. & Greenberg, M. E. Signaling mechanisms linking neuronal activity to gene expression and plasticity of the nervous system. & Lyons, D. A. Glia as architects of central nervous system formation and function. Nagai, J. et al. Behaviorally consequential astrocytic regulation of neural circuits. Endo, F. et al. Molecular basis of astrocyte diversity and morphology across the CNS in health and disease. Molofsky, A. V. et al. Astrocyte-encoded positional cues maintain sensorimotor circuit integrity. Siletti, K. et al. Transcriptomic diversity of cell types across the adult human brain. Murphy-Royal, C., Ching, S. & Papouin, T. A conceptual framework for astrocyte function. Induction of astrocytic Slc22a3 regulates sensory processing through histone serotonylation. Sardar, D. et al. Mapping astrocyte transcriptional signatures in response to neuroactive compounds. Williamson, M. R. et al. Learning-associated astrocyte ensembles regulate memory recall. Yu, X. et al. Context-specific striatal astrocyte molecular responses are phenotypically exploitable. Lee, H.-G. et al. Disease-associated astrocyte epigenetic memory promotes CNS pathology. Escartin, C. et al. Reactive astrocyte nomenclature, definitions, and future directions. Sun, W. et al. Spatial transcriptomics reveal neuron-astrocyte synergy in long-term memory. Making sense of astrocytic calcium signals - from acquisition to interpretation. Mu, Y. et al. Glia accumulate evidence that actions are futile and suppress unsuccessful behavior. Specific and behaviorally consequential astrocyte Gq GPCR signaling attenuation in vivo with iβARK. Engineered AAVs for efficient noninvasive gene delivery to the central and peripheral nervous systems. DeNardo, L. A. et al. Temporal evolution of cortical ensembles promoting remote memory retrieval. Nagai, J. et al. Hyperactivity with disrupted attention by activation of an astrocyte synaptogenic cue. Adamsky, A. et al. Astrocytic activation generates de novo neuronal potentiation and memory enhancement. Fanselow, M. S. Factors governing one-trial contextual conditioning. Serial two-photon tomography for automated ex vivo mouse brain imaging. Piantadosi, S. C. et al. Holographic stimulation of opposing amygdala ensembles bidirectionally modulates valence-specific behavior via mutual inhibition. Long-term memory is facilitated by cAMP response element-binding protein overexpression in the amygdala. Tronson, N. C., Wiseman, S. L., Olausson, P. & Taylor, J. R. Bidirectional behavioral plasticity of memory reconsolidation depends on amygdalar protein kinase A. Nat. Lin, Y. et al. Activity-dependent regulation of inhibitory synapse development by Npas4. Smith, N. A. et al. Fluorescent Ca2+ indicators directly inhibit the Na,K-ATPase and disrupt cellular functions. Rapid manifestation of reactive astrogliosis in acute hippocampal brain slices. Sun, W. et al. Glutamate-dependent neuroglial calcium signaling differs between young and adult brain. Pfeil, E. M. et al. Heterotrimeric G protein subunit Gαq is a master switch for Gβγ-mediated calcium mobilization by Gi-coupled GPCRs. Sun, X. et al. Functionally distinct neuronal ensembles within the memory engram. Sørensen, A. T. et al. A robust activity marking system for exploring active neuronal ensembles. Uematsu, A. et al. Modular organization of the brainstem noradrenaline system coordinates opposing learning states. Monitoring norepinephrine release in vivo using next-generation GRABNE sensors. Yokoyama, T. et al. A multicolor suite for deciphering population coding of calcium and cAMP in vivo. Suthard, R. L. et al. Basolateral amygdala astrocytes are engaged by the acquisition and expression of a contextual fear memory. Batiuk, M. Y. et al. An immunoaffinity-based method for isolating ultrapure adult astrocytes based on ATP1B2 targeting by the ACSA-2 antibody. Furube, E. et al. Neural stem cell phenotype of tanycyte-like ependymal cells in the circumventricular organs and central canal of adult mouse brain. Kohro, Y. et al. Spinal astrocytes in superficial laminae gate brainstem descending control of mechanosensory hypersensitivity. In vivo effects of cardiomyocyte-specific β-1 blockade on afterload- and frequency-dependent cardiac performance. Gleichman, A. J., Kawaguchi, R., Sofroniew, M. V. & Carmichael, S. T. A toolbox of astrocyte-specific, serotype-independent adeno-associated viral vectors using microRNA targeting sequences. Caldwell, A. L. M. et al. Aberrant astrocyte protein secretion contributes to altered neuronal development in multiple models of neurodevelopmental disorders. Tu, X. et al. Local autocrine plasticity signaling in single dendritic spines by insulin-like growth factors. Khan, S. et al. IGFBP2 plays an essential role in cognitive development during early life. Han, J.-H. et al. Increasing CREB in the auditory thalamus enhances memory and generalization of auditory conditioned fear. Ramsaran, A. I. et al. A shift in the mechanisms controlling hippocampal engram formation during brain maturation. McCall, J. G. et al. Locus coeruleus to basolateral amygdala noradrenergic projections promote anxiety-like behavior. Ramón y Cajal, S. Algunas conjeturas sobre el mecanismo anatómico de la ideación, asociación y atención. Cahill, M. K. et al. Network-level encoding of local neurotransmitters in cortical astrocytes. Gong, L., Pasqualetti, F., Papouin, T. & Ching, S. Astrocytes as a mechanism for contextually-guided network dynamics and function. Distinct temporal integration of noradrenaline signaling by astrocytic second messengers during vigilance. Sapkota, D. et al. Activity-dependent translation dynamically alters the proteome of the perisynaptic astrocyte process. Choucry, A., Nomoto, M. & Inokuchi, K. Engram mechanisms of memory linking and identity. Zaki, Y. et al. Offline ensemble co-reactivation links memories across days. & Byrne, J. H. The right time to learn: mechanisms and optimization of spaced learning. Nader, K., Schafe, G. E. & Le Doux, J. E. Fear memories require protein synthesis in the amygdala for reconsolidation after retrieval. & Ledoux, J. E. Disruption of reconsolidation but not consolidation of auditory fear conditioning by noradrenergic blockade in the amygdala. Naegeli, C. et al. Locus coeruleus activity mediates hyperresponsiveness in posttraumatic stress disorder. Cornell-Bell, A. H., Finkbeiner, S. M., Cooper, M. S. & Smith, S. J. Glutamate induces calcium waves in cultured astrocytes: long-range glial signaling. Ollivier, M. et al. Crym-positive striatal astrocytes gate perseverative behaviour. Rusakov, D. A., Bard, L., Stewart, M. G. & Henneberger, C. Diversity of astroglial functions alludes to subcellular specialisation. King, C. M. et al. Local resting Ca2+ controls the scale of astroglial Ca2+ signals. Rational engineering of XCaMPs, a multicolor GECI suite for in vivo imaging of complex brain circuit dynamics. Stress disrupts engram ensembles in lateral amygdala to generalize threat memory in mice. Levy, A., Ngai, S. H., Finck, A. D., Kawashima, K. & Spector, S. Disposition of propranolol isomers in mice. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. The Allen Mouse Brain Common Coordinate Framework: a 3D reference atlas. B., Epstein, C. L., Grossman, M. & Gee, J. C. Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain. Schildge, S., Bohrer, C., Beck, K. & Schachtrup, C. Isolation and culture of mouse cortical astrocytes. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Yu, X. et al. Reducing astrocyte calcium signaling in vivo alters striatal microcircuits and causes repetitive behavior. Yamasaki, A. et al. IRF8 and MAFB drive distinct transcriptional machineries in different resident macrophages of the central nervous system. Marsh, S. E. et al. Dissection of artifactual and confounding glial signatures by single-cell sequencing of mouse and human brain. We thank N. Fujimori, R. Ando, N. Miyasaka and N. Mataga for technical assistance; M. Motoyoshi for assisting with imaging; Y. Li for sharing GRAB sensors; and T. J. McHugh, J. P. Johansen, T. Kitamura, K. Tanaka, T. Kitazawa, T. Kanaya, B. Wulaer, E. Shigetomi, S. Okabe, B. Deneen and B. S. Khakh for valuable comments and discussions. STPT, cryosectioning and image acquisition were performed with common use equipment at the Support Unit for Bio-Material Analysis, RRD of RIKEN CBS. We thank the Division of Gene Expression Dynamics, the MEXT Cooperative Research Project Program, the Medical Research Center Initiative for High Depth Omics, and acknowledge support from CURE:JPMXP1323015486 for the Medical Institute of Bioregulation, Kyushu University for sequencing. groups are supported by Japan Agency for Medical Research and Development (AMED) grant JP24gm1910004. This work was supported in part by JSPS KAKENHI grant numbers 24K02128 (to J.N. by Japan Science and Technology Agency (JST) under grant numbers JPMJFR2249 (to J.N. ), by AMED under grant numbers JP21km0908001 (to J.N. ), Multidisciplinary Frontier Brain and Neuroscience Discoveries (Brain/MINDS 2.0) by AMED JP23wm0625001 (to H.S. ), by RIKEN-Osaka University Science and Technology Hub Collaborative Research Program (to J.N. ), and by the Takeda Science Foundation (to J.N. ), Daiichi Sankyo Foundation of Life Science (to T.M. ), Mochida Memorial Foundation for Medical and Pharmaceutical Research (to T.M. ), Astellas Foundation for Research on Metabolic Disorders (to T.M. ), Ono Pharmaceutical Foundation for Oncology (to T.M. and the Uehara Memorial Foundation (to T.M.). were partly supported by JSPS Research Fellowships (24KJ2074 and 22KJ3141, respectively). These authors contributed equally: Ken-ichi Dewa, Kodai Kaseda Ken-ichi Dewa, Kodai Kaseda, Aoi Kuwahara, Hideaki Kubotera, Atsuko Komori, Mika A. Holtz & Jun Nagai Department of Life Science and Medical Bioscience, Graduate School of Advanced Science and Engineering, TWIns, Waseda University, Tokyo, Japan Kodai Kaseda, Aoi Kuwahara & Jun Nagai Division of Allergy and Immunology, Medical Institute of Bioregulation, Kyushu University, Fukuoka, Japan Laboratory of Molecular Neuropharmacology, Graduate School of Pharamceutical Sciences, Osaka University, Suita, Japan Japan Science and Technology Agency (JST), Fusion Oriented REsearch for disruptive Science and Technology (FOREST), Kawaguchi, Japan Atsushi Kasai, Minako Ito & Jun Nagai Division of Brain Sciences, Institute for Advanced Medical Research, Keio University School of Medicine, Tokyo, Japan Department of Optical Neural and Molecular Physiology, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Center for Living Systems Information Science, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Department of Brain Development and Regeneration, Graduate School of Biostudies, Kyoto University, Kyoto, Japan Laboratory of Deconstruction of Stem Cells, Institute for Frontier Life and Medical Sciences, Kyoto University, Kyoto, Japan Department of Molecular and System Pharmacology, Graduate School of Pharmaceutical Sciences, Kyushu University, Fukuoka, Japan Kyushu University Institute for Advanced Study, Fukuoka, Japan Division of Cardiology, Department of Medicine, The Johns Hopkins Medical Institutions, Baltimore, MD, USA Precursory Research for Embryonic Science and Technology (PRESTO), Japan Science and Technology Agency, Kawaguchi, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out vector constructions with help from A. Komori, performed mouse surgeries and behaviour tests, and with M.A.H. A. Kasai provided guidance on the analysis of brain-wide data. performed ex vivo astrocyte calcium imaging and analysis. A. Kuwahara performed primary astrocyte experiments with help from K.D., who established the protocol. conducted in vivo fibre photometry and animal behaviour experiments, including associated surgeries with guidance from N.T. provided guidance on astrocyte sensors. provided key mouse lines and contributed intellectually to related analyses. Tissue preparation for single-cell transcriptomics was performed by K.D., K.K., H.K. and analysis of sequencing data by K.K. performed in vivo infusion of antibodies, for which K.K. performed mouse surgeries and histological analyses. conceptualized and directed the project, guided analyses and wrote the paper with help from K.D., K.K., A.K. All authors commented on and approved the manuscript. The authors declare no competing interests. Nature thanks Christian Henneberger, Gertrudis Perea and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic illustrating an AAV PHP.eB vector intravenous (i.v.) injection into retro orbital sinus of adult mice for expressing red fluorescence reporter mScarlet-I in astrocytes in whole brain. b, Representative images showing that whole-brain expression of mScarlet-I was observed in AAV injected mice, but not in saline injected mice. AAV PHP.eB-mediated mScarlet-I delivery resulted in expression within S100β/GFAP positive astrocytes. c, Bar plots showing that 97.3%–99.5% of astrocytes were positive for mScarlet-I, whose signals were enhanced with anti-RFP antibody, in the indicated 7 brain regions (left) and that those regions had few mScarlet-I-positive neurons (right). Data are mean ± SEM (error bars smaller than symbols in some cases). Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. AAV2/5-GfaABC1D-hM3Dq-mCherry was stereotaxically delivered into the orbitofrontal cortex (ORB) to induce Fos in astrocytes by 1 mg/kg CNO i.p. injection and tag those Fos+ astrocytes with mNG by 50 mg/kg 4-hydroxytamoxifen (4-OHT) i.p. b, hM3Dq activation with CNO administration increased Fos expression in ORB astrocytes. Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. b, Timeline of visual stimulation experiment to determine effective time window of astrocyte Fos tagging. c, Example images of Fos-tagged (mNG-positive) astrocytes in the primary visual cortex (VISp) of TRAP2 mice that underwent the visual stimulation experiment (representative of 9–12 slices from 3-4 mice/group). d, Quantification of mNG astrocyte density in TRAP2 mice VISp and primary somatosensory cortex (SSp): NS: n = 12 slices from 4 mice, −24 h: n = 12 slices from 4 mice, −12 h: n = 12 slices from 3 mice, −6 h: n = 12 slices from 4 mice, −3 h: n = 12 slides from 4 mice, 0 h: n = 12 slices from 4 mice, 3 h: n = 9 slices from 3 mice, 6 h: n = 9 slices from 3 mice, 12 h: n = 9 slices from 3 mice, 24 h: n = 9 slices from 3 mice. Two-way ANOVA, Šídák's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. a, Representative immunostaining of c-Fos expression in a subset of amygdalar astrocytes after fear conditioning (FC) and fear recall (FR). Arrowheads denote c-Fos+ Sox9+ cells (representative of 15 slices from 3 mice/group). b, Quantification of c-Fos+ astrocytes in the lateral and basolateral amygdala (LA/B) after FC and FR. Sample sizes and replicates in Supplementary Table 1. injection into Fos-iCreERT2 (TRAP2) mice for expressing green fluorescence reporter mNeonGreen (mNG) in Fos+ astrocytes. b, Timeline of fear recall experiment to determine effective time window of astrocyte Fos tagging. c, Example images of Fos-tagged (mNG-positive) astrocytes in the lateral and basolateral amygdala (LA/B) of TRAP2 mice that underwent the fear recall experiment (representative of 17–48 slices from 3–8 mice/group). Two-way ANOVA, Šídák's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. Alongside NoFC and FC groups (Fig. 1a), an additional cohort received 4-OHT immediately after CtxA exposure with shocks but no habituation (FC+Novelty). Brains were fixed and processed for STPT, 3D reconstruction, regional segmentation, and single-cell counting as in Fig. b, 3D rendering of the whole brain and three brain regions - the hippocampus (HPC; CA1, CA2, CA3 and dentate gyrus), the amygdala (lateral, basolateral and basomedial amygdalar subregions), primary visual cortex (VISp) - showing that Fos-mNG astrocytes (white dots). D dorsal, V ventral, A anterior, P posterior, M medial, L lateral. c-d, Heatmap of BAE count fold changes (vs NoFC in c, vs FC in d, n = 4 mice/group). *FDR < 0.05; full stats in Supplementary Table 2. Levene's test followed by one-way ANOVA or Kruskal-Wallis; post-hoc Tukey or Dunn with Bonferroni. Primary visual area (VISp), primary somatosensory area barrel field (SSp-bfd), orbital cortex (ORB), prelimbic cortex (PL), infralimbic cortex (ILA), entorhinal cortex (ENT), anterior cingulate cortex (ACA), olfactory cortex (OLF), retrosplenial cortex (RSP), lateral amygdala (LA), basal amygdala (BA), central amygdala (CEA), mediodorsal thalamus (MD), dorsal geniculate thalamus (GENd), anterodorsal thalamus (AD), anteroventral thalamus (AV), ventroposterior medial nucleus of thalamus (VPM), caudoputamen (CP), nucleus accumbens (ACB), dentate gyrus (DG), hippocampal CA1 (CA1), hippocampal CA3 (CA3), para-subiculum (PAR), and pre-subiculum (PRE). Sample sizes and replicates in Supplementary Table 1. a, Schematics illustrating the design of pharmacological screening. b, Representative images from 9 biological replicates showing high purity of primary astrocyte culture. The immunoreactivities of Iba1, a microglia marker, Tuj1, a neuronal marker and Olig2, an oligodendrocyte marker, were not evident, and most of the DAPI signal overlapped with GFAP-positive primary astrocytes. c, Graph showing that c-Fos intensity in GFAP-positive astrocytes was increased with noradrenaline (NA), but not with growth factors that are known to induce c-Fos expression in primary neurons. TTX (1 µM) was used throughout the culture experiments to eliminate the potential effect of neuronal firing. One-way ANOVA, Dunnett's test (vs Veh). Sample sizes and replicates in Supplementary Table 1. Mice underwent fear conditioning two weeks later followed by i.p. Brain samples were intracardially fixed for immunohistochemistry analysis. b, Graphs showing the percentage of RAM-expressed neurons in LA/B (left), co-expression of c-Fos and RAM (center), and overall c-Fos+ neurons (right). ****p < 0.0001. c, Representative images (from mKate2: n = 15 slices, hM4Di: n = 14 slices, 5 mice/group) showing expression of neuronal marker NeuN and RAM-tagged engram neurons (indicated by mCherry signals) in b. Overlapping c-Fos-expressing engram neurons were fewer in the RAM-hM4Di group. Sample sizes and replicates in Supplementary Table 1. a, Images (representative of n = 16 slices from 4 mice) showing FR-BAE (green) in the amygdala overlapping with astrocyte markers GFAP and S100β (cyan) and proximal to NA projections (white) and engram neurons (red). b, Graphs showing that engram neurons (left) and NA projections (center) are statistically more colocalized with FR-BAE than non-FR-BAE astrocytes. Astrocyte territory size is not different between BAE and non-FR-BAE astrocytes (right). Sample sizes and replicates in Supplementary Table 1. Termination sites were mapped onto a standard atlas (adapted from Paxinos, G. & Franklin, K.B.J. The Mouse Brain in Stereotaxic Coordinates, 2nd ed. Circles show fiber tips from individual mice, color-coded by experimental group. b, Freezing before, during, after FC, during FR, and in CtxB (distinct from CtxA). Freezing was lower in CtxB and in CtxA with Prop vs FR (FR: n = 14; CtxB: n = 13; Prop: n = 8 mice). c, dF/F traces: NA signals evoked by CtxA or CtxB (early) persisted longer in CtxA (late). d, Scatter plots: integrated areas of early and late NA signals in CtxA vs CtxB. e, Astrocyte cAMP traces: gradual buildup in HC, strong early responses to context exposure. FR further enhanced late cAMP in CtxA, not CtxB. f, Scatter plots: late cAMP signal areas higher in CtxA vs CtxB, consistent with GRABNE2h. g, Astrocyte Ca2+ traces: evoked by context exposure (early). At 7–15 min, signal frequency remained elevated in CtxA, not CtxB (late). h, Scatter plots: late Ca2+ frequency higher in CtxA vs CtxB. Wilcoxon signed-rank test or paired t test was used based on normality of the data distribution. Data are mean ± SEM (error bars smaller than symbols in some cases). Sample sizes and replicates in Supplementary Table 1. a, Representative gating strategy of FACS-sorting of ACSA2+ astrocytes from the amygdala for scRNA-seq. b, Violin plot showing relative expression levels of cell type marker genes in scRNA-seq. Aldoc, an astrocyte marker, was the only gene showing high expression. c, tSNE map showing astrocyte genes Aldoc, Atp1a2 (coding astrocyte cell surface antigen-2, ACSA2 with which we aimed to FACS-sort astrocytes from the amygdala tissue), and Slc1a3 with which we identified high expression in astrocytes across amygdalar astrocytes in our RNAscope experiments. d, Left: heatmap showing the top 20 genes that were upregulated in each astrocyte cluster relative to the others. Right: Example cluster marker genes identified were plotted onto the tSNE map. See also Supplementary Table 3. e, Left: tSNE plot of astrocyte clusters to illustrate that the 4 clusters are found in every single mouse examined. Right: Bar graph showing the proportion of each astrocyte cluster in the four experimental groups. a, Top 10 differentially expressed gene candidates, ranked by p value comparing Adrb1-positive vs. -negative astrocytes. Heatmap shows decreased (blue) or increased (red) expression. b, Violin plot showing Igfbp2 enrichment in Adrb1-positive astrocytes. c, RNAscope images (representative of 588 cells from 6 mice) of Adrb1 and Igfbp2 in Slc1a3-expressing astrocytes in LA/B from FR. e, Adrb1 expression enriched in Igfbp2-high astrocytes (n = 451 low, 137 high cells). Igfbp2 expression enriched in Adrb1-high astrocytes (n = 426 low, 162 high cells). f, Schematic of in vivo Igfbp2 perturbation. Mice implanted with bilateral cannulae into LA/B for Igfbp2-neutralizing antibody or control IgG infusion immediately after FR. g, Scatter graph: IgG infusion (n = 9 mice) showed no change in freezing between 1st and 2nd FR, whereas Igfbp2 antibody infusion (n = 12 mice) reduced freezing in 2nd FR, indicating destabilized memory. h, Freezing behaviour during 5-day Igfbp2 antibody infusions into LA/B within reconsolidation windows. Two-way ANOVA with Šídák's test. Sites mapped on a standard atlas (adapted from Paxinos, G. & Franklin, K.B.J. The Mouse Brain in Stereotaxic Coordinates, 2nd ed. Each circle indicates cannula termination, coloured by group. Data are mean and SEM (error bars smaller than symbols in some cases). a, Schematics showing the Gαq GPCR signalling cascade and the blocking of which with iβARK. Left: iβARK, a 122-residue inhibitory peptide from β-adrenergic receptor kinase26, directly binds to Gαq subunits and thus sequesters them to block subsequent Gαq-GPCR-induced signalling cascades. Right: iβARK2 with Lck allows it to localize to the plasma membrane, thus being more proximal to and binding Gαq subunits more efficiently. b, Top: AAV2/5 constructs for astrocytic iβARK or iβARKmut. c, Schematic for Ca2+ imaging of iβARK-positive LA/B astrocytes. d, Representative images showing expression of iβARK and iβARK2 in LA/B. e, Representative pseudo-coloured images showing the high fluorescence of GCaMP6f in LA/B astrocytes (representative of n = 28 iβARK2mut and 13 iβARK2 astrocytes). f, dF/F traces of Ca2+ responses in a iβARK(D110A)- or iβARK-expressing astrocyte evoked by PE (phenylephrine), an agonist of Gαq-coupled α1A-adrenoceptor (coded by Adra1a) analyzed for soma and processes separately. g, Scatter graph showing the average Ca2+ responses to 0, 1, and 3 µM PE, where iβARK significantly reduced the PE-induced Ca2+ responses. Two-way ANOVA, Tukey's multiple comparisons test. Sample sizes and replicates in Supplementary Table 1. a, Graphs showing no alteration in locomotion of Adrb1-OE mice during the open field test. Two-way ANOVA and Šídák's multiple comparisons test were used. b, Graphs showing no alteration in time spent in center zone of Adrb1-OE mice during the open field test. c-k, Minute-by-minute freezing behaviour of Ctrl and Adrb1-OE mice across conditioning and recall sessions. After three-shock FC (c), freezing was measured during the 1st (d) and 2nd FR (e). After one-shock FC (f), freezing was measured during the 1st (g) and 2nd FR (h), with Adrb1-OE mice showing significantly more freezing. In a separate three-shock FC cohort (i), freezing was measured during the 1st FR (j) and CtxB exposure (k), where Adrb1-OE mice also showed increased freezing. Two-way ANOVA with Šídák's test. Sample sizes and replicates in Supplementary Table 1. Details of raw replicates, statistical analyses, n numbers and relevant parameters, related to all figures. Fos-mNG astrocyte counts in 677 brain regions in NoFC, FC, FC + novelty, NoFR and FR groups. Single-cell transcriptomic data including the list of top 20 genes that differentiate astrocyte clusters and the differentially expressed genes estimated between Adrb1-positive and negative astrocytes. STPT images, astrocyte marking, and brain region segmentation in the NoFC group. STPT images, astrocyte marking, and brain region segmentation in the FR group. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The astrocytic ensemble acts as a multiday trace to stabilize memory. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/mens-brains-shrink-faster-than-womens-what-that-means-for-alzheimers/'>Men's Brains Shrink Faster than Women's. What That Means for Alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Women's brains age more slowly than men's, but they still have higher rates of Alzheimer's disease During ageing, men experience a greater reduction in volume across more regions of the brain than women do, according to a longitudinal study published today in the Proceedings of the National Academy of Sciences. The authors suggest this means that age-related brain changes do not explain why women are more frequently diagnosed with Alzheimer's disease than men are. “It's really important that we understand what happens in the healthy brain so that we can better understand what happens when people get these neurodegenerative conditions,” says Fiona Kumfor, a clinical neuropsychologist at the University of Sydney, Australia. Nearly twice as many women are diagnosed with Alzheimer's disease as men, and ageing is the biggest risk factor for the disease. This has prompted research into age-related sex differences in the brain. “If women's brains declined more, that could have helped explain their higher Alzheimer's prevalence,” says co-author Anne Ravndal, a PhD student at the University of Oslo. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Previous research investigating sex differences in brain ageing has shown mixed results, Ravndal adds. Several studies have found that men experience greater loss of total grey matter and hippocampus size compared with women, whereas other work has reported a sharper decline of grey matter in women. The latest study included more than 12,500 magnetic resonance imaging (MRI) brain scans from 4,726 people — at least two scans per person, taken an average of three years apart — who did not have Alzheimer's disease or any cognitive impairments and were control participants in 14 larger data sets. Overall, men experienced a greater reduction in volume across more regions of the brain than women did. For example, the postcentral cortex, which is responsible for processing sensations of touch, pain and temperature, as well as the body's own position and movements, declined by 2.0% per year in men and by 1.2% annually in women. Kumfor says the results suggest that men age faster than women do. That sex differences in brain volume don't seem to play a part in the condition is not entirely surprising, says Kumfor, because neurodegenerative conditions are complex. Understanding the disorders will require longitudinal studies involving people with Alzheimer's disease, to compare how their brains change over time. “Just looking at age-related changes in brain atrophy is unlikely to explain the complexities behind it,” she adds. Although the study is robust, further investigations with more diverse data sets are needed, says Brodtmann. For example, people included in the study had high levels of education — a protective factor against Alzheimer's disease — meaning that they were not fully representative of the general population. The data sets also lacked information about other factors that influence a person's risk of Alzheimer's disease, such as ethnicity and the age at which women experience menopause. The authors say that when they adjusted their analysis for education level, some parts of men's brains no longer showed a steeper decline than occurred in women. This article is reproduced with permission and was first published on October 13, 2025. First published in 1869, Nature is the world's leading multidisciplinary science journal. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251015032316.htm'>This European treatment for joint pain just passed a major scientific test</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 14:47:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Participants with mild to moderate knee osteoarthritis experienced notable pain reduction and improved mobility within four months of receiving the treatment. The radiation exposure was only a tiny fraction of what is used in cancer therapy. Because the study included a control group that underwent a simulated treatment, researchers were able to clearly separate genuine treatment effects from placebo responses, which are common in osteoarthritis trials. Early results from this Korean study were presented at the American Society for Radiation Oncology (ASTRO) Annual Meeting. "People with painful knee osteoarthritis often face a difficult choice between the risks of side effects from pain medications and the risks of joint replacement surgery," said Byoung Hyuck Kim, MD, PhD, principal investigator on the trial and an assistant professor of radiation oncology at Seoul National University College of Medicine, Boramae Medical Center. It develops as the cartilage that cushions bone ends wears away over time. Low-dose radiation therapy has long been used in European countries such as Germany and Spain to treat joint pain, where it is widely accepted. However, Dr. Kim noted that before this study, rigorous placebo-controlled evidence was scarce, and awareness of the treatment remained low among clinicians in other parts of the world. In this study, he explained, the radiation dose was less than 5% of what is typically used in cancer therapy, and no radiation-related side effects were observed. The multicenter study included 114 participants with mild to moderate knee osteoarthritis, recruited from three academic hospitals in Korea. Each participant underwent six treatment sessions without knowing which group they were assigned to. The effectiveness of treatment was evaluated using internationally recognized standards, identifying a participant as a "responder" if they achieved meaningful improvement in at least two of three measures: pain, physical function, and overall assessment of their condition. Participants also completed separate questionnaires addressing pain, stiffness, and mobility. For other secondary outcomes, including the amount of pain medication needed, there were no significant differences. Dr. Kim said this trial differed from previous research in two critical ways. "The sham-controlled design helped rule out placebo effects, and we limited stronger analgesics, which made differences between groups more clearly attributable to the radiation itself," he said. But using these pain relievers could mask the effects of radiation therapy," he said. Responses in the placebo arms were substantial -- about 40% met the criteria for treatment response without true radiation -- but are consistent with rates reported in prior osteoarthritis trials of injections or medications and in at least one similar European study, Dr. Kim said. We need to examine this more closely in future studies." Radiation therapy may be best suited for patients with underlying inflammation and preserved joint structure, he explained. He emphasized that low-dose radiation should be considered as part of shared decision-making alongside standard measures such as weight loss, physiotherapy, and medications. "In clinical practice, responses could be even stronger when radiation is properly combined with other treatments, and patient satisfaction may be higher than with current options alone." Planned studies also include larger, pragmatic trials to evaluate outcomes in specific subgroups and health-economic analyses comparing low-dose radiation with injections and medication regimens. Scientists Extend Lifespan by over 70% in Elderly Male Mice with New Treatment The “Really Big One” Might Trigger California's Next Catastrophe, Scientists Warn Promising New Treatment Could Calm Anxiety for Millions of Americans Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69001262/fault-lines-synched-together/'>Two of the Biggest Fault Lines in the World Are Synched Together. That Could Be Disastrous.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers hoping to unravel the dance of earthquake movement in North America's West Coast discovered a disturbing fact: Two of the world's largest fault lines sometimes work in conjunction. In a new study published in the journal Geosphere, researchers from Oregon State University—led by Chris Goldfinger, a marine geologist and geophysicist—show evidence of what's called “partial synchronization” of the northern San Andreas Fault and Cascadia Subduction Zone. That likely implies those unique turbidite formations were stacked by two earthquakes, one from each zone, in quick succession—a matter of years or even minutes apart. “When you tune an old radio, you're essentially causing one oscillator to vibrate at the same frequency as the other one,” he said. “When these faults synchronize, one fault could tune up the other and cause earthquakes in pairs.” “If they both went off together, then you've got potentially San Francisco, Portland, Seattle, and Vancouver [B.C.] “But it's pretty clear that if something like this happened—and we think the evidence for it is strong—we need to be prepared.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Go Deeper Into Mantle Than Ever Before Experts Found an Ancient  Tomb Full of Treasure An Elusive Rat Finally Showed Its Face to Science Scientists Just Solved a Major Mystery About Hail A Wild Wolf-Dog Hybrid Has Emerged in Greece</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69022085/stevia-hair-loss/'>Scientists Found a Chemical That Could Help Regrow Your Hair. It Might Be In Your Pantry.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-15 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>One of the most widespread medications for baldness, which is the active ingredient in products like Rogaine, is minoxidil (originally developed to treat high blood pressure). While this treatment has helped many people grow hair, it does come with a few downsides. Standard treatments typically have poor water solubility and skin permeability, so some hair growth experts resort to applying minoxidil with microneedling, a technique that punctures small holes in the top layer of the skin to stimulate growth factors and increase the medication's absorption. As described in a new study, scientists from China and Australia have developed a dissolving patch that can perform this microneedling work while also delivering a dose of minoxidil to the scalp, and it's derived from a somewhat unlikely ingredient: Stevioside (STV), the natural sweetener derived from the Stevia plant. “This method not only enhances drug penetration into skin but also eliminates other issues associated with traditional topical formulations, such as slow onset of action and/or inaccurate dosing.” Minoxidil, however, suppresses certain enzymes and inhibits androgen receptors, which has the net effect of decreasing the formation and binding of dihydrotestosterone. It also boosts aromatase, which converts testosterone into estradiol—a kind of estrogen that further stimulates hair growth. In 35 days, scientists saw hair regrowth in 67 percent of the target area, compared to only 25 percent from the control group—a promising start for this Stevia-based hair loss treatment. “While the mouse model results are promising, human hair growth cycles differ, and androgenetic alopecia is influenced by multiple factors,” Lifeng Kang, a co-author of the study from the University of Sydney, told Gizmodo. “Therefore, clinical trials are essential to confirm efficacy and safety in humans.” Unwanted hair loss isn't a thing of the past (at least, not yet), but science has it in its crosshairs—no pun intended. A Scientist Says We Can Slow Time With Our Minds Scientists Found a Surprising Way To Target Deadly Tumors</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            