
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-01</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250501122043.htm'>Ptero firma: Footprints pinpoint when ancient flying reptiles conquered the ground</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 17:44:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Fossils of footprints over 160 million years old have helped palaeontologists at the University of Leicester to narrow down when pterosaurs adapted to live on the ground. But new research shows that some of these ancient creatures were just as comfortable walking on the ground. In a groundbreaking new study published today in Current Biology (1 May), scientists at the University of Leicester have successfully linked fossilised footprints to the types of pterosaurs that produced them. By using 3D modelling, detailed analysis, and comparisons with pterosaur skeletons, the team has shown that at least three different types of tracks match up with distinct groups of pterosaurs. They reveal not only where these creatures lived and how they moved, but also offer clues about their behaviour and daily activities in ecosystems that have long since vanished." The study uncovered three distinct types of pterosaur footprints, each shedding light on different lifestyles and behaviours. Co-author Dr David Unwin from the School of Museum Studies, University of Leicester explained: "Finally, 88 years after first discovering pterosaur tracks, we now know exactly who made them and how." Perhaps the most striking discovery comes from a group of pterosaurs called neoazhdarchians which includes Quetzalcoatlus, with a 10 m wingspan one of the largest flying animals ever to have existed. Some of these tracks are present right up until the asteroid impact event, 66 million years ago, which led to the extinction of both pterosaurs and dinosaurs. One group of pterosaurs, ctenochasmatoids, known for their long jaws and needle-like teeth, left behind tracks most commonly found in coastal deposits. These animals likely waded along muddy shores or in shallow lagoons, using their specialised feeding strategies to catch small fish or floating prey. Known as dsungaripterids, these pterosaurs had powerful limbs and jaws, with toothless, curved beak tips designed for prising out prey, while large, rounded teeth at the back of their jaws were perfect for crushing shellfish and other tough food items. Smyth explains: "Tracks are often overlooked when studying pterosaurs, but they provide a wealth of information about how these creatures moved, behaved, and interacted with their environments. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/trump-quietly-halts-money-to-states-for-preventing-disaster-damage/'>Trump Quietly Halts Money to States for Preventing Disaster Damage</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 14:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Trump Quietly Halts Money to States for Preventing Disaster Damage President Trump is the first president in at least three decades to deny governors' requests for funding that's meant to protect people and property President Donald Trump is considering a major overhaul to the Federal Emergency Management Agency. CLIMATEWIRE | The Trump administration is scaling back a multibillion-dollar program that has been the backbone of state efforts to protect homes, hospitals and other structures from floods, hurricanes and earthquakes. President Donald Trump stopped approving new allocations in early April from a federal program that has been a top funding source for protecting people and property from disasters since 1989. The Hazard Mitigation and Grant Program has been used to elevate or demolish flood-prone homes, install tornado-safe rooms and strengthen buildings in hurricane or earthquake zones. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. On roughly one-third of the properties, flood-prone buildings were demolished and the land was left vacant as part of local efforts to reduce flood damage. More than $11 billion has been used to fortify medical facilities, power plants, roads and bridges so they will continue to operate during disasters, the analysis shows. On Monday, Trump named 13 members to a council he charged with reviewing the agency and recommending overhauls. Trump's spending halt from the hazard-mitigation program comes weeks after he canceled another multibillion-dollar FEMA grant program, raising concerns that states and localities will stop efforts to minimize damage from future disasters. FEMA's mitigation programs have been widely praised for reducing long-term disaster costs. “HMGP is more cost-effective in the long-term because it helps communities rebuild from disasters stronger than before,” Virginia Democratic Sens. North Carolina, for example, still has $1.7 billion that was approved by then-President Joe Biden after Hurricane Helene caused widespread damage last year. Neither FEMA nor the White House responded to requests for comment. But an internal FEMA document and Trump's handling of two recent disasters show the administration's new approach. But while Trump agreed to help 16 counties rebuild, he took the highly unusual step of refusing a request by Virginia Gov. Glenn Youngkin, a Republican, for hazard-mitigation money as part of the disaster-aid package. It was the first time in at least 27 years that a president had denied a state's request for hazard-mitigation money while approving its disaster declaration request, according to FEMA records that go back 27 years. Trump himself approved mitigation money when he declared disasters in Kentucky, West Virginia and Oklahoma in February and March — the first three declarations of his second term. “We are deeply concerned communities will have fewer resources available to protect against the impacts of extreme weather events,” Kaine and Warner said in their statement, noting Trump's recent cancellation of another FEMA mitigation program. In an April 12 memo to the White House, FEMA acting Administrator Cameron Hamilton suggested that Trump “not automatically approve the Hazard Mitigation Program” when declaring a disaster. “This is an action the President has already taken,” Hamilton wrote to the Office of Management and Budget in an apparent reference to Trump's denial of mitigation funds for Virginia. “This will serve as a precursor to further evaluating and redesigning mitigation grant programs.” The six-page memo, first reported by CNN, suggests making other cuts to FEMA programs, such as denying aid for smaller disasters and for snowstorms. Trump indicated his intention to follow Hamilton's recommendation on the mitigation program when he approved a second disaster declaration for Kentucky on April 25 without including mitigation money. “When I saw Kentucky, I saw that now there's a pattern, and this is consistent with the recommendation Hamilton made to OMB,” former FEMA chief of staff Michael Coen said, referring to the White House Office of Management and Budget. Andy Beshear, a Democrat, applauded FEMA for approving disaster aid for households hit by recent flooding and did not reply to E&E News questions about the absence of mitigation money. The program had come under scrutiny for its complexity, which resulted in states not spending billions of dollars they were allocated. Trump has said his overhaul of FEMA aims to give states more financial and logistical responsibility for disaster recovery — a position embraced by some experts and former FEMA leaders. But states are not likely to replace FEMA hazard-mitigation money, said Kelly McKinney, a former deputy commissioner of New York City Emergency Management. Many state programs are strongly backed by advocates who “weigh-in in a very big, noisy way” to protect spending, McKinney said. In early April, Trump canceled another FEMA mitigation program, named Building Resilient Infrastructure and Communities, and froze $3.6 billion of unspent funds that had been approved for states.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/why-evolving-dark-energy-worries-some-physicists/'>Latest Dark Energy Study Suggests the Universe Is Even Weirder Than We Imagined</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 14:15:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Latest Dark Energy Study Suggests the Universe Is Even Weirder Than We Imagined If dark energy is weakening, as suggested by recent results, then the cosmos is far stranger than most physicists had supposed New results suggest dark energy's influence on cosmic expansion has weakened over time. Against most experts' expectations, the result suggested that dark energy—the mysterious force driving the universe's accelerating expansion—was not an unwavering constant but rather a more fickle beast that was weakening over time. The shocking claim's source was the Dark Energy Spectroscopic Instrument (DESI), run by an international collaboration at Kitt Peak National Observatory in Arizona. And it was so surprising because cosmologists' best explanations for the universe's observed large-scale structure have long assumed that dark energy is a simple, steady thing. But as Joshua Frieman, a physicist at the University of Chicago, says: “We tend to stick with the simplest theory that works—until it doesn't.” Heady with delight and confusion, theorists began scrambling to explain DESI's findings and resurfaced old, more complex ideas shelved decades ago. Dark energy's implied fading, it seemed, was refusing to fade away. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “I'm particularly skeptical of DESI's press release,” Green says. DESI didn't do that many analyses.” The situation, Green says, is akin to looking for a lost set of car keys in a dark parking lot—but only where the light is bright: “When all you look under is one lamppost, you only see what you find there.” Other explanations exist for DESI's measurements, Green says, and not all of them require the cosmos-quaking prospect of an evolving dark energy. His preferred model instead invokes the putative decay of another mysterious aspect of cosmology, dark matter—thought to be a substance that gravitationally binds galaxies together but otherwise scarcely interacts with the rest of the universe at all. CDM is the type of dark matter that best accounts for observations of how galaxies form and grow, and—until DESI's proclamation suggested otherwise, that is—a constant dark energy has been the best fit for explaining the distributions of galaxies and other patterns glimpsed in large-scale cosmic structures. “Once they had this constant, everything snapped into place,” Green says. “All of the issues that had been around for 20 years that we'd been hoping were just small mistakes were really resolved by this one thing.” But dark energy's constancy has always been more of a clever inference rather than an ironclad certainty. DESI is an effort to clarify exactly what dark energy really is by closely monitoring how it has influenced the universe's growth. Since 2021 the project has been meticulously measuring the motions and distributions of galaxies across some 11 billion years of cosmic time. And its tracing of spatial distributions emerges from spying enormous bubblelike arrangements of galaxies thought to have formed from more primordial templates, called baryon acoustic oscillations (BAOs). Up until then, the researchers had worked on blinded data, meaning the true values were slightly but systematically altered so as to ensure that no one could deliberately or inadvertently bias the ongoing analysis to reach some artificially preordained result. These blinded data showed a huge divergence from LCDM. But when the real data were unveiled, “we saw all the points came very close to LCDM, and that was initially a huge relief,” she recalls. Those feelings quickly changed when the group noticed a small, persistent deviation in DESI's estimate for the value of lambda. But in DESI's latest results, which were posted to the preprint server arXiv.org last March and incorporated much larger and richer data sets, the statistical robustness of the unexpected lambda value soared, and most talk of flukes dwindled. The results rekindled preexisting ideas about dynamic dark energy first formulated decades ago, not long after dark energy's discovery itself in 1998. One popular theory posits a fifth fundamental force in addition to the known four (electromagnetism, gravity, and the strong and weak nuclear forces), emerging from some as-yet-undiscovered dark matter particle that can influence dark energy. Frieman says the data from DESI is so precise that if this particle is the correct explanation, physicists already know its crucial parameters. Constrained by the DESI data, Frieman says, the best-fitting model that would support this “fifth force” hypothesis “tells us that this [hypothetical] particle has a mass of about 10–33 electron volts.” To put that into perspective, this means such a particle would be 38 orders of magnitude lighter than an electron—which, Frieman notes, is “by far the lightest stable particle we know of that doesn't have zero mass.” But while some theorists used DESI's data to revive and sharpen intriguing theories of yesteryear, Green and others issued a warning. The problem: an evolving dark energy would seem to defy well-founded physical principles in other cosmic domains. The first major point of controversy involves something called the null energy condition, under which—among other things—energy can't propagate faster than light. Theorists have mathematically proven the condition's apparent necessity in numerous circumscribed scenarios within quantum and relativistic domains—but not for the universe at large. A discovery that something in the universe violates the null energy condition would be groundbreaking, to say the least: a more impolitic term would be “nonsensical.” This astounding violation is exactly what Green and others say most of DESI's analyses are showing, however. In 1917 Albert Einstein first introduced lambda as a way to ensure that a static universe would pop out of his equations. But after work led by Edwin Hubble proved the universe was expanding, Einstein abandoned his fudge factor (even calling it his “greatest blunder”). It wasn't until the late 1990s, when astronomers found that the universe's expansion wasn't constant but in fact accelerating, that lambda once again returned to theoretical prominence. This time theorists interpreted it to represent the magnitude of the universe's dark energy density, a constant that doesn't change with time. But if there's one thing modern cosmology has shown, it's that little, if anything, about the universe is ever so neat and tidy. So, despite a lack of evidence, theorists of the time reimagined LCDM as w(z)CDM, where w(z) is a time-varying term representing the ratio of dark energy's pressure to its energy density. For w(z) greater than –1, the universe's dark energy dilutes over time, consistent with DESI's findings. On the other hand, w(z) less than –1 leads to devastating consequences: dark energy's pressure overpowers its density, ultimately causing everything from galaxies all the way down to atoms to be ripped apart—a “big rip” that violates the null energy condition and would seemingly doom the universe to a violent death. Such criticisms “draw the wrong conclusions,” says Paul Steinhardt, a cosmologist at Princeton University. Researchers merely find it convenient for comparing different dark energy models and experiments. It rolls down to, but never crosses, the critical –1 value, consistent with a weakening dark energy that would avoid the universe ending in a big rip. But Gabriel Lynch, a Ph.D. student at the University of California, Davis, who has an alternative explanation for the DESI data, says that even if any of DESI's w(z) estimates are physical, coaxing out a theory to support them leads to incredibly fraught circumstances. Whether or not DESI's results would violate the null energy condition, everyone agrees on another problem. Frieman suggests that something else, perhaps an unknown particle, might be mimicking a negative-mass neutrino. But a new approach by Lynch and his thesis advisor Lloyd Knox, detailed in a preprint that was posted to arXiv.org in March, sidesteps this “negative neutrino” problem altogether. Fast forward about 14 billion years to today, and some would have decayed into dark radiation, erasing the heavy matter signal. If the idea holds true, DESI's data might be a way to find the exact value for neutrino masses as well as for dark matter particles, which would be a big deal. Owing to dynamic dark energy's paradoxes, “you really need to explore every alternative explanation [for the results], because evolving dark energy is the absolute last one that I would be willing to believe,” Green says. Despite such strong words, all parties caution that this debate is still in its early days. “This is only the first round of the fight,” Steinhardt says, and no model currently explains all of DESI's results. More data are needed, especially from even bigger and better cosmic surveys by planned next-generation telescopes. And, naturally, more analyses are needed, too, before the community can reach any consensus. Whether a resolution comes from dynamic dark energy, dark matter decay or something entirely different, the LCDM model has seemingly been stretched to its breaking point. Every reasonable explanation for DESI's data involves new, scarcely explored physics.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-58128-3'>Sex-differentiated placental methylation and gene expression regulation has implications for neonatal traits and adult diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 09:35:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 4004 (2025) Cite this article Sex differences in physiological and disease traits are pervasive and begin during early development, but the genetic architecture of these differences is largely unknown. Here, we leverage the human placenta, a transient organ during pregnancy critical to fetal development, to investigate the impact of sex in the regulatory landscape of placental autosomal methylome and transcriptome, and its relevance to health and disease. We find that placental methylation and its genetic regulation are extensively impacted by fetal sex, whereas sex differences in placental gene expression and its genetic regulation are limited. We identify molecular processes and regulatory targets that are enriched in a sex-specific manner, and find enrichment of imprinted genes in sex-differentiated placental methylation, including female-biased methylation within the well-known KCNQ1OT1/CDKN1C imprinting cluster of genes expressed in a parent-of-origin dependent manner. We establish that several sex-differentiated genetic effects on placental methylation and gene expression colocalize with birthweight and adult disease genetic associations, facilitating mechanistic insights on early life origins of health and disease outcomes shaped by sex. Sex differences in physiological traits and disease risk are pervasive, but the underlying mechanisms remain elusive. Phenotypic and developmental differences between males and females begin before birth, and differences manifest across the life span in disease prevalence, severity, and treatment response1. It has long been known that male fetus-bearing pregnancies have faster fetal growth rate2, higher vulnerability to complications such as pre-eclampsia, placental abruption, fetal growth restriction, premature delivery, and post-partum hemorrhage, and greater risk of perinatal mortality2,3,4. Recent studies have shown that autosomal genetic factors contribute to sex-differentiated phenotypes in adults, with some of these having prenatal molecular foundations1,5. Transcriptional regulation by methylation and gene expression can be a key underlying mechanism of these differences. Thus, investigating how fetal sex shapes the regulation of methylation and gene expression in tissues from early human development can unlock the molecular origins of health outcomes across the lifespan. Successful fetal development relies on the placenta, a temporary organ during pregnancy at the maternal-fetal interface that exhibits sex differences in function, morphology, and response to environmental exposures6,7,8,9,10,11. Placental dysfunction can lead to pregnancy complications and early programming of diseases that manifest in later life in a sex-dependent fashion12,13. Accumulating evidence suggests that fetal sex-specific placental transcript expression14,15,16,17 and methylation18,19,20 may be among key molecular antecedents to male and female differences in placental function and neonatal outcomes21,22. Studies that integrated placental multi-omics have offered insights into the molecular bases of human complex traits23,24. Yet, to what extent sex influences the genetic regulation of both placental methylation and gene expression is unknown. Whether sex-differentiated placental methylation is linked with gene expression regulation in their nearby genomic region is unclear. Moreover, studies on sex differences in placental methylation and gene expression are rare in ancestrally diverse human populations. In this study, we investigated the landscape of human placental methylation and gene expression impacted by fetal sex and its relevance to health and disease by integrating genome-wide placental methylation, gene expression, and genotype datasets from the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD) Fetal Growth Studies–Singleton cohort. We report (i) sex differences in placental methylation and its genetic regulation, and in placental gene expression and its genetic regulation; (ii) the biological significance of these sex-differentiated signals through functional annotations and omics integration; and (iii) the impact of these discoveries in advancing the molecular underpinnings of neonatal traits and adult diseases, revealing placental embedding of sex differences in human physiology and pathology. The discovery analyses employed data from participants of the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD) Fetal Growth Studies–Singleton cohort25. There was no significant sex difference in maternal race/ethnicity, age, pre-pregnancy body mass index, and gestational age at delivery (Supplementary Data 1). Figure 1 provides a flow chart of the study. Briefly, a total of 301 samples (152 males and 149 females) were used to identify autosomal differentially methylated cytosine-phosphate-guanine sites (CpGs) between male and female placentas (sex-DM); a sub-sample of 291 (147 males and 144 females) was used to identify sex-biased methylation quantitative trait loci, which are genetic variants associated with different methylation levels in male and female placenta (sex-mQTL); 80 samples (43 males and 37 females) were used to identify differentially expressed autosomal genes between male and female placentas (sex-DE); and 71 samples (38 males and 33 females) were used to identify sex-biased expression quantitative trait loci, which are genetic variants associated with different gene expression levels in male and female placenta (sex-eQTL). Convergence between sex-biased methylation and gene expression was evaluated, and biological insights were gained through functional annotations. Phenotypic relevance was examined by testing the correlation of sex-DM and sex-DE with neonatal and placental anthropometry traits and through colocalization analysis of sex-mQTL and sex-eQTL with published genome-wide association studies (GWAS) loci for complex diseases and traits. The novelty of sex-DM and sex-DE was determined by comparing with previous studies. Loci found to be sex-DE, sex-mQTL, or sex-eQTL were further assessed in an independent dataset from the Rhode Island Child Health Study (RICHS; 74 males and 74 females; more than 70% self-identified White; no sex difference in maternal race/ethnicity, age, pre-pregnancy body mass index, and gestational age at delivery; Supplementary Data 1)26. The workflow summarizes the procedures in identifying sex-biased methylation, gene expression, and genetic regulation in the placenta; functional characterization through assessment of co-occurrence among identified loci and functional annotation using various databases; and illustration of the relevance of the identified loci in disease and health traits in neonates and adults. sex-DM: sex-differentially methylated cytosine-phosphate-guanine sites (CpGs); sex-DE: sex-differentially expressed genes; sex-mQTL: sex-biased methylation quantitative trait locus; sex-eQTL: sex-biased expression quantitative trait locus; GWAS: a genome-wide association study. Genotype (DNA), and RNA-seq (RNA) figures were generated by NIAID Visual & Medical Arts. (10/7/2024) and methylation was adopted. NIAID NIH BIOART Source. NIAID Visual & Medical Arts. NIAID NIH BIOART Source. We identified 6077 sex-DM at a false discovery rate (FDR)-adjusted P < 0.05. Out of the 6077 sex-DM, 41.1% (2497/6077) were previously unreported, and the remaining 58.9% have previously been reported with consistent effect direction18,19,20,27,28,29 (Fig. Sex-DM CpGs were two times more likely to be male-hypermethylated than female-hypermethylated (66.9% vs. 33.1%), consistent with previous observations18,19,28. Each panel includes a Manhattan plot of -log10 P-values and the corresponding quantile-quantile plot of observed vs expected -log10 P-values and genomic control factor (λ). a sex-differences in methylation at cytosine-phosphate-guanine sites (sex-DM). b sex-biased genetic association with methylation (cis methylation quantitative trait loci (sex-mQTL). c sex-differences in gene expression (sex-DE). d sex-biased genetic association with gene expression (cis expression quantitative trait loci (sex-eQTL). Two-sided t tests were performed for (a) using limma with n = 301, df = 299, and Benjamini-Hochberg adjustment and BACON correction applied for multiple testing corrections. Two-sided t tests were performed for (b) using MatrixEQTL with n = 291, df = 289, and Benjamini-Hochberg adjustment applied for multiple testing corrections. Two-sided likelihood ratio tests were performed for (c) using edgeR with n = 80, df = 1, and Benjamini-Hochberg adjustment applied for multiple testing corrections. Two-sided t tests were performed for (d) using MatrixEQTL with n = 71, df = 69, and Benjamini-Hochberg adjustment applied for multiple testing corrections. Red dots indicate statistically significant results. We assessed whether the sex-differentially methylated sites are correlated with nearby gene expression more than expected by chance. Methylation at sex-DM CpGs was more likely to be significantly correlated with nearby gene expression in the placenta at FDR-adjusted P < 0.05 than non-sex-DM CpGs on the methylation array (63/3636 vs. 1288/277343; χ2 = 120.6, P = 4.67 × 10−28). An additional 13 CpG-gene correlations were identified by testing the correlation of each sex-DM CpG against all genes within a 200 kb distance. These findings suggest that sex-dependent methylation is not randomly distributed across the genome but is clustered at CpG loci more likely to have impacts on gene transcript levels (Supplementary Data 3). Differentially methylated sites may be involved in transcript regulation, but their precise effect is likely to depend on the genomic context30. In our data, positive sex-DM CpG-gene correlations (i.e., higher methylation correlated with higher gene expression) were more common than negative correlations. Moreover, the majority of CpGs positively correlated with gene expression were in the gene body, whereas the majority of those with negative correlation were in the gene transcription start site (TSS) (Supplementary Data 3). These findings are consistent with long-known theory and recent evidence that methylation within the gene body is often positively correlated with gene expression, while methylation in TSS is often negatively correlated with gene expression30,31,32,33,34,35,36. The top four strongest male-hypermethylated CpG-gene and top four strongest female-hypermethylated CpG-gene correlations are presented in Figs. Among male-hypermethylated sex-DM CpGs correlated with gene expression, methylation of cg11291313, located in ZNF300 TSS, and ZNF300 expression showed the strongest inverse correlation (Spearman r (rs) = − 0.56, P = 3.51 × 10−7); and cg02563011, located within CSMD1 gene body (167.4 kb from gene start), and CSMD1 showed the strongest positive correlation (rs = 0.73, P = 3.44 × 10−13). Previously, male bias has been found in a 225 kb locus of differentially methylated regions (DMRs) within the gene body of CSMD1 and in transcript abundance of CSMD1 in placenta37, although the relationship of the DMRs with CSMD1 transcript abundance was not elucidated. The two CpGs that we found to be correlated with ZNF300 and CSMD1 overlap with predicted active TSS-promoter and enhancer peaks, respectively, based on ENCODE and RoadMap Epigenomic annotations. Among female-hypermethylated sex-DM CpGs correlated with gene expression, cg01070760, located in PSMA8 exon (193 bp from gene start), and PSMA8 showed the strongest inverse correlation (rs = − 0.69, P = 3.07 × 10−11); and cg25948255, located within the gene body of CADM2 (504 kb from gene start), and CADM2 showed the strongest positive correlation (rs = 0.65, P = 1.17 × 10−9). The CpG linked to PSMA8 overlaps with a predicted enhancer and with a long-range chromatin interaction loop (based on high throughput chromosome conformation capture, Hi-C) in placental extravillous trophoblast (EVT) cells38. The CpG linked to CADM2 overlaps with a predicted enhancer and active chromatin in EVT cells38 and overlaps with a predicted polycomb-associated heterochromatin by ENCODE and RoadMap Epigenomic Annotations. ZNF300, CSMD1, and CADM2 have been implicated in cancer cell proliferation, migration, and invasion39,40,41, features mirrored by the invasive, immunologic, and angiogenic properties of placental cells42. ZNF300 methylation has been correlated with placental morphology43, and invasive placental trophoblast cells may drive sex differences at ZNF300 methylation27. Top associations were selected based on the strength of positive and negative correlations, two from each. The displayed figures include male-hypermethylated CpGs that showed the top two strongest negative correlations (a, b) and the top two strongest positive correlations (c, d) with their corresponding nearby genes and UCSC Genome Browser plots of regulatory sites. For each CpG, the left plot shows a correlation with gene expression in the placenta (two-sided Spearman correlation (ρ) with FDR-adjusted P < 0.05 based on S statistic was considered significant; n = 71); the middle plot shows UCSC Genome Browser's regulatory sites from ENCODE; the right plot shows UCSC Genome Browser's regulatory sites in placenta smooth chorion. Red highlights indicate the gene TSS; blue highlights indicate the CpG; and purple highlights indicate that the gene TSS and CpG are in close proximity to each other. Tracks include histone 3 lysine 4 monomethylation (H3K4me1: an enhancer mark), histone 3 lysine 4 trimethylation (H3K4me3: marks transcription start site), histone 3 lysine 27 trimethylation (H3K27me3: marks transcription repression), histone 3 lysine 36 trimethylation (H3K36me3: marks active transcription), and histone 3 lysine 27 acetylation (H3K27ac: marks active transcription) found in 7 cell lines from ENCODE and/or in placenta smooth chorion. Tracks labeled “DNase1” represent DNaseI hypersensitivity clusters in 125 cell types from ENCODE. Tracks labeled “ChIP-seq” represent ChIP-seq clusters in 338 factors and 130 cell types from ENCODE and ChIP-seq marks found in placenta smooth chorion. Tracks labeled “Promoter/enhancer” and “Interaction” represent GeneHancer Double Elite Regulatory Elements and GeneHancer Double Elite Clustered Interactions, respectively. Gray shade denotes a 95% confidence interval. Source data for Fig. 3a–d are provided as a Source Data file. Top associations were selected based on the strength of positive and negative correlations, two from each. The displayed figures include female-hypermethylated CpGs that showed the top two strongest negative correlations (a, b) and the top two strongest positive correlations (c, d) with their corresponding nearby genes and UCSC Genome Browser plots of regulatory sites. For each CpG, the left plot shows a correlation with gene expression in the placenta (two-sided Spearman correlation (ρ) with FDR-adjusted P < 0.05 based on S statistic was considered significant; n = 71); the middle plot shows UCSC Genome Browser's regulatory sites from ENCODE; the right plot shows UCSC Genome Browser's regulatory sites in placenta smooth chorion. Red highlights indicate the gene TSS; blue highlights indicate the CpG; and purple highlights indicate that the gene TSS and CpG are in close proximity to each other. Tracks included histone 3 lysine 4 monomethylation (H3K4me1: an enhancer mark), histone 3 lysine 4 trimethylation (H3K4me3: marks transcription start site), histone 3 lysine 27 trimethylation (H3K27me3: marks transcription repression), histone 3 lysine 36 trimethylation (H3K36me3: marks active transcription), and histone 3 lysine 27 acetylation (H3K27ac: marks active transcription) found in 7 cell lines from ENCODE and/or in placenta smooth chorion. Tracks labeled “DNase1” represent DNaseI hypersensitivity clusters in 125 cell types from ENCODE. Tracks labeled “ChIP-seq” represent ChIP-seq clusters in 338 factors and 130 cell types from ENCODE and ChIP-seq marks found in placenta smooth chorion. Tracks labeled “Promoter/enhancer” and “Interaction” represent GeneHancer Double Elite Regulatory Elements and GeneHancer Double Elite Clustered Interactions, respectively. Gray shade denotes a 95% confidence interval. Source data for Fig. 4a–d are provided as a Source Data file. We identified 1839 sex-mQTL consisting of 1529 unique genetic variants and 521 unique CpGs (FDR-adjusted P < 0.05) (Fig. The mean (± standard deviation [sd]) distance between the genetic variant and target CpG in a sex-mQTL was 394.3 (± 306.4) kb pairs, suggesting that genetic variants anywhere within the 1 Mb distance can have sex-dependent effects on methylation in the placenta. As expected44, the genetic variant's distance from its associated CpG was inversely correlated with its sex-biased effect on methylation (rs = −0.20, P = 7.32 × 10−19). However, genetic variants may induce epigenetic regulation in the placenta through long-range chromatin interactions38 and distal transcript regulatory elements such as enhancers45,46. Among our sex-mQTL SNPs, 37 SNPs overlap with predicted placental enhancers by Owen et al.46, and 36 SNPs overlap with predicted placental enhancers by Zhang et al.45. The majority of SNPs that showed enhancer overlaps (i.e., 94.5% (35/37) in ref. 45) were more than 100 kb away from the sex-mQTL CpG (Supplementary Data 4). A sex-QTL association can fall into a sex-specific, concordant, or opposite effect category based on the relative direction and magnitude of the allelic effect in males and females (see Methods section). In a sex-stratified analysis of the 1839 sex-mQTL pairs, 92.5% (1701/1839) had FDR-significant mQTL association in male and/or female placenta and further showed a significant difference in effect estimates between males and females at FDR-adjusted Pdiff < 0.05. Among the 1701 sex-mQTLs with effects that differed by sex, 83.8% (1426/1701) were specific to one sex only, the strongest male-specific effect being for rs12242275/rs7093024-cg09082518 [CCDC6] (βmale(M) = − 0.13, PM = 6.77 × 10−18, βfemale(F) = − 0.0026, PF = 0.49; Pdiff = 1.13 × 10−13; minor allele frequency (MAF) = 5.2%) and the strongest female-specific effect being for rs4351362-cg14735364 [NRF1] (βM = − 0.003, PM = 0.46, βF = 0.12, PF = 1.64 × 10−12; Pdiff = 7.06 × 10−10; MAF = 5.7%) (Figs. Downregulation of CCDC6 in the placenta has previously been linked to preterm deliveries47. NRF1 encodes a transcription factor involved in mitochondrial biogenesis and oxidative phosphorylation, and its reduced placental expression has been linked to preeclampsia48. 7.9% (134/1701) of sex-mQTLs had concordant effects in males and females (i.e., directionally consistent effects that differ in magnitude), rs530705354-cg14711243[ACAD9] being the top sex-differentiated with a stronger effect in males than in females (βM = 0.27, PM = 1.31 × 10−36, βF = 0.11, PF = 2.45 × 10−10; Pdiff = 9.12 × 10−10; MAF = 6.2%). The remaining 8.3% (141/1,701) sex-mQTLs had opposite effects in males and females, with rs71304466-cg07784293[WWTR1] exhibiting FDR-adjusted association in both males and females and the strongest sex difference (βM = − 0.04, PM = 7.23 × 10−8, βF = 0.01, PF = 3.43 × 10−3; Pdiff = 3.29 × 10−8; MAF = 6%) (Supplementary Data 4). WWTR1 encodes a transcription cofactor that regulates trophoblast cell fate during placentation, and defective expression in the placenta has associations with preterm birth, intrauterine growth restriction, and preeclampsia49. 778 sex-mQTL SNP-CpG pairs were available and tested in the RICHS dataset, and 19 were significant at FDR-adjusted P < 0.05 and with consistent effect direction. The strongest male-specific mQTL association identified (i.e., rs12242275/rs7093024-cg09082518 [CCDC6]) was among those replicated in the RICHS dataset (Supplementary Data 5). One of our sex-mQTLs (rs34571066-cg13299927 [ARHGEF10L]) has previously been reported20. a Pie chart displaying the distribution of sex-biased methylation quantitative trait associations based on allelic effect magnitude and direction in males and females (Blue = concordant effect direction, n = 134; Green = opposite effect direction, n = 141; Orange = single sex effect, n = 1426; Gray = Unclassified, n = 138). “Unclassified” includes associations that do not fall into the other categories (i.e., sex-biased mQTL associations with two-sided t test FDR-adjusted P ≥ 0.05 (n = 291, df = 289) in both sex strata and/or those with two-sided t test with unequal variance FDR-adjusted Pdiff ≥ 0.05 (n = 291, df = 289). b Regional plot of top male-specific sex-mQTL (rs12242275-cg09082518). The plot to the right indicates the absence of effect in females. c Regional plot of top female-specific sex-mQTL (rs4351362-cp14735364). The plot to the right indicates the absence of effect in males. Data span 200 kb centered at the mQTL SNP. The horizontal axis denotes the genomic position in build hg19, and the vertical axis denotes the association -log10 P-value and recombination rate (cM/Mb). The purple circle point represents the mQTL SNP. The purple triangle point represents the mQTL CpG. The color of each data point indicates its linkage disequilibrium value (r2) with the index SNP based on HapMap2. LocusZoom (http://locuszoom.org/) was used to generate the plot. Source data for Fig. 5b, c are provided as a Source Data file. We found 14 sex-DE (FDR-adjusted P < 0.05), of which sex-DE of ANGPT2 has been previously reported20 and 13 were previously unreported (Fig. In the RICHS dataset, 12 sex-DE transcripts were available, and 10 showed directionally consistent sex-DE associations, but none was significant after FDR adjustment (Supplementary Data 6). None of the 14 genes exhibited sex difference in non-placental tissues from the genotype tissue expression (GTEx) portal50, aligning with evidence of poor correlation of sex-differentiated autosomal gene expression between term placenta and 42 non-reproductive adult tissues from GTEx51. None of the sex-DE has placenta-specific RNA expression as compared to other human tissues based on tissue-specificity metrics in the Human Protein Altas (https://www.proteinatlas.org/about/download#protein_atlas_data) or in a placental RNA-sequencing study (Supplementary Data 6)52,53. We identified 13 previously unreported sex-eQTL (FDR-adjusted P < 0.05), consisting of 13 unique variants and 9 unique gene transcripts (Fig. The mean (± sd) distance between a sex-eQTL SNP and target gene was 419.9 (± 216.5) kb pairs, covering a broad region within the 1 Mb genomic window. In a sex-stratified analysis of the 13 sex-eQTL pairs, 92.3% (12/13) had FDR-significant eQTL association in male and/or female placenta and further showed a significant difference in effect estimate between males and females (FDR-adjusted Pdiff < 0.05). All 12 sex-eQTLs exhibited a genetic effect in one sex only, the strongest female-specific effect being for rs79910893-LPCAT3 (βM = − 0.002, PM = 0.96, βF = 1.42, PF = 4.72 × 10−7; MAF = 5.6%) and the strongest male-specific effect being for rs11986287-SLC52A2 (βM = 0.88, PM = 1.26 x 10-12, βF = − 0.03, PF = 0.14; MAF = 5.6%) (Fig. None of the sex-eQTLs overlap with sex-eQTLs in 44 non-placental tissues from GTEx (https://gtexportal.org/home/downloads/adult-gtex/qtl)50, consistent with evidence that sex differences in genetic regulation of gene expression are highly tissue-specific50,54,55. Out of the 13 sex-eQTLs, only rs10892219-ATP5MG was available in the RICHS dataset and did not exhibit significant association after FDR adjustment. a Pie chart displaying the distribution of sex-biased expression quantitative trait associations based on allelic effect magnitude and direction in males and females (Orange = single sex effect, n = 12; Gray = Unclassified, n = 1). “Unclassified” includes associations that do not fall into the other categories (i.e., sex-biased eQTL associations with two-sided t-test FDR-adjusted P ≥ 0.05 (n = 71, df = 69) in both sex strata and/or those with two-sided t-test with unequal variance FDR-adjusted Pdiff ≥ 0.05 (n = 71, df = 69). b Regional plot of top male-specific sex-eQTL (rs11986287-SLC52A2). The plot to the right indicates the absence of effect in females. c Regional plot of top female-specific sex-eQTL (rs79910893-LPCAT3). The plot to the right indicates the absence of effect in males. Data span 200 kb north of the gene TSS position to 200 kb south of the eQTL SNP. The horizontal axis denotes the genomic position in build hg19, and the vertical axis denotes the association -log10 P-value and recombination rate (cM/Mb). The purple circle point represents the eQTL SNP. The color of each data point indicates its linkage disequilibrium value (r2) with the index SNP based on HapMap2. A gene symbol and associated track from the UCSC Genome Browser shows the eQTL target gene's physical location. LocusZoom (http://locuszoom.org/) was used to generate the plot. Source data for Fig. 6b, c are provided as a Source Data file. To identify loci which may concurrently exhibit sex differences in methylation level, gene expression level, genetic regulation of methylation, and genetic regulation of gene expression, we looked for overlaps among our results (i.e., sex-DM, sex-DE, sex-mQTL, sex-eQTL). Only two overlaps were found: (i) A co-occurrence of sex-DM and sex-DE: male-biased cg25364822 methylation at the transcription start site of FNDC5 and female-biased expression of FNDC5. In turn, higher methylation at cg25364822 was significantly correlated with lower FNDC5 expression in males (rs = − 0.36, P = 0.023), but not in females (rs = 0.02, P = 0.91). FNDC5 is a precursor of irisin, which protects the placenta from oxidative stress, dysregulation of placental trophoblast differentiation, and insulin resistance56,57. Lower irisin levels have been linked to preeclampsia56,57, a placenta insufficiency-disorder correlated with smaller placental weight58. In our data, increased cg25364822 methylation and decreased FNDC5 expression were correlated with smaller placental weight (rs = − 0.12, P = 0.04; rs = 0.29, P = 0.01, respectively), suggesting a sex-biased epigenetic process that may regulate gene expression levels that impact placental function. (ii) A co-occurrence of sex-DM and sex-mQTL: male-biased cg27576576 methylation and male-biased association of the G allele of rs6459811 (PTPRN2) with higher cg27576576 methylation. The rarity of overlaps suggests that sex-QTL effects in the placenta are not largely explained by or accompanied with sex differences in methylation or sex differences in gene expression. Similarly, distinct genetic loci have been implicated in sex-biased eQTL and sex-biased gene expression in other tissues50,59. Furthermore, we found significantly high correlations in expression levels of sex-eQTL target genes between males and females (rs = 0.83, P = 0.0083) and in methylation levels of sex-mQTL target CpGs (rs = 1, P < 2.2 × 10−16) (Fig. 7a, b), suggesting largely non-overlapping sex-dependent placental methylation and gene expression. a DNA methylation level of sex-mQTL target CpGs (n = 521 CpGs). b Gene expression levels of sex-eQTL target genes (n = 9 gene transcripts). A two-sided Spearman correlation (ρ) test was implemented and was considered significant if P < 0.05. The p-value in a is approximately equal to zero. The error band in gray denotes a 95% confidence interval. Source data for Fig. 7a, b are provided as a Source Data file. As the placenta is a cell-heterogenous organ, we assessed whether the sex-QTL associations were due to sex differences in placental cell composition. We found no significant sex difference in placental cell type proportion derived in silico using methylation data60 or RNA-seq data61 (Supplementary Data 8). Second, cell type proportions were correlated with the top three methylation principal components (Supplementary Fig. 1) and top three RNA principal components (Supplementary Fig. 2), which were regressed out of the methylation and expression data, respectively, prior to sex-QTL analysis. Third, expression variance components (i.e., PEER factors), which may capture cell type signals were regressed out prior to the sex-eQTL analysis. Therefore, the sex-QTLs identified were unlikely to arise from sex differences in placental cell composition. Male-hypermethylated sex-DM CpGs in the placenta were more likely than female-hypermethylated sex-DM CpGs to be positively correlated (P < 0.05) with neonatal weight, length, or head circumference (48.4% vs. 0.8%, chi-squared test P = 2.61 × 10−321), whereas female-hypermethylated sex-DM were more likely than male-hypermethylated sex-DM and to be positively correlated with placental weight or placental weight/birth weight ratio (20.3% vs. 3.5%, chi-squared test P = 2.11 × 10−207) (Supplementary Data 2and Fig. Among male-hypermethylated CpGs, ten were positively correlated with all three neonatal anthropometry measures and negatively correlated with placenta-birthweight ratio (CpGs map to EGLN1, FNBP1L, TSPAN14, TEAD1, MIR130A, TMEM223, DOCK9, SLC1A5, DGCR6L, and ST6GALC6 genes). EGLN1 encodes a protein in the hypoxia-inducible factor (HIF) pathway and is important for placental development through its role as an oxygen sensor and regulator of HIF1A expression in trophoblast cells62,63. Among female-hypermethylated CpGs, three were positively correlated with placenta-birth weight ratio and negatively correlated with at least two neonatal anthropometry measures (CpGs map to SHANK3, POP4, and GRHL3 genes). SHANK3 encodes a synaptic protein that is crucial for the development of brain circuits64. The SHANK3 gene locus has been implicated in cognitive function measurement and schizophrenia65,66 and in autism spectrum disorders64, which is more commonly diagnosed in males than females67. Placental hypomethylation at a nearby locus (22q13.33) has been linked with autism68, and hypomethylation of our female-hypermethylated CpG in adult blood DNA has been associated with brain gray matter volume69. The figures display the number of male-hypermethylated (a) and female-hypermethylated (b) CpG sites that are positively correlated with neonatal and placental measures, and the number of male-hypermethylated (c) and female-hypermethylated (d) CpG sites that are negatively correlated with neonatal and placental measures. The measures are head circumference (HC), birth length (BL), birth weight (BW), placental weight (PW), and placental-birth weight ratio (PW/BW). Vertical lines connecting black-shaded circles denote measures that exhibited a shared correlation with the number of CpGs in bars. We assessed whether the sex-biased loci have previously been implicated in fetoplacental growth. A male-hypermethylated sex-DM at cg10806146 (5'UTR of SLC20A2) has previously been associated with higher birthweight70. Slc20a2 is required for normal placental phosphate transport function in mice, and its deficiency has been linked with defective vascularization, increased calcification of the placenta, and fetal growth restriction71. In addition, three genes near sex-DM CpGs (HSPA4, SLC45A4, and SLC6A2) and two genes near sex-mQTL CpGs (SLC45A4 and ARHGAP26) map fetal genetic variants associated with placental weight in previous GWAS72. Further, we looked up the sex-biased loci in the EWAS (epigenome-wide association study) Atlas, EWAS Catalog, and GWAS Catalog. Several sex-DM and sex-mQTL CpGs overlapped with CpGs in blood previously reported to be associated with pregnancy complications (e.g., preterm birth, preeclampsia, gestational diabetes mellitus), fetal and child growth, adult complex diseases (e.g., aging), and prenatal environmental exposures. Sex-mQTL genetic variants, genes near sex-DM CpGs, and sex-DE genes have links with GWAS traits in adults (Supplementary Data 9). To explore the usefulness of sex-QTLs in understanding the molecular basis of complex trait GWAS discoveries, we performed colocalization analysis by integrating sex-stratified QTL in the placenta with GWAS. Among sex-mQTLs, we identified 18 CpGs colocalized with 19 distinct GWAS loci for birthweight (eCAVIAR posterior probability of sharing the same causal variant ≥ 0.01), revealing the importance of sex-differentiated epigenetic mechanisms in the placenta to interpret genetic associations with fetal growth, a sex-differentiated trait tightly regulated by the placenta2. The strongest colocalizations were between cg10611863 (ENTPD4) and birthweight for female-stratified mQTL, and between cg02370022 (CCND1) and birthweight for male-stratified mQTL. Among sex-eQTLs, we identified 4 colocalized gene-adult trait pairs, of which female-stratified eQTLs at ATP5MG and FAM83A colocalized with asthma/hay fever/eczema and breast cancer and male-stratified eQTLs at FAM103A2P and SLC52A2 colocalized with breast cancer and educational attainment, respectively (Supplementary Data 10). Imprinting, an epigenetic regulation of gene expression in a parent-of-origin-dependent manner, has well-known roles in regulating placental development and function22,73. Sex bias has been reported in the regulation of imprinted genes in rodent placenta74,75, so we tested whether there is enrichment of imprinting regions among the sex-DM. We found significant enrichment of imprinted genes in sex-DM genes (61 imprinted genes out of 2225 unique sex-DM genes compared to 228 known imprinted genes in the human genome; hypergeometric test P = 2.599 × 10−9). The largest regional cluster of sex-DM CpGs was a 63 kb region in the maternally expressed TP73 gene and the homeobox family genes (HOXA3, HOXA4, HOXA5, HOXB2, HOXB3, and HOXC9), known for their crucial roles in regulating placental and fetal development76. Thirteen of the 61 imprinted genes are known to be imprinted specifically in the placenta only77 (Table 1 and Supplementary Data 11). The identified imprinting overlaps included a cluster of 13 female-hypermethylated sex-DM CpGs within the well-known KCNQ1OT1/CDKN1C imprinting domain in chromosome 11p15.5 genomic region. In the human placenta, the KCNQ1OT1/CDKN1C imprinting domain harbors genes such as CDKN1C, PHLDA2, and SLC22A18 that are maternally expressed in the first trimester and term placenta, a long non-coding RNA (KCNQ1OT1/kvDMR1) with paternal expression in the first trimester and biallelic expression in term placenta, and other genes such as KCNQ1 which exhibits maternal expression in first-trimester placenta and biallelic expression in term placenta78. The imprinting domain is regulated by a maternally methylated DMR imprinting control region (ICR) located in intron 10 of the KCNQ1 gene78. A genetic variant in intron 10 of KCNQ1 has shown a maternal parent-of-origin effect on placental weight72. Out of the 13 sex-DM CpGs, 8 were located within KCNQ1 at 13.7–165.1 kb distance from the ICR and overlap with histone 3 lysine 27 trimethylation (H3K27me3), a posttranslational epigenetic modification associated with transcriptional repression. Given these annotations, we assessed whether methylation at the 8 CpGs has a potential sex-dependent relationship with the placental expression of genes in the chr11p15.5 imprinting domain. Higher CpG methylation showed correlation with increased KCNQ1OT1 expression in females and trended with decreased KCNQ1OT1 expression in males (cg03030994-KCNQ1OT1: rm = − 0.26, Pm = 0.12; rf = 0.36, Pf = 0.04), increased CDKN1C expression in females (cg00446023-CDKN1C: rm = − 0.09, Pm = 0.58, rf = 0.35, Pf = 0.046), and decreased SLC22A18 and KCNQ1 expression in males (cg05457684-SLC22A18: rm = − 0.44, Pm = 0.006, rf = 0.04, Pf = 0.83; cg05457684-KCNQ1: rm = − 0.46, Pm = 0.004; rf = 0.05, Pf = 0.79; cg06960356-KCNQ1: rm = − 0.42, Pm = 0.009; rf = 0.14, Pf = 0.45). The KCNQ1OT1/CDKN1C domain has been linked to diseases and trait differences. For example, imprinting dysregulation of genes in the KCNQ1OT1/CDKN1C domain has been linked to Beckwith-Wiedemann syndrome, a disorder of growth regulation characterized by somatic overgrowth and tumor predisposition79. Moreover, increased placental expression of the maternally expressed CDKN1C, PHLDA2, and SLC22A18 genes has been linked with fetal growth restriction and smaller neonatal size80,81,82,83,84. We found that three female hypermethylated sex-DM CpGs showed correlation, albeit weakly, with lower birthweight (cg25548316: rs = − 0.13, P = 0.03), higher placental weight (cg13536051: rs = 0.11, P = 0.048; cg15782852: rs = 0.18, P = 0.002), and higher placenta-birthweight ratio (cg13536051: rs = 0.2, P = 0.001; cg15782852: rs = 0.14, P = 0.02). Most canonical pathways and hallmark gene sets that were enriched for genes near male-hypermethylated sex-DM CpGs differed from those enriched for female-hypermethylated sex-DM CpGs (Fig. The top canonical pathways enriched only in male-biased methylation included extracellular matrix organization, hemostasis, and immune response (e.g., chemokine and Family B G-protein-coupled receptors); pathways enriched only in female-biased methylation included transcription regulation, transportation of small molecules in cells, and regulation of genes involved in metabolism by the tumor suppressor protein TP53. The top hallmark gene sets enriched only in male-biased methylation included coagulation, epithelial-mesenchymal transition, upregulation of KRAS signaling, myogenesis, and allograft rejection; hallmark gene sets enriched only in female-biased methylation included mitotic spindle, early estrogen response, apoptosis, and glycolysis. For the minority of pathways and hallmark gene sets that overlapped between male- and female-hypermethylated sex-DM CpGs, few or no genes were common between males and females. Such gene set convergence through different genes suggests that male and female placentas may coordinate different CpGs/genes for an essentially identical biological process or potential compensatory mechanisms. The outer track represents the enriched canonical pathways obtained using the FUMA tool, ordered from the most to the least significant in the clockwise direction. The middle track represents the hypergeometric test for enrichment FDR-adjusted P-value. The orange bars in the inner-most track represent the number of genes near the sex-differentially methylated sites that overlapped with genes in the database of each pathway. The green lines connect enriched pathways shared between males and females, with width proportional to the number of shared genes. Database abbreviations: B: BIOCARTA, K: KEGG, N: NABA, P: PID, R: REACTOME, S: SIG. Source data are provided as a Source Data file. Transcription factor (TF) activity can contribute to sex-biased gene regulation55. We found that several transcription factor binding sites (TFBS) are enriched for the sex-biased methylation loci. The identified TFBS are associated with TFs that regulate target genes through diverse mechanisms. Specifically, the sex-mQTL genetic variants were significantly enriched for three TFBS associated with TFs that regulate target genes via functions such as nuclear hormone receptors (e.g., NR2C2), sequence-specific DNA binding (SP2), and epigenetic remodeling (REST) (FDR-adjusted P < 0.05). NR2C2 (nuclear receptor subfamily 2 group C member 2) encodes the testicular receptor 4 protein TR4. In vivo mice studies have shown that Nr2c2 plays key roles in fetal growth, early postnatal survival, fertility, and sensitivity to environmental stimuli85,86. Male Nr2c2-null mice exhibit delayed spermatogenesis and reduced fertility86, and females exhibit maternal behavioral abnormalities85. Sex-DM and sex-mQTL CpG genomic positions were significantly enriched for TFBS associated with TFs in the zinc finger family, cell cycle regulation (E2F), sequence-specific DNA binding (THAP), site-specific DNA binding and embryogenesis (AP-2) (Supplementary Fig. This study offered several unique insights about the landscape of sex differences in the level and genetic regulation of methylation and gene expression in the human placenta. Key findings include the following: (i) Placental autosomal methylation and its genetic regulation are extensively impacted by fetal sex, whereas sex differences in both the level and the genetic regulation of placental autosomal gene expression are limited. (ii) The vast majority of placental methylation sites and gene transcripts impacted by sex-biased genetic effects did not exhibit methylation and gene expression differences by sex. Moreover, methylation sites that were sex-biased were not enriched for predicted TF binding, did not largely correlate with annotated gene expression, and were enriched for molecular pathways largely distinct from previously described pathways related to sex-biased gene expression in placenta14,17. These observations suggest that distinct mechanisms may underlie sex-dependent placental methylation, gene expression, and their genetic regulation. (iii) The sex-biased methylation sites overlapped with genetic loci implicated in several human phenotypes. Notably, several sex-differentiated genetic effects on methylation and gene expression in the placenta colocalized with birthweight, and adult traits such as breast cancer and allergic diseases, contributing to the mechanistic interpretation of GWAS signals. These findings suggest the potential role of placenta-mediated sex differences in developmental and later-life physiological traits and diseases. Many genetic variants showing sex-biased association with placental methylation and gene expression had a single-sex effect, unveiling biological insights that would be masked or attenuated in standard combined-sex QTL studies. One such example is an mQTL near STMN3 (Stathmin-3) (rs1151625-cg14862171), identified previously by a combined-sex analysis of placenta samples87, and was found in the present study to be a sex-mQTL locus conforming to a female-specific effect. The genetic variant's effect on methylation was 45-fold stronger in females than male placenta and 1.29-fold stronger in female than combined-sex sample placenta. These results clarify that the previously reported association based on combined-sex analysis87 was most likely driven by the effect of the genetic variant on methylation in the female placenta only. STMN3 encodes a microtubule destabilizing protein that regulates placentation in early pregnancy in vitro88. The largely distinct sex-DM and sex-DE associations detected in the present study align with a recent report20. Our study implemented more omics-integrated analyses, including directly testing association of sex-DM CpGs with nearby gene expression, identifying sex-eQTL associations, and implementing analyses models that included interaction terms between sex and genetic variants. These approaches yielded unique insights that sex-DM CpGs were more likely to be associated with nearby gene expression alteration and enabled the detection of several previously undetected sex-mQTLs and sex-eQTLs with single-sex effects. Male fetus-bearing pregnancies are at increased risk for major pregnancy complications such as pre-eclampsia, placental abruption, fetal growth restriction, and post-partum hemorrhage, all of which have molecular origins tied to placentation and hemostasis4. In our study, male-biased placental methylation was markedly enriched for the hemostasis/coagulation hallmark gene set that has not previously been characterized as sex-differentiated. The placenta expresses coagulation components involved in hemostasis and placental vascular development89. The male-enriched coagulation hallmark gene set included tissue factor protein inhibitor-2 (TFPI2; aka placental protein 5 (PP5)), an extracellular matrix-associated protein known to be abundantly produced by the placenta and is actively involved in coagulation and hemostasis processes90. Transition to a state of hypercoagulability is maximal around the term to prevent maternal and child health complications due to excessive bleeding91. Our finding suggests that epigenetically induced suppression of coagulation pathways can be one potential explanation for male preponderance in pregnancy pathologies and contributes to emerging knowledge on coagulation-inflammation/immune response crosstalk during pregnancy. We found a previously unrecognized enrichment of imprinting genes in sex-differentiated placental methylation, including a cluster of female-biased methylation within the KCNQ1OT1/CDKN1C imprinting domain implicated in Beckwith-Wiedemann syndrome78,79. In our data, higher methylation of the female-hypermethylated CpGs was associated with female-specific upregulation or male-specific downregulation of genes in this imprinting domain. These findings suggest a potential crosstalk between sex and imprinting regulation in the placenta. The unique epigenetic architecture of imprinting in the human placenta may be permissive to epigenetic alterations. For example, in the KCNQ1OT1/CDKN1C cluster, loss of imprinting is variable, and gene repression by the KCNQ1OT1 lncRNA is incomplete92. Human placental DMRs are uniquely highly polymorphic73,78,93. Placental DMRs are associated with variable histone marks94 which are mechanistically linked to DNA methylation that influences imprinting stability95 and potentially with underlying genetic variation93. Collectively, these studies and our findings suggest that the imprinting domains may be vulnerable to epigenetic alterations from other factors that influence allelic bias in a sex-dependent manner, but future work should investigate this and its relevance to sex differences in pregnancy physiology and clinical complications. Our study has limitations. First, placental gene expression and methylation are dynamic over gestation16,17. However, the current study is unable to distinguish sex differences that emerge in early gestation from those that emerge in later gestation because the placenta samples were obtained at or near-term gestation. Second, genetic regulation of methylation and gene expression in the placenta may vary by cell type. We found that in silico-derived placental cell type composition did not vary by sex and did not explain the observed sex-QTLs. Similar to ours, the GTEx study did not find sex differences in cell composition for 41 out of 44 tissues tested50. These findings should be seen in light of the caveat that existing in-silico methods for estimating the cell type composition of the placenta do not fully characterize the spatial heterogeneity of the placenta. Better insights may be gained in the future through single-cell analysis of the placenta from different developmental stages. Third, differences related to sex chromosomes will require future methodological advances that can facilitate analyses that integrate the autosomes with the haploid Y chromosome and dosage compensation of the X chromosome1. Fourth, the detection of sex-eQTLs may have been limited by the sample size of our placental RNA-seq dataset. The inclusion of covariates to account for genetic population structure and cell heterogeneity may have further impacted the study's power in detecting weaker signals. Similar to our findings, the limited sex difference in gene expression regulation has been observed in 44 non-placental tissues from GTEx50 and in adult blood96,97. Moreover, one study demonstrated that samples of millions of individuals would be required to detect sex-biased eQTLs in the blood that appreciably mediate sex-biased genetic associations with complex traits96. Larger datasets and accounting for factors that influence QTLs can improve the transferability of sex-QTLs found to be limited in our study as well as others50,98. In conclusion, this multi-omics study in the placenta identified sex-specific regulatory processes and molecular pathways and potential crosstalk between placental sex and imprinting. Sex-biased genetic regulation of placental methylation and gene expression colocalized with GWAS loci for neonatal traits and adult diseases, revealing placental embedding of sex differences in human health and disease across the life span. The NICHD Fetal Growth study protocol was approved by the institutional review boards of NICHD and each of the participating clinic sites, namely, Columbia University, New York; New York Hospital, Queens, New York; Christiana Care Health System, Delaware; Saint Peter's University Hospital, New Jersey; Medical University of South Carolina, South Carolina; University of Alabama, Alabama; Northwestern University, Illinois; Long Beach Memorial Medical Center, California; University of California, Irvine, California; Fountain Valley Hospital, California; Women and Infants Hospital of Rhode Island, Rhode Island; and Tufts University, Massachusetts. The RICHS study was approved by institutional review boards of Emory University and Women and Infants Hospital of Rhode Island. Datasets from placenta samples obtained at delivery as part of the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD) Fetal Growth Studies – Singletons were included in the discovery analysis. The NICHD Fetal Growth Studies – Singletons recruited 2802 pregnant women between July 2009 and January 2013 at 12 clinical sites in the United States from four race/ethnic groups (i.e., non-Hispanic White, non-Hispanic Black, Hispanic, and Asian or Pacific Islander) between 8–13 gestational weeks and followed through delivery. To be enrolled, women had to have no past adverse pregnancy outcomes and no major pre-existing medical conditions, including autoimmune diseases, chronic hypertension, diabetes, chronic renal disease, cancer, HIV/AIDS, or psychiatric disorders. Gestational age was determined using the date of the last menstrual period and confirmed by ultrasound between 8 weeks to 13 weeks and 6 days of gestation. After the first ultrasound, pregnant women underwent up to five standardized ultrasounds with measurement of fetal biometry at a priori-defined gestational ages25. Birth weight was measured in grams (g) using an electronic infant scale or beam balance scale, and birth length and head circumference were measured in centimeters (cm). Placenta weight was measured in grams (g). Written informed consent was obtained from all study participants. Placental biopsies measuring 0.5 cm × 0.5 cm × 0.5 cm were taken from the fetal side (n = 312) within one hour of delivery, and samples were placed in RNAlater and frozen in − 80 °C for molecular analysis87. DNA extracted from the placental biopsies was genotyped using HumanOmni2.5 Beadchip (Illumina Inc., San Diego, CA). For the genotype dataset, detailed quality control procedures have been described previously21,87,99,100. Briefly, single nucleotide polymorphisms (SNPs) with >5% missing values, minor allele frequency <0.5%, and not in Hardy-Weinberg equilibrium (P < 10−4) we removed. SNP genotypes were imputed using the Michigan Imputation Server (https://imputationserver.sph.umich.edu/) using the 1000 Genomes Phase 3 genotype reference (https://www.internationalgenome.org/category/reference/), and filters were applied to remove insertion-deletions, SNPs with minor allele frequency < 0.5% and SNPs with imputation dosage r2 < 0.3. Infant sex (male or female) was obtained from medical charts and was compared with sex predicted based on X chromosome heterozygosity (inbreeding coefficient estimate F > 0.8 male, F < 0.2 female)21,87,99,100. DNA methylation was profiled on the 312 samples using Illumina's Infinium Human Methylation450 Beadchip (Illumina Inc., San Diego, CA). Probes which were cross-reactive, non-autosomal, had mean detection P ≥ 0.05, or had CpGs located within 20 base pairs of known SNPs were removed. Samples that had discrepancies between sex obtained from medical charts and predicted from genotypes, were outliers from the distribution of the samples' genetic clusters or had a mismatching sample identifier were removed, as previously described21,87,99,100. RNA from a subset of the placental samples (n = 80) was extracted using TRIZOL reagent (Invitrogen, MA), and RNA sequencing was performed using the Illumina HiSeq2000 system with 100 bp paired-end reads. The reads were mapped to the human reference genome (NCBI/build 37.2) using Tophat version 2.0.4. The raw placental RNA-seq dataset had a total of 33,690 RNA transcripts. Read count data were normalized across libraries using the trimmed mean of M values (TMM) normalization technique implemented in the R/Bioconductor package “edgeR”101. Transcripts in autosomal chromosomes without 6 or more reads in at least 3 samples or without depth-normalized counts per million (CPM) of 0.1 or higher in at least 3 samples were removed, as previously described22,87. The following datasets that passed previously performed quality control filters mentioned above were included in the present analyses: 301 samples (152 males and 149 females) with placental methylation data at 408,680 CpGs were included in sex-DM analysis, of which 291 samples (147 males and 144 females) with fetal genotype data at 5,359,103 genetic variants as well as placental methylation data at 408,680 CpGs were included in sex-mQTL analysis. A total of 80 sub-samples (43 males and 37 females) with placental RNA-seq data at 21,550 transcripts were included in sex-DE analysis, of which 71 samples (38 males and 33 females) with fetal genotype data at 5,337,343 genetic variants as well as RNA-seq at 21,550 transcripts were included in sex-eQTL analysis. A total of 301 samples (152 males and 149 females) with placental methylation data at 408,680 CpGs were included in the sex-DM analysis. We estimated placental cell type proportion in silico using DNA methylation beta values with the R/Bioconductor package “planet” (https://www.bioconductor.org/packages/release/bioc/html/planet.html)60. Genotype-based principal components (PCs) representing population structure were estimated using fetal genome-wide SNP genotype data. Methylation PCs were estimated using the R package “prcomp”102. To identify sex-differentiated methylation (sex-DM), epigenome-wide analyses were performed with fetal sex as the predictor and placenta DNA methylation at each CpG site as the outcome using the R/Bioconductor package “limma”103. The analysis included linear regression models that were adjusted for self-reported maternal race/ethnicity (non-Hispanic White, non-Hispanic Black, Hispanic, Asian), gestational age at delivery, methylation sample plate (n = 5), the first three methylation PCs, the first 10 genotype PCs to account for population structure, and predicted cell type proportions for six placental cell types60. To account for the inflation of statistical tests, we implemented a Bayesian method to obtain BACON-corrected inflation estimates and BACON-corrected P-values using the R/Bioconductor package “BACON”104. Quantile-quantile (QQ) plots of P-values and the corresponding inflation estimate after BACON-correction (λ = 1.271) are reported in Fig. BACON-corrected P-values were then controlled for false discovery rate (FDR), giving BACON-corrected FDR-adjusted P-values. To evaluate relations between sex-DM CpGs and gene expression, we tested whether methylation at each sex-DM CpG is correlated with the placental expression of its closest gene. This test was performed between 3636 sex-DM CpGs located in or near a gene (i.e., in gene body, 3'untranslated region (UTR), 5'UTR, exon, transcription start site (TSS) based on the University of California Santa Cruz (UCSC) genome reference annotation; https://genome.ucsc.edu/cgi-bin/hgGateway105) and placenta nearby gene expression levels in our dataset. In further exploration, we assessed the correlation between methylation at each sex-DM CpG and placental expression of all genes within 200 kb distance from the CpG site. A total of 291 samples (147 males and 144 females) with both placental methylation data at 408,680 CpGs and fetal genotype data at 5,359,103 genetic variants were included in the sex-mQTL analysis. Sex-mQTL analysis was performed to identify sex-mQTLs, i.e., SNPs whose effect on placental methylation differs in magnitude between male and female pregnancies. The sex-mQTL analysis was performed in two stages to accommodate the computational burden of the model for analysis in MatrixEQTL106. First, residuals of each methylation site were obtained by running a linear regression model with each methylation site as an independent outcome and race/ethnicity, gestational age at delivery, methylation plate, 3 methylation PCs, and 10 genotype PCs as covariates. Next, each methylation residual was fit as the outcome in a linear regression model with SNPs within 1 Mb distance and an interaction term between SNP and sex using MatrixEQTL106. The QQ plot of P-values for the SNP*sex interaction term of the mQTL mapping showed the absence of inflation (λ = 1.001) (Fig. Statistical significance for the interaction term was defined based on FDR-adjusted P < 0.05. Next, sex-stratified cis-mQTL analyses were performed on the FDR-significant mQTL-CpG pairs in a linear regression model with methylation as outcome, SNP within 1 Mb distance as predictor, and race/ethnicity, gestational age at delivery, methylation plate, 10 genotype PCs, and 3 methylation PCs as covariates. For pairs found to have FDR-significant mQTL association in either or both males and females (FDR-adjusted P < 0.05, accounting for the total number of mQTL-CpG pairs tested in the two strata), we performed a t-test of the difference between the standardized male-specific and female-specific effect estimates, assuming unequal variance (Pdiff). Lastly, we classified the sex-mQTLs with FDR-adjusted Pdiff < 0.05 into one of the following three categories: (i) concordant effect: association found to be FDR-significant in one sex and nominally significant (P < 0.05) or FDR-significant in the other sex with a consistent effect direction but different magnitude; FDR-adjusted Pdiff < 0.05. (ii) opposite effect: association found to be FDR-significant in one sex and nominally significant (P < 0.05) or FDR-significant in the other sex a with an opposite effect direction; FDR-adjusted Pdiff < 0.05. (iii) Single-sex effect: association found to be FDR-significant in one sex but not nominally significant (P < 0.05) in the other sex; FDR-adjusted Pdiff < 0.05. A total of 80 samples (43 males and 37 females) with placental RNA-seq data at 21,550 transcripts were included in the sex-DE analysis. Analysis was performed using edgeR, adjusted for race/ethnicity, gestational age at delivery, and 10 genotype PCs. Quantile-quantile (QQ) plots of P-values and the corresponding inflation estimate after BACON correction (λ = 1.326) are reported in Fig. A total of 71 individuals (38 males and 33 females) with both fetal genotype data at 5,337,343 genetic variants and RNA-seq at 21,550 transcripts were included in sex-eQTL analysis. Sex-eQTL analysis was performed in two stages to accommodate the computational burden of the model for MatrixEQTL106. First, residuals of each transcript were obtained by running a linear regression model with each transcript as the independent outcome and race/ethnicity, gestational age at delivery, 10 PEER factors (i.e., expression variance components)107, 3 RNA PCs, and 10 genotype PCs as covariates. Next, each transcript's residual was fit as an outcome in a linear regression model with SNPs within 1 Mb distance, and an interaction term between SNP and sex using MatrixEQTL106. The QQ plot of P-values for the SNP*sex interaction term of the eQTL mapping showed the absence of inflation (λ = 0.988) (Fig. Statistical significance for the interaction term was defined based on FDR-adjusted P < 0.05. Next, sex-stratified cis-eQTL analyses were performed in a linear regression model with gene expression as an outcome, SNP within 1 Mb distance as the predictor, and race/ethnicity, gestational age at delivery, 3 PEER factors, 4 genotype PCs, and 3 RNA PCs as covariates. The same workflow described for sex-mQTL was implemented to classify the sex-eQTLs with FDR-adjusted Pdiff < 0.05 into concordant effect, opposite effect, or single-sex effect. The sex-DE, sex-mQTL, and sex-eQTL associations discovered were assessed in a dataset from the RICHS cohort (n = 148; 74 males, 74 females). Mother and infant pairs were recruited following delivery at the Women and Infants Hospital of Rhode Island from 2009 to 2013. The study included infants born small for gestational age, large for gestational age, and controls born appropriate for gestational age-matched on sex, gestational age, and maternal age26. Placental RNA-seq data from a subset of samples (n = 200) were obtained using the Illumina Hi-Seq 2500 platform; placental DNA methylation data (n = 220) were obtained using the Infinium MethylationEPIC array (Illumina); and genotype data for 159 infants were obtained using the Illumina MegaEX array and imputed using the Haplotype Reference Consortium reference panel. Both sex-mQTL and sex-eQTL analyses were performed on 148 samples (74 male, 74 female) with genotype, DNA methylation data, and RNA-seq data. Of the 1839 SNP-CpG pairs in the sex-mQTL associations discovered in the NICHD Fetal Growth Studies cohort, 778 SNP-CpG pairs were available and tested in RICHS. Of the 14 SNP-gene pairs in the sex-eQTL associations discovered, only 1 pair (rs10892219-ATP5MG) was present and tested in RICHS. Of the 14 sex-DE associations discovered, 12 transcripts with RNA-seq data in RICHS were tested. Sex-QTL analyses were performed using linear regression models as in the discovery analysis with adjustment for self-reported ethnicity, methylation or RNA-seq batch, top 10 genotype PCs, and top 3 expression PCs (for sex-eQTL) or methylation PCs (for sex-mQTL). Sex-DE analyses were performed using linear regression models as in the discovery analysis with adjustment for race/ethnicity, gestational age at delivery, and 10 genotype PCs. For each of our samples, we derived proportions for 6 placental cell types (trophoblast, stromal, Hofbauer, endothelial, nucleated red blood cells (nRBC), syncytiotrophoblast) using methylation data and a reference panel of placental cell counts as implemented in the R package “planet” (https://www.bioconductor.org/packages/release/bioc/html/planet.html)60. We evaluated whether cell type proportions were different between males and females using a Wilcoxon rank-sum test (P < 0.05/6). We further assessed the correlation between cell proportion and methylation PCs 1-3, which were regressed out of the methylation data during sex-mQTL analysis. For RNA-seq, we also derived proportions for 27 placental cell types with CIBERSORTx61 based on placental single-cell sequencing reference composed of 19 fetal and 8 maternal cell types108 and tested whether cell type proportions were different between males and females using a Wilcoxon rank-sum test (P < 0.05/27). We further assessed the correlation between cell proportion and RNA PCs 1-3, which were regressed out of the transcript data during sex-eQTL analysis. To assess whether a shared causal genetic variant underlies both a complex phenotype and sex-biased gene expression/methylation in the placenta, we performed GWAS-QTL colocalization analysis using the ezQTL Web platform (https://analysistools.cancer.gov/ezqtl/#/home)109. ezQTL performs colocalization analysis using two methods (eCAVIAR and HyPrColoc) by integrating GWAS summary statistics, QTL results, and linkage disequilibrium (LD) matrix data. GWAS summary statistics for birthweight were obtained from the Early Growth Genetics Consortium (https://egg-consortium.org/birth-weight-2019.html). We performed colocalization analysis on the sex-QTL loci separately for male- and female-stratified QTL, each integrated with GWAS summary statistics and LD matrix data from the 1000 G samples (Supplementary Data 13). A sex-QTL locus was tested for GWAS-colocalization provided a given GWAS summary statistics dataset (from 423 publicly available GWAS trait summary statistics available in ezQTL) contains a SNP within 1 Mb window from the QTL lead SNP with a GWAS association P < 5 × 10−8. 63 eQTL-GWAS locus pairs representing 42 unique phenotypes and 96 mQTL-GWAS locus pairs representing 16 unique phenotypes which fulfilled this requirement were tested, separately for males and females. Colocalization was considered significant if eCAVIAR colocalization posterior probability was ≥0.01 or HyprColoc posterior probability was ≥ 0.5 based on a window of 50 SNPs around the GWAS lead SNP, as recommended in ezQTL. Associations of sex-DM CpGs or sex-DE transcripts with measures of neonatal anthropometry measured at birth (i.e., birthweight, birth length, head circumference) and placental size (i.e., placental weight and placental/birth weight ratio) were tested using the Spearman correlation test. The EWAS catalog (http://ewascatalog.org/)110 and EWASAtlas (https://ngdc.cncb.ac.cn/ewas/atlas/index)111 were searched to check whether the sex-DM and sex-mQTL CpGs overlap with previously known EWAS loci. The GWAS catalog v1.0.2 (https://www.ebi.ac.uk/gwas/; downloaded on December 2, 2021)112 was searched to check whether the sex-eQTL SNPs, sex-mQTL SNPs, sex-DE genes, and sex-DM genes overlap with previously known GWAS loci. To test the enrichment of transcription factor binding sites (TFBS) in the QTL genetic variants, we separately submitted sex-eQTL and sex-mQTL SNPs to SNP2TFBS (https://epd.expasy.org/snp2tfbs/; a web interface for querying genetic variants that affect transcription binding sites)113. The sex-DM and sex-mQTL target CpGs were annotated for regulatory features from ENCODE, Roadmap Epigenomics, and GENCODE (https://zwdzwd.github.io/InfiniumAnnotation/EPIC_hm450_hg19.html). Enrichment of TFBS in the genomic region flanking 200 bp from sex-DM or sex-mQTL target CpGs was tested using the JASPAR enrichment tool (https://jaspar.genereg.net/)114. JASPAR implements enrichment computations in the LOLA tool to test whether TFBS from the JASPAR database are enriched in our genomic regions of interest compared to CpG probes from the Illumina Infinium HumanMethylation450k microarray, separately for the following three sets of CpGs: i) CpGs hypermethylated in males, ii) CpGs hypermethylated in females, and iii) sex-mQTL targets CpG sites. We tested for enrichment of CpG positions with the functional elements DNase 1 hypersensitive sites (DHS), 15-state chromatin marks, and H3 histone marks compared to CpGs on the Illumina 450k array based on functional information from the Consolidated Roadmap Epigenomics database using eFORGE v2.0 (https://eforge.altiusinstitute.org/)115. Two sets of sex-DM genes (i.e., genes mapping male-hypermethylated CpG sites and female-hypermethylated CpGs) were separately annotated in biological context using the GENE2FUNC option using FUMA, a web-based platform that facilitates functional annotation of GWAS results (https://fuma.ctglab.nl/)116. Gene set enrichment analysis was performed using FUMA to test whether the input genes were enriched in GO Biological Process ontology and hallmark gene sets, which are biological states displaying coordinated expression as defined by the Molecular Signatures Database (MsigDB v7.0). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. As part of the NICHD Fetal Growth Studies, which is the discovery cohort in the present study, the genotypes, DNA methylation, and gene expression data have been deposited in the dbGaP database under accession code phs001717.v1.p1 [https://www.ncbi.nlm.nih.gov/gap/?term=phs001717.v1.p1]. Moreover, as part of the Rhode Island Child Health Study (RICHS), which is the replication cohort in the present study, the genotypes and gene expression data have been deposited in the dbGaP database under accession code phs001586.v1.p1 [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001586.v1.p1]. The following datasets made available by other groups were also used in the present analyses: the 1000 Genomes Reference Panel datasets were accessed at https://www.internationalgenome.org/category/reference/; the human genome reference made accessible by the Genome Reference Consortium was accessed at https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.13/; genotype imputation platform, as well as the Haplotype Reference Consortium reference panel, was accessed via the Michigan Imputation Server at https://imputationserver.sph.umich.edu/; and the Human Protein Altas dataset was downloaded at https://www.proteinatlas.org/about/download#protein_atlas_data. Source data are provided in this paper. Khramtsova, E. A., Davis, L. K. & Stranger, B. E. The role of sex in the genomics of human complex traits. Clifton, V. L. Review: Sex and the human placenta: mediating differential strategies of fetal growth and survival. Eriksson, J. G., Kajantie, E., Osmond, C., Thornburg, K. & Barker, D. J. Boys live dangerously in the womb. A. et al. Fetal sex and maternal pregnancy outcomes: a systematic review and meta-analysis. Sandovici, I., Fernandez-Twinn, D. S., Hufnagel, A., Constância, M. & Ozanne, S. E. Sex differences in the intergenerational inheritance of metabolic traits. Kim, D. W., Young, S. L., Grattan, D. R. & Jasoni, C. L. Obesity during pregnancy disrupts placental morphology, cell proliferation, and inflammation in a sex-specific manner across gestation in the mouse. Evans, L. & Myatt, L. Sexual dimorphism in the effect of maternal obesity on antioxidant defense mechanisms in the human placenta. van Abeelen, A. F. M. et al. The sex-specific effects of famine on the association between placental size and later hypertension. A., Moritz, K. M., Roberts, C. T., Walker, D. W. & Dickinson, H. The placental response to excess maternal glucocorticoid exposure differs between the male and female conceptus in spiny mice. Kalisch-Smith, J. I., Simmons, D. G., Pantaleon, M. & Moritz, K. M. Sex differences in rat placental development: from pre-implantation to late gestation. Tesfaye, M. et al. Prenatal social support in low-risk pregnancy shapes placental epigenome. PrabhuDas, M. et al. Immune mechanisms at the maternal-fetal interface: perspectives and challenges. & Thornburg, K. L. Placental programming of chronic diseases, cancer and lifespan: a review. Sood, R., Zehnder, J. L., Druzin, M. L. & Brown, P. O. Gene expression patterns in human placenta. Buckberry, S., Bianco-Miotto, T., Bent, S. J., Dekker, G. A. & Roberts, C. T. Integrative transcriptome meta-analysis reveals widespread sex-biased gene expression at the human fetal-maternal interface. Gonzalez, T. L. et al. Sex differences in the late first trimester human placenta transcriptome. Braun, A. E. et al. Examining sex differences in the human placental transcriptome during the first fetal androgen peak. Martin, E. et al. Sexual epigenetic dimorphism in the human placenta: implications for susceptibility during the prenatal period. Gong, S. et al. Genome-wide oxidative bisulfite sequencing identifies sex-specific methylation differences in the human placenta. Czamara, D. et al. Sex differences in DNA methylation across gestation: a large scale, cross-cohort, multi-tissue analysis. Sex differences in the associations of placental epigenetic aging with fetal growth. Chatterjee, S. et al. Sex-specific placental gene expression signatures of small for gestational age at birth. Tekola-Ayele, F. et al. Placental multi-omics integration identifies candidate functional genes for birthweight. Bhattacharya, A. et al. Placental genomics mediates genetic associations with complex health traits and disease. Grewal, J. et al. Cohortprofile: NICHD Fetal growth studies-singletons and twins. Marsit, C. J., Maccani, M. A., Padbury, J. F. & Lester, B. M. Placental 11-beta hydroxysteroid dehydrogenase methylation is associated with newborn growth and a measure of neurobehavioral outcome. Andrews, S. V., Yang, I. J., Froehlich, K., Oskotsky, T. & Sirota, M. Large-scale placenta DNA methylation integrated analysis reveals fetal sex-specific differentially methylated CpG sites and regions. Inkster, A. M. et al. A cross-cohort analysis of autosomal DNA methylation sex differences in the term placenta. Abignente, E. et al. Research on heterocyclic compounds. Phenyl derivatives of fused imidazole systems: antiinflammatory activity. Lancaster, E. E. et al. Large-scale integration of DNA methylation and gene expression array platforms identifies both cis and trans relationships. Yang, X. et al. Gene body methylation can alter gene expression and is a therapeutic target in cancer. The DNA methylation paradox. Functions of DNA methylation: islands, start sites, gene bodies and beyond. Laurent, L. et al. Dynamic changes in the human methylome during differentiation. Yin, Y. et al. Impact of cytosine methylation on DNA binding specificities of human transcription factors. Smith, J., Sen, S., Weeks, R. J., Eccles, M. R. & Chatterjee, A. Promoter DNA Hypermethylation and Paradoxical Gene Activation. Kass, S. U., Landsberger, N. & Wolffe, A. P. DNA methylation directs a time-dependent repression of transcription initiation. Varberg, K. M. et al. Extravillous trophoblast cell lineage development is associated with active remodeling of the chromatin landscape. Wang, T. et al. Overexpression of the human ZNF300 gene enhances growth and metastasis of cancer cells through activating NF-kB pathway. Wang, X. et al. CSMD1 suppresses cancer progression by inhibiting proliferation, epithelial-mesenchymal transition, chemotherapy-resistance and inducing immunosuppression in esophageal squamous cell carcinoma. Liu, N. et al. CADM2 inhibits human glioma proliferation, migration and invasion. Smith, Z. D. et al. Epigenetic restriction of extraembryonic lineages mirrors the somatic transition to cancer. Ladd-Acosta, C. et al. Placenta DNA methylation at ZNF300 is associated with fetal sex and placental morphology. Hannon, E. et al. Leveraging DNA-methylation quantitative-trait loci to characterize the relationship between methylomic variation, gene expression, and complex traits. Zhang, J., Simonti, C. N. & Capra, J. A. Genome-wide maps of distal gene regulatory enhancers active in the human placenta. Owen, D. M. et al. Genome-wide identification of transcriptional enhancers during human placental development and association with function, differentiation, and diseasedagger. Akram, K. M., Kulkarni, N. S., Brook, A., Wyles, M. D. & Anumba, D. O. C. Transcriptomic analysis of the human placenta reveals trophoblast dysfunction and augmented Wnt signalling associated with spontaneous preterm birth. Vangrieken, P. et al. Placental mitochondrial abnormalities in preeclampsia. Ray, S. et al. Hippo signaling cofactor, WWTR1, at the crossroads of human trophoblast progenitor self-renewal and differentiation. Oliva, M. et al. The impact of sex on gene expression across human tissues. Olney, K. C. et al. Sex differences in early and term placenta are conserved in adult tissues. Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. The RNA landscape of the human placenta in health and disease. Melé, M. et al. Human genomics. The human transcriptome across tissues and individuals. Sex differences in gene expression and regulatory networks across 29 human tissues. Kohan-Ghadr, H. R., Armistead, B., Berg, M. & Drewlo, S. Irisin protects the human placenta from oxidative stress and apoptosis via activation of the Akt signaling pathway. Drewlo, S. et al. Irisin induces trophoblast differentiation via AMPK activation in the human placenta. Burton, G. J., Redman, C. W., Roberts, J. M. & Moffett, A. Pre-eclampsia: pathophysiology and clinical implications. Dimas, A. S. et al. Sex-biased genetic effects on gene regulation in humans. Yuan, V. et al. Cell-specific characterization of the placental methylome. Determining cell type abundance and expression from bulk tissues with digital cytometry. Dynamic HIF1A regulation during human placental development. Yue, T. et al. Sex-biased regulatory changes in the placenta of native highlanders contribute to adaptive fetal development. Durand, C. M. et al. Mutations in the gene encoding the synaptic scaffolding protein SHANK3 are associated with autism spectrum disorders. Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Lam, M. et al. Pleiotropic meta-analysis of cognition, education, and schizophrenia differentiates roles of early neurodevelopmental and adult synaptic pathways. Werling, D. M., Parikshak, N. N. & Geschwind, D. H. Gene expression in human brain implicates sexually dimorphic pathways in autism spectrum disorders. Zhu, Y. et al. Placental methylome reveals a 22q13.33 brain regulatory gene locus associated with autism. Barbu, M. C. et al. Epigenome-wide association study of global cortical volumes in generation Scotland: Scottish family health study. Tekola-Ayele, F. et al. DNA methylation loci in placenta associated with birthweight and expression of genes relevant for early development and adult diseases. Wallingford, M. C., Gammill, H. S. & Giachelli, C. M. Slc20a2 deficiency results in fetal growth restriction and placental calcification associated with thickened basement membranes and novel CD13 and lamininα1 expressing cells. Beaumont, R. N. et al. Genome-wide association study of placental weight identifies distinct and shared genetic influences between placental and fetal growth. Hanna, C. W. Placental imprinting: Emerging mechanisms and functions. Thomas, K. N. et al. Maternal background alters the penetrance of growth phenotypes and sex-specific placental adaptation of offspring sired by alcohol-exposed males. & Sferruzzi-Perri, A. N. Loss of imprinting of the Igf2-H19 ICR1 enhances placental endocrine capacity via sex-specific alterations in signalling pathways in the mouse. Murthi, P. & Kalionis, B. Homeobox genes in the human placenta: Twists and turns on the path to find novel targets. Tucci, V., Isles, A. R., Kelsey, G. & Ferguson-Smith, A. C. Genomic imprinting and physiological Processes in Mammals. Monk, D. et al. Limited evolutionary conservation of imprinting in the human placenta. Weksberg, R., Smith, A. C., Squire, J. & Sadowski, P. Beckwith-Wiedemann syndrome demonstrates a role for epigenetic control of normal development. McMinn, J. et al. Unbalanced placental expression of imprinted genes in human intrauterine growth restriction. & Monk, D. Epigenetic Characterization of CDKN1C in Placenta Samples from Non-syndromic Intrauterine Growth Restriction. Cordeiro, A., Neto, A. P., Carvalho, F., Ramalho, C. & Doria, S. Relevance of genomic imprinting in intrauterine human growth expression of CDKN1C, H19, IGF2, KCNQ1 and PHLDA2 imprinted genes. Apostolidou, S. et al. Elevated placental expression of the imprinted PHLDA2 gene is associated with low birth weight. Lambertini, L. et al. Imprinted gene expression in fetal growth and development. Collins, L. L. et al. Growth retardation and abnormal maternal behavior in mice lacking testicular orphan nuclear receptor 4. Targeted inactivation of testicular nuclear orphan receptor 4 delays and disrupts late meiotic prophase and subsequent meiotic divisions of spermatogenesis. Delahaye, F. et al. Genetic variants influence on the placenta regulatory landscape. Expression of stathmin, a microtubule regulatory protein, is associated with the migration and differentiation of cultured early trophoblasts. & Brenner, B. Haemostatic mechanisms in human placenta. Muto, M. et al. Intersection of regulatory pathways controlling hemostasis and hemochorial placentation. & Gris, J. C. Coagulation and placenta-mediated complications. Rambam Maimonides Med. Vincenz, C., Lovett, J. L., Wu, W., Shedden, K. & Strassmann, B. I. Loss of imprinting in human placentas is widespread, coordinated, and predicts birth phenotypes. Pervasive polymorphic imprinted methylation in the human placenta. Ferguson-Smith, A. C. Genomic imprinting: the emergence of an epigenetic paradigm. Henckel, A. et al. Histone methylation is mechanistically linked to DNA methylation at imprinting control regions in mammals. Porcu, E. et al. Limited evidence for blood eQTLs in human sexual dimorphism. Kassam, I. et al. Autosomal genetic control of human gene expression does not differ across the sexes. Yao, C. et al. Sex- and age-interacting eQTLs in human complex diseases. Tekola-Ayele, F. et al. Trans-ethnic meta-analysis of genome-wide association studies identifies maternal ITPR1 as a novel locus influencing fetal growth during sensitive periods in pregnancy. Tekola-Ayele, F. et al. Admixture mapping identifies African and Amerindigenous local ancestry loci associated with fetal growth. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Akalin, A. et al. methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. van Iterson, M., van Zwet, E. W. & Heijmans, B. T. Controlling bias and inflation in epigenome- and transcriptome-wide association studies using the empirical null distribution. Raney, B. J. et al. The UCSC Genome Browser database: 2024 update. Nucleic Acids Res. A. Matrix eQTL: ultra fast eQTL analysis via large matrix operations. Stegle, O., Parts, L., Piipari, M., Winn, J. & Durbin, R. Using probabilistic estimation of expression residuals (PEER) to obtain increased power and interpretability of gene expression analyses. Campbell, K. A. et al. Placental cell type deconvolution reveals that cell proportions drive preeclampsia gene expression differences. Zhang, T., Klein, A., Sang, J., Choi, J. & Brown, K. M. ezQTL: A Web Platform for Interactive Visualization and Colocalization of QTLs and GWAS Loci. Battram, T. et al. The EWAS Catalog: a database of epigenome-wide association studies. Wellcome Open Res. Li, M. et al. EWAS Atlas: a curated knowledgebase of epigenome-wide association studies. Nucleic Acids Res. Sollis, E. et al. The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. Nucleic Acids Res. Kumar, S., Ambrosini, G. & Bucher, P. SNP2TFBS - a database of regulatory SNPs affecting predicted transcription factor binding site affinity. Nucleic Acids Res. A. et al. JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles. Nucleic Acids Res. Breeze, C. E. et al. eFORGE: A tool for identifying cell type-specific signal in epigenomic Data. Watanabe, K., Taskesen, E., van Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. This work was supported by the Intramural Research Program of the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD), National Institutes of Health (NIH) including American Recovery and Reinvestment Act funding via contract numbers HHSN275200800013C; HHSN275200800002I; HHSN27500006; HHSN275200800003IC; HHSN275200800014C; HHSN275200800012C; HHSN275200800028C; HHSN275201000009C and HHSN27500008. Additional support was obtained from the NIH Office of the Director, the National Institute on Minority Health and Health Disparities (NIMHD), and the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK). RICHS is partially supported by NIH-NIEHS R01ES022223 (C.J.M. The authors acknowledge the research teams at all participating clinical centers for the NICHD Fetal Growth Studies, including Christina Care Health Systems, Columbia University, Fountain Valley Hospital, California, Long Beach Memorial Medical Center, New York Hospital, Queens, Northwestern University, University of Alabama at Birmingham, University of California, Irvine, Medical University of South Carolina, Saint Peters University Hospital, Tufts University, and Women and Infants Hospital of Rhode Island. Genotyping was performed in the Department of Laboratory Medicine and Pathology, University of Minnesota. The authors also acknowledge C-TASC and The EMMES Corporations in providing data and imaging support. The authors are also thankful to the RICHS study participants for their participation, and the study staff at Women and Infants Hospital for their dedication to the project. This work utilized the computational resources of the NIH HPC Biowulf cluster (http://hpc.nih.gov). Division of Population Health Research, Division of Intramural Research, Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Bethesda, MD, USA Fasil Tekola-Ayele, Tesfa Dejenie Habtewold, Prabhavi Wijesiriwardhana & Marion Ouidir Glotech, Inc., contractor for Division of Population Health Research, Division of Intramural Research, Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Bethesda, MD, USA Gangarosa Department of Environmental Health, Rollins School of Public Health of Emory University, Atlanta, GA, USA Amber Burt & Carmen J. Marsit University of Grenoble Alpes, Inserm, Team of Environmental Epidemiology applied to Reproduction and Respiratory Health, Institute for Advanced Biosciences, Grenoble, France Department of Obstetrics and Gynecology, Columbia University, New York, NY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed this study and wrote the draft manuscript. performed statistical analyses and visualizations. contributed to data analysis. contributed to data analysis and write-up. contributed data and samples. interpreted the results, reviewed the draft manuscript, provided critical intellectual content, and approved the final manuscript. Correspondence to Fasil Tekola-Ayele. The authors declare that they have no competing interests. Nature Communications thanks Ionel Sandovici, Jennifer Frost, and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Sex-differentiated placental methylation and gene expression regulation has implications for neonatal traits and adult diseases. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250430142601.htm'>Increasing physical activity in middle age may protect against Alzheimer's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 01:50:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An increase in physical activity between the ages of 45 and 65 could help prevent Alzheimer's disease, while inactivity may be detrimental to brain health. This is the main conclusion of a scientific paper published in Alzheimer's & Dementia, which highlights the need to promote physical activity among middle-aged adults. It is estimated that 13% of Alzheimer's disease cases worldwide can be attributed to physical inactivity. In fact, the World Health Organization (WHO) recommends 150 to 300 minutes of moderate activity per week or 75 to 150 minutes of vigorous activity per week. While extensive research has shown that exercise reduces the risk of Alzheimer's disease by improving cardiovascular and mental health, recent studies suggest that physical activity may have a direct impact on the development of brain pathology associated with the disease. "We used physical activity questionnaires to assess changes in activity over a four-year period and neuroimaging tests to analyse the effects of exercise on brain structure and function," she adds. Participants were classified as adherent (meeting WHO recommendations), non-adherent (doing less than the recommended amount of physical activity), and sedentary (doing zero minutes of physical activity per week). Beta-amyloid (Aβ) is a protein that can impair neural communication when it accumulates in the brain and is considered the first pathological event in Alzheimer's disease. Participants who increased their physical activity to meet WHO-recommended levels showed less beta-amyloid accumulation than those who remained sedentary or reduced their physical activity. Non-sedentary participants also showed a greater cortical thickness in brain regions associated with Alzheimer's disease. Cortical thickness in the medial temporal area is crucial for memory, so its thinning or atrophy (loss of volume) is an early sign of neurodegeneration. "Even those who did less physical activity than recommended had greater cortical thickness than sedentary people, suggesting that any amount of exercise, no matter how minimal, has health benefits," explains Müge Akıncı. "These findings reinforce the importance of promoting physical activity in middle age as a public health strategy for Alzheimer's prevention," emphasises Eider Arenaza-Urquijo, ISGlobal researcher and lead investigator of the study. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250430142346.htm'>A virtual reality game integrating smell to fight cognitive decline</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 01:48:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>As the global population ages, supporting older adults in maintaining their cognitive and memory functions has become a pressing concern. One promising strategy to counter cognitive decline is through olfactory stimulation -- engaging the sense of smell. Smell signals travel directly to brain regions involved in memory and emotion. Building on this knowledge, a joint research team from Institute of Science Tokyo (Science Tokyo), University of the Arts London, Bunkyo Gakuin University, and Hosei University, Japan, has developed the world's first cognitive training method for older adults by combining olfactory stimulation with virtual reality (VR). By combining goal-oriented tasks with real-time feedback, our VR-based olfactory training approach can increase cognitive engagement and maximize its therapeutic impact," says Professor Takamichi Nakamoto from Science Tokyo. The method involves an olfactory display that emits specific scents during immersive VR gameplay, activating memory- and emotion-related brain regions. In the activity, participants are asked to memorize and later match scents within a virtual environment. Participants then explore the virtual landscape to locate a scent source. Upon reaching the odor source, shown as a stone lantern, they encounter three colored vapor clouds, each emitting a different scent. The final odor comparison phase engages olfactory discrimination and working memory retrieval, reinforcing cognitive function," explains Nakamoto. After just 20 minutes of playing the VR game, participants showed improvements in visuospatial rotation and memory. Visuospatial processing and cognitive function were assessed through different tasks. In the Hiragana Rotation Task, where they had to decide if rotated Japanese characters matched the original, scores improved from 19-82 to 29-85. In a word-based spatial memory recall task, where participants memorized word positions in a grid, scores rose from 0-15 to 3-15. With continued research and development toward more affordable olfactory displays or alternate scent delivery methods, olfactory-based VR activities could become an accessible and engaging tool for supporting mental health in older adults. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250430142233.htm'>Landmark experiment sheds new light on the origins of consciousness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 01:48:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An experiment seven years in the making has uncovered new insights into the nature of consciousness and challenges two prominent, competing scientific theories: Integrated Information Theory (IIT) and Global Neuronal Workspace Theory (GNWT). The two competing theories were tested against one another in 2019 in a collaborative experiment involving 256 human subjects, and the findings were just released. "Unravelling this mystery is the passion of my entire life." This discovery has implications for how we understand consciousness and may shed light on disorders of consciousness such as comas or vegetative states. Identifying where consciousness comes from could help detect "covert consciousness" in unresponsive patients with severe injuries -- a condition known to occur in about one-quarter of cases as reported in the New England Journal of Medicine last year. Integrated Information Theory (IIT) says consciousness comes from the interaction and cooperation of various parts of the brain as they work together to integrate information, like teamwork. It arises from how these parts are connected and how they share information with each other rather than any one individual area or part of the brain generating consciousness. "It was clear that no single experiment would decisively refute either theory. The theories are just too different in their assumptions and explanatory goals, and the available experimental methods too coarse, to enable one theory to conclusively win out over another," said Anil Seth, Ph.D., a professor of cognitive and computational neuroscience at the University of Sussex. "Having said all this, the findings of the collaboration remain extremely valuable -- much has been learned about both theories and about where and when in the brain information about visual experience can be decoded from. The study involved 256 subjects, which is unprecedented for this kind of experiment. Researchers showed them various visual stimuli and then used three common human brain measurement tools that track blood flow as well as magnetic and electrical activity to study their brains while they looked at the stimuli. This innovative approach brought together researchers with differing perspectives to test two theories in a collaborative, yet critical, environment aimed at reducing confirmation bias and accelerating scientific progress. "Adversarial collaborations are a powerful social process, little used because of its challenging nature, within any field that has competing theories," said Koch. "The bio-medical field could hugely profit by 'friendly' competition among theories -- neurobiological or others. But it requires a great deal of cooperation and work." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250430141835.htm'>'Scratching' more than the ocean's surface to map global microplastic movement</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-01 01:38:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Tiny fragments called microplastics -- ranging from 1 micron to 5 millimeters -- make up the vast majority of plastic pieces found and pose serious risks to ocean health. Most research has focused on surface waters, usually sampling just the top 15 to 50 centimeters using net tows. A researcher from Florida Atlantic University is among an international team of scientists who has moved beyond just "scratching the surface," marking a turning point in our understanding of how microplastics move through and impact the global ocean. For the study, researchers synthesized depth-profile data from 1,885 stations collected between 2014 and 2024 to map microplastic distribution patterns by size and polymer type, while also evaluating potential transport mechanisms. Ranging from a few to thousands of particles per cubic meter, their size determines how they move: smaller microplastics (1 to 100 micrometers) spread more evenly and penetrate deeper, while larger ones (100 to 5,000 micrometers) concentrate near the surface, especially within the top 100 meters of gyres. Gyres act like massive, slow-moving whirlpools that trap and concentrate floating debris -- especially plastic. Strikingly, microplastics are becoming a measurable part of the ocean's carbon cycle, making up just 0.1% of carbon particles at 30 meters but rising to 5% at 2,000 meters. This suggests that microplastics are not only persistent pollutants but may also be altering key biogeochemical processes in the deep sea. Researchers identified more than 56 types of plastic polymers in their synthesized microplastic dataset. Dense polymers become brittle and break down faster, particularly after prolonged exposure to environmental weathering. These small, persistent particles -- often originating from fishing gear and containers like polyester bottles -- can remain in the ocean for decades. Polypropylene, commonly found in items like yogurt containers and rope, photodegrades more quickly than polyethylene, which is used in plastic bags and water bottles. This may account for its lower abundance in offshore waters. Nonetheless, significant uncertainties remain in subsurface microplastic data due to inconsistent sampling techniques and limited coverage, highlighting the need for specialized equipment and greater collaboration to improve data reliability. As microplastics move through this vast space, they interact with natural particles and processes, potentially affecting how the ocean functions. "Our findings suggest microplastics are becoming a measurable part of the ocean's carbon cycle, with potential consequences for climate regulation and marine food webs," said Mincer. Several authors were FAU researchers in the Mincer lab including Luisa Galgani, Ph.D., who served as a Marie Curie Postdoctoral Fellow and is now faculty at the University of Siena; Ryan Bos, Ph.D., who was a Ph.D. student in the integrative biology program at FAU and is now a postdoc at Harvard; and Shiye Zhao, Ph.D., the lead author, who was an FAU postdoc for several years and is now tenured faculty at the Japan Agency for Marine-Earth Science and Technology. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            