
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/a-make-america-healthy-again-report-goes-easy-on-the-food-industry/'>A ‘Make America Healthy Again' Report Goes Easy on the Food Industry</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A ‘Make America Healthy Again' Report Goes Easy on the Food Industry Criticisms of the food industry have gone mute in a second White House report on improving childhood health—where they have been replaced by a lengthy list of research questions on the problem and calls for deregulating small farms. Released on September 9, this “Make Our Children Healthy Again Strategy” report comes from a commission that has been led by the U.S. Department of Health and Human Services' secretary Robert F. Kennedy, Jr., and has included White House officials and other agency heads. It cites poor diet, chemicals, inactivity and “overmedicalization” to explain a decline in U.S. children's health. In its introduction, the 20-page report says that it “outlines a strategic approach for executive actions to address the childhood chronic disease crisis through advancing research, realigning incentives, increasing public awareness, and fostering private sector collaborations.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Initially, the report offers a litany of 23 suggested research programs, ranging from studies on the effects of electromagnetic radiation and on the use of artificial intelligence for detecting cancer in young people to a research program on vaccine injuries. It also includes a list of some 35 policy changes, ranging from developing “MAHA Boxes” of food for participants of the Supplemental Nutrition Assistance Program (SNAP) to improving “regulatory processes” for over-the-counter sunscreen, all described in a few sentences. It then calls for the deregulation of whole-milk sales in schools, mobile grocery stores and small farms, among other agency actions. Finally, it calls for public awareness campaigns on childhood health, better exercise and nutrition, among other topics. “The report has a lot of ideas for actions that really could improve health but is short on specifics and weak on regulatory action,” says New York University nutrition expert Marion Nestle. The first MAHA report similarly blamed increasing U.S. childhood obesity, diabetes and autism rates—as well as other ailments—on unhealthy foods, chemicals, excessive screen time and overmedication. Notably, it blamed the food, chemical-manufacturing and pharmaceutical industries for these problems. It also initially backed its claims with citations to multiple nonexistent studies and frequently misstated real study findings (in a manner redolent of artificial intelligence authorship). “Discovery of the fake citations also came just days after President Donald Trump unveiled an executive order that called for ‘Restoring Gold Science Standards' to government activities,” Science magazine noted in May. The new report takes a different tack, describing current pesticide review standards as “robust” and calling for further study of “overprescription trends” in medicine. The September report also calls for improving the application process for farm-to-school grants without noting that, for this year, the Trump administration canceled such grants and eliminated a program to provide schools with food from local farms. “Scientifically, I think this report will soon be forgotten,” he adds. Ironically, the report echoes a 2010 effort led by then first lady Michelle Obama, Nestle notes. That campaign was, she adds, “greeted by outrage and disdain for ‘nanny-statism.' If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/math/a66026240/new-quantum-pi-formula-revolutionizes-mathematics/'>A New Formula for Pi Is Here. And It's Pushing Scientific Boundaries.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 16:31:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This breakthrough method optimizes complex computations like never before. But what does that mean, and why do we need different representations of pi? Because quantum mechanics looks at the tiniest particles, one at a time, even simple questions can have complex answers that require massive computing power. Rendering high-tech video games and movies like Avatar can take days or more, and that's still not at the level of reality. In this paper, published in the peer-reviewed journal Physical Review Letters, physicists Arnab Priya Saha and Aninda Sinha describe their new version of a quantum model that reduces complexity, but maintains accuracy. Think of the way early internet video buffered in chunks of similar colors, or how classic animators painted static bodies with individual moving parts on top. As detailed in their paper, Saha and Sinha combined two existing ideas from math and science: the Feynman diagram of particle scattering and the Euler beta function for scattering in string theory. Series can end up generalizing into overall equations or expressions, but they don't have to. And while some series diverge—meaning that the terms continue to alternate away from each other—others converge on one approximate, concrete result. The digits of pi extend into infinity, and pi is itself an irrational number, meaning it can't be truly represented by an integer fraction (the one we often learn in school, 22/7, is not very accurate by 2024 standards). But it can be represented pretty quickly and well by a series. That's because a series can continue to build out values well into the tiniest digits. If a mathematician compiles a series' terms, they can use the resulting abstraction to do math that isn't possible with an approximation of pi that's cut off at 10 digits by a standard desk calculator. “In the early 1970s,” Sinha said in a statement from the Indian Institute of Science, “scientists briefly examined this line of research, but quickly abandoned it since it was too complicated.” But math analysis like this has come a long way since the 1970s. They're able to build a sequence and see that it converges on the value of pi within far fewer terms than expected, making it easier for scientists to run the series and then use that for further work. It's a comment on the ongoing and collaborative nature of math theory, even when what results is a working model that might help scientists. “Doing this kind of work, although it may not see an immediate application in daily life, gives the pure pleasure of doing theory for the sake of doing it,” Sinha said in the statement. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. Experts Found First Branch On Animal Tree of Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a66025391/what-the-fourth-dimension-looks-like-where-it-is/'>Scientists Say the Fourth Dimension Could Be Hiding Right in Front of You</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 15:49:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Reality may be bigger than length, width, and height combined. String theorists believe our world encompasses more than three dimensions. Without experiential evidence, the mathematical theory of space and time as a fourth dimension has remained just that since the days of Albert Einstein: a theory. In the clip, Greene says that while “math suggests a real possibility there may be more dimensions than the ones we directly experience,” our minds have trouble comprehending just where they could be. Already, we move through three dimensions—left-right, back-forth, up-down—so how is there any room left? “They are new places that our experience doesn't allow us to access directly, but according to these theoretical ideas, might be there.” From far away, this garden hose may look one-dimensional to the naked eye. They could be so big we see them with our eyes, or they could curl up and become tiny and more difficult to detect, as is the case with the garden hose. “This idea might apply to space itself,” Greene says. “But just as the hose has a curled-up dimension, maybe space itself has curled up dimensions all around us, just curled up to such a fantastically small size we can't see them with our eyes,” he says. “We can't see them even with today's most powerful microscopes.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Found First Branch On Animal Tree of Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63011-2'>Cannabis impacts female fertility as evidenced by an in vitro investigation and a case-control study</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 15:36:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 8185 (2025) Cite this article Cannabis consumption and legalization is increasing globally, raising concerns about its impact on fertility. In humans, we previously demonstrated that tetrahydrocannabinol (THC) and its metabolites reach the ovarian follicle. An extensive body of literature describes THC's impact on sperm, however no such studies have determined its effects on the oocyte. Herein, we investigate the impact of THC on human female fertility through both a clinical and in vitro analysis. In a case-control study, we show that follicular fluid THC concentration is positively correlated with oocyte maturation and THC-positive patients exhibit significantly lower embryo euploid rates than their matched controls. In vitro, we observe a similar, but non-significant, increased oocyte maturation rate following THC exposure and altered expression of key genes implicated in extracellular matrix remodeling, inflammation, and chromosome segregation. Furthermore, THC induces oocyte chromosome segregation errors and increases abnormal spindle morphology. Finally, this study highlights potential risks associated with cannabis use for female fertility. Cannabis consumption for both medicinal and recreational use and legalization have been rising globally1. Cannabis contains several classes of chemicals with cannabinoids being the most prominent; among these, tetrahydrocannabinol (THC) is the primary psychoactive compound and the most studied2. Notably, the concentration of THC in cannabis products has increased significantly, from an average of 3% (by weight) in the 1980s to around 15% in 2020, with some strains reaching 30% of THC2. The increase in frequency, ease of availability, and escalation in potency raises concerns about broader impacts on global human health, including reproductive health. Indeed, the main apprehension regarding THC and reproductive health stems from the importance of the endocannabinoid system in human reproduction3. Endocannabinoids, including N-arachidonoylethanolamide and 2-arachidonoylglycerol, are endogenous cannabinoids that play a central role in both male and female reproduction3, whereas THC is an exogenous cannabinoid. Extensive research has documented the effects of THC on male reproduction, highlighting an impact on sperm deoxyribonucleic acid (DNA) methylation4,5,6,7 and sperm parameters8 including sperm concentration9,10,11, morphology12,13,14 and motility14. As for female health, literature reports the impact of cannabis use during pregnancy on pregnancy outcomes15,16,17,18, placental development18,19,20 and offspring health18,20,21,22. However, to our knowledge, no studies have investigated the impact of cannabis on the human female gamete, the oocyte, a gap partly due to the challenge associated with obtaining these samples. During in vitro fertilization (IVF) treatment, exogenous gonadotropins are administered in a process called “controlled ovarian hyperstimulation” which recruits multiple follicles and induces follicle growth. These recruited follicles, each containing an oocyte, are then collected by a physician in a procedure called oocyte retrieval. Oocytes are collected along with their surrounding microenvironment, including follicular fluid (FF) and supportive somatic cells (granulosa cells). The oocytes are isolated, and mature oocytes are used for subsequent in vitro fertilization. Using FF, our group has previously quantified Δ9-THC and its metabolites, 11-OH-THC and 11-COOH-THC23,24, demonstrating that these compounds could reach the follicular niche. This is significant as it suggests that THC may directly alter the microenvironment where the oocyte matures. Furthermore, our group has shown that THC exposure altered human granulosa cell methylation in a concentration dependent manner23, and in vitro exposure modulated cannabinoid receptor dynamics in granulosa cells24. However, no human studies and only a few animal model studies have investigated the impact of cannabis directly on oocyte development with conflicting results25,26,27,28,29. Maturation of the oocyte is a unique and highly specialized process beginning in utero during fetal development. It is widely accepted that female neonates are born with a finite number of oocytes, which, following menarche, are recruited to mature in cohorts with each menstrual cycle30. Although oocytes are protected in the ovary by the blood-follicle-barrier, they remain highly sensitive to environmental factors31. Given their essential role in reproduction, any perturbations in their development and maturation could have profound effects on fertility and on future generations. Thus, understanding the impact of THC on oocyte health is critical for providing informed guidance and counseling to patients of the potential risks to their fertility and future offspring. In this study, we determine the impact of physiologically relevant concentrations of THC on oocyte maturation, elucidate the transcriptomic changes induced by THC exposure and its effect on chromosome segregation, and compare our findings with a retrospective cohort study. Our investigation will aid in bridging the knowledge gap in our understanding of the sex-specific reproductive consequences of cannabis use and contribute to more effective and evidence-based patient counseling. Using a retrospective case-control design and mass spectrometry, we quantified the concentration of Δ9-THC and its metabolites,11-OH-THC and 11-COOH-THC, in the FF of patients undergoing IVF treatment to determine the reproductive consequences of THC consumption. Figure 1a illustrates the proportion of THC and its metabolites measured in all samples (n = 1059). Positivity rate was defined by the presence of 11-COOH-THC in the follicular fluid (62/1059, 6%). 11-COOH-THC was found alone in 13% of the samples (8/62) while Δ9-THC was co-detected in 37% of the samples (23/62) and 11-OH-THC co-detected in 2% (1/62). All three compounds were measured in 48% of the samples (30/62). Among the positive patients, 73% did not disclose their THC consumption on the patient intake questionnaire. The distribution of Δ9-THC and its metabolites showed a predominance of 11-COOH-THC (mean = 28.8 ng/mL), followed by Δ9-THC (mean = 7.5 ng/mL), with 11-OH-THC being the least abundant (mean = 1.7 ng/mL) (Fig. Notably, concentrations of these metabolites did not differ between FF and matched serum samples obtained at the time of oocyte retrieval (Fig. a Proportion of follicular fluid (FF) samples positive for Δ9-THC, 11-OH-THC and 11-COOH-THC (presence of 11-COOH-THC=positive sample). b Distribution of Δ9-THC (n = 53), 11-OH-THC (n = 31), and 11-COOH-THC (n = 62) concentrations in FF (data presented as median with interquartile range) and c concentrations of Δ9-THC, 11-OH-THC, and 11-COOH-THC in FF and matched serum samples (data presented as mean ± standard deviation, from n = 3 individual participants with matched serum samples (dotted fill pattern) and FF samples (plain)). d Correlation matrix of clinical and biochemical parameters in THC-Positive group (n = 62). The colors represent the two-sided Spearman ρ value *p < 0.05, **p < 0.01, ***p < 0.001. AMH Anti-Müllerian Hormone, BMI Body Mass Index, LH Luteinizing Hormone, FSH Follicle Stimulating Hormone, E2 Estradiol, GT Gonadotropins, THC Tetrahydrocannabinol, Mat. Rate Oocyte maturation rate, Fert. Source data are provided as a Source Data file. A Spearman correlation analysis identified significant correlations between THC metabolite concentrations and various clinical and biochemical parameters (Fig. Specifically, concentrations of Δ9-THC, 11-OH-THC and 11-COOH-THC were positively correlated with oocyte maturation rate in the THC-positive group (Δ9-THC: ⍴ = 0.370, p = 0.003; 11-OH-THC: ⍴ = 0.309, p = 0.014 and 11-COOH-THC: ⍴ = 0.295, p = 0.020). Interestingly, Δ9-THC levels were negatively correlated with a patient's Body Mass Index (BMI) (⍴ = −0.539, p = 0.000053). Patients undergoing IVF treatment and oocyte retrieval who consented for the collection of IVF waste material (immature oocytes, somatic cells and FF) and de-identified clinical data were included in this study. For each patient, a minimum of three immature oocytes at the germinal vesicle (GV) stage were collected following the removal of somatic cells. GV oocytes were cultured using our standard in vitro maturation (IVM) protocol for 24h32 (control group (Ctrl), n = 96) or with the addition of THC (treatment groups). Oocytes were treated with either a physiologically relevant (THC1, n = 95, 25 ng/mL Δ9-THC, 5 ng/mL 11-OH-THC, 50 ng/mL 11-COOH-THC) or a supraphysiologic (THC2, n = 93, 100 ng/mL Δ9-THC, 50 ng/mL 11-OH-THC, 200 ng/mL 11-COOH-THC) concentration where THC1 is based on the concentration of Δ9-THC and its metabolites measured in the follicular fluid of IVF patients and THC2 is based on previously reported concentrations in animal studies23,25,29. Subsequently, oocytes were classified based on their progression through key maturation checkpoints: germinal vesicle (GV) and Metaphase-I (MI) (after germinal vesicle breakdown (GVBD) and before polar body extrusion) were considered immature oocytes, while Metaphase-II (MII) oocytes (after visible polar body extrusion) were considered mature (Fig. Maturation rate was then calculated per treatment group. a Experimental design and in vitro oocyte maturation. b Maturation rate as the percentage of germinal vesicle (GV) oocytes that matured (progressed to metaphase-II (MII)) rate (Ctrl (44/96, THC1: 49/95, THC2: 54/93). Oocyte diameter (c) prior to exposure to THC (Ctrl: n = 92, THC1: n = 89, THC2: n = 85) and d after 24 h of culture (Ctrl: n = 91, THC1: n = 88, THC2: n = 83). Proportion of oocytes that underwent key maturation events after 24 h of culture: e germinal vesicle breakdown (GVBD) (Ctrl: n = 71, THC1: n = 72, THC2: n = 64) and f extrusion of the first polar body (MII-arrested stage) (Ctrl: n = 28, THC1: n = 30, THC2: n = 31). Error bars represent the mean ± standard deviation. Significance was assessed by two-sided Fisher's exact test or One-way ANOVA with two-sided Holm–Sidak multiple comparison test (ns not significant). IVF In vitro fertilization, IVM In vitro maturation, MII metaphase II, MI metaphase I, GV germinal vesicle, GVBD germinal vesicle breakdown, FF follicular fluid, THC tetrahydrocannabinol. Source data are provided as a Source Data file. Oocytes treated with THC1 showed no significant change in maturation rate (49/95, 52%, p = 0.6704), while THC2 exhibited a non-significant trend toward increased maturation (54/93, 58%, p = 0.1098), compared to Ctrl (44/96, 46%) (Fig. Utilizing timelapse imaging, oocyte morphology assessments were performed pre-IVM (Ctrl: n = 92, THC1: n = 89 and THC2: n = 85) and post-IVM (Ctrl: n = 91, THC1: n = 88, and THC2: n = 83), and key maturation events were recorded: GVBD (Ctrl: n = 71, THC1: n = 72 and THC2: n = 64) and extrusion of the first polar body (Ctrl: n = 28, THC1: n = 30 and THC2: n = 31). Examples of timelapse IVM images are provided in Supplementary Fig. 1 and corresponding videos are provided as Supplementary videos (Ctrl-Supplementary video 1, THC1-Supplementary video 2 and THC2- Supplementary video 3). There were no significant differences in oocyte diameter between treatment groups either before (Ctrl: 110.6 μm, THC1: 109.6 μm, p = 0.2120 and THC2: 109.6 μm, p = 0.2120) (Fig. 2c) or after 24 h of culture (Ctrl: 110.2 μm, THC1: 110.0 μm, p = 0.7416 and THC2: 108.8 μm, p = 0.1066). Similarly, the timing of GVBD (Fig. 2e) and polar body extrusion (Fig. 2f) remained unaffected by THC exposure. Demographic data of patients included in these analyses can be found in Supplementary Information - Supplementary Table 1. Single MII oocytes with good morphology and normal developmental progression were sequenced using our optimized ultra-low input RNA sequencing pipeline33 (n = 24 patients/n = 86 metaphase-II (MII) oocytes (28 Ctrl, 27 THC1 and 31 THC2). Differential expression analysis revealed 89 genes up-regulated and 227 genes down-regulated greater than 2-fold (|log2FC| > 1) and p < 0.05 (Fig. 3a) when assessing the impact of the THC1 vs Ctrl (Supplementary Data 1). Gene Set Enrichment Analysis (GSEA) identified that upregulated genes were principally associated with positive regulation of synaptic transmission, axonemal dynein complex assembly, and glutamate receptor signaling pathway, while the downregulated genes were associated with protein synthesis, expression regulation of SLITS and ROBOS and inflammatory processes (Fig. THC2 exposure induced a greater magnitude of transcriptomic dysregulation, with 402 up-regulated and 62 down-regulated genes identified (Fig. The upregulated genes were associated with the immune system and apoptotic pathways while downregulated genes were associated with attachment of spindle microtubules to kinetochores and inflammatory processes (Fig. As illustrated by the Venn diagram (Fig. 3e), THC1 exposure had 266 specific DEGs, while THC2 exposure had 414, with 50 being common to both treatment groups (gene lists are available in Supplementary Data 4). Of these 50 common DEGs, 32 were protein-coding, 19 were up-regulated (EPYC, RGS5, N4BP2L1, KRT19, PRR20G, KL, KCND3, ALDH3A1, SLC1A3, TSPAN8, PLAU, COL8A2, TFAP2E, SPTSSB, BRINP3, VANGL2, RGS18, RXFP1, and KCNMB3), 10 were down-regulated (OR4F15, MMP9, PRRX2, IRS4, INFG, CCIN, IL33, NEUROD1, MT1HL1, and MT1H), and 3 displayed bidirectional changes (S100B, ACTA1, and ARHGEF19) (Fig. 3f) (Detailed information available in Supplementary Data 5). Demographic data of patients included in these analyses can be found in Supplementary Information - Supplementary Table 2 and Sequencing Quality Control metrics can be found in Supplementary Data 6. Volcano plots of differentially expressed genes (DEGs) comparing (a) THC1 vs Ctrl and b pathway analysis by gene set enrichment analysis (GSEA) THC1 vs Ctrl. Volcano plots of DEGs comparing c THC2 vs Ctrl and d GSEA of the comparison THC2 vs Ctrl. e Venn diagram of DEGs in THC1 and THC2 compared to Ctrl. f Common protein-coding DEGs, colors representing fold change. DEGs were defined by p < 0.05 (twp-sided Wald test resulting in unadjusted p-values) and log2fold change (|log2FC)| > 1, n = 24 patients/n = 86 metaphase-II (MII) oocytes (28 Ctrl, 27 THC1 and 31 THC2). Significant pathways were defined as having a normalized enrichment score (|NES|) > 1.5 and p < 0.05 (two-sided permutation test resulting in unadjusted p-values). Subsets of MII oocytes from both the control and THC-treatment groups were used to assess polar body ploidy status (18 Ctrl, 21 THC1, and 21 THC2) and for spindle morphology (12 Ctrl, 12 THC1, and 12 THC2). Removal of the zona pellucida (ZP) and subsequent polar body biopsy (Supplementary Fig. 2) allowed for some oocytes to be used for ploidy determination by low-pass whole genome Next-Generation Sequencing (NGS) aneuploidy using VeriSeq PGT-A which is specialized in detecting aneuploidy in reproductive samples21,34 (Supplementary Fig. 3) and meiotic spindle organization by confocal microscopy allowing for precise visualization of spindle organization and chromosome alignment (Fig. Both THC1 and THC2 treatment led to a 9% increase in aneuploidy (Ctrl: 39%, THC1 and THC2: 48%, p = 0.7479) (Fig. 4b) and a higher proportion of complex aneuploidy, defined by the gain or loss of more than three chromosomes35 (Ctrl: 0%, THC1 and THC2: 42%, p = 0.1029) (Fig. Figure 4d reports a subset of oocytes where both ploidy status and spindle morphology were assessed (n = 17), without stratifying by treatment group. The majority of oocytes that completed meiosis I displayed normal spindle morphology (euploid: n = 8/13, 62% and aneuploid: n = 3/4, 75%, p > 0.9999) (Fig. 4d), but not all. The hallmark characteristics of “normal” meiotic spindles include bipolar barrel-shaped microtubules with the chromosomes aligned on the metaphase plate36. Whereas “abnormal” spindles are varied in their morphology and may include multipolar spindles, alterations in microtubule organization, and misaligned chromosomes36. Spindle disorganization and chromosome misalignment are shown by representative images in Fig. 4e, where oocytes were classified as having either “normal” or “abnormal” spindles. The proportion of oocytes with abnormal spindles was higher in the THC exposed groups compared to control (Ctrl (5/12), THC1 8/12), and THC2 (11/12), with a significant increase in THC2 (Ctrl: 42% and THC2: 92%, p = 0.0272) (Fig. (Spindle immunostaining negative controls ca be found in Supplementary Fig. a Experimental design for polar body sequencing and oocyte immunostaining. b Proportion of euploid oocytes deduced by polar body biopsy sequencing results in Ctrl (11/18), THC1 (11/21), and THC2 (11/21). c Proportion of aneuploid oocytes with a gain/loss of one chromosome (white), two chromosomes (gray) or three and more (black) in Ctrl (1 chromosome: n = 4, 2 chromosomes: n = 3, ≥3 chromosomes: n = 0, total n = 7), THC1 (1 chromosome: n = 2, 2 chromosomes: n = 4, ≥3 chromosomes: n = 4, total n = 10) and THC2 (1 chromosome: n = 4, 2 chromosomes: n = 2, ≥3 chromosomes: n = 4, total n = 10). d Proportion of normal morphology spindles in euploid (8/13) and aneuploid (3/4) oocytes. e Representative images of normal (n = 12) and abnormal (n = 24) metaphase-II-arrested oocyte spindles, chromosomes (Hoechst)are in blue, spindles (α-tubulin) are in green and cell membrane (Phalloidin) in red. Scale bar: 5 µm. f Proportion of oocytes with abnormal spindles in Ctrl (5/12), THC1 (8/12), and THC2 (11/12, p = 0.0272). Significance was assessed using two-sided Fisher's exact test. GV germinal vesicle, MI metaphase I, MII metaphase II, Oo oocyte, PB polar body, THC tetrahydrocannabinol, WGA whole genome amplification, ZP zona pellucida. Source data are provided as a Source Data file. Following pairwise case-control matching, where each THC-positive sample was matched to two THC-negative samples, a significant decrease in embryo euploidy rate was observed in the THC-positive group (n = 51, 60.0%), compared to the THC-negative group (n = 101, 67.0%, p = 0.0245) (Table 1). There was no significant change in maturation, fertilization and blastocyst rates (Table 1). To further evaluate the likelihood of adverse IVF outcomes, we conducted multiple logistic regression analyses, focusing on clinically relevant IVF outcome thresholds37: maturation rate (80%), fertilization rate (70%), blastocyst rate (50%) and euploidy rate (60%). We utilized backward stepwise logistic regression, including the following covariates: oocyte age, participant body mass index (BMI), anti-müllerian hormone (AMH), day 2/3 luteinizing hormone (LH), and follicle stimulating hormone (FSH), (estradiol) E2 on trigger, and total gonadotropin (GT) dose. The final model for both blastulation and euploidy rates retained THC status as a significant explanatory variable, with oocyte age being a significant covariate. In this pairwise matched cohort, THC positivity significantly decreased the odds of reaching a 50% blastulation rate or above (odds ratio: 0.45, p = 0.018) and the odds of achieving a euploidy rate above 60% (odds ratio: 0.47, p = 0.038) (Table 2). Age was also found to significantly impact blastulation and euploidy rates, with an odds ratio of 0.9144 (p = 0.0010) and 0.9150 (p = 0.0024), respectively. The models for predicting blastulation rate (>50%) and euploidy rate (>60%) demonstrated positive predictive power, with areas under the curve (AUCs) of 0.68 and 0.67, respectively (Supplementary Fig. Understanding the impact of environmental factors and lifestyle choices on female fertility is crucial for proper patient counseling. With cannabis being one of the most commonly used recreational drugs in the world1, it is critical to holistically evaluate its impact on mental and general health, in addition to reproductive health. This study, using donated human oocytes and an integrated multidisciplinary approach, reveals that exposure to THC affects oocyte maturation, transcriptome, and induces meiotic chromosomal imbalances associated with altered spindle morphology. Moreover, our retrospective analysis revealed that exposure to THC was associated with significantly lower embryo euploidy rate, likely partially explained by an altered chromosomal organization as demonstrated in the in vitro matured MII oocytes. In our retrospective study, we measured THC concentrations in 1059 follicular fluid samples from patients undergoing IVF treatment at CReATe Fertility Centre (Toronto, Canada) in a retrospective matched case-control cohort. Sixty-two samples tested positive for 11-COOH-THC, resulting in a 6% positivity rate (Fig. This rate is considerably lower than what was reported by a recent Health Canada survey where 23% of females reported recreational cannabis consumption within 1 year of being surveyed38. However, these patients were counseled pre-treatment not to use recreational drugs while undergoing IVF. The relative concentrations of Δ9-THC, 11-OH-THC, and 11-COOH-THC are consistent with metabolism of THC in the liver (Fig. Δ9-THC (half-life 1.3–10 days) is metabolized to 11-OH-THC (half-life 20 min–2 h) and 11-OH-THC is rapidly metabolized to 11-COOH-THC (half-life 3–5 days), which remains in the circulation for up to 30 days39. The consistent concentrations of THC metabolites in both the follicular fluid and serum suggests passive diffusion or transudation from the bloodstream into follicular fluid rather than active transport into or out of the follicle (Fig. In a clinical IVF setting, embryologists and physicians assess mature oocytes based on the extrusion of the first polar body30,40. This process, known as nuclear maturation, marks the oocyte's progression to metaphase-II and is considered the final stage of oocyte maturation30,41. When we compiled the IVF outcomes of our retrospective cohort, we observed a positive correlation between THC metabolites and oocyte maturation (Fig. This suggests that the levels of THC and its metabolites present in this cohort may support nuclear maturation through an unknown mechanism. Parallel to our clinical retrospective study, we treated pairwise-matched immature oocytes from patients who were negative for THC (naive GVs) with THC in vitro and we observed an increased proportion of oocytes which achieved the MII stage (Fig. Given that factors such as oocyte diameter have been previously reported to influence nuclear maturation42, we confirmed that there were no differences in oocyte size distribution across the three groups (Ctrl, THC1 and THC2) before and after culture for 24 h (Fig. Finally, we also monitored and recorded the timing of key maturation events using timelapse imaging technology and showed that the GVBD occurred slightly faster in THC treated oocytes (Fig. 2e), but this did not reach significance, and there were no differences in the timing of the first polar body extrusion between THC-exposed and Ctrl groups (Fig. Previous animal studies investigated the impact of THC on oocyte maturation using bovine oocytes29, which is considered a better proxy to human oocyte maturation when compared to murine model due to similar temporal dynamics. López-Cardona et al. reported a 15% increase in maturation rate after treating oocytes with 0.1 µM (31.45 ng/mL) Δ9-THC for 12 h (n = 30/group)29. In contrast, a more recent and larger study (n = 164/group) concluded that a 24 h treatment with ‘recreational cannabis doses' of 0.32 µM (100.63 ng/mL) and 3.2 µM (1,006.30 ng/mL) of Δ9-THC significantly reduced oocyte maturation from 80.1% to 65.3% and 60.1%, respectively25. Of note, the latter study used higher doses of Δ9-THC than what is measured in the FF of our patients but neither study examined the combined effects of Δ9-THC and its metabolites, which might alter its effect on the growing oocyte. Collectively, our results suggest that THC exposure, at both physiological and supraphysiological concentrations, appears to accelerate oocyte maturation speed and completion, consistent with results from the Lόpez-Cardona bovine study29 and previous mouse studies26. Oocytes must not only successfully progress through meiosis II to reach metaphase-II but also need sufficient time to reach cytoplasmic maturity to support early embryo development30,41. This process involves the precise and faithful packaging of various components, including maternal mRNA transcripts. mRNA production and storage in the ooplasm is critical, since, following establishment of the germinal vesicle, chromatin is condensed, and transcription is halted43,44. These transcripts are critical not only for meiosis resumption and fertilization, but also during the first 3 days of embryonic development45, since the early embryo remains transcriptionally silent, relying entirely on maternally inherited mRNA from the ooplasm to drive cellular processes prior to embryonic genome activation46. The selection and enrichment of critical transcripts in the ooplasm have been described in the literature and are regulated by post-transcriptional mechanisms. These complex processes involve a sophisticated network of RNA binding proteins (RBPs), polyadenylation factors and RNA translational and degradation machinery, which all regulate the storage, translation, and degradation of oocyte mRNAs46. To understand how THC affects stored maternal mRNA during oocyte maturation, we identified genes associated with transcripts that were differentially expressed following THC exposure (Fig. Focusing on the common protein-coding transcripts, 32 genes were identified and grouped into nine principal functions: G protein-coupled receptor (GPCR) signaling, extracellular matrix regulation, embryogenesis, cell-cell communication, inflammation, cytoskeleton, detoxification, transcription factor and ion channels (Supplementary Data 5). Among these, MMP9 was significantly downregulated in both THC treatment groups. MMP9 encodes for matrix metalloproteinase 9 (MMP-9) essential for local proteolysis of the extracellular matrix and leukocyte migration47,48,49. In animal models, MMP-9 has well-established role in ovulation and follicle rupture50,51,52, and in mice, protein expression in blastocyst and early embryo is critical for implantation53,54,55,56,57,58. In humans, several studies have investigated its role in implantation using various trophoblast and implantation models59,60,61. Dysregulation of its expression and activity is associated with pregnancy complications and recurrent pregnancy loss62,63,64,65. Moreover, THC has also been shown to decrease MMP-9 expression in human amniotic epithelium66 and endothelial cancer cells67. Thus, downregulation of MMP9 in the ooplasm may negatively contribute to key ovulation events needed for fertilization, embryo development, and implantation. G-protein coupled receptor (GPCR) signaling was also dysregulated following exposure to THC. GPCR signaling is crucial for oocyte growth and maturation, and many GPCRs are present at the oocyte surface, including the cannabinoid receptor 1 (CB1) and 2 (CB2)27,68. Genes coding for Regulators of G protein Signaling (RGS), RGS5 and RGS18, were significantly upregulated following THC treatment in our study. These regulators are known modulate GPCR signal transduction69. RGS5 and RGS18 belong to the RGS R4 family and have been shown to bind with the Gα proteins, thus reducing their inhibitory activity70,71. They have been shown to have a crucial role in the cell processing of the signal coming from the GPCRs72. Although the exact role of RGS5 and RGS18 in oocyte maturation is poorly understood, their upregulation suggests a potential impact on GPCR signaling pathways in response to THC stimulation. Immune pathways were also overrepresented in both treatment groups compared to controls. For instance, IFNG and IL33 were both significantly downregulated in THC exposed groups. It is well established that tight regulation of inflammatory processes is critical for implantation of the embryo in the endometrium73. Interferon-γ (IFNγ) is a cytokine widely known for its role in inflammation and the activation of macrophages74. Although its role during oocyte maturation is unclear, IFNγ secretion by the conceptus is essential for implantation in animal models75. On the other hand, Interleukin-33 (IL-33), a cytokine that belongs to the IL-1 family, binds to the ST2 (IL-1RL1) receptor76. In mice, IL-33 was found to be expressed in the oocyte and ST2 in granulosa cells and the uterus77. In humans, altered levels or expression of IL-33 or ST2 in various tissue and sample types were associated with pregnancy complications like preeclampsia78,79,80, preterm birth81,82,83, intrauterine growth restriction84, and miscarriage85. The downregulation of IL-33 we observed in THC-exposed oocytes suggests there may be potential disruption in the inflammatory processes necessary for successful pregnancy. A significant number of DEGs in THC-exposed oocytes are involved in the cytoskeletal function, including KRT19, COL8A2, ACTA1 and ARHGEF19 (Supplementary Data 5). In addition to these transcriptomic changes, we observed significant alterations to cytoskeleton machinery throughout this study. Indeed, the oocyte's cytoskeleton plays a crucial role in chromosome alignment, segregation, and polarity establishment86 and without appropriate formation and regulation of key cytoskeletal functions, the oocyte is vulnerable to chromosomal abnormalities. However, the cytoskeleton-associated DEGs identified in this study have not been previously characterized in oocyte development and thus require further investigation to gain a deeper understanding of the effects of THC on these processes. Taken together, THC exposure seems to impact critical transcripts involved in key oocyte maturation processes, fertilization, early embryo development and implantation. While these transcriptomic alterations likely result from post-transcriptional processes, the specific mechanisms by which THC affects these processes in human oocytes remains unknown46. Given the observed increase in nuclear maturation rate and the altered transcriptomic profiles related to chromosome organization, we next investigated the impact of THC on chromosome segregation. Indeed, errors in chromosome segregation during the first meiotic division is the most frequent cause of embryo aneuploidy87, making the faithful establishment of chromosome segregation machinery a critical bottleneck in the production of a chromosomally normal embryo88,89. To investigate oocyte ploidy, we performed polar body biopsy and low-pass whole genome sequencing. Notably, we observed that THC exposure led to a 9% increase in aneuploidy rates (Fig. Additionally, we observed an increase in the proportion of oocytes with complex aneuploidies (defined as a gain or a loss of 3 or more chromosomes)35 in the THC-treated group compared to controls (Fig. Aneuploidies are associated with implantation failure, miscarriage, and are incompatible with life90. It has been postulated that most aneuploidies arise from errors in maternal meiosis I91,92,93,94,95,96, but our data suggest that meiosis II may also be sensitive to perturbations as 38% of the euploid oocytes had abnormal spindle morphology (Fig. 4d) determining using confocal microscopy. We assessed spindle morphology after 24 h incubation of oocytes with and without THC. A normal spindle configuration is barrel shaped with chromosomes aligned at the metaphase plate, while ‘abnormal' configurations include multipolar spindles and misaligned chromosomes97 (Fig. In this study, we demonstrated a dose-dependent decrease in the proportion of oocytes with normal spindle morphology following THC exposure (Fig. Correct chromosome segregation during oocyte maturation is essential for producing euploid embryos, which have the highest chance of establishing a healthy pregnancy98. To address the primary clinical question regarding THC's impact on IVF outcomes, we used a pairwise case-control matching strategy, where each positive sample was matched to two negative samples based on demographic data, and we compiled the resulting matched cohort's IVF outcomes (Table 1). THC exposure was associated with a marginal increase in maturation rate (Table 1), concordant with what was obtained in our in vitro experimentations (Fig. Further, a significant decrease in embryo euploidy rate (Table 1) and reduced odds of obtaining a euploidy rate above 60% was observed (Table 2). These results indicate that THC-positive patients may have fewer euploid embryos from their IVF cycle and may experience a longer time to pregnancy. To deepen our understanding of our findings, we must extrapolate what is known about THC interactions and pathways from other cell types. THC primarily elicits its functions through binding the cannabinoid 1 and 2 receptors (CB1 and CB2), which are expressed at all stages of oocyte maturation68. CB1 and CB2 are G protein-coupled receptors (GPCRs) which are capable of inhibiting adenylate cyclase, the enzyme responsible for catalyzing adenosine triphosphate (ATP) to cyclic adenosine 3', 5'-monophosphate (cAMP). Activation of the CB receptors, through stimulation by THC, could thus lead to an inhibition of adenylate cyclase, resulting in lowering ooplasm cAMP levels99. Adenylate cyclase activity and the constant and high production of cAMP is critical to prevent premature meiotic resumption100. Here, we propose a hypothetical model of action of THC wherein THC binds to CB1/2 activating them, which in turn inhibits adenylate cyclase activity, reducing ooplasm cAMP concentration. Releasing the inhibition of meiotic resumption, would then result in premature resumption of meiosis. This untimely and premature resumption may increase the likelihood of aneuploidy arising in the oocyte and resulting embryo due to the premature separation of chromosomes misaligned on the metaphase plate and an asymmetrical division of the chromosomes into the first polar body. This hypothesis aligns with the correlation between THC concentrations and oocyte maturation rate observed in the retrospective cohort (Fig. 1d) and the increased oocyte maturation rate in vitro, as well as the associated reduction in euploid oocytes (Fig. 4b) and euploid embryo rates (Table 1) we observed. Further investigations are underway to dive deeper into THC signaling in the oocyte and better refine this hypothetical model. To conclude, this study comprehensively investigates and demonstrate the impact of THC on the human oocyte. Herein, our findings reveal significant effects on oocyte maturation, transcriptomic profiles, meiotic spindle organization, and oocyte ploidy. Collectively, this data presents compelling evidence that cannabis consumption may negatively impact female fertility. Our integrated and multi-faceted in vitro approach, utilizing multiple techniques and endpoints to assess chromosome segregation, is a major strength of this study. However, it was limited by the usage of immature GV oocytes following ovarian hyperstimulation, which are considered suboptimal for reproductive purposes, since they did not mature following initial stimulation. Furthermore, we acknowledge the importance of patient age on the oocyte ability to mature in vitro, but this study was not statistically powered to analyze results based on patient age. This limitation arose because the majority of GV oocytes were retrieved from patients younger than 37 years old (81%). Also, our study focused on identifying changes in the abundance of the pre-stored transcripts in response to THC exposure, rather than de novo transcription, limiting our ability to speculate on the impact of THC on gene expression before the GV stage. On the other hand, our retrospective cohort objectively measured THC and its metabolites to determine the impact of THC on IVF outcomes, overcoming biases inherent in self-reporting101. Indeed, 73% of our patients positive for THC did not report cannabis use when completing their patient intake questionnaire, potentially due to the persistent stigma of recreational drug consumption. A limitation of our retrospective study is the lack of data on cannabis consumption habits (e.g. frequency, timing, dosage, route of consumption), and our cohort is likely not representative of the general population, as all patients were undergoing IVF for fertility treatment or to altruistically donate their oocytes to intended parents. In addition, FF was not measured for the presence of other drugs, and even though none of the patients reported concomitant use of other drugs, self-reporting alone cannot rule out the exposure of the follicle to these substances. The limitations associated with the retrospective aspect of this study and the other potential contributors (e.g. lifestyle habits) to the observed outcomes are compensated by our in vitro study that used at least three oocytes (one in each exposure group) per patient. Finally, these findings underscore the need for increased awareness and caution among people with ovaries, particularly those undergoing fertility treatments. Our study highlights the importance of informing patients about the potential risks associated with cannabis consumption and provides a basis for regulatory bodies, medical professional societies, and public health organizations to establish recommendations and guidelines regarding cannabis consumption during fertility treatment. All patients undergoing ART procedures were provided with the opportunity to participate in the collection, and future use, of biological waste material for research purposes. Patients were provided with an Independent Review Board (IRB) approved informed consent package containing information regarding the types of material that would be collected following consent as well as examples of projects this material may be used for. Patients did not receive compensation or financial benefit for their participation in the collection of biological waste material. All patients included in this study provided informed consent for the donation of their biological waste material, which included follicular fluid (FF) and immature (GV) oocytes as well as associated de-identified demographic and clinical information, including age, Body Mass Index (BMI), ovarian reserve metrics and treatment regimens (Veritas IRB Approval #16487). To be included in the assessment of tetrahydrocannabinol (THC) on in vitro maturation (IVM), patients must have had 10 or more MII oocytes after stripping and a minimum of 3 GV oocytes in order to have at least one GV oocyte per treatment group. Last, all patients included in the assessment of THC on IVM were confirmed to be negative for THC by LC-MS/MS (described below). Patients were excluded if: they did not meet inclusion criteria, had low oocyte yield (<16 oocytes), low oocyte maturation rate (<62.5%), a previous cycle with poor fertilization rate (<75%) and/or blastulation rate (<40%), severe male factor, advanced maternal age (>40 years old), or who were undergoing fertility preservation (oncofertility and/or social egg freezing). Moreover, if a patient was consented for the donation of their biological waste material but the physician and embryologist deemed rescue-IVM (rIVM) was indicated for their clinical care, no oocytes would be collected for research purposes and patients would be informed of the addition of rIVM to their clinical treatment by the physician or another healthcare professional. All GV oocytes included in this study were collected and underwent IVM between July 2022 and January 2024. The request and use of samples and de-identified demographic and clinical data for this study was approved by Veritas IRB Approval #16518. For the retrospective analysis, FF was collected from all consenting patients undergoing IVF treatment between June 2016 and March 2023. All samples in this study were considered “female” as they are human oocytes which are chromosomally “XX”. Gender, race, ethnicity or other socially relevant groupings were not considered in this study. The purchase, storage and use of Δ9-THC and its metabolites for research purposes was approved by Health Canada and all procedures were conducted in accordance with the ‘Cannabis Act' and ‘Cannabis Regulations' (License #LIC-A4MUR820SB-2020). Measurements of Δ9-THC, 11-OH-THC and 11-COOH-THC in FF were performed for every patient who donated GV oocytes, as previously described, to exclude patients consuming cannabis for the IVM investigation23. For the retrospective study, FF and matched serum were measured using the same procedure. Briefly, proteins were precipitated using methanol (1:1 v/v), and the supernatants were assessed by LC-MS/MS using a QTRAP 5500 (SCIEZ, Concord, CA, USA) and Agilent 1290 HPLC (Agilent, Santa Clara, USA) with a calibration curve (0.001–200 ng) of known amount of the molecules of interest. Samples above the lower limit of quantification were considered positive. Cannabinol (CBN) and Cannabidiol (CBD) were also measured in the samples but were undetectable. GV oocytes were received from consenting patients, randomly split into three groups, and cultured using standard clinical IVM media (SAGE One-step (CooperSurgical, Canada) + 7.5 IU/mL of Menopur (Ferring, Canada)): Ctrl (n = 96, only IVM media), THC1 (n = 95, treated with a physiological concentration of cannabis based on previous measurements of cannabis in FF23: 25 ng/mL Δ9-THC (Sigma-Aldrich, Canada), 5 ng/mL 11-OH-THC (Sigma-Aldrich, Canada), 50 ng/mL 11-COOH-THC (Sigma-Aldrich, Canada)) and THC2 (n = 94, treated with THC at a concentration based on previous animal studies25,27,29: 100 ng/mL Δ9-THC, 50 ng/mL 11-OH-THC, 200 ng/mL 11-COOH-THC). Oocytes were obtained from 24 patients, and their demographic data are reported in the Supplementary Table 1. Images using an inverted bright field microscope were taken prior and following incubation to assess oocyte morphology. Oocytes were cultured for 24 h using the EVOS FL Auto 2 (ThermoFisher, Canada) imaging system or cell culture incubator (5% CO2 and 37 °C). Timelapse images were taken every 15 min and used for the assessment of GVBD and polar body extrusion (Ctrl: Supplementary video 1, THC1: Supplementary video 2 and THC2: Supplementary video 3). After 24 h of culture, oocytes were processed according to the specific endpoint (RNAseq or immunostaining/polar body biopsies). Oocytes destined for single-cell RNASeq were further stripped of any residual cumulus cells and snap-frozen in 0.2 mL tubes in less than 1 µL phosphate-buffered saline (1 X PBS). To reduce inter-sample variability, we selected patients with similar demographics and stimulation parameters (Supplementary Table 2). A total of 86 oocytes from 24 patients were selected for RNASeq, 28 in Ctrl, 27 in THC1 and 31 in THC2. cDNA from single oocytes were synthesized using the SMART-seq v4 Ultra Low Input RNA Kit (Takara Bio Inc., Japan). The amplified cDNA was purified using Agencourt AMPure XP beads (Beckman Coulter, USA) and eluted. The purified cDNA was quantified using the Qubit dsDNA high sensitivity assay (ThermoFisher, Canada) and length and molarity were assessed using the DNF-474 HS NGS Fragment Kit (1-6000 bp) with the Fragment Analyzer 5200 (Agilent Technologies, USA). The amplified cDNA (0.2 ng) was used to construct sequencing libraries using a modified and miniaturized Nextera XT library preparation protocol (Illumina, Canada) developed for the use with the Mosquito HV liquid handling robot (SPT labtech, Boston, USA). The quality of the libraries was assessed using the same Fragment Analyzer kit DNF-474 HS NGS Fragment Kit (1–6000 bp), quantified using Qubit dsDNA high sensitivity assay (ThermoFisher), normalized and pooled. Sequencing was performed on a NovaSeq 6000 S2 flow cell (Illumina, Canada) at Princess Margaret Genomics Centre (2 × 150 bp) (Toronto, Canada). The sequencing quality control metrics are compiled in the Supplementary Data 6. Raw sequencing reads were trimmed based on read quality (Phred > 28) and aligned and quantified to hg38 using STAR (Spliced Transcripts Alignment to a Reference; v2.7.8a)102. Low abundant transcripts were excluded (maximum <20) and normalized using the default normalization method built into DESeq2 (v3.5)103. We conducted differential expression (DE) using DESeq2 comparing THC1 vs. Ctrl and THC2 vs. Ctrl. Significantly differentially expressed genes were defined as p-value  <  0.05 and |log2fold change (FC)| of >1. The complete list of DE genes is available in the Supplementary data 1 (THC1 vs Ctrl) and the Supplementary data 2 (THC2 vs Ctrl)). This analysis was conducted in Partek Flow (version 11.0.23.1004). Gene Set Enrichment Analysis (GSEA)104,105 was conducted to determine what gene sets were impacted by exposure to THC. The resulting pathway list was cross referenced with a custom gene set created and supported by the Bader Lab (University of Toronto) which is comprised of all GO database, KEGG, Reactome, and Wiki pathways gene sets (v2024-01-01) (http://download.baderlab.org/EM_Genesets/)106 (Supplementary Data 3) Significant pathways were defined as having a |Normalized Enrichment Score (NES)| > 1.5 and p-value < 0.05. After 24 h in culture, the zona pellucida (ZP) was removed by incubating (30 s–2 min) with EmbryoMax® Acidic Tyrode's Solution (Millipore Sigma, CA) and polar bodies were mechanically separated from the oocytes. Oocytes were immediately fixed with 3.7% paraformaldehyde in PHEM (PIPES 12 mM, HEPES 5 mM, EGTA 2 mM and MgSO4・7H2O 0.8 mM, pH 6.9) for 30 min at room temperature (RT) and then permeabilized in PHEM + 0.25% Triton-X for 15 min at RT. After permeabilization, they were incubated overnight at 4 °C in blocking solution (3% bovine serum albumin (BSA) + 0.05% Tween-20 in 1X PBS). On the next day, the plate was brought to RT before transferring the oocytes in the primary antibody solution for 1 h at 37 °C (mouse anti-ɑ-Tubulin (1:250), T6199, Sigma-Aldrich, USA). Then, the oocytes were washed three times in the wash solution (0.5% BSA + 0.05% Tween-20 in PBS) at RT and moved in the secondary antibody solution for 2 h at 37 °C (goat anti-mouse AlexaFluor 488 (1:200), A-11001, Invitrogen, USA and Phalloidin AlexaFluor 555 (1:500) (A34055, Invitrogen, USA). After secondary antibody solution, the oocytes were washed three times in the wash solution and transferred into Hoechst 33342 (1:500) for 30 min. Finally, the oocytes were transferred into 1.5 µL drops in an imaging dish (Nunc Glass Bottom Dish, ThermoScientific, CA) covered with paraffin oil and 0.2 µm z-stacks were captured using Leica SP8 confocal microscope using the 63× objective with oil and a zoom factor set at 8 at the Advanced Optical Microscopy Facility (Toronto, Canada). Deconvolution was applied on the images using Huygens software version 23.10 (https://svi.nl/Huygens-Software) and images were visualized using ImarisViewer 10.1.1. Negative controls consisted of GV oocytes to observe the absence of spindle structure and stained MII-oocytes without the mouse anti-ɑ-Tubulin primary antibody (Supplementary Fig. Polar bodies were separated from the oocytes after removal of the ZP eliminating possible somatic cell contamination (as shown in Supplementary Fig. Polar bodies were individually snap-frozen at −80 °C in less than 2 µL and blinded samples were sent for whole genome chromosome copy number variation assessment (CNV) to the CReATe Reproductive Genetics sequencing platform. CNV analysis was performed by low-pass whole genome Next Generation Sequencing using validated clinical workflow on Illumina platform. Briefly, gDNA was amplified using SurePlex Whole Genome Amplification (WGA) (Illumina, CA), according to manufacturer's instructions. Amplified gDNA was then tagmented and indexed using Nextera XT (Illumina, CA). The indexed libraries were purified using AMPure XP beads (1:1 ratio) and normalized using magnetic beads. The normalized libraries were pooled, denatured, and sequenced on a NextSeq 550 (paired end, 2 × 75 bp). NxClinical version 6.0 (Bionano, CA) was used for chromosome CNV analysis and data visualization according to our standard clinical procedure (2 million reads/sample, CNV resolution of >10 Mb). Optimization experimentations demonstrated the concordance between polar body chromosome numbers and its sister oocyte (Supplementary Fig. Datasets were first assessed for normality using the Shapiro–Wilk test. Statistical significance was determined using two-sided Fisher's exact test for contingency analysis (maturation rate, spindle morphology and euploid rates), One-way ANOVA with a two-sided Holm–Sidak's multiple comparison test for continuous normally distributed datasets (oocyte diameter), or Kruskal–Wallis with a two-sided Dunn's multiple comparison test for continuous non-normally distributed datasets. The specific statistical test is indicated throughout table and figure legends. Significance was defined as p < 0.05. For the retrospective analysis, we performed pairwise case-control matching, where each THC-positive sample was matched to two THC-negative samples, as determined by the presence/absence of 11-COOH-THC in the FF. Matching was conducted using the FUZZY matching command in Statistical Package for Social Sciences (SPSS-v29) based on the following covariates: oocyte age, participant body mass index (BMI), anti-müllerian hormone (AMH), day 2/3 luteinizing hormone (LH), and follicle stimulating hormone (FSH), (estradiol) E2 on trigger, and total gonadotropin (GT) dose. AMH, LH, FSH and E2 were quantified during the clinical assessment using the Cobas e411 instrument (Roche, Basel, Switzerland). Matching success was determined using a two-tailed Mann–Whitney test for non-parametric distribution. Statistical significance of the IVF outcomes was determined using two-tailed Mann–Whitney test for non-parametric distribution (fertilization and euploid rates) and unpaired two-tailed t-test for parametric distribution (maturation and blastocyst rates) using GraphPad Prism 10.2.3. Numbers in parentheses in each figure legends represent distinct samples and not repeated measures. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data generated in this study are provided in the Supplementary Information/Source Data file. The normalized read count expression matrix and metadata have been deposited in the Gene Expression Omnibus (GEO) database with accession number GSE297757. Due to participant privacy concerns and institutional ethics restrictions, raw sequencing files (fastq) have been deposited at the European Genome-phenome Archive (EGA), which is hosted by the European Bioinformatics Institute (EBI) and the Centre for Genomic Regulation (CRG), under accession number EGAS50000001052. Further information about EGA can be found at https://ega-archive.org. Expected timeframe for response to access requests is 10 business days and the data will be available for the duration of the study as defined by the Data Access Agreement associated with this dataset. Source data are provided with this paper. Hansford, B. UNODC World Drug Report 2022 highlights trends on cannabis post-legalization, environmental impacts of illicit drugs, and drug use among women and youth (United Nations, 2022). About cannabis (Government of Canada, 2023). Parizek, A. et al. The endocannabinoid system and its relationship to human reproduction. Google Scholar Schrott, R. et al. Cannabis use is associated with potentially heritable widespread changes in autism candidate gene DLGAP2 DNA methylation in sperm. Google Scholar Murphy, S. K. et al. Cannabinoid exposure and altered DNA methylation in rat and human sperm. Google Scholar Kuzma-Hunt, A. G. et al. THC and sperm: impact on fertilization capability, pre-implantation in vitro development and epigenetic modifications. Google Scholar Hedges, J. C. et al. Cessation of chronic delta-9-tetrahydrocannabinol use partially reverses impacts on male fertility and the sperm epigenome in rhesus macaques. Google Scholar Hedges, J. C. et al. Chronic exposure to delta-9-tetrahydrocannabinol impacts testicular volume and male reproductive health in rhesus macaques. Google Scholar Kolodny, R. C., Masters, W. H., Kolodner, R. M. & Toro, G. Depression of plasma testosterone levels after chronic intensive marihuana use. Google Scholar Gundersen, T. D. et al. Association between use of marijuana and male reproductive hormones and semen quality: a study among 1,215 healthy young men. Google Scholar Hembree, W. C. III, Nahas, G. G., Zeidenberg, P. & Huang, H. F. Changes in human spermatozoa associated with high dose marihuana smoking. Google Scholar Payne, K. S., Mazur, D. J., Hotaling, J. M. & Pastuszak, A. W. Cannabis and male fertility: a systematic review. Google Scholar Belladelli, F. et al. Effects of recreational cannabis on testicular function in primary infertile men. Hehemann, M. C. et al. Evaluation of the impact of marijuana use on semen quality: a prospective analysis. Gunn, J. K. et al. Prenatal exposure to cannabis and maternal and child health outcomes: a systematic review and meta-analysis. Leemaqz, S. Y. et al. Maternal marijuana use has independent effects on risk for spontaneous preterm birth but not other common late pregnancy complications. Metz, T. D. et al. Cannabis exposure and adverse pregnancy outcomes related to placental function. Ayonrinde, O. T. et al. Association between gestational cannabis exposure and maternal, perinatal, placental, and childhood outcomes. Rokeby, A. C. E., Natale, B. V. & Natale, D. R. C. Cannabinoids and the placenta: receptors, signaling and outcomes. Rompala, G., Nomura, Y. & Hurd, Y. L. Maternal cannabis use is associated with suppression of immune gene networks in placenta and increased anxiety phenotypes in offspring. Harton, G. L. et al. ESHRE PGD Consortium/Embryology Special Interest Group–best practice guidelines for polar body and embryo biopsy for preimplantation genetic diagnosis/screening (PGD/PGS). Le, H. H. et al. Effects of in utero exposure to Delta-9-tetrahydrocannabinol on cardiac extracellular matrix expression and vascular transcriptome in rhesus macaques. Fuchs Weizman, N., Wyse, B. A., Montbriand, J., Jahangiri, S. & Librach, C. L. Cannabis significantly alters DNA methylation of the human ovarian follicle in a concentration-dependent manner. Fuchs Weizman, N. et al. Cannabis alters epigenetic integrity and endocannabinoid signalling in the human follicular niche. Misner, M. J. et al. Effects of delta-9 tetrahydrocannabinol (THC) on oocyte competence and early embryonic development. Totorikaguena, L. et al. The endocannabinoid system modulates the ovarian physiology and its activation can improve in vitro oocyte maturation. Totorikaguena, L., Olabarrieta, E., Lopez-Cardona, A. P., Agirregoitia, N. & Agirregoitia, E. Tetrahydrocannabinol modulates in vitro maturation of oocytes and improves the blastocyst rates after in vitro fertilization. Lopez-Cardona, A. P. et al. CB(1) cannabinoid receptor drives oocyte maturation and embryo development via PI3K/Akt and MAPK pathways. Lopez-Cardona, A. P. et al. Exocannabinoids effect on in vitro bovine oocyte maturation via activation of AKT and ERK1/2. He, M., Zhang, T., Yang, Y. & Wang, C. Mechanisms of oocyte maturation and related epigenetic regulation. Siu, M. K. & Cheng, C. Y. The blood-follicle barrier (BFB) in disease and in ovarian function. Shani, A. K. et al. The developmental potential of mature oocytes derived from rescue in vitro maturation. Fuchs Weizman, N. et al. Towards improving embryo prioritization: parallel next generation sequencing of DNA and RNA from a single trophectoderm biopsy. Montag, M., van der Ven, K., Rosing, B. & van der Ven, H. Polar body biopsy: a viable alternative to preimplantation genetic diagnosis and screening. McCoy, R. C. et al. Evidence of selection against complex mitotic-origin aneuploidy during preimplantation development. Wasielak-Politowska, M. & Kordowitzki, P. Chromosome segregation in the oocyte: what goes wrong during aging. ESHRE Special Interest Group of Embryology and Alpha Scientists in Reproductive Medicine The Vienna consensus: report of an expert meeting on the development of ART laboratory performance indicators. Canadian cannabis survey (Government of Canada, 2023). Human cannabinoid pharmacokinetics. Lemseffer, Y., Terret, M. E., Campillo, C. & Labrune, E. Methods for assessing oocyte quality: a review of literature. Coordination of nuclear and cytoplasmic oocyte maturation in eutherian mammals. Pors, S. E. et al. Oocyte diameter predicts the maturation rate of human immature oocytes collected ex vivo. Bonnet-Garnier, A. et al. Genome organization and epigenetic marks in mouse germinal vesicle oocytes. De La Fuente, R. Chromatin modifications in the germinal vesicle (GV) of mammalian oocytes. Innocenti, F. et al. Maternal effect factors that contribute to oocytes developmental competence: an update. Ermisch, A. F. & Wood, J. R. Regulation of oocyte mRNA metabolism: a key determinant of oocyte developmental competence. Takino, T. et al. Cleavage of metastasis suppressor gene product KiSS-1 protein/metastin by matrix metalloproteinases. Tschesche, H. et al. Latent collagenase and gelatinase from human neutrophils and their activation. Wilhelm, S. M. et al. SV40-transformed human lung fibroblasts secrete a 92-kDa type IV collagenase which is identical to that secreted by normal human macrophages. Ichikawa, S., Ohta, M., Morioka, H. & Murao, S. Blockage of ovulation in the explanted hamster ovary by a collagenase inhibitor. Brannstrom, M., Woessner, J. F. Jr., Koos, R. D., Sear, C. H. & LeMaire, W. J. Inhibitors of mammalian tissue collagenase and metalloproteinases suppress ovulation in the perfused rat ovary. Curry, T. E. Jr. & Osteen, K. G. The matrix metalloproteinase system: changes, regulation, and impact throughout the ovarian and uterine reproductive cycle. Zhang, S. et al. Matrix metalloproteinases improves trophoblast invasion and pregnancy potential in mice. Mahdavinezhad, F. et al. In vitro versus in vivo: development-, apoptosis-, and implantation- related gene expression in mouse blastocyst. Fontana, V. et al. Matrix metalloproteinase expression and activity in trophoblast-decidual tissues at organogenesis in CF-1 mouse. Harvey, M. B. et al. Proteinase expression in early mouse embryos is regulated by leukaemia inhibitory factor and epidermal growth factor. Wang, H. M. et al. Effect of ubiquitin-proteasome pathway on mouse blastocyst implantation and expression of matrix metalloproteinases-2 and −9. Kyprianou, C. et al. Basement membrane remodelling regulates mouse embryogenesis. Staun-Ram, E., Goldman, S., Gabarin, D. & Shalev, E. Expression and importance of matrix metalloproteinase 2 and 9 (MMP-2 and −9) in human trophoblast invasion. McGowen, M. R., Erez, O., Romero, R. & Wildman, D. E. The evolution of embryo implantation. Librach, C. L. et al. Interleukin-1 beta regulates human cytotrophoblast metalloproteinase activity and invasion in vitro. & Khalil, R. A. Matrix metalloproteinases in normal pregnancy and preeclampsia. Yan, Y. et al. Association of MMP2 and MMP9 gene polymorphisms with the recurrent spontaneous abortion: a meta-analysis. Goto, S. et al. MMP2 and MMP9 are associated with the pathogenesis of recurrent pregnancy loss through protein expression rather than genetic polymorphism. Karachrysafi, S. et al. Immunohistochemical study of MMP-2, MMP-9, EGFR and IL-8 in decidual and trophoblastic specimens of recurrent pregnancy loss cases. Fetal Neonatal Med. Yao, J. L. et al. Effects of Delta(9)-tetrahydrocannabinol (THC) on human amniotic epithelial cell proliferation and migration. Zhang, Y., Zheng, W., Shen, K. & Shen, W. ∆9-tetrahydrocannabinol inhibits epithelial-mesenchymal transition and metastasis by targeting matrix metalloproteinase-9 in endometrial cancer. Peralta, L. et al. Expression and localization of cannabinoid receptors in human immature oocytes and unfertilized metaphase-II oocytes. Neubig, R. R. & Siderovski, D. P. Regulators of G-protein signalling as new central nervous system drug targets. Zhou, J. et al. Characterization of RGS5 in regulation of G protein-coupled receptor signaling. Kimple, A. J., Bosch, D. E., Giguere, P. M. & Siderovski, D. P. Regulators of G-protein signaling and their Galpha substrates: promises and challenges in their use as drug discovery targets. B., Wilkinson, J. C. & Roman, D. L. Regulator of G-protein signaling (RGS) proteins as drug targets: progress and future potentials. Dekel, N., Gnainsky, Y., Granot, I. & Mor, G. Inflammation and implantation. Schroder, K., Hertzog, P. J., Ravasi, T. & Hume, D. A. Interferon-gamma: an overview of signals, mechanisms and functions. Johns, D. N. et al. Conceptus interferon gamma is essential for establishment of pregnancy in the pig. Cayrol, C. & Girard, J. P. Interleukin-33 (IL-33): a critical review of its biology and the mechanisms involved in its release as a potent extracellular cytokine. Begum, S. et al. Dynamic expression of interleukin-33 and ST2 in the mouse reproductive tract is influenced by superovulation. Granne, I. et al. ST2 and IL-33 in pregnancy and pre-eclampsia. Stampalija, T. et al. Maternal plasma concentrations of sST2 and angiogenic/anti-angiogenic factors in preeclampsia. Fetal Neonatal Med. Eda Gokdemir, I. et al. Evaluation of ADAMTS12, ADAMTS16, ADAMTS18 and IL-33 serum levels in pre-eclampsia. Fetal Neonatal Med. Li, M. et al. Amniotic fluid proteomic analysis identifies IL1RL1, APOE, and NECTIN4 as new biomarkers for preterm birth. BMC Pregnancy Childbirth 24, 530 (2024). Green, E. A. et al. The role of the interleukin-1 family in complications of prematurity. Cekmez, Y. et al. uPAR, IL-33, and ST2 values as a predictor of subclinical chorioamnionitis in preterm premature rupture of membranes. J. Interferon Cytokine Res. Franco-De Leon, K. et al. Interleukins IL33/ST2 and IL1-beta in intrauterine growth restriction and seropositivity of anti-toxoplasma gondii antibodies. Kaitu'u-Lino, T. J., Tuohey, L. & Tong, S. Maternal serum interleukin-33 and soluble ST2 across early pregnancy, and their association with miscarriage. & Sun, S. C. Actin cytoskeleton dynamics in mammalian oocyte meiosis. Mikwar, M., MacFarlane, A. J. & Marchetti, F. Mechanisms of oocyte aneuploidy associated with advanced maternal age. Charalambous, C., Webster, A. & Schuh, M. Aneuploidy in mammalian oocytes and the impact of maternal ageing. Gruhn, J. R. et al. Chromosome errors in human eggs shape natural fertility over reproductive life span. Wang, L. et al. IVF embryo choices and pregnancy outcomes. Kubicek, D. et al. Incidence and origin of meiotic whole and segmental chromosomal aneuploidies detected by karyomapping. Konstantinidis, M. et al. Aneuploidy and recombination in the human preimplantation embryo. Copy number variation analysis and genome-wide polymorphism genotyping. Capalbo, A. et al. Sequential comprehensive chromosome analysis on polar bodies, blastomeres and trophoblast: insights into female meiotic errors and chromosomal segregation in the preimplantation window of embryo development. Ottolini, C. S. et al. Genome-wide maps of recombination and chromosome segregation in human oocytes and embryos show selection for maternal recombination rates. Viotti, M. Preimplantation genetic testing for chromosomal abnormalities: aneuploidy, mosaicism, and structural rearrangements. Ariad, D. et al. Aberrant landscapes of maternal meiotic crossovers contribute to aneuploidies in human embryos. Thomas, C., Cavazza, T. & Schuh, M. Aneuploidy in human eggs: contributions of the meiotic spindle. Capalbo, A., Poli, M., Jalas, C., Forman, E. J. & Treff, N. R. On the reproductive capabilities of aneuploid human preimplantation embryos. & Volkow, N. D. Cannabis and cannabinoid signaling: research gaps and opportunities. Das, D. & Arur, S. Regulation of oocyte maturation: Role of conserved ERK signaling. Har-Gil, E., Heled, A., Dixon, M., Ahamed, A. M. S. & Bentov, Y. The relationship between cannabis use and IVF outcome-a cohort study. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Merico, D., Isserlin, R., Stueker, O., Emili, A. & Bader, G. D. Enrichment map: a network-based method for gene-set enrichment visualization and interpretation. received salary support through the University of Toronto Department of Physiology Yuet Ngor Wong Postdoctoral Fellowship Award. All other funding was provided by CReATe Fertility Centre through the reinvestment of clinical earnings. Human biological materials and associated de-identified data were collected by CReATe Biobank personnel. The CReATe Biobank is certified by the Canadian Tissue Repository Network (CTRNet) Biobank Program. The authors would like to thank all, past and present, CReATe Biobank staff, CReATe Fertility Centre clinical and IVF laboratory staff, CReATe Reproductive Genetics staff, and CReATe Research staff for their dedication to this study. Lastly, the authors would like to thank all patients for the donation of their biological material. CReATe Fertility Centre, Toronto, ON, Canada Cyntia Duval, Brandon A. Wyse, Noga Fuchs Weizman, Iryna Kuznyetsova, Svetlana Madjunkova & Clifford L. Librach Department of Physiology, Temerty Faculty of Medicine, University of Toronto, Toronto, ON, Canada Cyntia Duval & Clifford L. Librach Racine IVF Unit, Lis Maternity Hospital, Tel Aviv Sourasky Medical Center, Tel Aviv-Yafo, Israel Noga Fuchs Weizman Department of Obstetrics and Gynecology, Temerty Faculty of Medicine, University of Toronto, Toronto, ON, Canada Institute of Medical Sciences, Temerty Faculty of Medicine, University of Toronto, Toronto, ON, Canada Sunnybrook Research Institute, Toronto, ON, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar participated in the design of the study. established the collection of immature human oocytes through the CReATe Biobank in collaboration with IK and her team helping coordinate the oocyte collection. received, cultured and froze the oocytes for all experiments. optimized and conducted oocyte staining and confocal imaging, polar body biopsies, and RNA sequencing related manipulations. performed the transcriptomic analysis. compiled the retrospective data and analyzed the results. and her team conducted the polar body sequencing to assess the ploidy status. wrote the manuscript. hand-drawn all graphical elements. provided critical feedback on study design and the manuscript. All authors provided feedback on the manuscript and read and approved the final manuscript. Correspondence to Cyntia Duval. The authors declare no competing interests. Nature Communications thanks Norbert Gleicher and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Duval, C., Wyse, B.A., Fuchs Weizman, N. et al. Cannabis impacts female fertility as evidenced by an in vitro investigation and a case-control study. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02901-3'>My blue is your blue: different people's brains process colours in the same way</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 09:48:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). “Now we know that when you see red or green or whatever colour, that it activates your brain very similarly to my brain,” says study co-author Andreas Bartels, a cognitive neuroscientist at the University of Tubingen and the Max Planck Institute, both in Tubingen, Germany. Bartels and his colleague Michael Bannert wanted to explore how different colours are represented in parts of the brain associated with vision, and how consistent this is across different people. The pair used functional magnetic resonance imaging (fMRI) to compare activity in the brains of a group of participants while they viewed different colours. This allowed them to create a map of brain activity that showed how each hue was represented neurologically. They then trained a machine learning model called a linear classifier on this data, and used it to predict which colours were being viewed by members of a second group of study participants, on the basis of their brain activity. Depriving brain tumours of an amino acid could enhance chemotherapy Brain tumours in mice grow more slowly when starved of key amino acid Developing machine learning methods to infer multi-modal, condition-dependent causal gene networks from large-scale single-cell multi-omic datasets. Develop nanofab processes, operate advanced tools includes EBL, KrF Stepper, PL, CDSEM & PVD for cutting-edge BCI research. The Center for Metastasis Research eXcellence (MET-X) at the Fred Hutchinson Cancer Center (Fred Hutch) is launching an open-rank faculty search f... An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01481-x'>Combinatorial prediction of therapeutic perturbations using causally inspired neural networks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 09:39:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Phenotype-driven approaches identify disease-counteracting compounds by analysing the phenotypic signatures that distinguish diseased from healthy states. Here we introduce PDGrapher, a causally inspired graph neural network model that predicts combinatorial perturbagens (sets of therapeutic targets) capable of reversing disease phenotypes. Unlike methods that learn how perturbations alter phenotypes, PDGrapher solves the inverse problem and predicts the perturbagens needed to achieve a desired response by embedding disease cell states into networks, learning a latent representation of these states, and identifying optimal combinatorial perturbations. It also shows competitive performance on ten genetic perturbation datasets. An advantage of PDGrapher is its direct prediction, in contrast to the indirect and computationally intensive approach common in phenotype-driven models. It trains up to 25× faster than existing methods, providing a fast approach for identifying therapeutic perturbations and advancing phenotype-driven drug discovery. Target-driven drug discovery, which has been the dominant approach since the 1990s, focuses on designing highly specific compounds to act against targets, such as proteins or enzymes, that are implicated in disease, often through genetic evidence1,2,3. An example of target-driven drug discovery is the development of small molecule kinase inhibitors like imatinib. Imatinib stops the progression of chronic myeloid leukaemia by inhibiting BCR-ABL tyrosine kinase, a mutated protein involved in uncontrolled proliferation of leukocytes in patients with chronic myeloid leukemia4. Other notable examples include monoclonal antibodies such as trastuzumab, which specifically targets the HER2 receptor, a protein overexpressed in certain types of breast cancer. Trastuzumab inhibits cell proliferation while engaging the body's immune system to initiate an anti-cancer response5. These examples illustrate the success of target-driven drug discovery, yet the past decade has seen a revival of phenotype-driven approaches. This shift has been fuelled by the observation that many first-in-class drugs approved by the US Food and Drug Administration (FDA) between 1999 and 2008 were discovered without a drug target hypothesis6. Instead of the ‘one drug, one gene, one disease' model of target-driven approaches, phenotype-driven drug discovery focuses on identifying compounds or, more broadly, perturbagens—combinations of therapeutic targets—that reverse disease phenotypes as measured by assays without predefined targets1,7. Ivacaftor illustrates how these approaches can intersect. Although ivacaftor was developed through a target-driven strategy to modulate the cystic fibrosis transmembrane conductance regulator protein in individuals with specific mutations, its development relied on phenotypic assays to confirm functional improvements, such as increased chloride transport8,9,10. Phenotype-driven drug discovery has been bolstered by the advent of chemical and genetic libraries such as the Connectivity Map (CMap)11 and the Library of Integrated Network-based Cellular Signatures (LINCS)12. CMap introduced connectivity scores to quantify similarities between compound responses and disease gene expression signatures. Identifying compounds with gene expression signatures either similar to those of known disease-treating drugs or that counter disease signatures can help in selecting therapeutic leads13,14,15,16. These strategies have successfully identified drugs with high in vitro efficacy16 across a range of diseases17,18,19. Deep learning methods have been used for lead discovery by predicting gene expression responses to perturbagens, including perturbagens that were not yet experimentally tested20,21,22,23. However, these approaches rely on chemical and genetic libraries, meaning that they select perturbagens from predefined libraries and cannot identify perturbagens as new combinations of drug targets. Further, they are perturbation response methods that predict changes in phenotypes upon perturbations. Thus, they can identify perturbagens by exhaustively predicting responses to all perturbations in the library and then searching for perturbagens with the desired response. Unlike existing methods that learn responses to perturbations, phenotype-based approaches need to solve the inverse problem, which is to infer perturbagens necessary to achieve a specific response—that is, directly predicting perturbagens by learning which perturbations elicit a desired response. In causal discovery, the problem of identifying which elements of a system should be perturbed to achieve a desired state is referred to as optimal intervention design24,25,26. Using insights from causal discovery and geometric deep learning, here we introduce PDGrapher, an approach for the combinatorial prediction of therapeutic targets that can shift gene expression from an initial diseased state to a desired treated state. Given a genetic or chemical intervention dataset, PDGrapher pinpoints a set of genes that a perturbagen should target to facilitate the transition of node states from diseased to treated. PDGrapher uses protein–protein interaction (PPI) networks or gene regulatory networks (GRNs) as approximations of the causal graph, operating under the assumption of no unobserved confounders. PDGrapher tackles the optimal intervention design using representation learning, using a graph neural network (GNN) to represent structural equations. PDGrapher is trained on a dataset of disease–treated sample pairs to predict therapeutic gene targets that can shift the gene expression phenotype from a diseased to a healthy or treated state. Once trained, PDGrapher processes a new diseased sample and outputs a perturbagen—a set of therapeutic targets—predicted to counteract the disease effects in that specific sample. We evaluate PDGrapher across 19 datasets, comprising genetic and chemical interventions across 11 cancer types and two proxy causal graphs. We also consider different evaluation set-ups, including settings where held-out folds contain new samples in the same cell line with the training samples, and settings where held-out folds contain new samples from a cancer type that PDGrapher has never encountered during training. In held-out folds that contain new samples, PDGrapher detects up to 13.37% and 1.09% more ground-truth therapeutic targets in chemical and genetic intervention datasets, respectively, than existing methods. We also find that in chemical intervention datasets, candidate therapeutic targets predicted by PDGrapher are on average up to 11.58% closer to ground-truth therapeutic targets in the gene–gene interaction network than what would be expected by chance. Even in held-out folds containing new samples from a previously unseen cancer type, PDGrapher maintains robust performance. Unlike methods that indirectly identify perturbagens by predicting cell responses, PDGrapher directly predicts perturbagens that can shift gene expression from diseased to treated states. This feature of PDGrapher enables model training up to 25× faster than indirect prediction methods, such as scGen22 and CellOT27. As these approaches build a separate model for each perturbation, they become increasingly ineffective when applied to datasets with a large number of perturbagens. For example, with its default settings, CellOT needs 10 h to train for a single perturbagen in a cell line from the LINCS dataset. PDGrapher can aid in elucidating the mechanism of action of chemical perturbagens (Supplementary Fig. 3), which we show in the case of vorinostat, a histone deacetylase inhibitor used to treat cutaneous T cell lymphoma, and sorafenib, a multikinase inhibitor used in the treatment of several types of cancer (Supplementary Note 1). PDGrapher can also suggest potential anti-cancer therapeutic targets: it highlighted kinase insert domain receptor (KDR) as a top predicted target for non-small cell lung cancer (NSCLC; see ‘ PDGrapher identifies therapeutic targets validated through clinical and biological evidence' in Results). It identified associated drugs—vandetanib, sorafenib, catequentinib and rivoceranib—that inhibit the kinase activity of the protein encoded by KDR. These drugs block VEGF signalling, suppressing endothelial cell proliferation, migration and blood vessel formation, which tumours rely on for growth and metastasis28,29. By predicting combinatorial therapeutic targets based on phenotypic transitions, PDGrapher provides a scalable approach to phenotype-driven perturbation modelling. We evaluate our method across a total of 38 preprocessed datasets that span 2 types of intervention (genetic and chemical), 11 cancer types (lung, breast, prostate, colon, skin, cervical, head and neck, pancreatic, stomach, brain and ovarian), and 2 types of proxy causal graph: PPI networks and GRNs. Each dataset is uniquely defined by a combination of intervention type, causal graph type, cancer type, and cell line, and is denoted in the format: treatment type-graph type-cancer type-cell line. The genetic-PPI datasets include A549 (lung), MCF7 (breast), PC3 (prostate), HT29 (colon), A375 (skin), ES2 (ovary), BICR6 (head and neck), YAPC (pancreas), AGS (stomach) and U251MG (brain). Similarly, the chemical-GRN and genetic-GRN datasets span the same combinations of cancer types and cell lines as their PPI counterparts. This comprehensive collection of datasets enables systematic benchmarking of our model across diverse perturbation modalities, biological contexts and graph structures. Genetic interventions are single-gene knockout experiments by CRISPR–Cas9-mediated gene knockouts, while chemical interventions are multiple-gene treatments induced using chemical compounds. We use a PPI network from BIOGRID that has 10,716 nodes and 151,839 undirected edges. We additionally construct GRNs for each disease-treatment type pair using GENIE3 (ref. 30) (Supplementary Note 3), with GRNs on average having 10,000 nodes and 500,000 directed edges. Disease intervention data include paired healthy and diseased gene expression profiles, along with associated disease genes; however, these data are only available for cell lines corresponding to lung, breast and prostate cancers. In contrast, the treatment intervention data comprise paired diseased and treated gene expression samples, along with genetic or chemical perturbagens, and are available across all datasets. Supplementary Tables 11 and 12 summarize the number of samples for each cell line and intervention datasets. Given a diseased cell state (gene expression profile), the goal of PDGrapher is to predict the genes that, if targeted by a perturbagen, would shift the cell to a treated state (Fig. Unlike methods for learning the response of cells to a given perturbation22,27,31,32, PDGrapher focuses on the inverse problem by learning which perturbation elicits a desired response. PDGrapher predicts perturbagens that shift cellular states under the assumption that an optimal perturbagen is one that alters the gene expression profile of a cell to closely match a desired target state. Our approach comprises two modules (Fig. First, a perturbagen discovery module ƒp takes the initial and desired cell states and outputs a candidate perturbagen as a set of therapeutic targets \({{\mathcal{U}}}^{{\prime} }\) (Fig. Then, a response prediction module ƒr takes the initial state and the predicted perturbagen \({{\mathcal{U}}}^{{\prime} }\) and predicts the cell response upon perturbing genes in \({{\mathcal{U}}}^{{\prime} }\) (Fig. Our response prediction and perturbagen discovery modules are GNN models that operate on a proxy causal graph, where edge mutilations, or edge removals, represent the effects of interventions on the graph (Fig. b, Given a disease sample's gene expression, a proxy causal graph and a set of perturbagens, PDGrapher's response prediction module, ƒr, predicts the gene expression response of the sample to each perturbagen. ƒr represents perturbagen's effects in the graph as edge mutilations. c, ƒp is optimized using two losses: a cross-entropy cycle loss to predict a perturbagen \({{\mathcal{U}}}^{{\prime} }\) that aims to shift the diseased cell state closely approximating the treated state, \({\rm{CE}}\left({{\bf{x}}}^{\rm{t}},\,{f}_{\rm{r}}({{\bf{x}}}^{\rm{d}},\,{f}_{\rm{p}}({{\bf{x}}}^{\rm{d}},\,{{\bf{x}}}^{\rm{t}}))\right)\,\,({\rm{with}}\,{f}_{\rm{r}}\,{\rm{frozen}})\), and a cross-entropy supervision loss that directly supervises the prediction of \({{\mathcal{U}}}^{{\prime} }\), \({\rm{CE}}\left({{\mathcal{U}}}^{{\prime} },{f}_{\rm{p}}({{\bf{x}}}^{\rm{d}},\,{{\bf{x}}}^{\rm{t}})\right)\) (see Methods for more details). d, Both ƒr and ƒp follow the standard message-passing framework, where node representations are updated by aggregating the information from neighbours in the graph. PDGrapher is trained using an objective function with two components, one for each module, ƒr and ƒp. The response prediction module ƒr is trained using disease and treatment intervention data on cell state transitions so that the predicted cell states are close to the known perturbed cell states upon interventions. The perturbagen discovery module ƒp is trained only using the treatment intervention data; given a diseased cell state, ƒp predicts the set of therapeutic targets \({{\mathcal{U}}}^{{\prime} }\) that caused the corresponding treated cell state. The objective function for the perturbagen discovery module consists of two elements: (1) a cycle loss that optimizes the parameters of ƒp such that the response upon intervening on the predicted genes in \({{\mathcal{U}}}^{{\prime} }\), as measured by ƒr, closely approximates the actual treated cellular state; and (2) a supervision loss on the therapeutic target set \({{\mathcal{U}}}^{{\prime} }\) that directly pushes PDGrapher to predict the correct perturbagen. Both models are trained simultaneously using early stopping independently so that each model finishes training upon convergence. When trained, PDGrapher predicts perturbagens—as sets of candidate target genes—to shift cells from diseased to treated. Given a pair of diseased and treated samples, PDGrapher directly predicts perturbagens by learning which perturbations elicit target responses. In contrast, existing approaches are perturbation response methods that predict changes in phenotype that occur upon perturbation; thus, they can only indirectly predict perturbagens (Fig. Given a disease–treated sample pair, a response prediction module (such as scGen22, ChemCPA23, Biolord33, GEARS34 or CellOT27) is used to predict the response of the diseased sample to a library of perturbagens. We evaluate PDGrapher's performance in two separate settings (Fig. 2b,c): (1) a random splitting setting, where the samples are split randomly between training and test sets within a cell line (denoted as random for convenience); and (2) a leave-cell-out setting, where PDGrapher is trained in one cell line, and its performance is evaluated in a cell line the model never encountered during training to test how well the model generalizes to a new disease. Supplementary Tables 1 and 2 show the numbers of unseen perturbagens in chemical perturbation datasets in the random and leave-cell-out splits, respectively; Supplementary Tables 3 and 4 show the number of unseen perturbagens in genetic perturbation datasets in the random and leave-cell-out splits, respectively. The direct prediction means that PDGrapher directly infers the perturbation necessary to achieve a specific response. In contrast to direct prediction of perturbagens, existing methods predict perturbagens only indirectly through a two-stage approach. For a given diseased sample, the model learns the response to each candidate perturbagen from an existing library and identifies the perturbagen whose induced response most closely approximates the desired treated state. Existing methods learn the response of cells to a given perturbation22,27,31,32, whereas PDGrapher focuses on the inverse problem by learning which perturbagen elicits a given response, even in the most challenging cases when the combinatorial composition of perturbagen was never seen before. b,c, We evaluate PDGrapher's performance across two settings: given a cell line, randomly splitting samples between training and testing set (b), and by splitting samples based on cell lines, where the model is trained on one cell line and evaluated on a different, previously unseen cell line to assess PDGrapher's generalization performance (c). In the random splitting setting, we assess the ability of PDGrapher for combinatorial prediction of therapeutic targets across chemical PPI datasets (chemical-PPI-lung-A549, chemical-PPI-breast-MCF7, chemical-PPI-breast-MDAMB231, chemical-PPI-breast-BT20, chemical-PPI-prostate-PC3, chemical-PPI-prostate-VCAP, chemical-PPI-colon-HT29, chemical-PPI-skin-A375 and chemical-PPI-cervix-HELA). Specifically, we measure whether, given paired diseased–treated gene expression samples, PDGrapher can predict the set of therapeutic genes targeted by the chemical compound in the diseased sample to generate the treated sample. Given paired diseased–treated gene expression samples, PDGrapher ranks genes in the dataset according to their likelihood of being therapeutic targets. An nDCG value close to one indicates highly accurate predictions, with the top ranked gene targets closely matching the ground-truth targets, whereas lower nDCG values indicate poorer ranking performance. This metric provides a normalized and scalable measure of ranking quality, enabling consistent comparison across different datasets and models. PDGrapher outperforms competing methods in all cell lines, achieving nDCG values that are higher than the second-best competing method by 0.02 (chemical-PPI-lung-A549), 0.13 (chemical-PPI-breast-MDAMB231), 0.03 (chemical-PPI-breast-BT20), 0.004 (chemical-PPI-breast-MCF7), 0.07 (chemical-PPI-prostate-VCAP), 0.005 (chemical-PPI-prostate-PC3), 0.03 (chemical-PPI-skin-A375), 0.06 (chemical-PPI-cervix-HELA) and 0.001 (chemical-PPI-colon-HT29) (Fig. In addition to evaluating the entire predicted target rank, it is even more practically crucial to assess the accuracy of the top ranked predicted targets. As perturbagens target multiple genes, we measure the fraction of samples in the test set for which we obtain a partially accurate prediction, where at least one of the top N predicted gene targets corresponds to an actual gene target (denoted as the percentage of accurately predicted samples). PDGrapher consistently provides accurate predictions for more samples in the test set than competing methods. Specifically, it outperforms the second-best competing method by predicting ground-truth targets in an additional 7.73% (chemical-PPI-breast-MCF7), 9.32% (chemical-PPI-lung-A549), 13.37% (chemical-PPI-breast-MDAMB231), 4.50% (chemical-PPI-breast-BT20), 7.88% (chemical-PPI-prostate-PC3), 11.53% (chemical-PPI-prostate-VCAP), 7.56% (chemical-PPI-colon-HT29), 9.55% (chemical-PPI-skin-A375) and 8.41% (chemical-PPI-cervix-HELA) of samples (Fig. We also evaluate the performance of PDGrapher using recall@1, recall@10 and recall@100, which calculate the ratio of true target genes included in the top 1, top 10 and top 100 predicted gene targets, respectively. Although the absolute recall values are modest due to the inherent difficulty of the task, PDGrapher consistently outperforms all competing methods, showing its relative strength and robustness. We then consolidated the results using the rankings from experiments across different cell lines and metrics for each method. PDGrapher achieved the best overall rankings, with a median significantly higher than all competing methods (Fig. P values of the chemical perturbagen discovery tests are provided in the Source data. a,b, PDGrapher shows improved performance across nine chemical perturbation datasets with various diseases, yielding up to 13.37% more accurately predicted samples in the testing sets compared with the second-best model (for example, for chemical-PPI-breast-MDAMB231, 20.43% versus 7.05% (a)) and up to 0.13 higher nDCG than the second-best model (for chemical-PPI-breast-MDAMB231, 0.31 versus 0.18 (b)). In a and b, the bars show the average performance across five cross-validation test splits for each of the nine chemical datasets. The overlaid points represent performance values from individual data splits (n = 5 per cell line). Each data split contains 20% of samples in the dataset, with each sample corresponding to a perturbation-response instance. Where replicates exist for a given drug, they are treated as independent inputs during training and evaluation. c, PDGrapher recovers ground-truth therapeutic targets at higher rates (evaluated by recall 1–100) compared with competing methods for chemical-PPI datasets. d, Box plots show the distribution of average model rankings across 9 cell lines (n = 9); each dot corresponds to the aggregated ranking value across cross-validation splits and across all metrics for a distinct cell line. A higher value indicates better performance. e, Shown is the difference of shortest-path distances between ground-truth therapeutic genes and predicted genes by PDGrapher and a random reference across nine cell lines. Predominantly negative values indicate that PDGrapher predicts sets of therapeutic genes that are closer in the network to ground-truth therapeutic genes compared with what would be expected by chance (average shortest-path distances across cell lines for PDGrapher versus reference = 2.77 versus 3.11). PDGrapher not only provides accurate predictions for a larger proportion of samples and consistently predicts ground-truth therapeutic targets close to the top of the ranked list but it also predicts gene targets that are closer in the network (measured by the shortest-path distance) to ground-truth targets compared with what would be expected by chance (Fig. In all cell lines, the ground-truth therapeutic targets predicted by PDGrapher are significantly closer to the ground-truth targets compared with what would be expected by chance (Supplementary Table 6). For example, for chemical-PPI-lung-A549, the median distance between the predicted and ground-truth therapeutic targets is 3.0 for both PDGrapher and the random reference. However, the distributions show a statistically significant difference, with a 1-sided Mann–Whitney U-test that yields P < 0.001, an effect size (rank-biserial correlation) of 0.3531 (95% confidence interval (CI), [0.3515, 0.3549]) and a U-statistic of 1.29 × 1011. Similarly, for chemical-PPI-breast-MCF7, the median distance is 3.0 for both groups, yet the distributions are significantly different (P < 0.001, effect size = 0.2160 (95% CI, [0.2146, 0.2174]), U-statistic = 3.91 × 1011) (Supplementary Table 6). This finding suggests that PDGrapher predicts targets in a manner that reflects PPI network structure32. According to the local network hypothesis, which posits that genes in closer network proximity tend to be more functionally similar, PDGrapher's predictions are more functionally related to ground-truth targets than would be expected by chance35,36,37. PDGrapher also shows strong performance across genetic datasets, specifically genetic-PPI-lung-A549, genetic-PPI-breast-MCF7, genetic-PPI-prostate-PC3, genetic-PPI-skin-A375, genetic-PPI-colon-HT29, genetic-PPI-ovary-ES2, genetic-PPI-head-BICR6, genetic-PPI-pancreas-YAPC, genetic-PPI-stomach-AGS and genetic-PPI-brain-U251MG (Extended Data Fig. Briefly, PDGrapher successfully detected ground-truth targets in 0.87% (genetic-PPI-lung-A549), 0.50% (genetic-PPI-breast-MCF7), 0.24% (genetic-PPI-prostate-PC3), 0.38% (genetic-PPI-skin-A375), 0.36% (genetic-PPI-colon-HT29), 1.09% (genetic-PPI-ovary-ES2), 0.54% (genetic-PPI-head-BICR6), 0.11% (genetic-PPI-pancreas-YAPC) and 0.92% (genetic-PPI-brain-U251MG) more samples compared with the second-best competing method (Extended Data Fig. Its ability to effectively predict targets at the top of the ranks is further supported by the metrics recall@1 and recall@10 (Extended Data Fig. PDGrapher achieves the second-best overall rankings (Extended Data Fig. 2d), closely following scGEN, which obtained the highest nDCG values (Extended Data Fig. 2b) but showed weaker performance when evaluating only the top-ranked predicted targets (partially accurate prediction and recall@1). P values of the genetic perturbagen discovery tests are provided in the Source data. PDGrapher and competing methods perform worse on genetic data than on chemical data. This may be due to knockout interventions generating weaker phenotypic signals than small molecule interventions. While gene knockouts are essential for understanding gene function, single-gene knockout studies can offer limited insights into complex cellular processes due to compensatory mechanisms38,39,40. Despite the modest performance in genetic intervention datasets, PDGrapher outperforms competing methods in the combinatorial prediction of therapeutic targets. PDGrapher achieves the best performance in response prediction for both chemical (Extended Data Fig. 1) and genetic perturbation (Extended Data Fig. When using GRNs as proxy causal graphs, PDGrapher has comparable performance with GRNs and PPI networks across both genetic and chemical intervention datasets (Supplementary Figs. One difference is that GRNs were constructed individually for each cell line, which makes leave-cell-out splitting setting prediction particularly challenging. Therefore, we only conducted random splitting setting experiments for GRN datasets. We also used PDGrapher to clarify the mode of action of the chemical perturbagens vorinostat and sorafenib in chemical-PPI-lung-A549 (Supplementary Note 1). PDGrapher shows consistently strong performance on chemical intervention datasets in the leave-cell-out setting (Fig. In this setting, we use the trained models in the random splitting setting for each cell line to predict therapeutic targets in the remaining cell lines. PDGrapher successfully predicts perturbagens that describe the cellular dynamics and shift gene expression phenotypes from a diseased to a treated state in 7.16% (chemical-PPI-breast-MCF7), 6.50% (chemical-PPI-lung-A549), 5.00% (chemical-PPI-breast-MDAMB231), 8.67% (chemical-PPI-prostate-PC3), 7.72% (chemical-PPI-prostate-VCAP), 7.31% (chemical-PPI-skin-A375), 7.08% (chemical-PPI-colon-HT29) and 7.13% (chemical-PPI-cervix-HELA) additional testing samples compared with the second-best competing method (Fig. PDGrapher also outperforms the competing methods in 8 of 9 cell lines by predicting nDCG values that are 0.02 (chemical-PPI-breast-MCF7), 0.01 (chemical-PPI-lung-A549), 0.01 (chemical-PPI-breast-MDAMB231), 0.03 (chemical-PPI-prostate-PC3), 0.02 (chemical-PPI-prostate-VCAP), 0.01 (chemical-PPI-skin-A375), 0.03 (chemical-PPI-colon-HT29) and 0.02 (chemical-PPI-cervix-HELA) higher than those of the second-best competing method (Fig. Its strong performance is further supported by the recall metrics, particularly recall@10 (Fig. Considering the overall performance across different cell lines and metrics, PDGrapher achieves the highest rank, with a median surpassing competing methods (Fig. Combinations of therapeutic targets predicted by PDGrapher in chemical datasets are closer to ground-truth targets than expected by chance (Fig. However, the distributions show a statistically significant difference, with a 1-sided Mann–Whitney U-test yielding P < 0.001, an effect size (rank-biserial correlation) of 0.2191 (95% CI, [0.2182, 0.2200]) and a U-statistic of 2.46 × 1012. Similarly, for chemical-PPI-breast-MCF7, the median distance is 3.0 for both groups, yet the distributions are significantly different (P < 0.001, effect size = 0.2457 (95% CI, [0.2451, 0.2464]), U-statistic = 6.07 × 1012) (Supplementary Table 7). PDGrapher also outperforms existing methods in genetic perturbagen prediction across cell lines, as measured by the top targets on the predicted gene ranks (Supplementary Fig. PDGrapher also shows superior performance in response prediction for both chemical (Supplementary Fig. a,b, PDGrapher shows improved performance when trained on nine chemical perturbation datasets spanning various diseases and evaluated on the remaining eight cell lines. It achieves up to 8.67% more accurately predicted samples in the testing sets compared with the second-best baseline (for example, when trained on chemical-PPI-prostate-PC3, 12.81% versus 4.13% (a)) and an nDCG value of up to 0.03 higher than the second-best baseline (for example, when trained on chemical-PPI-colon-HT29, 0.19 versus 0.16 (b)). In a and b, the bars show the average performance across five cross-validation test splits for each of the nine chemical datasets. The overlaid points represent performance values from individual data splits (n = 5 per cell line). Each data split contains 20% samples in the dataset, with each sample corresponding to a perturbation-response instance. Where replicates exist for a given drug, they are treated as independent inputs during training and evaluation. c, PDGrapher recovers ground-truth therapeutic targets at higher rates (evaluated by recall 1–100) compared with competing methods for chemical-PPI datasets. d, Box plots show the distribution of average model rankings across 9 cell lines (n = 9); each dot corresponds to the aggregated ranking value across cross-validation splits, train cell lines and across all metrics for a distinct cell line. A higher value indicates better performance. Each dot represents a data point for a specific cell line and metrics. e, Shown is the difference of shortest-path distances between ground-truth therapeutic genes and predicted genes by PDGrapher and a random reference across nine cell lines. Predominantly negative values indicate that PDGrapher predicts sets of therapeutic genes that are closer in the network to ground-truth therapeutic genes compared with what would be expected by chance (average shortest-path distances across cell lines for PDGrapher versus random reference = 2.75 versus 3.11). Approaches that train individual models for each perturbagen (such as scGen and CellOT) generally achieve a better perturbagen prediction performance than those that use a single model for all perturbagens (Biolord, GEARS and ChemCPA). However, training individual models becomes infeasible for large-scale datasets with many perturbagens. For example, without parallelization, scGen would require about 8 years to complete the leave-cell-out experiments on the chemical and genetic perturbation data used in this study. Its training is up to 25× faster than scGen and more than 100× faster than CellOT when using the default setting of 100,000 epochs, substantially reducing computational costs. This efficiency highlights a key advantage of PDGrapher. This improved efficiency is due to PDGrapher's approach. Existing methods predict phenotypic responses to perturbations and identify perturbagens indirectly by searching through predicted responses for all candidates. In contrast, PDGrapher directly infers the perturbagen needed to achieve a specific response, learning which perturbations elicit a desired effect. We examined PDGrapher's ability to predict targets of anti-cancer drugs that were not encountered by the model during training time using chemical cell lines with matched healthy data: chemical-PPI-lung-A549, chemical-PPI-breast-MCF7, chemical-PPI-breast-MDAMB231, chemical-PPI-breast-BT20, chemical-PPI-prostate-PC3 and chemical-PPI-prostate-VCAP. PDGrapher was used to predict gene targets to shift these diseased cell lines into their healthy states. Figure 5a shows the recovery of targets of FDA-approved drugs for varying values of K (where K represents the number of predicted target genes considered in the predicted ranked list), indicating that PDGrapher can identify targets of approved anti-cancer drugs not seen during training among the top predictions. a, Performance of PDGrapher in the prediction of unseen approved drug targets to reverse disease effects across all cell lines with healthy counterparts in chemical perturbation datasets. b, Performance of sensitivity analyses evaluated by the percentage of accurately predicted samples for cell lines MDAMB231 and MCF7 under chemical and genetic perturbations, respectively. The PPI network used here is from STRING (string-db.org) with a confidence score for each edge. The edges are filtered by the 0.1, 0.2, 0.3, 0.4 and 0.5 quantiles of the confidence scores as cut-offs, resulting in 5 PPI networks with 625,818, 582,305, 516,683, 443,051 and 296,451 edges, respectively. Data are presented as mean values across five cross-validation data splits per PPI confidence quantile. from the mean (n = 5 computational replicates per quantile). Each point corresponds to performance on a specific data split. c, Performance metrics of the ablation study on PDGrapher's objective function components: PDGrapher-Cycle trained using only the cycle loss, PDGrapher-SuperCycle trained using the supervision and cycle loss, and PDGrapher-Super trained using only the supervision loss, evaluated by percentage of accurately predicted samples. d, Performance metrics of the second ablation study on PDGrapher's input data: PDGrapher—no disease intervention data using only treatment intervention data, and PDGrapher using both disease and treatment intervention data. The disease and treatment intervention data are organized as ‘healthy, mutation, disease' and ‘diseased, drug, treated', respectively. In c and d, bars show the average performance across five cross-validation test splits for each of the nine chemical datasets. The overlaid points represent performance values from individual data splits (n = 5 per cell line). Each data split contains 20% samples in the dataset, with each sample corresponding to a perturbation-response instance. Where replicates exist for a given drug, they are treated as independent inputs during training and evaluation. We analysed lung cancer by comparing the targets predicted by PDGrapher for lung cancer cell lines with the targets of candidate drugs in clinical development, curated from the Open Targets Platform41. This evaluation tested PDGrapher's ability to predict combinatorial chemical perturbagens. We compared the top ten targets predicted by PDGrapher for the A549 lung cancer cell line to ten randomly selected genes. The predicted targets had significantly higher Open Targets scores and more supporting resources than the random genes (Extended Data Fig. Using an Open Targets evidence cut-off score of 0.5, 8 of 10 predicted targets had evidence supporting their association with lung cancer, compared with only 2 of 10 in the random gene set. 42), targeting these predicted genes were not included in the training set but have been identified as potential treatments for NSCLC. We then evaluated PDGrapher's predictions by examining FDA-approved drugs that were not present in the training set of PDGrapher. Specifically, we assessed PDGrapher's performance using the chemical-PPI-lung-A549 dataset, focusing initially on pralsetinib, a targeted cancer therapy primarily used to treat NSCLC43. Pralsetinib is a selective Ret proto-oncogene (RET) kinase inhibitor designed to block the activity of RET proteins that have become aberrantly active due to mutations or fusions. Pralsetinib is known to target 11 key proteins: RET, DDR1, NTRK3, fms-related receptor tyrosine kinase 3 (FLT3), JAK1, JAK2, NTRK1, KDR, platelet derived growth factor receptor-β (PDGFRB), fibroblast growth factor receptor 1 (FGFR1) and FGFR2 (ref. RET, the gene encoding pralsetinib's primary target, was ranked 11th of 10,716 genes in the predicted list. This substantial overlap highlights the potential of the candidate targets identified by PDGrapher for pralsetinib-based lung cancer treatment, given that pralsetinib was not included in the training set of PDGrapher. Next, we examined KDR as a therapeutic target for lung cancer. KDR, also known as VEGFR2, has been identified as a critical therapeutic target in A549 lung adenocarcinoma cells. These cells express KDR at both mRNA and protein levels, facilitating autocrine signalling that promotes tumour cell survival and proliferation28. Activation of KDR enhances tumour angiogenesis and growth by upregulating oncogenic factors such as enhancer of zeste homologue 2 (EZH2), which is associated with increased cell proliferation and migration. Inhibiting KDR has showed promising therapeutic effects, including reduced cell proliferation and induced apoptosis. For instance, KDR inhibitors have been shown to decrease the malignant potential of lung adenocarcinoma cells by downregulating EZH2 expression and increasing sensitivity to chemotherapy29. These findings underscore the importance of KDR as a therapeutic target in A549 lung adenocarcinoma cells, highlighting its role in tumour progression and the potential benefits of its inhibition in cancer treatment strategies. Importantly, PDGrapher has successfully identified KDR among the top 20 predicted targets in chemical-PPI-lung-A549, validating its precision in detecting key therapeutic targets for lung cancer. Given that Open Targets offers more comprehensive evidence for targets currently under development, we conducted a second series of case studies using Open Targets data to evaluate PDGrapher's capability to identify candidate therapeutic targets and drugs. This analysis aims to identify targets for lung cancer. Figure 6a presents a bubble graph that illustrates the union of the top 10 predicted targets to transition cell states from diseased to healthy in the six cell lines of three types of cancer that have available healthy controls. Most predicted targets are supported by drugs, pathology and systemic biology, and somatic mutation databases, which were considered strong evidence sources. Two unique targets, DNA topoisomerase II-α (TOP2A) and cyclin-dependent kinase 2 (CDK2), are predicted exclusively for the lung cancer cell line (Extended Data Fig. TOP2A is ranked as the top predicted target by PDGrapher. This gene encodes a crucial decatenating enzyme that alters DNA topology by binding to two double-stranded DNA molecules, introducing a double-strand break, passing the intact strand through the break, and repairing the broken strand. This mechanism is vital for DNA replication and repair processes. TOP2A could be a potential therapeutic target for anti-metastatic therapy of NSCLC because it promotes metastasis of NSCLC by stimulating the canonical Wnt signalling pathway and inducing epithelial–mesenchymal transition45. Using the predicted target of TOP2A, PDGrapher then identified three drugs, aldoxorubicin, vosaroxin and doxorubicin hydrochloride, as candidate drugs. These drugs were not part of the training dataset of PDGrapher and are in the early stages of clinical development: aldoxorubicin and vosaroxin are in phase II trials (ClinicalTrials.gov), and doxorubicin hydrochloride is in phase I trials but has been shown to improve survival in patients with metastatic or surgically unresectable uterine or soft tissue leiomyosarcoma46. a, Union of the top 10 targets predicted by PDGrapher in lung, breast and prostate cancer. The details of the scoring system are provided in Supplementary Note 2. b, Predicted target rank from PDGrapher in 5 ranges, 1–10, 11–20, 51–60, 101–110, and 1,001–1,010 for lung cancer (chemical-PPI-lung-A549). Given that PDGrapher can rank all genes based on PPI network or GRN data, we assessed two questions: whether top-ranked genes have stronger evidence from Open Targets compared with lower-ranked genes, and what rank threshold should be used to identify reliably predicted genes. The analysis revealed a clear trend: both the number of supporting evidence sources and global scores decrease with increasing rank, validating the predictive accuracy of PDGrapher. Most targets ranked within the top 100 have strong evidence from Open Targets, indicating that a rank threshold of 100 could serve as a cut-off for selecting candidate targets. We conducted an ablation study to evaluate the components of PDGrapher's objective function using chemical datasets. We trained PDGrapher under three configurations: with only the cycle loss (PDGrapher-Cycle), only the supervision loss (PDGrapher-Super) and with both losses combined (PDGrapher-SuperCycle). The experiments were performed in the random splitting setting across all nine PPI chemical datasets. We assessed performance using several metrics, including the percentage of accurately predicted samples (Fig. 8b) and strength of evidence (Extended Data Fig. The results showed that PDGrapher-Super achieves the highest performance in predicting correct perturbagens but performs the worst in reconstructing treated samples. PDGrapher-SuperCycle (the configuration used throughout this study) strikes a balance between these two objectives, achieving competitive performance in predicting therapeutic genes while showing the best performance in reconstructing treated samples from diseased samples after intervening on the predicted genes. This makes PDGrapher-SuperCycle the most effective choice for balancing accuracy in perturbagen prediction with reconstruction fidelity. The findings show that supervision loss is essential for PDGrapher's overall performance. The PDGrapher-Cycle model consistently underperforms in all cell lines and metrics. Although PDGrapher-Super often excels in ranking performance, including cycle loss (in PDGrapher-SuperCycle) proves its value by moderately improving top prediction metrics such as recall@1 and recall@10. In addition, when healthy cell line data are available, the top predictions of PDGrapher-SuperCycle show stronger evidence compared with those of PDGrapher-Super in more than half (four of six) of the cell lines (Extended Data Fig. We chose PDGrapher-SuperCycle for this work because it provides accurate target gene predictions from the top-ranked genes in the predicted list and bases its predictions on the changes they would induce in diseased samples. Recognizing the role of biological pathways in disease phenotypes, PDGrapher-SuperCycle can identify alternative gene targets with close network proximity that may produce similar phenotypic outcomes. The organization of genes with similar functions, where each gene contributes to specific biochemical processes or signalling cascades, allows perturbations in different genes to yield analogous effects47. This function-based interconnectivity implies that targeting different genes with similar functions can achieve therapeutic outcomes, as these genes collectively influence cellular phenotypic states48. Although PDGrapher-SuperCycle shows slightly lower performance than PDGrapher-Super in pinpointing targets (Fig. 5c), it excels in identifying sets of gene targets capable of transitioning cell states from diseased to treated conditions (Supplementary Fig. We conducted four analyses to test the sensitivity of PDGrapher to the causal graph. The first analysis uses five PPI networks constructed with varying edge confidence cut-offs. The PPI network was obtained from STRING (https://string-db.org/)49, which assigns a confidence score to each edge. To create networks with different levels of confidence, we filtered edges based on the quantiles 0.1, 0.2, 0.3, 0.4 and 0.5 of the confidence scores, resulting in 5 networks with decreasing numbers of edges. For this analysis, we selected two cell lines: chemical-PPI-breast-MDAMB231 and genetic-PPI-breast-MCF7. The LINCS perturbation data for each cell line were processed using the five PPI networks (Supplementary Note 3). We trained PDGrapher with one, two and three GNN layers, selecting the best configuration based on the performance of the validation set. 3, PDGrapher performs robustly at all levels of confidence in PPI networks. It maintains stable performance on both chemical and genetic intervention datasets, even as an increasing number of edges is removed from the PPI networks. We created two sparse PPI networks using different edge removal strategies and one synthetic gene expression dataset with increasing levels of latent confounders. We applied two edge removal strategies to the PPI network: removing increasing numbers of either bridge edges or random edges. Results from the edge removal experiments indicate that although bridge edges are structurally critical, their limited number in the PPI graph reduces their overall impact on model predictions (Extended Data Fig. In contrast, the removal of random edges, which include both high-confidence and redundant connections, has a more pronounced effect on performance, highlighting the model's sensitivity to network perturbations (Extended Data Fig. PDGrapher showed stable performance in perturbagen prediction, with only a slight decrease in performance as stronger confounders were introduced (Extended Data Fig. We then evaluated whether PDGrapher can maintain robust performance in the absence of disease intervention data. In our training datasets, some cell lines lacked associated healthy control samples from disease-relevant tissues and cell types. For cell lines with healthy controls, we trained the response prediction module using both intervention datasets. For cell lines without healthy controls, we trained PDGrapher using only treatment intervention data. To evaluate PDGrapher's dependency on healthy control data, we trained the model on cell lines with available disease intervention data under two conditions: one using the disease intervention data for training and one excluding it. The results indicated that the two versions of PDGrapher perform consistently across cell types and data types (chemical and genetic; Fig. In half of the cell lines (four of nine), the model trained without disease intervention data outperformed the model trained with it. This shows that PDGrapher has a weak dependency on healthy control data and can perform well even when such data are unavailable. We formulate phenotype-driven lead discovery as a combinatorial prediction problem for therapeutic targets. Given a diseased sample, the goal is to identify genes that a genetic or chemical perturbagen should target to reverse disease effects and shift the sample towards a treated state that matches the distribution of a healthy state. This requires predicting a combination of gene targets, framing the task as combinatorial prediction. To address this, we introduce PDGrapher. Using a diseased cell state represented by a gene expression signature and a proxy causal graph of gene–gene interactions, PDGrapher predicts candidate target genes to transition cells to the desired treated state. PDGrapher includes two modules: a perturbagen discovery module that proposes a set of therapeutic targets based on the diseased and treated states, and a response prediction module that evaluates the effect of applying the predicted perturbagen to the diseased state. Both modules are GNN models that operate on gene–gene networks, which serve as approximations of noisy causal graphs. We use PPI networks and GRNs as two representations of these noisy causal graphs. PDGrapher predicts perturbagens that shift gene expression from diseased to treated states across 2 evaluation settings (random and leave-cell-out) and 19 datasets involving genetic and chemical interventions. Unlike alternative response prediction methods, which rely on indirect prediction to identify perturbagens, PDGrapher selects candidate gene targets to achieve the desired transformation36,37,50,51,52. PDGrapher has the potential to improve therapeutic lead design and expand the search space for perturbagens. It leverages large datasets of genetic and chemical interventions to identify sets of candidate targets that can shift cell line gene expression from diseased to treated states. By selecting sets of therapeutic targets for intervention instead of a single perturbagen, PDGrapher enhances phenotype-driven lead discovery. PDGrapher's approach to identifying therapeutic targets can enable personalized therapies by tailoring treatments to individual gene expression profiles. Its ability to output multiple genes is particularly relevant for diseases where dependencies among several genes affect treatment efficacy and safety. PDGrapher operates under the assumption that there are no unobserved confounders, a stringent condition that is challenging to validate empirically. Future work could focus on re-evaluating and relaxing this assumption. Another limitation lies in the reliance on PPI networks and GRNs as proxies for causal gene networks, as these networks are inherently noisy and incomplete53,54,55. PDGrapher posits that representation learning can overcome incomplete causal graph approximations. A valuable research direction is to theoretically examine the impact of such approximations, focusing on how they influence the accuracy and reliability of predicted likelihoods. Such analyses could uncover high-level causal variables with therapeutic effects from low-level observations and contribute to reconciling structural causality and representation learning approaches, which generally lack any causal understanding56. We performed two experiments to evaluate the robustness of PDGrapher. First, we tested PDGrapher on a PPI network with weighted edges, progressively removing low-confidence edges to assess its performance under increasing network sparsity. Second, we applied PDGrapher to synthetic datasets with varying levels of missing graph components and confounding factors in gene expression data. In both experiments, PDGrapher maintained stable performance. PDGrapher also showed robust performance across PPI networks with different numbers of edges. Phenotype-driven drug discovery using PDGrapher faces certain limitations, one of which is its reliance on transcriptomic data. Although transcriptomics is broadly applicable, including other data modalities, such as cell morphology screens, could produce more comprehensive models. Cell morphology screens, including cell painting, capture cellular responses by staining organelles and cytoskeletal components, generating image profiles that capture the effects of genetic or chemical perturbations57,58. These screens allow identification of phenotypic signatures that correlate with compound activity, mechanisms of action and potential off-target effects. The recent release of the JUMP Cell Painting dataset59 exemplifies how high-content morphological profiling can complement databases such as CMap and LINCS, creating integrated datasets for phenotype-driven discovery. By integrating multimodal data, including phenotypic layers from transcriptomic and image data, it becomes possible to uncover more comprehensive patterns of compound effects60. Such integration would broaden the scope of PDGrapher, allowing it to capture wider mechanistic insights and support more effective therapeutic discovery61,62. A limitation of our study is the use of NL20 as a control cell line for A549 (refs. Although NL20 is a normal human bronchial epithelial cell line and A549 is a human lung carcinoma cell line derived from the alveolar region, the two cell lines differ in anatomical origin and molecular characteristics. This mismatch could introduce biases in comparative analyses due to variations in baseline gene expression profiles and cellular behaviours. To mitigate this concern, we evaluate PDGrapher's performance across datasets with and without healthy control data. PDGrapher performs consistently regardless of the inclusion of healthy controls, indicating that its predictions are robust to the absence of matched control cells. Ablation analyses showed that incorporating cycle loss improved PDGrapher's performance in top target predictions for five of nine cell lines. On the basis of this improvement, we included the cycle loss in all experiments. Cycle loss helps maintain the robustness and biological relevance of model predictions. PDGrapher learns to predict drug targets that shift cells from a diseased state to a healthy or treated state. This bidirectional approach enforces the fidelity of predicted targets as they must contain sufficient information to reconstruct state B from state A. Cycle loss also serves as a regularizer that penalizes discrepancies between the original input and its reconstruction66. PDGrapher is a GNN approach for combinatorial prediction of perturbations that transition diseased cells to treated states. By leveraging causal reasoning and representation learning on gene networks, PDGrapher identifies perturbagens necessary to achieve specific phenotypic changes. This approach enables the direct prediction of therapeutic targets that can reverse disease phenotypes, bypassing the need for exhaustive response simulations across large perturbation libraries. Its design and evaluation lay the groundwork for future advances in phenotype-based modeling of therapeutic perturbations by improving the precision and scalability of perturbation prediction methods. Uppercase letter \({\mathsf{X}}\) indicates a random variable, and lowercase letter \({\mathsf{x}}\) indicates its corresponding value; bold uppercase \({\boldsymbol{\mathsf{X}}}\) denotes a set of random variables, and lowercase letter \({\boldsymbol{\mathsf{x}}}\) indicates its corresponding values. This section uses terminology and concepts from the framework of casual inference67. Intuitively, given a diseased cell line sample, we would like to predict the set of therapeutic genes that need to be targeted to reverse the effects of disease, that is, the genes that need to be perturbed to shift the cell gene expression state as close as possible to the healthy state. Next, we formalize our problem formulation. Let \({\mathtt{M}}= < {\boldsymbol{\mathsf{E}}},{\boldsymbol{\mathsf{V}}},{\mathcal{F}},P(\boldsymbol{{\mathsf{E}}}) >\) be a structural causal model (SCM; see the description of related works in Supplementary Note 4) associated with causal graph G, where \({\boldsymbol{\mathsf{E}}}\) is a set of exogenous variables affecting the system, \({\boldsymbol{\mathsf{V}}}\) are the system variables, \({\mathcal{F}}\) are structural equations encoding causal relations between variables and \(P({\boldsymbol{\mathsf{E}}})\) is a probability distribution over exogenous variables. Let \({\mathcal{T}}=\{{{\mathtt{T}}}_{1},\ldots ,{{\mathtt{T}}}_{m}\}\) be a dataset of paired healthy and diseased samples (namely, disease intervention data), where each element is a triplet \({\mathtt{T}}= < {{\boldsymbol{\mathsf{v}}}}^{\rm{h}},{\boldsymbol{\mathsf{U}}},{{\boldsymbol{\mathsf{v}}}}^{\rm{d}} >\) with \({{\boldsymbol{\mathsf{v}}}}^{\rm{h}}\in {[0,1]}^{N}\) being normalized gene expression values of a healthy cell line (variable states before perturbation), \({{\boldsymbol{\mathsf{V}}}}_{{\boldsymbol{\mathsf{U}}}}\) being the disease-causing perturbed variable (gene) set in \({\boldsymbol{\mathsf{V}}}\), and \({{\boldsymbol{\mathsf{v}}}}^{\rm{d}}\in {[0,1]}^{N}\) being gene expression values of a diseased cell line (variable states after perturbation). Our goal is to find, for each sample \({\mathtt{T}}= < {{\boldsymbol{\mathsf{v}}}}^{\rm{h}},{\boldsymbol{\mathsf{U}}},{{\boldsymbol{\mathsf{v}}}}^{\rm{d}} >\), the variable set \({{\boldsymbol{\mathsf{U}}}}^{{\prime} }\) with the highest likelihood of shifting variable states from diseased \({{\boldsymbol{\mathsf{v}}}}^{\rm{d}}\) to healthy \({{\boldsymbol{\mathsf{v}}}}^{\rm{h}}\) state. To increase generality, we refer to the desired variable states as treated (\({{\boldsymbol{\mathsf{v}}}}^{\rm{t}}\)). Our goal can then be expressed as: where \({P}^{\;{G}^{{\boldsymbol{\mathsf{U}}}}}\) represents the probability on the graph G mutilated by perturbations in variables in \({\boldsymbol{\mathsf{U}}}\). Under the assumption of no unobserved confounders, the above interventional probability can be expressed as a conditional probability on the mutilated graph \({G}^{{{\boldsymbol{\mathsf{U}}}}^{{\prime} }}\): which under the causal Markov condition is: where \({{\boldsymbol{\mathsf{Pa}}}}_{{{\boldsymbol{\mathsf{V}}}}_{i}}\) represents parents of variable \({{{\mathsf{V}}}}_{i}\) according to graph \({G}^{{{\boldsymbol{\mathsf{U}}}}^{{\prime} }}\) (that is, the mutilated graph upon intervening on variables in \({{\boldsymbol{\mathsf{U}}}}^{{\prime} }\)). Therefore, intervening on the variable set \({{\boldsymbol{\mathsf{U}}}}^{{\prime} }\) modifies the graph used to obtain conditional probabilities and determine the state of variables in \({{\boldsymbol{\mathsf{U}}}}^{{\prime} }\). In the previous section, we drew on the SCM framework to introduce a generic formulation for the task of combinatorial prediction of therapeutic targets. Instead of approaching the problem from a purely causal inference perspective, we draw upon representation learning to approximate the queries of interest to address the limiting real-world setting of a noisy and incomplete causal graph. Formulating our problem using the SCM framework allows for explicit modelling of interventions and formulation of interventional queries (see the description of related works in Supplementary Note 4). Inspired by this principled problem formulation, we next introduce the problem formulation using a representation learning paradigm. We let \(G=({\mathcal{V}},{\mathcal{E}})\) denote a graph with \(| {\mathcal{V}}| =n\) nodes and \(| {\mathcal{E}}|\) edges, which contains partial information on causal relationships between nodes in \({\mathcal{V}}\) and some noisy relationships. Let \({\mathcal{T}}=\{{{\mathtt{T}}}_{1},\ldots ,{{\mathtt{T}}}_{{\mathsf{M}}}\}\) be a dataset with an individual sample being a triplet \({\mathtt{T}}= < {{\bf{x}}}^{\rm{h}},{\mathcal{U}},{{\bf{x}}}^{\rm{d}} >\) with xh ∈ [0, 1]n being the node states (attributes) of a healthy cell sample (before perturbation), \({\mathcal{U}}\) being the set of disease-causing perturbed nodes in \({\mathcal{V}}\), and xd ∈ [0, 1]n being the node states (attributes) of a diseased cell sample (after perturbation). We denote by \({G}^{{\mathcal{U}}}=({\mathcal{V}},{{\mathcal{E}}}^{{\mathcal{U}}})\) the graph resulting from the mutilation of edges in G as a result of perturbing nodes in \({\mathcal{U}}\) (one graph per perturbagen; we avoid using superindices for simplicity). Here again we refer to the desired variable states as treated (xt). Our goal is then to learn a function: Under causal Markov condition, we can factorize \({P}^{{G}^{{{\mathcal{U}}}^{{\prime} }}}\) over graph \({G}^{{{\mathcal{U}}}^{{\prime} }}\): that is, the probability of each node i depending only on its parents \({{\mathcal{PA}}}_{i}\) in graph \({G}^{{{\mathcal{U}}}^{{\prime} }}\). We assume (1) real-valued node states, (2) G is fixed and given, and (3) atomic and non-atomic perturbagens (intervening on individual nodes or sets of nodes). Given that the value of each node should depend only on its parents in the graph \({G}^{{{\mathcal{U}}}^{{\prime} }}\), a message-passing framework appears especially suited to compute the factorized probabilities P. In the SCM framework, the conditional probabilities in equation (3) are computed recursively on the graph, each being an expectation over exogenous variables \(\boldsymbol{\mathsf{E}}\). Therefore, node states of the previous time point are not necessary. To translate this query into the representation learning realm, we discard the existence of noise variables and directly try to learn a function encoding the transition from an initial state to a desired state. An exhaustive approach to solving equation (5) would be to search the space of all potential sets of therapeutic targets \({{\mathcal{U}}}^{{\prime} }\) and score how effective each is in achieving the desired treated state. This is, how many cell response prediction approaches can be used for perturbagen discovery22,23,68. However, with moderately sized graphs, this is highly computationally expensive, if not intractable. Instead, we propose to search for potential perturbagens efficiently with a perturbagen discovery module (ƒp) and a way to score each potential perturbagen with a response prediction module (ƒr). Given that the prediction for each variable is dependent only on its parents in a graph, GNNs appear especially suited for this problem. We can formulate the query of interest under a graph representation learning paradigm as follows: given a graph \(G=({\mathcal{V}},{\mathcal{E}})\), paired sets of node attributes \({\mathcal{X}}=\{{{\bf{X}}}_{1},{{\bf{X}}}_{2},\ldots ,{{\bf{X}}}_{m}\}\) and node labels \({\mathcal{Y}}=\{{{\bf{Y}}}_{1},{{\bf{Y}}}_{2},\ldots ,{{\bf{Y}}}_{m}\}\), where each Y = {y1, …, yn}, with yi ∈ [0, 1], we aim at training a neural message-passing architecture that given node attributes Xi predicts the corresponding node labels Yi. There are, however, differences between our problem formulation and the conventional graph prediction tasks, namely, graph and node classification (summarized in Supplementary Table 13). In node classification, a single graph G is paired with node attributes X, and the task is to predict the node labels Y. Our formulation differs in that we have m paired sets of node attributes \({\mathcal{X}}\) and labels \({\mathcal{Y}}\) instead of a single set, yet they are similar in that there is a single graph in which GNNs operate. In graph classification, a set of graphs \({\mathcal{G}}=\{{G}_{1},\ldots ,{G}_{m}\}\) is paired with a set of node attributes \({\mathcal{X}}=\{{{\bf{X}}}_{1},{{\bf{X}}}_{2},\ldots ,{{\bf{X}}}_{m}\}\) and the task is to predict a label for each graph Y = {y1, …, ym}. Here graphs have a varying structure, and both the topological information and node attributes predict graph labels. In our formulation, a single graph is combined with each node attribute Xi, and the goal is to predict a label for each node, not for the whole graph. PDGrapher is an approach for combinatorial prediction of therapeutic targets composed of two modules. First, a perturbagen discovery module ƒp searches the space of potential gene sets to predict a suitable candidate \({{\mathcal{U}}}^{{\prime} }\). Next, a response prediction module ƒr checks the goodness of the predicted set \({{\mathcal{U}}}^{{\prime} }\), that is, how effective intervening on variables in \({{\mathcal{U}}}^{{\prime} }\) is to shift node states to the desired treated state xt. We optimize our response prediction module ƒr using cross-entropy (CE) loss on known triplets of disease intervention \(< {{\bf{x}}}^{\rm{h}},{\mathcal{U}},{{\bf{x}}}^{\rm{d}} >\) and treatment intervention \(< {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\): We optimize our intervention discovery module ƒp using a cycle loss, ensuring that the response to the predicted intervention set \({{\mathcal{U}}}^{{\prime} }\) closely matches the desired treated state (the first part of equation (7)). In addition, we provide a supervisory signal for predicting \({{\mathcal{U}}}^{{\prime} }\) in the form of cross-entropy loss (the second part of equation (7)). So, the total loss is defined as: We train ƒp and ƒr in parallel and implement early stopping separately (see ‘Experimental set-up' for more details). Trained module ƒp is then used to predict, for each diseased cell sample, which nodes should be perturbed (\({{\mathcal{U}}}^{{\prime} }\)) to achieve a desired treated state (Fig. Our response prediction module ƒr should learn to map pre-perturbagen node values to post-perturbagen node values through learning relationships between connected nodes (equivalent to learning structural equations in SCMs) and propagating the effects of perturbations downstream in the graph (analogous to the recursive nature of query computations in SCMs). Given a disease intervention triplet \(< {{\bf{x}}}^{\rm{h}},{\mathcal{U}},{{\bf{x}}}^{\rm{d}} >\), we propose a neural model operating on a mutilated graph, \({G}^{{\mathcal{U}}}\), where the node attributes are the concatenation of xh and \({{\bf{x}}}_{{\mathcal{U}}}^{{\prime} }\), predicting diseased node values xd. The first element is its gene expression value \({{\bf{x}}}_{i}^{\rm{h}}\) and the second is a perturbation flag, a binary label indicating whether a perturbation occurs at node i. In practice, we embed each node feature into a high-dimensional continuous space by assigning learnable embeddings to each node based on the value of each input feature dimension. Specifically, for each node, we use the binary perturbation flag to assign a d-dimensional learnable embedding, which is different between nodes but shared across samples for each node. To embed the gene expression value \({{\bf{x}}}_{i}^{\rm{h}}\in [0,1]\), we first calculate thresholds using quantiles to assign the gene expression value into one of the B bins. We use the bin index to assign a d-dimensional learnable embedding, which is different between nodes but shared across samples for each node. To increase our model's representation power, we concatenate a d-dimensional positional embedding (a d-dimensional vector initialized randomly following a normal distribution). Concatenating these three embeddings results in an input node representation of dimensionality 3d. For each node \(i\in {\mathcal{V}}\), an embedding zi is computed using a GNN operating on the node's neighbours' attributes. The most general formulation of a GNN layer is: We obtain an embedding zi for node i by stacking K GNN layers. The node embedding \({{\bf{z}}}_{i}\in {\mathbb{R}}\) is then passed to a multilayer feedforward neural network to obtain an estimate of the values of the post-perturbation nodes xd. Our perturbagen prediction module ƒp should learn the nodes in the graph that should be perturbed to shift the node states (attributes) from diseased xd to the desired treated state xt. Given a triplet \(< {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\), we propose a neural model operating on graph \({G}^{{{\mathcal{U}}}^{{\prime} }}\) with node features xd and xt that predicts a ranking for each node, where the top P ranked nodes should be predicted as the nodes in \({{\mathcal{U}}}^{{\prime} }\). In practice, we represent these binary features in a continuous space using the same approach as described for our response prediction module ƒr. For each node \(i\in {\mathcal{V}}\), an embedding zi is computed using a GNN operating on the node's neighbours' attributes. We obtain an embedding zi for node i by stacking K GNN layers. The node embedding \({{\bf{z}}}_{i}\in {\mathbb{R}}\) is then passed to a multilayer feedforward neural network to predict a real-valued number for node i. We implement PDGrapher using PyTorch 1.10.1 (ref. The implemented architecture yields a neural network with the following hyperparameters: number of GNN layers and number of prediction layers. We set the number of prediction layers to two and performed a grid search over the number of GNN layers (one to three layers). We train our model using a 5-fold cross-validation strategy and report PDGrapher's performance resulting from the best-performing hyperparameter setting. We next outline the evaluation set-up, baseline models and statistical tests used to evaluate PDGrapher. Random reference: Given a sample \({\mathtt{T}}= < {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\), the random reference baseline returns N random genes as the prediction of target genes in \({{\mathcal{U}}}^{{\prime} }\), where N is the number of genes in \({{\mathcal{U}}}^{{\prime} }\). The remaining genes are ranked randomly, and N is the number of genes in \({{\mathcal{U}}}^{{\prime} }\). Cancer drug targets: Given a sample \({\mathtt{T}}= < {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\), the cancer targets baseline returns the top N genes from an ordered list where the first M genes are cancer drug targets and the remaining genes are ranked randomly, and N is the number of genes in \({{\mathcal{U}}}^{{\prime} }\). The processing of drug target information is described in ‘Drug-targets information' and ‘Cancer drug and target information' in Supplementary Note 3. Perturbed genes: Given a sample \({\mathtt{T}}= < {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\), the perturbed genes baseline returns the top N genes from an ordered list where the first M genes are all perturbed genes in the training set and the remaining genes are ranked randomly, and N is the number of genes in \({{\mathcal{U}}}^{{\prime} }\). scGen22: scGen is a widely used gold-standard latent variable model for response prediction71,72,73,74. Given a set of observed cell types in control and perturbed states, scGen predicts the response of a new cell type to the perturbagen seen in training. To use scGen as a baseline, we first fit it to our LINCS gene expression data for each dataset type to predict response to perturbagens, training one model per perturbagen (chemical or genetic). Then, given a sample of paired diseased-treated cell line states, \({\mathtt{T}}= < {{\bf{x}}}^{\rm{d}},{{\mathcal{U}}}^{{\prime} },{{\bf{x}}}^{\rm{t}} >\), we compute the response of the cell line with gene expression \({{\bf{x}}}^{{\rm{d}}^{{\prime} }}\) to all perturbagens. The predicted perturbagen is that whose predicted response is closest to xt in R2 score, which quantifies the proportion of variance in treated state explained by the prediction. As scGen trains one model per perturbagen, it needs an exhaustively long training time for datasets with a large number of perturbagens, especially in the leave-cell-out setting. Therefore, we set the maximum training epochs to 100 and only conducted leave-cell-out tests for one split of data for scGen. Biolord33: Biolord can predict perturbagen response for both chemical and genetic datasets. To prevent memory and quota errors, we implemented two filtering steps: (1) instead of storing the entire response gene expression (rGEX) matrix of all input (control) cells for each perturbagen, we only store a vector of the averaged rGEX of the input cells per perturbagen, which is necessary for calculating R2 for evaluation; and (2) during prediction, if the number of control cells exceeds 10,000, we randomly downsample the control cells to 10,000. Similar to scGen, we predict the responses gene expression \({{\bf{x}}}^{{\rm{d}}^{{\prime} }}\) for all perturbagens and use them to calculate R2 to get the rank of predicted perturbagens. ChemCPA23: ChemCPA is specifically designed for chemical perturbation. We followed the official tutorials on GitHub for running this model (https://github.com/theislab/chemCPA), with all parameters set following the authors' recommendations. Data processing was also conducted using the provided scripts. We constructed drug embedding using RDKit with canonical SMILES sequences, as this is the default setting in the model and the tutorial. As the original ChemCPA model lacks functionality to obtain the predicted rGEX for each drug (averaging over the dosages), we developed a custom script to perform this task. These predictions, \({{\bf{x}}}^{{\rm{d}}^{{\prime} }}\), were subsequently used for calculating R2 to get the rank of predicted perturbagens. GEARS34: GEARS is capable of predicting perturbagen responses for genetic perturbation datasets, specifically for predicting the rGEX to unseen perturbagens. However, it is limited to predicting only those genes that are present in the gene network used as prior knowledge for model training. In addition, GEARS cannot process perturbagens with only one sample, so we filtered the data accordingly. After confirming with the authors, we established that GEARS is suitable only for within-cell-line prediction. Consequently, our experiments with GEARS were conducted exclusively within this scenario. CellOT27: CellOT is capable of working with both chemical and genetic datasets. We ran this model by following the official tutorial from GitHub (https://github.com/bunnech/cellot), ensuring that all parameters were set according to the provided guidelines. Due to CellOT's limitation in processing perturbagens with small sample sizes, we filtered the data to retain only those perturbagens with more than five samples or cells. We then used the predicted rGEX \({{\bf{x}}}^{{\rm{d}}^{{\prime} }}\) to calculate R2 and the predicted perturbagen ranks. Similar to scGen, CellOT trains one model per perturbagen, which results in an exhaustively long training time for datasets with a large number of perturbagens. This issue becomes even more pronounced when doing leave-cell-out evaluations. Therefore, for this method, we set the maximum training epochs to 100 and only conduct one split in leave-cell-out tests. We evaluate PDGrapher and competing methods on two different settings. For each cell line, the dataset is split randomly into train and test sets to measure our model performance in an independent and identically distributed setting. To measure model performance on unseen cell lines, we train our model with random splits on one cell line and test on a new cell line. Specifically, for chemical perturbation data, we train a model for each random split per cell line and test it on the entire dataset of the remaining eight cell lines. For genetic data, we train a model for each random split per cell line and test it on the entire dataset of the remaining nine cell lines. For example, with nine cell lines with chemical perturbation (A549, MDAMB231, BT20, VCAP, MCF7, PC3, A375, HT29 and HELA), we conducted an experiment where each split of cell line A549 was used as the training set, and the trained model was tested on the remaining eight cell lines (MDAMB231, BT20, VCAP, MCF7, PC3, A375, HT29 and HELA). Similarly, for cell line MDAMB231, we trained the model on each split of it and tested the model on the other eight cell lines (A549, BT20, VCAP, MCF7, PC3, A375, HT29 and HELA). This process was repeated for all cell lines, providing a comprehensive evaluation of PDGrapher and all competing methods. For all dataset split settings, our model is trained using 5-fold cross-validation, and metrics are reported as the average on the test set. Within each fold, we further split the training set into training and validation sets (8:2) to perform early stopping. We report average sample-wise R2 score and average perturbagen-wise R2 score to measure performance in the prediction of xt. The perturbagen-wise R2 score is adopted from scGen. It is computed as the square of the Pearson correlation of a linear least-squares regression between a set of predicted treated samples \({\hat{{\bf{X}}}}^{\rm{t}}\in {{\mathbb{R}}}^{N\times S}\) and a set of real treated samples \({{\bf{X}}}^{\rm{t}}\in {{\mathbb{R}}}^{N\times S}\) for the same perturbagen. Higher values indicate better performance in predicting the treated sample xt given the diseased sample xd and predicted perturbagen. This is used for evaluating the performance of response prediction. For evaluating perturbagen discovery, when the competing methods cannot predict perturbagen ranks for chemical perturbation data, we first calculate the rank of drugs based on the R2 score. We then build a target gene rank from the drug rank by substituting the drugs with their target genes acquired from DrugBank75 (accessed in November 2022; see details in Supplementary Note 3). A single drug can have multiple target genes, which we place in the rank in random order. As some methods cannot predict unseen drugs, their predicted target gene lists are often short, introducing bias in evaluation. To address this, we shuffle the missing target genes and attach them to the predicted ranks to create a complete rank. To evaluate the performance of our model in ranking predicted therapeutic targets, we use the nDCG, a widely used metric in information retrieval adapted for our setting. The raw DCG score is computed by summing the relevance of each correct target based on its rank in the predicted list, with relevance weighted by a logarithmic discount factor to prioritize higher-ranked interventions. To ensure comparability across datasets or experiments with different numbers of correct interventions, DCG is normalized by the ideal DCG, which represents the maximum possible score for a perfect ranking. This results in nDCG values in the range [0, 1], where higher values indicate better ranking performance and alignment with the ground truth. This metric is particularly suited for our task as it emphasizes the accuracy of top-ranked interventions while accounting for the diminishing importance of lower-ranked predictions. In addition, we report the proportion of test samples for which the predicted therapeutic targets set has at least one overlapping gene with the ground-truth therapeutic targets set (denoted as the percentage of accurately predicted samples). We also calculated the ratio of correct therapeutic targets that appeared in the top 1, top 10 and top 100 predicted therapeutic targets in the predicted rank, denoted as recall@1, recall@10 and recall@100, respectively. To assess the overall performance across all experiments and metrics, we calculated an aggregated metric, averaging all metric values for each method. In the benchmarking experiments, we performed a one-tailed pairwise t-test to evaluate whether PDGrapher significantly outperforms the competing methods. For other experiments, such as ablation studies, we used a two-tailed t-test to determine whether there is a significant difference in performance between the two models. A significance threshold of 0.05 was used for all tests. P values of perturbagen discovery and response prediction tests are presented in the Source data. In the ablation study, we evaluated PDGrapher by optimizing it with only the supervision loss (PDGrapher-Super) and with only the cycle loss (PDGrapher-Cycle) across all chemical datasets. We then compared the perturbagen prediction performance of these submodels with that of PDGrapher (PDGrapher-SuperCycle). To train PDGrapher-Super and PDGrapher-Cycle, for each cell line, we set the number of layers to that which was found optimal for the validation set in the random splitting setting for PDGrapher-SuperCycle. To test the sensitivity of PDGrapher on PPI networks, we used data from STRING (string-db.org), which provides a confidence score for each edge. The method for acquiring and preprocessing the PPI networks from STRING is detailed in Supplementary Note 3. For each cell line, we processed the data using the five PPI networks described in Supplementary Note 3. We optimized PDGrapher using 5-fold cross-validation as described in ‘Evaluation set-up' and optimized the number of GNN layers using the validation set in each split. Dataset with missing components removing bridge edges: this dataset is generated by progressively removing bridge edges from the existing PPI network. Bridge edges are those whose removal disconnects parts of the network. We vary the fraction of bridge edges removed in increments (from zero to one) and, for each fraction, we create a new edge list representing the modified network (Supplementary Table 5). This process ensures that different levels of network sparsity are introduced, affecting the overall structure and connectivity. We pair these networks with gene expression data from chemical-PPI-breast-MDAMB231. Dataset with missing components removing random edges: this dataset is generated by progressively removing random edges from the existing PPI network. We vary the fraction of bridge edges removed in increments ([0, 0.1, … 0.6]) and, for each fraction, we create a new edge list representing the modified network. The number of remaining directed edges in the network upon random edge removal are 273,319, 242,912, 212,525, 182,177, 151,811 and 121,472. The synthetic datasets were constructed with varying levels of confounding bias introduced into the gene expression data. To simulate latent confounder effects, Gaussian noise with distinct means and variances was progressively added to random subsets of genes. Genes were grouped into 50 predefined subsets, each representing a latent confounder group. For each group, a Gaussian distribution was defined, with the mean drawn randomly from a uniform distribution in the range [0.5, 0.5] and the standard deviation [0.1, 0.5]. A fraction ([0.2, 0.4, 0.6, 0.8, 1]) of these subsets was randomly selected for perturbation and, for each gene in these subsets, its expression value was incremented by a value sampled from the respective Gaussian distribution. The perturbed gene expression values were then clamped between zero and one to ensure validity. This strategy ensures that different latent biases are introduced globally to gene expression patterns while maintaining controlled variability. Let \({\mathcal{P}}\) be the set of predicted therapeutic targets, \({\mathcal{R}}\) be the set of ground-truth therapeutic targets, and spd(p, r) be the shortest-path distance between nodes in \({\mathcal{P}}\) and \({\mathcal{R}}\). We measure the closest distance between \({\mathcal{P}}\) and \({\mathcal{R}}\) as: As part of our performance analyses, we measure the network proximity of PDGrapher and competing methods. We compared the distributions of network proximity values using a Mann–Whitney U-test, along with a rank-biserial correlation to measure effect size. To assess the uncertainty of effect sizes, we performed bootstrapping with 1,000 resamples to estimate 95% CIs. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Processed datasets, including cell line gene expression datasets, PPI networks, drug targets and disease-associated genes, are available via the project website at https://zitniklab.hms.harvard.edu/projects/PDGrapher or directly at https://zenodo.org/records/15375990 (ref. Raw gene expression datasets were obtained from https://clue.io/releases/data-dashboard. Disease-associated genes were obtained from COSMIC at https://cancer.sanger.ac.uk/cell_lines/archive-download#:~:text=Complete. Source data are provided with this paper. Python implementation of PDGrapher is available at https://github.com/mims-harvard/PDGrapher (ref. Vincent, F. et al. Phenotypic drug discovery: recent successes, lessons learned and new directions. & Prunotto, M. Opportunities and challenges in phenotypic drug discovery: an industry perspective. Minikel, E. V., Painter, J. L., Dong, C. C. & Nelson, M. R. Refining the impact of genetic evidence on clinical success. Bange, J., Zwick, E. & Ullrich, A. Molecular targets for breast cancer therapy and prevention. Musa, A. et al. A review of Connectivity Map and computational approaches in pharmacogenomics. Rescue of CF airway epithelial cell function in vitro by a CFTR potentiator, VX-770. B. et al. Connectivity mapping: methods and applications. The Library of Integrated Network-based Cellular Signatures (LINCS) NIH program: system-level cataloging of human cells response to perturbations. The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. & Ravi, J. Reconciling multiple connectivity scores for drug repurposing. Guney, E. Reproducible drug repurposing: when similarity does not suffice. Reversal of cancer gene expression correlates with drug efficacy and reveals therapeutic targets. Chen, B. et al. Computational discovery of niclosamide ethanolamine, a repurposed drug candidate that reduces growth of hepatocellular carcinoma cells in vitro and in mice by inhibiting cell division cycle 37 signaling. In silico and in vitro drug screening identifies new therapeutic approaches for Ewing sarcoma. Prediction of drug efficacy from transcriptional profiles with deep learning. Pham, T. H., Qiu, Y., Zeng, J., Xie, L. & Zhang, P. A deep learning framework for high-throughput mechanism-driven phenotype compound screening and its application to COVID-19 drug repurposing. & Theis, F. J. scGen predicts single-cell perturbation responses. Predicting cellular responses to novel drug perturbations at a single-cell resolution. & Bühlmann, P. Two optimal strategies for active learning of causal models from interventional data. Ghassami, A. E., Salehkaleybar, S., Kiyavash, N. & Bareinboim, E. Budgeted experiment design for causal structure learning. 35th International Conference on Machine Learning (eds Dy, J. Agrawal, R., Squires, C., Yang, K., Shanmugam, K. & Uhler, C. ABCD-strategy: budgeted experimental design for targeted causal structure discovery. 22nd International Conference on Artificial Intelligence and Statistics (eds Chaudhuri, K. & Sugiyama, M.) 3400–3409 (Proceedings of Machine Learning Research, 2019). Bunne, C. et al. Learning single-cell perturbation responses using neural optimal transport. Barr, M. P. et al. Vascular endothelial growth factor is an autocrine growth factor, signaling through neuropilin-1 in non-small cell lung cancer. Riquelme, E. et al. VEGF/VEGFR-2 upregulates EZH2 expression in lung adenocarcinoma cells and EZH2 depletion enhances the response to platinum-based and VEGFR-2–targeted therapy. Huynh-Thu, V. A., Irrthum, A., Wehenkel, L. & Geurts, P. Inferring regulatory networks from expression data using tree-based methods. Yuan, B. et al. CellBox: interpretable machine learning for perturbation biology with application to the design of cancer combination therapy. Kamimoto, K. et al. Dissecting cell identity via network inference and in silico gene perturbation. & Nitzan, M. Disentanglement of single-cell data with biolord. Predicting transcriptional outcomes of novel multigene perturbations with gears. Network medicine: a network-based approach to human disease. Identification of disease treatment mechanisms through the multiscale interactome. Mutual interactors as a principle for phenotype discovery in molecular interaction networks. Gu, Z. et al. Role of duplicate genes in genetic robustness against null mutations. Deutscher, D., Meilijson, I., Kupiec, M. & Ruppin, E. Multiple knockout analysis of genetic robustness in the yeast metabolic network. The next-generation Open Targets Platform: reimagined, redesigned, rebuilt. The role of dexmedetomidine in tumor-progressive factors in the perioperative period and cancer recurrence: a narrative review. Gainor, J. F. et al. Pralsetinib for ret fusion-positive non-small-cell lung cancer (arrow): a multi-cohort, open-label, phase 1/2 study. Faraat Ali, G. C. & Kumari, N. Pralsetinib: chemical and therapeutic development with FDA authorization for the management of ret fusion-positive non-small-cell lung cancers. Expression and potential molecular mechanism of TOP2A in metastasis of non-small cell lung cancer. Pautier, P. et al. Doxorubicin–trabectedin with trabectedin maintenance in leiomyosarcoma. Hartwell, L. H., Hopfield, J. J., Leibler, S. & Murray, A. W. From molecular to modular cell biology. Menche, J. et al. Uncovering disease–disease relationships through the incomplete interactome. The string database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest. Bagherian, M. et al. Machine learning approaches and databases for prediction of drug-target interaction: a survey paper. Sparse dictionary learning recovers pleiotropy from human cell fitness screens. Huang, K. et al. Deeppurpose: a deep learning library for drug–target interaction prediction. Hart, G. T., Ramani, A. K. & Marcotte, E. M.How complete are current yeast and human protein-interaction networks? Zitnik, M., Sosič, R., Feldman, M. W. & Leskovec, J. Evolution of resilience in protein interactomes across the tree of life. Schölkopf, B. et al. Toward causal representation learning. Gustafsdottir, S. M. et al. Multiplex cytological profiling assay to measure diverse. Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes. Chandrasekaran, S. N. et al. JUMP Cell Painting dataset: morphological impact of 136,000 chemical and genetic perturbations. Akbarzadeh, M. et al. Morphological profiling by means of the Cell Painting assay enables identification of tubulin-targeting compounds. Using transcriptomics and cell morphology data in drug discovery: the long road to practice. Artificial intelligence foundation for therapeutic science. & Jacobson, M. R. Defects in mitochondrial fission protein dynamin-related protein 1 are linked to apoptotic resistance and autophagy in a lung cancer model. Le, C. F., Yusof, M. Y. Y., Hassan, H. & Sekaran, S. D. In vitro properties of designed antimicrobial peptides that exhibit potent antipneumococcal activity and produce synergism in combination with penicillin. Zhao, S., Song, P., Zhou, G., Zhang, D. & Hu, Y. METTL3 promotes the malignancy of non-small cell lung cancer by N6-methyladenosine modifying SFRP2. A. Unpaired image-to-image translation using cycle-consistent adversarial networks. (Department of Statistics, Harvard University, 2020); https://www.bradyneal.com/Introduction_to_Causal_Inference-Dec17_2020-Neal.pdf Predicting cellular responses to complex perturbations in high-throughput screens. Paszke, A. et al. Automatic differentiation in PyTorch. Fey, M. & Lenssen, J. E. Fast graph representation learning with PyTorch Geometric. Heumos, L. et al. Best practices for single-cell analysis across modalities. Gayoso, A. et al. A Python library for probabilistic analysis of single-cell omics data. Jovic, D. et al. Single-cell RNA sequencing technologies and applications: a brief overview. Luecken, M. D. et al. Benchmarking atlas-level data integration in single-cell genomics. Gonzalez, G. et al. Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks (genetic data). Gonzalez, G. et al. Combinatorial prediction of therapeutic perturbations using causally-inspired neural networks (chemical data). We thank D. Mohorcic for his help with the PDGrapher codebase. We acknowledge the support of NIH R01-HD108794, NSF CAREER 2339524, US DoD FA8702-15-D-0001, ARPA-H BDF programme, awards from Chan Zuckerberg Initiative, Bill & Melinda Gates Foundation INV-079038, Amazon Faculty Research, Google Research Scholar Program, AstraZeneca Research, Roche Alliance with Distinguished Scientists, Sanofi iDEA-iTECH, Pfizer Research, John and Virginia Kaneb Fellowship at Harvard Medical School, Biswas Computational Biology Initiative in partnership with the Milken Institute, Harvard Medical School Dean's Innovation Fund for the Use of Artificial Intelligence, Harvard Data Science Initiative, and Kempner Institute for the Study of Natural and Artificial Intelligence at Harvard University. was supported, in part, by the Summer Institute in Biomedical Informatics at Harvard Medical School. were supported by the ERC Consolidator Grant number 724228 (LEMAN). Any opinions, findings, conclusions or recommendations expressed in this material are those of the authors and do not necessarily reflect the views of the funders. These authors contributed equally: Guadalupe Gonzalez, Xiang Lin. Prescient Design, Genentech, South San Francisco, CA, USA Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA Merck & Co., South San Francisco, CA, USA Yale School of Public Health, New Haven, CT, USA Kempner Institute for the Study of Natural and Artificial Intelligence, Harvard University, Cambridge, MA, USA Broad Institute of MIT and Harvard, Cambridge, MA, USA Harvard Data Science Initiative, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar retrieved, processed and analysed gene expression data, PPI data and disease gene datasets. is currently employed by Genentech and I.H. was employed by Merck & Co. during the study. The other authors declare no competing interests. Nature Biomedical Engineering thanks Ping Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The R2 values are calculated between the predicted and actual gene expression for the top 20 (a), 40 (b), and 80 (c) differentially expressed genes per cell line. Dotted lines represent the average performance across cell lines, dots indicate individual data points, and bars represent the average R2 across five data splits. (a) PDGrapher provides accurate predictions for up to 1.09% more samples in the test set compared to the second-best baseline across Genetic-PPI datasets (genetic-PPI-ovary-ES2: 1.37% vs 0.28%). (b) scGen takes the leading position in nDCG across genetic-PPI datasets. (c) PDGrapher recovers ground-truth therapeutic targets at comparable rates as competing methods for genetic-PPI datasets. (d) PDGrapher has the best overall performance in perturbagen prediction within each cell line, evaluated by the averaged rank over multiple cell lines and metrics. (e-g) Shown is the R2 of the response prediction module of PDGrapher compared to competing baselines for the top 20 (e), 40 (f), and 80 (g) differentially expressed (DE) genes. Performance of sensitivity analyses evaluated by nDCG (a) and recalls (b-d) for datasets genetic-PPI-breast-MCF7 (left) and chemical-PPI-breast-MDAMB231 (right). The PPI used here is from STRING (string-db.org), which includes a confidence score for each edge. The edges are filtered by the 0.1, 0.2, 0.3, 0.4, and 0.5 quantiles of the confidence scores as cutoffs, resulting in five PPI networks with 625,818, 582,305, 516,683, 443,051, and 296,451 edges, respectively. The results of the percentage of accurately predicted samples are shown in Figure 5b. Performance of simulation analyses evaluated by percentage of accurately predicted samples (a) and nDCG (b) for the synthetic datasets with varying levels (0 to 1) of confounding bias introduced into the gene expression data. Gaussian noise, with distinct means and variances, was added progressively to random subsets of genes, simulating latent confounder effects in the treated gene expression data. The intensity of the confounding bias increases as more gene groups (representing network communities) are affected. This approach creates global, controlled variability in the gene expression data, paired with an unperturbed PPI network, allowing for the evaluation of algorithmic performance across different degrees of confounder noise. See Online Methods for more details on data generation. Performance of simulation analyses evaluated by percentage of accurately predicted samples (a) and nDCG (b) for the synthetic datasets with a [0, 0.1, …, 1] fraction of bridge edges removed in the simulated PPI. Bridge edges are those with high connectivity in the network, which, if removed, increase the number of disconnected communities. See more information in Supplementary Table S5. Performance in ablation studies evaluated by percentage of accurately predicted samples (a) and nDCG (b) for the synthetic datasets with a [0, 0.1, … 0.6] fraction of random edges removed in the PPI. See the Method section for more details. Boxplots display the evidence strength (a) and the number of evidence sources (b) for the top 10 predicted targets from PDGrapher versus 10 randomly selected genes. The evidence strengths are the global association scores and overall evidence sources provided by Open Targets. Details of the scoring system are in Supplementary Note 2. The strength of the evidence is the global association scores, which are based on all evidence sources provided by Open Targets. See details of the scoring system in Supplementary Note 2. Scoring system of evidence for target. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Gonzalez, G., Lin, X., Herath, I. et al. Combinatorial prediction of therapeutic perturbations using causally inspired neural networks. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-025-00947-6'>Emerging strategies, applications and challenges of targeting NAD</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 09:31:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Aging (2025)Cite this article Beyond their classical functions as redox cofactors, recent fundamental and clinical research has expanded our understanding of the diverse roles of nicotinamide adenine dinucleotide (NAD) and nicotinamide adenine dinucleotide phosphate (NADP) in signaling pathways, epigenetic regulation and energy homeostasis. Moreover, NAD and NADP influence numerous diseases as well as the processes of aging, and are emerging as targets for clinical intervention. Here, we summarize safety, bioavailability and efficacy data from NAD+-related clinical trials, focusing on aging and neurodegenerative diseases. We discuss the established NAD+ precursors nicotinic acid and nicotinamide, newer compounds such as nicotinamide riboside and nicotinamide mononucleotide, and emerging precursors. We also discuss technological advances including in industrial-scale production and real-time detection, which are facilitating NAD+ research and clinical translation. Finally, we emphasize the need for further large-scale studies to determine optimal dose, administration routes and frequency, as well as long-term safety and interindividual variability in response. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Navas, L. E. & Carnero, A. NAD+ metabolism, stemness, the immune response, and cancer. Signal Transduct. Google Scholar Xiao, W., Wang, R. S., Handy, D. E. & Loscalzo, J. NAD(H) and NADP(H) redox couples and cellular energy metabolism. Redox Signal. Google Scholar Braidy, N. et al. Role of nicotinamide adenine dinucleotide and related precursors as therapeutic targets for age-related degenerative diseases: rationale, biochemistry, pharmacokinetics, and outcomes. Redox Signal. Google Scholar Verdin, E. NAD+ in aging, metabolism, and neurodegeneration. Google Scholar Covarrubias, A. J., Perrone, R., Grozio, A. & Verdin, E. NAD+ metabolism and its roles in cellular processes during ageing. Cell Biol. Google Scholar Lautrup, S., Sinclair, D. A., Mattson, M. P. & Fang, E. F. NAD+ in brain aging and neurodegenerative disorders. Cell Metab. Google Scholar Zhu, Y., Liu, J., Park, J., Rai, P. & Zhai, R. G. Subcellular compartmentalization of NAD+ and its role in cancer: a sereNADe of metabolic melodies. Google Scholar Canto, C., Menzies, K. J. & Auwerx, J. NAD+ metabolism and the control of energy homeostasis: a balancing act between mitochondria and the nucleus. Cell Metab. Google Scholar Stein, L. R. & Imai, S. The dynamic regulation of NAD metabolism in mitochondria. Trends Endocrinol. Google Scholar Lautrup, S. et al. Decreased mitochondrial NAD+ in WRN deficient cells links to dysfunctional proliferation. Google Scholar Amjad, S. et al. Role of NAD+ in regulating cellular and metabolic signaling pathways. Google Scholar Xie, N. et al. NAD+ metabolism: pathophysiologic mechanisms and therapeutic potential. Signal Transduct. Google Scholar Janssens, G. E. et al. Healthy aging and muscle function are positively associated with NAD+ abundance in humans. Google Scholar Basse, A. L. et al. NAMPT-dependent NAD+ biosynthesis controls circadian metabolism in a tissue-specific manner. Natl Acad. Google Scholar Bonkowski, M. S. & Sinclair, D. A. Slowing ageing by design: the rise of NAD+ and sirtuin-activating compounds. Cell Biol. Google Scholar Yaku, K., Okabe, K. & Nakagawa, T. NAD metabolism: implications in aging and longevity. Ageing Res. Google Scholar McReynolds, M. R., Chellappa, K. & Baur, J. A. Age-related NAD+ decline. Google Scholar Peluso, A., Damgaard, M. V., Mori, M. A. S. & Treebak, J. T. Age-dependent decline of NAD+-universal truth or confounded consensus? Nutrients https://doi.org/10.3390/nu14010101 (2021). Wang, H. L. et al. Meeting Summary of The NYO3 5th NO-Age/AD Meeting and the 1st Norway-UK Joint Meeting on Aging and Dementia: recent progress on the mechanisms and interventional strategies. A Biol. Zhang, J., Wang, H. L. & Fang, E. F. in Molecular, Cellular, and Metabolic Fundamentals of Human Aging. E. F. Fang et al.) Ch. Guo, C., Sun, L., Chen, X. & Zhang, D. Oxidative stress, mitochondrial damage and neurodegenerative diseases. Neural Regen. Google Scholar Fang, E. F. Mitophagy and NAD+ inhibit Alzheimer disease. Google Scholar Fang, E. F. et al. NAD+ in aging: molecular mechanisms and translational implications. Trends Mol. Google Scholar Flones, I. H. et al. Mitochondrial complex I deficiency stratifies idiopathic Parkinson's disease. Google Scholar Toker, L. et al. Genome-wide histone acetylation analysis reveals altered transcriptional regulation in the Parkinson's disease brain. Google Scholar Dolle, C. et al. Defective mitochondrial DNA homeostasis in the substantia nigra in Parkinson disease. Google Scholar Hoyland, L. E. et al. Subcellular NAD+ pools are interconnected and buffered by mitochondrial NAD. Article PubMed Google Scholar Yoshino, J., Baur, J. & Imai, S. I. NAD+ intermediates: the biology and therapeutic potential of NMN and NR. Cell Metab. Google Scholar Mills, K. F. et al. Long-term administration of nicotinamide mononucleotide mitigates age-associated physiological decline in mice. Cell Metab. Google Scholar Camacho-Pereira, J. et al. CD38 dictates age-related NAD decline and mitochondrial dysfunction through an SIRT3-dependent mechanism. Cell Metab. Google Scholar Bai, P. et al. PARP-1 inhibition increases mitochondrial metabolism through SIRT1 activation. Cell Metab. Google Scholar Migaud, M. E., Ziegler, M. & Baur, J. Regulation of and challenges in targeting NAD+ metabolism. Cell Biol. Google Scholar Bogan, K. L. & Brenner, C. Nicotinic acid, nicotinamide, and nicotinamide riboside: a molecular evaluation of NAD+ precursor vitamins in human nutrition. Google Scholar Zapata-Perez, R., Wanders, R. J. A., van Karnebeek, C. D. M. & Houtkooper, R. H. NAD+ homeostasis in human health and disease. EMBO Mol. Google Scholar Lautrup, S., Hou, Y., Fang, E. F. & Bohr, V. A. Roles of NAD+ in health and aging. Cold Spring Harb. Kerr, J. S. et al. Mitophagy and Alzheimer's disease: cellular and molecular mechanisms. Trends Neurosci. Google Scholar Fang, E. F. et al. Mitophagy inhibits amyloid-beta and tau pathology and reverses cognitive deficits in models of Alzheimer's disease. Google Scholar Veverova, K. et al. Alterations of human CSF and serum-based mitophagy biomarkers in the continuum of Alzheimer disease. Autophagy https://doi.org/10.1080/15548627.2024.2340408 (2024). Zhang, J., Wang, H. L., Veverova, K., Vyhnalek, M. & Fang, E. F. Identification and potential clinical applications of novel autophagy/mitophagy proteins in the biofluids of Alzheimer's disease patients. Ageing Res. Google Scholar Mattson, M. P. & Arumugam, T. V. Hallmarks of brain aging: adaptive and pathological modification by metabolic states. Cell Metab. Google Scholar Wilson, D. M. III et al. Hallmarks of neurodegenerative diseases. Google Scholar Hou, Y. et al. Ageing as a risk factor for neurodegenerative disease. Article PubMed Google Scholar Hou, Y. et al. NAD+ supplementation normalizes key Alzheimer's features and DNA damage responses in a new AD mouse model with introduced DNA repair deficiency. Natl Acad. Google Scholar Xie, C. et al. Amelioration of Alzheimer's disease pathology by mitophagy inducers identified via machine learning and a cross-species workflow. Google Scholar Zhao, N. et al. Effect of alternating nicotinamide phosphoribosyltransferase expression levels on mitophagy in Alzheimer's disease mouse models. Acta Mol. Basis Dis. Google Scholar Gulaj, E., Pawlak, K., Bien, B. & Pawlak, D. Kynurenine and its metabolites in Alzheimer's disease patients. Med Sci. Google Scholar Giil, L. M. et al. Kynurenine pathway metabolites in Alzheimer's disease. J. Alzheimers Dis. Google Scholar Fathi, M. et al. Dynamic changes in metabolites of the kynurenine pathway in Alzheimer's disease, Parkinson's disease, and Huntington's disease: a systematic review and meta-analysis. Google Scholar Knapskog, A. B. et al. Higher concentrations of kynurenic acid in CSF are associated with the slower clinical progression of Alzheimer's disease. Alzheimers Dement. Google Scholar Knapskog, A. B. et al. Sex-specific associations of kynurenic acid with neopterin in Alzheimer's disease. Alzheimers Res. Google Scholar González-Sánchez, M. et al. Kynurenic acid levels are increased in the CSF of Alzheimer's disease patients. Biomolecules https://doi.org/10.3390/biom10040571 (2020). Jacobs, K. R. et al. Correlation between plasma and CSF concentrations of kynurenine pathway metabolites in Alzheimer's disease and relationship to amyloid-β and tau. Google Scholar Wennström, M. et al. Kynurenic acid levels in cerebrospinal fluid from patients with Alzheimer's disease or dementia with lewy bodies. J. Tryptophan Res. Google Scholar van der Velpen, V. et al. Systemic and central nervous system metabolic alterations in Alzheimer's disease. Alzheimers Res. Google Scholar Xue, C. et al. Tryptophan metabolism in health and disease. Cell Metab. Google Scholar Schwarcz, R., Bruno, J. P., Muchowski, P. J. & Wu, H. Q. Kynurenines in the mammalian brain: when physiology meets pathology. Google Scholar Yulug, B. et al. Combined metabolic activators improve cognitive functions in Alzheimer's disease patients: a randomised, double-blinded, placebo-controlled phase-II trial. Google Scholar Orr, M. E. et al. A randomized placebo-controlled trial of nicotinamide riboside in older adults with mild cognitive impairment. Google Scholar Orr, M. E. et al. Results from a pilot study: the effects of nicotinamide riboside on mild cognitive impairment. Alzheimer's Dement. Google Scholar Grill, J. D. et al. A Phase 2a proof‐of‐concept double‐blind, randomized, placebo‐controlled trial of nicotinamide in early Alzheimer's disease. In Alzheimer's Association International Conference, abstract: 77979 (AAIC, 2023). Bloem, B. R., Okun, M. S. & Klein, C. Parkinson's disease. Google Scholar Dickson, D. W. Parkinson's disease and parkinsonism: neuropathology. Cold Spring Harb. Dickson, D. W. Neuropathology of Parkinson disease. Parkinsonism Relat. Article PubMed Google Scholar Kalia, L. V. & Lang, A. E. Parkinson's disease. Google Scholar Wallings, R. L., Humble, S. W., Ward, M. E. & Wade-Martins, R. Lysosomal dysfunction at the centre of Parkinson's disease and frontotemporal dementia/amyotrophic lateral sclerosis. Trends Neurosci. Google Scholar Navarro-Romero, A., Montpeyo, M. & Martinez-Vicente, M. The emerging role of the lysosome in Parkinson's disease. Cells https://doi.org/10.3390/cells9112399 (2020). Hirsch, E. C. & Hunot, S. Neuroinflammation in Parkinson's disease: a target for neuroprotection? Lancet Neurol. Google Scholar Turconi, G. et al. Nicotinamide riboside first alleviates symptoms but later downregulates dopamine metabolism in proteasome inhibition mouse model of Parkinson's disease. Google Scholar SenGupta, T. et al. Base excision repair causes age-dependent accumulation of single-stranded DNA breaks that contribute to Parkinson disease pathology. Google Scholar Schondorf, D. C. et al. The NAD+ precursor nicotinamide riboside rescues mitochondrial defects and neuronal loss in iPSC and fly models of Parkinson's disease. Article PubMed Google Scholar Chang, K. H. et al. Alternations of metabolic profile and kynurenine metabolism in the plasma of Parkinson's disease. Google Scholar Heilman, P. L. et al. Tryptophan metabolites are associated with symptoms and nigral pathology in Parkinson's disease. Google Scholar Brakedal, B. et al. The NADPARK study: a randomized phase I trial of nicotinamide riboside supplementation in Parkinson's disease. Cell Metab. Google Scholar Gaare, J. J. et al. Nicotinamide riboside supplementation is not associated with altered methylation homeostasis in Parkinson's disease. Google Scholar Berven, H. et al. NR-SAFE: a randomized, double-blind safety trial of high dose nicotinamide riboside in Parkinson's disease. Google Scholar Feldman, E. L. et al. Amyotrophic lateral sclerosis. Google Scholar Lombardo, F. L. et al. A randomized double-blind clinical trial on safety and efficacy of tauroursodeoxycholic acid (TUDCA) as add-on treatment in patients affected by amyotrophic lateral sclerosis (ALS): the statistical analysis plan of TUDCA-ALS trial. Google Scholar Wang, X. et al. Deletion of nampt in projection neurons of adult mice leads to motor dysfunction, neurodegeneration, and death. Google Scholar Harlan, B. A. et al. Enhancing NAD+ salvage pathway reverts the toxicity of primary astrocytes expressing amyotrophic lateral sclerosis-linked mutant superoxide dismutase 1 (SOD1). Google Scholar Blacher, E. et al. Potential roles of gut microbiome and metabolites in modulating ALS in mice. Google Scholar Li, J. Y. et al. Alterations in metabolic biomarkers and their potential role in amyotrophic lateral sclerosis. Google Scholar de la Rubia, J. E. et al. Efficacy and tolerability of EH301 for amyotrophic lateral sclerosis: a randomized, double-blind, placebo-controlled human pilot study. Lateral Scler. Frontotemporal Degener. Article PubMed Google Scholar Han, S., Choi, J. R., Soon Shin, K. & Kang, S. J. Resveratrol upregulated heat shock proteins and extended the survival of G93A-SOD1 mice. Brain Res. Google Scholar Korner, S. et al. Differential sirtuin expression patterns in amyotrophic lateral sclerosis (ALS) postmortem tissue: neuroprotective or neurotoxic properties of sirtuins in ALS? Article PubMed Google Scholar Yoshino, M. et al. Nicotinamide mononucleotide increases muscle insulin sensitivity in prediabetic women. Google Scholar Fan, L., Cacicedo, J. M. & Ido, Y. Impaired nicotinamide adenine dinucleotide (NAD+) metabolism in diabetes and diabetic tissues: Implications for nicotinamide-related compound treatment. J. Diabetes Investig. Google Scholar Galderisi, A. et al. Metabolomics reveals new metabolic perturbations in children with type 1 diabetes. Google Scholar Gürcü, S. et al. Neopterin and biopterin levels and tryptophan degradation in patients with diabetes. Google Scholar Oxenkrug, G., van der Hart, M. & Summergrad, P. Elevated anthranilic acid plasma concentrations in type 1 but not type 2 diabetes mellitus. Google Scholar Orabona, C. et al. Deficiency of immunoregulatory indoleamine 2,3-dioxygenase 1in juvenile diabetes. JCI Insight https://doi.org/10.1172/jci.insight.96244 (2018). Oxenkrug, G. F. Increased plasma levels of xanthurenic and kynurenic acids in type 2 diabetes. Google Scholar Vangipurapu, J., Fernandes Silva, L., Kuulasmaa, T., Smith, U. & Laakso, M. Microbiota-related metabolites and the risk of type 2 diabetes. Diabetes Care 43, 1319–1325 (2020). Google Scholar Qi, Q. et al. Host and gut microbial tryptophan metabolism and type 2 diabetes: an integrative analysis of host genetics, diet, gut microbiome and circulating metabolites in cohort studies. Google Scholar Abedi, S., Vessal, M., Asadian, F. & Takhshid, M. A. Association of serum kynurenine/tryptophan ratio with poor glycemic control in patients with type2 diabetes. J. Diabetes Metab. Google Scholar Moon, J. Y. et al. Gut microbiota and plasma metabolites associated with diabetes in women with, or at high risk for, HIV infection. Google Scholar Dollerup, O. L. et al. A randomized placebo-controlled clinical trial of nicotinamide riboside in obese men: safety, insulin-sensitivity, and lipid-mobilizing effects. Article PubMed Google Scholar Dollerup, O. L. et al. Effects of nicotinamide riboside on endocrine pancreatic function and incretin hormones in nondiabetic men with obesity. Article PubMed Google Scholar Dollerup, O. L. et al. Nicotinamide riboside does not alter mitochondrial respiration, content or morphology in skeletal muscle from obese and insulin-resistant men. Google Scholar Remie, C. M. E. et al. Nicotinamide riboside supplementation alters body composition and skeletal muscle acetylcarnitine concentrations in healthy obese humans. Google Scholar Dunaif, A. et al. Evidence for distinctive and intrinsic defects in insulin action in polycystic ovary syndrome. Google Scholar Polderman, K. H., Gooren, L. J., Asscheman, H., Bakker, A. & Heine, R. J. Induction of insulin resistance by androgens and estrogens. Google Scholar Kane, A. E. et al. Long-term NMN treatment increases lifespan and healthspan in mice in a sex dependent manner. Preprint at bioRxiv https://doi.org/10.1101/2024.06.21.599604 (2024). Lapatto, H. A. K. et al. Nicotinamide riboside improves muscle mitochondrial biogenesis, satellite cell differentiation, and gut microbiota in a twin study. Google Scholar Qiu, Y. et al. NAD+ exhaustion by CD38 upregulation contributes to blood pressure elevation and vascular damage in hypertension. Signal Transduct. Google Scholar Lund, A. et al. Plasma kynurenines and prognosis in patients with heart failure. PLoS ONE 15, e0227365 (2020). Google Scholar Pedersen, E. R. et al. Associations of plasma kynurenines with risk of acute myocardial infarction in patients with stable angina pectoris. Google Scholar Shi, B. et al. Targeting gut microbiota-derived kynurenine to predict and protect the remodeling of the pressure-overloaded young heart. Google Scholar Konishi, M. et al. Impact of plasma kynurenine level on functional capacity and outcome in heart failure - results from Studies Investigating Co-morbidities Aggravating Heart Failure (SICA-HF). Article PubMed Google Scholar de Picciotto, N. E. et al. Nicotinamide mononucleotide supplementation reverses vascular dysfunction and oxidative stress with aging in mice. Aging Cell 15, 522–530 (2016). Google Scholar Abdellatif, M., Bugger, H., Kroemer, G. & Sedej, S. NAD+ and vascular dysfunction: from mechanisms to therapeutic opportunities. J. Lipid Atheroscler. Google Scholar Abdellatif, M., Sedej, S. & Kroemer, G. NAD+ metabolism in cardiac health, aging, and disease. Google Scholar Wang, D. D. et al. Safety and tolerability of nicotinamide riboside in heart failure with reduced ejection fraction. JACC Basic Transl. Google Scholar Fang, E. F. & Tao, J. Targeting on the NAD+-mitophagy axis to treat cardiovascular disease. Aging Med. Google Scholar Canner, P. L. et al. Fifteen year mortality in Coronary Drug Project patients: long-term benefit with niacin. Maher, V. M. et al. Effects of lowering elevated LDL cholesterol on the cardiovascular risk of lipoprotein(a). Matthan, N. R., Giovanni, A., Schaefer, E. J., Brown, B. G. & Lichtenstein, A. H. Impact of simvastatin, niacin, and/or antioxidants on cholesterol metabolism in CAD patients with low HDL. J. Lipid Res. Phan, B. A. et al. Effects of niacin on glucose levels, coronary stenosis progression, and clinical events in subjects with normal baseline glucose levels (<100 mg/dl): a combined analysis of the Familial Atherosclerosis Treatment Study (FATS), HDL-Atherosclerosis Treatment Study (HATS), Armed Forces Regression Study (AFREGS), and Carotid Plaque Composition by MRI during lipid-lowering (CPC) study. Haynes, R. et al. Serious adverse effects of extended-release niacin/laropiprant: results from the Heart Protection Study 2-Treatment of HDL to Reduce the Incidence of Vascular Events (HPS2-THRIVE) Trial. Group, H. T. C. HPS2-THRIVE randomized placebo-controlled trial in 25,673 high-risk patients of ER niacin/laropiprant: trial design, pre-specified muscle and liver outcomes, and reasons for stopping study treatment. Heart J. Teo, K. K. et al. Extended-release niacin therapy and risk of ischemic stroke in patients with cardiovascular disease: the Atherothrombosis Intervention in Metabolic Syndrome with low HDL/High Triglycerides: Impact on Global Health Outcome (AIM-HIGH) trial. Cheung, M. C., Zhao, X. Q., Chait, A., Albers, J. J. & Brown, B. G. Antioxidant supplements block the response of HDL to simvastatin-niacin therapy in patients with coronary artery disease and low HDL. McDermott, M. M. et al. Nicotinamide riboside for peripheral artery disease: the NICE randomized clinical trial. Abdellatif, M. et al. Nicotinamide for the treatment of heart failure with preserved ejection fraction. Gariani, K. et al. Inhibiting poly ADP-ribosylation increases fatty acid oxidation and protects against fatty liver disease. Dellinger, R. W. et al. Nicotinamide riboside and pterostilbene reduces markers of hepatic inflammation in NAFLD: a double-blind, placebo-controlled clinical trial. Article PubMed Pirinen, E. et al. Niacin cures systemic NAD+ deficiency and improves muscle performance in adult-onset mitochondrial myopathy. Cell Metab. Mukherjee, S. et al. Nicotinamide adenine dinucleotide biosynthesis promotes liver regeneration. Dall, M. et al. Hepatocyte-specific perturbation of NAD+ biosynthetic pathways in mice induces reversible nonalcoholic steatohepatitis-like phenotypes. Li, D. J. et al. NAD+-boosting therapy alleviates nonalcoholic fatty liver disease via stimulating a novel exerkine Fndc5/irisin. Zhang, C. et al. The acute effect of metabolic cofactor supplementation: a potential therapeutic strategy against non-alcoholic fatty liver disease. Basse, A. L. et al. Nampt controls skeletal muscle development by maintaining Ca2+ homeostasis and mitochondrial integrity. Frederick, D. W. et al. Loss of NAD homeostasis leads to progressive and reversible degeneration of skeletal muscle. Cell Metab. Chubanava, S. et al. NAD depletion in skeletal muscle does not compromise muscle function or accelerate aging. Cell Metab. Ryu, D. et al. NAD+ repletion improves muscle function in muscular dystrophy and counters global PARylation. de Zelicourt, A. et al. CD38-NADase is a new major contributor to Duchenne muscular dystrophic phenotype. EMBO Mol. Elhassan, Y. S. et al. Nicotinamide riboside augments the aged human skeletal muscle NAD+ metabolome and induces transcriptomic and anti-inflammatory signatures. Ito, K. & Barnes, P. J. COPD as a disease of accelerated lung aging. Article PubMed Lopez-Otin, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. Norheim, K. L. et al. Effect of nicotinamide riboside on airway inflammation in COPD: a randomized, placebo-controlled trial. Aging https://doi.org/10.1038/s43587-024-00758-1 (2024). Ghosh, T. S., Shanahan, F. & O'Toole, P. W. The gut microbiome as a modulator of healthy ageing. Shats, I. et al. Bacteria boost mammalian host NAD metabolism by engaging the deamidated biosynthesis pathway. Cell Metab. Chellappa, K. et al. NAD precursors cycle between host tissues and the gut microbiome. Cell Metab. Kim, L. J. et al. Host-microbiome interactions in nicotinamide mononucleotide (NMN) deamidation. FEBS Lett. Peluso, A. A. et al. Oral supplementation of nicotinamide riboside alters intestinal microbial composition in rats and mice, but not humans. NPJ Aging 9, 7 (2023). Rodionov, D. A. et al. Transcriptional regulation of NAD metabolism in bacteria: genomic reconstruction of NiaR (YrxA) regulon. Nucleic Acids Res. Biagi, E. et al. Gut microbiota and extreme longevity. Chu, X. et al. Nicotinamide adenine dinucleotide supplementation drives gut microbiota variation in Alzheimer's mouse model. Aging Neurosci. Fang, E. F. et al. Nuclear DNA damage signalling to mitochondria in ageing. Cell Biol. Fang, E. F. et al. Defective mitophagy in XPA via PARP-1 hyperactivation and NAD+/SIRT1 reduction. Fang, E. F. et al. NAD+ replenishment improves lifespan and healthspan in ataxia telangiectasia models via mitophagy and DNA repair. Cell Metab. Scheibye-Knudsen, M. et al. Cockayne syndrome group A and B proteins converge on transcription-linked resolution of non-B DNA. Natl Acad. Fang, E. F. et al. NAD+ augmentation restores mitophagy and limits accelerated aging in Werner syndrome. Maynard, S. et al. Lamin A/C impairments cause mitochondrial dysfunction by attenuating PGC1α and the NAMPT-NAD+ pathway. Nucleic Acids Res. Rothblum-Oviatt, C. et al. Ataxia telangiectasia: a review. Orphanet J. Rare Dis. Khanna, K. K., Lavin, M. F., Jackson, S. P. & Mulhern, T. D. ATM, a central controller of cellular responses to DNA damage. Cell Death Differ. Presterud, R. et al. Long-term nicotinamide riboside use improves coordination and eye movements in ataxia telangiectasia. Veenhuis, S. J. G. et al. Nicotinamide riboside improves ataxia scores and immunoglobulin levels in ataxia telangiectasia. Steinbrucker, K., Tiefenthaler, E., Schernthaner, E. M., Jungwirth, J. & Wortmann, S. B. Nicotinamide riboside for ataxia telangiectasia: a report of an early treated individual. Article PubMed Chen, L. & Oshima, J. Werner syndrome. Shamanna, R. A., Croteau, D. L., Lee, J. H. & Bohr, V. A. Recent advances in understanding Werner syndrome. Tian, Y. et al. WRN loss accelerates abnormal adipocyte metabolism in Werner syndrome. Cell Biosci. Shoji, M. et al. Nicotinamide riboside supplementation benefits in patients with Werner syndrome: a double-blind randomized crossover placebo-controlled trial. Aging Cell https://doi.org/10.1111/acel.70093 (2025). Huang, A. R., Jiang, K., Lin, F. R., Deal, J. & Reed, N. S. Hearing loss and dementia prevalence in older adults in the US. Okur, M. N. et al. Long-term NAD+ supplementation prevents the progression of age-related hearing loss in mice. Aging Cell 22, e13909 (2023). Okur, M. N. et al. Short-term NAD+ supplementation prevents hearing loss in mouse models of Cockayne syndrome. NPJ Aging Mech. Williams, P. A. et al. Vitamin B3 modulates mitochondrial vulnerability and prevents glaucoma in aged mice. Pang, H. et al. Aberrant NAD+ metabolism underlies Zika virus-induced microcephaly. Jiang, Y. et al. Treatment of SARS-CoV-2-induced pneumonia with NAD+ and NMN in two mouse models. Cell Discov. Chen, P. M. et al. CD38 reduces mitochondrial fitness and cytotoxic T cell response against viral infection in lupus patients by suppressing mitophagy. Kontani, K., Nishina, H., Ohoka, Y., Takahashi, K. & Katada, T. NAD glycohydrolase specifically induced by retinoic acid in human leukemic HL-60 cells. Identification of the NAD glycohydrolase as leukocyte cell surface antigen CD38. Aksoy, P., White, T. A., Thompson, M. & Chini, E. N. Regulation of intracellular levels of NAD: a novel role for CD38. Young, G. S., Choleris, E., Lund, F. E. & Kirkland, J. Decreased cADPR and increased NAD+ in the Cd38-/- mouse. Chini, C. et al. The NADase CD38 is induced by factors secreted from senescent cells providing a potential link between senescence and age-related cellular NAD+ decline. Chini, C. C. S. et al. CD38 ecto-enzyme in immune cells is induced during aging and regulates NAD+ and NMN levels. Covarrubias, A. J. et al. Senescent cells promote tissue NAD+ decline during ageing via the activation of CD38+ macrophages. Jin, D. et al. CD38 is critical for social behaviour by regulating oxytocin secretion. Migaud, M. E. et al. Efficient and scalable syntheses of nicotinoyl ribosides and reduced nicotinoyl ribosides, modified derivatives thereof, phosphorylated analogs thereof, adenylyl dinucleotide conjugates thereof, and novel crystalline forms thereof. Patent PCT WO2018089830A1 (2021). Brenner, C., Belenky, P. & Bogan, K. L. Yeast strain and method for using the same to produce nicotinamide riboside. Zhang, N. & Sauve, A. A. Synthesis of beta-nicotinamide riboside using an efficient two-step methodology. Nucleic Acid Chem. Article PubMed Zhang, M., Jiang, Y., Su, Z., Sun, W. & Zhan, L. Novel synthesis method of nicotinamide ribose. Sinclair, D. A. & Ear, P. H. Biological production of NAD precursors and analogs. WO Patent 2015/069860 (2015). Jin, C. & Wang, J. Carriers for enzyme or cell immobilization and immobilization method using the carriers. Kameda, A. et al. A novel ATP regeneration system using polyphosphate-AMP phosphotransferase and polyphosphate kinase. Zheng, C., Li, Y., Wu, X., Gao, L. & Chen, X. Advances in the synthesis and physiological metabolic regulation of nicotinamide mononucleotide. Nutrients https://doi.org/10.3390/nu16142354 (2024). Yang, Y., Zhang, N., Zhang, G. & Sauve, A. A. NRH salvage and conversion to NAD+ requires NRH kinase activity by adenosine kinase. Yang, Y., Mohammed, F. S., Zhang, N. & Sauve, A. A. Dihydronicotinamide riboside is a potent NAD+ concentration enhancer in vitro and in vivo. Zapata-Perez, R. et al. Reduced nicotinamide mononucleotide is a new and potent NAD+ precursor in mammalian cells and mice. FASEB J. Giroud-Gerbetant, J. et al. A reduced form of nicotinamide riboside defines a new path for NAD+ biosynthesis and acts as an orally bioavailable NAD+ precursor. Liu, Y. et al. Reduced nicotinamide mononucleotide (NMNH) potently enhances NAD+ and suppresses glycolysis, the TCA cycle, and cell growth. J. Proteome Res. Membrez, M. et al. Trigonelline is an NAD+ precursor that improves muscle function during ageing and is reduced in human sarcopenia. Ren, J. et al. Evidence of brain target engagement in Parkinson's disease and multiple sclerosis by the investigational nanomedicine, CNM-Au8, in the REPAIR phase 2 clinical trials. Vucic, S. et al. Efficacy and safety of CNM-Au8 in amyotrophic lateral sclerosis (RESCUE-ALS study): a phase 2, randomised, double-blind, placebo-controlled trial and open label extension. Writing Committee for the HEALEY ALS Platform Trial; Berryet al. CNM-Au8 in amyotrophic lateral sclerosis: the HEALEY ALS Platform Trial. Wang, H. L. et al. A luminescent-based protocol for NAD+/NADH detection in C. elegans, mice, and human whole blood. STAR Protoc. Zhu, X. H., Lu, M., Lee, B. Y., Ugurbil, K. & Chen, W. In vivo NAD assay reveals the intracellular NAD contents and redox state in healthy human brain and their age dependences. Natl Acad. Yu, Q. et al. A biosensor for measuring NAD+ levels at the point of care. Cambronne, X. & Kraus, W. L. Location, location, location: compartmentalization of NAD+ synthesis and functions in mammalian cells. Trends Biochem. Dziadosz, M. et al. Quantification of NAD+ in human brain with 1H MR spectroscopy at 3 T: comparison of three localization techniques with different handling of water magnetization. Peng, Y., Zhang, Z., He, L., Li, C. & Liu, M. NMR spectroscopy for metabolomics in the living system: recent progress and future challenges. Mevenkamp, J. et al. Development of a 31P magnetic resonance spectroscopy technique to quantify NADH and NAD+ at 3 T. Nat. Matsuyama, R. et al. Stabilization and quantitative measurement of nicotinamide adenine dinucleotide in human whole blood using dried blood spot sampling. Song, Q. et al. The safety and antiaging effects of nicotinamide mononucleotide in human clinical trials: an update. Damgaard, M. V. & Treebak, J. T. What is really known about the effects of nicotinamide riboside supplementation in humans. Airhart, S. E. et al. An open-label, non-randomized study of the pharmacokinetics of the nutritional supplement nicotinamide riboside (NR) and its effects on blood NAD+ levels in healthy volunteers. PLoS ONE 12, e0186459 (2017). Belenky, P., Christensen, K. C., Gazzaniga, F., Pletnev, A. & Brenner, C. Nicotinamide riboside and nicotinic acid riboside salvage in fungi and mammals. Quantitative basis for Urh1 and purine nucleoside phosphorylase function in NAD+ metabolism. Yaku, K. et al. BST1 regulates nicotinamide riboside metabolism via its glycohydrolase and base-exchange activities. Liu, L. et al. Quantitative analysis of NAD synthesis-breakdown fluxes. Cell Metab. Ratajczak, J. et al. NRK1 controls nicotinamide mononucleotide and nicotinamide riboside metabolism in mammalian cells. Mateuszuk, Ł. et al. Reversal of endothelial dysfunction by nicotinamide mononucleotide via extracellular conversion to nicotinamide riboside. Fletcher, R. S. et al. Nicotinamide riboside kinases display redundancy in mediating nicotinamide mononucleotide and nicotinamide riboside metabolism in skeletal muscle cells. Sociali, G. et al. Antitumor effect of combined NAMPT and CD73 inhibition in an ovarian cancer model. Article PubMed Grozio, A. et al. Slc12a8 is a nicotinamide mononucleotide transporter. Damgaard, M. V. et al. Intravenous nicotinamide riboside elevates mouse skeletal muscle NAD+ without impacting respiratory capacity or insulin sensitivity. Hawkins, J. et al. Randomized, placebo-controlled, pilot clinical study evaluating acute Niagen®+ IV and NAD+ IV in healthy adults. Preprint at medRxiv https://doi.org/10.1101/2024.06.06.24308565 (2024). Conlon, N. J. The Role of NAD+ in regenerative medicine. Lee, S. H. et al. Augmentation of NAD+ by dunnione ameliorates imiquimod-induced psoriasis-like dermatitis in mice. Han, K. et al. Boosting NAD preferentially blunts Th17 inflammation via arginine biosynthesis and redox control in healthy and psoriasis subjects. Cell Rep. Med. Zhang, Z., Lu, Y., Qi, J. & Wu, W. An update on oral drug delivery via intestinal lymphatic transport. Acta Pharm. Sun, S. et al. Design and preparation of NMN nanoparticles based on protein-marine polysaccharide with increased NAD+ level in d-galactose induced aging mice model. Colloids Surf. B Biointerfaces 239, 113903 (2024). Zhang, D. et al. Hydroxyapatite-based nano-drug delivery system for nicotinamide mononucleotide (NMN): significantly enhancing NMN bioavailability and replenishing in vivo nicotinamide adenine dinucleotide (NAD+) levels. Trammell, S. A. et al. Nicotinamide riboside is uniquely and orally bioavailable in mice and humans. Martens, C. R. et al. Chronic nicotinamide riboside supplementation is well-tolerated and elevates NAD+ in healthy middle-aged and older adults. Guyton, J. R. & Bays, H. E. Safety considerations with niacin therapy. Winter, S. L. & Boyer, J. L. Hepatic toxicity from large doses of vitamin B3 (nicotinamide). Rader, J. I., Calvert, R. J. & Hathcock, J. N. Hepatic toxicity of unmodified and time-release preparations of niacin. Knip, M. et al. Safety of high-dose nicotinamide: a review. Jenkins, D. J. A. et al. Supplemental vitamins and minerals for CVD prevention and treatment. Ferrell, M. et al. A terminal metabolite of niacin promotes vascular inflammation and contributes to cardiovascular disease risk. Ferrell, M. et al. Publisher correction: a terminal metabolite of niacin promotes vascular inflammation and contributes to cardiovascular disease risk. Diguet, N. et al. Nicotinamide riboside preserves cardiac function in a mouse model of dilated cardiomyopathy. Lv, H. et al. NAD+ metabolism maintains inducible PD-L1 expression to drive tumor immune evasion. Cell Metab. Bi, T. Q. & Che, X. M. Nampt/PBEF/visfatin and cancer. Cancer Biol. Santidrian, A. F. et al. Mitochondrial complex I activity and NAD+/NADH balance regulate breast cancer progression. Chen, A. C. et al. A phase 3 randomized trial of nicotinamide for skin-cancer chemoprevention. Surjana, D., Halliday, G. M., Martin, A. J., Moloney, F. J. & Damian, D. L. Oral nicotinamide reduces actinic keratoses in phase II double-blinded randomized controlled trials. Yiasemides, E., Sivapirabu, G., Halliday, G. M., Park, J. & Damian, D. L. Oral nicotinamide protects against ultraviolet radiation-induced immunosuppression in humans. Nacarelli, T. et al. NAD+ metabolism governs the proinflammatory senescence-associated secretome. Cell Biol. Fang, J. et al. NAD+ metabolism-based immunoregulation and therapeutic potential. Cell Biosci. Morandi, F., Horenstein, A. L. & Malavasi, F. The key role of NAD+ in anti-tumor immune response: an update. Sun, C. et al. Re-equilibration of imbalanced NAD metabolism ameliorates the impact of telomere dysfunction. EMBO J. Wang, H., Lautrup, S., Caponio, D., Zhang, J. & Fang, E. F. DNA damage-induced neurodegeneration in accelerated ageing and Alzheimer's disease. Ummarino, S. et al. NAD Modulates DNA methylation and cell differentiation. Cells https://doi.org/10.3390/cells10112986 (2021). Tomita, T. et al. Sirt1-deficiency causes defective protein quality control. Levine, D. C. et al. NAD+ controls circadian reprogramming through PER2 nuclear translocation to counter aging. Aman, Y. et al. Autophagy in healthy aging and disease. Ludewig, A. H. et al. Pheromone sensing regulates Caenorhabditis elegans lifespan and stress resistance via the deacetylase SIR-2.1. Natl Acad. Lin, J., Handschin, C. & Spiegelman, B. M. Metabolic control through the PGC-1 family of transcription coactivators. Cell Metab. Article PubMed Miwa, S., Kashyap, S., Chini, E. & von Zglinicki, T. Mitochondrial dysfunction in cell senescence and aging. Allen, A. R., Jones, A., LoBianco, F. V., Krager, K. J. & Aykin-Burns, N. Effect of Sirt3 on hippocampal MnSOD activity, mitochondrial function, physiology, and cognition in an aged murine model. Brain Res. Chini, C. C. S., Cordeiro, H. S., Tran, N. L. K. & Chini, E. N. NAD metabolism: role in senescence regulation and aging. Aging Cell 23, e13920 (2024). Pareja-Galeano, H., Sanchis-Gomar, F. & Lucia, A. p16INK4a, NAD+, and sestrins: new targets for combating aging-related chronic illness? Zhang, H. et al. NAD+ repletion improves mitochondrial and stem cell function and enhances life span in mice. Gomes, A. P. et al. Declining NAD+ induces a pseudohypoxic state disrupting nuclear-mitochondrial communication during aging. Ying, X., Zheng, X., Zhang, X., Yin, Y. & Wang, X. Kynurenine in IDO1high cancer cell-derived extracellular vesicles promotes angiogenesis by inducing endothelial mitophagy in ovarian cancer. Hou, Y. et al. NAD+ supplementation reduces neuroinflammation and cell senescence in a transgenic mouse model of Alzheimer's disease via cGAS-STING. Natl Acad. USA https://doi.org/10.1073/pnas.2011226118 (2021). Schmidlin, C. J., Dodson, M. B., Madhavan, L. & Zhang, D. D. Redox regulation by NRF2 in aging and disease. Free Radic. Devereaux, J. et al. Alterations in tryptophan metabolism and de novo NAD+ biosynthesis within the microbiota-gut-brain axis in chronic intestinal inflammation. Dehhaghi, M., Kazemi Shariat Panahi, H. & Guillemin, G. J. Microorganisms, tryptophan metabolism, and kynurenine pathway: a complex interconnected loop influencing human health status. J. Tryptophan Res. Wellman, A. S. et al. Intestinal epithelial sirtuin 1 regulates intestinal inflammation during aging in mice by altering the intestinal microbiota. Article PubMed Minhas, P. S. et al. Restoring hippocampal glucose metabolism rescues cognition across Alzheimer's disease pathologies. Dan, X. et al. Loss of smelling is an early marker of aging and is associated with inflammation and DNA damage in C57BL/6J mice. Aging Cell 22, e13793 (2023). Hamilton, H. L. et al. Nicotinamide adenine dinucleotide precursor supplementation modulates neurite complexity and survival in motor neurons from amyotrophic lateral sclerosis models. Redox Signal. Szot, J. O. et al. A metabolic signature for NADSYN1-dependent congenital NAD deficiency disorder. Gille, A., Bodor, E. T., Ahmed, K. & Offermanns, S. Nicotinic acid: pharmacological effects and mechanisms of action. Gindri, I. M. et al. Evaluation of safety and effectiveness of NAD in different clinical conditions: a systematic review. Article PubMed Schiuma, G., Lara, D., Clement, J., Narducci, M. & Rizzo, R. Nicotinamide adenine dinucleotide: the redox sensor in aging-related disorders. Redox Signal. Avalos, J. L., Bever, K. M. & Wolberger, C. Mechanism of sirtuin inhibition by nicotinamide: altering the NAD+ cosubstrate specificity of a Sir2 enzyme. Roos, J., Zinngrebe, J. & Fischer-Posovszky, P. Nicotinamide mononucleotide: a potential effective natural compound against insulin resistance. Signal Transduct. Zapata-Perez, R. et al. Biotechnological production of reduced and oxidized NAD+ precursors. Food Res. Li, F., Wu, C. & Wang, G. Targeting NAD metabolism for the therapy of age-related neurodegenerative diseases. Watne, L. O. et al. Cerebrospinal fluid quinolinic acid is strongly associated with delirium and mortality in hip-fracture patients. Suhs, K. W. et al. Kynurenine is a cerebrospinal fluid biomarker for bacterial and viral central nervous system infections. Quist-Paulsen, E. et al. High neopterin and IP-10 levels in cerebrospinal fluid are associated with neurotoxic tryptophan metabolites in acute central nervous system infections. De Pinto, V. Renaissance of VDAC: new insights on a protein family at the interface between mitochondria and cytosol. Biomolecules https://doi.org/10.3390/biom11010107 (2021). Magri, A. et al. AAV-mediated upregulation of VDAC1 rescues the mitochondrial respiration and sirtuins expression in a SOD1 mouse model of inherited ALS. Cell Death Discov. Luongo, T. S. et al. SLC25A51 is a mammalian mitochondrial NAD+ transporter. Canto, C. NAD+ precursors: a questionable redundancy. Metabolites https://doi.org/10.3390/metabo12070630 (2022). Kropotov, A. et al. Equilibrative nucleoside transporters mediate the import of nicotinamide riboside and nicotinic acid riboside into human cells. MacKay, D., Hathcock, J. & Guarneri, E. Niacin: chemical forms, bioavailability, and health effects. Article PubMed Chen, M. et al. SLC29A1 and SLC29A2 are human nicotinamide cell membrane transporters. Bahn, A. et al. Identification of a new urate and high affinity nicotinate transporter, hOAT10 (SLC22A13). Mathialagan, S. et al. Nicotinic acid transport into human liver involves organic anion transporter 2 (SLC22A7). Download references is supported by the Cure Alzheimer's Fund (282952, 284930), HELSE SØR-ØST (2020001, 2021021, 2023093), the Research Council of Norway (262175, 334361), Molecule AG/VITADAO (282942), NordForsk Foundation (119986), the National Natural Science Foundation of China (81971327), Akershus University Hospital (269901, 261973, 262960), the Civitan Norges Forskningsfond for Alzheimers sykdom (281931), the Czech Republic-Norway KAPPA program (with M. Vyhnálek, TO01000215), the Rosa sløyfe/Norwegian Cancer Society & Norwegian Breast Cancer Society (207819), Wellcome Leap's Dynamic Resilience Program (jointly funded by Temasek Trust; 104617) and the Alzheimer's Association (PTC-Gene-25-1439553). received funding from the European Union's Horizon Europe research and innovation program through the NADIS project (101073251). is supported by the Norwegian Health Association (43080). is partially supported by the Research Council of Norway through its Centres of Excellence scheme (project number 332713), KLINBEFORSK (26044), the Research Council of Norway (302483) and the Michael J. Fox Foundation (022355). is supported by the Novo Nordisk Foundation Center for Basic Metabolic Research, which is an independent research center at the University of Copenhagen, partially funded by the Novo Nordisk Foundation (NNF23SA0084103). is funded by the South-Eastern Norway Regional Health Authorities (2017095), the Norwegian Health Association (19536, 1513) and by Wellcome Leap's Dynamic Resilience Program (jointly funded by Temasek Trust) (104617). is supported by the Research Council of Norway (326461) and the South-Eastern Norway Regional Health Authority (272931). has received funding from the European Union's Horizon Europe research and innovation program under a grant agreement (NADIS; GA-101073251, with R.Z.-P., C.C., M.D., G.E.J., M.S.K., E.F.F., S.B. and R.H.) and from Research Council of Finland Profi 6 funding (336449) awarded to the University of Oulu. is supported by grants from the Research Council of Norway (325172, 309567, 302314). R.Z.-P. is supported by grants from Ministerio de Ciencia, Innovación y Universidades – Proyectos de Generación de Conocimiento 2023 (PID2023-147560OA-I00) and from Fundación Séneca – Agencia de Ciencia y Tecnología de la Región de Murcia (22011/JLI/22, Ayudas a Proyectos para la Gene-ración de Nuevo Liderazgo Científico Jóvenes Líderes en Investigación 2022). is supported by Marie Skłodowska-Curie HORIZON-MSCA-DN-2021 (NADIS; GA-101073251, with E.P., R.Z.-P., C.C., M.D., G.E.J., M.S.K., E.F.F. and RH), Fondazione Compagnia di San Paolo (Bando Trapezio) and the Italian Ministry of University and Research (PRIN 2022P9RM9M). is supported by ERC CoG (PD-GUT-101003329), AFM Telethon (28568), JPND (01ED2005B), Aligning Science Across Parkinson's (ASAP-000420) and Fondation De France (00147852/WB-2023-51647). is supported by the Japan Agency for Medical Research and Development (JP21bm0804016, JP21jm0210096, JP22ym0126066, JP23ek0109622) and the Ministry of Health, Labor and Welfare of Japan (JPMH21FC1016). is supported by grants from: The Research Council of Norway (288164), Bergen Research Foundation (BFS2017REK05), The KG Jebsen Foundation (SKGJ-MED-023) and the Western Norway Regional Health Authority (F-10229-D11661; IHF). is supported by Akershus University Hospital (263928) and the national key clinical specialties program of Hunan Province (20230146). is supported by the Norwegian Health Association (281956). is funded by a Hevolution/American Federation for Aging Research (AFAR) New Investigator award. These authors contributed equally: Jianying Zhang, He-Ling Wang. Department of Clinical Molecular Biology, University of Oslo and Akershus University Hospital, Lørenskog, Norway Jianying Zhang, He-Ling Wang, Sofie Lautrup & Evandro Fei Fang Key Laboratory of Oral Health Research in Hunan Province, Xiangya School of Stomatology, Central South University, Hu'nan, China Jianying Zhang The Norwegian Centre on Healthy Ageing (NO-Age) and the Norwegian National anti-Alzheimer's Disease (NO-AD) Networks, Oslo, Norway Jianying Zhang, He-Ling Wang, Sofie Lautrup & Evandro Fei Fang Institute of Clinical Medicine, University of Oslo, Oslo, Norway Hilde Loge Nilsen & Geir Selbæk Centre for Embryology and Healthy Development (CRESCO), University of Oslo, Oslo, Norway Hilde Loge Nilsen Department of Microbiology, Oslo University Hospital, Oslo, Norway Hilde Loge Nilsen Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark Jonas T. Treebak Institute of Clinical Medicine, Campus Ahus, University of Oslo, Lørenskog, Norway Leiv Otto Watne Department of Geriatric Medicine, Akershus University Hospital, Lørenskog, Norway Leiv Otto Watne Department of Geriatric Medicine, Oslo University Hospital, Oslo, Norway Geir Selbæk Norwegian National Centre for Ageing and Health, Vestfold Hospital Trust, Tønsberg, Norway Geir Selbæk School of Biomedical Sciences, UNSW Sydney, Sydney, New South Wales, Australia Lindsay E. Wu Department of Cardiology, Akershus University Hospital, Lørenskog, Norway Torbjørn Omland Jebsen Center for Cardiac Biomarkers, University of Oslo, Oslo, Norway Torbjørn Omland Research Unit of Biomedicine and Internal Medicine, Faculty of Medicine, University of Oulu, Oulu, Finland Eija Pirinen Medical Research Center Oulu, Oulu University Hospital and University of Oulu, Oulu, Finland Eija Pirinen Eija Pirinen Research Program for Clinical and Molecular Metabolism, Faculty of Medicine, University of Helsinki, Helsinki, Finland Eija Pirinen Hong Kong Longevity Science Laboratory Limited, Hong Kong Science Park, Pak Shek Kok, Hong Kong SAR, China Tin Cho Cheung & Jun Wang Department of Biomedicine, University of Bergen, Bergen, Norway Mathias Ziegler Department of Clinical Medicine, University of Bergen, Bergen, Norway Ole-Bjørn Tysnes & Charalampos Tzoulis Neuro-SysMed, Department of Neurology, Haukeland University Hospital, Bergen, Norway Ole-Bjørn Tysnes & Charalampos Tzoulis Group of Metabolism and Gene Regulation, UCAM HiTech Sport & Health Innovation Hub, Universidad Católica de Murcia, Guadalupe de Maciascoque, Murcia, Spain Rubén Zapata-Pérez Department of Experimental Medicine, Section of Biochemistry, University of Genova, Genova, Italy Santina Bruzzone Laboratory of Systems Biology and Genetics, Institute of Bioengineering, School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland Carles Canto Mechanisms and Therapy of Genetic Brain Diseases Institut Imagine, INSERM UMR 1163- Paris Cite' University, Paris, France Michela Deleidi Laboratory Genetic Metabolic Diseases, Amsterdam Gastroenterology, Endocrinology, and Metabolism (AGEM), Amsterdam Cardiovascular Sciences (ACS), Amsterdam UMC, University of Amsterdam, Amsterdam, The Netherlands Georges E. Janssens & Riekelt H. Houtkooper Center for Healthy Aging, Department of Cellular and Molecular Medicine, University of Copenhagen, Copenhagen, Denmark Morten Scheibye-Knudsen & Vilhelm A. Bohr Department of Endocrinology, Hematology and Gerontology, Graduate School of Medicine, Chiba University, Chiba, Japan Masaya Koshizaka & Koutaro Yokote Department of Diabetes, Metabolism and Endocrinology, Chiba University Hospital, Chiba, Japan Masaya Koshizaka & Koutaro Yokote UCSF Department of Medicine, San Francisco, CA, USA Eric Verdin Buck Institute for Research on Aging, Novato, CA, USA Eric Verdin Section on DNA Repair, National Institute on Aging, Baltimore, MD, USA Vilhelm A. Bohr Jebsen Center for Translational Research in Parkinson's disease, University of Bergen, Bergen, Norway Charalampos Tzoulis Department of Genetics, Blavatnik Institute, Paul F. Glenn Center for the Biology of Aging, Harvard Medical School, Boston, MA, USA David A. Sinclair Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Writing—original draft preparation: J.Z., H.L.W., S.L., H.L.N., J.T.T., L.O.W., G.S., L.E.W., T.O., E.P., T.C.C., J.W., M.Z., O.-B.T., R.Z.-P., S.B., C.C., M.D., G.E.J., R.H.H., M.S.K., K.Y., M.K., C.T. Writing—review and editing: J.Z., H.L.W., S.L., H.L.N., J.T.T., L.O.W., G.S., L.E.W., T.O., E.P., T.C.C., J.W., M.Z., O.-B.T., R.Z.-P., S.B., C.C., M.D., G.E.J., R.H.H., M.S.K., K.Y., M.K., E.V., V.B., C.T., D.A.S. Correspondence to Evandro Fei Fang. is a co-owner of Fang-S Consultation AS (organization no. 931 410 717) and NO-Age AS (organization no. 933 219 127); has a material transfer agreement with LMITO Therapeutics (South Korea), a CRADA arrangement with ChromaDex (USA), a commercialization agreement with Molecule AG/VITADAO and material transfer agreements with GeneHarbor (Hong Kong) Biotechnologies Limited and Hong Kong Longevity Science Laboratory (Hong Kong); is a consultant for MindRank AI (China), NYO3 (Norway), AgeLab (Vitality Nordic AS, Norway) and Hong Kong Longevity Science Laboratory (Hong Kong). is an owner of NilsenRepairAge Consulting (organization no. 932146797) and NO-Age AS (organization no. 933219127); has a CRADA arrangement with ChromaDex (USA) and is a consultant for Ambr Institute (Norway), MitoTerapeutics (Norway) and AgeLab (Vitality Nordic AS, Norway). is listed as inventor on international patent applications relating to the use of NR as a treatment for PD. These patents have been filed by the Technology Transfer Office ‘Vestlandets Innovasjonsselskap As (VIS)' on behalf of Haukeland University Hospital, Bergen, Norway (PCT/EP2022/067408, PCT/EP2022/067412 and PCT/EP2023/060962). has received honoraria for giving lectures at symposia sponsored by Eisai and Eli Lilly and has participated at advisory board meetings for Eisia, Roche and Eli Lilly concerning the manufacture of disease-modifying therapies for AD. has received institutional research support from ChromaDex (USA). reports a relationship with ChromaDex that includes speaking and lecture fees. is chief scientist at Blue Helix Health AS. have received research support from ChromaDex (USA) via their institution. is a member of the Chromadex Scientific Advisory Board. are inventors and consultants in the Hong Kong Longevity Science Laboratory. is a consultant, inventor, board member and, in some cases, an investor in Atai (2021–present) I; Galilei Biosciences SIRT6 activators (2014–present) F, I, E, A, B; InsideTracker (Segterra), Cambridge, MA I, A, IP (2011–present) B (2011–2017); Zymo Research Irvine, CA, molecular biology (2017–present) A; MetroBiotech, an EdenRoc company also known as Bauhaus; NAD boosters (2015–present) F, I, E, A, B (EdenRoc), IP; Life Biosciences Boston, and affiliates, Cantata Bio (formerly Dovetail Genomics and Arc Bio), Delavie Sciences (2017–present) F, I, E, A, B, IP; Levels (2021–present) E, A; Bold Capital (2021–present) E, A; Falcon Edge Capital (2021–present) E; AFAR (American Federation for Aging Research) B, A, F; Life Extension Advocacy Foundation (LEAF) A; Animal Biosciences (2017–present) I, E; Immetas (2018–present) F, I, E, A, B; Jumpstart Fertility, a Life Biosciences Company, F, I, E, A, B, IP; Fully Aligned Co, F, E, B, IP. F, founder; I, investor; E, equity; A, advisor/consultant; B, board of directors; IP, inventor on licensed patents; L, funding for laboratory (for further information about the affiliations of D.A.S., including those judged not relevant to this work, see https://sinclair.hms.harvard.edu/david-sinclairs-affiliations/). is a scientific advisor and shareholder in Metro Biotech and Jumpstart Fertility. The other authors declare no competing interests. Nature Aging thanks Eduardo Chini, Joseph Baur and Shinghua Ding for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Table 1. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zhang, J., Wang, HL., Lautrup, S. et al. Emerging strategies, applications and challenges of targeting NAD+ in the clinic. Nat Aging  (2025). Download citation Received: 27 August 2024 Accepted: 23 July 2025 Published: 09 September 2025 DOI: https://doi.org/10.1038/s43587-025-00947-6 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Aging (Nat Aging) ISSN 2662-8465 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02772-z'>Antibody–bottlebrush prodrug conjugates for targeted cancer therapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 09:25:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Biotechnology (2025)Cite this article Antibody–drug conjugates (ADCs) are effective targeted therapeutics but are limited in their ability to incorporate less-potent payloads, varied drug mechanisms of action, different drug release mechanisms and tunable drug-to-antibody ratios. Here we introduce a technology to overcome these limitations called ‘antibody–bottlebrush prodrug conjugates' (ABCs). An ABC consists of an IgG1 monoclonal antibody covalently conjugated to the terminus of a compact bivalent bottlebrush prodrug that has payloads bound through cleavable linkers and polyethylene glycol branches. This design enables the synthesis of ABCs with tunable average drug-to-antibody ratios up to two orders of magnitude greater than those of traditional ADCs. We demonstrate the functional flexibility and manufacturing efficiency of this technology by synthesizing more than 10 different ABCs targeting either HER2 or MUC1 using drugs with potencies spanning several orders of magnitude as well as imaging agents for ABC visualization and photocatalysts for proximity-based labeling of the ABC interactome. ABCs display high target engagement, high cell uptake and improved efficacy in tumor models compared to conventional HER2-targeted ADCs, suggesting promise for clinical translation. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data supporting the findings of this study are available within the article, in its supplementary information and on figshare (https://doi.org/10.6084/m9.figshare.29414048)90 and can be obtained from the corresponding authors upon reasonable request. Source data are provided with this paper. Siegel, R. L., Miller, K. D., Fuchs, H. E. & Jemal, A. CA Cancer J. Clin. Google Scholar Wang, Z., Li, H., Gou, L., Li, W. & Wang, Y. Antibody–drug conjugates: recent advances in payloads. Google Scholar Hobson, A. D. Chapter One - Antibody drug conjugates beyond cytotoxic payloads. Google Scholar Tarantino, P., Ricciuti, B., Pradhan, S. M. & Tolaney, S. M. Optimizing the safety of antibody–drug conjugates for patients with solid tumours. Google Scholar Dumontet, C., Reichert, J. M., Senter, P. D., Lambert, J. M. & Beck, A. Antibody–drug conjugates come of age in oncology. Google Scholar Chalouni, C. & Doll, S. Fate of antibody-drug conjugates in cancer cells. Google Scholar Tashima, T. Delivery of drugs into cancer cells using antibody–drug conjugates based on receptor-mediated endocytosis and the enhanced permeability and retention effect. Google Scholar ADCs' revival. Tarantino, P. et al. Antibody–drug conjugates: smart chemotherapy delivery across tumor histologies. CA Cancer J. Clin. Google Scholar Beck, A., Goetsch, L., Dumontet, C. & Corvaïa, N. Strategies and challenges for the next generation of antibody–drug conjugates. Google Scholar Z., Modi, S. & Chandarlapaty, S. Unlocking the potential of antibody–drug conjugates for cancer therapy. Google Scholar Gan, H. K., van den Bent, M., Lassman, A. & Scott, A. M. Antibody–drug conjugates in glioblastoma therapy: the right drugs to the right cells. Google Scholar Fu, Z., Li, S., Han, S., Shi, C. & Zhang, Y. Antibody drug conjugate: the ‘biological missile' for targeted cancer therapy. Signal Transduct. Google Scholar Tong, J. T. W., Harris, P. W. R., Brimble, M. A. An insight into FDA approved antibody-drug conjugates for cancer therapy. Google Scholar Criscitiello, C., Morganti, S. & Curigliano, G. Antibody–drug conjugates in solid tumors: a look into novel targets. Google Scholar Nagayama, A., Ellisen, L. W., Chabner, B. & Bardia, A. Antibody–drug conjugates for the treatment of solid tumors: clinical experience and latest developments. Google Scholar Jabbour, E., Paul, S. & Kantarjian, H. The clinical development of antibody–drug conjugates—lessons from leukaemia. Google Scholar Yu, B. & Liu, D. Antibody-drug conjugates in clinical trials for lymphoid malignancies and multiple myeloma. Google Scholar Zhao, P. et al. Recent advances of antibody drug conjugates for clinical applications. Google Scholar Nessler, I., Menezes, B. & Thurber, G. M. Key metrics to expanding the pipeline of successful antibody–drug conjugates. Trends Pharmacol. Google Scholar Mckertish, C. M. & Kayser, V. Advances and limitations of antibody drug conjugates for cancer. Google Scholar Su, Z. et al. Antibody–drug conjugates: recent advances in linker chemistry. Google Scholar Abelman, R. O., Wu, B., Spring, L. M., Ellisen, L. W. & Bardia, A. Mechanisms of resistance to antibody–drug conjugates. Google Scholar Nguyen, T. D., Bordeau, B. M. & Balthasar, J. P. Mechanisms of ADC toxicity and strategies to increase ADC tolerability. Google Scholar Chudasama, V., Maruani, A. & Caddick, S. Recent advances in the construction of antibody–drug conjugates. Google Scholar Lyon, R. P. et al. Reducing hydrophobicity of homogeneous antibody-drug conjugates improves pharmacokinetics and therapeutic index. Google Scholar Adams, E., Wildiers, H., Neven, P. & Punie, K. Sacituzumab govitecan and trastuzumab deruxtecan: two new antibody–drug conjugates in the breast cancer treatment landscape. Google Scholar Goldenberg, D. M. & Sharkey, R. M. Sacituzumab govitecan, a novel, third-generation, antibody-drug conjugate (ADC) for cancer therapy. Expert Opin. Google Scholar Modi, S. et al. Trastuzumab deruxtecan in previously treated HER2-positive breast cancer. Google Scholar Rubahamya, B., Dong, S. & Thurber, G. M. Clinical translation of antibody drug conjugate dosing in solid tumors from preclinical mouse data. Google Scholar Kukkar, D., Kukkar, P., Kumar, V., Hong, J. Deep, recent advances in nanoscale materials for antibody-based cancer theranostics. Google Scholar Yurkovetskiy, A. V. et al. Dolaflexin: a novel antibody–drug conjugate platform featuring high drug loading and a controlled bystander effect. Google Scholar Müllner, M. Molecular polymer bottlebrushes in nanomedicine: therapeutic and diagnostic applications. Google Scholar A. et al. Core-clickable PEG-branch-azide bivalent-bottle-brush polymers by ROMP: grafting-through and clicking-to. Google Scholar Golder, M. R. et al. Reduction of liver fibrosis by rationally designed macromolecular telmisartan prodrugs. Google Scholar Bhagchandani, S. H. et al. Engineering kinetics of TLR7/8 agonist release from bottlebrush prodrugs enables tumor-focused immune stimulation. Google Scholar Liao, L. et al. A convergent synthetic platform for single-nanoparticle combination cancer therapy: ratiometric loading and controlled release of cisplatin, doxorubicin, and camptothecin. Google Scholar Kolb, H. C., Finn, M. G. & Sharpless, K. B. Click chemistry: diverse chemical function from a few good reactions. Google Scholar Sletten, E. M. & Bertozzi, C. R. Bioorthogonal chemistry: fishing for selectivity in a sea of functionality. Google Scholar Agard, N. J., Prescher, J. & Bertozzi, C. R. A strain-promoted [3 + 2] azide-alkyne cycloaddition for covalent modification of biomolecules in living systems. Google Scholar Thirumurugan, P., Matosiuk, D. & Jozwiak, K. Click chemistry for drug development and diverse chemical–biology applications. Google Scholar Ogba, O. M., Warner, N. C., O'Leary, D. J. & Grubbs, R. H. Recent advances in ruthenium-based olefin metathesis. Google Scholar Fu, L., Zhang, T., Fu, G. & Gutekunst, W. R. Relay conjugation of living metathesis polymers. Google Scholar Knall, A. C. & Slugovc, C. Inverse electron demand Diels–Alder (iEDDA)-initiated conjugation: a (high) potential click chemistry scheme. Google Scholar Oliveira, B. L., Guo, Z. & Bernardes, G. J. L. Inverse electron demand Diels–Alder reactions in chemical biology. Google Scholar Blackman, M. L., Royzen, M. & Fox, J. M. Tetrazine ligation: fast bioconjugation based on inverse-electron-demand Diels–Alder reactivity. Google Scholar Oh, D. Y. & Bang, Y. J. HER2-targeted therapies—a role beyond breast cancer. Google Scholar Nakada, T., Sugihara, K., Jikoh, T., Abe, Y. & Agatsuma, T. The latest research and development into the antibody–drug conjugate, [fam-] trastuzumab deruxtecan (DS-8201a), for HER2 cancer therapy. Google Scholar Hou, Y. & Lu, H. Protein PEPylation: a new paradigm of protein–polymer conjugation. Google Scholar Chen, P., Yun, W., Sun, T., Lin, J. & Zhang, K. Enabling safer, more potent oligonucleotide therapeutics with bottlebrush polymer conjugates. Google Scholar Yu, X. et al. Reducing affinity as a strategy to boost immunomodulatory antibody agonism. Google Scholar Oostindie, S. C., Lazar, G. A., Schuurman, J. & Parren, P. W. H. I. Avidity in antibody effector functions and biotherapeutic drug design. Google Scholar Gall, V. A. et al. Trastuzumab increases HER2 uptake and cross-presentation by dendritic cells. Google Scholar B. et al. Microenvironment mapping via Dexter energy transfer on immune cells. Vohidov, F. et al. Design of BET inhibitor bottlebrush prodrugs with superior efficacy and devoid of systemic toxicities. Zafar, H., Liu, B., Nguyen, H. V. T. & Johnson, J. A. Caspase-3-responsive, fluorogenic bivalent bottlebrush polymers. ACS Macro Lett. Tsao, L. et al. Effective extracellular payload release and immunomodulatory interactions govern the therapeutic effect of trastuzumab deruxtecan (T-DXd). Article PubMed Central Pyzik, M., Kozicky, L. K., Gandhi, A. K. & Blumberg, R. S. The therapeutic age of the neonatal Fc receptor. Akaiwa, M., Dugal-Tessier, J. A. Antibody–drug conjugate payloads; study of auristatin derivatives. Junutula, J. R. et al. Site-specific conjugation of a cytotoxic drug to an antibody improves the therapeutic index. Ejigah, V. et al. Approaches to improve macromolecule and nanoparticle accumulation in the tumor microenvironment by the enhanced permeability and retention effect. Tolcher, A. W. et al. Randomized phase II study of BR96-doxorubicin conjugate in patients with metastatic breast cancer. Hendrikx, J. J. M. A. et al. Fixed dosing of monoclonal antibodies in oncology. Li, Y. et al. Targeted immunotherapy for HER2-low breast cancer with 17p loss. Burslem, G. M. & Crews, C. M. Proteolysis-targeting chimeras as therapeutics and tools for biological discovery. Dale, B. et al. Advancing targeted protein degradation for cancer therapy. Wu, T. et al. Targeted protein degradation as a powerful research tool in basic biology and drug target discovery. Li, X. & Song, Y. Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy. Raina, K. et al. PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer. Noblejas-López, M. et al. Activity of BET-proteolysis targeting chimeric (PROTAC) compounds in triple negative breast cancer. Sun, X. et al. A chemical approach for global protein knockdown from mice to non-human primates. Noblejas-López, M. et al. MZ1 co-operates with trastuzumab in HER2 positive breast cancer. Dragovich, P. S. Degrader-antibody conjugates. Chen, W. et al. MUC1: structure, function, and clinic application in epithelial cancers. Cheever, M. A. et al. The prioritization of cancer antigens: a National Cancer Institute pilot project for the acceleration of translational research. Liu, B. et al. An organometallic swap strategy for bottlebrush polymer–protein conjugate synthesis. Hamblett, K. J. et al. Effects of drug loading on the antitumor activity of a monoclonal antibody drug conjugate. Bai, S. et al. Cylindrical polymer brushes-anisotropic unimolecular micelle drug delivery system for enhancing the effectiveness of chemotherapy. Rabanel, J. M. et al. Deep tissue penetration of bottle-brush polymers via cell capture evasion and fast diffusion. Strasser, P. et al. Degradable bottlebrush polypeptides and the impact of their architecture on cell uptake, pharmacokinetics, and biodistribution in vivo. Qi, Y. et al. A brush-polymer/exendin-4 conjugate reduces blood glucose levels for up to five days and eliminates poly(ethylene glycol) antigenicity. Gil Alvaradejo, G. et al. Polyoxazoline-based bottlebrush and brush-arm star polymers via ROMP: syntheses and applications as organic radical contrast agents. ACS Macro Lett. Shieh, P., Nguyen, H. V. T. & Johnson, J. Tailored silyl ether monomers enable backbone-degradable polynorbornene-based linear, bottlebrush, and star copolymers through ROMP. Trowbridge, A. D. et al. Small molecule photocatalysis enables drug target identification via energy transfer. Oakley, J. V. et al. Radius measurement via super-resolution microscopy enables the development of a variable radii proximity labeling platform. Pan, C., Knutson, S. D., Huth, S. W. & MacMillan, D. W. C. µMap proximity labeling in living cells reveals stress granule disassembly mechanism. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Demichev, V. et al. dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. Tyanova, S. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Liu, B. et al. Data for ‘antibody-bottlebrush prodrug conjugates for targeted cancer therapy'. Download references We thank the National Institutes of Health (2R01CA220468-06A1) and the Yosemite–American Cancer Society Award (YACS-24-1343208-01-YACS) for support. thanks the Ludwig Center of the Koch Institute for Integrative Cancer Research at MIT for a postdoctoral fellowship and the Koch Institute Frontier Research Program for support. P.M. would like to acknowledge support from the National Science Foundation Graduate Research Fellowship Program (NSF-GRFP, grant no. are thankful for the financial support provided by the National Institutes of Health, National Institute of General Medical Sciences (R35GM134897-04); the Princeton Catalysis Initiative; Princeton University; and kind gifts from Merck, Janssen, Bristol Myers Squibb, Genentech, Genmab and Pfizer. N.J. is supported by the US Department of Defense (DOD award W81XWH-22-1-1122). We are grateful for the use of open-access tools, including DIA-NN, Perseus and BioGRID. Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA, USA Bin Liu, Hung V.-T. Nguyen, Aiden X. Wang, Valerie Lensch, Yutong Dai, Piper L. MacNicol, Yuyan Wang, Wencong Wang, Sachin Bhagchandani, Peyton Shieh, Samantha L. Kristufek & Jeremiah A. Johnson Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA, USA Bin Liu, Sachin Bhagchandani & Jeremiah A. Johnson Research Laboratory of Electronics, Massachusetts Institute of Technology, Cambridge, MA, USA Window Therapeutics, Boston, MA, USA Hung V.-T. Nguyen, Yivan Jiang & Sanjana Hemdev Thayer School of Engineering, Dartmouth College, Hanover, NH, USA Hung V.-T. Nguyen Department of Materials Science and Engineering, Massachusetts Institute of Technology, Cambridge, USA Zehao Sun Merck Center for Catalysis at Princeton University, Princeton, NJ, USA Zane H. Boyer, Philip A. Raftopoulos & David W. C. MacMillan Gene Expression Laboratory, Salk Institute for Biological Studies, La Jolla, CA, USA Nidhi Jyotsana, Michael Downes & Ronald M. Evans Division of Comparative Medicine, Massachusetts Institute of Technology, Cambridge, MA, USA Broad Institute of MIT and Harvard, Massachusetts Institute of Technology, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project idea. designed and performed the polymerization and bioconjugation. synthesized and characterized the macromonomers. performed the in vitro experiments. performed the in vivo experiments. assisted with the experiments. performed µMap optimization and proteomic analysis, under the guidance of D.W.C.M. provided the site-specific thiol-mAb. supervised the research. analyzed the data and wrote the manuscript, and all authors discussed and commented on it. Correspondence to Jeremiah A. Johnson. are shareholders of Window Therapeutics. declares an ownership interest in Dexterity Pharma, which has commercialized materials used in this work. The remaining authors declare no competing interests. Nature Biotechnology thanks Charles Dumontet and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Non-reducing SDS-PAGE analysis of model IgG1-based ABCs as a function of synthesis stoichiometry. BAR = average brush–antibody ratio. b. Non-reducing SDS-PAGE analysis of Trastuzumab-based ABCs as a function of synthesis stoichiometry. c. Flow cytometry histograms showing Cy5.5-HER2 uptake into HER2 + BT-474 cells as a function of synthesis stoichiometry (that is, the constructs are present as a mixture of BAR values). The x-axis represents the Cy5.5 fluorescence intensity. Since the number of Cy5.5 dyes per ABC increases with BAR, histograms are normalized to the total Cy5.5 loading for each construct to enable comparison (normalized to 40 µg/mL Cy5.5-BP, 1 h). d. Flow cytometry histograms for the same constructs shown in panel c normalized by antibody dose (normalized to 10 µg/mL mAb, 1 h). The x-axis represents the Cy5.5 fluorescence intensity. e. and f. Flow cytometry histograms for BT-474 cell uptake of isolated Cy5.5-HER2 ABCs with different BAR values (25 µg/mL, 1 h) (that is, ABCs with each BAR were separated from the synthesis mixture). Results are presented as mean ± SEM (n = 3 biological replicates). Statistical analysis was done using a 2-tailed t-test. For these statistical tests, **** denotes P < 0.001. a. Non-reducing SDS-PAGE analysis of site-specifically conjugated Cy5.5-HER ABCs. Two independent experiments were performed with similar results. Flow cytometry histograms showing enhanced BT474 cell uptake for site-specific Cy5.5-HER2 ABC compared to BPD alone. The x-axis represents the Cy5.5 fluorescence intensity. c. and d. Quantification of cell uptake based on mean fluorescence intensity from flow cytometry (n = 3 biological replicates). e. and f. Flow cytometry histograms (The x-axis represents the Cy5.5 fluorescence intensity) and quantification, respectively, for cell uptake studies comparing site-specific Cy5.5-HER2 to stochastically functionalized lysine-based Cy5.5-HER2 ABC with BAR of 1 (BT474 cells, 20 µg/mL, 60 min incubation). Both ABCs have the same 5% Cy5.5 concentration. ABCK is the lysine-conjugated Cy5.5-HER2 ABC prepared from commercial Trastuzumab. We note that this ABC was stored at 4 °C for ~1.5 years prior to this study, demonstrating excellent long-term storage stability. T-ABCK is a stochastic Lys conjugate prepared using the engineered Trastuzumab designed for cysteine conjugation; this construct is designed to rule out differences in cell uptake between Lys-conjugated commercial Trastuzumab and the engineered antibody. Finally, T-ABCC is the site-specific cysteine conjugate Cy5.5-HER2. Results are presented as mean ± SEM (n = 3 biological replicates). Statistical analysis was done using a 2-tailed t-test. For these statistical tests, NS denotes non-significant; **, P < 0.01. Cell uptake studies comparing Cy5.5-HER2 to non-targeted controls Cy5.5-IgG1 and Cy5.5-BP, each containing 1% Cy5.5 loading (40 µg/mL, 60 min incubation, BAR = 3 for ABCs). The x-axis represents the Cy5.5 fluorescence intensity. Cell uptake for similar constructs with 5% Cy5.5 loadings (40 µg/mL, 60 min incubation, BAR = 3 for ABCs). The x-axis represents the Cy5.5 fluorescence intensity. c. Cytotoxicity of PTX-HER2 ABC compared to non-targeted PTX-BP (BPD only), a mixture of PTX-BPD and Trastuzumab (aHER2), and Trastuzumab (aHER2) alone for 24 h incubation. Results are presented as mean ± SEM (n = 3 biological replicates). The ABC PTX-HER2 displays improved cytotoxicity compared to controls. d. Confocal fluorescence microscopy images showed greater cell engagement and uptake for Cy5.5-HER2 compared to non-targeted controls (50 μg/mL, 6 h incubation, 1% Cy5.5 labeled ABC). Two independent experiments were performed with similar results. a) Flow cytometry histograms demonstrating that Cy5.5-HER2 ABC uptake is dependent on HER2 expression. The x-axes represents the Cy5.5 fluorescence intensity. Cell uptake was studied using cell lines with varied HER2 expression: MCF-10A (HER2–); SKOV-3 (HER2 medium); SKBR-3 (HER2 high). Cells were treated with Cy5.5-HER2 ABC or non-targeting Cy5.5-BP polymer (1% Cy5.5 labeling) under the conditions listed at the bottom of the figure (varied times and concentrations). (b-d) Cytotoxicity results for HER2-targeted ABCs comprising different payloads in cell lines with varied HER2 expression. Results are presented as mean ± SEM (n = 3 biological replicates). b. MTT assay results for PTX-HER2 compared to non-targeted PTX-BP in HER2 high, medium, and negative (from left to right), respectively, cell lines following 24 h and 72 h incubation. c. HER2 + SKBR3 cell viability results for ABCs with different payloads (from left to right: MMAE, SN-38, and DOX). d. HER2– MCF-10A cell viability results for ABCs with different payloads (from left to right: MMAE, SN-38, and DOX). a. Confocal fluorescence images of BT-474 cells incubated with Cy5.5-HER2 (20 μg/mL) for different times. b. Confocal fluorescence images of BT-474 cells incubated with “theranostic” ABC SN38-Cy5.5-HER2 (50 μg/mL) for different times. White arrows point to cell nuclei where SN38 has localized following release from the ABC. In panels a and b, 4 h + 24 h and 4 h + 72 h mean that the cells were incubated with Cy5.5-HER2 or SN38-Cy5.5-HER2, respectively, for 4 h. Then, the cells were washed with PBS buffer three times and further incubated for an additional 24 h or 72 h before imaging. Two independent experiments were performed with similar results. First, ABCs bind to the cell surface through antibody-antigen interactions (upper left). Then, bound ABCs enter the cells through receptor-mediated endocytosis. Inside the endosome or lysosome, covalently attached payloads are released and subsequently diffuse to regions of the cell (for example, the nucleus or cytosol) to perform their MoA. Graphic created with BioRender.com. a. Ex vivo images of organs from mice (n = 3) at different time points following administration of Cy5.5-HER2 and non-targeted controls Cy5.5-IgG1 and Cy5.5-BP. b. Quantification of time-dependent BD for targeted and non-targeted constructs as quantified by fluorescence imaging. The bottom bar graphs correspond to the tumor fluorescence signals for different treatment groups, showing substantially greater tumor accumulation for HER2-targeting Cy5.5-HER2. Results are presented as mean ± SEM (n = 3 biological replicates). Statistical analysis was done using a 2-tailed t-test. For these statistical tests, NS denotes non-significant; *, P < 0.05; **, P < 0.01. Cell viability studies for ABCs with different payloads compared to their nontargeting BPs (top and middle row, from left to right: MMAE, SN-38, and DOX; top row: 2 days incubation; bottom row: 5 days incubation) and different ADCs (bottom row, from left to right: T-DM1 and T-DXd, 2 d or 5 d incubation). Each data point represents the mean of three independent replicates (n = 3 biological replicates). b. BT-474 tumor volumes at day 40 for mice given MMAE-based ABCs and non-targeted controls (n = 4 mice/group). c. Ex vivo images of the tumors at day 40 for mice given MMAE-based constructs. 3d for tumor volume measurement with dose schedule illustrated via green arrows (n = 6 mice/group). e. SDS-PAGE analysis of ABCs with DOX as the payload. f. BT-474 tumor volumes at day 40 for mice given DOX-based ABCs and non-targeted controls (n = 6 mice/group). g. Ex vivo images of the tumors at day 40 for mice given DOX-based constructs. Results are presented as mean ± SEM. Statistical analysis was done using a 2-tailed t-test. For these statistical tests, NS denotes non-significant; *, P < 0.05; **, P < 0.01. Tissue sections were formalin-fixed paraffin-embedded (FFPE), stained with hematoxylin and eosin (H&E), and evaluated by a board-certified veterinary pathologist. No sign of toxicity was observed in the heart, lung, liver, spleen, and kidney, supporting a good safety profile. b. Xenografts with different magnifications. Tumor cells were only visible in the PBS, MMAE-BP, and MMAE-IgG1 groups. Tumor-bearing mice were treated with MMAE-based constructs (5 mg/kg mAb; 1.7 mg/kg MMAE per dose). Mice were dosed once a week for a total of 4 doses as illustrated via green arrows in panel a of Fig. Scale bar = 300 μm. For each representative histology image, tissue sections from 4 mice were analyzed independently, with 3 slices examined per mouse, yielding similar results across all samples. Supplementary Discussion and Supplementary Figs. 1–45; Synthesis and Bioconjugation; Spectral Data; Supplementary References Uncropped gels for Supplementary Figures Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Liu, B., Nguyen, H.VT., Jiang, Y. et al. Antibody–bottlebrush prodrug conjugates for targeted cancer therapy. Received: 12 November 2024 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250908175454.htm'>Scientists build quantum computers that snap together like LEGO bricks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 04:10:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>What do children's building blocks and quantum computing have in common? Quantum computing relies on the manipulation of millions of information units called qubits, but these qubits are difficult to assemble. Like plastic children's bricks that lock together to create larger, more intricate structures, scientists can build smaller, higher quality modules and string them together to form a comprehensive system. Recognizing the potential of these modular systems, researchers from The Grainger College of Engineering at the University of Illinois Urbana-Champaign have presented an enhanced approach to scalable quantum computing by demonstrating a viable and high-performance modular architecture for superconducting quantum processors. Their work, published in Nature Electronics, expands on previous modular designs and paves the way toward scalable, fault-tolerant and reconfigurable quantum computing systems. Compared to these limited monolithic systems, modularity enables system scalability, hardware upgrades, and tolerance to variability, making it a more attractive option for building system networks. "Can I build a system that I can bring together, allowing me to manipulate two qubits jointly so as to create entanglement or gate operations between them? Can we do that at a very high quality? Typically, we only find out that something went wrong after putting it together. By constructing a system where two devices are connected with superconducting coaxial cables to link qubits across modules, Pfaff's team demonstrated ~99% SWAP gate fidelity, representing less than 1% loss. Their ability to connect and reconfigure separate devices with a cable while retaining high quality provides novel insight to the field in designing communication protocols. "Finding an approach that works has taken a while for our field," Pfaff said. "Many groups have figured out that what we really want is this ability to stitch bigger and bigger things together through cables, and at the same time reach numbers that are good enough to justify scaling. Moving forward, the Grainger engineers will turn their focus toward scalability, attempting to connect more than two devices together while retaining the ability to check for errors. Note: Content may be edited for style and length. New Research Reveals Not All Ultra-Processed Foods Are Bad Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250908175446.htm'>The sleep switch that builds muscle, burns fat, and boosts brainpower</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 03:24:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>And as every teenager should know, they won't reach their full height potential without adequate growth hormone from a full night's sleep. But why lack of sleep -- in particular the early, deep phase called non-REM sleep -- lowers levels of growth hormone has been a mystery. The findings provide a map for understanding how sleep and hormone regulation interact. The new feedback mechanism could open avenues for treating people with sleep disorders tied to metabolic conditions like diabetes, as well as degenerative diseases like Parkinson's and Alzheimer's. "People know that growth hormone release is tightly related to sleep, but only through drawing blood and checking growth hormone levels during sleep," said study first author Xinlu Ding, a postdoctoral fellow in UC Berkeley's Department of Neuroscience and the Helen Wills Neuroscience Institute. "We're actually directly recording neural activity in mice to see what's going on. The neurons that orchestrate growth hormone release during the sleep-wake cycle -- growth hormone releasing hormone (GHRH) neurons and two types of somatostatin neurons -- are buried deep in the hypothalamus, an ancient brain hub conserved in all mammals. Dysregulation of locus coeruleus neurons is implicated in numerous psychiatric and neurological disorders. "Understanding the neural circuit for growth hormone release could eventually point toward new hormonal therapies to improve sleep quality or restore normal growth hormone balance," said Daniel Silverman, a UC Berkeley postdoctoral fellow and study co-author. "There are some experimental gene therapies where you target a specific cell type. This circuit could be a novel handle to try to dial back the excitability of the locus coeruleus, which hasn't been talked about before." Mice sleep for short periods -- several minutes at a time -- throughout the day and night, providing many opportunities to study growth hormone changes during sleep-wake cycles. Somatostatin and GHRH surge during REM sleep to boost growth hormone, but somatostatin decreases and GHRH increases only moderately during non-REM sleep to boost growth hormone. But when the locus coeruleus becomes overexcited, it paradoxically promotes sleepiness, as Silverman showed in a study published earlier this year. Because growth hormone acts in part through the locus coeruleus, which governs overall brain arousal during wakefulness, a proper balance could have a broader impact on attention and thinking. Dan is the Pivotal Life Sciences Chancellor's Chair in Neuroscience. Other co-authors of the paper are Peng Zhong, Bing Li, Chenyan Ma, Lihui Lu, Grace Jiang, Zhe Zhang, Xiaolin Huang, Xun Tu and Zhiyu Melissa Tian of UC Berkeley; and Fuu-Jiun Hwang and Jun Ding of Stanford University. A New Weapon Against Cancer: Cold Plasma Destroys Hidden Tumor Cells Starving Cancer: New Diet Slows Growth of Deadliest Brain Tumors in Mice Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250908175442.htm'>Seven blood molecules that could explain why you're always sleepy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-09 03:24:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Approximately one in three Americans reports experiencing overwhelming drowsiness during the day -- a condition known as excessive daytime sleepiness (EDS). EDS is linked to an increased risk of serious conditions such as cardiovascular disease, obesity, and diabetes. A new study led by investigators from Mass General Brigham and Beth Israel Deaconess Medical Center identifies several molecules in the blood, known as metabolites, that are linked to EDS. Findings suggest that risk of the condition may be influenced by both internal body processes, such as hormone levels, and external factors such as diet. "Our study suggests diet and genetics may play an important role in EDS," said lead author Tariq Faquih, PhD, a postdoctoral fellow in the Division of Sleep and Circadian Disorders at Brigham and Women's Hospital, a founding member of the Mass General Brigham healthcare system. Researchers collected data on 877 metabolites, naturally occurring molecules in the body influenced by diet and hormones. An additional three metabolites were identified that varied by sex. The team found that omega-3 and omega-6 fatty acids, which are commonly found in foods that make up Mediterranean-like diets, were associated with lower risk of EDS. They also note some limitations to the study, including difficulty in interpreting exact values of metabolites and using a sleep questionnaire instead of bringing participants into a sleep lab for tests. Future directions could include conducting a clinical trial to see if dietary changes or supplements can help reduce daytime sleepiness. Additionally, the authors identified some unknown metabolites that they plan to explore further. "Conducting a clinical trial would be a big next step and could help us understand if omega-3s and omega-6s obtained from diet could help lower risk of EDS," said Faquih. Authorship: In addition to Faquih, MGB authors include Kaitlin S. Potts, Pavithra Nagarajan, Hanna M. Ollila, Tianyi Huang, Clary B. Clish, Susan Redline, Tamar Sofer, and Heming Wang. Disclosures: Redline discloses consulting relationships with Eli Lilly Inc., Jazz Pharma, and Apnimed Inc. Additionally, Redline serves as an unpaid board member for the Alliance for Sleep Apnoea Partners and has received loaned equipment for a multi-site study: oxygen concentrators from Philips Respironics and polysomnography equipment from Nox Medical. A New Weapon Against Cancer: Cold Plasma Destroys Hidden Tumor Cells Starving Cancer: New Diet Slows Growth of Deadliest Brain Tumors in Mice Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            