
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-11</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03876-4'>Lymph node-targeted, mKRAS-specific amphiphile vaccine in pancreatic and colorectal cancer: phase 1 AMPLIFY-201 trial final results</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 15:06:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cellular immunity, mediated by tumor antigen-specific CD4+ and CD8+ T cells, has a critical role in the success of cancer immunotherapy by targeting intracellular driver and passenger tumor mutations. We present the final results of the phase 1 AMPLIFY-201 trial, in which patients who completed standard locoregional treatment, with minimal residual mKRAS disease (n = 25, 20 pancreatic cancer and 5 colorectal cancer), received monotherapy vaccination with lymph node-targeting ELI-002 2P, including mutant KRAS (mKRAS) amphiphile-peptide antigens (G12D, G12R) and amphiphile-adjuvant CpG-7909. At a median follow-up of 19.7 months, efficacy correlated with mKRAS-specific T cell responses above or below a threshold 9.17-fold increase over baseline, with median radiographic relapse-free survival not reached, versus 3.02 months (hazard ratio (HR) = 0.12, P = 0.0002) and median overall survival not reached versus 15.98 months (HR = 0.23, P = 0.0099). Seventy-one percent of evaluable patients induced both CD4+ and CD8+ subsets, with sustained immunogenicity. Following ELI-002 2P treatment, antigen spreading was observed in 67% of patients, with increased T cells reactive to personalized, tumor antigens absent from the ELI-002 2P vaccine. Therefore, lymph node-targeting amphiphile vaccination induces persistent T cell responses targeting oncogenic driver KRAS mutations, alongside personalized, tumor antigen-specific T cells, which may correlate to clinical outcomes in pancreatic and colorectal cancer. Tumor-promoting driver mutations in KRAS occur in approximately 20–25% of human tumors, including colorectal cancer (CRC) (50%) and pancreatic ductal adenocarcinoma (PDAC) (93%). Despite curative intent, relapses are common following standard locoregional therapy, particularly for resectable PDAC. Subsequent elevated ctDNA or serum tumor antigen defines a biomarker-relapsed, minimal residual disease (MRD+) patient group at high risk for radiographic progression1,2. Further treatment following disease relapse/recurrence is primarily palliative and noncurative in intent (5-year survival = 23.3%3). Mutations in KRAS are attractive public neoantigens for immunotherapy due to their prevalence, truncal status and essential driver function. Growing evidence of mKRAS recognition by diverse human HLA alleles4,5,6 suggests that many patients could benefit from an effective off-the-shelf mKRAS-specific therapeutic vaccine. Recent analysis found T cells specific to G12D and/or G12V in 20/20 (100%) of healthy donors evaluated across a diverse HLA background, suggesting that the majority of patients include mKRAS-specific TCRs within their immune repertoire6. However, vaccination with conventional immunogens, including relatively small (<20 kDa) peptide antigens and molecular adjuvants, results in poor accumulation in lymph nodes where uptake by antigen-presenting cells programs adaptive immunity7. Conversely, chemical modification with albumin-binding lipid moieties facilitates delivery of amphiphile vaccines from peripheral injection sites to lymph nodes via size-dependent lymphatic transport by endogenous albumin (~65 kDa), resulting in improved antigen-specific T cell responses8. We previously reported the safety, immunogenicity and preliminary antitumor activity observed in a dose escalation phase 1 trial9. Patients received ELI-002 2P, including lymph node-targeted amphiphile mKRAS G12D-specific and G12R-specific, 18-mer peptide antigens, designed to be processed into HLA class I and HLA class II epitopes. These immunizing peptide antigens were co-administered with an amphiphile-adjuvant CpG-7909 in 25 patients with surgically resected stage I-IV PDAC (n = 20) and CRC (n = 5) who had no evidence of disease on imaging but had detectable MRD+ (ctDNA positive in n = 13/25, CA19-9 and/or carcinoembryonic antigen (CEA) criteria in n = 7/25, or both n = 5/25; Extended Data Fig. Preliminary results at the initial time of data cutoff (6 September 2023; median cohort follow-up time 8.5 months) showed that monotherapy treatment with ELI-002 2P-induced potent mKRAS-directed T cell responses in 21/25 (84%) of the participants, including both CD4+ and CD8+ T cells in 59%. T cell responses above the median 12.75-fold increase from baseline significantly correlated with improved tumor biomarker response and relapse-free survival (RFS)9. Herein we update post hoc analyses of immunogenicity and clinical outcomes with more than double the follow-up time to demonstrate that ELI-002 2P-induced T cell responses are potent, sustained and continue to correlate with freedom from relapse and death. Immune responses included sustained mKRAS-specific CD4+ and CD8+ T cell subsets exhibiting favorable effector cytokine function, cytotoxic markers and memory phenotype, as well as antigen spreading to personalized tumor antigens beyond the immunizing antigens. At a median follow-up of 19.7 months for the cohort (data cutoff: 24 September 2024), no new safety signals were identified in the primary endpoint assessment. The exploratory endpoints of radiographic RFS and overall survival (OS) were reassessed. At extended follow-up, the n = 25 cohort median OS was 28.94 months (Fig. The median RFS was maintained at 16.33 months (Fig. Analysis of the PDAC subset indicated that mRFS (15.31 months) and mOS (28.94 months) were similar to those observed for the whole cohort (Extended Data Fig. Study visits were concluded for AMPLIFY-201 in August 2024. a,b, OS defined as the time from first vaccine dose until death from any cause (a) and radiographic RFS defined as the time from first vaccine dose until confirmed radiographic progression according to iRECIST criteria or death in 25 patients (b). c–f, Patients were stratified by mKRAS-specific T cell response fold change from baseline at the threshold of 9.17. Patients at or above versus below threshold were compared for c, best overall biomarker response reported as percentage relative to the baseline value, error bars are mean ± s.d., P value calculated by two-tailed Mann–Whitney test, number of patients with or without iRECIST progression or death (d), OS (e) and radiographic RFS (f). HR indicates hazard ratio with 95% CI. P values calculated using two-tailed log-rank test. To assess the potential impact of mKRAS-specific T cell responses on clinical outcomes (OS and radiographic RFS), an exploratory receiver-operating-characteristics (ROC) analysis was conducted10. While prior supervision was performed empirically using the median9, we identified a T cell fold-change threshold of 9.17 that optimally separated patients with better (n = 17, 68%) and worse (n = 8, 32%) outcomes, as determined by ROC analysis (Extended Data Fig. As observed previously, the strength of ELI-002 2P-induced mKRAS-specific T cell response was correlated to tumor biomarker response (Fig. 1c), with patients exhibiting T cell response fold change above the 9.17 threshold universally achieving biomarker reductions, including six of six (100%) patients achieving complete ctDNA clearance. Presence or absence of radiographic progression or death (radiographic RFS) was also significantly associated with T cell response (Fig. 1d and Extended Data Table 1)—among the 17 patients with T cell responses above the 9.17 threshold, 11 (65%) were free from radiographic progression, including 5 who remained free of relapse and did not receive any subsequent therapy following ELI-002 2P; 6 (35%) patients received subsequent chemotherapy following tumor biomarker increase yet remained free from disease progression throughout follow-up. As three of six patients who remained free from radiographic progression received subsequent therapy, including immune checkpoint inhibition, following biomarker progression, it will be essential to evaluate these combinations prospectively in future trials. Although subsequent chemotherapy has the potential to negatively impact expanding tumor-specific T cells, prior use of sequential vaccination and chemotherapy regimens has shown that vaccine-induced T cell responses can be maintained following cytotoxic therapy11,12. The current data suggest that anti-mKRAS T cell induction may synergize with subsequent treatment to enable unexpectedly positive outcomes. In contrast to patients with T cell responses above the 9.17 threshold, radiographic progression was observed for all eight patients below the T cell threshold, and seven of eight (88%) died. This is consistent with an approximate threefold increase in risk of radiographic progression/death in the below threshold group (relative risk = 2.96), further suggesting that sufficient cellular immunity to driver mKRAS oncogenes may confer durable benefit. Moreover, the median OS of patients with above-threshold T cell responses was not reached, compared to 15.98 months (Fig. Finally, the median RFS was not reached compared to 3.02 months for patients with T cell fold change above or below the threshold (Fig. This was consistent with updated analysis of radiographic RFS and OS supervised by the prior 12.75 median T cell fold change (Extended Data Fig. The high proportion of patients with above-threshold T cell responses who remained without radiographic evidence of disease following subsequent therapy for biochemical progression (6/11, 55%) suggests that vaccination with ELI-002 or early treatment at the time of biochemical progression may be important strategies to evaluate in future prospective trials. At extended follow-up, 84% (21/25) of patients generated mKRAS-specific T cell responses following ELI-002 2P immunization with 100% responders at the two highest dose levels (5.0, 10.0 mg of adjuvant Amph-CpG-7909). The median response was 13.38-fold over baseline for all patients; however, 17 patients exhibited responses above the ROC threshold of 9.17-fold (Fig. Fifty-seven percent of patients induced responses to all seven mKRAS antigens evaluated (Fig. Seventy-one percent of patients induced both CD4+ and CD8+ mKRAS-specific T cells and the induction of both CD4+ and CD8+ T cells significantly correlated with overall tumor biomarker response, indicating the potential importance of a balanced T cell response for improved tumor biomarker response (Fig. All 12 patients who induced both CD4+ and CD8+ T cells had T cell responses above the 9.17 threshold, suggesting an association between these biomarkers (Extended Data Table 2). ELI-002 2P vaccination amplified mKRAS-specific CD4+ and CD8+ T cells secreting granzyme B and perforin, indicating cytolytic potential; 68% (13/19) of patients induced granzyme B and perforin secreting mKRAS-specific CD4+ T cells after vaccination with predominant central and effector memory phenotype (Extended Data Fig. Likewise, 84% (16/19) of patients had cytotoxic mKRAS-specific CD8+ T cells, substantially composed of TEMRA memory cells (Extended Data Fig. ELI-002 2P-induced mKRAS-specific T cells were persistent before and after booster vaccination with seven of eight (88%) evaluable patients maintaining elevated T cell responses relative to baseline levels following booster vaccinations (Fig. Furthermore, postboost, long-lived mKRAS-specific CD4+ T cells showed significantly increased central memory phenotype compared to baseline with associated decreases in the naive subset (Fig. Although not required in the AMPLIFY-201 study, collection of tumor tissue at the time of progression in future studies will allow for assessment of the tumor-immune microenvironment, which may elucidate potential mechanisms of resistance. Finally, antigen spreading was assessed to examine the potential for ELI-002 2P to promote the expansion of immune responses targeting personalized tumor neoantigens not present in the vaccine. Nonimmunizing antigens from each patient's personal tumor mutanome were selected for peripheral blood mononuclear cell (PBMC) immunogenicity assessment directly ex vivo. T cell responses targeting nonvaccine tumor neoantigens were significantly expanded from baseline levels in 67% (6/9) of the tested patients (Fig. Overall, 13 of 52 (25%) evaluated neoantigens showed increased T cell responses, including examples of both CD4+ and CD8+ T cells, as well as expansion of pre-existing, baseline-detectable responses or de novo expansion. Furthermore, five of six (83%) patients with positive antigen-spreading responses had mKRAS T cell counts above the ROC threshold, suggesting that robust mKRAS-specific T cell induction may be associated with diversification of the tumor-directed T cell response. Future assessment of larger patient groups will be critical to defining the potential contribution of antigen spreading to clinical outcomes. PBMCs were assessed for direct, ex vivo, T cell response at baseline and maximum postimmunization timepoints. a, mKRAS-specific fold change from baseline to response for T cell responders (n = 21/25 patients; 9.17-fold threshold indicated by dashed line). b, mKRAS-specific, background-subtracted IFNγ and/or GrB SFCs per 1 × 106 PBMCs. c, Frequencies of mKRAS-specific CD4+ or CD8+ T cells with intracellular IFNγ/TNF/IL2 assessed by flow cytometry. Response thresholds indicated by dashed lines in a–c and g. d, Frequency of T cell responders to 1–7 assessed mKRAS antigens. e, Frequency of T cell responders including CD4+, CD8+ or both CD4/8+ cytokine+ cells in ICS assay. f, Best overall tumor biomarker response for patients reported as percentage of the baseline value, stratified by CD4+/CD8+ T cell response; error bars are mean ± s.d. n = 22. g, Longitudinal IFNγ/GrB SFCs per 1 × 106 PBMCs for the eight patients with booster vaccinations. h, CD4+ memory phenotype of baseline cytokine negative and postboost mKRAS-specific T cells, n = 6 patients (CM, CCR7+ CD45RA−; EM, CCR7− CD45RA−; TEMRA, CCR7− CD45RA+; naive, CCR7+ CD45RA+); line indicates mean. i,j, Antigen-spreading assessment of patient-specific number of assessed and positive direct, ex vivo T cell responses to nonvaccine tumor neoantigens in FluoroSpot or ICS assay of postprime PBMCs (i) and representative ICS data from patient 11 at baseline and week 9 (j). All P values calculated by two-tailed Mann–Whitney test. Recently, multiple examples of next-generation therapeutic cancer vaccination have demonstrated promise in early phase and randomized clinical studies12,13,14,15. Innovative trial designs have focused on the adjuvant setting, where tumor burden is low, potentially allowing for robust and durable T cell responses to provide long-term protection from recurrence and death. Historically, PDAC has previously been considered a poor setting for immunotherapy; however, the favorable results from ELI-002 2P are consistent with the long-term follow-up of adjuvant patients with PDAC who received personalized mRNA vaccination in combination with atezolizumab and adjuvant mFOLFIRINOX12. In the long-term follow-up of ELI-002 2P treatment, mKRAS-specific T cell responses above a 9.17-fold threshold were significantly correlated to freedom from radiographic progression and death (radiographic RFS HR = 0.12, P = 0.0002; OS HR = 0.23, P = 0.0099). Outcomes in the PDAC subset, with a median radiographic RFS of 15.31 months and a median OS of 28.94 months, are notable given the historically rapid progression of patients with PDAC who have ctDNA+ MRD-relapse postsurgery, with a median radiographic RFS/DFS of 5.0–6.37 months and a median OS of 17.0 months12,16. Additionally, while the relatively small sample size and lack of an external validation cohort are limitations for the ROC analysis as performed, the subgroups were relatively well balanced for tumor type, tumor stage, prior therapies and baseline MRD characteristics (Extended Data Table 3). Notably, all five patients with G12R tumor mutations had T cell responses above the 9.17 threshold. Prior studies in PDAC have observed favorable clinical outcomes as well as decreased PDL1 expression for G12R17,18. This factor, together with potentially increased immunogenicity, may in part explain the more favorable clinical outcomes of PDAC patients with this mKRAS variant. ELI-002 2P-induced mKRAS-specific T cell responses obtained in the monotherapy setting were observed in 100% of patients treated at the RP2D (adjuvant Amph-CpG-7909 was dose escalated), included both CD4+ and CD8+ T cells, and were sustained throughout the follow-up period, including development of memory and maintenance of critical antitumor effector functions. In the majority of T cell responders (17/21), T cell responses were observed against the specific tumor antigen detected at the time of enrollment. Analysis of tumor-specific T cell response and antigen spreading among larger patient groups, and with additional serial PBMC samples, will be helpful to further understand the instances where responses to the tumor antigen identified during screening were not observed. Additional longitudinal data will be useful to inform whether long-term dosing may augment cellular immunity. In addition to expanded T cells targeting mKRAS driver mutations, treatment with ELI-002 2P frequently led to antigen spreading, with ex vivo-detectable expansion of CD4+ and CD8+ T cells targeting additional personalized tumor neoantigens similar to those discretely targeted by personalized vaccination. These are consistent with prior observations of antigen spreading following amphiphile vaccination19,20,21 suggesting a role for lymph node immune activation as a mechanism supporting the development of tumor-specific T cells in situ. Taken together, the long-term follow-up of the AMPLIFY-201 phase 1 study provides evidence that ELI-002 2P induces potent, polyfunctional CD4+ and CD8+ T cell immunity to mKRAS alongside frequent antigen spreading that may delay tumor recurrence. Beyond PDAC, off-the-shelf availability of ELI-002 supports broad development for various mKRAS-expressing tumor types. In conclusion, our observations support continued study of amphiphile lymph node-targeted immunotherapy for solid tumors. A phase 1, multicenter, open-label, first-in-human trial of ELI-002 2P monotherapy was conducted in five ascending dose cohorts at seven centers in the United States between 4 October 2021 and 24 September 2024 (the clinical cutoff date for the results presented here). A fixed dose of Amph-Peptides 2P (G12D and G12R, 0.7 mg each) was administered with escalating doses of 0.1, 0.5, 2.5, 5.0 and 10.0 mg Amph-CpG-7909 adjuvant. Eligible patients were 18 years or older, had mKRAS G12D-mutated or G12R-mutated pancreatic or colorectal cancers and were at high risk for relapse because of the presence of MRD (indicated by ctDNA-positivity or elevated serum CA19-9 and/or CEA). Clinical data was entered into Medidata Rave 2018.2.4. Additional details are provided in the study protocol (Supplementary Data 1). At two institutions, City of Hope and the University of Colorado School of Medicine, central institutional review board (IRB) approval was obtained from WIRB Copernicus IRB. Local IRB approvals were provided for Memorial Sloan Kettering Cancer Center's IRB, the University of Texas MD Anderson (Office of Human Subject Protection), the University of Iowa (Human Subjects Office/IRB), Northwell Health (Feinstein Institutes for Medical Research, Northwell Health IRB), the University of California, Los Angeles (Office of the Human Research Protection Program) and Massachusetts General Hospital (Dana–Farber Cancer Institute Office for Human Research Studies). Patients had recovered from prior surgery, chemotherapy or radiation without ongoing medical/surgical issues and were willing to use effective methods to avoid pregnancy and provided written informed consent. Baseline absolute neutrophil count ≥1.5 × 109 l−1, platelets ≥100 × 109 l−1, normal range liver function tests, serum creatinine <1.5 (or if serum creatinine was ≥1.5 mg dl−1, creatinine clearance calculated by the Cockcroft–Gault formula ≥60 ml min−1 was acceptable), albumin ≥2.5 g dl−1 and IL6 <500 pg ml−1 were required. PDAC patients had high risk tumor stages I, II, III or stage IV oligometastatic disease per current American Joint Committee on Cancer criteria with no evidence of disease on current imaging (equivocal radiographic findings such as subcentimeter lesions or potential resolving soft tissue changes after surgery were accepted), prior treatment with standard chemotherapy/chemoradiation administered in the neoadjuvant and/or adjuvant setting, and complete tumor resection (R0 or R1 pathologic margins), with focal use of intraoperative irreversible electroporation permitted. CRC patients had high risk stage II (T4N0), stage III (T4N1-2/TanyN2) or stage IV oligometastatic disease per current American Joint Committee on Cancer staging criteria, prior cytotoxic chemotherapy administered in the neoadjuvant or adjuvant setting, or as total neoadjuvant therapy, and complete surgical resection (R0 or R1 pathologic margins), with focal use of intraoperative irreversible electroporation permitted. We excluded patients who received antitumor therapy within 4 weeks, who had history of brain metastasis, other malignancies within the last 3 years (except for adequately treated carcinoma of the cervix, bladder, prostate, basal or squamous cell skin cancer), were receiving immunosuppressive drugs, those with serious comorbid illness including uncontrolled infection, class III or IV (New York Heart Association) cardiac failure, myocardial infarction within 6 months, active seizure disorders, autoimmune diseases or interstitial lung disease if requiring systemic steroids, pulse oximetry <92% on room air, prior organ transplants, HIV/AIDS, hepatitis B, hepatitis C (unless they had a sustained virologic response to direct-acting antiviral therapy) and those in the first two weeks of SARS-CoV-2. Women were excluded if pregnant or lactating. Treatment was divided into a ‘prime immunization series' (six subcutaneous doses of ELI-002 2P over 8 weeks), a 3-month ‘no dosing period' (observation) and a ‘booster immunization series' (4 weekly doses of ELI-002 2P). A follow-up period of up to 2 years was included after the first dose of ELI-002 2P to monitor safety and efficacy. The study was sponsored and designed by Elicio Therapeutics in collaboration with the academic authors. The trial protocol, amendments and supporting documents were approved by the local/central institutional review board for each study site, the US Food and Drug Administration and were registered on Clinicaltrials.gov (NCT04853017). All patients provided written informed consent. Cohorts ranged from three to six patients with expansions allowed after the first three patients completed 28 days without dose-limiting toxicity and when additional eligible patients had been identified. The initial protocol (version 1.0) was approved on 13 July 2020. Key protocol amendments are as follows: Amendment 2 (version 3.0) was approved on 23 February 2021 and included changes requested by the US Food and Drug Administration. On 8 April 2022, Amendment 4 (version 5.0) was approved and added serum tumor biomarkers (that is, CEA and CA19-9) to the MRD eligibility along with ctDNA. Amendment 5 (version 6.0), approved on 2 August 2022, added language regarding pseudo-progression and continued ELI-002 dosing. Amendment 6 (version 7.0) was approved on 25 January 2023 and added language for public record search for OS. Amendment 7 (version 8.0) was approved on 7 August 2023 and added another year of follow-up to collect additional RFS and OS. Primary endpoints of the study were safety (adverse events were graded per Common Terminology Criteria for Adverse Events, version 5.0), tolerability and determination of the RP2D. Secondary and exploratory endpoints include tumor biomarker reduction and clearance defined through assessment of ctDNA and/or serum tumor antigens (CA19-9 or CEA), radiographic relapse-free survival, defined as the time from initiation of ELI-002 treatment until confirmed radiographic progression using iRECIST criteria, and OS, and immunogenicity. PBMCs for immunogenicity analysis were processed from leukapheresis (baseline, week 9) or whole-blood collections (all other timepoints). Patient PBMCs were processed by the Ficoll-Hypaque gradient protocol for leukapheresis samples or cell processing tubes (BD) for whole-blood samples. PBMCs were resuspended in CS10 freezing media (Cryostor), frozen in aliquots of 10–20 million cells per cryovial and stored in a temperature-monitored liquid nitrogen vapor phase freezer. Only PBMCs collected before subsequent therapy are included in datasets and graphs, with the exception of the long-term duration graphs (Fig. The maximum T cell response was determined as the maximum fold change from baseline to any postvaccination timepoint in either the ‘Ex vivo FluoroSpot assay' or ‘Ex vivo ICS assay' for any of the seven mKRAS antigens or a pool of all seven antigens combined. Figure 2a includes data from all T cell responders (n = 21/25). Figure 2g contains data from all patients with booster doses that had responses in FluoroSpot assay (n = 8) while Fig. 2h contains data from the boosted patients in Fig. 2g that were also tested for memory markers in the intracellular cytokine staining (ICS) assay (n = 6/8). A direct IFNγ/granzyme B (GrB) FluoroSpot assay was performed on thawed PBMCs. Cryopreserved PBMCs were thawed in 10% human AB serum/RPMI media + Benzonase and rested overnight at 37 °C. Precoated human IFNγ/GrB FluoroSpot plates were washed with phosphate-buffered saline and blocked with AIM-V media for at least 30 min (MabTech). The 2 × 105 rested PBMCs were plated into each well and stimulated for 44 h as per the manufacturer's instructions with seven individual mKRAS peptide pools and a WT peptide pool. Each pool consisted of a KRAS 18-mer peptide along with the corresponding 9-mer and 10-mer overlapping peptides (OLPs), at a concentration of 2 µg per peptide per ml. No exogenous cytokines were added to the PBMCs during this assay. Dimethyl sulfoxide was used as the negative control (background wells) and anti-CD3 (MabTech) was used as the positive control. Plates were scanned and counted using the IRIS plate reader (MabTech) using the FITC and Cy3 filters. Data are background subtracted, averaged per triplicate measurements and normalized to 1 × 106 PBMCs. A responder in the FluoroSpot assay was defined as a patient with a ≥2-fold increase from baseline at any postvaccination timepoint and more than the minimum threshold of 50 total IFNγ and GrB spot-forming cells per 1 × 106 PBMCs. A direct ICS assay for IL2, IFNγ and TNF was performed by flow cytometry. PBMCs were thawed and rested overnight. In total, 106 PBMCs per well were plated and stimulated for 17 h at 37 °C with individual mKRAS peptide pools at 2 μg ml−1 per peptide (Supplementary Table 1). GolgiStop and GolgiPlug (BD) were also added to each well. The next day, cells were surface stained with antibodies against CD4 (BV421—clone, SK3; BD, 566907; 2.5 μl per well), CD8 (BV786—clone, RPA-T8; BD, 563823; 1:25), CD45RA (Alexa 700—clone, HI100; BioLegend, 304120; 1:25), CCR7 (PE-CF594—clone, 15053; BD, 562381; 1:12.5), Aqua Live/Dead marker (Thermo Fisher Scientific, L34966; 0.5 μl per well) and dump markers CD14 (PE-Cy5—clone, 61D3; Thermo Fisher Scientific, 15-0149-42; 1:200), CD16 (PE-Cy5—clone, 3G8; BioLegend, 302010; 1:200), and CD19 (PE-Cy5—clone, SJ25C1; BioLegend, 363042; 1:100). Cells fixed in 0.5% formaldehyde were acquired on a BD FACSymphony and data were analyzed with BD FlowJo V10 software (gating progression and example plots in Supplementary Fig. A responder in the ICS assay was defined as a patient having ≥2-fold increase in total IFNγ, IL2 and TNF from baseline at any postvaccination timepoint, along with a cytokine+ T cell frequency of ≥0.1%. Some patients were also tested in an extended ‘Ex vivo ICS assay' that included additional activation and cytotoxic markers. The next day, cells were surface stained with antibodies against CD8 (BUV805—clone, SK1, BD, 612889), CD45RA (PE-Cy7—clone, HI100; BD, 560675), CCR7 (BUV615—clone, 3D12; BD, 562381), Aqua Live/Dead marker (Thermo Fisher Scientific, L34966; 0.5 μl per well) and dump markers CD14 (PE-Cy5—clone, 61D3; Thermo Fisher Scientific, 15-0149-42; 1:400), CD16 (PE-Cy5—clone, 3G8; BioLegend, 302010; 1:100) and CD19 (PE-Cy5—clone, SJ25C1; BioLegend, 363042; 1:100). Comprehensive genomic profiling, whole-exome sequencing (WES), was performed to determine whether the patient's tumor harbored at least one of the two mKRAS alleles targeted by the ELI-002 2P (G12D or G12R). The Natera Signatera ctDNA test evaluated for the presence or absence of circulating tumor DNA. WES was performed on formalin-fixed, paraffin-embedded tumor samples with at least 20% tumor content confirmed by a pathologist under the Central Lab Improvement Amendments and College of American Pathologists guidelines. Genomic DNA was extracted from the patient's normal (whole blood) and tumor tissue. The assay was performed by target enrichment of the isolated DNA, followed by 440× coverage sequencing on an Illumina HiSeq 2500 or NovaSeq 6000 (Illumina). Somatic single-nucleotide variants (SNVs) that were present in the tumor and absent in the germline were identified. A proprietary Natera algorithm selected a set of 16 SNVs to maximize the detectability of tumor DNA if present in plasma. Polymerase chain reaction primers targeting the 16 personalized SNVs were designed and synthesized to be used to identify and track ctDNA in a patient's plasma. Cell-free DNA was extracted from plasma and analyzed using a multiplexed personalized polymerase chain reaction assay. Plasma samples with ≥2 SNVs detected above a predefined confidence threshold were deemed ctDNA+, and ctDNA concentration was reported as mean tumor molecules per milliliter of plasma. In patients without adequate tumor tissue, a plasma-based ctDNA assay for mKRAS variants was performed using Sysmex SafeSEQ RAS-RAF. Cell-free DNA was isolated from plasma and a next-generation sequencing-based assay that evaluated K/NRAS to detect SNVs was performed using NextSeq 550 (Illumina). Local testing was permitted if already available to confirm mKRAS status. Serum tumor biomarkers, CA19-9 and CEA, were analyzed by the local laboratories at each study site. To assess for antigen spreading, PBMCs were stimulated in the ‘Ex vivo FluoroSpot assay' and ‘Ex vivo ICS assay' as above, with neoantigens not included in the ELI-002 2P vaccine. Genomic DNA is extracted from the patient's normal and tumor samples using next-generation sequencing WES. Using the WES data for each patient, somatic single-nucleotide mutations present in the tumor and absent in the germline genomic DNA were identified using a validated bioinformatics tool (GEM ExTra pipeline NG2-LDT 1.14.0; Natera). The reference genome assembly used for alignment is NCBI GRCh37. Stop-gain and start-loss mutations were excluded. Up to ten neoantigens were randomly selected from the list of somatic SNVs generated by WES for each tested patient for antigen spreading testing. An algorithm was not used to select these neoantigens. First, an 18-mer was designed (generally with the mutation centered in the middle) and then Genscript synthesized two 15-mers overlapping by 11 to cover the mutated 18-mer (18-mer sequences found in Supplementary Table 1). Descriptive statistics were used to summarize demographic, medical history and safety data. Categorical variables were summarized using frequency counts and percentages. Clinical efficacy outcomes, such as tumor biomarker reduction or clearance, were examined for association with categorical variables, including high versus low T cell response, using the Mann–Whitney test. SAS v9.4 and R v4.4.3 were used to create Fig. GraphPad Prism v9.4 was used to create Figs. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Requests must be made to datarequest@elicio.com, with responses provided within 30 days of request. To ensure consistency with the underlying study consent, de-identified patient data that can be shared will be disclosed under data transfer agreements. Investigators and institutions who agree to the terms of the data transfer agreement, which will include, but will not be limited to, terms to address the use of these data for the purposes of a specific project and for research purposes only, to prohibit attempts to re-identify the data and to protect the confidentiality of the data, will be granted access to the data. The data will then be available for up to 12 months. Source data are provided with this paper. Circulating tumor DNA as a clinical test in resected pancreatic cancer. Botta, G. P. et al. Association of personalized and tumor-informed ctDNA with patient survival outcomes in pancreatic adenocarcinoma. Sener, S. F., Fremgen, A., Menck, H. R. & Winchester, D. P. Pancreatic cancer: a report of treatment and survival trends for 100,313 patients diagnosed from 1985–1995, using the National Cancer Database. Baleeiro, R. B. et al. Optimized anchor-modified peptides targeting mutated RAS are promising candidates for immunotherapy. Sim, M. J. W. et al. High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D. Conn, B. P. et al. Generation of T cell responses against broad KRAS hotspot neoantigens for cell therapy or TCR discovery. Trevaskis, N. L., Kaminska, L. M. & Porter, C. J. H. From sewer to saviour—targeting the lymphatic system to promote drug exposure and activity. Liu, H. et al. Structure-based programming of lymph-node targeting in molecular vaccines. Pant, S. et al. Lymph-node-targeted, mKRAS-specific amphiphile vaccine in pancreatic and colorectal cancer: the phase 1 AMPLIFY-201 trial. Measuring diagnostic and predictive accuracy in disease management: an introduction to receiver operating characteristic (ROC) analysis. Rojas, L. A. et al. Personalized RNA neoantigen vaccines stimulate T cells in pancreatic cancer. Sethna, Z. et al. RNA neoantigen vaccines prime long-lived CD8+ T cells in pancreatic cancer. Gainor, J. F. et al. T-cell responses to individualized neoantigen therapy mRNA-4157 (V940) alone or in combination with pembrolizumab in the phase 1 KEYNOTE-603 study. Yarchoan, M. et al. Personalized neoantigen vaccine and pembrolizumab in advanced hepatocellular carcinoma: a phase 1/2 trial. Braun, D. A. et al. A neoantigen vaccine generates antitumour immunity in renal cell carcinoma. Circulating tumor DNA as a potential marker of adjuvant chemotherapy benefit following surgery for localized pancreatic cancer. Distinct clinical outcomes and biological features of specific KRAS mutants in human pancreatic cancer. Moynihan, K. D. et al. Eradication of large established tumors in mice by combination immunotherapy that engages innate and adaptive immune responses. Lymph node–targeted vaccine boosting of TCR T-cell therapy enhances antitumor function and eradicates solid tumors. Ma, L. et al. Vaccine-boosted CAR T crosstalk with host immunity to reject tumors with antigen heterogeneity. & Tian, L. Joint confidence region estimation for area under ROC curve and Youden index. The study was sponsored and funded by Elicio Therapeutics. Employees of Elicio Therapeutics received salaries for their contributions to the study. Aside from grants to their institutions, no other authors received specific funding for this work. These authors contributed equally: Shubham Pant, Eileen M. O'Reilly. Massachusetts General Hospital, Boston, MA, USA Northwell Health, Lake Success, NY, USA James R. Perry, Thian Kheoh, Lisa K. McNeil, Esther Welkowsky, Peter C. DeMuth & Christopher M. Haqq University of Texas MD Anderson Cancer Center, Houston, TX, USA Memorial Sloan Kettering Cancer Center, New York City, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar obtained resources, managed data, performed project administration and contributed to the review and editing of the paper. Correspondence to Zev A. Wainberg, Christopher M. Haqq, Shubham Pant or Eileen M. O'Reilly. has received consultant/advisory fees from Alligator Bioscience, Bayer, Lilly Oncology, AstraZeneca, Merck, Merck KGaA, Daiichi Sankyo, Macrogenics, Amgen, Bristol-Myers Squibb, Astellas, Ipsen, Arcus, Novartis, PureTech, Roche, Seagen and Pfizer, and received research funding (institutional) from Elicio Therapeutics, Five Prime Therapeutics, Arcus, Pfizer, Plexxikon, Novartis and Merck. reports research funding from Elicio Therapeutics, Novartis, Actuate Therapeutics, Merck, AstraZeneca, Deciphera Pharmaceuticals, Genentech/Roche and Mirati, and advisory board relationship with Ipsen, Celgene, Merrimack, Actuate Therapeutics and Genentech. received research funding (institutional) from AbbVie/Stemcentrx, AstraZeneca/MedImmune, BeiGene, Biothera, Bristol-Myers Squibb/Celgene, Eli Lilly, Merck, Mirati Therapeutics, Novartis, Pfizer, Roche, Genmab, Elicio Therapeutics, Mirati, Amgen, Replimmune, Checkmate Pharmaceuticals, Gilead, GSK, Immunocore, Seagen, Tesaro and AbbVie, and also served on advisory boards for AbbVie, BeiGene, Jazz Pharma, Mirati, AstraZeneca/MedImmune, Novartis and Omega Therapeutics. reports consultancy/advisory board relationship with Elicio (Scientific Advisory Board Member/Stock Ownership); is a founder of Precision BioSensors; has consultancy/advisory board relationships with Agenus, Astellas, AstraZeneca, Beixon, Elicio Therapeutics, Guardant Health, Illumina, Natera, Foundation Medicine, Daiichi Sankyo, Tempus, Bayer, MSD Oncology/Merck, Neogenomics, Delcath Systems, QED, Taiho Oncology (self/institution), Exact Sciences, Eisai, BostonGene, Neogenomics, Saga Diagnostics, Regeneron, Servier, Seagen, Lilly, Xilio Therapeutics and Ipsen (to institution); and receives research funding or trial support from Merck, Novartis and Agenus Bio (all to institution). reports receiving honoraria from Vivacitas, a consultancy/advisory board relationship with Pfizer and research funding (to institution) from Elicio Therapeutics. reports receiving honoraria from the MedPro Specialty Advisory Board and holds institutional contracts with Elicio Therapeutics, Bristol-Myers Squibb, Exelixis, FameWave, Antengene, Arrys Therapeutics, Hutchison Medipharma, Corcept Therapeutics, Conjupro Biotherapeutics, Takeda and AbbVie for clinical trials in which she serves as the local Principal Investigator. has held consulting or advisory roles with Ipsen, Gritstone Oncology, Westwood Bioscience, Perthera, and Apeiron Biologics. have amphiphile vaccine patents pending to Elicio Therapeutics. reports clinical research funding (to institution) from Amal Therapeutics, Arqule, Bristol-Myers Squibb, Eli Lilly, Elicio Therapeutics, Holy Stone Healthcare, Immuneering, ImmunoMET, Ipsen, Mirati Therapeutics, Novartis, Purple Biotech, Rgenix, Sanofi-Aventis, Xencor, Astellas, Framewave, 4D Pharma, Boehringer Ingelheim, NGM Biopharmaceuticals, Janssen, Arcus Biosciences, BioNTech, Zymeworks and Pfizer. Other than financial relationships, consultant/advisory fees were received from Alligator Bioscience, Askgene Pharma, AstraZeneca, BPGBio, Zymeworks, Ipsen, Nihon Medi-Physics, Novartis, Janssen, Jazz Pharmaceuticals, US WorldMeds and Boehringer Ingelheim. reports receiving research funding (to institution) from Genentech/Roche, BioNTech, Bristol-Myers Squibb, AstraZeneca, Arcus, Elicio Therapeutics, Helsinn Healthcare, Parker Institute, NIH/NCI, Puma Biotechnology, Digestive Care, Break Through Cancer, Agenus, Amgen, Revolution Medicines, QED and Yiviva; has served as a consultant or on data and safety monitoring boards (DSMBs) for AbbVie, Arcus, AstraZeneca, Autem Medical, Ability Pharma, Alligator Bioscience, Agenus, Berry Genomics, BioNTech SE, Boehringer Ingelheim, Ipsen, Ikena, Merck, Moma Therapeutics, Novartis, Leap Therapeutics, Astellas, Bristol-Myers Squibb, Revolution Medicines, Regeneron and Tango; and also reports an uncompensated relationship with Thetis Pharma, travel support from BioNTech and Arcus and a disclosed financial relationship between her spouse and AbbVie. Nature Medicine thanks Mariano Ponz-Sarvise and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Saheli Sadanand, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Treatment with ELI-002 2P was conducted in a priming immunization period (B: baseline, S: Screening; week 0–week 7) and subsequent boosting immunization period (week 20–23) for CRC or PDAC patients exhibiting tumor expression of mKRAS G12D or G12R following completion of locoregional therapy, radiological confirmation of no evidence of disease and detection of either ctDNA or serum tumor biomarker indicating positivity for minimal residual disease (MRD). a,b, OS (a) and radiographic RFS (b) from study start in n = 20 PDAC patients. Scatter plot of T cell fold change threshold values by decreasing order of Youden Index. T cell fold change threshold values were rounded to the nearest tenth decimal. a, OS defined as the time from first vaccine dose until death from any cause and b, radiographic RFS defined as the time from first vaccine dose until confirmed radiographic progression according to iRECIST criteria or death in n = 25 patients. Patients were stratified by mKRAS-specific T cell response fold change from baseline at the previously determined median of 12.75. Patients at or above versus below median were compared for a (OS) and b (radiographic RFS). HR indicates hazard ratio with 95% CI. P values calculated using two-tailed log-rank test. PBMCs were collected for T cell response assessment at baseline and week 9 timepoints. a, Shown is the ex vivo mKRAS-specific cytotoxic CD4+ T cell fold change from baseline to week 9 in the extended ICS assay for n = 19 patients. T cell cytotoxic responders are defined as a patient having ≥2-fold increase from baseline at week 9 (dotted line) and >0.1% GrB+ and/or perforin+. b, Pie chart depicts the percentage of tested ELI-002 2P patients that induce CD4+ cytotoxic+ T cells in extended ICS assay. c, Representative flow cytometry plots of PBMCs from patient 11 at baseline and week 9 PBMCs that have been stimulated with OLPs to G12D mKRAS antigen and tested in an ex vivo ICS assay for cytokine and cytotoxic marker production. Shown are CD4+ IL2+ mKRAS-specific T cells that are then gated on granzyme B, perforin and CD45RA and CCR7 memory markers. PBMCs were collected for T cell response assessment at baseline and week 9 timepoints. a, Shown is the ex vivo mKRAS-specific cytotoxic CD8+ T cell fold change from baseline to week 9 in the extended ICS assay for n = 19 patients. T cell cytotoxic responders are defined as a patient having ≥2-fold increase from baseline at Week 9 (dotted line) and >0.1% GrB+ and/or perforin+. b, Pie chart depicts the percentage of tested ELI-002 2P patients that induce CD8+ cytotoxic+ T cells in extended ICS assay. c, Representative flow cytometry plots of PBMCs from Patient 19 at baseline and week 9 PBMCs that have been stimulated with OLPs to all 7 mKRAS antigen and tested in an ex vivo ICS assay for cytokine and cytotoxic marker production. Shown are CD8+ CD137+ mKRAS-specific T cells that are then gated on granzyme B, perforin and CD45RA and CCR7 memory markers. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Lymph node-targeted, mKRAS-specific amphiphile vaccine in pancreatic and colorectal cancer: phase 1 AMPLIFY-201 trial final results. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/deep-sea-desalination-pulls-drinking-water-from-the-depths/'>A Surprising Twist Puts Desalination Plants at the Bottom of the Ocean. Here's Why</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Companies are experimenting with deep-sea tech to produce cheaper fresh water Costs and energy requirements have kept desalination from going mainstream in most of the world. Early desalination involved boiling seawater and condensing the steam, a purely thermal method that used loads of energy. In the past 25 years reverse osmosis has become more common—it uses high pressure to push seawater through a membrane with holes so small that only water molecules squeeze through, leaving salt behind. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. What if we could let that movement happen naturally by harnessing the pressure hundreds of meters underwater? Reverse osmosis pods are submerged to depths of around 500 meters (1,600 feet), where immense hydrostatic pressure does the heavy lifting of separating water from salt. Far-fetched as it may sound, there are multiple prototypes already at work; the companies behind them aim to take cheap, large-scale desalination from pipe dream to reality. Its founder and CEO, Alexander Fuglesang, says there's no revolutionary new technology behind his business; it's “essentially a subsea pump cleverly coupled to existing membrane and filter technology,” he says. What's new is the energy savings—Flocean uses 40 to 50 percent less energy than conventional plants—and modular systems that can be deployed to many deep-sea location without bespoke engineering. “It's the same 365 days a year.” This isn't the case at land-based plants, which have to deal with algae blooms, river runoff, storms and seasonal temperature changes. Land-based desalination is several times more costly than pulling water from aquifers or lakes, even at giga-plants in the Middle East that benefit from abundant solar power and large economies of scale. Even if subsea technology cuts the cost of land-based desalination down by 40 percent, it will still be a pricey way to obtain potable water. “We need to remember that once the water is desalinated, it still needs to be pumped up from depths of up to 600 meters,” says Nidal Hilal, founding director of New York University's Water Research Center in Abu Dhabi, who has studied water-treatment engineering for more than 30 years. “Early pilot tests show promise, but the technology has yet to be proven at large scale.” Hilal's research group, for example, is developing electrically conductive reverse osmosis membranes that keep themselves clean by repelling salt ions and impurities. They would extend maintenance intervals, which Fuglesang says might be two to three years with existing membranes. Although proponents of the technology say it would have little effect on undersea life, others urge further research to gauge its impact on marine ecosystems. “Many organisms thrive at a 500-meter depth,” says Adina Paytan, a professor affiliated with the Institute of Marine Sciences at the University of California, Santa Cruz. “It's extremely important for many ocean processes such as the carbon cycle and nutrient cycling,” Paytan says. Companies will need to ensure that their water intake and salty brine by-product don't harm marine life or significantly alter these processes. Given the depths required, subsea desalination won't work in just any seaside location. “Many coastal cities lie on wide continental shelves, meaning deep water is far offshore,” Hilal says. Coastlines with steep drop-offs are ideal because shallow shelves would require long pipelines, adding to capital and operational costs. Fuglesang isn't worried about technical or engineering hurdles; he says the industry's biggest challenge will be aligning customers, governments and financial partners. Flocean is working on what will be the world's first large-scale subsea desalination plant, off the coast of Norway, and has a contract to start supplying water to an industrial facility in 2026. Netherlands-based Waterise has also secured its first industrial customer, with plans to start building a plant in the Red Sea's Gulf of Aqaba later this year, and Bay Area–based OceanWell is testing its prototype near Los Angeles. Long-term government contracts will most likely be needed for subsea desalination to really take off, and they may prove elusive. “The water-infrastructure industry is quite conservative,” Fuglesang says, noting that because new projects are so expensive, “nobody wants to be first” to go all-in on a new technology. Hilal says he believes subsea desalination could go mainstream and supply water to entire cities. But, he adds, “reaching true city-scale will take time, conceivably a decade or more.” Her work has appeared in Time magazine, Forbes, AI Frontiers, Scientific American and SingularityHub, among other outlets.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65653811/scientists-say-physical-warp-drive-is-now-possible/'>Scientists Announce a Physical Warp Drive Is Now Possible. Seriously.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 14:17:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Humans are one step closer to traveling at faster-than-light speeds. In a surprising paper, scientists say they've nailed down a physical model for a warp drive, which flies in the face of what we've long thought about the crazy concept of warp speed travel: that it requires exotic, negative forces. The colloquial term “warp drive” comes from science fiction, most famously Star Trek. Star Trek suggests that this extraordinary power alone pushes the ship at faster-than-light speeds. Scientists have been studying and theorizing about faster-than-light space travel for decades. One major reason for our interest is pure pragmatism: without warp drive, we're probably never making it to a neighboring star system. The closest such trip is still four years long at light speed. Our current understanding of warp speed dates back to 1994, when a now-iconic theoretical physicist named Miguel Alcubierre first proposed what we've called the Alcubierre drive ever since. “By a purely local expansion of spacetime behind the spaceship and an opposite contraction in front of it,” Alcubierre wrote in his paper's abstract, “motion faster than the speed of light as seen by observers outside the disturbed region is possible.” Essentially, an Alcubierre drive would expend a tremendous amount of energy—likely more than what's available within the universe—to contract and twist space-time in front of it and create a bubble. Some scientists have criticized the Alcubierre drive, however, because it requires too much mass and negative energy for humans to ever seriously construct a warp-based propulsion system. NASA has been trying to build a physical warp drive through Eagleworks Laboratories for most of the last decade, but hasn't yet made any significant strides. Plus, Alcubierre himself has endorsed the new model, which is like having Albert Einstein show up to your introductory physics class. Here's a helpful video in which Sabine Hossenfelder, a Professor and Research Fellow at the Frankfurt Institute for Advanced Studies, breaks down the findings: Of course, there's one gigantic caveat here: The concept in this paper is still in the “far future” zone of possibility, made of ideas that scientists still don't know how to construct in any sense. “While the mass requirements needed for such modifications are still enormous at present,” the APL scientists write, “our work suggests a method of constructing such objects based on fully understood laws of physics.” Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. This Clay Seal Might Belong to a Biblical Figure An AI System Found a New Kind of Plasma Physics</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/pessimistic-dogs-are-better-at-smelling-cancer-and-other-keys-to-disease/'>Pessimistic Dogs Are Better at Smelling Cancer—And Other Keys to Disease-Sniffing Success</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Pessimistic Dogs Are Better at Smelling Cancer—And Other Keys to Disease-Sniffing Success New research is revealing how disease-smelling dogs can excel Individual dogs' personalities, and how we interpret their behaviors, may be key to disease sniffing at scale. Cancer can change a person's “volatilome,” the unique set of volatile organic compounds found in breath, sweat, blood and urine. Billy and her cohort have learned to sniff out these subtle scent cues in masks worn by people with cancer diagnoses. But clinicians still do not routinely use dogs for diagnosis. Besides the obvious logistical challenges, dogs vary greatly in their olfactory accuracy. Researchers are increasingly finding that disease-sniffing prowess may come down to individual dogs' personality—and how well their handlers know them. New research efforts are focused on figuring out which dogs would be best for the job and on interpreting dogs' behaviors during a smell test. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Sharyn Bistre Dabbah, a veterinary scientist now at the University of Bristol in England, set out with colleagues at the U.K. charity Medical Detection Dogs to learn how the animals' personalities—especially their level of optimism or pessimism—affect disease-detection skills. The scientists then placed bowls behind two new screens between the positive and negative spots, and they classified the dogs as “optimistic” or “pessimistic” based on how quickly they investigated these new locations. Next, the team evaluated how accurately each dog could pick out a disease scent it was trained to detect among other smells. On average, the pessimistic dogs turned out to be more discerning. Other personality traits also play a role, says Clara Wilson, who researches disease- and stress-sniffing dogs at the University of Pennsylvania. Dogs that enjoy the thrill of a hunt—and thrive while searching for missing people or hidden bombs—might find sniffing through disease samples again and again rather repetitive. “We want a dog that doesn't get frustrated. Handlers' interpretations of dog behavior can also skew detection outcomes, says Akash Kulgod, co-founder of Dognosis, the Bengaluru-based start-up that trained Billy. Instead of teaching dogs to perform a specific behavior such as sitting or barking when they pick up an assigned scent—a process that takes extra time and can lead to dogs “lying” for treats—Kulgod and his team directly analyze each dog's natural body language. “They each have their own unique quirks—but all of it can be quantified because it's all related to this reward expectation that you have from the past sessions.” In a pilot study with 200 test samples involving 10 cancer types, presented at this year's American Society of Clinical Oncology conference, Dognosis dogs detected 96 percent of cancers. Next, the Dognosis team will scale up its study with 1,500 test samples. Researchers are always on the lookout for less invasive methods—including options directly involving our canine companions, as well as electronic noses inspired by them. But this primacy may not last, according to Andreas Mershin, chief science officer at the Boston-based start-up RealNose.ai. Mershin's team put mammalian smell receptors on an electronic chip and used machine-learning algorithms to interpret the output. The technique focuses on broader patterns among detected molecules rather than categorizing them individually. Dogs don't tick off a list of molecules in their heads, either; they just “know” what cancer smells like. This helps them to sense it accurately no matter which organ it is from or what the patient ate before giving their breath sample. They don't care about the font in which you write the scent; they just interpret it correctly,” Mershin says. Their findings, which built on work with diagnosis data from Medical Detection Dogs, suggest that focusing on this type of “scent character” might work as a scalable alternative to dogs, even if it's currently much slower. Rohini Subrahmanyam is a biologist turned science journalist. She loves writing about interesting creatures on our planet.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02032-y'>reverses pharmacological and neurodegenerative cognitive impairment in mice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 10:27:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Many brain disorders involve mitochondrial alterations, but owing to the lack of suitable tools, the causal role of mitochondrial dysfunction in pathophysiological processes is difficult to establish. Heterotrimeric guanine nucleotide-binding (G) proteins are key regulators of cell functions, and they can be found within mitochondria. Therefore, we reasoned that the activation of stimulatory mitochondrial G proteins (Gs) could rapidly promote the activity of the organelle and possibly compensate for bioenergetic dysfunction. Here, we show that a mitochondria-targeted recombinant designer receptor exclusively activated by designer drugs (mitoDREADD-Gs) can acutely trigger intramitochondrial signaling to increase mitochondrial membrane potential and oxygen consumption. In vivo activation of mitoDREADD-Gs abolished memory alterations in cannabinoid-treated mice and in two mouse models of Alzheimer's disease and frontotemporal dementia. Thus, mitoDREADD-Gs enables the establishment of causal relationships between mitochondria and biological or disease-related processes and represents an innovative potential therapeutic approach for disorders associated with mitochondrial impairment. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Source data are provided in Supplementary information. Source data are provided with this paper. No custom software codes were used to analyze the data. Mattson, M. P., Gleichmann, M. & Cheng, A. Mitochondria in neuroplasticity and neurological disorders. Chu, C. T. Mitochondria in neurodegeneration. & Frye, R. E. Mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis. Impaired mitochondrial function in psychiatric disorders. Anglin, R. E., Garside, S. L., Tarnopolsky, M. A., Mazurek, M. F. & Rosebush, P. I. The psychiatric manifestations of mitochondrial disorders: a case and review of the literature. Hollis, F. et al. Mitochondrial function in the brain links anxiety with social subordination. Schapira, A. H. et al. Mitochondrial complex I deficiency in Parkinson's disease. & Hebert-Chatelain, E. Damage in mitochondrial DNA associated with Parkinson's disease. Parkinson, G. M., Dayas, C. V. & Smith, D. W. Increased mitochondrial DNA deletions in substantia nigra dopamine neurons of the aged rat. Bell, S. M. et al. Mitochondrial dysfunction in Alzheimer's disease: A biomarker of the future? Anoar, S., Woodling, N. S. & Niccoli, T. Mitochondria dysfunction in frontotemporal dementia/amyotrophic lateral sclerosis: lessons from Drosophila models. Choe, M. & Titov, D. V. Genetically encoded tools for measuring and manipulating metabolism. Oldham, W. M. & Hamm, H. E. Heterotrimeric G protein activation by G-protein-coupled receptors. & Duman, R. S. Heterotrimeric G Proteins. In Basic Neurochemistry: Molecular, Cellular and Medical Aspects 6th edn (eds Siegel, G. J. et al.) Ch. Bhattacharya, M. et al. Nuclear localization of prostaglandin E2 receptors. Irannejad, R. et al. Functional selectivity of GPCR-directed drug action through location bias. Bénard, G. et al. Mitochondrial CB₁ receptors regulate neuronal energy metabolism. Belous, A. E. et al. Mitochondrial calcium transport is regulated by P2Y1- and P2Y2-like mitochondrial receptors. Abadir, P. M., Walston, J. D. & Carey, R. M. Subcellular characteristics of functional intracellular renin-angiotensin systems. Suofu, Y. et al. Dual role of mitochondria in producing melatonin and driving GPCR signaling to block cytochrome c release. Hebert-Chatelain, E. et al. A cannabinoid link between mitochondria and memory. & Voyno-Yasenetskaya, T. A. Gα12 is targeted to the mitochondria and affects mitochondrial morphology and motility. Benincá, C. et al. A new non-canonical pathway of Gαq protein regulating mitochondrial dynamics and bioenergetics. Roth, B. L. DREADDs for neuroscientists. Armbruster, B. N., Li, X., Pausch, M. H., Herlitze, S. & Roth, B. L. Evolving the lock to fit the key to create a family of G protein-coupled receptors potently activated by an inert ligand. Engineered AAVs for efficient noninvasive gene delivery to the central and peripheral nervous systems. Improving adeno-associated viral (AAV) vector-mediated transgene expression in retinal ganglion cells: comparison of five promoters. Furness, S. G. B. et al. Ligand-dependent modulation of G protein conformation alters drug efficacy. Gille, A., Wenzel-Seifert, K., Doughty, M. B. & Seifert, R. GDP affinity and order state of the catalytic site are critical for function of xanthine nucleotide-selective Gαs proteins. Jain, R., Watson, U., Vasudevan, L. & Saini, D. K. ERK activation pathways downstream of GPCRs. Torres-Quesada, O., Mayrhofer, J. E. & Stefan, E. The many faces of compartmentalized PKA signalosomes. Acin-Perez, R. et al. Cyclic AMP produced inside mitochondria regulates oxidative phosphorylation. Jewell, J. L. et al. GPCR signaling inhibits mTORC1 via PKA phosphorylation of Raptor. Lefkimmiatis, K., Leronni, D. & Hofer, A. M. The inner and outer compartments of mitochondria are sites of distinct cAMP/PKA signaling dynamics. Soria-Gomez, E. et al. Subcellular specificity of cannabinoid effects in striatonigral circuits. Cardone, L. et al. A-kinase anchor protein 84/121 are targeted to mitochondria and mitotic spindles by overlapping amino-terminal motifs. Jimenez-Blasco, D. et al. Glucose metabolism links astroglial mitochondria to cannabinoid effects. Monory, K. et al. Genetic dissection of behavioural and autonomic effects of Δ9-tetrahydrocannabinol in mice. Yoshiyama, Y. et al. Synapse loss and microglial activation precede tangles in a P301S tauopathy mouse model. Jankowsky, J. L. et al. Mutant presenilins specifically elevate the levels of the 42 residueβ-amyloid peptide in vivo: evidence for augmentation of a 42-specific γ-secretase. Trinchese, F. et al. Progressive age-related development of Alzheimer-like pathology in APP/PS1 mice. Greiser, M. et al. Calcium and bicarbonate signaling pathways have pivotal, resonating roles in matching ATP production to demand. Buck, J., Sinclair, M. L., Schapal, L., Cann, M. J. & Levin, L. R. Cytosolic adenylyl cyclase defines a unique signaling molecule in mammals. Fazal, L. et al. Multifunctional mitochondrial Epac1 controls myocardial cell death. Thibault, K. et al. Activation-dependent subcellular distribution patterns of CB1 cannabinoid receptors in the rat forebrain. Winters, B. D. et al. Cannabinoid receptor 1-expressing neurons in the nucleus accumbens. Delgado-Peraza, F. et al. Mechanisms of biased β-arrestin-mediated signaling downstream from the cannabinoid 1 receptor. Regulation of CB1 cannabinoid receptor trafficking by the adaptor protein AP-3. Cowburn, R. F., Wiehager, B., Ravid, R. & Winblad, B. Acetylcholine muscarinic M2 receptor stimulated [35S]GTP gamma S binding shows regional selective changes in Alzheimer's disease postmortem brain. Cowburn, R. F. et al. Adenylyl cyclase activity in postmortem human brain: evidence of altered G protein mediation in Alzheimer's disease. O'Neill, C. et al. Regionally selective alterations in G protein subunit levels in the Alzheimer's disease brain. Harrison, P. J., Barton, A. J. L., McDonald, B. & Pearson, R. C. A. Alzheimer's disease: specific increases in a G protein subunit (GSα) mRNA in hippocampal and cortical neurons. Avissar, S., Barki-Harrington, L., Herishanu, Y. & Schreiber, G. Reduced Gs protein function and Gαs levels in leukocytes of patients with Parkinson's disease. Mechanism of neuroprotective mitochondrial remodeling by PKA/AKAP1. Komen, J. C. & Thorburn, D. R. Turn up the power—pharmacological activation of mitochondrial biogenesis in mouse models. Imai, Y. et al. Light-driven activation of mitochondrial proton-motive force improves motor behaviors in a Drosophila model of Parkinson's disease. Dumont, M. et al. Behavioral deficit, oxidative stress, and mitochondrial dysfunction precede tau pathology in P301S transgenic mice. Wang, L. et al. Mitochondrial fusion suppresses tau pathology-induced neurodegeneration and cognitive decline. Expression of Cre recombinase in dopaminoceptive neurons. Di Benedetto, G., Scalzotto, E., Mongillo, M. & Pozzan, T. Mitochondrial Ca2+ Uptake induces cyclic AMP generation in the matrix and modulates organelle ATP levels. Oliveira da Cruz, J. F. et al. Specific hippocampal interneurons shape consolidation of recognition memory. Livigni, A. et al. Mitochondrial AKAP121 links cAMP and src signaling to oxidative metabolism. The metabolism of rat brain Mitochondria. De Rasmo, D. et al. Activation of the cAMP cascade in human fibroblast cultures rescues the activity of oxidatively damaged complex I. Acín-Pérez, R., Fernández-Silva, P., Peleato, M. L., Pérez-Martos, A. Respiratory active mitochondrial supercomplexes. Lopez-Fabuel, I. et al. Complex I assembly into supercomplexes determines differential mitochondrial ROS production in neurons and astrocytes. Nagashima, S. et al. Golgi-derived PI(4)P-containing vesicles drive late steps of mitochondrial division. Lurette, O. et al. Mitochondrial matrix-localized Src kinase regulates mitochondrial morphology. Jakic, B., Buszko, M., Cappellano, G. & Wick, G. Elevated sodium leads to the increased expression of HSP60 and induces apoptosis in HUVECs. We thank P. Roy of the Animal Facilities of the University of Moncton, and D. Gonzales, N. Aubailly, E. Huc and all personnel of the Animal Facilities of the NeuroCentre Magendie for mouse care. We also thank all members of Hebert-Chatelain's and Marsicano's lab for useful discussions and advice. We thank C. Le Gouill from the Bouvier laboratory (University of Montreal) for sharing Gs constructs and guidance. We would also like to thank the Bordeaux Imaging Center (BIC) and the Biochemistry and Biophysics platform (Bioprot, Bordeaux Neurocampus). This work was funded by the Natural Sciences and Engineering Research Council of Canada (grant number RGPIN-2015-05880 to E.H.-C.), Canadian Health Research Institute (grant number 156238 to E.H.-C. and L.B. ), Canada Research Chair program (to E.H.-C.) and ResearchNB (to E.H.-C.). This work was also supported by Inserm (to G.M. ); the European Research Council (Endofood, ERC–2010–StG–260515; CannaPreg, ERC-2014-PoC-640923; Micabra, ERC-2017-AdG-786467, to G.M. ); Fondation pour la Recherche Medicale (DRM20101220445 to G.M. ); the Human Frontiers Science Program (to G.M. ); Region Nouvelle Aquitaine (APP ESR NAq 2020 to G.M. ); and the French government in the framework of the University of Bordeaux's IdEx ‘Investments for the Future—France 2030' program / GPR BRAIN_2030 (to G.M. E.S.-G. was funded by MICIU/AEI (PGC2018-093990-A-I00 and PID2021-125763NB-I00). was funded by MICIU/AEI (PID2022-138813OB-I00 /10.13039/501100011033 and FEDER, UE), the Instituto de Salud Carlos III (CB16/10/00282) and the European Union's Horizon Europe research and innovation programme under MSCA Doctoral Networks 2021 (101072759, FuEl ThE bRaiN In healtThY aging and age-related diseases (ETERNITY). are recipients of a grant from la Caixa Foundation (agreement LCF/PR/HR23/52430016). received funding from the DIUE de la Generalitat de Catalunya (SGR 00022, 2023 FI-3 00034), from the H2020 Excellent Science—European Research Council (948217) and the Ministry of Economy and Competitiveness (RTI2018-093667-A-100; PID2021-122795OB-I00). was funded by the European Union's Horizon 2020 research and innovation programme under Marie Skłodowska-Curie grant agreement No. 676,144 (Synaptic Dysfunction in Alzheimer's Disease, SyDAD). Grant PID2019-107548RB-I00 funded by MICIU/AEI/10.13039/501100011033 (to P.G. ); The Basque Government (grant number IT1620-22 to P.G. ); and grant RD21/0009/0006 funded by Instituto de Salud Carlos III (ISCIII) and by the European Union NextGenerationEU, Mecanismo para la Recuperación y la Resiliencia (MRR), ISCIII-Redes de Investigación Cooperativa Orientadas a Resultados en Salud (RICORS)-Red de Investigación en Atención Primaria de Adicciones (RIAPAd) (to P.G.). G.H.-C. was supported by a fellowship from New Brunswick Health Research Foundation. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. These authors contributed equally: Antonio C. Pagano Zottola, Rebeca Martín-Jiménez, Gianluca Lavanco, Geneviève Hamel-Côté. These authors jointly supervised this work. Endocannabinoids and Neuroadaptation U1215 NeuroCentre Magendie, INSERM, Bordeaux, France Antonio C. Pagano Zottola, Gianluca Lavanco, Rui S. Rodrigues, Yamuna Mariani, Abel Eraso-Pichot, Sandra Beriain, Astrid Cannich, Rosmara Infantino, Francisca Julio-Kalajzić, Doriane Gisquet, Sandrine Pouvreau, Giovanni Marsicano & Luigi Bellocchio Antonio C. Pagano Zottola, Gianluca Lavanco, Rui S. Rodrigues, Yamuna Mariani, Abel Eraso-Pichot, Sandra Beriain, Astrid Cannich, Rosmara Infantino, Francisca Julio-Kalajzić, Doriane Gisquet, Ania Goncalves, Stephane Duvezin-Caubet, Anne Devin, Sandrine Pouvreau, Giovanni Marsicano & Luigi Bellocchio U1312 Bordeaux Institute of Oncology, INSERM, Pessac, France Canada Research Chair in Mitochondrial Signaling and Physiopathology, University of Moncton, Moncton, New Brunswick, Canada Rebeca Martín-Jiménez, Geneviève Hamel-Côté, Mehtab Khan, Stephanie Jean, Inas Al-Younis, Yann Baussan & Etienne Hebert-Chatelain Department of Biology, University of Moncton, Moncton, New Brunswick, Canada Rebeca Martín-Jiménez, Geneviève Hamel-Côté, Mehtab Khan, Stephanie Jean, Inas Al-Younis, Yann Baussan & Etienne Hebert-Chatelain Department of Health Promotion, Mother and Child Care, Internal Medicine and Medical Specialties of Excellence ‘G. Hospital del Mar Medical Research Institute, PRBB, Barcelona, Spain Carla Ramon-Duaso, Laura Vidal-Palencia & Arnau Busquets-Garcia Department of Biomedical and Biotechnological Sciences, Section of Pharmacology, University of Catania, Catania, Italy Laboratory of Ultrastructural and Functional Neuroanatomy of the Synapse, Department of Neurosciences, Faculty of Medicine and Nursing, University of the Basque Country UPV/EHU, Leioa, Spain Itziar Bonilla-Del Río, Nagore Puente & Pedro Grandes Achucarro Basque Center for Neuroscience, Science Park of the UPV/EHU, Leioa, Spain Itziar Bonilla-Del Río, Nagore Puente & Pedro Grandes Institute of Functional Biology and Genomics (IBGF), Universidad de Salamanca, CSIC, Salamanca, Spain Daniel Jimenez-Blasco & Juan P. Bolaños Institute of Biomedical Research of Salamanca (IBSAL), Hospital Universitario de Salamanca, Universidad de Salamanca, CSIC, Salamanca, Spain Daniel Jimenez-Blasco & Juan P. Bolaños Centro de Investigación Biomédica en Red de Fragilidad y Envejecimiento Saludable (CIBERFES), Madrid, Spain Daniel Jimenez-Blasco & Juan P. Bolaños Department of Neurosciences, Faculty of Medicine and Nursing, University of the Basque Country/ ‘Neuroglia basis of behavioral processes' Achucarro Basque Center for Neuroscience, Science Park of the UPV/EHU, Leioa, Spain Jon Egaña-Huguet & Edgar Soria-Gomez Interdisciplinary Institute for Neuroscience, UMR 5297, CNRS, Bordeaux, France IBGC, UMR 5095, CNRS, Bordeaux, France Stephane Duvezin-Caubet & Anne Devin Ikerbasque, Basque Foundation for Science, Leioa, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and E.H.-C. designed the study. and E.H.-C. performed biochemical experiments. G.L., C.R.-D., Y.M., F.D., L.V.-P., J. E.-H., I.B.-D.R., R.I., N.P., E.S.-G. and A.B.-G. performed viral injection, electron microscopy or behavioral experiments. S.D.-C. and A.D. supervised some biochemical experiments. and E.H.-C. analyzed the data. and E.H.-C. wrote the manuscript. and E.H.-C. supervised the work. All authors edited and approved the manuscript. Correspondence to Giovanni Marsicano, Luigi Bellocchio or Etienne Hebert-Chatelain. The authors declare no competing interests. One or more of the authors of this paper self-identifies as a member of the LGBTQ+ community. Nature Neuroscience thanks Istvan Katona, Bryan Roth and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Representative immunoblotting (n = 3) of Gs, the inner mitochondrial protein PHB2 and the outer mitochondrial membrane protein MFN1 in mitochondria of HEK cells treated with proteinase K (PK) as indicated. (b) Representative immunofluorescence and Pearson's colocalization scatter plots and indexes between HA and TOM20 or SDHA in HeLa and mouse embryonic fibroblasts (MEF) expressing control vector or the HA-tagged DREADD-Gs and mitoDREADD-Gs (n = 4 independent mitochondrial isolation for each group, two-tailed unpaired t-test, ****:p<0.0001). (c) Representative immunofluorescence and Pearson's colocalization scatter plots and indexes between HA and the nucleus marker DAPI, the plasma membrane marker wheat germ agglutinin (WGA) or the ER protein hERp57 in HEK cells expressing HA-tagged mitoDREADD-Gs (n = 3 independent cell preparations for each group, data were analyzed by one-way ANOVA, two-sided). (d) Representative immunofluorescence of HA and WGA in HEK cells expressing a control vector and HA-tagged DREADD constructs with quantification of cell area, perimeter and circularity based on WGA staining; n = 3 independent cell preparations for each group, data were analyzed by one-way ANOVA, two-sided, followed by Dunnett's multiple comparison test (for cell area); *p = 0.0226, **p = 0.0015. (e) Representative immunoblotting of HA in total cell lysate of HEK cells expressing control vector and mitoDREADD-Gs with C- or N-terminal HA tag (n = 3 independent cell preparations). TPL: total protein load. (f) Representative immunofluorescence of DAPI (in blue), GFAP (in white) and HA (in red) in the hippocampus of brain slices derived from mice expressing HA-tagged DREADD constructs as indicated. Yellow arrows indicate HA-positive neurons and green arrows indicate cells labeled with HA and GFAP. (g) Quantification (n = 6 brain slices from independent 7-14 weeks-old mice for each group) of HA-positive neurons and astrocytes based on immunofluorescence shown in f. Data were analyzed by two-way ANOVA, two-sided. (h) Representative immunoblotting of HA in neurons and astrocytes purified from mouse hippocampi expressing the empty vector or mitoDREADD-Gs (n = 3). (i) Electron micrographs of HA immunogold labeling in the hippocampus of mice expressing the empty vector. Green arrow indicates particles at the internal surface of the OMM. Quantification of the number of immunogold particles localized at specific compartments per image. Vector: n = 35 slices from one 7-14 weeks-old mice for each location; DREADD-Gs: n = 65 slices from two 7-14 weeks-old mice for each location; MitoDREADD-Gs: n = 126 slices from three 7-14-weeks-old mice for each location, two-way ANOVA, two-sided, followed by Tukey's multiple comparison test. Data are presented as mean ± SEM. (a) Representative immunofluorescence of HA and the mitochondrial membrane potential dye MitoTracker Red™ upon treatment with CNO (50 μM) in HeLa and MEF transiently expressing control vector or HA-tagged DREADD constructs, and in MEF stably expressing the same constructs as indicated (left panels). Quantification of MitoTracker Red™ staining (right panels), n = 4 independent cell preparations for each group except for vector in HeLa cells where n = 3. HeLa: **p = 0.0026 and ***p = 0.0007 according to one-way ANOVA followed by Tukey's multiple comparison test; ##p = 0.0065 according to one sample t test. MEF: ****p<0.0001 according to one-way ANOVA followed by Tukey's multiple comparison test; ###p = 0.0006 according to one sample t test. Stable MEF: ****p<0.0001 according to one-way ANOVA followed by Tukey's multiple comparison test; ####p<0.0001 according to one sample t test. (b) Effect of CNO (50 μM) on oxygen consumption rates (OCR) of intact HeLa cells expressing control vector or HA-tagged DREADD constructs. *p = 0.0170 and 0.0410, respectively, according to one-way ANOVA followed by Tukey's multiple comparison test Data are presented as mean ± SEM. (a) Representative immunofluorescence of HA and TOM20 upon treatment with CNO (50 μM) in HEK (n = 3 independent cell preparations for each group), HeLa (n = 3 independent cell preparations for each group), mouse embryonic fibroblast (MEF, n = 4 independent cell preparations for each group) transiently expressing control vector or HA-tagged mitoDREADD-Gs and in MEF stably (stable MEF, n = 4 independent cell preparations for each group) expressing the same constructs (left panels). Quantification of TOM20 staining in region of interest (ROI) (right panels). Data were analyzed by unpaired t test, two-sided and are presented as mean ± SEM. (a) Representative immunofluorescence of HA and SDHA upon treatment with CNO (50 μM) in HEK (n = 4 independent cell preparations for each group), HeLa, (n = 3 independent cell preparations for each group), mouse embryonic fibroblast (MEF, n = 3 independent cell preparations for each group) transiently expressing control vector or HA-tagged mitoDREADD-Gs and in MEF stably (stable MEF, n = 3 independent cell preparations for each group) expressing the same constructs (left panels). Quantification of SDHA staining in region of interest (ROI) (right panels). Data were analyzed by unpaired t test, two-sided and are presented as mean ± SEM. (a) Representative immunoblotting and quantification of pERK and ERK in HeLa cells expressing control vector, or HA-tagged DREADD constructs and treated with CNO (50 μM) as indicated. Vector, n = 6 independent cell preparations for each group; DREADD-Gs: n = 3 for each group; MitoDREADD-Gs: n = 3 for each group. ####p<0.0001 according to two-way ANOVA, two sided, followed by Sidak's multiple comparison test. (b) Levels of phospho-PKA substrate in total cell lysates of HeLa expressing control vector or HA-tagged DREADD constructs and treated with CNO (50 μM) as indicated (n = 3 independent cell preparations for each group). ***p = 0.0005, *p = 0.0390 and ***p = 0.0007, respectively according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. (c) Levels of phospho-PKA substrate in mitochondrial fractions of HEK expressing control vector, DREADD-Gs or mitoDREADD-Gs and treated with CNO (50 μM) as indicated (n = 4 independent mitochondrial isolations for each group). ####p<0.0001 according to two-way ANOVA, two sided, followed by Sidak's multiple comparison test. (d) Representative immunofluorescence of HA and the mitochondrial membrane potential dye MitoTracker Red™ (n = 3 independent cell preparations for each group) in HeLa expressing control vector, DREADD-Gs or mitoDREADD-Gs and treated with vehicle, CNO (50 μM) and the PKA inhibitor H89 (1 μM) as indicated (left panel). Quantification of MitoTracker Red™ staining (right panel). ****p<0.0001 according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. (e) Representative immunoblotting of HA, PKA catalytic subunit α (PKA-Cα) and total protein load (TPL) in HEK cells expressing control vector or mitoDREADD-Gs and treated with siPKA as indicated (left panel). Representative immunofluorescence of HA and the mitochondrial membrane potential dye MitoTracker Red™ (n = 4 independent cell preparations for each group) in HEK cells expressing control vector, MitoDREADD-Gs and treated with vehicle, CNO (50 μM) and/or siPKA (middle panel). Quantification of MitoTracker Red™ staining (right panel). ****p<0.0001 according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. (f) Representative immunoblotting of HA, PKA catalytic subunit α (PKA-Cα) and total protein load (TPL) in HeLa cells expressing control vector or mitoDREADD-Gs and treated with siPKA as indicated (left panel). Representative immunofluorescence of HA and the mitochondrial membrane potential dye MitoTracker Red™ (n = 4 independent cell preparations for each group) in HeLa cells expressing control vector, MitoDREADD-Gs and treated with vehicle, CNO (50 μM) and/or siPKA (middle panel). Quantification of MitoTracker Red™ staining (right panel). ****p<0.0001 according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. Data are presented as mean ± SEM. (a) Representative immunoblotting of HA, PKA catalytic subunit α (PKA-Cα), myc and total protein load (TPL) of PRKCA+/+ and PRKCA−/− cells expressing pcDNA, PKA-myc or MitoPKA-myc (n = 3). (b) Representative immunoblotting of AKAP121 and tubulin in HEK cells treated with two different shAKAP121 (n = 3). (c) Effect of CNO (50 μM) on oxygen consumption rates (OCR) of isolated mitochondria obtained from HEK cells expressing mitoDREADD-Gs co-transduced with shAKAP121 constructs. n = 4 independent cell preparations for Vector and MitoDREADD-Gs+vector; n = 6 for MitoDREADD-Gs + shAKAP121-1/2. *p=0.0339, *p = 0.0307, **p = 0.0026 according to one-way ANOVA, two sided, followed by Tukey's multiple comparison test and #p = 0.0265 accoding to one sample t test. (d) Representative immunofluorescence of HA, myc and Mitotracker Red™ in HEK cells co-expressing empty vector, mitoDREADD-Gs and NDUFS4-S173F-myc treated with vehicle or 50 μM CNO as indicated (left panel). Quantification of MitoTracker Red™ staining (right panel). n = 4 independent cell preparations for each construct. ****p<0.0001 according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. (e) Representative immunofluorescence of HA, myc and Mitotracker Red ™ in HeLa cells co-expressing empty vector, mitoDREADD-Gs and NDUFS4-S173F-myc treated with vehicle or 50 μM CNO as indicated (left panel). Quantification of MitoTracker Red™ staining from the left panel (right panel). n = 4 independent cell preparations for each construct. ****p<0.0001 according to two-way ANOVA, two sided, followed by Tukey's multiple comparison test. Data are presented as mean ± SEM. (a) Total exploration in the novel object recognition (NOR) task of D1R-CB1+/+ and D1R-CB1−/− mice expressing empty vector, CB1 or the non-mitochondrial DN22-CB1 within the hippocampus upon post-training intra-peritoneal (i.p.) injection of 5 mg/kg THC. D1R-CB1+/+: vector: n = 13 independent mice for veh, n = 11 for THC; D1R-CB1−/−: vector: n = 5 for veh, n = 6 for THC; D1R-CB1+/+: CB1: n = 7 for veh, n = 8 for THC; D1R-CB1+/+: DN22-CB1: n = 6 for veh, n = 9 for THC. Data were analyzed by two-way ANOVA, two sided. (b) Exploration time of familiar and novel objects in the NOR task of D1R-CB1+/+ and D1R-CB1−/− mice expressing empty vector, CB1 or the non-mitochondrial DN22-CB1 within the hippocampus upon post-training i.p. injection of 5 mg/kg THC. D1R-CB1+/+: vector: n = 13 independent mice for veh, n = 11 for THC; D1R-CB1−/−: vector: n = 5 for veh, n = 6 for THC; D1R-CB1+/+: CB1: n = 7 for veh, n = 8 for THC; D1R-CB1+/+: DN22-CB1: n = 6 for veh, n = 9 for THC. Data were analyzed by two-way ANOVA, two sided followed by Sidak's multiple comparison test. (c) Total exploration in the NOR task of D1R-CRE mice expressing empty vector, DREADD-Gs or mitoDREADD-Gs within the hippocampus upon post-training i.p. injections of vehicle, 5 mg/kg CNO and/or 5 mg/kg THC. Vector+ veh: n = 8 independent mice for veh and THC; Vector+THC: n = 8 for veh and n = 9 for CNO; DREADD-Gs+veh: n = 7 for each group; DREADD-Gs+THC: n = 8 for each group; MitoDREADD-Gs+veh: n = 6 for veh and n = 7 for CNO; MitoDREADD-Gs+THC: n = 6 for veh and n = 5 for CNO. Data were analyzed by two-way ANOVA, two sided. (d) Exploration time of familiar and novel objects in the NOR task of C57BL/6 mice expressing empty vector, and D1R-CRE mice expressing DREADD-Gs or mitoDREADD-Gs within the hippocampus upon post-training i.p. injections of vehicle, 5 mg/kg CNO and/or 5 mg/kg THC. Data were analyzed by two-way ANOVA, two-sided followed by Sidak's multiple comparison test. Data are presented as mean ± SEM. (a) Total exploration in the novel object recognition (NOR) task of WT and P301S mice expressing control vector, DREADD-Gs or mitoDREADD-Gs within the hippocampus upon intra-peritoneal (i.p.) injection of vehicle (veh) or 5 mg/kg CNO. WT+vector: n = 13 independent mice for each group; P301S+vector: n = 10 for each group; P301S + DREADD-Gs: n = 12 for each group; P301S+mitoDREADD-Gs: n = 12 for each group. Data were analyzed by two-way ANOVA, two-sided. (b) Exploration time of familiar and novel objects in the NOR task of WT and P301S mice expressing control vector, DREADD-Gs or mitoDREADD-Gs within the hippocampus upon i.p. injection of veh or 5 mg/kg CNO. WT+vector: n = 13 independent mice for each group; P301S+vector: n = 10 for each group; P301S + DREADD-Gs: n = 12 for each group; P301S+mitoDREADD-Gs: n = 12 for each group. Data were analyzed by mixed-effects analysis, two-sided followed by Bonferroni's multiple comparison test. (c) Total exploration in the NOR task of WT and APP/PS1 mice expressing control vector, DREADD-Gs or mitoDREADD-Gs within the hippocampus upon i.p. WT+vector: n = 9 independent mice for veh, n = 8 for CNO; APP/PS1 + DREADD-Gs: n = 10 for each group;APP/PS1+mitoDREADD-Gs: n = 7 for veh and n = 6 for CNO. Data were analyzed by two-way ANOVA, two-sided. (d) Exploration time of familiar and novel objects in the NOR task of WT and APP/PS1 mice expressing control vector or mitoDREADD-Gs within the hippocampus upon i.p. WT+vector: n = 9 independent mice for veh, n = 8 for CNO; APP/PS1 + DREADD-Gs: n = 10 for each group;APP/PS1+mitoDREADD-Gs: n = 7 for veh and n = 6 for CNO. Data were analyzed by two-way ANOVA, two-sided followed by Sidak's multiple comparison test. Data are presented as mean ± SEM. Representative immunofluorescence of HeLa cells stained with Mitotracker Red. The mitochondrial network (excluding the nucleus region) was selected using the freeform drawing tool and the integrated intensity was normalized by the area covered. Source Data for all figures and ED figures. Uncropped WB for main figures. Uncropped WB for Ext. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Pagano Zottola, A.C., Martín-Jiménez, R., Lavanco, G. et al. Potentiation of mitochondrial function by mitoDREADD-Gs reverses pharmacological and neurodegenerative cognitive impairment in mice. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41589-025-01978-1'>ZBTB11 depletion targets metabolic vulnerabilities in KRAS inhibitor-resistant PDAC</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 10:04:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Chemical Biology (2025)Cite this article Over 95% of pancreatic ductal adenocarcinomas (PDACs) harbor oncogenic mutations in KRAS. However, upon treatment with KRAS inhibitors, PDAC cells undergo rapid metabolic reprogramming toward an oxidative phosphorylation (OXPHOS)-dependent, drug-resistant state. Here, we demonstrate that this metabolic shift is associated with upregulation of the transcription factor ZBTB11 and both the metabolic state and resistance to KRAS inhibitors can be attenuated by ZBTB11 depletion. We develop molecular glue degraders of ZBTB11 and demonstrate that they reprogram the aberrant transcriptome, proteome, metabolome and bioenergetics of KRAS inhibitor-resistant PDAC, resensitizing them to KRAS inhibition. ZBTB11 degradation leverages cell-type-specific and cell-state-specific differences in gene-regulatory mechanisms controlling OXPHOS pathway transcripts to selectively target the KRAS inhibitor-resistant state in PDAC while sparing neurons derived from human induced pluripotent stem cells. Together, this work establishes ZBTB11 as a druggable vulnerability in KRAS inhibitor-resistant PDAC and provides a suite of molecular glue degrader tool compounds to investigate its function. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The RNA-seq transcriptomics data were deposited to the Gene Expression Omnibus data repository with dataset identifiers GSE298803 (ZBTB11 CRISPRi RNA-seq) and GSE298925 (sotorasib/JWJ-01-306 RNA-seq). The MS proteomics data were deposited to the ProteomeXchange Consortium through the PRIDE82 partner repository with dataset identifier PXD052360. Batch scripts for prepacking, relaxing and clustering Rosetta structure models, as well as the ten top-scoring structure clusters, are available upon request to N.L.T. Source data are provided with this paper. Cox, A. D., Fesik, S. W., Kimmelman, A. C., Luo, J. & Der, C. J. Drugging the undruggable RAS: mission possible? Strickler, J. H. et al. First data for sotorasib in patients with pancreatic cancer with KRAS p.G12C mutation: a phase I/II study evaluating efficacy and safety. Moore, A. R., Rosenberg, S. C., McCormick, F. & Malek, S. RAS-targeted therapies: is the undruggable drugged? Punekar, S. R., Velcheti, V., Neel, B. G. & Wong, K. K. The current state of the art and future trends in RAS-targeted cancer therapies. Zheng, Q., Zhang, Z., Guiley, K. Z. & Shokat, K. M.Strain-release alkylation of Asp12 enables mutant selective targeting of K-Ras-G12D. Zhang, Z., Morstein, J., Ecker, A. K., Guiley, K. Z. & Shokat, K. M. Chemoselective covalent modification of K-Ras(G12R) with a small molecule electrophile. Zhang, Z., Guiley, K. Z. & Shokat, K. M. Chemical acylation of an acquired serine suppresses oncogenic signaling of K-Ras(G12S). Skoulidis, F. et al. Sotorasib for lung cancers with KRAS p.G12C mutation. Riely, G. J. et al. 99O_PR KRYSTAL-1: activity and preliminary pharmacodynamic (PD) analysis of adagrasib (MRTX849) in patients (Pts) with advanced non-small cell lung cancer (NSCLC) harboring KRASG12C mutation. Strickler, J. H. et al. Sotorasib in KRAS p.G12C-mutated advanced pancreatic cancer. Zhao, Y. et al. Diverse alterations associated with resistance to KRAS(G12C) inhibition. Gulay, K. C. M. et al. Dual Inhibition of KRASG12D and pan-ERBB is synergistic in pancreatic ductal adenocarcinoma. Awad, M. M. et al. Acquired resistance to KRAS(G12C) inhibition in cancer. Tanaka, N. et al. Clinical acquired resistance to KRASG12C inhibition through a novel KRAS switch-II pocket mutation and polyclonal alterations converging on RAS–MAPK reactivation. Isermann, T., Sers, C., Der, C. J. & Papke, B. KRAS inhibitors: resistance drivers and combinatorial strategies. Brown, W. S. et al. Overcoming adaptive resistance to KRAS and MEK inhibitors by co-targeting mTORC1/2 complexes in pancreatic cancer. Muzumdar, M. D. et al. Survival of pancreatic cancer cells lacking KRAS function. Santana-Codina, N. et al. Oncogenic KRAS supports pancreatic cancer through regulation of nucleotide synthesis. Wasko, U. N. et al. Tumor-selective activity of RAS–GTP inhibition in pancreatic cancer. Viale, A. et al. Oncogene ablation-resistant pancreatic cancer cells depend on mitochondrial function. Santana-Codina, N. et al. Defining and targeting adaptations to oncogenic KRAS(G12C) inhibition using quantitative temporal proteomics. Bridges, H. R., Jones, A. J., Pollak, M. N. & Hirst, J. Effects of metformin and other biguanides on oxidative phosphorylation in mitochondria. Betarbet, R. et al. Chronic systemic pesticide exposure reproduces features of Parkinson's disease. Sanchez, M., Gastaldi, L., Remedi, M., Caceres, A. & Landa, C. Rotenone-induced toxicity is mediated by Rho-GTPases in hippocampal neurons. Yap, T. A. et al. Complex I inhibitor of oxidative phosphorylation in advanced solid tumors and acute myeloid leukemia: phase I trials. Wu, M., Arimura, G. K. & Yunis, A. A. Purification and characterization of a plasminogen activator secreted by cultured human pancreatic carcinoma cells. Arroyo, J. D. et al. A genome-wide CRISPR death screen identifies genes essential for oxidative phosphorylation. Sharma, K. R., Colvis, C. M., Rodgers, G. P. & Sheeley, D. M. Illuminating the druggable genome: pathways to progress. Schneider, M. et al. The PROTACtable genome. Harripaul, R. et al. Mapping autosomal recessive intellectual disability: combined microarray and exome sequencing identifies 26 novel candidate genes in 192 consanguineous families. Fattahi, Z. et al. Biallelic missense variants in ZBTB11 can cause intellectual disability in humans. Wilson, B. C. et al. Intellectual disability-associated factor ZBTB11 cooperates with NRF-2/GABP to control mitochondrial function. Wang, T. et al. Identification and characterization of essential genes in the human genome. Blomen, V. A. et al. Gene essentiality and synthetic lethality in haploid human cells. Tsherniak, A. et al. Defining a cancer dependency map. Tian, R. et al. CRISPR interference-based platform for multimodal genetic screens in human iPSC-derived neurons. Terman, A., Kurz, T., Navratil, M., Arriaga, E. A. & Brunk, U. T. Mitochondrial turnover and aging of long-lived postmitotic cells: the mitochondrial–lysosomal axis theory of aging. Sievers, Q. L. et al. Defining the human C2H2 zinc finger degrome targeted by thalidomide analogs through CRBN. Vetma, V. et al. Confounding factors in targeted degradation of short-lived proteins. Tran, N. L., Leconte, G. A. & Ferguson, F. M. Targeted protein degradation: design considerations for PROTAC development. Mahan, S. D., Riching, K. M., Urh, M. & Daniels, D. L. Kinetic detection of E3:PROTAC:target ternary complexes using NanoBRET technology in live cells. Wu, W. et al. Overcoming IMiD resistance in T-cell lymphomas through potent degradation of ZFP91 and IKZF1. Ting, P. Y. et al. A molecular glue degrader of the WIZ transcription factor for fetal hemoglobin induction. Matyskiela, M. E. et al. A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Drake, Z. C., Seffernick, J. T. & Lindert, S. Protein complex prediction using Rosetta, AlphaFold, and mass spectrometry covalent labeling. Watson, E. R. et al. Molecular glue CELMoD compounds are regulators of cereblon conformation. Kim, D., Lee, E. C., Kim, K. S. & Tarakeshwar, P. Cation–π–anion interaction: a theoretical investigation of the role of induction energies. Mullen, A. R. et al. Reductive carboxylation supports growth in tumour cells with defective mitochondria. Metallo, C. M. et al. Reductive glutamine metabolism by IDH1 mediates lipogenesis under hypoxia. Mullen, A. R. et al. Oxidation of α-ketoglutarate is required for reductive carboxylation in cancer cells with mitochondrial defects. Fendt, S. M. et al. Reductive glutamine metabolism is a function of the α-ketoglutarate to citrate ratio in cells. Cheng, G. et al. Profiling and targeting of cellular bioenergetics: inhibition of pancreatic cancer cell proliferation. Tiriac, H. et al. Organoid profiling identifies common responders to chemotherapy in pancreatic cancer. Tsuji, A., Akao, T., Masuya, T., Murai, M. & Miyoshi, H. IACS-010759, a potent inhibitor of glycolysis-deficient hypoxic tumor cells, inhibits mitochondrial respiratory complex I through a unique mechanism. Kronke, J. et al. Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS. Fink, E. C. et al. CRBN (I391V) is sufficient to confer in vivo sensitivity to thalidomide and its derivatives in mice. Donovan, K. A. et al. Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane radial ray syndrome. Pre, D., Wooten, A. T., Zhou, H., Neil, A. & Bang, A. G. Assaying chemical long-term potentiation in human iPSC-derived neuronal networks. Pre, D. et al. Development of a platform to investigate long-term potentiation in human iPSC-derived neuronal networks. Puppo, F., Pre, D., Bang, A. G. & Silva, G. A. Super-selective reconstruction of causal and direct connectivity with application to in vitro iPSC neuronal networks. Sherman, S. P. & Bang, A. G.High-throughput screen for compounds that modulate neurite growth of human induced pluripotent stem cell-derived neurons. Farge, T. et al. Chemotherapy-resistant human acute myeloid leukemia cells are not enriched for leukemic stem cells but require oxidative metabolism. Henkenius, K. et al. Maintenance of cellular respiration indicates drug resistance in acute myeloid leukemia. Lee, K. M. et al. Myc and MCL1 cooperatively promote chemotherapy-resistant breast cancer stem cells via regulation of mitochondrial oxidative phosphorylation. Navarro, P. et al. Targeting tumor mitochondrial metabolism overcomes resistance to antiangiogenics. Guieze, R. et al. Mitochondrial reprogramming underlies resistance to BCL-2 inhibition in lymphoid malignancies. Kuntz, E. M. et al. Targeting mitochondrial oxidative phosphorylation eradicates therapy-resistant chronic myeloid leukemia stem cells. Haq, R. et al. Oncogenic BRAF regulates oxidative metabolism via PGC1α and MITF. Boj, S. F. et al. Organoid models of human and mouse ductal pancreatic cancer. Schwinn, M. K. et al. CRISPR-mediated tagging of endogenous proteins with a luminescent peptide. Romero, N., Rogers, G., Neilson, A. & Dranka, B. P. Quantifying Cellular ATP Production Rate Using Agilent Seahorse XF Technology (Agilent, 2018). Desousa, B. R. et al. Calculation of ATP production rates using the Seahorse XF analyzer. Mudge, J. M. et al. GENCODE 2025: reference gene annotation for human and mouse. Nucleic Acids Res. Ewels, P. A. et al. The nf-core framework for community-curated bioinformatics pipelines. Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Chaudhry, A., Shi, R. & Luciani, D. S. A pipeline for multidimensional confocal analysis of mitochondrial morphology, function, and dynamics in pancreatic β-cells. Bunk, B. et al. MetaQuant: a tool for the automatic quantification of GC/MS-based metabolome data. Ewald, J. D. et al. Web-based multi-omics integration using the Analyst software suite. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Ma, M. et al. Proteome-wide profiling of asymmetric dimethylated arginine in human breast tumors. Perez-Riverol, Y. et al. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. Nucleic Acids Res. was supported by the National Science Foundation Graduate Research Fellowship (DGE-2038238) and the National Institutes of Health (NIH) National Cancer Institute (NCI) Cancer Cell Signaling and Communication Training Grant (T32CA009523). was supported by an NIH Molecular Biophysics Training Grant (T32GM139795). were supported by a grant from the Lustgarten Foundation and the Research for a Cure of Pancreatic Cancer fund. were supported by the Chan Zuckerberg Initiative Imaging Scientist Award (https://doi.org/10.37921/694870itnyzk) from the Chan Zuckerberg Initiative donor-advised fund, an advised fund of the Silicon Valley Community Foundation (https://doi.org/10.13039/100014989), and by the UC San Diego Goeddel Family Technology Sandbox and the Thermo Fisher–University of California, San Diego (UCSD) partnership. received additional financial support from the Spanish National Research Council. Research reported in this publication was supported by the NCI Cancer Center Support Grant P30 CA030199 and Shared Instrumentation Grant S10 OD036254. We would like to acknowledge support from the UCSD Biomolecular/Proteomics MS Facility Proteomics Core and the Sanford Burnham Prebys Proteomics, Cancer Metabolism, Genomics, Bioinformatics and Functional Genomics Core Facilities. We would also like to acknowledge N. Gray for providing the CRBN-targeting molecular glue screening library. These authors contributed equally: Nathan L. Tran, Jiewei Jiang. Department of Chemistry and Biochemistry, University of California, San Diego, La Jolla, CA, USA Nathan L. Tran, Jiewei Jiang, Min Ma, Gillian E. Gadbois & Fleur M. Ferguson Cancer Metabolism and Microenvironment Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, CA, USA Nathan L. Tran & Eric S. Wang Department of Surgery, Division of Surgical Oncology, UCSD Moores Cancer Center, University of California, San Diego, La Jolla, CA, USA Kevin C. M. Gulay, Anne G. Bang, Herve Tiriac & Andrew M. Lowy Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA, USA Department of Cell and Developmental Biology, University of California, San Diego, La Jolla, CA, USA Cara R. Schiavon, Elena Rebollo & Uri Manor Molecular Biology Institute of Barcelona, Barcelona, Spain Sanford Burnham Prebys Medical Discovery Institute, La Jolla, CA, USA Haowen Zhou, Chun-Teng Huang, Rabi Murad & David A. Scott Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, La Jolla, CA, USA Fleur M. Ferguson Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar generated sotorasib-resistant, HiBiT-tagged and NLuc-tagged ZBTB11 and ZBTB11 CRISPRi cell lines. performed Seahorse, reverse transcription–qPCR, cell counting proliferation, HiBiT and NLuc luminescence, NanoBRET, Rosetta computational modeling and mitochondrial staining, prepared figures and edited the paper. performed cell viability, immunoblots and molecular cloning. performed and analyzed transcriptomics. performed and analyzed metabolomics. performed and analyzed proteomics. performed CRBN engagement assays. designed and performed molecule synthesis. designed and generated CRISPRi constructs. developed MRTX1133-resistant cell lines, performed bright-field imaging, proliferation and synergy assays and edited the paper. designed and performed the mitochondrial imaging assays and analysis and prepared figures. performed and analyzed high-throughput iPS cell-derived neuron imaging assays. conceptualized the mitochondrial imaging assays and acquired funding. E.S.W supervised the research, acquired funding, performed study design and analysis, prepared figures and edited the paper. supervised the research, acquired funding, performed study design and analysis, prepared figures and wrote the paper (first draft). All authors read and approved the final paper. Correspondence to Eric S. Wang or Fleur M. Ferguson. are inventors on a patent application related to this work (US 63/515,472). lab receives or has received research resources in kind from SK Life Sciences. is a scientific cofounder and equity holder in Proximity Therapeutics and was previously a scientific advisory board member of Triana Biomedicines. is or was recently a consultant or received speaking honoraria from RA Capital, Tocris BioTechne, Eli Lilly, Amgen, Neomorph and Plexium. lab receives or has received research funding, travel funding or resources in kind from Ono Pharmaceutical, Eli Lilly, Promega and Merck. 's interests have been reviewed and approved by UCSD in accordance with its conflict-of-interest policies. The remaining authors declare no competing interests. Nature Chemical Biology thanks Seth Parker and the other, anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. MIA PaCa-2 cell viability following treatment with sotorasib for 3 days. Data are normalized to vehicle treatment controls. b. MIA PaCa-2 cell proliferation following treatment with the indicated compound over a 14-day period. c-d. MIA PaCa-2 and SUIT2 cell proliferation following treatment with the indicated compound(s) over a 14-day period. e. Brightfield images of MIA PaCa-2 cells in culture at 20X magnification. f. Confocal images of MIA PaCa-2 cells stained with α-HSP60 1° antibody and Alexa Fluor™ 568-conjugated 2° antibody at 63X magnification. g. Quantified 3D mitochondrial morphology in stained MIA PaCa-2 cells. h. Example confocal image of stained MIA PaCa-2 cells with peripheral region marked (area between the two blue lines). i. Confocal images of MIA PaCa-2 cells stained as in Extended Data Fig. 1F with peripheral regions marked. j. Quantified distribution of mitochondria in stained MIA PaCa-2 cells. k-l. MIA PaCa-2 and SUIT2 cells were grown with 13C-labeled glucose or glutamine for 24 hours, and targeted metabolomics analysis was performed by GC-MS/MS followed by MSEA. Representative top-scoring pathways are shown here with corresponding significance and enrichment (HitRatio) scores of n = 6 biological replicates. Significance was determined by performing 1000 permutation tests and comparing the observed enrichment score to the background distribution. Source Data in Supplementary Data 1. a-d. Data are depicted as the average +/- S. D. of n = 3 biological replicates. Data are depicted as the average +/- S. D. of at least n = 10 biological replicates. Significance was determined by performing Brown-Forsythe and Welch ANOVA tests with Dunnett's T3 test for multiple comparisons. Scale bars measuring indicated length are provided. a-f. MIA PaCa-2 CRISPRi cellular transcript abundance following treatment with doxycycline for the indicated amount of time. Data are normalized to vehicle treatment controls. MIA PaCa-2 CRISPRi cell proliferation following treatment with the indicated compound(s) over a 14-day period. k. Sequence alignment of ZBTB11 and other C2H2 zinc finger-containing proteins, as well as GSPT1, with the C2H2 motif and beta hairpin glycine highlighted. l. Protein half-life of ZBTB11 in the indicated cell line33. Data are depicted as the average +/- S. D. of n = 3 biological replicates. Chemical structures of validated screening hits. b. MIA PaCa-2 cellular ZBTB11-HiBiT abundance following treatment with the indicated concentration of compound for 5 hours. c. MIA PaCa-2 cellular protein abundance following treatment with JWJ-01-306 for the indicated amount of time. d. Data table of DC50 and Dmax values of JWJ-01-306 against ZBTB11 calculated from HiBiT lytic assay (Fig. 2f) and from Western blot (Extended Data Fig. e. HEK293 cellular CRBN engagement following treatment with the indicated concentration of compounds for 2 hours. f. MIA PaCa-2 cells were treated with 10 µM JWJ-01-368 or DMSO vehicle for 5 hours, and global proteomics analysis was performed by LC-MS/MS followed by differential gene expression analysis. Significance was determined by performing two-sample moderated t-tests with two-tailed distribution. Source Data in Supplementary Data 1. g-h. MIA PaCa-2 or SUIT2 cell proliferation following treatment with the indicated compound(s) over a 14-day period. i, k, m. Cellular protein abundance in the indicated cell line model following treatment with JWJ-01-306 for 5 hours. j, l. Jurkat cellular IKZF1-HiBiT or GSPT1-HiBiT abundance following treatment with the indicated concentration of compound for 5 hours. n. Sequence alignment of the 12 C2H2 zinc fingers of ZBTB11 with the C2H2 motif, beta hairpin glycine, and ZF10-unique lysine highlighted. o. MOLT-4 cellular NLuc-ZBTB11 ZF abundance following treatment with 10 µM ALV-05-184 for 5 hours. p. HEK293 cellular CRBN engagement following treatment with the indicated concentration of JWJ-01-306 for 2 hours prior to washout. q. HEK293T cellular ternary complex abundance following treatment with the indicated concentration of JWJ-01-306 for 5 hours prior to washout. r-s. MIA PaCa-2 cellular ZBTB11-HiBiT abundance following treatment with 10 µM JWJ-01-306 for 5 hours prior to washout and subsequent incubation for the indicated amount of time. b, e-h, j, l, p-s. Data are depicted as the average +/- S. D. of n = 3 biological replicates and are normalized to vehicle treatment controls. c, k. Data are representative of n = 2 independent experiments. M. Data are representative of n = 3 independent experiments. a, d, g. MIA PaCa-2 cellular OCR during a mitochondrial stress test following treatment with 10 µM of the indicated compound for 24 hours. From left to right, gray dotted lines indicate additions of oligomycin, BAM15, and rotenone/antimycin A. Data are depicted as the average +/- S. D. of n = 2 biological replicates. b, e, h. MIA PaCa-2 cellular glycoATP and mitoATP. Data are depicted as the average +/- S. D. of n = 2 biological replicates with n = 3 technical replicates each. c, f, i-j. MIA PaCa-2 cellular transcript abundance following treatment with 10 µM of the indicated compound for 24 hours. Data are depicted as the average +/- S. D. of n = 3 biological replicates and are normalized to vehicle treatment controls. k. MIA PaCa-2 SOTR cells were treated with 10 µM JWJ-01-306 or DMSO vehicle for 24 hours prior to incubation with media containing 13C-labeled glucose or glutamine for 24 hours, and targeted metabolomics analysis was performed by GC-MS/MS followed by MSEA. Representative top-scoring pathways are shown here with corresponding significance and enrichment (HitRatio) scores of n = 6 biological replicates. Significance was determined by performing 1000 permutation tests and comparing the observed enrichment score to the background distribution. Source Data in Supplementary Data 1. a. SUIT2 cellular OCR during a mitochondrial stress test following treatment with 10 µM of the indicated compound for 24 hours. From left to right, gray dotted lines indicate additions of oligomycin, BAM15, and rotenone/antimycin A. Data are depicted as the average +/- S. D. of n = 2 biological replicates. b. SUIT2 cellular glycoATP and mitoATP. Data are depicted as the average +/- S. D. of n = 2 biological replicates with n = 3 technical replicates each. c-d. MIA PaCa-2 cellular transcript abundance following treatment with 10 µM of the indicated compound for 24 hours. Data are depicted as the average +/- S. D. of n = 3 biological replicates and are normalized to vehicle treatment controls. e. Quantified 3D mitochondrial morphology in stained MIA PaCa-2 SOTR cells (Fig. f. Confocal images of MIA PaCa-2 SOTR cells stained as in Fig. 3n with peripheral regions marked. A scale bar measuring 10 µm is provided. g. Quantified distribution of mitochondria in stained MIA PaCa-2 cells. e, g. Data are depicted as the average +/- S. D. of at least n = 20 biological replicates. Significance was determined by performing Brown-Forsythe and Welch ANOVA tests with Dunnett's T3 test for multiple comparisons. a-d. MIA PaCa-2 cell proliferation following treatment with the indicated compound(s) over a 14-day period. e-h. SUIT2 cell proliferation following treatment with the indicated compound(s) over a 14-day period. MIA PaCa-2 ZBTB11WT cell proliferation following treatment with the indicated compound(s) over a 14-day period. k-l. SUIT2 ZBTB11WT cell proliferation following treatment with the indicated compound(s) over a 14-day period. m-p. MIA PaCa-2 and SUIT2 cellular OCR during a mitochondrial stress test following treatment with the indicated concentration of IACS-010759 for 24 hours. q-t. CAPAN-1, BxPC-3, AsPC-1, and PANC-1 PDAC cellular OCR during a mitochondrial stress test following treatment with 10 µM JWJ-01-306 for 24 hours. a-l. Data are depicted as the average +/- S. D. of n = 3 biological replicates. m-t. From left to right, gray dotted lines indicate additions of oligomycin, BAM15, and rotenone/antimycin A. Data are depicted as the average +/- S. D. of n = 3 biological replicates. a-f. MIA PaCa-2 cell viability following treatment with the indicated concentrations of sotorasib and JWJ-01-306, JWJ-01-368, or IACS-01-0759 for 7 days. Data are depicted as the average of n = 2 biological replicates. g-l. SUIT2 cell viability following treatment with the indicated concentrations of MRTX1133 and JWJ-01-306, JWJ-01-368, or IACS-01-0759 for 7 days. Data are depicted as the average of n = 2 biological replicates. MIA PaCa-2 SOTR cells were treated with 1 µM sotorasib, 10 µM JWJ-01-306, 1 µM sotorasib + 10 µM JWJ-01-306, or DMSO vehicle for 3 days, and global proteomics analysis was performed by LC-MS/MS. Protein abundances were filtered by one-way ANOVA (FDR < 0.05) and clustered using K-nearest neighbors clustering. Each protein cluster was then input to GSEA for pathway enrichment analysis using GO and KEGG pathways. The abundance of each protein in a given cluster is visualized as a line plot and is color-coded according to its distance from the trend of the cluster center, ranging from purple (close) to blue (far). Representative top-scoring pathways are listed to the right of the abundance line plots. Protein abundances and associated clusters are shown here with corresponding fold change values of n = 3 biological replicates. Source Data in Supplementary Data 1. MIA PaCa-2 SOTR cells were treated with 1 µM sotorasib, 10 µM JWJ-01-306, 1 µM sotorasib + 10 µM JWJ-01-306, or DMSO vehicle for 5 days, and global proteomics analysis was performed by LC-MS/MS. Protein abundances were filtered by one-way ANOVA (FDR < 0.05) and clustered using K-nearest neighbors clustering. Each protein cluster was then input to GSEA for pathway enrichment analysis using GO and KEGG pathways. The abundance of each protein in a given cluster is visualized as a line plot and is color-coded according to its distance from the trend of the cluster center, ranging from purple (close) to blue (far). Representative top-scoring pathways are listed to the right of the abundance line plots. Protein abundances and associated clusters are shown here with corresponding fold change values of n = 3 biological replicates. Source Data in Supplementary Data 1. Significant protein hits from the JWJ-01-306 treatment condition of the 3-day global proteomics (Extended Data Figs. 7) and 5-day global proteomics (Extended Data Fig. 8) were compared with protein entries in the MitoCarta 3.0 database. Protein abundances of ZBTB11, genes regulated by ZBTB11, and select other gene products in each treatment condition for both 3- and 5-day global proteomics experiments. Data are depicted as the median value, with the bounds indicating the 25th and 75th percentiles. All Source Data in Supplementary Data 1. Oligonucleotide sequences and processed transcriptomics, metabolomics and proteomics data. Detailed descriptions are listed on the first tab. Statistical source data. Unprocessed western blots. Statistical source data. Unprocessed western blots. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Unprocessed western blots. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Tran, N.L., Jiang, J., Ma, M. et al. ZBTB11 depletion targets metabolic vulnerabilities in KRAS inhibitor-resistant PDAC. Nat Chem Biol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemical Biology (Nat Chem Biol) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44185-025-00095-5'>Towards positive net outcomes for biodiversity, and developing safeguards to accompany headline biodiversity indicators</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 09:43:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Achieving the Global Biodiversity Framework will necessitate whole production systems contributing towards ‘halting and reversing' net biodiversity loss, counterbalancing negative impacts with comparable gains. Here, we report on an illustrative quantitative exploration into the feasibility of monitoring for positive net biodiversity outcomes for the Dutch dairy production sector, using a composite metric. We analysed performance data from 8,950 dairy farms across the Netherlands, combining these data into an integrated biodiversity index. Usefully, this index allowed us to calculate sectoral baseline biodiversity impacts, and explore possible biodiversity strategies. We show that the largest overall source of impacts is imported feed; interestingly, nutrient loads contribute little to the footprint, despite representing an important political issue nationally. This highlights a general risk in using single indices to track net biodiversity outcomes: that they could result in an exclusionary focus, and perverse outcomes. Consequently, we develop safeguards to accompany the index; showing the necessity of incorporating safeguards, but also that meeting them could reduce sectoral biodiversity impacts by ~94%. Our proposed strategies vary in feasibility, all requiring trade-offs between biodiversity, land availability, and production. In late 2022, the Convention on Biological Diversity adopted the Global Biodiversity Framework, aiming to “halt and reverse biodiversity loss to put nature on a path to recovery”1. Achieving this mission requires addressing biodiversity impacts not only from direct economic activities but also those embedded in global supply chains. Agriculture, as a leading driver of biodiversity loss, presents both challenges and opportunities in this context2,3. The path to global nature recovery will involve combining reactive mitigation for negative impacts on biodiversity with proactive biodiversity conservation and restoration initiatives4. Key principles will include that biodiversity impacts caused by economic development are quantified, reduced where possible, otherwise compensated for through biodiversity offsets, and accompanied by additional proactive conservation initiatives; leading to net positive biodiversity impacts overall. Here, we explore ‘net outcome' approaches on biodiversity at the scale of a national agricultural production system (the Dutch dairy sector, using data collected via the established Dutch ‘Biodiversity Monitor'), as one necessary step in quantifying pathways towards global biodiversity goals. The idea of ‘net outcome' type mechanisms for biodiversity (i.e. those that seek to quantify biodiversity losses and sum them against comparably quantified gains, to calculate a net outcome for biodiversity) is not new, and currently forms the basis of established or emerging policy in over 100 countries worldwide, a growing conservation portfolio, as well as underpinning environmental markets worth billions of dollars a year5. Many of the theoretical challenges associated with achieving neutral or even positive net outcomes have been explored in the scientific literature—with the most acute current barriers more related to practical considerations, e.g. monitoring, implementation, and demonstration in practice (e.g. refs. Crucially, in implementing the Global Biodiversity Framework, it will be insufficient to consider only the direct biodiversity impacts of economic activities, but also necessary to consider their indirect impacts on biodiversity that are embedded in the supply chain. This thinking partly underpins the emerging concept of ‘nature positive'3,8—a specific type of positive net outcome approach—current definitions for which are explored in ref. 8, but which under the definition we assume (see below and www.naturepositive.org) is directly aligned with the CBD post-2020 mission. The evaluation of biodiversity impacts for specific organisations—both direct and indirect—as a basis for developing positive net outcome-type strategies (again, of which ‘nature positive' is one example), has started to receive attention in the recent scientific literature9. This includes efforts to understand the organisational impacts of food consumption10. Nonetheless, of all of the key sectors in the global economy, agriculture is one that has seen relatively little attention in terms of net outcomes approaches. Agriculture will be an extremely influential sector in developing net outcome pathways: fundamental to society, but a leading global cause of biodiversity loss (e.g. refs. 11,12,13), and one that will play a key role in whether international agreements on climate change can be achieved14. Equally, agriculture could play an important role in biodiversity recovery—not only through reduction of impacts, but through restorative practices15. Further, agriculture as a sector represents a clear illustrative example of the challenges in trading off any efforts to stay within physical planetary boundaries with efforts to ensure society can meet core social objectives on wellbeing16. Here, we use an illustrative case study of dairy production in the Netherlands. Dairy farming could feasibly help support represent a ‘land sharing' approach to nature conservation: cows grazing at low densities in herb-rich grasslands, interspersed with certain well-connected landscape elements, could provide habitat for a substantial diversity of wild species. However, such landscapes have diminished globally because of a focus on dairy production efficiency, with dairy farmers primarily paid for milk sold in a competitive global market where their contribution to biodiversity receives little reward. The Dutch Sustainable Dairy Chain (DuurzameZuivelKeten, or DZK)—a collaboration between key public and private dairy organisations in the Netherlands—has recently shown interest in calculating net biodiversity outcomes associated with national dairy production; and understanding to what degree the sector could support nature positive goals. In defining nature positive, the Nature Positive Initiative states “Companies and financial institutions can contribute to the Nature Positive goal by taking these high-level actions: Assess their material impacts, dependencies, risks and opportunities; shift their business strategy and models; commit to science-based targets for nature; report their nature-related issues to investors and other stakeholders; transform by avoiding and reducing negative impacts, restoring, and regenerating nature; collaborate across land, seascapes and river basins; and advocate to governments for policy ambition”. A key point here is the concept of contributing towards an overarching absolute nature positive objective (say, on the scale of the Netherlands); which DZK could begin to do by first assessing impacts, and then developing a positive net outcome strategy within the production sector itself (see refs. Note also that a nature positive pathway is considered by the NPI to be one in which connected global challenges beyond biodiversity (climate change, water use, other forms of environmental pollution—alongside other sustainability issues, such as human wellbeing) are solved in tandem; but here, we focus on biodiversity. Similarly to multiple other organisations, DZK has sought in the first instance to track net biodiversity outcomes against a strategy using one main focal biodiversity metric, acting as a proxy. Our analyses are underpinned by data collected using the Dutch Biodiversity Monitor. The Biodiversity Monitor for Dairy Farming, developed by FrieslandCampina, Rabobank and WWF (World-Wide Fund for Nature) Netherlands, aims to enable Dutch dairy farms to improve biodiversity outcomes whilst ensuring sustainable revenue. It uses seven key performance indicators (KPIs) to measure the environmental impacts of individual dairy farms (both positive and negative). The Biodiversity Monitor consequently provides the basis of one potential system for calculating losses and gains of biodiversity towards a net positive impact; by combining KPI data treated as environmental pressures, characterising those pressures into biodiversity impacts in terms of the potentially disappeared fraction (PDF) of species, and aggregating these into a single measure of PDF.year (Fig. A flow diagram showing the calculation of the specific biodiversity indicator we use here to track net biodiversity outcomes. Process incorporates biodiversity monitor key performance indicator data into a single aggregated index, measured in ‘PDF.year'. Here—having developed an integrated biodiversity index to act as a metric for monitoring progress towards a nature positive goal, we: (a) calculate overall sectoral impacts for a baseline year; (b) define a set of biodiversity ‘safeguards', i.e. sub-targets to be achieved to ensure that a net positive outcome measured using the integrated index would not lead to perverse outcomes (e.g., by exceeding physical limits linked to one or more planetary boundaries); and (c) discuss broad strategies that could take the sector from baseline impacts towards positive net outcomes for biodiversity. 1) come from a screened dataset consisting of 8950 farms (see the “Methods” section), which was then factored up to represent the whole sector—a total of 14,542 farms covering 870,880 ha19. The largest source of biodiversity impacts under our analysis comes from land transformation—mostly linked to the production of ingredients used in purchased feeds (e.g., concentrates and other roughage and by-products). Ingredients derived from oil palm products were linked to ~60% of land use change impacts associated with purchased concentrate feed (noting that land transformation impacts from soy, one of the key ingredients, were assumed to be zero, due to the sector currently sourcing 100% RTRS or equivalent certified soy). The second highest source of impacts was associated with land use change on farms within the Netherlands (negative impacts from land transformation and positive impacts from land restoration). This was calculated from annual reductions and increases in ecologically valuable landscape components (“Nature & Landscape”) and areas of herb-rich grassland: for example, a farm that reduced/increased its area of Nature & Landscape elements relative to the previous year would incur an associated biodiversity loss/gain. These losses and gains were then summed across farms to obtain the sectoral values. Annual biodiversity impacts for the year 2020, in –PDF.year, broken down per KPI. LUC land use change, NL impacts within the Netherlands. Note: Other embedded impacts (non-LUC impacts) from purchased feed include those from land occupation, marine and freshwater eutrophication, and water consumption. Embedded greenhouse gas emissions associated with feed are included under the GHG emissions KPI. Impacts from other KPIs are comparatively lower than those from land-related KPIs, which is consistent with land use change being the primary driver of global biodiversity loss20. However, this does not negate the importance of other KPIs (e.g., nitrogen soil surplus, ammonia emissions) which are particularly important as drivers of local to national biodiversity loss, and highly contentious issues within the Netherlands. It is commonly advised not to focus on the results for biodiversity footprints in PDF.year or similar metrics in absolute terms, but on the relative impacts across activities, which is more instructive9. Though the practice is not uncommon, there are challenges in seeking to achieve a biodiversity net outcome measured using a single composite index. Not only are there considerable uncertainties in organisational biodiversity proxies (including those based on LCIA); there is also the potential to mask undesirable outcomes. Consequently, we identified a set of safeguards specific to the Dutch dairy case to be implemented in conjunction with the biodiversity index; seeking to avoid perverse outcomes, and allowing biodiversity strategy to be set at the level of midpoint pressures rather than on the basis of the biodiversity index. In the context of this project, based on literature review and stakeholder consultation (see the “Methods” section), we define a ‘safeguard' as: a standard put in place to ensure that a positive net outcome is achieved while limiting any unintended, undesirable, or perverse biodiversity outcomes (given that ‘net outcomes' approaches have at times led to the latter6). Table 1 lists identified values for each safeguard category (see the “Methods” section), defined at the sector level (i.e., not for individual farms), and categorised into two groups that apply at different points of the mitigation hierarchy (a widely used framework for the preferred sequence of actions to take in mitigating biodiversity impacts, see e.g. ref. Safeguards (1) and (2) define the minimum impacts that should first be avoided or reduced (i.e., the ‘unacceptable' impacts on biodiversity). These are determined from ‘static' goals or standards—such as the goal to buy 100% responsibly sourced soy, or to adhere to nitrogen standards as set out in the EU Nitrates Directive. Safeguards (3)–(5) define how unavoidable impacts should be compensated in line with best practices for biodiversity (e.g., through restoring or offsetting). These are determined by the type of impacts that have taken place (e.g., the habitat types and locations that have been affected), and when those impacts occurred. Given the potential importance of the identified safeguards in developing a net positive strategy, we calculated the baseline values for current sector performance against each Biodiversity Monitor KPI separately (Fig. herb-rich grassland (HRG) and Nature and Landscape KPIs have high levels of uncertainty having been extrapolated from fewer data points. While the change in herb-rich grassland between 2019 and 2020 matches well with the ‘Natural Grassland' figures reported by CBS for the dairy sector, our estimated total value of HRG is about three times larger than the Natural Grassland value reported by CBS for 2020. If all biophysical safeguards listed in Table 1 were met, the annual impact would be substantially less—but still, positive net outcomes (labelled for this specific project by DZK as net positive impact or NPI) would not be achieved without additional offsetting efforts (Fig. Comparison between 2020 impacts vs. impacts if all proposed preventative safeguards were met at the sectoral level. Our finding that a main contributor to overall sectoral biodiversity impacts is land use change in the supply chain is not necessarily unexpected (see refs. Thus, if any strategy was optimised using this index without safeguards as we proposed, it would run the risk of ignoring the importance of nutrient balances as a key issue in the Netherlands specifically. Biodiversity impacts can be considered at global, national, or local scales, and be assigned markedly different values across these scales—necessitating the inclusion of safeguards partly derived from local perspectives when performing calculations based on headline indicators and global trade flows. The safeguards employed here are specific to the Dutch case study, but the safeguard categories (Table 1) are not; the same categories could be used to structure safeguards for other agricultural production systems or even other sectors entirely. In essence, these categories draw from decades of work into biodiversity impacts mitigation hierarchies, extended to organisational or sectoral scales (see refs. Safeguards are likely especially important when one considers the degree of variability in results that can be obtained under the application of slightly different LCIA methods to calculate biodiversity footprints21; as without safeguards, any net biodiversity strategy developed based on that footprint would be highly sensitive to the specific LCIA route chosen. Such an approach would be entirely consistent with current thinking on biodiversity impact calculation and disclosure, as recommended by, e.g. the Taskforce on Nature-Related Financial Disclosures22 or equally by policies such as the EU Corporate Sustainability Reporting Directive; which themselves are explicitly aligned with guidance on biodiversity target setting as per the Science Based Targets Network. Interestingly, meeting the safeguards alone in this case would potentially take dairy production in the Netherlands a long way towards fully mitigating biodiversity impacts (Fig. It would also be a mechanism for ensuring alignment with other relevant existing policies and standards, at EU level and beyond (see detail in Table 1). Importantly, beyond the likelihood that the use of safeguards would help avoid perverse outcomes for biodiversity, there will be other red lines from the perspective of the Dutch dairy sector associated with ‘mission critical' (and consequently unavoidable) impacts on biodiversity (see ref. In particular, since there is a requirement for milk production to continue, there will remain some inevitable negative impacts on biodiversity, which will require ecological compensation (such as biodiversity restoration offsets). The overall amount of dairy produced by the sector every year is a fundamental consideration, and whether dairy production is to increase, remain stable, or decrease year on year. The types of scenarios displayed in Fig. 4 should be interpreted in that context. For instance, if we apply linear regression to farm-level data, we find that: kg milk produced per hectare trends downwards with increasing % protein produced on farms' own land (coefficient = −0.6), and trends upwards with increasing purchased concentrates per ha (coefficient = 0.9). It is important not to over-interpret these relationships, as they do not imply causation, and a full statistical treatment of the data would require far more nuanced analyses which is beyond scope here. However, such relationships would align with the reasonable contention that reducing biodiversity impacts via limiting feed imported from overseas will either require intensification of feed production within the Netherlands to maintain production or an overall reduction in milk production. So if the goal was to maintain dairy production while reducing the amount of feed purchased from overseas, there would be a requirement to give more land over to protein production within the Netherlands, implying less space to implement restoration of herb-rich grasslands and/or Nature and Landscape components to support domestic biodiversity gain. More extensive farms tend to be slightly larger on average, which may imply (given the positive relationship between purchased feed and production) that farm intensity rather than size plays a more substantial role in maintaining higher levels of production. In the case where the distribution of impacts by farm shifted to substantially lower impacts—such that farms in the current 90th percentile for impacts met current 50th percentile standards on those KPIs—we estimate that the result would be a 48% reduction in net impacts (in PDF.year) relative to 2020 values (Fig. That might be considered more of an ‘even contribution' approach towards net positive biodiversity outcomes, in which lower-performing farms were required to meet current ‘average' performance. However, extrapolating based on correlations between KPI scores and production, this would also result in a ~26% reduction in milk production (assuming the same efficiency in processes and number of farms). Such nuances as those discussed in this paragraph, and the trade-offs associated with them, demonstrate the need to couch discussions around biodiversity metrics and relevant safeguards within a broader framework for sustainable development, which requires more detailed investigation than we have the space to perform here. Considerations, though, would certainly include those around the socio-economic implications of any given strategy for farmers nationally, as well as the implications internationally; particularly, for instance, in those regions that currently produce feed that is then exported to the Netherlands. Comparison between 2020 impacts vs. impacts if farms below current 50th percentile standards on the index improved performance to current 50th percentile levels, based on the integrated index scores. Considering all components of the analyses reported here, what possible pathways exist for the Dutch dairy sector to contribute towards nature positive i.e. the Global Biodiversity Framework post-2020 mission? We lay out three broad hypothetical pathways, with different implications for the sector, and therefore varying feasibility (although, arguably, with each option sequentially showing increasing alignment with global targets under the Global Biodiversity Framework). ‘Adaptive compensation' approach: In the case in which the mid-point KPI safeguards are seen as aspirational rather than strict ‘red-line' limits, a scenario could involve making incremental progress year on year towards meeting all safeguards at the sector level—with all remaining annual impacts (initially close to baseline 2020 levels; Fig. 2) offset through restoration measures in the Netherlands and overseas. The offset activity required would be large but could be determined year on year via the methods here. Offsets would ideally commence in the same year as the impacts for which they compensate, with any delays leading to the use of multipliers (see ref. Under this category of approach, it would be recommended to evaluate progress at milestones towards 2030 and on to 2050, to determine (a) the degree of feasibility of meeting all KPI safeguards, and (b) the practicality and cost of eventually implementing sufficient biodiversity offsets to compensate residual impacts. 9), with small initial efforts at impact prevention that increase steadily over time. But this option could be costly in terms of financing biodiversity offsets requirements, where in any case offsetting should be a less preferred option on the spectrum of the mitigation hierarchy. ‘Even contribution' approach: In this case, the initial focus would be on working with all farms in the lower-ranked 50% (index performance), seeking improvements that bring them up to the equivalent index score as the current 50th percentile (Fig. Since these calculations are based on the integrated index score, and not on separate KPIs, they inherently incorporate some of the key interactions between KPIs. The result would be extensive prevention of biodiversity impacts for those farms, whilst those in the upper ranked 50% (again, by index score) would either continue business as usual or voluntarily seek improvements. The residual impacts would be compensated through biodiversity offsets, which would likely be far less extensive than in pathway 1 above. However, this is predicated on maintaining the number and area of existing farms; and the result of that (alongside achieving a drop in biodiversity impacts) would be a commensurate drop in dairy production. Our analyses—in the absence of any improved efficiencies in dairy production (which might allow greater production levels)—indicate a possible drop in dairy production of ~26%. The approach combines biodiversity impact prevention and compensation. However, it makes abundantly clear that there are inevitable trade-offs between biodiversity restoration, land availability for the dairy sector, and dairy production. In one version of this case, all safeguards would be met at the sector-level (broadly assuming trade-offs at farm-level can be at least partly avoided), reducing impacts substantially (by ~94% according to our calculations; Fig. In itself, this latter finding—that meeting basic biophysical and social safeguards (as outlined in Table 1) could proportionally deliver much of the progress needed towards neutralising biodiversity impacts—is worth underlining, and again emphasizes the usefulness of designing strategies based on multiple midpoint pressures and incorporating safeguards. Aside from the necessary changes in dairy production volumes, the deep net positive approach is appealing as it is likely not the case that a ‘one size fits all' approach can or should be taken to applying the KPIs safeguards to farms. What this type of scenario does usefully do is illustrate the extent of the change needed if the sector were to take a primarily ‘avoidance focused' approach towards meeting the net positive biodiversity target. While considering pathways towards net positive outcomes, it is important to consider the issue of uncertainty in calculating impacts in the first place. The uncertainties involved in estimating biodiversity impacts using LCIA approaches—though considered the best available current practice—are numerous, and well documented24. Analyses specifically pertaining to the biodiversity impacts of dairy organisations based in the Netherlands have shown that even the relative impacts of different organisational activities can vary depending on the precise methodology employed21. This means in turn that nature strategies building on biodiversity footprints must be designed to account for uncertainty in the underlying approaches. It is not possible to make quantitative estimates of the uncertainty bounds involved, which remains a key research question for nature-positive efforts more generally7,24; rather, net positive strategies will likely have to be designed flexibly and with adaptive management practices built in. Further, the safeguards outlined here were constructed around a combination of policy and literature review alongside non-systematic regional expert stakeholder consultation (see the “Methods” section). However, we note that the full application of a safeguards approach in practice would require a more detailed and nuanced stakeholder analysis, that is more widely built upon local knowledge, and systematically includes representatives of key stakeholder groups (see e.g. ref. The implementation of the Biodiversity Monitor has enabled us to perform a quantitative estimate of the Dutch dairy sector's biodiversity impacts; here, we have done so based on established methodologies for Life Cycle Impact Assessment. This study shows one path for developing an integrated biodiversity index for a sector, and strategically developing key areas for targeting biodiversity impact reduction (e.g. land use change linked to purchased feed). However, our key conclusion here is that the strategy must also be guided by efforts to meet a suite of safeguards—both biophysical and social—which we identified in this case through literature review and stakeholder consultation. Otherwise, the use of the single index to guide strategy might result in certain important impacts sources being overlooked (for example, if certain impacts were estimated to be less substantial using the index methodology in question than they were under another methodology21), or some other perverse outcome (e.g. there is considerable scope for perverse outcomes when implementing compensatory measures like biodiversity offsets19,23,25). Furthermore, given the limitations inherent in any biodiversity metrics, we suggest that actual nature strategies tracked using LCIA-type metrics might be usefully structured around the midpoint environmental pressures exerted on nature. This is even more important when considering that biodiversity is only one aspect of nature recovery, and of sustainable development more broadly. Consequently, any biodiversity metric used to track progress will necessarily sit within a much wider monitoring framework featuring multiple dimensions of sustainability; GHG emissions, water consumption, and beyond. The use of strategic safeguards linked to midpoint pressures, alongside an overarching index for tracking biodiversity outcomes, ends up providing a clear path forward for any biodiversity strategy. Meeting the safeguards alone would clearly take the sector a long way toward neutral net biodiversity outcomes. These conclusions are derived in relation to our case study but are likely generalisable to many other systems, in agricultural sectors and beyond: most sectors will have to construct indices for monitoring trends in net biodiversity outcomes if they are to track progress towards the Global Biodiversity Framework. Indeed, there are many regions with higher intact biodiversity than the Netherlands, for which balancing conservation and agricultural production is perhaps even more urgent. All countries have targets for biodiversity conservation alongside agricultural production under the Framework (e.g. Target 10 on enhancing sustainability in agriculture, Target 16 on enabling sustainable consumption), which approaches such as that taken here could support. For the reasons given here, organisations, sectors, and even nation-states should consider developing biodiversity strategies through mid-point pressures, and systematically develop safeguards alongside the key biodiversity metrics employed. Our focal biodiversity metric is based on LCIA approaches, as is current good practice for comprehensive biodiversity footprinting (see e.g. ref. Not only do these KPIs represent key pathways by which environmental pressures could impact biodiversity (e.g. GHG emissions, eutrophication); but they have also been agreed as relevant for DZK through a lengthy process of stakeholder engagement, and crucially, reflect data that are collected and therefore available to track long-term progress towards net biodiversity outcomes. To create a biodiversity index that integrated the KPI data (steps visualised in Fig. Gathered available KPI data (measures of environmental pressure) for the seven KPIs described in the Biodiversity Monitor, for all dairy farms in the Netherlands. Note that there are at least three additional possible KPIs not currently included in the Biodiversity Monitor which the authorship team felt could be relevant for incorporation into future iterations of the Biodiversity Monitor, but for which data are not currently available. These are pesticide use, water consumption, and phosphorus soil surplus; Treating each KPI as a ‘mid-point' environmental pressure10, converted each KPI value into an estimate of biodiversity impacts using characterisation factors from the LC-Impact methodology. This was done at the level of KPIs for each individual farm—fully anonymised—to then be aggregated into a biodiversity impact per farm. Again, this is measured using the unit ‘Potentially Disappeared Fraction' of species over time (PDF.year), which should be interpreted as an indicator for proportional contribution to global species extinction risk25. Here, PDF.year is used as a proxy indicator for combining and comparing biodiversity impacts across a range of mid-point pressures/KPIs; and, Summed PDF.year values across all farms to provide an estimate of biodiversity impacts over a given year for dairy production in the Netherlands. We include the seven KPIs described in the Biodiversity Monitor: GHG emissions, nitrogen soil surplus, ammonia emissions, area of permanent pasture, area of herb-rich grassland, area of managed land (labelled ‘Nature & Landscape'), and ‘protein produced on own land/in farmer's own region' (Table 2). ‘Protein produced on own land/in farmer's own region' is an indicator of the level of self-sufficiency (e.g., feed produced on own land), but is consequently indicative of the environmental footprint in other parts of the world (e.g., to grow ingredients like soy, used in concentrate feeds). To calculate the environmental impacts associated with purchased feed, we make use of the FeedPrint NL database developed by Wageningen University & Research, Blonk Consultants, and GFLI. the biodiversity index is calculated by farm and then aggregated. We assume that biodiversity gains across all farms have equal weight, and do not, for example, consider any strategic spatial placement of biodiversity gains as part of core areas or corridors; characterisation factors in LC-Impact24 are based on models that predict biodiversity losses per functional unit in terms of global species extinctions. To estimate biodiversity gains associated with certain activities e.g., Nature & Landscape management, we assume it is meaningful to apply these factors in reverse. This is a necessary assumption since, as far as we are aware, there are no comparable methods that model biodiversity gains from such a broad range of activities. Not only does this introduce some further uncertainty, but also we acknowledge that there is a temporal aspect here (in that reversing losses of biodiversity would likely take longer than causing it). Data were available at farm-level for 14,686 farms in total, with values spanning the time period 2018–2020. Data on the herb-rich grassland and Nature & Landscape KPIs were provided by Royal FrieslandCampina (RFC), with data on the remaining KPIs and contextual data provided from the KringloopWijzer (KLW) database. To convert KPI values into a biodiversity index value for the sector as a whole, we converted KPIs provided as relative values (e.g., kg CO2e per kg milk, or kg NH3 per ha) into total/absolute values per farm, by multiplying by either the total farm area or total milk production per farm. Farms were excluded from the analysis if they did not report comprehensive data under the Biodiversity Monitor across KPIs, for land area, or on production. We note that there is a possibility that exclusion could lead to analytical biases in terms of the types of farms that had data gaps/mistakes, though this is not something we can assume. Since the relevant farms were those that by definition we could not calculate biodiversity impacts for, due to data gaps/mistakes, this is not something we analysed. Instead, we note this as a potential issue and an avenue for further research. For each farm, the total area of combined temporary grassland and cropland combined was estimated by subtracting the area of permanent grassland from the total area of the farm. Yard area was assumed to be minor, and so not considered in this analysis. These Nature & Landscape management areas provided in the RFC dataset were separated into four categories, based on the type of land the measure would be applied to (e.g., grassland, cropland, or landscape element—as per the Cumulatie and Grondgebruik table provided by Boerennatuur) and also based on the biodiversity weighting per management package, (provided in the Beheerpakketten Biodiversiteitsmonitor (BBM) documentation and associated appendices26,27). These categories are shown in Table 3. The distinction between these four categories was necessary for biodiversity index calculations, separating areas of low-intensity farming (which it is assumed would have a relatively low biodiversity impact compared to more intensive areas) from farmland habitats capable of generating absolute biodiversity gains. It was necessary to exclude some management packages from calculations in order to avoid double counting by area: specifically, BBM107 and BB107 (‘Bodemverbetering grasland met ruige mest', ‘Bodemverbetering bouwland met ruige mest', ‘Bodemverbetering met ruige mest', ‘Chemie en kunstmestvrij land') as they are applied in combination with other management packages. Since it was not always possible to determine from the dataset which land use types were overlapping (e.g., whether HRG was included under temporary or permanent grassland). The following assumptions were therefore made: Areas of low-impact grassland/arable land and Nature & Landscape areas were subtracted from the assumed arable/temporary grassland area. If these values exceeded the arable/temporary grassland area, the remainder was subtracted from permanent grassland areas. While this involves making several assumptions, it was considered necessary and reasonable as a means to account for potential double counting of areas. For the 2020 analysis, areas of land transformation/restoration were calculated using the change in area of herb-rich grassland or Nature & Landscape management per farm relative to the previous year (2019). This calculation was only made when data were available for both years, which was the case for 1237 farms for herb-rich grassland (14% of the dataset, 9% of all farms in the Netherlands), and 3753 farms for Nature & Landscape areas (42% of the dataset, 26% of all farms in the Netherlands). To estimate biodiversity impacts associated with purchased feed, we converted data provided on quantities of feeds into an estimated level of environmental pressure using the life cycle analysis (LCA) database FeedPrint (written consent was provided by the owners for this project). FeedPrint is also referred to in the calculation guidance for the KLW, particularly when calculating GHG emissions associated with feed, so we use FeedPrint as a consistent source for estimating all environmental pressures. FeedPrint provides ingredient- and country-level breakdown for a broad range of dairy feeds, and calculates the associated environmental pressures using the PEFCR-Feed methodology. We exported environmental data for feed components per country, with default parameters selected. Data were provided for feed produced on the farms, and feed purchased by the farms. Feeds included concentrates, other roughage & by-products, maize, grass silage, and milk powder. It was assumed that the impacts of all feeds ‘produced' by the farm were already accounted for by the other environmental KPIs (areas of grassland, total area of farm, N/NH3/CO2e emissions associated with growing fodder, etc. Next, we assumed that ‘purchased' grass and maize silage were produced on other farms within the Dutch dairy sector, and so were also excluded to avoid double counting. Purchased concentrates and other roughage and by-products, however, were assumed to require additional analysis (i.e., produced by other sectors within the Netherlands, and/or by other countries). The analysis of purchased feeds therefore focuses on these two types of composite feeds. To estimate the component ingredients in concentrates we used ‘concentrate dairy standard' as a reference product, taking the ingredient and country-level breakdown directly from the FeedPrint database. ‘Other roughage & by-products' is however a much broader category of feed ingredients; to determine constituent ingredients in this category, we referred to data published by Blonk Consultants28, which provided an in-depth life cycle inventory for Dutch semi-skimmed milk and semi-mature cheese. We referred specifically to Tables 3–10 of this report (using the most recent dry matter estimates provided within FeedPrint). The country-level breakdown for each of the composite ingredients was then sourced from FeedPrint, and can be viewed directly within the FeedPrint database software. Environmental values per kilogram of ingredient per country were exported for land occupation, freshwater eutrophication, marine eutrophication, and water consumption. Carbon values from FeedPrint were not used in order to avoid double counting, as these were assumed to be accounted for under the GHG emissions KPI (an assumption that follows KLW guidance). While the area of land occupation is provided directly in FeedPrint, the area of land transformation (also known as land use change; LUC) is not. However, FeedPrint does provide estimates of carbon emissions associated with LUC, calculated using the PAS2050 methodology. These factors are in the form of tonnes CO2e per ha per year, and are provided at local level for a set of countries. For countries not included in this list, we used the continental average. The area of LUC was calculated for all feed ingredients except for those derived from soybean products. Soy was assumed to have no impacts associated with LUC due to the dairy sector's requirement for sourcing 100% certified sustainable soy (RTRS or equivalent; see Table 1). There are several biodiversity metrics currently available for use in impact analysis, though none are widely accepted as standard. LCIA is an approach supported by leading frameworks (e.g. ref. 22), has precedent in the literature (e.g. ref. 29), and allows us to incorporate the full scope of environmental pressures (KPIs), converting these KPI ‘mid-points' (pressures) into an estimated aggregated end-point impact on biodiversity. LC-Impact software was chosen here because it is one of the most recently developed LCIA methodologies (developed as part of an EU FP7 project, via a collaboration between 14 partners). It incorporates spatial differentiation for environmental impacts where relevant, as well as levels of species vulnerability and endemism—both of which are lacking to some degree in other LCIA methodologies. LC-Impact provides a set of characterisation factors (CFs), which can be used to calculate an estimate of biodiversity impact (in PDF.year) per unit of environmental pressure – for example, PDF.year per kg CO2e emitted, or PDF.year per m2 of land occupation. These CFs are based on a set of models from the scientific corpus that link the KPI to biodiversity via a particular ‘impact pathway' (e.g., climate change, eutrophication, acidification, habitat conversion, etc.). Here, we mainly used the core set of CFs, using the marginal CF in the case of terrestrial acidification impacts linked to ammonia emissions; marginal CFs calculate the biodiversity impact of an additional kilogram of ammonia (as opposed to the average effect of a kilogram of ammonia). For other KPIs, further adjustments were made as follows: Nitrogen soil surplus describes the balance between supply (e.g., from fertiliser, manure, fixation, etc.) and removal (e.g., via crops or emissions to air) of nitrogen compounds on farms. We ‘capped' negative nitrogen soil surplus values at zero—assuming that net removals of nitrogen would be highly localised, and should not be accounted for when aggregating across farms—before combining values with the biodiversity CFs. We also note for completeness that the CFs for nitrogen model eutrophication are based on a commonly applied (e.g. ref. 30, assumption of nutrient limitation—that is: nitrogen is assumed to be the limiting nutrient in marine ecosystems and phosphorus is assumed to be the limiting nutrient in freshwater ecosystems. As such, the impact of nitrogen soil surplus is modelled in terms of eutrophication of marine systems (this takes into account a generic soil leaching fraction for the Netherlands and nutrient transport via river systems—see Verones et al.31 for more information) rather than of freshwater systems. This could lead to impacts from nitrogen soil surplus being underestimated in our metric—although there is evidence that phosphorus is a primary driver of freshwater eutrophication, including within the Netherlands (e.g. refs. 32,33,34), and that nitrogen limitation tends to be stronger in marine systems35. Furthermore, relatively low levels of phosphorus soil surplus are reported for Dutch dairy farms—which would translate into a biodiversity impact from freshwater eutrophication several orders of magnitude lower than other KPIs ( ~ 1.4 × 10−08 PDF.year, based on an average of 7 kg P2O5 recorded on the WUR Agro & Food portal). So, though we have used the best method available, we acknowledge that this is a simplified approach. There are high percentages of eutrophic freshwater bodies in the Netherlands36, which would ideally be improved by CFs that take into account site-specific nutrient limitation and synergistic effects of nitrogen and phosphorus37,38. For land aspects, instead of applying the relevant factors provided via LC-Impact (based on a 2015 analysis), we substitute these for CFs calculated by Chaudhary and Brooks39. According to the authors of these resources, the 2018 factors are more up-to-date and reliable and can be directly substituted into LC-Impact. This also allows differences between three levels of land use intensity (minimal, light, intense) to be accounted for; definitions for each of the land use categories can be found in the supplementary material of Chaudhary and Brooks (2018). CFs for ‘light use' pasture were applied to areas of permanent grassland, whereas CFs for ‘minimal use' cropland and pasture were applied to areas of low-impact grassland/arable land. However, since ‘low-impact arable/grassland' encompasses a broad range of management approaches with varying benefits for biodiversity, we also apply BBM weightings (see https://biodiversiteitsmonitor.nl/certificatie.html). For herb-rich grassland or areas under Nature & Landscape measures: Any decrease in area was assumed to represent a loss of habitat, so we applied the CF for the Netherlands; conversely, the increase in area represented a gain, and we applied the CF in reverse. We acknowledge that using reversed CFs to estimate ‘biodiversity gains' is not standard; however, it was necessary since, as far as we are aware, there are no comparable methods that model biodiversity gains from such a broad range of activities. As for low-impact arable/grassland (above), herb-rich grassland and different types of Nature & Landscape management have variable effects on biodiversity, which are reflected in the BBM weightings. To avoid these weightings overestimating biodiversity gains (the maximum possible proportional change in habitat restoration is 1 i.e., 100% habitat recovery), the weightings were normalised such that the maximum possible weighting was 1. Though this means that we effectively assume complete restoration for these areas, which would not happen in reality on short timescales, this is likely within the large uncertainty bounds for applying LCIA to biodiversity footprinting in any case24. Areas undergoing LUC would also be associated with a change in GHG emissions; emissions associated with habitat loss, or sequestration through habitat restoration. We excluded GHG emissions from LUC linked to feed production, as those were already included within the values provided as part of the KLW dataset. Then, losses of habitat area were assumed to lead to an increase in GHG emissions, calculated using the values provided in Annex C of the PAS2050:2011 guidance. For any gains in habitat area (e.g., increases in herb-rich grassland or Nature and Landscape areas), GHG sequestration was estimated using the values provided by Table 3 of Schmidinger and Stehfest40, who calculate the average potential carbon sink per continental region and food product and provide factors in kg CO2/m2/year. GHG emissions and sequestration were both combined with the relevant CF in LC-Impact. Agricultural land released from productive use could, in the absence of action by other sectors, passively restore to natural habitat over time (e.g. ref. In general, however, land released from dairy production was not assumed to deliver a biodiversity gain via passive restoration as it would likely be used for other purposes (i.e. outside of the dairy sector). This is consistent with the literature on the treatment of counterfactual scenarios in net outcomes policies for biodiversity (e.g. ref. However, there could be cases in which land was set aside but still retained by dairy sector actors, and in that case, any passive restoration should be included in the overall calculations. Therefore, passive restoration of released land is included in our model in the specific scenario where total milk production remains the same or increases, alongside a reduction in land use. In such cases, we apply a high temporal risk multiplier to reflect the uncertainty in restoration timescales (see below). Biodiversity offsets are measurable conservation outcomes (e.g., restoring species and habitats) that are widely used to compensate for residual negative impacts on biodiversity. We did not include offsets in our models here, as these are not currently a component of DZK strategy. However, offsetting could be included in future versions of our models—and we note that biodiversity gains linked to biodiversity restoration offsets would likely be needed as part of achieving net positive impact overall. Impacts such as land use change may result in immediate biodiversity losses, but ecological gains from compensatory restoration activities may take time to accrue. Time lags are undesirable—particularly if species/habitats are threatened, or when the existence of biodiversity provides some ongoing ecosystem service that is diminished during the time lag. A common approach is to apply a multiplier to areas being restored, in order to account for uncertainties around immediate/certain losses being compensated by delayed/uncertain future gains. The multiplier can be considered a ratio between damaged and necessary compensated amounts of biodiversity. It would be applied here as a factor to biodiversity gains, to calculate gains that account for these temporal uncertainties. Laitila et al.23 propose a method to calculate minimum temporal multipliers associated with biodiversity restoration for offsets, which we apply to the biodiversity gains shown in Fig. Their method is based on the following parameters: Time taken to restore different habitats (in number of years); The change in habitat condition (e.g., the proportional increase in biodiversity that is achieved by the restoration activity); A discount rate, which mathematically determines the currently perceived value of biodiversity gains that are not achieved until future years (‘net present value'); and, Permanence of the positive and negative impacts. The values we applied for each parameter, for different restoration activities, are described in Table 4. The ‘permanence of impacts' parameter was assumed to be 30 years for all habitats, being approximately equivalent to one generation (and therefore a feasible period of time for maintaining activities). Time lag values for habitat restoration are as advised by Meli et al.41 and via habitat management guidance provided by BoerenNatuur. Discount rates are as suggested by Overton et al.43. To calculate the change/improvement in habitat conditions associated with Nature & Landscape measures, we used the existing biodiversity weightings developed for the BBM and ANLb packages, which again were normalised as previously discussed. After characterising all KPIs in terms of endpoint biodiversity impacts using the approaches described above, biodiversity impact values (in PDF.year) were summed across all farms (cf. The total impact estimated for the filtered dataset was factored up to an estimate for the production sector as a whole, based on the total number of farms (14,54219), and total area under dairy production in 2020 (870,880 ha19). Setting any ‘net outcomes' objective requires in part that any unavoidable biodiversity impacts are adequately compensated for (e.g., through biodiversity offsets). But to specify to what extent different biodiversity losses are permittable as part of net outcomes policies—and also in the context of uncertainties in calculating overall biodiversity impacts, as discussed throughout—we propose accompanying metrics with safeguards. These can be based both on empirical biophysical limits (at the level of the KPIs), as well as socio-political values. For example, a safeguard might specify that achieving net positive impacts on biodiversity should not be based on a strategy that includes deforestation of pristine rainforest through the Dutch dairy sector's feed supply chain, or one that leads to levels of nutrient pollution within the Netherlands that are unhealthy for humans or wildlife. It is unlikely that values can be maximised for all Biodiversity Monitor KPIs. For example, there may be farm-level trade-offs between KPIs (such as between protein produced on own land and nitrogen soil surplus, if production intensity is increased) or local constraints that limit performance. Safeguards help to define the safe operating space for the sector as a whole, allowing farm-level constraints and trade-offs to be navigated while striving for a genuine Net Positive impact on biodiversity at the sector level. Safeguards therefore need to be comprehensive enough to prevent perverse outcomes for nature, but practical enough to ensure feasibility when designing strategies towards a net positive target. In developing safeguards here, we drew upon existing good practice guidance around achieving ‘net outcomes' goals (such as net positive impacts on biodiversity), and associated topics (e.g. biodiversity offsetting). 6), business and financial institution standards and guidance (e.g., IFC Performance Standard 644, UK BNG Good Practice Principles), policy guidance (e.g., EU biodiversity strategy, Dutch Natuurpunten45), and stakeholder consultation sessions. These resources exhibit some common themes, which potentially serve as a good basis for determining categories of safeguards (Table 5). During our research period, two stakeholder consultation meetings were organised. The objective of these consultations was not only to inform relevant stakeholders about the project but also to receive feedback on the proposed methodologies. The stakeholder consultation meetings took place virtually on December 6, 2021, and May 25, 2022 (running for 2 h in both cases). Where identified stakeholders were unable to attend the meeting, they were invited to provide feedback by email. Both consultations were hosted by the research team. Separate documents outlining the status of methodologies were provided in advance of the meeting to attendees, and the research team presented research updates at each. Stakeholders asked questions, provided feedback, and made suggestions on how to improve the approach and methods. Finally, the project team hosted a focussed discussion about critical points where stakeholder input was essential to move forward with the project. KLW data from individual farms were linked to Nature and Landscape data from the same farms, and anonymized before being sent to the researchers i.e. completely anonymized. The data themselves are owned by ZuivelNL. Further, we use the FeedPrint NL database developed by Wageningen University & Research, Blonk Consultants, and GFLI; and, supporting data sourced from Royal FrieslandCampina and the KringloopWijzer database; again, these datasets can be requested from the corresponding owner organisations. Secretariat of the Convention on Biological Diversity. Bull, J. W. et al. Net positive outcomes for nature. Arlidge, W. N. S. et al. A global mitigation hierarchy for nature conservation. Maron, M. et al. etting robust biodiversity goals. The “nature-positive” journey for business: a conceptual research agenda to guide contributions to societal biodiversity goals. zu Ermgassen, S. O. S. E. et al. Are corporate biodiversity commitments consistent with delivering ‘nature-positive' outcomes? A review of ‘nature-positive' definitions, company progress and challenges. Taylor, I. et al. Meeting nature-positive goals for an organisation's food system. Maxwell, S. L. et al. Biodiversity: the ravages of guns, nets and bulldozers. & Nemecek, T. Reducing food's environmental impacts through producers and consumers. The direct drivers of recent global anthropogenic biodiversity loss. Global food system emissions could preclude achieving the 1.5 and 2 C climate change targets. Transforming Food and Agriculture to Achieve the SDGs: 20 Interconnected Actions to Guide Decision-makers (FAO, Rome, 2019). The social shortfall and ecological overshoot of nations. Don't dilute the term Nature Positive. The many meanings of no net loss in environmental policy. in Global Assessment Report on Biodiversity and Ecosystem Services of the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services (eds Brondizio, E. S., Settele, J., Díaz, S. & Ngo, H. T.) (IPBES secretariat, Bonn, Germany, 2019). Alternative Life Cycle Impact Assessment Methods for biodiversity footprinting could motivate different strategic priorities: a case study for a Dutch dairy multinational. Recommendations of the Taskforce on Nature-related Financial Disclosures. Laitila, J. et al. A method for calculating minimum biodiversity offset multipliers accounting for time discounting, additionality and permanence. Bromwich, T. et al. Navigating uncertainty in LCA-based approaches to biodiversity footprinting. Bull, J. W. et al. Biodiversity offsets in theory and practice. Beheerpakketten Biodiversiteitsmonitor (BBM) Natuur-en landschapspakketten en Kruidenrijk graslandpakketten https://biodiversiteitsmonitor.nl/docs/220131_BBM_pakketten_2022_vastgesteld_stichting_Biodiversiteitsmonitor.pdf (2021). BBM Weighting Table and Management Packages https://biodiversiteitsmonitor.nl/ (2022). LCA of Dutch Semi-skimmed Milk and Semimature Cheese Based on: Comparative LCA of Dutch Dairy Products and Plant-based Alternatives (Blonk Consultants, Gouda, the Netherlands, 2014). Taylor, I. et al. Nature-positive goals for an organization's food consumption. Morelli, B. et al. Critical review of eutrophication models for life cycle assessment. Verones, F. et al. LC-IMPACT: a regionalized life cycle damage assessment method. Lürling, M. & Mucci, M. Mitigating eutrophication nuisance: in-lake measures are becoming inevitable in eutrophic waters in the Netherlands. Schindler, D. W. et al. Reducing phosphorus to curb lake eutrophication is a success. Global analysis of nitrogen and phosphorus limitation of primary producers in freshwater, marine and terrestrial ecosystems. Henryson, K. et al. Spatially differentiated midpoint indicator for marine eutrophication of waterborne emissions in Sweden. & Brooks, T. M. Land use intensity-specific global characterization factors to assess product biodiversity footprints. Schmidinger, K. & Stehfest, E. Including CO2 implications of land occupation in LCAs—method and example for livestock products. Meli, P. et al. A global review of past land use, climate, and active vs. passive restoration effects on forest recovery. Restoration realities in the context of biodiversity offset policies. Performance Standard 6: Biodiversity Conservation and Sustainable Management of Living Natural Resources (IFC, World Bank Group, 2012). van Gaalen, F. et al. Natuurpunten: kwantificering van effecten op natuurlijke ecosystemen en biodiversiteit in het Deltaprogramma (‘Quantification of effects on natural ecosystems and biodiversity in the Delta Programme') (PBL, Den Haag, 2014). Grondgebondenheid als basis voor een toekomstbestendige melkveehouderij (‘Land-relatedness as a basis for future-proof dairy farming') (Commissie Grondgebondenheid, 2018). Werkgroep KPI natuur- en landschapsbeheer voor de biodiversiteitsmonitor melkveehouderij). Advies voor de uitwerking van de indicator natuur- en landschapsbeheer voor de biodiversiteitsmonitor melkveehouderij (‘Advice for the elaboration of the nature and landscape management indicator for the dairy farming biodiversity monitor') (Duurzame Zuivelketen (Werkgroep KPI natuur- en landschapsbeheer voor de biodiversiteitsmonitor melkveehouderij), Wageningen, 2019). Forest, Land, and Agriculture Science based Target Setting Guidance Draft for Public Consultation https://sciencebasedtargets.org/sectors/forest-land-and-agriculture (2022). (‘Threshold and Target Values for the KPIs of the Biodiversity Monitor Dairy Farming; Standardize from Ecology'), Report WENR 2968 (Wageningen Environmental Research, Wageningen, 2019). Beldman, A. et al. De Nederlandse melkveehouderij in 2030: verkenning van mogelijke ontwikkelingen op basis van economische modellering (‘Dutch Dairy Farming in 2030: Exploring Possible Developments Based on Economic Modelling'). Cormont, A. et al. Landscape complexity and farmland biodiversity: evaluating the CAP target on natural elements. Implementation gap between the theory and practice of biodiversity offset multipliers. Definitie, voorwarden en weging Kruidenrijk Grasland (‘Herb-rich grassland KPI. Definition, conditions and weighting for Herb-rich Grassland') (Duurzame Zuivelketen, WWF, Rabobank, 2020). The authors would like to acknowledge Dr. Talitha Bromwich for useful technical conversations on this topic while preparing the manuscript. Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar led the stakeholder workshops to develop safeguards, which all other authors participated in; J.W.B. have since moved on from DZK to new roles. The authors declare no competing interests. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Bull, J.W., Taylor, I., de Valença, A. et al. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01730-w'>A multichaperone condensate enhances protein folding in the endoplasmic reticulum</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 09:28:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Protein folding in the endoplasmic reticulum (ER) relies on a network of molecular chaperones that facilitates the folding and maturation of client proteins. How the ER chaperones organize in a supramolecular manner to exert their cooperativity has, however, remained unclear. Here we report the discovery of a multichaperone condensate in the ER lumen, which is formed around the chaperone PDIA6 during protein folding homeostasis. The PDIA6 condensates recruit further chaperones—Hsp70 BiP, J-domain protein ERdj3, disulfide isomerase PDIA1 and Hsp90 Grp94—which constitute some of the essential components of the early folding machinery. The chaperone condensates enhance folding of proteins, such as proinsulin, and prevent protein misfolding in the ER lumen. One-third of eukaryotic proteins are processed within the endoplasmic reticulum (ER) to obtain their correct structure1,2,3. This function is ensured by a network of molecular chaperones that recognizes client proteins and assists their folding4,5. A rate-limiting step in ER protein folding is the formation of disulfide bonds, and the protein disulfide isomerase (PDI) chaperone family is therefore indispensable for folding homeostasis6,7. PDIs contain one or more catalytically active thioredoxin-like domains, which can reduce, oxidize or isomerize disulfide bonds in client proteins via a characteristic CXXC motif. They typically combine catalytically active and inactive domains towards multidomain architectures. Although all PDIs share common structural features, they differ in terms of client-protein specificity and affinity, chaperone activity and redox properties, resulting in specialized roles for individual family members8. One particular family member, PDIA6, caught our attention because its malfunction is connected to several human diseases including cancer and diabetes, yet its exact function remained elusive9,10,11,12. We observed that endogenous PDIA6 localized as expected in the ER of human HeLa cells. Remarkably, however, the endogenous protein was not homogeneously dispersed but concentrated in well-defined clusters, both in wild-type (WT) cells as well as in PDIA6–Halo knock-in cells (Fig. Overexpressed PDIA6 formed clusters of similar shape and localization in living cells, albeit with substantially increased size (Fig. PDIA6 formed similar clusters in HEK and U2OS cells (Extended Data Fig. The PDIA6 clusters were mobile within the ER and underwent fission and fusion events, and their fluorescence recovery after photobleaching (FRAP) occurred within minutes (half life (t1/2) = 78 ± 14 s, diffusion coefficient (D) = 4 ± 1 × 10−3 μm2 s−1), a time scale that is common for biological phase-separated condensates13,14 (Fig. The PDIA6 condensates thus fulfil all hallmarks of biological phase separation15. a, Fixed HeLa cells expressing endogenous PDIA6 (left) and live-cell images of HeLa cells transfected with PDIA6–GFP (right). b, Time series of a HeLa cell transiently transfected with PDIA6–GFP. A fusion event is indicated by arrowheads. c, PDIA6–GFP FRAP experiment in HeLa cells. d, Fluorescence recovery analysis of the PDIA6–GFP FRAP experiment presented in c. In vivo analysis of three cells from three independent experiments each coloured in a different shade of grey. The black trace corresponds to Supplementary Video 3. e, Number of condensates in unstressed cells as well as cells subjected to tunicamycin or thapsigargin treatment and subsequent washouts. A total of 622, 0, 449, 0 and 455 condensates were observed in n = 25, 24, 21, 25 and 20 cells, respectively, from left to right, pooled from three independent biological replicates. f, Condensate size in unstressed and recovered cells after treatment with tunicamycin or thapsigargin; n = 25, 21 and 20 cells, respectively, pooled from three independent biological replicates. g,h, Time series images of individual HeLa cells transiently transfected with PDIA6–GFP during tunicamycin (g) and thapsigargin (h) treatment. i, In vitro phase separation of DyLight 488–PDIA6 in the presence of crowding agent (left), crowding agent plus 10 mM Ca2+ (middle) and crowding agent plus 10 mM Ca2+ and 20 mM EDTA (right). k, Number of in vitro PDIA6 droplets per field of view in the presence of crowding agent at different Ca2+ concentrations and fitted Hill equation binding model (green slope). ; n = 9 images for [Ca2+] ≥ 500 μM and n = 6 images for [Ca2+] < 500 μM per condition, pooled from three independent experiments. All images are representative of at least three independent experiments. Attempts to delete the PDIA6 gene in human cells were unsuccessful, indicating that PDIA6 is an essential protein, consistent with earlier findings in Caenorhabditis elegans16. Intriguingly, PDIA6 malfunction has been reported to lead to an imbalance in ER folding homeostasis16,17. We thus wondered whether a functional coupling exists between ER stress and condensate formation. To create a physiological ER-stress situation, we overexpressed Akita proinsulin, a misfolded mutant of proinsulin that accumulates in the ER and thereby causes ER stress18. This led to homogenous dispersion of PDIA6 in the entire ER lumen (Extended Data Fig. Notably, the stress-dependent condensate dissolution was reversible, as PDIA6 condensates reorganized after washout of the stressor (Fig. Two other stress inducers, thapsigargin and cyclopiazonic acid, induced the same effect (Fig. 1e,f and Extended Data Fig. PDIA6 condensates dissolved completely within 1 h following thapsigargin treatment, whereas dissolution was observed only after 6 h of tunicamycin treatment (Fig. 1g,h and Extended Data Fig. This kinetic difference correlates with the different modes of action of the two drugs. Thapsigargin blocks the sarco-/endoplasmic reticulum Ca2+ ATPase pumps19, leading to an almost immediate Ca2+ depletion in the ER lumen, whereas tunicamycin inhibits N-glycosylation of newly synthesized proteins20 and causes a delayed drop in Ca2+ levels as a secondary effect of the accumulation of unfolded proteins21. Overall, the presence of PDIA6 condensates is thus coupled to protein homeostatic conditions and their dissolution in response to ER stress seems to be regulated by luminal Ca2+ levels. To test this mechanistic hypothesis, we reconstituted droplets of purified recombinant PDIA6 protein in vitro. Homeostatic ER was mimicked by a high Ca2+ concentration, reducing conditions, physiological pH and a crowding agent. Under these conditions, recombinant PDIA6 reproducibly formed droplets that underwent fusion events and time-dependent growth in a time scale of seconds. The protein dynamically exchanged between the condensed and dispersed phase with similar kinetics as in vivo, as determined by FRAP experiments (t1/2 = 47 ± 18 s, D = 6 ± 2 × 10−3 μm2 s−1; Fig. Droplets did not form when Ca2+ was replaced by another divalent ion, Mg2+, that is, their formation was Ca2+-specific. Importantly, the droplets formed only at Ca2+ concentrations above about 500 μM and the saturation concentration was independent of the protein concentration (Fig. We determined the Ca2+ affinity of PDIA6 to be 420 ± 20 μM in vitro by quantifying the number of PDIA6 condensates as a function of Ca2+ concentration (Fig. Remarkably, this value corresponds to the physiological transition between protein folding homeostasis ([Ca2+]homeo ≈ 800 μM) and ER stress ([Ca2+]stress ≈ 100 μM)22, and therefore correlates very well with our observations in living cells. PDIA6 condensates are thus regulated by the Ca2+ concentration. We then employed an integrated structural biology approach to resolve the structure and mechanism of PDIA6 condensate formation. PDIA6 consists of two catalytically active thioredoxin-like domains, domains a0 and a, which are followed by the inactive domain b of unknown function (Fig. To monitor conformational changes and interactions using NMR spectroscopy, we assigned 94% of the Metε, Leuδ2 and Valγ2 methyl groups and established near-complete amide backbone assignments (Extended Data Fig. Through size-exclusion chromatography-multi-angle light scattering (SEC-MALS) and nuclear Overhauser effect (NOE) spectroscopy (NOESY) experiments, we found that PDIA6 dimerizes via helix α4 in domain a0 (Fig. 2b,c and Extended Data Fig. The dimerization interface was resolved by a methyl NOE network between residues L72 and L134 as well as between V75 and L131, L134 and L135, which corresponds to the arrangement observed in crystals of domain a0 (Protein Data Bank (PDB) identifier: 4ef0). The dimerization thus resulted from a carboxy-terminal extension of helix α4 in domain a0 by seven residues and contrasts the PDI family members PDIA1–5, which have shorter helices and are monomeric26 (Extended Data Fig. Dimerization occurred with a dissociation constant (KD) in the low nanomolar range (Extended Data Fig. We determined the crystal structure of domain b (residues 274–427), which adopts a thioredoxin-like fold, to a resolution of 1.8 Å (Extended Data Fig. a, Domain organization of PDIA6 including the cleaved signalling sequence (dashed), delimitations for domains a0, a and b, and the C-terminal tail (t). Intramolecular (grey) and intermolecular (pink) NOEs are indicated. c, Crystal structure of PDIA6 domain a0 (PDB: 4ef0) with subunits coloured in pink and grey. The NOE network detected in b is indicated by dashed blue lines. The interdomain NOE contact between L266 in domain a and M331 in domain b is represented by a blue line. e, Section of the two-dimensional [13C,1H]-heteronuclear multiple quantum correlation (HMQC) spectrum of 100 μM methyl-labelled PDIA6. f, Structural model of dimeric PDIA6 full-length (PDB: a0, 4ef0; a, 4gwr; b, 8cpq). g, Paramagnetic relaxation enhancement effect of 0.5 equivalent (eq.) Gd3+ on methyl groups of 100 μM methyl-labelled PDIA6 WT (top) and outcompetition by Ca2+ (bottom). h, Structural model of PDIA6 domain a (light grey) and b (dark grey) representing the methyl groups identified in g. i, Ca2+ binding sites I (left) and II (right). Ca2+ ions are modelled based on calsequestrin (PDB: 5kn3; Extended Data Fig. Methyl groups with substantial loss in intensity in g are represented as spheres. Side chains of Ca2+ binding residues are shown as sticks. The dimeric domain a0 is connected to domain a by a long positively charged linker, and domains a and b are connected by a short five-residue linker that presumably restricts the relative domain motions as it contains three conserved proline residues (Fig. Domains a and b unfold cooperatively, suggesting that they are stabilized by interdomain interactions (Extended Data Fig. The C-terminal tail of PDIA6 (residues 420–440) is highly flexible and disordered, as evidenced by a narrow chemical shift dispersion and high NMR signal intensities (Fig. Combining the experimental data, we established a full atomic model of PDIA6 (Fig. As PDIA6 condensate formation is Ca2+-dependent, we set out to determine PDIA6 Ca2+ binding sites. Titration of Ca2+ to PDIA6 did not lead to substantial changes in the methyl NMR spectra, demonstrating the absence of large structural rearrangements (Extended Data Fig. Titration of the paramagnetic analogue Gd3+, however, quenched NMR signals specifically in three structural regions, thus revealing three cation binding sites (I, II and III; Fig. 2g,h and Extended Data Fig. Bound Gd3+ was outcompeted by an excess of Ca2+, proving Ca2+ binding specificity. Binding sites I and II are located at the domain a–b interface and in domain b, respectively. Binding site III is located within the disordered C-terminal tail, as previously proposed25. We pinpointed the exact locations of binding sites I and II in the centre of the quenched regions by structural alignment with the Ca2+ binding protein calsequestrin and amide NMR experiments (Fig. We attempted to probe the binding sites experimentally, however, single point mutations in binding sites I or II resulted in drastically decreased protein yields, preventing further experiments (Extended Data Fig. In contrast, mutagenesis of the Ca2+ binding site III was readily possible. Disruption of this site did not affect condensate formation in vitro or in living cells (Extended Data Fig. Taken together, the Ca2+-dependent phase separation of PDIA6 is controlled by a molecular switch in domain b, which is activated following Ca2+ binding. Next, we set out to determine the molecular mechanism underlying condensate formation. Biological phase separation relies on multivalent interactions15 and therefore, we expected at least two intermolecular interactions to be involved. 25) did not form condensates (Extended Data Fig. In search for the second interaction, we first tested droplet formation of the individual domain subconstructs but none of these formed droplets in solution (Extended Data Fig. As mentioned earlier, we also probed a potential role of the disordered C-terminal tail, but neither neutralization of its positive charges (EQDN) nor its entire truncation (ΔC) abrogated the formation of condensates (Extended Data Fig. We then hypothesized that the highly flexible and charged linker connecting domains a0 and a (Fig. 3a) might drive condensate formation by interacting with another PDIA6 dimer. To resolve this interaction, we employed NMR spectroscopy on the disperse protein, that is, in the absence of crowding agent. The addition of the peptide indeed led to drastic spectral changes in domain b. In the absence of the peptide, the domain equally populated two conformations in slow exchange (two peaks per residue for all residues in domain b; 57:43 ± 8%). Following titration of the linker peptide, this equilibrium strongly shifted towards one conformation (93:7 ± 3%), evidencing a transient interaction between the linker and domain b (Fig. 3b,c and Extended Data Fig. High salt concentration as well as charge-neutralizing mutations in the linker peptide impaired the interaction, validating its electrostatic character (Fig. 3c,d and Extended Data Fig. Strikingly, the interaction was Ca2+-dependent, in full agreement with the physiological observation of phase separation (Fig. linker peptide (right) without crowding agent. The two peak sets of conformations P and O are shown on the example of residues V301 and M367. c, Relative populations of PDIA6 conformations P (dark blue) and O (light blue), in the presence of 10 mM Ca2+ and 10 eq. linker peptide (blue spheres) or following the addition of 1 M KCl. d, Relative populations of PDIA6 conformations P (dark blue) and O (light blue), as indicated. In c and d, data are the mean ± s.d. of n = 9 non-overlapping methyl group residues in domain b. NMR spectra are presented in Supplementary Fig. 1. e, In vitro phase separation of DyLight 488–PDIA6 constructs, as indicated, in the presence of crowding agent and 10 mM Ca2+. f, Live-cell images of HeLa cells transfected with PDIA6–GFP WT or linker mutants. In c–f, blue spheres represent positively charged residues in the peptide linker, while white spheres represent neutralizing mutations. g,h, Model of the domain a0a0 dimerization interface (g) as well as the intermolecular interaction between the linker and domain b of two PDIA6 dimers (h). One dimer of PDIA6 is coloured in dark pink, the other in light pink. i, Model of PDIA6 droplet formation (details in text). The data thus suggest a functional model for PDIA6 condensate formation, where the positively charged a0–a linker dynamically interacts with the negatively charged surface in domain b at homeostatic Ca2+ concentrations. We tested this functional model on the in vitro-reconstituted droplets of full-length PDIA6. In full agreement with the model, droplet formation was prevented by increased salt concentration or neutralization of the positively charged linker residues by mutagenesis (Fig. Importantly, the linker mutations did not affect the PDIA6 dimeric state or its disulfide isomerase and chaperone activities (Extended Data Fig. Encouraged by these findings, we then also probed the molecular model of condensate formation in cells. Cells transfected with PDIA6 linker mutants indeed no longer formed the large condensates observed for PDIA6 WT. Removal of two charges from the linker led to a reduction in condensate size, and complete neutralization of all five charged residues (5× linker mutant) fully impeded condensate formation, acting as dominant negative (Fig. The expression levels of endogenous PDIA6 and transfected PDIA6 5× linker mutant were approximately equal in our in vivo experiment, and at the same stoichiometry PDIA6 droplet formation was also abrogated in vitro (Extended Data Fig. BiP is one of the most abundant ER chaperones involved in protein folding and disaggregation and has been reported to interact with PDIA6 (refs. Immunostaining of HeLa cells showed that endogenous BiP co-localized in the PDIA6 condensates, both at endogenous and overexpressed PDIA6 levels. Overexpression of BiP, in turn, did not affect the PDIA6 condensates (Fig. We also tested whether BiP would form condensates on its own by staining endogenous BiP in cells transfected with PDIA6 5× linker mutant. As shown above, in these cells PDIA6 does not form condensates but is homogenously dispersed throughout the entire ER. Under this condition, BiP did not form condensates, thus establishing PDIA6 as the scaffold of the condensates to which BiP is recruited (Fig. a, Co-localization of endogenous PDIA6 and BiP in fixed HeLa cells. Arrowheads exemplarily indicate the position of condensates. b, Co-localization of overexpressed PDIA6–GFP and endogenous BiP in fixed HeLa cells. c, Co-localization of overexpressed PDIA6–GFP 5× linker mutant and endogenous BiP in fixed HeLa cells. PDIA6 was visualized using the eGFP signal. d, Docking model of PDIA6 and BiP, based on NMR data. Methyl groups with substantial chemical shift perturbation (CSP) upon interaction between PDIA6 and BiP are marked on the structures as coloured spheres. e, DyLight 633–BiP NBD in the presence of crowding agent and 10 mM Ca2+ (left), and co-localization with DyLight 488–PDIA6 (right) in vitro. Arrowheads exemplarily indicate the position of condensates. g, Co-localization of overexpressed PDIA6–GFP and endogenous calreticulin in fixed HeLa cells. In b and g, fluorescence intensities across the condensates were measured along the yellow line (left) and are shown as histograms (right). h, Co-localization analysis of various ER chaperones and PDIA6 (or BiP). PCC for pixel intensity of proteins in HeLa cells as indicated. Transfected PDIA6–GFP WT is indicated with an asterisk; all other proteins are endogenous. Individual data points (PCC per cell; n = 10 for PDIA6*–ERdj3 and PDIA6*–calreticulin, 13 for PDIA6*–BiP and 15 for all other protein co-localizations), pooled from three independent experiments, and the s.d. are shown (details in Supplementary Table 2). Next, we employed methyl and amide NMR experiments to determine the interaction sites between PDIA6 and BiP by chemical shift mapping. PDIA6 interacts via its domain b with the nucleotide binding domain (NBD) of BiP (Fig. The affinity of KD ≈ 50 μM (Extended Data Fig. 8l) is in the same range as known BiP–co-chaperone interactions35,36 and fully in line with a dynamic chaperone network. Importantly, the BiP binding site on domain b does not overlap with the interface required for condensate formation such that condensate formation and BiP recruitment by PDIA6 is simultaneously possible, as observed in cells. In full accordance with this model, the isolated BiP NBD co-localized in PDIA6 droplets in vitro, although it did not undergo phase separation by itself (Fig. Together, these experiments clearly establish that PDIA6 forms the scaffold of the condensates into which it recruits BiP as a secondary protein. Next, we investigated whether further ER chaperones were also concentrated into the PDIA6 condensates. We chose to examine the BiP co-chaperone ERdj3, an ER-resident member of the J-domain protein family, because it plays a central role in mediating client-protein transfer to BiP37. Strikingly, ERdj3 co-localized with the PDIA6–BiP condensates, both at endogenous protein levels and in PDIA6-transfected cells. Next, we probed the localizations of PDIA1 (the major PDI for numerous ER client proteins) and Grp94 (the ER-localized Hsp90), which cooperate with BiP in client-protein processing34,38. Both chaperones enriched in the PDIA6 condensates at endogenous protein levels and in PDIA6-transfected cells (Fig. In contrast to this, the lectin chaperones calreticulin and calnexin (responsible for the glycosylation of client proteins before their export from the ER)39 did not enrich in the PDIA6 condensates (Fig. From these observations, it emerges that the essential components of the early folding machinery—Hsp70, Hsp90, J-domain proteins and PDIs—specifically concentrate in the condensates, whereas the lectin chaperones involved in later-stage client maturation—calnexin and calreticulin—are not enriched (Fig. The co-localization of multiple key chaperones in the condensates strongly suggests a fundamental biological function in client-protein folding. Proinsulin, the precursor of insulin, is known to be folded in the ER through the assistance of various chaperones including BiP, Erdj3, PDIA6 and PDIA1 (refs. 40,41) before being exported to the Golgi for further processing and subsequent secretion in the form of insulin42. This makes it an ideal model client to study the potential role of the condensates as folding compartments. To assess the spatial localization of proinsulin within the ER, cells were transiently transfected with a proinsulin construct including a C-terminal myc tag. This allowed visualization of ER-localized proinsulin through staining with antibody to myc and revealed that proinsulin was highly enriched in endogenous PDIA6 condensates (Pearson's correlation coefficient, PCC = 0.93 ± 0.1), which strongly points towards a role of the condensates in client-protein folding. Similarly, proinsulin co-localized in PDIA6–GFP condensates (PCC = 0.88 ± 0.06; Fig. a, Co-localization of endogenous PDIA6 and overexpressed proinsulin–myc in fixed HeLa cells. Arrowheads exemplarily indicate the position of condensates. b, Co-localization of overexpressed PDIA6–GFP and proinsulin–myc in fixed HeLa cells. c, Secretion levels of insulin (medium), and expression levels of PDIA6 and α-tubulin (lysate) in HeLa cells transfected with proinsulin, proinsulin + PDIA6–GFP WT or proinsulin + PDIA6–GFP 5× linker mutant. Proinsulin was not detectable in the lysate. d, Secretion levels of insulin (medium) as well as expression levels of PDIA6, proinsulin and α-tubulin (lysate) in INS-1 cells transfected with mock, PDIA6–GFP WT or PDIA6–GFP 5× linker mutant. In c and d, representative blots from three independent replicates. e, Fold increase in spliced XBP1 (sXBP1) in HeLa cells transfected with PDIA6 WT or 5× linker mutant relative to untransfected cells, determined using quantitative PCR with reverse transcription (P = 0.00092). f, Flow cytometry-based cell aggresome analysis of HeLa cells transfected with PDIA6–GFP WT or PDIA6–GFP 5× linker mutant, detected by PROTEOSTAT aggresome dye (P = 0.007). Gating strategy summarized in Supplementary Fig. 2. g, Half recovery time of mApple–Sec61 in HeLa cells co-transfected with PDIA6–GFP WT or PDIA6–GFP 5× linker mutant. Data from three independent experiments were fitted individually and averaged (P = 0.003). In e–g, data are the mean ± s.d. ; n = 3 independent biological experiments per condition; two-sided Student's t-test. h, Model of PDIA6 chaperones condensates functioning as folding factories (details in text). To evaluate whether proinsulin is actively processed within the PDIA6 condensates, we assessed their effect on insulin secretion. If the PDIA6 condensates served as functional ER folding compartments, their absence should lead to a decrease in insulin secretion. We co-transfected cells with proinsulin and either PDIA6 WT (condensates) or the PDIA6 5× linker mutant (no condensates), and determined the amount of insulin secretion. Strikingly, cells transfected with PDIA6 5× linker mutant secreted less insulin than PDIA6 WT-transfected cells, supporting our hypothesis of a functional role of the PDIA6 condensates in client maturation (Fig. Notably, intracellular proinsulin was not detectable in these experiments, presumably due to its proteolytic degradation. We thus treated the cells with the proteasome inhibitor MG-132, which led to detectable proinsulin accumulation (Extended Data Fig. Thereby, proinsulin levels were notably higher in cells expressing PDIA6 5× linker mutant (no condensates) compared with PDIA6 WT (condensates), which further demonstrates that the absence of condensates impairs proinsulin folding. To corroborate this, we also used the rat insulinoma cell line INS-1, which exhibits an inherently high glucose-stimulated insulin secretion, overcoming the requirement of proinsulin co-transfection. Comparison of the secreted insulin with the intracellular proinsulin ratio of PDIA6 WT- and PDIA6 5× linker mutant-transfected INS-1 cells led to the same conclusion as before, namely that the absence of the chaperone condensates severely disrupts insulin processing (Fig. Together, these experiments provide clear evidence that the PDIA6 condensates as such play a fundamental role in insulin folding that goes beyond classical chaperone functions. Finally, we wanted to study the effect of the PDIA6–chaperone condensates on general ER client-protein turnover and proteostasis. To this end, we assessed the effect of chaperone condensates on three parameters that report on ER proteostasis: (1) XBP1 splicing, a well-established readout that directly reports on the accumulation of misfolded proteins in the ER lumen43,44, (2) protein aggregation using the PROTEOSTAT aggresome assay and (3) fluidity of the ER membrane, which has been reported to stiffen on accumulation of misfolded proteins, measured via the recovery dynamics of mApple–Sec61 in FRAP experiments. We transfected cells with PDIA6 WT or PDIA6 5× linker mutant and quantified the above parameters. Strikingly, compared with PDIA6 WT-expressing cells, the absence of PDIA6 condensates (5× linker mutant) led to a pronounced increase in XBP1 splicing (Fig. 5e) and greater protein aggregation (Fig. The fluorescence recovery of mApple–Sec61 was significantly slower in cells without PDIA6 condensates (t1/2 = 17 ± 5 s) than cells with PDIA6 condensates (t1/2 = 8 ± 4 s; Fig. Together, these results lend strong support for our model that the PDIA6 condensates play an active role in preventing protein aggregation and enhancing protein folding robustness. This is further reflected in a lower cell viability in the absence of condensates (Extended Data Fig. 10h–l), indicating that condensates avert cell death by preventing the accumulation of misfolded proteins. PDIA6 condensates thus serve to subcompartmentalize the ER into functional ‘folding factories'. In summary, our experiments reveal along multiple independent lines that crucial ER chaperones do not only cooperate in a functional network but form a self-organized supramolecular structure within the ER that guides client folding (Fig. The chaperone condensates seem to act in the early stages of client-protein folding and it is thus imaginable that client proteins are imported directly into the condensates following their translocation into the ER. The physiological benefits of subcompartmentalizing the ER by arranging chaperones into folding condensates indeed seem to be numerous: the elevated local concentrations of chaperones and clients increases both specificity and folding efficiency, thus reducing protein misfolding and aggregation. The simultaneous decrease in chaperone concentration outside the condensates may prevent non-specific interactions with folded proteins in the ER lumen, thus enabling a multistep processing pipeline for clients. Furthermore, the dynamic regulation of the chaperone condensates via the Ca2+ concentration might help to overcome an increasing load of misfolded proteins in the ER lumen during acute ER-stress situations by accelerating protein turnover at the cost of specificity. Condensate properties have recently been reported for the cytosolic chaperones Hsp70 and J-domain proteins, to specifically prevent aggregation of amyloid-forming proteins13,45. From these and our observations, the pattern seems to emerge that subcompartmentalization by chaperone condensates may be a general theme to structure cellular compartments. The chaperone superstructures in the ER not only provide a fascinating layer of functionality to enable protein biogenesis but might also result in novel therapeutical approaches to target protein misfolding diseases. Human PDIA6 lacking its ER signal sequence (residues 19–440), domain ab (residues 161–440) and domain b (residues 276–440), with an additional N-terminal His6 tag including a tobacco etch virus (TEV) cleavage sequence and domain a (residues 161–272) with an additional N-terminal His6 tag, followed by a SUMO tag were synthesised by GenScript. The genes were inserted using NcoI and XhoI restriction enzymes into a pET28a expression vector (Novagen). BL21-(DE3)-Lemo cells (New England Biolabs) were transformed and cultured at 37 °C in Luria–Bertani (LB) medium containing 30 mg ml−1 kanamycin. Expression was induced at an optical density at 600 nm (OD600) of 0.6 by adding 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG) and incubating at 25 °C for 12 h. The cells were harvested by centrifugation at 4,000g for 20 min. The pellet was resuspended in 35 ml lysis buffer per litre of culture (25 mM HEPES (pH 7.5), 300 mM NaCl, 10 mM MgCl2, 5 mM ATP (Sigma-Aldrich), 0.02 mg ml−1 ribonuclease RNase A (Roche), 0.01 mg ml−1 deoxyribonuclease DNase I (AppliChem) and 0.2 mg ml−1 phenylmethylsulfonyl fluoride). Cell lysis was performed using a microfluidizer (Microfluidics; three cycles at 4 °C). The soluble bacterial lysate was separated from cell debris and other components by centrifugation at 14,000g for 60 min and loaded onto a Ni-NTA column (Cytiva) equilibrated in buffer A (25 mM HEPES (pH 7.5), 300 mM NaCl and 10 mM imidazole). The PDIA6 was denatured with 6 M urea and loaded onto a Ni-NTA column (Cytiva) equilibrated in buffer A + 6 M urea to further separate the protein from nucleotide and protein contamination. No changes in the protein conformation of refolded and not refolded PDIA6 were observed in two-dimensional-HMQC spectra and SEC-MALS analyses. PDIA6 was eluted at a 300 mM imidazole concentration in buffer containing 6 M urea. PDIA6 refolding was achieved by overnight dialysis against buffer (25 mM HEPES (pH 7.5) and 300 mM NaCl). After refolding, the His tag was cleaved by overnight incubation with 1 mg TEV protease or Ulp1 SUMO protease per 50 mg of PDIA6 in cleavage buffer (25 mM HEPES (pH 7.5), 300 mM NaCl, 1 mM dithiothreitol and 0.5 mM EDTA) at 4 °C. The his–TEV–domain b and his–TEV–domain ab constructs were not cleaved. His–TEV–domain a0 and his–TEV–domain a used for amide backbone assignment were not cleaved. Cleaved PDIA6 constructs were separated from the tag, TEV or Ulp1 and uncleaved PDIA6 by a Ni-NTA column equilibrated in buffer A. Finally, PDIA6 was concentrated by ultrafiltration and injected into size-exclusion chromatography columns (Superdex-200 16/600 PG for constructs >30 kDa or Superdex-75 16/600 PG for constructs <30 kDa; Cytiva) to further purify the proteins and adjust the protein to its final buffer (25 mM HEPES (pH 7.5), 150 mM KCl and 10 mM MgCl2). The PDIA6 was concentrated by ultrafiltration and stored at −20 °C until use. The final yield of purified protein was 25 mg per litre of LB medium. To obtain fully reduced PDIA6, the protein was incubated with 10 mM tris(2-carboxyethyl)phosphine (TCEP) at room temperature for 1 h, buffer exchanged (3×) to SEC buffer (25 mM HEPES (pH 7.5), 150 mM KCl and 10 mM MgCl2) and immediately frozen. Before performing the calcium-binding experiments, PDIA6 was incubated with 20 mM EDTA for 1 h at room temperature and dialysed overnight against SEC buffer at 4 °C. Human BiP lacking its ER signal sequence (residues 19–654) with an N-terminal His6 tag including a TEV cleavage sequence was synthesized by GenScript. The gene was inserted through NcoI and XhoI into a pET28a expression vector (Novagen). BL21-(DE3)-Lemo cells (New England Biolabs) were transformed and cultured at 37 °C in LB medium. BiP expression was induced at an OD600 of 0.6 by adding 1 mM IPTG, and expression was continued at 25 °C for 12 h. The cells were harvested by centrifugation at 5,000g for 20 min. The pellet was resuspended in 20 ml lysis buffer per litre of culture (25 mM HEPES (pH 7.5), 150 mM NaCl, 10 mM MgCl2, 0.02 mg ml−1 ribonuclease RNase A (Roche), 0.01 mg ml−1 deoxyribonuclease DNase I (AppliChem) and 0.2 mg ml−1 phenylmethylsulfonyl fluoride). Cell lysis was performed using a microfluidizer (Microfluidics; three cycles at 4 °C). The soluble bacterial lysate was separated from cell debris and other components by centrifugation at 14,000g for 60 min and loaded onto a Ni-NTA column (Cytiva) equilibrated in buffer A (25 mM HEPES (pH 7.5), 150 mM NaCl and 10 mM MgCl2). The BiP was denatured with 6 M urea and loaded onto a Ni-NTA column equilibrated in buffer + 8 M urea. The BiP was eluted at a 500 mM imidazole concentration in buffer containing 8 M urea. BiP was separated from the tag, TEV and uncleaved BiP via a Ni-NTA column (Cytiva) equilibrated in buffer A. The BiP was then applied to an anion-exchange column (QHP, Cytiva) equilibrated in buffer QA (25 mM Tris (pH 8.5)) and eluted using a gradient of buffer QB (25 mM Tris (pH 8.5) and 1 mM KCl) at a concentration of 250 mM KCl. Finally, the BiP was concentrated by ultrafiltration and subjected to size-exclusion chromatography (Superdex-200 16/600 PG, Cytiva) to further purify the proteins and adjust to its SEC buffer. Finally, the BiP was concentrated by ultrafiltration and stored at −20 °C until use. The final yield of purified protein was 15–20 mg WT BiP per litre of LB medium. Methyl-labelled PDIA6 proteins were obtained by growing the expression cells in M9 minimal media prepared with 99.85% D2O (Sigma-Aldrich) containing 15NH4Cl (1 g l−1; Sigma-Aldrich) and d-glucose-d7 (2 g l−1; Sigma-Aldrich). At an OD600 of 0.8, a solution containing the labelled precursors was added as follows. For [U-2H, 15N, 12C], Met-[13CH3]ε, Val-[13CH3]γ2 and Leu-[13CH3]δ2 labelling of PDIA6 methionine leucine valine, 240 mg 2-hydroxy-2-[13C]methyl-3-oxo-4,4,4-tri-[2H]-butanoate (pro-S acetolactate-13C, NMR-Bio) and 100 mg of [13C]ε-L-methionine (Sigma-Aldrich). For [U-2H, 15N, 12C], Met-[13CH3]ε and Val-[13CH3]γ2 labelling of PDIA6 methionine valine, 240 mg 2-hydroxy-2-[13C]methyl-3-oxo-4,4,4-tri-[2H]-butanoate (pro-S acetolactate-13C, NMR-Bio), 30 mg of l-leucine-d10 and 100 mg of [13C]ε-l-methionine (Sigma-Aldrich). One hour after the addition of the precursors, protein expression was induced by adding 0.5–1 mM IPTG and incubating at 25 °C for 12 h. Methyl-labelled proteins were purified following the protocols described earlier. The QuikChange II mutagenesis protocol (Stratagene) was used to introduce the PDIA6 mutations M182L; M331L; M359L; M367L; L61M; V75I; V124I; L139M; L165M; L196M; I234V; L320M; L345M; L349M; L385M; I406V; V407I; L419M; V421I; I425V; L427M; L432M; L435M; L440M; K161 to stop (domain a0); L139STOP (domain a0 Δlinker); P274 to stop (domain a0a, domain a); S428 to stop (domain b ΔC(13)); 420STOP (ΔC(20); domain b ΔC(20); domain ab ΔC(20)); E422Q, D423N, D424N, D426N, D429N, E431Q, D433N and D434N (EQDN, dom b EQDN); R142Q, K150N and R153Q; K150N, R153Q and K159N; R142Q, K150N, R153Q, K159N and K160N (5× linker mutant) and the BiP mutation G407 to stop (NBD). All PCR primers were obtained from Microsynth. The plasmids were purified using the Zyppy plasmid mini prep kit (Zymo Research) and mutations were confirmed by sequencing by Microsynth AG. The experiments were recorded on Bruker Avance IIIHD 600 MHz, Avance III 700 MHz, Avance Neo 800 MHz or Avance IIIHD 900 MHz spectrometers running Topspin 3.6–4.0 and equipped with a cryogenically cooled triple-resonance probe. The NMR data were processed with nmrPipe46 2022.230.12.16 and ccpnmr47 2.5.2. Secondary chemical shifts were calculated relative to random-coil values48. linker peptide and 1 mM K2HPO4/KH2PO4 were used as indicated. To mimic ER homeostatic-like conditions, 10 mM TCEP or 15 mM glutathione (GSH) with 7:1 GSH:glutathione disulfide (GSSG) was used in the presence of 10 mM Ca2+. Samples behaved identically in the presence of 10 mM TCEP and 15 mM GSH with 7:1 GSH:GSSG. The samples were incubated before the experiment (about 1 h for TCEP, GSH and ADP; about 30 min for protein–protein interaction experiments, Ca2+ and Gd3+; and overnight for peptides). Data were analysed using Excel 16.97. PDIA6 Met[CH3]ε, Val[CH3]γ2 and Leu[CH3]δ2 assignment was obtained using a structure-based approach combining mutagenesis, specific valine labelling schemes49, subconstructs and three-dimensional 13C, 13C-resolved [1H, 1H]-NOESY experiments recorded at 37 °C in SEC buffer with 10 mM TCEP and 5% D2O. The following point mutations were used: M182L, M331L, M359L, M376L, L61M, V75I, V124I, V139M, L165M, L196M, I234V, L320M, L345M, L349M, L385M, I406V, V407I, L419M, V421I, I425V, L427M, L432M, L435M and L440M. The following subconstructs were used: his–domain a and his–domain ab. Each sample was recorded at 37 °C with an adjusted duration depending on the final concentration of each mutant (experimental time ranging from 120 to 240 min per sample). Together, this allowed for the assignment of 62/66 Met[CH3]ε and Leu[CH3]δ2 and Val[CH3]γ2 methyl groups (94%). All experiments were recorded at sample concentrations of 0.5–1.0 mM. All PDIA6 constructs were N-terminally tagged with DyLight 488 amine-reactive dyes (Thermo Scientific) at a concentration of 10 mg ml−1 in 25 mM MES (pH 6.5), 150 mM KCl and 10 mM MgCl2 in the dark for 1 h at 25 °C. Free dye was removed by overnight dialysis at pH 6.5 and 4 °C in the dark, and the labelled proteins were buffer exchanged (3×) to SEC buffer. BiP NBD was N-terminally tagged with DyLight 633 amine-reactive dyes (Thermo Scientific) following the same protocol. The protein concentration and extent of labelling were determined according to manufacturer's protocol for all constructs and only proteins labelled with 0.75 and 1.5 moles dye per mole protein were used for experiments. SEC-MALS measurements of PDIA6 constructs were performed at 25 °C in SEC buffer using a S200 10/300 GL column (Agilent Technologies) on an Agilent 1260 high-performance liquid chromatography system. Proteins were eluted at a concentration of 20 μM. Elution was monitored by multiangle light scattering (Heleos II 8+; Wyatt Technology), differential refractive index (Optilab T-rEX; Wyatt Technology) and absorbance at 280 and 254 nm (1260 UV; Agilent Technologies). The column was equilibrated overnight in the running buffer to obtain stable baseline signals from the detectors before data collection. All system parameters were calibrated using an injection of 2 mg ml−1 BSA solution (ThermoPierce) and standard protocols in ASTRA 6. Molar mass and mass distributions were calculated using the ASTRA 6 software (Wyatt Technology). Differential scanning fluorimetry data were acquired using a Prometheus NanoTemper NT.48 instrument (NanoTemper Technology) at protein concentrations of 100 μM in SEC buffer. The samples were scanned from 20 °C to 95 °C at a scan rate of 0.5 °C min−1. Protein unfolding was measured by detecting the temperature-dependent change in tryptophan fluorescence at emission wavelengths of 330 and 350 nm. For the determination of the dissociation constants N-terminally tagged DyLight 488–PDIA6 at 20 nM was incubated with increasing amounts of BiP in SEC buffer in the presence of 5 mM ADP, 1 mM KH2PO4/K2HPO4, 0.05% Tween and 1 mM CaCl2 prepared by serial dilution. The samples were incubated for 30 min at 25 °C before being loaded into capillary tubes. The LED power was set to 100% and the microscale thermophoresis (MST) power to medium. The pre-MST period was 5 s, the MST-acquisition period was 30 s and the post-MST period was 5 s. Measurements were performed in triplicates on a Monolith NT.115 system (NanoTemper) and integrated at an interval of 2.5 s. Mass photometry data were obtained using a Refeyn OneMP system in microscope coverslips prepared according to the manufacturer's instructions with silicone gaskets. To lock the focus, 18 µl SEC buffer was used, followed by the addition of 2 µl protein sample at the indicated final concentrations. PDIA6 his–domain bΔC (P274–L427) in 25 mM HEPES (pH 7.5), 150 mM KCl, 5 mM MgCl2 and 10 mM TCEP was crystallized at 10 mg ml−1 concentration at room temperature in sitting-drop vapour diffusion experiments in a 1:1 ratio of protein to precipitant. Crystals appeared in 1.9 M (NH4)2SO4, 0.1 M HEPES (pH 7.0) and 1% dimethylsulfoxide after two days and grew to their final size within ten days. The crystals were cryopreserved by the addition of ethylene glycol to a final concentration of 20% (vol/vol) and flash-cooled in liquid nitrogen. Data were collected at the SLS beamline X06DA (Swiss Light Source, Paul Scherrer Institute, Switzerland) at 100 K, and integrated, indexed and scaled using the XDS software51,52. The crystal structure was determined by molecular replacement with Phaser53 1.0 using PDB accession code 3uem (ref. Automated model building55 was carried out with Arp/Warp56 8.0, followed by Autobuild in PHENIX57 1.17, and completed by manual model building with Coot58 0.9.8 and structure refinement in PHENIX59 1.17. Model quality was validated with Molprobity60 4.4. Data collection and refinement statistics are in Supplementary Table 1. Disulfide reductase activity was assessed using the NADPH-based insulin reduction assay. Purified PDIA6 constructs (1 μM final concentration) were added to 1 mM reduced GSH, 800 μM NADPH, 0.16 units GSH reductase (Sigma-Aldrich) and 30 μM human insulin (Sigma-Aldrich) in SEC buffer and the absorbance of NADPH monitored at 340 nm using a Biotek Synergy H1 multi-mode microplate reader for 25 min at 25 °C. Chaperone activity was assessed using the citrate synthase aggregation assay. Citrate synthase (Escherichia coli; NZYTech) was dialysed against 40 mM HEPES (pH 7.5). Purified PDIA6 constructs (10 μM final concentration) were added to 1 μM citrate synthase in 40 mM HEPES (pH 7.5) buffer and incubated at 45 °C and 300 rpm. Protein aggregation was monitored at 350 nm using a Biotek Synergy H1 multi-mode microplate reader at the indicated time points. PDIA6 liquid–liquid phase separation (LLPS) was examined at 25 μM in SEC buffer in the presence of 10% polyethylene glycol (PEG3350, mass fraction, freshly prepared immediately before the experiment) as a crowding agent in the absence/presence of 15 mM GSH with 7:1 GSH:GSSG or 10 mM TCEP, 10 mM CaCl2 and/or 20 mM EDTA. The PEG was added last by simultaneously swirling the sample and gently pipetting up and down to thoroughly mix the sample. Images were taken 30 min after the addition of PEG. No PDIA6 construct formed liquid droplets in the absence of crowding agent and at concentrations below 1 mM. Phase-separated PDIA6 droplets formed under ER-mimicking conditions (15 mM GSH with 7:1 GSH:GSSG + 1–10 mM CaCl2) as well as under strongly reducing conditions (10 mM TCEP) in the presence of 1–10 mM CaCl2. Several salt concentrations (50–350 mM) were tested. Unlabelled proteins were used to confirm that they undergo phase separation without fluorescent labelling. Glass-bottomed 96-well plates were coated with 1% Pluronic F-127 overnight and the wells were washed three times with SEC buffer. For fluorescence microscopy, 1% N-terminally tagged DyLight protein was added to the unlabelled protein. BiP NBD LLPS was examined at 20 μM in SEC buffer in the presence of 5 mM ADP and 10% PEG as a crowding agent in the absence or presence of 20 μM PDIA6. Samples including BiP NBD were prepared as described for PDIA6. A wide-field microscope (FEI MORE) with a Hamamatsu ORCA flash 4.0 cooled sCMOS camera using a TIRF APON ×60/1.49 and U Plan S Apo ×100/1.4 oil objective with the Live Acquisition 2.5 software were used for capturing images. Images were processed in the OMERO biological image data management system. All images are representative of at least three independent sets of experiments. Post-acquisition colouring of the channels was performed according to artistic preference. HeLa CCL2, HEK293A and U2OS cells were cultured at 37 °C and 5% CO2 in high-glucose Dulbecco's modified Eagle's medium (Sigma-Aldrich) supplemented with 10% fetal bovine serum (Biowest), 2 mM l-glutamine (Gibco), 1 mM sodium pyruvate (Sigma) and 1×penicillin–streptomycin (Sigma). INS-1 832/13 cells were cultured at 37 °C and 5% CO2 in RPMI-1640 medium (Sigma) supplemented with 2 mM l-glutamine (Gibco), 1 mM sodium pyruvate (Sigma), 10 mM HEPES (BioConcept), 0.05 mM EmbryoMax β-mercaptoethanol (Sigma) and 10% fetal bovine serum. The HeLa CCL2 (ATCC CCL-2) and HEK293A (ATCC CVCL-6910) cell lines were a gift from M. Spiess; their identities were authenticated by short-tandem-repeat analysis by Microsynth AG (last in 2021). The U2OS cell line (ATCC HTB-96) was a gift from M. Hall. The INS-1 832/13 cell line was purchased from Sigma-Aldrich (catalogue number SCC207). All cell lines were confirmed to be mycoplasma-negative by PCR. For transient cell transfections, the cells were plated into six-well plates to reach 70% confluency the following day and transfected with 1 µg plasmid DNA complexed with Helix-IN transfection reagent (OZ Biosciences). For small interfering RNA (siRNA) knockdown experiments, cells were transfected with a final concentration of 50 pmol Silencer Select siRNA targeting PDIA6 (siRNA1, GAAUGUUCUGGACAGUGAAtt, Ambion, s19715 and siRNA2, GGCAGUGAAUGGUCUGUAUtt, Ambion, s19716) using RNAiMAX (Invitrogen) as per the manufacturer's protocol. The cells were replated onto coverslips or an imaging chamber the following day and fixed or imaged 24–48 h post transfection. For stress treatments, the cells were treated with 1 μM tunicamycin, 1.25 μg ml−1 thapsigargin or 8 μM cyclopiazonic acid for the specified duration. For washout experiments, the cells were treated with 1 μM tunicamycin, 1.25 μg ml−1 thapsigargin or 8 μM cyclopiazonic acid for 10 h, washed with PBS and cultured in fresh medium overnight. For experiments with blocked proteasomal degradation, the cells were treated with 5 μM MG-132 for 18 h. To ensure ER localization, the KDEL signal sequence (present in native PDIA6) was cloned at the C terminus of PDIA6–tGFP (Origene, RG201710) using a NEBuilder HiFi Assembly cloning kit (New England Biolabs) with primers designed by the NEBuilder assembly tool following the manufacturer's instructions. For simplicity, we refer to this plasmid as PDIA6–GFP. The C-terminal tail of PDIA6 was truncated by removing the last twelve amino acids from its sequence in the PDIA6-tGFP-KDEL plasmid using a NEB site-directed mutagenesis kit following the manufacturer's instructions and primers selected using the NEBaseChanger tool. PDIA6 L128A, L131A, L134A, V135A and L139A has been characterized to be monomeric23 and is hence referred to as ‘PDIA6 monomer'. The genes were inserted using BamHI and XhoI restriction enzymes into a pcDNA3.1(+) expression vector (Novagen). The QuikChange II mutagenesis protocol (Stratagene) was used to introduce the Akita proinsulin mutation C96Y. Hamster BiP–mCherry was a gift from E. Snapp (Addgene plasmid 62233) and mutated to the human sequence, and mApple–Sec61 was a gift from J. Lippincott-Schwartz. All PCR primers were obtained from Microsynth. The plasmids were purified using the Zyppy plasmid mini prep kit from Zymo Research and mutations were confirmed by sequencing by Microsynth AG. The guides were synthesized by Microsynth and cloned into the hSpCas9 pX330 plasmid (Addgene plasmid 42230) by annealing oligonucleotides and ligation into the vector, which was linearized with BbsI. The homology repair was designed as follows: 500 bp left homology arm–glycine-serin linker (GSGSGSGSGS)–HaloTag–linker (PSRLEEELRRRLTEP)–ER retention sequence (KDEL)–PolyA–G418/geneticin resistance cassette–500 bp right homology arm. Both plasmids were confirmed by sequencing (Microsynth). HeLa CCL2 cells at 70% confluency were transfected with 0.5 µg pX330 plasmid with the PDIA6 guide and 0.5 µg homology repair plasmid using Helix-IN transfection reagent (OZ Biosciences). Three days post transfection, 1.5 mg ml−1 G418 (Thermo Fisher) was added; the medium was exchanged every 2–3 days with fresh medium containing 1.5 mg ml−1 G418 until selection was complete (two weeks). To generate a CRISPR–Cas9-mediated PDIA6-knockout cell line, the PDIA6 genomic locus was targeted with the following guide RNAs: 5′-GCTCGTGAAGGATCGCCTCG-3′ (gRNA1, targeting exons 5–7, depending on the splice variant) and 5′-AGACCGCGTTGGGGATTGGA-3′ (gRNA2, targeting exons 11–13, depending on splice variant). The guides were synthesized by Microsynth and cloned into px458-mCherry (gRNA1) and px459-Puro (gRNA2) vectors by annealing oligonucleotides and ligation into the vector, which was linearized with BbsI. The plasmid was confirmed by sequencing (Microsynth). HeLa CCL2 and Hek293A cells at 70% confluency were transfected on a 10 cm dish with px458-mCherry (gRNA1) plasmid (2.5 µg per dish) and px459-Puro (gRNA2) plasmid containing the PDIA6 guides (2.5 µg per dish) using Helix-IN transfection reagent (OZ Biosciences). For selection, the cells were treated with 1.5 µg ml−1 puromycin one day post transfection for 24 h. FACS sorting (mCherry+) was carried out 48 h after transfection. The cells were trypsinized and resuspended in cell-sorting medium (2% FCS and 2.5 mM EDTA in PBS) and sorted on a Fortessa flow cytometer. Individual mCherry+ cells were collected, seeded into a 96-well plate and expanded. PDIA6 expression was determined by western blots. All of the surviving clones were positive for PDIA6, indicating that PDIA6 is essential. HeLa cells were plated onto coverslips 24 h before fixation or stress treatment. At specified time points, the cells were fixed with 4% paraformaldehyde, permeabilized with 0.1 % Triton X-100, blocked in PBS containing 5% fetal bovine serum and stained with polyclonal anti-PDIA6 (1:200; GeneTex, GTX33397), monoclonal anti-GRP78/BiP (1:200; Invitrogen, 1H11-1H7; lot: XI3695781), polyclonal anti-calreticulin (1:200; Proteintech, 27298-1-AP), polyclonal anti-DNAJB11/ERdj3 (1:200; Thermo Fisher, 15484-1-AP), monoclonal anti-Grp94 (1:200; Enzo, SPA-850; lot: 103418), anti-PDIA1 (1:200; GeneTex, GTX2279), polyclonal anti-calnexin (1:200; Enzo, ADI-SPA-860; a gift from the Speiss laboratory), followed by Alexa Fluor (AF)488- or AF633-conjugated goat anti-rabbit, AF633-conjugated goat anti-mouse, AF488- or AF633-conjugated goat anti-rabbit, AF488- or AF633-conjugated goat anti-rabbit, AF488-conjugated goat anti-rat or AF546-conjugated donkey anti-rat, AF633-conjugated goat anti-mouse and AF488- or AF633-conjugated goat anti-rabbit (all 1:500), respectively. Proinsulin was visualized within the condensates by staining pre-proinsulin–myc–his-transfected HeLa cells with anti-myc clone 9E10 (1:2,000; Sigma-Aldrich, M4439; lot: 029M4849V) primary antibody, followed by AF633-conjugated anti-mouse (1:500). PDIA6 in the PDIA6–HaloTag knock-in HeLa CCL2 cell line was stained with polyclonal anti-HaloTag (1:250; Promega, G928A), followed by AF488-conjugated goat anti-rabbit (1:500). Coverslips were mounted onto glass slides with Fluoromount G (Southern Biotech) and sealed with nail polish. Confocal images were acquired using an Olympus Fluoview FV3000 system and an UPLSAPO ×60/1.30 objective with silicone oil using the FV3000 (FV21S-SW version 2.5.1) system software. Laser intensities were at 0.5–3.0% for both the 488 (AF488) and 640 nm (AF633) wavelengths. All images are representative of at least three independent sets of experiments. All images for corresponding experiments were processed with the same settings to insure comparable results. Post-acquisition colouring of the channels was performed according to artistic preference. For live-cell imaging, cells were seeded on an imaging chamber (ibidi μ-slide) 24 h before data acquisition. Knock-in cells expressing PDIA6–Halo–KDEL fusion were labelled for 1 h using 200 nM JFX-554 (Promega) in the cell culture medium. After the labelling, the cells were rinsed and incubated for at least 1 h in culture medium. Live-cell imaging was performed in complete growth medium lacking phenol red at 37 °C with 5% CO2 on a wide-field microscope (FEI MORE) with a Hamamatsu ORCA flash 4.0 cooled sCMOS camera using a TIRF APON ×60/1.49 oil objective and an Olympus ‘SpinD' spinning-disc confocal microscope with a Hamamatsu ORCA-Fusion, sCMOS using an UPL APO ×60/1.5 oil objective. Laser intensities were set to 5–10% for the 488 nm wavelength. The system was preheated overnight (MORE) or 1 h (SpinD) to 37 °C to prevent thermal fluctuations. Images of the movies were recorded every 0.5–1.0 s. Post-acquisition colouring of the channels was performed according to artistic preference. The FRAP experiments were performed on an Olympus ‘SpinD' spinning-disc confocal microscope with a Hamamatsu ORCA-Fusion, sCMOS using an UPL APO ×60/1.5 oil objective. PDIA6 droplets and condensates were bleached using a 488 nm laser with 100% intensity (Rapp OptoElectronic Firefly). A total ten images were obtained prebleaching, and 200 images were recorded every 1.0–1.83 s (in vitro) or 0.5–1.0 s (in vivo) post bleaching. Sec61 was bleached using a 488 nm laser with 100% intensity (Rapp OptoElectronic Firefly). A total of ten images were obtained prebleaching and 30–200 images were recorded post bleaching with a time interval of 1.5 s (PDIA6–GFP WT co-transfected cells) or 2.3 s (PDIA6–GFP 5× linker mutant co-transfected cells). Intensities of FRAP regions were extracted with the Fiji 1.52p software tool, background corrected and reported relative to the prebleaching time point (PDIA6–GFP) or normalized as a percentage of recovery (mApple–Sec61). Images were analysed using Omero webserver 5.26.0, Fiji 1.52p and Python 3.9. Cell images were processed using a custom Python 3.9 script with OpenCV 4.10.00 and NumPy 1.26.4 to detect phase-separated condensates based on predefined morphological parameters. The detected condensates were quantified using circularity-based segmentation, enabling the extraction of key morphological properties, including size, shape and number, for further analysis. Microscopy images were processed using a custom Python 3.9 script with OpenCV 4.10.00 and NumPy 1.26.4, employing the Hough Transform to detect circular droplets. The number of detected droplets per image was then quantified, enabling further analysis of their distribution. Cell images were processed using a custom Python 3.9 script with OpenCV 4.10.00 and NumPy 1.26.4 for cell segmentation via Otsu thresholding and condensate detection. Following feature extraction, the relative intensities of condensed and dispersed protein populations in each cell were quantified to assess protein enrichment in condensates. Pearson correlation coefficients were determined using the Fiji 1.52p ‘JaCOP' plugin from 10–15 cells (see Supplementary Table 2). Statistical significance was calculated using a two-tailed Student's t-test. The fluorescence intensity histograms were extracted using the Fiji 1.52p software ‘plot profile' tool. To assess PDIA6 and BiP expression levels, HeLa CCL2 cells were lysed in lysis buffer (1% Triton X-100, 150 mM NaCl, 20 mM Tris pH 7.5, 1 mM EDTA, 1 mM EGTA and protease inhibitor) and denatured in Laemmli buffer at 65 °C for 10 min. The three biological replicates were prepared and lysed independently (n = 3). Samples were resolved by 10% SDS–PAGE and transferred onto nitrocellulose membrane (Amersham). The membranes were blocked with TBST (20 mM Tris (pH 7.6), 150 mM NaCl and 0.1% Tween 20) with 5% non-fat dry milk for 30 min and incubated overnight with primary polyclonal anti-PDIA6 (1:2,000; GeneTex, GTX33397), monoclonal primary anti-GRP78/BiP (1:2,000; Invitrogen, 1H11-1H7; lot: XI3695781) or primary monoclonal anti-α-tubulin (1:2,000; Sigma, T5168; lot: 0000242797) at 4 °C, followed by 2 h incubation with horseradish peroxidase (HRP)-conjugated secondary antibody (1:20,000; anti-mouse or anti-rabbit; Invitrogen, 31430 and 31460) in TBST. Chemiluminescence signals were detected using Immobilon western HRP substrate (Advansta) and imaged using a FusionFX imaging system (Vilber Lourmat). To asses insulin processing in HeLa cells, the cells were transfected with pre-proinsulin (and PDIA6 constructs). After 24 h, the cells were washed with preheated 1×PBS and 4×OPTIMEM GlutaMAX (Gibco), and incubated in OPTIMEM GlutaMAX (16 h). To assess insulin processing in INS-1 cells, the cells were washed 36 h post transfection (of PDIA6 constructs only) with HBSS (Gibco) + 2.5 mM glucose (Gibco) for 1 h and subsequently incubated with HBSS + 2.5 mM glucose for 2 h. The medium was collected, spun down (10 min, 13,000 rpm, 4 °C) and the supernatant was precipitated by the addition of 100% trichloroacetic acid (one volume trichloroacetic acid to four volumes protein). After 10 min incubation on ice, the samples were spun down (5 min, 13,000 rpm, 4 °C) and washed twice with cold acetone. The dried pellet was dissolved in 30 μl Laemmli buffer by vortexing and boiling (10 min, 300 rpm, 95 °C). The cells were lysed in RIPA buffer and denatured in Laemmli buffer at 65 °C for 10 min. The four biological replicates were prepared and lysed independently (n = 4). Protein levels were measured using a BCA assay (Merck) and the volumes of the precipitate and lysates were adjusted to equal lysate protein concentrations. The lysate blot was processed as described above. The blot of the secreted proteins was preblocked with TBST containing 5% non-fat dry milk for 5 min and the samples were fixed by incubation with TBST containing 0.2% glutaraldehyde for 15 min. The membrane was washed in TBST, blocked with TBST containing 5% non-fat dry milk for 30 min and incubated overnight with monoclonal anti-proinsulin (for secretion, 1:1,000; Invitrogen, MA1-83256; lot: 160454; for lysate, 1:2,000; Cell Signaling, 3014; lot: 10) in Can get signal solution (Toyobo) at 4 °C, followed by 2 h incubation with HRP-conjugated secondary antibody (1:20,000; anti-mouse or anti-rabbit; Invitrogen, 31430 or 31460, respectively) in Can get signal solution. Chemiluminescence signals were detected using Immobilon western HRP substrate (Advansta) and imaged using a FusionFX imaging system (Vilber Lourmat). HeLa cells were seeded in 96-well plates 24 h after transfection. After 24 h, MTT (Merck) was added at the indicated time points to a final concentration of 500 μg ml−1 and incubated for 2 h at 37 °C. The medium was removed and the crystals were dissolved in acidic (0.08 M HCl) isopropanol. The absorbance (A550nm − A690nm) was measured on a plate reader. All experiments were repeated in experimental and independent biological triplicates. Significance was calculated using a two-tailed Student's t-test (n = 9). RNA was extracted and purified from cells using an RNeasy kit according to the manufacturer's instructions. Complementary DNA was reverse-transcribed using GoScript reverse transcriptase primed with a mix of Oligo(dT)s and random hexamers (Promega). Real-time PCR was then performed using GoTaq qPCR master mix (Promega) and primers specific for sXBP1 (sXBP1, 5′-GCTGAGTCCGCAGCAGGT-3′; β-actin, 5′-CACCATTGGCAATGAGCGGTTC-3′). For PDIA6 WT-transfected cells, the sXBP1 levels were still below detection limit after 40 cycles of quantitative PCR and the Ct was thus set to 40 for statistical analysis. The significance was calculated from three independent biological replicates using a two-tailed Student's t-test (n = 3). For microscopy, PDIA6–GFP (WT and 5× linker mutant)-transfected HeLa cells were plated onto coverslips 24 h before fixation. Briefly, the cells were fixed for 30 min with 4% formaldehyde solution at room temperature, incubated with permeabilizing solution (1×assay buffer, 0.5 mM EDTA and 0.05% Triton X-100) on ice for 30 min, incubated for 30 min with the PROTEOSTAT detection reagent at room temperature in the dark, mounted on a microscopy slide with Fluoromount G (Southern Biotech) and sealed with nail polish. For aggresome detection analysis by flow cytometry, PDIA6–GFP (WT and 5× linker mutant)-transfected HEK293A cells and controls were pelleted, fixed with 4% formaldehyde solution for 30 min at room temperature and permeabilized (1×assay buffer, 0.5 mM EDTA and 0.05% Triton X-100), resuspended in FACS buffer (PBS containing 2% FCS and 0.05 μM EDTA) with PROTEOSTAT dye (1:1,250 dilution) and incubated for 30 min at room temperature in the dark. A Fortessa flow cytometer was used to measure mCherry fluorescence (PROTEOSTAT dye) of GFP-positive cells. Data were analysed using FlowJo 10.9.0. The gating strategy is in Supplementary Fig. The BiP structural model in the ADP undocked states was obtained by homology modelling using the SWISS-MODEL server and the DnaK undocked conformation (PDB 2kho)61 as a template. The structural model of PDIA6 FL has been modelled as a chimaera of the individual domain crystal structures (domain a0, PDB: 4ef0; domain a, PDB: 4gwr; domain b, PDB: 8cpq, determined in this study). The linker connecting domain a0 and a, and the C-terminal tail were modelled as disordered based on the NMR data presented in Fig. The interface between the domain a0 monomers is based on the crystal structure of domain a0 (PDB: 4ef0) and confirmed by NOE contacts between the domain, as presented in Fig. Importantly, the distances V75–L131 (7.3 Å), V75–L134 (11.4 Å), V75–V135 (12.7 Å) and L72–L131 (7.8 Å) within the monomer are too large to have (strong) intramolecular methyl NOE contacts, indicating that the resulting NOEs are intermolecular NOEs. The measured NOE crosspeak intensities fit to the expected intermolecular NOE intensities based on the distances calculated from the crystal structure (PDB: 4ef0; V75–L131, 4 Å; V75–L134, 4 Å; V75–L135, 7.1 Å and L72–L131, 7.5 Å). The interface between domains a and b was modelled using ESMFold62 based on the NOE contact between the domains presented in Fig. To determine the PDIA6 Ca2+ binding affinity (KD(Ca2+)), the Ca2+-dependent formation of droplets in the presence of a crowding agent was used as a direct measurable of the Ca2+ binding to PDIA6. The number of unlabelled PDIA6 droplets was quantified for a Ca2+ concentration between 100 μM and 1,000 μM from differential-interference-contrast microscopy images and fitted using nonlinear regression to the Hill equation binding model (equation (1)) to determine the KD(Ca2+). where nc is the number of condensates and n is the Hill coefficient. The structure model was generated with ESMFold61 for the full-length protein and manually trimmed to residues 161–419 to have a more compact, yet fully functional, construct. To generate equilibrated starting structures for the pulling simulations, each structure (WT and mutant) was placed in a dodecahedral box of TIP4 water, to which 150 mM NaCl was added, including neutralizing counterions. Following steepest descents minimization, each of the systems was equilibrated for 100 ps under a constant-pressure (constant number of particles, pressure, and temperature) ensemble, with position restraints applied to peptide heavy atoms throughout. Protein and non-protein atoms were coupled to separate temperature coupling baths and temperature was maintained at 300 K using the velocity rescale coupling method. Short-production molecular dynamics (MD) simulations were conducted for 100 ps in the absence of any restraints to relax the initial structures. All simulations were conducted using the GROMACS package (version 2024.3). Structures at the end of each of the previous relaxed trajectories (WT and 274PPP–274GGG) were used as starting configurations for pulling simulations. Residues 284–419 were pulled away from the structure over 500 ps, using a spring constant of 1,000 kJ mol−1 nm−2 and a pull rate of 0.1 Å ps−1. A final centre-of-mass collective variable distance of approximately >6.0 Å was achieved. From these trajectories, snapshots were taken to generate the starting configurations for the umbrella sampling windows. A distribution of sampling windows was used, such that the window spacing was 1 Å centre-of-mass separation, and resulted in each WT and mutant case in nine windows. In each window, 1 ns of MD was performed for a total simulation time of 9 ns utilized for umbrella sampling. Analysis of results was performed with the weighted histogram analysis method (WHAM). All in-cell data are based on multiple experiments with independent biological replicates. No statistical methods were used to pre-determine sample sizes but our sample sizes are similar to those reported in previous publications. Data distribution was assumed to be normal but this was not formally tested. A two-sided unpaired Mann–Whitney test was used for statistical analyses of the number of condensates per cell and condensate area. Statistical significance for all other analyses was calculated using a two-tailed Student's t-test. Statistical analyses were performed using Excel 16.97 or GraphPad Prism 9. Details are provided in the figure legends. Where statistical comparisons were performed, specific P values are indicated on the graphs. No samples or data points were excluded. The experiments were not randomized. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. PDIA6 methionine leucine valine methyl and his–domain a0, his–domain a and his–domain ab amide backbone resonance assignments have been submitted to the Biological Magnetic Resonance Data Bank under the following accession codes: 51863 and 51864. Domain delimitations for the generation of PDIA6 subconstructs were obtained from the UniProt database Q15084 (ref. Guide RNAs were designed using the human reference genome GRCh38 (RefSeq accession: NC_000001.11). Specific target sequences were identified using CRISPOR64 and confirmed against transcript ENST00000272227. Microscopy images have been deposited on Zenodo at https://doi.org/10.5281/zenodo.15497399 (ref. All other data supporting the findings of this study are available from the corresponding author on reasonable request. Source data are provided with this paper. Custom Python scripts used in this paper for image quantification are available without access restriction at https://doi.org/10.5281/zenodo.15424556 (ref. Anelli, T. & Sitia, R. Protein quality control in the early secretory pathway. Protein folding and modification in the mammalian endoplasmic reticulum. Vincenz-Donnelly, L. & Hipp, M. S. The endoplasmic reticulum: a hub of protein quality control in health and disease. & Braakman, I. Co- and post-translational protein folding in the ER. Hebert, D. N. & Molinari, M. In and out of the ER: protein folding, quality control, degradation, and related human diseases. Quality control in the endoplasmic reticulum. The protein disulfide isomerase family: from proteostasis to pathogenesis. Hatahet, F. & Ruddock, L. W. Substrate recognition by the protein disulfide isomerases. Essential cell-extrinsic requirement for PDIA6 in lymphoid and myeloid development. Chhabra, N. F. et al. A point mutation in the Pdia6 gene results in loss of pancreatic β-cell identity causing overt diabetes. Bai, Y. et al. PDIA6 modulates apoptosis and autophagy of non-small cell lung cancer cells via the MAP4K1/JNK signaling pathway. & Xu, J. W. Key genes associated with diabetes mellitus and hepatocellular carcinoma. Li, Y. et al. Hsp70 exhibits a liquid–liquid phase separation ability and chaperones condensed FUS against amyloid aggregation. Muzzopappa, F. et al. Detecting and quantifying liquid–liquid phase separation in living cells by model-free calibrated half-bleaching. Protein disulfide isomerase A6 controls the decay of IRE1α signaling via disulfide-dependent association. Both platelet- and endothelial cell-derived ERp5 support thrombus formation in a laser-induced mouse model of thrombosis. Rasmussen, U., Brøogger Christensen, S. & Sandberg, F. Thapsigargine and thapsigargicine, two new histamine liberators from Thapsia garganica L. Acta Pharm. Duksin, D. & Bornstein, P. Changes in surface properties of normal and transformed cells caused by tunicamycin, an inhibitor of protein glycosylation. Lebeau, P. F., Platko, K., Byun, J. H. & Austin, R. C. Calcium as a reliable marker for the quantitative assessment of the endoplasmic reticulum stress in live cells. Burdakov, D., Petersen, O. H. & Verkhratsky, A. Intraluminal calcium as a primary regulator of endoplasmic reticulum function. Kikuchi, M., Doi, E., Tsujimoto, I., Horibe, T. & Tsujimoto, Y. Functional analysis of human P5, a protein disulfide isomerase homologue. Okumura, M. et al. A unique leucine–valine adhesive motif supports structure and function of protein disulfide isomerase P5 via dimerization. Bastos-Aristizabal, S., Kozlov, G. & Gehring, K. Structural insight into the dimerization of human protein disulfide isomerase. Bakunts, A. et al. Ratiometric sensing of BiP–client versus BiP levels by the unfolded protein response determines its signaling amplitude. Melnyk, A., Lang, S., Sicking, M., Zimmermann, R. & Jung, M. Co-chaperones of the human endoplasmic reticulum: an update. Jessop, C. E., Watkins, R. H., Simmons, J. J., Tasab, M. & Bulleid, N. J. Protein disulphide isomerase family members show distinct substrate specificity: P5 is targeted to BiP client proteins. Meunier, L., Usherwood, Y. K., Chung, K. T. & Hendershot, L. M. A subset of chaperones and folding enzymes form multiprotein complexes in endoplasmic reticulum to bind nascent proteins. Mayer, M., Kies, U., Kammermeier, R. & Buchner, J. BiP and PDI cooperate in the oxidative folding of antibodies in vitro. Marcinowski, M. et al. Substrate discrimination of the chaperone BiP by autonomous and cochaperone-regulated conformational transitions. Yan, Y., Rato, C., Rohland, L., Preissler, S. & Ron, D. MANF antagonizes nucleotide exchange by the endoplasmic reticulum chaperone BiP. & Hendershot, L. M. ERdj3, a stress-inducible endoplasmic reticulum DnaJ homologue, serves as a cofactor for BiP's interactions with unfolded substrates. Structural transitions modulate the chaperone activities of Grp94. Pearse, B. R. & Hebert, D. N. Lectin chaperones help direct the maturation of glycoproteins in the endoplasmic reticulum. Jang, I. et al. PDIA1/P4HB is required for efficient proinsulin maturation and β cell health in response to diet induced obesity. Vasiljević, J., Torkko, J. M., Knoch, K. P. & Solimena, M. The making of insulin in health and disease. The unfolded protein response signals through high-order assembly of Ire1. Karagöz, G. E. et al. An unfolded protein-induced conformational switch activates mammalian IRE1. Gu, J. et al. Hsp40 proteins phase separate to chaperone the assembly and maintenance of membraneless organelles. Delaglio, F. et al. NMRPipe: a multidimensional spectral processing system based on UNIX pipes. The CCPN data model for NMR spectroscopy: development of a software pipeline. Kjaergaard, M. & Poulsen, F. M. Sequence correction of random coil chemical shifts: correlation between neighbor correction factors and changes in the Ramachandran distribution. Mas, G., Crublet, E., Hamelin, O., Gans, P. & Boisbouvier, J. Specific labeling and assignment strategies of valine methyl groups for NMR studies of high molecular weight proteins. Hiller, S., Fiorito, F., Wüthrich, K. & Wider, G. Automated projection spectroscopy (APSY). Kabsch, W. Integration, scaling, space-group assignment and post-refinement. McCoy, A. J. et al. Phaser crystallographic software. Wang, C. et al. Human protein-disulfide isomerase is a redox-regulated chaperone activated by oxidation of domain a'. Hoh, S. W., Burnley, T. & Cowtan, K. Current approaches for automated model building into cryo-EM maps using Buccaneer with CCP-EM. Cohen, S. X. et al. ARP/wARP and molecular replacement: the next generation. Terwilliger, T. C. et al. Iterative model building, structure refinement and density modification with the PHENIX AutoBuild wizard. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Adams, P. D. et al. PHENIX: building new software for automated crystallographic structure determination. Chen, V. B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR. Source images and videos to: A multi-chaperone condensate enhances protein folding in the endoplasmic reticulum. Mas, G. Supplemental code to: A multi-chaperone condensate enhances protein folding in the endoplasmic reticulum. Parashar, S. et al. Endoplasmic reticulum tubules limit the size of misfolded protein condensates. We thank the Biophysics Facility (T. Sharpe and T. Mühlethaler), the Imaging Core Facility (K. Schleicher and S. Roig), the Swiss NMR platform (T. Müntener) and the FACS facility (S. Malysheva and S. Stefanova) for technical support, and A. Mazur for MD simulations. J. Lippincott-Schwartz, F. Bottanelli and E. Snapp are acknowledged for sharing plasmids. We thank B. Reversade and E. Karagöz for fruitful discussions. This work was funded by the Swiss National Science Foundation (grant number 207755 to S.H. Anna Leder, Guillaume Mas, Viktória Szentgyörgyi, Roman P. Jakob, Timm Maier, Anne Spang & Sebastian Hiller Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar expressed and purified BiP protein. recorded and analysed NMR data. wrote the paper with contributions from all authors. Correspondence to Guillaume Mas or Sebastian Hiller. The authors declare no competing interests. Nature Cell Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Quantification of endogenous PDIA6 enrichment in condensates visualized by anti-PDIA6 antibody. Mean and standard deviation (n = 25 cells, pooled across three independent experiments). A total of 2,300 condensates were counted. b, PDIA6 was endogenously tagged with HaloTag in HeLa cells (PDIA6EN-Halo). Live-cell imaging of HeLa cells stably PDIA6EN-Halo fusion. PDIA6 was visualized by staining with the HaloTag ligand JFX-554. PDIA6 was visualized by the HaloTag ligand JFX-554. Mean and standard deviation (n = 25 cells, pooled across three independent experiments). A total of 2,320 condensates were counted. d, Fixed HeLa cells stably expressing PDIA6EN-Halo fusion. PDIA6 was stained by an α-Halo-antibody. Mean and standard deviation (n = 25 cells, pooled across three independent experiments). A total of 2,650 condensates were counted. f, Quantification of transiently transfected PDIA6–GFP enrichment in condensates visualized by the eGFP signal. Mean and standard deviation (n = 31 cells, pooled across three independent experiments). A total of 720 condensates were counted. g, Fixed HeLa cells transfected with PDIA6–GFP (eGFP signal) and stained against PDIA6 (anti-PDIA6 antibody, imaged in red channel) recorded at low (left) and high exposure (right). h, Fixed HeLa cells expressing endogenous PDIA6 (pink, left) and transfected with PDIA6–GFP (pink, right) stained for DAPI (blue). PDIA6–GFP was visualized by imaging eGFP. i, Live-cell imaging of HeLa cells transfected with PDIA6–GFP (pink), treated with ER tracker (red). j, Fixed HEK293A cells expressing endogenous PDIA6 (left) and live-cell imaging of HEK293A cells transfected with PDIA6–GFP (right). Endogenous PDIA6 was stained with an anti-PDIA6 antibody, overexpressed PDIA6–GFP was visualized by imaging eGFP. k, Fixed U2OS cells expressing endogenous PDIA6 (left) and live-cell imaging of U2OS cells transfected with PDIA6–GFP (right). Endogenous PDIA6 was stained with an anti-PDIA6 antibody, overexpressed PDIA6–GFP was visualized by imaging eGFP. l, Time series of a single HeLa cell stably expressing PDIA6EN-Halo fusion. Images were not bleach-corrected n, Fluorescence recovery analysis of PDIA6EN-Halo FRAP experiment presented in m. Representative data of three independent replicates. a, Fixed HeLa cells co-transfected with PDIA6–GFP (pink) and Akita proinsulin-myc (green). Akita proinsulin, a folding-defective mutant of proinsulin18, was used to mimic the accumulation unfolded/misfolded proteins in the ER lumen. Akita proinsulin forms puncta, as previously reported61. PDIA6 was visualized by imaging eGFP, Akita proinsulin using an anti-myc antibody. b, Western blot showing expression levels of PDIA6, BiP and α-tubulin and at indicated time points of Tm treatment in HeLa cells. Representative blot of three independent experiments. c, Quantification of PDIA6 and BiP levels from blot shown in b. Data representative of an experimental triplicate (n = 3). d–f, Fixed HeLa cells expressing endogenous PDIA6 and live-cell imaging of HeLa cells transfected with PDIA6–GFP after 10 h of Tm treatment (d), 10 h of Tg treatment (e) or 16 h of CPA (reversible Tg analogue) treatment (f) (left) and 24 h after drug washout (right). Endogenous PDIA6 was stained with an anti-PDIA6 antibody, overexpressed PDIA6–GFP was visualized by imaging eGFP. g, Fixed HeLa cells expressing endogenous PDIA6 observed at different time points after Tm (top) and Tg treatment (bottom). a, In vitro phase separation of PDIA6 (left) and Dylight 488–PDIA6 (right) with 10 mM Ca2+ in absence (top) and presence (bottom) of 10% PEG3350. b, In vitro time evolution of Dylight 488–PDIA6 droplets in presence of 10 mM Ca2+. d, Fluorescence recovery analysis of the Dylight 488–PDIA6 FRAP experiments presented in c. Experimental triplicate in shades of grey. f, In vitro phase separation of unlabelled PDIA6 in presence of crowding agent in absence (left) and presence of 10 mM Ca2+ (middle), and in presence of 10 mM Ca2+ and 20 mM EDTA (right). g, In vitro phase separation of Dylight 488–PDIA6 at different concentrations in presence of crowding agent and different Ca2+ concentrations after 30 min (left) and after 5 min (right), as indicated. Assignment of Met[13C1H]ε, Val-[13C1H]γ2 and Leu-[13C1H]δ2 methyl groups are indicated. Peaks annotated with a “t” correspond to residues in the C-terminal tail which could not be unambiguously assigned. b, Assigned methyl groups (pink spheres) displayed on the structure of PDIA6 established in Fig. Blue spheres represent partially assigned residues in the C-terminal tail. Sequence-specific resonance assignments are indicated. f, Percentages of assigned residues of methyl and amide groups in PDIA6 FL and domain constructs. a, SEC elution profiles (solid lines, left axis) and MALS apparent molecular mass (dotted lines, right axis) of PDIA6 full-length (FL) and domains. The extended helix α4 of PDIA6 is highlighted by a pink square. Residues forming the leucine-valine adhesive dimerization motif23 are marked by an orange square. Domain delimitations as defined on uniport.org or by available crystal structures. c, Structural comparison of PDI catalytically active domains helix α4. Occupancy and oligomeric state are indicated for each peak. e, Crystal structure of PDIA6 domain b at a resolution of 1.8 Å. f, Zoom on the electron density map of PDIA6 domain b crystal structure. g, Potential of mean force along the collective variable (CV). h, Protein logo analysis of PDIA6 of domain a-b linker of Homo sapiens, Rattus norvegicus, Mus musculus, Pongo abelii, Drosophila melanogaster and Caenorhabditis elegans. i, Thermal stability of reduced domain a (light grey), domain b (dark grey) and domain ab (pink)as determined by differential scanning fluorimetry. Full spectra shown in Supplementary Fig. Full spectra shown in Supplementary Fig. Gd3+ by 1 mM Ca2+ on full-length PDIA6 (bottom). Residues with substantial loss in intensity are highlighted in shades of purple (I, II and III). d,e, Zoom on Ca2+ binding site of calsequestrin low Ca2+ form (PDB: 5kn3, yellow) (left) and overlay with PDIA6 (grey) Ca2+ binding sites I (d) and II (e) (right). Side chains of Ca2+ binding residues shown as sticks. The Ca2+ ions in calsequestrin are coordinated by the amino acid residues Asp/Glu/Asp for site I and Asn/Asn/Asp/Asp for site II, while in PDIA6 by Asp/Ser/Ser and Asn/Glu/Gln/Thr, respectively, resulting in weaker Ca2+ binding. f, Overlayed sections of 2D [15N,1H]-HSQC spectra of 100 μM PDIA6 domain ab in absence (black) and in presence of excess Ca2+ showing assigned residues located at Ca2+ binding sites. Full spectra in Supplementary Fig. 4. g, Soluble expression yields of PDIA6 constructs with mutations (S263A, D271L, E280M, N284K, T337V or Q342K) targeting the Ca2+ binding sites. h, Fixed HeLa cells transfected with PDIA6–GFP ΔC. i, In vitro phase separation of Dylight 488–PDIA6 EQDN, which has been reported to not bind Ca2+ at the C-terminal tail23. b, In vitro phase separation of Dylight 488–PDIA6 subconstructs. The EQDN mutant has been described before25 to not bind Ca2+ to the C-terminal tail (binding site III). Scale bar, 10 μm c, Residue-specific populations of the conformation P (dark blue) and O (light blue). Each bar corresponds to one methyl group in domain b. d, Combined methyl chemical shift differences between conformations P and O in domain b. The significance threshold of 2 SDs is indicated by a pink line. e, Methyl groups with substantial chemical shift difference between conformations P and O marked as pink spheres on the structure of domain b. Residues located around helix α3 show largest chemical shift differences between the two conformations. f, Electrostatic surface potential (ESP) of domain b. Helix α3 forms part of a negatively charged patch. g, SEC elution profiles (solid lines, left axis) and MALS apparent molecular mass (dotted lines, right axis) of PDIA6 WT (pink), (R142Q, K150N, R153Q) (light blue), (K150N, R153Q, K159N) (dark blue) and 5x linker mutant (grey). All proteins eluted as a dimer. h, Melting temperature of PDIA6 WT and linker mutant constructs in SEC buffer and in presence of 10 mM of Ca2+ determined by DSF. Mean and standard deviation (n = 3 technical triplicates). Colour code as in g. i, Insulin reduction by PDIA6 WT and linker mutants. Insulin reduction rate of the linker mutants relative to the WT protein (right). Colour code as in g. Mean and standard deviation (n = 3 technical triplicates). j, Chaperone activity of PDIA6 WT and linker mutants assessed by the aggregation of citrate synthase as a model substrate. Colour code as in g. Mean and standard deviation (n = 3 technical triplicates). k, Quantification of condensates per cell of PDIA6 WT (same data as in Fig. 1e unstressed (us), pink) and PDIA6 5x linker mutant (grey) transfected cells (***P = 1.14·10−7, two-sided t-test), (n = 17 cells per condition pooled across three independent experiments). l, Protein expression levels of endogenous PDIA6 and transfected PDIA6–GFP in HeLa cells. m, In vitro phase separation of Dylight 488–PDIA6 WT + Dylight 488–PDIA6 5x linker mutant at 1:1 stoichiometry in presence of 10 mM Ca2+ at a final protein concentration of 25 μM (left) and 50 μM (middle) and 25 μM Dylight 488–PDIA6 WT for comparison (right). a, Quantification of enrichment of BiP in endogenous PDIA6 condensates (left, n = 25 cells, pooled across three independent experiments, a total of 2,300 condensates were counted) and in transiently transfected PDIA6–GFP condensates (right, n = 31 cells, pooled across three independent experiments, a total of 720 condensates were counted). b, Co-localization of overexpressed PDIA6–GFP (pink) and overexpressed BiP–mCherry (blue) in live HeLa cells (left) and of endogenous PDIA6 (pink) and overexpressed BiP–mCherry (blue) in fixed HeLa cells (right). c, Co-localization of overexpressed PDIA6–GFP (pink) and endogenous BiP (blue) in fixed HeLa cells at low (left) and high (right) exposure. Full spectra shown in Supplementary Fig. Full spectra shown in Supplementary Fig. The magnitudes of 1.5 SD and 2 SDs are indicated by pink lines. g, Combined amide chemical shift perturbation of PDIA6 domain ab upon titration of BiP. The magnitude of 1.5 SD is indicated by a purple line, disappearing residues in blue. h, Methyl groups with substantial CSP upon interaction with BiP shown as pink spheres on the structure of domain b. Amide groups with substantial CSP are shown as purple surface. Full spectra shown in Supplementary Fig. 6. j, Combined methyl chemical shift perturbation of BiP upon titration of 2 eq. The magnitudes of 2 SDs is indicated by a blue line. k, Methyl groups with substantial CSP upon interaction with PDIA6 shown as blue spheres on the structure of the undocked ADP-bound BiP (modelled on PDB: 2kho). l, Microscale thermophoresis signal of Dylight 488–PDIA6 plotted against BiP concentration in presence of 1 mM Ca2+. Mean and standard deviation (n = 3, technical triplicates). a, Co-localization of overexpressed PDIA6–GFP (pink) and endogenous Grp94 (blue) (top), overexpressed PDIA6–GFP (pink) and endogenous ERdj3 (blue) (bottom) and overexpressed PDIA6–GFP (pink) and endogenous PDIA1 (blue) (right) in fixed HeLa cells. PDIA6 was visualized using the eGFP signal. b, Co-localization of overexpressed PDIA6–GFP (pink) and endogenous calnexin (blue) in fixed HeLa cells. PDIA6 was visualized using the eGFP signal. Fluorescence intensities across the condensates were measured along the yellow line and are shown as histograms (right). c, Co-localization of endogenous BiP (pink) and endogenous calreticulin (blue) (top) or calnexin (blue) (bottom) in fixed HeLa cells. Fluorescence intensities across the condensates were measured along the yellow line and are shown as histograms (right). a, Co-localization analysis by Pearson Correlation Coefficient (PCC) for pixel intensity of proinsulin and endogenous and overexpressed (*) PDIA6 in HeLa cells. Individual data points (PCC per cell) (n = 15/11 cells for PDIA6-proinsulin and PDIA6*-proinsulin respectively) pooled across three independent experiments and standard deviation (for details see Supplementary Table 2). b, Secretion levels of insulin (medium) and expression levels of proinsulin, PDIA6 and α-tubulin (lysate) in proinsulin, proinsulin + PDIA6–GFP WT or proinsulin + PDIA6–GFP 5x linker mutant transfected HeLa cells after 18 h treatment with the proteasome inhibitor MG-132. c, Live-cell imaging of INS-1 cells transfected with PDIA6–GFP WT (left) or PDIA6–GFP 5x linker mutant (right). PDIA6–GFP was visualized by imaging eGFP. d, Aggresomes detected by PROTEOSTAT dye (green) in fixed HeLa cells transfected with PDIA6–GFP WT (left) or PDIA6–GFP 5x linker mutant (right) (pink). e,f, FRAP experiments of mApple–Sec61 (grey) in HeLa cells, co-transfected with PDIA6–GFP WT (e) or PDIA6–GFP 5x linker mutant (f) (pink). Due to strong bleaching of the mApple, contrast and brightness were adjusted for each image individually. g, Fluorescence recovery curve of photobleached mApple–Sec61 in PDIA6–GFP WT (pink) or PDIA6–GFP 5x linker mutant (grey) co-transfected HeLa cells. Average and standard deviation of experimental triplicates. Bleached areas did not overlap with condensates. h,i, Cell viability of PDIA6–GFP WT (pink) and PDIA6–GFP 5x linker mutant (grey) transfected cells relative to mock-transfected cells by MTT. (***P = 0.0003, two-sided t-test) (n = 9, triplicates of three independent biological experiments; h) and trypan blue assay (***P = 0.0003, two-sided t-test) (n = 9, parallel triplicates from three independent biological experiments; i). For the trypan blue assay a total of 12,760 (PDIA6–GFP WT), 8,519 (PDIA6–GFP 5x) and 16,150 (mock-transfected) cells were counted (i). j,k, Cell viability of PDIA6-knockdown cells, of PDIA6–GFP WT (pink) and PDIA6–GFP 5x linker mutant (grey) transfected cells relative to mock-transfected cells by MTT (PDIA6-siRNA1: **P = 0.003, PDIA6-siRNA2: ***P = 0.00008, two-sided t-test)(n = 9, triplicates of three independent biological experiments; j) and trypan blue assay (PDIA6-siRNA1: ** p = 1.64·10−8, PDIA6-siRNA2: *** p = 0.0004, two-sided t-test) (n = 9, triplicates of from three independent biological experiments; k). For the trypan blue assay a total of 90,350 (PDIA6–GFP WT, siRNA1), 76,070 (PDIA6–GFP 5x, siRNA1), 75,040 (mock-transfected, siRNA1), 54,850 (PDIA6–GFP WT, siRNA2), 66,210 (PDIA6–GFP 5x, siRNA2) and 41,530 (mock-transfected, siRNA2) PDIA6-knockdown cells were counted (k). Statistics on diffraction data and refinement of PDIA6 domain b (residues 274–427). *Values in parenthesis are for the highest resolution shell. Co-localization of ER chaperones in PDIA6 condensates. PCC for pixel intensity of the indicated proteins (protein 1 and protein 2) in 10–15 cells (n). Number of independent experiments with similar outcomes. All microscopy images and western blots are representative of the results of all independent repetitions. Fusion event of PDIA6–GFP condensates in HeLa cells. Fission event of PDIA6–GFP condensates in HeLa cells. Fluorescence recovery after photobleaching of PDIA6–GFP condensates in HeLa cells, showing a first phase of fast recovery due to diffusion, followed by a second phase of slow recovery due to exchange of material. HeLa cells stably expressing PDIA6–HaloTag fusion. PDIA6 is visualized by the HaloTag ligand JFX-554. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Leder, A., Mas, G., Szentgyörgyi, V. et al. A multichaperone condensate enhances protein folding in the endoplasmic reticulum. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250810093242.htm'>Scientists capture the secret quantum dance of atoms for the first time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-11 05:31:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>And yet, this quantum dance is far from chaotic; the dancers follow a strict choreography. In molecules, this strange behavior has another consequence: Even if a molecule should be completely frozen at absolute zero, it never truly comes to rest. The atoms it is made of perform a constant, never-ending quiet dance driven by so-called zero-point energy. For a long time, these patterned zero-point movements were considered impossible to measure directly. However, scientists at Goethe University Frankfurt and partner institutions have now succeeded in doing precisely that at the world's largest X-ray laser, the European XFEL in Hamburg, Germany. They captured the "dance of the atoms" by shining a "spotlight" on individual molecules and taking snapshots of their atoms - revealing each atom's precise choreography. Professor Till Jahnke from the Institute for Nuclear Physics at Goethe University Frankfurt and the Max Planck Institute for Nuclear Physics in Heidelberg explains: "The exciting thing about our work is that we were able to see that the atoms don't just vibrate individually, but that they vibrate in a coupled manner, following fixed patterns. This zero-point motion is a purely quantum mechanical phenomenon that cannot be explained classically." While the motion patterns of molecules with two or three atoms are fairly easy to follow, it quickly becomes complex with medium-sized molecules - like the studied iodopyridine, which consists of eleven atoms. Looking back, many puzzle pieces had to come together perfectly." Using a technique called Coulomb Explosion Imaging, molecules are triggered to undergo a controlled explosion by ultrashort, high-intensity X-ray laser pulses, allowing high-resolution images of their structure to be generated. This COLTRIMS reaction microscope has been developed over the past decades by Goethe University's Atomic Physics group. A version tailored specifically to the European XFEL was built by Dr. Gregor Kastirke during his PhD work. Seeing the device in action is something special, Kastirke says: "Witnessing such groundbreaking results makes me feel a little proud. After all, they only come about through years of preparation and close teamwork." "We're constantly improving our method and are already planning the next experiments," says Jahnke. "Our goal is to go beyond the dance of atoms and observe in addition the dance of electrons - a choreography that is significantly faster and also influenced by atomic motion. With our apparatus, we can gradually create real short films of molecular processes - something that was once unimaginable." This Little-Known Amino Acid Could Be the Secret to Rapid Weight Loss Could This Tiny Molecule Be Better Than Ozempic and Wegovy for Obesity? DNA From Three People Creates Healthy Babies Free of Genetic Disease Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            