
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-30</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09297-0'>ACLY inhibition promotes tumour immunity and suppresses liver cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 15:56:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Immunosuppressive tumour microenvironments are common in cancers such as metabolic dysfunction-associated steatohepatitis (MASH)-driven hepatocellular carcinoma (HCC) (MASH-HCC)1,2,3. Although immune cell metabolism influences effector function, the effect of tumour metabolism on immunogenicity is less understood4. ATP citrate lyase (ACLY) links substrate availability and mitochondrial metabolism with lipid biosynthesis and gene regulation5,6,7. Although ACLY inhibition shows antiproliferative effects in various tumours, clinical translation has been limited by challenges in inhibitor development and compensatory metabolic pathways8,9,10,11,12. Here, using a mouse model of MASH-HCC that mirrors human disease, genetic inhibition of ACLY in hepatocytes and tumours reduced neoplastic lesions by over 70%. To evaluate the therapeutic potential of this pathway, a novel small-molecule ACLY inhibitor, EVT0185 (6-[4-(5-carboxy-5-methyl-hexyl)-phenyl]−2,2-dimethylhexanoic acid), was identified via phenotypic screening. EVT0185 is converted to a CoA thioester in the liver by SLC27A2 and structural analysis by cryo-electron microscopy reveals that EVT0185-CoA directly interacts with the CoA-binding site of ACLY. Transcriptomic and spatial profiling in mice and humans linked reduced tumour ACLY with increases in the chemokine CXCL13, tumour-infiltrating B cells and tertiary lymphoid structures. Together, these findings illustrate how targeting tumour metabolism can rewire immune function and suppress cancer progression in MASH-HCC. These metabolic adaptations often involve enhanced glycolysis and de novo lipogenesis, enabling sustained biomass production and proliferation. At the same time, these pathways reshape the tumour microenvironment by limiting the availability of key nutrients, and by generating immunosuppressive byproducts such as lactate and succinate13,14. As a result, tumour metabolism not only fuels intrinsic cancer cell growth but also contributes to immune evasion. HCC exemplifies the interplay between metabolic dysfunction and immune suppression. Although once primarily linked to viral hepatitis, alcohol and toxins, HCC is increasingly driven by MASH1,15,16. MASH-HCC arises in a distinctly immunosuppressive hepatic microenvironment that is resistant to immunotherapy compared with viral-driven disease1,15,16. Although recent clinical observations suggest that features such as intratumoural steatosis17, reduced B cell infiltration3 and downregulation of the chemokine CXCL13 (ref. 17) may influence immune responsiveness in MASH-HCC, the mechanisms linking tumour metabolism to immune regulation remain poorly defined. ACLY functions at a key metabolic junction, converting citrate into acetyl-CoA and oxaloacetate, thereby linking carbohydrate availability to fatty acid and cholesterol synthesis and histone acetylation5,6,7. Through these roles, ACLY supports both the anabolic growth demands of tumour cells and epigenetic regulation of gene expression5,6,7. Although ACLY inhibition has been shown to suppress tumour proliferation in many preclinical models18,19,20, its therapeutic potential has been questioned due to compensatory upregulation of alternative acetyl-CoA-generating pathways, including acetate CoA synthetase 2 (ACCS2)8,10, pyruvate dehydrogenase (PDH)9,10, pantothenate kinase 2 (PANK2)11 and fatty acid oxidation12, that may bypass ACLY dependency. However, most studies have largely relied on cell lines and xenograft models that lack intact immune systems and do not reflect the immune-metabolic complexity of solid tumours such as MASH-HCC. Of note, although ACLY is upregulated in HCC and reduced expression correlates with improved survival, previous studies have focused predominantly on viral-related or toxin-related HCC1,21,22,23,24,25, leaving its role in MASH-HCC unexplored. Given the immunosuppressive nature of the MASH-HCC microenvironment and the emerging links with metabolism, we hypothesized that ACLY may serve as a key metabolic regulator of tumour–immune interactions. To model MASH-HCC, male C57BL/6J mice were injected with diethyl nitrosamine (DEN) and then maintained on a control chow (control-DEN) or a high-fat/fructose Western diet with physiologically relevant levels of cholesterol26 (WD-DEN) starting at 8 weeks for 28 weeks (Extended Data Fig. Compared with control-DEN mice, WD-DEN mice showed increased liver weights, liver:body weight ratios and significantly elevated levels of plasma α-fetoprotein (AFP), a marker of HCC (Extended Data Fig. Histological analysis confirmed enhanced steatosis, ballooning and inflammation in WD-DEN livers (Extended Data Fig. 1e,f), along with a marked increase in surface tumours and tumour burden (Extended Data Fig. Lesions were histologically classified as non-neoplastic or neoplastic, the latter showing features typical of human MASH-HCC, including macrovesicular or microvesicular steatosis, ballooning, Mallory–Denk bodies and lymphocyte infiltration (Extended Data Fig. WD-DEN mice developed more neoplastic lesions (Extended Data Fig. 1m) with significant macrovesicular or microvesicular steatosis, ballooning and lymphocyte infiltration that were pathologically similar to those of humans with MASH-HCC27 (Extended Data Fig. To assess relevance to human MASH-HCC, we performed bulk RNA sequencing of tumours and compared gene expression with patient samples and two other models: control-DEN28 and WD-CCl4 (FAT-MASH)29 (Extended Data Fig. The majority (8 out of 9) of WD-DEN tumours uniquely matched the Hoshida S1 subtype found in patients with MASH-HCC, whereas control-DEN and WD-CCl4 aligned with S2 or S3 subtypes (Extended Data Fig. Single-cell RNA sequencing (scRNA-seq) validated immunological similarity, with lymphocytes and AFP+ and GPC3+ tumour cells in WD-DEN and WD-CCl4 mice having similar correlation coefficients to humans with MASH-HCC (Extended Data Fig. Acly expression was broadly distributed across cell types in WD-DEN livers, mirroring the human condition (Extended Data Fig. Together, these findings demonstrate that the WD-DEN model recapitulates key pathological, molecular and immune features of advanced human MASH-HCC. To control for baseline tumour burden, mice were stratified by plasma AFP levels at 28 weeks, and two groups with equivalent AFP levels were injected with hepatocyte-targeted adeno-associated virus 8 (AAV8)-TTR vectors expressing either YFP (wild type (WT)) or Cre recombinase (Acly knockout (KO)) to induce hepatocyte-specific Acly deletion (Fig. Eight weeks post-injection, immunohistochemistry showed marked reductions in ACLY expression in both tumoural and non-tumoural hepatocytes of Acly-KO mice, but not in infiltrating immune or mesenchymal cells such as endothelial and Kupffer cells (Fig. Pathology analysis showed similar scores for liver steatosis, ballooning and inflammation between genotypes (Extended Data Fig. 2c), but biochemical assessment of non-tumour liver tissue revealed reduced levels of several fatty acids in Acly-KO mice (Extended Data Fig. Despite comparable liver pathology, Acly-KO mice exhibited markedly fewer surface tumours (Fig. 1f–i) and reduced tumour surface area (Extended Data Fig. Fewer than 30% of Acly-KO mice had more than 30 surface tumours, versus 90% of WT mice (Fig. Histological classification indicated that this reduction was driven primarily by an approximately 70% decrease in neoplastic lesions, with many Acly-KO mice having none (Fig. 1i), alongside a trend towards fewer non-neoplastic proliferative lesions (P = 0.0563; Extended Data Fig. Neoplastic lesions in Acly-KO mice had reduced steatosis (Fig. 1j), consistent with smaller lipid droplet size (Extended Data Fig. 1k) and reductions in fatty acids (Fig. These findings demonstrate that hepatocyte-specific Acly deletion in a physiologically relevant MASH-HCC model reduces tumour number, size, severity and lipid content. a, Experimental scheme of the Acly-KO model. b, Representative images of ACLY protein expression in WT and hepatocyte-specific Acly-KO liver and tumours. c,d, The H-score of ACLY protein expression in tumour (c) and tumour-adjacent liver (d). e, Representative Acly-KO mouse liver stained with ACLY antibody. The asterisk indicates inflammatory cell aggregation-positive ACLY staining; the blue arrows denote mesenchymal cell (endothelial or Kupffer cells)-positive ACLY staining; and the orange arrows show hepatocyte-negative ACLY staining. g, Visible number of tumours on the liver surface. P values were determined by unpaired, two-tailed Student's t-test. h, Percentage distribution graph of tumour numbers from livers of WT (n = 11) and Acly-KO (n = 11) mice. i, Number of neoplastic lesions in livers from WT (n = 11) and Acly-KO (n = 11) mice. P values by unpaired, two-tailed Student's t-test. k, Representative images showing tumour with lipid (left) and calculation of the percentage lipid area of WT (n = 5) and Acly-KO (n = 5) livers (right). P values were determined by unpaired, two-tailed Student's t-test. C:D, the total number of carbon atoms to the number of carbon–carbon double bonds. P values were determined by unpaired, two-tailed Student's t-test. Over the past three decades, several ACLY inhibitors have been described5. Bempedoic acid competitively inhibits ACLY activity and allosterically activates AMP-activated protein kinase-β1 (AMPKβ1)-containing complexes when converted to bempedoic acid-CoA in hepatocytes30. Potent allosteric ACLY inhibitors have also been developed32, but poor bioavailability and/or cell permeability have hindered development5. To identify more potent and cell-permeable ACLY inhibitors, we conducted a phenotypic screen in primary mouse hepatocytes, evaluating fatty acid and cholesterol synthesis across a library of compounds varying in hydrocarbon chain length, terminal functional groups and core chemical substitutions. Several candidates inhibited lipogenesis (Supplementary Table 1 and originally disclosed in ref. 33), with EVT0185, selected for further study due to strong inhibition of de novo lipogenesis (−84% at 100 µM, half-maximal inhibitory concentration (IC50) = 0.46 µM; Fig. Like bempedoic acid, EVT0185 was converted to EVT0185-CoA in rat liver microsomes (Extended Data Fig. Incubation of HEK293 cells with EVT0185 followed by mass spectrometry analysis indicated that only cells expressing SLC27A2 generated the CoA thioester (Extended Data Fig. Transcriptomic analyses revealed that SLC27A2 is upregulated in livers of people with MASH or MASH-HCC compared with healthy controls (Extended Data Fig. 3i), with no expression in immune cells per the protein atlas and single-cell datasets (Extended Data Fig. Consistent with this, EVT0185 had no effect on B cell or T cell proliferation in vitro (Extended Data Fig. These data indicate that EVT0185 is converted to a CoA thioester by SLC27A2, in the liver and tumours but not in immune cells of MASH-HCC. b, Effect of EVT0185 and EVT0185-CoA on ACLY activity. f, 2D cryo-EM class averages for the ACLY–(R,S)-EVT0185-CoA complex. ess, effective sample size; ptcls, particles. g, Cryo-EM reconstruction without symmetry applied. h,i, Sharpened cryo-EM map contoured at 9.94σ (h; absolute level of 0.12) and real space refinement atomic model (i) following local refinement after symmetry expansion and 3D classification. The interacting residues of ACLY are labelled. The disordered pantothenyl arm is tentatively indicated. ACLY is inhibited by palmitoyl-CoA5, therefore we hypothesized that the conversion of EVT0185 to its CoA thioester may drive the inhibition of ACLY activity. In cell-free assays, EVT0185-CoA (not the unconjugated diacid) inhibited recombinant human ACLY (hACLY) activity (Fig. 2b) and this effect was competitive with CoA (Fig. hACLY is a 1,101-residue polypeptide forming a functional 0.5-MDa tetramer and featuring an N-terminal citryl-CoA synthetase (CCS) module, consisting of CCSβ and CCSα regions, and a C-terminal citrate synthase homology (CSH) domain that serves as the oligomerization platform of the ACLY enzyme and contains the CoA-binding site32,35 (Fig. Shuttling of this high-energy reaction intermediate to the CSH module of ACLY, facilitated by flipping of the long pantothenyl-arm of citryl-CoA, positions the citryl-thioester moiety in the CSH active site where citryl-CoA undergoes retro-aldol cleavage into oxaloacetate and acetyl-CoA35,36. To directly examine the molecular interactions by which EVT0185-CoA inhibits hACLY, the ACLY–(R,S)-EVT0185-CoA complex was characterized using single-particle cryo-electron microscopy (cryo-EM) analysis (Fig. 4c) and revealed a pseudo-D2-symmetric ACLY assembly where four CCS modules arrange around the central CSH module (Fig. Markedly, the cryo-EM map for one CCS module was poorly defined, indicating structural heterogeneity in this region. To resolve this pseudo-D2-symmetric assembly at a higher resolution, we applied symmetry expansion in combination with local refinement followed by 3D classification (Extended Data Fig. This approach improved the overall map quality (Fourier shell correlation of 0.143 (FSC0.143) = 3.3 Å; Fig. 4e and Supplementary Table 2) and ligand density, which we interpreted as representing the adenosine 3′-phosphate 5′-diphosphate moiety of bound EVT0185-CoA with the pantothenyl arm and dicarboxylic acid moiety disordered (Fig. On the basis of this experimentally observed binding mode, which is analogous to the binding of CoA to ACLY (Extended Data Fig. 4f,g), we speculate that ACLY-bound EVT0185-CoA may adopt both compact and extended conformations in which the terminal carboxylic acid moiety may anchor itself in the citrate-binding sites of the CSH or CCS module, respectively. In addition to inhibiting ACLY, fatty acyl-CoAs or bempedoic acid-CoA can activate AMPK, which suppresses fatty acid and sterol synthesis by phosphorylating and inhibiting acetyl-CoA carboxylase (ACC) and HMG-CoA reductase30,37 (Extended Data Fig. To assess potential ACLY-independent effects of EVT0185, we measured lactate incorporation into fatty acids and sterols in hepatocytes from WT and Acly-KO mice, comparing results to bempedoic acid at equimolar doses (Extended Data Fig. We also tested both compounds for their ability to inhibit acetate incorporation, a process independent of ACLY but sensitive to ACC and/or ACSS2 inhibition (Extended Data Fig. In WT hepatocytes, EVT0185 more potently inhibited fatty acid and cholesterol synthesis from lactate and acetate than bempedoic acid, with differences amplified in Acly-KO cells, suggesting that additional targets may be involved (Extended Data Fig. Cell-free assays revealed that, unlike bempedoic acid30, EVT0185-CoA inhibited rather than activated AMPKβ1-containing complexes (Extended Data Fig. EVT0185 also suppressed clonogenic survival of human (Hep3B) and mouse (Hepa1-6) HCC cell lines more effectively than bempedoic acid (Extended Data Fig. Given the role of AMPK in pro-survival signalling38 and the compensatory upregulation of ACSS2 upon ACLY inhibition8,9,12, these findings highlight key mechanistic differences between EVT0185 and bempedoic acid and supported continued development of EVT0185 for HCC. Having established mechanisms for EVT0185 inhibition of ACLY, we subsequently examined in vivo efficacy. Compared with vehicle-treated controls, oral administration of EVT0185 reduced the respiratory exchange ratio within 1 h, an effect sustained for up to 5 h, indicating rapid and sustained inhibition of lipogenesis and/or increase in fatty acid oxidation (Extended Data Fig. Oral EVT0185 dosing inhibited hepatic lipogenesis from 14C-glucose in a dose-dependent manner, with a trend at 10 mg kg−1 (−18%) and significant reductions at 30 mg kg−1 (−56%, P = 0.02) and 60 mg kg−1 (−69%, P = 0.005; Extended Data Fig. We then tested EVT0185 in three mouse models, spanning both prevention and treatment of MASH-HCC. 1), mice were randomized by AFP and treated daily with vehicle, EVT0185 (30 or 100 mg kg−1) or bempedoic acid (100 mg kg−1) for 1 month (Extended Data Fig. EVT0185 reduced tumour burden, whereas bempedoic acid had limited efficacy (Fig. Similar to Acly-KO mice, EVT0185 also reduced tumour surface area and lipid accumulation (Extended Data Fig. In the WD-CCl4 prevention model, in which MASH and fibrosis are present by 12 weeks but tumours have not yet formed29,39, EVT0185 nearly eliminated tumour development (Fig. In an 18-month WD model without DEN or CCl4, mice with elevated levels of AFP were randomized to vehicle or EVT0185 (100 mg kg−1) for 4 weeks (Extended Data Fig. EVT0185 again drastically reduced tumour number (Fig. To assess therapeutic potential, we treated WD-CCl4 mice with established tumours starting at 19 weeks. 6i), and in combination with lenvatinib, led to complete tumour remission in 8% of animals (Fig. Finally, combining EVT0185 with anti-PDL1 and VEGFR antibodies in the WD-CCl4 model markedly reduced tumour burden and the proportion of animals with more than 25 tumours, overcoming the limited efficacy of immunotherapy alone (Fig. a, Representative images of livers isolated from mice WD-DEN for 8 months, then treated with vehicle, EVT0185 (30 or 100 mg kg−1) or bempedoic acid (BA-100 mg kg−1) for 1 month. b, Quantification of visible surface tumours from livers of mice treated with vehicle (n = 12), EVT0185 (30 or 100 mg kg−1; n = 12) or bempedoic acid (BA-100 mg kg−1; n = 4). P values were determined by one-way analysis of variance (ANOVA) with Fisher's least significant difference (LSD). c, Representative images of livers isolated from WD-CCl4 mice (12 weeks) treated with vehicle or EVT0185 (100 mg kg−1) for 18 weeks. d, Visible surface tumours on livers. n = 10 mice per group for vehicle or EVT0185 (100 mg kg−1). P value was determined by an unpaired, two-tailed Student's t-test. e, Percent distribution of tumour numbers per group. f, Representative images of livers isolated from mice maintained on WD for 18 months and treated with vehicle or EVT0185 (100 mg kg−1) for 4 weeks. g, Visible number of surface tumours. P value was determined by unpaired, two-tailed Student's t-test. h, Representative images of livers isolated from WD-CCl4 mice treated with vehicle, EVT0185 (50 or 100 mg kg−1), sorafenib (Sora; 15 mg kg−1), lenvatinib (Lenva; 7 mg kg−1), EVT0185 + sorafenib or EVT0185 + lenvatinib for 6 weeks. i, Tumour counts in the liver. n = 12 mice in the treatment groups and n = 10 mice in the vehicle group. j, Percent distribution of tumour numbers. k, Representative images of livers isolated from WD-CCl4 mice treated with isotype control or PDL1 and VEGFR2 antibody (Ab; 200 µg) with or without EVT0185 (100 mg kg−1). l, Visible surface tumours on the livers. P values were determined by one-way ANOVA with Fisher's LSD. m, Percent distribution of tumour numbers per group. To explore mechanisms underlying reduced tumour burden in Acly-KO mice, we performed bulk RNA-seq on liver tumours from WT (n = 9) and Acly-KO (n = 12) mice at two timepoints following AAV injection: a late timepoint (8 weeks; Fig. 2) and an early timepoint (4 weeks), before detectable differences in tumour burden (Extended Data Fig. Including the early timepoint allowed us to assess whether transcriptional changes precede tumour reduction. Acly expression was significantly reduced at both timepoints (Fig. 7c,d), although less so at the later stage, possibly due to clonal selection or infiltration by non-hepatocyte cell types. In contrast to previous reports in hepatocytes26,40, but similar to findings in pancreatic islets41, the expression of Acss2 was not significantly upregulated at either timepoint (Fig. a, Volcano plot of bulk RNA-seq of tumours showing upregulated and downregulated genes at early or late timepoints from Acly-KO (n = 12) versus WT (n = 9) mice. Significance was determined by Wald test with a false discovery-adjusted threshold of 5% as implemented in DESeq2. Horizontal dashed lines demarcate the P value threshold at a 5% false discovery rate (FDR). b, Gene Ontology analysis of selected biological processes involving significantly upregulated genes in tumours from Acly-KO (n = 12) versus WT (n = 9) mice. c, Top 10 Gene Ontology biological processes (GOBP) from clusters identified among significantly upregulated gene sets in tumours from Acly-KO (n = 12) versus WT (n = 9) mice. d, Correlation between B cell populations and Acly expression in tumours from WT (n = 9) and Acly-KO (n = 12) mice. e–p, Spatial transcriptomic analysis of livers from WT and Acly-KO mice and vehicle or EVT0185-treated mice. Cluster analysis representing the number of cell types in the liver and tumour (e,k). The top upregulated pathways in HCC cells from Acly-KO (g) and EVT0185-treated (m) mice. Statistical analysis was performed using Fisher's exact test. Expression level of metabolic genes (h,n). Cxcl13 expression levels in HCC cells (j,p). GC, germinal centre; HSC, hepatic stellate cell; MZ, marginal zone; Tr, regulatory; VSMC, vascular smooth muscle cell. Time-adjusted analysis revealed 367 upregulated and 333 downregulated genes in Acly-KO tumours, with enrichment of antitumour gene sets, including reduced tumour growth size (Extended Data Fig. Metabolically, Acly-KO tumours showed higher citrate and lower succinate levels, corresponding with reduced expression of Acly and succinate-CoA ligase subunits Suclg1 and Sucla2 (Extended Data Fig. Unexpectedly, genes linked to leukocyte proliferation and migration were significantly upregulated in Acly-KO tumours (Fig. Semantic clustering identified nine immune-enriched biological process clusters, including T cell and B cell activation, IFNα or IFNγ responses, and leukocyte adhesion (Fig. As tumour burden was unchanged at 4 weeks (Extended Data Fig. 7b), these early immune-related signatures suggested a potential causal role. Supporting this, gene networks inversely correlated with both Acly expression and tumour burden were enriched for immune processes (Extended Data Fig. Finally, gene expression deconvolution, using TIMER2, revealed a consistent negative correlation between Acly levels and B cell infiltration (Fig. 4d), suggesting a potentially important connection between Acly, immunogenicity and tumour burden. To further investigate tumour–immune interactions, we performed spatial transcriptomics on livers from WD-DEN, WT or Acly-KO mice (Fig. 4e,f) and from WD-CCL4-treated, vehicle-treated or EVT0185-treated (100 mg kg−1) mice (Fig. Gene Ontology enrichment analysis of spatially resolved tumour cells showed increased fatty acid and lipid metabolism in both Acly-KO mice (Fig. Spatial analysis also revealed a selective increase in the number of B cells, but not in the number of T cells, macrophages or natural killer T cells, in tumours from Acly-KO and EVT0185-treated mice (Fig. Using established markers, we found that these B cells were predominantly plasma cells, which mediate antibody production (Fig. Gene Ontology analysis of B cells confirmed enrichment of fatty acid metabolism pathways, supporting their role in plasma cell differentiation (Extended Data Fig. Similar findings were observed in scRNA-seq datasets from EVT0185-treated WD-DEN and WD-CCl4 tumours (Extended Data Fig. The levels of Cxcl13, a key B cell chemoattractant known to be reduced in MASH-HCC17, were elevated in tumours from both Acly-KO and EVT0185-treated mice (Fig. 4j,p), a finding replicated in publicly available RNA-seq data from WT and Acly-KO DEN-induced tumours cultured in vitro10 (GSE223966; Extended Data Fig. These data indicate that genetic or pharmacological inhibition of ACLY in MASH-HCC leads to increases in tumour CXCL13 levels and plasma B cell numbers. To further interrogate the inferred cell populations obtained from the RNA-seq and spatial transcriptomic analysis, we examined similar-sized, neoplastic lesions from the livers of WT and Acly-KO mice using multiplexed ion-beam imaging by time of flight (MIBI-TOF) imaging, which enables single-cell protein profiling42. Consistent with the transcriptomic datasets, Acly-KO mice had increases in the number of B cells at the tumour periphery, with non-significant increases in T cells or other immune cell populations (Fig. 5b–d); findings that were confirmed using immunohistochemical staining of CD19+ B cells (Fig. These changes were accompanied by reductions in the levels of lipid droplet-associated protein (PLIN2) and increases in CXCL13 protein expression (Extended Data Fig. Tertiary lymphoid structures (TLSs) are B cell aggregations that have been associated with improved HCC outcomes43. Detailed analysis of the tumour B cell infiltrates by a pathologist blinded to the treatments revealed that although diffuse and peripheral B cell infiltrations were observed in tumours from both genotypes, B cell aggregations resembling TLSs were predominantly observed in Acly-KO mice (80%; Fig. This pattern was also seen in tumours from WD-CCl4 EVT0185-treated mice without significantly altering T cell counts (Extended Data Fig. Supporting their functional role, scRNA-seq revealed upregulation of complement activation and cytotoxicity pathways in B cells from EVT0185-treated mice (Extended Data Fig. Reduced tumour burden in Acly-KO and EVT0185-treated mice was accompanied by lower Ki67 and higher cleaved caspase 3 levels at the tumour edge (Extended Data Fig. As AAV8-TTR-Cre does not target immune cells (Fig. 1e) and EVT0185 is inactive in them (Extended Data Fig. 3), these findings across multiple platforms (RNA-seq, spatial transcriptomics, scRNA-seq, MIBI and immunohistochemistry) strongly implicate B cells as key mediators of the antitumour effects of ACLY inhibition. a, t-distributed stochastic neighbour embedding (tSNE) plots showing cell phenotypes in WT and Acly-KO mice. b, Representative tumour images: 2 × 2 mm2 montage (25 fields of view (FOVs)), single 400 × 400 μm FOV, and 100 × 100 μm region of interest (ROI). c,d, B cell (c) and T cell (d) counts at the tumour–liver interface. P values were determined by Wilcoxon unpaired, two-sided test. e, Representative image of CD19+ staining (top) and quantification of CD19+ cells per mm2 tumour area in WT (n = 12) and Acly-KO (n = 10) lesions (bottom). P value was determined by unpaired, two-tailed Student's t-test. f, Percentage of mice with B cells present or absent in tumour TLSs. g, Pixel images of lymphoid marker levels with cell neighbourhood and phenotype maps overlaid on cell-boundary masks. The tumour interior is marked by a black star. h, Distribution of cells across cellular neighbourhoods. The whiskers extend to data within 1.5 × IQR. P value was determined by Wilcoxon non-parametric unpaired, two-sided test. j, Schematic of B cell depletion protocol in the Aclyf/f WD-DEN model. P values were determined by one-way ANOVA with Fisher's LSD: P = 7.51 × 10−5 (Acly-KO isotype versus WT isotype). P values were determined by two-tailed Student's t-test as implemented in limma. o, Hepatocyte-specific ACLY expression in MASH-HCC tumour and adjacent tissues (scRNA-seq). P values were determined by χ2 test of independence. p,q, Correlation of ACLY expression with B cell-specific markers (p), and B cell abundance based on cell-type deconvolution methods (q). To determine whether the immunogenic response was important for reducing tumour burden in response to genetic inhibition of Acly, we generated an inducible system for knocking down ACLY expression in the human Hep3B cell line (Extended Data Fig. 10g) for orthotopic implantation in immunodeficient NRG mice. As anticipated, Acly knockdown reduced glucose incorporation into triglycerides and led to compensatory upregulation of acetate incorporation into triglycerides (Extended Data Fig. In addition, inducible knockdown increased fatty acid oxidation (Extended Data Fig. However, despite these anticipated changes in metabolic activity, Acly knockdown did not impair orthotopic tumour growth in immunodeficient mice (Extended Data Fig. This suggests that alterations in lipid metabolism without a functional adaptive immune system was not sufficient for reducing tumour burden. After 28 weeks, AFP levels were assessed, and mice were randomized to four distinct groups based on this parameter before injection with either AAV8-YFP (WT) or AAV8-TTR-Cre (KO), and treatment with an anti-CD20 antibody, which depletes B cells44 (Fig. As anticipated, the anti-CD20 antibody eliminated B220+CD19+ B cells compared with isotype control-injected mice (Extended Data Fig. In WT animals, the depletion of B cells had no effect on the number of visible surface tumours (Fig. Consistent with our previous findings (Fig. 1), Acly-KO mice had large reductions in visible surface tumours when treated with isotype control; however, these effects were eliminated following the depletion of B cells (Fig. These data indicate that the reduction in tumour burden elicited by the inhibition of ACLY in mice with MASH-HCC requires B cells. To assess the relevance of ACLY inhibition in human MASH-HCC, we first analysed the microarray dataset15 (GSE164760), comparing tumour, non-tumour adjacent, MASH, cirrhotic and healthy liver tissue. ACLY was one of 73 genes consistently upregulated in MASH-HCC across all pairwise comparisons (Fig. 12a,b), suggesting that its overexpression in malignant tissue is not confounded by intermediate disease states. This pattern was not observed for other lipogenic enzymes such as ACACA, ACACB, ACSS2 or FASN (Extended Data Fig. Supporting this, snRNA-seq datasets confirmed that increased ACLY expression in malignant hepatocytes versus normal hepatocytes from patients with MASH-HCC (Fig. We next explored the relationship between ACLY and tumour immunogenicity. Similar to our mouse models, ACLY expression inversely correlated with B cell marker expression in human MASH-HCC (Fig. 5p), and cell deconvolution analyses showed higher ACLY levels associated with reduced B cell infiltration (Fig. To understand the pathways underlying ACLY overexpression, we performed differential gene expression modelling followed by weighted gene correlation network analysis (WGCNA) module analysis (Extended Data Fig. CXCL13 expression was significantly reduced in tumours with high ACLY expression and increased in those with low expression, mirroring mouse data (Extended Data Fig. Gene co-expression modules linked to high levels of ACLY were enriched for metabolic pathways—including macromolecule biosynthesis, aerobic respiration and acetylation—whereas they were negatively associated with immune-related features (Extended Data Fig. These findings reinforce a conserved inverse relationship between ACLY expression and tumour immunogenicity across human and mouse MASH-HCC. This study reveals an unexpected immunoregulatory role for ACLY in MASH-HCC. Although ACLY is well known for its metabolic function in suppressing fatty acid and cholesterol synthesis, we have demonstrated that hepatocyte-specific ACLY deletion not only suppresses intratumoural steatosis and proliferation but also enhances antitumour immunity. Genetic inhibition of ACLY increased tumour infiltration by B cells and upregulated the B cell chemoattractant CXCL13, promoting the formation of TLSs, immune niches associated with favourable prognosis in HCC and other solid tumours17,43,45,46,47. Crucially, B cell depletion abolished the antitumour effect of ACLY inhibition, confirming their essential role. As these tumour-suppressive effects in the Acly KO occurred independently of changes in liver histology, and the TTR-Cre promoter used to induce deletion does not affect immune cells26, these studies provide surprising findings about how inhibiting ACLY not only suppresses tumour proliferation but also enhances immunogenicity. Our findings indicating a critical role for B cells are consistent with emerging studies involving mouse models and patients with HCC that have shown favourable outcomes linked to increased tumoural B cell infiltration and TLSs48,49,50,51,52,53,54. These effects were associated with increases in the B cell chemokine CXCL13, which has been shown to be important for predicting response rates to immunotherapy in MASH-HCC. The precise mechanisms by which ACLY inhibition increases CXCL13 and TLS formation remain undefined; however, future studies investigating whether reductions in the levels of tumour metabolites such as fatty acids and succinate or increase in the levels of citrate directly impact tumoural B cell recruitment and formation of TLSs or, alternatively, whether there are reductions in the level of CXCL13 promoter acetylation will be important. We developed and characterized EVT0185, a novel, orally available small-molecule ACLY inhibitor, which phenocopied the immune and antitumour effects of genetic ACLY deletion. EVT0185 is activated in hepatocytes via SLC27A2-dependent conversion to its CoA-thioester form, allowing liver-specific action while sparing non-hepatic tissues. In multiple MASH-HCC mouse models, EVT0185 reduced tumour burden and synergized with current standards of care, including lenvatinib or anti-PDL1 and VEGFR antibodies. Structural cryo-EM analysis demonstrated that EVT0185-CoA binds directly to the ACLY CoA-binding pocket, supporting a clear mechanistic basis for its inhibitory action. Like the Acly-KO model, EVT0185 treatment was associated with increased plasma cell differentiation and complement-mediated cytotoxic pathways. Although our data strongly support ACLY as the principal target of EVT0185 in driving tumour immunogenicity, it remains possible that other targets may be important as we found that this compound also reduced acetate incorporation into fatty acids and sterols, an effect that was independent of ACLY. However, analysis of MASH-HCC tumours in humans mirrored our findings in Acly-KO mice, with ACLY expression but not ACSS2, ACC1/ACC2 or FAS being inversely correlated with B cell infiltration. Together, these data in both mice and humans strongly suggest that inhibition of ACLY is the primary driver of the enhanced tumour immunogenicity, mediated by EVT0185. EVT0185 was inactive in cultured immune cell populations, consistent with their low expression of Slc27a2, as activated T cells and macrophages require ACLY to elicit effector function. Together, these data position EVT0185 as a promising candidate for targeting ACLY in MASH-HCC. In conclusion, these studies have shown that genetic inhibition of ACLY in MASH-driven mouse models of HCC promotes the infiltration of tumour-infiltrating B cells and that this is critical for reducing tumour burden. Similar effects are also observed with EVT0185, a small-molecule competitive inhibitor of ACLY. These findings expand the role of tumour ACLY from a regulator of metabolism and proliferation to a mediator of antitumour immunity and suggest that targeting of tumour metabolism may be a largely untapped mechanism to improve immunotherapy response rates. As reductions in ACLY in clinical datasets are also associated with lower tumour burden and increased B cell infiltration, further studies investigating whether pharmacological inhibition of ACLY also increases immunosurveillance and reduces tumour burden in clinical populations will be important. EVT0185 and EVT0185-CoA were synthesized at Symeres (www.symeres.com); experimental methods and compound characterization are presented in the Supplementary Information. Animal experiments were carried out using the guidelines approved by the Animal Research Ethics Board at McMaster University (Steinberg Laboratory Animal Utilization Protocol #16-12-42, 21-01-04) or the Institutional Animal Care and Use Committee (IACUC) at Icahn School of Medicine at Mount Sinai (IACUC approval# PROTO202100080). The tumour development was monitored based on the established progression timelines for mouse HCC models. The welfare of the animals was evaluated through observation of visible masses, impaired mobility and marked weight loss. However, no specific tumour volume end points were defined for intrahepatic tumours. All experimental procedures did not exceed the limits set by the Animal Utilization Protocol or IACUC. C57BL/6J male mice were housed within the McMaster University Central Animal Facility. After weaning, mice were housed with littermates and fed a normal chow diet (8640 22/5, Teklad). At 10 weeks of age, mice were maintained on chow diet (control) or switched to a high-fat and high-fructose diet (WD), which consisted of 40% kcal fat (mostly palm oil), 20% kcal fructose and 0.02% wt cholesterol (D19101102, Research Diets) and housed at thermoneutral conditions (26–29 oC)26. Throughout the experiments, mice were housed in ventilated cage racks with ad libitum access to food and water. An automatic timing device was used to maintain an alternating 12-h cycle of light and dark. For the WD-DEN Acly-KO mouse model (Figs. 1 and 5), at 2 weeks of age, C57BL-6/Aclyfl/fl mice (obtained from JAX) were injected with DEN at 25 mg kg−1 body weight. After weaning, mice were housed with littermates and fed a normal chow diet (8640 22/5, Teklad). At 10 weeks of age, mice were switched to a high-fat and high-fructose diet (WD): 40% kcal fat (mostly palm oil), 20% kcal fructose and 0.02% cholesterol (D19101102, Research Diets) and housed at thermoneutral conditions. When the mice reached 7.5 months, they were bled from the tail vein, and plasma AFP levels were determined. Animals were grouped based on their AFP levels, so there were no differences between groups, and were then injected with AAV intravenously (AAV8-YFP or AAV8-TTR-Cre) as previously described26. After 4 (early timepoint) or 8 (late timepoint) weeks following AAV injection, mice were anaesthetized using ketamine–xylazine, and terminal blood was collected by cardiac puncture. For the B cell depletion experiments, AFP levels were assessed as described above after 7.5 months, and mice were randomized before being injected with AAV intravenously (AAV8-YFP or AAV8-TTR-Cre) and intraperitoneally injected with 250 µg of isotype control (400566, BioLegend) or anti-CD20 antibody (152104, BioLegend) with tissues collected 8 weeks later. For the WD-DEN EVT0185 and bempedoic acid study (Fig. 3), male mice housed within the McMaster University Central Animal Facility were used for these experiments. C57BL/6J mice were injected with DEN at 2 weeks of age and followed the above-mentioned protocol. Animals were grouped based on their AFP levels and orally gavaged with a single daily dose of vehicle (1.5% CMC and 0.2% Tween-20), EVT0185-30 or 100 mg kg−1 or bempedoic acid-100 mg kg−1. After 4 weeks of treatment, mice were anaesthetized using ketamine–xylazine, and terminal blood and tissues were collected. Male C57BL/6J mice obtained from Jackson Laboratory were used for these experiments. Mice at 6–8 weeks of age were fed ad libitum with WD (Envigo; TD.120528, Teklad Custom Research Diet) and maintained on sugar water (fructose (23.1 g l−1) + glucose (18.9 g l−1)) for the duration of the study. Animals received weekly intraperitoneal injections of CCl4 (0.2 μl, 100% CCl4 per gram of body weight) throughout the study as described below. The prevention study was conducted after 12 weeks of CCl4 injections + WD at McMaster University with mice orally gavaged with vehicle or EVT0185 (100 mg kg−1) for 18 weeks. The combination treatment study was conducted after 19 weeks of CCl4 injection + WD at Mont Sinai with mice orally gavaged with EVT0185 (50 or 100 mg kg−1), sorafenib p-tosylate (sorafenib; at 15 mg kg−1; S-8502, LC Laboratories), lenvatinib mesylate (lenvatinib; 7 mg kg−1; 29832, LC Laboratories) or sorafenib + EVT0185 (100 mg kg−1) or lenvatinib + EVT0185 (100 mg kg−1). The combination study with immunotherapy was conducted after 24 weeks of CCl4 injections + WD at McMaster University with mice being injected with InVivoPlus anti-mouse PDL1 (B7-81; BP0101), and InVivoPlus anti-mouse VEGFR2 (DC101; BP0060) alone or in combination with EVT0185 (100 mg kg−1). Isotype controls received InVivoPlus rat IgG1 isotype control, horseradish peroxidase (HRP; BP0088) and InVivoPlus rat IgG2b isotype control, anti-keyhole limpet haemocyanin (LTF-2; BP0090) as a single intraperitoneal injection containing 200 µg of each antibody in a total volume of 100 µl. Antibodies were prepared in InVivoPure pH 7.0 Dilution Buffer (IP0070). After 6 weeks of treatment, animals were anaesthetized using ketamine–xylazine, and terminal blood was collected by cardiac puncture. Male C57BL/6J mice obtained from Jackson Laboratory were used for these experiments. Mice at 6 weeks of age were fed ad libitum with WD, which consisted of 40% kcal fat (mostly palm oil), 20% kcal fructose and 0.02% wt cholesterol (D19101102, Research Diets). Throughout the experiments, mice were housed at thermoneutral conditions (26–29 °C). An automatic timing device was used to maintain an alternating 12-h cycle of light and dark. After 18 months, AFP-positive mice were divided into two groups and received a single daily dose via gavage of either vehicle (1.5% CMC and 0.2% Tween-20) or EVT0185 (100 mg kg−1). After 4 weeks of treatment, mice were anaesthetized using ketamine–xylazine, and terminal blood and tissues were collected. A portion of tumour-free liver (approximately 200 mg, caudate lobe) and tumours were removed, flash-frozen in liquid nitrogen and stored for molecular analysis. The remainder of the liver was placed in 10% formalin, fixed for 48 h and stored in 70% ethanol until it was embedded in paraffin, sectioned and mounted on slides for histological analysis. 1 day before surgery, male NOD-Rag1nullIL2rgnull (NRG) received intraperitoneal injections of 100 mg kg−1 cyclophosphamide and subcutaneous injections of 5 mg kg−1 carprofen. One hour before surgery, Hep3B-Luc cells were resuspended on ice in a 1:1 dilution of phenol red-free Matrigel (356237, Corning) and cold PBS (10010023, Gibco), aliquoted in 50 µl volumes at a cell density of 2 × 107 cells per millilitre. During surgery, mice were anaesthetized with 2% isoflurane (1001936040, Baxter), hair removed with 3-in-1 hair removal lotion (061700222611, Nair) and placed in a supine position on a heating pad, with nose fitted in an anaesthesia nose cone. Next, the left lobe of the liver was withdrawn, stabilized and tumour cells were injected into the liver parenchyma while applying pressure using a cotton tipped applicator (4305, Dynarex) to enable haemostasis. Following injection, the liver was returned into the peritoneal cavity and the peritoneum was closed with multiple single interrupted sutures using 4-0 Vicryl sutures (J743D, Ethicon). The skin was closed using 9-mm wound clips (RS-9262, Roboz). Finally, mice received post-surgical subcutaneous injections of 5 mg kg−1 carprofen and were placed in cages on a heating pad containing diet recovery gel (72-06-5022, Clear H2O) and monitored until they were ambulatory. Wound clips were removed on day 6, and mice were imaged for tumour progression on day 7 and weekly thereafter. Induction of in vivo short hairpin RNA (shRNA) expression was achieved via supplementing drinking water with 2 mg ml−1 doxycycline (Dox; D9891, Sigma-Aldrich) and 5% sucrose starting on day 7 post-surgery and every 2–3 days thereafter. Tumour progression was monitored via bioluminescence imaging using the IVIS Spectrum In Vivo Imaging System (124262, Perkin Elmer). IVIS imaging was performed on the auto-exposure setting at 10-min post-injection. Primary mouse hepatocytes were isolated from C57BL/6J mice as previously described55 and seeded in white opaque 96-well plates, and the next day were serum starved for 2 h followed by the treatment with EVT compounds (0, 0.1, 0.3, 0.5, 1, 3, 10, 30, 60 and 100 µM) in the presence of 14C-acetate (1 µCi ml−1) in a concentration-dependent manner. After 4 h of treatment, plates were washed two times with 1× PBS and 100 µl microscint fluid (Microscint O, part #601361) was added to each well. Plates were wrapped with aluminium foil and were shaken at 250 rpm for 2 h. After 2 h of shaking, 14C incorporation into the lipid fraction was determined by liquid scintillation counting using a TopCount NXT Microplate Scintillation and Luminescence Counter (Perkin Elmer). To compare the effect of bempedoic acid and EVT0185 in WT and Acly-KO mice, Aclyfl/fl mice were injected with hepatocyte-targeted AAV8-TTR expressing either YFP (WT) or Cre recombinase (Acly KO) via the tail vein to induce Acly genetic deletion as previously described26. After 2 weeks, hepatocytes from each mouse were collected and de novo lipogenesis (DNL) was assessed as previously described26. In brief, the cells were resuspended in complete William's Media E and plated in six-well plates and allowed to adhere for 4 h. Cells were then washed with warm PBS and switched to fresh fetal bovine serum (FBS)-free (serum-free) William's Media E for 2 h. Following a 2-h serum starvation, cells were washed in warm PBS, then treated with serum-free media supplemented with 1 µCi ml−1 [14C]-lactate (NEC599050UC, Perkin-Elmer) or 0.5 µCi ml−1 [14C]-acetate (NEC553050UC, Perkin-Elmer) for 18 h in the presence of 1 µM of EVT0185 or bempedoic acid both of which were dissolved in DMSO. After cooling, a 1:2 mixture of H2O:n-hexane was added to each sample, vortexed and centrifuged for 5 min (1,500 rpm). The top phase was transferred to a scintillation vial combined with Ultima Gold (NC0169557, Revvity) for counting of the sterol fraction. 2 N HCl and petroleum ether were added to the remaining bottom phase, which was vortexed and centrifuged for 5 min (1,500 rpm) at room temperature. The top phase was transferred to a scintillation vial for counting of the fatty acid fraction using a TopCount NXT Microplate Scintillation and Luminescence Counter (Perkin Elmer). Male C57BL/6J mice (8 weeks of age) were obtained from Jackson Laboratory and were fed a normal chow diet upon arrival. At approximately 10 weeks of age, mice were given a diet containing high-fat and high-fructose (rodent diet with 40% kcal fat (mostly palm oil), 20% kcal fructose and 0.02% cholesterol (D19101102, Research Diets) and housed at thermoneutral conditions (26–29 °C). After 7–8 months, the mice were divided into four groups and received a single daily dose via gavage of either vehicle (1.5% CMC and 0.2% Tween-20) or EVT0185 at a dose of 10, 30 or 60 mg kg−1 for 7 days. On the morning of day 8, animals received a final dose, and after 1 h, 14C glucose (PerkinElmer) was administered at a concentration of 12 μCi per mouse in a volume of 0.1 ml in 0.9% saline (intraperitoneal). One hour after 14C glucose was given, animals were anaesthetized by intraperitoneal injection of ketamine–xylazine (150 mg and 12.4 mg kg−1, respectively). Liver tissue was chipped on dry ice and the weight of the chip was weight recorded (30–50 mg of tissue). Liver tissue was homogenized in 1 ml of 2:1 chloroform:methanol using a bead homogenizer at 5,000 rpm for 2 × 12 s. Samples were incubated with gentle shaking at 4 °C for 2 h, vortexed for 2 × 12 s, and then centrifuged at 7,000 rpm for 10 min at 4 °C. The supernatant was transferred to a 1.5-ml Eppendorf tube and 200 μl of 0.9% saline was added. Next, 200 μl of the lower organic phase was removed and added to 5 ml of scintillation fluid. The number of disintegrations per minute was determined over 5 min and normalized to the amount of liver tissue. Of plasma obtained at termination, 5–10 µl was also counted and lipid per gram of tissue counts were normalized to plasma counts. Male C57BL/6J mice were fasted overnight (approximately 12 h) until food and 30% fructose water were again made accessible the next day at 7:30. Animals were allowed to feed ad libitum for approximately 2 h. The respiratory exchange ratio (RER) was monitored in metabolic cages using Oxymax/CLAMS (Comprehensive Laboratory Animal Monitoring System) equipment and software (Columbus Instruments). The calorimetry system consists of metabolic cages each equipped with water bottles and food hoppers connected to monitor food intake. All animals had ad libitum access to standard rodent chow and water throughout the study. Those animals with RER > 1 were administered with or without EVT0185 at a dose of 100 mg kg−1 via gavage. Animals remained in the metabolic cages and RER was monitored for 24 h. Plasma AFP levels were determined using a mouse AFP Duoset ELISA kit (DY5369-05, R&D Systems) following the manufacturer's protocol. In brief, 100 µl sample (1:1,000 in reagent diluent) was added to each well of a 96-well plate coated with mouse AFP capture antibody. Next, the plates were washed with wash buffer and 100 µl detection antibody was added to each well, followed by the Streptavidin HRP and substrate solution. The reaction was stopped by stop solution and the optical density of each well was measured in a microplate reader set to 450 nm. Following fixation with 10% neutral-buffered formalin for 48 h, the cassettes with a medial lobe of the liver were switched to 70% ethanol. Livers were then processed, paraffin-embedded, serially sectioned and stained with haematoxylin and eosin (H&E), antibodies targeted to CD3+ and CD19+ cells by the McMaster Immunology Research Centre Histology Core Facility. Images were taken using a Nikon 90i Eclipse upright microscope at indicated magnifications. Liver histology scores were assigned to liver sections by a pathologist who was blinded to the treatment conditions. Steatosis, inflammation and hepatocellular ballooning scores were assigned from H&E-stained liver sections as described by Kleiner et al.56. The surface area of tumour in the liver was quantified using qupath software from H&E-stained slides. The quantification of tumour CD3 and CD19-positive staining was completed from the same histological sections using HALO software by an analyst who was blinded to the treatment conditions. The H-score system was used for ACLY immunohistochemistry evaluation on tumoural and non-tumoural hepatic tissues. This was determined by the multiplication of the percentage of cells with cytoplasmic staining intensity ordinal value (0 for no, 1 for weak, 2 for medium and 3 for strong staining), which ranges from 0 to 300 possible values. The whole tumoural areas of each liver were evaluated in ×10 magnification as well as non-tumoural areas, and H-scores were calculated separately. Hepatic lesions were characterized into neoplastic and non-neoplastic proliferative lesions by a histopathologist according to the International Harmonization of Nomenclature and Diagnostic Criteria for Lesions in Rats and Mice (INHAND) guidelines57. All lesions had a hepatocellular phenotype. Non-neoplastic proliferative lesions, including foci of altered hepatocytes (FAHs), are referred to as premalignant lesions because they are composed of phenotypically altered cells demonstrating a higher risk of progression to malignancy than normal cells58,59. FAHs were defined as sharply demarcated circular, oval or irregularly shaped foci with preserved hepatic architecture and without compression of surrounding liver parenchyma. The basophilic-type FAH was the dominant type observed in this study, composed of slightly smaller hepatocytes with basophilic cytoplasm and slightly enlarged pleomorphic nuclei with prominent nucleoli, showing increased mitotic activity compared with background hepatic parenchyma. Scattered eosinophilic FAHs were also identified, composed of enlarged, polygonal hepatocytes with distinctly granular and pale pink, intensely eosinophilic or ground-glass appearance, as well as few clear cells and mixed cell types of FAHs. Neoplastic lesions as HCCs were characterized by loss of normal architecture (thickened hepatocytes plates), revealing mostly solid patterns as well as focal pseudoglandular and trabecular (micro or macro) patterns, including unpaired arteries or arterioles, consist of polygonal cells with more nuclear atypia (irregular nuclear membrane, pleomorphism, increased nuclear-to-cytoplasmic ratio, multinucleation and prominent nuclei), higher mitotic rate and cytoplasmic alterations include Mallory–Denk bodies and hyaline bodies57. Frozen tumour tissue (approximately 30 mg) was homogenized with Buffer RLT (RNeasy Mini Kit, 74106, Qiagen), and RNA isolation was performed using the RNeasy kit (74106, Qiagen) following the manufacturer's instructions. Sample quality was assessed using the Agilent 2100 Bioanalyzer G2938C with the Aligent RNA 6000 Nano Kit (Agilent). Only samples passing the RNA integrity number threshold of 7.0 were used for sequencing. mRNA was enriched using the NEBNext Poly(A) mRNA Isolation Module (NEB) followed by library preparation using the NEBNext Ultra II Directional RNA library kit (NEB). Next-generation sequencing was conducted at the McMaster Genomics Facility, Farncombe Institute, McMaster University, using Illumina HiSeq 1500 (Illumina). Samples were randomly distributed across lanes of a HiSeq Rapid v2 flow cell to eliminate lane-specific effects, and single-end 50-bp reads were generated at 12.5 million clusters per sample. Sequence quality was assessed using FastQC followed by the removal of low-quality reads and adapter sequences using Cutadapt. Quantification of reads was performed using feature Counts. We obtained raw RNA-seq files for tumour samples derived from control-DEN and WD-CCl4 models from PRJNA488497 (ref. Sequencing files were processed using the same method as described for in-house samples. Count-level expression data were adjusted for surrogate batch effect across all samples using ComBat-Seq as implemented in the sva package in R. Subsequently, adjusted expression data were normalized to transcript per million. Molecular classification of Hoshida subtypes for human sample and mouse models were performed using the nearest template prediction module as implemented in the Gene Pattern platform online. Pairwise correlation between each mouse sample and each human sample were averaged on a per-model basis for mouse models, and on a per-subtype basis for human samples. Consistent with a previous report15, we validated the unique transcriptomic enrichment of the Hoshida S1 subtype60 and depletion of the S2 subtype among patients with MASH-HCC relative to HCC of mixed aetiology shown in Extended Data Fig. Microarray CEL files were downloaded from GSE164760. Normalization was performed using the robust multiarray average expression method implemented in Affy (v1.74.0). The HGU219 custom chip definition file was obtained from the Brainarray Microarray Lab (http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/) and used for mapping probes to gene identifiers in a one-to-one format. For RNA-seq, overall and timepoint-specific differential gene expression results were obtained using the DESeq2 package (v1.36.0) by modelling the additive and interaction effect of timepoint and Acly KO. To obtain the overall result, we adjusted association of gene expression with Acly KO by a binary indicator of experimental timepoint (that is, early or late) as a confounding variable using an additive model (that is, time + Acly KO). Timepoint-specific results were obtained by modelling gene expression association using only samples belonging to either early or late timepoints. Finally, we modelled the modifying effect of timepoint on differential gene expression by using an interaction model (that is, time × Acly KO). We used a false discovery-adjusted threshold of 5% to determine gene significance. Individual sample read counts underwent variance stabilizing transformation. Gene ranks were obtained for downstream gene set enrichment analysis by performing natural logarithmic transformation of P value and applying the numerical sign of log2 fold change based on DESeq2 result. Differential expression analysis for microarray data was performed using limma (v3.52.3). Comparisons between tissue were performed by specifying all combinations of contrast involving MASH-HCC. Comparison using ACLY expression level as an independent variable was performed by first using a linear model to identify overall fold change direction followed by a natural cubic splines model with 5 degrees of freedom to identify gene significance. WGCNA (v1.71) in mouse tumour tissues was restricted to genes with greater than or equal to 15 counts in 75% of samples. A soft power of 14 was selected, resulting in a scale-free topology fit of 0.875 WGCNA for human MASH-HCC samples utilized a soft power of 12, resulting in a scale-free topology fit of 0.812. We constructed signed module networks for both mouse and human datasets using the robust correlation method and restricted the number of excluded outliers to 0.10 as recommended. Modules below the hierarchical clustering height of 0.25 were merged. Significant module genes and associated traits were identified using Pearson correlation and asymptotic P value for correlation. Over-representation and gene set enrichment analysis were conducted using clusterProfiler (v4.4.4). For over-representation analyses, we further calculated fold change as the ratio of a gene-to-background ratio. We used an adjusted P value of 0.05 as the threshold for significance. To identify clusters of similar Gene Ontology terms, we utilized the semantic similarity and binary cut method implemented in simplifyEnrichment (v1.6.1). Mouse cell-type deconvolution was performed using transcript per million normalized gene expression data and mMCP-counter (v1.1.0). Validation of B cell enrichment was performed using methods implemented in the online TIMER2.0 tool61. For human cell-type deconvolution, we used TIMER2.0 in addition to the ESTIMATE package (v1.0.13). We filtered cell types for those containing greater than 0 count in at least 75% of samples. Association with gene expression was assessed using univariate linear regression. Significance was determined using a false discovery-adjusted threshold of 5%. Liver tissues were collected and fixed with 10% neutral-buffered formalin for 36–48 h. After fixation, the samples were immersed in a 70% alcohol solution. The liver tissues were then processed, paraffin-embedded and sectioned. RNA quality of formalin-fixed, paraffin-embedded tissues was tested by TapeStation system with DV200 > 30% (DV200 (distribution value 200) refers to the percentage of RNA fragments that are greater than 200 nucleotides in length). Slides with 5 µm of tissue sections were processed with deparaffinization, H&E staining, imaging, decrosslinking, hybridization, ligation, probe extension, pre-amplification and probe-based library construction to generate gene expression libraries for each tissue for sequencing. Images were taken using the Nikon 90i Eclipse upright microscope. RNA-seq was performed using the Illumina NextSeq 2000 (P2 Flow cell, 2 × 50 bp configuration) system. Sequence data from formalin-fixed, paraffin-embedded tissues were analysed using the Space Ranger count pipeline from 10X Genomics for raw data processing and quality control. Downstream analyses included quality control, normalization, dimensional reduction and clustering, variable gene selection, spatially variable feature detection, annotation, differential expression and integration with multiple samples using Seurat62. Spatial transcriptomics data analyses were performed using the Linux system, R, RStudio software and Python programming language. Frozen mouse liver samples (WD-DEN and WD-CCl4 HCC models) were minced, fixed and dissociated following the 10X Genomics ‘tissue fixation and dissociation for chromium-fixed RNA profiling' protocol (document number CG000553, Rev B). Probe hybridization, library construction and sequencing followed the 10X Genomics ‘chromium-fixed RNA profiling reagent kits for multiplexed Samples' (document number CG000527, Rev D). Sequencing was performed using the Illumina Novaseq 6000 sequencing system (S4, 2 × 150, 2–2.5 billion reads, targeting 5,000 cells per sample at a depth of 25,000 read pairs per cell). Demultiplexing was performed using Cell Ranger via the 10X Genomics Cloud Analysis platform. Demultiplexed samples were integrated by condition and processed in R using Seurat v5. Mouse–human correlation analysis was performed in R. Mouse gene symbols were converted to their 1-to-1 orthologous human EnsemblIDs using the biomaRt package. Normalized count data were filtered for each dataset to include only genes common between all datasets. Each dataset was then filtered to include only cells expressing specific marker genes (those encoding CD45, AFP or GPC3, respectively). Pairwise Pearson correlation values were calculated between the human MASH-HCC data64 and WD-CCl4 mouse data15, using normalized gene expression in the relevant cell type. Results were visualized with the pheatmap package. Human gene-specific mean expression data were generated with Seurat (v5) in R. Wilcoxon rank-sum tests were used to assess the significance of differential expression of ACLY and SLC27A2 in different human conditions, relative to the MASH-HCC condition (P < 0.05 considered significant). Single-nucleus sequencing of cells derived from healthy, MASLD, MASH-HCC tumour and tumour-adjacent tissue were obtained from GSE174748 and GSE189175 (ref. 64) and processed using the Seurat package (v5.1.0) in R. We applied sctransform to normalize RNA read counts using the default selection of 3,000 genes as the number of variable features. Transformed counts underwent dimension reduction using principal component analysis. The 30 most variable components were then selected for uniform manifold approximation and projection. Clusters were then identified using the Louvain clustering method at a resolution of 0.2. Cell types were identified using the automated cell-type assignment pipeline as implemented in the sc-type package in R65. We utilized the custom gene set containing liver-specific cell-type expression markers provided by the GepLiver resource66. SCT normalized counts were used for this analysis. Low-confidence clusters based on the default score threshold was assigned with the ‘unknown' cell type. The proportion of cells expressing ACLY was determined for hepatocytes and malignant hepatocytes and compared between tumour and non-tumour adjacent tissues using χ2 test of independence. Tissues were sectioned (4-µm-thick sections) onto gold-coated MIBI slides. Antigen retrieval in a pH 9 Target Antigen Retrieval Solution (DAKO Agilent) occurred at 125 °C for 40 min in a Decloaking Chamber (Biocare Medical). After cooling to room temperature, slides were washed twice in TBS-T (Ionpath). The tissue was incubated in a blocking buffer consisting of 3% normal donkey serum (Jackson ImmunoResearch) in TBS-T for 20 min. The antibody panel consisting of metal-tagged antibodies supplied by Ionpath was diluted in blocking buffer and added to the tissue for overnight incubation at 4 °C in a moisture chamber. Slides were stored in a desiccator before MIBIscope analysis. Spectral images of stained liver lesions were collected using an Ionpath MIBIscope with multiplexed ion-beam imaging technology. Xenon primary ions from a Hyperion ion gun rastered across the slide to sputter stained tissue, which was detected by mass spectrometry by time of flight to reconstruct the spectral images, on a pixel-by-pixel basis, of each channel consisting of a single stained antibody. A more detailed description of the multiplexed ion-beam imaging technology has been previously described42. Spectral images of 400 × 400-µm regions of interest (ROIs) were selected with the assistance of a pathologist and the images were collected using an Ionpath MIBIscope with multiplexed ion-beam imaging technology. Multiplexed raw image sets were denoised and aggregate filtered using Ionpath's MIBI/O software using the default correction settings. Processed image data were stored as TIFF files for further data processing and analysis. Whole-cell segmentation was performed with the input of the nuclear stained and the membrane-stained marker channels using Mesmer67, and the segmentation mask images were stored as TIFF files for further analysis in R Studio. Single-cell data were extracted for all cell objects defined by the segmentation masks using a custom R script and packages described in Supplementary Table 5 (refs. Marker intensities were asinh transformed by a cofactor of 1. To classify cell types based on their marker expression levels, the FlowSOM clustering algorithm was used with the Bioconductor ‘FlowSOM' R package72. By inspecting a heatmap displaying normalized individual marker intensities, we annotated each of the 100 clusters into ten cell types. The tumour ROIs were selected based on the presence or high abundance of tumour-infiltrating lymphocytes. The tumour and non-tumour regions were omitted from analysis. We used the imcRtools package to detect cellular neighbourhoods. A spatial graph was constructed by detecting the k-nearest neighbours in 2D per cell73. First, for each cell, we used the aggregateNeigbors function to compute the fraction of cells of a certain cell type among its neighbours. The choice of six cellular neighbourhoods was determined by a parameter sweep across varying k values. Morphometric lipid droplet analysis was adapted from the previously published article74. In brief, H&E sections of tumour-bearing livers were processed using ImageJ for the average size of lipid droplets and average lipid area. Original H&E-stained slides were converted to an 8-bit greyscale image, which was then black and white inverted. These images were used for lipid droplet analysis using the following plugin: run (‘Analyze Particles…', ‘size = 50–20,000', circularity = 0.50–1.00 display summarize'), which defined the upper and lower limits to a lipid droplet area (50–20,000 µm2)75 to exclude irregular-shaped structures (lipid droplets are more spherical). Quantitation of free fatty acid levels was carried out using ultra-high-performance liquid chromatography–multiple reaction monitoring–mass spectrometry (UPLCMRM–MS) method as previously described76. In brief, a stock solution of fatty acids was prepared with their standard substances in LC–MS grade isopropanol:methanol (1:1). This solution was serially diluted 1:4 (v/v) to make 10 working standard solutions with the same solvent. A volume of the extractant equivalent to 2 mg of raw tissue was dried under a nitrogen gas flow. The residue was resuspended in 20 µl of LC–MS grade isopropanol:methanol (1:1). The mixtures were allowed to react at 40 °C for 60 min. Of aliquots of the resultant solutions, 6 µl was injected onto a C18 column (2.1 × 100 mm, 2.5 µm) to run UPLCMRM–MS with (−) ion detection on an Agilent 1290 UHPLC system coupled to a Sciex 4000 QTRAP MS instrument, with the use of 1 mM ammonium acetate in water (A) and acetonitrile:isopropanol (1:1, v/v) as the mobile phase for binary-solvent gradient elution in a range of 30–100% B over 16 min at 0.4 ml min−1 and 60 °C. The tumour tissues isolated were snap-frozen and stored in liquid nitrogen. Around 20–30 mg tissues from each group were chipped and homogenized (with 80% methanol)77. The supernatant was dried for gas chromatography (GC)–MS sample preparation78. Dried samples were further derivatized in 50 μl of 10 mg ml−1 methoxamine hydrochloride for 60 min at 42 °C, followed by 100 μl of N-tert-butyldimethylsilyl-N-methyl trifluoroacetamide for 90 min at 72 °C. Next, TCA metabolites were separated by Agilent 7890B gas chromatograph with HP-5ms Ultra Inert GC column (30 m, 0.25 mm, 0.25 µm, 7 inch; 19091S-433UI, Agilent). Metabolites were then analysed by Agilent 5977B mass-selective detector, using full-scan mode. Total metabolite abundances were measured as the area of the total ion counts normalized to protein content. Liver tumour CXCL13 protein levels were determined using the mouse CXCL13/BLC/BCA-1 Quantikine ELISA Kit (MCX130, R&D Systems) following the manufacturer's instructions. In brief, 50 µl of tissue homogenate was added to each well of a 96-well plate coated with a monoclonal antibody specific for mouse BLC or BCA-1 for 2 h. Next, the plates were washed with wash buffer and 100 µl mouse BLC–BCA-1 conjugate was added to each well for the next 2 h followed by the addition of substrate and stop solution. The optical density was measured using a microplate reader set at 450 nm. Paraffin-embedded, 5-µm sections of tumour-bearing livers were used for immunofluorescence. Slides were deparaffinized, rehydrated and subjected to antigen retrieval (10 mM sodium citrate buffer). Slides were blocked in BSA and subsequently incubated with primary antibody overnight at a concentration of 1:500 CCP3 (Cell Signaling Technology) and Ki67 (Thermo Fisher). Fluorophore-conjugated secondary antibodies were then applied to tissues and mounted with coverslips. Representative images were acquired and analysed using ImageJ analysis. All cell-free assays were conducted at Reaction Biology. Human ACC1 (50202), ACC2 (50201) and ACLY (50255) were obtained from BPS Biosciences. Recombinant protein of human ACSS2, transcript variant 1 (accession no. NM_018677) tagged with MYC–DDK was obtained from Origene (TP304260). ADP-Glo (V9101, Promega) was used to measure ACLY, ACC1 and ACC2 enzyme activity following the manufacturer's instructions. The assay was performed in two steps: (1) after the specific enzyme-mediated reaction that utilizes ATP and produces ADP, ADP-Glo reagent was added to terminate the kinase reaction and depleted the remaining ATP; and (2) kinase detection reagent was added to convert ADP to ATP and allow the newly synthesized ATP to be measured using a luciferase–luciferin reaction. ACSS2 activity was assessed using the Bellbrook Transcreener AMP2/GMP2 Assay kit, a far-red competitive fluorescence polarization assay. AMP produced in the reaction was detected by AMP/GMP Alexa fluor 633 tracer bound to an AMP2/GMP2 antibody. AMP displaced the tracer allowing free rotation of the fluorophore and decreased fluorescence polarization, which was quantified using an appropriate multimode plate reader. Human AMPKα1–AMPKβ1–AMPKγ1 heterotrimers were co-incubated with the AMPK substrate peptide at 20 μM, AMP at 125 μM and ATP at 10 μM using the HotSpot assay. Human recombinant ACLY was obtained from Sino Biological (11769-H07B). P53396) was expressed in insect cells using baculovirus expression host, with a poly histidine tag at the N terminus. The ACLY ADP-Glo assays were performed at room temperature. Enzyme and sodium citrate were prepared in reaction buffer at three times the reaction concentration and 5 µl was added to Corning 3572 reaction plate wells. Of compound concentrations, 100× were prepared by serial dilution in 100% DMSO. Then, the compound was transferred to the reaction plate, which contains enzyme and sodium citrate, using acoustic technology. The compound and enzyme were pre-incubated for 15 min at room temperature. Then, 5 µl of 3× coenzyme A was delivered to the reaction well, followed by 5 µl of 3× ATP to start the reaction. The assay was incubated for 2 h at room temperature. For the background wells, all assay components were added without enzyme. After a 2-h reaction, ADP-Glo reagent was added to stop the reaction and incubated for 40 min at room temperature following detection kit protocol. Then, ADP-Glo detection was added and incubated for 30 min at room temperature following the detection kit protocol. The luminescence signal was measured after 30 min of incubation with ADP-Glo detection. HEK293 cell line was obtained from American Type Culture Collection (ATCC) and grown in Dulbecco's modified eagle medium (319-005-CL, Wisent Bioproducts) supplemented with 10% FBS (098150, Wisent Bioproducts) in a humidified incubator at 37 °C and 5% CO2. Transfected cell lines were supplemented with G418 (A1720, Sigma-Aldrich) during maintenance at a concentration of 800 µg ml−1. Transfected SLC27A2 protein was detected by western blot using the antibody DYK tag (2368, Cell Signaling); while endogenous SLC27A2 was detected using the polyclonal antibody (PA5-30420, Invitrogen). The day before the experiment, HEK293 cells overexpressing SLC27A1/A2/A4/A5 were plated in six-well plates at 1–1.5 × 105 cells per cm2 in Dulbecco's modified eagle medium and cultured overnight. Cells were treated with 30 µM of EVT0185 for 3 h. At the end of the treatment period, medium was removed, cells were washed once with ice-cold PBS and were flash frozen in liquid N2. For extraction, plates were placed on ice, and cells were scraped in 266 µl of PBS per well. The cell suspension from three wells were pooled (approximately 800 µl total volume) and added to 1 ml of ice-cold 2:1 dichloromethane:methanol in a glass vial. The upper aqueous phase was transferred to a new tube and filtered through a 0.45-µm PTFE filter. Samples were delivered to the Centre for Microbial Chemical Biology at McMaster University for LC–MS/MS analysis. Samples were evaporated using a Turbovap LV (Biotage) evaporation system with the water bath set to 30 °C. Dried samples were resuspended in 100 µl of methanol, then analysed by injecting 5 µl into an Agilent 1290 Infinity series HPLC (Agilent) coupled to an LTQ-Orbitrap XL mass spectrometer (Thermo Fisher Scientific). Separation of the analytes was achieved on an Agilent Eclipse XDB-C18 (100 mm × 2.1 mm, internal diameter (i.d. ), 3.5 µm) analytical column using mobile phases consisting of (A) 10 mM ammonium acetate in water, pH 8.5, and (B) acetonitrile and a constant flow of 0.2 ml min−1. The analytical column was maintained at 30 °C. The separation of analytes was achieved using a gradient starting at 20% B, which was held for 1.5 min. The gradient increased to 95% B over the next 3.5 min and was held for 4.5 min. The total run time was 16 min. The HPLC eluent was introduced into the mass spectrometer using electrospray ionization in positive mode with a spray voltage of 3.6 kV. The mass spectrometer was set to acquire spectra in data-dependent acquisition mode. The full MS scan was set to an m/z range of 100–2,000 in the ion trap with a resolution of 30,000. The MS/MS was performed in the Orbitrap with a collision induced dissociation (CID) collision energy of 35. The activation time was set at 30 ms with the activation parameter q = 0.250 and an isolation window of 1.5 m/z. Hep3B (HB-8064) human hepatocarcinoma and Hepa1-6 (CRL-1830) mouse hepatoma cell lines were purchased from the ATCC and grown in eagle's minimum essential medium (10-009-CV, Mediatech) and Dulbecco's modified eagle medium (319-005-CL, Wisent Bioproducts), respectively, supplemented with 10% FBS (098150, Wisent Bioproducts) and 1% penicillin–streptomycin (15140122, Gibco, Thermo Fisher Scientific) in a humidified incubator at 5% CO2/95% air. Bempedoic acid was purchased from MedChemExpress. Hep3B and Hepa1-6 cells were seeded in 12-well plates at a cell density of 1,000 and 500 cells per well, respectively. On day 2, media in each well were aspirated and replaced with 1,000 µl of fresh complete media and cells were treated with or without the respective drugs in an increasing concentration followed by the incubation of the cells for 7 days in an incubator. On day 9, the media in each well were aspirated and cells were fixed with 10% formalin (500 µl) for 10 min at room temperature. Formalin was aspirated and the plates were washed with 1× PBS and stained with crystal violet for another 10 min. After 10 min of staining, crystal violet was poured over the cells, rinsed with tap water for three times and the plates were dried overnight. The next day, colonies (more than 50 cells) were counted and analysed. The pTrcHis2-hACLY expression construct35 for full-length human ACLY (Uniprot ID P53396-2) in frame with C-terminal MYC and His tags was transformed in BL21(DE3) cells. Cultures were grown in Luria–Bertani medium at 28 °C and expression was induced by the addition of 1 mM IPTG at an optical density at 600 nm of 0.6–0.7. After overnight expression, the bacterial culture was collected by centrifugation and bacterial cells were resuspended in IMAC-binding buffer, consisting of 50 mM sodium phosphate, pH 7.4, and 150 mM NaCl supplemented with cOmplete protease inhibitor cocktail without EDTA (Roche). The resulting supernatant was clarified using a 0.22-μm filter and loaded onto a Ni Sepharose column equilibrated with IMAC-binding buffer. The IMAC column was washed and His-tagged ACLY was eluted with binding buffer supplemented with increasing concentrations of imidazole. Elution fractions were pooled and concentrated using ultracentrifugation. ACLY was further purified by size-exclusion chromatography using HiLoad 16/600 Superdex 200 and Superose 6 (Increase) columns with 20 mM HEPES, pH 7.4, and 150 mM NaCl as a running buffer. The top fractions from the final size-exclusion chromatography elution peak were pooled and concentrated to 10 mg ml−1, aliquoted, flash frozen and stored at −80 °C in a freezer until further use. For cryo-EM grid preparation, purified ACLY (10 mg ml−1 in HBS buffer) was supplemented with 0.5% CHAPSO, 1 mM Mg2ATP and 4 mM EVT0185-CoA (freshly prepared from powder). Of the sample, 4 µl was applied to a glow-discharged C-Flat 1.2/1.3 300 mesh copper grid (Protochips), blotted for 4.5 s under 99% humidity at 22 °C and plunged into liquid ethane using a GP2 Leica grid plunger. Grids were screened using a JEOL 1400 Plus microscope equipped with a JEOL Ruby CCD camera at the VIB Bioimaging Core Ghent. Data collection was performed using a JEOL cryoARM 300 microscope equipped with a 6k × 4k GATAN K3 detector resulting in 7,263 movies with a raw pixel size of 0.72 Å (BECM). Movies were processed via patch-based motion correction and contrast transfer function estimation as implemented in cryoSPARC (v3.1.0)79. Good-quality 2D classes were initially obtained via the Blob Picker job in cryoSPARC, followed by template-based picking and neural network-based particle picking via TOPAZ as implemented in cryoSPARC80. Ab initio 3D classification (number of classes = 5) followed by homogeneous and non-uniform refinement81 without application of symmetry resulted in a 3D reconstruction that had a resolution of 3.7 Å following a gold-standard non-uniform refinement and represented the pseudo-D2-symmetric ACLY tetramer. In this cryo-EM map, the central, tetrameric CSH module was markedly better defined than the N-terminal CCS modules, indicating structural heterogeneity of the CCS modules with respect to the central CSH module. To address this flexibility and possibly resolve the CCS–CSH assembly at a higher resolution, we applied symmetry expansion in combination with local refinement followed by 3D classification. Following homogeneous and non-uniform refinement in D2 symmetry, the associated particle set was re-extracted without binning (0.72 Å per pixel) and symmetry expanded around the D2 axes. Using the molmap function in Chimera, a volume blurred to 25 Å that contained one CCS arm and the central CSH module was generated and transformed into a mask with the Volume Tools job in cryoSPARC (map threshold = 0.09, dilation radius = 7.2 Å and soft padding width = 14.4 Å). Gold-standard local refinement was performed by limiting the rotation and shift search extent around the original consensus refinement poses to 20° and 10 Å, respectively, in combination with a Gaussian prior over the pose or shift magnitudes (with SDROT = 15 Å and SDSHIFT = 7 Å). Ensuing 3D classification without alignment in cryoSPARC using the 3D classification job followed by another round of gold-standard local refinement resulted in a cryo-EM volume with a FSC0.143 resolution of 3.3 Å in which the atomic models for the CSS and CSH modules (extracted from Protein Data Bank (PDB) ID 6XHX) were fitted using Chimera and real-space refined in Phenix82 using reference restraints to the starting model. Restraints for EVT0185-CoA were generated via the de Grade Web Server (https://grade.globalphasing.org)83. A cryo-EM micrograph for the ACLY–EVT0185-CoA complex is provided in Extended Data Fig. Cryo-EM data and refinement statistics are summarized in Supplementary Table 2. Reported resolutions are based on the gold-standard FSC0.143 criterion84, and FSC curves were corrected for the effects of soft masking by high-resolution noise substitution85. Local resolution maps were computed using the blocres algorithm86 as implemented in cryoSPARC with an FSC threshold of 0.5. All representations of cryo-EM Coulomb potential density maps and structural models were prepared with ChimeraX87 and PyMol88. Cryo-EM map contour sigma levels have been reported based on map normalization in Coot. Hep3B cells (ATCC HB-8064) were cultured adherently using Eagle's minimum essential medium (ATCC 30-2003) supplemented with 10% (v/v) FBS (10437-028, Gibco) and 1% (v/v) antibiotic–antimycotic solution (15240112, Gibco). HEK293T (ATCC CRL-3216) was cultured adherently using Dulbecco's modified eagle medium, high glucose (11965092, Gibco) supplemented with 10% (v/v) FBS and 1% (v/v) antibiotic–antimycotic solution as aforementioned along with 1% (v/v) l-glutamine (25030081, Gibco), 1% (v/v) MEM non-essential amino acids solution (11140050, Gibco) and 1% (v/v) sodium pyruvate (11360070, Gibco). Dox-inducible shACLY plasmid vectors were generated using the inducible EZ-Tet-pLKO-Hygro vector (Addgene plasmid no. Two ACLY shRNA sequences—5′-CGTGAGAGCAATTCGAGATTA and 5′-GGCATGTCCAAGCTCAA—and one non-targeting shRNA sequence (5′-CCTAAGGTTAAGTCGCCCTCG) were ligated, transformed, amplified and validated using established protocol as previously described89. HEK293T cells were incubated with psPAX2 (Addgene plasmid no. 12259) and the EZ-Tet-pLKO-Hygro (Addgene plasmid no. 85972) transfer plasmid at a ratio of 1:1:2 pmol in the presence of Lipofectamine 2000 in Opti-MEM. Sixteen hours post-transfection, media were changed to 1% bovine serum albumin (A1470, Sigma-Aldrich) and 1 mM Na-butyrate (3850, Tocris) supplemented with antibiotic-free DMEM (11965092, Gibco) for HEK293T cell culture. Forty-eight hours post-transfection, lentivirus media were collected and mixed with Lenti-X concentrator (631232, Takara) before storing overnight at 4 °C. The resulting pellet was resuspended at 1:10 of initial media volume, aliquoted and frozen in liquid nitrogen for storage. An initial dilution of 1:50 v/v followed by serial dilutions of 1:4 v/v were used to transduce target cell lines at 60,000 cells per well in six-well plates using 5 µg ml−1 polybrene (107689, Sigma-Aldrich) supplemented cell culture media. Cells were transduced for 24 h, followed by recovery in lentivirus-free media for another 24 h before selection of successfully transduced cells using a pre-determined, cell line-dependent concentration of hygromycin B (H3274, Sigma-Aldrich) or puromycin (P8833, Sigma-Aldrich) for 7 days. Surviving cells post-antibiotic selection were further propagated to establish stable cell line. Cells were incubated in media supplemented with 1 µCi [14C] glucose (NEC042V250UC, Perkin Elmer) and [3H] acetate (NET003005MC, Perkin Elmer) for 4 h. Subsequently, cells were washed and scraped in PBS (10010023, Gibco). Lipids were extracted using a chloroform:methanol (1:2) solution, vortexed for 30 s and centrifuged at 13,000g for 10 min to isolate cellular debris and protein fractions from lipid supernatant. Lipids were further purified with chloroform:water (1:1) solution via vortex and centrifugation as described. Peptide concentration was quantified using the Pierce BCA Protein Assay Kit (23225, Thermo Scientific) for normalization calculation. Of lipid supernatant, 100 µl was extracted from the non-polar chloroform phase and mixed with 5 ml of Ultima Gold Scintillation Fluid (6013329, Perkin Elmer) for radioactivity quantification. Cells were incubated with serum-free cell culture media supplemented with 2% bovine serum albumin (A1470, Sigma-Aldrich), 500 µM sodium palmitate (P9787, Sigma-Aldrich), 0.5 µCi ml−1 [14C] palmitic acid (NEC075H050UC, Perkin Elmer) and 1 mM l-carnitine (11242008001, Roche) for 4 h. Cell culture supernatant was collected at the end of incubation and mixed with 1 ml of 1 M acetic acid (34256-1L-R, Fluka). Solution was sealed in a glass vial and shaken at 75 rpm at room temperature for 2 h. Released CO2 was absorbed in 450 µl of 1 M benzethonium hydroxide (B2156, Sigma-Aldrich) during this time. Subsequently, tube containing benzethonium hydroxide was mixed with 5 ml of Ultima Gold Scintillation Fluid (6013329, Perkin Elmer) for radioactivity quantification. Solution samples were collected and centrifuged at 13,000g for 10 min at 4 °C to isolate protein fraction from cell debris. Protein samples were quantified for peptide concentration using the Pierce BCA Protein Assay Kit (23225, Thermo Scientific) according to instruction. Samples were then diluted to 1 µg µl−1 using 4× SDS sample buffer (40% glycerol, 240 mM, Tris-HCl, pH 6.8, 8% SDS, 0.04% bromophenol blue, 5% β-mercaptoethanol and 20 mM dithiothreitol) and cell lysis buffer. Samples were then heated for 5 min at 95 °C before immunoblotting. Of the protein sample, 25 µg was loaded into each lane. Gel electrophoresis was performed first at 90 V as protein travels through the stacking gel and then at 120 V, through the separating gel. Protein samples were then transferred onto a PVDF membrane (03010040001, Roche) at 4 °C for 90 min following the standard wet transfer protocol. Subsequently, membranes were rinsed and rocked gently with 1× TBS (50 mM Tris, 150 mM NaCl and 1 M HCl, pH 7.4) for three cycles of 5 min each then blocked using a TBST-BSA solution (1× TBS, 0.5% bovine serum albumin and 0.1% Tween-20) for 1 h at room temperature. Membranes were then cut according to protein sizes and incubated in respective antibody solutions overnight at 4 °C with gentle rocking. The next day, primary antibody solutions were removed, and membranes were washed using TBST. Subsequently, membranes were incubated at room temperature with gentle rocking for 1 h in solutions containing 1:10,000 dilution of species-specific secondary antibody conjugated to HRP (anti-rabbit IgG HRP linked; 7074, Cell Signaling Technology). At the conclusion of incubation, membranes were washed again in TBST solution and kept in TBS solution until imaging and for storage. Peripheral blood mononuclear cells (PBMCs) were isolated from donors using the Ficoll Paque density gradient method. PBMCs were then washed with RPMI1640 and pelleted. Cells were resuspended with 10 ml of 3.5 µM CellTrace CFSE (C34570, Thermo Fisher) and incubated at room temperature for 20 min. After incubation, 40 ml complete RPMI1640 was added for the next 5 min to absorb excess CFSE. PBMCs were then washed, resuspended in complete RPMI1640 (10% FBS, 1% penicillin–streptomycin and 1% l-glutamine) and plated at a cell density of 1 × 106 cells per well in 12-well plates. For T cell proliferation, PBMCs were induced with 3 µg ml−1 of concanavalin A (C5275, Sigma-Aldrich) and 10 ng ml−1 of human recombinant IL-2 (200-02, Preprotech). For B cell proliferation, PBMCs were induced with 1 µg ml−1 of R848 (36611, mAb Tech) and 10 ng ml−1 of IL-2 (36611, mAb Tech). Next, PBMCs were incubated at 37 °C and 5% CO2 incubator for 5 days. Cells were then harvested, stained with E780 Live/Dead stain (C34570, Thermo Fisher), CD3 (300434, BioLegend) and CD19 (302241, BioLegend) antibody and analysed using flow cytometry. After 5 days, cells were then prepared for flow cytometry. For the B cell depletion study, the tail vein blood was collected from each animal after 15 days of single injection with 250 μg of anti-CD20 (152104, BioLegend) or isotype (400566, BioLegend) antibody. The cells were washed and blocked with Fc block (1:200; 553142, BD Biosciences) and stained with CD45.2 BV510 (1:25; 109838, BioLegend), B220 (1:100; 563894, BD Biosciences), CD19 (1:100; 152409, BioLegend) and 7AAD (1:100; A1310, Thermo Fisher Scientific). CytoFlex Flow Cytometer (Beckman Coulter Life Sciences) was used for data acquisition and was further analysed using FlowJo (v10.5). For PBMC analysis, the cells were harvested after 5 days of treatment and stained with E780 Live/Dead stain (C34570, Thermo Fisher), CD3 (300434, BioLegend) and CD19 (302241, BioLegend) antibody and analysed using flow cytometry. Independent t-test and analysis of variance (ANOVA) were performed using GraphPad Prism software (v9.5.1). Correlation and regression analysis were performed using R studio. All values are reported as mean ± s.e.m. Differences were considered significant (∗) when P < 0.05. Statistical significance between two independent groups was analysed by unpaired t-test. 1n and 9d,e were repeated at least four times. 4b shows a representative cryo-EM micrograph from the dataset (7,263 micrographs in total), illustrating the particle distribution and image quality. Histological scores were analysed using the non-parametric Mann–Whitney test when comparing two independent groups. Wilcoxon's test was used to analyse cell type and regional prevalence in MIBI-TOF images. Descriptions of additional software and statistical analysis used for transcriptomic analysis are mentioned in their respective methods. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data that support the findings of this study are available within the article and its Supplementary Information. The bulk RNA-seq data of the WD-DEN liver tumours, scRNA-seq data and spatial transcriptomic data of the WD-DEN and WD-CCl4 livers have been deposited at the NCBI Gene Expression Omnibus (GEO) and are accessible under accession numbers GSE296668, GSE297190 and GSE297081, respectively. For the mouse HCC model comparison, raw RNA-seq files for tumour samples derived from control-DEN and WD-CCl4 models were downloaded from the NCBI Sequence Read Archive under reference numbers PRJNA488497 and PRJNA386995, respectively. Single-nucleus sequencing of cells derived from healthy, MASLD, MASH-HCC tumour and tumour-adjacent tissue were obtained from GSE174748 and GSE189175. The gel source data are provided in Supplementary Fig. Source data are provided with this paper. The analysis code for MIBI-TOF data has been deposited in Zenodo90 (https://doi.org/10.5281/zenodo.15518613). Pfister, D. et al. NASH limits anti-tumour surveillance in immunotherapy-treated HCC. The immunosuppressive tumor microenvironment in hepatocellular carcinoma-current situation and outlook. Karin, M. New insights into the pathogenesis and treatment of non-viral hepatocellular carcinoma: a balancing act between immunosuppression and immunosurveillance. Stine, Z. E., Schug, Z. T., Salvino, J. M. & Dang, C. V. Targeting cancer metabolism in the era of precision oncology. Batchuluun, B., Pinkosky, S. L. & Steinberg, G. R. Lipogenesis inhibitors: therapeutic opportunities and challenges. Wellen, K. E. et al. ATP-citrate lyase links cellular metabolism to histone acetylation. Zhao, S. et al. ATP-citrate lyase controls a glucose-to-acetate metabolic switch. Vysochan, A., Sengupta, A., Weljie, A. M., Alwine, J. C. & Yu, Y. ACSS2-mediated acetyl-CoA synthesis from acetate is necessary for human cytomegalovirus infection. Izzo, L. T. et al. Acetylcarnitine shuttling links mitochondrial metabolism to histone acetylation and lipogenesis. Dibble, C. C. et al. PI3K drives the de novo synthesis of coenzyme A from vitamin B5. Kuna, R. S. et al. Inter-organelle cross-talk supports acetyl-coenzyme A homeostasis and lipogenesis under metabolic stress. The hallmarks of cancer metabolism: still emerging. Pinyol, R. et al. Molecular characterisation of hepatocellular carcinoma in patients with non-alcoholic steatohepatitis. Llovet, J. M. et al. Nonalcoholic steatohepatitis-related hepatocellular carcinoma: pathogenesis and treatment. Magen, A. et al. Intratumoral dendritic cell–CD4+ T helper cell niches enable CD8+ T cell differentiation following PD-1 blockade in hepatocellular carcinoma. Migita, T. et al. Inhibition of ATP citrate lyase induces an anticancer effect via reactive oxygen species: AMPK as a predictive biomarker for therapeutic impact. Hatzivassiliou, G. et al. ATP citrate lyase inhibition can suppress tumor cell growth. Hanai, J. I. et al. Inhibition of lung cancer growth: ATP citrate lyase knockdown and statin treatment leads to dual blockade of mitogen‐activated protein kinase (MAPK) and phosphatidylinositol‐3‐kinase (PI3K)/AKT pathways. Targeted inhibition of ACLY expression to reverse the resistance of sorafenib in hepatocellular carcinoma. Ma, Y.-N., Wang, S.-S., Liebe, R. & Ding, H.-G. Crosstalk between hepatic stellate cells and tumor cells in the development of hepatocellular carcinoma. Facciorusso, A. et al. Lenvatinib versus sorafenib as first-line therapy of advanced hepatocellular carcinoma: a systematic review and meta-analysis. Targeting ACLY attenuates tumor growth and acquired cisplatin resistance in ovarian cancer by inhibiting the PI3K–AKT pathway and activating the AMPK–ROS pathway. Xiang, W. et al. Inhibition of ACLY overcomes cancer immunotherapy resistance via polyunsaturated fatty acids peroxidation and cGAS-STING activation. Morrow, M. R. et al. Inhibition of ATP-citrate lyase improves NASH, liver fibrosis, and dyslipidemia. Lee, J. S. et al. Tumor stroma with senescence-associated secretory phenotype in steatohepatitic hepatocellular carcinoma. Dow, M. et al. Integrative genomic analysis of mouse and human hepatocellular carcinoma. Tsuchida, T. et al. A simple diet-and chemical-induced murine NASH model with rapid progression of steatohepatitis, fibrosis and liver cancer. Pinkosky, S. L. et al. Liver-specific ATP-citrate lyase inhibition by bempedoic acid decreases LDL-C and attenuates atherosclerosis. Nissen, S. E. et al. Bempedoic acid and cardiovascular outcomes in statin-intolerant patients. Wei, J. et al. An allosteric mechanism for potent inhibition of human ATP-citrate lyase. Oniciu, D. C. Functionalized long-chain hydrocarbon mono-and di-carboxylic acids and derivatives thereof, and their use for the prevention or treatment of disease. Falcon, A. et al. FATP2 is a hepatic fatty acid transporter and peroxisomal very long-chain acyl-CoA synthetase. Structure of ATP citrate lyase and the origin of citrate synthase in the Krebs cycle. & Savvides, S. N. Acetyl-CoA is produced by the citrate synthase homology module of ATP-citrate lyase. Pinkosky, S. L. et al. Long-chain fatty acyl-CoA esters regulate metabolism via allosteric control of AMPK β1 isoforms. Steinberg, G. R. & Hardie, D. G. New insights into activation and function of the AMPK. Positive selection of somatically mutated clones identifies adaptive pathways in metabolic liver disease. Zhao, S. et al. Dietary fructose feeds hepatic lipogenesis via microbiota-derived acetate. Carrer, A. et al. Acetyl-CoA metabolism supports multistep pancreatic tumorigenesis roles of acetyl-CoA metabolism in pancreatic tumorigenesis. Keren, L. et al. A structured tumor-immune microenvironment in triple negative breast cancer revealed by multiplexed ion beam imaging. Calderaro, J. et al. Intra-tumoral tertiary lymphoid structures are associated with a low risk of early recurrence of hepatocellular carcinoma. Immune checkpoint blockade in triple negative breast cancer influenced by B cells through myeloid-derived suppressor cells. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. Park, S. et al. Transcriptional upregulation of CXCL13 is correlated with a favorable response to immune checkpoint inhibitors in lung adenocarcinoma. Barrow, F., Khan, S., Wang, H. & Revelo, X. S. The emerging role of B cells in the pathogenesis of NAFLD. Zhang, Z. et al. Landscape of infiltrating B cells and their clinical significance in human hepatocellular carcinoma. Garnelo, M. et al. Interaction between tumour-infiltrating B cells and T cells controls the progression of hepatocellular carcinoma. Ho, W. J. et al. Neoadjuvant cabozantinib and nivolumab convert locally advanced hepatocellular carcinoma into resectable disease with enhanced antitumor immunity. Zhang, S. et al. Spatial transcriptomics analysis of neoadjuvant cabozantinib and nivolumab in advanced hepatocellular carcinoma identifies independent mechanisms of resistance and recurrence. The role of tumor-infiltrating B cells in the tumor microenvironment of hepatocellular carcinoma and its prognostic value: a bioinformatics analysis. Brunner, S. M. et al. Tumor-infiltrating B cells producing antitumor active immunoglobulins in resected HCC prolong patient survival. Li, W.-C., Ralphs, K. L. & Tosh, D. Isolation and culture of adult mouse hepatocytes. Kleiner, D. E. et al. Design and validation of a histological scoring system for nonalcoholic fatty liver disease. Feo, F., Pascale, R. M. & Calvisi, D. F. in The Cancer Handbook (ed. Feo, F. et al. Hepatocellular carcinoma as a complex polygenic disease. Interpretive analysis of recent developments on genetic predisposition. Dhanasekaran, R., Suzuki, H., Lemaitre, L., Kubota, N. & Hoshida, Y. Molecular and immune landscape of hepatocellular carcinoma to guide therapeutic decision making. Li, T. et al. TIMER2.0 for analysis of tumor-infiltrating immune cells. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Hu, C. et al. CellMarker 2.0: an updated database of manually curated cell markers in human/mouse and web tools based on scRNA-seq data. Alvarez, M. et al. Human liver single nucleus and single cell RNA sequencing identify a hepatocellular carcinoma-associated cell-type affecting survival. Ianevski, A., Giri, A. K. & Aittokallio, T. Fully-automated and ultra-fast cell-type identification using specific marker combinations from single-cell transcriptomic data. Li, Z. et al. GepLiver: an integrative liver expression atlas spanning developmental stages and liver disease phases. Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. Eling, N., Damond, N., Hoch, T. & Bodenmiller, B. cytomapper: An R/Bioconductor package for visualization of highly multiplexed imaging data. Nowicka, M. et al. CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. Integration, exploration, and analysis of high‐dimensional single‐cell cytometry data using Spectre. Stoltzfus, C. R. et al. CytoMAP: a spatial analysis toolbox reveals features of myeloid cell organization in lymphoid tissues. Van Gassen, S. et al. FlowSOM: using self‐organizing maps for visualization and interpretation of cytometry data. Windhager, J. et al. An end-to-end workflow for multiplexed image processing and analysis. In vivo percutaneous reflectance spectroscopy of fatty liver development in rats suggests that the elevation of the scattering power is an early indicator of hepatic steatosis. Swift, L. L., Love, J. D., Harris, C. M., Chang, B. H. & Jerome, W. G. Microsomal triglyceride transfer protein contributes to lipid droplet maturation in adipocytes. Han, J., Gagnon, S., Eckle, T. & Borchers, C. H. Metabolomic analysis of key central carbon metabolism carboxylic acids as their 3‐nitrophenylhydrazones by UPLC/ESI‐MS. Chronic cold exposure enhances glucose oxidation in brown adipose tissue. Cheng, T. et al. Pyruvate carboxylase is required for glutamine-independent growth of tumor cells. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Punjani, A., Zhang, H. & Fleet, D. J. Non-uniform refinement: adaptive regularization improves single-particle cryo-EM reconstruction. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. & Henderson, R. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. Chen, S. et al. High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. & Steven, A. C. One number does not fit all: mapping local variations in resolution in cryo-EM reconstructions. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Lilkova, E., Petkov, P., Ilieva, N. & Litov, L. The PyMOL molecular graphics system, version 2.0 (Schrodinger, LLC, 2015). Frank, S. B., Schulz, V. V. & Miranti, C. K. A streamlined method for the design and cloning of shRNAs into an optimized Dox-inducible lentiviral vector. McNicol, J. D. Multiplex ion-beam imaging single cell and spatial analysis in MASH-driven HCC. Espervita Therapeutics provided financial support for the conduct of the research related to EVT0185 and EVT0185-CoA. acknowledges the support of a Diabetes Canada Investigator Award (DI-5-17-5302-GS), a Canadian Institutes of Health Research Foundation Grant (201709FDN-CEBA-116200), a Tier 1 Canada Research Chair in Metabolism and Obesity and a J. Bruce Duncan Endowed Chair in Metabolic Diseases. acknowledges support from Research Foundation Flanders (FWO; grant 1524918N). is supported by a program grant from the VIB. We thank N. Henriquez and T. Campbell at the Centre for Microbial and Chemical Biology at McMaster University for assistance with the LC–MS experiments. We thank the VIB-VUB facility for Biological Electron Cryogenic Microscopy (BECM; Belgium) and the VIB Bioimaging Core Ghent (Belgium) for infrastructural access and technical support. These authors contributed equally: Jaya Gautam, Jianhan Wu, James S. V. Lally Centre for Metabolism, Obesity and Diabetes Research, Department of Medicine, McMaster University, Hamilton, Ontario, Canada Jaya Gautam, Jianhan Wu, James S. V. Lally, Russta Fayyazi, Elham Ahmadi, Sonia Rehal, Fiorella Di Pastena, Celina M. Valvano, Logan K. Townsend, Suhrid Banskota, Battsetseg Batchuluun, Maria Joy Therese Jabile, Alice Payne, Junfeng Lu, Eric M. Desjardins, Evangelia E. Tsakiridis, Bejal Mistry, Alex Aganostopoulos, Vanessa Houde, Paola Muti, Theodoros Tsakiridis, Dongdong Wang & Gregory R. Steinberg Espervita Therapeutics, Ann Arbor, MI, USA Jaya Gautam, Jianhan Wu, James S. V. Lally, Daniela Carmen Oniciu, Spencer Heaton, Roger S. Newton, Fiorella Di Pastena & Gregory R. Steinberg Jamie D. McNicol, Joanne A. Hammill, Ksenia Bezverbnaya & Jonathan L. Bramson Department of Oncology, McMaster University, Hamilton, Ontario, Canada Elham Ahmadi, Paola Muti & Theodoros Tsakiridis Division of Respirology, McMaster Immunology Research Centre, Schroeder Allergy and Immunology Research Institute, McMaster University, Hamilton, Ontario, Canada Ann Dansercoer, Koen H. G. Verschueren, Savvas N. Savvides & Kenneth Verstraete Ann Dansercoer, Koen H. G. Verschueren, Savvas N. Savvides & Kenneth Verstraete Department of Molecular and Cellular Endocrinology, Diabetes and Metabolism Research Institute, City of Hope Medical Center, Duarte, CA, USA Firestone Institute for Respiratory Health, The Research Institute at St. Joe's, St. Joseph's Healthcare, Hamilton, Ontario, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The overall conceptualization of studies related to model development, molecular and biochemical analysis, and characterization of Acly-KO mice were conducted by G.R.S., J.G., J.W. Representatives of Espervita Therapeutics were involved with the design and interpretation of studies involving EVT0185 (J.S.V.L., J.G., R.S.N., G.R.S., D.C.O. designed and performed the CoA detection experiment (Extended Data Fig. 3), an in vivo DNL experiment (Extended Data Fig. performed the MIBI-TOF imaging analysis presented in Fig. performed scRNA-seq analysis shown in Extended Data Figs. for the bulk RNA-seq analysis (Extended Data Fig. carried out liver histological assessments related to Fig. performed the lipid area and immunofluorescence imaging analysis (Fig. performed the SLC27A2 transfection experiment shown in Extended Data Fig. performed the human T cell and B cell proliferation experiments as presented in Extended Data Fig. performed the DNL experiments in WT and Acly-KO hepatocytes (Extended Data Fig. D.W. performed the bioinformatic analysis presented in Fig. assisted with viral injections used to generate Acly-KO mice (Fig. measured the TCA metabolite levels in tumour tissues (Extended Data Fig. assisted with animal euthanization and tissue collection. The other authors declare no competing interests. Nature thanks Raphaël Rodriguez and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Study design; DEN-injected animals fed Control Chow (Control-DEN) or Western Diet (WD-DEN). b, Liver mass, c, Liver mass/BW, and d, plasma AFP levels; mean ± SEM, WD-DEN (n = 10) vs Control-DEN (n = 8); unpaired two-tailed t-test: P = 3.86 × 10−6 (b), P = 1.41 × 10−5 (c). e, Representative H and E-stained liver sections. f, histological scores for steatosis, ballooning degeneration, inflammation, and NAS; mean ± SEM, WD-DEN (n = 10) vs Control-DEN (n = 8); unpaired Mann-Whitney two-tailed test: P = 2.29 × 10−5. g, Dorsal and ventral liver images. h, Tumor counts; mean ± SEM, WD-DEN (n = 10) vs Control-DEN (n = 8) mice; unpaired two-tailed t-test: P = 9.47 × 10−8. i, Tumor count distribution and j, tumor surface area; mean ± SEM, WD-DEN (n = 10) vs Control-DEN (n = 8); unpaired two-tailed t-test: P = 2.18 × 10−5. k, Distribution of tumor surface area per group. l, Histological feature of lesions (blue star: hepatic parenchyma, red star: Non-neoplastic proliferative lesion (FAH), orange arrow: slightly atypical hepatocyte (enlarged nuclei, higher N/C ratio, basophilic cytoplasm), black arrow: mitosis, blue arrows: atypical hepatocytes with enlarged and occasional binucleated nuclei, irregular nuclear membrane, prominent nucleoli and cytoplasmic alterations, red arrow: balloon cells with Mallory Denk bodies, yellow arrow: inflammatory infiltration). m, Number of neoplastic lesions in Control-DEN (n = 8) and WD-DEN (n = 10); mean ± SEM; unpaired two-tailed t-test. o, Average pairwise correlation on a per-model basis for mouse samples (column) and per-tissue or tissue molecular subtype basis for human samples (row). p, Proportion of mouse and human tumor tissue classification based on Hoshida molecular subtypes. q, Pairwise Pearson correlation between mouse models (columns) for cell types of interest (rows), using normalized gene expressions for all 1-to-1 orthologous genes between the mouse models (WD-DEN, WD-CCl4) and humans with MASH-HCC. r, Individual sample pairwise mouse to human correlation matrix. s-t, Mean Acly mRNA expression by cell types in livers from WD-DEN mice (s) and human MASH-HCC (t). a, Baseline serum AFP levels prior to AAV injection in WD-DEN treated mice. Data are presented as mean ± SEM. WT (n = 8) and Acly KO (n = 8) mice and statistical comparison was analyzed by two-tailed unpaired t-test. The diagram was created using BioRender (https://biorender.com). c, Pathological scoring of livers for steatosis, ballooning, inflammation, and NAFLD activity in WT (n = 8) and Acly KO (n = 8) mice. Data are presented as mean ± SEM. Statistical comparison for histological scores was analyzed by an unpaired Mann-Whitney two-tailed test. d, Fatty acid levels in tumor adjacent liver sections from WT and Acly KO mice (FA, Fatty acid; C:D, the total number of carbon atoms to the number of carbon-carbon double bonds). Data are presented as mean ± SEM, Acly KO (n = 10) vs WT (n = 11) mice by unpaired two-tailed t-test. e, Percentage of total liver area covered by tumors. Data are presented as mean ± SEM. f, Non-neoplastic proliferative lesions in livers from WT (n = 11) and Acly KO (n = 11) mice. Data are presented as mean ± SEM. g, Average lipid droplet size in liver tumors from WT and Acly KO mice. Data are presented as mean ± SEM, n = 5/group. Statistical significance within each genotype condition was analyzed by an unpaired two-tailed t-test. a, Overlay of the extracted ion chromatograms for EVT0185 reaction mixture at 0 and 30 min. c, Fragment ions of EVT0185-CoA at 30-min sample showing the fragment ion 605 m/z produced following the characteristic neutral loss of 3'-phosphonucleoside diphosphate (507 Da). Western blot showing d, Dkk tag expression in empty vector control (pCMV), SLC27A1, SLC27A2, SLC27A4, and SLC27A5 transfected HEK293 cells e, SLC27A2 (ACSVL1) protein expression in non-transfected cells (lane 1), pCMV controls (lanes 2–4) and SLC27A2 (lanes 5–7) transfected cells; β-actin as a loading control. For gel source data, see Supplementary Fig. 1. f, Extracted ion chromatogram from the MS1 and MS2 scans of HEK293 cells transfected with SLC27A2 and treated with 30 µM EVT0185. h, EVT0185-CoA detected only in extracts from HEK293 cells overexpressing SLC27A2 (ACSVL1); mean ± SEM, n = 6 biologically independent samples/group. j, Human SLC27A2 protein expression in different cell types. l, PBMCs stimulated with R848 and IL-2 showing CD19+ B cell proliferation compared to negative control. m, Average proliferation index of R848-induced CD19+ B cells with EVT0185 or vehicle, n = 2 biologically independent PBMC donors. n, PBMCs stimulated with Concanavalin A (Con A) and IL-2 showing CD3+ T cell proliferation compared to negative control. o, Average proliferation index of Con A-induced CD3+ T cells with EVT0185 or vehicle, n = 2 biologically independent PBMC donors. a, Global fit of Michaelis-Menten Plots by GraFit software for EVT0185-CoA effects on ACLY activity. b, Motion-corrected micrograph with picked particles encircled. The bottom scale bar is 20 nm. c, Cryo-EM data processing workflow in cryoSPARC for the ACLY:(R,S)-EVT0185-CoA complex. d, Sharpened cryo-EM map for ACLY:(R, S)-EVT0185-CoA complex following refinement without symmetry applied illustrating the ligand density at the four CoA-binding binding pockets of ACLY. The density is overlayed with the final real-space refined model molecular model for adenosine 3'-phosphate 5'-diphosphate (shown in Fig. e, Segment of the sharpened cryo-EM map following symmetry expansion, 3D classification, and local refinement and carved around residues ACLY residues 1054–1077 (carve radius = 2 Å). f-g, Comparison between the binding modes of (R, S)-EVT0185-CoA (panel f) and CoA bound to human ACLY in pdb 6hxh (panel g). a, Graphical representation showing metabolic enzymes in the regulation of sterol and fatty acid synthesis. The graphical representation was created using BioRender (https://biorender.com). Each stacked bar represents the mean ± SEM, % suppression with BA or EVT0185 in WT (n = 3) and Acly KO (n = 3) mice. Each line graph represents the mean ± SEM, enzyme activity inhibition by EVT0185-CoA (n = 3 samples/group). Each line graph represents the mean ± SEM colony formation in cells treated with EVT0185 or Bempedoic acid (n = 4 biologically independent samples/treatment groups). a, Effect of a single dose of EVT0185 on RER (Respiratory Exchange Ratio) in C57BL-6 mice. Each line graph represents the mean ± SEM, P values by unpaired two-tailed t-test, EVT0185 (n = 8) vs. vehicle-treated (n = 9) mice. b, 14C-glucose incorporation into fatty acids and cholesterol in the liver isolated from C57BL-6 mice treated with EVT0185 for 7 days. c, Experimental Scheme of WD-DEN HCC model. d, Serum-AFP levels before starting treatment, and e, tumor surface area after treatments. (f-g) f, Representative images showing tumor lipid droplets (Scale bars are 600 µm) and bar diagram showing percentage area of a lipid and g, average lipid droplet size in tumor, each bar represents the mean ± SEM, n = 6 mice/group, P values by unpaired two-tailed t-test. h, WD-fed mice with elevations in AFP at 18 months before starting treatment. Each bar represents the mean ± SEM, n = 5 mice/group, P values by unpaired two-tailed t-test. i, Sum of tumor diameter per mouse liver. Each bar represents the mean ± SEM, Isotype control (n = 11), PDL1 + VEGFR2 Ab (n = 10), and EVT0185 + PDL1 + VEGFR2Ab (n = 11) injected mice, P values by one-way ANOVA followed by Fisher's LSD multiple comparisons. of visible tumors on the liver surface in mice 4 weeks after adenovirus injection. Statistical analysis was performed using a two-tailed unpaired t-test. c, Differential expression analysis of Acly KO using an additive model adjusting for timepoint. Significance was determined by Wald test with a false discovery adjusted threshold of 5% as implemented in DESeq2. d, Modifying effect of time on Acly KO using an interaction model. Significance of interaction was determined by Wald test with a false discovery adjusted threshold of 5% as implemented in DESeq2. e, Comparison between the number of significant genes in each module negatively associated with both Acly expression and surface tumor burden. f, GSEA of decreased tumor growth size. g, TCA cycle showing a percentage change in tumor citrate and succinate levels in Acly KO mice compared to WT mice. Each bar represents the mean ± SEM, Acly KO (n = 12) vs WT (n = 9) mice, P values by two-tailed unpaired t-test. i, Hierarchical clustering of upregulated biological processes based on semantic similarity. j, Identification of co-expression modules significantly associated with Acly expression and surface tumor burden. k, Gene ontology analysis of co-expressed genes within module blue, Acly KO (n = 12) vs WT (n = 9) mice. c, Top upregulated pathways in B cells in Acly KO or EVT0185-treated mice (spatial transcriptomics analysis). Statistical analysis was performed using Fisher's Exact test. d and e, Single seq analysis of d, WD-DEN and e, WD-CCl4 mouse livers showing top upregulated pathways in B cells. Statistical analysis was performed using a one-sided hypergeometric test (enrichGO); p-values adjusted by Benjamini-Hochberg. f, Cxcl13 mRNA expression analyzed from publicly available RNA-seq dataset in WT and Acly KO DEN tumors cultured in vitro (GSE223966)10. Whiskers connect the minimum and maximum values. Significance was ascertained by an unpaired two-tailed t-test between Acly KO vs WT (n = 4 hepatocellular carcinoma cell lines derived from DEN-induced tumors in Aclyf/f mice). a and b, Quantification of lipid droplet area was extrapolated by converting the PLIN-2 signal into a mask and quantifying the area covered by this mask per 400 × 400 micron field of view (FOV). These calculations were performed in ImageJ using custom scripts. Each dot represents a single FOV, Wilcoxon non-parametric unpaired two-sided test. c, CXCL13 protein level in tumors from WT and Acly KO mice; mean ± SEM, n = 5 mice/group; unpaired two-tailed t-test. d, Representative images showing B cells at tumor centre and periphery (Orange star: Tumor front (border-periphery), Black star: Surrounding non-tumoral hepatic tissue). (g-j, l-n) WD-DEN mice treated with Vehicle or EVT0185. g, Representative MIBI images of liver tumors. h, B cell, and i, T cell counts from regions representing the tumor lesion interface with the liver; mean ± SEM, n = 4 mice/group; unpaired Wilcoxon non-parametric unpaired two-sided test. j, CD19+ positive cells count/mm2 tumor area; mean ± SEM, Vehicle (n = 29) and EVT0185 (n = 21) lesions; unpaired two-tailed t-test. k, CD3+ positive cells count/mm2 tumor area; mean ± SEM, WT (n = 12) and Acly KO (n = 10) lesions; unpaired two-tailed t-test. l, CD3+ positive cells count/mm2 tumor area; mean ± SEM, Vehicle (n = 29) and EVT0185 (n = 21) lesions; unpaired two-tailed t-test. m, Percentage of mice with/without B cells in TLS. n, TLS in EVT-treated mouse compared with vehicle control. o, GO enrichment analysis showing upregulated pathways related to complement activation in B cells in the EVT0185-treated group relative to Vehicle. Statistical analysis was performed using one-sided hypergeometric test (enrichGO); p-values adjusted by Benjamini-Hochberg. a-c, a, Representative images with percentage area fluorescence level of b, CCP3; mean ± SEM, Acly KO (n = 11) vs WT (n = 13) tumors by unpaired two-tailed t-test, and c, Ki67 protein expression in tumor leading edge; mean ± SEM, Acly KO (n = 12) vs WT (n = 15) tumors by unpaired two-tailed t-test: P = 8.74 × 10−8. d-f, d, Representative images and percentage area fluorescence level of e, CCP3 (P = 1.34 × 10−8) and f, Ki67 (P = 1.27 × 10−6) protein expression in the tumor leading edge. Data are presented as mean ± SEM. EVT0185 vs Vehicle control by unpaired two-tailed t-test, (n = 18 tumors per group). g, DOX inducible shRNA suppresses ACLY expression and phosphorylation in Hep3B cells. For gel source data, see Supplementary Fig. (h-j) Inducible knockdown led to h, reduced [14C] glucose-mediated DNL and i, increased [3H] acetate mediated DNL in Hep3B cells and j, increased fatty acid oxidation based on the sampling of [14C] incorporation in gaseous carbon dioxide obtained from cell culture media. Data are presented as mean ± SEM. Significance was determined by an unpaired one-tailed t-test. k, Outline of in vivo orthotopic experiment. l, Bioluminescence at 6 weeks was not significantly different between control and ACLY-deficient tumor. Each bar represents the mean ± SEM. Significance was determined by unpaired one-tailed t-test. a, Gating strategy to identify B220+CD19+ cells in Isotype control, and Anti-CD20 injected WT and Acly KO mice. Briefly, cells were gated as follows: SSC vs FSC plot of lymphocytes population, FSC-H vs FSC-A to identify single cell population, FSC vs 7-AAD to identify live cells population, B220 vs CD19 to identify B220+CD19+ B cell population. b. B220+CD19+ cells in Iso (Isotype control) and Anti-CD20 (B cell-depleted)-injected WT or Acly KO mice. Data are presented as mean ± SEM, Significant reductions were observed in WT-Anti-CD20 (n = 4; P = 2.37 × 10–6) and in Acly KO-Anti-CD20 (n = 7; P = 9.76 × 10–8), compared to WT-Isotype control (n = 11)-injected mice. The Acly KO-Isotype control group included n = 8 mice. Statistical analysis was performed using one-way ANOVA followed by Fisher's LSD test. b, Overlap between significantly upregulated and downregulated genes in each pairwise comparison. Whiskers connect the minimum and maximum values. d, ACLY upregulation in human MASH-HCC compared to all other tissue types using scRNAseq samples. Significance assessed using Wilcoxon rank-sum tests comparing each condition to MASH-HCC, with p-values adjusted using the Benjamini-Hochberg method. e, Differential expression analysis of genes associated with ACLY expression in MASH-HCC; two-tailed t-test; 5% FDR as implemented in limma. f, Principal component embeddings of genes differentially expressed with respect to ACLY expression validate stratification of MASH-HCC tissues by high, medium, and low ACLY expression levels. g, Upregulation of CXCL13 among human MASH-HCC samples with reduced (bottom tertile, n = 18) relative to elevated (top tertile, n = 18) ACLY expression. Significance determined by two-tailed t-test with a false discovery adjusted threshold of 5% as implemented in limma. Whiskers connect the minimum and maximum values. h, Identification of co-expression modules associated with ACLY expression and immune features. i, Significant biological processes associated with the immune and ACLY associated gene co-expression module. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. ACLY inhibition promotes tumour immunity and suppresses liver cancer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02420-1'>‘Sleeping' cancer cells in the lungs can be roused by COVID and flu</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 15:25:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). People infected with the virus that causes COVID-19 (lungs of a person with the disease shown here) have a nearly twofold increased risk of cancer-related death, data show.Credit: Rajaaisya/Science Photo Library Hidden in the lungs of some breast cancer survivors are tumour cells that can remain dormant for decades — until they one day trigger a relapse. Now, experiments in mice show that these rogue cells can be roused from their slumber by common respiratory illnesses such as COVID-19 or the flu. The findings, published in Nature on 30 July1, seem to extend to humans too: data from thousands of people show that infection with the SARS-CoV-2 coronavirus is linked with a nearly twofold increase in cancer-related death, possibly helping to explain why cancer death rates increased early during the COVID-19 pandemic. These cells, a precursor to metastasis — which is the spread to distant organs — pose a problem, even in survivors of these cancers. For instance, in about one-quarter of breast cancer survivors, such cells can trigger a relapse and metastasize. When exposed to flu virus, these dormant cancer cells (green) roused from their slumber, as indicated by a marker (magenta).Credit: Bryan Johnson DeGregori and his colleagues wondered whether acute inflammation caused by a respiratory infection could also reactivate dormant cancer cells. In these animals, the dormant cancer cells did not multiply nearly as quickly. The quest to prevent MS — and understand other post-viral diseases The quest to prevent MS — and understand other post-viral diseases This means that the infections do not directly cause cancer, but make it more likely that a future threat, whether it's an infection or a genetic mutation, could cause the cancer to become active again, DeGregori says. “You've awakened the flames and then it dies back down,” he says. “Seeing that these cancer cells were perverting the immune system to protect them as opposed to eliminate them was really quite shocking,” DeGregori says. Read the related News & Views ‘Inflammation during viral infection can rouse dormant cancer cells'. And listen to a podcast that discusses this study here. The quest to prevent MS — and understand other post-viral diseases Respiratory viral infections awaken metastatic breast cancer cells in lungs Respiratory viral infections awaken metastatic breast cancer cells in lungs Postdoctoral Associate – Systems Neuroscience: In Vivo Imaging & Electrophysiology Location: Virginia Tech, Fralin Biomedical Research Institute at... PhD program for life scientists and computational scientists/machine learning experts starting January 2026 The quest to prevent MS — and understand other post-viral diseases An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09332-0'>Respiratory viral infections awaken metastatic breast cancer cells in lungs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 15:25:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Breast cancer is the second most common cancer globally, with most deaths caused by metastatic disease, often following long periods of clinical dormancy1. Understanding the mechanisms that disrupt the quiescence of dormant disseminated cancer cells (DCCs) is crucial for addressing metastatic progression. Here we demonstrate, in mice, that influenza and SARS-CoV-2 infections lead to loss of the pro-dormancy phenotype in breast DCCs in the lung, causing DCC proliferation within days of infection and a massive expansion of carcinoma cells into metastatic lesions within two weeks. These phenotypic transitions and expansions are interleukin-6 dependent. We show that DCCs impair lung T cell activation and that CD4+ T cells sustain the pulmonary metastatic burden after the influenza infection by inhibiting CD8+ T cell activation and cytotoxicity. Crucially, these experimental findings align with human observational data. Analyses of cancer survivors from the UK Biobank (all cancers) and Flatiron Health (breast cancer) databases reveal that SARS-CoV-2 infection substantially increases the risk of cancer-related mortality and lung metastasis compared with uninfected cancer survivors. These discoveries underscore the huge impact of respiratory viral infections on metastatic cancer resurgence, offering new insights into the connection between infectious diseases and cancer metastasis. After initial remission, DCCs can stay dormant for years to decades before metastatic relapse, most commonly in lung, bone and liver4. Both cell-intrinsic factors and the tumour microenvironment determine whether metastatic cells stay dormant or progress5. Importantly, microenvironmental perturbations, such as those caused by increased inflammation, can be sufficient to increase metastasis5. Seasonal influenza affects more than 1 billion people each year6 and, by May 2025, SARS-CoV-2 infection had caused nearly 778 million recorded cases of COVID-19 (ref. Viral respiratory infections are typically associated with pulmonary inflammation, with a concomitant increase in pulmonary inflammatory cytokines, such as interleukin-6 (IL-6) and interferons (IFNs), and an expansion of immune cells, including neutrophils, macrophages and T lymphocytes2,3. Such inflammatory mechanisms, specifically involving IL-6 and STAT3 signalling8,9, neutrophils and neutrophil extracellular traps10, as well as the CD4+ cell–macrophage axis11, have been identified as regulators of metastatic processes in cancer. The observation that death rates from cancer rose in the first two years of the COVID-19 pandemic12, which is not fully accounted for by COVID-19 deaths or delayed screening and treatment, prompts an important hypothesis: that pulmonary viral infections increase cancer deaths by triggering the development of metastases from dormant DCCs. We sought to test this hypothesis through a dual approach: examining the effects of viral respiratory infections (influenza virus and SARS-CoV-2) on breast cancer dormancy in mouse models and correlating SARS-CoV-2 infection among cancer survivors to metastatic progression and cancer mortality. To study the effects of influenza virus infection on the awakening of dormant breast DCCs already lodged in the lung, we used the well-established MMTV-ErbB2/Neu/Her2 (hereafter MMTV-Her2) mouse model of breast cancer metastatic dormancy, in which mice overexpress rat Neu (Erbb2, a paralogue of human HER2) in epithelial mammary gland cells13,14. HER2+ early lesion cells in the mammary glands seed the lungs and other organs with DCCs within 10–14 weeks of life, where they remain largely as dormant single cells for up to one year before progressing to overt metastatic disease15. Thus, this model recapitulates the persistence of dormant DCCs in lungs and bone marrow in individuals who remain in remission for years to decades. MMTV-Her2 mice (FVB background) were infected with a sublethal dose of the influenza A virus (IAV) (Fig. Infected mice lost weight and recovered by 11–12 days post-infection (dpi) (Extended Data Fig. 1a), and wild-type and MMTV-Her2 mice showed a similar inflammatory response with increased cellularity of bronchoalveolar lavage (Extended Data Fig. The kinetics of viral clearance were similar between wild-type and MMTV-Her2 mice, in which IAV RNA copies peaked around 6 dpi, with a 100–1,000-fold reduction in viral load from 9–15 dpi (Extended Data Fig. a, MMTV-Her2 female mice in an FVB background were infected with a sublethal dose of Puerto Rico A/PR/8/34 H1N1 IAV by intranasal administration. Lungs and mammary glands were taken for analysis at the time points indicated after infection. b,c, Immunofluorescence (b) and quantification of HER2+ cells in lungs (c) at 3, 6, 9, 15, 28 and 60 dpi. The total number of HER2+ cells from three sections of the whole lung were quantified (n = 4 per group, n = 3 at 15 dpi). Lung sections were stained with DAPI (blue) and HER2 (green) as a marker for DCCs (b). d, Immunofluorescence and quantification of HER2+ cells in lungs 9 months after an influenza infection (n = 3 PBS, n = 5 IAV). e, Quantification of HER2+ cells in C57BL6/J MMTV-Her2 mouse lungs at 15 dpi with IAV (n = 4 PBS, n = 3 IAV). f, Immunohistochemistry and quantification of PyMT+ micrometastases defined by lesions with an area of less than 0.03 mm2 (n = 7 per group). g, EO771 mammary tumour cells were implanted into the mammary fat pads of C57BL/6 mice (n = 4 per group) and infected with IAV or PBS control after 31 days (experiment 1) and 20 days (experiment 2). The mice were implanted with 2 × 105 (experiment 1) or 1 × 106 EO771 (experiment 2) cells across two experiments, and combined results are shown. For each experiment, the average of the quantification of the PBS-treated mouse lungs was set to 1, so that a fold change could be calculated. Significance was determined by one-way analysis of variance (ANOVA). All box-and-whisker plots are presented as maximum value (top line), median value (middle line) and minimum value (bottom line), with all data points shown as dots. Illustration in a created using BioRender (De Dominici, M., https://BioRender.com/i40c047; 2025). The lungs of MMTV-Her2 mice were taken at 3, 6, 9, 15, 28 and 60 dpi (Fig. 1a) and examined for the abundance of HER2+ cells (Fig. Consistent with previous work14,16, we observed a small number of isolated DCCs or small clusters (fewer than 10 cells) in lungs before IAV infection. Strikingly, metastatic burden increased 100–1,000-fold between 3 and 15 dpi; the number of pulmonary HER2+ cells remained elevated even at 28 days, 60 days and 9 months after infection (Fig. IAV-mediated expansion of HER2+ DCCs was similarly observed in lungs of MMTV-Her2 mice in the C57BL/6J background at 15 dpi (Fig. Notably, the resultant expanded HER2+ cells exhibited a diffuse non-epithelial-like architecture, unlike the epithelial-like clusters and metastasis (more than 100 cells per cluster) of DCCs observed in lungs of MMTV-Her2 mice that are more than 10 months old14,16 (Extended Data Fig. Notably, we did not observe changes in the number of Ki67+HER2+ cells in mammary glands (Extended Data Fig. Furthermore, quantitative PCR (qPCR) analysis of transgenic rat Erbb2 expression in haematopoietic lineage-depleted cells from peripheral blood showed no changes with infection (Extended Data Fig. Thus, the increased number of HER2+ cells in the lungs does not seem to derive from increased seeding of cancer cells from mammary glands. We tested MMTV-PyMT mice with mammary-specific expression of polyoma middle T-antigen oncoprotein, which display early dissemination but shorter lung DCC dormancy16. MMTV-PyMT mice demonstrated an increased number of small tumour clusters in lungs after IAV infection (Fig. We also tested the effect of IAV infection using EO771 breast cancer cells (C57BL/6) implanted in the mammary gland, which seed the lungs and undergo a dormancy phase17. C57BL/6 mice with orthotopically implanted EO771 cells infected with IAV as above exhibited increased lung metastatic burden by 17–18 dpi compared with non-infected mice, in which EO771 cells remained largely dormant (Fig. 1h,i) Taken together, these findings show that IAV infection promotes DCC expansion in multiple models of breast DCC dormancy. When we examined the proliferation of DCCs in the lungs, we found a significant increase in the percentage of HER2+ cells expressing Ki67 (a marker of all cycle phases except G0) beginning at 3 dpi and peaking at 9 dpi (Fig. Similar results were obtained through in vivo incorporation of the thymidine analogue EdU (Fig. Although the fraction of HER2+ cells that express Ki67 decreased by 15 dpi, the total number of HER2+ cells expressing Ki67 remained highly elevated relative to baseline even at 60 dpi, given the overall increase in DCC burden in the lungs (Fig. These results show that IAV infection triggers DCC awakening in the lungs, increasing the metastatic burden. Lung sections from naive and IAV-infected mice were stained with antibodies against HER2 (green), Ki67 (magenta) and DAPI (blue) (a). Percentage of Ki67+ HER2+ cells (b, left), absolute number of Ki67+ HER2+ cells across three lung sections (middle, n = 4 per group, n = 3 at 15 dpi) and detection of EdU incorporation (right, n = 3 per group). c,d, Immunofluorescence and quantification of vimentin+ (Vim+) (c) and EpCAM+ HER2+ (d) cells in lungs after influenza infections, in which lung sections from naive and IAV-infected mice were stained with HER2 (green) and vimentin (magenta) (n = 3 per group, n = 4 PBS, 9 dpi, 60 dpi) or HER2 (green) and EpCAM (magenta) (n = 3 per group). Graphs show the percentage of vimentin+ HER2+ (c) or EpCAM+ HER2+ (d) cells. In a–d, statistical significance relative to PBS samples is shown, as determined by one-way ANOVA. e, GSEA analyses comparing DCCs from lungs of uninfected (PBS) and IAV-infected MMTV-Her2 mice at 9 dpi. f–h, Heatmaps of significantly differentially expressed collagen (Col) isoforms/lysyl oxidase (f), metalloproteinase (Mmp) (g) and vascular endothelial growth factor (Vegf) and intercellular adhesion molecule (Icam)/vascular cell adhesion molecule-1 (Vcam1) genes (h). All box-and-whisker plots are presented as maximum value (top line), median value (middle line) and minimum value (bottom line) with all data points shown by dots. Dormant DCCs in HER2+ and PyMT models maintain a ZFP281-driven mesenchymal-like state (vimentin+) until ZFP281 loss triggers an epithelial shift (EpCAM/E-cadherin+) during dormancy exit16. Consistent with previous results, most dormant DCCs present in uninfected lungs expressed vimentin and not EpCAM (Fig. The percentage of HER2+ cells expressing vimentin was not significantly affected early after infection (3–6 dpi). However, at 9 dpi the percentage of HER2+ cells expressing vimentin was decreased to around 50%, with a further decrease to less than 20% at 28 dpi (Fig. By contrast, early during IAV infection (3 dpi), a substantial fraction of HER2+ cells acquired EpCAM expression, associated with the awakening of DCCs (Fig. Thus, IAV infection drives sustained mesenchymal marker loss and a transient epithelial shift, with a persistent mixed or hybrid population over time, creating a hybrid phenotype that enables the awakening of dormant cells. To better understand the effect of IAV infection on DCCs, we performed flow cytometry sorting for HER2+ cells from the lungs of uninfected and IAV-infected mice at 9 dpi and performed bulk RNA-seq. As expected, IAV infection induced inflammatory, IFNα, IFNγ, TNF and IL-6–JAK–STAT3 signalling pathways (Fig. Notably, IAV infection also activated pathways in DCCs including collagen-containing extracellular matrix and angiogenesis (Fig. Collagen-1 abundance and crosslinking into fibrillar collagen has been linked to dormant DCC awakening18 that could be sustained by an angiogenic switch to maintain tumour growth. Indeed, angiogenesis and metalloproteinases have been shown to have a role in sustaining dormant cancer-cell awakening19,20. Notably, we observed striking changes in the expression of genes involved in mesenchymal or epithelial fate (Extended Data Fig. Although previous studies have shown conversion of DCCs from a dormant mesenchymal state to a more epithelial state after awakening16, we observed a unique and, to our knowledge, previously unrecognized hybrid and proliferative pattern after influenza virus infection, with increased expression of both mesenchymal markers (such as Cdh2, Cdh11, Fn1, Eng and Vim) and epithelial markers (such as Cdh1, Cldn2, Cldn5, Krt19, Klf4 and Ovol2) (all Padj ≤ 0.05; Supplementary Data 1). Notably, expression of Zfp281, which is a key mediator of the dormant mesenchymal state16, actually increased by 9 dpi in DCCs, further highlighting the hybrid state adopted by DCCs after IAV infection that seems to bypass the pro-dormancy function of ZFP281. We also observed increases in Cd274 (which encodes PDL1) and decreases in B2m (which is required for antigen presentation by major histocompatability complex class I), which could contribute to the avoidance of immune elimination (Supplementary Data 1). Furthermore, IL-6 produced during acute inflammation resulting from biopsy or chemotherapy contributes to the development of lung metastatic outgrowth of disseminated mammary tumour cells9,10. IL-6 is abundantly produced during IAV infection, partly because of the replication of the virus in lung epithelial cells24. Similarly, we also detected high levels of IL-6 in bronchoalveolar lavage fluid (BALF) from wild-type and MMTV-Her2 mice after IAV infection, with very low levels of IL-1β (Extended Data Fig. Moreover, we observed clear activation of the IL-6 signalling pathway in DCCs after IAV infection (Fig. Furthermore, in vitro IAV infection of primary mouse tracheal epithelial cells induced significantly increased secretion of IL-6 (Extended Data Fig. 2g), indicating that epithelial cells are a main source of IL-6 after IAV infection. To determine whether IL-6 production triggered by IAV infection contributes to the awakening of dormant DCCs, we used MMTV-Her2 mice crossed with Il6-knockout (KO) mice25. MMTV-Her2 and Il6-KO:MMTV-Her2 mice were infected with IAV and lungs taken for analysis at 9 and 28 dpi (at this dose of IAV, all mice recovered without excessive weight loss). Before infection, there was no difference in the number of dormant HER2+ cells between Il6-KO:MMTV-Her2 and MMTV-Her2 lungs (Fig. 3b,c), and these mice developed primary tumours requiring euthanization with similar timing at older ages (Extended Data Fig. Thus, IL-6 is not required for primary tumour growth or for early cancer-cell dissemination to the lungs. Strikingly, the number of HER2+ cells in lungs of IAV-infected Il6-KO:MMTV-Her2 mice was markedly decreased compared with infected MMTV-Her2 mice at both 9 and 28 dpi (Fig. 3a–c), with substantial reductions in Ki67+HER2+ cells, indicative of maintained DCC dormancy (Fig. Furthermore, in the MMTV-PyMT mouse model of breast cancer metastasis, IAV-induced proliferation of PyMT+ small lesions and formation of micro-metastases in the lungs was dampened by IL-6 deficiency (Extended Data Fig. Similar IL-6 dependency was found for the increased lung metastatic burden after IAV infection in the EO771 cell model (Extended Data Fig. Staining for vimentin and EpCAM demonstrated that most HER2+ cells in the lungs of Il6-KO:MMTV-Her2 mice retain vimentin expression and maintain EpCAM-negative status, which together with the failure to enter the cell cycle, as shown through immunofluorescence for Ki67, supports an IL-6 requirement for infection-induced DCC conversion from dormancy to awakening (Fig. d, Quantification of the percentage of HER2+ Ki67+ cells in MMTV-Her2 and IL-6 knockout:MMTV-Her2 at 9 dpi (n = 4 per group). e,f, Quantification of the percentage of vimentin+ HER2+ (e) and EpCAM+ HER2+ (f) cells in MMTV-Her2 and IL-6 knockout:MMTV-Her2 mice at 9 dpi (vimentin) and 3 dpi (EpCAM) (n = 3 for PBS, n = 4 IAV). Significance was determined by one-way ANOVA. g, Mean mammosphere area per well for HER2+ organoids after treating with 10 ng ml−1 IL-6 (n = 5 per group) (g); significance was calculated by Mann–Whitney test. All box-and-whisker plots are presented as maximum value (top line), median value (middle line) and minimum value (bottom line) with all data points shown by dots. We next digested mammary glands from 3-month-old MMTV-Her2 mice, grew them under organoid conditions and treated them with vehicle or IL-6. Mammary mammospheres treated with IL-6 showed significant increases in overall size (Fig. Similar increases in organoid size were observed for EO771 cells (Extended Data Fig. Although in vivo solitary DCC dormancy in the lung alveoli16,26 could not be fully replicated in vitro, mammosphere initiation assays from single cells to proliferative clusters have been used to study how perturbations affect single-cell growth initiation or growth arrest14,16. Despite its limitations, this assay confirmed that IL-6 directly affects HER2+ mammary tumour cells in a solitary state, which reproduced the IL-6 response observed in vivo. Overall, these results indicate that IAV infection-triggered IL-6 has a key role in mediating the awakening of dormant DCCs. Although IL-6 was essential for the awakening and the initial marked expansion of DCCs, minimal levels of IL-6 were detected in BALF of MMTV-Her2 mice 15 dpi (Extended Data Fig. 2e), indicating the presence of other factors that promote persistence after the expansion of DCCs at later times post-infection. Whereas recruitment of neutrophils to the lung occurs by 3 dpi with IAV, CD4+ T cells, CD8+ T cells and B cells accumulate in the lung from around 9 dpi in both wild-type and MMTV-Her2 mice (Extended Data Fig. Infection with IAV has also been shown to trigger the formation in the lungs of inducible bronchus-associated lymphoid tissues (iBALTs), which are lymphoid organizations that include primarily CD4+ and B cells. iBALTs can be detected in the lungs long after infection (up to 100 dpi)27. Accordingly, we also detected these CD4+ cell and B cell-enriched lymphoid organizations in the lung sections of wild-type and MMTV-Her2 mice 28 dpi (Fig. As expected, B cells in these iBALTs are also positive for the germinal centre B cell marker GL7. Moreover, consistent with the upregulation of many collagen genes in DCC after IAV infection (Fig. Collagens have been shown to limit T cell infiltration and activity28. b, As in a for IAV-infected MMTV-Her2 mice at 28 dpi, but for a region lacking CD4+ cells. c,d, Lung sections of MMTV-Her2 mice without or with CD4 depletion starting at −1 dpi or 10 dpi and taken for analysis at 28 dpi were stained for HER2 (green) and DAPI (blue) (shown for CD4 depletion starting at −1 dpi) (c). The number of HER2+ cells was quantified (n = 4 per group) (d). e, Quantification of HER2+ cells from MMTV-Her2 mice with CD4, CD8 or CD4/CD8 depletion (on −1 dpi) at 28 dpi (n = 4 per group, n = 3 CD8 depletion). f, Lung sections of MMTV-Her2 and CD4-depleted MMTV-Her2 mice at 28 dpi were stained for HER2 (green), CD8 (magenta) and DAPI (blue). g, Heatmap of the top 20 differentially expressed genes from scRNA-seq comparing CD4+ effector T cells from MMTV-Her2 + IAV versus wild type + IAV mice at 15 dpi. h, GSEA analysis showing pathway enrichment in effector CD8+ T cells in CD4-depleted MMTV-Her2 + IAV versus control MMTV-Her2 + IAV mice at 15 dpi. i, Concentration of IFNγ in a supernatant of enriched CD8+ cells enriched from lungs of MMTV-Her2 mice with or without CD4 depletion at 15 dpi and activated by anti-CD3/CD28 antibody (n = 4 IAV + IgG, n = 3 IAV + anti-CD4). j, Ex vivo CD8+ cytotoxic assay in which HER2+ cells were incubated with the same CD8+ cells (n = 3 per group). Significance was determined by one-way ANOVA (d and e) or two-tailed Student's t-test (i and j). All box-and-whisker plots are presented as maximum value (top line), median (middle line) and minimum (bottom line) with all data points shown as dots. To determine whether CD4+ cells maintain awakened DCCs later after IAV, CD4+ cell depletion was done before infection (−1 dpi) and HER2+ cells were examined in lung sections (Extended Data Fig. CD4+ cell depletion reduced the numbers of awakened DCCs at 28 dpi (Fig. However, CD4+ cell depletion did not affect DCC numbers or proliferation at 9 dpi (Extended Data Fig. 5c,d), consistent with the accumulation of CD4 cells late during infection (Extended Data Fig. Furthermore, the number of lung DCCs 28 dpi was also decreased when CD4+ cell depletion was initiated at 10 dpi (Fig. Thus, CD4+ cells contribute to the maintenance of awakened DCCs later after IAV infection. Previous studies have shown how neutrophil extracellular traps produced during inflammation can awaken DCCs in lungs29. However, in contrast to CD4+ cell depletion, the depletion of neutrophils with an anti-Ly6G antibody at the time of IAV infection did not alter the numbers or Ki67 positivity of HER2+ cells in the lungs (Extended Data Fig. Similarly, depletion of CD8+ cells before infection had no effect on the presence of DCCs (Fig. 4e), consistent with the paucity of CD8+ cells in lungs 28 dpi (Fig. Thus, maintenance of the awakened lung DCCs following IAV infection is selectively dependent on the presence of CD4+ cells. Interestingly, although only a low number of dispersed CD8+ cells were present in lungs 28 dpi in MMTV-Her2 mice, we found an increased accumulation of CD8+ cells in lungs of infected mice when CD4+ cells were depleted (Fig. These results indicated that CD4+ cells may repress the recruitment of CD8+ cells to the lung, potentially compromising immune surveillance against awakened DCCs. We therefore tested the effect of depleting both CD4+ and CD8+ cells on the maintenance of lung DCCs following IAV infection. Although CD4+ cell depletion resulted in a marked reduction of HER2+ cells in the lungs of infected mice, the dual depletion of CD8+ cells and CD4+ cells partly restored the numbers of HER2+ cells in the lungs (Fig. Thus, CD4+ cells maintain awakened lung DCCs after IAV infection, partly by suppressing the CD8+ immune response. We then examined whether the presence of awakened DCCs in the lungs following IAV infection could reprogram recruited T cells to a more suppressive or suppressed state by performing single-cell RNA-seq (scRNA-seq) of lungs of wild-type and MMTV-Her2 mice 9 and 15 dpi with IAV, at points in which the accumulation of T cells in the lungs is high (Extended Data Fig. As expected, multiple immune cell types, such as macrophages, natural killer cells, B cells, effector and memory CD4+ cells and effector and memory CD8+ cells, were present in the lung at both 9 and 15 dpi (Extended Data Fig. We had two replicates for most conditions, which exhibited very similar gene expression patterns (Extended Data Fig. IAV infection induced type I and II interferon responses and other innate immune pathways across these cell types, as expected (Extended Data Fig. Interestingly, there was substantially increased expression of a selective subset of genes in effector CD4+ cells from MMTV-Her2 mice relative to effector CD4+ cells in wild-type mice. In particular, Tnfaip3, Zfp36l2, Dusp5, Dusp1, Cxcr4, Klf6, Pdcd4 and Ctla4 were highly upregulated in effector CD4+ cells from MMTV-Her2 mice relative to wild-type mice (Fig. The differential expression of some of these genes was validated at the protein level either by flow cytometry (Cxcr4) or by western blot analysis (Dusp5) (Extended Data Fig. Tnfaip3, which encodes the E3 ubiquitin ligase A20, suppresses the antitumour activity of CD8+ cells30, whereas Zfp36l2, which regulates RNA stability, restrains CD8+-cell activation and expansion31,32. Dusp5, which encodes a dual phosphatase, suppresses T cell proliferation and promotes their survival33. Ctla4 induces Pdcd4 in cytotoxic T cells, and Pdcd4 deficiency enhances their antitumour effector functions36. These results indicated an impaired effector-cell phenotype in CD4+ cells from MMTV-Her2 mice relative to wild-type mice. Furthermore, comparing memory CD4+ cells from MMTV-Her2 mice versus wild-type mice at day 15 after IAV infection revealed similar changes in gene expression, with increased Tnfaip3, Klf6, Dusp1, Dusp5 and Dusp10 (Extended Data Fig. Similar changes were also observed in CD8+ cells isolated from MMTV-Her2 versus wild-type mice at 15 dpi (Extended Data Fig. Overall, these data indicated a bias to a more memory phenotype and increased survival with less effector function in CD4+ and CD8+ cells of infected MMTV-Her2 mice. Further analysis confirmed the increase in the ratio of memory to effector cells for both CD4+ and CD8+ cells in infected MMTV-Her2 mice relative to wild type (Extended Data Fig. 7f), consistent with DCC-mediated suppression of T cell effector function. Interestingly, the expression of multiple mitochondrial genes (including mt-Atp6, mt-Nd1, mt-Co3 and mt-Nd3) was greatly reduced in CD4+ cells from MMTV-Her2 mice, indicating a reduced mitochondrial content in these cells (Fig. The reduced mitochondrial content could result from increased autophagy mediated by the upregulation of Tnfaip3, because Tnfaip3 deficiency has been shown to increase mitochondrial content in CD4+ cells by promoting autophagy40. Reductions in mitochondrially and nuclear-encoded transcripts crucial for oxidative phosphorylation were observed across T cell types, including CD8 cells, in infected MMTV-Her2 mice by 15 dpi (but not at 9 dpi, before full DCC expansion), consistent with reduced T cell effector function (Extended Data Fig. We validated the reduction in mitochondria content in CD4+ and CD8+ cells from infected MMTV-Her2 mice relative to wild-type mice by Mitotracker staining and flow cytometry (Extended Data Fig. Pathway analysis further supports a compromised effector function of CD4+ cells from MMTV-Her2 mice. Over-representation analysis (ORA) of scRNA-seq comparing MMTV-Her2 and wild-type mice after IAV infection showed reduced interferon responses in MMTV-Her2 mice across multiple immune cell types (Extended Data Fig. 9a,b), as shown for macrophages (Extended Data Fig. 9c,d) and T cells (Extended Data Fig. 10a), consistent with the lower levels of type I and II IFNs produced in the lungs of infected MMTV-Her2 mice (Extended Data Fig. 10b; see Supplementary Data 3 for all significant gene set enrichment analysis (GSEA) results). We also observed significant suppression of the oxidative phosphorylation pathway in macrophages and T cells from infected MMTV-Her2 mice relative to infected wild-type mice by 15 dpi (Extended Data Figs. Together, these results indicate that the presence of DCCs skews the macrophage phenotype and impairs CD4+ and CD8+ cell activation in response to IAV infection, favouring tumour-cell persistence. Crucially, GSEA comparing CD8+ T cells from MMTV-Her2 mice with IAV infection (15 dpi) and CD4 depletion revealed significant increases in pathways involved in CD8+ cell activation, such as IL-2–STAT5, MTORC1 and type 1 and 2 interferon signalling pathways, indicating that both effector and memory CD8+ cells are more proliferative and activated when CD4+ cells are depleted (Fig. Mitochondrial and respiration phenotypes were also restored in CD8+ cells by CD4+ cell depletion by 15 dpi (Extended Data Fig. Thus, the effect of CD4+ cell depletion on eliminating DCCs seems to be mediated by enhanced CD8+ cell responses against DCCs. To further determine whether CD4+ cells in MMTV-Her2 mice are suppressing the activation of antitumour CD8+ cells, we isolated lung CD8+ cells from MMTV-Her2 mice or CD4+ cell-depleted MMTV-Her2 mice at 15 dpi with IAV. Increased production of IFNγ was specifically detected in lung CD8+ cells isolated from CD4-depleted infected MMTV-Her2 mice following ex vivo activation with anti-CD3/anti-CD28 antibodies (Fig. Moreover, we investigated the ability of these CD8+ cells to kill mammary cancer cells ex vivo using cultures of primary HER2+ cells isolated from the mammary glands of MMTV-Her2 mice. Lung CD8+ cells from CD4-depleted Her2 mice were also superior in killing MET-1 cancer cell lines generated from MMTV-PyMT mice that lacked HER2 (Extended Data Fig. Together, these data demonstrate the specific anti-mammary tumour activity of CD8+ cells from IAV-infected MMTV-Her mice and the suppressive effect that lung CD4+ cells have on antitumour CD8+ cells. To determine whether SARS-CoV-2 infection of lungs can promote the reawakening of dormant DCCs, we performed studies (analogous to those in Fig. 1 with IAV) using mouse-adapted SARS-CoV-2 that recognizes mouse ACE2, termed MA10, obtained by genetic modification of the spike gene41 followed by serial passages in mice42. This SARS-CoV-2 strain induces a COVID-19-like disease in mice including acute lung injury characterized by impaired pulmonary function, diffuse alveolar damage and infiltration of immune cells42. Similar to IAV infection, MA10 infection induced the production of high levels of IL-6 and IFNα in lungs (Extended Data Fig. Lower levels of IFNβ, IFNγ and IL-1β were also detectable in the BALF. Importantly, infection of MMTV-Her2 mice with MA10-SARS-CoV-2 resulted in a striking increase in HER2+ cells by 28 dpi (Fig. Analyses of earlier time points (3 and 9 dpi) demonstrated a stepwise increase in the number of HER2+ cells and Ki67+ HER2+ cells after MA10 infection (Fig. Notably, we demonstrated that these changes in HER2+ DCC proliferation, expansion and phenotypic transitions require IL-6, because MA10 infection-dependent changes are significantly reduced in Il6-KO:MMTV-Her2 mice, with no detectable change in MA10 virus replication (Fig. a, Quantification of HER2+ cells across three lung sections in C57BL6/J MMTV-Her2 mouse lungs at 28 dpi with MA10 SARS-CoV-2 (n = 6) or PBS control (n = 7). b,c, Quantification of HER2+ cells and percentage of Ki67+ HER2+ cells at 3 dpi and 9 dpi with MA10 in the lungs of FVB MMTV-Her2 mice (n = 3 PBS at 3 dpi and n = 4 at 9 dpi) (b) and comparing MMTV-Her2 mice without (WT) or with Il6 knockout (n = 4 per group) (c). Significance was determined by two-tailed Student's t-test (a,c). For b, we applied a negative binomial model for HER2+ cells per field comparing 9 dpi and PBS control (to accommodate the potential overdispersion); for HER2+Ki67+ cells per field, we determined whether cells per field in infected groups were significantly higher than 0 (all PBS samples were 0) using a negative binomial model. The analyses compared mortality risks between positive-test and negative-test participants, using censoring dates for death events from 1 December 2020 to 31 December 2022. e, Analyses from the Flatiron Health database evaluating the hazard ratio for the risk of progression to metastatic lung disease among patients with breast cancer who developed COVID-19 disease versus those who did not, adjusted for age, race and ethnicity (red) and multivariate analyses after also including co-morbidities, breast cancer subtype (for example, ER status) and other potential confounding factors (blue). All box-and-whisker plots are shown as maximum value (top line), median (middle line) and minimum (bottom line) with all data points shown as dots. The COVID-19 pandemic presents a unique opportunity to study the effect of pulmonary virus infections on cancer progression because, unlike influenza, data on virus infections and resulting outcomes were systematically collected in the first years of the pandemic. First, we analysed data from the UK Biobank to determine whether a SARS-CoV-2 positive test result, among a population of cancer survivors, was associated with an increased risk of all-cause, non-COVID-19 and cancer mortality (Extended Data Fig. To reduce potential confounding from vaccination and the use of at-home SARS-CoV-2 tests, we limited the analysis to subjects with PCR tests conducted before December 2020. Stratification of results based on primary tumour type and metastatic disease was not possible, owing to an insufficient number of observations. In the full study population with follow-up till 31 December 2022, which included 4,837 participants with a cancer diagnosis before 1 January 2015 (indicating inferred remission), we observed 413 deaths (298 in the test negatives and 115 in the test positives), yielding an odds ratio of 4.50 (95% confidence interval: 3.49–5.81) (Fig. When we excluded 120 deaths directly attributed to COVID-19, SARS-CoV-2-positive cases still showed increased mortality with an odds ratio of 2.56 (95% CI: 1.86–3.51). Based on the 128 cancer-related deaths as an outcome, we estimated a nearly twofold increase in cancer mortality in those who tested positive compared with those who tested negative (odds ratio, 1.85; 95% CI: 1.14–3.02). Analyses of participants diagnosed with cancer at least 10 years before the pandemic (before 1 January 2010) showed results consistent with the main analyses, although there was some loss of power owing to the reduced number of cases. SARS-CoV-2-positive participants had increased risks for all-cause mortality (odds ratio, 5.24; 95% CI: 3.66–7.48), non-COVID-19 mortality (odds ratio, 2.58; 95% CI: 1.64–4.06) and cancer mortality (odds ratio, 1.80; 95% CI: 0.85–3.80) compared with SARS-CoV-2-negative participants. When we reduced the follow-up period from 31 January 2022 to 1 December 2020, the odds ratio increased from 1.85 (1.1.4–3.02) to 8.24 (3.43–19.77) (Fig. 5d and Extended Data Table 2), with decreasing odds ratios across follow-up times reduced by subsequent 6-month intervals. This trend indicates that the strength of the association is greatest shortly after infection and diminishes over time. Some individuals in the test-negative group may have become infected during follow-up. These evolving differences in infection risk over time may have contributed to a weakening of the observed association. Even so, our findings indicate that the increased risk of cancer mortality is greatest in the first few months after SARS-CoV-2 infection. Together, the data indicate a markedly increased risk of death from cancer for cancer survivors who contract a SARS-CoV-2 infection. We used the Flatiron Health database with 36,845 female patients with breast cancer with complete information to specifically determine whether women with a primary diagnosis of breast cancer experienced an increased risk of progression to metastatic disease in the lungs after COVID-19 (Extended Data Table 3 and Extended Data Fig. Crucially, female patients with breast cancer who experienced COVID-19 disease after their initial breast cancer diagnosis exhibited an age, race and ethnicity-adjusted hazard ratio of 1.44 (95% CI: 1.01, 2.05; P = 0.043) for subsequent diagnosis of metastatic breast cancer in the lungs (red in Fig. Further adjusting our model for comorbidities and breast cancer subtype (oestrogen receptor (ER) status), we estimated a consistent, although slightly attenuated and not statistically significant, effect of COVID-19 on metastatic progression to the lungs with a hazard ratio of 1.41 (95%CI: 0.97, 2.04) (blue in Fig. These results indicate that the initial findings from the parsimonious model are robust and that potential confounding by additional covariates has a negligible effect in the Flatiron Health dataset. In all, these data show that COVID-19 increases lung metastasis risk in female patients with breast cancer. Our results indicate that respiratory virus infections promote the awakening and expansion of previously seeded dormant cancer cells. This process unfolds in two distinct phases. First, an IL-6-dependent switch of DCCs from a mesenchymal phenotype to a hybrid state promotes expansion. Second, this expansion is followed by a return to quiescence and establishment of CD4+ cell niches that inhibit DCC elimination, partly through the suppression of CD8+ cells (Extended Data Fig. We further show how the presence of HER2+ tumour cells results in a suppressive phenotype for CD4+ cells, and that depletion of CD4+ cells leads to the elimination of influenza virus infection-expanded DCCs, dependent on CD8+ cells with restored effector activity. Importantly, we show that a mouse-adapted SARS-CoV-2 virus similarly leads to IL-6-dependent DCC expansion in lungs. Although species differences warrant caution in interpreting mouse data, UK Biobank analyses show that cancer survivors had increased cancer mortality after SARS-CoV-2 infection. This risk peaked in the months after infection, paralleling mouse models showing greater than 100-fold DCC expansion into metastatic lesions within two weeks. Analyses based on the Flatiron Health database further reveal a marked rise in metastatic lung disease among breast cancer survivors following COVID-19. Collectively, these findings underscore the substantial metastatic risk COVID-19 posed to cancer survivors, with dormant DCC reactivation potentially driving this phenomenon. Our studies highlight the importance of developing interventions to minimize the risk of lung DCC awakening and metastatic disease in the millions of cancer survivors who experience respiratory virus infections. As well as primary prevention strategies, treatments for managing severe COVID-19 that have been approved by the US Food and Drug Administration include antagonistic antibodies against IL-6R46 and orally available JAK1/2 inhibitors47, raising the prospect of interventions that could reduce the risk of infection-induced metastatic cancer progression. The effectiveness and safety of these interventions, and the timing of their application to avoid impeding the resolution of the infection, will need to first be tested in preclinical and clinical studies. In conclusion, our studies reveal how respiratory virus infections can increase cancer recurrence risk and underscore the need for public health and clinical strategies to mitigate the increased risk of metastatic progression associated with SARS-CoV2 and other respiratory virus infections. Transgenic mouse models of breast cancer, using mouse mammary tumour virus (MMTV) long terminal repeats, are widely used. In brief, MMTV-PyMT and MMTV-erbB2/neu/Her2 (MMTV-Her2) mice express the oncogenes polyoma virus middle T antigen (PyMT) and rat Erbb2 (encoding HER2), respectively, upstream of the MMTV promoter, which confers expression in the mammary epithelium, as described elsewhere10,13,48. The MMTV-PyMT transgene is congenic in the FVB mouse background (a gift from William Muller). Given that the MMTV-PyMT mice exhibit substantial lung tumour burden within a few months of life, we limit our analyses to newly awakened DCCs in this model (forming micrometastases, defined as lesions with an area of less than 0.03 mm2). The MMTV-Her2 transgene is congenic in the FVB (Jackson Laboratory, 002376) and C57BL/6 (a gift from Ramon Parsons, congenic in C57BL/6J by backcrossing from the FVB background26) backgrounds. MMTV-Her2 mice (FVB) were crossed with IL-6-knockout (KO) mice9,25. For an orthotopic model of breast cancer, EO771 breast cancer cells49 (a gift from Diana Cittelly) were injected into the fourth right and left mammary fat pads with 2 × 105 or 1 × 106 cells per fat pad. Eight-week-old MMTV-PyMT and 12- to 14-week-old MMTV-Her2 female mice were infected with 500 EIU Puerto Rico A/PR/8/34 H1N1 IAV through intranasal administration in 50 μl PBS. For viral administration, mice were anaesthetized using 5% induction isoflurane and 2% maintenance, performed with a SomnoFlo Low-Flow electronic vaporizer machine in an induction chamber. After ensuring adequate anaesthesia with slow and deep breathing, droplets of viral fluid were placed on the mouse's nostrils. Once the fluid had been inhaled, the mouse was placed on a heating pad to recover. For immune-cell depletion experiments, mice were injected intraperitoneally with rat IgG as a control (MP Biochemicals, MPBio 0855951), 100 μg anti-CD4 (Bio X cell, clone GK1.5, BP003-1), or 100 μg anti-CD8 (Bio X cell, clone2.43) 1 day before IAV infection and every 6 days afterwards, or 200 μg anti-Ly6G (Bio X cell, clone 1A8, BP0075-1) on the day of the influenza virus infection, then 24 h and every other day afterwards, until being euthanized. For 5-Ethynyl-2′-deoxyuridine (EdU) incorporation, mice were injected with 50 mg per kg EdU (Sigma Aldrich, BCK488-IV-FC-S) 4 h before euthanasia. For both MMTV-PyMT and MMTV-Her2 mice, as a humane end point, mice were euthanized when the tumour reached 20 mm in any one dimension, tumours were ulcerated or infected, or if there was a major sign of discomfort, as determined by the institutional veterinarian. Mice were monitored every other day during the first week or until the tumour was palpated, and daily afterwards until the mice needed to be euthanized. All mice were co-housed in specific pathogen-free animal facilities, maintained at 21 °C (±1 °C) and 35% humidity with a 14 h:10 h light:dark cycle (light 06:00–20:00). Only female mice were used for the studies. An approved measure of CO2 followed by cervical dislocation was used for euthanasia. The University of Colorado Institutional Animal Care and Use Committee (IACUC) reviewed and approved all animal experiments (including humane end points described above), which were conducted in accordance with the NIH Guidelines for the Care and Use of Laboratory Animals. Mouse-adapted SARS-CoV-2 MA10 (BEI Resources, NR-55329) was propagated in Vero E6 cells (ATCC CRL-1586) as previously described42. When Vero E6 monolayers exhibited 70–75% cytopathic effect (2–3 dpi), supernatants were collected, clarified by centrifugation, supplemented with an additional 10% FBS, aliquoted and stored at −80 °C. SARS-CoV-2 titres were determined by plaque assay on Vero-E6 cells. Vero-E6 cells were maintained at 37 °C in Dulbecco's Modified Eagle medium (DMEM, HyClone 11965-084) supplemented with 10% fetal bovine serum (FBS), 10 mM HEPES (pH 7.3) and 100 U ml−1 of penicillin-streptomycin. MMTV-Her2 female mice (in both C57BL/6J and FVB backgrounds) at 14–19 weeks of age were anaesthetized by intraperitoneal injection of a mixture of ketamine (80 mg per kg) and xylazine (7.5 mg per kg) in a volume of 100–200 μl. Mouse weights were collected daily for 15 days, and mice inoculated with SARS-CoV-2 MA10 exhibited weight loss beginning at 2 dpi, with greatest loss achieved at 3-4 dpi, as previously reported42. As controls, MMTV-Her2 mice were mock inoculated with 50 μl of PBS/1% bovine calf serum. MA10 viral titre was determined as previously described50. Lung superior lobes were homogenized, serially diluted in DMEM with 2% FBS, HEPES, penicillin-streptomycin and incubated on Vero E6 cells for 1 h at 37 °C. Overlays were removed afterwards, and the plates were fixed with 4% paraformaldehyde for 20 min at room temperature. Fixed plates were stained with crystal violet (0.05% w/v) in 20% methanol for 10 min. Infectious viral titres were determined by manually counting the plaques formed. Lungs and mammary glands were collected and fixed in 10% neutral buffered formalin overnight, transferred to 70% ethanol the next day and then embedded in paraffin. Slides were deparaffinized in three incubations of 15 min in Histo-clear (Fisher Scientific, 50-899-90147) then descending 10-min ethanol incubations: three at 100%, followed by 95% and 70% followed by 10 min of H2O incubation. Heat-induced antigen retrieval was done for 10 min in a pressure cooker in citrate buffer (10 mM citric acid, pH 6.0). Sections were blocked for 1 h at room temperature with blocking solution (Abcam, AB64226) containing MOM blocking reagent (Vector Laboratories, MKB2213-1), incubated with primary antibodies (Supplementary Information Table 1) at 4 °C overnight in antibody diluent (Abcam, 64211), then washed 3 times for 30 min each in 0.1% triton-X 100 in PBS. For IHC samples, sections were incubated in ImmPRESS HRP goat anti-rabbit or rat IgG polymer detection kit (Vector Laboratories, MP-7451/MP7404) and ImmPACT DAB substrate, peroxidase HRP (Vector Laboratories, SK4105) according to the manufacturer's instructions. The IHC slides were mounted using micromount mounting medium (StatLab, MMC0126). For immunofluorescence, sections were incubated with secondary antibodies for 1 h at room temperature in antibody diluent (Abcam, 64211). Sections were then washed in 0.1% Triton-X 100 in PBS 3 times for 30 min each and were mounted using fluoroshield mounting media with DAPI (Abcam, 104139). Immunofluorescence images were collected using a Zeiss Axiovert 200-m fluorescence microscope. IHC images were collected using a Keyence BZ-X800 microscope. Section staining, image capturing and image analysis were done manually using ImageJ and were carried out by a researcher who was blinded to sample identities. Subsequent grouping and graphing were done by a different lab member who was unblinded after image analyses and quantification were completed. Collagen deposition was assessed using Masson's Trichrome stain. Bronchoalveolar lavage was done using 1 ml PBS (ThermoFisher, 14190-144) after mice were euthanized. Supernatant was flash frozen in liquid nitrogen and stored at −80 °C until analysis. Red blood cells were lysed using haemolytic buffer (150 mM NH4Cl, 1 mM NaHCO3, 1.1 mM Na2EDTA) for 3 min, flow buffer (PBS with 2% FBS and 2 mM EDTA) was added and cell suspensions were centrifuged at 500g for 5 min at 4 °C. Cells were resuspended in flow buffer and counted manually. Cytokines in the BALF were measured using custom-made high-sensitivity multiplex assays from Meso Scale Discovery according to the manufacturer's instructions. Cells recovered from BALF were stained with antibodies (Supplementary Information Table 1). Alternatively, whole lungs were taken and digested using a method described elsewhere52. In brief, lung digestion mix (1.5 mg ml−1 collagenase A (Sigma Aldrich, COLLA-RO), 0.4 mg ml−1 deoxyribonuclease I (Worthington, LS002139), 10 mM HEPES pH 7.2, 5% FBS) was injected into the lungs through cannulae and lungs were incubated in a shaking incubator at 37 °C for 30 min followed by vigorous vortexing. Digested lungs were passed through a 50 μm cell strainer and red blood cells were lysed using haemolytic buffer for 3 min, flow buffer were added and cell suspensions were centrifuged at 500g for 5 min at 4 °C. Single cells were resuspended in flow buffer and stained with antibodies (Supplementary Information Table 1) for flow cytometry. For mitochondrial mass analysis, lung cell suspensions were stained with Mitotracker green (Invitrogen, M7514) for 30 min at 37 °C. Data were collected on an LSR II flow cytometer (BD Biosciences) or Aurora (Cytek) and analysed using FlowJo software v.10. CD4 and CD8 cell populations were well defined (Extended Data Fig. For cell sorting of DCCs, lung cell homogenates were obtained from PBS or IAV (9 dpi)-infected MMTV-Her2 mice using a Lung Dissociation Kit Mouse according to the manufacturer's protocol (Miltenyi, 130-095-927). The single-cell suspensions were treated for red blood cell lysis. Single-cell suspensions were pre-incubated (5 min) with anti-CD16/CD32 Fc-Block (BD Biosciences, 553141) followed by staining for CD45 and HER2 and sorting using an Astrios EQ flow cytometer (Beckman Coulter). Sorted DCCs were used for bulk RNA-seq (described below). CD8+ cells were isolated from digested lungs using positive selection with CD8α (Ly-2) microbeads (Miltenyi, 130-117-044) according to the manufacturer's protocol. For CD8+ cell-mediated cytotoxicity experiments, Her2 cells isolated from mammary glands of MMTV-Her2 mice and expanded in culture, or immortalized PyMT cells (MET-1) isolated from mammary glands of MMTV-PYMT mice, were plated 2 days before the killing assay. Lung CD8+ cells (pooled from 3–4 mice) were added to the cancer cell cultures at a 1:1 effector:target ratio. Then, 48 h later, co-cultures were washed (removal of CD8+ cells), trypsinized and live cancer cells were counted. Isolated lung CD8+ cells were restimulated using anti-CD3/anti-CD28 coated beads53, and 20 h later, supernatant was collected and used for detection of IFNγ by ELISA, as previously described54, using anti-mouse IFNγ capture antibody (Biolegend, 505702) and biotinylated anti-mouse IFNγ antibody (Biolegend, 505804). CD4+ cells were isolated from IAV-infected wild-type and Her2 mice lung cell homogenate using positive selection with CD4 Microbeads (L3T4) (Miltenyi, 130-117-043) according to the manufacturer's protocol. Whole-cell extracts were prepared from CD4+ cells isolated from the lungs of wild-type and MMTV-Her2 mice (FVB) infected with IAV, following methods described elsewhere55. For western blot analysis, the following antibodies were used: β-actin monoclonal antibody (AC-15) (Invitrogen, AM4302), anti-DUSP5 (Invitrogen, PA5-85961), anti-rabbit HRP (Jackson ImmunoResearch Laboratories, 111-035-144) and anti-mouse HRP (Jackson ImmunoResearch Laboratories, 115-035-166). Single cells were generated as described in the section Flow-cytometric analyses. Cells exhibiting greater than 80% viability were fixed in a 4% formaldehyde solution using the Chromium Next GEM Single Cell Fixed RNA Sample Preparation Kit (10X Genomics). The whole-transcriptome probe pairs (10X Genomics) were added to the fixed single-cell suspensions to hybridize to their complementary target RNA during an overnight incubation at 42 °C. After hybridization, unbound probes were removed by washing. The fixed and probe-hybridized single-cell suspensions were loaded onto a Chromium X (10X Genomics) microfluidics instrument to generate partitioned nanolitre-scale droplets in oil emulsion. The target was for each droplet to contain a barcoded gel bead, a single cell and enzyme Master Mix (10X Genomics) for probe pair ligation and gel bead primer barcode extension. The single-cell barcoded, ligated probe products underwent library preparation using standard 10X Genomics protocols in preparation for Illumina next-generation sequencing. The gene expression library derived from single-cell barcoded, ligated probe product were sequenced as paired-end 150-base pair reads on an Illumina NovaSeq 6000 (Illumina) at the University of Colorado Genomics Shared Resource at a target depth of 20,000 reads per cell for all samples. The scRNA-seq fastq files were processed using Cell Ranger software (v.7.1.0, 10X Genomics)56 to assign reads to genes based on Cell Ranger's Chromium mouse transcriptome probe set (v.1.0.1). Genes found in fewer than 10 cells were excluded. After downstream processing, clusters were filtered if they contained canonical markers from multiple cell types. The data were then depth-normalized followed by natural-log transformation. The top 2,000 most variable genes were used to scale the data while regressing out cell cycle S/G2M difference, total UMI and percentage of mitochondrial UMIs. Principal component analysis was performed using the top 2,000 variable genes. Principal components (n = 30) that captured most of the variation were then included in further data-processing steps. Clusters were identified (at a resolution of 1.5) using the K-nearest neighbours algorithm. Clusters were annotated to cell types using enriched canonical markers and ORA59 with gene sets from the MSigDB60 and the PanglaoDB61. Broad T lymphocytes were identified and subclustered separately to increase cell-type resolution. Plots were produced using the Seurat57, ggplot263, ggpubr64 and pheatmap65 R packages. To focus on mitochondrial functions, we used our custom mitochondrial pathway gene lists, originally published in ref. Specifically, we examined overlaps between mitochondrial OXPHOS genes and our curated innate immune pathways associated with mitochondrial activity. Pathway analysis was done using fast GSEA67 with custom gene-set files previously curated in ref. All samples were compared with controls, and the ranked list of genes was defined using the −log10(P value) × log2(fold change). Statistical significance was assessed through 1,000 permutations of the gene sets60. Results are reported with a false discovery rate (FDR) threshold of less than 0.25 and visualized as heatmaps generated with the pheatmap package (v.1.0.12). RNA was isolated from sorted DCCs (flow sorting described above) with the RNeasy plus micro kit (Qiagen, 74034) and libraries were prepared using the SMART-Seq mRNA LP kit (Takara Bio, 634762) following the manufacturer's instructions. Pooled libraries were sequenced on the NovaSeq X (Illumina). Reads were trimmed with Cutadapt69 and aligned to the mouse transcriptome (GRCm38, Ensembl release 102) using STAR (v.2.7.9a)70 and quantified using Salmon (v.1.10.1)71. Differential expression analysis was done using limma72 with the voom method followed by GSEA as described above. Whole lung tissue was homogenized and RNA was isolated using TRIzol/chloroform extraction following the manufacturer's protocol (ThermoFisher and MilliporeSigma, respectively). RNA (1 μg) was reverse transcribed with an iScript cDNA synthesis kit (Bio-Rad Laboratories) and the viral load was determined by qPCR for the PR8 acid polymerase gene compared with a standard curve of known PR8 acid polymerase gene copy numbers as previously described73. Mice were euthanized using isoflurane and cervical dislocation. Whole mammary glands were minced and digested in 0.15% Collagenase 1A (Sigma, C-9891), 2.5% bovine serum albumin and 200 U DNAse I (Stemcell Technologies, NC9007308) solution at 37 °C with agitation for 30 min. Red blood cell lysis buffer (eBioscience, 4333-57) was used for 2 min at room temperature to remove blood cells. Cells were filtered through a 40-μm filter. Then 3 × 105 cells per well were seeded in six-well ultralow-adhesion plates in 1 ml mammosphere media (DMEM/F12 (Gibco, 11320-082), 1× B27 supplement (Gibco, 17504-044), 10 ng ml−1 EGF (Peprotech, AF-100-15-A), 50 U penicillin-streptomycin (Thermo Fisher, 15070-063)). An additional 1 ml of mammosphere medium was added 24 h after seeding. At day 4 after seeding, cells were treated with either PBS or 10 ng ml−1 IL-6 (R&D Systems, 406-ML-005) for 3 consecutive days. Using a Nikon Eclipse Ti-S microscope, mammospheres were imaged at 4× magnification with two images taken per well at the end of the treatment. The size and number of mammospheres were analysed using QuPath software. EO771 cells were seeded in a poly-HEMA-coated 12-well low-adhesion culture dish at a density of 1.5 × 105 cells per well in 1 ml organoid medium (DMEM/F12, 5% FBS, 1% penicillin-streptomycin 5,000 U ml−1, 20 ng ml−1 FGF2, 10 ng ml−1 EGF, 5 μM Y-27632, 4 μg ml−1 heparin plus 5% Matrigel). Cells were treated with either PBS or 10 ng ml−1 IL-6 (R&D Systems, 406-ML-005) for 3 days. Using an EVOS M7000 microscope, EO771 organoids were imaged at 4× magnification with five images taken per well every other day. The size and number of organoids were analysed using FIJI (ImageJ). MMTV-Her2 mice 12–14 weeks old were infected with 500 EIU Puerto Rico A/PR/8/34 H1N1 IAV or PBS as described above and euthanized using CO2 at 9 dpi. Blood was collected by intracardiac puncture and placed in heparin solution on ice. Following red blood cell lysis, lineage depletion was performed using a Miltenyi Direct Lineage cell depletion kit, mouse (Miltenyi, 130-110-470) following the manufacturer's instructions. After lineage depletion, RNA was extracted using an RNeasy Plus Micro Kit (Qiagen, 74034) following the manufacturer's instructions. Quantitative PCR with reverse transcription (RT–qPCR) was done using the iTaq Universal SYBR Green One-Step RT–qPCR (Bio-Rad, 172-5150) with primers for the MMTV-Her2 rat transgene; forward, 5′-CCCGAGTGTCAGCCTCAAA-3′; reverse, 5′-GCAGGCTGCACACTGATCA-3′. The RT–qPCR was run on a Bio-Rad thermocycler (CFX Opus 384). Statistical analyses were done using Prism 10.2.1 software (GraphPad). Quantification and image analysis were done in a blinded manner; n indicates the number of mice per group. A minimum of three slides per mouse were used for image analysis. Three lung sections at least 50 µm apart per mouse were counted and summed. We collected and analysed PBS groups at each time point; because no differences in DCC expansion or phenotype were observed at different time points, results for PBS samples were pooled. For other image quantifications, whole-lung images were divided into fields using the ImageJ grid function and 8–10 fields were selected at random per image and counted. For experiments with two groups, a two-tailed Student's t-test was used; for experiments with more than one group, one-way ANOVA tests were used unless otherwise stated. and P values ≤ 0.05 were interpreted as evidence against the null hypothesis (that is, no effect, no difference). Graphs are presented as box and whiskers with dots representing individual values; the three lines represent the maximum (top line), median (middle line) and minimum (bottom line) values of the dataset. Two complementary datasets from different regions of the world were analysed: the UK Biobank, which is a population-based study including 502,356 adult volunteers aged 40–69 years at recruitment from 2006 to 2010 (refs. 74,75), and the Flatiron Health electronic health record (EHR) database, which contains longitudinal data from about 280 US cancer clinics (around 800 sites of care) on patients with cancer and survivors76,77. Study 1 was an analysis of UK Biobank data including lifestyle, anthropometric, medical history, SARS-CoV-2 testing and mortality data linked to national registries. Previous cancer diagnoses were obtained through consented linkage to the national cancer registry and SARS-CoV-2 test status through linkage to national registers. Mortality data were obtained from the national death registries (NHS Digital, NHS Central Register and National Records of Scotland). We considered all-cause mortality (including both primary and secondary causes), non-COVID-19 mortality (by excluding deaths with ICD codes U07.1 and U07.2 (ref. 78) or any death within one month of the latest recorded positive test result) and cancer mortality (considering cause of death with ICD codes listed in Extended Data Table 1). To evaluate whether SARS-CoV-2 test positivity affected all-cause, non-COVID-19 or cancer mortality, we implemented a rigorous matching strategy. Cancer survivors with a primary cancer diagnosis at least five years before the start of the pandemic and a positive COVID-19 test result were matched to cancer survivors with negative test results with a similar risk profile. Of the 502,356 UK biobank participants, we excluded two groups: first, those with missing information on sex, age, body mass index, ethnicity, smoking status, alcohol consumption, education, employment status, household income, self-reported comorbidities, date of SARS-CoV-2 testing when the primary cause of death was COVID-19 and cancer diagnosis date if the primary cause of death was cancer (n = 65,245); and second, participants without any SARS-CoV-2 PCR test record (n = 195,559) (Extended Data Fig. This left 241,552 participants, of whom 48,958 had been diagnosed with cancer at the latest follow-up (18 December 2022). From this group, we excluded five groups: participants with inconsistent dates of death (n = 8); those diagnosed with multiple cancers (n = 4,421); participants with a primary cancer diagnosis after the start of the pandemic (defined as 1 January 2020; n = 7,650); those who tested positive for COVID-19 after the UK vaccination rollout (1 December 2020; n = 13,274); and participants with cancer diagnoses less than five years before the pandemic onset (n = 9,969); this was to ensure that participants were, in all likelihood, in full remission and thus any residual metastatic cancer cells were likely to be dormant. After these exclusions, the final cohort included 13,636 participants, of whom 531 tested positive for SARS-CoV-2, and 13,105 who tested negative before the vaccination rollout (Extended Data Fig. We used a non-parametric matching approach (without replacement)79 to identify (up to) ten negative-test participants for each positive-test participant. Matching was performed in two steps. We performed an exact matching based on cancer type and sex. Then, we matched for age, ethnicity, smoking status, alcohol consumption, education, employment status, household income and cancer diagnosis date (with a maximum allowable difference in cancer diagnosis of five years) using the nearest-neighbour method, an algorithm based on propensity score matching. The resulting matched population included 487 with positive tests (that is, we could not find good matches for 44 of those with positive tests) matched to 4,350 with negative tests. Using test positivity as the predictor, we ran a series of unconditional logistic regression models for all-cause, non-COVID-19 and cancer mortality. This was achieved by excluding the positive-test participants who were diagnosed with cancer between 1 January 2010 and 31 December 2019 and re-running the matching procedure, resulting in 266 with positive tests and 2,228 matched individuals with negative tests. Sensitivity analyses were conducted by varying censoring dates in six-month intervals from 1 June 2020 to 31 December 2022. Longer follow-up periods included more events, whereas shorter periods minimized potential bias from missing infection data and vaccination. Study 2 used Flatiron Health's nationwide EHR-derived database, including de-identified data from about 280 US cancer clinics (around 800 sites of care). The database is longitudinal, comprising de-identified patient-level structured and unstructured data, curated by technology-enabled abstraction76,77. Most patients in the database originate from community oncology settings, although the community and academic proportions may vary, based on study cohorts. Institutional Review Board approval of the protocol was obtained before the study was done and included an informed-consent waiver. Included in our study were women aged at least 18 years old at the time of initial cancer diagnosis, and who had: early breast cancer; the cohort includes a probabilistic sample of patients with a diagnosis of stage I–III breast cancer on or after 1 January 2011, including those who presented with non-metastatic disease but who subsequently developed recurrent or progressive disease, with at least two visits occurring on or after January 1, 2011; metastatic breast cancer; the cohort includes a probabilistic sample of patients diagnosed with stage IV breast cancer on or after 1 January 2011 and those who presented with earlier-stage breast cancer but who subsequently developed metastatic disease on or after 1 January 2011, and who had at least two clinic encounters evident in the database occurring on or after 1 January 2011; and adult female patients aged 18 years or more at the initial diagnosis. Baseline characteristics of gender, race, ethnicity and age at index date were obtained from structured data. Baseline characteristics were summarized using descriptive statistics. Univariable and multivariable Cox proportional hazard models were used to evaluate the effect of COVID-19 diagnosis on the risk of metastasis to the lungs, in which COVID-19 diagnosis status was treated as a time-varying covariate. The multivariable model was adjusted for patient characteristics considered relevant, including age, race and ethnicity. The median follow-up, the corresponding interquartile range (IQR) and the total number of accumulated person-years, for all patients, were 4.36 years, 6.21 years and 277,788 person-years, respectively. The median follow-up, the corresponding IQR and the total number of accumulated person-years, for patients' COVID-negative period, were 4.35 years, 6.21 years and 277,115 person-years, respectively. The median follow-up, the corresponding IQR and the total number of accumulated person-years, for patients' COVID-positive period, were 0.98 years, 1.08 years and 673 person years, respectively. The two-sided likelihood ratio tests were conducted. Patients without a date of pulmonary metastases were censored at the last confirmed activity date or death. Last confirmed activity was defined as the latest date of vitals record, medication administration or reported laboratory tests or results. We performed additional multivariate analyses (MVA) to control for additional potential confounding factors, including comorbidities and breast cancer subtypes as sensitivity analysis. Evidence of an ER-negative, progesterone receptor-negative and HER2-negative test result, in which HER2-negative is defined as negative with the cancer type not otherwise specified (NOS), next-generation sequencing (NGS) negative (ERBB2 not amplified), fluorescence in situ hybridization (FISH) negative/not amplified, IHC negative (0–1+) or IHC equivocal (2+); ER-positive and/or progesterone receptor-positive test result(s). A stratified Cox proportional hazard model with stratification factors stage, year of diagnosis, age group and cancer subgroup was used to evaluate the effect of COVID-19 diagnosis on the risk of metastases to the lungs while adjusting important covariates (age, race, ethnicity and comorbidity) at initial diagnosis. 81, indicating that there was no statistically significant evidence suggesting a violation of the proportional hazard assumption. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw and processed scRNA-seq data have been deposited in the Gene Expression Omnibus (GSE264175). For RNA-seq of DCCs, raw and processed RNA-seq data have been deposited in the Gene Expression Omnibus (GSE282438). Data availability for the UK Biobank and Flatiron Health analyses is described in the sections above. This study used the UK Biobank data under application number 69328 to M.C.-H. The UK Biobank received ethical approval from the North West Multi-centre Research Ethics Committee (REC reference11:/NW/0382) (http://www.ukbiobank.ac.uk/ethics/). The UK Biobank data are accessible on approval from the UK Biobank access committee. Preprocessing and recoding and analytical scripts are available on request to allow the replication of findings by researchers with active UK Biobank access. Requests for data sharing by licence or by permission for the specific purpose of replicating results in this manuscript can be submitted to PublicationsDataAccess@flatiron.com. The UK Biobank and Flatiron Health statistical analyses were conducted using R v.4.1.0 (ref. All other data are available from the corresponding author upon reasonable request. Source data are provided with this paper. The code used for the scRNA-seq analysis is available without restrictions on GitHub at https://github.com/Aeg22/dcc_flu. Source data are provided with this paper. Tan, L. Y., Komarasamy, T. V. & Rmt Balasubramaniam, V. Hyperinflammatory immune response and COVID-19: a double edged sword. Breast cancer subtypes predispose the site of distant metastases. The dormant cancer cell life cycle. Manore, S. G., Doheny, D. L., Wong, G. L. & Lo, H.-W. IL-6/JAK/STAT3 signaling in breast cancer metastasis: biology and treatment. Ganesan, R. et al. Taxane chemotherapy induces stromal injury that leads to breast cancer dormancy escape. Acute inflammation induced by the biopsy of mouse mammary tumors promotes the development of metastasis. DeNardo, D. G. et al. CD4+ T cells regulate pulmonary metastasis of mammary carcinomas by enhancing protumor properties of macrophages. Expression of the neu protooncogene in the mammary epithelium of transgenic mice induces metastatic disease. Mechanism of early dissemination and metastasis in Her2+ mammary cancer. Systemic spread is an early step in breast cancer. Nobre, A. R. et al. ZFP281 drives a mesenchymal-like dormancy program in early disseminated breast cancer cells that prevents metastatic outgrowth in the lung. Wang, J. et al. A synthetic metastatic niche reveals antitumor neutrophils drive breast cancer metastatic dormancy in the lungs. Di Martino, J. S. et al. A tumor-derived type III collagen-rich ECM niche regulates tumor cell dormancy. Balayan, V. & Guddati, A. K. Tumor dormancy: biologic and therapeutic implications. Barney, L. E. et al. Tumor cell-organized fibronectin maintenance of a dormant breast cancer population. Werner-Klein, M. et al. Interleukin-6 trans-signaling is a candidate mechanism to drive progression of human DCCs during clinical latency. Essential role of IL-6 in protection against H1N1 influenza virus by promoting neutrophil survival in the lung. Poli, V. et al. Interleukin-6 deficient mice are protected from bone loss caused by estrogen depletion. Dalla, E. et al. Lung-resident alveolar macrophages regulate the timing of breast cancer metastasis. et al. Inducible bronchus-associated lymphoid tissues (iBALT) serve as sites of B cell selection and maturation following influenza infection in mice. Rømer, A. M. A., Thorseth, M.-L. & Madsen, D. H. Immune modulatory properties of collagen in cancer. Albrengues, J. et al. Neutrophil extracellular traps produced during inflammation awaken dormant cancer cells in mice. The tumor necrosis factor alpha-induced protein 3 (TNFAIP3, A20) imposes a brake on antitumor activity of CD8 T cells. The ZFP36 family of RNA binding proteins regulates homeostatic and autoreactive T cell responses. Moore, M. J. et al. ZFP36 RNA-binding proteins restrain T cell activation and anti-viral immunity. Kutty, R. G. et al. Dual specificity phosphatase 5 Is essential for T cell survival. Andreatta, M. et al. A CD4+ T cell reference map delineates subtype-specific adaptation during acute and chronic viral infections. Smith, N. P. et al. Resident memory T cell development is gradual and shows AP-1 gene expression in mature cells. Lingel, H. et al. CTLA-4-mediated posttranslational modifications direct cytotoxic T-lymphocyte differentiation. Honey, K. Keeping CD4+ T cells under control. Ju, S. et al. Gadd45b and Gadd45g are important for anti-tumor immune responses. Yue, T. et al. SLFN2 protection of tRNAs from stress-induced cleavage is essential for T cell–mediated immunity. Matsuzawa, Y. et al. TNFAIP3 promotes survival of CD4 T cells by restricting MTOR and promoting autophagy. Dinnon, K. H. III et al. A mouse-adapted model of SARS-CoV-2 to test COVID-19 countermeasures. Leist, S. R. et al. A mouse-adapted SARS-CoV-2 induces acute lung injury and mortality in standard laboratory mice. Francescangeli, F. et al. COVID-19-induced Modifications in the tumor microenvironment: do they affect cancer reawakening and metastatic relapse? Rahimmanesh, I. et al. Cancer occurrence as the upcoming complications of COVID-19. Rubin, R. Baricitinib is first approved COVID-19 immunomodulatory treatment. Induction of mammary tumors by expression of polyomavirus middle T oncogene: a transgenic mouse model for metastatic disease. Casey, A. E., Laster, W. R. Jr & Ross, G. L. Sustained enhanced growth of carcinoma EO771 in C57 black mice. Davenport, B. J. et al. Phage-like particle vaccines are highly immunogenic and protect against pathogenic coronavirus infection and disease. Barbier-Torres, L. et al. Silencing hepatic MCJ attenuates non-alcoholic fatty liver disease (NAFLD) by increasing mitochondrial fatty acid oxidation. Yu, Y.-R. A. et al. A protocol for the comprehensive flow cytometric analysis of immune cells in normal and inflamed murine non-lymphoid tissues. Deleting the mitochondrial respiration negative regulator MCJ enhances the efficacy of CD8+ T cell adoptive therapies in pre-clinical studies. Valença-Pereira, F. et al. IL-6 enhances CD4 cell motility by sustaining mitochondrial Ca2+ through the noncanonical STAT3 pathway. Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Franzén, O., Gan, L.-M. & Björkegren, J. L. M. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data. clusterProfiler: an R package for comparing biological themes among gene clusters. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer, 2016). Kassambara, A. ggpubr: ‘ggplot2' Based Publication Ready Plots. Core mitochondrial genes are down-regulated during SARS-CoV-2 infection of rodent and human hosts. Korotkevich, G. et al. Fast gene set enrichment analysis. The nf-core framework for community-curated bioinformatics pipelines. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Persistent depots of influenza antigen fail to induce a cytotoxic CD8 T cell response. & Bell, S. Comparison of risk factor associations in UK Biobank against representative, general population based studies with conventional response rates: prospective cohort study and individual participant meta-analysis. Prospective study design and data analysis in UK Biobank. Ma, X., Long, L., Moon, S., Adamson, B. J. S. & Baxi, S. S. Comparison of Population Characteristics in Real-World Clinical Oncology Databases in the US: Flatiron Health, SEER, and NPCR. Birnbaum, B. et al. Model-assisted cohort selection with bias analysis for generating large-scale cohorts from the EHR for oncology research. Elliott, J. et al. COVID-19 mortality in the UK Biobank cohort: revisiting and evaluating risk factors. Ho, D., Imai, K., King, G. & Stuart, E. A. MatchIt: nonparametric preprocessing for parametric causal inference. Quan, H. et al. Coding algorithms for defining comorbidities in ICD-9-CM and ICD-10 administrative data. Grambsch, P. M. & Therneau, T. M. Proportional hazards tests and diagnostics based on weighted residuals. R: A Language and Environment for Statistical Computing. https://www.R-project.org (R Foundation for Statistical Computing, 2021). ; the Courtenay C. and Lucy Patten Davis Endowed Chair in Lung Cancer Research to J.D. ; the Kay Sutherland and Monika Weber Research Fund to J.D. ; and an Across the Finish Line grant from the University of Colorado School of Medicine to J.D., M.R. is supported by a National Institutes of Health (NIH)/National Cancer Institute (NCI) award R01CA260909. This work was also supported by NIH/NCI awards R01CA109182 and P30CA013330, the Samuel Waxman Cancer Research Foundation Tumor Dormancy Program to J.A.A.-G., who is also a Samuel Waxman Cancer Research Foundation Investigator. was supported by grant AWD-232405-SC from the Cancer League of Colorado. is supported by a NIGMS/NIH predoctoral fellowship (T32AR079114). F.G. acknowledges funding from the MRC Centre for Global Infectious Disease Analysis (reference MR/X020258/1), funded by the UK Medical Research Council (MRC); this UK-funded award is carried out in the frame of the Global Health EDCTP3 Joint Undertaking. H.M. is supported by the UK National Institute for Health Research's Comprehensive Biomedical Centre at University College London Hospitals. was funded by the Cancer Research Institute Irvington Postdoctoral Fellowship (CRI5290). was supported by NIH grants R01CA259635 and R01AG078814, DOD grant W81XWH-21-1-0128 and the Bill & Melinda Gates Foundation grant INV-046722. We thank the Rocky Mountain Regional VAMC Flow Core; the Biostatistics and Bioinformatics, Genomics, Human Immune Monitoring, and Flow Cytometry Shared Resources supported by National Cancer Institute grant P30CA046934 to the University of Colorado Cancer Center; and the Gates Institute Histology Core at the University of Colorado. This work was supported by the Alpine HPC system, which is jointly funded by the University of Colorado Boulder, the University of Colorado Anschutz, Colorado State University and the National Science Foundation (award 2201538). We thank G. Geno, W. Wang and M. Wang and J. Figura for guiding our studies with the Flatiron Health database; C. Swanton for connecting us with the team that did the UK Biobank analyses; and D. Merrick and P. Jedlicka for advice on the evaluation of DCC pathology. Present address: Blue Marble Space Institute of Science, Seattle, WA, USA These authors contributed equally: Shi B. Chia, Bryan J. Johnson These authors jointly supervised this work: Roel Vermeulen, Julio A. Aguirre-Ghiso, Mercedes Rincon, James DeGregori Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz Medical Campus, Aurora, CO, USA Shi B. Chia, Bryan J. Johnson, Marco De Dominici, Vadym Zaberezhnyy, Wolfgang E. Schleicher & James DeGregori Junxiao Hu, Dexiang Gao & James DeGregori University of Colorado Comprehensive Cancer Center, University of Colorado Anschutz Medical Campus, Aurora, CO, USA Junxiao Hu, Meher P. Boorgula, Varsha Sreekanth, Andrew Goodspeed, Dexiang Gao, James C. Costello, Mercedes Rincon & James DeGregori Felipe Valença-Pereira, Bennett Davenport, Thomas E. Morrison, Mercedes Rincon & James DeGregori Marc Chadeau-Hyam, Fernando Guntoro & Roel Vermeulen Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, London, United Kingdom MRC Centre for Global Infectious Disease Analysis, School of Public Health, Imperial College London, London, United Kingdom UConn Center On Aging, University of Connecticut School of Medicine and UConn Health, Farmington, CT, USA Department of Cell Biology, Cancer Dormancy Institute, Montefiore Einstein Comprehensive Cancer Center, Albert Einstein College of Medicine, Bronx, NY, USA Lucia Petriz-Otaño, Michael Papanicolaou & Julio A. Aguirre-Ghiso COVID-19 International Research Team, Medford, MA, USA Department of Surgery, McGowan Institute for Regenerative Medicine – Center for Space Biomedicine, University of Pittsburgh, Pittsburgh, PA, USA Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Oncology and Medicine Departments, The Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, MD, USA Van Andel Research Institute, Grand Rapids, MI, USA Center for Mitochondrial and Epigenomic Medicine, Division of Human Genetics, The Children's Hospital of Philadelphia, Philadelphia, PA, USA Julius Centre for Health Sciences and Primary Care, University Medical Centre, Utrecht University, Utrecht, The Netherlands Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar : analysed Flatiron Health data for patients with breast cancer and generated relevant figures; and edited the manuscript. : did mouse experiments involving SARS-CoV-2 MA10 infection; and performed the SARS-CoV-2 MA10 plaque assay. F.V.-P.: analysed experiments pertaining to MMTV-PyMT and assisted in tissue processing for flow cytometry; and did the isolation of DCC for RNA-seq, ex vivo CD8+ T cell cytotoxic assay, mitochondrial mass analyses and western blot for Dusp5. : did all the influenza PR8 infections and intraperitoneal injections of antibodies. : processed influenza-infected lungs and calculated the influenza viral load. : provided protocols and expertise for the immunofluorescence stains. : analyses of scRNA-seq data with generation of figures showing differential expression across cell types for innate immune and mitochondrial metabolism genes. : experimental guidance, particularly for scRNA-seq results. : interpreted scRNA-seq differential expression results for mitochondrial and oxidative phosphorylation-related genes and wrote Supplementary Note 1. J.A.A.-G. is a co-founder, advisory board member and equity holder in HiberCell, a Mount Sinai spin-off that develops cancer recurrence-prevention therapies. He consults for HiberCell and Astrin Biosciences, serves as chief mission advisor for the Samuel Waxman Cancer Research Foundation and has ownership interest in patent number WO2019191115A1/ EP-3775171-B1. are on the scientific advisory board for Mitotherapeutix. is a cofounder and chief scientific officer of OncoRx Insights. M.C.-H. holds shares in the O-SMOSE company and has no conflict of interest to disclose; consulting activities conducted by the company are independent of the present work. H.M. has consulted for Astra Zeneca relating to the use of monoclonal antibodies in the prevention and treatment of SARS-CoV-2 infection. All other authors declare no competing interests. Nature thanks Fangfang Zhou and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Weight change of MMTV-Her2 and WT mice post-IAV infection, error bars denote standard deviation (a) (n = 3/group). Total cell counts in BALF 3, 6, 9, 15 dpi (b)(n = 3/group). IAV lung viral load at 3, 6, 9, 15 dpi or for vehicle control mice (c)(n = 3/group). There were no significant differences between BALF cell counts or IAV viral load between MMTV-Her2 and WT mice. Lungs of an 18-week-old mouse 28 dpi with IAV and a naïve 9-month-old mouse stain with Her2 (green) (d). IF detection of Her2 and Ki67 in mammary glands (e) and quantification of Ki67+ ducts 9 dpi (f) (n = 4/group). Ct values from real-time RT-PCR for transgenic rat Erbb2 mRNA in CD45neg cells in circulating blood from PBS or IAV infected MMTV-Her2 mice at 9 dpi (g). IF for p53 and Ki67 (with DAPI) (h) and quantification (i) for Ki67 of EO771 tumor lesions (2×105 EO771 cells implanted – left; 1 × 106 EO771 cells implanted - right) (n = 2/group). Significance is determined by one-way ANOVA test. All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Heatmaps of genes for Interferon Alpha Response (a), Interferon Gamma Response (b), IL6/JAK/STAT3 Signaling (c), and Epithelial Mesenchymal Transition (d) pathways from RNA-seq for DCC comparing PBS to IAV at 9 dpi. Concentrations of IL-6 and IL-1β in BAL from MMTV-Her2 mice treated with PBS or infected with IAV at 3, 6, 9, 15 dpi (e, f). Concentration of IL-6 in supernatant of mouse tracheal epithelial cells (MTECs) treated with PBS or 24 h post IAV infection (g) (n = 3/group). Significance is determined by two tailed Student's t test. All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Kaplan-Meier curve for time to sacrifice due to primary tumor burden of MMTV-Her2 and IL-6 KO/MMTV-Her2 mice (n = 4/group) (a). Mice were sacrificed when primary tumors reached 2 cm in diameter; there was not a significant difference between the two groups. Quantification of PyMT+ lesions ( ≤ 5 cells) with at least one Ki67+PyMT+ cell in lungs 21 dpi with IAV (b) (n = 3/group, n = 4 for PBS). PyMT+ micrometastases (defined by lesions with an area <0.03 mm2) were quantified (c) (n = 7 WT, 4 IL6–/–); the WT groups are the same as in Fig. 1f, and are included to allow comparison with the IL-6 KO. EO771 cells (2 × 105) were implanted into WT or (n = 3) IL-6 KO C57BL/6 mice (n = 4), infected with IAV, and harvested at 12 dpi, stained with H&E, and tumor area and the numbers of lesions quantified (d and e). Significance is determined by one-way ANOVA test (b,c) or two tailed Student's t test (d,e). Size of mammospheres from MMTV-Her2 mice per well 72 h post daily treatment with either PBS (CTRL) or IL-6 (10 ng/mL) (f). Size of EO771 derived organoids per well 72 h post daily treatment with either PBS (CTRL) or IL-6 (10 ng/mL) (g) (n = 5/group). All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Counts of immune cells in bronchoalveolar lavage (BAL). Numbers of neutrophils, CD4+, CD8+, and B cells in BAL for mice treated with PBS or 3, 6, 9, 15dpi after IAV infection (a-d) (n = 3 per group. Representative IF stain of Her2 (green), B220 (magenta) (e), and GL7 (magenta) (f) in lungs of MMTV-Her2 mice treated with PBS or 28dpi post IAV infection (lung sections from 3 mice per group). Significance is determined by two-way ANOVA test. Representative Masson's Trichrome stain of lungs 28 dpi (lung sections from 3 mice per group) (l) and quantification (m) (n = 3/group) of the positively-stained area (blue) in parenchyma and iBALT for IAV infected WT and MMTV-Her2 mice. Significance is determined by two-way ANOVA test. All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Flow cytometric analysis of CD3+, CD4+ and CD8+ cells in lungs and spleens of WT mice 6 days post injection with 100 μg αCD4 antibody clone GK1.5 (a). The third column shows that the few CD3+ cells that are negative for CD8 persist following αCD4 treatment (demonstrating that the antibody was not simply preventing detection of CD4) (a). Quantification of Her2+ and Ki67+/Her2+ cells in lungs of MMTV-Her2 (IgG) and CD4-depleted MMTV-Her2 mice 9 dpi with IAV (c, d) (n = 4/group). For e-g, mice were injected with anti-Ly6G to deplete neutrophils or IgG control. Quantification of Her2+ cells 28 dpi with IAV is shown in (e) (n = 4/group). Significance is determined by two-tail Student's t test. All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Illustration in b created using BioRender (De Dominici, M., https://BioRender.com/i40c047; 2025). UMAP plot labelled by cell type (a, left). UMAP plots of each experimental group (a, right). Dot plot showing the expression of canonical marker genes used to identify the cell types (b). Quality control of the scRNAseq data set: distribution of UMIs, genes and mitochondrial transcripts within each sample pre-filtering (upper panel) and post-filtering (lower panel) with dashed lines indicating the filtering thresholds (c). Mice at 9 and 15 dpi with the replicates within each group highlighted (demonstrating reproducible patterns) (d). The top heatmap in (a) displays statistically significant changes in custom mitochondrial OXPHOS pathways, ranked by normalized enrichment score (NES) and identified through fGSEA analysis. Only pathways with a false discovery rate (FDR) < 0.25 are shown. The bottom heatmap shows individual log2(Fold-Change) values for custom innate immune genes across the experimental groups: Her2+IAV vs. HER2 + PBS, Her2+IAV vs. WT + IAV, and Her2+IAV+anti-CD4 vs. Her2+IAV. All genes are included, with statistical significance marked by * for adjusted p-value < 0.05 and # for raw p-value < 0.05. Note that for groups with depletion of CD4+ cells using anti-CD4 antibody, the residual cells in CD4+ effector, CD4+ memory and regulatory T-cell clusters expressed minimal CD4, and thus are not analyzed. Flow cytometric detection of Cxcr4 and western blotting for Dusp5 protein (b) (n = 4/group). Heatmap of top 20 differentially expressed genes from scRNAseq comparing CD4+ memory T-cells (c), CD8+ effector T-cells (d), CD8+ memory T-cells (e) in MMTV-Her2+IAV versus WT + IAV mice at 15dpi. Proportion of T-cell subtypes identified in scRNAseq (f). Mean fluorescence intensity of MitoTracker stain in CD4+ and CD8+ cells (g) (n = 4/group) from lungs of 15 dpi IAV infected WT or MMTV-Her2 mice. Significance is determined by two tailed Student's t test. CD4 and CD8 cell populations used to gate for MitoTracker staining (h) (n = 4/group). All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). The top-left heatmap illustrates statistically significant changes in custom mitochondrial OXPHOS pathways, ranked by normalized enrichment score (NES) and determined through fGSEA analysis. Only pathways with a false discovery rate (FDR) < 0.25 are displayed. The remaining heatmaps depict individual log2(Fold-Change) values for mitochondrial OXPHOS complex genes, comparing the experimental groups: Her2+IAV vs. HER2 + PBS, Her2+IAV vs. WT + IAV, and Her2+IAV+anti-CD4 vs. Her2+IAV. All genes are displayed, with statistical significance indicated by * for adjusted p-value < 0.05 and # for raw p-value < 0.05. For groups with depletion of CD4+ cells using anti-CD4 antibody, the residual cells in CD4+ effector, CD4+ memory and regulatory T-cell clusters exhibited minimal detection of CD4, and thus are not analyzed. Over-representation analysis heatmaps comparing the indicated samples across all cell types for interferon α (a) and γ (b) response enrichment at 15 dpi. GSEA analyses and heatmaps of top 20 differentially expressed genes comparing M1, M2, and migratory macrophages in MMTV-Her2+IAV versus WT + IAV mice 15dpi (c, d). Proportion of macrophage subtypes identified in scRNAseq samples (e). Concentrations of IFN-α, IFN-β, and IFN-γ in BAL at the indicated times post-IAV infection (b) (n = 3/group). Significance is determined by two-way ANOVA test (b). GSEA analyses of CD8+ memory T-cells in CD4-depleted MMTV-Her2+IAV versus MMTV-Her2+IAV mice at 15 dpi (c). All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). Ex vivo CD8+ cytotoxic assay where Her2+ mammary tumor cells were incubated with CD8+ cells enriched from lungs of WT, MMTV-Her2 mice, and MMTV-Her2 mice with CD4 depletion at 15 dpi (a) (n = 3/group). Ex vivo CD8+ cytotoxic assay where PyMT-expressing MET1 cells were incubated with CD8+ cells enriched from lungs of MMTV-Her2 mice with or without CD4 depletion 15 dpi (b) (n = 3/group). Concentrations of IFN-α, IFN-β, IFN-γ, IL-1β, and IL-6 in BAL at the indicated times post-MA10 infection (c, d). Infectious SARS-CoV-2 MA10 burden 3dpi with MA10 in lungs as quantified by plaque assay (f) (n = 6/group). Significance is determined by one-way ANOVA test (a,c,d) or two tailed Student's t test (b,e). All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots). IF stain of Her2 at 28 dpi (a) for C57BL6/J MMTV-Her2 mouse lungs at 28 dpi with MA10 SARS-CoV-2 or vehicle (mock). Significance is determined by one-way ANOVA test (b, left) or two tailed Student's t test (others). Summary representation of the selection, exclusion, and matching criteria implemented for UK Biobank (d). Summary representation of the selection, exclusion, and matching criteria implemented for the Flatiron Health Database (e). Model - pulmonary virus-dependent increases in IL-6 contribute to the awakening and expansion of dormant mesenchymal-like breast cancer cells that switch to a mixed epithelial/mesenchymal-like phenotype in lungs in the early phase of viral infection. Virus-dependent awakening and expansion of DCC in the lungs increases the risks of metastatic progression (f). All box-and-whisker plots are presented as maximum value (top line), median value (middle line), minimum value (bottom line) with all data points shown (dots).Illustration in f created using BioRender (De Dominici, M., https://BioRender.com/yxpclmg; 2025). Significant differentially expressed genes between DCCs at 9 dpi versus PBS controls. Significant differentially expressed genes in the scRNA-seq dataset comparing CD4 effector cells from MMTV-Her2 mice versus wild type. All significant GSEA results from the scRNA-seq experiments. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Chia, S.B., Johnson, B.J., Hu, J. et al. Respiratory viral infections awaken metastatic breast cancer cells in lungs. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09308-0'>Repurposing haemoproteins for asymmetric metal-catalysed H atom transfer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 15:11:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Transition metal–hydrides have been widely exploited in catalysis for the hydrofunctionalization of unsaturated moieties, including carbonyls, alkenes and alkynes1. To complement heterolytic metal–hydride bond cleavage, metal–hydride hydrogen atom transfer (MHAT) has recently gained attention, as a promising strategy for radical hydrofunctionalization of unactivated alkenes2, thus enabling late-stage diversification of complex molecules3,4. However, owing to the weak interactions between the prochiral organic radical and the enantiopure catalyst5, asymmetric MHAT6 remains challenging. Here we show that cytochrome P450 enzymes (CYPs) can be repurposed to catalyse asymmetric MHAT, a new-to-nature reaction. Directed evolution of P450BM3 yielded a triple mutant that catalyses MHAT radical cyclization of unactivated alkenes, producing diverse cyclic compounds—including pyrrolidines and piperidines—with up to 98:2 enantiomeric ratio under aerobic whole-cell conditions. Apart from electron-deficient alkenes, alternative radical acceptors—including hydrazones, oximes and nitriles—were converted by repurposed P450BM3 to enantioenriched cyclization products. Mechanistic investigations support an MHAT mechanism proceeding by homolytic cleavage of a fleeting iron(III)–hydride species2,6. Starting from CYP119, directed evolution afforded a stereocomplementary MHATase, highlighting the potential of repurposed CYPs for MHAT biocatalysis. Our study highlights the prospect of integrating homolytic metal–hydride reactivity into metalloenzymes, thus expanding the scope of asymmetric radical biocatalysis. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access $199.00 per year only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data that support the findings in this study are available in this paper and the Supplementary Information. Source data are provided with this paper. Maurizio Peruzzini, R. P. Recent Advances in Hydride Chemistry 1st edn (Elsevier Science, 2002). Crossley, S. W. M., Obradors, C., Martinez, R. M. & Shenvi, R. A. Mn-, Fe-, and Co-catalyzed radical hydrofunctionalizations of olefins. Google Scholar Wu, J. & Ma, Z. Metal-hydride hydrogen atom transfer (MHAT) reactions in natural product synthesis. Google Scholar Green, S. A. et al. The high chemofidelity of metal-catalyzed hydrogen atom transfer. Google Scholar Sibi, M. P., Manyem, S. & Zimmerman, J. Enantioselective radical processes. Google Scholar Shevick, S. L. et al. Catalytic hydrogen atom transfer to alkenes: a roadmap for metal hydrides and radicals. Google Scholar Schilter, D., Camara, J. M., Huynh, M. T., Hammes-Schiffer, S. & Rauchfuss, T. B. Hydrogenase enzymes and their synthetic models: the role of metal hydrides. Google Scholar Lee, W.-C. C., Wang, D.-S., Zhu, Y. & Zhang, X. P. Iron(III)-based metalloradical catalysis for asymmetric cyclopropanation via a stepwise radical mechanism. Google Scholar Mukaiyama, T. et al. Oxidation-reduction hydration of olefins with molecular oxygen and 2-propanol catalyzed by bis(acetylacetonato)cobalt(II). Google Scholar Waser, J. & Carreira, E. M. Convenient synthesis of alkylhydrazides by the cobalt-catalyzed hydrohydrazination reaction of olefins and azodicarboxylates. Google Scholar Choi, J., Tang, L. & Norton, J. R. Kinetics of hydrogen atom transfer from (η5-C5H5)Cr(CO)3H to various olefins: influence of olefin structure. Google Scholar Ishikawa, H. et al. Total synthesis of vinblastine, vincristine, related natural products, and key structural analogues. Google Scholar Ma, X. S. & Herzon, S. B. Intermolecular hydropyridylation of unactivated alkenes. Google Scholar Lo, J. C., Yabe, Y. & Baran, P. S. A practical and catalytic reductive olefin coupling. Google Scholar Discolo, C. A., Touney, E. E. & Pronin, S. V. Catalytic asymmetric radical-polar crossover hydroalkoxylation. Google Scholar Ebisawa, K. et al. Catalyst- and silane-controlled enantioselective hydrofunctionalization of alkenes by cobalt-catalyzed hydrogen atom transfer and radical-polar crossover. Google Scholar Zhang, G. & Zhang, Q. Cobalt-catalyzed HAT reaction for asymmetric hydrofunctionalization of alkenes and nucleophiles. Chem Catal. Google Scholar et al. Mimicking hydrogen-atom-transfer-like reactivity in copper-catalysed olefin hydrofunctionalization. Google Scholar Buzsaki, S. R. et al. Fe/thiol cooperative hydrogen atom transfer olefin hydrogenation: mechanistic insights that inform enantioselective catalysis. Google Scholar Hammer, S. C., Marjanovic, A., Dominicus, J. M., Nestl, B. M. & Hauer, B. Squalene hopene cyclases are protonases for stereoselective Brønsted acid catalysis. Google Scholar Gergel, S. et al. Engineered cytochrome P450 for direct arylalkene-to-ketone oxidation via highly reactive carbocation intermediates. Google Scholar Coelho, P. S., Brustad, E. M., Kannan, A. & Arnold, F. H. Olefin cyclopropanation via carbene transfer catalyzed by engineered cytochrome P450 enzymes. Google Scholar Vargas, D. A. et al. Biocatalytic strategy for the construction of sp3-rich polycyclic compounds from directed evolution and computational modelling. Google Scholar Bruffy, S. K. et al. Biocatalytic asymmetric aldol addition into unactivated ketones. Google Scholar Nakano, Y., Biegasiewicz, K. F. & Hyster, T. K. Biocatalytic hydrogen atom transfer: an invigorating approach to free-radical reactions. Google Scholar Van Stappen, C. et al. Designing artificial metalloenzymes by tuning of the environment beyond the primary coordination sphere. Google Scholar Chen, K. & Arnold, F. H. Engineering new catalytic activities in enzymes. Google Scholar Emmanuel, M. A. et al. Photobiocatalytic strategies for organic synthesis. Google Scholar Zhou, Q., Chin, M., Fu, Y., Liu, P. & Yang, Y. Stereodivergent atom-transfer radical cyclization by engineered cytochromes P450. Google Scholar Rui, J. et al. Directed evolution of nonheme iron enzymes to access abiological radical-relay C(sp3)-H azidation. Google Scholar Zetzsche, L. E. et al. Biocatalytic oxidative cross-coupling reactions for biaryl bond formation. Google Scholar Xu, Y. et al. A light-driven enzymatic enantioselective radical acylation. Google Scholar Chen, R., Kayrouz, C. S., McAmis, E., Clark, D. S. & Hartwig, J. F. Carbonic anhydrase variants catalyze the reduction of dialkyl ketones with high enantioselectivity. Google Scholar Ji, P., Park, J., Gu, Y., Clark, D. S. & Hartwig, J. F. Abiotic reduction of ketones with silanes catalysed by carbonic anhydrase through an enzymatic zinc hydride. Google Scholar Zhang, X. et al. Repurposing myoglobin into an abiological asymmetric ketoreductase. Google Scholar Wan, Z. et al. Stereoconvergent reduction of alkenes using a repurposed iron-based dioxygenase. Li, J., Kumar, A. & Lewis, J. C. Non-native intramolecular radical cyclization catalyzed by a B12-dependent enzyme. Wang, B. et al. Repurposing iron- and 2-oxoglutarate-dependent oxygenases to catalyze olefin hydration. Chen, D. et al. An evolved artificial radical cyclase enables the construction of bicyclic terpenoid scaffolds via an H-atom transfer pathway. Fansher, D. J., Besna, J. N., Fendri, A. & Pelletier, J. N. Choose your own adventure: a comprehensive database of reactions catalyzed by cytochrome P450 BM3 variants. ACS Catal. Kim, D., Rahaman, S. M. W., Mercado, B. Q., Poli, R. & Holland, P. L. Roles of iron complexes in catalytic radical alkene cross-coupling: a computational and mechanistic study. Li, H. Y., Darwish, K. & Poulos, T. L. Characterization of recombinant Bacillus megaterium cytochrome P-450 BM-3 and its two functional domains. Gröger, H., Gallou, F. & Lipshutz, B. H. Where chemocatalysis meets biocatalysis: in water. Andersson, M. P., Gallou, F., Klumphu, P., Takale, B. S. & Lipshutz, B. H. Structure of nanoparticles derived from designer surfactant TPGS-750-M in water, as used in organic synthesis. Turner, O. J., Murphy, J. A., Hirst, D. J. & Talbot, E. P. A. Hydrogen atom transfer-mediated cyclisations of nitriles. Saladrigas, M., Loren, G., Bonjoch, J. & Bradshaw, B. Hydrogen atom transfer (HAT)-triggered iron-catalyzed intra- and intermolecular coupling of alkenes with hydrazones: access to complex amines. ACS Catal. Drienovská, I., Mayer, C., Dulson, C. & Roelfes, G. A designer enzyme for hydrazone and oxime formation featuring an unnatural catalytic aniline residue. Kato, S., Abe, M., Gröger, H. & Hayashi, T. Reconstitution of myoglobin with iron porphycene generates an artificial aldoxime dehydratase with expanded catalytic activities. ACS Catal. Chen, K., Wang, Z., Ding, K., Chen, Y. & Asano, Y. Recent progress on discovery and research of aldoxime dehydratases. Green Synth. Mohamed, H., Ghith, A. & Bell, S. G. The binding of nitrogen-donor ligands to the ferric and ferrous forms of cytochrome P450 enzymes. Omura, T. & Sato, R. The carbon monoxide-binding pigment of liver microsomes: I. Evidence for its hemoprotein nature. Zheng, J., Kwak, K., Xie, J. & Fayer, M. D. Ultrafast carbon-carbon single-bond rotational isomerization in room-temperature solution. Yan, M., Lo, J. C., Edwards, J. T. & Baran, P. S. Radicals: reactive intermediates with translational potential. Koo, L. S., Immoos, C. E., Cohen, M. S., Farmer, P. J. & Ortiz de Montellano, P. R. Enhanced electron transfer and lauric acid hydroxylation by site-directed mutagenesis of CYP119. Download references thanks the NCCR Catalysis (grant no. 180544), a National Centre of Competence in Research funded by the Swiss National Science Foundation. Additional funding was provided by the NCCR Molecular Systems Engineering (grant no. We acknowledge T. Weymuth and M. Reiher for help with the computational studies. We acknowledge J. Stropp and D. Klose for their assistance with preliminary electron paramagnetic resonance studies. We acknowledge T. Kardashliev and Z. Zou for providing original plasmids of various haemoproteins. We acknowledge J. Zurflüh for the help with the GC-MS analysis. These authors contributed equally: Xiang Zhang, Dongping Chen Department of Chemistry, University of Basel, Basel, Switzerland Xiang Zhang, Dongping Chen, María Álvarez & Thomas R. Ward National Center of Competence in Research ‘Molecular Systems Engineering', Basel, Switzerland Xiang Zhang, Dongping Chen & Thomas R. Ward Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and D.C. conceived and designed the study. and D.C. contributed equally to the synthesis of the substrates and products in Fig. 3, directed evolution of both P450BM3 and CYP119, and the mechanistic study in Fig. 5 and Supplementary Information. and D.C. performed the catalytic experiment; X.Z. recorded the data. contributed to the synthesis of substrates and products in Fig. 4 as well as the double-blind experiments. and D.C. analysed the data. and D.C. wrote the paper. All authors gave approval for the final version of the paper. Correspondence to Thomas R. Ward. The authors declare no competing interests. Nature thanks Elaine O'Reilly and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Schematic representations of active sites in natural hydrogenases (a) and selected examples illustrating the application of MHAT in total synthesis (b). a, UV-Vis spectra of purified P450BM3_LQQ (black), purified P450BM3_LQQ with Na2S2O4 before (red) and after purging with CO (blue), and purified P450BM3_LQQ with PhSiH3 before (green) and after purging with CO (purple). (Inset) Expanded view of the region between 450 and 700 nm. All spectra were collected under anaerobic conditions. b, Investigation of the involvement of P450-FeII in the radical cyclization of diene 1. (Upper) Standard reaction conditions with addition of excess Na2S2O4 did not yield any pyrrolidine 2, in contrast to standard conditions without the additive. (Lower) Additional experiments were carried out in a glove box without any additive or with air or K3Fe(CN)6 as the additive. The clock icon was created with BioRender.com. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zhang, X., Chen, D., Álvarez, M. et al. Repurposing haemoproteins for asymmetric metal-catalysed H atom transfer. Download citation Received: 28 October 2024 Accepted: 19 June 2025 Published: 30 July 2025 DOI: https://doi.org/10.1038/s41586-025-09308-0 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1476-4687 (online) ISSN 0028-0836 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09116-6'>Towards more effective nature-based climate solutions in global forests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 15:06:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 643, pages 1214–1222 (2025)Cite this article Terrestrial ecosystems could contribute to climate mitigation through nature-based climate solutions (NbCS), which aim to reduce ecosystem greenhouse gas emissions and/or increase ecosystem carbon storage. Forests have the largest potential for NbCS, aligned with broader sustainability benefits, but—unfortunately—a broad body of literature has revealed widespread problems in forest NbCS projects and protocols that undermine the climate mitigation of forest carbon credits and hamper efforts to reach global net zero. Therefore, there is a need to bring better science and policy to improve NbCS climate mitigation outcomes going forward. Here we synthesize challenges to crediting forest NbCS and offer guidance and key next steps to make improvements in the implementation of these strategies immediately and in the near-term. We structure our Perspective around four key components of rigorous forest NbCS, illuminating key science and policy considerations and providing solutions to improve rigour. Finally, we outline a ‘contribution approach' to support rigorous forest NbCS that is an alternative funding mechanism that disallows compensation or offsetting claims. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Friedlingstein, P. et al. Global Carbon Budget 2022. Earth Syst. Griscom, B. W. et al. Natural climate solutions. Natl Acad. This paper is a seminal global synthesis on the potential of nature-based climate solutions to contribute to climate mitigation. Nolan, C. J., Field, C. B. & Mach, K. J. Constraints and enablers for increasing carbon storage in the terrestrial biosphere. Seddon, N. Harnessing the potential of nature-based solutions for mitigating and adapting to climate change. Ellis, P. W. et al. The principles of natural climate solutions. This review synthesizes the wide array of dimensions for high-quality nature-based climate solutions, including co-benefits and avoiding social harm. Buma, B. et al. Expert review of the science underlying nature-based climate solutions. Intergovernmental Panel on Climate Change (IPCC). Climate Change 2022: Mitigation of Climate Change. Contribution of Working Group III to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Shukla, P. R. et al.) (Cambridge Univ. Maguire, P. et al. A Green Growth Spurt: State of Forest Carbon Finance 2021 (Forest Trends' Ecosystem Marketplace, 2021). Donofrio, S., Procton, A., Weatherer, L., Calderon, C. & Bennett, G. All in on Climate: The Role of Carbon Credits in Corporate Climate Strategies (Forest Trends' Ecosystem Marketplace, 2023). Hale, T. et al. Assessing the rapidly-emerging landscape of net zero targets. Seddon, N. et al. Getting the message right on nature‐based solutions to climate change. Change Biol. Policy brief: The California Air Resources Board's U.S. Forest offset protocol underestimates leakage (2019). Hurteau, M. D., Hungate, B. & Koch, G. W. Accounting for risk in valuing forest carbon offsets. Carbon Balance Manag. Hurteau, M. D., Hungate, B. A., Koch, G. W., North, M. P. & Smith, G. R. Aligning ecology and markets in the forest carbon cycle. Anderegg, W. R. et al. Climate-driven risks to the climate mitigation potential of forests. This review paper synthesizes the increasing climate risks to durability of nature-based climate solutions in forests and highlights key gaps between present science and policy. Badgley, G. et al. California's forest carbon offsets buffer pool is severely undercapitalized. Badgley, G. et al. Systematic over‐crediting in California's forest carbon offsets program. Change Biol. Haya, B. K. et al. Quality Assessment of REDD+ Carbon Credit Projects (Berkeley Carbon Trading Project, 2023). Haya, B. K. et al. Comprehensive review of carbon quantification by improved forest management offset protocols. Stapp, J. et al. Little evidence of management change in California's forest offset program. This study demonstrates that a notable compliance forest offset programme shows little to no evidence of additionality in changing practices. West, T. A. P., Börner, J., Sills, E. O. & Kontoleon, A. Overstated carbon emission reductions from voluntary REDD+ projects in the Brazilian Amazon. West, T. A. P. et al. Action needed to make carbon offsets from forest conservation work for climate change mitigation. This landmark study documents very little additionality owing to poor baselines in several notable tropical Reducing Emissions from Deforestation and Forest Degradation (REDD) projects. Coffield, S. R. et al. Using remote sensing to quantify the additional climate benefits of California forest carbon offset projects. Change Biol. Randazzo, N. A., Gordon, D. R., & Hamburg, S. P. Improved assessment of baseline and additionality for forest carbon crediting. Guizar‐Coutiño, A., Jones, J. P. G., Balmford, A., Carmenta, R. & Coomes, D. A. A global evaluation of the effectiveness of voluntary REDD+ projects at reducing deforestation and degradation in the moist tropics. Macintosh, A. et al. Australian human-induced native forest regeneration carbon offset projects have limited impact on changes in woody vegetation cover and carbon removals. This study observes little additionality in a notable forest regeneration carbon offset programme using time-series satellite data. Live Carbon Prices Today. Novick, K. A. et al. Informing nature‐based climate solutions for the United States with the best‐available science. Change Biol. Seddon, N., Turner, B., Berry, P., Chausson, A. & Girardin, C. A. J. Grounding nature-based climate solutions in sound biodiversity science. Cook-Patton, S. C. et al. Protect, manage and then restore lands for climate mitigation. Novick, K. A. et al. We need a solid scientific basis for nature-based climate solutions in the United States. Pande, R. Fixing forest carbon credits. Chang, C. H. et al. Global evidence of human well-being and biodiversity impacts of natural climate solutions. Bonan, G. B. Forests and climate change: forcings, feedbacks, and the climate benefits of forests. Jackson, R. B. et al. Protecting climate with forests. Williams, C. A., Gu, H. & Jiao, T. Climate impacts of U.S. forest loss span net warming to net cooling. Bright, R. M., Zhao, K., Jackson, R. B. & Cherubini, F. Quantifying surface albedo and other direct biogeophysical climate forcings of forestry activities. Change Biol. Hasler, N. et al. Accounting for albedo change to identify climate-positive tree cover restoration. This study provides a global analysis of where albedo change from restoration or reforestation activities greatly alters the net climate impact. Randerson, J. T. et al. The impact of boreal forest fire on climate warming. Kuusinen, N. et al. Measured and modelled albedos in Finnish boreal forest stands of different species, structure and understory. Amiro, B. D. et al. The effect of post-fire stand age on the boreal forest energy balance. Rohatyn, S., Rotenberg, E., Tatarinov, F., Carmel, Y. & Yakir, D. Large variations in afforestation-related climate cooling and warming effects across short distances. Weber, J. et al. Chemistry-albedo feedbacks offset up to a third of forestation's CO2 removal benefits. Teuling, A. J. et al. Observational evidence for cloud cover enhancement over western European forests. Seidl, R. et al. Forest disturbances under climate change. Coop, J. D. et al. Wildfire-driven forest conversion in western North American landscapes. Davis, K. T. et al. Wildfires and climate change push low-elevation forests across a critical climate threshold for tree regeneration. Joos, F. et al. Carbon dioxide and climate impulse response functions for the computation of greenhouse gas metrics: a multi-model analysis. Archer, D. et al. Atmospheric lifetime of fossil fuel carbon dioxide. Earth Planet. Matthews, H. D. & Caldeira, K. Stabilizing climate requires near‐zero emissions. Allen, M. R. et al. Warming caused by cumulative carbon emissions towards the trillionth tonne. Matthews, H. D., Gillett, N. P., Stott, P. A. & Zickfeld, K. The proportionality of global warming to cumulative carbon emissions. Meinshausen, M. et al. Greenhouse-gas emission targets for limiting global warming to 2 °C. Zickfeld, K., Eby, M., Matthews, H. D. & Weaver, A. J. Setting cumulative emissions targets to reduce the risk of dangerous climate change. Allen, M. R. et al. Net zero: science, origins, and implications. Matthews, H. D. et al. Temporary nature-based carbon removal can lower peak warming in a well-below 2 °C scenario. Cullenward, D. A framework for assessing the climate value of temporary carbon storage. Carbon Market Watch https://carbonmarketwatch.org/publications/a-framework-for-assessing-the-climate-value-of-temporary-carbon-storage/ (2023). Fankhauser, S. What next on net zero? One Earth 4, 1520–1522 (2021). Cullenward, D., Badgley, G. & Chay, F. Carbon offsets are incompatible with the Paris Agreement. One Earth 6, 1085–1088 (2023). Oldfield, E. E. et al. Crediting agricultural soil carbon sequestration. Wu, C. et al. Uncertainty in US forest carbon storage potential due to climate risks. This study demonstrates substantially increasing climate-driven disturbance risks to durability for US forests and how it is likely to affect offset programmes. Wu, C. et al. Carbon reversal risks from climate-sensitive disturbances in US forests. In AGU Fall Meeting Abstracts Vol. Anderegg, W. R. et al. Future climate risks from stress, insects and fire across US forests. Anderegg, W. R. L., Trugman, A. T., Vargas G., G., Wu, C. & Yang, L. Current forest carbon offset buffer pool contributions do not adequately insure against disturbance‐driven carbon losses. Change Biol. Kriebel, D. et al. The precautionary principle in environmental science. Health Perspect. Schneider, S. H. & Lane, J. Dangers and thresholds in climate change and the implications for justice. In Fairness in Adaptation to Climate Change (eds Adger, W. N., Paavola, J., Huq, S. & Mace M. McFarland, B. J. Carbon reduction projects and the concept of additionality. Bushnell, J. The Economics of Carbon Offsets, Working Paper 16305 (National Bureau of Economic Research, 2010). McFarland, B. Origins, Development and Potential of the International REDD Market. Master's thesis, American Univ. West, T. A. P., Bomfim, B. & Haya, B. K. Methodological issues with deforestation baselines compromise the integrity of carbon offsets from REDD+. California Air Resources Board Offset Credit Regulatory Conformance and Invalidation Guidance (California Air Resources Board, 2015). Seyller, C. et al. The ‘virtual economy' of REDD+ projects: does private certification of REDD+ projects ensure their environmental integrity? Delacote, P., Le Velly, G. & Simonet, G. Revisiting the location bias and additionality of REDD+ projects: the role of project proponents status and certification. Energy Econ. van Kooten, G. C., Bogle, T. N. & de Vries, F. P. Forest carbon offsets revisited: shedding light on darkwoods. Xu, L. et al. Changes in global terrestrial live biomass over the 21st century. Wang, J. A., Baccini, A., Farina, M., Randerson, J. T. & Friedl, M. A. Disturbance suppresses the aboveground carbon sink in North American boreal forests. Nepstad, D. et al. Re-framing REDD+ (Amazon Environmental Research Institute (IPAM), 2012). Busch, J. et al. Structuring economic incentives to reduce emissions from deforestation within Indonesia. Teo, H. C. et al. Uncertainties in deforestation emission baseline methodologies and implications for carbon markets. Nepstad, D. et al. Slowing Amazon deforestation through public policy and interventions in beef and soy supply chains. Hansen, M. C. et al. High-resolution global maps of 21st-century forest cover change. Mertz, O. et al. Uncertainty in establishing forest reference levels and predicting future forest-based carbon stocks for REDD+. Land Use Sci. Schneider, L., Fuessler, J., Oberpriller, Q. & Spalding-Fecher, R. The ICVCM approval of three REDD methodologies presents risks to the integrity of the initiative. Öko-Institut https://www.oeko.de/blog/the-icvcm-approval-of-three-redd-methodologies-presents-risks-to-the-integrity-of-the-initiative (2024). Re: Comments on California's proposed REDD program and linkage with Acre, Brazil (2016). Teo, H. C., Sarira, T. V., Tan, A. R. P., Cheng, Y. & Koh, L. P. Charting the future of high forest low deforestation jurisdictions. Gan, J. Measuring transnational leakage of forest conservation. Ingalls, M. L., Meyfroidt, P., To, P. X., Kenney-Lazar, M. & Epprecht, M. The transboundary displacement of deforestation under REDD+: problematic intersections between the trade of forest-risk commodities and land grabbing in the Mekong region. & McCarney, G. Avoiding carbon leakage from nature-based offsets by design. One Earth 6, 790–802 (2023). Pan, W., Kim, M.-K., Ning, Z. & Yang, H. Carbon leakage in energy/forest sectors and climate policy implications using meta-analysis. Curtis, P. G., Slay, C. M., Harris, N. L., Tyukavina, A. & Hansen, M. C. Classifying drivers of global forest loss. Wang, J. A., Randerson, J. T., Goulden, M. L., Knight, C. A. & Battles, J. J. Losses of tree cover in California driven by increasing fire disturbance and climate stress. AGU Adv. European Union. Proposal for a DIRECTIVE OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL on substantiation and communication of explicit environmental claims (Green Claims Directive). State of California. AB-1305 Voluntary carbon market disclosures. California Legislative Information https://leginfo.legislature.ca.gov/faces/billNavClient.xhtml?bill_id=202320240AB1305 (2023). Isometric Standard. Validation and verification process. Isometric https://registry.isometric.com/standard#validation-and-verification-process (2025). Duflo, E., Greenstone, M., Pande, R. & Ryan, N. Truth-telling by third-party auditors and the response of polluting firms: experimental evidence from India. Trouwloon, D., Streck, C., Chagas, T. & Martinus, G. Understanding the use of carbon credits by companies: a review of the defining elements of corporate climate claims. Ivanova, I. Delta faces lawsuit alleging its ‘carbon-neutral' claim is greenwashing. CBS News https://www.cbsnews.com/news/delta-lawsuit-cabon-neutral-greenwashing-carbon-offsets/ (2023). Airlines Facing ‘Greenwashing' Litigation on Three Continents. Jones Day https://www.jonesday.com/en/insights/2023/08/ airlines-facing-greenwashing-litigation-on-three-continents (2023). Trencher, G., Nick, S., Carlson, J. & Johnson, M. Demand for low-quality offsets by major companies undermines climate integrity of the voluntary carbon market. Broekhoff, D., Gillenwater, M., Colbert-Sangree, T. & Cage, P. Securing Climate Benefit: A Guide to Using Carbon Offsets. Giles, C. What carbon offsets tell us about why environmental programs fail. The Regulatory Review https://www.theregreview.org/2023/12/18/ giles-what-carbon-offsets-tell-us-about- why-environmental-programs-fail/ (2023). Blanchard, L. et al. Funding forests' climate potential without carbon offsets. One Earth 7, 1147–1150 (2024). Kirschbaum, M. U. F. Temporary carbon sequestration cannot prevent climate change. European Union. Unfair commercial practices directive. European Commission https://commission.europa.eu/law/ law-topic/consumer-protection-law/unfair- commercial-practices-and-price-indication /unfair-commercial-practices-directive_en (2021). SBTi Corporate Net-Zero Standard. Version 2.0 - Initial Consultation Draft with Narrative. Science Based Targets Initiative https://sciencebasedtargets.org/ consultations/cnzs-v2-initialdraft (2025). Benson, S. et al. Above and Beyond: An SBTi Report on the Design and Implementation of Beyond Value Chain Mitigation (BVCM) Version 1.0. Science Based Targets Initiative https://sciencebasedtargets.org/resources/files/Above-and-Beyond-Report-on-BVCM.pdf (2024). Hewlett, O., Magrath, D., Höglund, R., Hutton, W. & Stanton, I. Funding Beyond Value Chain Mitigation: Step by Step Guidance for Organisations Taking Responsibility for Their Emissions Version 1.0. Gold Standard https://www.goldstandard.org/publications/ funding-beyond-value-chain-mitigation (2024). Moore, F. C. et al. Synthesis of evidence yields high social cost of carbon due to structural model variation and uncertainties. Rennert, K. et al. Comprehensive evidence implies a higher social cost of CO2. Tol, R. S. J. Social cost of carbon estimates have increased over time. Rogelj, J. et al. Scenarios towards limiting global mean temperature increase below 1.5 °C. Prest, B. C., Rennels, L., Errickson, F. & Anthoff, D. Equity weighting increases the social cost of carbon. Lou, J., Hultman, N., Patwardhan, A. & Mintzer, I. Corporate motivations and co-benefit valuation in private climate finance investments through voluntary carbon markets. Iovino, R., Testa, F. & Iraldo, F. Do consumers understand what different green claims actually mean? An experimental approach in Italy. Environmental Claims in Advertising. Qualitative Research Report. American Student Assistance https://www.asa.org.uk/static/6830187f-cc56-4433-b53a4ab0fa8770fc/CCE- Consumer-Understanding-Research-2022Final-090922.pdf (2022). Berrone, P., Fosfuri, A. & Gelabert, L. Does greenwashing pay off? Understanding the relationship between environmental actions and environmental legitimacy. Höglund, R. & Mitchell-Larson, E. Bridging the ambition gap: a framework for scaling corporate funds for carbon removal and wider climate action. Carbon Gap https://carbongap.org/wp-content/ uploads/2022/11/Ambition_Gap_Report_Nov22.pdf (2022). Download references This article is the product of the Wilkes Center for Climate Science and Policy at the University of Utah's Working Group on Rigorous Nature-based Climate Solutions in Forests. We thank S. Cook-Patton, who was a co-author on the initial manuscript and contributed to idea generation, writing and revisions, but is not listed in the final draft owing to differences in opinions about final recommendations. We thank K. Riley and P. Ellis for providing feedback on this work. acknowledges support from US National Science Foundation grants 2003017, 2044937 and 2330582 and Alan T. Waterman Award IOS-2325700. Wilkes Center for Climate Science and Policy, University of Utah, Salt Lake City, UT, USA William R. L. Anderegg, Libby Blanchard, Meng Liu & Linqing Yang School of Biological Sciences, University of Utah, Salt Lake City, UT, USA William R. L. Anderegg, Libby Blanchard, Meng Liu, Jonathan A. Wang & Linqing Yang World Wildlife Fund, Washington, DC, USA Christa Anderson CarbonPlan, San Francisco, CA, USA Kleinman Center for Energy Policy, University of Pennsylvania, Philadelphia, PA, USA Department of Earth and Ocean Sciences, University of North Carolina Wilmington, Wilmington, NC, USA Peng Gao Department of Earth System Science, University of California, Irvine, Irvine, CA, USA Michael L. Goulden & James Randerson Goldman School of Public Policy, University of California, Berkeley, Berkeley, CA, USA Lawrence Berkeley National Laboratory, Berkeley, CA, USA Jennifer A. Holm Department of Biology, University of New Mexico, Albuquerque, NM, USA Matthew D. Hurteau Department of Geography, University of British Columbia, Vancouver, British Columbia, Canada Marysa Lague O'Neill School of Public and Environmental Affairs, Indiana University, Bloomington, IN, USA Kimberly A. Novick Department of Geography, University of California, Santa Barbara, Santa Barbara, CA, USA Anna T. Trugman Graduate School of Geography, Clark University, Worcester, MA, USA Christopher A. Williams Department of Earth System Science, Ministry of Education Key Laboratory for Earth System Modeling, Institute for Global Change Studies, Tsinghua University, Beijing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the first draft of the manuscript. W.R.L.A., L.B., C.A., G.B., D.C., P.G., M.L.G., B.H., J.A.H., M.D.H., M. Lague, M. Liu, K.A.N., J.R., A.T.T., J.A.W., C.A.W., C.W. contributed to idea generation, literature synthesis, writing of several drafts and revisions. Correspondence to William R. L. Anderegg. D.C. previously consulted for Isometric and is a member of the Milkywire Climate Transformation Fund Advisory Board, the UNFCCC Article 6.4 Mechanism Methodological Expert Panel and California's Independent Emissions Market Advisory Committee. The other authors declare no competing interests. Nature thanks Jiehong Lou and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Anderegg, W.R.L., Blanchard, L., Anderson, C. et al. Towards more effective nature-based climate solutions in global forests. Download citation Received: 12 July 2024 Published: 30 July 2025 Issue Date: 31 July 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1476-4687 (online) ISSN 0028-0836 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/hidden-greenland-lake-punches-through-giant-blocks-of-ice-in-extremely/'>Enormous Fountain of Water Bursts through Greenland Ice, Surprising Geoscientists</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 13:29:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Hidden Lake Bursts through Greenland Ice in ‘Extremely Surprising' Event Water usually flows downward, but something strange happened under Greenland's ice sheet when a deluge punched through the surface to scour an area nearly twice the size of New York's Central Park Greenland's Harder Glacier and the subglacial flood outburst site (right bottom) can be seen in this Landsat satellite image acquired on August 1, 2014. “Having meltwater basically forcing its way up from the bed, that requires some extremely unique conditions,” Chu says. The event occurred in 2014 near Harder Glacier in far northern Greenland—a spot where a flash flood wouldn't get noticed on the ground. Mal McMillan, a professor of Earth observation at Lancaster University in England, and his colleagues discovered the flood a decade later while searching for signs of lakes beneath the ice in satellite imagery. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. CPOM, Lancaster University © DigitalGlobe, Inc. (2015), provided by European Space Imaging These lakes are important because they affect the overall water cycle and ice movement in Greenland. Increasingly good imagery has enabled researchers to see upwellings or divots in the ice that indicate that a subglacial lake is filling or draining. When looking at this historical data near Harder Glacier, the new study's team noticed a two-square-kilometer area of ice that had collapsed into an 85-meter-deep sinkhole. Beyond the fracture zone were six square kilometers of smooth ice, wiped clean by a giant deluge. About a kilometer downstream, however, the water hit some kind of obstacle. CPOM, Lancaster University © DigitalGlobe, Inc. (2015), provided by European Space Imaging The water then drained back under the ice sheet and may have ended up beneath Harder Glacier, which—coincidentally or not—saw a large calving event, in which a chunk of ice plopped into the sea, during that same 10-day period. “This is another piece in the puzzle of how big pieces of ice on our planet work,” says Robin Bell, a professor of marine and polar geophysics at Columbia University's Lamont-Doherty Earth Observatory, who was not involved in the research. The next question is how this puzzle piece fits in with the overall story of Greenland's rapid warming, McMillan says: “We need to understand whether this has happened on other occasions.” Stephanie Pappas is a freelance science journalist based in Denver, Colo.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a65490281/tiny-electric-motor/'>A Tiny 29-Pound Electric Motor Just Beat Everyone in Power Density</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Yasa, a small British engineering company, built an over-730-horsepower beast that weighs just under 30 pounds. No less than a week ago, a small British company called Yasa unofficially set the world record for the highest power density in an electric motor. Its prototype electric motor—weighing in at just 28.8 pounds—was able to output an eye-popping 737.5 horsepower. The biggest news here is that the motor is relatively inexpensive to manufacture and can be easily mass-produced, as Yasa claims it doesn't use any exotic materials like cobalt-iron laminations or litz wire. Koenigsegg's Dark Matter disc is still incredibly impressive, as it's nearly two years. It was designed to be part of the performance hybrid system in Koenigsegg's Gemera (as the Swedish automaker isn't interested in fully electric vehicles), and is capable of producing up to 2,300 horsepower in its most powerful configuration. (Somewhat surprisingly, the Lamborghini Urus SE we drove earlier this year featured an electric motor sourced from Volkswagen Group.) Mercedes is the only non-Italian automaker to use Yasa's electric motors in an EV, squeezing three of them into its latest 1,360 hp AMG GT XX Concept. Countless manufacturers have claimed to have an affordable EV on the way, and that makes us wonder: If Yasa's prototype motor cranks out 700 horsepower, how small could it make a motor that produces the mere 350 hp necessary to run a run-of-the-mill car? Many EV manufacturers are getting carried away with horsepower and torque figures, and the resulting horsepower arms race continues to worsen exponentially. The near-instant acceleration that electric powertrains deliver is a great party trick, but it's likely unnecessary overall. Power study from 2022 showed that the average horsepower for commuter vehicles sits around 180. Perhaps an even lighter, more modest electric motor from Yasa could be a catalyst for the next ultra-affordable, ultra-efficient EV. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. The Pros and Cons of Leasing vs Buying a Car The Best Car Waxes to Protect Your Paint Job How to Clean Your Headlights and Make Them Shine</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65530792/age-acceleration-50/'>Scientists Say Your Body Starts Aging Faster After 50—but Not All Parts at Once</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. As soon as we're born, we start aging, but scientists are quickly learning that not all aging is exactly the same. “Based on aging-associated protein changes, we developed tissue-specific proteomic age clocks and characterized organ-level aging trajectories,” the authors write. “Temporal analysis revealed an aging inflection around age 50, with blood vessels being a tissue that ages early and is markedly susceptible to aging.” In the study, scientists collected tissue samples across the body's major organ systems from 76 individuals of Chinese ancestry—aged 14 to 68—who all died from accidental brain injury. The tissues showed that certain organs aged at different rates. This isn't the first study to surmise that aging isn't quite as linear as we once thought. Last year, a study from Stanford University similarly confirmed that humans largely experience a period of accelerated aging at around 44 and the early 60s, which Stanford University's Michael Snyder, a professor of genetics and the study's senior author, surmised at the time could be related to aging. Speaking with Nature about this new study, Snyder says that the findings largely align with his own scientific conclusions. “It fits the idea that your hormonal and metabolic control are a big deal. That is where some of the most profound shifts occur as people age,” Snyder said. Understanding what those parts are will help human mechanics (AKA doctors) to keep things under the hood running for longer. As scientists continue exploring the mechanics of aging, findings will likely converge and begin tell the story of how the body more generally experiences aging throughout a lifetime. A Construction Survey Found an Ancient Celtic Town Rare ‘Book of the Dead' Found Buried in Egypt A Stolen Headless Statue Has Finally Returned Home Could New Physics Exist Below the Standard Model? An Enormous Desert Used to Be a 138-Foot-Deep Lake</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/this-summer-of-extreme-weather-features-flash-floods-and-corn-sweat/'>This Summer of Extreme Weather Features Flash Floods and Corn Sweat</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This Summer's Extreme Weather Explained: Flash Floods and Corn Sweat By Rachel Feltman, Andrea Thompson, Fonda Mwangi & Alex Sugiura With summer heat domes slamming down on parts of the U.S. and hurricane season ramping up, you've no doubt seen plenty of extreme weather stories in your feed over the last few weeks. Joining me today to demystify a few of those headlines is Andrea Thompson, a senior news editor for sustainability at Scientific American. Thanks so much for coming on to chat with us. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Feltman: So let's go over some of the topics that people might see trending in the headlines a lot, you know, during this time of year. Could you tell us a little bit about what happened in Texas and how it was possible for these floods to become so dangerous so quickly? Thompson: Yeah, so flash flood, it's, you know, sort of in the name—it happens really quickly and often takes people by surprise. It happens when you have really intense rains over a fairly small area, usually, over a relatively short time span. There was between six to 10 inches of rain in three hours, which is [laughs] a lot of rain. And it can be exacerbated by other aspects. You know, in cities you have a lot of pavement and a lot of asphalt, and those are impermeable to water, so water is going to collect even more than it would on, you know, soil. And then topography can play a role, too, and in Texas this was an area with a lot of riverbeds, a lot of steep topography that basically funnels all that water down into one area. And that's because there's just so much rain and it's all being funneled into sort of this one riverbed. And, you know, for reference, 26 feet is more than two stories in a building. Just six inches of quickly moving water can knock a person off their feet. Thompson: So it's not exponential, but you're getting much more force even for every little step in velocity ... Thompson: They're really hard to forecast, and that also takes people by surprise. So we can say, “It's gonna rain in this area on this day, and pockets will have, potentially, big downpours like this,” but you can't even say, usually, a few hours out, “It's going to bring exactly this much in exactly this place,” because these are such small features in the atmosphere that, you know, weather models just can't pick them out that far in advance. Feltman: Well, let's end on—not a fun note for people who are experiencing it but something that at least [laughs] feels more fun to talk about. Thompson: Yes, and it's, it's not just the actor who's in the new Superman movie [laughs], which—I've had lots of jokes about that [laughs]. So basically, there are heat waves in the summer. You know, this is especially true around the Gulf Coast, where you have this really abundant source of warm, moist air from the Gulf of Mexico. You know, the level of humidity can be affected by how wet a season has been—so we've had a pretty wet summer in the East, so everything is just really saturated with water, so when it's hot there's a lot of water to evaporate, or transpire, from plants. You know, these crops cover huge amounts of land, and when there's heat they transpire water vapor into the air, and that raises the humidity, and they call it “corn sweat,” which is a very funny term but [laughs] very grabby. But the Midwest is kind of notorious for these really high humidity levels, whereas when we think of humidity, we think of, like, “Oh, Florida,” or places like that ... Feltman: Well, and I feel like I ask you about this almost every time you come on, but it hasn't stopped [laughs] being important and useful: What can people do to keep cool in the summer and stay safe? Young children, older people, people who take certain medications or have certain illnesses, especially heart disease, are more susceptible. People who work outside are much more susceptible to heat illness. So it's important to be particularly aware for those people. Generally, you want to avoid any strenuous activity outside in the middle of the day, when the sun is at its highest and temperatures are at their highest. And we actually also have a story on how to keep your home cool that includes—you know, air-conditioning is obviously kind of the gold standard in terms of keeping things comfortable; it also has the added benefit of pulling humidity out of the air. But there's a lot you can do with fans in terms of keeping a home relatively cool, and part of that is because the motion, the air currents that it generates, means there's more air moving over the surface of your skin, so that is carrying heat away from your body, and it's also carrying sweat away, which sweat is basically the way our body naturally cools itself. You can also do things like making sure to seal any drafts, making sure your, like, your windows are very nice and sealed. You can put up blackout blinds, or if you don't even have those, you can even just do good old-fashioned aluminum foil on the outside to reflect some of the solar heat. I've done a little bit of that myself in my apartment [laughs]. You know, and there are other tips like that to basically just minimize the amount of heat coming into your apartment and maximize the amount of cooling that is happening for you. If you have any questions about the weather you'd like Andrea to answer for us in a future episode, let us know by sending us an email at ScienceQuickly@sciam.com. We'll be back on Friday with a fascinating conversation on the future of artificial intelligence—and why you shouldn't freak out if your favorite chatbot starts talking about its own sentience. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Andrea Thompson is a senior editor covering the environment, energy and earth sciences. She has been covering these issues for 16 years. Prior to joining Scientific American, she was a senior writer covering climate science at Climate Central and a reporter and editor at Live Science, where she primarily covered earth science and the environment. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. She holds a graduate degree in science, health and environmental reporting from New York University, as well as a B.S. in atmospheric chemistry from the Georgia Institute of Technology. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250729231241.htm'>Drones reveal 41,000-turtle nesting mega-site hidden in the Amazon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 03:32:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>By combining aerial imagery with statistical modeling, the researchers documented more than 41,000 Giant South American River Turtles gathered along the Amazon's Guaporé River. Their findings, published in the Journal of Applied Ecology, offer a new tool for conservationists seeking to monitor vulnerable animal populations with greater precision. "We describe a novel way to more efficiently monitor animal populations," said lead study author Ismael Brack, a post-doctoral researcher within the UF Institute of Food and Agricultural Sciences' (UF/IFAS) School of Forest, Fisheries and Geomatics Sciences. The project began with Wildlife Conservation Society (WCS) researchers in Brazil, Colombia and Bolivia. The turtles are exceptionally social creatures, and females congregate each year in July or August to nest in the Guaporé River sandbanks between Brazil and Bolivia. They create orthomosaics, which are highly detailed, high-resolution composite images made by stitching together hundreds of overlapping aerial photographs. Counting the animals shown in orthomosaics is a quicker, more accurate and less-invasive approach than counting animals from the ground. Together, researchers from UF and the WCS developed a method that improves counting accuracy by eliminating multiple sources of error, including double counts (the same individual counted multiple times) and missed individuals. Equipped with this data, they developed probability models that account for individuals entering and leaving the area, observed turtle behaviors and the likelihood of detecting an identifiable shell mark. Researchers who reviewed the orthomosaics but didn't account for animal movement or shell markings counted about 79,000 turtles. "These numbers vary greatly, and that's a problem for conservationists," Brack said. "If scientists are unable to establish an accurate count of individuals of a species, how will they know if the population is in decline or whether efforts to protect it are successful?" Past monitoring studies, for example, have involved clipping seals' fur, attaching high-visibility collars to elk and marking mountain goats with paintball pellets to keep track of animal movement during counts. The research team plans to perfect monitoring methods by conducting additional drone flights at the Guaporé River nesting site and in other South American countries where the Giant South American River Turtle gathers, including in Colombia and possibly Peru and Venezuela, Brack said. "By combining information from multiple surveys, we can detect population trends, and the Wildlife Conservation Society will know where to invest in conservation actions," he said. Note: Content may be edited for style and length. Rewriting Physics Textbooks: Scientists Propose a Bold New Theory About the Universe's Origins MIT Just Proved Einstein Wrong in the Most Famous Quantum Experiment Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/tsunami-warnings-issued-after-magnitude-8-7-earthquake-off-the-coast-of/'>Tsunami Warnings Issued after Magnitude 8.8 Earthquake off the Coast of Russia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-30 02:01:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Tsunami Warnings Issued after Magnitude 8.8 Earthquake Strikes off Russian Coast A photograph from March 2022 shows an aerial view of the South Kuril Islands. Waves 5.7 feet above normal reached Hawaii, flooding parking lots and streets, and wave heights reached up to eight feet in parts of California, the Guardian reported. The U.S. Tsunami Warning System has transitioned coastal areas from Hawaii to Alaska to the mainland Pacific Coast to a Tsunami Advisory, which means that “a tsunami with the potential to generate strong currents or waves dangerous to those in or very near the water is imminent, expected, or occurring” but that significant inundation is not expected. READ MORE: Russia's 8.8 Earthquake Is One of the Strongest Ever Recorded If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. This earthquake may be the eighth largest on record globally, wrote seismologist Alice-Agnes Gabriel of the Scripps Institution of Oceanography, on Bluesky. Evacuations were initially ordered in Japan and Hawaii after the July 30 earthquake. The epicenter is located 45 kilometers (28 miles) southeast of where a magnitude 9.0 temblor struck in 1952 and caused “a destructive, Pacific-wide tsunami,” according to the U.S. Geological Survey. A magnitude 7.4 earthquake that struck in the same region on July 20, 2025, was a foreshock to this July 30 quake, the USGS says. Editor's Note (7/30/25): This story will be periodically updated to reflect new information as the event unfolds. Andrea Thompson is a senior editor covering the environment, energy and earth sciences. She has been covering these issues for 16 years. Prior to joining Scientific American, she was a senior writer covering climate science at Climate Central and a reporter and editor at Live Science, where she primarily covered earth science and the environment. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. She holds a graduate degree in science, health and environmental reporting from New York University, as well as a B.S. in atmospheric chemistry from the Georgia Institute of Technology.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            