
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-24</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09312-4'>The Biodiversity Cell Atlas: mapping the tree of life at cellular resolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:44:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 645, pages 877–885 (2025)Cite this article Cell types are fundamental functional units that can be traced across the tree of life. Rapid advances in single-cell technologies, coupled with the phylogenetic expansion in genome sequencing, present opportunities for the molecular characterization of cells across a broad range of organisms. Despite these developments, our understanding of eukaryotic cell diversity remains limited and we are far from decoding this diversity from genome sequences. Here we introduce the Biodiversity Cell Atlas initiative, which aims to create comprehensive single-cell molecular atlases across the eukaryotic tree of life. This community effort will be phylogenetically informed, rely on high-quality genomes and use shared standards to facilitate comparisons across species. The Biodiversity Cell Atlas aspires to deepen our understanding of the evolution and diversity of life at the cellular level, encompassing gene regulatory programs, differentiation trajectories, cell-type-specific molecular profiles and inter-organismal interactions. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Goffeau, A. et al. Life with 6000 genes. C. elegans Sequencing Consortium.Genome sequence of the nematode C. elegans: a platform for investigating biology. Initiative, T. A. G. Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. Lander, E. S. et al. Initial sequencing and analysis of the human genome. Lewin, H. A. et al. Earth BioGenome Project: sequencing life for the future of life. Lewin, H. A. et al. Sullivan, P. F. et al. Leveraging base-pair mammalian constraint to understand genetic variation and human disease. Kapli, P., Yang, Z. & Telford, M. J. Phylogenetic tree building in the genomic age. Smith, S. D., Pennell, M. W., Dunn, C. W. & Edwards, S. V. Phylogenetics is the new genetics (for most of biodiversity). Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative cis-regulatory code. Oliver, S. G. From DNA sequence to biological function. Arendt, D. The evolution of cell types in animals: emerging principles from molecular studies. A., Kim, J. K., Svensson, V., Marioni, J. C. & Teichmann, S. A. The technology and biology of single-cell RNA sequencing. Rao, A., Barkley, D., França, G. S. & Yanai, I. Exploring tissue architecture using spatial transcriptomics. Minnoye, L. et al. Chromatin accessibility profiling methods. Regev, A. et al. The Human Cell Atlas. Haniffa, M. et al. A roadmap for the Human Developmental Cell Atlas. Lee, T. A. et al. A single-nucleus atlas of seed-to-seed development in Arabidopsis. Preprint at bioRxiv https://doi.org/10.1101/2023.03.23.533992 (2023). Schaum, N. et al. Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris. Li, H. et al. Fly Cell Atlas: a single-nucleus transcriptomic atlas of the adult fruit fly. Zhang, X. et al. A spatially resolved multi-omic single-cell atlas of soybean development. Marand, A. P., Chen, Z., Gallavotti, A. A cis-regulatory atlas in maize at single-cell resolution. Cao, J. et al. Comprehensive single-cell transcriptional profiling of a multicellular organism. References 18–23 represent comprehensive whole-organism cell atlases for animal and plant model species. A. et al. Divergence in neuronal signaling pathways despite conserved neuronal identity among Caenorhabditis species. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation. Mendieta, J. P. et al. Investigating the cis-regulatory basis of C3 and C4 photosynthesis in grasses at single-cell resolution. Triesch, S. et al. Single-nuclei sequencing of Moricandia arvensis reveals bundle sheath cell function in the photorespiratory shuttle of C3-C4 intermediate Brassicaceae. Li, J. et al. Deep learning of cross-species single-cell landscapes identifies conserved regulatory programs underlying cell types. Agarwal, V. & Shendure, J. Predicting mRNA abundance directly from genomic sequence using deep convolutional neural networks. Avsec, Ž. et al. Effective gene expression prediction from sequence by integrating long-range interactions. Janssens, J. et al. Decoding gene regulation in the fly brain. Pearce, J. D. et al. A cross-species generative cell atlas across 1.5 billion years of evolution: the TranscriptFormer single-cell model. Preprint at bioRxiv https://doi.org/10.1101/2025.04.25.650731 (2025). Taskiran, I. I. et al. Cell-type-directed design of synthetic enhancers. Mich, J. K. et al. Functional enhancer elements drive subclass-selective expression from mouse to primate neocortex. Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Church, S. H., Mah, J. L. & Dunn, C. W. Integrating phylogenies into single-cell RNA sequencing analysis allows comparisons across species, genes, and cells. Del Campo, J. et al. The others: our biased perspective of eukaryotic genomes. Ku, C. & Sebé-Pedrós, A. Using single-cell transcriptomics to understand functional states and interactions in microbial eukaryotes. Alacid, E. & Richards, T. A. A cell–cell atlas approach for understanding symbiotic interactions between microbes. Hu, M., Zheng, X., Fan, C.-M. & Zheng, Y. Lineage dynamics of the endosymbiotic cell type in the soft coral Xenia. Levy, S. et al. A stony coral cell atlas illuminates the molecular and cellular basis of coral symbiosis, calcification, and immunity. Serrano, K. et al. Spatial co-transcriptomics reveals discrete stages of the arbuscular mycorrhizal symbiosis. Fromm, A. et al. Single-cell RNA-seq of the rare virosphere reveals the native hosts of giant viruses in the marine environment. Ku, C. et al. A single-cell view on alga-virus interactions reveals sequential transcriptional programs and infection states. References 42–45 represent the first examples of using single-cell methods to study symbiotic interactions, simultaneously mapping host and symbiont gene expression programs within the same cells. Burki, F., Sandin, M. M. & Jamy, M. Diversity and ecology of protists revealed by metabarcoding. Delmont, T. O. et al. Functional repertoire convergence of distantly related eukaryotic plankton lineages abundant in the sunlit ocean. Cordier, T., Lanzén, A., Apothéloz-Perret-Gentil, L., Stoeck, T. & Pawlowski, J. Embracing environmental genomics and machine learning for routine biomonitoring. Marchetti, A. et al. Comparative metatranscriptomics identifies molecular bases for the physiological responses of phytoplankton to varying iron availability. Brunet, T. & King, N. The origin of animal multicellularity and cell differentiation. Sebé-Pedrós, A., Degnan, B. M. & Ruiz-Trillo, I. The origin of Metazoa: a unicellular perspective. Arendt, D. et al. The origin and evolution of cell types. Establishes a foundational conceptual framework for the study of cell type evolution and outlines key open questions and future research directions in the field. Archibald, J. M., Simpson, A. G. B. & Slamovits, C. H. Handbook of the Protists (Springer, 2017). Fritz-Laylin, L. K. et al. The genome of Naegleria gruberi illuminates early eukaryotic versatility. Brunet, T. et al. Light-regulated collective contractility in a multicellular choanoflagellate. Brunet, T. et al. A flagellate-to-amoeboid switch in the closest living relatives of animals. Dayel, M. J. et al. Cell differentiation and morphogenesis in the colony-forming choanoflagellate Salpingoeca rosetta. Phillips, M. A. et al. Malaria. Häckel, E. Monograph of Monera. Saville-Kent, W. A Manual of the Infusoria: Including a Description of All Known Flagellate, Ciliate, and Tentaculiferous Protozoa, British and Foreign, and an Account of the Organization and the Affinities of the Sponges Vol. Ramón y Cajal, S. Histologie Du Système Nerveux de l'homme & Des Vertébrés: Cervelet, Cerveau Moyen, Rétine, Couche Optique, Corps Strié, Écorce Cérébrale Générale & Régionale, Grand Sympathique Vol. Ramón y Cajal, S. Estructura de los centros nerviosos de las Aves. Revista Trimestral de Histología Normal y Patológica 1, 1–10 (1888). Virchow, R. Cellular Pathology as Based Upon Physiological and Pathological Histology (John Churchill, 1860). Hyman, L. H. The Invertebrates: Protozoa through Ctenophora (McGraw-Hill, 1940). Willmer, E. N. Cytology and Evolution (Academic, 1970). Vergara, H. M. et al. Whole-body integration of gene expression and single-cell morphology. Steinmetz, P. R. H. et al. Independent evolution of striated muscles in cnidarians and bilaterians. Ogino, K., Tsuneki, K. & Furuya, H. Distinction of cell types in Dicyema japonicum (phylum Dicyemida) by expression patterns of 16 genes. Novershtern, N. et al. Densely interconnected transcriptional circuits control cell states in human hematopoiesis. Minnoye, L. et al. Cross-species analysis of enhancer logic using deep learning. Alié, A. et al. The ancestral gene repertoire of animal stem cells. Cherbas, L. et al. The transcriptional diversity of 25 Drosophila cell lines. Evolutionary cell type mapping with single-cell genomics. Musser, J. M. et al. Profiling cellular diversity in sponges informs animal cell type and nervous system evolution. Sebé-Pedrós, A. et al. Cnidarian cell type diversity and regulation revealed by whole-organism single-cell RNA-seq. Sebé-Pedrós, A. et al. Early metazoan cell type diversity and the evolution of multicellular gene regulation. Najle, S. R. et al. Stepwise emergence of the neuronal gene expression program in early animal evolution. Plass, M. et al. Cell type atlas and lineage tree of a whole complex animal by single-cell transcriptomics. Fincher, C. T., Wurtzel, O., de Hoog, T., Kravarik, K. M. & Reddien, P. W. Cell type transcriptome atlas for the planarian Schmidtea mediterranea. References 75, 76, 78 and 79 represent the first whole-adult cell atlases for non-model animal species. Robertson, H. E. et al. Single cell atlas of Xenoturbella bocki highlights limited cell-type complexity. Álvarez-Campos, P. et al. Annelid adult cell type diversity and their pluripotent cellular origins. Ghaddar, A. et al. Whole-body gene expression atlas of an adult metazoan. Dogga, S. K. et al. A single cell atlas of sexual development in Plasmodium falciparum. Exemplifies the power of single-cell analysis to molecularly characterize cell states across the life cycle of unicellular eukaryotes. Wang, S. Y. et al. Role of epigenetics in unicellular to multicellular transition in Dictyostelium. Howick, V. M. et al. The Malaria Cell Atlas: single parasite transcriptomes across the complete Plasmodium life cycle. Wang, L. et al. The maturation and aging trajectory of Marchantia polymorpha at single-cell resolution. Scaling single-cell genomics from phenomenology to mechanism. Domcke, S. & Shendure, J. A reference cell tree will serve science better than a reference cell atlas. Shafer, M. E. R. Cross-species analysis of single-cell transcriptomic data. Yan, H. et al. Evolution of plant cell-type-specific cis-regulatory elements. Preprint at bioRxiv https://doi.org/10.1101/2024.01.08.574753 (2024). Hecker, N. et al. Enhancer-driven cell type comparison reveals similarities between the mammalian and bird pallium. One of the first examples of cross-species cell type comparisons based on regulatory sequence information rather than gene expression data. Bakken, T. E. et al. Comparative cellular analysis of motor cortex in human, marmoset and mouse. Song, Y., Miao, Z., Brazma, A. & Papatheodorou, I. Benchmarking strategies for cross-species integration of single-cell RNA sequencing data. Tarashansky, A. J. et al. Mapping single-cell atlases throughout Metazoa unravels cell type evolution. Mah, J. L. & Dunn, C. W. Cell type evolution reconstruction across species through cell phylogenies of single-cell RNA sequencing data. Lays out important considerations for cell type phylogenetic reconstruction and evolutionary models. Burkhardt, P. & Jékely, G. Evolution of synapses and neurotransmitter systems: the divide-and-conquer model for early neural cell-type evolution. Arendt, D., Bertucci, P. Y., Achim, K. & Musser, J. M. Evolution of neuronal types and families. Wagner, G. P. The developmental genetics of homology. Pacureanu, A., Silva, J. C. da, Yang, Y., Bohic, S. & Cloetens, P. Nanoscale three-dimensional imaging of biological tissue with X-ray holographic tomography. 10711 Biomedical Imaging and Sensing Conf. (eds Yatagai, T. et al.) 107112B (SPIE, 2018). Zinchenko, V., Hugger, J., Uhlmann, V., Arendt, D. & Kreshuk, A. MorphoFeatures for unsupervised exploration of cell types, tissues, and organs in volume electron microscopy. Di Tommaso, P. et al. Nextflow enables reproducible computational workflows. Denisenko, E. et al. Systematic assessment of tissue dissociation and storage biases in single-cell and single-nucleus RNA-seq workflows. Scully, T. & Klein, A. A mannitol-based buffer improves single-cell RNA sequencing of high-salt marine cells. Preprint at bioRxiv https://doi.org/10.1101/2023.04.26.538465 (2023). Chari, T. et al. Whole-animal multiplexed single-cell RNA-seq reveals transcriptional shifts across Clytia medusa cell types. Alles, J. et al. Cell fixation and preservation for droplet-based single-cell transcriptomics. García-Castro, H. et al. ACME dissociation: a versatile cell fixation-dissociation method for single-cell transcriptomics. B. et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Bageritz, J. et al. Glyoxal as an alternative fixative for single-cell RNA sequencing. Jiménez-Gracia, L. et al. FixNCut: single-cell genomics through reversible tissue fixation and dissociation. Fortmann, S. D. et al. Fixation before dissociation using a deep eutectic solvent preserves in vivo states and phospho-signaling in single-cell sequencing. Preprint at bioRxiv https://doi.org/10.1101/2023.02.13.528370 (2023). Martin, B. K. et al. Optimized single-nucleus transcriptional profiling by combinatorial indexing. Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Petrany, M. J. et al. Single-nucleus RNA-seq identifies transcriptional heterogeneity in multinucleated skeletal myofibers. Guillotin, B. et al. A pan-grass transcriptome reveals patterns of cellular divergence in crops. Jariani, A. et al. A new protocol for single-cell RNA-seq reveals stochastic gene expression during lag phase in budding yeast. Grones, C. et al. Best practices for the execution, analysis, and data storage of plant single-cell/nucleus transcriptomics. Heaton, H. et al. Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes. Guigó, R. Genome annotation: from human genetics to biodiversity genomics. Weisman, C. M., Murray, A. W. & Eddy, S. R. Mixing genome annotation methods in a comparative analysis inflates the apparent number of lineage-specific genes. Altenhoff, A. M. et al. Standardized benchmarking in the quest for orthologs. Glover, N. et al. Advances and applications in the quest for orthologs. Rosen, Y. et al. Toward universal cell embeddings: integrating single-cell RNA-seq datasets across species with SATURN. Rosen, Y. et al. Universal cell embeddings: a foundation model for cell biology. Preprint at bioRxiv https://doi.org/10.1101/2023.11.28.568918 (2023). Price, P. D. et al. Detecting signatures of selection on gene expression. Bertram, J. et al. CAGEE: computational analysis of gene expression evolution. Rohlfs, R. V., Harrigan, P. & Nielsen, R. Modeling gene expression evolution with an extended Ornstein–Uhlenbeck process accounting for within-species variation. Rohlfs, R. V. & Nielsen, R. Phylogenetic ANOVA: the expression variance and evolution model for quantitative trait evolution. Challis, R., Kumar, S., Sotero-Caio, C., Brown, M. & Blaxter, M. Genomes on a Tree (GoaT): a versatile, scalable search engine for genomic and sequencing project metadata across the eukaryotic tree of life. Wellcome Open Res. Zeng, H. What is a cell type and how to define it? Osumi-Sutherland, D. et al. Cell type ontologies of the Human Cell Atlas. The evolving concept of cell identity in the single cell era. We thank J. C. Carvalho for her expert work on the scientific illustrations and figure design. The BCA phase 0 is supported by the Gordon and Betty Moore Foundation (Grant GBMF12189). We also acknowledge financial support from the Wellcome Trust for the inaugural BCA meeting. A full list of members and their affiliations appears in the Supplementary Information Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain Arnau Sebé-Pedrós, Mafalda Dias, Anamaria Elek, Jonathan Frazer, Xavier Grau-Bové, Roderic Guigó, Manuel Irimia & Sebastián R. Najle Tree of Life, Wellcome Sanger Institute, Cambridge, UK Arnau Sebé-Pedrós, Mara K. N. Lawniczak & Mark Blaxter Universitat Pompeu Fabra, Barcelona, Spain Arnau Sebé-Pedrós, Mafalda Dias, Jonathan Frazer, Roderic Guigó & Manuel Irimia ICREA, Barcelona, Spain Arnau Sebé-Pedrós, Toni Gabaldón & Manuel Irimia Department of Computer Science and Applied Mathematics, Weizmann Institute of Science, Rehovot, Israel Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel Developmental Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany Laboratory of Computational Biology, VIB Center for AI and Computational Biology, Leuven, Belgium VIB-KU Leuven Center for Brain and Disease Research, Leuven, Belgium Department of Human Genetics, KU Leuven, Leuven, Belgium Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada Stazione Zoologica Anton Dohrn, Napoli, Italy Maria Ina Arnone Department of Embryology, Carnegie Science, Baltimore, MD, USA Department of Biology, Johns Hopkins University, Baltimore, MD, USA Department of Algal Development and Evolution, Max Planck Institute for Biology Tübingen, Tübingen, Germany Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT, USA Barcelona Supercomputing Centre (BSC-CNS), Plaça Eusebi Güell, Barcelona, Spain Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, Barcelona, Spain Donnelly Centre, University of Toronto, Toronto, Ontario, Canada Department of Physiology, University of Toronto, Toronto, Ontario, Canada Department of Biological Sciences, Howard Hughes Medical Institute, Columbia University, New York, NY, USA Centro de Biotecnología y Genómica de Plantas, Universidad Politécnica de Madrid (UPM)—Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA-CSIC), Campus de Montegancedo-UPM, Madrid, Spain Department of Systems Biology, Harvard Medical School, Boston, MA, USA Global Futures Laboratory, Arizona State University, Tempe, AZ, USA Department of Evolution and Ecology, University of California, Davis, CA, USA Department of Biology, Stanford University, Stanford, CA, USA Department of Organismal Biology and Anatomy, University of Chicago, Chicago, IL, USA Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT, USA Systems Biology Institute, Yale University, New Haven, CT, USA Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA, USA Department of Computer Science, California Institute of Technology, Pasadena, CA, USA The Rockefeller University, New York, NY, USA Earlham Institute, Norwich Research Park, Norwich, UK Medical School, University of East Anglia, Norwich Research Park, Norwich, UK Genomics Department, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA Berlin Institute for Medical Systems Biology, Max Delbrück Center for Molecular Medicine, Berlin, Germany German Center for Cardiovascular Research (DZHK), Berlin, Germany NeuroCluster of Excellence, Berlin, Germany Plant Resilience Institute, Michigan State University, East Lansing, MI, USA Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI, USA Department of Plant Biology, Michigan State University, East Lansing, MI, USA Department of Plant, Soil, and Microbial Sciences, Michigan State University, East Lansing, MI, USA Department of Biology, University of Oxford, Oxford, UK Stowers Institute for Medical Research, Kansas City, MO, USA Radcliffe Department of Medicine, University of Oxford, Oxford, UK Centre for Organismal Studies, Heidelberg University, Heidelberg, Germany Living Systems Institute, University of Exeter, Exeter, UK Department of Biosciences, University of Exeter, Exeter, UK European Molecular Biology Laboratory—European Bioinformatics Institute, Hinxton, UK Department of Neurosciences and Developmental Biology, Faculty of Life Sciences, University of Vienna, Vienna, Austria Department of Bioengineering, Stanford University, Stanford, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the paper based on discussions, comments and edits from S.A., J.A., M.I.A., M.B., P.C., S.M.C., M.D., C.D., A.E., J.F., T.G., J.G., X.G.-B., R.G., O.H., J.H.-C., M.I., A.K., H.L., C.J.L., H.M., J.M.M., L.G.N. Correspondence to Arnau Sebé-Pedrós. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. BCA single-cell dataset reporting checklist and recommended quality controls. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Sebé-Pedrós, A., Tanay, A., Lawniczak, M.K.N. The Biodiversity Cell Atlas: mapping the tree of life at cellular resolution. Issue Date: 25 September 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09489-8'>In vivo CRISPR screens identify modifiers of CAR T cell function in myeloma</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:42:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Chimeric antigen receptor (CAR) T cells are highly effective in haematological malignancies1. However, progressive loss of CAR T cells contributes to relapse in many patients2,3,4. Here we performed in vivo loss-of-function CRISPR screens in CAR T cells targeting B cell maturation antigen to investigate genes that influence CAR T cell persistence and function in a human multiple myeloma model. We tracked the expansion and persistence of CRISPR library-edited T cells in vitro and at early and late time points in vivo to track the performance of gene-modified CAR T cells from manufacturing to survival in tumours. The screens revealed context-specific regulators of CAR T cell expansion and persistence. Ablation of RASA2 and SOCS1 enhanced T cell expansion in vitro, whereas loss of PTPN2, ZC3H12A and RC3H1 conferred early growth advantages to CAR T cells in vivo. Notably, we identified cyclin-dependent kinase inhibitor 1B (encoded by CDKN1B), a cell cycle regulator, as the most important factor limiting CAR T cell fitness at late time points in vivo. CDKN1B ablation increased CAR T cell proliferation and effector function, significantly enhancing tumour clearance and overall survival. Our findings reveal differing effects of gene perturbation on CAR T cells over time and in different environments, highlight CDKN1B as a promising target to generate highly effective CAR T cells for multiple myeloma and underscore the potential of in vivo screening for identifying genes to enhance CAR T cell efficacy. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout Source data are provided with this paper. June, C. H., O'Connor, R. S., Kawalekar, O. U., Ghassemi, S. & Milone, M. C. CAR-T cell immunotherapy for human cancer. Rodriguez-Otero, P. et al. Ide-cel or standard regimens in relapsed and refractory multiple myeloma. San-Miguel, J. et al. Cilta-cel or standard care in lenalidomide-refractory multiple myeloma. Ruella, M., Korell, F., Porazzi, P. & Maus, M. V. Mechanisms of resistance to chimeric antigen receptor-T cells in haematological malignancies. Munshi, N. C. et al. Idecabtagene vicleucel in relapsed and refractory multiple myeloma. Berdeja, J. G. et al. Ciltacabtagene autoleucel, a B-cell maturation antigen-directed chimeric antigen receptor T-cell therapy in patients with relapsed or refractory multiple myeloma (CARTITUDE-1): a phase 1b/2 open-label study. Schmidt, R. et al. CRISPR activation and interference screens decode stimulation responses in primary human T cells. In vitro modeling of CD8+ T cell exhaustion enables CRISPR screening to reveal a role for BHLHE40. Shang, W. et al. Genome-wide CRISPR screen identifies FAM49B as a key regulator of actin dynamics and T cell activation. Simeonov, D. R. & Marson, A. CRISPR-based tools in immunity. Wang, D. et al. CRISPR screening of CAR-T cells and cancer stem cells reveals critical dependencies for cell-based therapies. Shifrut, E. et al. Genome-wide CRISPR screens in primary human T cells reveal key regulators of immune function. Carnevale, J. et al. RASA2 ablation in T cells boosts antigen sensitivity and long-term function. Enhanced T cell effector activity by targeting the Mediator kinase module. In vivo CD8+ T cell CRISPR screening reveals control by Fli1 in infection and cancer. Targeting REGNASE-1 programs long-lived effector T cells for cancer therapy. Lynn, R. C. et al. c-Jun overexpression in CAR T cells induces exhaustion resistance. Effective targeting of multiple B-cell maturation antigen-expressing hematological malignances by anti-B-cell maturation antigen chimeric antigen receptor T cells. Stock, S., Schmitt, M. & Sellner, L. Optimizing manufacturing protocols of chimeric antigen receptor T cells for improved anticancer immunotherapy. Long-term safety and activity of axicabtagene ciloleucel in refractory large B-cell lymphoma (ZUMA-1): a single-arm, multicentre, phase 1–2 trial. Xu, Y. et al. Closely related T-memory stem cells correlate with in vivo expansion of CAR.CD19-T cells and are preserved by IL-7 and IL-15. Cieri, N. et al. IL-7 and IL-15 instruct the generation of human memory stem T cells from naive precursors. Dwyer, C. J. et al. Fueling cancer immunotherapy with common γ chain cytokines. Abou-El-Enein, M. et al. Scalable manufacturing of CAR T cells for cancer immunotherapy. Amatya, C. et al. Optimization of anti-CD19 CAR T cell production for treatment of patients with chronic lymphocytic leukemia. Lane-Reticker, S. K. et al. Protocol for in vivo CRISPR screening using selective CRISPR antigen removal lentiviral vectors. The molecular basis of JAK/STAT inhibition by SOCS1. Sporri, B., Kovanen, P. E., Sasaki, A., Yoshimura, A. & Leonard, W. J. JAB/SOCS1/SSI-1 is an interleukin-2-induced inhibitor of IL-2 signaling. Flosbach, M. et al. PTPN2 deficiency enhances programmed T cell expansion and survival capacity of activated T cells. Simoncic, P. D., Lee-Loy, A., Barber, D. L., Tremblay, M. L. & McGlade, C. J. The T cell protein tyrosine phosphatase is a negative regulator of Janus family kinases 1 and 3. Matsushita, K. et al. Zc3h12a is an RNase essential for controlling immune responses by regulating mRNA decay. Mino, T. et al. Regnase-1 and roquin regulate a common element in inflammatory mRNAs by spatiotemporally distinct mechanisms. Disrupting roquin-1 interaction with regnase-1 induces autoimmunity and enhances antitumor responses. Uehata, T. et al. Malt1-induced cleavage of regnase-1 in CD4+ helper T cells regulates immune activation. Kloss, C. C. et al. Dominant-negative TGF-β receptor enhances PSMA-targeted human CAR T cell proliferation and augments prostate cancer eradication. Larson, R. C. et al. Anti-TACI single and dual-targeting CAR T cells overcome BCMA antigen loss in multiple myeloma. Lemoine, J., Ruella, M. & Houot, R. Overcoming intrinsic resistance of cancer cells to CAR T-cell killing. & Slingerland, J. M. p27 as a transcriptional regulator: new roles in development and cancer. Harrison, S. J. et al. CAR+ T-cell lymphoma post ciltacabtagene autoleucel therapy for relapsed refractory multiple myeloma. Perica, K. et al. CD4+ T-cell lymphoma harboring a chimeric antigen receptor integration in TP53. Ghilardi, G. et al. T cell lymphoma and secondary primary malignancy risk after commercial CAR T cell therapy. Larson, R. C. & Maus, M. V. Recent advances and discoveries in the mechanisms and functions of CAR T cells. Zheng, W. et al. Regnase-1 suppresses TCF-1+ precursor exhausted T-cell formation to limit CAR-T-cell responses against ALL. LaFleur, M. W. et al. A CRISPR–Cas9 delivery system for in vivo screening of genes in the immune system. LaFleur, M. W. et al. PTPN2 regulates the generation of exhausted CD8+ T cell subpopulations and restrains tumor immunity. Lin, C.-P. et al. Multimodal stimulation screens reveal unique and shared genes limiting T cell fitness. Comparative optimization of combinatorial CRISPR screens. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. received funding from the German Research Foundation (DFG; 466535590). received funding from the Gilead Sciences Research Scholar Program. received funding from the CRIS Foundation Out-Back Fellowship Programme (outback2021_6) and from the Spanish Society of Medical Oncology. received funding from the American–Italian Cancer Foundation and the Italian Association for Cancer Research. This work was funded by NIH R01 CA238268 (M.V.M.) and the Krantz Breakthrough Award. These authors contributed equally: Nelson H. Knudsen, Giulia Escobar, Felix Korell These authors jointly supervised this work: Robert T. Manguso, Marcela V. Maus Krantz Family Center for Cancer Research, Massachusetts General Hospital, Harvard Medical School, Charlestown, MA, USA Nelson H. Knudsen, Giulia Escobar, Felix Korell, Tamina Kienka, Alexander Armstrong, Amanda Bouffard, Michael C. Kann, Sadie Goncalves, Merle Phillips, Trisha R. Berger, Sangwoo Park, Diego Salas-Benito, Elijah P. Darnell, Filippo Birocchi, Mark B. Leick, Rebecca C. Larson, Debattama Sen, Robert T. Manguso & Marcela V. Maus Cellular Immunotherapy Program, Massachusetts General Hospital, Harvard Medical School, Charlestown, MA, USA Nelson H. Knudsen, Giulia Escobar, Felix Korell, Tamina Kienka, Alexander Armstrong, Amanda Bouffard, Michael C. Kann, Sadie Goncalves, Merle Phillips, Trisha R. Berger, Sangwoo Park, Diego Salas-Benito, Elijah P. Darnell, Filippo Birocchi, Mark B. Leick, Rebecca C. Larson, Robert T. Manguso & Marcela V. Maus Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, MA, USA Nelson H. Knudsen, Giulia Escobar, Felix Korell, Tamina Kienka, Celeste Nobrega, Seth Anderson, Andrew Y. Cheng, Maria Zschummel, Hans W. Pope, Mitra Pezeshki, Alexander Rojas, Juliette S. M. T. Suermondt, Merle Phillips, Sangwoo Park, Diego Salas-Benito, Filippo Birocchi, Mark B. Leick, Rebecca C. Larson, John G. Doench, Debattama Sen, Kathleen B. Yates, Robert T. Manguso & Marcela V. Maus Harvard Medical School, Boston, MA, USA Giulia Escobar, Felix Korell, Tamina Kienka, Maria Zschummel, Merle Phillips, Sangwoo Park, Diego Salas-Benito, Elijah P. Darnell, Filippo Birocchi, Mark B. Leick, Rebecca C. Larson, Debattama Sen, Kathleen B. Yates, Robert T. Manguso & Marcela V. Maus Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar CRISPR screen conception and experimental design: N.H.K., F.K., T.K., R.C.L., J.G.D., D.S., K.B.Y., R.T.M. Performed CRISPR screen experiments: N.H.K., F.K., T.K., A.Y.C., A.B., S.G., H.W.P., M. Pezeshki, A.R. Target validation and mechanistic experiments: N.H.K., G.E., F.K., T.K., A.Y.C., M.Z., A.B., A.A., M.C.K., S.G., H.W.P., M. Pezeshki, A.R., J.S.M.T.S., M. Phillips, S.P., D.S.-B., E.P.D. Animal handling was led by A.B. Analysis and interpretation of data: N.H.K., G.E., F.K., T.K., C.N., S.A., A.Y.C., M.Z., A.B., A.A., M.C.K., S.G., H.W.P., M. Pezeshki, A.R., J.S.M.T.S., M. Phillips, F.B., M.B.L., R.C.L., J.G.D., D.S., K.B.Y., R.T.M. Correspondence to Robert T. Manguso or Marcela V. Maus. are inventors on patents filed by MGH and the Broad Institute on these technologies. is an inventor on patents related to adoptive cell therapies, held by MGH (some licensed to ProMab and Luminary) and the University of Pennsylvania (some licensed to Novartis). receives grant and/or research support from Kite Pharma, Moderna and Sobi. has served as a consultant for multiple companies involved in cell therapies. 's competing interests are managed by Mass General Brigham. has received consulting or speaking fees from Bristol Myers Squibb, Gilead Sciences and Immunai Therapeutics, has equity ownership in OncoRev and receives research funding from Calico Life Sciences. is an inventor on patents related to adoptive cell therapies held by MGH and has served as a consultant for BioNTech and Cabaletta Bio. is an inventor on patents related to adoptive cell therapies held by MGH, has served as a consultant for Cargo and is now an employee of Link Cell Therapies. The other authors declare no competing interests. Nature thanks Joseph Fraietta and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Construct designs for the 4-1BB BCMA CAR (pCAR) and double-guide cassette (pGuide) containing the Mario sgRNA library (variable sgRNA). b. Timeline of the MM.1S stress model with 21 day tumor engraftment and 2E6 CAR treatment. Tumor growth was tracked by BLI. Data presented as mean +/– SEM. n = 3 mice (tumor only, Mario library (no CAR)) and n = 5 (CAR-T, Mario-CAR-T) per group from one healthy donor (ND202). Statistical significance amongst groups at day 35 as measured by two-way ANOVA with Tukey's multiple comparison. c. CD3 expression after day -5 CD3 negative enrichment (ND216, ND99, ND106). d. Representative Mario-CAR-T cell staining for CD34 indicating CAR transduced cells and NGFR for sgRNA library transduced cells from ND216. e. Replicate autocorrelation analysis scatter plots. Pearson's correlations are calculated for the library distribution of one animal versus any other animal, two averaged animals versus any other two, and so on. The mean of all possible combinations is plotted. f. Quantification of replicate autocorrelation analysis. Pearson's correlations are calculated for the library distribution of one animal versus any other animal, two averaged animals versus any other two, and so on. The mean of all possible combinations is plotted. g. z-scored abundance histograms of gene targeting or intergenic control sgRNAs across screening time points and conditions. h. Donor Pearson correlations across screening time points and conditions. Genes ranked by Log2(fold change) during in vitro manufacturing in IL-7/IL-15 (end-of-production vs. baseline, left panel) and early in vivo (day 7 vs. end-of-production, middle panel) or late in vivo (day 21 vs. end-of-production, right panel). Enriched genes are shown in red and depleted genes are shown in blue with circle size corresponding to –log10 (FDR). n = 2 healthy donor T cells (ND106, ND216). Frequency histograms of enrichment or depletion of sgRNAs for known T cell regulatory genes, grouped by respective period. n = 3 healthy donor T cells (ND216, ND99, ND106) c. CD4 and CD8 T cell composition (left panel) and phenotype (middle and right panels) at the beginning of CAR-T cell production (day 0) or at the EOP (day 13) from three healthy donors. Data are presented as mean +/– SEM. One-way ANOVA, with Sidak's multiple comparison test (statistical analysis is calculated between IL-2 and IL-7/IL-15 conditions for each genetic perturbation). a. Proportions of each knockout cell type amongst clusters. b. Hallmark gene set enrichment analysis (GSEA) of pseudo-bulk pooled PTPN2 KO (top panel) or CDKN1B KO (bottom panel) CAR-T cells compared to no guide CAR-T cells. c. Construct design for the 4-1BB BCMA CAR (pCAR) and double-guide cassette (pGuide) containing individual gene sgRNAs (variable sgRNA): CDKN1B, IL2RA, PTPN2, and RASA2. d. Quantification of insertions/deletions (indels) in sgRNA target region by next generation sequencing. Data represents target sgRNA 1 and 2 for CDKN1B, PTPN2, and RASA2, respectively (n = 2 samples); presented as mean +/– SEM. e. IL2RA expression by flow cytometry, with IL2RA KO CAR-T cells (before electroporation/CD3 negative selection and after) compared to UTD and intergenic control KO (both IL-2 and IL-7/IL-15-produced, respectively). 16-hour luciferase-based killing assay of knockout CAR-T cells co-cultured with MM.1S tumor cells (a, n = 4 technical replicates from CAR-T cells generated from two normal donors, ND116, ND202) or RPMI-8226 (b, n = 2 technical replicates from CAR-T cells generated from one normal donor), at different effector to target (E:T) ratios (from 10:1 to 1:100). Upper panels display comparison of intergenic KO CAR-T cells to UTD. Data presented as mean +/– SEM. Statistical significance was measured by two-way ANOVA with Tukey's multiple comparison test. a. MM.1S tumor burden as measured by bioluminescent imaging (BLI) of mice treated with intergenic control KO or CDKN1B KO BCMA CAR-T cells. NSG mice were injected intravenously with 1E6 MM.1S followed by transfer of CAR-T cells 21 days later. n = 3 mice per group from one healthy donor (ND202). Data presented as mean +/– SEM. Statistical significance was measured compared to the intergenic KO CAR-T cell treated group at day 77 (CDKN1B KO) as measured by two-way ANOVA with Tukey's multiple comparison test. Overall survival of mice treated with intergenic control KO or CDKN1B KO BCMA CAR-T cells. n = 13 for tumor only and intergenic control KO CAR-T cell treated animals and n = 16 for CDKN1B KO CAR-T cell treated animals. Data were combined from multiple experiments using T cells from two healthy human donors (ND202, ND116) and two sgRNA per gene target. Statistical significance was measured by log-rank (Mantel–Cox test) for Kaplan–Meier curves. c. Representative flow plot and quantification of the number of BCMA molecules expressed on the indicated tumor cell lines. a. MM.1S tumor burden as measured by bioluminescent imaging (BLI) of mice treated with intergenic control KO or CDKN1B KO BCMA CAR-T cells. NSG mice were injected intravenously with 1E6 MM.1S followed by transfer of CAR-T cells (from MM patient 1) 21 days later. n = 5 mice in the tumor only group. On day 66 post-CAR-T cell treatment, mice were re-challenged with 4E6 MM.1S cells (1st re-challenge), and again on day 84 with 8E6 MM.1S cells (2nd re-challenge). At each re-challenge, naïve mice (n = 3) were injected as controls. Data presented as mean +/– SEM. Percentage (b) and numbers (c) (mean +/– SEM, n = 8 mice per group) of intergenic control KO or CDKN1B KO CAR-T cells in the peripheral blood at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. d. CD8/CD4 CAR-T cells ratio (mean +/– SEM, n = 8 mice per group) in the peripheral blood of the mice at the indicated time post CAR-T cell infusion. e-f. Phenotype of intergenic control KO or CDKN1B KO CD4+ and CD8+ CAR-T cells at EOP (e) or in the mice (f, mean +/– SEM, n = 8 mice per group) at the indicated time post CAR-T cell infusion. Statistical significance measured by two-sided Student's t-test. Two-way ANOVA with Tukey's multiple comparison test. h) MM.1S tumor burden (mean +/– SEM, n = 8 mice per group) as measured by BLI on day 15 post 1st re-challenge. Each dot represents an individual mouse. i) Number of CAR-T cells (mean +/– SEM, n = 8 mice per group) in the blood of mice treated with intergenic control or CDKN1B KO CAR-T cells, measured 10 days before and 14 days after the 1st tumor re-challenge. Statistical significance measured by two-sided Student's t-test. j) MM.1S tumor burden (mean +/– SEM, n = 8 mice per group) as measured by BLI on day 41 post 2nd tumor re-challenge. Each dot represents an individual mouse. a. MM.1S tumor burden as measured by bioluminescent imaging (BLI) of mice treated with intergenic control KO or CDKN1B KO BCMA CAR-T cells. NSG mice were injected intravenously with 1E6 MM.1S followed by transfer of CAR-T cells (from MM patient 2) 21 days later. On day 71 post-CAR-T cell treatment, mice were re-challenged with 8E6 MM.1S cells. Naïve mice (n = 3) were injected as controls. Data presented as mean +/– SEM. Percentage (b) and numbers (c) (mean +/– SEM, n = 6 mice per group) of intergenic control KO or CDKN1B KO CAR-T cells the peripheral blood at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. d. CD8/CD4 CAR-T cells ratio (mean +/– SEM, n = 6 mice per group) over time in the peripheral blood of the mice. e-f. Phenotype of intergenic control KO or CDKN1B KO CD4+ and CD8+ CAR-T cells at EOP (e) or in the mice (f, mean +/– SEM, n = 6 mice per group) at the indicated time post CAR-T cell infusion. Statistical significance measured by two-sided Student's t test. Day 14: CD4 + CAR-T: TEMRA *p = 0.012 g. Percentage (mean +/– SEM, n = 6 mice per group) of PD1, TIM3 and CD39 expression in intergenic control KO or CDKN1B KO CD4+ and CD8+ CAR-T cells in the peripheral blood of the mice at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. h) MM.1S tumor burden (mean +/– SEM, n = 6 mice per group) as measured by BLI on day 41 post re-challenge. Each dot represents an individual mouse. a. RPMI-8226 tumor burden as assessed by caliper measurements of mice treated with intergenic control KO or CDKN1B KO BCMA CAR-T cells. NSG mice were implanted subcutaneously with 5E6 RPMI-8226 followed by transfer of CAR-T cells (from MM patient 1) 14 days later. n = 7 mice in tumor only and control sgRNA groups; n = 6 mice in CDKN1B KO group. Data presented as mean +/– SEM. Percentage (b) and numbers (c) (mean +/– SEM) of intergenic control KO (n = 7 mice) or CDKN1B KO (n = 6 mice) CAR-T cells the peripheral blood at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. d. CD8/CD4 CAR-T cells ratio (mean +/– SEM) in the peripheral blood of intergenic control KO (n = 7 mice) or CDKN1B KO (n = 6 mice) mice at the indicated time post CAR-T cell infusion. e-f. Phenotype of intergenic control KO (n = 7 mice) or CDKN1B KO (n = 6 mice) CD4+ and CD8+ CAR-T cells at EOP (e) or in the mice (f) at the indicated time post CAR-T cell infusion. Statistical significance measured by two-sided Student's t-test. Two-way ANOVA with Tukey's multiple comparison test. Comparison of cell cycle (G0/G1, S, G2/M) between CDKN1B KO and intergenic control KO CAR-T cells. a. MM.1S tumor burden as assessed by bioluminescence imaging (BLI) of mice treated with intergenic control KO or CDKN1B KO BCMA cilta-cel-like CAR-T cells. NSG mice were injected intravenously with 1E6 MM.1S followed by transfer of CAR-T cells (ND116) 21 days later. n = 5 mice in tumor only and control KO groups; n = 6 mice in CDKN1B KO group. Survival curve, n = 5 mice in tumor only and control sgRNA groups; n = 6 mice in CDKN1B sgRNA 1 group per group. Percentage (c) and numbers (d) (mean +/– SEM) of intergenic control KO (n = 5 mice) or CDKN1B KO (n = 6 mice) CAR-T cells in the peripheral blood at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. e. MM.1S tumor burden (mean +/– SEM, n = 6 mice in tumor only and CDKN1B sgRNA 1 groups; n = 5 mice in intergenic control KO group) as assessed by BLI in mice treated with intergenic control KO or CDKN1B KO BCMA cilta-cel-like CAR-T cells (ND202). Percentage (f) and numbers (g) (mean +/– SEM, n = 6 mice in CDKN1B sgRNA 1 group; n = 5 mice in control sgRNA group) of intergenic control KO or CDKN1B KO CAR-T cells the peripheral blood at the indicated time post CAR-T cell infusion. Two-way ANOVA with Tukey's multiple comparison test. h. CD8/CD4 CAR-T cells ratio (mean +/– SEM, n = 6 mice in CDKN1B KO group; n = 5 mice in intergenic control KO group) over time in the peripheral blood of the mice. Phenotype of intergenic control KO or CDKN1B KO (n = 6 mice) CD4+ and CD8+ CAR-T cells at EOP (i) or in the mice (j, mean +/– SEM, n = 6 mice in CDKN1B sgRNA 1 group; n = 5 mice in control sgRNA group) at the indicated time post CAR-T cell infusion. Statistical significance measured by two-sided Student's t-test. Day21: CD8 + CAR-T: TSCM *p = 0.0138; Day28: CD4 + CAR-T: TEMRA *p = 0.0328. k. Percentage (mean +/– SEM) of PD1, TIM3 and CD39 expression in intergenic control KO (n = 5 mice) or CDKN1B KO (n = 6 mice) CD4+ and CD8+ CAR-T cells in the peripheral blood of the mice at the indicated time post CAR-T cell transfer. Two-way ANOVA with Tukey's multiple comparison test. Gating strategy for flow cytometric analysis of peripheral blood from tumour-bearing NSG mice treated with CAR T cells. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Knudsen, N.H., Escobar, G., Korell, F. et al. In vivo CRISPR screens identify modifiers of CAR T cell function in myeloma. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09490-1'>Robot-assisted mapping of chemical reaction hyperspaces and networks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:29:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 645, pages 922–931 (2025)Cite this article Despite decades of investigation, it remains unclear (and hard to predict1,2,3,4) how the outcomes of chemical reactions change over multidimensional ‘hyperspaces' defined by reaction conditions5. Whereas human chemists can explore only a limited subset of these manifolds, automated platforms6,7,8,9,10,11,12 can generate thousands of reactions in parallel. Yet, purification and yield quantification remain bottlenecks, constrained by time-consuming and resource-intensive analytical techniques. As a result, our understanding of reaction hyperspaces remains fragmentary7,9,13,14,15,16. Are yield distributions smooth or corrugated? Do they conceal mechanistically new reactions? Can major products vary across different regions? Here, to address these questions, we developed a low-cost robotic platform using primarily optical detection to quantify yields of products and by-products at unprecedented throughput and minimal cost per condition. Scanning hyperspaces across thousands of conditions, we find and prove mathematically that, for continuous variables (concentrations, temperatures), individual yield distributions are generally slow-varying. At the same time, we uncover hyperspace regions of unexpected reactivity as well as switchovers between major products. Moreover, by systematically surveying substrate proportions, we reconstruct underlying reaction networks and expose hidden intermediates and products—even in reactions studied for well over a century. This hyperspace-scanning approach provides a versatile and scalable framework for reaction optimization and discovery. Crucially, it can help identify conditions under which complex mixtures can be driven cleanly towards different major products, thereby expanding synthetic diversity while reducing chemical input requirements. Owing to rapid advances in chemical automation, there has been a growing number of high-throughput campaigns to discover new modes of reactivity6,7,12,16,17, maximize reaction yields (often under the guidance of various optimization algorithms9,15,18) and produce standardized data sought urgently19 by chemical artificial intelligence. Our goals here were different, as we wished to reconstruct complete portraits of chemical reactions over multidimensional parameter spaces (concentrations, temperatures), with the quantification of yields not only for the major products but also for as many by-products as could be identified. Anticipating thousands of crude mixtures to analyse (here >9,000 in total), we wished to minimize the use of techniques such as nuclear magnetic resonance (NMR) or liquid chromatography–mass spectrometry (LC-MS), which, at their maximum throughput of only a few samples per hour, become costly and require prolonged or even dedicated access to equipment, which may be prohibitive for most academic groups. Accordingly, we relied heavily on inexpensive (cents per sample) and rapid (about 100 samples per hour) ultraviolet–visible (UV-Vis) detection augmented with algorithms to decompose complex spectra (that is, not just those featuring distinct and easy-to-interpret spectral features20,21) and with autocorrelation metrics to quantify missing information and detect anomalous outcomes. Unless otherwise stated, experiments were performed on a robotic platform (Fig. a, Parts of the house-built, inexpensive (about $25K) system: (1) horizontal gantry; (2) liquid-handling module; (3) racks of pipette tips; (4) stock solutions; (5) 54-vial plates; (6) UV-Vis spectrophotometer; (7) balance for weighing of solutions (see Supplementary Videos 1 and 2). b, Hypothetical A + B → C reaction space defined by initial concentrations [A]0 and [B]0 and temperature T. c, UV-Vis spectra of the HPLC-isolated species are measured at different concentrations (here for Claisen–Schmidt condensation from Fig. e, Stoichiometry of reaction places constraints on possible concentrations (blue) of components in the crude mixture (see Methods). f, Example of a perfect linear dependence (‘multicollinearity'), in which the brown spectrum in the top subplot happens to be a linear sum of the cyan and purple ones (scaled by, respectively, 0.50 and 0.25). Consequently, unmixing is an ill-conditioned problem, as the spectrum of the entire mixture (black) can be decomposed into infinitely many combinations of the brown, cyan and purple components (1:1:1 in the top and 0.5:0.75:2 in the bottom subplots). g, Correlation matrix for components A, B and C of the Claisen–Schmidt condensation from d. Low off-diagonal elements indicate no strong multicollinearity. h, Statistical distributions of yields with respect to the limiting reagent. Relative standard deviations are 2% between repeated optical detection performed on the same crude mixture (n = 54) and 5% between experimental repeats (n = 27) of the entire workflow for the same conditions (data from Fig. i, Differences between experimental and modelled spectra (from d) as a function of wavelength. The fit is adequate, as this trace closely resembles uncorrelated (‘white') noise. For a given reaction under study, this robot examines the hyperspace of conditions at points of some N-dimensional grid (for example, uniform in Fig. 1b), setting up reactions and acquiring UV-Vis absorption spectra at each point and at desired time(s). The acquisition of an individual spectrum takes, on average, roughly 8 s, whereas the accompanying pipetting and spectrometer washing–drying take about 50 s in total (Supplementary Video 2). In a subsequent (and perhaps counterintuitive) step, the crudes from all hyperspace points are combined (Fig. The resultant mixture is separated by chromatography and the isolated fractions are identified by traditional spectroscopic (NMR, MS) analyses. This ‘bulk' analysis is to identify the ‘basis set' of reaction products that form in any appreciable quantities anywhere in the hyperspace. The UV-Vis absorption spectra of these purified products (as well as all substrates, solvents and reagents used) at different concentrations are taken to construct concentration–absorbance calibration curves (Fig. Then the crude and often complex UV-Vis spectra acquired at each point of the hyperspace (Fig. 1b, right) are fitted by linear combinations of the reference spectra of the basis-set components (Fig. 1d) using the so-called vector decomposition techniques (also known as spectral unmixing22). Overall, this protocol requires conventional high-performance liquid chromatography (HPLC)/NMR/MS analysis of only one complex mixture combining all hyperspace samples (Fig. It returns on this time investment manifolds by rapidly estimating the yields and mixture compositions at thousands of individual data points within the hyperspace of the reaction. This general scheme merits two comments. First, it is important to reject potential unmixing solutions violating reaction stoichiometry (Fig. 1e) and to ensure stability of the fit. The latter benefits from the absorption spectra of components being of similar magnitudes and not linearly dependent (Fig. 1f), which is diagnosed by looking at off-diagonal elements of the correlation matrix between the concentrations of the components (Fig. Satisfying these conditions (1) is increasingly challenging as the number of components increases but (2) is helped by extending the spectral range as much as possible into the UV, at which the absorption bands of organic species are usually more numerous and narrower (see the example in Fig. With these precautions, yield estimates are within 5% (for example, a 20% yield would have a spread of 19–21%), with optical measurement and spectral unmixing contributing 2% and the remaining 3% caused by uncertainty of pipetting, fluctuations of temperature and residual evaporation (Fig. Second, the fitting procedure is accompanied by an algorithm to detect anomalous outcomes in some regions of the hyperspace. This is done by tracking differences between experimental and fitted spectra, calculating the variance of residuals (Fig. 1i) and evaluating the autocorrelation (across wavelengths, not time) of these residuals by the Durbin–Watson statistic. The mismatch is considered notable if the root mean square residual exceeds 0.01 absorbance units or if the Durbin–Watson statistic at 30 nm ‘lag' deviates from the value of 2 (which corresponds to the absence of autocorrelation) more strongly than the respective statistic for the baseline of a given spectrophotometer. If systematic deviation is detected, it then signifies formation of a product unexpected based on the initial scan of the hyperspace (see Fig. Further mathematical details of spectral unmixing and anomaly detection are discussed in Supplementary Information Section 3. We confirmed that this method is applicable to a range of reactions used widely in both academic groups and industry, including various couplings, condensations, cycloadditions and rearrangements (reactions 1–8 on the green background in Extended Data Fig. 1a), as well as substitutions, eliminations and multicomponent reactions (MCRs), discussed in detail in Figs. Expectedly, the approach is not suitable for, for example, reactions of aliphatic scaffolds whose products give no signal in the UV-Vis range above 220 nm and some reactions are borderline in terms of fit quality, owing to the obstruction of the product signal by reactant/solvent peaks (for example, 9–12 in the yellow and red portions of Extended Data Fig. Notably, yields quantified by the robotized approach correlate strongly, R2 = 0.96, with the yields of the same reactions performed, purified and analysed ex roboto (Extended Data Fig. a,b, Schemes of E1 (a) and SN1 (b) reactions and their mechanisms. For the former, we use 9-butyl-9H-fluoren-9-ol substrate 13a and for the latter, to prevent elimination, 9-phenyl-9H-fluoren-9-ol 14a. For E1, the parameters of the mechanism are the temperature dependences of equilibrium constant Kα for dynamic equilibrium between 13a and its protonated form, of equilibrium constant Kβ for dynamic equilibrium between 13b and its protonated form, of pKa of HBr in acetonitrile and the Eyring–Polanyi equation parameters for kinetic rate constants kE1 and k−E1 for water molecule elimination from the protonated 13a (Supplementary Information Section 5.2). For SN1 at a given temperature, the only parameter is the effective equilibrium constant KSN1 = [14b][H2O]/([14a][HBr]) (Supplementary Information Section 5.1). For SN1, the solvent was dioxane with admixture of AcOH (to homogenize the biphasic dioxane–water mixture formed in situ and complicating optical detection). c,d, Corresponding yield distributions of products 13b and 14b. For clarity, only 125 conditions per cube are shown. Marker colours and sizes are proportional to yields. Yield colour scales are the same for both cubes. Surfaces of equal yield are at 20%, 40%, 70% and 85%. e,f, Kinetics model (curves) fitted to experimental yields with respect to alcohol (circles) for E1 and SN1 reactions. To better visualize the quality of fits, only two-dimensional snapshots of the entire dataset are shown, here T = 21 °C for E1 and T = 36 °C for SN1 (for similar plots at all temperatures, see Supplementary Information Sections 5.1 and 5.2). Horizontal axis has the initial concentrations of HBr, with the colour corresponding to different initial concentrations of the alcohol indicated in the keys. a, Scheme of a reaction between alcohol 15a and HBr leading to major product 15b through anthranyl carbocation 15c. The minor anthraquinone product 15d originates from the cycloaddition of substrate 15a with singlet oxygen to give the endoperoxide, followed by subsequent fragmentations (for details, see Supplementary Fig. The ‘unexpected' carbocation dimer structure is assigned to 15e (for details of structural assignment, see Supplementary Information Sections 4.13 and 6.2). b, Space of reaction yields of the main product 15b under different conditions (reaction time: 24 h; for clarity, only 125 out of 1,085 points are shown; for all data, see Supplementary Information Section 4.15). Fitting the kinetic model to these experimental data reveals that yields of product 15b scale as \(1-\exp \left(-t{k}_{1}[{\rm{HBr}}]\exp \left(-\frac{{E}_{{\rm{a}}}}{{k}_{{\rm{B}}}T}\right)\right)\), in which t is reaction time, \({k}_{1}\exp \left(-\frac{{E}_{{\rm{a}}}}{{k}_{{\rm{B}}}T}\right)\) is the kinetic rate constant at a given temperature (prefactor k1 is a free parameter), Ea is the activation energy free parameter and [HBr] is the concentration of HBr—that is, these conversions increase with [HBr] and with temperature, do not depend on the initial concentration of the alcohol substrate and the rate-limiting step is formation of a carbocation 15c, all as expected for SN1 reaction. The isosurfaces correspond to 20%, 40%, 70% and 85% yield. c, Distribution of yields for the side product 15d. The isosurface corresponds to 20% yield. d, Pink cloud delineates a narrow region of conditions in which an anomalous outcome—that is, carbocation dimer 15e—is observed. e, Pink line is a typical UV-Vis spectrum from the region of conditions in which an anomalous outcome is observed. The dashed line is a typical experimental spectrum for which this anomaly is absent. Blue line is the ab initio theoretical absorption spectrum of 1.23 nM of 15e after summation with the dashed ‘baseline', blue band is the 1σ confidence interval (see Supplementary Information Section 6.2). a–d, Snapshots of the reaction's four-dimensional hyperspace. The axes of each plot are initial amine, aldehyde and isocyanide concentrations for initial concentrations of pTSA indicated above each plot. Experimental yields (after 16 h at 26 °C) of product 16e are colour-coded according to the legend in d. Only 125 interpolated conditions per cube are shown for clarity (see Supplementary Information Section 4.15 and Supplementary Video 3). Green and blue stars mark two distinct yield maxima. e–h, Corresponding yield distributions of 16e predicted by a kinetic model fitted simultaneously to all experimental data (yield isosurfaces are at 0.5%, 1.0%, 1.5% and 5.0%). This model is based on a kinetic network outlined in i and further side reactions (see Supplementary Information Scheme 5). The original reaction of 16a, 16b, 16c and DMF using initiator 16d and leading to product 16e is on the yellow background. The classic Ugi mechanism is summarized by blue arrows (for details, see Supplementary Information Scheme 6). A competing oxazoline-based path (brown) was experimentally excluded (red cross). A third mechanism (magenta), potentially active at high [pTSA]0 and involving aldehyde, amine and isocyanide in a 1:1:2 ratio, did not improve model fit. The theoretical model (Supplementary Tables 19, 20, 22 and 23) comprised this entire mechanistic network of 12 reactions and 15 proton transfer steps. Best-fitted model followed the data closely while retaining pKa values (in red font) of most species within 1–2 units from literature values (black font) used as initial guess (see Supplementary Information Section 5.3). j, Experimental and modelled yield profiles at the two maxima (corners A and B in a) along [pTSA]0 confirm that yields at global/local maxima are higher than at any four-dimensional paths connecting them (see Supplementary Video 3 and Supplementary Fig. k, Initial concentrations at global and local maxima. a, Identification of reaction products by closed-loop repurification and refitting against four-dimensional hypercube UV-Vis spectra (see Supplementary Information Section 4.15). Within eight repurification rounds, 16 components (14 products plus unreacted 19a and 19c) were identified. The mismatch between fitted and experimental spectra (vertical axis) decreased to the median value of instrumental uncertainty (dashed green line; see Supplementary Information Section 4.9). Box plot elements indicate the first quartile, median and third quartile, whiskers show the 5th and 95th percentiles, with outliers as black diamonds. Blue scatter plot shows individual data points. b, UV-Vis ‘basis set' of the 16 components. c, Reaction network: green, substrates; blue, isolated known products/intermediates; red, new products/intermediates. Network connectivity was verified by separate reactions (green arrows): (1) NH4OAc, AcOH, 80 °C, 18 h; (2) ethyl acetoacetate, EtOH, 80 °C, 18 h; (3) EAB, EtOH, 80 °C, 24 h. Note the presence of products of two separate ‘named reactions', Hantzsch (19d) and Petrenko-Kritschenko (19e) (mechanisms are shown in Supplementary Information Schemes 2 and 3. d–f, Within the hyperspace, the reaction network at 80 °C can be switched between three main products observed in different corners of the space: Hantzsch ester 19d (maximum yield 63.5%, <3% 19p) (d); Petrenko-Kritschenko product 19e (maximum yield about 66%) (e); benzylidene-extended Hantzsch derivative 19k (maximum yield 61.7%) (f). These yields (after 48 h) were quantified by HPLC and are colour-coded according to the legend. Isosurfaces correspond to 20%, 30%, 40%, 50% and 60% yield. For full experimental details, see Supplementary Information Sections 4.7, 4.9 and 4.10. a, 756 PBAs, catalysts with a general formula KMB[MA(CN)6], were prepared by the robot in separate vials using the co-precipitation method. Up to two aqueous solutions of MA and up to five of MB were pipetted automatically with specified volumes to prepare the PBA catalyst suspensions, each with a final volume of 200 µl in individual vials, followed by 24 h of ageing at room temperature. b, Irrespective of the composition, these nanomaterials were particles approximately 50 nm in size. Uniformity of metal distributions was confirmed by energy-dispersive spectroscopy. For large-area scanning electron microscopy images and all experimental details, see Supplementary Information Section 8. c, Five-dimensional hyperspace of yields of styrene oxide reconstructed by spectral unmixing. d, Five-dimensional hyperspace of selectivity of styrene oxide reconstructed by spectral unmixing. e, At least five PBAs (red markers 1–5 here and black circles in the corresponding hyperspace in c) showed yield–selectivity properties better than previously studied PBAs (blue markers), as well as some other controls (grey markers: Mn2+, Fe2+, Co2+, Ni2+, Cu2+, Fe(CN)63+, Co(CN)63− and the last one without any catalyst; see raw data in Supplementary Table 34). f, The robotically reconstructed hyperspace allowed for the identification of four products unreported in previous PBA studies of this reaction (20d, 20e, 20f and 20g shown on green backgrounds) and for the reconstruction of the reaction's mechanistic network involving four intermediates 20h, 20i, 20j and 20k. The blue fishhook arrows on species 20d indicate the electron flow to furnish aldehyde 20b (R = t-BuOO, t-BuO or HO), whereas the purple fishhook arrows on species 20k indicate the electron flow to furnish epoxide 20c (ref. 56). For the generation of radicals from tert-butyl hydroperoxide, see Supplementary Information Section 8.7. With these capabilities, we began to explore and analyse hyperspaces of several classic reaction types differing in mechanistic complexity. In doing so, we focused on yield distributions at reaction time of several hours (that is, the situation relevant to synthetic organic practice) and on hyperspace regions featuring maximal degree of variability (as opposed to regions in which, for example, concentrations are so high that yields are all ‘saturated'; see Supplementary Fig. 2–6, we plot only the subsets of data points, with all raw data available at Zenodo (https://doi.org/10.5281/zenodo.14880579). Starting simple, we considered the spaces of the E1 elimination and SN1 substitution23,24 using, respectively, substrates 13a and 14a. The space of E1 was examined at t = 4 h for 775 conditions and that of SN1 at t = 48 h for 930 conditions. The three-dimensional yield distributions shown in Fig. 2c,d for reaction products 13b and 14b are approximately concave, steadily increasing towards the global maximum (in bimolecular SN1, there is shoulder maximum in the region in which the concentration of the limiting reagent 14a is low; see also Extended Data Fig. Mathematically, the ‘steepness' of these surfaces can be quantified by the slopes of the product concentration with respect to the initial concentration of substrate(s), Dij = ∂Ci/∂C0,j—as shown in Supplementary Fig. 166, the absolute values of these slopes, |Dij| are small, at or below unity. Analysis of residuals detects no anomalies and, under all conditions, only substrates and the product are present in various proportions (for example, E1 is not accompanied by SN1 side reaction). Of note, yield data contained in the ‘cubes' allow for fitting approximate kinetic models (by time-integrating the underlying kinetic equations) and for deriving reasonable values of some kinetic and thermodynamic parameters, such as ΔH, ΔH‡ or ΔS (see caption to Fig. Even though the data correspond to only one time point, the abundance of yield values for various substrate concentrations imposes hundreds of constraints against which a candidate model must simultaneously fit, thus limiting the acceptable parameters of the model. In the ‘A four-dimensional hyperspace' section, we will use this approach to interrogate more complex mechanisms. Next, we revisited the SN1 reaction but with 15a used as a substrate (Fig. The algorithm also detects a minor anthraquinone by-product 15d congruent with the Diels–Alder reaction with singlet-state oxygen25,26 and distributed as illustrated in Fig. Most notably, analysis of residuals (Fig. 1i) provides the first example of a systematic anomaly and an unexpected outcome in a narrow region of low HBr concentrations marked in pink in Fig. This species exhibits an intensely pink colour that gradually wanes over two days (and disappears rapidly during purification attempts). 3e) does not agree with absorption spectra of 15c or structurally similar carbocations27,28,29, but extensive analyses based on MS as well as time-dependent density functional theory (TD-DFT) calculations (Supplementary Information Sections 4.13 and 6.2) support assignment as a carbocation dimer 15e resulting from a reaction between 15c and 15a (with enough HBr to create 15c, but with the amount of water introduced by this HBr insufficient to quench the intermediate into 15b). To our best knowledge, dimerization of a substrate with its own derived carbocationic intermediate has only been observed under superacidic conditions at temperatures down to −50 °C (ref. 30) but never in the presence of quenching nucleophiles (here H2O and Br−) and under ambient conditions. We emphasize, however, that even with the presence of this anomaly, the hyperspace remains simple—that is, each of its constituent species features only one yield maximum—and the yield distributions of all species are slowly varying, |Dij| ≈ 1 (Supplementary Information Fig. 166), including 15e, which forms over a narrow range but in only approximately 1 nM concentration. In search for more topologically complex hyperspaces, we turned to reactions that involve more than two substrates and are based on much more complex mechanisms. As the first case, we interrogated a four-component Ugi-type reaction from ref. 4 and chosen because of distinct UV-Vis signal of the cyclized product 16e. For this cyclization reaction, 3,234 conditions were investigated within a four-dimensional space defined by different initial concentrations of 4-nitrobenzaldehyde 16a, n-butylamine 16b and p-tosylmethyl isocyanide 16c substrates (DMF substrate was also a solvent of constant concentration), as well as p-toluenesulfonic acid monohydrate, pTSA 16d, acting as a reaction initiator. The hyperspace illustrated in Fig. 4a–d and Supplementary Video 3 now reveals the presence of two distinct yield maxima for the heterocyclic product 16e—a global maximum marked by a green star and a local maximum marked by a blue star. Further sampling of the conditions' grid in the region between the maxima confirms that they are separated in four-dimensional space by a region of low yield (Fig. Notably, MS analyses (Supplementary Information Section 4.8) evidence distinct signals at these two maxima: the local maximum gives a peak m/z ratio corresponding to the iminium ion 18c expected for the classical Ugi mechanism32,33, whereas the global maximum also features a peak attributable to the oxazoline 17e. This, in turn, may suggest an extra mechanism (brown arrows) through the oxazoline 17e intermediate—alternative mechanisms for the Ugi reaction have, indeed, been postulated for decades but never proved34,35. However, such a possibility is directly disqualified by separate, ex roboto experiments in which isolated 17e fails to give product 16e on further reaction with n-butylamine in the presence of pTSA (Supplementary Information Section 4.8.4). Although other mechanisms can also be considered (see magenta arrows in Fig. 4 and corresponding caption), fitting the experimental hyperspace data to the kinetic network of 12 reactions and 15 proton transfer steps (Fig. 4i and theoretical details in Supplementary Information Section 5.3) suggests that an excellent agreement can be achieved (Fig. 4e–h) based only on the Ugi mechanism—that is, inclusion of extra mechanisms offers no perceptible improvement. We also note that evolution of these maxima following changes in pTSA concentration is very gradual (Supplementary Video 3) and the yield distributions are characterized by very small ‘slopes', |Dij| ≲ 1 (Supplementary Fig. Next, we considered the classic Hantzsch pyridine synthesis (Fig. 5), studied for almost 150 years (refs. 36,37,38) and interesting for several known intermediates and competing pathways (coloured in blue in Fig. Notably, examining the four-dimensional hyperspace (concentrations of the three substrates, 26 °C and 80 °C temperatures; a total of 2,582 conditions) revealed the presence of many more components than previously thought. Their isolation required several HPLC repurification cycles performed in a closed-loop manner. Specifically, after each cycle, the newly identified substances were added to the spectral unmixing algorithm and the global fits to experimental spectra from the entire hypercube gradually improved (Fig. 5a,b); meanwhile, more fractions were selected for the next round of purification if they featured yet-unassigned signals (Supplementary Information Sections 4.9 and 4.10). After eight such fitting–purification cycles, the mismatch between the fitted and experimental spectra reached instrumental noise (dashed green line in Fig. 5a), at which point our knowledge of the hyperspace composition can, for all practical reasons, be deemed nearly complete. This knowledge spans not only the seven known species36,37,38 but also nine new ones (coloured red in Fig. 5c) that have not been reported in the classical Hantzsch reaction but can be of interest in the context of biological activity39. By considering the causal relationships between these and other species discovered within the hyperspace, it is possible to establish the synthetic connectivity of the isolated species (Supplementary Information Section 4.10). This analysis, ultimately, reconstructs the complete network of the Hantzsch reaction detailed in Fig. 5c with key steps confirmed by further ex roboto reactions (marked by green arrows). Notably, analysis of yield distributions (with the help of HPLC, as spectral unmixing of all 16 components is no longer unique) revealed that, by adjusting substrate concentrations at 80 °C, the network can be switched between three different major products (19d, 19e, 19k), each forming in >60% yield and with maxima located at different corners of the conditions' cube (Fig. One of these switchovers is between the Hantzsch ester (19d) and the Petrenko-Kritschenko product (19e)40, meaning that even the so-called named reactions can, in reality, be part of the same hyperspace. Another observation is that, despite the switchovers, the individual yield distributions for individual species remain smooth, as visualized by the isosurfaces in Fig. Finally, we consider a hyperspace in which dimensions correspond to systematic variations in composition. Here these compositions are the contents of different metals in the Prussian blue analogues (PBAs)41 (Fig. 6), which are perovskite-type materials described by a general formula KMB[MA(CN)6] and widely studied in the context of catalysis and energy storage42. The robot surveys a five-dimensional space defined by the contents of metals at sites MA (two types, Fe and Co) and MB (five types, Mn, Fe, Co, Ni and Cu) over a uniform grid with the granularity of 0.2 (for example, Mn0.2Ni0.4Cu0.4–Fe0.6Co0.4 PBA versus Mn0.2Ni0.2Cu0.6–Fe0.4Co0.6 PBA, with molar fractions at A and B sites each summing to unity). Each of the 756 PBAs is prepared in situ by straightforward co-precipitation method (Fig. 6a,b) and is used to catalyse reaction of styrene 20a and t-BuOOH to give styrene oxide 20c (given that styrene oxide is the key intermediate to synthesize many fine chemicals and pharmaceuticals43, this reaction has been the subject of several studies using various catalysts, including PBAs44,45). Figure 6c shows the distribution of reaction yields and Fig. As seen, the yield hyperspace is much more corrugated than concentration–temperature hyperspaces from Figs. 2–5 and features several local yield maxima. Notably, it contains several PBA compositions that offer better yield–selectivity characteristics than previously reported PBAs44,45 or other controls (Fig. As in other examples that we considered, the hyperspace reconstruction reveals the presence of unreported intermediates and by-products and helps reconstruct a mechanistic network (Fig. One of the insights from these studies is the relatively simple structure of the concentration–temperature spaces, with yield distributions of any single species featuring, at most, two yield maxima and with the slope values |Dij| at or below unity (Supplementary Fig. As described in Supplementary Information Sections 7.1 and 7.2, it can be rigorously proved that, for strongly connected, directed hypergraphs47,48 (that is, those that cannot be split into non-interacting subsets of reactions) and for first-order and pseudo-first-order reactions, |Dij| ≤ 1. For higher-order kinetics of individual reactions, the theoretical upper bound on |Dij| is too high to be practically relevant, but numerical studies of networks comprising first-order and second-order steps and no cycles confirm that |Dij| is also typically on the order of unity. A corollary to this result is that, along concentration coordinates C0,j, j = 1,…,k, the hypersurfaces can be examined faithfully at sparse intervals (see Supplementary Information Section 7.6). This, in turn, can help reduce the number of experiments during conditions' screening and optimization campaigns. For more discussion of hyperspace properties (in terms of topology, differential yields and Shannon information theory), see Supplementary Figs. Naturally, these considerations do not apply to hyperspaces in which compositions are varied. For instance, different catalysts (as in Fig. 6) can lower the activation barriers substantially and to different extents. As a result, they can alter the network-wide kinetics more than changes over a limited range of concentrations or temperatures, resulting in ‘steeper' concentration derivatives and in the hyperspace featuring many local yield maxima. The method by which we reconstructed reaction networks is loosely analogous to the approaches used in electronics to reverse-engineer a circuit inside a ‘black box' by applying inputs and analysing corresponding outputs. In the cases studied here, the wirings could be reconstructed relatively readily by humans but, as illustrated in Supplementary Information Section 4.11, this process can also be aided by network algorithms operating either at the level of mechanistic steps4,49 or full reactions50, thereby linking the hyperspace of conditions to the ‘space of reaction grammars'5, a link argued to be key to chemical innovation5. In this effort, the broad investigation of the hyperspace is essential, as different inputs can serve to ‘activate' or ‘deactivate' different branches of the network (by modifying reaction rates and shifting equilibria), enabling the formation of as many products as possible. This also means investigating the reaction space at stoichiometries that a human chemist might not necessarily find intuitive or relevant to a particular reaction. Here this approach more than doubled the knowledge of the Hantzsch hyperspace and we have seen similar discovery enhancements in other reactions that we have studied since then (for example, Pechmann, Biginelli). From a technical point of view, our ability to interrogate entire spaces of conditions capitalizes on the use of inexpensive (cents per sample) optical detection, which reduces the need for costly (about $45–300 per sample) HPLC/quantitative NMR analyses by a factor of several hundred (for example, one HPLC/NMR cycle per 1,085 UV-Vis experiments to reconstruct the SN1 network in Fig. 3; eight closed-loop cycles to ensure complete knowledge of the Hantzsch space examined by UV-Vis at 2,582 conditions). As long as the equilibrium is not fully reached (Supplementary Figs. 154 and 160), sweeping the reaction rates through control of starting concentrations accesses the type of data similar to conventional kinetic experiments (that ‘sweep' the time to observation instead) and estimation of kinetic parameters becomes possible. A related aspect is the practical ability to direct these reaction networks towards desired outcomes by simply adjusting concentrations (that is, without using different reagents, as done in some recent excellent papers, such as ref. 51). For simple reactions, such stoichiometric control is, of course, well known (for example, we can influence the ratio of monosubstituted to disubstituted product of alkyl dibromide by using one versus two equivalents of the nucleophile). However, for multicomponent reaction mixtures, it is less obvious that they can be pushed cleanly towards different major products. We have seen this capability realized for the Hantzsch reaction, in which the yields of major products 19d, 19e and 19k were maximized at concentration ratios very different from those we may expect based on product stoichiometry alone. For instance, product 19e incorporates one copy of substrate 19a, two copies of 19b and two copies of 19c, but in the hyperspace in Fig. 5e, the regions of appreciable yield, >40%, start from about 1:4.5:2 ratio (and continue for even higher molar excess of 19b at the yield maximum, which, however, becomes wasteful in terms of substrate use; see Extended Data Fig. In conclusion, this work is an initial effort—made possible by modern reaction automation—to understand the structure of reaction hyperspaces, one of the five foundational spaces of mathematical chemistry5. It reinforces the view of chemical reactions as networks4,51,52 embedded in multidimensional spaces of conditions and, at least in some cases, switchable between different major products, which is reminiscent of some biochemical networks53,54 and promising in terms of diversity-oriented synthesis55. The rapid and cost-effective approach to hyperspace reconstruction can (1) systematize and accelerate reaction discovery and optimization and (2) foster fundamental research on reaction networks as dynamic systems. This effort can be aided by hyperspace visualization and analysis tools such as those we developed and make available at Zenodo (https://doi.org/10.5281/zenodo.14880579; see also Supplementary Video 4). In a broader context, inclusion of our experimental yield maps to the benchmarks used for testing the general-purpose yield optimization algorithms9,11,15 would appreciably widen the diversity of these benchmarks, given the current scarcity of such datasets7,9,13,15. In future work, we aim to extend our robotic platform to reactions that require solid dispensing and/or strictly oxygen-free conditions (this will broaden the scope of hyperspaces we can analyse), to accelerate dispensing and measuring operations (to analyse fast reactions) and also to monitor hyperspace evolution over time, which will be necessary to reconstruct highly nonlinear mechanisms (for example, oscillating reactions). Reagent addition was carried out using a commercial pipetting module (ZEUS, Hamilton), optimized for pipetting a range of organic solvents in aliquots ranging from 10 µl to 1 ml, with volumetric errors of less than 1% for volumes greater than 50 µl and less than 2% for volumes between 10 and 50 µl. Liquids were dispensed into 2-ml glass vials arranged in custom-designed 54-well plates, with each reaction typically set to a final volume of 500 µl. Following reagent addition, vials were hermetically sealed using a flat lid, with a rubber sheet and perfluoroalkoxy film placed between the lid and the vial. Initial mixing was performed on an orbital shaker at 250 rpm for 5 min. Subsequent stirring is omitted, as separate studies (see Supplementary Information Sections 2.9 and 4.5) have shown that, for vials of these dimensions, passive mixing over the course of the reaction (hours) is as efficient as mechanical stirring, thereby eliminating the need for cumbersome stir bars. After reactions were run for desired times, the crude mixture from each vial was, if necessary, automatically diluted to align with the detection range of the UV-Vis spectrophotometer (NanoDrop, Thermo Fisher Scientific). The operation of the spectrophotometer was also automated, including lid opening and closing, along with the flushing and drying of the measuring pedestal (see Supplementary Video 2). All of these operations were orchestrated by house-written software. The entire assembly was placed inside a hood and, if needed, constantly purged with nitrogen. For some hyperspaces, two such systems were used. Further technical details and the blueprints for system replication are provided in Supplementary Information Section 2 and at Zenodo (https://doi.org/10.5281/zenodo.14880579)57. The software design is illustrated in Supplementary Fig. 2 and the source code is deposited at https://github.com/yaroslavsobolev/robowski-maps. The full cubes of 775 E1 conditions (five temperatures, 16–36 °C; five initial substrate concentrations, 1.5–15.0 mM; 31 HBr concentrations, 0.1–10.0 mM; reaction time: 4 h) and 930 SN1 conditions (six temperatures; 16–66 °C; five 9-butyl-9H-fluoren-9-ol substrate concentrations, 0.03–0.30 M; 31 HBr concentrations; reaction time: 48 h) are provided in Supplementary Information Section 4.15, with all raw data in the Zenodo repository. Stoichiometry constraints, as shown in Fig. 1e, were included into the spectral unmixing algorithm by modifying the log-likelihood function L to be minimized: L is a sum of squared spectral mismatch (yellow) of experimental Ameasured(λ) and modelled Amodel(λ) absorbances, divided by the instrumental variance of the spectrophotometer σ2(λ, Ameasured(λ)), and the term representing the violation of stoichiometric inequalities (red) containing the Heaviside function θ(x) and the experimental uncertainties \({\sigma }_{{[{\rm{A}}]}_{0}}\) and \({\sigma }_{{[{\rm{B}}]}_{0}}\) of preparing the reaction mixture with given starting concentrations. See Supplementary Information Section 3 for more details about the spectral unmixing algorithm 2–5 were extracted by the marching cubes algorithm58 operating on a 50 × 50 × 50 regular grid obtained through the radial basis functions interpolator applied to the raw data. This method was implemented in the HyperspaceViewer software, which is provided in the Supplementary Information and is open-sourced at a GitLab repository (https://gitlab.com/az-steak/hyperspace_viewer). The kinetic fitting in Fig. This procedure gave reasonable estimates of the reactions' thermodynamic parameters: ΔH = −30.7 ± 1.4 kJ mol−1 for SN1 and \(\Delta H={81.98}_{-1.64}^{+3.52}\,{\rm{kJ}}\,{{\rm{mol}}}^{-1}\), \(\Delta S={273.8}_{-4.9}^{+10.8}\,{\rm{J}}\,{{\rm{mol}}}^{-1}\,{{\rm{K}}}^{-1}\) and \({\Delta H}^{\ddagger }={21.10}_{-0.49}^{+0.33}\,{\rm{kJ}}\,{{\rm{mol}}}^{-1}\) (transition state enthalpy) for E1. For derivations of these values and comparisons against related literature examples, see Supplementary Information Sections 5.1 and 5.2. Initial guess of kinetic parameters (Fig. 4e–h) was provided by the Markov chain Monte Carlo algorithm59, kinetic equations at each condition were integrated numerically and the discrepancy against the experimental yield distribution (over the hyperspace) was minimized using the trust region reflective algorithm60. See Supplementary Information Section 5.3 for further details of the kinetic model. PBAs were synthesized following a reported co-precipitation method61. Two stock solutions containing metal MA (Fe, Co) were prepared by dissolving K3[Fe(CN)6] or K3[Co(CN)6] in deionized water to form clear solutions of type A (0.10 M). Five stock solutions containing metal MB (Mn, Fe, Co, Ni, Cu) were prepared separately by dissolving corresponding metal nitrates/chlorides (Mn(NO3)2, FeCl2, Co(NO3)2, Ni(NO3)2, Cu(NO3)2, each at 0.10 M) in the presence of trisodium citrate dihydrate (Na3C6H5O7·2H2O, 0.1125 M) in deionized water to form clear solutions of type B. Afterwards, mixing of solutions of type A and type B were performed in specified volume ratios using the automated liquid-handling system according to the target composition. For instance, for a PBA with composition Mn0.2Ni0.4Cu0.4–Fe0.6Co0.4, the sequence and volume of stock solution addition were as follows: 20 μl Mn(NO3)2, 40 μl Ni(NO3)2, 40 μl Cu(NO3)2, 60 μl K3[Fe(CN)6] and 40 μl K3[Co(CN)6]. The total volume of solutions of type A was 100 μl and the total volume of solutions of type B was also 100 μl—therefore, the total volume of each PBA solution was always 200 μl. After sealing, each 54-vial well plate housing different mixed solutions was shaken for 15 min (250 rpm) and aged at room temperature for 24 h to form PBAs. For a typical set of styrene epoxidation reactions, the reaction stock solution was prepared as follows: first, styrene (0.50 M) and tert-butyl hydrogen peroxide (0.75 M) were dissolved in 100 ml acetonitrile. Then, 30 ml cetyltrimethylammonium bromide aqueous solution (0.25 M) was added to the mixture. Subsequently, the obtained mixture was sonicated for 5 min to obtain a homogeneous stock solution. Afterwards, the robotic system pipetted 1.3-ml aliquots of the thus prepared stock solution into the vials containing previously prepared PBAs (200 µl aqueous suspension). The reaction vials were then sealed and every reaction plate was placed on a thermal shaker (72 °C, 700 rpm) for 6 h. After reaction, when the PBAs sedimented, the robot acquired 40 µl of the supernatant from every vial, diluted it 1,000 times and then measured the UV-Vis spectrum with the NanoDrop spectrophotometer. The yield of styrene oxide (yso) and benzaldehyde (yalde) for each sample can be obtained by spectral unmixing and the selectivity of styrene, calculated by yso/(yso + yalde) × 100%, was then obtained. All data in support of the findings of this study are included in the Supplementary Information; furthermore, raw spectra and their fitted results for all automated reactions (9,362 data points in total) are deposited at Zenodo (https://doi.org/10.5281/zenodo.14880579)57. Deposition numbers 2418839 (15b) and 2418838 (15b·C3H7NO) containing the crystallographic data are available free of charge from the Cambridge Crystallographic Data Centre at www.ccdc.cam.ac.uk/data_request/cif. All codes and pipelines used for this study are provided at GitHub (https://github.com/yaroslavsobolev/robowski-maps). HyperspaceViewer software for the visualization and analysis of hyperspaces is deposited at GitLab (https://gitlab.com/az-steak/hyperspace_viewer) and Zenodo (https://doi.org/10.5281/zenodo.14880579)57, along with the source code and user manual, and as the HyperspaceViewer.zip Supplementary File. Furthermore, We release a Google Colab notebook (https://colab.research.google.com/drive/1uXaoWjkHYaPoCt8N6q4GzYY3e92T_nC-?usp=sharing) with the step-by-step tutorial of the core functionality of calibration and spectral unmixing for the Claisen–Schmidt condensation case shown in Fig. Predicting the outcomes of organic reactions via machine learning: are current descriptors sufficient? Saebi, M. et al. On the use of real-world datasets for reaction yield prediction. The challenge of balancing model sensitivity and robustness in predicting yields: a benchmarking study of amide coupling reactions. Szymkuć, S., Wołos, A., Roszak, R. & Grzybowski, B. Restrepo, G. Spaces of mathematical chemistry. Granda, J. M., Donina, L., Dragone, V., Long, D. L. & Cronin, L. Controlling an organic synthesis robot with machine learning to search for new reactivity. Lin, S. et al. Mapping the dark space of chemical reactions with extended nanomole synthesis and MALDI-TOF MS. Science 361, eaar6236 (2018). Coley, C. W. et al. A robotic platform for flow synthesis of organic compounds informed by AI planning. Angello, N. H. et al. Closed-loop optimization of general reaction conditions for heteroaryl Suzuki-Miyaura coupling. Rohrbach, S. et al. Digitization and validation of a chemical synthesis literature database in the ChemPU. Automated self-optimization, intensification, and scale-up of photocatalysis in flow. Autonomous mobile robots for exploratory synthetic chemistry. Buitrago Santanilla, A. et al. Nanomole-scale high-throughput chemistry for the synthesis of complex molecules. Davies, I. W. The digitization of organic synthesis. Shields, B. J. et al. Bayesian reaction optimization as a tool for chemical synthesis. Wilbraham, L., Mehr, S. H. M. & Cronin, L. Digitizing chemistry using the chemical processing unit: from synthesis to discovery. Rapid planning and analysis of high-throughput experiment arrays for reaction discovery. Wang, J. Y. et al. Identifying general reaction conditions by bandit optimization. Artificial intelligence for retrosynthetic planning needs both data and expert knowledge. Stadler, E. et al. A versatile method for the determination of photochemical quantum yields via online UV-Vis spectroscopy. Lu, J.-M. et al. Roboticized AI-assisted microfluidic photocatalytic synthesis and screening up to 10,000 reactions per day. Bioucas-Dias, J. M. et al. Hyperspectral unmixing overview: geometrical, statistical, and sparse regression-based approaches. & Sidhu, H. Mechanisms of competing solvolytic elimination and substitution reactions. The role of ion-pair intermediates in aqueous solvents. Klaper, M., Wessig, P. & Linker, T. Base catalysed decomposition of anthracene endoperoxide. Ammer, J., Sailer, C. F., Riedle, E. & Mayr, H. Photolytic generation of benzhydryl cations and radicals from quaternary phosphonium salts: how highly reactive carbocations survive their first nanoseconds. Nishimae, Y., Kurata, H. & Oda, M. Arylbis(9-anthryl)methyl cations: highly crowded, near infrared light absorbing hydrocarbon cations. Nojima, M., Takagi, M., Morinaga, M., Nagao, G. & Tokura, N. Reaction of some triarylmethyl radicals, polyarylalkenes, and 9,10-dihydro-9,10-epidioxyanthracenes with sulphur dioxide; detection of radicals and/or cations derived from the corresponding cation radicals. Hollenstein, S. & Laali, K. K. Efficient conversion of 9-isopropenylphenanthrene to 4,6,6-trimethyl-6H-benz[de]anthracene in FSO3H; 5,6-dihydro-4H-benzanthracen-4-ium ion and its charge delocalization mode. & Krchňák, V. p-TSA-mediated four-component reaction: one-step access to mesoionic 1H-imidazol-3-ium-4-olates, direct NHC precursors. Medeiros, G. A. et al. Probing the mechanism of the Ugi four-component reaction with charge-tagged reagents by ESI-MS(/MS). A. D. Review on the Ugi multicomponent reaction mechanism and the use of fluorescent derivatives as functional chromophores. Alvim, H. G. O., da Silva Júnior, E. N. & Neto, B. A. D. What do we know about multicomponent reactions? Mechanisms and trends for the Biginelli, Hantzsch, Mannich, Passerini and Ugi MCRs. Chéron, N., Ramozzi, R., Kaïm, L. E., Grimaud, L. & Fleurat-Lessard, P. Challenging 50 years of established views on Ugi reaction: a theoretical approach. Hantzsch, A. Condensationsprodukte aus Aldehydammoniak und ketonartigen Verbindungen. Shen, L. et al. A revisit to the Hantzsch reaction: unexpected products beyond 1,4-dihydropyridines. The multicomponent Hantzsch reaction: comprehensive mass spectrometry monitoring using charge-tagged reagents. Chang, C.-C. et al. Antagonism of 4-substituted 1,4-dihydropyridine-3,5-dicarboxylates toward voltage-dependent L-type Ca2+ channels CaV1.3 and CaV1.2. Petrenko-Kritschenko, P. Über die kondensation des acetondicarbonsäureesters mit aldehyden, ammoniak und aminen. & Indra, A. Prussian blue- and Prussian blue analogue-derived materials: progress and prospects for electrochemical energy conversion. Li, W. et al. Chemical properties, structural properties, and energy storage applications of Prussian blue analogues. Choo, J. P. S. & Li, Z. Styrene oxide isomerase catalyzed Meinwald rearrangement reaction: discovery and application in single-step and one-pot cascade reactions. Guo, S. et al. Synthesis of trimetallic Prussian blue analogues and catalytic application for the epoxidation of styrene. Liang, Y. et al. Prussian blue analogues as heterogeneous catalysts for epoxidation of styrene. Pal, A. et al. Finding thermodynamically favorable pathways in chemical reaction networks using flows in hypergraphs and mixed-integer linear programming. & Wilmer, C. E. The ‘wired' universe of organic chemistry. Krzeszewski, M. et al. Computer-generated, mechanistic networks assist in assigning the outcomes of complex multicomponent reactions. Mikulak-Klucznik, B., Klucznik, T., Beker, W., Moskal, M. & Grzybowski, B. A. Catalyst: curtailing the scalable supply of fentanyl by using chemical AI. Mahjour, B., Shen, Y., Liu, W. & Cernak, T. A map of the amine–carboxylic acid coupling system. Baltussen, M. G., de Jong, T. J., Duez, Q., Robinson, W. E. & Huck, W. T. S. Chemical reservoir computation in a self-organizing reaction network. & Winfree, E. Enzyme-free nucleic acid logic circuits. Daniel, R., Rubens, J. R., Sarpeshkar, R. & Lu, T. K. Synthetic analog computation in living cells. Wołos, A. et al. Computer-designed repurposing of chemical wastes into drugs. Insight of solvent effect on CeO2 catalyzed oxidation of styrene with tert-butyl hydroperoxide: a combined experimental and theoretical approach. Jia, Y. et al. Code and raw data for ‘Robot-assisted mapping of chemical reaction hyperspaces and networks'. The Visualization Toolkit: An Object-oriented Approach to 3D Graphics 4th edn (Kitware, 2006). Ensemble samplers with affine invariance. A subspace, interior, and conjugate gradient method for large-scale bound-constrained minimization problems. Du, M. et al. High‐entropy Prussian blue analogues and their oxide family as sulfur hosts for lithium‐sulfur batteries. We thank A. Shevlyakov for helpful discussions. We thank Fan Guo for assistance with the EDS analysis. This work was generously supported by the taxpayers of South Korea through the Institute for Basic Science, project code IBS-R020-D1. These authors contributed equally: Yankai Jia, Rafał Frydrych Center for Algorithmic and Robotized Synthesis (CARS), Institute for Basic Science (IBS), Ulsan, Republic of Korea Yankai Jia, Rafał Frydrych, Yaroslav I. Sobolev, Wai-Shing Wong, Bibek Prajapati, Daniel Matuszczyk, Yasemin Bilgi, Louis Gadina, Juan Carlos Ahumada, Galymzhan Moldagulov, Namhun Kim, Eric S. Larsen, Maxence Deschamps, Yanqiu Jiang & Bartosz A. Grzybowski Yankai Jia, Galymzhan Moldagulov & Bartosz A. Grzybowski Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar constructed the automation hardware, designed its software, conducted experiments with R.F. contributed to automation design, led the chemical part of the project, selected, performed and interpreted all reactions in roboto and performed MS and MS/MS mechanistic studies. designed software for spectral unmixing and numerical models of reaction kinetics (with input from R.F. ), fitted kinetics to the experimental data, performed TD-DFT calculations of absorption spectra, quantum thermochemistry calculations and theoretical study of the smoothness of yield maps. performed all Hantzsch reactions with the robotic platform and partial HPLC purifications of the Hantzsch reaction and performed one reaction for versatility test. proposed the reaction, made the proof-of-concept test and determined products structure and reaction mechanism for PBA catalysts. worked on synthesis, products isolation, structure determination and mechanism analysis for PBA-catalysed reactions. also performed and analysed the Suzuki–Miyaura coupling experiments. Y. Jiang worked on PBA synthesis, reactions on robotic platform, data analysis and wrote the initial draft for the PBA part. worked on four reactions for the versatility test, assigned structures to unexpected Hantzsch reaction products and proposed the corresponding mechanisms. and others conducted a reaction versatility test. conducted the system accuracy test. contributed to robotic synthesis (Hantzsch reaction), separation of crude mixtures, purification, NMR analysis of isolated compounds and performed one reaction for the versatility test. analysed single-crystal X-ray diffraction data. conducted synthesis and aided with the analysis and determination for the rearrangement reaction of anthracen-9-yldiphenylmethanol and analysed several sets of single-crystal X-ray diffraction data to determine their three-dimensional structures. conceived and supervised the project. Correspondence to Yanqiu Jiang or Bartosz A. Grzybowski. The authors declare no competing interests. Nature thanks Guillermo Restrepo and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Examples of reactions compatible (green background), borderline (yellow) and incompatible (pink) with yield determination using UV-Vis and spectral unmixing. The compatible reactions include but are not limited to Claisen–Schmidt condensation 1, Ullmann-type coupling 2, Suzuki–Miyaura cross-coupling 3, Cu(I)-catalysed alkyne-azide cycloaddition 4, Friedel–Crafts acylation 5, imine condensation 6, Glaser-type coupling 7 or Beckmann rearrangement 8. The Diels–Alder reaction 9 shown on yellow background is borderline in terms of the quality of unmixing (the overlap of the UV-Vis signals from the product and from dimethyl acetylenedicarboxylate prevents accurate fitting, leading to an overestimation of the yield). Other reactions unsuitable for the method are, for example, SN2 reaction of alkyl bromide 11 (not absorbing above 220 nm) or condensation of dichlorodiaminobenzene with phenanthrenequinone 12 (poor solubility of the product). b, R2 = 0.96 correlation between the optically determined yields versus the yields of the same reactions determined by conventional means (HPLC for E1 from Fig. 3 and Ugi-type MCR from Fig. 4; 1H NMR for Claisen–Schmidt condensation 1 and imine condensation 6; the weights of isolated compounds for Cu(I)-catalysed alkyne-azide cycloaddition 4 and Beckmann rearrangement 8). For numerical values of each data point, see Supplementary Tables 5–7 in Supplementary Information Sections 4.3 and 4.4. a, Taking as an example a basic bimolecular reaction A + B → C (with kinetic rate k[A][B], reaction time t = 1 and kinetic constant k = 1), the plot has the yield and concentration of C for various combinations of A and B substrates. As seen, yield—defined with respect to the concentration of the limiting reagent—tends to maximize at the boundaries of the region (orange curve in a) but the concentration of the product tends to maximize in its interior (blue curve in a). There are two implications of this ‘inverse' dependence: (1) yields at very low concentrations may be artificially high, whereas—in reality—they involve very high (wasteful) excesses of some reagents; (2) detecting and quantifying the product reliably may be compromised in such a low-concentration regime. Accordingly, in the current work—especially for the multicomponent mixtures—we have worked over concentration ranges that do not extend to very small values, thereby assuring reliable detection while avoiding highly wasteful excess ratios. Such a ‘practical' range is shown schematically in a by the dashed vertical lines. Another corollary is that discovery of unexpected products may be severely compromised when using a strategy of investigating only the edges and boundaries of an n-dimensional region while leaving the interior unexplored—a strategy that may otherwise seem intuitive to a chemist and has some merit in maximization of the yield of known products. c,d, Comparison of final concentration map (c) to yield map (d) for product 19e in Hantzsch reaction. Note the difference between the conditions that maximize concentration versus ones that maximize yield: yield is maximized at the vertices of the cube, whereas concentration is maximized in the middle of the front face and top edge, with higher concentrations protruding deeper into the interior of the cube (see blue curve in a). Supplementary Information about materials, robotic platform, data processing, experimental procedures and results, kinetic models, quantum chemical calculations and theoretical perspective on the ‘slopes' of yield maps. HyperspaceViewer software for interactive exploration of multidimensional yield maps. Functionality is shown in Supplementary Video 4. Liquid transfer for volume calibration, reaction preparation and UV-Vis spectrum measurement. 00:00-00:11, pipetting to analytical balance for volume calibration. 00:25-00:41, pipetting to UV-Vis spectrophotometer for spectrum acquisition. 00:00-00:16, transferring reaction crude from a vial to the spectrophotometer. 00:22-00:28, cleaning detection area of spectrophotometer. 00:29-00:57, exemplary measurements of another nine samples in the same reaction plate. Experimental (left) and theoretical (right) four-dimensional map of the yield of Ugi multicomponent reaction. Yield is shown by colour and size of spheres. Three-dimensional space coordinates are the starting concentrations of isocyanide, amine and aldehyde. Experimental data have been smoothed for this illustration, as described in Supplementary Information Section 3.13. For comparison of the smoothed and raw data, see Supplementary Information Section 4.15.4. Theoretical yield map (right half of the video) corresponds to the kinetic model best fit to the data, as described in Supplementary Information Section 5.3. The same theoretical yields are shown by the blue curves in Supplementary Figs. 116–128 (Supplementary Information Section 4.13.4) for all 3,234 conditions tested in experiments. Illustration of the house-written HyperspaceViewer software allowing for direct visualization/analysis of three-dimensional and four-dimensional hyperspace data. The software features multi-isosurface rendering, interactive data point analysis and intuitive cube manipulation. The isosurfaces are generated through three sequential steps: octree generation, data interpolation and meshing by the marching cubes algorithm. The application is available in the Supplementary Files of this article. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Robot-assisted mapping of chemical reaction hyperspaces and networks. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09531-9'>Industry-compatible silicon spin-qubit unit cells exceeding 99% fidelity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:15:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Among the many types of qubit presently being investigated for a future quantum computer, silicon spin qubits with millions of qubits on a single chip are uniquely positioned to enable quantum computing. However, it has not been clear whether the outstanding high-fidelity operations and long coherence times shown by silicon spin qubits fabricated in academic settings1,2,3,4,5,6,7,8 can be reliably reproduced when the qubits are manufactured in a semiconductor foundry9,10,11. Here we show precise qubit operation of silicon two-qubit devices made with standard semiconductor tooling in a 300-mm foundry environment. Of the key metrics, single- and two-qubit control fidelities exceed 99% for all four devices, and the state preparation and measurement fidelities reach up to 99.9%, as evidenced by gate set tomography. We report spin lifetime and coherence up to T1 = 9.5 s, \({T}_{2}^{* }=40.6\,{\rm{\mu }}{\rm{s}}\) and \({T}_{2}^{{\rm{Hahn}}}=1.9\,{\rm{ms}}\). We determine that residual nuclear spin-carrying isotopes contribute substantially to operational errors, identifying further isotopic purification as a clear pathway to even higher performance. The early academic successes of silicon spin-qubit technology1,2,3,4,5,6,7,8 created enthusiasm for the idea that these qubits could be fabricated in industrial 300-mm manufacturing lines9,10,11,12. Potential advantages include the ability to leverage advanced fabrication processes, to integrate qubits with modern electronics and to benefit from the existing capabilities of mass manufacturing at low costs12. Over the past decade, there have been a number of proposals for potential architectures that could be used to realize a scalable quantum computer based on silicon spin qubits13,14,15,16,17. However, recent attempts to fabricate spins-in-silicon quantum dots using industrial-scale fabrication has raised awareness about the material challenges that must be faced. For example, it is crucial to reduce the charge noise and static disorder arising due to defects and traps at interfaces and oxides18,19,20. High-fidelity single-qubit operations can be performed with minimal impact from charge noise because the electron spin couples to the noisy electric fields only through its weak spin–orbit effects21,22,23,24,25. By contrast, other operations, including qubit initialization, read-out and exchange-based two-qubit gates, are more sensitive to the charge noise present in conventional metal–oxide–semiconductor gate stacks19,24. This sensitivity has motivated the search for ways of mitigating the effects of charge noise, including moving spin states away from the interface to quantum wells26,27. However, from a scaling perspective, it is desirable to reconcile as much as possible conventional complementary metal–oxide–semiconductor (CMOS) processes with qubit fabrication to leverage the mature CMOS industry to the fullest extent. Moreover, the formation of quantum dots against thin oxides has several benefits, such as a large electric field that leads to large valley splitting18,28,29 and efficient electrostatic coupling between the gate voltage bias and the channel, allowing for well-isolated states, efficient exchange coupling control29 and fast gate-based read-out30. In this work, we study four two-qubit devices fabricated using a long-established CMOS geometry, namely a planar metal–oxide–semiconductor with polysilicon gates31. We demonstrate for all devices all basic qubit operations, including one- and two-qubit gates, initialization and read-out, with error rates approaching and surpassing the fault-tolerance threshold for the widely studied surface error correction code32. We obtain these error rates using state-of-the-art gate set tomography (GST) tools33,34, allowing us to pinpoint effects such as crosstalk and the breakdown between stochastic and coherent errors. Our studies indicate unprecedented reproducible low error levels while also identifying clear pathways to further reducing errors. The qubit devices were designed by Diraq and fabricated at imec in a 300-mm spin-qubit process flow that is described in Methods. The resulting wafer consists of several devices covering a range of feature sizes and configurations. We chose the design that we estimated would have optimal device parameters, and we tested four of these devices (denoted A–D). The design consists of a double quantum dot and a nearby single-electron transistor (SET) for spin read-out1 (Fig. a, Schematic of a Diraq two-qubit device fabricated on a 300-mm wafer, showing the full wafer, single die and single device level. b, Charge stability diagram of device A as a function of plunger (P1 and P2) voltage detuning ΔVP = ΔVP1 = −ΔVP2 and exchange gate (J) voltage VJ, showing four isolated electrons in the double-dot system. Devices B–D were also measured in the (3,1) charge configuration and showed analogous charge stability diagrams. Voltage points for single-qubit operation (Joff < 1 kHz), two-qubit operation (Jon ≈ 1 MHz) and read-out are shown as a triangle, inverted triangle and star, respectively. The faint charge transition starting from VJ = 1.05 V is very weakly coupled to the quantum dot and probably corresponds to charge movement outside the active dot confinement area. In device D, we observed similar charge movements outside the active dot region; devices B and C showed none. Inset, Cross section of the double-quantum-dot device indicating the number of electrons under the plunger gates (P1 and P2), exchange gate (J) and double potential well. We operated the devices in a 3He/4He dilution refrigerator with a base temperature of 10 mK in isolated mode, with four electrons in the double dot formed under the plunger gate electrodes (P1 and P2). A voltage applied to an interstitial exchange gate (J) controlled the tunnel coupling between the dots1,35,36. The electrons were loaded from a two-dimensional electron gas formed under the reservoir gate, which overlaps with an n++ implanted ohmic region. SET charge-noise measurements reveal a 1/fα power spectral density with α between 0.50(7) and 1.19(5) and an amplitude of 0.79(4)–\(0.98(7)\,{\rm{\mu }}{\rm{eV}}/\sqrt{{\rm{Hz}}}\) at 1 Hz (Extended Data Fig. 1), in line with previous low charge-noise results from devices fabricated using similar methods19,21. A static magnetic field B0 in the range 0.662–0.7 T split the spin states to form the single-qubit states, which were manipulated with on-resonance microwave signals applied to the electron spin resonance antenna with frequency fESR. Single-qubit Xπ/2 gates were formed with timed microwave pulses on resonance with the qubit Larmor frequencies fL,Q1 and fL,Q2 (Fig. 2), whereas Zπ/2 gates were implemented as virtual gates through rotations of the reference frame37. The single-qubit metrics, including spin lifetime and Ramsey and Hahn coherence times, for the four devices are shown in Table 1. Probability of reading out an even or blockaded spin state Peven, when applying microwave signal at the frequency fESR – fLO for a duration of tESR. c,d, Coherent Rabi oscillations at fL,Qi for qubits 1 (c) and 2 (d). Real-time feedback is implemented to counteract the Larmor frequency deviation39. The local oscillator frequency is set to fLO = 18.610 GHz. The microwave power applied to qubit 1 is approximately 7% larger than that to qubit 2 resulting in matching Rabi frequencies of fRabi = 658.6(3) kHz. Figure 3 details our implementation of two-qubit control. A voltage pulse, typically of duration tCZ between 100 ns and 500 ns, was applied to the J gate to control the exchange interaction strength (Fig. We extracted from the fitted exchange frequencies in Fig. 3c an exchange controllability of 18.4 dec V−1 (devices A–D are compared in Fig. By combining a pulsed exchange interaction with single-qubit phase rotations, we were able to implement a controlled-Z (CZ) two-qubit gate1 (Fig. Using real-time feedback, we tracked and corrected the SET voltage operation point and the single-qubit Larmor and Rabi frequencies39 (see Extended Data Fig. We used a heralded initialization protocol to check if the qubit system was successfully initialized in the \(| \downarrow \downarrow \rangle \) state, and we repeated the initialization if unsuccessful7,40. a,b, Probability of detecting an even spin state Peven after a microwave burst of fixed power and duration at different J gate voltages VJ when preparing a mixed odd state of \(| \downarrow \uparrow \rangle \) and \(| \uparrow \downarrow \rangle \) (a) and a pure even state \(| \downarrow \downarrow \rangle \) (b). The power and duration of the microwave burst are roughly calibrated to a single-qubit π rotation. The local oscillator frequency is set to fLO = 18.610 GHz. The mixed odd state is initialized with a tramp = 1 μs ramp at VJ = 0.96 V over the anti-crossing and the even state with heralded initialization based on a tramp = 1 μs ramp at VJ = 1.12 V. All experiments throughout the paper were conducted with an even state \(| \downarrow \downarrow \rangle \) initialization, unless otherwise specified. c, Controlled phase (CZ) oscillation as a function of exchange time texchange and exchange gate voltage VJ. We apply dynamical decoupling by a consecutive π rotation on both qubits to filter out other phase-accumulating effects, such as Stark shift. d, Decoupled exchange oscillation fingerprint for fixed exchange time texchange = 10 μs as a function of plunger voltage detuning ΔVP and exchange gate voltage VJ. e, Exchange strength as a function of exchange gate voltage extracted from fitting exchange oscillations in c. The controllability is fitted assuming Jexchange ∝ a exp bVJ, resulting in a range between b = 9.78 dec V−1 and 18.44 dec V−1 across devices A–D. The variation in exchange controllability across the devices is probably due to fabrication variability. f,g, Calibration of the CZ single-qubit phase correction by preparing the target spin in superposition, applying a CPHASE followed by a virtual phase rotation for qubits 1 (f) and 2 (g)4,7. Vertical lines indicate the phase values for which the spin state of Q1 (Q2) is flipped if Q2 (Q1) is in the on/spin-up state. The read-out probability is unscaled in all data. To precisely capture errors in qubit operation, we designed a GST experiment comprising the full gate set {II, XI, IX, ZI, IZ, CZ}, from which we constructed a list of ‘preparation' and ‘measurement fiducials' circuits. Combining these fiducials with ‘germ' circuits up to length 16, we obtained a list of 12,263 unique sequences34. We chose GST as the benchmarking method because it yields the error taxonomy with specific error channels and is generally a higher bar for fidelities than the more commonly used interleaved randomized benchmarking, which often overestimates fidelities8. We implemented an idle gate with a total idle time equal to the duration of a CZ gate. Single-qubit gates in the two-qubit context were achieved by rotating one qubit π/2 around the \(\widehat{x}\) axes (XI and IX) and \(\widehat{z}\) axes (ZI and IZ), while the other qubit was idling. The XI and IX gates were implemented by applying a microwave square pulse of around 400 ns, including a 20-ns smooth sine ramp, at fL,Q1 and fL,Q2 for qubits 1 and 2, respectively. The ZI and IZ gates were implemented by virtually rotating the reference frame for qubits 1 and 2, respectively. The entangling two-qubit CZ gate was implemented by applying a voltage pulse to the exchange gate J for a duration of tCZ. Figure 4 breaks down the error generators and their contribution to the infidelity for each gate and device. Across all four devices operation accuracies are above 99%. For example, for device A, the gate fidelities are 99.45(5)%, 99.55(4)%, 99.95(4)%, 99.96(4)%, 99.74(9)% and 99.37(5)% for the XI, IX, ZI, IZ, II and CZ, respectively (see first row of Table 1). The combined state preparation and measurement (SPAM) fidelity even exceeds 99.9% with FSPAM = 99.95(8)%, which is one of the highest reported values for industrially fabricated spin qubits (Extended Data Fig. Single-qubit on-target gate fidelities FX,Q1 = 99.97(15)% and FX,Q2 = 99.81(11)% are comparable to previously reported values for devices made using the same fabrication processes21. a–d, Infidelity contributions of Hamiltonian and stochastic error generators across all four devices for the XI (a), IX (b), II (c) and CZ (d) gates from GST. Hamiltonian errors contribute to the infidelity in second order, whereas stochastic errors contribute in first order. Stochastic errors, particularly the idling ZI and IZ error generator components, dominate the infidelity by more than an order of magnitude. Error bars represent the 95% confidence level. For visibility, only those of the IZ and ZI error generators are displayed. Furthermore, the stochastic IZ and ZI error generators dominate the infidelity contributions across all devices. In particular, the stochastic IZ component is larger for the CZ gate compared with the I gate, which is probably related to the (3,1) charge configuration used in all devices. In future work, a systematic study of the (1,3) charge configuration, as well as higher electron occupations such as (3,3) across several devices, would provide valuable insights into the physical origin of the observed error mechanisms. SPAM fidelities reproducibly exceed 99.9% for three out of the four devices (devices A, C and D). For device B, we achieved a SPAM fidelity of 99.33(9)%. We attribute this slightly reduced SPAM fidelity to signs of gate leakage between the SET barrier and the ohmic contacts. Similarly, device D shows leakage between the SET barrier and the plunger gate used to form qubit 2, which is the probable cause of the reduced spin relaxation and coherence time of qubit 2 (Table 1). Additionally, we performed d.c. cryo-probing measurements on a further 16 devices: one with the same design parameters as devices A–D and 15 with different design parameters, all from the same imec wafer. These measurements indicate excellent uniform electrostatic control over all device gate electrodes (Extended Data Fig. Device A exemplifies the high performance of qubits on the 300-mm wafer and can, therefore, be used as a case study to examine the sources of infidelity. In the following sections, we analyse the GST error contributions and their probable physical origins in device A, as well as its qubit performance as a function of magnetic field. 4 and the similarity of the Larmor frequency dynamics in Extended Data Fig. 3, we expect this conclusion to hold for all four devices. The ZI and IZ gates were implemented virtually and are, therefore, in principle error-free. However, the clock update on the field-programmable gate array took a finite time, leading to slight imperfections in these virtual operations due to idling errors. Hence, we attribute their infidelities, 0.05(4)% (0.04(4)%) for ZI (IZ), to a small time delay in the control hardware. Similarly, the infidelity contribution to the II gate from the stochastic dephasing error generator IZ (0.15%) is larger than for ZI (0.09%). First, these error contributions could be explained by the coupling of 29Si nuclear spins to the electron spin state of qubit 2 (ref. Qubit 1 also experiences quasi-static changes in the Larmor frequency, but these are of lower magnitude, despite the Stark shifts of qubit 1 being larger (Extended Data Fig. This is another indication that hyperfine-coupled residual 29Si nuclear spins are the origin of that noise. As expected, these idling errors are roughly doubled (tXI = tIX = 400 ns) in comparison with the idling gate, which only takes around half the time. However, during CZ gate operation, the tunable exchange barrier is lowered to enable the Heisenberg exchange. Second, it might seem contradictory that the gate infidelity is dominated by noise in qubit 2, given that \({T}_{2,{\rm{Q}}2}^{{\rm{Hahn}}} > {T}_{2,{\rm{Q}}1}^{{\rm{Hahn}}}\) and \({T}_{2,{\rm{Q}}2}^{* }\approx {T}_{2,{\rm{Q}}1}^{* }\). However, the GST experiments are designed to amplify, not echo out, persistent noise sources. They are, therefore, heavily affected by slow noise, such as Larmor frequency changes arising from hyperfine-coupled nuclear spins. On the other hand, Hahn echo experiments are limited by fast charge noise. The Larmor frequency variation, tracked during the GST experiment using real-time feedback39,40, shows a higher standard deviation for the qubit 2 frequency variation compared with qubit 1 (Extended Data Fig. Third, sequences between which the Larmor frequency feedback exhibits a large frequency change can be removed from the GST dataset, but this has a significant effect only on the stochastic IZ error generator of the XI gate (Extended Data Fig. 8 and 9) and the spin–orbit interaction in qubit 1 is stronger compared with qubit 2, as can be seen in Fig. The standard deviation of the Larmor frequency feedback increases in a step-like manner as a function of total experiment runtime for distinct instances across all magnetic fields (Extended Data Fig. Hence, these slow timescale jumps probably originate from qubit 2 coupling strongly to the nuclear spin states of the residual 29Si atoms. These slow jumps in the Larmor frequency do not contribute to \({T}_{2}^{{\rm{Hahn}}}\) and, therefore, do not contradict \({T}_{2,{\rm{Q}}2}^{{\rm{Hahn}}} > {T}_{2,{\rm{Q}}1}^{{\rm{Hahn}}}\). In terms of qubit device performance, Fig. 5 shows spin lifetime and coherence metrics as functions of the magnetic field B0. T1 and \({T}_{2}^{* }\) first increase up to B0 = 0.3–0.4 T, followed by a decrease for larger B0. Because \({T}_{2}^{* }\) is sensitive to d.c. noise, it samples 1/f noise down to frequencies corresponding to the total time of the experiment, which renders it a bad metric for comparing different qubits or qubit platforms. Including Larmor frequency feedback in the experiment reduces the contribution of low-frequency 1/f noise44. We demonstrate this in Extended Data Fig. 11a,b by plotting \({T}_{2}^{* }\) as a function of the cumulative measurement time. Most curves are approximately flat, showing the effectiveness of the feedback mechanism. 11c,d show the variation of \({T}_{2}^{* }\) as a function of laboratory time. a–c, Spin lifetime T1 (a), Ramsey \({T}_{2}^{* }\) (b) and Hahn echo \({T}_{2}^{{\rm{Hahn}}}\) (c) metrics as functions of the applied magnetic field amplitude B0. We measured both qubits for each experiment interleaved and repeated this 50 (T1), 10,000 (\({T}_{2}^{* }\)) and 2,000 (\({T}_{2}^{{\rm{Hahn}}}\)) times. The Hahn echo experiment at B0 = 1.3 T was repeated 500 times. Between each repeat we used real-time Larmor frequency feedback following the protocol described in ref. The data were fitted using the functions described in Extended Data Fig. Data shown as stars correspond to metrics measured during two-qubit benchmarking (Extended Data Fig. 2), using a method and gate voltages that differed from those used for the magnetic field sweep. Error bars represent the 95% confidence level. The longest measured coherence times for device A include Hahn echo coherence time \({T}_{2}^{{\rm{Hahn}}}=\text{1,900}\,{\rm{\mu }}{\rm{s}}\) (1,805 μs) for qubit 2 (qubit 1) measured at the low magnetic field of B0 = 50 mT, as well as \({T}_{2}^{* }\) = 40.6 μs (39.0 μs) and T1 = 9.5 s (4.7 s). The long \({T}_{2}^{{\rm{Hahn}}}\) and its small variation between both qubits indicate that magnetic noise dominates at low fields. On increasing B0, the Stark shift increases linearly (Extended Data Fig. 9), with steeper slopes for qubit 1 (ΔΩStark(P1, J, P2) = (−19.9(4), 31.3(3), 14.2(4)) MHz V−1 T−1) than for qubit 2 ((0, 3.2(2), 16.7(4)) MHz V−1 T−1), manifesting in stronger coupling to charge noise through the spin–orbit interaction and resulting in reduced coherence, with a 33% larger \({T}_{2,Q2}^{{\rm{Hahn}}}=617(26)\,{\rm{\mu }}{\rm{s}}\) (\({T}_{2,Q1}^{{\rm{Hahn}}}=464(22)\,{\rm{\mu }}{\rm{s}}\)) at B0 = 1.3 T. The observed Stark shift is entirely caused by the intrinsic spin–orbit coupling of silicon electrons confined at the Si/SiO2 interface. This type of spin–orbit coupling is largely influenced by atomistic roughness at the Si/SiO2 interface29. In device A, we obtained −20 MHz V−1 T−1 to 30 MHz V−1 T−1, which falls within that range. The magnetic field dynamics of the spin coherence times are distinct from previous results for academic devices7, and the exact physical origin requires a more detailed study, which is beyond the scope of this work. Looking forward, the increase in \({T}_{2}^{{\rm{Hahn}}}\) for low fields is extremely encouraging because it indicates that the qubits are amenable to more scalable control techniques. Such approaches continuously drive the spin by using a modulated microwave field to decouple it from noise and eliminate free precession (\({T}_{2}^{{\rm{Hahn}}}\gg {T}_{2}^{* }\))36,45. Operation at lower fields requires a different modality of the two-qubit gate. To achieve similarly fast gates, a SWAP-like operation will be implemented. Furthermore, at low fields an always-on, global control is beneficial for avoiding idling errors and using the long decoupling times \({T}_{2}^{{\rm{Hahn}}}\gg {T}_{2}^{* }\) observed. The key challenge for that operation modality is sufficiently large Stark shifts. Our chosen implementation of an entangling gate (CZ) is faster than decoupling schemes, but it relies on the exchange interaction being smaller than the Zeeman energy difference of the two qubits (Jexc ≪ ΔEZ). We chose B0 values in the range 0.662–0.7 T to meet this requirement, but in future, we intend to explore other two-qubit gate implementations8. We have demonstrated high-fidelity control of semiconductor qubits industrially manufactured in a 300-mm pilot line. In contrast to previous publications in which charge noise is identified as the key limiting factor3,4,5,8,38,46,47, our results are strongly encouraging from a noise-analysis perspective, both in terms of the fidelity values themselves, and in view of the clear pathway they pave towards further improvement through further nuclear purification. Further isotopic enrichment of the 400-ppm 29Si used in this work to levels at or below 50 ppm has already been demonstrated in academic prototype devices7. This is consistent with previous results showing the effect of nuclear spin noise on coherence times in Si/SiGe quantum-dot spin qubits20. We have shown that using engineering practices in an industrial setting can lead to state-of-the-art spin-qubit performance when coupled with a systematic search for target performance metrics. It is crucial to understand the connection between qubit performance and measurements that can be performed during the fabrication process, such as electrical noise and Hall bar transport19,48. These investigations are in their infancy and will require an increasingly accurate model of the quantum behaviour of electrons in semiconductor devices. In the long term, further improvements will be required on the qubit front, for which our noise analysis provides a first-order angle of attack. To reduce the overhead for fault-tolerant quantum computing32, 99.9% fidelities across all operations is a realistic target. At this stage, a statistical analysis of these qubits at a larger scale is not yet available. The consistent calibration of qubits at the level demonstrated here remains an intensive manual process with a number of scientific insights gained along the way. Automation of this process needs to mature to consistently achieve these results in a mass-characterization campaign49. Having silicon–metal–oxide–semiconductor (SiMOS) qubits fabricated in a foundry environment readily available will facilitate the future demonstration of large-scale SiMOS qubit arrays. This has so far been challenging for SiMOS spin qubits because it has been difficult to fabricate devices with small gate pitches and low charge-noise interfaces. These results could be leveraged to design quantum error correction strategies to cater to the specific demands of spin qubits. However, further studies will be needed to characterize qubit operations in a more scalable setting, such as under a single global microwave field and under elevated temperatures, which will inevitably be imposed by the heat dissipated by the co-integrated control electronics on the CMOS chip50,51,52. Our reproducible demonstrations of high-quality qubit operation made using 300-mm foundry processes offer a proof of principle that should encourage further industrial research into far more complex technologies that are at the disposal of foundries. The fabrication method used at imec was optimized for low charge noise, high uniformity19,31,53,54 and qubit-specific integration with a gate pitch smaller than 100 nm. Fast fabrication cycles and design flexibility were ensured by optimally combining optical lithography and electron-beam lithography. Fabrication started with the epitaxial growth of an isotopically enriched layer of silicon with a residual concentration of 400 ppm 29Si. Dedicated cleaning processes and 300-mm pilot line process control are key to the development of the high-quality interface19. Next, a triple-layer overlapping polysilicon gate stack was formed using electron-beam lithography and dry etching, with each gate separated by a thin interstitial high-temperature oxide deposit19. Finally, an aluminium strip line was patterned so that single spin states could be manipulated using electron spin resonance, offering a control solution at the few-qubit level50,52 (Extended Data Fig. Devices A, B, C and D were measured in Bluefors XLD400, Bluefors LD400, and two different Oxford Instruments K100 Kelvinox dilution refrigerators, respectively, and mounted on the cold finger. Previous work conducted with similar experimental set-ups indicates that the electron temperature was 200–300 mK (ref. An American Magnetics AMI430 supplied static external magnetic fields to devices A, B and C, with the field for A pointing in the [110] direction of the Si lattice and those for B and C pointing in the [1\(\overline{1}\)0] direction. We used a Q-Devil QDAC-II to supply d.c. voltages through filtered lines with a bandwidth of up to 20 Hz. Dynamic voltage pulses were generated with a field-programmable gate array, in this experiment the Quantum Machines Operator-X+ (OPX+). The pulses were combined with d.c. biases using custom linear bias combiners at room temperature. The OPX+ had a sampling time of 1 ns. The dynamic pulse lines in the fridge had a bandwidth of up to 50 MHz, which translated into a minimum rise time of 20 ns. We accounted for this by smoothing out fast a.c. pulsing. Microwave pulses were generated by a vector signal generator (PSG8267D, Keysight). The in-phase and quadrature and pulse modulation waveforms were generated by the OPX+. The charge sensor comprised a single-island SET in d.c. mode. The SET current integrated for tint = 100 μs was amplified using a room-temperature I–V converter (Basel SP983c) and sampled by the OPX+. At this stage, we used d.c. mode read-out, which was limited by the bandwidth and noise of the room-temperature transimpedance amplifier, leaving high-bandwidth cryogenic current amplification for future work. Gate times were optimized based on a trade-off between speed and lifetime. The magnetic field amplitude B0 was chosen in the range 0.662–0.7 T, such that the single-qubit Rabi QRabi = fRabiTRabi was maximized. We chose our B0 values to meet this requirement. The data supporting this work are available at Zenodo (https://doi.org/10.5281/zenodo.15571656)57. Veldhorst, M. et al. A two-qubit logic gate in silicon. Huang, W. et al. Fidelity benchmarks for two-qubit gates in silicon. Noiri, A. et al. Fast universal quantum gate above the fault-tolerance threshold in silicon. Xue, X. et al. Quantum logic with spin qubits crossing the surface code threshold. Mills, A. R. et al. Two-qubit silicon quantum processor with operation fidelity exceeding 99%. Weinstein, A. J. et al. Universal logic with encoded spin qubits in silicon. Huang, J. Y. et al. High-fidelity spin qubit operation and algorithmic initialization above 1 K. Nature 627, 772–777 (2024). Tanttu, T. et al. Assessment of the errors of high-fidelity two-qubit gates in silicon quantum dots. Zwerver, A. M. et al. Qubits made by advanced semiconductor manufacturing. Maurand, R. et al. A CMOS silicon spin qubit. Camenzind, L. C. et al. A hole spin qubit in a fin field-effect transistor above 4 kelvin. Dumoulin Stuyck, N. et al. CMOS compatibility of semiconductor spin qubits. Veldhorst, M., Eenink, H. G., Yang, C. H. & Dzurak, A. S. Silicon CMOS architecture for a spin-based quantum computer. Li, R. et al. A crossbar network for silicon quantum dot qubits. Scaling silicon-based quantum computing using CMOS technology. The SpinBus architecture for scaling spin qubits with electron shuttling. & Fogarty, M. Towards early fault tolerance on a 2 × N array of qubits equipped with shuttling. Saraiva, A. et al. Materials for silicon quantum dots and their impact on electron spin qubits. Elsayed, A. et al. Low charge noise quantum dots with industrial CMOS manufacturing. Neyens, S. et al. Probing single electrons across 300-mm spin qubit wafers. Demonstration of 99.9% single qubit control fidelity of a silicon quantum dot spin qubit made in a 300 mm foundry process. Jock, R. M. et al. A silicon metal-oxide-semiconductor electron spin-orbit qubit. Controlling spin-orbit interactions in silicon quantum dots using magnetic field direction. Burkard, G., Ladd, T. D., Pan, A., Nichol, J. M. & Petta, J. R. Semiconductor spin qubits. Cifuentes, J. D. et al. Impact of electrostatic crosstalk on spin qubits in dense CMOS quantum dot arrays. Lawrie, W. I. L. et al. Quantum dot arrays in silicon and germanium. The germanium quantum information route. Yang, C. H. et al. Spin-valley lifetimes in a silicon quantum dot with tunable valley splitting. Cifuentes, J. D. et al. Bounds to electron spin qubit variability for scalable CMOS architectures. Gonzalez-Zalba, M. F. et al. Gate-sensing coherent charge oscillations in a silicon field-effect transistor. Li, R. et al. A flexible 300 mm integrated Si MOS platform for electron- and hole-spin qubit exploration. Fowler, A. G., Mariantoni, M., Martinis, J. M. & Cleland, A. N. Surface codes: towards practical large-scale quantum computation. Demonstration of qubit operations below a rigorous fault tolerance threshold with gate set tomography. Nielsen, E. et al. Probing quantum processor performance with pyGSTi. Chittock-Wood, J. F. et al. Exchange control in a MOS double quantum dot made using a 300 mm wafer process. Seedhouse, A. E. et al. Pauli blockade in silicon quantum dots with spin-orbit control. Vandersypen, L. M. K. & Chuang, I. L. NMR techniques for quantum control and computation. Reduced sensitivity to charge noise in semiconductor spin qubits via symmetric operation. Dumoulin Stuyck, N. et al. Silicon spin qubit noise characterization using real-time feedback protocols and wavelet analysis. Steinacker, P. et al. Bell inequality violation in gate-defined quantum dots. Muhonen, J. T. et al. Storing quantum information for 30 seconds in a nanoelectronic device. Hensen, B. et al. A silicon quantum-dot-coupled nuclear spin qubit. Zhao, R. et al. Single-spin qubits in isotopically enriched silicon at low magnetic field. Coherence of a driven electron spin qubit actively decoupled from quasistatic noise. Hansen, I. et al. Entangling gates on degenerate spin qubits dressed by a global field. Struck, T. et al. Low-frequency spin qubit energy splitting noise in highly purified 28Si/SiGe. Yoneda, J. et al. A quantum-dot spin qubit with coherence limited by charge noise and fidelity higher than 99.9%. Camenzind, T. N. et al. High mobility SiMOSFETs fabricated in a full 300 mm CMOS process. Zwolak, J. P. & Taylor, J. M. Colloquium: Advances in automation of quantum dot devices control. Coherent control of electron spin qubits in silicon using a global field. Bartee, S. K. et al. Spin-qubit control with a milli-kelvin CMOS chip. Vahapoglu, E. et al. Single-electron spin resonance in a nanoelectronic device using a global field. Dumoulin Stuyck, N. et al. An integrated silicon MOS single-electron transistor charge sensor for spin-based quantum information processing. Uniform spin qubit devices with tunable coupling in an all-silicon 300 mm integrated process. 2021 Symposium on VLSI Circuits 1–2 (IEEE, 2021). Spence, C. et al. Probing low-frequency charge noise in few-electron CMOS quantum dots. Yang, C. H. et al. Operation of a silicon quantum processor unit cell above one kelvin. Steinacker, P. Data for ‘Industry-compatible silicon spin qubit unit cells exceeding 99% fidelity'. Spin lifetime and charge noise in hot silicon quantum dot qubits. Elsayed, A. et al. Comprehensive 300 mm process for silicon spin qubits with modular integration. 2023 IEEE Symposium on VLSI Technology and Circuits (VLSI Technology and Circuits) 1–2 (IEEE, 2023). We thank C. Bohémier for his assistance with the cryogenic screening and A. Morello and J. Cole for useful discussions. We thank S. Pauka, T. Gaebel, J. Dehollain and A. Torgovkin for assistance with the cryogenic set-ups. We acknowledge support from the Australian Research Council (Grant No. FL190100167) and the U.S. Army Research Office (W911NF-23-10092). acknowledge support from the Sydney Quantum Academy. acknowledges support from the Baxter Charitable Foundation. are recipients of an Australian Research Council Industrial Fellowship (Project Nos. IE240100252 and IM230100396) funded by the Australian Government. For open access purposes, we have applied a CC BY public copyright licence to any author-accepted manuscript version arising from this submission and have chosen to publish the final version of record as open access under a CC BY licence. is the Proximus Chair in Quantum Science and Technology. Open access funding provided through UNSW Library. These authors contributed equally: Paul Steinacker, Nard Dumoulin Stuyck School of Electrical Engineering and Telecommunications, University of New South Wales, Sydney, New South Wales, Australia Paul Steinacker, Nard Dumoulin Stuyck, Wee Han Lim, Tuomo Tanttu, MengKe Feng, Santiago Serrano, Andreas Nickl, Marco Candido, Jesus D. Cifuentes, Ensar Vahapoglu, Samuel K. Bartee, Fay E. Hudson, Kok Wai Chan, Arne Laucht, Chih Hwan Yang & Andrew S. Dzurak Diraq, Sydney, New South Wales, Australia Nard Dumoulin Stuyck, Wee Han Lim, Tuomo Tanttu, MengKe Feng, Santiago Serrano, Jesus D. Cifuentes, Ensar Vahapoglu, Samuel K. Bartee, Fay E. Hudson, Kok Wai Chan, Arne Laucht, Chih Hwan Yang, Andre Saraiva, Christopher C. Escott & Andrew S. Dzurak Stefan Kubicek, Julien Jussot, Yann Canvel, Sofie Beyne, Yosuke Shimura, Roger Loo, Clement Godfrin, Bart Raes, Sylvain Baudot, Danny Wan & Kristiaan De Greve Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar 's supervision and with input from T.T., A.S., A.L., C.H.Y., M.F., A.N., J.D.C., C.C.E. assisted with the experimental set-up. assisted with measurements of devices C and D. N.D.S. wrote the GST experiment implementation with input from M.F. helped with the GST analysis. The imec team, consisting of S.K., J.J., Y.C., S. Baudot, Y.S., R.L., C.G., B.R., S. Beyne and D.W. and led by K.D.G., developed the 300-mm spin-qubit process, fabricated the device and performed an initial electrical device screening at wafer-scale. performed the initial cryogenic device screening characterization under W.H.L. performed the cryo-prober measurements under N.D.S. and A.S. wrote the paper with input from all authors. Correspondence to Paul Steinacker, Nard Dumoulin Stuyck or Andrew S. Dzurak. is chief executive officer and a director of Diraq Pty Ltd. N.D.S, W.H.L., T.T., M.F., S.S., J.D.C., E.V., S.K.B., F.E.H., K.W.C., A.L., C.H.Y., A.S., C.C.E. declare equity interest in Diraq. The other authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Voltage noise spectrum is first calculated using a Fast Fourier transform on extracted SET current measurements divided by the slope of the Coulomb peak, and finally converted into a charge noise spectrum in μeV using the lever arm58. Charge noise at 1 Hz ranges between \(0.79(4)-0.98(7)\,{\rm{\mu }}{\rm{e}}{\rm{V}}/\sqrt{{\rm{H}}{\rm{z}}}\). Solid line is a power law fit of the power spectral density with exponent ranging between a = 0.50(7) − 1.19(5). a,b, Spin relaxation of qubit 1 (a) and 2 (b) spin up to the ground state \(| \downarrow \downarrow \rangle \). Curves are fitted to an exponential decay resulting in a relaxation time of T1,Q1 = 2.4(2) s (a = 0.999, b = 0.847) and T1,Q2 = 6.3(6) s (a = 0.989, b = 1.183), respectively. For long wait times, the pure \(| \downarrow \downarrow \rangle \) state decays to its thermal equilibrium following the Fermi-Dirac distribution at finite temperatures. c,d, Ramsey experiment for qubit 1 (c) and 2 (d) fitted to a sinusoidal exponential decay resulting in a coherence time of \({T}_{2,{\rm{Q}}1}^{* }=30.2(6)\,{\rm{\mu }}{\rm{s}}\) (a = −0.498, fdet = 95 kHz, b = 1.874, c = 0.498) and \({T}_{2,{\rm{Q}}2}^{* }=29.1(6)\,{\rm{\mu }}{\rm{s}}\) (a = −0.488, fdet = 102 kHz, b = 1.884, c = 0.499), respectively. The oscillation is induced by a phase shift corresponding to a 100 kHz detuning. Every data point is averaged for 1,000 repeats, each integrating the readout signal for tint = 100 μs. We use real-time Larmor frequency feedback between repeats using the protocol described in ref. 39. e,f, Hahn echo experiment for qubit 1 (e) and 2 (f) fitted to an exponential decay resulting in a coherence time of \({T}_{2,{\rm{Q}}1}^{{\rm{Hahn}}}=445(6)\,{\rm{\mu }}{\rm{s}}\) (a = 0.931, b = 1.263, c = 0.035) and \({T}_{2,{\rm{Q}}2}^{{\rm{Hahn}}}=803(6)\,{\rm{\mu }}{\rm{s}}\) (a = 0.908, b = 3.125, c = 0.040), respectively. We measure all six single qubit projections and fit the state purity γstate. Error bars represent the 95% confidence level. a–d, Single qubit Larmor frequency deviation of qubit 1 and 2 across devices A – D (a–d) as a function of lab time during a GST experiment nominally equivalent to the ones discussed in the main text (Fig. Figures are plotted with the same y-axis range. The standard deviation of the frequency feedback of qubit 2 (73–99 kHz) is larger compared to qubit 1 (27–52 kHz) across all devices. The microwave local oscillator frequency is set to fLO = 18.610−19.590 GHz (compare Table 1). Frequency feedback is implemented by following the protocol described in ref. GST-estimated SPAM operations, with the initial state represented as a density matrix and the terminating parity measurement as a POVM, yield a \(| \downarrow \downarrow \rangle \) state initialization fidelity of Finit = 99.996(49)% and an even/odd parity measurement of Fmeas = 99.959(59)%, respectively. The combined state preparation and measurement fidelity is calculated to FSPAM = 99.95(8)%. Error bars represent the 95% confidence level. Each plot shows I–V data for 16 quantum-dot devices (coloured curves) measured with a cryo-prober at around 1 K. One device with same design parameters as devices A–D, and 15 with different design parameters, all from the same 300 mm wafer. Different panels pertain to the gate electrodes (EB1, P1, J, P2, EB2, RG, CBs) or to all gates (top left). Gate infidelities \(1-{\mathcal{F}}\) benchmarked using GST as a function of Larmor frequency feedback rate (the frequency at which the feedback is executed and the Larmor frequency updated) at B0 = 0.666 T. We estimate the precision to be ~5 kHz, based on the deviation from the induced 100 kHz oscillations in Ramsey experiments shown in Extended Data Fig. The virtual ZI and IZ gates are independent of the Larmor frequency and omitted. Each GST experiment consists of 12,263 sequences repeated 100 times. The Larmor frequency feedback circuit is executed with increasing rate, i.e. after every 3000th, 1000th, 300th, 100th, 30th, 10th, 3rd, and single GST gate sequence. All gate infidelities decrease with increasing Larmor frequency feedback rate. The dominant infidelity contribution to the XI, II, and CZ gate is the stochastic IZ error generator, originating from idling of qubit 2. The IX gate infidelity is dominated by the respective ZI error generator, originating from idling of qubit 1. Hamiltonian ZI and IZ error generators, not plotted here, contribute about 0.4% to the infidelity of the CZ gate. Better calibration, as done in the main text (Fig. Error bars represent the 95% confidence level. Infidelity contribution of Hamiltonian and stochastic error generators of the full gate set XI (a), IX (b), ZI (c), IZ (d), II (e), and CZ (f) comparing the effect of removing gate sequences with Larmor frequency jumps > 50 kHz in post-selection at B0 = 0.666 T using GST. Hamiltonian errors contribute to the infidelity in second order, while stochastic errors contribute in first order. The ZI and IZ gates are implemented virtually and are independent of that post selection within error bars. The original (without post-selection) GST experiment consisted of 12,263 sequences, with Larmor frequency feedback after every 10th sequence, repeated 400 times. 171 repeats passed the post-selection for analysis. Error bars represent the 95% confidence level. a,b, Larmor frequency feedback σ(ΔfL) during interleaved Ramsey and Hahn echo measurements for different magnetic field amplitudes B0 of qubit 1 and 2, respectively. Each Ramsey experiment is repeated at least 10,000 times, with Larmor frequency feedback before each repeat, corresponding to 10,000 data points and roughly 25 min. For magnetic field amplitudes of B0 = 0.05 T, 0.1 T, 0.8 T, and 1.1 T, 50,000, 100,000, 50,000, and 40,000 Ramsey repeats are measured and then truncated in post batched in chunks of 10,000 repeats displayed here shaded. Each Hahn experiment is repeated 2,000 times for all six state projections [+X,−X,+Y,−Y,+Z,−Z], with Larmor feedback between each repeat for each projection, corresponding to 12,000 data points and roughly 30 min. The Hahn echo experiment for B0 = 1.3 T is only repeated 500 times. a,b, Standard deviation of the Larmor frequency variation σ(ΔfL) obtained from real-time Larmor frequency feedback during interleaved Ramsey and Hahn echo measurements as a function of magnetic field amplitudes B0 of qubit 1 and 2, respectively. Dashed lines follow a linear fit ΔΩStark ⋅ B0, resulting in ΔΩStark(P1, J, P2) = (−19.9(4), 31.3(3), 14.2(4)) MHz/(V·T) and = (0, 3.2(2), 16.7(4)) MHz/(V·T) for qubit 1 and 2, respectively. Error bars for the standard deviation of the Larmor frequency variation are only extracted for magnetic field amplitudes of B0 = 0.05 T, 0.1 T, 0.8 T, and 1.1 T with 50,000, 100,000, 50,000, and 40,000 Ramsey repeats batched in chunks of 10,000, respectively. Error bars represent the 95% confidence level. a,b, Standard deviation of the Larmor frequency variation σ(ΔfL) obtained from real-time Larmor frequency feedback during interleaved Ramsey measurements as a function of total run time for a range of magnetic field amplitudes B0 of qubit 1 and 2, respectively. a,b, Cumulative \({T}_{2}^{* }\) of qubit 1 and 2 as a function of increasing number of experiment repeats and total measurement time for a range of magnetic field amplitudes B0, respectively. Each experiment is defined by 101 shots of the Ramsey circuit with varying wait time. c,d, \({T}_{2}^{* }\) of qubit 1 and 2 as a function of lab time, respectively. Each data point is obtained by 100 repeated Ramsey experiments taking a total of approximately 15 s. We use real-time Larmor frequency feedback between Ramsey experiment repeats. Shaded error regions represent the 95% confidence level. Panel a adapted with permission from ref. 59, IEEE (The Japan Society of Applied Physics, 2023). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Industry-compatible silicon spin-qubit unit cells exceeding 99% fidelity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09549-z'>Design of facilitated dissociation enables timing of cytokine signalling</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:13:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Protein design has focused on the design of ground states, ensuring that they are sufficiently low energy to be highly populated1. Designing the kinetics and dynamics of a system requires, in addition, the design of excited states that are traversed in transitions from one low-lying state to another2,3. This is a challenging task because such states must be sufficiently strained to be poorly populated, but not so strained that they are not populated at all, and because protein design methods have focused on generating near-ideal structures4,5,6,7. Here we describe a general approach for designing systems that use an induced-fit power stroke8 to generate a structurally frustrated9 and strained excited state, allosterically driving protein complex dissociation. X-ray crystallography, double electron–electron resonance spectroscopy and kinetic binding measurements show that incorporating excited states enables the design of effector-induced increases in dissociation rates as high as 5,700-fold. We highlight the power of this approach by designing rapid biosensors, kinetically controlled circuits and cytokine mimics that can be dissociated from their receptors within seconds, enabling dissection of the temporal dynamics of interleukin-2 signalling. Protein–protein interactions orchestrate much of biological function. These two properties cannot usually be achieved simultaneously in binary interactions because they depend on the interaction off-rate in opposite ways: high affinity usually requires slow dissociation (low off-rate), whereas rapid exchange requires fast dissociation (high off-rate) (Supplementary Fig. Several natural systems exhibit ‘facilitated dissociation'10,11,12,13,14,15,16,17,18,19,20, in which an effector (E) can bind to a target–host (TH) complex to form an excited ternary complex (THE)20,21,22,23,24,25,26 from which the target dissociates quickly (Fig. In such a system, the target can bind tightly to the host, yet can also be rapidly released by adding the effector27. In engineered DNA systems, the kinetic control afforded by an analogous phenomenon (toehold-mediated strand displacement) has enabled the construction of many complex functions28,29, but DNA systems have limited utility for directly interfacing with biology. Protein binding and unbinding can be readily coupled to biological processes, but there has been no general approach to design kinetic control over protein interactions. a–c, A high-affinity interaction can rapidly exchange through facilitated dissociation (bottom pathways), but not through mutually exclusive competition (top pathways). c, Schematic of induced-fit facilitated dissociation (bottom) compared with slow mutually exclusive competition (top). The host protein (H, subscripted by conformational state X or Y) is shown in blue, the target (T) in pink and the effector (E) in orange. d, Structural models of starting components (effector-responsive switch and arbitrary binder–target pair) combined to construct facilitated dissociation systems. e, Structural models of example proteins designed to undergo a facilitated dissociation process starting from a tightly interacting state X (left) through a structurally frustrated ternary intermediate in state Y (right, solid). State X (transparent) is included to show the conformational change. We set out to design protein systems that undergo facilitated dissociation. Given an interacting protein binder–target pair, we reasoned that we could construct host proteins with controllable dissociation kinetics by fusing an effector-responsive conformational switch to the binder such that when the effector is not bound, the target can bind normally, but in the alternate effector-bound conformation, the switch clashes with the target, leading to strain in the target–host–effector ternary complex that resolves when the target dissociates30 (Fig. 1d,e; nomenclature detailed in Supplementary Note 1). This would allosterically couple the effector and target, and facilitated dissociation could proceed through the strained ternary complex intermediate faster than spontaneous dissociation of the target in mutually exclusive competition (Fig. Crucially, the energy of this ternary intermediate must be neither too high (otherwise the facilitated dissociation pathway would not be faster) nor too low (otherwise the target would not dissociate) (Fig. To set the ternary intermediate energy within this optimal range (Supplementary Fig. To access this strained state effectively, we reasoned that binding the effector should rapidly drive the conformational change against the resisting force associated with generating strain. Such a ‘driven' motion would be akin to the power strokes of motor proteins: a large conformational change that is both thermodynamically and kinetically favoured (a low-barrier descent down a steep energy gradient)8,31. Compared with the mechanism of direct steric overlap between target and effector, as in many existing facilitated dissociation systems17,18,19,20,26,32, this allosteric mechanism should be quite modular: such a force-generating switch could be fused to almost any binder to enable facilitated dissociation of its target. We began by using a previously designed effector-responsive conformational switch (hinge protein cs221; ref. This switch can undergo a rigid-body hinge motion to transition from a closed state (X) to an open state (Y), and in the open state Y can bind an effector peptide quite tightly (koff = 5 × 10−6 s−1). 34) modified as described in Supplementary Fig. 3) with dissociation slow enough (koff = 9 × 10−5 s−1) to allow us to easily measure substantial effector-induced acceleration of target dissociation, but not slower than the effector dissociation so that the target would be more likely than the effector to dissociate from the ternary complex. To allosterically couple binding of the effector to dissociation of the target, we designed structured fusions of the hinge switch and binder such that when the switch is in state X, the target can bind, but when in the effector-bound state Y, it will clash strongly (Fig. We obtained synthetic genes encoding 12 designs, expressed and purified the proteins from Escherichia coli and found that the best (allosteric switch 0; AS0) showed slow and reduced effector association in the presence of the target (Supplementary Fig. 5d), indicating the desired allosteric coupling but also the need for designs with a faster, driven pathway for effector binding against the target. In these first designs, because the effector-binding cleft is closed in state X, the hinge switch must first transition to the open state Y before the effector can bind (a conformational selection mechanism). Because state Y clashes with the target, this conformational change is slow when the target is bound, limiting the rate of effector association and the overall rate of facilitated dissociation (Supplementary Fig. This slow step could be bypassed if the effector could instead first bind to state X and accelerate the transition to state Y. An induced-fit mechanism could provide the driving force: an intrinsically disordered effector could weakly engage with state X, and fold and make more extensive interactions in the strained ternary complex in state Y35, driving the transition in a power-stroke-like motion36,37. Such a mechanism would require new switches that, in state X and throughout the conformational transition, retain an open effector-binding cleft where a flexible effector could associate (Fig. Starting from AS0, we constructed a new state X by shifting the two domains from their state Y positions relative to each other (by one heptad along the helix of domain 1 that contacts domain 2), building a new loop between the domains, and optimizing single sequences that could adopt both this new state X and the original effector-bound state Y (Supplementary Fig. This approach maintains the open cleft in state X by introducing minimal rotation from state Y and simplifies the multi-state sequence-design challenge by minimizing the local structural differences between the two states. Transition between conformational states occurs by a register shift throughout which the cleft could remain open and bind the effector (Supplementary Fig. Because these new hosts retain the state Y backbone validated to clash with the target in AS0, we should observe allosteric coupling if the switch works as designed (Fig. We obtained synthetic genes encoding ten such designs, and found that four tightly bound the effector (dissociation constant (Kd) < 1 nM; Supplementary Fig. To measure facilitated dissociation kinetics, we used surface plasmon resonance (SPR): with the target affixed to the SPR surface, we incubated with the host, then measured target–host dissociation under flow of various concentrations of effector. 7), but minimally affects the target off-rate from a control static binder fusion that lacks an effector-binding site (Supplementary Fig. a, Slow dissociation of the target from the host in the absence of effector (solid line) and fast dissociation in the presence of 2 μM effector (dashed line) assessed by SPR. Slow dissociation data (solid grey) fitted with a double exponential (pink). b, Kinetic model describing pathways of competition. Top, mutually exclusive competition; middle, facilitated dissociation with effector binding rate-limited by conformational selection; bottom, facilitated dissociation with induced-fit effector binding. The k labels are rate constants. c, Cartoon representations of the peptide (left) and 3hb (right) effectors; interface residues are shown in grey. d, Circular dichroism spectra of the peptide and 3hb effectors. e,f, Kinetic characterization of the formation and breakage of the ternary complex intermediate with the peptide (e) and 3hb (f) effectors. Top left, fast dissociation of the target from the ternary complex; data (grey) fitted with double exponentials (orange and green). Bottom left, effector association to form the ternary complex and extremely slow subsequent dissociation; data (grey) fitted with single exponentials (colours) in the association phase. Right, apparent effector on-rate constant plotted against effector concentration (circles) and a linear (e) or hyperbolic (f) fit. g,h, Kinetic characterization of the full facilitated dissociation pathway with the peptide (g) and 3hb (h) effectors. Left, effector-concentration-dependent dissociation of the target; data (grey) fitted (colours) as described in the Methods. Right, rate constant of facilitated target dissociation plotted against effector concentration (circles) and fitted with a hyperbolic equation (black line). In a and in the left plots of e–h, schematics show the arrangement of proteins relative to the SPR chip (grey). To investigate the mechanism of facilitated dissociation (Fig. 2b), we focused on AS1, the design with the tightest effector binding (Kd,H:E ≈ 10 pM). We directly measured the rate constant of target dissociation from the strained ternary complex (koff,T:HE) by flowing pre-incubated host–effector complex at a high concentration to form the strained ternary complex with the target on the SPR surface, then tracking target–host dissociation under continued flow of the effector (Fig. In the full facilitated dissociation process, as the concentration of added effector increases, the rate constant of facilitated target dissociation approaches koff,T:HE (Fig. 7), strongly suggesting that the ternary complex is an intermediate in the facilitated dissociation process (Fig. To further analyse the mechanism of forming the strained ternary complex, we characterized facilitated dissociation with two different effectors: the peptide and a three-helix bundle ‘3hb' (3hb21; ref. The two effectors make nearly identical interactions with AS1, but when unbound, the peptide is disordered whereas the 3hb is structured (Fig. With AS1 affixed to the SPR surface, we measured the rate of effector association to form the strained ternary complex by first saturating AS1 with target, then flowing varying concentrations of effector mixed with constant excess target (to ensure that the target remains bound after effector association). 2e) but hyperbolically for the 3hb, saturating at the rate of a concentration-independent step (Fig. Notably, with the 3hb, the rate of facilitated target dissociation saturates at this same value (Fig. In contrast to the 3hb, peptide effector binding (Fig. 2e) and resulting target destabilization (Fig. 2g) can both occur more rapidly than the THX → THY conformational change, suggesting that the more flexible peptide effector can bind to AS1 in state X to accelerate this conformational change through an induced-fit mechanism12,35,38 (Fig. We used X-ray crystallography to structurally characterize multiple states of our designed systems. 1a) and of the host–effector complexes (Fig. 1b) closely match the design models (maximum 1.3 Å Cα root mean square deviation (RMSD)). 3c, top), with some strain suggested by small variations among the structures (Fig. a, Crystal structure of AS1 alone (grey) overlaid with the design model of AS1 in state X (blue). Inset, detailed view of side chains in the partially open effector-binding cleft. b, Cocrystal structure of AS1 and peptide effector (grey) overlaid with the design model of the AS1–effector complex in state Y (blue and orange). Inset shows the same view as in a. c, Top, cocrystal structure of AS1 (with intact cleft) and target (grey) overlaid with the design model of the target–AS1 complex in state X (blue and pink). Bottom, cocrystal structure of AS1 (with collapsed cleft) and target (grey) overlaid with the design model of the target–AS0 complex (blue and pink) whose state X resembles this collapsed state. d, Cocrystal structure of AS1 (with hydrophobic surface mutations), target and peptide effector (grey) aligned at the switch region with the design models of the target (pink) and AS1–effector complex in state Y (blue and orange) showing the designed clash. e, Top, detailed view of the target interface side chains in the ternary complex (grey) and the target–AS1 complex (pink) interacting with AS1 (blue). Bottom, detailed view of the backbone hydrogen bonding in the interfacial strand pairing. The target–AS1 complex (pink and blue) hydrogen bonds (green) are less strained than the ternary complex (grey) hydrogen bonds (red). We next sought to determine how the target–host–effector ternary complex deforms to resolve the designed clash. At the concentrations of the components used in the above experiments, the AS1 ternary complex is an only transiently populated excited state, but at high concentrations it becomes the dominant state (Kd,T:HE = 200 nM; Supplementary Fig. This enabled us to solve structures of AS1 in the ternary complex intermediate with both target and effector. 3e, top) and shearing the interfacial strand pairing (Fig. Double electron–electron resonance (DEER) spectroscopy (Extended Data Fig. 2) and molecular dynamics (MD) simulations (Extended Data Fig. 3) suggest that the ternary complex is dynamic, with varying amounts of strain at each location (Supplementary Note 3). We next investigated the factors that contribute to the dissociation kinetics, seeking to maximize the dissociation rate enhancement by (1) reducing the base rate of target dissociation and (2) increasing the rate of facilitated dissociation. For (1), the target dissociates 20-fold faster from AS1 than from an unhindered binder fusion (Supplementary Fig. 7), probably because of minor strain in the target–AS1 complex (Extended Data Fig. For (2), with AS1, effector association can occur at least five times faster than target dissociation from the ternary complex (Fig. Assuming a simple spring model of the ternary complex, this could be achieved by deforming with a greater magnitude or in a direction of higher stiffness (Supplementary Fig. Avoiding any clash in state X while maintaining a strong clash in state Y, we sampled a variety of target + binder positions relative to the switch from AS1; rebuilt fusions between the switch and the newly located binder; and selected variants that were predicted by AlphaFold2 (AF2) to have substantial deformations spanning a variety of directions (AF2 predictions of the strained AS1 ternary complex were within 1.0 Å Cα RMSD of the target–AS1–effector crystal structure). These changes to the fusion region, although distant from both binding sites, caused considerable variation in the kinetics of target dissociation and their modulation by the effector (Supplementary Fig. Most of these variants showed reduced target off-rates (compared with AS1) in the absence of effector that increased in the presence of effector (Supplementary Fig. 11 and Supplementary Table 1), and for the fastest variant (AS117), adding effector accelerated target dissociation by 2,400-fold, reaching a rate exceeding that of the original AS1 (Fig. 10) and that strain can distribute non-uniformly throughout the structure, leading to kinetic asymmetry24 (Supplementary Fig. a,b, Comparison of three representative designs with different facilitated dissociation kinetics. a, For each design: left, model of host in state X (blue) aligned to the target (pink) to show any clash influencing the target off-rate in the absence of effector; right, model of host–effector complex in state Y (blue and orange) aligned to the target (pink) to show the designed clash, and (grey) AF2 prediction of the target position relative to the switch in the ternary complex to show how the clash resolves through global strain. b, Forward and reverse facilitated dissociation rates: target off-rate constants versus effector concentration (orange circles) and effector off-rate constants versus target concentration (pink circles) fitted with hyperbolics (black lines). Cartoons illustrate the forward and reverse facilitated dissociation pathways. Dashed lines mark the base and accelerated off-rate constants for forward (orange) and reverse (pink) facilitated dissociation. FRET time courses showing slow transfer of a kinetically trapped effector (blue, top schematic) and accelerated transfer through facilitated dissociation (orange, bottom schematic). Luminescence time courses showing breakage of a reversible split luciferase through slow direct competition (blue, top schematic) and faster facilitated dissociation (orange, bottom schematic). Luminescence time course (orange) showing rapid sensing of SARS-CoV-2 through facilitated dissociation (schematic). c–e, Data fitted with single exponentials (black lines). We next set out to use facilitated dissociation to construct protein systems with kinetic behaviours that were previously inaccessible to design. First, inspired by toehold-mediated strand displacement in DNA28, we sought to create a kinetically trapped system which, on stimulation, quickly reconfigures through a chain reaction. To investigate this, we fused the effector peptide (E2) for a second reporter hinge (H2) to the target such that E2 is occluded when this fusion (E2–target) is bound to a host (Extended Data Fig. As intended, the release of E2–target from AS114 and the subsequent switching of H2 is slow, but accelerates markedly upon addition of the original effector, which accelerates E2–target dissociation (we include excess original target to prevent E2–target rebinding39; Fig. In principle, multiple orthogonal hosts could be constructed and chained together through such target–effector fusions. Second, we reasoned that our designs could complement split protein systems with high affinity and enable them to be switched off rapidly. To test this, we tagged AS1 and the target with NanoBiT split luciferase fragments, LgBiT and SmBiT (ref. When combined, these components exhibit high luciferase activity that disappears much more rapidly upon addition of effector and excess untagged target than upon addition of excess untagged target alone (Fig. Third, in previously designed thermodynamically controlled biosensors that operate through a conformational selection mechanism, there is a trade-off between dynamic range and response time (reducing background by lowering the energy of the ‘off' state generally slows interconversion with the ‘on' state)41,42. We reasoned that sensors based on facilitated dissociation could be limited by the rate of target association rather than by internal conformational switching, and found that by caging the SmBiT peptide within the effector bound to a host, facilitated dissociation (in the reverse direction compared with previously) upon binding the target uncaged the SmBiT rapidly to enable luciferase reconstitution (Methods). Using this strategy together with a designed SARS-CoV-2 receptor-binding domain (RBD) binder (LCB1; ref. 43), we generated 16 ‘AScov' sensor designs in which the RBD clashes with the switch in state Y but not in state X. After addition of SARS-CoV-2 RBD, the best sensor shows a 30-fold increase in luciferase activity, with a half-time of 30 s (Fig. 6)—70 times faster than a previously designed LOCKR-based SARS-CoV-2 sensor that relies on conformational selection41. Thus, using this platform, a binder to almost any target can be turned into a single-component sensor that is sufficiently fast that, in most practical applications, its response time will be limited by target association rather than by a slow conformational change. Finally, we investigated whether facilitated dissociation could be used to control cellular processes with high temporal resolution. In cellular signalling, the residence times of ligands on their cognate receptors is thought to modulate the signalling and cellular responses44,45,46. The central immune cytokine interleukin-2 (IL-2) activates the IL-2 receptor (IL-2Rβγc) by inducing heterodimerization of chains β and γc (ref. The resulting complexes dissociate or degrade on timescales of hours48,49, so controlling the temporal dynamics of IL-2 signalling is difficult: there is no off-switch (Fig. a, Natural pathways for terminating IL-2 signalling are slow. b, Through facilitated dissociation, signalling could be rapidly terminated. c, Model of ASNeo2 binding IL-2Rβγc to activate signalling (left), which quickly terminates after adding effector (right). d, Left, accelerated dissociation of γc; data (grey) fitted (colours) as described in the Methods. Right, γc dissociation rate constant versus effector concentration (circles) fitted with a hyperbolic (black line). e, Relative IL-2Rβ/γc dimerization on the cell surface at first (grey; n = 37), after adding ASNeo2 (blue; n = 32) and after subsequently adding effector (orange; n = 33). f, Time courses of IL-2Rβ/γc dimerization after pre-stimulation with ASNeo2 then adding nothing (blue) or effector (orange), fitted with an exponential (black) yielding the rate constant kapp. g, Dose–response of STAT5 phosphorylation from stimulation with ASNeo2 alone (blue) or with effector (orange) (n = 1). h, Time courses of STAT5 phosphorylation after stimulation with ASNeo2 for 5 min then adding nothing (blue), effector (orange) or ruxolitinib (green) (n = 3). i–n, Human T cells were stimulated with ASNeo2 or left untreated as a control (grey). Signalling was sustained (blue) or terminated with effector (orange) after the indicated duration. i,j, Cell division (i, by carboxyfluorescein succinimidyl ester (CFSE) staining) and survival (j) three days after stimulation (n = 4); statistics from ANOVA with two-sided Tukey's post-test. k, Time courses of BCL2 expression (by quantitative PCR (qPCR); n = 3). l–n, RNA-seq analysis six hours after stimulation (n = 3). l, Principal component (PC) analysis. m, Changes in gene expression after transient stimulation. Points denote differentially expressed genes; the most significant are labelled. n, Heat map of differentially expressed genes from hallmark gene sets with high gene correlation. In e,h–k, lines and bars represent medians (e) or means (h–k); error bars and shaded areas represent 95% confidence intervals. n refers to biologically independent samples. We set out to construct a switchable IL-2 mimic that enables control over the lifetime of the IL-2Rβγc complex with a time resolution of seconds (Fig. Sampling a variety of γc positions relative to the switch from AS1, we rigidly fused Neo2 to the switch such that in state X, γc can bind, but in the effector-bound state Y, it would strongly clash (Fig. We identified several designs for which binding the effector markedly accelerates the dissociation of γc from the active signalling complex (Supplementary Fig. For one of these, ASNeo2, binding the effector induces a 1,500-fold increase in the γc off-rate (Fig. We designed variants of ASNeo2 with different dissociation kinetics and topological safeguards against degradation (Supplementary Note 5); for one, the effector accelerated γc dissociation by 5,700-fold, the highest fold change of all our designed systems (Extended Data Fig. To investigate the switching capability of ASNeo2 in a physiological context, we quantified the dimerization of labelled IL-2Rβ and γc in the plasma membrane of live cells by single-molecule fluorescence microscopy. Dual-colour co-tracking and mobility analyses confirmed that ASNeo2 efficiently dimerizes IL-2Rβ and γc and that adding the effector reverses this association rapidly and completely, even at a high excess of γc (Fig. By double labelling ASNeo2 on opposite sides, we could observe the effector-induced intramolecular conformational change at the plasma membrane using single-molecule Förster resonance energy transfer (smFRET) (Extended Data Fig. ASNeo2 activates signalling in human natural killer (NK) cells (YT-1 cell line), and its activity is greatly reduced in the presence of the effector (Fig. After the effector is added, STAT5 phosphorylation immediately stops accumulating and gradually decreases to a low level (Fig. To examine how the duration of IL-2 signalling affects the downstream cellular response, we stimulated primary human T cells with ASNeo2 and induced dissociation at various time points. Whereas sustained stimulation was required for proliferation51 (Fig. 5i), protection from apoptosis was evident after a short transient stimulation: cells treated with ASNeo2 and then with effector 5 min afterwards survived three days later at double the rate of an unstimulated control (Fig. 5j), despite the suppression of most downstream IL-2 target proteins (Extended Data Fig. This likely reflects reduced caspase-3 activity and increased BCL2 expression after transient stimulation (Fig. To further investigate the dependence of IL-2 signalling on the lifetime of the signalling complexes, we compared RNA sequencing (RNA-seq) data from T cells that were treated with ASNeo2 transiently, continuously or not at all (Fig. On activation, T cells typically shift their energy production from oxidative phosphorylation to glycolysis52, and we observed that transient stimulation downregulates oxidative-phosphorylation genes (Fig. 15; glycolysis genes are not yet upregulated). Sustained stimulation activated genes associated with mTORC-driven metabolic changes and MYC- and E2F-driven cell-cycle progression (for example, CDK4 and POLD2), but transient stimulation did not (Fig. 15), suggesting that IL-2 signalling must be sustained to pass the G1/S checkpoint. Transient stimulation did activate genes associated with formation of the mitotic spindle (for example, DOCK2 and KIF1B) (Fig. 15), suggesting that preparations for mitosis are made immediately after T cell activation, before cell-cycle checkpoints. These results show how designed facilitated dissociation can be used to tackle unanswered questions in cell biology. We show here that by explicitly considering excited intermediate states when designing coupled protein systems, a broad range of facilitated dissociation systems can be designed. We use a switchable target binder (host) and a flexible effector that can rapidly bind to and switch the target–host complex to induce a large steric clash with the target, forming a strained excited ternary complex. Crystal structures throughout the facilitated dissociation process confirm our ability to design excited states and large register-shift conformational changes. Our designed dynamic systems show both a high dynamic range and a rapid stimulus response, demonstrating the kinetic advantage of facilitated dissociation over mutually exclusive competition. Power-stroke mechanisms8,31 can generate force more efficiently than can ratchet mechanisms (in which a single large step is rectified at the end of the conformational change)37. In contrast to this induced-fit mechanism of the flexible effector, we find that a rigid effector provides reduced rate acceleration even though it binds more tightly38. In kinesin and other biological systems, it is difficult to directly assess the role of flexibility and folding upon binding in overall function; by contrast, our designed model systems allow the direct comparison of flexible and rigid effectors and show that the former yield faster conformational transitions against loads. Most known examples of facilitated dissociation couple the target and the effector through a combination of direct steric overlap17,18,19,20,26,32 and intricate allostery14,15,22,23,24,25. By contrast, because our allosteric mechanism of switching steric clashes places no specific requirements on the binder or target, our approach can be used to dynamically regulate protein–protein interactions quite generally: by fusing to our switch, almost any binding interaction can be made to switch off rapidly in the presence of an effector. Our approach transferred immediately to rapidly switching active IL-2-like signalling complexes: we obtained several working designs on the first attempt (in which 24 designs were tested). Controlling the duration of IL-2 signalling with such switchable cytokines could enable investigation of early events in signalling or tuning of the cell response through timing-sensitive regulatory mechanisms further downstream51 (for example, disrupting signalling complexes at the cell surface or later in the endosome49 could be used to distinguish the cellular responses induced by signalling from each compartment55). Therapeutically, systemic administration of the effector and local administration of the switchable cytokine could elicit strong immune activation at only the site of administration (because any cytokine that escapes into circulation would be deactivated by the effector). More generally, our work provides a route to designing the rates and pathways of protein motion and change, which should ultimately enable the construction of complex, lifelike protein machinery. In PyMOL, we manually positioned the switch relative to the binder–target complex subject to several constraints: there is no steric overlap between the target and switch state X; there is large steric overlap between the target and switch state Y; the smallest deformation that could be undergone by the switch and the target to resolve this clash is in the desired direction; and the switch C terminus and binder N terminus are relatively oriented such that sensible additional structure could be built between the switch and the binder to rigidly fuse them as positioned. To aid in visualizing this additional structure, we also included placeholder helices while manually positioning the switch and binder, effectively ‘sketching' the fusion (Supplementary Fig. We then refined these sketches into plausible backbones. For later fusions, including the AS1 variants and ASNeo2 designs, we first used Rosetta FastDesign57 to sample around the starting sketch for designable positions of the switch, binder and placeholder helices while keeping the region of the state X switch that clashes in state Y fixed relative to the target, then used RFDiffusion7 (conditioned on the secondary structure and block adjacency of the sketches) to build structure between the sampled switch and binder positions. During both structure-generation approaches, we masked noncritical residues on the switch and binder interfacing with the fusion structure. To filter designs, we used AF259 (with initial guess (AF2-IG)60 for complex predictions) to predict the structure of the fusions alone, with the target, and with the effector, selecting designs for which each structural state is correctly and confidently predicted by a majority of the five model weight sets. Finally, for the AS1 variants and ASNeo2 designs, we used AF2-IG to predict the structure of the strained ternary complexes, selecting sets with a diversity of deformation directions. Designs AS1, AS2, AS5 and AS7 were generated starting from design AS0, which contains the hinge switch cs221. When cs221 was designed, its state Y was generated by copying the N-terminal domain (helices 1–4) of its parent scaffold (DHR20; ref. 61), aligning helix 4 of the copy to the corresponding helix of DHR20 offset by three residues, and combining the transformed N-terminal domain and original C-terminal domain into a single protein33. Thus, relative to the C-terminal domain, the N-terminal domain is both rotated around and translated along the axis of helix 4, exposing a cleft between the domains for binding a helical peptide. Now, to generate a new state X for this switch that retains an open cleft, we repeated this procedure but aligned the N-terminal domain with a residue offset of −4 instead of 3, so that overall this position of the N-terminal domain is shifted from state Y along helix 4 by one heptad (Supplementary Fig. This introduces minimal rotation and thus maintains the open cleft. We combined this new position of the N-terminal domain with the entire C-terminal domain of AS0 (including the fusion to LHD101B), then used Rosetta FastDesign to further sample designable positions of the N-terminal domain around this starting point. In half of the design trajectories, we included a placeholder helix in the cleft to help ensure it remains open. We then used inpainting with RosettaFold to generate loops connecting the domains, then paired these complete state X backbones with the original effector-bound state Y of AS0 (also generating new loops between domains in state Y as necessary to match the loop lengths of each new state X). To generate single sequences that support both states, we first performed multi-state design with Rosetta FastDesign (enforcing sequence symmetry between states) to further refine the paired backbones so that they are more mutually compatible, then used ProteinMPNN with residue probabilities tied at corresponding positions between states to optimize sequences simultaneously for both conformations33. To filter designs, we used AF2-IG to predict the structure of the switches both with and without the effector, selecting designs for which the new state X (in the absence of effector) and state Y (in the presence of effector) are correctly and confidently predicted by a majority of the five model weight sets. To further ensure that these designs favour state X in the absence of effector and state Y in the presence of effector, we only selected designs that scored more favourably in Rosetta in state X than in state Y, but also scored more favourably in state Y with the effector bound than the sum of the scores of state X and the unbound effector. We first sought to cage the SmBiT peptide within the effector so that, through reverse facilitated dissociation, binding of the target would rapidly uncage the SmBiT to enable luciferase reconstitution. We grafted SmBiT onto the effector peptide at a range of positions, then screened these ‘SmBiTgraft' variants for binding to AS0 and rapid dissociation after binding of the target. We used AS0 as the base design, reasoning that its closed state X would slow undesired peptide reassociation. For each working SmBiTgraft, we then generated a flexibly linked LgBiT–SmBiTgraft–AS0 construct. The best construct shows low luciferase activity that increases rapidly after addition of target, but only slowly after addition of effector peptide (which must compete directly with SmBiTgraft) (Extended Data Fig. To demonstrate the modularity of this platform, using the methods described above for designing hosts, we rigidly fused a SARS-CoV-2 RBD binder (LCB1; ref. 43) to the switch in place of the target binder such that the RBD clashes with the switch in state Y but not in state X, and tested 16 of these AScov fusion designs. Synthetic DNA fragments encoding each design were obtained from IDT as eBlocks and cloned into custom vectors using Golden Gate assembly62. Designs usually contained a C-terminal sequence-specific nickel-assisted cleavage (SNAC) tag63 and a 6×His tag (MSG-Protein-GSGSHHWGSTHHHHHH). Proteins to be captured on the chip for SPR experiments contained an N-terminal AviTag and a C-terminal 6×His tag (MSGLNDIFEAQKIEWHESSG-Protein-GSGHHHHHH). For the size-exclusion chromatography (SEC) binding experiments shown in Supplementary Fig. 6 and for screening SmBiTgraft effector peptide variants with SPR, the effector was fused to superfolder GFP in a sfGFP-GSSG-Effector-GSHHHHHH construct. To rapidly break split luciferases, AS1 was fused to LgBiT in a MSG-AS1-linker-LgBiT-GSHHHHHH construct and the target was fused to SmBiT in a MSG-Target-linker-SmBiT-GSHHHHHH construct. In the rapid sensors, AS0 and AScov were fused to LgBiT and a SmBiT-containing effector in a MSGHHHHHHGS-LgBiT-linker-SmBiTgraft-linker-(AS0 or AScov)-GS construct. Sequences for all designs are available in Supplementary Table 4. All proteins were expressed from NEB BL21(DE3) E. coli cells using TBII (MpBio) autoinduction medium with 0.5% (w/v) glycerol, 0.05% (w/v) glucose, 0.2% (w/v) lactose, 20 mM MgSO4, trace metal mix and 50 μg ml−1 kanamycin. Expression cultures (50 ml) were grown either at 37 °C for 16–20 h or at 37 °C for 6–8 h, then at 18 °C for 16–20 h. Cells were collected by centrifugation, resuspended in 5 ml lysis buffer (100 mM Tris HCl pH 8.0, 200 mM NaCl, 50 mM imidazole, 1 mM PMSF, and 1 Pierce Protease Inhibitor Mini Tablets, EDTA-free per 50 ml) and lysed by sonication. The lysate was clarified by centrifugation at 14,000g for 30 min. Protein in the soluble lysate was bound to 1 ml Ni-NTA resin (QIAGEN), washed with 5 ml low-salt wash buffer (20 mM Tris HCl pH 8.0, 200 mM NaCl and 50 mM imidazole), 5 ml high-salt wash buffer (20 mM Tris HCl pH 8.0, 1 M NaCl, 50 mM imidazole) and 5 ml low-salt wash buffer, and eluted in 1.2 ml elution buffer (20 mM Tris HCl pH 8.0, 200 mM NaCl and 500 mM imidazole) after a 0.4 ml pre-elution. The proteins were further purified by SEC on a fast protein liquid chromatography (FPLC) system with a Superdex 200 Increase 10/300 GL column in Tris-buffered saline (TBS; 20 mM Tris pH 8.0 and 100 mM NaCl) with 1-ml fractions. Where possible, fractions that probably corresponded to protein monomers were selected. Final protein concentrations were estimated using molar extinction coefficients predicted from the protein sequence and integrating the absorbance at 280 nm over the selected fractions. Correct protein molecular weights were confirmed using liquid chromatography–mass spectrometry (LC–MS). The effector peptide cs221B was chemically synthesized by GenScript. The TAMRA-labelled effector used in fluorescence polarization (FP) experiments was synthesized as previously described33. Individual host proteins, sfGFP–effector and 1:1 host:sfGFP–effector mixtures were prepared at 20 μM in TBS (20 mM Tris pH 8.0 and 100 mM NaCl). A 0.5-ml quantity of each solution was injected onto a Superdex 200 Increase 10/300 GL column in TBS and the absorbance at 230 nm was monitored for changes in the retention volumes of the mixture compared with the individual proteins. FP binding experiments with TAMRA-labelled effector were performed at 25 °C in TBS (20 mM Tris pH 8.0 and 100 mM NaCl) with 0.05% v/v TWEEN20 in 96-well plates (Corning 3686). Parallel and perpendicular fluorescence intensity was measured using a Synergy Neo2 plate reader with an FP 530/590 filter cube. Fluorescence polarization P (in units of mP) was calculated by the following expression: For affinity measurements, host proteins were titrated by twofold serial dilution across TAMRA–effector (at a constant concentration between 0.1 and 1 nM) through 24 wells, with a final volume of 80 μl in each well. Plates were incubated for at least 12 h at room temperature to fully equilibrate before measurement. To determine affinities, the following binding isotherm function was fitted to the measured polarization values using nonlinear least-squares minimization: When the fit Kd is lower than [E], affinities are too strong to be accurately measured with this method, so ‘Kd < [E]' is reported. For kinetic competition measurements, the target LHD101An1 was titrated by twofold serial dilution across the host protein (at a constant concentration of 22 nM) through five wells (a sixth well with just the host was also included) with a final volume of 72 μl in each well. Plates were incubated for one hour to allow the host and the target to equilibrate. To each well, 8 μl of 200 nM TAMRA–effector was added and rapidly mixed using a multichannel pipette, and the measurement was started immediately afterwards. This resulted in a 20 nM final concentration of both host and effector in a final volume of 80 μl per well. The following single exponential decay function was fitted to the measured polarization time courses using nonlinear least-squares minimization: To measure facilitated dissociation in the reverse direction, host and TAMRA–effector were incubated at a 2:1 ratio to fully saturate the effector with host, diluted to 4 nM host and 2 nM effector, and distributed across 11 wells (40 μl in each well). In nine separate wells, the target was titrated by a fourfold serial dilution with constant excess unlabelled effector (40 μM). Just excess unlabelled effector at 40 μM was prepared in the 10th separate well (to measure the base rate of effector dissociation), and buffer was prepared in the 11th (to confirm the baseline remains stable). Forty microlitres of these target + effector solutions were added to the 11 host + TAMRA–effector wells and rapidly mixed using a multichannel pipette, and the measurement was started immediately afterwards. This resulted in final concentrations of 2 nM host, 1 nM TAMRA–effector, 20 μM unlabelled effector (to make the dissociation of TAMRA–effector irreversible) and varying concentrations of target in a final volume of 80 μl per well. The following single exponential decay function was fitted to the measured polarization time courses using nonlinear least-squares minimization: where koff,H:E is the base effector off-rate constant, koff,TH:E is the off-rate constant for accelerated effector dissociation from the ternary complex, K1/2 is the concentration at which half the rate increase from koff,H:E to koff,TH:E is reached and [T] is the concentration of target present. Proteins to be captured on the SPR chip were expressed with an N-terminal AviTag and purified as described above, except that the proteins were biotinylated after elution from the Ni-NTA resin: to the elutions, 5 μg ml−1 BirA (Avidity), 10 mM ATP, 10 mM Mg(OAc)2 and 100 μM d-biotin were added and allowed to incubate at room temperature for at least 4 h before further purification by SEC. Successful biotinylation was confirmed using LC–MS. SPR measurements were performed at 25 °C in HBS-EP+ buffer (Cytiva) on a Biacore 8K instrument. Biotinylated proteins were immobilized on the chip using the Biotin CAPture system (Cytiva). For measurements of target off-rate constants, biotinylated target protein was immobilized on the chip. To measure base off-rate constants koff,T:H, host proteins at 50 nM were flowed over the chip for 60 s, then dissociation was measured for 2 h. To measure accelerated off-rate constants koff,T:HE, pre-incubated host–effector complexes (host at 1 μM and effector at 5 μM to ensure that the host is saturated with effector) were flowed for 60 s to form the ternary complex on the chip, then dissociation was measured for 4–20 min under a constant flow of 5 μM effector. Most dissociation data were fitted with the following double exponential decay function to account for populations of host protein with different dissociation kinetics: where S is the modelled SPR response, S0 is the baseline, S1 and S2 are amplitudes relating to the sizes of each host population, kapp,1 and kapp,2 are apparent rate constants corresponding to each host population, t is the time and t0 is the time at which dissociation initiates. The reported rate constant typically corresponds to the faster and higher amplitude exponential in the fit; instances in which other criteria are used to determine which rate constant corresponds to the change of interest are noted. When clearly only one host population is present (often indicated by kapp,1 ≈ kapp,2 or a large difference between S1 and S2 when fitting a double exponential), the following single exponential decay function was fitted instead, in which the parameters are the same as above (instances in which single exponentials were used are noted): To measure the rate constant of effector association to form the ternary complex, biotinylated host protein was immobilized on the chip. The host protein was saturated with target by flowing 5 μM target over the chip for 4 min, then a varying concentration of effector and 5 μM target was flowed over the chip for 2–4 min to associate the effector, and finally 5 μM target was flowed over the chip for 4 min to monitor effector dissociation. Target was included at 5 μM (higher than its affinity to the ternary complex) throughout the experiment to ensure that the host remained saturated with target, preventing changes in target binding from convoluting the response from effector binding. Association data were fitted with the following single exponential decay function: where kon,TH:E and koff,TH:E are on-rate and off-rate constants and [E] is the concentration of effector flowed over the chip. where kswitch is the rate constant for the THX → THY conformational change, kunswitch is the rate constant for the THY → THX conformational change, kon,TH:E and koff,TH:E are on-rate and off-rate constants and [E] is the concentration of effector flowed over the chip. This function describes the slow relaxation rate constant of binding by conformational selection64. During the fit, koff,TH:E was constrained to a low value (<1 × 10−4 s−1), as observed. To measure the effector-concentration-dependent rate constant of the full facilitated dissociation process, either biotinylated target protein or biotinylated common gamma γc ectodomain (Acro Biosystems ILG-H85E8) was immobilized on the chip. Each experiment involved multiple cycles of host association and induced dissociation under the flow of various concentrations of effector obtained by twofold serial dilution (Supplementary Fig. Throughout these cycles, a small population of host that is unresponsive to the effector (‘Hn', owing to partial degradation or misfolding induced by the strain in the ternary complex) could accumulate on the chip (Supplementary Fig. with initial values computed for each cycle by where S is the modelled SPR response, [THE], [TH] and [THn] are concentrations of complex states on the chip, t is the time after the start of the dissociation cycle, [E] is the concentration of effector flowed over the chip, koff,T:H, koff,T:HE, kon,TH:E, koff,TH:E and koff,T:Hn are on-rate and off-rate constants, aTH, aTHE and aTHn are amplitudes relating the concentration of each state on the chip to an SPR response, fn is an amplitude fudge factor for cycle n accounting for small differences in amplitude across cycles and fresponsive is the fraction of host that is responsive to the effector. Varying all of these parameters, this model is then globally fitted to the dissociation curves of all cycles using nonlinear least-squares minimization. Note that with a sufficiently high value of kon,TH:E, this model's dissociation kinetics are determined mainly by the target dissociation parameters koff,T:H and koff,T:HE; meanwhile, the effector binding parameters kon,TH:E and koff,TH:E tend to tightly covary and cannot be accurately determined from these fits. Also note that for designs with low values for koff,T:HE, this model may be less accurate because the assumption that [THE] = 0 at the beginning of each cycle may no longer be valid. To compute the effective rate constant of the full facilitated dissociation process for intact host, we used the following simplified system of differential equations, which no longer accounts for a small population of unresponsive host: The following hyperbolic function was fitted to the effective rate constants (constrained to the values of koff,T:HE and koff,T:H obtained from the global fit): where koff,T:H is the base target off-rate constant, koff,T:HE is the off-rate constant for accelerated target dissociation from the ternary complex, K1/2 is the concentration at which half the rate increase from koff,T:H to koff,T:HE is reached and [E] is the concentration of effector flowed over the chip. 2f (right) and the target facilitated dissociation experiment in Fig. All SPR measurements in the main figures were repeated at least once, with similar results. Circular dichroism spectra were measured at 25 °C on protein samples at 0.2 mg ml−1 in TBS (20 mM Tris pH 8.0, 100 mM NaCl) using a Jasco J-1500 spectrophotometer. This was accomplished by lysine methylation (for crystals AS1_TH 1, AS1_TH 2 and AS1_THE 1) or with the hydrophobic surface mutations K46L, E50W, K172W and E173Y (for crystal AS1_THE 2). Protein was expressed from NEB BL21(DE3) E. coli cells using TBII autoinduction medium as above but at a larger scale: either 8 × 50 ml or 1–2 × 500 ml cultures. Cells were collected by centrifugation, resuspended in lysis buffer and lysed by sonication. The lysate was clarified by centrifugation at 14,000g for 30 min. Protein in the soluble lysate was bound to 8 ml Ni-NTA resin (QIAGEN), washed with 10 ml low-salt wash buffer, 30 ml high-salt wash buffer and 10 ml SNAC cleavage buffer (100 mM 2-(N-cyclohexylamino)ethanesulfonic acid (CHES), 100 mM acetone oxime, 100 mM NaCl and 500 mM guanidinium chloride, pH 8.6)63 and incubated in 40 ml SNAC cleavage buffer + 2 nM NiCl2 for 12 h at room temperature to cleave. The amount of cleaved protein in the flowthrough and wash was assessed with SDS–PAGE, and fractions with enough cleaved protein were concentrated and further purified using SEC on an FPLC system with either a Superdex 75 Increase 10/300 GL column or a HiLoad 20/600 Superdex 75 pg column in either TBS (20 mM Tris pH 8.0 and 100 mM NaCl) or lysine methylation buffer (50 mM HEPES pH 7.5 and 250 mM NaCl) if lysine residues in the protein were to be methylated. For protein complex cocrystallization, the purified proteins and/or chemically synthesized effector were mixed at equimolar ratios. For some samples (resulting in crystals AS1_TH 1, AS1_TH 2 and AS1_THE 1), lysine residues were methylated as previously described65, and the reaction was quenched using SEC on an FPLC system to buffer-exchange into TBS. Finally, the samples were concentrated to crystallization levels. Crystallization experiments were done using the sitting drop vapour diffusion method. Initial crystallization trials were set up in 200-nl drops using the 96-well plate format at 20 °C. Crystallization plates were set up using a Mosquito LCP from SPT Labtech, then imaged using UVEX microscopes and UVEX PS-256 from JAN Scientific. Diffraction data were collected at the National Synchrotron Light Source II beamline 17-ID-1 (FMX/AMX), the Advanced Light Source beamline 821/822 or the Advanced Photon Source NECAT 24ID-C. X-ray intensities and data reduction were evaluated and integrated using XDS66 and merged and scaled using Pointless or Aimless in the CCP4 program suite67. Structure determination and refinement starting phases were obtained by molecular replacement with Phaser68, using the designed model for the structures. After molecular replacement, the models were improved using phenix.autobuild with rebuild-in-place to false and using simulated annealing. Model building was performed using Coot70. The final model was evaluated using MolProbity71. Spin-label modelling and distance distribution predictions were performed as previously described33 using chiLife72 with the off-rotamer sampling method73. Site pair selections were performed as previously described33. Host protein variants containing cysteine residues at the selected sites were purified as described above, except that 0.5 mM TCEP was included in the lysis buffer and the first two Ni-NTA resin washes, and the proteins were labelled immediately after elution from the Ni-NTA resin: to the elutions, 50 μl of 200 mM of the nitroxide spin label 1-oxyl-2,2,5,5-tetramethyl-3-pyrroline-3-methyl)methanethiosulfonate (MTSL) in dimethyl sulfoxide (DMSO) was added and allowed to incubate for at least 2 h at room temperature before further purification by SEC. Successful labelling was confirmed using LC–MS. DEER samples were prepared at 20 µM labelled host protein in deuterated solvent buffered by 20 mM Tris at pH 8.0 with 100 mM NaCl and 20% d8-glycerol (Cambridge Isotope Laboratories) as a glassing agent. When appropriate, target and effector were added to a concentration of 100 µM each. Then, 15–30 µl of each sample was loaded into a quartz capillary (Sutter Instrument, 1.1 mm inner diameter, 1.5 mm outer diameter) and flash-frozen with liquid nitrogen. DEER experiments were performed as previously described33. An ELEXSYS E580 spectrometer (Bruker) at Q-band (around 34 GHz) with an EN5107D2 resonator (Bruker) was used for all experiments. The temperature was maintained at 50 K using a cryogen-free cooling system (ColdEdge). The four-pulse DEER sequence was used, using 60-ns Gaussian observer pulses with a full width at half maximum (FWHM) of 30 ns and a frequency near the centre of the field-swept spectrum and 150-ns sech/tanh probe pulses with centre 80 MHz above the observer frequency, 80 MHz bandwidth and a truncation parameter of 10. All shaped pulses were generated using the SpinJet arbitrary waveform generator (Bruker). Pulse shapes were calculated using PulseShape (https://github.com/stolllab/PulseShape), using both resonator compensation and transmitter nonlinearity compensation. All data were collected using a 2 ms shot repetition time, 8-step phase cycling and τ1 averaging from 400 ns to 528 ns in 16-ns steps. All other experimental parameters (including pump pulse time step, τ2 delays, number of scans and others) were chosen on a per-sample basis and are reported in Supplementary Table 3. All DEER data were analysed using the DeerExp module of the eprTools Python package (https://github.com/mtessmer/eprTools). All data were fitted using separable nonlinear least squares74. The foreground signal was modelled using Tikhonov regularization with the second derivative operator. The regularization parameter was selected using generalized cross-validation. The background was modelled using a 3D-homogeneous spin distribution. An additional penalty restraining the modulation depth to be low was used to prevent the fitting of long-distance artefacts in the foreground, as was done previously75. Confidence intervals were estimated using bootstrap sampling with 100 samples using a fixed regularization parameter. Input files for the MD simulation were prepared with CHARMM-GUI76,77, with AF2 structure as the initial structure. A rectangular water box of edge length 10 Å was placed around the protein. The water box contained potassium and chlorine ions of concentration 0.15 M that neutralized the protein's net charge. All three steps were performed with the GROMACS 2020.2 MD engine79,80. The steepest descent method was used for energy minimization, for 5,000 steps with an energy tolerance of 1000 kJ mol−1 nm−1. The neighbour list was updated every ten steps with a cut-off distance of 1.2 nm. Cut-off was used to calculate Van der Waals interactions with a switch distance of 1.0 nm and a cut-off distance of 1.2 nm. The fast smooth particle-mesh Ewald method was used to calculate electrostatics with a cut-off distance of 1.0 nm. All bonds with hydrogen atoms were treated as rigid using the linear constraint solver (LINCS) algorithm81. The equilibration step was performed with a leap-frog algorithm using a time step of 1 fs and a total simulation time of 125 ps. The solute and solvent were coupled with a time constant for coupling of 1 ps. The centre-of-mass translational velocity was removed every 100 steps to prevent the system from drifting. The cut-off schemes for Van der Waals and electrostatic interactions were the same as those used in the minimization step, except that the neighbour list was updated every 20 steps. The velocities were generated from a Maxwell distribution with a temperature of 303.15 K. The same LINCS method was used to constrain the hydrogen atoms during the equilibration step. The production step was performed with a leap-frog algorithm using a time step of 2 fs and a total simulation time of 1 μs. The solute and solvent were coupled with a time constant of 1 ps. Exponential relaxation pressure coupling and isotropic coupling with a time constant of 5.0 ps was used to maintain the pressure at 1.0 bar. The cut-off schemes for Van der Waals and electrostatic interactions were the same as those used in the minimization step, except that the neighbour list was updated every 20 steps and the Coulomb cut-off distance was set as 1.2 nm. Analysis of the trajectories, such as RMSD and root mean square fluctuation (RMSF) calculations, was performed with MDAnalysis82. To simulate DEER distance distributions from these trajectories, structures along each trajectory were clustered using the Gromos clustering algorithm83 in GROMACS, distance distributions were predicted (using chiLife72 with the off-rotamer sampling method73) for the centre structure of each cluster and the resulting distributions were averaged weighted by the occupancies of their corresponding clusters. Hinge protein cs201F_E249L (H2) was purified as described above, except that 0.5 mM TCEP was included in the lysis, wash and elution buffers, and the SEC buffer was phosphate-buffered saline (PBS; 20 mM sodium phosphate pH 7.0, 100 mM NaCl and 0.5 mM TCEP). To double label with dyes, 50 μM H2 was incubated with 250 μM Alexa Fluor 555 C2 maleimide (donor; Thermo Fisher Scientific) and 250 μM Alexa Fluor 647 C2 maleimide (acceptor; Thermo Fisher Scientific), shaking at room temperature for at least 2 h. The reaction was quenched by adding DTT to 10 mM, and the proteins were separated from excess dye by SEC in TBS (20 mM Tris pH 8.0, 100 mM NaCl). FRET binding experiments were performed at 25 °C in TBS (20 mM Tris pH 8.0 and 100 mM NaCl) with 0.05% v/v TWEEN20 in 96-well plates (Corning 3686). Fluorescence intensity was measured using a Synergy Neo2 plate reader, exciting the donor at a 520-nm wavelength and reading acceptor emission at a 665-nm wavelength. To measure the E2–target on-rate constant, 40 μl H2 at 10 nM was prepared in six wells. To each well, 40 μl E2–target at various concentrations was added and rapidly mixed using a multichannel pipette, and the measurement was started immediately afterwards. The following single exponential decay function was fitted to the measured FRET time courses using nonlinear least-squares minimization, A control solution of just 1,067 nM AS114 was also prepared. A 37.5-μl quantity of 42.7 nM H2 was prepared in eight wells; 37.5 μl of AS114 + E2–target was added to four wells and 37.5 μl of AS114 was added to the other four wells using a multichannel pipette, and the measurement was started immediately afterwards. Five minutes later, four solutions (5 μl buffer, 100 μM target, 16 μM effector or 100 μM target + 16 μM effector) were added to each set of four wells (each solution to a different well of each set) using a multichannel pipette, and the measurement was immediately continued for 1 h. Mixing components at these concentrations resulted in final concentrations of 500 nM AS114, 250 nM E2–target, 20 nM H2, 1 μM effector and 6 μM target. A baseline drift function of the following form was fitted to the AS114 + buffer data and subtracted from the other time courses: AS114 was used because, in state X, it does not clash with E2 extending past the target in E2–target. Luminescence experiments were performed at 25 °C in TBS (20 mM Tris pH 8.0 and 100 mM NaCl) with 0.05% v/v TWEEN20 in 96-well plates (Corning 3686). Luminescence was measured using a Synergy Neo2 plate reader with a LUM filter cube. To measure rapid breakage of a split luciferase, 111 pM AS1–LgBiT and 22 nM target SmBiT were incubated for one hour to load AS1 with the target and reconstitute the split luciferase. Next, 72 μl of this mixture was added to two wells, then 8 μl of 1/10-diluted Nano-Glo substrate (Promega N1130) and either 10 μM effector and 200 μM target or just 200 μM target was added to these wells using a multichannel pipette, and the measurement was started immediately afterwards. Excess target was included to fully outcompete target–SmBiT to enable measurement of the dissociation rate. Mixing components at these concentrations resulted in final concentrations of 100 pM AS1–LgBiT, 20 nM target–SmBiT, 1 μM effector, 20 μM target and 1/100-diluted Nano-Glo substrate. To measure rapid analyte sensing, 64 μl of 12.5-pM sensor was added to eight wells, 8 μl of 1/10-diluted Nano-Glo substrate was added to these wells using a multichannel pipette and the measurement was started immediately afterwards. Five minutes later, 8 μl of various concentrations of analyte obtained by tenfold serial dilution was added using a multichannel pipette, and the measurement was immediately continued for 30–60 min. Mixing components at these concentrations resulted in final concentrations of 10 pM sensor and 1/100-diluted Nano-Glo substrate. The analytes were target, effector or SARS-CoV-2 RBD (Acro Biosystems SPD-C52H3). Figure 4e shows a time course with 800 nM SARS-CoV-2 RBD. For cell-surface labelling, receptors were N-terminally fused to suitable tags using a pSems vector including the signal sequence of Igκ (pSems leader). Common gamma chain (γc) was fused to the ALFA-tag84 and IL-2Rβ was fused to nonfluorescent monomeric GFP (mXFP)85. HeLa cells (ACC 57, DSMZ) were cultured as previously described86. Labelling, washing and subsequent imaging were performed after mounting the coverslips into custom-made incubation chambers with a volume of 1 ml. Cells were equilibrated in medium with fetal bovine serum (FBS) but lacking phenol red, supplemented with an oxygen scavenger and a redox-active photoprotectant (0.5 mg ml−1 glucose oxidase (Sigma-Aldrich), 0.04 mg ml−1 catalase (Roche), 5% w/v glucose, 1 μM ascorbic acid and 1 μM methylviologene) to minimize photobleaching88. Selective cell-surface receptor labelling was achieved by using anti-GFP and anti-ALFA-tag nanobodies (NBs), which were site-specifically labelled by maleimide chemistry via a single cysteine residue at their C termini88. Anti-ALFA NB labelled with Cy3B (degree of labelling (DOL): 1.06) and anti-GFP NB labelled with ATTO 643 (DOL: 1.0) were added at concentrations of 3 nM each, at least 10 min before imaging. Coverslips were precoated with poly-l-lysine-graft-poly(ethylene glycol) to minimize unspecific binding of NBs and were functionalized with RGD peptide for efficient cell adhesion89. During the imaging experiments, ASNeo2 was used at 100 nM, and 10 μM effector was used to induce receptor dissociation. Dual-colour imaging was performed by total internal reflection fluorescence (TIRF) microscopy using an inverted microscope (IX83, Olympus) equipped with a spectral image splitter (QuadView, Photometrics) and an EMCCD camera (iXon Ultra, Andor) as described in detail elsewhere90. Fluorophores were excited by sequential illumination with a 561-nm laser (2RU-VFL-P-2000-560-B1R, 2,000 mW, MPB Communications) and a 642-nm laser (2RU-VFL-P-2000-642-B1R, 2,000 mW, MPB Communications). Alternating laser excitation was achieved with a simple micro-controller (Arduino Uno) and open-source acquisition software91 synchronizing laser shuttering with an acousto-optic tunable filter (AOTF; AA.AOTFnC-400.650-TN, AA Opto Electronic) and camera triggering. For long-term tracking experiments, 1,500 frames per channel were acquired at 40 fps. The resulting image stacks were divided into five-frame stacks and dimerization was determined for each stack. 5f, three time courses, each normalized to its average initial relative dimerization, are overlaid for each condition. For all other tracking experiments, 200 frames per channel were acquired at 40 fps and dimerization was determined over the whole image stack. Dual-colour single-molecule co-tracking time-lapse images were evaluated using an in-house-developed MATLAB software (SLIMfast4C, https://zenodo.org/record/5712332)88. After channel registration based on calibration with fiducial markers, molecules were localized using the multi-target tracking algorithm92. Immobile emitters were filtered out by spatio-temporal cluster analysis93. Frame-by-frame co-localization within a cut-off radius of 150 nm was applied followed by tracking of co-localized emitters using the utrack algorithm94. Molecules co-diffusing for ten frames or more were then identified as co-localized. Relative levels of co-localization were determined on the basis of the fraction of co-localized particles relative to all localizations in the ALFA-γc channel (561 nm). Diffusion properties were determined from the pooled single trajectory using mean squared displacement analysis for all trajectories with a lifetime greater than ten frames. Diffusion constants were determined from the mean squared displacement by linear regression. Relative dimerization was estimated by: FRET efficiencies were evaluated using an in-house-developed MATLAB software (provided and described in detail elsewhere as supplementary software90). In brief, alternating laser excitation FRET experiments provide three separated emission channels: directly excited donor \({D}_{{D}_{{\rm{ex}}}}^{{D}_{{\rm{em}}}}\) and acceptor \({A}_{{A}_{{\rm{ex}}}}^{{A}_{{\rm{em}}}}\) channels, as well as a sensitized FRET \({F}_{{D}_{{\rm{ex}}}}^{{A}_{{\rm{em}}}}\) channel. First, channels were aligned by calibration with fiducial markers. Then, after applying a single-molecule localization algorithm95, single-molecule intensities were determined from background subtracted images (\({I}_{{D}_{{\rm{ex}}}}^{{D}_{{\rm{em}}}}\), \({I}_{{A}_{{\rm{ex}}}}^{{A}_{{\rm{em}}}}\), \({I}_{{D}_{{\rm{ex}}}}^{{A}_{{\rm{em}}}}\)). To evaluate FRET efficiencies, donor–acceptor pairs were co-localized with an optimized search radius. For these pairs, the apparent FRET efficiency was calculated by: To achieve accurate FRET efficiencies, standard further corrections were applied. These include the donor leakage coefficient, cross-talk-corrected proximity ratio and the correction factor γ (refs. Human NK (YT-1) cells were cultured in RPMI 1640 complete medium, supplemented with 10% FBS, 2 mM l-glutamine, minimum essential non-essential amino acids, sodium pyruvate, 25 mM HEPES and penicillin–streptomycin (Gibco). For the flow-cytometry-based pSTAT5 detection assay, 2–5 × 105 IL-2Rα-positive YT-1 cells were seeded in 350 µl of medium per well in a 96-well plate. The cells were stimulated with 1 nM ASNeo2 or Neo2 for 5 min at 37 °C. As a control, 50 µl of untreated YT-1 cells were set aside at the start of each experiment and evaluated after 45 min alongside the treated cells. After stimulation, all cells were transferred to three separate wells containing a control (no treatment), 10 μM effector or 40 μM ruxolitinib. One-seventh of the cells (50 µl) were resuspended in 17 µl of 16% paraformaldehyde (PFA) for immediate fixation. This process was repeated at 5-, 10-, 15-, 30- and 45-min intervals. After all of the time points were fixed, the cells were refixed in 4% PFA for 15 min at room temperature. After fixation, the cells were washed once with PBS containing 0.5% bovine serum albumin (BSA) (PBSA) and permeabilized with 100% methanol for 45 min at 4 °C. After permeabilization, the cells were washed twice with PBSA and stained for one hour at room temperature using Alexa Fluor 647-conjugated phospho-STAT5 (Tyr694) rabbit monoclonal antibody (Cell Signaling Technology, 9365, clone C71E5) diluted 1:100. After three washing steps, the cells were analysed using a CytoFlex S flow cytometer (Beckman Coulter). Data were analysed with CytExpert software, and cells were gated on SSC-A versus FSC-A. Each experiment was done in triplicate, and the results were analysed accordingly. 5g, pSTAT5 signal is normalized to the background level. 5h, n = 3 time courses were normalized to the signal at time 0 (just before effector addition) and a baseline signal was determined by the average of this normalized pSTAT5 of the untreated cells, then each time course was renormalized to this baseline signal. Some batches of cells were unresponsive to stimulation; data from these batches were excluded from analysis. HeLa cells for single-molecule imaging were sourced from the DSMZ (ACC 57); YT-1 (CD25+) cells were sourced from ATCC. Cell lines were authenticated by short tandem repeat profiling. HeLa cells tested negative for mycoplasma contamination by PCR. Primary human peripheral blood mononuclear cells (PBMCs) isolated from a healthy donor by leukapheresis were thawed and resuspended at a cell density of 2 × 106 cells per ml in T cell medium, which contained RPMI 1640 (Gibco), FBS (10% v/v, Gibco, Thermo Fisher Scientific), HEPES (25 mM, Gibco, Thermo Fisher Scientific), penicillin–streptomycin (1% v/v, Gibco, Thermo Fisher Scientific), sodium pyruvate (1% v/v, Gibco, Thermo Fisher Scientific), MEM non-essential amino acids solution (1% v/v, Gibco, Thermo Fisher Scientific) and 2-mercaptoethanol (0.1% v/v, Gibco, Thermo Fisher Scientific), and supplemented with human IL-2 (100 U ml−1). For human T cell activation, cells were activated with plate-bound anti-human CD3ε (1 μg ml−1, clone OKT-3, BioXCell) and soluble anti-human CD28 (5 μg ml−1, clone 9.3, BioXCell) for 48 h. Cells were then expanded in T cell medium with human IL-2 (100 U ml−1) for eight days. On day 9, IL-2 was withdrawn for 36 h, after which cells were resuspended at 1 × 106 cells per ml and stimulated with ASNeo2 (1 nM or 5 nM) for either 5 or 25 min, as indicated. Effector peptide (10 μM) was then added to terminate signalling. After 48 or 72 h, cells were collected for counting and phenotypic analysis by flow cytometry. 5i,j, cells were stimulated with 1 nM ASNeo2. 5k–n, cells were stimulated with 5 nM ASNeo2. For surface-marker staining, cells were collected into U-bottom 96-well plates (Thermo Fisher Scientific), blocked with Human TruStain FcX (BioLegend) and incubated with the indicated antibodies at 4 °C for 20 min, followed by live–dead staining by 4',6-diamidino-2-phenylindole (DAPI; Thermo Fisher Scientific). Cells were then washed and resuspended with FACS buffer (PBS containing 0.2 % BSA; Sigma-Aldrich) for flow-cytometry analyses. For phospho-STAT staining, primary human T cells were rested in T cell medium lacking IL-2 for 24 h before signalling assays. Cells were stimulated with 50 µl ASNeo2 for 5 min or 25 min, followed by the addition of effector peptide and incubation for another 20 min at 37 °C, and the reaction was terminated by fixation with 1.5% PFA for 15 min at room temperature with agitation. Cells were washed and permeabilized with ice-cold 100% methanol for 60 min on ice. Afterward, cells were washed with FACS buffer before staining with pSTAT5 antibodies for one hour at 4 °C in the dark. Cells were washed and resuspended in FACS buffer for flow-cytometry analyses. For caspase-3 staining, cells were first stained for surface markers and Zombie Violet Fixable Dye (BioLegend), followed by staining with a FITC Active Caspase-3 Apoptosis Kit (BD Biosciences) according to the manufacturer's protocol. For transcription factor staining, cells were first stained for surface markers and Zombie Violet Fixable Dye, then fixed and permeabilized using a Foxp3/Transcription Factor Staining Buffer Set (eBioscience) as per the manufacturer's instructions. Cells were subsequently incubated with the indicated antibodies for intracellular staining. Detection was performed using a CytoFlex (Beckman Coulter), and data were analysed with FlowJo (v.10.10.0). The following antibodies or staining reagents were purchased from BD Biosciences: pSTAT5 (47/Stat5, 612599), and BD Pharmingen FITC Active Caspase-3 Apoptosis Kit (571606). DAPI was purchased from Thermo Fisher Scientific. Antibodies were validated by the manufacturers using flow cytometry. Human T cells on day 9 after activation were subjected to IL-2 starvation for 36 h. The unstimulated control cells were maintained in culture without IL-2. For transient IL-2 signalling, cells were stimulated with ASNeo2 (5 nM) for 25 min, followed by the addition of effector peptide (10 μM) to terminate signalling. For sustained signalling, ASNeo2 (5 nM) was continuously maintained in the culture medium. Total RNA was extracted using the Quick-RNA Miniprep Kit (Zymo Research) following the manufacturer's instructions. cDNA was synthesized using the High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific) with random primers. qPCR was performed using TaqMan master mix (Thermo Fisher Scientific) on a StepOnePlus Real-Time PCR System. BCL2 expression was measured using the TaqMan BCL2 assay, and gene expression was normalized to GAPDH (Thermo Fisher Scientific). Relative expression levels were calculated using the ΔΔCt method. Human T cells on day 8 after activation were subjected to IL-2 starvation for 36 h. The unstimulated control cells were maintained in culture without IL-2. For transient IL-2 signalling, cells were stimulated with ASNeo2 (5 nM) for 25 min, followed by the addition of effector peptide (10 μM) to terminate signalling. For sustained signalling, ASNeo2 (5 nM) was continuously maintained in the culture medium. After six hours, cells were collected, and total RNA was extracted using the Quick-RNA Miniprep Kit (Zymo Research). RNA libraries were prepared using a poly(A) enrichment-based mRNA library preparation kit following the manufacturer's instructions. Libraries were pooled and sequenced on the NovaSeq X Plus Series (PE150). Reads were aligned to the reference genome (GRCh38) using Rsubread, and gene expression was quantified with featureCounts. For analysing pathway enrichment, gene set co-regulation analysis (GESECA) was performed with hallmark gene sets from the Human Molecular Signatures Database (MSigDB). To reduce redundancy, we performed hierarchical clustering of hallmark gene sets with Jaccard's distance to yield eight gene-set clusters with minimal gene overlaps. Only one gene set with the lowest false discovery rate (FDR)-adjusted P value per cluster is shown in Fig. Differential expression analysis was performed using DESeq2, with FDR-adjusted P < 0.05 as a threshold for differential expression. Heat maps were created using normalized counts of genes identified as differentially upregulated or downregulated in both sustained versus unstimulated and transient versus unstimulated comparisons. The main-text SPR, circular dichroism, FP, FRET and luminescence experiments, and also a subset of the DEER and SEC experiments, were performed twice (SPR in Figs. 2d,h and 4b; FRET in Fig. 4c; and SEC in Supplementary Fig. 9a) or three times (SPR in Fig. 4d,e; SEC in Supplementary Fig. 6b; and DEER with AS1 in Extended Data Fig. 2) to ensure reproducibility and low variance, and one representative experiment was reported owing to low variance among replicates. Having established low variance for these experiments, some similar measurements reported in the Supplementary Information were performed once. For all microscopy, cell staining, qPCR and RNA-seq experiments, three or four independent biological replicates were performed unless otherwise noted in the figure legend. All attempted replications of all experiments were successful, with the exception that some batches of YT-1 cells were unresponsive to stimulation; the findings in Fig. 5h were not observed with those batches. No sample size calculations were performed; rather the sample sizes were chosen based on experience and were sufficient for the important observed differences between groups to be strongly statistically significant. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated during this study are included either in the main text or as Supplementary Information. Sequences for all designs are available in Supplementary Table 4. Atomic coordinates and structure factors for all crystal structures reported in this paper have been deposited in the Protein Data Bank (PDB; https://www.rcsb.org) with accession codes 9DCX, 9DCY, 9DCZ, 9DD0, 9DD1, 9DD2, 9DD3, 9DD4, 9DD5 and 9OLQ. PDB models and sequences for all designs and source data with analysis scripts have been deposited at Zenodo under https://doi.org/10.5281/zenodo.16749448 (ref. Single-molecule tracking data have been deposited at Zenodo: calibration beads, unstimulated samples, long-term measurements and labelled ligand experiments under https://doi.org/10.5281/zenodo.13957447 (ref. 98), Neo2 and Neo2 + effector under https://doi.org/10.5281/zenodo.13957498 (ref. The raw RNA-seq data have been deposited in the NCBI Sequence Read Archive (SRA) under BioProject accession code PRJNA1302552. The SKEMPI database can be accessed at https://life.bsc.es/pid/skempi2/database/index. The reference genome GRCh38 (accession code GCF_000001405.26) can be accessed at https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/. Hallmark gene sets can be accessed from MSigDB at https://www.gsea-msigdb.org/gsea/msigdb/collections.jsp. Code used to generate designs and analysis scripts used to generate all figures has been deposited at Zenodo (design code: https://doi.org/10.5281/zenodo.16749263 (ref. Bogetti, A. T., Presti, M. F., Loh, S. N. & Chong, L. T. The next frontier for designing switchable proteins: rational enhancement of kinetics. & Kortemme, T. Design principles of protein switches. Huang, P.-S. et al. RosettaRemodel: a generalized framework for flexible backbone protein design. Koga, N. et al. Principles for designing ideal protein structures. Huang, P.-S. et al. High thermodynamic stability of parametrically designed helical bundles. Watson, J. L. et al. De novo design of protein structure and function with RFdiffusion. Kinetic enhancement of NF-κB·DNA dissociation by IκBα. Facilitated dissociation of transcription factors from single DNA binding sites. Sottini, A. et al. Polyelectrolyte interactions enable rapid association and dissociation in high-affinity disordered protein complexes. Holzbaur, E. L. F. & Johnson, K. A. Microtubules accelerate ADP release by dynein. Reubold, T. F., Eschenburg, S., Becker, A., Kull, F. J. A structural model for actin-induced nucleotide release in myosin. Packschies, L. et al. GrpE accelerates nucleotide exchange of the molecular chaperone DnaK with an associative displacement mechanism. Singh, A. K. et al. Substrate-induced facilitated dissociation of the competitive inhibitor from the active site of O-acetyl serine sulfhydrylase reveals a competitive-allostery mechanism. Abdiche, Y. N. et al. Antibodies targeting closely adjacent or minimally overlapping epitopes can displace one another. & Wright, P. E. Hypersensitive termination of the hypoxic response by a disordered protein switch. Accelerated disassembly of IgE–receptor complexes by a disruptive macromolecular inhibitor. Directed evolution of and structural insights into antibody-mediated disruption of a stable receptor-ligand complex. Alverdi, V., Hetrick, B., Joseph, S. & Komives, E. A. Direct observation of a transient ternary complex during IκBα-mediated dissociation of NF-κB from DNA. & Wolynes, P. G. Molecular stripping in the NF-κB/IκB/DNA genetic regulatory network. Chen, W., Lu, W., Wolynes, P. G. & Komives, E. A. Single-molecule conformational dynamics of a transcription factor reveals a continuum of binding modes controlling association and dissociation. & Wright, P. E. Role of backbone dynamics in modulating the interactions of disordered ligands with the TAZ1 domain of the CREB-binding protein. Davies, A. M. et al. Allosteric mechanism of action of the therapeutic anti-IgE antibody omalizumab. & Marko, J. F. How do DNA-bound proteins leave their binding sites? The role of facilitated dissociation. Åberg, C., Duderstadt, K. E. & van Oijen, A. M. Stability versus exchange: a paradox in DNA replication. & Seelig, G. Dynamic DNA nanotechnology using strand-displacement reactions. Sarraf, N., Rodriguez, K. R. & Qian, L. Modular reconfiguration of DNA origami assemblies using tile displacement. Miyashita, O., Onuchic, J. N. & Wolynes, P. G. Nonlinear elasticity, proteinquakes, and the energy landscapes of functional transitions in proteins. Wang, H. & Oster, G. Ratchets, power strokes, and molecular motors. Structural basis of omalizumab therapy and omalizumab-mediated IgE exchange. Praetorius, F. et al. Design of stimulus-responsive two-state hinge proteins. Sahtoe, D. D. et al. Reconfigurable asymmetric protein assemblies through implicit negative design. Kinetic advantage of intrinsically disordered proteins in coupled folding–binding process: a critical assessment of the “fly-casting” mechanism. & Tomko, E. J. Conformational selection and induced fit as a useful framework for molecular motor mechanisms. & Dill, K. A. Molecular motors: power strokes outperform Brownian ratchets. & Wright, P. E. Conformational propensities of intrinsically disordered proteins influence the mechanism of binding and folding. Paramanathan, T., Reeves, D., Friedman, L. J., Kondev, J. A general mechanism for competitor-induced dissociation of molecular complexes. Dixon, A. S. et al. NanoLuc complementation reporter optimized for accurate measurement of protein interactions in cells. Quijano-Rubio, A. et al. De novo design of modular and tunable protein biosensors. Cao, L. et al. De novo design of picomolar SARS-CoV-2 miniprotein inhibitors. The drug–target residence time model: a 10-year retrospective. Instructive roles for cytokine-receptor binding parameters in determining signaling and functional potency. Martinez-Fabregas, J. et al. Kinetics of cytokine receptor trafficking determine signaling and functional selectivity. Shouse, A. N., LaPorte, K. M. & Malek, T. R. Interleukin-2 signaling in the regulation of T cell biology in autoimmunity and cancer. Lowenthal, J. W. et al. High and low affinity IL 2 receptors: analysis by IL 2 dissociation rate and reactivity with monoclonal anti-receptor antibody PC61. Fujii, M. et al. High-affinity receptor-mediated internalization and degradation of interleukin 2 in human T cells. et al. De novo design of potent and selective mimics of IL-2 and IL-15. Arneja, A., Johnson, H., Gabrovsek, L., Lauffenburger, D. A. & White, F. M. Qualitatively different T cell phenotypic responses to IL-2 versus IL-15 are unified by identical dependences on receptor signal strength and duration. Jones, R. G. & Thompson, C. B. Revving the engine: signal transduction fuels T cell activation. Khalil, A. S. et al. Kinesin's cover-neck bundle folds forward to generate force. Kumar, A., Moreau, J. L., Gibert, M. & Thèze, J. Internalization of interleukin 2 (IL-2) by high affinity IL-2 receptors is required for the growth of IL-2-dependent T cell lines. Wang, J. et al. Scaffolding protein functional sites using deep learning. The Rosetta all-atom energy function for macromolecular modeling and design. Robust deep learning–based protein sequence design using ProteinMPNN. Highly accurate protein structure prediction with AlphaFold. Improving de novo protein binder design with deep learning. Wicky, B. I. M. et al. Hallucinating symmetric protein assemblies. Dang, B. et al. SNAC-tag for sequence-specific chemical protein cleavage. Di Cera, E. Mechanisms of ligand binding. Walter, T. S. et al. Lysine methylation as a routine rescue strategy for protein crystallization. Winn, M. D. et al. Overview of the CCP4 suite and current developments. McCoy, A. J. et al. Phaser crystallographic software. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. Tessmer, M. H. & Stoll, S. chiLife: an open-source Python package for in silico spin labeling and integrative protein modeling. Tessmer, M. H., Canarie, E. R. & Stoll, S. Comparative evaluation of spin-label modeling methods for protein structural studies. Fábregas Ibáñez, L., Jeschke, G. & Stoll, S. DeerLab: a comprehensive software package for analyzing dipolar electron paramagnetic resonance spectroscopy data. Hasanbasri, Z., Tessmer, M. H., Stoll, S. & Saxena, S. Modeling of Cu(ii)-based protein spin labels using rotamer libraries. Jo, S., Kim, T., Iyer, V. G. & Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. Lee, J. et al. CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Berendsen, H. J. C., van der Spoel, D. & van Drunen, R. GROMACS: a message-passing parallel molecular dynamics implementation. Van Der Spoel, D. et al. GROMACS: Fast, flexible, and free. Hess, B., Bekker, H., Berendsen, H. J. C. & Fraaije, J. G. E. M. LINCS: a linear constraint solver for molecular simulations. & Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. Daura, X. et al. Peptide folding: when simulation meets experiment. The ALFA-tag is a highly versatile tool for nanobody-based bioscience applications. Mechanism of homodimeric cytokine receptor activation and dysregulation by oncogenic mutations. Wilmes, S. et al. Receptor dimerization dynamics as a regulatory valve for plasticity of type I interferon signaling. Reed, S. E., Staley, E. M., Mayginnes, J. P., Pintel, D. J. & Tullis, G. E. Transfection of mammalian cells using linear polyethylenimine is a simple and effective means of producing recombinant adeno-associated virus vectors. Sotolongo Bellón, J. et al. Four-color single-molecule imaging with engineered tags resolves the molecular architecture of signaling complexes in the plasma membrane. You, C., Richter, C. P., Löchte, S., Wilmes, S. & Piehler, J. Dynamic submicroscopic signaling zones revealed by pair correlation tracking and localization microscopy. Winkelmann, H., Richter, C. P., Eising, J., Piehler, J. & Kurre, R. Correlative single-molecule and structured illumination microscopy of fast dynamics at the plasma membrane. Edelstein, A. D. et al. Advanced methods of microscope control using μManager software. Sergé, A., Bertaux, N., Rigneault, H. & Marguet, D. Dynamic multiple-target tracing to probe spatiotemporal cartography of cell membranes. Niewidok, B. et al. Single-molecule imaging reveals dynamic biphasic partition of RNA-binding proteins in stress granules. Robust single-particle tracking in live-cell time-lapse sequences. Aguet, F., Antonescu, C. N., Mettlen, M., Schmid, S. L. & Danuser, G. Advances in analysis of low signal-to-noise images link dynamin and AP2 to the functions of an endocytic checkpoint. Accurate FRET measurements within single diffusing biomolecules using alternating-laser excitation. Design of facilitated dissociation enables timing of cytokine signaling - designed proteins, analysis scripts, and raw data. Pollmann, C. Design of facilitated dissociation enables control over cytokine signaling duration - SMT raw data - unstimulated - calibration beads - long term tracking - labelled ligand. Pollmann, C. Design of facilitated dissociation enables control over cytokine signaling duration - SMT raw data - neo2 - neo2 + Effector. Pollmann, C. Design of facilitated dissociation enables control over cytokine signaling duration - SMT raw data - ASneo2 - ASneo2 + Effector - intraFRET ASneo2. Broerman, A. J. abroerman/facilitated-dissociation-code: code for Facilitated Dissociation Paper. & Wright, P. E. Multivalency enables unidirectional switch-like competition between intrinsically disordered proteins. This work was supported by the Audacious Project at the Institute for Protein Design (A.J.B., A.K., J.D.L.C., E.B. ); by a gift from Microsoft (A.J.B. ); by the Nordstrom Barrier Institute for Protein Design Directors Fund (M.H.A. ); by Bill and Melinda Gates Foundation OPP1156262 (A.K. ); by the Open Philanthropy Project Improving Protein Design Fund (E.B. ); by the National Institutes of Health (NIH) National Institute of Allergy and Infectious Disease grant R0AI160052 (A.K.B. ); by CRI Irvington Postdoctoral Fellowship 315511 (Y.Z. ); by National Cancer Institute K00 award 4K00CA274708 (M.O. ); by the DFG grants PI 405/15 and SFB 1557 (C.P. The EPR spectrometer used for the DEER experiments was in part supported by NIH grant S10OD021557. This research used resources (FMX/AMX) of the National Synchrotron Light Source II, a US Department of Energy (DoE) Office of Science User Facility operated for the DOE Office of Science by Brookhaven National Laboratory under contract DE-SC0012704. The Center for BioMolecular Structure (CBMS) is supported mainly by the NIH National Institute of General Medical Sciences (NIGMS) through a Center Core P30 Grant (P30GM133893), and by the DoE Office of Biological and Environmental Research (KP1607011). This work is based on research performed at the Northeastern Collaborative Access Team beamlines, which are funded by the NIGMS (P30 GM124165). The research used resources of the Advanced Photon Source, a US DoE Office of Science User Facility operated for the DoE Office of Science by Argonne National Laboratory under contract DE-AC02-06CH11357. The Advanced Light Source is supported by the Director, Office of Science, Office of Basic Energy Sciences and US DoE (DE-AC02-05CH11231). Institute for Protein Design, University of Washington, Seattle, WA, USA Adam J. Broerman, Mauriz A. Lichtenstein, Mohamad H. Abedi, Danny D. Sahtoe, Aza Allen, Alex Kang, Joshmyn De La Cruz, Evans Brackenbrough, Asim K. Bera, Florian Praetorius & David Baker Adam J. Broerman, Mohamad H. Abedi, Danny D. Sahtoe, Aza Allen, Alex Kang, Joshmyn De La Cruz, Evans Brackenbrough, Asim K. Bera, Florian Praetorius & David Baker Department of Molecular and Cellular Physiology, Stanford University School of Medicine, Stanford, CA, USA Yang Zhao, Masato Ogishi & K. Christopher Garcia Department of Structural Biology, Stanford University School of Medicine, Stanford, CA, USA Yang Zhao, Masato Ogishi & K. Christopher Garcia Mark D. Jackson, Maxx H. Tessmer & Stefan Stoll Department of Biomedical Engineering, Oregon Health and Science University, Portland, OR, USA Won Hee Ryu & Daniel M. Zuckerman Howard Hughes Medical Institute, University of Washington, Seattle, WA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the facilitated dissociation design concept. developed the computational design pipeline, designed the allosteric switches and characterized their binding kinetics with SPR, FP, FRET, luminescence and SEC experiments, with mentorship from F.P. performed and analysed single-molecule localization microscopy and NK cell signalling activity experiments. performed and analysed transient stimulation experiments on primary T cells. designed and characterized the rapid sensors. performed and analysed DEER experiments. performed and analysed MD simulations. performed initial signalling activity experiments. and A.A. performed site-saturation mutagenesis on LHD101. determined and analysed crystal structures. All authors read and commented on the manuscript. Correspondence to Adam J. Broerman, Florian Praetorius or David Baker. are in the process of filing a provisional patent application that incorporates discoveries described in this article. The remaining authors declare no competing interests. Nature thanks Hideki Aihara, Doug Barrick and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Crystal structure of AS5 alone (grey) overlaid with the design model of AS5 in state X (blue). Inset shows a detailed view of side chains in the partially open effector-binding cleft. b, Cocrystal structure of AS5 and peptide effector (grey) overlaid with the design model of the AS5–effector complex in state Y (AS5 in blue, effector in orange). Inset shows the same view of the side chains in the effector-binding cleft as in a. c, Design model of AS1 in state X (blue) aligned to the target (pink), showing a minor clash. d, Three cocrystal structures of AS1 (with intact cleft) and target with methylated lysines (grey) overlaid with the AF2 model of the target–AS1 complex in state X (AS1 in blue, target in pink), showing fluctuation in the target binding conformation. a, Raw DEER traces (black), foreground fits (colours), and background fits (grey) for AS1 and AS114 with all combinations of target and effector. Experiments on complexes included the target, effector, or both in excess over the host at concentrations higher than required to fully form the complex (Supplementary Fig. 9), and spin labels were placed far from the target and effector binding sites. Thus, changes in the DEER distance distributions with different combinations of target and effector should only reflect changes in host conformation. b, Distance distributions from experiment (colours) and simulated from the structural state represented by the cartoons (black). For AS1, the simulated and experimental distance distributions agree well, further validating that each state adopts its designed conformation. c, Experimental distance distributions of all states, coloured corresponding to b. For both AS1 and AS114, the ternary complex distribution (green) aligns with the host–effector complex distribution (orange) and not with the host alone (blue) or target–host complex (pink) distributions, confirming that the ternary complex is primarily in state Y. b,c, Lines represent the distance distribution which best fits the time domain data; shaded regions represent 95% confidence intervals from bootstrapping (Methods). b, Cα RMSD of the switch (blue) and target (pink) from their position in the crystal structure when the entire structures are aligned. Compared to trajectories 1 and 2, trajectory 3 shows reduced target deformation and increased switch deformation, showing that these trajectories differ in where they localize strain to resolve the clash. c, Per-residue Cα RMSF of the host (blue), target (pink), and effector (orange) in the ternary complex computed from each trajectory. The clashing region of the target (highlighted in grey) shows considerable flexibility, according with this region being disordered in the crystal structure. d, Comparison of MD simulations to experimental data. Left, crystal conformation of the ternary complex (grey) aligned to representative conformations from each MD trajectory (red, yellow, and light blue). DEER spin-label positions are shown in green. In the MD simulations, although flexible, this region remains mostly ordered, causing additional deformation compared to the crystal structure. Right, experimentally measured DEER distance distribution of the ternary complex (green line representing the best fit to the time domain data and shaded region representing 95% confidence interval from bootstrapping (Methods)) and distance distributions simulated from the crystal structure (grey line) or MD trajectories (dashed lines, colours correspond to the conformations shown at left). a, Ratios of fold accelerations in the forward direction to those in the reverse direction, given by the equation: acceleration ratio = (koff,T:HE/koff,T:H) ÷ (koff,TH:E/koff,H:E). Acceleration ratios corresponding to designs with asymmetric distributions of strain are coloured orange, and to symmetric blue. b, Plot of forward vs reverse acceleration ratios, with linear fits for the symmetric and asymmetric groups. c, For each design, design model of host–effector complex in state Y (blue and orange) aligned to the target (pink) to show the designed allosteric clash, and (grey) AF2 prediction of the target position relative to the switch in the ternary complex to show how the clash resolves through global strain. The target deforms downward in designs with an asymmetric distribution of strain, shearing the beta sheet, whereas it deforms outward in designs with a more symmetric distribution of strain, bending the beta sheet. a, Design model of E2–target, comprising the target LHD101A (with mutations R43V and V69Q) fused to the effector peptide “E2” (cs201B) for hinge cs201. b, Design models of E2–target (green/pink) bound to AS114 (blue) in state X showing no clash (left) and in state Y with the effector peptide (orange) showing a strong clash (right). c, Design models of the reporter hinge “H2” (cs201F with mutation E249L (sticks) which increases E2 on-rate and labelled with Alexa Fluors 555 and 647 at positions indicated by stars) in state X (left) and in state Y with E2–target (right). AS114 and H2 would overlap considerably if simultaneously bound to E2–target, so their binding should be mutually exclusive: AS114 should cage E2 until its release by the effector. d, Kinetics of E2–target and H2 association, measured by a change in FRET efficiency due to the conformational change in H2 upon binding. (Left) FRET time courses (normalized to the initial signal) with varying concentrations of E2–target and 5 nM H2; data (circles) fit with single exponentials (lines). (Right) apparent on-rate constants plotted against E2–target concentration (circles) and a linear fit. The value of the association rate constant (5e + 4 M−1s−1) is higher than the reported value (4.5e + 3 M−1s−1) for the original hinge cs201F with effector cs201B, suggesting that mutation E249L on H2 biases its conformational pre-equilibrium toward state Y to increase the apparent association rate. e, Additional data for the kinetically governed chain reaction shown in Fig. In the grey control time course, 500 nM AS114 was added to 20 nM H2, then 1 μM effector and 6 μM target was added at time 0, showing that none of these components bind to H2 to cause a change in FRET signal. In the other time courses, pre-incubated 500 nM AS114 and 250 nM E2–target was added to 20 nM H2, then buffer (blue), 1 μM effector (green), 6 μM target (pink), or both (orange) were added at time 0. A baseline drift (obtained from 500 nM AS114 after adding 20 nM H2 then at time 0 adding buffer) was subtracted from each time course, and time courses were normalized to the initial signal. The chain reaction proceeds faster when just excess target is added, probably due to blocking rebinding of E2–target to AS114 after transient dissociation39, but this effect is insufficient to achieve full acceleration. The chain reaction also proceeds faster when just effector is added, but probably due to transient rebinding of E2–target to re-form the strained ternary complex, this also does not achieve full acceleration. Note that if a single-chain effector is desired to fully accelerate the chain reaction, the effector and target could be flexibly fused into a single construct. Such a multivalent effector would be reminiscent of CITED2, whose multivalency enables rapid and unidirectional competition against HIF-1α (ref. a, Structural model of the best SARS-CoV-2 sensor construct, comprising AScov (blue), the SmBiTgraft peptide with the effector (orange) and grafted SmBiT (green), LgBiT (grey or cyan), and flexible linkers (black). In this design, SmBiT is caged in a helical conformation when SmBiTgraft is bound to the switch and is free to reconstitute the luciferase when SmBiTgraft is released. To form SmBiTgraft, SmBiT was grafted onto the effector peptide such that most of its hydrophobic residues are buried within the switch–SmBiTgraft interface when in the bound helical conformation. Because the original effector peptide binds so strongly to the switch, it could accommodate replacing some of its interface residues with residues from SmBiT without reducing its affinity so much that it no longer effectively cages the SmBiT. b, SPR data showing sfGFP-SmBiTgraft binding to AS0 (blue and orange, association phase), slow subsequent dissociation in the absence of target (blue), and rapid subsequent dissociation upon addition of 10 μM target (orange) caused by rapid target binding to form a transient ternary complex, causing the spike at the beginning of the dissociation phase. c–f, Rapidly sensing the target through a facilitated dissociation mechanism (top), slowly sensing the effector limited by the slow base exchange rate of SmBiTgraft between binding AS0 and LgBiT (middle), and rapidly sensing the SARS-CoV-2 RBD with facilitated dissociation (bottom). c, Schematics showing the mechanism of sensing. d, Luminescence time courses (normalized to the initial signal) of 10 pM sensor construct then at time 0 adding varying concentrations of analyte (target, effector, or SARS-CoV-2 RBD); data (colours) fit (black) with single exponentials up to the maximum signal for time courses which showed appreciable signal increase. In some time courses, signal slowly decreases due to depletion of luciferase substrate. e, Luminescence signal fold change plotted against analyte concentration. f, Sensor response rate constant plotted against analyte concentration for time courses that showed appreciable signal increase. a, Schematic depiction of the labelling strategy for single-molecule tracking experiments. b, Single-molecule trajectories of IL-2Rβ (red), γc (blue) and ASNeo2-induced heterodimers (magenta). 5e as box plots to display datapoint variation including Neo2 (+/- effector) (green and orange, left side) and single-molecule tracking experiments with labelled ASNeo2 and γc (right side). e, Diffusion properties of IL-2Rβ and γc are reverted to the ground state after addition of effector. f, Immobile particles are increased upon stimulation, but not decreased after effector addition, potentially indicating receptors internalizing in membrane proximal endosomes. For e,f, the left box always corresponds to IL-2Rβ and the right one to γc. Sample sizes for d–f are as in c. g, Normalized localization density over time confirms minimal single-molecule bleaching in long-term single-molecule tracking experiments. Sample sizes and independent repeats are: without Effector: 5 and 5; with Effector: 3 and 3. h,i, Dissociation of ASNeo2-induced IL-2Rβ/γc dimers at the cell surface upon addition of 10 µM effector as detected by time-lapse single-molecule co-tracking (h) with colour-coded corresponding co-trajectories (i). j,k, Conformational change of ASNeo2 bound to the cell-surface receptor probed by smFRET. j, FRET efficiency histograms for ASNeo2 E4C/K211C labelled with Cy3B and ATTO643 bound to cells expressing IL-2Rβ and γc in the absence (blue) and presence (yellow) of 10 µM effector. Sample sizes are: without Effector: 7; with Effector: 5. k, smFRET co-localizations of one individual cell before the effector was added (left) and after it was added (right) colour-coded for FRET efficiency, highlighting the observation of individual molecules. Statistics for c,e,f were performed using two-sided two-sample Kolmogorov–Smirnov tests (ns, not significant, P values noted). Scale bars in b,i: 5 µm. a, Design models of ASNeo2 and selected cyclic permutations in state X, rainbow-coloured from N terminus (blue) to C terminus (red) to illustrate the protein topology. This way, the regulatory switch cannot degrade without also breaking Neo2. b, SEC purifications performed on a Superdex 200 Increase 10/300 GL column. The cyclic permutations are prone to aggregation during expression, but distinct monomer peaks can be picked out. c, Fast effector-concentration-dependent dissociation of γc from the ASNeo2–IL-2Rβ–γc complex upon addition of peptide effector. Data (grey) fit (colours) as described in methods (neglecting the accumulation modelling because accumulation on the SPR surface was negligible with these proteins). d, Rate constants of facilitated γc dissociation computed from the model fit by ln(2)÷{half-time of γc–host interaction} plotted against effector concentration (circles) and fit with hyperbolic equations (black lines). Human T cells were stimulated with 1 nM (a–g) or 5 nM (i) ASNeo2 for 5 min (a–h), 30 min (h), or 25 min (i) or left untreated as a control. Signalling was either sustained by continued ASNeo2 treatment or terminated by the addition of 10 μM effector. Cells were collected for counting and phenotypic analysis by flow cytometry after 72 h (b,c; n = 4) or 48 h (a,e–g; n = 4), for pSTAT signalling analysis by flow cytometry after 20 min (h, n = 3), or for RNA-seq analysis after 6 h (i, n = 3). a, Changes in viable T cell counts from 0 h to 48 h across each group. b,c, Representative flow-cytometry histogram of CFSE (b) and quantitative analysis of divided cells (c). d, Frequencies of live T cells. e, Mean fluorescence intensity (MFI) of Ki-67, CD25, GATA3, CD69, and BCL2. f,g, Representative flow-cytometry plots (f) and quantitative analysis of caspase-3+ cells (g; n = 4). i, Transcripts per million (TPM) of CDK4 and POLD2 in the MYC_TARGETS_V1 gene set (left); TPM of DOCK2 and KIF1B in the MITOTIC_SPINDLE gene set (right). (a,h,i) or SD (c–e,g). n refers to biologically independent samples. Example of IL-2Rβγc facilitated dissociation on the cell surface. Microscopy recording showing single-molecule trajectories of fluorescently labelled IL-2Rβ (red) and γc (blue) on the plasma membrane of live cells. The cells had been incubated with 100 nM ASNeo2, causing co-localization of one IL-2Rβγc pair (circled in white) within this frame. Seven seconds into the recording, 10 μM effector was added, leading to γc dissociation four seconds later. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Broerman, A.J., Pollmann, C., Zhao, Y. et al. Design of facilitated dissociation enables timing of cytokine signalling. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68035032/hidden-edge-state-atoms-energy-transmission/'>Scientists Found the Hidden 'Edge State' That May Lead to Practically Infinite Energy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 13:32:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Take, for example, what is known as the quantum Hall effect. First discovered in 1980 by German physicist Klaus von Klitzing, this effect describes the behavior of electrons (under the influence of a magnetic field and approaching absolute zero temperatures) as they pass through 2D materials, such as graphene. Usually, you'd expect the electrons to experience resistance and scatter, but under these conditions, they formed lossless energy states locked along the material's boundary. This quantization of electrical resistance, known as an “edge state,” is particularly useful if you want to create exotic materials free of electrical resistance. “These states occur over femtoseconds, and across fractions of a nanometer, which is incredibly difficult to capture,” Richard Fletcher, an assistant professor at MIT, said in a press statement. “The beauty is seeing with your own eyes physics which is absolutely incredible but usually hidden away in materials and unable to be viewed directly.” To actually study this quantum interaction on a more reasonable scale, Fletcher—along with his colleagues at MIT's Research Laboratory of Electronics and the MIT-Harvard Center for Ultracold Atoms—decided on a novel method to essentially scale up this phenomenon using a cloud of ultracold sodium atoms instead of electrons. According to the researchers, this allowed the team to watch these edge states form “over milliseconds and microns,” which are much more manageable experimental parameters. The team used one million ultracold sodium atoms and essentially trapped them in a complex set-up of lasers. However, to simulate the experience of living in a flat space, the researchers also spun them like “riders on an amusement park Gravitron.” “The trap is trying to pull the atoms inward, but there's centrifugal force that tries to pull them outward,” Fletcher said. There's also a third force, the Coriolis effect, such that if they try to move in a line, they get deflected. So these massive atoms now behave as if they were electrons living in a magnetic field.” Once atoms encountered this light, they flowed in just one direction—much like electrons at ultra-small quantum scales. “You can imagine these are like marbles that you've spun up really fast in a bowl, and they just keep going around and around the rim of the bowl,” Martin Zwierlein, a co-author of the study, said in a press statement. To test these atoms' resistances, the team then placed obstacles—such as a point of light—in their paths, and the atoms passed by without any measurable resistance. Now that scientists have a reliable stand-in for this quantum process, future experiments can push these interactions to the “edge” and begin to explore unknown frontiers of this fascinating piece of quantum physics. Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-025-00957-4'>Herbal terpenoids activate autophagy and mitophagy through modulation of bioenergetics and protect from metabolic stress, sarcopenia and epigenetic aging</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 10:41:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Small molecular food components contribute to the health benefits of diets rich in fruits, vegetables, herbs and spices. The cellular mechanisms by which noncaloric bioactives promote healthspan are not well understood, limiting their use in disease prevention. Here, we deploy a whole-organism, high-content screen in zebrafish to profile food-derived compounds for activation of autophagy, a cellular quality control mechanism that promotes healthy aging. We identify thymol and carvacrol as activators of autophagy and mitophagy through a transient dampening of the mitochondrial membrane potential. Chemical stabilization of thymol-induced mitochondrial depolarization blocks mitophagy activation, suggesting a mechanism originating from the mitochondrial membrane. Supplementation with thymol prevents excess liver fat accumulation in a mouse model of diet-induced obesity, improves pink-1-dependent heat stress resilience in Caenorhabditis elegans, and slows the decline of skeletal muscle performance while delaying epigenetic aging in SAMP8 mice. Thus, terpenoids from common herbs promote autophagy during aging and metabolic overload, making them attractive molecules for nutrition-based healthspan promotion. Despite an extraordinary increase in lifespan over the past 170 years1, the prevalence of chronic diseases among the rapidly growing population of older adults remains high due to environmental, lifestyle and nutritional factors2. Among these, inadequate nutrition makes a major contribution to the onset and progression of noncommunicable diseases and mortality. The intake of excess calories, as well as the overconsumption or underconsumption of specific food and nutrient groups, has been associated with a wide range of negative health outcomes3. Prevention through nutritional approaches, sometimes referred to as ‘food as first medicine', is therefore gaining importance in lowering the burden of noncommunicable diseases4. Beyond recommendations for adequate portion sizes and balanced consumption of nutrient-dense foods, specific dietary patterns are effective tools for disease prevention. For instance, caloric restriction and regimens under the umbrella term of intermittent fasting, such as alternate-day fasting, 5:2 fasting and time-restricted eating, show promise in promoting healthy aging not only through weight loss but also through other mechanisms independent of changes in weight5,6,7,8. At the cellular level, various molecular adaptations promote the health benefits of caloric restriction and intermittent fasting, among which the activation of autophagy is believed to be a principal mechanism explaining their geroprotective effects9. Autophagy prevents proteotoxicity and cellular damage by recycling protein aggregates and dysfunctional organelles, leading to improved organism resilience against metabolic challenges and other age-related cellular stressors10. The requirement for intact autophagy to mediate the benefits of caloric restriction in extending healthspan and lifespan has been demonstrated by genetic loss-of-function experiments in a wide range of experimental models (including yeast, worms and flies) carrying deletions in genes related to autophagy11. Similar findings have been confirmed in mammals. For example, genetic ablation of autophagy in the liver, adipose tissue, skeletal muscle or hypothalamic proopiomelanocortin neurons of mice abolishes the system-wide beneficial effects of intermittent fasting, suggesting a conserved role of autophagy for healthy aging in mammals12. The uptake of dietary patterns that activate autophagy is complicated by the need for low-calorie intake over an extended period before autophagy is efficiently induced. In addition, the efficacy with which fasting activates autophagy shows a large variability across individuals13. To circumvent these challenges, small molecules could be used to mimic the benefits of fasting and caloric restriction while following a diet with normal or only slightly reduced caloric intake14. Such a strategy should ideally be based on food-grade ingredients that activate autophagy and can be used in a dietary context for preventive health before drugs are medically indicated. For example, spermidine, urolithin A and resveratrol have been shown to provide broad health benefits across organ systems, with a remarkable level of conservation across species15,16,17,18,19. However, the number of natural bioactive compounds that can be safely used at efficacious levels in food and beverage formats remains small, limiting the potential to promote autophagy as part of diets that can be widely adopted for disease prevention. In this study, we developed and deployed a rapid testing approach in zebrafish as a suitable whole-organism model for small-molecule discovery, with the aim of finding food-grade autophagy activators20. Zebrafish have been used to measure autophagy in vivo, providing a vertebrate model that recapitulates organismal complexity21,22. In this study, we established an in vivo workflow for the discovery of autophagy modulators using a transgenic construct in which ZsGreen is tagged to the autophagosome-localized protein LC3 (encoded by map1lc3), combined with high-content image acquisition and analysis in live zebrafish. Mechanistically, thymol transiently lowers the mitochondrial membrane potential (MMP), triggering a cellular stress response that promotes the recycling of damaged mitochondria. Thymol supplementation reduces hepatic fat content in a mouse model of diet-induced obesity, delays age-related phenotypes in worms, and lowers epigenetic age in conjunction with improved muscle performance in a naturally occurring mouse strain affected by premature mobility decline. The change from a dispersed cytosolic presence of LC3-I to the formation of vesicular structures rich in LC3-II can be exploited in confocal microscopy to visualize autophagosomes. We developed a zebrafish model in which we tagged the fluorescent reporter protein ZsGreen in frame to the 5′ end of map1lc3. To drive the expression of the reporter gene in skeletal muscle, we used the skeletal muscle-specific promoter of actc1b and generated Tg(actc1b:ZsGreen-map1lc3;cryaa:TdTomato) zebrafish, hereafter named actc1b:ZsGreen-LC3 (Fig. Next, we set up an image acquisition and analysis pipeline that detects LC3-positive puncta for in vivo quantification of autophagosomes (Fig. Profiling of zebrafish larvae showed that the number of autophagosomes increased from 3 to 6 days post fertilization (dpf) (Fig. Cotreatment with the lysosomotropic agent ammonium chloride (NH4Cl), an inhibitor of autophagosome degradation, increased the accumulation of LC3 puncta compared to baseline from 4 dpf, demonstrating a bona fide increase in autophagic flux (Fig. This result is consistent with autophagy activation during the transition of zebrafish larvae from the ‘yolk-feeding' stage at 3 dpf to a state of limited energy substrate availability due to yolk consumption at 6 dpf (ref. To enable the discovery of previously unidentified autophagy modulators in vivo, we implemented a semiautomated multistep protocol as summarized in Fig. We opted for an early time window, with analysis at 3 dpf when endogenous autophagic flux is low. Specifically, we distributed larvae at 2 dpf into 96-well microplates. After incubation with the compound of interest, the larvae were immobilized through treatment with an anesthetic before image acquisition as described above. Following the first acquisition, the anesthetic was washed out and the larvae were treated with NH4Cl before a second round of imaging was performed. a, Schematic of the transgenic Tg(actc1b:ZsGreen-map1lc3;cryaa:TdTomato) zebrafish line for live monitoring of autophagy. b, Representative fluorescence image of transgenic ZsGreen-LC3 zebrafish larvae at 3 dpf, acquired at 4× (top) and 20× (bottom) magnifications. c, Detection of LC3-positive vesicles through semiautomated high-content imaging in transgenic larvae at 3 dpf (top) and 6 dpf (bottom). The resulting objects were identified and counted as described in Extended Data Fig. e, Workflow for rapid testing of small molecules in 96-well microplates to identify modulators of autophagy. f, Representative images of transgenic zebrafish larvae at 3 dpf treated for 16 h with 1 µM rapamycin (top), followed by treatment with NH4Cl (bottom). g, Quantification of autophagic flux in zebrafish larvae treated with rapamycin (3 dpf; vehicle −/+ NH4Cl, n = 68; rapamycin − NH4Cl, n = 67; rapamycin + NH4Cl, n = 71) and normalized to control (vehicle, 1% DMSO). Violin plots in d and g report the median (solid lines) and quartiles (dotted lines). Data were pooled from three independent experiments and analyzed by two-way ANOVA followed by Sidak multiple comparisons. Scale bars, 100 µm (b top) and 20 µm (b bottom, c and f). Panels a and e created with BioRender.com. For validation, larvae were treated with rapamycin, an mTOR inhibitor with well-established effects on autophagy induction in mammals. mTOR activity was abolished after 20 h of treatment of wild-type larvae, as demonstrated by S6 protein phosphorylation (Extended Data Fig. Accordingly, treatment of actc1b:ZsGreen-LC3 larvae for 16 h enhanced autophagic flux in the absence of NH4Cl, and the same effect was observed following 4 h of incubation with NH4Cl (Fig. Next, we tested the evolutionary conservation of autophagy modulation in zebrafish by performing a pilot screen using a commercial collection of 240 synthetic and natural pharmacophores. Among the seven hits that were identified, three small molecules—ceramide, calpeptin and retinoic acid—have been previously reported to induce autophagy in mammalian cells (Extended Data Fig. These results validate zebrafish as a model for quantifying autophagy levels in vivo, while the semiautomatic analysis pipeline provides a rapid workflow for the discovery of autophagy inducers. To enrich a screening library for molecules with a favorable safety profile for human translation, we next curated a list of natural bioactive compounds from edible sources and endogenous metabolites, including vitamins, bioactive molecules from herbs and spices, and human serum metabolites (Supplementary Table 2). The molecules were screened at concentrations from 10 to 250 µM and were ranked according to their maximum efficacy (Emax) in the presence of NH4Cl (Fig. The analysis included high-content imaging of LC3-positive puncta in more than 2,500 larvae in the presence or absence of NH4Cl. Carvacrol and thymol, terpenoids abundant in essential oils derived from common thyme (Thymus vulgaris) and oregano (Origanum vulgare), were the most potent autophagy activators among all the screened bioactive ingredients (Fig. Both molecules showed a dose–response relationship, with substantial effects on autophagy starting from 50 µM for carvacrol and 10 µM for thymol (Extended Data Fig. Thymol and carvacrol constitute up to 60% of all phytochemicals in the oil fraction of oregano and thyme31. Therefore, we hypothesized that oregano essential oil, a natural product extracted from dry oregano leaves and containing the monoterpenoids carvacrol, p-cymene and thymol (Extended Data Table 1), exhibits bioactivity related to autophagy. Consistent with this, we observed that treatment with oregano essential oil induced a robust increase in autophagic flux (Fig. Notably, thymol, carvacrol and oregano essential oil demonstrated similar efficacy to rapamycin at the doses tested, making them promising candidates for exploring autophagy-dependent health benefits. Considering the structural similarity and comparable efficacy of thymol and carvacrol, we chose thymol as the lead ingredient for subsequent experiments, based on the availability of pharmacokinetic studies in humans32. 2d,e), indicating that the conjugation of thymol does not interfere with its bioactivity. Thymol did not affect S6 and AMP-activated protein kinase (AMPK) phosphorylation, suggesting that autophagy is not directly activated through these canonical regulatory pathways (Extended Data Fig. a, Heat map of autophagy induction in response to bioactives tested at four ascending doses with and without treatment with NH4Cl in zebrafish larvae at 3 dpf. The colors represent the relative area of LC3 puncta normalized to the vehicle control, ranging from low (blue) to high (red) activation. White crossed cells represent data excluded due to treatment toxicity (3 dpf; n = 4–21 per condition). b, Representative images of ZsGreen-LC3 zebrafish larvae treated for 16 h with the indicated molecules, either alone or in combination with NH4Cl. Data were pooled from three independent experiments and analyzed by two-way ANOVA followed by Sidak multiple comparisons. In summary, we identified thymol and carvacrol as potent autophagy inducers that are naturally found in oregano and thyme. We next aimed to determine how thymol induces autophagy and whether the mode of action is conserved across species. Jurkat cells, a human cell line that grows in suspension, can be used to measure autophagic flux through flow cytometry. Based on previous reports that thymol intercalates into lipid bilayers, we hypothesized that thymol acts at the level of the mitochondrial membrane to activate autophagy33,34,35. Thymol caused a mild decrease in the MMP, as detected by a fluorometric shift in JC-10 emission from orange, which represents the aggregate form, to green, representing the monomeric form, compared to the chemical uncoupler FCCP (Fig. Cell viability was not affected even in the presence of 250 µM thymol for up to 6 h, suggesting that autophagy is not a result of cellular toxicity (Extended Data Fig. To confirm similar bioactivity on mitochondrial membranes, we quantified the MMP in Jurkat cells treated with carvacrol and found a comparable effect to that of thymol (Extended Data Fig. Consistent with membrane depolarization, treatment with thymol and carvacrol increased mitochondrial superoxide generation, suggesting a mechanism that originates at the mitochondrial membrane (Extended Data Fig. a, LC3 signals measured by flow cytometry in Jurkat cells treated with rapamycin or thymol (blue data points: native; green data points: lysosomal (Lys) inhibitor cotreatment). Each data point represents the mean of a biological replicate (n = 2). Data points represent the means of biological replicates (n = 2). c, MMP in Jurkat cells treated with vehicle (0.1% DMSO), thymol (100 µM) or FCCP (100 nM), measured by flow cytometry as the ratio between the monomeric form (emission wavelength (Em) = 590 nm) and the aggregated form (Em = 525 nm) of JC-10 (n = 3 wells per condition). Ordinary one-way ANOVA followed by Sidak multiple comparisons. d, ADP-stimulated oxygen consumption rate (OCR) of mitochondria isolated from livers treated with thymol (green solid line). Values were normalized to the vehicle (dashed blue line) and are presented as means ± s.e.m. e, Microscopy images of MAFs treated with vehicle (0.1% DMSO) or thymol (100 µM) in the presence of MitoTracker (green) and TMRM (red). g, Gene set enrichment analysis in MAFs ranked by level of significance. Dotted lines indicate the significance threshold. h, Twenty-four significantly enriched pathways related to autophagy and cellular energy metabolism. Experiments (e–h) were conducted on MAFs with a 24-h exposure to vehicle (0.1% DMSO) or thymol (100 µM) (control, n = 6; thymol, n = 6). j, OCR in zebrafish continuously treated with thymol compared to the washout and control groups (vehicle, 1% DMSO). Rotenone and antimycin A (R + AA) were used to determine nonmitochondrial respiration (3 dpf; control, n = 18; thymol, n = 21; washout, n = 21 pools of larvae). k, Basal and maximal OCR calculated as the average of the last three points before the addition of FCCP and R + AA, respectively (3 dpf; control, n = 18; thymol, n = 20; washout, n = 21 pools of larvae). Data were pooled from three independent experiments. Ordinary one-way ANOVA followed by Sidak multiple comparisons. l, Survival analysis of zebrafish larvae treated with thymol or vehicle and exposed to mitochondrial stress through rotenone treatment. Data were pooled from two independent experiments (3 dpf; n = 48 per group). Pairwise comparisons to control using a log-rank test. P values were adjusted using Bonferroni correction. Flow cytometry gating schemes are shown in Supplementary Fig. Changes in the MMP can affect the respiratory chain and modulate mitochondrial metabolism. Therefore, we measured the oxygen consumption rate (OCR) in mitochondria purified from mouse livers that were treated ex vivo with a range of thymol doses. Consistent with our previous observation that thymol affects the MMP, we found a dose-dependent effect on mitochondrial respiration with a half-maximal inhibitory concentration (IC50) of 295.7 ± 6.7 µM in healthy mitochondria (Fig. Based on these results, we validated the effects through a different method for quantifying the MMP, in which we used primary mouse adult fibroblasts (MAFs) from the same animal model used for OCR quantification. MAFs were treated with thymol for 24 h and then exposed to the fluorescent dye TMRM, a sensitive marker for MMP changes. As expected, TMRM fluorescence was reduced in thymol-treated cells (Fig. 3e), leading to a significant decrease in the TMRM signal compared to that of the control (Student's t test, P < 0.001) (Fig. Carvacrol had similar effects on the MMP (Extended Data Fig. We next asked whether thymol treatment leads to transcriptional adaptations consistent with changes in cellular energy homeostasis. QuantSeq 3′ mRNA sequencing revealed 273 differentially expressed genes in thymol-treated cells compared to controls (Extended Data Fig. Unbiased gene set analysis of cellular hallmarks revealed an anti-inflammatory gene expression signature, with three of the top seven pathways indicating downregulated inflammatory signaling, including tumor necrosis factor (TNF) signaling via nuclear factor-κB (NF-κB) and IL-6–JAK–STAT3 signaling (Fig. Notably, a gene signature consistent with the activation of an unfolded protein response was upregulated, indicative of a cellular stress response (Extended Data Fig. Further, a targeted Gene Ontology analysis of 24 curated biological processes related to cellular energy metabolism and autophagy showed upregulation of pathways linked to macroautophagy, autophagy and protein catabolic processes. In contrast, Gene Ontology terms related to the response to starvation were downregulated (Fig. Depolarization of mitochondria is thought to induce autophagy and mitophagy, thereby initiating the targeted removal and replacement of cellular components, which results in improved resilience to energetic stressors and damage36. To be beneficial, this mechanism needs to be transient, without long-term inhibitory effects on cellular respiration, and should be moderate enough to predominantly affect damaged mitochondria. We first tested the kinetics of thymol's effects on mitochondrial respiration by measuring the OCR in zebrafish larvae. In this setup, we compared a vehicle group to a group of animals that were continuously exposed to thymol and to another group in which thymol treatment was replaced by vehicle treatment 30 min before the analysis (Fig. Consistent with the in vitro experiments in mammalian cells, thymol reduced basal respiration in larvae, with minor effects on uncoupled respiration, suggesting that thymol does not affect maximal respiratory capacity. Importantly, mitochondrial function was fully rescued within 30 min following the removal of thymol (Fig. We hypothesized that transient inhibition of mitochondrial respiration leads to autophagy, thereby removing toxic materials and improving overall cellular quality. To test whether thymol increases organismal resilience, we treated zebrafish larvae with the mitochondrial complex I inhibitor rotenone, a toxin whose effects have been previously shown to be counteracted by the activation of autophagy and mitophagy37. Pretreatment of larvae with thymol for 16 h at three different doses reduced rotenone-induced toxicity, indicating that thymol primes cells for improved resilience to mitochondrial damage (Fig. In summary, these results show that the effects of thymol on mitochondrial membrane integrity and respiratory function are transient, reversible, and confer protection at both the cellular and organismal levels. The activation of autophagy and the transient inhibition of mitochondrial respiration suggest that both mechanisms induced by thymol are functionally connected. To test whether thymol's effect of lowering of the MMP triggers the activation of mitophagy in addition to macroautophagy, we obtained MAFs derived from mito-QC reporter mice that carry a tandem mCherry-GFP tag fused to the mitochondrial targeting sequence of the outer mitochondrial membrane protein FIS1 (encoded by Fis1)38 (Extended Data Fig. Fibroblasts were derived from male mito-QC mice. Of note, all mice used in the study, including those receiving in vivo treatments, were male. When mitophagy is activated in these mice, mitochondria tagged by the reporter construct are delivered to lysosomes, where GFP fluorescence is quenched by the acidic environment of the lumen, resulting in red foci of mitochondria due to the remaining fluorescence emission of the mCherry signal. Treatment with thymol for 24 h led to a strong activation of mitophagy, measured by the appearance of red fluorescent puncta and quantified as the mitophagy index, defined as the ratio of mCherry-positive mitolysosomes to the total number of mCherry-GFP-positive mitochondria (Fig. In contrast, cotreatment with oligomycin, an inhibitor of mitochondrial ATP synthase that blocks proton efflux and prevents mitochondrial depolarization, attenuated the effect of thymol on mitophagy (Fig. Thus, thymol-induced mitochondrial depolarization is necessary to initiate the mitophagy process. Next, we turned to an in vivo mammalian model and performed oral gavage with two administrations of thymol at 20 mg kg−1 per dose using mito-QC reporter mice (Fig. Gastrocnemius muscle biopsies and liver samples were collected 2 h after the last treatment, and histological sections were imaged to assess mitophagy levels (Fig. Oral thymol treatment resulted in a striking induction of mitophagy in skeletal muscle, as demonstrated by increases in the mitophagy index and the number of mCherry-positive mitochondria (Fig. In contrast, mitophagy activation in liver samples was similar between the thymol and vehicle groups, potentially limited by the timing of sampling relative to the pharmacodynamics of thymol, or due to the mosaic expression of the transgene as reported previously (Extended Data Fig. a, Representative confocal microscopy images of mito-QC MAFs treated with vehicle (0.1% DMSO), oligomycin (1 nM), thymol (100 µM) or thymol (100 µM) + oligomycin (1 nM) for 24 h. Scale bar, 20 µm. b, Quantification of mitophagy activation as the mitophagy index, which represents the relative mitolysosomal area in mito-QC MAFs. Ordinary one-way ANOVA followed by Dunnett multiple comparisons. c, Study scheme for the acute exposure of transgenic mito-QC reporter mice to vehicle or thymol. Mice received two doses of thymol by oral gavage at a dose of 20 mg per kg body weight, or vehicle, 16 and 2 h before liver and muscle tissue collection (10 weeks old, n = 5 per group). d, Representative confocal microscopy images of gastrocnemius skeletal muscle 2 h after the second treatment. f, Number of mCherry-positive mitophagy foci. Ten fields were analyzed for each mouse, with each data point in the box plots in e and f representing the value from an individual microscopic field (control, n = 50; thymol, n = 50). Taken together, our results show that mitophagy activation by oregano terpenoids occurs both in vitro and in vivo and is triggered by a mechanism that requires mild and transient mitochondrial depolarization. Autophagy enables the degradation of triglyceride-rich lipid droplets in hepatocytes, protecting against aberrant liver fat accumulation, a common phenotype in aging that is compounded by calorie-dense but nutrient-poor Western diets40. As thymol inhibits the respiratory capacity of liver mitochondria at elevated doses (Fig. 3d), we tested its effects on hepatic macroautophagy and mitophagy by western blot analysis in wild-type mice. Total tissue lysate and mitochondria were purified from liver tissue to determine markers of autophagy and mitophagy in response to oral treatment (Fig. Thymol increased the autophagic flux, measured as the LC3-II/LC3-I protein ratio, in conjunction with hyperphosphorylation of mitochondrial ubiquitin at the Ser-65 residue, suggesting mitophagy activation through canonical PINK-1/Parkin signaling (Fig. a, Schematic overview of acute intervention with thymol in C57BL/6 mice. Mice received either thymol 20 mg kg−1 or vehicle by two gavages 16 and 2 h before they were killed, and proteins or mitochondria were isolated from liver tissues after animal killing (10 weeks old, n = 5 per group). b, Western blot analysis of total lysates and mitochondrial extracts from liver of mice treated with thymol or vehicle using the indicated antibodies. c, Densitometric quantification of the western blots. d, Schematic overview of chronic intervention study in wild-type mice. Ten-week-old mice were fed standard chow (control) or a HFD. Thymol or vehicle was administered per os to mice fed a HFD 5 days per week for 8 weeks. The control group was treated with vehicle only. e, Body weight changes throughout the intervention. f, Representative histological images of liver tissue stained with Oil Red O (ORO) and hematoxylin and eosin (H&E). g, Quantification of mean lipid droplet size from ORO staining. h, Liver triglyceride (TG) content (n = 13 per group). i, Western blot analysis of liver tissues using the indicated antibodies for the quantification of LC3-II/LC3-I and P62 protein contents. j, Densitometric quantification of the western blots (n = 15 per group). Ordinary one-way ANOVA followed by Sidak multiple comparisons. Panels a and d created with BioRender.com. Following this acute study, we set out to test whether thymol prevents the accumulation of excess liver fat in chronic conditions. Groups of mice were fed a high-fat diet (HFD) and supplemented with 20 mg kg−1 of thymol or a vehicle control by gavage for 5 days per week over 8 weeks. A group of mice was fed a standard chow diet and received vehicle treatment by gavage to serve as a reference (Fig. At the end of the treatment, body weight was higher in mice fed a HFD than in their littermates that were fed a standard diet (Fig. Thymol had no effects on weight gain (Fig. 5a,b) or food intake (Extended Data Fig. Blood glucose and insulin responses to an oral glucose tolerance test (OGTT) were similar between the thymol-treated and control HFD-fed groups (Extended Data Fig. In contrast, liver histological analysis revealed robust protection against steatosis and preserved tissue morphology in thymol-treated mice compared to the vehicle-treated controls (Fig. Oil Red O (ORO) analysis showed that the amount of lipid droplets decreased in the livers of animals treated with thymol, along with a reduction in tissue triglyceride content (Fig. Consistent with an enhancement of autophagy, we observed lower levels of P62, an autophagy cargo adaptor and substrate that becomes depleted in response to increased autophagic flux10 (Fig. No differences between groups were observed for the LC3-II/LC3-I ratio, suggesting that autophagic flux stabilizes over the treatment period and leads to the degradation of autophagosomes and their cargo, including P62, without further changes in the LC3-II/LC3-I ratio. Taken together, oral thymol intake acutely activates liver autophagy and mitophagy, preventing excess liver fat accumulation during a chronic HFD challenge, independent of changes in body weight. Caenorhabditis elegans is a well-suited model organism for studying epistatic mechanisms that mediate the effects of small-molecule interventions on healthspan42. We first aimed to confirm an evolutionarily conserved effect of thymol and carvacrol on autophagy induction. We treated wild-type worms that carry a transgene in which GFP is fused to the C. elegans ortholog of MAP1LC3, leading to the expression of a GFP::LGG-1 reporter. Worms were exposed to thymol or carvacrol at different life stages and for varying durations of exposure. Thymol and carvacrol increased GFP::LGG-1 puncta in young (day 1) and adult worms in response to lifelong treatment (Fig. The same effect was observed in aged animals (10 days old) treated throughout their lives or in response to a subchronic duration of 72 h (Fig. Notably, autophagy levels in aged C. elegans treated with the terpenoids were robust and comparable to those observed in young worms, despite lower baseline autophagy levels (Fig. Based on the ability to induce autophagy in C. elegans, we used a transgenic line carrying a mitochondria-targeted Rosella (mtRosella) reporter in the muscle cells of the body wall. This dual-fluorescent reporter coexpresses pH-sensitive GFP (mtGFP) and pH-insensitive DsRed. In this setup, we tested the ability of thymol to activate mitophagy in wild-type worms and pink-1 mutants lacking the membrane-bound PINK-1 protein, which initiates canonical mitophagy upon membrane depolarization43. Thymol robustly increased mitophagy signals in both young and aged worms (Fig. The reversal of age-related autophagy decline was absent in worms lacking functional PINK-1 signaling, demonstrating that thymol signals through canonical mitophagy activation (Fig. Ordinary one-way ANOVA followed by Sidak multiple comparisons. b, Epifluorescence microscopy images of C. elegans expressing an mtRosella biosensor in body wall muscle cells to visualize mitophagy in response to control (0.1% DMSO) and thymol (25 µM) treatment in young (day 1) and old (day 10) wild-type and pink-1 (RNAi)-treated animals. Two-way ANOVA followed by Sidak multiple comparisons. e,f, Activity score (arbitrary units) in response to 4 h of thermal shock at 37 °C. Before heat exposure, worms were treated with thymol (vehicle, n = 2,319; thymol, n = 1,728) (e) or carvacrol (vehicle, n = 2,266; carvacrol, n = 1,980) (f) for 72 h at 50 μM from days 1 to 3 of adulthood. Ordinary one-way ANOVA followed by Dunnett multiple comparisons. In e–l, data were pooled from three independent experiments and are presented as means ± s.e.m. m, Percentage of damaged myofibrils in 10-day-old C. elegans in response to lifelong treatment with thymol. Ordinary one-way ANOVA followed by Sidak multiple comparisons. Data were pooled from two independent experiments, with 200 muscle fibers analyzed for each condition. Exposing cells to nonphysiological temperatures induces the expression of heat shock proteins, a group of chaperones that protect cellular components by forming protein aggregates44. We hypothesized that thymol increases resilience to heat stress through the turnover of protein aggregates and damaged mitochondria, thereby reinstating proteostasis that is perturbed by high temperatures. Exposure of C. elegans to a temperature of 37 °C for 4 h sharply reduced their activity score, a measure that assesses animal motility as a readout of recovery from the heat exposure (Fig. Thymol or carvacrol treatments 72 h before the heat shock robustly improved thermotolerance, measured as the percentage of active animals during a time course of 32 h after the thermal shock (Fig. We repeated the same experiment in worms with genetic deletions of daf-16, which is critical for autophagy signaling, and pink-1, the gatekeeper of canonical mitophagy signaling, to test dependency on these pathways45. Thymol did not protect worms deficient in daf-16 and pink-1, indicating that these genes are required for the benefits of thymol (Fig. C. elegans deficient in aak-2, the ortholog of genes encoding proteins of the AMPK complex in vertebrates, also interfered with thymol-dependent motility benefits, consistent with the upstream regulatory role of AMPK in the induction of autophagy and mitophagy46 (Fig. In contrast, the effects of thymol did not depend on skn-1, which encodes a transcription factor with conserved functions in oxidative stress responses to NRF2 in mammals47, indicating that the protective effects do not rely on reactive oxygen species signaling (Fig. Next, we assessed motility as a surrogate for improved healthspan during lifelong treatments in microfluidic chambers. Activity scores improved in worms exposed to thymol at concentrations of 25 and 50 µM from 7 to 10 days of adulthood (Fig. Similar benefits regarding age-related motility were also observed when using conventional microplates, with thymol and carvacrol at concentrations of 25 and 50 µM consistently enhancing velocity and total traveled distance following treatments from the egg stage to 10 days of adulthood (Extended Data Fig. Lastly, we asked whether the improved motility outcomes were also reflected in preserved anatomical structures. To this end, we used a strain that expresses MYO-3 tagged with GFP in body wall muscles and quantified the damage to myofibrils, which consist of the contractile proteins required for efficient movement. During aging, myofibers exhibit severe damage and decreased MYO-3 expression48. Thymol and carvacrol treatments preserved myofiber structure at 25 and 50 µM, while higher doses did not reduce damage (Fig. 6f), suggesting that higher doses may lack benefits under chronic exposure. Moreover, both thymol and carvacrol increased the MYO-3::GFP fluorescence intensity at all tested doses, indicating healthier myofibrils (Extended Data Fig. We did not observe an effect on overall lifespan, potentially due to antagonistic effects under chronic exposure, among other reasons (Extended Data Fig. In summary, thymol improves age-related motility and slows skeletal muscle decline by enhancing cellular resilience to aging and stress through the activation of autophagy and mitophagy. The effects of thymol on autophagy across species and tissue types suggest broad benefits that collectively support healthy aging. To this end, we aimed to translate these findings to a mammalian model of progressive aging by studying SAMP8 (senescence-accelerated mouse prone 8) mice. This model is particularly suited for studying the aging of the brain and physical performance, two domains that have a major role in age-related disabilities in humans. SAMP8 mice are an inbred strain selected from a genetically heterogeneous population of mice. Specifically, these mice naturally develop early signs of aging, such as reduced locomotor performance, sarcopenia, and cognitive decline linked to an accumulation of protein aggregates and a loss of mitochondrial respiratory function. Therefore, they are considered a relevant model for human aging49,50. We administered thymol at a dose of 20 mg kg−1 per day or vehicle by oral gavage for 5 days per week over 12 weeks starting at 8 months of age (Fig. First, we analyzed DNA methylation patterns that predict age-adjusted health status. These methylation-based epigenetic clocks are emerging as relevant biomarkers of aging in mammals and are used to detect the beneficial effects of interventions designed to extend healthspan51. Notably, the skeletal muscle tissue of mice supplemented with thymol showed a reduction in epigenetic age, as assessed using a pan-tissue epigenetic clock52, of approximately 20% compared to the control group (Student's t test, P = 0.033) (Fig. a, Schematic overview of chronic interventions with thymol in SAMP8 mice (n = 15 per group). b, Biological age measured in skeletal muscle through the quantification of epigenetic clock-specific methylation sites, expressed as biological age minus chronological age (BioAge − age) in SAMP8 mice (control, n = 10; thymol, n = 13). c, Representative immunofluorescence images of the tibialis anterior muscle stained with laminin. d, Frequency distribution of skeletal muscle fibers according to their cross-sectional area. e, Mean cross-sectional area (CSA) of tibialis muscle fibers. f, Treadmill analysis at baseline and after 12 weeks of treatment (control, n = 11 mice; thymol, n = 13 mice). g, Treadmill analysis results expressed as the ratio of running distance at the end of the study to that at baseline (control, n = 11; thymol, n = 13). h, Grip strength measured at baseline and after 12 weeks of treatment (control, n = 12 mice; thymol, n = 13 mice). i, Grip strength expressed as the ratio of force between the end of the study and baseline (control, n = 12 mice; thymol, n = 13 mice). j, Gene set enrichment barcode plot for the TNF signaling via NF-κB gene set from the 3′ QuantSeq analysis of the gastrocnemius. k, Western blot analysis of skeletal muscle from mice treated with thymol or vehicle using the indicated antibodies. l, Densitometric quantification of the western blots. Each data point represents the densitometric value obtained from individual muscle lysates (control, n = 9; thymol, n = 10). Data points in the before–after plots in f and h represent values from each mouse at baseline and the end of the study (12 weeks). Similar to findings in mice fed a HFD, thymol treatment did not affect body weight (Extended Data Fig. Histological analysis of the tibialis anterior muscle showed larger myofibers in mice treated with thymol compared to controls (Fig. The frequency distribution of the muscle fiber area further showed a shift toward larger myofibers compared to untreated mice (Fig. 7d), resulting in an approximately 13% increase in the cross-sectional fiber area (Student's t test, P < 0.001) (Fig. Endurance was strikingly higher in the thymol-treated group than in the control group (Fig. Grip strength was preserved throughout the study in the treatment group, showing a robust preventive effect against the decline of muscle strength compared to control animals (Fig. Numerically, strength decreased by 36% from baseline in the vehicle-treated controls but remained stable from baseline in the thymol-treated group (Fig. Furthermore, motor coordination and working memory capacity, measured by the rotarod test and the continuous Y maze task, improved in the group of mice treated with thymol (Extended Data Fig. Despite the remarkable phenotypic improvements, the levels of mitochondrial respiratory chain proteins and protein markers of the mitochondrial unfolded protein response remained unchanged, suggesting a relatively selective effect on mitophagy independent of major changes in the total mitochondrial mass (Extended Data Fig. Gene expression changes were moderate, with only one gene set related to anti-inflammatory responses showing significant enrichment (Fig. Of note, the gene set—hallmark TNF signaling via NF-κB—was similar to that found in MAFs in response to thymol treatment (Extended Data Fig. Western blot analysis of gastrocnemius tissue biopsies showed an increased amount of PINK-1 protein, indicating robust activation of mitophagy through canonical PINK-1 signaling (Fig. Together, these data indicate that thymol treatment delays the age-related decline in muscle performance and lowers epigenetic age in SAMP8 mice. Autophagy has emerged as a principal mechanism that maintains cellular functions during aging53. Here, we describe thymol and carvacrol, two isomeric phenolic monoterpenes derived from thyme and oregano, as inducers of autophagy and mitophagy both in vitro and in vivo, with conserved effects across species. Thymol and carvacrol are hydrophobic molecules with bactericidal properties due to their ability to intercalate into bacterial membranes, and they are thought to provide antipathogenic defenses33,34,35. Mitochondria evolved from ancestral proteobacteria through an endosymbiotic process, giving rise to the bioenergetic properties of aerobic eukaryotes54. Autophagy induction by thymol in our eukaryotic models is consistent with an effect on mitochondrial membranes that resembles bacterial membrane disruption. Molecules that intercalate into mitochondrial membranes, such as the mitochondria-targeted antioxidant MitoQ, have been shown to depolarize mitochondria and trigger autophagy36. This depolarization stimulates the PINK-1-dependent recruitment of Parkin, which in turn mediates the elimination of mitochondria in lysosomes through mitophagy43. In support of this hypothesis, we show that thymol lowers oxygen consumption in live zebrafish and impairs oxygen flux in preparations of freshly isolated mitochondria from mouse hepatocytes. Mechanistically, the induction of mitophagy by thymol depends on PINK-1, while oligomycin—a chemical that inhibits proton flux across the mitochondrial membrane and thereby counteracts membrane depolarization—blunts thymol-induced mitophagy. These results provide robust evidence for a mitochondrial membrane-originating mechanism through which thymol activates autophagy by depolarizing mitochondria, thereby enhancing mitochondrial clearance both in vitro and in vivo. Mild, transient mitochondrial perturbations activate cellular stress response mechanisms, including autophagy and mitophagy, leading to long-term benefits through a process known as mitohormesis55,56. Previous reports show that not all damaged mitochondria can be efficiently cleared by mitophagy unless a certain threshold of damage is crossed and autophagy is activated by independent mechanisms57,58. We propose that thymol has a favorable pharmacokinetic profile for achieving mitohormetic effects in vivo. Previous pharmacokinetic studies in animals and humans show that thymol has a rapid Tmax of 0.5 and 1.97 h, respectively, followed by relatively fast clearance, with a half-life of 10.2 h in humans32,59. This prompt clearance of circulating thymol suggests that cells are only transiently exposed to its effects. As a result, mitophagy is primarily activated in damaged mitochondria, while autophagy is not activated in healthy mitochondria with normal electrochemical coupling. Rapid absorption in the duodenum further suggests that oral intake leads to transient peaks in thymol and its conjugates, reaching levels sufficient to activate autophagy in tissues before they are cleared60. While the exact pharmacokinetic profile and pharmacodynamic mechanism of action of thymol on mitochondrial turnover remain to be established, its common use in traditional medicine supports its safety in humans61. In metabolic dysfunction-associated fatty liver disease, autophagy is known to hydrolyze triacylglycerol from lipid droplets40. Mitophagy concomitantly increases the turnover of defective mitochondria, thereby facilitating fatty acid oxidation and efficient metabolic coupling, as well as protecting hepatocytes against aberrant liver fat accumulation62. Our findings show that chronic administration of thymol to mice fed a HFD resulted in a reduction in liver lipid deposition and lower levels of P62 protein. Thymol did not affect body weight, which aligns with the potential for caloric restriction mimetic effects independent of weight loss. Autophagy has been reported to promote thermotolerance due to its role in the removal of protein aggregates that form when cellular temperatures rise to toxic levels63. Thymol efficiently preserved activity in response to heat shock, suggesting that autophagy is the underlying protective mechanism. Finally, we showed that thymol-treated worms had increased motility with age, accompanied by a lower degree of myofiber damage, suggesting protection against the age-related decline in body wall muscle function and thereby preserving mobility. Based on these results, we turned to SAMP8 mice, which are characterized by the occurrence of protein aggregates in the brain and skeletal muscle, among other tissues50. Thymol demonstrated a remarkable restoration of motor performance, including improvements in strength and endurance. The functional benefits were accompanied by a lower epigenetic age determined using a pan-tissue clock readout, suggesting a potent antiaging effect at the tissue level and its relevance for translation. A limitation of our study is the use of male mice across the different in vivo experiments, a choice made due to the availability constraints of aged animals. Although testing the efficacy of thymol and carvacrol in females is an important next step, the consistent effects observed in zebrafish larvae (a stage before sex determination) and in the hermaphrodite C. elegans suggest a largely sex-independent mechanism. While deploying different species and experimental models is a strength of this work, the approach of using zebrafish larvae has some limitations. Our discovery paradigm is likely to miss compounds that depend on signaling pathways not yet established during early larval stages, as well as those specific to old age. Other limitations include the difficulty in deciding doses for oral efficacy in preclinical models, a problem that we mitigated by relying on the known pharmacokinetics of thymol in humans and rodents. Terpenoids and polyphenolic compounds are secondary plant metabolites enriched in diets characterized by a high contribution of fruits, vegetables, herbs and spices64. Some of these noncaloric phytonutrients are proposed to stimulate autophagy and mitophagy, potentially forming part of the overall health impact of these diets through the cumulative effects of bioactive mixtures65. Analyses of dietary habits among people living in areas with a high percentage of centenarians indicate that individuals with extraordinary healthspans consume diets rich in mitohormetic phytochemicals66. The limited knowledge of the overall intake doses of thymol from aromatic plants makes it challenging to perform a systematic analysis of its contribution to long-term benefits through diet. Nonetheless, carvacrol- and thymol-containing plants have been used in traditional medicine for their anti-inflammatory, antioxidant and bactericidal properties, although their mechanism of action has remained largely elusive67,68. Secondary plant metabolites have numerous biological functions, including symbiotic advantages such as producing colors and flavors, conferring resistance to stressors (such as drought, heat and radiation) and possessing antipathogenic properties, among other functions that may explain their bioactivity in humans. In fact, a surprising number of plant bioactives have inhibitory or modulatory effects on cellular bioenergetic functions in eukaryotic cells. For example, biguanides from the French lilac (Galega officinalis) inhibit mitochondrial respiratory complex I69,70 and have served as the lead structures for the antidiabetic medication metformin. Here, we demonstrate that thymol activates autophagy and mitophagy through a mechanism initiated at the level of mitochondrial bioenergetics, which may be related to the bactericidal properties of thymol as part of its original function in pathogen defense. Thus, the activation of cellular stress pathways by thymol, and potentially other secondary plant metabolites that target bacterial membranes, may reflect moonlighting functions in mitochondrial bioenergetics, autophagy and mitophagy when ingested and absorbed into mammalian cells. Beyond the induction of autophagy and mitophagy at the mitochondrial level, pleiotropic effects likely exist. For example, thymol's effects on bacteria could influence the gut microbiome, which mediates positive health outcomes from fasting and metformin treatment72,73. While our cellular experiments demonstrate a cell-autonomous effect on autophagy and mitophagy, the microbial activity of thymol may influence the composition of the host microbiome. Interestingly, oregano essential oil is used in animal feed to lower intestinal inflammation and improve immunity by modulating the composition of intestinal microbiota74,75. Transcriptional profiling in MAFs and within a systemic context in skeletal muscle biopsies from SAMP8 mice showed an enrichment of gene signatures compatible with a reduced inflammatory response. Additional studies are warranted to assess the effect of thymol on microbiota, particularly as thymol is believed to be readily absorbed in the upper gastrointestinal tract and may not reach the distal small intestine76. In summary, this study broadens the mechanistic understanding of the health benefits mediated by oregano and thyme terpenoids and shows a potential link between their functions as secondary metabolites in plants and their effects on mitochondrial bioenergetics, autophagy and mitophagy in eukaryotic cells. Thymol is present at high concentrations in common oregano and thyme species and can be enriched in oregano essential oils, making it a relatively simple and cost-effective ingredient for nutrition-based prevention. A range of medical conditions are additional avenues to explore, including neurodegenerative diseases characterized by proteotoxicity, such as Parkinson's disease and Alzheimer's disease, as well as the spectrum of inherited mitochondrial diseases. A list of antibodies and reagents used in this study is available in the Reporting Summary and Supplementary Table 7. Adult zebrafish of the AB line were raised at 28 °C under standard conditions. All procedures followed the Swiss and EU ethical guidelines and were approved by the ethics committee of the Canton of Vaud (permits VD3177 and VD3546). Transgenic Tg(actc1b:ZsGreen-map1lc3;cryaa:TdTomato) zebrafish were generated using I-SceI meganuclease-mediated insertion into AB embryos at the one-cell stage. The line is registered at the Zebrafish Model Organism Database (ZFIN) central repository under the designation Tg(actc1b:ZsGreen-map1lc3;cryaa:TdTomato)nei014 and can be found at https://zfin.org. All experiments adhered to Swiss regulations on animal experimentation and the European Community Council Directive 2010/63/EU. All procedures were approved by the Nestlé Ethical Committee (ASP-16-38-EXT), the Office Vétérinaire Cantonal Vaudois (VD2770, VD3195 and VD3484), and the Directorate General for Animal Health and Veterinary Drugs of the Italian Ministry of Health (authorization nos. Mice were housed under standard conditions with ad libitum access to food and water. C57BL/6 wild-type mice were purchased from Janvier Labs and were fed either a standard diet (D12450J, Research Diets) or a HFD (D12492, Research Diets). Mito-QC mice were kindly provided by I. Ganley, and SAMP8 AKR/J mice were obtained from Envigo. Thymol was dissolved in corn oil and administered orally. The dose was selected using allometric scaling based on previous pharmacokinetic and pharmacodynamic studies in mice and humans32,67. For the acute studies, 10-week-old male mito-QC or wild-type mice on a C57BL/6 background received thymol at a dose of 20 mg per kg body weight, followed by a second dose 14 h later. Organs from mito-QC mice were postfixed in 3.7% paraformaldehyde and 200 mM HEPES (pH 7.0) at 4 °C for 24 h. Tissues were then cryopreserved in 30% sucrose/PBS at 4 °C for 2 days, frozen in chilled isopentane on dry ice and cryosectioned at 8 µm. For chronic treatments, 10-week-old C57BL/6 or 8-month-old SAMP8 male mice received thymol at a dose of 20 mg per kg body weight per day or vehicle (corn oil). The following C. elegans strains were used in this study: wild-type N2, DA2123 (adIs2122[lgg-1p::GFP::lgg-1+rol-6(su1006)]), CF1038 (daf-16(mu86)), RB2547 (pink-1(ok3538)), TG38 (aak-2(gt33)), QV225 (skn-1(zj15)), RW1596 (stEx30[pmyo-3GFP::MYO-3 + rol-6(su1006)]) and IR2539 (unc-119(ed3); Ex[pmyo‐3TOMM‐20::Rosella; unc‐119(+)]). All strains were obtained from the Caenorhabditis Genetics Center (funded by the National Institutes of Health Office of Research Infrastructure Programs P40 OD010440), except for the strain expressing the mtRosella biosensor, which was a kind gift from the Tavernarakis laboratory in Greece. For RNAi experiments, Escherichia coli HT115 bacteria were transformed with either the pink-1 (RNAi) construct or the empty PL4440 vector. Bacteria were grown overnight in liquid cultures containing ampicillin and tetracycline, followed by subculturing in liquid medium with ampicillin. Cultures were mixed with IPTG (2 mM), seeded onto plates and dried overnight. Human lymphocytic T Jurkat cells (clone E6.1, ATCC, TIB-152) were used to measure autophagic flux in vitro. Cells were grown in complete RPMI (Thermo Fisher Scientific, 11875093) under standard conditions. For the experiment, cells were washed, counted and incubated at 1 × 105 cells per well in RPMI plus 10% human AB serum (GemCell, GeminiBio, 100-512) in duplicate in a 96-well plate with either rapamycin (Sigma-Aldrich, C2920), thymol (Sigma-Aldrich, 16254), carvacrol (Sigma-Aldrich, W224502), FCCP (Sigma-Aldrich, C2920), or 0.5% DMSO (Sigma-Aldrich, D8418) as the vehicle. Primary MAFs were obtained from C57BL/6J or mito-QC mice. Biopsy tissues were cut into small pieces with a scalpel blade and distributed on coated dishes with 1.5 ml DMEM high glucose (Thermo Fisher Scientific, 10569010) supplemented with 20% FBS (Thermo Fisher Scientific, 100-18B). Thymol (100 μM), carvacrol (100 µM), valinomycin (1 µM) (Sigma-Aldrich, V0627) and oligomycin (1 nM) (Sigma-Aldrich, O4876) were each dissolved in DMSO, added to the cell medium, and left for 24 h. Three transgenic embryos per well were placed in 96-well plates and treated with compounds diluted in DMSO. After 16 h, the embryos were anesthetized with 0.016% tricaine (Sigma-Aldrich, E10521) and imaged using the ImageXpress Micro Confocal High-Content Imaging System (Molecular Devices). Tricaine was then washed out, and the embryos were treated with 100 mM NH4Cl (Sigma-Aldrich, 09718) for 4 h before being imaged again. For automated imaging, larval images were initially acquired using a 4× objective and analyzed with MetaXpress software (version 6.7.2). A custom Python code has been developed for the automated detection of larvae. Twenty-six Z stacks of each area are acquired using a 20× objective. For the identification and quantification of ZsGreen-LC3 puncta, two-dimensional projections were analyzed. Images were processed as follows: individual Z stacks were projected onto single images using maximal projection. The projection was subsequently smoothed using serial operations, and this smoothed image was then subtracted from the maximal projection to remove the fish background from the puncta signal, resulting in a subtracted image. The smoothed image was then binarized to create the pre-fish body mask, which was cleaned using area filtering. Finally, an LC3 puncta mask was created by keeping the pre-puncta objects encompassed within the fish body mask. Morphological measurements were recorded from the two final masks. The LC3 punctate total area relative to body surface area was used as a parameter for quantification. Python and MetaXpress codes have been deposited on GitHub and are accessible at https://github.com/giulializzo/zebrafish_high_content_imaging.git. The OCR in zebrafish larvae was measured using the Seahorse XF24 instrument (Seahorse Bioscience). 101174-100, Seahorse Bioscience), maintained with an islet capture screen and overlaid with 600 µl of egg water without air bubbles. Larvae were allowed to equilibrate for 10 min and maintained at 28.5 °C throughout the entire experiment. Respiration rates were determined every 5 min. Basal respiration was calculated as the average of the last three measurement points (from 40 to 52 min) before the injection of FCCP to a final concentration of 0.8 µM. Uncoupled respiration was determined by the maximum value observed after FCCP injection. Larvae that did not respond to FCCP and exhibited OCR values lower than basal respiration were excluded from the analysis. Rotenone (Sigma-Aldrich, R8875) and antimycin A (Sigma-Aldrich, A8764) were injected at concentrations of 1 µM and 1 µg l−1, respectively, to inhibit the electron transport chain, with residual OCR accounting for nonmitochondrial respiration. Total proteins were extracted from mouse tissues or from pools of 20 zebrafish larvae at 3 dpf, as described in the Supplementary Methods. For SDS–PAGE, lysates were diluted, mixed with NuPAGE LDS sample buffer and a sample reducing agent (Thermo Fisher Scientific, NP0008 and NP0009), and heated at 70 °C for 10 min in heating blocks. Proteins (40 µg) were loaded onto precast NuPAGE Bis–Tris 1.0 mm Midi Protein Gels (Thermo Fisher Scientific, WG1402 and WBT01020), and electrophoresis was performed for 90 min at 130 V. Gels were incubated in 20% ethanol for 10 min and transferred onto PVDF membranes using the iBlot2 instrument (Thermo Fisher Scientific). Membranes were rehydrated in TBS–Tween (Thermo Fisher Scientific, 28360) and blocked with 5% skim milk in TBS–Tween for 1 h at room temperature. Anti-rabbit or anti-mouse IgG H&L (HRP) secondary antibodies (1:10,000, Thermo Fisher Scientific, 31460 and 31430) were incubated for 1 h at room temperature and detected using an enhanced chemiluminescence reagent (Thermo Fisher Scientific, 32106). To measure autophagy in Jurkat cells, we used the Guava LC3 antibody-based assay kit (Luminex, FCCH100171), following the manufacturer's instructions. The kit includes reagent A (an inhibitor of lysosomal activity), reagent B (a permeabilization solution) and an anti-LC3 antibody conjugated with FITC (clone 4E-12). Half of the replicates were incubated with reagent A for 30 min before the end of the treatment to block lysosomal degradation of LC3 vesicles, thereby measuring autophagic flux. Cells were then transferred to a 96-well V-bottom plate, permeabilized with reagent B to remove the cytoplasmic form of LC3 (LC3-I) and incubated with the FITC-conjugated LC3 antibody. Samples were then analyzed using a Becton Dickinson LSR Fortessa SORP analyzer. Offline analyses were performed using FCS Express Software (De Novo Software), with results expressed as the median fluorescence intensity in the FITC channel. MMP and mitochondrial superoxide measurements in Jurkat cells were conducted using flow cytometry. For the MMP, JC-10 was used according to the manufacturer's instructions (Enzo Life Sciences, ENZ-52305). Cells were treated with thymol, carvacrol or FCCP for 30 min, and JC-10 staining was added 15 min before analysis. For mitochondrial superoxide measurements, Jurkat cells were incubated with MitoSOX Red (5 µM) (Thermo Fisher Scientific, M36008) for 20 min before acquisition. Samples were analyzed using a Becton Dickinson LSR Fortessa SORP flow cytometry analyzer. In MAFs, the MMP was assessed by TMRM and MitoTracker costaining. Thymol or carvacrol (100 μM) was added to the medium and left for 24 h. Untreated cells received an equal volume of DMSO. Oligomycin (1 nM) was used to prevent proton efflux and depolarization. For MMP measurement, cells were grown in a 24-well glass-bottom plate in the presence of conditioned medium for 24 h, then costained with TMRM (20 nM) (Thermo Fisher Scientific, T668) and MitoTracker Deep Red (100 nM) (Thermo Fisher Scientific, M22426). Following a 30-min incubation in the dark at 37 °C, the medium was replaced with dye-free conditioned medium. TMRM and MitoTracker signals were analyzed using a TCS SP8 confocal microscope (Leica Biosystems), as described in the Supplementary Methods. After dissection, liver tissue was rapidly cut into small pieces with scissors and placed in ice-cold medium A (0.32 M sucrose, 10 mM Tris–HCl, 1 mM EDTA; pH adjusted to 7.4). The pieces were transferred into a 5-ml Dounce homogenizer (glass–Teflon) at a ratio of 5 ml of medium A per 1 g of tissue. After four to six strokes at 600 rpm on ice, the suspension was transferred into a 15-ml Falcon tube and centrifuged at 1,000g for 5 min at 4 °C. The pellet containing mitochondria was resuspended in 1 ml of ice-cold medium A. The mitochondrial protein concentration was then determined using the BCA method (Thermo Fisher Scientific, 23225). The OCR in mitochondria isolated from mouse livers was measured using high-resolution respirometry (OROBOROS Oxygraph-2k, Oroboros Instruments). Respiration in isolated mitochondria was determined using SUIT (substrate–uncoupler–inhibitor titration) protocols, with modifications. A 2-µl volume of thymol solution (50 mM thymol in ethanol) was repeatedly injected to reach incremental concentrations of thymol in the respirometry chamber. The second (control) chamber was injected with the same volume of carrier (ethanol) alone. After a sufficient number of titration steps, respiration was halted by the addition of the complex I inhibitor rotenone (Sigma-Aldrich, R8875, 0.5 µM). Values obtained were normalized to the baseline respiration measured in the control chamber. Total RNA was extracted as described in the Supplementary Methods. Total RNA (75 ng; 8.9 < RNA quality number < 10) was used to generate the libraries using the QuantSeq 3′ mRNA-Seq Library Prep Kit FWD for Illumina (Lexogen, 015.384) following 20 cycles of PCR amplification. Libraries were quantified using the Quant-iT PicoGreen kit (Invitrogen, Q33140) on a FilterMax F3. Libraries were pooled at an equimolar ratio at a concentration of 40 nM, and sequencing was performed by loading a concentration of 650 pM with a P3 50 kit for 65 cycles on an Illumina NextSeq 2000 platform (Illumina) according to the manufacturer's instructions. Body composition was measured using nuclear magnetic resonance with the Minispec device (Bruker). OGTT was performed on mice that had been fasted for 6 h. Tail vein glucose levels were measured using a Contour glucometer (Ascensia Diabetes Care) immediately before glucose administration and at 15, 30, 60, 90 and 120 min after glucose administration (1 g glucose per kg body weight). Plasma insulin levels before glucose administration and at 15, 30 and 60 min after glucose administration during the OGTT procedure were quantified from heparinized plasma samples using an ultrasensitive ELISA kit (Crystal Chem, 90080). The Triglyceride-Glo Assay kit (Promega, J3160) was used to measure triglyceride levels in mouse liver homogenates. Homogenates were prepared from 40 mg of liver tissue in 1 ml ice-cold PBS, using a TissueLyser II (Qiagen) at a frequency of 30 oscillations per second for 2 min. For experiments on plates, C. elegans strains were cultured at 20 °C on nematode growth medium (NGM) agar plates seeded with the E. coli strain OP50. Thymol and carvacrol were mixed into the agar medium before the plates were poured. DMSO was used as a control (maximum concentration of 0.4% for 100 µM). Fresh plates were prepared weekly, and animals were transferred to freshly seeded plates daily after reaching the L4 stage. For pharmacological treatment on chips, worms were exposed to carvacrol, thymol or 1% DMSO in liquid NGM medium with heat-killed E. coli OP50 (20 mg ml−1). Compounds were dissolved in DMSO and prepared at a 2× concentration in liquid NGM. Once the animals reached the L4 stage, they were loaded into microfluidic chips (Infinity Chips, NemaLife) and maintained on a diet of heat-killed E. coli OP50 for 24 h. On day 1 of adulthood, worms were treated with the appropriate compound or 1% DMSO. Medium exchange was performed daily to maintain consistent exposure until the end of the assays. The number of autophagosomes was assessed in the DA2123 C. elegans strain that expresses GFP::LGG-1, with autophagosomes detected as GFP+ puncta. Animals were collected, washed to remove bacteria and anesthetized using metamizole at a final concentration of 20 mM. The animals were then placed on slides and observed under the 40× lens of a Leica DM5500 upright microscope. Mitophagy was measured using a strain that expresses the mtRosella biosensor in the mitochondria of body wall muscle cells. Mitophagy was calculated as the ratio of pH-insensitive DsRed to pH-sensitive GFP, as previously described77. For the healthspan assay, one 90-s video per chip was recorded for each time point. The activity score analysis involved determining the fraction of pixels (representing a worm within a bounding box) that changed over 30 s. Highly active animals that completely moved their bodies out of the bounding box were assigned a score of 1, whereas completely inactive animals received a score of 0. Animals with intermediate scores still had a portion of their bodies remaining in the bounding box after 30 s. For thermotolerance assays, L4 animals and C. elegans carrying deletions of daf-16, pink-1, aak-2 and skin-1 were loaded into microfluidic chips at 20 °C with NGM. On day 1 of adulthood, the medium was exchanged with NGM containing the appropriate treatment, and the animals were allowed to grow until day 3 of adulthood. Subsequently, the animals were exposed to a 37 °C heat shock for 4 h, followed by recovery and scoring for the number of alive animals in the chips. The mean speed and distance crawled by C. elegans on the plate were measured on day 10 of adulthood using the Movement Tracker software, as previously described78. Lifespan tests were performed at 20 °C, and the worms were monitored daily for survival. Lifespan data were recorded until all the worms had died. Lifespan metrics were calculated and analyzed using the Kaplan–Meier method. RW1596 animals were grown on NGM plates seeded with E. coli OP50 bacteria and containing either thymol, carvacrol or vehicle. The animals were maintained on these plates until day 10 of adulthood, after which they were collected, washed to remove bacteria and anesthetized with tetramisole at a final concentration of 20 mM. They were then placed on slides and observed under the 40× lens of a Leica DM5500 upright microscope. At least 200 muscle cells were counted for each condition. Muscle cells containing broken myofibrils or myofibrils with abnormal orientation were classified as damaged, while those with intact myofibrils showing normal, parallel orientation were classified as normal. Finally, the percentage of damaged myofibers relative to the total number of myofibers was calculated. DNA extraction was performed on 100 mg of muscle tissue using the Qiagen DNeasy Blood and Tissue kit, following the manufacturer's recommendations. DNA was quantified using DropSense technology (Unchained Labs). Bisulfite conversion was performed on 500 ng of DNA using the Zymo EZ DNA methylation kit, following the manufacturer's instructions. Converted DNA (200 ng) was used to proceed with the Infinium HD Methylation Assay Protocol, following the manufacturer's recommendations. DNA methylation data were generated using custom arrays (HorvathMammalMethyl40, Illumina) and scanned on an iScan reader (Illumina). Biological age was quantified based on a pan-tissue epigenetic clock algorithm52. Endurance was evaluated using a treadmill apparatus (Ugo Basile) with a gradually accelerating protocol, starting at 5 rpm for 2 min and increasing to 50 m s−1 over 2,700 s. The test was terminated by exhaustion, defined as ten falls per minute in the motivational air puff. Weakness was assessed by measuring forelimb grip strength using a grip strength meter (Ugo Basile). Mice grasped the grid with their forelimbs and were gently pulled away by holding the base of their tails. Measurements were recorded in triplicate, and the average was used for the analysis. The Y maze task was used to measure spontaneous alternation behavior. A rotarod apparatus (Ugo Basile) was used to assess coordination skills. After two acclimation sessions, the apparatus was set to a starting velocity of 4 rpm and an acceleration rate of 20 rpm per minute. Mice underwent three trial sessions at least 20 min apart, and the latency to fall was recorded for each mouse. For histochemical analysis, tissues were frozen in liquid nitrogen-precooled isopentane. The tibialis anterior muscle was cut into 12-μm sections at −25 °C, and the sections were thaw-mounted on slides. The slides were dried at room temperature and stained using a monoclonal anti-laminin antibody (Sigma-Aldrich, L9393) diluted 1:100 and a goat anti-rabbit IgG H&L (Alexa Fluor 405) secondary antibody (Thermo Fisher Scientific, A48254) diluted 1:500 for detection. Livers were cryodissected at 8 µm and fixed overnight in 4% paraformaldehyde. The ORO stock stain was prepared by dissolving 0.5 g of ORO (Sigma-Aldrich, O0625) in 100 ml of isopropanol. The working solution was prepared by diluting 30 ml of the stock with 20 ml of distilled water. Liver tissues were cut into 2-mm-thick slices, fixed in formalin, paraffinized and hydrated. After deparaffinization and rehydration with xylene and ethanol, the slides were stained with ORO working solution for 15 min, rinsed with 60% isopropanol and lightly stained with hematoxylin for 4 min. After rinsing with distilled water, the slides were mounted. Lipid droplets were stained red and assessed using an Olympus VS120 slide scanner. The size of lipid droplets per field was calculated based on the average size of lipid droplets in ten fields from each mouse liver slide, using the multipoint function of ImageJ. Raw counts were mapped to the mouse reference genome (GRCm38-101) using STAR (v.2.5.3) and counted using HTSeq-count (v.0.6.1). Data were checked for quality, and no outliers were detected in the principal component analysis. A filter was applied to select only genes with a minimum of 30 reads in at least six samples (corresponding to a threshold of 3.809 on the count per million values), and nonannotated genes were discarded. A total of 10,697 features were retained using these filtering criteria. To account for composition bias between libraries, we applied TMM (trimmed mean of M-value) normalization79. Differential expression analysis between groups was performed using edgeR (v.4.2.1), which estimates gene-wise negative binomial dispersions by calculating the adjusted profile log-likelihood for each gene and maximizing it using the weighted likelihood empirical Bayes method80. A quasi-likelihood negative binomial generalized log-linear model was fitted to the count data, and a gene-wise empirical Bayes quasi-likelihood F test was conducted for each comparison. Resulting P values were adjusted for multiple testing using the Benjamini–Hochberg correction method. Gene set enrichment analysis was performed using the Camera function in the edgeR package81, which conducts a competitive test to assess whether the genes in the set are highly ranked in terms of differential expression relative to genes not in the set, while accounting for intergene correlation. The Fry algorithm from edgeR was applied to assess the enrichment of a preselected list of pathways. Fry is a fast approximation of a rotation gene set test that performs self-contained gene set tests to assess whether any of the genes in the set are differentially expressed83. P values are obtained through simulation, followed by adjustment for multiple testing. For each coverslip containing mito-QC fibroblasts or tissues, ten random images were acquired using a Leica TCS SP8 confocal microscope with a 63× oil immersion objective. Images were processed with ImageJ using a custom macro, as described in the Supplementary Methods. GraphPad Prism software version 9.0 (GraphPad Software) was used to perform all statistical analyses as indicated in the main text and figure legends, except for the gene expression and pathway analyses. Box plot elements show the following data: median (center line), upper and lower quartiles (box limits), minimum and maximum values (whiskers), and points. Differences between two groups were assessed using Student's t test. Differences among more than two groups were assessed using one-way analysis of variance (ANOVA). The interaction between two factors was tested using a two-way ANOVA. Data distribution was assumed to be normal, but this was not formally tested. Survival analysis in zebrafish and worms was performed using the Kaplan–Meier method, and pairwise comparisons were conducted using the log-rank test. For high-content imaging, zebrafish larvae that were displaced during acquisition, resulting in undetectable LC3-positive puncta, were excluded from the analysis. No randomization method was used in zebrafish, cell or worm experiments. Mice were randomized using simple randomization based on body weight within the experimental groups. All phenotyping experiments in mice were conducted with the experimenters blinded to the treatment, and data analyses were performed without knowledge of the experimental conditions. Chronic mouse interventions were performed once, but they followed blinding and randomization to ensure reproducibility. For mouse studies, animals that showed weight loss greater than 20%, those found dead in their cages, and those displaying signs of infection and inflammation in their wounds were killed and excluded from the study. Histopathology, confocal imaging and image analysis were performed in a blinded manner. For all other experiments, blinding was not formally conducted. For short-term thymol supplementation studies in mice, the sample size was determined based on previous publications to detect significant changes in mitophagy or autophagy induction, as measured by the mitophagy index or the LC3-II/LC3-I ratio, respectively38,84. For the metabolic dysfunction-associated fatty liver disease model, the sample size was similar to that reported in other studies85. Based on pilot studies, a 20% decline in motor performance was expected between 8 and 11 months of age. With a study power (β) set at 0.80, an α error of 0.05, and assumptions of a halt in motor decline from the start of treatment and a 25% mortality rate, a sample size of 15 animals per group was deemed necessary. SAMP8 mice that died before the end of the experiment were excluded from subsequent analyses. For C. elegans studies, sample sizes were determined based on previous publications42,86. For all representative images showing cells, zebrafish or C. elegans, experiments were repeated a minimum of three times independently and checked to ensure the reproducibility of the results. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Unprocessed transcriptomic data based on 3′ QuantSeq analysis have been deposited in the Gene Expression Omnibus (GEO, https://www.ncbi.nlm.nih.gov/geo/) database and are accessible through the identifiers GSE298195 and GSE298196. Epigenetic data were generated through a custom array for mammalian species (HorvathMammalMethyl40, Illumina) and can be obtained from the nonprofit research organization Epigenetic Clock Development Foundation (‘Clock Foundation', https://clockfoundation.org/) upon reasonable request. For all other datasets underlying the results, no data restriction applies to the raw data generated in this study, and they can be requested from the corresponding authors. Source data are provided with this paper. For the quantification of confocal images of zebrafish larvae, an automatic workflow has been generated using MetaXpress software version 6.7.2, as illustrated in Extended Data Fig. Custom Python and MetaXpress codes have been deposited on GitHub and are accessible via the following link: https://github.com/giulializzo/zebrafish_high_content_imaging.git. No new code was generated for the quantification of gene expression and DNA methylation analyses. Nyberg, S. T. et al. Association of healthy lifestyle with years lived without major chronic diseases. GBD 2019 Diseases and Injuries Collaborators.Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Bishop, C. E. Economics of aging: new insights. de Cabo, R. & Mattson, M. P. Effects of intermittent fasting on health, aging, and disease. Stekovic, S. et al. Alternate day fasting improves physiological and molecular markers of aging in healthy, non-obese humans. Elortegui Pascual, P. et al. A meta-analysis comparing the effectiveness of alternate day fasting, the 5:2 diet, and time-restricted eating for weight loss. Madeo, F., Zimmermann, A., Maiuri, M. C. & Kroemer, G. Essential role for autophagy in life span extension. Mizushima, N. & Komatsu, M. Autophagy: renovation of cells and tissues. Hansen, M., Rubinsztein, D. C. & Walker, D. W. Autophagy as a promoter of longevity: insights from model organisms. Martinez-Lopez, N. et al. System-wide benefits of intermeal fasting by autophagy. The effect of fasting or calorie restriction on autophagy induction: a review of the literature. Madeo, F., Pietrocola, F., Eisenberg, T. & Kroemer, G. Caloric restriction mimetics: towards a molecular definition. Aman, Y. et al. Autophagy in healthy aging and disease. Mechanism and medical implications of mammalian autophagy. & Kroemer, G. Caloric restriction mimetics against age-associated disease: targets, mechanisms, and therapeutic potential. Morselli, E. et al. Autophagy mediates pharmacological lifespan extension by spermidine and resveratrol. Gut, P., Reischauer, S., Stainier, D. Y. R. & Arnaout, R. Little fish, big data: zebrafish as a model for cardiovascular and metabolic disease. Moss, J. J., Hammond, C. L. & Lane, J. D. Zebrafish as a model to study autophagy and its role in skeletal development and disease. In vivo autophagy quantification: measuring LC3 and P62 puncta in 3D image system from zebrafish larvae. He, C., Bartholomew, C. R., Zhou, W. & Klionsky, D. J. Assaying autophagic activity in transgenic GFP-Lc3 and GFP-Gabarap zebrafish embryos. Mizushima, N. The ATG conjugation systems in autophagy. Underwood, B. R. et al. Antioxidants can inhibit basal autophagy and enhance neurodegeneration in models of polyglutamine disease. Gut, P. et al. Whole-organism screening for gluconeogenesis identifies activators of fasting metabolism. Scarlatti, F. et al. Ceramide-mediated macroautophagy involves inhibition of protein kinase B and up-regulation of beclin 1. & Bossis, I. Retinoic acid induces autophagosome maturation through redistribution of the cation-independent mannose-6-phosphate receptor. Novel targets for Huntington's disease in an mTOR-independent autophagy pathway. Cohen, S. M. et al. FEMA GRAS assessment of natural flavor complexes: Origanum oil, thyme oil and related phenol derivative-containing flavoring ingredients. Systemic availability and pharmacokinetics of thymol in humans. Valero, M. & Francés, E. Synergistic bactericidal effect of carvacrol, cinnamaldehyde or thymol and refrigeration to inhibit Bacillus cereus in carrot broth. Xu, J., Zhou, F., Ji, B.-P., Pei, R.-S. & Xu, N. The antibacterial mechanism of carvacrol and thymol against Escherichia coli. McWilliams, T. G. et al. mito-QC illuminates mitophagy and mitochondrial architecture in vivo. McWilliams, T. G. & Ganley, I. G. Investigating mitophagy and mitochondrial morphology in vivo using mito-QC: a comprehensive guide. Singh, R. & Cuervo, A. M. Lipophagy: connecting autophagy and lipid metabolism. Hertweck, M., Hoppe, T. & Baumeister, R. C. elegans, a model for aging with high-throughput capacity. Jin, S. M. et al. Mitochondrial membrane potential regulates PINK1 import and proteolytic destabilization by PARL. Lee, S. S., Kennedy, S., Tolonen, A. C. & Ruvkun, G. DAF-16 target genes that control C. elegans life-span and metabolism. The Caenorhabditis elegans AMP-activated protein kinase AAK-2 is phosphorylated by LKB1 and is required for resistance to oxidative stress and for normal motility and foraging behavior. Tang, L., Dodd, W. & Choe, K. Isolation of a hypomorphic skn-1 allele that does not require a balancer for maintenance. Zhang, H. et al. Handelin extends lifespan and healthspan of Caenorhabditis elegans by reducing ROS generation and improving motor function. Takeda, T., Hosokawa, M. & Higuchi, K. Senescence-accelerated mouse (SAM): a novel murine model of accelerated senescence. Targeting multiple mitochondrial processes by a metabolic modulator prevents sarcopenia and cognitive decline in SAMP8 mice. Thompson, M. J. et al. A multi-tissue full lifespan epigenetic clock for mice. Mozhui, K. et al. Genetic loci and metabolic states associated with murine epigenetic aging. López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. & Kamikawa, R. The origin and diversification of mitochondria. Mottis, A., Herzig, S. & Auwerx, J. Mitocellular communication: shaping health and disease. de Vries, R. L. A., Gilkerson, R. W., Przedborski, S. & Schon, E. A. Mitophagy in cells with mtDNA mutations: being sick is not enough. Civiletto, G. et al. Rapamycin rescues mitochondrial myopathy via coordinated activation of autophagy and lysosomal biogenesis. Improvement of thymol properties by complexation with cyclodextrins: in vitro and in vivo studies. Targeting lipophagy as a potential therapeutic strategy for nonalcoholic fatty liver disease. Liang, V. et al. Altered proteostasis in aging and heat shock response in C. elegans revealed by analysis of the global and de novo synthesized proteome. & Goñi, I. Intake and bioaccessibility of total polyphenols in a whole diet. Martel, J. et al. Hormetic effects of phytochemicals on health and longevity. Davinelli, S., Willcox, D. C. & Scapagnini, G. Extending healthy ageing: nutrient sensitive pathway and centenarian population. Nagoor Meeran, M. F., Javed, H., Al Taee, H., Azimullah, S. & Ojha, S. K. Pharmacological properties and molecular mechanisms of thymol: prospects for its therapeutic potential and pharmaceutical development. Salehi, B. et al. Thymol, thyme, and other plant sources: health and potential uses. Structural basis of mammalian respiratory complex I inhibition by medicinal biguanides. Matsuzaki, S. & Humphries, K. M. Selective inhibition of deactivated mitochondrial complex I by biguanides. Gledhill, J. R., Montgomery, M. G., Leslie, A. G. W. & Walker, J. E. Mechanism of inhibition of bovine F1-ATPase by resveratrol and related polyphenols. Maifeld, A. et al. Fasting alters the gut microbiome reducing blood pressure and body weight in metabolic syndrome patients. Pryor, R. et al. Host–microbe–drug–nutrient screen identifies bacterial effectors of metformin therapy. Effects of dietary oregano essential oil supplementation on growth performance, intestinal antioxidative capacity, immunity, and intestinal microbiota in yellow-feathered chickens. Jia, L. et al. Oregano essential oils mediated intestinal microbiota and metabolites and improved growth performance and intestinal barrier function in sheep. Effects of thymol and thymol α-d-glucopyranoside on intestinal function and microbiota of weaned pigs. Palikaras, K., Lionaki, E. & Tavernarakis, N. Coordination of mitophagy and mitochondrial biogenesis during ageing in C. elegans. The Movement Tracker: a flexible system for automated movement analysis in invertebrate model organisms. A scaling normalization method for differential expression analysis of RNA-seq data. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Wu, D. & Smyth, G. K. Camera: a competitive gene set test accounting for inter-gene correlation. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Wu, D. et al. ROAST: rotation gene set tests for complex microarray experiments. Chrisam, M. et al. Reactivation of autophagy by spermidine ameliorates the myopathic defects of collagen VI-null mice. Recena Aydos, L. et al. Nonalcoholic fatty liver disease induced by high-fat diet in C57bl/6 models. Rahman, M. et al. NemaLife chip: a micropillar-based microfluidic culture device optimized for aging studies in crawling C. elegans. We thank N. Rochat and E. Jiminez for their assistance in the zebrafish facility, the Nestlé Institute of Health Sciences community for the discussions and support, and I. Ganley from the University of Dundee, Scotland, UK, for providing the mito-QC mice. We acknowledge support from the Fondazione Regionale per la Ricerca Biomedica (Regione Lombardia-FRRB) (Project ID 1740526 to D.B. and the Ricerca Corrente (RRC) provided to the Fondazione IRCCS Istituto Neurologico Carlo Besta. We are grateful to V. Sorrentino for the expert advice on C. elegans experiments and to the Advanced Technology Center for Aging Research, IRCCS INRCA (Ancona, Italy), for the support with mouse experiments. These authors jointly supervised this work: Jerome N. Feige, Philipp Gut. Gabriele Civiletto, Giulia Lizzo, Kamila Muller, Guillaume E. Jacot, Federico Sizzano, Minji Huh, José L. Sanchez-Garcia, Bertrand J. Bétrisey, Alix Zollinger, Patricia Lino, Christopher Neal, Anne-Laure Egesipe, Joy Richard, Myriam Chimen, Aurélie Hermant, Benjamin Brinon, Jerome N. Feige & Philipp Gut Health, Nutrition and Care, DSM-Firmenich, Kaiseraugst, Switzerland Unit of Medical Genetics and Neurogenetics, Fondazione IRCCS Istituto Neurologico Carlo Besta, Milan, Italy Dario Brunetti, Ivano Di Meo, Maria Nicol Colombo & Valeria Tiranti Department of Clinical Sciences and Community Health, Dipartimento di Eccellenza 2023–2027, University of Milan, Milan, Italy Laboratory of Integrative Systems Physiology, Interfaculty Institute of Bioengineering, École Polytechnique Fédérale de Lausanne, Lausanne, Switzerland Ioanna Daskalaki, Arwen W. Gao, Amelia Lalou & Johan Auwerx Mohammed Adnan Qureshi, Dhaval S. Patel & Siva A. Vanapalli Texas Tech University, Lubbock, TX, USA Advanced Technology Center for Aging Research and Geriatric Mouse Clinic, IRCCS INRCA, Ancona, Italy Department of Clinical and Molecular Sciences (DISCLIMO), Università Politecnica delle Marche, Ancona, Italy Experimental Animal Models for Aging Unit, Scientific Technological Area, IRCCS INRCA, Falconara Marittima, Italy Epigenetic Clock Development Foundation, Los Angeles, CA, USA Altos Labs, San Diego, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar : conceptualization, methodology, investigation, validation, formal analysis, visualization, supervision, writing—original draft; D.B. : conceptualization, methodology, investigation, formal analysis, visualization, writing—review and editing; G.L. : conceptualization, interpretation, writing—review and editing, project administration, supervision. All authors read and approved the manuscript. This statement is based on the Contributor Roles Taxonomy (CRediT). Correspondence to Gabriele Civiletto, Jerome N. Feige or Philipp Gut. Patents owned by Société des Produits Nestlé SA have been filed related to this work, and the list of inventors includes authors of this article. Patent application publications can be found using the following identifiers: WO2023165870A1, WO2024099886A1 and WO2022128870A1. are employees of NemaLife Inc., which has received fees from Nestlé Research for preclinical services. is a founder and paid consultant of the nonprofit Epigenetic Clock Development Foundation, which licenses these patents. No new patents were filed for epigenetic age analysis as part of this study. is a principal investigator at Altos Labs. is a cofounder and employee of the nonprofit Epigenetic Clock Development Foundation. is a consultant and/or scientific advisory board member of MitoBridge/Astellas, MetroBiotech, Amazentis, Vandria, OrsoBio and NovMetapharma. The other authors declare no competing interests. Nature Aging thanks Viktor Korolchuk, David Vilchez and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a Graphical representation of the construct used to generate transgenic Tg(actc1b:ZsGreen-map1lc3;cryaa:TdTomato) zebrafish to monitor autophagy. b Schematic workflow for image analysis of transgenic zebrafish larvae. 20X (0.75 NA) objective Z stacks (26 images, 2 µm apart) of the bodies were acquired in FITC (536 nm) channel using automated microscope (ImageXpress, Molecular Devices) and a 60 µm pinhole spinning disk. c Image acquisition of transgenic zebrafish larvae in 96 microwell plate. Scale bars, 10 mm (top); 1 mm (bottom-left); 100 µm (bottom-right) d Western blot images showing time-course experiment of inhibition of mTORC1 activity in 3 dpf zebrafish larvae treated with 1 µM rapamycin. e Densitometric quantification of western blot. f Scatterplot showing autophagy flux in zebrafish larvae treated with 240 selected chemicals from TOCRIS library at 10 µM (3 dpf, n = 12 per compound). a Representative images of transgenic zebrafish larvae at 3 dpf treated for 16 h with vehicle (1% DMSO), carvacrol and thymol at the indicated concentrations, followed by treatment with NH4Cl for 4 h. Scale bar, 50 µm. b,c Quantification of autophagic flux in transgenic zebrafish larvae (3 dpf, n = 17–21). Data are shown as mean ± SEM. Two-way ANOVA followed by Sidak multiple comparisons. d Western blot analysis of zebrafish larvae at 3 dpf treated with indicated doses of thymol or 2 µM of rapamycin for phospho-S6 signal activation e Western blot analysis of P-AMPK immunoreactivity with same treatments as in (d). a Cytofluorimetric analysis of autophagy flux in Jurkat cells treated with vehicle (ctrl) or thymol for two hours and stained with LC3-II antibody. Representative LC3-II plots show fluorescence intensities in presence (upper panel) or absence (lower panel) of an inhibitor of lysosomal activity. b Time-course and dose-response cytofluorimetric analysis of cell viability in Jurkat cells treated with thymol and rapamycin (n = 1). c Quantification of mitochondrial membrane potential (MMP) in Jurkat cells treated with control (DMSO 0.1%), carvacrol or FCCP (100 nM) measured as ratio between monomeric (Em = 590 nm) and aggregate forms (Em = 525 nm) of JC-10 (n = 3 wells/condition). Data in c and d are shown as means ± SEM. Ordinary one-way ANOVA followed by Sidak multiple comparisons. e Representative confocal microscopy images of Mouse Adult Fibroblasts (MAFs) treated with vehicle (0.1% DMSO), carvacrol (100 µM) and the uncoupler valinomycin (1 µM) in the presence of MitoTracker (green) and TMRM (red) to visualize changes in MMP. One-way ANOVA followed by Sidak multiple comparisons. g Scatterplot of QuantSeq 3' mRNA sequencing analysis in MAFs treated for 24 h with thymol (100 µM) or control (0.1% DMSO). Differentially regulated genes are highlighted in red (up-regulated) and blue (down-regulated); (control, n = 6; thymol, n = 6). h Unbiased gene set enrichment analysis in MAFs treated with vehicle and thymol. Gene expression analysis shows remaining significantly enriched gene sets following the top seven presented in Fig. a Transgenic construct to quantify mitophagy in mito-QC mice. b Representative confocal microscopy images of liver tissue 2 h hours after the second treatment. Note mosaic expression of the transgenic reporter proteins in hepatic tissue. d Number of mCherry-positive mitophagy foci. For each mouse, ten fields were analyzed, with each data point in the boxplots c and d representing the value from an individual microscopy field. a,b Whole body composition analysis in mice by multi-echo magnetic resonance imaging (MRI). Percentages of fat mass (a) and lean mass (b) were measured at the end of the treatment. One-way ANOVA followed by Sidak multiple comparisons. c Cumulative food intake of mice fed with control and high fat diet and treated with thymol or vehicle only. d Oral glucose tolerance test (OGTT) performed after 7 weeks of intervention. e Blood insulin concentrations after glucose challenge at the indicated timepoints. Data show mean ± SEM (n = 15) and are analyzed by one-way ANOVA followed by Sidak multiple comparisons. P values in d and e are color coded: orange for high fat diet versus control and green for high fat diet and thymol versus control. a Fluorescence microscopy images of transgenic GFP::LGG-1 reporter C. elegans visualizing the activation of autophagy through quantification of green fluorescent foci in body wall muscle cells. Young (Day 1) and aged (Day 10) worms were treated with thymol and carvacrol at 25 µM compared to vehicle (DMSO, 0.1%). Duration of administration was lifelong treatment (left panels) or 72 h (right panels) prior to imaging. Arrows indicate GFP::LGG-1 positive punctae, indicative of autophagosomes. b-e Motility assays in Day 10 C. elegans treated with thymol or carvacrol at the indicated doses. Mean speed (b,c) and total distance travelled (d,e) were quantified to assess age-related decline in motility functions. At least 92 worms were analyzed for each condition. f Percentage of damaged myofibrils in Day 10 old C. elegans in response to lifelong treatment with thymol at indicated doses. Ordinary One-way ANOVA followed by Sidak multiple comparisons. Data are pooled from two independent experiments, with at least 200 muscle fibers analyzed for each condition. g-i Quantification of damaged myofibrils as function of fluorescence intensity in response to thymol or carvacrol at indicated concentrations. Data are pooled from two independent experiments, with at least 300 muscle fibers analyzed for each condition. Ordinary One-way ANOVA followed by Sidaj multiple comparisons. j,k Survival analysis of C. elegans exposed to indicated doses of thymol (j) and carvacrol (k) throughout lifespan. At least 50 worms per condition have been analyzed and compared with vehicle using log rank test. a Weekly body weight analysis of mice from start to end of treatment (n = 15 mice/group). Mice that died were excluded from subsequent analyses b, c Average weights of tibialis anterior (b) and gastrocnemius (c) muscles (Control, n = 10; thymol, n = 13) d Spatial memory performance measured by Y maze at the baseline and after 12 weeks of treatment (Control, n = 11; thymol, n = 13). e Motor performance measured as latency to fall in a rotarod test (Control, n = 11; thymol, n = 13). f Western blot analysis of total lysate extracts of skeletal muscle from mice treated with thymol or vehicle using the indicated antibodies. g Gene enrichment for tnfα signaling via nfkb pathway from 3' QuantSeq analysis of MAFs shown as barcode plot. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Civiletto, G., Brunetti, D., Lizzo, G. et al. Herbal terpenoids activate autophagy and mitophagy through modulation of bioenergetics and protect from metabolic stress, sarcopenia and epigenetic aging. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250924012222.htm'>Tylenol in pregnancy linked to higher autism risk, Harvard scientists report</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 06:10:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When children are exposed to acetaminophen -- also known by the brand name Tylenol or as paracetamol -- during pregnancy, they may be more likely to develop neurodevelopmental disorders (NDDs) including autism and ADHD, according to a new study. The study was published recently in BMC Environmental Health. The researchers used the Navigation Guide Systematic Review methodology -- a gold-standard framework for synthesizing and evaluating environmental health data -- which enabled them to conduct a rigorous, comprehensive analysis that supported evidence of an association between acetaminophen exposure during pregnancy and increased incidence of NDDs. High fever can raise the risk of neural tube defects and preterm birth. "We recommend judicious acetaminophen use -- lowest effective dose, shortest duration -- under medical guidance, tailored to individual risk-benefit assessments, rather than a broad limitation," they wrote. In late September, the Food and Drug Administration announced it would issue a letter to clinicians urging them to be cautious about the use of acetaminophen in pregnancy. Baccarelli said he had discussed his study with Health and Human Services Secretary Robert F. Kennedy Jr. in the weeks leading up to that announcement and provided the White House team with an statement noting his research found "evidence of an association" between prenatal exposure to acetaminophen and neurodevelopmental disorders. The statement continued: "Further research is needed to confirm the association and determine causality, but based on existing evidence, I believe that caution about acetaminophen use during pregnancy -- especially heavy or prolonged use -- is warranted." Baccarelli noted in the "competing interests" section of the research paper that he has served as an expert witness for plaintiffs in a case involving potential links between acetaminophen use during pregnancy and neurodevelopmental disorders. Note: Content may be edited for style and length. Green Tea Shows Promise in Fighting Obesity and Diabetes Even One Drink May Raise Dementia Risk, Landmark Study Warns Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            