
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-24</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/nightmare-bacteria-infections-spiking-leaving-key-carbapenem-antibiotics/'>‘Nightmare Bacteria' Infections Spiking, Leaving Key Carbapenem Antibiotics Useless, CDC Warns</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 22:15:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists say people receiving treatment in hospitals are at highest risk Microbiologist holds a petri dish growing bacteria, Klebsiella pneumoniae, one species that can become resistant to last-resort antibiotics. The U.S. Centers for Disease Control and Prevention is raising alarm about a sharp spike in infections from dangerous bacteria that are resistant to some of the strongest antibiotics. A report released on Tuesday by CDC scientists found that, between 2019 and 2023, there was as much as a 461 percent increase in the infection rate of certain bacteria in the group Enterobacterales that can thwart many antibiotic treatments, including a powerful class of drugs known as carbapenems. The report's authors note that CRE infections are still considered rare and mostly occur in hospital settings. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “The biggest thing for us is that we understand where this is happening because we want to ensure that this does not go outside of health care settings [and] into the community and cause more difficult to treat infections.” The group Enterobacterales encompasses a wide number of common germs, including Escherichia coli and Klebsiella pneumoniae. One way bacterial species turn into CRE is by picking up specific genes that allow them to evade antibiotics, such as by giving them the ability to make carbapenemases, enzymes that inactivate carbapenems. They looked at overall changes in CRE infection rates, as well as the prevalence of five different types of genes that code for carbapenemases. “We saw this increase in at least every region,” Rankin says. Among CRE with these genes, those with a gene that codes for an enzyme called New Delhi metallo-β-lactamase, or NDM, had the largest jump in infection rate—461 percent. People receiving treatment in these care settings are currently at highest risk, but Rankin says it's not out of the realm of possibility for NDM-CRE infections to occur in other environments. They are not an oral antibiotic you can take,” Rankin says. “We are concerned because there is risk that this could spread into communities, meaning that common infections like urinary tract infections that are usually treated with the oral antibiotics may increasingly need to be treated with the IV antibiotics and require hospitalization.” “These are genes that can transfer resistance across different bacterial species,” she says. NDM-CRE can spread from improper hand hygiene among health care providers or inadequate cleaning and disinfection of equipment. Other key factors are insufficient testing and limited access to detection tools. “When NDM-CRE infections are not identified quickly, earlier treatment with effective antibiotics and infection control interventions may be delayed, which can then create more opportunities for transmission from patient to patient,” Rankin says. She hopes the report shows the importance of using specialized testing to detect these drug-resistant genes so that health care providers can deliver effective—and potentially lifesaving—treatments. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03093-6'>Exclusive: RFK Jr cancelled mRNA research — but the US military is still funding it</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 21:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). The abrupt termination last month of nearly half a billion dollars in US government contracts for mRNA-vaccine research rattled scientists working inside and outside industry. Yet not all large-scale research into mRNA vaccines in the United States is being dismantled. In the United States, the government considers such research crucial not only because these pathogens threaten soldiers deployed abroad, but also because they could ignite a global outbreak. “A lot of us are at least relieved the Department of Defense [DoD] is not abandoning mRNA research,” says Amesh Adalja, an infectious-disease specialist at the Johns Hopkins Center for Health Security in Baltimore, Maryland. “The whole biodefence structure is completely derailed,” Adalja says. Peter Berglund learnt that his company's federally backed vaccine programme was being cut the same way that many other affected firms did — in a 5 August notice from the HHS's Biomedical Advanced Research and Development Authority (BARDA), which ordered an immediate shutdown of ongoing studies. For Berglund, chief scientific officer at HDT Bio in Seattle, Washington, the news was a gut punch, as he told colleagues at a conference on RNA-based therapeutics in Boston, Massachusetts, this month. Within days, HDT executives heard from project managers at the DoD's Joint Program Executive Office (JPEO) for Chemical, Biological, Radiological and Nuclear Defense, which had been co-funding the CCHF vaccine research. HDT was told to restart its trial, with the JPEO pledging support through at least this first phase of clinical evaluation. The DoD funding, although substantial, is less than what had originally been pledged in conjunction with BARDA. “But, at least now, we can advance it through phase I” and worry about the rest later, he adds. Each is designed to protect against a different strain of avian influenza. Clinical-trial registries still list both BARDA and the JPEO as collaborators. An AstraZeneca spokesperson declined to comment on the US government's role in funding the trial against bird flu — which has been infecting US poultry and dairy cattle and raising the spectre of a leap into humans. The JPEO did not respond to requests for comment. In a statement, HHS press secretary Emily Hilliard disputed suggestions that withdrawing from joint projects would weaken the nation's pandemic preparedness, writing that “BARDA is prioritizing evidence-based, ethically grounded solutions”. Like its relative Ebola, Marburg virus can also cause haemorrhagic fever.Credit: NIAID/Science Photo Library The JPEO and BARDA had also been jointly funding a preclinical-stage vaccine programme for biotechnology firm Moderna in Cambridge, Massachusetts3. The mRNA shot is aimed at Marburg virus — a close but even deadlier relative of Ebola — which caused an outbreak earlier this year in northwest Tanzania, resulting in ten deaths. Neither Moderna nor its collaborator, the University of Texas Medical Branch in Galveston, responded to e-mails from Nature seeking comment on the project's funding status. mRNA vaccines for HIV trigger strong immune response in people Ebola outbreak in the DRC: why is it so deadly? Hotly anticipated US vaccine meeting ends with confusion — and a few decisions Three ways ex-CDC chief says that Trump team is sidelining science What research might be lost after the NIH's cuts? Machine learning reveals potential consequences of cuts to US research What research might be lost after the NIH's cuts? Machine learning reveals potential consequences of cuts to US research Trump's $100K visa fee for foreign talent: how will it affect researchers? 16 PhD positions available for highly motivated early-career researchers to be part of the new MSCA Doctoral Network – SHIELD. The Robotic Materials group at ETH Zurich is looking for three postdocs and one PhD for the project in soft robotics funded by ERC StG ColorfulAct. mRNA vaccines for HIV trigger strong immune response in people An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03112-6'>She lived to 117: what her genes and lifestyle tell us about longevity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 16:18:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Maria Branyas Morera had been verified as the oldest living person when she died last year at age 117.Credit: Xavier Dengra (Public Domain) “We wanted to learn from her particular case to benefit other people,” says Manel Esteller, a physician specializing in genetics at the University of Barcelona in Spain. At the time, Branyas was living in the small town of Olot, in the Catalonia region of Spain, where she enjoyed reading books, playing with dogs and spending time with friends and family, including her two daughters — each in their 90s. One of the main insights from the work, Esteller says, is that it is possible to distinguish molecular changes that happen in the body because of ageing from those that occur because of poor health. “This is telling us that the loss of telomeres is not necessarily associated with disease, it's simply associated with being old,” Esteller says. Mayana Zatz, a geneticist at the University of São Paulo in Brazil who studies the genetics of healthy centenarians, says the study is thorough but notes that conclusions that are based on a single individual are limited. “It would be interesting to compare the findings with supercentenarians in other populations,” she says. Ageing accelerates around age 50 ― some organs faster than others Ageing is linked to inflammation — but only in the industrialized world What molecular ‘clocks' can tell you about your health A Chinese AI tool can manage chronic disease — could it revolutionize health care? Ageing accelerates at around age 50 ― some organs faster than others One potent gene raises risk of Alzheimer's, Parkinson's and other brain diseases 16 PhD positions available for highly motivated early-career researchers to be part of the new MSCA Doctoral Network – SHIELD. The Robotic Materials group at ETH Zurich is looking for three postdocs and one PhD for the project in soft robotics funded by ERC StG ColorfulAct. Ageing is linked to inflammation — but only in the industrialized world What molecular ‘clocks' can tell you about your health An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09312-4'>The Biodiversity Cell Atlas: mapping the tree of life at cellular resolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:44:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 645, pages 877–885 (2025)Cite this article Cell types are fundamental functional units that can be traced across the tree of life. Rapid advances in single-cell technologies, coupled with the phylogenetic expansion in genome sequencing, present opportunities for the molecular characterization of cells across a broad range of organisms. Despite these developments, our understanding of eukaryotic cell diversity remains limited and we are far from decoding this diversity from genome sequences. Here we introduce the Biodiversity Cell Atlas initiative, which aims to create comprehensive single-cell molecular atlases across the eukaryotic tree of life. This community effort will be phylogenetically informed, rely on high-quality genomes and use shared standards to facilitate comparisons across species. The Biodiversity Cell Atlas aspires to deepen our understanding of the evolution and diversity of life at the cellular level, encompassing gene regulatory programs, differentiation trajectories, cell-type-specific molecular profiles and inter-organismal interactions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Goffeau, A. et al. Life with 6000 genes. C. elegans Sequencing Consortium.Genome sequence of the nematode C. elegans: a platform for investigating biology. Initiative, T. A. G. Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. Lander, E. S. et al. Initial sequencing and analysis of the human genome. Lewin, H. A. et al. Earth BioGenome Project: sequencing life for the future of life. Lewin, H. A. et al. Sullivan, P. F. et al. Leveraging base-pair mammalian constraint to understand genetic variation and human disease. Kapli, P., Yang, Z. & Telford, M. J. Phylogenetic tree building in the genomic age. Smith, S. D., Pennell, M. W., Dunn, C. W. & Edwards, S. V. Phylogenetics is the new genetics (for most of biodiversity). Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative cis-regulatory code. Oliver, S. G. From DNA sequence to biological function. Arendt, D. The evolution of cell types in animals: emerging principles from molecular studies. A., Kim, J. K., Svensson, V., Marioni, J. C. & Teichmann, S. A. The technology and biology of single-cell RNA sequencing. Rao, A., Barkley, D., França, G. S. & Yanai, I. Exploring tissue architecture using spatial transcriptomics. Minnoye, L. et al. Chromatin accessibility profiling methods. Regev, A. et al. The Human Cell Atlas. Haniffa, M. et al. A roadmap for the Human Developmental Cell Atlas. Lee, T. A. et al. A single-nucleus atlas of seed-to-seed development in Arabidopsis. Preprint at bioRxiv https://doi.org/10.1101/2023.03.23.533992 (2023). Schaum, N. et al. Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris. Li, H. et al. Fly Cell Atlas: a single-nucleus transcriptomic atlas of the adult fruit fly. Zhang, X. et al. A spatially resolved multi-omic single-cell atlas of soybean development. Marand, A. P., Chen, Z., Gallavotti, A. A cis-regulatory atlas in maize at single-cell resolution. Cao, J. et al. Comprehensive single-cell transcriptional profiling of a multicellular organism. References 18–23 represent comprehensive whole-organism cell atlases for animal and plant model species. A. et al. Divergence in neuronal signaling pathways despite conserved neuronal identity among Caenorhabditis species. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation. Mendieta, J. P. et al. Investigating the cis-regulatory basis of C3 and C4 photosynthesis in grasses at single-cell resolution. Triesch, S. et al. Single-nuclei sequencing of Moricandia arvensis reveals bundle sheath cell function in the photorespiratory shuttle of C3-C4 intermediate Brassicaceae. Li, J. et al. Deep learning of cross-species single-cell landscapes identifies conserved regulatory programs underlying cell types. Agarwal, V. & Shendure, J. Predicting mRNA abundance directly from genomic sequence using deep convolutional neural networks. Avsec, Ž. et al. Effective gene expression prediction from sequence by integrating long-range interactions. Janssens, J. et al. Decoding gene regulation in the fly brain. Pearce, J. D. et al. A cross-species generative cell atlas across 1.5 billion years of evolution: the TranscriptFormer single-cell model. Preprint at bioRxiv https://doi.org/10.1101/2025.04.25.650731 (2025). Taskiran, I. I. et al. Cell-type-directed design of synthetic enhancers. Mich, J. K. et al. Functional enhancer elements drive subclass-selective expression from mouse to primate neocortex. Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Church, S. H., Mah, J. L. & Dunn, C. W. Integrating phylogenies into single-cell RNA sequencing analysis allows comparisons across species, genes, and cells. Del Campo, J. et al. The others: our biased perspective of eukaryotic genomes. Ku, C. & Sebé-Pedrós, A. Using single-cell transcriptomics to understand functional states and interactions in microbial eukaryotes. Alacid, E. & Richards, T. A. A cell–cell atlas approach for understanding symbiotic interactions between microbes. Hu, M., Zheng, X., Fan, C.-M. & Zheng, Y. Lineage dynamics of the endosymbiotic cell type in the soft coral Xenia. Levy, S. et al. A stony coral cell atlas illuminates the molecular and cellular basis of coral symbiosis, calcification, and immunity. Serrano, K. et al. Spatial co-transcriptomics reveals discrete stages of the arbuscular mycorrhizal symbiosis. Fromm, A. et al. Single-cell RNA-seq of the rare virosphere reveals the native hosts of giant viruses in the marine environment. Ku, C. et al. A single-cell view on alga-virus interactions reveals sequential transcriptional programs and infection states. References 42–45 represent the first examples of using single-cell methods to study symbiotic interactions, simultaneously mapping host and symbiont gene expression programs within the same cells. Burki, F., Sandin, M. M. & Jamy, M. Diversity and ecology of protists revealed by metabarcoding. Delmont, T. O. et al. Functional repertoire convergence of distantly related eukaryotic plankton lineages abundant in the sunlit ocean. Cordier, T., Lanzén, A., Apothéloz-Perret-Gentil, L., Stoeck, T. & Pawlowski, J. Embracing environmental genomics and machine learning for routine biomonitoring. Marchetti, A. et al. Comparative metatranscriptomics identifies molecular bases for the physiological responses of phytoplankton to varying iron availability. Brunet, T. & King, N. The origin of animal multicellularity and cell differentiation. Sebé-Pedrós, A., Degnan, B. M. & Ruiz-Trillo, I. The origin of Metazoa: a unicellular perspective. Arendt, D. et al. The origin and evolution of cell types. Establishes a foundational conceptual framework for the study of cell type evolution and outlines key open questions and future research directions in the field. Archibald, J. M., Simpson, A. G. B. & Slamovits, C. H. Handbook of the Protists (Springer, 2017). Fritz-Laylin, L. K. et al. The genome of Naegleria gruberi illuminates early eukaryotic versatility. Brunet, T. et al. Light-regulated collective contractility in a multicellular choanoflagellate. Brunet, T. et al. A flagellate-to-amoeboid switch in the closest living relatives of animals. Dayel, M. J. et al. Cell differentiation and morphogenesis in the colony-forming choanoflagellate Salpingoeca rosetta. Phillips, M. A. et al. Malaria. Häckel, E. Monograph of Monera. Saville-Kent, W. A Manual of the Infusoria: Including a Description of All Known Flagellate, Ciliate, and Tentaculiferous Protozoa, British and Foreign, and an Account of the Organization and the Affinities of the Sponges Vol. Ramón y Cajal, S. Histologie Du Système Nerveux de l'homme & Des Vertébrés: Cervelet, Cerveau Moyen, Rétine, Couche Optique, Corps Strié, Écorce Cérébrale Générale & Régionale, Grand Sympathique Vol. Ramón y Cajal, S. Estructura de los centros nerviosos de las Aves. Revista Trimestral de Histología Normal y Patológica 1, 1–10 (1888). Virchow, R. Cellular Pathology as Based Upon Physiological and Pathological Histology (John Churchill, 1860). Hyman, L. H. The Invertebrates: Protozoa through Ctenophora (McGraw-Hill, 1940). Willmer, E. N. Cytology and Evolution (Academic, 1970). Vergara, H. M. et al. Whole-body integration of gene expression and single-cell morphology. Steinmetz, P. R. H. et al. Independent evolution of striated muscles in cnidarians and bilaterians. Ogino, K., Tsuneki, K. & Furuya, H. Distinction of cell types in Dicyema japonicum (phylum Dicyemida) by expression patterns of 16 genes. Novershtern, N. et al. Densely interconnected transcriptional circuits control cell states in human hematopoiesis. Minnoye, L. et al. Cross-species analysis of enhancer logic using deep learning. Alié, A. et al. The ancestral gene repertoire of animal stem cells. Cherbas, L. et al. The transcriptional diversity of 25 Drosophila cell lines. Evolutionary cell type mapping with single-cell genomics. Musser, J. M. et al. Profiling cellular diversity in sponges informs animal cell type and nervous system evolution. Sebé-Pedrós, A. et al. Cnidarian cell type diversity and regulation revealed by whole-organism single-cell RNA-seq. Sebé-Pedrós, A. et al. Early metazoan cell type diversity and the evolution of multicellular gene regulation. Najle, S. R. et al. Stepwise emergence of the neuronal gene expression program in early animal evolution. Plass, M. et al. Cell type atlas and lineage tree of a whole complex animal by single-cell transcriptomics. Fincher, C. T., Wurtzel, O., de Hoog, T., Kravarik, K. M. & Reddien, P. W. Cell type transcriptome atlas for the planarian Schmidtea mediterranea. References 75, 76, 78 and 79 represent the first whole-adult cell atlases for non-model animal species. Robertson, H. E. et al. Single cell atlas of Xenoturbella bocki highlights limited cell-type complexity. Álvarez-Campos, P. et al. Annelid adult cell type diversity and their pluripotent cellular origins. Ghaddar, A. et al. Whole-body gene expression atlas of an adult metazoan. Dogga, S. K. et al. A single cell atlas of sexual development in Plasmodium falciparum. Exemplifies the power of single-cell analysis to molecularly characterize cell states across the life cycle of unicellular eukaryotes. Wang, S. Y. et al. Role of epigenetics in unicellular to multicellular transition in Dictyostelium. Howick, V. M. et al. The Malaria Cell Atlas: single parasite transcriptomes across the complete Plasmodium life cycle. Wang, L. et al. The maturation and aging trajectory of Marchantia polymorpha at single-cell resolution. Scaling single-cell genomics from phenomenology to mechanism. Domcke, S. & Shendure, J. A reference cell tree will serve science better than a reference cell atlas. Shafer, M. E. R. Cross-species analysis of single-cell transcriptomic data. Yan, H. et al. Evolution of plant cell-type-specific cis-regulatory elements. Preprint at bioRxiv https://doi.org/10.1101/2024.01.08.574753 (2024). Hecker, N. et al. Enhancer-driven cell type comparison reveals similarities between the mammalian and bird pallium. One of the first examples of cross-species cell type comparisons based on regulatory sequence information rather than gene expression data. Bakken, T. E. et al. Comparative cellular analysis of motor cortex in human, marmoset and mouse. Song, Y., Miao, Z., Brazma, A. & Papatheodorou, I. Benchmarking strategies for cross-species integration of single-cell RNA sequencing data. Tarashansky, A. J. et al. Mapping single-cell atlases throughout Metazoa unravels cell type evolution. Mah, J. L. & Dunn, C. W. Cell type evolution reconstruction across species through cell phylogenies of single-cell RNA sequencing data. Lays out important considerations for cell type phylogenetic reconstruction and evolutionary models. Burkhardt, P. & Jékely, G. Evolution of synapses and neurotransmitter systems: the divide-and-conquer model for early neural cell-type evolution. Arendt, D., Bertucci, P. Y., Achim, K. & Musser, J. M. Evolution of neuronal types and families. Wagner, G. P. The developmental genetics of homology. Pacureanu, A., Silva, J. C. da, Yang, Y., Bohic, S. & Cloetens, P. Nanoscale three-dimensional imaging of biological tissue with X-ray holographic tomography. 10711 Biomedical Imaging and Sensing Conf. (eds Yatagai, T. et al.) 107112B (SPIE, 2018). Zinchenko, V., Hugger, J., Uhlmann, V., Arendt, D. & Kreshuk, A. MorphoFeatures for unsupervised exploration of cell types, tissues, and organs in volume electron microscopy. Di Tommaso, P. et al. Nextflow enables reproducible computational workflows. Denisenko, E. et al. Systematic assessment of tissue dissociation and storage biases in single-cell and single-nucleus RNA-seq workflows. Scully, T. & Klein, A. A mannitol-based buffer improves single-cell RNA sequencing of high-salt marine cells. Preprint at bioRxiv https://doi.org/10.1101/2023.04.26.538465 (2023). Chari, T. et al. Whole-animal multiplexed single-cell RNA-seq reveals transcriptional shifts across Clytia medusa cell types. Alles, J. et al. Cell fixation and preservation for droplet-based single-cell transcriptomics. García-Castro, H. et al. ACME dissociation: a versatile cell fixation-dissociation method for single-cell transcriptomics. B. et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Bageritz, J. et al. Glyoxal as an alternative fixative for single-cell RNA sequencing. Jiménez-Gracia, L. et al. FixNCut: single-cell genomics through reversible tissue fixation and dissociation. Fortmann, S. D. et al. Fixation before dissociation using a deep eutectic solvent preserves in vivo states and phospho-signaling in single-cell sequencing. Preprint at bioRxiv https://doi.org/10.1101/2023.02.13.528370 (2023). Martin, B. K. et al. Optimized single-nucleus transcriptional profiling by combinatorial indexing. Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Petrany, M. J. et al. Single-nucleus RNA-seq identifies transcriptional heterogeneity in multinucleated skeletal myofibers. Guillotin, B. et al. A pan-grass transcriptome reveals patterns of cellular divergence in crops. Jariani, A. et al. A new protocol for single-cell RNA-seq reveals stochastic gene expression during lag phase in budding yeast. Grones, C. et al. Best practices for the execution, analysis, and data storage of plant single-cell/nucleus transcriptomics. Heaton, H. et al. Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes. Guigó, R. Genome annotation: from human genetics to biodiversity genomics. Weisman, C. M., Murray, A. W. & Eddy, S. R. Mixing genome annotation methods in a comparative analysis inflates the apparent number of lineage-specific genes. Altenhoff, A. M. et al. Standardized benchmarking in the quest for orthologs. Glover, N. et al. Advances and applications in the quest for orthologs. Rosen, Y. et al. Toward universal cell embeddings: integrating single-cell RNA-seq datasets across species with SATURN. Rosen, Y. et al. Universal cell embeddings: a foundation model for cell biology. Preprint at bioRxiv https://doi.org/10.1101/2023.11.28.568918 (2023). Price, P. D. et al. Detecting signatures of selection on gene expression. Bertram, J. et al. CAGEE: computational analysis of gene expression evolution. Rohlfs, R. V., Harrigan, P. & Nielsen, R. Modeling gene expression evolution with an extended Ornstein–Uhlenbeck process accounting for within-species variation. Rohlfs, R. V. & Nielsen, R. Phylogenetic ANOVA: the expression variance and evolution model for quantitative trait evolution. Challis, R., Kumar, S., Sotero-Caio, C., Brown, M. & Blaxter, M. Genomes on a Tree (GoaT): a versatile, scalable search engine for genomic and sequencing project metadata across the eukaryotic tree of life. Wellcome Open Res. Zeng, H. What is a cell type and how to define it? Osumi-Sutherland, D. et al. Cell type ontologies of the Human Cell Atlas. The evolving concept of cell identity in the single cell era. We thank J. C. Carvalho for her expert work on the scientific illustrations and figure design. The BCA phase 0 is supported by the Gordon and Betty Moore Foundation (Grant GBMF12189). We also acknowledge financial support from the Wellcome Trust for the inaugural BCA meeting. A full list of members and their affiliations appears in the Supplementary Information Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain Arnau Sebé-Pedrós, Mafalda Dias, Anamaria Elek, Jonathan Frazer, Xavier Grau-Bové, Roderic Guigó, Manuel Irimia & Sebastián R. Najle Tree of Life, Wellcome Sanger Institute, Cambridge, UK Arnau Sebé-Pedrós, Mara K. N. Lawniczak & Mark Blaxter Universitat Pompeu Fabra, Barcelona, Spain Arnau Sebé-Pedrós, Mafalda Dias, Jonathan Frazer, Roderic Guigó & Manuel Irimia ICREA, Barcelona, Spain Arnau Sebé-Pedrós, Toni Gabaldón & Manuel Irimia Department of Computer Science and Applied Mathematics, Weizmann Institute of Science, Rehovot, Israel Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel Developmental Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany Laboratory of Computational Biology, VIB Center for AI and Computational Biology, Leuven, Belgium VIB-KU Leuven Center for Brain and Disease Research, Leuven, Belgium Department of Human Genetics, KU Leuven, Leuven, Belgium Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada Stazione Zoologica Anton Dohrn, Napoli, Italy Maria Ina Arnone Department of Embryology, Carnegie Science, Baltimore, MD, USA Department of Biology, Johns Hopkins University, Baltimore, MD, USA Department of Algal Development and Evolution, Max Planck Institute for Biology Tübingen, Tübingen, Germany Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT, USA Barcelona Supercomputing Centre (BSC-CNS), Plaça Eusebi Güell, Barcelona, Spain Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, Barcelona, Spain Donnelly Centre, University of Toronto, Toronto, Ontario, Canada Department of Physiology, University of Toronto, Toronto, Ontario, Canada Department of Biological Sciences, Howard Hughes Medical Institute, Columbia University, New York, NY, USA Centro de Biotecnología y Genómica de Plantas, Universidad Politécnica de Madrid (UPM)—Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA-CSIC), Campus de Montegancedo-UPM, Madrid, Spain Department of Systems Biology, Harvard Medical School, Boston, MA, USA Global Futures Laboratory, Arizona State University, Tempe, AZ, USA Department of Evolution and Ecology, University of California, Davis, CA, USA Department of Biology, Stanford University, Stanford, CA, USA Department of Organismal Biology and Anatomy, University of Chicago, Chicago, IL, USA Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT, USA Systems Biology Institute, Yale University, New Haven, CT, USA Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA, USA Department of Computer Science, California Institute of Technology, Pasadena, CA, USA The Rockefeller University, New York, NY, USA Medical School, University of East Anglia, Norwich Research Park, Norwich, UK Genomics Department, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA Berlin Institute for Medical Systems Biology, Max Delbrück Center for Molecular Medicine, Berlin, Germany German Center for Cardiovascular Research (DZHK), Berlin, Germany NeuroCluster of Excellence, Berlin, Germany Plant Resilience Institute, Michigan State University, East Lansing, MI, USA Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI, USA Department of Plant Biology, Michigan State University, East Lansing, MI, USA Department of Plant, Soil, and Microbial Sciences, Michigan State University, East Lansing, MI, USA Department of Biology, University of Oxford, Oxford, UK Stowers Institute for Medical Research, Kansas City, MO, USA Radcliffe Department of Medicine, University of Oxford, Oxford, UK Centre for Organismal Studies, Heidelberg University, Heidelberg, Germany Living Systems Institute, University of Exeter, Exeter, UK Department of Biosciences, University of Exeter, Exeter, UK European Molecular Biology Laboratory—European Bioinformatics Institute, Hinxton, UK Department of Neurosciences and Developmental Biology, Faculty of Life Sciences, University of Vienna, Vienna, Austria Department of Bioengineering, Stanford University, Stanford, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the paper based on discussions, comments and edits from S.A., J.A., M.I.A., M.B., P.C., S.M.C., M.D., C.D., A.E., J.F., T.G., J.G., X.G.-B., R.G., O.H., J.H.-C., M.I., A.K., H.L., C.J.L., H.M., J.M.M., L.G.N. Correspondence to Arnau Sebé-Pedrós. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. BCA single-cell dataset reporting checklist and recommended quality controls. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Sebé-Pedrós, A., Tanay, A., Lawniczak, M.K.N. The Biodiversity Cell Atlas: mapping the tree of life at cellular resolution. Issue Date: 25 September 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09568-w'>Collective homeostasis of condensation-prone proteins via their mRNAs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:44:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The concentration of proteins containing intrinsically disordered regions must be tightly controlled to maintain cellular homeostasis1,2. However, mechanisms for collective control of these proteins, which tend to localize to membraneless condensates, are less understood than pathways mediated by membrane-bound organelles3,4. Here we report ‘interstasis', a homeostatic mechanism in which increased concentration of proteins within RNA–protein condensates induces the sequestration of their own mRNAs. The selectivity of interstatic mRNA capture relies on the structure of the genetic code and conserved codon biases, which ensure that similar multivalent RNA regions encode similar low-complexity domains. For example, arginine-enriched mixed charge domains (R-MCDs) tend to be encoded by repetitive purine-rich sequences in mRNAs. The multivalent regions are bound by specific RNA-binding proteins, including TRA2 proteins, which relocalize to speckles upon interstasis to promote selective mRNA capture. CLK-mediated phosphorylation of TRA2 proteins counters their localization to speckles, thereby modulating interstasis. Thus, the condensation properties of nuclear speckles act as a sensor for interstasis, a collective negative-feedback loop that co-regulates mRNAs of highly dosage-sensitive genes, which primarily encode nuclear condensation-prone proteins. Fitness screens in multiple eukaryotic species have shown that proteins containing intrinsically disordered regions (IDRs) are more likely to be toxic when their dosage is increased1. The toxicity induced by the increased dosage of individual IDR-containing proteins in yeast has been linked to their propensity to phase separate and form biomolecular condensates3. Proteins that are prone to phase separation have the greatest mismatch between protein and RNA abundance, indicating that post-transcriptional mechanisms strongly affect the protein abundance1,2. Many proteins can interact with the RNA or DNA of their own gene, yet no sensors or effectors are known that could regulate the collective expression of condensation-prone proteins5,6,7,8. Co-condensation of proteins is partly determined by the molecular grammar of short motifs and biophysically similar amino acids in their IDRs9,10,11,12. Many IDRs contain repetitive amino acid arrangements and are thus classified as low-complexity domains (LCDs). One type of LCD is the R-MCD, where arginines are interspersed among other positively and negatively charged amino acids. R-MCDs promote localization of proteins into nuclear speckles10. Major overexpression of artificial R-MCDs has been shown to increase speckle cohesion to the extent that all polyadenylated mRNAs can become retained in the speckles, highlighting the need for homeostatic co-regulation of such proteins10. Biophysically similar amino acids tend to have similar codons, a characteristic of genetic code that is thought to have emerged to mitigate the effects of non-synonymous point mutations or translation errors13,14. We asked whether the mRNA regions that encode LCDs contain repetitive arrangements of similar sequences, termed ‘multivalent regions'. To generalize the analysis of multivalent RNA sequences beyond specific motif sets, we designed a scoring algorithm that assigns a generalized RNA multivalency (GeRM) score for each nucleotide within provided RNA sequences (Fig. For a given k-mer at each position along a transcript, a GeRM score is calculated by assessing the similarity of surrounding k-mers, weighted based on the proximity between k-mers (Fig. A high GeRM score thus indicates that the k-mer is present in highly multivalent RNA regions, where large numbers of similar k-mers are found in close proximity (Fig. We applied the GeRM algorithm to the longest coding transcript isoform for every human gene and then focused our analyses on the coding sequences (CDSs). For example, the profile of LUC7L3 mRNA reveals a region of high GeRM scores (‘GeRM region') that is highly GA rich and that encodes exclusively charged amino acids (Fig. b, An example of two 5-mers with either high or low GeRM scores. c, An example transcript from the gene LUC7L3. d, The native DNA and amino acid sequences of LUC7L3 within the GeRM peak and a synonymously codon-shuffled sequence. Conservation across 100 vertebrates (PhyloP) for each position that can tolerate synonymous mutation is shown by the height of the letter. Below, a ratio of GeRM scores for the native codon choice to average scores of any synonymous mutation is also shown. e, The mean entropy for amino acid sequences encoded inside high GeRM CDS regions (black line) outside but within the same protein (grey line). f, As in e, but comparing the AlphaFold-predicted pLDDT values inside (black line) and outside (grey line) protein regions encoded by high GeRM regions. h, The normalized conservation across 100 vertebrates of synonymously mutable positions in coding sequences that either encode LCDs or do not. Unless otherwise stated, all pairwise significance testing were performed using FDR-corrected Welch t-tests, where *P < 1−15. Precise P values can be found in the Source Data. To assess the complexity of amino acid sequences in GeRM regions, we calculated their information entropy using a sliding window and found that their entropy was significantly lower than the remaining sequences of the same proteins and to the average throughout the proteome, demonstrating that GeRM regions commonly encode LCDs (Fig. Moreover, we used AlphaFold2 prediction confidence to approximate protein disorder and found that GeRM regions preferentially encode IDRs, as they are not predicted to be structured (Fig. If RNA multivalency was functionally important, one would expect a selection pressure for biases in codon usage that reinforce the multivalency of LCD-encoding regions. To examine such biases, we synonymously shuffled the codons within each transcript and calculated the mean GeRM potential across ten shuffles (Fig. In the example of LUC7L3, codon choices reinforce the GA richness of the CDS region that encodes an R-MCD and is almost entirely composed of charged amino acids (Fig. Conversely, in the example of CCDC61, codon choices reinforce the G richness of the CDS region that encodes an LCD that contains only approximately 40% of charged amino acids and is rich in arginines, alanines and glycines (Extended Data Fig. To more broadly assess how codon biases reinforce the multivalency of LCD-encoding CDS regions, we then defined LCDs as regions in the bottom 2% of amino acid entropy (Supplementary Table 7). This suggests that, especially in regions encoding LCDs, codon choices tend to promote the multivalency potential of the RNA. We speculated that if codon usage supports high multivalency of sequences encoding LCDs, then these codon usage biases would be evolutionarily conserved. We tested every possible synonymous codon substitution in the transcriptome and calculated the change in GeRM potential of each overlapping 5-mer (Extended Data Fig. The average ratio for all but four possible synonymously mutable codons was positive, with more common codons having higher ratios on average (Extended Data Fig. To account for the general differences in amino acid composition and conservation of LCDs (Extended Data Fig. We found that the more strongly a native codon promotes multivalency, the more conserved the codon is across mammals or vertebrates, and that this effect is especially strong within the regions that encode LCDs (Fig. This analysis suggests that evolutionary selection pressure tends to preserve codons that enhance high multivalency potential of RNA sequences encoding LCDs. To understand whether RNA multivalency could serve as a template for co-regulation of co-condensing proteins, we assessed its relationships to the LCD molecular grammar. We calculated the contribution of each 5-mer to the total GeRM score of each region, reduced the dimensionality of the data with UMAP and clustered all protein-coding GeRM regions using HDBSCAN. We observed diverse clusters, some of which represented trinucleotide repeats, whereas the largest clusters lacked stereotyped repetition: a GC-rich cluster, a C-rich cluster and three GA-rich clusters (one cluster with more adenines, one with more guanines and one with interspersed cytosines; Fig. All types of GeRM regions encoded LCDs (Extended Data Fig. 2d), and GeRM regions with similar multivalent RNA motifs tend to encode LCDs that are dominated by similar amino acids (Fig. The GA-rich GeRM regions encoded domains rich in charged amino acids, with the ratio of adenine to guanine in the multivalent motifs biasing the encoded domain towards positive or negative charge, respectively. b, UMAP embedding from a, with each GeRM region recoloured to show the most common amino acid encoded within the region. c, Heatmap of Gene Ontology fold enrichments (colour) and significance values (size) for genes containing GeRM regions from the clusters shown in a. NS, not significant; proc., process; tx, transcription. d, The correlation (Spearman's rank) of arginine codon usage within LCDs containing at least 20% arginine (R-LCDs). e, The Gene Ontology fold enrichments for genes containing R-LCDs encoded by GA-rich codons or CG-rich codons. f, Analysis of codon usage correlations within R-LCDs in different species. The black dashed line denotes no correlation. Next, we asked whether each type of GeRM region is enriched in mRNAs that encode proteins with related functions. We performed an ontology analysis of gene sets containing each type of GeRM region and found that each set was associated with specific terms (Fig. The GA-rich GeRM regions were primarily associated with nuclear RNA regulation or chromatin-related terms, whereas GC-rich and CAG-repeat GeRM regions were enriched mainly for transcription-related terms, which agrees with previous studies showing GC bias for codons in transcription factors15. The C-rich GeRM regions were most enriched in the SH3 domain and actin-binding terms, whereas the CU-rich regions were enriched for keratin filament genes. Therefore, each type of RNA multivalency in CDS regions is linked to specific classes of LCDs and enriched in functionally related groups of transcripts. All RNA multivalency classes in coding regions are reinforced by biased codon usage, as evident by significantly decreased RNA multivalency in all types of GeRM regions after codon shuffling (Extended Data Fig. These biases are driven mainly by codon choice for amino acids that contain a large number of possible codons, such as arginine, which is common in multiple types of LCDs (Fig. We calculated the arginine codon usage within each LCD that contained at least 20% arginine (R-LCDs), which could be separated based on codon-usage principal component into those encoded primarily by GA-rich or CG-rich codons (Extended Data Fig. Those with GA-rich arginine codons were highly enriched for other charged residues and thus included R-MCDs (Extended Data Fig. 3c,f), whereas R-LCDs with CG-rich codons contained more proline, alanine and glycine (Extended Data Fig. Of note, the GA-biased R-LCDs are much more highly enriched for nuclear speckle, mRNA-processing and RNA-binding Gene Ontology terms compared with GC-rich R-LCDs (Fig. Conversely, mRNAs encoding proteins with nuclear speckle or RNA-binding Gene Ontology terms show a strong bias towards GA-rich arginine codons (Extended Data Fig. Therefore, arginine codon choice within GeRM regions conforms to the multivalent identity of the region by mimicking the sequences of other codons encoding charged amino acids, thereby maximizing the multivalency of RNA regions encoding functionally distinct classes of R-LCD. As representative examples, the arginine codon biases in LUC7L3 and CCDC61 R-LCDs reinforce their distinct AG-rich and CG-rich multivalency, respectively (Fig. As a measure of the distinct arginine codon preference in the two classes of R-LCDs, we observed a positive correlation in R-LCDs between AGA, AGG and CGA arginine codons, and between CGG and CGC codons, and strong anticorrelations between these two groups (Fig. We used this approach to ask at what point in evolution this codon bias in R-LCDs emerged. We observed the correlations and anticorrelations between AG and CG arginine codons in R-LCDs across a diverse range of species. Although the total proportions of R-LCDs did not dramatically change across species (Extended Data Fig. 3g,h), we observed that the correlations between CG-rich codons, between GA-rich codons and the anticorrelation between these groups were stable from humans to reptiles, but that this pattern was less pronounced in amphibians and other species that diverged earlier in evolutionary history (Fig. Thus, the codon biases that reinforce the distinct RNA multivalency of two R-LCD classes have become most pronounced in terrestrial vertebrates. As the accumulation of R-MCDs affects the properties of nuclear speckles10, we investigated whether speckles enable any feedback regulation of R-MCD proteins, and whether GA-rich GeRM regions have any role in such regulation. To assess this, we combined the region encoding the R-MCD C terminus of peptidylprolyl isomerase G (PPIG; Fig. 3a) fused to mScarlet into a doxycycline-inducible PiggyBac vector, which was integrated into a stable polyclonal HeLa cell population. After 16 h of induction, we observed mScarlet–PPIGLCD localization to the nucleus, where it colocalized with SC35, a nuclear speckle marker (Extended Data Fig. Expression of PPIGLCD resulted in fewer, larger speckles per nucleus, without reducing the intensity of SC35 staining (Extended Data Fig. 4e), consistent with previously reported effects of R-MCDs on speckles10. b, Expression of PPIGLCD over time after doxycycline (dox) induction from a single experiment. n = 6,367, 566 and 141 genes for the bottom 90%, top 10% and top 2%, respectively, with calculations performed on values from 4 independent replicates. d, The change in proportion of labelled peptides after 8 h of SILAC labelling per protein when inducing PPIGLCD expression for 12 h compared with uninduced cells. e,f, Example images from a set of three independent experiments showing the HCR-FISH signal for PSAP (control; e) or BRD4 (GA multivalent; f) mRNAs, with SC35 immunofluorescence and mScarlet–PPIGLCD. Three independent replicates are plotted in different colours, and regression slopes are plotted in dashed lines. h, The slopes of linear regression models of the relationship between mScarlet–PPIGLCD expression and mRNA enrichment within the speckle for control and GA-multivalent mRNAs. Precise P values can be found in the Source Data. We performed 3′ end sequencing on the nuclear and cytoplasmic fractions of our reporter cell line after 0, 4, 8 and 12 h of mScarlet–PPIGLCD induction to assess the effect of its gradual accumulation (Fig. We then categorized genes based on their total CDS GA multivalency and looked at the nuclear–cytoplasmic distribution of these genes over time. To investigate the effect of the observed nuclear retention of GA-rich multivalent transcripts on protein synthesis, we performed pulsed stable isotope labelling by amino acids in cell culture (pSILAC). PPIGLCD expression was induced for 12 h, followed by an 8-h pulse labelling period to selectively label and quantify proteins newly synthesized during the PPIGLCD induction window. Comparison of heavy-to-light peptide ratios between doxycycline-induced and uninduced controls revealed a selective decrease in the proportion of labelled peptides encoded by mRNAs with high GA multivalency (Fig. 3d), with minimal effect observed on the translation of all other proteins. Thus, the nuclear retention of GA-rich multivalent transcripts upon PPIGLCD expression corresponds with downregulated translation of the proteins that these mRNAs encode. To determine whether GA-rich transcripts are specifically sequestered into nuclear speckles, we designed hybridization chain reaction fluorescence in situ hybridization (HCR-FISH) probes against five GA-rich multivalent mRNAs and two non-multivalent control mRNAs. We then induced PPIGLCD expression for 16 h and performed HCR-FISH along with SC35 immunofluorescence (Fig. These observations correspond to changes in nuclear–cytoplasmic distribution in our sequencing experiments (Extended Data Fig. Under these conditions we also observed a slight dose-dependent enrichment of poly-A+ mRNA in nuclear speckles, accompanied by a mild increase in the proportion of nuclear mRNA, indicating that mRNA sequestration is selective (Extended Data Fig. We next asked whether this effect is generalizable across R-MCDs. We transfected HeLa cells with mGreenLantern-fused R-MCDs from three additional proteins (LUC7L3, PRPF38B and SRSF11) and repeated HCR-FISH 24 h post-transfection, with two GA-rich multivalent mRNAs and two non-multivalent controls (Extended Data Fig. This induced the selective speckle retention of GA-rich mRNAs, consistent with our observations with PPIGLCD (Extended Data Fig. We did not observe this effect when transfecting other LCDs with different amino acid identities (Extended Data Fig. We conclude that variations in the concentration of any given R-MCD, or variations in the combined concentrations of all R-MCDs, drive the selective retention of GA-rich mRNAs in nuclear speckles. We quantified PPIGLCD expression at the protein and RNA level and found that a 15% increase in the total abundance of R-MCDs in the cell is sufficient to induce selective mRNA retention (Extended Data Fig. Thus, modest changes in the concentration of R-MCDs can selectively control the export of mRNAs containing GA-rich GeRM regions, which encode R-MCD or other charged proteins. To assess whether the nuclear retention of GA-rich mRNAs decreases the synthesis of encoded proteins, we created codon-biased versions of a GA-rich region in LUC7L3 mRNA that encodes an R-MCD (Fig. 1c,d), such that it either had a high or low GA multivalency but equivalent GC content. We then cloned this region as a 3′ untranslated region (UTR) sequence downstream of a CDS encoding mGreenLantern. We transfected each of these two constructs into our mScarlet–PPIGLCD reporter cell line, induced PPIGLCD expression for 16 h, and measured the intensity of mScarlet and mGreenLantern in each cell. We found that as the expression of mScarlet–PPIGLCD increased, expression of mGreenLantern strongly decreased when the 3′ UTR of mGreenLantern contained a highly multivalent GA-rich sequence, whereas the effect was significantly weaker when the sequence was codon biased to decrease the GA content (Extended Data Fig. Thus, modestly increased concentration of an R-MCD protein decreases the synthesis of other R-MCD proteins due to the broad effect on the retention of endogenous GA-rich mRNAs. We refer to this mutual homeostatic co-regulation of LCD-containing proteins as interstasis. These proteins are enriched in IDRs, which are longer than average and have increased densities of charged amino acids (Fig. 4c) and to a lesser extent also non-charged polar amino acids (Extended Data Fig. Moreover, their genes are enriched for ontology terms related to the nuclear speckle, chromatin remodelling, RNA binding and RNA processing (Extended Data Fig. To account for the strong enrichment of Gene Ontology terms related to nuclear compartments, we compared all proteins that localize to speckles, nucleoplasm or nuclear bodies as defined by the Human Protein Atlas16 to the subset of these proteins that are co-regulated through interstasis, and found that these have extremely high condensation potential as predicted by the FuzDrop model17 (Extended Data Fig. Proteins encoded by genes that showed significant nuclear mRNA retention upon expression of PPIGLCD are compared with proteins from expressed genes that did not exhibit retention (n = 6,543 and 531, respectively). b, Proportions of the same proteins as in a predicted to be disordered by AlphaFold2. c, DosPS scores for the same proteins as in a. d, ROC curve (purple) indicates the mean performance of the balanced random forest model, trained on the indicated features, across fourfold cross-validation (grey curves) in distinguishing between nuclear-retained mRNAs and mRNAs with unchanged nuclear:cytoplasmic ratios (Methods). e, Schematic illustrating how mRNP packaging effects driven by exon length could influence assembly of interstasis-promoting RBPs. g, The nuclear:cytoplasmic abundance ratio of reporter transcripts depending on their sequence multivalency and different lengths of expression induction via doxycycline. The distribution of ratios is shown via a ridgeline plot, whereas the individual group means for each replicate are shown as a dot plot (n = 3 independent replicates). h, The nuclear:cytoplasmic abundance ratio of reporter transcripts over time depending on the number of exons and their multivalency (n = 3 independent replicates). Precise P values can be found in the Source Data. Black dashed lines represent no change between conditions. Genes encoding IDR-containing proteins were reported to be more dosage sensitive in yeast, Drosophila melanogaster and Caenorhabditis elegans1. This identified a striking trend for bidirectional dosage sensitivity of the genes encoding nuclear proteins that are under interstatic co-regulation, with high pHaplo and pTriplo scores that correspond to haploinsufficiency and triplosensitivity (Extended Data Fig. Furthermore, bidirectional dosage sensitivity was confirmed by gnomAD scores, showing higher mutational constraint of genes under interstatic regulation than controls19 (Extended Data Fig. Thus, interstasis enables homeostatic dosage co-regulation of genes that are bidirectionally dosage sensitive and mutationally constrained. Finally, the genes under interstatic co-regulation had higher scores by the dosage sensitivity-based phase separation predictor model (DosPS)20, confirming the combined trends for strong dosage sensitivity and the high condensation potential of their proteins (Fig. We thus propose that interstasis protects from the toxicity that can arise from deregulation of the most dosage-sensitive genes through its collective homeostatic regulation of the most condensation-prone proteins. The packaging and compaction of mRNPs have been recognized as important steps in mRNA maturation and export21,22. Previous work has shown that both mRNA sequence and gene architecture can influence mRNA export, but the relationship between sequence-specific RBP assemblies and mRNP packaging remains incompletely understood23,24,25. We reasoned that EJC deposition upstream of exon–exon junctions could limit the accessibility of multivalent sequences, preventing interstatic regulation. To assess this, we trained a random forest classifier to predict transcripts that become retained in the nucleus after 12 h of PPIGLCD expression. The input features of the model included total CDS length, mean exon length (inversely proportional to EJC density) and multivalent RNA features. We trained an array of models using a stepwise feature-selection approach to find that a combination of four features — CDS length, mean exon length, GA-rich and CAG-rich multivalency — led to a model with the best predictive performance (area under the receiver operating characteristic curve (AUROC) = 0.87; Fig. All of these features individually contributed to mRNA responsiveness; CDS length (AUROC = 0.83) was the strongest determinant of responsiveness, whereas GA-rich and CAG-rich multivalency, as well as mean exon length, further improved the performance of the model (Extended Data Fig. Furthermore, when we categorized mRNAs based on whether 50% of their GA-rich multivalency was within 50 nt of the expected EJC-binding region, GA-rich multivalent sequences were less capable of inducing nuclear mRNA retention upon PPIGLCD overexpression when they were proximal to the EJC (Extended Data Fig. To address the importance of EJC density and sequence multivalency in the absence of variable CDS length, we designed a novel reporter system in which mScarlet–PPIGLCD is encoded with variable codon choices and variable exon number, whereas its CDS length remains constant. This allowed us to assess whether EJC density could limit the accessibility of multivalent RNA for RBP assembly (Fig. The reporter plasmid pool was generated by Gibson assembly of eight fragments in sequence, each of which varied in two ways: having GA-rich or GA-poor codons, and either containing or lacking an intron, in which intron splicing efficiency was maximized using in silico mutagenesis to maximize SpliceAI scores26. The combination of these two variables leads to four possible options for each fragment of the CDS (Fig. This generated a library of reporter sequences that all produced the same protein but differed at the RNA level in their degree of GA-rich multivalency and in the number of constitutive introns (Fig. As previous work has implicated transcript GC content in the steady-state nuclear–cytoplasmic abundances, we ensured that the GC content of the CDS was kept constant25. Owing to the constraints on GC content and splicing efficiency, the most highly multivalent sequence was marginally less multivalent than the native RNA sequence that encodes PPIGLCD, whereas the least multivalent sequence was still more multivalent than most random CDSs of the same length (Supplementary Fig. Gibson assembly yielded a pool of approximately 45,000 uniquely barcoded plasmids as identified by targeted sequencing (Supplementary Fig. 2c), with a comparable representation of all possible fragments (Supplementary Fig. Sixteen hours after transfecting this plasmid pool, we isolated the RNA and confirmed with long-read sequencing that all the introns in the reporter were efficiently spliced (Supplementary Fig. Given the roles of nuclear speckles in enhancing splicing efficiency27, this enabled us to rule out inefficient splicing as a confounding factor influencing RNA localization. Next, we collected RNA from nuclear and cytoplasmic fractions at 16 h post-transfection and performed targeted sequencing of the plasmid barcodes. To test whether the degree of nuclear retention depends on the extent of the R-MCD expression, we generated a stable inducible pooled cell line expressing the library of reporter constructs, induced expression for 4, 8 and 12 h, and sequenced the nuclear and cytoplasmic barcodes at each timepoint. The nuclear retention of highly multivalent sequences was significantly stronger at 12 h than at 4 or 8 h (Fig. We replicated this effect by transfecting the reporter plasmid pool for 8 or 24 h and sequencing the nuclear and cytoplasmic barcodes (Extended Data Fig. We also addressed the role of splicing-mediated EJC loading with our reporter system, because in the reporter pool each of the eight fragments had a chance to contain an intron, thus the total number of exons varied between reporters. If all 8 fragments included an intron, there were 9 exons between 162 and 211 nt in length, whereas most reporters had 5–6 exons. Exon number alone had a weak influence on the nuclear–cytoplasmic distribution of reporter transcripts (Extended Data Fig. However, for the more multivalent reporter transcripts, exon content had a strong effect on nuclear retention: if a reporter transcript had a small number of longer exons, it was increasingly retained over time, but if a reporter had a greater number of shorter exons, it did not accumulate in the nucleus (Fig. We observed the same effect, although weaker, when transfecting the plasmid pool, suggesting that chromatin context may promote the EJC-dependent mRNP packaging (Extended Data Fig. This confirms that multivalent GA-rich RNA sequences confer the greatest potential for nuclear retention when they are positioned inside long exons, where multivalent RBP assemblies are less able to be influenced by EJC-mediated mRNP packaging. Previous insights from the RNA binding of TDP-43 demonstrated that its condensation promotes binding to highly multivalent RNA regions, which includes binding to its own GU-multivalent mRNA that promotes homeostatic control of TDP-43 dosage28. Here we asked whether increased condensation of nuclear speckles could promote the capacity of specific RBPs to recruit GA-multivalent mRNAs into speckles and thus enable interstasis. To identify the RBPs that might bind to GA-multivalent mRNAs in interstasis, we first investigated how the GA multivalency of reporter constructs correlates with the abundance of high-affinity binding motifs for different RBPs. We obtained significantly enriched 5-mers for 79 RBPs (P < 0.05) from RNA-Bind-N-Seq experiments29,30,31 and used the abundance of these 5-mers and their Z scores to assign a binding potential score to each reporter sequence. We found that GA multivalency had the strongest correlation with the binding potential scores of nuclear speckle proteins TRA2A (r = 0.97) and TRA2B (r = 0.95; Fig. Next, we gathered public iCLIP and eCLIP datasets32,33,34,35,36 and identified RBPs that preferentially bind to multivalent regions in the CDS. To identify RBPs with preference for multivalent sequences, we determined whether RBPs preferred to bind to a motif when it was in a highly multivalent context, rather than when the same motif was surrounded by dissimilar sequences. For example, UPF1 shows no bias towards multivalency for its most bound motifs, whereas TRA2B strongly prefers to bind to its most bound motifs in a multivalent context (Extended Data Fig. This identified 34 CLIP datasets in which protein had a preference to bind its favoured motifs in a more multivalent context (Fig. The GA-rich multivalent regions were bound by a large number of RBPs that have been observed to be enriched in nuclear speckles32,37,38 (Fig. 5a, bold), including six SR proteins that preferentially assemble on GA-rich GeRM regions, with the TRA2 SR proteins showing the strongest enrichment on these regions. Datasets in bold are from proteins with known nuclear speckle localization. b, Example CLIP crosslinking profiles for SR proteins across an APP transcript, with the smoothed GeRM score, amino acid entropy and proportion of charged amino acids. c, Example TRA2B and SC35 immunofluorescence images from two independent experiments showing nuclei containing variable amounts of mScarlet–PPIGLCD. Two independent replicates are plotted in different colours, and the regression lines are plotted in dashed lines. e, EIF3A HCR-FISH in nuclei expressing mScarlet–PPIGLCD and treated with either scrambled siRNA or TRA2A-targeting and TRA2B-targeting siRNA for 48 h (three independent replicates). f, Quantification of the enrichment of the EIF3A HCR-FISH signal in the nuclear speckle over the nucleoplasm for cells expressing variable amounts of PPIGLCD and treated with either scrambled siRNA or TRA2A-targeting and TRA2B-targeting siRNA for 48 h. All pairwise statistical comparisons were performed using a Welch t-test with FDR correction for multiple testing, where *P < 0.05. Precise P values can be found in the Source Data. For example, TRA2 and several other SR proteins bind strongly and specifically to the GA-rich GeRM region that encodes the charged disordered domain of APP (Fig. We also performed iCLIP for endogenous TRA2B from mouse embryonic stem cells, which confirmed the strong enrichment around the GA-rich GeRM regions of CDSs (Extended Data Fig. 8a,b), and additionally showed that iCLIP reads were more likely to contain a splice junction when the protein was bound to a GA-rich exon, confirming that TRA2B remains bound to multivalent sites post-splicing (Extended Data Fig. TRA2 proteins are known to enhance splicing of GA-rich exons33. We therefore analysed public data in which TRA2A and TRA2B were co-depleted, which increases skipping of both alternative and constitutive exons33, but found no trend for skipping of the exons with GA-multivalent GeRM regions (Extended Data Fig. Moreover, data from VastDB39 demonstrate that the GA-multivalent exons from the mRNAs that are retained upon interstasis are highly constitutive across all available human tissues (Extended Data Fig. In addition, our randomized reporter analysis did not show any link between GA-rich exons and splicing changes of either the exons or their flanking introns during interstasis (Supplementary Fig. We conclude that binding of SR proteins to highly multivalent GA-rich exons is rarely associated with the modulation of alternative splicing. For a protein to be an effector of the interstatic dosage loop, it should be sensitive to the dose of R-MCD proteins. Indeed, TRA2B was diffuse throughout the nucleoplasm in the absence of PPIGLCD expression, with only mild colocalization with the nuclear speckle marker SC35, but became enriched in nuclear speckles 16 h after the induction of PPIGLCD (Fig. The degree of speckle localization correlated with the dose of PPIGLCD, similar to retained GA-rich mRNAs. The same dose variations of R-MCD protein induced speckle retention of TRA2B and nuclear GA-rich mRNA to a similar extent, indicating that TRA2B could be an effector that drives the sequestration of GA-rich mRNAs in the nuclear speckle. To demonstrate the role of TRA2 proteins in modulating selective purinergic mRNA sequestration, we performed short-interfering (siRNA)-mediated depletion of TRA2A and TRA2B in cells expressing PPIGLCD (Extended Data Fig. Cells were grouped by their PPIGLCD expression intensity to ensure that observed effects were not attributed to changes in PPIGLCD expression driven by TRA2 depletion itself. At lower levels of PPIGLCD expression that are sufficient to promote the retention of EIF3A mRNA in nuclear speckles in control conditions, double TRA2A/B knockdown significantly decreased speckle retention of EIF3A (Fig. When R-MCD concentrations become very high, the effect of TRA2A/B knockdown diminished, indicating that at such concentrations, other RBPs that assemble on GA-rich multivalent regions could also drive speckle mRNA retention (Fig. This suggests that TRA2 proteins are key effectors of interstasis upon modest accumulation of R-MCDs, which more probably represents physiological variation. To determine whether the R-MCD proteins or their GA-rich mRNA itself drives the relocalization of TRA2B to the nuclear speckle, we created a construct encoding mGreenLantern fused to the R-MCD of LUC7L3 (Extended Data Fig. Two versions of each construct were made with opposing codon biases in LUC7L3, to create either GA-rich or GA-poor sequences (Extended Data Fig. Upon transfection of the constructs, the LUC7L3 R-MCD protein localized to nuclear speckles, and it drove the speckle localization of TRA2B regardless of whether it was expressed from GA-rich or GA-poor sequences (Extended Data Fig. We also created a construct in which a stop codon had been inserted upstream of the GA-rich sequence that would otherwise encode the LUC7L3 R-MCD, thus making it part of the 3′ UTR, and this construct had no effect on TRA2B localization (Extended Data Fig. This confirmed that accumulation of R-MCD proteins, rather than their mRNAs, drives TRA2B relocalization, and TRA2B promotes the sequestration of GA-rich mRNA into the speckles. Localization of SR proteins within nuclear speckles is regulated by their phosphorylation via various kinases, such as the CDC2-like (CLK) family of kinases40,41. Previous work has indicated that serine phosphorylation in SR domains disrupts arginine-driven self-association in nuclear speckles10. CLK kinase activity is regulated under a diverse array of physiological contexts, including during the cell cycle and in response to small variations in temperature42,43. Given the role of TRA2, and probably other SR proteins, as mediators of interstasis, we asked whether the set point of interstasis could be modulated by the activity of CLK kinases. We treated HeLa cells for 8 h with 1 µM CLK-IN-T3, a selective inhibitor of CLK1, CLK2 and CLK3 (ref. 44), and confirmed that TRA2B and other SR proteins became less heavily phosphorylated (Extended Data Fig. We observed that CLK-IN-T3 treatment induced a strong relocalization of TRA2B to nuclear speckles, and a mild enrichment of poly-A+ mRNA in the nucleus, especially within speckles (Fig. Upon 16 h of CLK-IN-T3 treatment, speckles became larger and fewer in number (Extended Data Fig. Thus, CLK kinase inhibition produces a similar phenotype to the overexpression of an R-MCD with regards to mRNA and TRA2B localization and nuclear speckle morphology. a, TRA2B and SC35 immunofluorescence in cells treated either with DMSO or 1 µM CLK-IN-T3 for 8 h. b, Example image of SON immunofluorescence and EIF3A HCR-FISH in cells treated either with DMSO or 1 µM CLK-IN-T3 for 16 h. c, The nuclear–cytoplasmic distribution of reporter transcripts binned by their total GeRM scores. d, Schematic detailing the interstasis of proteins with charged low-complexity domains. All statistical comparisons were performed using a Welch t-test with FDR correction for multiple testing, where *P < 0.05 and **P < 0.01. Precise P values can be found in the Source Data. The black dashed line represents no change between conditions. To test the selectivity of speckle mRNA localization, we repeated HCR-FISH after 16 h of CLK-IN-T3 treatment, which significantly increased the localization of the GA-rich EIF3A and BRD4 mRNAs, but not control mRNAs, to nuclear speckles relative to the nucleoplasm (Fig. To assess the sequence-specific effect on mRNA retention using our reporter system, we treated cells with 1 µM CLK-IN-T3 or DMSO for 2 h, then induced the expression of the reporter transcript pool with doxycycline for 6 h, at which point the expression of the PPIGLCD protein is still negligible, and performed targeted sequencing of the reporter mRNAs. We found that CLK inhibition significantly increased the nuclear retention of the GA-multivalent reporter transcripts (Fig. To assess the effect on the nuclear–cytoplasmic distribution of endogenous mRNAs, we performed 3′ end sequencing of nuclear and cytoplasmic fractions after 8 h of CLK-IN-T3 treatment, which increased the nuclear retention of mRNAs with GA-rich GeRM regions in CDSs (Extended Data Fig. The extent of nuclear retention of each transcript was correlated between 8-h CLK inhibition and 12-h PPIGLCD expression (Extended Data Fig. 10k), indicating that both R-MCD abundance and CLK activity can modulate the retention of multivalent mRNAs in nuclear speckles. We report the discovery of interstasis, a collective feedback loop mediated by a protein–RNA condensate that promotes the homeostasis of a broad range of condensation-prone proteins. Most of these are proteins that interact with RNA, DNA and chromatin, which tend to be enriched in IDRs across all domains of life45. Using a novel combinatorially assembled reporter strategy to study mRNA selectivity upon interstasis, we validated the importance of codon biases in the nuclear retention of multivalent mRNAs that encode condensation-prone proteins. Our findings suggest a model in which nuclear mRNA retention is promoted by multivalent assemblies of specific RBPs that localize to nuclear speckles, whereas EJC-dependent packaging counteracts such mRNA capture. We showed that upon increasing the dosage of condensation-prone proteins in speckles, TRA2B relocalizes to speckles, acting as a dosage effector that promotes the capture of multivalent mRNAs (Fig. By managing the most condensation-prone nuclear proteins that tend to localize into speckles, interstasis probably protects speckles from reaching the toxic levels of cohesion that could non-specifically capture mRNAs10. Thus, the properties of a condensate can serve as the long-sought mechanistic link between dosage sensing of IDR-containing proteins and dosage effectors that control their gene expression. The multivalency of CDS regions encoding IDRs emerges partly due to the structure of the genetic code, that is, the high similarity of codons for biophysically similar amino acids, which tend to cluster within IDRs. Moreover, we found that codon biases further increase such RNA multivalency, especially in the transcriptomes of land vertebrates. All classes of multivalent CDS regions contain strong codon biases that reinforce multivalency, and each class is bound by specific RBPs and encodes IDRs within functionally coherent sets of proteins, indicating a broad potential for their co-regulation. Although autologous interaction between RBPs and their own mRNAs have been linked to affinities between amino acids and their cognate codons46,47, broad feedback regulation of a class of related IDRs and their mRNAs, or the role of codon biases in reinforcing such feedback have not been previously observed. Understanding how codon biases might enable feedback regulation between specific sets of proteins and their multivalent mRNAs has implications for various fields, such as interpreting the effect of synonymous mutations on human diseases48, and for the design of synonymous gene-recoded therapeutics49. Nuclear retention of several spliced mRNAs has been identified in human tissues50, and phosphorylation of SR proteins has been shown to promote speckle mRNA retention; however, the selectivity of such retention remained unclear51. We found that the activity of CLK family kinases modulate selective speckle retention of highly multivalent, GA-rich mRNAs. Eight-hour inhibition of CLK kinases phenocopies the same selectivity of mRNA retention as the increased R-MCD expression, as it leads to hypophosphorylation of SR proteins such as TRA2B, which we found to be a key effector of the GA-rich mRNA sequestration into nuclear speckles. CLK signalling is responsive to various physiological states, including cell-cycle progression and small variations in temperature42,43, and is investigated as a therapeutic target for many diseases40, indicating a broad potential for altered signalling to modify the threshold of interstasis in specific cellular contexts. It has been proposed that an additional function of endogenous condensates could be to buffer cellular noise due to their concentration-dependent assembly52, but it remained unclear how heterotypic interactions between diverse sets of proteins and RNAs could contribute to such buffering53. We now identify such a function for nuclear speckles, which serve as collective sensors of the combined dosage and modifications of co-condensing proteins. By capturing the mRNAs encoding these proteins, speckles promote the dosage homeostasis of proteins that are expressed from some of the most bidirectionally dosage-sensitive and mutationally constrained genes. It remains to be seen whether nuclear speckles thereby help to protect proteostasis during conditions that affect the relative concentrations of proteins in nucleoplasm, or in diseases with aberrant protein condensates, thereby complementing known pathways for protein quality control that primarily sense cytoplasmic disturbances54. Many age-related diseases, especially in neurodegeneration, involve aberrant protein–RNA condensates that can eventually promote aggregation of IDR-containing proteins4,55,56, and are often caused by low-complexity repeat expansions that can involve speckle recruitment of the repeat RNA57,58. Understanding the mechanisms that manage the homeostasis of proteins in condensates could open new therapeutic opportunities for these diseases55. GeRM is calculated from a string of consecutive overlapping nucleotide sequences of length k (k-mers). where \(w=\frac{n+1}{2}\), the function d represents the Hamming distance, and λ represents a scaling constant. In non-mathematical terms, the GeRM score is calculated by comparing a k-mer to all the other k-mers that surround it in a fixed window. For each of the surrounding k-mers, the sequence similarity to the central k-mer is calculated from the negative exponent of the Hamming distance, such that k-mers with identical sequences have a high score and those with unrelated sequences have a low score. The constant λ determines how quickly this similarity score decays as sequences become more dissimilar to the central k-mer. This sequence similarity score is multiplied by a distance score, which decays linearly from 1 to 0 with distance from the central k-mer. k-mers that overlap with the central k-mer are ignored. For the calculation of codon-shuffled multivalency, all codons for a given amino acid in each transcript were swapped such that the number of instances of each codon was preserved for each transcript, but their order was randomized. All GeRM scores for each transcript were calculated, and the mean GeRM score per position across ten shuffles was calculated. The GeRM algorithm is available as an R package from GitHub (https://github.com/ulelab/germ). For each protein-coding gene in GENCODE 29 or GENCODE M22, the transcript with the longest CDS was selected and ties were broken by the longest total transcript length. Only spliced transcripts were used for analysis. CDS GeRM regions were defined by taking all positions where the smoothed GeRM scores exceeded the 98th percentile of smoothed GeRM scores within the CDS, and GeRM regions were adjusted to contain all k-mers that fell within the smoothing window. GeRM regions with at least 41 nt of overlap were merged. GeRM regions for which at least one-third of the region fell within an untranslated region were excluded. For each GeRM region, the relative proportion of the total multivalency of the region that was accounted for by each possible k-mer was calculated. For clustering, UMAP was used to reduce the dimensionality of the data to 4 dimensions, using 50 neighbours and a minimum distance of 0.001. The final clustering was performed with DBSCAN, yielding eight clusters. GeRM regions falling outside of a cluster were excluded from further analysis. For presentation purposes, UMAP was used to reduce the original data to have two dimensions, with other parameters kept the same. Codon shuffling was performed by shuffling the codons either within the entire transcriptome or within each transcript. Shuffling was performed five times, and GeRM scores were calculated from the shuffled sequences. The mean GeRM score per k-mer across the five shuffles was used for further analysis. To generalize the scoring of identified multivalent types (that is, the GA-multivalency score and transcript multivalency), we summarized each GeRM cluster with the k-mers that contributed most to multivalency of regions in that cluster. The proportion of total multivalency accounted for by each 5-mer was calculated, and 5-mers were sorted in descending order based on this proportion. The cumulative sum was calculated, and all 5-mers that accounted for the first 50% of the multivalency in each GeRM cluster were selected as representative 5-mers for this multivalency type. To define the complexity of amino acid sequences, information entropy was calculated using the R package HDMD in a sliding window of 41 amino acids along the translated CDSs from the set of transcripts used for GeRM scoring. LCDs were defined as regions with entropy in the bottom 2% of all entropy scores, with all amino acids in the sliding window being considered part of the LCD. AlphaFold59 predicted local distance difference test (pLDDT) scores were obtained for the same set of proteins where available from UniProt. First, ‘mutable positions' were defined as any position that could be synonymously mutated. Primarily, these are the final position of the codon, but the first positions of some leucine and arginine codons can also be synonymously mutated. For each mutable position, sequences were produced in which every possible synonymous mutation was made. When this ratio is positive, the native nucleotide at the mutable position is associated with higher GeRM than the other possible mutations that could exist at that position on average. PhyloP conservation across 100 vertebrates and 470 mammals in hg38 coordinates were downloaded from the UCSC Golden Path. These values were converted into transcriptomic coordinates based on the single transcripts per genes that were used for GeRM calculation. Mutable positions were binned based on their GeRM codon ratio with the binning calculations being applied to each codon separately, such that each bin contained an equal proportion of each codon. For raw conservation analysis, the average PhyloP conservation values of mutable positions in each GeRm codon ratio bin were used. For analysis of normalized conservation values, the PhyloP of mutable positions were normalized for each codon by subtracting the median conservation of that mutable position in that codon by the median conservation across all mutable positions of the same type. For example, the conservation value associated with one instance of the A in the codon CGA would be normalized by subtracting the median conservation of all As in CGA codons. Subtraction was used for normalization because PhyloP scores operate on a logarithmic scale. For analysis of arginine codon usage in R-LCDs, all human LCDs were filtered for those that obtain at least 20% arginine residues. Pairwise correlations between arginine codon usages using Spearman's rank correlation. To define CG-rich and GA-rich codon groups, principal component analysis was performed based on the arginine codon usage for all R-LCDs, and the top and bottom third of LCDs based on the first principal component were defined as CG-rich and GA-rich R-LCDs, respectively. For the definition of R-LCDs in different species, all CDSs for each species were downloaded from Ensembl and a single transcript per gene was identified according to the previously described approach. LCDs encoded by these sequences were defined as previously described, using the complexity threshold defined for the human proteome for each species. A full list of the species and annotation versions can be found in Supplementary Table 1. HeLa cells were obtained from Cell Science at The Francis Crick Institute and cultured in DMEM + GlutaMAX (Thermo Scientific) with 10% FBS and split every 3–4 days using TrypLE Express (Thermo Scientific). Low-passage, wild-type mouse embryonic stem cells (mESCs; 129S8/B6 background) were obtained from the Genetic Modification Service at The Francis Crick Institute and cultured feeder free on Nunc cell culture dishes (Thermo Scientific) coated with 0.1% gelatin (ES-006-B, Millipore). mESCs were maintained in 2i media60, fed every day and split every 2–3 days using Accutase (A6964, Sigma). All cell lines were sourced from, authenticated by and mycoplasma tested at The Francis Crick Institute. All cell transfections were performed using Lipofectamine 3000 (Thermo Scientific) according to manufacturer's instructions. Doxycycline induction was performed by changing cell culture media to fresh, pre-warmed media containing 100 ng ml−1 doxycycline. CLK-IN-T3 (SML2649, Sigma) was dissolved to 5 mM in DMSO. In experiments using CLK-IN-T3 treatments, control cells were treated with the equivalent volume of DMSO. Double-stranded DNA fragments were ordered as gBlocks or eBlocks from IDT. All PCRs utilized Q5 High-Fidelity Master Mix (NEB) or Phusion High-Fidelity Master Mix (NEB). Backbone plasmids were linearized via PCR followed by treatment with DpnI (NEB) before assembly with fragments using NEBuilder HiFi DNA Assembly (NEB) according to the manufacturer's protocol. Assembly reactions were purified using magnetic SPRI beads (Mag-bind TotalPure NGS; Omega-Bio-tek) before DNA transformation into 5α competent Escherichia coli (NEB). All sequences were confirmed via Sanger sequencing (Source Bioscience). pTwist-AMP high-copy plasmid was purchased from Twist Bioscience and was used as a vector for all overexpression via transient transfection. Sequences of all LUC7L3 constructs can be found in Supplementary Table 5. For the three additional R-MCD plasmids, the following regions of LUC7L3221–415, PRPF38B251–534 and SRSF11171–389 were fused to mGreenLantern and assembled into pTwist-CMV plasmid backbones. Nuclear–cytoplasmic fractionation was carried out by trypsinizing whole cells and washing twice with PBS. Cells were then pelleted and resuspended in 200 µl cytoplasmic lysis buffer: 50 mM HEPES pH 7.5, 2 mM MgCl2, 50 mM 2-mercaptoethanol, 0.025% NP-40, 0.05% saponin and 1X cOmplete EDTA-free protease inhibitors (Sigma). Cells were rotated at 4 °C for 10 min, pelleted and the supernatant was taken as the cytoplasmic fraction. The nuclear pellet was washed in 1 ml cytoplasmic lysis buffer for 3 min, then pelleted again. The supernatant was removed and the nuclei were lysed in 200 µl iCLIP lysis buffer61 to obtain nuclear fractions. All buffers were kept ice cold. For each fraction, 20 µl was used for western blotting to confirm the efficiency of the fractionation, which was stable across all samples (Extended Data Fig. In all western blots, vinculin was used as a loading control (1:2,000; 700062, Thermo Scientific). RNA extractions were performed from cell pellets using the Maxwell RSC simplyRNA kit (Promega) using a Maxwell RSC Instrument (Promega). For each sample, 250 ng µl−1 of RNA was fragmented in 10 mM Tris-HCl pH 7.5 and 10 mM MgCl2 buffer for 5 min at 95 °C. Of fragmented RNA, 2 µl was mixed with 2 µl water, 0.5 µl of 10 mM dNTPs and 0.5 µl of 5 µM of oligo-dT reverse transcription primers containing Illumina P7 sequences, and primers were annealed by heating at 65 °C for 3 min then cooling to 42 °C at 1 °C per second. At this point, reverse transcription was carried out using SuperScript IV (Invitrogen) according to manufacturer's instructions, with the addition of 0.25 µl of 40 µM of a template-switching oligo containing Illumina P5 sequences and unique molecular identifiers (UMIs). The reaction was incubated for 1 h at 42 °C. Following alkaline hydrolysis of RNA, the product was purified using magnetic SPRI beads and amplified using 0.5 µM i5/i7 Illumina indexed primers in Q5 High-Fidelity Master Mix (NEB). All sequencing data were mapped to the GRCh38.p12 genome using the GENCODE v29 basic genome annotation. All 3′ end sequencing data were trimmed using cutadapt (v4.4)62 to remove poly-A sequences and Illumina adapters. All data were processed using the nf-core RNA-seq Nextflow pipeline (v3.12.0)63, and in the case of the 3′ end sequencing data, the additional option --noLengthCorrection was provided to Salmon to prevent length correction for gene expression. Differential expression analysis was conducted using DESeq2 in R64, with effect size shrinkage using the apeglm package65. To classify genes based on their degree of GA multivalency, a GA-multivalency score was calculated for each gene based on the CDS of its primary transcript, defined by the longest annotated CDS, with ties settled by total transcript length. First, for each of the three previously identified GA-rich GeRM clusters (A-rich and G-rich AG, AG + C), representative k-mers were derived as described in the section ‘Identification and classification of GeRM regions'. Then, raw multivalency scores of 5-mers were scaled to all other instances of the same 5-mer, such that their mean score was 0 and their standard deviation was 1. For each transcript, the sum of all these scaled GeRM scores was summed across all representative 5-mers. On the basis of this metric, transcripts that have many instances of GA-rich 5-mers specifically in multivalent contexts are rewarded. All processed data from 3′ end sequencing experiments can be found in Supplementary Table 4. Cells were grown for 48 h in lysine-free and arginine-free DMEM (Thermo Scientific), supplemented with 10% dialysed FBS (Thermo Scientific), 600 mg l−1 proline and 100 mg l−1 arginine and lysine (‘light' media). Cells were induced with 500 ng ml−1 doxycycline for 12 h before switching to either medium arginine 13C6 (88210, Thermo Scientific) and lysine 4,4,5,5-D4 (DLM-2640, Cambridge Isotope Laboratories; ‘medium' media) or heavy arginine 13C6 15N4 (89990, Thermo Scientific) and lysine 13C6 15N2 (88209, Thermo Scientific; ‘heavy' media) for 8 h. Media were refreshed every 2 h, with doxycycline treatment maintained throughout. Cells were washed with PBS and lysed in 4% SDS, 100 mM Tris-HCl pH 7.5 buffer supplemented with protease inhibitors (Sigma). Samples were next sonicated, and their protein concentrations quantified via DC protein assay (Bio-Rad). Lysates were reduced by the addition of dithiothreitol to a final concentration of 5 mM, vortexed briefly and incubated at 56 °C for 30 min. Samples were cooled to room temperature and alkylated with iodoacetamide to a final concentration of 20 mM. Following brief vortexing and incubation in the dark at room temperature for 20 min, residual iodoacetamide was quenched with 5 mM dithiothreitol for 10 min. Trypsin was added at a 1:100 enzyme:substrate ratio. Peptide samples (1 µg on column) were analysed by liquid chromatography with tandem mass spectrometry (LC–MS/MS). Chromatographic separation was performed using the U3000 UHPLC NanoLC system (Thermo Scientific) and peptides were resolved by reversed phase chromatography on a 75-µm C18 Pepmap column (50 cm length) via a three-step linear gradient comprising 80% acetonitrile in 0.1% formic acid. The gradient was delivered to elute peptides at a flow rate of 250 nl min−1 over 120 min, initially at 4% solvent at 0–10 min, then increasing to 30% solvent (10–75 min) and 40% solvent at 75–80 min. This was followed by a wash step with 99% solvent (85–90 min) and a final equilibration step with 4% solvent (90–120 min). Peptides were ionized by electrospray via an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific) operating under Xcalibur (v4.3). Data were acquired in data-dependent acquisition mode using an Orbitrap-Ion Trap method with a 3-s cycle time between a full MS scan and MS/MS fragmentation by collision-induced dissociation. Orbitrap spectra were collected at 120,000 resolution over an m/z range of 350–1,600, with an automatic gain control of 4.0 × 105 (100%) and maximum injection time of 35 ms. Monoisotropic precursor selection was enabled for charge states +2 to +5 with an intensity threshold of 5 × 103–1 × 1020 and dynamic exclusion of 35 s ± 10 ppm. MS2 precursor ions were isolated in the quadrupole with a mass width filter of 1.6 m/z. Ion trap fragmentation spectra were collected with an automatic gain control target setting of 1.0 × 104 (100%) with a maximum injection time of 35 ms with collision-induced dissociation collision energy set at 35%. Raw mass spectrometry data were processed into peak list files using Proteome Discoverer (v3.1; Thermo Scientific) and searched with the Sequest algorithm67 against the Uniprot Human isoform database (188,476 entries; September 2024). This was concatenated with the mScarlet fluorescent protein sequence. Database searching was performed at a stringency of 1% FDR including a decoy search with precursor ion intensity quantification. Post-translational modifications to amino acid residues included carbamidomethylation (C) and variable oxidation (M), SILAC 13C6 (R) and SILAC 2H42 (K). The proportion of SILAC-labelled peptides for each protein was compared between doxycycline-treated and untreated samples to determine the change in translation or protein turnover. The mean log2 fold change for all proteins belonging to different multivalency classes in each replicate was calculated and this per-replicate mean was used for pairwise significance testing. The proportion of the total proteome or total R-MCD abundance attributed to mScarlet–PPIGLCD overexpression was calculated using the peptides assigned to mScarlet, and R-MCDs were classified as previously defined R-LCDs (those containing at least 20% arginine), with the additional constraint that the net charge must be positive and the total fraction of charged residues in the LCD must be greater than 40%. Immunofluorescence was performed by fixing cells in 4% paraformaldehyde and 0.1% glyoxal at room temperature for 10 min. Samples were permeabilized by incubating with PBS with 0.5% Triton-X for 5 min at room temperature. Samples were blocked using PBS with 3% BSA and 0.1% Tween (blocking solution). Primary antibody incubations were carried out in blocking solution for 1 h at room temperature or overnight at 4 °C. Samples were washed three times with PBS with 0.1% Tween for 5 min, and secondary antibody incubations were carried out in blocking solution for 1 h at room temperature in the dark. Samples were washed twice, then incubated with 200 ng ml−1 DAPI for 15 min in the dark at room temperature. DAPI was washed out and samples were placed in glycerol mounting medium (90% glycerol, 20 mM Tris pH 8 and PBS). The following antibodies were used at the following concentrations: rabbit anti-TRA2B (1:1,000; ab31353, abcam), rabbit anti-SON (1:1,000; HPA023535, Sigma) and mouse anti-SC35 (1:500; S4045, Sigma). Secondary antibodies to mouse and rabbit, conjugated to Alexa Fluor 488 or Alexa Fluor 647 were used at 1:500 dilution (Abcam). This included goat pAb anti-rabbit IgG Alexa Fluor 647 (ab150079), goat pAb anti-rabbit IgG Alexa Fluor 488 (ab150077), goat pAb anti-mouse IgG Alexa Fluor 488 (ab150113) and goat pAb anti-mouse IgG Alexa Fluor 647 (ab150115). For each mRNA, HCR 12 probe pairs were designed with the B4 amplifier sequences using the HCR 3.0 Probe Maker68. HCR 3.0 was performed on cells in µ-Slide eight-well glass bottom slides (80827, Ibidi) as described in the original protocol69, with the following modifications. First, fixation and permeabilization was performed as described in the immunofluorescence section. After pre-hybridization, primary probe hybridization was performed with a probe concentration of 10 nM for 3 h. Overnight HCR amplification was performed in a volume of 125 µl with half the concentration of HCR hairpins (Molecular Instruments). Following HCR amplification, samples were washed according to the original protocol, and then immunofluorescence was performed as previously described, but with all buffers containing 2XSSC in place of PBS. In all cases of R-MCD overexpression transfections, HCR-FISH was performed on HeLa cells 24 h post-transfection with 50 ng of mGreenLantern-fused R-MCD plasmid, in combination with 200 ng of pUC19 filler DNA. Samples were incubated with HCR 3.0 hybridization buffer for 30 min at 37 °C, then in hybridization buffer containing 1 µg ml−1 oligo-dT(25)-Cy5 for 2 h. Samples were washed twice in 5XSSC with 0.1% Tween, stained with 200 ng ml−1 DAPI in the same buffer and washed once more before glycerol mounting medium was added. Microscope images were acquired using an Olympus IX3 Series (IX83) inverted microscope, equipped with a Yokogawa W1 spinning disk and a Hamamatsu Orca Fusion CMOS camera (pixel size of 6.5 μm, 2,304 × 2304, 5.3 megapixels). All images were z-projected using a maximum projection. Segmentation of nuclear speckles was performed using CellProfiler using the Otsu thresholding algorithm, and nucleoplasms were considered all non-speckle regions of the nucleus. The mean signal intensity for HCR-FISH or immunofluorescence within speckles was compared with the mean intensity in the nucleoplasm for each cell to create a speckle enrichment ratio. Mean intensities in the nucleoplasm and nuclear speckle were calculated per cell, but statistical comparisons between groups counted each replicate as a single observation, taking the mean values across all cells in the replicate. Using predictive modelling, we aimed to evaluate the contribution of CDS length, EJC density and different types of multivalent features on nuclear retention of endogenous transcripts in interstasis. Representative k-mers were assigned for each of the eight GeRM clusters as described in the section ‘Identification and classification of GeRM regions'. Subsequently, k-mers belonging to multiple clusters were retained only in the cluster to which they contributed the highest proportion of multivalency, ensuring no k-mers overlapped between clusters. Next, the raw k-mer multivalencies for all k-mers belonging to a cluster were summed across CDSs of representative transcripts, defined as the transcript with the longest CDS per gene, and normalized by CDS length. These length-normalized multivalency sums were then standard scaled across the transcriptome so that the mean transcriptome-wide multivalency of each GeRM cluster was zero. Given this high correlation and the similarity of their k-mers, we combined these clusters into a single GC-rich cluster for training. This approach efficiently minimized collinearities between features, except for a moderate anti-correlation (Spearman's r = −0.75) between the GC-rich and A-rich clusters. Features used for training are summarized in Supplementary Table 8. Then, we trained a classification model to differentiate between mRNAs retained in the nucleus after PPIGLCD induction and control mRNAs that did not show a significant change in their nuclear-to-cytoplasmic ratio under these conditions. mRNAs were classified as retained if they had a q < 0.05 and a log2 fold change greater than 1 at 12 h post-PPIGLCD induction, resulting in 275 transcripts. Control mRNAs were defined by a q ≥ 0.05 and a log2 fold change between −0.1 and 0.1, yielding 1,697 transcripts. The classifier was configured with 100 trees (n_estimators = 100) and used bootstrap sampling with replacement to enhance generalization. Gini impurity criterion was used for node splitting and the minimum number of samples required at a leaf node was set to 15. For evaluation of the classifier's performance, we used fourfold stratified cross-validation. This method splits the data into fourfolds, with each fold serving as a test set once while the remaining threefolds serve as the training set. Stratified k-fold cross-validation ensures that each of the folds maintains the same proportion of positive and negative samples as the entire dataset, thereby preventing skewed performance metrics. In this case, the model was trained four times, with each training set comprising 75% of the data and each test set comprising 25%, ensuring balanced class representation in both training and testing. Visualized AUROC metrics were averaged across folds to obtain a robust estimate of the classifier's performance. To identify the optimal set of input features and assess their effect on predictive performance, we used a stepwise feature-selection approach. Initially, we trained models with individual features and progressively expanded the feature set, by selecting the best-performing single feature or feature combination and adding the remaining features on top of it. This process was repeated until adding more features no longer enhanced the performance of the model. All models were evaluated using fourfold cross-validation, as described. Untranslated regions were added to this sequence. Seven equally spaced sequences of 28 nt were chosen along the sequence to be used as overlap sequences for future Gibson assembly. These overlap sequences were protected from future in silico mutagenesis. Twenty-six constitutively spliced introns ranging from 83 nt to 384 nt in length were selected from housekeeping genes. We generated 25 different combinations by randomly choosing 8 of these introns, then performed the following iterative optimization algorithm to each of these 25 combinations to generate sequences with optimized predicted splicing. First, introns were inserted in the reporter CDS in positions that were exactly between the previously defined overlap sequences or the start and end codons. The probability of donor and acceptor site usage was predicted using SpliceAI, and all probabilities were summed. Next, 100 coding sequences with 5 random synonymous mutations were generated, and a sequence was selected that best maintained GC content and kept GA 5-mer content low. After this, an intronic sequence was selected with a 80% chance to be mutated at 5% of its positions and a 50% chance to be shifted by up to 20 nt. If the total probability of splice site usage was increased in the mutated sequence, then the mutated sequence was put forwards for another round of mutation, until 2,000 iterations were completed. To create a GA-rich version of the reporter CDS, the iterative process was repeated, but with synonymous mutations that maintained GC content but maximized GA-rich 5-mer content. To determine which of the 25 possible combinations of 8 introns resulted in the best-predicted splicing in both GA-poor and GA-rich reporter sequences, we selected the intron set with the highest summed probability of predicted donor and acceptor site usage. This resulted in GA-poor and GA-rich codon-biased reporter sequences in which both splice sites for each of the eight introns was predicted to have at least a 98.5% probability of being spliced. For both the GA-rich and the GA-poor sequence, eight sequence segments were generated that spanned from each overlap sequence to the next (or to the untranslated region) and that either contained an intronic sequence or not. Therefore, each of the eight segments of the final reporter gene varied in two possible ways: GA richness and the presence or absence of an intron, and each of these variable segments shared a fixed overlap sequence with the neighbouring segment. All 32 possible segment variations were ordered as double-stranded DNA fragments from IDT. In addition to the overlap sequences that would be part of the final reporter transcript, additional overlap sequences were added for intermediate assembly steps. Then, two pools containing all variations of the first four segments or all variations of the last four segments were made, such that intron-containing variants had twice the molarity of intronless variants at each position, but all positions had the same overall molarity. A Gibson assembly reaction was carried out on each pool, and due to the additional overlaps on the first, fourth, fifth and final segments, circular pieces of double-stranded DNA were formed, each containing one of the four variations at each of the four positions used for that pool. These circular pieces of DNA were then linearized and amplified by PCR, and the linearized fragments deriving from the first and second half of the reporter gene were again Gibson assembled together. After each reaction, the product was purified using Mag-Bind TotalPure beads. All PCRs were performed using Q5 High-Fidelity Master Mix (NEB). All Gibson assemblies were performed using NEBuilder HiFi DNA Assembly (NEB). To generate a doxycycline-inducible PiggyBac integration vector containing the reporter library (ePB-ExportReporter), reporter genes and their barcoded UTRs were amplified out of the original reporter plasmid library with a low number of PCR cycles, before being inserted into an all-in-one PiggyBac plasmid (ePB-Bsd-TT-NIL) containing a doxycycline-inducible promoter as well as a constitutive promoter driving the expression of the rtTA protein and blasticidin resistance gene. All designed fragments, as well as example reporter sequences and plasmids, are provided in Supplementary Table 2. For sequencing of the plasmid pool, the reporter genes were amplified from the plasmid DNA with a minimal number of PCR cycles. For sequencing of the reporter RNA, one well of a six-well plate of HeLa cells were transfected with 500 ng of the reporter plasmid pool and 2,000 ng of a filler pUC19 plasmid for 16 h. RNA was extracted and 500 ng of total RNA was used as input for a SuperScript IV reverse transcription (Invitrogen) reaction using an oligo-dT reverse transcription following the manufacturer's instructions. In both cases, the PCR product was purified with magnetic beads (Mag-bind TotalPure NGS, Omega-Bio-tek) and Nanopore adapters were added using the Nanopore Ligation Sequencing Kit (Oxford Nanopore), according to the manufacturer's instructions. The libraries were sequenced separately using MinION R9.4.1 Flow Cells (Oxford Nanopore). Raw Nanopore data were basecalled with Guppy (v6.0.1). The basecalled reads were mapped to a genome of possible reporter transcripts using minimap2 (ref. 72), and barcodes were extracted by matching the upstream and downstream sequences using the R package stringdist73. Targeted sequencing libraries were prepared from 500 ng of RNA input per sample. A targeted reverse transcription was performed with SuperScript IV (Invitrogen) using a reverse transcription primer that was specific to the reporter 3′ UTR and contained a 4-nt experimental barcode, a 10-nt UMI and an Illumina p7 adapter sequence. The reaction was performed according to manufacturer's instructions. Samples were multiplexed in groups of six at this point, and the RNA was removed by alkaline hydrolysis for 15 min, the pH was neutralized with hydrochloric acid, and the cDNA was purified with magnetic beads. A nested PCR was then performed in which ten cycles with one set of primers was performed. Of the reaction, 5% was spiked into the next six-cycle reaction, in which an Illumina p5 adapter sequence was added. The PCR was purified with magnetic beads (Mag-bind TotalPure NGS, Omega-Bio-tek), and Illumina i5 and i7 indices were added by a final PCR. Libraries were sequenced on a NovaSeq 6000 by the Advanced Sequencing Facility at The Francis Crick Institute. All primer sequences can be found in Supplementary Table 2. Targeted sequencing libraries were analysed in R. Libraries were demultiplexed using the barcodes introduced by the reverse transcription primer, and the plasmid barcode was extracted by matching the flanking constant regions using fuzzy string matching using the R package stringdist. Reads containing the same barcode were deduplicated by their UMI sequences. Barcodes were counted and related to their reporter gene characteristics based on the long-read sequencing data. Barcode counts were normalized to the total number of assigned barcodes for each sample. Barcodes with low numbers of counts were excluded from further analysis. All processed data for each experiment can be found in Supplementary Table 3. We cloned the reporter pool into a doxycycline-inducible PiggyBac vector and generated the PiggyBac cell lines by co-transfecting HeLa cells with equal amounts of the ePB-ExportReporter plasmid pool and a plasmid expressing the PiggyBac transposase. A transposase-negative control was also prepared. Cells were selected once with 5 µg ml−1 blasticidin for 4 days, then expanded for 1 week, before being selected again with blasticidin until all cells in the transposase-negative control were dead. This pool of ePB-ExportReporter-positive cells was then expanded and stocks were frozen. To understand which RBPs have the potential to bind to GA-multivalent reporters with high affinity, we designed a ‘binding potential' score, based on RBP-binding motifs obtained with the RNA-Bind-N-Seq method. We downloaded enrichment scores (R scores) for 78 RBPs published by Dominguez et al.30, and for TRA2B32 from the ENCODE portal, yielding motifs for 79 distinct RBPs. Accession codes and relevant concentrations of RNA-Bind-N-Seq experiments are specified in the supplementary material of the original publication (see supplementary table 3 in ref. The R scores for the RBP concentration that yielded the highest motif enrichment were converted to z scores using mean and standard deviation. A tail test was then applied to compute P values, and motifs with a P < 0.05 were selected for further analysis. To evaluate the reporter sequences in terms of RBP-binding potential, we divided their CDSs into 5-mers using a rolling window of 5 nt. We then counted the 5-mers that were labelled as significant for a given RBP and multiplied the count for each significant 5-mer by its z score. The values obtained were summed into a single score for each RBP (Fig. The binding potential scores for all evaluated RBPs and reporter sequences can be found in Supplementary Table 9. All public CLIP data were processed as previously described74, with replicates merged. All crosslinks falling within CDS per sample were counted and the proportion of CDSs to total crosslinks was calculated. Samples in the bottom third were excluded. Remaining samples in the bottom third of raw CDS crosslink counts were also excluded, to remove low-quality datasets. To determine whether a protein tends to bind to RNA in multivalent contexts, the percentile of the GeRM score was calculated for each possible 5-mer in all CDSs. If a given 5-mer is crosslinked preferentially in multivalent contexts (relative to the average multivalency of that 5-mer), then the mean GeRM percentile will be high. The mean 5-mer GeRM percentiles for the top 50 most crosslinked 5-mers was compared with all other 5-mers, producing a ratio. This ratio represents the general bias of a protein to bind its preferred motifs in a multivalent context. Only samples with a ratio of at least 1.1, implying a preference for multivalent motifs, were kept for further analysis. Only transcripts in the top half of total crosslinks per nucleotide in the CDS across all samples were used for further analysis, as samples with a low crosslinking density typically did not contain enough crosslinks per GeRM region to produce meaningful results. The mean of this ratio within each GeRM cluster was calculated for each sample. Samples were then hierarchically clustered and plotted as a heatmap (Fig. iCLIP was performed according to the iiCLIP protocol61, using crosslinked cell lysate containing 1.5 mg of protein per sample. Two replicates were prepared for each CLIP target. For immunoprecipitation, 5 µg anti-TRA2B (ab31353, abcam) was used. Membrane transfer was performed overnight at 4 °C with reduced voltage (15 V). The entire lane was cut from 40 kDa upwards. CLIP libraries were trimmed and demultiplexed using Ultraplex75 and mapped to a small RNA genome containing all rRNA, small nuclear RNA, tRNA and small nucleolar RNA sequences from GENCODE vM22 using STAR (v2.7.0)76. Metaprofiles around different GeRM regions were created by taking the midpoint of each GeRM region and counting the transcriptome-mapped crosslinks in each position relative to the midpoint. All counts were summed across GeRM regions of the same cluster, then normalized by the number of GeRM regions within the cluster and the number of millions of unique transcriptome-mapping crosslinks in each sample. For siRNA-induced double TRA2 knockdowns, 100 nM of ON-TARGETplus siRNA pools, consisting of four siRNAs targeting both TRA2A (LQ-019480-00-0005, Dharmacon) and TRA2B (LQ-007278-00-0005, Dharmacon), were transfected using Lipofectamine RNAiMAX (Thermo Scientific) following the manufacturer's reverse transfection protocol. Transfections with a 100 nM scrambled non-targeting control pool (D-001810-10-05, Dharmacon) were carried out in parallel. Thirty-six hours after transfection, doxycycline was added to the transfection media to induce mScarlet–PPIGLCD expression over a period of 12 h. A total of 48 h post-transfection, cells were either fixed for HCR-FISH as previously described or collected for western blotting to confirm efficient knockdown of TRA2A (1:500; H00029896-B01P, Novus Biologicals) and TRA2B. This was quantified via ImageStudio 5.x Odyssey Clx (LI-COR). For each sample, a single well of a six-well plate of ePB-ER HeLa cells was transfected with one of the mGreenLantern LUC7L3-LCD reporter plasmids in which the LUC7L3 sequence was used as a 3′ UTR but was either GA rich or GA poor. Four replicates were performed for each plasmid, as well as untransfected and uninduced controls. After 16 h, the cells were dissociated, washed twice with ice-cold PBS and kept on ice while the samples were measured with an LSR Fortessa (BD Biosciences). The intensities of mGreenLantern and mScarlet were measured using a 488-nm blue laser (Coherent Sapphire 100 mW) with a 530/30-nm filter, and a 561-nm yellow green laser (Coherent Sapphire 100 mW) with a 610/20-nm filter, respectively. Gating of singlets was performed using FlowJo, and subsequent analysis was performed in R. For each plasmid, the mGreenLantern intensities were normalized to the mean intensity values of the uninduced control samples. Cells were binned according to their mScarlet intensity values, and the mean mGreenLantern intensity values for all cells in each bin were calculated per replicate. All statistics were performed in R (v4.2.0). All tests were two-tailed when possible. All pairwise comparisons were made using a Welch t-test unless otherwise stated. When appropriate, P values were corrected for multiple testing using the Benjamini–Hochberg procedure. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All sequencing data have been deposited via ArrayExpress. The 3′ end sequencing experiments following PPIGLCD expression and CLK-IN-T3 treatment have been deposited under the accession numbers E-MTAB-13304 and E-MTAB-13328, respectively. Processed versions of these datasets can be found on Flow.bio: https://app.flow.bio/projects/710458278460049547. Targeted sequencing data from the reporter experiments have been deposited under the accession number E-MTAB-13329. Long-read sequencing data used to characterize the reporter system have been deposited under the accession number E-MTAB-13330. All iCLIP data from mESCs have been deposited under the accession number E-MTAB-13331. All scripts for data analysis and figure generation (https://github.com/ulelab/interstasis-paper), and the GeRM package (https://github.com/ulelab/germ) are available on GitHub. Vavouri, T., Semple, J. I., Garcia-Verdugo, R. & Lehner, B. Intrinsic protein disorder and interaction promiscuity are widely associated with dosage sensitivity. & Babu, M. M. Tight regulation of unstructured proteins: from transcript synthesis to protein degradation. Bolognesi, B. et al. A concentration-dependent liquid phase separation can cause toxicity upon increased protein expression. A. Biomolecular condensates at the nexus of cellular stress, protein aggregation disease and ageing. & Medenbach, J. Auto-regulatory feedback by RNA-binding proteins. & Žagrović, B. Coding relationship links RNA G-quadruplexes and protein RGG motifs in RNA-binding protein autoregulation. Goldberger, R. F. Autogenous regulation of gene expression. Evidence of direct complementary interactions between messenger RNAs and their cognate proteins. Wang, J. et al. A molecular grammar governing the driving forces for phase separation of prion-like RNA binding proteins. A. et al. Arginine-enriched mixed-charge domains provide cohesion for nuclear speckle condensation. Pritišanac, I. et al. A functional map of the human intrinsically disordered proteome. King, M. R. et al. Macromolecular condensation organizes nucleolar sub-phases to set up a pH gradient. Haig, D. & Hurst, L. D. A quantitative measure of error minimization in the genetic code. Alff-Steinberger, C. The genetic code and error transmission. Nakachi, Y. et al. Nucleotide compositional constraints on genomes generate alanine-, glycine-, and proline-rich structures in transcription factors. Thul, P. J. et al. A subcellular map of the human proteome. Hardenberg, M., Horvath, A., Ambrus, V., Fuxreiter, M. & Vendruscolo, M. Widespread occurrence of the droplet state of proteins in the human proteome. Collins, R. L. et al. A cross-disorder dosage sensitivity map of the human genome. Collins, R. L. et al. A structural variation reference for medical and population genetics. Phase separation as a possible mechanism for dosage sensitivity. Pacheco-Fiallos, B. et al. mRNA recognition and packaging by the human transcription–export complex. & Ulitsky, I. Sequences enriched in Alu repeats drive nuclear localization of long RNAs in human cells. Zuckerman, B., Ron, M., Mikl, M., Segal, E. & Ulitsky, I. Gene architecture and sequence composition underpin selective dependency of nuclear export of long RNAs on NXF1 and the TREX complex. Mordstein, C. et al. Codon usage and splicing jointly influence mRNA localization. Predicting splicing from primary sequence with deep learning. Chaturvedi, P. & Belmont, A. S. Nuclear speckle biology: at the cross-roads of discovery and functional analysis. Hallegger, M. et al. TDP-43 condensation properties specify its RNA-binding and regulatory repertoire. Van Nostrand, E. L. et al. Principles of RNA processing from analysis of enhanced CLIP maps for 150 RNA binding proteins. Dominguez, D. et al. Sequence, structure, and context preferences of human RNA binding proteins. An integrated encyclopedia of DNA elements in the human genome. Van Nostrand, E. L. et al. A large-scale binding and functional map of human RNA-binding proteins. Best, A. et al. Human Tra2 proteins jointly control a CHEK1 splicing switch among alternative and constitutive target exons. Krchňáková, Z. et al. Splicing of long non-coding RNAs primarily depends on polypyrimidine tract and 5′ splice-site sequences due to weak interactions with SR proteins. Daniels, N. J. et al. Functional analyses of human LUC7-like proteins involved in splicing regulation and myeloid neoplasms. Braun, S. et al. Decoding a cancer-relevant splicing decision in the RON proto-oncogene using high-throughput mutagenesis. The Human Cell Atlas: from vision to reality. Tapial, J. et al. An atlas of alternative splicing profiles and functional associations reveals new regulatory programs and genes that simultaneously express multiple major isoforms. Song, M. et al. Cdc2-like kinases: structure, biological function, and therapeutic targets for diseases. The Clk/Sty protein kinase phosphorylates SR splicing factors and regulates their intranuclear distribution. Haltenhof, T. et al. A conserved kinase-based body-temperature sensor globally controls alternative splicing and gene expression. Dominguez, D. et al. An extensive program of periodic alternative splicing linked to cell cycle progression. Funnell, T. et al. CLK-dependent exon recognition and conjoined gene formation revealed with a novel small molecule inhibitor. Frege, T. & Uversky, V. N. Intrinsically disordered proteins in the nucleus of human cells. Widespread autogenous mRNA–protein interactions detected by CLIP-seq. Zagrovic, B., Adlhart, M. & Kapral, T. H. Coding from binding? Molecular interactions at the heart of translation. Sauna, Z. E. & Kimchi-Sarfaty, C. Understanding the contribution of synonymous mutations to human disease. Lin, B. C., Kaissarian, N. M. & Kimchi-Sarfaty, C. Implementing computational methods in tandem with synonymous gene recoding for therapeutic development. Bahar Halpern, K. et al. Nuclear retention of mRNA in mammalian tissues. B. R. et al. Phosphorylation of a nuclear condensate regulates cohesion and mRNA retention. Phase separation provides a mechanism to reduce noise in cells. & Brangwynne, C. P. Can phase separation buffer cellular noise? & Murcko, M. A. Modulating biomolecular condensates: a novel approach to drug discovery. Vendruscolo, M. & Fuxreiter, M. Protein condensation diseases: therapeutic opportunities. Lester, E. et al. Tau aggregates are RNA–protein assemblies that mislocalize multiple nuclear speckle components. Malik, I., Kelley, C. P., Wang, E. T. & Todd, P. K. Molecular mechanisms underlying nucleotide repeat expansion disorders. Urbanek, M. O., Jazurek, M., Switonski, P. M., Figura, G. & Krzyzosiak, W. J. Nuclear speckles are detention centers for transcripts containing expanded CAG repeats. Highly accurate protein structure prediction with AlphaFold. The ground state of embryonic stem cell self-renewal. Lee, F. C. Y. et al. An improved iCLIP protocol. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. The nf-core framework for community-curated bioinformatics pipelines. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Zhu, A., Ibrahim, J. G. & Love, M. I. Heavy-tailed prior distributions for sequence count data: removing the noise and preserving large differences. Shen, S. et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Eng, J. K., McCormack, A. L. & Yates, J. R. An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. Segment number threshold determines juvenile onset of germline cluster expansion in Platynereis dumerilii. Third-generation in situ hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust. Pachitariu, M. & Stringer, C. Cellpose 2.0: how to train your own model. Lemaître, G., Nogueira, F. & Aridas, C. K. Imbalanced-learn: a Python toolbox to tackle the curse of imbalanced datasets in machine learning. Li, H. Minimap2: pairwise alignment for nucleotide sequences. The stringdist package for approximate string matching. Kuret, K., Amalietti, A. G., Jones, D. M., Capitanchik, C. & Ule, J. Positional motif analysis reveals the extent of specificity of protein–RNA interactions observed by CLIP. Wilkins, O. G., Capitanchik, C., Luscombe, N. M. & Ule, J. Ultraplex: a rapid, flexible, all-in-one fastq demultiplexer. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & Sudbery, I. UMI-tools: modeling sequencing errors in unique molecular identifiers to improve quantification accuracy. The PRIDE database at 20 years: 2025 update. We thank M. Bordenx, L. Gnidovec and F. Mardakheh for technical support and advice; M. Modic for the transposase plasmid; S. Lynham and X. Yang at the proteomics facility of King's College London; the Advanced Sequencing, Advanced Light Microscopy, Flow Cytometry and Cell Science Platforms at The Francis Crick Institute; and the BioOptics Facility at the Institute of Molecular Pathology. This work was supported by the European Research Council under the European Union's Horizon 2020 research and innovation programme (835300-RNPdynamics to J.U. was supported by a Marie Sklodowska-Curie fellowship (101150910). is a Lady Edith Wolfson Fellow funded by the MND Association and the Rosetrees Charity. laboratory is supported by the European Union (European Research Council; RiboTrace, CoG-866166), the Austrian Science Fund (FWF; 10.55776/F80 and 10.55776/DOC177) and the Vienna Science and Technology Fund (WWTF; LS23-053). is supported by Boehringer-Ingelheim, the European Research Council under the Horizon 2020 research and innovation programme (ERC-2020-STG 949081 RNApaxport) and by the Austrian Science Fund (FWF) doc.funds program DOC177-B (RNA@core: molecular mechanisms in RNA biology). For the purpose of Open Access, the authors have applied a CC BY public copyright licence to any Author Accepted Manuscript version arising from this submission. Open Access funding provided by The Francis Crick Institute. These authors contributed equally: Rupert Faraway, Neve Costello Heaven The Francis Crick Institute, London, UK Rupert Faraway, Neve Costello Heaven, Holly Digby, Oscar G. Wilkins, Anob M. Chakrabarti, Ira A. Iosub, Lea Knez & Jernej Ule Rupert Faraway, Neve Costello Heaven, Holly Digby, Ira A. Iosub, Rhys Dore & Jernej Ule Department of Basic and Clinical Neuroscience, Institute of Psychiatry Psychology and Neuroscience, King's College London, London, UK Rupert Faraway, Neve Costello Heaven, Holly Digby, Ira A. Iosub, Rhys Dore & Jernej Ule National Institute of Chemistry, Ljubljana, Slovenia Klara Kuret Hodnik, Jure Rebselj, Neža Vadnjal & Jernej Ule Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and jointly supervised the study. and N.V. also performed the experiments, whereas K.K.H. and J.R. also performed the data analyses. assisted with turning the GeRM code into an R package. wrote the manuscript, with contributions from all co-authors. All authors agreed with the content and consented to submit the manuscript with their contributions. Correspondence to Rupert Faraway or Jernej Ule. The authors declare no competing interests. Nature thanks Igor Ulitsky and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A codon shuffled sequence in which all codons must be synonymously shifted is shown. Below, a ratio is shown that compares the GeRM scores associated with the native codon choice to the GeRM scores associated with any possible synonymous mutation. c, a schematic detailing the calculation of the ratio of GeRM between native codon choices and synonymous mutations. d, the mean log2-transformed ratio for each type of possible synonymous mutation. Relative codon usage is calculated by scaling the transcriptomic codon usage such that the mean is 0 and the standard deviation is 1. e, The normalised conservation across 470 mammals of synonymously mutable positions in coding sequences that either encode LCDs or do not. Conservation values are normalised to the middle position of the codon, which is never synonymously mutable, and each codon is normalised to have a median conservation value of 0. f, PhyloP scores across 100 vertebrates of the middle positions of codons inside or outside of LCDs. 1e, but using unnormalised PhyloP scores across 100 vertebrates. The mean GeRM scores in those regions after synonymously reordering the codons within each transcript (dashed lines) Unless otherwise stated, pairwise significance testing are FDR-corrected Welch t-tests, where * = p < 1−15. Precise p-values are found in Source Data. a, The contribution to the total GeRM score in each GeRM cluster that is accounted for by a given 5-mer. Only the three most common 5-mers per cluster are shown. b, The relative occurrence of amino acids encoded by different GeRM regions relative to their occurrence in the entire proteome. Only amino acids with notable enrichments are shown. c, The number of GeRM regions per cluster encoding at least two-fold more arginines than expected based on occurrence in the proteome. d, The mean amino acid entropy calculated in sliding windows for amino acids encoded by different GeRM regions or the amino acid sequences encoded by the rest of the CDS (grey line). Statistical tests are one sample t-tests, with FDR correction. a, Principal component analysis of the arginine codon usage within LCDs containing at least 20% arginine (R-LCDs). R-LCDs are categorised into three groups of equal size using the first principal component. b, The codon usage of the groups of R-LCDs defined in panel a. c, The representation of amino acids within the groups of R-LCDs defined in panel a. d-e, The codon usage of R-LCDs from genes with either ‘nuclear speck' or ‘RNA binding' gene ontology terms compared to R-LCDs without those annotations. f, The proportion of charged amino acids within the R-LCD categories defined in panel a. g, The percentage of proteins containing at least one R-LCD in different species. h, the percentage of LCDs that contain at least 20% arginine in different species. i, the Spearman's rank correlations of codon usage within R-LCDs in different species. Precise p-values are found in Source Data. a, Normalised counts for the nuclear lncRNA NEAT1 and the efficiently exported mRNA RPL12 in nuclear and cytoplasmic fractions at different PPIGLCD expression timepoints. b, Normalised counts for control (PSAP, HNRNPDL) and GA-rich (EIF3A, BRD4) RNAs, as in panel a. c, Co-localisation of mScarlet-PPIGLCD with nuclear speckles. d, Oligo-dT FISH of cells expressing mScarlet-PPIGLCD. e, Changes in the nuclear SC35 intensity and the size and number of speckles per nucleus after expression of mScarlet-PPIGLCD across 3 replicates, with at least 100 nuclei per replicate. Points are coloured based on the length of expression induction with doxycycline. Ratios are compared the intensity of mScarlet-PPIGLCD in the nucleus, and points are coloured as in panel f. h-i, Example images showing HCR-FISH signal for HNRNPDL (control) or BRD4 (GA-multivalent) mRNAs, with SC35 immunofluorescence and mScarlet-PPIGLCD. j, Quantification of the enrichment of HCR-FSH signal within nuclear speckles versus nucleoplasm per nucleus with respect to PPIGLCD expression. 3 independent replicates are plotted in different colours and regression lines are plotted in dashed lines. l, Quantification of the proportion of total R-MCD encoding transcripts or total mRNAs that are the overexpressed PPIGLCD transcript in the dox induction time course sequencing experiment. a-c, Example images showing HCR-FISH signal for PSAP (control) or EIF3A (GA-multivalent) mRNAs, with SC35 immunofluorescence in cells expressing different R-MCDs (LUC7L3221–415, PRPF38B251–534 and SRSF11171–389) fused to mGreenLantern. d, Quantification of enrichment of HCR-FISH signal within nuclear speckles compared to the nucleoplasm for cells expressing varying amounts of the R-MCDs shown in panels a-c. e-g, As in panels a-c, but showing EIF3A HCR-FISH signal for cells expressing either an R-MCD with GA-rich GeRM (LUC7L3221–415) or a LCD associated with a different GeRM signature (C-rich GeRM, Pro-rich: mGreenLantern-PRR36832–1099; GC-rich GeRM, Ala/Glu/Gly-rich: mGreenLantern-SIRT12–151). Precise p-values are found in Source Data a, Flow cytometry measurements of mGreenLantern signal from cells transfected with constructs encoding mGreenLantern with a GA-rich or less GA-rich 3'UTR. PPIGLCD was expressed for 16 h. mGreenLantern intensities are normalised to mGreenLantern transfection without PPIGLCD expression. b, The proportion of proteins made up of highly polar (non-charged) regions (regions of 40 amino acids with greater than 40% polar, non-charged residues). Proteins encoded by genes that showed significant nuclear mRNA retention upon expression of PPIGLCD are compared to proteins from expressed genes that did not exhibit retention. c, length of longest predicted disordered region. d, FuzDrop scores for retained and control genes. e, Dosage sensitivity scores (haploinsufficiency) for retained and control genes. f, Dosage sensitivity scores (triplosensitivity) for retained and control genes. g, The distribution of predicted probabilities of loss-intolerance as determined by gnomAD for retained and control genes. h, The proportion of retained or control genes with a gnomAD score greater than 0.5. i, Per mRNA interaction effects describing the change in nuclear-cytoplasmic distribution after 12 h of PPIGLCD expression for mRNAs with different degrees of GA multivalency. j, Interaction effects from panel b for mRNAs in the top 2% of GA multivalency. mRNAs are classified based on whether 50% of the total GA-multivalency occurs within 50 nt of a predicted exon junction complex site. l, GA-multivalency for retained and control genes. m, Mean exon length for retained and control genes. n, CDS length for retained and control genes. o, CAG-multivalency for retained and control genes. Precise p-values are found in Source Data. a, The nuclear-cytoplasmic distribution of reporter transcripts binned by the number of introns in their reporter gene. Data from cells transfected with the reporter plasmid pool for 16 h. b, Nuclear-cytoplasmic distributions of reporter transcripts, but comparing cells transfected for either 8 (short) or 24 h (long) with the plasmid pool. c, The nuclear-cytoplasmic distribution of reporter transcripts binned by the number of introns in their reporter gene. Data from cells in which the reporter library is stably integrated and expressed for either 4, 8 or 12 h by the addition of doxycycline. d, The nuclear-cytoplasmic abundance ratio of reporter transcripts over time depending on the number of exons and their multivalency. Data is taken from cells transfected for either 8 (short) or 24 h (long) with the plasmid pool. Data from cells in which the reporter library is stably integrated and expressed for either 4, 8 or 12 h by the addition of doxycycline. f, The slopes from regression models of relative reporter transcript abundance over time were calculated by expressing the reporter pool for 16 h, then collecting fractions at 0, 2, 4 and 8 h after withdrawing dox. A positive slope indicates that the transcript is relatively more stable in the pool, while a negative slope indicates that the transcript is relatively less stable. Precise p-values are found in Source Data. a, proportion of mapped Tra2b iCLIP crosslinks falling in different genomic regions. Solid lines denote the mean of two replicates, while shaded regions denote the standard error. c, Taking a window of 50 nt from the 3' end of every internal exon of the longest coding transcript per gene, the proportion of iCLIP reads that contain a splice junction. d, Changes in exon proportion spliced in (dPSI) after double knockdown of TRA2A and TRA2B from publicly available data33. Exons are all coding exons from the longest coding transcript per gene with a GA-multivalent region, categorised based on whether the exon overlaps a GA-multivalent region. e, The proportion of exons spliced in (PSI) in the control samples, using the same data and categories as in panel d. f, Using all available human tissues from VastDB, the percentage of exons in each tissue that are constitutive (PSI > .95, groups as defined in panel d). g, Western blot of the knockdown efficiency of TRA2A and TRA2B siRNA after 36 h. For gel source data, see Supplementary Fig. h, Quantification of TRA2B knockdown efficiency from immunofluorescence in cells used for HCR-FISH experiments. i, CLIP crosslinking frequency to each 5-mer and the mean percentile of the GeRM score of crosslinked 5-mers for UPF1. j, as in i, but for TRA2B. b-c, A representative image showing TRA2B and SC35 signal in cells expressing mGreenLantern either with the GA-rich LCD of LUC7L3 as a 3'UTR sequence or as a translated sequence. d, Quantification of the enrichment of TRA2B signal within nuclear speckles versus nucleoplasm per nucleus with respect to mGreenLantern expression for the different constructs shown in panel a. Independent replicates are plotted in different colours and regression lines are plotted in dashed lines. e, The slopes of linear regression models of the relationship between mGreenLantern expression and TRA2B signal within the speckle for the different constructs shown in panel a. Pairwise significance testing are FDR-corrected Welch t-tests, where * = p < 0.05, ** = p < 0.01, *** = p < 0.001. Precise p-values are found in Source Data. a, Oligo-dT FISH in cells treated either with DMSO or 1 µM CLK-IN-T3 for 8 h. b, The nuclear-cytoplasmic ratio of oligo-dT signal per cell at different doses of CLK-IN-T3 for 8 h. c, The ratio of upper quartile to median oligo-dT signal per nucleus at different doses of CLK-IN-T3 for 8 h. d, Example image of SON immunofluorescence and HCR-FISH of control (PSAP) mRNA in cells treated either with DMSO or 1uM CLK-IN-T3 for 16 h. Scale bars are 10 µm. e, Quantification of the enrichment of HCR-FSH signal within nuclear speckles versus nucleoplasm per nucleus per replicate for control (PSAP, HNRNPDL) or GA-multivalent (BRD4, EIF3A) mRNAs. f, as in d, but with GA-multivalent (BRD4) mRNA. g, The average size of speckles per nucleus per replicate in cells treated with either DMSO or 1 µM CLK-IN-T3 for 16 h. h, The number of speckles normalised by nuclear area per nucleus per replicate in cells treated with either DMSO or 1 µM CLK-IN-T3 for 16 h. i, Per gene interaction effect of subcellular fraction and drug treatment from 3' end sequencing of nuclear and cytoplasmic fractions of cells treated with DMSO or 1uM CLK-IN-T3 for 8 h. Genes are classified based on the amount of GA-multivalency that they contain in their coding sequences. j, Western blot showing changes in TRA2B phosphorylation (green, lower band) and pan-SR protein phosphorylation (red) following CLK-IN-T3 treatment. For gel source data, see Supplementary Fig. k, Per-gene log-transformed fold-changes for the interaction terms from PPIGLCD overexpression (12 h doxycycline) or 1 µM CLK-IN-T3 (8 h) where a positive fold change means that the gene was more nuclear after treatment. Points are coloured by their scaled GA-multivalency. Precise p-values are found in Source Data. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Faraway, R., Costello Heaven, N., Digby, H. et al. Collective homeostasis of condensation-prone proteins via their mRNAs. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09574-y'>SPP1 is required for maintaining mesenchymal cell fate in pancreatic cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 15:42:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Elucidating the complex network of communication between tumour cells is central to understanding cell fate decisions and progression of pancreatic ductal adenocarcinoma (PDAC)1,2. We previously showed that constant suppression of BMP activity by the BMP antagonist GREM1 secreted by mesenchymal PDAC cells is essential for maintaining the fate of epithelial PDAC cells3. Here we identify SPP1 (also known as osteopontin)4 as a key regulator of mesenchymal cell fate in pancreatic cancer. Proteomic analysis of plasma from patients with PDAC showed that SPP1 is substantially upregulated in late-stage disease. Inactivation of Spp1 led to a delay in tumorigenesis in mouse PDAC models and abolished metastasis formation. Spp1 was expressed in epithelial PDAC cells, and Spp1 inactivation resulted in a conversion of mesenchymal to epithelial PDAC cells. Mechanistically, SPP1 bound the CD61 receptor on mesenchymal PDAC cells to induce Bmp2 and Grem1 expression, and GREM1 inhibition of BMP signalling was required for Spp1 expression in epithelial cells, thereby forming an intercellular regulatory loop. Concomitant inactivation of Grem1 reverted the epithelial phenotype of Spp1 knockout to fully mesenchymal PDAC. Conversely, Grem1 heterozygosity combined with Spp1 knockout resulted in wild-type PDAC histology, a result that confirmed the direct antagonistic functions of these factors. Hence, mesenchymal and epithelial PDAC cell fates are determined by the reciprocal paracrine regulation of the soluble factors GREM1 and SPP1. To identify factors that control the progression of pancreatic cancer, we compared protein levels in the plasma of patients with early-stage (stage I or II) or late-stage (stage III or IV) PDAC (Fig. The abundance of SPP1 was significantly increased at late stages (Fig. SPP1 plasma levels were also increased in Pdx1-cre;KrasLSL-G12D/+;Trp53fl/fl;Rosa26LSL-YFP (KPCY) mice, a commonly used mouse model of PDAC6, compared with wild-type (WT) animals (Fig. a, Schematic showing a comparison of the proteomic profiles of plasma proteins of patients with stage I or II PDAC and patients with stage III or IV PDAC. b, Volcano plot showing differential protein expression between plasma samples from patients with early-stage pancreatic cancer (stage I or II) or late-stage pancreatic cancer (stage III or IV). Blue dots indicate proteins with increased expression in early-stage pancreatic cancer (stage I and II), and red dots indicate proteins with increased expression in late-stage pancreatic cancer (stage III and IV). Statistical analysis was performed using two-sided t-tests. d, Immunofluorescence analysis of VIM (red), YFP (grey) and SPP1 (green) in KPCY tumours. f, RT–qPCR analysis of expression of the epithelial markers Krt19, Cdh1 and Epcam and the mesenchymal markers Fn1, S100a4 and Vim between Spp1WT/WT KPFV and Spp1Δ/Δ KPFV organoids (n = 3 biological replicates), normalized using Spp1WT/WT KPFV values. h, Sizes of subcutaneous tumours formed from Spp1WT/WT KPF and Spp1Δ/Δ KPF organoids (n = 7 mice). P values were calculated using two-sided t-tests. To explore the role of SPP1 in the development and progression of pancreatic cancer, we first analysed single-cell sequencing data from public databases. Cancer cell populations with high SPP1 expression had significantly higher levels of genes involved in the epithelial–mesenchymal transition (EMT) signature7,8 than populations with low SPP1 expression (Extended Data Fig. Moreover, by analysing publicly available transcriptome data of 644 patients with pancreatic cancer, we observed that patients with high SPP1 expression exhibited significant enrichment of the hallmark EMT gene set8. Accordingly, these patients showed worse prognosis compared with those with low SPP1 expression (Extended Data Fig. Together, these results suggest that SPP1 is a regulator of EMT in pancreatic cancer. Immunofluorescence staining of KPCY tumours revealed that SPP1 was predominantly expressed in epithelial PDAC cells (EPCs) and not in vimentin (VIM)-positive mesenchymal PDAC cells (MPCs) (Fig. To understand SPP1 function, we established organoids from tumours of Pdx1-Flp;KrasFSF-G12D/+;Trp53frt/frt;Vim-GFP (KPFV) mice9, which enabled the differentiation between EPCs and MPCs on the basis of GFP fluorescence (Extended Data Fig. We performed Spp1 knockout in KPFV organoids using CRISPR–Cas9 technology (Extended Data Fig. The VIM–GFP+ MPC population was abolished after Spp1 knockout (Fig. 1e), which suggested that Spp1 is required for MPC fate. Quantitative PCR with reverse transcription (RT–qPCR) analysis showed a marked decrease in expression of the mesenchymal markers Vim, Fn1 and S100a4, concomitant with a notable increase in the epithelial markers Krt19, Cdh1 and Epcam10 (Fig. In line with this result, there were fewer allograft tumours generated from Pdx1-Flp;KrasFSF-G12D/+;Trp53frt/frt (KPF) organoids deficient in SPP1 (Spp1Δ/Δ KPF), and the tumours were small and consisted almost exclusively of EPCs. By contrast, tumours formed from control KPF organoids contained both EPC and MPC populations (Fig. Furthermore, Spp1 overexpression in organoids from LSL-KrasG12D/+;LSL-Trp53R172H/+;Pdx-1-Cre (KPR172H/+C) mice induced mesenchymal differentiation of PDAC cells (Extended Data Fig. Notably, these phenotypes were also observed in human PDAC organoids and xenografts with SPP1 knocked out or overexpressed (Extended Data Fig. Thus, SPP1 produced by EPCs is required for the maintenance of MPCs in both mouse and human PDAC. Several receptors have been identified as receptors for SPP1, including the integrin CD61 (encoded by Itgb3)11,12,13. Immunofluorescence staining of KPCY tumours showed a markedly complementary expression pattern of SPP1 and CD61, and CD61 was predominantly expressed in MPCs (Fig. To further investigate the relationship between SPP1 and CD61, we performed CRISPR–Cas9-mediated knockout of Itgb3 in KPFV organoids (Extended Data Fig. VIM–GFP+ cells were not observed in Itgb3-deficient KPFV organoids, which also had a significant reduction in mesenchymal marker expression (Fig. Furthermore, allograft formation was significantly reduced (Fig. 2d,e), and the proportion of MPCs was decreased in Itgb3-deficient allografts (Fig. a, Immunofluorescence analysis of CD61 (red), YFP (grey) and SPP1 (green) expression in KPCY orthotopic tumours (n = 3 mice). e, Sizes of subcutaneous tumours formed from Itgb3WT/WT KPF and Itgb3Δ/Δ KPF organoids (n = 4 mice). f, Immunofluorescence staining for GFP (grey), KRT19 (green) and VIM (red) of Itgb3WT/WT KPF and Itgb3Δ/Δ KPF subcutaneous tumours. g, Comparison of the percentages of different cancer cell subpopulations between subcutaneous tumours formed from Itgb3WT/WT KPF and Itgb3Δ/Δ KPF organoids (n = 3 mice). P values were calculated using two-sided t-tests. We next investigated the potential function of SPP1 in PDAC progression. We combined PDAC induced by the Flp/Frt recombination system (KPF mice)14 with a floxed Spp1 (Spp1fl) allele, a Flp-inducible Rosa26FSF-creER allele14 that allows Spp1 deletion in the pancreatic parenchyma during tumour development and a Tomato/GFP-Cre reporter allele15 to generate compound Pdx1-Flp;KrasFSF-G12D/+;Trp53frt/frt;R26FSF-creER;R26mTmG;Spp1fl/fl (Spp1fl/fl KPFCT) mice (Fig. a, Schematic showing how the Spp1fl/fl KPFCT mouse model was generated. b, Haematoxylin and eosin (H&E) staining of Spp1WT/WT KPFCT and Spp1fl/fl KPFCT tumours (n = 3 mice). c, Immunofluorescence staining for GFP (grey), KRT19 (green) and VIM (red) in Spp1WT/WT KPFCT and Spp1fl/fl KPFCT tumours. d, Comparison of the proportions of different cancer cell subpopulations between Spp1WT/WT KPFCT and Spp1fl/fl KPFCT tumours (n = 5 mice). g, Images of liver and lung metastases in Spp1WT/WT KPhetFCT and Spp1fl/fl KPhetFCT mice. h, Percentage of mice with liver metastases (left) and quantification of the number of liver metastases (right) in Spp1WT/WT KPhetFCT (n = 15) and Spp1fl/fl KPhetFCT (n = 20) mice. i, Percentage of mice with lung metastases (left) and quantification of the number of lung metastases (right) in Spp1WT/WT KPhetFCT (n = 15) and Spp1fl/fl KPhetFCT (n = 20) mice. P values were calculated using two-sided Fisher-exact tests (h (left) and i (left)) or two-sided t-tests (d,h (right), i (right)). Histology of Spp1-deficent tumours showed a notable increase in epithelial structures (Fig. 3b), and immunofluorescence analysis revealed that PDAC tumours contained a marked increase in EPCs (KRT19+VIM–) and a substantial decrease in MPCs (KRT19–VIM+) compared with control KPFCT tumours (Fig. Notably, Spp1fl/fl KPFCT tumours exhibited a considerable reduction in cancer-associated fibroblasts (Extended Data Fig. However, despite the reduction in cancer-associated fibroblasts, the degree of immune infiltration was unchanged (Extended Data Fig. An increase in MPCs is known as a crucial factor that leads to poor prognosis in patients with PDAC16,17. Accordingly, Spp1fl/fl KPFCT mice with tumours exhibited significantly increased survival compared with control KPFCT mice (Fig. Moreover, the absence of Spp1 also substantially reduced PDAC development in the KPCY model, with a substantial percentage of Spp1-deficient animals surviving more than 400 days (Fig. EMT is as an important mechanism of tumour metastasis formation18. To investigate the impact of Spp1 on PDAC metastasis, we used (Pdx1-Flp;KrasFSF-G12D/+;Trp53WT/frt;R26FSF-creER;R26mTmG) KPhetFCT mice, because the heterozygosity of Trp53 means that there is a longer latency period in the formation of the tumour, which enables metastases to form and disseminate3. Ultrasound imaging showed that most mice developed tumours at around 100 days of age. Therefore, tamoxifen was administered at this point, and tumour progression and metastasis formation were monitored (Fig. Consistent with Spp1fl/fl KPFCT mice, Spp1fl/fl KPhetFCT mice (which have Spp1 knocked out) had tumours that predominantly comprised EPCs, whereas KPhetFCT mice (with WT Spp1) formed tumours with both EPCs and MPCs (Extended Data Fig. In KPhetFCT mice, liver metastases were detected in 23% of animals and 30% had lung metastases. By contrast, Spp1fl/fl KPhetFCT mice had no metastases in the liver or lung (Fig. Furthermore, orthotopic transplantation of Spp1-overexpressing KPR172H/+C organoids significantly promoted liver and lung metastases (Extended Data Fig. By contrast, in vivo administration of a SPP1-blocking antibody reduced MPC cell fate and suppressed liver and lung metastases (Extended Data Fig. Taken together, these data demonstrate that SPP1 is essential for maintaining the fate of MPCs and promoting metastasis formation. We next sought to investigate the molecular mechanism of SPP1 function. We previously identified Grem1 as a BMP2 target gene to control epithelial cell identity3. RNA sequencing (RNA-seq) analysis of Spp1Δ/Δ KPFV and control organoids revealed that SPP1 function is required for Bmp2 and Grem1 expression (Fig. 4a), a result that was confirmed by RT–qPCR (Fig. Similarly, Itgb3 knockout significantly attenuated Bmp2 and Grem1 expression (Fig. Conversely, overexpressing Itgb3 increased Bmp2 and Grem1 expression (Extended Data Fig. This result suggests that the signalling axis SPP1–CD61–BMP2–GREM1 operates between EPC and MPC PDAC subpopulations. Blue dots indicate genes with higher expression in Spp1Δ/Δ KPFV organoids, and red dots indicate genes with higher expression in Spp1WT/WT KPFV organoids. P values were calculated using edgeR (two-sided). b, RT–qPCR analysis of Grem1 and Bmp2 expression in Spp1WT/WT KPFV and Spp1Δ/Δ KPFV organoids (n = 3 biological replicates). d, RT–qPCR analysis of Spp1, Id1, Id2, Id3, Bmp2 and Grem1 levels in KPF organoids treated with recombinant BMP2 protein or vehicle (n = 3 biological replicates), normalized to vehicle values. f, Quantification of the proportion of KRT19+VIM– and KRT19–VIM+ cancer cells in e. Data are the mean ± s.e.m. P values were calculated using two-sided t-tests. RNA-seq analysis revealed that Itgb3 knockout organoids showed impaired NF-κB signalling (Extended Data Fig. RT–qPCR confirmed downregulation of the NF-κB target genes Ccl2 (ref. 20) and Tnf19, whereas their expression increased when Itgb3 was overexpressed (Extended Data Fig. Next, using the JASPAR database21, we identified two putative NF-κB-binding sites in the Bmp2 promoter, and chromatin immunoprecipitation assays for RelA confirmed binding to these two sites in the Bmp2 promoter (Extended Data Fig. The TAK1 kinase inhibitor 5Z-7-oxozeaenol22 significantly inhibited the binding of RelA to the site at +1597 and reduced Bmp2 mRNA levels (Extended Data Fig. BMP2 treatment increased Bmp2 expression, which indicated the presence of an autoregulatory feedback loop23,24, and also upregulated Grem1 but decreased Spp1 expression (Fig. BMP2 treatment led to a rapid induction of the BMP target genes Id1, Id2 and Id3, but Spp1 downregulation occurred only after 24 h and coincided with an increase in Vim expression. By contrast, treatment with the BMP inhibitor LDN193189 led to the opposite effect (Extended Data Fig. 8i–k), which indicated that Spp1 is expressed as a consequence of epithelial identity. Thus, BMP2, SPP1 and GREM1 are components of a complex regulatory loop that determines PDAC cell fate. To better understand the mechanism of PDAC cell fate determination, we generated Spp1 and Grem1 compound double-knockout models (Extended Data Fig. Concomitant inactivation of Grem1 reverted the epithelial phenotype of Spp1 knockout to fully mesenchymal PDAC. Moreover, Grem1 heterozygosity combined with Spp1 knockout resulted in WT PDAC histology (Fig. 9c), a result that highlighted their directly opposing functions. Furthermore, subsequent Grem1 knockout reversed the effect of Spp1 knockout and produced a mesenchymal phenotype. Conversely, Grem1 overexpression reversed the effects of Spp1 overexpression and induced an epithelia phenotype (Extended Data Figs. These results further emphasize the antagonistic relationship between SPP1 and GREM1. Cellular heterogeneity is a prominent feature used for the classification of pancreatic cancer25,26,27. We showed here that epithelial and mesenchymal PDAC subpopulations are required for the maintenance of each other through a reciprocal, paracrine signalling network of three diffusible molecules: SPP1, BMP2 and GREM1 (Extended Data Fig. We speculate that the mutual dependency of epithelial and mesenchymal subpopulations may underlie the obligatory cellular heterogeneity seen in PDAC. Moreover, our data suggested that cellular heterogeneity is required for efficient pancreatic tumour formation, as decreasing the mesenchymal PDAC subpopulation through Spp1 inactivation led to a substantial extension of lifespan in the aggressive KPF and KPC PDAC models (Fig. Hence therapeutic strategies that aim to reduce cellular heterogeneity could be considered for PDAC treatment. Peripheral venous blood samples were collected from 26 patients with pancreatic cancer at the Seventh Affiliated Hospital of Sun Yat-sen University. Plasma was isolated from the collected blood samples, and plasma proteomic analysis was subsequently performed. Informed consent was obtained from all participants, and approvals were secured from the ethics board of the Seventh Affiliated Hospital of Sun Yat-sen University for the use of these specimens in research. All clinical samples were collected with informed consent, and all experiments adhered to the regulations and approvals of the internal review and ethics board at the Seventh Affiliated Hospital of Sun Yat-sen University. The Institutional Review Board approval number is KY-2024-249-01. Clinical information on the patients, including age, sex, TNM stage, histology and differentiation, was obtained from medical and follow-up records (Supplementary Table 1). Human pancreatic tissue samples were obtained from patients at the Seventh Affiliated Hospital of Sun Yat-sen University who underwent surgery for pancreatic neoplasia. Informed consent was obtained from all participants, and approvals were secured from the ethics board of the Seventh Affiliated Hospital of Sun Yat-sen University for the use of these specimens in research. The tissue samples were collected in Advanced DMEM/F12 base medium with 100 µg ml–1 Primocin and minced into 1 mm2 pieces with a scissor. Minced tissue was then washed in the base medium and resuspended in 10 ml digestion medium containing Advanced DMEM/F12, 5 mg ml–1 collagenase XI, 2 mg ml–1 dispase II and 10 µg ml–1 DNase I. Tumour fragments were digested for 2–4 h on a Thermo Scientific tube rotator at 37 °C at speed 25. Digestion was stopped by adding the base medium. Tissue was then washed and centrifuged twice with the base medium and digested for 10 min with TrypLE Express (Gibco) supplemented with Y-27632 (10 µM). Tissue was then filtered through a 100 µm cell filter (Falcon), washed and centrifuged twice in the base medium. The pellet was resuspended in 5 ml red cell lysis buffer (Corning) for 5 min. Cells were then washed and centrifuged a further 2 times in the base medium, resuspended in 50 µl Matrigel per well (Corning) and plated onto pre-warmed 24-well plates. Next, 500 µl medium per well was added 15–30 min later once the Matrigel had solidified. The medium contained 1× B27 supplement, 10 mM nicotinamide, 1.25mM N-acetylcysteine, 10 nM gastrin, 100 ng ml–1 human FGF10, 100 ng ml–1 mouse Noggin, 500 nM A83-01, 100 µg ml–1 Primocin, 50 ng ml–1 EGF, 10 µM Y-27632, 10% (v/v) R-Spondin 1 conditioned medium and 50% (v/v) WNT3A conditioned medium. For protein extraction, cellular debris from the plasma samples was removed through centrifugation at 12,000g for 10 min at 4 °C. The supernatant was then carefully transferred to a fresh centrifuge tube, and the protein was quantified using a BCA kit (Bio-Rad) according to the manufacturer's protocol. For trypsin digestion, high-abundant proteins were first removed from the plasma samples using a kit (A36369, Thermo), and the solution was then treated with 5 mM dithiothreitol for 30 min at 56 °C, followed by alkylation with 11 mM iodoacetamide for 15 min at room temperature in the dark. Next, 70 μl enzyme digestion buffer was added to the beads and incubated at 95 °C for 10 min, followed by allowing the samples to return to room temperature. Trypsin was then added to a final concentration of 20 ng μl–1 for overnight digestion. The resulting peptides were desalted using C18 ZipTips (Millipore) according to the manufacturer's instructions and then dried for subsequent mass spectrometry (MS) analysis. For liquid chromatography–MS/MS analysis, the tryptic peptides were resuspended in solvent A and directly loaded onto a self-fabricated reversed-phase analytical column (15 cm in length, 100 μm internal diameter). The mobile phase was composed of solvent A (0.1% formic acid in water) and solvent B (0.1% formic acid and 80% acetonitrile in water). The separated peptides were analysed using an Orbitrap Astral mass spectrometer equipped with a nano-electrospray ion source. The electrospray voltage was set to 1,900 V. Precursors were analysed in the Orbitrap detector, whereas fragments were analysed in the Astral detector. The full MS scan resolution was set to 240,000 with a scan range of 380–980 m/z. The MS/MS scan was fixed with a first mass of 150.0 m/z at a resolution of 80,000. Higher-energy collisional dissociation fragmentation was performed at a normalized collision energy of 25%. The automatic gain control target was set at 500%, with a maximum injection time of 3 ms. For database searches, the data-independent acquisition (DIA) data were processed using the DIA-NN search engine (v.1.8). Tandem mass spectra were searched against the Homo_sapiens_9606_SP_20231220.fasta database (20,429 entries) concatenated with a reversed decoy database. Excision on the amino-terminal Met and carbamidomethylation on Cys were specified as fixed modifications. The false discovery rate (FDR) was adjusted to be less than 1%. To address missing values in our dataset, we used the K-nearest neighbours imputation method using the KNNImputer function from the scikit-learn library in Python28. 34) mouse lines have been previously described. The VIM–GFP transgenic line developed by the Gene Expression Nervous System Atlas (GENSAT) Project was obtained from the Mutant Mouse Resource & Research Centers9. Spp1fl mice (NM-CKO-210205) were purchased from the Shanghai Model Organisms Center. These lines were intercrossed to generate the desired genotypes of both sexes on a C57BL/6 background in Charles River Laboratories. Mouse genotypes were determined using real-time PCR with specific probes designed for each gene (Transnetyx). Pdx1-Flp;KrasFSF-G12D;Trp53WT/frt;Rosa26FSF-creER;Rosa26mT/mG;Spp1fl/fl (Spp1fl/fl KPhetFCT) and Spp1WT/WT KPhetFCT mice were given tamoxifen at the same doses at 100 days of age. Sample sizes were determined on the basis of the results of our preliminary experiments. Mice from autochthonous tumour models were selected according to their correct genotypes. Plasma was isolated from the collected blood samples from mice after humane euthanasia. The samples were centrifugated at 12,000g for 10 min at 4 °C, and the supernatant was carefully transferred to a fresh centrifuge tube. Protein levels were quantified using a BCA kit according to the manufacturer's protocol. ELISA was performed according to the Mouse & Rat Osteopontin (OPN) ELISA Kit–Quantikine protocol (R&D System). The immunocompromised Nu/Nu mice used have been previously described35, and adults (aged 8–12 weeks) obtained from Charles River Laboratories were used for transplantation. Cells dissociated from organoids were subcutaneously injected in 50 μl cold growth-factor-reduced Matrigel (Corning). For Spp1 knockout organoid transplantation, a total 10,000 Spp1Δ/Δ KPF and Spp1WT/WT cells were injected into contralateral flanks of Nu/Nu mice. For Itgb3 knockout organoid transplantation, a total 100,000 Itgb3Δ/Δ KPF and Itgb3WT/WT cells were injected into contralateral flanks of Nu/Nu mice. For the inducible Grem1 deletion after Spp1 deletion analysis, we used CRISPR–Cas9-mediated knockout of Spp1 in GFP-labelled Grem1fl/fl KPFC organoids, and a total 100,000 cells dissociated from GFP-labelled Grem1fl/flSpp1Δ/Δ KPFC organoids were injected into the flanks of Nu/Nu mice. After tumours reached a size of 100 mm3, tamoxifen or peanut oil (vehicle) was intraperitoneally injected at a dose of 100 mg kg−1 body weight once a day for 5 days. For the inducible Grem1 overexpression after Spp1 overexpression analysis, we used Plv-SPP1–GFP and pINDUCER-GREM1 overexpression lentiviral vectors in KPFC organoids, and a total of 100,000 cells dissociated from Plv-SPP1–GFP and pINDUCER-GREM1 KPF organoids were injected into the flanks of Nu/Nu mice. After tumours reached a size of 100 mm3, doxycycline or saline (vehicle) was intraperitoneally injected at a dose of 2 mg kg–1 body weight twice weekly. Tumour growth was monitored twice a week and mice were humanely euthanized when the tumour size reached a mean diameter of 1.2 cm. The tumours were weighed and fixed in 10% neutral buffered formalin (Cellstor) for histological analyses. All animal experiments were approved by the Institute of Cancer Research, London's Animal Welfare and Ethical Review Body and conformed to UK Home Office regulations under the Animals (Scientific Procedures) Act 1986, including Amendment Regulations 2012. The mice were housed in individually ventilated cages at constant temperature and humidity (23 ± 2 °C, 50–60%) in a pathogen-free controlled environment, with a standard 12 h light–12 h dark cycle and unlimited access water and food. For subcutaneous transplantation experiments, the maximum tumour size did not exceed the mean diameter of 1.2 cm, in compliance with project licence PP9490916. The injection technique was adapted from published literature36,37. Under anaesthesia, mice were placed in a supine position on a heated (37 °C) ultrasound platform. KPR172H/+C Spp1 overexpression (n = 10,000) cells and KPR172H/+C control (n = 10,000 cells) cells, suspended in 25 μl of 90% cold growth-factor-reduced Matrigel (Corning), were slowly injected into the pancreas using a 1 ml syringe with a 29 gauge needle. The needle was slowly withdrawn after 15 s of delay to prevent leakage. The precise acquisition of ultrasound images of pancreatic tumours was performed using a Microscan transducer MS-550D (VEVO 2100; Visualsonics). Mice were gently anaesthetized with 1.5–2.5% isoflurane in oxygen at a flow rate of 1 l min−1 and placed in a supine position on a preheated (37 °C) ultrasound platform. Pancreatic tumours were located and labelled using the 2D B-mode setting. For tumour volume measurement, consecutive 2D-mode images were acquired by scanning across the entire tumour at intervals of 0.2 mm using the 3D stage control system, and then reconstructed into a 3D volume. The heart rate, respiration rate and body temperature were monitored throughout the scanning process to ensure stable vital signs of the mice. Organoids were grown to 80–90% confluence before injection. Organoids were dissociated using TrypLE Express supplemented with Y-27632 (10 µM) into single cells and small clusters, and then counted using an automated cell counter. Cells were then resuspended to the indicated concentration in a 50% Matrigel and 50% organoid culturing medium. Next, 50 µl of the cell mixture was injected into the flank of male NSG mice aged 6 weeks. The tumour size was monitored twice weekly with callipers, and tumours were allowed to grow for about 2 months. Tumour volumes were calculated using the formula volume = length × width2/2, as previously described3. Mice were killed at either the study end point or when the tumour reached the maximum allowable volume (tumour volume = 1.5 cm2), and tissues were collected for downstream analyses. The animal use protocol listed was reviewed and approved by the Institutional Animal Care and Use Committee (IACUC), Sun Yat-Sen University (approval no. KPR172H/+CY mice (100 days old) were intraperitoneally injected 3 times a week for 3 months with mouse SPP1 antibody (BioxCell, clone MPIIIB10; 200 μg per mouse per injection) or control antibody (mouse IgG isotype control, Cell Sciences; 200 μg per mouse per injection) in independent experiments. After humane euthanasia, mice were disinfected with 70% ethanol and pancreatic tumours were dissected and placed into ice-cold DMEM–F-12 containing 50 U ml–1 penicillin–streptomycin (Thermo Fisher Scientific). The tissue samples were minced on ice using sterile tissue scissors and then incubated in DMEM containing 1 mg ml–1 collagenase V (Sigma-Aldrich) in a shaking incubator for 1 h. After digestion, the process was halted by adding ice-cold G solution, and the cells were centrifuged at 300g for 5 min at 4 °C. Subsequently, the cells underwent further digestion with TrypLE (Thermo Fisher Scientific) for 10 min at 37 °C, followed by stopping the reaction with ice-cold 2% FCS in PBS and centrifugation. The cells were resuspended, filtered through a 40 μm nylon mesh, centrifuged again and then plated in Matrigel. Organoid medium was used as previously described38. To ensure the selection of cancer cells in the culture, for the first five passages, they were cultured in medium without EGF. After the fifth passage, complete organoid medium was used for culturing and experimentation. Primary antibodies were incubated in blocking buffer at 4 °C overnight. Fluorophore-conjugated secondary antibodies together with 3 μM DAPI were incubated in blocking buffer for 1 h at room temperature. The antibodies used are listed in Supplementary Table 2. Stained organoids were imaged on a Leica SP8 confocal microscope and analysed with Imaris (v.9.5.1). After standard dissection of mice, tumour tissue samples were collected and fixed in 10% neutral buffered formalin for 24 h. The tumours were then transferred to 70% ethanol for storage, followed by gradual dehydration in ethanol and embedding in paraffin. The paraffin-embedded tissues were sectioned into 4-μm-thick slices. H&E staining and DAB immunohistochemistry were conducted by the Histopathology Core Facility Breast Cancer Research at The Institute of Cancer Research using standard procedures. All staining was carried out using a Dako Autostainer Link48 (Agilent Technologies) immunostaining platform with epitope retrieval and simultaneous dewaxing using the Dako PT Link module (Agilent Technologies) according to the manufacturer's instructions. Formalin-fixed paraffin-embedded sections were cut at 4 µm onto APEX adhesive slides (Leica Biosystems), dried at 60 °C for 60 min and subject to epitope retrieval at 97 °C for 20 min using either Dako Target Retrieval Solution pH 9 (Agilent Technologies, K800421-2) or pH 6 (Agilent Technologies, K800521-2) as described below. Primary antibodies were diluted in Dako FLEX antibody diluent (Agilent, K800621-2) and endogenous peroxidases were blocked using Dako REAL peroxidase block (Agilent, S202386-2). Antibody binding reactions were visualized using Dako DAB+ (Agilent, K346811-2) and sections were counterstained using Dako FLEX Haematoxylin (Agilent, K800821-2). The following antibodies were used for the indicated experiments: CK19 DSHB rat monoclonal TROMA III (primary dilution 1:200), epitope retrieval = pH 9, detection system = anti-rat N-histofine polymer-HRP (Nichirei Biosciences, 414311F); vimentin, Cell Signaling Technology rabbit monoclonal D21H3, 5741 (primary dilution 1:200), epitope retrieval = pH 9, detection system = Dako rabbit EnVision polymer-HRP (Agilent, K400311-2). Stained slides were imaged on a NanoZoomer (Hamamatsu Photonics) slide scanner and analysed with QuPath (v.0.4.2)39. Analyses and grading of PDACs were performed by a certified pathologist based on H&E staining. For liver and lung metastases, H&E-stained sections of paraffin-embedded tissues were analysed, and the number of metastases per section was quantified using ImageJ. Paraffin sections from tumour tissues were prepared as described above, and antigen retrieval was performed by heating in 10 mM sodium citrate (Sigma-Aldrich) buffer (adjust pH 6.0 by using hydrochloric acid). The sections were blocked using blocking buffer (PBS containing 10% FCS and 1% BSA from Sigma-Aldrich) at room temperature for 1 h, followed by overnight incubation with primary antibodies prepared in blocking buffer at 4 °C in a cold room. On the next day, fluorophore-conjugated secondary antibodies (1:200) along with 3 μM DAPI (Sigma-Aldrich) in blocking buffer were applied in the dark at room temperature for 1 h. A list of the antibodies used is provided in Supplementary Table 2. Before mounting, slides were incubated in 0.1% (w/v) Sudan Black B (Sigma-Aldrich) in fresh 70% ethanol to reduce background signal. Stained slides were imaged on a Leica confocal microscope or a NanoZoomer (Hamamatsu Photonics) slide scanner and analysed using Imaris. The protocol for engineering organoids was adapted and modified from a previous study40. Organoids were dissociated into single cells using TrypLE and diluted to 1 × 106 cells per electroporation. The cell pellet was then resuspended in a precise 100 μl solution containing BTXpress solution (BTX), 7.2 μg PB-CMV-MCS-EF1a-RFP-T2A-Puro (System Biosciences) or pPB[Exp]-CMV>Hygro (Vectorbuilder), 2.8 μg CMV-pHyPBase or pRP[Exp]-mCherry-CAG>hyPBase (Vectorbuilder), and 10 μg PX330 (provided by F. Zhang, Addgene, 4223062) with a target gRNA. The cell–DNA mixture was transferred to 2 mm electroporation cuvettes (Nepagene) and electroporated using a NEPA21 super electroporator (Nepagene), with settings described in our previous publication41. After electroporation, the cell suspension was transferred to complete medium containing Y-27632 (10 μM) without antibiotics. After 30 min, cells were centrifuged at 300g for 5 min at 4 °C, resuspended in Matrigel and cultured in antibiotic-free complete medium supplemented with 10 μM Y-27632. On day 5 after electroporation, organoids were selected with 2 μg ml–1 puromycin or 100 mg ml–1 hygromycin B. After 3 days of selection, organoids were flow sorted into single cells into a 96-well plate with 30 μl Matrigel per well. gDNA was extracted using a Purelink Genomic DNA Minikit per the manufacturer's instructions (Thermo Fisher Scientific). PCR was performed using Q5 High-Fidelity 2× master mix (New England Biolabs) with pre-designed primers and gDNA. Subsequently, PCR products were amplified and cloned using a Zero Blunt PCR Cloning kit (Thermo Fisher Scientific) and sequenced by Sanger sequencing from Genewiz. A list of the gRNAs used is provided in Supplementary Table 3. Organoids or tissues were dissociated into single cells as mentioned above. Single cells were suspended in FACS buffer (PBS containing 2% FCS). The cells were incubated with fluorophore-conjugated isotype control or experimental antibodies in FACS buffer on ice in the dark for 30 min. A list of the antibodies used is provided in Supplementary Table 2. Cells were washed 3 times with PBS, filtered through a 40 μm nylon mesh (Corning) and then incubated in FACS buffer containing 3 μM DAPI or propidium iodide. Single-colour and fluorescence minus one staining control were used as necessary. The cells were analysed using FACSymphony A5 (BD Biosciences) analyzer. Data were analysed using FlowJo 10 software. The gating strategy is shown in Supplementary Fig. RNA was extracted from organoids using either a RNAqueous-Micro Total RNA Isolation kit (Thermo Fisher Scientific) or a RNeasy Mini kit (Qiagen) following the manufacturer's instructions. For BMP2 and LDN193189 treatment, organoids were starved in DMEM with 1% FCS overnight and treated with vehicle, 100 ng ml–1 recombinant BMP2 Protein (R&D Systems) or 100 nM LDN193189 (Tocris) for 12 h, followed by RNA extraction from the cells. cDNA was generated using an iScript cDNA synthesis kit (Bio-Rad) according to the manufacturer's instructions. The cDNA was diluted 1:10 in distilled water, and 2 μl was used per qPCR reaction. PowerUp SYBR Green master mix (Thermo Fisher Scientific) was used for qPCR on an ABI7500 or QuantStudio 6 Flex (Applied Biosystems) system. Primers were designed using the Primer-BLAST tool at NCBI and made by Sigma-Aldrich. Housekeeping gene mRNA levels (Actb or Rplp0) were used for normalization. mRNA levels were quantified using the difference in Ct values and calculated using the \({2}^{-\Delta \Delta {C}_{{\rm{t}}}}\) method42. A list of the qPCR primers used is provided in Supplementary Table 4. Matrigel was removed from cells on ice using Cell Recovery solution (Corning) following the manufacturer's instructions. The Matrigel-depleted cells were lysed in ice-cold cell lysis buffer (NEB) as previously described3. After brief sonication, lysates were centrifuged at 13,000 r.p.m. Protein quantification was performed using a Protein Assay kit (Bio-Rad), and proteins were separated by SDS–PAGE on 10% gels (Bio-Rad), then transferred onto PVDF membranes (Bio-Rad) using a Trans-Blot Turbo system (Bio-Rad) according to the manufacturer's instructions. The membranes were blocked in TBST containing 3% BSA at room temperature for 1 h, then incubated with primary antibodies prepared in TBST with 1% BSA overnight at 4 °C. HRP-conjugated secondary antibodies were subsequently incubated at room temperature for 1 h. Chemiluminescence was detected using Amersham ECL Prime Western Blotting Detection reagent (GE Healthcare) in an Amersham ImageQuant 800 imaging system. A list of the antibodies used is provided in Supplementary Table 2. GFP and inducible Grem1 were cloned into pLenti6 V5-DEST (Thermo Fisher Scientific) or pInducer20-Blast (Addgene, 109334) using the BP and LR Recombination System (Thermo Fisher Scientific). Lentiviral particles were produced in HEK293T cells using psPAX2 and pMD2.G helper plasmids (Addgene) and transfected into HEK293T cells using jetPRIME transfection reagent (jetPRIME). For organoid infection, a single-cell suspension was prepared and incubated with lentiviruses supplemented with 4 μg ml–1 polybrene (Santa Cruz Biotechnology) in a rocking incubator for 6 h. Cells were then seeded in Matrigel and antibiotic selection was applied 3 days later. To predict transcription factor binding motifs in the Bmp2 promoter, DNA sequences ranging from −1000 to +500 bp from the transcription start site were scanned in the JASPAR 2000 database (http://jaspar.genereg.net/) using a relative profile score threshold of 80%. Organoids were dissociated and plated on 2D culture dishes and cultured in DMEM with 5% FCS. After exponential growth, the cells were switched to DMEM containing 1% FCS and starved overnight, followed by treatment with vehicle or 5Z-7-oxozeaenol for 6 h. The cells were fixed and crosslinked chromatin was prepared as previously described3. For immunoprecipitation, 10 μg chromatin was incubated with 10 μl anti-histone H3 rabbit IgG (CST, used as a positive control), 2 μl normal rabbit IgG (CST) or 5 μl of anti-RelA antibody (CST) overnight at 4 °C. Before immunoprecipitation, 2% of the chromatin (v/v) was used as input. Protein G magnetic beads were used to capture chromatin–protein–antibody complexes, followed by reversal of crosslinking to release chromatin and purification using a SimpleChIP Enzymatic Chromatin IP kit (CST). DNA was quantified by qPCR using primers targeting predicted Rela-binding regions at the Bmp2 promoter. A list of the chromatin immunoprecipitation qPCR primers is provided in Supplementary Table 5. A list of the antibodies used is provided in Supplementary Table 2. For the RNA-seq analysis of Spp1 and Itgb3 WT and knockout organoids, RNA was extracted from organoids using a RNeasy Mini kit (Qiagen) following the manufacturer's instructions. mRNA capture and library preparation were performed by Genewiz. Biological replicate libraries were sequenced on an Illumina NovaSeq platform at Genewiz, which generated an average of approximately 25 million 150 bp paired-end reads per sample. Sequence reads were processed using Trimmomatic (v.0.36) to remove possible adapter sequences and nucleotides with poor quality43. Reads were aligned to the mouse reference assembly to the Mus musculus GRCm38 reference genome available on ENSEMBL using STAR aligner (v.2.5.2b)44. Normalization of raw counts and regularized log-transformation were performed as previously described3. For the RNA-seq dataset of Spp1 and Itgb3 WT and knockout organoids, GSEA (v.4.0.3) analysis was performed using the rlog-transformed counts, with the following settings applied: number of permutations = 1000, permutation type = gene set, enrichment statistics = weighted, metric for ranking genes = Signal2Noise. Normalized expression values of PDAC samples from the datasets GSE71729, ICGCarray, ICGCseq, TCGA, GSE62452 and GSE79668 were batch corrected using ComBat algorithm. Samples were grouped into SPP1high and SPP1low based on the median expression level. For GSEA, the following settings were applied: including the number of permutations = 1000, permutation type = phenotype, enrichment statistics = weighted, metric for ranking genes = Signal2Noise. The EMT gene set of 200 genes was checked for significant enrichment with an FDR q < 0.2. For human PDAC scRNA-seq data, fastq files of 24 PDAC and 11 normal pancreas samples were downloaded from the Chinese National Genomics Data Center (Genome Sequence Archive: CRA001160) and processed as previously described45. Uniform manifold approximation and projection plots were generated using the R package Seurat (v.3.2.2) with default settings. Cells were coloured according to cluster assignments from the reference. For evaluating SPP1 expression and its association with EMT in ductal cells (type 1 and 2), raw counts were normalized to counts per million (CPM) using the edgeR package in R (v.4.0.3) and log2[CPM + 1] transformed. Cells belonging to ductal clusters 1 and 2 were categorized into SPP1high and SPP1low based on expression levels. 7a, 10a, 11a and 12 were created in BioRender (https://biorender.com). All images and graphs represent the results of multiple experiments, which were conducted on different biological replicate samples and different dates. Differential gene expression analysis for RNA-seq data was performed using the edgeR package in R. All other statistical analyses were conducted using GraphPad Prism 8 for Windows. Specific statistical tests are specified in the figure legends. The survival plots for Spp1fl/fl KPFCT and Spp1WT/WT KPFCT mice receiving tamoxifen treatment and for Spp1fl/fl KPCY, Spp1WT/WT KPCY mice and SPP1 neutralization experiments were created and analysed using two-sided log-rank tests in GraphPad Prism 8. The following PDAC datasets were used: GSE71729, ICGCarray, ICGCseq, TCGA, GSE62452 and GSE79668. The normalized ComBat batch-corrected expression values were used for categorizing samples to SPP1high and SPP1low based on surv_cutpoint function in survminer R package. This function determines the optimal cut-off by maximally selected rank statistics. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. RNA-seq data for Spp1WT/WT KPFV and Spp1Δ/Δ KPFV organoids and for Itgb3WT/WT KPFV and Itgb3∆/∆ KPFV organoids have been deposited into the Gene Expression Omnibus under the accession codes GSE269971 and GSE269972, respectively. Source data are provided with this paper. Distant metastasis occurs late during the genetic evolution of pancreatic cancer. Lan, L. et al. GREM1 is required to maintain cellular heterogeneity in pancreatic cancer. Bill, R. et al. CXCL9:SPP1 macrophage polarity identifies a network of cellular programs that control human cancers. Ferreira, R. M. M. et al. Duct- and acinar-derived pancreatic ductal adenocarcinomas show distinct tumor progression and marker expression. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Heintz, N. Gene expression nervous system atlas (GENSAT). Tonelli, C. et al. A mucus production programme promotes classical pancreatic ductal adenocarcinoma. Nallasamy, P. et al. Pancreatic tumor microenvironment factor promotes cancer stemness via SPP1–CD44 axis. Dai, B. et al. Blockage of osteopontin–integrin β3 signaling in infrapatellar fat pad attenuates osteoarthritis in mice. & Johnson, G. A. Secreted phosphoprotein 1 (SPP1, osteopontin) binds to integrin alpha v beta 6 on porcine trophectoderm cells and integrin alpha v beta 3 on uterine luminal epithelial cells, and promotes trophectoderm cell adhesion and migration. Schönhuber, N. et al. A next-generation dual-recombinase system for time- and host-specific targeting of pancreatic cancer. Muzumdar, M. D., Tasic, B., Miyamichi, K., Li, L. & Luo, L. A global double-fluorescent Cre reporter mouse. Zheng, X. et al. Epithelial-to-mesenchymal transition is dispensable for metastasis but induces chemoresistance in pancreatic cancer. Yamada, S. et al. Epithelial-to-mesenchymal transition predicts prognosis of pancreatic cancer. Aiello, N. M. et al. EMT subtype influences epithelial plasticity and mode of cell migration. Taniguchi, K. & Karin, M. NF-κB, inflammation, immunity and cancer: coming of age. Both cIAP1 and cIAP2 regulate TNFα-mediated NF-κB activation. Fornes, O. et al. JASPAR 2020: update of the open-access database of transcription factor binding profiles. Ruiz, E. J. et al. LUBAC determines chemotherapy resistance in squamous cell lung cancer. Ghosh-Choudhury, N. et al. Autoregulation of mouse BMP-2 gene transcription is directed by the proximal promoter element. Ghosh-Choudhury, N., Abboud, S. L., Mahimainathan, L., Chandrasekar, B. & Choudhury, G. G. Phosphatidylinositol 3-kinase regulates bone morphogenetic protein-2 (BMP-2)-induced myocyte enhancer factor 2A-dependent transcription of BMP-2 gene in cardiomyocyte precursor cells. The roles of intratumour heterogeneity in the biology and treatment of pancreatic ductal adenocarcinoma. Milan, M., Diaferia, G. R. & Natoli, G. Tumor cell heterogeneity and its transcriptional bases in pancreatic cancer: a tale of two cell types and their many variants. Espinet, E., Klein, L., Puré, E. & Singh, S. K. Mechanisms of PDAC subtype heterogeneity and therapy response. Introduction to machine learning: k-nearest neighbors. Analysis of lung tumor initiation and progression using conditional expression of oncogenic K-ras. Marino, S., Vooijs, M., van Der Gulden, H., Jonkers, J. Induction of medulloblastomas in p53-null mutant mice by somatic inactivation of Rb in the external granular layer cells of the cerebellum. Olive, K. P. et al. Mutant p53 gain of function in two mouse models of Li–Fraumeni syndrome. Hingorani, S. R. et al. Preinvasive and invasive ductal pancreatic cancer and its early detection in the mouse. Srinivas, S. et al. Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus. Conditional deletion of gremlin causes a transient increase in bone formation and bone mass. Flanagan, S. P. ‘Nude', a new hairless gene with pleiotropic effects in the mouse. Huynh, A. S. et al. Development of an orthotopic human pancreatic cancer xenograft model using ultrasound guided injection of cells. Hay, C. A., Sor, R., Flowers, A. J., Clendenin, C. & Byrne, K. T. Ultrasound-guided orthotopic implantation of murine pancreatic ductal adenocarcinoma. Boj, S. F. et al. Organoid models of human and mouse ductal pancreatic cancer. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Fujii, M., Matano, M., Nanki, K. & Sato, T. Efficient genetic engineering of human intestinal organoids using electroporation. Nelson, J. K. et al. USP25 promotes pathological HIF-1-driven metabolic reprogramming and is a potential therapeutic target in pancreatic cancer. & Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the \({2}^{-\Delta \Delta {C}_{{\rm{t}}}}\) Method. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Peng, J. et al. Single-cell RNA-seq highlights intra-tumoral heterogeneity and malignant progression in pancreatic ductal adenocarcinoma. We thank Breast Cancer Now for funding this work as part of Programme Funding to the Breast Cancer Now Toby Robins Research Centre and grants from the National Natural Science Foundation of China (no. We thank staff at the Breast Cancer Now Toby Robins Research Centre Translational Research Support in Vitro Modelling Team, the Human Organoid Facility at the CRUK Convergence Science Centre (CTRQQR-2021\100009) and the Institute of Cancer Research Proteomics Core Facility for their technical support. Present address: Department of Hepatobiliary Surgery, Jining Public Health Medical Center, Jining, China Present address: Department of Oncology, Affiliated Hospital of YouJiang Medical University, Baise, China Present address: Centre for Physical Mechanics and Biophysics, Sun Yat-sen University, Guangzhou, China These authors contributed equally: Huafu Li, Linxiang Lan, Hengxing Chen The Breast Cancer Now Toby Robins Research Centre, The Institute of Cancer Research, London, UK Huafu Li, Linxiang Lan, May Zaw Thin, Jessica K. Nelson, Ian M. Evans, E. Josue Ruiz, Rongjie Cheng, Li Tran & Axel Behrens The Seventh Affiliated Hospital, Sun Yat-sen University, Shenzhen, China Linxiang Lan, Hengxing Chen, Jian Ma, Tingzhuang Yi, Chunming Wang, Yulong He & Changhua Zhang Systems and Precision Cancer Medicine Laboratory, Division of Molecular Pathology, The Institute of Cancer Research, Sutton, UK Biological Services Unit, The Institute of Cancer Research, London, UK Nina Barough Pathology Core Facility, The Breast Cancer Now Toby Robins Research Centre, The Institute of Cancer Research, London, UK Aix Marseille University, CNRS, CPT (UMR 7332), Turing Centre for Living Systems, Marseille, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar participated in manuscript preparation and formal analyses and performed most of the experiments. and H.C performed proteomics on blood samples from patients with pancreatic cancer and in vivo and in vitro experiments using patient-derived organoids. helped with ultrasound imaging of pancreatic tumours and pancreas orthotopic transplantation. assisted with the study design and manuscript preparation. All authors discussed the results and implications and commented on the manuscript. Correspondence to Changhua Zhang or Axel Behrens. The authors declare no competing interests. Nature thanks Dieter Saur and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, UMAP plot of single cells from a scRNA-seq dataset for human PDACs (24 PDAC and 11 normal pancreas samples). b, Cancer cells were categorized into the high SPP1 expression population (SPP1hi, n = 6980 cells) and the low SPP1 expression population (SPP1lo, n = 6981 cells) groups based on the expression of SPP1. c, EMT signature score plot for SPP1 high (n = 6980 cells) and SPP1 low (n = 6981 cells) cancer cell populations. d, Normalised RNA read counts (RSEM) for SPP1 from the PDAC dataset of GSE71729, ICGCarray, ICGCseq, TCGA, GSE62452 and GSE79668. The samples were categorised into SPP1hi (n = 321) and SPP1lo groups (n = 323) based on its expression, higher or lower than the median value. e, GSEA plot for enrichment of the hallmark EMT gene set in the transcriptome of SPP1hi PDACs. NES, normalized enrichment score; FDR, false discovery rate. f, Kaplan-Meir test for the survival analysis of SPP1 high expression patients (n = 321) and SPP1 low expression patients (n = 323). P value was calculated with the logrank test. g, KPFV pancreatic cancer mouse model schematic diagram. h, Validation of CRISPR/Cas9 Spp1 knockout by Western blot. Gel source data are provided in Supplementary Fig. j, Immunofluorescence staining for DAPI (Blue), GFP (gray), Krt19 (green), and Vimentin (red) of KPF; Spp1wt/wt (n = 3 mice) and KPF; Spp1Δ/Δ subcutaneous tumours (n = 2 mice). For b, c, d data, P value is from two-sided Mann-Whitney test. a, Immunofluorescent analysis of Vimentin (yellow), DAPI(blue), and Krt19 (red) in KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression organoids. b, RT-qPCR analysis of the expression of epithelial markers Krt19, Cdh1, and Epcam, and mesenchymal markers Fn1, S100a4, and Vim between KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression organoids (n = 3 biological replicates), normalised using KPR172H/+C; Plv-con values. c, Immunofluorescent staining for GFP (gray), Krt19 (green), and Vimentin (red) in KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression allografts. P values were calculated using two-sided t-tests. a, RT-qPCR analysis of the expression of epithelial markers KRT19, CDH1, and EPCAM, and mesenchymal markers FN1, S100A4, and VIM between Human PDAC; SPP1wt/wt and Human PDAC; SPP1Δ/Δ organoids (n = 3 biological replicates), normalised using SPP1wt/wt organoids values. b, Validation of CRISPR/Cas9 SPP1 knockout by Western blot. Gel source data are provided in Supplementary Fig. 1. c, Image of the subcutaneous tumours formed by Human PDAC; SPP1wt/wt and Human PDAC; SPP1Δ/Δ organoids. Scale bar length is 0.5 cm. e, Immunofluorescence staining for KRT19 (green), and VIMENTIN (red) of subcutaneous tumours from (c). i, RT-qPCR analysis of the expression of epithelial markers KRT19, CDH1, and EPCAM, and mesenchymal markers FN1, S100A4, and VIM between Human PDAC; Plv-con and Human PDAC; SPP1 overexpression organoids (n = 3 biological replicates), normalised using Plv-con organoids values. j, Image of the subcutaneous tumours formed by Human PDAC; Plv-con and Human PDAC; SPP1 overexpression organoids. Scale bar length is 0.5 cm. l, Immunofluorescence staining for KRT19 (green), and VIMENTIN (red) of subcutaneous tumours from (j). n, o, H&E and immunohistochemistry for KRT19 of subcutaneous tumours from (j). For a, d, f, i, k, m data are mean ± s.e.m. P values were calculated using two-sided t-tests. a, Flow cytometry for Cd61 in Cd61wt/wt and Cd61Δ/Δ PDAC organoids. b, GSEA of the hallmark EMT gene set between of KPFV; Cd61wt/wt and KPFV; Cd61Δ/Δ transcriptomes. d, Comparison of the percentages of Cd45−/Cd31−/GFP− in tumors from KPFCT; Spp1wt/wt and KPFCT; Spp1fl/fl mice (n = 5 mice). e, Immunofluorescence staining for aSMA (red), GFP (green), and Vimentin (Pink) of KPFCT; Spp1wt/wt and KPFCT; Spp1fl/fl tumours (n = 5 mice). f, Comparison of the percentages of aSMA cell subpopulations with Cancer cells between tumours formed by KPCTF; Spp1wt/wt and KPFCT; Spp1 fl/fl. For d, f, g, h, i, j, k, l, m and n data are mean ± s.e.m. P values were calculated using two-sided t-tests. a, H&E and immunohistochemistry of Krt19 in tumors from KPhetFCT; Spp1wt/wt and KPhetFCT; Spp1fl/fl mice. b, Immunofluorescence staining for GFP (gray), Krt19 (green), and Vimentin (red) of KPhetFCT; Spp1wt/wt and KPhetFCT; Spp1fl/fl tumours. a, Ultrasound imaging of pancreatic tumours in Pancreas Orthotopic transplantation mice. b, H&E immunohistochemistry in tumour from KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression Pancreas Orthotopic transplantation mice. c, Percentage of mice with of liver metastases (left) and quantification of number of liver metastases (right) in KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression Pancreas Orthotopic transplantation mice (n = 4 mice). d, Percentage of mice with of lung metastases (left) and quantification of number of lung metastases (right) in KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression Pancreas Orthotopic transplantation mice (n = 4 mice). e, Immunohistochemistry for Vimentin in livers from KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression Pancreas Orthotopic transplantation mice. f, Immunohistochemistry for Krt19 in lung from KPR172H/+C; Plv-con and KPR172H/+C; Spp1 overexpression Pancreas Orthotopic transplantation mice. c (left) and d (left) P values were calculated using two-sided Fisher exact test. a, Scheme depicting the experimental approach for Spp1 neutralization experiments in KPR172H/+CY Model. b, Kaplan-Meir test for the survival analysis of KPR172H/+CY mice treat with Spp1 antibody or IgG control. P value was calculated with the logrank test. c, Percentage of mice with of liver metastases (left) and quantification of number of liver metastases (right) in KPR172H/+CY mice treat with Spp1 antibody or IgG control (n = 12 mice). d, Percentage of mice with of lung metastases (left) and quantification of number of lung metastases (right) in KPR172H/+CY mice treat with Spp1 antibody or IgG control (n = 12 mice). e, Immunofluorescent staining for YFP (gray), Krt19 (green), and Vimentin (red) in KPR172H/+CY mice treat with Spp1 antibody or IgG control. f, Comparison of the proportions of different cancer cell subpopulations between KPR172H/+CY mice treat with Spp1 antibody or IgG control tumours (n = 5 mice). g, H&E immunohistochemistry in tumors from KPR172H/+CY mice treat with Spp1 antibody or IgG control. h, Immunohistochemistry of Krt19 in tumors from KPR172H/+CY mice treat with Spp1 antibody or IgG control. i, Immunohistochemistry for Krt19 in livers from KPR172H/+CY mice treat with Spp1 antibody or IgG control mice. j, Immunohistochemistry for Krt19 in lung from KPR172H/+CY mice treat with Spp1 antibody or IgG control. c (left) and d (left) P values were calculated using two-sided Fisher exact test. c (right), d (right) and f, P values were calculated using two-sided t-tests. a, RT-qPCR analysis of the expression of Id1, Id2, Id3 and Grem1 in control and Cd61 overexpressing KPFV organoids. b, GSEA of the TNFα NF-κB hallmark gene set between KPFV; Cd61wt/wt and KPFV; Cd61Δ/Δ organoids. c, RT-qPCR analysis of the NF-κB signaling pathway target genes Ccl2, Clap2, and Tnf in KPFV; Cd61wt/wt and KPFV; Cd61Δ/Δ organoids. d, RT-qPCR analysis of Ccl2, Clap2, and Tnf expression in control and Cd61 overexpressing KPFV organoids. Values are normalised to the control. e, Potential binding sites of RelA in the Bmp2 promoter were predicted using JASPAR. f, Schematic diagram of the binding sites of RelA to the Bmp2 promoter. g, ChIP analysis for RelA binding to the Bmp2 promoter and Tnf promoter in KPFV organoids. ChIP was conducted using IgG control or RelA antibody, with or without treatment with TAK1 inhibitor (5Z-7), followed by qPCR analysis. Values are normalised to the input. h, RT-qPCR analysis of the expression of Bmp2 in Vehicle and TAK1 inhibitor (5Z-7) treated organoids. Values are normalised to vehicle treatment. i, RT-qPCR analysis of the expression of Spp1, Id1, Id2, Id3, Bmp2 and Grem1 in KPF organoids treated with the Bmp2 inhibitor LDN193189 and vehicle. Values are normalised to the vehicle. j, Comparative RT-qPCR analysis of Id1, Id2, Id3, Spp1, Krt19 and Vim, in KPF organoids treated with Bmp2 for 0, 3, 6, 9 and 24 h. Values are normalised to Bmp2 0 h treatment. k, Comparative RT-qPCR analysis of Id1, Id2, Id3, Spp1, Krt19 and Vim, in KPF organoids treated with LDN193189 for 0, 3, 6, 9 and 24 h. Values are normalised to the LDN193189 0 h treatment. For a, c, d, g, h, i, j, k data are mean ± s.e.m and n = 3 biological replicates. P values were calculated using two-sided t-tests. a, Schematic diagram showing the mouse model KPFCT; Grem1fl/fl. b, Schematic diagram showing the KPFCT; Spp1fl/fl; Grem1fl/fl mouse model. a, Scheme depicting the experimental approach for establishing tumours from Spp1 CRISPR/Cas9 knockout and Tamoxifen-inducible Grem1 knockout organoids. b, RT-qPCR analysis of the expression of epithelial markers Krt19, Cdh1, and Epcam, and mesenchymal markers Fn1, S100a4, and Vim in organoids from either KPFC; Spp1 wt/wt; Grem1wt/wt, KPFC; Spp1Δ/Δ; Grem1fl/fl organoids and subsequently treated with either vehicle or with 4-OH Tamoxifen as noted in the figure (n = 3 biological replicates); all data was normalised against the KPFC; Spp1 wt/wt; Grem1wt/wt group. c, Image of the subcutaneous tumours generated by the implantation of KPFC; Spp1Δ/Δ; Grem1fl/fl organoids into nude mice and subsequently treated with either vehicle or with Tamoxifen as noted in the figure. Scale bar length is 1 cm. d, Quantification of the subcutaneous tumours detailed in (c) (n = 5 mice). e, Immunofluorescence staining for GFP (gray), Krt19 (green), and Vimentin (red) of KPFC; Spp1Δ/Δ; Grem1fl/fl & GFP (treated with vehicle) and KPFC; Spp1Δ/Δ; Grem1fl/fl & GFP (treated with Tamoxifen) subcutaneous tumours. f, Comparison of the proportions of epithelial cancer cell subpopulations between the tumours detailed in (e) (n = 5 mice). g, Comparison of the proportions of mesenchymal cancer cell subpopulations the tumours detailed in (e). h, Immunohistochemistry for H&E from subcutaneous tumours formed by KPFC; Spp1Δ/Δ; Grem1fl/fl &GFP treated with vehicle or Tamoxifen. For b, d, f and g data are mean ± s.e.m. P values were calculated using two-sided t-tests. a, Scheme depicting the experimental approach for establishing tumours from Spp1 overexpression and doxycycline-inducible Grem1 overexpression organoids. b, RT-qPCR analysis of the expression of epithelial markers Krt19, Cdh1, and Epcam, and mesenchymal markers Fn1, S100a4, and Vim in organoids from either KPF; Plv-GFP&pINDUCER-con or KPF; Plv-Spp1-GFP&pINDUCER-Grem1 mice treated with either vehicle or doxycycline as noted in the figure (n = 3 biological replicates). c, Image of the subcutaneous tumours formed by KPF; Plv-Spp1-GFP&pINDUCER-Grem1 organoids treated with either vehicle or Doxycycline as indicated. Scale bar length is 1 cm. d, Quantification of the subcutaneous tumours detailed in (c). e, Immunofluorescence staining for GFP (gray), Krt19 (green), and Vimentin (red) of the subcutaneous tumours formed by KPF; Plv-Spp1-GFP&pINDUCER-Grem1 organoids treated with either vehicle or Doxycycline as indicated (n = 5 mice). f, Comparison of the proportions of epithelial cancer cell subpopulations between of the subcutaneous tumours detailed in (e) (n = 5 mice). h, Immunohistochemistry for H&E from subcutaneous tumours formed by KPF; Plv-Spp1-GFP&pINDUCER-Grem1 treated with vehicle or Doxycycline. For b, d, f and g data are mean ± s.e.m. P values were calculated using two-sided t-tests. Schematic cartoon depicting the mechanism for the paracrine interaction between epithelial and mesenchymal cancer cells. This diagram was created in BioRender. Uncropped raw images for western blots. 1h, anti-SPP1 and β-actin were run on the same gel. 3b, anti-SPP1 and β-actin were run on the same gel. Gating strategies for flow cytometry analysis. Gating strategies for flow cytometry analysis of Extended Data Fig. Clinicopathological characteristics of patients with pancreatic cancer. Information includes the patient's sex, age, histology, differentiation, TNM stage and AJCC (8th edn) staging. A detailed list of antibodies used in the study. Sequences of gRNAs used in the study. Included are the sequences of sgSpp1-1, sgSpp1-2, sgSPP1 -1, sgSPP1-2, sgCd61-1 and sgCd61-2. Sequences of RT–qPCR primers used in the study. Sequences of ChIP–qPCR primers used in the study. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Li, H., Lan, L., Chen, H. et al. SPP1 is required for maintaining mesenchymal cell fate in pancreatic cancer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68035032/hidden-edge-state-atoms-energy-transmission/'>Scientists Found the Hidden 'Edge State' That May Lead to Practically Infinite Energy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 13:32:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Take, for example, what is known as the quantum Hall effect. First discovered in 1980 by German physicist Klaus von Klitzing, this effect describes the behavior of electrons (under the influence of a magnetic field and approaching absolute zero temperatures) as they pass through 2D materials, such as graphene. Usually, you'd expect the electrons to experience resistance and scatter, but under these conditions, they formed lossless energy states locked along the material's boundary. This quantization of electrical resistance, known as an “edge state,” is particularly useful if you want to create exotic materials free of electrical resistance. “These states occur over femtoseconds, and across fractions of a nanometer, which is incredibly difficult to capture,” Richard Fletcher, an assistant professor at MIT, said in a press statement. “The beauty is seeing with your own eyes physics which is absolutely incredible but usually hidden away in materials and unable to be viewed directly.” To actually study this quantum interaction on a more reasonable scale, Fletcher—along with his colleagues at MIT's Research Laboratory of Electronics and the MIT-Harvard Center for Ultracold Atoms—decided on a novel method to essentially scale up this phenomenon using a cloud of ultracold sodium atoms instead of electrons. According to the researchers, this allowed the team to watch these edge states form “over milliseconds and microns,” which are much more manageable experimental parameters. The team used one million ultracold sodium atoms and essentially trapped them in a complex set-up of lasers. However, to simulate the experience of living in a flat space, the researchers also spun them like “riders on an amusement park Gravitron.” “The trap is trying to pull the atoms inward, but there's centrifugal force that tries to pull them outward,” Fletcher said. There's also a third force, the Coriolis effect, such that if they try to move in a line, they get deflected. So these massive atoms now behave as if they were electrons living in a magnetic field.” Once atoms encountered this light, they flowed in just one direction—much like electrons at ultra-small quantum scales. “You can imagine these are like marbles that you've spun up really fast in a bowl, and they just keep going around and around the rim of the bowl,” Martin Zwierlein, a co-author of the study, said in a press statement. To test these atoms' resistances, the team then placed obstacles—such as a point of light—in their paths, and the atoms passed by without any measurable resistance. Now that scientists have a reliable stand-in for this quantum process, future experiments can push these interactions to the “edge” and begin to explore unknown frontiers of this fascinating piece of quantum physics. Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a67994479/new-discovery-cleopatras-tomb/'>A New Discovery Could Help Lead Archaeologists to Cleopatra's Tomb</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>She wielded political power by allying her kingdom with Rome and, at one point, was the richest woman on Earth. Rumors of its location have been swirling since Cleopatra's death around two thousand years ago. She and Antony were supposedly buried together after Egypt was defeated by the forces of Augustus Caesar, (with him falling on his sword and her reportedly allowing a deadly asp to take her life), but neither body has ever surfaced. According to the Egyptian Ministry of Tourism and Antiquities, archaeologist and Cleopatra enthusiast Kathleen Martinez (along with her research team) recently discovered a port on the ocean floor that was unmistakably linked to a temple that Martinez believes could hold the body of the infamous queen. It has long been thought that Cleopatra had been interred somewhere in her birthplace of Alexandria, but Martinez had a different idea. She believes that the queen actually sought her final resting place in a temple of Isis. Built by pharaoh Ptolemy II Philadelphus (an ancestor of Cleopatra) between 280 and 270 B.C.E., it used to be a grand temple, but over time, Taposiris Magna has been reduced to crumbling walls and eroded monuments rising out of the sand. Thousands of years ago, the port stood firmly on land. The coastline was four kilometers (about 2.5 miles) further out during Cleopatra's time, but today, it has encroached to the point that the port is in a much more waterlogged state than it would have been upon construction. The seafloor was littered with anchors and amphora.Martinez had previously explored a subterranean tunnel that led from Taposiris Magna to the Mediterranean Sea, which ended in a group of structures her team refers to as Salam 5. The sunken port is an extension of that, and adds to the trove of objects previously unearthed at Taposiris Magna. Prior finds include coins stamped with Cleopatra's profile, bronze statuettes, oil lamps, ceramic vessels used in rituals, and limestone vessels for storage. Whether the tomb of Cleopatra could have also have ended up submerged is still unknown, and its location still eludes Martinez. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. An Archaeology Student Dug Up a Rare Metallic Face A ‘Dead' Man Woke Up—and Showed Death Isn't Final This Crystal Is a Map of Earth's Galactic Journey</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250924012222.htm'>Tylenol in pregnancy linked to higher autism risk, Harvard scientists report</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-24 06:10:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When children are exposed to acetaminophen -- also known by the brand name Tylenol or as paracetamol -- during pregnancy, they may be more likely to develop neurodevelopmental disorders (NDDs) including autism and ADHD, according to a new study. The study was published recently in BMC Environmental Health. The researchers used the Navigation Guide Systematic Review methodology -- a gold-standard framework for synthesizing and evaluating environmental health data -- which enabled them to conduct a rigorous, comprehensive analysis that supported evidence of an association between acetaminophen exposure during pregnancy and increased incidence of NDDs. High fever can raise the risk of neural tube defects and preterm birth. "We recommend judicious acetaminophen use -- lowest effective dose, shortest duration -- under medical guidance, tailored to individual risk-benefit assessments, rather than a broad limitation," they wrote. In late September, the Food and Drug Administration announced it would issue a letter to clinicians urging them to be cautious about the use of acetaminophen in pregnancy. Baccarelli said he had discussed his study with Health and Human Services Secretary Robert F. Kennedy Jr. in the weeks leading up to that announcement and provided the White House team with an statement noting his research found "evidence of an association" between prenatal exposure to acetaminophen and neurodevelopmental disorders. The statement continued: "Further research is needed to confirm the association and determine causality, but based on existing evidence, I believe that caution about acetaminophen use during pregnancy -- especially heavy or prolonged use -- is warranted." Baccarelli noted in the "competing interests" section of the research paper that he has served as an expert witness for plaintiffs in a case involving potential links between acetaminophen use during pregnancy and neurodevelopmental disorders. Note: Content may be edited for style and length. Green Tea Shows Promise in Fighting Obesity and Diabetes Even One Drink May Raise Dementia Risk, Landmark Study Warns Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            