
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44460-025-00005-z'>Wearable lateral flow assays for cortisol monitoring with time-dynamic sweat sampling and sensing by electrochromic timers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 16:54:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Sensors volume 1, pages 85–98 (2026)Cite this article Cortisol is a key regulator of stress and circadian physiology, yet current monitoring relies on invasive blood sampling or saliva assays that are prone to contamination and provide limited temporal resolution. Wearable sweat cortisol sensors are promising, but require electronic sensing systems and have limited capability for long-term, time-sequenced monitoring. Here we present a wearable paper-based microfluidic platform that integrates plasmonic-gold-nanoflower-based colorimetric assays to enable non-invasive tracking of cortisol in eccrine sweat. Sweat is induced by carbachol iontophoresis and directed through collection channels using either electronically timed sequential activation or paper-based delay valves with self-powered electrochromic indicators. In human studies, the system resolved circadian variations, acute stress responses to cold pressor challenges, and jet-lag-associated disruptions, with the results closely matching those from saliva and serum assays. This wearable lateral flow technology establishes sweat as a viable medium for real-time hormone monitoring and may enable personalized management of stress, sleep and circadian misalignment. You have full access to this article via your institution. Managing stress is a critically important aspect of maintaining good physical and mental health1. Chronic exposure to elevated concentrations of cortisol can have detrimental effects on the cardiovascular, immune, renal and endocrine systems1. Misalignment of circadian rhythms, also controlled by cortisol, can likewise lead to adverse health effects4,5. Knowledge of cortisol levels can, therefore, potentially assist in the management of sleep patterns and stress-related disorders, including those related to post-traumatic stress disorder and chronic fatigue syndrome6. Despite the importance of cortisol, monitoring its concentration in biofluids remains challenging. Saliva is an attractive alternative to blood, but one that is prone to contamination by substances in the mouth7. Eccrine sweat is therefore of increasing interest3,6,8,9,10,11. Wearable devices for electrochemical sensing of cortisol in sweat12,13,14 offer important capabilities but require electronics for sensing and offer limited capacity for the time-dynamic detection of cortisol over hours. Conventional colorimetric sensors have some potential, but face challenges due to the extremely low concentrations of cortisol present in sweat3. Lateral flow immunoassays that use colorimetric readout have proven utility in sensing biomarkers in blood serum, urine, nasal swabs and saliva. For measuring cortisol, such assays operate based on competitive binding between cortisol and a conjugate of bovine serum albumin (BSA) and cortisol (BSA–CTS), both of which bind anti-cortisol antibodies on plasmonic nanoparticles. The sensitivity depends on various physical, chemical and biological factors, including label type, bioreceptor affinity and flow rate15. Among these, the optical label is critically important. Relative to commonly used gold nanoparticles (AuNPs), gold nanoflowers (AuNFs) are particularly attractive for this purpose, due to their high surface areas and enhanced extinction coefficients associated with localized surface plasmon resonances16,17,18. Their complex surface structure also improves antibody immobilization efficiency18. Previous studies have demonstrated these enhancements in lateral flow assays (LFAs), but their application in detecting cortisol remains unexplored. This Article reports a skin-interfaced device that combines paper-based microfluidic structures and LFAs that use AuNFs for the sampling and analysis of sweat in a time-sequential fashion following iontophoretic delivery of carbachol through the surface of the skin. The results include versions of the device with two different sampling modes: (1) a pre-programmed electronic timer circuit for sequenced iontophoresis events; and (2) a passive valve structure and self-powered electrochromic timer with which to capture distinct sampling events. Demonstrations of this technology involve studies on the diurnal cycle, cold pressor tests (CPTs) and responses to jet lag. Levels of cortisol in sweat measured in this fashion align well with separate assays based on saliva and serum. The results suggest the potential for routine biochemical assessments of stress and mental health, to guide therapeutic intervention. The platform comprises four major components: an encapsulation layer that caps the entire system; an integrated iontophoresis module with an electronic timer for time-triggered sweat stimulation; a set of agarose hydrogels containing carbachol (blue; carbagel) or KCl (yellow); and a skin-interfaced LFA module with four separate assays (Fig. Each channel has a separate sample pad to transfer sweat towards a conjugation pad, which contains AuNFs that are conjugated with an anti-cortisol antibody (AuNF–Ab conjugates) (Fig. An exploded view of the LFA (Fig. The assay includes a polyethylene terephthalate (PET) substrate with an adhesive layer, microfluidic spacers, nitrocellulose membranes, conjugation pads, sample pads, absorbent pads and viewing windows made of superhydrophilic and superhydrophobic materials. Figure 1h presents an image of an LFA on the body, displaying the test and control lines. Figure 1i illustrates the principles and operation for time-sequenced iontophoretic stimulation at 20-min intervals, with sweat flow visualized by green food dyes spotted on the conjugation pad. a, Exploded schematic of the device, comprising an encapsulation cover, an iontophoretic stimulation module, hydrogels containing carbachol (blue) and KCl (yellow), and the skin-interfaced LFA. b, 3D representation of the device with an iontophoretic stimulation module and encapsulation layer. c, Magnified schematic of simultaneous sweat stimulation, collection onto cellulose paper and transport to an LFA strip via a sample pad. d, Detailed schematic of the LFA on a PET substrate. f, Photograph of an LFA with patterned hydrogels placed on the skin for iontophoretic stimulation. g, Photograph of the wearable LFA with the iontophoretic stimulation module and an encapsulation layer. i, A series of optical images demonstrating a time-sequenced series of iontophoretic stimulation events. The sample pad contains a phosphate buffer (1 M; 2 μl) that adjusts the pH of entering sweat to 6.5. At physiologically relevant sweat rates (0.5–1.0 μl min−1), the transit time of ~6–12 min provides sufficient incubation for cortisol to fully bind to the antibody3. This scheme offers limits of detection that span into picomolar concentrations, consistent with physiological concentrations of cortisol in sweat. a, Schematic of the LFA for sensing sweat cortisol based on the use of AuNFs and competitive mechanisms. d, Size distributions of AuNFs measured by dynamic light scattering. e–g, Standard calibration curves of the LFA with artificial sweat using the test line (e), control line (f) and control/test ratio (g). n = 3. h, Effect of flow rate on the control/test line ratios for cortisol concentrations of 0 and 100 ng ml−1. n = 3. i, Performance of LFA calibration curves for estimating the cortisol concentration in real sweat of various baseline cortisol concentrations versus artificial sweat. Sweat samples were drop-cast onto the LFA assays. n = 3. j, LFA versus ELISA for estimating the concentration of cortisol in real sweat. n = 3. k, Stability of the LFA signal with time at cortisol concentrations of 0 and 100 ng ml−1. n = 3 before optimization of the AuNF volume. Upon reduction of gold ions by thiolate complex formation, the clear solution turns reddish brown, indicating the formation of AuNFs. Figure 2b shows the ultraviolet absorbance spectrum of AuNFs, with a maximum peak at 558 nm, resulting from their larger size compared with typical AuNPs. Figure 2c presents a transmission electron microscopy image of a typical AuNF, with an average size of 164 nm, as determined from dynamic light scattering (Fig. In this competitive LFA assay, AuNF–Ab conjugates bind to the immobilized BSA–CTS antigen at the test line in the absence of cortisol in sweat, forming a distinct and intense band (Extended Data Fig. The residual AuNF–Ab conjugates, both with and without cortisol, subsequently interact via their fragment crystallizable regions with the anti-mouse IgG antibody at the control line. Increasing the cortisol concentration (0–100 ng ml−1) results in a progressive fading of the test line colour due to the partial or full occupation of anti-cortisol antibody with sweat cortisol and competition with BSA–CTS at the test line, which disappears entirely at approximately 1,000 ng ml−1 (Fig. Similarly, the control line becomes more pronounced with increasing cortisol concentration (Fig. Thus, the LFA demonstrates high-contrast bright bands at both the test and control lines, particularly at low and high cortisol concentrations. As such, the control-to-test-line ratios provide sensitive quantification of cortisol concentrations. The optimization of adjusted sweat pH levels (Extended Data Fig. 1h,i), working buffers (Extended Data Fig. 1l,m) and anti-cortisol antibody (Extended Data Figs. 1m and 2a) yields a highly sensitive assay, as demonstrated by the standard calibration curve for artificial sweat solution spiked with cortisol concentrations ranging from 100 pg ml−1 to 100 ng ml−1 at a flow rate of 0.5 μl min−1 (Fig. The calibration curve covers the physiologically relevant concentrations of cortisol in sweat, which extends to 100 ng ml−1. The results indicate negligible sensitivity to sweat flow rate and injection volume, over relevant ranges, after optimization (Fig. The standard calibration curves obtained for three different real sweat samples with baseline cortisol concentrations of 1.0, 0.9 and 0.3 ng ml−1, as well as artificial sweat samples, all of which are spiked with cortisol, are consistent (Pearson's correlation coefficient, r = 0.99; Fig. Furthermore, cortisol concentrations obtained from the LFA assay correlate with those measured using an enzyme-linked immunoassay (ELISA) kit (r = 0.94; Fig. Quantitative readouts of test line intensity under varying illumination conditions (4,500–5,500 K) and using various imaging devices (digital and smartphone cameras) show negligible signal variation, supporting the robustness of the assay (Extended Data Fig. Protocols for analysing the LFA signals appear in Extended Data Fig. 3a–g and the Supplementary Information section ‘Analysis of wearable LFA'. The LFA signals remain consistent for up to 6 h after assay completion, which is critical for time-sequenced iontophoretic sweat sampling and detection over several hours (Fig. Optical images of the time-sequenced iontophoresis module appear in Extended Data Fig. 4d–f show a block diagram and connection scheme. First, activating the long-term timer (TPL5110) with a switch generates a pulse signal that triggers the short-term timer (TLC551). The short-term timer, pre-configured in one-shot mode, then activates one of the step-up converters (R1218) from the four channels (initially set to 00) through a 2-to-4 decoder (SN74LVC1G139) connected to a 2-bit counter (74HC73). During this activation, direct electrical current flows from positive (+) to negative (−) bottom electrodes, thereby delivering carbachol locally through the skin to trigger sweat production. After a set duration (for example, 5 min), the short-term timer turns off and shifts the counter by one position. 3b also confirms the time-sequenced series of iontophoretic sweat stimulation. This stimulation sequence repeats according to the preset timing parameters. 4g,h presents the results for stimulation time interval (tinterval) versus long timer resistance (RL) and stimulation time (ton) versus short timer resistance (Rs), respectively. 4i demonstrates a low level of timing variability. a, Electrical circuit block diagram and connection mechanisms for the time-sequenced iontophoretic stimulation module. b, Timing diagram showing output voltages for each iontophoretic channel in response to the voltage-triggering signals for long- and short-term timers. e, Effect of the separation distance between the stimulation area and collection channel on the volume of sweat for participants 1 and 2 and FEA simulation. n = 3. f, Effect of the separation distance between the stimulation area and collection channel on sweat rate for participants 1 and 2. n = 3. Dashed line corresponds to FEA simulation. g, Electrical field simulation for operation of the module (first stimulation bay) at a current of 130 μA, demonstrating the effectiveness of the separation. h, Demonstration of the time-sequenced series of iontophoresis (IP) events for 4 collection channels with 6-mm spacing (first iontophoresis event). i, Electric field simulation for the iontophoretic module (second iontophoresis event). The colorimetric bar has a logarithmic scale. j, Demonstration of the second iontophoresis event. k, Electric field simulation for the third iontophoresis event. l, Demonstration of the third iontophoresis event. The data reveal no evidence of cross-contamination at different time points. Figure 3c,d shows a schematic of the flexible printed circuit board with electronic components on the top and bottom of the iontophoretic stimulation module. The remaining components, including a switch, a battery and wireless charging circuits, are located on separate flexible printed circuit board islands, which fold onto the main timer modules during packaging. Experiments and finite element analysis (FEA) simulations that include the properties of the hydrogel (Extended Data Fig. 5a,b), the current densities (Extended Data Fig. 3e,f) serve as guides to maximize the sweat rate that follows from iontophoretic delivery. Carbagel produces about 20 μl sweat, compared with 5 μl for the alternative pilogel (pilocarpine in hydrogel), at 0.5% wt/vol, 4.3 μA mm−2 and a 1.5-mm distance between the stimulation site and collection channel (Extended Data Fig. Carbagel is better suited for the application presented here than pilogel, as the miniaturized LFA requires more than 10 μl sweat for operation. For values from 500–200 Ω, the current through the skin ranges from 100–500 μA (Extended Data Fig. 5c), with a corresponding increase in sweat volume (Extended Data Fig. 5i–k illustrates FEA results for the effect of current density on electric field distribution through the skin. The LFA can be filled within 30 min of stimulation for the case of a paper-based microfluidic collection channel with dimensions of 2 mm × 10 mm (Extended Data Fig. The distance between the stimulation site and collection channel is a critical parameter for maximizing the collected sweat volume and minimizing cross-contamination between each collection channel (Fig. At distances greater than 5.5 mm from the stimulation site, little sweat emerges from the collection channel, thereby defining a minimum distance to avoid cross-contamination. At a distance of 1.5 mm, sweat volumes of over 20 μl can be produced while forming a tight seal with the skin. A series of FEA simulations and on-body stimulation experiments confirm capabilities in time-sequenced iontophoretic stimulation with zero cross-contamination (Fig. Superhydrophilic surfaces prevent droplet condensation by causing water to spread evenly as a thin film21,22. However, the exclusive use of superhydrophilic materials leads to wicking of sweat away from the nitrocellulose membrane (Fig. The placement of superhydrophobic material on top of the sample and conjugation pad resolves this issue by blocking sweat flow through the top window (Fig. Figure 4d demonstrates the effectiveness of a superhydrophilic window (contact angle: 9.4°) in preventing droplet formation compared with superhydrophobic (contact angle: 151°) or untreated PET (contact angle: 82°) windows. The configuration with both superhydrophobic and superhydrophilic materials enables reliable operation and clear visualization after 15 min (Fig. 4f) of continuous heating on a hotplate at 30°. Characterization of the contact angle (Extended Data Fig. 7a,b), atomic force microscopy (Extended Data Fig. 4g–i) and reactive-ion etching process parameters (Extended Data Fig. 7e) of these materials provided additional insights, as discussed in the Supplementary Information section ‘Characterization of superhydrophilic and superhydrophobic surfaces'. a, Cross-sectional view of the LFA showing the mechanism of sweat vapour condensation on the surface of a superhydrophobic or hydrophobic viewing window. b, Cross-sectional view of the LFA showing a failure mechanism due to sweat flow through a viewing window entirely comprising a superhydrophilic window. c, Cross-sectional view of the LFA assembly with a combination of superhydrophobic and superhydrophilic materials comprising the viewing window. d, Optical image of a skin-interfaced device that uses iontophoretic stimulation of sweat to illustrate the effect of various material types on vapour condensation. e,f, Photograph of an LFA device that uses a superhydrophobic and superhydrophilic viewing window at 15 min (e) and 4 h (f). g, Transmittance change as a function of time for normal PET, superhydrophilic PET and a superhydrophobic viewing window (PET + PSA) at 30 °C in the presence of water. n = 1. i, Transmittance spectra of glass, normal PET, superhydrophilic PET and superhydrophobic PET + PSA after 6 h of heating at 30 °C in the presence of water. A simple, passive alternative to the electronic module described previously comprises a series of LFAs coupled with BSA-gated paper-based microfluidic delay valves and electrochromic timers. Narrow paper-based microfluidic cellulose channels with drop-cast layers of BSA (4%; 1.5 μl) serve as delay valves that direct sweat in a time-sequenced manner into separate LFAs. These valves temporarily restrict the flow until the BSA dissolves, allowing sequential delivery of sweat into each LFA and electrochromic timer pair. Electrochromic timers aligned with the valves and LFAs operate through a colour change associated with the reduction of a layer of polyaniline. Digital image capture and analysis quantifies the change in colour, thereby allowing determination—with an appropriate calibration factor—of the time for sweat collection into each LFA. a, Exploded view of a wearable LFA with BSA-gated delay valves, LFAs and an electrochromic timer. b, Exploded view of an electrochromic timer to define the sweat collection time for each channel. c, On-body demonstration of chronosampling functionality by delay valves for a series of LFAs. d, Delay times of the valves for experimental (squares) and simulation (circles and dashed line) data as a function of BSA concentration. e, Contact angles of the BSA-loaded paper-based microfluidic surface as a function of the BSA concentration. f, Delay time of the valves at a concentration of 3% for experimental (squares) and simulation (circles and dashed line) data for varying valve widths. g, Simulated progression of water saturation (blue) and BSA dissolution (red). h, Galvanostatic charge/discharge traces (at a constant current of 5 µA) for a polyaniline cathode (black trace, bottom plot) and a nickel anode (green trace, bottom plot) with representative images and b values (blue symbols, top plot) obtained from the polyaniline electrode at different charge states. i, Discharge traces obtained from electrochromic timers equipped with different load resistors bridging the anode and cathode. j, Electrochromic discharge traces obtained at different temperatures. m, Timer discharge traces obtained on the body both with (blue symbols) and without (red symbols) pre-treatment chemistry to remove reducing species from the sweat volume (benchtop data are provided for reference; black trace). n, Practical demonstration of the fully integrated device for two participants with iontophoretically stimulated sweat followed by exercise. Increasing valve widths lead to shorter delay times (Fig. The computational framework for simulating these valves integrates multiphase flow dynamics in porous media with a temporal evolution model of dissolution. As the BSA concentration asymptotically approaches 0, the contact angle θ approaches superhydrophilic conditions (θ → 0°)23,24, thereby enabling controlled fluid permeation through the valve architecture (Fig. Figure 5h displays information on the timer, specifically the galvanostatic charge/discharge traces obtained from both the polyaniline cathode (black trace) and nickel anode (purple trace) under an applied current of 5 µA. The data include colorimetric b values (blue diamonds) and representative digital photographs of the polyaniline electrode. During reduction, polyaniline displays a sloping voltage plateau at 0.3 V associated with its conversion from the fully oxidized pernigraniline phase to the partially oxidized emeraldine phase. During this transition, the colour of the electrode changes substantially, transitioning from dark blue to pale green (appearing red in photographs due to the roughened Au electrode). Further reduction results in complete conversion of the emeraldine phase to the leucoemeraldine phase, but this conversion does not provide a substantial colour change. Similarly, this colorimetric and redox transition is completely reversible upon application of an oxidizing current, as evidenced by the charging trace provided in the bottom half of Fig. An optical image of an optimized device on the wrist appears in Extended Data Fig. 9d, showing the paper-based microfluidic sweat collection inlet, BSA valves, electrochromic timers and LFA channels. In the assembled device, spontaneous reduction initiates with the introduction of sweat into the timing well, resulting in polyaniline reduction and a concomittant colour change analogous to that displayed in Fig. The total timing duration in this assembly depends on a load resistor placed between the anode and cathode that limits the rate of battery discharge. Appropriate selection of the resistance can address the requirements of a specific application (Fig. Timing durations from several minutes to over one hour are achievable in this way. As timing in this structure relies on the standard reduction potentials of the anode and cathode, it is comparatively less sensitive to temperature effects than analogous diffusion-based colorimetric timing systems8, as displayed in Fig. Reducing species, such as ascorbic acid, can act as interferents by reacting with the oxidized polyaniline membrane on the timer and discolouring the timer prematurely. A trace of polyaniline on the cellulose paper-based microfluidic channel before the timer reacts with any reducing species before sweat reaches the timer, thereby protecting the timer from discolouration by sweat. Subjecting polyaniline cathodes to progressively increasing concentrations of ascorbic acid reveals the effect of exogenous reducing species on electrochromic timing elements (Fig. Increasing concentrations of reductants induce spontaneous reduction of the electrode (and a concomitant lightening of the apparent coloration) with full reduction obtained at 100 mM. Following solution pre-reduction, the effect of reductive interferents can be effectively eliminated. Detailed schematics, characterizations and simulations of the timer appear in Extended Data Fig. The resulting structure exhibits excellent performance for real sweat in benchtop (Fig. Figure 5n shows a practical demonstration of the device with two participants. Iontophoresis initially induced sweat, which filled the first set of LFAs and electrochromic timers. After 15 min, participants began exercising while collecting sweat for the next sets of LFAs and timers. Both participants showed a slight increase in sweat cortisol concentration following exercise (Fig. 6c show variations in cortisol concentrations aligned with circadian rhythms for three participants over three days. Cortisol concentrations measured in saliva using an ELISA kit provide data for comparison. Collection of sweat and saliva samples occurred at 17:00 on the first day; 9:30, 12:00, 14:30 and 17:00 on the second day; and 9:30 on the third day. For both saliva and sweat, cortisol concentrations started low for all participants on the first day (17:00), peaked on the second day morning (09:30), gradually decreased throughout the second day (17:00) and then increased again on the third day morning (09:30). Data from circadian rhythm studies reveal a strong correlation between sweat cortisol and saliva cortisol (r = 0.73) for six participants (Fig. b, Photograph of a device on the skin. c, Time-dynamic monitoring of sweat (black) and saliva cortisol (red) over 3 days using the integrated LFA module for 3 participants: P1 (female; 37 years), P2 (male; 24 years) and P3 (male; 38 years). d, Correlation between sweat and saliva cortisol collected from the forearms of n = 6 participants (4 males and 2 females; 26 datapoints; r = 0.73). f–h, Time-dynamic monitoring of sweat and serum cortisol concentrations before, during and after a CPT for 3 participants: P1 (f; male; 38 years), P2 (g; male; 26 years) and P3 (h; male; 24 years). i,j, Serum (red) and sweat (grey) cortisol concentrations in the morning (09:30) and afternoon (16:00) before travel, immediately after travel and after recovery for a participant (male; 32 years) visiting Seoul from Chicago for 26 days (i) and a participant (female; 24 years) visiting Taipei from Chicago for 20 days (j). k, Sweat cortisol concentrations in the morning (09:30) and afternoon (16:00) before travel, immediately after travel and after recovery for a participant (male; 34 years) visiting Seoul from Chicago for 17 days. l, Correlation between sweat and serum cortisol collected from the forearms of n = 10 participants (8 males and 2 females; 33 datapoints). Investigations of the effect of CPTs on sweat and serum cortisol concentrations further highlight the versatility of the integrated system in studying psychoneurological responses to acute stress (Fig. Sweat iontophoretic stimulation and serum collection occurred on an interval set to 20 min. Both serum and sweat cortisol concentrations increase upon exposure to ice water, reaching peak values at 20 min after exposure, followed by recovery at 40 min after exposure (Fig. Monitoring disruptions to circadian rhythms is particularly relevant for night shift workers, intercontinental airline travellers and patients with insomnia or excessive sleepiness5. Jet lag, in particular, affects travellers crossing multiple time zomes, sometimes leading to daytime fatigue, insomnia, nausea and early waking25. Measurements of sweat and serum cortisol concentrations offer valuable insights into the impact of jet lag on circadian rhythms before travel, immediately after travel and during recovery (Fig. In one case, a participant travelled from Chicago to Seoul and returned 26 days later, with a nine-hour time difference (Fig. Cortisol concentrations in sweat and serum were measured 3 days before departure, 1 day after returning to Chicago and 12 days after recovery. In another case, a participant travelled from Chicago to Taiwan and returned 20 days later, with a 13-h time difference (Fig. Cortisol concentrations for sweat and serum were measured 1 day before departure, 1 day after returning to Chicago and 12 days after recovery. Similar to the first participant, this participant exhibited a typical circadian rhythm before travel, a reversed circadian rhythm after travel and a normalized rhythm after recovery. Additionally, we investigated sweat cortisol concentrations for a participant who travelled from Chicago to Seoul and returned after 17 days, with a 9-h time difference (Fig. Sweat cortisol concentrations were measured 5 days before departure, 1 day after returning to Chicago and 12 days after recovery. Similar circadian rhythm profiles to those of the other two participants were observed for this participant. A positive correlation between sweat and serum cortisol concentrations was observed, with a Pearson's correlation coefficient of r = 0.73 for 10 participants (Fig. 6l), demonstrating the relevance of sweat cortisol in monitoring the effects of acute stressors, as well as disruptions to circadian rhythms by jet lag, effectively. The wearable platform introduced here provides a rapid, convenient method for time-sequential sampling of eccrine sweat and measurements of cortisol concentrations, with accuracy and selectivity across a range of physiologically relevant values. The simplicity of the electronics design and the use of quantitative colorimetric readout schemes enabled by AuNF-based LFAs represent two essential features of the technology that are advantages relative to other approaches. The timing circuit and chronosampling valves both enable time-resolved sampling and quantification of sweat cortisol. The circuit autonomously controls sweat stimulation and sampling over extended periods, while the valves support short-term electronics-free operation and compatibility with natural modes of sweating. These two strategies provide complementary options for dynamic sweat monitoring across different use-case scenarios. Comprehensive benchtop studies with artificial sweat provide a basis for understanding all aspects of the operation and for optimizing essential design choices. Human participant studies, including quantitative correlations with concentrations of cortisol measured in saliva and serum using conventional techniques, confirm the practical value of the technology. Additional physiological studies demonstrate the ability to track circadian rhythms, monitor responses to acute stresses (such as those induced by CPTs) and determine the effects of jet lag on circadian misalignment. Potential applications span these scenarios and others, where elevated cortisol levels—particularly those that are persistent—can guide therapeutic intervention. Combination of this platform with biophysical monitoring systems26,27 will extend the capabilities to develop a more comprehensive understanding of stress. Human trials were approved by the Institutional Review Board (STU00220834) at Northwestern University and all participants gave complete, informed, signed consent before participating in the on-body experiments. All 13 human participants (nine males and four females) from the laboratory participated randomly in the experiments based on no specific selection criteria and no compensation. A priori power analysis for a two-tailed bivariate normal model correlation indicated that the minimum sample size to yield a statistical power of at least 0.8 with an alpha of 0.05 and a Pearson's correlation coefficient of r = 0.7 was 13 for Fig. Unless otherwise noted, all materials were used without further purification. The materials for the absorbent pad (Advanced Microdevices Pvt Type GFB-R4), sample pad (Whatman GF/F glass microfibre filters) and conjugation pad (ClaremontBio glass fibre) were purchased from Advanced Microdevices. The immunopore RP nitrocellulose membrane was purchased from Cytiva. Anti-cortisol antibody (10R-145a) and BSA–CTS (80-IC10) were purchased from Fitzgerald. Gold(III) chloride trihydrate (≥99.9%), gold wire (diameter: 0.5 mm; 99.99%), ammonium chloride (EMSURE), sulfuric acid (95.0–98.0%), aniline (≥99.5%), nickel wire (diameter: 0.5 mm; 99.99%), pilocarpine nitrate, BSA, Agarose Low EEO, sodium phosphate monobasic, sodium phosphate dibasic, aminobenzenethiol, gold chloride hydrate, polyvinylpyrrolidone, cortisol standard (C-106), Tween 20 and mouse anti-IgG antibody (M8642-1MG) were purchased from Sigma–Aldrich. Polyimide films (75 µm thickness; 15.24 cm width) were purchased from Argon. Gold (99.999%) and chromium (99.95%) pellets for electron beam evaporation were obtained from Kurt J. Lesker Company. Double-sided copper-clad laminate (M916137; Pyralux) was purchased from DuPont. Capillary blood collection tubes (15 μl), Protein LoBind Eppendorf tubes (1.5 and 2.0 ml), Whatman cellulose (602H; 1575), alcohol prep pads and Pierce 20× borate buffer were purchased from Thermo Fisher Scientific. Carbachol was purchased from AA Blocks. Artificial sweat (1700-0020) was purchased from Pickering Laboratories. The battery holders, switches, low-dropout regulators (1.8 and 3.3 V), rectifiers, voltage regulators, 10 nF capacitors (0201), 1 μF capacitors (0201 and 0402), DC/DC converters, 0.22 μF capacitors (0402), 10 μH inductors (50 mA; 0603), red LEDs (0603), resistors (0201), programmable IC SOT-23-THIN timers, 551-type timer/oscillators (single), hex inverters, JK flip-flop 2 elements and 2-to-4 decoder/demultiplexer were purchased from DigiKey. LiPO batteries were purchased from LiPol Battery. 23 G safety lancets and superhydrophilic PET sheets (B0827YLFV9; Frienda) were purchased from Amazon. Sheets of PET (LX000464) were purchased from Flexconn. Skin adhesives (PC2723U) were purchased from Scapa Healthcare. Human Cortisol ELISA kits (LS-F10024) for sweat and serum were purchased from LifeSpan Biosciences. Cortisol ELISA kits (ab154996; saliva samples) were purchased from Abcam. A 25-mM solution of 4-aminobenzene-1-thiol (ABT) in 95% ethanol was added to 10 mM HAuCl4 solution containing 0.001% polyvinylpyrrolidone (PVP) in ultrapure water with a reaction ratio of 1 to 9 (vol/vol). for 30 min to remove excess reagents. Conjugation of antibody to the AuNFs involved mixing 200 μl AuNF stock solution with 800 μl borate buffer (5 mM) and adding 0.5 μl anti-cortisol antibody (1.16 mg ml−1). The borate buffer (5 mM) was obtained by diluting 20 mM borate buffer in deionized water (2% vol borate buffer in deionized water). After ensuring complete mixing, the mixture was incubated for 2 h in a rotating mixer at 10 r.p.m. and then left undisturbed for an additional 5 min at room temperature. At this stage, AuNFs were bound with antibodies. Adding 100 μl 1% BSA solution to the mixture, followed by subsequent incubation for 1 h in a rotating mixer at 10 r.p.m., blocked the unbound surfaces of the AuNFs. After 5 min, the mixture was centrifuged at 2,500g for 35 min at 4 °C and washed with 1 ml borate buffer (5 mM) three times for complete removal of the supernatant. After final centrifugation, the precipitate was reconstituted in 100 μl 0.5% BSA in phosphate-buffered saline and stored at 4 °C until later use. The sample pad (2.0 mm × 1.5 mm) was treated with 2 μl buffer (pH 6.5) and dried under vacuum at room temperature for 10 min. The sample pad was designed with a staircase structure to facilitate efficient sweat absorption and transport towards the assay components. To reduce non-specific biofouling of AuNF–Ab conjugates, the conjugate pad was initially impregnated with 5 μl of a working buffer containing 30% (wt/vol) BSA and Tween 20 and dried under vacuum at room temperature for 20 min. Subsequently, AuNF–Ab conjugates were deposited onto the conjugate pad at a volume of 2 μl and dried under the same conditions. Nitrocellulose membranes were miniaturized for wearable implementation and cut into 2-mm-wide pieces using an infrared laser (LPKF) before assembly into standard LFA cassettes. The test line reagent comprised BSA–CTS at a 1/8 dilution factor with a final concentration of 0.125 mg ml−1, whereas the control line contained an anti-mouse IgG antibody at 1 mg ml−1. Both reagents were dispensed onto the nitrocellulose membrane using predefined parameters, including a dispensing volume of 0.325 μl mm−1 and a speed of 100 mm s−1, followed by drying in a vacuum chamber at room temperature for 20 min. The test and control lines were spaced 5 mm apart for optimal visualization and signal differentiation. The preparation of superhydrophobic surfaces involved fluorine plasma treatment using a deep reactive-ion etching system (SPTS). The PET substrate with a thin layer of adhesive (FLX000464; Flexcon) was placed in the chamber with the adhesive side facing up, followed by evacuation to a pressure of 1.6 Pa. CF4 gas was then introduced at a flow rate of 50 sccm, with a glow discharge at 200 W for 2 min. Dropcasting 1.5 μl BSA (4% wt/vol; Sigma–Aldrich) solution onto the designated valve location in the cellulose paper completed the preparation of BSA-gated paper-based microfluidic delay valves, followed by drying under vacuum for 20 min. A 3D printer (Formlabs) created the guide jig for the alignment and assembly of different layers. LFA components and covers were assembled by flipping the device and assembling from the opposite side. Assembly of system was complete after all four LFA modules were placed in the chamber and covered by the superhydrophilic viewing window. Timing elements were fabricated from thin-film Au electrodes supported on flexible polyimide, as follows. Substrate materials were initially prepared by electron beam deposition of metal films (100 nm Au, with a 20-nm Cr adhesion layer; AJA International) onto a large polyimide substrate (75 µm thickness; surface area: ~700 cm2) whose surface was cleaned (sequential rinsing with water and isopropanol) and treated with ultraviolet ozone (UVOCS; 5 min). Deposition was performed at a constant rate of 1 Å s−1. Following deposition, electrode traces were patterned using an ultraviolet laser micromachining system (ProtoLaser U4; LPKF Laser and Electronics SE) at an output power of 0.5 W in a single pass. Arrays of eight timers were patterned in this fashion in preparation for subsequent chemical treatment by electrodeposition. Rough gold was deposited by cyclic voltammetry between −1.8 and 0.2 V (scan rate: 0.5 V s−1) from a 10-mM solution of HAuCl4 in 2.5 M NH4Cl supporting electrolyte using an Au slug as the counter and reference electrode28. A total of 25 cycles were applied, pausing every five cycles to mix the electrolyte solution with a pipette to reduce the accumulation of bubbles on the electrode surface. Following roughening, polyaniline was deposited from a monomer solution of 0.5 M aniline in 1 M H2SO4 by the application of a constant reducing current of 50 µA for 400 s using an Au slug as the counter electrode and a double-junction Ag/AgCl electrode (BASI) as the reference. To ensure even and dark coloration of polyaniline cathodes, electrodes were subsequently oxidized by polarization to 0.4 V in pH 4.5 solution before cutout and assembly (again using Ag/AgCl as the reference and Au as the counter electrode). Nickel anodes were applied by electrodeposition from a commercially available bright nickel plating bath (Gold Plating Services) by the application of a reducing potential of −1.5 V versus Ni metal for 120 s using an Ni wire as the counter and reference electrode. Following electrodeposition, individual timers were cut from the substrate (ProtoLaser U4; 0.5 W; 30 repetitions) for integration with the paper-based microfluidic assembly. Numerical simulations were conducted using COMSOL Multiphysics, employing tetrahedral mesh elements for all of the computational analyses. A refined mesh with feature sizes smaller than one-fifth of the electrode width was adopted to ensure accuracy. The numerical solver dynamically controlled time step sizes through backward differentiation formulas, with the initial step sizes kept small to prevent singularity. The material properties and key parameters used in the simulation included: a skin conductivity σ of 0.026 S m−1 (ref. 29); a diffusion coefficient D of 14.8 × 10−6 cm2 s−1 (ref. Numerical simulations were conducted using COMSOL Multiphysics31,32. Darcy's law interface23 was utilized to describe the fluid flow through the filter paper (porous medium). A no-flow condition was applied across the vertical boundaries, whereas the upper and lower boundaries were set to a zero-pressure outlet condition. The domain ordinary differential equation and differential algebraic equation interface was employed to describe the BSA dissolution. Smooth-On Dragon Skin 10 (with a mixing ratio of 1:1) was poured into the mould, degassed in a vacuum chamber for 30 min and baked in a 75 °C oven for 3 h. Five healthy volunteers (four male and one female) with regular sleep–wake rhythms participated in a diurnal cortisol cycle test. Participants did not have any food intake for at least 60 min before each testing. The on-body testing comprised sweat stimulation sessions throughout three consecutive days, including: (1) day one at 17:00; (2) day two at 09:30, 12:00, 14:30 and 17:00; and (3) day three at 09:00. After sanitizing the forearm, sweat was stimulated and collected simultaneously by the iontophoresis-integrated LFA device, where 130-μA current was applied for 5 min. Saliva samples were collected immediately after the sweat stimulation and stored under −18 °C until further use. A saliva salivate (Sarstedt) was used to collect saliva samples by keeping a cotton swab inside the mouth for 5 min. After 5 min, the cotton swab was returned to the salivate and centrifuged for 2 min at 2,000g. The supernatant of saliva sample was then collected using a pipette and stored in a −18 °C freezer until use. Five healthy volunteers (four males and one female) participated in the standard CPT in the afternoon (between 15:00 and 17:00) to avoid the effect of diurnal cortisol cycle variation. After baseline sample collection, the iontophoresis device with a timer was turned on for sequential sweat stimulation. Fresh capillary blood was collected from the non-dominant hand at different stages synchronized with iontophoresis time (immediately after the CPT, at 20 min and at 40 min). For capillary blood collection, an alcohol pad was used to sanitize the volunteer's fingers before collection. A 21-G lancet punctured the sanitized finger to draw blood droplets. A 15-μl capillary blood collection tube (PTS Diagnostics) was used to collect blood droplets of over 50 μl into a 1.5-ml Eppendorf microtube. The collected blood sample was set aside for 40 min at room temperature for blood clotting. After 40 min resting, the blood sample was centrifuged at 2,000g for 15 min to separate the serum from the clotted white and red blood cells. The supernatant serum was collected in a 1.5-ml Eppendorf microtube and stored in a −18 °C freezer until use. Four healthy volunteers (three males and one female) participated in the jet lag study. All volunteers experienced a time difference of more than eight hours, as well as varying degrees of jet lag symptoms. All volunteers took flights from the USA to East Asia and stayed there for more than two weeks. Before travel, baseline sweat cortisol concentrations were quantified using the LFA device in the morning (09:30) and afternoon (16:00). For two participants, capillary blood samples were collected at the same time, with sweat cortisol measurement. All statistical analyses were performed using OriginPro (version 2022; OriginLab) and G*Power 3.1.9. The selected image is representative of approximately 85 nanoparticles. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. All code related to the LFA data presented in this paper is available at https://github.com/cho8690/LFA.git. Weber, C. J., Clay, O. M., Lycan, R. E., Anderson, G. K. & Simoska, O. Ok, J., Park, S., Jung, Y. H. & Kim, T. I. Wearable and implantable cortisol-sensing electronics for stress monitoring. Kim, S. et al. Soft, skin-interfaced microfluidic systems with integrated immunoassays, fluorometric sensors, and impedance measurement capabilities. Sleep and circadian regulation of cortisol: a short review. Treatment of shift work disorder and jet lag. Upasham, S., Churcher, N. K. M., Rice, P. & Prasad, S. Sweating out the circadian rhythm: a technical review. Mohammadi, M. H. et al. Saliva lab-on-a-chip biosensors: recent novel ideas and applications in disease detection. Cho, S. et al. A skin-interfaced microfluidic platform supports dynamic sweat biochemical analysis during human exercise. A. Skin-interfaced systems for sweat collection and analytics. Kim, J., Campbell, A. S., de Avila, B. E. & Wang, J. Wearable biosensors for healthcare monitoring. Shajari, S. et al. MicroSweat: a wearable microfluidic patch for noninvasive and reliable sweat collection enables human stress monitoring. Parlak, O., Keene, S. T., Marais, A., Curto, V. F. & Salleo, A. Molecularly selective nanoporous membrane-based wearable organic electrochemical device for noninvasive cortisol sensing. Wang, B. et al. Wearable aptamer-field-effect transistor sensing system for noninvasive cortisol monitoring. Torrente-Rodriguez, R. M. et al. Investigation of cortisol dynamics in human sweat using a graphene-based wireless mHealth system. Rivas, L., Merkoci, A., Hu, L. M., Parolo, C. & Idili, A. Rational approach to tailor Au–IrO2 nanoflowers as colorimetric labels for lateral flow assays. Liang, R. S. et al. A sensitive gold nanoflower-based lateral flow assay coupled with gold staining technique for the detection of SARS-CoV-2 antigen. Lai, W. H. et al. Gold nanoflowers labelled lateral flow assay integrated with smartphone for highly sensitive detection of clenbuterol in swine urine. Cai, P., Wang, R., Ling, S. & Wang, S. Rapid and sensitive detection of tenuazonic acid in agricultural by-products based on gold nano-flower lateral flow. Gupta, R. et al. Ultrasensitive lateral-flow assays via plasmonically active antibody-conjugated fluorescent nanoparticles. & Chen, H. Transparent and superhydrophilic antifogging coatings constructed by poly (N-hydroxyethyl acrylamide) composites. Highly transparent superhydrophilic graphene oxide coating for antifogging. Patari, S. & Mahapatra, P. S. Liquid wicking in a paper strip: an experimental and numerical study. & Mahapatra, P. S. Dynamics of liquid flow through fabric porous media: experimental, analytical, and numerical investigation. Cingi, C., Emre, I. E. & Muluk, N. B. Jetlag related sleep problems and their management: a review. Xu, C. et al. A physicochemical-sensing electronic skin for stress response monitoring. Yoo, J. Y. et al. Wireless broadband acousto-mechanical sensing system for continuous physiological monitoring. & Heinemann, U. Carbachol effects on hippocampal neurons in vitro: dependence on the rate of rise of carbachol tissue concentration. & Tiwari, M. K. Autonomous transport and splitting of a droplet on an open surface. Brooks, R. H. & Corey, A. T. Properties of porous media affecting fluid flow. Masoodi, R. & Pillai, K. A general formula for capillary suction-pressure in porous media. & Siepmann, F. Mathematical modeling of drug dissolution. This work was supported by the Querrey Simpson Institute for Bioelectronics at Northwestern University. This work made use of the NUFAB facility of Northwestern University's NUANCE Center, which has received support from the SHyNE Resource (NSF ECCS-2025633), the International Institute for Nanotechnology and Northwestern University's Materials Research Science and Engineering Center programme (NSF DMR-2308691). This work made use of the EPIC facility of Northwestern University's NUANCE Center, which has received support from the SHyNE Resource (NSF ECCS-2025633), the International Institute for Nanotechnology and Northwestern University's Materials Research Science and Engineering Center programme (NSF DMR-2308691). This work was supported by the Northwestern University High Throughput Analysis Laboratory and Keck Biophysics Facility, the shared resource of the Robert H. Lurie Comprehensive Cancer Center of Northwestern University, supported partly by the National Cancer Institute Cancer Center (Support Grant P30 CA060553). We acknowledge use of the facility at IMSERC at Northwestern University, which has received support from SHyNE Resource (NSF ECCS-2025633) and the National Institutes of Health (1S10OD012016-01 and 1S10RR019071-01A1). The study was supported by a National Institutes of Health sleep and circadian training grant (T32HL007909 to K.M. ), a postdoctoral fellowship provided by the Natural Sciences and Engineering Research Council of Canada (to S.S.) and the National Natural Science Foundation of China (12202241 to Z.L. Present address: Department of Mechanical, Industrial and Mechatronics Engineering, Toronto Metropolitan University, Toronto, Ontario, Canada Present address: Institute for Biomedical Engineering, Science and Technology (iBEST), St. Michael's Hospital–Unity Health Toronto, Toronto, Ontario, Canada These authors contributed equally: Soongwon Cho, Shaghayegh Shajari, Yirui Xiong, Kenneth Madsen, Zengyao Lv, Seunghee Cho. Querrey Simpson Institute for Bioelectronics, Northwestern University, Evanston, IL, USA Soongwon Cho, Shaghayegh Shajari, Yirui Xiong, Kenneth Madsen, Seunghee Cho, Ruihao Song, Ivy Huang, Ravi F. Nuxoll, Yibo Zhou, Yong-woo Kang, Chanho Park, Jeonghwan Park, Yu-Ting Huang, Haris Bukaric, Xander Mueller, Alexander J. Aranyosi, Roozbeh Ghaffari, Yonggang Huang & John A. Rogers Center for Bio-Integrated Electronics, Northwestern University, Evanston, IL, USA Soongwon Cho, Shaghayegh Shajari, Seunghee Cho, Yong-woo Kang, Chanho Park, Jeonghwan Park, Alexander J. Aranyosi, Roozbeh Ghaffari & John A. Rogers Department of Material Science and Engineering, Northwestern University, Evanston, IL, USA Yirui Xiong, Ravi F. Nuxoll, Ziyu Chen, Anubhap Taechamahaphan, Yonggang Huang & John A. Rogers Department of Civil and Environmental Engineering, Northwestern University, Evanston, IL, USA Zengyao Lv, Shupeng Li & Yonggang Huang Center for Systems Biology, Massachusetts General Hospital, Boston, MA, USA Department of Radiology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Department of Biomedical Engineering, Northwestern University, Evanston, IL, USA Michelle Li, Sarena Wapnick, Haris Bukaric, Roozbeh Ghaffari & John A. Rogers Department of Mechanical Engineering, Northwestern University, Evanston, IL, USA Department of Physics and Astronomy, Northwestern University, Evanston, IL, USA Department of Electrical and Computer Engineering, Northwestern University, Evanston, IL, USA Segal Design Institute, Northwestern University, Evanston, IL, USA College of Engineering and Computer Science, Australian National University, Canberra, Australian Capital Territory, Australia Department of Neurological Surgery, Northwestern University, Evanston, IL, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and developed the AuNF-based LFA. S.S., Soongwon Cho and Jinho Park developed the method and codes for the LFA analysis. developed the integrated iontophoresis with a smart timer. developed the BSA-gated delay valves for chronosampling. assisted with and participated in the human participant trials. provided the sweat collection equipment and devices. J.A.R., Y.X., S.S., K.M., Z.L., M.L., S. Oh and Soongwon Cho wrote the paper. All authors read and approved the paper. Correspondence to Seyong Oh or John A. Rogers. are co-founders of a company, Epicore Biosystems, that develops and commercializes microfluidic devices for sweat analysis, but using other sensing techniques. The other authors declare no competing interests. Nature Sensors thanks Yei Hwan Jung, Hui Kong and Sihong Wang for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic illustration of AuNF synthesis chemistry. b, Measurement of zeta potential for AuNF incubated for 1 h with varying concentrations of anti-cortisol antibody. n = 3. c-f, Demonstration of chronosampled LFA with increasing cortisol concentrations: 0 (c), 1 (d), 10 (d) and 100 ng/mL (f). h, Adjustment of artificial sweat and real sweat to pH 6.5 using phosphate buffer within the sample pads. Measured color (a*, black) and pH values (red) are shown. n = 1. j, Evaluation of different working buffer concentrations (1:1 ratio of BSA to Tween 20) for LFA sensitivity. n = 3. k, Performance comparison of different working buffer types (Trixton, Tw20, 10G and PVP) for AuNF transport through the NC membrane. l, Effect of varying BSA-CTS concentrations (from 1 mg/mL to 0.125 mg/mL) on the visual appearances of the test lines. Data are presented as mean values ± SD (n = 3 technical replicates). a, Effect of cortisol ab dilutions (1X, 2X, 4X, 6X and 8X) on the control/test line ratios at cortisol concentrations of 0, and 100 ng/mL. n = 3. b, Effect of flow rate on the control/test line ratios for cortisol concentrations of 0, and 100 ng/mL prior to working buffer optimization. n = 3. c, Effect of injection volumes (10, 15, and 20 μL) on the control/test line ratios at cortisol concentrations of 1 and 100 ng/mL. n = 3 prior to optimization of assay components. d, Effect of lighting color temperature (4500, 5000, 5500 K) on the control/test line ratios at cortisol concentrations of 0, 0.1, 1, 10, and 100 ng/mL. n = 3. e,f, Comparison of control/test line ratios of LFA images, at cortisol concentrations of 0, 0.1, 1, 10, and 100 ng/mL, taken with a smartphone camera (Iphone 16 pro) and a digital camera (Canon) at lighting color temperatures of e, 5500 K, and f, 4500 K. n = 3. Data are presented as mean values ± SD (n = 3 technical replicates). a, Original image of an assay with color checker for calibration. b, Image of the assay after applying color calibration. d, Green channel image with a yellow line for LFA profile analysis. g, Flow chart for LFA analysis. a, An optical photograph of the implemented electrical circuit including electrical timer, iontophoresis stimulation and power management integrated circuit (PMIC). b, A close-up photograph of the implemented electrical timer circuit for time-sequenced iontophoretic stimulation, fabricated on a flexible PCB. c, An optical photograph of the wireless charging module. d, Schematic of the electrical timer circuit. e, Fully implemented circuit for time-sequenced iontophoretic stimulation. f, Power management integrated circuit and operating principle for wireless charging. g, Experiental results showing interval time as a function of long timer resistance. n = 3. h, Iontophoresis stimulation time as a function of short timer resistance. n = 3. i, Experimental results showing the cycle variation in sweat stimulation time and time interval between stimulation. Data are presented as mean values ± SD (n = 3 technical replicates). b, Total sweat volume collected by the paper-based microfluidic channel following iontophoretic stimulation using carbagel (black) and pilogel (red). n = 3. c, Current stability of the iontophoretic modules with different resistances. d, Simulation setup for estimating sweat volume, including the anode, cathode, and extended region for modeling carbachol diffusion within the skin (total current: 130 μA, duration: 10 min). f, Experimental (red, black) and simulated data (blue) for total sweat volume collected at varying current densities (1% carbagel, 1.5 mm spacing). n = 3. g, Experimental sweat rate in two subjects over 40 min under carbagel stimulation at varying current densities. n = 3. h, Simulated sweat rate over 40 min at varying current densities. i-k, Top and cross-sectional views of the electrical field distribution in the iontophoretic module at current densities of 130 μA (i), 41 μA (j) and 19 μA (k). l,m, Successful generation of sufficient sweat volume using the integrated iontophoresis system, paper-based microfluidic collection channel, and LFA. Data are presented as mean values ± SD (n = 3 technical replicates). b, Photograph of a wet LFA strip with dyed artificial sweat, showing water droplet condensation on the viewing window. c,d, Schematic illustrations of LFA device with a test paper for visualizing unwanted hydrophilic wicking through a superhydrophilic cover (c) and a combination of superhydrophilic and superhydrophobic covers (d). e,f, Schematic illustrations of LFA devices after pumping artificial sweat with a superhydrophilic cover with a wet test paper (e) and with a combination of superhydrophilic and superhydrophobic cover with a dry test paper (f). g,h, Optical images of LFA devices with a test paper before sweat wicking with a superhydrophilic cover (g) and a combination of superhydrophilic and superhydrophobic cover (h). i,j, Optical images of LFA devices after pumping artificial sweat with a superhydrophilic cover with a wet test paper (i) and a combination of superhydrophilic and superhydrophobic cover with a dry test paper (j). b, Contact angle measurements for a superhydrophilic PET film. c, Atomic force microscopic (AFM) image of a PET film. d, AFM image of a superhydrophilic PET surface formed by a coating of silica nanoparticles. e, The effect of reactive ion etching power on the contact angle of a superhydrophobic PET + PSA substrate. a, Boundary conditions for the simulation. b, Water saturation at the outlet (right) for BSA loading concentrations (w/v, 1 μL) of 0%, 1%, 2%, and 3%. d, Fully integrated system incorporating paper-based microfluidics, electrochromic timing elements, and LFAs to facilitate and characterize the time-dependent collection and chemical analysis of sweat. a, Schematic illustration of an electrochromic timer. b, Optical and scanning electron micrographs of timer electrodes at different points during assembly. c, Cyclic voltammetry of smooth and electrochemically roughened Au electrodes, illustrating an increase in the electroactive surface area following roughening. d, Electroactive surface areas computed from the results in c. e, Electrochemical impedance spectroscopy of bare and roughened Au electrodes. f, Electrochemical impedance spectroscopy of an electrochromic timer using artificial sweat (pH = 4.5) as the electrolyte. i, Correlation between discharge current and cell capacity. j, Colorimetric response obtained from timers both in the presence and absence of chemical pretreatments to remove reductants from incoming sweat. Data are presented as mean values ± SD. (n = 3) k, Simulated pH gradients within electrochromic timing elements with different load resistors computed at the end of cell discharge. Irrespective of load resistor, only modest changes to solution pH are obtained, although discharging at elevated current densities (low resistance) results in exaggerated concentration gradients. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Cho, S., Shajari, S., Xiong, Y. et al. Wearable lateral flow assays for cortisol monitoring with time-dynamic sweat sampling and sensing by electrochromic timers. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative You have full access to this article via your institution. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69999529/consciousness-without-cortex-theory/'>Consciousness May Not Be Uniquely Human, New Theories Suggest—Rewriting What We Know About the Mind</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 16:48:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Now, scientists think consciousness might be older and more widespread than previously believed. Not only are researchers unable to define what consciousness is—whether it be basic self-awareness or some kind of higher connection with the universe—but they also aren't exactly sure where it comes from. In their respective November 2025 papers, two research groups at the Ruhr University Bochum (RUB) in Germany suggest that looking at other species, particularly birds, could redefine the illusive concept and explain where our own consciousness evolved from. In the first paper, researchers Albert Newen, PhD, and Carlos Montemayor, PhD, argue that across species, consciousness evolved in distinct waves, each serving a particular function. These stages challenge previous theories of consciousness, as they provide a new framework that relies less on the cortex, the outermost layer of your brain, which is commonly associated with consciousness theories. Likewise in the second paper, authors Gianmarco Maldarelli, PhD, and Onur Güntürkün, PhD, present evidence that there may be more structures capable of generating consciousness than previously thought. 🔮 Check Out Our New Video Series, Pop Mech Explains: PrecognitionYou've probably had that eerie feeling before—knowing what's about to happen before it actually does. In Newen and Montemayor's evolutionary framework, basic arousal developed first. It's like nature's security alarm, warning the body of any dangers that may arise. This is “what we humans experience as our everyday consciousness, but it is also clearly proven to be realized in many mammals,” including mice, the researchers say in the paper. General alertness allows you to focus on one detail amid a slew of otherwise overwhelming information, giving your brain the tools to judge the outcome of decisions. For instance, if you're holding a hot plate, you're focused on a piece of information—pain in your hand—but have the capacity to weigh two choices: drop it or burn yourself. For a wild animal, this may look like weighing the risk of running into a predator while collecting food in an open area or going hungry. More simply put, general alertness enables organisms to develop complex associations, helping them better navigate their environment. The researchers explain that, at its core, this level of self awareness “makes it possible for us to better integrate into society and coordinate with others.” Passing the mirror self-recognition test—a behavioral test where animals are prompted to examine themselves in a mirror—is commonly associated with reflexive self-consciousness. During the test, the subject is marked with paint or a sticker on an area of their body that they normally wouldn't be able to see. Human children are able to pass the test at around 18 months old, and animals including chimpanzees, dolphins, and magpies also perform well, meaning linguistic abilities aren't necessary in proving reflexive self-consciousness. In Maldarelli and Güntürkün's paper, the two researchers argue “that consciousness is not the ultimate triumph of human evolution but rather represents a more basic cognitive process, possibly shared with other animal phyla,” meaning consciousness might be much older and more widespread than researchers previously believed. The researchers present growing evidence that various birds display self awareness—or what Newen and Montemayor would likely consider reflexive self-consciousness. The authors list several species who performed well on “ecologically significant” mirror tests unique to their family. Maldarelli and Güntürkün argue that the nidopallium caudolaterale—the bird version of a prefrontal cortex—is still “immensely connected,” meaning it's capable of processing information in complex ways. In other words, “bird brain” isn't quite the insult you intend it to be. The paper suggests that consciousness is possible without a mammalian prefrontal cortex—a well-organized brain structure commonly associated with the profound consciousness we humans are capable of. However, the authors note that no notable conclusions can be drawn yet. “When we focus only on human minds, it can be difficult to tell which of our features are essential to consciousness and which are merely part of our particular form,” Sebo writes in an email. “By studying consciousness across species, we can better understand our own feelings and emotions, as well as those of other humans who may differ from us.” Aside from what we might learn about human consciousness, studying animal awareness is also valuable because of the ethical implications it poses, according to Sebo. For now, the true root of consciousness remains just outside of our understanding—but know that next time a bird perches on your windowsill, it might be thinking the same thing you are. Emma Frederickson graduated from Pace University where she studied communication and media. Prior to her time as an editor, she was a freelance science reporter. She enjoys covering everything from shipwrecks to pimple popping, but her favorite topics include climate change, conspiracy theories, and weird biology. This Method Could Upload Your Brain—And Bring It Back to Life Later They Thought It Was a Home Reno, Not an Exorcism</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/americans-overwhelmingly-support-science-but-some-think-the-u-s-is-lagging/'>Americans Overwhelmingly Support Science, but Some Think the U.S. Is Lagging Behind: Pew</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Americans Overwhelmingly Support Science, but Some Think the U.S. Is Lagging Behind Americans are proud of their country's science prowess: a majority believe it is important for the U.S. to be a world leader in science, according to the Pew Research Center's latest report on trust in science. But people who voted Democratic in the 2024 presidential election tended to hold a very different view than Republican voters on whether the country is living up to its promise. Between 2023 and 2025, the proportion of Democrats who believe that the U.S. is losing ground in science compared with other countries jumped by 28 percentage points. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In 2022 and 2023 the difference between Democratic and Republican opinion was “far more modest,” Kennedy says, with both groups responses' within 7 percentage points of each other. Amanda Montañez; Source: Do Americans Think the Country Is Losing or Gaining Ground in Science? Last year the Trump administration cut federal funding for science. Meanwhile experts have warned of a “brain drain,” partly motivated by the administration's strict immigration policies, with researchers choosing to study or live overseas instead of in the U.S. Despite these cuts, the majority of Americans—84 percent—thought federal investments in science aimed at advancing knowledge were worthwhile. Republican voters, however, were more likely than their left-leaning peers to be open to private companies playing a key role in science, Kennedy says. “One thing we've seen in our surveys over a number of years is that support for science funding is pretty widespread among both Republicans to Democrats,” he says. Indeed, lawmakers on both sides of the aisle have advanced several legislative efforts to claw back some of the targeted federal funding for science agencies. Ultimately, the report shows that Americans' trust in science and scientists remains broadly strong—but not as strong as it was before the COVID pandemic. “There's a broader context of trust and confidence going on in society,” Kennedy says. Still, he points out that Pew survey participants have consistently ranked scientists among the most trustworthy groups in society for the past 10 years. Claire Cameron is breaking news chief at Scientific American. Originally from Scotland, she moved to New York City in 2012. Her work has appeared in National Geographic, Slate, Inc. Magazine, Nautilus, Semafor, and elsewhere. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260115022812.htm'>Statins may help almost everyone with type 2 diabetes live longer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 14:18:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A large long-term study has found that statins, a widely used class of cholesterol-lowering medications, significantly reduce the risk of death and serious heart-related problems in adults with type 2 diabetes. This challenges a long-standing debate over whether preventive statin treatment is worthwhile for patients who appear to be at lower cardiovascular risk. High LDL levels are linked to clogged arteries, heart attacks, and strokes. People with type 2 diabetes already face a higher risk of cardiovascular disease, but doctors have not always agreed on whether statins are necessary for those whose short-term heart risk appears minimal. The new findings suggest that statins may offer protective effects for a much wider group of diabetes patients than previously believed. Their goal was to assess both the effectiveness and safety of starting statin therapy for primary prevention. Primary prevention refers to preventing a first heart attack or stroke before any such event has occurred. This risk estimate is commonly used in clinical practice to guide treatment decisions. Across all risk categories, statin use was linked to lower rates of death from any cause and fewer major cardiovascular events such as heart attacks and strokes. Even participants classified as low risk experienced measurable benefits, which directly challenges the assumption that statins only help people already at high risk of heart disease. Myopathy refers to muscle-related side effects, which can include weakness or soreness and are a known but uncommon concern with statin use. No increase in liver-related problems was found, addressing another common worry among patients and clinicians. Based on these results, the authors concluded that doctors should carefully consider the advantages of statin therapy for all adults with type 2 diabetes, even when a person's short-term predicted risk of cardiovascular disease is low. The findings suggest that relying solely on short-term risk estimates may cause some patients to miss out on treatments that could help them live longer and avoid serious heart complications. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/evacuated-iss-astronauts-return-to-earth-as-nasa-prepares-for-artemis-moon/'>Evacuated ISS Astronauts Return to Earth as NASA Looks Ahead to Artemis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Astronauts Return to Earth in First ISS Medical Evacuation One of the most notable chapters in the history of NASA is coming to something like a close: after calling for an unprecedented medical evacuation of four astronauts onboard the International Space Station (ISS), these space farers are safely home. When asked at a Thursday press conference if NASA planned to release further information about the medical situation that prompted the evacuation, agency chief Jared Isaacman said it is “very committed to being transparent.” That said, to the extent that we are in a position to share more information publicly and have the necessary consent, we would do so,” he said. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “Obviously, we took this action because it was a serious medical condition,” Isaacman said. Crew-11 splashed down in a SpaceX Crew Dragon capsule off the coast of California at approximately 3:41 A.M. EST. The evacuating Crew-11 includes NASA astronauts Mike Fincke and Zena Cardman, Japanese Aerospace Exploration Agency astronaut Kimiya Yui and Russian cosmonaut Oleg Platonov. NASA has not identified which of these astronauts experienced the medical issue. The crew is undergoing medical evaluation on a receiver ship and is headed to a hospital in San Diego, Calif., for further investigation and care, said Joel Montalbano, deputy associate administrator of NASA's Space Operations Missions Directorate, at the same conference. Whatever happened to Crew-11 could influence how the agency prepares for future human spaceflight missions, including the upcoming Artemis II moon flyby. NASA will conduct a full debrief and review of the Crew-11 mission, Isaacman said. The ISS is equipped with an array of medical equipment, drugs and diagnostic tools—all of which the station's crew know how to use. The agency plans for these contingencies on every mission, Isaacman said. There are options to bring astronauts back from the space station in hours, not days. Claire Cameron is breaking news chief at Scientific American. Originally from Scotland, she moved to New York City in 2012. Her work has appeared in National Geographic, Slate, Inc. Magazine, Nautilus, Semafor, and elsewhere. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02948-0'>Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 11:05:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Spatially mapping the transcriptome and proteome in the same tissue section can profoundly advance our understanding of cellular heterogeneity and function. Here we present Deterministic Barcoding in Tissue sequencing plus (DBiTplus), an integrative multimodal spatial omics approach combining sequencing-based spatial transcriptomics and multiplexed protein imaging on the same section, enabling both single-cell-resolution cell typing and transcriptome-wide interrogation of biological pathways. DBiTplus utilizes spatial barcoding and RNase H-mediated cDNA retrieval, preserving tissue architecture for multiplexed protein imaging. We developed computational pipelines to integrate these modalities, allowing imaging-guided deconvolution to generate single-cell-resolved spatial transcriptome atlases. We demonstrate DBiTplus across diverse samples including frozen mouse embryos, and formalin-fixed paraffin-embedded human lymph nodes and lymphoma tissues, highlighting its compatibility with challenging clinical specimens. DBiTplus uncovered mechanisms of lymphomagenesis, progression and transformation in human lymphomas. Thus, DBiTplus is a unified workflow for spatially resolved single-cell atlasing and unbiased exploration of biological mechanisms in a cell-by-cell manner at transcriptome scale. The advent of high-throughput single-cell technologies has revolutionized our understanding of biological systems, enabling comprehensive analyses at the molecular level across diverse biological contexts1. These approaches lack spatial context, limiting our understanding of intercellular interactions within tissues. Spatial omics addresses this limitation using a wide range of approaches. Array-based approaches such as Slide-seq2 utilize spatially barcoded surfaces to capture mRNA transcripts. Tissue barcoding approaches such as DBiT-seq3, spatial CITE-seq4 and spatial ATAC-seq5 use spatially defined delivery of DNA barcodes to profile the transcriptome, proteome or epigenome. Imaging-based methods such as MERFISH6 utilize combinatorial labeling and sequential imaging to achieve subcellular resolution. Similarly, technologies such as STARmap7 sequence nucleic acids directly within tissues or cells. Broadly, in situ sequencing or imaging techniques offer higher spatial resolution and sensitivity but may lack transcriptome-wide coverage, whereas sequencing-based methods provide transcriptome-wide coverage, often at the expense of high spatial resolution. Multiplexed protein imaging technologies such as CODEX8 have also been developed, offering single-cell resolution profiling of protein markers for in situ cell typing. Currently, most spatial multi-omic technologies involve running single spatial omics assays separately on adjacent or serial tissue sections, followed by computational data integration of the multimodal datasets. Due to heterogeneity in cellular composition and tissue architecture, even between adjacent sections from the same block, integrating multimodal data computationally may be suboptimal, as perfect concordance between tissue sections is almost unattainable. Thus, new methods for spatially resolved multimodal and multi-omics measurements on the same tissue sections are necessary. Single-cell spatial multimodal metabolomics approaches such as REDCAT combine protein and metabolite measurements in the same tissue section9. IN-DEPTH was also recently developed for same-slide spatial multi-omics integration10. Here, we developed DBiTplus, which combines spatially resolved transcriptomics sequencing with multiplexed immunofluorescence (mxIF) imaging for the unbiased co-profiling of the whole-transcriptome and protein markers on the same tissue section. We tested several cDNA retrieval methods, settling on an enzymatic approach using RNase H after spatial barcoding, which maintains tissue integrity and morphology before mxIF imaging. A computational approach, modified from the MaxFuse algorithm11, was developed to integrate the multi-omic spatial transcriptomic and mxIF datasets, and a robust cell-type decomposition (RCTD)-like approach12 used for DBiTplus spot cell-type deconvolution and spot splitting to generate pure cell-type sub-spots. We applied DBiTplus to elucidate the process of embryogenesis in optimal cutting temperature (OCT)-frozen and paraffin-embedded C57BL/6 (C57) mouse embryo sections. Applying DBiTplus to healthy human lymph node and lymphoma tissues demonstrated the capability to generate high-quality single-cell-resolved transcriptome and protein data from the same tissue section, addressing the hurdles associated with data integration and registration from adjacent sections. In the standard DBiT-seq workflow, mRNAs are reverse-transcribed in situ within the tissue matrix, and DNA barcodes A (Ai; i = 1–50) and B (Bj; j = 1–50) delivered perpendicularly through microfluidic chips with 50 parallel channels. The barcodes ligate to form a unique two-dimensional array of barcoded spots. After tissue lysis, barcoded cDNAs are recovered, purified and amplified for paired-end sequencing to generate spatial gene expression maps. This technique has been adapted to profile the transcriptome, epigenome and proteome and, more recently, applied to archival formalin-fixed paraffin-embedded (FFPE) tissue blocks3,4,5,13. Two chemical approaches (using sodium hydroxide (NaOH) and dimethylsulfoxide (DMSO)) and an enzymatic approach (using RNase H) were tested (Extended Data Fig. Following successful retrieval of the cDNA and preparation of a sequencing library, the intact tissue section was imaged using Akoya Biosciences' PhenoCycler-Fusion platform or Bruker Spatial Biology's CellScape platform, and routine hematoxylin and eosin (H&E) staining performed on the same tissue section (Fig. For FFPE samples, the Patho-DBiT workflow13 was used (until spatial barcoding was completed), following which the tissue section was incubated at 55 °C with a mix of Triton X-100 and Thermostable RNase H enzyme to break down RNA strands in RNA–DNA hybrids and to facilitate the diffusion through the permeabilized cell membranes. Another overnight incubation at 37 °C was performed to increase cDNA recovery from the tissue. The retrieved cDNAs were pooled and a sequencing library built. The intact tissue section can be stored at −20 °C until mxIF imaging. FFPE tissue sections underwent brief rehydration and antigen retrieval steps before mxIF imaging. For fresh frozen samples, the DBiTplus workflow was identical, excluding FFPE-specific steps. We developed a workflow integrating the mxIF cell-by-protein and DBiTplus spot-by-gene matrices into a unified feature matrix with cell-type labels and spatial coordinates (Fig. 1b), starting with whole-cell segmentation of the mxIF data using Mesmer14 and quality-control steps. The mxIF datasets were annotated via MaxFuse11 integration with a reference single-cell RNA-sequencing (scRNA-seq) dataset such as the Mouse Organogenesis Cell Atlas15. Utilizing tissue boundary detection and image transformation, mxIF and DBiT-seq images were co-registered to enable accurate matching and alignment of the same cells across modalities (Extended Data Fig. Cell-type counts from mxIF segmentation masks were mapped to DBiTplus spots, which were subdivided into pure cell-type sub-spots for cell-type-specific gene expression estimation, enabling single-cell annotation beyond existing deconvolution tools (Fig. Preliminary experiments on OCT-frozen embryonic day 13 (E13) mouse C57 embryo sections tested cDNA retrieval methods. The barcoded region of the tissue was covered with a clean polydimethylsiloxane (PDMS) well gasket and incubated at room temperature with 50 μl of 0.1 M NaOH for 15 min, which was then collected and neutralized with equimolar hydrochloric acid (HCl). NaOH disrupted tissue morphology (Extended Data Fig. In mouse spleen serial sections, each section was subjected to a different cDNA retrieval approach: 0.1 M NaOH for 5 min, 90% DMSO for 30 min and 10 U of RNase H at 37 °C for 30 min. The samples were then imaged with a 25-marker mouse immune panel by Akoya Biosciences. DMSO-treated and NaOH-treated sections yielded poor-quality CODEX staining, whereas the RNase H-treated section yielded 14 positively staining markers including CD45R (B cell lineage) and CD169 (macrophages; Extended Data Fig. Thus, the enzymatic approach was selected for further optimization (Extended Data Fig. E13 mouse embryo sections underwent spatial barcoding with proteinase K lysing (control) or RNase H-mediated cDNA release (test). Sequencing detected 24,102 and 20,973 genes, respectively (Extended Data Fig. Adding 0.5% Triton X-100 to the RNase H mix did not compromise tissue integrity, but saw an increase in mean gene counts per spot (Extended Data Fig. Spatial clustering identified eight clusters, with clusters 2 and 3 corresponding to the diencephalon, mesencephalon and telencephalon, evidenced by the expression of Zbtb20 and Id4 (embryonic neocortex, forebrain; Extended Data Fig. Spatial patterns of select genes—Sox11, Col2a1 and Sox2—matched in situ hybridization data from the Allen Brain Atlas and the Mouse Organogenesis Spatiotemporal Transcriptomic Atlas (MOSTA) from STOmicsDB16 (Extended Data Fig. Clusters 0 (control) and 2 (test) showed remarkably similar spatial distributions and 1,109 overlapping genes, underscoring the ability of DBiTplus to generate high-quality spatial transcriptome data and recapitulate tissue biology (Extended Data Fig. We profiled human cerebellum and lymph node FFPE sections, which retained their architecture after spatial barcoding and cDNA recovery (cDNA size ranged from 200 to 800 base pairs; Supplementary Fig. AQP4+ glial cells were observed across all three layers, and the single layer of large pear-shaped CALB1+ Purkinje cell bodies was most prominent in the Purkinje cell layer (Supplementary Fig. Lymph node CODEX staining recapitulated expected architecture, including CD20+ B cells and CD21+ follicular dendritic cells in the follicle, and CD3ε+ T cells in interfollicular regions (Supplementary Fig. DBiTplus was applied to an E11 paraffin-embedded mouse embryo (Fig. 2a) and showed a strong correlation (R = 0.99) with the standard DBiT-seq workflow (on the adjacent section) with 27,884 overlapping genes. Each spot captured ~1,200 genes and 3,300 unique molecular identifiers (UMIs; Fig. The same section was stained with a 26-marker CODEX panel. Unsupervised clustering identified ten transcriptomic clusters, which closely aligned with anatomic structures and spatial protein patterns from the CODEX data (Fig. Cluster 4 expressed liver markers (Hba.a1, Hba.a2, Hbb.bt, Afp and Serpina6), while cluster 8 expressed embryonic heart markers (Myh6, Myl7, Myh7 and Tnnt2; Extended Data Fig. mRNA and protein expression showed consistent spatial patterns for genes such as Mki67, Nefl and Sox2 (Fig. Furthermore, spatial expression of Sox2 (pluripotency, cluster 0) Hoxa10 (limb muscle development, cluster 1) and Sox11 (progenitor cell behavior regulation including neurogenesis, cluster 2) matched in situ hybridization data from the Allen Brain Atlas and the MOSTA dataset (Extended Data Fig. A trained support vector machine model enabled comprehensive cell-type annotation of the whole CODEX dataset (Supplementary Figs. Integration of CODEX and DBiT-seq allowed deconvolution of DBiTplus spots into constituent cell types (Fig. Using Seurat weighted-nearest-neighbor (WNN) methodology, further enhanced cell-type separation, clearly distinguishing epithelial cells from other cell types. a, Brightfield images of an E11 FFPE mouse embryo section before (top) and after (bottom) the DBiTplus workflow, showing that tissue integrity and morphology are preserved following cDNA retrieval (n = 2). b, Correlation analysis between two FFPE E11 mouse embryo section samples comparing the standard DBiT-seq and the DBiTplus workflows from a two-sided Pearson correlation test (Pearson correlation = 0.99 and P value < 2.2 × 10−16). c, Venn diagram showing overlap of genes sequenced between the standard DBiT workflow and the DBiTplus workflow. d, Distribution of detected genes and UMIs per spatial spot. e, Top, UMAP clustering of spatial transcriptomic data identified ten distinct transcriptomic clusters from the E11 mouse embryo. Bottom, mxIF staining performed on the same tissue section. f, Comparison of the spatial gene expression (DBiTplus) and protein expression (CODEX) for selected markers reveals concordant spatial localization across both modalities. g, CODEX-informed spot deconvolution of DBiTplus data. Cells were annotated by label transfer from the MOCA dataset using MaxFuse. We applied DBiTplus to adjacent sections of a benign human lymph node, using 50-μm (Fig. 3) and 25-μm microfluidic devices, respectively (Supplementary Fig. After cDNA retrieval, sections were stained with a 35-plex CODEX panel and H&E staining was done (Fig. Unsupervised clustering revealed five transcriptomic clusters, including B cells (MS4A1, cluster 1), smooth muscle cells in the medulla (MYH11 and CALD1, cluster 3) and macrophages lining the medullary cords (MARCO, cluster 4; Fig. Uniform manifold approximation and projection (UMAP) embedding of the reference scRNA-seq and CODEX datasets using MaxFuse is shown in Fig. Cell-type distributions matched known lymph node biology, with T cells and B cells predominating (Fig. Joint embedding of DBiTplus and CODEX (Fig. 3g) and violin plots of the cell-type-specific modality weights from WNN analysis, showed that the CODEX (protein) contributed more for T cell-subtype identification, relative to DBiTplus (transcriptome), and the converse was observed for B cell subtypes such as GCB and plasma B cells (Fig. This can be explained by the marker composition of the CODEX panel, which included more T cell than B cell markers. a, Spatial transcriptomic profiling of an FFPE benign human lymph node section at 50-μm resolution using DBiTplus (n = 1). Left, brightfield and H&E images of the same tissue section after DBiTplus and CODEX. Right, spatial clustering of spots reveals five distinct transcriptomic regions that spatially correspond to distinct histological regions within the human lymph. b, Heat map showing the top five differentially expressed genes (DEGs) across spatial transcriptomic clusters. c, mxIF staining using CODEX on the same tissue section (n = 1). Bottom, high-resolution view of the boxed region. e, UMAP projection of integrated protein (CODEX) and RNA (DBiTplus) modalities shows strong concordance following MaxFuse integration. f, Cell-type composition derived from CODEX-labeled cells within DBiTplus spots, visualized as bar plots (left) and pie charts (right) for T cells and B cells. g, UMAP plots of DBiTplus, CODEX and Seurat WNN-integrated datasets colored by cell-type annotation. h, Seurat WNN-derived modality weights for each cell type. The strongest weights were observed for T cell subtypes and the lowest were observed for B cell subtypes. i, CODEX-informed spatial deconvolution of DBiTplus transcriptomic spots. Left, deconvolved spatial map, number of cells and cell types inferred from CODEX cell segmentation and cell-type annotation; middle: zoomed-in spot-level cell-type compositions; right, multiplexed image of the same region from CODEX (DAPI, CD20, CD8, CD4, FOXP3, CD31). j, Spatial deconvolution of DBiTplus data using TACCO. 50% of spots have Pearson correlation scores >0.6 when compared with ground truth (CODEX-informed spot-level cell-type deconvolution from i). k, Cell2location-based deconvolution and spatial mapping of cell types. Left to right, cell-type map, spot-level pie charts, correlation heat map and correlation score distribution. 36% of spots have Pearson correlation scores >0.6 when compared with ground truth (CODEX-informed spot-level cell-type deconvolution from i). Each DBiTplus spot was subdivided into pure cell-type sub-spots, with cell-type identities assigned through MaxFuse label transfer and validated against CODEX markers (4′,6-diamidino-2-phenylindole (DAPI), CD20, CD8, CD4, FOXP3, CD31) from the same region of the tissue. Three methods—TACCO (Optimal transport-based)17, Cell2location (Bayesian probabilistic model)18 and RCTD (Poisson regression-based deconvolution)12—were tested to benchmark spatial cell-type deconvolution, relative to the ground-truth CODEX-informed spot-level cell-type dataset (Fig. TACCO performed best, with 50% of spots showing Pearson scores > 0.6, versus 36% for Cell2location (Fig. Metrics for RCTD are shown in Extended Data Fig. Average silhouette width (ASW) scores, with or without CODEX (0.47 versus 0.46), were similar for the DBiTplus transcriptome dataset, indicating comparable clustering structure in the DBiTplus dataset alone. However, adjusted Rand index (ARI) scores improved from 0.09 to 0.21 with CODEX guidance, reflecting more accurate cell-type assignment (Extended Data Fig. UMAP of the DBiTplus (transcriptome) revealed diverse immune (B cell and T cell subsets) and stromal populations (Extended Data Fig. Cell-type-specific markers MS4A1 (canonical B cell marker), IL7R (naive CD4+ T cells), CTLA4 (regulatory T cells) and CALD1 and MYH11 (endothelial and vascular smooth muscle cells) distinguished immune subsets (Extended Data Fig. IL4R was enriched in GCB/naive/activated B cell subsets, indicating activation or a poised state for activation, while MKI67 and TOP2A were upregulated in cycling B cells, indicating active proliferation. These patterns validate B cell-subtype annotations and capture functional transitions from naive to proliferative and terminally differentiated states (Extended Data Fig. To delineate transcriptional differences between antigen-inexperienced and antigen-experienced B cells, we compared naive and activated B cells using differential gene expression and pathway analysis. Naive B cells upregulated MS4A1, IGHM, IL4R and POU2F2, consistent with a resting phenotype capable of antigen sensing and homing. Correspondingly, B cell antigen receptor (BCR) signaling, CXCR4 signaling and RHO GTPase cycle pathways were upregulated, whereas apoptosis signaling was downregulated, underscoring reflecting migratory readiness and pro-survival status. These findings underscore functional divergence between naive and activated B cells and validate the resolution of our spatial transcriptomic profiling. We evaluated RNA–protein concordance in DBiTplus, comparing spatial transcriptomics with CODEX imaging. This is important because the relationship between protein levels and their coding transcripts can be discordant, influenced by spatial and temporal mRNA variations and the protein biosynthesis machinery19. As validation, we compared our normalized gene expression to a reference lymph node dataset from Bai et al.13 (also using Patho-DBiT), noting generally consistent gene expression levels (R = 0.89, P < 2.2 × 10−16), particularly for highly abundant transcripts like PTPRC, ACTB and HLA.DRA. Low-expressing genes, such as GZMB and IFNG, were present at low levels in both datasets. Interestingly, CD68 transcripts were detectable in the reference dataset (Extended Data Fig. Given the documented discrepancies between transcript and protein expression in immune cells, we evaluated the correlation between average normalized mRNA and protein levels in our dataset20. As expected, we observed minimal correlation (R = 0.23, P = 0.18), consistent with prior findings in peripheral blood mononuclear cells, particularly for T cell markers such as CD4 (Extended Data Fig. CD68 protein expression was clearly visible in the CODEX images (Extended Data Fig. 6e,f), and other macrophage-associated markers such as CD14 and CD163 were detected in both modalities (Extended Data Fig. These findings underscore the strength of DBiTplus in co-mapping RNAs and proteins on the same tissue section, allowing for cross-validation and enhancing confidence in cell-type annotation and spatial localization. Assessment of CODEX data from the DBiTplus workflow revealed staining quality comparable to control lymph node sections (two sections away from the Fig. 3 sample), with strong correlations (Pearson > 0.85) for key cell-type markers including CD3ε, CD20 and podoplanin (Supplementary Figs. H&E staining after DBiTplus, evaluated against standard H&E and post-CODEX H&E (no prior DBiT), showed partially diminished nuclear detail, but still allowed trained pathologists to identify major cell types based on cellular morphology and spatial context (Supplementary Fig. This suggests the utility of these H&E images for inferring single-cell spatial gene expression in the dead space regions of the microfluidic channels using tools like iStar21. It is worth noting that post-CODEX H&E imaging can be challenging due to potential tissue damage during the flow cell removal, which may compromise reliable histological analysis. To investigate lymphoma progression, we profiled a marginal zone lymphoma (MZL) sample that showed increased large cell populations, high proliferation index and clinical progression without overt histological transformation (Fig. Unlike healthy lymph nodes with well-defined follicular structure, MZL is characterized by effacement of normal tissue architecture through the infiltration of large, atypical B cells and proliferation of small to medium-sized lymphocytes with irregular nuclei, condensed chromatin and scant cytoplasm (Supplementary Fig. Interestingly, we observed robust follicular dendritic cell networks marked by CD21, with clusters of CD3+ and CD5+ T cells. a, Schematic overview of the study design. b, H&E staining of the FFPE lymph node section showing both small and large malignant lymphocytes (n = 1). c, Multiplexed CODEX imaging of the same tissue section (n = 1), showing CD20, CD3E, CD31 and Ki67. Zoomed-in image showing detailed views of proliferative tumor zones and vasculature. d, Spatial cell-type mapping using CODEX-guided DBiTplus transcriptomics. e, Bar plot of cell-type proportions across the lymphoma section dominated by B cells (small and large) and T cells. g, Volcano plot highlighting the DEGs in small B cells and large B cells, with key transformation-associated genes labeled. Each dot represents a specific gene, with upregulation in small and large B cells colored in blue and red, respectively. (Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction.) h,i, Pathway enrichment analysis of small B cells (h) and large B cells (i) showing distinct pathway activation profiles during transformation. Small B cells are enriched for signaling pathways such as BCR and NF-kB signaling, while large B cells show enrichment for PI3K–AKT and NOTCH signaling pathways. j, Monocle 3 UMAP embedding colored by B cell subtype shows a continuum from small to large B cells. k, Pseudotime trajectory analysis reveals a dynamic transcriptional progression along the transformation axis. A 2.5-mm × 2.5-mm region was profiled with DBiTplus (25-μm microfluidic device) and CODEX (44-marker panel) on the same tissue section. CODEX revealed extensive CD20+ Ki67+ proliferating B cells (Fig. B cells were classified as large or small B cells based on size and CD20 intensity. Integration of protein and RNA data revealed concordant expression of CD31, CD4 and TIGIT in the joint UMAP embedding (Supplementary Fig. The transcriptional profile of large B cells was consistent with biological evolution with upregulated genes like IL6ST (interleukin-6 signaling), CLU (apoptosis inhibition) and SYNE2, AHNAK and MARCKSL1 (protein synthesis and cell growth)22,23 indicating increased survival, and dysregulated inflammatory signaling (Extended Data Fig. DEG analysis between small and large B cells showed upregulation of NFKBIA, suggesting potential dysregulation of the frequently overactive nuclear factor (NF)-κB pathway, as well as upregulation of PPRX1, ANTRX1 and VMP1, associated with migration/metastasis, PI3K–AKT–mTOR signaling and autophagy, respectively, in large B cells24,25,26. Small B cells showed upregulation of PKHD1L1 and EXOSC10, associated with B cell lymphoma growth, activation and DNA repair27,28 (Fig. Upstream regulator analysis of DEGs in large B cells identified SOX4 (a MYC upstream regulator through ATK family and TP53 inhibitor) and downregulation of RAD21 (cohesion complex gene), whose reduced expression in diffuse large B cell lymphoma (DLBCL) correlates with decreased survival, suggesting underlying pro-tumor survival mechanisms29 (Extended Data Fig. Causal network analysis (which identifies upstream regulators of gene expression in large B cells) highlighted SYK, an SRC family kinase involved in BCR and PI3K–AKT signaling, as a key master regulator. Additional regulators included upregulated DOCK2 (proliferation regulator through RAC and ERK activation in B cell lymphomas) and downregulated SPEN (Notch signaling pathway regulator; Extended Data Fig. 7g,h), These transcriptional shifts offer insight into the biological programs underlying biological progression of MZL. We performed pseudotime analysis on the deconvoluted data of small and large B cell transcriptomes using Monocle 3 (ref. 15) and found small and large B cells on a differentiation continuum, revealing dynamic transcriptional changes across MZL progression (Fig. ANTXR1 is known to be elevated in the progression of several cancers and has recently been identified as a therapeutic target in DLBCL30. These findings suggest that MZL progression (with the increased presence of large B cells as well as aggressive clinical features) is driven by a coordinated loss of B cell identity and immune function, coupled with the upregulation of proliferation, transcriptional reprogramming and epigenetic remodeling pathways, characteristic of aggressive lymphoma transformation. Richter's transformation is the evolution of chronic lymphocytic leukemia (CLL) into aggressive lymphomas, often DLBCL, although rare cases involve Hodgkin or T cell lymphomas31. Recent multi-omics studies have identified distinct Richter's transformation subtypes, clonal origins and high-risk genomic features32. Most DLBCL-Richter's transformation cases (~80%) are clonally related to the original CLL, representing true transformations, with poor prognosis (median overall survival <1 year), although de novo DLBCL can occur in individuals with concurrent CLL33. We profiled a rare case of concurrent CLL and Richter's transformation-DLBCL within the same lymph node, providing a unique spatially contiguous context to study Richter's transformation (Fig. H&E staining revealed a region of densely packed small monomorphic cells transitioning into a sheet of large pleomorphic cells with open chromatin corresponding to the CLL and DLBCL regions, respectively (Fig. We applied DBiTplus to 5-µm-thick sections followed by imaging with a 30-marker panel (using CellScape, with relevant CLL/DLBCL markers such as CD5, LEF1 and CD20). An adjacent section was also imaged as a quality-control measure (Extended Data Fig. mxIF imaging revealed proliferating CD20brightKi67high large B cells (DLBCL region), contrasting with the CD20dimKi67low small B cells (CLL region), as well as a higher density of T cells within the DLBCL region (Fig. Leveraging the distinct features of CLL and DLBCL (CD20dimKi67low versus CD20brightKi67high), B cells were initially classified as large or small based on size, and the remaining cells in the TME, annotated using MaxFuse (Fig. T cell infiltration was higher in DLBCL than CLL, with small B cells, on average, located closer to the nearest T cell compared to large B cells (Extended Data Fig. Large B cells had the largest diameters of ~20 μm (Fig. Some overlap of large B cells in CLL and small B cells in DLBCL suggested a gradual transition between the two regions (Extended Data Fig. To refine CLL cell classification, we applied a gating strategy based on cell with size and normalized CD20 and Ki67 expression (Extended Data Fig. a, Schematic overview of transformation from CLL (CD5+CD20dimCD23+LEF1+Ki67low) to DLBCL (CD5+CD20brightCD23+LEF1−Ki67high). b, H&E staining of FFPE lymph nodes showing histologically distinct regions of CLL and DLBCL (n = 1). Right, brightfield image of the DBiTplus-barcoded region. c, mxIF imaging (using CellScape) of the same tissue section showing CD20, CD3E, Ki67, COL4A1 and DAPI staining (n = 1). Arrowheads point to highly proliferative large B cells. d, Spatial distribution of T cell markers (CD4, CD45RO, CD3E and CD45RA) in DLBCL versus CLL regions, highlighting increased T cell infiltration in transformed zones. e, Spatial cell-type deconvolution using CellScape-guided DBiTplus transcriptomics reveals regional differences in immune cell composition across the tissue section. Cell-type annotation legend shown in f. f, Bar plot showing cell-type proportions; large B cells dominate DLBCL regions, while small B cells are enriched in CLL regions. Significance level was calculated with unpaired two-tailed Welch t-test, ****P < 0.0001. h, Spatial gradient expression of selected protein markers (CD20, CD3, CD274/PD-L1, CD163) highlights differences in tumor and immune cell distribution across the tissue. i, Violin plot of the quantification of functional immune signatures (exhaustion, activation, cytotoxicity, suppression and proliferation) comparing DLBCL and CLL regions. k, Volcano plot showing DEGs in the small cells. Each dot represents a specific gene, colored in blue (downregulated) and red (upregulated), respectively. (Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction.) l, Pathway analysis of small B cells. z-score is computed and used to reflect the predicted activation level. n, Spatial BCR activation score map derived from DBiTplus transcriptomics, revealing localized activation patterns across the tissue section. Spatial gradient plots revealed enrichment of CD163+ macrophages, CD274 (PD-L1) and CD20+ cells in DLBCL regions (Fig. 5h), indicating an immunologically active yet suppressed microenvironment. Regional scores for activation (CD38, CD45RO), cytotoxicity (granzyme B, CD56, CD8), proliferation (Ki67), immunosuppression (CD274, FOXP3, CD163) and exhaustion (CD279) using multiplexed imaging data (Fig. 9) were all higher in DLBCL than CLL regions (two-tailed Mann–Whitney test, ****P < 0.0001), mirroring the distribution of CD8+ T cells with highest exhaustion and large B cells, macrophages and regulatory T cells showing highest immunosuppression scores within the TME. This aligns with reports of aberrant PD-1+ neoplastic B cells and increased infiltration of FOXP3+ T cells and CD163+ macrophages in Richter's transformation versus CLL, potentially influencing responses to immune checkpoint blockade. Additionally, CLL cells have been shown to overexpress PD-L1, which engages PD-1 on T cells, promoting immune tolerance through downstream inhibitory signaling36. UMAP and spatial clustering revealed clear separation of cell types (Extended Data Fig. Large B cells uniquely expressed ROR2, SMOC2 and PDE1C, implicated in proliferation, migration and regulation of the PI3K–Akt pathway, while small B cells overexpressed anti-apoptotic BCL2, consistent with their more indolent and chronic nature, and AFF3, suggesting IGHV-mutated CLL (Extended Data Fig. Pathways upregulated in small B cells included BCR signaling, histone modification and T cell exhaustion, whereas CTLA4 signaling in cytotoxic T lymphocytes was downregulated (Fig. In large B cells, p53 and apoptosis signaling were upregulated, although PTEN was downregulated, suggesting activation of intrinsic stress responses, whereas canonical proliferative and epigenetic programs such as mTOR, ERK/MAPK, PI3K signaling, and DNA methylation and transcriptional repression were downregulated. This reflects the hypomethylation typically observed in Richter's transformation versus CLL and de novo DLBCL and indicates a shift from canonical proliferative circuits. Additionally, suppression of T cell antigen receptor signaling may reflect impaired immune engagement, and upregulation of extracellular matrix remodeling suggests altered tumor–stroma interactions (Extended Data Fig. Aberrant cell-surface expression of co-inhibitory receptors CTLA4 and LAG-3 on large B cells may play a role in immune escape (Fig. Spatial scoring of T cell activation and exhaustion (based on previously published gene lists38,39,40) highlighted DLBCL hotspots, while higher scores of BCR activation were observed in the CLL region, suggesting reduced BCR survival signaling dependency in the transformed lymphoma (Supplementary Fig. DBiTplus further enables spatial profiling of small noncoding RNAs such as microRNAs (miRNAs), which regulate mRNA synthesis and gene expression41. The miR-17-92 cluster, miR-150 and miR-15b are critical for B cell differentiation and germinal center selection, whereas miR-21, miR-155 and miR-222 are known to be upregulated in lymphoid malignancies, with high miR-21 expression linked to the activated B cell subtype of DLBCL42. Unsupervised clustering at the DBiTplus spot level revealed seven distinct miRNA clusters, with clusters 0 and 1 corresponding to the histological transition from CLL to DLBCL (Extended Data Fig. Differential analysis identified several miRNAs relevant to disease progression: miR-34a, miR-21 and miR-155 (Extended Data Fig. Interestingly, miR-21 is known to inhibit expression of PTEN and activate the PI3K–AKT pathway, increasing chemotherapy resistance43, confirming the small and large B cell pathway analysis (Extended Data Fig. Other miRNAs such as miR-342 and miR-132, implicated in other lymphomas, warrant further investigation in the context of Richter's transformation. These findings demonstrate the unique capability of DBiTplus to spatially map miRNAs and interrogate their roles in hematological malignancies. We applied Monocle 3 to the DBiTplus-deconvoluted single-cell transcriptomes of this CLL-DLBCL transformation. UMAP visualization revealed distinct clusters for large B cells and small B cells, indicating substantial transformation-associated transcriptional shifts (Extended Data Fig. Pseudotime analysis of small B cells revealed dynamic gene expression changes: ATM (linked to NF-κB activation), MKI67 (proliferation) and LEF1 (diagnostic CLL marker) increased over pseudotime, whereas AFF3, BCL2 and TCL1A decreased (Extended Data Fig. Pseudotime heat map highlighted progressive upregulation of several CLL-associated genes across multiple pathways such as DNA damage response (TP53, ATM), chromatin modification (ASXL1, SETD2) and RNA splicing (SF3B1; Extended Data Fig. BIRC3, a tumor suppressor and negative regulator of noncanonical NF-κB signaling in CLL, exhibited transient upregulation before declining, consistent with 11q deletion-associated poor prognosis. These transcriptional dynamics reveal gradual remodeling of the CLL transcriptome toward a more aggressive state, underscoring the molecular changes that may underlie Richter's transformation. Spatial multi-omics integrates genomics, transcriptomics, proteomics and metabolomics while preserving spatial context, providing a comprehensive view of molecular processes in tissues. Traditional approaches utilize separate assays on adjacent tissue sections limiting alignment and multimodal integration accuracy due to tissue heterogeneity. To overcome this, we developed DBiTplus, which combines unbiased transcriptome-wide spatial sequencing with mxIF (CODEX or CellScape) on the same tissue section, compatible with both OCT-frozen and FFPE samples. We optimized enzymatic cDNA retrieval using RNase H, while preserving tissue architecture for multiplexed imaging. By registering DBiTplus data with high-resolution imaging, we achieve precise colocalization of transcriptomic and proteomic data. This enabled mxIF-informed DBiTplus spot deconvolution, allowing for accurate identification of cell types. Our approach involved splitting spots into pure cell-type sub-spots and utilizing the Seurat WNN methodology for reliable cell-type deconvolution and splitting the transcriptomes of individual sub-spots into single-cell transcriptional profiles. Thus, leveraging mxIF data to guide the splitting of DBiT spatial transcriptomes can enable the creation of truly single-cell-level spatially resolved transcriptome atlases. DBiTplus also captures the whole milieu of small RNAs, particularly miRNAs, providing insights into the molecular biological mechanism of disease evolution. DBiTplus shares common limitations with sequencing-based spatial transcriptomics approaches; low capture depth can lead to dropout of low-abundance transcripts, which may be exacerbated by the lower transcript recovery rate of DBiTplus, compared to the standard DBiT-seq workflow, an important consideration when profiling rare cell populations. Nonetheless, these limitations are counterbalanced by several strengths: DBiTplus enables whole-transcriptome spatial profiling without being constrained by predesigned panels, can be integrated with high-plex protein imaging on the same tissue section, and is flexible and cost-effective for broad adoption. While Xenium and CosMx achieve single-cell spatial resolution through direct imaging of transcripts, DBiTplus even allows for unbiased profiling of total RNAs including small noncoding RNAs (that is, miRNAs), which is unique for discovery of new RNA biological mechanisms. In summary, DBiTplus represents a spatial multi-omics approach that integrates sequencing-based and imaging-based spatial assays on the same tissue section, enabling image-guided deconvolution into single-cell-resolved spatial transcriptomes. By combining multiple molecular layers at single-cell resolution, DBiTplus provides unprecedented insights into tissue architecture and cellular interactions, opening new avenues for spatial multi-omics. De-identified archived benign FFPE human lymph node and lymphoma tissue blocks were obtained from Yale Pathology Tissue Services (YPTS). Written informed consent for participation in any cases where identification was collected alongside the specimen, was obtained from individuals or their guardians, in accordance with the principles of the Declaration of Helsinki. Each sample was handled in strict compliance with HIPAA regulations, University Research Policies, Pathology Department diagnostic requirements and Hospital bylaws. The mouse tissue used in this study was obtained from a commercial vendor, Zyagen (San Diego, CA), which procured and handled the animals under their in-house Institutional Animal Care and Use Committee (IACUC)-approved protocols. Because no live animal procedures were conducted at our institution, separate IACUC approval was not required. E11 mouse whole-embryo paraffin sagittal sections were made of freshly collected tissues, fixed in 10% neutral buffered formalin, and processed for paraffin embedding. Both OCT and paraffin blocks were also sectioned at a thickness of 7–10 μm and mounted on the center of poly-L-lysine-covered glass slides (63478-AS, Electron Microscopy Sciences). After review and selection by a board-certified pathologist, optimal paraffin blocks were sectioned by YPTS at a thickness of 7–10 μm and mounted on the center of poly-L-lysine-coated 1 × 3-inch glass slides. Serial tissue sections were collected simultaneously for DBiT-seq and H&E staining. Human brain cerebellum paraffin sections (HP-202) were purchased from Zyagen and made of freshly collected tissues, fixed in 10% neutral buffered formalin and processed for paraffin embedding. Paraffin sections were stored at −80 °C until use. To perform RNA integrity number tests, 15–20-μm-thick curls were obtained from YPTS. The RNeasy FFPE Kit for RNA Extraction from Qiagen was used. Details of the fabrication process for the PDMS wafers and microfluidic chips can be found in a prior publication3. The DNA oligonucleotides were obtained from Integrated DNA Technologies, with the sequences provided in Supplementary Tables 6 and 7. The mixes were placed in a thermal cycle and heated to 97 °C to anneal and slowly cooled to room temperature at a rate of −0.1 °C s−1. The barcodes can be stored at −20 °C for up to 6 months. The section was then fixed with 4% formaldehyde for 20 min and washed three times with 0.5× DPBS-RI (1× DPBS diluted with nuclease-free water and 0.05 U μl−1 RNase Inhibitor). The tissue was permeabilized for 20 min at room temperature using 0.5% Triton X-100 in DPBS, followed by a wash with 0.5× DPBS-RI (1× DPBS diluted with nuclease-free water and 0.05 U μl−1 RNase Inhibitor) to stop the permeabilization. After air-drying, a PDMS reservoir was placed over the region of interest (ROI) on the tissue slide. In situ polyadenylation was performed with Escherichia coli poly(A) polymerase. To remove excess reagents, the slide was dipped in 50 ml DPBS and shake-washed for 5 min. The sample was incubated at room temperature for 30 min and then at 42 °C for 90 min, followed by a wash with 50 ml DPBS. The chip was imaged to record positions for downstream alignment and analysis. An acrylic clamp was used to secure the PDMS to the slide, preventing interchannel leakage. After incubating at 37 °C for 30 min, the PDMS chip was removed, and the slide was washed with 50 ml of DPBS. Next, a second PDMS device with 50 perpendicular channels was attached to the air-dried slide over the ROI. The tissue section was then washed with nuclease-free water to remove any residual salts, and a final brightfield image was taken to mark the boundaries of the microfluidic channels on the tissue ROI. Tissue sections were baked for 1 h at 60 °C to facilitate the removal of paraffin and increase adhesion of the tissue section to the slide. Finally, the tissue sections were immersed in distilled water for 5 min. Next, the tissue slide was immersed into preheated antigen retrieval buffer (Discovery CC1 buffer (Roche, Basel) or Tris-EDTA Buffer, pH 9.0 (Abcam)) and allowed to boil at 95–100 °C for 10 min and then allowed to cool to room temperature. For cDNA retrieval, the barcoded region of the tissue was covered with a clean PDMS well gasket, and 100 μl cDNA extraction solution (10 μl 5% Triton X-100, 74 μl nuclease-free water, 10 μl 1× RNase H Reaction Buffer and 6 μl Thermostable RNase H (M0523S, New England Biolabs)) was loaded into it. The reservoir was then clamped tightly with the slide to avoid any leakage and sealed with parafilm. The clamped tissue slide was incubated in a humidified box at 55 °C for 3 h. Following this, the cDNA extraction solution was collected and 1 μl of 0.5 M EDTA was added to inactivate the RNase H enzyme. The intact tissue slide was washed with 100 μl of nuclease-free water, which was then collected. Following this, 100 μl of cDNA extraction solution was added to the tissue slide as described previously, and the clamped slide was incubated in a humidified box at 37 °C overnight. The cDNA extraction solution was collected and 1 μl of 0.5 M EDTA was added to inactivate the RNase H enzyme. The intact tissue slide was washed with 100 μl of nuclease-free water, which was also collected and an additional wash step with 0.1× SSC buffer done. The collected cDNA extraction solution was then pooled and stored at −80 °C until use. For control slides, the standard lysing process described in previous publications from the lab was followed3. The tissue clamps were removed, and the intact tissue was washed in nuclease-free water and then with 1× PBS. Following this, the tissue slide was incubated with 500 μl of DAPI solution (two drops of NucBlue Fixed Cell ReadyProbes Reagent in 500 μl of 1× PBS) and incubated at room temperature for 5 min. This image is used to co-register the DBiTplus and mxIF images. The tissue slide was then washed with 1× PBS three times and stored at −80 °C until the multiplexed fluorescence imaging step. Before use, the beads were washed three times with 1× B&W buffer containing 0.05% Tween 20 and resuspended in 100 µl of 2× B&W buffer (10 mM Tris-HCl pH 7.5, 1 mM EDTA, 2 M NaCl). The beads were then mixed with the purified cDNA in a 1:1 volume ratio and incubated with gentle rotation at room temperature for 60 min. The beads were subsequently washed twice with 1× B&W buffer and once with 1× Tris buffer containing 0.1% Tween 20. The template switch reaction was performed at room temperature for 30 min and then at 42 °C for 90 min with gentle rotation. Afterward, the beads underwent a single wash with 10 mM Tris-HCl pH 7.5 containing 0.1% Tween 20 and another wash with nuclease-free water. Second-strand synthesis was then performed as follows: the beads were washed twice with TE-TW buffer (10 mM Tris pH 8, 1 mM EDTA, 0.01% Tween 20) and resuspended in freshly prepared 200 μl 0.1 M NaOH for 5 min with gentle rotation. The beads were washed once with 500 μl of TE-TW, and once with 500 μl 1× TE buffer (10 mM Tris pH 8, 1 mM EDTA). This suspension was then distributed into PCR strip tubes. The PCR product was purified using SPRIselect beads at a 0.8× ratio, according to the manufacturer's standard protocol. The resulting cDNA amplicon was then analyzed using the TapeStation system with D5000 DNA ScreenTape and reagents. 20 ng of cDNA was used as the input amount, and three rounds of depletion were performed. We observed that two rounds of depletion could suffice. The library product was purified using SPRIselect beads at a 0.7× ratio, according to the manufacturer's standard protocol, and sent out to Novogene Corporation to be sequenced on an Illumina NovaSeq 6000 or NovaSeq X Plus Sequencing System with paired-end reads of 150 base pairs in length. A modified version of the CODEX PhenoCycler-Fusion protocol (https://www.akoyabio.com/wp-content/uploads/2021/01/CODEX-User-Manual.pdf) was adopted for tissue sections used in the DBiTplus workflow. Since the tissue had already been deparaffinized and rehydrated during the DBiTplus workflow, the CODEX process began with a gentle antigen retrieval step using 1× AR9 buffer for 5–10 min. The tissue was then allowed to cool to room temperature and was rinsed twice with nuclease-free water and hydration buffer, followed by staining buffer as the antibody cocktail was prepared. The tissue slide was incubated with the antibody cocktail at room temperature for 3 h in a humidified chamber. After incubation, the tissue underwent a series of steps including post-fixation, ice-cold methanol incubation and a final fixation step. Attached to the flow cell, the tissue section was incubated in 1× PhenoCycler buffer with additive for at least 10 min to improve adhesion. A final .qptiff file was generated, at the end that could be viewed using QuPath (V0.5.1)44. For further details on the PhenoCycler antibody panels, experimental cycle design and reporter plate volumes, see Supplementary Data 1. MxIF staining and imaging was performed using the CellScape platform (Bruker Spatial Biology). Human FFPE CLL samples were prepared as previously described and shipped to Bruker Spatial Biology in Saint Louis at 4 °C. Pre-DBIT samples were shipped in standard slide mailers. Post-DBIT samples were shipped in 50 ml conical tubes with PBS to prevent sample dehydration in transit. In preparation for staining, pre-DBIT samples were deparaffinized and rehydrated according to the CellScape user manual. Immediately following deparaffinization (pre-DBIT samples), or 24 h before running the CellScape experiment (post-DBIT), heat-induced epitope retrieval was performed at 110–120 °C under low pressure in a pressure cooker with samples placed in plastic Coplin jars (Fisher, Waltham) filled with Discovery CC1 buffer (Roche, Basel) for 15 min. Thereafter, samples were allowed to cool for 25 min on the benchtop. Slides were then washed with CellScape Wash Buffer, mounted in the CellScape Whole-Slide Imaging Chamber, which was immediately filled with CellScape Storage Buffer and stored at 4 °C until use. The multiplex proteomic assay was performed on the CellScape platform using automated iterative cycles of fluorophore-conjugated primary antibody staining, imaging and photobleaching. The assay consisted of 12 cycles, with each cycle beginning with a 10-s photobleach and subsequent background measurement for each channel to be stained. This was followed by automated staining of up to three antibodies incubated for 15 min per cycle. The 31-plex assay included Sytox Green (Thermo, S7020), the VistaPlex Cell Boundaries (Bruker Spatial Biology, VISTAPLEX3101), Immune Profiling (Bruker Spatial Biology, VISTAPLEX3102) and Architecture (Bruker Spatial Biology, VISTAPLEX 3103) kits, as well as the following custom-conjugated primary antibodies: LEF1 (Abcam, ab137872), CD5 (Leica, CD5-4C7-L-U) and CD23 (Leica, CD23-1B12-L-U). A razor was then used to carefully detach the flow cell from the tissue slide. The tissue slide was then rinsed thoroughly with deionized water and then with 1× PhenoCycler Buffer without additive three times for 10 min each. Histological H&E staining on the FFPE sections was conducted by YPTS. The H&E images were taken using the Motic EasyScan digital slide scanner at a magnification of ×40. The standard CODEX data preprocessing and filtering pipeline was first applied to extract cell positions and cell-level features from two adjacent CODEX slices separately. Six marker points were then manually identified from the two stacked CODEX images. This transformation was subsequently applied to align cell positions from the two slices into a common coordinate system. To ensure statistical robustness, the spatial dimensions were discretized into bins with a pixel size of 100 (~4,000 common bins), and the mean values of selected features were computed by aggregating the cells within each bin. Common bins across both datasets were identified, and Pearson correlation coefficients were calculated to quantify the strength of linear relationships between corresponding features, and scatterplots were generated to visually represent the correlations. Read 2 from the FASTQ file was processed to extract UMIs and spatial barcodes A and B. Read 1, containing cDNA sequences, was aligned to the mouse GRCm39 or human GRCh38 reference genome using STAR (V2.7.8a)45. Spatial barcode sequences were demultiplexed with ST_Pipeline (V1.8.1)46, using the predefined coordinates of the microfluidic channels, and ENSEMBL IDs were converted to gene names. This generated a gene-by-spot expression matrix for downstream analysis. Entries in the matrix that corresponded to spot positions without tissue were excluded. 5) was preprocessed following the ASTRO Pipeline47, also utilized by Bai et al.13. For comparative analyses, and to account for varying sequencing depths, the raw sequencing reads were uniformly downsampled to the read count of the sample with the minimum number of reads using Seqkit (V2.3.1)48. The downsampled reads were processed as described in the sequence alignment and generation of gene expression matrix section above. The average number of UMIs and genes per pixel were calculated during the spatial gene expression analysis using the Seurat (V4.3.0)49 pipeline, taking into account only useful pixels (pixels actually covering tissue) and were visualized as violin plots. Initially, gene expression within each spot was normalized and variance-stabilized using the SCTransform method, specifically designed for scRNA-seq datasets. Linear dimensional reduction was then performed with the ‘RunPCA' function, and the optimal number of principal components for further analysis was determined using a heuristic approach, visualized by an ‘Elbow plot' that ranks principal component analysis (PCA) components by their variance percentages. Subsequently, the ‘FindNeighbors' function embedded spots into a k-nearest-neighbor graph structure based on Euclidean distance in PCA space, and the ‘FindClusters' function applied a modularity optimization technique to cluster the spots. Finally, DEGs defining each cluster were identified using the ‘FindMarkers' function for pairwise comparisons between spot groups or ‘FindAllMarkers' for DEGs for each identity (cluster or cell type) versus all other cells combined. All scRNA-seq data utilized in this study are publicly available. For this study, cells with a ‘development_stage' of 10.5 to 11.5 days post-coitum were selected to correspond with CODEX measurements, using the MaxFuse method as described below. The scRNA-seq dataset for human lymph nodes was retrieved from the Cell2location repository, and is available for download at https://urldefense.com/v3/__https:/cell2location.cog.sanger.ac.uk/paper/integrated_lymphoid_organ_scrna/RegressionNBV4Torch_57covariates_73260cells_10237genes/sc.h5ad__;!!IBzWLUs!XDud8EZcXLnWNMGYjYHAFdgJyM_lWL-_j9l6RweZDqQXLP1uY3a78B7_-BgAEzSHFVLk6TfuCwekjLtKGhyb4z8FJVvCtJhUcRVGqg$/. Cell-type annotations were provided by the original study. For cell mask prediction, the default training resolution of 0.5 μm per spot was adopted. Cells within the [0.05, 0.95] cell-size quantile range and possessing DAPI signal intensities exceeding the 0.1 quantile threshold were retained for analysis. mxIF features were extracted by summing the signal for each feature per cell. For each mxIF feature, the 0.05 and 0.95 quantiles were calculated, and each single-cell-level mxIF feature was subsequently scaled to a [0, 1] range, with the 0.05 quantile mapped to 0 and the 0.95 quantile mapped to 1. Values exceeding this range were clipped to 0 or 1, as appropriate. Before integration, standard preprocessing protocols were applied to all scRNA-seq data using Scanpy. This preprocessing included count normalization, log1p transformation and the identification of highly variable genes, resulting in the selection of 5,000 genes exhibiting the highest variability. Linked features between the scRNA-seq and mxIF datasets were identified based on corresponding protein and gene names. From these linked features, those with a standard deviation greater than 0.01 were selected to enhance integration performance. Following integration, low-quality pivots were removed to ensure the reliability of the cross-modal pivot pairs. Approximately 10% of cells from the mxIF datasets, representing high-confidence matches, were selected to construct these pivot pairs. For each pivot pair, cell-type labels from the scRNA-seq data were transferred to the matched mxIF cells. To extend cell-type annotation to the entire mxIF dataset, a support vector machine model was trained on the pivot mxIF cells to predict cell-type labels based on protein expression measurements. To isolate the highly proliferative, enlarged B cell population in the MZL and CLL to DLBCL samples, we applied a three-way thresholding strategy based on protein abundance and morphometric size. Any cell whose CD20 intensity exceeded the CD20 0.6 quantile and whose Ki67 intensity exceeded the Ki67 0.6 quantile and whose physical cell size exceeded the area 0.6 quantile simultaneously was labeled as a ‘large B cell'. Conversion of cell size from pixel to area was based on the resolution of 0.5 μm per pixel. To align the high-resolution mxIF image with the low-resolution DBiTplus spot image of the FFPE mouse embryo dataset, tissue boundaries were initially identified in both images. The input image was first smoothed using a Gaussian filter (implemented via the ‘filters.gaussian' function from the scikit-image package) to reduce noise and enhance relevant structures. Subsequently, Otsu's thresholding method (used through the ‘filters.threshold_otsu' function from the scikit-image package) was applied to the smoothed image to compute an optimal threshold value. This threshold was then used to generate a binary image, effectively separating the foreground (potential tissue) from the background. The final output was a binary image with no holes enabling identification of the tissue's outer boundary. Following the identification of tissue boundaries, an optimal similarity transformation was determined and applied to the mxIF image. The reference scRNA-seq dataset was normalized to a common total count per cell corresponding to the dataset's median sequencing depth. Following normalization, the average expression profile for each cell type was computed based on the normalized counts. μk,j denotes the average raw expression of gene j of cell type k in the scRNA-seq data. The deconvolution of DBiTplus spots is achieved by computing the cell-type proportions of mxIF cells aligned to each spot. Let βk,i denote the proportion of the contribution of cell type k to spot i, and xi,j denote the total raw expression of gene j in DBiTplus spot i. To attribute gene expression to specific cell types within each DBiTplus spot, we split each spot into pure cell-type sub-spots. This is done by computing the expected cell-type-specific gene expression at each DBiTplus spot. For single-cell-level gene data analysis, to normalize gene expression values, we first calculated the median UMI counts per pure cell-type sub-spot. This median value was then used as the scale.factor in the NormalizeData function so that all sub-spots are normalized to a common library size representative of the dataset's central tendency. Linear dimension reduction was then performed with the ‘RunPCA' function, and the optimal number of principal components for further analysis was determined using a heuristic approach, visualized by an ‘elbow plot' that ranks PCA components by their variance percentages. Subsequently, the ‘FindNeighbors' function embedded spots into a k-nearest-neighbor graph structure based on Euclidean distance in PCA space, and the ‘FindClusters' function (resolution = 0.7, all other parameters at default values) applied a modularity optimization technique to cluster the sub-spots. Finally, DEGs defining each cluster were identified using the ‘FindMarkers' function for pairwise comparisons between spot groups or ‘FindAllMarkers' for differential expression for each identity (cluster or cell type) versus all other cells combined. mxIF cells were aggregated into pure cell-type sub-spots, and DBiTplus spots were similarly divided into pure cell-type sub-spots. Subsequent analyses were performed using the Seurat WNN methodology, with measurements from the pure cell-type sub-spots serving as input data. The RNA data underwent normalization using a scale factor derived from the median UMI count. Following normalization, variable features were identified, and PCA was conducted on the RNA assay. For each cell, the nearest neighbors within the dataset were determined based on a weighted combination of RNA and protein similarities. The integrated results were then utilized for visualization and clustering. We used Cell2location for the deconvolution analysis. First, a negative binomial regression model was trained to estimate reference transcriptomic profiles for all cell types profiled with scRNA-seq data (sc_subsampled_park2020.h5ad). The ‘major celltype' annotation was used to ensure consistency. Lowly expressed genes were excluded using the filtering strategy recommended by Cell2location (cell_count_cutoff = 5, cell_percentage_cutoff2 = 0.03, nonz_mean_cutoff = 1.12). The model was trained for 600 epochs and reached convergence. The following Cell2location hyperparameters were applied: N_cells_per_location = 33 (estimated from nuclei counts in H&E images using QuPath, version 0.50). Training was stopped after 50,000 iterations, with all other parameters set to default values. Correlations were summarized by: (i) the proportion of spots above thresholds (>0.9, >0.8, >0.7, >0.6), and (ii) spatial heat maps of correlation values derived from spot coordinates. We also plotted the overall correlation distribution using 0.1-wide bins. RCTD provides a principled, likelihood-based framework for inferring which single-cell transcriptional programs underlie the mRNA captured in each spatial spot. Conceptually, it treats the observed counts as a mixture of reference cell-type profiles and estimates the mixture fractions that best explain the data, while explicitly accounting for technical variability such as total UMI depth. In our workflow, the analysis was implemented in R (spacexr v2.2.1) with Seurat, SeuratDisk and zellkonverter facilitating seamless interchange between Seurat and AnnData formats. The DBiTplus dataset supplied the query layer: raw spot-level counts were extracted from the Spatial assay, two-dimensional spot coordinates were obtained through GetTissueCoordinates() and spot-specific library sizes were computed with colSums(counts). These three components were wrapped in a SpatialRNA() object, which together with a preprocessed single-cell reference served as input to create.RCTD(). Model fitting proceeded via run.RCTD() with doublet_mode = ‘full'. The resulting cell-type weight matrix was normalized so that proportions across all reference types sum to one in every spot (normalize_weights()), appended to the DBiT Seurat object with AddMetaData(), and finally exported for downstream comparison with other deconvolution methods and for integration into our spatial analyses. Cell-type deconvolution with TACCO (v0.2.2) was performed in Python. Deconvolution was carried out with tc.tl.annotate, which treats every spatial spot as a compositional mixture of the single-cell expression profiles. Specifically, we passed the DBiT AnnData object (adata) and the gene-matched reference (ref) to tc.tl.annotate, using ‘celltype' as the label to be transferred, result_key = ‘TACCO' to store the output, and reconstruction_key = ‘rec' to save reconstructed expression values. Setting multi_center = 1 let the algorithm place a single, data-driven centroid in expression space for each cell type, striking a balance between flexibility and over-fitting. After inference, ‘tc.pp.filter' was applied to mirror any gene or cell filtering decisions across reference and query, ensuring that subsequent comparisons rested on an identical feature space. Pseudotemporal analysis was performed with Monocle 3 (ref. 15) on average gene expressions of pure cell-type sub-spots obtained from proteome-informed cell-type deconvolution of the DBiTplus spots. The Seurat object was subset to include only cell types of interest. The raw count matrix was converted converted to a monocle cell dataset using the ‘new_cell_data_set' function. Pseudotime plots of specific genes were plotted using the ‘plot_genes_in_pseudotime' function. The ‘graph_test' function was used to identify DEGs and the heat map of the DEGs plotted (both k-means and hierarchical clustering) with a modified version of code from ref. Ingenuity pathway analysis (Qiagen)51 was used to explore the biological pathways implicated in the DEGs from our Seurat clusters or cell types. This replication step ensured proper sample sizes for P-value calculations. The DEGs per cluster or cell type are generated using the FindAllMarkers function or FindMarkers (for pairwise comparisons) in Seurat. The list of genes with corresponding log2fold change values, P value and adjusted P value of each gene, was exported as a CSV file and used as input for the Qiagen ingenuity pathway analysis software. The Ingenuity Knowledge Base (genes only) served as the reference set for performing core expression analysis. Positive z-scores denote predicted activation or upregulation, whereas negative z-scores indicate inhibition or downregulation. The significance (−log10P value) of each pathway enrichment is further evaluated using a right-tailed Fisher's exact test. Pathways were plotted as bar charts using Prism V10 or R (4.2.0-foss-2020b). The upstream regulator analysis identifies a list of upstream regulators that may be responsible for the observed gene expression changes in the list of DEGs in our datasets. Prism V10 (GraphPad) was used for statistical analyses and the specific tests used are indicated in the main text and figure captions. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw and processed data reported are deposited in the Gene Expression Omnibus under accession GSE308167. The resulting fastq files were aligned to the mouse reference genome (GRCm39) or human reference genome (GRCh38). mxIF imaging datasets and histological images are available at https://doi.org/10.5281/zenodo.17153112 (ref. Published data for data comparison are available online at STOmicsDB MOSTA (E13.5_E1S2.MOSTA.h5ad; https://db.cngb.org/stomics/mosta/spatial/) and Allen Mouse Brain Atlas (Developing Mouse Brain—age, E13.5; Theiler stage, TS21; https://developingmouse.brain-map.org/). Patho-DBiT reference lymph node data are available on the Gene Expression Omnibus under accession number GSE274641 (sample GSM8454083). Baysoy, A., Bai, Z., Satija, R. & Fan, R. The technological landscape and applications of single-cell multi-omics. Rodriques, S. G. et al. Slide-seq: a scalable technology for measuring genome-wide expression at high spatial resolution. Liu, Y. et al. High-spatial-resolution multi-omics sequencing via deterministic barcoding in tissue. Liu, Y. et al. High-plex protein and whole transcriptome co-mapping at cellular resolution with spatial CITE-seq. Farzad, N. et al. Spatially resolved epigenome sequencing via Tn5 transposition and deterministic DNA barcoding in tissue. & Zhuang, X. Spatial transcriptome profiling by MERFISH reveals subcellular RNA compartmentalization and cell cycle-dependent gene expression. Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Black, S. et al. CODEX multiplexed tissue imaging with DNA-conjugated antibodies. Li, Y. et al. All-optical multimodal mapping of single cell-type–specific metabolic activities via REDCAT. Yeo, Y. Y. et al. Same-slide spatial multi-omics integration reveals tumor virus-linked spatial reorganization of the tumor microenvironment. Integration of spatial and single-cell data across modalities with weakly linked features. Robust decomposition of cell type mixtures in spatial transcriptomics. Bai, Z. et al. Spatially exploring RNA biology in archival formalin-fixed paraffin-embedded tissues. Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. The single-cell transcriptional landscape of mammalian organogenesis. Chen, A. et al. Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays. Mages, S. et al. TACCO unifies annotation transfer and decomposition of cell identities for single-cell and spatial omics. Kleshchevnikov, V. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Li, J., Zhang, Y., Yang, C. & Rong, R. Discrepant mRNA and protein expression in immune cells. Zhang, D. et al. Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology. Arribas, A. J. et al. Resistance to PI3Kδ inhibitors in marginal zone lymphoma can be reverted by targeting the IL-6/PDGFRA axis. & Moreaux, J. Gene expression-based risk score in diffuse large B-cell lymphoma. Cai, C. et al. Anthrax toxin receptor 1/tumor endothelial marker 8 promotes gastric cancer progression through activation of the PI3K/AKT/mTOR signaling pathway. Ferreres, J. R. et al. PRRX1 silencing is required for metastatic outgrowth in melanoma and is an independent prognostic of reduced survival in patients. Distinct genetically determined origins of Myd88/BCL2-driven aggressive lymphoma rationalize targeted therapeutic intervention strategies. Liu, Y. et al. Palmitoylation by ZDHHC family members regulate B-cell lymphoma growth. Zhang, W. et al. Exosome complex genes mediate RNA degradation and predict survival in mantle cell lymphoma. Rivas, M. A. et al. Cohesin core complex gene dosage contributes to germinal center derived lymphoma phenotypes and outcomes. Puvvada, S. D. et al. Pharmacogenetic approaches identify ANTXR1 as a potential mediator of response to therapeutic NF-κB inhibition in diffuse large B cell lymphoma. Broséus, J. et al. Molecular characterization of Richter syndrome identifies de novo diffuse large B-cell lymphomas with poor prognosis. Evolutionary history of transformation from chronic lymphocytic leukemia to Richter syndrome. Parikh, S. A. et al. Diffuse large B-cell lymphoma (Richter syndrome) in patients with chronic lymphocytic leukaemia (CLL): a cohort study of newly diagnosed patients. The immune microenvironment shapes transcriptional and genetic heterogeneity in chronic lymphocytic leukemia. Mahmoud, A. M., Gaidano, G. & Mouhssine, S. Immunological aspects of Richter syndrome: from immune dysfunction to immunotherapy. Mouhssine, S. & Gaidano, G. Richter syndrome: from molecular pathogenesis to druggable targets. Zhang, C. et al. Prioritizing exhausted T cell marker genes highlights immune subtypes in pan-cancer. The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. Diamant et al. Harmonizome 3.0: integrated knowledge about genes and proteins from diverse multi-omics resources. & Turner, M. Regulation of B-cell differentiation by microRNAs and RNA-binding proteins. Malpeli, G. et al. MYC-related microRNAs signatures in non-Hodgkin B-cell lymphomas and their relationships with core cellular pathways. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Zhang, D. et al. ASTRO: automated spatial whole-transcriptome RNA-expression workflow. & Hu, F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. Integrated analysis of multimodal single-cell data. Tambalo, M., Mitter, R. & Wilkinson, D. G. A single cell transcriptome atlas of the developing zebrafish hindbrain. Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus-multiplexed IF images. Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus-multiplexed IF and H&E images. We thank Yale West Campus cleanroom team for assistance with microfluidic wafer fabrications and the YPTS team for FFPE tissue sectioning and staining. Computational data analysis was conducted with Yale High Performance Computing clusters. We acknowledge support received from US National Institutes of Health grants U54CA274509, U54CA268083, UH3CA257393, RF1MH128876, U54AG079759, U54AG076043, R01CA245313 and RM1MH132648 (all to R.F.) ), and from the US National Science Foundation grants DMS 2345215 (to Z.M.) These authors contributed equally: Archibald Enninful, Zhaojun Zhang. Department of Biomedical Engineering, Yale University, New Haven, CT, USA Archibald Enninful, Mingyu Yang, Zhiliang Bai, Negin Farzad, Graham Su, Alev Baysoy, Jungmin Nam, Yao Lu, Shuozhen Bao, Siyan Deng & Rong Fan Department of Statistics and Data Science, The Wharton School, University of Pennsylvania, Philadelphia, PA, USA Akoya Biosciences, Menlo Park, CA, USA Bruker Spatial Biology, St Louis, MO, USA Department of Statistics and Data Science, Yale University, New Haven, CT, USA Human and Translational Immunology Program, Yale School of Medicine, New Haven, CT, USA Yale Center for Research on Aging (Y-Age), Yale School of Medicine, New Haven, CT, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors reviewed, edited and approved the manuscript. Correspondence to Mina L. Xu, Zongming Ma or Rong Fan. were reviewed and managed by Yale University Provost's Office in accordance with the University's conflict of interest policies. has served as consultant for Treeline Biosciences, Pure Marrow and Seattle Genetics. The other authors declare no competing interests. Nature Methods thanks Qiangyuan Zhu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Rita Strack and Lei Tang, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic overview of three tested approaches for cDNA release from FF or FFPE tissue sections. b, Brightfield images of E13 mouse embryo FF sections before and after cDNA retrieval using NaOH, DMSO, or RNase H. Morphology is best preserved with the RNase H condition. c, Assessing the quality of CODEX imaging on FF mouse spleen after DBiTplus workflow (n = 1). The RNase H-treated sample shows positive staining makers including CD45R (B cell lineage) and CD169 (macrophages) similar to control staining. d, Schematic overview for RNase H cDNA retrieval protocol. The three tubes are pooled for cDNA purification, PCR amplification, and library preparation. e, Image registration pipeline between DBiTplus and mxIF datasets. a Violin plot showing the number of detected genes/UMI counts per spatial spot from from two controls and two replicates FF E13 mouse embryo samples. Box plots show the median (center line), the first and third quartiles (box limits) and 1.5x interquartile range (whiskers). b Brightfield image of E13 FF mouse embryo. Right: spatial mRNA and UMI count maps demonstrates average of 1,029 genes and 1,601 UMIs detected per 25 μm spot. c Pairwise correlation plots between replicates (DBiTplus workflow) and control (standard DBiT-seq workflow) showing the reproducibility of the DBiTplus workflow from a two-sided Pearson correlation test, with a Pearson correlation coefficient of 0.87 and 0.72 respectively. Pearson correlation between the two replicates is 0.84. For Replicate 2, 0.5% Triton X-100 was added to the RNase H to further improve cDNA release efficiency. d Spatial UMAP reveals 8 transcriptionally distinct clusters that align with anatomical features of the E13 mouse embryo. e Heat map of the top 5 differentially expressed genes per spatial cluster. f Validation of DBiTplus gene expression using independent reference datasets. In-situ Hybridization (ISH) staining image from the Allen Mouse Brain Atlas (Developing Mouse Brain - Age: E13.5, Theiler Stage: TS21) and gene expression from Mouse Organogenesis Spatiotemporal Transcriptomic Atlas (MOSTA) dataset (E13.5_E1S2.MOSTA.h5ad). Scale bar 1 mm for Allen Mouse Brain Atlas images. g Gene Ontology (GO) enrichment analysis showing the biological process associated with E13 mouse embryos (cluster 0 from the standard DBiT-seq workflow and cluster 2 from DBiTplus workflow), both of which are associated with the embryonic diencephalon. a UMAP showing 10 distinct clusters of DBiTplus spots/spots. c Heat map showing the top 5 differentially expressed genes for each spatial cluster. d Spatial distribution of clusters 0, 1, and 2 alongside their principal defining genes. In-situ Hybridization (ISH) staining image from the Allen Mouse Brain Atlas (Developing Mouse Brain - Age: E13.5, Theiler Stage: TS21) and gene expression from STOmics Mouse Organogenesis Spatiotemporal Transcriptomic Atlas (MOSTA) dataset (E13.5_E1S1.MOSTA.h5ad). Scale bar 1.5 mm for Allen Mouse Brain Atlas images. b Spatial distribution of annotated cell types across the entire lymph node tissue. c Left: Spatial expression patterns of key lineage markers - CD14 (myeloid lineage), CD20 (B cells), and CD3ε (T cells) derived from the CODEX dataset. Right: spatial plots showing subtypes corresponding to each major lineage, including T cell subsets, B cell subsets and myeloid populations. d Spatial deconvolution results from RCTD. e Performance metrics for CODEX-informed versus uninformed deconvolution. b Heat map of differentially expressed genes across annotated cell types. c Violin plots of representative (canonical) marker genes for various B cell subtypes. d Volcano plots showing differentially expressed genes in naive (left) and activated (right) B cells compared to all other cells. Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction) e Pathway analysis comparing enriched pathways between naïve B cells and activated B cells. f Ingenuity Pathway Analysis (IPA) graphical summary showing predicated canonical pathways, upstream regulators and transcriptional and translational programs in activated B cells. a Venn diagram comparing 35 markers in the CODEX panel with corresponding gene expression data from DBiTplus. CD68 is the only protein marker without a corresponding gene transcript detected in DBiTplus. b Bar plot showing normalized expression of matched genes between DBiTplus lymph node (blue) and a Patho-DBiT reference lymph node (green). c Scatter plot showing correlation of log-transformed gene expression between DBiTplus (n = 1) and Patho-DBiT reference lymph node sample (n = 1). Strong concordance observed (Pearson R = 0.89, p < 2.2 × 10−16). Weak but positive association (R = 0.23, p = 0.18). e Multiplexed CODEX image showing CD68+ macrophages within the lymph node architecture (blue: DAPI, green: CD68). g Comparative visualization of CD14 and CD163 expression in both CODEX (top row, protein) and DBiTplus (bottom row, RNA), showing similar spatial enrichment patterns. h Spatial distribution of macrophages as identified from CODEX-guided DBiTplus cell type deconvolution. a Heat map of top differentially expressed genes across annotated cell types from the spatial transcriptomics dataset. Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction) d, IPA-inferred upstream regulator network predicting TF-driven control over gene expression changes in tumor B cells. f Causal network diagram centered on MYC, highlighting upstream regulation from SOX4 through the AKT family. g Activation z-scores vs. –log10(p-values) from causal network analysis showing predicted activators (red) and inhibitors (blue). P-values calculated using the right-tailed FET (Fisher's Exact Test). h Expanded regulatory network involving of SYK, a key upstream regulator of multiple genes in lymphoma transformation. i Monocle3 UMAP embedding showing four key genes. a Representative high-resolution images of DLBCL (top) and CLL (bottom) regions showing differences in cell size (n = 1). b Schematic of the DBiTplus workflow using CellScape on an FFPE tissue section. c Protein marker panel used in CellScape, including immune, stromal, and segmentation markers. d Stacked barplot showing cell type composition in CLL and DLBCL regions. Computed using a two-tailed Mann-Whitney test (**** represents p < 0.0001, nCLL = 5,525 and nDLBCL = 4,924). f Violin plots of the distance of T cell subtypes from tumor cells in DLBCL versus CLL regions. Violin plots show the median (center line), the first and third quartiles (box limits) and 1.5x interquartile range (whiskers). h Histogram of cell size distributions comparing large B cells (DLBCL) and small B cells (CLL). Histogram bin width of 0.25 i Spatial maps showing localization of large B cells (red) and small B cells (blue) within the tissue section (left), and CLL cells only (right). j, 2D density scatter plot showing CD20 and Ki67 expression as a function of cell size in large B cells (left) versus other B cells (right). k, Density distribution plots of marker expression (CD23, CD5, LEF1) in small (top) and large B cells (bottom). l Spatial maps showing localization of CLL-like cells. a Spatial expression gradients for representative protein markers across the tissue section reveal distinct spatial localization (within CLL or DLBCL regions) patterns of B cells (CD19), macrophages (CD68), immune checkpoints (CD279/PD-1, FOXP3), proliferating cells (Ki67), and cytotoxic activity (GranzymeB). b UMAP plot showing unsupervised clustering of annotated cell type across the lymphoma tissue. c Heat map of top differentially expressed genes across all cell types. d Volcano plot showing DEGs between large B cells (DLBCL) and small B cells (CLL), with key upregulated genes in each population highlighted. Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction e Heat map of top 30 DEGs distinguishing large B cells from small B cells. f Ingenuity Pathway enrichment analysis of large B cells. a Distribution of microRNAs and corresponding UMI counts per spatial pixel. Dashed lines indicate the average number of microRNAs or UMI counts. b Spatial clustering of clusters 0 and 1 whose spatial distributions mirrored CLL and DLBCL regions from histological staining of Richter's transformation sample. c Differentially expressed microRNAs between clusters 0 and 1. Differential gene expression computed from two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction) d Spatial mapping of select microRNAs implicated in lymphoid malignancies. e–g, Monocle3 UMAP visualization of large B cells and small B cells e showing distinct partitions in transcriptomic space f Monocle3 UMAP visualization of small B cells g Monocle3 pseudotime UMAP highlighting a dynamic trajectory. h Pseudotime heat map expression dynamics of genes associated with transformation, including anti-apoptotic regulators (BCL2, BIRC3), proliferation (MKI67), and key signaling nodes (ATM, LEF1, TCL1A). i Heat map of pathway-associated gene expression (z-score) across pseudotime reveals coordinated activation of DNA damage response, chromatin modifiers, and NF-κB and MAPK-ERK signaling, consistent with progressive transformation. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-025-01053-3'>DNA damage in macrophages drives immune autoreactivity via nuclear antigen presentation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 10:18:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Aging and DNA damage increase the risk of chronic inflammation and autoimmunity, yet the molecular underpinnings remain unclear. In this study, we uncover a DNA damage-driven mechanism in macrophages that triggers immune autoreactivity. Here, using Er1Lyz2/− mice with a macrophage-specific DNA repair defect in ERCC1−XPF, we demonstrate that monocyte-derived macrophages accumulate DNA damage, activate the immune system, drive polyclonal T cell responses and generate antinuclear autoantibodies. Proteomic and immunopeptidomic analyses reveal a distinct major histocompatibility complex class II (MHC-II) antigen repertoire enriched in nuclear and ribosomal peptides, relying on autophagy for nuclear cargo delivery to MHC-II. Aged macrophages exhibit a similar lysosomal cargo profile, linking autophagy-driven nuclear antigen presentation to immune activation. Notably, inhibiting autophagy in Er1Lyz2/− mice suppresses autoimmune features, pinpointing autophagy-facilitated nuclear antigen processing as a central driver of age-related autoimmunity. These findings establish DNA damage-induced autophagy in macrophages as a pivotal mechanism linking aging to autoimmunity, unveiling potential therapeutic targets to mitigate age-related immune dysregulation. Aging is characterized by the accumulation of DNA damage, a hallmark that fuels chronic inflammation, accelerates immunosenescence and drives a spectrum of age-related pathologies1,2,3,4,5,6. Persistent DNA lesions and impaired repair mechanisms activate inflammatory pathways, including type I interferon signaling through cytoplasmic nucleic acid release and the activation of the senescence-associated secretory phenotype (SASP)1,7,8,9,10,11. Recent studies reveal that genotoxic stress stimulates adaptive immunity by generating peptides presented on human leukocyte antigen class I (HLA-I) molecules, highlighting the intricate interplay between DNA damage and immune responses12. Macrophages have a critical role in the immune system as sentinels, antigen-presenting cells (APCs) and modulators of immunity13,14. Aging impairs macrophage or monocyte function through reduced clearance of apoptotic cells, debris and pathogens15,16, increased MHC expression17, skewed polarization18 and chronic cytokine secretion19. In autoimmune responses, macrophages can contribute to the accumulation and presentation of immunogenic antigens. Identifying these peptides is essential for the development of targeted therapies, such as peptide-based vaccines or immunomodulatory agents, to selectively suppress autoimmune responses without broadly suppressing the immune system20,21. Autophagy, a fundamental cellular degradation and recycling process, maintains homeostasis by clearing damaged organelles, misfolded proteins and intracellular debris, particularly in response to genotoxic stress22,23. Additionally, autophagy intersects with antigen presentation pathways, playing a pivotal role in MHC-II-mediated antigen presentation24,25. Dysregulation of autophagy-mediated antigen processing is implicated in the pathogenesis of autoimmunity, underscoring its therapeutic potential. Despite its importance, the precise composition of autophagy-derived antigens and their impact on immune activation remain largely undefined26,27. In the present study, we reveal how DNA damage in macrophages drives immune activation using Er1Lyz2/− mice, a model with myeloid-specific DNA repair defects. ERCC1−XPF is a heterodimeric, structure-specific endonuclease complex required for lesion excision in nucleotide excision repair (NER)28,29. It is also known to participate in DNA double-strand break (DSB) repair, DNA interstrand-crosslink (ICL) repair, base excision repair (BER), telomere maintenance and for the processing of various alternative DNA structures that hinder replication and transcription events30,31,32,33. Er1Lyz2/− mice exhibit autoimmune symptoms, driven by MHC-II presentation of nuclear and ribosomal peptides. Aged macrophages show similar increase in antigen presentation, linking these changes to the DNA damage response (DDR). Notably, inhibiting autophagy reduces immune activation, highlighting autophagy as a key driver of nuclear antigen processing and a promising therapeutic target for age-related autoimmune diseases. To study how persistent DNA damage in monocyte-derived macrophages affects the immune system response, we studied mice with an engineered Ercc1 defect in the myeloid cell lineage (Lyz2-cre+;Ercc1fl/−, hereafter referred to as Er1Lyz2/− mice). By 8 months of age, Er1Lyz2/− mice develop a systemic inflammatory response and an increase in tissue infiltrates compared to their wild-type counterparts—that is, Lyz2-cre−;Ercc1fl/+ mice34. Hematoxylin and eosin (H&E) staining revealed chronic inflammation in multiple intraparenchymal areas of the kidneys isolated from Er1Lyz2/− mice, including around the pelvicalyceal system. The inflammatory infiltrates consisted primarily of small lymphocytes, large activated lymphocytes and numerous plasma cells, indicative of an active immune response (Fig. Immunofluorescence analysis of kidney cryosections showed an increased deposition of complement protein C3 and immune complexes according to IgM and IgA staining (Extended Data Fig. These proteins are commonly localized in the kidneys of patients with autoimmune glomerulonephritis and mouse models. Despite this, skin preparations of Er1Lyz2/− mice displayed no inflammatory foci or other alterations according to H&E analysis, and there were no apparent changes in the thickness of paws or ankles (Extended Data Fig. Notably, periodic acid−Schiff (PAS) staining in kidneys and an albumin ELISA argued against the presence of tubular atrophy or fibrosis of the kidney parenchyma or proteinuria in Er1Lyz2/− mice. PAS stain, however, additionally revealed the expanse of the mesangium of 30−40% of the glomeruli in Er1Lyz2/− kidneys, indicating glomerular damage (Extended Data Fig. Supporting this, an indirect immunofluorescence assay using mouse embryonic fibroblasts (MEFs) as a substrate demonstrated the presence of autoantibodies in the sera of 8-month-old Er1Lyz2/− mice. The increase in autoantibody production was similar to that observed in naturally aged 24-month-old wild-type mice (Extended Data Fig. Antibody staining patterns included homogenous or speckled nuclear and cytoplasmic patterns (Extended Data Fig. Antinuclear antibody enzyme-linked immunoabsorbent assay (ANA-ELISA) confirmed the presence of antinuclear autoantibodies in the sera of Er1Lyz2/− mice (Fig. Adoptive transfer of sera derived from 8-month-old wild-type or Er1Lyz2/− mice to young wild-type recipients and analysis of kidney cryosections resulted in an increase in the localization of IgM immune complexes in the glomeruli of mice receiving Er1Lyz2/− sera when they were compared to the recipients of wild-type sera (Extended Data Fig. Histological analysis of wild-type spleens showed normal morphological features, with a normal ratio of white to red pulp. Er1Lyz2/− spleens, however, showed marked hyperplasia of the white pulp, with foci of plasmacytic cells at the white−red pulp border. Spleen weights complemented these results, with Er1Lyz2/− spleens weighing more than their wild-type controls, pointing to an increase in cellularity (Extended Data Fig. Staining of splenocytes for CD19 (B cell maker) and CD138 (plasma cell marker) revealed increased plasma cell percentages and numbers in the secondary lymphoid organs and the bone marrow and further supported the increased antibody production in 8-month-old Er1Lyz2/− mice (Fig. This aligns with findings in aged C57BL/6 mice, which spontaneously developed antinuclear autoantibodies driven by MHC-II antigen presentation to CD4+ T cells, accompanied by an immune complex deposition in the kidneys of old mice35. Flow cytometry analysis of splenocytes from Er1Lyz2/− and wild-type mice showed increased numbers of additional cell populations that regulate autoimmunity: CD11b+ myeloid cells, CD4+ cells, CD8+ cells, NK1.1+CD3− (natural killer) cells and regulatory CD4+ T cells in Er1Lyz2/− spleens but not granulocyte−monocyte progenitors in Er1Lyz2/− bone marrow (Fig. Splenic macrophages (CD11b+F4/80+SinglecF−) and CD11b+Ly6G− cells of monocytic origin from Er1Lyz2/− mice exhibited higher percentages of cells expressing MHC-II antigen presentation and CD86 co-stimulatory molecules, respectively, along with increased protein expression levels of both markers (Fig. Accordingly, flow cytometry analysis of cells from spleens and lymph nodes stained for the CD4 glycoprotein, the CD44 adhesion molecule responsible for T cell migration and the L-selectin/CD62L naive T cell marker revealed a gradual increase in the percentages and numbers of memory CD4+ T cells (CD44hiCD62Llo) in 8−10-month-old Er1Lyz2/−mice36,37 (Fig. Conversely, a decrease in the percentage of naive CD4+ T cells (CD44loCD62Lhi) was observed, arguing for a polyclonal CD4+ T cell expansion in Er1Lyz2/− mice (Extended Data Fig. This resembled the T cell expansion observed in 24-month-old wild-type mice (Fig. To assess T cell function, CD4+ T cells from 10-month-old mice were stimulated with phorbol 12-myristate 13-acetate (PMA) and ionomycin for 4−6 h, which activates protein kinase C and induces calcium signaling, important for T cell activation and cytokine production38. Stimulation of CD4+ T cells from 10-month-old Er1Lyz2/− mice with PMA and ionomycin led to increased interferon-γ (IFNγ) secretion, indicating a more robust T helper 1 (TH1) immune response (Fig. Correspondingly, a higher percentage of CD4+ T cells expressing T-bet, the master regulator of TH1 differentiation, was observed in the lymph nodes of these mice compared to controls (Fig. Along with their activated profile and increased numbers, FOXP3− T cells from Er1Lyz2/− mice increased expression of the inhibitory receptor programmed cell death protein 1 (PD-1) (Fig. 1l) and the proliferation marker Ki-67 (Extended Data Fig. 3I), indicating prolonged exposure to activation signals. Interestingly, adoptive transfer of CD4+ T cells isolated from 10-month-old wild-type and Er1Lyz2/− mice to NOD scid gamma (NSG) hosts led to a higher frequency of T cell infiltration in the kidneys of mice receiving Er1Lyz2/− T cells, as suggested by the inflammatory foci observed during H&E analysis and CD4+ T cell numbers measured by flow cytometry analysis of kidney Percoll fractions (Extended Data Fig. a, H&E stain of kidneys isolated from 8-month-old wild-type (WT) and Er1Lyz2/− mice (n = 7−8, P = 0.0040). The magnification in each image is indicated, and black arrows point to large activated lymphocytes and plasma cells (inflammation score: 0: no/focal, 1: moderate, 2: intense). b, Antinuclear autoantibody (ANA) detection in the sera of 8−10-month-old WT and Er1Lyz2/− mice by ELISA (n = 8−10, P = 0.0449). Autoantibody patterns of 8-month-old WT and Er1Lyz2/− and 24-month-old WT (aged) mice are shown in Extended Data Fig. All bar plots show the total number of CD19+CD138+ plasma cells (n = 7, P = 0.0201). Myeloid cells (n = 4−5, P= 0.04) (c), CD4+ cells (n = 9−10, P = 0.0469) (d), regulatory T (Treg) cells (n = 3, P = 0.0483) (e) and natural killer (f) and CD8+ T (g) cells (n = 4, P = 0.0004 and P = 0.0395). Representative scatter plots and gating strategies are shown in Extended Data Fig. h,Flow cytometry analysis of activated macrophages in spleens of 8-month-old mice using CD11b and Ly6G for gating in cells of monocytic origin (gating strategy in Extended Data Fig. Representative scatter plots (top) as well as bar plots (bottom) depict the percentage of MHC-II+CD86+ macrophages. MFIs are shown in Extended Data Fig. i, Graphs showing the percentage of memory CD4+ T cells in the spleens of 2-month-old, 6-month-old and 10-month-old WT and Er1Lyz2/− mice as well as 24-month-old (aged) WT mice, quantified by flow cytometry analysis. Representative scatter plots for the activation status of T cells isolated from 10-month-old mice are included in the same figure, and representative plots for cells isolated from 2-month-old and 24-month-old mice are shown in Extended Data Fig. j, ELISA analysis for the detection of IFNγ in the supernatants of PMA and ionomycin-stimulated CD4+ T cells (n = 3, P = 0.0124). k, Bar plot displaying the expression levels of T-bet (MFI) in CD4+ T cells in 8-month-old WT and Er1Lyz2/− spleens. A representative histogram plot can be found in Extended Data Fig. l, Histogram showing the MFI of PD-1 inhibitory marker in the FOXP3−CD44+ population of WT and Er1Lyz2/− CD4+ T cells in the spleen, at 10 months of age (n = 4, P = 0.0063). m, Adoptive transfer of WT or Er1Lyz2/− BMDMs in young WT mice for a time period of 8−12 weeks, every 2 weeks (experimental scheme shown in top legt). The percentage of activated (CD44hiCD62Llo) CD4+ cells in the inguinal lymph nodes (LN) and the levels of antinuclear autoantibodies in the sera of WT mice receiving WT or Er1Lyz2/− BMDMs are plotted (bottom left and right, respectively). Plasma cells (CD19+CD138+) in the spleens of WT mice receiving WT or Er1Lyz2/− BMDMs are shown in Extended Data Fig. Representative scatter plots are shown in Extended Data Fig. To determine whether Er1Lyz2/− macrophages could trigger the immune activation observed in Er1Lyz2/− mice, we performed a bone marrow-derived macrophage (BMDM) adoptive transfer experiment. In this experiment, wild-type and Er1Lyz2/− BMDMs, differentiated in vitro, were intravenously administered into young wild-type mice every 2 weeks for 8−12 weeks. Notably, wild-type mice receiving Er1Lyz2/− macrophages showed a higher percentage of memory CD4+ T cells (CD44hiCD62Llo) in their lymph nodes and plasma cells (CD19+CD138+) in their spleens, accompanied by an increase in sera autoantibodies compared to mice receiving wild-type macrophages (Fig. These results indicate that Ercc1 ablation in BMDMs induces spontaneous immune activation and autoantibody production in mice, further supporting the role of DNA damage in promoting autoimmune and inflammatory responses. DSBs caused by low-dose chemotherapeutic agents or irradiation upregulate MHC-II via the ATM−NF-κB−IRF1 pathway, a mechanism described in cancer cells and B cells39,40. Similarly, genotoxic stress, such as X-rays or chemotherapeutic drugs, has been predicted to stimulate HLA-I-mediated T cell activation12. DNA-damaging agents such as doxorubicin further link genotoxicity to immunogenicity by inducing senescence and triggering CD8+ T cell responses to neoantigens41. Co-localized foci of DDR markers γH2AX and 53BP1 in Er1Lyz2/− macrophages indicated the accumulation of DNA DSBs, which were partly diminishedby reducing oxidative stress with the use of antioxidants N-acetylcysteine and Mito-TEMPO. This was in line with the higher mean fluorescence intensity (MFI) when we tested for the detection of 8-oxoguanine (8-oxoG) oxidative lesions in Er1Lyz2/− BMDMs (Extended Data Fig. These DNA breaks can activate an ATM-mediated DDR, shown by the increased protein levels of the phosphorylated form of ATM (Extended Data Fig. To test whether Er1Lyz2/− macrophages harboring DNA damage could directly drive CD4+ T cell responses in an MHC-II-dependent manner, we treated BMDMs with the ATM inhibitor KU-55933 (ref. Er1Lyz2/− BMDMs showed elevated MHC-II levels, a response abrogated by ATM inhibition, demonstrating that ATM activation is critical for DNA damage-driven antigen presentation (Fig. Likewise, treatment of BMDMs with ATR and DNA-PK inhibitors, also key mediators of the DDR, led to a similar decrease in MHC-II levels (Fig. DNA damage has been linked to the development of systemic lupus erythematosus (SLE) as a risk factor. For this reason, we considered the role of the DDR in MHC-II antigen presentation in monocyte-origin cells derived from 11-month-old New Zealand Blank (NZB)/New Zealand White (NZW) F1 lupus-prone mice. We performed flow cytometry analysis in splenocytes from lupus-prone and control mice and found increased levels of γ-Η2Α.Χ DNA damage marker in CD11b+Ly6G− monocyte-origin cells (Extended Data Fig. In addition, the increased antigen presentation capacity of lupus-prone compared to control monocyte-origin cells dropped upon ATM kinase inhibition (Extended Data Fig. DNA damage thus renders monocytes and macrophages able to initiate innate immune responses in a lupus mouse model, based on immunogenic MHC-II peptide generation. Accordingly, treatment of wild-type BMDMs with etoposide and camptothecin, potent genotoxins that induce DSBs and single-strand breaks (SSBs) through inhibition of topoisomerase II and I, respectively43, activated the DDR. This response, observed 24 h after treatment, resulted in an increase in MHC-II protein levels (Fig. By contrast, stress responses such as tunicamycin-induced endoplasmic reticulum (ER) stress or starvation did not significantly elevate MHC-II levels (Extended Data Fig. In addition to MHC-II epitope binding to the T cell receptor (TCR), effective T cell activation relies on the cytokine and chemokine milieu provided by APCs44,45,46. Quantitative PCR (qPCR) analysis revealed upregulation of T cell chemoattractant genes in Er1Lyz2/− macrophages (Extended Data Fig. Building upon findings that DNA damage in brain-resident macrophages induces type I interferon production ex vivo9, we assessed interferon-β (IFNβ) levels in BMDM culture supernatants. Protein concentration and western blot analysis confirmed increased levels of IFNβ in Er1Lyz2/− BMDM supernatants. Additionally, western blotting of BMDM lysates showed increased phosphorylation of STAT1 and upregulation of IRF5, indicating the activation of type I interferon signaling pathways (Extended Data Fig. MHC-II-mediated antigen presentation is crucial for determining CD4+ T cell reactivity, playing a key role in both peripheral tolerance maintenance and breakdown47,48. To investigate whether Er1Lyz2/− BMDMs could directly activate antigen-experienced CD4+ T cells, we performed a BMDM−T cell co-culture. wild-type or Er1Lyz2/− BMDMs were co-cultured with antigen-experienced CD4+ T cells isolated from 10-month-old Er1Lyz2/− mice, which were more activated in vivo but maintained normative ERCC1 expression levels (Extended Data Fig. Flow cytometry analysis showed an increase in the percentage of CD4+ T cells expressing CD69, an early activation marker49, when they were co-cultured with Er1Lyz2/− BMDMs compared to wild-type controls (Fig. Consistently, IFNγ ELISpot assays, used to measure the effector function of CD4+ T cells in the co-culture, revealed higher IFNγ secretion in the span of 48 h. Although blocking MHC-II or IFNβ levels both led to a reduction in the number of IFNγ spots, MHC-II led to a substantial reduction in the spot numbers, whereas IFNβ blockade led to a smaller but consistent decrease (Fig. These results confirm the enhanced capacity of Er1Lyz2/− macrophages to activate T cells, linking DNA damage in macrophages to an activated adaptive immune response and implicated antigen presentation and type I interferon in this process. Considering the activation status of macrophages, we hypothesized that MHC-II-mediated antigen presentation would be critical for sustaining CD4+ T cell activation in Er1Lyz2/− mice. To test this, we blocked the MHC-II−TCR interaction in vivo by intraperitoneally administering an anti-MHC-II or an isotype control antibody to 6-month-old wild-type and Er1Lyz2/− mice, which showed a similar percentage of activated CD44hiCD62Llo T cells at this age (Figs. Five weeks after treatment, at the point when Er1Lyz2/− untreated mice showed an activated immune system (Fig. 1−3), there was a reduction in MHC-II surface levels in CD11b+ myeloid and in CD4+ T cell activation percentages in Er1Lyz2/− mice treated with anti-MHC-II compared to the isotype control-treated group. This was evident by the CD44hiCD62Llo population decrease and reduced IFNγ secretion after PMA and ionomycin stimulation. By contrast, no changes in T cell activation were observed in wild-type mice, regardless of anti-MHC-II or isotype control treatment (Fig. 2g,h and Extended Data Fig. Moreover, we reported a drop in the C3 complement protein deposition in the glomeruli of anti-MHC-II-treated Er1Lyz2/− mice when they were compared to their isotype controls (Extended Data Fig. To further support our data, we next performed a depletion of CD4+ T cells by administration of a neutralizing anti-CD4 antibody to 6-month-old Er1Lyz2/− mice for 5 weeks. This led to a reduction in the inflammatory foci in the kidneys and a decrease in plasma cell numbers and percentages in spleens derived from anti-CD4-treated mice compared to the ones derived from isotype control-treated mice (Fig. 2i,j and Extended Data Fig. a, Flow cytometry analysis of WT and Er1Lyz2/− BMDMs and Er1Lyz2/− BMDMs treated or not with an ATM inhibitor (ATMi), stained with the antigen presentation protein MHC-II. The histograms show the surface MHC-II expression levels of BMDMs (MFI). Isotype controls are shown in gray. Gating strategy is indicated in Extended Data Fig. b, Western blotting of MHC-II protein levels in whole-cell extracts from WT and Er1Lyz2/− DMSO-treated controls (Ctrl) or ATMi-treated or ATR kinase inhibitor (ATRi)-treated BMDMs. NEL, normalized expression levels. c, Flow cytometry analysis of the MHC-II expression levels of WT control and WT etoposide (ETO)-treated BMDMs. An overlay of representative histograms is shown, and MFIs are plotted. An isotype control is shown in gray (n = 3, P = 0.0189). d,e, Co-culture of WT or Er1Lyz2/− BMDMs and 10-month-old Er1Lyz2/− CD4+ T cells in a 1:1 or 1:4 ratio (n = 5, P = 0.0316). The rectangle gate and the corresponding bar plots mark the percentage of activated CD4+ T cells in culture (CD69+CD4+ cells). ELISpot assay measuring the secretion of T-cell-derived IFNγ. Representative images and corresponding bars in the plot show the number of spots per 2 × 105 cells plated. Er1Lyz2/− BMDMs were untreated or treated with anti-IFNβ (aIFNβ) or anti-MHC-II (aMHC-II) neutralizing antibodies (n = 4−5, P = 0.0006, P = 0.0354 and P < 0.0001). f, The experimental scheme indicates that mice were treated with an aMHC-II or an isotype control antibody for a total of 6 weeks, receiving intraperitoneal injections once per week. g, The graph shows the percentage of activated CD4+ T cells (CD44hiCD62Llo) in the spleens of WT and Er1Lyz2/− mice treated with an aMHC-II or an isotype control antibody (n = 5, P = 0.0213 and P = 0.0265). Representative plots are shown in Extended Data Fig. The plot depicts the secreted IFNγ protein levels in the PMA and ionomycin-stimulated CD4+ T cells isolated from Er1Lyz2/− mice receiving either an MHC-II-blocking or an isotype control antibody (n = 4−5, P = 0.0447). i, Experimental scheme for a 6-week CD4+ T cell depletion in Er1Lyz2/− mice. Mice received injections once per week intravenously. j, CD4+ T cell depletion experiment. Representative scatter plots and graphs of the plasma cell percentages and numbers in spleens of Er1Lyz2/− mice receiving isotype an isotype control or an anti-CD4 (aCD4) antibody (n = 5−6, P = 0.0020 and P = 0.0357 for percentages and P = 0.0002 and P = 0.0007 for counts). The MHC-II-mediated generation of CD4+ T cells in Er1Lyz2/− mice prompted us to examine if there are DNA damage-related epitopes driving this phenotype. We characterized the immunopeptidome of wild-type and Er1Lyz2/− BMDMs as well as wild-type macrophages treated with lipopolysaccharide (LPS), a well-known immunostimulant50. LPS induces higher MHC-II protein expression while maintaining a similar DDR in wild-type and LPS-treated BMDMs (Extended Data Fig. After MHC-II peptide immunoprecipitation (Extended Data Fig. 7c) and peptide purification, we performed liquid chromatography−tandem mass spectrometry (LC−MS/MS) (Fig. 3a), identifying 1,237 peptides from wild-type mice, 1,290 peptides from Er1Lyz2/− mice, and 999 peptides from LPS-treated wild-type BMDMs. Of these, 798, 823 and 661 peptides were predicted to be potential MHC-II binders for each condition, respectively. a, Experimental outline for the preparation and mass spectrometry analysis of MHC-II-bound peptides from WT and Er1Lyz2/− untreated and LPS-treated BMDMs. b, Histogram displaying the range of the peptide amino acid (aa) length. All peptides from all samples (WT, Er1Lyz2/− and LPS-treated WT) were used for this graph. c,Gibbs clustering depicting the predominant MHC-II core amino acids. The peptides that were classified as low or high binders of MHC-II from all samples were included for this analysis. NetMHCIIpan (version 4.3; H-2-IAb and H-2-IAq alleles) was used for the identification of peptides with any binding affinity. d, Volcano plot of differentially presented peptides in WT (downregulated, blue) and Er1Lyz2/− (upregulated, red) cells. log2(fold change) of −6 or +6 represents peptides uniquely identified in the WT or Er1Lyz2/− MHC ligandome. Statistical significance was set at P ≤ 0.05 (ANOVA analysis) (horizontal black dashed line) and peptide enrichment at log2(fold change) ≥ 0.3 (upregulated in Er1Lyz2/−) or log2(fold change) ≤ −0.3 (downregulated in Er1Lyz2/−) (vertical black dashed line). Nuclear and ribosomal as well as uniquely identified proteins are labeled with their corresponding gene symbol. e, Bar charts showing the intracellular origin of peptide antigens significantly enriched in the WT, Er1Lyz2/− and LPS-treated WT samples. WT overrepresented peptides were compared to Er1Lyz2/− ones, whereas Er1Lyz2/− and LPS-treated WT enriched peptide sequences were both compared to WT controls. Bold numbers correspond to the percentages of peptides derived from nuclear proteins. f, Venn diagram for the comparison of Er1Lyz2/− overrepresented peptides with the previously characterized peptides in the IEDB. g, Bubble plot of the Gene Ontology (GO) term enrichment analysis (Cellular Component, Mann−Whitney U-test) of significantly overrepresented Er1Lyz2/− peptides, when compared to WT controls (P ≤ 0.05 and log2(fold change) ≥ 0.3). The x axis indicates the fold enrichment derived from pathway analysis. h, IFNγ ELISpot analysis of splenocytes isolated from either 8-month-old WT or Er1Lyz2/− mice and pulsed with peptides derived from the indicated source proteins in the presence of IL-2. Representative images are shown in this figure or in Extended Data Fig. Er1Lyz2/− unpulsed, IL-2-exposed splenocytes were used as a negative control (red dotted line) (n = 4; exact P value is provided in the Source Data file). Most peptides were 14–17 amino acids in length (Fig. 3b), aligning with the most common peptide length presented by MHC-II proteins. Moreover, peptide clustering using GibbsCluster confirmed the presence of the typical MHC-II binding motifs (Fig. Notably, more presented peptides were enriched or uniquely identified in Er1Lyz2/− BMDMs compared to untreated and LPS-treated wild-type cells, indicating that DNA damage alters the antigenic landscape of macrophages (Fig. These overrepresented peptides predominantly originated from subcellular compartments including the ER (calnexin), ribosomal proteins (RPL30 and RPS19) and, notably, the nucleus (such as histone H1, NOLC1, HNRNPL, EXOSC4 and NME2). By contrast, peptides from wild-type cells were mostly derived from plasma membrane and cytosolic proteins, whereas peptides presented by LPS-treated wild-type cells were from extracellular proteins (MMP14 and FN1) and plasma membrane proteins (ANPEP and MARCKSL1) (Fig. Comparison of the Er1Lyz2/− enriched peptides with Immune Epitope Database (IEDB) peptides identified 72 epitopes arising under genotoxic stress, including those from histone H1, NOLC1, HNRNPL and NME2 proteins (Fig. Gene Ontology (Cellular Component) enrichment analysis of the significantly overrepresented Er1Lyz2/− peptides further highlighted an increased abundance of epitopes associated with the nucleus, euchromatin, perinuclear space and ribonucleoprotein complexes (Fig. Conversely, pathway analysis in the overrepresented LPS-treated wild-type antigens confirmed their distinct origin (Extended Data Fig. To assess the predicted immunogenicity of the Er1Lyz2/− overrepresented peptides, we synthesized peptides, isolated splenocytes from 8-month-old wild-type or Er1Lyz2/− mice and performed IFNγ ELISpot assays to restimulate CD4+ T cells. We selected peptides that were uniquely identified or of nuclear origin, as nuclear antigens are crucial for the development of autoimmune responses51. We found that peptides of nuclear origin (H1 and HNRPL) and the ribosomal and nuclear peptide RPL30 elicited a more robust IFNγ T cell response in Er1Lyz2/− splenocytes (Fig. To explore how genotoxic stress shapes the antigen presentation profile in Er1Lyz2/− cells, we focused on the mechanisms that enable self-peptides to be presented by MHC-II molecules. These peptides are typically delivered to MHC-II via two pathways: the endocytosis/phagocytosis pathway, where external materials are engulfed by APCs, and the autophagy-mediated pathway. Given that DNA damage activates autophagy in macrophages34, we examined its role in enhancing immunogenicity. To assess the contribution of autophagy to MHC-II-dependent T cell responses, we inhibited autophagosome formation in Er1Lyz2/− BMDMs by treating them with 3-methyladenine (3-MA)54. When treated macrophages were co-cultured with CD4+ T cells from 8-month-old Er1Lyz2/− mice, we observed a reduction in the number of spots in ELISpot assays (Fig. This provides direct evidence that autophagy has a crucial role in MHC-II peptide presentation in DNA-damaged macrophages. a, IFNγ ELISpot analysis of recall assay: Er1Lyz2/− CD4+ T cells isolated from 8-month-old mice were co-cultured with WT, Er1Lyz2/− or Er1Lyz2/− 3-MA-treated BMDMs. Representative images of the wells are shown, and the number of spots per reaction is plotted (n = 4, P = 0.033 and P = 0.0379). b−d, Autophagosome content identification in the U2-OS cell line. b, Experimental scheme of the APEX2−LC3B-based proteomics approach. c, Volcano plot of proteins enriched in the autophagosomes of DMSO-treated (downregulated, blue) or ETO-treated (upregulated, red) U2-OS cells. Statistical significance (two-tailed Student's t-test) was set at P ≤ 0.05 (horizontal black dashed line) and peptide enrichment at log2(fold change) ≥ 0.85 (overrepresented in autophagosomes derived from ETO-treated cells, ≥1.75 fold change) or log2(fold change) ≤ −0.85 (overrepresented in autophagosomes derived from DMSO-treated cells, ≤−1.75 fold change) (vertical black dashed line). Nuclear, ribonucleoprotein complex and ribosomal proteins are labeled with their corresponding gene symbol. d, Bubble plot of the GO term enrichment analysis (Cellular Component, Mann−Whitney U-test) of significantly overrepresented proteins found in autophagosomes isolated from ETO-treated cells (P ≤ 0.05 and log2(fold change) ≥ 0.85). e, Western blot detection of lamin A/C and lamin B1 protein levels in whole-cell extracts from WT and Er1Lyz2/− BMDMs. f, Immunofluorescence staining for the detection of cytoplasmic chromatin fragments and histone H1 upon treatment of WT cells with ETO and/or autophagy inhibitor chloroquine and Er1Lyz2/− cells with autophagy inhibitors chloroquine (CQ), 3-MA and BafA1. White square boxes indicate areas selected for zoomed-in images per genotype per treatment, displayed on the top left of each image panel. White arrows indicate H1+DAPI+ cytosolic chromatin fragments, and green arrowheads point to the focal accumulation of cytoplasmic H1 species. The graphs show the percentage of H1+DAPI+ structures in the cytoplasm of WT or Er1Lyz2/− cells (graph on the left) and the MFI of histone H1 measured in the cytoplasm of cells (graph on the right) (5−8 fields for cyto-DAPI and three independent optical fields for cyto-H1 were counted from each biological replicate; exact P value is provided in the Source Data file). Cells experiencing senescence or DNA damage rely on autophagy to remove excess material55,56,57,58,59,60,61. We hypothesized that DNA damage-induced alterations in autophagosome content could determine the availability of proteins for lysosomal processing and MHC-II loading. To explore this, we employed an in vivo proximity biotinylation approach, combining the APEX2−LC3B system with proteomic analysis62. We constructed U2-OS cells constitutively expressing the APEX2−LC3B construct, cultivated them in SILAC (stable isotope labeling by amino acids in cell culture) media for metabolic labeling and exposed them to etoposide or dimethyl sulfoxide (DMSO). Cells were labeled with ‘medium' and ‘heavy' amino acids, respectively, and subjected to biotinylation by addition of biotin-phenol and H2O2, whereas non-biotinylated controls were labeled with ‘light' amino acids. After treatment, cells were exposed to bafilomycin A1 (BafA1), a vacuolar H+ ATPase (V-ATPase) inhibitor, to accumulate autophagosomes and prevent degradation of their contents63,64. Autophagosomal cargo was purified by streptavidin pulldown in the presence of proteinase K to prevent contaminants from the outer membrane of autophagosomes, and the protein content was quantified using LC−MS/MS (Fig. Etoposide treatment led to the enrichment of 128 proteins within autophagosomes, including vesicle-resident proteins such as MAP1LC3B and LAMP1, as well as nuclear-derived proteins such as histones H2A, H2B and H3, LMNA, NME2 and HNRNPL and ribosomal proteins such as FAU, RPL23A and RPL4 (Fig. These results are consistent with the peptides identified in the Er1Lyz2/− BMDM immunopeptidome, highlighting a substantial overlap in the pathways involved. This suggests that DNA damage-induced changes in the autophagic cargo are pivotal in shaping the antigenic profile of macrophages, contributing to the MHC-II loading of nuclear-derived peptides. Our findings uncover a pivotal role for autophagy in maintaining nuclear homeostasis under genotoxic stress. Previous studies linked the loss of lamin A/C and lamin B1 to DNA damage, which may lead to the release of nuclear constituents into the cytoplasm, where they can be cleared via autophagy. To test this, we examined the levels of these nuclear lamins in wild-type and Er1Lyz2/− BMDMs. We observed a reduction in both lamin A/C and lamin B1 in Er1Lyz2/− BMDMs (Fig. 4e), suggesting that DNA damage disrupts nuclear integrity, despite the lack of cell cycle arrest (Extended Data Fig. Interestingly, ATR-mediated but not ATM-mediated signaling seems to be associated with the decrease in lamin B1 levels (Extended Data Fig. We further investigated the consequences of autophagic inhibition by treating BMDMs with 3-MA, BafA1 and chloroquine, which led to the accumulation of cytoplasmic chromatin structures (DAPI+H1+) and nucleosome protein H1 in Er1Lyz2/− macrophages (Fig. Similarly, etoposide-treated wild-type cells displayed a phenotype resembling that of Er1Lyz2/− macrophages. No DSBs were induced within a 3-h timeframe by these treatments, as evidenced by the co-localization of γH2AX and 53BP1 markers (Extended Data Fig. The cGAS−STING pathway, a key sensor of cytosolic DNA, is involved in the removal of DNA through autophagy, as autophagy proteins interact with cGAS−STING to promote clearance of cytosolic double-stranded DNA65,66. We tested the levels of the cytosolic cGAS in wild-type and Er1Lyz2/− BMDMs and found increased levels in untreated Er1Lyz2/− BMDMs compared to wild-type controls. Autophagic induction via starvation in Er1Lyz2/− macrophages, however, reduced cytosolic cGAS levels, suggesting that increased autophagy turnover facilitates the removal of chromatin fragments (Extended Data Fig. To monitor autophagic flux in these cells, we transfected BMDMs with a fluorescent mCherry−GFP−LC3 plasmid67. In Er1Lyz2/− macrophages, we observed a higher red-to-green fluorescence ratio, indicating increased loss of GFP fluorescence (sensitive to acidic lysosomal environments), whereas red fluorescence was retained in the lysosomes, suggesting an enhanced autophagic flux (Extended Data Fig. Moreover, lysosomes in Er1Lyz2/− BMDMs were larger and more acidic than in wild-type controls, as shown by flow cytometry analysis with LysoTracker and LysoSensor dyes (Extended Data Fig. Collectively, our data point to an enhanced lysosomal function, likely enabling the more efficient and timely transfer of nuclear material for the generation of antigens in Er1Lyz2/− BMDMs. To further investigate the role of autophagy in the transport of nuclear material to lysosomes, we performed immunofluorescence studies with antibodies against LAMP-1, p62, histone H1 and lamin B1. Cells were treated with chloroquine to inhibit the degradation of autophagic cargo. We observed co-localization of H1 and lamin B1 proteins with autophagosomes, which were either fusing with or in close proximity to lysosomes. This was quantified by measuring the percentage of cells with triple co-localized foci and the total number of these foci per each individual cell (Fig. Interestingly, treatment of Er1Lyz2/− cells with Dynasore68, a dynamin inhibitor that blocks endocytosis and phagocytosis in macrophages (early endosome antigen EEA1; Extended Data Fig. 9a), did not diminish the levels of H1 or lamin B1 co-localizing with autolysosomes. This enhanced the notion that autophagic vacuoles dispose of dispensable cytoplasmic content originating from within Er1Lyz2/− cells rather than cell debris derived from neighboring macrophages (Fig. 5a,b and Extended Data Fig. Furthermore, this is supported by the lack of cell death, as measured by flow cytometry analysis of Annexin V/propidium iodide-stained cells (Extended Data Fig. Finally, we purified and analyzed lysosomes from chloroquine-treated wild-type and Er1Lyz2/− BMDMs. Western blotting confirmed the presence of nuclear proteins such as histone H1, lamin B1, lamin A/C and FAU ubiquitin-like as well as ribosomal protein S30 and the LAMP-1 lysosomal protein (Fig. These findings underscore the enhanced nucleophagy and turnover of ribosomal and nuclear proteins in Er1Lyz2/− macrophages, highlighting the crucial role of autophagy in maintaining nuclear integrity on the one hand but enrichment of nuclear antigens in the lysosomes on the other hand. Notably, evidence from immunofluorescence studies in the THP-1 human monocytic cell line upon etoposide treatment and monocyte-derived CD11b+Ly6G− cells isolated from SLE model mice additionally revealed the presence of small cytoplasmic chromatin fragments. These DNA moieties are coated with histone H1 and co-localized with autophagy protein p62 (Fig. Etoposide-induced DNA damage additionally upregulated HLA-DR (MHC-II cell surface receptor) in THP-1 cells, further verifying our hypothesis in human monocytes (Extended Data Fig. a,b, Co-localization studies of autophagy (p62), lysosomes (LAMP-1) and histone H1 (a) or lamin B1 (LMNB1) (b) upon chloroquine or chloroquine and Dynasore (Dyn) treatment. Magenta arrowheads point to H1+p62+LAMP-1+ (a) or LMNB1+p62+LAMP-1+ (b) foci. White arrows point to cytoplasmic chromatin fragments. Single-channel or two-channel images and higher magnifications of a are shown in Extended Data Fig. The two graphs illustrate the percentage of cells with any triple co-localized foci (n = 3 biological replicates, 4−7 optical fields) or the total number of triple co-localized foci per each individual cell counted (n ≥ 144 cells for H1 and n ≥ 162 cells for LMNB1). c, Western blot analysis of lysosomes purified from WT and Er1Lyz2/− BMDMs, after chloroquine treatment for 3 h. Membranes were probed for ribosomal (FAU) and nuclear (H1, LMNB1 and LMNA/C) markers. LAMP-1 was used as a resident protein of lysosomes. The splice point is indicated by a vertical black line. Equal amounts of protein (up to 10 μg) were loaded, and protein levels were normalized according to Ponceau stain. protein levels are plotted (n = 3−4). d,e, Immunofluorescence staining of THP-1 human monocytes (n = 3, P = 0.0053) (d) and cells of monocytic origin sorted from NZB/NZW F1 mice (SLE) (n = 3 P = 0.0115) (e), stained for DAPI, H1 and p62. Cells were fixed after a 3-h treatment with chloroquine. Magenta arrowheads point to chromatin fragments co-localized with p62. The graphs illustrate the percentage of cells with cytoplasmic chromatin fragments. Exact P values are provided in the Source Data. To further investigate the role of autophagy in modulating immune responses in Er1Lyz2/− mice, we generated double knockout (DKO) mice by simultaneously deleting Ercc1 and Atg5, which is crucial for autophagosome formation, in myeloid cells (Ercc1fl/−;Atg5fl/fl;Lyz2-cre+) (Extended Data Fig. In comparison to their Er1Lyz2/− littermates, DKO mice exhibited a reduction in inflammatory foci in their kidneys at 8 months of age, accompanied by lower levels of antinuclear autoantibodies in their sera and moderate hyperplasia of the spleen white pulp with plasmacytic cell foci (Fig. 6a,b and Extended Data Fig. The numbers of CD11b+F4/80+ splenic macrophages and Lin−c-Kit+Sca-1−CD34+CD16/32+ myeloid granulocyte−monocyte progenitor cells in the bone marrow were similar between Er1Lyz2/− and DKO mice (Extended Data Fig. Building on this, we checked for the percentages of activated CD4+ T cells and plasma cells and found them to be decreased in DKO compared to Er1Lyz2/− spleens (Fig. 6c,d and Extended Data Fig. Assessment of cell viability in wild-type, Er1Lyz2/− and DKO BMDMs exhibited no alterations among genotypes, indicating that neither Ercc1 loss nor autophagy ablation in the absence of Ercc1 induces cell death in BMDMs (Extended Data Fig. To test whether the lack of Er1Lyz2/− nuclear and ribosomal antigen presentation could induce the mitigation of adaptive immune system activation, we next performed MHC-II peptidomics analysis in DKO BMDMs (Supplementary Table 3). We selected the potential MHC-II binders and searched for the protein names of the peptides uniquely identified in the case of Er1Lyz2/− immunopeptidomes when they were compared to wild-type control peptidomes. Notably, no peptides derived from H1.1, RPL30, EXOSC4 and ATP5ME were identified in any of the wild-type or DKO MHC-II ligandomes. Likewise, most of the rest of the Er1Lyz2/− peptides (corresponding to 103 out of 120 proteins) were not in common with the DKO overrepresented peptides. We then checked for the subcellular origin of the DKO overrepresented peptides and traced them back to mainly plasma membrane, cytosolic and extracellular matrix proteins, whereas 8% of the peptides consisted of nuclear proteins where approximately 30% of them were overrepresented in Er1Lyz2/− cells (Fig. Considering that DNA damage in BMDMs triggers cytoplasmic DNA sensing coupled with a type I interferon response, we checked for the cytoplasmic cGAS levels also in DKO BMDMs, which were increased in comparison to wild-type cells (Extended Data Fig. DKO BMDM−CD4+ T cell co-culture data well aligned with these results, eliciting a dampened IFNγ production compared to T cells cultured with Er1Lyz2/− BMDMs (Extended Data Fig. Furthermore, ELISpot assays using splenocytes from 8-month-old mice demonstrated a decline in the recall response of antigen-specific CD4+ T cells isolated from DKO mice (Fig. 6f), indicating a critical role for autophagy in systemic inflammation and autoimmune feature appearance in this model. Interestingly, when we purified lysosomes from DKO BMDMs, there was a reduction in the levels of histone H1 and nuclear lamins B1 and A/C compared to lysosomes from Er1Lyz2/− BMDMs. These findings suggest that autophagy is essential for the presentation of immunogenic nuclear and ribosomal antigens (Fig. a, H&E staining in kidney sections derived from 8-month-old WT, Ercc1Lyz2/− and DKO mice. Black arrows point to inflammatory foci. The inflammation score is shown, where 0: no/focal and 1: moderate. The red dotted line represents the mean inflammation score in the kidneys of Er1Lyz2/− mice (n = 8, P = 0.0313). b, ANA-ELISA for the detection of antinuclear autoantibodies in the sera of 8-month-old DKO mice. The red dotted line indicates the average titer of antinuclear antibodies in the Er1Lyz2/− sera (n = 6, P = 0.0449). c, Percentages of CD44hiCD62Llo activated CD4+ T cells in the spleens of 8-month-old WT, Er1Lyz2/− and DKO mice (n = 5−6, P = 0.0009 and P= 0.0404). d, Percentages (left) and numbers (right) of plasma cells in the spleens of 8-month-old WT, Er1Lyz2/− and DKO mice (n = 5−6). e, Bar chart depicting the subcellular origin of the peptides overrepresented in DKO BMDMs compared to WT BMDMs. PEAKS software was used. The bold number corresponds to the percentage of peptides derived from nuclear proteins f, IFNγ ELISpot assay. Splenocytes were isolated from 8-month-old Ercc1Lyz2/−, and DKO mice were pulsed with immunogenic peptides. Representative images are shown, and the number of spots is plotted (n = 3−4, P = 0.005838 and P = 0.000025). g, Western blot analysis of H1 and lamin A/C nuclear proteins present in lysosomal extracts derived from Ercc1Lyz2/− and DKO BMDMs. Equal amounts of protein (up to 10 μg) were loaded, and protein levels were normalized according to Ponceau stain. Exact P values are provided in the Source Data. The lysosomal proteome of senescent cells changes significantly, contributing to the lysosome-derived SASP69. In addition to the exocytosis of extracellular matrix components, we hypothesized that lysosomes from senescent or aged cells might be enriched with proteins also found in the lysosomes of BMDMs accumulating DNA damage (such as in Er1Lyz2/− cells), potentially influencing the immunopeptidome. To test this, we analyzed the lysosomal proteome of senescent SK-MEL-103 cells induced by palbociciclib70. We found that approximately 25% of the Er1Lyz2/− MHC-II-presented peptides were shared with the lysosomal cargo of senescent cells, including nuclear proteins H1 and NME2 and ribosomal proteins such as RPL30 and RPL35A (Fig. Next, we investigated whether macrophages from aged mice display similar traits of senescence and genotoxic stress, potentially altering their lysosomal content and immunopeptidome. We examined different monocyte and macrophage subtypes from both young (2 months) and aged (24 months) mice, including BMDMs (Ly6G−CD11b+CD115+), cells of monocytic origin (Ly6G−CD11b+) and peritoneal macrophages staining positive for marker F4/80 (thioglycolate-elicited macrophages (TEMs)) (Supplementary Fig. Aged BMDMs exhibited elevated levels of γH2X compared to their young counterparts. Similarly, more splenic macrophages and TEMs from aged mice were positive for γH2AX (Fig. Flow cytometry analysis showed that MHC-II levels were increased across all three aged macrophage subtypes, indicating enhanced antigen presentation (Supplementary Fig. Notably, a greater proportion of aged TEMs tested positive for the senescence-associated β-galactosidase (SA-β-gal) marker, similar to Er1Lyz2/− BMDMs34 (Fig. Aged TEMs also showed lamin A/C loss (Supplementary Fig. Immunofluorescence analysis revealed a higher percentage of cells with chromatin fragments and higher co-localization of histone H1, p62 and LAMP-1 in aged TEMs (Fig. These results suggest that DNA damage in aged macrophages leads to profound alterations in their autophagosomal and lysosomal cargo, similar to senescent cells, thereby contributing to higher antigen presentation. Finally, we investigated whether nuclear proteins such as histone H1 and lamin A/C were transported more frequently in lysosomes of aged TEMs and found that they contained higher levels of both proteins. This increase was reduced with 3-MA, implicating macroautophagy in this process (Fig. Thus, both senescent cells and aged macrophages exhibit significant changes in their lysosomal proteomes, with increase in antigen presentation and, likely, alterations in their immunopeptidome. These changes, driven by DNA damage and macroautophagy, highlight the potential for lysosomes to influence immune responses during aging. a, Venn diagram showing the comparative analysis of the lysosomal proteome derived from palbociciclib-induced senescent SK-MEL-103 cells (blue) and of the MHC-II-bound peptidome of Er1Lyz2/− BMDMs (green). The numbers indicate the number of proteins in each dataset. b, γH2AX levels in aged monocytes or macrophages. controls are shown in gray. (i) Flow cytometry analysis of cells isolated from bone marrow of young (2 month (m)) and aged (24 month) mice, stained for Ly6G, CD11b, CD115 surface and γΗ2ΑΧ DNA damage markers. Representative histogram overlay of γH2AX in bone marrow monocytes (Ly6G−CD11b+CD115+ cells) and representative bar plot of the γH2AX MFI (n = 3−4). (ii) Flow cytometry analysis of splenocytes isolated from young and aged mice, stained for Ly6G, CD11b and γΗ2ΑΧ. A representative histogram overlay of γH2AX levels in cells of monocytic origin (Ly6G−CD11b+ cells), with the black bisector gate indicating the γH2AX+ population. The bar plot indicates the corresponding percentages of γH2AX+ cells (n = 4). (iii) Immunofluorescence staining of TEMs isolated from young and aged peritonea using a γH2AX antibody. Representative images and quantification of the percentage of γH2AX+ cells are shown (5−10 optical fields were counted) (n = 6). c, Representative images of the SA-β-gal assay using young and aged TEMs. The percentage of β-gal+ cells is plotted (three independent optical fields, and over 360 cells were counted per biological replicate) (n = 4, P < 0.0001). d, Representative images of the immunofluorescence detection of H1, p62 and LAMP-1 in young and aged TEMs. The white arrow points to a cytoplasmic structure stained positive for DAPI and all three proteins. The graphs depict the percentage of cells with chromatin fragments (top, DAPI+H1+) and of cells with H1+p62+LAMP-1+ foci (4−7 independent optical fields, and n > 67 cells were counted, respectively) (n = 3−5) P = 0.0093 and P = 0.0133. Single-channel images of d are shown in Supplementary Fig. e, Western blot analysis of lysosomal preparations derived from young and aged TEMs, treated or not with 3-MA, as indicated. All were treated with chloroquine for the inhibition of lysosomal cargo degradation. LAMP-1 protein levels are shown as a positive control and Ponceau stain as a loading control. In each case, 7 μg of protein was loaded. A quantitative analysis is presented in the bar chart (n = 3−5). The splice point is indicated by a vertical black line. Exact P values are provided in the Source Data. Our study reveals how DNA damage in macrophages links genotoxic stress to autoimmune disorders and identifies autophagy as key in nuclear and ribosomal antigen processing. This mirrors findings in DNA repair-deficient conditions such as xeroderma pigmentosum, where impaired genome maintenance predisposes individuals to autoimmune conditions such as type 1 diabetes and SLE71. Similarly, type 1 diabetes has been associated with Cockayne syndrome, driven by transcription-coupled repair defects that impair DNA damage removal from actively transcribed genes72,73. Individuals with Cockayne syndrome exhibit increased inflammation74. Consistently, exposure to genotoxins can trigger autoimmune responses75,76,77,78. In line, individuals with SLE exhibit higher DNA damage levels compared to healthy individuals, and several proteins related to genome maintenance have polymorphisms79,80,81. Pharmacological targeting of ATR in B cells of patients with active SLE disease attenuates antibody production82. Accordingly, monocyte-derived cells isolated from NZB/NZW F1 mouse spleens exhibited higher levels of γH2AX and MHC-II depending on the DDR protein ATM. Native DNA turns immunogenic when modified or mislocalized—a shift evident in SLE, where the presence of anti-DNA antibodies is a hallmark. In our study, Er1Lyz2/− mice developed antinuclear autoantibodies, kidney inflammation and complement deposition in the glomeruli, hyperplasia of the spleen white pulp and activation of adaptive immunity. Kidney targeting of immune complexes may be a result of the accumulation of tissue-external nuclear antigens and their corresponding antinuclear antibodies in the kidney filtration barrier or the outcome of MHC-II antigen presentation from BMDMs in the glomerular microvasculature and resident macrophages83,84. DNA damage or repair defects lead to cytoplasmic accumulation of single-stranded DNA10, double-stranded DNA9 or telomeric DNA85, activating a type I interferon response. Cells clear cytoplasmic DNA via various mechanisms including autophagy to maintain homeostasis and prevent immune activation. In addition, autophagy is implicated as a critical mechanism in the defense against pathogens86. Besides its role in suppressing inflammatory responses, macroautophagy is also known to contribute to the development of intracellular MHC-II antigens87,88. Genetic depletion of autophagy in APCs—that is, dendritic cells or macrophages—results in impaired epitope presentation via MHC-II and epitope-specific CD4+ T cell responses89,90,91,92,93. In vivo, dendritic cell-specific Atg5-deficient mice have abrogated autoimmune disease appearance upon myelin oligodendrocyte glycoprotein (MOG) immunization and fail to mount effective TH1 responses in the presence of a viral challenge, due to defective processing and presentation of MHC-II antigens89,90,93. In Er1Lyz2/− cells and genotoxin-treated macrophages and human monocytes, DNA damage boosts MHC-II antigen presentation, displaying nuclear self-peptides, as revealed by MHC-II peptidome analysis. This underscores the role of autophagy in clearing nuclear proteins for antigen presentation. The enrichment of nuclear proteins and ribosomal protein FAU in lysosomes from Er1Lyz2/− BMDMs supports this hypothesis. Our data align with previous reports that different pathologies result in the formation of unique MHC-II peptidomes94,95. Myeloid cell-specific DNA repair and autophagy-deficient mice (Ercc1fl/−;Atg5fl/fl;Lyz2-cre+, DKO) showed improved autoimmune symptoms, reduced antigen-driven CD4+ T cell responses and reversed nuclear and ribosomal MHC-II epitope presentation and nuclear protein enrichment in lysosomes (Fig. This supports that the mechanism of action of hydroxychloroquine and chloroquine—drugs commonly used to treat rheumatic diseases—involves reducing the ability of APCs to present immunogenic peptides to CD4+ T cells.96. Consistently, monocytic-origin cells isolated from NZB/NZW F1 mice had a higher percentage of chromatin fragments in their cytoplasm that were co-localized with p62. Conversely, enhancing autophagy via starvation in Er1Lyz2/− BMDMs lowered the levels of cytoplasmic double-stranded DNA sensor cGAS, indicating the role of autophagy in preventing toxic buildup and dampening antiviral responses to cytoplasmic DNA. Over time, however, this protective mechanism may shift, driving chronic antigen load and heightened T cell responses, ultimately contributing to the development of immunological memory with age. Aged monocytes in mice and humans show elevated MHC-II levels, signaling higher antigen loads and increased autoimmune risk with age17. In 24-month-old macrophages, this rise in antigen presentation, along with increased DNA damage, was evident. We propose that increased nuclear component trafficking to lysosomes via macroautophagy drives heightened MHC-II presentation of nuclear peptides. Together with the observed DNA damage in aged monocytes, this suggests that aging fosters the generation of immunogenic epitopes in macrophages. SASP components secreted during aging significantly contribute to macrophage immunogenic potential and likely also autoimmune responses. In this work, besides the aged macrophage secretome, we suggest that senescent features also determine their lysosomal proteome and MHC-II peptidome. Studies in animal models have shown that chloroquine extends the lifespan in Caenorhabditis elegans, mice and rats while reducing systemic inflammation97,98,99. Similarly, 3-MA alleviates colitis and inflammation in aged mice100. In this respect, boosting DNA repair capacity101, developing DNA damage-centered peptide vaccines102, leveraging exosome-based therapies to remove cytoplasmic nucleic acids9,10,85 or inhibiting autophagy in age-related autoimmune disorders is promising and could enhance immune tolerance and reduce autoimmune risk. Wild-type and Er1Lyz2/− mice were generated as previously described by intercrossing Lyz2-cre (C57BL/6 background), Ercc1fl/fl (FVB background) and Ercc1+/− (C57BL/6 background) mice. This breeding strategy produced wild-type mice carrying a floxed and a wild-type Ercc1 allele (Ercc1fl/+) and Er1Lyz2/− mice carrying one floxed and one knockout allele in conjunction with the Lyz2-cre transgene (Lyz2-cre;Ercc1fl/−). Similarly, Atg5fl/fl mice (C57BL/6) were used to achieve the conditional knocking out of both Atg5 and Ercc1 genes in the same background. Eleven-month-old male F1 mice exhibiting lupus-like disease resulted from the cross of NZB × NZW mice. Mice were housed in a specific pathogen-free facility at the Institute of Molecular Biology and Biotechnology-Foundation for Research and Technology (IMBB-FORTH) where the light/dark cycle (12 h) and temperature were controlled. Mice were fed a normal chow diet and were provided water ad libitum. This work received ethical approval by an independent animal ethics committee at IMBB-FORTH. All relevant ethical guidelines for the work with animals were adhered to during this study. For the duration of all in vivo experiments, mice were monitored daily. Mice were initially anesthetized using ketamine/xylazine. For the acquisition of serum, the blood was centrifuged twice at 10,000g for 10 min at 4 °C. The supernatant was kept, and pellets were discarded. For the adoptive transfer of mouse sera to young hosts, sera from 10-month-old wild-type and Er1Lyz2/− mice were diluted 1:3 in 1× PBS and injected intravenously once per week for a total of 5 weeks. Bone marrow was harvested from the tibias and femurs of mice, and precursor cells were differentiated in DMEM supplemented with 10% FBS, antibiotics (50 μg ml−1 streptomycin, 50 U ml−1 penicillin from Sigma-Aldrich, 2 mM l−1 glutamine from Gibco) and 30% L929 conditioned media for 6 days. On the seventh day, 30% L929 media were replaced with fresh DMEM containing 10% FBS, antibiotics and 10% L929. All treatments were performed on the seventh day of differentiation. In more detail, etoposide (ETO; Sigma-Aldrich, E1383) was added at the concentration of 25 μM for 1 h, and cells were recovered for 24 h before MHC-II flow cytometry analysis. Inhibitors targeting ATM kinase signaling (10 μM ATMi; KU 60019; Sigma-Aldrich, 531978), ATR kinase signaling (10 μΜ ATRi; Millipore, 189299) and DNAPK kinase signaling (2.5 μΜ; NU7441; STEMCELL Technologies, 74082) were added for a 6-h duration. N-acetylcysteine (Sigma-Aldrich, A9165) and Mito-TEMPO (Sigma-Aldrich, SML0737) were added at the concentration of 1 mM and 20 μM, respectively, for 24 h. Autophagy inhibitors 3-MA (Sigma-Aldrich, 189490), 10 mM, and chloroquine (Sanofi Aventis), 50 μM, were added for a duration of 3 h. For the inhibition of endocytosis, cells were treated with 80 μM Dynasore (Sigma-Aldrich, 324410) for a total of 3 h, including a 2-h chloroquine treatment in the case of autolysosome−nuclear protein co-localization studies. LPS-induced activation of BMDMs was carried out at a concentration of 100 ng ml−1 for a 16-h timeframe. For the determination of the cells' autophagic flux, FUW mCherry−GFP−LC3 lentivirus (Addgene, plasmid no. 110060; http://n2t.net/addgene:110060; RRID: Addgene_110060) production was performed in HEK293T cells with helper plasmids psPAX2 and pMD2.G. The supernatant was collected 72 h later, filtered using a 45-μm filter and precipitated with polyethylene glycol before immediate use or storage at −80 °C until use. Viral transfection of BMDMs with the mCherry−GFP–LC3 plasmid was performed in 10% L929 conditioned media on day 6 of differentiation. Cells were fixed 48 h later, permeabilized with 0.1% Triton in 1× PBS and stained with DAPI for the detection of nuclei. Cells were purchased from the American Type Culture Collection (no. TIB-202) and cultured in RPMI 140 (Gibco). Approximately 2 × 105 cells were treated with 25 μM ETO for 1 h or left untreated and recovered in fresh RPMI medium for 24 h before flow cytometry analysis or 4% formaldehyde fixation for immunofluorescence staining. Cells analyzed with immunofluorescence were treated with chloroquine for 3 h beforehand. CD4+ T cells were isolated from spleens of mice. Single-cell suspensions of splenocytes were obtained by mushing spleens in 40-μm strainers, collecting cell pellets by centrifugation at 300g for 5 min at room temperature, removing red blood cells (RBCs) by resuspending the pellets in RBC lysis buffer for 2 min at room temperature and, finally, centrifuging again at 300g for 5 min at room temperature. MACS MicroBeads (CD4 L3T4; Miltenyi Biotec, 130-117-043) were used for positive selection of the desired cell population, as per the manufacturer's instructions. For the T cell−BMDM co-culture, isolated BMDMs were seeded on a 96-well plate with CD4+ T cells in a ratio of 1:1 in the presence of 0.5 μg ml−1 CD28 (Invitrogen, 16-0281-86), in RPMI 140 medium (Gibco). BMDMs were differentiated as described, and 5 × 106 cells were injected intravenously at the timepoints indicated. CD4+ T cells were isolated as described, and 2 × 106 cells were injected intravenously once per week for a total of 8 weeks in young NSG hosts. Intraperitoneal injections of 70 μg of anti-MHC-II (I-A/I-E, M5/114 monoclonal antibody by Bio X Cell) blocking or isotype control antibodies were administered weekly, for 4−6 weeks. Intravenous injections of 150 μg of anti-CD4 (Bio X Cell) blocking or isotype control antibodies were administered weekly, for 6 weeks. BMDMs, TEMs, THP-1 monocytes, CD11b+Ly6G− monocytes from NZB/NZW F1 mice or MEFs were fixed in 1× PBS/4% formaldehyde for 10 min at room temperature. Cells were washed three times in 1× PBS, blocked and permeabilized using 1% BSA and 0.1% Triton X-100 in 1× PBS. Primary antibodies or mouse sera were incubated in 1% BSA/0.1% Triton X-100/PBS for 1 h at room temperature or overnight at 4 °C. Afterwards, cells were washed three times with 0.1% Triton/PBS, and fluorochrome-conjugated secondary antibodies were added in 1% BSA/0.1% Triton X-100/PBS for 1 h at room temperature, followed by three more washes in 0.1% Triton X-100/1× PBS. Mounting was done with 80% glycerol, and samples were imaged with a Leica SP8 confocal microscope. Cells were fixed in methanol, on ice, for 10 min and washed three times with 1× PBS. After fixation, the coverslips were air dried and incubated in 0.05 N HCl for 5 min, on ice, washed three times in 1× PBS and then incubated in a 100 μg ml−1 RNase A (Macherey-Nagel, 740397), 150 mM NaCl and 15 mM sodium citrate solution for 1 h, at 37 °C. A 1× PBS wash was performed for 3 min, followed by sequential ethanol dehydration steps: 35%, 50% and 75% ethanol for 3 min. Then, 0.15 N NaOH in 70% ethanol was added for 4 min; two PBS washes were performed; and cells were fixed with 4% formaldehyde in 70% ethanol for 2 min. The fixation buffer was exchanged with 50% and then 35% ethanol, and cells were again washed with 1× PBS for 2 min and treated with 5 μg ml−1 proteinase K in Tris-EDTA buffer for 5−10 min, at 37 °C. After a PBS wash, blocking and primary antibody incubation was performed as described before. An 8-oxoG antibody (Millipore, MAB3560) was used, at a 1:100 concentration, overnight. Kidneys were fresh frozen in OCT compound and stored at −80 °C until further analysis. Kidneys were sliced using a Leica CM1850 UV cryostat (7 μm). Tissue sections were fixed in 4% formaldehyde for 15 min, washed three times with 1× PBS and blocked with 1% BSA/0.1% Triton X-100/PBS for 1 h at room temperature. A similar protocol as the one for immunofluorescence of fixed cells was followed. Skins, kidneys and spleens were dissected from mice, fixed overnight in 4% formaldehyde, washed three times with 1× PBS and then embedded in paraffin blocks. Tissue sections were used for H&E or PAS staining. For surface protein staining, cells were stained with fluorochrome-conjugated antibodies diluted in staining buffer (1× PBS/5% FBS or 1× HBSS/5% FBS) for 20 min on ice, using the concentrations indicated by the manufacturer. Cells were washed by staining buffer and centrifuged at 300g for 5 min at 4 °C. For the staining of intracellular proteins, True-Nuclear Transcription Factor Buffer Set (BioLegend, 424401) was used, and cells were centrifuged at 400g for 5 min at room temperature, after fixation. Staining for the detection of granulocyte−monocyte progenitors in the bone marrow was as follows. Bone marrow was collected by flushing the femur, and cells were incubated with RBC buffer for the removal of erythrocytes. Then, 1 × 106 cells were stained with Pacific Blue anti-mouse lineage antibody cocktail (1:10), PE anti-mouse CD34 (1:50), FITC anti-mouse c-Kit (1:100), APC anti-mouse CD16/32 (1:50) and PerCP anti-mouse SCA-1 antibody (1:100) for 3 h at 4 °C, before a PBS/5% FBS wash and flow cytometry analysis. Lysosomal dyes were purchased from Thermo Fisher Scientific: LysoTracker Red DND-99 (L7528) and LysoSensor Green DND-189 (L7535). For Annexin V/propidium iodide staining in BMDMs, the FITC Annexin V Apoptosis Detection Kit (BD Pharmingen, 556547) was used. Cell analysis was eventually performed in a FACSCanto II flow cytometer or a FACSCalibur (BD Biosciences), and data analysis was performed using FlowJo software (Tree Star). The gating strategies were as follows: forward scatter/side scatter (FSC/SSC) for live cell selection and debris removal; forward scatter area/side scatter area (FSC-A/SSC-A) for the subsequent removal of cell aggregates; and then fluorophore-conjugated specific antibodies for the next gates. Spleens were obtained from NZB/NZW F1 mice, and single-cell suspensions were either cryopreserved and thawed for analysis or directly stained with PE anti-mouse Ly6G (1:200) and APC anti-mouse CD11b (1:100) for 20 min, at 4 °C, in 1× PBS/5% FBS and 2 mM EDTA. Cell sorting was performed in a FACSAria III flow cytometer, and monocytic-origin cells were identified as Ly6G−CD11b+. Cells were seeded on poly-l-lysine-coated coverslips, in 48-well plates (105 cells per well), and cultured for 24 h in RPMI 1640 and recombinant M-CSF (250 ng ml−1 working concentration; PeproTech, 315-02). Cells were treated with chloroquine for 3 h prior to fixation. The splenocytes that were not separated through cell sorting were seeded on 48-well plates (105 cells per well), in RPMI 1640 and recombinant M-CSF, and treated with an ATM inhibitor for a duration of 16 h. Cells were collected and stained with DAPI for dead cell exclusion, CD11b and Ly6G for monocyte labeling and MHC-II antigen presentation protein. Quantitative qPCR was performed using a CFX Duet Real-Time PCR system device (Bio-Rad), and data were analyzed as previously described34. The Hprt1 (hypoxanthine phosphoribosyltransferase 1) gene was used for normalization. At least 30 × 106 BMDMs or TEMs pooled from four mice were collected for the isolation of lysosomes using the Lysosome Isolation Kit (Abcam, ab234047). Lysosomes were then lysed using RIPA, and their protein content was detected through immunoblotting analysis. Cells or lysosomes were lysed with RIPA buffer, containing 50 mM Tris-HCl (pH 8.0), 150 mM NaCl, 0.5% sodium deoxycholate, 1% Nonidet P-40 and 0.1% sodium dodecyl sulfate and protease and phosphatase inhibitors. For IFNβ detection from BMDMs, culture supernatants from the same amount of cells were concentrated using Amicon Ultra Centrifugal Filter, 10-kDa molecular weight cutoff (Merck Millipore, UFC901024). The concentrated supernatants were mixed with equal volumes of 2× Laemmli and boiled at 80 °C for 10 min before being loaded into the gel for SDS−PAGE. For cell and lysosome lysates, protein concentration was determined using Bradford protein assay, and equal amounts of protein were loaded (50−80 μg for cells and 5−8 μg for lysosomes) for SDS−PAGE. Equal parts of concentrated supernatant proteins were loaded. Proteins were transferred to nitrocellulose membranes (Amersham Hybond), blocked using 5% skim milk diluted in 1× PBS with 0.1% Tween 20 (PBS-T) for 1 h and probed with antibodies. β-tubulin or actin was used for the normalization in the case of cell lysates and supernatants and Ponceau staining for the normalization in the case of lysosomal lysates. An ECL (Thermo Fisher Scientific and Amersham) development was performed, and results were imaged using ImageBlot (Bio-Rad). Quantification was performed using Fiji (ImageJ). Antibodies against the following proteins were used: MHC-II (Bio X Cell, clone M5/114; western blot: 1:800), ERCC1 (Santa Cruz Biotechnology, clone D-10; western blot: 1:500), Ki-67 (Cell Signaling Technology, 9129S, clone D3B5; FACS: 1:500), γH2AX (Millipore, 05-636; immunofluorescence: 1:12,000), 53BP1 (NB100-304; immunofluorescence: 1:200), IFNβ (Cell Signaling Technology, 97450; immunofluorescence: 1:500, western blot: 1:1,000), pSTAT1 (Cell Signaling Technology, 9167; western blot: 1:250) and STAT1 (Cell Signaling Technology, 14994; western blot: 1:500), β-tubulin (Abcam, ab6046; western blot: 1:1,000), IRF5 (Proteintech, 10547-1-AP; western blot: 1:500), actin (Cytoskeleton, BK037; western blot: 1:5,000), H1 (Santa Cruz Biotechnology, sc-8030; immunofluorescence: 1:50, western blot: 1:200), LAMN A/C (Proteintech, 10298-1-AP; western blot: 1:2,000), lamin B1 (Abcam, ab16048; immunofluorescence: 1:500, western blot: 1:1,000), cGAS (Proteintech, 26416-1-AP; immunofluorescence: 1:200), EEA1 (Proteintech, 28347-1-AP; western blot: 1:200), LAMP1 (Santa Cruz Biotechnology, sc-19992; immunofluorescence: 1:100), LAMP1 (Developmental Studies Hybridoma Bank; western blot: 1:200), p62 (Abnova; immunofluorescence: 1:1,000), p62 (Cell Signaling Technology; immunofluorescence: 1:500), GAPDH (Abcam, ab8245; western blot: 1:2,000), FAU (Proteintech, 13581-1-AP; western blot: 1:200), ATG5 (Proteintech, 10181-2-AP; western blot: 1:1,000) and γH2AX (Cell Signaling Technology; immunofluorescence: 1:500, FACS: 1:500). Approximately 3 × 108 BMDMs, derived from a pool of isolated cells from four mice of the same genotype, were used per each MHC-II immunoprecipitation sample. Cells were initially lysed using a buffer containing 0.5% NP-40, 50 mM Tris (pH 8.0), 150 mM NaCl and protease inhibitors. Native MHC-II−peptide complexes were purified using InVivoMAb anti-mouse MHC-II (I-A/I-E, M5/114 monoclonal antibody by Bio X Cell) along with Protein G Sepharose beads (Millipore). Immunopeptides were purified using Sep-Pak tC18 columns containing 100 mg of sorbent (Waters Corporation). The elution of peptides from the tC18 sorbent was conducted with 32% acetonitrile (ACN) in 0.1% trifluoroacetic acid (TFA). Eluates were volume reduced using a vacuum evaporator until almost all liquid was evaporated. The peptides were then resolved with 2% ACN in 0.5% TFA and stored at −80 °C until further analysis103. An LC−MS/MS analysis was performed on a Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) online coupled to an UItiMate 3000 RSLC nano-HPLC (Dionex/Thermo Fisher Scientific). The peptides were automatically injected and loaded onto a C18 trap column (300-µm inner diameter × 5 mm, Acclaim PepMap100 C18, 5 µm, 100 Å, LC Packings; Thermo Fisher Scientific) at a 30 µl min−1 flow rate prior to performing C18 reversed-phase chromatography on the analytical column (nanoEase MZ HSS T3 Column, 100 Å, 1.8 µm, 75 µm × 250 mm; Waters Corporation) at a 250 nl min−1 flow rate in a 95-min nonlinear ACN gradient from 3% to 40% in 0.1% formic acid. Profile precursor spectra from 300 m/z to 1,650 m/z were recorded at 60,000 resolution with an automatic gain control (AGC) target of 3 × 106 and a maximum injection time of 100 ms. The 15 most abundant peptide ions of charges 1 to 4 were selected from the mass spectrometry scan and fragmented using higher-energy collisional dissociation (HCD) with a normalized collision energy of 28, an isolation window of 1.6 m/z and a dynamic exclusion of 15 s. MS/MS spectra were recorded at a resolution of 30,000 with an AGC target of 1 × 105 and a maximum injection time of 100 ms. Proteome Discoverer 2.5 software (version 2.5.0.400; Thermo Fisher Scientific) was used for peptide and protein identification via a database search (Sequest HT search engine) against the SwissProt murine database (release 2020_02; 17,070 sequences). The database search was performed with an unspecified peptide cleavage. The precursor mass tolerance was 10 ppm, and the fragment mass tolerance was 0.02 Da. The carbamidomethylation of cysteine was set as static modification. Dynamic modifications included the deamidation of asparagine and glutamine, the oxidation of methionine and a combination of methionine loss with acetylation on the protein N terminus. Peptide spectrum matches (PSMs) and peptides were validated with the Percolator algorithm105. Only the top-scoring hits for each spectrum were accepted with a false discovery rate (FDR) < 1% (high confidence). The mass spectrometry data for Fig. 6 were acquired in DDA-PASEF mode on a timsTOF Ultra 2 mass spectrometer (Bruker). Peptides were loaded on Evotips (one Evotip for each injection). They were placed in an Evosep autosampler until analysis. The 40 samples per day whisper method employing a 27-min gradient with solvents A (0.1% formic acid, water) and B (0.1% formic acid, MeCN) was chosen, and a 15-cm column (PepSep C18, 1.9-µm beads, 75-µm inner diameter) was used for separation of peptides. The DDA-PASEF method (‘MHC class II') covered a mass range from 100 m/z to 1,700 m/z and a mobility range from 0.64 to 1.45 1/K0. A duty cycle of 100% was achieved by setting the ramp time and accumulation time to 100 ms each. Estimated cycle time was 0.6 s. PEAKS Studio 12.5 software (Bioinformatics Solutions) was employed for peptide identification using the DeepNovo algorithm for de novo peptide sequencing against the SwissProt Mouse database (release 2020_02; 17,061 entries). Mass spectrometry data were searched without enzymatic specificity constraints, with precursor mass error tolerances set to 10.00 ppm and 20.00 ppm for different experiments, and fragment mass error tolerance was maintained at 0.02 Da. Peptide length was constrained between 6 and 30 amino acids to accommodate the broader range typical for MHC-II immunopeptidomics applications. Variable post-translational modifications included N-terminal acetylation (+42.01 Da), asparagine and glutamine deamidation (+0.98 Da) and methionine oxidation (+15.99 Da), with a maximum of three variable modifications permitted per peptide. PSMs were filtered using stringent criteria including DeepNovo confidence scores ≥70.00% and PSM significance thresholds of −log10(P) ≥ 20.0, with confident amino acid assignment requiring ≥2.00% threshold. Label-free quantification was performed using identification-directed quantification with feature intensity thresholds ranging from 300 to 100,000. Ion mobility tolerance was set to 0.05 (1/K0) where applicable. Data refinement included mass correction and chimera association algorithms to improve spectral quality. No normalization methods were applied to preserve the inherent biological variance in the quantitative data. Co-culture supernatants were collected after two centrifugations: 300g, 5 min, room temperature, for the removal of cells and 2,000g, 15 min, 4 °C, for the removal of cell debris. Protease inhibitors were added, and the samples were stored at −80 °C until use. Mouse IFNγ protein levels were quantified using Mouse IFNγ ELISA MAX Deluxe Set (BioLegend, 430804). Antinuclear autoantibodies were quantified using Mouse Anti-Nuclear Antigens (ANA/ENA) Ig (total (A + G + M)) ELISA Kit (Alpha Diagnostics International, 5210). Albumin levels in the urine were detected using Mouse Albumin ELISA Kit (Bethyl Laboratories, E99-134). ELISpot assays were performed according to the manufacturer's instructions (Mabtech, 3321-4APT-2). In total, 200.000 splenocytes were stimulated for 48 h in the presence of 3 μg of the indicated synthesized peptide (Macrogen) and 30 U ml−1 recombinant interleukin-2 (rIL-2; PeproTech, 0717108). rIL-2-stimulated splenocytes derived from Er1Lyz2/− mice were used as a negative control. For the T cell−macrophage co-cultures, T cells and BMDMs were mixed in a 1:4 ratio for 48 h, in the presence of 0.5 μg ml−1 anti-CD28 and 30 U ml−1 rIL-2. BMDMs were pretreated with 10 mM 3-MA for 3 h and 20 μg ml−1 anti-MHC-II (I-A/I-E, M5/114 monoclonal antibody by Bio X Cell) or 20 μg ml−1 anti-IFNβ (BioLegend, 508107) for 24 h where necessary. Samples were imaged with a Leica M205 FA dissection microscope, and spots were counted using ImageJ. The peptides used for the assays were as follows: H1-1: KKPKVVKAKKVAKSPA, RPL30: PGDSDIIRSMPEQTGEK, COL1A: TPAKNSYSRAQANKH, HRNRPL: YGNVEKVKFMKSKPG, EXOSC4: GPHEIRGSRSRALPD. U2-OS cells constitutively expressing APEX2−Flag−LC3B were subjected to autophagosome content profiling as described in Le Gerroué et al.62. Quantification of the proteasomal content of autophagosomes via mass spectrometry was based on SILAC. In brief, APEX2−Flag−LC3B proximal proteins were biotinylated in ‘medium' and ‘heavy' labeled cells by inducing APEX2 activity after incubation with biotinphenol (500 mM) for 2 hat 37 °C and a 1-mimpulse with H2O2 (1 mM). Biotinphenol was also added to ‘light' labeled cells, but biotinylation was not induced by omission of H2O2 application. In heavy labeled cells, autophagy was induced with genotoxic stress for 16 h of etoposide treatment (10 µM), whereas medium and light labeled cells were treated with DMSO. Autophagosome enrichment was induced by adding BafA (200 nM) to all cells for 2 h simultaneously to the biotinphenol treatment. Quenching solution (1 mM sodium azide, 10 mM sodium ascorbate and 5 mM Trolox in DPBS) was added to all cells to stop remaining biotinylation reactions, followed by three washing steps with PBS. Cells were collected from 15-cm dishes by trypsinization (3 min, 37 °C). Afterwards, cells were washed twice in PBS prior to mixing them in a 1:1:1 ratio based on cell numbers. An autophagosome-enriched fraction was recovered from the cells by the steps below performed at 4 °C prior to a streptavidin pulldown. Cells were washed and incubated in homogenization buffer I (10 mM KCl, 1.5 mM MgCl2, 10 mM HEPES-KOH and 1 mM DTT (pH 7.5)) for 20 min in an overhead shaker. Afterwards, cells were transferred into a dounce homogenizer and lysed with tight-fitting pestle B. The lysate was transferred into a new reaction vessel and diluted in homogenization buffer II (75 mM KCl, 22.5 mM MgCl2, 220 mM HEPES-KOH and 0.5 mM DTT (pH 7.5)). After centrifugation, the autophagosome-rich supernatant was treated with proteinase K (30 µg ml−1) and 1 mM CaCl2 for 30 min. Then, 5 mM PMSF was added to inactivate proteinase K. The fraction was cleared by centrifugation at 17,000g for 15 min, and the pellet was resuspended and incubated in RIPA buffer (50 mM Tris-HCl (pH 7.4), 1% Triton X-100, 0.5% sodium deoxycholate, 0.1% SDS and 150 mM NaCl) for 30 min. Afterwards, the lysate was cleared by centrifugation at 20,000g for 15 min. The supernatant was incubated overnight with pre-equilibrated NeutrAvidin beads. Beads were washed four times with RIPA buffer, and proteins were eluted by boiling in 3× sample buffer supplemented with 1 mM DTT for 20 min at 95 °C. Proteins were incubated with CAA (4.5 mM) in the dark and resolved on SDS-PAGE. In-gel digestion using trypsin followed prior to subjection of the peptides to LC−MS/MS analysis. Differential protein analysis was performed using rule-based (frequency of identification, fold change) and statistical tests (t-test). Uniquely identified proteins were marked as those identified in all three biological repeats of the group and in none of the repeats of the comparing condition. Commonly identified but differentially abundant proteins between the two conditions were reported as those with greater than or equal to 1.75-fold change of average protein abundance between groups and t-test P < 0.05, for proteins identified in all three biological repeats in each of the comparing conditions. The t-test was performed on log2-transformed protein intensity values between the heavy and medium isotopically labeled conditions. Analysis was performed in Python programming language using common libraries (scipy, statmodels, numpy, pandas and matplotlib). Plots were created by a free online platform for data visualization, SRplot (https://www.bioinformatics.com.cn/en), and GraphPad Prism 8.0. All statistical analyses were performed in GraphPad Prism 8.0. No statistical methods were used to predetermine sample sizes, but our sample sizes are similar to those reported in previous publications34. No data were excluded from the analyses. Data distribution was assumed to be normal, but this was not formally tested. The investigators were not blinded to allocation during experiments and outcome assessment. Samples were allocated to experimental groups according to genotypes or treatments. No method of randomization was used to assign samples to experimental groups. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE106 partner repository with dataset identifiers PXD058775 and PXD058936. All data needed to evaluate the conclusions in this paper are present in the paper and/or the supplementary materials. All data supporting the findings are also available from the corresponding author upon reasonable request. The DNA-damage response in human biology and disease. The central role of DNA damage in immunosenescence. The central role of DNA damage in the ageing process. Yousefzadeh, M. J. et al. An aged immune system drives senescence and ageing of solid organs. Zhao, Y. et al. DNA damage and repair in age-related inflammation. The senescence-associated secretory phenotype: the dark side of tumor suppression. Hartlova, A. et al. DNA damage primes the type I interferon system via the cytosolic DNA sensor STING to promote anti-microbial innate immunity. Arvanitaki, E. S. et al. Microglia-derived extracellular vesicles trigger age-related neurodegeneration upon DNA damage. Chatzidoukaki, O. et al. R-loops trigger the release of cytoplasmic ssDNAs leading to chronic inflammation upon DNA damage. Karakasilioti, I. et al. DNA damage triggers a chronic autoinflammatory response, leading to fat depletion in NER progeria. Uchihara, Y. et al. DNA damage promotes HLA class I presentation by stimulating a pioneer round of translation-associated antigen production. Sheu, K. M. & Hoffmann, A. Functional hallmarks of healthy macrophage responses: their regulatory basis and disease relevance. Park, M. D. et al. Macrophages in health and disease. Takahashi, R. et al. Attenuated phagocytosis of secondary necrotic neutrophils by macrophages in aged and SMP30 knockout mice. Rawji, K. S. et al. Deficient surveillance and phagocytic activity of myeloid cells within demyelinated lesions in aging mice visualized by ex vivo live multiphoton imaging. Barman, P. K. et al. Production of MHCII-expressing classical monocytes increases during aging in mice and humans. Cui, C. Y. et al. Skewed macrophage polarization in aging skeletal muscle. Franceschi, C. et al. Inflammaging: a new immune-metabolic viewpoint for age-related diseases. Prinz, J. C. Immunogenic self-peptides - the great unknowns in autoimmunity: identifying T-cell epitopes driving the autoimmune response in autoimmune diseases. Kroemer, G., Marino, G. & Levine, B. Autophagy and the integrated stress response. White, E., Lattime, E. C. & Guo, J. Y. Autophagy regulates stress responses, metabolism, and anticancer immunity. Munz, C. Autophagy beyond intracellular MHC class II antigen presentation. Rock, K. L., Reits, E. & Neefjes, J. Li, Y. et al. Nuclear accumulation of UBC9 contributes to SUMOylation of lamin A/C and nucleophagy in response to DNA damage. Mucino-Hernandez, G. et al. Nucleophagy contributes to genome stability through degradation of type II topoisomerases A and B and nucleolar components. & Garinis, G. A. Nucleotide excision repair: new tricks with old bricks. Apostolou, Z. et al. Nucleotide excision repair and transcription-associated genome instability. The structure-specific endonuclease complex SLX4−XPF regulates Tus−Ter-induced homologous recombination. Kumar, N., Raja, S. & Van Houten, B. The involvement of nucleotide excision repair proteins in the removal of oxidative DNA damage. Zhu, X. D. et al. ERCC1/XPF removes the 3′ overhang from uncapped telomeres and represses formation of telomeric DNA-containing double minute chromosomes. Li, T. T. & Vasquez, K. M. Multi-faceted roles of ERCC1-XPF nuclease in processing non-B DNA structures. Goulielmaki, E. et al. Tissue-infiltrating macrophages mediate an exosome-based metabolic reprogramming upon DNA damage. The development of autoimmune features in aging mice is closely associated with alterations of the peripheral CD4+ T-cell compartment. Baaten, B. J., Li, C. R. & Bradley, L. M. Multifaceted regulation of T cells by CD44. Cutting edge: L-selectin (CD62L) expression distinguishes small resting memory CD4+ T cells that preferentially respond to recall antigen. Chatila, T. et al. Mechanisms of T cell activation by the calcium ionophore ionomycin. Oda, T. et al. DNA-double strand breaks enhance the expression of major histocompatibility complex class II through the ATM-NF-κB-IRF1-CIITA pathway. Lhuillier, C. et al. Radiotherapy-exposed CD8+ and CD4+ neoantigens enhance tumor control. Marin, I. et al. Cellular senescence is immunogenic and promotes antitumor immunity. Emerging cancer therapeutic opportunities target DNA-repair systems. van Maanen, J. M. et al. Curtsinger, J. M., Lins, D. C. & Mescher, M. F. Signal 3 determines tolerance versus full activation of naive CD8 T cells: dissociating proliferation and development of effector function. Curtsinger, J. M. & Mescher, M. F. Inflammatory cytokines as a third signal for T cell activation. & Luster, A. D. Chemokines: critical regulators of memory T cell development, maintenance, and function. Turley, S. J., Fletcher, A. L. & Elpek, K. G. The stromal and haematopoietic antigen-presenting cells that reside in secondary lymphoid organs. Koncz, B., Balogh, G. M. & Manczinger, M. A journey to your self: the vague definition of immune self and its practical implications. Cibrian, D. & Sanchez-Madrid, F. CD69: from activation marker to metabolic gatekeeper. Richert, I. et al. A TLR4 agonist induces osteosarcoma regression by inducing an antitumor immune response and reprogramming M2 macrophages to M1 macrophages. Pisetsky, D. S. Pathogenesis of autoimmune disease. & Furuta, K. The ins and outs of MHC class II-mediated antigen processing and presentation. Impaired lysosome reformation in chloroquine-treated retinal pigment epithelial cells. & Kroemer, G. Autophagy in the pathogenesis of disease. & Dillin, A. Macroautophagy in quiescent and senescent cells: a pathway to longevity? The coordinate regulation of the p53 and mTOR pathways in cells. A. et al. Mitochondrial dysfunction in ataxia-telangiectasia. Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Lan, Y. Y. et al. Extranuclear DNA accumulates in aged cells and contributes to senescence and inflammation. Repair, reuse, recycle: the expanding role of autophagy in genome maintenance. Juretschke, T. & Beli, P. Causes and consequences of DNA damage-induced autophagy. Le Guerroué, F. et al. Autophagosomal content profiling reveals an LC3C-dependent piecemeal mitophagy pathway. Mauthe, M. et al. Chloroquine inhibits autophagic flux by decreasing autophagosome-lysosome fusion. Mauvezin, C. & Neufeld, T. P. Bafilomycin A1 disrupts autophagic flux by inhibiting both V-ATPase-dependent acidification and Ca-P60A/SERCA-dependent autophagosome-lysosome fusion. Liang, Q. et al. Crosstalk between the cGAS DNA sensor and Beclin-1 autophagy protein shapes innate antimicrobial immune responses. Zhao, M. et al. CGAS is a micronucleophagy receptor for the clearance of micronuclei. Leeman, D. S. et al. Lysosome activation clears aggregates and enhances quiescent neural stem cell activation during aging. Macia, E. et al. Dynasore, a cell-permeable inhibitor of dynamin. & Finkel, T. Lysosomes in senescence and aging. The lysosomal proteome of senescent cells contributes to the senescence secretome. Taieb, A. et al. Clinical, immunological, and genetic investigations in a rare association of type 1 diabetes with xeroderma pigmentosum. & Berry, S. A. Cockayne syndrome: review of 140 cases. Ozdirim, E. et al. Cockayne syndrome: review of 25 cases. Immunity in the progeroid model of Cockayne syndrome: biomarkers of pathological aging. Ocadlikova, D. et al. Chemotherapy-induced tumor cell death at the crossroads between immunogenicity and immunotolerance: focus on acute myeloid leukemia. Broecker, F. et al. Chemotherapy-induced, broadly reactive autoantibodies in a colon cancer patient. Tobacco smoking and autoimmune rheumatic diseases. B. DNA repair and systemic lupus erythematosus. Micheli, C. et al. UCTD and SLE patients show increased levels of oxidative and DNA damage together with an altered kinetics of DSB repair. Stergioti, E. M. et al. Transcriptomic and proteomic profiling reveals distinct pathogenic features of peripheral non-classical monocytes in systemic lupus erythematosus. Manolakou, T. et al. ATR-mediated DNA damage responses underlie aberrant B cell activity in systemic lupus erythematosus. Westhorpe, C. L. V. et al. Effector CD4+ T cells recognize intravascular antigen presented by patrolling monocytes. Distinct fate, dynamics and niches of renal macrophages of bone marrow or embryonic origins. Transcription stress at telomeres leads to cytosolic DNA release and paracrine senescence. Deretic, V. Autophagy in inflammation, infection, and immunometabolism. Dengjel, J. et al. Autophagy promotes MHC class II presentation of peptides from intracellular source proteins. Towards a systems understanding of MHC class I and MHC class II antigen presentation. Alissafi, T. et al. Tregs restrain dendritic cell autophagy to ameliorate autoimmunity. Keller, C. W. et al. ATG-dependent phagocytosis in dendritic cells drives myelin-specific CD4+ T cell pathogenicity during CNS inflammation. Romao, S. et al. Autophagy proteins stabilize pathogen-containing phagosomes for prolonged MHC II antigen processing. Ma, J. et al. Dectin-1-triggered recruitment of light chain 3 protein to phagosomes facilitates major histocompatibility complex class II presentation of fungal-derived antigens. In vivo requirement for Atg5 in antigen presentation by dendritic cells. Clement, C. C. et al. Pleiotropic consequences of metabolic stress for the major histocompatibility complex class II molecule antigen processing and presentation machinery. The MHC class II immunopeptidome of lymph nodes in health and in chemically induced colitis. Schrezenmeier, E. & Dorner, T. Mechanisms of action of hydroxychloroquine and chloroquine: implications for rheumatology. Li, W. et al. Low-dose chloroquine treatment extends the lifespan of aged rats. Long-term treatment with chloroquine increases lifespan in middle-aged male mice possibly via autophagy modulation, proteasome inhibition and glycogen metabolism. Boosting ATM activity alleviates aging and extends lifespan in a mouse model of progeria. Liu, A. et al. Autophagy inhibitor 3-methyladenine mitigates the severity of colitis in aged mice by inhibiting autophagy. The DREAM complex functions as conserved master regulator of somatic DNA-repair capacities. Therapeutic potential of tolerance-based peptide vaccines in autoimmune diseases. Marino, F. et al. High-throughput, fast, and sensitive immunopeptidomics sample processing for mass spectrometry. Zolg, D. P. et al. INFERYS rescoring: boosting peptide identifications and scoring confidence of database search results. Kall, L. et al. Semi-supervised learning for peptide identification from shotgun proteomics datasets. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. The Horizon 2020 Marie Curie ITN ‘HealthAge' (GA 812830); the Horizon-WIDERA-2023-Talents-01-01: ERA Chairs ‘RACE' (1011803090); the Horizon-WIDERA-Access Pathways to Synergies 2023 ‘TRIAD' (101158508); the Horizon-WIDERA-2023-Talents-01, ERA Chair InflaCare (101180309); the Fondation Santé; ELIDEK grant 11578; ELIDEK PhD fellowships 11034 and 11330; the Hevolution Foundation; the Uni-Pharma Kleon Tsetis Pharmaceutical Laboratories S.A. (PAR00838) and Pharmathen S.A. (PAR00863) research funds; and the Greece 2.0, National Recovery and Resilience Plan Flagship program TAEDR-0535850 supported this work. The research in the Beli laboratory was funded by the Deutsche Forschungsgemeinschaft (German Research Foundation; Project-ID 259130777 – SFB 1177). These authors contributed equally: George Niotis, Ermioni S. Arvanitaki. Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, Heraklion, Greece George Niotis, Ermioni S. Arvanitaki, Emmanouil Theodorakis, Konstantinos C. Tsolis & George A. Garinis Department of Veterinary Sciences, LMU Munich, Martinsried, Germany Core Facility – Metabolomics and Proteomics Core, Helmholz Center Munich, German Research Center for Environmental Health (GmbH), Munich, Germany Institute of Molecular Biology (IMB), Mainz, Germany Thomas Juretschke & Petra Beli Department of Rheumatology, Clinical Immunology and Allergy, University Hospital of Heraklion, Heraklion, Greece Department of Pathology, Medical School, University of Crete, Rethymno, Greece Institute of Developmental Biology and Neurobiology (IDN), Johannes Gutenberg Universität, Mainz, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to George A. Garinis. The authors declare no competing interests. Nature Aging thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Immunofluorescence analysis of kidney cryosections stained with antibodies raised against IgM and IgA immunoglobulins or C3 complement protein. Kidney glomeruli are shown in all images. The plots on the bottom depict the mean fluorescence intensity (MFI) of each staining (n = 5-7, pval = 0.0039 and pval = 0.0327). Histological analysis (H&E staining) of skins derived from 8-month-old wt or Er1Lyz2/− mice. (C.) Caliper measurements of the thickness of paws and ankles from 8-month-old wt and Er1Lyz2/− animals (n = 6, pval > 0.05). (D.) PAS staining of kidney paraffin sections. The white arrow indicates glomerular damage (n = 3). (E.) ELISA for the detection of albumin in the urine of wt and Er1Lyz2/− animals (n = 5, pval > 0.05). (F.) Autoantibody detection in the sera of 8-month-old wt and Er1Lyz2/− mice using an indirect immunofluorescence assay. Primary wild-type mouse embryonic fibroblasts (MEFs) were seeded and incubated with the mouse sera, as indicated. Here, representative images of 1:50 sera dilutions are presented. The autoantibody positivity for mice is also expressed as the percentage indicated on the top right of each image (n = 10 biological replicates and 5 independent optical fields/mouse serum were assessed).(G.) Patterns of fluorescence in MEFs incubated with 8-month-old Er1Lyz2/− sera are indicated(n = 10). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01 (two-tailed Student's t-test). Adoptive transfer of sera derived from wt or Er1Lyz2/− animals to young wt hosts. A schematic diagram is shown (top). The immunofluorescence images depict glomeruli from kidney cryosections, stained with an IgM antibody (bottom) (n = 6, pval = 0.0143). H&E analysis of spleens isolated from 8-month-old wt and Er1Lyz2/− animals. The magnification is indicated (n = 6-7). (C.) Weights of spleens isolated from 8-month-old wt and Er1Lyz2/− animals (n = 8, pval = 0.0174). (D.) Flow cytometry analysis of live splenocytes. Gating sequence for analysis of plasma cells in wt and Er1Lyz2/− splenocytes. The plot shows the percentages of CD19+CD138+ plasma cells (n = 3, pval = 0.0293). (E.) Bar charts of the number of CD19+CD138+ plasma cells in the lymph nodes and bone marrows of 8-month-old mice (n = 4-5, pval = 0.0295). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01 (two-tailed Student's t-test). Flow cytometry analysis of bone marrows derived from 8-month-old wt and Er1Lyz2/− animals for the detection of myeloid granulocyte-monocyte progenitors. The scatter plots show the gating sequence and representative images. Gating sequence for the flow cytometry analysis of splenic macrophages and representative plots for the quantification of the MHC-II MFIs (n = 3, pval = 0.0474). (C.) MHC-II (left) and CD86 (right) protein expression levels in wt and Er1Lyz2/− splenic macrophages. The overlaying histograms represent the fluorescence intensities of wt, Er1Lyz2/− or isotype (iso) control samples (in the dashed black line, continuous black line and in gray color respectively). Mean fluorescence intensities are plotted in the bar graphs (n = 4-6, pval = 0.0283 and pval = 0.0261). (D.) Numbers of memory CD4+ T cells (CD44hiCD62Ll°) in wt and Er1Lyz2/− spleens (n = 5, pval = 0.026). (E.) Representative scatter plots and corresponding bar charts depicting the percentage of activated CD4+ T cells (CD44hiCD62Ll°) in 8- and 10-month-old wt and Er1Lyz2/− inguinal lymph nodes (n = 3, pval=0.031 and pval=0.0276). (F.) Plots showing the percentages of naïve CD4+ T cells (CD44l°CD62Lhi) in 2- and 10-month-old spleens (n = 4-6, pval = 0.0131). (G.) Representative scatter plots of CD44 and CD62L expression in 2- (young) and 24-month-old (aged) wt spleens (n = 4). (H.) Representative histogram plots of T-bet protein levels in CD4+ T cells after staining for T-bet and CD4 in 10-month-old wt and Er1Lyz2/− lymph node single cell suspensions (shown in the dashed black line and continuous black line respectively). Histograms overlaying the expression of ki67 protein in CD4+CD44+ cells in splenocytes isolated from 10-month-old wt and Er1Lyz2/− animals and stained for the ki67 proliferation, CD4 T cell and CD44 activation marker (shown in gray and in the continuous black line respectively) (n = 4, pval = 0.0487). Asterisk indicates the significance set at p-value: *≤0.05 (two-tailed Student's t-test). Schematic diagram showing the CD4+ T-cell transfer experiment. H&E staining of kidney paraffin sections derived from wt NSG hosts receiving wt or and Er1Lyz2/− CD4+ T cells. T cells were isolated from 8-10-month-old animals. The arrows show the focal inflammation observed in all the kidneys of mice receiving Er1Lyz2/− CD4+ T cells (n = 6). Representative images from all individual mice (left) and a higher magnification image (right, 400x magnification) are depicted. (C.) Gating sequence for the analysis of CD4+ T cells in kidney Percoll fractions. The bar plot shows the quantification of CD4+ T cell numbers localized in the kidneys of animals receiving wt or Er1Lyz2/− T cells (n = 5, pval = 0.0477). (D-E.) Representative plots of (D.) activated CD4+T cells and (E.) plasma cells in the spleens of animals receiving wt or Er1Lyz2/− BMDMs. Plasma cell percentages are quantified (n = 4, pval = 0.0433). Asterisk indicates the significance set at p-value: *≤0.05 (two-tailed Student's t-test). Immunofluorescence analysis for the detection of DNA damage levels in wt untreated, Er1Lyz2/− untreated and Er1Lyz2/− anti-oxidant-treated (mito-TEMPO and N-acetylcysteine (NAC)) BMDMs. The percentage of cells with 3 or more γ-H2A.X+53BP1+ co-localized foci is plotted (n = 3-4, pval = 0.0036, pval = 0.0204 and pval = 0.0093). Single-channel images of Extended Data Fig. 5A are shown in Supplementary file 2D(B.) Immunofluorescence analysis for the detection of 8-oxoguanine (8-oxoG) lesions in wt and Er1Lyz2/− BMDMs. The plot shows the MFI of 8-oxoG in each condition (n = 4, pval = 0.0086). (C.) Western blot analysis for the quantification of phosphorylated ATM and total ATM levels in whole-cell extracts. Beta tubulin was used for normalization (n = 4-5, pval = 0.0307). (D.) Gating strategy for the flow cytometry analysis of single cell suspensions of BMDMs and MHC-II levels in wt and Er1Lyz2/− untreated macrophages and Er1Lyz2/− cells treated with an ATR or a DNA-PK inhibitor (n = 3). (E.) Flow cytometry analysis of the quantification of γ-H2A.X levels in cells of monocytic origin isolated from lupus prone NZB/W 11-month-old mice (SLE) or from their age-matched controls (cont). An isotype control histogram is shown in gray (n = 4, pval = 0.0365). (F.) Quantification of the MHC-II levels in control, lupus prone untreated (SLE) and lupus monocyte-derived cells treated with an ATM kinase inhibitor (SLE+ATMi). The MFI in each corresponding condition is plotted (n = 5-6, pval = 0.0037 and pval = 0.0074). (G.) Histograms showing the expression of MHC-II protein in wt BMDMs treated with camptothecin (wtCPT, left), tunicamycin or underwent nutrient starvation (wttun and wtstarv respectively, right), as indicated, and their corresponding plots for their quantification (n = 3-4, pval = 0.0138). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01 (two-tailed Student's t-test). Immunofluorescence detection of the colocalized DNA damage markers γ-H2A.X and 53BP1 in wt untreated and wt etoposide-treated BMDMs. The percentage of positive cells for over 2 colocalized γ-H2A.X+53BP1+ foci/cell is plotted. (At least 4 independent optical fields were counted from n = 3 biological replicates, pval = 0.0018) Single-channel images of Extended Data Fig. 6A are shown in Supplementary file 3A(B.) Quantitation of the mRNA levels of chemokine genes in Er1Lyz2/− BMDMs, as shown. The mRNA levels of these genes in wt BMDMs are indicated with the red dotted line (n = 3-6, exact pvalue provided in Source Data file). (C.) (Top) Western blotting for the detection of interferon beta (IFN-β) protein levels secreted in the supernatants of wt and Er1Lyz2/− BMDMs. The splice point is indicated by a vertical black line. Equal volumes of concentrated supernatants were loaded on the gel. For the normalization of the secreted protein levels, equal volumes of cells lysed with RIPA buffer were loaded and the membranes were probed with beta tubulin. (Bottom) Blots for the quantification of interferon regulatory factor 5 (IRF5) and phosphorylated signal transducer and activator of transcription 1 (pSTAT1) versus total STAT1 protein levels. (D.) Western blot analysis of ERCC1 protein levels in CD4+ T cells purified from wt or Er1Lyz2/− mice at the age of 10 months (n = 3). (E.) Gating strategy for the flow cytometry analysis of CD4+ T cells in a BMDM-CD4+ T-cell co-culture. MHC-II levels exhibit a drop in the CD11b+ population (n = 3, pval = 0,0384). (H.) Immunofluorescence analysis of C3 complement protein localized in the kidney glomeruli of Er1Lyz2/− isotype control-treated and Er1Lyz2/− anti-MHC-II-treated animals. Representative images and plots with the MFI quantification of C3 are shown (n = 3, pval = 0.0343). H&E staining of kidneys derived from Er1Lyz2/− anti-CD4-and isotype control-treated animals. The arrows point to inflammatory foci (n = 6-8, pval = 0,0452). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01, ***≤0.01 (two-tailed Student's t-test). Immunofluorescence staining of γ-Η2A.X and 53BP1 in wt untreated, wt LPS-treated (wtLPS) and Er1Lyz2/− BMDMs. Cells with over 2 colocalized foci of the two proteins were labeled positive and indicated using white arrows. The percentage of positive cells is plotted. (At least 4 independent optical fields were counted from n = 3 biological replicates.) Single-channel images of Extended Data Fig. 7A are shown in Supplementary file 3B (B.) Flow cytometry analysis of the MHC-II expression levels in wt untreated, wtLPS and Er1Lyz2/− BMDMs. Representative histograms and MFIs are plotted (n = 3-5). (C.) Immunoprecipitation and western blot detection of the MHC-II protein in BMDMs using an anti-MHC-II (IP) or isotype control antibody (IgG). (D.) Volcano plot of differentially presented peptides in wt (downregulated, blue) and wtLPS cells (upregulated, red). Log2(Fold Change) of -6 or 6 represents peptides uniquely identified in the wt or wtLPS MHC-peptidome. Statistical significance (ANOVA analysis) was set at p-value ≤ 0.05 (horizontal black dashed line) and peptide enrichment at Log2(Fold Change)≥0.3 (upregulated in wtLPS) or Log2(Fold Change)≤-0.3 (downregulated in wtLPS) (vertical black dashed line). Extracellular matrix proteins are labeled with their corresponding gene symbol. (E.) Bubble plot of the Gene Ontology (GO) term enrichment analysis (cellular component, Mann-Whitney U test) of significantly over-represented wtLPS peptides, when compared to wt controls (p-value ≤ 0.05 and Log2Fold change≥0.3). The x axis indicates the fold enrichment derived from pathway analysis. (F.) IFN-γ ELISpot analysis of splenocytes isolated from either 8-month-old wt or Er1Lyz2/− mice and pulsed with the indicated peptides. Representative images are shown. Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01 (two-tailed Student's t-test). Cell cycle analysis of wt and Er1Lyz2/− BMDMs stained with DAPI-ki67. Western blotting of whole-cell extracts derived from wt and Er1Lyz2/− untreated cells and cells treated with ATM and ATR inhibitors for the quantification of Lamin B1. Actin was used as a loading control (n = 4, pval=0.0487 and pval= 0.0025). (C.) Immunofluorescence staining of γ-Η2AX and 53BP1 in Er1Lyz2/− untreated, 3-MA treated and CQ-treated BMDMs. Cells with over 2 colocalized foci of the two proteins were considered positive and the percentage of positive cells is plotted. (At least 5 independent optical fields were counted from n = 3 biological replicates.) Single-channel images of Extended Data Fig. 8C are shown in Supplementary file 3C. (D.) Immunofluorescence detection of cGAS in wt untreated, Er1Lyz2/− untreated and Er1Lyz2/− BMDMs that underwent FBS-starvation (Er1Lyz2/− +starv). (At least 4 independent optical fields were counted from n = 3 biological replicates, pval=0.0095 and pval=0.0069). Single-channel images of Extended Data Fig. 8D are shown in Supplementary file 3B(E.) Autophagic flux measurement after lentiviral infection of wt and Er1Lyz2/− BMDMs with a construct for the expression of LC3-GFP-mCherry. Green fluorescence corresponds to autophagosomes, while red fluorescence represents the autophagosomes and lysosomes in each cell. The graph shows the ratio of average red to green fluorescence per cell. (At least 3 independent optical fields were counted from n = 4 biological replicates, pval=0.0170) (F-G.) An overlay of the fluorescence intensity of (F.) LysoTracker Red (n = 5, pval=0.0289) and (G.) LysoSensor Green (n = 7-8, pval=0.0302) in Er1Lyz2/− BMDMs and wt corresponding controls, measured by flow cytometry analysis. Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01 (two-tailed Student's t-test). Immunofluorescence staining of early endosome antigen 1 (EEA1) in Er1Lyz2/− untreated and dynasore-treated BMDMs. White insets indicate a higher magnification on the right (n = 3). Single- or two-channel images and higher magnifications of Fig. Magenta arrowheads point to H1+p62+LAMP-1+ foci. White arrows point to cytoplasmic chromatin fragments. (C.) Flow cytometry analysis of wt and Er1Lyz2/− BMDMs stained with annexin V and propidium iodide (PI) for the detection of cell death. Scatter plots are and the percentages of annexin−PI− cells are shown (n = 4). (D.) LAMP-1 protein levels in equally loaded lysosomal extracts isolated from wt and Er1Lyz2/− BMDMs. Ponceau stain was used for normalization. (n = 4, pval=0.0147) (E.) Quantification of MHC-II in untreated and etoposide-treated THP-1 human monocyte cells. An isotype control histogram is shown in gray (n = 4, pval=0.0006). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01, ***≤0.01 (two-tailed Student's t-test). Western blotting for the detection of ERCC1 and ATG5 protein levels in wt and DKO (Ercc1fl/−; Atg5fl/fl; Lyz2-Cre+) BMDMs. The bar charts indicate the protein levels in the DKO cells while the red dotted line marks wt protein levels for comparison. GAPDH or actin were used for normalization (n = 4 pval<0,0001 and pval<0,0001). The splice point is indicated by a vertical black line (B.) H&E analysis of DKO spleens (n = 4). (C.) Quantification of macrophage numbers in the spleens of 8-month-old wt, Er1Lyz2/− and DKO mice (n = 4, pval = 0,0334). (D.) Detection of granulocyte-monocyte progenitors in the bone marrows of 8-month-old wt, Er1Lyz2/− and DKO mice. The red dotted line in the graph marks the mean percentage of GMPs in Er1Lyz2/− mice, comparable to the one in wt mice (the individual points of the measurements of wt and Er1Lyz2/− GMPs are shown in Supplementary Figure S3A) (n = 6). (E.) Representative scatter plots of the plasma cells in splenocytes isolated from 8-month-old wt, Er1Lyz2/− and DKO mice. Total numbers are shown in Fig. (F.) Flow cytometry analysis of wt and DKO BMDMs stained for annexin V and PI. The percentage of live (annexin−PI−) cells is plotted (n = 3). (G.) Immunofluorescence staining of cGAS in wt, Er1Lyz2/− and DKO BMDMs. (H.) IFN-γ ELISpot of 1:4 BMDM: CD4+ T cell co-cultures: Er1Lyz2/− or DKO BMDMs were mixed with Er1Lyz2/− CD4+ T cells. The numbers of IFN-γ spots are plotted (n = 4, pval = 0,0091). Asterisk indicates the significance set at p-value: *≤0.05, **≤0.01, ****<0.0001 (two-tailed Student's t-test). 1−4 and legends for Supplementary Figs. Peptides identified in immunopeptidomics analysis. The peptides that were classified as low or high binders of MHC-II from all samples were included for this analysis (NetMHCIIpan version 4.3, H-2-IAb and H-2-IAq alleles). Proteins identified in the autophagosomes of DMSO-treated or etoposide-treated U2-OS cells Peptides identified in immunopeptidomics analysis of DKO and WT control BMDMs. The peptides that were classified as low or high binders of MHC-II from all samples were included for this analysis (NetMHCIIpan version 4.3, H-2-IAb and H-2-IAq alleles). The peptides with fold change > 1.5, P < 0.05 and group profile ratio >0.85:1.15 or >1.15:0.85 were taken into consideration. Unprocessed western blots and/or gels for Supplementary Fig. Statistical Source Data for Supplementary Fig. Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Unprocessed western blots and/or gels Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Niotis, G., Arvanitaki, E.S., Theodorakis, E. et al. DNA damage in macrophages drives immune autoreactivity via nuclear antigen presentation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02196-7'>Investigating the methodological foundation of lesion network mapping</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 10:18:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Lesion network mapping (LNM) is a neuroimaging framework that uses normative functional connectivity (FC) data to link heterogeneous brain lesions and functional alterations to brain networks implicated in neurological and psychiatric conditions. However, many of the networks identified by LNM and related methods appear to be highly similar across diverse conditions such as addiction, depression, psychosis and epilepsy. To understand this similarity, we re-examined the data from multiple LNM studies and assessed the methodological roots of the method. Our findings reveal a foundational limitation: at its core, LNM involves a repetitive sampling of one and the same FC matrix. As a result, it systematically maps sets of local brain changes—whether they are patient lesions, magnetic resonance imaging-derived alterations, synthetic or random—onto the same nonspecific properties of the used FC data, producing highly similar networks across conditions. This central limitation cautions the use of LNM as a method for studying distinct biological networks underlying brain disorders. Our work may aid the development of a new generation of network-mapping methods from first principles. Identifying brain regions and circuits that give rise to neurological and psychiatric symptoms is a central goal of fundamental and clinical neuroscience. Charting the relationship between brain alterations and behavior has long served as a cornerstone of this effort, from linking brain injury to behavioral outcomes1,2,3 to systematic studies leveraging modern neuroimaging techniques4. Progress has, however, been more elusive for complex neurological and psychiatric conditions, where patients can often exhibit highly spatially distributed and heterogeneous brain abnormalities5,6,7. The method of ‘lesion network mapping' (LNM)8,9, also known in literature under alternative terms such as ‘causal brain mapping'10, ‘causal network localization'11, ‘lesion network-symptom mapping'12,13,14,15, ‘network localization'16,17, ‘atrophy network mapping'18, ‘remission network mapping'19, ‘coordinate network mapping' or ‘coordinate-based network mapping'20,21,22, ‘activation network mapping'23, ‘network-based meta-analytic' analysis24, among others (Supplementary Table 1), has rapidly gained traction as a framework to trace and unite topographically heterogenous lesions and other brain alterations to underlying brain circuits10,11,15. Collectively referred to as the LNM framework, this method maps the anatomical locations of brain alterations onto normative functional brain connectivity (FC) to examine whether, and if so how, these alterations converge onto a common underlying network. The framework posits that alterations in different brain regions can give rise to similar clinical symptoms when they disrupt the same functional brain network. Over the past years, LNM studies have reported such functional networks for a broad range of neurological and psychiatric disorders, including post-traumatic stress disorder (PTSD)25, epilepsy26,27, autism spectrum disorder (ASD)28, schizophrenia29, obsessive-compulsive disorder (OCD)30 and migraine20, among many others (see refs. 31,32,33 and a 2025 PubMed/ClinicalTrials.gov search for review; Supplementary Table 1 and Supplementary Note 1). Notable LNM findings include the ‘causal depression network'15,34,35,36, a ‘psychosis circuit'37 and brain circuits related to addiction38, all highlighted as promising for clinical application15,25,26,38,39,40. However, many of these reported LNM networks—purportedly delineated as disease-specific—seem to converge on strikingly similar brain networks. As illustrated in Fig. 1a,b, the LNM networks reported for psychiatric conditions such as addiction38, migraine20, PTSD25 and schizophrenia29, but also for neurological conditions such as vertigo41, Capgras syndrome42, Parkinson's disease43 and disrupted volition16, appear to implicate one and the same system, a network involving bilateral insular cortices, the anterior cingulate cortex (ACC) and parts of the frontopolar cortex, thalamus and cerebellum. This observation is unexpected, considering the substantial heterogeneity in etiology and symptomatology of these conditions. a,b, Images of LNM-related circuitry maps from recent LNM and sLNM publications (from refs. Panel a is reproduced with permission. c, Correlation between sLNM networks for reduced PTSD risk25 and cognitive decline induced by DBS in Parkinson's disease43 (shown in b). d, Recomputed LNM maps resulting from the application of voxel-wise Lead-DBS54 on publicly available lesions for addiction38, migraine20, neurogenic stuttering44, neglect syndrome53, insomnia53 and disrupted agency16. Reconstruction of LNM maps (d, first two images) compared to those reported in the original study (a) is high. e–g, Correlations between reconstructed LNM maps depicted in d are shown. h–j, Results show high similarity between LNM circuits derived from cortical deviations for six psychiatric conditions (BP and OCD are shown) and healthy controls; data taken from ref. 28. k, The most reported regions across 102 LNM networks from a literature survey (Supplementary Tables 1 and 2), highlighting the prevalence of the top 10% highest correlated and anticorrelated voxels. Extensive overlap is evident in the insula, ACC and frontal pole. l–n, LNM networks derived from random lesions also show highly similar LNM outcomes. For example, lesions that disrupted agency16 and spin-randomized versions of these lesions (middle row) across the brain, as well as completely randomized seed locations (bottom row), result in similar LNM outcomes (shown in n). o,q, Plot of the spatial correlation between the original LNM map (disrupted agency16) and a typical example from the randomized conditions. p,r, Randomization of lesions was repeated 1,000 times, with almost all occasions resulting in highly similar LNM maps between the original (disrupted agency) and random conditions (box plots show values of n = 1,000 permutations; (p) minima = 0.06, maxima = 0.92, center = (median) 0.75, bounds of box (Q1 25th percentile–Q3 75th percentile) = 0.66–0.81, whiskers = 0.43–0.92; (r) minima = 0.58, maxima = 0.96, center = (median) 0.84, bounds of box (Q1 25th percentile–Q3 75th percentile) = 0.81–0.87, whiskers = 0.72–0.96). s, The application of LNM (Lead-DBS) on lesions associated with addiction remission (top left, lesion masks taken from ref. The panel also shows LNM outputs on the same lesion set but now spin-randomized across the cortex (top right, exemplary spin, r = 0.48), following a random selection of 100 lesions with mixed symptomatology (bottom left, ‘mixed lesions', r = 0.93), and based on 100 synthetic lesions (bottom right, r = 0.71). All approaches yield very similar LNM maps. t–v, Plots show data (ASD28) from an alternative null analysis, with the connections of the group connectome C binarized and randomized (t, left = original matrix, right = randomized matrix). Once again, LNM analyses resulted in very similar maps. Plot in u shows a representative example (ASD) and v shows a box plot of all randomizations (box plot shows values of n = 1,000 permutations; minima = 0.93, maxima = 0.98, center = (median) 0.96, bounds of box (Q1 25th percentile–Q3 75th percentile) = 0.96–0.96, whiskers = 0.94–0.98). Examining this spatial overlap between published LNM networks in more detail substantiates the observed high spatial alignment. For example, published LNM networks for PTSD25 and cognitive decline in Parkinson's disease43 show high spatial correlation (r = 0.73; see Fig. 1b,c, Supplementary Note 2 and Supplementary Table 2 for data sources). Similar overlap is observed among networks for addiction38, migraine20, neurogenic stuttering44 and disrupted agency16 (r = 0.62–0.89; voxel-wise P < 0.001; Fig. This spatial alignment remains highly significant after correcting for spatial autocorrelation effects (spin test45 and BrainSMASH46; Pspin, Pbrainsmash < 0.001; r and P values for all examined networks are listed in Supplementary Table 3). Similar overlap is evident for LNM networks linked to aphasia47 and epilepsy27 (r = 0.40), amnesia48 and psychosis37 (r = 0.80), as well as for networks further linked to individual symptom data like networks related to risk of depression in multiple sclerosis34 and remission for smoking addiction38 (r = 0.57; all P, Pspin, Pbrainsmash < 0.001). LNM maps derived based on focal neurological lesions (for example, dyskinetic cerebral palsy49) or associated with deep brain stimulation (DBS)-related targets (for example, treatment for OCD50) also appear to show surprisingly high similarity (r = 0.64; P, Pspin, Pbrainsmash < 0.001; Supplementary Table 3). Remarkably, several of these LNM networks—for example, disruption of agency16 (Fig. 1l–n), ASD28, addiction38, but also epilepsy27 (Supplementary Fig. 8)—seem to be indistinguishable from networks derived when lesions are randomly shuffled across the brain (r = 0.73–0.95; Fig. 1l–r), derived from a mix of lesions not associated with one specific disorder (Fig. 1s), or even from completely random synthetic lesions (Fig. 1s and Supplementary Note 6). Also, randomizing the connections of the normative connectome dataset does not appear to markedly disrupt the LNM outcomes, resulting in rather similar networks (degree-preserving randomization51,52; for example, LNM for neglect syndrome53, r = 0.66, addiction38, r = 0.72, agency16, r = 0.75, and ASD28, r = 0.94, illustrated in Fig. The breadth of this spatial similarity is indicated by a literature survey, identifying 201 studies that discussed and/or used the LNM framework in context of studying 101 neurological and psychiatric conditions (2015–2025; see details in Supplementary Notes 1 and Supplementary Table 1). Re-analyzing 102 LNM networks across 72 of these studies confirmed an overall high alignment of LNM maps (|r| = 0.40, s.d. = 0.25; Supplementary Notes 2 and 3), with regions such as the bilateral insula, ACC and frontal cortex appearing in up to 74% of reported LNM networks (Fig. 1k; see Supplementary Note 5 for details). To explain this notable similarity among reported LNM networks, we examined the core principles of the method. Our systematic analysis reveals a fundamental limitation of LNM methods: LNM projects sets of lesions—regardless of their clinical association—onto only elementary properties of the standard connectivity matrix, primarily the row sum of that matrix (that is, node ‘degree'). Below, we provide a step-by-step walkthrough of the LNM pipeline, illustrating how its procedural stages can be expressed compactly in linear matrix notation. This formalization exposes the inherent constraint of the method that explains why the majority of published LNM networks converge to highly similar outcomes instead of identifying disorder-specific circuits. LNM (for methodologically equivalent variants and approaches published under different nomenclature, see Supplementary Table 1, from now on collectively referred to as LNM) typically consists of three methodological steps. Figure 2a presents a schematic of these steps, as implemented in popular LNM toolboxes like Lead-DBS54 (Supplementary Notes 8 and 17). We can consider a group of patients, each with one or more brain lesions, and study them using a large standard resting-state functional magnetic resonance imaging (fMRI) dataset from normative healthy individuals (for example, 1,000 healthy participants from the GSP1000 (ref. 55) or Human Connectome Project56). In step 1 of the LNM procedure, each lesion is mapped to corresponding voxels in the standardized space (for example, MNI152) of the normative dataset. Next, in step 2, the FC of a lesion is computed by correlating the average resting-state time series of the lesion's matching voxels with all other voxels in the brain and standardizing the correlation values using a Fisher r-to-z transformation. This is repeated across all healthy datasets in the normative connectivity dataset, resulting in over 1,000 FC maps per lesion, which are then combined into a single map using a one-sample t test to assess voxel-wise deviation from zero FC. A threshold (for example, |t| > 7) can be applied to identify the strongest connections57. Steps 1 and 2 are repeated for all studied lesions, producing a set of individual FC t maps, one for each lesion. a, The procedure of LNM involves three major steps—first, the lesion(s) of a single patient s (step 1) is placed into standard space. Next, the FC profile of that lesion ms of patient s is computed by means of the fMRI resting-state data in a large normative dataset, with the FC maps combined in a one-sample t test (two-sided) to obtain a single FC map for each lesion of patient s. Optionally, the t map can be thresholded to select the strongest connections (step 2). Steps 1 and 2 are repeated for all lesions of the group of patients S. Afterwards, the individual FC lesion maps are combined in a group analysis (step 3) to define their underlying common network. b, Step 2 of the LNM procedure can be streamlined (left, middle row) using an atlas-based approach in which the cortex and subcortical areas are parcellated according to a high-resolution atlas—for example, the Yeo-Schaefer1000/Melbourne54 atlas107,108. Middle, an atlas-based approach allows for precomputation of all lesion-to-region FC for all datasets in the normative connectome dataset. Right, all individual matrices can be grouped into a single group connectome C, with the resulting group matrix containing the same information as the one-sample t test performed in step 2. c, Taken together, the entire LNM procedure is now compressed to selecting row i corresponding to lesion ms of patient s from the group matrix C (optionally, threshold the resulting vector), repeat this for all lesions of all patients s in S, and summing over the selected rows Cm to obtain the final LNM network map. C, group connectivity matrix; GSP1000, Brain Genomics Superstruct Project 1000; h, normative participants; r, correlation coefficient; S, all participants. Next, in the group-analysis step 3 of the LNM procedure, the lesion FC t maps are combined to produce the group LNM network. This is typically done by averaging the lesion FC t maps, identifying regions consistently connected across lesions (for example, ≥75% (ref. The resulting map is referred to as the LNM network9 or LNM sensitivity map8. Alternatively, when individual symptom data are available, the group-analysis step 3 can involve correlating the lesion FC maps with symptom scores (~16% of reviewed studies; Supplementary Table 1) or contrast subgroups with differing symptom levels (~11%); variants of the method referred to as ‛lesion network-symptom mapping' or symptom-based LNM12,13,14,15. The sign of the resulting r values or t values in the symptom lesion network mapping (sLNM) depends on the behavioral scale that is used, and may indicate, for example, risk level11,25, symptom change15 or clinical state (for example, relapse versus remission)38. We found that the LNM methodological steps can be considerably compressed, without losing information. This compression is illustrated in Fig. 2b,c, and a mathematical derivation is provided in Supplementary Note 18. First, precomputing the correlation among the time series of all brain voxels yields all possible lesion-to-voxel FC maps beforehand. These precomputed matrices, for all normative participants in the normative connectivity dataset (H), can replace step 2 in the LNM approach (Fig. To improve practical feasibility, a high-resolution brain atlas can be used to divide the brain into, for example, R = 1,000 equally sized regions58. Furthermore, inferring equal variance across the connections in H (which we empirically validated, r = 0.99; Supplementary Note 8), the one-sample t test in step 2 can be replaced by taking the mean of the precomputed individual matrices54. This allows replacing the entire set of 1,000 normative FC matrices with a single mean group connectivity matrix C (Fig. This approach eliminates the need for looping the procedure over all normative datasets for each lesion, repetitively, reducing the computation time for a standard dataset of 50 lesions from ~10–12 h using the Lead-DBS toolbox54 to under 10 s. We empirically validated this compressed approach, with both the full Lead-DBS implementation and the atlas-based accelerated version producing effectively identical LNM maps (examined across 100 patient and 100 synthetic lesions, mean r = 0.96; Supplementary Notes 8 and 20). The compressed version (Fig. 2c) describes the LNM procedure now as: (step 1) matching lesion ms of participant s to the region(s) i in the used brain atlas; (step 2) selecting the matching row(s) i in the group connectivity matrix C; repeat steps 1–2 for all lesions; and (step 3 group analysis) taking the sum (or mean, which are equivalent) of all selected rows to obtain the final LNM map. Formally, we can express LNM as where S denotes the total set of patients, s one specific participant, ms the lesion of participant s, |ms| the size of lesion ms, i the row(s) in C matching the region(s) of lesion ms in participant s, C the group average functional matrix of size R × R, R all voxels or brain regions in the chosen brain mask or atlas and r a specific region in R (scaled with a fixed constant; for exact formal notation, see Supplementary Notes 8 and 18). We can also rewrite equation (1) in a vector notation: where \({{\bf{m}}_{s}}\) is a row vector of size 1 × R, indicating the lesion region with entries of 1 or 1/\(\left|{m}_{s}\right|\) when a lesion covers multiple rows, and 0 otherwise. We can now make one final compression—combining all lesion vectors \({{\bf{m}}}_{s}\) of all participants into a single lesion matrix M = (\({\vec{{\bf{m}}}}_{1}\), \({\vec{{\bf{m}}}}_{2}\), …,\({{\vec{\bf{m}}}}_{s}\)) (Fig. This summarizes the entire LNM procedure (steps 1, 2 and 3 combined) to a linear matrix multiplication: where M denotes the lesion matrix, C the standard group connectivity matrix. In the sLNM variant, the group-analysis step is slightly modified (illustrated in Supplementary Fig. In step 3, at each voxel, the FC values across the individual lesion maps (size S × 1) are further correlated with the participants' symptom scores (size S × 1), instead of taking the mean over all maps without further weighting. With steps 1 and 2 the same (and given by M × C, equation (3)), it can be obtained that the calculation of the final sLNM r map of all voxels in step 3 scales with: where M and C are again the lesion matrix and the normative group connectivity matrix, and \({{\bf{sv}}}\) now a standardized row vector describing the individual symptom scores (Supplementary Notes 9 and 19 provide a step-by-step and more formal derivation of sLNM). We provide exemplary code for the voxel-wise Lead-DBS implementation of LNM and sLNM, along with the equivalent linear matrix form of equations (3) and (4) in Supplementary Note 20. The above formal characterization brings to light a key limitation at the core of the LNM method, explaining the observed similarity between published networks (Fig. Specifically, the approach involves a repetitive sampling of one and the same matrix C, with the lesions M (and additionally the symptom scores sv in the sLNM variant) involving only linear operations on the input matrix. Let us consider two simple cases. First, for a single patient with exactly one unifocal lesion, applying LNM yields an intermediate tensor (equation (1)) of size S × M × R = 1 × 1 × 1,000. Averaging over lesions M of participants S (both equal to 1 here) results in an LNM brain map that mirrors row i of the input matrix C. Similarly, with five distinct lesions across five patients, LNM selects five rows from C, and the resulting LNM map corresponds to the sum or mean of those rows. Now consider a larger sample of S » 1 participants, each with a single lesion (Fig. For S = 1,000 with minimal spatial overlap between lesions, each lesion approximately corresponds to a unique region in the set of R = 1,000 regions, and thus to a unique row of C. It now emerges that step 2 of the LNM procedure involves selecting all rows of C, effectively reproducing the entire matrix. In the group-analysis step 3, the resulting LNM map contains the same information as the row-summation vector of the original connectivity matrix C. This convergence to the row-summation vector of C is even clearer when viewed in matrix notation (equation (3)). In this example, the lesion matrix M is the identity matrix I, leaving steps 1 and 2 as I × C, and the final group-analysis map as the row-summation vector of C (Fig. Visual illustration of how the method of LNM represents a matrix multiplication M × C. M is the lesion matrix containing the full lesion information across all participants S. Each row defines a unique lesion vector ms describing the brain region(s) affected by the lesion(s) of participant s (1) and which are not (0). C is the normative functional connectivity matrix of size R × R. The LNM procedure samples the corresponding rows of the normative matrix C. In the case of the number of lesions to approximate all regions of the brain, M becomes the identity matrix I, leading to the entire LNM procedure to copy C. After (optional) thresholding and summing across rows, the resulting LNM map equals the summation vector, or degree, of the normative connectome C. It is readily obtained that this alignment to degree will also occur when sets are smaller in size than R, with a uniform sampling of C approximating the degree of the matrix. Such convergence arises rapidly for any reasonably sized set of spatially heterogeneous lesions, which represent the typical input to LNM studies (Supplementary Table 1). When LNM (equation (3)) is applied to lesion sets of ≥10 spatially heterogeneous lesions, the resulting map already approximates the summation vector of C (Supplementary Fig. 2; r > 0.44, 10,000 runs, Pspin < 0.05). For sets of 20–25 heterogeneous lesions, a typical size for LNM studies (Supplementary Table 4), the correlation increases further quickly (r > 0.62; Supplementary Fig. 2), approaching the degree distribution of the input matrix for almost all spatially heterogeneous lesion sets. This systematic alignment with the summation vector of C also occurs when lesions exhibit substantial spatial overlap. Although most LNM studies focus on spatially heterogeneous lesion sets (for example, refs. 8,22,24,28,38; Supplementary Table 1), some have examined localized, overlapping lesions—for example, localized stroke or other lesion data linked to peduncular hallucinosis8, coma59, psychosis37, as well as spatially proximal transcranial magnetic stimulation (TMS) or DBS stimulation sites43,50,60. In these cases (with empirical examples reported below), the lesion vectors in matrix M contain duplicates or mark rows of C corresponding to spatially adjacent regions, resulting in the repeated selection of identical or highly similar rows. Consequently, the resulting LNM map still converges to the sum of the selected rows, primarily reflecting the inherent FC pattern of the underlying seed region(s). Even in the extreme case where all lesions fall within a single region, the probability that the LNM map reflects the degree structure of C remains non-negligible (|r| > 0.3, 74% of all possible cases; Supplementary Note 10). More formally, in such scenarios, the LNM map converges toward the sum of the row-induced subgraph Cm of C, that is, the sum of rows corresponding to the lesion regions (i, j, …, k). Variants like sLNM refine the LNM map using individual symptom scores, but they still fundamentally rely on information drawn from one and the same connectivity matrix C. The linear operation of a vector, such as the symptom/phenotype vector sv on a structured (formally, low-rank) matrix, will produce patterns of correlation r values that are shaped by the limited set of latent factors defining the matrix (we provide a more detailed explanation of this phenomenon together with examples in Supplementary Note 9). Consequently, sLNM maps based on a structured matrix, such as the FC matrix C, will align with the elementary properties of C. This leaves systematic traces in the sLNM map, most strongly aligned with the dominant latent factors of C (for example, PC1 of C, which overlaps with degree, |r| = 0.82), resulting in predictable sLNM outcomes regardless of whether the lesions or symptom scores are clinically informed or random (Supplementary Notes 9 and 20). We empirically tested the predicted systematic alignment of published LNM and sLNM maps to degree and other basic elementary properties (see below) of the normative functional connectome. We computed the row-summation vector of the group-average connectivity matrix of the GSP1000 dataset as used in Lead-DBS55. Then we correlated the resulting voxel-wise and atlas-based degree map with a series of reported LNM networks. Results support the prediction that LNM network maps strongly represent the summation vector of the normative connectome (multiple examples shown in Fig. For example, LNM networks presented for addiction38 (three conditions, r = 0.81/0.70/0.82), neglect syndrome53 (r = 0.70, Fig. 4m), disrupted agency16 (r = 0.59, Fig. 4p), bipolar disorder28 (r = 0.97) and OCD28 (r = 0.96) all show a strong association with the summation vector of C (P, Pspin, Pbrainsmash < 0.001; Supplementary Note 10; r and P values listed in Supplementary Table 4). a, Brain plots displaying the degree of the group average functional connectome in standard space, with warmer colors indicating regions of high degree. b–e, Same slices as in a for LNM maps for addiction38 (b), neurogenic stuttering44 (c), disrupted agency16 (d) and neglect syndrome53 (e). f–q, Correlations between functional degree of the normative connectome and (s)LNM networks (from left to right) for political involvement109 (f), aphasia62 (g), epilepsy27 (h), depression circuit in multiple sclerosis (MS-depression)34 (i), addiction38 (j), migraine20 (k), insomnia53 (l), neglect syndrome53 (m), disrupted agency16 (n), major depressive disorder (MDD)28 (o), schizophrenia (SCZ)28 (p) and neurogenic stuttering44 (q). Red dots represent voxels, black dots denote brain regions (atlas-based LNM). r,s, Systematic relationship between the first principal component (PC1) of the normative connectome and LNM maps derived from sLNM12, a variant of LNM in which lesion functional maps are further tuned by correlating them to individual symptom scores, for TMS target sites for depression64 (r), and DBS-related networks for cognitive decline in Parkinson's disease43 (s). t,u, Association between LNM maps and row sum of the matching subset of rows of the normative connectome C corresponding to the voxels (or regions) affected by the set of lesions (Cm) for psychosis37 (t) and amnesia48 (u). In f–u, spatial spin permutation (Main and Methods) was used to assess statistical significance (Pspin < 0.001, two-sided, n = 10,000 permutations, P values shown in Supplementary Table 4). Similarly, published network maps derived by means of sLNM and related variants, for example, networks reported from addiction38 (Fig. 4j, r = 0.63), risk for depression in multiple sclerosis34 (Fig. 4i, r = 0.44), or networks hypothesized to reduce anxiety and depression symptoms14 (r = 0.56) showed a significant trace of degree (P, Pspin, Pbrainsmash < 0.001; Supplementary Table 4) and more specifically the first principal component of C (|r| = 0.77–0.89; see also Supplementary Note 9). LNM networks derived from small, homogeneous and/or highly focal lesions can similarly exhibit strong traces of degree. Examples are found in the application of LNM derived on the basis of smaller lesion datasets (for example, aphasia62, n = 20, r = 0.74; migraine20, n = 11, r = 0.70; Fig. 4g,k; delusional misidentification63, n = 17, r = 0.65; Supplementary Table 4). The same holds for application of LNM and sLNM to DBS and TMS target sites43,64, where the stimulation sites are often highly localized within a radius of millimeters or centimeters (for example, TMS to DLPFC target sites to treat depression symptoms, r = 0.53, DBS related to cognitive decline in Parkinson's disease, r = 0.64, P, Pspin, Pbrainsmash < 0.001; PC1 |r| = 0.84/0.92; Fig. Similarly, traces of degree are present when considering LNM maps derived from lesions with considerable spatial overlap (psychosis37, ~30% lesions in midbrain, |r| = 0.52; amnesia48, ~50% lesions in the thalamus, |r| = 0.12), but with these maps even more strongly reflecting the row-summation vector of their selected row graphs Cm (r = 0.95–0.98; Fig. In total, of the 102 published LNM and sLNM networks we re-analyzed, 78 showed a significant trace of degree (Pspin < 0.05, 91 of 102 for Pbrainsmash < 0.05; Supplementary Table 4). Below, we will further discuss the (non)specificity of these LNM maps, showing that the spatial patterns of almost all LNM networks can be explained by means of the same elementary properties of the standard matrix C. The fundamental properties of functional connectome organization—for example, modularity58,65,66, hubs67, anticorrelation68, gradient structure69—constrain LNM and sLNM maps to reflect the network membership of lesion sites. The linear nature of LNM (equations (3) and (4)) implies repeated sampling of one and the same fixed matrix C, leaving lesion projections M, and joint symptom projections sv in the case of sLNM, inherently constrained to the principal subspace defined by C (Supplementary Note 9). Distributed lesions yield LNM maps that approximate the global degree sequence (as in many reported LNM networks; Fig. Conversely, clustered lesions cause the procedure to mirror the functional module or resting-state network(s) in which the lesions are located. Simulations confirm that >90% of generated FC lesion maps correlate with the canonical resting-state networks derived from modularity analysis of C (Methods), a pattern replicated in patient lesions and published circuits (r > 0.3; Supplementary Table 4). Moreover, the anticorrelated architecture of the connectome (that is, ~47% connections in the GSP1000 connectivity matrix are anticorrelated) ensures that LNM maps often show negative correlations with maps derived from lesions in opposing networks. Thus, patterns of anticorrelated LNM networks often interpreted as biologically meaningful in LNM studies14,70,71 are likely predictable consequences of the combined modular and anticorrelated structure of the standard connectome dataset. We examined the level of disease-specificity of LNM networks. On average, each LNM network showed a strong spatial overlap of |r| > 0.6 with 24 of the other 102 networks (P, Pspin < 0.05). This supports the very low—if not negligible—disease-specificity of LNM maps. This lack of specificity reflects the intrinsic nature of the LNM procedure. As we have seen above (equations (3) and (4)), LNM and sLNM repeatedly use the same low-rank matrix C, which limits the outcomes of the procedure to main patterns already present in C. To further illustrate this, we constructed a linear regression model that reflects nine basic factors describing the elementary properties of C—that is, its subcortical and cortical degree and its modular (n = 4) and functional gradient (n = 3) architecture69, core aspects of functional brain connectivity documented extensively in the field (for example, refs. Regressing LNM maps of which lesion data was available against this simple model showed that 93% (mean, s.d. = 5.0%) of the variance in LNM networks is explained by the basic properties of C (Supplementary Note 14). We found similar findings for published sLNM-derived networks (R2 = 79%, s.d. Any remaining variance falls well within the expected noise level of fMRI and LNM data74,75. These findings suggest that published disease LNM networks include no substantive information, other than unspecific signal already captured by global properties of functional connectome organization. LNM studies typically include statistical tests to support the sensitivity and specificity of presented networks16,32,76 (Supplementary Fig. We briefly discuss the validity and meaning of these statistical tests in light of the above observations, with an extended discussion in Supplementary Note 15. In the LNM approach, step 2 often involves the use of a one-sample t test to assess whether voxel-wise FC differs from zero. However, the large size of the normative dataset often leads to widespread significance77. For example, running 50 synthetic lesions in Lead-DBS shows that, on average, 64% voxels exceed the common |t| > 7 threshold. Moreover, this step contributes little additional statistical value. As shown above (Fig. 2c), the one-sample t test can be replaced altogether by simply taking the mean of the Fisher r-to-z-transformed correlations. The specificity test evaluates the disease- or condition-specificity of the examined LNM map (Supplementary Fig. This is typically conducted using a two-sample t test contrasting the derived map from a set of localized patient lesions with a set of random lesions drawn from other disorders32,76. Although this procedure appears to constitute an additional null test, it is largely redundant with the liberal sensitivity test. With LNM of random lesions to converge to the degree sequence of the normative matrix C (equation (3); Supplementary Fig. 2), the specificity test effectively re-assesses the same signal as the sensitivity test, but now relative to the matrix degree rather than zero. This highlights a lack of statistical independence between procedures intended to capture distinct information (for simulation analyses, see Supplementary Note 15). A commonly performed final test involves generating a conjunction or convergence map22,37,76, identifying voxels that pass both the sensitivity and specificity tests (Supplementary Fig. Given the relative ease of passing the sensitivity test and its interdependence with specificity, such conjunctions are easily obtained. We modeled ~500,000 lesions across all brain regions (Yeo-Schaefer1000/Melbourne54) with varying levels of overlap using standard LNM settings (sensitivity |t| > 7, G = 75%; specificity |t| > 10 (for example, ref. Marginal overlap between lesions (Dice = 0.08) resulted already in significant group results (10% sets), with minimum levels of overlap (Dice = 0.16) yielding 64% significant sets, increasing to almost all sets to reveal significant regions (97% tested sets) as spatial overlap increased (Dice > 0.25; Supplementary Fig. LNM has emerged as a widely used approach for identifying brain circuits linked to neurological and psychiatric conditions, as well as their symptoms9,10,17. Our analysis reveals a foundational limitation of the LNM framework—it maps circumscribed brain changes mostly to one and the same outcome, reflecting only elementary properties of the normative connectome. Given that these challenges arise from the LNM method rather than from extensive circuit-level findings in clinical neuroscience, this knowledge may aid the development of new network-mapping techniques. Our findings have broad implications for a wide range of existing work regarding disease networks and circuits derived by means of LNM, used here as an umbrella term that unifies various related terminologies and methods in literature ((for example, the method is also commonly applied under labels such as “atrophy network mapping,” “activation network mapping,” or “network-based meta-analytic” analysis; Main and see for an overview Supplementary Table 1). The current results suggest that a substantial proportion of the presented LNM networks are nonspecific and may not accurately reflect genuine biological brain networks. In practice, the LNM captures only a small set of factors that describe broad features of the input connectivity matrix and has limited ability to identify subtle disorder-specific properties. The convergence of LNM networks onto elemental properties of the connectome could be interpreted as support for the biological plausibility of LNM networks and circuits, like reflecting a transdiagnostic network underlying multiple disorders78. High-degree brain hubs66,67,79,80,81, for example, have been extensively theorized to have a central role in the pathophysiology of a wide range of disorders82,83,84,85. However, this interpretation in the context of LNM is misleading. The convergence of LNM methods and variants to the row sum of the used connectivity matrix (equation (3)) or, in more general terms, to the latent factors of C (Supplementary Notes 9,10 and 14), is purely a mathematical consequence of the procedure, not evidence of correspondence with the brain's hub, resting-state modular network or otherwise complex wiring architecture. Accordingly, when the empirical connectivity matrix is replaced by a randomized counterpart C′, or by other structured nonbiological matrices, the LNM outcome remains the product of the latent properties that govern C′ (Supplementary Fig. LNM is increasingly proposed as a framework to guide therapeutic applications of TMS and DBS14,25,37,39,40,50, with case studies performed86,87 and protocols for larger randomized controlled trials based on LNM networks registered (see Supplementary Table 1 for review). However, many such proposed targets—for example, the anticorrelated frontopolar cortex for substance use disorder38, or peak voxels to refine DBS target sites for epilepsy26,27—seem to primarily reflect the mean signal of the standard connectivity data, rather than identifying disease-specific loci. Indeed, the same regions emerge when LNM is applied across unrelated conditions11,25,27,44,48,59 (Supplementary Fig. 7) or just summing FC across all voxels in the GSP1000 dataset (Supplementary Fig. Given the clinical impact of these procedures, it appears essential to thoroughly reassess these targets before substituting traditional stimulation sites with demonstrated efficacy88,89,90. LNM studies have often been motivated by the observation that brain alterations in neuropsychiatric and neurological disorders are spatially diverse and heterogeneous—indeed, 55% studies describe them as such (Supplementary Table 1)22,23,24,27,28,29,91,92,93. This heterogeneity is frequently cited as a rationale for performing the LNM analysis, in search of an underlying common functional network that unites these brain alterations. Our findings propose a re-appreciation of disease heterogeneity, further studying how brain disorders may involve spatially distributed, heterogeneous alterations that converge on shared phenotypes6,94. A remaining question is whether the methodological limitations of LNM can be alleviated through refinements of its statistical procedures, for example by using random(ized) lesions or seed locations as a null-model. We approach such a solution with caution. The observation that almost all meaningful variance in LNM maps is explained by basic properties of the connectivity matrix suggests that deviations from a reference model or baseline are likely minimal, if they exist. Indeed, 70 of 78 LNM maps where lesion data were available failed to reach even a liberal significance criterion set by a generative null-model based on random synthetic lesions (nominal two-sided α = 0.05, uncorrected for 78,000 tests; Supplementary Note 16). A similar outcome was observed from a permutation-based null model in which lesion locations were randomly shuffled while preserving modular prevalence (71/78, PFDR > 0.05; Supplementary Note 16). We are also hesitant to propose a null-model solution for the LNM framework from a conceptual standpoint. Permutation-based null models estimate effects under random conditions by randomizing the input data. In LNM, only two variables exist at its core—M and C. With C describing the connectivity and remaining fixed in LNM studies and approaches8,9,17,24, the set of lesions (M) is left to be permuted. As predicted from equation (3), LNM on random sets of lesions consistently produces similar solutions dominated by degree (Supplementary Fig. This inherent limitation of the LNM framework hinders the construction of a null distribution that fulfils the essential criterion of spanning a meaningful range of alternative maps for a valid null test. The framework of network mapping has profoundly contributed to modern concepts of psychiatric and neurological disorders as network-based conditions83,95,96,97,98. LNM8,9,23,99 is proposed as a powerful and promising method within this framework to gain deeper insight into the mechanistic role of brain circuits in disorders9,10,11. Regrettably, our findings indicate that a substantial proportion of networks and disease circuits derived from LNM may not accurately reflect genuine disease-specific biological brain networks. However, it is crucial to separate the ‘theory' from the ‘method'. While we, and experts in the field with whom we discussed our findings, were not able to find an enduring solution to the foundational methodological issues of LNM, a continuous community effort to study the role of brain circuits in neurological and psychiatric disorders is imperative for advancing our understanding and developing new, effective treatments for these conditions. Linking deviations in brain organization to behavioral outcomes has long served as a cornerstone of this effort—from early clinical observations1 to systematic studies leveraging modern neuroimaging techniques4. Embedded in efforts collectively referred to as disease connectomics82,83,100, proposed fruitful future directions for the field may lie in revisiting the original rationale of lesion and ‘voxel-based lesion–symptom mapping' in context of connectivity mapping techniques1,2,3,101,102,103 to systematically chart how lesions impact brain circuitry and behavior. In parallel, network neuroscience72,79,104 offers the framework to more broadly investigate the central role of core network nodes in brain function and dysfunction79,82,83,84. Future efforts could focus in combining real patient lesions with in silico simulations to identify brain areas that may serve as general targets for intervention105,106. Such community efforts in revising the field of LNM from first principles may help ongoing work to develop brain network methods to map, understand and ultimately translate network-level approaches into clinical applications. A systematic literature search (December 2025) on LNM studies was performed. This identified 201 LNM studies, including 187 LNM data studies, 9 reviews and 5 commentaries with LNM, sLNM and/or other related variants the focus of the study, published between 2015 and 2025 (see Supplementary Note 1 and Supplementary Table 1 for details). From these articles, we extracted data on 102 published (s)LNM networks across 72 studies on 50 neurological, 18 psychiatric and 4 behavioral conditions (creativity, political, healthy, facial emotion), including 18 downloaded LNM maps, 11 datasets with reported lesion prevalence, 350 original lesion masks, 935 lesions manually segmented from original papers and 8 coordinate-based LNM (n = 1,442 brain coordinates). Details about the data extracted from these studies are presented in Supplementary Note 2 and Supplementary Table 2. Voxel-wise LNM was performed using the Lead-DBS toolbox54 (settings, FullSet of GSP1000 participants55; Supplementary Note 3 and Fig. Equivalently, atlas-based LNM involved mapping lesions to the 1,000 cortical regions of the Yeo-Schaefer1000 atlas107 and the Melbourne54 subcortical atlas108 and selecting the matching rows of the selected parcels from the group connectome matrix C (Fig. Spatial overlap of LNM maps was computed using Pearson correlation coefficients, using voxel-wise correlation for available voxel-wise maps and atlas-space for atlas-based maps. Significance was further assessed using the spin-null model45 and the BrainSMASH generative null model46 (10,000 permutations) to account for spatial autocorrelation effects. For 102 downloaded and reconstructed LNM maps (see Supplementary Table 2 for sources), the top 10% correlated and anticorrelated voxels were binarized and averaged to generate an overlap map of LNM regions across published studies. A group functional connectome matrix C was formed by mapping the same functional time series of the GSP1000 participant data as used in voxel-wise LNM54,55 (Supplementary Note 4) to the Yeo-Schaefer1000/Melbourne54 atlas and averaging the computed individual FC matrices into the group matrix (no thresholding). The summation vector of the group connectivity matrix C, or degree, was calculated as the row sum of the connectivity matrix C using the Brain Connectivity Toolbox110. Functional modules66, reflecting the composition of resting-state networks58,73, were identified using the Newman modularity algorithm110. A simple model describing the elementary factors of the connectome matrix was formed on the basis of the derived network metrics. Subcortical, whole-brain and modular degree were computed as the mean connectivity of matching regions from the brain atlas, together with three FC gradients69 taken as the first three components of a principal component analysis on C (Supplementary Note 13). Randomized lesions were generated by several randomization strategies, including spatially rotating the original cortical lesions across the cortical surface (spin-null permutation45). Alternative randomization methods included the generation of random synthetic lesions by taking random samples from the brain atlas (matching lesion size) and a biologically driven randomization that drew random lesions from the total collection of clinically informed lesions associated with a wide range of conditions and disorders (Supplementary Note 6). A randomized normative connectome matrix was generated by randomizing the connections in the connectivity matrix C (threshold r > 0.2, other thresholds yielded similar results) using the rewiring method described in refs. LNM maps were compared between the original connectome and its randomized counterparts (1,000 permutations performed; see Supplementary Note 7 for details). Alternatively, full degree-disrupting randomization of C was examined. Synthetic lesions were constructed by randomly selecting regions with equal probability P = 1/R from all regions in the atlas, dilated by including the closest neighboring parcels (n = 4), and for voxel-wise LNM further mapped to corresponding voxels in the MNI atlas volume. Lesion sets (n = 50) with varying levels of overlap between lesions were created by randomly selecting a first lesion with probability P = 1/R from all brain regions, with the second to nth lesion placed in that region as 1/R × q, and in all other brain regions with probability P ~ 1/R. As such, parameter q ensured a variable level of overlap between some of the lesions in the set, while all other lesions remained completely randomly distributed across the brain. The level of lesion overlap within each set as a function of q was quantified by means of the average Dice coefficient among all lesion pairs in the set, ranging from zero (no overlap) to one (complete overlap; see Supplementary Note 16 for details). We refer to Supplementary Note 16 for null-model simulations using synthetic lesions and randomizing patient lesions, preserving modular assignment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data used in the present study are publicly available. The preprocessed normative FC time series from the GSP1000 dataset are available at https://doi.org/10.7910/DVN/ILXIKS and in the Lead-DBS toolbox54. Neuroimaging data from the Human Connectome Project are available at https://www.humanconnectome.org. All LNM maps used in this study are available at https://neurovault.org and on GitHub (https://github.com/dutchconnectomelab/lesionnetworkmapping). Lesion masks associated with amnesia, hypersomnia, insomnia, neglect syndrome and Alice in Wonderland syndrome are available at https://www.lesionbank.org/. All other reported lesion or LNM data are directly available from the referenced papers. Voxel-wise LNM was applied using the open-source Lead-DBS toolbox54 (https://www.lead-dbs.org/). Spin-null permutation was conducted using the BrainSpace toolbox111. Network analysis of the normative connectome was performed using the Brain Connectivity Toolbox110. Steps of the streamlined LNM procedure, mathematical derivation, simplification of the method and example code are presented in Supplementary Notes 17–20 and at https://github.com/dutchconnectomelab/lesionnetworkmapping. Squire, L. R. The legacy of patient H.M. for neuroscience. Translation of Broca's 1865 report. Localization of speech in the third left frontal convolution. Roelofs, A. Wernicke's functional neuroanatomy model of language turns 150: what became of its psychological reflex arcs?. The history and future of resting-state functional magnetic resonance imaging. Selvaraj, S. et al. Grey matter differences in bipolar disorder: a meta-analysis of voxel-based morphometry studies. Muller, V. I. et al. Altered brain activity in unipolar depression revisited: meta-analyses of neuroimaging studies. Brugger, S. P. & Howes, O. D. Heterogeneity and homogeneity of regional brain structure in schizophrenia: a meta-analysis. Boes, A. D. et al. Network localization of neurological symptoms from focal brain lesions. Fox, M. D. Mapping symptoms to brain networks with the human connectome. Siddiqi, S. H., Kording, K. P., Parvizi, J. & Fox, M. D. Causal mapping of human brain function. Siddiqi, S. H. et al. Causal network localization of brain stimulation targets for trait anxiety. Preprint at Research Square https://doi.org/10.21203/rs.3.rs-4221074/v1 (2024). Wawrzyniak, M., Klingbeil, J., Zeller, D., Saur, D. & Classen, J. The neuronal network involved in self-attribution of an artificial hand: a lesion network-symptom-mapping study. Klingbeil, J., Wawrzyniak, M., Stockert, A., Karnath, H. O. & Saur, D. Hippocampal diaschisis contributes to anosognosia for hemiplegia: evidence from lesion network-symptom-mapping. Siddiqi, S. H. et al. Distinct symptom-specific treatment targets for circuit-based neuromodulation. Siddiqi, S. H. et al. Brain stimulation and brain lesions converge on common causal circuits in neuropsychiatric disease. Darby, R. R., Joutsa, J., Burke, M. J. & Fox, M. D. Lesion network localization of free will. Darby, R. R., Joutsa, J. & Fox, M. D. Network localization of heterogeneous neuroimaging findings. Tetreault, A. M. et al. Network localization of clinical, cognitive, and neuropsychiatric symptoms in Alzheimer's disease. Ji, G. et al. Distinct antidepressant therapies act on a common brain network. Preprint at Research Square https://doi.org/10.21203/rs.3.rs-6261102/v1 (2025). Burke, M. J. et al. Mapping migraine to a common brain network. Zhukovsky, P. et al. Coordinate-based network mapping of brain structure in major depressive disorder in younger and older adults: a systematic review and meta-analysis. Stubbs, J. L. et al. Heterogeneous neuroimaging findings across substance use disorders localize to a common brain network. Peng, S., Xu, P., Jiang, Y. & Gong, G. Activation network mapping for integration of heterogeneous fMRI findings. Cash, R. F. H., Müller, V. I., Fitzgerald, P. B., Eickhoff, S. B. & Zalesky, A. Altered brain activity in unipolar depression unveiled using connectomics. Siddiqi, S. H. et al. A potential target for noninvasive neuromodulation of PTSD symptoms derived from focal brain lesions in veterans. Schaper, F. et al. Mapping lesion-related epilepsy to a human brain network. Ji, G. J. et al. A generalized epilepsy network derived from brain abnormalities and deep brain stimulation. Segal, A. et al. Regional, circuit and network heterogeneity of brain abnormalities in psychiatric disorders. Makhlouf, A. T. et al. Heterogeneous patterns of brain atrophy in schizophrenia localize to a common brain network. Cotovio, G. et al. Obsessive-compulsive disorder secondary to focal brain lesions: from lesions to networks. Preprint at medRxiv https://doi.org/10.1101/2025.01.09.25320060 (2025). Webler, R. D. et al. An atlas and evidence-based appraisal framework of psychiatric brain stimulation targets generated by causal network mapping. Preprint at medRxiv https://doi.org/10.1101/2025.02.25.25322842 (2025). Joutsa, J., Corp, D. T. & Fox, M. D. Lesion network mapping for symptom localization: recent developments and future directions. Nabizadeh, F. & Aarabi, M. H. Functional and structural lesion network mapping in neurological and psychiatric disorders: a systematic review. Siddiqi, S. H. et al. Lesion network localization of depression in multiple sclerosis. Li, Y., Jin, Y., Wu, D. & Zhang, L. A depression network caused by brain tumours. Padmanabhan, J. L. et al. A human depression circuit derived from focal brain lesions. Pines, A. R. et al. Mapping lesions that cause psychosis to a human brain circuit and proposed stimulation target. Joutsa, J. et al. Brain lesions disrupting addiction map to a common human brain circuit. Horn, A. et al. Deep brain stimulation induced normalization of the human functional connectome in Parkinson's disease. Al-Fatly, B. et al. Connectivity profile of thalamic deep brain stimulation to effectively treat essential tremor. Li, Y. et al. A vertigo network derived from human brain lesions and brain stimulation. Darby, R. R. & Fox, M. D. Reply: Capgras syndrome: neuroanatomical assessment of brain MRI findings in an adolescent patient. Reich, M. M. et al. A brain network for deep brain stimulation induced cognitive decline in Parkinson's disease. Theys, C. et al. Localization of stuttering based on causal brain lesions. Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. B., Helmer, M., Shinn, M., Anticevic, A. & Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Souter, N. E. et al. Mapping lesion, structural disconnection, and functional disconnection to symptoms in semantic aphasia. Ferguson, M. A. et al. A human memory circuit derived from brain lesions causing amnesia. De Almeida Marcelino, A. L. et al. Lesion distribution and network mapping in dyskinetic cerebral palsy. Li, N. et al. A unified connectomic target for deep brain stimulation in obsessive-compulsive disorder. Maslov, S. & Sneppen, K. Specificity and stability in topology of protein networks. Rubinov, M. & Sporns, O. Complex network measures of brain connectivity: uses and interpretations. LesionBank: a platform for lesion network mapping analysis. A. Lead-DBS: a toolbox for deep brain stimulation electrode localizations and visualizations. Cohen, A., Soussand, L., McManus, P. & Fox, M. GSP1000 Preprocessed Connectome (Harvard Dataverse, 2020). Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. Cohen, A. L. & Fox, M. D. Reply: the influence of sample size and arbitrary statistical thresholds in lesion-network mapping. Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. Fischer, D. B. et al. A human brain network derived from coma-causing brainstem lesions. Fox, M. D. et al. Resting-state networks link invasive and noninvasive brain stimulation across diverse psychiatric and neurological diseases. Mollica, A. et al. The network-based underpinnings of persisting symptoms after concussion: a multimodal neuroimaging meta-analysis. Zarifkar, P. et al. Lesion network mapping of eye-opening apraxia. Darby, R. R., Laganiere, S., Pascual-Leone, A., Prasad, S. & Fox, M. D. Finding the imposter: brain connectivity of lesions causing delusional misidentifications. Fox, M. D., Buckner, R. L., White, M. P., Greicius, M. D. & Pascual-Leone, A. Efficacy of transcranial magnetic stimulation targets for depression is related to intrinsic functional connectivity with the subgenual cingulate. Damoiseaux, J. et al. Consistent resting-state networks across healthy subjects. & Betzel, R. F. Modular brain networks. Van den Heuvel, M. P. & Sporns, O. Network hubs in the human brain. Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Schaper, F. et al. Brain lesions causing parkinsonism versus seizures map to opposite brain networks. Trapp, N. T. et al. Large-scale lesion symptom mapping of depression identifies brain regions for risk and resilience. Van den Heuvel, M. P. & Hulshoff Pol, H. E. Exploring the brain network: a review on resting-state fMRI functional connectivity. Van den Heuvel, M., Mandl, R. & Hulshoff Pol, H. Normalized cut group clustering of resting-state fMRI data. Marek, S. et al. Reproducible brain-wide association studies require thousands of individuals. Helwegen, K., Libedinsky, I. & van den Heuvel, M. P. Statistical power in network neuroscience. Kim, N. Y. et al. Lesions causing hallucinations localize to one common brain network. Sperber, C. & Dadashi, A. The influence of sample size and arbitrary statistical thresholds in lesion-network mapping. Taylor, J. J. et al. A transdiagnostic network for psychiatric illness derived from atrophy and lesions. Bullmore, E. & Sporns, O. The economy of brain network organization. Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. & Petersen, S. E. Evidence for hubs in human functional brain networks. Tomasi, D. & Volkow, N. D. Functional connectivity hubs in the human brain. Van den Heuvel, M. P. & Sporns, O. A cross-disorder connectome landscape of brain dysconnectivity. Fornito, A., Zalesky, A. & Breakspear, M. The connectomics of brain disorders. Crossley, N. A. et al. The hubs of the human connectome are generally implicated in the anatomy of brain disorders. Buckner, R. L. et al. Cortical hubs revealed by intrinsic functional connectivity: mapping, assessment of stability, and relation to Alzheimer's disease. Raymond, N., Trotti, R., Oss, E. & Lizano, P. Lesion network guided neuromodulation to the extrastriate visual cortex in Charles Bonnet syndrome reduces visual hallucinations: a case study. Raymond, N., Trotti, R. & Lizano, P. Case report: lesion network guided transcranial direct current stimulation targeting treatment refractory hallucinations and delusions: a traditional and accelerated stimulation case study. Berlim, M. T., van den Eynde, F. & Jeff Daskalakis, Z. Clinically meaningful efficacy and acceptability of low-frequency repetitive transcranial magnetic stimulation (rTMS) for treating primary major depression: a meta-analysis of randomized, double-blind and sham-controlled trials. Perera, T. et al. The clinical TMS society consensus review and treatment recommendations for TMS therapy for major depressive disorder. Plewnia, C. et al. Theta burst stimulation of temporo-parietal cortex regions for the treatment of persistent auditory hallucinations: a multicentre, randomised, sham-controlled, triple-blind phase 3 trial in Germany. Peng, S. et al. Heterogenous brain activations across individuals localize to a common network. Wang, Y. et al. Heterogeneous brain abnormalities in schizophrenia converge on a common network associated with symptom remission. Kutsche, J. et al. Mapping neuroimaging findings of creativity and brain disease onto a common brain circuit. Reisinger, S. N. & Hannan, A. J. Developmental origins of mental health and disorders (DOMHaD): an approach to understanding, preventing and treating psychiatric disorders. Modern network science of neurological disorders. Pol, H. H. & Bullmore, E. Neural networks in psychiatry. Contributions and challenges for network models in cognitive neuroscience. Lydon-Staley, D. M. & Bassett, D. S. Network neuroscience: a framework for developing biomarkers in psychiatry. Jimenez-Marin, A. et al. Multimodal and multidomain lesion network mapping enhances prediction of sensorimotor behavior in stroke patients. Vogel, J. W. et al. Connectome-based modelling of neurodegenerative diseases: towards precision medicine and mechanistic insight. Bates, E. et al. Voxel-based lesion-symptom mapping. Corbetta, M. et al. Common behavioral clusters and subcortical anatomy in stroke. Thiebaut de Schotten, M., Foulon, C. & Nachev, P. Brain disconnections link structural connectivity with function and behaviour. Bassett, D. S. & Sporns, O. Deco, G. & Kringelbach, M. L. Great expectations: using whole-brain computational connectomics for understanding neuropsychiatric disorders. Naze, S. et al. Mechanisms and interventions promoting healthy frontostriatal dynamics in obsessive-compulsive disorder. Schaefer, A. et al. Local-global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Tian, Y., Margulies, D. S., Breakspear, M. & Zalesky, A. Topographic organization of the human subcortex unveiled with functional connectivity gradients. Siddiqi, S. H., Balters, S., Zamboni, G., Cohen-Zimerman, S. & Grafman, J. H. Effects of focal brain damage on political behaviour across different political ideologies. Rubinov, M., Kötter, R., Hagmann, P. & Sporns, O. Brain connectivity toolbox: a collection of complex network measurements and brain connectivity datasets. Vos de Wael, R. et al. BrainSpace: a toolbox for the analysis of macroscale gradients in neuroimaging and connectomics datasets. This study was supported by an ERC Consolidator grant from the European Research Council (101001062 CONNECT to M.P.v.d.H.) and an NWO VICI grant from the Netherlands Organization for Scientific Research (VI.C.241.074 BrainDiversity to M.P.v.d.H.). J.R. was supported by the LOEWE program of the Hessian Ministry of Science and Arts (grant LOEWE1/16/519/03/09.001(0009)/98) and by the Deutsche Forschungsgemeinschaft (DFG; German Research Foundation; grants 565437584 and 571864092). is supported by the Australian NHMRC (grant 2027597). The authors thank M. Gruber, R. Mandl, M. Benders and R. Brouwer for conceptual discussions and feedback. The authors thank the authors of papers we contacted, discussing the results and implications for published work. The authors also appreciate their time and respect the wishes of those who preferred to remain anonymous. Department of Neurosciences, Center for Neurogenomics and Cognitive Research, Amsterdam Neuroscience, Vrije Universiteit Amsterdam, Amsterdam, the Netherlands Martijn P. van den Heuvel, Ilan Libedinsky & Sebastian Quiroz Monnens Department of Child and Adolescent Psychiatry and Psychology, Amsterdam UMC location Vrije Universiteit Amsterdam, Amsterdam, the Netherlands Martijn P. van den Heuvel Department of Psychiatry, Psychosomatic Medicine and Psychotherapy, Goethe University Frankfurt, University Hospital, Frankfurt, Germany Institute for Translational Psychiatry, University of Münster, Münster, Germany Goethe University Frankfurt, Cooperative Brain Imaging Center—CoBIC, Frankfurt, Germany Center for Clinical Neuroscience and Cognition, University Medical Center Groningen, Groningen, the Netherlands Brain and Mental Health Program, QIMR Berghofer, Brisbane, Queensland, Australia School of Biomedical Sciences, University of Queensland, Brisbane, Queensland, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the study, performed analysis, interpreted the data, wrote the manuscript and Supplementary Information, designed and made the figures, and supervised the project. collected the data, performed analysis, wrote the Supplementary Information and provided feedback on the manuscript. performed analyses and made the figures. provided feedback on the analyses, results and manuscript. provided feedback and expertise on the analyses, interpretation of the data and results and wrote the manuscript. All authors discussed the results and implications and commented on the manuscript at all stages. Correspondence to Martijn P. van den Heuvel. has participated in a project as a data consultant for ROCHE (ROCHE had no role in this study) and is part of the editorial board of Wiley Human Brain Mapping (Wiley Human Brain Mapping had no role in this study). J.R. received speaker's honoraria from Janssen, Hexal, Neuraxpharm and Novartis (they had no role in this study). is involved in a clinical neuromodulation center, the Queensland Neurostimulation Centre (QNC, trading as the Australian Brain Foundation), which offers neuroimaging-guided neurotherapeutics; is not paid by QNC and this center had no role in the study; has served as a co-inventor on a patent application by the National University of Singapore covering neuroimaging-based personalized TMS; and is involved in the development of imaging-based personalized TMS for depression with ANT Neuro and Resonait. The provisional patent and products from ANT Neuro and Resonait are not directly related to this work. serves on the editorial boards of Wiley Human Brain Mapping and Elsevier NeuroImage: Clinical, neither of which had any role in the study. The other authors declare no competing interests. Nature Neuroscience thanks Janine Bijsterbosch and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–8 and Supplementary Tables 1–4. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions van den Heuvel, M.P., Libedinsky, I., Quiroz Monnens, S. et al. Investigating the methodological foundation of lesion network mapping. Version of record: 15 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02533-5'>Accounting for ocean impacts nearly doubles the social cost of carbon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 10:12:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Climate Change (2026)Cite this article Oceans provide essential benefits to people and the economy, underpinned by the extent and condition of marine ecosystems and infrastructure—or ‘blue' capital. However, the impacts of climate change on blue capital have been largely overlooked in influential indicators such as the social cost of carbon (SCC). Here we integrate the latest ocean science and economics into a climate-economy model, capturing climate change impacts on corals, mangroves, seaports, fisheries and mariculture to estimate their welfare repercussions at a global scale. Conceptually, this ocean-based SCC (blue SCC) represents a component of the total SCC currently omitted in standard estimates. We estimate the 2020 blue SCC to be US$48 per tCO2 (US$38–70, 25th–75th percentile) with baseline discounting, representing an almost doubling of the SCC estimate from the same model without considering ocean-related impacts. The blue SCC increases to US$168 for a discount rate of 2%. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The underlying analysis can be found at http://github.com/berbastien/blue-scc (https://doi.org/10.5281/zenodo.17649110). Source data are provided with this paper. Code for the blue SCC replication is provided at http://github.com/witch-team/RICE50xmodel/releases/tag/v2.6.0 (https://doi.org/10.5281/zenodo.17649366). The Blue Economy: 10 Years, 100 Innovations, 100 Million Jobs (Paradigm Publications, 2010). Urban, E. R. Jr & Ittekkot, V. (eds) Blue Economy: An Ocean Science Perspective (Springer, 2022). Wenhai, L. Successful blue economy examples with an emphasis on international perspectives. Halpern, B. S. et al. An index to assess the health and benefits of the global ocean. World Bank & United Nations Department of Economic and Social Affairs. The Potential of the Blue Economy—Increasing Long-Term Benefits of the Sustainable Use of Marine Resources for SIDS and Coastal Least Developed Countries (World Bank, 2017). Abraham, J. P. et al. A review of global ocean temperature observations: implications for ocean heat content estimates and climate change. Keeling, R. F., Körtzinger, A. & Gruber, N. Ocean deoxygenation in a warming world. Doney, S. C., Fabry, V. J., Feely, R. A. & Kleypas, J. Ocean acidification: the other CO2 problem. Bernier, N. B. et al. Storm surges and extreme sea levels: review, establishment of model intercomparison and coordination of surge climate projection efforts (SurgeMIP). Valuing Climate Changes: Updating Estimation of the Social Cost of Carbon Dioxide (National Academies Press, 2017); https://doi.org/10.17226/24651. Technical Support Document: Social Cost of Carbon, Methane and Nitrous Oxide Interim Estimates under Executive Order 13990 (Interagency Working Group on Social Cost of Greenhouse Gases, United States Government, 2021). Report on the Social Cost of Greenhouse Gases: Estimates Incorporating Recent Scientific Advances (US Environmental Protection Agency, 2023). Prest, B. C., Wingenroth, J. & Errickson, F. Challenges and opportunities for incorporating climate change's impacts on ocean systems into the social cost of greenhouse gases https://www.rff.org/documents/4601/Report_24-17_IX6Vq3m.pdf (Resources for the Future, 2024). Gazzotti, P. et al. Persistent inequality in economically optimal climate policies. Drupp, M. A. et al. Accounting for the increasing benefits from scarce ecosystems. Rennert, K. et al. Comprehensive evidence implies a higher social cost of CO2. Carleton, T. et al. Valuing the global mortality consequences of climate change accounting for adaptation costs and benefits. Rode, A. et al. Estimating a social cost of carbon for global energy consumption. Hultgren, A. et al. Impacts of climate change on global agriculture accounting for adaptation. Drupp, M. A., Freeman, M. C., Groom, B. & Nesje, F. Discounting disentangled. Anthoff, D., Estrada, F. & Tol, R. S. J. Shutting down the thermohaline circulation. Schaumann, F. & Alastrué De Asenjo, E. Weakening AMOC reduces ocean carbon uptake and increases the social cost of carbon. Unequal climate impacts on global values of natural capital. Social cost of carbon estimates have increased over time. Burke, M. et al. Opportunities for advances in climate change economics. Wagner, G. et al. Eight priorities for calculating the social cost of carbon. Sully, S., Hodgson, G. & van Woesik, R. Present and future bright and dark spots for coral reefs through climate change. A. et al. A warming ocean threatens mangrove restoration targets and deepens global inequities in ecosystem service losses. Verschuur, J., Koks, E. E. & Hall, J. W. Systemic risks from climate-related disruptions at ports. A. et al. Modelling the effects of climate change on the distribution and production of marine fishes: accounting for trophic interactions in a dynamic bioclimate envelope model. Cheung, W. W. L. et al. Climate change exacerbates nutrient disparities from seafood. Cornwall, C. E. et al. Global declines in coral reef calcium carbonate production under ocean acidification and warming. Ricke, K. L., Orr, J. C., Schneider, K. & Caldeira, K. Risks to coral reefs from ocean carbonate chemistry changes in recent earth system model projections. Drupp, M. A., Turk, Z. M., Groom, B. Limited substitutability, relative price changes and the uplifting of public natural capital values. Preprint at https://arxiv.org/abs/2308.04400 (2024). & Hänsel, M. C. Relative prices and climate policy: how the scarcity of nonmarket goods drives policy evaluation. & Moore, F. C. Use and non-use value of nature and the social cost of carbon. Carbone, J. C. & Smith, V. K. Valuing nature in a general equilibrium. Bertram, C. et al. The blue carbon wealth of nations. Kim, H. et al. Towards a better future for biodiversity and people: modelling Nature Futures. Gauvreau, A. M., Lepofsky, D., Rutherford, M. & Reid, M. ‘Everything revolves around the herring': the Heiltsuk-herring relationship through time. Pereira, O. S., Jacobsen, M., Carson, R., Cortés, J. Understanding and valuing human connections to deep-sea methane seeps off Costa Rica. Levin, L. A. et al. Deep-sea impacts of climate interventions. National Academies of Sciences, Engineering, and Medicine; Division on Earth and Life Studies; Ocean Studies Board; Committee on A Research Strategy for Ocean-based Carbon Dioxide Removal and Sequestration. A Research Strategy for Ocean-Based Carbon Dioxide Removal and Sequestration (National Academies Press, 2021). Massicotte, P. & South, A. rnaturalearth: World map data from Natural Earth. R package version 1.1.0.9000 https://docs.ropensci.org/rnaturalearth/ (2026). Coral reefs of the world (1:10,000,000 scale) https://databasin.org/datasets/957dde289e764bf69d9c69e3fa8890d6/ (Conservation Biology Institute, 2011). Bunting, P. et al. Global Mangrove Watch (1996–2020) Version 3.0 Dataset (3.0). Flanders Marine Institute. Maritime Boundaries Geodatabase, version 12 https://www.marineregions.org/.10.14284/628 (2023). Verschuur, J., Koks, E. E. & Hall, J. W. Ports' criticality in international trade and global supply-chains. Leach, N. J. et al. FaIRv2.0.0: a generalized impulse response model for climate uncertainty and future scenario exploration. Nordhaus, W. D. & Yang, Z. A regional dynamic general-equilibrium model of alternative climate-change strategies. Limits to substitution between ecosystem services and manufactured goods and implications for social discounting. Hoel, M. & Sterner, T. Discounting and relative prices. Brander, L. M. et al. Economic values for ecosystem services: a global synthesis and way forward. Guidelines for Preparing Economic Analyses (US Environmental Protection Agency, 2014). Free, C. M. et al. Realistic fisheries management reforms could mitigate the impacts of climate change in most countries. PLoS ONE 15, e0224347 (2020). & Sumaila, U. R. Economic impact of ocean fish populations in the global fishery. Prest, B. C., Rennels, L., Errickson, F. & Anthoff, D. Equity weighting increases the social cost of carbon. Download references We are grateful to S. Sandin and his laboratory members at Scripps Institution of Oceanography for invaluable discussions on the climate impacts on corals. Our thanks also go to Angie Creativist Studio for their assistance in refining the visual elements, and to S. Guilford from the National Geographic Society for his help in enhancing the maps presented in this Article. thanks K. Scherrer, J. Guiet and D. Bianchi for enlightening conversations on the climate impacts on fisheries, and B. Prest and J. Wingenroth for crafting thoughtful spaces to discuss the role of oceans in the SCC. acknowledges financial support from the Institutional Fellowship of Scripps Institution of Oceanography that made this research possible. acknowledge funding from the European Union's Horizon Europe research and innovation programme under grant agreements no. 101081369 (SPARCCLE) and no. acknowledges the support of the Edward A. Frieman Endowed Presidential Chair in Climate Sustainability. Scripps Institution of Oceanography, University of California San Diego, La Jolla, CA, USA Bernardo A. Bastien-Olvera, Octavio Aburto-Oropeza & Katharine Ricke Instituto de Ciencias de la Atmósfera y Cambio Climático, Universidad Nacional Autónoma de México, Mexico City, Mexico CMCC Foundation – Euro-Mediterranean Center on Climate Change, Milan, Italy Bernardo A. Bastien-Olvera, Johannes Emmerling, Francesco Granella & Massimo Tavoni RFF-CMCC European Institute on Economics and the Environment, Milan, Italy Bernardo A. Bastien-Olvera, Johannes Emmerling, Francesco Granella & Massimo Tavoni Programa de Investigación en Cambio Climático, Universidad Nacional Autónoma de México, Mexico City, Mexico Institute of Earth System Sciences (IESW), Leibniz Universität Hannover, Hannover, Germany Institute for Environmental Studies, Vrije Universiteit Amsterdam, Amsterdam, The Netherlands Institute for the Oceans and Fisheries, The University of British Columbia, Vancouver, British Columbia, Canada Marine Science Institute, University of California Santa Barbara, Santa Barbara, CA, USA Bren School of Environmental Science and Management, University of California Santa Barbara, Santa Barbara, CA, USA Department of Management, Economics and Industrial Engineering, Politecnico di Milano, Milan, Italy Faculty of Technology, Policy and Management, Delft University of Technology, Delft, Netherlands Oxford Programme for Sustainable Infrastructure Systems, University of Oxford, Oxford, UK School of Global Policy and Strategy, University of California San Diego, La Jolla, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Except for the first and last authors, all other authors are listed in alphabetical order. provided conceptualization, investigation, methodology, visualization and formal analysis, and contributed to writing the original draft. contributed to methodology, visualization, formal analysis and software. contributed to methodology, supervision and resources. contributed to methodology and supervision, writing of the original draft, resources and funding acquisition. All authors contributed equally to data curation, validation, and review and editing. Correspondence to Bernardo A. Bastien-Olvera. The authors declare no competing interests. Nature Climate Change thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Estimated Blue SCC across five SSPs, reflecting different trajectories of socioeconomic development and associated emissions. Each SSP results in a different increase in global mean temperature by 2100 relative to pre-industrial levels. End-of-century global mean surface temperature increases are as follows: SSP1: 2.60 °C, SSP2: 3.58 °C, SSP3: 4.18 °C, SSP4: 3.82 °C, SSP5: 3.95 °C. Mean PAWN index values and 95% confidence intervals showing the influence of each input parameter on the blue SCC, based on global sensitivity analysis with bootstrapping. The elasticity of substitution between blue capital components (θ) is the most influential parameter. Other important drivers include the non-use value of corals, the fraction of non-substitutable nutrients from fisheries, and the Value of a Statistical Life (VSL). Left: Percentage increase in the SCC when adding ocean-based damages to three alternative baseline damage functions in RICE50 + . Right: Blue SCC estimates using alternative pure rate of time preference (prtp) and elasticity of marginal utility of consumption (emuc) pairs calibrated to near-term discount rates of 1.5–3%. Supplementary Discussion and Figs. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Bastien-Olvera, B.A., Aburto-Oropeza, O., Brander, L.M. et al. Accounting for ocean impacts nearly doubles the social cost of carbon. Download citation Version of record: 15 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Climate Change © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260114084109.htm'>How everyday foam reveals the secret logic of artificial intelligence</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-15 05:22:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Foams appear in everyday life as soap suds, shaving cream, whipped toppings and food emulsions like mayonnaise. For many years, scientists believed foams behaved much like glass, with their tiny components locked into disordered but essentially fixed positions. It may even help scientists better understand living structures that must continually reorganize themselves, such as the internal scaffolding of cells. Instead of eventually becoming stationary, the bubbles kept wandering through many possible arrangements. From a mathematical viewpoint, this behavior closely resembles how deep learning works. During training, an AI system repeatedly adjusts its parameters -- the information that defines what an AI "knows" -- rather than locking into a single final state. "It's striking that foams and modern AI systems appear to follow the same mathematical principles. Understanding why that happens is still an open question, but it could reshape how we think about adaptive materials and even living systems." They generally keep their shape and can spring back after being squeezed. At much smaller scales, however, foams are considered "two-phase" materials, made of bubbles suspended in a liquid or solid background. Because foams are easy to make and observe while still displaying complex mechanical behavior, scientists have long used them as model systems to study other dense and dynamic materials, including living cells. Traditional theories treated foam bubbles like rocks rolling across an energy landscape. In this view, bubbles move downhill into positions that require less energy to maintain, then stay there. When researchers examined real foam data, they found the behavior did not align with these predictions. According to Crocker, signs of this mismatch appeared nearly two decades ago, but there were no suitable mathematical tools to fully explain what was happening. Modern AI systems learn by continuously adjusting numerical parameters during training. Early approaches tried to push these systems toward a single optimal solution that perfectly matched their training data. Deep learning relies on optimization methods related to a mathematical technique called gradient descent. Over time, researchers realized that pushing models too far into the deepest possible solutions caused problems. Systems that fit their training data too precisely became fragile and performed poorly on new information. "Keeping it in flatter parts of the landscape, where lots of solutions perform similarly well, turns out to be what allows these models to generalize." Instead, they continue moving within broad regions where many configurations are equally viable. This ongoing motion closely parallels how modern AI systems operate during learning. The same mathematics that helps explain why deep learning works also captures what foams have been doing all along. The findings raise new questions in a field many believed was already well understood. That alone may be one of the study's most important contributions. By showing that foam bubbles are not frozen in glass-like states but instead move in ways similar to learning algorithms, the research encourages scientists to rethink how other complex systems behave. Like foam, the cytoskeleton must continually reorganize while preserving its overall structure. "Why the mathematics of deep learning accurately characterizes foams is a fascinating question," says Crocker. Scientists May Have Finally Found the “Holy Grail” of Sugar Substitutes Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            