
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-01</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65950288/archaeologists-find-317-skeletons-buried-under-department-store/'>Archaeologists Found 317 Skeletons Buried Under a Department Store</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 16:57:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The remains of 317 individuals from medieval and post-medieval burial grounds were discovered by archaeologists from Cotswold Archaeology in Kings Square, Gloucester, as the site was being redeveloped into the University of Gloucester's City Campus. “Every time we work in Gloucester, we make new discoveries,” Cliff Bateman, Cotswold Archaeology Senior Project Officer, said in a press release. Many earlier artifacts from the Roman period also surfaced, which makes sense, considering that what is now King's Square is thought to have once been the northeast quadrant of an ancient Roman town. Other recent finds include brick burial vaults and a crypt from St. Aldate's Church—the external wall and porch of which appeared when remodeling efforts started. The later St. Aldate's stood until 1960, but has long since been demolished. Evidence of its medieval predecessor has not yet appeared, but is thought to be in the area. The bones of twelve individuals were only exhumed for research before being reinterred in their original graves. Bateman also said that he is just about positive “there will be Roman buildings in situ” beneath the post-medieval necropolis. This isn't surprising to archaeologists, who first started finding mosaics and ruins of Roman buildings in the basement of the empty Debenham's. Preliminary studies on the teeth of skeletons have found that the people they came from likely consumed a diet high in sugar. Excavations in another part of the city previously revealed even more skeletons, this time from the Late Roman period, which were buried both on their backs and facedown. “These objects have been retained on site, following archaeological recording, and will be displayed on site for students, staff and visitors to City Campus to appreciate once the site is fully operational,” Steve Sheldon, Acting Principal Manager of Cotswold Archaeology, said in a more recent press release. There could be no more epic way to kick off an ancient history class. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. We May Have Just Found a Hidden Shipwreck Gut Feelings Are Memories From the Future, Scientists Say An Expert Just Found a Coin With Huge Implications</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02735-z'>How ageing changes our genes — huge epigenetic atlas gives clearest picture yet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 15:41:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Chris Simms is a science writer based in Somerset, UK. A huge analysis of DNA methylation in human tissue has revealed new anti-ageing targets.Credit: Lawrence Lawry/SPL The study assessed DNA methylation patterns in human tissue samples and revealed that some tissues seem to age faster than others. The analysis also found universal epigenetic markers of ageing across different organs. This ‘epigenetic atlas' might help researchers to study the link between DNA methylation and ageing and could aid the identification of molecular targets for anti-ageing treatments. “I think this is a great resource” to understand ageing, says Joao Pedro Magalhaes, a molecular biologist at the University of Birmingham, UK. I am sure that it will be valuable to researchers.” Researchers can already analyse DNA methylation patterns in people's genomes to create ageing clocks — tools that measure biological age. To elucidate how methylation relates to ageing, Nir Eynon at Monash University in Melbourne, Australia, and his colleagues conducted a meta-analysis of more than 15,000 samples from 17 human tissues taken from adults of different ages. They mapped out methylation changes across 900,000 potential sites in the DNA, then created an open-access atlas. “We had examples from people from 18 years old till 100 or so,” says Eynon, so we can look at the epigenetic markers and how they change across the human lifespan. Study co-author Macsue Jacques, also at Monash University, says almost all tissues have increased DNA methylation as they age. Their analysis also found that different organs have distinct ageing patterns of DNA methylation. “Each tissue has a different shift that happens,” Jacques says. “We wanted to find a common ageing mechanism that goes across all the tissue types,” says Jacques. An epigenetics gold rush: new controls for gene expression These genes can have the opposite effects depending on which parent they came from Parent-of-origin effects on complex traits in up to 236,781 individuals A male-essential miRNA is key for avian sex chromosome dosage compensation Spouses tend to share psychiatric disorders, massive study finds 20 Postdoctoral positions available at IChF PAS, funded by the Horizon Europe MSCA-Cofund programme. Seeking exceptional Senior/Junior PIs, Postdocs, and Core Specialists globally year-round The Director is initially appointed for a three-year term, renewable for a second term, and concurrently holds the title of Research Fellow. An epigenetics gold rush: new controls for gene expression Human body's ageing ‘clock' ticks faster after heat stress An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63335-z'>Catalytic aromatic C−H borylation via strained Si and Ge metallacycles</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 13:12:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 8163 (2025) Cite this article Arenes bearing silyl and boryl, or germyl and boryl groups, represent a significant class of bifunctional molecules that have garnered considerable attention in organic synthesis due to their unique reactivity and versatility in undergoing diverse chemical transformations. Despite their importance, the synthesis of these mixed-metalloid compounds, particularly those with ortho-substituted aromatic frameworks, often involves complex, multi-step procedures. Herein, we present a highly efficient, single-step method for synthesizing ortho-boronated arylhydrosilanes and arylgermanes via catalytic C−H borylation reaction. The use of Si- and Ge-containing directing groups has been demonstrated to effectively suppress undesirable metathesis reactions, a side effect typically associated with their metallic nature. Our approach employs an iridium catalyst in combination with a ligand and an acetate base, which facilitates the formation of benzo-fused four-membered metallacycles within the catalytic cycle. The method's broad applicability and practicality are further underscored by its ability to generate a wide array of valuable intermediates that can be easily transformed into various functionalized molecules. Both experimental and computational studies provide profound insights into the reaction dynamics and the factors influencing the formation of these strained metallacycles. Catalytic C−H activation has emerged as a powerful strategy for modifying carbon centers in organic molecules, offering atom-efficient and streamlined synthetic pathways1,2,3,4,5,6,7,8,9,10. In complex molecules where multiple C–H bonds often reside in similar environments, the incorporation of a directing group has proven to be the most effective method for achieving selective transformation (Fig. These directing groups, whether inherently present or artificially introduced, serve to precisely position the metal catalyst at the desired C–H bond. Such process is thermodynamically favored due to the formation of stable five-, six-, and seven-membered metallacyclic intermediates, which ensure regioselectivity through conformational rigidity. Non-metallic elements, particularly those from the chalcogen (O, S) and pnictogen (N, P) groups, are widely utilized as coordinating atoms in directing groups. In contrast, the integration of directing groups containing metalloid elements has seen limited progress, primarily due to their propensity to induce undesirable metathesis reactions in the presence of transition metals, a consequence of their metallic character17. Although some success has been achieved with silicon atoms, current methods are only confined to specific substrate classes that form five-membered metallacyclic intermediates, significantly limiting their broader applicability18,19,20. Consequently, there is a pressing need to develop more versatile C−H activation strategies that can accommodate a wider range of metalloid elements while moving beyond the conventional constraints of rigid metallacycle formation. a General modes for directed C−H activation. b Precedent discovery of strained silametallacycles. c Pd-catalyzed β-selective C–H activation of aliphatic amines. d Metalloid Si/Ge-directed C−H borylation (this work). Arenes ortho-bisfunctionalized with silyl and boryl, or germyl and boryl groups, have been extensively utilized in photoelectronics, semiconductors, catalysis, and organic synthesis due to the unique properties of these distinct metalloids21,22,23,24,25,26,27. However, current synthetic methods for these compounds often require intricate, multi-step processes, particularly when forming two C−metalloid bonds individually28,29,30,31. Seminal studies by Milstein32,33, Tilley, and Bergman34 have demonstrated that iridium complexes derived from triphenylsilane can produce benzo-fused four-membered silametallacycles through oxidative addition of ortho C−H bonds (Fig. Building on these pioneering works, we are inspired to explore three key questions: whether such strained metallacycles can be generated in a catalytic fashion, whether they can further couple with boron species, and whether the metalloid elements can be extended to germanium atoms. Notably, Gaunt and coworkers have reported palladium-catalyzed β-selective C−H activation of aliphatic amines, predominantly achieved via the formation of four-membered metallacycles (Fig. In contrast, catalytic C−H borylation leading to the formation of benzo-fused metallacycles introduces even greater ring strain and a metalloid motif that is more susceptible to uncharted side reactions37,38,39,40,41. Here, we introduce an ortho-selective C−H borylation directed by silicon and germanium atoms (Fig. Our system, which utilizes arylhydrosilanes or arylgermanes as substrates, combines an iridium catalyst with a bipyridine-type ligand, an acetate base, and HBpin to efficiently facilitate the formation of a benzo-fused four-membered metallacycle within the catalytic cycle. Importantly, this methodology effectively overcomes the common issue of byproduct formations from C−metalloid cleavage26,42,43,44,45 and metalloid−H bond cleavage46,47,48,49,50. This approach enables the one-step preparation of a diverse array of ortho-boronated arylhydrosilanes and arylgermanes, offering superior efficiency in terms of atom, step and redox economy. The ability to selectively functionalize these aryl compounds with boron atoms while maintaining the integrity of the metalloid atoms represents a significant advancement in the field of C−H activation. To establish a platform for this chemical process, we initially opted for the reaction between triphenylsilane (1a) and HBpin (Table 1). The reaction was employed [Ir(cod)OMe]2 as the catalyst along with a tridentate ligand L1 at a temperature of 125 °C in THF. However, merely trace amounts of the desired C–H borylation product 2a were observed via GC-MS analysis (entry 1). To enhance the overall yield of the reaction, we explored the influence of various additives. The incorporation of LiOAc into the reaction system could slightly improve the reactivity (entry 2). Additionally, we examined alternative salts such as NaOAc (19%) and KOAc (26%), which exhibited superior effectiveness in facilitating the formation of 2a (entries 3–4). Conversely, the utilization of K2CO3 resulted in a significant decrease in yield (entry 5). These discoveries underline the significance of both the K+ cation and OAc- anion in influencing the overall outcome of the reaction. Moreover, we investigated the impact of different ligands on the reaction. The inclusion of bipyridine-type ligands L2–L4 proved to be crucial in promoting the reaction (entries 6–8), with the highest achievable yield of 2a (75%) obtained in the presence of L4 (entry 8). Under the optimized reaction conditions, we observed the absence of diborylated products, with Ph-Bpin emerging as the primary byproduct, presumably resulting from C-Si bond cleavage. The structure of product 2a was further confirmed by X-ray crystallography. In contrast, employing 1,10-phenanthroline (L5) as the ligand led to a significantly lower yield (entry 9). Other solvents such as dioxane, toluene, and DMF resulted in much lower reactivity (entries 10–12). We also explored the potential of alternative boron reagents, such as B2pin2, which yielded inferior results (entries 13). Interestingly, alternative iridium sources such as [Ir(cod)Cl]2 and [Ir(cod)acac]2 resulted in a substantial decrease in yield (entries 14–15), while other transition metals like [Rh(cod)Cl]2 completely failed to promote the reaction (entry 16). Reducing the loading of [Ir(cod)OMe]2 to 1.0 mol% still exhibited an acceptable reactivity (entry 17). Finally, a control experiment confirmed that the reaction did not proceed in the absence of the metal catalyst (entry 18). Having the optimized reaction conditions in hand, we proceeded to explore the suitability of various arylhydrosilanes for ortho C–H borylation (Fig. A wide range of triarylsilanes bearing substituents such as methyl (1b), isopropyl (1c), tert-butyl (1d), phenyl (1e), methoxy (1f), and methylthio (1g) were subjected to the system producing products 2b–2g with modest to good yields. Notably, the reaction of tri(naphthalen-2-yl)silane (1h), which possesses two potential sites for C–H borylation, demonstrated remarkable selectivity towards borylation at the less hindered C3 position. The structure of product 2h was also confirmed by X-ray crystallography. However, when identical reaction conditions were applied to dialkylarylhydrosilanes, the reaction efficiency decreased markedly, necessitating further optimization. The inclusion of NaOAc in the conditions with a mixture of B2pin2 and HBpin was found to significantly enhance the reaction efficiency. For example, silane 1i, bearing two isopropyl substituents, was successfully converted to product 2i in a 54% yield. In contrast, without NaOAc, the reaction became very chaotic. Substrate 1j, containing a para-methyl group, displayed favorable reactivity and selectivity at the less sterically hindered position. Conversely, substrate 1k, with a meta-methyl group, exhibited significantly reduced reactivity. Furthermore, other dialkylarylhydrosilanes with various substituents at the para position, including iPr (1m), tBu (1n), Ph (1o), OMe (1p), TMS (1q), F (1r), Cl (1s), CF3 (1t), and OCF3 (1u), were successfully utilized under these conditions. Additionally, the reaction of diisopropyl(naphthalen-2-yl)silane (1v) showed exclusive C3 selectivity. Under these optimized conditions, when diarylalkylhydrosilane 1w, containing two aryl and one methyl substituent, was tested, product 2h was only 17% yield, and a byproduct 2a, resulting from σ-bond metathesis, was observed. Interestingly, substrates with isopropyl (1x) and cyclopentyl (1y) groups exhibited compatible reactivity, indicating that the introduction of bulky substituents on the silicon atom can be advantageous in suppressing σ-bond metathesis. Finally, the reaction of silane 1z, containing two aryl and one benzyl substituent, displayed extraordinary selectivity for C–H bond activation within the benzyl motif, resulting in the formation of the dual C–H borylation product 2z with a yield of 68%. This intriguing outcome suggests a preferential formation of five-membered silametallacycles during the C–H activation process. Reaction conditions: [Ir(cod)OMe]2 (2.5 mol%), L4 (5.0 mol%), 1 (0.50 mmol), HBpin (1.25 mmol), KOAc (1.25 mmol) in 1.5 mL of THF at 125 °C for 6 h under argon; For entries 8–25: [Ir(cod)OMe]2 (5.0 mol%), L4 (10.0 mol%), HBpin (0.15 mmol), B2pin2 (0.75 mmol), NaOAc (1.25 mmol) at 150 °C. aIsolated yield. Based on the above findings, we extended our investigation to explore the application of hydrogermanes in C–H borylation (Fig. Initially, when triphenylgermane (3a) was substituted with 1a under the standard conditions, a significant presence of byproducts arising from σ-C–Ge bond metathesis was observed. To address this challenge, we performed a comprehensive screening and identified an optimized reaction protocol employing a stoichiometric quantity of the tridentate ligand L1. This ligand effectively inhibited the metathesis pathway, enabling the formation of the desired product 4a with a 72% yield (entry 1). Diminishing the catalyst load significantly reduced the yield of the target product (entry 2). Nonetheless, decreasing the ligand amount to 30 mol% still facilitated the formation of 4a with an acceptable yield (56%). Notably, substituting ligand L1 with L4 resulted in only trace amounts of the desired product (entry 4). Additionally, KOAc was found to be indispensable for the reaction's progression (entry 5). Leveraging these insights, we investigated the feasibility of C–H borylation using a variety of arylhydrogermanes. Triarylhydrogermanes with different substituents on the benzene ring, such as methyl (3b), isopropyl (3c), methoxy (3d), methylthio (3e), fluoro (3f), and trimethylsilyl (3g), were tested in the reaction system, yielding products 4b–4g in moderate yields. Notably, the reaction involving tri-3-methylphenylgermane (3h) and tri-2-naphthylgermane (3i), which have two potential sites for borylation, showed a significant preference for the less sterically hindered position. Furthermore, hydrogermanes 3j and 3k, substituted with two phenyl groups and one alkyl group, were effectively transformed into products 4j and 4k with high efficiency. However, under these updated conditions, the reaction of a hydrogermane, containing one phenyl and two isopropyl substituents, completely failed. Instead, we only observed the byproduct resulting from σ-bond metathesis (not shown in the Figure). Reaction conditions: [Ir(cod)OMe]2 (5.0 mol%), L1 (1.5 mmol), 3 (0.50 mmol), HBpin (1.25 mmol), KOAc (1.25 mmol) in 3.0 mL of THF at 115 °C for 24 h under argon. aYield was determined by GCMS. bIsolated yield. The adept synthesis of mixed-metalloid compounds has paved the way for more in-depth investigations into their reactivity (Fig. Our initial studies focused on the reactivity of ortho-boronated arylhydrosilanes (Fig. For instance, compound 2a can efficiently undergo a Pd-catalyzed Suzuki–Miyaura coupling reaction with iodoarenes 5, leading to the rapid formation of biaryl product 6 in a yield of 54%. Subsequently, in the presence of Wilkinson's catalyst, compound 6 can be further transformed through dehydrogenative cyclization, yielding silole 7 with an impressive 88% yield51. Moreover, the reaction of compound 2a with ClCH2I and nBuLi results in the in situ generation of the CH2BrLi reagent, which ultimately leads to the formation of boronate 8, now featuring an additional methylene unit52. Notably, the methylation of the Si–H bond in 2a, achieved via treatment with CH2I2 and Et2Zn, delivers product 9 with full conversion. This compound can then be effectively oxidized using NaBO3·4H2O, providing phenol 10 in nearly quantitative yield. Further diversification of compound 9 is achieved by subjecting it to NIS in the presence of CuI, resulting in the iodination product 11 with a yield of 73%. Additionally, we developed a Rh-catalyzed cyclization protocol wherein compound 9 reacts with alkyne 12, enabling the synthesis of benzosilole 13 with a 63% yield53. We also explored the reactivity of ortho-boronated arylgermanes, using reagent 3a as the starting material (Fig. Through a sequential C–H borylation and methylation of the Ge atom, compound 14 can be synthesized in a one-pot process with a yield of 64%. Utilizing the above rhodium-catalyzed cyclization conditions, we successfully obtained benzogermole 15 with a good yield from compound 14 and alkyne 12. a Follow-up transformations of compound 2a. b One-pot C–H borylation and methylation of 3a and further cyclization. A series of deuterium labeling experiments were initially conducted in the absence of HBpin to elucidate the potential reaction intermediates (Fig. When the reaction utilizing Ph3Si–D (1a–d) was performed without KOAc, 76% deuterium incorporation was observed at the ortho position of 1a, while only 5% D was detected at the silicon atom (Fig. The observed D incorporation suggests that the oxidative addition of Si–H bond to iridium center is irreversible and that the reaction may proceed through a four-membered silacyclic metal intermediate. The ortho C–D incorporation increased upon the addition of KOAc, indicating that KOAc accelerates the ortho C–H metalation. Furthermore, subsequent addition of D2O to this system resulted in significant deuterium labeling at the Si atom (43% D), suggesting the formation of a potassium silylide within the system, which can undergo hydrolysis with water. Importantly, no deuterium incorporation was observed in 1a with D2O in the absence of the catalyst (Fig. This discovery further confirms the essential presence of the iridium catalyst for the formation of four-membered silacyclic-iridium intermediate. A series of deuterium labeling experiments were then conducted under standard conditions using HBpin. During the work-up stage, deuterium was introduced into the reaction mixture of 1a and HBpin, resulting in 68% Si–D incorporation in the final product 2a (Fig. This observation confirms that the final product 2a is derived from a potassium silyl precursor. To further elucidate the mechanistic details of the Ir-catalyzed C–H borylation process, two kinetic isotope effect (KIE) experiments were performed. In the first experiment, parallel reactions were carried out using Ph3SiH (1a) and Ph3Si–D (1a–d1) with HBpin, yielding a KIE value of 1.29 (Fig. In the second experiment, parallel reactions were conducted using Ph3SiH (1a) and (C6D5)3Si-H (1a–d15) with HBpin, revealing a KIE value of 1.52 (Fig. These findings suggest that the cleavage of the Si–H bond and the ortho C–H bond are not the rate-determining steps in the reaction54. a Deuterium experiment of 1a–d in the absence of HBpin. b Deuterium incorporation of 1a in the absence of catalyst. c Deuterium experiment of 1a with D2O under standard conditions. d KIE experiment of 1a and 1a–d. e KIE experiment of 1a and 1a–15. Based on the above results, density functional theory (DFT) calculations were conducted to elucidate the reaction pathway of triphenylsilane 1a in the absence of HBpin, aiming to gain a deeper understanding the formation of strained silametallacycles (Fig. The catalytic cycle initiates with the dissociation of the dimeric iridium catalyst into the monomeric iridium intermediate INT1A. Following this, the Si−H bond of 1a undergoes oxidative addition to the iridium center of INT1A through the transition state TS2A, with an activation energy of 13.6 kcal mol−1 relative to the zero-energy point. In the absence of KOAc, the formed intermediate INT2A undergoes a reversible intramolecular oxidative addition of ortho C–H bond, forming a four-membered strained silametallacycles, designated as INT3A. In the presence of KOAc, intermediate INT2A activates the ortho C–H bond via a base-assisted internal electrophilic-type substitution (BIES) mechanism55,56. This process proceeds through the transition state TS3A-OAc, characterized by an activation energy of 18.1 kcal mol−1. Remarkably, this pathway is substantially more energetically favorable compared to the oxidative addition route through TS3A, which exhibits a significantly higher activation energy of 33.7 kcal mol−1. The reduced activation energy associated with the BIES mechanism underscores the efficiency of KOAc in facilitating the C–H activation, thereby promoting the formation of the strained silametallacyclic intermediate. Intermediate INT4A further undergoes reductive elimination via TS5A-Si with an activation energy of 25.3 kcal mol−1, transferring the hydrogen atom from silicon to the ortho-position of silicon center. This finding is consistent with experimental observations of deuterium incorporation. a The energy profile for the generation of intermediate INT4A. b The calculated key transition states. M06-D3/6–311 + G(d, p)-SDD(Ir)/SMD(tetrahydrofuran)//B3LYP-D3/6–31 G(d)-LANL2DZ(Ir). We further investigated the competition reaction between silane 1a and HBpin with iridium catalyst (Fig. The calculated energy barrier for the oxidative addition of the H–B bond to iridium is 18.8 kcal mol−1 relative to the energy of Ir-cat, significantly higher than that of the transition state TS2A for the oxidative addition of the H–Si bond (18.8 kcal mol−1 for TS2A-B vs 13.6 kcal mol−1 for TS2A). This energy difference suggests that silane 1a preferentially reacts with the iridium catalyst to form the strained silametallacyclic intermediate INT4A. This mechanistic pathway deviates notably from the previously directed aromatic C−H borylation triggered by Ir-Bpin species (see Supporting Information Figs. a Calculated transition state for the oxidative addition of the H–B bond to iridium center. b Key transition state of TS2A-B. Based on the above results, we examined the pathway leading to the final products by DFT calculations (Fig. A molecule of HBpin coordinates to the Ir center in INT4A, undergoing oxidative addition via transition state TS6A, with an energy barrier of 1.5 kcal mol−1, to form the Ir-H intermediate INT6A. Subsequently, reductive elimination and H2 release generate the penta-coordinated intermediate INT7A, with an activation energy of 2.3 kcal mol−1. A competitive pathway involving direct reductive elimination from INT6A to form the C–B bond was also considered but ruled out due to its prohibitively high energy barrier. The reductive elimination of INT7A proceeds through transition state TS8A, which has an activation energy of 20.3 kcal mol−1, leading to the formation of the desired C–B bond. In TS8A, the forming C–B bond length is 1.68 Å, indicating a well-defined transition state for this critical step. The generated intermediate INT8A then undergoes a transmetalation process with excess KOAc in the system, forming a silicate-potassium salt, pro-K. The structural information of transition state TS9A reveals the forming K–Si bond length to be 3.19 Å and the breaking Ir–Si bond length to be 3.45 Å. The activation energy barrier for this step is calculated to be 34.0 kcal mol−1 relative to INT6A, making it the highest energy barrier in the calculated free energy profile and thus the rate-determining step. During the work-up with D2O, the formed pro-K can further undergo hydrolysis, resulting in the formation of products 2a–d, as shown in the experimental results in Scheme 4a. Concurrently, the Ir intermediate INT10A, which is also generated, participates in an oxidative addition with another silane 1a, followed by a BIES-type C−H bond activation. This sequence of events culminates in the generation of the strained silametallacyclic intermediate INT4A, thereby completing the catalytic cycle efficiently. a The further transformation of intermediate INT4A. b The calculated key transition states. In conclusion, we have successfully developed a highly efficient and practical method for the synthesis of mixed-metalloid products through C−H borylation, enabled by iridium catalysts. This breakthrough addresses several long-standing challenges in the field of C−H activation, particularly the catalytic formation of strained benzo-fused metallacycles and the use of metalloid directing atom. The dual functionality of the formed mixed-metalloid compounds is a significant advantage, as it enables the synthesis of a wide range of valuable molecules with high versatility. The findings represent a significant advancement in organic synthesis, opening new avenues for the creation of complex and functionalized organic molecules with improved efficiency and selectivity. In a glove box, a 25 mL dried Schlenk tube equipped with a stirring bar was charged with silane (0.5 mmol), [Ir(cod)OMe]2 (8.3 mg; 0.0125 mmol, 2.5 mol%), dtbpy (6.7 mg; 0.025 mmol, 5 mol%), AcOK (123 mg; 1.25 mmol, 2.5 equiv.) and HBpin (160 mg; 1.25 mmol, 2.5 equiv.). The Schlenk tube was then removed from the glove box into air, and THF (1.5 mL, super dry) was added under a nitrogen flow. The mixture was heated to 125 °C and stirred for 6 h. The solvent was removed under reduced pressure, and the residue was purified by silica gel column chromatography to afford the products 2b–2h. In a glove box, a 25 mL dried Schlenk tube equipped with a stirring bar was charged with silane (0.5 mmol), [Ir(cod)OMe]2 (16.6 mg; 0.025 mmol, 5 mol%), dtbpy (13.4 mg; 0.05 mmol, 10 mol%), AcONa (102.5 mg; 1.25 mmol, 2.5 equiv. ), HBpin (19.2 mg; 0.15 mmol, 30 mol%) and B2pin2 (190.6 mg; 0.75 mmol, 1.5 equiv.). The Schlenk tube was then removed from the glove box into air, and THF (1.5 mL, super dry) was added under a nitrogen flow. The mixture was heated to 150 °C and stirred for 6 h. The solvent was removed under reduced pressure, and the residue was purified by silica gel column chromatography to afford the products 2i–2z. In a glove box, a 25 mL Schlenk tube equipped with a stirring bar was charged with [Ir(cod)OMe]2 (6.64 mg; 0.01 mmol; 5 mol%), 2,2′:6′,2″-terpyridine (139.8 mg; 0.6 mmol; 3 equiv. ), and AcOK (49 mg; 0.5 mmol; 2.5 equiv.). Then, the Schlenk tube was removed from the glove box into air. Germane (0.2 mmol), HBpin (64 mg; 0.5 mmol; 2.5 equiv.) and THF (3 mL) were added to the Schlenk tube under a nitrogen flow. The reaction mixture was then heated to 115 °C and stirred for 24 h. The solvent was removed under reduced pressure, and the residue was purified by silica gel column chromatography to afford the products 4b–4k. Crystallographic data for the structures reported in this Article have been deposited at the Cambridge Crystallographic Data Centre, under deposition numbers CCDC 2378961 (2a) and 2378962 (2h). Copies of the data can be obtained free of charge via https://www.ccdc.cam.ac.uk/structures/. Source Data are provided with this manuscript. Supplementary information and chemical compound information are available in the online version of the paper. Reprints and permission information is available online at http://www.nature.com/reprints/. Correspondence and requests for materials should be addressed to Z.S. Source data are provided with this paper. Google Scholar Wencel-Delord, J., Dröge, T., Liu, F. & Glorius, F. Towards mild metal-catalyzed C–H bond activation. Google Scholar Dong, Z., Ren, Z., Thompson, S. J., Xu, Y. & Dong, G. Transition-metal-catalyzed C–H alkylation using alkenes. Google Scholar Yang, Y., Lan, J. & You, J. Oxidative C–H/C–H coupling reactions between two (Hetero)arenes. Google Scholar Wang, C.-S., Dixneuf, P. H. & Soulé, J.-F. Photoredox catalysis for building C–C bonds from C(sp2)–H bonds. Google Scholar Davies, H. M. L. & Liao, K. Dirhodium tetracarboxylates as catalysts for selective intermolecular C–H functionalization. Google Scholar Gandeepan, P. et al. 3d transition metals for C–H activation. Google Scholar Guillemard, L., Kaplaneris, N., Ackermann, L. & Johansson, M. J. Late-stage C–H functionalization offers new opportunities in drug discovery. Google Scholar Zhao, B., Prabagar, B. Modern strategies for C–H functionalization of heteroarenes with alternative coupling partners. Google Scholar Dalton, T., Faber, T. & Glorius, F. C–H activation: toward sustainability and applications. ACS Cent. Google Scholar Lyons, T. W. & Sanford, M. S. Palladium-catalyzed ligand-directed C−H functionalization reactions. Google Scholar Colby, D. A., Bergman, R. G. & Ellman, J. A. Rhodium-catalyzed C−C bond formation via heteroatom-directed C−H bond activation. Google Scholar Zhang, F. & Spring, D. R. Arene C–H functionalisation using a removable/modifiable or a traceless directing group strategy. Google Scholar Liu, B. et al. Transition-metal-catalyzed, coordination-assisted functionalization of nonactivated C(sp3)–H bonds. Google Scholar Sambiagio, C. et al. Comprehensive overview of directing groups applied in metal-catalysed C–H functionalisation chemistry. Google Scholar Li, Z. & Shi, Z. Late-stage diversification of phosphines by C–H activation: a robust strategy for ligand design and preparation. Google Scholar & Tilley, T. D. Synthesis of Sm−SiH3 complexes via σ-bond metathesis of the Si−C bond of phenylsilane. Google Scholar & Hartwig, J. F. Silyl-directed, iridium-catalyzed ortho-borylation of arenes. a one-pot ortho-borylation of phenols, arylamines, and alkylarenes. Google Scholar Cho, S. H. & Hartwig, J. F. Iridium-catalyzed borylation of secondary benzylic C–H bonds directed by a hydrosilane. Google Scholar Su, B. & Hartwig, J. F. Iridium-catalyzed, silyl-directed, peri-borylation of C−H bonds in fused polycyclic arenes and heteroarenes. Google Scholar & Mills, J. S. Chemistry challenges in lead optimization: silicon isosteres in drug discovery. Drug Discov. Google Scholar Franz, A. K. & Wilson, S. O. Organosilicon molecules with medicinal applications. Google Scholar Langkopf, E. & Schinzer, D. Uses of silicon-containing compounds in the synthesis of natural products. Fleming, I., Barbero, A. & Walter, D. Stereochemical control in organic synthesis using silicon-containing compounds. & Hiyama, T. Silicon-based cross-coupling reaction: an environmentally benign version. Fricke, C., Deckers, K. & Schoenebeck, F. Orthogonal stability and reactivity of aryl germanes enables rapid and selective (multi)halogenations. Rogova, T., Ahrweiler, E., Schoetz, M. D. & Schoenebeck, F. Recent developments with organogermanes: their preparation and application in synthesis and catalysis. Tobisu, M., Onoe, M., Kita, Y. & Chatani, N. Rhodium-catalyzed coupling of 2-silylphenylboronic acids with alkynes leading to benzosiloles: catalytic cleavage of the carbon−silicon bond in trialkylsilyl groups. Ihara, H., Koyanagi, M. & Suginome, M. Anthranilamide: a simple, removable ortho-directing modifier for arylboronic acids serving also as a protecting group in cross-coupling reactions. Tobisu, M., Baba, K. & Chatani, N. Rhodium-catalyzed synthesis of germoles via the activation of carbon–germanium bonds. Ihara, H. & Suginome, M. Easily attachable and detachable ortho-directing agent for arylboronic acids in ruthenium-catalyzed aromatic C−H silylation. Aizenberg, M. & Milstein, D. Competitive generation of C–H and C–Si bonds by reductive elimination: formation of silametallacycles by metalation of silyl ligands. Aizenberg, M. & Milstein, D. Facial (methyl)(hydrido)(silyl) complexes of iridium: synthesis, X-ray structures, and reductive elimination reactions. Facile formation of silametalacycles by metalation of silyl ligands. Klei, S. R., Tilley, T. D. & Bergman, R. G. Reactions of Cp*(PMe3)Ir(Me)OTf with silanes: role of base-free silylene complexes in rearrangements of the resulting silicon-based ligands. McNally, A., Haffemayer, B., Collins, B. S. L. & Gaunt, M. J. Palladium-catalysed C–H activation of aliphatic amines to give strained nitrogen heterocycles. Willcox, D., Chappell, B. G. N., Hogg, K. F., Calleja, J., Smalley, A. P. & Gaunt, M. J. et al. A general catalytic β-C–H carbonylation of aliphatic amines to β-lactams. A. I., Barnard, J. H., Marder, T. B., Murphy, J. M. & Hartwig, J. F. C−H activation for the construction of C−B bonds. Hartwig, J. F. Regioselectivity of the borylation of alkanes and arenes. Ros, A., Fernández, R. & Lassaletta, J. M. Functional group directed C–H borylation. Xu, L. et al. Recent advances in catalytic C−H borylation reactions. Wright, J. S., Scott, P. J. H. & Steel, P. G. Iridium-catalyzed C−H borylation of heteroarenes: balancing steric and electronic regiocontrol. Fricke, C. & Schoenebeck, F. Organogermanes as orthogonal coupling partners in synthesis and catalysis. Dahiya, A., Fricke, C. & Schoenebeck, F. Gold-catalyzed chemoselective couplings of polyfluoroarenes with aryl germanes and downstream diversification. & Xiao, B. Germatranes and carbagermatranes: (hetero)aryl and alkyl coupling partners in Pd-catalyzed cross-coupling reactions. Dahiya, A., Schoetz, M. D. & Schoenebeck, F. Orthogonal olefination with organogermanes. & Hartwig, J. F. Iridium-catalyzed preparation of silylboranes by silane borylation and their use in the catalytic borylation of arenes. de la Vega-Hernández, K. et al. Radical germylzincation of α-heteroatom-substituted alkynes. Debrauwer, V. et al. Ligand-controlled regiodivergent palladium-catalyzed hydrogermylation of ynamides. Shishido, R. et al. General synthesis of trialkyl- and dialkylarylsilylboranes: versatile silicon nucleophiles in organic synthesis. Marciniec, B., Pietraszuk, C., Pawluć, P. & Maciejewski, H. Inorganometallics (transition metal−metalloid complexes) and catalysis. Ureshino, T., Yoshida, T., Kuninobu, Y. & Takai, K. Rhodium-catalyzed synthesis of silafluorene derivatives via cleavage of silicon−hydrogen and carbon−hydrogen bonds. Okamoto, K. et al. External flash generation of carbenoids enables monodeuteration of dihalomethanes. Onoe, M. et al. Rhodium-catalyzed carbon–silicon bond activation for synthesis of benzosilole derivatives. Simmons, E. M. & Hartwig, J. F. On the interpretation of deuterium kinetic isotope effects in C–H bond functionalizations by transition-metal complexes. Balcells, D., Clot, E. & Eisenstein, O. C–H bond activation in transition metal species from a computational perspective. Davies, D. L., Macgregor, S. A. & McMullin, C. L. Computational studies of carboxylate-assisted C–H activation and functionalization at group 8–10 transition metal centers. Tamura, H., Yamazaki, H., Sato, H. & Sakaki, S. Iridium-catalyzed borylation of benzene with diboron. Theoretical elucidation of catalytic cycle including unusual iridium(V) intermediate. Vanchura, B. A. et al. Electronic effects in iridium C–H borylations: insights from unencumbered substrates and variation of boryl ligand substituents. Roosen, P. C. et al. Outer-sphere direction in iridium C–H borylation. We would like to thank financial support from National Key R&D Program of China (2022YFA1503200), the National Natural Science Foundation of China (Grants 92361201, 22025104 and 22171134), the Natural Science Foundation of Jiangsu Province (Grant BK20240059, BK20220033), the Fundamental Research Funds for the Central Universities (Grant 020514380326) for their financial support. We are also grateful to the High-Performance Computing Center of Nanjing University for performing the numerical calculations in this paper on its blade cluster system. State Key Laboratory of Coordination Chemistry, Chemistry and Biomedicine Innovation Center (ChemBIC), School of Chemistry and Chemical Engineering, Nanjing University, Nanjing, China Qianwei Chen, Minyan Wang, Yue Zhao & Zhuangzhi Shi Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the experiments, mechanistic study, and analyzed the data. performed the crystallographic studies. conceived the study and wrote the paper. performed the DFT calculation and conceived the study. Correspondence to Minyan Wang or Zhuangzhi Shi. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Chen, Q., Wang, M., Zhao, Y. et al. Catalytic aromatic C−H borylation via strained Si and Ge metallacycles. Nat Commun 16, 8163 (2025). Received: 19 February 2025 Accepted: 17 August 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02760-y'>‘The wolf is not the bad guy': working with farmers to protect a reintroduced species</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 12:20:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Carmen García-Chávez is a biologist and co-founder of the Tonkawa Foundation in Nuevo Casas Grandes, Mexico. The Mexican wolf's return not only helps to conserve biodiversity but also represents an act of reconciliation with a species that humans persecuted for decades. We estimate that 25–30 Mexican wolves now live in the region. It's early days but their presence is already affecting the distribution of other predators such as pumas, hinting at an ecological rebalance. Wolves are not always welcome, especially in areas with livestock farms, so we work closely with landowners to monitor the wolves without generating conflict. The grant lottery: award rates at UK national funding agency fall below 20% How LinkedIn helped me to carve out a career path after my PhD Global phenology maps reveal the drivers and effects of seasonal asynchrony Engineered yeast provides rare but essential pollen sterols for honeybees States now have a legal duty to prevent climate harm — justice is in reach 20 Postdoctoral positions available at IChF PAS, funded by the Horizon Europe MSCA-Cofund programme. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02759-5'>Do social-media bans benefit young people? These data could offer clues</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 10:17:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Andrew Mackinnon is professor of biostatistics, The Black Dog Institute, Sydney, New South Wales, Australia. Smartphone use among teenagers is a hot-button topic for many people. The law will prevent people under 16 years of age from having accounts for a range of social-media platforms, including Instagram, Facebook, TikTok and YouTube. The legislation, introduced in response to concerns over children's online safety and well-being1, provides a golden opportunity for researchers to investigate how social-media use might affect mental health among adolescents — a complex issue that has been the subject of heated debate. Social-media bans won't work — there are better ways to keep kids safe Social-media bans won't work — there are better ways to keep kids safe There's little time to plan and set up the ideal experiment. It would require substantial organization in schools, the recruitment of research staff, agreement about the research measures used and fast ethical review and parental consent, all of which takes many months. Nevertheless, there are more-feasible approaches that could provide valuable information if researchers act promptly. On the downside, it takes time for official data to become available. Routinely collected data do not reveal anything about individual mental-health trajectories and social-media use, making it hard to determine whether social media contributed to peoples' actions. These records also do not allow investigation of the many instances of anxiety and depression that do not lead to self-harm or suicide. This is because these conditions often go untreated or are addressed through primary health-care services or private psychologists, from which data are not easily collated. These surveys typically involve a representative sample of the population. To study the social-media ban, they might instead oversample at-risk subgroups — teens from sexual and gender minorities (LGBTQIA+) or those with pre-existing mental health conditions, for instance. The surveys could be used to probe risk factors for anxiety and depression, including online bullying, and could ask detailed questions about mental ill-health and social-media use on various platforms, going beyond generic measures such as screen time. They could also include measurements of lifestyle and any activities or hobbies taken up to fill the time previously spent on social media. Analyses would help to answer questions about which individuals are protected by social-media bans, and what online activities are associated with improved well-being. Young people might adopt different lifestyle interests when Australia's social-media ban begins — a trend that researchers could study.Credit: Vlad Sokhin/Panos Pictures A gap of four years between surveys would make the data coarse with respect to any trajectories of change — but narrowing that gap to, say, two years is probably both financially and politically unfeasible. These studies have the advantage of individual retrospective data. Australia has several such studies that include mental-health measures. The Future Proofing Study, for instance, began in 2019 and is examining the onset and trajectory of mental-health conditions in 6,000 adolescents from age 13 onwards3. How social media affects teen mental health: a missing link How social media affects teen mental health: a missing link Trials evaluating public-health interventions could also be used, if their timespan encompasses the social-media ban (the control groups in such trials are essentially longitudinal observational studies). It could continue its follow-up assessments for longer than the current limit of 36 months. But comparisons are likely to be complex, owing to probable differences in sample demographics, risk assessments, age in the cohorts and outcome measurements. Social-media bans won't work — there are better ways to keep kids safe How social media affects teen mental health: a missing link Use fines from EU social-media act to fund research on adolescent mental health The vibrant colours of a house-hunting crab — August's best science image Research posts on Bluesky are more original — and get better engagement Trump called for ‘gold-standard science': how the NIH, NSF and others are answering ORCID launched more than a decade ago, but has yet to fulfil its potential Organs on chips could make biomedical research more equitable 20 Postdoctoral positions available at IChF PAS, funded by the Horizon Europe MSCA-Cofund programme. Seeking exceptional Senior/Junior PIs, Postdocs, and Core Specialists globally year-round The Director is initially appointed for a three-year term, renewable for a second term, and concurrently holds the title of Research Fellow. Social-media bans won't work — there are better ways to keep kids safe How social media affects teen mental health: a missing link Use fines from EU social-media act to fund research on adolescent mental health An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02805-0'>Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 09:40:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Genetic variants (both coding and noncoding) can impact gene function and expression, driving disease mechanisms such as cancer progression. The systematic study of endogenous genetic variants is hindered by inefficient precision editing tools, combined with technical limitations in confidently linking genotypes to gene expression at single-cell resolution. We developed single-cell DNA–RNA sequencing (SDR-seq) to simultaneously profile up to 480 genomic DNA loci and genes in thousands of single cells, enabling accurate determination of coding and noncoding variant zygosity alongside associated gene expression changes. Using SDR-seq, we associate coding and noncoding variants with distinct gene expression in human induced pluripotent stem cells. Furthermore, we demonstrate that in primary B cell lymphoma samples, cells with a higher mutational burden exhibit elevated B cell receptor signaling and tumorigenic gene expression. SDR-seq provides a powerful platform to dissect regulatory mechanisms encoded by genetic variants, advancing our understanding of gene expression regulation and its implications for disease. Genomic variation in both coding and noncoding regions of the genome drives human population differences and disease1,2,3. Over 90% of predicted genome-wide association study variants for common diseases are located in the noncoding genome, while their gene regulatory impact is challenging to assess. Genetic loss-of-function screening of coding genes and CRISPR interference (CRISPRi)/CRISPR activation screens in noncoding regions have provided valuable insights into disease mechanisms. However, they neglect precise genomic variation potentially masking more complex cellular disease phenotypes caused by individual variants4,5,6,7. Existing precision genome editing tools to introduce variants have limited efficiency and variable editing outcomes in mammalian cells8,9,10. This makes it difficult to use guide RNAs (gRNAs) as a proxy to annotate the variant perturbation in pooled screens. Although some droplet-based technologies enable assessment of variants within transcripts, they neglect the impact of noncoding variants, which constitute the vast majority of disease-associated variants11. Exogenous introduction of sequence variants, via episomal massively parallel reporter assays for noncoding variants or open reading frame expression for coding sequences, allows for high-throughput screening of variants for functional effects but lacks endogenous genomic position and sequence context12,13,14,15. These limitations hinder systematic studies of endogenous genetic variation and its impact on disease-relevant gene expression. To confidently link precise genotypes to gene expression in their endogenous context, a combined single-cell genomic DNA (gDNA) and RNA assay is required to directly assess coding and noncoding variants and gene expression in the same cell. Current technologies that enable simultaneous high-sensitivity readout of both gDNA and RNA are well established and laborious with low throughput16,17,18,19,20,21,22,23,24,25. High-throughput droplet-based or split-pooling approaches can measure thousands of cells simultaneously but lack combined high-sensitivity and tagmentation-independent readout of gDNA and RNA26,27,28,29. This results in sparse data with high allelic dropout (ADO) rates (>96%), making it impossible to correctly determine zygosity of variants on a single-cell level. Here, we developed targeted droplet-based single-cell DNA–RNA sequencing (SDR-seq), a scalable and sensitive method to screen genetic variation in high throughput, linking it to gene expression and distinct cellular states. We developed SDR-seq to simultaneously measure RNA and gDNA targets in the same cell with high coverage across all cells. The assay combines in situ reverse transcription (RT) of fixed cells with a multiplexed PCR in droplets using Tapestri technology from Mission Bio (Fig. Cells are dissociated into a single-cell suspension, fixed and permeabilized. In situ RT is performed by using custom poly(dT) primers, adding a unique molecular identifier (UMI), a sample barcode (BC) and a capture sequence (CS) to cDNA molecules. Cells containing cDNA and gDNA are loaded onto the Tapestri machine. After generation of the first droplet, cells are lysed, treated with proteinase K and mixed with reverse primers for each intended gDNA or RNA target. During generation of the second droplet, forward primers with a CS overhang, PCR reagents and a barcoding bead containing distinct cell BC oligonucleotides with matching CS overhangs are introduced. A multiplexed PCR amplifies both gDNA and RNA targets within each droplet. Cell barcoding is achieved through the complementary CS overhangs on PCR amplicons and cell BC oligonucleotides. Distinct overhangs on reverse primers containing either R2N (gDNA, Nextera R2) or R2 (RNA, TruSeq R2) allow for separation of next-generation sequencing (NGS) library generation for gDNA and RNA. This enables optimized sequencing of each library: (1) full-length to entirely cover variant information on gDNA targets along with the cell BC and (2) transcript and BC information (cell BC, sample BC and UMI) for RNA targets. R2N (Nextera) or R2 (TruSeq) overhangs on reverse primers enable separate NGS library generation for gDNA and RNA. b, Outline of the POP experiment. The fixation conditions and number of gDNA/RNA targets are indicated. c, Knee plot of ranked cells by sequencing depth (gDNA + RNA). d, Number of cells found per fixation condition. e, Correct sample BC detection per cell. Read count for maximum sample BCs found was divided by the total amount of RNA reads per cell; n = 4,391 (PFA) and 4,553 (glyoxal) cells from one SDR-seq experiment. Color indicates the percentage of reads per cell with max sample BCs. g,h, Number of gDNA targets per cell (g) and gDNA reads per cell (h); n = 4,391 (PFA) and 4,553 (glyoxal) cells from one SDR-seq experiment. i, Individual gDNA targets are shown per fixation condition. Size indicates the percentage of cells detected in. Color indicates read coverage; chr, chromosome. j,k, Number of genes per cell (j) and RNA UMIs per cell (k); n = 4,391 (PFA) and 4,553 (glyoxal) cells from one SDR-seq experiment. l, Individual genes are shown per fixation condition. Size indicates the percentage of cells detected in. m, Comparison of expressed genes to bulk RNA-seq data; z score data are scaled by row. n, Pearson correlation of expressed genes to bulk RNA-seq data. o, Average expression and variance of genes assayed in the POP experiment using SDR-seq, 10x Genomics and ParseBio. To test SDR-seq, we performed a proof-of-principle (POP) experiment amplifying a small number of gDNA (28) and RNA (30) targets in human induced pluripotent stem (iPS) cells (Fig. As fixation is critical for in situ RT, we tested two different fixatives, paraformaldehyde (PFA) and glyoxal. PFA is commonly used in in situ RT reactions but can impair gDNA and RNA quality as it cross-links nucleic acids30. Glyoxal does not cross-link nucleic acids and was expected to provide a more sensitive readout31,32. For simplicity, overhangs on reverse primers for gDNA and RNA were the same (R2N) in this experiment (Extended Data Fig. After filtering high-quality cells and removing doublets using distinct sets of sample BCs during in situ RT for each fixation condition, we obtained ~9,000 cells from a single SDR-seq run (Fig. Cells were evenly distributed over the two fixation conditions, with over 95% of reads per cell mapping to the correct sample BC on average (Fig. For downstream analysis, contaminating reads were removed from each cell. gDNA target coverage is expected to be uniform across cells as each cell contains the same gDNA input. We detected 23 of 28 gDNA targets (82%) with high coverage and in the vast majority of cells (Fig. Minimal differences in gDNA target detection and coverage were observed between PFA and glyoxal conditions (Extended Data Fig. RNA target coverage is expected to vary as they were chosen based on a range of expression levels. Indeed, individual RNA targets showed varying expression levels, with some only expressed in a subset of cells (Fig. RNA target detection and UMI coverage increased when using glyoxal compared to PFA (Extended Data Fig. Ubiquitously expressed housekeeping and iPS cell maintenance genes were detected in all cells, whereas other genes showed specific expression only in a subset of cells, consistent with published data (Extended Data Fig. Comparing bulk RNA-seq data of human stem cells to pseudo-bulked SDR-seq gene expression showed comparable levels of expression for the vast majority of targets with high correlation (Fig. SDR-seq showed reduced gene expression variance and higher correlation between individually measured cells than iPS cell data from 10x Genomics and ParseBio, indicating greater measurement stability (Fig. To test for potential cross-contamination of gDNA and RNA between cells during in situ RT, we performed a species-mixing experiment using human WTC-11 iPS cells and mouse NIH-3T3 cells. Cells were processed either separately or as a mixed population during in situ RT (Extended Data Fig. This allowed us to distinguish contamination introduced during in situ RT from general ambient nucleic acids by comparing the mixed-species condition to the single-species controls. We obtained a total of 16,000 cells across the different in situ RT conditions with the vast majority of doublets effectively removed using the sample BC information introduced during in situ RT (Extended Data Fig. Cross-contamination of gDNA was minimal (<0.16% on average), with no difference between the mixed-species and single-species conditions (Extended Data Fig. RNA cross-contamination was low (0.8–1.6% on average), with increased levels in the mixed-species condition compared to in the single-species controls (Extended Data Fig. The majority of cross-contaminating RNA from ambient RNA could be removed using the sample BC information introduced during in situ RT (Extended Data Fig. These data indicate that overall levels of cross-contaminating nucleic acids are low in SDR-seq. Together, these results demonstrate that SDR-seq enables highly sensitive detection of DNA and RNA targets across thousands of single cells in a single experiment, with the potential to link both modalities in a high-throughput manner. Next, we tested whether SDR-seq is scalable to detect hundreds of gDNA and RNA targets simultaneously. We designed an experiment using panels of 120, 240 and 480 targets, with equal numbers of gDNA and RNA targets in iPS cells (Fig. To enable cross-panel comparison, 60 gDNA and 30 RNA targets were shared between panels. To adjust for differences in sequencing depth, reads were subsampled for gDNA and RNA based on panel size to achieve equal average read coverage per cell for shared targets (Extended Data Fig. We confirmed that separately prepared NGS libraries for gDNA and RNA mapped with high specificity to their respective references (Extended Data Fig. Overall, 80% of all gDNA targets were detected with high confidence in more than 80% of cells across all panels, with only a minor decrease in detection for larger panel sizes (Extended Data Fig. Detection and coverage of shared gDNA targets were highly correlated between panels, indicating that gDNA target detection is largely independent of panel size (Fig. The minor decrease in detection for the larger panel sizes predominantly affected low-coverage targets (Extended Data Figs. Similarly, RNA target detection showed a minor decrease in larger panels compared to the 120 panel (Extended Data Fig. Detection and gene expression of shared RNA targets were highly correlated between panels (Fig. 4i,j), indicating robust and sensitive detection independent of panel size. Variability was predominantly observed for lowly expressed genes (Extended Data Fig. a, Outline of panel size testing experiments. gDNA and RNA targets are equal within panels; shared targets are indicated. b,c, Pearson correlation of detection (b) and coverage (c) of shared gDNA targets between panels. d,e, Pearson correlation of detection (d) and coverage (e) of shared genes between panels. f, Outline of chromatin sites tested. A combination of the chromatin marks detected and the relative distance to a gene define regulatory elements; PLS, promotor-like sequence; pELS, proximal enhancer-like sequence; dELS, distal enhancer-like sequence. g, Detection of different chromatin sites between panels. Size indicates the percentage of cells detected in. h, Outline of expression levels tested. i, Detection of genes with different expression levels between panels. Size indicates the percentage of cells detected in. j, Heat map of expression of all shared genes between panels; z score data are scaled by row. To assess whether chromosomal context influences gDNA detection using SDR-seq, we included target sites among the shared panels that were either overlapping expressed genes (OEGs) or not OEGs (NOEGs). Additionally, we tested for different chromatin marks and states (H3K3me3, H3K27ac and DNase sensitive), reflecting different genomic regulatory element types depending on their proximity to the transcription start site (TSS; Fig. We did not observe a strong impact on detection and coverage across panels based on OEG or NOEG location (Fig. Notably, no specific regulatory element type showed a systematic detection bias, and even sites with low DNase signal were confidently recovered. Genes were selected based on a range of expression levels and grouped into high, medium and low expression (Fig. High- and medium-expression genes were detected in almost all cells, whereas low-expression genes showed reduced detection rates across panel sizes (Fig. This aligns with published data suggesting that some genes are not expressed in all iPS cells33. Overall expression levels of shared genes were highly similar across the different panel sizes tested (Fig. To determine the ADO rate of SDR-seq, we selected gDNA amplicon loci containing heterozygous single-nucleotide polymorphisms based on bulk sequencing data. In amplicons detected in more than 80% of cells, heterozygous variants were correctly called in an average of 87–94% of cells (Extended Data Fig. The primary cause of ADO in larger panels was overall low detection rates of a gDNA target (Extended Data Fig. Noise levels of both miscalled variants and deletions or insertions were low (<0.15%) and showed comparable levels across panels (Extended Data Fig. The frequency of these lowly abundant miscalled variants depends on the reference base, with PCR deamination byproducts likely being the most common. Variant allele frequencies (VAFs) of individual variants could distinguish true heterozygous alleles from variant noise (Extended Data Fig. SDR-seq is thus scalable to assay hundreds of gDNA and RNA targets simultaneously with high reproducibility and sensitivity across different panel sizes, independently of chromatin state and expression level. This makes it a versatile tool to analyze variants at hundreds of loci in single cells, while simultaneously measuring cellular gene expression. Genomic variants can increase or decrease gene expression, but effect sizes are often small. Therefore, a sensitive readout of these gene expression changes is essential. We probed the ability of SDR-seq to detect strong and subtle gene expression changes across different perturbation systems designed to repress gene expression or introduce variants that alter expression levels. To assess whether SDR-seq can detect strong gene expression changes, we designed a CRISPRi experiment composed of four gRNA categories: (1) nontargeting control gRNAs (NTC), (2) gRNAs targeting expression quantitative trait loci (eQTLs), (3) gRNAs targeting the TSS of genes predicted to be affected by those eQTLs (CRISPRi controls) and (4) gRNAs targeting gene bodies to possibly introduce STOP codons through editing (STOP controls; Fig. CRISPRi-expressing human iPS cells infected with a lentiviral CROP-seq gRNA library were selected via fluorescence-activated cell sorting, followed by SDR-seq. The SDR-seq primer panel amplified the gDNA sites of the eQTLs together with associated transcripts, the viral CROP-seq transcript to assign cells to gRNAs and multiple housekeeping genes to normalize data. Cells were successfully assigned to gRNAs (75%), with an average of 30 cells per gRNA (Extended Data Fig. NTC gRNAs showed no significant effect on any of the genes measured, whereas most (95%) TSS-targeting CRISPRi control gRNAs caused a strong reduction in target gene expression (Fig. Seven eQTLs (24%) and three STOP control gRNAs (60%) significantly reduced target gene expression. Significantly scoring eQTL and STOP control gRNAs were located within a 2-kb window of the TSS, suggesting a direct inhibitory effect similar to CRISPRi control gRNAs (Extended Data Fig. This demonstrates that SDR-seq can confidently detect gene expression changes mediated by CRISPRi. Additionally, these data highlight the importance of directly assessing variants in proximity to the TSS to evaluate their impact on gene expression rather than approximating such effects with CRISPRi. a, Outline of the CRISPRi screen. b, Volcano plot for the CRISPRi screen with different gRNA classes indicating fold change and P value calculated using MAST with a Benjamini–Hochberg correction for multiple testing. For NTCs, all genes measured are shown. For other gRNA classes, only the intended target for each gRNA is shown. c, Outline of the PE screen. d, Volcano plot for the PE screen with different gRNA classes indicating fold change and P value calculated using MAST with a Benjamini–Hochberg correction for multiple testing. e, Alleles and gene expression for SOX11, ATF4 and MYH10 STOP controls are shown; **P < 10−3 and ***P < 10−4 calculated using MAST with a Benjamini–Hochberg correction for multiple testing; n = 4,152 (SOX11: REF), 117 (SOX11: HET), 9 (SOX11: ALT), 4,916 (ATF4: REF), 53 (ATF4: HET), 4 (ATF4: ALT), 4,925 (MYH10: REF) and 18 cells (MYH10: HET) from one SDR-seq experiment; P = 3.05 × 10−4 (ATF4: REF–HET), 5.38 × 10−8 (ATF4: REF–ALT), 4.95 × 10−3 (ATF4: HET–ALT) and 7.90 × 10−10 (MYH10: REF–HET). f, Outline of the BE screen. g, Volcano plot for different gRNA classes indicating fold change and P value calculated using MAST with a Benjamini–Hochberg correction for multiple testing. Comparisons between the different alleles are shown as shapes. h, Variants in the POU5F1 locus and their impact on gene expression are shown; UTR, untranslated region. The impact of each variant is color coded. REF, HET and ALT alleles are shown for each genotype. Fold change between the combination of variants is indicated in color (green), and P value (–log10) is indicated as size calculated using MAST with a Benjamini–Hochberg correction for multiple testing. Next, we aimed to directly install eQTL variants and measure more subtle effects on gene expression (Fig. We generated two human iPS cell lines expressing a prime editing (PE) transgene, with or without the coexpression of a dominant-negative regulator of the mismatch repair pathway designed to enhance editing efficiency (Pemax or Pemax-MLH1dn)10. To validate the system, we used a fluorescent lentiviral reporter system that measures editing efficiency via reconstitution of a nonfunctional enhanced green fluorescent protein (eGFP; Extended Data Fig. Using a PE gRNA (pegRNA) that repairs eGFP, we observed ~50% editing efficiency, demonstrating the system's editing potential in human iPS cells (Extended Data Fig. We lipofected these PE iPS cells with a pegRNA library designed to introduce the same eQTLs tested in the CRISPRi screen, as well as STOP codons to assess nonsense-mediated decay. Following fluorescence-activated cell sorting enrichment of lipofected cells, we performed SDR-seq (Fig. Editing efficiency was limited in both PE cell lines, complicating the interpretation of many variants (Extended Data Fig. Despite this limitation, we performed differential gene expression testing for between called reference (REF), heterozygous (HET) and alternative (ALT) variant alleles (Extended Data Fig. Significant gene expression changes were only observed for the STOP controls (Fig. In addition to installing eQTLs with PE, we tested the use of base editing (BE) in human iPS cells. We selected 56 high-likelihood eQTLs with a potential association for gene expression changes based on multiple studies, including noncoding variants that are located in open chromatin and editable with ABE8e or CBE base editors36,37,38 (Fig. None of these variants have previously been experimentally validated in an endogenous context as causative for transcriptional regulation, to our knowledge. After introducing gRNA libraries into iPS cells, cells were selected, and SDR-seq was performed. We found several eQTL variants with a significant effect on target gene expression (Fig. Additionally, we measured the effect of non-BE-associated mutations using SDR-seq. Human iPS cells accumulate somatic mutations during cell culture, while they undergo constant competitive selection for variants that are advantageous in culture conditions39. We found a synonymous variant in the 3′ end of POU5F1, a gene encoding a critical pluripotency factor, which significantly altered gene expression in the same direction as observed in prior eQTL studies36 (Extended Data Fig. However, after assessing variants that may have accumulated during culturing along the entire amplicon, we found that certain combinations of variants showed different effects on POU5F1 expression (Fig. In particular a set of variants in the 3′ untranslated region was associated with significantly different transcript levels. The presence of these variants was confirmed by bulk amplicon sequencing of this locus (Extended Data Fig. This highlights the importance of directly assessing variants at the locus of interest to accurately resolve their impact on gene expression. SDR-seq can confidently detect variants at the single-cell level and associate them with gene expression differences, demonstrating sensitivity even for subtle changes. This is the case even under conditions of limited editing efficiency in our experiments, which confound the interpretation of many tested eQTLs. Linking genetic variants to gene expression profiles is crucial for understanding cancer pathogenesis yet remains challenging in primary samples. B cell lymphomas are heterogenous cancers of the lymphatic system arising from distinct stages of B cell maturation. In this maturation process, naive B cells are stimulated to migrate through the dark zone (DZ) and light zone (LZ) of the germinal center, where they undergo somatic hypermutation and selection, followed by maturation into memory B cells and plasma cells40,41,42,43. Although the cell of origin is central to the classification of B cell lymphomas, it was recently shown that cancer cells retain their ability to differentiate. Thereby tumors acquire multiple maturation states from the same cell of origin while simultaneously undergoing clonal evolution through the accumulation of heterogenous genetic variants over time44,45. We used B cell lymphomas to investigate how genetic variation impacts gene expression and differentiation within tumors. We analyzed primary tumor samples from two individuals with follicular lymphoma and one individual with germinal center subtype diffuse large B cell lymphoma using SDR-seq (Fig. A targeted gDNA panel, based on variants from bulk DNA sequencing, was applied to profile 3,600 to 8,400 cells per sample. Using a reference mapping approach based on mutual nearest neighbors and canonical correlation analysis, we mapped B cell maturation states from a dataset of nonmalignant reactive lymph nodes to tumor samples (Extended Data Fig. Immunoglobulin light chain restriction confirmed monoclonality and malignancy of tumor B cells (Extended Data Fig. Somatic HET or ALT variants detected in both malignant B cells and non-B cells suggested limited contributions to disease progression, whereas variants occurring exclusively in malignant B cells may be oncogenic (Fig. Variants found in bulk gDNA sequencing of the same samples could also be recovered using SDR-seq (Extended Data Fig. The three samples showed a number of distinct variants, while some predominately somatic variants were shared. Primary samples and target panels are indicated. b, Uniform manifold approximation and projection (UMAP) highlighting the different samples clustered by either gene expression (RNA) or variants (gDNA). The numbers of cells for each sample are indicated as a bar graph. c, UMAP highlighting the maturation states clustered by either gene expression (RNA) or variants (gDNA). The numbers of cells within a maturation state are indicated as a bar graph (percentage of total); Mem, memory B cells. d, Variants detected in the experiment. Color indicates the percentage within B cells and non-B cells for each variant. Samples and HET/ALT alleles are indicated by color. Venn diagram showing the overlap of variants that occur with more than 5% frequency in each sample. e, Subset cells for DZ and LZ maturation states clustered by variants (gDNA) with clones indicated by color for each sample. The numbers of cells within DZ or LZ maturation states are indicated as a bar graph (percentage of total). f, Differentially abundant variants between DZ and LZ states (P < 0.05, χ2 test with a Benjamini–Hochberg correction). Summed counts of genes that the variants map to are shown in a bar graph. ΔDZ – LZ (percentage of the respective allele in DZ minus LZ) is shown in a heat map. Genes that the variants map to and patient of origin are indicated by color. g, Gene expression of the most frequently differentially expressed genes across samples in LZ and DZ states. Color indicates expression (z score, data are scaled by column), primary samples and maturation state. h, Gene Ontology (GO) term analysis of the most frequently differentially expressed genes. P values were computed using a Fisher's exact test with the weight01 algorithm (topGO), correcting for GO hierarchy structure. cell., cellular; comp., compounds; DE, differentially expressed; proc., process; resp., response; trans., transduction. i, Genes involved in B cell receptor signaling in cells with high (top 20%) and low (bottom 20%) variant burden in DZ and LZ states. Color indicates expression (z score, data are scaled by column), primary samples, genes, maturation state and variant burden. Next, we focused on a comparative analysis between DZ and LZ maturation states as most B cells belonged to these states (>80%). Clustering DZ and LZ cells based on variant information covered with our targeted gDNA panel revealed that two samples (FL2 (follicular lymphoma) and GCB1 (germinal center subtype diffuse large B cell lymphoma)) showed clonal structures (Fig. Genetic clones showed differences in proportions of the DZ and LZ states annotated by gene expression, indicating that clonal evolution and differentiation are predominantly separate processes. Our data suggest that genetic clones with different variant composition continue differentiating after they arise and can have an impact on differentiation rates. Differential abundance testing showed that BCL2 variants, a gene encoding an antiapoptotic factor frequently overexpressed in B cell lymphomas and central to B cell maturation, were enriched in the LZ compared to DZ (Fig. Variants were also enriched in many immunoglobulin variable region genes, which are targeted during somatic hypermutation. LZ cells predominantly exhibited an increase in ALT or REF variant alleles compared to DZ cells (Fig. Next, we tested if frequent variants impact gene expression in cells belonging to either the LZ or DZ state. We subset cells within each state into variant containing or not containing and performed differential gene expression testing. This revealed a number of genes involved in B cell receptor signaling and tumorigenesis, frequently affected in both DZ and LZ states, with increased participant-specific expression levels predominantly in the LZ compared to in the DZ (Fig. Elevated B cell receptor signaling is associated with repressing apoptosis in B cell lymphomas49,50. Cells with higher mutational burden, characterized by frequent HET and ALT variants, showed elevated levels of B cell receptor signaling compared to cells with lower mutational burden (Fig. LZ cells in the geminal center can revert to the DZ following unsuccessful binding to antigens from antigen-presenting cells and thereby undergo multiple rounds of somatic hypermutations51. Our data suggest that cells with a high mutational burden may have undergone more rounds of somatic hypermutation and have increased B cell receptor signaling and tumorigenic gene expression patterns to evade apoptosis induced by unsuccessful antigen binding in the LZ. This is in line with the distinct enrichment of variants in the LZ compared to in the DZ that we observe. Using SDR-seq, we profiled variants and gene expression simultaneously in primary tumor samples, linking cell states to mutational burden. We could distinguish variants present in malignant B cells and non-B cells, perform clustering analysis based on variants identifying clonal structures, test for enrichment of variants in maturation states and assess their impact on gene expression. This revealed elevated tumorigenic and antiapoptotic signaling in cells with higher mutational burden. Here, we developed SDR-seq to directly measure gene expression combined with coding and noncoding variants in single cells with high throughput and sensitivity. This method uses targeted primer panels for droplet-based multiplexed PCR to assess both gDNA and RNA in the same cells. Importantly, SDR-seq enables variant detection in noncoding regions of the genome, where the vast majority of disease-associated variants are located1,2,3,11. The targeted approach of SDR-seq facilitates high coverage of gDNA and RNA targets, allowing for confident detection of genomic variants and their zygosity, sensitive gene expression readout and reduced sequencing costs. This contrasts with existing split-pooling or droplet-based approaches, which rely on tagmentation of nucleosome-depleted chromatin and require whole-genome sequencing of each cell, resulting in sparse data and difficulties in correctly determining variant zygosity26,27,28,29. For these methods, ADO rates are high (>96%), whereas SDR-seq enables accurate detection of ~90% of alleles at the single-cell level. ADO levels of SDR-seq are comparable to targeted single-cell DNA sequencing using Tapestri (ADO < 10%)52. Lower-throughput single-cell methods that are plate based rely on tagmentation, amplification via multiple displacement or primary template-directed amplification (PTA) for gDNA readouts while enabling a whole-genome sequencing readout16,17,18,19,20,21,22,23,24,25. Although tagmentation- and multiple displacement amplification-based technologies also have high ADO rates, PTA achieves a high recovery rate for correctly determined alleles (>90%) when sequencing libraries are at saturation20,23,24,53. SDR-seq achieves a drastic ~100-fold increase in cell throughput compared to PTA-based single-cell DNA and RNA-sequencing technologies, while reducing total genome coverage due to its targeted approach24. Our results demonstrate that SDR-seq can assay hundreds of gDNA loci and genes simultaneously with high reproducibility and sensitivity across different panel sizes, covering up to 42.8 kb of gDNA per cell. Variants could be detected independent of chromatin context across hundreds of gDNA loci in the same cell. Distinct RNA targets can be picked and adjusted according to experimental needs. The scalability and sensitivity of SDR-seq make it a versatile tool for studying a wide range of coding and noncoding genetic variants and their effects on gene expression across diverse cell types. We can detect variants at a frequency of around 0.15% depending on the type and length of the variant. In both human iPS cells and primary human samples, we link variants to distinct gene expression patterns and can sensitively detect subtle gene expression changes. Advances in PE and pegRNA prediction tools might overcome limitations that we observed in this study constraining the interpretation of several infrequently edited eQTLs. In B cell lymphoma samples, SDR-seq enabled the identification of tumor-specific variants and their associated gene expression profiles, highlighting its potential for studying intratumor heterogeneity and cancer evolution. We could associate cells with higher mutational burden to elevated B cell receptor signaling and tumorigenic gene expression in primary B cell lymphoma samples. In future applications, SDR-seq could be combined with other readouts, including a targeted protein readout or DNA methylation, to provide a more holistic view of cellular regulation54,55. Targeting the mitochondrial genome with SDR-seq could enable clonal tracing of cell populations based on mitochondrial somatic variants56,57. Enhanced gene expression readouts might enable measurement of larger RNA panels or a whole-transcriptome readout in parallel to a highly sensitive targeted gDNA readout for multiple loci. Although our attempts for a combined whole-transcriptome readout by using template switch oligonucleotides during the in situ RT reaction were unsuccessful, other experimental approaches might be successful, thereby broadening the scope of potential applications. SDR-seq offers a powerful, scalable and sensitive approach to link genomic variants to gene expression in single cells, and this method is flexible to assay both genetically engineered cell lines and primary tissue samples. With the vast majority of predicted variants for common diseases located in the noncoding genome, SDR-seq enables the study of these variants systematically at scale1,2,3. In combination with gene editing tools, it holds great potential to decipher the regulatory mechanisms that underlie endogenous variants, complementing other high-throughput approaches that assay the gene expression-to-variant link of endogenous loci or via barcoding approaches12,13,14,15,58,59. This method advances our ability to study gene expression regulation and its implications for disease, providing insights that could drive the development of therapeutic strategies and enhance our understanding of complex genetic disorders. A detailed protocol for SDR-seq is published on protocols.oi at https://doi.org/10.17504/protocols.io.6qpvr9q43vmk/v1. WTC-11 iPS cells (Coriell Institute for Medical Research, GM25256) were verified to display a normal karyotype, were contamination free and were regularly tested for mycoplasma. Cells were cultured in Essential 8 medium (E8; Thermo Fisher Scientific, A1517001) on Vitronectin XF-coated (StemCell Technologies, 07180) tissue culture plates. iPS cells were split using Accutase (StemCell Technologies, 07922) and E8 supplemented with 10 µM Y-27632 dihydrochloride (RI; Tocris, 1254). After single-cell dissociation, 1 volume of E8 + RI was added, and cells were spun at 200g (5 min), resuspended and plated in E8 + RI. Mouse NIH-3T3 (DSMZ, ACC 59) cells were cultured in DMEM (Gibco, 11965092) supplemented with 10% fetal bovine serum (FBS), 100 U ml−1 penicillin/streptomycin and 1× nonessential amino acids on gelatin-coated tissue culture plates at 37 °C with 5% CO2. WTC-11 iPS cells were dissociated into single cells using Accutase, filtered through a 40-µm cell strainer and counted. For all experiments performed in human iPS cells, 1.5 × 106 cells were used as input for fixation. This was the minimum number of cells that were used as input for fixation in any experiment that involved human iPS cells. For glyoxal fixation, cells were resuspended in 200 µl of glyoxal fixation solution (3% glyoxal, 20% ethanol and 0.75% acetic acid (glacial), pH 4.0) and incubated for 7 min at room temperature. One milliliter of ice-cold wash buffer 1 (1× PBS with 2% bovine serum albumin (BSA), 1 mM DTT and 0.5 U µl−1 RNasin Plus ribonuclease inhibitor; Promega, N2615) was added, and cells were spun at 500g for 3 min at 4 °C. The supernatant was carefully removed, and the wash step was repeated with wash buffer 1 for a total of two washes. Cells were resuspended in 175 µl of ice-cold permeabilization buffer (10 mM Tris-HCl (pH 7.5), 10 mM NaCl, 3 mM MgCl2, 0.1% Tween 20, 0.2 U µl−1 RNasin Plus ribonuclease inhibitor, 1 mM DTT, 2% BSA, 0.1% IGEPAL CA-630 and 0.01% digitonin) and incubated for 4 min on ice. One milliliter of ice-cold wash buffer 2 (10 mM Tris (pH 7.5), 10 mM NaCl, 3 mM MgCl2, 0.1% Tween 20, 0.2 U µl−1 RNasin Plus ribonuclease inhibitor, 1 mM DTT and 2% BSA) was added, and the tube was gently inverted four to six times. Cells were spun at 500g for 5 min at 4 °C, resuspended in ice-cold resuspension buffer (1× PBS, 2% BSA, 1 mM DTT and 0.2 U µl−1 RNasin Plus ribonuclease inhibitor), filtered through a 40-µm strainer, counted and diluted to 1.4 × 106 cells per ml. PFA fixation was performed as described elsewhere with adaptations60. In short, cells were resuspended in 1 ml of 1× PBS with 0.2 U µl−1 RNasin Plus ribonuclease inhibitor, 3 ml of 1.3% PFA solution (in 1× PBS) was added, and cells were fixed for 10 min on ice. One hundred and sixty microliters of permeabilization buffer (5% Triton X-100 with 0.2 U µl−1 RNasin Plus ribonuclease inhibitor) was added, and the tube was gently inverted four to six times and incubated for 3 min on ice. Cells were spun at 500g for 3 min at 4 °C, the supernatant was carefully removed, and cells were resuspended in 500 µl of 1× PBS with 0.2 U µl−1 RNasin Plus ribonuclease inhibitor. Ice-cold 100 mM Tris-HCl at pH 8.0 (500 µl) was added and mixed by inverting the tube. Then, 20 µl of permeabilization buffer was added and mixed by inverting the tube four to six times. Cells were spun at 500g for 3 min at 4 °C, the supernatant was removed, resuspended in 300 µl of 0.5× PBS with 0.2 U µl−1 RNasin Plus ribonuclease inhibitor, filtered through a 40-µm strainer, counted and diluted to 1.4 × 106 cells per ml. Cell loss during fixation ranged between 10 and 30%. This was achieved by performing spins in swinging-bucket rotors and using 15-ml polypropylene centrifuge tubes during the entire process. RT master mix consisting of a final concentration of 1× RT buffer, 0.25 U µl−1 Enzymatics RNase Inhibitor (Biozym, 180520), 0.2 U µl−1 RNasin Plus ribonuclease inhibitor, 500 mM dNTPs and 20 U µl−1 Maxima H Minus Reverse Transcriptase (Thermo Fisher, EP0752) was prepared on ice in 8 µl for a total reaction volume of 20 µl. Four microliters of RT oligonucleotides (12.5 µM) was combined in each 96-well plate with 8 µl of RT master mix (Supplementary Tables 1 and 2). Eight microliters of fixed and permeabilized cells (10,000 cells) was added to each well, yielding a total reaction volume of 20 µl. We used a total of 48 RT reactions, yielding 480,000 cells, and recommend this number as it provides enough surplus to be in the range of the optimal cell concentration needed for the Tapestri microfluidic device from Mission Bio (105,000–200,000 cells). All RT reactions were pooled into a 15-ml conical tube containing 10 ml of 1× PBS with 1% BSA, and cells were spun at 500g for 5 min. Samples were processed using a Tapestri microfluidic device from Mission Bio (version 2, MB51-0007, MB51-0010 and MB51-0009) according to the manufacturer's protocol, with modifications. The in situ RT-processed cell pellet was resuspended in cell buffer from Mission Bio, and cells were counted and diluted to the appropriate concentration of 4,000–8,000 cells per µl. Custom primers were used in the multiplexed droplet PCR amplification step. RNA primers were designed using the TAP-seq primer prediction tool with a targeted optimal melting temperature of 60 °C (minimum 58 °C and maximum 62 °C) and a product size range from 150 to 300 bp using published single-cell RNA-sequencing iPS cell data for primer prediction (https://www.ebi.ac.uk/biostudies/arrayexpress with E-MTAB-6687)4,33. gDNA primers were designed using the Tapestri Designer (https://designer.missionbio.com). Primers were not validated before use; a dropout of around 10–20% is expected for gDNA primers. Especially for gDNA primers a dropout of custom primers is to be expected. An overview of version 1 and version 2 primer sequences with corresponding overhangs can be found in Supplementary Tables 1 and 2. Version 1 gDNA and RNA primers both had CS and R2N overhangs (only used in the POP experiment). Version 2 gDNA primers had CS and R2N overhangs, whereas RNA primers had CS and R2 overhangs. Detailed information on sample multiplexing using RT primers can be found in Supplementary Table 3. Forward and reverse stock primers had concentrations of 20 µM and 120 µM for both versions, respectively. Both reverse and forward primer mixes contained equimolar amounts of gDNA and RNA targeting primers. For version 1, final sequencing libraries were generated according to the Mission Bio user guide. For version 2, RNA and gDNA sequencing libraries were generated separately using the corresponding library amplification primers (Supplementary Table 2). Human iPS cells (WTC-11) and mouse fibroblasts (NIH-3T3) were fixed as described above with glyoxal. Cells of each species origin were either used individually or mixed during subsequent in situ RT using a distinct sample BC-introducing RT primer per well (48 in total). Human and mouse genes to profile by SDR-seq were selected to display a range of expression. RNA targeted primers were designed as described above using public data (https://www.10xgenomics.com/datasets/500-1-1-mixture-of-human-hek-293-t-and-mouse-nih-3-t-3-cells-3-lt-v-3-1-chromium-x-3-1-low-6-1-0) from 10x Genomics for NIH-3T3 primer design. Genomic sites were randomly selected, and gDNA targeted primers were designed as described above. Samples were processed using a Tapestri microfluidic device from Mission Bio (version 3, MB03-0091, MB03-0092 and MB03-0093) with modifications as described above. Informed consent from every participant was gathered beforehand. Lymph node samples were processed and frozen following previously described methods61,62. Frozen samples were thawed, added to 10 ml of RPMI (Gibco, 11875093) supplemented with 10% FBS and 0.5 mM EDTA and spun at 400g for 5 min. Cells were resuspended in 10 ml of 1× DPBS supplemented with 5% FBS, filtered through a 70-µm strainer and spun at 400g for 5 min. Cells were resuspended in 100 µl of bead solution from a Dead Cell Removal kit (Miltenyi Biotec, 130-090-101) and incubated for 15 min in the dark. Binding buffer was prepared according to the manufacturer's protocol, and the LS column (Miltenyi Biotec, 130-042-401) was washed with 500 µl of binding buffer. Cells were applied to the column and washed four times with binding buffer while collecting the flow through. Cells were spun at 400g for 5 min and resuspended in 1 ml of binding buffer. We proceeded with glyoxal fixation and SDR-seq as described above. For primary cells, input can be a limiting factor to perform SDR-seq. As described above, we recommend 48 wells with 10,000 cells each for the in situ RT reaction yielding 480,000 cells, enough surplus to be in the optimal range for the Mission Bio microfluidic device (105,000–200,000 cells). If there are not enough cells in an individual primary sample, they can be multiplexed on the same Tapestri run by using distinct sample BCs during the in situ RT reaction to achieve the optimal cell concentration needed as input for the Tapestri device. To fill up an entire lane of a Tapestri run, the minimum number of cells that was used in this study for a primary sample was 380,000 cells as input for the glyoxal fixation and 350,000 cells as input for the in situ RT. The gDNA panel was constructed for regions with >20% VAF detected in the selected tumor samples from the targeted DNA-sequencing data, which were sampled previously in Fitzgerald et al.44. Genes for a targeted expression readout of the profiled B cell lymphoma samples were chosen based on both the literature and CITE-seq data from the same samples44. These included genes from maturation markers found in the literature and variable features, differentially expressed genes and housekeeping genes based on single-cell RNA-sequencing data (Supplementary Table 4). Primers were designed as described above. For each SDR-seq dataset generated, we first performed custom BC identification and error correction, mapped reads to custom reference sequences and built read and deduplicated UMI matrices. This was performed with a software package we named SDRranger (https://github.com/hawkjo/SDRranger; https://doi.org/10.5281/zenodo.14762618 (ref. The full BC structure for the RNA targeted libraries is of the format cell BC1 (variable-length linker (14–17 bp)), cell BC2 (constant length linker (15 bp)) and sample BC (UMI). To identify these, we first aligned each read to all possible linker backbone sequences to account for the variable-length linker sequences. We discarded alignments with length-normalized alignment scores more than two standard deviations below average, measured from the first 10,000 reads. We then performed error correction on BC1 and BC2 to unique corrected BCs with a Levenshtein distance of 0 or 1. Due to the adjacent UMI, the sample BC does not have an identifiable end point in the case of insertions and deletions, so we corrected sample BCs with free divergence of 0 or 1 and with no other BC with free divergence only 1 higher64. Following BC identification, reads were mapped to custom alignment references built for each gDNA and RNA library. For gDNA, the chromosomal locations of the amplicons were used to extract reference sequences. Custom fasta and .gtf files were generated and used to build references using the genomeGenerate function of STAR (v2.7.11a). For the POP experiment, reads were separated into gDNA and RNA reads before BC identification by a separate mapping step to the corresponding references. Final bam files are produced, which contain tags with cell BC and sample BC sequences for each read, both before and after error correction. Matrices were then constructed by tallying reads by cell BC and sample BC versus gene or gDNA amplicon. For all reads from a given cell and given gene or amplicon, a connectivity graph of all observed UMIs is constructed. Each node is a unique UMI sequence and read count of that sequence, and directed edges are added between nodes A and B if the two UMIs have a free divergence of 1 to allow for indels, but only if nA ≥ 2nB – 1 reads, where nA and nB are the respective numbers of reads. This is based on the observation that each additional error to a UMI sequence should reduce the frequency of observing that sequence. Furthermore, only one incoming ‘parent' edge is allowed per node to avoid artifactual connections through singletons. This is repeated for each cell and sample BC for each gene or amplicon to build the full matrix. The resulting cell–gene (UMIs–RNA) and cell–amplicon (reads–gDNA) matrices were analyzed in R using the Seurat R package (v5.0.3). First, a general threshold per cell was set on reads per cell (RNA + gDNA) based on rank–rank plots of reads per cell ranked by size to determine an initial set of cells to include in the analysis. Then, multiple metrics were used to filter for high-quality cells. Only one distinct set of sample BCs is expected to be found per cell; therefore, this can be used to effectively remove doublets in the dataset. Contaminating reads that did not belong to the maximum sample BC found per cell were removed. RNA count matrixes were processed (log-normalized, scaled). A principal component analysis (PCA) was performed on all genes measured for RNA matrices. A probabilistic PCA was performed on all variants measured for VAF matrices. This was followed by subsequent UMAP embedding. For clustering, the shared nearest neighbor graph was calculated and used as input for Louvain clustering. Each cell that was defined as high quality was then used to call variants using the GATK HaplotypeCaller (v4.2.3.0)66. Each individual cell bam file was modified to contain the cell BC in the read name and indexed using samtools (v1.17), and the MAPQ scores were set from 255 (STAR output) to 60 and to be compatible as an input in the GATK HaplotypeCaller. GATK HaplotypeCaller was run using no maximum read threshold per cell and using a diploidy of two, and resulting vcf files were merged to yield a matrix of cells to variants compatible as input for Seurat. Low-frequency variants (<0.1% for editing and <0.3% for primary human samples) were removed, and remaining variants were input into the Ensembl Variant Effect Predictor for functional annotation67. This functional annotation was added as metadata. Genotypes of the GATK HaplotypeCaller were added as an assay to the previous Seurat object, while the remaining output was added as metadata. Wild-type alleles were included based on the read depth for a given amplicon per cell. Both wild-type alleles and variant alleles were excluded from subsequent analysis if the read depth was low (<10 reads) or the genotype quality score of the GATK HaplotypeCaller was low (<30 GQ). For comparison of variance in the proof-of-concept experiment, published 10x Genomics and ParseBio data (https://www.parsebiosciences.com/customer-datasets/multi-omics-approach-for-near-full-length-human-ipsc-transcriptomes-in-cardiomyocyte-models/#download) were used33. For constitutive CRISPRi and PEmax and PEmax-MLHdn1 cell lines, the corresponding transgene was inserted into the AAVS1 locus in WTC-11 iPS cells as previously described using specific TALENs5,68. For the CRISPRi plasmid, pHR-UCOE-SFFV-dCas9-mCherry-ZIM3-KRAB (Addgene, 154473) was modified to pHR-UCOE-SFFV-dCas9-mCherry-KRAB-MECP2 with DNA fragments ordered from Twist Bioscience containing KRAB–MECP2. dCas9, KRAB–MECP2 and dTomato were amplified and cloned into the AAVS1 targeting vector described above with NEBuilder HiFi DNA Assembly Master Mix (New England Biolabs, M5520). For the PEmax and PEmax-MLHdn1 plasmids, the CRISPRi plasmid was used as a backbone while inserting the PEmax or PEmax-P2A-MLHdn1 (Addgene, 174828) sequence with NEBuilder HiFi DNA Assembly Master Mix. To generate transgenic iPS cells expressing the CRISPRi, PEmax or PEmax-MLHdn1 transgene from the AAVS1 locus, WTC-11 iPS cells were electroporated with the corresponding homology plasmid (3 µg per electroporation) and two TALEN plasmids (0.75 µg per electroporation each) targeting the AAVS1 locus (Addgene, 52341 and 52342). iPS cells were dissociated into a single-cell suspension and counted, and 1 × 106 cells were electroporated using the CB-150 program of the 4D-Nucleofector System and the P3 Primary Cell 4D-Nucleofector X Kit L (Lonza, V4XP-3024), according to the manufacturer's protocol and plated in E8 + RI. Cells were sorted 7–10 days after electroporation for dTomato using a BD Fortessa instrument running Diva (V9.0.1) sofware, plated at low density in E8 + RI, grown to colonies, picked and genotyped (Supplementary Table 5). Positively genotyped clones were checked for homogenous dTomato signal and validated for activity in a corresponding assay, and three clones of each cell line were subjected to a genotyping array screening using an Infinium Global Screening Array-24 kit (Illumina, 20030770) to check for chromosomal rearrangements in iPS cell clones. Only clones that showed no or minor differences to the WTC-11 wild-type parental cell line were used in this study. All gRNA and pegRNA libraries were cloned in pools. eQTLs were selected based on high confidence from published data, and both lowly expressed genes (counts per million (CPM) > 150) and essential genes in iPS cells were removed also based on published data5,36,37,69. eQTLs for the base editor screen were further filtered by overlap with ATAC-seq peaks, expression (transcripts per million > 10) and compatibility with transversion by adenine or cytosine base editors (A > G, C > T, G > A, T > C)38. Sites to introduce STOP codons were chosen manually in the selected genes. pegRNAs were designed using PrimeDesign (https://drugthatgene.pinellolab.partners.org), and linkers to separate the pegRNA from the tevopreQ1 3′ stabilizing sequence were designed using pegLIT (https://peglit.liugroup.us)70,71. For the CRISPRi experiment, the spacer sequences of the above pegRNAs were used for eQTLs and STOP controls, whereas gRNAs targeting the TSS for genes predicted to be affected by eQTLs were designed using CRISPick (https://portals.broadinstitute.org/gppx/crispick/public), and NTCs were chosen from the GeCKO-v2 library72,73,74. BE gRNAs were designed and selected based on highest predicted editing efficiency using the BE-Hive tool75. The pegRNA screening vector was a kind gift from J. This vector was modified to remove the ERT2-Cre-ERT2 sequence and the gRNA scaffold and include a 3′ stabilizing tevopreQ1 after the insertion site for pegRNAs via BbsI Golden Gate cloning. The BE vectors were all-in-one cytosine or adenine base editor + guide expression constructs (Addgene, 158581 and 179097). The gRNA screening vector was a modified CROP-seq vector (Addgene, 86708) to also express eGFP and include a distinct gRNA CS in the scaffold of the gRNA76. Oligonucleotide pools for pegRNA and gRNA libraries were checked for the presence of BbsI and Esp3I sites within the spacer/RT/PBS sites and ordered from IDT as oPools. pegRNA oligonucleotides included a spacer sequence, PBS and RT with overhangs for amplification that included BbsI sequences compatible with Golden Gate cloning. Spacer and PBS/RT sequences were separated by a constant sequence containing two Esp3I sites for a second round of Golden Gate cloning to introduce the pegRNA scaffold. gRNA oligonucleotides consisted of spacer sequences and overhangs for amplification that included BbsI sequences compatible with Golden Gate cloning. Oligonucleotides were amplified (eight cycles) with compatible primers. The purified PCR product was cloned into the respective pegRNA or gRNA screening vector described above using BbsI and Golden Gate cloning. Electrocompetent bacteria (Lucigen, 60242-1) were electroporated (10 µF, 600 Ω, 1,800 V, E = 184 V cm−1) with purified ligation product and grown in a pool for 10 h at 30 °C before extracting plasmid DNA. For pegRNA and base editor guide libraries, a scaffold sequence was ordered with overhangs that included Esp3I overhangs (IDT), amplified with complementary primers (eight cycles), purified and cloned as described above using Esp3I Golden Gate cloning. Sequence overviews for cloning of the respective gRNA/pegRNA libraries can be found in Supplementary Table 5. Lentiviruses were produced in HEK293T (ATCC, CRL-3216) cells grown in DMEM supplemented with 10% FBS, 1× GlutaMAX (Gibco, 35050061), 100 U ml−1 penicillin–streptomycin (Gibco, 15140122) and 1× MEM nonessential amino acids (Gibco, 11140050) and coated using VSV-G. The day before transfection, HEK293T cells were plated at 80% confluency, plasmids were lipofected using Lipofectamine 3000 Transfection reagent (L3000001), and the cells were split 1:10 5 h after lipofection. The supernatant was collected 3 days after lipofection, cell debris was pelleted at 200g for 5 min at 4 °C, the remaining supernatant was spun at 28,000g for 5 h, and the virus pellet was resuspended in the appropriate volume of E8 + RI. On the day of infection, human iPS cells were split 1:2.5 2 h before infection using Accutase. Infections were performed overnight in E8 + RI. Medium was replaced the next day with E8. For some editing experiments, constructs were only expressed transiently in human iPS cells. For this, transfection was performed using Lipofectamine Stem Transfection Reagent (STEM00003) according to the manufacturer's protocol. Public variant information data for the WTC-11 cell line was downloaded from University of California, Santa Cruz (https://s3-us-west-2.amazonaws.com/downloads.allencell.org/genome-sequence/AH77TTBBXX_DS-229105_GCCAAT_recalibrated.vcf.gz). Variants were filtered using bcftools for heterozygous variants and quality (GT = ‘het', filter = ‘PASS', format/DP > 70, format DP < 150, QUAL > 1,000, INFO/MQ > 59.8) and subset to contain single-nucleotide polymorphisms, insertions/deletions or multinucleotide polymorphisms. Candidate cis-regulatory elements (cCREs) for five human iPS cell lines (H1, H7, H9, iPS DF 6.9 and iPS DF 19.11) were obtained from SCREEN (https://screen.encodeproject.org), and corresponding regulatory elements were subset from these34. Genomic regions were defined as OEGs or NOEGs if the gene overlapping that genomic region was expressed in bulk RNA-seq data (>10 CPM)5. These OEG or NOEG regions were then overlapped with the filtered WTC .vcf file to select for regions containing high-quality variant information. One hundred and twenty regions were randomly subsampled for each cCRE within the OEG and NOEG classes, and primers were designed as described above. Each cCRE with OEG and NOEG was equally represented in both the total and shared panels. Genes were subset into highly (>400 CPM), medium (<400, >40) and lowly (<40, >4) expressed gene groups. Primers were designed as described above. After determining high-quality cells in all panels, as described above, they were subset from the bam file, and reads per cell for gDNA and RNA were scaled according to panel size in a way so that the average number of reads per cell for shared gDNA and RNA targets was the same. Variants were called as described above. B cell maturation states were mapped to each tumor sample from a published reactive lymph node single-cell RNA-sequencing dataset through shared gene expression features using previously described methods44,46. Gene expression was used to determine immunoglobulin light chain restriction. Log-normalized counts (without batch effect correction to prevent bias introduced by sample integration) were used to find transfer anchors and project samples on the reference PCA (50 dimensions) and UMAP (2 dimensions) reductions. The expression of genes encoding immunoglobulin-κ (IGKC) and immunoglobulin-λ (IGLC1–IGLC7) light chain was used to determine cell malignancy through light chain restriction47. Separate NGS sequencing libraries for gDNA and RNA were analyzed with SDRranger, and variants were called as described above. GO term analysis was performed using the R package topGO (v2.54.0), the weigth01 algorithm and a Fisher's exact test. Enrichment for biological processes was computed for the top 21 differentially expressed genes in the LZ versus DZ across all samples versus all genes measured. No data were excluded from analysis, and cutoffs for defining high-quality cells in SDR-seq were set as described above. Differential gene expression testing in the single-cell data was performed using MAST and by subsetting cells in the respective genotype within a given cell or perturbation state77. Differential abundance testing of variants between maturation states in primary B cell lymphoma samples was performed using χ2 testing, followed by adjusting P values with the Benjamini–Hochberg method. All box plots shown in this study show the center line as the median, box limits indicate 25th and 75th percentiles, and whiskers indicate 1.5× the interquartile range; all outliers are displayed. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequencing data and processed data for nonprimary human samples are available on Gene Expression Omnibus under accession number GSE268646. Sequencing data and processed data for primary human samples are available on the European Genome–Phenome Archive under study number EGAS50000000374 and dataset ID EGAD50000000551. The dataset on European Genome–Phenome Archive is read-only under ega-archive.org/datasets/EGAD50000000551. Access to the data will be granted for appropriate use in research and will be governed by the provisions laid out in the terms contained in the Data Access Agreement. Source data are provided with this paper. All relevant code will be deposited on GitHub upon publication. Code containing SDRranger to generate count/read matrices from RNA or gDNA raw sequencing data is available under https://github.com/hawkjo/SDRranger. Code for TAP-seq prediction, generation of custom STAR references and processing of the data is available under https://github.com/DLindenhofer/SDR-seq. Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. The NHGRI-EBI GWAS catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Eichler, E. E. Genetic variation, comparative genomics, and the diagnosis of disease. Targeted Perturb-seq enables genome-scale genetic screens in single cells. Discovery of target genes and pathways at GWAS loci by pooled single-cell CRISPR screens. Kornienko, J. et al. Mislocalization of pathogenic RBM20 variants in dilated cardiomyopathy is caused by loss-of-interaction with transportin-3. Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Porto, E. M., Komor, A. C., Slaymaker, I. M. & Yeo, G. W. Base editing: advances and therapeutic opportunities. Nam, A. S. et al. Somatic mutations and cell identity linked by genotyping of transcriptomes. Arnold, C. D. et al. Genome-wide quantitative enhancer activity maps identified by STARR-seq. McAfee, J. C. et al. Focus on your locus with a massively parallel reporter assay. Ursu, O. et al. Massively parallel phenotyping of coding variants in cancer with Perturb-seq. Yu, L. et al. scONE-seq: a single-cell multi-omics method enables simultaneous dissection of phenotype and genotype heterogeneity from frozen tumors. Rodriguez-Meira, A. et al. Unravelling intratumoral heterogeneity through high-sensitivity single-cell mutational analysis and parallel RNA sequencing. Kong, S. L. et al. Concurrent single-cell RNA and targeted DNA sequencing on an automated platform for comeasurement of genomic and transcriptomic signatures. Zachariadis, V., Cheng, H., Andrews, N. & Enge, M. A highly scalable method for joint whole-genome sequencing and gene-expression profiling of single cells. Macaulay, I. C. et al. G&T-seq: parallel sequencing of single-cell genomes and transcriptomes. Han, K. Y. et al. SIDR: simultaneous isolation and parallel sequencing of genomic DNA and total RNA from single cells. Hou, Y. et al. Single-cell triple omics sequencing reveals genetic, epigenetic, and transcriptomic heterogeneity in hepatocellular carcinomas. Dey, S. S., Kester, L., Spanjaard, B., Bienko, M. & van Oudenaarden, A. Integrated genome and transcriptome sequencing of the same cell. Chung, C. et al. Cell-type-resolved mosaicism reveals clonal dynamics of the human forebrain. Li, W., Calder, R. B., Mar, J. C. & Vijg, J. Single-cell transcriptogenomics reveals transcriptional exclusion of ENU-mutated alleles. & Hwang, D. Single-cell multiomics: technologies and data analysis methods. Yin, Y. et al. High-throughput single-cell sequencing with linear amplification. Olsen, T. R. et al. Scalable co-sequencing of RNA and DNA from individual nuclei. Otoničar, J. et al. HIPSD&R-seq enables scalable genomic copy number and transcriptome profiling. Phan, H. V. et al. High-throughput RNA sequencing of paraformaldehyde-fixed single cells. Richter, K. N. et al. Glyoxal as an alternative fixative to formaldehyde in immunostaining and super‐resolution microscopy. Channathodiyil, P. & Houseley, J. Glyoxal fixation facilitates transcriptome analysis after antigen staining and cell sorting by flow cytometry. Nguyen, Q. H. et al. Single-cell RNA-seq of human induced pluripotent stem cells reveals cellular heterogeneity and cell state transitions between subpopulations. The ENCODE Project Consortium et al. Expanded encyclopaedias of DNA elements in the human and mouse genomes. The rules and impact of nonsense-mediated mRNA decay in human cancers. Cuomo, A. S. E. et al. Single-cell RNA-sequencing of differentiating iPS cells reveals dynamic genetic effects on gene expression. Large-scale profiling reveals the influence of genetic variation on gene expression in human induced pluripotent stem cells. Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath. Volpato, V. & Webber, C. Addressing variability in iPSC-derived models of human disease: guidelines to promote reproducibility. Küppers, R. Mechanisms of B-cell lymphoma pathogenesis. Seifert, M., Scholtysik, R. & Küppers, R. Origin and pathogenesis of B cell lymphomas. Koues, O. I., Oltz, E. M. & Payton, J. E. Short-circuiting gene regulatory networks: origins of B cell lymphoma. Fitzgerald, D. et al. A single-cell multi-omic and spatial atlas of B cell lymphomas reveals differentiation drives intratumor heterogeneity. Nowell, P. C. The clonal evolution of tumor cell populations. Stuart, T. et al. Comprehensive integration of single-cell data. Ratech, H. & Litwin, S. Surface immunoglobulin light chain restriction in B-cell non-Hodgkin's malignant lymphomas. Opferman, J. T. & Kothari, A. Anti-apoptotic BCL-2 family members in development. & Cambier, J. C. Src-family kinases in B-cell development and signaling. The B-cell-specific Src-family kinase Blk is dispensable for B-cell development and activation. De Silva, N. S. & Klein, U. Dynamics of B cells in germinal centres. Morita, K. et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. Luquette, L. J. et al. Single-cell genome sequencing of human neurons identifies somatic point mutation and indel enrichment in regulatory elements. Demaree, B. et al. Joint profiling of DNA and proteins in single cells to dissect genotype–phenotype associations in leukemia. Bianchi, A. et al. scTAM-seq enables targeted high-confidence analysis of DNA methylation in single cells. Identification of leukemic and pre-leukemic stem cells by clonal tracking from single-cell transcriptomics. Ludwig, L. S. et al. Lineage tracing in humans enabled by mitochondrial mutations and single-cell genomics. Cooper, S. E. et al. scSNV-seq: high-throughput phenotyping of single nucleotide variants by coupled single-cell genotyping and transcriptomics. Martyn, G. E. et al. Rewriting regulatory DNA to dissect and reprogram gene expression. B. et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Roider, T. et al. Multimodal and spatially resolved profiling identifies distinct patterns of T cell infiltration in nodal B cell lymphoma entities. & Dietrich, S. Processing human lymph node samples for single-cell assays. & Press, W. H. Indel-correcting DNA barcodes for high-throughput sequencing. & Sudbery, I. UMI-tools: modelling sequencing errors in unique molecular identifiers to improve quantification accuracy. Hockemeyer, D. et al. Genetic engineering of human pluripotent cells using TALE nucleases. Essential gene profiles for human pluripotent stem cells identify uncharacterized genes and substrate dependencies. Hsu, J. Y. et al. PrimeDesign software for rapid and simplified design of prime editing guide RNAs. Engineered pegRNAs improve prime editing efficiency. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR–Cas9. Sanson, K. R. et al. Optimized libraries for CRISPR–Cas9 genetic screens with multiple modalities. & Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Arbab, M. et al. Determinants of base editing outcomes from target library analysis and machine learning. Replogle, J. M. et al. Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing. Finak, G. et al. MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. We thank the Genomics Core Facility (GeneCore) at EMBL for consultation and sequencing. We thank J.-P. Mallm and the Single-Cell Open Lab (scOpenLab) at DKFZ for expertise and access to the Tapestri instrument from Mission Bio. Support by the DKFZ scOpenLab is gratefully acknowledged. We thank the Flow Cytometry Core Facility at EMBL for support and flow cytometry services. received funding from an HFSP postdoctoral fellowship (LT0023/2022-L). was supported by the European Research Council (Synergy Grant DECODE under grant agreement 810296) and a fellowship through state funds approved by the State Parliament of Baden-Württemberg for the Innovation Campus Health + Life Science Alliance Heidelberg Mannheim. received funding from a grant from the German Federal Ministry of Education and Research (SIMONA, 031L0263A). received funding from an EMBL-GSK Collaboration Fund. was supported by the EMBL predoctoral fund. was supported by the Dieter Schwarz Foundation Endowed Professorship and the National Human Genome Research Institute, National Institutes of Health (R01HG011664). at EMBL was supported by a grant from the Else Kröner Fresenius Stiftung (2021_EKS. Open access funding provided by European Molecular Biology Laboratory (EMBL). These authors contributed equally: Julia R. Bauman, John A. Hawkins. Dominik Lindenhofer, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Moritz Kueblbeck, Kyung-Min Noh, Wolfgang Huber, Oliver Stegle & Lars M. Steinmetz DZHK (German Centre for Cardiovascular Research), Partner Site Heidelberg/Mannheim, Heidelberg, Germany Division of Computational Genomics and Systems Genetics, German Cancer Research Center (DKFZ), Heidelberg, Germany Donnacha Fitzgerald, Sascha Dietrich & Wolfgang Huber Department of Oncology, Hematology and Clinical Immunology, Medical Faculty of Heinrich-Heine-Universität, Düsseldorf University Hospital, Düsseldorf, Germany Center for Integrated Oncology Aachen–Bonn–Cologne–Düsseldorf (CIO ABCD), Düsseldorf, Germany Stanford Genome Technology Center, Palo Alto, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed experiments and analyzed the data. developed the SDRranger pipeline for processing SDR-seq data. designed, performed and interpreted the study of B-cell lymphoma patient samples. provided protocols for fixation compatible with in situ RT. have submitted a patent application on ‘High-throughput multiomic readout of RNA and genomic DNA within single cells' (PCT/US2024/029950). is a cofounder and board member of Anthropocene Bio. The remaining authors declare no competing interests. Nature Methods thanks Xiao Dong and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Lei Tang, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Overview of targeted scDNA-scRNA-seq (SDR-seq) method for proof-of-principle (POP) experiment. R2N (Nextera) reverse primer overhangs were used for both gDNA and RNA. b, Quality control plots to remove doublets showing RNA reads per cell for each set of sample barcodes used to discriminate PFA and glyoxal fixation conditions. Cell numbers are indicated on the right. c–f, Quality control plots before (c, d) and after (e, f) filtering for low quality cells. Color indicates fraction of detected gDNA or RNA targets respectively. g, h, Coverage and detection of each gDNA (g) or RNA (h) target. a, UMAP clustering of POP SDR-seq data based on gene expression. b, Color coding of UMAP by fixation condition. c, UMAP plots of ubiquitously expressed genes GAPDH, POU5F1 and SOX2. d, UMAP plots of cluster-specifically expressed genes KLF5, SALL4 and ESRRB. e, Gene expression correlation (Pearson) between 100 subsampled cells comparing them individually against each other for SDR-seq, 10x Genomics and ParseBio. n = 4950 comparisons for SDR-seq, 10x Genomics and ParseBio. f, Outline of cell mixing experiment using either human (WTC-11) or mouse (NIH-3T3) cells. During in situ RT cells were either used in individual wells or mixed together. g, h, Number of cells found for each species per in situ RT condition before (g) and after (h) doublet removal using sample BC information during analysis. i, j, Cross contamination of gDNA (i) and RNA (j) molecules between species for each in situ RT condition. k–n, Quality control plots to show either gDNA reads/cell (k, l) or RNA UMIs/cell (m, n) for each in situ RT condition before and after doublet removal and ambient RNA/UMI removal using sample BC information. e, Overview of SDR-seq with separate library generation for RNA and gDNA. Distinct R2 (RNA) or R2N (gDNA) overhangs for each library. Sequencing ready libraries can be generated using specific library primers binding to R2 or R2N, respectively. f, Specificity of gDNA and RNA NGS libraries. Data from gDNA or RNA libraries was mapped to either gDNA or RNA references. a, Quality metrics of panel size experiment. Color indicates fraction of gDNA targets/cell recovered. b, gDNA coverage and detection for all targets across panels tested. c, gDNA target detection per cell for all targets across panels tested. d, gDNA coverage and detection for shared targets across panels tested. e, gDNA target detection per cell for shared targets across panels tested. f, Quality metrics of panel size experiment. Color indicates fraction of RNA targets/cell recovered. g, RNA coverage and detection for all targets across panels tested. h, RNA target detection per cell for all targets across panels tested. i, RNA coverage and detection for shared targets across panels tested. j, RNA target detection per cell for shared targets across panels tested. a, b, Comparison of coverage and detection for shared targets across panels tested for each gDNA (a) and RNA (b) target. c, Coverage and detection for shared targets across panels. Dashed line indicates 80% of cells detected in, which is the threshold used for categorizing highly covered amplicons that were used to determine allelic dropout (ADO). d, Percentage of correctly called heterozygous alleles across panels on shared targets. n = 58 variants from 1 independent SDR-seq experiment for each panel size testing. e, Influence of detection and gDNA coverage on correctly called heterozygous alleles. f, g, Normalized frequency of miscalled variants across panels on shared targets for conversions (f) or deletions/insertions (g). h, Example gDNA target amplicon with indicated frequencies of variants. Dashed lines indicate primer binding sites. i, Density of variant allele frequencies for heterozygous variants and noise. a, Overview of gRNA assignment for CRISPRi screen. b, Coverage for each gRNA in CRISPRi screen. n = individual gRNAs from 1 SDR-seq experiment. Size indicates P-value calculated using MAST with Benjamini-Hochberg correction for multiple testing. d, Outline of testing for PE iPSCs. Editing can be measured by repairing a non-functional EGFP that was integrated via a lentivirus. e, Flow cytometry indicating editing in PE iPSCs. a, Editing efficiency in PE screen. CRISPRi is indicated as a control. b, Editing efficiency for each locus that was assessed. Loci that had somatic alleles in the PE iPCSs that were either HET or ALT are not shown. c–e, Called genotypes and genotype quality vs. variant allele frequency (VAF) for SOX11 (c), ATF4 (d) and MYH10 (e). a, Intended edit to be introduced by base editing gRNA is shown at the POU5F1 locus and its impact on gene expression. P-value = 5.78x10−20 (POU5F1: HET-ALT), b, All measured variants in combination along the measured gDNA site shown for POU5F1 with their impact on gene expression. Number of cells are indicated on top. c, Comparison of variants at POU5F1 locus identified with SDR-seq to bulk amplicon sequencing. Each cell is assigned a state, higher score indicates better assignment of this cell to the respective state. b, UMAP of cells belonging to the geminal center (GC) maturation states including DZ and LZ. Color coded are either kappa or lamda Ig light chains. d, Comparison of variant allele frequencies (VAF) of variants found in bulk or in SDR-seq in either B-cells or non-B-cells. e, Summed counts for differentially expressed (DE) genes between variant containing and non-containing cells within both DZ and LZ for most frequent variants for each patient. Genes picked for profiling B cell lymphoma samples. Primer sequences for cloning and genotyping. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-01903-y'>Head–tail carboboration of multisubstituted alkenes enabled by chain recognition</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 09:18:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Precision in site-selective functionalization of molecules bearing multiple potentially reactive positions is a longstanding challenge in organic synthesis. Although migratory difunctionalization offers a powerful strategy for programmed functional group installation along carbon chains, its implementation with multisubstituted alkenes has been impeded by the increased complexity of site-selectivity. Here we report a head–tail carboboration of multisubstituted alkenes with exceptional site-selectivity, enabled by ligand steric exclusion in a nickel-catalysed chain-walking system. This catalytic system enables selective migratory transformations of tertiary alkyl–metal intermediates across a range of thermodynamically and kinetically accessible reaction site combinations. The importance of this study is evident in not only its capacity to efficiently couple structurally diverse carbon electrophiles with complex substrates, including natural terpenes, but also its contribution to streamlining the synthesis of natural products and materials. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data supporting the findings of this study are available in the article and its Supplementary Information. Crystallographic data for the structures reported in this article have been deposited at the Cambridge Crystallographic Data Centre, under deposition numbers CCDC 2376317 (52), CCDC 2376318 (54-1) and CCDC 2313837 (56-1). Copies of the data can be obtained free of charge via https://www.ccdc.cam.ac.uk/structures/. Trost, B. M. Selectivity: a key to synthetic efficiency. Clayden, J., Greeves, N. & Warren, S. Organic Chemistry (Oxford Univ. Zhu, J., Wang, Q. & Wang, M. Multicomponent Reactions in Organic Synthesis (Wiley, 2014). Davies, H. M. L. & Liao, K. Dirhodium tetracarboxylates as catalysts for selective intermolecular C–H functionalization. Google Scholar He, J., Wasa, M., Chan, K. S. L., Shao, Q. Palladium-catalyzed transformations of alkyl C–H bonds. Google Scholar McDonald, R. I., Liu, G. & Stahl, S. S. Palladium(II)-catalyzed alkene functionalization via nucleopalladation: stereochemical pathways and enantioselective catalytic applications. Google Scholar Whyte, A., Torelli, A., Mirabi, B., Zhang, A. & Lautens, M. Copper-catalyzed borylative difunctionalization of π-systems. ACS Catal. Google Scholar Yin, G., Mu, X. & Liu, G. Palladium(II)-catalyzed oxidative difunctionalization of alkenes: bond forming at a high-valent palladium center. Google Scholar Wu, D. et al. Overcoming limitations in non-activated alkene cross-coupling with nickel catalysis and anionic ligands. Google Scholar Wickham, L. M. & Giri, R. Transition metal (Ni, Cu, Pd)-catalyzed alkene dicarbofunctionalization reactions. Google Scholar Li, Y. Y. & Yin, G. Y. Nickel chain-walking catalysis: a journey to migratory carboboration of alkenes. Google Scholar Sommer, H., Juliá-Hernández, F., Martin, R. & Marek, I. Walking metals for remote functionalization. ACS Cent. Google Scholar Vasseur, A., Bruffaerts, J. & Marek, I. Remote functionalization through alkene isomerization. Google Scholar Rodrigalvarez, J., Haut, F.-L. & Martin, R. Regiodivergent sp3 C–H functionalization via Ni-catalyzed chain-walking reactions. JACS Au 3, 3270–3282 (2023). Google Scholar Li, Y., Wu, D., Cheng, H.-G. & Yin, G. Difunctionalization of alkenes involving metal migration. Google Scholar Ding, C., Ren, Y., Sun, C., Long, J. & Yin, G. Regio- and stereoselective alkylboration of endocyclic olefins enabled by nickel catalysis. Google Scholar Pang, H., Wu, D., Cong, H. & Yin, G. Stereoselective palladium-catalyzed 1,3-arylboration of unconjugated dienes for expedient synthesis of 1,3-disubstituted cyclohexanes. ACS Catal. Google Scholar Clardy, J. & Walsh, C. Lessons from natural molecules. Sharifi-Rad, J. et al. Biological activities of essential oils: from plant chemoecology to traditional healing systems. Lam, Y. A., Lawson, T. G., Velayutham, M., Zweier, J. L. & Pickart, C. M. A proteasomal ATPase subunit recognizes the polyubiquitin degradation signal. Rahighi, S. et al. Specific recognition of linear ubiquitin chains by NEMO is important for NF-κB activation. Beal, R. E., Toscano-Cantaffa, D., Young, P., Rechsteiner, M. & Pickart, C. M. The hydrophobic effect contributes to polyubiquitin chain recognition. Qian, C. & Zhou, M. M. SET domain protein lysine methyltransferases: structure, specificity and catalysis. Life Sci. Wu, H. et al. Structural biology of human H3K9 methyltransferases. PLoS ONE 5, e8570 (2010). Luo, M. Chemical and biochemical perspectives of protein lysine methylation. Bhat, K. P., Kaniskan, H. Ü., Jin, J. & Gozani, O. Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. Botuyan, M. V. et al. Structural basis for the methylation state-specific recognition of histone H4-K20 by 53BP1 and Crb2 in DNA repair. Braude, E. A., Coles, J. A., Evans, E. A. & Timmons, C. J. Steric effects in anionotropic rearrangements. Ramadoss, B., Jin, Y., Asako, S. & Ilies, L. Remote steric control for undirected meta-selective C–H activation of arenes. Wu, D. et al. Alkene 1,1-difunctionalizations via organometallic-radical relay. Li, Y. et al. Modular access to substituted cyclohexanes with kinetic stereocontrol. Wang, W., Ding, C. & Yin, G. Catalyst-controlled enantioselective 1,1-arylboration of unactivated olefins. Sakai, H. A., Liu, W., Le, C. C. & MacMillan, D. W. C. Cross-electrophile coupling of unactivated alkyl chlorides. Logan, K. M., Sardini, S. R., White, S. D. & Brown, M. K. Nickel-catalyzed stereoselective arylboration of unactivated alkenes. Joung, S., Bergmann, A. M. & Brown, M. K. Ni-catalyzed 1,2-benzylboration of 1,2-disubstituted unactivated alkenes. Jansen, D. J. & Shenvi, R. A. Synthesis of medicinally relevant terpenes: reducing the cost and time of drug discovery. Future Med. Wang, S. et al. Cobalt-catalysed allylic fluoroalkylation of terpenes. Nairoukh, Z., Cormier, M. & Marek, I. Merging C–H and C–C bond cleavage in organic synthesis. Di Martino, R. M. C., Maxwell, B. D. & Pirali, T. Deuterium in drug discovery: progress, opportunities and challenges. Li, Z. et al. Nickel-catalyzed regio- and enantioselective borylative coupling of terminal alkenes with alkyl halides enabled by an anionic bisoxazoline ligand. Sun, C., Ding, C., Yu, Y., Li, Y. & Yin, G. Ligand-modulated regiodivergent alkenylboration of allylarenes: reaction development and mechanistic study. Hansch, C., Leo, A. & Taft, R. W. A survey of Hammett substituent constants and resonance and field parameters. Falivene, L. et al. Towards the online computer-aided design of catalytic pockets. Falivene, L. et al. SambVca 2. A web tool for analyzing catalytic pockets with topographic steric maps. Escayola, S., Bahri-Laleh, N. & Poater, A. %VBur index and steric maps: from predictive catalysis to machine learning. Crabtree, R. H. The Organometallic Chemistry of the Transition Metals (Wiley, 2009). Li, X. et al. Regio- and enantioselective remote dioxygenation of internal alkenes. Matteson, D. S. Stereodirected Synthesis with Organoboranes (Springer Science & Business Media, 2012). Sandford, C. & Aggarwal, V. K. Stereospecific functionalizations and transformations of secondary and tertiary boronic esters. Li, C. et al. Non-fullerene acceptors with branched side chains and improved molecular packing to exceed 18% efficiency in organic solar cells. Gao, Y. et al. All-small-molecule organic solar cells with 18.1% efficiency and enhanced stability enabled by improving light harvesting and nanoscale microstructure. Chen, C. L. et al. Tetrahedral liquid-crystalline networks: an A15-like Frank–Kasper phase based on rod-packing. Schumm, J. S., Pearson, D. L. & Tour, J. M. Iterative divergent/convergent approach to linear conjugated oligomers by successive doubling of the molecular length: a rapid route to a 128 Å-long potential molecular wire. Yang, Z. et al. Reptilian chemistry: characterization of a family of dianeackerone-related steroidal esters from a crocodile secretion. Natl Acad. Krückert, K., Flachsbarth, B., Schulz, S., Hentschel, U. & Weldon, P. J. Ethyl-branched aldehydes, ketones, and diketones from caimans (Caiman and Paleosuchus; Crocodylia, Reptilia). Yang, Z., Attygalle, A. & Meinwald, J. Reptilian chemistry: enantioselective syntheses of novel components from a crocodile exocrine secretion. Download references National Natural Science Foundation of China (grant nos. and 22401220 to Yangyang Li), Guangdong Basic and Applied Basic Research Foundation (grant no. ), Fundamental Research Funds for the Central Universities (grant no. 413100070 to Yangyang Li) and Scientific Research Innovation Capability Support Project for Young Faculty (ZYGXQNJSKYCXNLZCXM-H17 to G.Y.) are acknowledged for financial support. We thank J. Min from Wuhan University for providing the intermediate compounds essential for synthesizing organic solar cells. We also thank the Core Facility of Wuhan University for their assistance with X-ray crystallographic analysis. Additionally, the numerical calculations for this research were conducted on the supercomputing system at the Supercomputing Center of Wuhan University. These authors contributed equally: Weiyu Kong, Dong Wu, Hong Wei. The Institute for Advanced Studies, TaiKang Center for Life and Medical Sciences, State Key Laboratory of Metabolism and Regulation in Complex Organisms, Wuhan University, Wuhan, China Weiyu Kong, Dong Wu, Hong Wei, Ben Gao, Penghui Li, Yangyang Li & Guoyin Yin Shanghai AI Laboratory, Shanghai, China Yuqiang Li Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed the project and directed the work. conducted the experiments and data analysis. and Yuqiang Li performed all density functional theory calculations. G.Y., W.K., D.W. and Yangyang Li wrote the paper. Correspondence to Yangyang Li or Guoyin Yin. The authors declare no competing interests. Nature Chemistry thanks Albert Poater and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Tables 1–8, Figs. 1–162, reaction development, experimental procedures, characterization data and computational details. Coordinate files from the computational study. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Kong, W., Wu, D., Wei, H. et al. Head–tail carboboration of multisubstituted alkenes enabled by chain recognition. Download citation Received: 10 October 2024 Accepted: 10 July 2025 Published: 01 September 2025 DOI: https://doi.org/10.1038/s41557-025-01903-y Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemistry ISSN 1755-4349 (online) ISSN 1755-4330 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02320-9'>Tissue-specific mRNA delivery and prime editing with peptide–ionizable lipid nanoparticles</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-01 09:14:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Materials (2025)Cite this article Lipid nanoparticles for mRNA delivery and gene editing have the potential to transform the current therapeutic landscape. Nonetheless, a major bottleneck using this technology is the difficulty in achieving cell- and tissue-specific delivery and avoiding liver accumulation. Here we report the rational design of peptide ionizable lipids to assemble lipid nanoparticles with organ-selective mRNA delivery. Structure–activity and structure–selectivity relationship analyses enable us to obtain a general and predictable strategy for peptide ionizable lipid design. By incorporating artificial ionizable and natural amino acids and/or functional molecules into peptide ionizable lipids, we create lipid nanoparticles with tissue-specific targeting, including the lungs, liver, spleen, thymus and bone. In particular, lipid nanoparticles containing peptide lipids targeting the liver show comparable efficacy and safety compared with FDA-approved formulations. Furthermore, lipid nanoparticles with peptide lipids achieve the efficient co-delivery of PEmax mRNA and engineered prime editing guide RNA for prime editing of the liver and lungs. Overall, our platform offers a predictable methodology for the rational design of tissue-targeting lipid nanoparticles that might aid the development of improved mRNA-based gene editing therapeutics. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data associated with this study are present in the article and its Supplementary Information. The raw and analysed datasets generated during the study are available for research purposes from the corresponding authors on request. NGS sequencing data are available in the Sequence Read Archive under accession no. Source data are provided with this paper. Cullis, P. R. & Felgner, P. L. The 60-year evolution of lipid nanoparticles for nucleic acid delivery. Drug Discov. Google Scholar A. et al. The current landscape of nucleic acid therapeutics. Google Scholar Chen, S. et al. Nanotechnology-based mRNA vaccines. Google Scholar Akinc, A. et al. Targeted delivery of RNAi therapeutics with endogenous and exogenous ligand-based mechanisms. Google Scholar Kim, J., Eygeris, Y., Ryals, R. C., Jozić, A. & Sahay, G. Strategies for non-viral vectors targeting organs beyond the liver. Google Scholar Herrera-Barrera, M. et al. Peptide-guided lipid nanoparticles deliver mRNA to the neural retina of rodents and nonhuman primates. Google Scholar Kedmi, R. et al. A modular platform for targeted RNAi therapeutics. Google Scholar Rurik, J. G. et al. CAR T cells produced in vivo to treat cardiac injury. Google Scholar Cheng, Q. et al. Selective organ targeting (SORT) nanoparticles for tissue-specific mRNA delivery and CRISPR–Cas gene editing. Google Scholar Lian, X. et al. Bone-marrow-homing lipid nanoparticles for genome editing in diseased and malignant haematopoietic stem cells. Google Scholar Han, X. et al. Ligand-tethered lipid nanoparticles for targeted RNA delivery to treat liver fibrosis. Google Scholar Lei, J. et al. Development of mannosylated lipid nanoparticles for mRNA cancer vaccine with high antigen presentation efficiency and immunomodulatory capability. Google Scholar Gautam, M. et al. Lipid nanoparticles with PEG-variant surface modifications mediate genome editing in the mouse retina. Google Scholar Melamed, J. R. et al. Ionizable lipid nanoparticles deliver mRNA to pancreatic β cells via macrophage-mediated gene transfer. Google Scholar Abd Elwakil, M. M. et al. Harnessing topology and stereochemistry of glycidylamine-derived lipid nanoparticles for in vivo mRNA delivery to immune cells in spleen and their application for cancer vaccination. Google Scholar Zhang, R. et al. Esterase-labile quaternium lipidoid enabling improved mRNA-LNP stability and spleen-selective mRNA transfection. Google Scholar Chen, J. et al. Combinatorial design of ionizable lipid nanoparticles for muscle-selective mRNA delivery with minimized off-target effects. Google Scholar Li, Z. et al. Enzyme-catalyzed one-step synthesis of ionizable cationic lipids for lipid nanoparticle-based mRNA COVID-19 vaccines. ACS Nano 16, 18936–18950 (2022). Google Scholar Han, X. et al. Fast and facile synthesis of amidine-incorporated degradable lipids for versatile mRNA delivery in vivo. Google Scholar Miao, L. et al. Delivery of mRNA vaccines with heterocyclic lipids increases anti-tumor efficacy by STING-mediated immune cell activation. Google Scholar Zhu, Y. et al. Screening for lipid nanoparticles that modulate the immune activity of helper T cells towards enhanced antitumour activity. Google Scholar Dahlman, J. E. et al. Barcoded nanoparticles for high throughput in vivo discovery of targeted therapeutics. Google Scholar Xue, L. et al. High-throughput barcoding of nanoparticles identifies cationic, degradable lipid-like materials for mRNA delivery to the lungs in female preclinical models. Google Scholar Rhym, L. H., Manan, R. S., Koller, A., Stephanie, G. & Anderson, D. G. Peptide-encoding mRNA barcodes for the high-throughput in vivo screening of libraries of lipid nanoparticles for mRNA delivery. Google Scholar Li, B. et al. Accelerating ionizable lipid discovery for mRNA delivery using machine learning and combinatorial chemistry. Google Scholar Eygeris, Y., Gupta, M., Kim, J. & Sahay, G. Chemistry of lipid nanoparticles for RNA delivery. Google Scholar Qiu, M., Li, Y., Bloomer, H. & Xu, Q. Developing biodegradable lipid nanoparticles for intracellular mRNA delivery and genome editing. Google Scholar Zhang, Y., Sun, C., Wang, C., Jankovic, K. E. & Dong, Y. Lipids and lipid derivatives for RNA delivery. Google Scholar Thalmayr, S. et al. Molecular chameleon carriers for nucleic acid delivery: the sweet spot between lipoplexes and polyplexes. Lin, Y. et al. Chemical evolution of amphiphilic xenopeptides for potentiated Cas9 ribonucleoprotein delivery. Dong, Y. et al. Lipopeptide nanoparticles for potent and selective siRNA delivery in rodents and nonhuman primates. Cheng, M. H. Y. et al. Induction of bleb structures in lipid nanoparticle formulations of mRNA leads to improved transfection potency. Jayaraman, M. et al. Maximizing the potency of siRNA lipid nanoparticles for hepatic gene silencing in vivo. Dilliard, S. A., Cheng, Q. On the mechanism of tissue-specific mRNA delivery by selective organ targeting nanoparticles. Tilstra, G. et al. Iterative design of ionizable lipids for intramuscular mRNA delivery. Qiu, M. et al. Lung-selective mRNA delivery of synthetic lipid nanoparticles for the treatment of pulmonary lymphangioleiomyomatosis. Fenton, O. S. et al. Synthesis and biological evaluation of ionizable lipid materials for the in vivo delivery of messenger RNA to B lymphocytes. Fei, Y. et al. Simplified lipid nanoparticles for tissue- and cell-targeted mRNA delivery facilitate precision tumor therapy in a lung metastasis mouse model. Xue, L. et al. Rational design of bisphosphonate lipid-like materials for mRNA delivery to the bone microenvironment. Paunovska, K. et al. A direct comparison of in vitro and in vivo nucleic acid delivery mediated by hundreds of nanoparticles reveals a weak correlation. Whitehead, K. A. et al. In vitro–in vivo translation of lipid nanoparticles for hepatocellular siRNA delivery. Huang, P., Deng, H., Wang, C., Zhou, Y. Cellular trafficking of nanotechnology-mediated mRNA delivery. Hashiba, K. et al. Branching ionizable lipids can enhance the stability, fusogenicity, and functional delivery of mRNA. Liu, S. et al. Membrane-destabilizing ionizable phospholipids for organ-selective mRNA delivery and CRISPR–Cas gene editing. Miao, L. et al. Synergistic lipid compositions for albumin receptor mediated delivery of mRNA to the liver. Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Sun, Y. et al. In vivo editing of lung stem cells for durable gene correction in mice. Wei, T. et al. Lung SORT LNPs enable precise homology-directed repair mediated CRISPR/Cas genome correction in cystic fibrosis models. An, M. et al. Engineered virus-like particles for transient delivery of prime editor ribonucleoprotein complexes in vivo. Davis, J. R. et al. Efficient prime editing in mouse brain, liver and heart with dual AAVs. Nelson, J. W. et al. Engineered pegRNAs improve prime editing efficiency. Chen, P. J. et al. Enhanced prime editing systems by manipulating cellular determinants of editing outcomes. Zou, Q. et al. Photoactivatable base editors for spatiotemporally controlled genome editing in vivo. This work was supported by the National Key Research and Development Program of China (2023YFC3405000 and 2024YFB3814800 to Q.C. ); Noncommunicable Chronic Diseases—National Science and Technology Major Project (2023ZD0500600 to T.W. ); the National Natural Science Foundation of China (82272139 to Q.C. ); the Beijing Natural Science Foundation (Z240028 to T.W. ); the ‘Open Competition to Select the Best Candidates' Key Technology Program for Cell and Gene Therapy of NCTIB (NCTIB2023XB02001 to Q.C. ); Beijing–Tianjin–Hebei Basic Research Cooperation Project (22JCZXJC00030 to Q.C. ); Project from Beijing Institute for Stem Cell and Regenerative Medicine (2023FH206 to T.W. ); Initiative Scientific Research Program, Institute of Zoology, Chinese Academy of Sciences (2023IOZ0204 to T.W. ); Fund from the Key Laboratory of Organ Regeneration and Reconstruction, Chinese Academy of Sciences (2024KF11 to Q.C. ); the China Postdoctoral Science Foundation (2023M740101 to Y.L. ); and the Open Research Fund of Beijing Advanced Center of RNA Biology (BEACON) at Peking University. In addition, we would like to thank Y. Chen from Tianjin Medical University for kindly providing PE reporter cells and mice and also thank W. Zhang for her help on cryo-TEM sampling and imaging at the Cryo-Electron Microscopy Platform, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences (IGDB). We would also like to thank S. Xiang and L. Yu from Starna Therapeutics for their help on the analysis of RNA integrity. These authors contributed equally: Yi Lin, Mengyao Li. Department of Biomedical Engineering, College of Future Technology, Peking University, Beijing, China Yi Lin, Yan Zong, Xiaoqiong Tan & Qiang Cheng State Key Laboratory of Organ Regeneration and Reconstruction, Institute of Zoology, Chinese Academy of Sciences, Beijing, China Mengyao Li, Zijin Luo, Yanan Meng, Hongyu Ren, Xiaolu Yu, Fan Liu & Tuo Wei Beijing Institute for Stem Cell and Regenerative Medicine, Beijing, China Mengyao Li, Zijin Luo, Yanan Meng, Hongyu Ren, Xiaolu Yu, Fan Liu & Tuo Wei University of Chinese Academy of Sciences, Beijing, China Mengyao Li, Zijin Luo, Yanan Meng, Hongyu Ren, Xiaolu Yu, Fan Liu & Tuo Wei Beijing Advanced Center of RNA Biology (BEACON), Peking University, Beijing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the research. performed the experiments. All authors were involved in the data analysis. wrote the paper, and all authors discussed and commented on it. directed the research. Correspondence to Tuo Wei or Qiang Cheng. are authors of a patent application related to this work (PCT/CN2024/104390) filed by Peking University in China. are authors of a patent related to this work (CN202510238302.0) filed by Institute of Zoology, Chinese Academy of Sciences, in China. The other authors declare no competing interests. Nature Materials thanks Dan Peer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Lipid molar ratios and chemical structures of four lipid components used for formulating PIL-based mRNA-LNPs. b–d, Hydrodynamic particle size (b), polydispersity index (c), and zeta potential (d) of PIL-based mRNA-LNPs. e,f, mRNA encapsulation efficiency (e) and apparent pKa (f) of PIL-based mRNA-LNPs. g, Enhanced intramolecular electrostatic repulsion of PILs with increasing AIFA numbers causes a decrease in their apparent pKa due to more closely packed ionizable amino acids. h, Intensity size distribution of a12K4 LNPs. i, TNS assay curve for determining the apparent pKa of a12K4 LNPs (pKa = 6.21). Apparent pKa was defined as the point at which 50% of TNS fluorescence was achieved. j-m, Hydrodynamic particle size (j), zeta potential (k), polydispersity index (l), and mRNA encapsulation efficiency (m) of LNPs containing PILs with hybrid tails. n–r, Hydrodynamic particle size (n), zeta potential (o), polydispersity index (p), mRNA encapsulation efficiency (q), and apparent pKa (r) of LNPs composed of different lysine analogues. s, Representative cryo-TEM image of a12Dab4 LNPs. Scale bar: 50 nm. The Cryo-TEM experiment was repeated for 4 times and similar results were obtained. All data are presented as mean ± s.d. (n = 3 biologically independent samples). Panel g created with BioRender.com. a16K2 LNPs initially mediate mRNA expression predominantly in the spleen (> 90.0%) and can be shifted to the liver and lungs by modifications with histidine (H)/cysteine (C) and lysine (K), respectively. aam12K5 LNPs initially mediate mRNA expression predominantly in the spleen (> 80.0%) and can be shifted to the liver and lungs by modifications with histidine (H) and lysine (K)/arginine (R), respectively. a, Schematic illustration of improved specificity of mRNA expression in the thymus by reducing expression in the lungs using the Thymus PILOT LNPs with miR126 targeting site (126ts)-incorporated Luc mRNA. b, Bioluminescence images and quantification of mRNA expression in major organs and the thymus at 6 h post-injection. Am-KYa12K4 Thymus PILOT LNPs containing 1.0 mg kg-1 Luc mRNA or 126ts-Luc mRNA were used in this study. The specificity of mRNA expression in the thymus was increased to 82.4% using 126ts-Luc mRNA. c, Schematic illustration shows that Cre mRNA delivery by the Bone PILOT LNPs activates tdTomato expression in Ai9 reporter mice. d, Specific Cre-edited BM mature immune cells and hematopoietic stem cells (HSCs) in the bone marrow. LT-HSC: long-term hematopoietic stem cell; LMPP: lympho-myeloid primed progenitor; ST-HSC: short-term hematopoietic stem cell. Data are presented as mean ± s.d. (n = 3 biologically independent samples). Panels a and c and schematics in d created with BioRender.com. The in vivo activity of PILs are mainly determined by the type and length of alkyl tails, lysine and its analogues, and the number of AIFA building blocks, while the in vivo selectivity of PILs are influenced by the type and length of alkyl tails, the number of AIFA building blocks, and terminal modifications. a, Chemical structures of phospholipids and PILs used for the optimization of PILOT LNPs. b, c, Bioluminescence images (b) and quantification (c) of mRNA expression in various organs at 6 h following treatment of mice with a12K4 LNPs containing different phospholipids. d, e, Bioluminescence images (d) and quantification (e) of mRNA expression in various organs at 6 h following treatment of mice with a12Dab4 LNPs containing different phospholipids. f, Lipid molar ratios used for the optimization of PILOT LNPs. g, h, Bioluminescence images (g) and quantification (h) of mRNA expression in various organs at 6 h following treatment of mice with a12K4 LNPs prepared at different ratios of lipid components. i, j, Bioluminescence images (i) and quantification (j) of mRNA expression in various organs at 6 h following treatment of mice with a12Dab4 LNPs prepared at different ratios of lipid components. k,l, Bioluminescence images (k) and quantification (l) of mRNA expression in various organs at 6 h following treatment of mice with a12Dab4 LNPs formulated at different nitrogen-to-phosphate (N/P) ratios and the benchmark ALC-0315 LNP at an N/P ratio of 10. m, Relative expression in the liver at 6 h following treatment of mice with a12Dab4 Liver PILOT LNPs formulated at different N/P ratios. Data are presented as mean ± s.d. (n = 3 biologically independent samples). Statistical significance was analyzed by the two-tailed unpaired t-test; **** P < 0.0001; *** P < 0.001; ** P < 0.01; * P < 0.05; ns, not significant. Supplementary Figs. 1–91, Tables 1–6, Methods and references. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Extended Data Fig. Statistical source data for Extended Data Fig. Statistical source data for Extended Data Fig. Statistical source data for Extended Data Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Lin, Y., Li, M., Luo, Z. et al. Tissue-specific mRNA delivery and prime editing with peptide–ionizable lipid nanoparticles. Received: 29 July 2024 DOI: https://doi.org/10.1038/s41563-025-02320-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Materials © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            