
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-19</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09756-8'>Structural basis of regulated N-glycosylation at the secretory translocon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 17:12:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Most human secretory pathway proteins are N-glycosylated by oligosaccharyltransferase (OST) complexes as they enter the endoplasmic reticulum (ER)1,2,3. Recent work revealed a substrate-assisted mechanism by which N-glycosylation of the chaperone glucose-regulated protein 94 (GRP94) is regulated to control cell surface receptor signalling4. Here we report the structure of a natively isolated GRP94 folding intermediate tethered to a specialized CCDC134-bound translocon. Together with functional analysis, the data reveal how a conserved N-terminal extension in GRP94 inhibits OST-A and how structural rearrangements within the translocon shield the tethered nascent chain from inappropriate OST-B glycosylation. These interactions depend on a hydrophobic CCDC134 groove, which recognizes a non-native conformation of nascent GRP94. Our results define a mechanism of regulated N-glycosylation and illustrate how the nascent chain remodels the translocon to facilitate its own biogenesis. Protein N-glycosylation is catalysed by OST complexes in the ER membrane. About 20% of human proteins are modified with N-glycans, which affect their folding, degradation, trafficking and function1,2. Defects in the cellular N-glycosylation machinery cause a group of inherited diseases called congenital disorders of glycosylation (CDG)5, and pathogenic missense mutations resulting in a gain or loss of N-glycosylation have been described6,7. Metazoan genomes encode two OST complexes3. OST-A associates with ER membrane-bound ribosomes docked to SEC61 and TRAP (the ‘secretory' translocon), giving it priority access to the unfolded nascent chain as it enters the lumen8,9. Sites skipped by OST-A can be modified by OST-B, which can act co- or post-translationally but does not associate with the ribosome–translocon complex (RTC)10. Notably, the presence of an acceptor sequon (Asn-X-Thr/Ser, X≠Pro) does not ensure its modification11,12. Factors such as sequence context, folding kinetics and cellular metabolic state influence acceptor site usage, leading to variability across cells, tissues and disease states13,14. The functional consequences of this variability remain poorly understood. Glycosylation of normally unused acceptor sequons can target proteins for degradation by the ERAD pathway15,16. An important example is GRP94 (or HSP90B1), an abundant ER chaperone with six acceptor sequons (Fig. Whereas GRP94 is typically glycosylated at a single site (N217), glycosylation of up to five ‘facultative' sites has been observed in cells subjected to different forms of ER stress17,18,19,20. a, Domain organization of GRP94, highlighting the one constitutive (N217) and five facultative N-glycosylation acceptor sites. The signal peptide (SP), pre-N segment (pN) and first helix of the N-domain (αN1, yellow) are also indicated. b, log2[transcript enrichment] (IP abundance/input abundance) plotted against log2[length of the longest lumenal (translocated) segment]; data are the average of two biological replicates for all secretory and transmembrane transcripts recovered following FLAG–CCDC134 or FLAG–FKBP11 affinity purification (n = 2,296). c, Comparison of log2[transcript enrichment] for all secretory and transmembrane transcripts recovered in the CCDC134 and FKBP11 samples. Note that FKBP11 and CCDC134 are among the most enriched transcripts in their respective samples, because each protein is tagged at its N terminus (immediately following the SP); this serves as an internal control for the immunoprecipitation. d, Comparison of log2[transcript abundance] (transcripts per million, TPM) for all transcripts recovered following FKBP11 and CCDC134 affinity purification (n = 9,242). e, Volcano plots for all secretory and transmembrane transcripts showing −log10[adjusted P-value] versus log2[enrichment] for each sample. For clarity, FKBP11 and CCDC134 transcripts are not shown. f, CCDC134 and FKBP11 interaction profiles for GRP94; mean (line) and range (shading) for two biological replicates are indicated. For source data related to panels b–e, see Supplementary Table 1. A pathway regulating GRP94 hyperglycosylation was recently identified through genome-wide CRISPR–Cas9 screening4. This process depends critically on the lumenal ER protein CCDC134 and components of OST-A that recognize a conserved, metazoan-specific N-terminal extension of GRP94 to suppress glycosylation of its facultative sites during synthesis. Indeed, mutations that cause loss of CCDC134 function impair WNT and IGF1R signalling, causing the bone disorder osteogenesis imperfecta4,23,24,25. The structural basis by which these factors control GRP94 acceptor site usage at the secretory translocon remains unclear. In earlier work, we used affinity purification, mRNA sequencing and mass spectrometry to identify FKBP11 as a transmembrane protein that can bind ribosome-associated secretory translocons during synthesis of proteins with long lumenal segments26. Notably, CCDC134 and GRP94 were identified as co-purifying factors. In a reciprocal experiment, FKBP11 was recovered following affinity purification by means of a FLAG tag on CCDC134, and this interaction was dependent on the ribosome, as neither FKBP11 nor CCDC134 recovered the other component in the absence of ribosomes (Extended Data Fig. To identify cotranslational clients of FKBP11 and CCDC134, we used a recently developed ribosome profiling method optimized for low-input samples27,28 (Extended Data Fig. As observed previously for FKBP11 (ref. 26), affinity-purified CCDC134-bound ribosomes were modestly enriched for transcripts encoding proteins with long translocated segments (Fig. This seemed to depend on FKBP11, as most transcripts enriched by CCDC134 were also enriched by FKBP11, but not vice versa (Fig. Notably, GRP94 was strongly enriched by both factors and was also the most abundant transcript in each elution (Fig. Positional analysis revealed distinct interaction profiles for FKBP11 and CCDC134 during synthesis of the 803-residue GRP94 polypeptide. FKBP11 arrived first, when the nascent chain was about 161 residues long (Fig. At this point, GRP94 residues 22–106 were in the lumen (assuming that approximately 55 residues are buried in the ribosome exit tunnel and SEC61 channel). CCDC134 began to arrive shortly thereafter, when the nascent chain was about 204 residues long. This is consistent with a previous analysis of GRP94 translocation intermediates assembled by in vitro translation4. CCDC134 was stabilized further as synthesis continued, peaking as the GRP94 N-domain finished translocating into the lumen. Once recruited, both factors persisted until translation was terminated. Thus, FKBP11 and CCDC134 engage the translocon early during GRP94 synthesis and remain bound until synthesis is complete. To understand how these factors are organized, we prepared FKBP11-bound ribosomes from FLAG–FKBP11 HEK293 cells and solved the structure by single-particle cryo-electron microscopy (cryo-EM) (Fig. The maps show well-defined density for SEC61, OST-A, A/P- and P/E-site transfer RNAs and the ribosome. Density was also visible for the TRAP complex, portions of the lipid-linked oligosaccharide (LLO) donor and several N-glycans. Nascent chain density was visible in the ribosome exit tunnel and SEC61 channel, which adopts an open conformation with its displaced plug helix packed against SEC61β, SEC61γ and DC2 (ref. Thus, these natively isolated FBKP11–ribosome complexes represent physiologically relevant complexes captured in the act of translation and translocation. a, Overview of the cryo-EM translocon map after post-processing using DeepEMhancer. b, Close-up of CCDC134 (blue), FKBP11 (purple) and nascent GRP94 (magenta) fit to the cryo-EM translocon map low-pass filtered to 8 Å resolution. c, Structural comparison of folded, nucleotide-bound GRP94 (PDB ID 5ULS; grey) and nascent GRP94 (magenta), showing conformational differences in the lid and helix αN1 (yellow). Secondary structure annotation for the GRP94 pre-N segment and flanking regions is shown at the bottom, based on experimental structures and AlphaFold (AF) models. d, Close-up of CCDC134 and part of the GRP94 ‘bridge' helix fit to the cryo-EM translocon map low-pass-filtered to 6 Å resolution. e, Overview of the final model showing the arrangement of CCDC134, FKBP11 and nascent GRP94 relative to components of the ribosome-bound secretory translocon. Additional lumenal density was visible near OST-A (Fig. This density is distinct from weak, unassigned densities visible near OST-A in a recent cryo-electron tomography (cryo-ET) study29. Although FKBP11 and CCDC134 do not interact in the absence of ribosomes, AlphaFold predicted a high-confidence complex that was fit into a portion of this density with only minor adjustments to each subunit's flexible C-terminal region (Fig. This configuration buries the canonical ligand binding site of FKBP11 and positions its conserved and positively charged C-terminal helix, previously shown to be important for ribosome binding26, adjacent to uL22 and 28S rRNA H24 on the cytosolic side of the membrane (Extended Data Fig. The remaining density was assigned to residues 25–283 of nascent GRP94, which is observed in a non-native conformation (Fig. The pre-N segment (residues 22–74), which is largely unstructured or disordered in previously determined structures of GRP94, was modelled into continuous density that wraps around the OST-A active site, in good agreement with high-confidence AlphaFold modelling (Extended Data Figs. AlphaFold was also used to guide placement of the first N-domain helix (αN1, residues 82–91), which normally packs against the ATP-binding site in folded GRP94 (Fig. 2c), into density extending along the surface of CCDC134 (Fig. Portions of the globular N-domain, including its central helix and most of the eight-stranded β-sheet, were placed into weak but characteristic density adjacent to CCDC134 (Fig. No density was observed for bound nucleotide, the flexible N-domain active-site ‘lid' or the M- and C-terminal domains. Thus, the structure reveals a bona fide GRP94 translation intermediate(s) captured in a partially folded state while bound to the secretory translocon and two accessory factors, FKBP11 and CCDC134 (Fig. The pre-N segment extends deep into the OST-A active-site cavity, burying more than 3,100 Å2 of surface area against STT3A (about 1,680 Å2), DC2 (about 540 Å2) and SEC61α (about 930 Å2) (Fig. This negatively charged region (residues 25–79) (Fig. 3c,d) is rich in conserved residues4 and forms an extensive network of intra- and inter-molecular contacts. This region is highly sensitive to mutation, as most variants tested in RKO cells led to GRP94 hyperglycosylation and degradation and, consequently, to reduced cell surface expression of its clients IGF1Rβ and LRP6 (Fig. Notably, these mutations did not affect GRP94 folding or chaperone activity, as they did not impair the activity of a variant lacking all five facultative sequons (5N; Extended Data Fig. Thus, the metazoan-specific pre-N segment regulates GRP94 hyperglycosylation, not its catalytic activity. a, Orthogonal views of the nascent GRP94 pre-N segment (magenta) wedged between two OST-A subunits, STT3A (green) and DC2 (salmon), and SEC61α (yellow). b, Close-up showing pre-N segment contacts with β-hairpins in DC2 and SEC61α. c, Overview of the pre-N segment fit to the cryo-EM translocon map after low-pass filtering by local resolution. The positions of residues tested in cells are indicated. d, Close-up of the same cryo-EM map, highlighting residues in the pre-N SRT region. e, ER and plasma membrane (PM) levels of IGFR1β and LRP6 and levels of monoglycosylated (mg) and hyperglycosylated (hg) GRP94 in GRP94 knockout RKO cells stably expressing wild-type (WT) GRP94 or the indicated variant. f, Superposition of OST structures bound to short substrate analogues (cyan, blue) or the SRT motif of nascent GRP94 (magenta). g, Organization of the GRP94 SRT motif and LLO within the STT3A (green) active site. h, Glycosylation of GRP94 and PSAP (an OST-A specific substrate) and cell surface abundance of LRP6 in STT3A knockout RKO cells stably expressing WT STT3A or the indicated variant. Close-up views showing the location of E623R (i) and D363R point mutants in STT3A (green) (j) that selectively affect GRP94 (but not PSAP) glycosylation. Data shown in panels e and h are representative of at least two independent experiments. For gel source data, see Supplementary Fig. The pre-N interactions within the acceptor site binding pocket mimic those of previously determined structures of short (about seven residues) peptides bound to yeast OST and human OST-B (refs. Density was observed for portions of the LLO glycan, polyprenyl chain and pyrophosphate, but not an active-site divalent cation (Extended Data Fig. The lumenal EL5 loop of STT3A is in an ordered, ‘engaged' conformation, packed against the LLO and GRP94. Notably, the conserved GRP94 ‘SRT' motif (residues 42–44) forms a tight 180° turn that positions S42 close to the donor LLO substrate (Fig. Here, the hydroxyl group of T44 contacts the STT3A WWD motif, which is essential for acceptor sequon recognition33,34, explaining why mutations in either site lead to GRP94 hyperglycosylation and degradation4. The central (+1) position of an acceptor sequon can be any residue except proline, which is incompatible with the tight turn conformation required for OST-A binding. As expected of a substrate mimic, an arginine-to-proline mutation at GRP94 position 43 resulted in hyperglycosylation and degradation (Fig. Notably, mutating this position to a negatively charged glutamate (and, to a lesser extent, alanine or isoleucine) also caused GRP94 hyperglycosylation, whereas lysine was tolerated4. A positive charge at the +1 position of GRP94 is probably required to neutralize the flanking negative charges on E40 and D46 that are brought together on binding (Fig. These observations define the pre-N segment as a pseudo-substrate inhibitor that occludes the STT3A acceptor site binding pocket during GRP94 synthesis to limit glycosylation of the facultative sites. This mechanism differs from that proposed for the small-molecule OST inhibitor NGI-1, which traps a catalytically inactive conformation of LLO but does not prevent acceptor peptide binding21. The persistent binding of the 53-residue pre-N segment contrasts with the transient engagement of nascent polypeptides as they enter the lumen, suggesting differences in how they interact with OST-A35,36,37. Unlike the conserved and extensive interface used by the pre-N domain, recognition of bona fide acceptor sequences is primarily mediated by contacts between the acceptor sequon and STT3A31,32,38. Consistent with this distinction, we identified distal STT3A variants (D363R and E623R) that caused GRP94 hyperglycosylation and reduced surface expression of LRP6 but had no effect on the OST-A-specific substrate prosaposin (PSAP)10 (Fig. Notably, these residues are close to the known CDG mutations Y360S and V626A39,40,41, the latter of which causes GRP94 hyperglycosylation in patient fibroblasts19. This suggests that the disease phenotypes of some CDG-causing STT3A mutations may be because of GRP94 and its clients, rather than (or as well as) a general N-glycosylation defect. Pre-N binding to OST-A is necessary but not sufficient to fully suppress GRP94 hyperglycosylation, which also requires CCDC134 (ref. 4). CCDC134 is recruited by a network of interactions with the translocon and nascent GRP94 (Fig. Although FKBP11 is well positioned to help recruit CCDC134, this is not strictly required, because HEK293 and RKO cells lacking FKBP11 did not show a GRP94 hyperglycosylation defect (Extended Data Fig. However, STT3A variants carrying mutations in the small CCDC134–STT3A interface were unable to fully suppress GRP94 hyperglycosylation4. Similarly, a CCDC134 variant lacking its OST-A-binding C-terminal extension (Δ194–224) (but retaining the ‘QSEL' ER retention signal) did not fully rescue a CCDC134 knockout, probably because of impaired assembly at the translocon (Fig. a, Overview of CCDC134 (blue) interactions at the secretory translocon. The top inset shows a close-up of the GRP94 ‘bridge' helix (magenta, with αN1 in yellow) connecting the OST-A active site (green) and CCDC134. The bottom inset shows the helical C-terminal extension of CCDC134 packed against a cleft between OST48 (olive) and RPN2 (lime green). b, Structural comparison of folded, nucleotide-bound GRP94 (PDB ID 5ULS; grey) and nascent GRP94 (magenta), showing CCDC134 bound to a non-native conformation of nascent GRP94. In this tethered conformation, the canonical αN1 binding site in the N-domain is occluded by CCDC134. c, GRP94 αN1 binding surface of CCDC134 coloured by hydrophobicity. d, Close-up of hydrophobic residues lining the CCDC134 groove. e, Glycosylation of GRP94 in CCDC134 knockout cells stably expressing wild-type (WT) CCDC134 or the indicated variant. The Δ194–224 construct is only visible in the anti-HA blot because the anti-CCDC134 antibody recognizes a C-terminal epitope that is deleted from this construct. f, ER and plasma membrane (PM) levels of IGFR1β and levels of monoglycosylated (mg) and hyperglycosylated (hg) GRP94 in GRP94 knockout RKO cells stably expressing the indicated GRP94 or GRP94-5N variant. See also Extended Data Fig. Data shown in panels e and f are representative of at least two independent experiments. For gel source data, see Supplementary Fig. The globular domain of CCDC134 makes further interactions with nascent GRP94, burying almost 1,100 Å2 of CCDC134 surface area in two distinct interfaces (Fig. The smaller interface (about 350 Å2) involves the partially folded GRP94 N-domain, which packs loosely against CCDC134. The larger interface (about 720 Å2) involves the amphipathic GRP94 helix αN1 (including residues 82–91 and several flanking residues), which flips about 180° from its canonical position in folded GRP94 to pack against a conserved hydrophobic groove on the surface of CCDC134 (Fig. In this configuration, the canonical αN1 binding site in the N-domain is occluded by CCDC134 (Fig. Notably, the αN1 region is required for CCDC134-dependent suppression of GRP94 hyperglycosylation4. Consistent with this, CCDC134 variants carrying structure-guided hydrophobic-to-charge mutations within the hydrophobic groove were recruited to the translocon but did not prevent GRP94 hyperglycosylation in CCDC134 knockout RKO cells (Fig. 4d,e and Extended Data Fig. In nascent GRP94, αN1 and the pre-N segment are connected by a long helical element, termed the bridge helix (residues 63–91) (Fig. This helix is not observed in experimental and predicted structures of GRP94, which show considerable conformational flexibility in this region42,43 (Fig. The paucity of direct contacts between CCDC134 and STT3A suggested that rigidity of the bridge helix may be functionally important. Consistent with this, individual helix-disrupting proline substitutions at positions along the bridge helix (I66, L69) caused GRP94 hyperglycosylation, whereas alanine substitutions were tolerated (Fig. Notably, these mutations did not affect folding or chaperone activity of the mature protein, as they did not impair the activity of the 5N variant. These data show how CCDC134 recognizes a non-native conformation of nascent GRP94 to occlude the OST-A active site and tether the growing polypeptide chain to the translocon. This explains why CCDC134 is stabilized at the secretory translocon during GRP94 synthesis4 (Fig. 1f) and why fully folded GRP94 does not inhibit OST-A or OST-B post-translationally, despite its abundance in the ER lumen. The pre-N and N-domain of GRP94 remain tethered to the translocon until translation is terminated (Fig. Density for the M- and C-domains—which account for nearly 60% of the mature protein—is not visible in the structure, and superimposed full-length GRP94 monomers42,43 show numerous intra- and inter-molecular clashes, including with CCDC134 and subunits of OST-A (Extended Data Fig. This suggests that the native N–M and M–C domain interfaces cannot form until synthesis is complete. The TRAP complex adopts different positions in previously determined translocon structures29,44,45. In the CCDC134-bound secretory translocon, it shifts away from the tethered and partially folded GRP94 N-domain. In its place, we observed density for a small four-helix bundle, which was assigned to KCP2, the DC2-binding subunit of OST-A (refs. The DC2 N-terminus extends more than 20 Å along the membrane to bind KCP2 by means of a cleft on its cytosolic face. This interaction buries more than 1,200 Å2 of DC2 surface area and loosely tethers KCP2 near the SEC61α hinge, SEC61γ, the ribosome and the repositioned TRAP TMDs. a, Comparison of subunit organization near the SEC61α hinge, viewed from the cytosol (left, cryo-ET structure of the secretory translocon, PDB ID 8B6L; right, present structure). Arrow indicates movement of the TRAP transmembrane segments, which is accompanied by displacement of the TRAPα TMD. The KCP2 subunit of OST-A is visible in its place, stabilized by a long N-terminal extension of DC2 (salmon) and packed loosely against the TRAP transmembrane bundle and the SEC61α hinge. b, Comparison of TRAP complex position in actively translocating RTCs, highlighting the large 15–30-Å displacement of its lumenal domains. RAMP4 (orange) is bound at the lateral gate of fully open, plug-displaced SEC61 (yellow) in both structures. c, View of a new interface formed between RPN2 (green) and the repositioned TRAPδ (tan) and TRAPβ (light tan) lumenal domains (inset). d, View of the lumenal vestibule formed by TRAP, OST-A, CCDC134 and the nascent GRP94 N-domain. e, Model for regulated GRP94 N-glycosylation. For clarity, only the lumenal domain of FKBP11 is depicted. Notably, the TRAP lumenal domain shifts from its position at the SEC61α hinge loop44 to form a new interface with RPN2 (Fig. This creates a large lumenal vestibule adjacent to the SEC61 channel exit that is lined by TRAP, OST-A, CCDC134 and the partially folded GRP94 N-domain (Fig. Here, the downstream domains of tethered GRP94 can begin folding in close proximity to the translocon, thereby limiting inappropriate cotranslational glycosylation of the M-domain by OST-B acting in trans10. Our findings lead to a molecular model for regulated GRP94 N-glycosylation (Fig. FKBP11 is recruited to the secretory translocon as the pre-N segment translocates through SEC61 and begins to occlude the OST-A active site. Once FKBP11 is bound, CCDC134 is progressively stabilized through a network of interactions with FKBP11, OST-A and the elongating GRP94 chain. Key to this is binding of the CCDC134 hydrophobic groove to a transiently exposed amphipathic helix (αN1) in GRP94. This configuration tethers GRP94 to the translocon and stabilizes pre-N binding to the STT3A active site to prevent glycosylation of downstream sites by OST-A. The relatively complex architecture and high contact order49 of the N-domain suggest that it folds slowly, leaving its constitutive glycosylation site (N217) accessible to OST-B (acting in trans) as it enters the lumen. Once packed against CCDC134, however, the partially folded N-domain contributes to a lumenal vestibule that shields downstream facultative sites from OST-B while the M-domain attempts to fold. As synthesis finishes and GRP94 is released, the αN1 helix packs against the N-domain, leaving the mature (folded) GRP94 incapable of inhibiting OST-A or OST-B. This pathway does not seem to be strictly required for proper GRP94 folding, as constructs lacking all five facultative sites are functional in RKO cells, even when harbouring mutations that disrupt their interactions with OST-A (Extended Data Fig. We propose that they are part of a cotranslational quality control system that has evolved to rapidly detect and degrade GRP94 misfolding events. This would be analogous to cotranslational ubiquitination of nascent polypeptides on cytosolic ribosomes50,51,52, which preferentially targets proteins that fold inefficiently. It is also one of the most abundant proteins in the ER lumen. Thus, even a fraction of misfolded GRP94 might be sufficiently toxic in certain cell types or developmental stages to necessitate a dedicated quality control checkpoint at the RTC. Rather than facilitating folding, the function of CCDC134 would be to shield the facultative sites from glycosylation—by occluding the OST-A active site and restricting access of the tethered nascent chain to OST-B—while GRP94 attempts to fold. Our data suggest that FKBP11 functions as a ribosome-binding adaptor to help recruit CCDC134 to the secretory translocon during GRP94 biosynthesis. Burial of its nominal catalytic site by CCDC134 argues against a role as a nascent chain prolyl isomerase. Although we did not detect GRP94 hyperglycosylation defects in knockout HEK293 or RKO cells, FKBP11 may be required in other contexts. Indeed, FKBP11 is highly expressed in specialized secretory cells and is upregulated in response to various ER stresses55,56,57,58. Notably, FKBP11 knockdown was recently reported to suppress proinflammatory cytokine production in endothelial cells by disrupting NF-κB signalling59, consistent with a link to GRP94 biogenesis and cell-surface receptor signalling. Our selective ribosome profiling data hint at an additional, more general function for FKBP11 and CCDC134 at the translocon. Hundreds of transcripts were enriched by FKBP11 and CCDC134, albeit at much lower levels than GRP94 (Fig. Notably, these proteins lack obviously conserved pre-N-like sequences, suggesting that FKBP11 and CCDC134 do not regulate their glycosylation. An intriguing possibility is that FKBP11 and CCDC134 also function more broadly to shield aggregation-prone hydrophobic segments that are transiently exposed during translocation into the ER lumen. An important concept emerging from recent work is that the subunit composition of the translocon is dynamic29,60,61,62. Understanding how different substrates drive translocon remodelling to support their own biogenesis is an important goal. For nascent GRP94, a key signal is the transient exposure of an amphipathic helix that is recognized by CCDC134 to suppress glycosylation at the secretory translocon. Further studies are needed to explore how the cotranslational folding and modification of other nascent polypeptides is coordinated at the translocon. The following secondary antibodies conjugated to horseradish peroxidase were used: Peroxidase AffiniPure Donkey Anti-Mouse IgG (H+L) (Jackson ImmunoResearch Laboratories, #715-035-150, RRID: AB_2340770, 1:10,000); Peroxidase AffiniPure Donkey Anti-Rabbit IgG (H+L) (Jackson ImmunoResearch Laboratories, #111-035-144, RRID: AB_2307391, 1:10,000); Peroxidase Donkey Anti-Rabbit IgG (Fc specific) (Sigma-Aldrich, #SAB3700863, RRID: AB_3675584, 1:10,000); Peroxidase Rabbit Anti-Mouse IgG H&L (Abcam, #ab6728, RRID: AB_955440, 1:10,000). Secondary antibodies conjugated to IRDye 800CW were obtained from LI-COR (IRDye 800CW Donkey anti-Mouse IgG Secondary Antibody, 1:10,000). Doxycycline-inducible 3xFLAG–CCDC134 (tag inserted after the ER signal sequence) was cloned by polymerase chain reaction (PCR) amplification followed by Gibson assembly (New England Biolabs) into pcDNA5-FRT/TO. Doxycycline-inducible 3xHA–CCDC134 (tag inserted after the ER signal sequence) mutants were cloned by PCR amplification followed by Gibson assembly (New England Biolabs) into pLenti-TRE-rtta3G-BLAST4. pENTR2B-3xFLAG-GRP94 (wild type) and pENTR2B-3xFLAG-GRP945N (includes mutations in all five facultative sequons: S64A, S109A, S447A, T483I, T504I)4 were used as templates to clone all mutant constructs used to generate RKO addback cell lines. All mutant constructs were generated using PCR amplification followed by Gibson assembly and cloned into pLenti CMV Puro DEST63 using gateway methods. Doxycycline-inducible 3xFLAG–GRP94 (tag inserted after the ER signal sequence) constructs for transient expression in HEK293 cells were cloned by PCR amplification followed by Gibson assembly into pcDNA5/FRT/TO. These were used as templates to generate mutant constructs by QuikChange Site-Directed Mutagenesis (Agilent, 210519, 200514). pENTR2B-STT3A-FLAG4 was used as a template to generate STT3A mutants using PCR amplification followed by Gibson assembly and cloned into pLenti CMV Puro DEST63 using Gateway methods. All constructs were fully sequenced to confirm accuracy. Flp-In T-REx 293 cells (Invitrogen) were maintained at 37 °C and 5% CO2 in DMEM (Corning, MT10013CV) and supplemented with 10% foetal bovine serum (GeminiBio, 900-108) and 100 U ml−1 penicillin plus 100 µg ml−1 streptomycin (GeminiBio, 400-109). Cells were checked approximately every 6 months for mycoplasma contamination using the Universal Mycoplasma Detection Kit (ATCC, 30-1012K) and were found to be negative. 26) and STT3A60 knockout Flp-In T-REx 293 cell lines were generated by CRISPR–Cas9 as previously described. A CCDC134 knockout cell line was generated similarly. In brief, an sgRNA (5′-AGAAGATGTTTGAGGTGAAG-3′) targeting exon 3 of human CCDC134 was designed using the Synthego CRISPR design tool (https://design.synthego.com) and cloned into pSpCas9(BB)-2A-Puro plasmid (PX459; Addgene #62988). Twenty-four hours after transfection, cells were selected under 1 µg ml−1 of puromycin (InvivoGen, ant-pr-1) for 72 h. Single cells were isolated by sorting in 96-well plates and allowed to grow clonally. Further validation was performed by PCR amplification of ± 200 base pairs of PAM sequence and tracking INDEL by decomposition of Sanger sequencing data using the TIDE web tool64 and Synthego ICE analysis tool. An STT3B knockout cell line was generated similarly with sgRNA (5′-ACACATCATCTTGCATCTCA-3′) targeting exon 2 of human STT3B and validated by immunoblotting and tracking INDEL. A stable, doxycycline-inducible 3xFLAG-FKBP11 Flp-In T-Rex 293 cell line was generated in FKBP11 knockout cells as previously described26. A stable, doxycycline-inducible 3xFLAG-CCDC134 Flp-In T-REx 293 cell line was generated similarly. Briefly, pOG44 (encoding Flp- recombinase) and pcDNA5 SSER-3xFLAG-CCDC134 at a ratio of 9:1 with 3 µl of TransIT-293 transfection reagent (Mirus Bio, MIR2700) were co-transfected into CCDC134 knockout Flp-In T-REx 293 cells. After 24 h, cells were selected with 100 µg ml−1 of hygromycin for 12 days to obtain the stably integrated cell lines. For doxycycline-inducible expression of 3xFLAG–CCDC134, cells were tested in a range of doxycycline concentrations; overexpression was observed without further doxycycline in the culture media. Stable addback of tagged GRP94, STT3A or CCDC134 were introduced into clonally derived knockout RKO cells using the lentiviral expression system as described previously4. Briefly, virus was generated by transfecting HEK293T cells in six-well plates with 200 ng pMD2.G (Addgene), 400 ng psPAX2 (Addgene) and 800 ng of the desired pLenti CMV Puro DEST or pLenti-TRE-rtta3G-BLAST construct using 7 μl of 1 mg ml−1 polyethylenimine (Polysciences) per well. Lentivirus-conditioned media was collected after 48 h, filtered through a 0.45-μm filter and 0.5 ml of filtered lentivirus was mixed with 1.5 ml of complete media containing 8 μg ml−1 polybrene (MilliporeSigma). The diluted virus was then added to the indicated cells seeded on six-well plates. 24 h post-infection, cells were split and selected with puromycin (2 μg ml−1) or blasticidin (10 µg ml−1) for 3–7 days or until all of the cells on the control plate are dead. For doxycycline-inducible expression of 3xHA–CCDC134, cells were grown for 24 h in a range of doxycycline concentrations with 5 nM inducing low, near-endogenous expression levels. Typically, ten 15-cm dishes of the desired HEK293 cell line were grown to approximately 80% confluency, washed once with ice-cold PBS and collected by scraping in ice-cold PBS. Cells were collected by centrifugation at 1,000 × g for 5 min at 4 °C, flash-frozen in liquid nitrogen and stored at −80 °C for future use. For microsome preparation, the frozen cell pellet was thawed, resuspended in hypotonic homogenization buffer (10 mM HEPES pH 7.4, 10 mM KOAc, 1 mM MgCl2) and incubated on ice for 30 min. Cells were then mechanically lysed with 50 strokes in a glass Dounce homogenizer chilled in ice and sucrose was added to a final concentration of 200 mM. Whole cells and nuclear fragments were removed by two rounds of centrifugation at 2,000 × g for 10 min at 4 °C and the ER-enriched membrane fraction was collected from the supernatant by centrifugation at 10,000 × g for 15 min at 4 °C. Pelleted membrane fractions were resuspended to a total volume of 1 ml in microsome buffer (50 M HEPES pH 7.4, 250 mM KOAc, 10 mM MgCl2, 250 mM sucrose) supplemented with 0.5× cOmplete EDTA-free protease inhibitor cocktail (Roche, 11836170001) and 1 mM CaCl2. The microsome suspension was treated with 10,000 U ml−1 micrococcal nuclease (MNase) (NEB, M0247S) at 37 °C for 25 min followed by 5 U ml−1 RQ1 RNase-Free DNase (Promega, M610A) for 5 min at room temperature. Nuclease digestion was quenched with 2 mM EGTA and microsomes were collected by centrifugation at 10,000 × g for 15 min at 4 °C. Pelleted microsomes were resuspended in microsome buffer supplemented with 50 U ml−1 SUPERaseIn (Invitrogen, AM2696) to the desired concentration and flash-frozen in liquid nitrogen and stored at −80 °C. Microsomes were prepared as described above, with the following modifications. All buffers were supplemented with 100 ng ml−1 cycloheximide. Samples were kept on ice or at 4 °C throughout, unless otherwise noted. Homogenization buffer was supplemented with 0.5× cOmplete EDTA-free protease inhibitor cocktail (Roche). Nuclear fragments were removed by pelleting twice at 2,800 rpm (1,578 × g) for 10 min. Before MNase treatment, microsomes were resuspended with 1.5 ml of 100 mM KOAc microsome buffer. Micrococcal nuclease-treated 3xFLAG–CCDC134 and 3xFLAG–FKBP11 microsome pellets (from nine 15-cm dishes) were resuspended with 1 ml microsome buffer supplemented with 2.5% digitonin and 1× cOmplete EDTA-free protease inhibitor cocktail (Roche). The sample was rotated to solubilize for 30 min and insoluble material was removed by pelleting at 13,500 × g for 15 min. The soluble fraction was layered over a 1-ml sucrose cushion (50 mM HEPES pH 7.4, 150 mM KOAc, 1 M sucrose, 5 mM MgCl2, 0.25% digitonin) and centrifuged at 250,000 × g for 2 h in a TLA100.3 rotor. The resulting ribosomal pellet was resuspended in 1 ml microsome buffer supplemented with 1.25% digitonin. A portion of this was reserved for sequencing (which served as the ‘input' sample for ribosome profiling) and the remainder was immunoprecipitated in batch using 60 μl anti-FLAG M2 affinity gel (MilliporeSigma, A2220) and end-over-end mixing overnight at 4 °C. The sample was subsequently washed four times with 12 column volumes of microsome buffer supplemented with 0.4% digitonin. The sample was eluted twice with 1.5 column volumes of microsome buffer (with 200 mM KOAc, 0.4% digitonin, 0.5 mg ml−1 FLAG peptide). The eluate (which served as the ‘IP' sample for ribosome profiling) was collected using a pre-equilibrated spin filter column, frozen in liquid nitrogen and stored at −80 °C (‘IP' sample for sequencing). To prepare the ribosome profiling library, the RNA concentration in both input and IP samples was determined using the Qubit RNA high-sensitivity assay. To generate RNase footprints, samples with 800 ng of RNA were digested with RNase 1 (LGC Biosearch Technologies, N6901K) at 0.5 U µl−1 in a 90-µl reaction at room temperature for 1.5 h. The reactions were stopped by adding 400 µl of TRIzol (Ambion, 15596026), vortexing thoroughly and then adding 100 µl of chloroform. RNA in the aqueous layer was separated by centrifugation at 12,000 × g for 15 min at 4 °C and precipitated overnight by adding 0.1 volumes of 3 M sodium acetate (Invitrogen, AM9740), 10 mg of GlycoBlue (Invitrogen, AM9515) and 1.2 volumes of isopropanol (Fisher Scientific, BP2618). Purified RNase footprints were subjected to Rfoot-seq library preparation following a previously published method27,28. Ribosome profiling data were processed by trimming adaptors with Cutadapt v4.1, removing rRNA reads with Bowtie v2.2.6 (ref. 65) and aligning remaining reads to the human hg38 genome and RefSeq-defined transcriptome with TopHat v2.1.0 (ref. 66). Uniquely mapped reads were used for downstream analyses. High-quality reads showing 3-nt periodicity were adjusted to the ribosomal A-site using RibORF (refs. Transcript abundance was quantified as transcripts per million using HTSeq v2.0.3 (ref. 70). Transcript enrichment was calculated as the log2 ratio of IP to input samples, median-centred to all genes and tested for significance using chi-squared tests with Benjamini–Hochberg correction. Positional interaction profiles were generated from codon-level enrichment, computed as the log2 ratio of IP to input at each codon position. 3xFLAG-FKBP11 HEK293 cells were seeded in ten 15-cm dishes and induced with 0.1 ng ml−1 doxycycline for 24 h. Microsomes (A260 of 35) were prepared as described above and solubilized in microsome buffer supplemented with 1.5% digitonin and 1× cOmplete EDTA-free protease inhibitor cocktail (Roche). After 1 h at 4 °C on a rotating wheel, the solubilized microsomes were diluted 2× with microsome buffer (150 mM KOAc) and cleared by centrifugation at 15,000 × g for 15 min at 4 °C. The cleared supernatant was incubated overnight at 4 °C with approximately 70 µl of anti-FLAG M2 affinity gel (MilliporeSigma, A2220) pre-equilibrated with wash buffer (50 mM HEPES pH 7.4, 200 mM KOAc, 2 mM MgCl2, 1 mM glutathione, 0.15% digitonin). Resin was washed twice with five column volumes of wash buffer and bound material was eluted with 200 µl of the same buffer supplemented with 0.5 mg ml−1 FLAG peptide (APExBIO, A6001) for 10 min at room temperature. The eluate was passed through a pre-equilibrated approximately 30 µm Pierce spin column (Thermo Fisher, 69725), layered over a 300-µl sucrose cushion (50 mM HEPES pH 7.4, 150 mM KCl, 5 mM MgCl2, 1 mM glutathione, 500 mM sucrose, 0.15% digitonin) and centrifuged at 355,000 × g for 1 h at 4 °C in a TLA120.1 rotor (Beckman Coulter; Optima MAX-XP). The ribosome pellet was resuspended in 20 µl of wash buffer supplemented with 0.5 mg ml−1 FLAG peptide to a final A260 of 0.57 and immediately used to freeze grids for single-particle cryo-EM. Quantifoil R1.2/1.3 400 mesh gold grids with 2 nm ultrathin carbon were glow-discharged for 25 s immediately before use. Using a Thermo Fisher Vitrobot Mark IV, 3 μl of freshly prepared sample was applied to the grid, incubated at 22 °C and 100% humidity for 60 s, blotted for 7 s, drained for 0.5 s and frozen in liquid-nitrogen-cooled ethane. Three datasets (811, 1,684 and 4,074 videos) were collected on an FEI Titan Krios at 300 KV with EPU software, using a defocus range from −1.9 to −0.9 μm. Super-resolution videos (pixel size = 0.84 Å) were recorded at a nominal magnification of 53,000× using a K3 BioQuantum direct electron detector (Gatan) and a total electron exposure of 60 e− Å−2 over 52 frames. All data processing was performed using RELION 5.0 (ref. 71). Videos were motion-corrected using MotionCor2 (ref. The contrast transfer function (CTF) was estimated with CTFFIND4.1 (ref. 73). An initial particle set was generated by automated picking using the Laplacian-of-Gaussian method with a diameter range of 200 to 300 Å and a threshold of 1, applied to 4,074 videos binned by 2 during motion correction. Particles were extracted using a 128-pixel box (5.04 Å per pixel) and subjected to 2D classification (k = 100, T = 2) to select ribosome-like classes. Selected particles were re-extracted with a 384-pixel box (1.68 Å per pixel) and used for 3D refinement. The resulting map was used to train a Topaz74 model for autopicking across the full combined dataset (6,569 unbinned videos), yielding 741,983 particles. Topaz-picked particles were extracted with a 256-pixel box (2.52 Å per pixel) and subjected to 2D classification (k = 100, T = 2) using a 550 Å spherical mask and 100 classes. From these, 26 ribosome-like classes were selected (676,520 particles). 3D refinement was performed using a low-pass-filtered (60 Å) mammalian ribosome reference, followed by 3D classification (k = 9, T = 4). This yielded four classes corresponding to the secretory translocon (73.8%), one class with weak 40S subunit density (4.5%) and four classes containing poorly aligned particles (21.7%). The 498,812 particles from secretory translocon-containing classes were re-extracted with a 768-pixel box (0.84 Å per pixel) and refined in 3D. Two rounds of 3D refinement and CTF refinement were performed before each dataset was individually polished. The polished particles were then combined for a third round of 3D refinement. To further resolve compositional heterogeneity, focused classification with signal subtraction (k = 9, T = 4, 416-pixel box, 0.84 Å per pixel) was performed using a mask around the translocon region. This yielded three classes with density for the secretory translocon, GRP94, CCDC134 and FKBP11. These were combined (135,132 particles) and refined with local angular searches. An extra round of focused classification with signal subtraction (k = 4, T = 4) targeting GRP94, CCDC134, FKBP11 and STT3A yielded two classes: one with (41.2%) and one without (58.8%) density for the three accessory factors. The particle subset containing GRP94, CCDC134 and FKBP11 (55,750 particles) was subjected to local refinement, yielding Map 1 (translocon-only map). These particles were then re-extracted using a 480-pixel box (1.4 Å per pixel) and further refined. Maps used for model building and display were uniformly low-pass-filtered to different resolutions, low-pass-filtered by local resolution (in RELION) or post-processed using DeepEMhancer75. The human 60S subunit from PDB ID 6ZMI and the A/P- and P/E-site tRNAs from PDB ID 6W6L were used as starting models for the ribosome. The TRAP complex from PDB ID 8RJC30 (mutated to human) and the OST-A complex from PDB ID 8B6L29 were used as starting models for the secretory translocon. 77), as described below (Extended Data Fig. Protein sequences were obtained from UniProt78. The quality of each AlphaFold2 multimer prediction was initially assessed by the predicted local-distance difference test (pLDDT), which provides a per-residue confidence score for each subunit, the predicted aligned error (PAE), which provides a confidence measure of the predicted protein–protein interface(s), and by the overall (pTM) and interface (ipTM) predicted TM scores. The AlphaFold2 complexes were also validated by their fit to the experimental density, comparison with previously determined structures and mutational analysis. Low-confidence regions not supported by the cryo-EM density were removed from the model. All maps were used for model building. The 60S subunit, A/P and P/E tRNAs were placed as rigid bodies into Map 2 and a 43-residue poly-Ala segment of the nascent chain was modelled into exit tunnel density extending into the open SEC61 channel. The model was fit using tightly restrained real-space refinement in Coot79, including planar and trans peptide restraints, Ramachandran restraints and Geman-McClure local distance restraints. The OST-A complex from 8B6L was placed into Map 1 as a rigid body and adjusted as a single unit using tightly restrained real-space refinement in Coot, as above. The AlphaFold2 model of KCP2 (bound to the N-terminal region of DC2) was placed into the map and adjusted as a single unit (to maintain predicted inter-chain contacts) using tightly restrained real-space refinement in Coot. Weak density in the STT3A catalytic site was visible for a co-purifying LLO ligand, including the pyrophosphate group and portions of the isoprenyl tail and glycan moiety, and was built using PDB ID 8AGC as a guide. Several ordered N-glycans were also built for STT3A and RPN1. Further weak density was visible for what are probably digitonin and lipid molecules, but these were not modelled. The AlphaFold2 model for SEC61–RAMP4 complex was placed as a rigid body into Map 1 and the complex adjusted as a single unit using tightly restrained real-space refinement in Coot. The displaced plug region of the fully opened SEC61 channel was rebuilt using PDB ID 8RJB as a guide. Density for the TRAP complex, which was strongest in the lumenal regions, showed substantial displacement from its canonical position adjacent to SEC61. To model this, we divided the model into three separate regions: (1) the TRAPβ,γ,δ membrane and cytosolic segments; (2) TRAPα,β,δ lumenal domains; and (3) the TRAPα TMD and cytosolic tail. The membrane and lumenal regions were fit using tightly restrained real-space refinement in Coot. The TRAPα cytosolic tail was adjusted manually and fit using real-space refinement; no density was observed for the TRAPα TMD, which was not modelled. The FKBP11–CCDC134 AlphaFold2 model was docked into Map 1 as a rigid body. The C-terminal, positively charged helix of FKBP11 was modelled into weak density adjacent to the ribosome. The flexible C-terminus of CCDC134 was placed into helical density at the OST48–RPN2 interface, guided by an additional AlphaFold2 prediction. The resulting model was adjusted using tightly restrained real-space refinement in Coot. The AlphaFold2 model of GRP94 (residues 22–97) bound to DC2 and STT3A was placed into Map 1 by superimposing onto the previously placed STT3A subunit. GRP94 residues 73–97 showed poor fit to the helical cryo-EM density and were removed and the three-subunit complex (including GRP94 residues 22–72) was adjusted using tightly restrained real-space refinement in Coot. Next, the AlphaFold2 model of GRP94 (residues 22–97) bound to CCDC134 was docked into Map 1 by superimposing onto the previously placed model of CCDC134–FKBP11. GRP94 residues 63–92 are predicted by AlphaFold2 to form a single extended helix (‘bridge helix'), in excellent agreement with the experimental cryo-EM density. GRP94 residues 22–72 were removed and the two-subunit complex was adjusted using tightly restrained real-space refinement in Coot. Finally, the globular N-domain of GRP94 was docked into low-resolution density adjacent to CCDC134. Placement was guided by the central helical element of the N-domain and its β-sheet. No density was visible in the GRP94 nucleotide binding site or for the dynamic active-site ‘lid', which was removed from the model. Similarly, no density was visible for the GRP94 ‘charged linker' motif or for the M- and C-domains. The final GRP94 model (residues 22–283) was adjusted as a single complex with CCDC134, FKBP11, DC2 and STT3A using tightly restrained real-space refinement in Coot. The translocon model was subjected to real-space refinement in PHENIX80 alone (versus Map 1) or after combining with the 60S model (versus Map 2). Five rounds of global minimization and group B-factor refinement were carried out with Ramachandran and rotamer restraints (rotamer outliers were fixed using the fit option ‘outliers' and the target was set to ‘fix_outliers'), reference model restraints (starting model) and hydrogen-bonding, base-pair and stacking parallelity restraints applied to the rRNA. Secondary structure restraints were turned off. Final model statistics for the translocon and the 60S-translocon models are provided in Extended Data Table 1. Structure figures were generated with ChimeraX81. RKO whole-cell lysates were prepared in lysis buffer: 50 mM Tris at pH 8.0, 150 mM NaCl, 2% NP-40, 0.25% deoxycholate, 0.1% SDS, 0.5 mM TCEP, 10% glycerol, 1× SIGMAFAST protease inhibitor cocktail (MilliporeSigma) and 1× PhosSTOP phosphatase inhibitor cocktail (Roche). Samples were placed on a shaker for 30 min at 4 °C, centrifuged for 30 min at 20,000 × g at 4 °C and the supernatant was collected and measured by BCA (Thermo Fisher Scientific). To resolve LRP6 and full-length GRP94 glycoforms, samples were run on Novex Tris-Glycine 4 to 12% gels (Thermo Fisher Scientific). For immunoblot analysis of IGF1R, cells were collected by scraping, as IGF1R is sensitive to trypsinization. The resolved proteins were transferred onto nitrocellulose membrane (Bio-Rad Laboratories) using a wet electroblotting system (Bio-Rad Laboratories) followed by immunoblotting. For EndoH analysis of wild-type and knockout (ΔFKBP11, ΔCCDC134, ΔSTT3A, ΔSTT3B) HEK293 cells, microsomes were prepared from two 15-cm dishes as described above. The samples were treated with 2,500 U EndoH (New England Biolabs, P0702L), according to the manufacturer's protocol. Samples were run on home-made tris-glycine 5.5% gels at 4 °C and constant voltage (150 V) or on 4–20% Bio-Rad TGX precast gels, followed by immunoblotting. For analysis of GRP94 mutants in HEK293 cells, wild-type Flp-In T-REx 293 cells were seeded onto a six-well plate at a density of 200,000 cells per well one day before transfection. A transfection mixture was prepared containing 1 μg pcDNA5-SS-3xFLAG-GRP94, 150 μl Opti-MEM and 3 μl TransIT293, which was incubated at room temperature for 20 min before adding dropwise to each well. After 24 h, cells were collected using cold 1× PBS and centrifuged at 500g for 5 min at 4 °C. The supernatant was discarded and cells were resuspended in 100 μl lysis buffer (50 mM Tris pH 7.4, 150 mM NaCl, 1% NP-40, 0.5% sodium deoxycholate, 0.1% SDS, 0.5 mM PMSF). Samples were incubated on ice for 15 min (with vortexing every 5 min) and the supernatant was collected after centrifugation at 20,000g for 10 min at 4 °C. Samples were analysed by sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE) and immunoblotting. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequencing data are available at the NCBI Gene Expression Omnibus (GEO) repository with accession number GSE303507. Protein models and cryo-EM maps are available at the RCSB Protein Data Bank with accession codes 9YGY (translocon-only model) and 9N9J (60S subunit and translocon). The corresponding maps are available at the Electron Microscopy Data Bank with accession codes EMD-72945 (Map 1) and EMD-49171 (Map 2). All other data are available in the main text or the supplementary materials. & Aebi, M. Roles of N-linked glycans in the endoplasmic reticulum. Hebert, D. N. & Molinari, M. In and out of the ER: protein folding, quality control, degradation, and related human diseases. Kelleher, D. J., Karaoglu, D., Mandon, E. C. & Gilmore, R. Oligosaccharyltransferase isoforms that contain different catalytic STT3 subunits have distinct enzymatic properties. Regulated N-glycosylation controls chaperone function and receptor trafficking. Chang, I. J., He, M. & Lam, C. T. Congenital disorders of glycosylation. Vogt, G. et al. Gains of glycosylation comprise an unexpectedly large group of pathogenic mutations. Grasbon-Frodl, E. et al. Loss of glycosylation associated with the T183A mutation in human prion disease. Structural basis for coupling protein transport and N-glycosylation at the mammalian endoplasmic reticulum. Shibatani, T., David, L. L., McCormack, A. L., Frueh, K. & Skach, W. R. Proteomic analysis of mammalian oligosaccharyltransferase reveals multiple subcomplexes that contain Sec61, TRAP, and two potential new subunits. & Gilmore, R. Cotranslational and posttranslational N-glycosylation of polypeptides by distinct mammalian OST isoforms. & Wormald, M. R. Statistical analysis of the protein environment of N-glycosylation sites: implications for occupancy, structure, and folding. Zacchi, L. F. & Schulz, B. L. N-glycoprotein macroheterogeneity: biological implications and proteomic characterization. Zhang, Q., Ma, C., Chin, L. S. & Li, L. Integrative glycoproteomics reveals protein N-glycosylation aberrations and glycoproteomic network alterations in Alzheimer's disease. Controlling N-linked glycan site occupancy. Sato, T. et al. STT3B-dependent posttranslational N-glycosylation as a surveillance system for secretory protein. Moriconi, C. et al. Interactions between N-linked glycosylation and polymerisation of neuroserpin within the endoplasmic reticulum. Dersh, D., Jones, S. M., Eletto, D., Christianson, J. C. & Argon, Y. OS-9 facilitates turnover of nonnative GRP94 marked by hyperglycosylation. Cherepanova, N. A., Venev, S. V., Leszyk, J. D., Shaffer, S. A. & Gilmore, R. Quantitative glycoproteomics reveals new classes of STT3A- and STT3B-dependent N-glycosylation sites. & Nicchitta, C. V. Purification and partial molecular characterization of GRP94, an ER resident chaperone. Positive selection CRISPR screens reveal a druggable pocket in an oligosaccharyltransferase required for inflammatory signaling to NF-κB. Bernaleau, L. et al. CCDC134 controls TLR biogenesis through the ER chaperone Gp96. Dubail, J. et al. Homozygous loss-of-function mutations in CCDC134 are responsible for a severe form of osteogenesis imperfecta. & Charoenngam, N. Fetal fractures in an infant with maternal Ehlers-Danlos syndrome, CCDC134 pathogenic mutation and a negative genetic test for osteogenesis imperfecta. The recurrent homozygous translation start site variant in CCDC134 in an individual with severe osteogenesis imperfecta of non-Morrocan ancestry. The prolyl isomerase FKBP11 is a secretory translocon accessory factor. Li, Q., Stroup, E. K. & Ji, Z. Rfoot-seq: transcriptomic RNase footprinting for mapping stable RNA-protein complexes and rapid ribosome profiling. Li, Q., Yang, H., Stroup, E. K., Wang, H. & Ji, Z. Low-input RNase footprinting for simultaneous quantification of cytosolic and mitochondrial translation. Gemmer, M. et al. Visualization of translation and protein biogenesis at the ER membrane. & Hegde, R. S. Structural analysis of the dynamic ribosome-translocon complex. Ramirez, A. S. et al. Molecular basis for glycan recognition and reaction priming of eukaryotic oligosaccharyltransferase. & Locher, K. P. Cryo-electron microscopy structures of human oligosaccharyltransferase complexes OST-A and OST-B. Studies on the function of oligosaccharyl transferase subunits: Stt3p is directly involved in the glycosylation process. Lizak, C., Gerber, S., Numao, S., Aebi, M. & Locher, K. P. X-ray structure of a bacterial oligosaccharyltransferase. Whitley, P., Nilsson, I. M. & von Heijne, G. A nascent secretory protein may traverse the ribosome/endoplasmic reticulum translocase complex as an extended chain. Nilsson, I. et al. Photocross-linking of nascent chains to the STT3 subunit of the oligosaccharyltransferase complex. Shrimal, S. & Gilmore, R. Glycosylation of closely spaced acceptor sites in human glycoproteins. Zielinska, D. F., Gnad, F., Wisniewski, J. R. & Mann, M. Precision mapping of an in vivo N-glycoproteome reveals rigid topological and sequence constraints. Shrimal, S., Ng, B. G., Losfeld, M. E., Gilmore, R. & Freeze, H. H. Mutations in STT3A and STT3B cause two congenital disorders of glycosylation. Ghosh, A. et al. Phenotypic heterogeneity in a congenital disorder of glycosylation caused by mutations in STT3A. Factor VIII and vWF deficiency in STT3A-CDG. Dollins, D. E., Warren, J. J., Immormino, R. M. & Gewirth, D. T. Structures of GRP94-nucleotide complexes reveal mechanistic differences between the hsp90 chaperones. Huck, J. D., Que, N. L., Hong, F., Li, Z. & Gewirth, D. T. Structural and functional analysis of GRP94 in the closed state reveals an essential role for the pre-N domain and a potential client-binding site. Jaskolowski, M. et al. Molecular basis of the TRAP complex function in ER protein biogenesis. Gemmer, M., Chaillet, M. L. & Förster, F. Exploring the molecular composition of the multipass translocon in its native membrane environment. Roboti, P. & High, S. The oligosaccharyltransferase subunits OST48, DAD1 and KCP2 function as ubiquitous and selective modulators of mammalian N-glycosylation. Roboti, P. & High, S. Keratinocyte-associated protein 2 is a bona fide subunit of the mammalian oligosaccharyltransferase. Contact order revisited: influence of protein size on the folding rate. Sato, S., Ward, C. L. & Kopito, R. R. Cotranslational ubiquitination of cystic fibrosis transmembrane conductance regulator in vitro. & Huibregtse, J. M. A cotranslational ubiquitination pathway for quality control of misfolded proteins. Principles of cotranslational ubiquitination and quality control at the ribosome. Rajasekaran, N. & Kaiser, C. M. Co-translational folding of multi-domain proteins. Jahn, M. et al. Folding and domain interactions of three orthologs of Hsp90 studied by single-molecule force spectroscopy. Shaffer, A. L. et al. XBP1, downstream of Blimp-1, expands the secretory apparatus and other organelles, and increases protein synthesis in plasma cell differentiation. Wang, X. et al. Spliced X-box binding protein 1 stimulates adaptive growth through activation of mTOR. Herrema, H. et al. FKBP11 rewires UPR signaling to promote glucose homeostasis in type 2 diabetes and obesity. Weighted gene co-expression network analysis identifies FKBP11 as a key regulator in acute aortic dissection through a NF-kB dependent pathway. Sundaram, A. et al. Substrate-driven assembly of a translocon for multipass membrane proteins. Smalinskaite, L., Kim, M. K., Lewis, A. J. O., Keenan, R. J. & Hegde, R. S. Mechanism of an intramembrane chaperone for multipass membrane proteins. Global analysis of translocon remodeling during protein synthesis at the ER. Campeau, E. et al. A versatile viral system for expression and depletion of proteins in mammalian cells. Brinkman, E. K., Chen, T., Amendola, M. & van Steensel, B. Easy quantitative assessment of genome editing by sequence trace decomposition. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Kim, D. et al. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Ji, Z. RibORF: identifying genome-wide translated open reading frames using ribosome profiling. & Struhl, K. Many lncRNAs, 5′UTRs, and pseudogenes are translated and some are likely to express functional proteins. Yang, H., Li, Q., Stroup, E. K., Wang, S. & Ji, Z. Widespread stable noncanonical peptides identified by integrated analyses of ribosome profiling and ORF features. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. Scheres, S. H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. Mirdita, M. et al. ColabFold: making protein folding accessible to all. Protein complex prediction with AlphaFold-Multimer. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Zhao, J. Fuller and the University of Chicago Advanced Electron Microscopy Core Facility (RRID: SCR_019198) for discussions and support. Computational resources were provided by the University of Chicago Research Computing Center (RCC). This work was supported by: National Institutes of Health grants R35GM145374 (to R.J.K. ), and National Science Foundation Graduate Research Fellowships DGE-1746045 (to M.Y.) These authors contributed equally: Melvin Yamsek, Mengxiao Ma, Roshan Jha Department of Biochemistry and Molecular Biology, The University of Chicago, Chicago, IL, USA Melvin Yamsek, Roshan Jha & Robert J. Keenan Mengxiao Ma, Katherine DeLong & Rajat Rohatgi Mengxiao Ma, Katherine DeLong & Rajat Rohatgi Department of Biomedical Engineering, McCormick School of Engineering, Northwestern University, Evanston, IL, USA Department of Molecular Genetics and Cell Biology, The University of Chicago, Chicago, IL, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Rajat Rohatgi or Robert J. Keenan. The authors declare no competing interests. Nature thanks Friedrich Förster, Linda Hendershot and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Overview of the sample preparation strategy for analysis by western blotting, single-particle cryo-EM and selective ribosome profiling. b, Analysis of input (I), flow-through (FT), ribosome-bound (pellet, P, loaded at 200× or 400× input) and ribosome-free (supernatant, S, loaded at 3× input) fractions by SDS-PAGE and immunoblotting. Data are representative of at least two independent experiments. For gel source data, see Supplementary Fig. d, Reads were grouped by read lengths (18–35 nt) and the fraction of reads assigned to the three different frames is shown. Strong 3-nt periodicity of read distribution was observed for most footprint sizes. e, Ribosomal A-site adjusted read distribution around the start and stop codons of mRNAs. Initial rounds of 3D classification yielded four classes containing characteristic density for the secretory translocon (TRAP, SEC61 and OST-A complexes). These were subjected to further 3D refinement and focused classification with signal subtraction using a mask around the translocon, micelle and part of the ribosomal large subunit (cyan). This yielded three main secretory translocon classes, which were refined and subjected to further focused classification with signal subtraction using a mask around GRP94, CCDC134, FKBP11 and STT3A (green). The class with density for GRP94, CCDC134 and FKBP11 was refined to generate final maps of the translocon (Map 1) and the RTC (Map 2). a, Overview of the translocon-only map (Map 1) and whole-particle map after focused refinement on the 60S subunit (Map 2), coloured and low-pass-filtered by local resolution as estimated in RELION 5.0. Maps are coloured on the same scale, from 3.5 to 9.5 Å. b, Map 1 shown at a lower contour and coloured by local resolution as in panel a. c, Fourier shell correlation (FSC) curves (masked). The height and colour (blue to red) of the bars is proportional to the number of particles in those views. All predictions were obtained using the ColabFold2 implementation of AlphaFold2 multimer v3 (ref. 77). Predicted alignment error matrices are also shown. a, CCDC134 (blue) and FKBP11 (purple) fit to the cryo-EM translocon map low-pass-filtered to 8-Å resolution. In this and subsequent panels, the corresponding AlphaFold2 (AF2) predictions (grey) are superimposed for comparison with the final models. b, Overview (left) and close-up (right) views of the OST-A-bound pre-N segment (magenta) fit to the translocon map after low-pass-filtering by local resolution. The AF2 pre-N model (grey) was superimposed by means of the STT3A subunit (not shown for clarity). c, Overview of CCDC134 and the GRP94 bridge helix (magenta) fit to the translocon map low-pass-filtered to 6-Å resolution. d, Orthogonal views of KCP2 (green) and the DC2 N-terminal extension (salmon) fit to the same map. a, Slice through the whole-particle map low-pass-filtered by local resolution, highlighting the hybrid state A/P (green) and P/E (orange) tRNAs on the ribosome (grey). b, Close-up of the translocon map low-pass-filtered by local resolution, highlighting the LLO ligand bound in the STT3A (green) active site. A portion of the GRP94 SRT motif (including S42) is also shown (magenta). c, Close-up of an N-linked glycan at RPN1 N299, fit into the translocon map low-pass-filtered by local resolution. d, Slice through the translocon map low-pass-filtered by local resolution, highlighting nascent GRP94 (magenta) density in the ribosome (grey) exit tunnel and SEC61 (yellow) channel. The translocated pre-N and N-domains of nascent GRP94 are also visible in the OST-A (green) active-site cavity and packed against CCDC134 (blue). e, Close-up of the SEC61 plug helix fit to the translocon map after post-processing using DeepEMhancer. f, View of FKBP11 (purple) fit into the translocon map low-pass-filtered to 6 Å (left) and a close-up of its C-terminal helix fit to the whole-particle map low-pass-filtered to 6 Å (right). The ribosome is shown in grey. g, Density for FKBP11 and its C-terminal helix is not visible in other RTC maps, including EMD-4316, shown here after low-pass-filtering to 6 Å. h, Orthogonal views of the globular GRP94 N-domain (magenta) fit to the translocon map low-pass-filtered to 6 Å. i, Close-up views of the STT3A TMDs fit to the translocon map after post-processing using DeepEMhancer. j, As in panel i, but for the DC2 TMDs. a, Levels of monoglycosylated (mg) and hyperglycosylated (hg) GRP94 following transient transfection of the indicated variant into HEK293 cells. b, ER and plasma membrane (PM) levels of IGFR1β and LRP6 and levels of GRP94-5N (mg) in GRP94 knockout RKO cells stably expressing the indicated GRP94-5N variant. c, EndoH analysis of endogenous GRP94 hyperglycosylation (hg) in the indicated HEK293 cell lines. d, As in c but for the indicated RKO cell lines. The last four lanes show GRP94 knockout RKO cells stably expressing a GRP94 T44A mutant, +/− treatment with the proteasome inhibitor, bortezomib, for 18 h at 1 µM. e, Analysis of endogenous GRP94 hyperglycosylation (hg) levels in RKO cell lines expressing sgRNAs targeting the indicated genes (NTC, non-targeting control). f, Input (I), ribosome pellet (P), flow-through (FT) and elution (E) fractions from purification of HA-tagged CCDC134 or the indicated mutants stably expressed in CCDC134 knockout RKO cells. g, Levels of monoglycosylated (mg) and hyperglycosylated (hg) GRP94 following transient transfection of the indicated variant into HEK293 cells. The fastest migrating band present in all lanes corresponds to endogenous, monoglycosylated GRP94. Data shown in panels a–g are representative of at least two independent experiments. For gel source data, see Supplementary Fig. a, Structural comparison of folded GRP94 monomers from the indicated experimentally determined structures, aligned on the nascent GRP94 N-domain (magenta) to highlight N–M domain interface flexibility in the closed- and open-dimer conformations. b, Overlay of a folded GRP94 monomer (PDB ID 5uls) in a closed-dimer conformation, aligned on the N-domain of nascent GRP94. Severe clashes are observed between the M-domain (dark grey) and STT3A (green), DC2 (salmon), CCDC134 (blue) and portions of the GRP94 pre-N domain (magenta), including the bridge helix. c, Overlay of a folded GRP94 monomer (PDB ID 2o1v) in an open-dimer conformation, aligned on the N-domain of nascent GRP94. Severe clashes are observed between M-domain (rosy brown) and STT3A, CCDC134 and RPN1 (not visible in this view). Further clashes are observed between the C-domain and RPN1. FKBP11 was removed for clarity in panels b and c. d, The charged linker (CL) (red) from an AlphaFold2 model of GRP94 was superimposed onto the structure of nascent GRP94 to highlight its location at the translocon. This antiparallel two-stranded β-sheet projects away from the N-domain, where it may help shield the M- and C-domains as they begin to fold. Profiling source data for Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Yamsek, M., Ma, M., Jha, R. et al. Structural basis of regulated N-glycosylation at the secretory translocon. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/11/251118220058.htm'>Nanoscale trick makes “dark excitons” glow 300,000 times stronger</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 17:00:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A research group at the City University of New York and the University of Texas at Austin has found a method to make dark excitons, a class of previously unseen light states, emit bright light and be controlled with nanoscale precision. The study, published November 12 in Nature Photonics, points toward future technologies that could operate faster, use less energy, and shrink to even smaller sizes. Dark excitons form in ultra-thin semiconductor materials and normally remain undetectable because they release only faint light. Even so, scientists have long viewed them as promising for quantum information and advanced photonics because they interact with light in unusual ways, remain stable for relatively long periods, and experience less disruption from their surroundings, which helps reduce decoherence. This structure increased the brightness of dark excitons by an extraordinary factor of 300,000, making them clearly observable and allowing their behavior to be precisely controlled. "This work shows that we can access and manipulate light-matter states that were previously out of reach," said principal investigator Andrea Alù, Distinguished and Einstein Professor of Physics at the CUNY Graduate Center and founding director of the Photonics Initiative at the Advanced Science Research Center at the CUNY Graduate Center (CUNY ASRC). This level of control could support new designs for on-chip photonics, highly sensitive detectors, and secure quantum communication. "Our study reveals a new family of spin-forbidden dark excitons that had never been observed before," said first author Jiamin Quan. "This discovery is just the beginning -- it opens a path to explore many other hidden quantum states in 2D materials." The findings also address a long-standing question of whether plasmonic structures can boost dark excitons without altering their fundamental nature when placed in close proximity. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09732-2'>Prime editing-installed suppressor tRNAs for disease-agnostic genome editing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 16:09:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Precise genome-editing technologies such as base editing1,2 and prime editing3 can correct most pathogenic gene variants, but their widespread clinical application is impeded by the need to develop new therapeutic agents for each mutation. For diseases that are caused by premature stop codons, suppressor tRNAs (sup-tRNAs) offer a more general strategy. Existing approaches to use sup-tRNAs therapeutically, however, require lifelong administration4,5 or show modest potency, necessitating potentially toxic overexpression. Here we present prime editing-mediated readthrough of premature termination codons (PERT), a strategy to rescue nonsense mutations in a disease-agnostic manner by using prime editing to permanently convert a dispensable endogenous tRNA into an optimized sup-tRNA. We optimized prime editing agents to install an engineered sup-tRNA at a single genomic locus without overexpression and observed efficient readthrough of premature termination codons and protein rescue in human cell models of Batten disease, Tay–Sachs disease and cystic fibrosis. In vivo delivery of a single prime editor that converts an endogenous mouse tRNA into a sup-tRNA extensively rescued disease pathology in a model of Hurler syndrome. PERT did not induce detected readthrough of natural stop codons or cause significant transcriptomic or proteomic changes. Our findings suggest the potential of disease-agnostic therapeutic genome-editing approaches that require only a single composition of matter to treat diverse genetic diseases. Therapeutic genome-editing efforts, including more than 70 clinical trials so far, have predominantly used programmable nucleases, base editors or prime editors to disrupt or correct disease-associated genes in an allele-specific manner. These approaches have proven to be effective in patients or in animal models for the treatment of disorders such as sickle-cell disease6,7, T cell leukaemia8, hypercholesterolaemia9,10, alpha-1-antitrypsin deficiency10, chronic granulomatous disease11, progeria12, spinal muscular atrophy13, prion disease14, alternating hemiplaegia of childhood15 and many other genetic diseases. Although allele-specific therapeutic genome-editing strategies offer treatments for many serious diseases with few treatment options, the breadth of the global genetic disease crisis, in which more than 8,000 genetic diseases collectively affect hundreds of millions of patients, demands new approaches that more rapidly bring the benefits of therapeutic genome editing to large numbers of patients16. Until substantial streamlining of regulatory, development and manufacturing costs occurs, the development of the thousands of distinct drugs needed to treat any large fraction of patients with genetic disease remains impractical, despite the fact that base editing and prime editing can collectively correct the vast majority of known pathogenic mutations1,2,3. The versatility of prime editing, which uses a programmable nickase and a reverse transcriptase to replace targeted segments of DNA with new sequences of our choosing3, can be used in creative ways that may benefit far larger cross-sections of patients with genetic disease. Nonsense mutations account for 24% of pathogenic alleles in the ClinVar database16. The common molecular consequence of nonsense mutations—termination of translation before functional protein can be made—suggests more generalized approaches to their therapeutic rescue that might be applied in an allele-agnostic or even disease-agnostic manner. Nonsense sup-tRNAs have an anticodon that complements a premature termination codon (PTC) and support the installation of an amino acid rather than termination of translation. Although sup-tRNAs can theoretically read through both PTCs and natural termination codons (NTCs), extensive evidence suggests that sup-tRNAs can be well-tolerated in eukaryotes. Both naturally occurring and exogenous sup-tRNAs can be expressed without apparent toxicity4,5. Multiple biological mechanisms explain the observation of surprisingly low levels of sup-tRNA-mediated readthrough at NTCs. Second, the frequent presence of redundant and diverse in-frame stop codons following NTCs reduces the likelihood of extending a protein past a suppressed stop codon by more than a few amino acids18. Third, recruitment of polypeptide chain release factors to the 3′ untranslated region (3′ UTR) near NTCs can outcompete sup-tRNAs19. Finally, sup-tRNAs mediate nonsense suppression only in transcripts that are being expressed in a given cell, minimizing the risk of toxicity from ectopic overexpression. Despite the promise of sup-tRNAs as a potential therapeutic strategy, current approaches such as lipid nanoparticle (LNP)-based or adeno-associated virus (AAV)-mediated delivery of sup-tRNAs4,5 face the challenge of supporting sup-tRNA production throughout the lifetime of the patient. AAV-based methods, in particular, may be limited to a single dose, owing to the generation of high levels of neutralizing antibodies following dosing, which may not be sufficient to treat genetic diseases that require restoration of protein expression for the lifetime of the patient. As a result, recent efforts to rescue nonsense mutations have relied on the overexpression or delivery of high levels of sup-tRNAs with suboptimal suppression efficiency4,5, even though tRNA overexpression can alter global translation22,23,24 and raise the possibility of toxicity. A one-time treatment that would enable permanent expression of endogenous levels of a sup-tRNA optimized for suppression efficiency without apparent toxicity has the potential to transform the treatment of diseases caused by PTCs. Because such a treatment may ameliorate a variety of diseases arising from the targeted type of stop codon, it also has the potential to benefit a large number of patients with multiple disorders caused by many pathogenic PTCs. Here we evaluated tens of thousands of variants of all 418 high-confidence human tRNAs to engineer highly active sup-tRNAs with sufficient potency to mediate efficient nonsense mutation suppression even when expressed from a single genomic copy with endogenous regulatory elements. We then used the versatility of prime editing to permanently convert a redundant endogenous human tRNA to these optimized sup-tRNAs. We call this strategy PERT (prime editing-mediated readthrough of premature termination codons). In human cell models of nonsense mutations in genes known to cause Batten disease (TPP1 p.L211X and TPP1 p.L527X), Tay–Sachs disease (HEXA p.L273X and HEXA p.L274X) and Niemann–Pick disease type C1 (NPC1 p.Q421X and NPC1 p.Y423X), treatment with the same prime editor programmed to install an optimized sup-tRNA resulted in restoration of 20–70% of normal enzyme activity. We experimentally quantified the effectiveness of our prime editing-installed sup-tRNA for reading through all clinically relevant TAG PTCs in the ClinVar database, observing PTC readthrough for the vast majority of sequences tested. Treatment with a single composition of matter mediated around 25% production of full-length GFP in reporter mice, and therapeutic levels of approximately 6% IDUA enzyme activity restoration, resulting in nearly complete rescue of disease pathology in Hurler syndrome mice. These findings provide a foundation for the development of disease-agnostic therapeutic genome-editing strategies that have the potential to substantially increase the number of patients that can benefit from a single genome-editing drug. The human genome encodes 47 isodecoder (anticodon) tRNA families comprising 418 high-confidence genes (Supplementary Table 1). This redundancy, coupled with transcription-induced mutagenesis, yields many tolerated tRNA mutations in humans, including anticodon mutations that create natural suppressor or mis-charged tRNAs25,26. Fourteen isodecoder families have no corresponding tRNA gene and are decoded by pseudocognate tRNAs27, and entire tRNA families have been deleted without consequence28. Therefore, we hypothesized that converting a single endogenous tRNA to a sup-tRNA with prime editing could be well-tolerated and enable readthrough of nonsense mutations regardless of the gene in which they originate. To screen endogenous human tRNAs for their ability to become sup-tRNAs, we designed mCherry-STOP-GFP reporters in which GFP expression occurs only after PTC readthrough (Supplementary Fig. Next, we designed prime editing reagents to convert the anticodon loops of two endogenous tRNAs (tRNA-Gln-CTG-6-1 and tRNA-Arg-CCG-2-1) into sup-tRNAs. For clarity, we follow standard tRNA nomenclature throughout this work. We installed these sup-tRNAs at the endogenous loci of tRNA-Gln-CTG-6-1 and tRNA-Arg-CCG-2-1 using prime editing in HEK293T cells. We observed on average 29% conversion of each endogenous tRNA into a sup-tRNA (range: 19%–37%) (Extended Data Fig. We then overexpressed the reporter construct via plasmid transfection, or expressed it from a single gene copy using lentiviral transduction. We characterized PTC readthrough using two metrics: the percentage of GFP-positive cells (% GFP) and the GFP mean fluorescence intensity relative to a wild-type GFP control (relative protein yield) (Extended Data Fig. Although both sup-tRNAs led to GFP rescue with an overexpressed reporter, neither led to significant PTC readthrough with a single-copy reporter (Extended Data Fig. Even with an overexpressed reporter, the per cent rescue of GFP relative to a wild-type control was modest, with on average 7.8% GFP-positive cells and a relative GFP protein yield of 10%, and overexpression of the reporter alone without any sup-tRNA yielded 3.9% GFP-positive cells, presumably from pseudocognate tRNA-mediated PTC readthrough. Previous sup-tRNA studies have heavily relied on high tRNA levels in transfection, LNP or viral settings, often using elevated copy numbers of the sup-tRNA and/or the PTC-containing mRNA4,5. These findings revealed the need to identify sup-tRNAs that are sufficiently potent to mediate endogenous genomic PTC readthrough at low sup-tRNA levels that are compatible with genomic installation and minimize the risk of perturbing global translation. To comprehensively identify sup-tRNAs that promote stop codon readthrough with low copy number expression, we designed three prime editing screens to convert the anticodon of each endogenous human tRNA (n = 418 genes) to CUA, UCA, or UUA anticodons, thereby generating sup-tRNAs that recognize TAG, TGA or TAA stop codons, respectively (Fig. For each tRNA gene, we identified the two nearest 20-bp spacers flanking the anticodon with an NGG protospacer-adjacent motif and varied primer-binding site (PBS) lengths (11–14 bp) and reverse transcription template (RTT) homology lengths (6–10 bp), where feasible. To enhance editing efficiency, we appended the structured RNA motif tevopreQ1 to each prime editing guide RNA (pegRNA), creating engineered pegRNAs (epegRNAs)29. In total, we designed 17,579 epegRNAs for each of the three sup-tRNA anticodons plus 420 negative-control epegRNAs (Fig. a, Schematic of prime editing strategy to generate sup-tRNAs. b, Lentiviral epegRNA design and screening strategy. gDNA, genomic DNA; NGS, next-generation sequencing. c, Fold enrichment of epegRNAs, designed as controls (red) or to generate sup-tRNA (blue), in GFP-sorted cells, compared to the plasmid pool in the TAG screen. d, Fold enrichment of epegRNAs, colour-coded by the amino acids of the isoacceptor tRNA family that they target. We cloned the epegRNA libraries into a lentiviral backbone and transduced them into a corresponding HEK293T cell line containing the mCherry-STOP-GFP reporter. Fluorescence-activated cell sorting (FACS) isolation of GFP-positive cells enriched cells expressing epegRNAs that generate functional sup-tRNAs (Fig. We identified arginine, leucine, tyrosine and serine tRNA backbones capable of reading through TAG stop codons (Fig. 2a–f) and leucine and arginine tRNA backbones that read through TGA stop codons (Extended Data Fig. A similar screen for TAA stop codons did not yield effective sup-tRNAs (Supplementary Fig. 6), consistent with previous observations that TAA is the most stringent stop codon and is more resistant to sup-tRNA-mediated readthrough30. Whereas epegRNAs that simultaneously targeted multiple tRNA sequences generally performed best, half of the most enriched positive hits in the TAG screen were epegRNAs targeting a single tRNA (Extended Data Fig. Although the epegRNA screen successfully identified endogenous tRNAs that could be converted into effective sup-tRNAs, even the best-performing sup-tRNA from this screen achieved less than 5% protein yield relative to a wild-type GFP control (Extended Data Fig. To address this limitation, we next engineered all major features of each sup-tRNA to enhance readthrough efficiencies. Whereas tRNA expression is typically driven by an internal Pol III promoter, the leader sequence is thought to regulate gene expression31. Thus far, most sup-tRNA studies have relied on exogenous promoters for expression5, although the role of internal tRNA promoters is being investigated32. We performed three screens using a pool of all 418 mature tRNA sequences, modifying their anticodons to CUA, UCA or UUA to target TAG, TGA or TAA stop codons, respectively. The pooled sup-tRNA libraries were cloned into lentiviral constructs with each promoter variant and transduced into the mCherry-STOP-GFP reporter lines (Supplementary Fig. Unlike the prime editing screen, lentiviral expression of sup-tRNAs with exogenous promoters yielded fewer TAG sup-tRNAs and activity depended on promoter choice (Extended Data Fig. Leu-TAA sup-tRNAs were strongest with hU6 (tRNA-Leu-TAA-4-1, tRNA-Leu-TAA-3-1, tRNA-Leu-TAA-1-1 and tRNA-Leu-TAA-2-1, in descending order of activity), whereas Tyr-GTA (tRNA-Tyr-GTA-7-1) performed best with minU6 or no promoter, suggesting that this tRNA may have stronger internal promoter elements (Extended Data Fig. No effective TGA sup-tRNAs were identified when expressing the sup-tRNAs from a lentiviral construct (Supplementary Fig. Since exogenous promoters seem to hinder the proper expression of many sup-tRNAs identified in the prime editing screen, we next sought to investigate how endogenous leader sequences and terminator sequences affect sup-tRNA function. To further evaluate the effect of leader sequences, we analysed the 40 bp upstream of all 418 high-confidence human tRNAs. These sequences are less conserved than the tRNA gene bodies and may influence their expression31,32. We cloned each leader sequence upstream of a sup-tRNA candidate (tRNA-Leu-TAA-3-1) (Supplementary Fig. 2 and Supplementary Table 4), transduced this pool of constructs into the mCherry-STOP-GFP reporter cell line, and isolated GFP-positive cells via FACS. Eighty-six per cent of the cells expressing a sup-tRNA in this screen were GFP-positive, indicating that most leader sequences support sup-tRNA expression (Extended Data Fig. We individually tested leader sequences with the highest and lowest enrichment among GFP-positive cells, as well as eight randomly generated control sequences. The top hits performed well (averaging 72% GFP-positive cells) and the bottom hits performed poorly (averaging 6.1% GFP-positive cells), but the random leader controls also performed reasonably well (averaging 57% GFP-positive cells) (Extended Data Fig. Thus, whereas many diverse leader sequences can support robust sup-tRNA activity, others can nearly abolish activity. No apparent leader consensus sequences explained the observed variation in sup-tRNA activity. Next, we investigated whether leader sequences could modulate PTC suppression levels across different sup-tRNAs. 4e); and two leaders upstream of highly expressed tRNAs: (5) tRNA-Cys-GCA-3-1; and (6) tRNA-Ser-GCT-3-1, which could in principle be overwritten with alternative sup-tRNA sequences using prime editing34. In parallel, we explored how downstream termination sequences might affect tRNA expression levels. We identified four potential downstream sequences to test: the endogenous 100-bp sequence downstream of each mature tRNA sequence, and terminators consisting of TTT (3T), TTTT (4T) or TTTTT (5T), which are predicted to terminate transcription from a Pol III promoter with efficiencies of approximately 0%, 75% or 99%, respectively35. We generated a lentiviral library containing all 6 leader sequences paired with 418 human tRNAs with their anticodons switched to CUA, UCA or UUA. We transduced each reporter cell line with the lentiviral library, isolated GFP-positive cells and sequenced the enriched tRNA variants. Unlike the lentiviral screening with exogenous promoters, this screen successfully identified at least one variant of all TAG sup-tRNAs that had been identified in the prime editing screen. The most effective sup-tRNAs performed best when paired with their own endogenous leader sequence (Extended Data Fig. Thus leader sequences have a critical regulatory role in tRNA activity. The screen results also indicated that a terminator is essential for sup-tRNA activity, with a directly adjacent 5T terminator outperforming most endogenous terminator sequences (Extended Data Fig. We hypothesize that the absence of a terminator may lead to run-off of Pol III, or weakened expression due to a lack of the traditional Pol III recycling mechanism in which Pol III is rapidly reused for a new round of transcription after completing a transcript36. Notably, only 93% and 48% of natural downstream sequences have a 4T or 5T termination sequence, respectively, within 100 bp of their mature tRNA sequence37 (Supplementary Fig. We next hypothesized that introducing mutations within the tRNA itself could further enhance sup-tRNA activity. We also sought to identify silent mutations that could improve prime editing outcomes by evading cellular mismatch repair (MMR)38 and by preventing rebinding of the prime editor to the target tRNA locus following repair. We selected three candidate sup-tRNAs (tRNA-Arg-CCT-4-1, tRNA-Tyr-GTA-2-1 and tRNA-Leu-TAA-4-1) and a corresponding mouse sup-tRNA (tRNA-Leu-TAA-2-1) and generated lentiviral libraries that include every possible 1-bp deletion, single-nucleotide substitution variant (SNV), and simultaneous base pair substitution in each tRNA's paired bases (Supplementary Fig. We transduced the reporter cell line with these lentiviral libraries and compared their performance to that of the corresponding anticodon-only (ac-only) sup-tRNA sequences. Most mutations reduced sup-tRNA performance (Extended Data Fig. Deletions were poorly tolerated across sup-tRNAs (Extended Data Fig. By contrast, for each sup-tRNA, a subset of SNVs and paired-base substitutions improved sup-tRNA performance (Extended Data Fig. We also found that beneficial mutations in tRNA-Leu-TAA-4-1 could apply to the related family members tRNA-Leu-TAA-1-1, tRNA-Leu-TAA-2-1, and tRNA-Leu-TAA-3-1 (Extended Data Fig. Since the tRNA-Leu-TAA family exhibited the highest baseline suppression activity, we selected this sup-tRNA backbone for further engineering (Fig. We focused on the mutations that were capable of being introduced at the same time as anticodon conversion with prime editing. Since mutations in sup-tRNAs can have additive effects4,32, we generated 19 combinations of the 5 anticodon-proximal mutations and introduced them into the top-performing tRNA-Leu-TAA-1-1 sup-tRNA using either prime editing at the endogenous locus or by expressing the variant from a single-copy lentiviral construct (Fig. For clarity, we refer to tRNAs whose anticodon alone was replaced as ac-only sup-tRNAs. When additional mutations were introduced to enhance readthrough, we designate them engineered sup-tRNAs and specify the type of mutation introduced; for example, engineered sup-tRNA (tRNA-Leu-TAA-1-1+hp12ta>cg+hp13gc>cg). a, Results of saturation mutagenesis of tRNA-Leu-TAA-4-1, with mutations that are comparable or better than an ac-only sup-tRNA shown in purple. Bases that would easily be accessible with prime editing (PE-accessible bases) at the same time as changing the anticodon are circled with a dashed line. In this representation, each hairpin dinucleotide is represented by a number (hp1–hp25). b, Heat map showing the relative fold change in activity of hairpin mutations compared with the ac-only sup-tRNA. Hairpin positions correspond with hp1–hp25 in a. c, Sequences of the anticodon stem loops of five prime editing-accessible mutation variants and an ac-only control, from which 19 variant combinations were generated. d, Protein yield relative to wild type (WT) after readthrough with an endogenously converted tRNA-Leu-TAA-1-1 sup-tRNA, a single-copy lentiviral tRNA-Leu-TAA-1-1 sup-tRNA with the indicated mutations, or with alternative engineered sup-tRNAs (described in Porter et al.32 and Wang et al.5). Of note, we observed that sup-tRNA mutations can be additively combined, supporting the production of up to fivefold higher levels of full-length GFP protein than the ac-only sup-tRNA, yielding up to 35% of the amount of GFP as control cells expressing an uninterrupted GFP gene (Fig. Engineered sup-tRNAs developed in this study substantially outperform previously engineered sup-tRNAs, including a recently reported variant that yielded 16% GFP protein yield (Fig. 2d) but would require changing 83 bp of a native tRNA sequence for genomic installation32. To evaluate the cellular abundance of prime editing-installed sup-tRNAs compared with a wild-type endogenous tRNA, we performed targeted tRNA sequencing for tRNA-Leu-TAA-1-1 on a polyclonal edited population. We observed that the ac-only sup-tRNA and the engineered sup-tRNA are 5.6-fold and 2.5-fold less abundant, respectively, than the endogenous tRNA-Leu-TAA-1-1 that they replace (Extended Data Fig. Both sup-tRNAs were modified at positions 26 and 37, as expected39 (Extended Data Fig. These findings suggest that mutations in the anticodon stem can increase engineered sup-tRNA performance in part by increasing its cellular abundance. Encouraged by the greatly enhanced suppression efficiency achieved through iterative engineering of the mature tRNA sequence, we next optimized prime editing strategies to convert the mammalian endogenous tRNA-Leu-TAA-1-1 tRNA gene into the ac-only sup-tRNA or engineered sup-tRNA. We designed a lentiviral epegRNA library of 17,280 epegRNAs that exhaustively tested spacer variants, PBS lengths from 8 to 16 bp, RTT lengths from 21 to 36 bp, and combinations of each of the 19 mutation variants of interest alongside 720 control epegRNAs (Fig. We encoded a tRNA-Leu-TAA-1-1 synthetic target site directly adjacent to the lentiviral epegRNA to read out both the epegRNA and the outcome of editing in the same sequencing read, transfected cells transduced with this library with a panel of plasmids encoding prime editor proteins40 (PE6a, PE6b, PE6c, PE6d, PE6e, PE6f, PE6g, PEmax and PEmaxΔRNaseH) (Fig. Furthermore, we performed experiments in both MMR-deficient (HEK293T) cells and MMR-proficient (HeLa) cells and included conditions in which MMR was transiently inhibited by co-transfection of a dominant-negative MMR protein (MLH1dn)38. a, Pooled screening strategy and prime editing system variants. b,c, Per cent desired editing at a synthetic target site for each of the indicated prime editors in HeLa cells (b) and HEK293T cells (c). d, Per cent editing at a synthetic target site in the pooled screen plotted against the per cent editing at the endogenous locus in arrayed validation in HEK293T cells (left) and HeLa cells (right) for the top 3 and bottom 3 epegRNA architectures per mutation variant. Pearson correlation is indicated in the top right. e, Example of prime editing efficiencies at a synthetic target site across PBS length and RTT length combinations for installing the hp12ta>cg+hp13gc>cg engineered sup-tRNA. Across all conditions tested, PE6c yielded the highest mean editing efficiency (Fig. 7a) and the inclusion of MLH1dn resulted in greater mean editing efficiencies for most tested variants, suggesting that MMR evasion strategies can enhance editing efficiency for these edits (Fig. We observed a high correlation (R = 0.78) between the results from the HeLa and HEK293T datasets (Extended Data Fig. To validate that editing of the lentivirally integrated target site used in the screen reflects editing of the endogenous tRNA-Leu-TAA-1-1 locus, we performed arrayed validation on 122 high-performing and low-performing epegRNAs in HEK293T and HeLa cells. Editing efficiency at the endogenous locus and at the lentiviral target site in both cell types were correlated (Fig. 3d,e, R = 0.80 in HEK293T cells and R = 0.70 in HeLa cells), indicating that the lentiviral target-matched screen reflects prime editing outcomes at the endogenous locus. To finalize our editing strategy, we tested whether a PE3 or PE3b strategy—both of which use a nicking guide RNA (ngRNA) to nick the unedited strand of the heteroduplex editing intermediate in order to favour the use of the edited strand as a template for DNA repair—could further improve editing outcomes3. Top-performing PE3 strategies yielded mean editing efficiencies of ~60–80% in HEK293T cells, substantially outperforming earlier designs that yielded <8% editing and completing the optimization of the proof-of-concept editing strategy for PERT (Supplementary Fig. We assessed off-target editing from PERT using two independent methods. First, we identified all sites in the human genome with up to four mismatches from the protospacer of the best-performing engineered sup-tRNA (Extended Data Fig. We sequenced at depth each of these sites following prime editing to install an engineered sup-tRNA (tRNA-Leu-TAA-1-1+hp12ta>cg+hp13gc>cg) and no significant off-target editing was observed above that of an untreated control (Extended Data Fig. Second, as a genome-wide unbiased method for detecting potential off-target edits, we performed a screen in mammalian cells with a large pool of lentiviral integrated putative off-target sites. We computationally identified off-target sites in the genome with up to 6 mismatches from the optimized epegRNA's protospacer or with up to 4 mismatches and up to two bulges, identifying 17,206 candidates (Supplementary Table 11). This coverage is identical to the computational filtering step of circularization for in vitro reporting of cleavage effects by sequencing (CIRCLE-seq)41, a commonly used genome-wide off-target detection method for RNA-guided nucleases. Following lentiviral integration of the pool of off-target candidate sites and transfection to install an engineered sup-tRNA (tRNA-Leu-TAA-1-1+hp12ta>cg+hp13gc>cg) with prime editing, we sequenced the lentiviral cassette and analysed editing at each off-target site. This analysis nominated 16 candidate off-target sites with potential prime editing events occurring above the level of the untreated control sample in prime-edited cells (Extended Data Fig. To further investigate these potential off-target sites, we individually amplified the genomic region corresponding to each of the 16 nominated off-target sites in cells transfected with prime editing reagents to install an engineered sup-tRNA (tRNA-Leu-TAA-1-1+hp12ta>cg+hp13gc>cg). We observed no off-target prime editing or indel formation above background for all amplified genomic loci (Extended Data Fig. These data are consistent with previous reports of the high sequence specificity of prime editing, which probably arises from its mechanism requiring multiple independent DNA or RNA hybridization events in order to make a productive edit3,42,43. We evaluated the potential for several types of abnormalities in human cells following conversion of an endogenous tRNA into a sup-tRNA. To detect transcriptome perturbations, we performed RNA-seq after converting the endogenous tRNA-Leu-TAA-1-1 gene into an engineered sup-tRNA (Leu-TAA-1-1+hp12ta>cg+hp13gc>cg) with prime editing. We observed no differentially expressed genes (FDR < 0.05 and |log2(fold change)| > 1), suggesting no evident stress response from converting the tRNA-Leu-TAA-1-1 tRNA gene into a sup-tRNA gene in a human cell line (Extended Data Fig. Next, we evaluated the effect of repurposing the tRNA-Leu-TAA-1-1 tRNA gene on global tRNA homeostasis by performing quantitative PCR to measure the pooled expression of 28 different tRNA gene families following prime editing-mediated installation of an ac-only or engineered sup-tRNA (Leu-TAA-1-1+hp12ta>cg+hp13gc>cg). We observed no significant difference (adjusted P value > 0.05) in the expression of any of these 28 tRNA gene families in either treatment condition relative to wild-type cells (Extended Data Fig. 8f), suggesting that the tRNA-Leu-TAA-1-1 tRNA gene can be repurposed without significant perturbation to general tRNA homeostasis. To evaluate potential readthrough of NTCs, we performed mass spectrometry to detect peptides that are translated into the 3′ UTR for protein-coding genes terminating with a TAG stop codon. After confirming that Leu-TAA sup-tRNAs retain 100% fidelity of leucine charging (Extended Data Fig. 9a), we computationally predicted all peptides that could be generated by reading through the 4,036 natural TAG termination codons and extending translation to the second in-frame stop codon. We then examined whether these predicted peptides were present by performing label-free quantitative mass spectrometry on trypsin-digested total protein lysates from both control and prime-edited HEK293T cells expressing and mCherry-STOP-GFP reporter and either the ac-only sup-tRNA or the engineered sup-tRNA (Leu-TAA-1-1+hp12ta>cg+hp13gc>cg). At the global protein abundance level, the only protein that was significantly (adjusted P value < 0.05, more than 2-fold change) different in abundance between control samples and prime-edited samples was the reporter GFP, which was 8.1-fold and 7.3-fold more abundant in cells edited with the ac-only sup-tRNA and the engineered sup-tRNA, respectively (Extended Data Fig. Although we observed robust readthrough of the PTC and peptides generated for GFP, we detected no peptides from translation past the NTC for any TAG-terminated protein (Fig. a,b, Normalized abundance of peptides identified past a PTC (top) or past natural TAG termination codons (NTCs) (bottom) for cells prime edited to express an ac-only sup-tRNA (a) or an engineered sup-tRNA (b). Dashed lines correspond to predicted potential trypsin cleavage sites. c, Desired editing outcomes in six HEK293T cell models of disease for each specified edit. d, TPP1 enzyme activity in treated and untreated human cell models of Batten disease relative to wild-type controls. e, HEXA enzyme activity in treated and untreated human cell models of Tay–Sachs disease relative to wild-type controls. To test with enhanced sensitivity the ability of a prime editing-installed sup-tRNA to induce NTC readthrough, we identified the 69 most abundant proteins that use a TAG stop codon from the above dataset and performed target-ion mass spectrometry to detect each of 111 predicted peptides that would be generated from reading past the NTC for these highly expressed proteins. We observed a single peptide corresponding to potential readthrough of an NTC (encoding 18 amino acids after the end of the tyrosyl-tRNA synthetase protein), but the abundance of this peptide was not significantly different (adjusted P value > 0.05) between untreated cells and cells that were prime edited to express a sup-tRNA (Extended Data Fig. Thus both proteome-wide and target-ion protein mass spectroscopy indicate that prime editing-installed sup-tRNAs do not broadly perturb the proteome or induce detected NTC readthrough under the conditions tested, probably owing to cellular mechanisms that safeguard against NTC readthrough. To assess the potential of PERT to rescue the effects of genomic PTCs with disease relevance, we evaluated readthrough across diverse, endogenously encoded PTCs. We generated models in HEK293T cells of known human pathogenic PTCs corresponding to a TAG stop codon in TPP1, HEXA and NPC1 genes that cause Batten disease, Tay–Sachs disease and Niemann–Pick disease type C1, respectively. We chose six mutations that either originally encoded a leucine amino acid or are known to tolerate a leucine at the indicated position (TPP1 p.L211X, TPP1 p.L527X, HEXA p.L273X, HEXA p.L274X, NPC1 p.Q421X and NPC1 p.Y423X)45. As with many loss-of-function genetic diseases, previous studies suggest that even low levels of protein activity for these disorders can provide therapeutic benefits46,47,48. We transfected these six cell models with the same optimized PE3 editing components to convert the endogenous genomic tRNA-Leu-TAA-1-1 locus into an ac-only sup-tRNA or engineered sup-tRNAs (tRNA-Leu-TAA-1-1+hp13gc>ta+hp12ta>cg or tRNA-Leu-TAA-1-1+hp13gc>cg+hp12ta>cg). Engineered sup-tRNA installation led to substantial (17–70%) restoration of enzyme activity in the four TPP1 and HEXA pathogenic PTC models, and partial restoration of full-length protein expression in both models of PTCs in NPC1 (which encodes a cholesterol transporter rather than an enzyme) using the same prime editing reagents, establishing the disease-agnostic therapeutic potential of this approach (Fig. In each case, treatment resulted in full-length protein expression that meets or exceeds the documented therapeutic threshold for each disease target46,47,48. Together, these results establish that a single prime editing agent to convert an endogenous genomic tRNA gene into a sup-tRNA can rescue a substantial fraction of enzyme activity and full-length protein production lost to nonsense mutations. To evaluate the performance of PERT at a broad spectrum of nonsense mutations, we cloned 14,746 pathogenic PTCs in the ClinVar database flanked on either side by the 18 nucleotides present in the native mRNA sequence into a nonsense-mediated decay-sensitive reporter construct (Supplementary Fig. This phenomenon allows quantification of the ability of a sup-tRNA to read through a given PTC by comparing the abundance of mRNA transcripts for each PTC-containing sequence (Methods). We transduced this library of PTC-containing sequences into a clonal ac-only sup-tRNA cell line and observed an average readthrough score of 69 ± 30% across 14,746 pathogenic PTCs, indicating stabilization of the mRNA transcript and readthrough of the PTC for most sequences. We independently confirmed readthrough of full-length CFTR cDNAs containing 15 distinct pathogenic PTC mutations that cause cystic fibrosis using a prime editing-installed ac-only sup-tRNA. 11c,d and Supplementary Table 16), with a maximum observed protein yield of 14% relative to wild-type CFTR. Together, these results demonstrate that PERT can effectively rescue diverse pathogenic genomic PTCs and can be used to generate therapeutically relevant levels of full-length protein and active enzymes in human cells. Next, we sought to evaluate the feasibility of applying PERT to rescue full-length protein production in vivo. We then designed an AAV9 eGFP reporter cassette containing a premature stop codon that only yields fluorescence when read through by a sup-tRNA. We administered these three AAV9 vectors, two for prime editing and one for the eGFP reporter, via intracerebroventricular injection into wild-type C57/BL6 mice on postnatal day 0 (P0) and evaluated prime editing and readthrough efficiency (Fig. Whereas all sup-tRNA engineering was done with a TAG sup-tRNA, for this experiment we additionally tested the installation of an analogous TGA sup-tRNA with a TGA-containing eGFP reporter construct. HTS, high-throughput sequencing; ICV, intracerebroventricular; Npu, Nostoc punctiforme; vg, vector genomes; Cbh, chicken β-actin hybrid promoter; RT, reverse transcriptase domain; KASH, Klarsicht, ANC-1 and Syne homology domain; ITR, inverted terminal repeat. b, In vivo readthrough efficacy of an exogenously supplied TAG or TGA premature stop codon reporter construct. P values are calculated using a one-sided Welch's t-test. d, Left, desired editing efficiencies in treated homozygous IduaW392X mice. Right, IDUA enzyme activity in treated homozygous IduaW392X mice. P values are calculated using a one-sided Welch's t-test compared with non-prime edited mice (Extended Data Fig. f, Tissue pathology score based on GAG storage evaluated by microscopy. At 3 weeks after injection, we collected the brain tissue for analysis of editing and readthrough efficiency. We observed an average editing efficiency of 11% for the TAG sup-tRNA and 23% for the TGA sup-tRNA in the bulk brain cortex of treated mice (Extended Data Fig. To measure readthrough efficacy, we dissociated the cells from whole hemisphere halves of treated mice and measured GFP expression by flow cytometry. Therefore, although editing efficiency would benefit from further optimization in mice, cells that receive prime editing agents can achieve readthrough levels that could ameliorate many diseases. To assess the tolerability of PERT in vivo, we performed whole-proteome mass spectrometry on tissues that had been prime edited to express an ac-only sup-tRNA. Similar to our observations in human cell lines (Fig. 4a,b), we did not detect any peptides corresponding to readthrough of natural stop codons in tissues from mice treated with PERT (Supplementary Table 17). In addition, we monitored a separate cohort of mice over the course of 15 weeks and observed no significant difference in body weight in the PERT-treated mice (n = 7) relative to untreated controls (n = 9) (Extended Data Fig. Finally, we investigated the in vivo therapeutic application of PERT in a mouse model of the human lysosomal storage disease mucopolysaccharidosis type I (MPS-I). MPS-1 is caused by a lack of α-L-iduronidase, leading to the pathogenic accumulation of glycosaminoglycans (GAGs). This model harbours a TAG stop codon, Idua p.W392X, which is orthologous to the most common pathogenic human mutation, IDUA p.W402X, and has been used previously to evaluate PTC readthrough in vivo5,51. For this experiment we further engineered the mouse sup-tRNA using mutations identified via saturation mutagenesis screening of mouse tRNA-Leu-TAA-2-1, generating an engineered sup-tRNA (mouse tRNA-Leu-TAA-2-1+hp13gc>ta+mut38a>t) (Supplementary Fig. We packaged prime editing agents to install this sup-tRNA into the v3em dual-AAV9 architecture50 and treated homozygous and heterozygous P0 neonatal IduaW392X littermates via intracerebroventricular injection (Fig. At 7 weeks after treatment, we evaluated prime editing efficiencies across tissues and observed mean desired prime editing efficiencies of 6.3%, 6.1%, 6.8% and 0.05% in the cortex, heart, liver and kidney, respectively (Fig. 5d), consistent with the known tissue tropism of AAV9 (ref. Editing led to marked restoration of IDUA enzymatic activity compared with untreated controls, with mean enzymatic activity relative to wild-type mice of 7.6%, 6.3%, 1.7% and 0.31% in the heart, cortex, liver and kidney, respectively (Fig. These data demonstrate the ability of in vivo PERT to restore enzyme function that has been abrogated by pathogenic endogenous PTCs. To assess whether restoration in enzymatic activity was sufficient to rescue disease phenotypes, we performed histological analysis of brain, liver and spleen tissue from homozygous IduaW392X mice. We scored phenotypic severity on the basis of MPS-I related pathology, including hepatic and splenic foam cell accumulation, heart vessel-associated foam cell accumulation, cerebellar Purkinje cell vacuolization, splenic stromal vacuolization and pathogenic GAG accumulation. MPS-I GAG storage pathology was observed across all tissues analysed from untreated IduaW392X homozygous mice, but was minimal or absent in treated counterparts. Purkinje cells of the cerebellum and thalamic neurons displayed moderate vacuolization in untreated homozygous IduaW392X mice and was resolved following PERT treatment with absent or minimal vacuoles (Fig. We observed a similar pattern of attenuated vacuolization in the spleen, liver and heart in treated cohorts (Fig. Finally, the liver of untreated mice displayed mild Alcian blue staining that was minimal in the treated mice, indicating depletion of GAG accumulation (Fig. These data are consistent with previous reports that restoration of only 1% or more IDUA enzyme activity is sufficient to rescue most or all observed MPS-1 disease phenotypes5. Together, these results illustrate the in vivo applicability of PERT for PTC readthrough and rescue of disease phenotypes in a clinically relevant mouse model. Clinical genome-editing technologies, including nucleases, base editors and prime editors, have advanced the treatment of many genetic disorders. These therapeutic approaches have predominantly been disease-specific, with most efforts requiring the design of distinct gene-correction strategies for each pathogenic mutation. Given that more than 200,000 distinct mutations collectively affect hundreds of millions of individuals, developing, manufacturing and meeting regulatory requirements for thousands of individual therapies needed to treat even a modest fraction of patients with genetic disease is a monumental challenge, creating an urgent need for allele- and disease-agnostic therapeutic approaches that can address large classes of mutations with a single therapeutic composition. As one potential disease-agnostic strategy, we developed PERT as a prime editing-mediated approach to convert redundant endogenous tRNAs into optimized sup-tRNAs. This strategy combines the durable, one-time treatment potential of genome editing with the broad applicability of sup-tRNAs. Through high-throughput evaluation of thousands of variants spanning all 418 high-confidence tRNA genes, we identified sup-tRNAs that are capable of reading through 32% of all possible nonsense mutations with precise amino acid correction (installing the wild-type amino acid in response to the PTC), including Arg and Leu sup-tRNAs that can read through TGA stop codons and Leu, Ser and Tyr sup-tRNAs that can read through TAG stop codons. Whereas precise amino acid correction is the most conservative strategy, many nonsense mutations may still be rescued with non-wild-type amino acid substitutions, although characterization of the missense protein variant in these cases is necessary53. By swapping the anticodon of the endogenous tRNA-Leu-TAA-1-1 gene to CTA and introducing prime editing-accessible mutations in the anticodon stem, we engineered sup-tRNAs variants capable of restoring more than 35% of wild-type protein activity and/or expression from a gene containing a TAG nonsense mutation, surpassing the therapeutic threshold known for many loss-of-function genetic diseases46,47,48. In mouse models, including those for Hurler syndrome, PERT treatment demonstrated substantial protein restoration and near-complete rescue of disease pathology. During the course of this study, we did not observe any evident toxicity from PERT in any cultured cells or mice, changes in the transcriptome of cells, including the abundance of other tRNAs, readthrough of natural stop codons in human cells or in tissues from PERT-treated mice using proteome-wide mass spectrometry, or any off-target editing using two genome-wide detection methods. Since the stoichiometry of cellular tRNAs can influence global translation outcomes22,23,24, we hypothesize that conversion of a single endogenous genomic tRNA gene into a sup-tRNA may be less perturbative to global translation outcomes than previous approaches to introduce sup-tRNAs by exogenous overexpression or delivery4,5. Not all endogenous tRNA genes could be converted into effective sup-tRNAs with a simple anticodon change. In some cases, only specific isodecoders within a family were functional sup-tRNAs (for example, Tyr-GTA but not Tyr-ATA). Some sup-tRNAs favoured TAG stop codon readthrough (for example, Tyr-GTA), whereas others supported readthrough of only TGA stop codons (for example, Arg-TCG). We were unable to identify an effective prime editing-installed sup-tRNA for TAA stop codons with low-copy screening, although future efforts could explore reporter sensitivity adjustments or additional tRNA mutations in order to achieve this goal. Although certain patterns emerge in these data—such as the identification of Ser and Leu sup-tRNA backbones, which do not rely on the anticodon loop for recognition by their aminoacyl-tRNA synthetases54,55—it remains unclear why a functional Ala sup-tRNA, which also is thought to lack synthetase recognition of the anticodon, was not identified. Our data suggest that the determinants of sup-tRNA effectiveness, despite extensive study, remain incompletely understood. Across the sequence contexts of 14,746 pathogenic TAG stop codons in ClinVar, more than 70% showed readthrough from PERT. However, additional factors outside of the immediate sequence context are also likely to influence sup-tRNA activity, including ribosome velocity and PTC position in the mRNA. Further improvements may come from mutations beyond the anticodon stem, such as alterations in the D- or T-loops, which may enhance aminoacylation, transcription or ribosome accommodation of sup-tRNAs. Alternative methods for integrating a sup-tRNA sequence into the genome, such as twin prime editing (twinPE)34, eePASSIGE56, CASTs57,58 or methods based on homology-directed repair58, may facilitate the incorporation of more advanced combinations of mutations. Combining PERT with readthrough-promoting small molecules may further boost functional protein restoration. We also anticipate that, although our primary goal was to optimize a sup-tRNA specifically for one-time genomic installation, the best-performing variants we identified could improve readthrough efficiency when delivered through tRNA-supplementation approaches such as AAV- or LNP-mediated delivery of sup-tRNAs. Additional studies are needed to investigate and optimize the safety, efficacy and ability of the strategy developed in this study to be delivered to relevant tissues through viral and non-viral in vivo delivery methods. Future work could explore tissue-targeted capsid engineering, receptor-specific LNP-mediated or engineered virus-like particle-mediated59,60 delivery, and regulated or transient expression systems that avoid unnecessary long-term exposure to editing agents. Nevertheless, the findings of this initial study highlight the potential of PERT as a disease-agnostic therapeutic strategy in which a single composition of matter may offer a one-time treatment that benefits many different cohorts of patients with diverse genetic diseases. Although direct DNA-level correction of pathogenic mutations remains the most straightforward approach to precision genetic medicine, the large effort currently required to develop, meet regulatory requirements and manufacture many gene editing agents needed to correct many mutations underscores the importance of complementary allele-agnostic and disease-agnostic approaches such as PERT to maximize the ability of patients to benefit from advances in gene editing. By enabling a single treatment to address a broad cross-section of nonsense mutation-related diseases, PERT represents an advance toward making therapeutic genome editing accessible to a much larger patient population. We hope that PERT and the flexibility of prime editing will stimulate additional allele-agnostic and mutation-agnostic therapeutic gene editing strategies that may provide broadly applicable treatments for diverse serious disorders. DNA amplification was conducted by PCR using Phusion U Green Multiplex PCR Master Mix (ThermoFisher Scientific) or Q5 Hot Start High-Fidelity 2× Master Mix (New England BioLabs) unless otherwise noted. Plasmids expressing epegRNAs were constructed by Gibson assembly using a custom acceptor plasmid. Sequences of sgRNA and epegRNA constructs used in this work are listed in Supplementary Table 1. All vectors for mammalian cell experiments were purified using Plasmid Plus Midiprep kits (Qiagen) or PureYield plasmid miniprep kits (Promega), which include endotoxin removal steps. All experiments using live animals were approved by the Broad Institute Institutional and Animal Care and Use Committees. Wild-type C57BL/6 mice were obtained from Charles River (027). Cell lines with homozygous PTCs in TPP1, HEXA and NPC1were generated using prime editing. HEK293T (ATCC CRL-3216), Neuro-2a (ATCC CCL-131) and HeLa (CCL-2) cells were purchased from ATCC and cultured and passaged in Dulbecco's Modified Eagle's Medium (DMEM) plus GlutaMAX (ThermoFisher Scientific), supplemented with 10% (v/v) fetal bovine serum (Gibco, qualified). All cell types were incubated, maintained, and cultured at 37 °C with 5% CO2. Cell lines were authenticated by their respective suppliers and tested negative for mycoplasma. 16–24 h after seeding, cells were transfected at approximately 60% confluency with 600 ng, 200 ng and 60 ng of PEmax plasmid, epegRNA plasmid and ngRNA plasmid, respectively using Lipofectamine 3000 according to manufacturer's instructions (Thermo Fisher Scientific). 4 d after transfection, single cell clones were isolated by limiting dilution cloning and expanded over a 2-week period. The resulting colonies were further expanded and genotyped by high-throughput sequencing of the targeted locus and those found to be homozygous for the expected edit were retained for downstream experiments. HEK293Ts (ATCC CRL-3216) were transfected with pMD2.G (Addgene #12259) and delta8.2 (Addgene Plasmid #8455) packaging plasmids alongside the appropriate lentiviral backbone using Lipofectamine 2000. Medium was changed 24 h after transfection. Virus-containing supernatant was collected and filtered through a 0.45-μM filter 48 h after transfection. Virus was used immediately or stored at 4 °C for up to 1 week before cell transduction. All oligonucleotides for high-throughput screening were ordered from Twist Biosciences as single-stranded oligonucleotide pools. Oligonucleotide pools were amplified using Q5 Hot Start High-Fidelity 2X Master Mix (NEB M0494L) to create double-stranded inserts for isothermal assembly. Primers used are indicated in each specific screening section of the Methods. The minimum number of PCR cycles was used to amplify (typically between 11 and 13 cycles) and double-stranded inserts were checked for size and/or inappropriate products by TapeStation (Agilent). For each 20-μl reaction cloned into the pSEP0308, pSEP0309 or pSEP0310 lentiviral backbones with a 300-bp insert, 50 ng of the appropriate lentiviral backbone was assembled with 10 ng of the appropriate double-stranded insert using NEBuilder HiFi DNA Assembly Master Mix (NEB E2621L). For each 1,000 elements, an additional 20-μl reaction was set up. Reactions were incubated at 50 °C for 2 h and then pooled and purified using the QIAquick PCR purification kit (Qiagen 28104) according to manufacturer's instructions. Reaction products were eluted in a minimum of 20 μl ddH2O, but otherwise were eluted in 2.5 μl per original 20-μl reaction. Isothermal assembly reactions were electroporated into NEB 10-beta Electrocompetent Escherichia coli (NEB C3020) and plated onto LB plates aiming for at least 1,000× coverage of each library element. 14 h later, colonies were scraped and prepared using Qiagen Plasmid Plus kits (Qiagen 12945 and 12963) according to manufacturer's instructions. Medium was changed 24 h after transfection. Virus-containing supernatant was collected and filtered through a 0.45-μM filter 48 h after transfection. Cells were transduced to ensure ≥1,000× coverage of transduced cells per element of the library. 2 d after transduction, cells were passaged with 1 μg ml−1 puromycin for 3–4 days to enrich for transduced cells. For screens requiring FACS isolation of GFP-positive cells, we calculated approximate coverage by multiplying the per cent of GFP-positive cells by the number of elements in the library and aiming for 1,000× coverage of that number. For example, if 10% of cells were GFP-positive for a 500 element library, we sorted at least 50,000 cells per replicate. Cells were pelleted and flash frozen on dry ice. Genomic DNA was isolated using QIAamp DNA Micro kits (Qiagen 56304) or QIAamp Mini kits (Qiagen 51304) depending on the cell number isolated. All genomic DNA for each replicate was input into an initial set of PCR reactions using Q5 Hot Start High-Fidelity 2X Master Mix (NEB M0494L) to amplify the integrated lentiviral cassette and to add sequencing adapters, with a maximum of 5 μg genomic DNA per 50-μl reaction. Primers used are indicated in each screening section and are also listed in Supplementary Table 1. PCR reactions were purified using the QIAquick PCR purification kit (Qiagen 28104) according to manufacturer's instructions and 2 μl of PCR product was input into a second PCR reaction to add unique sample indices and flow cell adapters to each amplicon. Final PCR reactions were bead purified with a ratio of 0.7× using SPRI beads, quality controlled using a TapeStation (Agilent) and quantified using a Qubit prior to high-throughput sequencing on an Illumina MiSeq, Illumina NextSeq, or Element Biosciences AVITI instrument. Sequencing conditions and downstream analyses are specific to each screen and are indicated in each separate screening section. To convert each endogenous tRNA into a sup-tRNA, we identified 418 high-confidence tRNA genes and identified the two closest 20-bp spacers with an NGG protospacer-adjacent motif upstream and downstream of the anticodon. We allowed prime editing to target multiple tRNAs in the case of identical mature tRNA sequences. We designed three prime editing screens, each with RTTs that replaced the anticodon of each endogenous tRNA with one of three sup-tRNA anticodons (CUA, UCA or UUA). To enhance prime editing efficiency, we appended a structured RNA motif, tevopreQ1, to the 3′ end of each pegRNA, creating epegRNAs29. In total, we designed 17,579 epegRNAs to convert endogenous tRNAs to sup-tRNAs for each of the three possible sup-tRNA anticodons. Each pool also included 420 control epegRNAs targeting serine and arginine tRNAs, swapping their anticodons for those of other serine and arginine anticodons, which are not expected to give signal in a readthrough-based screen (Supplementary Table 2). We included a unique barcode sequence 3′ of the polyT terminator sequence for each epegRNA to improve our ability to assign each sequencing read to the correct element. The final oligonucleotides had the following design: a common 5′ end for isothermal assembly (5′-tatcttgtggaaaggacgaaacacc-3′), a unique epegRNA sequence followed by a polyT for Pol III termination, an adjustable length linker used to ensure that each element of the library was the same length, a unique 25 bp barcode, and a common 3′ end for isothermal assembly (5′-ctcgagtactaggatccattaggcg-3′). Oligonucleotides were amplified using oSEP0114 and oSEP0115 prior to isothermal assembly into a linearized pSEP0308 with a hU6 promoter for driving expression of the epegRNAs. Final libraries for next generation sequencing were sequenced on an Illumina MiSeq instrument using oSEP0213 as a custom Read1 primer with 300 cycles. Reads were normalized on the basis of sequencing depth and sorted samples were compared with the plasmid pool representation of each element. Oligonucleotides and results for this screen are listed in Supplementary Table 2. To evaluate a wide range of tRNA promoter variants in high-throughput, we took the 418 high-confidence tRNA sequences from the human genome and swapped their anticodon to CUA, UCA or UUA to target the stop codons TAG, TGA or TAA, respectively. A unique barcode was assigned to each tRNA variant to reduce the likelihood that sequencing errors would influence assignment of library members. In addition, a pre-integrated Nextera Read1 adapter was included so that Read1 sequences on an Illumina instrument will read straight into the highly complex barcode for defining clusters. We cloned this pool using isothermal assembly into three different lentiviral backbones to evaluate the effect of different upstream elements on sup-tRNA efficacy: pSEP0308, which has a hU6 promoter, pSEP0309, which has a minU6 promoter and pSEP0310, which contains no exogenous promoter. Reads were normalized on the basis of sequencing depth and sorted samples were compared with the plasmid pool representation of each element. Oligonucleotides and results for this screen are listed in Supplementary Table 3. For the initial leader sequence screen, the 40 bp upstream of each endogenous high-confidence tRNA in the human genome was determined and placed upstream of the mature sequence for tRNA-Leu-TAA-3-1 with its anticodon changed to CUA followed by a polyT termination sequence. As controls, the same leader sequences were placed upstream of the mature sequence for tRNA-Leu-TAA-3-1 with its native anticodon. Oligonucleotides and results for this screen are recorded in Supplementary Table 4. To next screen leader sequences across a wide range of sup-tRNAs, we generated a lentiviral library containing six leader sequences (two top-performing, two bottom-performing and two random sequences) paired with 418 human tRNAs with their anticodons switched to CUA, UCA or UAA. We included a unique 20-bp barcode sequence 3′ of the terminator sequence for each tRNA to improve our ability to assign each sequencing read to the correct element. For analysis, alignment was confirmed between tRNA elements and the 20-bp barcode region and then subsequently the 20-bp barcode was used to assign each read to the appropriate library element. Reads were normalized on the basis of sequencing depth and sorted samples were compared with the plasmid pool representation of each element. Oligonucleotides and results for this screen are recorded in Supplementary Table 5. To design variants for saturation mutagenesis screening, every sequence containing a SNV, paired substitution at all hairpin positions, and 1-bp deletion was generated computationally. Then, 25 bp ends compatible with isothermal assembly were added to either side (see Supplementary Table 1) and a unique barcode was assigned to each tRNA variant to reduce the likelihood that sequencing errors would influence assignment of library members. In addition, a pre-integrated Nextera Read1 adapter was included so that Read1 sequences on an Illumina instrument will read straight into the highly complex barcode for defining clusters. Variants were ordered as a Twist oligonucleotide pool for each individual tRNA for tRNA-Leu-TAA-4-1, tRNA-Arg-CCT-4-1, mouse tRNA-Leu-TAA-2-1 and tRNA-Tyr-GTA-2-1 (sequences in Supplementary Table 6). A pooled library of all possible variants for tRNA-Leu-TAA-1-1, tRNA-Leu-TAA-2-1, tRNA-Leu-TAA-3-1, and tRNA-Leu-TAA-4-1 was ordered as a separate Leu-TAA focused library (sequences in Supplementary Table 7). For the tRNA-Arg-CCT-4-1, mouse tRNA-Leu-TAA-2-1, and tRNA-Tyr-GTA-2-1 screens, Twist oligonucleotides contained a leader sequence element for each tRNA and they were cloned by isothermal assembly into the pSEP0310 lentiviral backbone with no exogenous Pol III promoter. For the tRNA-Leu-TAA-4-1 and Leu-TAA focused library, Twist oligonucleotides did not contain a leader sequence element for each tRNA and they were cloned by isothermal assembly into the pSEP0308 lentiviral backbone with an exogenous hU6 promoter driving expression of the tRNAs. Libraries were sequenced on an Illumina instrument with 26 cycles for Read1 covering the barcode region and 86 cycles for Read2 covering the tRNA region. For analysis, alignment was confirmed between tRNA elements and the 25-bp barcode region and then subsequently the 25-bp barcode was used to assign each read to the appropriate library element. To optimize epegRNA architectures, we designed a lentiviral epegRNA library of 17,280 epegRNAs to test out five spacer variants, PBS lengths from 8 to 16 nt, RTT lengths from 21 to 36 nt, and combinations of each of the 19 mutation variants of interest alongside 720 control epegRNAs with RTTs that do not encode an anticodon edit. We transfected cells transduced with this library with a panel of prime editor proteins that included PEmax and several engineered PE6 variants40 listed in Supplementary Data Tables 9 and 10. We performed experiments in both MMR-deficient (HEK293T) cells and MMR-proficient (HeLa) cells and included conditions in each case in which MMR was transiently inhibited by co-transfection of a dominant-negative MMR protein (MLH1dn)38. For experiments in HeLa cells, we collected genomic DNA 3 d after transfection. We then performed high-throughput sequencing of the lentivirally integrated cassette. For analysis, sequencing reads were initially demultiplexed into individual fastq files by first aligning each read to a corresponding member of the epegRNA library using bowtie2 (ref. Next, each individual fastq file was then trimmed to only include the full-length synthetic target site. Editing outcomes for each epegRNA sequence were quantified using CRISPResso2. HEK293T cells were transfected with our optimized epegRNA sequence and ngRNA alongside PE6c or with an unrelated epegRNA and ngRNA pair targeting the HEK3 locus alongside PE6c (see Supplementary Table 1). RNA was extracted 6 d after transfection using the Qiagen RNeasy kit (Qiagen) according to manufacturer's instructions. RNA-seq libraries were generated using the SMART-Seq mRNA LP kit (Takara Bio) according to manufacturer's instructions and sequenced 2× 75-bp on an Element Biosciences AVITI instrument. Fastq reads were trimmed of adapter sequences using Trim Galore, aligned to the human genome using STAR, and differential expression analysis was performed using DESeq2 and custom R scripts. Total RNA, including small RNAs, was isolated from prime edited cells using miRNeasy kits (Qiagen) according to manufacturer's protocols. RNA was reversed transcribed with SuperScript IV (Thermo Fisher Scientific) using a primer specific to each tRNA gene family queried66 (Supplementary Table 1). Next, quantitative PCR was performed using Power SYBR Green PCR master mix (Thermo Fisher Scientific) and primers specific to each tRNA gene family queried (Supplementary Table 1). For tRNA-Leu-TAA-1-1, we performed targeted tRNA sequencing. Specifically, we measured the relative abundance of the desired tRNA edit versus unedited in a polyclonal edited population in both genomic DNA and total RNA. For targeted tRNA sequencing, we performed reverse transcription using a primer specific to tRNA-Leu-TAA-1-1 and the Induro RT enzyme that is tolerant to RNA modifications39. To generate a diverse PTC sequence context reporter library, 14,746 naturally occurring premature TAG stop codons in protein-coding genes were identified in ClinVar that were annotated as pathogenic, likely pathogenic, or of uncertain significance. Each TAG stop codon was flanked on either side by the 18 nucleotides present in the native mRNA sequence. Positive controls were generated that contained the same sequence context but a TTG Leucine codon instead of a TAG premature stop codon. For negative controls, 2,800 ‘redundant stop' control library members were generated, which were a subset of the ClinVar variants and their TTG controls but with the codon following the TAG stop codon or TTG codon changed to a TAA stop codon followed by a +1 frameshift to prevent readthrough. These sequences were cloned into the pSEP0211 lentiviral backbone between mCherry and GFP via Gibson assembly. Two days following transduction, we extracted both mRNA and genomic DNA from the transduced cell population, generated cDNA by reverse transcription, and sequenced the integrated reporter construct from both the cDNA and genomic DNA samples. For analysis, we calculated an ‘RNA score' metric by quantifying the frequency of each element in the cDNA and dividing that value by the frequency of that element in the genomic DNA. To control for the impact each unique sequence context might have on transcript expression level independent of readthrough activity, we defined a ‘readthrough score' for each ClinVar PTC by dividing the RNA score of each variant by the RNA score of its corresponding no-premature-stop equivalent. Oligonucleotides and results for this screen are recorded in Supplementary Table 15. To account for target-site binding and potential annealing of the full-length RTT product, we extracted the local sequence surrounding these sites to include 3 bp upstream of the putative target site and 34 bp downstream. We included 100 uniquely barcoded positive-control sequences corresponding to the endogenous tRNA-Leu-TAA-1-1 site and 694 negative-control sequences that shared no homology with the epegRNA spacer sequence. These sequences were cloned into the pSEP0310 lentiviral backbone and transduced into HEK293T cells after lentiviral production. These cells were then transfected with an epegRNA expression plasmid alone as a negative control, or with a PE6c prime editor expression plasmid alongside the epegRNA expression plasmid. 3 d after transfection, genomic DNA was isolated and the lentiviral integrated target site was amplified for high-throughput sequencing. The amplified target-site was sequenced using 2× 115 paired-end high-throughput sequencing. For analysis, high-throughput sequencing reads were initially demultiplexed into individual fastq files by first aligning each read to a corresponding member of the target-site library using bowtie2 (ref. Next, each individual fastq file was then trimmed to only include the full-length putative off-target site. Putative off-target editing events were characterized as previously described34. The presence of this off-target marker sequence was quantified using output from CRISPResso2 and was used as a proxy for off-target prime editing. Oligonucleotides and results for this screen are recorded in Supplementary Table 11. A pooled sequencing primer was generated for nominated human off-target sites using the rhAmpSeq design tool (IDT). Genomic DNA was extracted from editor-treated HEK293T cells and amplified with rhAmpSeq pooled sequencing primers according to the manufacturer's protocol. Sequences for rhAmpSeq amplicons were extracted using the R Bioconductor BSGenome package (v.1.4.3) using the GRCh37/hg19 (human) reference genomes. Cells were trypsinized, resuspended in media containing 10% FBS, and the solution was filtered through a 45-μm cell strainer prior to flow cytometry or FACS isolation. Flow cytometry analysis was performed using the CytoFLEX LX Flow Cytometer (Beckman Coulter, C06779) at the Broad Institute Flow Cytometry Core, with CytExpert Acquisition and Analysis Software (v.2.4). FACS was performed on the SONY MA900 Cell Sorter (Sony Biotechnology) and cells were sorted into medium prior to being spun down. Plasmid vectors for mammalian expression of epegRNAs or ngRNAs were cloned via isothermal assembly as previously described67. In brief, a human U6 promoter vector was linearized via polymerase chain reaction and incubated with IDT eBlocks encoding the full-length epegRNA or ngRNA sequence flanked by sequences necessary for isothermal assembly using NEBuilder HiFi DNA Assembly Master Mix (New England BioLabs). A list of epegRNA and ngRNA sequences used in this study are provided in Supplementary Table 1. AAV vector genomes were cloned via isothermal assembly as previously described50. In brief, the v3em vector genome construct (Addgene, #198735) was linearized by restriction digest and new epegRNA and ngRNA sequences, encoded across two IDT eBlocks, were inserted via isothermal assembly using NEBuilder HiFi DNA Assembly Master Mix (New England BioLabs). The PE6d and PE6e v3em vectors were assembled via isothermal assembly using NEBuilder HiFi DNA Assembly Master Mix (New England BioLabs). For all experiments performed in HEK293T or HeLa readthrough reporter polyclonal cell lines and experiments performed in Neuro-2a cells, cells were seeded at 12,000 cells per well into 96-well plates (Corning) and transfected the following day with Lipofectamine 3000 (Thermo Fisher Scientific). A total of 200 ng, 66 ng and 22 ng of prime editor plasmid, epegRNA plasmid and ngRNA plasmid (where indicated), respectively, were transfected per well. 3 d after transfection, cells were lysed and genomic DNA was collected by incubation in a lysis buffer containing 10 mM Tris-HCl, pH 8.0, 0.05% SDS and 800 units per μl of proteinase K (New England BioLabs) at 37 °C for 1 h, followed by enzyme inactivation at 80 °C for 30 min. For rescue experiments performed in the HEK293T disease models, cells were initially seeded at 200,000 cells per well in 12-well plates (Corning). The following day, cells were transfected with 700 ng of PE6c prime editor plasmid, 200 ng of epegRNA plasmid and 75 ng of ngRNA plasmid using Lipofectamine 3000 (Thermo Fisher Scientific). 3 d after transfection, genomic DNA was extracted as described above. For assays using human cells, cells were collected by trypsinization, pelleted, washed in 1× phosphate-buffered saline and repelleted. For the tripeptidyl peptidase (TPP1) assay, pelleted cells were lysed at 4 °C for 30 min in a buffer containing 0.1% Triton X-100 (Sigma) and 10% SDS (Thermo Fisher Scientific) in 1× phosphate-buffered saline. For the hexosaminidase (HEXA) assay, pelleted cells were lysed in a RIPA homogenizing buffer supplemented with a protease inhibitor cocktail (Roche) at 4 °C for 30 min. Supernatant was collected and total protein concentration was quantified using the Pierce BCA protein assay kit according to the manufacturer's protocol (Thermo Fisher Scientific). For both HEXA and TPP1 assays, the sensitivity and accuracy of the assay was determined by a standard curve derived from measurements of known quantities of wild-type protein lysate (Supplementary Fig. For the TPP1 assay, 10 μg of protein lysate was incubated at 37 °C overnight in a 0.1 M sodium acetate buffer at pH 4 containing the Ala-Ala-Phe-7-amido-4-methylcoumarin substrate (Sigma, A3401) at a final concentration of 250 μM in a reaction volume of 40 μl. Enzymatic activity relative to wild-type was calculated by dividing the mean fluorescence values from each treatment condition by the mean fluorescence values of mock-treated wild-type HEK293T cells. For the HEXA assay, 5 μg of protein lysate was incubated for 1 h at 37 °C in a 0.1 M citrate phosphate buffer at pH 4.5 containing either 4-methylumbelliferone-f-N-acetylglucosamine (MUG, Sigma, 69585) or 4-methylumbelliferone-O-N-acetylglucosamine-6-sulfate (MUGS, Sigma, 454428) substrate at a concentration of 3.2 mM. Enzymatic activity relative to wild-type was calculated by dividing the derived mean HEXA activity values for each treatment condition by that of the mean HEXA activity values from mock-treated wild-type HEK293T cells. For measurements of α-L-iduronidase in mouse tissue, protein was extracted by first homogenizing the tissue using the TissueLyser II (QIAGEN) in T-PER tissue protein extraction reagent (Thermo Fisher Scientific, 78510) containing cOmplete Protease Inhibitor Cocktail (Roche). Lysates were then centrifuged at 20,000g for 20 min at 4 °C and supernatant was collected and total protein concentration was quantified using the Pierce BCA protein assay kit according to the manufacturer's protocol (Thermo Fisher Scientific). Up to 40 μg of whole protein lysate was incubated overnight in a 130 mM sodium formate buffer containing 0.42 mg ml−1 of d-saccharic acid 1,4-lactone monohydrate (Sigma-Aldrich, S0375) and 4MU-iduronic acid (0.12 mM; Santa Cruz Biotechnology, sc-220961) at pH 3.5. Cells were collected by trypsinization, pelleted, washed in 1× phosphate-buffered saline then lysed in a RIPA homogenizing buffer supplemented with a protease inhibitor cocktail (Roche) at 4 °C for 30 min. Supernatant was collected and total protein concentration was quantified using the Pierce BCA protein assay kit according to the manufacturer's protocol (Thermo Fisher Scientific). Up to 10 μg of whole protein lysate was separated by SDS–PAGE using a 4–12% Bolt Bis-Tris Plus Mini Protein Gel (Invitrogen). Proteins were transferred to a nitrocellulose membrane using the iBlot 2 Dry Blotting system (Thermo Fisher Scientific) then incubated in a 5% milk solution in 1 × Tris-Buffered Saline, 0.1% Tween-20 (TBST). The blocked membrane was incubated at 4 °C overnight in NPC1 (Abcam, ab134113, 1:2,500 dilution) or GAPDH (Santa Cruz, sc-47724, 1:5,000 dilution) primary antibody. The following day, the membrane was incubated with horseradish peroxidase-conjugated goat anti-rabbit IgG (Abcam, ab6721; 1:10,000 dilution) for NPC1 and anti-mouse IgG (Abcam, ab205719; 1:10,000 dilution) for GAPDH for 1 h at room temperature. Protein was isolated as described for Western blotting. To prepare samples for mass spectrometry, samples were passed over S-trap micro spin columns (Profiti) according to manufacturer's protocols with the following modifications: 10 mM DTT (final concentration) was used instead of TCEP. After adding DTT, tubes were placed on a heating block for 10 min at 95 °C. Disulfides were alkylated with 20 mM iodoacetamide (final concentration) instead of methyl methanethiosulfonate, and after adding the iodoacetamide the samples were incubated at 25 °C for 30 min in the dark. After digestion with 5 µg trypsin (Thermo Fisher 90057), samples were desalted using Pierce Peptide Desalting Spin Columns (Thermo Fisher 89852) following the manufacturer's protocol. The desalted tryptic peptides were resuspended in 100 µl of 100 mM triethylammonium bicarbonate buffer (TEAB), vortexed, and briefly centrifuged. For labelling with tandem mass tags (TMTs), lyophilized TMTpro Label reagents (Thermo Fisher A44520) were used according to manufacturer's protocols. At the time of labelling, aliquots were equilibrated at room temperature and 20 µl of anhydrous acetonitrile was added to each tube. The TMT reagents were vortexed, briefly centrifuged, and allowed to dissolve for 5 min at 25 °C. TMT reagents were added to each 100-µl sample, vortexed, and briefly centrifuged. The samples were incubated for 1 h at 25 °C. After 1 h, 5 µl of 5% hydroxylamine was added to each sample and incubated for 15 min to quench the reaction. Equal amounts of each labelled sample were then combined together and speed-vacuumed to dryness. The TMT-labelled tryptic peptides were separated by reverse phase HPLC (Thermo Ultimate 3000) using a Thermo PepMap RSLC C18 column (2 µm tip, 75 µm x 50 cm ES903) over a gradient before nano-electrospray using a Orbitrap Exploris 480 mass spectrometer (Thermo). Solvent A was 0.1% formic acid in water and solvent B was 0.1% formic acid in acetonitrile. The mass spectrometer was operated in a data-dependent mode. The parameters for the full scan MS were: resolution of 60,000 across 450–1,600 m/z and maximum IT 50 ms. The full mass spectrometry scan was followed by MS/MS for as many precursor ions in a three-second cycle with a NCE of 32, dynamic exclusion of 30 s and resolution of 45,000. Raw mass spectral data files (.raw) were searched using Sequest HT in Proteome Discoverer (Thermo). Sequest search parameters were: 10 ppm mass tolerance for precursor ions; 0.02 Da for fragment ion mass tolerance; 2 missed cleavages of trypsin. Fixed modifications were carbamidomethylation of cysteine and TMTpro modification on lysines and peptide N termini. AAV production was performed using HEK293T clone 17 cells (ATCC, CRL-11268) maintained in DMEM plus GlutaMAX (Thermo Fisher Scientific) with 10% heat-inactivated FBS without antibiotic in 150 mm2 dishes (Thermo Fisher Scientific). The day before transfection cells were plated at a density of 18 million cells per 150 mm2 plate. The following day, 5.7 μg of AAV genome plasmid, 11.4 μg of pHelper (Clontech) and 22.8 μg of rep-cap plasmid per plate were delivered via polyethyleneimine transfection (PEI MAX, Polysciences). Three days after transfection, cells were collected by cell scraping and pelleted at 3,000g for 10 min. Medium was decanted into a solution of poly(ethylene glycol) (PEG) 8000 (Sigma-Aldrich) and NaCl at a final concentration of 8% PEG and 500 mM NaC and incubated on ice for 2 h. The cell pellet was resuspended in 500 µl per plate of a hypertonic buffer containing 40 mM Tris base, 500 mM NaCl, 2 mM MgCl2 and 100 U ml−1 salt active nuclease (ArcticZymes) and incubated for one hour at 37 °C. The medium-containing solution was then pelleted by centrifugation at 3,000g for 30 min and the resulting pellet was resuspended in 500 µl per plate of a hypertonic lysis buffer and combined with the cell lysate. This lysate was then added to Beckman Coulter Quick-Seal tubes via 16-gauge, 5-inch needles (Air-Tite N165) in a discontinuous gradient of iodixanol in sequentially floating layers as follows: 9 ml of 15% iodixanol in 500 mM NaCl and 1× phosphate-buffered saline-MK (1× phosphate-buffered saline plus 1 mM MgCl2 and 2.5 mM KCl), 6 ml of 25% iodixanol in 1× phosphate-buffered saline-MK and 5 ml each of 40% and 60% iodixanol in 1× phosphate-buffered saline-MK with phenol red at a concentration of 1 μg ml−1 in the 15%, 25% and 60% layers to aid layer visualization This gradient was then ultracentrifuged using a fixed-angle Ti 70 rotor in an Optima XPN-100 Ultracentrifuge (Beckman Coulter) at 68,000 rpm for one hour at 18 °C. Next, 3 ml of virus-containing solution was extracted from the 40–60% iodixanol interface via an 18-gauge needle. Buffer was exchanged for cold phosphate-buffered saline with 0.001% F-68 using a PES 100 kD MWCO column (Thermo Fisher Scientific) and concentrated. The resulting AAV-containing solution was sterile filtered using a 0.22-μm filter and quantified by quantitative PCR (qPCR) (AAVpro Titration Kit version 2, Clontech). Purified virus was stored at 4 °C until use. All experiments involving live animals were approved by the Broad Institute Institutional Animal Care and Use Committee (D16-00903; 0048-04-15-2). Mouse housing facilities were maintained at 20–22 °C with 30–50% humidity, on a 12 h light/12 h dark cycle with ad libitum access to standard rodent diet and water. For experiments involving IduaW392X mice, we used Strain from Jackson Laboratory. Syringes for microinjection were prepared by pulling PCR Micropipettes (Drummond Scientific Company, 5-000-1001-X10) with the Sutter P1000 micropipette puller for a tip diameter size of 100 μm. A total of 4 μl of injection solution was front-loaded to the micropipette syringe for injection. Neonatal mice were anaesthetized on ice. Litters were randomized for injection with a given AAV composition. No statistics were performed to pre-determine sample size for each group. In experiments demonstrating readthrough of an exogenous reporter co-administered via AAV, we evaluated both TAG and TGA stop codon readthrough. The reporter construct contained an eGFP expression cassette with either a TAG or TGA PTC at codon 81 or the wild-type glutamine codon. In experiments evaluating the long-term tolerability of PERT, we delivered editing reagents in the v3em dual-AAV architecture introducing either an CUA anticodon into the mouse tRNA-Leu-TAA-2-1 gene, or a control +5 G-to-T edit in the Dnmt1 locus. At three weeks, the cortices and livers from mice were processed for whole proteome mass spectrometry to evaluate readthrough past natural stop codons. Mice used in this study were sacrificed by CO2 asphyxiation, and unperfused tissues were immediately dissected. For protein and DNA analysis, dissected tissues were immediately frozen in liquid nitrogen. Protein extraction was performed as described above. Genomic DNA and RNA was extracted using the QIAGEN AllPrep DNA/RNA kit according to the manufacturer's protocol. For histology and immunohistochemistry, dissected tissues were placed in 10% neutral buffered formalin and then soaked in 70% ethanol prior to paraffin embedding. Following routine processing and paraffin embedding, tissue sections were stained with either haematoxylin and eosin, Alcian blue, or prepared for immunohistochemistry. For immunohistochemistry, 4-μm-thick sections were deparaffinized and rehydrated, followed by antigen retrieval using a sodium citrate buffer. After quenching endogenous peroxidase and application of a protein block (Dako), sections were incubated with either an anti-GFP antibody (Abcam, ab183734) or an anti-iduronidase (R&D systems, AF4119) antibody. Staining was detected using a species-specific detection kit, diaminobenzidine was used as the chromogen, and Mayer's Haematoxylin (Dako) was used as the counterstain. For Alcian blue staining, 4-μm formalin-fixed, paraffin-embedded tissue sections were deparaffinized and rehydrated. Sections were stained with the Alcian Blue–1%, pH 2.5 kit (Newcomer, 9102A). Tissue histopathology was performed at the University of Minnesota Comparative Pathology Shared Resource and pathology scoring was performed by a board-certified veterinary pathologist blinded to the treatment conditions. All screens were performed in independent biological duplicates. Sample sizes for all other experiments and analyses are defined in the corresponding figure legends. Where performed, select comparisons are indicated within the figure, and P values can be found in corresponding Supplementary Tables. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Amplicon next generation sequencing data have been deposited to the NCBI Sequence Read Archive database under accession PRJNA1335297. Source data are provided with this paper. Bioinformatics analysis was performed using publicly available programs and parameters described in the Methods. The code for CRISPResso is available at https://github.com/pinellolab/CRISPResso2. Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Wang, J. et al. AAV-delivered suppressor tRNA overcomes a nonsense mutation in mice. Newby, G. A. et al. Base editing of haematopoietic stem cells rescues sickle cell disease in mice. Everette, K. A. et al. Ex vivo prime editing of patient haematopoietic stem cells rescues sickle-cell disease phenotypes after engraftment in mice. Chiesa, R. et al. Base-edited CAR7 T cells for relapsed T-cell acute lymphoblastic leukemia. Lee, R. G. et al. Efficacy and safety of an investigational single-course CRISPR base-editing therapy targeting PCSK9 in nonhuman primate and mouse models. Beam Therapeutics announces positive initial data for BEAM-302 in the phase 1/2 trial in alpha-1 antitrypsin deficiency (AATD), demonstrating first ever clinical genetic correction of a disease-causing mutation. Prime Medicine announces breakthrough clinical data showing rapid restoration of DHR positivity after single infusion of PM359, an investigational prime editor for chronic granulomatous disease. Arbab, M. et al. Base editing rescue of spinal muscular atrophy in cells and in mice. Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. A meta-analysis of nonsense mutations causing human genetic disease. Liang, H., Cavalcanti, A. R. & Landweber, L. F. Conservation of tandem stop codons in yeasts. Wu, C., Roy, B., He, F., Yan, K. & Jacobson, A. Poly(A)-binding protein regulates the efficiency of translation termination. Ito-Harashima, S., Kuroha, K., Tatematsu, T. & Inada, T. Translation of the poly(A) tail plays crucial roles in nonstop mRNA surveillance via translation repression and protein destabilization by proteasome in yeast. Pavon-Eternod, M. et al. tRNA over-expression in breast cancer and functional consequences. Gingold, H. et al. A dual program for translation regulation in cellular proliferation and differentiation. & O'Donoghue, P. Pathways to disease from natural variations in human cytoplasmic tRNAs. Targeted sequencing reveals expanded genetic diversity of human transfer RNAs. Ehrlich, R., Davyt, M., Lopez, I., Chalar, C. & Marin, M. On the track of the missing tRNA genes: a source of non-canonical functions? Ding, W. et al. Rare codon recoding for efficient noncanonical amino acid incorporation in mammalian cells. Engineered pegRNAs improve prime editing efficiency. Ho, A. T & Hurst, L. D. Stop codon usage as a window into genome evolution: mutation, selection, biased gene conversion and the TAG paradox. Orellana, E. A., Siegal, E. & Gregory, R. I. tRNA dysregulation and disease. Porter, J. J., Ko, W., Sorensen, E. G. & Lueck, J. D. Optimization of ACE-tRNAs function in translation for suppression of nonsense mutations. Preece, R. et al. ‘Mini' U6 Pol III promoter exhibits nucleosome redundancy and supports multiplexed coupling of CRISPR/Cas9 effects. Programmable deletion, replacement, integration and inversion of large DNA sequences with twin prime editing. Gao, Z., Herrera-Carrillo, E. & Berkhout, B. Delineation of the exact transcription termination signal for type 3 polymerase III. Facilitated recycling pathway for RNA polymerase III. Widespread occurrence of non-canonical transcription termination by human RNA polymerase III. Nakano, Y. et al. Genome-wide profiling of tRNA modifications by Induro-tRNAseq reveals coordinated changes. Doman, J. L. et al. Phage-assisted evolution and protein engineering yield compact, efficient prime editors. Tsai, S. Q. et al. CIRCLE-seq: a highly sensitive in vitro screen for genome-wide CRISPR–Cas9 nuclease off-targets. Jin, S. et al. Genome-wide specificity of prime editors in plants. Cassan, M. & Rousset, J. P. UAG readthrough in mammalian cells: effect of upstream and downstream stop codon contexts reveal different signals. Saturation variant interpretation using CRISPR prime editing. Residual levels of tripeptidyl-peptidase I activity dramatically ameliorate disease in late-infantile neuronal ceroid lipofuscinosis. & Rattazzi, M. C. Low levels of beta hexosaminidase A in healthy individuals with apparent deficiency of this enzyme. Guatibonza Moreno, P. et al. At a glance: the largest Niemann-Pick type C1 cohort with 602 patients diagnosed over 15 years. Ishigaki, Y., Li, X., Serin, G. & Maquat, L. E. Evidence for a pioneer round of mRNA translation: mRNAs subject to nonsense-mediated decay in mammalian cells are bound by CBP80 and CBP20. Efficient prime editing in mouse brain, liver and heart with dual AAVs. Wang, D. et al. Ataluren suppresses a premature termination codon in an MPS I-H mouse. Zincarelli, C., Soltys, S., Rengo, G. & Rabinowitz, J. E. Analysis of AAV serotypes 1-9 mediated gene expression and tropism in mice after systemic injection. Ko, W. et al. ACE-tRNAs are a platform technology for suppressing nonsense mutations that cause cystic fibrosis. Identity elements of human tRNALeu: structural requirements for converting human tRNASer into a leucine acceptor in vitro. Efficient site-specific integration of large genes in mammalian cells via continuously evolved recombinases and prime editing. Targeted DNA integration in human cells without double-strand breaks using CRISPR-associated transposases. Ran, F. A. et al. Genome engineering using the CRISPR–Cas9 system. Engineered virus-like particles for transient delivery of prime editor ribonucleoprotein complexes in vivo. Engineered virus-like particles for efficient in vivo delivery of therapeutic proteins. Castellani, C. & team, C. CFTR2: How will it help care? Schulz, H. L. et al. Mutation spectrum of the ABCA4 gene in 335 Stargardt disease patients from a multicenter german cohort—impact of selected deep intronic variants and common SNPs. The genetic landscape and epidemiology of phenylketonuria. Improved molecular diagnosis of dystrophinopathies in an unselected clinical cohort. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Udagawa, T., Seki, M. & Inada, T. Optimized protocol for tRNA identification in the ribosomal complexes from human cell lines. & Kim, J. S. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. This study was funded by US National Institutes of Health grants R35GM118062 and RM1HG009490; Broad Institute Chemical Biology and Therapeutics Science program funds; and Howard Hughes Medical Institute (HHMI). We thank the Koch Institute's Robert A. Swanson (1969) Biotechnology Center for technical support, including R. Schiavoni at the Biopolymers and Proteomics Core Facility. HHMI lab heads have previously granted a nonexclusive CC BY 4.0 license to the public and a sublicensable license to HHMI in their research articles. Pursuant to those licenses, the author-accepted manuscript of this article can be made freely available under a CC BY 4.0 license immediately upon publication. These authors contributed equally: Sarah E. Pierce, Steven Erwood Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA, USA Sarah E. Pierce, Steven Erwood, Keyede Oye, Meirui An, Nicholas Krasnow, Emily Zhang, Aditya Raguram & David R. Liu Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA, USA Sarah E. Pierce, Steven Erwood, Keyede Oye, Meirui An, Nicholas Krasnow, Emily Zhang, Aditya Raguram & David R. Liu Howard Hughes Medical Institute, Harvard University, Cambridge, MA, USA Sarah E. Pierce, Steven Erwood, Keyede Oye, Meirui An, Nicholas Krasnow, Emily Zhang, Aditya Raguram & David R. Liu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed the research, performed experiments and analysed data. prepared protein samples for mass spectrometry. are inventors on a patent application filed by the Broad Institute on the design and applications of PERT. is a co-founder of Beam Therapeutics, Prime Medicine, Pairwise Plants, Editas Medicine and nChroma Bio, companies that use or deliver genome-editing agents. The other authors declare no competing interests. Nature thanks John Lueck, who co-reviewed with Joseph Porter, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. For example, in indicating “tRNA-Gln-CTG-6-1”: Gln indicates the amino acid that the tRNA normally carries (here, glutamine); CTG is the anticodon sequence that recognizes the corresponding codon in mRNA; 6 is the isodecoder index, which distinguishes different tRNA genes that share the same amino acid and anticodon but differ slightly in sequence elsewhere; and 1 is the locus copy number, identifying which genomic copy of this isodecoder is being referenced. (b) Sequencing reads with the specified edit for two endogenous tRNAs that were converted to sup-tRNAs with prime editing. (c) Schematic of readthrough quantification from flow cytometry data. Values and error bars reflect mean ± s.d. (d) % GFP+ cells (top) and relative protein yield (bottom) for two sup-tRNAs delivered via transfection or as a single-copy PE-edited tRNA, paired with either an overexpressed reporter or a single copy reporter. (a) Comparison of fold enrichment between TAG and TGA stop codon reporters. Values and error bars reflect mean ± s.d. (b) Validation of top-performing epegRNAs in an arrayed transfection format. (c) Sequencing reads with the specified edit for 16 epegRNAs validated individually from the TAG PE2 screen. (d) Fold enrichment in GFP-sorted cells compared to plasmid pool for arginine, serine, leucine, and tyrosine tRNA isoacceptors. (e,f) Fold enrichment in GFP-sorted cells compared to plasmid pool for epegRNAs targeting the endogenous tRNA-Leu-TAA-1-1 (e) and tRNA-Ser-AGA-4-1 (f) genes with the indicated spacers, RTT homology lengths, and PBS lengths. (a) Fold enrichment of epegRNAs generating sup-tRNAs compared to controls in GFP sorted cells compared to the plasmid pool in the TGA screen. (d) Sequencing reads with the specified edit for 8 epegRNAs validated individually from the TGA PE2 screen. (a) Iterative pooled screening pipeline for lentiviral sup-tRNA optimization. (b) Comparison of Pol III promoters and their respective sizes for sup-tRNA expression. (c) Fold enrichment of sup-tRNA variants in GFP sorted cells compared to the plasmid pool representation when driven by a hU6, minU6, or endogenous promoter with no specified leader sequence (no exogenous promoter). Variants are separated by controls (with a non-sup-tRNA anticodon) or sup-tRNAs with a CUA anticodon. (d) Fold enrichment in GFP sorted cells compared to bulk transduced cells for a single sup-tRNA (tRNA-Leu-TAA-3-1, UAA > CUA) downstream of 418 distinct tRNA leader sequences. (e) Arrayed lentiviral validation of a single sup-tRNA (tRNA-Leu-TAA-4-1, UAA > CUA) downstream of the top and bottom hits from the screen in (d) compared to the endogenous leader sequence for the indicated tRNA as well as randomly designed synthetic 40-bp sequences. (f-g) Fold enrichment of sup-tRNA variants in GFP sorted cells compared to plasmid pool separated based on upstream (f) or downstream (g) sequence. Values and error bars reflect mean ± s.d. (a) Log2 fold change in abundance among GFP-positive cells of mutant sup-tRNAs compared to an ac-only sup-tRNA for tRNA-Tyr-GTA-2-1 with its anticodon changed to CUA. Mutation position is indicated on the x-axis. (b) Heatmap representing log2 fold change in abundance of mutant sup-tRNAs among GFP-positive cells compared to the plasmid pool for each single-nucleotide variant (SNV) in the four Leu-TAA sup-tRNA genes. Mutation position is indicated on the x-axis. (c) Log2 fold change in abundance of mutant sup-tRNAs among GFP-positive cells compared to the plasmid pool for all naturally occurring human variants of tRNA-Leu-TAA-1-1 documented in tRNAdb. (d) Sequencing reads with the specified edit (%) for epegRNAs used to install each of the indicated combinations of mutations in the endogenous tRNA-Leu-TAA-1-1 gene alongside changing its anticodon to CUA. (a) Targeted tRNA-seq of tRNA-Leu-TAA-1-1 measures the abundance of engineered sup-tRNA, ac-only sup-tRNA, or wild-type tRNA at the DNA and RNA levels. (b) Sequence logo of targeted tRNA-seq for reads corresponding to the wild-type tRNA, ac-only sup-tRNA, and engineered sup-tRNA at the RNA and DNA levels. (c-d) Probability of a G base at position 26 (c) or a G base at position 37 (d) for reads corresponding to the engineered sup-tRNA, ac-only sup-tRNA, or wild-type tRNA. The Induro RT used for cDNA generation will result in an “N” nucleotide at any position that is modified in the RNA. (a) Percent editing efficiency across all epegRNAs with the indicated spacer from the PE6c screen in HeLa (top) and HEK293T (bottom) cells. Trends across the given mutation combinations are shown. (b) epegRNA spacer-dependent percent editing in HeLa cells transfected with the PE6c prime editor. Each point indicates a unique epegRNA sequence. (c) Fold change in desired editing efficiency when cells were transfected with MLH1dn and PEmax (PE4) compared to being transfected with PEmax alone (PE2) in HeLa (top) and HEK293T (bottom) cells. Pearson correlation is shown in the top right. (b) Percent of sequencing reads containing single nucleotide polymorphisms (SNPs) (top) or insertions or deletions (indels) (bottom) at each of the indicated locations in the genome, either on-target at the tRNA-Leu-TAA-1-1 locus or at 18 potential off-target sites. (c) Percent prime editing at each of the indicated target site types in the off-target screen after subtracting background frequencies observed in cells transfected with epegRNA only and no prime editor. (d) Percent prime editing marker frequency (top) or percent indel frequency after background subtraction (bottom) for individual validation of nominated off-targets from the off-target screen. (e) RNA-seq comparing cells transfected with PE6c and an epegRNA/ngRNA designed to convert tRNA-Leu-TAA-1-1 into a sup-tRNA compared to an unrelated epegRNA/ngRNA pair designed to edit the HEK3 locus. Each data point represents the expression of a gene. No genes had an adjusted p-value < 0.05 and a |log2 fold change |>1. (f) Quantitative PCR analysis of the expression levels of 28 different tRNA families in cells expressing a PE-installed ac-only sup-tRNA, engineered sup-tRNA, or wild-type cells. (b-d) Protein-level analysis of whole proteome mass spectrometry data from cells prime edited to install an ac-only sup-tRNA (b), engineered sup-tRNA (c), or unedited control cells (d). Log2 fold change in protein abundance compared to unedited control cells. Dashed lines indicate an adjusted p-value of 0.05 on the y-axis and log2 fold change in abundance of ±1 compared to unedited control cells. (e) Normalized abundance of peptides identified past a premature termination codon (PTC) (top) or past natural TAG termination codons (NTCs) (bottom) for non-prime edited control HEK293T cells. Dotted lines correspond to trypsin cleavage sites. (f-g) Target-ion mass spectrometry showing change in abundance of peptides predicted to result from NTC readthrough of the 69 most abundant proteins that use a TAG stop codon in HEK293T cells expressing an ac-only sup-tRNA (f) or an engineered sup-tRNA (g) compared to non-prime-edited cells. All peptides identified post-PTC (GFP) are significant (adjusted p < 0.05). Peptides for pre-NTC controls and for one post-NTC peptide identified are not significant (adjusted p > 0.05). The data shown in f and g are only a subset of the complete data set, which is provided in Supplementary Table 14. (a) Sequencing reads with the specified edit (%) for the brain cortex of BL/6 mice treated with prime editing agents to convert tRNA-Leu-TAA-2-1 into CUA or UCA suppressors alongside a wild-type GFP reporter (WT) or a GFP reporter containing a TAG or TGA stop codon. (b) Sequencing reads with the specified edit (%) for the brain cortex of BL/6 mice treated with prime ediing agents to convert tRNA-Leu-TAA-2-1 into a CUA suppressor or to introduce a benign Dnmt1 edit. (c) Body weight in grams (g) over time following treatment with prime editing agents, corresponding to editing shown in (b). (d) Idua enzymatic activity in Hurler syndrome mice without prime editing treatment. Arrows show vacuolated cells and inset is a high power magnification of a Purkinje cell. (f) Tissue pathology score based on GAG storage evaluated by microscopy. Values and error bars reflect mean ± s.d. Design and results for PE2 epegRNA screens to convert endogenous tRNAs into sup-tRNAs with TAG, TGA, and TAA reporters. Design and results for initial lentiviral sup-tRNA screen. Design and results for sup-tRNA leader sequence screen. Design and results for saturation mutagenesis screening of individual sup-tRNAs. Design and results for epegRNA screening of Leu-TAA-1-1 into a sup-tRNA with an adjacent synthetic target site in HeLa cells. Design and results for epegRNA screening of Leu-TAA-1-1 into a sup-tRNA with an adjacent synthetic target site in HEK293T cells. Design and results for lentiviral off-target screening of the epegRNA. Evaluation of first and second stop codons for each protein in the human proteome. Whole proteome mass spectrometry on human cells treated with PERT. Target-ion mass spectrometry on potential 3′UTR peptides for the top 69 proteins with a TAG stop codon in cells treated with PERT. Design and results for sequence contexts amenable to readthrough with a CUA sup-tRNA. CFTR cDNA constructs used in validation of sequence context screening results. Whole proteome mass spectrometry on mouse livers and cortices treated with PERT. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Pierce, S.E., Erwood, S., Oye, K. et al. Prime editing-installed suppressor tRNAs for disease-agnostic genome editing. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69485120/is-homo-floresiensis-human-ancestor-still-alive/'>A Tiny Apelike Humanoid May Still Be Living in Plain Sight, Scientist Says</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 15:56:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. The scientific community believe a small species of human known as homo floresiensis once lived on the island of Flores, Indonesia, around 50,000 years ago. But one professor thinks the apelike humanoids could still live there, evolution be damned. Think of this as the hunt for Bigfoot, only with a much smaller target. He wrote a book in 2022, Between Ape and Human: An Anthropologist on the Trail of a Hidden Hominoid, and The Debrief ran an interview with Forth on the quest. “What really interested me in the lai ho'a is that it was small, like the figures in Nage country,” Forth told The Debrief, “but it was reckoned still to be alive. And indeed, there were a few people around, it seemed, who claimed to have seen one or more.” These creatures have a human-like upright gait, come hairier than humans but not as hairy as an ape, and have a distinct ape-like face, according to the Lio people's accounts to Forth. The professor's hopes of a living homo floresiensis were emboldened at the finding of fossils roughly 20 years ago. Forth's book purports that these ape-man creatures lived at least into modern times, and he believes credible sightings mean there's a chance a small population still exists. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Found a Mass Grave of Roman Soldiers</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251118220049.htm'>Inflammation turns bone marrow into a breeding ground for disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 15:23:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Every moment, the bone marrow generates millions of fresh blood and immune cells. Aging, chronic inflammation, or somatic mutations can disrupt communication among these cell groups, reducing normal stem-cell renewal and allowing mutated HSCs to expand unnoticed. Myelodysplastic syndrome (MDS), a related disorder involving clonal HSCs, causes inefficient blood-cell production and gradual failure of the bone marrow. To better understand how mutated HSC clones gain dominance, an international research team co-led by Judith Zaugg from EMBL and University of Basel and Borhane Guezguez from UMC Mainz carried out an extensive molecular and spatial analysis of human bone marrow. Using single-cell RNA sequencing, biopsy imaging, proteomics, and co-culture models, the researchers created a detailed map of the bone marrow microenvironment in healthy donors (including those with CHIP) and in patients with MDS. Their analysis revealed an unexpected cellular shift that begins long before clinical signs appear. "I was surprised to observe such pronounced remodeling of the bone marrow microenvironment already in individuals with CHIP, although the underlying cause-and-effect relationships remain unclear," said Zaugg, co-senior author, EMBL Group Leader, and Professor at Basel University. Unlike healthy stromal cells, these inflammatory MSCs (iMSC) produce large amounts of interferon-induced cytokines and chemokines. These molecules attract and activate interferon-responsive T cells, which then intensify the inflammatory activity. This creates a feed-forward loop that maintains chronic inflammation, disrupts normal blood formation, and contributes to vascular changes in the marrow. Interestingly, the researchers did not find signs that mutated hematopoietic cells in MDS directly trigger this inflammatory response. They were able to separate mutated from non-mutated cells using SpliceUp, a computational method developed by co-lead author and EMBL alumnus Maksim Kholmatov in collaboration with Pedro Moura and Eva Hellström-Lindberg from Karolinska Institute. SpliceUp identifies mutated cells in single-cell datasets by detecting abnormal RNA-splicing patterns. This failure may help explain why the bone marrow stops working properly," said Karin Prummel, co-lead author and EMBL postdoc. "It was quite surprising to see the lack of a direct inflammatory effect that we could attribute to the mutant cells," said Maksim Kholmatov, co-lead author and EMBL alumnus. Anti-inflammatory drugs or therapies that adjust interferon signaling may help preserve marrow function in older adults with CHIP. The specific molecular features of iMSCs and interferon-responsive T cells may also serve as early biomarkers for people at elevated risk. "As advances in molecular profiling allow us to detect pre-leukemic states years before clinical onset, understanding how stromal and immune cells interact provides a foundation for preventive therapies that intercept disease progression before leukemia develops." Beyond blood disorders, the results contribute to a broader understanding of 'inflammaging', the low-level, chronic inflammation that supports many age-related conditions, including cancer and cardiovascular and metabolic disease. The bone marrow, once considered only a site of blood production, now appears to be both affected by and responsible for systemic inflammatory aging. By showing how interactions between immune and stromal cells drive these changes, the study offers a model for exploring inflammatory remodeling in other myeloid malignancies and advanced leukemia. "It will be crucial to study these processes over time; our current findings are based on cross-sectional data," Zaugg said. "This has important implications for therapies that replace malignant cells but leave the bone marrow niche intact, such as blood stem cell transplantation. The work appears alongside a complementary study examining the MDS bone marrow microenvironment, also published in Nature Communications and led by Marc Raaijmakers from Erasmus MC Cancer Institute in Rotterdam. Satellites Reveal Ancient “Mega Traps” Hidden in the Chilean Andes Frozen 40,000-Year-Old Mammoth Reveals Shockingly Intact RNA and Hidden Genetic Secrets Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-safe-is-melatonin-and-how-does-the-sleep-aid-work-experts-explain/'>Should You Really Be Taking Melatonin? What Experts Want You to Know</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Melatonin supplements have become a ubiquitous sleep aid, but research shows that benefits are modest, and the heart health effects of long-term use are unknown As many as 70 million people in the U.S. have problems with sleep, and many are increasingly turning to melatonin supplements to help them fall asleep. People generally perceive these over-the-counter gummies, liquids and tablets as harmless. But recent research has raised new questions about whether the popular sleep aid is as benign as people assume, especially the longer it's used. Melatonin is a hormone that humans naturally produce, as do all other mammals, including bears, and birds. Instead of acting like a sedative, melatonin sends the body biological signals that it's nighttime. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Melatonin supplements are generally considered safe, but some people do report headaches, dizziness or nausea after exposure. Taking too much at one time can disrupt sleep rather than improve it. Supplements are sold with doses that are “much, much higher than anybody is naturally producing,” says Jamie Zeitzer, a sleep specialist at Stanford University. The hormone is commonly sold in five-milligram concentrations, though some “extra strength” products say they contain up to 40 milligrams. Melatonin is sold as a supplement, which means such products reach the market without the Food and Drug Administration first evaluating their safety or production standards. Shalini Paruthi, a sleep medicine physician at John J. Cochran Veterans Hospital in St. Louis and spokesperson for the American Academy of Sleep Medicine, says to look for U.S. Pharmacopeia–certified products, which are tested for the quantity of their listed ingredients. At a conference in early November, researchers at SUNY Downstate Health Sciences University and their colleagues presented an analysis of U.K. medical records that raised concerns about long-term melatonin use. Almost 5 percent of 65,414 adults with insomnia who took melatonin for at least a year experienced heart failure (whereas almost 3 percent of those who did not take the sleep aid did so). People who took melatonin for at least a year were also more than three times as likely to be hospitalized for heart failure and nearly twice as likely to die from any cause compared with those who had no medical record of taking melatonin. But some experts believe the findings, which have not yet been published or peer-reviewed, might say more about the consequences of poor sleep rather than melatonin use. “I don't see a very good reason for melatonin to be directly involved in [heart failure],” Zeitzer says. Bad sleep is more likely putting extra pressure on the cardiovascular system, he says. Chronic insomnia is already known to be related to a higher risk of cardiovascular disease, metabolic dysfunction and issues with memory, learning and mental health. A more likely explanation for the new study's trends is that people with poor sleep are relying on melatonin, which might not be helping them, Martin says. Other long-term studies haven't found evidence that melatonin harms the heart. But there isn't much substantial, long-term safety data for melatonin. Parents are increasingly giving their children melatonin to help with sleep. Evidence suggests it can make a real difference for some children with neurodivergent conditions. In such cases, “we would definitely recommend a low-dose melatonin to see if that helps,” she says. “And for a lot of these kids, it really does.” In placebo-controlled trials, people who take melatonin fell asleep just seven minutes faster on average and slept about eight minutes longer than those who took a placebo. But improvements are larger when participants knew they were getting melatonin. The American Academy of Sleep Medicine recommends melatonin for some sleep issues related to time adjustments such as jet lag and shift work but not for insomnia. If melatonin works for you, keep using it, Zeitzer says. Evidence isn't strong enough yet to show long-term use improves or worsens health, Martin says. She suggests people should work with a specialist if their sleep problems persist. Lori Youmshajekian is a science journalist who reports on consumer health, environmental issues and scientific misconduct. She holds a master's degree in science journalism from New York University and has written for National Geographic, Wired and Retraction Watch, among other outlets. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/these-birds-learned-to-tweet-like-r2-d2-listen-to-the-uncanny-results/'>These Birds Learned to Tweet Like R2-D2. Listen to the Uncanny Results</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The lovable Star Wars droid is helping to shed light on why some bird species are better at mimicking sounds than others Some birds such as starlings (right) are better at mimicking R2-D2 (left) than others. A long time ago in a galaxy far, far away, R2-D2 beeped and booped—and now birds that copy the Star Wars character are giving scientists fresh insight into how different species imitate complex sounds. A study, published recently in Scientific Reports, analyzed the sounds of nine species of parrots, including Budgies, as well as European Starlings to see how accurately each bird mimicked R2-D2's robotic whirring. Researchers did acoustic analyses on samples of birds imitating the plucky droid that were already available online to compare how statistically similar each bird's noises were to a model of R2-D2's sounds. The starlings, a type of songbird, emerged as star vocalists: their ability to produce “multiphonic” noises—in their case, two different notes or tones expressed simultaneously—allowed them to replicate R2-D2's complex chirps more accurately. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “Starlings can produce two sounds at once because they control both sides of the syrinx independently,” says study co-author Nick Dam, an evolutionary biologist at Leiden University in the Netherlands. “Parrots are physically incapable of producing two tones simultaneously.” It isn't exactly known why different species developed differing control over their syrinx. “Likely, some ancestor of songbirds happened to evolve the ability to control the muscles on both sides of the syrinx, and this helped them in some way,” says University of Northern Colorado biologist Lauryn Benedict, who wasn't involved in the study but sometimes works with its authors. (And she and other scientists are looking for public submissions of other examples of parrots imitating sounds for their "The Many Parrots Project") “A wider sample would be really neat,” she says, “and they could test all kinds of different sounds, not just R2-D2!” Kate Graham-Shaw is a journalist based in New York City. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03714-0'>Mind-reading devices can now predict preconscious thoughts: is it time to worry?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 10:41:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Liam Drew is a freelance writer based near London, UK. Years later, Smith started making music again, thanks to an implant that recorded and analysed her brain activity. When she imagined playing an on-screen keyboard, her brain–computer interface (BCI) translated her thoughts into keystrokes — and simple melodies, such as ‘Twinkle, Twinkle, Little Star', rang out1. The rise of brain-reading technology: what you need to know The rise of brain-reading technology: what you need to know For Smith, it seemed as if the piano played itself. Smith is one of roughly 90 people who, over the past two decades, have had BCIs implanted to control assistive technologies, such as computers, robotic arms or synthetic voice generators. These volunteers — paralysed by spinal-cord injuries, strokes or neuromuscular disorders, such as motor neuron disease (amyotrophic lateral sclerosis) — have demonstrated how command signals for the body's muscles, recorded from the brain's motor cortex as people imagine moving, can be decoded into commands for connected devices. But Smith, who died of cancer in 2023, was among the first volunteers to have an extra interface implanted in her posterior parietal cortex, a brain region associated with reasoning, attention and planning. Andersen and his team think that by also capturing users' intentions and pre-motor planning, such ‘dual-implant' BCIs will improve the performance of prosthetic devices. “The surprise was that when we go into the posterior parietal, we can get signals that are mixed together from a large number of areas,” says Andersen. “There's a wide variety of things that we can decode.” The ability of these devices to access aspects of a person's innermost life, including preconscious thought, raises the stakes on concerns about how to keep neural data private. It also poses ethical questions about how neurotechnologies might shape people's thoughts and actions — especially when paired with artificial intelligence. Ethicists worry that, left unregulated, these devices could give technology companies access to new and more precise data about people's internal reactions to online and other content. “Whole-brain interfacing is going to be the future,” says Tom Oxley, chief executive of Synchron, a BCI company in New York City. Along the way, he says, AI will continue to improve decoding capabilities and change how these systems serve their users. “It leads you to the final question: how do we make that safe?” Consumer neurotech products capture less-sophisticated data than implanted BCIs do. This measures ripples of electrical activity that arise from the averaged firing of huge neuronal populations and are detectable on the scalp. Still, EEG can reveal overall brain states, such as alertness, focus, tiredness and anxiety levels. Companies already offer headsets and software that give customers real-time scores relating to these states, with the intention of helping them to improve their sports performance, meditate more effectively or become more productive, for example. AI has helped to turn noisy signals from suboptimal recording systems into reliable data, explains Ramses Alcaide, chief executive of Neurable, a neurotech company in Boston, Massachusetts, that specializes in EEG signal processing and sells a headphone-based headset for this purpose. And there is widespread anticipation that AI will allow further aspects of users' mental processes to be decoded. For example, Marcello Ienca, a neuroethicist at the Technical University of Munich in Germany, says that EEG can detect small voltage changes in the brain that occur within hundreds of milliseconds of a person perceiving a stimulus. Such signals could reveal how their attention and decision-making relate to that specific stimulus. And ethicists say that a big tech company could suddenly catapult the devices to widespread use. “There's a wild west when it comes to the regulatory standards,” he says. In 2018, Ienca and his colleagues found that most consumer BCIs don't use secure data-sharing channels or implement state-of-the-art privacy technologies2. That means most firms can use the information as they please, including selling it. But Ienca and Nita Farahany, an ethicist at Duke University in Durham, North Carolina, fear that such laws are insufficient because they focus on the raw data and not on the inferences that companies can make by combining neural information with parallel streams of digital data. Inferences about a person's mental health, say, or their political allegiances could still be sold to third parties and used to discriminate against or manipulate a person. “The data economy, in my view, is already quite privacy-violating and cognitive- liberty-violating,” Ienca says. Several key international bodies, including the United Nations cultural organization UNESCO and the Organisation for Economic Co-operation and Development, have issued guidelines on these issues. Furthermore, in September, three US senators introduced an act that would require the Federal Trade Commission to review how data from neurotechnology should be protected. While their development advances at pace, so far no implanted BCI has been approved for general clinical use. This relatively simple BCI allows users to select on-screen options by imagining moving their foot. Elon Musk's neurotech firm Neuralink in Fremont, California, has surgically implanted its more complex device in the motor cortices of at least 13 volunteers who are using it to play computer games, for example, and control robotic hands. Company representatives say that more than 10,000 people have joined waiting lists for its clinical trials. At least five more BCI companies have tested their devices in humans for the first time over the past two years, making short-term recordings (on timescales ranging from minutes to weeks) in people undergoing neurosurgical procedures. Researchers in the field say the first approvals are likely to be for devices in the motor cortex that restore independence to people who have severe paralysis — including BCIs that enable speech through synthetic voice technology. As for what's next, Farahany says that moving beyond the motor cortex is a widespread goal among BCI developers. Last year, Andersen's group published a proof-of-concept study5 in which internal dialogue was decoded from the parietal cortex of two participants, albeit with an extremely limited vocabulary. Some even became active when the player decided whether to stick with their current hand or take another card. Both Oxley and Matt Angle, chief executive of BCI company Paradromics, based in Austin, Texas, agree that BCIs in brain regions other than the motor cortex might one day help to diagnose and treat psychiatric conditions. BCIs could potentially track such symptoms in a person, deliver stimulation that adjusts neural activity and quantify how the brain responds to that stimulation or other interventions. Shanechi does not yet know whether the neural correlates of psychiatric symptoms will be trackable across many brain regions or whether they will require recording from specific brain areas. Such models, constructed by training AI algorithms on thousands of hours of neural data from numerous people, would in theory be generalizable across individuals' brains. Genser, J., Damianos, S. & Yuste, R. Safeguarding Brain Data: Assessing the Privacy Practices of Consumer Neurotechnology Companies (Neurorights Foundation, 2024). The rise of brain-reading technology: what you need to know Tiny robots swim through blood, deliver drugs — and then dissolve Why some researchers think we're on the cusp of a major longevity breakthrough Integrator dynamics in the cortico-basal ganglia loop for flexible motor timing Shared and language-specific phonological processing in the human temporal lobe If the AI bubble bursts, what will it mean for research? Semantic design of functional de novo genes from a genomic language model We are seeking a highly motivated Research Assistant to join our dynamic team at i-BRAIN. The rise of brain-reading technology: what you need to know An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-65788-8'>Position-specific ORF nucleoside-ribose modifications enabled by complete chemical synthesis enhance mRNA stability and translation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-19 10:29:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite the remarkable success of mRNA vaccines, improving the translational efficiency of mRNA therapeutics remains a critical challenge to their widespread clinical application. Here we systematically evaluate chemical modifications to improve the translational activity and stability of uncapped mRNA. We employ a primarily chemistry-based synthetic approach, which is crucial for the position-specific introduction of chemical modifications, enabling detailed structure-activity relationship studies, hitherto unattainable with conventional methods. A pivotal innovation herein is the introduction of 2´-F modification at the first nucleoside of the codon in the open reading frame, which significantly bolsters the stability of mRNA without compromising its translation. Additional modifications at the 5´-UTR and poly(A) tail with other types of nucleoside and phosphate analogs also exemplify the importance of terminal modifications for improved translation. Precise control of these modification patterns achieves higher peptide expression than conventional in vitro-transcribed mRNA. These findings offer a unique framework for designing effective mRNA-based therapeutics. Research on the development of messenger RNA (mRNA) therapeutics using mRNA as drug molecules has been greatly accelerated by the successful application of mRNA vaccines against SARS-CoV-21,2. mRNA therapeutics3,4 have a variety of applications, including vaccines against infectious diseases using viral and bacterial antigens5, cancer vaccines based on cancer cell-specific antigens6,7, and protein replacement therapy for diseases caused by genetic mutations8,9,10. Compared to protein replacement therapy using proteins as drug molecules, mRNA therapeutics offer advantages in terms of molecular design simplicity, predictability of molecular properties, and inherent eukaryotic post-translational modifications vital for protein function. Unlike conventional DNA-based gene therapy, on the other hand, mRNA therapeutics do not require transport to the nucleus for gene expression, have little risk of genome integration, and allow for transient expression of therapeutic effects, enabling better therapeutic control. The following issues are generally recognized for the future application of mRNA therapeutics as general-purpose medicines3,5: mRNA in vivo stability, insufficient translational activity, induction of undesired immune responses, and devising effective delivery methods to target tissues and cells. Among these challenges, introducing non-canonical chemical modifications has shown promise in enhancing mRNA stability and translational activity and reducing immune responses11,12,13. Improvement of mRNA function by introducing chemical modifications has been studied mainly for the 5´-cap, and various cap derivatives have been developed to improve stability by imparting decapping resistance14,15,16,17,18,19, as well as for tool-oriented applications20,21,22. Although less common, modifications to the poly(A) tail23,24,25 and region-specific thiophosphate modifications in the 5´-untranslated region (UTR)26 have been explored to enhance protein production. In addition, the use of non-canonical nucleobases27,28,29,30 can effectively mitigate undesired immune responses31,32,33,34,35 triggered by exogenous mRNA. It is well recognized that the use of these modified nucleobases plays a significant role in the success of mRNA vaccine development against SARS-CoV-2. These studies underscore the potential of chemical modifications and the use of non-canonical structures to address the challenges of mRNA therapeutics. However, conventional enzyme-based preparative methods for mRNA greatly limit the types and patterns of chemical modifications that can be introduced. Modification of the ribose ring would significantly improve biological stability, as exemplified by oligonucleotide therapeutics36,37,38,39 such as antisense oligonucleotides (ASOs) and siRNAs. However, there are two significant challenges in applying the same concept to mRNAs. One is that ribose modifications in ORFs, as known from examples such as 2´-O-methyl (2´-OMe)30,40,41, would significantly decrease translational activity. The second issue is the synthetic method used for chemically modified mRNA. mRNA is basically synthesized by enzymatic transcription, but even in transcription using engineered RNA polymerases, the types of sugar-modified nucleosides that can be introduced are very limited42,43,44,45,46. In addition, when preparing mRNA with a single composition, as in pharmaceutical applications, substitutions with modified nucleosides are inevitably made throughout for the specific nucleobase. In other words, by using conventional transcription-based synthetic methods, it is impossible to introduce position-specific chemical modifications to maximize both mRNA stability and translational activity. On the other hand, our group has also been conducting research to solve issues in mRNA therapeutics from different aspects, including nano-structure design of mRNAs such as circular mRNA47,48,49,50, introduction of chemical modifications such as specific phosphorothioate modification to 5´-UTR26, development of cap analogs for in vitro transcription (IVT) that enable purification of capped mRNAs with high-purity and translational activity51, and chemical capping reaction on RNA for complete chemical synthesis of mRNA52. Here, we show the possibility of introducing chemical modifications into the ribose scaffold of nucleosides, especially those in the open reading frame (ORF) regions, as a pioneering approach for developing effective mRNAs (Fig. Chemically modified mRNAs developed in this study are prepared by first synthesizing the corresponding RNA oligonucleotides using phosphoramidite chemistry and then concatenating these RNA fragments by enzymatic or chemical ligation reactions (Fig. 1B) to obtain full-length mRNAs. This method enables the introduction of many different types of chemical modifications in various patterns in the synthesis of RNA fragments and allows precise evaluation of their effects on the translational activity and stability of the mRNA. A Concept of the study. B Two ligation methods employed in this study. First, to screen for modification patterns, we designed a sequence of 91 nucleotides (nt) uncapped RNA that could be directly synthesized without ligation, and evaluated its translation activity in a cell-free translation system with HeLa lysate (Fig. The sequence consisted only of 5´-UTR (β-globin UTR)53 and ORF, and was designed to encode Flag-His6 peptide52 to perform a sandwich ELISA for the evaluation of translational activity. As modifications, 2´-fluoro (2´-F), 2´-OMe, 2´-O-methoxyethyl (2´-O-MOE), locked nucleic acid (LNA), and DNA etc., were selected. mRNA modified with 2´-OMe at both ends (NK002) showed 4-fold higher translational activity than unmodified RNA (NK001) (Fig. Other types of modifications, such as 2´-F, LNA, DNA, 2´-O-MOE, and phosphorothioate, were also examined for possible introduction into the mRNA terminus (Fig. LNA (NK011) modification resulted in a significant loss of translational activity (Fig. S1B), whereas DNA (NK012) and 2´-O-MOE (NK009) modifications maintained translational activity, including additional phosphorothioate modifications (NK014 and NK015), as well as for 2´-OMe (NK013) (Fig. Next, the introduction of the 2´-F modification into a specific position in the codon unit was tested; the modification was specifically introduced at the codon's first, second, or third nucleoside over the ORF (NK003, NK004, NK005) (Fig. Interestingly, while the translational activity of mRNA with the modification at all of the second or third nucleoside in the codon unit (2nd NC, 3rd NC) was suppressed by nearly 30–50 % level, the modification at the first nucleoside in the codon unit (1st NC) did not have strong deleterious effects on translational activity. This translational acceptability of the 2´-F modification of the 1st NC was also observed in another sequence (67 nt mRNA coding Flag-His6). In contrast, other types of sugar modifications showed a negative effect on translation (Fig. 2C) when all of the 1st NC were modified, translation was suppressed in the case of the 2´-OMe (NK006) and 2´-O-MOE (NK007) modifications, and a nearly 50% reduction was observed for the deoxyribose modification (NK008). A Design and sequence of sugar modifications of chemically synthesized mRNAs. B, C The concentration of encoded peptides from each mRNA obtained after 30 min of reaction of the HeLa cell lysate in the in vitro translation system (n = 3 biological replicates). Data are presented as mean ± standard error. The statistically significant differences between “NK001” and “NK002” (B, C) in two-tailed, unpaired Student's t-test are marked as follows. Statistically significant differences for each mRNA from “NK002” (B, C) in one-way ANOVA followed by Dunnett's test are marked as follows. Source data are provided as a Source Data file. Subsequently, screening for the optimum chemical modification patterns of the terminal region (6 nt at the 5´ terminus and 3 nt at the 3´ terminus) was performed with a longer mRNA (145 nt) encoding three repeats of FLAG peptide and His-Tag (Fig. The mRNAs were prepared by chemical or enzymatic ligation reactions between the 5´ side 80 nt and the 3´ side 65 nt RNA fragments that were prepared using an automated oligonucleotide synthesizer. The method of ligation (enzymatic or chemical) was determined based on the relative positions of the ligation point and chemically modified sites. For substrates without chemical modifications near the ligation point, enzymatic ligation was selected based on the reported tolerance of sugar-modified nucleoside analogs to ligation by RNA ligase 254. In cases where the region around the ligation point contains 2´-modified nucleosides, the chemical ligation method was selected with the 3´ end of the 5´side -RNA fragment on being 2´-F, 3´-phosphate, and the 5´ end of the 3´ side- RNA fragment being a hydroxyl group. The conditions for the chemical ligation method were slightly modified from those previously reported55,56 In the case of RNA prepared by chemical ligation, the target ligated RNA strand was confirmed by LC-MS analysis (Fig. 2´-F modification on the terminus increased peptide production (NK024) to some extent, while LNA and BNA-NC (N-Me) modification significantly restrained translation (NK026, NK027) (Fig. 2´-O-MOE modification showed a positive effect on the translation (NK025) with a similar level as 2´-OMe modification (NK023), and the positive effect was further improved by the additional phosphorothioate modification to the terminal region (NK028). A, C, E Design and sequence of sugars and backbone modifications of chemically synthesized mRNAs. B, D, F The concentration of encoded peptides from each mRNA obtained after 30 min of reaction with the HeLa cell lysate in vitro translation system (n = 3 biological replicates). Data are presented as the mean ± standard error. The statistically significant differences between “NK025” and “NK029” (D) in two-tailed, unpaired Student's t-test are marked as follows. Both standard errors of “NK034” and “NK035” (D) were 0 and described as “+”. The statistically significant differences for each mRNA from “NK022” (B) or “NK029” (D) in one-way ANOVA followed by Dunnett's test are marked as follows. Source data are provided as a Source Data file. Based on these findings, terminal modification of mRNA with polyadenine (poly(A)) tails was evaluated. To the 145 nt RNA with 2´-O-MOE modification at the terminus, 20 nt of poly(A) with various modification types was also prepared (Fig. The types of modifications tested on poly(A) were as follows: 2´-F modification for every 2 nt (NK030), cross modification with 2´-F and 2´-OMe (NK031), complete modification with 2´-OMe (NK032), and 2´-O-MOE (NK033). The terminal 3 nt of the modified poly(A) was set as 2´-O-MOE with a phosphorothioate linkage. Evaluation of the translational activity revealed that the addition of non-modified poly(A) (20 nt) increased the translated peptide amount, and the positive effect was further enhanced by these modifications on poly(A), with 2´-F modification for every 2 nt (NK030) being slightly better than the others (Fig. In contrast to the ORF cases, various modifications were accepted on poly(A) without loss of translational activity. To evaluate whether the modified poly(A) tail retained the intrinsic biological function, we measured the binding affinity of Poly(A)-Binding Protein (PABP) to chemically-synthesized poly(A) fragments based on the Surface Plasmon Resonance binding assay. The 5´-biotinylated poly(A) fragments were immobilized onto a streptavidin-coated sensor chip, and the response rate for PABP binding was measured (Fig. As expected, PABP exhibited strong binding to unmodified poly(A) RNA (NK065; KD = 4.4 ± 0.3 × 10−10 mol/L), while it did not bind to the randomized sequence (C5A5U5G5; NK067).57 Regarding chemically modified fragments, modified poly(A) showed a weaker binding to PABP than unmodified poly(A) (NK066; KD = 4.8 ± 0.2 × 10−9 mol/L); however, the binding was superior to nonspecific binding observed for C5A5U5G5 sequence with the same modification pattern (NK068; KD = 4.4 ± 0.1 × 10−8 mol/L). These results supported the binding of NK066 to PABP. A previous study reported that sugar modifications of poly(A) impair PABP binding.58 Nevertheless, the combined effect of sugar and PS modification has not been addressed, suggesting the potential for a positive effect from this combination. The biological function of chemically modified poly(A) tail was also evaluated by the subsequent translation competition assays. The translation from synthesized mRNA with modified poly(A) (NK041) was competitively inhibited by the addition of poly(A) RNA fragment, while it remained unchanged by the addition of poly(C) (Fig. To clarify the effect of chemical modification of mRNA at various positions, 5´-UTR, ORF, and poly(A), mRNA samples with different modification sites and patterns were prepared (Fig. 3E), and their translational activities were compared in HeLa cell extracts. The samples tested were as follows: RNA with no modification (NK034), modified only at the 5´-UTR (NK035), ORF (NK036), poly(A) (NK037), modified both at the 5´-UTR and poly(A) (NK038) and modified over all regions (NK039). As shown in Fig. 3F, each modification to either the 5´-UTR, ORF, or poly(A) increased translational activity, and a stronger increment of the translated peptide was observed by the modification of ORF and poly(A) than that for the 5´-UTR. Interestingly, the combination of the 5´-UTR and poly(A) modification (NK038) counteracted the positive effects of each modification, and the additional modification of the ORF enhanced translation (NK039). Additionally, LC-MS/MS analysis was performed for the extracted peptide after the in vitro translation in rabbit reticulocyte lysate, and both non-modified (NK034) and chemically modified (NK041) mRNA administration predominantly afforded the target peptide MS peak (Fig. S4), suggesting no negative effect on mRNA decoding due to chemical modification in the ORF. After identifying the candidates for the optimum modification patterns, four mRNA samples were designed and synthesized with various modification patterns for each region (Fig. All four samples contained ribose modification in the ORF except for the start codon, 2´-F modification at the 1st NC, and the level of chemical modification on both terminals was changed among these four samples, as shown in Fig. Evaluation of the translational activity of these modified mRNA revealed that, as the level of terminal modification increased, the amount of translated peptide slightly decreased (NK030 vs. NK031, NK040, NK041) (Fig. The time course of translation was also evaluated over 30 min (Fig. While mRNA without any modifications (NK034) showed maximum translation at 5 min after administration, the modified RNA (NK030, NK031, NK041) increasingly produced the peptide over 30 min, with moderately modified RNA (NK030) showing the highest translation level at every time point. A Design and sequence of sugars and backbone modifications of chemically synthesized mRNAs. B The concentration of encoded peptides from each mRNA obtained after 30 min of reaction with the HeLa cell lysate in vitro translation system (n = 3 biological replicates). C, D The time course of the concentration of translated peptides (C, n = 3 biological replicates) and remaining RNA (D, n = 4 biological replicates) obtained 30 min reaction of HeLa cell lysate in an in vitro translation system. E, F Time course of the concentration of encoded peptide from HeLa cells after lipofection using messengerMAX (E) and electroporation (F) of each mRNA (n = 3 biological replicates). G Time course of the remaining RNA after electroporation in HeLa cells (n = 4 biological replicates). Data are presented as the mean ± standard error. The statistically significant differences between “NT” and “37 °C, 30 min” (D) in two-tailed, unpaired Student's t-tests were marked as follows. Statistically significant differences for each mRNA from “NK034” (B, C, E–G) in one-way ANOVA followed by Dunnett's test are marked as follows. Source data are provided as a Source Data file. The stability of mRNA in HeLa lysates was evaluated using reverse transcription quantitative polymerase chain reaction (RT-qPCR) for the reaction mixture (Fig. While the unmodified RNA (NK034) showed fast clearance from the mixture, the modified mRNAs (NK030, NK031, and NK041) stably remained in the reaction mixture without degradation for 30 min at 37 oC. While the amount of remaining mRNA was nearly comparable for the three types of modified mRNAs, there was a significant difference in the amount of the translation product (Fig. These results suggest that moderate and high levels of chemical modification at the terminus improved the stability of the mRNA, but the latter negatively influenced its translational activity. The effect of introducing chemical modifications to each region of the mRNA on translation activity was also evaluated in a cell-based system (NK034 – NK039, Fig. Consistent with the results in the cell lysate (Fig. 3F), the introduction of modifications to the ORF led to a strong enhancement in translation activity (NK036 and NK039). Additional modification both in the 5´-UTR and poly(A) cooperatively augmented the translational activity (NK039), which was more obvious than the case with the cell lysate system (Fig. The 2'-F modification at the 1st NC of start codon did not affect the translation activity of the mRNA, demonstrating it could be incorporated throughout the ORF (NK041 vs. NK052, Fig. The time course of translation was evaluated using HeLa cells over an extended incubation period of up to 48 h. In the case of lipofection (Fig. 4E), mRNAs that were highly modified in the poly(A) moiety (NK031 and NK041) afforded the highest amount of translational product over 48 h compared to mRNAs with the other two types of modification (NK030 and NK040). Notably, the translation level of the unmodified mRNA (NK034) was significantly lower than that of the modified mRNAs, clearly reflecting the results of the stability evaluation (Fig. While the modification level in the 5´-UTR region had little effect on the translation level in the case of mRNAs with a higher level of modification in the poly(A) region (NK031 vs. NK041), a lower level of modification of the 5´-UTR resulted in better translation activity for mRNA with less modification of the poly(A) region (NK030 vs. NK040). In the case of electroporation (Fig. 4F), the trend was somewhat different from that observed in the lipofection cases; mRNA with high modification at both termini (NK041) showed the highest level of translation without decreasing the peptide amount over 24 h. Both these experiments suggested that modification of the poly(A) region was more critical for increasing the translation amount than that on the 5´-UTR region in the cellular system. The amount of mRNA remaining in HeLa cells after electroporation was quantified by RT-qPCR (Fig. 4F); the mRNA was more stable as the level of chemical modification increased. In addition, no cytotoxicity was observed for the transfection of both non-modified and chemically modified mRNA (Fig. S7, NK034, 030, 031, 040, 041) at the concentration range evaluated for translational activity. Various reasons can be considered for the different optimal modification patterns observed in HeLa cell lysates and cell lines. One possible explanation for this is the varying impact of the mRNA degradation system. Specifically, in the HeLa cell lysate system, where the effect of mRNA degradation was relatively minor, the introduction of extensive chemical modifications at both ends of the mRNA inhibited translation. In contrast, in the cell system, the significant influence of the mRNA degradation system suggests that resistance to degradation plays a major role in determining translational activity. A similar trend in the positive effect of the chemical modification was also observed in other cell lines, where mRNA was administrated to hAoSMCs (human aortic smooth muscle cells) by electroporation, and the time course of the translation was evaluated (Fig. In addition, the versatility of the design strategy was confirmed by other mRNA sequences, the ones encoding the same hybrid peptide, three repeats of FLAG and HisTag, but with different 5´-UTR sequences (Fig. S5B, C), and the others encoding a different peptide composed of FLAG and Epidermal Growth Factor (EGF) (Fig. For the FLAG-EGF construct, the time course of translation was also evaluated. By introducing chemical modifications into mRNA (NK051), it was found that the peak translation activity was around 12 h, and the half-life from the maximum translation level was more than 36 h (Fig. Considering a previous report that the stability of recombinant EGF in several buffer solutions is limited to a few hours,59 it could be suggested that mRNA, particularly chemically modified mRNA, has advantages over direct peptide administration. Nonetheless, further investigation is required to test whether chemically modified mRNA administration with established delivery methods such as LNP60,61,62 afforded more sustained pharmacological effects in vivo than direct administration of recombinant EGF.63 Finally, the stability and translation activity of chemically modified and canonical mRNA prepared by IVT (NK060) with co-transcriptional capping with CleanCap were compared (Fig. The capping ratio of the co-transcriptionally capped mRNA was 84% based on DNAzyme-mediated 5´-terminus cleavage reaction (Fig. No damage to a cap structure nor structural change during isolation by dPAGE was confirmed by LC-MS analysis of the isolated mRNA (Fig. Based on the above studies, chemically modified uncapped RNA (NK041) was designed to have intensive chemical modification at both termini and 2´-F modification at the 1st NC in ORF except for the start codon and was entirely prepared by chemical method; the 5´ side 80 nt RNA fragment with 3´ phosphate terminus was chemically ligated with the 3´-side 85 nt RNA fragment with a 5´ -hydroxy terminus. As additional control samples, m1Ψ-modified co-transcriptionally capped IVT RNAs with 20 nt (NK062) and 80 nt (NK063) poly(A) were also prepared. A Design and sequence of chemically synthesized mRNAs and in vitro-transcribed mRNAs. B Time course of concentration of remaining RNA after incubating with diluted mouse serum (n = 3 biological replicates). C, D Time course of concentration of encoded peptide from HeLa cells after lipofection using messengerMAX (n = 4 biological replicates) (C) and electroporation (D) of each mRNA (n = 4 biological replicates). The statistically significant differences for each mRNA from “NK041” (B–D) in one-way ANOVA followed by Dunnett's test were marked as follows. Source data are provided as a Source Data file. For stability evaluation, mRNA was incubated in 1.6% diluted mouse serum at 37 oC, and the amount of remaining mRNA was evaluated by RT-qPCR (Fig. 5B), where a calibration curve was created for each mRNA sample to offset the effects of modifications on reverse transcription (Fig. IVT mRNA (NK034) degraded rapidly, leaving less than 5 % of the sample after 90 min. Introducing m1Ψ and extending poly(A) length were both effective, as these modified mRNAs remained at approximately 6.0% (NK060) and 21% (NK063) at the same time point. In contrast, chemically modified RNA without a cap (NK041) showed higher stability; ~48% of RNA remained after 90 min of incubation. In terms of translational activity, while non-modified mRNAs (NK034 and 060) afforded a very low translation level, regardless of the presence or absence of a 5´ cap (Figs. 5C, D, S8A), m1Ψ-modified capped mRNAs with 20 nt (NK062) and 80 nt (NK063) poly(A) enhanced the translational activity, especially at earlier time points, with longer poly(A) effectively enhancing the activity. Chemically modified uncapped mRNA (NK041) stably expressed much higher levels of the peptide in HeLa cells over 24–48 h compared to other mRNA (Fig. Notably, m1Ψ-modified mRNA with longer poly(A) showed higher translational activity than chemically modified mRNA at the early time point, but its decay was faster than that of NK041. After 24 h, NK041 showed 6–16 fold higher translational activity than NK063, suggesting a strong stabilizing effect by chemical modification (Fig. The same trend in translational level was observed in the electroporation system (Fig. These results clearly demonstrate that appropriate chemical modifications enhance peptide production from mRNA in cellular systems. Immune responses (IL6, TNF, and IL1B) upon the administration of mRNA were also evaluated (Fig. Chemically synthesized mRNAs (NK034 and NK041), as well as m1Ψ-modified mRNAs (NK062 and 063), did not show significant immunogenicity, while non-modified IVT mRNA (NK060) showed a high levels of immune-related gene expression, especially at higher doses. Uncapped IVT mRNA (NK056-059) showed the same trend of immune responses (Fig. S9A, D-I), and these results were consistent with the dsRNA detection results (Fig. The effect of cap introduction on this short mRNA sequence was also evaluated using several control samples (NK056-NK063, Fig. S9A-C); In terms of translational activity, only capped and m1Ψ-modified mRNA (NK062 and NK063) showed a significant level of peptide expression 5 h after administration. Collectively, no immune response was observed with m1Ψ-modified mRNA, and the introduction of the cap surely enhanced translational activity. In other words, in this construct, if the influence of the immune response is eliminated, enhancement of translational activity by the introduction of the cap can be observed. Furthermore, the effect of cap introduction on chemically modified mRNA was evaluated (Fig. An improvement in translational activity by the introduction of modifications in capped mRNA (NK055 vs. NK053, 054) was confirmed. On the other hand, in comparisons among chemically modified mRNAs, uncapped mRNA (NK052) showed higher translational activity than capped mRNA (NK055), suggesting certain contribution of cap-independent translation mechanisms.64,65,66 Cap-dependency of translation activity on sugar-modified synthetic mRNA remains a topic for future investigation. In conclusion, we have systematically demonstrated the effectiveness of chemical modification of the sugar moiety of ribonucleosides in enhancing peptide production from uncapped mRNA. Augmented by the additional chemical modification on both termini, the chemically modified mRNA, even without a 5´ cap, yielded a greater amount of peptide over a longer duration than the IVT-derived canonical mRNA. Site-specific chemical modification for a high level of translation was enabled by chemical synthesis of the RNA fragments. Ligation reactions with these RNA fragments allowed for the preparation of chemically modified mRNA exceeding 200 nt in length. Building on the SAR insights from this study, we are pursuing the development of highly efficacious mRNA therapeutics. Owing to their high and long-lasting activity, chemically modified mRNA might provide novel therapeutic options that are difficult to achieve with IVT-derived mRNAs in the evolving area of mRNA therapeutics, such as cancer vaccines7, protein replacement, and tissue regeneration therapies. RNA fragments and splint DNAs for enzymatic or chemical ligation reactions were purchased from Gene Design or Hokkaido System Science, where they were synthesized using standard phosphoramidite methods. The oligonucleotides used in this study are listed in Tables S1–S6. The dPAGE analysis of the evaluated RNA samples is presented in Figs. The concentration of the oligonucleotide and ligated mRNA was measured by NanoDrop2000 (Thermo Fisher Scientific) using the extinction coefficient calculated by OligoAnalyzer (Integrated DNA Technologies, Inc.). The structures of the solid-phase synthesized oligonucleotides were determined by liquid chromatography-mass spectrometry (LC-MS) using an Agilent 6120 series single quadrupole LC-MS system (Agilent Technologies). Liquid chromatography was performed using an ACQUITY BEH C18 column (1.7 μm, 2.1 × 50 mm; Waters) with buffers A (8.6 mmol/L trimethylamine and 100 mmol/L hexafluoroisopropanol in water) and B (methanol). The column temperature was 60 °C. The gradient ran from 10% to 90% B over 9.0 min (for Tables S1–S3) or 18.0 min (for Fig. Water injections served as blanks. Mass spectrometry was performed in negative ion mode with a negative capillary voltage of 3000 V, full scan range m/z 500–2000. The acquired MS data were deconvoluted and reviewed using Agilent OpenLab CDS ChemStation Edition (Version C.01.09). All data were obtained from single measurements of each sample (n = 1). A mixture containing RNA fragments No.1 and 2 (each 10 nmol, final conc. of 50 μmol/L) (Table S2) and splint DNA (10 nmol, final conc. of 50 μmol/L) (Table S3) in T4 RNA Ligase 2 Reaction Buffer (New England BioLabs, NEB) was heated at 90 °C for 5 min and then gently cooled to room temperature. Then, T4 RNA Ligase 2 (10 units) and 60% PEG6000 (final conc. of 15%) were added and incubated at 37 °C for 16 h. The ligated RNAs were purified using preparative 5% denaturing PAGE and isolated using the crush-and-soak method to obtain the ligated products. The gel extraction was performed with 1× TE buffer and desalting with Amicon 10 K (Merck Millipore) and ethanol precipitation was performed and the mRNA was stored in 1 mmol/L sodium citrate (pH 6.5) buffer. For the preparation of some lots of NK023 and NK064, the following RP-HPLC purification was performed. After the enzymatic ligation, the crude samples were treated with Recombinant DNase I and then purified by HPLC with XBridge Oligonucleotide BEH C18 OBD Prep Column (130Å, 2.5 μm, 10 mm × 50 mm) and gradient buffer containing 0–50% MeCN, 100 mmol/L hexylammonium acetate in H2O. The yield of each ligated mRNA is shown in Table S4. A mixture containing RNA fragments Nos. 56 and 57 (each 10 nmol, final conc. of 50 μmol/L) (Table S2), splint DNA (10 nmol, final conc. of 100 μmol/L) (Table S3) in 100 mmol/L NaCl was heated at 90 °C for 5 min and gently cooled to room temperature. of 5 mmol/L) and N-Cyanoimidazole (final conc. of 5 mmol/L) in DMSO were added and incubated at 30 °C for 20 h. The ligated uncapped RNA was purified using a NAP-10 column and preparative 5% denaturing PAGE. The yield of each ligated mRNA is shown in Table S4. Generally, the ligation yield was moderate (approximately 40 %, Fig. S20); however, the final isolated yield remained low, mainly because of sample loss during the isolation process, which could be overcome by scaling up the reaction. 69 was prepared from starting fragment No. 68 based on a previously described chemical capping reaction.52 After lyophilizing the RNA solution, the following solvents and reagents were added at the final concentrations listed below, and the solution was incubated at 55 °C for 3 h: DMSO (10 µmol/L RNA), 7-methylguanosine 5´-diphosphate imidazolide (10 mmol/L), and N-methyl imidazole (1.3 mol/L). After the reaction, the mixture was diluted with MQ and desalted using Amicon 10 K, followed by isopropanol precipitation. The product was purified using reverse-phase HPLC (Column: YMC-Triart Bio C18, Eluent: A: 50 mmol/L TEAA, 5% MeCN, B: MeCN, Linear gradient 0–40 % B over 30 min). Nearly quantitative capping was confirmed by dPAGE analysis (Fig. LC-MS analysis of the isolated fragment was performed; calculated: 26877.6, observed: 26880.7. The condition was shown below; Column: ACQUITY UPLC BEH C18 Column, 130 Å, 1.7 µm, 2.1 mm × 50 mm; Column temp. ; Linear gradient of 5–30% over 20 min; Solvent A: MeOH; Solvent B: 8.6 mmol/L TEA/100 mmol/L HFIP solution; and Flow rate: 0.3 mL/min. The obtained fragment No. 69 was determined by LC-MS as described above. 69 and 70 were chemically ligated and then treated with Recombinant DNase I as described above. The reaction mixture was then purified by HPLC with a COSMOSIL RNA-SEC-1000 Packed Column (7.5 × 300 mm) and 20 mmol/L phosphate buffer (pH 7.4). Template DNA for IVT was prepared by PCR using pUC57amp/G-3×FLAG-6×His-DHFR as the template. The plasmid and PCR primer was purchased from IDT and Sigma-Aldrich, respectively. These sequences are listed in Table S5. To prepare 5′ capped mRNA by co-transcriptional capping, the following PCR primers were used: FW-AGG and RV-PA20 (for preparation of poly(A)20) or FW-AGG and RV-PA80 (for preparation of poly(A)80). The PCR mixture consisted of 0.25 μmol/L primers, 0.25 ng/μL pUC57amp/G-3×FLAG-6×His-DHFR vector, 1× PCR buffer of PrimeSTAR MAX DNA Polymerase (Toyobo). The reaction was treated with 100 U/μL DpnI and then analyzed by agarose gel electrophoresis, and gel-excised dsDNA was purified with the QIAquick PCR Purification Kit (QIAGEN). Transcription reactions were conducted with MEGAScript T7 Transcription Kit (Invitrogen) based on the manufacturer's protocol. A reaction mixture containing 1 ng/μL DNA (PCR product) and 8 mmol/L NTPs with or without 6 mmol/L CleanCap AG was incubated at 37 °C for 6 h. To the reaction mixture was added Turbo DNase I, which was further incubated at 37 °C for 15 min, and then purified using the Monarch RNA Cleanup Kit (NEB). The purified capped-RNA was then treated with 5 U/μL Antarctic Phosphatase Enzyme (NEB) and 1×Antarctic Phosphatase Buffer at 37 °C for 1 h. The obtained mRNA was analyzed by dPAGE, and gel-purified mRNA was extracted using buffer. mRNA was recovered by alcohol precipitation. To prepare 5′ capped mRNA by post-transcriptional enzymatic capping, FW-GGG and PA20 were used as PCR primers. PCR was performed with the KAPA HiFi HotStart ReadyMixPCR Kit (Roche) based on the manufacturer's protocol. Transcription reactions were conducted using the MEGAScript T7 Transcription Kit, 4 ng/μL DNA, and 9 mmol/L NTPs. After DNase I treatment and purification, the resultant RNA was capped with Vaccinia Capping System (NEB) and ScriptCap 2'-O-Methyltransferase Kit (CELLSCRIPT) based on the manufacturer's protocol. The reaction mixture containing 0.5 μg/μL RNA, 0.1 mmol/L SAM, 0.5 mmol/L GTP, 0.5 U/μL Vaccinia Capping Enzyme, 2.5 U/μL 2'-O-Methyltransferase, 2 U/μL RNase Inhibitor, and 1× Capping Buffer was incubated at 37 °C for 1 h and then purified with Monarch RNA Cleanup Kit. Purified capped-RNA was then treated with Antarctic Phosphatase and gel-purified. The other procedures were the same as those described above. The sequences of the prepared mRNAs are described in Table S6. Chemically synthesized uncapped mRNAs were preheated at 90 °C for 3 min and cooled to 4 °C before the translation assay and degradation evaluation. The in vitro translation reaction was performed using a 1-Step Human Coupled IVT Kit (Thermo Fisher Scientific) according to the manufacturer's protocol. The reaction mixture (10 μL) containing each concentration of mRNA and 0.8 U/μL Murine RNase Inhibitor was incubated at 37 °C. The reaction was stopped by incubation at 72 °C for 5 min. The in vitro translation reaction was performed using a rabbit reticulocyte lysate kit (Promega) according to the manufacturer's protocol. The reaction mixture (10 μL) containing 300 nmol/L mRNA, 10 μmol/L poly(A) fragments and 1.6 U/μL Murine RNase Inhibitor was incubated at 37 °C. After incubation, reaction mixture was diluted in 0.1 mol/L Carbonate buffer (pH 9.4) to stop the reaction. The vector for expressing human PABP, pCS-6xHis/TEV/hPABPC1[NM_002568.4], was constructed and packaged by VectorBuilder Inc. with vector IDs VB231222-2539wqp. The protein was expressed in E. coli BL21 (DE3) Star cells (Invitrogen) and cell pellets were lysed with B-PER (Thermo Fisher Scientific) containing 125 units/mL benzonase (Merck). Nucleic acids bound to PABP were digested by benzonase and further removed with poly(ethyleneimine) precipitation. The proteins were purified using cOmplete His-Tag Purification Resin (Roche), followed by a Superdex 200 increase GL 10/300 column (Cytiva). The binding affinity of human PABP to the poly(A) fragments was determined using Biacore T200 (GE Healthcare). 5'-biotinylated poly(A) fragments were purchased from Gene Design. The poly(A) fragments were captured on streptavidin-coated sensor chip (GE Healthcare) in a running buffer (10 mmol/L HEPES, 600 mmol/L NaCl, 3 mmol/L EDTA, 0.005% v/v surfactant P20 pH 7.4). Recombinant PABP was injected for 2 min and allowed to dissociate for 3 min. The flow-cell surface was regenerated with 0.1% SDS for 2 min to completely remove PABP bound to the poly(A) fragments. Kinetic parameters were determined using 1:1 binding model. The translational activity of each uncapped mRNA was evaluated in vitro using HeLa cells. In the lipofection experiment, cells were seeded and cultured at 1.0 × 104 cells/well in RPMI1640 medium containing 10% FBS in 96-well cell culture plates at 37 °C and 5% CO2. After overnight cultivation, 10 μL/well of transfection reagent containing each concentration of mRNA and 0.3% Lipofectamine MessengerMAX Transfection Reagent diluted with Opti-MEM were added to 40 μL/well of the cultured cells. After 5 h of transfection, the culture supernatant was removed, replaced with RPMI1640 medium containing 10% FBS, and the cells were cultivated. Cultured cells were then washed with ice-cold D-PBS(−) and lysed with 20 μL/well iScript RT-qPCR Sample Preparation Reagent (Bio-Rad) containing 2% Protease Inhibitor Cocktail (EDTA free) (Nacalai Tesque). For the FLAG-EGF mRNA experiment, 0.45% Lipofectamine MessengerMAX was used. Electroporation of mRNAs was performed using the Nucleofector 96-well Shuttle System (Lonza) and the SE Cell Line 96-well Nucleofector Kit (Lonza). Twenty microliters of reaction mixture containing each concentration of mRNA, 2.0 × 105 HeLa cells, Nucleofector Solution and Supplement 1 was pulsed with FF-150 condition, and then seeded and cultivated at 5.0 × 104 cells/well. Cell lysis was performed under the same conditions as the lipofection experiment. Electroporation of uncapped human primary aortic smooth muscle cells (hAoSMC) (Lonza) was performed using the Nucleofector 96-well Shuttle System and P1 Primary Cell 96-well Nucleofector Kit (Lonza). hAoSMC were pulsed with FF-130 condition. After electroporation, 2.0 × 104 cells/well were seeded and cultivated with Smooth Muscle Cell Growth Medium 2 (Lonza) and lysed as described above. The translational activity of each mRNA was evaluated as described in the section on assays using HeLa cells. FLAG-His tag peptides in the translation reaction were measured using sandwich ELISA. The following standard peptides were purchased from COSMO-BIO: 1×FLAG-His6: NH2-MDYKDDDDKGGHHHHHH-COOH and 3×FLAG-His6: NH2-MDYKDDDDKIIDYKDDDDKGGDYKDDDDKHHHHHH-COOH. Maxisorp 96-well plate (Sigma-Aldrich) was coated with 3 μg/mL of Anti-His-Tag Antibody (Proteintech) in 0.1 mol/L Carbonate buffer (pH 9.4) and then blocked with 3% BSA-TBST. The plate was washed with 1×TBST, and then each translation reaction and standard peptides in 3% BSA-TBST were added. After 1 h of incubation, a washed plate was added with 0.01 % Monoclonal ANTI-FLAG M2-Peroxidase (HRP) Ab produced in mice (Thermo Fisher Scientific) in 3% BSA-TBST. After washing, the plate was developed with 1-Step Ultra TMB-ELISA and then stopped with 0.5 mol/L sulfuric acid. 570 was measured using a spectrophotometer (Bio-Rad). To evaluate immune stimulation by each mRNA transfected into in vitro cultured cells, cDNA was prepared with the SuperPrep Cell Lysis & RT Kit for qPCR (Toyobo) from cultured cells 5 h after the transfection of mRNAs. The gene expression of IL6 (Hs00174131_m1), TNF (Hs00174128_m1), and IL1B (Hs01555410_m1) was evaluated by RT-qPCR using TaqMan probes and TaqMan Gene Expression Master mix (Applied Biosystems) and normalized by GAPDH (Hs02786624_g1) expression. The viability of HeLa cells at 24 h after lipofection with 30 nmol/L chemically modified mRNA was evaluated with the CellTiter-Glo Luminescent Cell Viability Assay (Promega) based on the manufacturer's protocol. The sequences and structures of the primers (Sigma-Aldrich) and TaqMan probes (Applied Biosystems) are described below. Reverse transcription of mRNA was performed using the iScript Select cDNA Synthesis Kit (Bio-Rad) according to the manufacturer's protocol. The reaction mixture contained 0.2 μmol/L RT primer, and the translation reaction was diluted with RNase/DNase-free water containing 0.2 U/μL Recombinant RNase Inhibitor. RT-qPCR of the obtained cDNAs was performed with Taqman qPCR Master Mix (Applied Biosystems) and QuantStudio 12 K Flex Real-Time PCR System (Applied Biosystems), as described below. Twenty microliters of reaction mixture containing 4 μL of cDNA, 1.65 μmol/L FW primer, 1.40 μmol/L RV primer, 0.10 μmol/L FAM-MGB probe and 1×reaction master mix were subjected to the thermal cycling reaction as follows: 50 °C for 2 min, 95 °C for 10 min, (95 °C, 15 s, 60 °C, 1 min) × 50 cycles. The stability of each mRNA in mouse serum was measured as described below. Ten microliters of reaction mixture containing 1.6% mouse serum and 1 μmol/L of each mRNA in THE RNA Storage Solution (Ambion) was incubated at 37 °C for 0, 15, 30, and 60 min, and then 1.2 U/μL of Recombinant RNase inhibitor was added to stop the degradation. The remaining mRNAs in the reaction mixture were measured using RT-qPCR, as described in the section of “Evaluation of mRNA stability in vitro”. dsRNA content in each mRNA sample was detected with a Double-stranded RNA (dsRNA) ELISA kit (K1 based) (Exalpha). Poly(I:C) HMW (InvivoGen) was used as the standard dsRNA. The general procedure was based on the manufacturer's protocol. Chemically synthesized mRNAs were translated using a rabbit reticulocyte in vitro translation system (Promega) at 37 °C for 30 min. The translated peptides were purified with an anti-FLAG antibody (Millipore). LC-MS analysis of the peptides and synthetic standards (NH2-MDYKDDDDKIIDYKDDDDKGGDYKDDDDKHHHHHH-COOH) was performed by Medical ProteoScope (Kanagawa, Japan). Peptides were resuspended in [TFA]:[ACN]:[H2O] = 0.1:2:98 (%vol). Analysis was conducted using an Ultimate 3000 RSLCnano LC system coupled to a Q Exactive Orbitrap MS (Thermo Fisher Scientific). Chromatography used a Nano HPLC Capillary Column (3.0 µm, 75 µm × 15 cm) with buffers A (0.1% formic acid) and B (0.1% formic acid, 90% acetonitrile). The gradient ran from 5% to 95% B over 50 min at 350 nL/min. Water injections served as blanks. Mass spectrometry was performed in positive ion mode with a capillary temperature of 250 °C, full scan range m/z 310–1500, and resolution 70,000 (MS) and 17,500 (MS/MS). Lock mass correction used m/z 391.28429 and 445.12003. Data were reviewed with Xcalibur Qual Browser (v4.2.47). MS/MS data were analyzed with Mascot (Matrix Science) using standard settings; however, MS/MS spectra are not shown. Peptide identifications were confirmed by high-resolution LC-MS and comparison with synthetic standards. Technical replicates ensured reproducibility. The solution containing RNA (0.5 μmol/L, as final conc. ), DNAzyme (1.0 μmol/L), Tris-HCl (pH 8.0, 5 mmol/L), MgCl2 (5 mmol/L) in MQ was prepared and the mixture was incubated at 37 °C for 1 h. DNAzyme sequence: CTGCAAAAAGAAGGCTAGCTACAACGAAAGTAGCTTG (Eurofins Genomics). After the reaction, iPrOH precipitation was performed and the sample was subjected to dPAGE analysis. No statistical method was used to determine the sample size. No data were excluded from the analysis. The experiments were not randomized. Statistical analyses were performed using GraphPad Prism9 ver. The methods used are described in each figure. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated or analyzed in this study are included in this published article, in the supplementary information file, or in the source data files, and are available from the corresponding author(s) upon request. Source data are provided with this paper. Verbeke, R., Lentacker, I., De Smedt, S. C. & Dewitte, H. The dawn of mRNA vaccines: the COVID-19 case. Fang, E. et al. Advances in COVID-19 mRNA vaccine development. Rohner, E., Yang, R., Foo, K. S., Goedel, A. & Chien, K. R. Unlocking the promise of mRNA therapeutics. Şahin, U., Karikó, K. & Türeci, Ö mRNA-based therapeutics—developing a new class of drugs. Chaudhary, N., Weissman, D. & Whitehead, K. A. mRNA vaccines for infectious diseases: principles, delivery and clinical translation. Liu, C. et al. mRNA-based cancer therapeutics. & Pichon, C. Neoantigen vaccine nanoformulations based on chemically synthesized minimal mRNA (CmRNA): small molecules, big impact. Papukashvili, D. et al. Self-amplifying RNA approach for protein replacement therapy. Qin, S. et al. mRNA-based therapeutics: powerful and versatile tools to combat diseases. Vavilis, T. et al. mRNA in the context of protein replacement therapy. Design, assembly, production, and transfection of synthetic modified mRNA. Elkhalifa, D., Rayan, M., Negmeldin, A. T., Elhissi, A. M. A. Chemically modified mRNA beyond COVID-19: potential preventive and therapeutic applications for targeting chronic diseases. De La Vega, R. E. et al. Efficient healing of large osseous segmental defects using optimized chemically modified messenger RNA encoding BMP-2. Muttach, F., Muthmann, N. & Rentmeister, A. & Kore, A. R. Recent advances in modified cap analogs: synthesis, biochemical properties, and mRNA based vaccines. Kore, A. R., Shanmugasundaram, M., Charles, I., Vlassov, A. V. & Barta, T. J. Locked nucleic acid (LNA)-modified dinucleotide mRNA cap analogue: synthesis, enzymatic incorporation, and utilization. A. et al. 5'-phosphorothiolate dinucleotide cap analogues: reagents for messenger RNA modification and potent small-molecular inhibitors of decapping enzymes. Structural insights into the interaction of clinically relevant phosphorothioate mRNA cap analogs with translation initiation factor 4E reveal stabilization via electrostatic thio-effect. Towards superior mRNA caps accessible by click chemistry: synthesis and translational properties of triazole-bearing oligonucleotide cap analogs. Ogasawara, S. Duration control of protein expression in vivo by light-mediated reversible activation of translation. & Rentmeister, A. Chemo-enzymatic modification of the 5′ cap to study mRNAs. Klöcker, N. et al. Photocaged 5′ cap analogues for optical control of mRNA translation in cells. Strzelecka, D. et al. Phosphodiester modifications in mRNA poly(A) tail prevent deadenylation without compromising protein expression. Chemically modified mocRNAs for highly efficient protein expression in mammalian cells. Li, C. Y. et al. Cytidine-containing tails robustly enhance and prolong protein production of synthetic mRNA in cell and in vivo. Kawaguchi, D. et al. Phosphorothioate modification of mRNA accelerates the rate of translation initiation to provide more efficient protein synthesis. The pivotal role of chemical modifications in mRNA therapeutics. Effects of mRNA modifications on translation: an overview. & Yi, C. Chemical modifications to RNA: a new layer of gene expression regulation. Effects of chemically modified messenger RNA on protein expression. Anderson, B. R. et al. Incorporation of pseudouridine into mRNA enhances translation by diminishing PKR activation. Freund, I., Eigenbrod, T., Helm, M. & Dalpke, A. H. RNA modifications modulate activation of innate toll-like receptors. Karikó, K., Buckstein, M., Ni, H. & Weissman, D. Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. Karikó, K. et al. Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. Karikó, K. & Weissman, D. Naturally occurring nucleoside modifications suppress the immunostimulatory activity of RNA: implication for therapeutic RNA development. Glazier, D. A. et al. Chemical synthesis and biological application of modified oligonucleotides. McKenzie, L. K., El-Khoury, R., Thorpe, J. D., Damha, M. J. & Hollenstein, M. Recent progress in non-native nucleic acid modifications. Egli, M. & Manoharan, M. Chemistry, structure and function of approved oligonucleotide therapeutics. & Seth, P. P. The medicinal chemistry of therapeutic oligonucleotides. Aurup, H., Siebert, A., Benseler, F., Williams, D. & Eckstein, F. Translation of 2′-modified mRNA in vitro and in vivo. Choi, J. et al. 2′-O-methylation in mRNA disrupts tRNA decoding during translation elongation. Padilla, R. & Sousa, R. Efficient synthesis of nucleic acids heavily modified with non-canonical ribose 2'-groups using a mutantT7 RNA polymerase (RNAP). Transcription yield of fully 2′-modified RNA can be increased by the addition of thermostabilizing mutations to T7 RNA polymerase mutants. Houlihan, G., Arangundy-Franklin, S. & Holliger, P. Engineering and application of polymerases for synthetic genetics. Ohashi, S., Hashiya, F. & Abe, H. Variety of nucleotide polymerase mutants aiming to synthesize modified RNA. Wang, G. et al. Thermophilic nucleic acid polymerases and their application in xenobiology. Abe, N. et al. Rolling circle amplification in a prokaryotic translation system using small circular RNA. Abe, N. et al. Rolling circle translation of circular RNA in living human cells. & Abe, H. Preparation of circular RNA in vitro. Nakamoto, K. & Abe, H. Chemical synthesis of circular RNAs with phosphoramidate linkages for rolling-circle translation. Inagaki, M. et al. Cap analogs with a hydrophobic photocleavable tag enable facile purification of fully capped mRNA with various cap structures. Complete chemical synthesis of minimal messenger RNA by efficient chemical capping reaction. Kozak, M. Features in the 5′ non-coding sequences of rabbit α and β-globin mRNAs that affect translational efficiency. Nandakumar, J. et al. How an RNA Ligase Discriminates RNA versus DNA Damage. Chen, H. et al. Template-directed chemical ligation to obtain 3′-3′ and 5′-5′ phosphodiester DNA linkages. Bowler, F. R. et al. Prebiotically plausible oligoribonucleotide ligation facilitated by chemoselective acetylation. Khanam, T., Sondekoppa, M. R., Avak, K., Nahum, S. & Brosius, J. Poly(A)-binding protein binds to a-rich sequences via RNA-binding domains 1+2 and 3+4. Perzanowska, O., Smietanski, M., Jemielity, J. Chemically modified Poly(A) analogs targeting PABP: structure activity relationship and translation inhibitory properties. Yang, C.-H., Huang, Y.-B., Wu, P.-C. & Tsai, Y.-H. The evaluation of stability of recombinant human epidermal growth factor in burn-injured pigs. Hou, X., Zaks, T., Langer, R. & Dong, Y. Lipid nanoparticles for mRNA delivery. Jeong, M., Lee, Y., Park, J., Jung, H. & Lee, H. Lipid nanoparticles (LNPs) for in vivo RNA delivery and their breakthrough technology for future applications. & Xia, T. Recent advances in site-specific lipid nanoparticles for mRNA delivery. Kim, H. et al. Hyaluronate—epidermal growth factor conjugate for skin wound healing and regeneration. Meyer, K. D. et al. 5′ UTR m6A promotes cap-independent translation. Shatsky, I. N., Terenin, I. M., Smirnova, V. V. & Andreev, D. E. Cap-independent translation: What's in a name?. Trainor, B. M., Ghosh, A., Pestov, D. G., Hellen, C. U. T. & Shcherbik, N. A translation enhancer element from black beetle virus engages yeast eIF4G1 to drive cap-independent translation initiation. This work was supported by the Japan Society for the Promotion of Science (JSPS), International Leading Research (JP22K21346 to Y.K. ), and Japan Agency for Medical Research and Development (AMED) (JP22gm0010008, JP22fk0310506 to HA, JP23fk0210133 to Y.K.). This work was funded by Kyowa Kirin Co., Ltd. These authors contributed equally: Hiroto Iwai, Yasuaki Kimura. Hiroto Iwai, Masakazu Honma, Atsushi Hashimoto, Keiichi Motosawa, Takayuki Atago, Kana Asano, Keiko Kobayashi, Hiroki Yamada, Keiko Hiraishi, Seiji Saito & Junichiro Yamamoto Department of Chemistry, Graduate School of Science, Nagoya University, Nagoya, Japan Yasuaki Kimura, Kosuke Nakamoto, Naoko Abe, Ryoko Ogisu & Hiroshi Abe Research Center for Materials Science, Nagoya University, Nagoya, Japan Fumitaka Hashiya & Hiroshi Abe Institute for Glyco-core Research (iGCORE), Nagoya University, Nagoya, Aichi, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized and designed the study. designed and performed the biochemical experiments, including cell experiments. Correspondence to Yasuaki Kimura, Junichiro Yamamoto or Hiroshi Abe. is a cofounder of Crafton Biotechnology. The company focuses on the development of mRNA therapeutics. are employees of Kyowa Kirin Co. Ltd. International patents application covering part of this work has been filed by Nagoya University and Kyowa Kirin Co., Ltd. The remaining authors declare no competing interests. Nature Communications thanks anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Iwai, H., Kimura, Y., Honma, M. et al. Position-specific ORF nucleoside-ribose modifications enabled by complete chemical synthesis enhance mRNA stability and translation. Version of record: 19 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            