
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-13</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250513112306.htm'>Growth before photosynthesis: How trees regulate their water balance</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 17:16:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A study by the University of Basel shows how trees react to drought -- and revises previous perceptions. Plants have small pores on the underside of their leaves, known as stomata. When the sun rises, these pores open and the plants absorb carbon dioxide (CO2) from the atmosphere, which they need, in addition to sunlight and water, for photosynthesis. At the same time, water evaporates through the open stomata; for a tree, this may be several hundred liters per day. When water is scarce, plants can close their stomata and thus prevent it from evaporating too much water. The fact that plants have this protective mechanism at their disposal is nothing new. Water loss is therefore the price a plant pays for photosynthesis. "When it comes to plants, researchers have traditionally focused on photosynthesis. So it was previously assumed that trees treated this process as a priority and therefore kept the stomata open for as long as possible to absorb CO2, only closing them when there was no other option," explains study leader Professor Ansgar Kahmen. This inhibits tree growth even in intermediately dry conditions. This results in air bubbles, known as embolisms. "When this happens, irreparable damage occurs, the water transport system collapses and the plant eventually dies," says Ansgar Kahmen. Water supply in the tree is key It used to be assumed that, in order to maintain photosynthesis for as long as possible, trees would close their stomata only shortly before the onset of these embolisms. The new study now shows that the stomata remain closed at an earlier point in time, namely when water absorption at night has become difficult. "For the first time, we were able to show that a tree does not even open its stomata in the morning if it cannot absorb enough water overnight," says Kahmen. This means that the tree forgoes photosynthesis in favor of growth. According to Kahmen, this prioritization makes sense: If the plant stops growing due to a shortage of water, then, no matter how much photosynthesis it carries out, it will not be able to use the resulting products. The findings could also influence calculations relating to carbon sequestration by forests. "Climate models that assume a certain growth in carbon storage volume would therefore have to be adapted," says lead author Richard L. Peters, a former postdoc at the University of Basel and now professor at the Technical University of Munich. Particularly in the context of climate change, which is leading to warmer and, above all, drier summers in countries including Switzerland, carbon uptake could change more dramatically than previously assumed. "What is remarkable is that our early stomatal closure observations apply to all tree species, whether deciduous or coniferous. How well a tree species copes with drought therefore cannot be solely determined by the process of stomata closure" says Peters. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250513112158.htm'>Combinations of chronic illnesses could double risk of depression</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 16:57:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers found that some combinations of illnesses -- particularly cardiometabolic ones like diabetes and heart disease -- could more than double the likelihood of a future depression diagnosis. With multimorbidity -- when patients live with two or more chronic conditions -- continuing to put pressure on an already stretched healthcare system, experts say the findings highlight the need for integrated care models that address both mental and physical health. Researchers from the University of Edinburgh used data from more than 142,000 people in the UK Biobank study to examine how physical illnesses interact to influence the risk of depression -- a condition that often goes underdiagnosed in people managing long-term physical diseases. Participants were aged 37-73 years and had at least one chronic physical condition but no history of depression. Scientists used statistical clustering techniques to group individuals by their physical illness profiles and tracked how these clusters related to later diagnoses of depression. This group had no single dominant illness, but rather a complex mix of issues. People with both heart disease and diabetes were also found to be at high risk, as were those with chronic lung conditions like asthma or COPD -- chronic obstructive pulmonary disease. While the biological burden of illness may play a role, researchers say social and systemic factors could also help explain why physical multimorbidity leads to worse mental health outcomes. Lauren DeLong, lead author and PhD student at the University of Edinburgh's School of Informatics, said: "We saw clear associations between physical health conditions and the development of depression, but this study is only the beginning. Bruce Guthrie, Professor of General Practice at the University of Edinburgh's Advanced Care Research Centre, said: "Healthcare often treats physical and mental health as completely different things, but this study shows that we need to get better at anticipating and managing depression in people with physical illness." NIHR's research in this area is helping to paint a full picture of what patients are dealing with, rather than just focusing on one health condition at a time." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64738853/4-hours-of-sleep-mutation/'>A Gene Mutation Lets You Function Perfectly on 4 Hours of Sleep. It Might Be in Your DNA.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. But according to a new study, some people have a genetic mutation that makes them feel perfectly alive after sleeping only half as much. While the human body typically takes about 8 or 9 hours to do this, having what's called the “short sleep” mutation makes this possible within just 3 to 6 hours. Neuroscientist Ying-Hui Fu, who researches gene mutations related to sleep, has now discovered another mutation that contributes to natural short sleep (NSS), adding to the five already known mutations that she and her team previously discovered. “Our bodies continue to work when we go to bed, detoxifying themselves and repairing damage,” Fu said in a recent press release. “These people, all these functions our bodies are doing while we are sleeping, they can just perform at a higher level than we can.” Fu began investigating genetic reasons for needing so little sleep after she and her team were approached by a mother and daughter who felt refreshed after sleeping six hours or less. Analyzing their genome led to finding a rare mutation in the gene DEC2, which regulates circadian rhythm by binding to the gene MyoD1, which in turn switches on expression of the neuropeptide known as orexin. The DEC2 mutation results in increased levels of orexin, which explains why short sleepers can wake up after only a few hours and have no issue with feeling as if they are about to pass out during the day. In contrast, narcolepsy (characterized by excessive sleepiness during the day and disrupted sleep at night) is caused by an orexin deficit. SIK3 has a part in regulating both the need for sleep and the amount of NREM (non-rapid-eye-movement) sleep we get each night. This gene is active in synapses—spaces between neurons where messages are sent from one neuron to another—and is expressed most in the cerebellum and adrenal glands. When Fu genetically modified mice (which usually need about 12 hours of sleep) to have the SIK3 mutation, she found that they needed half an hour less of sleep to function. Another SIK3 mutation was previously found to trigger hypersomnia, or the need for more sleep. This study found that one amino acid in this gene, known as S551, is especially important when it comes to sleep regulation. Future studies might be able to use knowledge about such genetic mutations to help with sleep conditions like narcolepsy or insomnia. “[They] highlight the broader implications of kinase activity in sleep regulation across species, and provide further support for potential therapeutic strategies to enhance sleep efficiency.” Someday, we may never feel undead in the morning again. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Every Human Body Gives Off an Invisible Glow</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41576-025-00841-2'>Predicting gene expression from DNA sequence using deep learning models</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 12:16:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Transcription of genes is regulated by DNA elements such as promoters and enhancers, the activity of which are in turn controlled by many transcription factors. Owing to the highly complex combinatorial logic involved, it has been difficult to construct computational models that predict gene activity from DNA sequence. Recent advances in deep learning techniques applied to data from epigenome mapping and high-throughput reporter assays have made substantial progress towards addressing this complexity. Such models can capture the regulatory grammar with remarkable accuracy and show great promise in predicting the effects of non-coding variants, uncovering detailed molecular mechanisms of gene regulation and designing synthetic regulatory elements for biotechnology. Here, we discuss the principles of these approaches, the types of training data sets that are available and the strengths and limitations of different approaches. You have full access to this article via your institution. Gene expression patterns are predominantly driven by regulatory sequences that surround the genes and control their transcriptional activity. These regulatory elements are themselves bound by a diverse array of transcription factors (TFs), each recognizing specific short DNA sequence motifs. In many cases, the precise positioning of these motifs relative to each other is critical. Because most genes are regulated by multiple regulatory elements and TFs acting in combination, the complexity of the regulatory network is daunting. Yet, the importance of accurate predictive models of gene expression cannot be understated. In the fields of basic biology, gene regulation controls all aspects of development and cell signalling. In biotechnology, enhanced control of gene expression can improve the quality and yield of products and crops. In the clinic, better understanding of the impact of non-coding sequence variants on gene expression can lead to new diagnostics and treatments. Traditional machine-learning approaches have been widely used to model gene expression1, as have algorithms that incorporate 3D folding of the genome2,3. However, the predictive power of these models has remained limited. Recent advances in deep learning (DL) offer exciting opportunities to overcome this hurdle and to revolutionize the study of the regulatory genome. In particular, the so-called sequence-to-expression (S2E) models, which enable predictions of gene expression levels from DNA sequence alone, greatly aid in uncovering the sequence grammar of gene regulation. A key advantage of S2E models over previous machine-learning methods is their ability to extrapolate to previously unseen sequences, enabling them to predict the effect of sequence variants or to generate entirely synthetic regulatory elements not present in the genome. Here, we provide an overview of the main DL models that have been recently developed and applied to build S2E models. Compared with recent broad reviews4,5, we focus mainly on S2E models, with an emphasis on transcriptional regulation; modelling of other aspects of RNA metabolism such as splicing and decay is reviewed elsewhere (for example, refs. We discuss the basic principles of DL modelling, the diversity of data types that are used to train gene expression models, how the models may be validated and the strengths and limitations of each approach. Furthermore, we illustrate that DL-based S2E models are no longer ‘black boxes' but can be queried to uncover the sequence grammar that underlies gene regulation. Finally, we discuss the opportunities that S2E models offer in the fields of genetics and synthetic biology. Traditional machine-learning methods typically rely on pre-extracted features of DNA sequences (such as k-mer counts8), which lack crucial positional information (such as the distance between TF binding sites (TFBs)). By contrast, DL uses multiple layers of operations (‘deep' layers) to process long genome sequences directly. This end-to-end approach enables DL models to capture complex, potentially hierarchical and nonlinear patterns in the genome sequence data, leading to its breakthrough capabilities in S2E modelling9. Using only DNA sequences as input, S2E models can predict one or more transcriptional activity measurements associated with the provided sequence. This can include direct measurements of RNA levels (for example, predicting RNA sequencing (RNA-seq) or cap analysis of gene expression sequencing data) or proxies of gene expression such as epigenome mapping data (for example, predicting chromatin immunoprecipitation followed by sequencing (ChIP–seq) or assay for transposase-accessible chromatin with sequencing (ATAC-seq) data). However, for the model to make accurate predictions, it must first undergo training. During training, the model is provided with both the DNA sequences and their corresponding experimentally determined activity measurements. The model's parameters are adjusted to minimize the difference between its predictions and the experimental measurements (Box 1). This modelling involves the fitting of thousands or even millions of parameters, and very large input data sets are required to train the models9. It should be noted that current S2E models have a fundamental limitation: there currently is no evidence to suggest that they can make reliable predictions beyond the cell types and conditions for which training data were provided. For example, they cannot predict gene expression in a particular cell type treated with a certain hormone unless the training data were generated from the same cell type treated with this hormone. Training a deep learning model involves optimizing parameters (or weights) to predict outputs that closely match the target data. This optimization is guided by a loss function, which quantifies the difference between the model's predictions and the experimentally measured data. A common challenge in model training is the risk of overfitting, whereby the model memorizes the training data rather than learning the predictive motifs and patterns, resulting in poor performance on unseen data. To avoid overfitting, the original data set is typically divided into at least three folds with non-overlapping sequences: training, validation and test sets. After training, the validation set is used to evaluate the performance of the model and detect possible overfitting (that is, when the model performs better on the training data than on the validation data). This validation process avoids overfitting or underfitting (that is, poor performance on both data sets) by selecting appropriate hyperparameters, such as the model architecture or the model size (that is, the number of parameters). Once the parameters and hyperparameters of the model are set, the test set provides an unbiased estimate of the performance of the final model on unseen data. In practice, especially when training data are limited, cross-validation is often used. In this case, the training and validation sets are merged and split into k folds (see the figure, represented as four folds). This approach involves training k models, each time using a different fold as the validation set while the remaining fold serves as the training set (not shown in the figure). Cross-validation provides a more reliable estimate of the performance of the model and improves the stability of predictions and interpretations when all the models trained in the different folds are used simultaneously (ensemble of models)21,27,58,138. By contrast, training multiple models can be challenging for large models that require substantial computational resources and time. When creating data folds, it is important to ensure that sequences in different folds do not overlap the same genomic regions, which is referred to as ‘data leakage', as this could artificially inflate the perceived performance of the model (that is, similar performance on training and validation data), whereas in reality, it is overfitting. In sequence-to-expression models, the common practice is to group all sequences from the same chromosomes into the same fold to prevent any potential data leakage among training, validation or test folds. Other models take into account homology, whereby homologous sequences are used in the same fold18,109. However, some researchers advocate the use of random sequences as an alternative, as the extensive homology within the genome may prevent a proper estimate of overfitting76. To achieve accurate predictions, the design of the model's architecture is crucial. The architecture refers to the structure of the layered network, including the types of operations performed in each layer, the number of layers and how they are connected. In DL, the architecture is typically chosen based on prior knowledge of the data structure and the patterns that should be recognized. S2E models should identify patterns in the DNA sequence corresponding to TFBSs, DNA structural flexibility and the interaction between these patterns. Given the sequential nature of DNA sequences, the most common architectures are convolutional neural networks (CNNs)10 and transformers11. Other architectures have also been explored; for a detailed overview, see ref. A neural network architecture can be regarded as a series of arithmetic operations applied to its input. For this reason, the DNA sequence must be represented in a numerical format. Most commonly, the so-called one-hot encoding is used in which each DNA base is converted to a unique binary vector: for example A = [1, 0, 0, 0], C = [0, 1, 0, 0], G = [0, 0, 1, 0] and T = [0, 0, 0, 1] (Fig. a, A convolutional neural network architecture consists of a stack of convolutional layers: initial layers learn simple patterns (such as transcription factor (TF) motifs), whereas subsequent or deeper layers learn more complex patterns. The DNA sequence is first transformed into a one-hot encoded matrix. In the first convolution layer, a dot product is performed between a subregion of the sequence (highlighted in yellow) and a kernel. The kernel acts as a motif detector. The resulting scores are stored in a vector: higher values indicate stronger matches between the motif and the subsequence at a given position. Multiple kernels can be applied in a single layer, each learning different (partial) motifs. Several convolutional layers can be stacked, ultimately combining their output with a dense layer to produce the final output. The number of convolutional layers, the size of the kernels or the number of kernels applied in each layer are hyperparameters that can be tuned. However, some of these hyperparameters align with known biological priors (for example, the size of kernels tends to be in the length range of TF binding sites). b, Sequence-to-expression models that use a transformer architecture usually start by using convolutional layers. The convolutional layers initially learn local patterns, compressing the sequence, and the self-attention layers learn more distal patterns. The attention mechanism can be thought of as a similarity matrix, whereby each element computes how relevant it is for the other elements. In this example, the interaction of the promoter and its closest enhancers is highly important for the prediction, whereas the furthest enhancer does not interact with the promoter owing to the presence of an insulator. CNNs have a hierarchical architecture that is well suited to detect and combine features at multiple hierarchical scales. For the S2E challenge, this is important because the regulatory grammar manifests at different spatial resolutions: the initial layers capture local and relatively simple features, such as motifs representing TFBSs, whereas the deeper layers identify more complex patterns such as interactions between potentially more distal TFBSs. This layer performs a dot product between a kernel (or filter) and a subset of the sequence matching the kernel size (Fig. The matrix multiplication is executed in a sliding window along the length of the input sequence, producing a vector where each entry represents the similarity between positions in the raw sequence and the applied kernel. The convolutional kernels, which are a result of fitting the model to the data, can be conceptualized as position weight matrices that scan the input sequence for motif matches (for extensive review, see ref. After the convolution step, an activation function is applied to the matrix multiplication output. Typically, this is a nonlinear function, allowing the network to fit to complex nonlinear data and uncover complex patterns. The number and size of kernels are hyperparameters that can be optimized during the hyperparameter tuning process. Several convolution layers are repeated until the last fully connected layer produces the output. The output can vary based on the task: it might be a single value representing the overall prediction for the entire input sequence, or a vector representing higher resolutions, potentially down to individual nucleotides. Nearly a decade ago, pioneering studies demonstrated that CNNs trained on epigenomic data outperformed the then gold-standard algorithms based on shallow machine-learning algorithms14,15,16. Although these models did not directly predict gene expression, they successfully predicted TF binding and histone marks based solely on genomic sequences. These initial models could learn from sequences of up to 1,000 nt, limiting their ability to capture long-range interaction effects between promoters and distal elements, such as enhancers, which can act over distances of hundreds of thousands of base pairs. To address this limitation, Basenji17 introduced a type of convolution layer that increased the input size of the model to sequences of 131 kb. Compared with previous models such as Basset, Basenji achieved greater precision in peak calling of DNA accessibility measurements obtained using DNase I hypersensitive sites sequencing (DNase-seq) experiments, increasing the area under the precision-recall curve average from 0.435 to 0.577. As the demand for larger receptive fields grew, CNNs reached their limits: the substantially deeper layers reduce resolution and make them less efficient for detecting long-range dependencies. By contrast, transformer architectures can handle large receptive fields and study long-range interactions. Transformers use self-attention, a mechanism by which each element in a sequence can be used to weigh the importance of other elements in the same sequence when processing information, capturing relationships and dependencies without regard to their distance (Fig. Positional encoding is also used along with the self-attention mechanism, as it preserves the relative positions of elements within the DNA sequence. In practice, hybrid architectures are often used, with initial CNN layers that learn local interactions (such as motifs) followed by self-attention layers that focus on long-range dependencies18. Enformer18 was one of the first S2E models to successfully implement transformers. It could handle sequences up to 196 kb and was shown to outperform all previous gene expression predictors. This improvement was attributed to a wider receptive field, which allowed the model to learn promoter–enhancer interactions. As an example, Enformer correctly predicted several distal enhancers (more than 20 kb away) that control the HNRNPA1 gene, in accordance with experimental evidence. However, subsequent analyses19,20 indicated that most of the predictive signal was derived from proximal regions and promoters rather than distal enhancers. A plausible explanation for this is that long-range regulatory interactions are relatively rare, thus providing fewer examples for the model to learn from. A recent successor of Enformer, named Borzoi21, took an even bigger input size of 524 kb and successfully modelled RNA-seq coverage profiles, thereby simultaneously incorporating transcription initiation, termination and splicing. Borzoi's improved performance over Enformer in all downstream tasks related to gene expression levels might also be attributed to its holistic approach to modelling multiple regulatory layers, rather than solely relying on a larger receptive field. Despite their advantages, transformers have quadratic computational and memory complexity, which require large computational power to make training and prediction feasible. As a result, constructing and interpreting these models and identifying their failure points become more challenging22. Additionally, in settings in which training occurs based on the experimentally observed activity of short sequences (such as in massively parallel reporter assays (MPRAs), discussed subsequently), CNN models can outperform transformer-based models23,24. Consequently, recently published and preprint studies that use genome-wide measurements still prefer simpler models such as CNNs25,26,27,28. Various data types have been used to train DL models in the context of gene regulation. These include not only direct measurements of transcriptional activity and mRNA abundance but also epigenome mapping data that provide estimates of TF binding, the activity of putative regulatory elements and local chromatin states that are linked to gene activity (Fig. Often these data types are combined to improve predictive power. MPRAs, which directly test the regulatory activity of large numbers of short DNA fragments, provide a fundamentally different class of training data. Subsequently, we discuss the utility of these diverse data types as input for DL models. Binding of transcription factors (TFs) influences gene expression and can be measured with chromatin immunoprecipitation followed by sequencing (ChIP–seq)129. Regulatory elements often reside in open chromatin, which can be measured with DNase I hypersensitive sites sequencing (DNase-seq) or assay for transposase-accessible chromatin with sequencing (ATAC-seq)130. Histone modifications that correlate with gene expression patterns can also be measured with ChIP–seq131. RNA sequencing (RNA-seq) or 5′mRNA sequencing (5′mRNA-seq) provides measurements of gene expression. 5′mRNA-seq techniques include cap analysis of gene expression132, global run-on with cap capture133 or transcription start site sequencing134, which measure the 5′-capped ends of nascent RNAs (reviewed elsewhere135). Unlike RNA-seq, these techniques do not measure transcription termination, transcript stability or splicing events. All of these different data types can be used to train sequence-to-expression models There are extensive mutual causality links between these features (indicated by arrows): DNA sequence and chromatin accessibility dictate TF binding; some TFs can open up chromatin or recruit histone-modifying enzymes, and transcription is controlled by all of these features but conversely also affects chromatin states36,45,46. Early efforts explored whether genome-wide measurements of gene expression may suffice as training data for S2E models. This was explored in Saccharomyces cerevisiae using mRNA expression data from about 4,000 different experimental conditions29. A CNN model could predict the median mRNA levels across these conditions from DNA sequence alone with substantial accuracy. The sequences of promoters, terminators, untranslated regions and even coding regions collectively contributed to this predictive power. However, this study did not address whether condition-specific patterns of gene expression could be predicted. A similar CNN approach could predict the median mRNA levels taken from a broad range of human and mouse cell types with reasonable accuracy, using 10 kb of DNA sequence surrounding the transcription start sites30. Most of this predictive power was driven by sequences within 1 kb from the transcription start sites. However, it remained difficult to predict cell-type-specific expression patterns30. Methods that measure expression of the 5′ regions of mRNA, such as the global nuclear run-on sequencing variant GRO-cap, have also been used extensively in both preprint and peer-reviewed studies to train S2E models and explain the regulatory logic of promoters and enhancers25,26,27. These findings highlight that although gene activity measurements have limitations when used as the sole training data, they remain an important source of information for S2E models. As an alternative to direct modelling of gene activity, several research groups have used genome-wide maps of regulatory elements, such as enhancers and promoters, as a basis for DL modelling. A diversity of mapping data is available to infer the activity of regulatory elements throughout the genome and across cell types, in particular maps of TF binding that are mostly generated by ChIP–seq and related methods31. Although these mapping data are valuable by themselves, for example, to measure effects of variants on TF binding32, here we will focus on their use as training data for S2E models. Several CNN models, when trained on ChIP–seq data, could accurately predict patterns of TF binding from sequence at base resolution and uncover underlying sequence patterns15,33,34,35, providing useful insights into the biology of TFs and the regulatory elements that they bind to. Active promoters and enhancers are often in a relatively open chromatin state, which can be mapped using either DNase-seq or ATAC-seq36,37. CNN models trained on sets of DNaseI hypersensitive sites could predict cell-type-specific open chromatin sites with good reliability16,38. A DL model based on ATAC-seq data was able to classify multiple classes of putative regulatory elements in human melanoma cell lines39. This model could also predict similar elements in melanoma cell lines from other mammals, even when these elements were not evolutionarily conserved. This implies that the model has learned at least some of the sequence grammar that dictates open chromatin regions in multiple species. Another study compared CNN models of both TF binding and ATAC-seq to identify TFs that are likely to be responsible for chromatin opening (the so-called pioneer factors)35. Enhancers are typically marked by specific histone modifications such as H3K27ac and H3K4me1 (ref. 40) and by the production of short unstable RNAs named enhancer RNAs41. Lists of genomic regions that are marked by such features have been extensively used as a basis to optimize DL models designed to predict regulatory elements (for example, refs. We note, however, that the vast majority of elements in these lists have so far not been experimentally validated, and it is highly likely that only a subset actually has a role in gene regulation36. Predictive DL models based on these epigenome features, while useful, should thus be interpreted with caution. Although tens of thousands of genes in a genome seem a lot, this number of data points might not be enough for robust training of S2E models. Multitask learning may help to tackle this problem. This strategy aims to improve performance by training several related tasks — such as gene activity in multiple cell types, or multiple types of epigenome data — together, within a single model. The rationale is that the partial similarities between multiple enable more effective learning. For example, several histone modifications and open chromatin are known to mark active genes (albeit with somewhat different patterns), and hence their maps jointly provide more robust estimates of gene activity. Additionally, there are extensive mutual causality links between these features: DNA sequence and chromatin accessibility dictate TF binding; some TFs can open up chromatin or recruit histone-modifying enzymes, and transcription is controlled by all of these features but conversely also affects chromatin states36,45,46 (Fig. Likewise, diverse cell types share a substantial part of their gene expression patterns, and hence joint learning on data from multiple cell types may be more effective. For these reasons, multitask learning has been widely explored. Several studies have combined multiple types of experimental data as well as data from a diversity of tissues and cell types to build predictive integrated models of the epigenome and transcriptional activity (for example, refs. Often, thousands of input data sets are combined and, together with optimized model architectures, this has yielded very impressive predictive power. Although multitask models offer advantages over single-task models in terms of their predictive power and their potential for knowledge transfer between data sources50,51, the practice of multitasking is a subject of debate. Multitask models are typically larger than their single-task counterparts. As a result, their high predictive power may be due to the large number of parameters that can be fit, rather than capturing the underlying biological mechanisms more accurately. One study found that multitask models such as Enformer18 and Sei48 underperformed on cell-type-specific accessibility data compared with single-task models52. Enformer also poorly captured distal regulatory elements19 and made errors in predicting the effects of sequence variants53. This suggests that cell-type-specific features might be under-represented in multitask models52,54. Two possible solutions have been proposed in preprint studies: one is to balance the importance for each task54; and the other is to account for expression variation across cell types for each locus20,55,56. As the advantages of multitask training are still debated, training of smaller, single-task models may result in equal or better performance, faster training, quicker predictions and facilitate interpretability, provided that high-quality training data are available. Another solution to overcome data scarcity is to use transfer learning. These models are first trained on large, related data sets, after which the pretrained models are fine-tuned to specific tasks. Published and preprint studies have shown that these methods increase model accuracy and training speeds20,28,57,58,59,60,61,62,63. For example, a CNN trained on ATAC-seq data could be refined on a much smaller set of enhancer assay data to yield a predictive model of enhancer activity across multiple tissues in Drosophila melanogaster58. Although epigenome maps have proven to be extremely valuable as input for DL models, these data types have inherent limitations. Maps of open chromatin and histone modifications are intrinsically correlative and do not offer direct measurements of causal regulatory activity36. This may be confounded further by the fact that the genome is partitioned into large domains of autocorrelated histone modifications64, which poses a challenge to pinpoint causally relevant DNA sequences. In fact, ~15–50% of regions marked by open chromatin are not detectably active as enhancers (for example, refs. MPRAs provide an independent source of training data by directly measuring the ability of thousands up to billions of short DNA sequences (up to ~2 kb), to act as either enhancers or as promoters69,70,71 (Fig. Thus, MPRAs assign regulatory activity directly to short DNA fragments. Moreover, deep MPRAs will contain many partially overlapping fragments that aid in pinpointing which locus contains the actual regulatory signal. For these reasons, using MPRA data as input for DL modelling facilitates inferences of the local regulatory grammar of these elements. By contrast, the short sizes of the measured DNA fragments preclude the modelling of long-range regulatory interactions between regulatory elements, and large-scale MPRAs have so far been limited to cultured cells. A massively parallel reporter assay (MPRA) is established from DNA query fragments, which can be either fragmented genomic DNA or synthetically designed sequences. These sequences are cloned into an MPRA vector. There are many variations on MPRA designs, but we show a few of the most frequently used designs. The top design is the survey of regulatory elements (SuRE) design, in which there is no (minimal) promoter in the vector, allowing measurements of autonomous promoter activity136. The middle design is the self-transcribing active regulatory region sequencing (STARR-seq) design, in which enhancer activities are measured by self-transcription of the queried sequence instead of barcodes137. In the bottom design, queried enhancer fragments are measured by barcode expression (for extensive review, see ref. Transfection of these MPRA libraries will result in activity measurements for all fragments present in the library. CRE, cis-regulatory element; ORF, open reading frame; PAS, poly-adenylation site. Following a first proof-of-concept CNN model of a relatively small MPRA data set72, several teams have expanded and refined this approach. Of note is the model named DeepSTARR, developed to unravel the regulatory grammar in D. melanogaster cells73. Trained on activity measurements of thousands of enhancer fragments in combination with either housekeeping or developmental promoters, this model could uncover some of the regulatory logic that underlies the difference between these two fundamentally different regulatory programmes73. A preprint reports that another CNN model trained on promoter MPRA data from five different human cell lines identified cell-type-specific activities of TFs and uncovered a striking positional logic of TFs74, which was experimentally confirmed74,75. DeepLiver, a transfer-learning CNN incorporating MPRA data, was used to dissect the regulatory grammar of enhancers in multiple mouse liver cell types59. One MPRA strategy is to include completely random DNA query fragments. Interestingly, one study using four different MPRA designs to probe both enhancer and promoter activity found that models trained on billions of random sequences were somewhat more predictive of known transcription start site positions than the models trained on actual genomic sequences67. It has been argued that this higher predictive power of random sequences can be attributed to limited sequence diversity within the human genome76. However, this approach requires MPRA libraries of extremely high complexity77,78, which are technically challenging to construct and introduce with sufficient coverage into mammalian cells. MPRAs with transiently transfected reporters, which typically remain episomal, do not interrogate reporter activity in native chromatin context and may therefore fail to capture chromatin dependencies. To overcome this concern, lentivirus or transposable element vectors can be used to integrate reporters into the genome. However, the activity of an integrated reporter can vary up to ~1,000-fold depending on the genomic location79. To average out such context effects, each reporter must be integrated into tens of random sites, which limits the throughput of the MPRA. With this condition met, one study found good but imperfect correlations between integrated and episomal MPRAs80. Another study trained on lentiviral MPRA data in three different cell types resulted in a high-quality model that could partially model the effects of genetic variants65. Similarly, an S2E model trained on only ~100k fragments measured in a lentiviral MPRA in primary human cortical cells and cerebral organoids was able to dissect neurodevelopmental regulatory grammar68. Whether integrated reporters truly yield better S2E models than episomal reporters remains to be established. Taken together, these studies indicate that MPRA data can be valuable in explaining gene regulatory logic and can be seen as complementary to epigenome maps. Once a well-performing S2E model is available, it becomes reasonable to ask how the model reaches its predictions and whether new biological insights can be gained from it. Broadly speaking, this relies on identifying (combinations of) sequence features learnt by the model to predict its output. This is generally referred to as model interpretation or explainable artificial intelligence. Interpreting S2E models can thus yield a treasure trove of information on regulatory sequence, such as identifying key TFBS motifs and the specific sequence context in which they can be functional. However, S2E model interpretation poses substantial challenges owing to the complexity of these models. In recent years, S2E model interpretability has received much attention in the field (extensively reviewed elsewhere5). Here, we describe the most common techniques specific to the S2E task. Attribution methods aim to evaluate the importance (or contribution) of individual nucleotides in a sequence to the predicted output. This can be visualized in an attribution map. One frequently used approach is in silico saturated mutagenesis (ISM)14,15. Here, the input sequence is mutated exhaustively, one nucleotide at a time, and the model is queried for the predicted effect of each mutation on the transcriptional activity (Fig. Attribution maps are sometimes represented as the average effect of the three possible mutations at a given position. These maps often highlight consecutive nucleotides with high impact on the predictions, and such nucleotide stretches frequently overlap with known TF motifs. It is important to note that this method goes beyond simply identifying TF motifs in a sequence, which could be achieved with a standard position weight matrix scan. Instead, it uncovers a new layer of complexity by detecting motifs that are functional specifically within their sequence context. This distinction is critical, as the presence of a motif in a sequence does not necessarily imply that the TF will bind at that location81; or that the TF, even if bound, will be functionally active. Moreover, the attribution of a given motif can be used to infer motif affinity35,82. In recent years, progress has been made in understanding what models are learning, both to better understand regulatory grammar and to gain insights that can improve the models. For each position in a given sequence, the reference sequence and its three possible mutations are generated. The model predicts scores for all versions, and the score difference between each mutated sequence and the reference sequence is calculated to determine the variant effect. This process is repeated across all positions, resulting in a matrix that indicates the effect of each mutation on the prediction. b, Attribution methods with backpropagation. Alternative methods to in silico mutagenesis leverage the backpropagation process (Box 1) to optimize the internal neural network parameters. As a result, each gradient indicates how a change of any of the nucleotides of the input sequence influences the prediction85. c, Algorithms such as transcription factor (TF)-Modisco89 cluster similar motifs in the attribution maps into a consensus motif. This approach enables the recognition of shared patterns across different sequences18,21,25,27,28,33,59,73,90,91. d, Dissecting sequence context rules. To unravel the regulatory logic learned by the model, it is possible to apply it to synthetic sequences designed for this purpose. For example, synthetic sequences may be designed in which two TF binding sites are inserted in a random background sequence at various distances from each other. The relation between the predicted expression and distance between the in silico inserted TF binding sites can inform about a potential interaction between the factors binding these motifs. Predictions from an existing sequence-to-expression model on a subset of sequences are used to fit a linear model, called a surrogate model. This simpler model includes convolutional filters that represent TF motifs, with their output combined in a linear fashion. The coefficients indicate the contribution of TFs to gene expression. One drawback of ISM is its computational expense, as it requires the model to make predictions for all possible single substitutions in the sequence of interest. Some approaches aim to reduce prediction time by performing only a subset of the model's operations83,84 Moreover, despite being intuitive, ISM struggles with the redundancy problem (sometimes referred to as the saturation problem5,12); motifs that are robust to single mutations (such as redundant motifs that regulate the same gene) might not be detected. An alternative approach to obtain attribution maps is backpropagation method (Fig. Unlike ISM, these methods require just a single computation to obtain an attribution map for a single sequence, and preprints suggest this makes them more computationally efficient, although their attribution maps tend to be noisier85,86,87,88. Some of these tools86,87,88 also solve the redundancy problem. Attribution maps uncover the relevant motifs within single sequences (Fig. Algorithms such as TF-MoDISco, which was reported in a preprint, can aggregate these motifs across multiple sequences (for example, sets of candidate enhancers) to yield commonly used regulatory sequences89 (Fig. This can help to identify not only known TFs but also novel motifs involved in gene regulation18,21,25,27,28,33,59,73,90,91. Although attribution maps are extremely useful to find systematic patterns such as motifs, other approaches are needed if the goal is to determine how a specific sequence context influences the predictions. To address this, the activity of sequences designed to test specific hypotheses can be predicted computationally, effectively mimicking in vivo assays on a scale that would not be experimentally feasible22,33,73,74. By generating tailored sequences, it is possible to test feature combinations to uncover general rules (Fig. For example, inserting two motifs into a set of background sequences and progressively varying their distance can reveal the optimal spacing for motif cooperativity. In this way, pairs of motifs that cooperate preferentially when closely spaced have been found33,73, such as binding motifs for Sox2 and Nanog in mouse embryonic stem cell when placed in a random genetic background33. Conversely, examples of motifs that exhibit stronger cooperativity across longer distances can also be uncovered, such as two GATA motifs in D. melanogaster in a plasmid setting73. Similarly, a recent preprint reports that computational insertion of specific TF motifs throughout human promoters revealed position-dependent activating and repressive effects of these TFs74. Several recent efforts have aimed to combine the predictive power of DL models and the interpretability of linear models26,92,93,94. Linear models are easier to interpret because their coefficients indicate the contribution of input variables, such as a TF binding motif, to gene expression. Explainable models are a class of models that do not rely on post hoc algorithms for interpretation. An example is ExplaiNN93, which uses a linear combination of independent convolutional layers to predict TF binding and enhancer activity. Unlike attribution methods that focus on local sequences, ExplaiNN can identify global patterns intuitively and computationally efficiently. However, because explainable models are simpler, they learn less fine-grained cis-regulatory rules. An emerging alternative approach to reaching model explanations is to approximate a trained DL model with a simpler model called a surrogate model26,82,94 (Fig. For example, a recent study developed the SQUID framework, which uses complex S2E models such as Enformer to predict the activity of sequences of interest that are subsequently used as training data to fit a simpler model, the coefficients of which are easier to interpret94. A preliminary study also suggests that principles of thermodynamics may also be incorporated to understand the biophysical properties of TF binding learned by the S2E models82. The design of explainable and surrogate models still heavily depends on existing biological knowledge, limiting their potential to discover novel insights. Therefore, methods that directly interrogate S2E models, such as attribution methods and applying S2E models to synthetic sequences, remain more powerful tools to uncover new biology. Surrogate and explainable methods are likely to become more popular in the future as our understanding of the regulatory grammar improves. As with any prediction model, thorough validation of performance and robustness under various conditions is essential. Individual S2E predictions (for example, the effect of a particular mutation) may be experimentally verified by standard molecular genetics approaches such as reporter assays or CRISPR editing. However, testing the overall accuracy of the model is more challenging, because it requires large-scale genome perturbations and functional data. Databases of expression quantitative trait loci (eQTL) are often used to obtain an indication of the predictive power of S2E models. However, the resolution of eQTL maps is generally limited by linkage disequilibrium, making it hard to pinpoint individual causal variants. Although fine-mapping techniques such as SuSiE95 can partially overcome this problem, most studies report modest correlations between S2E model predictions and eQTL data (for example, refs. Nevertheless, eQTL benchmarking may be useful for side-by-side comparisons and to identify shortcomings of models53. Published and preprint studies suggest that among the most scalable experimental tools available to verify predictions are MPRAs, which can probe up to hundreds of thousands of sequences20,96,97. The ability to synthesize these sequences de novo offers extensive opportunities for rigorous validation, both by testing variants of natural sequences and by testing completely novel sequences. However, the length of these synthetic sequences has so far been limited to a few hundred base pairs. Mutating sequences within the natural genomic context at a large scale is much more challenging. A promising approach is helicase-assisted mutagenesis, which can distribute random point mutations throughout a single genomic region of about 1,000 bp. These mutations can then be linked to the expression level of a gene of interest98. CRISPR interference (CRISPRi), a multiplexable tool that represses targeted sets of candidate enhancers99, can be used to verify that an S2E model has correctly predicted activating elements, but not to test individual sequence variants within these elements. A complementary approach is Targeted Perturb-seq, which can mutagenize >1,000 candidate regulatory elements in parallel and monitor the effects on the expression of dozens of nearby genes100. However, the mutations generated in Perturb-seq are a mix of short insertions and deletions that cannot be easily determined within the experimental setup. Finally, methods to relocate regulatory elements to thousands of alternative positions79,101 may in the future be applied to test S2E predictions of the interplay between regulatory elements and their broader genomic environment. The successful training of S2E models, combined with robust and stable interpretation techniques, has allowed researchers to interrogate the syntax learned by these DL models (Fig. This has already resulted in tangible new insights into regulatory processes. An early approach, BPNet33, investigated the impact of motif spacing on TF cooperativity, revealed how Nanog binds with helical periodicity and demonstrated how Sox2 and Nanog motifs cooperate in a directional-specific manner. Later efforts built on BPNet's architecture102 showed that short tandem repeats can facilitate TF binding. Similarly, several published26 and emerging25,27 efforts have focused on modelling transcription initiation. Taken together, these studies reveal that local sequences alone can largely explain initiation activity and the positioning of transcription initiation depends on just a few key motifs, whereas additional TFs modulate its strength. Furthermore, by leveraging chromatin accessibility CNN models, an orthogonal study showed how TF perturbation affects chromatin accessibility28 and demonstrated that low-affinity motifs are crucial for controlling regulatory elements and for increasing sensitive responses to perturbation. Sequence-to-expression (S2E) models can be trained on transcription or epigenomic maps, massively parallel reporter assay (MPRA) data or a combination of these data types. These models have a number of applications. a, S2E models allow the regulatory logic of regulatory elements such as promoters and enhancers to be dissected. For example, S2E models can be used to identify cooperativity between transcription factors. b, These models can be used to identify the effects of non-coding sequence variants that occur within the human population. Genome-wide association studies often lack the resolution to directly identify the causal variant of the associated phenotype. S2E models could help in fine mapping, that is, in determining the genetic variant responsible for the trait under study. c, S2E models can be used to create synthetic regulatory elements either by iterative optimization, such as through evolutionary algorithms, or simply by evaluating millions of predictions and selecting the one with the desired predicted activity. This can aid with the design of regulatory elements with a desired activity or cell-type specificity or for use in biotechnology applications or for gene therapy purposes. S2E models have also been extensively applied to enhancer regions to broaden our understanding of their DNA sequence rules35,48,73,90,103. These models have facilitated the study of motif-flanking sequences and the effect of motif distance on cooperativity73,103. Furthermore, they have helped distinguish between pioneer TF motifs that initiate DNA accessibility and the TF motif combinations that activate enhancers while increasing their accessibility during embryogenesis35. Genome-wide association studies (GWAS) aim to link genetic variation to specific phenotypes or diseases by comparing allele frequencies. One of the applications of S2E modelling is the ability to predict the effect of any sequence variant, even those not seen during training (Fig. Several S2E models have been tested on SNPs used in GWAS14,16,17,47,48 and have typically assigned higher scores to SNPs found to be significant in GWAS compared with negative-control SNPs. However, GWAS are often limited by sample size and linkage disequilibrium, which make it challenging to pinpoint the actual causal SNP and the gene affected. S2E models may be an effective way to address these challenges and provide a means of prioritizing and fine-mapping GWAS variants, while also providing a way to shed light on the underlying mechanisms104,105,106. S2E models have also been used to help elucidate gene expression variance in human individuals. Although two studies53,107 identified a subset of genes in which S2E model predictions performed poorly across individuals, the models accurately predicted the magnitude of single-nucleotide variant effects but struggled to predict their direction. Moreover, incorrect predictions across models were not always observed for the same genes, which might imply that the underlying cause of the inaccurate predictions is not due to an intrinsic regulatory mechanism. To tackle this challenge, both studies suggest training on personalized genomes, which would increase the sequence variation for training28. Changes in gene expression are key drivers of speciation and adaptation, and shifts in gene activity across species are primarily linked to sequence variation in cis-regulatory elements rather than differences in transcriptional machinery108. To support this idea, many studies have shown that S2E models, especially enhancer activity predictors, can be trained in one mammalian species and accurately generalize to predict across related tissues in other mammalian species26,39,108,109,110,111,112. Therefore, S2E models have been used to explore functional conservation in enhancer evolution108, link putative enhancers to phenotype evolution111 and uncover unexpected regulatory grammar similarities between species112. As S2E models offer the opportunity to evaluate the functional activity of any sequence of interest, at a massive scale, they are also used to design synthetic sequences de novo (Fig. One approach is to use in silico evolution to iteratively ‘evolve' a sequence to possess a desirable function, whereby the S2E model is used as the evolutionary objective function78,113,114. This has yielded a diversity of synthetic sequences that exhibit higher activity than any genomic sequence present in naturally occurring genomes114. Similarly, this approach can be adapted to yield sequences that are active only in specific cell types or are condensed into shorter sequences while retaining functionality58,73,74,78,113,114,115,116. Such synthetic sequences are particularly interesting for biotechnology and gene therapy purposes. DL seems to be a very fruitful way to tackle the S2E challenge. Recent studies have demonstrated its potential to study the intricate mechanisms of gene regulation, predict the pathogenicity of non-coding variants and design synthetic DNA with tailored functionalities. Despite this substantial progress, several challenges remain in developing truly comprehensive and robust DL models for gene expression. Addressing these challenges will be crucial for realizing the full potential of DL in genome biology. S2E models have evolved to use increasingly larger model architectures and are trained on bigger data sets and multiple tasks simultaneously. Nevertheless, models still struggle to capture long-range regulatory interactions. Incorporating genomic principles and biological constraints into model architectures can substantially enhance their performance. This approach has been successfully demonstrated in AlphaFold, a DL model that achieved unprecedented accuracy in protein structure prediction by integrating biophysical properties into its architecture117. In recent years, genomic language models (gLMs), the genome equivalent of the large language models that have disrupted the language processing field, have gained attention. These models learn motif conservation and co-occurrence patterns by processing genome sequences, either across individuals or species118,119,120,121. Trained gLMs can be used as a basis for training S2E models, theoretically boosting their performance or enabling the use of smaller training data sets. However, according to recent preprints122,123, despite their success with making predictions for coding sequences, the utility of gLMs for tasks involving the regulatory genome remains limited, likely owing to differences in levels of conservation and grammar complexity. Future generations of gLMs may demonstrate potential for the S2E task. With ‘only' tens of thousands of coding genes and upstream regulatory sequences in a typical mammalian genome, some argue that the genome itself may lack sufficient examples to learn all possible TF interactions, in particular for TF motifs that are rare76. Randomizing genomic sequences or generating fully random sequences could address this issue by expanding the variety of training sequences. Fully random sequences will, however, lack any evolutionary selection and may drown the model in negative examples. Instead, cleverly designing partially randomized DNA sequences that remain functional, experimentally feasible and address model blind spots is likely to yield better results. Early attempts to use ‘active learning', where the model is presented with MPRA-based training data on sequences it is most uncertain about, are promising124. Alternatively, increasing the training data in silico by using evolutionarily related sequences may be fruitful125,126. Consequently, training models on a combination of genomic sequences and strategically generated synthetic sequences may substantially improve model performance, especially when studying the effect of TFs whose binding sites might be under-represented in the genome. A major promise of S2E models is to aid in prioritizing likely pathogenic non-coding variants. However, current models can be inconsistent in their predictions and often struggle to predict the effects, or even the direction of effects, of non-coding variants across individuals53,107. This limitation has been partially attributed to the fact that most models are trained on a single reference genome. Emerging preprint studies report improved performance when expression variability across individuals is taken into account20,61,127, suggesting that training models on measurements paired with personalized genomes may be a promising path forward. Nevertheless, further progress is required to deliver on the promise of non-coding variant effect prediction before it is ready for clinical use. Current S2E models are still far from capable of reliably extrapolating to unknown cell types or experimental conditions not represented in the training data. Of note, some very recent progress utilizing foundation models to obtain prediction for unseen cell types is promising128. To reach this goal, an ideal S2E model should not only rely on DNA sequence data but also be able to incorporate data on the abundance and post-translational modifications of all TFs. With sufficient training across diverse cell types, such a model could learn which combinations of TFs and modifications are required for gene regulation in each context. To reach this comprehensive model, it will be important to massively scale up data acquisition efforts to generate training data that cover more diverse cell types and cellular contexts, such as cells with specific perturbations. For the near future, it is likely more fruitful to make it easy to train specific models for a cell type and perturbation of interest: a ‘lean and mean' model. To keep such an endeavour economical, it would be essential to know the minimal amount and type of training data needed to achieve a model capable of: accurately predicting the activity of every gene and correctly identifying the underlying molecular mechanisms, such as which TF binds and controls which regulatory element. Finally, despite their power, it is well known that DL models can ‘hallucinate', that is, make certain predictions that do not match the real world. It will therefore remain essential to rigorously validate predictions by independent methodology. This will be particularly important when predictions are to be used for practical applications such as personalized medicine and biotechnology. Libbrecht, M. W. & Noble, W. S. Machine learning applications in genetics and genomics. Fulco, C. P. et al. Activity-by-contact model of enhancer–promoter regulation from thousands of CRISPR perturbations. Ying, P. et al. Genome-wide enhancer-gene regulatory maps link causal variants to target genes underlying human cancer risk. Sokolova, K., Chen, K. M., Hao, Y., Zhou, J. & Troyanskaya, O. G. Deep learning sequence models for transcriptional regulation. Novakovsky, G., Dexter, N., Libbrecht, M. W., Wasserman, W. W. & Mostafavi, S. Obtaining genetics insights from deep learning via explainable artificial intelligence. This Review provides a detailed description of interpretation methods for sequence-to-expression models. Capitanchik, C., Wilkins, O. G., Wagner, N., Gagneur, J. From computational models of the splicing code to regulatory mechanisms and therapeutic implications. La Fleur, A., Shi, Y. & Seelig, G. Decoding biology with massively parallel reporter assays and machine learning. van Helden, J., Andre, B. Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies. LeCun, Y., Bottou, L., Bengio, Y. & Haffner, P. Gradient-based learning applied to document recognition. Attention is all you need. Deep learning: new computational modelling techniques for genomics. Stormo, G. D. Modeling the specificity of protein–DNA interactions. & Troyanskaya, O. G. Predicting effects of noncoding variants with deep learning-based sequence model. Describing DeepSEA, this pioneering paper uses a convolutional neural network to predict the effects of non-coding variants on epigenomic tracks using only DNA sequence as input. Alipanahi, B., Delong, A., Weirauch, M. T. & Frey, B. J. Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning. & Rinn, J. L. Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks. Kelley, D. R. et al. Sequential regulatory activity prediction across chromosomes with convolutional neural networks. Effective gene expression prediction from sequence by integrating long-range interactions. Describing Enformer, this study was among the first to effectively use transformers to capture long-distance enhancer–promoter interactions, enabling the prediction of epigenomic and expression tracks across multiple cell types. Karollus, A., Mauermeier, T. & Gagneur, J. Current sequence-based models capture gene expression determinants in promoters but mostly ignore distal enhancers. This article highlights some limitations of current transformer sequence-to-expression models. He, A. Y., Palamuttam, N. P. & Danko, C. G. Training deep learning models on personalized genomic sequences improves variant effect prediction. Preprint at bioRxiv https://doi.org/10.1101/2024.10.15.618510 (2024). Linder, J., Srivastava, D., Yuan, H., Agarwal, V. & Kelley, D. R. Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation. This paper describes Borzoi, the current state-of-the-art transformer-based model that predicts RNA-seq, among other tracks, by capturing transcription, splicing and poly-adenylation signals. Toneyan, S. & Koo, P. K. Interpreting cis-regulatory interactions from large-scale deep neural networks. Penzar, D. et al. LegNet: a best-in-class deep learning model for short DNA regulatory regions. Rafi, A. M. et al. A community effort to optimize sequence-based deep learning models of gene regulation. Cochran, K. et al. Dissecting the cis-regulatory syntax of transcription initiation with deep learning. Preprint at bioRxiv https://doi.org/10.1101/2024.05.28.596138 (2024). Dudnyk, K., Cai, D., Shi, C., Xu, J. & Zhou, J. Sequence basis of transcription initiation in the human genome. & Danko, C. G. Dissection of core promoter syntax through single nucleotide resolution modeling of transcription initiation. Preprint at bioRxiv https://doi.org/10.1101/2024.03.13.583868 (2024). Transfer learning reveals sequence determinants of the quantitative response to transcription factor dosage. Deep learning suggests that gene expression is encoded in all parts of a co-evolving interacting gene regulatory structure. Predicting mRNA abundance directly from genomic sequence using deep convolutional neural networks. Lee, B. H. & Rhie, S. K. Molecular and computational approaches to map regulatory elements in 3D chromatin structure. Zhang, Y. et al. MLSNet: a deep learning model for predicting transcription factor binding sites. Avsec, Z. et al. Base-resolution models of transcription-factor binding reveal soft motif syntax. Zhang, Q. et al. Base-resolution prediction of transcription factor binding signals by a deep learning framework. Brennan, K. J. et al. Chromatin accessibility in the Drosophila embryo is determined by transcription factor pioneering and enhancer activation. Gasperini, M., Tome, J. M. & Shendure, J. Towards a comprehensive catalogue of validated and target-linked human enhancers. Marr, L. T., Jaya, P., Mishra, L. N. & Hayes, J. J. Whole-genome methods to define DNA and histone accessibility and long-range interactions in chromatin. Liu, Q., Xia, F., Yin, Q. & Jiang, R. Chromatin accessibility prediction via a hybrid deep convolutional neural network. Minnoye, L. et al. Cross-species analysis of enhancer logic using deep learning. Rada-Iglesias, A. et al. A unique chromatin signature uncovers early developmental enhancers in humans. Noguchi, S. et al. FANTOM5 CAGE profiles of human and mouse samples. Predicting enhancers with deep convolutional neural networks. Li, Y., Shi, W. & Wasserman, W. W. Genome-wide prediction of cis-regulatory regions using supervised deep learning methods. Boosting tissue-specific prediction of active cis-regulatory regions through deep learning and Bayesian optimization techniques. Insights into gene regulation: from regulatory genomic elements to DNA–protein and protein–protein interactions. Kim, S. & Wysocka, J. Deciphering the multi-scale, quantitative cis-regulatory code. Deep learning sequence-based ab initio prediction of variant effects on expression and disease risk. Chen, K. M., Wong, A. K., Troyanskaya, O. G. & Zhou, J. A sequence-based global map of regulatory activity for deciphering human genetics. Fast decoding cell type-specific transcription factor binding landscape at single-nucleotide resolution. Kaiser, L. et al. One model to learn them all. Vandenhende, S. et al. Multi-task learning for dense prediction tasks: a survey. Kathail, P. et al. Current genomic deep learning models display decreased performance in cell type-specific accessible regions. Sasse, A. et al. Benchmarking of deep neural networks for predicting personal gene expression from DNA sequence highlights shortcomings. Lakkapragada, A., Sleiman, E., Surabhi, S. & Wall, D. P. Mitigating negative transfer in multi-task learning with exponential moving average loss weighting strategies. Schwessinger, R., Deasy, J., Woodruff, R. T., Young, S. & Branson, K. M. Single-cell gene expression prediction from DNA sequence at large contexts. Preprint at bioRxiv https://doi.org/10.1101/2023.07.26.550634 (2023). Lal, A. et al. Decoding sequence determinants of gene expression in diverse cellular and disease states. Preprint at bioRxiv https://doi.org/10.1101/2024.10.09.617507 (2025). Novakovsky, G., Saraswat, M., Fornes, O., Mostafavi, S. & Wasserman, W. W. Biologically relevant transfer learning improves transcription factor binding prediction. de Almeida, B. P. et al. Targeted design of synthetic enhancers for selected tissues in the Drosophila embryo. Bravo Gonzalez-Blas, C. et al. Single-cell spatial multi-omics and deep learning dissect enhancer-driven gene regulatory networks in liver zonation. Hingerl, J. C. et al. scooby: modeling multi-modal genomic profiles from DNA sequence at single-cell resolution. Preprint at bioRxiv https://doi.org/10.1101/2024.09.19.613754 (2024). Drusinsky, S., Whalen, S. & Pollard, K. S. Deep-learning prediction of gene expression from personal genomes. Preprint at bioRxiv https://doi.org/10.1101/2024.07.27.605449 (2024). & Ioannidis, N. M. Leveraging genomic deep learning models for non-coding variant effect prediction. Murphy, A. E., Beardall, W., Rei, M., Phuycharoen, M. & Skene, N. G. Predicting cell type-specific epigenomic profiles accounting for distal genetic effects. Waszak, S. M. et al. Population variation and genetic control of modular chromatin architecture in humans. Agarwal, V. et al. Massively parallel characterization of transcriptional regulatory elements. Liu, Y. et al. Functional assessment of human enhancer activities using whole-genome STARR-sequencing. Sahu, B. et al. Sequence determinants of human gene regulatory elements. Deng, C. et al. Massively parallel characterization of regulatory elements in the developing human cortex. Trauernicht, M., Martinez-Ara, M. & van Steensel, B. Deciphering gene regulation using massively parallel reporter assays. & Lea, A. J. Leveraging massively parallel reporter assays for evolutionary questions. & VanDusen, N. J. Massively parallel reporter assays for high-throughput in vivo analysis of cis-regulatory elements. Movva, R. et al. Deciphering regulatory DNA sequences and noncoding genetic variants using neural network models of massively parallel reporter assays. de Almeida, B. P., Reiter, F., Pagani, M. & Stark, A. DeepSTARR predicts enhancer activity from DNA sequence and enables the de novo design of synthetic enhancers. This study applied a convolutional neural network model to predict enhancer activity in a plasmid-based assay to understand the grammatical rules of enhancers in Drosophila melanogaster cells. The regulatory grammar of human promoters uncovered by MPRA-trained deep learning. Preprint at bioRxiv https://doi.org/10.1101/2024.07.09.602649 (2024). Duttke, S. H. et al. Position-dependent function of human sequence-specific transcription factors. de Boer, C. G. & Taipale, J. Hold out the genome: a roadmap to solving the cis-regulatory code. de Boer, C. G. et al. Deciphering eukaryotic gene-regulatory logic with 100 million random promoters. The evolution, evolvability and engineering of gene regulatory DNA. This study was one of the first to design functional synthetic sequences and investigate regulatory evolution in Saccharomyces cerevisiae using sequence-to-expression models. Akhtar, W. et al. Chromatin position effects assayed by thousands of reporters integrated in parallel. Klein, J. C. et al. A systematic evaluation of the design and context dependencies of massively parallel reporter assays. Badis, G. et al. Diversity and complexity in DNA recognition by transcription factors. Alexandari, A. M. et al. De novo distillation of thermodynamic affinity from deep learning regulatory sequence models of in vivo protein–DNA binding. Preprint at bioRxiv https://doi.org/10.1101/2023.05.11.540401 (2023). Nair, S., Shrikumar, A., Schreiber, J. & Kundaje, A. fastISM: performant in silico saturation mutagenesis for convolutional neural networks. Accelerating in silico saturation mutagenesis using compressed sensing. Deep inside convolutional networks: visualising image classification models and saliency maps. Learning important features through propagating activation differences. Describing DeepLIFT, this study represents pioneering work on attribution methods for interrogating sequence-to-expression models. A unified approach to interpreting model predictions. & Yan, Q. Axiomatic attribution for deep networks. Shrikumar, A. et al. Technical note on transcription factor motif discovery from importance scores (TF-MoDISco) version 0.5.6.5. Janssens, J. et al. Decoding gene regulation in the fly brain. Yuan, H. & Kelley, D. R. scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks. Ribeiro, M. T., Singh, S. & Guestrin, C. “Why Should I Trust You?”: explaining the predictions of any classifier. 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining 1135–1144 (Association for Computing Machinery, 2016). Novakovsky, G., Fornes, O., Saraswat, M., Mostafavi, S. & Wasserman, W. W. ExplaiNN: interpretable and transparent neural networks for genomics. Seitz, E. E., McCandlish, D. M., Kinney, J. & Koo, P. K. Interpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models. Wang, G., Sarkar, A., Carbonetto, P. & Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. Siraj, L. et al. Functional dissection of complex and molecular trait variants at single nucleotide resolution. Preprint at bioRxiv https://doi.org/10.1101/2024.05.05.592437 (2024). Integration of multiple epigenomic marks improves prediction of variant impact in saturation mutagenesis reporter assay. Chen, X. D. et al. Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes. Yao, D. et al. Multicenter integrated analysis of noncoding CRISPRi screens. Targeted Perturb-seq enables genome-scale genetic screens in single cells. Eder, M., Moene, C. J. I., Dauban, L., Leemans, C. & van Steensel, B. Functional maps of a genomic locus reveal confinement of an enhancer by its target gene. Preprint at bioRxiv https://doi.org/10.1101/2024.08.26.609360 (2024). Horton, C. A. et al. Short tandem repeats bind transcription factors to tune eukaryotic gene expression. Reiter, F., de Almeida, B. P. & Stark, A. Enhancers display constrained sequence flexibility and context-specific modulation of motif function. Evaluating the informativeness of deep learning annotations for human complex diseases. Trevino, A. E. et al. Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution. Wang, S. K. et al. Single-cell multiome of the human retina and deep learning nominate causal variants in complex eye diseases. Personal transcriptome variation is poorly explained by current genomic deep learning models. Prediction of gene regulatory enhancers across species reveals evolutionarily conserved sequence properties. Kelley, D. R. Cross-species regulatory sequence activity prediction. Kaplow, I. M. et al. Inferring mammalian tissue-specific regulatory conservation by predicting tissue-specific differences in open chromatin. Relating enhancer genetic variation across mammals to complex phenotypes using machine learning. Hecker, N. et al. Enhancer-driven cell type comparison reveals similarities between the mammalian and bird pallium. Taskiran, I. I. et al. Cell-type-directed design of synthetic enhancers. In this paper, several approaches are used to construct cell-type-specific enhancers using sequence-to-expression models. Gosai, S. J. et al. Machine-guided design of cell-type-targeting cis-regulatory elements. Lal, A., Garfield, D., Biancalani, T. & Eraslan, G. Designing realistic regulatory DNA with autoregressive language models. Sarkar, A., Tang, Z., Zhao, C. & Koo, P. K. Designing DNA with tunable regulatory activity using discrete diffusion. Preprint at bioRxiv https://doi.org/10.1101/2024.05.23.595630 (2024). Improved protein structure prediction using potentials from deep learning. Ji, Y., Zhou, Z., Liu, H. & Davuluri, R. V. DNABERT: pre-trained bidirectional encoder representations from transformers model for DNA-language in genome. Dalla-Torre, H. et al. Nucleotide transformer: building and evaluating robust foundation models for human genomics. Benegas, G., Batra, S. S. & Song, Y. S. DNA language models are powerful predictors of genome-wide variant effects. Nguyen, E. et al. HyenaDNA: long-range genomic sequence modeling at single nucleotide resolution. Tang, Z., Somia, N., Yu, Y. & Koo, P. K. Evaluating the representational power of pre-trained DNA language models for regulatory genomics. Preprint at bioRxiv https://doi.org/10.1101/2024.02.29.582810 (2024). Isa Marin, F. et al. BEND: benchmarking DNA language models on biologically meaningful tasks. Active learning of enhancer and silencer regulatory grammar in photoreceptors. Preprint at bioRxiv https://doi.org/10.1101/2023.08.21.554146 (2023). & Moses, A. M. Improving the performance of supervised deep learning for regulatory genomics using phylogenetic augmentation. Lee, N. K., Tang, Z., Toneyan, S. & Koo, P. K. EvoAug: improving generalization and interpretability of genomic deep neural networks with evolution-inspired data augmentations. Rastogi, R., Reddy, A. J., Chung, R. & Ioannidis, N. M. Fine-tuning sequence-to-expression models on personal genome and transcriptome data. Preprint at bioRxiv https://doi.org/10.1101/2024.09.23.614632 (2024). Fu, X. et al. A foundation model of transcription across human cell types. Johnson, D. S., Mortazavi, A., Myers, R. M. & Wold, B. Genome-wide mapping of in vivo protein–DNA interactions. Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. & Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Barski, A. et al. High-resolution profiling of histone methylations in the human genome. Shiraki, T. et al. Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage. Kruesi, W. S., Core, L. J., Waters, C. T., Lis, J. T. & Meyer, B. J. Condensin controls recruitment of RNA polymerase II to achieve nematode X-chromosome dosage compensation. Massive transcriptional start site analysis of human genes in hypoxia cells. & Zentner, G. E. Global approaches for profiling transcription initiation. van Arensbergen, J. et al. Genome-wide mapping of autonomous promoter activity in human cells. Arnold, C. D. et al. Genome-wide quantitative enhancer activity maps identified by STARR-seq. Bajwa, A., Rastogi, R., Kathail, P., Shuai, R. W. & Ioannidis, N. M. Characterizing uncertainty in predictions of genomic sequence-to-activity models. Preprint at bioRxiv https://doi.org/10.1101/2023.12.21.572730 (2023). The authors acknowledge V. Franceschini-Santos for extensive discussion and help with the generation of figures. Research at the Netherlands Cancer Institute is supported by an institutional grant of the Dutch Cancer Society and of the Dutch Ministry of Health, Welfare and Sport. The Oncode Institute is partially funded by the Dutch Cancer Society. Oncode Institute, Utrecht, The Netherlands Lucía Barbadilla-Martínez, Noud Klaassen, Bas van Steensel & Jeroen de Ridder Lucía Barbadilla-Martínez & Jeroen de Ridder Division of Molecular Genetics, Netherlands Cancer Institute, Amsterdam, The Netherlands Noud Klaassen & Bas van Steensel You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar All authors contributed substantially to discussion of the content, wrote the article and reviewed and/or edited the manuscript before submission. Correspondence to Bas van Steensel or Jeroen de Ridder. The authors have applied for a patent related to the S2E model PARM. Nature Reviews Genetics thanks Julia Zeitlinger and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Interpretation technique that visualizes the importance of each nucleotide in a sequence to the prediction of the model. A neural network architecture using convolutional layers that extract local patterns at different spatial hierarchies. A class of machine-learning approaches capable of identifying highly complex patterns in large data sets. Unlike classical machine learning, DL methods can automatically learn the best representation through stacking artificial neural networks in multiple layers, thereby minimizing the need for manual feature engineering. A basic linear algebra computation to multiply matrices. Genomic loci that regulate expression levels of mRNA or proteins. A neural network layer in which every input contributes to the computation of every output. These are typically the last layers in a model. Configuration settings that control how the model learns, such as model size and learning rate, which are set before training and may be optimized during validation. Also referred to as a filter, a kernel is a small matrix that can detect specific patterns in the input sequence through the process of convolution. Which features each kernel recognizes is determined by the parameters in the matrix, which are learned during training. A short substring of length ‘k' found within a larger biological sequence (such as DNA or RNA). Describes a wide range of algorithms that can learn from data to make predictions on new and unseen data. Examples include random forest, support vector machines and gradient boosting. Machine-learning methods often require manual feature engineering. Also referred to as weights, parameters are values in the model that are learned during training to optimize performance. Representation of motifs in a DNA sequence as matrices that indicate the frequency or importance of each nucleotide at each position. Region of DNA sequence that influences the prediction of the model at a given position. A mechanism used in transformer deep learning models that allows the model to weigh the importance of different parts of an input sequence when processing it, effectively allowing the model to ‘focus' on the most relevant part of the input DNA sequence. A technique whereby a model trained on one task is used as a starting point for a model on a different but related task, leveraging previously learned knowledge to improve performance. A neural network architecture based on the concept of self-attention that weights the importance of different parts of the input, allowing long-range dependencies to be captured. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Barbadilla-Martínez, L., Klaassen, N., van Steensel, B. et al. Predicting gene expression from DNA sequence using deep learning models. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative You have full access to this article via your institution. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64745894/biophotons-emit-glow/'>Every Human Body Gives Off an Invisible Glow. Here's What It Means.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ultraweak biophotons largely caused by metabolic processes could play a key role in noninvasive health testing. “You're glowing” is a well-known complimentary phrase meant to convey a perceived level of health, happiness, or other biological condition—pregnancies often inspire the platitude, for example. But in reality, humans (along with all other living things) truly do emit a glow largely caused by an organism's metabolic and cellular processes. This glow, in scientific parlance, is created by what are known as biophotons (or ultraweak photons). Scientists have studied them for decades, probing their changes due to age, gender, health, and many other factors. It's really something that comes from all living things.” To capture this process in real time, Oblak and his team captured two one-hour exposures using advanced digital cameras capable of capturing individual photons emitted from four mice. “While the live mice emit robust UPE, likely indicative of ongoing biological processes and cellular activity, the dead mice's UPE emission is nearly extinguished, with only a few remaining ‘bright spots', which correspond to ‘brighter spots' in living mice, reflecting the cessation of metabolic and energy variation,” the authors wrote. Of course, these photon emissions are not exclusive to the animal kingdom, so Oblak and his team used a similar process to analyze UPEs in a plant—specifically, an umbrella tree (Heptapleurum arboricola). They tested the emission of biophotons due to plant injury, as well as the application of certain chemicals like alcohol (isopropanol), hydrogen peroxide (H2O2), and benzocaine. Moreover, UPE may be used as an indicator of injury effects in plants, where the injured parts emit more photons,” the authors wrote. “The investigation of UPE from plants may be used as a simple method for non-invasive monitoring of health abnormality and plant growth under different environmental conditions for plant biology and agricultural practices.” As New Scientist notes, previous studies have similarly captured the disparate “glowing” effect in living cells (and even individual body parts), but never an entire animal organism as examined in this study. A Surprising Reason Why Your Pee May Turn Red Humans Could Grow New Teeth in Just a Few Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-trumps-national-weather-service-cuts-could-cost-lives/'>The National Weather Service Is Understaffed and Facing Huge Cuts—That Could Cost Lives</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Weather experts warn that staff cuts at the National Weather Service that have been made by the Trump administration are a danger to public safety as tornadoes, hurricanes and heat loom this spring and summer A house submerged in floodwaters, inPointe-Aux-Chenes, Terrebonne Parish, La. Just more than 100 years ago, on March 18, 1925, a tornado slashed across the U.S. Midwest with no warning at all and killed 695 people—a massive number for a single outbreak. But by last year, forecasters could give several days' warning that the then approaching storms Helene and Milton were likely to abruptly morph into monsters. “Our worst nightmare is that weather forecast offices will be so understaffed that there will be needless loss of life,” wrote five former NWS directors from both Democratic and Republican administrations in an open letter on May 2. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Ultimately, storm experts say, disruption caused by existing and proposed cuts will hit multiple fronts. “The net result is going to be massive economic harm,” said climate scientist Daniel Swain during one of his regular talks hosted on YouTube. “We have [a more] efficient level of [staff compared] to the number of people we're serving than any other country in the world by two orders of magnitude,” says Louis Uccellini, who was NWS director from 2013 to 2022 and signed the open letter. “It's a cup of coffee,” says JoAnn Becker, president of the National Weather Service Employees Organization, a union that represents the NWS and several NOAA offices. In comparison, the NWS's entire budget for 2024 was less than $1.4 billion. Chris Wood, Flight Engineer Rusty Dittoe, and Hurricane Aircraft Commander Adam Arbitbol flies towards Tropical Storm Debby on Aug. 3, 2024. “NOAA is a $12-billion agency trapped in a $5-and-a-half-billion budget,” said Craig McLean, then acting chief scientist of NOAA, in testimony to Congress three years ago. Fifty-two offices are now considered “critically understaffed,” meaning a shortage of more than 20 percent. “When you're down four people, it starts to hurt,” she adds. “There comes a point where you don't have enough people to cover everything.” Those data are then fed into weather models and are crucial to making sure the models start with the most accurate possible information. This is particularly true during tornado outbreaks or prior to a hurricane landfall. For the former, forecasters need to understand the atmospheric patterns that influence an outbreak to better pinpoint where tornadoes might spin up. And understanding atmospheric patterns over the country is critical to forecasting where a hurricane will make landfall. “And the effects may not be obvious until there's a major tornado outbreak or hurricane landfall downwind that doesn't go so well.” The fact that a function as essential as balloon launches is being cut is a clear sign of how much staff are feeling the crunch, Swain said. Another concern regards forecasting equipment, such as the nation's Doppler radar system, which is the only tool that forecasters can use to spot tornadoes inside storms systems, allowing for better warnings. Staffing cuts and spending freezes mean that if any radars or other equipment go down, offices may not have the staff or money to repair them, Becker and others say. Research has shown that including these data clearly makes hurricane forecasts more accurate and reliable. Diminished crews mean some flights could be cancelled, leaving coastal communities more vulnerable to approaching storms. Some of the NWS offices will become so short-staffed that they may have to operate part-time, the agency's former directors warned in their open letter. This could include making fewer highly tailored forecasts, as well as performing less outreach on social media and to local officials and emergency managers. Such outreach has been a major goal of the NWS to make sure communities are better prepared before extreme weather hits. The forecasters in the NWS offices are “community experts” who have close working relationships with emergency managers, school districts and other local decision-makers, Becker says. Without those proactive efforts, “you're basically watching the storm,” Uccellini says. Being down so many people means “you have to cut corners—and cutting corners is dangerous with lives and property at stake,” says Jeff Masters, a writer at Yale Climate Connections and a former Hurricane Hunter at NOAA. Uccellini likens what is happening to stretching a rubber band: “You can stretch and stretch it, and then it breaks,” he says. Neither artificial intelligence forecasts nor private weather companies will be able to fill in the gap; both rely on the data NOAA collects. Without robust NOAA data collection, “the Weather Channel, Accuweather ... will be unable to function as they have,” says Rick Spinrad, who served as NOAA administrator from 2021 to 2025. Industries that depend on weather and climate data, such as the insurance industry, have also spoken out. The Union of Concerned Scientists has also sent congressional leaders an open letter to urge them to reinstate NOAA's staffing and funding that has been signed by more than 3,300 scientists and other experts. Morale is extremely low in offices across the NWS, according to Swain's video and to Uccellini and many others who know current employees at the agency. Funding cuts are forcing many employees to bring in their own toilet paper and soap. There is also “an extreme culture of fear” Swain said in his video, with “threatening and demeaning communications” from agency leaders that have called employees “lazy” and “low productivity.” In their open letter, the former NWS directors contradicted this characterization, speaking of the dedication of the agency's employees: “They will often sleep in weather forecast offices to make sure poor weather conditions don't stop them from being on time for their shifts to do their critical work. Instead, Spinrad says, the Trump administration has made “easy” cuts such as firing “probationary” employees (those who were newly hired or recently promoted, making them easier to fire). This approach “is trying to use a chainsaw instead of a scalpel to fix the patient” in terms of addressing bureaucratic inefficiencies, Masters says. These adjustments are also temporary and we will continue to fulfill our core mission of providing life-saving forecasts, warnings, and decision support services.” “In an era of climate change causing increased extreme weather, we should be spending more on NOAA and the National Weather Service, not less,” Masters says. “This is a very poor way to spend our tax dollars.” Andrea Thompson is an associate editor covering the environment, energy and earth sciences. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. in atmospheric chemistry from the Georgia Institute of Technology.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/california-told-companies-to-label-toxic-chemicals-instead-theyre-quietly/'>California Told Companies to Label Toxic Chemicals. Instead They're Quietly Dropping Them</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Businesses are making moves to avoid consumer warning labels, and the effects reach far beyond California Requiring warning labels on products with potentially toxic ingredients can obviously help keep them out of a careful consumer's shopping cart. But a recent study shows that these “right-to-know” laws may also halt such formulations long before they hit the shelves or are released into the air—and can even protect people outside a law's geographic range. One of the most significant such laws ever passed in the U.S., California's Proposition 65, requires businesses to post a warning when chemical exposures, whether through product ingredients or air emissions, exceed a safe standard. For the recent study, published in Environmental Science & Technology, researchers interviewed business leaders and found that California's rule has caused many companies to reformulate their products by reducing amounts of flagged ingredients to safer levels—or by dropping them entirely. The interviews covered dozens of industries such as cleaning products, electronics and home improvement. They included top-earning brands across all sectors as well as leading green cleaning brands—although the companies remain anonymous in the study, says lead author Jennifer Ohayon, a scientist at the nonprofit research organization Silent Spring Institute. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Ohayon and her colleagues found that companies commonly replaced the warning-requiring ingredients altogether, in part to avoid possible litigation. Michael Freund is a lawyer who spent decades representing groups aiming to stop toxic chemical emissions; he says the California proposition's incentives can help fill a key gap. In the cases he worked on, “every one of those companies had permits that allowed them to do what they were doing,” he says. Although the 1986 law is specific to California, the study results suggest its effects cross state borders as manufacturers reformulate their products nationally. A parallel study published last year by the Silent Spring Institute backs this idea up with data. Of the chemicals, 26 were listed in Prop 65, and samples from before and after listing were available for 11 of those, which allowed for a comparison. In a metaphorical game of Twister, the researchers had to figure out what publicly available data could be matched to Prop 65 chemicals because “the data weren't collected for that purpose,” Schwarzman says. Monitoring all listed chemicals over time in future work would show any patterns much more clearly. Ripley Cleghorn is a science communicator and data visualization designer.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41565-025-01921-4'>Battery research needs more reliable, representative and reproducible synchrotron characterizations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 10:34:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Synchrotron techniques can probe battery materials and devices at unprecedented scales of time and space, providing in-depth mechanistic understanding. However, the lack of standardization in synchrotron measurements and analyses can lead to biased interpretations of data and results. Here, we propose possible strategies to address the reliability, representativeness and reproducibility issues of synchrotron characterizations in battery research. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access Buy this article Prices may be subject to local taxes which are calculated during checkout Frith, J. T., Lacey, M. J. Cheng, W. et al. Exploration 4, 20230056 (2023). Fordham, A. et al. Joule 7, 2622–2652 (2023). Lübke, E. et al. Energy Environ. Jacquet, Q. et al. Adv. Atkins, D. et al. Adv. Yang, C. et al. Joule 6, 8–15 (2022). Puls, S. et al. Nat. Bras, W., Myles, D. A. Jousseaume, T., Colin, J.-F., Chandesris, M., Lyonnard, S. & Tardif, S. ACS Energy Lett. Clark, S. et al. Adv. The European Synchroton. Kasper, M. et al. J. The authors acknowledge the Battery Pilot Hub (proposal MA-4929) and the European Battery Hub (proposal MA-6230) at ESRF. T. Jousseaume and S. Tardif contributed to the figures. European Synchrotron Radiation Facility (ESRF), Grenoble, France You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Jakub Drnec or Sandrine Lyonnard. The authors declare no competing interests. Reprints and permissions Drnec, J., Lyonnard, S. Battery research needs more reliable, representative and reproducible synchrotron characterizations. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02189-z'>Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorder</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-13 09:30:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Gene-based approaches identified 249 potential effector genes for OCD, with 25 of these classified as the most likely causal candidates, including WDR6, DALRD3 and CTNND1 and multiple genes in the major histocompatibility complex (MHC) region. OCD genetic risk was associated with excitatory neurons in the hippocampus and the cortex, along with D1 and D2 type dopamine receptor-containing medium spiny neurons. OCD genetic risk was shared with 65 of 112 additional phenotypes, including all the psychiatric disorders we examined. The first GWAS (ncases = 14,140, ncontrols = 562,117)12 found one genome-wide significant locus associated with OCD, while the second (ncases = 37,015, ncontrols = 948,616)13 identified 15 independent genome-wide significant loci. As with other complex traits, increased sample sizes are needed for a more comprehensive understanding of the underlying genetic etiology of OCD and its genetic relationships with related disorders. Based on the results from the meta-analysis, we conducted secondary analyses, including positional and functional fine-mapping of SNPs and genes, structural equation modeling to examine possible genetic differences in sample ascertainment across cohorts, protein and transcriptome-wide association analyses, single-cell enrichment and genetic correlations with other traits (Supplementary Fig. Our results provide more detailed insight into the genetic underpinnings and biology of OCD. Ascertainment of cases varied across cohorts: OCD diagnosis was determined (1) by a healthcare professional in a clinical setting (18 cohorts, n = 9,089 cases), (2) from health records or biobanks (seven cohorts, n = 9,138 cases), (3) in a clinical setting or from health records with the additional characteristic that all OCD cases were primarily collected for another psychiatric disorder (three cohorts, n = 5,266 cases) or (4) by self-reported diagnosis in a consumer-based setting (23andMe, Inc., n = 30,167 cases). Cohort details, including phenotypic assessment, quality control and individual cohort GWAS analyses, are described in Supplementary Note 2 and Supplementary Table 1. We identified 30 independent (defined in Supplementary Note 3) loci among the 1,672 SNPs that exceeded the genome-wide threshold for significance (\(P < 5{\times 10}^{-8}\); Manhattan plot in Fig. Analysis of the X chromosome, conducted in a subset of the data for which this information was available (23andMe), yielded no significant associations (Supplementary Note 4 and Supplementary Fig. Using MiXeR15, we estimated that approximately 11,500 (standard error of the effect estimate (s.e.) = 607) causal variants account for 90% of the OCD SNP-based heritability. The y axis represents −log10 (P values) (two sided, not adjusted for multiple testing) for the association of variants with OCD using an inverse-variance-weighted fixed-effects model (ncases = 53,660 and ncontrols = 2,044,417). Index variants of genome-wide significant loci are highlighted as green diamonds. No statistically significant heterogeneity was observed across individual cohorts for the 30 genome-wide significant loci, as assessed with Cochran's Q-test (Supplementary Fig. 33–37) showed evidence that sample ascertainment impacted results at a genome-wide scale, although not beyond what is observed with closely related psychiatric disorders17,18. We observed moderate to high genetic correlations across the subgroups (between 0.63, s.e. A common factor GWAS based on the one-factor GenomicSEM model resulted in 20 significant loci, all of which were also significant in the primary GWAS (Supplementary Table 8 and Supplementary Fig. = 0.3%), with slightly higher estimates for the clinical (\({h}_{{\rm{SNP}}}^{2}\) = 16.4%, s.e. We prioritized putative risk genes for OCD using six positional and functional QTL gene-based mapping approaches. Functional expression quantitative trait locus (eQTL) mapping was performed with transcriptome-wide association study (TWAS)20, using PsychENCODE gene expression weights21, and summary-based Mendelian randomization (SMR)22 using the whole-blood eQTLGen23 and MetaBrain24 datasets. Finally, we used the psychiatric omnilocus prioritization score (PsyOPS)26, which combines positional mapping with biological annotations, to further prioritize risk genes within genome-wide significant loci. In total, 251 genes were significantly associated with OCD through at least one gene-based approach, and 48 were implicated by at least two methods (Methods, Supplementary Note 7 and Supplementary Tables 9–14). From the 48 genes implicated by at least two approaches, we prioritized likely causal genes for OCD using colocalization (TWAS-COLOC)27,28 and SMR–heterogeneity in dependent instruments (SMR-HEIDI)22 tests. Colocalization was used to identify significant TWAS associations for which the underlying GWAS and eQTL summary statistics are likely to share a single causal variant. Similarly, HEIDI was used to select SMR associations for which the same causal variant affects gene expression and trait variation. Another gene of interest, CTNND1 (catenin δ1), was implicated by three of our five approaches (multivariate set-based association test (mBAT-combo), TWAS, PWAS) and showed evidence for colocalization. For a discussion of the overlap between the gene findings with rare coding variants in OCD, see Supplementary Table 6 and Supplementary Note 7. a, List of 25 genes that were implicated in at least two of the five different gene-based tests (significance indicated by gray dots) and passed the TWAS colocalization and/or SMR-HEIDI filters (significance indicated by orange dots). genes within each locus are indicated by blue dots. b, Enrichment of OCD GWAS signal in human brain-related tissues from GTEx (version 8). c, Top 20 groups of brain cell types (n = 35 total tested) enriched with OCD GWAS signal using MAGMA. Vertical crosses represent the mean −log10(P value) observed for each brain cell type group. Blue crosses represent a significant enrichment of OCD GWAS signals (FDR across 35 groups, FDR < 0.05), while pink crosses indicate nonsignificant enrichment. Gray points represent the association (−log10(P value)) for each single cell cluster (‘level 5' analysis defined by Zeisel et al.30) in a given cell type (for example, excitatory neurons, cerebral cortex). After mapping significantly associated SNPs from the GWAS meta-analysis to likely causal genes, we explored which tissues or cell types showed enriched gene expression of OCD-associated genetic signals using a previously described approach29 on published human gene expression datasets from bulk tissue RNA-seq data from the Genotype–Tissue Expression (GTEx) project and single-cell RNA-sequencing data from the adult mouse central and peripheral nervous systems30. In the adult mouse central and peripheral nervous systems, we found enrichment of OCD GWAS signals in 41 of 166 tested specific single cell types using the MAGMA gene set enrichment test (Supplementary Table 16). When summarizing results of individual single cell types into groups of cell types defined by the same region or tissue and cell type, nine of 35 were enriched for OCD GWAS signals (top 20 shown in Fig. Strong enrichment of OCD GWAS signal was especially observed in excitatory neurons of the hippocampus and the cerebral cortex as well as in D1 and D2 medium spiny neurons (MSNs). Using phenome-wide association analysis, we examined whether the 30 independent OCD-associated loci identified by our GWAS meta-analysis have previously been associated with other phenotypes (see Supplementary Tables 17a–d for lookups in four, partially overlapping GWAS databanks and Table 1 for highlighted associations). We further used bivariate linkage disequilibrium score regression (LDSC)31 to investigate the extent of genetic correlations between OCD and 112 previously published GWASs encompassing psychiatric, substance use and neurological phenotypes, among others (Fig. We found that 65 phenotypes were significantly correlated with OCD after correcting for multiple testing using the Benjamini–Hochberg32 procedure to control the false discovery rate (FDR) at a threshold of 0.05. Significant positive genetic correlations were also obtained for neuroticism (\({r}_{\rm{G}}=0.53\)), in particular for the worry subcluster (\({r}_{\rm{G}}=0.64\)), and all individual items in the worry subcluster, with slightly lower estimates for the depressive subcluster (\({r}_{\rm{G}}=0.35\)). Of the assessed neurological disorders, OCD was only significantly correlated with migraine (\({r}_{\rm{G}}=0.15\)). This includes psychiatric, substance use, cognition–socioeconomic status (SES), personality, psychological, neurological, autoimmune, cardiovascular (cardiovasc. Red circles indicate significant associations with a P value adjusted for multiple testing with the Benjamini–Hochberg procedure to control the FDR (<0.05). a., after; ADHD, attention-deficit hyperactivity disorder; ALS, amyotrophic lateral sclerosis; BMI, body mass index; embarras., embarrassment; freq, frequency; fr., from; HDL, high-density lipoprotein; IQ, intelligence quotient; LDL, low-density lipoprotein; neurot., neuroticism; nr., number; PTSD, post-traumatic stress disorder; sat., satisfaction; VN, verbal-numerical. The OCD GWAS reported here, comprising over 53,000 cases, identified 30 independent genome-wide significant loci. Common SNPs explained 6.7% of the variation in OCD risk in our meta-analysis (LDSC with an assumed population prevalence of 1%), a significant reduction from the 28% reported previously10. However, differences in the assumed population prevalence (where a lower assumed prevalence for LDSC heritability calculation results in a lower heritability estimate) and an increase in sample heterogeneity likely contributed to this discrepancy. The reduction in SNP heritability is in line with previous observations for closely related psychiatric disorders such as attention deficit hyperactivity disorder (ADHD)33,34 or depression17,35,36,37, where expanding the phenotype definition increased genetic heterogeneity, potentially accounting for the observed decrease in SNP heritability. The current estimates are comparable to those of other psychiatric and substance use disorders, with SNP heritability estimates ranging between 9% and 28%38. The most significant SNP (rs78587207 (\(P=5.28\times 1{0}^{-12}\))) identified in the GWAS is located on chr11q12.1 and has been previously associated with several traits, including neuropsychiatric phenotypes39 such as depressive symptoms40 and neuroticism40. Gene-based analyses identified four putative causal genes within this locus. This gene was associated with OCD using three gene-based tests (mBAT-combo, TWAS and PWAS), and we found strong evidence for colocalization of the TWAS signal for CTNND1 in the dlPFC. The protein product of CTNND1 is a regulator of cell–cell adhesion43 and has a crucial role in gene transcription, Rho GTPase activity and cytoskeletal organization44,45,46. Other credible causal genes in the locus include CLP1 (cleavage factor polyribonucleotide kinase subunit 1), TMX2 (thioredoxin-related transmembrane protein 2) and ZDHHC5 (zinc finger DHHC type palmitoyltransferase 5). Rare genetic mutations in CLP1 are associated with pontocerebellar hypoplasia type 10, a very rare autosomal recessive neurodegenerative disease characterized by brain atrophy and delayed myelination resulting in intellectual disability47. ZDHHC5 has not been implicated in brain development but has been linked to lung acinar adenocarcinoma and lung papillary adenocarcinoma in prior studies49. We identified a total of 25 credible causal genes based on robust evidence using multiple positional and functionally informed gene-based approaches. Of the 25 genes that were implicated, 15 were within 6.5 kb of a SNP that surpassed genome-wide significance in the meta-analysis. These genes lie in a gene-rich region on chr3p21.31, which, in addition to harboring multiple genome-wide significant SNPs, has been previously associated with a broad range of psychiatric disorders and related traits, including schizophrenia39, well-being52 and the worry subcluster of neuroticism53. DALRD3 is located on chromosome 3 in the same region as WDR6. DALRD3, when fully disrupted, is implicated in a form of epileptic encephalopathy with associated developmental delay57. Finally, a third gene in the 3p21 locus, CELSR3 (cadherin EGF LAG seven-pass G type receptor 3), encodes a protocadherin that is highly expressed in the developing basal ganglia58. Four other genes identified through these analyses are located in the MHC locus, a region on chromosome 6 that has a major role in the adaptive immune system and has been repeatedly linked to major psychiatric disorders61. Genetic pleiotropy may underlie this connection, with variants predisposing individuals to both autoimmune conditions and OCD65. Furthermore, some OCD subtypes, such as pediatric acute-onset neuropsychiatric disorders associated with Streptococcus and pediatric acute-onset neuropsychiatric syndrome, may have autoimmune origins66,67. Nevertheless, we were surprised to discover several negative genetic correlations between OCD and autoimmune disorders such as Crohn's disease, ulcerative colitis and inflammatory bowel disease in our analyses, suggesting that there is heterogeneity (and perhaps pleiotropy) in the genetic relationships between autoimmune disorders and OCD. These findings are in line with traditional neural circuitry models of OCD, which focus on frontal cortical–striatal pathways68,69. Interestingly, the frontal and anterior cingulate cortices, which were enriched in our tissue-based analyses, as well as the hippocampus and the striatum, which were implicated in our cell type-based analyses, are among the regions that are consistently implicated in neuroimaging studies of OCD41,71,72,73. However, MSNs are also enriched in major depressive disorder75, schizophrenia76 and intelligence77, suggesting that the observed enrichment is not specific for OCD. Our analyses of the shared genetic risk between OCD and other psychiatric disorders provides further insights into the etiology of OCD. The strongest genetic correlations were observed for anxiety disorders, depression and anorexia nervosa, all of which are highly comorbid with OCD79. This aligns with previous findings from cross-disorder analyses suggesting a shared genetic susceptibility among most psychiatric disorders38,80,81. A notable exception is our finding that risk variants for OCD are protective for alcohol dependence82, which is at odds with epidemiological evidence strongly linking OCD and alcohol-related disorders83 but in line with a recent paper79 reporting a lower-than-expected lifetime comorbidity of substance use disorders in OCD. In both categories, the patterns of genetic correlations appear to follow a gradient across disorders and traits. 20), where few to no genome-wide significant loci are identified10,12,51,85, to the linear phase of SNP discovery, where even relatively small increases in sample size identify additional genome-wide significant loci18. Owing to the nature of our study, imputation references used in the different cohorts were heterogeneous and did not allow for confident analysis of rare variant associations. Future larger-scale sequencing studies that are currently underway will be needed to identify associations in this allele frequency spectrum. We also note that the genetic correlation analyses are impacted by residual heterogeneity in genetic signals owing to the employment of heterogeneous ascertainment strategies. In summary, this work substantially advances the field of OCD genetics by identifying new OCD genetic risk loci and multiple credible candidate causal genes, including those expressed in brain regions and cell types previously implicated in OCD86. The observation that common variants explain only a modest amount of the phenotypic variation in OCD suggests that other types of genetic variation may also contribute to the etiology of OCD. Notably, whole-exome-sequencing studies have suggested that a substantial proportion of OCD cases (22%) may be influenced by rare de novo coding variants87, especially in genes that are intolerant to loss of function88. Similarly, rare potentially damaging copy number variations represent part of the risk architecture for OCD9. These findings emphasize the need for a comprehensive exploration of the contribution of both common and rare genetic factors as well as their interplay to OCD risk. Finally, with the implication of the MHC complex, we provide additional evidence for potential shared genetic influences underlying both OCD and increased liability to autoimmune processes, although the directionality of those relationships remains to be definitively elucidated. Finally, with the emergence of drug databases describing the relations between drugs and molecular phenotypes89, our results may be useful for drug repurposing (that is, identifying existing drugs targeting OCD risk genes), leading to new opportunities to find more effective treatments. All relevant ethics approvals have been obtained by the respective cohort's institutions, and a list of all respective approvals can be found in Supplementary Note 2. Supplementary Table 1 provides an overview of the individual cohorts. 93) or ICD-10 (https://icd.who.int/) criteria for OCD as assessed by a healthcare professional or derived from (electronic) health records, while the remaining 32,233 cases were based on self-reported OCD diagnosis (23andMe, AGDS and parts of UKBB). Cohort-specific sample and analytic details can be found in Supplementary Note 2. Data collections were approved by the relevant institutional review boards at all participating sites, and all participants provided written informed consent. First, the data of each participating cohort were analyzed individually (see Supplementary Note 2 for details). The resulting GWAS summary statistics were then harmonized before a conjoint meta-analysis of all autosomes was conducted. Each summary statistic dataset was transformed to the ‘daner' file format following RICOPILI96 specifications. All variants had to meet the following criteria for inclusion: minor allele frequency (MAF) > 1% in cases and controls, INFO score > 0.8 and <1.2. Furthermore, strand-ambiguous A/T and C/G SNPs were removed if their MAF was >0.4. In the case that A/T and C/G SNPs showed a MAF < 0.4, allele frequencies were compared to frequencies in the HRC reference. If an allele mismatch was found, that is, the allele had a frequency > 0.5 in the HRC reference, it was assumed that alleles were reported on different strands, and alleles were flipped subsequently. Marker names were uniformly switched to those present in the HRC reference. Inverse-variance-weighted meta-analysis was conducted on 28 European cohorts using METAL97. Weighting was based on standard error primarily to account for the large case–control imbalances in cohorts that used linear mixed model approaches in their primary GWAS. Heterogeneity was assessed with Cochran's Q statistic and the I2 statistic98,99 (see Supplementary Note 5 for details). The genomic control factor lambda (λ) was calculated for each individual GWAS and for the overall meta-analysis to identify residual population stratification or systematic technical artifacts. The 23andMe data included information on the X chromosome; as this information was not present for all other cohorts, analysis of the X chromosome was only conducted in this subcohort (see Supplementary Note 4 for details). We further conducted GWAS meta-analyses on the following four subgroups, defined by differences in their sample ascertainment: (1) clinical OCD cases diagnosed by a healthcare professional in a clinical setting (ncases = 9,089, ncontrols = 21,077; including IOCDF, IOCDF_trio, EPOC, NORDiC-nor, NORDiC-swe, EGOS, OCGAS, OCGAS-ab, OCGAS-gh, OCGAS-nes, Psych_Broad, WWF, MVP, Michigan/Toronto IGS, YalePenn, Chop, CoGa), (2) comorbid individuals who were primarily ascertained for another comorbid psychiatric disorder (ncases = 5,266, ncontrols = 43,760; AGDS, iPSYCH), (3) biobank data from large-scale biobanks or registries with ICD or DSM codes (ncases = 9,138, ncontrols = 1,049,776; BioVU, EstBB, FinnGen, HUNT, MoBa, UKBB) or (4) 23andMe data (ncases = 30,167, ncontrols = 929,804). While these groups are not exclusive (for example, diagnoses in health records were originally given in a clinical setting or comorbid cases were also assessed in a clinical setting or derived from health records), we defined these groups by the cohort's primary characteristic. We also conducted one meta-analysis including all clinical, comorbid and biobank subgroups, while excluding the 23andMe data, resulting in 23,493 cases and 1,114,613 controls. We performed a conditional and joint analysis (GCTA-COJO)14 to identify independent signals within significant OCD loci. This approach performs a conditional and joint analysis on the basis of conditional P values before calculating the joint effects of all selected SNPs. We used the stepwise model selection procedure to select independently associated SNPs. The distance assumed for complete linkage disequilibrium was 10 Mb, and we used the default P-value threshold of \(5\times {10}^{-8}\) to define a genome-wide significant hit. Through this approach, we aimed to address potential concerns about heterogeneity in genetic liability for individual subgroups following different ascertainment strategies. MTAG is a multi-trait analysis that is usually used to combine different but related traits into one meta-analysis by leveraging the shared heritability among the different traits and thereby gaining power. In this case, our aim was to generate ascertainment-specific estimates, while boosting power by leveraging the high shared heritability between the subgroups. The MTAG analysis resulted in four different GWAS summary statistics, one for each subgroup (clinical, comorbid, biobanks, 23andMe). First, we ran a common factor model without individual SNP effects, following the tutorial ‘Models without individual SNP effects' on the GenomicSEM GitHub website (Code availability). Second, we ran a multivariate GWAS of the common factor (see Supplementary Note 5 for details). This ensures that we capture how much of each subgroup contributes to the latent factor. GenomicSEM also generates \({Q}_{{\rm{SNP}}}\) values, which indicate possible heterogeneous effects across the subgroups. To omit a downward bias in our estimates of liability-scale heritability, following Grotzinger et al.102, we accounted for varying levels of ascertainment across cohorts in our meta-analysis by summing the effective sample sizes across the contributing cohorts and using that as the input sample size for LDSC. We used cross-trait LDSC31, a method that computes genetic correlations between GWASs without bias from ancestry differences or sample overlap to calculate genetic correlations between the primary OCD meta-analysis and other phenotypes of interest. The genome-wide linkage disequilibrium information used by these methods was based on European populations from the HapMap 3 reference panel101, and GWAS summary statistics were filtered to only include SNPs that were part of the 1,290,028 HapMap 3 SNPs. To ensure the internal consistency of the datasets included in our meta-analysis, we calculated genetic correlations between all cohorts we considered to have a sample size large enough for LDSC (effective sample size of ≥1,000) and between the four ascertainment-specific subgroups. To match the significant SNPs to the genes for which they likely influence function, we conducted a series of positional and functional gene-mapping analyses. The positional mapping employed MBAT-combo19, while the functional mapping tested whether genetic variants associated with OCD were also associated with differential expression of nearby genes (within a 1-Mb window) using (1) TWAS20 using PsychENCODE data and included colocalization with COLOC27,28, and (2) SMR22 using whole-blood eQTL information and brain tissues from MetaBrain, alongside the HEIDI test, which tests for heterogeneity in GWAS signal and eQTL association. As a final step, genes within each locus were prioritized using PsyOPS26, which integrates both positional and functional information. A gene-based analysis was conducted using mBAT-combo19 within GCTA version 1.94.1 (ref. A gene list consisting of 19,899 protein-coding genes was used to map the base pair position of genes using genome build hg19 (see Supplementary Note 7 for details). TWAS FUSION uses reference linkage disequilibrium and reference gene expression panels with GWAS summary statistics to estimate the association between gene expression and OCD risk. We corrected for multiple testing using Bonferroni correction. We performed colocalization analyses using the COLOC R function27,28 implemented in TWAS FUSION. Colocalization is a Bayesian method used to calculate the posterior probabilities (PP) that individual lead SNPs within a significant TWAS locus are (1) independent (for example, two causal SNPs in linkage disequilibrium, one affecting transcription and one affecting OCD; PP3) or (2) share the same associated variant (for example, a single causal SNP affects both transcription and OCD (PP4)). This was performed using the FUSION software, which jointly estimates the effect of all significant features within each locus by using residual SNP associations with OCD after accounting for the predicted expression of other features. SMR22 was performed using default settings and eQTL meta-analysis summary statistics from European populations for whole blood from eQTLGen23 and all five nervous system tissues from MetaBrain (basal ganglia, cerebellum, cortex, hippocampus and spinal cord)104. The HEIDI test was performed alongside SMR to test for effect size heterogeneity between the GWAS and eQTL summary statistics. We used the gene prioritization method PsyOPS26 to rank genes within genome-wide significant loci. This supervised approach integrates biological annotations on mutational intolerance, brain-specific expression and involvement in neurodevelopmental disorder for genes within significant loci. We performed a PWAS using protein expression data from human brain samples. The ROS/MAP proteomes were generated from the dlPFC of 376 participants of European ancestry and included 1,476 proteins with significant SNP-based heritability (P < 0.01). The Banner PWAS weights were generated from 152 individuals of European ancestry and included 1,147 proteins with significant SNP-based heritability. We corrected for multiple testing using Bonferroni correction. We created a list of prioritized genes using both gene-based tests and colocalization–HEIDI filters. Results from each gene-based test were first restricted to protein-coding genes with unique gene identifiers based on the release from GENCODE (version 40) for hg19. The following criteria were then used to prioritize genes: (1) a significant (Bonferroni-corrected) association from at least two gene-based tests (mBAT-combo, TWAS FUSION, SMR or PsyOPS) and (2) evidence of colocalization (COLOC PP4 > 0.8) and/or significant SMR association with HEIDI P > 0.05. Joint–conditional tests of association and significant PWAS associations were used as ancillary approaches to further annotate the prioritized gene list. An analysis of tissue and cell type enrichment of OCD GWAS association signals was conducted using MAGMA (version 1.08)105 and partitioned LDSC106. We used the previously described approach29 to determine gene expression specificity in bulk tissue RNA-seq data from 37 tissues in GTEx (version 8) and single-cell RNA-sequencing data from 19 regions in the mouse central and peripheral nervous systems30. The analysis was limited to protein-coding genes with 1:1 orthologs between mice and humans. Gene expression in each tissue or cell type was calculated relative to total expression across all tissues or cell types. Enrichment analysis was performed on genes with the top 10% specificity values in each tissue or cell type, as previously defined29. To evaluate the enrichment of tissue- and cell type-specific genes in OCD genetic association signals, we applied MAGMA and partitioned LDSC. We restricted the analysis to summary statistics for SNPs with a high INFO score (>0.6) and frequency in the entire cohort (MAF > 0.01). Using MAGMA (version 1.08), we tested whether genes with the top 10% specificity in a tissue or cell type showed enrichment in gene-level genetic associations for OCD, with the 1000 Genomes Phase 3 European sample genotypes serving as the linkage disequilibrium reference panel. We used standard gene boundaries (35 kb upstream of the transcription start site to 10 kb downstream of the transcription stop site). Partitioned LDSC was used to examine whether SNPs within 100-kb regions of the top 10% specifically expressed genes were enriched for SNP-based heritability for OCD. All results were corrected for multiple testing with an FDR threshold of 0.05. Multiple resources were used to identify previously reported associations of our 30 significant SNPs with other phenotypes. CAUSALdb estimates causal probabilities of all genetic variants in GWAS significant loci using three state-of-the-art fine-mapping tools including PAINTOR, CAVIARBF and FINEMAP107,108,109,110. We performed a lookup of SNPs identified to be significantly associated with OCD-related phenotypes in previous GWASs. Note that this is not an independent replication, as previous studies partially overlap with the cohorts included in this GWAS. We performed a bidirectional lookup, assessing (1) whether gene findings from our GWAS showed evidence for rare variant involvement and (2) vice versa, whether findings from rare variant testing showed evidence of common variant association in our GWAS. First, we comprehensively assessed the overlap between 251 genes that we highlighted in our study as carrying common risk variation for OCD (Supplementary Table 14) and current gene-based summary statistics from OCD exome-sequencing data. We used data from Halvorsen et al.88 because it is the largest published exome-sequencing study of OCD presently. The supplementary materials from that paper include de novo variant calls from 771 case trios and 1,911 controls (supplementary table 14 in ref. We compared the burden of de novo variants, partitioned by variant annotation (synonymous, missense, loss of function) in trio cases versus trio controls within these 251 GWAS-prioritized genes. As described previously88, we only included de novo variants that were in loci well covered in both case and control data (In_Jointly_Covered_Loci==TRUE). We also excluded all calls from quartet samples in ref. For each of the four variant annotation classes, we compared the proportion of cases that had at least one qualifying de novo variant to the proportion of controls using a two-sided Fisher's exact test. Second, as Halvorsen et al.88 describe an overall excess of loss-of-function variants in OCD cases relative to controls specifically within loss-of-function intolerant genes (supplementary table 13 in ref. We looked up 200 genes with a probability of loss-of-function intolerance > 0.995 (derived from ref. We further tested for a difference in the proportion of these pLI > 0.995 genes with effect size estimate > 1 versus ≤1 within the set of genes highlighted in the OCD GWAS (n = 251) versus outside this set using a two-sided Fisher's exact test. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. In line with 23andMe regulations, 10,000 SNPs from the full GWAS including 23andMe are also being made available at https://www.med.unc.edu/pgc/download-results/. Datasets will be made available at no cost for academic use. MVP summary statistics are made available through dbGAP request under accession https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v12.p1 phs001672.v12.p1. Core analysis code for RICOPILI can be found at https://sites.google.com/a/broadinstitute.org/ricopili/. GenomicSEM, specifically the tutorial ‘Models without Individual SNP effects' can be found here: https://github.com/GenomicSEM/GenomicSEM/wiki/3.-Models-without-Individual-SNP-effects. PWAS: for access to the protein weights, see https://www.synapse.org/#!Synapse:syn24872746. GCTA (mBAT-combo and COJO) can be found at https://yanglab.westlake.edu.cn/software/gcta/#Overview. LDSC and partitioned heritability can be found at https://github.com/bulik/ldsc. Fawcett, E. J., Power, H. & Fawcett, J. M. Women are at greater risk of OCD than men: a meta-analytic review of OCD prevalence worldwide. The Global Burden of Disease: 2004 Update (WHO Press, 2008). Suicide in obsessive–compulsive disorder: a population-based study of 36 788 Swedish patients. Meier, S. M. et al. Mortality among persons with obsessive–compulsive disorder in Denmark. The genetic epidemiology of obsessive–compulsive disorder: a systematic review and meta-analysis. Burton, C. L. et al. Heritability of obsessive–compulsive trait dimensions in youth from the general population. van Grootheest, D. S., Cath, D. C., Beekman, A. T. & Boomsma, D. I. Twin studies on obsessive–compulsive disorder: a review. Revealing the complex genetic architecture of obsessive–compulsive disorder using meta-analysis. Davis, L. K. et al. Partitioning the heritability of Tourette syndrome and obsessive compulsive disorder reveals differences in genetic architecture. Strom, N. I. et al. Genome-wide association study identifies new locus associated with OCD. Strom, N. I. et al. Genome-wide association study identifies new loci associated with OCD. Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Beyond SNP heritability: polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model. Grotzinger, A. D. et al. Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. Strom, N. I. et al. Genome-wide association study of major anxiety disorders in 122,341 European-ancestry cases identifies 58 loci and highlights GABAergic signaling. Li, A. et al. mBAT-combo: a more powerful test to detect gene–trait associations from GWAS data. Gusev, A. et al. Integrative approaches for large-scale transcriptome-wide association studies. Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Qi, T. et al. Identifying gene targets for brain-related traits using transcriptomic and methylomic data from blood. Brain proteome-wide association study implicates novel proteins in depression pathogenesis. Predicting causal genes from psychiatric genome-wide association studies using high-level etiological knowledge. Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. Wallace, C. Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses. Bryois, J. et al. Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson's disease. Zeisel, A. et al. Molecular architecture of the mouse nervous system. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder. Demontis, D. et al. Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains. Hall, L. S. et al. Genome-wide meta-analyses of stratified depression in Generation Scotland and UK Biobank. Als, T. D. et al. Depression pathophysiology, risk prediction of recurrence and comorbid psychiatric disorders using genome-wide analyses. Minimal phenotyping yields genome-wide association signals of low specificity for major depression. Derks, E. M., Thorp, J. G. & Gerring, Z. F. Ten challenges for clinical translation in psychiatric genetics. Baselmans, B. M. L. et al. Multivariate genome-wide analyses of the well-being spectrum. Shephard, E. et al. Toward a neurocircuit-based taxonomy to guide treatment of obsessive–compulsive disorder. Yanagisawa, M. et al. A p120 catenin isoform switch affects Rho activity, induces tumor cell invasion, and predicts metastatic disease. The catenin p120ctn interacts with Kaiso, a novel BTB/POZ domain zinc finger transcription factor. Vandervore, L. V. et al. TMX2 is a crucial regulator of cellular redox state, and its dysfunction causes severe brain developmental abnormalities. Mattheisen, M. et al. Genome-wide association study in obsessive–compulsive disorder: results from the OCGAS. Watanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits. Nagel, M., Watanabe, K., Stringer, S., Posthuma, D. & van der Sluis, S. Item-level analyses reveal genetic heterogeneity in neuroticism. Association of LKB1 with a WD-repeat protein WDR6 is implicated in cell growth arrest and p27Kip1 induction. Adams, D. M., Reay, W. R. & Cairns, M. J. Multiomic prioritisation of risk genes for anorexia nervosa. Identification of candidate Parkinson disease genes by integrating genome-wide association study, expression, and epigenetic data sets. Lentini, J. M., Alsaif, H. S., Faqeih, E., Alkuraya, F. S. & Fu, D. DALRD3 encodes a protein mutated in epileptic encephalopathy that targets arginine tRNAs for 3-methylcytosine modification. Willsey, A. J. et al. De novo coding variants are strongly associated with Tourette disorder. Debnath, M., Berk, M., Leboyer, M. & Tamouza, R. The MHC/HLA gene complex in major psychiatric disorders: emerging roles and implications. Mataix-Cols, D. et al. A total-population multigenerational family clustering study of autoimmune diseases in obsessive–compulsive disorder and Tourette's/chronic tic disorders. Tylee, D. S. et al. Genetic correlations among psychiatric and immune-related phenotypes based on genome-wide association data. Westwell-Roper, C. et al. Immune-related comorbidities in childhood-onset obsessive compulsive disorder: lifetime prevalence in the Obsessive Compulsive Disorder Collaborative Genetics Association Study. Swedo, S. E. et al. Pediatric autoimmune neuropsychiatric disorders associated with streptococcal infections: clinical description of the first 50 cases. Wilbur, C. et al. PANDAS/PANS in childhood: controversies and evidence. Structural neuroimaging biomarkers for obsessive–compulsive disorder in the ENIGMA-OCD consortium: medication matters. Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium. Trans-ancestry genome-wide study of depression identifies 697 associations implicating cell types and pharmacotherapies. Trubetskoy, V. et al. Mapping genomic loci implicates genes and synaptic biology in schizophrenia. Savage, J. E. et al. Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence. Exploring the genetic overlap between twelve psychiatric disorders. Sharma, E. et al. Comorbidities in obsessive–compulsive disorder across the lifespan: a systematic review and meta-analysis. Lee, P. H. et al. Genomic relationships, novel loci, and pleiotropic mechanisms across eight psychiatric disorders. Walters, R. K. et al. Transancestral GWAS of alcohol dependence reveals common genetic underpinnings with psychiatric disorders. Stewart, S. E. et al. Genome-wide association study of obsessive–compulsive disorder. Piantadosi, S. C. et al. Transcriptome alterations are enriched for synapse-associated genes in the striatum of subjects with obsessive–compulsive disorder. Cappi, C. et al. De novo damaging DNA coding mutations are associated with obsessive–compulsive disorder and overlap with Tourette's disorder and autism. Halvorsen, M. et al. Exome sequencing in obsessive–compulsive disorder reveals a burden of rare damaging coding variants. Wang, J. et al. CAUSALdb: a database for disease/trait causal variants identified using summary statistics of genome-wide association studies. The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Diagnostic and Statistical Manual of Mental Disorders: DSM-5 237–242 (American Psychiatric Association, 2013). McCarthy, S. et al. A reference panel of 64,976 haplotypes for genotype imputation. Lam, M. et al. RICOPILI: Rapid Imputation for Consortias Pipeline. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. Turley, P. et al. Multi-trait analysis of genome-wide association summary statistics using MTAG. Integrating common and rare genetic variation in diverse human populations. Grotzinger, A. D., de la Fuente, J., Privé, F., Nivard, M. G. & Tucker-Drob, E. M. Pervasive downward bias in estimates of liability-scale heritability in genome-wide association study meta-analysis: a simple solution. Wang, D. et al. Comprehensive functional genomic resource and integrative model for the human brain. de Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Benner, C. et al. FINEMAP: efficient variable selection using summary data from genome-wide association studies. Chen, W. et al. Fine mapping causal variants with an approximate Bayesian method using marginal test statistics. Integrating functional data to prioritize causal variants in statistical fine-mapping studies. Kichaev, G. & Pasaniuc, B. Leveraging functional-annotation data in trans-ethnic fine-mapping studies. A list of members of the 23andMe Research Team, HUNT, CoGa and the MVP who contributed to this study can be found in Supplementary Note 1. We thank the research participants and employees of 23andMe for making this work possible. Genotype quality control and imputation were conducted by the K.G. NORDiC is funded by National Institute of Mental Health (NIMH) R01 MH110427 (PI J.J.C. We thank the collection team that worked to recruit them: A. Juréus, J. Pege, M. Rådström, R. Satgunanthan-Dawoud, M. Krestelica and B. Ohlander, as well as the data manager B. Iliadou. The AGDS was primarily funded by National Health and Medical Research Council (NHMRC) of Australia grant 1086683. This work was further supported by NHMRC grants 1145645, 1078901 and 1087889. Communications for their effort in promoting the study. We also acknowledge D. Whiteman and C. Olsen from QSkin. EGOS was supported by a grant from the Beatrice and Samuel A. Seaver Foundation to D.E.G. LifeGene was supported by the Swedish Research Council, Karolinska Institutet–Stockholm County Council research grants, AFA Insurance and the Torsten and Ragnar Söderbergs Foundation. GENOS was supported by the German Research Foundation (GR 1912/1-1). The OCD Collaborative Genetics Association Study (OCGAS) is a collaborative research study and was funded by the following NIMH grant numbers: MH071507, MH079489, MH079487, MH079488 and MH079494. This work (OCGAS and IOCDF) is supported by the Netherlands Organization for Scientific Research—Gravitation project ‘BRAINSCAPES: a Roadmap from Neurogenetics to Neurobiology' (024.004.012) and the European Research Council advanced grant ‘From GWAS to Function' (2018-ADG 834057). The OCGAS and IOCDF samples are supported through NIMH grant numbers MH071507 (G.N. High-performance computer capacity for handling and statistical analysis of iPSYCH data in the GenomeDK HPC facility was provided by the Center for Genomics and Personalized Medicine and the Centre for Integrative Sequencing, iSEQ, Aarhus University, Denmark (grant to A.D.B.). BioVU: CTSA (S.D., Vanderbilt Resources) was supported by the National Center for Research Resources, grant UL1 RR024975-01 and is now at the National Center for Advancing Translational Sciences (grant 2 UL1 TR000445-06). The dataset(s) used for the analyses (BioVU) described were obtained from Vanderbilt University Medical Center's BioVU, which is supported by numerous sources: institutional funding, private agencies and federal grants. S.A. acknowledges a Miguel Servet contract (CP22/00026) awarded by the Instituto de Salud Carlos III and cofunded by the European Union Fund: Fondo Social Europeo Plus, FSE+. HYPERGENES and InterOmics cohorts provided controls of Italian origin for the present study. J.C.-D. acknowledges her contract from the Network Center for Biomedical Research (CIBER). acknowledges the Clinical Investigation Centre, Robert Debré Hospital. A. Havdahl was also supported by South East Norway Health Authority (2020022). A. Abdellaoui was supported by the Foundation Volksbond Rotterdam. was supported by NHMRC project grant 1145645 and the University of Queensland Health Research Accelerator Program. was supported by the Italian Ministry of Health grant RC18-19-20-21/A. K. Hagen was supported by the Trond Mohn Foundation. Fabrizio and Federica Piras are supported by the Italian Ministry of Health RC18-19-20-21/A grant. This work was in part supported by the German Research Foundation (DFG) grants RA1971/8-1 and RA1971/7-1 and by the Bundesministerium für Bildung und Forschung grant 01ED2007A to A. Ramirez. was supported by the NIMH (OCGAS and OCGS). is supported by an Australian NHMRC Investigator Grant (APP1172917). is supported by the Italian Ministry of Health RC18-19-20-21/A grant. collected data as part of the NIH grant 1R01MH093381. (MoBa) has received grant support from RCN (324499, 273291, 262656, 248778, 223273), KG Jebsen Stiftelsen and NordForsk 164218. J. Kaprio has been supported by the Academy of Finland (grant 336823). was supported by grants from the NIH including R01NS102371, R01MH113362, R01MH118223, R01NS105746 and R56MH120736. was supported by an NIH Training Grant in Human Genetics (2T32GM080178). has been funded by the US Veterans Affairs Administration. These authors contributed equally: Nora I. Strom, Zachary F. Gerring, Marco Galimberti, Dongmei Yu. These authors jointly supervised this work: Jeremiah M. Scharf, Murray B. Stein, Joel Gelernter, Carol A. Mathews, Eske M. Derks, Manuel Mattheisen. Nora I. Strom, Jakob Grove, Anders D. Børglum & Manuel Mattheisen Department of Mental Health and Neuroscience, Translational Neurogenomics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Department of Psychiatry, Human Genetics, Yale University, New Haven, CT, USA Department of Center for Genomic Medicine, Psychiatric and Neurodevelopmental Genetics Unit, Massachusetts General Hospital, Boston, MA, USA Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Center for Research in Molecular Medicine and Chronic Diseases (CiMUS), Genomics and Bioinformatics, University of Santiago de Compostela, Santiago de Compostela, Spain National Institute for Health and Care Research Maudsley Biomedical Research Centre, South London and Maudsley NHS Trust, London, UK Behrang Mahjani, Michael Breen, Joseph D. Buxbaum, Magdalena Janecka, Laura G. Sloofman, Sven Sandin & Dorothy E. Grice Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden Behrang Mahjani, Christina Hultman, Nancy L. Pedersen, Cynthia M. Bulik, Mikael Landén & Sven Sandin Mental Health and Neuroscience Program, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Department of Neurosciences and Mental Health, Hospital for Sick Children, Toronto, Ontario, Canada Second Opinion Outpatient Clinic, GGNet, Warnsveld, the Netherlands Psychiatric Genetics Unit, Group of Psychiatry, Mental Health and Addiction, Vall d'Hebron Research Institute, Universitat Autònoma de Barcelona, Barcelona, Spain Silvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta Ribasés Department of Mental Health, Hospital Universitari Vall d'Hebron, Barcelona, Spain Silvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta Ribasés Biomedical Network Research Centre on Mental Health (CIBERSAM), Madrid, Spain Silvia Alemany, Judit Cabana-Dominguez, María Soler Artigas & Marta Ribasés Christine Andre, Brian P. Brennan, Jesse Crosby, Martha J. Falkenstein, Lauryn Garner, Christina Gironda, Eric Jenike, Kara Kelley, Brittany Mathes, Sriramya Potluri & Eric Tifft Laboratory of Neuropsychiatry, IRCCS Santa Lucia Foundation, Rome, Italy Department of Child Study Center and Psychiatry, Yale University, New Haven, CT, USA Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Department of Psychiatry, Harvard Medical School, Boston, MA, USA Brian P. Brennan, Jesse Crosby, Martha J. Falkenstein, Dan A. Geller & David L. Pauls Department for Congenital Disorders, Statens Serum Institut, Copenhagen, Denmark Department of Psychiatry and Behavioral Sciences, Johns Hopkins Medical Institutions, Baltimore, MD, USA Child and Adolesccent Psychiatry Department, APHP, Paris, France Psychiatry, McLean Hospital OCDI, Harvard Medical School, Belmont, MA, USA Adult Psychological Services, CBTeam LLC, Lexington, MA, USA qGenomics (Quantitative Genomics Laboratories), Esplugues de Llobregat, Spain Department of Medical Epidemiology and Biostatistics, Center for Eating Disorders Innovation, Karolinska Institutet, Stockholm, Sweden Department of Psychiatry, Johns Hopkins University, Baltimore, MD, USA The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark Genetic Epidemiology Research Branch, National Institute of Mental Health, Bethesda, MD, USA Department of Psychiatry, Møre og Romsdal Hospital Trust, Molde, Norway Department of Mental Health, Norwegian University for Science and Technology, Trondheim, Norway Nic Waals Institute, Lovisenberg Diaconal Hospital, Oslo, Norway Department of Medical Genetics, Dr. Einar Martens Research Group for Biological Psychiatry, Haukeland University Hospital, Bergen, Norway Department of Psychiatry and Behavioral Sciences, Johns Hopkins University, Baltimore, MD, USA Kristi Krebs, Andres Metspalu, Tõnu Esko, Reedik Mägi, Mari Nelis, Georgi Hudjashov & Lili Milani Department of Psychiatry, Yale University, West Haven, CT, USA COBRE Center for Neuromodulation, Butler Hospital, Providence, RI, USA Department of Psychiatry and Human Behavior, Alpert Medical School, Brown University, Providence, RI, USA Department of Psychiatry, Dalhousie University, Halifax, Nova Scotia, Canada Department of Psychiatry and Behavioral Science, Johns Hopkins University, Baltimore, MD, USA Department of Psychiatry, Massachusetts General Hospital, Boston, MA, USA Psychiatric and Neurodevelopmental Genetics Unit, Harvard Medical School, Boston, MA, USA Department of Clinical Neuroscience and Neurorehabilitation, Neuropsychiatry Laboratory, IRCCS Santa Lucia Foundation, Rome, Italy CIBERER, Centro de Investigación Biomédica en Red, Madrid, Spain Department of Psychiatry and Glenn Biggs Institute for Alzheimer's and Neurodegenerative Diseases, UT Health San Antonio, San Antonio, TX, USA Department of Psychiatry, McLean Hospital, Harvard Medical School, Belmont, MA, USA Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA, USA Site Berlin–Potsdam, German Center for Mental Health (DZPG), Berlin, Germany Department of Clinical and Molecular Medicine, NTNU, Trondheim, Norway Jebsen Center for Genetic Epidemiology, NTNU, Trondheim, Norway Department of Molecular Medicine and Surgery, CMM, Karolinska Institutet, Stockholm, Sweden OCD Institute, Division of Depression and Anxiety, McLean Hospital, Belmont, MA, USA Department of Psychiatry, Harvard Medical School, Belmont, MA, USA Human Genetics Institute of New Jersey, Rutgers University, Piscataway, NJ, USA Laboratory of Clinical Science, NIMH Intramural Research Program, Bethesda, MD, USA Department of Psychiatry, Veterans Affairs Connecticut Healthcare System, West Haven, CT, USA Department of Psychiatry, OCD Clinical and Research Unit, Bellvitge Hospital, Barcelona, Spain Department of Psychiatry and Mental Health, Bellvitge Biomedical Research Institute IDIBELLL, Barcelona, Spain CIBERSAM, Mental Health Network Biomedical Research Center, Madrid, Spain Department of Specialized Training, Drenthe Mental Health Care Institute, Groningen, the Netherlands Department of Psychiatry and Psychotherapy, University Hospital LMU, Munich, Germany Child Study Center and Psychiatry, Yale School of Medicine, New Haven, CT, USA Department of Psychiatry, Columbia University Medical Center, New York, NY, USA Department of Medicine, Mass General Brigham, Boston, MA, USA COBRE Center on Neuromodulation, Butler Hospital, Providence, RI, USA Department of Psychiatry and Human Behavior, Alpert Medical School, Brown University, Providence, RI, USA Department of Addictology and Psychiatry, Université Paris-Est Créteil, AP-HP, Inserm, Paris, France Department of Mental Health, Psychiatric Genetics, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA Psiquiatría, Instituto Nacional de Medicina Genómica, Mexico City, México Department of Psychiatry, Yale University, New Haven, CT, USA Department of Psychiatry, Hospital Universitari Vall d'Hebron, Barcelona, Spain Group of Psychiatry, Mental Health and Addictions, Psychiatric Genetics Unit, Vall d'Hebron Research Institute, Barcelona, Spain Department of Psychiatry, Sunnybrook Health Sciences Centre, Toronto, Ontario, Canada Department of Psychiatry and Behavioral Neurosciences, Child and Adolescent Psychiatry, Wayne State University School of Medicine, Detroit, MI, USA Department of Psychiatry and Neuroscience Institute, SAMRC Unit on Risk and Resilience in Mental Disorders, University of Cape Town, Cape Town, South Africa Department of Genetic Epidemiology, QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Program in Genetics and Genome Biology, Hospital for Sick Children, Toronto, Ontario, Canada PsychGen Center for Genetic Epidemiology, Norwegian Institute of Public Health, Oslo, Norway Department of Psychiatry, Veterans Affairs Connecticut Healthcare Center, West Haven, CT, USA Department of Mental Health and Neuroscience, QIMR Berghofer, Brisbane, Queensland, Australia You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Sealock, T.B., J.R.C., B. Mahjani, J.G.T., K.B., C.L.B., J.J.L., G.Z., S.A., C.A., K.D.A., J.B., N.B., C.B., J.B.N., O.J.B., D.B., M.H.B., S.B., R.B., M.B., B.P.B., H.B., J.D.B., J.B.-G., E.M.B., J.C.-D., B.C., A. Camarena, C.C., A. Carracedo, M.C., M.C.C., V.C., E.H.C., J.C., B.A.C., E.J.D.S., R.D., S.D., J.A.E., X.E., M.J.F., B.T.F., L.G., C. Gironda, F.S.G., M.A.G., J. Grove, W.G., J.H., K. Hagen, K. Harrington, A.H., K.D.H., A.G.H., D.H., C.H., M.J., E.J., E.K.K., K. Kelley, J. Klawohn, J.E.K., K. Krebs, C. Lange, N.L., D. Levey, K.L.-T., F.M., B. Maher, B. Mathes, E.M., N.M., N.C.M., S.M., E.C.M., M. Mulhem, P.S.N., E.L.N., K.S.O'C., L.O., O.T.O., T.P., N.L.P., Fabrizio Piras, Federica Piras, S.P., R.R., A. Ramirez, S. Rauch, A. Reichenberg, M.A. Riddle, S. Ripke, M.C.R., A.S.S., M.A.S., A.H.S., L.G.S., J.S., M.S.A., L.F.T., E.T., H.V., N.v.K., J.V.-V., N.N.V., C.P.W., Y.W., J.R.W., B.S.W., Y.Y., H.Z., A. Agrawal, P.A., G. Berberich, K.K.B., C.M.B., D.C., D.D., V.E., H.E., P.F., T.V.F., A.J.F., J.M.G., D.A.G., H.J.G., B.D.G., G.L.H., I.B.H., D.M.H., N.K., J. Kennedy, D. Lai, M. Landén, S.L.H., M. Leboyer, C. Lochner, J.T.M., S.E.M., P.B.M., B.M.N., H.N., M. Nordentoft, M.P., C. Pato, D.L.P., J.P., C. Pittenger, D.P., J.A.R.-Q., S.A.R., M.A. and M. Mattheisen provided samples and/or processed individual cohort data. is on the scientific advisory board for Myriad Genetics and the medical advisory board for the Tourette Association of America and received clinical trial funding from Emalex and Octapharma Pharmaceuticals. has received travel grants and speaker's honoraria from Fresenius Medical Care, Neuraxpharm, Servier and Janssen-Cilag as well as research funding from Fresenius Medical Care. The Brain and Mind Centre operates early-intervention youth services at Camperdown under contract to headspace. He is the chief scientific advisor to and a 3.2% equity shareholder in InnoWell, which aims to transform mental health services through the use of innovative technologies. is a member of the scientific advisory board at Deep Genomics and Neumora. C. Pittenger consults and/or receives research support from Biohaven Pharmaceuticals, Freedom Biosciences, Ceruvia Lifesciences, Transcend Therapeutics, UCB BioPharma and F-Prime Capital Partners. These relationships are not related to the current work. He was formerly a consultant for Brainsway and Biohaven Pharmaceuticals in the past 12 months. He owns stock less than $5,000 in nView–Proem for distribution related to the YBOCS scales. He receives book royalties from Elsevier, Wiley, Oxford, the American Psychological Association, Guildford, Springer, Routledge and Jessica Kingsley. D.M.-C. receives royalties for contributing articles to UpToDate and Wolters Kluwer Health and personal fees for editorial work from Elsevier, all unrelated to the current work. has in the past 3 years received consulting income from Acadia Pharmaceuticals, Big Health, Biogen, Bionomics, Boehringer Ingelheim, Clexio, Eisai, EmpowerPharm, Engrail Therapeutics, Janssen, Jazz Pharmaceuticals, NeuroTrauma Sciences, Otsuka, PureTech Health, Sage Therapeutics, Sumitomo Pharma and Roche–Genentech. has stock options in Oxeia Biopharmaceuticals and EpiVario. He has been paid for his editorial work on Depression and Anxiety (editor in chief), Biological Psychiatry (deputy editor) and UpToDate (co-editor in chief for psychiatry). J. Gelernter is paid for editorial work by the journal Complex Psychiatry. has received funding from Biohaven, Boston Scientific and Medtronic. All other authors report no conflicts of interest. Nature Genetics thanks Sarah Gagliano Taliun and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Strom, N.I., Gerring, Z.F., Galimberti, M. et al. Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorder. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            