
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-04</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a69631946/chernobyl-dogs-dna-rapid-evolution-science/'>The Dogs of Chernobyl Are Experiencing Rapid Evolution, Study Suggests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 14:28:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Animals of all kinds have thrived in humanity's absence. As the world's greatest nuclear disaster approaches its 40th anniversary, biologists are now taking a closer look at the animals located inside the Chernobyl Exclusion Zone (CEZ), which is about the size of Yosemite National Park, and investigating how decades of radiation exposure may have altered animals' genomes—and even, possibly, sped up evolution. “Do they have mutations that they've acquired that allow them to live and breed successfully in this region?” co-author Elaine Ostrander, a dog genomics expert at the National Human Genome Research Institute, told The New York Times. The idea of radiation speeding up natural evolution isn't a new one. Scientists have been analyzing certain animals living within the CEZ for years, including bacteria, rodents, and even birds. One study back in 2016 found that Eastern tree frogs (Hyla orientalis), which are usually a green color, were more commonly black within the CEZ. This made scientists ponder: could something similar be happening to Chernobyl's wild dogs? While this may seem to heavily imply that these dogs have undergone some type of rapid mutation or evolution due to radiation exposure, this study is only a first step in proving that hypothesis. However, one environmental scientist, speaking with Science News, says that these studies can be tricky business, largely due to the fact that sussing out radiation-induced mutations from other effects, like inbreeding, is incredibly difficult. And in the time since the original study was published, other researchers have dug into this question and come up with competing results. To establish a baseline for comparison, the team compared the genome of Chernobyl City dogs located 10 miles from the CEZ to dogs found in regions of Russia, Poland, and other nearby countries. Once they determined that the populations were genetically similar, they then used the Chernobyl City dogs as a representative control for their study. “We know that, for example, exposure to high doses of radiation can introduce instability from the chromosomal level on down,” Matthew Breen, senior author of the study from North Carolina State University, said in a press statement. “While this dog population is 30 or more generations removed from the one present during the 1986 disaster, mutations would likely still be detectable if they conferred a survival advantage to those original dogs. But we didn't find any such evidence in these dogs.” Despite a current lack of firm conclusions, the study has shown once again that an area that—by all rights—should be a wasteland has become an unparalleled scientific opportunity to understand radiation and its impact on natural evolution. Climbers Found Evidence of a Mass Panic Attack Bees Can Also Learn the Basics of Morse Code This Strange Blue Goo Contains Evidence of Life A Carnivorous ‘Death Ball' Emerged from the Deep</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69536299/wipe-out-worlds-deadliest-infectious-disease/'>Scientists Just Discovered How to Wipe Out the World's Deadliest Infectious Disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Caused by the bacteria Mycobacterium tuberculosis, it has seen a resurgence in the U.S., emerging in California, Kansas, Maine, North Carolina, and New York. Shelley Haydel—an infectious disease microbiologist from Arizona State University—and her research team have now found that the deadly bacterium has a molecular system it literally cannot live without. Haydel attacked it with CRISPR interference (CRISPRi), which can repress certain genes, and when PrrAB was switched off, the bacterium's life was over. “The essentiality of the PrrAB two-component system (TCS) in Mycobacterium tuberculosis positions it as a promising therapeutic target, prompting investigation into its potential as a universal drug target across mycobacterial species,” she said in a study recently published in American Chemical Society: Infectious Diseases. While not all mycobacteria cultured in Haydel's lab were taken down by the removal of PrrAB, it was effective against tuberculosis in vitro. PrrAB regulates genes necessary for respiration and oxidative phosphorylation, a process through which oxygen is reduced (gaining electrons and losing its oxidation state) to create bonds for the cell's primary energy molecule, adenosine triphosphate (ATP). (Considering how these bacteria infect the lungs, it's a bit ironic that eliminating them involves targeting their own respiratory system.) Diarylthiazole-48 (DAT-48)—previously known to be an experimental anti-tuberculosis compound—kills the bacteria by inhibiting PrrAB. Even without PrrAB repression, the researchers found that using DAT-48 together with some tuberculosis drugs (such as bedaquiline or telacebec) made it more potent, showing that the bacteria's susceptibility to DAT-48 is somehow linked to how PrrAB regulates its respiratory system. “DAT-48, an experimental anti-TB compound, acted within the PrrAB-regulated pathwayexhibiting potent activity against M. tuberculosis and synergy with respiratory inhibitors,” Haydel said. While these treatments have not been tested on humans yet, if they prove to be successful in vivo, tuberculosis might just go the way of smallpox and vanish. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Gut Feelings Are Memories From the Future, Scientists Say A Missing World War II Destroyer Has Been Found Scientists Just Upended the Age of Life on Earth</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251204083601.htm'>Simple nutrient mix delivers surprising autism breakthrough in mice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 13:43:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers led by Tzyy-Nan Huang and Ming-Hui Lin at Academia Sinica in Taiwan have found that a low-dose combination of zinc, serine, and branched-chain amino acids may ease behavioral difficulties in three mouse models of autism. Autism spectrum disorder (ASD) is associated with atypical neural development that disrupts how neurons form connections. Nutrition is also recognized as a factor that can influence ASD. Zinc, serine, and branched-chain amino acids have each been linked to potential improvements in neural connectivity. To test this, they worked with three mouse models of ASD and examined synapse-related protein levels, tracked amygdala activity using calcium imaging, and evaluated social behavior. Social behaviors improved as well, but only when the nutrients were given as a mixture; the same doses of individual supplements produced no measurable change. Yi-Ping Hsueh explained, "As hundreds of genes are implicated in autism, each with distinct molecular functions, a 'one gene-one therapy' approach is impractical for addressing the complexity of ASD. Our findings show that a low-dose nutrient mixture containing zinc, branched-chain amino acids (BCAAs), and serine -- working synergistically to improve synaptic function and social behaviors across three ASD mouse models -- offers a safer and more practical strategy for long-term, broad application, even beginning in childhood." It is exciting to see that combining these nutrients at low doses successfully restores synaptic proteomes and enhances social behaviors in three different mouse models of autism." Co-first author Ming-Hui Lin said, "I was thrilled to observe that just seven days of treatment with the nutrient mixture significantly modulated neuronal circuit activity and connectivity in real time. These results provide strong support for the beneficial effects of low-dose nutrient supplement combinations." This work was supported by grants from Academia Sinica, Taiwan (AS-IA-111-L01 to Y.-P.H.) Scientists Warn This Popular Cooking Oil May Be Quietly Fueling Weight Gain Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03955-z'>Aluminium is crucial to vaccines — and safe. Why are US advisers debating it?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 13:31:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Many childhood vaccines contain small amounts of aluminium to boost the immune response. Credit: GettyWhen US vaccine advisers meet this week, they will turn their spotlight on little-known ingredients that are crucial to the effectiveness of many vaccines: adjuvants. Many childhood vaccines contain small amounts of aluminium to boost the immune response. Adjuvants are compounds that boost immune responses, improving the ability of vaccines to elicit long-lasting immunity against infectious diseases. At this week's meeting, the US Advisory Committee on Immunization Practices (ACIP) is expected to discuss aluminium, an adjuvant widely used in inoculations against diphtheria, tetanus and hepatitis B, among others. Such adjuvants could prove crucial to the success of the next generation of vaccines against diseases including tuberculosis and malaria, and against viruses such as HIV, says Darrell Irvine, a vaccine immunologist at the Scripps Research Institute in La Jolla, California. “And adjuvants are likely going to be part of the solution.” Some adjuvants trigger low levels of inflammation at the injection site, thereby boosting recruitment of immune cells to where they are needed and encouraging the cells to linger there. Others are more specific, activating molecular signalling pathways in certain classes of immune cells. Aluminium is in the first category, and its vaccine-boosting properties have been known for more than a century, says Irvine. Even so, US health secretary Robert F. Kennedy Jr, who has long campaigned against vaccines, has argued that aluminium adjuvants are linked to autism and food allergies. The ACIP advisers, handpicked by Kennedy, will discuss the use of aluminium in vaccines this week, but they have not yet scheduled a vote. Measles makes a comeback: four charts show where and how 154 million lives and counting: 5 charts reveal the power of vaccines Will RFK Jr's vaccine agenda make America contagious again? Potent neutralization of Marburg virus by a vaccine-elicited antibody Postdoctoral scholarship in a project on microbial system characterization using OPTIR and Raman spectroscopy The Intramural Research Program (IRP), National Institute on Drug Abuse (NIDA) Measles makes a comeback: four charts show where and how 154 million lives and counting: 5 charts reveal the power of vaccines Will RFK Jr's vaccine agenda make America contagious again? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69561987/knight-elongated-skull/'>Archaeologists Discovered the Bizarre Skull of an Ancient Christian Knight</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Sometime between the 13th and 15th centuries, a knight suffered defeat in combat. It would be centuries before archaeologists unearthed what was left of his remains, including fragments of a coffin, scattered bones, and a skull that was unexpectedly elongated. Warrior monks in white robes joined forces with nobles and laypeople to drive out invaders. Biological anthropologist Carme Rissech from the University of Rovira i Virgili in Reus, Spain, had been excavating the Counts' Courtyard cemetery with her research team when she unearthed the bizarrely shaped skull—it was found to be ultradolichocephalic, meaning that it was especially long, narrow and flat—prompting an investigation. “Despite the craniofacial abnormalities, there is no skeletal evidence of severe cognitive impairment,” Rissech said in a study recently published in the journal Heritage. Unhealed perimortem trauma caused by a bladed weapon (penetrating the left temple and occipital region) and a blunt force injury to the left tibia suggest that he could have died in battle.” Rissech ruled out several conditions before tentatively diagnosing this individual with Crouzon syndrome, which causes the sutures in a baby's skull to fuse too soon (the skulls of newborns have open sutures that help them through the birth canal, and in the average human, these sutures gradually close until they completely fuse around age 3). Other symptoms of Crouzon syndrome include hypertelorism (in which the eyes are especially far apart), a narrow back of the nasal cavity that can impede breathing, inverted teeth from a receding upper jaw, a sunken palette, and hearing loss. Despite such difficulties, Crouzon Syndrome does not typically affect cognitive ability, so the deceased individual wouldn't have encountered any mental impediment to knighthood. Without treatment, Crouzon Syndrome can be fatal in childhood, but this newfound skull is evidence that even in an era with only rudimentary medicine, some individuals with this condition did survive to adulthood and beyond. So, even though surgery for this condition would not exist for hundreds of years, the unnamed knight defied the odds and was estimated to have been between 45 and 47 years old at his time of death. While his lower legs were apparently not used much, marks on the femur indicated muscular thighs often seen in those who ride horses. “This case stands out for its rarity and significance,” said Rissech. “It documents a possible adult case of Crouzon syndrome in a possible medieval knight, who not only survived into middle age but also could have served actively in warfare.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Scientists Just Upended the Age of Life on Earth</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/can-we-image-alien-earths-this-newfound-object-could-show-the-way/'>Can We Image Alien Earths? This Newfound Object Could Show the Way</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Using direct imaging, astronomers have found a brown dwarf that could help test technology for taking snapshots of Earth-like exoplanets Astronomers have found what could become the first target for a crucial test of NASA's upcoming Nancy Grace Roman Space Telescope, a soon-to-launch observatory that serves as a pathfinder mission for discovering Earth-like worlds around other stars. In a pair of new studies, an international research team has revealed two newfound objects around nearby stars: a gas-giant exoplanet orbiting the star HIP 54515 and a brown dwarf around the star HIP 71618. Both discoveries relied on fresh observations from a planet-imaging instrument at Japan's Subaru Telescope in Hawaii, as well as archival data from the European Space Agency's Gaia spacecraft. When it soars into orbit as early as next year, Roman will carry an instrument called a coronagraph, which is designed to blot out most of a star's bright glare so that the far fainter light from accompanying planets can be seen. Roman's coronagraph is a critical precursor for more ambitious starlight-blocking hardware planned to fly on another future NASA mission, the Habitable Worlds Observatory, which could launch in the late 2030s to discover, image and study potential Earth twins around dozens of sunlike stars. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The brown dwarf, Currie explains, is essentially a dim “failed star” that's neither too close nor too far from its stellar host, putting it at exactly the right distance and brightness for Roman to show its stuff. In particular, the direct-imaging techniques used to discover these two objects mark a major step forward for the field, which still mostly relies on other, indirect methods, such as tracing wobbles in a star's celestial motion to find exoplanets. “Direct imaging allows us to see these worlds themselves, which is historically significant not just as a technical milestone but as a shift in how we study distant planetary systems,” she says. Claire Cameron is the Breaking News Chief at Scientific American. Originally from Scotland, she moved to New York in 2012. Her work has appeared in National Geographic, Slate, Inc. Magazine, Nautilus, Semafor, and elsewhere. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02151-6'>In vivo CRISPR screen reveals regulation of macrophage states in neuroinflammation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 11:12:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Here we established an in vivo CRISPR screening pipeline using genetically editable progenitor cells to dissect macrophage regulation in mouse models of multiple sclerosis (MS). Screening over 100 cytokine receptors and signaling molecules identified interferon-γ, tumor necrosis factor, granulocyte-macrophage colony-stimulating factor and transforming growth factor-β as essential regulators of macrophage polarization in vivo. Single-cell transcriptomics confirmed that transferred progenitor cells generate all blood-derived CNS myeloid cell populations, enabling Perturb-seq analysis of cytokine actions in neuroinflammation. Combined with biosensor expression, our approach allows monitoring cytokine effects on myeloid cell migration, debris phagocytosis and oxidative activity in vivo. Comparative transcriptomic analyses revealed conserved neuroinflammatory cytokine signatures across myeloid populations, CNS compartments and species, elucidating cytokine cues shaping myeloid function in the cerebrospinal fluid and parenchyma of individuals with MS. This versatile pipeline thus provides a scalable framework for high-resolution analysis of macrophage states and uncovers the cytokine signals that underlie their regulation in MS and MS models. MS is an autoimmune disorder characterized by the infiltration of adaptive and innate immune cells into the CNS, resulting in demyelination and axonal loss1. Among the infiltrating immune cells, macrophages are the most abundant2, and their numbers correlate with the extent of tissue damage3,4. In experimental models, monocyte-derived macrophages are critical for disease manifestation and progression5,6,7 as they secrete pro-inflammatory mediators, proteolytic enzymes, and reactive oxygen and nitrogen species that exacerbate tissue damage8,9. Conversely, macrophages also contribute to repair by clearing myelin debris, remodeling tissue and releasing trophic factors10,11. These divergent roles illustrate the remarkable plasticity of macrophages, which adopt context-dependent polarization states shaped by their microenvironment and cellular condition12,13,14. In MS models, macrophages transition from an inducible nitric oxide synthase (iNOS)-positive, lesion-promoting phenotype to a lesion-resolving state characterized by arginase-1 (Arg1) expression8,15. However, the molecular cues that instruct these transitions in vivo remain incompletely defined, particularly within the inflamed CNS. Recently, pooled CRISPR screens have shown to be capable of identifying the essential signaling components that underlie complex cellular phenotypes16,17. In vitro, such pooled CRISPR screens have helped to comprehensively dissect important macrophage functions including regulation of their survival and phagocytic capabilities18,19; yet, these techniques have not enabled CRISPR screening of macrophages in an inflamed tissue environment. Here, we established a rapid, versatile and scalable in vivo CRISPR screening approach that allows functional analysis of more than 100 genes in myeloid cells in a single experiment. We used this approach to deconstruct the regulation of macrophage states in MS models and identified four essential cytokines that instruct macrophage polarization in neuroinflammation. To dissect the cytokine signals that drive macrophage states in neuroinflammatory lesions, we established a new approach to perform pooled CRISPR knockout (KO) screens in monocyte-derived macrophages in vivo using Hoxb8FL cells, a hematopoietic progenitor line20 that can be easily expanded and gene edited in culture. We incubated hBCL2-overexpressing Hoxb8FL Cas9 cells transduced with an enhanced green fluorescent protein (eGFP) retrovirus (Hoxb8FL cells) in vitro with macrophage colony-stimulating factor (M-CSF) for 48 h to initiate myeloid differentiation, and then injected them intravenously (i.v.) into mice 8–9 days after immunization with MOG1–125 (Fig. This allowed the primed Hoxb8FL-derived cells to differentiate into CD45+CD11b+ myeloid cells in vivo before CNS lesions form and first experimental autoimmune encephalomyelitis (EAE) symptoms become apparent around 12 days after immunization. Indeed, Hoxb8FL-derived cells were detectable based on their eGFP expression in blood, spleen, lymph nodes and bone marrow (Extended Data Fig. 1b) and exhibited similar expression of monocyte genes in the blood, and of macrophage surface markers including F4/80 and major histocompatibility complex (MHC) class II in peripheral lymphoid tissues, as endogenous myeloid cells (Extended Data Fig. Notably, Hoxb8FL-derived CD11b+ cells also readily entered spinal EAE lesions in a CCR2-dependent manner where they made up about 15% of the CD11b+ cells and showed comparable distribution in the lesion to endogenous CD11b+ cells. The cell transfer did not affect the overall disease course (Fig. 1i,l–o), and Hoxb8FL-derived myeloid cells did not contribute to the pool of resident meningeal and perivascular macrophages of the spinal cord (Extended Data Fig. In the EAE lesions, Hoxb8FL-derived myeloid cells polarized into iNOS+ and Arg1+ expressing cell populations, the transcriptomes of which closely correlated to the corresponding endogenous macrophage populations (Fig. 1c,d and Extended Data Fig. a, Scheme of the new experimental approach. Hematopoietic precursor (Hoxb8FL) cells were pre-differentiated in vitro for 48 h toward the myeloid lineage then i.v. injected into EAE-induced mice before disease onset. b, Histological characterization of the distribution of Hoxb8FL-derived and endogenous myeloid cells in EAE lesions in the spinal cord. Right, relative abundance of cells through lesions (depth from the spinal cord surface). N = 3 animals, 8, 11 and 11 lesions per animal. c, Representative flow cytometry plots of the transferred Hoxb8FL-derived cells (bottom row, green) compared to endogenous cells (top row, gray) in the spinal cord of the same mouse with EAE at the peak of disease. d, Quantification of the flow cytometry data. N = 4 animals for the myeloid marker panel and 3 animals for the polarization marker panel. e, Scheme of the pooled CRISPR screen targeting 109 cytokine receptors and downstream signaling genes with 15 control sgRNAs. f, Cytokine receptors regulating CNS migration, determined by comparing the spinal cord to the bone marrow compartment. g,h, Cytokine receptors regulating M-iNOS (g) or M-Arg1 (h) polarization, determined by comparing the single iNOS+ (g) or Arg1+ (h) spinal cord macrophages to the iNOS and Arg1 double-negative macrophages. Two-way analysis of variance (ANOVA) with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons, F = 1.98 × 10−16, P > 0.9999 used in b; multiple paired two-tailed t-tests or Wilcoxon tests with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons used in d; NS, P value > 0.05; *P value < 0.05, **P value < 0.01, ***P value < 0.001, **** P value < 0.0001; figures show the mean ± sd; not all genes included in the screen are shown for f–h. Asterisks indicate P values and false discovery rate (FDR) < 0.05 and absolute log2(fold change) > 3 times the s.d. To resolve the cytokine regulation of macrophage phenotypes in neuroinflammatory lesions, we performed a pooled CRISPR screen based on the transfer of Hoxb8FL-derived myeloid cells. We first constructed a CRISPR library of 343 single guide RNAs (sgRNAs), with three sgRNAs per gene and 15 non-targeting controls, to target 64 cytokine receptors and 45 of their downstream signaling mediators. We then transduced Hoxb8FL cells in vitro at a multiplicity of infection < 0.3, to ensure a single sgRNA transduced each cell. We selected infected cells, keeping the library coverage at a minimum of 100 cells containing each sgRNA (100× coverage). After in vitro pre-differentiation, we injected the cells into mice with EAE and about 6 days later, around the peak of EAE symptoms, we isolated Hoxb8FL-derived CD45 + CD11b+ myeloid cells from the bone marrow, and iNOS− Arg1−, iNOS+ and Arg1+ Hoxb8FL-derived myeloid cells from the spinal cord by fluorescence-activated cell sorting (FACS; Fig. We then compared the relative distribution of sgRNAs across these cell populations as depletion of sgRNAs in a given population indicates an essential role for the respective targeted genes in the induction of the corresponding cell state. Indeed our results showed that: signaling through CCR2 alone was required for CNS entry of phagocytes (Fig. 1f); the interferon gamma (IFNγ) receptor (IFNGR) and tumor necrosis factor (TNF) receptors (TNFR) were required for iNOS+ polarization, which was decreased by signaling through the TGFβ receptor 1 (TGFBR1; Fig. 1g); while Arg1+ polarization was promoted by the TGFBR1 and granulocyte-macrophage colony-stimulating factor (GM-CSF) receptor (CSF2R), and inhibited by the IFNGR1 (Fig. We confirmed these results by performing single-gene knockouts in Hoxb8FL cells and compared the polarization of Hoxb8FL-derived myeloid cells deficient for a given cytokine receptor with co-transferred control-edited Hoxb8FL-derived myeloid cells (Fig. These experiments confirmed that signaling via the IFNGR1 and the TNFR1 was required for iNOS+ polarization, while signaling through the TGFBR1 and the CSF2RA promoted Arg1+ polarization (Fig. Single-gene knockouts further confirmed that cytokines including interleukin (IL)-4 and IL-10, which induce Arg1+ polarization in vitro21, are not required for Arg1+ polarization in vivo, at least in this model at this disease time point (Extended Data Fig. Next, we leveraged our CRISPR screen to define the downstream signaling pathway components that are essential for transducing cytokine effects on macrophage polarization. We found that JAK1, JAK2 and STAT1 are required to mediate the effects of IFNGR, while IKK2 (Ikbkb), the key kinase mediating canonical nuclear factor-κB activation, is the only essential component of TNFR1 downstream signaling; moreover, only the transcription factor SMAD4 is essential for mediating the effects of TGFBR1 on both iNOS+ and Arg1+ polarization (Fig. Taken together, four cytokine receptors, IFNGR1, TNFR1, TGFBR1 and CSF2RA, and defined components of their downstream signaling pathways, are essential for establishing iNOS+ and Arg1+ macrophage polarization in neuroinflammatory lesions in the active MOG EAE model. a, Scheme of the validation experiment. Hoxb8FL control (tdTomato) cells were co-transferred with two different KO lines (GFP and blue fluorescent protein (BFP)) into the same immunized mouse before disease onset. b,d,f,h, Percentage of iNOS+ cell polarization in the single KO validation experiments compared to control, in the same animal (left) and scheme of the cytokine signaling pathway (right) for Ifngr1-KO (b), Tnfrsf1a-KO (d), Tgfbr1-KO (f) and Csf2ra-KO (h). c,e,g,i, Percentage of Arg1+ cell polarization in the single KO validation experiments compared to control of the same animal (left) and scheme of the cytokine signaling pathway (right) for Ifngr1-KO (c), Tnfrsf1a-KO (e), Tgfbr1-KO (g) and Csf2ra-KO (i). b–i, n = 9 animals for Ifngr1-KO and Tgfbr1-KO transfer, n = 8 animals for Tnfrsf1a-KO and Csf2ra-KO transfer. In b–i, asterisks indicate P values and FDR < 0.05 and absolute log2(fold change) > 3 times the s.d. To determine which of these cytokine regulation pathways were unique to this particular MS model and which might be more universal, we repeated the screen in a passive EAE model induced by the transfer of MOG-reactive IL-17-producing helper T (TH17) cells. We transferred CRISPR-edited Hoxb8FL cells at the onset of disease symptoms and isolated Arg1+ and Arg1− Hoxb8FL-derived macrophages 6 days later as at this time point during lesion resolution most macrophages were Arg1+ (Extended Data Fig. Our results showed that—as in the active MOG EAE model—TGFBR supported Arg1+ polarization, while IFNGR1 reduced Arg1+ polarization; similarly, the effects of TGFBR were mediated by the transcription factor SMAD4, while the effects of IFNGR1 relied on STAT1. However, in contrast to the MOG EAE model, Arg1+ polarization in the TH17 transfer model was not affected by CSF2R deletion but instead was limited by interfering with IL-4/IL-13 signaling, at the level of either IL13RA or the transcription factor STAT6. Furthermore, in the TH17 transfer EAE model, Arg1+ polarization was also inhibited by type I interferon signaling mediated by IFNAR1 (Extended Data Fig. Taken together, our findings indicate that IFNγ and TGFβ reciprocally regulate macrophage polarization across MS models while the regulatory effects of other cytokines—among them GM-CSF and IL-4/IL-13–depend on the inflammatory context and thus might differ between lesion types and/or disease stages. To obtain a higher resolved understanding of these essential cytokine regulators, we first generated bone-marrow-derived macrophages, exposed them to different combinations of cytokines in vitro, and subjected them to transcriptional analysis (Extended Data Fig. As predicted by our screen results, IFNγ and TNF acted synergistically to induce iNOS+ polarization, while TGFβ and GM-CSF collaborated during the induction of Arg1+ polarization: however, these cytokines did so by inducing distinct, cytokine-specific transcriptional modules (Extended Data Fig. Notably, the highest proportion of Arg1+ cells was induced by the combination of all four cytokines, which may be related to the observation that myeloid cells can transition between MiNOS and MArg1 states in vivo and in vitro8. Therefore, we next aimed to uncover the effect of these cytokines on macrophage differentiation and molecular specification in vivo. To do so, we conducted three separate single-cell RNA-sequencing (scRNA-seq) experiments on CD11b+Ly6G− monocytes/macrophages isolated at the peak of disease from mice with active MOG EAE: (i) a wild-type (WT) EAE experiment as reference; (ii) Hoxb8FL-derived CRISPR KO experiments targeting the IFNGR1, TNFR1, TGFBR1 and CSF2RA; and (iii) bone-marrow chimera experiments having reconstituted the bone marrow with individual CRISPR KOs of the same receptors (Fig. When we integrated the transcriptional data from the three sets of experiments, we found that the same myeloid cell clusters were present in all of them; moreover, cells in the same clusters exhibited high transcriptional similarity independent of their origin (Fig. Through analysis of marker gene expression and pseudotime analysis of the integrated object (Fig. 3c), we assigned the myeloid cell populations from monocyte-like cells to three clusters of CXCL10+ macrophages22 (‘macrophages CXCL10 1–3') and eight clusters of other macrophages (‘macrophages 1–8'). To relate the single-cell analysis to our CRISPR screen, we mapped the expression of our screen markers Nos2 and Arg1 in the scRNA-seq data and found that the populations expressing these genes were consistent across experiments and remained mostly distinct from each other: Nos2 expression was more restricted and primarily found in early-to-intermediate phenotypes (based on pseudotime), most prominently in the cluster ‘macrophages CXCL10 3'; while Arg1 was broadly expressed in intermediate and later macrophage populations (Fig. Furthermore, both Hoxb8FL-transfer and bone-marrow chimera-based CRISPR editing experiments confirmed the effects of cytokine receptor deletion on iNOS+ and Arg1+ polarization on the transcriptomic level as predicted by our initial CRISPR screen results (Extended Data Fig. Hoxb8FL (top) or LSK (bottom) cells deficient for the receptors of the four essential cytokines driving macrophage polarization in EAE as well as control-edited cells were either transferred as a Hoxb8FL cell pool, as previously described (top), or used to reconstitute the immune system after irradiation in a bone-marrow chimera model (bottom). At the peak of disease, endogenous (not shown on scheme), Hoxb8FL-derived or chimeric macrophages were harvested from the spinal cord and subjected to single-cell transcriptional sequencing. b, UMAP plots of the three EAE macrophage single-cell sequencing datasets after integration. c, Pseudotime across the different monocyte and macrophage clusters in the integrated dataset. Clusters are color coded according to the scale on the right. d, Feature plot of Nos2 and Arg1 expression in the three datasets; expression levels are color coded according to the scales on the right of the plot. e, Density difference plot showing differences in the distribution of myeloid cell subpopulations in the KOs compared to the control cells; density differences are color coded according to the scale on the right. Next, we investigated how the lack of IFNγ, TNF, GM-CSF or TGFβ signaling in neuroinflammatory lesions affects the differentiation of myeloid cell subpopulations. We compared the distribution of monocyte/macrophage cell clusters in the uniform manifold approximation and projection (UMAP) between the individual KOs and the control-edited cells in both Hoxb8FL-transfer and bone-marrow chimera experiments. Among them only the deletion of IFNGR1 affected the monocyte-like cell phenotypes and resulted in a relative increase in the proportion of the monocyte population, as previously observed23. Among the macrophage clusters, deletion of IFNGR1 and TNFR1 showed opposite phenotypes, with IFNγ signaling needed for maintaining earlier—presumably more pro-inflammatory—macrophage clusters, and TNF signaling being required for progression to later macrophage cell phenotypes (Fig. In contrast, deletion of TGFBR1 and CSF2RA showed overlapping effects on the distribution of macrophage subpopulations, with both cytokines blocking the induction of CXCL10+ macrophages and promoting the differentiation of macrophages to the later macrophage cell phenotypes (Fig. To further characterize the effects of the essential cytokine regulators on the molecular specification of macrophages, we next manually curated gene signatures that represent distinct lesion-promoting and lesion-resolving properties of macrophages (Methods). In line with the transcriptional similarity of macrophage clusters across experimental settings, these signatures were highly conserved between endogenous macrophages and macrophages derived from bone-marrow chimera or Hoxb8FL-progenitor cells (Extended Data Figs. When we investigated the expression of these signatures in our single-cell transcriptomic data, we found that the expression of signatures related to lesion-promoting functions across macrophage clusters was highly correlated, and decreased from early to later macrophage populations in neuroinflammatory lesions (Extended Data Fig. We next assessed how each cytokine receptor deletion affected the expression of the gene signatures in each of the individual macrophage clusters. Our results showed that overall lesion-promoting actions of macrophages are underpinned by IFNGR1, TNFR1 and CSF2RA signaling, and inhibited by TGFBR1 signaling—with the exception of ‘antigen presentation', which primarily depends on IFNGR1 signaling (Fig. To further investigate the effect of these cytokine regulators on a defined lesion-promoting function of myeloid cells, we focused on the regulation of oxidative bursting, a key contributor to tissue damage in experimental and human neuroinflammatory lesions9,24,25,26. To directly read out the redox state of myeloid cells, we expressed the Grx1-roGFP2 sensor27 in CRISPR-edited Hoxb8FL-derived myeloid cells. Our results showed that in neuroinflammatory lesions in vivo IFNγ signaling appears to be a primary driver of oxidative bursting (Fig. a, Left, expression of each transcriptional signature related to lesion-promoting properties in Hoxb8FL control cells; right, phenotype of the KOs in Hoxb8FL-derived cells for the respective signature across macrophage clusters (Ma_Cx_1 to Ma_8) and overall (All) compared to the control cells. For statistical analysis, gene-set enrichment analysis (GSEA) was used (Methods). b, Scheme illustrating the detection of oxidized/reduced cellular states by excitation ratiometric imaging of the Grx1-roGFP2 sensor. c, Experimental design of the intravital imaging experiments based on the transfer of Hoxb8FL-derived myeloid cells expressing the Grx1-roGFP2 sensor. d, Representative images of the dorsal lumbar spinal cord along the midline vein, showing a neuroinflammatory lesion infiltrated by Hoxb8FL-derived myeloid cells expressing the Grx1-roGFP2 sensor. Filled white arrowheads indicate myeloid cells in an oxidized state, while outlined arrowheads indicate myeloid cells in a reduced state. e,f, Quantification of the redox state of Ifngr1-KO (e) or Tnfrsf1a-KO (f) Hoxb8FL-derived myeloid cells expressing the Grx1-roGFP2 sensor. Left, proportion of control (gray) or KO (purple) cells per animal that fall in each oxidation/reduction Grx1-roGFP2 ratio bin, where lower bin numbers mean low ratio; the whole range is divided into ten bins and each bin is equally sized; middle, cumulative distribution of the oxidized/reduced Grx1-roGFP2 ratios of all cells; right, percentage of cells per animal that show an oxidized/reduced ratio above the 75th percentile ratio of their experiment. In a, KO names marked by an asterisk at the top of the plots and black ‘All' labels indicate that the phenotype of the signature for that KO versus control in the global Hoxb8FL-derived cells is significant. Blue indicates downregulated and red indicates upregulated in the KO compared to the control. Asterisks and black cluster name labels indicate the phenotype is significant for that cluster for the Hoxb8FL-derived KO versus the control. Significance was determined with the GSEA pathway analysis algorithm for a nominal (NOM) P value < 0.05 or an FDR q value < 0.25 and absolute normalized enrichment score (NES) > 1.5 (Methods); e,f, (left), Ordinary two-way ANOVA with a single pooled variance, F = 2.009, P = 0.0509 (e) and F = 0.2027, P = 0.9929 (f) for the interaction of genotype × bin. e,f, (right) one-tailed unpaired t-test. In line with these results and the antagonistic actions of IFNγ and TGFβ signaling observed in the CRISPR screens across MS models, transcriptional signatures related to lesion-resolving actions of macrophages were primarily induced by TGFBR1 signaling and often inhibited by IFNGR1 signaling. Notably, both TGFBR1 signaling and CSF2RA signaling appear to drive the expression of genes associated with ‘tissue remodeling', indicating that—as observed in the CRISPR screens and for macrophage polarization in vitro—CSF2RA receptor signaling can synergize with TGFBR1 signaling in distinct functional domains (Fig. Furthermore, we observed prominent effects of TGFBR1 ablation on transcriptional signatures related to lipid signaling and lipid processing. As clearing the lipid-rich myelin debris is an important prerequisite for the resolution of neuroinflammatory lesions, we further investigated how essential cytokine regulators affect lipid uptake and digestion. In line with the regulation of related transcriptional signatures, we observed that specifically Hoxb8FL-derived myeloid cells lacking TGFBR1 but not those lacking IFNGR1, TNFR or CSF2RA showed an increase in cell size, lipid content and lysosomal marker expression (Fig. Notably, our subsequent analysis revealed that this lipid accumulation is not related to an impairment of phagocytosis or lysosomal acidification but rather due to an alteration of the lipid efflux capacity of myeloid cells that appear to depend on TGFβ signaling (Fig. In line with these results, TGFBR1-deficient Hoxb8FL-derived myeloid cells showed an increased content of lipid droplets (Fig. 5o) that can drive a pro-inflammatory foamy cell phenotype28. Indeed, myeloid cells lacking TGFBR1 showed an increased expression of the foamy cell marker GPNMB29,30 in the inflamed spinal cord (Fig. a, Same representation as in Fig. b,e, Representative image of neutral-lipid loaded (b) or Lamp1-positive (e) control and Tgfbr1-KO Hoxb8FL-derived myeloid cells in EAE lesions at peak of disease. Scale bars, 10 μm; white arrows indicate macrophages with a foamy appearance (b); inset indicates area in e. c,d,f, Quantification of neutral-lipid droplet volume (c), cell volume (d) and Lamp1 intensity (f) in control (gray) and cytokine receptor KO (orange) Hoxb8FL-derived myeloid cells. g, Quantification of pHrodo intensity over time in vitro upon pHrodo-myelin treatment of control and Tgfbr1-KO Hoxb8FL-derived macrophages. N = 3 independent in vitro experiments with 6 technical replicates per experiment. Data are plotted from all technical replicates. h, Scheme illustrating the detection of lysosomal acidification using the mTFP1-mCherry sensor and the experimental design of the intravital imaging experiment co-transferring dye-labeled mTFP1-mCherry Hoxb8FL control-edited or Tgfbr1-KO cells. i, Representative image of the dorsal lumbar spinal cord along the midline vein, showing a neuroinflammatory lesion infiltrated by Hoxb8FL-derived dye-labeled control-edited and Tgfbr1-KO myeloid cells expressing mTFP1-mCherry. j,k, Quantification of the lysosome acidification state in Hoxb8FL-derived myeloid cells expressing mTFP1-mCherry. j, Left, proportion of control (gray) or Tgfbr1-KO (orange) myeloid cells per animal that fall in each acidification mCherry/mTFP1 ratio bin, where lower bin numbers mean a low ratio, the whole range is divided into ten bins and each bin is equally sized; right, cumulative distribution of the acidification mCherry/mTFP1 ratio of all cells. k, Percentage of cells per animal that have an acidification ratio above the 75th percentile ratio of their experiment. l,n, Quantification of MBP absolute intensity (l, left), MBP intensity normalized to Lamp1 intensity (l, right) or Plin2 intensity (n) in Hoxb8FL-derived control (gray) or Tgfbr1-KO (orange) myeloid cells in EAE lesions at the peak of disease. m, Scheme of the cholesterol efflux assay (left) and quantification of cholesterol-TMR in the supernatant of TGFβ-treated control or Tgfbr1-KO Hoxb8FL-derived macrophages. N = 3 independent experiments with 3 technical replicates each; data are plotted from the technical replicates normalized to untreated control average. o, Quantification of the presence of cholesterol crystals by confocal reflection microscopy in control or Tgfbr1-KO Hoxb8FL-derived myeloid cells in EAE lesions at the peak of disease. p, Representative images showing Gpnmb labeling in control and Tgfbr1-KO Hoxb8FL-derived myeloid cells in EAE lesions at the peak of disease. White arrowheads indicate Gpnmb puncta. 4a for details; c, d, f, k, l, m, o and p, Two-tailed paired t-test or Wilcoxon test; g and j, ordinary two-way ANOVA with a single pooled variance. g, F = 55.05, P < 0.0001 for the genotype. j, F = 1.336, P = 0.2237 for the interaction of genotype × bin; j (right), K–S statistic. Taken together, these analyses demonstrate how combining CRISPR gene editing with single transcriptomics and in situ and in vivo imaging can be applied to comprehensively reveal the regulation of macrophage states, their differentiation and molecular specification in vivo. In this context, it is interesting to note that similar regulatory patterns with regard to both the temporal evolution of gene signatures and their regulation by individual cytokine receptor KOs were observed between Hoxb8FL-derived and bone-marrow chimeric macrophages (Extended Data Fig. 8), supporting the view that Hoxb8FL-derived myeloid cells are a valid tool to dissect the molecular regulation of macrophages in vivo. To reveal additional regulatory aspects of cytokine signaling beyond the curated signatures, we performed an unbiased analysis of the differentially regulated genes in the cytokine receptor-deficient compared to the control-edited macrophage populations (Methods). We observed that most of the regulated genes in any of the cytokine receptor deletion experiments were similarly altered in multiple macrophage clusters, suggesting that the cytokine regulators induce common transcriptional responses across macrophage subpopulations (Extended Data Fig. Notably, we again saw a sizable overlap in the genes differentially regulated by TGFβ and GM-CSF, while IFNγ induced expression of a distinct gene set (Extended Data Fig. An unbiased biological processes Gene Ontology (GO) term analysis of the differentially regulated genes revealed that GO terms related to cell adhesion and migration were among the top biological processes regulated by CSF2RA and TGFBR1 (Fig. Indeed, a curated signature including genes related to macrophage adhesion and migration identified TGFBR1 as a putative regulator of these functions (Fig. To further investigate how the migration pattern of myeloid cells in neuroinflammation might be modulated by the essential cytokine regulators, we co-transferred Hoxb8FL-derived myeloid cells deficient for IFNGR1, TNFR1, CSF2RA or TGFBR1 with control-edited cells and characterized the distribution of transferred cells within neuroinflammatory lesions. Here we observed a striking difference in the distribution of TGFBR1-deficient macrophages between leptomeninges and CNS parenchyma with TGFBR1-deficient cells being basically absent from the pia and enriched in the CNS lesion parenchyma. This altered distribution appeared to be specific for TGFBR1-deficient macrophages as it was not observed for any of the other cytokine receptor-deficient cells (Fig. To assess whether the differential distribution of TGFBR1-deficient myeloid cells in the inflamed CNS results from an altered mobility of these cells, we used in vivo microscopy to track the movement pattern of control-edited and TGFBR1-deficient Hoxb8FL-derived myeloid cells in EAE lesions in the superficial dorsal spinal cord. Our results showed that neither the proportion of cells that move nor the speed at which they move are affected by the absence of TGFBR1 (Fig. Taken together, these experiments thus reveal that TGFBR1 signaling instructs the distribution of macrophages in the inflamed CNS without affecting their ability to move within neuroinflammatory lesions. a, Unbiased gene-set analysis of GO biological process gene sets downregulated by cytokine receptor KOs (see Methods for details on the calculation of the KO-specific differentially regulated genes). GO terms related to cell migration or adhesion are highlighted in bold. b, Left, expression of the transcriptional signature ‘Adhesion & Migration' in the Hoxb8FL-derived control cells; right, phenotype of the KOs in control Hoxb8FL-derived cells for this signature across clusters (Ma_Cx_1 to Ma_8) and overall (All) compared to the control cells. For statistical analysis, GSEA was used (Methods). A mix of tdTomato+ control cells and eGFP+ KO Hoxb8FL cells was transferred into an immunized mouse before disease onset; at the peak of disease the animal was perfused, and then the spinal column was decalcified and cut coronally. d, Representative confocal microscopy image of a cross-section of the whole spinal column of an EAE animal co-transferred with control (red) and Tgfbr1-KO (green) cells. Higher-magnification image on the right shows myeloid cell infiltration in the pia delimited by laminin staining, and in adjacent parenchymal lesions. e, Quantification of the distribution of the Hoxb8FL-derived cytokine receptor KO myeloid cells relative to the control cells in the same animal across compartments. f, Scheme of the experiment for tracking the movement of Hoxb8FL-derived myeloid cell in neuroinflammatory lesions (left); time-lapse images showing cell movement along the midline vein in the dorsal spinal cord at peak of EAE of control-edited and Tgfbr1-KO Hoxb8FL-derived myeloid cells (middle); and quantification of movement parameters (right). Filled white arrowheads indicate control cells, outlined white arrowheads indicate Tgfbr1-KO cells, and lines denote cell tracks. Pathways shown in a include only GO_BP and GO_MF terms, with an adjusted P value < 0.01, an intersection size ≥ 5 and term size < 2,000; b, KO names marked by asterisks at the top of the plots and black ‘All' labels indicate that the phenotype of the signature for that KO versus control in the global Hoxb8FL-derived cells is significant. Asterisks and black cluster name labels indicate the phenotype is significant for that cluster for the Hoxb8FL-derived KO versus the control. Significance was determined with the GSEA pathway analysis algorithm for a NOM P value < 0.05 or an FDR q value < 0.25 and absolute NES > 1.5 (Methods); e, Two-tailed paired t-test or Wilcoxon test for KO versus control for pia and for parenchyma separately; data are shown as a ratio for convenience. f, Two-tailed paired t-test or Wilcoxon test. We next examined whether insights from experimental neuroinflammatory lesions could reveal cytokine-driven regulation of myeloid cells in individuals with MS. Using genes induced by a given cytokine across mouse myeloid populations, we defined ‘neuroinflammatory cytokine signatures' representing cytokine-regulated programs in inflammatory contexts (Fig. These signatures only partially overlapped with those from homeostatic conditions31 (Extended Data Fig. 10a) and were prominently expressed in myeloid clusters within MOG EAE white matter lesions, particularly in populations dependent on the respective cytokine (Figs. We further analyzed immune cells from the cerebrospinal fluid (CSF) and gray matter of mice with MOG EAE and cortical MS-like pathology. In both compartments, IFNγ, TNF, GM-CSF and TGFβ signatures were strongly enriched in myeloid but not adaptive immune cells, with high expression of the signatures induced by IFNγ, GM-CSF and TGFβ also detected in microglia (Fig. These findings indicate that neuroinflammatory cytokine signatures are conserved across myeloid populations and CNS compartments, including gray matter and CSF. a, Scheme illustrating how we derived neuroinflammatory cytokine signatures. b, Expression of the averaged neuroinflammatory cytokine signatures in the WT EAE UMAP. c, Scheme of the CSF extraction procedure from the cisterna magna (left) and UMAP of the immune cells present in CSF of EAE animals at the peak of disease (right). d,e, Expression of the averaged neuroinflammatory cytokine signatures in the CSF EAE UMAP. f, Scheme of the induction of the cortical MS (cMS) disease model (left) and UMAP of the immune cells present in the cortex of cMS animals at the peak of inflammation (right). g,h, Expression of the averaged neuroinflammatory cytokine signatures in the cMS UMAP. We next investigated whether neuroinflammatory gene signatures could reveal cytokine actions on myeloid cells in individuals with MS. To this end, we analyzed induction of these signatures in CSF monocytes from people with multiple sclerosis. A published scRNA-seq dataset32 was integrated with our own dataset comprising untreated individuals early in the course of MS (Methods), together with a control set from individuals with mild cognitive impairment (MCI) and Alzheimer's disease (AD33; Fig. Pseudobulk analysis across monocyte populations revealed that CSF monocytes from people with multiple sclerosis—unlike those from MCI/AD controls—showed clear induction of the IFNγ signature (Fig. This is consistent with the reported induction of IFNγ-dependent genes in neurons as previously observed, for example, in the cortex of people with multiple sclerosis34. a, UMAP of human CSF monocyte/macrophages across datasets after integration. b, Pseudobulked average expression per individual of the neuroinflammatory cytokine signatures across human MS CSF datasets and disease stages, and a control CSF dataset from MCI/AD. N = 5 idiopathic intracranial hypertension (IIH), 6 radiologically isolated syndrome (RIS) and 14 individuals with MS, n = 5 individuals with IIH and 5 individuals with MS from ref. 33. c, UMAP of human microglia from ref. d, Pseudobulked average expression per individual of the neuroinflammatory cytokine signatures in microglia derived from gray matter of control individuals or people with MS or MCI/AD. N = 12 samples for control gray matter (GM), 15 for normal-appearing gray matter (NAGM) in MS and 13 for gray matter lesion (GML) in MS, n = 13 healthy individuals, 8 individuals with MCI and 20 individuals with AD from ref. 36. e, Density distribution of microglial clusters across white matter lesion types in human MS. WM, control white matter; NAWM, normal-appearing white matter; AL, active lesion; CAL, chronic active lesion; CIL, chronic inactive lesion; RL, remyelinating lesion. f, Pseudobulked average expression per individual of the neuroinflammatory cytokine signatures across microglial clusters in the white matter. Respectively, n = 36, 23, 19, 30 and 18 samples in homeostatic to MHC class II-recovery clusters. b, One-way ANOVA, IFNγ F = 4.9165, P = 0.0171, TNF F = 0.7544, P = 0.4821, GM-CSF F = 0.2334, P = 0.7938, TGFβ F = 0.727, P = 0.5088, followed by two-tailed t-tests or Wilcoxon tests comparing RIS/MS/MCI-AD to their dataset control (IIH or healthy); d, One-way ANOVA IFNγ F = 2.0908, P = 0.1379, TNF F = 0.2422, P = 0.7861, GM-CSF F = 2.0325, P = 0.1454, TGFβ F = 1.5998, P = 0.2156, followed by two-tailed t-tests NAGM and GML to healthy GM for ref. 35 MS data; IFNγ F = 1.803, P = 0.168, TNF F = 1.1469, P = 0.3284, GM-CSF F = 6.0476, P = 0.0052, TGFβ F = 0.7814, P = 0.465, followed by two-tailed t-tests comparing MCI/AD to healthy for ref. 36 MCI/AD data; f, one-way ANOVA TNF F = 2.3413, P = 0.0588, Kruskal–Wallis test IFNγ P = 0.0039, GM-CSF P < 0.0001, TGFβ P < 0.0001, followed by two-tailed t-tests or Wilcoxon tests comparing all clusters to the homeostatic cluster, adjusting the P values by the FDR method; In the box plots, the line shows median, the box extends from Q1 to Q3, and the whiskers extend to the smallest and largest values within 1.5 times the interquartile range from Q1 and Q3. To investigate cytokine activity in MS lesions, we analyzed a large single-nucleus RNA-seq dataset from postmortem brain samples of people with multiple sclerosis35. Tissue myeloid cells, predominantly microglia, were grouped into five clusters based on established molecular markers (Fig. We then examined the presence of neuroinflammatory cytokine signatures in microglia from gray matter lesions and normal-appearing gray matter and compared them with microglia from MCI/AD donors36. Pseudobulk analysis per individual revealed induction of IFNγ and GM-CSF signatures in MS gray matter—most prominently within lesions—whereas no such induction was observed in microglia from MCI/AD samples (Fig. These results suggest that essential cytokine regulators are active in the MS brain, prompting us to examine whether distinct microglial subsets are differentially influenced by IFNγ, TNF, GM-CSF and TGFβ. In MS white matter lesions35, microglial cluster frequencies varied by pathology stage: homeostatic microglia dominated healthy tissue, whereas inflammatory/interferon-responsive (IRM), disease-associated (DAM), lipid-processing and MHC class II-high microglia were enriched in MS lesions (Fig. IRM cells were abundant in active lesions and surrounding white matter, DAM and lipid-processing microglia clustered in active and chronic active lesions, while MHC class II-high cells appeared in inactive and remyelinating areas, indicating a recovery-associated phenotype37. Differential expression of cytokine signatures (relative to homeostatic microglia) revealed strong induction of an IFNγ signature in IRM, broad GM-CSF activation across lesion-associated populations, absent enrichment of a TNF gene signature and robust induction of a TGFβ signature in DAM and MHC class II-high microglia (Fig. Taken together, these observations highlight that the neuroinflammatory cytokine signatures we delineated and validated in MS models are selectively induced in myeloid cells in neuroinflammatory but not in neurodegenerative conditions and can be used to uncover the differential regulation of myeloid cell states in MS across disease locations and stages. We developed a CRISPR-based screening platform enabling systematic dissection of molecular factors underlying macrophage state regulation in inflamed tissues. Compared to classical conditional gene deletion or bone-marrow chimera approaches38, our method expedites experimental timelines, reduces animal use and ensures uniform sgRNA distribution through consistent myeloid cell expansion, facilitating large-scale analysis of over 100 genes in a single screen. Applying this strategy to cytokine receptor screening identified IFNγ, TNF, GM-CSF and TGFβ as key in vivo drivers of macrophage polarization, whereas IL-4, IL-10 and IL-13, which modulate polarization in vitro21,39, do not direct macrophage polarization in vivo at least in the active MS model at the studied time point. Using naive C57BL/6 recipients and in vitro-edited precursors enhances experimental flexibility, enabling integration with biosensor technologies and avoiding confounds from irradiation or genetic drift40,41,42. As Hoxb8FL-derived myeloid populations were detected in all analyzed tissues, this methodology is broadly applicable for interrogating myeloid cell regulation across diverse tissues and disease settings. Successful application of this approach relies on an understanding of its inherent limitations. The method targets monocyte-derived myeloid cells, as microglia and most CNS-resident macrophages largely persist independently of blood-derived monocytes43. It is optimally suited for probing acute, cell-autonomous phenotypes, since Hoxb8FL-derived cells replace only ∼15% of endogenous populations, minimizing capacity to affect overall tissue pathology or disease trajectory. Nevertheless, this framework enables direct analysis of myeloid cell phenotypes within largely unaltered lesion contexts, isolating cell-intrinsic signaling pathways. The efficiency of large-scale CRISPR screens remains contingent on the yield of genetically modified cells from tissue; given the short lifespan of monocyte-derived cells, rapid immune infiltration models are preferable. Additionally, our approach relies on a finite set of sortable phenotypes—here, iNOS+ for lesion-promoting and Arg1+ for lesion-resolving myeloid cells, reflecting established roles in MS models and MS tissue linking iNOS+ polarization to active lesion formation, with subsequent transition to Arg1+ during recovery in MS models8,15,25,44. While this binary scheme encapsulates extremes, macrophage phenotypes likely span a broader continuum influenced by disease context, cell development and analytical methodology13,14,45. Importantly, our screening strategy can be extended to alternative markers, and cytokine-based regulation provides a foundation for dissecting additional regulatory pathways influencing myeloid states. To delineate cytokine-induced macrophage states in neuroinflammation with single-cell resolution, we implemented Perturb-seq using scRNA-seq for CRISPR perturbation readout46,47. To validate the utility of the Hoxb8FL transfer approach for single-cell analyses, we performed parallel Perturb-seq in bone-marrow chimeras optimized for CRISPR editing of myeloid cells. Both methods demonstrated that cytokine receptor deletion induces similar shifts in macrophage subpopulation composition and transcriptional profiles (Fig. 5–8), confirming the broad utility of CRISPR-edited progenitors for comprehensive, unbiased analysis of macrophage state regulation. A key insight from our Perturb-seq analysis is that macrophage polarization is chiefly governed by the balance of IFNγ and TGFβ signaling. Both pathways not only promote distinct polarization states but also reciprocally inhibit the alternative phenotype, a phenomenon consistent across MS models. These findings implicate IFNγ as a central mediator of pro-inflammatory signatures, tissue damage and oxidative burst in myeloid cells, aligning with previous studies in MS models and people with MS48,49,50. Nonetheless, IFNγ actions are highly context dependent, potentially varying with EAE model, lesion stage, cofactors and responding cell type48,51,52,53. Similarly to IFNγ, TNF signaling promoted pro-inflammatory, tissue-damaging macrophage signatures, corroborating studies where TNF blockade improved outcomes in MS models54,55 and TNFR1-deficient mice were protected from EAE56. However, neuroinflammatory TNF signatures were not prominently detected in lesion-associated microglia in MS. Clinically, anti-TNF therapies, effective in other autoimmune diseases57,58, have exacerbated MS pathology, a dichotomy partially explained by an MS-risk single nucleotide polymorphism (rs1800693) in TNFRSF1A that yields a soluble TNFR1 antagonizing TNF59. These converging findings suggest that TNF's protective effects in MS may be mediated through non-macrophage populations, such as its known support for myelin repair via TNFR2 on oligodendrocyte precursor cells60. Our CRISPR screen revealed that both GM-CSF and TGFβ are essential for Arg1+ macrophage polarization in EAE. This was unexpected given GM-CSF's established pro-inflammatory role in neuroinflammation: mice lacking CSF2R in monocytes are resistant to EAE due to impaired induction of inflammatory and oxidative stress-related genes23,61. Perturb-seq confirmed these pro-inflammatory effects yet uncovered an unanticipated contribution of GM-CSF to tissue remodeling, resembling TGFβ activity. The considerable overlap between GM-CSF-regulated and TGFβ-regulated gene sets (Extended Data Fig. 9c) mirrors their reported synergy in promoting scar-associated macrophages62, highlighting the complexity of GM-CSF as both a pro-inflammatory and reparative cytokine—an important consideration given its candidacy as a therapeutic target in MS and other inflammatory conditions63,64,65. Consistent with earlier studies, TGFβ induced transcriptional programs of homeostasis, debris clearance and tissue repair. Loss of TGFBR1 impaired lipid efflux, resulting in foamy macrophages expressing GPNMB and accumulating lipids and cholesterol crystals, a phenotype linked to progressive MS pathology66,67,68. An intriguing aspect of this regulation is that it appears to be distinct in monocyte-derived macrophages69 compared to microglia in which TGFβ was shown to limit effective lipid clearance at least in a model of toxic myelin damage70. Beyond its established homeostatic functions, we identified a critical role for TGFβ signaling in regulating myeloid cell infiltration during neuroinflammation. Although TGFβ has long been associated with cell motility and epithelial–mesenchymal transition in cancer71,72,73, our data indicate a distinct mechanism in the CNS: TGFβ does not affect macrophage movement within lesions but instead governs their progression from meninges into the parenchyma, an effect that might well contribute to the worsening of CNS inflammation observed in the absence of TGFβ signaling in myeloid cells74. Single-cell analysis of cytokine actions revealed broadly consistent effects of individual cytokines on myeloid cell specification across macrophage clusters, indicating that response intensity rather than direction depends on the differentiation state of the cell. However, cytokine-induced gene programs were partially shaped by the local inflammatory milieu, as transcriptional responses in neuroinflammatory lesions only partly overlapped with those observed after cytokine stimulation in lymph nodes31 (Extended Data Fig. The uniform cytokine responsiveness within lesions suggests that core signaling mechanisms are conserved across myeloid cell types and possibly species, providing a framework for cross-species comparison of myeloid activation in disease. From our Perturb-seq data, we defined neuroinflammatory cytokine signatures for IFNγ, TNF, GM-CSF and TGFβ, and confirmed their enrichment across murine myeloid populations not only in inflamed white matter, but also in gray matter lesions and the CSF compartment. Analysis of human CSF monocytes revealed selective upregulation of IFNγ-responsive genes in people with MS, paralleling IFNγ presence in CSF75 and its induced transcriptional signatures in MS neurons34 and gray matter microglia as we observe here. In white matter, IFNγ signatures predominated in inflammatory microglia, while GM-CSF—but not TNF—signatures extended across lesion-associated clusters, implicating GM-CSF in sustained microglial activation76,77. In contrast, TGFβ signatures were enriched in MHC class II+ microglia from inactive and remyelinating lesions. This indicates that TGFβ not only maintains the homeostatic phenotype of microglia78, but also provides an important signal that drives a distinct microglial phenotype associated with tissue recovery37. Collectively, our findings demonstrate that neuroinflammatory cytokine signatures can delineate which cytokines influence specific myeloid cell populations across CNS compartments and neurological conditions. This analytical approach is adaptable to diverse mediators and transcriptomic datasets, offering a powerful tool for uncovering molecular drivers of myeloid cell heterogeneity in humans. In conclusion, we established a rapid, scalable and versatile in vivo CRISPR screening approach to dissect the signals driving myeloid cell states across tissue compartments and models, providing insights into myeloid cell regulation in human disease, an important step toward the rational design of therapies targeting innate immune cells. Animal care and experiments were carried out in accordance with the regulations of the applicable animal welfare acts and using protocols approved by the responsible regulatory authority (Regierung von Oberbayern, commission number 15). For human data, the collection of samples was approved by the local ethics committees of the LMU, Munich (ethical vote: 163-16, equivalent of the Institute Review Board (IRB) number for the ethics committee). Written informed consent was obtained from all individuals according to the Declaration of Helsinki. Further information and requests for resources and reagents should be directed to and will be fulfilled by the corresponding authors, M.K. Plasmids generated in this study are available upon reasonable request. All primer sequences are listed in Supplementary Table 1. The MSCV-v2-U6-(BbsI)-Pgk-Puro-T2A-eGFP (with v2 improved scaffold) vector was generated as follows: first, the pU6-Pgk-Puro-T2A construct was PCR amplified from pKLV2-U6gRNA5(BbsI)-PGKpuro2ABFP-W (Addgene), and the eGFP construct was PCR amplified from pMSCV-Cas9-eGFP (in house); then, the PCR product was assembled into the SalI (NEB) + XhoI (NEB) digested pMSCV-neo (Takara Clontech) vector using the Gibson Assembly Master Mix (NEB). The pMSCV-v2-U6-(BbsI)-Pgk-Puro-T2A-BFP (with v2 improved scaffold) vector was generated as follows: the pU6-Pgk-Puro-T2A-BFP construct was PCR amplified with overhangs for SalI + XhoI from pKLV2-U6gRNA5(BbsI)-PGKpuro2ABFP-W, then the PCR product was digested and ligated into SalI + XhoI digested pMSCV-neo vector with Quick Ligase (NEB). The pMSCV-v2-U6-(sgNon-Targeted)-Pgk-Puro-T2A-Tdtomato (with v2 improved scaffold) vector was generated as follows: first, the pU6-Pgk-Puro-T2A construct was PCR amplified from pKLV2-U6-(sgNon-Targeted)-PGKpuro2ABFP-W, and the Tdtomato construct was PCR amplified from pAAV-CAG-tdTomato (Addgene); then, the PCR product was assembled into the SalI + XhoI digested pMSCV-neo vector using the Gibson Assembly Master Mix. The pXPR_053-hPgk-VEX vector was obtained from Addgene68. The pMSCV-v2-U6-(BbsI)-Pgk-Grx1-roGFP2 vector was generated as follows: the pLPCX cyto Grx1-roGFP2 (Addgene) and pMSCV-v2-U6-(BbsI)-Pgk-Puro-T2A-BFP vectors were digested with BglII + ClaI and the Grx1-roGFP2 and pMSCV-v2-U6-(BbsI)-Pgk constructs were ligated with Quick Ligase. The pMSCV-v2-U6-(BbsI)-Pgk-mTFP1 vector was generated as follows: the mTFP1 construct was PCR amplified from pLJM1-FIRE-pHLy (Addgene), then the PCR product was assembled into the NotI + XhoI digested pKLV2-U6-(sgRNA)-PGKpuro2ABFP-W vector using the Gibson Assembly Master Mix. C57BL/6 mice were purchased from Charles River or Janvier and bred in the Core Facility for Animal Models of the Biomedical Center, LMU. R26-Cas9-eGFP animals were ordered from The Jackson Laboratory (024858) and backcrossed onto a C57BL/6 background several times. Animal care and experiments were carried out in accordance with the regulations of the applicable animal welfare acts and using protocols approved by the responsible regulatory authority (Regierung von Oberbayern, commission number 15). All animals had free access to food and water. Animals were kept at a room temperature of 22 °C ± 2 °C and humidity of 55% ± 10%, with a 12-h/12-h light–dark cycle (6:30–18:30). Male and female mice between 2 and 8 months old at the start of the experiment were used. All cells were incubated in a humidified incubator at 37 °C and 5% CO2 in air. All media were supplemented with 10% FBS (Bio&SELL) and 1% penicillin–streptomycin (Thermo Fisher). HEK293T cells (American Type Culture Colllection) were kept in DMEM GlutaMAX (Thermo Fisher). Primary bone marrow cells and Hoxb8FL cells were kept in RPMI GlutaMAX (Thermo Fisher). Hoxb8FL cells were additionally supplemented with 0.1% 2-mercaptoethanol (Thermo Fisher), 1 μM β-estradiol (Sigma) and supernatant from a Flt3L-producing B16 melanoma cell line, to a final concentration of 35 ng ml−1. During macrophage differentiation the RPMI medium was additionally supplemented with M-CSF (10–20 ng ml−1; PeproTech). T cells were kept in RMPI 1640 (Sigma) and additionally supplemented with 10 mM HEPES, 2 mM L-glutamine, 1% non-essential amino acids, 1 mM sodium pyruvate and 50 µM β-mercaptoethanol. For freezing cells, a 10% dimethylsulfoxide (Sigma) and 90% FBS (Bio&SELL) mixture was used, and cryovials were kept in a freezing box for several days at −80 °C before transferring to liquid nitrogen. Cell lines were tested for the absence of mycoplasma. HEK239T cells were detached by using 0.05% Trypsin-EDTA (Thermo Fisher). Macrophages were detached with Accutase solution (Sigma). hBCL2-overexpressing Hoxb8FL Cas9 (Hoxb8FL) lines were generated as follows. Bone marrow cells were harvested from femurs and tibias of 6–10-week-old animals and cultured in RPMI supplemented with recombinant mouse IL-3 (5 ng ml−1), IL-6 (20 ng ml−1) and 1% cell culture supernatant from SCF-producing B16 melanoma cells. After 2 days, the cells were spin infected with MSCV-ERHBD-Hoxb8FL-carrying retrovirus. A day after spin-infection, the cells were cultured in Hoxb8FL medium until infected cells were enriched in the culture in the presence of β-estradiol20. The medium was replaced every 2–3 days. The hBCL2 overexpression improved the survival of these cells both during in vitro differentiation experiments and in vivo. All sgRNA sequences are listed in Supplementary Table 1. For individual sgRNA cloning, 20-nucleotide-long sgRNAs were picked from the GPP sgRNA designer tool from the Functional Genomics Consortium of The Broad Institute, Massachusetts, USA. sgRNA sequence and reverse complemented sequence were ordered as two separate oligonucleotides from Metabion with overhangs on the 5′ side of CAAC for forward and AAAC for reverse. Complementary oligonucleotides with overhangs were phosphorylated and annealed in the presence of 10x T4 Ligation Buffer (NEB) and T4 PNK (NEB) by increasing the temperature to 95 °C and ramping down to 25 °C at 5 °C min−1. Annealed oligonucleotides were ligated into Bpil- or BsmbI-digested (Thermo Fisher) gRNA cargo plasmid by Quick Ligase (NEB) for 6 min at room temperature. Ligated plasmids were then transformed into Stellar competent cells (Takara Clontech) with heat shock at 42 °C for 55 s. Bacterial plates were incubated overnight, and single clones were picked and prepped with the Qiagen plasmid miniprep kit. The correct ligation product was confirmed with Sanger sequencing (Sequencing service, LMU Biozentrum) using the hU6 primer. sgRNAs were picked from the GPP sgRNA designer tool from Functional Genomics Consortium of The Broad Institute. All sgRNA sequences that were selected, and the library cloning primers, as well as protocols, are listed in Supplementary Table 1. sgRNAs for the mouse Cytokine Receptor library were ordered from Integrated DNA Technologies as custom oPools (50 pmol per oligonucleotide) as a 79-mer with a sequence of 5'-GCAGATGGCTCTTTGTCCTAGACATCGAAGACAACACCGN20GTTTTAGTCTTCTCGTCGCC-3', with N20 indicating the sgRNA sequence. Each gene was targeted with three different sgRNAs (Stat6 with 4 sgRNAs), and 15 non-targeted (from here onward, control) control gRNAs were also included (343 oligonucleotides in total). The oligonucleotide pool was dissolved in Qiagen TE buffer to get a 100 μM stock concentration, and then the single-stranded oligonucleotides (100 ng) were PCR amplified for two cycles with Q5 High-Fidelity DNA Polymerase (NEB) using Oligo_Amp_F and Oligo_Amp_R primers to generate double-stranded DNAs. The PCR products were purified with the Nucleotide Removal Kit (Qiagen). Amplified double-stranded DNAs were digested with FastDigest BpiI (BbsI, Thermo Fisher) for 2 h at 37 °C in a total of two reactions, and then purified with the Nucleotide Removal Kit (Qiagen). Ligation was performed with a T4 DNA Ligase (NEB) using a 3 ng insert and 40 ng BpiI-digested MSCV-v2-U6-(BbsI)-Pgk-Puro-T2A-eGFP for 16 h at 16 °C per reaction in a total of two reactions. The ligated product was cleaned with a PCR Purification Kit (Qiagen), and the concentration was measured with Qubit 4 (Thermo Fisher). Ten nanograms of the ligated product was transformed into 50 μl of NEB Stable Competent cells in a total of eight reactions and incubated at 30 °C overnight. A library representation above 100× was confirmed by plating transformed competent cells in serial dilutions. The plasmid DNA was prepared with an Endofree Plasmid Maxi Kit (Qiagen). For viral transduction of Hoxb8FL cells, HEK293T cells were plated into a six-well plate 18–24 h before transfection. For the transfection per well of six-well plate, 1.5 µg pMSCV retroviral plasmid and 1.5 µg pCL-Eco packaging vector were added in 400 µl of RPMI medium without serum and antibiotics. Then 7.5 μl of the transfection reagent TransIT-LT1 were added into the mix, vortexed, and then incubated at room temperature for 30 min. After 30 min, the solution was added dropwise to HEK293T cells. Retrovirus-containing supernatant was collected at 48–72 h after transfection. Hoxb8FL cells were spin infected with freshly harvested virus at 1,200g for 30 min at room temperature. For individual KO cell line generation, individual sgRNA viruses were produced and transduced. For the CRISPR screen experiments, the library oligonucleotide pool was used to generate a virus mix containing all the library sgRNAs, which was then transduced at a maximum multiplicity of infection of 0.3 or below (< 30% transduction efficiency) to prevent multiple integrations of sgRNAs into a single cell, and enough cell numbers were always kept ensuring a minimum 1,000× coverage (1,000 cells having the same sgRNA). For the Perturb-seq experiment, viruses for six individual non-targeted sgRNAs and two different sgRNAs per gene targeted were produced individually, and single KO or control cell lines were generated in parallel and mixed in equal proportions before expansion before i.v. On the next day, puromycin (Thermo Fisher) was added at a final concentration of 5 µg ml−1 to select for transduced cells. After 4 days of puromycin selection, cells expressing fluorescent markers (for example, eGFP) were sorted to purity to ensure high expression of the fluorescent marker by using a FACS Aria III (BD) or FACS Fusion (BD) at the Flow Cytometry Core Facility of the Biomedical Center, LMU. The combination of positive selection of the infected cells by puromycin with sorting of the pure fluorescent population ensured all the Hoxb8FL transferred cells in the in vivo experiments were gene edited. All single KO lines used in validation experiments were assessed for KO efficiency before the experiment with the TIDE assay. This assay works as follows: to assess the extent of genetic editing at DNA level for single sgRNAs, genomic DNA from sgRNA transduced (see above) Hoxb8FL cells was isolated with the DNeasy Blood and Tissue Kit (Qiagen), and the DNA region targeted by the sgRNA was amplified with Q5 High-Fidelity DNA Polymerase (NEB) and specific PCR primers for each different sgRNA. All TIDE primers are listed in Supplementary Table 1. The samples from KO and control cells were submitted to Sanger Sequencing with either the forward or the reverse PCR primer (Sequencing service, LMU Biozentrum). The Cas9-dependent generation of insertion–deletion repair errors and the subsequent KO efficiency (frame shifts that are not a multiple of 3 are considered to generate a protein KO) were assessed by the ICE v2 software tool80. In vitro-expanded Hoxb8FL cells were washed twice with PBS to remove β-estradiol and cultured in RPMI medium (with 10 ng ml−1 M-CSF) for 2 days to initiate myeloid differentiation. On day 8–9 after active EAE induction, or at disease onset in TH17 adoptive transfer EAE (onset of clinical symptoms or >1 g of weight loss, or else the Hoxb8FL transfer stopped disease development), cells were washed twice with PBS and 10–15 × 106 cells in 200 μl of PBS were injected i.v. For the intravital imaging experiments, Hoxb8FL cells were labeled before transfer with CellTrace Far Red (Thermo Fisher Scientific, C34564), CellTrace Yellow (Thermo Fisher Scientific, C34567) or CellTrace Violet (Thermo Fisher Scientific, C34571) for in vivo detection. After reconstituting the CellTrace according to the manufacturer's instructions, cells were collected and the residual medium was washed once with PBS. Then, the cells were incubated 30 min at 37 °C in the dark in a 1:500 dilution of CellTrace solution in PBS at a concentration of 5 × 106 cells per ml. Staining was stopped by adding five times the volume of cell medium and incubating for 5 min at 37 °C in the dark, followed by two PBS washes before injection. Animals with EAE clinical signs were euthanized by isoflurane overdose. When CSF was collected, it was collected immediately after euthanasia from the cisterna magna. Briefly, the animal was head-fixed in a stereotactic frame, and the cisterna magna was exposed by keeping the head bent down at a 45° angle and removing skin and muscle from the neck. Then, a pulled glass pipette was used to carefully penetrate the cisterna magna and suction the CSF. Between 10 μl and 25 μl of CSF was collected per animal and immediately resuspended on FACS buffer (0.5% BSA, 1 mM EDTA, 25 mM HEPES on calcium and magnesium-free PBS) on ice. Blood samples were collected immediately after euthanasia or after CSF extraction by cardiac puncture, and the animal's body was perfused with PBS-heparin before collecting the spleen, inguinal lymph nodes, bone marrow (femur and vertebra), spinal cord and brain cortex through microdissection. Cells from bone marrow were collected by flushing with ice-cold PBS, whereas cells from vertebrae were isolated after crushing with a pestle and mortar. Spleen, lymph nodes, spinal cord and brain tissues were homogenized with a glass dounce homogenizer (Wheaton) with a loose pestle, transferred into a PBS solution containing collagenase D (0.8 mg ml−1; Roche) and DNase I (10 ng ml−1; Roche) and incubated for 20 min at 37 °C with shaking (1,000 rpm) for dissociation of cells. For scRNA experiments, cells were incubated in digestion buffer for only 5 min to minimize changes to the transcriptome. Cells from spleen, lymph nodes (after digestion) and bone marrow were treated with ACK Lysing Buffer (Thermo Fisher) for 5 min, whereas cells from blood were incubated for 20 min on ice for red blood cell lysis. All cell suspensions were passed through a 100-μm-pore-diameter cell strainer (Corning). Cells from spinal cord and brain were isolated with Percoll (Sigma) gradient to remove the myelin. Cells were resuspended with 1 ml of 100% FBS and 9 ml of 33% Percoll (in PBS), and 1 ml of 10% FBS (in PBS) was added on the top slowly to form a layer. The myelin layer was carefully sucked with a vacuum, and pelleted cells were washed with PBS to remove the Percoll solution. For the cytokine library experiment, around ten animals were combined for one replicate to ensure sufficient cells (as, depending on the disease score, cell numbers found in the spinal cord vary). Before starting flow cytometry labeling, monocytes/macrophages from the spleen and bone marrow were enriched with MACS using anti-CD11b microbeads (Milteny) to reduce sorting time. All cells were blocked with TruStain fcX (anti-mouse CD16/CD32, BioLegend) and stained with Live/Dead Near-IR (Thermo Fisher) to exclude dead cells during blocking for 30 min at 4 °C. After washing with PBS, cells were incubated with antibodies recognizing extracellular markers for 30 min at 4 °C, with anti-CD11b-PerCP antibody to include monocytes/macrophages and anti-Ly6G-BV786 antibody to exclude granulocytes. Cells from the spinal cord were then fixed and permeabilized using Cytofix/Cytoperm kit (BD Biosciences) for 15 min at 4 °C, before intracellular labeling. Cells were incubated with anti-Arg1-APC and anti-iNOS-Pacific Blue antibodies for 30 min at 4 °C in Perm/Wash buffer solution. Hoxb8FL-derived cells were identified by their eGFP fluorophore expression. Populations of interest were isolated using a FACS Aria III (BD) or FACS Fusion (BD) at the Flow Cytometry Core Facility of the Biomedical Center. Enough cells were sorted for each population to ensure a library coverage of a minimum of 100× (100 cells containing each sgRNA). Genomic DNA from sorted cells was isolated by using the QIAamp DNA Micro Kit. Cells were lysed in the presence of Proteinase K at 56 °C overnight (modified from the manufacturer's protocol) for more efficient de-crosslinking of fixed DNA before isolation of genomic DNA. For in vitro macrophage polarization experiments, bone marrow-derived cells were cultured in RPMI medium with 10 ng ml−1 M-CSF for 5–7 days to allow differentiation. Differentiated cells were then re-seeded and the respective recombinant mouse cytokines were added at a concentration of 10–20 ng ml−1 as reported in the literature81 for 48 h before FACS: IL-4 (R&D Systems), TGFβ (BioLegend), GM-CSF (PeproTech), IFNγ (R&D Systems) or TNF (PeproTech) were used. Cells were detached with Accutase followed by two washing steps with PBS. Cells were fixed and permeabilized using the Cytofix/Cytoperm kit for 15 min at 4 °C before incubation with anti-Arg1 and anti-iNOS antibodies for 30 min at 4 °C in Perm/Wash buffer solution. Cells were analyzed with a Fortessa (BD) or a Cytoflex S (Beckman Coulter). For characterization experiments of Hoxb8FL-derived cells, nonspecific binding was blocked with TruStain fcX (anti-mouse CD16/32, BioLegend) and cells were stained with Live/Dead reagent (Thermo Fisher) to exclude dead cells, for 30 min at 4 °C. After washing with PBS, cells were incubated with antibodies recognizing extracellular markers for 30 min at 4 °C: anti-CD11b, anti-CD45, anti-Ly6G, anti-Ly6C, anti-MHC class II and anti-F4/80 were used. Cells then were incubated with anti-Arg1 and anti-iNOS antibodies for 30 min at 4 °C in Perm/Wash buffer solution. All antibodies—unless otherwise stated—were used at 1:100 dilution. Hoxb8FL-derived cells were identified by eGFP fluorophore expression, and eGFP-negative cells were assigned as the endogenous population. Cells then were analyzed using a FACS Aria III (BD) or FACS Fusion (BD) at the Flow Cytometry Core Facility of the Biomedical Center. Neutrophils were excluded from all analyses by expression of Ly6G. For single KO validation experiments, before the transfer of Hoxb8FL cells into animals, Hoxb8FL cells expressing tdTomato and control sgRNA were mixed at a 1:1:1 ratio with Hoxb8FL cells expressing either BFP or eGFP and the gene targeting sgRNA to minimize the inter-animal experimental variation. For the 10x scRNA-seq experiments, before euthanasia, 3 μg per mouse of anti-CD45 antibody was injected i.v. in the tail vein to permit exclusion of cells derived from blood. Cells from 7 mice were pooled from a single EAE round for the WT EAE experiment; cells from 6 (round 1) and 12 (round 2) mice were pooled from two independent Hoxb8FL cell generation and EAE rounds for the Hoxb8FL-derived cell experiment; for the chimera experiment, three mice each with control, Ifngr1-KO, Tnfrsf1a-KO and Csfr2ra-KO BM chimera, and two mice with Tgfbr1-KO BM chimera were pooled; for CSF sequencing, 11 (round 1) and 12 mice (round 2) were pooled; for cEAE, 3 healthy, 3 d3 cEAE and 3 d14 cEAE mice were pooled. All cells were incubated with TruStain fcX (anti-mouse CD16/32, BioLegend) and Live/Dead Near-IR (Thermo Fisher) to exclude dead cells for 30 min at 4 °C. After washing with PBS, cells were incubated with anti-CD11b-PerCP antibody to include monocytes/macrophages and anti-Ly6G-BV786 antibody to exclude granulocytes, for 30 min at 4 °C. For the cEAE experiment, cells were additionally incubated with CD45-PE antibody to include all immune cells. Hoxb8FL-derived cells were identified by eGFP fluorophore expression, and eGFP-negative cells were assigned as the endogenous population. Chimeric cells were identified by Vex fluorophore expression. Populations of interest (WT, Hoxb8FL and chimeric EAE: CD11b+Ly6G−; CSF and cEAE CD45+Ly6G−) were sorted for purity using a FACS Aria III (BD) or FACS Fusion (BD) at the Flow Cytometry Core Facility of the Biomedical Center. Sorted cells were immediately prepared for 10x single-cell experiments according to the 10x protocol. The gating strategy to sort for Ly6G− cells resulted in minimal numbers of neutrophils in the spinal EAE scRNA-seq experiments: 310 cells, 2% of total, in the WT dataset, 31 cells, 0.28% of total, in the Hoxb8FL-transfer dataset and 174 cells, 0.53% of total, in the bone-marrow chimera dataset. No statistical methods were used to predetermine sample sizes, but our sample sizes are similar to those reported in previous publications8,82,83. For next-generation sequencing sample demultiplexing, Je-Demultiplex85 was used in Galaxy86. For analysis of CRISPR screens, cutadapt87, trimmomatic88, the MAGeCK software89, Galaxy86 and R84 were used. For analysis of bulk Galaxy86, RNA STAR90 (version 2.7.2b in Galaxy), HTSeq-count91 (version 1.0.0 in galaxy), DESeq2 (ref. For image analysis, Fiji/ImageJ93 and Imaris (Oxford Instruments) were used. For scRNA-seq data, Seurat94,95,96,97,98 (versions 4+) and R84 were used. Figures were made using the aforementioned software and Adobe Illustrator 2025. Data are represented as the mean ± standard deviation unless otherwise stated; box plots show the median and quartiles 1 and 3 for the box limits, and whiskers extend to 1.5 times the interquartile range. Sample sizes are reported in the figure legends. All replicates are biological unless otherwise stated, measurements were not repeated, and all tests are two tailed unless otherwise stated. The choice to do a one-tail or two-tailed test depended on whether there were prior data suggesting the expected effect would be in a specific direction. Animals were randomly assigned to experiments or, when not possible, the conditions were appropriately blocked such that there were animals of the same condition in all cages. Due to the experimental design and animal laws, no blinding was possible during FACS experiments and Hoxb8FL characterization histology. The analysis of all histological data was done blinded. scRNA-seq exclusion criteria for low quality control are detailed in the Supplementary Methods. No other data were excluded. For statistical testing of the cell distribution in the lesion (Fig. 1b), an ordinary two-way ANOVA with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons was used. For the Hoxb8FL transfer experiments (Figs. 1, 2 and 7), multiple paired t-tests or Wilcoxon tests (when the sample was not normally distributed) with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons with FDR (q) of 5% when needed were used. For comparison of EAE disease course in WT and Hoxb8FL transferred animals (Extended Data Fig. For the intravital imaging experiments (Figs. 4 and 5), for the proportion of cells in each ratio bin, an ordinary two-way ANOVA with a single pooled variance, for the interaction of genotype and bin was run; a Kolmogorov–Smirnov test was run for cumulative distributions, and a one-tailed unpaired (Fig. 5) t-test for the proportion of cells above Q3 of the ratio. For the histological quantifications, paired t-test or Wilcoxon tests were run. For the cholesterol efflux assay, unpaired t-tests were run. For the pHrodo-myelin phagocytosis assay, an ordinary two-way ANOVA with a single pooled variance for the genotype differences was run. For the single KO phenotype differences in polarization between Hoxb8FL and chimera, unpaired t-tests or Wilcoxon tests were run. For the KO/control distribution phenotype (Fig. 6), paired t-tests or Wilcoxon tests comparing KO and control were used, but data are plotted as the KO/control ratio for visualization purposes only. For the motility analysis, paired t-tests were run. For CRISPR screen results (Figs. 2 and 3), the MAGeCK software was used as described above. Asterisks indicate P value and FDR < 0.05 and absolute log2(fold change) > 3 times the standard deviation of the noise distribution, as previously described. For comparison of in vitro bone marrow-derived macrophage polarization conditions (Extended Data Fig. 4), an ordinary one-way ANOVA with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons was used. Bulk RNA-seq differentially expressed genes (Extended Data Fig. 4) were defined as having an adjusted P value < 0.05 and log2(fold change) > 3 times the standard deviation of all log2(fold change) in the comparison. For individual gene KO-versus-control phenotypes in the scRNA-seq experiments (Extended Data Figs. 5, 6 and 7), the FindMarkers function of the Seurat package (version 5) was used with default parameters other than logfc.threshold = 0 and min.pct = 0.1, and differentially expressed genes were defined as adjusted P value < 0.05 and absolute log2(fold change) > 3 times the standard deviation of the log2(fold change) distribution of the cluster and as described above for Fig. 4 and 5), the GSEA preranked function of the GSEA software99,100 (version 4.2.3) was used, with a custom-made database of the signatures, as described elsewhere. A signature was considered significantly enriched when the NOM P value < 0.05 or the FDR q value < 0.25 and absolute NES > 1.5. For unbiased pathway analysis (Fig. 6), the pathways depicted in the figure were filtered based on the output parameters of gprofiler101 as described above. For statistical analysis of human samples (Fig. 8), the Shapiro normality test was used to evaluate the normal distribution of the sample, and after, according to the results, a one-way ANOVA was performed for each cytokine signature for newly generated CSF samples followed by t-tests or Wilcoxon tests comparing disease conditions against controls. For datasets from Schafflick et al.32 and Piehl et al.33, t-tests or Wilcoxon tests comparing disease conditions against controls were used. For samples from Macnair et al.35 and Mathys et al.36, the Shapiro normality test was used to evaluate the normal distribution of the sample, and after, according to the results, a one-way ANOVA or Kruskal–Wallis test per cytokine-induced signature per dataset was ran, followed by t-tests or Wilcoxon tests comparing all disease/lesion conditions to control (gray matter or healthy, respectively) and all clusters to the homeostatic cluster. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated or analyzed during this study is included in the Supplementary Information, on Zenodo https://doi.org/10.5281/zenodo.15808138 (ref. 102) or via the corresponding authors upon reasonable request. Source data are provided with this paper. All custom code used for analysis during this study is included in the published article as Supplementary Information or is available from the corresponding authors upon reasonable request. Attfield, K. E., Jensen, L. T., Kaufmann, M., Friese, M. A. & Fugger, L. The immunology of multiple sclerosis. Henderson, A. P., Barnett, M. H., Parratt, J. D. & Prineas, J. W. Multiple sclerosis: distribution of inflammatory cells in newly forming lesions. Bitsch, A., Schuchardt, J., Bunkowski, S., Kuhlmann, T. & Bruck, W. Acute axonal injury in multiple sclerosis. Correlation with demyelination and inflammation. Moccia, M. et al. Assessing disability and relapses in multiple sclerosis on tele-neurology. Huitinga, I., van Rooijen, N., de Groot, C. J., Uitdehaag, B. M. & Dijkstra, C. D. Suppression of experimental allergic encephalomyelitis in Lewis rats after elimination of macrophages. Ajami, B., Bennett, J. L., Krieger, C., McNagny, K. M. & Rossi, F. M. Infiltrating monocytes trigger EAE progression, but do not contribute to the resident microglia pool. Hwang, D. et al. CSF-1 maintains pathogenic but not homeostatic myeloid cells in the central nervous system during autoimmune neuroinflammation. Locatelli, G. et al. Mononuclear phagocytes locally specify and adapt their phenotype in a multiple sclerosis model. Mendiola, A. S. et al. Transcriptional profiling and therapeutic targeting of oxidative stress in neuroinflammation. Miron, V. E. et al. M2 microglia and macrophages drive oligodendrocyte differentiation during CNS remyelination. Harnessing the benefits of neuroinflammation: generation of macrophages/microglia with prominent remyelinating properties. Locati, M., Curtale, G. & Mantovani, A. Diversity, mechanisms, and significance of macrophage plasticity. Sanin, D. E. et al. A common framework of monocyte-derived macrophage activation. & Miller-Jensen, K. Reframing macrophage diversity with network motifs. Giles, D. A. et al. Myeloid cell plasticity in the evolution of central nervous system autoimmunity. Bock, C. et al. High-content CRISPR screening. Schraivogel, D., Steinmetz, L. M. & Parts, L. Pooled genome-scale CRISPR screens in single cells. Covarrubias, S. et al. High-throughput CRISPR screening identifies genes involved in macrophage viability and inflammatory pathways. Shi, J. et al. A genome-wide CRISPR screen identifies WDFY3 as a regulator of macrophage efferocytosis. Redecke, V. et al. Hematopoietic progenitor cell lines with myeloid and lymphoid potential. Huang, X., Li, Y., Fu, M. & Xin, H. B. Polarizing macrophages in vitro. Giladi, A. et al. Cxcl10+ monocytes define a pathogenic subset in the central nervous system during autoimmune neuroinflammation. Amorim, A. et al. IFNγ and GM-CSF control complementary differentiation programs in the monocyte-to-phagocyte transition during neuroinflammation. Fischer, M. T. et al. NADPH oxidase expression in active multiple sclerosis lesions in relation to oxidative tissue damage and mitochondrial injury. Cross, A., Manning, P., Keeling, R., Schmidt, R. & Misko, T. Peroxynitrite formation within the central nervous system in active multiple sclerosis. Nikić, I. et al. A reversible form of axon damage in experimental autoimmune encephalomyelitis and multiple sclerosis. Gutscher, M. et al. Real-time imaging of the intracellular glutathione redox potential. Defective cholesterol clearance limits remyelination in the aged central nervous system. The unique phenotype of lipid-laden macrophages. Androvic, P. et al. Spatial transcriptomics-correlated electron microscopy maps transcriptional and ultrastructural responses to brain injury. Dictionary of immune responses to cytokines at single-cell resolution. Integrated single cell analysis of blood and cerebrospinal fluid leukocytes in multiple sclerosis. Piehl, N. et al. Cerebrospinal fluid immune dysregulation during healthy brain aging and cognitive impairment. Schirmer, L. et al. Neuronal vulnerability and multilineage diversity in multiple sclerosis. Macnair, W. et al. snRNA-seq stratifies multiple sclerosis patients into distinct white matter glial responses. Mathys, H. et al. Single-cell transcriptomic analysis of Alzheimer's disease. Yshii, L. et al. Astrocyte-targeted gene delivery of interleukin 2 specifically increases brain-resident regulatory T cell numbers and protects against pathological neuroinflammation. LaFleur, M. W., et al. X-CHIME enables combinatorial, inducible, lineage-specific and sequential knockout of genes in the immune system. Cao, X. et al. Irradiation induces bone injury by damaging bone marrow microenvironment for stem cells. Busch, K. & Rodewald, H. R. Unperturbed vs. post-transplantation hematopoiesis: both in vivo but different. Characterization of the long-term effects of lethal total body irradiation followed by bone marrow transplantation on the brain of C57BL/6 mice. & Quintana, F. J. Microglia and central nervous system–associated macrophages—from origin to disease modulation. Klotz, L. et al. Broad rim lesions are a new pathological and imaging biomarker for rapid disease progression in multiple sclerosis. Z. Macrophage diversity in cancer revisited in the era of single-cell omics. Dixit, A. et al. Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens. In vivo perturb-seq reveals neuronal and glial abnormalities associated with autism risk genes. Arellano, G., Ottum, P. A., Reyes, L. I., Burgos, P. I. & Naves, R. Stage-specific role of interferon-gamma in experimental autoimmune encephalomyelitis and multiple sclerosis. & Johnson, K. P. Treatment of multiple sclerosis with gamma interferon: exacerbations associated with activation of the immune system. Maintenance of chronic neuroinflammation in multiple sclerosis via interferon signaling and CD8 T cell-mediated cytotoxicity. Tichauer, J. et al. Interferon-gamma ameliorates experimental autoimmune encephalomyelitis by inducing homeostatic adaptation of microglia. Therapeutic role of interferon-γ in experimental autoimmune encephalomyelitis is mediated through a tolerogenic subset of splenic CD11b+ myeloid cells. Stoolman, J. et al. An IFNγ/CXCL2 regulatory pathway determines lesion localization during EAE. Ruddle, N. H. et al. An antibody to lymphotoxin and tumor necrosis factor prevents transfer of experimental allergic encephalomyelitis. Baker, D. et al. Control of established experimental allergic encephalomyelitis by inhibition of tumor necrosis factor (TNF) activity within the central nervous system using monoclonal antibodies and TNF receptor-immunoglobulin fusion proteins. Eugster, H. P. et al. Severity of symptoms and demyelination in MOG-induced EAE depends on TNFR1. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the monoclonal anti-tumor necrosis factor antibody cA2. The Lenercept Multiple Sclerosis Study Group and The University of British Columbia MS/MRI Analysis Group. TNF neutralization in MS: results of a randomized, placebo-controlled multicenter study. The Lenercept Multiple Sclerosis Study Group and The University of British Columbia MS/MRI Analysis Group. Gregory, A. P. et al. TNF receptor 1 genetic risk mirrors outcome of anti-TNF therapy in multiple sclerosis. Madsen, P. M. et al. Oligodendroglial TNFR2 mediates membrane TNF-dependent repair in experimental autoimmune encephalomyelitis by promoting oligodendrocyte differentiation and remyelination. The cytokine GM-CSF drives the inflammatory signature of CCR2+ monocytes and licenses autoimmunity. Identification of a broadly fibrogenic macrophage subset induced by type 3 inflammation. Galli, E. et al. GM-CSF and CXCR4 define a T helper cell signature in multiple sclerosis. Constantinescu, C. S. et al. Randomized phase 1b trial of MOR103, a human antibody to GM-CSF, in multiple sclerosis. Multiple sclerosis severity variant in DYSF-ZNF638 locus associates with neuronal loss and inflammation. Distinct transcriptional changes distinguish efficient and poor remyelination in multiple sclerosis. Vliet, D. V. D. et al. Foamy microglia link oxylipins to disease progression in multiple sclerosis. Fatal demyelinating disease is induced by monocyte-derived macrophages in the absence of TGFβ signaling. Bosch-Queralt, M. et al. Diet-dependent regulation of TGFβ impairs reparative innate immune responses after demyelination. Hao, Y., Baker, D. & Ten Dijke, P. TGF-β-mediated epithelial-mesenchymal transition and cancer metastasis. Transforming growth factor-β1 regulates macrophage migration via RhoA. Perrot, C. Y. et al. Mast-cell expressed membrane protein-1 is expressed in classical monocytes and alveolar macrophages in idiopathic pulmonary fibrosis and regulates cell chemotaxis, adhesion, and migration in a TGFbeta-dependent manner. Parsa, R. et al. TGFβ regulates persistent neuroinflammation by controlling Th1 polarization and ROS production via monocyte-derived dendritic cells. Bai, Z. et al. Cerebrospinal fluid and blood cytokines as biomarkers for multiple sclerosis: a systematic review and meta-analysis of 226 studies with 13,526 multiple sclerosis patients. Bras, J. P. et al. TNF-alpha-induced microglia activation requires miR-342: impact on NF-kB signaling and neurotoxicity. Stanley, E. R., Biundo, F., Gokhan, S. & Chitu, V. Differential regulation of microglial states by colony stimulating factors. Spittau, B., Dokalis, N. & Prinz, M. The role of TGFβ signaling in microglia maturation and activation. Ran, F. A. et al. Genome engineering using the CRISPR-Cas9 system. Inference of CRISPR edits from Sanger trace data. Ying, W., Cheruku, P. S., Bazer, F. W., Safe, S. H. & Zhou, B. Investigation of macrophage polarization using bone marrow derived macrophages. Jafari, M. et al. Phagocyte-mediated synapse removal in cortical neuroinflammation is promoted by local calcium accumulation. Kendirli, A. et al. A genome-wide in vivo CRISPR screen identifies essential regulators of T cell migration to the CNS in a multiple sclerosis model. R: a language and environment for statistical computing. Girardot, C., Scholtalbers, J., Sauer, S., Su, S. Y. & Furlong, E. E. Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers. Galaxy, C. The Galaxy platform for accessible, reproducible, and collaborative data analyses: 2024 update. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Anders, S., Pyl, P. T. & Huber, W. HTSeq–a Python framework to work with high-throughput sequencing data. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Integrated analysis of multimodal single-cell data. Dictionary learning for integrative, multimodal and scalable single-cell analysis. A., Gennert, D., Schier, A. F. & Regev, A. Spatial reconstruction of single-cell gene expression data. Stuart, T. et al. Comprehensive integration of single-cell data. Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Kolberg, L. et al. g:Profiler-interoperable web service for functional enrichment analysis and gene identifier mapping (2023 update). In vivo CRISPR screen reveals regulation of macrophage states in neuroinflammation. We thank A. Schmalz and B. Fiedler for excellent technical assistance, D. Matzek, B. Stahr, N. Ntaraklitsas for animal husbandry. We further wish to thank L. Robinson of Insight Editing London for critical review and editing of the manuscript. We also thank LAFUGA (Stefan Krebs, Helmut Blum) and CCGA Kiel (Sören Franzenburg, Janina Fuß) for next-generation sequencing, the Core Facility Flow Cytometry at the BMC (B. Tast, L. Richter and P. Khosravani) for supporting flow cytometry and cell sorting and the LMU Biozentrum Sequencing Service for Sanger sequencing. For advice or assistance during the experiments, we thank A. Ricci, Z. Zebec and S. Anak. 's laboratory is supported by grants from the Deutsche Forschungsgemeinschaft (DFG; including Collaborative Research Center -Transregio128, projects B10 and B13; Transregio 167 – project ID 259373024, Transregio 274 - project ID 408885537, projects C02, C05; the Research Unit FOR5705 – KE774/6-1), the Munich Cluster for Systems Neurology (SyNergy EXC 2145 – ID 390857198), the ‘Klaus-Faber Stiftung' and the ‘Verein Therapieforschung für MS-Kranke e.V.'. are further supported by a grant from the DFG (Transregio TRR152 - project ID 239283807, project P27). and M.S.-S. were funded by the European Research Council under the European Union's Horizon 2020 research and innovation program (grant agreement 682435). received support from the Fundación Rafael del Pino and is part of the DFG-funded Graduate School of Systemic Neurosciences (GSC 82 - ID 24184143). pen access funding provided by Ludwig-Maximilians-Universität München. These authors contributed equally: Clara de la Rosa, Arek Kendirli. Institute of Clinical Neuroimmunology, University Hospital, Ludwig-Maximilians-Universität Munich, Munich, Germany Clara de la Rosa, Arek Kendirli, Franz Bauernschmitt, Anna S. Thomann, Ilgin Kisioglu, Daniela Beckmann, Yves Carpentier Solorio, Veronika Pfaffenstaller, Yi-Heng Tai, Niel Mehraein, Paula Sanchez, Lisa Ann Gerdes, Eduardo Beltran, Klaus Dornmair, Anneli Peters & Martin Kerschensteiner Biomedical Center (BMC), Medical Faculty, Ludwig-Maximilians-Universität Munich, Martinsried, Germany Clara de la Rosa, Arek Kendirli, Franz Bauernschmitt, Anna S. Thomann, Ilgin Kisioglu, Daniela Beckmann, Yves Carpentier Solorio, Veronika Pfaffenstaller, Yi-Heng Tai, Niel Mehraein, Paula Sanchez, Lisa Ann Gerdes, Eduardo Beltran, Klaus Dornmair, Anneli Peters & Martin Kerschensteiner Clara de la Rosa, Daniela Beckmann, Yves Carpentier Solorio & Veronika Pfaffenstaller Arek Kendirli, Lisa Ann Gerdes, Mikael Simons & Martin Kerschensteiner Institute of Experimental Haematology, Center for Translational Cancer Research (TranslaTUM), School of Medicine and Health, Technical University Munich, Munich, Germany Institute of Neuronal Cell Biology, Technical University Munich, Munich, Germany German Center for Neurodegenerative Diseases (DZNE), Munich, Germany Institute of Stroke and Dementia Research, University Hospital, Ludwig-Maximilians-Universität Munich, Munich, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the experiments. performed single-cell transcriptomic analysis in mice, and F.B. performed all bioinformatic and statistical analysis. contributed to in vivo microscopy analysis. performed histological analysis and N.M. performed in vitro experiments on bone marrow-derived macrophages. performed and supervised the in vitro analysis of lipid efflux. contributed to the TH17 transfer EAE experiments and L.A.G., E.B. wrote the paper with input from all authors. Correspondence to Arek Kendirli or Martin Kerschensteiner. The authors declare no competing interests. Nature Neuroscience thanks V. Wee Yong and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Representative flow cytometry plots of myeloid differentiation during the in vitro 48h treatment before i.v. b, Percentage of myeloid Hoxb8FL-derived cells in vitro and in vivo (left) and quantification of the Hoxb8FL-derived myeloid cells in different tissue compartments in vivo. N = 4 animals c, Correlation of bulk RNA transcript counts in sorted Hoxb8FL-derived monocytes, compared to the same population of endogenous blood monocytes. d, Representative flow cytometry plots of the transferred Hoxb8FL-derived cells (bottom row, green) compared to endogenous cells (top row, gray) in the blood of the same mouse with EAE at the peak of disease, with two different panels (left) and quantification (right). N = 4 animals for the myeloid marker panel and three animals for the polarization marker panel. e-h, Percentage of Hoxb8FL-derived cells (green) and endogenous cells (gray) in the same animal in the spleen (h), femur bone marrow (i), vertebrae bone marrow (j) and inguinal draining lymph nodes (k), at the peak of EAE. i, Histological characterization of the distribution of Hoxb8FL-derived and endogenous myeloid cells in EAE lesions in the spinal cord pia matter and parenchyma, blue dashed line indicates pia-parenchyma separation. Right, percentage of total endogenous (gray) or Hoxb8FL (green) cells present in the pia (top) or parenchymal (bottom) compartments of the lesion. N = 3 animals, 3 sections per animal. j, Representative image of the meningeal niche containing CD206+ resident macrophages but no infiltrated Hoxb8FL-derived myeloid cells. Dashed line indicates the dura. k, Representative image of the perivascular niche containing Lyve+ resident macrophages but no infiltrated Hoxb8FL-derived myeloid cells. Dashed line indicates the vessel lumen. l, Scheme of the experimental approach to study the migration phenotype of Ccr2-KO myeloid cells. m, Representative flow cytometry plots of control-tdTomato or Ccr2-KO GFP Hoxb8FL-derived myeloid cells in the bone marrow (left) or spinal cord (right) of a mouse with EAE at the peak of disease. n, Relative amounts of control-derived and Ccr2-KO-derived monocytes/macrophages or granulocytes across different tissues in mice with EAE at the peak of disease. o, EAE disease course of mice with (green) or without (gray) a Hoxb8FL cell transfer. N = 40 animals per group p-q, Correlation of bulk RNA transcript counts in sorted polarized Hoxb8FL-derived macrophages compared to the same population of endogenous spinal cord iNOS+ (p) or Arg1+ (q) macrophages at the peak of EAE. (a) Two-tailed paired t-test; (d-h) multiple two-tailed paired t-tests or Wilcoxon tests with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons; (n) ordinary one-way ANOVA left F = 87.58 P < 0.0001, right F = 1.242, P = 0.3274; Kolmogorov-Smirnov test used in (o); ns p-value > 0.05, * p-value < 0.05, ** p-value < 0.01, *** p-value < 0.001, **** p-value < 0.0001; figures show mean ± sd. a, Representative flow cytometry gating to assess gene KOs in Hoxb8FL-derived myeloid cells. b, Representative flow cytometry plots of myeloid cell polarization for Ifngr1-KO, Tnfrsf1a-KO, Tgfbr1-KO and Csf2ra-KO with their intra-animal control. c-f, Percentage of iNOS+ (c, e) and Arg1+ (d, f) cell polarization for Il4ra-KO and the downstream transcription factor Stat6-KO (c, d) and Il10ra-KO (e, f) compared to control (barplot). On the right, scheme of the cytokine signaling pathways, color-coded for the phenotype of the gene KO in M-iNOS (c, e) and M-Arg1 (d, f) polarization derived from the CRISPR screen shown in Fig. a, Scheme of the Hoxb8FL cell transfer in the Th17 T cell adoptive transfer EAE model. b, Cytokine receptors affecting M-Arg1 polarization, determined by comparing the Arg1+ spinal cord macrophages (13–29% of total Hoxb8FL-derived macrophages, n = 2 experiments) to the iNOS and Arg1 double negative macrophages (71–87% of total Hoxb8FL-derived macrophages, n = 2 experiments). Stars in (b, c) indicate p-value and fdr < 0.05 and absolute log2(Fold Change) > 3 x sd of noise distribution (see Methods for details). a, e, Representative flow cytometry plots of iNOS (a) or Arg1 (e) induction in bone-marrow derived macrophages (BMDMs) in different cytokine incubation conditions. b, f, Percentage of iNOS (b) or Arg1 (f) positive cells among BMDMs in different cytokine incubation conditions. N = 8 in vitro experiments, except for conditions TNFα + TGFβ and TNFα + GMCSF which have n = 3. c, g, Heatmap of genes induced by different cytokine combinations in BMDMs during M-iNOS (c) or M-Arg1 (g) polarization across biological replicates, color coded for the log2(Fold Change) value of the replicate vs the mean of the naïve M0 macrophages. Naïve macrophage and four cytokine combination samples are repeated in both plots, showing different genes. Only the genes significantly upregulated in any of the shown cytokine combinations compared to naïve macrophages are included on the heatmap. d, h, Venn diagram of the overlap in differentially regulated genes induced in BMDMs by incubation with the listed cytokines compared to naïve M0 macrophages. In brackets, total number of DEGs per cytokine condition. Purple to yellow color code indicates increasing numbers of differentially regulated genes. Ordinary one-way ANOVA with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli for multiple comparisons used in (b, f); (b) F = 244.6, P < 0.0001, (f) F = 77.85, P < 0.0001; ns p-value > 0.05, * p-value < 0.05, ** p-value < 0.01, *** p-value < 0.001, **** p-value < 0.0001; figures show mean ± sd; Significantly upregulated genes in (c, g) were defined as having an adjusted p-value < 0.05 and log2(Fold Change) > 3 x standard deviation of all log2(Fold Change) in the comparison. a, Expression of key genes determining cluster cell identity in the scRNAseq experiments across clusters. b-d, Correlation of the gene log2(Fold Change) of each cluster vs all the other clusters between the WT EAE macrophage experiment clusters and the Hoxb8FL-derived macrophage experiment clusters (b), the WT EAE macrophage experiment clusters and the chimeric macrophage experiment clusters (c) and the Hoxb8FL-derived macrophage experiment clusters and the chimeric macrophage experiment clusters (d). e, Nos2 (top) and Arg1 (bottom) regulation across KOs in the Hoxb8FL-derived (left) and chimeric (right) macrophages compared to control cells of the same experiment. Boxplot line shows median, box Q1 and Q3, and whiskers the interquantile range x 1.5. Dashed line shows control median. f, UMAP plots of control-edited and cytokine receptor deficient Hoxb8FL-derived and bone-marrow chimeric myeloid cells (b-d), Top left values indicate Pearson correlation; (e), stars indicate adjusted p-value < 0.05 in the KO vs control comparison. a, Pearson correlation of the individual signatures' average gene expression across clusters between Hoxb8FL control and WT EAE macrophages (top), chimeric control and WT EAE macrophages (middle), and chimeric control and Hoxb8FL control macrophages (bottom). b, Expression pattern of curated transcriptional signatures related to lesion-promoting properties of macrophages across macrophage clusters derived from Hoxb8FL control-edited cells. c-i, Heatmaps of the log2(Fold Change) of individual genes comparing Hoxb8FL-derived KO cells to control, for each transcriptional signature in which the corresponding KO has a significant phenotype (as shown in Fig. Stars in (c-i) indicate adjusted p-value < 0.05 and absolute log2(Fold Change) > 3 x standard deviation of the log2(Fold Change) distribution of the cluster as analyzed by Seurat FindMarkers. The genes are arranged from highest to lowest log2(Fold Change) in the overall KO vs control comparison, so their order changes between plots of the same signature. a, Pearson correlation of the individual signatures' average gene expression across clusters between Hoxb8FL control and WT EAE macrophages (top), chimeric control and WT EAE macrophages (middle), and chimeric control and Hoxb8FL control macrophages (bottom). b, Expression pattern of curated transcriptional signatures related to lesion-resolving properties of macrophages across macrophage clusters derived from Hoxb8FL control-edited cells. c-f, Heatmaps of the log2(Fold Change) of individual genes comparing Hoxb8FL-derived KO cells to control, for each transcriptional signature in which the corresponding KO has a significant phenotype (as shown in Fig. g, Left, representative image of fibronectin (Fn1) immunolabeling in control-edited and Tgfbr1-KO Hoxb8FL-derived myeloid cells in EAE lesions at peak of disease. h, Representative images of myelin-pHrodo uptake over time by control-edited or Tgfbr1-KO Hoxb8FL-derived macrophages in vitro. i, Quantification of cholesterol-TMR in the supernatant of untreated (left) or IFNγ, TNFα, GM-CSF and TGFβ treated (right) control or Tgfbr1-KO Hoxb8FL-derived macrophages. N = 3 independent experiments with 3 technical replicates each, data plotted from the technical replicates normalized to untreated control average. (c-f) Stars indicate adjusted p-value < 0.05 and absolute log2(Fold Change) > 3 x standard deviation of the log2(Fold Change) distribution of the cluster as analyzed by Seurat FindMarkers. The genes are arranged from highest to lowest log2(Fold Change) in the overall KO vs control comparison, so their order changes between plots of the same signature. a, Scheme of the experimental approach to perform CRISPR editing of myeloid cells in bone marrow chimeras. b, Gating strategy for the sorting of CRISPR-edited Vex+ chimeric cells (left) and quantification of total CRISPR-edited cells amongst the monocyte/macrophages in the spinal cord at peak of EAE. c, Comparison of the iNOS+ and Arg1+ polarization phenotype in flow cytometry between single-gene KOs in Hoxb8FL-derived and the chimera-derived macrophages. The data derived from Hoxb8FL-cells are the same as shown in Fig. d, e, Expression pattern of curated transcriptional signatures reflecting distinct properties of macrophages presumably contributing to tissue destruction (c) or lesion resolution (d) across macrophage clusters derived from bone-marrow chimeras. f, Correlation of the log2(Fold Change) phenotype of all genes between the chimera-derived and the Hoxb8FL-derived macrophage KOs compared to controls. g, Pearson correlation of the individual signatures' gene regulation in the KO macrophages compared to control between Hoxb8FL-derived and chimera-derived macrophage phenotypes. Only significant signatures (as shown in Figs. 4 and 5) per KO are shown. (b, c), two-tailed unpaired t-test or Wilcoxon test; (e, f), Pearson correlation. a, Proportion of downregulated genes in the KO vs the control that are unique to the cluster or shared between more than one cluster. Numbers at the top of each column indicate the total number of downregulated DEGs per cluster. Downregulated DEGs per KO are defined as genes significantly downregulated (adjusted p-value < 0.05 and log2(Fold Change) < 2 x standard deviation of the log2(Fold Change) distribution of the comparison as analyzed by Seurat FindMarkers) for the same cluster in either the Hoxb8FL-derived KO cells compared to the Hoxb8FL control or the chimeric KO cells compared to the chimeric control. b, Downregulated gene overlap across KOs. See Methods for definition of downregulated genes in the KO. Numbers below the KO name indicate total genes downregulated in that KO, numbers inside of the circles indicate genes uniquely downregulated in that KO, and numbers on the edges indicate genes regulated by the two connected KOs. a, Neuroinflammatory cytokine signature expression across different cytokine injected samples based on Cui et al. (ref. Dots represent the mean, lines the standard deviation b, Expression of myeloid markers in CSF monocyte/macrophages c, Distribution of the gene counts of all detected genes across CSF datasets. d, Characterization of human microglial clusters. e, Distribution of the gene counts of all detected genes across Macnair et al. (ref. f, Distribution of the gene counts of all detected genes across Mathys et al. (ref. g, Distribution of the gene counts of all detected genes across Macnair et al. (ref. b, e-g, Dots represent the mean. Boxplot line shows median, box Q1 and Q3, and whiskers the interquantile range x 1.5. Primer, oligonucleotides, sgRNAs and PCR protocols for cloning, library preparations and sequencing. scRNA-seq cluster markers, gene signatures used for analysis for mouse and human data, and human clinical data. Cell Ranger version information for human data scRNA-seq. Code for all scRNA-seq analysis, mouse and human. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Source data and statistical analysis when applicable. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. de la Rosa, C., Kendirli, A., Baygün, S. et al. In vivo CRISPR screen reveals regulation of macrophage states in neuroinflammation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02402-8'>Ultrafast time-resolved observation of non-thermal current-induced switching in an antiferromagnetic Weyl semimetal</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 10:48:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Materials (2025)Cite this article Antiferromagnets have gained a growing interest for next-generation spintronic applications. Among them, the antiferromagnetic Weyl semimetal Mn3Sn stands out because of its electrical and magnetic properties driven by its non-collinear spin structure at room temperature. Despite research progress on the current-induced switching of the magnetic octupole in Mn3Sn, the ultrafast switching inherent to the antiferromagnet remains to be resolved, and the underlying mechanism is yet elusive. Here we measure the spatiotemporally resolved current-induced switching dynamics in polycrystalline Mn3Sn films using ultrafast magneto-optical Kerr effect imaging, with current pulses as short as 140 ps. Our results directly reveal two distinct switching regimes depending on the intensity and duration of the current pulse: a non-thermal process that does not require the transient melting of antiferromagnetic order, and a temperature-assisted process that relies on heating above the magnetic ordering temperature. Our work highlights the potential of Mn3Sn towards ultrafast magnetic recording devices. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Data that support the findings of this work are available in the Supplementary Information. Source data are provided with this paper. Satoh, T. et al. Spin oscillations in antiferromagnetic NiO triggered by circularly polarized light. Article PubMed Google Scholar Bossini, D. et al. Macrospin dynamics in antiferromagnets triggered by sub-20 femtosecond injection of nanomagnons. Google Scholar Jungwirth, T., Marti, X., Wadley, P. & Wunderlich, J. Antiferromagnetic spintronics. Google Scholar Baltz, V. et al. Antiferromagnetic spintronics. Google Scholar Li, J. et al. Spin current from sub-terahertz-generated antiferromagnetic magnons. Google Scholar Vaidya, P. et al. Subterahertz spin pumping from an insulating antiferromagnet. Google Scholar Bai, H. et al. Functional antiferromagnets for potential applications on high-density storage and high frequency. Google Scholar Miwa, S. et al. Giant effective damping of octupole oscillation in an antiferromagnetic Weyl semimetal. Small Sci. Google Scholar Jhuria, K. et al. Spin–orbit torque switching of a ferromagnet with picosecond electrical pulses. Google Scholar Garello, K. et al. Ultrafast magnetization switching by spin-orbit torques. Google Scholar Polley, D. et al. Picosecond spin-orbit torque–induced coherent magnetization switching in a ferromagnet. Google Scholar Díaz, E. et al. Energy-efficient picosecond spin–orbit torque magnetization switching in ferro- and ferrimagnetic films. Yang, Y. et al. Ultrafast magnetization reversal by picosecond electrical pulses. Google Scholar Nakatsuji, S., Kiyohara, N. & Higo, T. Large anomalous Hall effect in a non-collinear antiferromagnet at room temperature. Google Scholar Higo, T. et al. Anomalous Hall effect in thin films of the Weyl antiferromagnet Mn3Sn. Google Scholar Ikeda, T. et al. Anomalous Hall effect in polycrystalline Mn3Sn thin films. Google Scholar Ikhlas, M. et al. Large anomalous Nernst effect at room temperature in a chiral antiferromagnet. Google Scholar Li, X. et al. Anomalous Nernst and Righi-Leduc effects in Mn3Sn: Berry curvature and entropy flow. Article PubMed Google Scholar Higo, T. et al. Large magneto-optical Kerr effect and imaging of magnetic octupole domains in an antiferromagnetic metal. Google Scholar Kuroda, K. et al. Evidence for magnetic Weyl fermions in a correlated metal. Google Scholar Nakatsuji, S. & Arita, R. Topological magnets: functions based on Berry phase and multipoles. Matter Phys. Google Scholar Suzuki, M.-T., Koretsune, T., Ochi, M. & Arita, R. Cluster multipole theory for anomalous Hall effect in antiferromagnets. Google Scholar Nomoto, T. & Arita, R. Cluster multipole dynamics in noncollinear antiferromagnets. Google Scholar Železný, J., Zhang, Y., Felser, C. & Yan, B. Spin-polarized current in noncollinear antiferromagnets. Article PubMed Kimata, M. et al. Magnetic and magnetic inverse spin Hall effects in a non-collinear antiferromagnet. Rout, P. K., Madduri, P. V. P., Manna, S. K. & Nayak, A. K. Field-induced topological Hall effect in the noncoplanar triangular antiferromagnetic geometry of Mn3Sn. Wang, X. et al. Topological Hall effect in thin films of an antiferromagnetic Weyl semimetal integrated on Si. ACS Appl. Li, X. et al. Chiral domain walls of Mn3Sn and their memory. Chen, X. et al. Octupole-driven magnetoresistance in an antiferromagnetic tunnel junction. Tsai, H. et al. Electrical manipulation of a topological antiferromagnetic state. Higo, T. et al. Perpendicular full switching of chiral antiferromagnetic order by current. Xie, H. et al. Magnetization switching in polycrystalline Mn3Sn thin film induced by self-generated spin-polarized current. Zheng, Z. et al. Effective electrical manipulation of a topological antiferromagnet by orbital torques. Sugimoto, S. et al. Electrical nucleation, displacement, and detection of antiferromagnetic domain walls in the chiral antiferromagnet Mn3Sn. Wu, M. et al. Current-driven fast magnetic octupole domain-wall motion in noncollinear antiferromagnets. Sakamoto, S. et al. Antiferromagnetic spin-torque diode effect in a kagome Weyl semimetal. Yamada, S. et al. Observation of electrical fast switching in the antiferromagnet Mn3Sn. Meeting Abstracts Phys. Krishnaswamy, G. K. et al. Time-dependent multistate switching of topological antiferromagnetic order in Mn3Sn. Takeuchi, Y. et al. Chiral-spin rotation of non-collinear antiferromagnet by spin–orbit torque. Yan, G. Q. et al. Quantum sensing and imaging of spin–orbit-torque-driven spin dynamics in the non-collinear antiferromagnet Mn3Sn. et al. Handedness anomaly in a non-collinear antiferromagnet under spin–orbit torque. Shukla, A., Qian, S. & Rakheja, S. Order parameter dynamics in Mn3Sn driven by d.c. and pulsed spin–orbit torques. APL Mater. Xu, Z. et al. Deterministic spin-orbit torque switching including the interplay between spin polarization and kagome plane in Mn3Sn. Pal, B. et al. Setting of the magnetic structure of chiral kagome antiferromagnets by a seeded spin-orbit torque. Kobayashi, Y., Shiota, Y., Narita, H., Ono, T. & Moriyama, T. Pulse-width dependence of spin–orbit torque switching in Mn3Sn/Pt thin films. Yoo, M.-W., Lorenz, V. O., Hoffmann, A. & Cahill, D. G. Thermal contribution to current-driven antiferromagnetic-order switching. APL Mater. Ogawa, K. et al. Ultrafast stroboscopic time-resolved magneto-optical imaging of domain wall motion in Pt/GdFeCo wires induced by a current pulse. Schlauderer, S. et al. Temporal and spectral fingerprints of ultrafast all-coherent spin switching. Baumgartner, M. et al. Spatially and time-resolved magnetization dynamics driven by spin–orbit torques. Cai, K. et al. Ultrafast and energy-efficient spin–orbit torque switching in compensated ferrimagnets. Matsuo, T. et al. Crossover between intrinsic and temperature-assisted regimes in spin-orbit torque switching of antiferromagnetic order. Preprint at https://arxiv.org/abs/2510.27138 (2025). Download references This work was supported by a JST-MIRAI program (JPMJMI20A1). acknowledges financial support from JSPS KAKENHI (grant number JP24KJ0653). We are grateful to K. Kondo for fruitful discussions. Tomoya Higo Present address: Department of Electronics and Electrical Engineering, Keio University, Yokohama, Japan These authors contributed equally: Kazuma Ogawa, Hanshen Tsai. Department of Physics, The University of Tokyo, Bunkyo-ku, Japan Kazuma Ogawa, Hanshen Tsai, Naotaka Yoshikawa, Takumi Matsuo, Yutaro Tsushima, Mihiro Asakura, Hanyi Peng, Takuya Matsuda, Tomoya Higo, Satoru Nakatsuji & Ryo Shimano Institute for Quantum Matter and Department of Physics and Astronomy, Johns Hopkins University, Baltimore, MD, USA Takumi Matsuo & Satoru Nakatsuji The Institute for Solid State Physics, The University of Tokyo, Kashiwa, Japan Tomoya Higo & Satoru Nakatsuji Trans-scale Quantum Science Institute, The University of Tokyo, Bunkyo-ku, Japan Satoru Nakatsuji & Ryo Shimano Cryogenic Research Center, The University of Tokyo, Bunkyo-ku, Japan Ryo Shimano Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out the optical experiments and analysed the data. and N.Y. developed the ultrafast TR-MOKE measurement system under the supervision of R.S. and T. Matsuda optimized the device structure for the TR-MOKE measurements with feedback from K.O., S.N. grew and characterized the thin films. fabricated the Hall-bar devices and conducted the transport measurements of current-induced switching phenomena under the supervision of S.N. conceived the project of this study. wrote the manuscript with feedback from all co-authors. Correspondence to Kazuma Ogawa or Ryo Shimano. The authors declare no competing interests. Nature Materials thanks Davide Bossini, Zhiqi Liu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Sections 1–15 and Figs. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Ogawa, K., Tsai, H., Yoshikawa, N. et al. Ultrafast time-resolved observation of non-thermal current-induced switching in an antiferromagnetic Weyl semimetal. Download citation Received: 06 February 2025 Accepted: 10 October 2025 Published: 04 December 2025 Version of record: 04 December 2025 DOI: https://doi.org/10.1038/s41563-025-02402-8 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Materials ISSN 1476-4660 (online) ISSN 1476-1122 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251204024226.htm'>Scientists capture flu viruses surfing into human cells in real time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 08:48:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Fever, aching limbs and a runny nose -- as winter returns, so too does the flu. The disease is triggered by influenza viruses, which enter our body through droplets and then infect vulnerable cells. A research team from Switzerland and Japan has taken an exceptionally close look at how this virus behaves. Using a microscopy approach they created themselves, the scientists can zoom in on the outer surface of human cells in a Petri dish. This setup has enabled them to watch, live and in sharp detail, the moment an influenza virus penetrates a living cell. "The infection of our body cells is like a dance between virus and cell," says Yamauchi. This system normally brings essential substances such as hormones, cholesterol or iron into the cell. The virus moves along the surface, latching onto one molecule after another, until it arrives at a site rich in these receptors. A structural protein named clathrin shapes and supports this deepening pocket. Previous attempts to study this crucial moment in infection relied on methods like electron microscopy, which require destroying the cells to obtain an image. As a result, they captured only single moments in time. Fluorescence microscopy, another common tool, offers live imaging but at low spatial resolution. The new method, which merges atomic force microscopy (AFM) with fluorescence microscopy, is called virus-view dual confocal and AFM (ViViD-AFM). Because ViViD-AFM allows scientists to observe infection while it is happening, it offers a valuable way to test antiviral drug candidates directly in cell cultures. The team notes that the technique may also be applied to studying other viruses or even vaccines, giving researchers a real-time view of how these particles interact with cells. Note: Content may be edited for style and length. Scientists Close In on a Universal Cancer Vaccine Scientists Warn This Popular Cooking Oil May Be Quietly Fueling Weight Gain Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251203004736.htm'>Surprising optics breakthrough could transform our view of the Universe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 05:59:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Gravitational-wave detectors may soon get a major performance boost, thanks to a new instrumentation advance led by physicist Jonathan Richardson of the University of California, Riverside. In a paper published in the journal Optica, Richardson and his colleagues describe FROSTI, a full-scale prototype that successfully controls laser wavefronts at extremely high power inside the Laser Interferometer Gravitational-Wave Observatory, or LIGO. LIGO is an observatory that measures gravitational waves -- tiny ripples in spacetime created by massive accelerating objects such as colliding black holes. It was the first facility to directly detect these waves, providing strong support for Einstein's Theory of Relativity. Using two 4-km-long laser interferometers located in Washington and Louisiana, LIGO senses incredibly small disturbances, giving scientists a new way to study black holes, cosmology, and matter under extreme conditions. LIGO relies on mirrors that are among the most carefully engineered components in modern science. To detect distortions in spacetime that are smaller than 1/1,000th the diameter of a proton, these mirrors must be held almost perfectly still. Even tiny vibrations or environmental noise can drown out the faint gravitational-wave signals that LIGO is trying to detect. "At the heart of our innovation is a novel adaptive optics device designed to precisely reshape the surfaces of LIGO's main mirrors under laser powers exceeding 1 megawatt -- more than a billion times stronger than a typical laser pointer and nearly five times the power LIGO uses today," said Richardson, an assistant professor of physics and astronomy. "This technology opens a new pathway for the future of gravitational-wave astronomy. FROSTI, short for FROnt Surface Type Irradiator, is a precision wavefront control system designed to cancel out distortions produced when intense laser light heats LIGO's optics. Existing systems can only make relatively coarse corrections, but FROSTI uses a more advanced thermal projection method to apply fine, higher-order adjustments to the mirror surfaces. Despite its icy name, FROSTI operates by warming the surface of the mirror in a very controlled way that returns it to its ideal optical shape. This smooths out optical distortions while avoiding extra noise that could be mistaken for real gravitational-wave signals. "That means pushing the limits on both laser power and quantum-level precision," Richardson said. Our new technology solves this tension by making sure the optics remain undistorted, even at megawatt power levels." That increase in reach could allow astronomers to detect millions of black hole and neutron star mergers across cosmic history, and to study them with unprecedented detail. "We're already designing new versions capable of correcting even more complex optical distortions. Richardson conducted the research in collaboration with scientists at UCR, MIT, and Caltech. Quantum Spin Ice Is Real: Physicists Confirm Exotic State of Matter Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-66985-1'>Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-04 03:26:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications , Article number:  (2025) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. scRNA-seq and Visium spatial transcriptomics raw and processed mouse data generated for this study have been deposited in NCBI's Gene Expression Omnibus143 and are accessible through GEO Series accession numbers GSE272236 and GSE272237, respectively. All other generated data are provided in the main text, the supplementary information file, or the source data file. The publicly available human GBM scRNA-seq data are available through Broad Institute's Single Cell Portal (https://singlecell.broadinstitute.org/single_cell; study: SCP1985) and https://github.com/parveendabas/GBMatlas. The previously published human GBM stRNA-seq was downloaded from https://datadryad.org/stash/dataset/doi:10.5061/dryad.h70rxwdmj. The Xenium human brain cancer dataset was downloaded from the 10X website https://www.10xgenomics.com/datasets/ffpe-human-brain-cancer-data-with-human-immuno-oncology-profiling-panel-and-custom-add-on-1-standard and visualized in Xenium Explorer (v4.0.0, 10x Genomics). The human patient HGG TMA images are available from The Human Protein Atlas (https://images.proteinatlas.org/18144/43118_A_6_8.jpg and https://images.proteinatlas.org/13531/31458_A_5_5.jpg). Source data are provided with this paper. The code used for the scRNA-seq and spatial transcriptomics analyses is available at https://github.com/KPLab/Pericytes_GBM. Schaff, L. R. & Mellinghoff, I. K. Glioblastoma and other primary brain malignancies in adults: a review. Google Scholar Stupp, R. et al. Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma. Google Scholar Wang, L. et al. A single-cell atlas of glioblastoma evolution under therapy reveals cell-intrinsic and cell-extrinsic therapeutic targets. Google Scholar Varn, F. S. et al. Glioma progression is shaped by genetic evolution and microenvironment interactions. Google Scholar Hambardzumyan, D. & Bergers, G. Glioblastoma: defining tumor niches. Trends Cancer 1, 252–265 (2015). Google Scholar De Silva, M. I., Stringer, B. W. & Bardy, C. Neuronal and tumourigenic boundaries of glioblastoma plasticity. Trends Cancer 9, 223–236 (2023). Google Scholar Prager, B. C., Bhargava, S., Mahadev, V., Hubert, C. G. & Rich, J. N. Glioblastoma stem cells: driving resilience through chaos. Trends Cancer 6, 223–235 (2020). Google Scholar Diaz-Flores, L. et al. Disproportion in pericyte/endothelial cell proliferation and mechanisms of intussusceptive angiogenesis participate in bizarre vessel formation in glioblastoma. Cells 10, https://doi.org/10.3390/cells10102625 (2021). Farin, A. et al. Transplanted glioma cells migrate and proliferate on host brain vasculature: a dynamic analysis. Google Scholar Seano, G. & Jain, R. K. Vessel co-option in glioblastoma: emerging insights and opportunities. Google Scholar Calabrese, C. et al. A perivascular niche for brain tumor stem cells. Cancer Cell 11, 69–82 (2007). Google Scholar Charles, N. A., Holland, E. C., Gilbertson, R., Glass, R. & Kettenmann, H. The brain tumor microenvironment. Google Scholar Armulik, A., Genove, G. & Betsholtz, C. Pericytes: developmental, physiological, and pathological perspectives, problems, and promises. Google Scholar Armulik, A. et al. Pericytes regulate the blood-brain barrier. Google Scholar Bell, R. D. et al. Pericytes control key neurovascular functions and neuronal phenotype in the adult brain and during brain aging. Google Scholar Caspani, E. M., Crossley, P. H., Redondo-Garcia, C. & Martinez, S. Glioblastoma: a pathogenic crosstalk between tumor cells and pericytes. PLoS ONE 9, e101402 (2014). Google Scholar Valdor, R. et al. Glioblastoma ablates pericytes antitumor immune function through aberrant up-regulation of chaperone-mediated autophagy. Google Scholar Zhang, X. N. et al. Pericytes augment glioblastoma cell resistance to temozolomide through CCL5-CCR5 paracrine signaling. Cell Res. Google Scholar Cheng, L. et al. Glioblastoma stem cells generate vascular pericytes to support vessel function and tumor growth. Google Scholar Zhou, W. et al. Targeting glioma stem cell-derived pericytes disrupts the blood-tumor barrier and improves chemotherapeutic efficacy. Google Scholar Jain, S. et al. Single-cell RNA sequencing and spatial transcriptomics reveal cancer-associated fibroblasts in glioblastoma with protumoral effects. J. of Clin. Mae, M. A. et al. Single-cell analysis of blood-brain barrier response to pericyte loss. Oudenaarden, C., Sjolund, J. & Pietras, K. Upregulated functional gene expression programmes in tumour pericytes mark progression in patients with low-grade glioma. Google Scholar Abdelfattah, N. et al. Single-cell analysis of human glioma and immune cells identifies S100A4 as an immunotherapy target. Google Scholar van Splunder, H., Villacampa, P., Martinez-Romero, A. & Graupera, M. Pericytes in the disease spotlight. Trends Cell Biol. Google Scholar Jin, S. et al. Inference and analysis of cell-cell communication using CellChat. Google Scholar Efremova, M., Vento-Tormo, M., Teichmann, S. A. & Vento-Tormo, R. CellPhoneDB: inferring cell-cell communication from combined expression of multi-subunit ligand-receptor complexes. Google Scholar Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Google Scholar Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. Google Scholar Ponten, F., Jirstrom, K. & Uhlen, M. The Human Protein Atlas—a tool for pathology. Google Scholar Lindblom, P. et al. Endothelial PDGF-B retention is required for proper investment of pericytes in the microvessel wall. Genes Dev. Google Scholar Villasenor, R. et al. Region-specific permeability of the blood-brain barrier upon pericyte loss. Blood Flow. Google Scholar Abramsson, A., Lindblom, P. & Betsholtz, C. Endothelial and nonendothelial sources of PDGF-B regulate pericyte recruitment and influence vascular pattern formation in tumors. Google Scholar Hertwig, F. et al. Definition of genetic events directing the development of distinct types of brain tumors from postnatal neural stem/progenitor cells. Cancer Res. Google Scholar Szatmari, T. et al. Detailed characterization of the mouse glioma 261 tumor model for experimental glioblastoma therapy. Cancer Sci. Google Scholar Ozawa, T. et al. Most human Non-GCIMP glioblastoma subtypes evolve from a common proneural-like precursor glioma. Cancer Cell 26, 288–300 (2014). Google Scholar Squatrito, M. et al. Loss of ATM/Chk2/p53 pathway components accelerates tumor development and contributes to radiation resistance in gliomas. Cancer Cell 18, 619–629 (2010). Couturier, C. P. et al. Single-cell RNA-seq reveals that glioblastoma recapitulates a normal neurodevelopmental hierarchy. Darmanis, S. et al. Single-cell RNA-Seq analysis of infiltrating neoplastic cells at the migrating front of human glioblastoma. Eberhart, C. G. & Bar, E. E. Spatial enrichment of cellular states in glioblastoma. Acta Neuropathol. He, L. et al. Analysis of the brain mural cell transcriptome. Mizrak, D. et al. Single-cell analysis of regional differences in adult V-SVZ neural stem cell lineages. Ochocka, N. et al. Single-cell RNA sequencing reveals functional heterogeneity of glioma-associated brain macrophages. Neftel, C. et al. An integrative model of cellular states, plasticity, and genetics for glioblastoma. Chanoch-Myers, R., Wider, A., Suva, M. L. & Tirosh, I. Elucidating the diversity of malignant mesenchymal states in glioblastoma by integrative analysis. Genome Med. Zhou, Y. et al. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Hellstrom, M. et al. Lack of pericytes leads to endothelial hyperplasia and abnormal vascular morphogenesis. Cell Biol. Holash, J. et al. Vessel cooption, regression, and growth in tumors mediated by angiopoietins and VEGF. Gupta, R. K. et al. Tumor-specific migration routes of xenotransplanted human glioblastoma cells in mouse brain. Griveau, A. et al. A glial signature and Wnt7 signaling regulate glioma-vascular interactions and tumor microenvironment. Cancer Cell 33, 874–889 e877 (2018). Hong, J. et al. Role of tumor pericytes in the recruitment of myeloid-derived suppressor cells. Cancer Inst. Pombo Antunes, A. R. et al. Single-cell profiling of myeloid cells in glioblastoma across species and disease stage reveals macrophage competition and specialization. Sousa, S. et al. Human breast cancer cells educate macrophages toward the M2 activation status. Breast Cancer Res. Jeannin, P., Paolini, L., Adam, C. & Delneste, Y. The roles of CSFs on the functional polarization of tumor-associated macrophages. FEBS J. Wu, G. et al. Targeting Gas6/TAM in cancer cells and tumor microenvironment. Hara, T. et al. Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma. Cancer Cell 39, 779–792 e711 (2021). Wang, Q. et al. Vascular niche IL-6 induces alternative macrophage activation in glioblastoma through HIF-2alpha. Sorensen, M. D., Dahlrot, R. H., Boldt, H. B., Hansen, S. & Kristensen, B. W. Tumour-associated microglia/macrophages predict poor prognosis in high-grade gliomas and correlate with an aggressive tumour subtype. Marques, C. et al. NF1 regulates mesenchymal glioblastoma plasticity and aggressiveness through the AP-1 transcription factor FOSL1. eLife 10, https://doi.org/10.7554/eLife.64846 (2021). Qin, Y. et al. Overexpression of HGF/MET axis along with p53 inhibition induces de novo glioma formation in mice. Tao, W. et al. Dual Role of WISP1 in maintaining glioma stem cells and tumor-supportive macrophages in glioblastoma. Chen, Y. et al. GRB10 is a novel oncogene associated with cell proliferation and prognosis in glioma. Cancer Cell Int. Hao, J. et al. Sema3C signaling is an alternative activator of the canonical WNT pathway in glioblastoma. Garcia-Alonso, L., Holland, C. H., Ibrahim, M. M., Turei, D. & Saez-Rodriguez, J. Benchmark and integration of resources for the estimation of human transcription factor activities. Genome Res. Badia, I. M. P. et al. decoupleR: ensemble of computational methods to infer biological activities from omics data. Müller-Dott, S. et al. Expanding the coverage of regulons from high-confidence prior knowledge for accurate estimation of transcription factor activities. Nucleic Acids Res. Bazzoni, R. & Bentivegna, A. Role of notch signaling pathway in glioblastoma pathogenesis. Cancers 11, https://doi.org/10.3390/cancers11030292 (2019). Sadahiro, H. et al. Activation of the receptor tyrosine kinase AXL regulates the immune microenvironment in glioblastoma. Cancer Res. Zhan, T., Rindtorff, N. & Boutros, M. Wnt signaling in cancer. Barthel, L. et al. Glioma: molecular signature and crossroads with tumor microenvironment. Cancer Metastasis Rev. Cable, D. M. et al. Robust decomposition of cell type mixtures in spatial transcriptomics. Li, L., Zhang, W., Zhao, S. & Sun, M. FOS-like antigen 1 is a prognostic biomarker in hepatocellular carcinoma. Liu, Y., Yue, M. & Li, Z. FOSL1 promotes tumorigenesis in colorectal carcinoma by mediating the FBXL2/Wnt/beta-catenin axis via Smurf1. Zhang, M. et al. FOSL1 promotes metastasis of head and neck squamous cell carcinoma through super-enhancer-driven transcription program. Elangovan, I. M. et al. FOSL1 promotes Kras-induced lung cancer through amphiregulin and cell survival gene regulation. Cell Mol. El Zarif, T. et al. Epigenomic signatures of sarcomatoid differentiation to guide the treatment of renal cell carcinoma. Zhang, Y. et al. Single-cell RNA sequencing identifies critical transcription factors of tumor cell invasion induced by hypoxia microenvironment in glioblastoma. Cheng, F. & Guo, D. MET in glioma: signaling pathways and targeted therapies. Cancer Res. Hu, H. et al. Mutational landscape of secondary glioblastoma guides MET-targeted trial in brain tumor. Al-Ghabkari, A., Huang, B. & Park, M. Aberrant MET receptor tyrosine kinase signaling in glioblastoma: targeted therapy and future directions. Cells 13, https://doi.org/10.3390/cells13030218 (2024). Petterson, S. A. et al. High levels of c-Met is associated with poor prognosis in glioblastoma. Mulcahy, E. Q. X., Colomicronn, R. R. & Abounader, R. HGF/MET signaling in malignant brain tumors. Cerami, E. et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. Gao, J. et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Hoogstrate, Y. et al. Transcriptome analysis reveals tumor microenvironment changes in glioblastoma. Cancer Cell 41, 678–692 e677 (2023). Ravi, V. M. et al. Spatially resolved multi-omics deciphers bidirectional tumor-host interdependence in glioblastoma. Cancer Cell 40, 639–655 e613 (2022). Charles, N. & Holland, E. C. The perivascular niche microenvironment in brain tumor progression. Cell Cycle 9, 3012–3021 (2010). Jung, E. et al. Tumor cell plasticity, heterogeneity, and resistance in crucial microenvironmental niches in glioma. Valdor, R. et al. Glioblastoma progression is assisted by induction of immunosuppressive function of pericytes through interaction with tumor cells. Mader, L. et al. Pericytes/vessel-associated mural cells (VAMCs) are the major source of key epithelial-mesenchymal transition (EMT) factors SLUG and TWIST in human glioma. Nisancioglu, M. H., Betsholtz, C. & Genove, G. The absence of pericytes does not increase the sensitivity of tumor vasculature to vascular endothelial growth factor-A blockade. Cancer Res. Vanlandewijck et al. A molecular atlas of cell types and zonation in the brain vasculature. Albiach, A. M. et al. Glioblastoma is spatially organized by neurodevelopmental programs and a glial-like wound healing response. Preprint at bioRxiv https://doi.org/10.1101/2023.09.01.555882 (2023). Cheng, J. et al. Targeting pericytes for therapeutic approaches to neurological disorders. Acta Neuropathol. Ozen, I. et al. Brain pericytes acquire a microglial phenotype after stroke. Acta Neuropathol. Bocci, M. et al. Infection of brain pericytes underlying neuropathology of COVID-19 patients. McQuaid, C. & Montagne, A. SARS-CoV-2 and vascular dysfunction: a growing role for pericytes. Cardiovasc Res. Daneman, R., Zhou, L., Kebede, A. & Barres, B. A. Pericytes are required for blood-brain barrier integrity during embryogenesis. Tsai, H. H. et al. Oligodendrocyte precursors migrate along vasculature in the developing nervous system. Pichol-Thievend, C. et al. VC-resist glioblastoma cell state: vessel co-option as a key driver of chemoradiation resistance. Wang, W. et al. Identification of hypoxic macrophages in glioblastoma with therapeutic potential for vasculature normalization. Cancer Cell https://doi.org/10.1016/j.ccell.2024.03.013 (2024). Buonfiglioli, A. & Hambardzumyan, D. Macrophages and microglia: the cerberus of glioblastoma. Acta Neuropathol. Greenwald, A. C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Cell https://doi.org/10.1016/j.cell.2024.03.029 (2024). Cooke, V. G. et al. Pericyte depletion results in hypoxia-associated epithelial-to-mesenchymal transition and metastasis mediated by met signaling pathway. Cancer Cell 21, 66–81 (2012). Seol, D. W., Chen, Q. & Zarnegar, R. Transcriptional activation of the hepatocyte growth factor receptor (c-met) gene by its ligand (hepatocyte growth factor) is mediated through AP-1. Pecce, V. et al. The role of FOSL1 in stem-like cell reprogramming processes. Pombero, A., Garcia-Lopez, R. & Martinez, S. Pericyte-glioblastoma cell interaction: a key target to prevent glioblastoma progression. Cells 12, https://doi.org/10.3390/cells12091324 (2023). Dias Moura Prazeres, P. H. et al. Pericytes are heterogeneous in their origin within the same tissue. Piao, Y. et al. Acquired resistance to anti-VEGF therapy in glioblastoma is associated with a mesenchymal transition. Cancer Res. Lu, K. V. et al. VEGF inhibits tumor cell invasion and mesenchymal transition through a MET/VEGFR2 complex. Cancer Cell 22, 21–35 (2012). Piao, Y. et al. Glioblastoma resistance to anti-VEGF therapy is associated with myeloid cell infiltration, stem cell accumulation, and a mesenchymal phenotype. Lee, S. T. et al. Loss of pericytes in radiation necrosis after glioblastoma treatments. Falchetti, M. L. et al. Glioblastoma endothelium drives bevacizumab-induced infiltrative growth via modulation of PLXDC1. Martin, J. D., Seano, G. & Jain, R. K. Normalizing function of tumor vessels: progress, opportunities, and challenges. Tolaney, S. M. et al. Role of vascular density and normalization in response to neoadjuvant bevacizumab and chemotherapy in breast cancer patients. Holland, E. C., Hively, W. P., DePinho, R. A. & Varmus, H. E. A constitutively active epidermal growth factor receptor cooperates with disruption of G1 cell-cycle arrest pathways to induce glioma-like lesions in mice. Genes Dev. Holland, E. C. & Varmus, H. E. Basic fibroblast growth factor induces cell migration and proliferation after glia-specific gene transfer in mice. Hu, X. et al. mTOR promotes survival and astrocytic characteristics induced by Pten/AKT signaling in glioblastoma. Uhrbom, L., Nerio, E. & Holland, E. C. Dissecting tumor maintenance requirements using bioluminescence imaging of cell proliferation in a mouse glioma model. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Zudaire, E., Gambardella, L., Kurcz, C. & Vermeren, S. A computational tool for quantitative analysis of vascular networks. PLoS ONE 6, e27385 (2011). Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Zheng, G. X. et al. Massively parallel digital transcriptional profiling of single cells. McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. Cell Syst. Yang, S. et al. Decontamination of ambient RNA in single-cell RNA-seq with DecontX. Genome Biol. Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Hao, Y. et al. Integrated analysis of multimodal single-cell data. Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. & Regev, A. Spatial reconstruction of single-cell gene expression data. Stuart, T. et al. Comprehensive integration of single-cell data. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Marsh, S. E. et al. Dissection of artifactual and confounding glial signatures by single-cell sequencing of mouse and human brain. Blanco-Carmona, E. Generating publication ready visualizations for single cell transcriptomics using SCpubr. Preprint at bioRxiv https://doi.org/10.1101/2022.02.28.482303 (2022). Wei, T. & Simko, V. R package 'corrplot': Visualization of a Correlation Matrix. R Package Version 0.92. Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Marsh, S. scCustomize: custom visualizations & functions for streamlined analyses of single cell sequencing. R Package Version 2.1.2. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Genome Biol. Kassambara, A. ggpubr: 'ggplot2' Based Publication Ready Plots. R Package Version 0.6.0. Interlandi, M., Kerl, K. & Dugas, M. InterCellar enables interactive analysis and exploration of cell-cell communication in single-cell transcriptomic data. Elosua-Bayes, M., Nieto, P., Mereu, E., Gut, I. & Heyn, H. SPOTlight: seeded NMF regression to deconvolute spatial transcriptomics spots with single-cell transcriptomes. Nucleic Acids Res. Therneau, T. A Package for Survival Analysis in R. R package version 3.5-5. Kassambara, A., Kosinski, M. & Biecek P. survminer: Drawing Survival Curves using 'ggplot2'. R Package Version 0.4.9. R Core Team. R: a language and environment for statistical computing. R Foundation for Statistical Computing. Edgar, R., Domrachev, M. & Lash, A. E. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res. Download references is the Grosskopf Professor of Molecular Medicine at Lund University. This work was supported by grants from the Fru Berta Kamprad foundation to the L2 Cancer Bridge at Create Health, the Swedish Research Council (2023-03136 to K.P. ), the Swedish Cancer Society (24 3812 Pj to K.P. and 22 0536 PT to P.B. ), the Swedish Childhood Cancer Society (PR2024-0121 to K.P. ), the Knut and Alice Wallenberg foundation (to G.B.J. ), Swedish State Support for Clinical Research through Region Skåne ALF (to K.P. ), the Göran Gustafsson foundation (to K.P. ), the Mats Paulsson foundations (L2 Cancer Bridge to K.P. ), the Cancera foundation (L2 Cancer Bridge to K.P.). The authors thank the Centre for Translational Genomics (Lund University), and Clinical Genomics Lund (SciLifeLab) for sequencing services; Uppsala Multidisciplinary Center for Advanced Computational Sciences (UPPMAX), the Swedish National Infrastructure for Computing (SNIC), and the National Academic Infrastructure for Supercomputing in Sweden (NAISS) for the provided computing and storage resources. The authors would like to thank Ulrike Nuber for kindly providing the p53−/−, H-Ras over-expressing glioma cell line, Laurent Roybon for kindly providing the CMMP vector, Massimo Squatrito for kindly providing the Ntv-a mice, and Christer Betsholtz, as well as Guillem Genové for kindly providing the Pdgfbret/ret mice. The authors would like to thank Wondossen Sime, David Lindgren, Eliane Cortez, and Christina Möller for helpful technical advice and assistance. Finally, the authors would like to thank all reviewers for their feedback. Open access funding provided by Lund University. These authors contributed equally: Sebastian Braun, Paulina Bolivar. Division of Translational Cancer Research, Department of Laboratory Medicine, Lund University Cancer Centre, Lund University, Lund, Sweden Sebastian Braun, Paulina Bolivar, Jonas Sjölund, Matteo Bocci, Mehrnaz Safaee Talkhoncheh, Rebecca Rosberg, Elinn Johansson, Alexander Pietras & Kristian Pietras Biotech Research and Innovation Centre, University of Copenhagen, Copenhagen, Denmark Clara Oudenaarden Division of Oncology and Pathology, Department of Clinical Sciences, Lund, Lund University Cancer Centre, Lund University, Lund, Sweden Katja Harbst, Bengt Phung & Göran B. Jönsson Department of Clinical Sciences, Malmö, Lund University Diabetes Centre, Lund University, Malmö, Sweden Eugenia Cordero SciLifeLab, Department of Laboratory Medicine, Lund University, Lund, Sweden Kristian Pietras Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Formal analysis: P.B., C.O., J.S., K.H., K.P., S.B. Data curation: P.B., C.O., J.S., S.B., K.P. Writing—review and editing: P.B., C.O., J.S., M.B., K.H., M.S.T., B.P., E.C., R.R., E.J., G.B.J., A.P., K.P., S.B. Funding acquisition: K.P. Correspondence to Kristian Pietras. The authors declare no competing interests. Nature Communications thanks Dritan Agalliu, Justin Lathia, and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Braun, S., Bolivar, P., Oudenaarden, C. et al. Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma. Nat Commun  (2025). Download citation Received: 24 September 2024 Accepted: 18 November 2025 Published: 04 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            