
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-22</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44286-025-00315-6'>A synthetic cell-free pathway for biocatalytic upgrading of formate from electrochemically reduced CO</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-22 11:13:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Chemical Engineering (2025)Cite this article Electrochemical reduction of carbon dioxide (CO2) can produce important one-carbon (C1) feedstocks for sustainable biomanufacturing, such as formate. Unfortunately, natural formate assimilation pathways are inefficient and constrained to organisms that are difficult to engineer. Here we establish a synthetic reductive formate pathway (ReForm) in vitro. ReForm is a six-step pathway consisting of five engineered enzymes catalyzing nonnatural reactions to convert formate into the universal biological building block acetyl-CoA. We establish ReForm by selecting enzymes among 66 candidates from prokaryotic and eukaryotic origins. Through iterative cycles of engineering, we create and evaluate 3,173 sequence-defined enzyme mutants, tune cofactor concentrations and adjust enzyme loadings to increase pathway activity toward the model end product malate. We demonstrate that ReForm can accept diverse C1 substrates, including formaldehyde, methanol and formate produced from the electrochemical reduction of CO2. Our work expands the repertoire of synthetic C1 utilization pathways, with implications for synthetic biology and the development of a formate-based bioeconomy. This is a preview of subscription content, access via your institution Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data are available in the Article or its Supplementary Information. Source data are provided with this paper. Atomic structures reported in this Article are deposited to the Protein Data Bank under accession codes 9CD3 and 9CD4. The cryo-EM data were deposited to the Electron Microscopy Data Bank under EMD-45461 and EMD-45462. Lee, H. et al. in IPCC, 2023: Climate Change 2023: Synthesis Report (eds Core Writing Team, Lee, H. & Romero, J.) Global Energy Review 2025 (IEA, 2025); https://www.iea.org/reports/global-energy-review-2025 Cai, W. et al. The 2024 China report of the Lancet Countdown on health and climate change: launching a new low-carbon, healthy journey. Lancet Public Heal. Google Scholar Li, H. et al. Integrated electromicrobial conversion of CO2 to higher alcohols. Google Scholar Bar-Even, A., Noor, E., Flamholz, A. & Milo, R. Design and analysis of metabolic pathways supporting formatotrophic growth for electricity-dependent cultivation of microbes. Google Scholar Yishai, O., Lindner, S. N., Cruz, J. G., de la, Tenenboim, H. & Bar-Even, A. Google Scholar Bang, J. et al. Synthetic formatotrophs for one-carbon biorefinery. Google Scholar Li, L. et al. Stable, active CO2 reduction to formate via redox-modulated stabilization of active sites. Google Scholar Crowther, G. J., Kosály, G. & Lidstrom, M. E. Formate as the main branch point for methylotrophic metabolism in Methylobacterium extorquens AM1. Google Scholar Kim, S. et al. Growth of E. coli on formate and methanol via the reductive glycine pathway. Google Scholar Bang, J., Hwang, C. H., Ahn, J. H., Lee, J. & Lee, S. Y. Escherichia coli is engineered to grow on CO2 and formic acid. Google Scholar Claassens, N. J. et al. Replacing the Calvin cycle with the reductive glycine pathway in Cupriavidus necator. Google Scholar Bar-Even, A. Formate assimilation: the metabolic architecture of natural and synthetic pathways. Google Scholar Bar-Even, A., Noor, E., Lewis, N. E. & Milo, R. Design and analysis of synthetic carbon fixation pathways. Natl Acad. Google Scholar Schwander, T., Borzyskowski, L. S. von, Burgener, S., Cortina, N. S. & Erb, T. J. A synthetic pathway for the fixation of carbon dioxide in vitro. Google Scholar Luo, S. et al. Construction and modular implementation of the THETA cycle for synthetic CO2 fixation. Google Scholar Chou, A., Lee, S. H., Zhu, F., Clomburg, J. M. & Gonzalez, R. An orthogonal metabolic framework for one-carbon utilization. Google Scholar Lu, X. et al. Constructing a synthetic pathway for acetyl-coenzyme A from one-carbon through enzyme design. Google Scholar Siegel, J. B. et al. Computational protein design enables a novel one-carbon assimilation pathway. Proc Natl Acad. Google Scholar Wang, J., Anderson, K., Yang, E., He, L. & Lidstrom, M. E. Enzyme engineering and in vivo testing of a formate reduction pathway. Nattermann, M. et al. Engineering a new-to-nature cascade for phosphate-dependent formate to formaldehyde conversion in vitro and in vivo. Google Scholar Hu, G. et al. Synergistic metabolism of glucose and formate increases the yield of short-chain organic acids in Escherichia coli. ACS Synth. Google Scholar Lee, S. Y. et al. A comprehensive metabolic map for production of bio-based chemicals. Google Scholar Sodium Malate Market Size By Grade (Pharmaceutical Grade, Food Grade, Industrial Grade), By Form (Powdered, Liquid, Granular), By Application (Food & Beverages, Pharmaceuticals, Personal Care & Cosmetics, Industrial Applications) & Forecast 2023–2032 (GMI, 2022); https://www.gminsights.com/industry-analysis/sodium-malate-market Chou, A., Clomburg, J. M., Qian, S. & Gonzalez, R. 2-Hydroxyacyl-CoA lyase catalyzes acyloin condensation for one-carbon bioconversion. Google Scholar Nattermann, M. et al. Engineering a highly efficient carboligase for synthetic one-carbon metabolism. ACS Catal. Scheffen, M. et al. A new-to-nature carboxylation module to improve natural and synthetic CO2 fixation. Google Scholar Trudeau, D. L. et al. Design and in vitro realization of carbon-conserving photorespiration. Natl Acad. Zarzycki, J., Sutter, M., Cortina, N. S., Erb, T. J. In vitro characterization and concerted function of three core enzymes of a glycyl radical enzyme—associated bacterial microcompartment. Karim, A. S. et al. In vitro prototyping and rapid optimization of biosynthetic enzymes for cell design. Hunt, A. C. et al. A rapid cell-free expression and screening platform for antibody discovery. Silverman, A. D., Karim, A. S. & Jewett, M. C. Cell-free gene expression: an expanded repertoire of applications. Hunt, A. C. et al. Cell-free gene expression: methods and applications. Yang, Y. et al. Construction of an artificial phosphoketolase pathway that efficiently catabolizes multiple carbon sources to acetyl-CoA. PLoS Biol. Hopf, T. A. et al. Mutation effects predicted from sequence co-variation. Shroff, R. et al. Discovery of novel gain-of-function mutations guided by structure-based deep learning. Zhang, J. & Liu, Y. Computational studies on the catalytic mechanism of phosphoketolase. Landwehr, G. M. et al. Accelerated enzyme engineering by machine-learning guided cell-free expression. Starai, V. J., Celic, I., Cole, R. N., Boeke, J. D. & Escalante-Semerena, J. C. Sir2-dependent activation of acetyl-CoA synthetase by deacetylation of active lysine. Marx, C. J., Laukel, M., Vorholt, J. & Lidstrom, M. E. Purification of the formate-tetrahydrofolate ligase from Methylobacterium extorquens AM1 and demonstration of its requirement for methylotrophic growth. Bar-Even, A. et al. The moderately efficient enzyme: evolutionary and physicochemical trends shaping enzyme parameters. Mordhorst, S., Maurer, A., Popadić, D., Brech, J. & Andexer, J. N. A flexible polyphosphate-driven regeneration system for coenzyme A dependent catalysis. Cai, T. et al. Cell-free chemoenzymatic starch synthesis from carbon dioxide. Diehl, C., Gerlinger, P. D., Paczia, N. & Erb, T. J. Synthetic anaplerotic modules for the direct synthesis of complex molecules from CO2. Meng, X. et al. A synthetic methylotroph achieves accelerated cell growth by alleviating transcription-replication conflicts. Sherkhanov, S. et al. Isobutanol production freed from biological limits using synthetic biochemistry. Black, W. B. et al. Engineering a nicotinamide mononucleotide redox cofactor system for biocatalysis. Black, W. B. et al. Activation-free upgrading of carboxylic acids to aldehydes and alcohols. Preprint at bioRxiv https://doi.org/10.1101/2025.07.28.667276 (2025). Liew, F. E. et al. Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale. Chen, Y. et al. Efficient multicarbon formation in acidic CO2 reduction via tandem electrocatalysis. Kay, J. E. & Jewett, M. C. Lysate of engineered Escherichia coli supports high-level conversion of glucose to 2,3-butanediol. Jiang, Y. et al. Multigene editing in the Escherichia coli genome via the CRISPR–Cas9 system. Bassalo, M. C. et al. Rapid and efficient one-step metabolic pathway integration in E. coli. Kwon, Y.-C. & Jewett, M. C. High-throughput preparation methods of crude extract for robust cell-free protein synthesis. Jewett, M. C. & Swartz, J. R. Mimicking the Escherichia coli cytoplasmic environment activates long-lived and efficient cell-free protein synthesis. Jewett, M. C. & Swartz, J. R. Substrate replenishment extends protein synthesis with an in vitro translation system designed to mimic the cytoplasm. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Suzuki, R. et al. Crystal structures of phosphoketolase thiamine diphosphate-dependent dehydration mechanism. Morin, A. et al. Collaboration gets the most out of software. Luc, W., Rosen, J. & Jiao, F. An Ir-based anode for a practical CO2 electrolyzer. Download references Molecular graphics and analyses were performed with UCSF ChimeraX, developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco, with support from National Institutes of Health R01-GM129325 and the Office of Cyber Infrastructure and Computational Biology, National Institute of Allergy and Infectious Diseases. This work made use of the IMSERC MS facility at Northwestern University, which has received support from the Soft and Hybrid Nanotechnology Experimental (SHyNE) Resource (NSF ECCS-2025633), the State of Illinois and the International Institute for Nanotechnology (IIN). We thank Stanford University Cryo-EM center (cEMc) and particularly B. Singal for providing support for cryo-EM grid preparation, data collection, processing and structure determination pipeline. We thank F. ‘Ralph' Tobias for his help in developing analytical methods for malate detection and K. Seki for his help in gathering intact protein MS data on the deacetylated acyl-CoA synthetases. We also thank J. W. Bogart for conversations regarding this work. Funding was provided by the Department of Energy (DE-SC0023278) (G.M.L., B.V., K.Z., I.M., A.G., R.L., C.T., E.H.S., A.S.K. and Stanford University Cryo-electron Microscopy Center (cEMc) (B.S. Department of Chemical and Biological Engineering, Northwestern University, Evanston, IL, USA Grant M. Landwehr, Bastian Vogeli, Anika Gupta, Rebeca Lion, Ashty S. Karim & Michael C. Jewett Center for Synthetic Biology, Northwestern University, Evanston, IL, USA Grant M. Landwehr, Bastian Vogeli, Anika Gupta, Rebeca Lion, Ashty S. Karim & Michael C. Jewett Department of Chemistry, Northwestern University, Evanston, IL, USA Cong Tian & Edward H. Sargent Stanford SLAC CryoEM Initiative, Stanford University, Stanford, CA, USA Bharti Singal Department of Bioengineering, Stanford University, Stanford, CA, USA Kyle Zolkin, Irene Martinez & Michael C. Jewett Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Funding acquisition: A.S.K., M.C.J. Correspondence to Ashty S. Karim or Michael C. Jewett. have filed an invention disclosure based on the work presented. has a financial interest in National Resilience, Gauntlet Bio, Pearl Bio, Inc., and Stemloop Inc. 's interests are reviewed and managed by Northwestern University and Stanford University in accordance with their competing interest policies. All other authors declare no competing interests. Nature Chemical Engineering thanks Mattheos Koffas, Zaigao Tan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Methods, Figs. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Landwehr, G.M., Vogeli, B., Tian, C. et al. A synthetic cell-free pathway for biocatalytic upgrading of formate from electrochemically reduced CO2. Nat Chem Eng  (2025). Download citation Accepted: 03 November 2025 Version of record: 22 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemical Engineering (Nat Chem Eng) ISSN 2948-1198 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01828-1'>Interphase chromosome conformation is specified by distinct folding programmes inherited through mitotic chromosomes or the cytoplasm</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-22 10:55:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Identity-specific chromosome conformation must be re-established at each cell division. To uncover how interphase folding is inherited, we developed an approach that segregates chromosome-intrinsic mechanisms from those propagated through the cytoplasm during G1 nuclear reassembly. Here we show that genome compartmentalization is driven entirely by chromosome-intrinsic factors. In addition to conventional compartmental segregation, the chromosome-intrinsic folding programme leads to prominent genome-scale microcompartmentalization of mitotically bookmarked cis-regulatory elements. The microcompartment conformation forms transiently during telophase and is subsequently modulated by a second folding programme inherited through the cytoplasm in early G1. This programme includes cohesin-mediated loop extrusion and factors involved in transcription and RNA processing. The combined and interdependent action of chromosome-intrinsic and cytoplasmic inherited folding programmes determines the interphase chromatin conformation as cells exit mitosis. Interphase chromosome folding is specific to the transcriptional programme of a given cell type1,2,3,4. Two major processes contribute to this folding: loop extrusion mediated by cohesin and spatial compartmentalization that is probably driven by homotypic affinities of chromatin domains distributed across the genome5. Within the three-dimensional (3D) space of the nucleus the genome is segregated into broad euchromatin/heterochromatin compartments and multiple constituent subcompartments that form within (in cis) and between (in trans) chromosomes6. Heterochromatic loci cluster together, often at the nuclear periphery, and active chromatin domains self-associate most frequently in the nuclear interior. The molecular factors that determine compartmentalization remain poorly understood7,8. The distribution of histone modifications along the genome is strongly correlated with subcompartment formation6,9,10 and it is possible that these modifications display homotypic affinities (that is, histone 3 lysine 9 trimethylation, H3K9me3; ref. In addition, factors that bind at specific histone modifications—that is, BRD proteins for histone 3 lysine 27 acetylation (H3K27ac)11,12 and HP1 proteins for H3K9me3 (refs. 13,14)—can bridge interactions between marked loci. As gene expression is cell type-specific, changes in chromatin composition and histone modification along the linear genome and, in turn, the spatial positioning of these domains in subcompartments within the nucleus also differ between cell types. Interphase chromosome folding is actively modulated by cohesin-dependent loop formation. The resulting interaction patterns are also cell type-specific due to the reliance of cohesin activity on the positioning of cis-regulatory elements (CREs). Cohesin can be loaded throughout the genome15 but is more efficiently recruited at open chromatin sites such as enhancers16,17,18. Furthermore, loop extrusion is blocked in a directional manner at CTCF-bound sites9,19,20,21 and cohesin complexes are unloaded near the 3′ ends of active genes16,22. These dynamics specify a genome-wide ‘cohesin traffic pattern' that gives rise to a complex identity-specific chromosome folding pattern observed by the chromosome conformation capture-based technology Hi-C that includes loops (for example, CTCF–CTCF and promoter-enhancer loops), stripes, flares, insulated boundaries and topologically associating domains (TADs)23. During mitosis most features of cell type-specific chromosome folding, such as compartments and cohesin-mediated loops, are undetectable by Hi-C. A largely cell type-invariant compressed array of partially self-entangled loops is instead formed by condensin complexes to give rise to rod-shaped mitotic chromatids24,25,26,27,28. These loops are not positioned at specific CREs. Many of the chromatin-associated factors that contribute to interphase organization are absent from mitotic chromosomes. Condensins actively remove loop-extruding cohesin complexes during prophase of mitosis26, and cell-cycle kinases drive the phosphorylation and dissociation of a large proportion of the chromatin-bound factors, including CTCF, transcription factors and, to a large extent, RNA polymerases29,30,31,32,33. However, not all regulatory factors dissociate from mitotic chromosomes and a number remain bound at specific cis-elements. This phenomenon, referred to as mitotic bookmarking34, records active loci for reactivation in the next cell cycle. Cell cycle stage-dependent changes in chromosome structure are also observed at the scale of nucleosomes. Although promoters that are active in a given cell type remain largely nucleosome-free, enhancers are less accessible during mitosis and are probably bookmarked to be reopened by remodellers during the next cell cycle29,35. Similarly, CTCF sites are thought to be bookmarked by active histone modifications and become occupied by nucleosomes in mitosis, although the extent to which this occurs differs between cell types29,36,37. The dramatic changes to chromosome organization that occur during mitosis pose a particular challenge to cycling cells. Every time cells re-enter G1, the cell type-invariant mitotic chromosome structure must be converted to an identity-specific interphase conformation. The relative timing of events during this process has been delineated in several recent studies38,39,40,41,42. First, condensins dissociate during telophase and a transient chromosome folding state devoid of extruded loops is formed39. Although CTCF is rapidly enriched at post-mitotic chromatin, the cohesin complex is recruited later during cytokinesis38,39, at which point extrusion rapidly re-establishes loops, TADs and other extrusion-related features. Genome compartmentalization is first detected around telophase but does not reach the full extent of segregation until well into G1 (ref. Although the kinetics of post-mitotic chromosome reorganization are increasingly known, it remains unclear how the cell type-specific folding programme is transmitted through mitosis. We have established an experimental system that allows us to discriminate the contributions of factors required for cell identity-specific chromosome folding based on whether they are inherited on mitotic chromosomes or transmitted to daughter nuclei through the cytoplasm. We find that the interphase chromatin state is achieved through the combined action of two folding programmes inherited in distinct ways. During telophase, the nuclear envelope re-forms around newly segregated chromosomes, establishing regulated nucleocytoplasmic transport. We sought to prevent nuclear import from telophase onwards to understand the extent to which type-specific interphase chromosome folding can be re-established independently of G1 cytoplasmic factors. To this end, RanGAP1 and Nup93 were biallelically fused with auxin-inducible degron (AID) and NeonGreen fluorescent protein sequences in DLD-1 cells using CRISPR–Cas9 (Methods, Fig. 1), enabling disruption of nuclear transport by two mechanisms. Nup93 is a scaffold nucleoporin essential for assembly and stability of nuclear pore complexes43,44,45, and RanGAP1 critically regulates the RanGTP gradient required for nucleocytoplasmic shuttling of proteins46,47. a, Workflow for the depletion of AID-tagged RanGAP1 and Nup93 proteins from the onset of mitotic exit into G1. Auxin-induced degradation is initiated 2 h before nocodazole/mitotic release (time (t) = 0). b, Representative western blot images of whole-cell lysates derived from control and auxin-treated DLD-1 cell lines 5 h after mitotic release showing efficient depletion of RanGAP1–AID and Nup93–AID (left). normalized to vinculin for depleted lysates relative to untreated controls for three replicates (right). c, Representative immunofluorescence images for RanGAP1 control and depleted cells demonstrate congruous kinetics of mitotic release. Loss of histone H3 serine 10 phosphorylation (H3pS10) and recruitment of the nuclear-envelope proteins emerin and lamin B receptor (LBR) are shown for control and RanGAP1–AID-depleted cells. d, Representative DNA-content (stained with PI) flow cytometry measurements for RanGAP1 control and depleted cells demonstrate congruous kinetics of mitotic release. The percentage of cells in each quadrant of the flow cytometry plots is provided. e, Representative immunofluorescence images of RanGAP1–AID control and depleted (ΔRanGAP1) cells 5 h after mitotic release demonstrating the presence of nuclear lamina (lamin A), nuclear envelope (LBR) and nuclear pore complex (Elys, Nup160 and FG-rich nucleoporins (FG-Nups))-resident proteins in the absence of RanGAP1–AID. f, The nuclear volumes of control and RanGAP1- or Nup93-depleted cells 5 h after mitotic exit indicate a reduced size of import-incompetent nuclei. Individual data points and mean values for each of three independent replicates (R1, R2 and R3) are indicated. g, Import competence is not re-established after mitosis in RanGAP1–AID- or Nup93–AID-depleted cells. This is indicated by the reduced relative nuclear-to-total intensity of MBP–mScarlet–NLS after 5 h release from Ro-3306. Ratios are shown for eight cells in three independent fields from onset of mitotic release for one experiment. Source numerical data and unprocessed blots are provided. To prevent nuclear transport during mitotic exit, RanGAP1–AID or NUP93–AID were degraded in arrested prometaphase cells (Supplementary Methods and Fig. The cells were then released through anaphase into G1. Depletion of RanGAP1–AID or Nup93–AID did not impact the kinetics of the initial mitotic exit stages (Fig. 1c,d and Extended Data Fig. The absence of Nup93 after mitotic release resulted in daughter nuclei sealed by a continuous nuclear envelope and an underlying nuclear lamina devoid of mature nuclear pore complexes (Extended Data Fig. Conversely, apparently mature nuclear pore complexes were found at the nuclear envelope of RanGAP1–AID-depleted cells (Fig. Despite differences in nuclear morphology, depletion of either RanGAP1–AID or Nup93–AID impeded chromatin decondensation and nuclear volume expansion characteristic of control cells progressing through G1 (Fig. Nucleoli and nuclear speckles, two import-dependent organelles with important roles in nuclear organization48,49,50,51, were not found in RanGAP1–AID- or Nup93–AID-depleted nuclei and strong speckle protein-containing granules were found exclusively in the cytoplasm (Extended Data Fig. The transport competence of Nup93- and RanGAP1-depleted cells for large macromolecules was quantified by expressing a canonical importin-α nuclear localization signal (NLS) fused to maltose-binding protein (MBP) and mScarlet fluorescent protein (MBP–mScarlet–NLS; Fig. Time-lapse fluorescence imaging of control cells demonstrated efficient nuclear import of MBP–mScarlet–NLS following chromosome segregation, with minimal protein observed in the cytoplasm by 5 h. In contrast with the untreated controls, depletion of either Nup93–AID or RanGAP1–AID prevented the nuclear accumulation of MBP–mScarlet–NLS throughout mitotic exit and G1 entry entirely, consistent with a failure to establish nucleocytoplasmic transport. Thus, in this experimental context, the genome is isolated from the cytoplasm once the nuclear envelope has formed in late telophase. To assess how chromosome-intrinsic factors inform post-mitotic genome folding, we performed Hi-C 3.0 (refs. Pure G1 populations of cells that had progressed through mitosis in the absence or presence of RanGAP1 or Nup93 were obtained by fluorrescence-activated cell sorting (FACS) (Supplementary Fig. Despite an inability to establish active nucleocytoplasmic transport, not only did broad euchromatin/heterochromatin compartments form but they also seemed more pronounced in RanGAP1–AID- and Nup93–AID-depleted cells (Fig. In the absence of nuclear import, we observed an additional prominent network of small (<100 kb) highly interacting chromatin domains that interacted ubiquitously with each other in cis and trans. The chromosome-wide grid-like nature of these interactions implies the formation of a distinct subcompartment in the absence of factors that would normally enter the nucleus after telophase. We refer to these interactions as microcompartments on the basis of their scale and similarity to previously described structures8,54. a, Hi-C interaction frequency maps at 250 kb (chromosome (Chr) 6: 105.0–170.8 Mb–Chr 7:0–56 Mb; left), 50 kb (Chr 6: 125–145.5 Mb; middle) and 10 kb (Chr 6: 129–130.5 and 136.1–138.2 Mb; right) resolutions showing genome compartmentalization in control (top) and auxin-treated (bottom) RanGAP1–AID cells released from prometaphase for 5 h. First Eigenvector values for cis interactions phased by gene density (high gene density > 0). b, Hi-C contact matrix at 10 kb resolution (Chr 6: 129–130.5 and 136.1–138.2 Mb) highlighting microcompartments detected in RanGAP1-depletion and constituent MCDs (top). Control tracks define CREs, which coincide with MCDs. c, Heat maps for all 2,105 MCDs centred on contact frequency summits with a 100-kb flank and sorted by length, demonstrating the prevalence of the ATAC–seq, H3K27ac, H3K4me3 and H3K27me3 coverage in control and RanGAP1-depleted cells, and RNA expression in control cells. Intra-chromosomal EV1 values from 10-kb matrices indicate altered compartmentalization at MCDs following RanGAP1 depletion. d, Relative enrichment of control candidate CREs genome-wide, at 10 kb control A-compartment bins (EV1 > 0), and MCDs demonstrating the predominance of active promoters (PLS) and enhancers (proximal and near enhancer-like (pELS), promoter–proximal; dELS, promoter–distal). e, Pairwise mean observed/expected contact frequency between all MCDs (top) projected in cis (10 kb) and trans (25 kb), showing dramatically enhanced interactions at all length scales, and between chromosomes in RanGAP1-depleted G1 cells (bottom). f, Aggregate pairwise observed/expected contact frequencies between cCREs assigned in control cells and subjected to hierarchical binning at 10 kb resolution showing enhanced homo- and heterotypic interactions between active promoters and enhancers in RanGAP1-depleted cells at multiple genomic distances in cis and in trans. Source numerical data are provided. The properties of chromosome-intrinsic microcompartments were explored by identifying the domains of 25–125 kb that mediate strong focal pairwise enrichments in the Hi-C interaction data of RanGAP1–AID-depleted G1 cells. We identified 2,105 strong microcompartment domains (MCDs) along the linear genome encompassing all autosomal chromosomes9,55 (Supplementary Fig. Microcompartment domains consist of gene-dense active regions corresponding to high assay for transposase-accessible chromatin using sequencing (ATAC–seq) coverage and peaks of H3K4me3 and H3K27ac found in both control and RanGAP1-depleted cells in G1 (Fig. Accordingly, active promoters and enhancers defined in control cells (candidate CREs, cCREs; Methods) were highly enriched at MCDs (Fig. Furthermore, the A-compartment profile at MCDs captured by the first Eigenvector (EV1) in control cells was focally enriched in RanGAP1–AID- and Nup93–AID-depleted cells (Fig. 2a,c and Extended Data Fig. 3a,b), indicating a more prominent subcompartment. We confirmed the compartment nature of these regions by genome-wide spectral clustering analysis of RanGAP1-depletion Hi-C data (Methods and Extended Data Fig. More than 60% of the 2,105 MCDs were specifically covered by a single cluster corresponding to highly active euchromatin (Extended Data Fig. Based on the comprehensive grid of microcompartments observed in RanGAP1–AID-depleted cells, we leveraged the possibility that all MCDs possess an intrinsic affinity for each other. To quantify the strength of these interactions, we projected all pairwise combinations of MCDs, giving rise to 73,970 and 2,079,615 microcompartment interactions within and between chromosomes, respectively. Aggregate pileups demonstrate an average focal enrichment of interaction frequency of at least fourfold in RanGAP1–AID- and Nup93–AID-depleted cells extending to all genomic distances in cis and to trans chromosomal contacts (Fig. Enriched interactions between the MCDs were also observed in control G1 cells but with substantially reduced frequency (Fig. 3c), consistent with previous observations of weakly enriched interactions between active promoters and enhancers in wild-type cells8,54,57. The quantitatively lower enrichment of MCD contact frequency in Nup93–AID-depleted cells compared with RanGAP1 depletion may reflect clonal variation, as MCDs were assigned in the latter context, but we also noted marked differences in the short-range contact frequency heat maps of Nup93-depleted cells, including a more prominent maintenance of some control loops compared with the RanGAP1 depletion (Extended Data Fig. In a complementary analysis we determined the interaction frequencies between cCREs defined in control RanGAP1–AID cells. Pairs of CTCF sites interacted frequently in control cells at genomic distances of <1 megabases (Mb), as expected, but were dramatically reduced in RanGAP1–AID- or Nup93–AID-depleted G1 cells (Fig. Conversely, we found that active promoters and enhancers interacted with each other at considerably elevated frequencies regardless of genomic separation when nuclei formed in the absence of RanGAP1 or Nup93 (Fig. These interactions also occurred in control cells but at lower frequencies. Together, we found that the propensity to form a distinct microcompartment of active CREs during G1 is an intrinsic capacity of post-mitotic chromosomes that emerges as a pronounced feature in the absence of active nuclear transport. Telophase is the last mitotic stage when chromosomes can recruit cytoplasmic factors independently of nuclear import. We therefore sought to determine the folding state of telophase chromosomes using Hi-C. Pro-metaphase-arrested control and RanGAP1–AID-depleted cells were released and fixed after 1.25 or 1.50 h, the point where we observed the greatest proportion of telophase cells in the post-mitotic population (Fig. 1d, upper right quadrant), and stained with propidium iodide (PI). 3a), which briefly peaks in the transition from metaphase to G1 (ref. At 1.25 h post release, at least 60% of the sorted cells were in telophase, as evidenced by elongated mid-body microtubules connecting the two masses of highly condensed chromosomes (Extended Data Fig. Cells sorted in the same way at 1.5 h post release had progressed through telophase and we could identify more than half of the sorted cells having a clear abscission mid-body, indicating they were in cytokinesis. In our experimental system, nuclear volume increased by about 50% from telophase to cytokinesis in control cells and dramatically expanded by almost threefold in early G1 (Fig. a, Enrichment of telophase and cytokinesis cells by FACS for Hi-C analysis (left). Fixed cells were collected 1.25–1.50 h following prometaphase release and sorted based on DNA (PI) content and width (top right). The percentage of cells in each quadrant of the flow cytometry plots is provided. Immunofluorescence images showing enrichment of cells in telophase or cytokinesis based on chromatin (DAPI) and microtubule (ɑ-tubulin) morphology (bottom right). Scale bar, 5 µm b, The nuclear volume of control and RanGAP1-depleted cells enriched in telophase, cytokinesis or G1 indicates a lack of nuclear growth after telophase in import-incompetent nuclei. Data for 25 DAPI-stained nuclei per condition are provided along with the mean of each of the two replicates (R1 and R2). c, Hi-C interaction frequency maps (Chr 14: 53.5–55.5 versus 67.8–69.0 Mb) at 10 kb resolution showing genome organization in control and auxin-treated RanGAP1–AID cells in prometaphase, telophase, cytokinesis or early G1. MCDs detected in RanGAP1-depleted cytokinesis- or G1-sorted cell populations are shown. d, Pairwise mean observed/expected contact frequency between MCD anchors in cis (10 kb) and trans (25 kb) showing enhanced interactions in telophase that peak at cytokinesis in control cells. 2,105 G1 and 1,791 cytokinesis MCD anchors are categorized as cytokinesis-specific, shared G1-cytokinesis or G1-specific in RanGAP1-depleted cells. e, Heat maps for subsetted MCD anchors centred on contact frequency summits with a 100-kb flank and sorted by MCD length demonstrate the prevalence of H3K4me3 and H3K27ac coverage in mitotic, control G1 and RanGAP1-depleted G1 (ΔG1) cells at MCD anchors (bottom). The cis 10 kb EV1 values for control and ΔG1 cells, control CTCF coverage and convergent CTCF loop anchor assignment of control G1 cells are shown. Cyto, cytokinesis; pro-Meta, prometaphase; Telo, telophase. Source numerical data are provided. We performed Hi-C with FACS-sorted populations of prometaphase, telophase, cytokinesis and G1 cells, and observed the progressive acquisition of interphase chromosome compartments and loops in control cells (Fig. Remarkably, unperturbed cells were found to form MCD contacts that were visually apparent in contact frequency heat maps by cytokinesis. We compared the 2,105 MCDs identified in G1 to those identified in cytokinesis-sorted RanGAP1-depleted cells and found that more than half of all G1 MCDs were detected at the earlier time point (Extended Data Fig. In addition, a substantial number of microcompartment anchors were only called in cytokinesis or only in G1 but aggregate analysis demonstrated their interactions did indeed occur in cis and trans in both cell-cycle stages, albeit with different relative frequencies (Fig. Pairwise interactions between MCDs identified as cytokinesis-specific, G1-specific or shared cytokinesis-G1 were formed in cis and trans during telophase and increased in strength towards cytokinesis in both control and RanGAP1–AID-depleted cells (Fig. In the absence of nuclear import, these active microcompartments continued to strengthen as cells entered G1. By contrast, progression into G1 in control cells coincided with a strong reduction in the frequency of pairwise MCD interactions. Together, these data demonstrate that extensive cis and trans microcompartmentalization of active cCREs is a normal transient process that peaks during cytokinesis and is then lost or at least severely reduced during G1. Similar short-range microcompartment dynamics have been seen during mitotic exit in mouse cells59. This loss was not observed in RanGAP1–AID- or Nup93–AID-depleted cells and MCD interactions continued to increase in frequency as cells progressed through G1. The MCDs detected during cytokinesis were relatively enriched for marks of active CREs compared with those detected later in G1 (Fig. Although cytokinesis-specific MCDs displayed slightly different interaction dynamics during the time course, they were similarly enriched for active CREs. Accordingly, contacts between active promoters and enhancers mirrored the interactions between MCDs during mitotic exit (Extended Data Fig. The average contact frequency between CREs increased in telophase, reaching a peak at all length scales during cytokinesis, and was relatively diminished in a distance-dependent way in control cells by early G1. As mitotic exit coincides with transcriptional reactivation60,61, we sought to examine the relationship between transcriptional activity and microcompartment formation in control and RanGAP1-depleted cells. We used Thiol(SH)-linked alkylation for the metabolic sequencing of RNA (SLAM–seq)62,63 to directly quantify nascent RNA molecules synthesized in 1-h increments from prometaphase to early G1 (Fig. We found that the new-to-total RNA ratios (NTRs) increased relative to prometaphase levels from 2 to 5 h in control cells but globally plateaued after the initial minor increase of nascent synthesis, 2 h following prometaphase release, in RanGAP1-depleted cells (Fig. 4b,c and Extended Data Fig. a, Workflow for pulse labelling of nascent transcripts during prometaphase or mitotic release in the presence or absence of RanGAP1. Nascent transcripts were labelled by the addition of 4-thio-uridine (4sU) for 1 h before RNA extraction at prometaphase (t = 0) and 1, 2, 3, 4 or 5 h after release. The addition of auxin leads to thiol-specific alkylation and enables site-specific substitution of a guanine. SLAM–seq identifies T-to-C conversions by high-throughput sequencing and enables the quantification of nascent and previously transcribed RNA pools. b, Coverage tracks of 4SU-converted reads in control and RanGAP1-depleted cells collected hourly following mitotic release normalized to the prometaphase (t = 0)-arrested signal demonstrating attenuated post-mitotic transcription in RanGAP1-depleted cells. Substantial transcriptional activity starting 2 h after mitotic release that continues to increase into G1 characterizes genes in control samples but is attenuated in the absence of RanGAP1. Boxplots indicate the median and interquartile range (box bounds) of the NTR fold changes for three independent SLAM–seq libraries per time point. d, Heat maps of the mean NTRs for genes classified by the temporal stage at which NTR ≥ 10% (n = 9,486 control (left) and 2,872 RanGAP1-depleted (right) genes): prometaphase, 0 h (mito); early, 1–2 h and late, 3–5 h. Transcriptional clusters identified in control samples contrast the attenuated transcriptional activity for clusters identified in RanGAP1-depleted samples. e, Odds ratios (ORs) showing the strength of association between genes at MCDs and post-mitotic gene reactivation clusters. Genes expressed by the early samples are enriched in control but not RanGAP1-depleted conditions at strong G1-cytokinesis-intersected MCDs (bottom). *P < 0.1; one-sided Fisher's exact test, followed by Benjamini–Hochberg correction. Source numerical data are provided. Three clusters of genes were defined by the time at which post-mitotic nascent RNA synthesis first exceeded an NTR of 10%: prometaphase (t = 0), early (t = 1–2 h), and late (t = 3–5 h) genes. In control cells, nascent RNA synthesis increased over time through waves of gene activation with earlier expressed genes generally reaching higher relative levels of new transcription by G1 (Fig. In RanGAP1-depleted cells, these genes displayed reduced transcriptional activation during mitotic exit and generally reduced nascent synthesis (Extended Data Fig. The relatively few genes with substantial levels of nascent synthesis in RanGAP1-depleted cells were predominantly early expressed genes that were transcribed at reduced levels by G1 (Fig. 4d (right)), consistent with a minimal capacity for new transcription during the early stages of mitotic exit that is severely attenuated by G1. Prometaphase and early expressed genes were significantly enriched at MCDs, particularly those detected in cytokinesis (Fig. However, this association was found for early expressed control genes but not the RanGAP1-depleted set (Fig. 4e), suggesting a greater correspondence of MCDs to the control transcriptional programme inherited from the previous cell cycle. The global reduction in nascent RNA synthesis in RanGAP1-depleted cells indicates that ongoing transcription is not required to maintain or strengthen these interactions in G1. However, the enrichment of early expressed genes at strong MCDs, implies a role for the microcompartmentalization of active elements in driving post-mitotic gene activation in control cells. The appearance of a microcompartment during telophase that overlaps the specific active CREs of wild-type cells implies an inherited feature. To investigate this possibility, we performed Omni-ATAC–seq64 (Methods) and cleavage under targets and release using nuclease (CUT&RUN)65 targeting H3K4me3, H3K27ac or CTCF for cells arrested in prometaphase or released to early G1 (Fig. Consistent with previous work29, the ATAC–seq fragment-length distributions indicated more regularly spaced nucleosomes during mitosis compared with G1 cells, which was not impacted by the depletion of RanGAP1 (Extended Data Fig. We identified 159,072 peaks of ATAC–seq coverage across all conditions and compared their intersection (Fig. Although the largest fraction of peaks found genome-wide were G1-specific (left), accessible regions at MCDs were predominantly mitotically retained and RanGAP1 independent (right). These peaks correspond to highly accessible H3K4me3- and H3K27ac-dense loci (Fig. 5a,c), consistent with mitotic bookmarking at CREs. More than 90% of the MCDs overlapped at least one bookmarked promoter- or enhancer-annotated ATAC–seq peak and the valency of these transcriptional elements was increased at the stronger domains that were detected in cytokinesis (Fig. A small subset of G1-specific peaks that became accessible in the import-deficient G1 cells were also enriched at MCDs. However, more than half of the MCDs did not overlap any promoter or enhancer specific to these G1-specific peaks, suggesting that the G1-acquired open sites are not essential for MCD interactions (Extended Data Fig. Together, these data suggest that the microcompartments formed during telophase are driven by mitotically inherited regions of robust chromatin accessibility. a, Representative Hi-C contact matrices at 10 kb resolution (Chr 4: 74.7–74.8 × 73.9–78.7 Mb) highlighting microcompartment interactions for MCDs detected in cytokinesis and G1 RanGAP1-depleted cells. Coverage tracks from ATAC–seq as well as CTCF, H3K4me3 and H3K27ac CUT&RUN generated in prometaphase-arrested, G1 control and RanGAP1-depleted G1 cells indicate the prevalence of mitotic booking at active genomic elements. Gene annotations are shown for bulk RNA sequencing. b, Intersections of all (left) or MCD-overlapped (right) ATAC–seq peaks detected in prometaphase, G1 control and RanGAP1-depleted G1 cells plotted as a fraction of the total. The number of called peaks are indicated. c, Heat maps centred on the union set of ATAC–seq peaks detected in prometaphase-arrested or G1 cells released for 5 h with or without RanGAP1. Stacks sorted by prometaphase ATAC–seq signal demonstrate co-occurrence of G1 signals at mitotic peaks and the prevalence of H3K27ac, H3K4me3, RNA sequencing and MCD anchors. d, Prevalence of bookmarked CREs assigned on control prometaphase and G1 ATAC–seq, H3K4me3 and H3K27ac coverage. Fractions of 50 kb bins genome-wide or having EV1 > 0 as well as all G1 shared G1-cytokinesis (cyto) or G1-specific MCDs overlapping the indicated valency of promoter and enhancer elements are shown. e, Representative western blot images of whole-cell lysates derived from control and RanGAP1-depleted cells arrested in prometaphase or 5 h after mitotic release, followed by treatment with either JQ1 or dBET6 indicate efficient degradation of both BRD2 and BRD4 in the dBET6-treated cells but not JQ1-treated cells. Vinculin was used as a loading control. f, Pairwise mean observed/expected contact frequency between MCD anchors in control and RanGAP1-depleted cells demonstrating unchanged cell cycle-dependent interaction strength following treatment with either JQ1 or dBET6, used to inhibit or degrade BET proteins, respectively. Source numerical data and unprocessed blots are provided. Histone H3K27 acetylation is implicated in both post-mitotic gene activation40,41,61,66 and 3D genome organization8,57. We observed that bookmarked peaks of enriched H3K27ac not only corresponded to MCDs but could also recapitulate microcompartment dynamics during mitotic exit, forming enriched contacts in cis and trans that increased from telophase to cytokinesis and were attenuated as cells entered G1 in the presence of RanGAP1 (Extended Data Fig. These observations are consistent with a capacity for chromatin readers, such as Bromo and Extra-Terminal (BET) domain proteins, to bind acetylated histones during mitosis12,66,67 and form phase-separated condensates11,68,69,70. To determine whether they mediate microcompartment formation during mitotic exit, BET domain proteins were targeted using the competitive inhibitor JQ1 or dBET6 (a proteolysis targeting chimaera, PROTAC), which degrades BRD2 and BRD4 in prometaphase-arrested cells (Fig. Surprisingly, inhibition or loss of BET domain proteins did not attenuate pairwise interaction frequencies between MCDs or bookmarked H3K27ac peaks in control or RanGAP1-depleted cells during mitotic exit (Fig. 7f), indicating that another factor or H3K27ac itself must drive the transient clustering of MCDs. The condensin-mediated loop array that compacts mitotic chromosomes is replaced by the extrusive activity of cohesin around telophase38,39,42, giving rise to CTCF loops, TADs and insulating domain boundaries. Average Hi-C contact probability plotted as a function of genomic separation, P(s), provides an estimation of extruded loop size at the distance of minimal decay25,71,72,73,74. In control cells, condensin loops of 200–300 kb disappeared at telophase and were undetectable by cytokinesis. Following G1 entry, loops of approximately 100 kb were observed, consistent with cohesin-mediated extrusion (Fig. In the absence of RanGAP1, the condensin loops were lost with similar kinetics to controls but 100-kb loops were not established and the P(s) curve indicates the absence of any extruded loops from cytokinesis onwards. Accordingly, loops at convergent CTCF motifs first appeared in telophase and strengthened by at least fivefold in early G1 control cells, but these interactions remained rare throughout mitotic exit in the absence of RanGAP1 (Fig. Depletion of Nup93–AID similarly prevented the global establishment of strong interphase loops (Extended Data Fig. 8a,b), suggesting that nuclear import is a general requirement for cohesin-mediated loop extrusion during G1 entry. Immunofluorescence confirms exclusion of the cohesin subunit Rad21 from post-mitotic nuclei assembled in the absence of RanGAP1 or Nup93, whereas CTCF retained access to chromatin in the nucleus (Fig. Accumulation of CTCF on the chromatin provides support for previous findings suggesting the rapid recruitment of CTCF following mitotic exit precedes the cohesin-mediated extrusion of interphase loops38. a, P(s) plots for Hi-C data from control and RanGAP1-depleted FACS-sorted cells at discrete phases. Brown shaded lines indicate the average size of cohesin- or condensin-extruded loops, 100kb or 200–300 kb loops, respectively. b, Mean observed/expected Hi-C contact frequency at 18,613 convergent CTCF loops demonstrating RanGAP1-dependent progression during mitotic exit. Interactions averaged over three 10 kb bins across the 200 kb CTCF motif-centred window and stack-ups sorted by G1 loop strength are shown. c, Representative immunofluorescence images of RanGAP1–AID control and depleted cells in G1 demonstrating nucleocytoplasmic localization of Rad21 and CTCF. d,e, Mean convergent CTCF loop strength (d) and mean strength of cis MCD–MCD contacts (e) of RanGAP1–AID control and depleted cells during mitotic exit. f, Mean strength of cis MCD–MCD contacts in G1 demonstrating the distance-dependent reduction in interaction strength beyond the range of control convergent CTCF loops in control cells. The 90th percentile of loop length is indicated by a grey line g, Mean strength over time of MCD–MCD contacts classified by the presence (> 0) or absence of at least one CTCF loop anchor. h, Mean observed/expected MCD–MCD contact frequency at 10 kb during mitotic exit categorized by the presence of a convergent CTCF loop or loop anchor (>0) and looping domain status. i, Schematic of the microcompartment fates in G1 cells. j, Representative Hi-C interaction frequency maps at 25 kb (Chr 7: 20.5–28.5 Mb) resolution and a 10 kb zoom-in (Chr 7: 25.7–28.0 Mb) showing genome organization between looping domains in control (top) and RanGAP1–AID-depleted G1 cells (bottom). k, Representative Hi-C interaction frequency maps at 25 kb (Chr 8: 29.5–39.5 Mb) and 10 kb (Chr 8: 29.5–31.4 Mb versus 36.5–39.8 Mb) resolutions showing genome organization at loop-anchored and extrusion-free MCDs in control and RanGAP1–AID-depleted G1 cells. j,k, MCD positions, control cell EV1 values, H3K27ac CUT&RUN signal and gene annotations are shown. Colored arrows indicate fates of different microcompartments in control and RanGAP1–AID-depleted G1 cells. Ctrl, control; Cyto, cytokinesis; pro-Meta, prometaphase; Telo, telophase. Source numerical data are provided. Loop extrusion counteracts epigenetically defined chromatin compartmentalization72,74,75,76 and we observed a decrease in MCD interaction strength in control cells as the strength of extruded loops increased in G1 (Fig. The reduced average MCD–MCD interaction frequency was driven by long-range interactions >750 kb, beyond the range of most extruded loops (Fig. To address the relationship between extrusion and the loss of telophase microcompartments as cells enter G1, MCDs were categorized by the presence of a CTCF loop anchor. At 10 kb resolution, extrusion anchors were found at more than 75% of the MCDs (Fig. The MCD interactions that overlapped at least one CTCF loop anchor were only maintained over shorter distances (<750 kb) from cytokinesis to G1, whereas extrusion-free MCD contacts lacked a distance-dependent loss in contact frequency. These findings imply a role for loop extrusion in modifying the global network of pairwise MCD interactions as cells exit mitosis through a process we refer to as ‘pruning'. Microcompartments that appeared in cytokinesis and coincided with interphase CTCF loops were not only retained but strengthened in control G1 cells (Fig. 6h–j), suggesting that although the initial MCD–MCD interactions are cohesin-independent, cohesin-mediated loop extrusion specifically reinforces these contacts. The fact that cohesin loops are typically less than 1 Mb can explain why only relatively short-range microcompartments are maintained in this manner, whereas longer-range contacts are pruned in G1 (Fig. As these MCDs become tethered to a small set of nearby MCDs to give rise to interphase extrusion loops, interactions with other more distal MCDs become far less probable. We further classified microcompartment interactions based on the domain identity of the constituent MCDs. We found the most reduced MCD–MCD interactions in control G1 cells overlap a loop anchor and span distinct looping domains (inter-TAD). Conversely, intra-TAD microcompartments and inter-TAD contacts that did not overlap any CTCF loop anchors were reduced to a much lesser extent as control cells exited mitosis (Fig. Notably, the differences in MCD–MCD interactions with respect to loop extrusion anchors observed 5 h after mitotic release in RanGAP1–AID cells could be recapitulated in Nup93–AID cells, confirming generality in DLD-1 (Extended Data Fig. Together, we find that the extensive indiscriminate cCRE microcompartment formed during mitotic exit is pruned following the establishment of active nucleocytoplasmic transport. We propose that cohesin-mediated loop extrusion, and the resulting TADs and CTCF–CTCF loops, impose constraints on which pairwise MCD–MCD interactions remain and which microcompartments melt into larger subcompartments (Fig. To evaluate whether the pruned state can be reversed, we performed Hi-C in cells depleted of RanGAP1 specifically after mitosis by the addition of auxin 3.5 h after prometaphase release (Extended Data Fig. At 3.5 h, loops had formed and MCD interactions were mostly pruned in control cells, whereas RanGAP1-depleted cells lacked extrusion loops and possessed extensive microcompartment interactions (Extended Data Fig. RanGAP1 loss after early G1 leads to a modest gain of long-range contacts between MCDs that are infrequent in control cells and indicates the capacity to re-form MCD contacts after pruning has occurred. This gain in MCD interactions coincided with reduced nuclear localization of cohesin (Extended Data Fig. 9d), suggesting that continuous active nuclear transport is required to maintain nuclear cohesin levels and the fully pruned microcompartment. The relatively minor impact of G1 RanGAP1 depletion could be explained by retained levels of nuclear cohesin. Degradation of RanGAP1 during G1 most closely resembled the early G1 (3.5 h) control condition (Supplementary Fig. This state is characterized by an increase in the size of cohesin-extruded loops (Extended Data Fig. 9e, vertical arrows) and the presence of both MCD–MCD interactions and convergent CTCF loops (Extended Data Fig. Nonetheless, loss of nuclear transport after the occurrence of G1 pruning facilitated an additional increase in average MCD interactions at all genomic distances particularly across looping domains (Extended Data Fig. 9h), indicating a requirement for continuous active nuclear transport to maintain pruning of non-specific MCD interactions and a retained capacity to form these long-range contacts in interphase. Using EV1 from Eigenvector decomposition of the Hi-C data to define A and B compartments revealed the expected acquisition of homotypic compartmental segregation after mitotic exit in both control and RanGAP1-depleted cells, which increased in strength through telophase, cytokinesis and G1 (Fig. The absence of active nuclear transport as cells progressed to G1 quantitatively impacted active and inactive chromatin compartmentalization (Supplementary Fig. 2c), leading to finer-scale active domains captured by EV1 (Fig. 10a,b) and enhanced A/B segregation (Fig. 10c) without global changes to A/B identity (Fig. These observations are consistent with previous findings in Nipbl-depleted cells where cohesin-dependent loop extrusion was effectively absent72. The binary classification of active and inactive chromatin from Hi-C data is an oversimplification of genome compartmentalization77 that can be expanded to include cell type-specific subcompartments of different types by various methods9,15,78,79. To further investigate intrinsic compartmentalization, we employed interaction profile groups (IPGs) that were previously defined in wild-type DLD-1 cells80 yielding active (A1 and A2), inactive (pooled to ‘all B') and transcriptionally intermediate (V/VI) chromatin subcompartments. Preferential homotypic interactions between these subcompartment loci increased as cells progressed from cytokinesis to G1 in control and RanGAP1-depleted cells (Fig. 7a,d,e (top)), indicating that compartmentalization at the scale of IPGs is entirely driven by determinants and factors associated with telophase chromosomes. a, Representative Hi-C interaction frequency maps at 50 kb resolution (Chr 17: 47.5–73.4 Mb) showing changes in genome organization from cytokinesis (upper triangle) to G1 (t = 5 h; lower triangle). EV1 values and IPGs are shown. Arrows indicate microcompartments observed in control cytokinesis-sorted cells (grey), which are resorbed into subcompartments by early G1 (black). b, Saddle plots representing the segregation of active (A) and inactive (B) chromatin compartments in cis for control and RanGAP1-depleted cells 5 h after mitotic release. The EV1 from each condition was used to rank 25 kb genomic bins into equal quantiles and the average interaction frequency between these ranked bins was normalized to the expected interactions to build the heat map. Average preferential A–A and B–B interactions for the strongest 20% A and B loci are indicated (white squares). c, Average preferential A–A and B–B interaction strength in control and RanGAP1–AID-depleted cells demonstrating increased A and B segregation over the course of mitotic exit. Ranked bins designating A and B compartments were derived from the EV1 of the 5 h control G1 cells at 25 kb resolution for comparison. d, Pairwise aggregate intra (cis) and interchromosomal (trans) observed/expected contact frequency between DLD-1 IPGs showing enhanced homotypic interactions in control and RanGAP1–AID-depleted cells from cytokinesis to early G1. Aggregate observed/expected interactions between MCDs as a subset of each IPG demonstrate the cell-cycle dynamics of a distinct microcompartment in the presence or absence of RanGAP1 (bottom). e, Quantification of the aggregate saddle plots in d. The mean observed/expected homotypic interaction frequency is shown for all IPGs (top) and IPGs stratified according to the presence or absence of an MCD. Lines are coloured as per the subcompartments in d. Source numerical data are provided. Microcompartment domains occurred at similar frequencies at A1 (6.6%), A2 (6.7%) and V/VI (5.9%) IPGs, and were largely absent from the B compartment (0.3%). We found that MCDs interacted most frequently with other MCDs, regardless of their subcompartment status, during cytokinesis (Fig. This preference for MCD–MCD interactions was lost in control G1 cells when MCD interaction frequencies reflected the canonical subcompartment interaction affinities. Conversely, the MCD–MCD interactions continued to strengthen during G1 in the absence of RanGAP1, maintaining a strong preference for microcompartmental interactions over contacts with other loci of the same subcompartment type. These data are consistent with the modified spectral clustering performed on RanGAP1-depletion Hi-C data, which demonstrated a distinct interaction profile for MCDs that did not correspond to a single specific IPG defined in wild-type cells (Extended Data Fig. These results demonstrate that the cCRE microcompartment is quantitatively and temporally distinct from subcompartments defined in control G1 cells. In control cells, MCDs initially interacted regardless of subcompartment status but these interactions dissolved (were ‘resorbed') and replaced by interactions with loci, including with loci that were not MCDs, that were generally assigned the same corresponding subcompartment status during mitotic exit. This resorption did not occur in nuclear import-deficient cells for which the cCRE microcompartment continued to strengthen (Fig. 7a, e(arrows) and Extended Data Fig. For a better understanding of the chromatin state that gives rise to the cCRE microcompartment and to identify cytoplasmic factors that normally modify this pattern, we employed stable isotope labelling by amino acids in cell culture (SILAC)81, followed by liquid chromatography with mass spectrometry (LC–MS) of nuclei isolated from cells exiting mitosis (Fig. Focusing on common changes to the nuclear proteome between RanGAP1 and Nup93 depletion, we found that universal changes to the protein composition were mostly factors that failed to localize to the nucleus as cells entered G1 (Fig. Classification of these proteins according to annotated functions82,83 identified biological processes that were retained or lost in import-deficient nuclei (Fig. As expected, chromatin-associated processes were enriched in the proteomes of isolated control nuclei. Consistent with our observations of normal mitotic exit kinetics, the absence of active nuclear transport did not consistently impact the presence of factors related to mitotic cell division, including proteins essential to mitotic chromosome condensation (Fig. We confirmed that the cohesin complex was completely absent in transport-incompetent nuclei. Other strongly depleted processes included DNA replication and repair as well as RNA processing and ribosome biogenesis, the latter being consistent with the absence of nuclear speckles and nucleoli in depleted nuclei (Extended Data Fig. Proteins involved in DNA-templated transcription were found to require nuclear import for access to the genome in early G1, which was confirmed by a dramatic reduction in components of the basal transcription machinery (Fig. 8d) and cytoplasmic localization of RNA pol II by immunofluorescence (Fig. Finally, although we found that chromatin organization factors were relatively less consistent between depletions, many chromatin remodelling enzymes were substantially absent from the nucleus in both RanGAP1 and Nup93 depletions, whereas structural components of heterochromatin and euchromatin may be less dependent on transport for nuclear localization in newly divided cells. We conclude that loop extrusion, transcription and RNA processing probably do not occur, or occur to a dramatically reduced degree, in nuclei formed during mitotic exit in the absence of active nuclear transport. Given that canonical chromatin compartments and the cCRE microcompartment form under these conditions, our data suggest that these active processes are mechanistically dispensable for compartmentalization during mitotic exit. a, Experimental workflow for SILAC-based LC–MS quantification of nuclear proteome alterations found in cells entering G1 (t = 5 h) in the absence of either RanGAP1 or Nup93. Auxin-induced degradation was initiated 2 h before nocodazole/mitotic release (t = 0). b, Volcano plots for the consensus list of proteins identified in early G1 nuclear isolates from RanGAP1– (top) and Nup93–AID (bottom) cells. For each protein, the enrichment in auxin-treated versus control (log2-tranformed) is plotted against the multiple-testing Benjamini–Hochberg (BH)-adjusted P values from two-sided non-parametric permutation tests (P = 0.05 and the P = BH threshold are indicated). Two pooled replicates for heavy and light reversed experiments are shown and each protein is coloured according to the auxin-treated/control ratio in the RanGAP1–AID. c, Gene ontology over-representation analysis for selected biological processes among all proteins identified in RanGAP1–AID and Nup93–AID control G1 nuclei (all found), or proteins reduced by at least twofold in the nuclei of both RanGAP–AID- and Nup93–AID-depleted cells (down (δ)). Dots are coloured according to the false discovery rate (FDR) and dot sizes indicate fold enrichment. The log2-transformed fold enrichment in auxin-treated versus control nuclei was calculated separately for each replicate and for the pooled (reversed) LC–MS spectra. Any significant difference in the pooled samples, based on Benjamini–Hochberg (BH)-adjusted P values, are indicated by an asterix. e, Representative immunofluorescence images (at least two independent experiments) of RanGAP1–AID or Nup93–AID control and depleted cells fixed 5 h after mitotic release demonstrating the nucleocytoplasmic localization of RNA pol II (red). Lamin A/C indicates the nuclear periphery. We propose the existence of two folding programmes that specify interphase chromosome conformation as cells exit mitosis and enter G1. The first programme is driven by factors that associate with chromosomes no later than telophase to drive genome-wide compartmentalization based on the epigenetic signatures of the previous cell cycle. This programme includes and may even be driven by mitotically bookmarked cis elements and trans factors that form a prominent indiscriminate cCRE microcompartment starting in telophase. Although inherited in the same way, the cCRE microcompartment is distinct from conventional subcompartments, both in composition and cell cycle-dependent dynamics. A second folding programme starts after nuclear-envelope formation and requires active nucleocytoplasmic transport, implying that the relevant factors are inherited through mitosis in the cytoplasm. This programme includes cohesin, which drives the formation of interphase loops and TADs. It is intriguing that the two main known mechanisms of chromosome folding, compartmentalization and loop extrusion are inherited through mitosis in two distinct and physically segregated ways. The extensive cCRE microcompartment observed during mitotic exit is a transient folding state that is extensively modulated as cells enter and progress through G1. Microcompartments constitute a discrete compartment type derived from different active subcompartments that display preferential homotypic interactions during cytokinesis and persist into G1 in the absence of nuclear active transport. The cCRE microcompartment, and canonical A and B compartments and subcompartments are all formed without import of factors from the cytoplasm, indicating that affinity-driven compartmentalization is generally mediated by factors stably associated with mitotic chromosomes or rapidly recruited before telophase. We found that the cCREs at strong MCDs are bookmarked during mitosis. Furthermore, histone modifications associated with active bookmarked loci (for example, H3K4Me3 and H3K72Ac) and more generally corresponding to subcompartments are at least partly retained during mitosis. It is possible that histone modifications are sufficient to mediate affinity-driven compartmentalization, although in vitro studies have suggested that bridging factors, such as BET proteins, mediate interactions between loci marked with acetylated histones11. Previous work has shown that Brd2 and Brd4 contribute to genome compartmentalization during interphase84 and at condensin-depleted mitotic chromosomes85, respectively. However, we found that inhibition or degradation of BRD2 and BRD4 did not impact microcompartment interactions during mitotic exit, which implies an alternative mechanism for microcompartment affinity. Consistent with previous reports60,61, we observed that a burst of transcription occurs shortly after mitotic exit. The earliest genes expressed following mitotic exit in control cells are preferentially enriched at MCDs compared with genes of the late transcriptional programme, suggesting that microcompartment formation in telophase–cytokinesis could contribute to early post-mitotic gene reactivation. We noted that by G1, when microcompartment interactions are most frequent in RanGAP1-depleted cells, nascent RNA synthesis was strongly attenuated and the transcriptional machinery was excluded from the nucleus. Together, these findings suggest that microcompartment contacts formed during mitotic exit could promote early post-mitotic gene reactivation in control cells but that microcompartment formation itself is independent of transcription. Similarly, compartmentalization has been linked to genome localization at speckles86 and nucleoli87 but the absence of these structures in nuclear transport-deficient cells did not adversely impact any level of global compartmentalization we examined. We found that a second folding programme is inherited through the cytoplasm at the end of mitosis and thus relies on nuclear transport-dependent factors. Our ATAC–seq data demonstrate that the cell type-specific G1 chromatin landscape relies on chromosome-intrinsic and cytoplasmic factors. Although many cCREs overlapping MCDs remain accessible, bookmarked sites may also be inaccessible during mitosis and require active remodelling during G1 (ref. Accordingly, our proteomics analysis identified a number of import-dependent chromatin remodelling complexes that may contribute to re-establishing the G1-specific architecture. Cohesin can be recruited anywhere along the genome but loop extrusion patterns are sensitive to the presence and location of active cis elements16,22 and transcriptional machinery88. The second folding programme inherited through the cytoplasm may therefore open distal regulatory sites and define a cell type-specific loop-extrusion traffic pattern. The nuclear import requirement for cohesin accumulation at post-mitotic chromosomes, recently observed by microscopy42, physically and temporally segregates two modes of chromatin extrusion at the end of mitosis, which is consistent with the existence of an extrusion-free chromosome folding intermediate during telophase39. Although the relevance of this separation remains to be seen, it enabled functional segregation of the two main chromatin folding mechanisms in our system. The chromosome-intrinsic capacity for previously active and mitotically bookmarked promoters and enhancers to interact gives rise to a comprehensive array of non-specific contacts during telophase–cytokinesis. Initial microcompartment formation is promiscuous in cis and trans, and does not require loop extrusion. The formation of extrusion-independent promoter-enhancer contacts has been observed previously in different cell contexts and cell-cycle stages54,59,89,90,91,92,93. It is possible that this microcompartment serves to rapidly re-establish gene-regulatory contacts during mitotic exit. As cytoplasmic factors enter the nucleus, long-range (>1 Mb) MCD–MCD interactions are substantially reduced or pruned. We propose that G1 pruning of indiscriminate MCD contacts is driven by cohesin-mediated loop extrusion. Most MCDs overlap CTCF sites that anchor extruded loops in G1. These MCDs retain short-range interactions with other MCDs, possibly reinforced by loop extrusion. Pruning during mitotic exit ensures that MCD–MCD interactions start to obey rules of TAD formation such that interactions within extrusion domains are more likely to be retained than interactions between domains. Because the effect of pruning can be partially reversed by RanGAP1 depletion in G1, we conclude that interphase chromatin folding is the result of continuous interplay between intrinsic affinity-based compartmentalization and cytoplasmic inherited processes, such as loop extrusion. We propose that the chromosome-intrinsic folding programme reveals a universal propensity for active gene-regulatory elements to interact through affinity-driven interactions. A second folding programme superimposes a more deterministic logic through regulated cohesin-mediated loop extrusion that ensures the specific pairing of cCREs to ensure cell type-specific gene expression. With the exception of nuclear import assays, all experiments described here employed human colorectal adenocarcinoma DLD-1 cells (American Type Culture Collection, CCL-221) expressing either RanGAP1 or Nup93 homozygously tagged with NeonGreen and an AID, infra-red protein (IFP)-tagged RCC1 and Tir1, as described previously. We refer to these cell lines as RanGAP1–AID and Nup93–AID. For cell lines used in nuclear import assays, the sequences of MBP and mScarlet were amplified by PCR from pMAL (NEB) and pmScarlet_ɑTubulin_C1 (Addgene, catalogue number 85045), respectively. The NLS sequence was synthesized and all fragments were inserted by Gibson reaction (NEB, E2611S) into the multiple cloning site of AAVS1_Puro_PGK1 vector (Addgene, catalogue number 68375) through replacement of 3×FlagTwinStep-Tag. MBP–mScarlet–NLS was inserted into the AAVS1 locus in RanGAP1–AID and NUP93–AID DLD-1 cell lines. Modified DLD-1 cells lines (RanGAP1–AID, Nup93–AID, RanGAP1–AID/MBP–NLS and Nup93/MBP–NLS) were cultured at 37 °C with 5% CO2 in DMEM, high glucose, GlutaMAX supplement, pyruvate medium (Gibco, 10569010) with 10% fetal bovine serum (Gibco, 16000069) and 1% penicillin–streptomycin (Gibco, 15140122). The cells were synchronized in prometaphase of mitosis using a standard double thymidine-nocodazole block protocol. All incubations and washes were carried out in solutions pre-warmed to 37 °C. The cells were seeded at low density in DMEM medium containing 4 mM thymidine (Sigma, T1895) and incubated for 17 h, released (after two PBS washes) for 7 h in fresh medium to replicate the genome and enter mitosis, and incubated with 4 mM thymidine for an additional 17 h to obtain synchronous populations blocked in S-phase. After the second thymidine block, the cells were released (after two PBS washes) in fresh medium for 5 h, followed by an additional 5 h incubation in the presence of 50 ng ml−1 nocodazole (Sigma, M1404) to accumulate in prometaphase. Following incubation for an additional 2 h, the control and depleted cells were either collected for subsequent analyses (described later) or released into fresh medium to progress through mitosis for 1.25 h (telophase), 1.5 h (cytokinesis) or 5 h (early G1). To deplete RanGAP1 specifically in G1, the medium was exchanged 3.5 h after nocodazole release and adherent cells were incubated with auxin for an additional 6.5 h into late G1 (10 h). Matched 10 h control and depleted conditions were also subjected to removal of non-adherent cells at 3.5 h to ensure similar G1 synchrony. To inhibit BET protein activity during mitotic exit, 500 nM JQ1 (Selleckchem, S7110) or dBET6 (PROTAC; Selleckchem, S8762) was added to prometaphase-arrested cells in the presence or absence of auxin. After another 2 h of incubation, the control and RanGAP1-depleted cells were released for 5 h into early G1 in the presence or absence of BET inhibitors. DLD-1 RanGAP1–AID and NUP93–AID cell lines, expressing MBP–mScarlet–NLS, were cultured on four-well glass-bottomed chambers (Ibidi) and synchronized with CDK1 inhibitor (Selleckchem, Ro-3306) for 16 h. After 15 h in the CDK1 inhibitor, the cells were treated with fresh medium or medium containing 1 mM auxin for 1 h in the presence of the CDK1 inhibitor. The cells were imaged on an Eclipse Ti2 inverted microscope (Nikon) equipped with a spinning disk confocal system (UltraVIEW Vox Rapid Confocal Imager; PerkinElmer) and controlled with the Volocity software (PerkinElmer) utilizing a Nikon PlanFluor ×40/1.3 oil-immersion objective lens. The cells were imaged in FluoroBrite DMEM (Thermo Fisher Scientific) medium. The microscope was equipped with a temperature-, CO2- and humidity-control chamber that maintained a 5% CO2 atmosphere and 37 °C. RanGAP1-NG, NUP93-NG, MBP–mScarlet–NLS and RCC1iRFP670 fluorescent protein signals were excited with 488-nm (20–24% power, 200–250 ms exposure), 568-nm (5% power, 50–60 ms) and 640-nm (100% power, 150–200 ms exposure) laser lines, and binning set to 2×, respectively. A series of six z-slices, 2 µm optical sections, were acquired every 2 min and monitored for 6 h for three independent fields. Images were captured and analysed using the Volocity (PerkinElmer) and Image J (National Institutes of Health) software. Images represent a single z-stack. To calculate relative nuclear intensity, we used the following equation: The relative nuclear-to-cytoplasmic intensity of MBL–mScarlet–NLS was plotted. Data are expressed as individual values analysed every 10 min for eight control or auxin-treated cells of two independent fields of one experiment. Assessment of MBL–mScarlet–NLS import was performed in two independent experiments. Adherent cells were dissociated with accutase and equal numbers of cells were lysed with 2×Laemmli buffer (60 mM Tris pH 6.8, 10% glycerol, 2% SDS, 100 mM dithiothreitol) by boiling for 10 min. Proteins were separated on 4–12% NuPage bis-Tris gels using 1×MES running buffer (NuPAGE 20X buffer NP0002) for 45 min at 175 V in an Invitrogen XCell Sure-Lock mini gel and blotting system. The gels were transferred to 0.2-µm nitrocellulose membrane in Pierce 10X Western Blot Transfer Buffer, Methanol-free (Thermo Fisher Scientific, 35040) at 30 V for 1 h. For immunoblotting, the membranes were blocked with 5% milk in TBS-T (1×TBS + 0.1% Tween-20) at room temperature for at least 1 h. Primary antibodies were diluted in block buffer and incubated overnight at 4 °C and horseradish peroxidase (HRP)-linked secondary antibodies for 1 h at room temperature. The blots were developed and imaged using SuperSignal West Dura Extended Duration Substrate (Thermo Fisher Scientific, 34076) and a Bio-Rad ChemiDoc system. The following primary antibodies were used: mouse anti-RanGAP1 (OTI1B4; Novus Biologicals, NBP2-02623), mouse anti-Nup93 (F-2; Santa Cruz Biotechnology, sc-374400), rabbit anti-BRD4 (E2A7X; Cell Signaling Technologies, 13440), rabbit anti-BRD2 (EPR7642; Abcam, ab139690) and rabbit anti-vinculin (EP18185; Abcam, ab129002). The following secondary antibodies were used: goat anti-mouse IgG–HRP (Cell Signaling Technologies, 7076) and goat anti-rabbit IgG–HRP (Cell Signaling Technologies, 7074). Immunofluorescence was performed using standard methods. For early G1 immunofluorescence, cells were released from prometaphase onto glass coverslips and fixed with 4% paraformaldehyde in PBS for 30 min at room temperature. For mitotic release experiments, the cells were first fixed and then concentrated onto glass coverslips using an Epredia Cytospin at 150g. The fixed samples were incubated with PBS containing 2% Triton X-100 for 1 h before primary and secondary antibody staining in PBS containing 0.1% Triton X-100 for 3 and 1 h, respectively. After staining with 10 µg ml−1 DAPI in PBS for 10 min at room temperature, the coverslips were mounted in Vectashield antifade medium (Vector Labs, H-1000-10) for confocal imaging. The following secondary antibodies were used: goat anti-mouse IgG H&L Alexa Fluor 488 (Abcam, ab150113), goat anti-mouse IgG H&L Alexa Fluor 568 (Abcam, ab175473), goat anti-rabbit IgG H&L Alexa Fluor 488 (Abcam, ab15007) and goat anti-rabbit IgG H&L Alexa Fluor 568 (Abcam, ab175471). Images were acquired using a Nikon A1 point-scanning confocal microscope with GaAsP detectors (488 and 561 nm lasers) or a high sensitivity MultiAlkali PMT (405 nm laser) and an Apo TIRF, 1.49 numerical aperture, ×60 oil-immersion objective (Nikon). For chromatin volume estimation, fixed DAPI-stained cells were imaged in 40–60 consecutive 0.2-µm z-slices. The 3D volume of DAPI-containing signal was used as a proxy for chromatin volume and measured using the ‘3D Objects Counter' Plugin97 of the Fiji software98 on pre-filtered (Gaussian, σ = 2) image stacks. Cells were collected at various points of mitotic exit. Adherent cells were dissociated with accutase (Thermo Fisher Scientific, A11105-01) and pooled with non-adherent collected cells to assess the entire population. To assess the cell-cycle profile (DNA content), cell pellets were resuspended in 200 µl PBS and fixed with 800 µl of cold 100% ethanol. The cells were stored at −20 °C for at least 24 h. Approximately 1 × 106 fixed cells were stained with 50 µg ml−1 PI (Thermo, P1304MP), diluted in 1 ml PBS containing 50 µg ml−1 RNase A (Roche, 10109169001) and 0.1% saponin, for 1 h at room temperature. After staining, the cells were spun, and the pellets were resuspended in 1 ml PBS and passed through a 35-µm filter (Falcon, 352235). Flow cytometry was performed on a MACSQUANT set-up. Analysis was performed using the FlowJo software (v10) and plots reflect populations gated for debris but not doublets. Cells were collected at various points of mitotic exit and fixed for Hi-C 3.0 analysis52 with a few modifications to facilitate cell sorting. Adherent cells collected 5 and 10 h after prometaphase release were dissociated with accutase. Prometaphase (t = 0) and early (t = 1.25–1.50 h) released cells were directly collected by shake-off. Cell suspensions were pelleted and treated with accutase for an additional 5 min at room temperature to prevent aggregation and washed with HBSS (Thermo Fisher Scientific, 14025134). Fixation proceeded first with 1% formaldehyde (Fisher Scientific, BP531-25) in HBSS for 10 min, which was quenched with 0.125 M glycine for 5 min at room temperature and 15 min on ice. Next, the cells were fixed with 3 mM disuccinimidyl glutarate in PBS for 40 min at room temperature with rotation, followed by a second quenching with 0.125 M glycine. The fixed cells were washed twice with PBS + 0.1% BSA and snap-frozen in liquid nitrogen before staining for FACS analysis. To sort cells by DNA content, approximately 10 × 106 fixed cells were stained with 50 µg ml−1 PI, diluted in 5 ml PBS containing 50 µg ml−1 RNase A and 0.1% saponin, for 1 h at room temperature. The cells were then spun and washed with PBS before resuspension in 2 ml PBS + 0.1% BSA and passage through a 35-µm filter. Propidium iodide-stained cell suspensions were sorted on a BD FACS Melody system using the 561 nm laser for forward scatter, side scatter and PI. All populations were gated based on forward scatter/side scatter to eliminate cell debris and cells sorted for either prometaphase (4n) or G1 (2n) DNA content were also subject to doublet discrimination. To enrich for telophase or cytokinesis, cells fixed 1.25 and 1.50 h after mitotic release, respectively, were sorted based on doubled PI signal (DNA content) area and width. All sorted cells were collected in PBS containing 1% BSA and washed twice in PBS before being snap-frozen. Chromosome conformation capture was performed as previously described52 with some modifications. Synchronized cells were crosslinked with 1% formaldehyde and 3 mM disuccinimidyl glutarate, and enriched in specific cell-cycle stages by FACS, as described above. Sorted cells (1–5 × 106) were collected and snap-frozen for storage at −80 °C before lysis. After lysing the cells and digesting the chromatin with 400 U each of DpnII and DdeI (NEB, R0543M and NEB, R0175) overnight, the DNA ends were labelled with biotinylated dATP (LifeTech, 19524016) using 50 U Klenow DNA polymerase (NEB, M0210). Blunt-end ligation was performed with 50 U T4 ligase (Life Technologies, 15224090) at 16 °C for 4 h, followed by overnight reverse crosslinking with 400 μg ml−1 proteinase K (Thermo Fisher Scientific, 25530031) at 65 °C. The DNA was purified using phenol–chloroform extraction and ethanol precipitation, and concentrated on a 30 kDa Amicon Ultra column (EMD Millipore, UFC5030BK). Biotin was removed from the unligated ends in 50 μl reactions using 50 U T4 DNA polymerase (NEB, M0203) per 5 mg of DNA. Following DNA sonication (Covaris S220 system) and solid-phase reversible immobilization bead-size fractionation to generate DNA fragments 100–300 bp in length. The DNA ends were repaired using 7.5 U T4 DNA polymerase, 25 U T4 polynucleotide kinase (NEB, M0201) and 2.5 U Klenow DNA polymerase (NEB, M0210). Libraries were enriched for ligation products by biotin pulldown with MyOne streptavidin C1 beads (Invitrogen, 65001). To prepare for sequencing, A-tailing was performed using 15 U Klenow DNA polymerase (3′–5′ exo-; NEB, M0212) and either TruSeq DNA LT kit indexed adaptors (Illumina, 20015964) or NEBNext multiplex oligos (NEB, E7780S) were employed. Libraries were amplified in PCR reactions for 5–7 cycles using a TruSeq DNA LT kit (Illumina, 15041757) and subjected to solid-phase reversible immobilization bead-size selection before sequencing on either an Illumina HiSeq 4000 or a NovaSeq 6000 system using the paired-end 50 bp or 100 bp modules. Two biological replicates were performed for each condition, with the exception of early mitotic exit populations in telophase or cytokinesis. Hi-C datasets have been deposited at National Center for Biotechnology Information (NCBI)'s Gene Expression Omnibus (GEO; GSE277875; SuperSeries, GSE278023). Chromatin accessibility was investigated using Omni-TAC–seq99,100 with some modifications. Prometaphase-arrested cells were collected by mitotic shake-off and adherent cells were dissociated using accutase 5 h after prometaphase release. Viable cells (50,000) were permeabilized in 50 µl cold Resuspension Buffer (10 mM Tris–HCl pH 7.4, 10 mM NaCl and 3 mM MgCl2) containing 0.1% NP-40 (MP Biomedicals, 0219859680), 0.1% Tween-20 and 0.01% digitonin (Promega, G9441) for 3 min on ice. The cells were isolated by centrifugation at 500g and 4 °C for 10 min, and the buffer was exchanged by washing once with the detergent-free Resuspension Buffer. Tagmentation was performed in permeabilized cells resuspended in 50 µl Tagmentation Buffer (Diagenode, C01019043) containing 100 nM adaptor-loaded Tn5 transposase (Diagenode, C01070012-30), 0.001% digitonin and 0.1% Tween-20 at 37 °C for 30 min with intermittent mixing. The reaction was stopped by the addition of Binding Buffer (Qiagen MinElute PCR Kit, 28004) and DNA was purified using a Qiagen MinElute PCR kit according to the manufacturer's protocol. Purified DNA was eluted in 21 μl Elution Buffer and stored at −20 °C for library preparation. To prepare ATAC–seq libraries for sequencing, custom barcoded primers based on a previous design101 were used in an initial five-cycle pre-amplification using NEBNext high fidelity PCR master mix (NEB, M0541). Following PCR amplification, libraries were purified using solid-phase reversible immobilization beads, with a sample-to-bead ratio of 1.0:1.5. ATAC–seq libraries were sequenced on an Illumina NextSeq 2000 machine using the 50 bp paired-end reagents. Two biological replicates were performed for each condition. ATAC–seq datasets have been deposited at the NCBI GEO accession GSE277731 (SuperSeries GSE278023). The CUT&RUN protocol was modified from Skene and Henikoff65 and applied to control or RanGAP1-depleted cell populations arrested in prometaphase or released for 5 h into G1. Approximately 5 × 106 cells per condition were harvested and lysed (0.1% Triton X-100, 20 mM HEPES-KOH pH 7.9, 10 mM KCl, 0.5 mM spermidine, 20% glycerol and Roche EDTA-free protease inhibitor; Sigma-Aldrich, 11873580001) for 10 min on ice. Due to the inclusion of prometaphase samples, the cells were centrifuged for 3 min at 600g and 4 °C after every wash or buffer exchange. The lysed cells were incubated for 5 min on ice with wash buffer (20 mM HEPES pH 7.5, 150 mM NaCl, 0.5 mM spermidine, 0.1% BSA and Roche protease inhibitor) containing 2 mM EDTA and washed once more with wash buffer before resuspension in 1 ml wash buffer for antibody binding. Primary antibodies to histone H3K4me3 (Abcam, ab8580; 1:100), H3K27ac (D5E4; Cell Signaling Technologies, 8173; 1:100), H3K27me3 (Cell Signaling Technologies; 1:100) or CTCF (Cell Signaling Technologies, 2899; 1:50) were bound overnight on a rotator at 4 °C. A control IgG (guinea pig anti-rabbit; ABIN101961; 1:100) sample was included for each replicate in control prometaphase and G1 cell populations. The samples were washed three times (2 min each) in wash buffer with rotation at 4 °C, followed by pAG-MNase binding (1/20 dilution; CUTANA pAG-MNase for ChIC/CUT&RUN workflows; EpiCypher, 15-1016) for 1 h at 4 °C with rotation. Following three washes (5 min each at 4 °C) with rotation, the samples were resuspended in 150 µl wash buffer and MNase was activated on wet ice by the addition of CaCl2 (final concentration of 2 mM) for 30 min. The reaction was quenched with 150 µl of 2×STOP buffer (200 mM NaCl, 20 mM EDTA, 4 mM EGTA, 50 µg ml−1 RNase A and 40 µg ml−1 glycogen) containing 1:1,000 of 10 ng ml−1 heterologous DNA (CUTANA E. coli spike-in DNA; EpiCypher, 18-1401) and incubated for 20 min at 37 °C. After centrifugation at 16,000g for 5 min, the supernatant was incubated at 65 °C for 10 min with 0.1% SDS, 0.2 mg ml−1 proteinase K and 50 µg ml−1 RNase A. Extraction of DNA using 1:1 phenol:chloroform extraction and ethanol precipitation was followed by AMpure XP bead (Beckman Coulter, A63881) size selection to remove fragments larger than about 700 bp. Libraries were prepared for sequencing using the NEBNext Ultra II end repair/dA-tailing module (NEB, E7546) and NEBNext multiplex oligos for Illumina (NEB, E7600 and E7780). The libraries were amplified in PCR reactions for 14–18 cycles and subjected to AMpure XP bead-size selection before sequencing on an Illumina NextSeq 2000 system using the paired-end 50 bp module. Two biological replicates were performed for each condition. The CUT&RUN datasets have been deposited at the NCBI GEO (GSE308844; SuperSeries, GSE278023). DLD-1 cells were arrested at prometaphase, as described earlier, and released to progress through mitosis over 5 h, with samples collected every hour. The cells were labelled with 1 mM 4sU in 1-h non-overlapping intervals comprising the final hour of culture for each collection time point. Unlabelled controls were included at prometaphase arrest and at the end of the 5-h time course, and each experiment was performed in triplicate. At the end of each time point, the cells were collected and snap-frozen in liquid nitrogen, and the cell pellets were lysed in TRIzol reagent before total RNA extraction using a Zymo RNA Clean & Concentrate kit. Before RNA extraction, six ERCC spike-ins were added to the lysed pellet at a concentration relative to the number of cells harvested: three ERCCs labelled with 10% 4sU and three unlabelled ERCCs, as described previously102. SLAM–seq libraries were prepared as previously described62. Briefly, ribosomal RNA was depleted as previously described103, the RNA was treated with 10 mM iodoacetamide to alkylate 4sU bases and libraries were prepared using a NEBNext Ultra II directional RNA library prep kit according to the manufacturer's recommendations. Library quality was confirmed using Agilent TAPE station reagents and libraries were sequenced on an Illumina NextSeq 2000 platform (2 × 100 paired-end reads) to an average depth of 67 × 106 reads per library. Before cell-cycle synchronization, DLD-1 RanGAP1–AID and Nup93–AID cells were labelled by culturing them for five days in heavy (L-arginine-13C6, 15N4 hydrochloride and L-lysine:2HCl 13C6 15N2; Sigma-Aldrich, 608033 and Cambridge Isotope, CNLM-291-H, respectively) or light (L-arginine hydrochloride and L-lysine; Sigma-Aldrich, A6969 and L5501, respectively) DMEM for SILAC medium (Thermo Fisher Scientific, A33822) with dialysed fetal bovine serum (Sigma-Aldrich, F0392). SILAC was maintained throughout the cell synchronization experiment and nuclei were isolated for LC–MS. For each cell line, one replicate experiment was performed using light control and heavy IAA-treated samples, and labelling was then reversed for the second biological replicate. Nuclei were isolated by manual disruption based on a previous protocol105 for mass spectrometry analysis. Early G1 labelled cells were collected 5 h after prometaphase release and washed with PBS. The cells (10 × 106) were carefully resuspended with a broad pipette tip in 1 ml hypotonic buffer (10 mM HEPES pH 7.9, 1.5 mM MgCl2 and 10 mM KCl) containing 0.1 mM phenylmethyl sulfonyl fluoride, 0.5 mM dithiothreitol (Thermo Fisher Scientific, BP172-25) and 1×protease inhibitor cocktail (Thermo Fisher Scientific, 78440), and incubated on ice for 25 min. The cells were then ruptured by douncing 40× in a pre-chilled homogenizer (tight pestle (B)) and nuclei were isolated by centrifugation for 5 min at 1,500g and 4 °C, and resuspended in a 10 mM Tris buffer pH 7.4 containing 2 mM MgCl2. The samples were adjusted to 1×Laemmli buffer (60 mM Tris pH 6.8, 10% glycerol, 2% SDS and 100 mM dithiothreitol) and heated to >85 °C for 10 min before gel electrophoresis. Heavy and light samples were mixed at a 1:1 ratio of total protein based on quantification of stained gels (GelCode Blue Safe Protein Stain; Thermo Fisher Scientific, 24594). The final protein gels were stained with GelCode blue safe protein stain and bands were excised for LC–MS sample processing. Overnight trypsin digestion was performed at 37 °C. After protein digestion and drying in a SpeedVac, the samples were reconstituted in 25 µl MS solvent (5% acetonitrile and 0.1% formic acid) and 3.8 µl was injected into a Fusion Lumos Orbitrap mass spectrometer in OTOT mode with a 90 min gradient. Peptides were searched against the SwissProt human database in Maxquant and proteins were subject to a two-peptide cutoff. Two replicates were performed for each condition, reversing the SILAC labelling, and >3,000 proteins were detected in each sample at 1% false discovery rate (summarized in Supplementary Tables 4 and 5). Fold changes and Benjamini–Hochberg-corrected P values comparing control and depleted cell conditions were determined using the Q+ analysis by Scaffold. Full processed datasets are in Supplementary Tables 4 and 5, and raw data have been deposited to the ProteomeXchange Consortium via the PRIDE106 partner repository with the dataset identifier PXD056346. Publicly available CTCF chromatin immunoprecipitation with sequencing (ChIP–seq) coverage from untreated DLD-1 cells and the corresponding bigWig (GSM4238559) was used to define cCREs and throughout the paper for visualization unless otherwise indicated. Variant 1 of the MA0139 CTCF motif annotation from the Jaspar database107 was used to define convergent extrusion loops. Publicly available RNA-sequencing data for the DLD-1-RanGAP1 cell line (GSE132363: GSM3860900, GSM3860901 and GSM3860902) were used to define active genes for visualization and ATAC–seq analysis. Raw data were processed using the nf-core/rnaseq pipeline (v.3.15.0; https://github.com/nf-core/rnaseq )108. Genes with fragments per kilobase of transcript per million mapped reads (FPKM) > 0 in all three replicates and FPKM > 1 in the pooled dataset were defined as active. This information is summarized in Supplementary Table 3. We used a combination of publicly available CTCF ChIP–seq coverage for DLD-1 together with the control G1 histone H3K4me3 and H3K27ac CUT&RUN and ATAC–seq data generated in this study to define CREs according to the procedure defined by ENCODE109. Briefly, we annotated the list of cell line-independent DNase hypersensitive sites from ENCODE v2 with z-score-transformed ATAC–seq, H3K4me3 and H3K27ac signal, and used the distance to the nearest transcription start site (TSS) to assign DHSs to seven CRE groups: PLS (22,555 promoter-like: open, H3K4me3 enriched, within 200 bp of the nearest TSS), pELS (28,863 combined proximal and near enhancer-like: open, H3K27ac enriched and within 2 kb of the nearest TSS), dELS (43,807 distal enhancer-like: open, H3K27ac enriched and at least 2 kb away from the nearest TSS), H3K4me3 (1,417 open H3K4me3: open, H3K4me3 enriched and at least 200 bp away from the nearest TSS), CTCF (9,911 open, CTCF enriched and not enriched in any other marks) and finally, open (43,316 open, not enriched in any marks). Where indicated, binned genomic loci were assigned one cCRE based on the following hierarchy: PLS > pELS > dELS > CTCF > just open > not open. The fold enrichment of binned cCREs in specific genomic regions was determined using the ChromHMM ‘OverlapEnrichment' function110, which normalizes the fraction of bases in a particular state (that is, PLS) found at a subset of genomic loci (that is, MCDs) to the fraction of those loci region genome-wide. Stackups demonstrate the behaviour of a given signal (for example, H3K4me3 CUT&RUN) at a set of genomic loci, typically centred at those loci and presented as a heat map, where each row represents the signal around the individual genomic locus. We took advantage of the Python application programming interface (API)111 built around UCSC BBI library112 to extract a signal stored in a bigWig or bigBed files given a set of same-sized genomic intervals using the ‘stackup' function. We ensured our intervals had the identical size by centering on the loci of interest and providing fixed-length upstream and downstream flanks. For every stackup, we also generated a summary signal profile by averaging the stackup across all rows for every column. We used 100 kb for upstream and downstream flank sizes, and aggregated the signal into 100 bins for every row of the stackup, logarithmic colour scales were used throughout the stack-ups with the exception of EV1 profiles and bigBed-derived signals—coverages of loop anchors, MCD anchors and so on. Hi-C libraries were processed using the distiller-nf pipeline113 (v.0.3.4): paired-end reads were mapped to hg38 human reference genome using bwa mem114 in a single-sided fashion (-SP); read alignments were parsed and classified into pairwise interactions, or pairs, by parse from the pairtools package115 (v.1.0.2) and the additional ‘–walk-policy all' option was used to rescue multiway interactions (walks); after the removal of duplicates, uniquely mapped and rescued pairs were further filtered according to their alignment quality (MAPQ > 30) and subsequently aggregated into binned contact matrices in the cooler format116 at a resolution of 1, 2, 5, 10, 25, 50, 100, 250, 500 and 1,000 kb; contact matrices were normalized using the iterative correction normalization77 with the default parameters (for example, the first two diagonals were excluded from balancing at each resolution to avoid short-range ligation artifacts); bins with extreme genomic coverage, as detected by the MADmax (maximum allowed median absolute deviation) filter72, were masked and excluded from the analysis. Summary mapping statistics, as produced by the pairtools module from MultiQC117, are in Supplementary Table 1 and Supplementary Fig. We used insulation tracks and cis Eigvectors to assess the reproducibility of the replicates, which were pooled together as depicted in Supplementary Fig. To extract Hi-C features relevant for our analyses we used the cooltools package55 (v.0.7.0); specifically, we leveraged the cooltools Python API for scripting our analyses in the form of Jupyter Notebooks118. We used arms of human autosomal chromosomes (Chr 1–22) as a genome partitioning for the analyses (parameter ‘view_df') and default parameters in the API function calls unless specified otherwise. In the following sections, we provide a brief description of specific functions that were used to extract each feature. Frequency of interactions as a function of genomic separation (scaling plots, P(s)) was calculated using balanced Hi-C data binned at 1 kb using the ‘expected_cis' function with smoothing in log-space enabled, data for chromosome arms were aggregated to generate an average genome-wide scaling. Most downstream analyses require Hi-C matrices to be ‘flattened', that is, normalized to the decay of interaction frequency with genomic distance. We use the expected_cisfunction to calculate such an ‘expected'. In case of trans or interchromosomal data, matrices were normalized to average levels of interchromosomal interactions, calculated using the ‘expected_trans' function. Results of these functions were passed to the downstream analyses when applicable. Diamond insulation score120 was calculated using the ‘insulation' function at 10 kb resolution and 100 kb averaging window size (size of the insulation diamond). Eigenvector analysis77 was performed separately for each chromosome arm using the ‘eigs_cis' function at 10, 25, 50 and 250 kb, where gene density was used to ‘phase' Eigenvectors, that is, Eigenvector tracks were ‘flipped' if they anticorrelated the gene density track. The EV1 Eigenvectors (ones with the highest Eigenvalues, which typically correspond to Hi-C compartments) of select samples were saved as bigWig files to use in stackup analysis and visualization. To evaluate how different classes of genomic loci interact with each other in 3D, that is, saddle plot analysis77, we used the ‘saddle' function. First, assignment of the classes to genomic loci (for example, compartment status, CRE status and so on) was done on a bin level and passed as a ‘track' parameter to the function. Second, the average level of interactions was calculated for each possible combination of classes from the flattened contact map (observed/expected). Finally, a class-pairwise average interaction matrix was constructed. Assigning different classes to genomic bins was done in a specific manner. For published DLD-1 IPGs80, we used the chromatin state assignments directly at 50 kb resolution after merging B2/3 and B4 heterochromatin classes into ‘B'. Candidate CREs were hierarchically assigned to 10 and 25 kb genomic bins in the following order: PLS, pELS, dELS, H3K4me3-open, CTCF and finally, open sites (for example, if a given 10 kb bin contained a pELS element, then the entire bin was assigned pELS status, regardless of other elements present in that bin). ‘Continuous' EV1 tracks were digitized into 38 quantiles after excluding 2.5% of the extreme EV1 values from each end of the spectrum. We estimated the strength of the A compartment as an enrichment of AA interactions over AB: AA / ((AB + BA) / 2), where AA is an average of observed–expected interactions between the EV1 quantiles with the 20% strongest A-compartment identity, and AB(=BA) is an average of the observed–expected interactions between the quantiles with 20% strongest A and B identities. Similarly for the B compartment, strength was estimated as: BB / ((AB + BA) / 2). The ‘pileup' function was used to explore the local interaction pattern of a set of two-dimensional genomic features (defined by a pair of genomic locations, for example, in our case an all-by-all grid of MCDs, with their various subsets and a set of called loops). Briefly, local contact maps (normalized to the expected) centred on a given feature and with a fixed flank of 100 kb, a ‘snippet', were extracted as a stack and then averaged all together or in groups of features that meet a certain criterion, for example, in our case subgroups of the microcompartment grid by genomic distance, subgroups of the grid overlapping other features like extrusion dots or domains and so on. Cis-chromosomal pileups were performed on 10 kb contact maps, whereas trans-chromosomal pileups were performed on 25 kb maps, unless specified otherwise. Local interaction patterns, average pileups, were also used to quantify average strength of a given genomic feature. Specifically, we used a 50 × 50 kb window in the centre together with four 60 × 60 kb corners (as a periphery signal) for the grid of MCDs in cis, a 25 × 25 kb centre (single pixel in the middle) and four 50 × 50 kb corners (periphery) for the grid of MCDs in trans, and a 30 × 30 kb centre and four 70 × 70 kb corners (periphery) for loop strength calculations. To detect a reference set of extrusion loops for our analyses, we pooled control Hi-C data for 5 and 10 h at 10 kb resolution (given their similarity, as demonstrated in Supplementary Fig. 1c,d) to achieve higher sequencing depth and then used the ‘dots' function to call significantly enriched interactions. We used ‘cluster_filtering=False' and otherwise the default parameters to apply more stringent singleton filtering afterwards. Detected interactions were further filtered to ensure they were compatible with the convergent CTCF–CTCF interaction. The resulting list of 18,615 interactions/loops were also used to define extrusion domains, intuitively, it is the outermost loop from a subgroup of nested loops that defines a domain. Specifically, loops were clustered by their anchor 1 to group those situated on the same extrusion line, then most upstream anchor 1 of the cluster was used together with the most downstream anchor 2 of the cluster to define such fully inclusive intervals. Nested and significantly overlapping (when the overlap between intervals was >70% of either of the intervals) intervals from the resulting list were merged, yielding the final list of 3,401 domains. We defined MCDs as the genomic loci/anchors that give rise to the strongly interacting off-diagonal rectangular domains that are clearly visible on the 5 h RanGAP1-depletion contact map and set out to detect them using screening procedure akin to the detection of dots: detect enriched pixels that stand out relative to the local background and after grouping them by proximity, detect those groups that continue interacting with others across distances, that is, those that continue “checkering”. Such checkering anchors are the target feature that we aimed to detect in the first place, that is the MCDs. Next, we described specific steps involved in MCD detection. Variable size and shape of observed MCD–MCD interaction domains dictated the choice of convolution kernels that “describe” the local vicinity for a given group of pixels (Supplementary Fig. 2b): the vertical (V) kernel is meant to facilitate detection of horizontally elongated domains, whereas the horizontal (H)–vertically elongated domains, where a square-shaped group of pixels M (middle)—corresponds to the part of the domain being tested. Convolutional kernels M, V and H were swept across distance decay-corrected contact map (observed/expected) at 10 kb resolution, up to 30 MB for computational efficiency, and enriched pixels were selected with a simple thresholding approach: pixels for which the M was at least twice as bright as either the V or H. The density-based clustering approach ‘OPTICS' from the sklearn package121 (v.1.4.1; Supplementary Fig. 2c) was used to filter out singletons and small groups of enriched pixels, and preserve larger more robust groups of enriched pixels (‘min_samples = 5' and ‘max_eps = 33 kb' parameters were used). Pixels that remained after the clustering step were used to calculate the coverage of enriched pixels (or anchor valency; Supplementary Fig. 2d), and finally, we applied the 1D peak detection function ‘find_peaks' of the scipy package122 to the coverage track to detect prominent peaks (Supplementary Fig. 2e), which were treated as the final MCD anchors. Each anchor was characterized by its footprint interval and a summit—we used footprints for most of the downstream analysis, except for the average pileups and stack-ups, where we used the summits as an MCD genomic coordinate instead of the centre of the footprint. The exact details of implementation are available on GitHub (https://github.com/dekkerlab/inherited-folding-programs.git). We detected a total of 2,105 MCDs in the 5 h RanGAP1 depletion sample and 1,791 MCDs in the RanGAP1-depleted sample at cytokinesis, 68% of which were contained in the former sample. We used three subsets of MCDs for downstream analysis: 565 cytokinesis MCDs that did not overlap the 5 h ones (‘Cyto-specifc'), 1,218 cytokinesis MCDs that were also present in G1 (‘Cyto at G1' or ‘Cyto + G1') and 876 G1 MCDs that were not present in the cytokinesis sample (‘G1-specific'). We also identified 4,623 permissive MCDs by pooling the 5 h and 10 h G1 RanGAP1-depleted Hi-C samples—we refer to this set as Mega or permissive MCDs. This set generally demonstrated the same trends, albeit with the addition of weaker MCDs and potentially some erroneous MCD calls, and was only used for the spectral clustering characterization (details in the next section). Motivated by the recent progress in the application of spectral clustering15,56 to the analysis of Hi-C data, we set out to apply this methodology specifically to the RanGAP1-depletion data. Such an analysis complements our enrichment-based MCD screening procedure, enabling us to test whether MCDs indeed demonstrate similar interaction patterns genome-wide. To date, we performed spectral clustering of combined 5 h and 10 h G1 RanGAP1-depletion Hi-C data, following a previously described procedure15, and here we provide a brief description of our implementation and highlight key differences. We start by combining RanGAP1 depletion G1 5 h and 10 h Hi-C samples at 10 kb resolution to reduce data sparsity. The combined contact frequency map is flattened by normalizing it to the expected: average contact frequency decay with distance for intra- and interchromosome arm regions and pairwise interchromosomal interaction levels for trans data55. Additional ‘bad bins' corresponding to translocated loci were manually identified (Supplementary Table 6) and masked for further analysis to ensure leading Eigenvectors of the contact map explain compartmentalization patterns instead of the discordance between reference assembly and the state of the genome in DLD-1 cell line15. The pre-processed contact map was further balanced using an iterative correction procedure77 and finally eigendecomposed. This procedure differs from the published approach15 by performing Eigendecomposition on a full genome-wide contact map that includes both cis and trans interactions. The primary motivation for this is to achieve Eigendecomposition at a high resolution (10 kb) relevant for MCD detection. Dense matrix Eigendecomposition at such resolution is challenging, as it would require approximately 1 TB of local memory. To overcome this, we used modified version of the publicly available prototype of sparse Hi-C contact map Eigendecomposition (https://github.com/open2c/open2c_vignettes/blob/main/sparse_eigendecomp.ipynb). This prototype leverages ARPACK123 implementation of sparse matrix Eigendecomposition, available in SciPy122 as the scipy.sparse.linalg.eigsh function. The sparse contact map was fed into the function as a scipy.sparse.linalg.LinearOperator vector function that defines the matrix-vector product between our sparse contact map and said vector. Performing Eigendecomposition on combined cis and trans data presents a challenge for interpretability due to the variation in chromosome sizes, stark contrast in sparsity between cis- and trans-contact frequencies and potential differences in cis and trans compartmentalization patterns. However, in this particular case of RanGAP1-depleted Hi-C data, we did not observe obvious effects of chromosome size variation until Eigenvector 10 (sorted by importance, that is, absolute Eigenvalue) and confirmed that the leading non-trivial Eigenvector is highly correlated to a concatenated list of chromosome-wide EV1s calculated using the cooltools.eigs_cis procedure55 at 10 kb (Extended Data Fig. Thus, we continued our downstream analysis with the 2–9 most significant Eigenvectors (Extended Data Fig. 4b), where E1 was excluded for clustering purposes as it is a trivial ‘flat' Eigenvector corresponding to the sum of rows/columns of the contact map. Eigenvectors E2–E9 were unit-normalized, weighted by the square root of the corresponding Eigenvalue and subjected to K-means clustering as described previously15. The number of clusters k = 8 was chosen as it provided optimal overlap between the permissive set of MCDs and a given MCD-associated cluster, that is, >60% of the cluster is covered with MCDs and >60% of MCDs are covered with intervals from that cluster (holds true by counting both overlapping nucleotides and number of intervals). Imposing cluster-informed grouping and sorting onto epigenetic tracks (Extended Data Fig. 4c) reveals strong enrichment for active marks H3K27ac/H3K4me3 in the MCD-associated cluster C2. There is one apparent deviation from this trend we have described so far and it is the cluster most closely associated with the A1 IPG identified in wild-type DLD-1 cells80: this cluster harbours a subset (approximately 20%) of ‘stubborn' MCDs that ‘refuse' to cluster with the ‘mainstream' MCD-associated cluster. Overall, we were able to capture the main trends and characteristics of MCDs as defined using our enrichment-based method; however, further development into clustering approaches might enable a more precise and comprehensive MCD detection. Batch pre-processing of raw ATAC–seq data from control and auxin-treated RanGAP1–AID cells synchronized in prometaphase or released for 5 h to early G1 was performed using the nf-core/atacseq pipeline (v.2.1.0; https://github.com/nf-core/atacseq )108. Adaptor-trimmed paired-end reads were mapped to the hg38 reference genome using bwa mem and filtered according to the standard pipeline for mapping quality, mitochondrial reads, PCR duplicates and read length (<2,000 bp). Similarity between replicates was confirmed using DESeq2 (ref. The summary mapping statistics of ATAC–seq samples generated in this study are in Supplementary Table 2. Fragment-length distributions were determined from binary alignment map (BAM) files using deepTools125. Read ends were derived from the filtered alignments (BAM files) and modified to account for tn5 by shifting ± reads by +4/−5 bp for use in all downstream applications. ATAC–seq coverage tracks were generated from shifted read ends using BEDtools126 and scaled to 1 × 106 mapped reads. We used MACS3127 to find ATAC–seq peaks of accessibility in the pooled and single replicate datasets using default parameters with a shift/extend of −75/+150. Peaks called in pooled datasets were only considered true when they overlapped a peak in each of the two constituent replicates by at least 50%. To compare these pooled peaks between conditions, the union set was merged using bioframe128. Batch pre-processing of raw CUT&RUN data from control and auxin-treated RanGAP1–AID cells synchronized in prometaphase or released for 5 h to early G1 was performed using the nf-core/atacseq pipeline (v.2.1.0; https://github.com/nf-core/atacseq )108. Adaptor-trimmed paired-end reads were mapped to the hg38 reference genome using bwa mem and filtered according to the standard pipeline for mapping quality, mitochondrial reads, PCR duplicates and read length (<2,000 bp). Similarity between replicates was confirmed using DESeq2 (ref. The summary mapping statistics of CUT&RUN samples generated in this study are in See Supplementary Table 2. Read ends were derived from the filtered alignments (BAM files) and extended ±25 bp for most visualizations. Coverage tracks were generated from read ends using BEDtools126 and scaled to 1 × 106 mapped reads. Peaks of enriched H3K27ac signal were called from paired-end bedgraph files using SEACR129 with normalization to cell cycle-matched IgG controls. Bookmarked peaks are considered the subset of control G1 peaks that overlap a peak called in the control prometaphase sample. SLAM–seq reads were aligned to Ensembl GRCh38v95 with Hisat-3N130, specifying T>C conversions. T>C conversions in sequenced reads were obtained by filtering Hisat-3N-aligned BAM files for reads with a Yf:i tag (corresponding to the number of T>C converted nucleotides) greater than one and thereafter annotated as nascent. Conversion efficiency was confirmed in ERCC spike-ins by obtaining the proportion of reads with T>C substitutions in 4sU-labelled versus unlabelled ERCC species. Global conversion efficiency was estimated by calculating the percentage of total RNA reads harbouring T>C substitutions in labelled and unlabelled samples. GRAND-SLAM was used to estimate the RNA NTR of each gene63 and genes with at least 100 reads were retained for further analyses. Fisher's exact tests were used to test for significant associations between genes expressed in different stages of mitotic exit and genes found in MCDs (alternative = greater); the resulting P values were corrected using the Benjamini–Hochberg method. No statistical method was used to pre-determine sample size in any experiments. Two replicates for each condition were performed for most experiments (Hi-C, ATAC–seq, CUT&RUN and SILAC LC–MS). For SLAM–seq, three independent experiments were performed. Insulation tracks and cis Eigvectors were used to assess the reproducibility of the Hi-C replicates (Supplementary Fig. 2c,d), which were pooled for most of the presented plots and analyses (Supplementary Fig. ATAC–seq similarity between replicates was confirmed using DESeq2 (ref. All conclusions were verified in individual replicates. Unless otherwise indicated, all immunofluorescence (Figs. 1c,e, 6c, 8e and Extended Data Figs. 2a,c,e–g, 8c, 9d) and western blot images (Figs. 1a,b, 9b) are representative of at least two independent replicates, and flow cytometry panels are representative of at least three independent experiments (Fig. 1g) and images (Extended Data Fig. 2g) are representative of two independent experiments. No data were excluded from these analyses, and randomization or blinding of experiments was not applicable. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Any requests regarding cell lines and plasmids should be directed to Mary Dasso's laboratory. The datasets generated in this publication have been deposited in the NCBI GEO as a SuperSeries accessible through the accession number GSE278023, consisting of GSE277875 (Hi-C), GSE277731 (ATAC–seq), GSE308844 (CUT&RUN) and GSE309609 (SLAM–seq). The following published datasets were used in this study (Supplementary Table 3): GSE132363, GSE178593 and GSE214012. All other data supporting the findings of this study are available from the corresponding author on reasonable request. Open2C scripts and notebooks used in this study are publicly available in GitHub: https://github.com/open2c and https://github.com/dekkerlab/inherited-folding-programs. No other customized codes were developed for this study. & van Steensel, B. Genome architecture: domain organization of interphase chromosomes. & Mirny, L. The 3D genome as moderator of chromosomal communication. & Cavalli, G. Understanding 3D genome organization by multidisciplinary methods. Mirny, L. A., Imakaev, M. & Abdennur, N. Two major mechanisms of chromosome organization. Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Hildebrand, E. M. & Dekker, J. Mechanisms and functions of chromosome compartmentalization. Harris, H. L. et al. Chromatin alternates between A and B compartments at kilobase scale for subgenic organization. Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. SPIN reveals genome-wide landscape of nuclear compartmentalization. A. et al. Organization of chromatin by intrinsic and regulated phase separation. Dey, A., Chitsaz, F., Abbasi, A., Misteli, T. & Ozato, K. The double bromodomain protein Brd4 binds to acetylated chromatin during interphase and mitosis. Selective recognition of methylated lysine 9 on histone H3 by the HP1 chromo domain. Keenen, M. M. et al. HP1 proteins compact DNA into mechanically and positionally stable phase separated domains. Diverse silent chromatin states modulate genome compartmentalization and loop extrusion barriers. Valton, A.-L. et al. A cohesin traffic pattern genetically linked to gene regulation. Galitsyna, A. et al. Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation. Rinzema, N. J. et al. Building regulatory landscapes reveals that an enhancer can recruit cohesin to create contact domains, engage CTCF sites and activate distant genes. de Wit, E. et al. CTCF binding polarity determines chromatin looping. Guo, Y. et al. CRISPR inversion of CTCF sites alters genome topology and enhancer/promoter function. Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Busslinger, G. A. et al. Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl. Fudenberg, G., Abdennur, N., Imakaev, M., Goloborodko, A. Emerging evidence of chromosome folding by loop extrusion. Naumova, N. et al. Organization of the mitotic chromosome. Gibcus, J. H. et al. A pathway for mitotic chromosome formation. Samejima, K. et al. Rules of engagement for condensins and cohesins guide mitotic chromosome formation. Two ways to fold the genome during the cell cycle: insights obtained with chromosome conformation capture. Hildebrand, E. M. et al. Mitotic chromosomes are self-entangled and disentangle through a topoisomerase-II-dependent two-stage exit from mitosis. Oomen, M. E., Hansen, A. S., Liu, Y., Darzacq, X. & Dekker, J. CTCF sites display cell cycle-dependent dynamics in factor binding and nucleosome positioning. Sekiya, T., Murano, K., Kato, K., Kawaguchi, A. & Nagata, K. Mitotic phosphorylation of CCCTC-binding factor (CTCF) reduces its DNA binding activity. Martínez-Balbás, M. A., Dey, A., Rabindran, S. K., Ozato, K. & Wu, C. Displacement of sequence-specific transcription factors from mitotic chromatin. Parsons, G. G. & Spencer, C. A. Mitotic repression of RNA polymerase II transcription is accompanied by release of transcription elongation complexes. Samejima, I. et al. Mapping the invisible chromatin transactions of prophase chromosome remodeling. Ito, K. & Zaret, K. S. Maintaining transcriptional specificity through mitosis. Hsiung, C. C.-S. et al. Genome accessibility is widely preserved and locally modulated during mitosis. Chervova, A., Festuccia, N., Altamirano-Pacheco, L., Dubois, A. & Navarro, P. A gene subset requires CTCF bookmarking during the fast post-mitotic reactivation of mouse ES cells. Oomen, M. E. et al. Mitotic chromosomes harbor cell type- and species-specific structural features within a universal loop array conformation. Zhang, H. et al. Chromatin structure dynamics during the mitosis-to-G1 phase transition. Abramo, K. et al. A chromosome folding intermediate at the condensin-to-cohesin transition during telophase. Pelham-Webb, B. et al. H3K27ac bookmarking promotes rapid post-mitotic activation of the pluripotent stem cell program without impacting 3D chromatin reorganization. Dynamic regulation of histone modifications and long-range chromosomal interactions during postmitotic transcriptional reactivation. Brunner, A. et al. Quantitative imaging of loop extruders rebuilding interphase genome architecture after mitosis. Sachdev, R., Sieverding, C., Flötenmeyer, M. & Antonin, W. The C-terminal domain of Nup93 is essential for assembly of the structural backbone of nuclear pore complexes. The nuclear pore complex consists of two independent scaffolds. Bischoff, F. R., Klebe, C., Kretschmer, J., Wittinghofer, A. & Ponstingl, H. RanGAP1 induces GTPase activity of nuclear Ras-related Ran. Prasanth, K. V., Sacco-Bubulya, P. A., Prasanth, S. G. & Spector, D. L. Sequential entry of components of the gene expression machinery into daughter nuclei. Leung, A. K. L. et al. Quantitative kinetic analysis of nucleolar breakdown and reassembly during mitosis in live human cells. & Gibcus, J. H. Hi-C 3.0: improved protocol for genome-wide chromosome conformation capture. Systematic evaluation of chromosome conformation capture assays. Goel, V. Y., Huseyin, M. K. & Hansen, A. S. Region Capture Micro-C reveals coalescence of enhancers and promoters into nested microcompartments. Open2c, Abdennur, N. & Abraham, S. Cooltools: enabling high-resolution Hi-C analysis in Python. Reimonn, T. et al. Joint decomposition of Hi-C maps reveals salient features of genome architecture across tissues and development. Friman, E. T., Flyamer, I. M., Marenduzzo, D., Boyle, S. & Bickmore, W. A. Ultra-long-range interactions between active regulatory elements. Gasnereau, I. et al. Flow cytometry to sort mammalian cells in cytokinesis. Goel, V. Y., et al. Dynamics of microcompartment formation at the mitosis-to-G1 transition. Palozola, K. C. et al. Mitotic transcription and waves of gene reactivation during mitotic exit. Hsiung, C. C.-S. et al. A hyperactive transcriptional state marks genome reactivation at the mitosis-G1 transition. Herzog, V. A. et al. Thiol-linked alkylation of RNA to assess expression dynamics. Jürges, C., Dölken, L. & Erhard, F. Dissecting newly transcribed and old RNA using GRAND-SLAM. Grandi, F. C., Modi, H., Kampman, L. & Corces, M. R. Chromatin accessibility profiling by ATAC–seq. & Henikoff, S. An efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites. Behera, V. et al. Interrogating histone acetylation and BRD4 as mitotic bookmarks of transcription. The oncogenic BRD4-NUT chromatin regulator drives aberrant transcription within large topological domains. Sabari, B. R. et al. Coactivator condensation at super-enhancers links phase separation and gene control. Daneshvar, K. et al. lncRNA DIGIT and BRD3 protein form phase-separated condensates to regulate endoderm differentiation. Rosencrance, C. D. et al. Chromatin hyperacetylation impacts chromosome folding by forming a nuclear subcompartment. Gassler, J. et al. A mechanism of cohesin-dependent loop extrusion organizes zygotic genome architecture. Schwarzer, W. et al. Two independent modes of chromatin organization revealed by cohesin removal. Polovnikov, K. E. et al. Crumpled polymer with loops recapitulates key features of chromosome organization. The cohesin release factor WAPL restricts chromatin loop extension. Nuebler, J., Fudenberg, G., Imakaev, M., Abdennur, N. & Mirny, L. A. Chromatin organization by an interplay of loop extrusion and compartmental segregation. Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. Imakaev, M. et al. Iterative correction of Hi-C data reveals hallmarks of chromosome organization. Revealing Hi-C subcompartments by imputing inter-chromosomal chromatin interactions. Chen, Y. et al. Mapping 3D genome organization relative to nuclear compartments using TSA-Seq as a cytological ruler. Scelfo, A. et al. Tunable DNMT1 degradation reveals DNMT1/DNMT3B synergy in DNA methylation and genome organization. Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics. Ashburner, M. et al. Gene ontology: tool for the unification of biology. Gene Ontology Consortium et al. The Gene Ontology knowledgebase in 2023. Xie, L. et al. BRD2 compartmentalizes the accessible genome. Zhao, H. et al. Genome folding principles uncovered in condensin-depleted mitotic chromosomes. & Belmont, A. S. Genome organization around nuclear speckles. & Kaufman, P. D. Close to the edge: heterochromatin at the nucleolar and nuclear peripheries. Transcription shapes 3D chromatin organization by interacting with loop extrusion. Lam, J. C. et al. YY1-controlled regulatory connectivity and transcription are influenced by the cell cycle. Calderon, L. et al. Cohesin-dependence of neuronal gene expression relates to chromatin loop length. Thiecke, M. J. et al. Cohesin-dependent and -independent mechanisms mediate chromosomal contacts between promoters and enhancers. Hsieh, T.-H. S. et al. Enhancer-promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1. Liu, N. Q. et al. WAPL maintains a cohesin loading cycle to preserve cell-type-specific distal gene regulation. & Dasso, M. Analysis of nucleoporin function using inducible degron techniques. Aksenova, V. et al. Nucleoporin TPR is an integral component of the TREX-2 mRNA export pathway. Chu, V. T. et al. Increasing the efficiency of homology-directed repair for CRISPR-Cas9-induced precise gene editing in mammalian cells. Bolte, S. & Cordelières, F. P. A guided tour into subcellular colocalization analysis in light microscopy. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Corces, M. R. et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. & Greenleaf, W. J. ATAC-seq: a method for assaying chromatin accessibility genome-wide. Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. & Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Schwalb, B. et al. TT-seq maps the human transient transcriptome. & Zamore, P. D. Strand-specific libraries for high throughput RNA sequencing (RNA-Seq) prepared without poly(A) selection. Ong, S.-E. & Mann, M. A practical recipe for stable isotope labeling by amino acids in cell culture (SILAC). Herrmann, C., Avgousti, D. C. & Weitzman, M. D. Differential salt fractionation of nuclei to analyze chromatin-associated proteins from cultured mammalian cells. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. Rauluseviciute, I. et al. JASPAR 2024: 20th anniversary of the open-access database of transcription factor binding profiles. The nf-core framework for community-curated bioinformatics pipelines. ENCODE Project Consortium et al. Expanded encyclopaedias of DNA elements in the human and mouse genomes. & Kellis, M. Chromatin-state discovery and genome annotation with ChromHMM. Kent, W. J., Zweig, A. S., Barber, G., Hinrichs, A. S. & Karolchik, D. BigWig and BigBed: enabling browsing of large distributed datasets. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Open2C et al. Pairtools: from sequencing data to chromosome contacts. Abdennur, N. & Mirny, L. A. Cooler: scalable storage for Hi-C data and other genomically labeled arrays. Ewels, P., Magnusson, M., Lundin, S. & Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Kluyver, T. et al. Jupyter Notebooks—a publishing format for reproducible computational workflows. International Conference on Electronic Publishing (eds Loizide, F. & Schmidt, B.) Crane, E. et al. Condensin-driven remodelling of X chromosome topology during dosage compensation. Pedregosa, F. et al. Scikit-learn: Machine Learning in Python. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Lehoucq, R. B., Sorensen, D. C. & Yang, C. ARPACK Users' Guide. (Society for Industrial and Applied Mathematics, 1998). Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Zhang, Y. et al. Model-based analysis of ChIP–Seq (MACS). Open2C et al. Bioframe: operations on genomic intervals in Pandas dataframes. Meers, M. P., Tenenbaum, D. & Henikoff, S. Peak calling by Sparse Enrichment Analysis for CUT&RUN chromatin profiling. Zhang, Y., Park, C., Bennett, C., Thornton, M. & Kim, D. Rapid and accurate alignment of nucleotide conversion sequencing reads with HISAT-3N. We thank the members of the Dekker laboratory, the Open2C community (especially N. Abdennur), for discussion on experiments and data analysis, and C. Navarro for discussion and help revising and editing the paper. We thank the UMass Electron Microscopy Core (K. Reddig and G. Hendricks, supported award numbers S10OD025113-01 and S10OD021580 from the National Center For Research Resources), the UMass Proteomics Core, the UMass FACS core and the UMass Deep Sequencing Core (E. Kittler, M. Zapp and D. Wilmot). We thank R. Kaufhold for help with constructing the MBP–mScarlet–NLS construct. This work was supported by grants from the National Human Genome Research Institute (grant numbers HG003143 and HG011536 to J.D. and the National Institute for General Medical Sciences (R35GM133762 to A.A.P.). is an investigator of the Howard Hughes Medical Institute. were supported by the Intramural Research Program of the Eunice Kennedy Shriver National Institute of Child Health and Human Development at the National Institutes of Health, USA (grant number ZIAHD008954). This research was in part supported by the Intramural Research Program of the National Institutes of Health (NIH). was supported by the National Institute of Allergy and Infectious Diseases (grant number F31AI189160). was supported by the Graduate Research Fellowship Program from the National Science Foundation. Department of Systems Biology, University of Massachusetts Chan Medical School, Worcester, MA, USA Allana Schooley, Sergey V. Venev & Job Dekker Howard Hughes Medical Institute, Chevy Chase, MD, USA Division of Molecular and Cellular Biology, National Institute for Child Health and Human Development, National Institutes of Health, Bethesda, MD, USA RNA Therapeutics Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA Institute for Medical Engineering and Science and Department of Physics, Massachusetts Institute of Technology, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the study. All cell lines were designed and engineered in the M.D. A.S. performed Hi-C, epigenomic profiling assays, proteomics and all experiments except for the nuclear import assays. analysed Hi-C data, developed analytical methods and supported all other analyses. A.S. analysed Hi-C, ATAC–seq and other relevant datasets. performed preliminary analysis of RanGAP1-depletion Hi-C. A.S. and J.D. wrote the paper with input from S.V.V., V.A., J.W.L., E.N., M.C.D. is a member of the scientific advisory board of Arima Genomics, San Diego, CA, USA and Omega Therapeutic, Cambridge, MA, USA. is inventor on patent application US 12,146,186 B2, held by the University of Massachusetts Chan Medical School, Harvard College, the Whitehead Institute for Biomedical Research, and the Massachusetts Institute of Technology, which covers Hi-C technology. The other authors declare no competing interests. Nature Cell Biology thanks Pedro Rocha and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b, Representative western blot images of whole-cell lysates derived from parental, RanGAP1–AID, and Nup93–AID DLD-1 cell lines demonstrating relative size and abundance of wt and AID-tagged RanGAP1 (a) or AID-tagged Nup93 (b). Vinculin was probed as a loading control. a, Representative immunofluorescence images for Nup93 control and depleted cells. Loss of Histone H3 serine 10 phosphorylation (H3pS10P) and DNA content at indicated time points indicates similar mitotic exit kinetics in the presence or absence of Nup93. b, DNA-content flow cytometry measurements for Nup93 control and depleted cells indicating similar mitotic exit kinetics in the presence or absence of Nup93. c, Representative fluorescent images of endogenous NeonGreen-tagged RanGAP1–AID or Nup93–AID demonstrate efficient degradation following 2 h auxin treatment during mitotic arrest and 5 h release to G. Scale bar, 10 um. d, Representative transmission electron micrographs of RanGAP1–AID- or Nup93–AID-depleted cells fixed in G1 reveal relatively small nuclei with hyper-condensed chromatin. Arrows indicate nuclear pores embedded in the double lipid bilayers of the nuclear envelope, which are not found in Nup93-depleted nuclei. e, Representative immunofluorescence images of Nup93 control and depleted cells 5 h after mitotic release demonstrating presence of nuclear lamina (lamin A) and nuclear envelope (LBR) proteins as well as the DNA-binding nuclear pore complex protein, Elys, in the absence of Nup93. Structural (Nup160) and late associating (FG-Nups) nucleoporins are not found in Nup93-depleted nuclei. f. Nuclear speckle (SON) and nucleolar (NPM1) resident proteins are mis-localized to the cytoplasm in RanGAP1–AID and Nup93–AID-depleted nuclei. Representative immunofluorescence images of cells fixed 5 h after mitotic exit are shown. g, Representative time-lapse fluorescent images of RanGAP1-NG-AID or Nup93-NG-AID control and depleted cells released from Ro-3306 for 5 h. Endogenously tagged RCC1-RFP demarcating chromatin and the MBP–mScarlet-NLS import substrate are shown. a, Representative Hi-C interaction frequency maps at 250 kb (Chr 6: 105–170.8 Mb–Chr 7: 0–56 Mb), 50 kb (Chr 6: 125–145.5 Mb), and 10 kb (Chr 6: 129–130.5 v. 136.1–138.2 Mb) resolution showing genome compartmentalization in control and auxin-treated DLD-1 Nup93–AID cells released from prometaphase arrest for 5 h. First eigenvector (EV1) values for cis interactions are phased by gene density. Examples of canonical loops and MCDs present in the Nup93 depletion are indicated by arrows. b, Heat maps for all RanGAP1 MCDs centred on contact frequency summits with 100 kb flanking regions and sorted by anchor length demonstrate differences in intra-chromosomal EV1 values derived from 10 kb matrices in control and Nup93-depleted cells. c, Pairwise mean observed/expected contact frequency at 10 kb between all RanGAP1 MCDs projected in cis and trans showing enhanced interactions at all length scales and between chromosomes in Nup93-depleted G1 cells. d, Pairwise aggregate observed/expected contact frequency between cCREs assigned in control cells and subjected to hierarchical binning at 10 kb resolution showing enhanced homo- and heterotypic interactions between active promoters (PLS) and enhancers (promoter-proximal:pELS, promoter-distal:dELS) in Nup93-depleted cells compared to controls at multiple genomic distances in cis and in trans. a, Scatter plot between genome-wide Eigenvector E2 (derived from spectral clustering analysis) on x-axis, and chromosome-level cis-eigenvectors E1 concatenated together for all autosomal chromosomes on y-axis. b, Eigenvalues derived from spectral clustering analysis of RanGAP1-depleted Hi-C data sorted by their absolute values. E2–E9 were used in the clustering analysis. c, Overview of spectral clustering of Hi-C data from 5 h G1 RanGAP1-depleted cells binned at 10 kb showing enrichment of MCDs in a specific chromatin profile. Heat maps of mean signal intensity for various functional genomics features binned the same way from top to bottom: ATAC–seq, H3K4me3, H3K27ac, H3K27me3 and CTCF CUT&RUN, extrusion dot anchors (“loop anchors” detected in control G1 Hi-C data), bulk asynchronous cell RNA-seq coverage, SLAM–seq derived nascent RNA coverage over time (vs time 0, -/+ RanGAP1), MCD density (Cyto-specific, shared between Cyto and G1, G1-specific, all Cyto and G1 combined, permissive set of MCDs detected in a combined 5 h and 10 h G1 RanGAP1 depletion sample (“Mega”)), and DLD-1 IPGs (B-combined, V-combined, A2 and A1). Weighted eigenvectors E2–E9 are depicted below the epigenetic marks. Different clusters are demarcated by black vertical lines on each track and the EVs heat map. Clusters are ordered by their average E2, and sorted by E2 within each cluster. Loci corresponding to masked “bad bins”, chrX, chrY and chrM are omitted for clarity. a, Cell-cycle staging, based on DAPI and tubulin immunofluorescent morphology, for control and RanGAP1-depleted cell populations isolated by FACS 1.25 or 1.5 h after prometaphase release. Manual scoring of at least 25 cells in each of two replicates per condition demonstrates the successful enrichment of telophase or cytokinesis cells. b, Overlap of microcompartment domains identified in either cytokinesis or G1 RanGAP1-depleted cells specifies a majority of shared (1,218) MCDs as well as cytokinesis- (565) or G1-specific (876) subsets. c, Representative Hi-C interaction frequency maps at 10 kb (Chr 14: 53.5–55.5 v. 67.8–69 Mb) resolution showing genome organiszation in individual replicates of control and auxin-treated DLD-1 RanGAP1–AID cells enriched in prometaphase, telophase, cytokinesis, or early G1 (5 h after release). Matched first eigenvector (EV1) values for cis interactions are phased by gene density (A > 0). MCDs detected in RanGAP1-depleted cytokinesis- or G1-sorted cell populations are shown. d, Relative fold enrichment of control cCREs at cytokinesis-specific, cytokinesis-G1 shared, or G1-specific MCDs demonstrating the predominance of active promoters and enhancers particularly at the shared domains. e, Pairwise aggregate observed/expected contact frequency between cCREs assigned in control cells and subjected to hierarchical binning at 10 kb resolution showing transient enhanced homo- and heterotypic interactions between active promoters (PLS) and enhancers (promoter-proximal:pELS, promoter-distal:dELS) at multiple genomic distances in cis and in trans during mitotic exit.Ctrl, control; cyto, cytokinesis; pro-Meta, prometaphase; telo, telophase. Source numerical data are provided. a, Proportion of uniquely aligned SLAM–seq reads mapping to ERCC spike-ins that contain greater than one T>C substitution. Reads from three ERCC species in vitro transcribed with 10% 4sU have significantly more T>C substitutions (left) than reads mapping to 3 ERCC species with no 4sU-molecules (right) (Welch's two-sample t-test; p < 2.2 × 10−16). Boxplots indicate median and interquartile range of NTR fold changes for three independent SLAM–seq libraries per time point. b, Percentage of uniquely- aligned reads per SLAM–seq sample that contain greater than one T>C substitutions. 4sU-labelled samples have significantly more substituted reads than unlabelled controls (Welch's two-sample t-test; p = 0.0017). Data shown across time points, with three independent SLAM–seq libraries per time point and condition. Boxplots indicate median and interquartile range of NTR fold changes for three independent SLAM–seq libraries per time point. c, Global distributions of mean new-to-total RNA ratios (NTRs) from SLAM–seq libraries estimated using GRAND-SLAM. Control samples show increased transcriptional activity following mitotic exit relative to RanGAP1-depleted samples. Boxplots indicate median and interquartile range of NTR fold changes for three independent SLAM–seq libraries per time point. d, Heat map of gene timing classification derived on control sample NTRs, but shown for NTRs observed for those genes in RanGAP1-depleted samples. NTRs from depletion conditions reveal a temporally disrupted gene expression program. Source numerical data are provided. a, Relative fragment-length distributions of ATAC–seq reads indicating regular nucleosome positioning in mitotically arrested cells and more dynamic G1 architecture in control and RanGAP1-depleted cells released for 5 h. b, Prevalence of bookmarked CREs based on control prometaphase and G1 ATAC–seq, H3K4me3, and H3K27ac coverage. The fraction of 50 kb bins genome-wide or having EV1 > 0, as well as all G1, shared G1-Cyto, or G1-specific MCDs, which overlap the indicated valency of either promoter or enhancer elements individually are shown. The fraction of 50 kb bins genome-wide or having EV1 > 0, as well as all G1, shared G1-Cyto, or G1-specific MCDs, which overlap the indicated valency of promoter and enhancer elements (left) or either promoter (PLS) or enhancer (ELS) elements individually are shown. d, Heat maps centred on 5,202 peaks of enriched H3K27ac signal called in the G1 control with 5 kb (top) or 75 kb flanking regions and sorted by strength of the prometaphase H3K27ac signal demonstrating prevalence MCDs and of elevated intra-chromosomal EV1 values from 10 kb matrices in control and RanGAP1-depleted cells. H3K27ac and H3K4me3 coverage in prometaphase and G1 control or RanGAP1-depleted cells, as well as CTCF and RNAseq from asynchronous cells are shown. e, Pairwise mean observed/expected contact frequency between 5,202 mitotically bookmarked peaks of enriched H3K27ac signal projected in cis (10 kb) and trans (25 kb) demonstrating enhanced interactions in telophase that peak around cytokinesis of mitotic exit in control cells and continue to strengthen in the absence of RanGAP1. f, Pairwise mean observed/expected contact frequency between bookmarked H3K27ac peaks in control and RanGAP1-depleted cells demonstrating unchanged cell cycle-dependent interaction strength upon treatment with JQ1 or dBET6, to inhibit or degrade BET proteins, respectively. Ctrl, control; cyto, cytokinesis; pro-Meta/M, prometaphase; telo, telophase. Source numerical data are provided. a, P(s) and derivative P(s) plots for Hi-C data from FACS-sorted early G1 (t = 5 h) control and Nup93-depleted cells. b, Pairwise mean observed/expected Hi-C contact frequency between convergent CTCF loops identified in pooled interphase RanGAP1–AID control Hi-C data demonstrating Nup93-dependent looping interactions in early G1 (t = 5 h). Average signal for three central 10 kb bins across the 200 kb CTCF motif-centred window and stack-ups sorted by G1 loop strength are shown. c, Representative immunofluorescence images (at least two independent experiments) of Nup93–AID control and depleted cells fixed 5 h after mitotic release demonstrating the nucleocytoplasmic localization of the cohesin complex subunit, Rad21, and boundary transcription factor, CTCF. d, Pairwise mean observed/expected contact frequency between MCDs in early G1 (t = 5 h) in control and RanGAP1-depleted cells. MCD–MCD contacts are categorized by the presence of a convergent CTCF loop or loop anchor (>0) and the looping domain status of the constituent MCDs, as indicated. a, Workflow for RanGAP1–AID depletion in G1. Control and RanGAP1-depleted cells were fixed at t = 3.5 and 10 h and compared to G1 depletion from 3.5–10 h. b, Representative western blot images of whole-cell lysates collected every 30 min following G1 depletion at t = 3.5 h. c, Hi-C interaction frequency maps at 10 kb resolution (Chr 6: 129.1–130.5 & 136.1–138.2 Mb) for RanGAP1–AID control and depleted cells released from prometaphase for 3.5 or 10 h or G1 depletion from 3.5–10 h. EV1 values phased by gene density (A > 0). Arrows indicate MCD–MCD interactions detected in G1 RanGAP1 depletion. d, Representative immunofluorescence images of RanGAP1–AID control and mitotic (t = 0) or early G1-depleted (t = 3.5 h) cells fixed 10 h after mitotic release demonstrating nucleocytoplasmic localization of Rad21. e, P(s) plots for Hi-C data from RanGAP1–AID control and mitotic (t = 0) or early G1-depleted (t = 3.5 h) cells 10 h after mitotic release. Vertical line indicates average control loop size, which is increased (arrows) in RanGAP1–AID-depleted cells. f, Pairwise mean observed/expected Hi-C contact frequency in late G1 (t = 10 h) at convergent CTCF loops, demonstrating retained loops in early G1 (t = 3.5 h) RanGAP1–AID-depleted cells. Average signal for three 10 kb bins across the 200 kb CTCF motif-centred window and stack-ups sorted by G1 loop strength are shown. g, Observed/expected Hi-C interaction pileups at forward-oriented loop anchor CTCF motifs plotted in a 400 kb window at 10 kb resolution. Ratios for depleted vs control cells are shown. Contacts are categorized by distance or the presence of a convergent CTCF loop or loop anchor (“>0”) and looping domain status. a, Distributions of EV1 values from Eigenvector decomposition of 25 kb binned Hi-C data from control and auxin-treated RanGAP1–AID and Nup93–AID cells in G1. b, Representative examples of 25 kb EV1 tracks from the Hi-C data of control and RanGAP1–AID- or Nup93–AID-depleted cells in early G1. c, Saddle plots representing the segregation of active (A) and inactive (B) chromatin compartments in cis for control and Nup93–AID-depleted cells 5 h after mitotic release (G1). The first Eigenvector from each condition was used to rank 25 kb genomic bins and quantification of the average preferential A–A and B–B interactions for the top 20% strongest A and B loci are indicated. d, Saddle plots representing the segregation of active (A) and inactive (B) chromatin compartments defined in control cells at 25 kb resolution and plotted for Nup93–AID-depleted cells 5 h after mitotic release (G1). Quantification of the average preferential A–A and B–B interactions for the top 20% strongest A and B loci are indicated. e, Pairwise aggregate observed/expected contact frequency between IPGs derived from DLD-1 cells and further categorized by the presence or absence of MCDs showing enhanced homotypic interactions between MCDs in Nup93–AID-depleted cells in early G1. Hi-C libraries generated and analysed in this study. Summary of nuclear proteome changes in RanGAP1–AID-depleted cells. Summary of nuclear proteome changes in Nup93–AID-depleted cells. Excluded regions in spectral clustering. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Schooley, A., Venev, S.V., Aksenova, V. et al. Interphase chromosome conformation is specified by distinct folding programmes inherited through mitotic chromosomes or the cytoplasm. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-04114-7'>Vagus nerve-mediated neuroimmune modulation for rheumatoid arthritis: a pivotal randomized controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-22 10:14:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The inflammatory reflex, in which vagus nerve signaling modulates cytokine production, is dysregulated in rheumatoid arthritis (RA). RESET-RA, a pivotal, double-blind, randomized, sham-controlled trial, evaluated a vagus nerve-targeted neuromodulation system for RA in 242 patients with inadequate response/intolerance to biological/targeted synthetic disease-modifying antirheumatic drugs. Patients were randomized to active or sham stimulation for 3 months, and then all received open-label stimulation with results reported to 12 months. The primary end point was 3-month American College of Rheumatology 20% (ACR20) response. ACR20 rates were higher with active simulation than with sham at 3 months (35.2% versus 24.2%, P = 0.0209), which further improved in open-label to 50.0% at 6 months and 52.8% at 12 months (all-completers). Related serious adverse events (rate = 1.6%) were all perioperative, and resolved. Vagus nerve-mediated neuroimmune modulation for RA achieved its primary efficacy end point and produced durable clinical benefits with a favorable safety profile. Patients with uncontrolled rheumatoid arthritis (RA) suffer from painful chronic joint inflammation, systemic inflammation and progressive disability due to ongoing structural joint damage. Despite the availability of numerous conventional synthetic disease-modifying antirheumatic drugs (csDMARDs; for example, methotrexate) and biological or targeted synthetic DMARDs (b/tsDMARDs) with distinct mechanisms of action, treatment failure remains a challenge for many patients due to lack of initial response, loss of response over time or intolerance to b/tsDMARDs1,2. The central nervous system governs homeostatic control of immune responses and bone turnover through innate neuroimmune and osteo-targeted reflexes, including the vagus nerve-mediated ‘inflammatory reflex'. This reflex is dysregulated in RA; tonic vagus nerve activity is diminished, and reduction in vagal tone precedes the onset of clinical disease3,4,5. Actively stimulating the vagus nerve can engage the inflammatory reflex, resulting in acetylcholine release and specific agonism of α7 nicotinic acetylcholine receptors on immune cells6,7. This broad-spectrum immunomodulation retains cytokine network bioavailability, thereby allowing reduction and resolution of inflammation while maintaining competent immunosurveillance against foreign pathogens and precancerous cells9,10,11. Prior clinical studies of implanted devices support this mechanism of action as a treatment approach for RA12,13,14. In these studies, a total of 27 patients with RA were stimulated with implanted neurostimulators targeting the vagus nerve and although not designed to determine efficacy, demonstrated clinical improvements with reductions in proinflammatory cytokines12,13. Accordingly, we studied the safety and efficacy of an integrated neuromodulation system to treat patients with moderately-to-severely active RA following an inadequate response or intolerance to one or more b/tsDMARDs, in the pivotal (similar to a phase III drug trial) RESET-RA trial. We observed in this randomized, controlled trial of 242 patients that the primary end point of a difference between stimulation and sham in American College of Rheumatology 20% (ACR20) responses at 3 months was achieved, with further clinical improvements documented at 12 months. Following approval by the institutional review board (Advarra centrally and three local boards), a total of 405 patients diagnosed with RA were screened for study participation, 243 were consented and enrolled, and 242 completed standard b/tsDMARD washout per protocol requirements before device implantation (intent-to-treat (ITT) population). Figure 1 provides patient disposition from screening through 12 months. Patients were randomized to either arm 1 (stimulation treatment for 3 months, then continued to open-label stimulation; 122 patients) or arm 2 (sham treatment for 3 months, then crossover to open-label stimulation; 120 patients) following postsurgical recovery. One consented patient was discontinued from the study before device implantation for logistical reasons; the patient was not randomized but was followed for safety per protocol. A single patient randomized to arm 1 did not continue in the study after the 3-month primary end point. The demographics and clinical characteristics of the patients at baseline are presented in Table 1 and Supplementary Tables 1–3. The remainder received at least two csDMARDs. BMI, body mass index; CVA, cerebrovascular accident; TIA, transient ischemic attack; JAKi, Janus kinaseinhibitor, NSAID, nonsteroidal anti-inflammatory drug. The magnetic resonance imaging (MRI)-conditional pulse generator, the ‘implant', was implanted and fitted within a contoured silicone pod that was secured directly to the left cervical vagus nerve15 (Fig. Once implanted, targeted electrical pulses were delivered to the vagus nerve to engage the inflammatory reflex and modulate immune activity. The stimulation parameters were controlled by site-specific programmers using a tablet-based software application that transmitted information through a charger worn by the patient (Fig. Those patients randomized to sham stimulation always received 0 mA, regardless of the stimulation strength set by the programmers, who were blinded to treatment arm assignment. The integrated neuromodulation system consists of an implant and pod. The implant is placed in the pod to position and hold it in place on the left cervical vagus nerve to ensure direct contact for precise stimulation. The implant is approximately 2.5 cm in length and weighs 2.6 g. To charge the implant, patients wear a wireless device (charger) around the neck for a few minutes, once a week. The implant is programmed by healthcare providers (HCPs) using a proprietary application (programmer). 16, by permission of Cold Spring Harbor Laboratory Press. The active stimulation intensity was set to an upper comfort level (maximum = 2.5 mA) and delivered a 1-min train of pulses to the vagus nerve once daily at 10 Hz16 (arm 1 = 1.8 mA average; arm 2 = 0 mA). The 3-month assessments were completed for 99.2% of patients, with two missed visits (one in each arm). All patients, healthcare providers, investigators, joint evaluators and the sponsor were blinded to group assignment until all patients completed assessments at 3 months and the database was locked for primary efficacy analysis. Following the primary end point assessment at 3 months, all patients were eligible to continue in the study for open-label active stimulation treatment. Adjunctive pharmacological treatments (‘augmented therapy') were permitted throughout the open-label stimulation period at the discretion of the rheumatologist in consultation with the patient, with 17.8%, 24.8% and 32.2% of patients receiving protocol-defined augmented therapy at 6, 9 and 12 months, respectively. At these timepoints, 88.0%, 80.6% and 75.2% of patients remained free from adjunctive b/tsDMARD therapy. The primary end point was a difference in proportion of patients in the ITT population receiving stimulation versus sham who achieved an ACR20 response at the 3-month visit from baseline (day of consent). ACR20 response, as defined by the American College of Rheumatology, is a dichotomous composite end point representing the proportion of patients who achieve at least a 20% improvement in both the tender and swollen joint counts (out of a maximum of 28 joints) and in ≥3 of the following 5 additional measures—Health Assessment Questionnaire Disability Index (HAQ-DI) score, patient global assessment of disease, patient pain, evaluator's global assessment of disease and high-sensitivity C-reactive protein concentration17. The primary end point was met at 3 months; ACR20 response was achieved by 35.2% of arm 1 (stimulation) and by 24.2% of arm 2 (sham; Fig. The stratification-adjusted difference in ACR20 response between arms was 11.8% (P = 0.0209, 95% confidence interval (CI) = 0.6, 23.1). At 3 months, Bang's blinding index scores were <0.3 for patients, joint assessors and co-investigators, which indicated satisfactory blinding at the time of primary end point assessment (Supplementary Table 4). a, The percentage of patients who had an ACR20 response at 3 months, the primary efficacy end point of this study. Patients with missing data or who received rescue treatment were imputed as nonresponders. Stimulation, n = 122; sham, n = 120. b, The percentage of patients who had an ACR20 response through 12 months. c, The percentage of patients who had a DAS28-CRP good or moderate response according to the EULAR criteria. d, The percentage of patients who had achieved LDA or remission by DAS28-CRP criteria (score < 3.2). All data are presented as the percentage of each group with s.e. Formal statistical analyses were only performed at the 3-month study visit (double-blind) with the Cochran–Mantel–Haenszel (CMH) test using stratification factors based on prespecified criteria (prior inadequate or lost response to a tsDMARD, prior inadequate or lost response to ≥4 biological DMARDs with ≥2 mechanisms of action and RA disease severity defined as either <4 TJC28 or <4 SJC28 at day 0), at a one-sided significance level of 0.025. After 3 months (open-label stimulation), patients were permitted to augment therapy with adjunctive drugs without restriction. Data presented at 6, 9 and 12 months in b–d include all patients completing the visit who had not used augmented therapy. End points included a DAS28-CRP good/moderate response according to the European League Against Rheumatism (EULAR) criteria; a DAS28-CRP minimal clinically important difference (MCID; −1.2); a HAQ-DI MCID (−0.22); and an ACR20 response from day 0 (randomization). Among the key secondary end points, multiplicity adjustment was performed using Hochberg's step-up procedure. All secondary end points showed a higher response rate for stimulation compared to sham, although significance was achieved for only EULAR good/moderate response (Supplementary Table 5). EULAR good/moderate response was achieved by 60.7% of arm 1 (stimulation) and by 41.7% of arm 2 (sham) (multiplicity-adjusted P = 0.0048, 95% CI = 7.3, 31.7). HAQ-DI MCID was achieved by 45.9% of arm 1 and by 36.7% of arm 2 (multiplicity-adjusted P = 0.0797, 95% CI = −3.3, 21.4). ACR20 from day 0 was achieved by 31.1% of arm 1 and by 22.5% of arm 2 (multiplicity-adjusted P = 0.0797, 95% CI = 1.1, 25.3). All other efficacy end points at 3 months and within the open-label stimulation period were exploratory. Open-label stimulation was initiated in both arms after the 3-month controlled-blind period. Response rates for primary and all secondary end points increased further in arm 1, while patients in arm 2 demonstrated clinical improvements following initiation of stimulation. Clinical end point responses were sustained in both arms through 12 months (Supplementary Table 5). Prespecified clinically relevant composite outcome measures included EULAR good/moderate response, low disease activity (LDA) or remission by DAS28-CRP, and LDA or remission by the Clinical Disease Activity Index (CDAI) evaluated at 3 months (sham-controlled-blinded period) and also during the open-label stimulation period at 6 months, 9 months and 12 months (Fig. Difference in rates for CDAI LDA/remission at 3 months did not reach significance (% rate ± s.e.m.—arm 1 = 23.3% ± 4, arm 2 = 16.0% ± 3; P = 0.0648, 95% CI = −2.1, 17.7), although favored stimulation. All composite outcome measures improved further in both arms during the open-label period, when all patients received stimulation, with responses maintained at 12 months. Comparable results were observed by an all-completers analysis, which additionally included those patients who received augmented therapy (Supplementary Table 6). Patient satisfaction rate using a five-point Likert rating scale revealed that 78.1% of patients were somewhat to very satisfied with the therapy at 6 months (satisfaction %—arm 1 = 75.6%, arm 2 = 80.7%; Supplementary Table 10). Images were analyzed with the validated Outcome Measures in Rheumatology (OMERACT) RA MRI score (RAMRIS) to objectively quantify bone erosion progression18. In the ITT population, a total of 216 patients had RAMRIS scores measured at both baseline and 3 months (arm 1, n = 109; arm 2, n = 107). From baseline to 3 months, a smaller proportion of patients in arm 1 (stimulation) exhibited progression of bone erosions (>0.5 increase in score) in the evaluated hand and wrist compared with arm 2, although the difference was not significant (P = 0.248; Fig. In the prespecified subgroup analysis of patients with a phenotype enriched for erosive damage risk (defined as synovitis score of 2 or more on any individual joint, at least four joints with a score of 1 or any joint with osteitis at baseline), a total of 105 patients met the erosive phenotype criteria (arm 1, n = 57; arm 2, n = 48). In this subgroup, the rate of progression of bone erosion from baseline to 3 months was significantly decreased in arm 1 (stimulation = 18.9%) as compared with arm 2 (sham = 37.8%, P = 0.016; Fig. During the open-label stimulation period from 3 months to 6 months, the rate of progression of bone erosion declined in arm 2 (Fig. in the number of tender and swollen joints and as compared to baseline are shown in a (tender-joint count) and in b (swollen-joint count). c–f, The cumulative probability of a change in erosion score, as assessed via the OMERACT RAMRIS. c,e, Plot change from baseline to 3 months during the controlled-blind period. Patients who completed the 3-month and 6-month visits without augmented therapy are included in c and d. Patients at risk for erosion progression are included in e and f. g,h, The percentage of patients who had erosion progression, defined as >0.5 increase in RAMRIS erosion score by MRI. Data are presented as the mean (a,b) or the percentage of each group (g,h) with s.e. Statistical analyses were only performed at points through 3 months; a and b with mixed-effect model repeated measures and g and h with the CMH test. All tests use one-sided significance level of 0.025. Primary and secondary end point response measures during the open-label period were consistent with analyses using nonresponder imputation to account for missed visits or study dropouts (Supplementary Table 5). The safety evaluation was based on all available data at the time of reporting, with a mean implant duration of >700 days. No deaths or unanticipated adverse device effects occurred at any point during the trial. Overall, adverse events occurred in a similar proportion of patients in both arms during the controlled period (Table 2 and Supplementary Table 11). Nonserious related adverse events were predominantly associated with the implantation procedure (Supplementary Table 13), reported in 38 patients (15.6%; 52 events). These were consistent with those seen in other devices implanted near the cervical vagus nerve. The most frequent events were mild to moderate hoarseness, classified as either vocal cord paresis (4.5%, n = 11) or dysphonia (2.9%, n = 7). These adverse events resolved over the course of up to one year; three patients received bulk injection fillers into the left vocal cord and some underwent voice therapy. Active stimulation (1 min daily) was generally well tolerated. Mild to moderate stimulation-related events, most commonly pain, occurred in 4.2% of patients (n = 10) during the controlled period and 4.6% (n = 11) during long-term follow-up. These typically resolved after reducing stimulation strength or adjusting the time of delivery, without interruption of therapy. Overall, during the controlled period and long-term follow-up, 4 of 242 ITT patients (1.7%) experienced a serious adverse event related to surgical procedure, all having an onset during the perioperative period (1.6% of the 243 enrolled patients in the safety population). All these events resolved without clinically significant sequelae. There was one event of postoperative incision-site swelling, one event of transient vocal cord paresis (presented as hoarseness) with dysphagia, one intraoperative pharyngeal perforation (that occurred during an explantation procedure and was immediately repaired) and one event of postoperative dysphonia (that presented as hoarseness) possibly associated with postoperative progression of age-related vocal cord bowing (presbylarynges; Table 2). There were no serious adverse events related to the active stimulation. A total of six patients underwent device removal (explantation) before 12 months. All explantations were performed as outpatient, elective procedures. The reasons for explantation included a nonfunctioning device (n = 1), chronic pain at the incision site (n = 1), gastrointestinal symptoms attributed to stimulation therapy (n = 1) and patients who requested removal due to perceived lack of benefit (n = 3). Details of the safety results during the open-label stimulation period are provided in Table 2 and Supplementary Tables 17–21. No safety concerns have been identified through other protocolized safety monitoring assessments (for example, vital signs, hematology, ECG, blood pressure, heart rate). All adverse events of infection, major adverse cardiac events and malignancies were reviewed and determined to be unrelated, and the rates were within the expected range for the target RA patient population. RESET-RA is the first randomized, sham-controlled trial to demonstrate the safety and efficacy of a neuroimmune modulation device to treat any autoimmune disease, specifically RA. Compliance with therapy and preservation of treatment blinding were achieved through automated nighttime delivery of active stimulation. Efficacy was observed in the 3-month blinded-control period and further supported by improvement during the open-label stimulation period through 12 months, with low usage of adjunctive b/tsDMARDs. In patients with high baseline risk for structural damage, active stimulation substantially reduced progression of bone erosions, as assessed by quantitative MRI joint imaging. The enrolled study population reflected a spectrum of treatment experiences, including 43% classified with ‘difficult-to-treat' (D2T) RA, that is, those who had previously failed multiple b/tsDMARDs, with at least 2 different mechanisms of action19, and 39% who had failed a single b/tsDMARD. The patient's choice to undergo surgical implantation of an experimental device rather than switch to another b/tsDMARD indicated strong patient preference for nonpharmacologic treatment options. Unlike most trials for new therapies in this population, RESET-RA did not impose a requirement for elevated CRP at baseline, allowing inclusion of patients with moderate-to-severe RA regardless of CRP status who would normally be excluded from studies of new RA therapies. This inclusive design reflects the broader United States (US) RA population, where CRP is not uniformly elevated despite active disease20. Efficacy measures were consistent with those used in phase 3 RA drug trials. Rates of ACR20 response, EULAR moderate/good response and LDA/remission by DAS28-CRP criteria were all higher with stimulation compared with sham at 3 months (LDA/remission by CDAI was numerically higher). Blinding across all assessed parties was satisfactory at the primary clinical end point, and the sham response rate was consistent with placebo response rates reported in phase 3 trials of pharmacologic therapies in bDMARD-refractory patients21,22,23,24. During the open-label period, therapeutic responses in arm 1 continued to improve over time, while patients in arm 2 demonstrated clinical benefits following crossover to active stimulation. The observed clinical improvement past 3 months was consistent with the pilot study of this neuromodulation system in patients with RA12,14. In both arms, the ACR20, clinical LDA/remission and EULAR good/moderate response rates were comparable to those achieved with effective pharmacologic therapies in similar bDMARD-refractory patients by 6 months, and were sustained through 12 months21,22,23,24. Protection from progressive structural damage was documented by MRI. Although most patients who have had an inadequate response to b/tsDMARD therapy were unlikely to have had MRI-observable erosive progression over a 3-month period (as was observed in the ITT population), the subgroup with elevated synovitis and/or osteitis at baseline was at higher risk for progression25. In this higher erosive-risk phenotype, active stimulation attenuated the rate of erosion progression, a clinically relevant effect, given that progressive MRI-detectable erosions at 3 months predict radiographic progression at 12 months and subsequent functional decline26,27. The observed protection from bone erosions is consistent with several known mechanisms of vagus nerve-targeted neuroimmune modulation. For example, vagotomized mice become osteoporotic, while stimulating the vagus nerve leads to increased bone formation12,28,29,30. Electrical stimulation of the vagus nerve induces release of specialized pro-resolving mediators and neurotransmitters that signal through specific receptors including the ChemR23 lipid receptors, acetylcholine receptors and adrenergic receptors on osteoblasts, osteoclasts and osteocytes resulting in reduction of osteoclastogenesis and osteoclastic activity28,30,31,32,33,34,35. The protective effect of this therapy on bone may therefore be mediated by the canonical RANKL/osteoprotegerin pathway (neuroimmune modulation decreases RANKL and increases osteoprotegerin, the decoy ligand for RANKL) as well as direct effects on osteoblasts, osteoclasts and osteocytes28,30,31,32,33,34,35. While these pathways are biologically plausible, confirmation in RA patients will require future targeted mechanistic studies. In evaluating the safety of drugs used to treat RA, there has been a critical focus on increased rates of serious infections, major cardiovascular, venous and/or arterial thrombotic events and malignancies, reflecting the immunosuppressive mechanisms of action of b/tsDMARDs36. The safety data from the RESET-RA trial through 12 months showed no increase in these adverse events, consistent with the well-established safety data of vagus nerve stimulation accumulated over decades of use in nonautoimmune disease populations37. No hemodynamic changes, including hypotension or bradycardia, were detected during the study. The rate of related serious adverse events was low, associated with the surgery, all of which were successfully managed to clinical resolution, and with no events observed in the open-label period. Nonserious adverse events were predominantly associated with the implantation procedure, were mild or moderate in severity and consistent with the inherent risks of any surgical procedure performed near the cervical vagus nerve37. Clinical application of electrical stimulation to the vagus nerve with a variety of devices has been used in the U.S. since 1997, initially as an adjunctive treatment for patients with drug-refractory epilepsy, later for difficult-to-treat depression, and most recently for use in stroke patients to improve motor function when paired with physical rehabilitation37. Relative to this extensive prior clinical experience, no new risks or safety signals were observed in the RESET-RA trial. The new integrated neuromodulation system used in the RESET-RA trial represents an important advancement in device design, as it uses a rechargeable battery and integrated electrodes, obviating the need to tunnel a lead wire from an implantable pulse generator, typically placed in the chest, to the cervical vagus nerve. This design eliminates the risk of lead breakage and chest-pocket infections seen with other systems. Nonimplanted devices could also mitigate these risks; however, to date, randomized controlled trials of transcutaneous stimulation targeting neuroimmune pathways have not demonstrated efficacy in RA38. A limitation of this trial was the 3-month controlled phase, which was restricted in duration by U.S. Food and Drug Administration (FDA) guidelines to protect patients randomized to placebo from an extended period without treatment. In drug trials, peak ACR20 responses are typically observed by 3 months, that is, continued group-level improvement after 3 months is minimal21,22,23,24. The response is consistent with clinical experience with vagus nerve stimulation for other indications, where some patients took longer to achieve threshold benefit37. The extended time to peak therapeutic response is likely influenced by modulation of innate neuroimmune pathways rather than by acute inhibition of discrete inflammatory pathways, the mechanism of most pharmaceutical interventions. This may explain why, despite a statistically significant difference in ACR20 response rates between arms at 3 months (meeting the primary efficacy end point), the effect size of 11.8% was smaller than the projected effect size of 25% and smaller than that typically reported in drug trials at 3 months21,22,23,24. During open-label, the therapeutic response to active stimulation increased between 3 and 6 months, as was first observed in the pilot study of this neuromodulation system12,14. Demonstrating longer-term sustained reductions in disease activity (beyond short-term efficacy measured at 3 months) is clinically meaningful, as RA is a chronic disease that requires prolonged therapy. Caution should be exercised in interpreting the low rate of augmentation therapy, as observed disease activity after 3 months to 12 months was not paired against protocol-mandated decision-making to either advance or not advance therapy. In conclusion, evidence from this large, pivotal, randomized, double-blind, sham-controlled trial of an integrated neuromodulation system using an implantable device demonstrated significant sustained clinical benefits and high patient satisfaction with low rates of adverse events through 12 months of follow-up in a largely D2T, moderately-to-severely active RA population. Patients benefited from automatic delivery of 1 min of active treatment daily to achieve substantial improvement in their disease activity, resulting in a significant decrease in swollen and tender joints, and reduction in MRI-observed erosions without the use of b/tsDMARDs. Vagus nerve-mediated neuroimmune modulation offers a first-in-class nonpharmacologic therapeutic option for RA. The RESET-RA trial (ClinicalTrials.gov registration: NCT04539964, registered on 31 August 2020) was a pivotal, randomized, double-blind, sham-controlled trial conducted at 41 sites across the U.S. based on study design considerations from the FDA previously used to approve therapeutic trials for RA39,40. Within 30 days of completing the baseline assessments performed at time of informed consent, all eligible patients who provided informed consent underwent an outpatient surgical procedure to implant the integrated vagus nerve stimulation device, followed by randomization (day 0) in a 1:1 ratio to either an active stimulation (arm 1) or a sham stimulation (arm 2) group. Randomization incorporated stratification for prior inadequate or lost response to a tsDMARD, exposure to ≥4 biological DMARDs with ≥2 mechanisms of action and RA disease severity at day 0. The randomization scheme was generated by the study biostatisticians and implemented centrally through Interactive Response Technology (IRT). End points were assessed at 3 months after randomization as change from baseline. Data collection used the Veeva System of Electronic Data Capture, with clinical sites directly entering data through a password-protected portal, with source documentation verification completed by clinical monitors. Data collection by Electronic Data Capture was compliant for data integrity, security and traceability through adherence to regulations like FDA 21 CFR Part 11 and ICH GCP, and global standards. After the primary end point assessment at 3 months, all patients were eligible for open-label treatment with stimulation, and use of adjunctive pharmacological treatments were permitted at the discretion of the rheumatologist in consultation with the patient (‘augmented therapy'). Augmented therapy was defined as initiation of a b/tsDMARD, use of additional csDMARD(s), high-dose steroids or steroid injections in combination with continued active stimulation during open-label follow-up. A detailed schematic of trial design and visits is provided in Extended Data Fig. All data and results associated with the dataset are available in the ‘Results' section and Supplementary Note. All patients provided written informed consent using an institutional review board (IRB)-approved document. Patients were compensated with a nominal payment, approved by the IRB, for each completed visit. Eligible patients were adults (age = 22–75 years, inclusive) with moderately-to-severely active RA defined at baseline (day of informed consent) by the presence of >4 tender joints and >4 swollen joints of 28 joints, as assessed by a designated joint evaluator. Prior treatment with at least one csDMARD (for example, methotrexate) for a minimum of 12 weeks was required at a continuous and nonchanging dose for at least 4 weeks before consent. Patients were required to maintain this csDMARD dose throughout the double-blind, sham-controlled period. Patients were also required to have experienced an inadequate response or intolerance to at least one or more b/tsDMARDs before consent, with standard b/tsDMARD washout completed before the surgical procedure for device implantation. An elevated level of an acute phase reactant, such as serum C-reactive protein, was not required for eligibility. Exclusion criteria included history of vagotomy, partial or complete splenectomy, clinically significant cardiovascular disease or regular use of or dependency on nicotine products within the two years preceding study participation. Participants meeting final eligibility criteria and in whom the implantation procedure was attempted were considered enrolled (safety cohort). The implantation procedure occurred within 30 days of informed consent and took place in an operating room under general anesthesia. The Implant and Pod were placed on the left cervical vagus nerve by surgeons trained and experienced in procedures involving the implantation of vagus nerve stimulation devices. All enrolled and randomized patients were included in the ITT analysis. Stimulation therapy was delivered by a neuromodulation device comprised of an implant with an integrated, rechargeable battery approximately 2.5 cm in length. The implant was placed in a silicon positioning pod on the left cervical vagus nerve during an outpatient surgical procedure under general anesthesia (Fig. Stimulation parameters chosen for this study have been designed, translated and validated in 3 prior clinical studies12,13,16,43. All patients, regardless of treatment assignment, completed the same stimulation titration protocol that occurred weekly over a period of 4 weeks. The device was titrated to an upper comfort level that did not exceed 2.5 mA. For treatment, a pulse train was delivered to the vagus nerve for 1 min once daily at 10 Hz (stimulation group), while patients randomized to sham stimulation always received 0 mA, regardless of the stimulation strength set by the blinded programmer. Stimulation was programmed to be delivered in the early morning hours, when patients would typically be asleep. Patients were instructed that feeling stimulation was not necessary to be receiving active treatment, and if stimulation was felt, the stimulation setting may not be at a therapeutic dose during the double-blind, sham-controlled period. Blinding to group assignment was maintained for patients, healthcare providers, investigators, joint evaluators and the sponsor until all patients completed assessments at 3 months and the database was locked for primary efficacy analysis. Because all patients crossed over to open-label follow-up before the study was unblinded, they repeated the initial titration protocol, as it was unknown whether they had been receiving stimulation or sham during the 3-month double-blind period. The primary end point was the difference in proportion of patients receiving stimulation versus sham who achieved an ACR20 response at the 3-month visit from baseline (day of consent). ACR20 response is a dichotomous composite end point indicating the proportion of patients that achieve at least 20% improvement in the number of both tender and swollen joints of a maximum of 28 joints and ≥3 of 5 additional measures—HAQ-DI score (scale 0 = mild disability to 3 = severe disability), patient global assessment of disease (0 = inactive to 10 = very active, patient pain (0 = no pain to 10 = worst), evaluator's global assessment of disease (0 = inactive to 10 = very active) and high-sensitivity C-reactive protein concentration (mg ml−1)17. Additional prespecified end points included DAS28-CRP good/moderate response according to the EULAR criteria, achievement of low disease activity (LDA) or remission by DAS28-CRP criteria (DAS28-CRP, score < 3.2 for LDA or remission) and LDA or remission by the CDAI (score <10 for LDA or remission). Additional outcomes were collected that were not the focus of this study and may be the topic of subsequent publications. Detailed descriptions of end points can be found in the Supplementary Note—Description of efficacy end points. To objectively evaluate treatment effect on joint inflammation and erosions, a standardized, validated imaging method, using gadolinium contrast-enhanced MRI of the hand and wrist, was used. The OMERACT RAMRIS was used to evaluate progression of bone erosions18. All images were assessed centrally by two independent radiologists blinded to treatment allocation, clinical information and the order in which the images were acquired to ensure objective and unbiased scoring. Prespecified analyses included the percentage of patients exhibiting progression in bone erosions (increase in score > 0.5 at 3 months compared to first MRI) among patients with a highly erosive phenotype (defined as synovitis score of 2 or more on any individual joint, at least four joints with a score of 1 or any joint with osteitis at baseline)25,44. To evaluate reported satisfaction with the therapy, a participant satisfaction questionnaire, administered at 6 months, was comprised of five-point Likert rating scale and question on whether the participant would recommend the SetPoint System. An optional section was available for the participant to provide comments. This was an exploratory end point and there were no prespecified analyses. Adverse events and serious adverse events were tabulated and coded using the Medical Dictionary for Regulatory Activities (version 23.1) and the incidence of events was tabulated based on system organ class and preferred term. All enrolled patients were analyzed for safety. Adverse events were evaluated by investigators for relatedness to the implant device, implantation procedure, charger, stimulation therapy or explantation procedure (if completed). Safety was also monitored by clinical laboratory tests, including complete blood count, measurement of vital signs, electrocardiogram and other safety assessments performed at baseline and scheduled visits. An investigational device exemption was approved by the FDA, and the trial was conducted in accordance with national and local regulations, the ethical principles of the Declaration of Helsinki and Good Clinical Practice guidelines. The protocol was approved by an institutional review board at each participating site. All patients provided written informed consent before undergoing study-related activities. An independent data safety and monitoring committee oversaw the study conduct. Data was analyzed by the sponsor's biostatistician and interpreted by the sponsor and authors. Advarra served as the central IRB, with three sites using local IRBs. Power calculations were performed solely for the primary end point. A sample size of 120 per study group would provide 91.6% power to detect a 25% difference in ACR20 response at 3 months, with one-sided α of 0.025, assuming response rates of 45% and 20% in the stimulation and sham groups, respectively. These analyses were performed as an all-completers analysis (all patients who completed the specified follow-up visit) and a nonaugmented analysis (all patients who completed the visit without use of augmented therapy). Binary responses, including the ACR20 response at 3 months, were analyzed with the Cochran–Mantel–Haenszel test using stratification factors based on prespecified criteria (prior inadequate or lost response to a tsDMARD, prior inadequate or lost response to ≥4 biological DMARDs with ≥2 mechanisms of action and RA disease severity defined as either <4 TJC28 or <4 SJC28 at day 0). The secondary efficacy end points that were continuous variables (change from baseline) were analyzed using mixed-effect model repeated measure statistics. Among the key secondary end points, multiplicity adjustment using Hochberg's step-up procedure was performed to control the familywise type 1 error rate at a one-sided significance level of 0.025 (ref. Assessment of LDA or remission were evaluated by DAS28-CRP (score < 3.2) and CDAI (score < 10) as prespecified exploratory end points. The Bang's blinding index and its associated 95% CIs were calculated for patients, investigators and joint evaluators (scale range = −1 to 1 with │scores│ < 0.3 considered as satisfactory blinding)41. Given that the primary efficacy end point of ACR20 response was composed of seven components, missing data were handled as follows: if either tender-joint count or swollen-joint count were missing, or if ≥3 of 5 remaining ACR measures were missing, the participant was considered as a nonresponder. For binary secondary end points, participants with missing efficacy data, early withdrawals (before 3 months) or participants who received rescue treatment before 3 months were imputed as nonresponders. Rescue treatment was defined as any change in RA treatment made before 3 months for the reason of addressing worsening of RA symptoms by adding a b/tsDMARD, increasing the dose or adding a csDMARD, increasing the dose or adding a corticosteroid or a corticosteroid injection within 30 days of a study visit. For continuous end points, the change from baseline was set to missing at visits with missing postbaseline values, where data were imputed to missing, or for patients who received rescue treatment before 3 months. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Access to the clinical trial data in this paper can be requested from the corresponding author after one year from publication by any qualified researchers who engage in rigorous, independent scientific research. Data will be provided following review and approval of a research proposal and Statistical Analysis Plan (SAP) and execution of a Data Sharing Agreement (DSA). The use of custom code was not applicable in this paper. Evrengul, H. et al. Heart rate variability in patients with rheumatoid arthritis. Baker, M. C., Nagy, D., Tamang, S., Horvath-Puho, E. & Sorensen, H. T. Vagotomy and the incidence of rheumatoid arthritis and osteoarthritis: a Danish register-based study. Koopman, F. A. et al. Autonomic dysfunction precedes development of rheumatoid arthritis: a prospective cohort study. & Tracey, K. J. Mechanisms and therapeutic relevance of neuro-immune communication. Rethinking inflammation: neural circuits in the regulation of immunity. & Tracey, K. J. Bioelectronic medicine: preclinical insights and clinical advances. Dalli, J., Colas, R. A., Arnardottir, H. & Serhan, C. N. Vagal regulation of group 3 innate lymphoid cells and the immunoresolvent PCTR1 controls infection resolution. Genovese, M. C. et al. Safety and efficacy of neurostimulation with a miniaturised vagus nerve stimulation device in patients with multidrug-refractory rheumatoid arthritis: a two-stage multicentre, randomised pilot study. Koopman, F. A. et al. Vagus nerve stimulation inhibits cytokine production and attenuates disease severity in rheumatoid arthritis. Gaylis, N. B. et al. Neuroimmune modulation for drug-refractory rheumatoid arthritis: long-term safety and efficacy in patients enrolled in a pilot vagus nerve stimulation study. Peterson, D. et al. Clinical safety and feasibility of a novel implantable neuroimmune modulation device for the treatment of rheumatoid arthritis: initial results from the randomized, double-blind, sham-controlled RESET-RA study. Felson, D. T. et al. American College of Rheumatology. Preliminary definition of improvement in rheumatoid arthritis. The OMERACT rheumatoid arthritis magnetic resonance imaging (MRI) scoring system: updated recommendations by the OMERACT MRI in arthritis working group. Hofman, Z. L. M. et al. Difficult-to-treat rheumatoid arthritis: what have we learned and what do we still need to learn?. Kay, J. et al. Clinical disease activity and acute phase reactant levels are discordant among patients with active rheumatoid arthritis: acute phase reactant levels contribute separately to predicting outcome at one year. Genovese, M. C. et al. Safety and efficacy of upadacitinib in patients with active rheumatoid arthritis refractory to biologic disease-modifying anti-rheumatic drugs (SELECT-BEYOND): a double-blind, randomised controlled phase 3 trial. Burmester, G. R. et al. Tofacitinib (CP-690,550) in combination with methotrexate in patients with active rheumatoid arthritis with an inadequate response to tumour necrosis factor inhibitors: a randomised phase 3 trial. Genovese, M. C. et al. Abatacept for rheumatoid arthritis refractory to tumor necrosis factor α inhibition. Genovese, M. C. et al. Baricitinib in patients with refractory rheumatoid arthritis. Determining a magnetic resonance imaging inflammatory activity acceptable state without subsequent radiographic progression in rheumatoid arthritis: results from a followup MRI study of 254 patients in clinical remission or low disease activity. Very early MRI responses to therapy as a predictor of later radiographic progression in early rheumatoid arthritis. Schett, G. & Gravallese, E. Bone erosion in rheumatoid arthritis: mechanisms, diagnosis and treatment. Bajayo, A. et al. Skeletal parasympathetic innervation communicates central IL-1 signals regulating bone mass accrual. Tamimi, A. et al. Could vagus nerve stimulation influence bone remodeling?. A. et al. Neurostimulation of the cholinergic anti-inflammatory pathway ameliorates disease in rat collagen-induced arthritis. & Jouvene, C. Novel mediators and mechanisms in the resolution of infectious inflammation: evidence for vagus regulation. & Walboomers, X. F. Application of specialized pro-resolving mediators in periodontitis and peri-implantitis: a review. Bassi, G. S. et al. Modulation of experimental arthritis by vagal sensory and central brain stimulation. Rosch, G., Zaucke, F. & Jenei-Lanzl, Z. Autonomic nervous regulation of cellular processes during subchondral bone remodeling in osteoarthritis. Frisell, T. et al. Safety of biological and targeted synthetic disease-modifying antirheumatic drugs for rheumatoid arthritis as used in clinical practice: results from the ARTIS programme. & Nguyen, D. K. Learnings from 30 years of reported efficacy and safety of vagus nerve stimulation (VNS) for epilepsy treatment: a critical review. Baker, M. C. et al. A randomized, double-blind, sham-controlled, clinical trial of auricular vagus nerve stimulation for the treatment of active rheumatoid arthritis. Clinical Development Programs for Drugs, Devices, and Biological Products for the Treatment of Rheumatoid Arthritis (RA). Rheumatoid Arthritis: Developing Drug Products for Treatment. & Park, J. Blinding assessment in clinical trials: a review of statistical methods and a proposal of blinding assessment protocol. D'Haens, G. et al. Neuroimmune modulation through vagus nerve stimulation reduces inflammatory activity in Crohn's disease patients: a prospective open-label study. Baker, J. F., Conaghan, P. G., Emery, P., Baker, D. G. & Ostergaard, M. Validity of early MRI structural damage end points and potential impact on clinical trial design in rheumatoid arthritis. A sharper Bonferroni procedure for multiple tests of significance. The authors thank all patients and their families who were willing to participate in this trial. We thank Spire Sciences for performing the MRI image analyses. The RESET-RA trial was sponsored by SetPoint Medical. Arizona Arthritis & Rheumatology Research, Avondale, AZ, USA Willow Rheumatology and Wellness, Willowbrook, IL, USA Great Lakes Center of Rheumatology, Lansing, MI, USA Clinical Trials of Texas, LLC, San Antonio, TX, USA IRIS Research and Development, Plantation, FL, USA Arthritis & Rheumatic Disease Specialties, Aventura, FL, USA Arthritis & Osteoporosis Consultants of the Carolinas, Charlotte, NC, USA Physician Research Collaboration, Lincoln, NE, USA Massachusetts General Hospital, Boston, MA, USA Inland Rheumatology Clinical Trials, Upland, CA, USA Parris and Associates Rheumatology, Atlanta, GA, USA Rheumatology, Western Washington Medical Group, Bothell, WA, USA Southwest Rheumatology Research, Mesquite, TX, USA Long Island Regional Arthritis & Osteoporosis Care, Babylon, NY, USA Yaakov A. Levine, Melissa L. Evangelista, Amy A. Derosier & David Chernoff The Feinstein Institutes for Medical Research, Manhasset, NY, USA Donald and Barbara Zucker School of Medicine at Hofstra/Northwell, Hempstead, NY, USA Harvard Medical School, Cambridge, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar reports consulting, grant/research support and/or honoraria (including speakers bureau, symposia and expert witness) from AbbVie, AstraZeneca, Boehringer Ingelheim, Bristol Myers Squibb, Celgene, Eli Lilly, Genentech, Gilead, GlaxoSmithKline, IGM Biosciences, Janssen, Merck, Novartis, Pfizer, Roche and UCB; and consulting from SetPoint Medical. reports consulting, advisory roles, speaker honoraria and/or grant/research support from AbbVie, Amgen, AstraZeneca, Eli Lilly, Janssen, Novartis and UCB. reports advisory, consultancy, grant/research support and/or speaker honoraria from AbbVie, Alexion, Amgen, Artiva Biotherapeutics, AstraZeneca, Boehringer Ingelheim, Bristol Myers Squibb, Celgene, Centocor, Eli Lilly, Esaote, Exagen, Genentech, Gilead, Global Healthy Living Foundation, Horizon, Image Analysis Group, Janssen, Mallinckrodt, Merck, Novartis, Pfizer, Pharmacia, Radius, Regeneron, Sandoz, Sanofi, Takeda, Theramex, UCB and Vividion Therapeutics. reports research funding from AbbVie, Acelyrn, Alumis, Amgen, Aqtual, Artiva, Biogen, Bristol Myers Squibb, Eli Lilly, Galapagos, Genentech, Gilead, GlaxoSmithKline, Horizon, IGM Biosciences, Moonlake, Novartis, Sanofi, SetPoint Medical, Takeda and UCB; is an officer/board member of AB Solutions; and an advisor/review panel member for Inmedix. reports advisory roles, consultancy, speaker honoraria and/or research support from AbbVie, Amgen, AstraZeneca, Bristol Myers Squibb, GlaxoSmithKline, Janssen, Pfizer, SetPoint Medical and UCB. reports research funding from AbbVie, Amgen, Eli Lilly, Horizon, Janssen, SetPoint Medical and UCB. reports consulting, advisory roles, research funding, intellectual property/patents and/or speaker honoraria from Amgen, Arthrosi, Bristol Myers Squibb, Eli Lilly, Johnson & Johnson, Novartis, SOBI and UCB. reports consulting, research funding and/or stock options from AbbVie, Amgen, Johnson & Johnson, Merck, Novartis, Takeda and UCB. reports research funding from AstraZeneca, Scipher Medicine and SetPoint Medical. are employees of and hold equity/stock options in SetPoint Medical. reports consulting and/or research funding from AbbVie, Amgen, Bendcare, Bristol Myers Squibb, Corrona, Crescendo, Eli Lilly, Genentech, GlaxoSmithKline, Janssen, Moderna, Novartis, Pfizer, Roche, Sanofi and UCB; and is an officer/board member of FASTER. reports consulting and/or research funding from Neuropace, Medtronic, Inbrain and SetPoint Medical. The other authors declare no competing interests. Nature Medicine thanks Rushna Ali, Sejong Bae and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 15, licensed under a Creative Commons Attribution 4.0 International License. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Vagus nerve-mediated neuroimmune modulation for rheumatoid arthritis: a pivotal randomized controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251221043237.htm'>A traditional Brazilian plant shows unexpected strength against arthritis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-22 05:48:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A research team in Brazil has found strong evidence that the Joseph's Coat plant (Alternanthera littoralis) is both safe and effective at reducing inflammation, easing pain, and protecting against arthritis. Joseph's Coat grows naturally along Brazil's coastline and has long been used in traditional medicine to treat inflammation, infections, and parasitic illnesses. Despite its widespread use, there had been little scientific research confirming whether these benefits were real or whether the plant was safe. This phase of the work was led by Marcos Salvador, a pharmacist from the Institute of Biology (IB) at UNICAMP. A team led by pharmacologist Cândida Kassuya from the Faculty of Health Sciences at UFGD evaluated how well the extract reduced inflammation in experimental models of arthritis. "Finally, we performed the toxicological analyses under my coordination," explains Arielle Cristina Arena, associate professor in the Department of Structural and Functional Biology at the Institute of Biosciences at UNESP's Botucatu Campus. Laboratory Results Show Reduced Inflammation and Joint Damage "In the experimental models, we observed reduced edema, improved joint parameters, and modulation of inflammatory mediators, suggesting antioxidant and tissue-protective actions," says Arena. These findings indicate that the plant does more than reduce swelling. The results also suggest that it may help protect joint tissue and limit damage associated with inflammatory conditions like arthritis. The findings also point to a favorable safety profile at therapeutic doses, which could be encouraging for eventual human use. Regulatory approval would also be necessary before any therapeutic use. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            