
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-19</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250519131126.htm'>Experimental painkiller could outsmart opioids -- without the high</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-19 17:59:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The drug, called SBI-810, is part of a new generation of compounds designed to target a receptor on the nerves and spinal cord. "What makes this compound exciting is that it is both analgesic and non-opioid," said senior study author Ru-Rong Ji, PhD, an anesthesiology and neurobiology researcher who directs the Duke Anesthesiology Center for Translational Pain Medicine. Even more encouraging: it prevented common side effects like constipation and buildup of tolerance, which often forces patients to need stronger and more frequent doses of opioids over time. SBI-810 is in early development, but Duke researchers are aiming for human trials soon and they've locked in multiple patents for the discovery. Drug overdose deaths are declining, but more than 80,000 Americans still die each year most often from opioids. Researchers said the drug could be a safer option for treating both short-term and chronic pain for those recovering from surgery or living with diabetic nerve pain. Using a method known as biased agonism, it switches on a specific signal -- β-arrestin-2 -- linked to pain relief, while avoiding other signals that can cause side effects or addiction. "It's a promising target for treating acute and chronic pain." SBI-810 effectively relieved pain from surgical incisions, bone fractures, and nerve injuries better than some existing painkillers. When injected in mice, it reduced signs of spontaneous discomfort, such as guarding and facial grimacing. Duke scientists compared SBI-810 to oliceridine, a newer type of opioid used in hospitals, and found SBI-810 worked better in some situations, with fewer signs of distress. Unlike opioids like morphine, SBI-810 didn't cause tolerance after repeated use. It also outperformed gabapentin, a common drug for nerve pain, and didn't cause sedation or memory problems, which are often seen with gabapentin. Researchers said the compound's dual action -- on both the peripheral and central nervous systems -- could offer a new kind of balance in pain medicine: powerful enough to work, yet specific enough to avoid harm. Additional Duke authors include first authors Ran Guo and Ouyang Chen; Sangsu Bang, Sharat Chandra, Yize Li, Gang Chen, Rou-Gang Xie, Wei He, Jing Xu, Richard Zhou, Shaoyong Song, Ivan Spasojevic, Marc G. Caron, William C. Wetsel and Lawrence S. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/military/aviation/a64794264/boeing-f47-fighter-jet-design-analysis/'>The Air Force's Secret New Fighter Jet Will Be Unstoppable—But What You've Seen Is a Clever Deception</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-19 17:43:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The F-47's illustrated features might have been intended as red herrings—or had their proportions exaggerated—all along. When the Air Force made a long anticipated announcement in March that it had finally selected Boeing to produce its secretive, new sixth-generation fighter jet known as the F-47, it also released two murky “graphical artist renderings” purporting to show a partial frontal view of the aircraft, which will combine next-gen stealth, sensor fusion, and long-range strike capabilities. “Its adaptability and modular design ensure seamless integration with emerging technologies, positioning it as a dominant platform for decades to come,” the U.S. Air Force said in a press release accompanying the images. In various configurations, canards can enhance lift, horizontal stabilization, and vertical axis pitch control. Further, they say, the renderings may even be U.S. propaganda intended to deceive foreign intelligence. Such measures are hardly unprecedented, as the service has obscured or misleadingly altered the initial imagery of all but one of its prior stealth aircraft. In fact, by mid-April, Air Force officials told Air & Space Forces Magazine the images don't “accurately portray the aircraft,” should be “taken with a large grain of salt,” “aren't giving anything away,” and “maybe” bear a “resemblance” to the final aircraft. This imparts greater lateral stability against rolls, at the expense of maneuverability, while downward-swept anhedral wings are less stable but more roll-conducive. Civilian aircraft typically favor stabilizing dihedrals, while jet fighters often sport flat or anhedral wings. But flight control technologies have evolved to make such a design more possible, while the demand for maximum maneuverability may have decreased. Indeed, the F-47 art's dihedral wings and shovel nose evoke an Area 51 black project aircraft developed by Boeing's Phantom Works division in the 1990s named the Bird of Prey due to its similarities to a Klingon spaceship from Star Trek. Designed to push the limits of stealth technology, the tailless Bird of Prey proved stable enough to pilot without computer assistance, and its innovations were subsequently integrated into Boeing's tailless X-45 stealth combat drone prototype. Despite this possible Boeing lineage for anhedral wings, and rationales favoring canards, the recent Air Force interview raises the possibility that these illustrated features might have been intended as red herrings—or had their proportions exaggerated—all along. That includes next-generation adaptive-cycle turbofan engines, for which two rival designs are being competed by Pratt & Whitney and General Electric; they feature a revolutionary capability to change the amount of air bypassing the engine's compressor midflight. But with adaptive cycle engines, you can have your cake and eat it, too. An F-47 could use high-bypass mode to efficiently transit the battlespace and lurk in waiting, then transition to low bypass mode when a combat situation demands maximum performance, including Mach 2+ speeds (twice the speed of sound). The F-47's engines will also be designed to generate much more electricity while compensating for the generated heat, likely by redirecting air flows for internal cooling. This will enable more powerful radar and computer processors which could improve ability to detect targets at great distances, or with very small radar cross-sections. Admittedly, drone-control hardware could be packed into an external pod adaptable to older aircraft, but the F-47's systems will be stealthily conformal, and its open-architecture computer will support drone command capabilities more effectively. Because F-47s may be heavily outnumbered in many combat scenarios, and can only carry so many missiles internally, drones will allow each F-47 to bring additional pieces to the air combat chessboard. Compared to the faster and more valuable F-47 “queen,” the less costly drones under its control could serve as pawns or bishops, providing extra missile shots, drawing away enemy attacks, and scanning or jamming from additional angles. Currently, the Air Force is evaluating the competing General Atomics YFQ-42 and Anduril YFQ-44 drones to serve as missile carriers, while a more expensive Increment 2 Loyal Wingman is being solicited. While there's still little certainty as to the F-47's true design geometry, we have a pretty good idea what the Air Force wants this jet to do and how it might go about accomplishing that. Until then, we must accept that the Air Force is not above teasing the public with less-than-faithful artwork of its secret aircraft. He holds a Master's degree from Georgetown University and served with the Peace Corps in China. This Ship Once Rammed Through Ice 40 Feet Thick Why the USS Gerald R. Ford Is Such a Badass Ship We're Already Fighting the World's First AI War Is the U.S. Ready to Fight a New Kind of War?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44221-025-00445-4'>Intensive irrigation buffers groundwater declines in key European breadbasket</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-19 10:47:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Water (2025)Cite this article Metrics details The Po Plain in northern Italy is a critical agricultural region and one of the largest water users in the European Union. Recent dry conditions have put future water resource availability into question. This study examines spatio-temporal variations in groundwater storage observed by the Gravity Recovery and Climate Experiment satellites and more than 1,000 groundwater wells from 2002 to 2022. We find that the rate of groundwater storage decline more than doubled from 2015 to 2022 as compared to the 2002–2022 rate. We also show that seasonal and long-term groundwater availability is strongly influenced by irrigation activities. Groundwater storage in irrigated areas is highly correlated to snow accumulation in the Alps and shows more stability as compared to non-irrigated areas, which experience dramatic declines during drought years. This indicates that inefficient irrigation practice, using water largely supplied by snowmelt, recharges underground aquifers and helps maintain high water tables, making aquifers underlying irrigated farmland resilient to the negative consequences of drought. These findings can help guide climate-driven adaptations to irrigation systems that account for the impact on groundwater recharge. This is a preview of subscription content, access via your institution Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout GRACE JPL mascon solutions can be downloaded from https://podaac.jpl.nasa.gov/dataset/TELLUS_GRAC-GRFO_MASCON_CRI_GRID_RL06.1_V3. GLDAS models VIC, Noah and CLSM can be downloaded from https://disc.gsfc.nasa.gov/datasets?keywords=GLDAS. GLEAM4 can be downloaded via SFTP at https://www.gleam.eu. ERA5-Land can be downloaded from https://cds.climate.copernicus.eu/cdsapp#!/dataset/10.24381/cds.68d2bb30?tab=form. SPEI can be downloaded from https://spei.csic.es/map/. Monthly Po River discharge data provided by D. Zanchetin are available via Zenodo at https://doi.org/10.5281/zenodo.7225699 (ref. Monthly precipitation is available from NASA's IMERG project at https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGM_07/summary?keywords=%22IMERG%20final%22. SWE is provided by the Meteorological Reanalysis Italian Dataset at https://merida.rse-web.it. Monthly seasonal mean groundwater levels and trends for 1,024 groundwater wells across the northern Italian plains are available via Zenodo at https://doi.org/10.5281/zenodo.14013762 (ref. All data were processed in MATLAB. Wavelet software was provided by C. Torrence and G. Compo and is available at http://atoc.colorado.edu/research/wavelets/. Immerzeel, W. W. et al. Importance and vulnerability of the world's water towers. Google Scholar Siebert, S. et al. A global data set of the extent of irrigated land from 1900 to 2005. Google Scholar Siebert, S. et al. Groundwater use for irrigation—a global inventory. Google Scholar Mehta, P. et al. Half of twenty-first century global irrigation expansion has been in water-stressed regions. Google Scholar Scanlon, B. R. et al. Global water resources and the role of groundwater in a resilient water future. Earth Environ. Google Scholar Koster, D. et al. Regions of strong coupling between soil moisture and precipitation. Google Scholar McDermid, S. et al. Irrigation in the Earth system. Earth Environ. Google Scholar Qin, Y. et al. Agricultural risks from changing snowmelt. Google Scholar Borin, M. A wise irrigation to contribute to integrated water resource management. Google Scholar Zucaro, R. Atlante nazionale dell'irrigazione ‘National atlas of irrigation' (INEA, 2011). Colombo, N. et al. Long-term trend of snow water equivalent in the Italian Alps. Google Scholar Montanari, A. et al. Why the 2022 Po River drought is the worst in the past two centuries. Google Scholar Avanzi, F. et al. Winter snow deficit was a harbinger of summer 2022 socio-hydrologic drought in the Po Basin, Italy. Earth Environ. Google Scholar IPCC Climate Change 2021: The Physical Science Basis (Cambridge Univ. Spinoni, J. et al. Future global meteorological drought hot spots: a study based on CORDEX data. Google Scholar Qin, Y. et al. Snowmelt risk telecouplings for irrigated agriculture. Google Scholar Brussolo, E. et al. Aquifer recharge in the Piedmont Alpine zone: historical trends and future scenarios. Google Scholar ISTAT Utilizzo e qualità della risorsa idrica in Italia (Istat, 2019); https://www.istat.it/produzione-editoriale/utilizzo-e-qualita-della-risorsa-idrica-in-italia/ Rossi, M., Donnini, M. & Beddini, G. Nationwide groundwater recharge evaluation for a sustainable water withdrawal over Italy. Lasagna, M., Mancini, S. & De Luca, D. A. Aquifer protection from overexploitation: example of actions and mitigation activities used in the Maggiore Valley (Asti Province, NW Italy). Sartirana, D. et al. Quantifying groundwater infiltrations into subway lines and underground car parks using MODFLOW-USG. Bellafiore, D. et al. Saltwater intrusion in a Mediterranean delta under a changing climate. Teatini, P., Ferronato, M., Gambolati, G., Bertoni, W. & Gonella, M. A century of land subsidence in Ravenna, Italy. Teatini, P., Ferronato, M., Gambolati, G. & Gonella, M. Groundwater pumping and land subsidence in the Emilia-Romagna coastland, Italy: modeling the past occurrence and the future trend. Carminati, E. & Martinelli, G. Subsidence rates in the Po Plain, northern Italy: the relative impact of natural and anthropogenic causation. Rotiroti, M. et al. The effects of irrigation on groundwater quality and quantity in a human-modified hydro-system: the Oglio River basin, Po Plain, northern Italy. Bonaldo, D. et al. The summer 2022 drought: a taste of future climate for the Po valley (Italy)? Colombo, N. et al. Unprecedented snow-drought conditions in the Italian Alps during the early 2020s. van Tiel, M. et al. Cryosphere–groundwater connectivity is a missing link in the mountain water cycle. Van Lanen, H. A. J., Wanders, N., Tallaksen, L. M. & Van Loon, A. F. Hydrological drought across the world: impact of climate and physical catchment structure. Tapley, B. D., Bettadpur, S., Ries, J. C., Thompson, P. F. & Watkins, M. M. GRACE measurements of mass variability in the Earth system. Rodell, M., Velicogna, I. & Famiglietti, J. S. Satellite-based estimates of groundwater depletion in India. Torrence, C. & Compo, G. P. A practical guide to wavelet analysis. Vicente-Serrano, S. M., Beguería, S. & López-Moreno, J. I. The Standardized Precipitation Evapotranspiration Index—SPEI. Beguería, S., Vicente-Serrano, S. M., Reig-Garcia, F., & Latorre Garcés, B. SPEIbase v.2.9 DIGITAL.CSIC v. 2.9 (DIGITAL.CSIC, 2023). Giuliano, G. Ground water in the Po basin: some problems relating to its use and protection. Atlante dell'agricoltura italiana - 6° Censimento generale dell'agricoltura [Atlas of Italian Agriculture - 6th General Census of Agriculture] (Istat, 2014). Humphrey, V., Rodell, M. & Eicker, A. Using satellite-based terrestrial water storage data: a review. Giustini, F., Brilli, M. & Patera, A. Mapping oxygen stable isotopes of precipitation in Italy. Wilson, J. L. and Guan, H. in Groundwater Recharge in a Desert Environment: The Southwestern United States (eds Hogan, J. F. et al.), 113–137 (AGU, 2004); https://doi.org/10.1029/009WSA08 Masetti, M. et al. Impact of a storm-water infiltration basin on the recharge dynamics in a highly permeable aquifer. Ministero delle Politiche Agricole Alimentari e Forestali. L'agricoltura nel distretto idrografico padano: Contributo tematico al Piano di Gestione del Distretto Idrografico Padano (Autorità di bacino del fiume Po, 2010). Directive 2000/60/EC of the European Parliament and of the Council Establishing a Framework for Community Action in the Field of Water Policy (European Council, 2000). Grafton, R. Q. et al. The paradox of irrigation efficiency. & Liesch, T. Quantification and possible causes of declining groundwater resources in the Euro-Mediterranean region from 2003 to 2020. Masseroni, D. et al. The 63-year changes in annual streamflow volumes across Europe with a focus on the Mediterranean basin. Cowherd, M., Leung, R. & Girotto, M. Evolution of global snow drought characteristics from 1850 to 2100. Watkins, M. M., Wiese, D. N., Yuan, D.-N., Boening, C. & Landerer, F. W. Improved methods for observing Earth's time variable mass distribution with GRACE using spherical cap mascons. Res: Solid Earth 120, 2648–2671 (2015). Wiese, D. N., Landerer, F. W. & Watkins, M. M. Quantifying and reducing leakage errors in the JPL RL05M GRACE Mascon solution. Wiese, D. N., Yuan, D.-N., Boening, C., Landerer, F. W. & Watkins, M. M. JPL GRACE Mascon Ocean, Ice, and Hydrology Equivalent Water Height CRI Filtered. Landerer, F. W. et al. Extending the global mass change data record: GRACE follow‐on instrument and science data performance. Li, B., Beaudoing, H. and Rodell, M. NASA/GSFC/HSL. GLDAS Catchment Land Surface Model L4 Monthly 1.0 x 1.0 Degree V2.1 (GES DISC, 2020); https://doi.org/10.5067/FOUXNLXFAZNY Rodell, M. et al. The global land data assimilation system. Beaudoing, H. and Rodell, M. NASA/GSFC/HSL. GLDAS Noah Land Surface Model L4 Monthly 1.0 x 1.0 Degree V2.1 (GES DISC, 2020); https://doi.org/10.5067/LWTYSMP3VM5Z Muñoz-Sabater, J. et al. ERA5-Land: a state-of-the-art global reanalysis dataset for land applications. Miralles, D. G. et al. Global land-surface evaporation estimated from satellite-based observations. Beaudoing, H. and Rodell, M. NASA/GSFC/HSL. GLDAS VIC Land Surface Model L4 Monthly 1.0 x 1.0 Degree V2.1 (GES DISC, 2020); https://doi.org/10.5067/VWTH7S6218SG Woessner, W. W. & Poeter, E. P. Hydrogeologic Properties of Earth Materials and Principles of Groundwater Flow (The Groundwater Project, 2020). Bonomi, T. Database development and 3D modeling of textural variations in heterogeneous, unconsolidated aquifer media: application to the Milan plain. Bonomi, T., Fumagalli, L., Rotiroti, M., Bellani, A. The hydrogeological well database TANGRAM©: a tool for data processing to support groundwater assessment. ISPRA Dati provenienti dall'Archivio nazionale delle indagini nel sottosuolo ai sensi della Legge 464/84, ISPRA - Dipartimento per il Servizio Geologico d'Italia – Servizio GEO-APP (ISPRA, 2024). Smith, R. Aquifer stress history contributes to historic shift in subsidence in the San Joaquin Valley, California. Zanchetin, D. Monthly discharge of Po River at Pontelagoscuro. Bonanno, R., Lacavalla, M. & Sperati, S. A new high-resolution meteorological reanalysis Italian dataset: MERIDA. Martinelli, G. et al. Nitrate sources, accumulation and reduction in groundwater from northern Italy: insights provided by a nitrate and boron isotopic database. Carlson, G., Girotto, M., Wilder, A. & Massari, C. Groundwater level seasonal behavior and trends in wells across the northern Italian plains. Download references We thank the following monitoring agencies: Agenzia Regionale per la Protezione dell'Ambiente (ARPA) Lombardia, L'Agenzia regionale per la prevenzione, l'ambiente e l'energia dell'Emilia Romagna (ARPAE), Agenzia Regionale per la Prevenzione e Protezione Ambientale del Veneto (ARPAV) and Agenzia Regionale per la Protezione Ambientale (ARPA) Piemonte for providing the in situ groundwater observation well time series and lake-level change time series. NASA grants 80NSSC20K1240 and 80NSSC22K1831 and the Hellman Fellows Fund supported G.C. and M.G., European Space Agency (ESA) grant 4000136272/21/I-EF supported C.M. and the Historically Black Colleges and Universities-Berkeley Environmental Scholars for Change Program supported D.W. Department of Environmental Science, Policy and Management, University of California, Berkeley, Berkeley, CA, USA Grace Carlson, Andrew Wilder, Destinee Whitaker & Manuela Girotto Research Institute for Geo-Hydrological Protection, National Research Council CNR, Perugia, Italy Department of Earth and Environmental Sciences, University of Milano-Bicocca, Milano, Italy Marco Rotiroti & Tullia Bonomi Water Research Institute, National Research Council CNR, Montelibretti, Italy Spelman College, Atlanta, GA, USA Destinee Whitaker West Atlanta Watershed Alliance, Atlanta, GA, USA Destinee Whitaker You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the experiments. performed the experiments. and D.W. analysed the data. contributed materials and analysis tools. led the writing of the paper with all authors making contributions to the writing and editing process. Correspondence to Grace Carlson. The authors declare no competing interests. Nature Water thanks Jenna Dohman, David Ketchum and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The High Plain and Low Plain (dark blue and orange, respectively) correspond to ∆GWS time series in Fig. Lombardia irrigated and Lombardia not irrigated (yellow and purple, respectively) correspond to the time series in Fig. Veneto and Piemonte (green and purple, respectively) correspond to the time series in Fig. 5c and Emilia Romagna (red) correspnds to the time series in Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Carlson, G., Massari, C., Rotiroti, M. et al. Intensive irrigation buffers groundwater declines in key European breadbasket. Nat Water  (2025). Download citation Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Water (Nat Water) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01531-7'>Structure, assembly and inhibition of the Toxoplasma gondii respiratory chain supercomplex</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-19 09:48:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The apicomplexan mitochondrial electron transport chain is essential for parasite survival and displays a divergent subunit composition. Here we report cryo-electron microscopy structures of an apicomplexan III2–IV supercomplex and of the drug target complex III2. The supercomplex structure reveals how clade-specific subunits form an apicomplexan-conserved III2–IV interface with a unique, kinked architecture, suggesting that supercomplexes evolved independently in different eukaryotic lineages. A knockout resulting in supercomplex disassembly challenges the proposed role of III2–IV in electron transfer efficiency as suggested for mammals. Nevertheless, knockout analysis indicates that III2–IV is critical for parasite fitness. They were also inhibited with endochin-like quinolone (ELQ)-300, an inhibitor in late-stage preclinical development. Notably, in the apicomplexan binding site, ELQ-300 is flipped compared with related compounds in the mammalian enzyme. On the basis of the binding modes and parasite-specific interactions discovered, we designed more potent ELQs with subnanomolar activity against T. gondii. Our findings reveal critical evolutionary differences in the role of supercomplexes in mitochondrial biology and provide insight into cytochrome b inhibition, informing future drug discovery. The mitochondrial electron transport chain (mETC) is essential for nearly all organisms from the divergent domain of eukaryotes, yet our knowledge of how it works is primarily informed by studies in yeast and mammals. The occurrence of respiratory supercomplexes has been known for decades1 and recently, insight from structural studies exposed the interactions that mediate their formation. However, whether supercomplex formation is functionally relevant for electron transfer or whether this reoccurring arrangement confers a different advantage, such as increased complex stability, is still a subject of ongoing investigations2,3,4,5,6. This study provides an evolutionary perspective on the currently proposed supercomplex functions through the structural and functional characterization of a protozoan parasite supercomplex. Apicomplexans are parasites that belong to the myzozoan clade of eukaryotes. These parasites were recently shown to have expanded mETC complexes with numerous clade-specific subunits7,8; however, their role remains unknown. Likewise, in the apicomplexan mETC, complex III2 (CIII) is the primary target for antiparasitic drugs. However, the molecular basis for this sensitivity is not fully understood owing to the lack of a parasite CIII structure. We address both of these questions through structural analyses and genetic studies. The Toxoplasma gondii respiratory supercomplex was purified from the rapidly proliferating tachyzoite stage. Using single-particle cryo-electron microscopy (cryo-EM), we determined the structure of the III2–IV supercomplex in the presence of atovaquone and endochin-like quinolone (ELQ)-300 (10 µM each) at 2.8 Å resolution, allowing atomic model construction (Fig. 1a,b, Table 1 and Extended Data Fig. The 960 kDa supercomplex consists of 37 subunits and contains the conserved reaction centers and cofactors (Supplementary Table 1). In total, 13 subunits are apicomplexan conserved, of which 2 are found in CIII (TgQCR12 and 13). Unlike previously predicted8,9, the complex contains a newly assigned homolog of QCR10, which could only be identified via structural similarity. Complex IV (CIV) is greatly augmented, containing 11 apicomplexan-conserved subunits and substantial clade-specific subunit extensions (Fig. 1c and Supplementary Table 1), giving it a molecular mass of 405 kDa, much larger than the 207 kDa mammalian homolog10. For the 11 apicomplexan-conserved subunits of T. gondii CIV (ApiCox7, 10, 13, 15, 16, 18, 19, 20, 22, 24 and 30), we extend an existing nomenclature indicating the approximate molecular weight11 (Methods and Supplementary Table 1). The increased molecular masses of CIII and CIV are in line with the larger complex II and adenosine triphosphate (ATP) synthase assemblies found in Apicomplexa12,13. These findings confirm that the apicomplexan oxidative phosphorylation complexes display augmented subunit compositions, raising questions about the role of new subunits and extensions, as well as their evolution. An overall reduced protein hydrophobicity resulting from a splitting of structural elements into several subunits may enable mitochondrial protein targeting following gene transfer from the mitochondrial genome to the nuclear genome13,14. In our structure, T. gondii Cox2 is split into three proteins, each providing canonical helices but displaying reduced overall hydrophobicity compared with canonical Cox2 (Extended Data Fig. 2), providing support that this mechanism enabled the marked mitochondrial genome reduction in apicomplexans to just three protein-coding genes. a, The side view (left) and top view (right) of the III2–IV composite map containing 13 clade-specific subunits. The transmembrane region of CIV is kinked by 21° with respect to CIII. b, An atomic model of III2–IV. c, Conserved and clade-specific structural elements. Supernumerary subunits may also mediate higher-order oxidative phosphorylation assemblies that suit clade-specific mitochondrial functions and contribute to membrane curvature induction. Our T. gondii III2–IV structure reveals an unusual supercomplex architecture, in which the transmembrane region of CIV is tilted against the membrane plane of CIII by ~20° (Fig. This finding is further supported by the positions of lipid-binding sites within the curved membrane region (Extended Data Fig. By contrast, the transmembrane region of the mammalian and yeast supercomplex homologs are flat (Extended Data Fig. The kink is probably induced by the presence of clade-specific subunits TgQCR12 in CIII and ApiCox7 in CIV, which act as spacers in the transmembrane and matrix regions and would clash in a flat architecture, as present in the mammalian supercomplex (Extended Data Fig. Recently, kinked interfaces have been observed between complexes I and III2 in ciliate and plant supercomplexes15,16 and the ciliate I–II–III2–IV2 structure was shown to contribute to membrane curvature induction. In T. gondii, pentagonal pyramid arrays of ATP synthase generate a distinct bulbous cristae morphology that is characteristic for mitochondria of apicomplexan parasites13. Using electron cryo-tomography and subtomogram averaging, we identified the III2–IV supercomplex in situ and revealed its localization in the curved, lateral regions of the cristae membranes (Fig. This architecture differs markedly from the lamellar cristae found in mammals, where respiratory supercomplexes reside in the flat membrane regions17 (Fig. a, A slice of an electron cryo-tomogram of mitochondrial membranes from T. gondii. ATP synthase (yellow) and supercomplexes (red) are indicated. b, The subtomogram average map (gray) is consistent with a III2–IV1 arrangement. The individually fitted CIII dimer (blue) and CIV monomer structures (yellow) are from S. cerevisiae (PDB 6T15). c, A three-dimensional close-up of a showing apical ATP synthase and lateral supercomplex. The apices of the mitochondrial membranes (blue) are occupied by ATP synthase pyramids (yellow, described in ref. 13), with supercomplexes (red) in the flatter lateral regions. Both structures were obtained by subtomogram averaging. d, The arrangement of ATP synthase and supercomplexes in cristae of mammals and T. gondii. The rim of flat cristae in mammals are in line with ATP synthase rows (PDB 7ajb), whereas the apices of bulbous T. gondii cristae are shaped by ATP synthase pyramids (PDB 6TML). Mammalian supercomplexes (PDB 7o37) are found in the lateral, flat membrane regions, whereas the kinked III2–IV T. gondii supercomplex (this study) is accommodated by the curved, lateral regions. The matrix is shown in yellow. The cristae lumen and intermembrane space are shown in blue. e, A schematic highlighting the resulting cristae morphologies, adapted from ref. 43. f, Overlay of supercomplex structures aligned on CIV highlights different architectures, as revealed by differing positions of CIII from mammals (Mus musculus, PDB 7o37 mature supercomplex), yeast (S. cerevisiae, PDB 6giq) and plant (Vigna radiata, PDB 7jrp). The mammalian and T. gondii CIII homologs bind to opposite sides of CIV. g, Comparison of the T. gondii (this study) and M. musculus III2–IV (PDB 7o37) showing the rotation of CIV relative to CIII, thereby placing different CIV subunits (COX6B and COX7c) at the interface with QCR6, or at the distal end of the supercomplex. h, Open-book view of the III2–IV interface with lumenal contacts with respective subunits highlighted by spheres. i, Top view of the III2–IV interface. The T. gondii CIV is unusual in its augmented subunit composition and orientation within the supercomplex. The largest clade-specific subunit, ApiCox13, contains a CDGSH-type iron–zinc finger domain on the matrix side, with some residues contributing to cardiolipin binding (Extended Data Fig. As this protein fold is known to have evolved varying metal binding capacities (Zn2+ or Fe2S2)18, we performed sequence alignments and structure prediction of ApiCox13 homologs. This analysis indicated that Fe2S2 sites are well conserved in Apicomplexa, while homologs from marine species in related phyla instead contain tetrahedrally coordinated zinc (Extended Data Fig. These findings are in line with the observed essentiality of Fe2S2 binding capability for CIV integrity and parasite survival in T. gondii19. The architecture of the T. gondii supercomplex is unique among all previously described respiratory supercomplexes and this is mediated by a series of clade-specific subunits and extensions. When compared with the mature mammalian supercomplex III2–IV, the T. gondii CIV is found in a similar position relative to CIII, but rotated by ~180° (Extended Data Fig. 5a), therefore interacting with a different set of subunits. This leads to a unique relative positioning of T. gondii complexes III and IV, compared with mammalian, yeast and plant supercomplexes20,21,22 (Fig. 5a), indicating that non-opisthokont lineages may represent most of respiratory supercomplex diversity. Whereas the mammalian assembly factor SCAF1 ties CIV to CIII in proximity to QCR6, TgQCR6 is facing toward NDUFA4 and the clade-conserved subunits, ApiCox10 and 15, located on the opposite side of CIV, thus generating a unique interface and III2–IV supercomplex architecture (Fig. Furthermore, owing to the kinked architecture of the T. gondii supercomplex, CIV is associated to CIII only via the conserved TgQCR6, forming lumenal interactions with the four CIV subunits COX6B, ApiCox10, NDUFA4 and ApiCox15 (spheres in Fig. The interaction with ApiCox10 is mediated via an apicomplexan-conserved N-terminal extension (residues 2–23) of TgQCR6, which forms hydrophobic interactions with a helix hairpin (H3–H4) of ApiCox10 (Extended Data Fig. ApiCox15 contains a single TM-helix and a 38-residue lumenal region that contacts the TgQCR6 helix hairpin. Finally, unlike its mammalian homolog, the apicomplexan NDUFA4 C-terminus contains a structured horizontal helix that interacts with the TgQCR6 hairpin (Fig. 2g,h,i) on the lumenal side. The clade-specific interacting subunits and phylum-specific protein extensions are conserved in the malaria-causing Plasmodium falciparum (Extended Data Fig. 5d,e) and other apicomplexans, indicating that the newly described III2–IV interface and unusual supercomplex architecture are probably conserved in apicomplexan parasites. Given the observed interactions of the clade-specific ApiCox10 and ApiCox15 with TgQCR6 at the supercomplex interface, we hypothesized that deletion of either of those proteins would result in supercomplex disassembly, allowing us to address the role of supercomplex formation. We attempted to generate ApiCox10 and ApiCox15 individual knockouts (KOs) using a clustered regularly interspaced short palindromic repeats (CRISPR)–Cas9 system. Five independent transfections and a screen of 50 clones failed to isolate an ApiCox15-KO mutant suggesting that this gene is refractory for full deletion. On the other hand, we were able to generate an ApiCox10-KO in two separate background lines, one where the CIV subunit Cox2a is endogenously tagged with a C-terminal hemagglutinin (HA) tag, and another where the CIII subunit QCR2 is endogenously HA tagged8. Additionally, we generated a complemented line where a Ty-tagged ApiCox10 is reintroduced (Extended Data Fig. Native gel electrophoresis analysis demonstrated the loss of III2–IV supercomplexes in both ApiCox10-KO lines (Fig. 3a–c), and this was further confirmed via proteomics analysis of Cox2a-HA immunoprecipitation (Extended Data Fig. Importantly, the individual CIV and CIII2 remain intact (Fig. 6i), confirming the unique interface architecture of the apicomplexan III2–IV supercomplex and its dependence on a clade-specific subunit for its assembly. a, Native PAGE analysis of Rieske-HA (CIII2 tagged line), Cox2a-HA (CIV tagged line) and ApiCox10-KO in the Cox2a-HA background (ApiCox10-KO/Cox2a-HA). Total lysates were treated with digitonin and separated by BN–PAGE, followed by immunoblot analysis with anti-HA antibodies, as well as anti-TOM40 as a loading control. Positions of complexes are indicated. b, Native PAGE analysis of QCR2-HA (CIII2 tagged line) and ApiCox10-KO in the QCR2-HA background (ApiCox10-KO/QCR2-HA) treated with digitonin, as in a. Samples were also separated by SDS–PAGE and immunoblot analysis with anti-TOM40 antibodies performed as a loading control. c, Native PAGE analysis of Cox2a-HA ApiCox10-KO/Cox2a-HA treated with digitonin, followed by cytochrome c DAB staining to visualize CIV activity. d–f, Native PAGE analysis of the lines in a (d), b (e) and c (f) extracted using β-DDM. g, Measurement of mitochondrial membrane potential using JC-1 dye via flow cytometry analysis. (i): T. gondii stained with the dye JC-1 indicated that both lines possess a mitochondrial membrane potential that is sensitive to the ionophore valinomycin. (ii): quantification of mitochondrial membrane potential by population that is positive for JC-1 staining. Graphs show mean ± s.d., from eight independent experiments. One-way ANOVA followed by Tukey's multiple pairwise comparisons was performed, and P values from relevant pairs are displayed. ****P < 0.0001. h, Extracellular flux analysis of (i) basal mitochondrial OCR, (ii) maximal mitochondrial OCR and (iii) ECAR of parental and ApiCox10-KO parasites. P value was determined from a two-tailed unpaired Student's t-test. i, A mixed culture growth competition assay of ApiCox10-KO or parental mNEON fluorescent parasites with tdTomato parasites. Relative abundance (compared with passage 0) of ApiCox10-KO or parental parasites after six passages. P value was determined using a two-tailed one-sample t-test comparing values to passage 0, ** P = 0.0011. The supercomplex structure raises the question of whether the observed kinked architecture contributes to the induction of membrane curvature or whether it has evolved to allow accommodation in the unique bulbous mitochondrial cristae found in apicomplexans. Analysis of mitochondria in electron microscopy thin sections of the ApiCox10-KO line pointed to unchanged cristae morphology and density, in support of the latter scenario (Extended Data Fig. Likewise, ApiCox10-KO had no impact on the mitochondrial redox state (Extended Data Fig. Furthermore, examination of the mitochondrial membrane potential and oxygen consumption rate (OCR) in extracellular ApiCox10-KO, as proxy measurements for respiratory activity in tachyzoites, revealed no alteration compared to the parental line (Fig. 3g,h and Extended Data Fig. This finding is in stark contrast to Saccharomyces cerevisiae, where supercomplex disruption impairs electron transfer efficiency in a cytochrome c level-dependent manner4. Interestingly, analysis of the surface charges on the T. gondii supercomplex shows a continuous negatively charged patch on the lumen region (Extended Data Fig. 6n), which is conserved in the yeast supercomplex, where it has been suggested to facilitate intracomplex cytochrome c transfer by two-dimensional (2D) diffusion23. Together, our findings suggest that cytochrome c diffusion does not limit respiration in T. gondii tachyzoites. Finally, we tested whether the inability to form respiratory supercomplexes inflicts a fitness cost on the parasite. Analysis of parasite replication revealed a potential delay in reaching eight parasites per vacuole (Extended Data Fig. We thus created an ApiCox10-KO in a fluorescent background line and performed a growth competition assay. ApiCox10-KO parasites grown in the same culture as wild-type parasites were consistently outcompeted, indicating decreased fitness (Fig. Overall, parasites that are unable to form supercomplexes have no observable defects in respiratory activity, but display a fitness penalty, indicating that supercomplexes play an important fitness-conferring role in mitochondrial function that is not directly related to primary catalytic function. The finding that clade-specific subunits, which mediate supercomplex formation are important for competitive fitness may indicate that they convey increased stability or aid localization of the kinked supercomplexes to the uniquely shaped cristae (as opposed to the inner boundary membrane) to enable efficient formation of a membrane potential. The fitness defects associated with loss of supercomplexes in a tachyzoite competition assay suggest that supercomplex functions may become more relevant and observable in vivo or in different stages of the T. gondii life cycle. CIII (cytochrome bc1 complex) plays a crucial role in electron transfer and proton pumping. The pumping of protons into the cristae lumen occurs via the Q-cycle, which involves sequential ubiquinol oxidation and ubiquinone reduction. Both reactions occur in specific quinone binding sites within the cytochrome b (Cyt-b) subunit, called Qo (oxidation) and Qi (reduction), which are essential for catalytic activity. Atovaquone is a Food and Drug Administration-approved antimalarial drug that competitively inhibits the Cyt-b Qo site and is also effective against toxoplasmosis24 (EC50 of 138 nM (ref. However, resistance to atovaquone has led to the development of novel inhibitors, such as ELQs, which target the Qi site25. The basis of selectivity of inhibition at either site is not fully understood. The current understanding of species-specific Qo site inhibition by atovaquone is deduced from homology models of CIII structures of other species26, but experimental structures from apicomplexans have not been determined. Moreover, ELQ-300 is a Qi site inhibitor undergoing late-stage preclinical testing as a new antimalarial drug in the Medicines for Malaria portfolio, which is effective against blood, liver and mosquito-stage malaria, and also displays efficacy against T. gondii and other apicomplexan pathogens27,28. When used in combination therapy with atovaquone, ELQ-300 is highly synergistic in a murine malaria model29. However, the atovaquone/ELQ-300-bound structure of CIII has not been reported, and the structural basis for parasite-specific Cyt-b inhibition is not known. The cryo-EM map shows both ELQ-300 (Qi) and atovaquone (Qo) bound, allowing insight into the binding mode of the two inhibitors (Fig. 4a,b and Extended Data Fig. As our mitochondria preparations of the obligate intracellular parasite T. gondii also contained substantial amounts of mitochondria from the host cells (‘Vero' cells from the African green monkey, Cholorocebus sabaeus), the final cryo-EM sample contained a mixture of mitochondrial complexes from both species, which we classified computationally (Extended Data Fig. From the same cryo-EM dataset, we thus also determined the structure of CIII within the co-purified Cholorocebus sabaeus I–III2–IV respirasome at 2.8 Å resolution (with atovaquone bound in Qi and Qo sites, see below), allowing direct comparison of architecture and inhibitor binding between parasite and host (Extended Data Table 1 and Extended Data Fig. In our structure, TgCyt-b displays the conserved fold including eight transmembrane helices (A–H) and conserved redox centers (Extended Data Fig. Notably, helices F, G and H are substantially remodeled. In opisthokonts (which includes yeast and mammals), helix F is curved and contains a 310 helix that introduces a bend in the transmembrane segment and is followed by a conserved proline residue (P305 in humans and C. sabaeus) that acts as a helix breaker. This conserved proline residue is missing in apicomplexan Cyt-b homologs, resulting in a straight α-helical helix F. Likewise, the T. gondii FG-loop (N306–W324, human 309–315) is remodeled, followed by a shortened hairpin of helices G and H. These features affect the positioning of residues that interact with the chlorophenyl moiety of atovaquone. a, View of the Qo site of T. gondii occupied by atovaquone, which prevents electron transfer to heme bL and Fe2S2 in the lumenal domain of the Rieske subunit, which occupies the b-state. Residues responsible for apicomplexan-specific atovaquone sensitivity are shown in red. b, A ligand diagram of atovaquone interactions in the T. gondii Qo site. d, A schematic of atovaquone interactions in the C. sabaeus Qo site. The binding pocket for the hydroxynaphthoquinone group of the inhibitor is conserved between C. sabaeus and T. gondii, with our structure revealing interactions with residues F128, Y131, P265, M138 and I263 (Fig. 4), of which the latter two were previously found mutated in atovaquone-resistant T. gondii lines30. The difference in Qo site affinity arises from a group of parasite-specific residues (I124, Y272 and F289) that interact with the chlorophenyl moiety of atovaquone. Among these, Y272 (T. gondii numbering) on the EF-helix is occupied by an alanine in higher primates (A277 in humans and C. sabaeus). In the inhibited Cyt-b structure from S. cerevisiae, the chlorophenyl and cyclohexane rings of atovaquone are modeled in a near parallel configuration26. Our structure reveals that in T. gondii, the two moieties are instead oriented orthogonally, leading to a pi-stacking interaction of the chlorophenyl group with Y272 (Fig. The aromatic character of this residue is also conserved in Plasmodium species causing malaria in humans (F267). Indeed, the single mutations F267V and F267I convey atovaquone resistance in P. falciparum and P. yoelii, respectively31,32, indicating that aromatic interactions, rather than unspecific hydrophobic contacts, are required for atovaquone binding. Our structure reveals that Y272 acts to position the chlorophenyl group of atovaquone to convey parasite-specific ligand interactions. Owing to the curved helix F structure, in the S. cerevisiae Cyt-b, the I299 sidechain is partially inserted between the respective aromatic residue (F278) and the chlorophenyl group, preventing a stacking interaction as observed in our T. gondii structure (Extended Data Fig. This may explain the potential difference in sensitivity between yeast26,33 and P. falciparum27,34. Notably, the binding of atovaquone to the Qo site generates an induced fit. Comparison to our T. gondii structure with unoccupied Qo site (see below), revealed that the EF loop and cd1 helix undergo movements to widen the pocket. In this process, I263 on the EF loop, which would result in a clash with atovaquone in the unoccupied state, moves by 2 Å to accommodate the ligand (Supplementary Movie 1). While the atovaquone EC50 value is in the low nanomolar range for both P. falciparum and T. gondii cytochrome bc135,36, the EC50 for the mammalian homolog is more than 100 times higher27 (EC50 T. gondii of 138 nM, 38 µM human foreskin fibroblasts (HFF)25). Remarkably, we also found atovaquone bound not only in the Qo site, but also the Qi site of the C. sabaeus Cyt-b, which is part of the I–III2–IV supercomplex. Our structure indicates that in the dual-site inhibited mammalian Cyt-b, the atovaquone molecules of the Qi and Qo sites probably adopt two different tautomers to enable the formation of hydrogen bond networks in the two quinone binding sites (Extended Data Fig. This observed dual atovaquone inhibition of the mammalian complex (Qi + Qo) indicates that ELQ-300 binds the mammalian Qi site very weakly or not at all. Thus, by co-determining the structures of both the mammalian and T. gondii CIII in the presence of equimolar atovaquone/ELQ-300 concentrations (10 µM) from the same heterogeneous sample and revealing their different drug binding, we provide structural evidence for the high parasite selectivity of ELQ-300 as a Qi-site inhibitor27. To reveal its binding mechanism at high resolution, we affinity-purified the T. gondii CIII embedded in amphipols via the 3xFLAG-tagged Rieske subunit (Extended Data Fig. 9a–e), inhibited by ELQ-300 only, and determined its cryo-EM structure to 1.8 Å resolution (Extended Data Fig. Surprisingly, the C2-symmetric CIII structure contains two bound ELQ-300 molecules per Cyt-b monomer, one at the Qi site and, unexpectedly, another in the central cavity of the Qo channel (Fig. In the Qi pocket, ELQ-300 displays an unexpected binding mode, which is well supported by our cryo-EM density (Fig. The 4-(1H)-quinolone group is pinned by hydrogen bonds with H197 and D223, interacting with ketone and amino groups of the ligand at N–O distances of 3.0 Å, respectively (Fig. The ketone group of ELQ-300 also interacts with the heme bH propionate group via an ordered water (Fig. This binding mode differs markedly from the orientation of other 3-diaryl 4-(1H)-quinolones in the Qi site of the bovine enzyme, which was found to be rotated 180° (ref. Owing to the fixed orientation of the quinolone group in our T. gondii structure, the diarylether group in position-2 extends toward helix D (on the matrix side), which is markedly different from the binding pose observed in the bovine homolog, in which quinolones with both 2- or 3-diaryl-groups extend in the opposite direction, toward helix A (in the lumen side; Fig. 5c,e and Extended Data Fig. This finding is similar to the binding mode of atovaquone in the Qi site observed in our C. sabaeus structure, in which the chlorophenyl-cyclohexyl group also points toward helix A, indicating a similar binding mode to that seen for other inhibitors of the mammalian Qi site26,29,30. In our T. gondii structure, the orientation of the first aryl group is determined by the adjacent I193 on helix D, leading to a 63° angle between the aryl and quinolone planes. The peripheral trifluoro-methoxy-phenoxy group displays flexibility and adopts two distinct conformations (Fig. Conformer A makes contacts along helix D with I189, V190 and I193, whereas through a rotation around the ether bond, conformer B extends toward helix a to interact with a set of hydrophobic residues (Extended Data Fig. The hydrophobic character of the Qi site entrance is probably enhanced by the specific recruitment of an adjacent cardiolipin molecule (Extended Data Fig. a, View of ELQ-300 bound in the Qi site of T. gondii, showing both conformer A and B. b, A close-up view of conformer B of ELQ-300 bound in the Qi site. Parasite-specific aqueous pocket denoted by gray-dashed line. c, TgCyt-b with heme and ELQ-300 binding sites. Helices A–H are connected by loops including AB, CD and EF, with interspersed horizontal helices a, ab, cd1, cd2 and ef. The overlapping atovaquone site (Qo) is shown in transparent orange. e, Ligand diagram of ELQ-300 in the Qi site highlighting hydrogen bonds and hydrophobic interactions. f, Name, formula and EC50 values (nM) of the ELQs that were tested. One conspicuous residue that may contribute to determining the binding mode of ELQ-300 is F34 from helix A, which interacts with the methyl group in the 2-position (Fig. This residue is conserved in P. falciparum, but not in human (S35) and it is thus possible that its bulky character contributes to dictating the orientation of the 3-aryl quinolones in apicomplexan parasites by restricting the entrance of the Qi site. Thus, our structure reveals an unexpected binding mode of the ELQ-300 in the apicomplexan Qi site, which differs markedly from previously determined structures of the inhibited mammalian CIII and provides an accurate molecular model for developing antimalarial inhibitors. This finding corroborates the observation that an I22L mutation of the respective residue in P. falciparum (clone D1) causes a 24-fold reduction in sensitivity to ELQ-300 (ref. L26 further contacts the chlorine substituent at the van der Waals distance (Extended Data Fig. 9r), suggesting it is important for ligand positioning within the Qi site. This finding agrees with previous evidence showing that the size of the atomic radius of the 6-position substituent, fluorine (ELQ-316) or hydrogen (ELQ-298), is inversely correlated to potency against T. gondii25,28. Furthermore, our structure indicates that species-specific affinity may not be solely owing to steric hindrance introduced by L26, but due to additional contacts of the 3′-methyl group of the P. falciparum I22 sidechain with the 7-methoxy group of ELQ-300 (Extended Data Fig. We concluded that this may be owing to a deeper penetration of the inhibitor into the Qi site, allowing a stronger interaction with the substituent at the position 7 (Extended Data Fig. Indeed, an analysis of the apicomplexan Qi site revealed that a parasite-specific aqueous pocket around the 7-methoxy-substituent is generated by replacement of mammalian residues Y224 and K227 with M219 and T222 in T. gondii, respectively. Moreover, a T222P substitution resulted in resistance to 7-methoxy ELQs36. To test this hypothesis, we synthesized and tested ELQs with a longer 7-ethoxy substituent in combination with chlorine (ELQ-804), fluorine (ELQ-807) and hydrogen (ELQ-627) at position 6 (Fig. 5d,f and Extended Data Table 1). The EC50 values of each 7-methoxy compound compared with its 7-ethoxy analog was statistically different (P < 0.0001). The EC50 of ELQ-627 was 0.12 nM, which was 27-fold more potent than ELQ-316, a lead preclinical candidate for toxoplasmosis that is highly effective and well-tolerated in mouse models of acute and latent toxoplasmosis compared with clinically used drugs and advanced preclinical compounds42. The discovery and validation of a unique binding pocket in the T. gondii Cyt-b Qi site provides a readily exploitable feature of the apicomplexan target for structure-guided drug design. The cryo-EM structure of the apicomplexan CIII will advance the development of pathogen-specific, highly potent Cyt-b inhibitors to treat devastating apicomplexan diseases. T. gondii tachyzoites were cultured in HFF, sourced from American Type Culture Collection (SCRC-1041). HFFs and parasites were cultured in Dulbecco's modified Eagle medium, containing 4.5 g l−1 glucose, supplemented with 10% (v/v) fetal bovine serum (FBS), 4 mM L-glutamine and penicillin or streptomycin antibiotics and grown at 37 °C with 5% CO2. To generate the Rieske-3xFLAG line (C-terminal triple FLAG epitope tagging), a gRNA targeting the stop codon of TGGT1_320220 was identified using ChopChop44 and cloned into a U6 promoter and Cas9–green fluorescent protein expressing vector (Tub-Cas9YFP-pU6-ccdB-tracrRNA)45 using the BsaI restriction site. The CAT selection cassette and triple FLAG epitope were amplified by PCR from a pLIC.TEV.3xFLAG.CATΔpacI plasmid46. 47) line by electroporation and cassette integration was selected with chloramphenicol. For the ApiCox knockout line, a gRNA targeting the stop codon of TGGT1_316255 was identified via ChopChop44 and cloned as above into Tub-Cas9YFP-pU6-ccdB-tracrRNA. A dihydrofolate reductase (DHFR) resistance cassette was amplified using the pDT7S4 plasmid as template47 and using primers containing 50 bp of sequence homology to regions upstream and downstream of the TGGT1_316255 open reading frame. 48) parental line by electroporation and cassette integration was selected with pyrimethamine. For the complementation of the knockout cell line, cDNA was cloned into a pTUB8mycGFPMyoATy expression vector via EcoRI and NsiI restriction sites49, plasmid electroporated and selected with mycophenolic acid (25 mg ml−1) and xanthine (50 mg ml−1). Replication assay: parental or ApiCox10-KO parasites were inoculated into HFF cells grown on a glass coverslip and left to invade and replicate for 24 h. Parasites were detected via immunofluorescence using the GAP45 antibody [1:1,000]50. The number of vacuoles containing 1, 2, 4 or 8+ parasites were counted for more than 250 vacuoles per replicate. Four independent experiments were performed. Competition assay: parental or ApiCox10-KO parasites in the mNEON::Δku80 background48 were counted, resuspended in fluorescence-activated cell sorting (FACS) buffer (25 mM HEPES, 5 mM EDTA and 1% w/v FBS) and mixed in a ≈1:1 ratio with wild-type tdTomato::Δku80 parasites and the mixed population inoculated in cell culture. These mixed populations were passaged six times. At each passage a portion of the population was analyzed using BD FACS Celesta (BD Biosciences). Data were analyzed using the FlowJo v10.8.1 software (BD biosciences). Growth inhibition for ELQ EC50: T. gondii proliferation inhibition with ELQs was measured in a 96-well assay using an RH T. gondii strain expressing beta-galactosidase cultured in HFF. Compounds dissolved in dimethylsulfoxide (DMSO) were diluted serially across the plate in four replicate rows by fourfold dilutions with a control column receiving no compounds. Then, 4,000 T. gondii tachyzoites were added to each well. After 3 days of incubation (37 °C, 5% CO2), the media were replaced with a solution of chlorophenol red-β-d-galactopyranoside and NP-40. The absorbance of each well was measured at 575 nm in a Molecular Devices SpectraMax 190 plate reader. Each compound was tested in at least three independent experiments. Absorbance was plotted against the base-10log of compound concentration and fitted to a four-parameter model of the Hill equation to estimate the EC50 for each compound. The 95% CIs of the mean, two-tailed unpaired t-tests and EC50 values were calculated using GraphPad Prism v8.4.3 software. For native polyacrylamide gel electrophoresis (PAGE) analysis, parasites were filtered through a 3.0-µm polycarbonate filter, washed with ice cold phosphate buffered saline (PBS) and resuspended in native PAGE sample buffer (Thermo Fisher) supplemented with 1% n-dodecyl β-D-maltoside (β-DDM) or 1% digitonin. After 30 min incubation at 4 °C, samples were centrifuged at 18,000g for 30 min at 4 °C and the supernatant mixed with Coomassie G250 to a final concentration of 0.25% w/v. Samples were separated on a native PAGE 4–16% (for monomer detection) or a 3–12% (for supercomplex detection) Bis-Tris gel. NativeMark and a bovine mitochondrial membrane preparation was used as a molecular weight marker. SDS–PAGE, immunoblot analysis and DAB staining were performed as described previously8,46. The following primary antibodies were used: anti-HA (Roche, clone 3F10) (1:500), anti-TOM40 (1:2,000)51, anti-MYS (1:2,000)52, anti-Ty53 (1:800), anti-CDPK1 (1:10,000)54 and anti-FLAG (Thermo Fisher, clone FG4R) (1:2,000). SDS–PAGE immunoblots were then labeled with secondary fluorescent antibodies (LI-COR: anti-mouse IRDye 800CW, anti-rabbit IRDye 680RD and anti-rat IRDye 800CW, all 1:10,000) and imaged with an Odyssey CLx. Native PAGE immunoblots were labeled with secondary horseradish peroxidase-conjugated antibodies (anti-rat, immunoglobulin G H&L horseradish peroxidase, abcam, 1:5,000; anti-rabbit, immunoglobulin G H&L horseradish peroxidase conjugate, Promega, 1:10,000) and chemiluminescence detection using Pierce ECL western blotting substrate and either an X-ray film or an iBright FL1000 imager (Invitrogen). OCR and extracellular acidification rate (ECAR) were measured using a Seahorse XF HS Mini Analyzer v3.0.0.41 (Agilent Technologies) as described previously55. Each of the six independent experiments was performed with parental and ApiCox10-KO parasites with two technical replicates. Parasites were filtered through a 3.0 µm polycarbonate filter and incubated with 1.5 µM JC-1 (5,5′,6,6′-tetrachloro-1,1′,3,3′-tetraethylbenzimidazolocarbocyanine iodide, Thermo Fisher Scientific, stock 1.5 mM in DMSO) for 15 min at 37 °C. Treatment with 10 μM valinomycin was used as a depolarizing control. Cells were pelleted and resuspended in 1–2 ml FACS buffer (25 mM HEPES, 5 mM EDTA and 1% w/v FBS) before analysis using a BD FACS Celesta analyzer and data acquisition using FACSDiva software v9 (BD Biosciences). Unstained controls were used to define gates for analysis. In total, 50,000 events per treatment were collected, and data were analyzed using FlowJo v10.8.1 software (BD Biosciences). After 1 day, cells were fixed with 4% paraformaldehyde. Cells were permeabilized and blocked with a solution of 2% bovine serum albumin and 0.2% triton X-100 in PBS before incubation with primary antibodies (anti-Ty53, anti-MYS52 and anti-TOM40 (ref. 51)), 1:1,000, followed by secondary antibodies (Alexa Fluor Goat anti-Mouse 488 Invitrogen A-11001, 1:1,000 and Alexa Fluor Goat anti-Rabbit 594 Invitrogen A-11012, 1:1,000). Coverslips were mounted on slides with Fluoromount-G mounting media containing 4′,6-diamidino-2-phenylindole (Southern Biotech, 0100–20). Slides were visualized on a DeltaVision Core microscope (Applied Precision) using the 100× objective and z-stacking. Images were deconvolved using SoftWoRx v5.5 software and processed using FIJI software v1.5.2 (ref. Cells were then fixed with fixation buffer (2.5% (v/v) glutaraldehyde, 4% (w/v) paraformaldehyde, in 0.1 M cacodylate buffer, pH 7.2), washed in 0.1 M cacodylate buffer, pH 7.2 and post-fixed in 1% (w/v) OsO4, 1.25% (w/v) K4[Fe(CN)6] for 1 h on ice. After several washes in the same buffer, the samples were en bloc stained with 0.5% (w/v) uranyl acetate in water for 30 min. Thereafter, samples were washed with water, dehydrated in ascending acetone series and resin embedded. Ultrathin sections (~50 nm thick) were collected and imaged on a JEOL 1200 Transmission electron microscope (JEOL) operated at 80 kV. Images were analyzed in FIJI software and the number of cristae per unit mitochondrial area was calculated. MitoSOX staining to assess oxidative stress in the mitochondria was performed as described previously48 with minor modifications. Briefly, parasites were grown in the presence or absence of 2 mM ferric ammonium chloride, shown to increase mitochondrial oxidative stress, for 6 h, before parasites were filtered through a 3.0 µm polycarbonate filter and incubated with 1 µM MitoSOX (Thermo Fisher, M36008) for 30 min at 37 °C. Cells were pelleted and resuspended in 2 ml FACS buffer (25 mM HEPES, 5 mM EDTA and 1% w/v FBS) before analysis using BD FACS Celesta analyzer and data acquisition using FACSDiva software v9 (BD Biosciences). Parasites were gated on forward and side scatter and on green fluorescence before the red fluorescent signal was analyzed. In total, 100,000 events per treatment were collected and data were analyzed using the FlowJo v10.8.1 software (BD biosciences). Immunoprecipitations for identification by mass spectrometry were performed as described previously55 with minor modifications. Cox2a-HA and ApiCox10-KO/Cox2a-HA parasites were lysed in a lysis buffer containing 1% digitonin. Elutions containing equal amount of protein were sent for mass spectrometry analysis. Four independent experiments were performed. Proteins detected in at least three out of the four experiments are displayed in a volcano plot. Data analysis to generate a volcano plot was performed using Perseus v1.6.12.0; samples were compared using a two-sided t-test, the false discovery rate was set to 0.01 and the significance threshold was set to 2. Immunoprecipitation of Rieske-TEV-3xFLAG parasites was performed using anti-FLAG M2 affinity agarose gel (Merck). Parasites (≈1 × 1010) were incubated in 4 ml buffer (150 mM NaCl, 2 mM EDTA, 50 mM Tris–HCl pH 7.4) containing 2% β-DDM for 2 h at 4 °C before centrifugation at 18,000g for 30 min at 4 °C. The supernatant was incubated with FLAG affinity gel overnight at 4 °C. Then the affinity gel was washed three times with buffer containing 0.05% β-DDM before elution with a FLAG peptide solution (150 µg ml−1) for 1 h at 4 °C. The eluate was concentrated to ≈50 µl in a vivaspin 500 filter (100 kDa molecular weight cutoff) and subjected to gel filtration on a Superose 6 Increase 3.2/300 column (GE Healthcare) in 150 mM NaCl, 2 mM EDTA, 50 mM Tris–HCl pH 7.4 and 0.05% β-DDM to separate it from aggregates and FLAG peptide. CIII-containing fractions were pooled and incubated with amphipol A8-35 (Anatrace) in a molar ratio of 1:5 for 4 h at 4 °C, followed by the addition of Bio-Beads SM-2 Resin (Bio-Rad) in a molar ratio of 1:20 for 16 h at 4 °C to remove detergent. The sample was concentrated and gel filtrated as before, followed by a final spin-column concentration to 0.94 mg ml−1. ELQ-300 (in DMSO) was added to a final concentration of 10 µM before sample vitrification. Parasite culturing and mitochondria purification was performed as previously described13. Briefly, T. gondii RH tachyzoites were grown in Vero cells in Dulbecco's modified Eagle medium supplemented with 10% (v/v) FBS, 4 mM of l-glutamine and penicillin or streptomycin antibiotics at 37 °C with 5% (v/v) CO2. For each mitochondrial preparation ≈100 T150 flasks were collected at >80% host-cell lysis and passed through 23G needles to fully lyse any remaining host cells. Parasites were lysed by successive rounds of nitrogen cavitation (2,500 PSI, 15 min incubation on ice) until >95% lysis (confirmed by light microscopy). The final combined lysate was centrifuged as before to remove unbroken cells and the supernatant was centrifuged at 16,000g for 30 min at 4 °C. The resulting crude mitochondrial pellet was further purified on a discontinuous sucrose gradient in 20 mM of HEPES–KOH pH 7.4, 2 mM of EDTA, 15%/23%/32%/60% (w/v) sucrose by centrifugation (103,745g, 1 h, 4 °C) in an SW41 rotor (Beckman Coulter) and enriched mitochondria were collected from the 32–60% (w/v) interface. Enriched mitochondria were lysed in a total volume of 34 ml buffer containing 25 mM HEPES–KOH pH 7.5, 25 mM KCl, 15 mM MgOAc2, 2% (w/v) digitonin, 2 mM DTT and one tablet of EDTA-free protease inhibitor cocktail for 2 h at 4 °C and the lysate was cleared by centrifugation at 30,000g for 20 min at 4 °C. The supernatant was layered on a sucrose cushion in buffer of 1 M sucrose, 25 mM HEPES–KOH pH 7.5, 25 mM KCl, 15 mM MgOAc2, 2% digitonin and 2 mM DTT, and centrifuged at 230,759g for 4 h at 4 °C in a Ti70 rotor (Beckman Coulter). Fractions corresponding to the respiratory chain supercomplex were pooled and concentrated to 25 µl in a vivaspin 500 filter (100 kDa molecular weight cutoff). Atovaquone and ELQ-300 (in DMSO) were added in a 100-fold dilution to a final concentration of 10 µM before sample vitrification. Next, 3 µl of sample (~0.2–0.5 mg ml−1) were applied to glow-discharged Quantifoil R2/2 Cu grids with a 2-nm-thin carbon support layer and vitrified by plunge-freezing into liquid ethane after blotting for 3 s. Cryo-EM was performed on a Titan Krios operated at 300 kV at a magnification of 165 kx (0.83 Å per pixel) with a K3 quantum camera (slit width 20 eV) at an exposure rate of 17 electrons pixel−1 s−1 with a 1.75 s exposure fractionated into 40 frames using EPU 1.12 software (Thermo Fisher Scientific). Initial rounds of 2D classification were performed to generate classes for reference-based particle picking, which was performed in RELION4, resulting in 2,658,324 picked particles. To address sample heterogeneity, the resulting classes of a subsequent 2D classification (cryoSPARC4.6) were manually grouped and subjected to ab initio model generation (cryoSPARC), which resulted in six different 3D references, corresponding to the T. gondii ATP synthase dimer and supercomplex, the C. sabaeus ATP synthase and I–III2–IV supercomplex, and mitochondrial large ribosomal subunit, as well as an unidentified C7-symmetric complex (Supplementary Fig. These six maps were used as input to heterogeneous refinements, resulting in the particle distribution shown in Supplementary Fig. Subsequent masked refinements of the T. gondii supercomplex yielded maps of the CIII and CIV subcomplexes at 2.79 and 3.16 Å, respectively. Data collection of the ELQ-300-inhibited T. gondii complex-III dataset was performed on a Titan Krios, as described above, resulting in 46,370 movies. Image processing was performed in cryoSPARC. Templates for particle picking were generated by initial rounds of 2D classification. To reduce map anisotropy and increase interpretability, the number of top views was reduced, resulting in a final number of 2,056,878 particles. Non-uniform refinement resulted in a 1.9 Å structure which was improved to 1.83 Å by cryo-EM density modification in PHENIX 1.21 (ref. All final maps were generated from contrast transfer function (CTF)-refined particles. All resolution estimates are according to Fourier shell correlations (FSC) that were calculated from independently refined half-maps using the 0.143 criterion with correction for the effect of the applied masks (Extended Data Fig. An initial atomic model of the T. gondii III2–IV supercomplex was assembled by fitting the known conserved subunits, as predicted by AlphaFold2 (ref. Newly identified subunits were built de novo and identified by BLAST searches following manual modeling in Coot 0.96 (ref. At the modeled N-terminus of the T. gondii Cyt-b, we identified cryo-EM density preceding residue Met10, which is consistent with a Phe9. While this observation is not explained by the mRNA sequence, it is consistent with the high-resolution cryo-EM map. For atomic model building of the C. sabaeus CIII, a homology model of the murine CIII (Protein Data Bank (PDB) 7o37) was generated using the SWISS-MODEL webserver, followed by rebuilding and inhibitor fitting in Coot and structure refinement in PHENIX 1.19. Homology modeling of the P. falciparum QCR6 homolog was performed in SWISS-MODEL. For the mapping of Cyt-c binding sites, PDB 3CX5 (ref. 61) were fitted to the T. gondii complexes III2 and IV, respectively, to reveal cytochrome binding sites. Prediction of membrane positions was performed using the OREMPRO webserver62. Structure-based multiple sequence alignment of QCR6 was performed in Chimera-1.14 (ref. Images of the structures were generated with ChimeraX 1.6.1 (ref. For CIII subunits, TGGT1_214250, previously named apicomplexan-specific QCR11 (ref. 8), was reassigned as a homolog of conserved subunit QCR10 and thus renamed TgQCR10. For CIV subunits, we followed a previously established ‘ApiCox' nomenclature11. However, structural similarity showed that previously named ApicCox23, 25, 35 and 14 correspond to conserved subunits Cox4, 6A, 6C and 7a, respectively. Furthermore, we newly identified the conserved subunit Cox2c and Cox7c directly from the cryo-EM density. We also reassigned the previously detected subunit TGGT1_257160 as conserved Cox8a and, following the established nomenclature, novel clade-conserved subunits ApiCox22, 20, 5, 10 and 7. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The atomic coordinates were deposited in the PDB under accession numbers 9I4X (T. gondii complex III with ELQ-300/ATQ), 9I4Y (T. gondii CIV), 9H8T (C. sabaeus CIII with ATQ), and 9G9T (T. gondii CIII with ELQ-300). The cryo-EM maps have been deposited in the Electron Microscopy Data Bank (EMDB) under the respective accession numbers: EMD-52621 (T. gondii complex III with ELQ-300/ATQ), EMD-52622 (T. gondii CIV), EMD-51939 (C. sabaeus CIII with ATQ), and EMD-51157 (T. gondii CIII with ELQ-300). The atomic coordinates that were used in this study are: PDB 7O3C (murine III2–IV supercomplex), 5IY5 (cytochrome c) and 3CX5 (complex III with bound cytochrome c). Full versions of all gels are provided in the source file. It includes Supplementary Methods (parasite cell culture and genetic manipulation; mitochondrial isolation; purification of respiratory supercomplexes; electron cryo-microscopy and data processing; atomic model building, homology modeling and analysis; data visualization and multiple sequence alignment; growth analysis; blue native (BN) and SDS–PAGE and immunoblot analysis; analysis of respiratory rate; compound synthesis); Supplementary Tables 1 and 2, Supplementary Discussion and Supplementary Notes. Source data are provided with this paper. Schägger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast and mammalian mitochondria. Enhanced respiratory chain supercomplex formation in response to exercise in human skeletal muscle. Fedor, J. G. & Hirst, J. Mitochondrial supercomplexes do not enhance catalysis by quinone channeling. Berndtsson, J. et al. Respiratory supercomplexes enhance electron transport by decreasing cytochrome c diffusion distance. Structural rather than catalytic role for mitochondrial respiratory chain supercomplexes. Preserved respiratory chain capacity and physiology in mice with profoundly reduced levels of mitochondrial respirasomes. Composition and stage dynamics of mitochondrial complexes in Plasmodium falciparum. Maclean, A. E. et al. Complexome profile of Toxoplasma gondii mitochondria identifies divergent subunits of respiratory chain complexes including new subunits of cytochrome bc1 complex. A., Rajendran, E., Zwahlen, S. M., Faou, P. & van Dooren, G. G. Divergent features of the coenzyme Q:cytochrome c oxidoreductase complex in Toxoplasma gondii parasites. Structure of the intact 14-subunit human cytochrome c oxidase. Seidi, A. et al. Elucidating the mitochondrial proteome of Toxoplasma gondii reveals the presence of a divergent cytochrome c oxidase. A., Huet, D., van Dooren, G. G. & Sheiner, L. The mystery of massive mitochondrial complexes: the apicomplexan respiratory chain. Mühleip, A. et al. ATP synthase hexamer assemblies shape cristae of Toxoplasma mitochondria. Daley, D. O., Clifton, R. & Whelan, J. Intracellular gene transfer: reduced hydrophobicity facilitates gene transfer for subunit 2 of cytochrome c oxidase. Zhou, L., Maldonado, M., Padavannil, A., Guo, F. & Letts, J. Klusch, N. et al. Cryo-EM structure of the respiratory I + III(2) supercomplex from Arabidopsis thaliana at 2 Å resolution. Davies, K. M. et al. Macromolecular organization of ATP synthase and complex I in whole mitochondria. Shimberg, G. D., Pritts, J. D. & Michel, S. L. J. Iron-sulfur clusters in zinc finger proteins. Leonard, R. A., Tian, Y., Tan, F., van Dooren, G. G. & Hayward, J. An essential role for an Fe-S cluster protein in the cytochrome c oxidase complex of Toxoplasma parasites. The architecture of respiratory supercomplexes. Rathore, S. et al. Cryo-EM structure of the yeast respiratory supercomplex. Atomic structures of respiratory complex III(2), complex IV, and supercomplex III(2)-IV from vascular plants. Moe, A., Di Trani, J., Rubinstein, J. L. & Brzezinski, P. Cryo-EM structure and kinetics reveal electron transfer by 2D diffusion of cytochrome c in the yeast III–IV respiratory supercomplex. & Jaffe, G. J. Atovaquone for the treatment of toxoplasma retinochoroiditis in immunocompetent patients. Doggett, J. S. et al. Endochin-like quinolones are highly efficacious against acute and latent experimental toxoplasmosis. Birth, D., Kao, W. C. & Hunte, C. Structural analysis of atovaquone-inhibited cytochrome bc1 complex reveals the molecular basis of antimalarial drug action. Nilsen, A. et al. Quinolone-3-diarylethers: a new class of antimalarial drug. Targeted structure-activity analysis of endochin-like quinolones reveals potent Qi and Qo site inhibitors of Toxoplasma gondii and Plasmodium falciparum cytochrome bc1 and identifies ELQ-400 as a remarkably effective compound against acute experimental toxoplasmosis. Stickles, A. M. et al. Atovaquone and ELQ-300 combination therapy as a novel dual-site cytochrome bc1 inhibition strategy for malaria. & Boothroyd, J. C. Characterization of cytochrome b from Toxoplasma gondii and Q(o) domain mutations as a mechanism of atovaquone-resistance. Srivastava, I. K., Morrisey, J. M., Darrouzet, E., Daldal, F. & Vaidya, A. Resistance mutations reveal the atovaquone-binding domain of cytochrome b in malaria parasites. Nam, T. G. et al. A chemical genomic analysis of decoquinate, a Plasmodium falciparum cytochrome b inhibitor. Kessl, J. J. et al. Molecular basis for atovaquone binding to the cytochrome bc1 complex. B. Hemozoin-free Plasmodium falciparum mitochondria for physiological and drug susceptibility studies. Biagini, G. A. et al. Generation of quinolone antimalarials targeting the Plasmodium falciparum mitochondrial respiratory chain for the treatment and prophylaxis of malaria. Alday, P. H. et al. Genetic evidence for cytochrome b Qi site inhibition by 4(1H)-quinolone-3-diarylethers and antimycin in Toxoplasma gondii. Targeting the ubiquinol-reduction (Q(i)) site of the mitochondrial cytochrome bc(1) complex for the development of next generation quinolone antimalarials. Zhang, Y. et al. Synthesis and structure-activity relationships of antimalarial 4-oxo-3-carboxyl quinolones. de Souza, J. O. et al. Parasitological profiling shows 4(1H)-quinolone derivatives as new lead candidates for malaria. Subtle changes in endochin-like quinolone structure alter the site of inhibition within the cytochrome bc1 complex of Plasmodium falciparum. Discovery, synthesis, and optimization of antimalarial 4(1H)-quinolone-3-diarylethers. Orally bioavailable endochin-like quinolone carbonate ester prodrug reduces Toxoplasma gondii brain cysts. Mühleip, A. Mechanismen der cristae-biogenese in humanparasiten. Curt-Varesano, A., Braun, L., Ranquet, C., Hakimi, M. A. Identification of the Toxoplasma gondii mitochondrial ribosome, and characterisation of a protein essential for mitochondrial translation. Sheiner, L. et al. A systematic screen to discover and analyze apicoplast proteins identifies a conserved and essential protein import factor. Herm-Gotz, A. et al. Toxoplasma gondii myosin A and its light chain: a fast, single-headed, plus-end-directed motor. Plattner, F. et al. Toxoplasma profilin is essential for host cell invasion and TLR11-dependent induction of an interleukin-12 response. van Dooren, G. G., Yeoh, L. M., Striepen, B. Ovciarikova, J., Lemgruber, L., Stilger, K. L., Sullivan, W. J. & Sheiner, L. Mitochondrial behaviour throughout the lytic cycle of Toxoplasma gondii. Bastin, P., Bagherzadeh, Z., Matthews, K. R. & Gull, K. A novel epitope tag system to study protein targeting and organelle biogenesis in Trypanosoma brucei. Silva, M. F., Douglas, K., Sandalli, S., Maclean, A. E. & Sheiner, L. Functional and biochemical characterization of the Toxoplasma gondii succinate dehydrogenase complex. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Terwilliger, T. C., Ludtke, S. J., Read, R. J., Adams, P. D. & Afonine, P. V. Improvement of cryo-EM maps by density modification. Highly accurate protein structure prediction with AlphaFold. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Solmaz, S. R. & Hunte, C. Structure of complex III with bound cytochrome c in reduced state and definition of a minimal core interface for electron transfer. Shimada, S. et al. Complex structure of cytochrome c–cytochrome c oxidase reveals a novel protein-protein interaction mode. & Gelly, J. C. OREMPRO web server: orientation and assessment of atomistic and coarse-grained structures of membrane proteins. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. Wimley, W. C. & White, S. H. Experimentally determined hydrophobicity scale for proteins at membrane interfaces. Moon, C. P. & Fleming, K. G. Side-chain hydrophobicity scale derived from transmembrane protein folding into lipid bilayers. McPhillie, M. et al. New paradigms for understanding and step changes in treating active and chronic, persistent apicomplexan infections. We acknowledge the Scottish Center for Macromolecular Imaging (SCMI) for access to cryo-EM instrumentation, funded by the MRC (MC_PC_17135) and SFC (H17007). We thank the Swedish National cryo-EM facility at SciLifeLab (funded by the KAW, EPS and Kempe foundations), for instrument access and assistance with data acquisition. We thank the Infrastructure Team–Information Services (University of Glasgow) for computational support. We thank the Cellular Analysis Facility from MVLS SRF for their support and assistance for the flow cytometry work. We thank C. Ettles for assistance with parasitology work. We thank L. Lemgruber of the Cellular Analysis Facility from MVLS SRF for his support and assistance in in the transmission electron microscopy work. We would like to acknowledge S. Kosto of the FingerPrints Proteomics Facility at the University of Dundee, which is supported by the ‘Wellcome Trust Technology Platform' award (097945/B/11/Z). The Wellcome Centre for Integrative Parasitology is supported by core funding from the Wellcome Trust (104111). We thank the OHSU Medicinal Chemistry Core (research resource ID: SCR_019048). This work was supported by Wellcome Investigator Award (217173_Z_19_Z) (to L.S. ); by a FutureScope Fellowship by the Wellcome Centre for Integrative Parasitology (to A. Mühleip); and Sir Henry Wellcome Fellowship (to A.E.M., grant number 221681/Z/20/Z). S.S. was supported by Swiss National Science Foundation Early Postdoc Mobility Fellowship (grant no. A. Meir was supported by MRC grants MC_UU_00034/1 and MC_UU_12014/7. received funding for this work from the US Department of Veterans Affairs Biomedical Laboratory Research VA Merit Review award BX004522. receives funding from the US Department of Veterans Affairs, Office of Research and Development Program Award number BX003312. Research reported in this publication was supported by the US National Institutes of Health under award numbers R01AI100569 and R01AI141412 and by the US Department of Defense Peer Reviewed Medical Research Program (PR181134). The National Science Foundation provided instrument funding for the BioAnalytical Mass Spectrometry Facility at Portland State University (NSF, MRI 1828573), which was used to generate HRMS analytical measurements. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the paper. Open Access funding provided by University of Helsinki (including Helsinki University Central Hospital). Andrew E. MacLean, Shikha Shikha, Mariana Ferreira Silva, Amit Meir, Lilach Sheiner & Alexander Mühleip Max J. Gramelspacher, Aaron Nilsen, Katherine M. Liebman, Sovitj Pou, Rolf W. Winter, Michael K. Riscoe & J. Medicinal Chemistry Core, Oregon Health and Science University, Portland, OR, USA Department of Microbiology and Molecular Immunology, Oregon Health and Science University, Portland, OR, USA School of Medicine Division of Infectious Diseases, Oregon Health and Science University, Portland, OR, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed genetic manipulation of parasites. performed isolation of mitochondria and protein purification. A. Meir performed protein purification and optimization for cryo-EM studies. A. Mühleip performed cryo-EM sample preparation, data collection, data processing, atomic model building and analysis. performed cell biological, biochemical and microscopy assays. contributed to conceptualization, synthesis and characterization of compounds. All authors contributed to figure and table preparation and/or to the writing of the methods. wrote the paper with contributions from A.E.M. Correspondence to Lilach Sheiner or Alexander Mühleip. The authors declare no competing interests. Nature Structural & Molecular Biology thanks Leonid Sazanov, Rita Tewari, Janet Vonck and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Dimitris Typas was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Representative micrograph with T. gondii and C. sabaeus supercomplex particles indicated in blue and orange, respectively. (b) Representative 2D classes of the T. gondii supercomplex. (c) Representative 2D classes of C. sabaeus supercomplex. (d) Data processing scheme, highlighting the reference-based 3D classification to separate T. gondii and C. sabaeus ATP synthase (purple, light green), mitochondrial large subunit (yellow) and putative mitochondrial chaperonin (teal) from T. gondii and C. sabaeus supercomplexes (blue and orange). (e) Fourier Shell Correlation (FSC) curves. (f) Representative micrograph with T. gondii complex III from IP. (g) Representative 2D classes of the T. gondii complex III. (h) Data processing scheme for complex III. (a) Summary of homology search outcomes for the new subunits. (c) Heat map indicating the mean hydrophobicity (calculated by grand average of hydropathy (KD)65 or according to Wimley-White (WW)66 or the Moon-Fleming (MF)67 of Cox2 homologs in divergent organisms. Lipid positions indicate a bend membrane region (dashed lines). (b) T. gondii supercomplex structure with estimated membrane plane regions showing a rotational offset between the two subcomplexes CIII and CIV. (a) T. gondii CIV structure (b) TgApiCox13 with bound lipids (CDL, cardiolipin; PE, phosphatidylethanolamine) (c) ApiCox13 from T. gondii (left, this study), predicted homolog structures from Perkinsus marinus and Plasmodium falciparum. Of the two Fe2S2 clusters in T. gondii, one is conserved in P. falciparum, whereas P. marinus seem to utilize zinc. (d) Phylogenetic tree with conservation of Fe2S2 and zinc (e) Multi-sequence alignment with residues involved in Fe2S2 or zinc binding shown in yellow and orange respectively. (f) Cofacors of the T. gondii respiratory supercomplex. (a) Overlay of supercomplex structures aligned on CIV highlights different architectures (luminal view), as revealed by differing positions of CIII from mammals (M. musculus, PDB 7o37 mature supercomplex), yeast (S. cerevisiae, PDB 6giq) and plant (V. radiata, 7jrp). The mammalian and T. gondii CIII homologs bind to opposite sides of CIV. (b) TgQCR6 (hinge protein) interaction with four contact sites on CIV highlighted. (c) T. gondii III2-IV interface (this study) of TgQCR6 with ApiCox10 involving contacts via an apicomplexan-conserved N-terminal extension. (d-e) Structural overlay (D) and structure-based multiple sequence alignment (E) of the murine, P. falciparum and T. gondii QCR6 homologs showing a conserved hairpin structure. The N-terminal QCR6 extension that interacts with ApiCox10 is also found in P. falciparum. The mammalian N-terminus is cleaved as part of a mitochondrial targeting sequence. The expected size of integration PCRs are shown. (b) PCR to test integration of HA-epitope tag and CAT selection cassette into the endogenous locus, as outlined in (A). (c) Immunoblot analysis of whole cell lysate extracted from Cox2a-HA and parental parasites. Samples were separated by SDS-PAGE, blotted, and detected using anti-HA and anti-TOM40 as a loading control. (d) Schematic of the strategy used to replace the coding sequence of ApiCox10 with a DHFR selection cassette. The expected size of integration PCRs are shown. (e) PCR to test integration of the DHFR selection cassette into the endogenous locus, in Cox2a-HA, QCR2-HA and mNEON::Δku80 parental background, as outlined in (D). (h) Volcano plot of proteomic data from Cox2a-HA vs ApiCox10/Cox2a-HA immunoprecipitations, showing the -log10 P values and the log2 fold changes of proteins detected by mass spectrometry in 3 or more replicates, from 4 independent experiments. P values were calculated using a two-tailed t-test. Horizontal dotted gray line denotes P= 0.05 and P = 0.01. Vertical dotted line denotes 5x enrichment. Complex III subunits are labeled in red and complex IV subunits are labeled in the blue. Total lysate was treated with digitonin and separated by BN-PAGE, followed by immunoblot analysis. Positions of complexes are indicated. (j) Transmission electron microscopy (TEM) of parasites to visualize mitochondrial cristae. (k,l) Quantification of number of cristae cross sections per mitochondrial surface area (K) and mitochondrial area (L) in parental and ApiCox10-KO parasites from TEM. Data points from 100 mitochondrial profiles, with mean shown ( ± s.d. P value from a two-tailed unpaired t-test. (m) MitoSOX labeling to detect mitochondrial ROS at steady state or upon treatment with ferric ammonium chloride (FAC) in ApiCox10 knockout and parental parasites in mNEON background via flow cytometry analysis. Left: red fluorescence of T. gondii, in the absence or presence of ferric ammonium chloride (FAC), stained with MitoSOX. Right: Quantification of population that is positive for MitoSOX signal. Graphs show mean ± s.d., from 4 independent experiments One-way ANOVA followed by Turkey's multiple pairwise comparisons was performed, and P value from relevant pairs displayed. (n) Surface electrostatics of the III2-IV (calculated with the Adaptive Poisson-Boltzmann Solver, APBS4) reveals a negative lumenal patch. Cytochrome-c binding sites were inferred from overlays of PDBs 5iy5 and 3cx5. (o) Quantification of number of parasites per vacuole for parental and ApiCox10-KO parasites. from 4 independent experiments, for which over 250 vacuoles were counted for each replicate. The P value determined by multiple two-tailed t-tests with a Holm-Sidak correction applied. (p) Mixed culture growth competition assay of ApiCox10-KO or parental mNEON fluorescent parasites with tdTomato parasites. Relative abundance (compared to passage 0) of ApiCox10-KO parasites across 6 passages. 3g,i and Extended Data Fig. Gates were drawn as indicated manually using FlowJo software, using forward and side scatter plots (SSC-A vs FSC-A) to gate on the parasite population and separate it from host cell debris. Single cells were then selected (FSC-A vs FSC-H). Parasites were then analyzed for fluorescence. (b) Flow cytometry fluorescence analysis (i) analysis of red fluorescence for JC-1 assay to determine mitochondrial membrane potential in Fig. 3g (left) (ii) analysis of green and red fluorescence for JC-1 assay to determine mitochondrial membrane potential in Fig. Comparison of parasites showing red and green or green only fluorescence was made, with positions of quadrants determined by unstained and valinomycin controls. (iii) analysis of green and red fluorescent parasites for growth competition assay in Fig. Green and red parasite quadrants determined by analyzing mNEON::Δku80 and tdTomato::Δku80 parental parasites separately (iv) analysis of red fluorescence for MitoSOX assay to determine mitochondrial ROS generation in Extended Data Fig. (a) T. gondii supercomplex structure with heme molecules and inhibitors (ELQ-300 and atovaquone (ATQ)) (b) Cryo-EM map (left) and model (right) of the C. sabaeus CIII. (c) Schematic of the C. sabaeus CIII with heme groups and two ATQ copies (orange) per monomer shown. Inset depicts close-up view of the Qi site occupied with atovaquone. (d) Overlay of Cyt-b structures from T. gondii, C. sabaeus (both this study) and S. cerevisiae with the module of helices F-H highlighted. (e) Comparison of helices F,G,H. The T. gondii helices F and H are straight and lack the conserved proline residues (red) found in the opisthokont homologs. (f) Overlay of the atovaquone-inhibited Cyt-b structures from T. gondii and C. sabaeus showing that the kink in the mammalian helix F causes it to extend further into the Qo site. Two affected residues interacting with ATQ are shown. (g) Multiple sequence alignment showing the absence of conserved proline residues (red) in helices F and H. Tg, Toxoplasma gondii; Pf, Plasmodium falciparum; Pm, Plasmodium malariae, Pk, Plasmodium knowlesi; Pv, Plasmodium vivax; Po, Plasmodium ovale; Hs, Homo sapiens; Cs, Chlorocebus sabaeus; Sc, Saccharomyces cerevisiae. (h) T. gondii Qo site with ATQ and surrounding conserved residues (map in blue). (i) Comparison of atovaquone bound Qo sites in T. gondii (left) and S. cerevisiae (right; PDB 4pd4). (j) Ligand diagram of atovaquone in the C. sabaeus Qi site. (a) Schematic of the strategy used to C-terminally triple FLAG-epitope tag the Rieske protein. The expected size of integration PCRs are shown. (b) PCR to test integration of FLAG-epitope tag and CAT selection cassette into the endogenous locus, as outlined in (A). (c) Immunoblot analysis of whole cell lysate extracted from Rieske-3xFLAG and parental parasites. Samples were separated by SDS-PAGE, blotted, and detected using anti-FLAG and anti-MYS as a loading control. (d) Immunoblot analysis of fractions from FLAG immunoprecipitation. (e) Purified complex III after one round of gel filtration: 5 µg of protein separated by SDS-PAGE and stained with instant blue. (f-j) Cryo-EM densities (transparent blue) for residues surrounding the Qi site showing the ELQ front view (F), back view (G), closeup on the cytochrome-b N-terminus (H) and conformers A and B of ELQ (I), closeup of a cardiolipin molecule (J). (k) In T. gondii (this study) the carbonyl and amino groups interact with H197 and D223, respectively. (l-m) In X-ray structures of the B. taurus homolog, the orientation of the 3-aryl quinolones is rotated by 180 degrees, leading to a different position of the diaryl-rest when compared to the parasite structure (L: PDB ID 5NMI, ref. Shown are inhibitors MJM170 (L) and WDH-1U-4 (M). (n-p) By comparison, owing to a flipped quinolone orientation 2-aryl-quinolones bind with the diaryl-like group facing in the same direction as in other structures of the bovine enzyme (N: PDB ID 7R3V; O: 5OKD; P: 6HAW), shown are inhibitors CK-2-67 (N), SCR0911 (O) and WDH2G7 (P). (q-r) Species-specific mutations in the apicomplexan Qi site explain susceptibility to ELQ-300. (q) ELQ-300 bound in the T. gondii cytochrome-b (this study, cryo-EM density in transparent blue) with overlaid a predicted model of the P. falciparum homolog (AlphaFold2). (r) Van der Waals atomic radii of the 3-position chlorine and L26 shown in comparison with the smaller fluorine atom found in ELQ-316. Supplementary Methods, Discussion, Notes and References. Mass spectometry data from Cox2a-HA and ApiCox10-KO/Cox2a-HA immunoprecipitations. Complex III and IV subunits detected. Uncropped gel images (pdf) and original data values (xlsx). Subunit homology search E values and hydrophobicity calculations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. MacLean, A.E., Shikha, S., Ferreira Silva, M. et al. Structure, assembly and inhibition of the Toxoplasma gondii respiratory chain supercomplex. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59759-2'>LLPS REDIFINE allows the biophysical characterization of multicomponent condensates without tags or labels</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-19 07:53:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Liquid-liquid phase separation (LLPS) phenomenon plays a vital role in multiple cell biology processes, providing a mechanism to concentrate biomolecules and promote cellular reactions locally. Despite its significance in biology, there is a lack of conventional techniques suitable for studying biphasic samples in their biologically relevant form. Here, we present a label-free and non-invasive approach to characterize biomolecular condensates termed LLPS REstricted DIFusion of INvisible speciEs (REDIFINE). Relying on diffusion NMR measurements, REDIFINE exploits the exchange dynamics between molecules in the condensed and dispersed phases to determine not only diffusion constants and the fractions in both phases but also the average radius of the condensed droplets and the exchange rate between the phases. Observing proteins, RNAs, water, as well as small molecules, and even assessing the concentrations of biomolecules in both phases, REDIFINE analysis allows a rapid biophysical characterization of multicomponent condensates which is important to understand their functional roles. In comparing multiple systems, REDIFINE reveals that folded RNA-binding proteins form smaller and more dynamic droplets compared to the disordered ones. Cells organize complex biochemical reactions in space and time by forming compartments1,2,3. Besides the organelles enclosed by membranes, proteins and nucleic acids can phase separate into membraneless liquid-like compartments such as the prominent nucleolus, nuclear speckles, processing bodies, or stress granules4,5,6. These cellular structures, formed through the process of liquid-liquid phase separation (LLPS), have been described as coacervates7,8 and are optically resolvable as micron-sized structures. Given their involvement in RNA metabolism, many of these biomolecular condensates are enriched in proteins that contain RNA-binding domains (RBDs) and LLPS-prone intrinsically disordered regions (IDRs)9,10,11. Numerous RNA-binding proteins (RBPs) undergo LLPS under physiological conditions, either in the presence or absence of RNA. One of the examples of RBPs that recently attracted a lot of attention in the scientific community during the COVID-19 pandemic, SARS-CoV-2 Nucleocapsid (Nu), binds the RNA and engages in multivalent homo- and heterotypic interactions leading to LLPS12,13. It is hypothesized that its phase separation with RNA is essential for genome RNA packaging and the formation of viral ribonucleoprotein (vRNP)14,15. Another RBP protein, Polypyrimidine tract-binding protein 1 (PTBP1), can form condensates with RNA, driving the production of the compartments in the nucleus to promote gene silencing16,17. Moreover, LLPS is associated with several diseases, including amyotrophic lateral sclerosis (ALS) and frontotemporal dementia, where stress may induce yet another RNA-binding protein FUS (FUsed in Sarcoma)18,19 to form granules that mature into solid aggregates. Disease-associated mutations accelerate this liquid-to-solid transition20. Besides FUS, a mammalian DEAD-box helicase (DDX4) can also undergo LLPS by itself (by self-association of the disordered N terminus)21, forming condensed droplets that can act as a molecular filter concentrating single-stranded DNA but excluding double-stranded DNA22. The domain schemes of these proteins are shown in Supplementary Fig. Despite the evidence of the functional importance of phase-separation for these proteins, a wide spectrum of questions about the biophysical properties of these biomolecular condensates and their relation to (patho)physiology in biological systems remain unanswered23,24, due to a lack of suitable experimental procedures that are not invasive and that can probe the biomolecules in their relevant form25,26,27. Although multicolor labeling can provide a plethora of information about the condensates, the fluorescent tags often affect protein conformation and dynamics, which can influence their phase separation, introducing potential bias in the conclusions28,29,30,31,32,33,34. In this work, we examined several biomolecular condensates (protein and protein-RNA complexes) using LLPS REDIFINE, an approach that utilizes the restricted diffusion of biomolecules inside the droplets and chemical exchange as an NMR lens to probe the physical properties of the condensates. Without the need for specific labels, we could comprehensively characterize the biomolecular condensates formed by wild-type FUS, DDX4, Nu, and PTBP1 proteins and correlate the extracted information to their biophysical characteristics. For our first application of LLPS REDIFINE, we examined the biphasic sample of the N-terminal domain (NTD) of the human FUS protein, which we studied earlier19. It comprises the low complexity QGSY-rich segment and the first arginine-glycine repeat (RGG1)19. FUS NTD domain is completely unstructured (Supplementary Fig. 1) and is considered to be essential for the phase separation of FUS protein35. FUS NTD droplets were stabilized in vitro using an agarose hydrogel, which allows prolonged spectroscopical analysis (Supplementary Fig. Previously, we have shown that the large diffusion difference between FUS NTD in dilute and condensed phases can be used to quantify their respective populations19. However, the choice of longer diffusion times, Δ, could impair this quantification due to the presence of protein exchange between the two phases. With this in mind, we performed multiple diffusion measurements on FUS NTD biphasic sample stabilized in agarose, where we varied the gradient strength as well as the diffusion time Δ (Fig. As expected, the decay of protein signal is more pronounced at the longer diffusion times. Figure 1b illustrates that indeed, the population of the condensed phase that would be extrapolated from the plateau19 becomes lower with increasing Δ. We observed that the apparent diffusion constants of both the slow- and fast-diffusing proteins are also strongly influenced by the diffusion time Δ (Fig. This implies that these experimental decay curves cannot be simply explained by two independent protein populations and that only by taking into account exchange between the populations one could interpret these curves. a A series of 1D proton spectra showing signal decay with increasing gradient strength in diffusion NMR experiment acquired on fresh FUS NTD sample using different diffusion times. Resonance highlighted in green contains an overlapping signals of tyrosine aromatic protons from both protein in dilute and condensed phase dispersed in agarose. b Decay of the integral of tyrosine signals plotted as \(I/{I}_{0}=f(g)\) and \({{\mathrm{ln}}}(I/{I}_{0})=f({q}^{2})\) illustrating the influence of diffusion time on droplet fraction and diffusion coefficients. c LLPS REDIFINE model and the ensuing fitting of the FUS NTD diffusion data set. Following parameters are determined: Ddil = 8.0 ± 0.2 × 10−11 and Dcond = 8.9 ± 0.7 × 10−13 m2/s, νcond = 0.753 ± 0.002, Rdrop = 1.21 ± 0.03 μm, p = 0.405 ± 0.007 μm/s and kcd = 1.00 ± 0.03 s−1 is derived from Rdrop and p. d Spider chart summarizes the properties of FUS NTD condensate and represents its REDIFINE fingerprint. 10 ms gradient length was used in (a, b). The uncertainties for each parameter are calculated using covariance matrix and report on the ambiguity of determined values. e FRAP experiment performed on FUS NTD spiked with 1% Atto488-labeled FUS NTD in 0.5% agarose. Data are presented as mean values +/− the standard deviation from the measurements on 4 different droplets. We therefore introduced the model outlined in the theoretical section of the Methods and in Notes 2 and 3 of Supplementary Information to interpret these decay curves. Our model considers a population of protein \({\nu }_{{cond}}\) inside the condensed droplets and \({\nu }_{{dil}}\) in the dilute phase in equilibrium that can exchange between each other, given by \({\nu }_{{cond}}{k}_{{cd}}={\nu }_{{dil}}{k}_{{dc}}\) where \({k}_{{cd}}\) and \({k}_{{dc}}\) are the forward and reverse exchange rates, respectively. The droplet-bulk interface is characterized by a permeability for a molecule to move between the phases, and it is defined as the permeability factor p. The exchange rate from condensed to dilute phase \({{{\rm{k}}}}_{{{\rm{cd}}}}\) is expressed as \({k}_{{cd}}=\frac{3p}{{R}_{{drop}}}\) where \(\frac{3}{{R}_{{drop}}}\) the surface-to-volume ratio of a sphere38. As the \({D}_{{cond}}^{{app}}\) is too slow to be measured by conventional NMR probes due to technical limitations, our method exploits the chemical exchange between the two phases to relay the information about slow restricted diffusion onto more easily observable NMR signals of the dilute phase. Hence the name LLPS REstricted DIFusion of INvisible speciEs (REDIFINE). The framework of REDIFINE model is illustrated in Supplementary Note 2. We ran extensive simulations to show how various parameters including \({R}_{{drop}}\), permeability and \({\nu }_{{cond}}\) affect diffusion curves. Not surprisingly, the choice of diffusion time Δ influences the extent of the modulation in diffusion curves caused by different condensate parameters (Supplementary Note 2). Using REDIFINE simulation, we could fully reproduce the acquired data for FUS NTD. As the chemical exchange is not a priori known, our simulations also revealed that in order to unambiguously determine condensate parameters by REDIFINE, multiple experiments with several diffusion times Δ, are required (Supplementary Note 2). To test this, we subjected FUS NTD sample prepared in agarose to LLPS REDIFINE, integrating signals stemming from the aromatic protons of tyrosines (Fig. 1a), which were assigned based on their characteristic chemical shift. REDIFINE provided excellent global fitting to the curves (Fig. In just 2 h of experimental recording followed by REDIFINE fitting, key FUS LLPS properties could be determined including FUS diffusion constants in both phases (8.0 × 10−11 and 8.9 × 10−13 m2/s for the dilute and the condensed phases, respectively), the fraction of the protein in the droplets (νcond of 0.75 at 600 μM FUS NTD) and the droplets average radius (1.2 μm) as well as their permeability (0.4 μm/s) from which we can derive the exchange rate from condensed to dilute phase (kcd = 1 s−1). These parameters of the biphasic sample obtained with REDIFINE can be illustrated by the spider chart shown in Fig. 1d, which represents a fingerprint for FUS NTD LLPS. We can also define a global exchange rate that accounts for the partitioning between phases \({k}_{{ex}}={\nu }_{{cond}}{k}_{{cd}}={\nu }_{{dil}}{k}_{{dc}}\), which can be somewhat comparable to the rates obtained in fluorescence recovery after photobleaching (FRAP)39,40 experiments. This results in a \({k}_{{ex}}\) rate of 0.75 s−1, corresponding to a half-life time (\({t}_{1/2}\)) of 0.9 s according to the equation \({t}_{1/2}=\frac{{\mathrm{ln}}2}{{k}_{{ex}}}\). This result is very consistent with the FRAP data acquired on the FUS QGSY-rich region41 (recovery \({t}_{1/2} \sim 1{{\rm{s}}}\)) and yet with another unstructured protein, α-Synuclein42 (recovery \({t}_{1/2} \sim 3{{\rm{s}}}\)). LLPS REDIFINE droplet size is also in very good agreement with the values determined by a thorough analysis of fluorescent images taken for FUS NTD in agarose19, which further indicated that FUS NTD droplet size does not depend on initial protein concentration. We prepared two additional FUS NTD samples at a concentration of 200 μM and measured REDIFINE three times for each sample. Supplementary Note 4 summarizes these measurements and confirms very consistent fitting parameters throughout the technical and biological replicates. As an independent control, we prepared a fresh sample of FUS NTD in 0.5% agarose and visualized it under the microscope (Supplementary Fig. FUS NTD droplets stabilized in agarose indeed have a very uniform size of circa 1 μm in radius, validating the REDIFINE methodology. To test droplet dynamics, we performed FRAP experiments by adding 1% FUS NTD which was labeled with Atto488 dye (Supplementary Fig. Figure 1e shows the fluorescence recovery upon bleaching of the FUS NTD droplet, indicating liquid-like condensate properties. Due to the low bleaching efficiency of Atto488 dye, the full laser power was required to reach a significant reduction in fluorescence. The FRAP recovery that we obtained (9 ± 4 s−1) was slower than the REDIFINE exchange rate and the literature values. We attribute this discrepancy to the local heating effects caused by the high-power irradiation, as the bleached droplet slightly changed shape over time (Fig. This could also be a reason for the lower recovery levels that we achieved, as high irradiation power can cause degradation and/or enhanced oligomerization of FUS NTD. Next, we extended REDIFINE to the full-length FUS, which, despite the presence of two folded RNA binding domains, is still mostly unstructured (with only 8.9 % of the sequence adopting a stable structure). Akin to FUS NTD, full-length FUS spontaneously forms droplets by itself. At a concentration of 200 μM, the large majority of full-length FUS is found inside the condensed phase based on an almost non-decaying signal in NMR diffusion experiment19 (Fig. 3a shows the LLPS REDIFINE fitting for this FUS biphasic sample. From REDIFINE, we indeed found that 90% of the protein is inside the droplets (νcond = 0.9, translating to 1.6 mg of protein) and consequently very low population of FUS outside which is primarily why the diffusion constant of the protein in the dilute phase could not be accurately determined. Furthermore, in this biphasic condensate of FUS full-length, we could quite surprisingly spectroscopically observe a small population of slowly diffusing water with a shifted frequency implying different interactions in the condensed phase. This population corresponds to structured water43,44,45 found in the droplets, and as the water decay curve showed a complex biexponential shape (Fig. 2b), we could apply LLPS REDIFINE to the water signal as well. 3b presents an excellent fit of the water LLPS REDIFINE data, confirming a droplet radius of around 1 μm consistent with the radius obtained independently using the FUS FL signal decay. REDIFINE revealed also that there is around 3% of water trapped in the droplets (accounting for a volume of 4 μl of water). Similar results are detected for FUS NTD, though with slightly lower water content in the droplets (Supplementary Note 5). Due to such a small population of water in the droplets, the uncertainty for the diffusion coefficient in the condensed phase is large, but still revealed that water is diffusing two to three orders of magnitude slower than in the dispersed phase and within the same range as the FUS protein. FUS condensate parameters of the protein and the water are summarized in Fig. 2c, d. As expected, they both report the same size, however the interface permeability for water is 3-fold larger than for the protein. Knowing the initial amounts of protein and water, and accounting for protein and water fractions in the condensed phase result in a FUS concentration of 6.8 mM in the droplets and correspondingly 20 μM in the dilute phase (340−fold less) as illustrated in Fig. Although these correspond to the concentrations of a macromolecule in a biphasic LLPS sample in equilibrium, our result is in agreement with previously reported concentrations for the FUS protein in a single condensed phase after physical separation of the two phases41. a A series of 1D proton spectra showing full-length FUS signal with increasing gradient strength in NMR diffusion experiment. Non-decaying signal implies a very high population of slow-diffusing FUS in the droplet. b Similarly, a series of 1D proton spectra showing the water signal from the same FUS sample. Note that based on different chemical shifts, there is a distinct population of bound water in the droplets. Water diffusion also exhibits biexponential behavior. c Summary of LLPS REDIFINE data set acquired on both protein and water signal, providing multicomponent characterization of FUS FL condensate. d LLPS REDIFINE allows us to “visualize” the FUS FL condensates, providing average droplet size, interface permeabilities, and concentrations in corresponding phases. e Fluorescent images acquired on FUS FL sample using different fluorescent tags. f FRAP recovery time course for GFP and mCherry tags. Data are presented as mean values +/− the standard deviation from the measurements on 4 different droplets. Experiments in (a) are acquired using 75 ms diffusion encoding time and 10 ms gradient length while in (b) for water 70 ms and 2 ms respectively. The uncertainties for each parameter in (c) are calculated using a covariance matrix and report on the ambiguity of determined values. The representative images are from three independent experiments. Again, to independently compare our measurements with other methods, we added 1% of Atto488 labeled protein to visualize condensates by fluorescent microscopy. This labeling unfortunately caused the whole sample of FUS FL protein to aggregate (Fig. 2e) possibly due to the presence of 15 lysine residues in FL protein where the dye can be potentially bound affecting the net charge and leading to cumulative effects of increased hydrophobic interactions. On the contrary, by spiking FUS FL with 1% GFP- and mCherry-tagged fusion protein, we could observe liquid-like FUS FL droplets. FRAP experiment showed similar recovery for both GFP- and mCherry-tagged protein (Fig. 2f) while the aggregated sample with Atto488 label never recovered. Akin to the FUS NTD, the FRAP recovery rates are by the same amount slower compared to REDIFINE chemical exchange. Impaired bleaching recovery correlated with an observation that FRAP rates closely depend on the size of the droplet that was bleached (Supplementary Fig. Faster FRAP recovery detected after partial bleaching of larger droplets compared to the small ones can be rationalized by the steep reduction in exchange efficiency when the fraction of protein in the dilute phase is below 10% (Supplementary Fig. As the exchange process between phases depends on droplet size and populations in each phase, REDIFINE provides much more comprehensive analysis of the condensates than FRAP. In addition, the example of FUS FL further shows evidence that even a small amount of fluorescent tag can affect phase separation and even introduce bias in the fluorescent measurement raising the demand for the development of label-free methodologies. Still puzzled by the difference between REDIFINE and FRAP, we analyzed another intrinsically disordered N-terminal domain from DDX4 protein (SSP = 0.052, Supplementary Figs. Diffusion constant in the condensed phase of around 7 × 10−13 m2/s that we determined with REDIFINE perfectly matched with the value obtained on a monophasic dense phase of DDX4 using very elegant NMR experiments that exploit methyl triple-quantum states for diffusion encoding47. Besides the protein-specific diffusion and population parameters, we could observe that the average droplet size (Rdrop = 1.05 μm) and global chemical exchange rate (kex = 0.61 s−1) of DDX4 condensates is very similar to FUS indicating a potentially common mechanism of condensate formation for unstructured proteins. On this sample, we also measured an independent NMR experiment that could be adapted to LLPS systems to measure directly the rate of chemical exchange between phases. Filter-Exchange NMR Spectroscopy (FEXSY)48,49,50 can filter out the signal stemming from the dilute phase and consequently measure the rate of signal recovery as a result of chemical exchange from the condensed phase. The fitting of FEXSY datasets confirmed the same exchange rate and protein fraction in the condensed phase as determined by LLPS REDIFINE (Supplementary Fig. To further test this, we compared REDIFINE and FEXSY on another biphasic sample of FUS FL incubated for two months at room temperature (Supplementary Fig. Again, the chemical exchange determined by REDIFINE and FEXSY closely matched. Note that due to its lower sensitivity, FEXSY experiments took 5-fold longer to acquire compared to LLPS REDIFINE but could not report on interface permeabilities, droplet size, and reliably on diffusion coefficients. Besides previously discussed systems, many proteins can undergo LLPS only in the presence of binding molecules, forming bi- or multimolecular condensates3. A well-known example is the SARS-CoV-2 Nucleocapsid protein that binds negatively charged RNA, which in turn induces condensate formation. Nu readily phase-separates with substoichiometric quantities of structured viral RNA fragments such as the stem-loop-2 s2m RNA (Supplementary Fig. DOSY decay curves acquired on free Nu can be easily fitted to a single exponential decay indicative of a single dispersed phase (Supplementary Fig. With RNA present, phase separation occurs, and decay curves experience a steep decrease of the signal with no significant plateau compared to FUS samples. The data could not anymore be quantitatively explained by a single exponential function, however fitting with LLPS REDIFINE (Supplementary Fig. 7b, c), allowed delineating the exchange-averaged diffusion information about the Nu condensates. \({D}_{{cond}}\) is circa two orders of magnitude slower than \({D}_{{dil}}\), and around 27% of the protein resides in the droplets. The average exchange rate is determined to be 4.8 s−1 which is faster than for fully unstructured proteins, explaining the steeper diffusion decay at higher gradient strengths. The size of the droplets determined by REDIFINE is around 0.8 μm, which is confirmed by fluorescence microscopy (Supplementary Fig. As Nucleocapsid protein can promiscuously interact and form droplets even with short single-stranded RNAs, we then examined a Nu condensate bound to the 6-mer poly-A RNA (A6). Maximum turbidity is reached at 6 equivalents of this single-stranded RNA (Supplementary Figs. Unlike s2m, the NMR RNA signals are now detectable allowing us to apply LLPS REDIFINE to both the protein and RNA signals (Fig. This is equivalent to multicolor fluorescent labeling, however without any specific dye as every type of polymer possesses a distinct the NMR signature. Figure 3a illustrates the aliphatic peaks of the protein and aromatic resonances of the A6 RNA that we used for observing the decay and fitting with REDIFINE. Reassuringly, both protein and RNA (Supplementary Fig. 3b) data sets independently report the same average droplet size but more surprisingly we obtained a significantly different permeability for the interphase exchange. We could see that A6 RNA exchanges almost eight-fold faster than the Nucleocapsid protein, probably due to being smaller and in large excess. Although having a better fit (χ = 0.23 vs 0.70), the uncertainty of the RNA's \({D}_{{cond}}\) is large (Supplementary Fig. 8b) due to fast chemical exchange averaging. From the equal fractions of protein and RNA in the droplet (~0.3), it can be implied that the condensate contains 6 equivalents of A6 for one Nu protein in both phases, which is the ratio needed to reach maximum turbidity (Supplementary Fig. a Nucleocapsid:A6 condensates prepared at 1:6 protein:RNA ratio. Both protein and RNA peaks are visible in NMR spectrum and are analyzed separately by LLPS REDIFINE. Spectra are acquired using 75 ms diffusion time and a gradient length of 10 ms for protein and 3.5 ms for RNA. b LLPS REDIFINE results obtained from both Nucleocapsid and RNA side. c Overlay of condensate properties obtained on condensates formed by Nucleocapsid protein and structured s2m RNA and by another structured protein, PTBP1 in the presence of 3 × UCUCU RNA. The uncertainties for each parameter are calculated using covariance matrix and report on the ambiguity of determined values. In contrast to the Nucleocapsid that promiscuously binds the RNA, we also investigated the multidomain PTBP1 protein that forms a specific complex of high affinity with an RNA containing several UCUCU motifs. With 0.1 equivalent of 3 × UCUCU RNA (Supplementary Figs. 6a, b, and 8c), PTBP1 readily phase separates, and at room temperature, around 50% of PTBP1 is found in droplets (consistent with independent spectrophotometric measurements). Surprisingly, we found that PTBP1 is exchanging very quickly between the condensed and dilute phases (90 s−1, see Supplementary Fig. These rates are very high on the diffusion time-scale (tens to hundreds of milliseconds) and REDIFINE could not determine unambiguously the diffusion coefficients and the droplet size. However, at 278 K the droplets are still present in the sample and the \({k}_{{cd}}\) rate is significantly reduced (around three-fold) allowing REDIFINE to provide information about the droplet size and permeability as well as the diffusion of PTBP1 in the condensed phase (Supplementary Fig. For comparison, Nu:s2m results from Supplementary Fig. 7 are also presented in Fig. 3c, showing a very similar REDIFINE fingerprint to Nu:A6 condensate and even to PTBP1:3 × UCUCU in terms of diffusion constant, size, and fraction of proteins, despite being different in terms of droplet permeability and exchange. This suggests that RNA-induced condensates have similar droplet properties. Altogether, we showed here that LLPS REDIFINE allows the biophysical characterization of multicomponent biological condensates such as protein-RNA condensates (Supplementary Table 1), and the method could be easily extended to multiprotein or drug-protein condensates. After examining more than 15 different condensate samples involving several different proteins and RNAs (Supplementary Table 1), we indeed observed an intriguing trend in the droplet properties among different systems. The condensates of unstructured proteins generally tend to form larger droplets and experience slower interphase exchange (i.e., FUS, DDX4). Proteins with structured domains that require the addition of RNA to phase separate exchange faster and usually form smaller droplets (i.e., Nu, PTBP1). This is illustrated in Fig. 4a by a 2D plot showing an inverse correlation between the condensed-to-dilute exchange rate and the droplet size. Interestingly, the FUS biphasic sample in the presence of RNA (green dot) contains droplets that are more dynamic and closer to the region defined by structured protein condensates which phase separated in the presence of RNA (Supplementary Fig. The effect of protein structural properties on droplet dynamics may find its origin by the positive correlation between the global chemical exchange rate and the ratio between the protein secondary structure prediction factor (SSP) and the protein net charge Z (Fig. The fastest exchange is observed for PTBP1 that has the largest SSP/Z ratio while the fully unstructured FUS NTD and DDX4 LCD domains have the slowest rate and smallest SSP/Z ratio (Fig. A possible explanation for this relationship is proposed in Fig. 4c that highlights the smaller potential for multivalent interactions in structured RBPs compared to fully unstructured proteins to form condensates. a A correlation plot of chemical exchange vs. the average radius of the droplets showing that disordered proteins make larger droplets that exchange slower to the dilute phase compared to RNA-binding structured proteins. The latter cover much broader space with much more heterogeneous interphase chemical exchange. Addition of RNA to one of the FUS samples shifted the position in the plot closer to RBPs suggesting that the RNA makes droplets more dynamic (green dot). b A plot of average global exchange rate determined by LLPS REDIFINE with respect to the ratio between protein secondary structure prediction factor SSP and protein net charge Z. An obvious correlation could be deduced. Values presented for FUS NTD, FUS and Nu condensates are the averages with standard deviation of the results from multiple (n > 3) different samples containing these proteins while for DDX4 and PTBP1 are the fitting uncertainties. c A potential explanation for such behavior. RBDs are mostly folded proteins that protect their hydrophobic parts with their fold and can engage in far less polyvalent binding interactions compared to IDPs. Therefore, IDPs have larger affinity to be in the condensed, interaction-reach phase. The concept of restricted diffusion in biomolecular condensates occurs as a result of the droplets' interfacial tension52. Polyvalent interactions in the condensed phase prevent molecules from diffusing to the dilute phase, manifesting as complex diffusion. Interestingly, even dispersed molecular complexes experience restricted diffusion. Being in dynamic equilibrium with the complex, constituent molecules move in solution as they are constantly being dragged by these intermolecular interactions, which makes their diffusion coefficient a function of interaction affinity and kinetics. Diffusion NMR experiments are sensitive to detect this apparent restricted diffusion only when diffusion coefficients of free molecules and complex are significantly different and in an intermediate exchange regime (10−1000 ms). Fortuitously, this is often the case with protein-RNA complexes prone to phase separation, resulting in complex, exchange-averaged diffusion detectable by the NMR experiment. Considering this, we extended our model with a multi-pool exchanging system in a dilute phase. This is illustrated in Fig. This can be used to further calculate the dissociation constant \({K}_{d}\) and the on-rate \({k}_{{on}}\) (“Methods”). We applied the methodology to soluble PTBP1 complex with one equivalent of 3 x UCUCU RNA (Fig. Using this extended model, we obtained a population for the complex of 0.945 ± 0.006 and a \({k}_{{off}}\) of 1.41 ± 0.02 s−1. Based on the previous experience with this system, we presumed a 1:1 binding stoichiometry and could determine a \({K}_{d}\) of 500 ±100 nM. This value is in very good agreement with the independent electrophoretic mobility shift assay (EMSA) performed for the same complex with a measured \({K}_{d}\) of 490 ± 20 nM (Fig. The additional advantage of our methodology is that it involves a single protein–RNA sample for an acquisition/processing time of only 2 h and importantly does not require titration. a REDIFINE with dispersed complexes. If biomolecules coexist in solution in multiple exchanging states with different diffusion coefficient (protein/RNA and a complex), then PGSTE curve contains exchange averaged information of all the states present in solution. If exchange happens to be in intermediate regime, REDIFINE approach can be used to calculate \({K}_{d}\) and \({k}_{{on}}\) rate based on fitted \({k}_{{off}}\) and the complex population parameters. b Fitting of REDIFINE data for PTBP1 : 3 × UCUCU. \({K}_{d}\) of 500 ± 100 nM is determined. Values of the parameters are the average values of two independent measurements and two independent processing (different integration regions) with the standard error. c Native gel showing the 3 × UCUCU RNA shift upon binding to PTBP1 protein and ensuing fitting of the protein-RNA complex band formation. \({K}_{d}\) of 490 ± 20 nM is determined which is in agreement with the REDIFINE. The \({K}_{d}\) value is the mean of three independent EMSA experiments with standard error. Here we presented a methodology that can be used to study liquid–liquid phase separation and complex biophysical systems in general. REDIFINE approach allowed us to gain detailed insights into the physical properties of biphasic condensates by examining them in vitro in their biologically relevant form. By determining diffusion coefficients in dilute and condensed phases co-existing in equilibrium, partition coefficients, average droplet size, surface permeability, and chemical exchange, REDIFINE provides a fingerprint for biomolecular condensates, akin to fluorescent microscopy LLPS assays, FRAP, and microrheology combined25, though acquired in a single NMR experiment and without requiring any specific detection labels. As NMR spectroscopy provides distinct molecular fingerprints, LLPS REDIFINE can be applied to any species present in the condensates, given that they are observable by NMR, i.e., in multicomponent condensates. Although we primarily focused on monitoring proteins (FUS, DDX4, PTBP1, and SARS-CoV-2 Nu), we could also observe the properties of RNA and water within the condensates. This multicomponent characterization, often not accessible by other techniques, can be extended to ions such as phosphate, chloride, and sodium53 that often influence phase separation behavior, but also multiprotein and drug-protein condensates. Applying LLPS REDIFINE on various protein condensates, we observed an unexpected trend in droplet properties. We could see an inverse correlation between the droplet size and the chemical exchange between the condensed and dilute phases for our samples. This might be explained by the fact that smaller droplets have a larger surface-to-volume ratio promoting more efficient exchange processes54. Moreover, RBPs containing structured domains tend to form smaller and more dynamic droplets compared to mostly unstructured proteins. We hypothesize that this is due to the much smaller valency of structured RBP homotypic interactions compared to IDPs. This is also why structured proteins need an anionic RNA to significantly increase the polyvalency of interactions in order to enable phase separation, while IDPs can engage in interactions with other protein molecules and phase-separate on their own. Once in the droplets without tertiary structure protecting their hydrophobic or other interaction-prone groups, IDPs have a higher affinity to remain in the high-valency droplet environment compared to the dilute phase. This would explain consistently slower exchange rates among IDPs. LLPS REDIFINE requires for NMR analysis only one sample of unlabeled biomolecules stabilized in agarose hydrogel. We performed an extensive analysis to show that the loose mesh of 0.5% agarose does not affect the structure and behavior of the large biomolecules of interest (Supplementary Note 6). A minimum of 100-150 μM protein in 130 μl sample volume is needed for analysis, although even lower protein concentrations can be used since NMR experiments can be acquired with more extensive signal-averaging. As the spectral resolution is not essential, LLPS REDIFINE is in theory also applicable at low-field benchtop machines with gradient capabilities. Currently, LLPS REDIFINE is based on a model that considers droplets of uniform size. Consequently, it provides information about the average droplet radius that best fits the experimental data. Although agarose-mimicking cytoskeleton prevents droplet fusion, in general, the droplets in the biphasic sample are somewhat heterogeneous in size. Although not encountered in this study, very heterogeneous size distributions can affect the fitting. Principally, if the distribution shape is assumed (normal or log-normal) LLPS REDIFINE could provide also a standard deviation of the distribution besides the average radius55,56,57,58 and this is currently being investigated. It should be mentioned that very fast exchange could also constrain the applicability of REDIFINE, which is, in principle, limited to the kinetic processes occurring on a millisecond to second time scale. Other limitations are discussed in the Supplementary Information. Furthermore, while the model currently accounts only for the surface-to-volume ratio of the droplets, it is also possible to include different liquid droplet shapes, other than spheres. This could be ellipsoids and cylinders for example38, which could be applicable in the case of aggregated condensates and also for in cell (bacterial and eukaryotic) studies, which are currently being explored. Biphasic condensates exhibit complex diffusion behavior that manifests as a deviation from mono-exponential trend and resembles the so-called kurtosis effect59 utilized traditionally in Magnetic Resonance Imaging (MRI). This kurtosis effect, defined with a parameter K, represents a deviation from gaussian diffusion (Supplementary Note 7) and implies a physically-restricted molecular motion36. Diffusion kurtosis imaging of water has proven to be very sensitive to the tissue type and is conventionally used to assess structural abnormalities in the brain associated with neuropathologies. It is illustrated in Supplementary Note 7 that molecular diffusion in all biomolecular condensates experiences a kurtosis effect to a different extent. Following this link, we anticipate that REDIFINE methodology can be eventually utilized for in cell NMR microimaging60 to spatially characterize various membraneless assemblies. Additionally, this methodology exhibits a broad range of applications, such as for soluble biomolecular complexes, as it can provide thermodynamic and kinetic information of protein-RNA binding in the intermediate to fast exchange regime. As the diffusion coefficient of a small molecule becomes significantly slower upon binding to a large biomolecule, we anticipate that our approach can be exploited in drug screening campaigns as ligand-observed experiments. Altogether, LLPS REDIFINE provides a label-free approach to study droplet properties in a biphasic environment for protein-only condensates but also for homogenous multicomponent condensates composed of protein and RNA. Application to multiprotein and protein-drugs condensates is underway. Considering the importance of the surface of biological condensates for droplet maturation and reaction kinetics61,62,63,64,65, the exchange parameters obtained by LLPS REDIFINE will be crucial to understand the role of condensates in biology and disease. Since the early development of the pulsed gradient spin echo (PGSE) technique by Stejskal and Tanner in 196566, diffusion measurement experiments have been extensively used in the field of NMR and MRI67,68,69. In the conventional diffusion experiment, a normalized signal of a studied analyte is mapped as a function of the gradient amplitude applied in the experiment. The application of gradients can impart a specific phase to molecules depending on their position in the NMR tube, and their translational diffusion will lead to the mixing of different phases, ultimately resulting in decay of the signal. The attenuation of the observable normalized signal depends on the experimental parameters and diffusion rates (Supplementary Note 1) and follows a simple mono-exponential decay66 according to the formula where parameter \(b={\gamma }^{2}{g}^{2}{\delta }^{2}\cdot \Delta\) \({{\rm{\gamma }}}\) is the nuclear gyromagnetic ratio, \(g\) and \(\delta\) the strength and duration of the gradient coming from the NMR probe, \(\Delta\) diffusion time, and finally \(D\) a diffusion coefficient. It is often sensible to write this equation as where \(q=\gamma g\delta\) it excludes diffusion time and depends only on the pulse gradient parameters. Usually, \(\delta\) it is kept constant, but should be adjusted according to the molecular weight of the analyte. When this dependence is visualized on a semi-logarithmic plot as Then, a slope of the line simply represents the diffusion coefficient (Supplementary Note 1). Laying the foundations of LLPS REDIFINE (REstricted DIFusion of INvisible speciEs) methodology, the model considers a biphasic sample containing a relative population of protein νcond in condensed and νdil in dilute phase. For simplicity, we considered a mono-component condensate formed solely by one type of biomolecule (i.e., FUS condensates). Since the chance for hindered diffusion outside the droplets is very low due to the relative volumes of condensed and dilute phase, we consider free diffusion of biomolecules in the dilute phase described with the diffusion coefficient \({{{\rm{D}}}}_{{{\rm{dil}}}}\). Diffusion of molecules inside the condensed droplets is restricted by the interfacial tension caused by the very different interaction affinity within the two phases. Related NMR diffusion measurements were first applied in the 2000s to characterize emulsion systems based on restricted diffusion55,56,57,58. This defines certain permeability38,70 of droplet interface described with permeability factor \({{\rm{p}}}\). Restricted diffusion will depend on the size of the droplets, and we assume perfect spherical geometry for the condensed droplets defined by the average radius \({{{\rm{R}}}}_{{{\rm{drop}}}}\). For the sake of simplicity, if we use \({{\rm{q}}}={{\rm{\gamma }}}{{\rm{g}}}{{\rm{\delta }}}\), the restricted diffusion36,37,38,70,71,72,73 can be expressed as: To describe the chemical exchange between the condensed and dilute phase, we used the exchange rate \({{{\rm{k}}}}_{{{\rm{cd}}}}\) that is dependent on the droplet surface-to-volume ratio and interface permeability. As the surface-to-volume ratio for the spheres is equal \(\frac{3}{{R}_{{drop}}}\), then the chemical exchange rate38 is given by: If the decay of magnetization \({{\rm{M}}}\) due to the gradients of PGSE experiment can be expressed in the differential form as: Then the evolution of the biphasic system in exchange can be shown as: where \({k}_{{dc}}=\frac{{\nu }_{{cond}}{k}_{{cd}}}{{\nu }_{{dil}}}\) and \({\nu }_{{cond}}+{\nu }_{{dil}}=1\) as the system is at equilibrium during rather short measurement time. We can also define a global chemical exchange as a rate more comparable with the rates obtainable by FRAP experiments. The system of differential equations in Eq. can be solved analytically, and the total magnetization at given time \({{t}}=\Delta\), and gradient strength and length \({{\rm{g}}}\) and \({{\rm{\delta }}}\), respectively, is given by: Solution for \(S\left({q}^{2},\varDelta \right)\) has been used to fit the experimental data. There are 5 independent physical parameters that describe the model system: \({D}_{{dil}}\), \({D}_{{cond}}\), \({\nu }_{{cond}}\), \({R}_{{drop}}\), and average permeability \(p\). The diffusion decay curves for various \(\Delta\) have been fit globally at the same time. Fitting was performed using fminsearch, a nonlinear programming solver in MATLAB, that finds the minimum of multivariable function. The uncertainties of fitted values were calculated using the covariance matrix of parameters. Covariance matrix reports on the correlations between all the parameters and therefore reflects on the ambiguities about their fitted values. In other words, large uncertainty reports that the parameter is not well correlated with the other parameters and therefore cannot be determined with certainty by the multiparametric fitting procedure. For example, this is the case with diffusion coefficient \({D}_{{cond}}\) in case of low population of the condensed phase (<0.2) or fast chemical exchange (>30 s−1). The model to calculate binding affinity in soluble complexes considers a two-pool system with different isotropic diffusion coefficients (as in a dispersed sample) that can exchange with each other. There are 4 independent parameters that describe the model system: \({D}_{{dil\_free}}\), \({D}_{{dil\_complex}}\), \({\nu }_{{complex}}\), and \({k}_{{off}}\). Based on known initial concentrations of the reactants, the population in the bound state (\({\nu }_{{complex}}\)), and a priori determined binding stoichiometry, one can calculate the equilibrium concentrations of the species in a free state. Having all the equilibrium concentrations, a dissociation constant can be determined as Based on the \({k}_{{off}}\) and \({K}_{d}\), one can also calculate the \({k}_{{on}}\) rate of complex association: Full-length Nucleocapsid construct was cloned into a pESPRIT vector between the AatII and NotI cleavage sites with His6-tag and TEV protease cleavage sites at the N terminus (GenScript Biotech, The Netherlands). All Nucleocapsid protein constructs were expressed in Escherichia coli BL21 (DE3) overnight at 18 °C after induction at an optical density of 0.6 with 0.6 mM IPTG. Cells were harvested by centrifuging at 3,750 x g and resuspended in buffer containing 20 mM Tris-HCl, pH 8.0 and 1 M NaCl. The cells were lysed by cell cracking and the lysate centrifuged again at 17,000 × g at 4 °C. The supernatant was subjected to standard Ni-NTA purification. Following TEV cleavage and removal of the excess N-terminal tag and TEV by Ni-NTA affinity, samples were concentrated and exchanged to 20 mM sodium phosphate, pH 6.0, 50 mM NaCl (NMR buffer). For certain protein preparations, when purity was not satisfactory, it was additionally subjected to size exclusion chromatography (SEC; Superdex 75/200) into the NMR buffer. The coding sequence of PTBP1 was cloned in pET28a (Novagen). All plasmids were sequenced and transformed into BL21-Codon Plus (DE3)-RIL cells (Agilent Technologies) for protein expression. Protein purification of PTBP1 was performed as described previously74. The NMR samples of PTBP1 were prepared in 10 mM sodium phosphate buffer with 20 mM NaCl at pH 6.5. Full-length FUS sequence was cloned into a pET24b vector by XhoI and BamHI restriction digestion and ligation that enables the translation of a TEV-cleavable GB1-His6-tagged protein. RNA-binding mutants were generated by classical site-directed mutagenesis. FUS protein constructs were expressed in Escherichia coli BL21(DE3) overnight at 20 °C after induction at an optical density of 0.6 with 0.1 mM isopropyl-β-d-thiogalactopyranoside (IPTG). Cells were harvested by centrifugation at 5,000 ×g and were directly resuspended in Suspension Buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, protease inhibitors). Two times 12 min sonication on ice for 7 cycles with pulse-pause intervals at power level 70% was used to lyse the cells. The cell lysate was centrifuged at 15,000 × g at 4 °C, and the resulting cell pellet was homogenized on ice (15 mL, Dounce Homogenizer) in Buffer A (8 M urea, 50 mM HEPES, pH 7.5, 500 mM NaCl). The homogenized protein solution was centrifuged again at 15,000 × g at 4 °C for 25 min. The supernatant was collected and subjected to Ni-NTA purification. The Ni-NTA (Qiagen) column was equilibrated with Buffer A. After loading the filtered protein solution onto the column, the column was washed with 2 column volumes of Buffer A and 4 column volumes of Buffer A1 (1 M urea, 50 mM HEPES, pH 7.5, 150 mM NaCl). TEV protease and 5 mM 2-mercaptoethanol were added into the eluted protein solution followed by a dialysis in Buffer B (1 M urea, 50 mM HEPES, pH 7.5, 150 mM NaCl, 250 mM imidazole, 5 mM ME) overnight, prior to a second dialysis in Buffer C (6 M urea, 50 mM HEPES, pH 7.5, 150 mM NaCl). A second Ni-NTA purification was performed by applying the protein solution on a Ni-NTA column equilibrated in Buffer C. Six column volumes of Buffer C were used for washing. The flow-through of protein solution and washes were collected together and concentrated via centrifugation using 15 mL Amicon UltraCentrifugal Filter Units (Millipore) with a molecular weight cut-off of 30 kDa. N-terminus disordered domain of DDX4, DDX4(1−233), and FUS NTD low-complexity domains were expressed and purified as previously described19. The final buffer used in LLPS REDIFINE was 30 mM HEPES with 200 mM KCl at pH 7.5 with a residual urea concentration of 0.6 M. For reasons unrelated to the present study FUS NTD (600 μM sample in Fig. S2m (5′-GGUUCACCGAGGCCACGCGGAGUACGAUCGAGUGUACAGUGAACC-3′), 3 × UCUCU (5′-GGGAGAUCUCUAAAAAUCUCUAAAAAUCUCUAAAAA-3′), and SL34 (5′-GGGAUCCGAUUUCCCCAAAUGUGGGAAACUCGACUGCAUAAUUUGUGGUAGUGGGGGACUGCGUUCGCGCUUUCCCCUG-3′) RNAs were produced by in vitro run-off transcription with T7 RNA polymerase (purified in-house) from two complementary DNA primers containing a T7 promoter or in the case of SL34 based on previously reported plasmids75. Magnesium concentration was optimized for in vitro transcription reactions with both commercially available unlabeled NTPs (Applichem). The RNAs were purified by anion exchange chromatography in denaturing conditions. The purified RNA was precipitated by butanol extraction to eliminate urea and salts. Lyophilized RNA was re-suspended in water or respective buffers. A6 RNA oligo was purchased from Dharmacon. The turbidity (light scattering at 600 nm) of the samples was measured using a UV-Vis spectrophotometer (ND-1000 Spectrophotometer, NanoDrop). For Nu, a defined volume of protein and RNA stock solutions was diluted in Nucleocapsid NMR buffer (20 mM sodium phosphate, pH 6.0, 50 mM NaCl) to yield a 60 µM of protein and varying RNA concentrations in a total reaction volume of 20 µL. The RNA is added in the last step to initiate the phase separation, and the measurements are performed after 1 min of incubation in two technical replicates. In total, three biological replicates were acquired using the same procedure for Nu:s2m and two biological replicates (each with 2 technical replicates) for Nu:A6. A similar procedure was used for PTBP1 and SL34 (3 biological replicates) prepared at 100 µM protein concentration in PTBP1 buffer (10 mM sodium phosphate, pH 6.5, 20 mM NaCl) in a total reaction volume of 10 µL. For measurements of FUS, 20 µM protein samples were prepared in FUS Turbidity Buffer (5 mM potassium phosphate, pH 6.0, 100 mM KCl, 2 µM ZnCl2, 1 mM TCEP) and measured in three replicates with varying concentrations of RNA. All biphasic samples examined in this study with LLPS REDIFINE method were prepared in 0.5% final mass concentration of agarose in order to stabilize liquid droplets as previously described19. For structured protein, low-melting agarose is used while for unstructured proteins we used a regular agarose. Corresponding protein buffers were first boiled with agarose powder to solubilize it, and then kept warm to remain liquid. Agarose buffers used to prepare FUS and DDX4 samples were still liquid at 55 °C, while the low-melting agarose buffer is in liquid state even at 36 °C. It is crucial to use low-melting agarose with folded proteins in order not to denature them by the addition of very hot agarose stock buffer. To form a biphasic sample stabilized by agarose hydrogel, protein (and RNA) stocks were mixed with respective warm agarose buffer in a 1.5-ml Eppendorf and quickly transferred to the NMR tube. Agarose gelation occurs shortly after transferring to the NMR tube, and the sample is ready for analysis. Note that the final samples of FUS and DDX4 also contain residual urea19. FUS NTD (100 μΜ), full-length FUS (150 μΜ), Nucleocapsid:A6 (110:660 μΜ), and PTBP1:3 × UCUCU (150:15 μΜ) samples were prepared in 0.5% agarose hydrogel to ensure the same conditions as in REDIFINE experiments and imaged using bright-field microscopy. FUS NTD, FL FUS and Nucleocapsid protein were labeled post-translationally with Atto488 dye (Jena Bioscience) on primary amino groups as per the manufacturer's protocol. Unbound dye was removed from the labeled protein by overnight dialysis against fresh protein buffer. GFP- and mCherry-labeled FL FUS were expressed as fusion proteins and expressed as described above for FUS FL. Enzymes and unbound dye were removed by phenol-chloroform extraction and ethanol precipitation. Unlabeled FUS NTD and FUS FL were spiked 100:1 with Atto488 and GFP/mCherry-labeled protein. All samples except for Nucleocapsid:s2m were prepared in corresponding buffers containing 0.5% agarose. FUS NTD, FUS FL, Nucleocapsid:s2m and PTBP1:3xUCUCU samples were imaged on a Nikon Spinning Disk SoRa microscope using a 100 × 1.45 CFI Plan Apo Oil objective, while Nucleocapsid:A6 image was acquired using a Sony α6400 APS-C DSLM camera body mounted to an Olympus CKX41 inverted microscope with a 40× objective. FRAP experiments were performed using a standard procedure76. Ensuing recovery rates are reported as a multiple droplets average (>4) with standard deviation. Purified RNA 3 × UCUCU (10 pmol) was first 5'-end dephosphorylated by 5 units of Antarctic phosphatase (NEB, 5 U/µL) in Antarctic phosphatase buffer (NEB, 50 mM Bis-Tris-Propane-HCl, 1 mM MgCl2, 0.1 mM ZnCl2) in a total reaction volume of 10 µL. The reaction was incubated at 37 °C for 30 min, and the enzyme was heat-inactivated at 80 °C for 2 min Dephosphorylated RNA (1.8 pmol) was then 5′-end labeled with 2 pmol of ATP [γ32P] (10 uCi/uL, 5000 Ci/mmol, Hartmann Analytic GmbH) using 10 units of T4 polynucleotide kinase (NEB, 10 U/µL) in T4 polynucleotide kinase buffer (NEB, 70 mM Tris-HCl, 10 mM MgCl2, 5 mM DTT) in a total reaction volume of 10 µL. The reaction was incubated at 37 °C for 1 h, and then the enzyme was heat-inactivated at 95 °C for 5 min. The labeled RNA was purified using an Illustra MicroSpin G-25 column (GE Healthcare) and diluted to 7.2 fmol/µL. Different protein-RNA samples containing 7.2 fmol 3 × UCUCU and an increasing amount of PTBP1 full-length (final concentration from 0 to 1.5 µM) were prepared in D buffer (20 mM HEPES, pH 8.0, 100 mM KCl, 1.5 mM MgCl2, 10% glycerol, 0.5 mM DTT). The RNA-protein complexes were run on a native 10% TBE gel in TBE running buffer (40 mM Tris-HCl, pH 8.3, 45 mM boric acid, 1 mM EDTA) at 100 V for 1 h. The gel was soaked in fixing solution 1 (10% acetic acid, 20% methanol, 5% glycerol) for 15 min and in fixing solution 2 (20% methanol, 5% glycerol) for another 15 min. The gel was subsequently dried using a drying device (Hoefer Slab Gel Dryer GD2000) at 80 °C under vacuum for 1 h. The gel was screened overnight in a phosphor cassette and imaged using Typhoon FLA 9000 (GE Healthcare). All NMR experiments were recorded with a 3-mm NMR sample tube at 298 K (unless otherwise specified) on 600 MHz and 700 MHz Avance NEO spectrometers equipped with TCI cryo-probes and on a 500 MHz Avance NEO with QCI cryo-probe. In total 700 MHz cryoprobe was equipped with xyz-gradient system, while the 500 and 600 MHz probes had only z-axis gradients. All experiments were run and processed using TopSpin 4 software (Bruker). LLPS REDIFINE data sets were measured using a conventional 2D Stimulated-Echo experiment using bipolar gradients (stebpgp1s19) to which a pseudo 3rd dimension was added to vary the diffusion encoding time (Supplementary Note 3). Watergate3919 was used as a preferred water suppression scheme. Alternatively, we also tested the LED experiment using bipolar gradients (ledbpgppr2s) and water suppression using presaturation, and, besides lower signal-to-noise (SNR), the same LLPS REDIFINE results were obtained. LLPS REDIFINE of the water molecule in the FUS condensate (Fig. 2) was acquired without water presaturation using ledbpgp2s pulse sequence. All diffusion experiments were run in 16 increments of gradient strength (g), ranging from 2 to 95%. On Bruker spectrometers, this is defined in percentage of the maximum gradient amplitude of the system, and it is specific for each NMR probe. Typically, most probes can deliver a maximum output of around 53.5 G/cm at 100% gradient amplitude. LLPS REDIFINE needs to contain at least 16 gradient amplitude points defining the diffusion decay for a good fitting. 16−32 scans of signal averaging to acquire sufficient SNR per every gradient strength were required on our NMR systems, and condensate samples were prepared at 100−150 µM protein concentration (except FUS NTD at 600 µM). Depending on the protein/RNA size, diffusion gradient length (δ) can range from 2 to 10 ms. In this study, we used δ of 9 or 10 ms for the protein detection, 3.5 ms for the A6 RNA, and 2 ms for water, and always smoothed rectangular-shaped gradients SMSQ10.100. All experiments were run using an AU program “dosy” available in the TopSpin that also calculates the gradient ramp and feeds it to the spectrometer. It is described that the LLPS REDIFINE data set contains multiple diffusion curves acquired for different diffusion encoding times. To provide the most accurate multiparametric fitting, a minimum of six diffusion curves have to be acquired spanning as wide as possible diffusion encoding time. The maximum diffusion time that can be applied is constrained by the longitudinal T1 relaxation of the target molecule, leading to the signal decay, and for RBDs it is around 700 ms, while for unstructured proteins it can be up to 1000 ms. In case of slow chemical exchange, long diffusion times are absolutely crucial to resolve the fitting parameters. As chemical exchange is not a priori known, acquisition of sufficient diffusion curves spanning the region between the short and the maximum diffusion time is acquired for precise parameter determination by LLPS REDIFINE. In this work, we acquired from 6 to 11 curves in total that were used for fitting. Also, samples containing similar fractions of protein in the condensed phase and dilute phase can be characterized well with only 6 curves, while more curves are required otherwise. Further aspects of the REDIFINE methodology, including prospects and limitations, are discussed in Note 8 of the Supplementary Information. Another parameter worth mentioning is the d1 recovery delay that we used between 1.5 and 5 s. To acquire the full LLPS REDIFINE data set, the course depends on all aforementioned parameters, but in this study, it usually took 1−2 h. The presented differential equations defining the model behind LLPS REDIFINE are incorporated within a home-written MATLAB code. Differential equations are first solved and their analytical solution is used for all the simulations (Supplementary Note 2) and all the fitting. The experimental data were first processed in TopSpin (Fourier transformation, apodization, baseline correction) and then fed to the MATLAB code, which used the least squares regression to minimize the difference between the experimental data and the analytical model function. Five independent parameters were fit: Ddil, Dcond, νcond, average Rdrop, and p, while kcd is derived as explained in Eq. Because of using analytical solutions for the differential equations presented in Eq. (7), the multiparametric fitting procedure was very fast and lasted only a couple of seconds on a regular laptop. The uncertainties of fitted values were calculated using the covariance matrix of parameters. The code containing the model functions and fitting procedures is provided as Source data. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Unless otherwise stated, all data supporting the results of this study can be found in the article, supplementary, and source data files. All raw NMR data presented in this work have been deposited to Zenodo repository https://doi.org/10.5281/zenodo.15228787. Source data are provided with this paper. The Matlab codes used for simulations and fitting have been deposited in the same Zenodo repository https://doi.org/10.5281/zenodo.15228787. & Rosen, M. K. Biomolecular condensates: organizers of cellular biochemistry. Alberti, S. Phase separation in biology. Mehta, S. & Zhang, J. Liquid–liquid phase separation drives cellular function and dysfunction in cancer. Mitrea, D. M. & Kriwacki, R. W. Phase separation in biology; functional organization of a higher order. Active liquid-like behavior of nucleoli determines their size and shape in Xenopus laevis oocytes. Solution nuclear magnetic resonance structure of membrane-integral diacylglycerol kinase. & Brangwynne, C. P. Beyond stereospecificity: liquids and mesoscale organization of cytoplasm. Langdon, E. M. & Gladfelter, A. S. A new lens for RNA localization: liquid-liquid phase separation. Nandana, V. & Schrader, J. M. Roles of liquid–liquid phase separation in bacterial RNA metabolism. & Mittag, T. How do intrinsically disordered protein regions encode a driving force for liquid–liquid phase separation? Perdikari, T. M. et al. SARS‐CoV‐2 nucleocapsid protein phase‐separates with RNA and with human hnRNPs. Savastano, A., Ibáñez de Opakua, A., Rankovic, M. & Zweckstetter, M. Nucleocapsid protein of SARS-CoV-2 phase separates into RNA-rich polymerase-containing condensates. Iserman, C. et al. Genomic RNA elements drive phase separation of the SARS-CoV-2 nucleocapsid. Jack, A. et al. SARS-CoV-2 nucleocapsid protein forms condensates with viral genomic RNA. & Earnshaw, W. C. Structure and function in the nucleus. The polypyrimidine tract-binding protein affects coronavirus rna accumulation levels and relocalizes viral RNAs to novel cytoplasmic domains different from replication-transcription sites. Murthy, A. C. et al. Molecular interactions underlying liquid−liquid phase separation of the FUS low-complexity domain. Emmanouilidis, L. et al. NMR and EPR reveal a compaction of the RNA-binding protein FUS upon droplet formation. Patel, A. et al. A liquid-to-solid phase transition of the ALS protein FUS accelerated by disease mutation. Yuwen, T., Brady, J. P. & Kay, L. E. Probing conformational exchange in weakly interacting, slowly exchanging protein systems via off-resonance R1ρ experiments: application to studies of protein phase separation. Phase transition of a disordered nuage protein generates environmentally responsive membraneless organelles. & Mittag, T. Considerations and challenges in studying liquid-liquid phase separation and biomolecular condensates. Alberti, S. & Dormann, D. Liquid–liquid phase separation in disease. Ganser, L. R. & Myong, S. Methods to study phase-separated condensates and the underlying molecular interactions. Ibrahim, K. A., Naidu, A. S., Miljkovic, H., Radenovic, A. & Yang, W. Label-free techniques for probing biomolecular condensates. Specific protein-RNA interactions are mostly preserved in biomolecular condensates. Uebel, C. & Phillips, C. Phase-separated protein dynamics are affected by fluorescent tag choice. Sánchez‐Rico, C., Voith von Voithenberg, L., Warner, L., Lamb, D. C. & Sattler, M. Effects of fluorophore attachment on protein conformation and dynamics studied by spFRET and NMR spectroscopy. & Boxer, S. G. Choose your label wisely: water-soluble fluorophores often interact with lipid bilayers. Diverse effects of fluorescent labels on alpha-synuclein condensate formation during liquid-liquid phase separation. Pandey, N. K. et al. Fluorescent protein tagging promotes phase separation and alters the aggregation pathway of huntingtin exon-1. Phase separation of α-crystallin-GFP protein and its implication in cataract disease. Dörner, K. et al. Tag with Caution — How protein tagging influences the formation of condensates. Lin, Y., Currie, S. L. & Rosen, M. K. Intrinsically disordered sequences enable modulation of protein phase separation through distributed tyrosine motifs. Restricted self-diffusion of protons in colloidal systems by the pulsed-gradient, spin-echo method. Balinov, B., Jonsson, B., Linse, P. & Soderman, O. The NMR self-diffusion method applied to restricted diffusion. Simulation of echo attenuation from molecules in spheres and between planes. & Henkelman, R. M. An analytical model of restricted diffusion in bovine optic nerve. Measurement of membrane protein lateral diffusion in single cells. From fixed to FRAP: measuring protein mobility and activity in living cells. Burke, K. A., Janke, A. M., Rhine, C. L. & Fawzi, N. L. Residue-by-residue view of in vitro FUS granules that bind the C-terminal domain of RNA polymerase II. Ray, S. et al. α-Synuclein aggregation nucleates through liquid–liquid phase separation. Otting, G. & Wüthrich, K. Studies of protein hydration in aqueous solution by direct NMR observation of individual protein-bound water molecules. Saenger, W. Structure and dynamics of water surrounding biomolecules. Kriwacki, R. W., Hill, R. B., Flanagan, J. M., Caradonna, J. P. & Prestegard, J. H. New NMR methods for the characterization of bound waters in macromolecules. Aumiller, W. M., Pir Cakmak, F., Davis, B. W. & Keating, C. D. RNA-based coacervates as a model for membraneless organelles: formation, properties, and interfacial liposome assembly. Structural and hydrodynamic properties of an intrinsically disordered region of a germ cell-specific protein on phase separation. Åslund, I., Nowacka, A., Nilsson, M. & Topgaard, D. Filter-exchange PGSE NMR determination of cell membrane permeability. Eriksson, S. et al. NMR quantification of diffusional exchange in cell suspensions with relaxation rate differences between intra and extracellular compartments. Constant gradient FEXSY: a time-efficient method for measuring exchange. Padroni, G. et al. A hybrid structure determination approach to investigate the druggability of the nucleocapsid protein of SARS-CoV-2. Wang, Z., Yang, C., Guan, D., Li, J. & Zhang, H. Cellular proteins act as surfactants to control the interfacial behavior and function of biological condensates. van der Veen, J. W. C., van Gelderen, P., Creyghton, J. H. N. & Bovée, W. M. M. J. Diffusion in red blood cell suspensions: separation of the intracellular and extracellular NMR sodium signal. Gil-Garcia, M. et al. Local environment in biomolecular condensates modulates enzymatic activity across length scales. Characterization of food emulsions by PFG NMR. Johns, M. L. & Hollingsworth, K. G. Characterisation of emulsion systems using NMR and MRI. Hollingsworth, K. G., Sederman, A. J., Buckley, C., Gladden, L. F. & Johns, M. L. Fast emulsion droplet sizing using NMR self-diffusion measurements. Wolf, F., Hecht, L., Schuchmann, H. P., Hardy, E. H. & Guthausen, G. Preparation of W1/O/W2 emulsions and droplet size distribution measurements by pulsed‐field gradient nuclear magnetic resonance (PFG‐NMR) technique. Balanda, K. P. & Macgillivray, H. L. Kurtosis: a critical review. Jenner, C. F., Xia, Y., Eccles, C. D. & Callaghan, P. T. Circulation of water within wheat grain revealed by nuclear magnetic resonance micro-imaging. He, C., Wu, C. Y., Li, W. & Xu, K. Multidimensional super-resolution microscopy unveils nanoscale surface aggregates in the aging of FUS condensates. Choi, C.-H., Lee, D. S. W., Sanders, D. W. & Brangwynne, C. P. Condensate interfaces can accelerate protein aggregation. The liquid-to-solid transition of FUS is promoted by the condensate surface. Morelli, C. et al. RNA modulates hnRNPA1A amyloid formation mediated by biomolecular condensates. Emmanouilidis, L. et al. A solid beta-sheet structure is formed at the surface of FUS droplets during aging. & Tanner, J. E. Spin diffusion measurements: spin echoes in the presence of a time-dependent field gradient. Merboldt, K.-D., Hanicke, W. & Frahm, J. Self-diffusion NMR imaging using stimulated echoes. & Dobson, C. M. Characterisation of protein unfolding by NMR diffusion measurements. Le Bihan, D. et al. Diffusion tensor imaging: concepts and applications. Karunanithy, G. et al. INDIANA: an in-cell diffusion method to characterize the size, abundance and permeability of cells. Pfeuffer, J., Flögel, U., Dreher, W. & Leibfritz, D. Restricted diffusion and exchange of intracellular water: theoretical modelling and diffusion time dependence of 1H NMR measurements on perfused glial cells. Price, W. S., Barzykin, A. V., Hayamizu, K. & Tachiya, M. A model for diffusive transport through a spherical interface probed by pulsed-field gradient NMR. Panagiotaki, E. et al. Noninvasive quantification of solid tumor microstructure using VERDICT MRI. Structural modeling of protein-RNA complexes using crosslinking of segmentally isotope-labeled RNA and MS/MS. Jutzi, D. et al. Aberrant interaction of FUS with the U1 snRNA provides a molecular mechanism of FUS-induced amyotrophic lateral sclerosis. Ray, S., Singh, N., Patel, K., Krishnamoorthy, G. & Maji, S. K. FRAP and FRET investigation of α-synuclein fibrillization via liquid-liquid phase separation in vitro and in HeLa cells. The authors acknowledge Prof. Lucio Frydman, Prof. Thomas Michaels and Dr. Fred F. Damberger for useful discussions. The authors also thank Drs. Uwe Schmitt, Tarun Chadha, and Agnieszka Ilnicka from ETH IT Services for constructive advice that helped speed up the fitting procedure in Matlab. This work was supported by Swiss National Science Foundation (SNSF) grants 310030-215555 (F.H.-T.A. ), and NCCR RNA & Disease grant 51NF40-182880 (F.H.-T.A. These authors contributed equally: Yaning Han, Nina C. Kathe, Yinan Ni. Department of Biology, Institute of Biochemistry, ETH Zurich, Zurich, Switzerland Mihajlo Novakovic, Yaning Han, Nina C. Kathe, Yinan Ni, Leonidas Emmanouilidis & Frédéric H.-T. Allain You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed the simulations and experiments, as well as process the data; M.N., L.E. prepared protein and RNA samples. reviewed and edited the manuscript. provided supervision and acquired funding. All authors read and approved the final manuscript. Correspondence to Mihajlo Novakovic, Leonidas Emmanouilidis or Frédéric H.-T. Allain. The authors declare no competing interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Novakovic, M., Han, Y., Kathe, N.C. et al. LLPS REDIFINE allows the biophysical characterization of multicomponent condensates without tags or labels. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            