
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-28</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/bird-watching-tips-as-migration-season-peaks/'>Birds Are Migrating All across the U.S. Here's How to Make the Most of It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 15:20:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Bird Migration Is Peaking in the U.S.—Here's How to Watch The spring migration of birds from their southern wintering grounds to their breeding grounds in the North is about to reach a fever pitch. Over the next few weeks, most of the U.S. is heading into peak migration, when hundreds of millions of birds can wing their way across the sky in a single night on a mission to claim a prime bit of real estate, attract a suitable mate and get to work raising the next generation. Many of these migratory birds travel vast distances between their winter and summer homes, taking advantage of seasonal abundances in food and daylight. The Red Knot, a stout shorebird with an orange-pink breast, may journey nearly twice that distance, sometimes covering more than 600 miles in a single day. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The birds are now looking extra snazzy—bright and glossy, dressed to impress in their breeding plumage. Colorful, shiny feathers can signal good health to potential mates. Get outside early if you can; birds tend to be most active in the morning. For a closer look at their beautiful colors, patterns and other details, use binoculars. A field guide can help you figure out which species you're observing— The Sibley Guide to Birds is especially good and is available as an app. Another guide in app form is Merlin Bird ID from the Cornell Lab of Ornithology. It has a nifty feature that can help you identify birds by their songs and calls. Spring is not only a perfect time for veteran bird-watchers to spy an elusive species. The availability of amazing technology means that even a novice can easily identify and learn about the birds they observe. People often wonder about the link between dinosaurs and birds. Here you can learn about how, over millions of years, evolution forged small, flight-capable birds with feathers and wings from nonavian dinosaurs. Kate Wong is an award-winning science writer and senior editor at Scientific American focused on evolution, ecology, anthropology, archaeology, paleontology and animal behavior. Her reporting has taken her to caves in France and Croatia that Neandertals once called home, to the shores of Kenya's Lake Turkana in search of the oldest stone tools in the world, to Madagascar on an expedition to unearth ancient mammals and dinosaurs, to the icy waters of Antarctica, where humpback whales feast on krill, and on a "Big Day" race around the state of Connecticut to find as many bird species as possible in 24 hours. Kate is co-author, with Donald Johanson, of Lucy's Legacy: The Quest for Human Origins. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64579357/100-roman-horses/'>A Construction Crew Accidentally Uncovered the Remains of 100+ Horses</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Construction crews discovered the cemetery during preparations for a housing project. Evidence shows that the carcasses were slid into shallow pits and buried lying on their side with outstretched or bent legs. And even with the dense burials, there were few overlaps of graves, exposing that the site was likely clearly defined with above-ground markers. Even if a horse could walk on its own, if it was deemed beyond military usefulness, it would have been taken to the cemetery and killed there. “The horses do not seem to have all died at the same time in a major event such as a battle or epidemic,” Roth said. “Rather, here are the animals that either died during the presence of the ala in Bad Cannstatt due to illness, injury, or other reasons, or who no longer did their task as a military horse.” “Here we see a particularly close connection of the owner to his horse,” Roth said. Experts plan to study the sex, age at death, and size of the horses while looking for possible diseases and causes of death. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A Building Crew Found an Iron Age Chariot Wheel Experts Found an Ancient Tomb with a Fake Door Messages Found on the Walls of Last Supper Site</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a64581842/hailstone-formation/'>Scientists Found Fingerprints in Hailstones That May Overturn the Simple Truths of Weather</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Every year, hailstorms cause billions in damage to homes, businesses, and crops, yet how these ice balls of destruction form in the upper troposphere has remained somewhat of a mystery. The predominant theory was that hailstones went through a recycling motion, which explained why they often contained varying layers of a clear and opaque ice. Now, a new study from the Chinese Academy of Sciences suggests that this recycling motion is only a small part of the story. “This work fundamentally changes how we understand hail formation,” Zhang said in a press statement. “By moving beyond assumptions to actual chemical evidence, we're building a more accurate picture of these destructive weather phenomena.” Some 10 stones showed that they formed while steadily descending toward the atmosphere, and another 13 displayed evidence of only one single upward push. According to the researchers, hail typically forms in a temperature “sweet spot” of between roughly -30 and -10 degrees Celsius. Although this largely refutes the previous theory behind hailstone formation, larger hailstones do require extended time (at least one upward growth) in this zone for supercooled water to form more layers. This explains why stronger storms—with more powerful updrafts—often produce larger hailstones. Zhang hopes that understanding the inner workings of hailstorm formation will help improve weather forecasting and our ability to estimate the potential dangers of a passing hailstorm (this research is in collaboration with the U.S. National Center for Atmospheric Research). Meteorologists at NWS have already reported degraded forecasting services due to curtailed weather balloon launches used to help gather data for accurate storm prediction. This Ancient Larva Still Has Its Brain And Guts This “Battery In a Rock” Changes the Energy Game This Dark Ocean Pit Has a Bleak Weather Forecast A DNA Mutation Helps Some Fish Survive Deep Waters</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/this-bitcoin-encryption-comes-from-renaissance-art-and-math/'>This Cutting-Edge Encryption Originates in Renaissance Art and Math</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This Cutting-Edge Encryption Originates in Renaissance Art and Math An architectural view by Francesco di Giorgio Martini showcases Renaissance perspective drawing. He planted his instrument, which involved an easel, a mirror and a wire framework, near the then unfinished cathedral of Florence in Italy—a cathedral whose monumental dome he would soon design. This demonstration of his recently discovered laws of perspective is said to have occurred sometime between 1415 and 1420, if his biographers are correct. This is the security scheme that underpins Bitcoin and other cryptocurrencies and has become a fast-growing encryption method on other Internet platforms as well. But how did Renaissance art lead to the mathematics that govern modern cryptography? Its characters include a French prisoner of war and two mathematicians struck down in their prime—one by illness and the other by a duelist's pistol. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. His findings, however, were couched in rather obscure language and struggled to find an audience. His key contributions were included in a book that had a print run of 50 copies, small even for that era, and many of those copies were eventually bought back by the publisher and destroyed. During Desargues's lifetime, only fellow French mathematician Blaise Pascal became an ardent disciple of his work. Pascal contributed his own theorem to the study of what became known as “projective geometry.” Despite Desargues's obscurity, he made a revolutionary breakthrough by adding the concept of points and lines at infinity to Euclidean geometry. By including those points, projective geometry could be merged with Euclidean geometry in a way that was consistent for both systems. Furthermore, parabolas and hyperbolas are equivalent to ellipses, with the addition of one or two points at infinity, respectively. These insights, though valuable, would languish in obscurity for more than 100 years. Rather, a different French mathematician, Gaspard Monge, began to work on the same questions and derived similar results. The most comprehensive work on projective geometry in this era, however, came in the 19th century from French engineer and mathematician Jean-Victor Poncelet, under rather trying circumstances. He then joined France's corps of military engineers as a lieutenant and was ordered to what is now Belarus to support Napoleon's invasion of Russia in 1812. Poncelet remained with the French army right up to the Battle of Krasnoye in Russia, where he was separated from his unit and possibly left for dead. After the battle, he was scooped up by the Russian army and marched to Saratov, Russia, more than 700 miles from Krasnoye and more than 2,000 miles from his home in Metz, France. Although Poncelet was not confined to a prison, he was “deprived of books and comforts of all sorts,” according to an English translation of his introduction to his first book on projective geometry. He could not carry out this plan, however, saying that he was “distressed above all by the misfortune of my country and my own lot.” In hindsight, it is perhaps not surprising that a prisoner of war thousands of miles from home and unsure of when, or even if, he would be repatriated would focus his efforts on understanding points at infinity—a distance that might have seemed quite intelligible to someone in Poncelet's situation. After the war that had been sparked by that invasion ended, Poncelet returned to France and his two-volume work on projective geometry, published in 1822, was far more well-received and widely read than Desargues's work. At around the same time that Poncelet was finishing his book on projective geometry, Norwegian mathematician Niels Henrik Abel was studying elliptic integrals. These integrals are rather difficult expressions that started off as parts of an attempt to measure the circumference of an ellipse. The curves, it turned out, are much easier to work with. Further research into elliptic curves would be left to others, however; Abel died from tuberculosis at age 26 in 1829, mere months after publishing an important paper on the subject. Galois would die tragically but also stubbornly in a duel at age 20, but before his death he laid out the principles of group theory, in which mathematical objects and operations that follow certain rules constitute a group. The system he developed, which uses what are called homogeneous coordinates, play a pivotal role in elliptic curve cryptography. Several decades later, in 1901, another French mathematician, Henri Poincaré, realized that points with rational coordinates—that is, points with coordinates that can be represented as fractions on the graph of an elliptic curve—composed a group. This process onlyworked if you used the homogeneous coordinates discovered by Möbius that include a point at infinity, however. Importantly, elliptic curve groups turned out to be Abelian, which meant that the order in which those addition operations were performed didn't matter. This is where matters stood until the mid-1980s, when Victor S. Miller, then a researcher at IBM, and Neal Koblitz of the University of Washington independently realized that you could build a public-private key cryptographic system based off elliptic curve groups. Importantly, both keys are required to decrypt the message being sent. Because elliptic curve groups are commutative, meaning that it doesn't matter in what order addition is carried out, both parties will arrive at a number corresponding to the same final point on the curve, and this is the number that will be used to encrypt and decrypt the data. Elliptic curve cryptography is a relative latecomer to the encryption game. Its status as the de facto standard for cryptocurrencies made people more familiar with it and more comfortable implementing it, although it still lags behind RSA encryption, the standard method in use today, by a wide margin. Yet elliptic curve cryptography has distinct advantages over RSA cryptography: it provides stronger security per bit and is faster than RSA. An elliptic curve cryptographic key of just 256 bits is roughly as secure as a 3,072-bit RSA key and considerably more secure than the 2,048-bit keys that are commonly used. Richard Jensen is a freelance writer from Sioux Falls, S.D.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02675-x'>Repeated behavioural evolution is associated with convergence of gene expression in cavity-nesting songbirds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 10:15:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Ecology & Evolution (2025)Cite this article Uncovering the genomic bases of phenotypic adaptation is a major goal in biology, but this has been hard to achieve for complex behavioural traits. Here we leverage the repeated, independent evolution of obligate cavity nesting in birds to test the hypothesis that pressure to compete for a limited breeding resource has facilitated convergent evolution in behaviour, hormones and gene expression. We used an integrative approach, combining aggression assays in the field, testosterone measures and transcriptome-wide analyses of the brain in wild-captured females and males. Our experimental design compared species pairs across five avian families, each including one obligate cavity-nesting species and a related species with a more flexible nest strategy. We find behavioural convergence, with higher levels of territorial aggression in obligate cavity nesters, particularly among females. Across species, levels of testosterone in circulation were not associated with nest strategy nor aggression. Phylogenetic analyses of individual genes and co-regulated gene networks revealed more shared patterns of brain gene expression than expected by drift, although the scope of convergent gene expression evolution was limited to a small percentage of the genome. When comparing our results to other studies that did not use phylogenetic methods, we suggest that accounting for shared evolutionary history may reduce the number of genes inferred as convergently evolving. Altogether, we find that behavioural convergence in response to shared ecological pressures is associated with largely independent evolution of gene expression across different avian families, punctuated by a narrow set of convergently evolving genes. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available in the Supplementary Information, on Github via Zenodo at https://doi.org/10.5281/zenodo.14715061 (ref. 86), and via Figshare repository at https://doi.org/10.6084/m9.figshare.26367061 (ref. Raw sequence reads and count data can be obtained from the Gene Expression Omnibus database (GEO accession number GSE244480). Scripts for processing and analysing data are available on Github via Zenodo at https://doi.org/10.5281/zenodo.14715061 (ref. Darwin, C. R. On the Origin of Species (Harvard Univ. Rosenblum, E. B., Parent, C. E. & Brandt, E. E. The molecular basis of phenotypic convergence. Google Scholar Arendt, J. & Reznick, D. Convergence and parallelism reconsidered: what have we learned about the genetics of adaptation? Trends Ecol. Google Scholar Gallant, J. R. & O'Connell, L. A. Studying convergent evolution to relate genotype to behavioral phenotype. Google Scholar Jourjine, N. & Hoekstra, H. E. Expanding evolutionary neuroscience: insights from comparing variation in behavior. Google Scholar Evolutionary endocrinology and the problem of Darwin's tangled bank. Google Scholar Rittschof, C. C. et al. Neuromolecular responses to social challenge: common mechanisms across mouse, stickleback fish, and honey bee. Google Scholar Toth, A. L. et al. Brain transcriptomic analysis in paper wasps identifies genes associated with behaviour across social insect lineages. Google Scholar Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Google Scholar Mikheyev, A. S. & Linksvayer, T. A. Genes associated with ant social behavior show distinct transcriptional and evolutionary patterns. Google Scholar Warner, M. R., Qiu, L., Holmes, M. J., Mikheyev, A. S. & Linksvayer, T. A. Convergent eusocial evolution is based on a shared reproductive groundplan plus lineage-specific plastic genes. Google Scholar Boyle, E. A., Li, Y. I. & Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Google Scholar Young, R. L. et al. Conserved transcriptomic profiles underpin monogamy across vertebrates. Google Scholar Pease, J. B. et al. Layered evolution of gene expression in ‘superfast' muscles for courtship. Google Scholar Harrison, M. C. et al. Hemimetabolous genomes reveal molecular basis of termite eusociality. Google Scholar Johnson, Z. V. et al. Cellular profiling of a recently-evolved social behavior in cichlid fishes. Google Scholar Nelson, R. J. & Trainor, B. C. Neural mechanisms of aggression. Google Scholar Wingfield, J. C., Hegner, R. E., Dufty, J., Alfred, M. & Ball, G. F. The ‘Challenge Hypothesis': theoretical implications for patterns of testosterone secretion, mating systems, and breeding strategies. Google Scholar Horton, B. M. et al. Estrogen receptor alpha polymorphism in a species with alternative behavioral phenotypes. Google Scholar Rosvall, K. A. et al. Neural sensitivity to sex steroids predicts individual differences in aggression: implications for behavioural evolution. Google Scholar Saul, M. C. et al. Cross-species systems analysis of evolutionary toolkits of neurogenomic response to social challenge. Genes Brain Behav. Zhang-James, Y. et al. An integrated analysis of genes and functional pathways for aggression in human and rodent models. Kelly, A. M. & Wilson, L. C. Aggression: perspectives from social and systems neuroscience. O'Connell, L. A. & Hofmann, H. A. Evolution of a vertebrate social decision-making network. Rosvall, K. A. Proximate perspectives on the evolution of female aggression: good for the gander, good for the goose? Clutton-Brock, T. Sexual selection in males and females. Rosvall, K. A. Intrasexual competition in females: evidence for sexual selection? Wright, A. E. & Mank, J. E. The scope and strength of sex-specific selection in genome evolution. Goymann, W. & Wingfield, J. C. Male-to-female testosterone ratios, dimorphism, and life history—what does it really tell us? Rosvall, K. A., Bentz, A. & George, E. M. How research on female vertebrates contributes to an expanded challenge hypothesis. Smiley, K. O. et al. Beyond a biased binary: a perspective on the misconceptions, challenges, and implications of studying females in avian behavioral endocrinology. Duckworth, R. A. Adaptive dispersal strategies and the dynamics of a range expansion. Leffelaar, D. & Robertson, R. J. Nest usurpation and female competition for breeding opportunities by tree swallows. Wilson Bull. Merilä, J. & Wiggins, D. A. Interspecific competition for nest holes causes adult mortality in the collared flycatcher. Albers, A. N., Jones, J. & Siefferman, L. Behavioral differences among eastern bluebird populations could be a consequence of tree swallow presence: a pilot study. & Getty, T. Fitness benefits to intrasexual aggression in female house wrens, Troglodytes aedon. Sexual selection on aggressiveness in females: evidence from an experimental test with tree swallows. Rubenstein, D. R. & Lovette, I. J. Reproductive skew and selection on female ornamentation in social species. Schuppe, E. R., Sanin, G. D. & Fuxjager, M. J. The social context of a territorial dispute differentially influences the way individuals in breeding pairs coordinate their aggressive tactics. Lipshutz, S. E. & Rosvall, K. A. Nesting strategy shapes territorial aggression but not testosterone: a comparative approach in female and male birds. Fang, Y. T., Tuanmu, M. N. & Hung, C. M. Asynchronous evolution of interdependent nest characters across the avian phylogeny. Zenil-Ferguson, R., McEntee, J. P., Burleigh, J. G. & Duckworth, R. A. Linking ecological specialization to its macroevolutionary consequences: an example with passerine nest type. Jawor, J. M. et al. Seasonal and individual variation in response to GnRH challenge in male dark-eyed juncos (Junco hyemalis). Rosvall, K. A., Bergeon Burns, C. M., Jayaratna, S. P. & Ketterson, E. D. Divergence along the gonadal steroidogenic pathway: implications for hormone-mediated phenotypic evolution. Bukhari, S. A. et al. Temporal dynamics of neurogenomic plasticity in response to social interactions in male threespined sticklebacks. Goodson, J. L. The vertebrate social behavior network: evolutionary themes and variations. Stelzer, G. et al.The GeneCards Suite: from gene data mining to disease genome sequence analyses. Ghanem, N. et al. The Rb/E2F pathway modulates neurogenesis through direct regulation of the Dlx1/Dlx2 bigene cluster. Chandra, B., Voas, M. G., Davies, E. L. & Roberts-Galbraith, R. H. Ets-1 transcription factor regulates glial cell regeneration and function in planarians. Hung, H. H. et al. Selective involvement of UGGT variant: UGGT2 in protecting mouse embryonic fibroblasts from saturated lipid-induced ER stress. Yan, C. et al. Correction: IRE1 promotes neurodegeneration through autophagy-dependent neuron death in the Drosophila model of Parkinson's disease. Cell Death Dis. Zhu, J. Y., Chen, M., Mu, W. J., Luo, H. Y. & Guo, L. The functional role of Higd1a in mitochondrial homeostasis and in multiple disease processes. Genes Dis. Crombie, E. M., Cleverley, K., Timmers, H. T. M. & Fisher, E. M. C. The roles of TAF1 in neuroscience and beyond. Open Sci. Baran, N. M., McGrath, P. T. & Streelman, J. T. Applying gene regulatory network logic to the evolution of social behavior. Experimental competition induces immediate and lasting effects on the neurogenome in free-living female birds. Filby, A. L., Paull, G. C., Hickmore, T. F. & Tyler, C. R. Unravelling the neurophysiological basis of aggression in a fish model. Storz, J. F. Causes of molecular convergence and parallelism in protein evolution. West-Eberhard, M. J. Sexual selection, social competition, and speciation. Harts, A. M., Booksmythe, I. & Jennions, M. D. Mate guarding and frequent copulation in birds: a meta-analysis of their relationship to paternity and male phenotype. Drury, J. P., Cowen, M. C. & Grether, G. F. Competition and hybridization drive interspecific territoriality in birds. Husak, J. F. et al. Life history and environment predict variation in testosterone across vertebrates. Goymann, W., Moore, I. T. & Oliveira, R. F. Challenge hypothesis 2.0: a fresh look at an established idea. Wingfield, J. C., Ramenofsky, M., Hegner, R. E. & Ball, G. F. Whither the challenge hypothesis? Lischinsky, J. E. & Lin, D. Neural mechanisms of aggression across species. Lipshutz, S. E., George, E. M., Bentz, A. Evaluating testosterone as a phenotypic integrator: from tissues to individuals to species. Fischer, E. K., Song, Y., Hughes, K. A., Zhou, W. & Hoke, K. L. Nonparallel transcriptional divergence during parallel adaptation. Renn, S. C. P. et al. Gene expression signatures of mating system evolution. Alaux, C. et al. Honey bee aggression supports a link between gene regulation and behavioral evolution. Mullon, C., Wright, A. E., Reuter, M., Pomiankowski, A. & Mank, J. E. Evolution of dosage compensation under sexual selection differs between X and Z chromosomes. Grath, S. & Parsch, J. Sex-biased gene expression. Fridolfsson, A.-K. & Ellegren, H. A simple and universal method for molecular sexing of non-ratite birds. Soma, K. K., Schlinger, B. A., Wingfield, J. C. & Saldanha, C. J. Brain aromatase, 5 alpha-reductase, and 5 beta-reductase change seasonally in wild male song sparrows: relationship to aggressive and sexual behavior. Grabherr, M. G. et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Rhie, A. et al. Towards complete and error-free genome assemblies of all vertebrate species. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Plaisier, S. B., Taschereau, R., Wong, J. & Graeber, T. G. Rank–rank hypergeometric overlap: identification of statistically significant overlap between gene-expression signatures. Nucleic Acids Res. Langfelder, P. & Horvath, S. WGCNA: an R package for weighted correlation network analysis. Mi, H., Muruganujan, A., Ebert, D., Huang, X. & Thomas, P. D. PANTHER version 14: more genomes, a new PANTHER GO-slim and improvements in enrichment analysis tools. Nucleic Acids Res. Article PubMed Central Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13, 2498–2504 (2003). Hadfield, J. D. MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Jetz, W., Thomas, G. H., Joy, J. B., Hartmann, K. & Mooers, A. O. The global diversity of birds in space and time. Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019). Revell, L. J. phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol. Seabold, S. & Perktold, J. Statsmodels: Econometric and Statistical Modeling with Python. In Proc. 9th Python in Science Conference (eds van der Walt, S. & Millman, J.) Kolberg, L. et al. g:Profiler—interoperable web service for functional enrichment analysis and gene identifier mapping (2023 update). Nucleic Acids Res. Lipshutz, S. & Hibbins, M. slipshut/CavityNesting: cavity nesting github release (v1.1.1). Lipshutz, S. et al. Repeated behavioral evolution is associated with convergence of gene expression in cavity-nesting songbirds. Figshare https://doi.org/10.6084/m9.figshare.26367061 (2024). Kumar, S., Stecher, G., Suleski, M. & Hedges, S. B. TimeTree: a resource for timelines, timetrees, and divergence times. Download references For facilitating fieldwork, we thank the Indiana Department of Natural Resources; Indiana University's Research and Teaching Preserve; T. Rothfus and C. Hagie at Therkilsden Field station; A. Fournier and S. Havera at Forbes Field Station; and J. Hoover, N. Antonson, H. Scharf and A. Funk. For vocal stimuli, we thank M. Wilkins, D. Reichard, C. Krieg and E. Chen. Thanks to E. Mueller for help with 3D printing; S. Nallu and J. Huang for TapeStation and library preparation; E. D. Curole and J. K. for assistance in the lab; and B. Young for assistance with data analysis. This work was funded by the US National Science Foundation grant DBI-1907134 (S.E.L. and CAREER grant IOS-1942192 (K.A.R. ), as well as support from Indiana University Biology, Loyola Biology and Duke Biology. Department of Biology, Indiana University, Bloomington, IN, USA Sara E. Lipshutz, Mark S. Hibbins, Alexandra B. Bentz, Tara A. Empson, Elizabeth M. George, Samuel J. Torneo, Abbigail M. Turner, Sarah E. Wolf, Mary J. Woodruff, Matthew W. Hahn & Kimberly A. Rosvall Department of Biology, Loyola University Chicago, Chicago, IL, USA Sara E. Lipshutz & Quinn K. Thomas Department of Biology, Duke University, Durham, NC, USA Sara E. Lipshutz Department of Ecology and Evolutionary Biology, University of Toronto, Toronto, Ontario, Canada Mark S. Hibbins School of Biological Sciences, University of Oklahoma, Norman, OK, USA Alexandra B. Bentz Center for Genomics and Bioinformatics, Indiana University, Bloomington, IN, USA Aaron M. Buechlein & Douglas B. Rusch Department of Ecology and Evolutionary Biology, University of Tennessee, Knoxville, TN, USA Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH, USA Elizabeth M. George Department of Evolution, Ecology, and Behavior, University of Illinois Urbana-Champaign, Urbana, IL, USA Mark E. Hauber, Wendy M. Schelsky & Abbigail M. Turner Illinois Natural History Survey, Prairie Research Institute, University of Illinois Urbana-Champaign, Champaign, IL, USA Mark E. Hauber & Wendy M. Schelsky Advanced Science Research Center and Programs in Biology and in Psychology, Graduate Center of the City University of New York, New York, NY, USA The Grainger College of Engineering, University of Illinois Urbana-Champaign, Urbana, IL, USA School of Biological Sciences, University of Edinburgh, Edinburgh, Scotland, UK Sarah E. Wolf United States Department of Agriculture, Animal and Plant Health Inspection Services, Veterinary Services, Center for Epidemiology and Animal Health, Fort Collins, CO, USA Department of Computer Science, Indiana University, Bloomington, IN, USA Matthew W. Hahn You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived of the study; S.E.L. led the data collection, supported by A.B.B., A.M.T., E.M.G, M.E.H., M.J.W., S.J.T, S.E.W., T.A.E. analysed and interpreted the data; S.E.L., M.S.H. wrote the paper, and all authors contributed to revisions. The manuscript reflects the contributions and ideas of all authors. Correspondence to Sara E. Lipshutz or Kimberly A. Rosvall. The authors declare no competing interests. Nature Ecology & Evolution thanks Natasha Bloch, Chih-Ming Hung and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Sections 1–12, Tables 1–13 and Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Lipshutz, S.E., Hibbins, M.S., Bentz, A.B. Repeated behavioural evolution is associated with convergence of gene expression in cavity-nesting songbirds. Nat Ecol Evol  (2025). Download citation Accepted: 04 March 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Ecology & Evolution (Nat Ecol Evol) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-58282-8'>An anti-virulence drug targeting the evolvability protein Mfd protects against infections with antimicrobial resistant ESKAPE pathogens</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 09:52:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The increasing incidence of antibiotic resistance and the decline in the discovery of novel antibiotics have resulted in a global health crisis, particularly, for the treatment of infections caused by Gram-negative bacteria, for which therapeutic dead-ends are alarming. Here, we identify and characterize a molecule, NM102, that displays antimicrobial activity exclusively in the context of infection. NM102 inhibits the activity of the non-essential Mutation Frequency Decline (Mfd) protein by competing with ATP binding to its active site. Inhibition of Mfd by NM102 sensitizes pathogenic bacteria to the host immune response and blocks infections caused by the clinically-relevant bacteria Klebsiella pneumoniae and Pseudomonas aeruginosa, without inducing host toxicity. Finally, NM102 inhibits the mutation and evolvability function of Mfd, thus reducing the bacterial capacity to develop antimicrobial resistance. These data provide a potential roadmap for the development of drugs to combat antimicrobial resistance. The fight against pathogenic bacteria is becoming one of the greatest challenges for our societies, particularly with the spread of multi- and extensively-drug resistant bacteria, which are seriously jeopardizing the use of traditional antibiotics1. Estimates suggest that at least 700,000 people die annually from drug-resistant infections, and without intervention, this number could rise to 10 million by 2050, becoming the major cause of death worldwide2. In particular, infections caused by the ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter spp.) are increasingly associated with therapeutic dead-ends. As such, they are at the top of the World Health Organization's priority list of bacteria for which novel therapeutics are urgently needed. The identification and characterization of antibiotics with new molecular scaffolds, innovative targets and/or novel mechanisms of action, associated with a low potential for resistance induction, are urgently needed and particularly challenging for Gram-negative pathogens. Indeed, since 1990, only six new classes of antibiotics have been approved3. Recent efforts have begun to revitalize antibiotic research, with compounds being explored either through drug repurposing or through molecular mechanisms that are often redundant with traditional antibiotics. This is the case for finafloxacin, a fluoroquinolone antibiotic recently approved to treat ear infections caused by P. aeruginosa4,5, darobactin, which specifically targets Gram-negative bacteria6, teixobactin, which is only effective against Gram-positive bacteria7, and the newly developed irresistin-16, which exhibits bactericidal activity against both Gram-negative and Gram-positive bacteria without inducing resistance, at least thus far8. These findings pave the way for the discovery of new molecules, a crucial step in providing effective antibiotics that act on novel targets, thereby avoiding cross-resistances. In this context, this study focuses on a new bacterial target, the Mutation Frequency Decline (Mfd) protein, and its inhibition by innovative small molecules. Mfd recognizes RNA polymerase (RNAP) stalled at noncoding lesions and utilizes ATP to translocate along DNA, likely forcing RNAP forward and ultimately dissociating it from the bulky DNA template10. The structure of Mfd has been solved, revealing the presence of an ATPase motor module, which is highly conserved among bacteria and is functionally necessary12,15. Stress-induced mutagenesis can help pathogens generate drug-resistant cells during antibiotic therapy. Mfd has been associated with the development of antibiotic resistance in Campylobacter jejuni16,17, and Helicobacter pylori Mfd-deficient cells are sensitized to antibiotics16,17. Furthermore, Mfd has been suggested to be an evolvability factor that promotes hypermutation in bacteria, thereby accelerating the evolution of antimicrobial resistance (AMR)18. In line with that, a recent study identified, through an in vivo screen, a molecule called ARM-1 for Antimicrobial Resistance Molecule that targets Mfd to reduce mutagenesis and significantly delays antibiotic resistance development across highly divergent bacterial pathogens19. Taken together, these data emphasize that blocking Mfd could hinder molecular evolution, thereby inhibiting the development of resistance in many bacterial pathogens. Furthermore, Mfd is critical for virulence in the Gram-positive Bacillus cereus and in the Gram-negative Shigella flexneri species and it confers resistance to nitric oxide (NO) stress20,21. The production of reactive nitrogen species is a key step in the immune response following infection22. Mfd is involved in DNA repair following DNA damage and is thus required for bacterial resistance to the host response. As Mfd is widely conserved in the bacterial kingdom, these data highlight a mechanism—possibly ubiquitous in bacteria—that overcomes the host immune response and tackles its mutagenic properties. The function of Mfd in the bacterial bypass of host immunity, its role as an evolvability factor during antimicrobial resistance, its ubiquity in the prokaryotic world as well as its absence in higher eukaryotes, all support the selection of this protein as an innovative bacterial target for anti-infective therapies. Molecules that inhibit Mfd activity would neither directly inhibit nor kill bacteria, but instead curb bacterial evolution while impeding the bacteria's ability to resist the host immune response. Thus, Mfd inhibitory compounds should boost the immune system's response against pathogenic bacteria while acting exclusively at the inflammation site, thereby preventing collateral damage. Herein, we report a targeted, structure-based, high-throughput in silico screening, using the druggable ATP binding site of Mfd, to identify molecules capable of acting as competitors to the cognate ligand ATP, thereby inhibiting Mfd activity. One molecule, NM102, was identified as a potent molecule that specifically targets the ATP-binding site of Mfd. In silico molecular analysis and in vivo assays revealed that NM102 displays antimicrobial activity, specifically in the context of infection, even against clinically relevant Gram-negative bacterial pathogens such as resistant K. pneumoniae, P. aeruginosa, and pathogenic E. coli. In animal models, NM102 was able to block infections with these pathogens without inducing host toxicity nor damaging the microbiota. Finally, NM102 also blocked the function of Mfd as an evolvability factor, thereby reducing the bacteria's ability to develop antimicrobial resistance. Overall, our findings identify and characterize a promising antimicrobial candidate that could be used either alone, as a booster for the host immune system, or in combination with other antibiotics by inhibiting Mfd's evolvability activity. To identify antimicrobials with a new mechanism of action, we performed a rational, targeted, and structure-based high-throughput in silico screening of small molecules to challenge their efficiency to accommodate to the ATP binding site of the bacterial protein Mfd. 3D modeling of E. coli Mfd in an active conformation was designed (Supp Fig. Using this template, a large library of 4.8 million compounds was screened in silico, to challenge their capacity to bind the ATPase site of Mfd. This virtual screening led to the identification of 95 molecules, selected for subsequent experimental validation (Supp Fig. The 95 molecules were challenged to inhibit the ATPase function of E. coli Mfd in vitro (Fig. The inhibition rates of Mfd activity ranged from 10% to 85%. Dose-response assays on ATPase activity (Fig. A In vitro high-throughput screening for inhibitors of Mfd-C ATPase activity. The indicated compounds were tested at a final concentration of 100 mg/mL. The results are the average of two independent experiments done in duplicate with standard deviation. B Michaelis Menten plot for inhibition activity of NM102 (0–100 µM) on E. coli Mfd-C (0.35 µM) ATPase activity with ATP (0–0.3 mM). The results are the average of at least three independent experiments with standard deviation. C Lineweaver-Burk plot showing NM102 inhibition of Mfd-C ATPase activity through a competitive mode of action. The results are the average of at least three independent experiments with standard deviation. D Chemical structure of the NM102 compound. NM102 is a small molecule with a chemical scaffold relatively close to ATP as it is composed of an indole-like ring close to the adenosine ring followed by a ribose-like ring and eventually polar sulfur groups that could mimic phosphate moieties (Fig. To measure the specific binding of NM102 to Mfd, Isothermal Titration Calorimetry (ITC) experiments were conducted (Fig. ITC is based on the determination of the heat involved during the binding of a ligand to a receptor, measuring power units as a function of time. The obtained stoichiometry value (n = 1.1) indicates a 1:1 binding interaction between the Mfd protein and NM102 with a Kd of 83 ± 9 µM. By contrast, in the presence of NM102, ATP binds to Mfd with a Kd of 430 ± 50 µM. Thus, NM102 binds to Mfd with a better affinity than ATP and successfully inhibits Mfd-ATP interaction which decreased by ca 70% (A = Ka/Kapp = 0.34). These data demonstrate that NM102 binds specifically to the ATP binding site of Mfd and inhibits Mfd-ATP binding in a competitive manner. The heat rate in the presence of NM102 was compared with and without NM102 and plotted as a function of time (higher panel), and the normalized fit was measured with an increased mole ratio (lower panel). B ITC thermodynamic parameters associated with the interaction of Mfd with ATP in the absence or presence of NM102. Kd, dissociation constant Kd, Ka, association constant, A, ratio of Ka in the presence vs absence of inhibitor NM102. To assess the specificity of the NM102 inhibitor towards the ATPase activity of Mfd, we evaluated its capacity to decrease the ATPase activity of various eukaryotic proteins, including functionally related and unrelated ATPase proteins. The activity towards four proteins was tested: ERCC3, ERCC6, XPD, and yUpf1 (Fig. NM102 sharply inhibited Mfd ATPase activity, while it had no effect on all other ATPase protein activity, strongly suggesting that NM102 is selective for Mfd activity. In addition, the activity of NM102 was also tested towards the bacterial helicase RecG since the library screen that identified NM102 was performed on the Mfd sequence of E. coli morphed with respect to the active ADP binding site of RecG (Supp Fig. Consistently, a slight activity of NM102 was observed, nevertheless significantly lower than for Mfd, further emphasizing NM102 specificity for Mfd. A NM102 inhibition of bacterial Mfd and eukaryotic proteins (ERCC3, ERCC6, XPD, yUpf1) ATPase activity. ATPase activity of the proteins, in the absence and presence of 100 µM of NM102, was assessed by using BIOMOL® Green reagent microtiter-plate assay. Data were normalized to that of the DMSO control. The graph shows the mean of at least two independent experiments with standard deviation. B Michaelis Menten plot for inhibition activity of NM102 (0 to 100 µM) on E. coli RecG (0.35 µM) ATPase activity with ATP (0 to 0.3 mM). The results are the average of two independent experiments done in duplicate with standard deviation. C Binding energy measured in silico in kcal/mol between ATP or NM102 and Mfd (left) anf Upf1 (right). NM102 showed a marked enhanced affinity for Mfd as compared to its cognate ATP ligand, with binding energies of −9.8 kcal.mol−1 vs -6.5 kcal.mol−1, respectively, consistently with ITC data. 4A) highlighted that Mfd accommodates NM102 at its ATP binding site, within the same binding pocket that encompasses the motif I with residues from Asp629 to Glu636, that also includes the catalytic Lys634. Additionally, we were able to define a sphere of interaction through twenty-one positions involved in the binding of both ATP and NM102 (Suppl. NM102 is a slightly longer molecule as compared to ATP, especially in its extended conformation. Thus, its accommodation extends to interact with amino-acid residues that are remote from the catalytic center, namely Gln664, His665, Asn668, and D876 of motif V. This feature could explain the increased affinity for NM102 compared to ATP. A Upper part: strip of the eight domains with the following color code blue D1a-D2-D1b, orange D3, pink D4, yellow D5, green D6, and red D7. Lower part: Mfd rendered surfaces of conformations L0 inactive (pdb id 2eyq) and L1 active (pdb id 6 × 26), and their corresponding view at 90°. Right part, top insert: Close-up of Mfd from E. coli with the docking of ATP (in stick with carbon colored in cyan) and down insert: similar view with NM102 (in stick with carbon colored in deepteal). The residues involved in the sphere of binding of ligands are shown as sticks. B Close-up of the docking of ATP (cyan sticks) and NM102 (orange stick) in Mfd of E. coli harboring the site-directed mutations F597A, F599A, K634A, and E730Q. For the sake of comparison, position of ATP in the non-mutated Mfd is shown as black lines. In parallel to Mfd, bindings of ATP and NM102 were simulated in Mfd that was mutated in silico at its catalytic center (Fig. Lys634A and Glu730Q since they are known to define the active site9. Accordingly, they were mutated into alanine residues to abolish the apolar contribution of their aromatic ring, while preserving local and global folding of the protein. Importantly, the docking showed that mutated Mfd completely changed the accommodated poses of both ATP and NM102, and that their alternative positions were inconsistent with a competitive occupancy of the ATP site by ATP. These data further highlight the interaction of NM102 at the ATP binding site of Mfd. Mfd is widely observed in the bacterial kingdom10. To assess whether NM102 could more generally bind and inhibit the function of Mfd from a wide range of bacteria implicated in AMR, multiple sequence alignment and homology modeling of Mfd were performed for one strain of each species of the ESKAPE bacteria (Supp Fig. The alignment revealed that Mfds are distinctly clustered into Gram-positive and Gram-negative bacteria, nevertheless they overall share more than 45% of sequence similarity along 1200 amino acid residues of the ESKAPE bacteria. Markedly, this average on Mfd full-length sequence does not reflect the trend observed for each functional module. Indeed, modules composed of domains D1a-D3, D4, D5-D6, and D7 show 47%, 58%, 68% and 44% of sequence similarity, respectively (Supp Fig. Particularly, the ATPase motor module, composed of D5 and D6 domains, is by far the most conserved. Furthermore, the ATP binding site, which locates at the interface between D5 and D6, showed fairly high to strict conservation of the catalytic motifs, the so-called motifs I to VI (Supp Fig. To profile the affinity of NM102 towards Mfds from the ESKAPE bacteria and identify the probable conserved features of binding, NM102 was docked into each 3D homology model of Mfd and the binding energy was measured (Suppl. Molecular binding in all the modeled Mfds revealed that both ATP and NM102 maintain similar interactions at the ATP binding site, with residues conserved in all species, particularly the catalytic Lys634 and Glu730 that are strictly conserved, and Phe597 and Phe599 that are replaced by tyrosine residues in S. aureus, A. baumannii, and E. faecium (Mfd E. coli numbering). Their mode of binding can be clustered into two rather isoenergetic groups. One cluster gathers NM102 pose similar in Mfd E. coli, S. aureus, A. baumannii, K. pneumoniae, and P. aeruginosa, while the other gathers an NM102 pose similar in Mfd E. faecium and E. cloacae. Also, 21 residues were identified as identical between species and always positioned within the 5 Å coordination sphere (Suppl. Particularly, P1 Phe597 and P2 Phe599 could be responsible for the π stacking pose of the adenine-like moiety while polar P12 Thr635, and charged P11 Glu636 and P21 R905 could interact closely with the sulfonamide and phosphate-like groups of NM102. Overall, ATP binding ranged from −5.5 to −8 kcal mol−1, whilst NM102 binding ranged from -9.0 to −10.3 kcal.mol−1, suggesting that NM102 binds to all ESKAPE Mfds with a significantly better affinity as compared to the physiological ATP. Taken together, our results strongly suggest that NM102 could compete with ATP at the ATP binding site in all Mfd proteins of the ESKAPE group bacteria, hence, specifically inhibiting Mfd ATPase activity. As the challenging urgency is to identify molecules efficient against Gram-negative pathogenic bacteria of the ESKAPE group, the rest of the study is focused on clinically relevant K. pneumoniae, P. aeruginosa and E. coli strains. Mfd is a non-essential protein in unstressed conditions. However, we have previously shown that Mfd is essential to resist NO stress during infection20,21. Our innovative antimicrobial strategy consists in identifying inhibitors that have no antibiotic activity under unstressed conditions, but are only active under stressed conditions, where Mfd is required. NM102 was thus first tested for its antimicrobial activity in the absence of stress. Under these conditions, no antimicrobial activity on K. pneumoniae was observed (Fig. Conversely, under NO stress conditions, NM102 inhibited clinical K. pneumoniae survival (IC50 of 78.8 µM) (Fig. These results highlight that NM102 has an antimicrobial activity against K. pneumoniae under NO stress conditions. Strains were grown to exponential phase in LB medium. Bacteria were exposed for 4 h at 37 °C to 50 µL of increasing concentrations of NM102 alone A K. pneumoniae DOU or with NOC-5 as a NO donor, B K. pneumoniae DOU, C E. coli ATCC 25922, D P. aeruginosa CIP27853). These data show that NM102 acts against Gram-negative pathogenic bacteria under conditions that mimic immune stress. To assess the antimicrobial activity of NM102 as an antimicrobial in vivo, Bombyx eri larvae were injected with P. aeruginosa CIP27853 and treated with NM102 (Fig. A P. aeruginosa CIP27853 wt and ∆mfd strains were grown to late exponential phase in LB medium at 37 °C and injected into forth instar silkworm larvae without or with NM102 (2.78 µg/larvae). At 24 h post-infection larvae were crushed and the content of each tube was serially diluted on LB plates for CFU numeration. Each graph represents the values obtained for n insects with the mean value indicated. P values are calculated against the condition wild type-strain without NM102, using Mann-Whitney test (p < 0.0001). For mice experiments, K. pneumoniae ATCC 700603 wt and ∆mfd (B), K. pneumoniae ALE (C), P. aeruginosa CIP27853 (D) were grown to late exponential phase and diluted in PBS. bacterial administration was performed through slow instillation of 20 µL of bacterial suspension (1 × 107 CFU). 200 µL of NM102 nanoformulation (NF-NM102 at 1.5 mg/kg or 6 mg/kg) or empty nanoformulation (control) were administrated in mice via the intraperitoneal (i.p.) Alternatively, 20 µl of a mixture containing bacterial suspension (1×107 CFU) and NM102 nanoformulation (1.5 or 6 mg/kg) or empty nanoformulation was administered via the i.n. Mice were sacrificed by cervical dislocation after 24 h and the log CFU in the lung was calculated per gram of organ. The exact P values, minima, maxima, center, bound of box, and whisker are all provided as source data. First, the amount of CFU of the ∆mfd mutant was lower compared to the CFU recovered following insect infection with the wild-type strain, showing that Mfd is a critical virulence factor for P. aeruginosa. Second, NM102 did not decrease the bacterial load of the ∆mfd mutant, confirming that NM102 is specifically active on Mfd in vivo. It is interesting to note that NM102 decreased bacterial load to the same level that could be reached with the ∆mfd mutant, highlighting that NM102 had a maximum of activity, completely inhibiting the Mfd activity, similarly to a complete deletion of the target. To further investigate the potential of NM102 in vivo, a mouse lung model of infection with K. pneumoniae and P. aeruginosa was chosen23. Due to its poor solubility in most common solvents (Suppl. Table 2), the above studies were performed using a solution of NM102 in DMSO. Thus, an alternative to the DMSO vehicle was developed to further assess the therapeutic effect of NM102 and to offer a more viable medication. Herein, a DMSO-free nanoformulation was prepared via a sonication assisted-nanoprecipitation method in the presence of poly(d-L lactide-co-glycolide) (PLGA) (Suppl. The resulting NM102 nanoformulation (drug content = 0.6 mg/mL) exhibited an average diameter of 180 nm, with narrow particle size distribution (polydispersity index, PDI = 0.2) and a neutral surface charge (ζ potential = −2 ± 1 mV). An “empty” nanoformulation, used as a control, was prepared according to the same protocol but without the drug (average diameter = 136 nm ± 7, PDI = 0.08 ± 0.04, ζ potential = −4 ± 1 mV). The NM102 nanoformulation showed a good colloidal stability for up to 2 weeks at both 4 °C and room temperature (Suppl. Mice were infected intranasally with K. pneumoniae and treated via the intraperitoneal route with the NM102 nanoformulation. The NM102 nanoformulation significantly reduced the bacterial burden in lungs compared to control group treated with empty nanoformulation in a dose-dependent manner (by 4 times with 1.5 mg/kg of NM102 and by 15 times with 6 mg/kg of NM102, corresponding to 75% and 94 % reduction of the bacterial load, respectively), showing that NM102 nanoformulation acts as an efficient antimicrobial against K. pneumoniae infection in vivo. The efficacy of the NM102 molecule was compared under the same conditions to that of meropenem (MPN), a beta-lactam antibiotic that is commonly used in the treatment of various infections including K. pneumoniae and P. aeruginosa23. NM102 nanoformulation showed similar efficacy to MPN in reducing the K. pneumoniae CFU in the lung of the infected mice at 6 and 10 mg/kg, respectively (Fig. 6B), resulting in 94% and 93% reduction of the bacterial load, respectively (Suppl. To confirm that NM102 targets Mfd in vivo, mice were also infected with the K. pneumoniae ∆mfd mutant. NM102 decreased the CFU recovered of the wild type strain to the level of CFU obtained with a ∆mfd mutant, and NM102 had no impact on the ∆mfd mutant, which further highlights that NM102 acts on Mfd in vivo. The efficacy of NM102 was further investigated on the clinical K. pneumoniae ALE strain, which is resistant to several antibiotics including MPN (Fig. Strikingly, MPN at 10 mg/kg had no significant effect on the resistant K. pneumoniae ALE, highlighting the potential of the NM102 molecule on resistant Gram-negative bacteria in vivo. For P. aeruginosa, the NM102 nanoformulation at 6 mg/kg decreased by 14 times the mean bacterial load (although not statistically significant), and MPN at 10 mg/ml decreased the bacterial load by 26 times (Fig. The combination of the two treatments was drastically efficient against P. aeruginosa in an NM102-dose dependent manner compared to single treatments (decrease by 127 times for NM102 1.5 mg/kg + MPN 10 mg/kg, and by 730 times for NM102 1.5 mg/kg + MPN 10 mg/kg). These data suggest the possible use of NM102 nanoformulation as an antimicrobial drug during combination therapy. Overall, NM102 proved to be an antimicrobial molecule active against clinically-relevant pathogens, including multidrug-resistant bacteria that may be associated with difficult-to-treat infections and sometimes therapeutic failure. In the development of anti-infectives, an important feature that needs to be assessed is their impact on the host. When tested for toxicity towards human cells (Fig. 7A), NM102 and the NM102 nanoformulation did not induce cytotoxicity either to HeLa or to Vero cells at the highest doses used in insect and mouse experiments, respectively. MPN showed no toxicity to HeLa cells, but induced cytotoxicity in Vero cells at 10 and 100 µM. Cells were treated for 1 h with MPN at 10 and 100 µM, NM102 at 10 and 300 µM, NM102 nanoformulation at 1.29 and 5.16 mM or their respective control 10% DMSO or empty nanoformulation. Values for treated cells were normalized to the untreated control. The results reported are mean ± SD of three independent experiments each in triplicates. Changes in the mice body weight were assessed for 7 days. Treatment with NM102 had neither impact on insect or mouse survival nor on the mice weight (Fig. Importantly, PLGA, which was used for the nanoformulation of NM102, is an FDA-approved copolymer for which neither systemic nor organ toxicity has been reported in animal models25,26. As NM102 is active on bacteria only in the context of a stress-like inflammation, we hypothesized that following pulmonary infection, the inflammation will be localized at the site of infection and that NM102 treatment should not affect the distant gut microbiota species, in contrast to treatment with conventional antibiotics. To assess the innocuity of NM102 towards the gut microbiome in vivo, mice were first treated with NM102 without bacterial infection, and feces were recovered after treatment. NM102 did not impact the overall quantity of bacteria in the feces, after 1 and 7 days of treatment. By contrast, streptomycin decreased sharply the overall quantity of bacteria in the feces after 1 day. We further assessed the composition of the gut microbiome following nasal infection by K. pneumoniae and NM102 nanoformulation treatment. We did not detect changes in gut microbiome diversity between untreated and NM102-treated mice. The family-level microbiome profile, following NM102 treatment, largely resembled that of untreated mice (Fig. The Shannon indexes of empty nanoformulation and MN102 nanoformulation-treated mice were similar to untreated mice (Fig. Similarly, we did not detect significant differences in beta-diversity between treatments using weighted and unweighted UniFrac distances (Fig. Gut microbiome diversity is unaffected by NM102. Mice were infected with K. pneumoniae and left untreated or treated with NM102 nanoformulation (NF-NM102) or the empty nanoformulation (NF) as control. After 24 h, feces were collected and the microbiome diversity was analyzed by 16S rRNA gene sequencing. Relative abundance of bacterial families (A), Shannon index (B), and weighted and unweighted UniFrac distances (C) were determined. The bar colors represent bacterial families with mean abundance >1%. Each dot (B, C) and each column (A) represents the value obtained for one mouse. 6 A, B, C is n = 7 mice per group. 6C is a two-tailed Wilcoxon test with Benjamini-Hochberg correction for multiple comparisons. Mfd has been shown to play a critical role in promoting bacterial resistance to commonly used antibiotics and has been proposed as a target for the development of anti-evolvability drugs18. We thus assessed whether NM102 could also inhibit Mfd evolvability activity. First, the mutation rate of E. coli wild type, ∆mfd mutant, and ∆mfd complemented strains was assessed following exposure to rifampicin that commonly induces a high frequency of resistance in vitro (Fig. An intermediate value of mutation frequency was obtained for the complemented strain, indicating that the complementation is likely partial, which is not surprising due to the use of an inducible promoter. A The mutation rate of E. coli wt, Δmfd, and Ω∆mfd/mfd strains were measured following exposure to NM102 (100 µM) or the DMSO control, using the frequency of spontaneous accumulation of resistant mutants to rifampicin as reference measurement. Data in this figure correspond to 12 independent cultures. Lea Coulson analysis was used on webSalvador to determine significance using an efficiency parameter of ε = 0,5 (∗p < 0.05; ∗∗p < 0.01; ****p < 0.001, the exact P values are provided as source data). Resistance evolution of E. coli to rifampicin (B) and streptomycin (C) was measured following exposure to NM102 (100 µM) or the DMSO control. Line plots show the median antibiotic concentration at each sampled time point for three independent experiments. Then, the impact of NM102 on the evolution rate was evaluated following treatment of E. coli with several antibiotics. NM102 significantly decreased the evolution rate of E. coli following treatment with rifampicin (Fig. 9C) by eight and two folds, respectively. These data strongly suggest that NM102 targets the evolvability activity of Mfd and could thus be used as a combinatory drug to decrease antimicrobial resistance. To address the need for novel bacterial drug targets and the design and synthesis of unique anti-infectives with new mechanisms of action, especially towards Gram-negative bacteria, we have identified and characterized an innovative bacterial target, the Mutation Frequency Decline protein, Mfd, and promoted its inhibition by a newly discovered therapeutic molecule. Mfd is a nonessential transcription repair coupling factor that is ubiquitous and conserved in bacteria but absent in eukaryotes. In this study, we identified a molecule, NM102, as an inhibitor of Mfd activity. We showed its dual capacity, first to display antimicrobial activity in animals against K. pneumoniae and P. aeruginosa, two major nosocomial Gram-negative bacteria of the ESKAPE group, and second to inhibit Mfd evolvability function, thus reducing the frequency of antibiotic resistance appearance. As such, NM102 is active on its own to prevent bacterial infections. Furthermore, it is also predicted to strongly reinforce the efficacy of existing antibiotics and prevent the emergence of resistance, an aspect rather unique and essential in the field of drug development and antibiotic therapy. Since Mfd is not an essential protein, standard minimum inhibitory concentration (MIC) assays could not be used to identify inhibitory molecules. This rational approach allowed screening of a large library of molecules to identify potential inhibitory molecules. Hence, we propose a strategy that can be applied to other non-essential bacterial targets, providing a rapid and efficient pathway to screen the potential of small molecule without the need to develop laborious and specific in vitro activity tests. The main advantage of such method is its high interpretability regarding pocket properties. Additionally, once a potentially druggable pocket has been identified, it can be screened using various in silico tools to propose potential ligands for experimental validation and further optimization. We chose to target the ATP-binding domain of Mfd to identify inhibitory molecules because: (i) ATP plays a central role in Mfd activity; (ii) the 3D-structure of the ATP-binding site structure in Mfd is known; (iii) ATP-binding sites, despite their large structural diversity, are known to be druggable, as evidenced by inhibitors of many targets (e.g., protein kinases, HSP-90a)28; and (iv) although a large proportion of proteins have ATP-binding sites, these sites exhibit significant structural diversity, resulting in distinct specificity and selectivity for ligands28. In silico, its molecular mode of interaction explains how NM102 is competitive and docks into the ATP pocket with greater affinity than ATP. Notably, its length could reinforce the interaction between D5 and D6 domains, which is expected for ATPase activity, through direct and closer interaction with the Q664-H665 stretch and an ancillary salt bridge between the D876 and R905 residues. Consistently, NM102 competed with ATP in vitro. Moreover, such interaction could trap Mfd in its active ATPase conformation and prevent it from engaging in the subsequent DNA translocation step. NM102 interacts strongly through polar interaction with the catalytic residue E730, whose mutation into glutamine is defective in ATP hydrolysis. Furthermore, E730 is sandwiched between R733 and R953, and R953A has been reported to result in the loss of DNA translocation9. Any modification of interaction within the E730 nexus, which connects both ATPase and DNA translocation activities, could impact the coordination of the two activities, which must be exquisitely tuned. Notably, the pseudo-adenine part of NM102 conserves π-stacking with F599, similar to that seen in the adenine moiety of ATP. Furthermore, we have mutated the four residues F597A, F599A, K634A, and E730Q that are possibly critical in all species for the binding of NM102 in the ATP binding pocket of Mfd. Indeed, K634A and E730Q were previously described as important for ATP binding9 and F597A and F599A are evidenced in this work as crucial for the recognition of the indole-like ring of NM102. Neither ATP nor NM102 could correctly be positioned into the mutated Mfd ATP pocket, further demonstrating the specific binding of NM102 in the ATP binding pocket of Mfd. These data provide mechanistic and molecular insights into NM102 binding with respect to its target and specificity. Regarding specificity, NM102 neither binds nor inhibits the ATPase activity of various eukaryotic ATPase proteins, including closely functionally related ATPase proteins involved in transcription-coupled nucleotide excision repair with ATP-dependent DNA helicase activity. These data further suggest that the ATP pockets of ATPase proteins exhibit structural diversity, allowing for distinct ligand specificity. Mfd activity enables bacteria to overcome the host's defense responses20. Host defense against bacteria is predominantly mediated by cellular immune mechanisms. Various cells, such as macrophages, neutrophils, and epithelial cells produce toxic species, including NO, during most infections. NO can severely damage biological molecules, such as proteins and nucleic acids, thereby inducing DNA damage and strand breaks29. Bacteria express sensor proteins that can detect NO and, accordingly, activate the expression of enzymes that detoxify NO before it reaches lethal levels33,34. Additionally, we previously demonstrated a link between NO-induced bacterial DNA damage and DNA repair by Mfd21. Mfd is ubiquitous in bacteria, and almost all bacteria induce an NO response from the host, so the Mfd-targeted inhibitor is expected to have a broad spectrum of activity. We have consistently shown that NM102 docks in silico to the Mfd of all ESKAPE pathogens and has antibacterial activity against Gram-negative bacteria under NO stress. As a proof of concept for efficacy against Gram-positive bacteria, we also confirmed that NM102 has antimicrobial activity against B. cereus under NO stress, the same B. cereus in which we initially identified the function of Mfd during immune stress (Suppl. Mfd activity is not limited to NO stress. Indeed, Mfd has been shown to protect against oxidative stress in Bacillus subtilis35. This may explain why NM102 was efficient in vitro during ATPase assay, with a relatively low IC50 value, and also effective in vivo, while exhibiting higher IC50 values during in vitro NO stress, which may not fully represent the conditions encountered by bacteria during in vivo infections. The ability to evolve is critical for bacterial survival, particularly in the context of pathogenesis, where evading host immunity is essential and requires constant adaptation. Therefore, Mfd may be involved in bacterial virulence, likely by preventing DNA damage induced by the host through its overall immune response and/or stress conditions encountered in vivo. This question is difficult to address technically in vivo as bacteria encounter many different stresses during infection. Another hypothesis could be the role of Mfd during the modulation of transcription. Indeed, it has been recently suggested that Mfd acts as a transcription factor able to modulate the activity of RNAP and gene expression. These activities may also explain the observed effect of Mfd in stressed bacteria9,16,36. Thus, inhibition of Mfd may not be effective in the treatment of some pathogens. We then focused our study on the impact of NM102 on the mutagenic property of Mfd to assess its role in antibiotic resistance. Indeed, Mfd has been shown to be a general evolvability factor promoting bacterial mutagenesis during infection9. Mfd is reportedly required for developing high levels of drug resistance upon initial exposure to sub-inhibitory concentrations of antibiotics, accelerating AMR development18. Additionally, stress-induced mutagenesis can help pathogens generate drug-resistant cells during antibiotic therapy38. Here we show that NM102 inhibits Mfd's function as a mutation and evolvability factor during antibiotic stress. As such, NM102 can target the evolvability property of Mfd. By decreasing the resistance pressure, these new drugs may have a longer efficacy, which in turn will help limit the spread of antibiotic resistance. Interestingly, a small molecule, ARM1, has very recently been identified as an inhibitor of Mfd-dependent mutagenesis, further demonstrating the potential of fighting AMR by targeting this protein involved in the acceleration of evolution leading to increased AMR19,39. The mechanisms leading to Mfd inhibition are different between NM102 and ARM1. Indeed, in contrast with NM102, ARM1 did not bind to the ATP binding site of Mfd and had no impact on Mfd ATPase activity19. Combination therapies have played and will continue to play an important role in clinical treatment. We have shown that NM102 is effective as antimicrobial in vivo, both alone and in combination with meropenem, and that NM102 treatment can kill bacteria that are resistant to current antibiotics. Thus, a combination treatment, including an Mfd inhibitor coupled with a classical antibiotic, could be a promising strategy with increased antimicrobial efficacy and a reduced likelihood of resistance development at the onset of treatment. As recently proposed, targeting the evolutionary capacity of bacteria could also have broad implications beyond AMR development, from reducing cancer evolution to limiting pathogenic diversity in the context of host immunity18. Drug loaded into nanocarriers has been widely proposed for the delivery of poorly soluble drugs, not only overcoming the drug's physicochemical limitation but also ensuring enhanced accumulation at the site of action and sustained drug release40,41,42,43. Herein, we provide proof of the efficacy of the aqueous nanoformulation of NM102, paving the way for developing a novel antibiotic formulation suitable for widespread use. Classical antibiotics target essential bacterial pathways and, as such, are highly efficient but not specific, thus also affecting commensal members of the microbiome44. The gut microbiome is essential for human and animal health, as it is involved in various health-associated processes, including digestion, metabolism, and immune development, and it also protects against invading pathogens. Since Mfd is not an essential protein, its inhibition by NM102 should not impact the distant host microbiome in the absence of inflammatory stress. To test this hypothesis, we assessed the gut microbiome composition after NM102 treatment in healthy mice and mice infected intranasally with K. pneumoniae. NM102 had no detectable effect on the diversity of the gut microbiome, which can be collaterally damaged with classical, non-selective antibiotics. This is a unique feature that may help develop microbiome-friendly molecules that not only treat infectious diseases but also promote the overall health of patients. However, in enteric infections, local gut inflammation may make Mfd essential for the survival of bacterial commensals as well. Therefore, in those cases NM102-like compounds may also negatively impact the gut microbiome, causing collateral damage similar to that of current antibiotics, but with the advantage still of being effective in combating resistance. Besides, since Mfd is a non-essential protein, targeting it should decrease the resistance pressure by NM102. NM102 by itself does not kill bacteria and therefore, resistance is not observed in the absence of any stress. In vivo, during inflammation, Mfd is essential and NM102 treatment hinders bacterial survival. This is likely to cause bacterial resistance eventually. Nonetheless, we can speculate that this resistance will be reduced in comparison to traditional antibiotics, which apply continuous pressure to both pathogenic and microbiota bacteria over the course of treatment. More research will be required on this topic. Overall, new drugs targeting Mfd, a non-essential virulence and evolvability factor, could be used alone or as adjunctive therapy during the treatment of infections to improve the potency of current antibiotics and reduce the emergence of resistance. We are confident that these innovative molecules could provide a promising therapeutic option and expand the arsenal of drugs available to combat AMR and potentially other diseases. All animals were handled in strict accordance with good animal practice as defined by the local animal welfare bodies (Unité IERP, INRAE Jouy en Josas, France, Agreement No. All animal experiments were performed in accordance with European Directive 2010/63/EU. All efforts were made to minimize animal suffering and adhere to the 3Rs principle (Reduce, Refine, Reuse). As we aimed to explore the capacity of Mfd to bind ligands that could be competitive with ATP, a prerequisite was to model Mfd in a conformation conductive ligand binding, meaning that the ATP-binding site must adopt an open and accessible conformation. At the time of computation, all available crystal structures of E. coli Mfd were in an inactive form due to the so-called walker A motif closing over the active site, thus precluding any binding (pdb id 2EYQ)12. The D2 domain of RecG helicase from Thermotoga maritima (pdb id 1GM5) is a structural homolog of Mfd, solved in an active conformation complexed with ADP45. The Bioinfo-DB database (http://bioinfo-pharma.u-strasbg.fr/bioinfo) is an in-house developed database of 4.8 million commercially available compounds as powder (1–50 mg) filtered according to internal rules to contain only drug-like compounds. The database was first filtered to retain compounds grossly resembling nucleotides and fulfilling the following properties: (i) at least one hydrogen-bond donor, (ii) at least two hydrogen-bond acceptors, (iii) at least one aromatic ring, (iv) predicted aqueous solubility higher than 50 µM (predicted with PipelinePilot v.9.5, Dassault Systèmes, Paris), (v) topological polar surface area lower than 120 Å2. The filtered set of 1.2 million compounds was then converted into three-dimensional (3D) structures using Corina v3.40 (Molecular Networks, Erlangen, Germany). Up to four stereoisomers were created for compounds with undefined stereocenters. Altogether, 3D atomic coordinates were defined for 1,874,034 compounds, constituting the docking set. A prototyping molecular dynamics object or “protomol” was first generated from the list of binding site residues. Potential hits were selected according to the following two strategies (Suppl. Then, compound redundancy was removed (the highest score was retained if more than two poses originated from the same compounds) to yield 91 compounds. A chemical diversity selection by maximum common substructures was performed using the LibMCS algorithm (ChemAxon Ltd., Budapest, Hungary) to retain a first set (SET1) of 23 chemically diverse compounds. A chemical diversity selection by maximum common substructures was performed using the LibMCS algorithm (ChemAxon Ltd., Budapest, Hungary) to retain a second set (SET2) of 74 chemically diverse compounds. Previously defined SET1 and SET2 were merged, yielding a final selection of 95 unique hits (two hits were common to both sets) that were further purchased in 5 mg quantities. They were aligned using the multiple sequence alignment MAFFT algorithm with default parameters48. The alignment was visualized using Jalview 249. For homology modeling, the recent structure of E. coli Mfd in the so-called L1 state and in complex with the RNA polymerase was chosen as template because L1 state-Mfd adopts an active and opened conformation, that renders the ATP pocket accessible. The L1 state was solved at 4.6 Å resolution, using cryo-electronic microscopy; its pdb id is 6X2650. Based on this template, one hundred models were computed for each Mfd of the ESKAPE bacteria, in their wild-type form, using the homology building software Modeller version 9.1851. Each model was ranked and selected upon the lowest DOPE and function scores of Modeller. Eventually, the stereochemistry of selected model was checked using Procheck50. To obtain Mfd with the mutations F597A, F599A, K634A, and E730Q, the best model of non-mutated Mfd was iteratively substituted four times, using the Pymol command mutagenesis. ATP and NM102 were individually docked into the active site of each modeled Mfd, in L1-state conformation, with respect to the homology modeling described above. AutoDock Tools 4.2 (ADT4) was used with a grid box conserved in size and position, centered onto the active site of the ATPase, with the genetic algorithm of Lamarck and the default parameters for 20 runs. ATP and NM102 coordinates were obtained from the ZINC database (https://zinc.docking.org/substances/home/) in mol2 format, converted into ADT4's PDBQT format with their dihedrals set free to rotate. Schrödinger, LLC), and protein-ligand interactions were characterized using the Protein–Ligand Interaction Profiler (PLIP) program52. The same protocol was used for Mfd mutated at F597A, F599A, K634A and E730Q. K. pneumoniae DOU CTXM-15, ALE CTXM-15+, P. aeruginosa CIP27853, E. coli ATCC25922 were provided by Dr. Thierry Naas (CHU de Bicêtre, Le Kremlin-Bicêtre, France). The MIC of antibiotics was determined on those strains. Briefly, a 0.5 McFarland standard bacterial solution was prepared in physiological saline and then diluted 1/1000 in Mueller Hinton broth. Then, 50 µL of bacterial solution was added in 96 well plates containing 100 µL of various concentrations of antibiotics (Penicillin, Ampicillin, Ceftriaxone, Meropenem, Cefotaxin, Oxacillin, Streptomycin, and Ciprofloxacin). The plates were incubated overnight at 37 °C. Bacterial growth was measured in each well to determine the MIC (Suppl. E. coli K12 wild-type and ∆mfd mutants were previously described53. The complemented E. coli K-12 Δmfd Ωmfd mutant strain was obtained by inserting into the E. coli K12 Δmfd mutant strain the complementation plasmid pET21a containing the entire mfd gene (synthesized by Biobasic, genetic environment Suppl. Briefly, the receptor strain was suspended in 100 µL of a buffer containing LB, 10% PEG4000, 5% DMSO, MgCl2 0.01 M, MgSO4 0.01 M and mixed the complemented plasmid. A heat shock was performed and complemented strains were selected after regeneration on LB agar plates supplemented with 100 µg/mL of ampicillin. The corresponding mutant was named E. coli Δmfd Ωmfd. All Strains were stored at −80 °C as 20% (v/v) glycerol stocks. The P. aeruginosa CIP27853 ∆mfd mutant strain was constructed by double-crossover region deletion. Briefly, using the available sequencing information of P. aeruginosa CIP27853 strain, approximately 1-kb regions upstream (region 3360879 to 3361826) and downstream (region 3363535 to 3364310) of the mfd gene were synthesized by Genecust (Boynes, France). The two fragments were cloned and juxtaposed into the suicide vector pEXTK by Genecust. The resulting plasmid, pPa001, was transformed into electrocompetent E. coli SM10 λpir and conjugated into P. aeruginosa CIP27853 strain as previously described (Hmelo et al., 2015). Briefly, donor and recipient strains were mixe 1:2 on a sterile 0.05 μm-pore-size-membrane (VMWP02500 Milipore) placed on LB agar plates supplemented with irgasan 25 µg/ml and gentamicin 75 µg/ml and incubated for 18 h at 37 °C. After mating, bacteria were resuspended from filter in LB with IPTG 1 mM, incubated for 3 h at 37 °C, and plated on LB agar plates with Azithromycine (AZT) 100 µg/ml and IPTG 1 mM for 18 h at 37 °C. Transconjugants were selected for loss of gentamicin resistance by picking individual colonies on LB agar and LB agar supplemented with 75 μg/mL gentamicin at 37 °C. Deletion of the mfd gene by double recombination was verified by PCR using primers upstream (GAACACCACCAGTTCCACCTG) and downstream (GGTAGAGCTGGCCAATCAGC) of the cloned region and by sequencing. The resulting mutant was named P. aeruginosa ∆mfd. The K. pneumoniae ATCC 700603 ∆mfd mutant strain was constructed by double-crossover region deletion. Briefly, using the available sequencing information of K. pneumoniae ATCC700603 strain, ~1-kb region upstream (region 2056347 to 2057137) and downstream (region 2060189 to 2060810) of the mfd gene were synthesized by Genecust (Boynes, France). The synthesized DNA fragments and the CatR cassette were digested with the appropriate enzymes and ligated to produce a “mfd-upstream”-“CatR”-“mfd-downstream” KpnI-EcoRI fragment, which was then inserted between the KpnI and EcoRI sites of the pDG704 vector (Addgene). The resulting plasmid, pKp002, was transformed into electrocompetent E. coli SM10 λpir and then conjugated into K. pneumoniae ATCC700603. Briefly, the donor and recipient strains were mixed 2:1 on a sterile 0.05 μm-pore-size-membrane filter (VMWP02500Milipore) placed on LB agar plates for 4 h at 37 °C. After mating, bacteria were resuspended from the filter in 1.5 mL of LB medium. Transconjugants were selected on LB agar supplemented with 25 μg/mL of chloramphenicol at 37 °C. Deletion of the mfd gene by double recombination was verified by PCR using primers located upstream (CCCCAGGATGTTGTTTTGCA) and downstream (CCGTTATCCTGCACCACATG) of the cloned region and by sequencing. The resulting mutant was named K. pneumoniae ∆mfd. The 95 molecules were sourced from commercial vendors: Enamine, Chembridge, Chemspace, AbamaChem, Asinex, Chemdiv, Interbioscreen, BCH Research and Specs. Each compound was dissolved in DMSO (Sigma-Aldrich) and stored at room temperature until further use. Roswell Park Memorial Institute (RPMI) 1640 GlutaMAXTM medium was purchased from Gibco. The Nitric Oxide (NO) donor, 3-[2-hydroxy-1-(1-methylethyl)-2-nitrosohydrazino]-1-propanamine NOC-5, was purchased from Calbiochem, Sigma-Aldrich. NOC-5 was dissolved in NaOH 0.01 N to a final concentration of 200 mM. Under these conditions, a stable NO-amine complex can spontaneously release two NO equivalents22. Meropenem (MPN) was purchased from Sigma-Aldrich and dissolved in water to a stock concentration of 25 mM. Poly(lactide-co-glycolide) (PLGA) (Resomer® RG502 H, acid terminated, MW = 7–17 kDa), poly(vinyl alcohol) (PVA) (87−89% hydrolyzed, MW = 30−70 kDa), trehalose and all other reagents and solvents were supplied by Sigma-Aldrich (France). E. coli MG1655 His6-tagged MfdC (residues 451-1148 Uniprot accession code P30958) was synthesized and purified by BioBasic Inc. (Markham, ON, Canada) with a purity >90%. Briefly, the expression plasmid pET21a containing mfdC was transformed into E. coli BL21(DE3), and expression of His6-tagged MfdC was induced with 0.5 mM IPTG at 37°C for 4 h. Cells were lysed, and protein was purified using a Ni-IDA column. For ITC experiment the full-length Mfd protein was synthesized by BioBasic Inc. (Markham, ON, Canada) and purified following the same protocol with a purity >90%. The XPD/ ERCC2 protein is involved in transcription-coupled nucleotide excision repair and is an integral member of the basal transcription factor BTF2/TFIIH complex. The XPB/ ERCC3 protein is an ATP-dependent DNA helicase that functions in nucleotide excision repair. It is a subunit of basal transcription factor 2 (TFIIH) and, therefore, also functions in class II transcription helicases55. It has an ATPase activity, interacts with several transcription and excision repair proteins, and may promote complex formation at DNA repair sites56. The three purified proteins were purchased from Abbexa Ltd. Saccharomyces cerevisiae protein yUpf1 HD (residues 221–851 Uniprot accession code P30771, which corresponds to the helicase domain) was kindly provided by Dr Hervé Le Hir (Institut de biologie de l'Ecole normale supérieure (IBENS), Paris, France) and described in ref. Mfd-C enzyme activity was evaluated by measuring the quantity of inorganic phosphate (PO4i) released using BIOMOL® Green reagent microtiter-plate assay (Enzo Life Sciences). For the initial drug screening, Mfd-C (0.35 µM) was incubated with DMSO (2% (v/v)) or with the 95 compounds (100 mg/mL) and 1 mM ATP for 10 min at 37 °C. To further evaluate NM102 activity, Mfd-C (0.35 µM) was incubated with NM102 at the indicated concentration (0 to 100 µM). ATPase reaction was measured in Tris buffer 0,05 M at pH 8 in the presence of ATP (0 µM to 0.3 mM) for 30 min at 37 °C. Then, the proteins were incubated with 2% DMSO or 100 µM NM102. The ATPase reaction was quantified after 30 min of incubation at 30 °C. For all proteins, 50 µL of each reaction medium was transferred into clear, flat-bottom 96-well plates, and the reaction was stopped by adding 100 µL of BIOMOL® Green reagent. Absorbance at 620 nm was measured in a microplate reader (Tecan). Absorbance values were then converted to nmols of released PO4i based on a PO4i standard curve prepared as recommended by the supplier. Data analysis and IC50 calculations were performed using GraphPad Prism 7.05 (San Diego, CA). Lineweaver-Burk plots were generated using SigmaPlot Software. The affinity constant and thermodynamic parameters of the interaction between NM102 and Mfd were determined by ITC using NanoITC (TA Instruments)58. The Mfd protein was dissolved in 0.1 M Tris + 0.2 M NaCl with 4% DMSO at pH = 8 (buffer A) and used in the assay at 20 µM. For competitive assay, Mfd was incubated with NM102 and Mfd protein interaction with ATP was evaluated. Each titration involved 20 injections at 300 s intervals of 2.5 μL aliquots of the compound NM102 solution (200 μM) into the sample cell (volume 200 μL) containing the Mfd solution (20 μM). Bacteria were grown in LB medium at 37 °C with shaking at 200 rpm until the late exponential phase. Bacterial suspensions were prepared in RPMI medium at concentrations between 5 × 104 and 1 × 105 CFU/mL, and 150 μL aliquots were dispensed into a 96-well plate. Bacteria were exposed to NO by adding 50 μL of NOC-5 at final concentrations ranging from 0 to 1000 μM. The bacterial survival rate was quantified by plating on LB agar after 4 h at 37 °C and normalizing the bacterial load in NOC-5-treated samples to that of untreated control samples (0 μM of NOC-5). NO dose–response curves were generated by plotting log10-transformed NOC-5 concentrations and fitting the data to a non-linear model with a variable Hill slope using GraphPad Prism 7.05. The NOC-5 concentrations required to induce 10 and 20% of mortality were calculated using R and the drc package (Christian Ritz and Jens C. Strebig). For efficacy assays, bacterial solutions were prepared as described above. Bacterial survival rate was quantified by plating on LB agar after 4 h at 37 °C and normalizing the bacterial load in NM102-treated samples to that of the control samples. The toxicity of NM102 in the absence of NO was assessed by exposing the bacterial suspension to NM102 at concentrations ranging from 0 to 80 µM of NM102 as described above but without NOC-5. The NM102 concentration required to induce 50% of mortality (IC50) was calculated using R and the drc package. NM102 was formulated in the presence of poly(d-L lactide-co-glycolide) (PLGA) according to a previously described method with some modifications59. It consists of an ultrasonic emulsification process, followed by dialysis to remove the organic solvent. First, 120 mg of PLGA and 28 mg of NM102 were dissolved in 4 mL of DMSO and then the organic phase was injected through a 23 G-needle into 50 mL of a 0.5% w/v aqueous solution of PVA under sonication in ice (5 min, amplitude 20%, cycle of sonication: 10 s ON and 10 s OFF) using a tip sonicator (Vibra-cellTM–75041, Fisher Bioblock Scientific, Belgium). The organic phase was then removed by dialysis (24 h, room temperature) using a 100 kDa MW cut-off cellulose ester membrane (Thermo Fisher Scientific, France). Intensity-weighted mean diameters (Dz) and particle size distributions were measured at 25 °C by dynamic light scattering using a Nano ZS instrument (173° scattering angle, Malvern, France) after 1/20 dilution in Milli-Q water (Millipore, France). All measurements were performed in triplicate. For storage, 20-mL flat-bottom glass vials were filled with 2 mL of nanoformulation supplemented with 5% w/v of trehalose as a cryoprotectant. Samples were frozen at −20 °C for 12 h and then freeze-dried (24 h) using an Alpha 2–4 LD plus freeze-drier (Martin Christ, Germany) (condenser temperature <70 °C, vacuum <0.1 mbar). Freeze-dried samples were capped, weighted, and stored at 4 °C. The drug content was determined by direct quantification of NM102 in freeze-dried NM102 nanoformulation samples (without cryoprotectant). Briefly, one sample was dissolved in DMSO (1 mL) by vortexing. The amount of NM102 was then quantified by UV spectroscopy at 282 nm (PerkinElmer UV/VIS spectrophotometer, Germany). The drug concentration was calculated using an NM102 calibration curve in DMSO (correlation coefficient >0.999), as follows: drug content (mg/mL) = weight of drug/volume of nanoformulation dispersion. Fourth instar silkworm larvae Bombyx eri were purchased from « L'office pour les insectes et leur environnement » (OPIE), Guyancourt, France. Larvae were reared with Ligustrum vulgare or Ligustrum japonicum until they reached a weight of [0,6 –0,9] g. Prior to assays, Bombyx eri larvae were kept under starvation overnight and randomly distributed in groups of 30. P. aeruginosa wt and ∆mfd strains were grown in LB medium at 37 °C under agitation at 180 rpm until late exponential growth phase. Bacterial cultures were then diluted in Phosphate Saline Buffer (PBS). The last dilution to obtain the desired concentration was done in PBS with 10% DMSO with or without NM102. Bacteria were plated right before infection to determine the quantity injected. Larvae were injected with 20 µL of bacterial suspension (3 × 103 CFU/injection in 10% DMSO) containing NM102 (2.78 µg/larvae) or 10% DMSO in the hemolymph via the last pro-leg. At 4 h and 8 h post first injection, larvae were injected with 50 µL of: (i) PBS, 10% DMSO or (ii) PBS, 10% DMSO with NM102 (6.97 µg/larvae) via the last pro-leg on the other side of the larvae. All injections were performed using a 1 mL hypodermic syringe with a 0.5 × 25 mm needle and an automated syringe pump (KD Scientific KDS 100). 24 h post infection, insects were placed in 2 mL sterile tubes containing 500 µL of PBS and 1.4 mm ceramic lysing beads (MP Biomedicals, Solon, OH, USA) and crushed using a Fast Prep instrument (MP Biomedicals, Solon, OH, USA). The bacterial content in each tube was assessed by serial dilutions on LB plates and CFU numeration. The log CFU was calculated per gram of larvae. For all mice experiments, nine-week-old, specific-pathogen-free, female, strain C57BL, substrain 6J mice were purchased from Janvier Labs (Le Genest-Saint-Isle, France). They were housed with a maximum of five animals per cage and had ad libidum access to food and water. The experiments were done with 12 h dark/light cycle at ambient temperature and controlled humidity. Prior to infection, mice were mildly anaesthetized by intraperitoneal (i.p.) Bacterial strains were grown to late exponential phase and diluted in PBS. bacterial administration was performed through slow instillation of 20 µL of bacterial suspension (1 × 107 CFU) with a micropipette (10 µL in each naris). Inoculated suspensions were plated on LB agar for CFU determination prior to injection. Empty and NM102-containing formulations were administrated in mice either: (i) via the i.p. route with 200 µL of suspension containing 1.5 or 6 mg/kg (1.29 and 5.16 mM, respectively) of drug or (ii) via the i.n. route with 20 µl of a mixture containing bacterial suspension (1 × 107 CFU) and NM102 nanoformulation (1.5 or 6 mg/kg) or empty nanoformulation. For combination therapy studies, NM102 nanoformulation and MPN were inoculated via the i.n. Mice were sacrificed by cervical dislocation after 24 h and the left lung was placed in 2 mL sterile tubes containing 600 μL of PBS and 1.4 mm ceramic lysing beads (MP Biomedicals, Solon, OH, USA). Organs were homogenized using a FastPrep instrument (MP Biomedicals, Solon, OH, USA), and aliquots from each tube were serially diluted on LB plates for CFU enumeration. The log CFU was calculated per gram of organ. For the toxicity study, mice were treated (i.p., 200 L/injection) with NM102 (6 mg/kg) or DMSO as control, and mice weight was monitored for 7 days. HeLa (human cervical epithelial cell line, ATCC CCL-2) and Vero (monkey kidney epithelial cell line, ATCC CCL-81)) cells were cultured in Dulbecco's Modified Eagles Medium (DMEM), supplemented with 10% heat-inactivated fetal bovine serum and 1 U/mL penicillin, and 1 µg/mL streptomycin at 37 °C and 5% CO2. Once reaching a confluency of 70%, cells were washed with Dulbeccos Phosphate Buffered Saline (DPBS) and detached by treatment with 0.05% Trypsin-EDTA. Total cell number was determined by using a Neubauer Cell chamber and cells were seeded at a concentration of 4 × 104 cells/well in a flat-bottom Tissue Culture 96-well plate. Cells were then incubated at 37 °C and 5% CO2 overnight before starting the cytotoxicity assay. Two hours before the assay, medium in each well was replaced by an antibiotic-free DMEM medium. Cells were washed and treated with the respective chemical (MPN at 10 and 100 µM; NM102 at 10 and 300 µM; NM102 nanoformulation at 1.29 and 5.16 mM) or their respective control (10% DMSO or empty nanoformulation) in a total volume of 100 µL in Hanks' Balanced Salt Solution (HBSS) medium (ThermoFischer). After 1 h of incubation, 20 µL of CellTiter96®AQueous was added to each well and cells were incubated again for 1 h. Absorbance was measured at 490 nm after 5 s of shaking to determine cytotoxicity. Treated cells were compared and normalized to the untreated control. Mice were treated (i.p., 200 µL/injection) with DMSO or NM102 (6 mg/kg). Feces were collected (day 1 and day 7) and the quantity of bacteria was assessed by qPCR as previously described60. In addition, microbiota diversity was assessed by 16S rRNA sequencing. Mice were treated (i.p., 200 µL/injection) with DMSO or NM102 (6 mg/kg) after i.n. Microbial DNA was extracted using the Dneasy PowerSoil Pro Kit following the provider's instructions (Qiagen, Hilden, Germany). DNA samples were sent to the NGS Competence Center Tübingen (Tübingen, Germany) for quantification, library construction, and sequencing. DNA concentration was quantified using a Qubit fluorometer (Thermo Fischer, Waltham, MA). The V4 hypervariable region of the 16S rRNA gene was amplified using primers F515 (5′-GTGCCAGCMGCCGCGGTAA-3′) and R806 (5′-GGACTACHVGGGTWTCTAAT-3′)61 and sequenced with the Illumina MiSeq sequencing platform with v2 chemistry. To process 16S rRNA amplicon sequences, we used the dada2 v.1.21.0 package of R (v.4.2.0)62 following its standard operating procedure as available at https://benjjneb.github.io/dada2/bigdata.html. The filtered forward and reverse reads were dereplicated separately and used for inference of the amplicon sequence variants (ASVs) with default parameters, after which they were merged in a per-sample basis. We next filtered the merged reads to retain only those with a length between 250 and 256 bp and carried out chimera removal. We then performed the taxonomic assignment on the final set of ASVs using a curated dada2-formated database based on the genome taxonomy database (GTDB) release R06-RS20263. A maximum-likelihood tree of the ASVs was obtained using the R packages DECIPHER v.2.24.0 and phangorn v.2.8.164 after performing a multiple sequence alignment with each of the representative sequences. We used the PERFect v.1.10 package of R65 to identify spurious ASVs, defined as those that made insignificant contributions to the total covariance of the microbiome dataset. For alpha- and beta-diversity analyses, samples were rarefied to 21300 reads/sample to account for differences in library size. To compare alpha-diversity between samples, we used the Shannon index as implemented in the vegan v.2.6-2 package of R. Global differences in Shannon index between treatments were assessed using the Kruskal-Wallis test and pairwise contrasts between treatments were performed using the Wilcoxon signed-rank test. We assessed differences in beta-diversity estimates using the adonis function (analysis of variance using distance matrices) of the permutational multivariate analysis of variance (PERMANOVA) on the distance matrices with 9999 permutations, as implemented in the vegan v.2.6-2 package of R. For pairwise PERMANOVA tests, we used the pairwiseAdonis v.0.4 package of R (available at https://github.com/pmartinezarbizu/pairwiseAdonis). P values were adjusted for multiple comparisons using the Benjamini-Hochberg method. E. coli Keio wt, Δmfd and complemented ∆mfd/mfd strains were grown onto LB agar O/N at 37 °C. Each culture was then divided in two with one culture exposed to 100 µM NM102 and the other to 2% final DMSO as control. Bacteria were incubated until the culture reached OD 600 nm of 1.2, which corresponds to the early stationary phase. Bacteria were plated onto LB agar plates with 50 µg/mL rifampicin and incubated for 24 h at 37 °C. The total number of bacteria was determined following serial dilution and plating onto LB agar plate. The test was performed 12 times with independent cultures. Lea-Coulson comparisons of rate analysis were performed on Web Salvador to determine significance between conditions, using an efficiency parameter of ε = 0,5 (∗p < 0.05; ∗∗p < 0.01; ****p < 0,001). A serial passage broth microdilution protocol was performed based on previously described methods18 to identify the effect of NM102 on the evolution of antibiotic resistance. Briefly, the culture of E. coli Keio WT started from a single colony and was diluted to OD 600 nm = 0.005. The bacterial suspension was subsequently transferred into a 96-well microplate (180 μL/well) and incubated with NM102, DMSO 2%, or left untreated. Treated and untreated bacteria were grown in LB with a gradient of concentrations of the indicated antibiotic to select for resistance for 18–24 h at 37 °C under stationary conditions. The OD 600 nm was measured and MICs were determined. Serial passage was performed using cells growing (defined by an OD 600 nm at a minimum of 50% of the untreated condition) in the highest concentration of antibiotic. Antimicrobial concentrations were adjusted during the process to compensate for the rise in MIC values. This process was repeated for four or five passages. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The data generated in this study are provided in the Source Data file. The amplicon sequence data generated during this study have been deposited in the European Nucleotide Archive with accession PRJEB86897. The R notebooks with the code used for the analysis of amplicon sequencing data are available at https://github.com/Lisa-Maier-Lab/NM102_Project Source data are provided with this paper. & Wright, G. D. The past, present, and future of antibiotics. Collaborators, A. R. Global burden of bacterial antimicrobial resistance in 2019: a systematic analysis. Randall, L. B., Georgi, E., Genzel, G. H. & Schweizer, H. P. Finafloxacin overcomes Burkholderia pseudomallei efflux-mediated fluoroquinolone resistance. Imai, Y. et al. A new antibiotic selectively kills Gram-negative pathogens. & Spoering, A. L. A new antibiotic kills pathogens without detectable resistance. Martin, J. K. et al. A dual-mechanism antibiotik kills Gram-negative bacteria and avoids drug resistance. Deaconescu, A. M. Mfd - at the crossroads of bacterial DNA repair, transcriptional regulation and molecular evolvability. Pomerantz R. T., O'Donnell M. Direct restart of a replication fork stalled by a head-on RNA polymerase. Structural basis for bacterial transcription-coupled DNA repair. Million-Weaver, S. et al. An underlying mechanism for the increased mutagenesis of lagging-strand genes in Bacillus subtilis. Effects of the bacterial transcription-repair coupling factor during transcription of DNA containing non-bulky lesions. Brugger, C. et al. Molecular determinants for dsDNA translocation by the transcription-repair coupling and evolvability factor Mfd. The Helicobacter pylori Mfd protein is important for antibiotic resistance and DNA repair. Key role of Mfd in the development of fluoroquinolone resistance in Campylobacter jejuni. Ragheb, M. N. et al. Inhibiting the evolution of antibiotic resistance. Carvajal-Garcia, J. et al. A small molecule that inhibits the evolution of antibiotic resistance. Darrigo, C., Guillemet, E., Dervyn, R. & Ramarao, N. The bacterial Mfd protein prevents DNA damage induced by the host nitrogen immune response in a NER-independent but RecBC-dependent pathway. Porrini, C., Ramarao, N. & Tran, S. L. Dr. NO and Mr. Toxic—the versatile role of nitric oxide. Liu, S. et al. Pterostilbene restores carbapenem susceptibility in New Delhi metallo-beta-lactamase-producing isolates by inhibiting the activity of New Delhi metallo-beta-lactamases. Toxic effects of dimethyl sulfoxide on red blood cells, platelets, and vascular endothelial cells in vitro. In vivo evaluation of the biodistribution and safety of PLGA nanoparticles as drug delivery systems. Safety evaluation of poly(lactic-co-glycolic acid)/poly (lactic-acid) microspheres through intravitreal injection in rabbits. Rognan, D. The impact of in silico screening in the discovery of novel and safer drug candidates. & Thornton, J. M. Shape variation in protein binding pockets and their ligands. Akuta, T., Zaki, M. H., Yoshitake, J., Okamoto, T. & Akaike, T. Nitrative stress through formation of 8-nitroguanosine: insights into microbial pathogenesis. Yoshitake, J. et al. Nitric oxide as an endogenous mutagen for Sendai virus without antiviral activity. Zaki, M. H., Akuta, T. & Akaike, T. Nitric oxide-induced nitrative stress involved in microbial pathogenesis. Implication of a key region of six bacillus cereus genes involved in siroheme synthesis, nitrite reductase production and iron cluster repair in the bacterial response to nitric oxide stress. Laver, J. R. et al. Bacterial nitric oxide detoxification prevents host cell S-nitrosothiol formation: a novel mechanism of bacterial pathogenesis. Martin, H. A. et al. Mfd protects against oxidative stress in Bacillus subtilis independently of its canonical function in DNA repair. & Merrikh, H. Oxidative stress drives mutagenesis through transcription-coupled repair in bacteria. Increased toxin expression in a Clostridium difficile mfd mutant. & Ohshiro, T. Antibiotic resistance mutations induced in growing cells of Bacillus-related thermophiles. Merrikh, H. & Kohli, R. M. Targeting evolution to inhibit antibiotic resistance. Tibbitt, M. W., Dahlman, J. E. & Langer, R. Emerging frontiers in drug delivery. Hwang, D., Ramsey, J. D. & Kabanov, A. V. Polymeric micelles for the delivery of poorly soluble drugs: from nanoformulation to clinical approval. Lee M. K. Liposomes for enhanced bioavailability of water-insoluble drugs: in vivo evidence and recent approaches. Anselmo, A. C. & Mitragotri, S. Nanoparticles in the clinic: an update post COVID-19 vaccines. Maier, L. et al. Unravelling the collateral damage of antibiotics on gut bacteria. Structural analysis of DNA replication fork reversal by RecG. Marcou, G. & Rognan, D. Optimizing fragment and scaffold docking by use of molecular interaction fingerprints. Katoh, K., Misawa, K., Kuma, K. & Miyata, T. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. B., Martin, D. M., Clamp, M. & Barton, G. J. Jalview Version 2-a multiple sequence alignment editor and analysis workbench. Laskowski, R. A., Jablonska, J., Pravda, L., Varekova, R. S. & Thornton, J. M. PDBsum: structural summaries of PDB entries. & Blundell, T. L. Comparative protein modelling by satisfaction of spatial restraints. Adasme, M. F. et al. PLIP 2021: expanding the scope of the protein–ligand interaction profiler to DNA and RNA. Baba, T. et al. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Li, Q. et al. ERCC2 helicase domain mutations confer nucleotide excision repair deficiency and drive cisplatin sensitivity in muscle-invasive bladder cancer. Fan, L. & DuPrez, K. T. XPB: an unconventional SF2 DNA helicase. Kokic, G., Wagner, F. R., Chernev, A., Urlaub, H. & Cramer, P. Structural basis of human transcription-DNA repair coupling. Kanaan, J. et al. UPF1-like helicase grip on nucleic acids dictates processivity. & Freire, E. Characterization of protein-protein interactions by isothermal titration calorimetry. Van de Ven, H. et al. PLGA nanoparticles and nanosuspensions with amphotericin B: potent in vitro and in vivo alternatives to Fungizone and AmBisome. Mayeur, C. et al. Faecal D/L lactate ratio is a metabolic signature of microbiota imbalance in patients with short bowel syndrome. Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. Parks, D. H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. Schliep, K. P. phangorn: phylogenetic analysis in R. Bioinformatics 27, 592–593 (2011). Smirnova, E., Huzurbazar, S. & Jafari, F. PERFect: PERmutation filtering test for microbiome data. Associating microbiome composition with environmental covariates using generalized UniFrac distances. We thank Antoine Allier for his help with R, Andrea Villarino for her input on the in vitro assay, and Isabelle Barbosa for her input in the yUpf1 assay. We thank all members of the IERP animal facility (INRAE, Jouy en Josas) and Catherine Cailleau (Institut Galien, Orsay) for their technical assistance with animal studies. We thank Marion Leclerc for the qPCR experiments. We warmly thank Erwin Bohn and his team at the University of Tübingen for their help, advice, and biological material used to construct the P. aeruginosa ∆mfd mutant strain. We thank Cordula Gekeler for excellent technical assistance, Constance Porrini for her participation in designing the NO assay, Saoussen Oueslati for her help during the MIC tests, Anouk Espel and Louise Pelven for their participation in the mutation assays. We thank the NCCT of Tübingen for sequencing. We warmly thank Natalia Lisa for her help in the protein purification (IBR, CONICET) and the STAN CONICET facility (Argentina) for the ITC assays. We warmly thank Dr. Libera Lo Presti for proofreading the manuscript. This work was supported by research grants from: SATT PARIS-SACLAY (CM2016-0067, DR, ST, NR), Postdoctoral Research Fellowship, Campus France (Prestige−2016−2-0011, ST), Prematuration IDEX (CDE−2018-002324/IRE 2018-0022, N.R. ), DIM One health region Ile de France (R19104DD, T.N., N.R. ), DFG (CMFI Cluster of Excellence EXC 2124, L.M. ), the Mécénat des Mutuelles AXA (laureate 2022, N.R.) and l'Agence Nationale de la Recherche (ANR) project NO ESKAPE ANR−23-CE18-0028-01 (S.M., J.N., G.A., N.R. These authors contributed equally: Seav-Ly Tran, Lucie Lebreuilly, Delphine Cormontagne. Université Paris-Saclay, INRAE, Micalis Institute, Jouy-en-Josas, France Seav-Ly Tran, Lucie Lebreuilly, Delphine Cormontagne, Samantha Samson, Thu Ba Tô, Marie Stosskopf, Rozenn Dervyn & Nalini Ramarao Université Paris-Saclay, INRAE, MaIAGE, Jouy-en-Josas, France Thu Ba Tô, Simona Mura & Julien Nicolas Anne Grießhammer, Jacobo de la Cuesta-Zuluaga & Lisa Maier Cluster of Excellence ‘Controlling Microbes to Fight Infections', University of Tübingen, Tübingen, Germany Anne Grießhammer, Jacobo de la Cuesta-Zuluaga & Lisa Maier Team ReSIST, INSERM U1184, School of Medicine Université Paris-Saclay, LabEx LERMIT, Assistance Publique/Hôpitaux de Paris, French NRC for Carbapenemase-Producing Enterobacterales. Université de Strasbourg, CNRS, UMR 7200 LiT, Illkirch, France You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar : analyzed data, supervision, writing of the manuscript, provided materials and funding. declare the following competing interests: a patent application (EP3868376, 2021) has been published, describing the use of NM102 and its derivatives as antibiotics. The remaining authors declare no competing interests. Nature Communications thanks the anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Tran, SL., Lebreuilly, L., Cormontagne, D. et al. An anti-virulence drug targeting the evolvability protein Mfd protects against infections with antimicrobial resistant ESKAPE pathogens. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02170-w'>Pangenome analysis reveals structural variation associated with seed size and weight traits in peanut</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 09:52:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Peanut (Arachis hypogaea L.) is an important oilseed and food legume crop, with seed size and weight being critical traits for domestication and breeding. However, genomic rearrangements like structural variations (SVs) underlying seed size and weight remain unclear. Here we present a comprehensive pangenome analysis utilizing eight high-quality genomes (two diploid wild, two tetraploid wild and four tetraploid cultivated peanuts) and resequencing data of 269 accessions with diverse seed sizes. We identified 22,222 core or soft-core, 22,232 distributed and 5,643 private gene families. The frequency of SVs in subgenome A is higher than in subgenome B. We identified 1,335 domestication-related SVs and 190 SVs associated with seed size or weight. Notably, a 275-bp deletion in gene AhARF2-2 results in loss of interaction with AhIAA13 and TOPLESS, reducing the inhibitory effect on AhGRF5 and promoting seed expansion. This high-quality pangenome serves as a fundamental resource for the genetic enhancement of peanuts and other legume crops. Peanut (Arachis hypogaea L.) is an important legume crop for vegetable oil and protein1. Cultivated peanut is an allotetraploid species that originated from hybridization of two wild diploid progenitors, Arachis duranensis (AA) and Arachis ipaensis (BB)2,3. Pod size is a critical quantitative agronomic trait that impacts peanut yield. During the domestication process, peanuts were selectively bred from wild accessions with small pods to modern varieties characterized by larger pods. Despite limited exploration of causal genes in peanuts4,5,6,7,8,9,10,11, the molecular mechanisms regulating pod size remain largely unresolved. In addition, currently available genomes still contain numerous gaps and high-quality genomic resources for peanuts are lacking because of the complexities of assembling polyploid genomes2,12,13,14,15,16. To optimize the exploration of genomic variations, it is important to have access to high-quality genomic resources across various populations of peanuts. Different individual genomes of a crop species can vary greatly17,18,19. A pangenome, which amalgamates genomics data from diverse individuals or populations, offers an holistic perspective for a more profound understanding of the genome structural and functional diversity of a species or population18,19,20,21. The pangenome of a crop can be much larger than its reference genome19. Developing a pangenome can identify missing or unannotated genes in the current reference genome, aiding in the discover of genes linked to yield, disease resistance and adaptability, and thus enhancing crop improvement. In practice, the utilization of genetically diverse germplasm resources for varietal improvement is critical for enhancing peanut yield22. However, the use of germplasm resources to improve traits is still limited, and genomic variations affecting phenotypic diversity remain underexplored. Recently, resequencing of germplasm resources was imperative for the identification of genomic variations and fundamental to the acceleration of genomic research23,24. With the resequencing data of a large number of accessions, genome-wide association studies (GWAS) can be conducted to identify genomic variations associated with specific traits in natural populations. This approach has been extensively utilized in various gramineous and leguminous crops25,26,27,28,29,30,31,32,33,34. However, research of the mechanism and application after GWAS in peanuts remains limited23,24,35,36, especially for large-sized variations such as SVs. In this study, we generated a comprehensive pangenome for both wild and cultivated peanuts, encompassing two diploid wild species, two tetraploid wild species and four tetraploid cultivated species. We assessed the genome-wide diversity of 269 peanut accessions, including wild species, landraces and improved species, and conducted SV-GWAS to investigate agronomic traits related to yield. Our analysis of the peanut pangenome showed significant genomic variations from breeding and highlighted trait-related variations affecting seed size and weight, providing a valuable resource for future improvements. To explore genetic diversity and population structure in peanut, we collected ~8 TB of resequencing data from 269 accessions with various seed sizes. These accessions come mainly from four populations (32 diploid wild species, 8 tetraploid wild species, 155 tetraploid cultivated landrace, 67 tetraploid improved accessions and 7 unclassified accessions) that are geographically widespread (Fig. Two accessions were included from previous studies12,15, and 267 accessions were newly sequenced (Supplementary Table 1). After mapping reads and calling variants to the reference genome (Methods), we obtained 5,989,854 (AA: 5,048,032; BB: 941,822) high-quality SNPs (minor allele frequency >0.05 and a missing rate <0.5). The phylogenetic tree showed that peanut accessions have diverged from wild and cultivated accessions (Fig. During the domestication of cultivated accessions from wild accessions, seed size and weight were increased significantly (Fig. Principal component analysis (PCA) of SNPs in both subgenomes A and B showed clear separation of wild and cultivated accessions. A large genetic distance was found between the A subgenomes of AA wild diploids and AABB allotetraploids, whereas high genetic diversity existed in the B subgenome of landrace peanuts (Fig. 1d,e), perhaps due to the limited availability of BB diploid accessions. a, Different species (A. hypogaea, their tetraploid A. monticola ancestors, and two wild diploid ancestors A. duranensis and A. ipaensis) and populations (AA/BB, AABB wild, AABB landrace and AABB improved) of peanut, including 269 peanut accessions with different seed sizes, are widely distributed geographically. b, Relationships of peanut accessions are illustrated using a phylogenetic tree based on subsampling 100,000 SNPs. The labeled accessions were used for downstream high-quality genome assembly or pangenome construction. c, Seed size (seed weight and length) differs significantly across four peanut populations. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. P values were calculated by two-tailed Student's t-test. d,e, PCA based on SNPs from the A subgenome (d) and the B subgenome (e). f,g, Summary of nucleotide diversity (π) and population divergence (FST) in the A subgenome (f) and B subgenome (g) between peanut populations. h, Results of population structure analysis using ADMIXTURE clustering when K = 8, 9 and 10. The cluster with combination of AABB wild, landrace and improved accessions is highlighted in a dashed box. Clu, cluster; PC1, first principal component; PC2, second principal component. AA wild diploid accessions had higher genetic diversity (π; 8.57 × 10−4) than BB wild diploid accessions (5.69 × 10−5). However, the allotetraploid accessions had higher values of π in the B subgenome than in the A subgenome (AABB wild: 9.62 × 10−5 > 7.48 × 10−5; AABB landrace: 1.46 × 10−4 > 1.17 × 10−4; AABB improved: 1.26 × 10−4 > 1.06 × 10−4). Of the four populations, AABB landrace accessions had the highest value of π in the B subgenome, which was consistent with the conclusion from PCA (Fig. Population fixation statistics (FST) values between AA wild diploids and other allotetraploids were high (0.739–0.866), whereas FST values between BB wild diploids and other allotetraploids were low (0.153–0.201) (Fig. The findings indicate an asymmetrical domestication process between subgenomes A and B. ADMIXTURE population structure analysis revealed peanuts possess ancestry proportions that derived from a combination of AABB wild and landrace accessions, with an estimated cluster number K = 9 (Fig. 1h and Supplementary Fig. All tetraploid wild accessions (Amon_N249, A7, US10 and so on) and several landrace accessions (SA080, DL183, SA117 and so on) were clustered into one group and shared more than 35% proportions with their ancestry BB accessions (H104, US38, US37, US36), suggesting that cultivated accessions likely originated from wild allotetraploid accessions. We selected the genomes of eight representative peanut accessions that demonstrate a range of diverse pod sizes for the construction of a high-quality peanut pangenome: two previously published genomes and six newly published genomes, including one AA wild diploid, two AABB wild allotetraploids, one AABB landrace allotetraploid and two AABB improved allotetraploids (Fig. Cultivated peanuts (A. hypogaea (Ahy)) originated from tetraploid ancestors (Arachis monticola (Amon)) nearly 4,000 years ago, and these were derived from hybridization between two wild diploids (A. duranensis (Adu) and A. ipaensis (Aip)) approximately 11,000 years ago37 (Fig. The genomes of the six new sequenced accessions were assembled de novo based on the combination of Nanopore ultra-long, PacBio HiFi and Hi-C technologies, mainly using NextDenovo and HiC-Pro38,39 (Methods). These chromosome-level genome assemblies were more contiguous and complete than the previously published ones, with scaffold N50 values of 134–148 Mb, contig N50 values of 25–79 Mb, long terminal repeat (LTR) Assembly Index values of 21.27–29.37 and BUSCO values of 98.6–99.4% (Supplementary Tables 2 and 3), which indicated the high quality of these genomes. Peanut genomes had a repeat sequence content of nearly 85%, with repeats mainly including retroelements (64–67%) and DNA transposons (10–12%) (Supplementary Fig. In total 39,736–77,824 protein-coding genes were annotated (Supplementary Table 5). a, Comparison of pods from representative diploid, tetraploid and cultivated peanuts. b, Pod size measurements, namely pod length, pod weight, seed length and seed weight. Data are given as the mean ± s.d. of five biological replicates. c, Possible ancestry and evolution of cultivated peanuts. d, Gene family PAVs in representative accessions. e, Pan-core gene family curve in representatives. n = 100 (iterations of random sampling without replacement). Fitting of power law model of new gene family (n) and sample (N) with n = kN−a, where k is a constant, −a is the law's exponent. R2 is the Pseudo R-squared. f, Number and percentage of core (eight of eight), soft-core (seven of eight), distributed (two of eight to six of eight) and private (one of eight) gene members in representative accessions. g–i, Gene length (g), Ka/Ks values (h) of core, soft-core, distributed and enriched KEGG pathways, and private gene family members in the A and B subgenomes (i). k, PAV of genes with the domain PF00931 (NB-ARC domain) present in the A subgenome. The white number in the boxes is the hit count of protein–genome alignment, which is unlike the strict standard in gene annotation. l, Phylogenetic tree of genes in the family OG0025534 based on multiple sequence alignment. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. P values were calculated by two-tailed Student's t-test (b, g and h). ka, thousand years ago. We constructed a pangenome of peanut using protein-coding genes. The pangenome was composed of 50,097 gene orthogroups (family): 17,137 core, 5,085 soft-core, 22,232 distributed and 5,643 private. Clustering of the gene family by presence–absence variations (PAVs) showed that the newly de novo assembled genomes had significantly more gene families (~37,000) than the two public genomes (~30,000; Wilcoxon rank sum test (WRST) P < 0.001) (Fig. 2d and Supplementary Fig. RNA sequencing (RNA-seq) for five tissues showed that 1,086 genes from 1,021 novel families had strong expression evidence in at least one tissue (transcript coverage ≥95% and fragments per kilobase of transcript per million fragments mapped fragments (FPKM) ≥ 2; WRST P < 0.001) (Supplementary Fig. 3b,c) indicating that a large proportion of the novel genes were real. Of the peanut pangenome, nearly 26,493–37,829 gene families were present in each peanut genome (Supplementary Table 5). The numbers of novel gene families and samples followed a power law distribution (R2 = 0.987, k = 10,150, a = 1.2) (Fig. 2e), indicating that the constructed peanut pangenome was closed. All genes were located near the two telomeres of each chromosome, except for the private genes, which showed peaks on chr. Core and private genes in the A subgenome, regardless of the type of gene family, were longer than those in the B subgenome (WRST P < 0.001 (core) and P < 0.05 (private)) (Fig. The core and soft-core families had longer genes, transcripts, exons and coding sequences (CDSs) than the distributed and private families (WRST, gene, transcript, exon and CDS length, P < 0.001) (Supplementary Fig. The core and soft-core family genes in the B subgenome had higher Ka/Ks values than those in A subgenome (WRST, missense and synonymous rates, P < 0.001) (Fig. The genes of core and soft-core families had lower missense and synonymous rates than those of distributed and private families (WRST P < 0.001) (Supplementary Fig. 5e–g) and higher expression levels in all tissues investigated (WRST P < 0.001) (Supplementary Fig. The genes of core and soft-core families in both the A and B subgenomes were enriched in glycosphingolipid biosynthesis and signaling pathways regulating the pluripotency of stem cells, whereas the genes of private gene families were enriched in mannose-type O-glycan biosynthesis. The genes of soft-core families in the A subgenome were uniquely enriched in aflatoxin biosynthesis and the Notch signaling pathway (Fig. Core genes found in cultivated peanuts but absent from wild peanuts were enriched in photosynthesis and lipid metabolism, indicating that they underwent convergent evolution during domestication (Supplementary Fig. We also investigated members from the distributed and private genes families of accession ZP06, which had the largest pod size (Methods). These genes were enriched in Pfam terms such as NB-ARC (nucleotide-binding adaptor shared by Apaf-1, R proteins, and Ced-4) domain, Rx (Potato Virus X Resistance Protein) N-terminal domain, F-box domains and hydroxymethylglutaryl-coenzyme A synthase N-terminal domain (Fig. For example, most members of the distributed and private gene families with domain PF00931 (NB-ARC domain) were present in the A subgenome (Fig. However, a member of the gene family OG0025534 in the A subgenome of ZP06 (SL03G07910) was unique among the gene families of improved peanuts (Fig. We noted that gain of a core gene family OG0003529 in NDH108 and ZP06 occurred at chr. 3 of the A subgenome during hybridization and polyploidization, which mainly includes the plant growth regulator indole-3-acetic acid (IAA) amino acid hydrolase ILR1-like 4 or 6 genes (Supplementary Fig. A graph-based peanut pangenome was constructed utilizing a combination of assembly-based and read-based SV detection methods, exemplified by the NB-ARC gene that contains both SVs and small variants (Fig. Initially, we conducted a synteny analysis across multiple genomes, which revealed significant collinearity in the chromosomes of both the A and B subgenomes (Supplementary Fig. From eight peanut genome assemblies, we identified a total of 86,308 high-confidence SVs, with insertions and deletions being the predominant types (Fig. 3b and Supplementary Fig. We performed genotyping on tetraploid accessions using resequencing data, and validated ten SVs in genes by polymerase chain reaction analysis (PCR; Supplementary Fig. Insertions were positively correlated with deletions, duplications, inversions and SNPs or insertion–deletions (indels) (Supplementary Fig. There were more SVs in subgenome A than in subgenome B, and these were not evenly distributed across each chromosome, mainly spreading in chr. 8 of A subgenome (Fig. 3c and Supplementary Fig. The size of SVs in the A and B subgenomes varied from 50 to 100,000 bp. The size of insertions was lower in the A subgenome than in the B subgenome, while the sizes of deletions and duplications were higher in A subgenome (Fig. Of the different SVs, 10–16% overlapped with genes and 15–30% overlapped with the upstream (promoter) or downstream 3,000 bp of genes (Supplementary Fig. 12a); 40–80% of SVs overlapped with repeat elements, particularly LTRs (Supplementary Fig. The sizes of insertions in promoter and exon regions were extremely variable and were in the three ranges, 50–500 bp, 500–5,000 bp and 5,000+ bp, while the sizes of deletions in promoter and exon regions were not similar (Fig. 3e and Supplementary Fig. Up to 40% of insertions and deletions in the promoter and exon regions were LTR and DNA transposons (Fig. 3f and Supplementary Fig. Genes with insertions in exons were enriched in glyoxylate and dicarboxylate metabolism and the PPRP (Peroxisome proliferator-activated receptors) signaling pathway, while the promoter insertions were enriched in beta-alanine metabolism, and valine, leucine and isoleucine degradation (Supplementary Fig. SVs in promoter and exon rather than downstream regions could regulate gene expression (Fisher's exact test) (Fig. 3g and Supplementary Fig. a, Plot of a nucleotide-binding oligomerization domain-like receptor (NLR) gene, ROQ1, with both large SVs (red) and small variants (SNPs and indels) (blue). b, SVs in the A and B subgenomes of representative accessions. c, Distribution of SVs (the circles from outer to inner are nonredundant insertions (INS), deletions (DEL), duplications (DUP), inversions (INV) and translocations) in chromosomes. d, Sizes of deletions, duplications, insertions and inversions in the A and B subgenomes. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. e, Sizes of insertions in promoter and exon regions. f, Repeat content of insertions in promoter and exon regions. g, Expression of single-copy genes with SVs in promoter and exon regions. P values were calculated by two-tailed Student's t-test. ASV, SV in subgenome A; BND, break ends; BSV, SV in subgenome B; DTA, hAT. DTC, CACTA; DTC, CACTA; DTH, PIF-Harbinger; DTM, Mutator; DTT, Tc1-Mariner; FKPM, Fragments per kilobase of transcript per million mapped fragments; MITE, Miniature inverted repeat transposable elements; NS, no significant difference; U3k, Upstream 3,000 bp promoter. Our analysis particularly concentrated on accession ZP06 with the largest pod size. In this accession, we observed 2,130 insertions and 2,160 deletions in the A subgenome, and 1,401 insertions and 1,352 deletions in the B subgenome (Fig. In addition, fewer SVs were found to overlap with CDS, promoter regions and downstream regions in both subgenomes from Amon to ZP06 than from diploids to Amon (Amon to ZP06, 36–338; K30076 to ZP06, 40–650; Adu to ZP06, 269–6,185) (Supplementary Fig. There were more deletions in the B subgenome than in the A subgenome from Amon to ZP06 (SVs overlapping CDS, exon, promoter, gene, downstream: 55, 59, 140, 167 and 231 (A); 77, 86, 223, 186 and 338 (B)) (Supplementary Fig. These SV-related genes in the A subgenome were enriched in Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways such as arachidonic acid metabolism and metabolism of xenobiotics by cytochrome P450, whereas those in the B subgenome were enriched in KEGG pathways such as stilbenoid, diarylheptanoid and gingerol biosynthesis, and inositol phosphate metabolism (Fig. SVs had the potential to affect gene structure or gene expression directly, including some important genes related to yield and disease resistance. For example, NDH16G00520 (Snakin-2) expression was higher in the flowers and fruits of ZP06, in which there was a 214-bp DNA/DTM (DNA transposon, Mutator) insertion in the promoter, than in other accessions without the insertion (Fig. In the exon of NDH14G21970 (glutamine-dependent NAD(+) synthetase), the gene with a 7.9-kb deletion had strong evidence of supported whole-genome sequencing (WGS) of ZP06 (Fig. A 1.2-kb insertion of a DNA/DTA (DNA transposon, hAT) transposon in an exon of NDH05G28200 (glycosyl transferase) divided it into two genes (Fig. A 1.7-kb deletion in ZP06 (encompassing exons of NDH18G35100 and NDH18G35110) resulted in two truncated genes (Fig. These results indicate that SVs have varied effects on gene functions. a, Number of insertions and deletions from diploids to tetraploid in the A and B subgenomes. b, Enriched KEGG pathways for genes with SVs in the A and B subgenomes of ZP06. c, An allele of NDH16G00520 (Snakin-2) with a 214-bp insertion has higher expression levels in flowers and fruits than the allele without the insertion. d, NDH14G21970 (glutamine-dependent NAD(+) synthetase) with a 7.9-kb deletion has strong evidence of supported WGS. e, A 1.2-kb insertion in NDH05G28200 (glycosyl transferase) divides it into two genes. f, A 1.7-kb deletion in NDH18G35100 (enhanced disease susceptibility 1) and NDH18G35110 (γ-aminobutyric acid, GABA permease) generated two truncated genes with different protein sequences. To better understand how genomic variations affect the function of genes during domestication, we further investigated SVs under selection between wild and landrace accessions. Genome-wide analysis of selective sweeps using FST, the nucleotide diversity ratio (πW/πL or πL/πI) and XP-CLR (the cross population composite likelihood ratio test) showed the unbalanced selection of chromosomes during the evolution of landrace accessions from wild accessions (AmToAhL) and improved accessions from landraces (AhLToAhI) (Fig. 5a,b and Supplementary Fig. 20 (5.7 Mb) had more selection during the evolution of landrace accessions from wild accessions, whereas chr. 19 (31.4 Mb) had more regions under selection during the development of improved accessions from landraces (Supplementary Fig. Regions more than twice the size (AhLToAhI: 29 Mb (A) and 67 Mb (B)) were under selection in the B subgenome than in the A subgenome during the development of improved accessions from landraces (Fig. a,b, Major genomic regions with evidence of selective sweeps during the evolution of landrace accessions from wild accessions (a) and during the evolution of improved accessions from landrace accessions (b). FST, nucleotide diversity ratio (πW/πL or πL/πI) and XP-CLR tests were used for selection analysis. Regions with evidence of selection (top 5% for at least two metrics) are highlighted. Different SVs from two populations were used for searches of SV-related genes. c, Genomic length, genes, SV-related genes and SV exon-related genes under selection in the A (sub-A) and B subgenomes (B-sub). d, Pfam enrichment analysis of genes and SV-associated genes with evidence of selection. e, Repeat overlap with SVs of selection in the A and B subgenomes. We identified 1,335 domestication-related SVs overlapping 329 genes (Supplementary Tables 7–9). Significantly, 108 SV-related genes in the A subgenome were subject to selection during the evolution of landraces from wild accessions, which were enriched in the salt stress response, antifungal and cullin family, whereas 114 genes in the B subgenome were enriched in bromodomain, leucine-rich repeat and sucrose synthase. During the evolution of improved accessions from landraces, 59 SV-related genes under selection in the A subgenome exhibited enrichment in the fasciclin domain and aminotransferase class-V, while 50 genes in the B subgenome were enriched in the legume lectin domain and pathogenesis-related protein bet v1 family (Fig. Some 41–55% of domestication-related SVs overlap with LTR or Gypsy elements and 35–55% domestication-related SVs overlapped with simple repeats. Of domestication-related SVs from the A and B subgenomes, the percentage of SVs with simple repeats (35% ≪ 52.8–54.7%), Helitron (8% ≪ 15–20%) and low complexity (3% ≪ 7–8%) in the B subgenome were the lowest during the evolution of improved accessions from landrace accessions (Fig. Among the genes with domestication-related SVs, 19 were associated with traits such as fruit size or known disease resistance (Supplementary Tables 8 and 9). For example, a higher proportion of diploids and wild allotetraploids had a 629-bp deletion in CRK26 (a calcium-dependent protein kinase gene), which is found in a cluster of up to 19 CRK genes on chr. A 27.7-kb deletion in IRK (a leucine-rich repeat gene) was difficult to detect because of alignment deviation near tandem gene duplications, but assemblies can accurately report its position and predict its length (Fig. A tandem unit consisting of the pod size gene NTF6 (mitogen-activated protein kinase, brassinosteroid signaling and homeostasis and regulation of grain size and plant height) and the disease-resistance gene FBRL2 (negative transcriptional regulator of immune responsive genes, including PR1), had high copy number variants (Fig. 6c, left), which was unique in peanut and did not exist in rice, soybean and cotton genomes. In addition, higher expression levels of both NTF6 and FBRL2 were observed in accessions with larger pod sizes (Fig. 6c, middle), whereas lower expression levels of both were observed in roots after bacterial wilt infection (Fig. 6c, right), indicating strong consistency in the expression of these two genes. These results indicate that the pangenome provides a valuable resource for gene exploration and expansion associated with fruit size and disease resistance during peanut domestication. 9, and genotyping of the population for CRK26 genes with (red) or without a deletion (steel blue). b, A 27.7-kb deletion in an IRK gene (NDH10G19410) from an IRK gene cluster at chr. 10 from haplotype assembly and long-read alignment. c, Genome collinearity (left), expression level of seed (middle) from various accessions and expression level of root from accession NDH108 with bacterial wilt infection (right) in the tandem unit of pod size genes NTF6 (NDH13G04110/NDH13G04130/NDH13G04150, mediator of RNA polymerase ll transcription subunit 36a) and disease-resistant related genes FBRL2 (NDH13G04100/NDH13G04120/NDH13G04140, protein kinase superfamily protein) at chr. Three biological replicates. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. Cytokinins play crucial roles in plant development by regulating cell division. The mechanisms of cytokinin metabolism and signal transduction are well understood40,41. Zeatin and isopentenyladenine-type are considered the predominant active forms of cytokinins42,43, and cytokinin oxidase/dehydrogenase (CKX) catalyzes the irreversible degradation of cytokinins44,45. In this study, we identified 117 SVs or indels associated with seed weight (P < 0.001), and one significant SV for seed weight was identified on chr. 3 by SV-GWAS (P = 1.84 × 10−4) (Fig. Linkage disequilibrium (LD) block analyses revealed a block (chr. 3: 9.71–9.73 Mb) next to the significantly associated SV in NDH03G08990 (Fig. Gene annotation showed that NDH03G08990 (AhCKX6) encodes a CKX containing an FAD-binding domain and a CK-binding domain (Fig. This gene might be involved in regulating cell division, thereby affecting yield, as previously reported in wheat46 and chickpea47. Gene structure analysis revealed that AhCKX6 has two insertions (TTACAAATATTATTTATTACTGT and CAAATTTGTC) in the 3′-UTR of accessions with larger seeds (Fig. Genotype-based association analysis revealed that the 3′-UTR insertions were absent in all 61 wild species. By contrast, three-quarters of the landrace and improved varieties exhibited insertions in the 3′-UTR (Fig. Furthermore, association analysis revealed that these insertions were significantly correlated with seed length and 100-seed weight (Fig. a, Graph-based SV-GWAS using genotype (SVs and indels from Minigraph-Cactus) and phenotype (seed weight) data from resequencing accessions. Horizontal lines represent the significant threshold (P = 1 × 10−3, 1 × 10−5). b, LD heatmap and candidate gene AhCKX6. c, Conserved domains of AhCKX6. d, Gene structure of AhCKX6 and location of the insertion. e, Proportions of wild, landrace and improved peanut accessions with the two haplotypes of AhCKX6. f, Box plots for seed length and 100-seed weight according to the genotype of the SV in AhCKX6. The numbers of accessions with the HapI and HapII genotypes are 61 and 183, respectively. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. g, Evolution of CKX6 proteins in different plant lineages. Phylogenetic analysis of CKX proteins from various plant lineages using the alignment of 25 full-length CKX6 protein sequences. The ancestral CKX gene sequence evolved in the kingdom Plantae along with the diversification into monocot and eudicot plant species >150 Ma. h, Expression levels of AhCKX6 in different tissues of HapI and HapII. i, The expression level of AhCKX6 was higher in HapI than in HapII in developing seeds. j, Seeds of HapII accessions contain more cytokinins than those of HapI accessions. n = 3 biological replicates. k, Model showing the mechanism by which AhCKX6 regulates peanut seed size. Data in h, i and j are given as mean ± s.e.m. n = 3 biological replicates. P values were calculated by two-tailed Student's t-test (f, h, i and j). CK, cytokinin; cZRMP, cis-zeatin riboside monophosphate; DHZ7G, dihydrozeatin-7-glucoside; DHZR, dihydrozeatin ribonucleoside; DHZROG, dihydrozeatin-O-glucoside riboside; DZ, dihydrozeatin; tZOG, trans-zeatin-O-glucoside; tZRMP, 9-ribosyl-trans-zeatin 5′-monophosphate. CKX evolution was traced back only to the first primitive land plant, Physcomitrium patens48. The ancestral CKX gene evolved in the kingdom Plantae along with the diversification into monocot and eudicot plant species >150 million years ago (Ma)49. Following a whole-genome duplication event approximately 58 Ma, CKX genes appeared in legumes. They then emerged in the Arachis genus around 2.2 Ma (ref. A phylogenetic association study revealed that AhCKX genes exhibit a high degree of similarity to soybean CKX genes (Fig. The expression levels of AhCKX6 were observed to be higher in the roots, leaves and seeds of HapI compared with HapII (Fig. In addition, AhCKX6 expression levels were elevated in small-seed accessions relative to large-seed accessions at various stages of seed development (Fig. Conversely, large-seed accessions exhibited higher cytokinin content levels (Fig. In summary, the insertion in the 3′-UTR of the CKX6 gene results in reduced expression levels. This increase in cytokinin promotes early cell division, ultimately leading to the development of larger seeds (Fig. Seed size is a critical agronomic trait that is closely associated with grain yield. Here, we identified 73 SVs associated with seed size (P < 0.001) and a gene (NDH08G29450) exhibiting an exon SV was found encode auxin response factor 2 (ARF2). This was the most significant SV associated with seed length in SV-GWAS (P = 3.63 × 10−6) (Fig. 18 and Supplementary Table 14), which acts as a negative regulator of seed size and weight in Arabidopsis50. In the ZP06 variety, the AhARF2-2 gene contains a 275-bp deletion and a 7-bp insertion in the 12th exon, as well as premature stop codons in the 13th exon (HapII) (Fig. 8b and Supplementary Fig. 19), resulting in loss of the conserved Auxin/Indole-3-Acetic Acid domain (AUX/IAA, PF02309) (Fig. 8c and Supplementary Fig. This loss is significantly correlated with variations in seed length and 100-seed weight (Fig. Molecular markers have been developed based on this SV to differentiate between the HapI and HapII haplotypes (Supplementary Fig. The AhARF2-2 gene was highly expressed in the seed and its expression was substantially higher in HapII accessions than in HapI accessions (Fig. Furthermore, after exogenous application of 1-naphthalene acetic acid (NAA) at the seedling stage, HapI accessions showed a significant upregulated response compared with HapII accessions (Fig. Further analysis indicated that the mutation did not alter the subcellular localization of its corresponding protein (Supplementary Fig. a, Graph-based SV-GWAS using genotype (SVs from Assembly and Read) and phenotype (seed length) data from resequencing accessions. Horizontal lines represent the significance threshold (P = 1 × 10−3, 1 × 10−5). b, Gene structure of AhARF2-2. c, Protein structure of AhARF2-2. Red boxes represent the missing domain. d, Box plots for seed length and 100-seed weight according to the genotype of the SV in AhARF2-2. Center line, median; box lower and upper edges, 25% and 75% quartiles, respectively. e, Relative expression levels of AhARF2-2 in different tissues. f, Expression pattern of ARF2-2 after NAA treatment. g, Bimolecular fluorescence complementation (BiFC) assay of the interaction of AhARF2-2, AhIAA13 and AhTPL. h,i, Y2H of the interaction of AhARF2-2 with AhIAA13 (h) or AhTPL (i). BD-T7-Lam (pGBKT7-Lam)/AD-T7-T (pGADT7-T) was used as the negative control, and BD-T7-53 (pGBKT7-53)/AD-T7-T was used as the positive control. j, Y1H assay of AhARF2-2 and AhGRF5 promoter. k, Dual-luciferase assay in tobacco leaves expressing LUC. Renilla luciferase (REN) was used as the internal control to normalize values in each assay. l, Overexpression of AhARF2-2HapI/HapII in Arabidopsis thaliana. m, Relative expression levels of overexpression (OE) and wild-type (Col) Arabidopsis. n–p, Comparison of seed area (n), seed length (o) and seed width (p) between the OE and Col plants. Col n = 22, HapI-3 n = 9, HapII-3 n = 19. q, Model of AhARF2-2 determining peanut seed size: AhIAA13 binds AhARF2-2 via the AUX/IAA domain. Auxin releases AhARF2-2, which recruits TPL to suppress AhGRF5. Data are given as mean ± s.e.m. n = 3 biologically independent samples (e, f, k and m). P values were calculated by two-tailed Student's t-test (d, e, f, k, m, n, o and p). Bimolecular fluorescence complementation and yeast two-hybrid (Y2H) assays confirmed that the AhARF2 protein from HapI accessions can interact with AhIAA13 and inhibit AhTPL, whereas the protein from HapII accessions has lost this interaction capability (Fig. In peanut, transient overexpression of the AhARF2-2HapI and AhARF2-2HapII genes resulted in a significant decrease in AhGRF5 expression, with HapI causing more pronounced downregulation than HapII (Supplementary Fig. These findings suggest that AhARF2-2 may negatively regulate AhGRF5 expression in peanuts. Yeast one-hybrid (Y1H) assays and firefly luciferase (LUC) complementation assays demonstrated that both the HapI and HapII AhARF2-2 proteins can bind to the promoter of the growth-regulating factor 5 (AhGRF5) gene (Fig. Reverse transcription quantitative PCR (RT–qPCR) analysis revealed that AhGRF5 exhibited the highest expression levels in seeds, with significantly higher expression in HapII accessions than in HapI accessions (Supplementary Fig. Moreover, AhARF2-2 could inhibit AhGRF5 expression, with HapI exhibiting a stronger inhibitory effect than HapII (Fig. TOPLESS (TPL) acts as an inhibitory factor in multiple biological processes51. AhARF2 has the capability to recruit TPL, thereby further suppressing the expression of AhGRF5; thus, HapI exhibits stronger inhibitory capability than HapII (Supplementary Fig. Results from transgenic Arabidopsis showed that the seed area and seed length of HapII lines were significantly larger than Hap I transgenic lines (Fig. 8l–p and Supplementary Fig. In summary, we propose that AhARF2-2 interacts with AhIAA13 via the C-terminal AUX/IAA domain. In the presence of auxin, AhARF2-2 is released and subsequently recruits TPL to suppress the expression of downstream AhGRF5. However, in large-seed accessions with SV deletion, AhARF2-2 is unable to interact with AhIAA13 and TPL, which leads to the reduction of the suppression of AhGRF5, thereby promoting seed expansion (Fig. Peanut (Arachis hypogaea L.), recognized as one of the most important oilseed and food legume crops, exhibits the unique characteristic of ripening its seeds underground. Throughout the processes of domestication and cultivation, there has been a progressive increase in the size and weight of peanut pods and seeds1,12,49. However, our comprehension of the molecular mechanisms and evolutionary factors influencing peanut pod size and weight remains limited. In this study, we developed comprehensive peanut pangenomes by integrating data from eight high-quality peanut genomes and 269 resequenced accessions. This resulted in an extensive resource of genomic variations that contribute to key agronomic traits in peanuts. The newly constructed pangenomes and identified variations enhance our understanding of the genetic basis underlying traits such as seed size and weight, and will facilitate advancements in crop science and peanut breeding, thereby potentially improving global food security. Along with SNPs and epigenetic differences, SVs are newly emerging as important variation features contributing to the genetic and phenotypic diversity observed in and between species. Understanding the impact of SVs on plant phenotypic variation is crucial for breeders aiming to develop superior cultivars51. SV identification has long been challenging in terms of both a lack of accuracy and comprehensiveness when using short-read resequencing data. Nevertheless, the important role of SVs has been highlighted in recent crop studies51,52. Our analysis revealed that combining assembly-based and read-based methods using long-read sequencing data improves the identification of SVs in plant genomes, especially large SVs. Notably, a significant observation is that most insertion SVs are associated with LTR retrotransposons and DNA transposons. We identified 1,335 domestication-related SVs and 190 SVs associated with seed size or weight. Our study revealed that SVs could influence gene expression, functional dynamics and uneven domestication between the two subgenomes, ultimately affecting seed size and weight. Throughout the extensive process of domestication and evolution, differential selection pressures have been exerted on subgenomes A and B, resulting in distinct functional specializations, similar to those observed in cotton (Gossypium spp. )53 and rapeseed (Brassica napus)54. During the transition from landraces to improved cultivars, chr. 3, 16 and 19 experienced intense artificial selection. The genes in the B subgenome were enriched in legume lectin domain and pathogenesis-related protein, such as a tandem unit of two peanut-specific genes associated with pod size and disease resistance separately (Fig. In crop breeding, enhancing both disease resistance and yield presents major challenges, with trade-offs between these traits being a common phenomenon in crop production55. Our research suggested that resistance was compromised as yields increased during peanut domestication. SV-GWAS identified two significant regions potentially related to seed size and weight. Subsequent experiments confirmed that SVs in these genes influenced seed size and weight. Notably, an SV in the 3′-UTR of the AhCKX6 gene on chr. 3 was associated with seed weight. Another significant candidate gene, AhARF2-2, encodes an auxin response factor and negatively regulates seed size. However, our research still has the limitation on the sample size of tetraploid wild peanuts in population selection, owing to the specificity of Arachis monticola acting as an intermediate between cultured and wild peanut. In summary, we have developed an extensive dataset comprising high-quality reference genomes, pangenomes and significant genomic variations, such as SVs, elucidating the influence of SVs on critical traits, including seed size and weight in peanuts. This study furnishes a genetic resource for the identification of functional genes associated with yield and disease resistance, offering valuable tools for breeding and crop enhancement for peanut as well as other polyploid species. All 269 analyzed accessions (two accessions, Tifrunner and Shitouqi, were previously published and others were newly sequenced) were collected from major peanut-growing regions, including India, the United States and China. These accessions, which are currently one of the most complete and representative collections worldwide, were used to analyze the major haplotype of the candidate gene related to the seed traits. Plant DNA was extracted using the Plant DNA Extraction Kit (catalogue number SM262; Seven). Peanut seedlings were grown in pots in a growth chamber set to 28 °C, with a 16 h light and 8 h dark photoperiod and ~60% relative humidity. Experiments were conducted when the seedlings reached the four-leaf stage. All Arabidopsis thaliana wild-type and transgenic plants were from the Columbia (Col-0) ecotype background. Arabidopsis thaliana Col-0 plants and tobacco (Nicotiana benthamiana) plants were grown in pots in a growth chamber at 23 °C with a 16 h light and 8 h dark photoperiod and ~60% relative humidity. In this study, previously collected WGS and phenotypic data from 269 peanut accessions with various seed sizes were reanalyzed using an updated high-quality reference genome NDH108. Raw sequencing data was preprocessed using FASTP (v.0.23.2)56. Trimmed reads were aligned to the reference genome (Near T2T genome: NDH108) using BWA (v.0.7.17)57. Moreover, for wild diploid accessions, homologous chromosomes of subgenomes were retained as the reference genomes (NDH108-A or NDH108-B subgenome). SNPs and indels were called using bcftools (v.1.10.2)58 (parameter: bcftools mpileup --min-ireads 1 -Ou -f $ref ${sample}.bam | bcftools call --ploidy 1 -mv -Oz). The SNPs and indels were then filtered using bcftools (parameter: bcftools filter -g3 -G10 -e ‘QUAL<30 || DP<5 || MQ<20'). Variants from all accessions were merged into one file using bcftools (parameter: merge -0). To conduct the analysis for both diploid and tetraploid peanut genomes, SNPs with the A or B subgenome were filtered with a minor allele frequency of >0.05 and a missing rate of <0.5 using VCFtools (v.0.1.16)59 (parameter: --max-missing 0.5 --maf 0.05), respectively. These SNPs and indels were annotated using SnpEff (v.5.1)60. SNPs in variant call format (VCF) were converted to PHYLIP format using Vcf2phylip (v.2.8)61 for phylogenetic analysis. The phylogenetic relationship was inferred using the neighbor-joining method with PHYLIP (v.3.697)62. To estimate the genetic similarity in and among populations, we calculated genetic diversity (π or pi) in four population using VCFtools (v.0.1.16)59 (parameter: --window-pi 100000 --window-pi-step 20000), as well as the population fixation statistics FST using VCFtools (v.0.1.16)59 (parameter: --fst-window-size 100000 --fst-window-step 20000). LD pruning was performed using PLINK (v.1.90b6.24)63 (parameter: --indep-pairwise 50 10 0.2). Population structure was estimated using ADMIXTURE (v.1.3.0)64, which identified different numbers of clusters (K) according to cross-validation error. Considering both agronomic traits and genetic variability, we newly selected several representative accessions with smaller to larger pods for high-quality genome assembly. These include an accession of Ad (Arachis duranensis; Adu), two accessions of Am (Arachis monticola; Amon and H16-5) and three accessions of Ahy (Arachis hypogaea; mH8, NDH108 and ZP06). Raw Oxford Nanopore Technologies (ONT) long reads with a mean quality score >7 were retained at the pre-processing step. De novo genome assembly was performed with ONT Ultra-long reads or PacBio HiFi reads using NextDenovo (v.2.4.0)38. To improve the accuracy of the assembly, the contigs were refined with Racon (v.1.3.1)65 using long reads and Nextpolish (v.1.3.1)66 using Illumina short reads with default parameters. Quality controlling of Hi-C raw data was performed using HiC-Pro (v.2.8.1)39 as former research. First, low-quality sequences (quality scores <20), adapter sequences and sequences shorter than 30 bp were filtered out using FASTP (v.0.23.2)56, and the clean paired-end reads were then mapped to the draft assembled sequence using Bowtie 2 (v.2.3.2)67 (parameter: -end-to-end --very-sensitive -L 30) to get the unique mapped paired-end reads. Valid interaction paired reads were identified and retained by HiC-Pro from unique mapped paired-end reads for further analysis. Invalid read pairs, including dangling-end, self-cycle, re-ligation and dumped products were filtered by HiC-Pro. The scaffolds were further clustered, ordered and oriented scaffolds onto chromosomes by LACHESIS68 (parameters: CLUSTER_MIN_RE_SITES=100, CLUSTER_MAX_LINK_DENSITY=2.5, CLUSTER NONINFORMATIVE RATIO=1.4, ORDER MIN N RES IN TRUNK=60, ORDER MIN N RES IN SHREDS=60). Transpose elements (TEs) were annotated using EDTA (v.2.0.0)69 () pipeline (parameter: --sensitive 1 --anno 1), which incorporates several well-performed structure-based and homology-based programs, including LTRharvest (v.1.5.10)70 (in GenomeTools), LTR_FINDER (v.1.07)71, LTR_retriever (v.2.9.0)72, Generic Repeat Finder (v.1.0)73, TIR-learner (v.2.5)74, HelitronScanner (v.1.0)75, TEsorter (v.1.3.0)76, RepeatModeler (v.2.0.2)77 and RepeatMasker (v.4.1.1)78. The interspersed repeats and tandem repeats of genomes were annotated using RepeatMasker (v.4.1.1)78 with a TE library generated from EDTA (v.2.0.0)69 and TRF (v.4.09)79. Three independent approaches, ab initio prediction, homology search and reference guided transcriptome assembly, were used for gene prediction in a repeat-masked genome. In detail, GeMoMa (v.1.6.1)80 was used to align the homologous peptides from related species to the assembly and then obtain the gene structure information, which was homolog prediction. For transcript-based gene prediction, filtered RNA-seq reads were aligned to the reference genome using STAR (v.2.7.3a)81. The transcripts were then assembled using Stringtie (v.1.3.5)82 d and ORFs were predicted using PASA (v.2.3.3)83. For the de novo prediction, RNA-seq reads were de novo assembled using Stringtie and analyzed with PASA to produce a training set. Augustus (v.3.3.1)84 with default parameters was then utilized for ab initio gene prediction with the training set. Finally, EVidenceModeler (v.1.1.1)85 was used to produce an integrated gene set from which genes with TE were removed using the TransposonPSI package (http://transposonpsi.sourceforge.net/)86 and the miscoded genes were further filtered. UTRs and alternative splicing regions were determined using PASA (v.2.3.3)83 based on RNA-seq assemblies. Proteins of each accession were assessed using BUSCO (v.5.3.2)87 with database embryophyta_odb10 v2020-09-10. Domains of proteins were annotated using InterProScan (v.5.55-88.0)88. The Gene Ontology of proteins was predicted using PANNZER2 (v.15.12.2020)89. Enrichment analysis was preformed using ClusterProfiler (v.3.16.1)90. The longest protein sequences of each gene were selected as representatives. Short proteins with <50 amino acids were ignored. The orthogroups (or gene families) were found using Orthofinder (v.2.5.4)91 with the parameter ‘-S blast'. For each accession, orthogroups with at least one member gene were defined as being present. The orthogroups PAVs matrix was used for downstream analysis. The power model92 was chosen to fit the new gene family count (n) and sample count (N) as reported previously. Every new gene family count was the median of values from 100 iterations of random sampling without replacement. The curve was fitted using the ‘nls' function in R v.4.0.2. Pseudo-R2 was computed using the R package aomisc v.0.648. The orthogroups in all (eight of eight) accessions were considered as core families. Orthogroups present in more than 90% of samples but not all (seven of eight) accessions were considered as soft-core families, the same threshold used for ‘soft-core' in previous studies18,93. Orthogroups present in between two and six of eight accessions were considered as distributed families. The orthogroups present in only one (of eight) accession were considered as private families. Single-copy gene orthogroups with fewer than one member per accession were aligned in parallel using ParaAT (v.2.0)94. The gene pairs were aligned using MUSCLE (v.5.1)95 and nonsynonymous (Ka) and synonymous (Ks) substitution rates were calculated using KaKs_Calculator (v.3.0)96 with the model averaging method97. Synteny and collinearity analyses were performed at both the genome and gene levels. We used D-Geneies (v.1.4)98 with whole genomes' alignment from Minimap2 (v.2.22)99 and MCscan (in JCVI) v.1.2.7 (ref. 100) with the CDS of longest protein sequences of each gene alignment from Last (v.1418)101. SVs of each accession were called using assembly-based and read-based SV detection methods, generating three graph-based pangenome sets. (1) Assembly-based SVs, graph pangenome construction from variants (SVAss): SVs of each accession were detected with assembly-based methods Svim-asm (v.1.0.2)102 (parameter: haploid) and CuteSV (v.1.0.13)103 (parameter: -s 1 --genotype --report_readid -p -1 -mi 500 -md 500 –max_cluster_bias_INS 1000 --diff_ratio_merging_INS 0.9 --max_cluster_bias_DEL 1000 --diff_ratio_merging_DEL 0.5). Insertions (INS) and deletions (DEL) from the intersection of two tools were compared and selected using Truvari (v.3.4.0)104 (parameter: bench). Duplications (DUP), inversions (INV) and breakends (BND) were selected from the results of CuteSV (v.1.0.13)103. SVs with supporting sequence names (RNAMES in the INFO column) from the same chromosomes or unplaced contigs were retained. Moreover, for two wild diploid accessions (Adu and K30076), the SVs from homologous chromosomes (Adu, A subgenome; K30076, B subgenome) or unplaced contigs were retained. The nonredundant SVs set was merged using SURVIVOR (v.1.0.7)105 (parameter: merge 1000 1 1 -1 -1 50). We constructed graph-based pangenome from variants and reference sequences using vg v.1.53.0. (2) Assembly-based SVs and small variants, graph pangenome construction from assemblies (FASTA) (Minigraph-Cactus): base-level pangenome graphs were constructed of each chromosome using Minigraph-Cactus (v.2.7)106. The DNA sequences (nodes) in the sequence graph (GFA) of each chromosome were renamed (chromosome prefixes + raw node identification) and merged into a graph of all chromosomes. (3) Assembly-based + read-based SVs, graph pangenome construction from variants (SV Assembly and Read): we added a read-based SV detection method to avoid heterozygous and unassembled-region SVs omitted in the single haplotype of genome assembly. SVs of each accession were detected using the read-based method CuteSV (v.2.0.3)103 from ONT reads and PacBio HiFi reads (parameter: --max_cluster_bias_INS 100 --diff_ratio_merging_INS 0.3 --max_cluster_bias_DEL 100 --diff_ratio_merging_DEL 0.3 (ONT); --max_cluster_bias_INS 1000 --diff_ratio_merging_INS 0.9 --max_cluster_bias_DEL 1000 --diff_ratio_merging_DEL 0.5 (PacBio HiFi)). The SVs with the ‘PRECISE' and ‘PASS' tags were filtered. Next, these SVs were divided into two subsets according to the regions of assembly alignment: SVs in assembly-covered regions and SVs in assembly-uncovered regions. For subset 1, heterozygous insertions and deletions were compared and merged with assembly-based SVs using Truvari (v.3.4.0)104 (parameter: bench). For subset 2, insertions and deletions were added with assembly-based SVs. We constructed graph-based pangenome from variants and reference sequences using vg (v.1.53.0)107. SVs were annotated with gene regions (gene, exon, CDS and gene upstream and downstream 3,000 bp) and repeat regions (TE, low complexity and simple repeat) using Vcfanno (v.0.3.3)108. The alternative allele sequences of insertions were extracted and annotated for repeats using RepeatMasker (v.4.1.1)78 with a TE library generated from EDTA (v.2.0.0)69 and TRF (v.4.09)79. Five tissues (flower, fruit, stem, leaf and root) of accessions Adu, Amon, H16-5, mH8, NDH108 and ZP06 were selected for RNA-seq experiments. The raw paired-end reads were filtered and trimmed using FASTP (v.0.23.2)56. The trimmed reads were mapped to the reference genome (NDH108) using HISAT2 (v.2.1.0)109 (parameter: --dta) and sorted using Samtools (v.1.10)110. The coverage of each transcript of genes was calculated using Bedtools (v.2.29.2)111 (parameter: genomecov -bga -split -ibam). The expression of each transcript of genes and each gene was calculated using Stringtie (v.1.3.5)82. The FPKM was used to estimate the gene expressions. We used three metrics to identify selective region of domestication sweeps, including genetic diversity ratio, FST and XP-CLR112. We compared the genetic diversity ratio (πAA/πAABBw, πBB/πAABBw, πAABBw/πAABBc) in four groups in 100-kb windows sliding 20 kb using VCFtools (v.0.1.16)59 (parameter: --window-pi 100000 --window-pi-step 20000). Windows with the top 5% of values were identified as candidate domestication-sweep regions. To identify additional domestication effects, we calculated the population fixation statistics (FST) in 100-kb windows sliding 20 kb using VCFtools (v.0.1.16)59 (parameter: --fst-window-size 100000 --fst-window-step 20000). Windows with the top 5% FST value were regarded as highly differentiated regions. We also calculated XP-CLR112 to scan for domestication-sweep regions in 100-kb windows sliding 20 kb and identified windows with the top 5% XP-CLR values (parameter: --ld 0.95 --size 100000 --step 20000). Windows filtered with at least two metrics were considered as domestication-sweep regions. NDH108 was cultivated using the hydroponic method (14 h light and 10 h dark photoperiod). When peanut roots reached 5–6 cm, seedlings were inoculated with Ralstonia solanacearum (strain number 180731-1) provided by the Institute of Plant Protection, Henan Provincial Academy of Agricultural Sciences, China. We performed RNA-seq of roots in NDH108 with bacterial wilt infection, at three time points (0, 12 and 24 h) of three independent experiments in replicates. The analysis process of raw reads was the same as described in the section ‘Gene expression analysis'. For genotyping of resequencing accessions, we used an ensemble genotyper EVG (v.1.2.0)113 combining GraphTyper2 (v.2.7.7)114, GraphAligner (v.1.0.13)115, vg (v.1.53.0)107 and Pangenie (v.3.0.1)116 with the graph pangenome constructed from both genome assemblies and long reads to genotype variations of SVs and indels (≥10 bp) from short-read sequencing data in tetraploid accessions. For SV-GWAS, two phenotypes (seed weight and seed length) were surveyed and the insertions and deletions from the graph pangenome were selected. Genotype files were prepared using PLINK (v.1.90b6.24)63. GWAS was performed using EMMAX (v.beta-07Mar2010)117. Plots were generated using R (v.4.0.2) with the R packages ggplot2 (v.3.3.6), ggmap (v.4.0.0) (Plotting spatial data), GOplot (v.1.0.2), ggbreak (v.0.1.1) (axis breaks plots), gghalves (v.0.1.4) (half plots), qqman (v.0.1.8) (GWAS Manhattan plots), ggpubr (v.0.4.0) (significant level in box plots) Pheatmap (v.1.0.12) (heatmap; https://CRAN.R-project.org/package=pheatmap), iTOL (v.6.6)118 (phylogenetic tree), Bandage (v.0.8.1)119 (graph pangenome visualization) and Rcirco (v.1.2.2)120 (genomic circos plots). Plots of the genome browser including tracks of gene annotations, repeat annotations, SVs and alignments were generated using Samplot (v.1.3.0)121, JBrowse2 (v.2.3.2)122 and IGV (v.2.5.3)123. The LD heatmap was generated using LDBlockShow (v.1.40)124. The amino acid sequences were retrieved from the National Center for Biotechnology Information (https://www.ncbi.nlm.nih.gov/) and UniProt (https://www.uniprot.org/). The phylogenetic tree was constructed using MEGA 5.2 (ref. 125) (maximum-likelihood method), FastTree (v.2.1.11)126 (approximately maximum-likelihood method) and modified by iTOL (v.6.6)118. The multiple sequence alignment was performed using MUSCLE (v.5.1)95. The protein structure was predicted using AlphaFold (v.2.3)127 and displayed using PyMol (v.2.3.4) (http://www.pymol.org). The CDSs of AhARF2-2HapI and AhARF2-2HapII (NDH08G29450) were cloned into a pCAMBIA1300-GFP vector using the Hieff Clone Universal One Step Cloning Kit (catalogue number 10911ES20, Yeasen). The recombinant plasmids pCAMBIA1300-HapI/HapII-GFP (35S:HapI/HapII-GFP), marker vector and empty control vector were then transformed into Agrobacterium tumefaciens strain GV3101 (AC1001, WEIDI). Agrobacterium mediated transient expression assays in tobacco (Nicotiana tabacum) and peanut according to our previous work11. Fluorescence images of transiently infected tobacco leaf subepidermal cells were captured utilizing laser scanning confocal microscopy (catalogue number LSM710, Carl Zeiss). Two-week-old seedlings (mH8 and ZP06), cultivated on plates, were subjected to treatment through spraying with 1 μM NAA. Samples were collected at 0, 2, 4, 6 and 12 h following the treatment with NAA. Total RNA was extracted using FastPure Universal Plant Total RNA Isolation Kit (catalogue number RC411, Vazyme) and TransScript One-Step gDNA Removal and cDNA Synthesis SuperMix Kit (catalogue number AT311-02, TransGen Biotech). Gene expression levels were assayed using the PerfectStart Green qPCR SuperMix Kit (catalogue number AQ601, TransGen Biotech) in conjunction with the Bio-Rad CFX96 real-time PCR system. The 2−∆∆CT method was used to calculate the relative expression levels of each gene128. Ahactin7 (XM_025826875) and Atactin2 (U37281.2) were used as the internal control genes. Primers were designed using Primer 3 (v.4.1.0)129. Cytokinin measurements were performed by Wuhan Metware Biotechnology Co., Ltd, utilizing seeds from the Amon and ZP06 varieties, approximately 15 days following the peg's penetration into the soil. The AhGRF5 (Arahy.V829EQ) promoter was integrated into the pAbAi vector. The recombinant plasmid pAbAi-AhGRF5 was digested with the restriction enzyme BstBI (catalogue number R0519V, New England Biolabs) and transferred into the Y1H yeast strain and tested on SD/-Ura medium (catalogue number PM2271, Coolaber) with different concentrations of aureobasidin A. The CDS of AhARF2-2HapI and AhARF2-2HapII were cloned into the pGADT7 and transferred into the Y1H yeast strain, containing pAbAi–AhGRF5 plasmid, tested on SD/-Leu medium (catalogue number PM2201, Coolaber) with 200 ng ml−1 of aureobasidin A. The CDSs of AhARF2-2HapI and AhARF2-2HapII were cloned into the N-terminal half of yellow fluorescent protein (nYFP). The CDSs of AhIAA13 and AhTPL were cloned into the C-terminal half of YFP (cYFP) to generate HapI-nYFP, AhIAA13-cYFP, HapII-nYFP and AhTPL-cYFP vectors. The appropriate pairs of constructs (AhTPL-cYFP and HapI-nYFP; AhTPL-cYFP and HapII-nYFP; AhIAA13-cYFP and HapI-nYFP; AhIAA13-cYFP and HapII-nYFP) were infiltrated into N. benthamiana leaves via Agrobacterium-mediated transient infiltration and detected by laser scanning confocal microscopy (catalogue number LSM710, Carl Zeiss). MDB4JZ) and AhTPL (Arahy.UFRA39) CDSs were cloned into pGBKT7 vector (bait vector). These recombinant vectors were transformed into the Y2H yeast strain. The bait vectors pGBKT7-53 (BD-T7-53) and pGBKT7-Lam (BD-T7-Lam), serving as positive and negative controls respectively, were cotransformed with the prey vector pGADT7-T. Yeast cells and control cells, carrying recombinant plasmids, were grown on deficient medium: lacking tryptophan and leucine (SD/-Trp/-Leu; catalogue number PM2221, Coolaber) and leucine, tryptophan histidine and adenine (SD/-Leu/-Trp/-His/-Ade; catalogue number PM2221, Coolaber), respectively. The CDSs of AhTPL, AhARF2-2HapI and AhARF2-2HapII were cloned into pCAMBIA1300-GFP vector to construct the pCAMBIA1300-TPL-GFP (35S:TPL), 35S:HapI and 35S:HapII as effector plasmids. The AhGRF5 promoter (850 bp) was inserted into pGreenII 0800-LUC to construct the GRF5pro:LUC reporter plasmid. These vectors were individually transformed into GV3101 (harboring the pSoup plasmid). The pairs of constructs (35S:GFP and GRF5pro:LUC; 35S:HapI and GRF5pro:LUC; 35S:HapII and GRF5pro:LUC; 35S:HapI, 35S:TPL and GRF5pro:LUC; 35S:HapII, 35S:TPL and GRF5pro:LUC) were infiltrated into N. benthamiana leaves using the method reported previously130. LUC activity was detected using a low-light cooled CCD imaging apparatus (Tanon 5200). LUC or Renilla luciferase was assayed using the Double-Luciferase Reporter Assay Kit (catalogue number FR201, TransGen Biotech) with dual LUC assay reagents (Promega). Primers used for the LUC assay are listed in Supplementary Table 16. Agrobacterium tumefaciens GV3101 containing the recombinant vectors 35S:HapI-GFP or 35S:HapII-GFP was used to transform Arabidopsis (Col-0) using the floral dipping method131 Transgenic Arabidopsis plants were selected on MS medium supplemented with 30 mg l−1 hygromycin and their identity was confirmed through PCR. Subsequently, the T3 generation of transgenic plants was utilized for phenotypic analysis of the candidate gene. Seed sizes were photographed with an anatomical microscope and were tested using the Ween SC-G automated test analysis system (WSeen). Primers used for gene cloning are listed in Supplementary Table 16. The statistical analyses were mostly performed in R (v.4.0.2). For biochemical and molecular biology analysis, at least three individuals were mixed in each sample with three biological replicates. Standard deviations and P values were calculated using two-tailed Student's t-test and analysis of variance. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The eight genomic sequences for pangenome and all the 269 genomic sequence data for GWAS analysis have been deposited in the National Genomics Data Center (NGDC) database under BioProject numbers PRJCA029798 and PRJCA029800 (GSA numbers CRA019399 and CRA019260). The published transcriptomic datasets for candidate gene expression analysis can be downloaded from the NGDC under accession number PRJCA029802 (GSA number CRA019264). The transcriptomic datasets for gene expression analysis on accessions with bacterial wilt infection can be downloaded from the NGDC under accession number PRJCA030060 (GSA numbers CRA019399 and CRA020567). The result data has been mirrored deposited at https://cgm.sjtu.edu.cn/PeanutPan/index.html. Source data are provided with this paper. The codes for this study are available via Zenodo at https://doi.org/10.5281/zenodo.15003999 (ref. 132) and via GitHub at https://github.com/SJTUCGM/PeanutPan. Akram, N. A., Shafiq, F. & Ashraf, M. Peanut (Arachis hypogaea L.): a prospective legume crop to offer multiple health benefits under changing climate. Bertioli, D. J. et al. The genome sequences of Arachis duranensis and Arachis ipaensis, the diploid ancestors of cultivated peanut. Fávero, A. P., Simpson, C. E., Valls, J. M. & Velo, N. A. Study of evolution of cultivated peanut through crossability studies among Arachis ipaënsis, A. duranensis, and A. hypogaea. Alyr, M. H. et al. Fine-mapping of a wild genomic region involved in pod and seed size reduction on chromosome A07 in peanut (Arachis hypogaea L.). Liu, Y. et al. Multi-omics profiling identifies candidate genes controlling seed size in peanut. Guo, F. et al. Transcriptome analysis and gene expression profiling of the peanut small seed mutant identified genes involved in seed size control. Wu, Y. et al. Comparative transcriptomics analysis of developing peanut (Arachis hypogaea L.) pods reveals candidate genes affecting peanut seed size. Yang, L. et al. Global transcriptome and co-expression network analyses revealed hub genes controlling seed size/weight and/or oil content in peanut. Yang, H. et al. Fine mapping of qAHPS07 and functional studies of AhRUVBL2 controlling pod size in peanut (Arachis hypogaea L.). Gangurde, S. S. et al. Nested-association mapping (NAM)-based genetic dissection uncovers candidate genes for seed and pod weights in peanut (Arachis hypogaea). Zhao, K. et al. PSW1, an LRR receptor kinase, regulates pod size in peanut. Bertioli, D. J. et al. The genome sequence of segmental allotetraploid peanut Arachis hypogaea. Chen, X. et al. Sequencing of cultivated peanut, Arachis hypogaea, yields insights into genome evolution and oil improvement. Yin, D. et al. Genome of an allotetraploid wild peanut Arachis monticola: a de novo assembly. Zhuang, W. et al. The genome of cultivated peanut provides insight into legume karyotypes, polyploid evolution and crop domestication. Xue, H. et al. A near complete genome of Arachis monticola, an allotetraploid wild peanut. Wang, W. et al. Genomic variation in 3,010 diverse accessions of Asian cultivated rice. Liu, Y. et al. Pan-genome of wild and cultivated soybeans. Zhang, F. et al. Long-read sequencing of 111 rice genomes reveals significantly larger pan-genomes. Zekic, T., Holley, G. & Stoye, J. Pan-genome storage and analysis techniques. Shi, J., Tian, Z., Lai, J. Plant pan-genomics and its applications. Pandey, M. K. et al. Advances in Arachis genomics for peanut improvement. Lu, Q. et al. A genomic variation map provides insights into peanut diversity in China and associations with 28 agronomic traits. Zheng, Z. et al. Chloroplast and whole-genome sequencing shed light on the evolutionary history and phenotypic diversification of peanuts. Tian, G. et al. Allelic variation of TaWD40-4B.1 contributes to drought tolerance by modulating catalase activity in wheat. Tian, F. et al. Genome-wide association study of leaf architecture in the maize nested association mapping population. Li, H. et al. Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels. Huang, X. et al. Genome-wide association studies of 14 agronomic traits in rice landraces. Huang, X. et al. A map of rice genome variation reveals the origin of cultivated rice. Zhou, Z. et al. Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean. Fang, L. et al. Genomic analyses in cotton identify signatures of selection and loci associated with fiber quality and yield traits. Kang, L. et al. Genomic insights into the origin, domestication and diversification of Brassica juncea. Varshney, R. K. et al. Resequencing of 429 chickpea accessions from 45 countries provides insights into genome diversity, domestication and agronomic traits. Varshney, R. K. et al. A chickpea genetic variation map based on the sequencing of 3,366 genomes. Zhang, X. et al. Genome-wide association study of major agronomic traits related to domestication in peanut. Liu, Y. et al. Genomic insights into the genetic signatures of selection and seed trait loci in cultivated peanut. Yin, D. et al. Comparison of Arachis monticola with diploid and cultivated tetraploid genomes reveals asymmetric subgenome evolution and improvement of peanut. Hu, J. et al. NextDenovo: an efficient error correction and accurate assembly tool for noisy long reads. Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Sakakibara, H. CYTOKININS: activity, biosynthesis, and translocation. Werner, T. & Schmülling, T. Cytokinin action in plant development. Schmitz, R. Y. et al. Cytokinins: synthesis and biological activity of geometric and position isomers of Zeatin1. Einset, J. W. Conversion of N6-isopentenyladenine to zeatin by Actinidia tissues. Frébort, I. et al. Evolution of cytokinin biosynthesis and degradation. Zhang, L. et al. TaCKX6-D1, the ortholog of rice OsCKX2, is associated with grain weight in hexaploid wheat. Khandal, H. et al. Root-specific expression of chickpea cytokinin oxidase/dehydrogenase 6 leads to enhanced root growth, drought tolerance and yield without compromising nodulation. Paces, V., Werstiuk, E. & Hall, R. H. Conversion of N-(delta-isopentenyl) adenosine to adenosine by enzyme activity in tobacco tissue. von Schwartzenberg, K. et al. Cytokinins in the bryophyte Physcomitrella patens: analyses of activity, distribution, and cytokinin oxidase/dehydrogenase overexpression reveal the role of extracellular cytokinins. Khuman, A., Kumar, V. & Chaudhary, B. Evolutionary expansion and expression dynamics of cytokinin-catabolizing CKX gene family in the modern amphidiploid mustard (Brassica sp.). Schruff, M. C. et al. The AUXIN RESPONSE FACTOR 2 gene of Arabidopsis links auxin signalling, cell division, and the size of seeds and other organs. Yuan, Y., Bayer, P. E., Batley, J. & Edwards, D. Current status of structural variation studies in plants. Alonge, M. et al. Major impacts of widespread structural variation on gene expression and crop improvement in tomato. Wang, M. et al. Asymmetric subgenome selection and cis-regulatory divergence during cotton domestication. Hu, J. et al. Genomic selection and genetic architecture of agronomic traits during modern rapeseed breeding. Derbyshire, M. C. et al. The complex relationship between disease resistance and yield in crops. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Danecek, P. & McCarthy, S. A. BCFtools/csq: haplotype-aware variant consequences. Danecek, P. et al. The variant call format and VCFtools. Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Edgardo M. O. Vcf2phylip v2.0: convert a VCF matrix into several matrix formats for phylogenetic analysis. Felsenstein, J. PHYLIP—phylogeny inference package (Version 3.2). Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Alexander, D. H., Novembre, J. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Vaser, R. et al. Fast and accurate de novo genome assembly from long uncorrected reads. Hu, J. et al. NextPolish: a fast and efficient genome polishing tool for long-read assembly. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Burton, J. N. et al. Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions. Ou, S. et al. Benchmarking transposable element annotation methods for creation of a streamlined, comprehensive pipeline. Gremme, G., Steinbiss, S. & Kurtz, S. GenomeTools: a comprehensive software library for efficient processing of structured genome annotations. IEEE/ACM Transactions on Computational Biology and Bioinformatics Vol. & Wang, H. LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. Nucleic Acids Res. Ou, S. & Jiang, N. LTR_retriever: a highly accurate and sensitive program for identification of long terminal repeat retrotransposons. & Liang, C. Generic Repeat Finder: a high-sensitivity tool for genome-wide de novo repeat detection. Su, W., Gu, X. & Peterson, T. TIR-Learner, a new ensemble method for TIR transposable element annotation, provides evidence for abundant new transposable elements in the maize genome. Xiong, W. et al. HelitronScanner uncovers a large overlooked cache of Helitron transposons in many plant genomes. Zhang, R. G. et al. TEsorter: an accurate and fast method to classify LTR-retrotransposons in plant genomes. Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. Tarailo-Graovac, M. & Chen, N. Using RepeatMasker to identify repetitive elements in genomic sequences. Benson, G. Tandem repeats finder: a program to analyze DNA sequences. Nucleic Acids Res. Keilwagen, J. et al. Using intron position conservation for homology-based gene prediction. Nucleic Acids Res. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Kovaka, S. et al. Transcriptome assembly from long-read RNA-seq alignments with StringTie2. Campbell, M. A. et al. Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis. Stanke, M. et al. Using native and syntenically mapped cDNA alignments to improve de novo gene finding. Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments. Urasaki, N. et al. Draft genome sequence of bitter gourd (Momordica charantia), a vegetable and medicinal plant in tropical and subtropical regions. Manni, M. et al. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. Jones, P. et al. InterProScan 5: genome-scale protein function classification. Törönen, P., Medlar, A. & Holm, L. PANNZER2: a rapid functional annotation web server. Nucleic Acids Res. Yu, G., Wang, L. G., Han, Y. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. Tettelin, H., Riley, D., Cattuto, C. & Medini, D. Comparative genomics: the bacterial pan-genome. Li, X. et al. Large-scale gene expression alterations introduced by structural variation drive morphotype diversification in Brassica oleracea. Zhang, Z. et al. ParaAT: a parallel tool for constructing multiple protein-coding DNA alignments. Edgar, R. C. Muscle5: high-accuracy alignment ensembles enable unbiased assessments of sequence homology and phylogeny. Zhang, Z. KaKs_Calculator 3.0: calculating selective pressure on coding and non-coding sequences. Genomics Proteomics Bioinformatics 20, 536–540 (2022). Posada, D. Using MODELTEST and PAUP* to select a model of nucleotide substitution. Cabanettes, F. & Klopp, C. D-GENIES: dot plot large genomes in an interactive, efficient and simple way. Li, H. New strategies to improve minimap2 alignment accuracy. Tang, H. et al. Synteny and collinearity in plant genomes. Kiełbasa, S. M. et al. Adaptive seeds tame genomic sequence comparison. Heller, D. & Vingron, M. SVIM-asm: structural variant detection from haploid and diploid genome assemblies. Jiang, T. et al. Long-read-based human genomic structural variation detection with cuteSV. English, A. C. et al. Truvari: refined structural variant comparison preserves allelic diversity. Jeffares, D. C. et al. Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast. Hickey, G. et al. Pangenome graph construction from genome alignments with Minigraph-Cactus. Garrison, E. et al. Variation graph toolkit improves read mapping by representing genetic variation in the reference. Pedersen, B. S., Layer, R. M. & Quinlan, A. R. Vcfanno: fast, flexible annotation of genetic variants. Kim, D. et al. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Li, H. et al. The Sequence Alignment/Map format and SAMtools. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Chen, H., Patterson, N. & Reich, D. Population differentiation as a test for selective sweeps. A comprehensive benchmark of graph-based genetic variant genotyping algorithms on plant genomes for creating an accurate ensemble pipeline. Eggertsson, H. P. et al. GraphTyper2 enables population-scale genotyping of structural variation using pangenome graphs. Rautiainen, M. & Marschall, T. GraphAligner: rapid and versatile sequence-to-graph alignment. Ebler, J. et al. Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes. Kang, H. M. et al. Variance component model to account for sample structure in genome-wide association studies. & Bork, P. Interactive Tree of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. Wick, R. R. et al. Bandage: interactive visualization of de novo genome assemblies. Zhang, H., Meltzer, P. & Davis, S. RCircos: an R package for Circos 2D track plots. Belyeu, J. R. et al. Samplot: a platform for structural variant visual validation and automated filtering. Diesh, C. et al. JBrowse 2: a modular genome browser with views of synteny and structural variation. Robinson, J. T. et al. Integrative genomics viewer. Dong, S. S. et al. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. Tamura, K. et al. MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Pfafl, M. W. A new mathematical model for relative quantification in real-time RT–PCR. Nucleic Acids Res. Untergasser, A. et al. Primer3—new capabilities and interfaces. Nucleic Acids Res. Zhang, Y. et al. The circadian-controlled PIF8-BBX28 module regulates petal senescence in rose flowers by governing mitochondrial ROS homeostasis at night. & Bent, A. F. Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Xue, H. PeanutPAN: pan-genome for peanut genomics study (1.0.0). The computations in this paper were run on the π 2.0 cluster supported by the Center for High Performance Computing at Shanghai Jiao Tong University. This work was supported by grants from the Key Program of National Natural Science Foundation of China (NSFC)-Henan United Fund (grant number U22A20475), Key Scientific and Technological Project of Henan Province (grant numbers 221111110500, 222301420026 and HARS-22-05-G1), National Natural Science Foundation of China (grant number 32170643), Natural Science Foundation of Shanghai (grant numbers 20ZR1428200, and 22ZR1433600), Shanghai Key Program of Computational Biology (grant number 23JS1400800) and National key R&D program (grant number 2023YFF1001600). acknowledges the start-up grant from the Food Futures Institute of Murdoch University, Australia. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. These authors contributed equally: Kunkun Zhao, Hongzhang Xue, Guowei Li. College of Agronomy, Henan Agricultural University, Zhengzhou, China Kunkun Zhao, Yi Fan, Zenghui Cao, Kai Zhao, Lin Zhang, Ding Qiu, Rui Ren, Fangping Gong, Zhongfeng Li, Xingli Ma & Dongmei Yin School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China Hongzhang Xue, Xiaorui Dong, Huimin Lu & Chaochun Wei Institute of Crop Germplasm Resources, Shandong Academy of Agricultural Sciences, Ji'nan, China Guowei Li & Shubo Wan WA State Biotechnology Centre, Centre for Crop and Food Innovation, Food Futures Institute, Murdoch University, Murdoch, Western Australia, Australia Annapurna Chitikineni & Rajeev K. Varshney You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the study. performed data analysis. Kunkun Zhao, Y.F., Z.C., D.Q., L.Z., A.C. and Kai Zhao performed the experiments. and Kai Zhao prepared the samples and reagents. measured the agronomic traits. and Kunkun Zhao wrote the paper. All authors read and approved the final paper. Correspondence to Rajeev K. Varshney, Chaochun Wei or Dongmei Yin. The authors declare no competing interests. Nature Genetics thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Source data Supplementary Figs. 21–24 Statistical source data. 1c Statistical source data. 2b Statistical source data. 4c Statistical source data. 7e,f,h,I,j Statistical source data. 8d,e,f,k,m,n,o,p Statistical source data. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Zhao, K., Xue, H., Li, G. et al. Pangenome analysis reveals structural variation associated with seed size and weight traits in peanut. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02650-8'>A call for built-in biosecurity safeguards for generative AI tools</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 09:49:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. It also enhances biomedical imaging, personalized medicine and laboratory automation, which enables faster and more efficient scientific advancements. However, these breakthroughs have also raised biosecurity concerns, which has prompted policy and community discussions1,2,3,4. The power of generative AI lies in its ability to generalize from known data to the unknown. Deep generative models can predict novel biological molecules that might not resemble existing genome sequences or proteins. This capability introduces dual-use risks and serious biosecurity threats — such models could potentially bypass the established safety screening mechanisms used by nucleic acid synthesis providers5, which presently rely on database matching to identify sequences of concerns6. AI-driven tools could be misused to engineer pathogens, toxins or destabilizing biomolecules, and AI science agents could amplify risks by automating experimental designs7. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Wittmann, B. J. et al. Preprint at bioRxiv https://doi.org/10.1101/2024.12.02.626439 (2024). Fast Track Action Committee On Synthetic Nucleic Acid Procurement Screening. (White House Office of Science and Technology Policy, 2024). Neural Information Processing Systems (eds Oh, A.) Certain tools and software are identified in this Correspondence to foster understanding. Such identification does not imply recommendation or endorsement by the National Institute of Standards and Technology, nor does it imply that the tools and software identified are necessarily the best available for the purpose. Defense Advanced Research Projects Agency, Ballston, VA, USA National Institute of Standards and Technology, Gaithersburg, MD, USA School of Computer Science, Carnegie Mellon University, Pittsburgh, PA, USA Mohamed bin Zayed University of AI, Abu Dhabi, United Arab Emirates You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar invented some of the technologies mentioned in this Correspondence, with patent applications filed by Princeton University and Stanford University. is scientific advisor to Acrobat Genomics and Arbor Biotechnologies. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02671-1'>Meeting European Union biodiversity targets under future land-use demands</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 09:02:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Ecology & Evolution (2025)Cite this article The European Union is committed to achieving ambitious area-based conservation and restoration targets in the upcoming decade. However, there is concern that these targets risk conflicting with socioeconomic needs, particularly for food, timber and bioenergy production. Here we develop an integrated spatial planning approach to identify where restoration, conservation and production allocation could maximize benefits to species conservation and climate mitigation, while acknowledging future land demands of the bio-economy. We show that, while changing production demands risk driving further biodiversity loss by 2030, when these demands are met alongside strategic restoration measures, as outlined by the EU Nature Restoration Regulation, future landscapes could improve the conservation status of populations for more than 20% of species of conservation concern while also increasing terrestrial carbon stocks. Our analysis demonstrates how critical the Nature Restoration Regulation is to achieving biodiversity targets and how integrated planning can align biodiversity policy objectives with future socioeconomic demands. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All required data to run the land allocation optimization problem are available via Zenodo at https://doi.org/10.5281/zenodo.14720069 (ref. All code to process data and solve the optimization problem are available via Zenodo at https://doi.org/10.5281/zenodo.14720069 (ref. Dıaz, S. et al. Assessing Nature's contributions to people. Google Scholar Kunming-Montreal Global Biodiversity Framework (Convention on Biological Diversity, 2022). Xu, H. et al. Ensuring effective implementation of the post-2020 global biodiversity targets. Google Scholar Regulation (EU) 2024/1991 on Nature Restoration and Amending Regulation (EU) 2022/869 (ESO, 2024). Arneth, A. et al. Making protected areas effective for biodiversity, climate and food. Google Scholar Visconti, P. et al. BIOCLIMA: Assessing Land Use, Climate and Biodiversity Impacts of Land-based Climate Mitigation and Biodiversity Policies in the EU (EU, 2024). Global priority areas for ecosystem restoration. Google Scholar Jung, M. et al. Areas of global importance for conserving terrestrial biodiversity, carbon and water. Google Scholar Jung, M. et al. An assessment of the state of conservation planning in Europe. Google Scholar Impact Assessment Accompanying the Proposal for a Regulation of the European Parliament and of the Council on Nature Restoration (EU, 2022). Climate change and EU foreign policy: the negotiation of burden sharing. Google Scholar Visconti, P., O'Connor, L. & Beher, J. Approaches to identify terrestrial priority areas for achieving the 30% and 10% protection target in the EU (ETC BE Report 2024/4) Zenodo https://doi.org/10.5281/zenodo.14529890 (2024). Eckert, I., Brown, A., Caron, D., Riva, F. & Pollock, L. J. 30 × 30 biodiversity gains rely on national coordination. Google Scholar Chaplin-Kramer, R. et al. Conservation needs to integrate knowledge across scales. Adame, M. F., Hermoso, V., Perhans, K., Lovelock, C. E. & Herrera-Silveira, J. Selecting cost-effective areas for restoration of ecosystem services: cost-effective restoration. Montesino Pouzols, F. et al. Global protected area expansion is compromised by projected land-use and parochialism. Ringius, L., Torvanger, A. Burden sharing and fairness principles in international climate policy. Chapman, M., Boettiger, C. & Brashares, J. S. Leveraging private lands to meet 2030 biodiversity targets in the United States. Chaplin-Kramer, R. et al. Global modeling of Nature's contributions to people. De Groot, R. S. et al. Benefits of investing in ecosystem restoration: investing in ecosystem restoration. Martin, D. M. & Lyons, J. E. Monitoring the social benefits of ecological restoration. Malkamaki, A. et al. A systematic review of the socio-economic impacts of large-scale tree plantations, worldwide. Staccione, A. et al. Exploring the effects of protected area networks on the European land system. Buckley, M. C. & Crone, E. E. Negative off-site impacts of ecological restoration: understanding and addressing the conflict. et al. Overcoming the coupled climate and biodiversity crises and their societal impacts. Lehtomaki, J., Tomppo, E., Kuokkanen, P., Hanski, I. Applying spatial conservation prioritization software and high-resolution GIS data to a national-scale study in forest conservation. Heiner, M. et al. Making space: putting landscape-level mitigation into practice in Mongolia. Jung, M. An integrated species distribution modelling framework for heterogeneous biodiversity data. Crosswalk Between EUNIS Habitats Classification and Corine Land Cover (European Environment Agency, 2014). Dou, Y. et al. A new European land systems representation accounting for landscape characteristics. Hengl, T. et al. Global mapping of potential natural vegetation: an assessment of machine learning algorithms for estimating land potential. Walker, W. S. et al. The global potential for increased storage of carbon on land. Calvo Buendia, E. et al. 2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories (IPCC, 2019). Kindermann, G. E. et al. Potential stocks and increments of woody biomass in the European Union under different management and climate scenarios. Carbon Balance Manag. Hanson, J. O. et al. Systematic conservation prioritization with the prioritizr R package. Fastre, C., Van Zeist, W.-J., Watson, J. & Visconti, P. Integrated spatial planning for biodiversity conservation and food production. Roscher, S., Conde, S. & Maitre, J. Final Database on Linkages Between Species/Habitat-Types and Broad Ecosystems (ETC/BD, 2015). Frank, S. et al. Dynamics of the land use, land use change, and forestry sink in the European Union: the impacts of energy and climate targets for 2030. Frank, S., Gusti, M., Valin, H., Forsell, N. & Havl´ık, P. Documentation for Estimating LULUCF Emissions/Removals and Mitigation Potentials With GLOBIOM/G4M (IIASA, 2020). Global land-use implications of first and second generation biofuel targets. Forsell, N. et al. Impact of modelling choices on setting the reference levels for the EU forest carbon sinks: how do different assumptions affect the country-specific forest reference levels? Carbon Balance Manag. Weitzel, M. et al. Model-based assessments for long-term climate strategies. Capros, P. et al. Description of models and scenarios used to assess European decarbonisation pathways. EU Reference Scenario 2020: Energy, Transport and GHG Emissions: Trends to 2050 (EU, 2021). Chapman, M., Boettiger, C., Jung, M. & Visconti, P. milliechapman/EU-restoration-prioritization: v1.1.0. would like to acknowledge financial support from the International Institute for Applied Systems Analysis, Laxenburg (Austria) and the National Member Organizations that support the institute in taking part in the Young Scientists Summer Program. and M.J. acknowledge funding from the European Union Biodiversity and Climate Strategies Assessment (EU BIOCLIMA), Service Contract 07.0202/2020/836131/SER/ENV.D.2. and M.J. acknowledge funding under the Horizon Europe project NaturaConnect. NaturaConnect receives funding under the European Union Horizon Europe research and innovation programme under grant agreement no. We would like to also acknowledge the helpful feedback from members of the Biodiversity, Ecology and Conservation Research Group at IIASA and F. Claeys and A. Teller on early iterations of the analysis. Department of Environmental Systems Science, ETH Zürich, Zurich, Switzerland Department of Environmental Science, Policy, and Management, University of California Berkeley, Berkeley, CA, USA Melissa Chapman & Carl Boettiger Biodiversity, Ecology and Conservation Research Group, International Institute for Applied Systems Analysis (IIASA), Vienna, Austria Martin Jung & Piero Visconti Integrated Biosphere Futures Research Group, International Institute for Applied Systems Analysis (IIASA), Vienna, Austria David Leclère, Andrey L. D. Augustynczik, Mykola Gusti & Leopold Ringwald You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar and M.J. designed the study. developed code, scenarios and visuals. assisted with the data analysis. provided underlying data layers used in the analysis. and M.J. drafted the article with revisions from all authors. Correspondence to Melissa Chapman or Piero Visconti. The authors declare no competing interests. Nature Ecology & Evolution thanks Isaac Eckert, Stefan Wirsenius and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–9, Methods and Tables 1–8. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Chapman, M., Jung, M., Leclère, D. et al. Meeting European Union biodiversity targets under future land-use demands. Nat Ecol Evol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Ecology & Evolution (Nat Ecol Evol) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44358-025-00044-x'>Utilizing aquatic environmental DNA to address global biodiversity targets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-28 02:32:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Biodiversity (2025)Cite this article Achieving global biodiversity goals requires assessing, attributing and reversing the ongoing, unprecedented biodiversity decline in aquatic ecosystems, and relies on adequate data to inform policy and action. Analysis of environmental DNA (eDNA) has become established as a novel and powerful approach to assess the state and functioning of aquatic ecosystems, and although increasingly implemented by stakeholders its potential is not yet fully tapped. In this Perspective, we review the current state of aquatic eDNA research, focusing in particular on the policy relevance of eDNA and its utility in contributing towards the Kunming–Montreal Global Biodiversity Framework. We summarize key technological developments in eDNA science to measure organismal diversity, its potential for spatial and temporal upscaling to become a key reference for local to global biodiversity action, and the next steps needed to effectively implement eDNA for decision-making and reaching biodiversity targets. Using eDNA to support biodiversity assessment will particularly benefit the understanding of understudied ecosystems and allow the direct calculation of ecological indices and implementation of FAIR (findable, accessible, interoperable and reusable) and inclusive data curation. Important next steps for eDNA require proper method standardization and commonly agreed quality standards, populating reference databases, and overcoming methodological constraints in retrofitting novel eDNA-based approaches to existing biodiversity monitoring approaches. Aquatic biodiversity is declining from local to global scales, yet in most regions, no or only minimal data on state and change of biodiversity are available. Representative, scalable and replicable monitoring of aquatic biodiversity is needed to achieve the Kunming–Montreal Global Biodiversity Framework targets. Environmental DNA (eDNA) analysis is a key technology to achieve a global measurement network of state and trends in biodiversity, and many of its technical aspects are ready to be implemented. eDNA analysis allows whole-community assessments, broad taxonomic coverage, high spatiotemporal resolution and calculation of environmental indices. Particularly for undersampled regions, large rivers, lakes and marine systems, eDNA metabarcoding might be an effective technology to rapidly gain biodiversity data. To make eDNA-based monitoring policy frameworks successful and trusted, inclusive development and uptake of international method standards are needed. You have full access to this article via your institution. The unprecedented decline of global biodiversity in terrestrial, freshwater and marine ecosystems1,2,3, including the loss of functional and genetic diversity, is exceeding the planetary boundary of biosphere integrity4 and threatening many ecosystem functions and services. Consequently, biodiversity loss has been identified as an urgent global challenge5, and increasing political consensus exists that rapid measures to halt and reverse biodiversity loss are needed. The Kunming–Montreal Global Biodiversity Framework (GBF), adopted by the Parties of the Convention on Biological Diversity in December 2022 (ref. 6), sets out four overarching goals for 2050: (1) halt human-induced species extinction, (2) achieve sustainable use of biodiversity, (3) implement equitable sharing of benefits from biodiversity and (4) mobilize resources to close the biodiversity finance gap. The GBF also contains 23 targets to be reached by 2030, including conserving at least 30% of the world's land and sea areas, restoring degraded ecosystems, reducing pollution, ensuring sustainable use of biodiversity and performing adequate monitoring of biodiversity globally6,7. Freshwater and coastal marine systems are among the most affected by global change8, and the main drivers of biodiversity decline in these ecosystems are well known9: habitat modification, pollution, invasive alien species, direct exploitation and climate change. Attributing the effects of these large-scale drivers to localized spatial and temporal scales is challenging, primarily because of the transport and mixing of water in aquatic systems. For example, diffuse and point-source inputs of chemicals in rivers have cumulative effects on aquatic biodiversity at a downstream catchment scale10. Furthermore, aquatic ecosystems cross countries, continents and are — via oceans and through the global water cycle — far more continuously integrated than terrestrial systems11. Consequently, effects of anthropogenic pressures have the potential to spread over large spatial extents and across political jurisdictions. To implement actions that successfully address aquatic biodiversity loss, adequate assessments are necessary7,12 and require monitoring of all aquatic biodiversity — including groundwater, surface water and marine systems. Such monitoring must be broadly applicable, comparable and scalable, and must be implementable for countries worldwide that have different baseline information on species diversity and abundance to effectively inform policymaking. Environmental DNA (eDNA)13 analysis is a key advance in biodiversity monitoring14 that will help to achieve the GBF targets. eDNA is directly extracted from an environmental sample15, such as water, soil or air (in contrast to DNA directly isolated from specimens) and analysed. Environmental samples might contain the DNA of all organisms present (including microorganisms, meiofauna and macrofauna), and include both intra- and extracellular DNA. eDNA in aquatic systems can also provide information on the diversity of organisms in adjacent terrestrial systems whose DNA is washed into the water bodies16,17. Environmental samples might also contain RNA (eRNA)18,19, which is more difficult to handle, but can reflect more recent and metabolically active community signals than eDNA20,21. As eDNA can be sampled and analysed across all aquatic ecosystems to detect organisms in a universal manner, it can be applied to meet multiple GBF targets (Fig. Methods based on eDNA are already established and being implemented for the early detection and mitigation of invasive alien species22,23 (GBF target 6: reduce the introduction of invasive alien species by 50%) and the evaluation of the state of ecosystems under multiple stressors and in pre- and post-restoration states24,25 (target 2: restore 30% of all degraded ecosystems). Collecting eDNA samples is becoming increasingly fast and cost-efficient26, rendering this approach highly scalable across space and time. The simplicity of gathering water samples also grants eDNA analysis a high potential for automatization27,28 and for its inclusion in citizen science projects29,30,31. Methods based on eDNA could, therefore, be implemented across the world, even in regions for which we currently lack adequate biodiversity data. a, The Kunming–Montreal Global Biodiversity Framework (GBF) identified 23 action-oriented global targets to be reached by 2030 (ref. Many of these targets require information on the state, change and trends of biodiversity for scientific, political and economic decision-making. Using environmental DNA (eDNA) to provide information on biological diversity offers a potentially universal approach that can support GBF targets by providing baseline data and action–response information to guide decision-making. Establishing, tracking and assessing biodiversity hotspots, biodiversity trends and change is particularly important. b, Multiple targets can be directly assessed using existing eDNA technologies, which are sufficiently developed for implementation but not yet routinely used in most countries. In this Perspective, we examine the current state of aquatic eDNA research, with a particular focus on the policy relevance of eDNA and its utility in contributing towards GBF targets (Fig. We focus on eDNA only (not eRNA) because of its higher readiness level for biodiversity monitoring. First, we explain the technical aspects of eDNA-based biodiversity monitoring required for assessing baseline states and action–response effects. Second, we assess the use of eDNA at the level of communities and ecosystems, targeted species, and its ability to attribute environmental change to underlying drivers. Finally, we outline necessary steps, including standardization, data curation and data deposition, to make eDNA analysis globally accessible. DNA is universal across all species, and traces of DNA are left in the environment by all organisms, for example, by shedding skin cells. Extraction of this DNA from an environmental sample, in theory, enables the detection of any organism, either by targeted screening for single species or assessment of whole communities13,32 (Fig. In both cases15, the first step is sampling water in the focal environment and extracting the DNA present in these samples. For species-specific assays, a portion of DNA is amplified using primers specific to this taxon33,34 and the number of copies present in the original sample is evaluated using a quantitative polymerase chain reaction (qPCR)35 approach, such as real-time qPCR or digital PCR. When assessing whole communities via metabarcoding14, a barcode gene region is amplified using primers with a broader taxonomic scope. The amplified DNA is sequenced via high-throughput sequencing technologies36, which yield millions to billions of sequences (termed ‘reads'). These sequences are then bioinformatically processed and clustered into meaningful units (either as operational taxonomic units or as  amplicon sequence variants) and compared with reference sequences in databases to assign them to specific organisms. These two approaches are well established and are currently the most used methods applied to eDNA samples32,37. New approaches are being developed to investigate the functional or genetic diversity of a group of organisms38, to investigate trophic relationships39 or to obtain information on the whole genome of the organisms present in the samples40,41, but are not yet sufficiently developed for widespread biodiversity monitoring. Generally, sampling of eDNA is applicable to systems with minimal prior information and can also be integrated with data from other monitoring techniques such as remote sensing, camera or audio recordings17,42. a, Aquatic environmental DNA (eDNA) is sampled from marine, freshwater and groundwater ecosystems, enabling spatial (for example, catchment wide) and temporal (for example, across seasons) integration. The basic principle of aquatic eDNA analysis include sampling and capturing the DNA (usually via filtration) followed by DNA extraction, amplification (either of single species or through metabarcoding), sequencing and taxonomic assignment by comparison to DNA reference databases. b, Environmental DNA collected in river networks enables gathering information on biodiversity from a wider spatial extent by integrating hydrological first principles and estimated production and decay of DNA. Collection of samples can be enhanced via automated methods (such as autonomous samplers) or via citizen science approaches, which enables accumulation of data at high spatiotemporal resolution. Following pioneering work in the 1980s and 1990s to isolate microbial communities from environmental samples43,44 and the development of DNA barcoding technology for species identification45, the combination of these technologies in the late 2000s46 led to eDNA becoming established as a research field in its own right13. Uptake of eDNA-based approaches has since vastly expanded in the field of biodiversity research owing to their non-invasiveness and increased efficiency for detecting species in a range of habitats at decreasing costs47. Technological advancements36,48 have enabled progression from detecting single species — initially focused on protected or invasive alien taxa46,49 — to comprehensive biodiversity assessments of many taxa with a metabarcoding approach. Consequently, the past 15 years have seen substantial growth in the number of datasets produced using this approach (Fig. 3), alongside the development of guidelines, large-scale projects and end-user applications15,29,37,47,50,51,52,53,54. a, A schematic chronological timeline of important steps in the development of the concept of environmental DNA (eDNA) (yellow dots), progress of molecular techniques (red dots) and use of eDNA (blue dots). b, Temporal trends in aquatic data volume over time as the number of datasets published in the NCBI Sequence Read Archive on 20 September 2024. Here, a dataset is defined as a data package with a specific run number. Metadata on these datasets were retrieved using Entrez Programming Utilities and parsed using a list of keywords to include only eDNA and eRNA datasets from aquatic environments. HTS, high-throughput sequencing. The uptake of eDNA research has been particularly prevalent in aquatic ecosystems, including rivers, lakes and marine water bodies, because of the ease of sample collection. Gaps in biodiversity knowledge are often larger for aquatic ecosystems than for terrestrial ecosystems, necessitating the implementation of novel approaches tailored to aquatic ecosystems. Current eDNA research in aquatic environments has mostly focused on freshwater ecosystems (65% of studies compared with 25% on marine ecosystems and 10% on other/multiple systems) and more than half of the studies have focused on fish (52%)51 — probably due to their high commercial and recreational values. Additionally, fish tend to shed greater amounts of DNA than species with sclerotized exoskeletons (such as crayfish) and are easier to detect owing to their high biomass and mobility55. The primary strength of using eDNA analysis for assessing biodiversity is the ability to infer species' presence without direct collection or observation13, and that the same approach can be applied across the whole tree of life. The minimally invasive sampling involved is desirable from ethical, work-safety and systems perspectives, as massive upscaling can be achieved without unwanted side effects. However, indirect inference — detecting DNA and not the organisms themselves — is sometimes also seen as the greatest weakness of eDNA analysis. This approach limits inference and information on population structure (abundance or life stages) and requires cautious interpretation of the signal as positive detections might be linked to transport of DNA56, sequencing errors57 or database classification errors58. Therefore, eDNA workflows require rigorous methods and best practices at each stage to minimize the risk of false positives59. Successfully meeting the GBF targets requires the acquisition of three types of data (Fig. First, biodiversity must be mapped across all scales and systems to identify priority areas for conservation and restoration (‘map biodiversity'; for example, GBF targets 3, 6 and 14). Second, the spatial movements and temporal changes in abundance of species of interest (such as pathogens, invasive alien or endangered species) must be tracked (‘track species'; for example, GBF targets 4 or 6). Finally, the responses of ecosystems to anthropogenic changes must be monitored and understood, including identifying pollution levels that do not harm biodiversity (as mentioned in GBF target 7) or tracking recovery success when restoring degraded ecosystems (see the ‘Understand and attribute drivers of biodiversity change' section). The analysis of eDNA is a powerful approach to address all these aspects and to provide common information needed for implementing efficient regulation. In this section, the potential use of eDNA-based approaches for these three objectives is explored and examples of their usefulness for GBF targets presented. Effective ecosystem assessment for policymaking, such as monitoring systems under pressure or identifying biodiversity hotspots, hinges upon efficient characterization of biodiversity, including adequate baseline data and mapping biodiversity trends over space and time60. Traditional sampling approaches such as gillnet fishing or trawling are still widely used, but are not sufficient to reach the ambitious targets of GBF and other regional initiatives (such as the European Union (EU) Water Framework Directive) because they are too labour intensive, destructive and costly to scale up or implement — particularly in undersampled regions, such as tropical streams, open ocean or groundwater systems. By contrast, eDNA analysis allows greater spatial resolution and increased temporal and taxonomical scales compared with traditional approaches, and eDNA sampling is increasingly adopted by public agencies as part of their routine biomonitoring programmes61,62,63. eDNA-based approaches effectively complement traditional aquatic biodiversity monitoring as they overcome the challenges of morphology-based classification (such as juvenile life stages or cryptic species64) and enable access to undersampled and less-accessible systems such as groundwater ecosystems65 or oceanic trenches66. Technological advances are currently driving this transition. The ever-increasing pace of technological developments in eDNA science has inevitably reduced the cost of processing water samples67, enabling the collection of samples at high spatial and temporal resolution68,69. Although sample collection and eDNA capture can still be labour intensive, collecting water samples is becoming considerably faster owing to the development of autonomous robotic and passive samplers. Existing autonomous samplers70,71,72,73 are capable of multisample capture and preservation. When coupled with a microfluidic block, they can even collect, filter, extract and perform multiple targeted qPCR assays of eDNA samples in situ74,75. These automated samplers are highly attractive for early detection of invasive alien species (addressing GBF target 6; see the ‘Tracking endangered, harmful or invasive alien species' section), and can be operated for months at a time without intervention76; however, further technological improvements are possible and the cost of fully automated samplers is currently prohibitive for some applications. Low-cost and less technologically advanced alternative samplers are based on passive capture of eDNA on substrates, such as non-charged cellulose ester or even medical gauze secured in three-dimensional hollow printed housing77,78. Traditionally, monitoring biodiversity was often restricted to specialists, whereas these simple eDNA capture methods have the potential to include citizen scientists in eDNA-based biodiversity assessment29,30,31, contributing to increased participation and improving integration of biodiversity into decision-making at every level (GBF targets 22 and 14, respectively). Owing to transportation and mixing processes in water bodies, aquatic eDNA samples taken at a specific location reflect both the local community and organisms living in the surrounding environment. This feature can be leveraged to understand the spatial structure of biodiversity underpinning the observed data. In rivers, for example, depending on the discharge and other abiotic factors, eDNA can travel downstream from a few hundred metres to tens of kilometres56,79,80, acting as a ‘conveyor belt of biodiversity information'16. Thus, eDNA-based biodiversity measures not only correlate to upstream-averaged measures of land use and habitat types17,81,82,83, but also provide space-filling projections of biodiversity by integrating signals from discrete sampling points into a continuous representation along a watercourse (Fig. Similarly, in marine systems, eDNA is transported by currents and shaped by the stratification of the water column84,85. Coupling stream86,87 and marine88,89 eDNA data with hydrodynamic models is therefore essential to unravel and understand the spatial distribution of biodiversity. Specific models90,91 enable spatial projection of pointwise eDNA samples beyond their collection point into space-filling taxon richness maps at high spatial resolution92. As such, they provide spatially integrated information on biodiversity. The assessment of overall biodiversity is particularly relevant for GBF targets that focus on area-based or area-projected measures, such as target 1 (‘plan and manage all areas to reduce biodiversity loss'), target 3 (‘conserve 30% of land, water and seas') or target 14 (‘integrate biodiversity into decision-making'). These targets are most effectively reached when overall biodiversity assessments of ponds, streams, groundwater and marine systems are comparable, implementable and accessible from local to global scales. Tracking or detecting species of particular interest or concern is often a key piece of information needed for policymaking and communication. In this context, eDNA analysis is a valuable tool for the detection and monitoring of individual species. Indeed, the first applications of eDNA were for the detection of either rare and endangered49 or invasive alien species46,93. The detection and monitoring of individual species of interest is directly associated with several GBF targets, including target 4 (‘halting extinction of [individual] endangered species') and target 6 (‘reducing the introduction of invasive alien species'). Two examples of integrating eDNA information on targeted species into policy are the tracking of the endangered great crested newt in the UK using eDNA94 and the almost globally implemented eDNA (and eRNA) tracking of pathogens (in particular SARS-CoV-2) in wastewater samples95. In both cases, real-time implementation and analysis of eDNA samples enables rapid dissemination to inform policy. eDNA-based methods are now implemented in many countries for routine monitoring of harmful algal blooms96,97, invasive alien species, such as the quagga mussel98 and European green crab99, and of endangered species such as the above-mentioned great crested newt94 or the harbour porpoise49. Importantly, however, an eDNA signal does not automatically indicate an organism's immediate temporal or spatial occurrence. The signal can be transported in space (for example, by water current or in the faeces of predators), reflect past occurrences (ancient DNA), be a transient signal of an organism passing by, or even enter the water via wastewater or terrestrial run-off16,100,101,102. An eDNA signal therefore contains different information compared with the direct observation of an organism, including potential false positives103,104, and thus requires different interpretation. For example, detection of an alien species using eDNA analysis in a location where it has not previously been observed should not trigger a policy response for removal, but rather incentivize the deployment of other methods to confirm the species' presence. Similarly, eDNA-based techniques do not allow the collection of population demographics, such as age and size distribution or sex ratio, which can be required for endangered or invasive alien species monitoring105. Some of these limitations can be overcome by sampling eRNA rather than eDNA, as eRNA represents genes expressed in the sampled environment and degrades faster than eDNA20. As eRNA provides a more contemporary signal than eDNA, its analysis can potentially distinguish between live and dead material to reduce the number of false positives18,21 or, in specific cases, even differentiate life stages (for example, in amphibians). However, eRNA analysis is still under evaluation and, with a few exceptions19, is not yet ready for use in monitoring contexts to inform policymakers. Approaches based on eDNA are potentially relevant for GBF target 4, which specifically links the halt of species extinctions to the prevention of genetic diversity loss. Addressing this target currently relies on indirect measures of genetic diversity based on abundance data for many species and subsequent estimation of effective population size. Determining abundance estimates from eDNA is challenging, but at least possible in some circumstances. Both targeted106,107 and metabarcoding108 approaches have demonstrated correlation between copy number and/or read counts from eDNA and biomass or abundance, indicating that the method is at least semi-quantitative. However, this approach is strongly debated owing to the high potential for bias throughout eDNA workflows, which can create noise and error in the signal109,110,111,112,113. Possible solutions include adding internal standards of a known amount of synthetic DNA (so-called DNA spike-in) to compare read counts within species and across sites. In addition to indirect measures of genetic diversity based on abundance, eDNA analysis could enable direct measurement of intraspecific genetic diversity, even in hard-to-sample environments and for rare and elusive species. This approach is still in development and is not yet ready for direct implementation; however, it has shown promising results in particular cases. So far, investigations have focused on the detection of haplotypes in short sections of mitochondrial (mt)DNA used for metabarcoding, and demonstrated that eDNA captures the genetic diversity of sampled populations38,114,115,116,117. Such approaches can differentiate populations and identify regions with unique genetic variation based on the mtDNA marker genes118. However, these short mtDNA fragments generally have low power to resolve finer genetic structures within or between populations. Targeting longer fragments and nuclear DNA provides increased resolution but presents additional challenges related to the shorter persistence of longer DNA fragments in the environment, and that eDNA represents a mixture of DNA from different individuals. Current methods do not allow assigning genotypes to individuals, which restricts most genetic approaches to population-level estimates119. Likelihood-based DNA mixture models could be applied to address the problem of redundancy (multiple individuals sharing the same allele) as long as a relatively large panel of multi-allelic markers is available120. Technical challenges notwithstanding, microsatellite allele frequencies of round gobies (Neogobius melanostomus) or an Arctic diatom (Thalassiosira hyalina) estimated from aquatic eDNA have been shown to closely resemble allele frequencies from genotyped tissues120,121. A considerable challenge for estimating genetic diversity from eDNA data is the detection of errors generated during the amplification and sequencing steps. For species detection, many bioinformatic procedures remove genetic variability during regular data cleansing and cluster sequences at the species level. For population genetic-level analysis, keeping exact sequence variants while removing erroneous sequences is the primary goal and can be more challenging119. Resolving this challenge requires combined improvements in sequencing technologies and the computational steps to identify and remove errors. For policymaking and management, attribution of biodiversity change is key: only when drivers of change (both positive and negative) are linked to their biodiversity effects can effective policy and management be achieved. Traditionally, attributing drivers to aquatic biodiversity change has relied on monitoring and morphological identification of a relatively small number of species (such as macroinvertebrates, benthic macrofauna, fish and diatoms) that are biological indicators of environmental pressures such as nutrient pollution122. However, these methods are often unsuitable for monitoring emerging drivers, such as pollution by nanoparticles or microplastics, and hinge on the (often incorrect) assumption that a few indicator groups represent overall ecosystem responses. eDNA-based methods have already led to marked improvements in our ability to document biodiversity123 and identify pollutant thresholds124 by integrating data from diverse taxonomic groups with only one sample. This approach, in comparison with traditional methods, will provide a more comprehensive — and at least more diversified — response of aquatic communities to anthropogenic changes, individually and in combination24,25,125. Current monitoring, whether with traditional or molecular tools, often focuses on the structure of communities (their richness and composition) rather than their functional properties123. However, the effect of biodiversity loss on ecosystem services cannot be fully understood from estimates of taxonomic diversity alone. To characterize ecosystem health in its entirety, structural and functional biodiversity must be measured as these parameters can respond differently to different stressors. This type of monitoring requires a shift from focusing on which species provides the function to understanding how the functional properties of the whole community respond to pressures123,126. For example, co-occurrence data generated from eDNA metabarcoding across the tree of life allow reconstructing multitrophic ecological networks, from which functional properties such as connectance and nestedness can be estimated127. Other emerging approaches for addressing ecosystem changes are based on the analysis of expressed genes via eRNA. The strength of these eRNA methods lies in monitoring the physiological state of populations or communities by detecting changes in the activity of single or multiple genes18. Shotgun sequencing of total RNA from environmental samples (metatranscriptomics) allows the identification of genes that are actively and collectively expressed under different conditions128. Changes in freshwater biofilm microbial community-level expression in response to pollutants have been demonstrated, even when changes in community composition were not detected129. Additionally, experiments have revealed changes in community-wide gene expression of aquatic organisms under heat stress through sampling experimental tank water, demonstrating that metatranscriptomics of extra-organismal eRNA can reveal gene expression response to environmental change130. These emerging measures of ecological condition, especially when using eRNA, are promising but not yet ready for widespread use for practitioners and to inform decision-making. Advances in artificial intelligence and modelling are enabling new approaches for attributing biodiversity change to environmental and/or anthropogenic drivers. For example, eDNA coupled with machine-learning models is used to infer the ecological status of aquatic systems131,132, and allows inclusion of a broader set of indicator taxa. Microorganisms, for example, differ considerably in their metabolism and sensitivity to different pollutant sources133, but have been largely ignored in biomonitoring because they are difficult to culture and identify, and because knowledge of their ecological function is unavailable134,135. Validated approaches based on profiling communities using amplicon sequence variants and/or operational taxonomic units detect changes in the structure of communities of bacteria133,136, freshwater invertebrates25,137 and diatoms138 against well-understood disturbance gradients, and have demonstrated strong improvements in the accuracy of ecological quality assessment139,140. Supervised machine learning combined with metabarcoding will complement and expand current morphology-based methods to produce the same ecological quality assessments at increased speed and cost efficiency, and without necessarily depending on taxonomic assignment using barcode reference databases135 (Fig. The ability of supervised machine learning to outperform traditional indicator value approaches, using training datasets with reference disturbance levels to estimate biotic indices, has been demonstrated131,132,140,141,142,143. The principle of the machine-learning approach is to link genomic features of the environmental DNA (eDNA) samples (such as operational taxonomic units (OTUs), amplicon sequence variants (ASVs) or taxonomically assigned information) to environmental reference states. Specifically, during the training phase, the model learns relationships between genomic features and known environmental conditions, iteratively refining its predictive rules. In the predicting phase, the trained model applies these learned rules to analyse new eDNA samples, generating straightforward, actionable predictions. Through continuous feedback cycles, the model enhances its accuracy, improving its utility for environmental monitoring and management. Overall, eDNA analysis enables attributing changes in biodiversity to specific environmental factors17,65,144, evaluating the impact of implementing protected areas145,146, and calculating biodiversity and ecosystem state indices47. Aquatic eDNA analysis can also attribute change beyond aquatic ecosystems: imprints of terrestrial land use on aquatic eDNA are well established, showing that aquatic eDNA signals associate to the structure and diversity of terrestrial land use assessed by remote sensing at 400–1,000-m upstream distances17. This information can then be translated into effective practice for the prevention of biodiversity decline. eDNA-based biodiversity monitoring has the potential to scale from a local to a regional and eventually a global perspective, and enables data integration and the use of this information to downscale back to regional and local interventions42. eDNA approaches are already widely used across different scales, which helps to contribute to monitoring progress on GBF targets. These methods are particularly useful for spearheading activities in regions in which previous knowledge on biodiversity is limited; for example, eDNA-based approaches are being tested to study the biodiversity of deep marine habitats147,148. Similarly, subterranean habitats are one of the least studied environments and the use of molecular tools such as eDNA contributes to mapping their biodiversity65,149. Moreover, eDNA-based assessments are being developed and implemented in many of the world's largest rivers such as the Danube150, Yangtze144 or Rhine151, which are not only home to enigmatic megafauna but are also severely threatened by anthropogenic activities. Achieving target 1 of the GBF (‘plan and manage all areas to reduce biodiversity loss') depends on understanding the initial state and change of overall biodiversity. In many cases, eDNA data are the very first baseline data, and linking these data to possible drivers of biodiversity change will ultimately guide restoration planning152. The ability of eDNA analysis to provide data on the state and change of individual species focuses on target 4 (‘halt species extinction, protect genetic diversity, and manage human–wildlife conflicts'), which directly refers and links to the monitoring and managing of biodiversity, at the species or within-species level. Similarly, target 6 (‘reduce the introduction of invasive alien species by 50% and minimise their impact') operates at the species level, aiming to decrease the impact of alien species in part through preventing their introduction in aquatic systems. To support this approach, eDNA analysis provides early detection and monitoring of alien species. Although initial attempts to detect invasive alien species (for example, the Asian carps Hypophthalmichthys nobilis and Hypophthalmichthys molitrix93,153) were partially challenged by detection failures (false absences), false presence indications and unknown lower thresholds of detectability, increasing standardization and the adoption of structured decision-making processes have resulted in a relatively broad application of eDNA methods to monitor Asian carp species in North America23,154. Similar efforts have been established for other aquatic alien species, such as invasive kelp (Undaria pinnatifida) and sea stars (Asterias amurensis)155, tunicates (Didemnum vexillum)156, dreissenid mussels (Dreissena spp. )22 or freshwater aquatic plants (Najas minor)157. The contribution of eDNA analysis to GBF targets 1, 4 and 6 focuses on simply detecting the presence of species. However, eDNA analysis can provide insight into why and how biodiversity is affected; that is, mechanistic monitoring of aquatic systems and the effects of stressors. Classic assessments of environmental toxicology and environmental chemistry have focused on monitoring a highly diversified set of chemicals10. To reach target 7 (‘reduce pollution to levels that are not harmful to biodiversity'), pollutant monitoring must be linked to an ecosystem's ecological health and resilience status. eDNA is effective in capturing how environmental stressors, particularly chemical pollution, have combined negative effects on the structure, stability and composition of aquatic communities across a wide range of taxa158,159,160. Target 10 (‘enhance biodiversity and sustainability in agriculture, aquaculture, fisheries, and forestry') requires similar monitoring of effects on biodiversity, but in relation to the production and harvesting of natural resources. eDNA is already routinely used to monitor impacts of aquaculture (such as salmon farming) by examining marine benthic sediments161 and has also been used to monitor the outflow of wastewater into natural streams, focusing on possible effects of different agricultural runoff100. eDNA-based methods have the potential to greatly improve our understanding of aquatic ecosystems and contribute to national and global biodiversity targets. For successful inclusion of eDNA analysis in monitoring programmes, careful action is needed as methods must be broadly applicable, comparable and scalable, and must be implementable for countries across the globe. Two key barriers exist to the implementation of eDNA: first, the perceived limited comparability between results generated with eDNA-based approaches and those obtained with traditional methods162,163,164; and second, unequal access to technical infrastructure. eDNA metabarcoding technology is considered ready for implementation and allows — when needed — retrofitting of indices to current standards47,135. Discrepancies with traditional assessments occur because of amplification biases and incomplete reference databases, but also because eDNA can detect cryptic or elusive species47,123,165 previously missed by traditional tools. Assessing biodiversity and achieving successful conservation outcomes will require effectively using eDNA alongside traditional methods, as biodiversity monitoring globally remains under-resourced. In high-income countries, existing monitoring programmes should be complemented with eDNA-based approaches, creating continuity for existing long-term monitoring and generally allowing comparison at the index level. Maintenance and overlap with other programmes will increase costs and require extra work over the short term, but secure existing data series in the long term. To encourage the rapid adoption of eDNA globally, in low- and middle-income countries, novel (and first) monitoring could use eDNA only. This section discusses necessary steps towards a global application of eDNA to assess biodiversity within the context of the GBF. First, methods of standardization must be agreed and promoted (agree on minimal standards on sampling and processing, provide training opportunities and institutionalize quality control). Second, adequate reference databases, digital sequence data deposition and general access to these resources must be established. Finally, efforts must be made to increase inclusivity and remove technology barriers (collective benefit, authority to control, responsibility and ethics (CARE) principles and Nagoya Protocol). The development and implementation of international standards is an essential next step to make eDNA approaches broadly applicable. These next steps, including establishing standardized protocols and best practices for general implementation of eDNA, are particularly relevant for global initiatives, such as the GBF, that rely on common agreement beyond the borders and jurisdiction of individual countries. eDNA has always been developed as a tool for practitioners, rather than for academic research only. Consequently, discussions on an adequate and replicable implementation paralleled the rapid advances in the field153,162,166,167. General protocols and guidelines were developed from the first species-based assessments, such as amphibians in the UK29, to the monitoring of whole communities of invertebrates and fish, for example, in Switzerland168, the EU37,169, China170 or the USA93. Despite these standardization efforts, eDNA-based biodiversity assessments still use various approaches throughout the world171. The selection of a certain filter material, DNA extraction methods, primers, sequencing technologies and post-processing steps can drastically affect the inferred taxa lists172,173. For multispecies detection approaches, such as eDNA metabarcoding, the effects of the choice of methods are most pronounced for rare species that are easily missed. Broader use of eDNA for monitoring thus requires standardization of practices63,174, in particular a global agreement on general minimum standards for sampling and processing, reporting and data publishing. Here, specific recommendations for different freshwater ecosystems (for example, small stream versus large, stratified lake), environmental factors (turbidity, flow, pH and so on) and specific research questions and tasks (single species detection versus multispecies detection) must be accounted for37,168,175. This standardization is well underway — a first ISO standard is available in draft form after international voting176, and several others are in development under the framework ISO/TC 147/SC 5 (Biological methods) — but must be continued. Given the ambitious 2030 GBF timeline, the implementation and standardization of the method in its current form might be more effective than deferring implementation considering the promise of upcoming new technologies. Comparability and scalability across taxa and regions are the biggest assets of eDNA analysis. These capabilities hinge, however, on adequate access to reference databases as most uses of eDNA rely on assigning specific sequences to taxonomic and genetic reference databases (although taxonomy-free approaches exist139). These reference databases are globally still highly unevenly distributed, and large geographic and taxonomic gaps are present177,178,179. For example, (sub)tropical systems and many invertebrate groups are far less represented than temperate systems and vertebrate groups. Populating, expanding and maintaining these reference databases is imperative. Advances in sequencing technologies enable the upscaling of reference database generation by several orders of magnitude while also drastically reducing costs. However, many — even formally described — species are still absent from public genomic repositories180. Novel nanopore sequencing technologies could allow the barcoding of thousands of specimens from a single amplicon pool at a sequencing cost of few US$ per specimen, offering the potential to catalogue all species within two decades181. The portability and applicability of sequencing technologies in field conditions is also rapidly improving; for example, portable sequencers have been used for DNA barcoding in the Ecuadorian Chocó Rainforest182 and other remote locations. Populating databases is still a major bottleneck as implementation is lagging behind this progress in technologies, and also critically depends on taxonomists' knowledge and expertise. Scientists and end-users must be mobilized alike and commit to complementing the databases as the task is important, necessary and cannot be delegated. This challenge is particularly pronounced given the large number of undescribed or underrepresented taxa, discrepancies in taxonomic classifications and the need for rigorous validation of reference sequences. The complexity of maintaining and curating these databases has been widely discussed in the literature, touching on issues such as funding177 and standardization efforts58. A comprehensive discussion of these aspects is, however, beyond the scope of this Perspective. Alongside data generation, the raw data produced from environmental samples must be deposited in repositories (Table 1) whose long-term operability is guaranteed183. Priority should be given to open platforms specifically adapted to host genetic data, enabling better discoverability through advanced application programming interfaces (which enable the exchange of data between existing programmes) and research tools184. The International Nucleotide Sequence Database Collaboration, linking the Sequence Read Archive, the European Nucleotide Archive and the DDBJ Sequence Read Archive are currently among the most suitable options for depositing and indexing eDNA data. Ideally, eDNA repositories should facilitate the seamless exchange and integration of data across different platforms and disciplines. For example, biodiversity data portals such as the Ocean Biodiversity Information System (OBIS) and the Global Biodiversity Information Facility (GBIF) can include biodiversity records derived from raw DNA data while providing a permanent link to the original repositories185. Publication of eDNA datasets in these global biodiversity databases is crucial to ensure that data can be used by policymakers with little to no specialist knowledge186. However, so far, few eDNA datasets have been made publicly available through these platforms. The key to successful adoption is the mandatory deposition of eDNA raw data along with relevant metadata provided in standardized data models, such as provided by the Genomics Standards Consortium (see the MIxS MIMARKS checklists187) or the Biodiversity Information Standards consortium TDWG (Darwin Core extended data model188,189). As part of ongoing international task forces, new metadata standards are currently being developed190. These data standards ensure that the most critical metadata are recorded and made available through a standardized vocabulary191, which in turn can be aligned with other biodiversity community standards. Such metadata (Table 1) are crucial as they ensure that the data can be accurately interpreted and reused in various contexts, which is necessary to reach a broad range of stakeholders, including citizens. eDNA data producers must focus on creating datasets that can be used and reused186, which involves ensuring the longevity of data, adopting standard formats and metadata for ease of interpretation, and promoting open access to facilitate wide usage across different sectors. Thus, published eDNA data must adhere to the FAIR principles — findable, accessible, interoperable and reusable192 — as this approach enhances the discoverability and usability of eDNA data across various scientific and non-scientific domains186 (Table 1). Therefore, setting minimum guidelines for data reporting and analysis is recommended, and minimum information to be reported as part of any aquatic eDNA study has been proposed190,193. Data deposition must include the raw sequencing data and the essential metadata specific to the samples and their processing in the laboratory, but also the final community data matrix, easily interpretable by the stakeholders, as well as the bioinformatics tools and reference databases required to produce the results. These minimum guidelines will ensure a baseline of quality and comparability for eDNA research and application. The successful, widespread implementation of eDNA hinges on removing barriers to its access and making the methods inclusive. Technological barriers and uneven distribution of knowledge and resources between high-income countries and low- and middle-income countries exist across biodiversity sciences. As eDNA analysis (similar to remote sensing and other technology-driven approaches) depends on specialized laboratory and analysis techniques, overcoming discrepancies in lack of knowledge, training and resources is particularly relevant. If countries cannot afford eDNA-based monitoring or do not have access to the technologies to process and assess the samples, the existing inequality in biodiversity knowledge will further increase. In addition to ensuring equitable access to eDNA technology, local and Indigenous knowledge on biodiversity must be both valued and integrated into technology-driven monitoring efforts. Although measuring biodiversity variables through proxies such as eDNA is technologically feasible, maintaining taxonomic and natural history knowledge, as well as connections to ecosystems and nature, is essential. Datafication of biodiversity and reliance on technological solutions run the risk of disconnecting people with nature194, as sequencing information and eDNA approaches cannot replace observing and experiencing nature directly. Additionally, limits exist to the quality and applicability of data available from methods based on indirect collection of information, such as remote sensing or eDNA. Conversely, eDNA also has the potential to be a powerful tool that provides people with a new perspective on their surrounding environment and biodiversity, strengthening their interest in and commitment to nature29,30,31. Any eDNA-based approaches should, therefore, consider how to include the public and taxonomic experts such that the information on biodiversity from eDNA is accompanied by increased understanding and perception of biodiversity. Addressing the sociocultural dimensions of eDNA, particularly when data originate from territories of Indigenous communities, necessitates the adoption of the CARE principles195. These principles ensure that eDNA research is conducted in a manner that respects the rights and traditions of local communities, promoting an ethical approach and facilitating the local populations acceptance of biodiversity research. Incorporating the CARE principles aligns eDNA practices with broader societal values and legal frameworks, such as the Nagoya Protocol196 and GBF target 13 (‘fair and equitable sharing of genetic resources'). In some situations, however, open-access goals such as the population and maintenance of reference databases can bypass traditional mechanisms for benefit sharing, potentially disadvantaging the countries or communities where the genetic material originated. Public deposition without consultation can undermine the authority of these communities over resources tied to their cultural identity and governance systems. Moreover, eDNA can also complicate traditional frameworks, especially when DNA is transported across national boundaries by natural processes such as ocean currents, river flows and air197. All stakeholders — nations, Indigenous groups and local communities — must be included in discussions about the implementation of CARE principles, data sharing and benefit sharing to ensure that the benefits of eDNA research are distributed fairly and with due regard to the sovereignty and cultural significance of the data sources194,198. One of the four overarching goals of the GBF is to mobilize the business and finance sectors to close the gap in funding biodiversity. In particular, target 19 (‘Mobilize $200 billion per year for biodiversity') encourages solutions to incentivize international companies and finance to engage in the monitoring of biodiversity and contribute monetarily to its protection. Several initiatives are being promoted to market biodiversity and to incentivize the development of activities and technologies that are not harmful for the environment or that mitigate deleterious actions, such as green bonds or biodiversity credits (quantifiable units of biodiversity improvement that should provide an incentive for funding conservation projects)199,200. Although such market-based solutions to environmental challenges are gaining substantial attention at the global scale, deep uncertainties exist regarding the process and their benefits199. Without proper transparency and accountability, these schemes might be ineffective and lead to greenwashing and inequitable outcomes, as has occurred for carbon credits201. A challenge to the broad use of biodiversity credits is the difficulty in quantify biodiversity improvements accurately and in a standardized manner. Sampling and identification methods vary substantially, are not all easily implementable across the globe, and the multidimensionality of biodiversity cannot be captured in one single number. eDNA metabarcoding has been suggested as a useful method to help the development of marketing metrics for biodiversity, as its simplicity and scalability could allow the collection of comparable data across the globe. However, this approach also requires global standardization of methods and a clear reference to its limits. Moreover, eDNA is particularly well suited for aquatic environments but is less proven for terrestrial ecosystems, which might limit the comparability of this data201,202. Addressing biodiversity decline and implementing global measures to assess, manage and restore biodiversity are essential and at the core of the GBF framework. The huge progress in molecular-based biodiversity monitoring, particularly the use of eDNA, has created the first feasible and coherent approach to monitor aquatic biodiversity. To successfully implement eDNA biomonitoring as a reliable and widely trusted basis for policy and decision-making, a new common agreement on its use is needed, including common protocols, data standards and standardization for quality assurance. Further, implementation of eDNA approaches hinges on increased FAIRness and open accessibility of biodiversity data globally, which also presents an opportunity to enhance collaboration, improve conservation efforts and advance research. Rapid completion of reference database, curation and integration of FAIR principles are needed. Although progress towards these goals, in particular where and by whom data are stored, is becoming more systematically addressed in academic research, agreement has not yet been reached and implemented regarding data produced by public stakeholders and private companies. Sample repositories (such as the Global Genome Biodiversity Network) and data centres established for eDNA data should not only be accessible and implemented for high-income countries (for example, the International Nucleotide Sequence Database Collaboration), but also for low- and middle-income countries. As these repositories are funded by public money and provide collective benefits, all data collected through governmental programmes should be included — even when data collection and analysis is executed by private companies. Coherent, integrated and complete data storage and access will allow establishing new universally valid metrics for describing biodiversity and elevate the quality and comparability of biodiversity data. In the framework of the GBF, regional to global progress on reaching targets could be tracked by new eDNA-based indicators on the state, change and pressures of biodiversity. Particularly in aquatic systems, these metrics can be calculated at high spatial and temporal resolutions: transport of eDNA in the water allows predictions of biodiversity across catchments and water bodies, whereas matching eDNA sampling to existing monitoring stations of water quality allows synergy with existing infrastructure. A pragmatic approach should be chosen when designing new metrics versus retrofitting to existing monitoring programmes: that is, complementing rather than replacing existing monitoring programmes, yet building new monitoring timelines where no data are available. Swiftly establishing a global network of coherent aquatic biodiversity monitoring using eDNA is realistic and required to achieve GBF targets. A possible way forwards is institutionalizing eDNA monitoring activities through regional centres (examples of such attempts include the Defra DNA Center of Excellence in the UK, the Joint Research Center in the EU and the National eDNA Reference Centre in Australia). These centres coordinate activities that include funding, supporting and driving research that is relevant to policy implementation, reaching out to local monitoring personnel, receiving samples from citizens, performing quality checks through internal positive control, and generating reference material and certification standards. Such regional eDNA excellence centres can facilitate capacity building and act as coordinating and executing building blocks, linking both local and global biodiversity monitoring centres (similar to GEO BON). The Canadian Centre of Biodiversity Genomics at the University of Guelph and other federal initiatives are examples of such centres. Global join up of these initiatives and centres is crucial to ensure international standards and comparability of datasets across countries. Finally, to future proof the data acquired and ensure long-term comparability, adequate biobanking of samples is a necessary next step, which should, at a minimum, include storing metadata, raw sequencing output, and the extracted DNA samples in quality and quantity sufficient to be used by generations to come203. Together, these measures will elevate the accessibility and extent of aquatic biodiversity data to levels needed for achieving GBF targets and securing global biodiversity. Keck, F. et al. The global human impact on biodiversity. Pereira, H. M. et al. Global trends and scenarios for terrestrial biodiversity and ecosystem services from 1900 to 2050. Google Scholar Loreau, M. et al. Do not downplay biodiversity loss. Google Scholar Rockström, J. et al. A safe operating space for humanity. Google Scholar World Economic Forum. Global Risks Report 2022, 17th edn (WEF, 2022). United Nations Convention on Biological Diversity. Kunming–Montreal Global Biodiversity Framework. CBD https://www.cbd.int/doc/decisions/cop-15/cop-15-dec-04-en.pdf (UN, 2022). Gonzalez, A. et al. A global biodiversity observing system to unite monitoring and guide action. Google Scholar Almond, R. E., Grooten, M. & Peterson, T. World Wildlife Fund. Living Planet Report 2020 — Bending the Curve of Biodiversity Loss (WWF, 2020). Intergovernmental Science–Policy Platform on Biodiversity and Ecosystem Services. Global assessment report on biodiversity and ecosystem services (IPBES, 2019). Schwarzenbach, R. P. et al. The challenge of micropollutants in aquatic systems. Google Scholar Vörösmarty, C. J. et al. Global threats to human water security and river biodiversity. Google Scholar Leclère, D. et al. Bending the curve of terrestrial biodiversity needs an integrated strategy. Google Scholar Taberlet, P., Coissac, E., Hajibabaei, M. & Rieseberg, L. H. Environmental DNA. Google Scholar Taberlet, P., Coissac, E., Pompanon, F., Brochmann, C. & Willerslev, E. Towards next-generation biodiversity assessment using DNA metabarcoding. Google Scholar Pawlowski, J., Apothéloz-Perret-Gentil, L. & Altermatt, F. Environmental DNA: what's behind the term? Clarifying the terminology and recommendations for its future use in biomonitoring. Google Scholar Deiner, K., Fronhofer, E. A., Mächler, E., Walser, J.-C. & Altermatt, F. Environmental DNA reveals that rivers are conveyer belts of biodiversity information. Google Scholar Zhang, H. et al. A spatial fingerprint of land–water linkage of biodiversity uncovered by remote sensing and environmental DNA. Google Scholar Yates, M. C., Derry, A. M. & Cristescu, M. E. Environmental RNA: a revolution in ecological resolution? Google Scholar A. et al. Environmental monitoring: inferring the diatom index from next-generation sequencing data. Google Scholar Kagzi, K., Hechler, R. M., Fussmann, G. F. & Cristescu, M. E. Environmental RNA degrades more rapidly than environmental DNA across a broad range of pH conditions. Google Scholar Pochon, X., Zaiko, A., Fletcher, L. M., Laroche, O. Wanted dead or alive? Using metabarcoding of environmental DNA and RNA to distinguish living assemblages for biosecurity applications. Google Scholar Sepulveda, A. et al. Using structured decision making to evaluate potential management responses to detection of dreissenid mussel (Dreissena spp.) Google Scholar US Fish and Wildlife Service. Great Lakes eDNA monitoring program. Asian carp Canada https://www.asiancarp.ca/surveillance-prevention-and-response/great-lakes-edna-monitoring-program/ (US FWS, 2020). Romero, F., Acuña, V. & Sabater, S. Multiple stressors determine community structure and estimated function of river biofilm bacteria. Google Scholar Beermann, A. J., Zizka, V. M. A., Elbrecht, V., Baranov, V. & Leese, F. DNA metabarcoding reveals the complex and hidden responses of chironomids to multiple stressors. Google Scholar Fediajevaite, J., Priestley, V., Arnold, R. & Savolainen, V. Meta-analysis shows that environmental DNA outperforms traditional surveys, but warrants better reporting standards. Google Scholar Buchner, D., Macher, T.-H., Beermann, A. J., Werner, M.-T. & Leese, F. Standardized high-throughput biomonitoring using DNA metabarcoding: strategies for the adoption of automated liquid handlers. Google Scholar Ruppert, K. M., Kline, R. J. & Rahman, M. S. Past, present, and future perspectives of environmental DNA (eDNA) metabarcoding: a systematic review in methods, monitoring, and applications of global eDNA. Google Scholar Biggs, J. et al. Using eDNA to develop a national citizen science-based monitoring programme for the great crested newt (Triturus cristatus). Google Scholar Larson, E. R. et al. From eDNA to citizen science: emerging tools for the early detection of invasive species. Google Scholar Couton, M. et al. Integrating citizen science and environmental DNA metabarcoding to study biodiversity of groundwater amphipods in Switzerland. Google Scholar Deiner, K. et al. Environmental DNA metabarcoding: transforming how we survey animal and plant communities. Google Scholar Sanger, F., Nicklen, S. & Coulson, A. R. DNA sequencing with chain-terminating inhibitors. Google Scholar Mullis, K. et al. Specific enzymatic amplification of DNA in vitro: the polymerase chain reaction. Cold Spring Harb. Google Scholar Higuchi, R., Fockler, C., Dollinger, G. & Watson, R. Kinetic PCR analysis: real-time monitoring of DNA amplification reactions. Google Scholar Margulies, M. et al. Genome sequencing in microfabricated high-density picolitre reactors. Google Scholar Bruce, K. et al. A Practical Guide to DNA-Based Methods for Biodiversity Assessment (Pensoft, 2021). Sigsgaard, E. E. et al. Population-level inferences from environmental DNA — current status and future perspectives. Google Scholar Abad-Recio, I. L., Alonso-Sáez, L. & Lanzén, A. Toward functional profiling for eDNA‐based monitoring in coastal environments: a comparison of three approaches. Google Scholar MacKenzie, M. & Argyropoulos, C. An introduction to nanopore sequencing: past, present, and future considerations. Google Scholar Bovo, S. et al. Shotgun metagenomics of honey DNA: evaluation of a methodological approach to describe a multi-kingdom honey bee derived environmental DNA signature. Google Scholar Thomsen, P. F., Jensen, M. R. & Sigsgaard, E. E. A vision for global eDNA-based monitoring in a changing world. Google Scholar Ogram, A., Sayler, G. S. & Barkay, T. The extraction and purification of microbial DNA from sediments. Google Scholar Steffan, R. J., Goksøyr, J., Bej, A. K. & Atlas, R. M. Recovery of DNA from soils and sediments. Google Scholar Hebert, P. D. N., Cywinska, A., Ball, S. L. & deWaard, J. R. Biological identifications through DNA barcodes. Google Scholar Ficetola, G. F., Miaud, C., Pompanon, F. & Taberlet, P. Species detection using environmental DNA from water samples. Google Scholar Blackman, R. et al. Environmental DNA: the next chapter. Google Scholar Satam, H. et al. Next-generation sequencing technology: current trends and advancements. Google Scholar Foote, A. D. et al. Investigating the potential use of environmental DNA (eDNA) for genetic monitoring of marine mammals. Google Scholar Leese, F. et al. DNAqua-Net: developing new genetic tools for bioassessment and monitoring of aquatic ecosystems in Europe. Google Scholar Takahashi, M. et al. Aquatic environmental DNA: a review of the macro-organismal biomonitoring revolution. Google Scholar De Brauwer, M. et al. Best practice guidelines for environmental DNA biomonitoring in Australia and New Zealand. Google Scholar Ferrante, J. et al. Gaining decision-maker confidence through community consensus: developing environmental DNA standards for data display on the USGS Nonindigenous Aquatic Species database. Google Scholar Minamoto, T. et al. An illustrated manual for environmental DNA research: water sampling guidelines and experimental protocols. Google Scholar Andruszkiewicz Allan, E., Zhang, W. G., C. Lavery, A. Environmental DNA shedding and decay rates from diverse animal forms and thermal regimes. Google Scholar Deiner, K. & Altermatt, F. Transport distance of invertebrate environmental DNA in a natural river. Google Scholar Burian, A. et al. Improving the reliability of eDNA data interpretation. Google Scholar Keck, F., Couton, M. & Altermatt, F. Navigating the seven challenges of taxonomic reference databases in metabarcoding analyses. Google Scholar Goldberg, C. S., Strickler, K. M. & Pilliod, D. S. Moving environmental DNA methods from concept to practice for monitoring aquatic macroorganisms. Google Scholar & Londoño, M. C. Monitor biodiversity for action. Google Scholar Norros, V. et al. Roadmap for implementing environmental DNA (eDNA) and other molecular monitoring methods in Finland — vision and action plan for 2022–2025 (Finnish Environment Institute, 2022). Blancher, P. et al. A strategy for successful integration of DNA-based methods in aquatic monitoring. Google Scholar Kelly, R. P. et al. Toward a national eDNA strategy for the United States. Mason, D. H. et al. Certain detection of uncertain taxa: eDNA detection of a cryptic mountain sucker (Pantosteus jordani) in the Upper Missouri River, USA. Google Scholar Couton, M., Hürlemann, S., Studer, A., Alther, R. & Altermatt, F. Groundwater environmental DNA metabarcoding reveals hidden diversity and reflects land-use and geology. Google Scholar Laroche, O., Kersten, O., Smith, C. R. & Goetze, E. From sea surface to seafloor: a benthic allochthonous eDNA survey for the abyssal ocean. Lee, K. N., Kelly, R. P., Demir-Hilton, E., Laschever, E. & Allan, E. A. Adoption of environmental DNA in public agency practice. Sander, M. et al. Environmental DNA time series analysis of a temperate stream reveals distinct seasonal community and functional shifts. River Res. Google Scholar Tillotson, M. D. et al. Concentrations of environmental DNA (eDNA) reflect spawning salmon abundance at fine spatial and temporal scales. Google Scholar Formel, N., Enochs, I. C., Sinigalliano, C., Anderson, S. R. & Thompson, L. R. Subsurface automated samplers for eDNA (SASe) for biological monitoring and research. Google Scholar Hendricks, A. et al. A miniaturized and automated eDNA sampler: application to a marine environment. In OCEANS 2022, Hampton Roads https://doi.org/10.1109/oceans47191.2022.9977218 (IEEE, 2022). Hendricks, A. et al. Compact and automated eDNA sampler for in situ monitoring of marine environments. Google Scholar George, S. D. et al. Field trials of an autonomous eDNA sampler in lotic waters. Google Scholar Preston, C. M. et al. Underwater application of quantitative PCR on an ocean mooring. Google Scholar Hansen, B. K. et al. Remote, autonomous real-time monitoring of environmental DNA from commercial fish. Google Scholar Sepulveda, A. J. et al. Robotic environmental DNA bio-surveillance of freshwater health. Google Scholar Maiello, G. et al. Little samplers, big fleet: eDNA metabarcoding from commercial trawlers enhances ocean monitoring. Google Scholar Chen, X. et al. Comparative evaluation of common materials as passive samplers of environmental DNA. Google Scholar Pont, D. Predicting downstream transport distance of fish eDNA in lotic environments. Google Scholar Van Driessche, C., Everts, T., Neyrinck, S. & Brys, R. Experimental assessment of downstream environmental DNA patterns under variable fish biomass and river discharge rates. Google Scholar Brantschen, J. et al. Habitat suitability models reveal the spatial signal of environmental DNA in riverine networks. Cantera, I. et al. Low level of anthropization linked to harsh vertebrate biodiversity declines in Amazonia. Google Scholar Zong, S. et al. Combining environmental DNA with remote sensing variables to map fish species distributions along a large river. Google Scholar et al. Water stratification in the marine biome restricts vertical environmental DNA (eDNA) signal dispersal. Google Scholar et al. Environmental DNA (eDNA) metabarcoding reveals strong discrimination among diverse marine habitats connected by water movement. Google Scholar Laporte, M. et al. Caged fish experiment and hydrodynamic bidimensional modeling highlight the importance to consider 2D dispersion in fluvial environmental DNA studies. Google Scholar & Sassoubre, L. M. Environmental DNA (eDNA) shedding and decay rates to model freshwater mussel eDNA transport in a river. Google Scholar Andruszkiewicz, E. A. et al. Modeling environmental DNA transport in the coastal ocean using Lagrangian particle tracking. Fukaya, K. et al. Estimating fish population abundance by integrating quantitative data on environmental DNA and hydrodynamic modelling. Google Scholar Carraro, L. & Altermatt, F. eDITH: an R‐package to spatially project eDNA‐based biodiversity across river networks with minimal prior information. Google Scholar Carraro, L., Mächler, E., Wüthrich, R. & Altermatt, F. Environmental DNA allows upscaling spatial patterns of biodiversity in freshwater ecosystems. Google Scholar Blackman, R. C., Carraro, L., Keck, F. & Altermatt, F. Measuring the state of aquatic environments using eDNA-upscaling spatial resolution of biotic indices. Google Scholar Jerde, C. L. et al. Detection of Asian carp DNA as part of a Great Lakes basin-wide surveillance program. Google Scholar Rees, H. C. et al. The application of eDNA for monitoring of the great crested newt in the UK. Google Scholar Jahn, K. et al. Early detection and surveillance of SARS-CoV-2 genomic variants in wastewater using COJAC. Google Scholar Feist, S. M. & Lance, R. F. Genetic detection of freshwater harmful algal blooms: a review focused on the use of environmental DNA (eDNA) in Microcystis aeruginosa and Prymnesium parvum. Google Scholar Abdul Manaff, A. H. N. et al. Mapping harmful microalgal species by eDNA monitoring: a large-scale survey across the southwestern South China Sea. Google Scholar Blackman, R. C. et al. Targeted and passive environmental DNA approaches outperform established methods for detection of quagga mussels, Dreissena rostriformis bugensis in flowing water. Google Scholar Danziger, A. M. & Frederich, M. Challenges in eDNA detection of the invasive European green crab, Carcinus maenas. Google Scholar Mansfeldt, C. et al. Microbial community shifts in streams receiving treated wastewater effluent. Google Scholar Yamamoto, S. et al. Environmental DNA metabarcoding reveals local fish communities in a species-rich coastal sea. Inoue, Y., Miyata, K., Yamane, M. & Honda, H. Environmental nucleic acid pollution: characterization of wastewater generating false positives in molecular ecological surveys. ACS ES&T Water 3, 756–764 (2023). A., Jerde, C. L. & Sepulveda, A. J. What do you mean by false positive. Ficetola, G. F., Taberlet, P. & Coissac, E. How to limit false positives in environmental DNA and metabarcoding? McCauley, M., Koda, S. A., Loesgen, S. & Duffy, D. J. Multicellular species environmental DNA (eDNA) research constrained by overfocus on mitochondrial DNA. Pilliod, D. S., Goldberg, C. S., Arkle, R. S. & Waits, L. P. Estimating occupancy and abundance of stream amphibians using environmental DNA from filtered water samples. Doi, H. et al. Environmental DNA analysis for estimating the abundance and biomass of stream fish. Di Muri, C. et al. Read counts from environmental DNA (eDNA) metabarcoding reflect fish abundance and biomass in drained ponds. Pont, D. et al. Quantitative monitoring of diverse fish communities on a large scale combining eDNA metabarcoding and qPCR. Nakagawa, H., Fukushima, K., Sakai, M., Wu, L. & Minamoto, T. Relationships between the eDNA concentration obtained from metabarcoding and stream fish abundance estimated by the removal method under field conditions. Fonseca, V. G. Pitfalls in relative abundance estimation using eDNA metabarcoding. Yates, M. C., Fraser, D. J. & Derry, A. M. Meta‐analysis supports further refinement of eDNA for monitoring aquatic species‐specific abundance in nature. Sepulveda, A. J. et al. It's complicated environmental DNA as a predictor of trout and char abundance in streams. Sigsgaard, E. E. et al. Population characteristics of a large whale shark aggregation inferred from seawater environmental DNA. Weitemier, K. et al. Estimating the genetic diversity of Pacific salmon and trout using multigene eDNA metabarcoding. Parsons, K. M., Everett, M., Dahlheim, M. & Park, L. Water, water everywhere: environmental DNA can unlock population structure in elusive marine species. Elbrecht, V., Vamos, E. E., Steinke, D. & Leese, F. Estimating intraspecific genetic diversity from community DNA metabarcoding data. Turon, X., Antich, A., Palacín, C., Praebel, K. & Wangensteen, O. S. From metabarcoding to metaphylogeography: separating the wheat from the chaff. Couton, M., Viard, F. & Altermatt, F. Opportunities and inherent limits of using environmental DNA for population genetics. Andres, K. J., Sethi, S. A., Lodge, D. M. & Andrés, J. Nuclear eDNA estimates population allele frequencies and abundance in experimental mesocosms and field samples. Wolf, K. K. E. et al. Revealing environmentally driven population dynamics of an Arctic diatom using a novel microsatellite PoolSeq barcoding approach. Barbour, M. T., Gerritsen, J., Snyder, B. D. & Stribling, J. US Environmental Protection Agency. Rapid Bioassessment Protocols for Use in Streams and Wadeable Rivers (US EPA, 1999). Cordier, T. et al. Ecosystems monitoring powered by environmental genomics: a review of current strategies with an implementation roadmap. Yang, J. et al. Ecogenomics of zooplankton community reveals ecological threshold of ammonia nitrogen. Nuy, J. K. et al. Responses of stream microbes to multiple anthropogenic stressors in a mesocosm study. Li, F. et al. Application of environmental DNA metabarcoding for predicting anthropogenic pollution in rivers. Blackman, R. C., Ho, H.-C., Walser, J.-C. & Altermatt, F. Spatio-temporal patterns of multi-trophic biodiversity and food-web characteristics uncovered across a river catchment using environmental DNA. Stevens, J. D. & Parsley, M. B. Environmental RNA applications and their associated gene targets for management and conservation. Bergsveinson, J. et al. Metatranscriptomic insights into the response of river biofilm communities to ionic and nano-zinc oxide exposures. Hechler, R. M., Yates, M. C., Chain, F. J. J. & Cristescu, M. E. Environmental transcriptomics under heat stress: can environmental RNA reveal changes in gene expression of aquatic organisms? Cordier, T. et al. Predicting the ecological quality status of marine environments from eDNA metabarcoding data using supervised machine learning. Keck, F., Brantschen, J. & Altermatt, F. A combination of machine-learning and eDNA reveals the genetic signature of environmental change at the landscape levels. Salis, R. K., Bruder, A., Piggott, J. J., Summerfield, T. C. & Matthaei, C. D. High-throughput amplicon sequencing and stream benthic bacteria: identifying the best taxonomic level for multiple-stressor research. Sagova-Mareckova, M. et al. Expanding ecological assessment by integrating microorganisms into routine freshwater biomonitoring. Cordier, T., Lanzén, A., Apothéloz-Perret-Gentil, L., Stoeck, T. & Pawlowski, J. Embracing environmental genomics and machine learning for routine biomonitoring. Martínez-Santos, M. et al. Treated and untreated wastewater effluents alter river sediment bacterial communities involved in nitrogen and sulphur cycling. Andújar, C. et al. Metabarcoding of freshwater invertebrates to detect the effects of a pesticide spill. Vasselon, V., Rimet, F., Tapolczai, K. & Bouchez, A. Assessing ecological status with diatoms DNA metabarcoding: scaling-up on a WFD monitoring network (Mayotte island, France). Apothéloz-Perret-Gentil, L. et al. Taxonomy-free molecular diatom index for high-throughput eDNA biomonitoring. Feio, M. J. et al. A taxonomy-free approach based on machine learning to assess the quality of rivers with diatoms. Frühe, L. et al. Supervised machine learning is superior to indicator value inference in monitoring the environmental impacts of salmon aquaculture using eDNA metabarcodes. Cordier, T. et al. Supervised machine learning outperforms taxonomy-based environmental DNA metabarcoding applied to biomonitoring. Wilkinson, S. P. et al. TICI: a taxon-independent community index for eDNA-based ecological health assessment. Zhang, Y., Zhang, X., Li, F. & Altermatt, F. Fishing eDNA in one of the world's largest rivers: a case study of cross-sectional and depth profile sampling in the Yangtze. Gold, Z., Sprague, J., Kushner, D. J., Zerecero Marin, E. & Barber, P. H. eDNA metabarcoding as a biomonitoring tool for marine protected areas. Stewart, K., Ma, H., Zheng, J. & Zhao, J. Using environmental DNA to assess population-wide spatiotemporal reserve use. McClenaghan, B. et al. Harnessing the power of eDNA metabarcoding for the detection of deep-sea fishes. Fujiwara, Y. et al. Detection of the largest deep-sea-endemic teleost fish at depths of over 2,000 m through a combination of eDNA metabarcoding and baited camera observations. van der Heyde, M. et al. Taking eDNA underground: factors affecting eDNA detection of subterranean fauna in groundwater. Savio, D. et al. Bacterial diversity along a 2600 km river continuum. De Ventura, L., Kopp, K., Seppälä, K. & Jokela, J. Tracing the quagga mussel invasion along the Rhine River system using eDNA markers: early detection and surveillance of invasive zebra and quagga mussels. Adams, A. J. et al. From eDNA to decisions using a multi-method approach to restoration planning in streams. Mahon, A. R. et al. Validation of eDNA surveillance sensitivity for detection of Asian carps in controlled and field experiments. US Fish and Wildlife Service. Quality assurance project plan eDNA monitoring of bighead and silver carps. USFWS Great Lakes region 3 (US FWS, 2022). Ellis, M. R. et al. Detecting marine pests using environmental DNA and biophysical models. Matejusova, I. et al. Environmental DNA based surveillance for the highly invasive carpet sea squirt Didemnum vexillum: a targeted single-species approach. Sepulveda, A. J. et al. When are environmental DNA early detections of invasive species actionable? Li, F. et al. Human activities' fingerprint on multitrophic biodiversity and ecosystem functions across a major river catchment in China. Lanzén, A., Dahlgren, T. G., Bagi, A. & Hestetun, J. T. Benthic eDNA metabarcoding provides accurate assessments of impact from oil extraction, and ecological insights. Suzzi, A. L. et al. eDNA metabarcoding reveals shifts in sediment eukaryote communities in a metal contaminated estuary. Stoeck, T. et al. Environmental DNA metabarcoding of benthic bacterial communities indicates the benthic footprint of salmon aquaculture. Keck, F. et al. Meta-analysis shows both congruence and complementarity of DNA and eDNA metabarcoding to traditional methods for biological community assessment. Leese, F. et al. Why we need sustainable networks bridging countries, disciplines, cultures and generations for aquatic biomonitoring 2.0: a perspective derived from the DNAqua-net COST action. Pont, D. et al. The future of fish-based ecological assessment of European rivers: from traditional EU Water Framework Directive compliant methods to eDNA metabarcoding-based approaches. Pawlowski, J., Bonin, A., Boyer, F., Cordier, T. & Taberlet, P. Environmental DNA for biomonitoring. Meyer, A. et al. Morphological vs. DNA metabarcoding approaches for the evaluation of stream ecological status with benthic invertebrates: testing different combinations of markers and strategies of data filtering. Blackman, R. C. et al. Advancing the use of molecular methods for routine freshwater macroinvertebrate biomonitoring — the need for calibration experiments. Pawlowski, J., Apothéloz-Perret-Gentil, L., Mächler, E. & Altermatt, F. Environmental DNA Applications for Biomonitoring and Bioassessment in Aquatic Ecosystems. Environmental Studies no. 2010: 71 (Federal Office for the Environment, 2020). Laamanen, T. et al. Technology readiness level of biodiversity monitoring with molecular methods – where are we on the road to routine implementation? Yang, J., Zhang, L., Mu, Y. Small changes make big progress: a more efficient eDNA monitoring method for freshwater fish. Shea, M. M. et al. Systematic review of marine environmental DNA metabarcoding studies: toward best practices for data usability and accessibility. Li, J., Lawson Handley, L.-J., Read, D. S. & Hänfling, B. The effect of filtration method on the efficiency of environmental DNA capture and quantification via metabarcoding. Deiner, K. et al. Optimising the detection of marine taxonomic richness using environmental DNA metabarcoding: the effects of filter material, pore size and extraction method. Loeza-Quintana, T., Abbott, C. L., Heath, D. D., Bernatchez, L. & Hanner, R. H. Pathway to increase standards and competency of eDNA surveys (PISCeS)— advancing collaboration and standardization efforts in the field of eDNA. Altermatt, F. et al. Quantifying biodiversity using eDNA from water bodies: general principles and recommendations for sampling designs. Water Quality — Sampling, Capture and Preservation of Environmental DNA from Water (ISO, 2023). Weigand, H. et al. DNA barcode reference libraries for the monitoring of aquatic biota in Europe: gap-analysis and recommendations for future work. Li, F. et al. Gap analysis for DNA-based biomonitoring of aquatic ecosystems in China. Briski, E., Ghabooli, S., Bailey, S. A. Are genetic databases sufficiently populated to detect non-indigenous species? Mc Cartney, A. M. et al. The European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics. Hebert, P. D. N., Floyd, R., Jafarpour, S. & Prosser, S. W. J. Barcode 100K specimens: in a single nanopore run. Pomerantz, A. et al. Real-time DNA barcoding in a rainforest using nanopore sequencing: opportunities for rapid biodiversity assessments and local capacity building. Lin, D. et al. The TRUST principles for digital repositories. Leigh, D. M. et al. Best practices for genetic and genomic data archiving. Nilsson, R. H. et al. Introducing guidelines for publishing DNA-derived occurrence data through biodiversity data platforms. Berry, O. et al. Making environmental DNA (eDNA) biodiversity records globally accessible. Yilmaz, P. et al. Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications. Meyer, R. et al. Aligning standards communities for omics biodiversity data: sustainable Darwin Core–MIxS interoperability. Abarenkov, K. et al. Publishing DNA-derived data through biodiversity data platforms v1.3 (GBIF Secretariat, 2023). Klymus, K. E. et al. The MIEM guidelines: minimum information for reporting of environmental metabarcoding data. MBMG https://doi.org/10.3897/mbmg.8.128689 (2024). Takahashi, M. et al. Best practice for publishing environmental DNA (eDNA) data according to FAIR principles. Wilkinson, M. D. et al. The FAIR guiding principles for scientific data management and stewardship. Goldberg, C. S. et al. Critical considerations for the application of environmental DNA methods to detect aquatic species. Shen, E. W., Vandenberg, J. M. & Moore, A. Sensing inequity: technological solutionism, biodiversity conservation, and environmental DNA. Carroll, S. R. et al. The CARE principles for indigenous data governance. Secretariat of the Convention on Biological Diversity. Nagoya Protocol on Access to Genetic Resources and the Fair and Equitable Sharing of Benefits Arising from their Utilization to the Convention on Biological Diversity (Convention on Biological Diversity, United Nations, 2011). Stammnitz, M. R., Hartman Scholz, A. Environmental DNA without borders: let's embrace decentralised genomics to meet the UN's biodiversity targets. Handsley-Davis, M., Kowal, E., Russell, L. & Weyrich, L. S. Researchers using environmental DNA must engage ethically with Indigenous communities. Wauchope, H. S. et al. What is a unit of nature? Measurement challenges in the emerging biodiversity credit market. Bhutta, U. S., Tariq, A., Farrukh, M., Raza, A. & Iqbal, M. K. Green bonds for sustainable development: review of literature on development and impact of green bonds. Watt, R. The fantasy of carbon offsetting. Ford, H. V. et al. A technological biodiversity monitoring toolkit for biocredits. Jarman, S. N., Berry, O. & Bunce, M. The value of environmental DNA biobanking for long-term biomonitoring. Funding is from the Swiss National Science Foundation (grant nos. are supported by the National Key Research and Development Program of China (2022YFC32021001 and 2021YFC3201003). These authors contributed equally: Florian Altermatt, Marjorie Couton, Rosetta C. Blackman. Department of Aquatic Ecology, Eawag, Swiss Federal Institute of Aquatic Science and Technology, Dübendorf, Switzerland Florian Altermatt, Marjorie Couton, Luca Carraro, François Keck & Rosetta C. Blackman Department of Evolutionary Biology and Environmental Studies, Universität Zürich, Zürich, Switzerland Florian Altermatt, Marjorie Couton, Luca Carraro, François Keck & Rosetta C. Blackman Lake Ecosystems Group, UK Centre for Ecology and Hydrology, Lancaster Environment Centre, Bailrigg, UK Aquatic Ecosystem Research, University of Duisburg-Essen, Essen, Germany Florian Leese Centre for Water and Environmental Research, University of Duisburg-Essen, Essen, Germany Florian Leese School of Ecology and Environmental Science, Yunnan University, Kunming, China State Key Laboratory of Water Pollution Control and Green Resource Recycling, School of the Environment, Nanjing University, Nanjing, China You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Overall project lead and coordination by F.A. Overall conceptualization by F.A., M.C. Discussion of content by all authors. Lead writing of article by F.A., M.C. Writing and lead for specific sections by F.A., M.C., L.C., F.K., L.L.-H., F.L., X.Z., Y.Z. Conceptualization of figures by F.A., M.C., R.C.B., L.C. Reviewing and editing of manuscript before submission by all authors. Correspondence to Florian Altermatt. directs a translation project at Nanjing University that develops apparatus for routine eDNA biomonitoring. The remaining authors declare no competing interests. Nature Reviews Biodiversity thanks David Duffy, Adam Sepulveda and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. DDBJ Sequence Read Archive: https://www.ddbj.nig.ac.jp/dra/index-e.html DEFRA DNA Center of Excellence: https://jncc.gov.uk/our-work/dna-coe/ European Nucleotide Archive: https://www.ebi.ac.uk/ena/browser/home GEO BON: https://geobon.org/ Global Genome Biodiversity Network: https://www.tdwg.org/standards/ggbn/ International Nucleotide Sequence Database Collaboration: https://www.insdc.org/ Joint Research Center: https://joint-research-centre.ec.europa.eu/ Nagoya Protocol: https://www.cbd.int/abs/default.shtml National eDNA Reference Centre: https://www.ecodna.org.au/nrc Sequence Read Archive: https://www.ncbi.nlm.nih.gov/sra Inferred unique sequence(s) derived from high-throughput sequencing after removal of erroneous sequences. Taxonomic group, such as fish, macroinvertebrates or diatoms, specifically used to assess environmental conditions in relation to legislative frameworks. Identification of a specimen using a short DNA fragment called a (genetic) marker. Approaches used to sequence millions of DNA sequences in a rapid and cost-effective manner (also known as next-generation sequencing). A DNA sequence of a gene or part of a gene with a known location in the genome used to identify specific species. Identification of the multiple organisms represented in a sample by sequencing a common DNA marker using high-throughput sequencing. An operational definition of clustered sequences based on their sequence similarity (for example, >97% similarity) to reflect approximated taxonomic units. The process used to multiply target DNA sequences in a sample to facilitate their identification. A short, single-stranded DNA sequence (~18–25 bp) used to target a region of the gene to be amplified during the polymerase chain reaction. Dye-based or probe-based PCR method that allows the quantification of target DNA at each PCR amplification cycle. Individual sequence of base pairs (here, amplified through PCR) that corresponds to a single DNA fragment. An approach in which a single species is targeted, typically using standard, quantitative or digital PCR (as opposed to metabarcoding). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Altermatt, F., Couton, M., Carraro, L. et al. Utilizing aquatic environmental DNA to address global biodiversity targets. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative You have full access to this article via your institution. Nature Reviews Biodiversity © 2025 Springer Nature Limited Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            