
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-18</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250818102941.htm'>Forget LASIK: Safer, cheaper vision correction could be coming soon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 15:24:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Millions of Americans have altered vision, ranging from blurriness to blindness. But not everyone wants to wear prescription glasses or contact lenses. Accordingly, hundreds of thousands of people undergo corrective eye surgery each year, including LASIK -- a laser-assisted surgery that reshapes the cornea and corrects vision. Michael Hill, a professor of chemistry at Occidental College, will present his team's results at the fall meeting of the American Chemical Society (ACS). ACS Fall 2025 is being held Aug. 17-21; it features about 9,000 presentations on a range of science topics. But if the cornea is misshapen, it doesn't focus light properly, resulting in a blurry image. Hill explains that "LASIK is just a fancy way of doing traditional surgery. This is what Hill and collaborator Brian Wong are exploring through a process known as electromechanical reshaping (EMR). "I was looking at living tissues as moldable materials and discovered this whole process of chemical modification." In the body, the shapes of many collagen-containing tissues, including corneas, are held in place by attractions of oppositely charged components. These tissues contain a lot of water, so applying an electric potential to them lowers the tissue's pH, making it more acidic. But one collagen-rich tissue that they were eager to explore was the cornea. In this work, the team constructed specialized, platinum "contact lenses" that provided a template for the corrected shape of the cornea, then placed each over a rabbit eyeball in a saline solution meant to mimic natural tears. They repeated this setup on 12 separate rabbit eyeballs, 10 of which were treated as if they had myopia, or nearsightedness. Additionally, in other experiments, the team demonstrated that their technique might be able to reverse some chemical-caused cloudiness to the cornea -- a condition that is currently only treatable through a complete corneal transplant. Next up is what Wong describes as, "the long march through animal studies that are detailed and precise," including tests on a living rabbit rather than just its eyeball. They also plan to determine the types of vision correction possible with EMR, such as near- and far-sightedness and astigmatism. "There's a long road between what we've done and the clinic. But, if we get there, this technique is widely applicable, vastly cheaper and potentially even reversible," concludes Hill. Anatomical variation, birth defects, trauma, and various pathologies can alter the shape, structural stability, and transparency of the cornea, thus affecting vision. Here we report our efforts to apply electromechanical reshaping (EMR) as a molecular- based, non-ablative/non-incisional alternative to laser vision refraction, using ex vivo rabbit globes. EMR relies on short electrochemical pulses to electrolyze interstitial water, with subsequent diffusion of protons into the extracellular matrix of collagenous tissues; protonation of immobilized anions within this matrix disrupts the ionic-bonding network that provides structural integrity. Optical coherence tomography (OCT), second-harmonic generation (SHG), and confocal microscopy suggest that EMR enables control over corneal contouring while maintaining the underlying macromolecular collagen structure and stromal cellular viability. Note: Content may be edited for style and length. Megaquake Triggers Rare Tsunami Caught by NASA's SWOT Satellite How Recharging the Brain's “Batteries” Restored Lost Memory Webb Telescope Spots Oldest Black Hole, Shattering Cosmic Records Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02077-6'>A defined microbial community reproduces attributes of fine flavour chocolate fermentation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 15:09:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cocoa (Theobroma cacao L.) bean fermentation is a spontaneous process involving interactions between abiotic and biotic factors that contribute to the final flavours of chocolate. Understanding these underlying interactions could enable desired flavour profiles to be reproduced under controlled conditions. Here, using bean fermentation samples from Colombian farms, we established that pH, temperature and microbiota composition, including both bacteria and fungi, influence key flavour attributes of premium chocolate. Genome-resolved metagenomics revealed that metabolic traits necessary for the development of the flavour profile of chocolate are redundantly present in the fermentation microbial community. Using a defined and metabolically competent microbial consortium, the feasibility of replicating fine flavour attributes of chocolate under controlled conditions was confirmed via omics, metabolic networks and a trained tasting panel. Our results provide the basis for the design of fermentation starters to robustly reproduce fine chocolate characteristics. The production of fermented foods, such as chocolate, relies on the metabolic activities of microbial communities. Members of these communities transform the raw substrate, cocoa (Theobroma cacao L.) beans, into the precursor for chocolate production1,2. Although after harvest, cocoa beans undergo several processing steps to produce the final chocolate, the fermentation step, however, is a spontaneous process, involving a dynamic interplay between abiotic and biotic factors, resulting in the production of metabolites that define the chocolate flavour3,4,5,6,7,8,9. Unfermented cocoa beans develop little flavour potential when roasted and tend to be bitter and astringent4,5. In contrast, well-fermented beans can exhibit complex flavour notes with reduced bitterness and astringency. Typically, cocoa bean fermentation takes place on cocoa farms, within wooden boxes, baskets or heaps covered with various materials to regulate environmental temperatures1,2. In contrast to other fermented foods such as wine10,11, cheese12 or beer13, where microbes are often intentionally introduced or controlled, the microbial communities in cocoa bean fermentation assemble spontaneously from the environment with minimal human intervention. Therefore, variations in the assembly of microbial communities for fermentation across different locations are expected14,15,16, and this can influence the chocolate quality and flavour from different cocoa origins. Although various approaches have been used to analyse microbial communities in fermenting cocoa beans17,18,19,20,21,22,23,24,25, our understanding of microbial community assembly during cocoa bean fermentation remains limited. Particularly, there is a knowledge gap regarding how abiotic and biotic factors influence fermentation outcomes and impact chocolate flavour26. This lack of knowledge limits the optimization of cocoa bean fermentation as an industrial process similar to other fermented food industries. Therefore, further studies are imperative to delve deeper into this system to unlock its full potential. We monitored abiotic factors, such as temperature and pH, during bean fermentations in Colombia over two growing seasons. We selected a farm in the Santander district, the country's leading cocoa-producing region (Extended Data Fig. Temperature changes in fermenting beans were measured daily at two depths over 7 days (Extended Data Fig. We found that the temperature of fermenting beans increased after 24 h, exhibiting a sigmoidal pattern that resembled typical microbial growth curves (Fig. Therefore, we hypothesized that this temperature increase was associated with exothermic metabolic reactions in the bean pulp, probably driven by microbial activity. a, Temperature and pH changes of the fermenting beans during fermentation. Shaded areas represent 95% confidence intervals (CIs). Temperature recordings were taken at two depths: below the surface (top) and midway through the fermenting mass (mid-box). b, Pearson correlation analysis (two-sided) between the bean temperature and testa/pulp pH. c,d, Alpha and beta diversity estimates of the bacterial (c) and fungal (d) communities in the fermentation at different time points. Microbial samples were collected from two independent fermentations. A total of 21 bacterial and 20 fungal community profiles were used. One-way ANOVA was used to discern significant differences among groups (bacteria: P = 4.2759 × 10−4; fungi: P = 1.9382 × 10−7) and means were separated using Tukey's post hoc test. Shared letters denote no significant difference (P > 0.05), whereas different letters indicate statistically significant (P < 0.05) variation between groups. Beta diversity was visualized using canonical analysis of principal coordinates (CAP). Group differences were assessed using PERMANOVA with 9,999 permutations. PERMANOVA R2 and P value are displayed. e,f, Phylogram showing the relative abundance profiles of the main bacterial (e) and fungal (f) families at different fermentation time points. g, Mantel correlation between bacterial and fungal Bray–Curtis dissimilarity matrices. The correlation was assessed using a two-sided Mantel test with the Pearson method and 10,000 permutations. The Mantel correlation coefficient (r) and corresponding P value are displayed. To reinforce this hypothesis, we measured changes in pH in the beans' testa associated with the pulp (Extended Data Fig. The bean pulp is rich in primary and secondary substrates that can be used as carbon sources by the microbiota present3,5,19,27. Subsequently, the pH increased linearly probably due to chemical transformations of substrates in the testa/pulp5. In addition, we found a positive correlation between pH changes measured in the testa/pulp and temperature changes recorded in the fermenting cocoa mass (Fig. Both parameters exhibited similar magnitudes of variation during fermentation, indicating a link between changes in pulp pH and temperature shifts (Extended Data Fig. This is consistent with chemical transformations occurring within the pulp during fermentation28,29. We next investigated pH changes in bean cotyledons, a measure more directly associated with bean fermentation (Fig. A plausible explanation for this negative correlation between the pH in the testa/pulp and cotyledons (Extended Data Fig. 1d) could be that cocoa fermentation is a compartmentalized process that progresses simultaneously in the pulp and cotyledon. Supporting this hypothesis, we found that pH changes in the testa/pulp do not explain the pH shifts detected in the cotyledon (Extended Data Fig. Nevertheless, Pearson's correlation analysis revealed a strong negative correlation (r = −0.91307, P = 5.4793 × 10−16) between bean temperature and pH changes in the cotyledon (Extended Data Fig. These results strongly support the idea that changes in temperature and pH during cocoa bean fermentation are the result of chemical reactions probably driven by the microbes present8,19,32. Overall, we conclude that bean temperature and cotyledon pH could serve as markers for bean fermentation progression. In fact, these parameters significantly correlated with colour changes in beans, which are traditionally used to assess the fermentation end point5,33,34 (Extended Data Fig. To determine whether changes in microbial composition are associated with the progression of bean fermentation, we analysed microbial community dynamics using whole-metagenome shotgun Oxford Nanopore sequencing (Extended Data Fig. We found significant changes in alpha diversity (within-sample diversity) among microbial populations across different fermentation time points (Fig. Consistent with previous studies14,35, bacterial alpha diversity generally declined as fermentation progressed (Fig. This decline in microbial diversity could be driven by metabolic changes in the pulp and bean composition because of microbial action19,32,35. These microbial activities may underlie the observed changes in temperature and pH during bean fermentation. Analysis of beta diversity (between-sample variation) revealed that both bacterial and fungal communities separate along the first axis of a canonical analysis of principal coordinates (CAP) according to fermentation time (Fig. 1c,d), indicating clear shifts in community composition between early and late fermentation time points. In general, during the first 24 h of fermentation, the bacterial community was enriched in Erwiniaceae and depleted in Enterobacteriaceae and Pectobacteriaceae (Fig. Consistent with previous studies1,2, the bacterial community composition shifted after 48 h towards enrichment in Acetobacteraceae, which remained dominant for the rest of fermentation (Fig. In the case of fungi, we detected an initial enrichment in Saccharomycetaceae, aligning with previous reports7,21,36,37,38, and a concurrent depletion in Nectriaceae and Debaryomycetaceae within the first 24 h (Fig. The relative abundance of Saccharomycetaceae increased further at later time points during fermentation (Fig. We also analysed species-level enrichment of bacterial and fungal taxa across all fermentation time points relative to time zero (Supplementary Fig. Overall, these results underscore coordination between bacteria and fungi during bean fermentation (Fig. 1g) that may be influenced by changes in pH and temperature, spatial distribution and substrate availability. This proposed model of interkingdom microbial coordination could explain observed shifts in fermentation temperature and progressive acidification of the bean cotyledon, probably driven by the metabolic activities of Lactobacillaceae and Acetobacteraceae. To begin assessing the impact of the abiotic and biotic parameters on chocolate flavour, we selected two additional farms in Colombia in the regions of Huila and Antioquia (Extended Data Fig. First, we confirmed that the cocoa varieties from all three farms had comparable genetic backgrounds (Fig. This analysis allowed us to exclude genotype as a relevant factor influencing bean fermentation and chocolate flavour in our study. Population structure was determined on the basis of 84 high-quality SNP markers. The reference SNP profiles were primarily generated from cacao accessions at the International Cocoa Genebank Trinidad and were selected across the 10 cacao genetic clusters identified in ref. Each vertical line represents an individual. b, Temperature and pH changes of the fermenting beans in Huila and Antioquia farms. Daily temperature recordings were taken at two depths within the fermentation box: 7 cm below the top surface (top) and midway through the fermenting mass (mid-box). Bean samples from these positions were collected and separated into testa with pulp and cotyledons, and the pH of both tissues was measured. c,d, Canonical analysis of principal coordinates (CAP) showing the bacteria (c) and fungi (d) community compositions in the fermenting beans across the three cocoa farms analysed in this study. Differences in beta diversity estimates are shown across fermentation time points (left), farm locations (middle) and bean harvesting periods (right). Group differences were assessed using PERMANOVA with 9,999 permutations. e,f, Phylogram displaying the relative abundance profiles of the main bacterial (e) and fungal (f) families at different fermentation time points. Next, we analysed the temperature and pH kinetics during bean fermentation at these two farms, across two consecutive harvests. Our findings showed that in general, the changes in these abiotic fermentation indicators in Huila, but not in Antioquia, resembled those observed previously at the Santander farm (Figs. This suggests that the fermenting beans from Antioquia underwent chemical transformations distinct from those of the other two farms, probably due to the presence of different microbial communities. To test this hypothesis, we characterized bacterial and fungal compositional changes in Huila and Antioquia fermentations (Extended Data Fig. Generally, the Huila and Antioquia fermentations showed microbiota characteristics similar to those in Santander (Figs. Supporting our hypothesis however, we found very different dynamics in the community in Antioquia (Figs. These differences in microbial composition dynamics in Antioquia may explain the distinct temperature and pH patterns observed. Moreover, consistent with previous studies14,15,16, distinct differences in microbiota composition were detected across geographical locations, with a stronger signal in fungal populations (Fig. 2d and Supplementary Result 5), suggesting that local environment or microbial source origin may influence fermentation characteristics. Consequently, we explored the influence of environmental microbial sources to the fermentation process (Supplementary Fig. Overall, the results suggest that bacteria harboured within fermentation box surfaces might function as a ‘memory', serving as a stable inoculation source over time, whereas initial sources of fungi for fermentation probably originate from diverse environmental sources, explaining the stronger geographic signal. These divergent contributions from environmental sources among farms, coupled with temporal disparities, may enrich flavour diversity across locations. We analysed whether changes in the identified fermentation signatures (temperature, pH and microbial composition) could offer insights into the quality of the fermented beans. We evaluated nine quality parameters39,40,41 commonly used in the chocolate industry to rapidly assess the commercial quality of fermented cocoa beans (Supplementary Fig. We found no differences in the fermentation degree quality parameters across locations, despite Antioquia fermentation signatures being very atypical (Supplementary Fig. These results suggest that the fermentation markers identified here are not predictive of the overall degree of bean fermentation. Therefore, we postulated that variations in these fermentation signatures, which are associated with chemical processes occurring within the beans, could offer insights into chocolate flavour characteristics. To validate this hypothesis, we prepared cocoa liquors from the fermented beans from the three farms and evaluated their flavours using a trained tasting panel. Cocoa beans are classified as ‘bulk' or ‘fine flavour'39,41. Fine flavour beans offer complex, desirable notes with less bitterness and are prized in premium chocolate, whereas bulk beans have a simpler cocoa flavour and are typically more bitter. Consistent with the abiotic and biotic fermentation signature changes (Figs. 1 and 2), liquors from Santander and Huila shared flavour attributes that contrasted with those from Antioquia (Fig. We observed similar organoleptic qualities between the Santander and Huila liquors and a fine flavour reference liquor from Madagascar (Fig. In contrast, the Antioquia liquor clustered with bulk references from Ivory Coast and Ghana, showing a narrower flavour range (Fig. These results underscore the pivotal role of bean fermentation, especially the dynamics of its abiotic and biotic markers defined here, as critical factors driving the development of fine flavour in chocolate. a, CAP showing the clustering of cocoa liquor sensory profiles from fermented beans produced in Santander, Huila and Antioquia. Group differences were assessed using PERMANOVA with 9,999 permutations. The proportions of sensory attribute variance explained by farm location, harvesting period and sensory panelists, estimated by PERMANOVA, are displayed on the left bar. PERMANOVA R2 and P value explained by farm location are shown. b, Heat map illustrating the sensory characteristics of cocoa liquors from the three farms. Reference liquors from Madagascar (fine cocoa) and Ivory Coast and Ghana (bulk cocoa) were included. c, Heat map showing the extracted features of the temperature and pH kinetic curves of fermenting beans from the three farms and their importance to each chocolate sensory attribute. The heat map is clustered hierarchically on the basis of sensory attributes, and selected temperature and pH features, and is coloured on the basis of feature importance determined using the random forest model. The bars at the top denote the sensory groups defined by the Cocoa of Excellence guidelines and highlight attributes typically present in bulk cocoa, fine cocoa and undesirable flavours. d, Heat maps showing extracted features from pivotal bacteria and fungi growth curves in fermenting beans and their importance to sensory attributes. Important bacteria and fungi taxa driving differences in beta diversity in cocoa fermentation were selected on the basis of PERMANOVA coefficients. The heat map is coloured on the basis of feature importance, which represents the percentage increase in mean squared error (%IncMSE) for each feature, determined using the random forest model. Bacteria are indicated in black text and fungi are illustrated in red text. The bars at the top denote the sensory groups defined by the Cocoa of Excellence guidelines for cocoa liquors and chocolate, and highlight attributes typically present in bulk cocoa, fine or flavour cocoa and undesirable flavours. To assess the predictive value of temperature and cotyledon pH, in predicting chocolate flavour attributes, we fitted their kinetic curves using a 5-parameter model across locations and identified 13 fermentation kinetic features (Extended Data Fig. To reduce redundancy, we clustered pairwise correlations and retained the most variable feature from each cluster for predicting chocolate flavour attributes (Extended Data Fig. Using these selected features, we identified two main clusters linking abiotic features to flavour notes common to both bulk and fine chocolate (Fig. Consistent with our previous results (Fig. 3a,b), we observed a strong association between fine chocolate notes, such as light wood, flowers, caramel/panela, brown and dark wood, and abiotic features, such as time to pH maximum kinetic energy, temperature inflection point and time to temperature inflection point (Fig. This indicates that highly variable features related to temperature and pH strongly predict chocolate flavour characteristics. We investigated whether microbiota composition changes during fermentation were associated with chocolate flavour attributes. We identified microbial groups driving compositional differences across fermentation time points and locations (Extended Data Fig. CAP analysis using these biotic markers recapitulated both temporal and spatial microbiota variations (Extended Data Fig. Seven growth features, capturing shifts in the microbial marker abundances, were then extracted, and redundancy was reduced using the same clustering approach applied to the abiotic traits (Extended Data Fig. We observed that changes in the features identified for certain microbes predicted distinct flavour characteristics (Fig. Notably, changes in the features identified in two fungi genera, Torulaspora and Saccharomyces, strongly associated with flavour attributes often found in fine chocolate (Fig. To further validate the predictive power of the identified markers, we analysed abiotic and microbial feature associations with flavour attributes across 19 and 11 independent natural fermentations, respectively, conducted over multiple years in diverse agroecological regions of Trinidad, a country known for fine flavour cocoa. Despite this broad panel in sensory attributes, strong and consistent associations between several identified markers and key flavour attributes emerged, many matching or exceeding those from Colombian fermentations (Supplementary Fig. These findings indicate that, alongside temperature and pH dynamics, manipulating microbial community composition and dynamics during fermentation can shape chocolate flavour, enabling the development of reproducible starter cultures for consistent, high-quality chocolate production. To guide the design of fermentation starters, we investigated whether a reduced microbial consortium, defined by key taxonomic and metabolic traits, could reproduce the essential characteristics of the cocoa fermentation process. We first constructed metagenome assembled genomes (MAGs) from the shotgun sequencing data of bacteria and fungi associated with fermentation. After removing duplicate MAGs and those that did not meet our genome completeness and contamination thresholds, 55 MAGs remained for further analysis (Fig. We confirmed that the constructed MAGs reliably captured the microbial community dynamics during fermentation (Supplementary Fig. To assess microbial functional dynamics, we identified enriched Gene Ontology (GO) terms in contigs assembled from the microbial communities that served as the source for MAG reconstruction. Consistent with microbial composition patterns (Supplementary Fig. 4d), functional analysis confirmed similar enrichment dynamics in Santander and Huila fermentations, contrasting sharply with Antioquia. Key enriched GO categories included microbial division, growth and processes typical of food fermentation, such as pH response, alcohol biosynthesis and metabolism, and responses to heat, starvation, osmotic and oxidative stresses42,43,44 (Supplementary Fig. Again, pathway and protein enrichment analyses further highlighted key enzymatic groups enriched during fermentation that probably contribute to cocoa bean flavour development, including aminotransferases (linked to aroma precursors such as pyrazines45,46), l-aspartate 4-carboxy-lyase (producing sweet and savoury flavour precursors47,48) and ethanolamine ammonia-lyase (generating fruity volatiles such as acetaldehyde3,9,49) (Supplementary Fig. Taken together, these findings suggest that the sequenced microbial communities from the fermentations and the corresponding assembled MAGs potentially capture essential metabolic pathways for cocoa bean fermentation and the development of diverse chocolate flavour attributes. However, due to the incomplete nature of some MAGs, certain metabolic pathways may still be underrepresented. a, Neighbour-joining tree showing the 55 MAGs derived from the fermenting cocoa beans metagenomic dataset. The MAGs were constructed using both single-sample assembly and co-assembly approaches, employing different binning tools. The dendrogram was constructed utilizing marker gene sequences identified across the genomes, with colours indicating the main bacterial and fungal phyla detected. b, PCoA showing the projected metabolic potential of the full microbial community (left), defined community identified using our metabolic approach (middle) and the synthetic community (SYNCOM) (right) designed to recapitulate the defined community. Genome-scale metabolic networks were reconstructed from the MAGs and from the genome sequences that were generated from the cultured microbes. Metabolites reachable are based on cocoa pulp as the precursor. The colours depict different bacteria and fungi families. c, Venn diagram depicting the number of metabolites attainable by the full, defined and synthetic microbial communities when seeded with cocoa pulp. The total number of metabolites produced by a community (n) reflects the cumulative metabolic capabilities of the individual microbes and the added value to produce new metabolites as a result of cooperation among individual microbes in the community. Overlapping sections denote metabolites shared by multiple communities, while non-overlapping areas signify metabolites exclusive to each community. d, Venn diagram illustrating the taxonomic overlap at the family level between the defined community identified in our metabolic network analysis and our synthetic community (SYNCOM). Each circle represents the total number of taxonomic families within a community. The overlapping regions highlight families shared by both communities, while the non-overlapping sections represent families unique to each. The percentages within each region denote the proportion of taxonomic families in that section relative to the total number across both communities. To evaluate the metabolic capabilities of the MAGs, we reconstructed in silico, genome-scale metabolic networks for 44 MAGs specifically recovered from fermenting beans. Using cocoa pulp substrates19,27,32,50,51 to seed the network, we predicted metabolite production for individual MAGs to assess the cumulative metabolic capacities of the MAG set, a representative proxy of the fermentation microbiota (Supplementary Fig. We observed strong taxonomic clustering, with closely related microbes producing similar metabolites (Supplementary Fig. As expected, microbes with similar metabolic capacities followed comparable abundance trajectories in the Santander and Huila fermentations but diverged markedly in the Antioquia fermentation (Extended Data Fig. These findings indicate that taxonomic and metabolic features essential for cocoa fermentation are redundantly present within the microbial community. Indeed, our metabolic network analysis pinpointed 10 MAGs with metabolic capacities equivalent to the microbial community in cocoa fermentations (Fig. To evaluate this reduced consortium, we established a collection of bacterial and fungal strains isolated from fermenting cocoa beans (Supplementary Result 13). This representative collection captured the dominant microbial families involved in Colombian cocoa fermentations (Extended Data Fig. We selected a subset of 9 strains (5 bacteria, 4 fungi; Supplementary Table 8) that approximated the taxonomy (55.6%) and metabolic potential (94.9%) of the 10 identified MAGs (Fig. We then tested the consortium in controlled cocoa fermentations using microboxes. We inoculated beans with the full consortium, individual strain dropouts (where individual strains were removed from the consortium), a randomly selected consortium, and included uninoculated controls (Extended Data Fig. Inoculated beans with the synthetic microbial communities (full, dropouts and random) generally recapitulated the pH kinetics in the testa/pulp and cotyledons observed during fermentations at Santander and Huila farms, both associated with fine flavour chocolates (Figs. In contrast, these pH shifts were absent in uninoculated beans (Extended Data Figs. Further analyses revealed that elevated temperatures and acidic conditions, typical in natural fermentations, significantly impaired the growth of consortium member strains (Extended Data Fig. These findings suggest that dynamic pH and temperature shifts may drive microbial succession and interkingdom coordination during fermentation. We reproduced key microbiota features of the Santander and Huila fermentations (Figs. Microbial cell density increased progressively during the process (Extended Data Fig. Inoculation with the full synthetic community led to a decline in bacterial and fungal alpha diversity during fermentation, mirroring farm observations (Fig. In dropout experiments, this diversity decline was lost in five bacterial and one fungal dropout combinations (Extended Data Fig. Analysis of community structure revealed that the microbial composition was primarily shaped by the initial starter formulation and fermentation time (Fig. 8), highlighting the critical role of specific strains for maintaining microbial community structure. Together, these findings demonstrate that defined synthetic starters, under controlled conditions, can replicate the key abiotic and biotic features of spontaneous farm cocoa fermentations. a,b, Bacterial (a) and fungal (b) alpha diversity estimate of in vitro fermented beans inoculated (SYNCOM) or not (No SYNCOM) with a synthetic microbial consortium. Three independent fermentations were conducted and 32 bacterial and 22 fungal community profiles were generated. ANOVA was used to identify significant differences among groups, and means were separated using Tukey's post hoc test. Letters indicate similarities and differences between groups (P < 0.05). c,d, Canonical analysis of principal coordinates illustrating bacterial (c) and fungal (d) beta diversity estimates from in vitro fermented beans. Differences in beta diversity are illustrated for the experimental treatment effect (inoculated and not inoculated; left) and the fermentation time effect (right). Group differences were assessed using PERMANOVA with 9,999 permutations. The PERMANOVA R2 and corresponding P value are shown. e,f, Phylogram showing the relative abundance profiles of the main bacterial (e) and fungal (f) families in the in vitro fermentations. g, PCA of the volatiles detected in liquors made from beans fermented in vitro in comparison to liquors from Santander, Huila and Antioquia. h, Heat maps showing the enrichment of volatiles detected in liquors made from beans fermented in vitro as well as from the three farms. Squares outlined in black are volatiles significantly enriched (red) and depleted (blue) with respect to the No SYNCOM control (q < 0.05 and log2 fold change > ±2). Volatiles are clustered according to their log2 fold change values. i, Heat map illustrating the sensory characteristics of liquors made from beans fermented in vitro in comparison to liquors from the three farms. We tested whether the reproducibility of general fermentation characteristics under controlled conditions with synthetic starters extended to metabolic transformations linked to chocolate flavour. We prepared cocoa liquors from beans fermented under controlled conditions and analysed volatile organic compounds (VOCs) associated with chocolate flavour3,9,46,52,53,54,55,56,57. We compared these to cocoa liquors from the on-farm fermentations in Santander, Huila and Antioquia. Principal component analysis revealed a clear separation between the metabolic profiles of inoculated and farm fermentations versus non-inoculated controls (Fig. VOC enrichment patterns were similar between the controlled and on-farm fermentations, but distinct from non-inoculated controls (Fig. We confirmed that changes in VOC metabolic profiles were linked to initial starter composition and evolved throughout fermentation (Extended Data Fig. Kinetic analyses across the full, dropout and random synthetic starters revealed significant differences in VOC concentrations, while VOCs in non-inoculated fermentations remained largely unchanged compared to time zero (Extended Data Fig. Non-volatile metabolite profiles also differed across the synthetic communities and diverged further over time (Extended Data Fig. At later stages, non-inoculated and random-inoculated fermentations were metabolically distinct from others (Extended Data Fig. Hundreds to thousands of metabolites showed distinct accumulation kinetics (Extended Data Fig. 9a–c), confirming limited redundancy in the metabolic capacities of the reduced starter (Extended Data Fig. These results highlight the reproducibility and tunability of flavour-associated metabolic profiles using defined synthetic communities. A trained tasting panel confirmed that liquors from beans fermented with the full synthetic community exhibited flavour notes characteristic of fine chocolates from Santander, Huila and Madagascar, clearly distinguishing them from bulk chocolate references (Fig. In contrast, dropouts, the random community and non-inoculated fermentations produced liquors with altered or diminished flavour complexity (Extended Data Fig. These results demonstrate that defined microbial starters can reliably reproduce and modulate fine chocolate traits under controlled conditions. We establish the previously suggested role of pH and temperature changes1,58 as robust predictors of chocolate flavour characteristics. In addition, we characterize microbial dynamics during bean fermentation to reveal interkingdom microbial interactions that promote flavour profile development in cocoa beans. Our study demonstrates that both abiotic and biotic fermentation markers influence the flavour characteristics of chocolate liquors, supporting previous hypotheses6,58. We expose the origins of spontaneous microbial assembly driving fermentations on farms, shedding light on flavour attributes of cocoa beans linked to specific geographic locations. Furthermore, by defining the relationship between abiotic and biotic fermentation signatures and flavour notes, our findings provide a foundation for manipulating flavour profiles of cocoa beans. These conclusions have been validated across diverse agroecological cocoa-growing regions, spanning two countries and multiple harvest seasons. We demonstrate using synthetic community dropout experiments that the metabolic capabilities required to ferment cocoa are redundantly represented within the fermentation microbial community and therefore can be encapsulated in a reduced microbial consortium under controlled conditions. While the dropout experiments, removing only one microbial strain at a time, strongly support a reduced functional redundancy present in the fermentation starter designed in this work, these dropout experiments may not be sufficient to capture higher-order metabolic interactions within the microbial community. Therefore, a fully combinatorial design in the dropout experiments may be required to convincingly demonstrate that the metabolic capabilities contributed by each starter member are unique. Similarly, due to the incomplete nature of some eukaryotic MAGs, certain metabolic pathways may be underrepresented or absent in our analysis. Therefore, we speculate that increasing the quality of eukaryotic MAGs could lead to the design of new starters with specific metabolic capabilities. These new starters are expected to generate novel chocolate flavour profiles through mechanisms similar to those described here. The results of this work expand our understanding of how the microbial community composition present in fermentation is a key determinant of chocolate flavour characteristics. We developed a robust pipeline enabling the design of fermentation starters that will contribute to the domestication of spontaneous and unpredictable microbial fermentation of cocoa occurring on farms. This sets the stage for the emergence of a modern chocolate industry akin to the beer or cheese industry, based on controlled cocoa fermentations, driven by synthetic microbial starters capable of robustly reproducing unique flavour attributes in cocoa beans and chocolate. Cocoa plantations used in this work were distributed throughout Colombia and were separated into three agroecological zones on the basis of climatic conditions, topography and soil composition. The selected agroecological zones were: (1) Santander, a mountainous region with the largest production rates of cocoa in the country, (2) Huila, an inter-Andean dry valley region and (3) Antioquia, a Pacific region. For each zone, one farm was selected for sampling: Santander (3,888 ft above sea level), Huila (3,640 ft above sea level) and Antioquia (3,993 ft above sea level) (Extended Data Fig. The Huila and Antioquia farms were located ~430 km apart, and ~550 km and 198 km away from the analysed farm in Santander, respectively (Extended Data Fig. For the farms selection, we also considered best agricultural practices, a well-established infrastructure and characteristics of cocoa fermentation. All regions had a tropical and humid climate, and the monthly average temperature during sample collection was from 22.8 °C to 28.7 °C. All cocoa bean fermentations were performed on the farms using the farmers' traditional practices and were analysed during the mid (May) and main harvests (October–November) on the three farms except for Antioquia, where only the main harvest period was characterized. Briefly, mature ripe cacao pods were harvested and opened manually in the fields. Cocoa beans and surrounding pulp were scooped out by hand and placed into pre-washed wooden fermentation boxes. Natural cocoa bean fermentation proceeded at ambient temperatures ranging from an average minimum temperature of 22 °C (night-time) to a maximum of 33 °C (daytime). In all cases, the beans were turned at 48 h and 96 h after fermentation began. To ensure the reproducibility of our observations, we only evaluated fermentation events that followed the traditional practices of local farmers who use wooden boxes to ferment cocoa beans. By maintaining these traditional protocols, we minimized the risk of introducing experimental bias (other types of practices such as sacks, bamboo baskets, plastic baskets, styrofoam and others) that could have altered the natural fermentation trajectory and influenced the final flavour profiles. Consequently, this consistency helped in achieving reproducibility of the fermentation process and flavour profiles under the experimental conditions. The temperature of the fermentation mass within the boxes was recorded daily using a Brannan digital thermometer (Extended Data Fig. Measurements were taken at three different zones in the box (bottom left corner, middle and top right corner) at two depths: (1) 7 cm below the top surface of the beans and (2) midway through the fermenting mass. For pH measurement, three beans were collected from the boxes, 7 cm below the top surface in each of the three selected fermentation zones. Testa (seed coats) covered with pulp were separated from the cotyledons of the beans and both tissues (testa/pulp and cotyledons) were macerated in 10 ml distilled water using a mortar and pestle. Then, the pH of the suspensions was determined using a Hanna Checker H198103 pH tester. The colour changes of the beans were determined using images taken daily with a Samsung SM-G9600 camera in automatic mode. RGB values were extracted from the bean images using ImageJ v.1.54d. A minimum of eight points on each bean image were selected for each analysed time point. Greyscale and luminance values were derived using the formulas (R + G + B)/3 and 0.299 R + 0.587 G + 0.114B, respectively. Samples for the microbial community analyses were collected from the fermenting beans daily using a Zymo Collection Swab (R1104) (Extended Data Fig. The swab buds were then placed in Zymo DNA/RNA Shield Lysis and Collection tubes (R1104), and the tube contents were mixed by vigorous shaking for 10 s. A total of n = 66 fermentation samples were collected across the three farms. Using the same protocol, swab samples were also collected from various environmental sources on the farms, including the surface of cacao leaves and pods, the inner surface of the fermentation box, the hands of farm workers involved in scooping beans and transferring them to the fermentation box, and their pod cracking tools. Soil samples were collected by discarding the top 7 cm of soil and transferring 150–250 mg of soil into Zymo DNA/RNA Shield Lysis and Collection tubes using a clean spatula. To analyse the microbiota on fruit flies around the fermentation, fruit flies were caught and incubated in 0.5 ml Zymo DNA/RNA Shield Lysis solution with agitation for 5 min. The flies were then removed, and the solution was transferred into a Zymo DNA/RNA Shield Lysis and Collection tube with an additional 0.5 ml of lysis solution. In total, n = 70 farm environment samples were collected for metagenomic analysis across the three farms. Clean swabs were used as control samples without DNA. Samples for microbial community analysis were homogenized using the SuperFastPrep-2 cell lysis homogeniser (MP Biomedicals) at maximum speed for 10 s and were subsequently centrifuged for 1 min and 30 s at 9,800 × g. DNA was extracted using the ZymoBIOMICS DNA Miniprep kit (Zymo, D4300) following manufacturer instructions, and the resulting DNA concentration was determined using a Qubit fluorometer (Thermo Fisher). DNA sequencing libraries were prepared using the Rapid PCR Barcoding kit (SQK-RPB004) from Oxford Nanopore Technologies (ONT). In brief, 1 µl fragmentation mix was added to 3 µl DNA (2–10 ng µl−1), and the reaction was mixed by gentle finger-flicking. The tube was placed in a miniPCR mini16 thermal cycler (Amplyus) and the DNA was fragmented using the following conditions: 30 °C for 1 min, then 80 °C for 1 min. The fragmented DNA was cooled and amplified in a PCR reaction containing 20 µl nuclease-free water, 25 µl LongAmp Taq 2× master mix (New England Biolabs (NEB), M0287L), 4 µl fragmented DNA and 1 µl barcode adaptor. The reaction was gently mixed and amplified using the following conditions: 95 °C for 3 min, 20 cycles of denaturation at 95 °C for 15 s, annealing at 56 °C for 15 s and extension at 65 °C for 6 min, and a final extension of 65 °C for 6 min. The resulting DNA library was purified using 0.6× Agencourt AMPure XP beads (Beckman Coulter, A63881) and eluted in 10 µl 10 mM Tris-HCl pH 8.0 and 50 mM NaCl. The library concentration was determined using a Qubit fluorometer (Thermo Fisher). Equimolar quantities of individual barcoded sample libraries were pooled and the volume adjusted to 10 µl using 10 mM Tris-HCl pH 8.0 and 50 mM NaCl. Subsequently, 1 µl of Rapid Adapter solution was added to the pooled library and the tube was incubated at room temperature for 5 min. Then, 34 µl sequencing buffer, 25.5 µl loading beads and 4.5 µl nuclease-free water were added to the tube, and the contents were mixed gently. The prepared pooled library was added to a verified and primed FLO-MIN106 R9.4.1 flow cell (ONT, FLO-MIN106D) in a MinION DNA sequencer (ONT) following manufacturer instructions. DNA sequencing was conducted with default parameters using MinIT (ONT) with MinKNOW v.2.1.12 (ONT). Fast5 files were base called with Guppy v.4.0.15 using the ‘template_r9.4.1_450bps_hac.jsn' high-accuracy model (ONT). Temperature and pH distributions were compared using the Kolmogorov–Smirnov test with the ks.test() function from the stats v.4.3.0 package in R and plotted with ggplot2 v.3.4.2. To explore the correlations between temperature and pH in the testa/pulp and cotyledons, scatterplots with correlation coefficients and P values were generated using the stat_cor() function from the ggpubr v.0.6.0 package. To assess dissimilarities in bean colour at the different time points during fermentation, principal component analysis (PCA) was performed with the prcomp() function in R, and the first two principal components were plotted using ggplot2 v.3.4.2. In addition, scatterplots with correlation coefficients and P values were employed to explore correlations between bean colour values, temperature and pH. We obtained a total of 29,714,777 base-called reads (97.7 Gbp) from the whole-metagenome shotgun sequencing across the three farms. The initial dataset underwent demultiplexing, and primer and barcode sequences were trimmed using qcat v.1.1.0 (ONT). Reads with ambiguous barcode assignments were excluded from further analysis. The reads were filtered with NanoFilt (v.2.8.0)59 to discard low quality sequences (Q-score < 9) and sequences <100 bp. Reads were mapped to the Theobroma cacao Criollo v.2.0 reference genome60 as well as the Homo sapiens reference genome GRCh38.p14 (RefSeq GCF_000001405.40) using minimap2 (v.2.17)61 to identify and remove potential contaminating DNA in samples. Reads that mapped were removed using SAMtools (v.1.9)62 and Seqtk v.1.3 scripts. Following these processing steps, we retained 24,300,820 reads (80.8 Gbp) with an average read length of 3,326.4 bp (Extended Data Fig. 1h) and a mean read quality of Q13.2. The reads were classified using the Kraken 2 archaea, bacteria, viral, plasmid, human, UniVec_Core, protozoa and fungi reference databases (k2_pluspf_20220607). To estimate relative abundances, the Bracken v.2.7 pipeline64 was applied to the classification results. Subsequently, Pavian v.1.0 facilitated the extraction of abundance and taxonomic tables. To assess alpha diversity across the samples, we calculated the Shannon diversity index using phyloseq v.1.44.0. We used analysis of variance (ANOVA) to test for significant differences in Shannon diversity indices between groups, and means were separated using Tukey's honestly significant difference (HSD) test from the agricolae v.1.3.5 R package. For beta diversity, Bray–Curtis dissimilarity matrices were calculated using the phyloseq v.1.44.0 ‘bray' method, and the variances explained by fermentation time, farm location and harvest season were estimated by performing permutational multivariate analysis of variance (PERMANOVA) using the adonis2() function in the vegan v.2.6.4 R package. Unconstrained and constrained ordination of beta diversity was plotted using principal coordinate analysis (PCoA) and canonical analysis of principal coordinates (CAP), respectively, on the basis of Bray–Curtis dissimilarity matrices calculated with vegan v.2.6.4. We visualized differences in fermentation time, farm location and harvest season with the CAP analysis, using the following models: The relative abundance of taxa was plotted as a stacked bar representation using phyloseq v.1.44.0. The tax_glom() function in phyloseq v.1.44.0 was used to agglomerate taxa, and the aggregate_rare() function in microbiome v.1.22.0 was used to aggregate rare groups. Mantel correlations between bacterial and fungal Bray–Curtis dissimilarity matrices were performed using the mantel() function of vegan v.2.6.4, with the Pearson method and 10,000 permutations. We used DESeq2 (v.1.40.0)65 to calculate the enrichment profiles at Santander by fitting a generalized linear model (GLM) with the following design: To elucidate the genetic backgrounds of the cacao varieties cultivated across the three cocoa farms, we first conducted a survey focusing on diverse fruit morphologies to gauge the diversity present. The parameters assessed encompassed fruit characteristics such as form, basal constriction, apex, rugosity, ridging, length, diameter, wall thickness and the colour of the mature unripe fruit. Following this morphological survey, young and healthy leaf samples were collected from each distinct morphotype identified (ntotal = 24; Santander n = 12, Huila n = 5 and Antioquia n = 7). The leaf samples were washed and dried, and DNA isolation was carried out using a ZR Plant/Seed DNA MiniPrep kit (Zymo, D6020) with modifications detailed in ref. The samples were genotyped at 96 SNP sites (Supplementary Table 1) on a Fluidigm Juno System using a Juno 96.96 Genotyping IFC (Standard BioTools) in accordance with manufacturer instructions. The SNP sites analysed were selected from the Theobroma cacao global reference SNP panel67,68. Briefly, genotyping assays were prepared using the Juno SNP Type Genotyping Reagent kit (Standard BioTools, 100-8364) and specific SNP type genotyping assays manufactured by Standard BioTools. The Juno 96.96 thermal cycling protocol included a multiplex specific target amplification (STA) step before the SNP genotyping to enrich the template molecules. For SNP genotyping, reactions were initiated at 95 °C for 10 min, followed by 4 cycles at 95 °C for 15 s, 64–61 °C (1 °C decrease with each cycle) for 45 s, and 72 °C for 15 s. This was followed by 39 cycles at 95 °C for 15 s, 60 °C for 45 s, and 72 °C for 15 s. Fluorescence intensity was quantified using the Fluidigm EP1 software (Standard BioTools), and genotypic calls were automatically made using Fluidigm SNP Genotyping Analysis software v.4.1.3 (Standard BioTools). SNP data generated are available in Supplementary Table 2. The fermented beans were spread on a wooden surface in a 3–4-cm layer for sun drying. The drying mass was mixed every 1.5 h for the first 3 days of drying and every 3 h from the fourth day onward. Drying was carried out carefully to ensure that off-flavours did not develop. The beans were covered overnight and during rainy conditions. All batches underwent drying until reaching a final moisture content of 7%, taking ~7–8 days, after which they were stored in jute bags. Quality assessments were conducted on 100-g samples from each bean lot using cut tests following the procedure outlined in ref. The evaluation included identifying characteristics such as underfermented (purple/violet), overfermented (grey/slaty), insect-damaged, chopped/broken, germinated, mouldy, double, or flat bean defects. Beans categorized as fully fermented with no defects and those that were partially purple were considered desirable/high quality. For the roasting process, the beans were placed on shallow perforated trays and roasted at 140 °C for 25 min a convection oven (Sheldon Manufacturing). Subsequently, the beans were cooled to ambient temperature, broken and winnowed to produce cocoa nibs. The nibs were transformed into cocoa liquor on a granite-wheeled melangeur (CocoaTown), reducing the particle size to 16–18 μm. Liquor samples (60 °C) were evaluated through coded, randomized tastings by 3–6 trained sensory panelists in duplicate or triplicate. The panelists consisted of members of the Food Technology, Quality and Sensory Evaluation team at the Cocoa Research Centre, Trinidad. The panel members were trained in accordance with the Cocoa of Excellence Programme guidelines69 under the supervision of the panel leader, and are experienced in cocoa sensory evaluation. Flavour descriptors assessed by the panel members were based on the cocoa liquor sensory evaluation template of E.S. Reference liquors from Madagascar (fine or flavour cocoa) and Ivory Coast and Ghana (bulk cocoa) were included in the sensory analysis. To neutralize palates between tastings, soda crackers and mouth rinsing with still water were employed. SNP genotypic data were generated at 96 SNP sites for each morphotype in the study (ntotal = 24; Santander n = 12, Huila n = 5 and Antioquia n = 7). This dataset was then combined with the SNP profiles from 228 cacao reference accessions sourced from the Cocoa Research Centre, Trinidad, SNP database. The reference SNP profiles were primarily generated from cacao accessions at the International Cocoa Genebank Trinidad and were selected across the 10 cacao genetic clusters identified in ref. In addition, 30 Amelonado-Criollo hybrid accessions were included. The combined dataset was filtered by removing SNPs with >10% missing data and monomorphic SNPs. The result was a final dataset of 84 high-quality SNP markers, with a missing data range between 0 and 3.57% and a mean of 0.52% across all accessions. For phylogenetic analysis, SNP profiles were converted into DNA strings, aligned using DECIPHER v.2.24.0 and transformed into a distance matrix with seqinr v.4.2.16. A neighbour-joining tree was constructed with ape v.5.6.2, and the resulting tree was visualized using ggtree v.3.8.0 with ggtreeExtra v.1.10.0. The genetic distances were further analysed through a PCoA for visualization. The PCoA involved converting genetic distances to 2 dimensions using classical multidimensional scaling with the stats v.4.3.0 package, and plotting with ggplot2 v.3.4.2. Ancestry was inferred using STRUCTURE (v.2.3.4)72, employing structure-threader (v.1.3.10)73 for parallelized runs across multiple CPU cores. To facilitate the analysis, reference accessions were replicated to ensure a minimum representation of 60 individuals for each of the 10 cacao genetic groups. Simulations were calculated using the admixture model with alpha inferred and independent allele frequency with 200,000 burn-ins and 500,000 Monte Carlo Markov Chain repetitions without any previous genetic or geographic origin information. The number of clusters (K) was set from 8 to 12 with 30 iterations for each K value. CLUMPAK (Cluster Markov Packager Across K) (v.1.1)74 was utilized to assess the congruence among independent STRUCTURE runs for each K value, and the optimum K value was determined according to ref. To explore how the surrounding microbial environmental sources in the cocoa plantations may be contributing to cocoa fermentation communities, we used FEAST v.0.1.0 to perform community-wide microbial source tracking analysis. Bean quality assessments were analysed using Fisher's exact test with the fisher.test() function of the stats v.4.3.0 package. For the analysis of sensory attributes in the cocoa liquors, we estimated the variances explained by farm location, harvesting period and sensory panelists by performing PERMANOVA using distance matrices with the adonis2() function in the vegan v.2.6.4 R package. This analysis allowed us to examine how location, harvest and panelist, and their interactions, contributed to the variation in the sensory data (Fig. A constrained ordination of the sensory attributes was plotted using CAP with vegan v.2.6.4 using the following model: To illustrate the sensory characteristics of individual cocoa liquors, the mean scores for each sensory attribute were calculated across panelists for each liquor sample. These scores were then transformed to a scale between 0 and 6 using the rescale() function of the scales v.1.2.1 R package. The transformed scores were visualized on a heat map generated with ggplot2 v.3.4.2. This involved randomly selecting a minimum of 15 temperature versus time data points and fitting the temperature curve using a 5-parameter general model equation. From the model, we derived several key features including: (1) maximum growth rate (peak rate at which the system grows during a specified period); (2) time to maximum kinetic energy (duration for the system to reach its maximum kinetic energy level); (3) temperature at maximum kinetic energy (the specific temperature value at the point of maximum kinetic energy); (4) exponential phase duration (period of rapid increase in numbers or activity); (5) linear phase duration (phase where the kinetics rate becomes relatively constant); (6) exponential decay phase duration (timeframe when the system starts to decline after reaching its maximum kinetics); (7) temperature change during exponential phase (change in temperature during rapid exponential kinetics); (8) temperature change during linear phase (alterations in temperature during stable kinetics); (9) temperature change during exponential decay (variations in temperature during the decline following exponential kinetics); (10) rate of temperature change during the exponential phase (speed at which temperature changes during rapid exponential kinetics); (11) rate of temperature change during the linear phase (speed at which temperature changes during stable kinetics); (12) time to inflection point (duration for the system to reach the inflection point, indicating a shift in kinetics pattern); and (13) inflection point (point on the kinetics curve where the curvature changes, signifying a transition in kinetics rate or pattern). The entire process was repeated at least three times for each farm's fermentation. In addition, we extended our analysis to model the kinetics of the inverse cotyledon pH, extracting similar curve features. The mean normalized feature values were then visually represented on a heat map using ggplot2 v.3.4.2. Subsequently, Pearson correlation coefficients and corresponding P values between these features were computed using the rcorr() function in the Hmisc v.5.0.1 package. Significant correlations (P < 0.05) were emphasized with black squares on the plots. Furthermore, the coefficient of variation for the feature values was calculated and depicted using ggplot2 v.3.4.2. For each cluster identified, we selected the feature with the highest coefficient of variation as a representative for the cluster. Following this, for each sensory attribute, we employed the randomForest v.4.7.1.1 R package to construct a random forest model. This was done to pinpoint the most significant features associated with each sensory attribute. Subsequently, we visualized the percentage increase in mean squared error (%IncMSE) for each feature by generating a heat map using ggplot2 v.3.4.2. To identify taxonomic markers associated with the sensory attributes of the cocoa liquors, we began by pinpointing the pivotal bacteria and fungi responsible for the notable variances in beta diversity, specifically focusing on those exerting the most influence on beta diversity disparities observed among fermentation time points and across the three farm locations. To identify the top bacteria and fungi driving the differences in beta diversity across fermentation time and farm location, we calculated PERMANOVA coefficients of the taxa using the adonis() function in the vegan v.2.6.4 R package at the genus level, and assessed their prevalence with microbiome v.1.22.0. The results were visualized using ggplot2 v.3.4.2. Bacteria with coefficients ≥10 and fungi with coefficients ≥5 were selected for further analysis. Subsequently, the relative abundances of the selected bacteria and fungi were extracted from the metagenomic dataset for each fermentation across the three farms and different fermentation time points. Following this, we utilized the gcplyr v.1.5.2 R package to extract growth features of the selected taxa. The extracted features encompassed: (1) first local maxima (the initial peak density achieved during growth before a subsequent decline); (2) initial density (starting density of taxa, corresponding to the first local minima in taxa density); (3) area under the curve (overall taxa growth); (4) maximum density (peak taxa density, offering insights into the taxa carrying capacity within a specific environment, or alternatively, measures of taxa growth yield or efficiency); (5) time to maximum density (duration taken to reach the maximum taxa density in the environment); (6) midpoint (the moment when the density initially reaches half of the maximum density); and (7) inflection point (the instance when the derivative of the growth curve attains its maximum value). The mean normalized feature values were then visually represented on a heat map using ggplot2 v.3.4.2. Subsequently, Pearson correlation coefficients and corresponding P values between these features were computed using the rcorr() function in the Hmisc v.5.0.1 package. Significant correlations (P < 0.05) were emphasized with black squares on the plots. Furthermore, the coefficients of variation for the feature values were calculated and depicted using ggplot2 v.3.4.2. For each cluster identified, we selected the feature with the highest coefficient of variation as a representative for the cluster. Following this, for each sensory attribute, we employed the randomForest v.4.7.1.1 R package to construct a random forest model. This was done to pinpoint the most significant features associated with each sensory attribute. Subsequently, we visualized the percentage increase in mean squared error (%IncMSE) for each feature by generating a heat map using ggplot2 v.3.4.2. Hierarchical clustering of the feature importance was applied using the ward.D2 method based on Euclidean distances. To assess the robustness of the association between abiotic and taxonomic markers and sensory attributes in cocoa liquors, we modelled the kinetics of temperature and pH changes during bean fermentation across 19 independent fermentations, conducted on cocoa farms in diverse agroecological regions of Trinidad between 2018 and 2022. Farms and fermentation events were selected to capture the full spectrum of cocoa flavour profiles found in Trinidad, a country renowned for producing high-quality fine or flavour cocoa beans. From the kinetic curves, we extracted the following features: temperature inflection point, time to temperature inflection point, duration of the temperature exponential phase, duration of the temperature exponential decay phase, rate of temperature change during the exponential phase, pH exponential decay phase duration, and pH change rate during the exponential phase. In addition, the relative abundances of the selected bacterial and fungal taxa were extracted from metagenomic data of 11 fermentations. Subsequently, growth curves were plotted as described earlier, and the following growth features were extracted: area under the curve, inflection point, initial density and midpoint. As before, we constructed random forest models to identify the most important abiotic and taxonomic features associated with each sensory attribute. Feature importance was visualized using heat maps, displaying the percentage increase in mean squared error (%IncMSE). Various strategies were utilized to construct the MAGs. Initially, a single-sample assembly and binning approach was adopted, where reads from individual samples were assembled into contigs using metaFlye77 in the Flye v.2.9 package with default mode. Reads from each sample were subsequently mapped to the respective assembly using minimap2 (v.2.17)61, and the corresponding abundance files were generated using SAMtools (v.1.12)62. The abundance files were used for metagenomic binning of the contigs using two different binning tools: MaxBin (v.2.2.4)78 with default parameters and MetaBAT (v.2.15)79 with specific parameters (percentIdentity=85, minContigLength=1000, minContigDepth=1). To help capture low-abundance microbes, a co-assembly and binning approach was implemented. This involved pooling reads from fermentation samples within each region (Santander, Huila, Antioquia), assembling contigs and generating metagenomic bins. In addition, a concatenation of reads from fermentation samples across all three farms was performed, followed by contig assembly and binning. These diverse strategies resulted in the construction of 1,591 MAGs. Subsequently, MAGs underwent dereplication using dRep (v.3.4.0)80, with genome filtering options set at 10,000 bp minimum length, 10% minimum completeness, 10% maximum contamination and 95% average nucleotide identity (ANI) threshold for species-level dereplication (see ref. The dereplicated MAGs were evaluated using CheckM (v.1.1.6)82 to determine their genome completeness and contamination levels. MAGs were assigned to be either low, medium or high quality based on the completeness and contamination levels recommended by ref. After excluding MAGs with contamination levels >10% and those with completeness <50% for bacteria or 30% for fungi, 55 MAGs were retained for further analysis. The completeness and contamination statistics for the final MAG set were visualized using ggplot2 v.3.4.2 (Supplementary Fig. 4a,b), and additional quality and genome statistics can be found in Supplementary Table 4. To assess how well the MAGs represented the fermentation and farm environment communities, sequence reads were mapped to the MAGs using minimap2 (v.2.17)61, and mapped reads were identified with SAMtools (v.1.9)62 and Seqtk v.1.3 (Supplementary Fig. This process entailed identifying open reading frames (ORFs) within each contig, followed by mapping the predicted ORFs against the NCBI NR protein database. Taxonomic classification of the MAGs can be found in Supplementary Table 4. The marker gene sequences for each MAG were initially merged into a string and aligned using Clustal W in the msa v.1.32.0 package. The alignment was trimmed with microseq v.2.1.6, transformed into a distance matrix with seqinr v.4.2.16, and a neighbour-joining tree was constructed with ape v.5.6.2. The resulting tree was visualized using ggtree v.3.8.0 with ggtreeExtra v.1.10.0 (Fig. The ribosomal RNA (rRNA) genes in the MAGs were identified using Barrnap v.0.9 (https://github.com/tseemann/barrnap). Open reading frames from each MAG were predicted using FragGeneScanRs (v.1.1.0)85 with default settings. Functional annotation of predicted proteins was performed using eggNOG-mapper (v.2.1.9)86 with the eggNOG v.5.0.2 database87 with Diamond (v.2.0.11)88 and MMseqs2 release 12-113e3 (ref. To assess the relative abundance of the MAGs during fermentations, reads from each fermentation sample across the three regions were mapped to the MAGs using minimap2 (v.2.17)61, and CoverM (v.0.6.1)90 was used to extract the relative abundance counts with the ‘genome' mode and transcripts per million coverage method. Significant differences between comparisons (q value < 0.05) with log2 fold change > ±2 were highlighted with black squares (Supplementary Fig. To identify metabolic processes that were enriched within the microbial communities during the fermentation, we selected the contigs assembled from individual samples (380,365 contigs) and subsequently subjected them to deduplication using the dedupe.sh tool in BBTools v.38.76 to eliminate redundancies. Taxonomic classification of the contigs was performed using the CAT v.8.22 taxonomic classification pipeline84. Subsequently, the contigs were filtered to retain only bacterial and fungal sequences. DESeq2 (v.1.40.0)65 was utilized to determine the contig enrichment profiles in the three farms by fitting a GLM with the following design: We extracted the following comparisons from the fitted model: 48 h vs 0 h, 72 h vs 0 h and 96 h vs 0 h. Contigs meeting the criteria of an FDR-adjusted P value (q value) < 0.05 and a log2(fold change) > ±2 were selected for further analysis. Open reading frames encoded within the contigs were predicted using FragGeneScanRs (v.1.1.0)85 with default settings. This was followed by functional annotation of the predicted proteins using the eggNOG-mapper (v.2.1.9)86 pipeline with the eggNOG v.5.0.2 database87 with Diamond (v.2.0.11)88 and MMseqs2 release 12-113e3 (ref. Enriched Kyoto Encyclopedia of Genes and Genomes (KEGG) biochemical reactions, along with associated enzymes and proteins, were identified on the basis of an adjusted P-value threshold of <0.01 and visualized using a heat map generated with ggplot2 v.3.4.2. The genes annotated with Gene Ontology (GO) classifications were subsequently extracted, and a GO enrichment analysis focusing on biological processes was conducted. This involved employing adaptive GO clustering in conjunction with Mann–Whitney U testing, utilizing the GO_MWU tool as previously described91. In this analytical approach, genes were ranked on the basis of signed log2 fold change values. Significantly enriched and depleted GO categories were determined by an adjusted P value < 0.05 (Supplementary Table 5). The most prominent enriched and depleted GO categories shared across comparisons were visualized in ggplot2 v.3.4.2 and coloured on the basis of the square root transformed delta rank values (enrichment score) of the GO categories (Supplementary Fig. Similarly, enriched biochemical reactions annotated in the KEGG database, and proteins, were identified using a generalized linear model with an adjusted P-value threshold of <0.05 (Supplementary Tables 6 and 7). Prominently enriched and depleted categories shared across comparisons were visualized as heat maps (Supplementary Fig. For each MAG, we combined the predicted coding DNA sequence and corresponding translated amino acid sequences identified with FragGeneScanRs (v.1.1.0)85 with the functional annotations predicted by eggNOG-mapper (v.2.1.9)86, into genbank-formatted files using emapper2gbk (v.0.3.0)92 in ‘genes mode'. Subsequently, these files were utilized to generate the metabolic network of the fermentation community using the Metage2Metabo (v.1.5.3)92,93 graph-based metabolic analysis pipeline. Briefly, genome-scale metabolic networks (GSMNs) were reconstructed for the 44 MAGs detected in the fermenting bean using the Metage2Metabo (v.1.5.3)92,93 pipeline with Pathway Tools (v.26.0)94. Metabolites known to be present in cocoa pulp (artificial pulp components: 0.14% (w/v) high-viscosity carboxymethyl cellulose, 0.77% (w/v) low-viscosity carboxymethyl cellulose, 1.09% (w/v) pectin, 2.5% (w/v) sucrose, 4% (w/v) glucose, 5% (w/v) fructose, 1% (w/v) citric acid, 0.5% (w/v) yeast extract, 0.5% (w/v) peptone, 0.1% (w/v) calcium lactate pentahydrate, 0.1% (v/v) Tween 80, 0.05% (w/v) magnesium sulfate heptahydrate and 0.02% (w/v) manganese sulfate monohydrate) (see refs. 27,50) were used as metabolic precursors to seed the network. The added value of cooperation was used as the metabolic target to compute the key species and a defined community. Defined microbial communities were then identified by simplifying the complexity of the full community into a defined community with equivalent metabolic capabilities. This matrix was visualized using ComplexHeatmap v.2.12.1 with circlize v.0.4.15 (Supplementary Fig. To visualize changes in the abundance of microbes with different metabolic potential over time, the data were transformed into a distance matrix using the dist() function in R95 with the ‘euclidean' method and converted to two dimensions using classical multidimensional scaling of the dissimilarities with the stats v.4.3.0 package. Results were plotted with ggplot2 v.3.4.2 and coloured on the basis of the relative abundance of each MAG during the fermentation at each time point for each region (Extended Data Fig. Finally, our metabolic network analysis identified 10 MAGs possessing metabolic capabilities equivalent to the full community. The metabolites reachable by the 10 MAGs, based on the cocoa pulp as the precursor, were compiled and transformed into a distance matrix, then converted to two dimensions and plotted with ggplot2 v.3.4.2 as described above. We used ggvenn v.0.1.10 to visually represent the count of metabolites generated by the microbial communities, as well as to demonstrate the overlap of metabolites shared among them. We isolated bacterial and fungal strains from fermenting cocoa beans from the Cocoa Research Centre's fermentation facility at the University of the West Indies in Trinidad. For this, mature ripe cacao pods were harvested from the International Cocoa Genebank, Trinidad (ICGT), and opened manually in the field by trained staff. The beans and surrounding pulp were scooped out by hand, placed into clean plastic bags and transported to the Cocoa Research Centre fermentation facility. The beans were placed into a pre-washed wooden fermentation box, covered with jute bags to minimize heat loss, and allowed to undergo natural fermentation at ambient temperatures ranging from a minimum of 22 °C (night-time) to a maximum of 34 °C (daytime). Using sterile surgical gloves, two beans were collected daily from the fermentation mass at a depth of 7 cm at the centre of the fermentation box. These beans were cut into small pieces with a sterile scalpel blade. A few pieces of each bean were placed into tubes containing 80% glycerol, and the tubes were stored at −80 °C until further used. Samples (1 ml) of the homogenate were serially diluted (1/100, 1/1,000, 1/10,000, 1/100,000) with 10 mM MgCl2, from which 100-µl aliquots of the dilutions were plated on different selective agar media. The selective media included: (1) acetic acid medium (AAM): 1% (w/v) d-glucose (Sigma-Aldrich, G7021-5KG), 0.5% (v/v) ethanol, 0.3% (v/v) acetic acid, 1.5% (w/v) bacteriological peptone (Millipore) (Sigma-Aldrich, 91249-500 G), 0.8% (w/v) yeast extract (Millipore) (Sigma-Aldrich, 70161-500 G), 2% (w/v) agar (Sigma-Aldrich, A6686-500G), pH 4.5 with nystatin (30 mg l−1) (Sigma-Aldrich, N3503-25MU) and penicillin (50 mg l−1) (Sigma-Aldrich, 13752-5G-F); (2) De Man Rogosa Sharpe agar (MRS) (Millipore)96 (Sigma-Aldrich, 69964-500 G) with 0.1% (v/v) Tween 80 (Sigma-Aldrich, P4780-100ML) and nystatin (30 mg l−1); (3) yeast peptone glucose agar (YPG): 1% (w/v) yeast extract, 2% (w/v) peptone, 2% (w/v) glucose, 2% (w/v) agar, pH 5.6 with chloramphenicol (100 mg l−1) (Sigma-Aldrich, C1919-25G); and (4) nutrient agar (NA) (Millipore) (Sigma-Aldrich, 70148-500G). These media were optimized for the culture of the main taxonomic groups present in fermenting cocoa beans. The plates were incubated at various temperatures (25 °C, 30 °C, 37 °C and 42 °C) for 1–3 weeks. Colonies with distinct morphologies based on colony appearance, colour, optimal growth temperature and growth rate were selected and purified through successive subculturing until no visible signs of contamination were observed. The purified isolates were grown in liquid culture, mixed at a ratio of 1:1 with 80% (v/v) glycerol and stored at −80 °C for future use. To identify the isolates, we conducted amplification and sequencing of the 16S rRNA gene for bacteria and the internal transcribed spacer (ITS) region for fungi that were cultured. A single colony of each isolate was inoculated into LB media (500 µl) and incubated overnight at 28 °C with agitation at 200 r.p.m. Subsequently, 10 µl of the culture was heated at 95 °C for 5 min and then centrifuged at 10,000 × g for 1 min. The reactions were initially heated to 94 °C for 3 min, followed by 30 cycles of denaturation at 94 °C for 30 s, annealing at the optimized primer temperature (16S rRNA: 58 °C, ITS: 64 °C) for 30 s, extension at 72 °C for 1 min and 30 s, and a final extension at 72 °C for 10 min. To confirm the specific amplification of target DNA regions, half of the PCR volume was visualized on an agarose gel via electrophoresis. The resulting amplicons were prepared for sequencing by combining 14 µl of Milli-Q water, 1 µl of the PCR reaction and 2 µl of sequencing primer (10 pmol µl−1). Sanger sequencing of the PCR products was conducted using the 8F and ITS1 primers for bacteria and fungi, respectively, and amplicons were sequenced at Eurofins Genomics. This allowed us to characterize the individual metabolic capabilities of each isolate and, in turn, understand their collective metabolic capabilities. To accomplish this, each isolate was cultivated in its respective medium. This involved inoculating 3 ml of the medium with a single colony and incubating under optimal conditions until saturation. The cells were collected through centrifugation at 4,000 × g for 10 min, and the supernatant was discarded. The collected cells were resuspended in 3 ml 10 mM MgCl2, subjected to centrifugation as before and finally resuspended in 1 ml 10 mM MgCl2. The suspension was transferred to a 2-ml tube with glass beads (150–212 μm and 425–600 μm in size) and centrifuged at 13,000 × g for 2 min. The supernatant was discarded and 1 ml of DNA extraction buffer (50 mM Tris-HCl pH 8.0, 5 mM EDTA pH 8.0, 350 mM sorbitol, 1% N-lauryl sarcosine, 71 mM NaCl, 0.1% CTAB) with 1 µl of Monarch RNase A (NEB, T3018L) was added. The cells were lysed in a Qiagen TissueLyser II Bead Mill (QIAGEN), which involved shaking at 30 Hz for 10 min. Subsequently, the sample was incubated at 60 °C for 20 min; then an equal volume of chloroform (Scientific Laboratory Supplies, CHE1574) was added. The sample was mixed by inverting several times and centrifuged at 13,000 × g for 5 min. The aqueous layer (top layer) was transferred to a new tube and an equal volume of ice-cold isopropanol was added. The tube was inverted several times and incubated at −20 °C overnight. Afterwards, the tube was centrifuged at 13,000 × g for 5 min, the supernatant was discarded, and the tube was inverted on tissue paper to allow the DNA pellet to air dry for 10 min. Following this, the DNA was resuspended in 50 µl of Milli-Q water. The suspension was centrifuged at 13,000 × g for 5 min, and the supernatant containing the DNA was transferred to a 1.5-ml tube and quantified using a Qubit fluorometer (Thermo Fisher). For the preparation of DNA libraries, the DNA was digested with NEBNext dsDNA fragmentase (NEB, M0348L) in the following reaction mix: 200 ng of DNA in 16 µl Milli-Q water, 2 µl 10× fragmentase reaction buffer v.2 and 2 µl NEBNext dsDNA fragmentase. The reactions were incubated at 37 °C for 20 min, and the process was halted by adding 5 µl 0.5 M EDTA pH 8. The volume was adjusted to 50 µl with Milli-Q water, and DNA fragments between 300 and 500 bp were selectively isolated using double-sided DNA selection with Agencourt AMPure XP beads (Beckman Coulter, A63881). After incubation at 20 °C for 30 min, the fragments were purified again using Agencourt AMPure XP beads. The mixture was incubated at 37 °C for 30 min, followed by 70 °C for 5 min, and then purified using Agencourt AMPure XP beads. Individual samples were indexed through ligation using a mix comprising 10.25 µl DNA, 1 µl 600 U µl−1 T4 DNA ligase, 12.5 µl of 2× Rapid Ligation buffer and 1.25 µl 2.5 µM indexing adapter from the KAPA Dual-Indexed Adapter kit (Kapa Biosystems, KK8722). Samples were incubated at 25 °C for 15 min; then 5 µl 0.5 M EDTA pH 8 was added. The fragments were purified twice with Agencourt AMPure XP beads and then enriched in the following reaction: 20 µl DNA, 25 µl of 2× KAPA HiFi HS Mix (Kapa Biosystems, KK2602), 2.5 µl 5 μM I5 primer (5′-AATGATACGGCGACCACCGAGATCTACAC-3′) and 2.5 µl 5 μM I7 primer (5′-CAAGCAGAAGACGGCATACGAGAT-3′). The resulting DNA libraries were purified using Agencourt AMPure XP beads, quantified on a Qubit fluorometer (Thermo Fisher), and equimolar quantities of individual barcoded DNA libraries were pooled and sequenced (PE150 bp) on an MGI Tech MGISEQ-2000 sequencing platform at Beijing Genomics Institute. A glycerol stock sample of each isolate was plated on the respective selective agar media and incubated at 30 °C for 72–96 h. Subsequently, a 15-ml tube containing 3 ml of selective medium was inoculated with a single colony from the agar plate. for bacteria, in a shaking incubator for 72–120 h. Following this incubation period, cells were collected by centrifugation at 4,000 × g at 4 °C for 8 min and subjected to three washes with 10 mM MgCl2 to remove residual media and cellular debris. The cells were then resuspended in 10 mM MgCl2, and the optical density at 600 nm (OD600) was measured to estimate cell concentrations. A pooled inoculum containing all isolates of the defined community was prepared, with the final concentration of each isolate in the pool set at 109 colony-forming units per millilitre (c.f.u.s ml−1) assuming that 1 OD600 unit is equal to 109 c.f.u.s ml−1. In addition, individual strains were systematically removed (single-strain dropout) from the 9-member microbial community to evaluate how the absence of each strain impacts overall community structure and function. Mature, ripe, healthy and undamaged cacao pods were harvested from the ICGT. The pods were thoroughly washed with water to dislodge any debris and surface sterilized using 20% hypochlorite solution containing 0.05% Triton X-100 for 10 min in a sterile hood. The pods were carefully opened and the beans with pulp were extracted, maintaining sterile conditions. All beans from all the pods were pooled, mixed to homogeneity and divided into six sterile microboxes (Sac O2, TP1600 + TPD1600), each containing ~1 kg of beans. Three of the microboxes received inoculation with 100 µl of the defined synthetic community inoculum (SYNCOM), while the remaining three microboxes were inoculated with only 100 µl 10 mM MgCl2, serving as the No SYNCOM control. Daily pH measurements were taken by collecting three beans from each fermentation. For monitoring the microbial community during fermentation, swab samples of the beans were collected at 0, 48 and 96 h using a Zymo Collection Swab (R1104). Samples were collected in duplicate for each fermentation box and placed in Zymo DNA/RNA Shield Lysis and Collection tubes (Zymo, R1104). The tube contents were vigorously shaken for 10 s and the tubes were stored at −20 °C until used. Following the fermentation, the beans were spread on foil trays and placed in an oven at 35 °C for 5 days to reduce the moisture content to <7%. The beans were stirred on drying days 1, 2 and 3 to prevent bean clumping. For the single-strain dropout experiments, 125 g of dried unfermented cocoa beans was sterilized with a 70% ethanol solution containing 1% Tween 20 and then rehydrated with sterile water. Excess water was discarded and the beans were transferred to a sterile microbox (Sac O2, TP1600 + TPD1600) containing 200 ml of sterile artificial pulp. The experimental design included a full synthetic microbial consortium (T1, 9-member SYNCOM) and modified versions where individual strains were removed (T2–T10). Control groups consisted of non-inoculated beans (T11 and T12) and beans inoculated with a randomly selected 9-member microbial consortium (T13). For each treatment, four independent fermentations were performed. However, for T12, the beans were maintained at a constant temperature of 30 °C for the entire 120-h incubation period. pH measurements of the testa/pulp and cotyledons were recorded daily from a single bean of each fermentation. Swab samples for microbial community analysis were collected at 0, 24 and 48 h (n = 156), while 5 beans from each treatment replicate were sampled at 0, 48 and 120 h for metabolomic analysis (n = 156). After fermentation, beans from each treatment replicate (n = 52) were oven dried at 35 °C, as previously described, to produce cocoa liquors. In addition, total cell counts in the 9-member SYNCOM were measured using a BlauBrand Thoma counting chamber (Brand). To assess potential growth limitations, whether due to nutrient deficiencies in the pulp or environmental stresses from temperature and/or pH, each isolate was cultured in artificial pulp at pH values of 3.6, 4.6, 5.6 and 6.6. Cultures were incubated under three fermentation temperature conditions (30 °C, 35 °C and 45 °C) for 70 h with 200 r.p.m. Growth was monitored across all conditions by measuring OD600. The DNA samples were placed in a Qiagen TissueLyser II Bead Mill (QIAGEN) and homogenized at 30 Hz for 10 min. Following this, DNA was extracted using the ZymoBIOMICS DNA Miniprep kit (Zymo, D4300) following manufacturer instructions, and the resulting DNA concentration was determined using a Qubit fluorometer (Thermo Fisher). Unique frameshifting tags were added to the 5′ end of both primers following the method outlined in ref. 97 to enhance library diversity and enable efficient multiplexing of samples for sequencing. Each sample was amplified in triplicate, and for each 96-well PCR plate of reactions, three unique sets of frameshifting tag combinations were employed with both the forward and reverse primers. This approach facilitated the effective multiplexing of samples for sequencing across multiple plates. The reaction mix for each sample included 1 µl DNA, 5 µl 2× KAPA HiFi HS Mix (Kapa Biosystems, KK2602), 0.25 µl of 338F forward primer frameshift mix (10 pmol µl−1), 0.25 µl of 806R reverse primer frameshift mix (10 pmol µl−1) and 3.5 µl Milli-Q water. The PCR products from the triplicate reactions were combined and purified using Agencourt AMPure XP beads (Beckman Coulter, A63881). Subsequently, the PCR products were indexed using 96 unique reverse indexing primers. The indexing mix for each sample included 4.5 µl PCR product DNA, 5 µl 2× KAPA HiFi HS Mix (Kapa Biosystems, KK2602), 0.25 µl forward enrichment primer (10 pmol µl−1) and 0.25 µl reverse enrichment-indexing primer (10 pmol µl−1). The DNA libraries were purified using Agencourt AMPure XP beads (Beckman Coulter, A63881) and quantified with a Qubit fluorometer (Thermo Fisher). Subsequently, the libraries were pooled in equal amounts and diluted to 10 pM for sequencing. For fungal profiling, the ITS2 region was amplified and sequenced using the ITS3-F (5′-GCATCGATGAAGAACGCAGC-3′) and ITS4-R (5′-TCCTCCGCTTATTGATATGC-3′) universal primer sequences described in ref. To enhance library diversity and facilitate sample multiplexing for sequencing, unique frameshifting tags were incorporated at the 5′ end of both primers using the methodology outlined in ref. Every sample was subjected to triplicate amplification. In addition, for each 96-well PCR plate of reactions, three distinct sets of unique frameshifting tag combinations of the forward and reverse primers were used to enable the multiplexing of samples for sequencing from multiple plates. Amplicons from the triplicate reactions were consolidated and purified using Agencourt AMPure XP beads (Beckman Coulter, A63881). Subsequently, the samples were indexed using the TruSeq Read 2-annealing reverse Illumina adapter, pooled as previously described (see ‘Bacteria 16S rRNA library preparation and sequencing') and sequenced on an Illumina MiSeq platform using the 600-cycle V3 Reagent kit (Illumina) at the University of Nottingham's DeepSeq Sequencing Facility. For the validation of the defined community experiment, dried beans from the three 9-member SYNCOM inoculated fermentations were combined into a single pool, while the three No SYNCOM control batches were pooled separately. The pooled beans were then processed into cocoa liquors and subjected to sensory evaluation as described previously (see ‘Drying, roasting and sensory evaluation of liquor samples'), and well as to gas chromatography–mass spectrometry (GC–MS) analysis. For the single-strain dropout experiment, cocoa beans collected at 0, 48 and 120 h of fermentation were freeze dried for 72 h and ground into a fine powder for liquid chromatography–mass spectrometry (LC–MS) and GC–MS analysis. In addition, the dried fermented beans were processed into cocoa liquors and analysed using sensory evaluation and GC–MS. The aroma and other volatile compounds in the cocoa bean and liquor samples were analysed at the International Flavour Research Centre (IFRC) at the University of Nottingham. Milled powder (1 g) of the cocoa samples was mixed with 10 µl 3-heptanone (0.01 μg µl−1) internal standard in hermetically sealed 20-ml vials and incubated for 5 min at 50 °C in a thermostatic agitator. A 50/30 μm DVB/CAR/PDMS SPME Fibre (Supelco) was used to extract volatile compounds from the headspace of each sample. The SPME fibre extracted for 15 min at 50 °C and desorbed for 0.5 min at 240 °C. The volatiles were analysed by GC–MS using splitless injection into a TRACE 1300 series gas chromatograph coupled with a single quadrupole mass spectrometer (Thermo Fisher). A ZB-WAX-plus column of 30 m length, 0.25 mm internal diameter and 0.250 μm film thickness (Phenomenex) was used with the following time–temperature programme: 40 °C for 2 min, followed by a temperature increase from 40–240 °C at a rate of 6 °C min−1, and then held at 240 °C for 5 min. A minimum of 3 replicates per liquor sample were analysed with randomized sample injections for the validation of the defined community experiment, including 9-member SYNCOM-inoculated samples, No SYNCOM samples, Santander, Huila, Antioquia and reference liquors. For the single-strain dropout experiment, individual biological replicates were utilized. The SPME fibre was conditioned for 3 min at 240 °C between samples. The quality of the headspace GC–MS runs was assessed by running the internal standard after 5–20 consecutive sample runs and estimating the variations in retention time and peak areas. Volatile compounds were identified by comparing each mass spectrum with either the spectra from standard compounds or with spectra in reference libraries (NIST/EPA/NIH Mass Spectral Library). The relative abundance of volatiles was calculated from GC peak areas by comparison with the peak area of the internal standard. A 100-mg portion of the powdered sample was weighed and placed into a 1.5-ml microcentrifuge tube. The sample was defatted by adding 800 µl ice-cold hexane, vortexing and incubating in a sonic water bath. This defatting process was repeated twice more, and the defatted pellet was dried using nitrogen gas with a sample concentrator (Techne). To extract metabolites, 460 µl 80% methanol was added to the dried pellet, and the mixture was vortexed, sonicated and centrifuged. The combined supernatant was centrifuged again and the final supernatant was transferred to a new tube. The extract was dried using a Savant SpeedVac SPD140DDA vacuum concentrator (Thermo Scientific) and then stored at −20 °C. Before analysis, the dried pellet was reconstituted with 50% aqueous ethanol, followed by sonication. The sample was centrifuged and the supernatant was transferred to a new 1.5-ml tube. For each time point (0, 48 and 120 h), aliquots from biological replicates of each treatment were pooled separately, transferred to LC–MS vials, capped and stored for further analysis. The generation of the untargeted metabolic profiles was performed using an Agilent 1260 Infinity II Ultra High-Performance Liquid Chromatography system coupled to an Agilent 6546 tandem quadrupole time-of-flight mass spectrometer (Agilent Technologies). Chromatographic separation was performed with an Acuity UPLC HSS T3 column (2.1 × 100 mm, 1.8 μm; Waters) fitted with a KrudCatcher pre-filter (Phenomenex). The flow rate of the mobile phase (A: 5% acetonitrile, versus B: 95% acetonitrile, both with 0.1% formic acid v/v) was at 0.3 ml min−1, with the analytical gradient starting at 5% solvent B, increasing to 15%, 25%, 35%, 45% and 65% at 2, 4, 8, 10 and 12 min, respectively, followed by column washing and re-equilibration (total run time 22 min). After injection of ×10 QC samples to condition the system, each sample was randomized to ×5 injections across the batch. QC samples were injected after every 10 consecutive runs to assess system performance across the batch. Data were collected in MS1 mode scanning 50–1,700 m/z. Reference masses were continuously injected for mass correction. We performed multiple sequence alignments with the 16S rRNA and ITS sequences for the bacteria and fungi, respectively, using DECIPHER v.2.24.0. 3b) and fungi ITS sequences (Extended Data Fig. 3c), were visualized using ggtree v.3.8.0 with ggtreeExtra v.1.10.0. Subsequently, we computed the mean relative abundance of each family at each fermentation time point and plotted the results using ggplot2 v.3.4.2. We used Cutadapt (v.4.6)100 to eliminate primer and barcode sequences, as well as low-quality sequences, from the paired-end reads of the sequenced genomes of the isolates. Subsequently, the high-quality filtered reads were de novo assembled into a draft genome for each isolate using SPAdes (v.3.15.5)101 with default parameters. The assembled genomes were evaluated for contiguity and completeness using BUSCO (v.5.6.1)102. Open reading frames in the genomes were predicted with FragGeneScanRs (v.1.1.0)85 with default settings. Functional annotation of the predicted proteins was carried out using the eggNOG-mapper (v.2.1.9)86 pipeline, utilizing the eggNOG v.5.0.2 database87 with Diamond (v.2.0.11)88 and MMseqs2 release 12-113e3 (ref. The predicted coding DNA and translated amino acid sequences, along with the predicted functional annotations, were combined into genbank-formatted files using emapper2gbk (v.0.3.0)92 in ‘genes mode'. Metabolites reachable by each of the isolates in the network, based on the cocoa pulp metabolites as the precursor, were compiled into a data matrix and visualized as described previously (see ‘Metabolic network analysis and identification of a defined microbial community'). Raw reads were demultiplexed and trimmed with Cutadapt (v.4.6)100. Subsequently, the processed sequences were denoised and collapsed into amplicon sequence variants (ASVs) using the DADA2 v.1.24.0 pipeline. In brief, paired reads were filtered by removing sequences with uncalled bases, eliminating reads with >2 expected errors, and truncating reads when the average quality score dropped to <2. Error rates for forward and reverse reads were determined separately through the learnErrors() function. These error rates were then utilized to infer ASVs individually for both the forward and reverse reads, and the forward and reverse sequences were subsequently merged. The merged ASVs were used to construct an ASV sequencing table, and chimaeras were removed. To assess alpha diversity across the samples, we calculated the Shannon diversity index using phyloseq v.1.44.0. We used ANOVA to test for significant differences in Shannon diversity indices between groups, and means were separated using Tukey's HSD test in the agricolae v.1.3.5 R package. For beta diversity, Bray–Curtis dissimilarity matrices were calculated using the phyloseq v.1.44.0 ‘bray' method, and the variances explained by treatment and fermentation time were estimated by performing PERMANOVA using the adonis2() function in the vegan v.2.6.4 R package. Constrained ordination of beta-diversity was plotted using CAP on the basis of Bray–Curtis dissimilarity matrices calculated with vegan v.2.6.4. We visualized differences in treatment and time with the CAP analysis, using the following models: PCoA based on Bray–Curtis dissimilarities was used to visualize shifts in microbial community composition across fermentation treatments (T1–T13). The relative abundance of taxa was plotted as a stacked bar representation using phyloseq v.1.44.0. The tax_glom() function in phyloseq v.1.44.0 was used to agglomerate taxa, and the aggregate_rare() function in microbiome v.1.22.0 was used to aggregate rare groups. The data were first pre-processed, followed by analysis in R. PCA was performed using Euclidean distances with the prcomp() function, while PCoA was conducted using a dissimilarity matrix computed with the Manhattan distance in the dist() function. Classical multidimensional scaling was then applied using the cmdscale() function. The results were visualized using ggplot2 v.3.4.2. A bar plot of Manhattan distances between each treatment centroid and T1 (the full synthetic community) was generated to quantify dissimilarity from the baseline. To identify enriched volatile compounds among samples, we employed DESeq2 (v.1.40.0)65, fitting a GLM with the following design: From the fitted model, we extracted key comparisons. A volatile compound was deemed significant if it exhibited an FDR-adjusted P value (q value) < 0.05. Black squares were used to highlight significant differences (q value < 0.05) with log2 fold change > ±2 between the aforementioned comparisons. For data analysis, the total ion chromatograms of repeat QC injections were visually assessed to check comparability of runs throughout the dataset. Initially, global MS1 features (peak height >20,000) were first extracted using Mass Profiler software (MP; v.10 Agilent Technologies) and exported to a common .CEF file for each polarity. Thereafter, files for each replicate group were time aligned to the central QC sample in Profinder software (v.10, Agilent); then features were extracted (peak height >5,000) in ‘batch targeted' mode using the global MS1 features .CEF file as a reference library. Following this pre-processing, PCoA was conducted using Euclidean distances in vegan v.2.6.4. The dissimilarity matrix was computed with the vegdist() function and then used to perform PCoA. The results were visualized using ggplot2 v.3.4.2. A bar plot depicting the Euclidean distances between each treatment centroid and T1 (the full synthetic community) was created to measure dissimilarity from the baseline. We employed DESeq2 (v.1.40.0)65 to discern enriched compounds among samples. This was achieved by fitting a GLM with the design: From the fitted model, we identified key comparisons between treatments and fermentation times, highlighting significant differences where q values were <0.05 and log2 fold changes exceeded ±2. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Nanopore sequencing data, as well as 16S rRNA and ITS amplicon sequencing data generated for this study, have been archived in the NCBI Sequence Read Archive under project accession PRJNA1104253. Microbial genomes and metagenomes produced in this work are publicly available via https://www.gabrielcastrillo.com/ and through Zenodo at https://doi.org/10.5281/zenodo.15985598 (ref. Datasets required to reproduce the results of this study are available in the associated GitHub repository at https://github.com/David-Lee86/min-com-choc (ref. Reference genomes used include Theobroma cacao Criollo v.2.0 (Cocoa Criollo B97-61/B2 version 2; https://cocoa-genome-hub.southgreen.fr/download) and Homo sapiens GRCh38.p14 (RefSeq GCF_000001405.40). The strain collection used in this study is available upon request by contacting G.C. Scripts, source data and metabolic models required to reproduce the analyses of this study can be accessed through GitHub at https://github.com/David-Lee86/min-com-choc (ref. Schwan, R. F. & Wheals, A. E. The microbiology of cocoa fermentation and its role in chocolate quality. De Vuyst, L. & Leroy, F. Functional role of yeasts, lactic acid bacteria and acetic acid bacteria in cocoa fermentation processes. Afoakwa, E. O., Paterson, A., Fowler, M. & Ryan, A. Kongor, J. E. et al. Factors influencing quality variation in cocoa (Theobroma cacao) bean flavour profile—a review. Moreira, I. M. D. V., Miguel, M. G. D. C. P., Duarte, W. F., Dias, D. R. & Schwan, R. F. Microbial succession and the dynamics of metabolites and sugars during the fermentation of three different cocoa (Theobroma cacao L.) hybrids. Herrera-Rocha, F. et al. Dissecting fine-flavor cocoa bean fermentation through metabolomics analysis to break down the current metabolic paradigm. Rodriguez-Campos, J., Escalona-Buendía, H. B., Orozco-Avila, I., Lugo-Cervantes, E. & Jaramillo-Flores, M. E. Dynamics of volatile and non-volatile compounds in cocoa (Theobroma cacao L.) during fermentation and drying processes using principal components analysis. Garofalo, C. et al. Starter cultures for sparkling wine. Roudil, L. et al. Non-Saccharomyces commercial starter cultures: scientific trends, recent patents and innovation in the wine sector. Fox, P. F., Guinee, T. P., Cogan, T. M. & McSweeney, P. L. H. in Fundamentals of Cheese Science 121–183 (Springer, 2017). Lodolo, E. J., Kock, J. L. F., Axcell, B. C. & Brooks, M. The yeast Saccharomyces cerevisiae—the main character in beer brewing. Pacheco-Montealegre, M. E., Dávila-Mora, L. L., Botero-Rute, L. M., Reyes, A. Fine resolution analysis of microbial communities provides insights into the variability of cocoa bean fermentation. Bortolini, C., Patrone, V., Puglisi, E. & Morelli, L. Detailed analyses of the bacterial populations in processed cocoa beans of different geographic origin, subject to varied fermentation conditions. Papalexandratou, Z., Camu, N., Falony, G. & De Vuyst, L. Comparison of the bacterial species diversity of spontaneous cocoa bean fermentations carried out at selected farms in Ivory Coast and Brazil. Miguel, M. G. D. C. P. et al. Cocoa fermentation: microbial identification by MALDI-TOF MS, and sensory evaluation of produced chocolate. Kostinek, M. et al. Diversity of predominant lactic acid bacteria associated with cocoa fermentation in Nigeria. Camu, N. et al. Dynamics and biodiversity of populations of lactic acid bacteria and acetic acid bacteria involved in spontaneous heap fermentation of cocoa beans in Ghana. Identification of predominant yeasts associated with artisan Mexican cocoa fermentations using culture-dependent and culture-independent approaches. Batista, N. N., Ramos, C. L., Ribeiro, D. D., Pinheiro, A. C. M. & Schwan, R. F. Dynamic behavior of Saccharomyces cerevisiae, Pichia kluyveri and Hanseniaspora uvarum during spontaneous and inoculated cocoa fermentations and their effect on sensory characteristics of chocolate. Determination of the microbial community in Amazonian cocoa bean fermentation by Illumina-based metagenomic sequencing. Agyirifo, D. S. et al. Metagenomics analysis of cocoa bean fermentation microbiome identifying species diversity and putative functional capabilities. Illeghems, K., De Vuyst, L., Papalexandratou, Z. & Weckx, S. Phylogenetic analysis of a spontaneous cocoa bean fermentation metagenome reveals new insights into its bacterial and fungal community diversity. de Melo Pereira, G. V., Magalhães-Guedes, K. T. & Schwan, R. F. rDNA-based DGGE analysis and electron microscopic observation of cocoa beans to monitor microbial diversity and distribution during the fermentation process. Taylor, A. J., Cardenas-Torres, E., Miller, M. J., Zhao, S. D. & Engeseth, N. J. Microbes associated with spontaneous cacao fermentations—a systematic review and meta-analysis. Pettipher, G. L. Analysis of cocoa pulp and the formulation of a standardised artificial cocoa pulp medium. The chemistry behind chocolate production. Fang, Y. et al. Chemical and flavor profile changes of cocoa beans (Theobroma cacao L.) during primary fermentation. Andersson, M., Koch, G. & Lieberei, R. Structure and function of the seed coat of Theobroma cacao L. and its possible impact on flavour precursor development during fermentation. Afoakwa, E. O., Quao, J., Takrama, J., Budu, A. S. & Saalia, F. K. Chemical composition and physical quality characteristics of Ghanaian cocoa beans as affected by pulp pre-conditioning and fermentation. Ardhana, M. M. & Fleet, G. H. The microbial ecology of cocoa bean fermentations in Indonesia. Ndukwu, M. C. & Udofia, M. Kinetics of change in colour and some bio-chemical composition during fermentation of cocoa bean. Fernández-Niño, M. et al. Dissecting industrial fermentations of fine flavour cocoa through metagenomic analysis. Van de Voorde, D., Díaz-Muñoz, C., Hernandez, C. E., Weckx, S. & De Vuyst, L. Yeast strains do have an impact on the production of cured cocoa beans, as assessed with Costa Rican Trinitario cocoa fermentation processes and chocolates thereof. Menezes, A. G. T. et al. Investigation of chocolate produced from four different Brazilian varieties of cocoa (Theobroma cacao L.) inoculated with Saccharomyces cerevisiae. Species diversity, community dynamics, and metabolite kinetics of the microbiota associated with traditional Ecuadorian spontaneous cocoa bean fermentations. & Ali, N. A. in Achieving Sustainable Cultivation of Cocoa – Genetics, Breeding, Cultivation and Quality Vol. Ilangantileke, S. G., Wahyudi, T. & Bailon, M. A. G. Assessment methodology to predict quality of cocoa beans for export. Sukha, D. A. in Drying and Roasting of Cocoa and Coffee (eds Hii, C. L. & Borém, F. M.) Ch. Giraffa, G. Studying the dynamics of microbial populations during food fermentation. Vinicius De Melo Pereira, G. et al. A review of selection criteria for starter culture development in the food fermentation industry. Sharma, R., Garg, P., Kumar, P., Bhatia, S. K. & Kulshrestha, S. Microbial fermentation and its role in quality improvement of fermented foods. Schnermann, P. & Schieberle, P. Evaluation of key odorants in milk chocolate and cocoa mass by aroma extract dilution analyses. Counet, C., Callemien, D., Ouwerx, C. & Collin, S. Use of gas chromatography−olfactometry to identify key odorant compounds in dark chocolate. Comparison of samples before and after conching. A. et al. Genetics of amino acid taste and appetite. Origin of 2,3-pentanedione and 2,3-butanedione in d-glucose/l-alanine Maillard model systems. Rodriguez-Campos, J. et al. Effect of fermentation time and drying temperature on volatile compounds in cocoa. Lefeber, T., Janssens, M., Camu, N. & De Vuyst, L. Kinetic analysis of strains of lactic acid bacteria and acetic acid bacteria in cocoa pulp simulation media toward development of a starter culture for cocoa bean fermentation. Lee, A. H. et al. A laboratory-scale model cocoa fermentation using dried, unfermented beans and artificial pulp can simulate the microbial and chemical changes of on-farm cocoa fermentation. Braga, S. C. G. N. et al. Study of volatile profile in cocoa nibs, cocoa liquor and chocolate on production process using GC×GC-QMS. Ducki, S., Miralles-Garcia, J., Zumbé, A., Tornero, A. & Storey, D. M. Evaluation of solid-phase micro-extraction coupled to gas chromatography–mass spectrometry for the headspace analysis of volatile compounds in cocoa products. Frauendorfer, F. & Schieberle, P. Changes in key aroma compounds of Criollo cocoa beans during roasting. & Barringer, S. A. Alkylpyrazines and other volatiles in cocoa liquors at pH 5 to 8, by selected ion flow tube‐mass spectrometry (SIFT‐MS). & Blanch, C. Effect of cocoa roasting time on volatile composition of dark chocolates from different origins determined by HS-SPME/GC-MS. CyTA 19, 81–95 (2021). & Heimdal, H. Effect of fermentation method, roasting and conching conditions on the aroma volatiles of dark chocolate. Calvo, A. M. et al. Dynamics of cocoa fermentation and its effect on quality. De Coster, W., D'Hert, S., Schultz, D. T., Cruts, M. & Van Broeckhoven, C. NanoPack: visualizing and processing long-read sequencing data. The cacao Criollo genome v2.0: an improved version of the genome for genetic and functional genomic studies. Li, H. Minimap2: pairwise alignment for nucleotide sequences. Improved metagenomic analysis with Kraken 2. Lu, J., Breitwieser, F. P., Thielen, P. & Salzberg, S. L. Bracken: estimating species abundance in metagenomics data. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Gopaulchan, D. et al. Morphological and genetic diversity of cacao (Theobroma cacao L.) in Uganda. Motilal, L. A., Sankar, A., Gopaulchan, D. & Umaharan, P. in Biotechnology of Plantations Crops (eds Chowdappa, P. et al.) Ch. Mahabir, A. et al. Development of a core SNP panel for cacao (Theobroma cacao L.) identity analysis. Guide for the Assessment of Cacao Quality and Flavour (Bioversity International, 2023). Geographic and genetic population differentiation of the Amazonian chocolate tree (Theobroma cacao L). Pritchard, J. K., Stephens, M. & Donnelly, P. Inference of population structure using multilocus genotype data. & Paulo, O. S. Structure_threader: an improved method for automation and parallelization of programs structure, fastStructure and MavericK on multicore CPU systems. Kopelman, N. M., Mayzel, J., Jakobsson, M., Rosenberg, N. A. & Mayrose, I. Clumpak: a program for identifying clustering modes and packaging population structure inferences across K. Mol. Evanno, G., Regnaut, S. & Goudet, J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Garcia-Neto, M., Kebreab, E. & Faria-Júnior, M. J. D. A. PPFM: practical program for forces modeling. Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. & Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. & Banfield, J. F. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. & Aluru, S. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. B., Arkhipova, K., Cambuy, D. D., Coutinho, F. H. & Dutilh, B. E. Robust taxonomic classification of uncharted microbial sequences and bins with CAT and BAT. Van der Jeugt, F., Dawyndt, P. & Mesuere, B. FragGeneScanRs: faster gene prediction for short reads. Cantalapiedra, C. P., Hernández-Plaza, A., Letunic, I., Bork, P. & Huerta-Cepas, J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Sensitive protein alignments at tree-of-life scale using DIAMOND. Steinegger, M. & Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Aroney, S. T. N. et al. CoverM: read alignment statistics for metagenomics. Wright, R. M., Aglyamova, G. V., Meyer, E. & Matz, M. V. Gene expression associated with white syndromes in a reef building coral, Acropora hyacinthus. Belcour, A. et al. Metage2Metabo, microbiota-scale metabolic complementarity for the identification of key species. Frioux, C., Fremy, E., Trottier, C. & Siegel, A. Scalable and exhaustive screening of metabolic functions carried out by microbial consortia. Karp, P. D. et al. Pathway Tools version 23.0 update: software for pathway/genome informatics and systems biology. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2023); http://www.R-project.org/ de Man, J. C., Rogosa, M. & Sharpe, M. E. A medium for the cultivation of lactobacilli. Lundberg, D. S., Yourstone, S., Mieczkowski, P., Jones, C. D. & Dangl, J. L. Practical innovations for high-throughput amplicon sequencing. White, T. J., Bruns, T., Lee, S. J. W. T. & Taylor, J. in PCR Protocols: A Guide to Methods and Applications (eds Innis, M. A. et al.) 315–322 (Academic, 1990). Altschul, S. F., Gish, W., Miller, W., Myers, E. W. & Lipman, D. J. Basic local alignment search tool. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Prjibelski, A., Antipov, D., Meleshko, D., Lapidus, A. Using SPAdes de novo assembler. Manni, M., Berkeley, M. R., Seppey, M., Simão, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Abarenkov, K. et al. UNITE general FASTA release for eukaryotes 2. Gopaulchan, D. Microbial genomes associated with cocoa bean fermentation [data set]. Gopaulchan, D. Controlled cocoa fermentation with defined microbes. We thank the Federación Nacional de Cacaoteros (Fedecacao), Bogotá, Colombia, for providing personnel and logistical support in Colombia. We also thank CasaLuker S.A, Bogotá, Colombia for support in harvesting activities and access to laboratory space at CasaLuker S.A in Bogotá as part of the collaboration contract with the University of Nottingham to carry out the project ‘Controlling cocoa bean fermentation for enhanced chocolate flavour'. To perform this project, CasaLuker S.A obtained permit 284 (2020) to access genetic resources and derived products granted by the Ministry of Environment and Sustainable Development of Colombia. All harvesting, processing and analysis of the material was carried out in Colombia as part of a CasaLuker S.A working group. This study was funded by the Prosperity Fund through Innovate UK (UKRI Innovate UK Agri-Tech Catalyst Colombia grant project number 133788) to D.G., C.M., D.E.S. BioRender was used to create the schematic diagram illustrating the field cocoa bean fermentation analysis conducted in this study. Future Food Beacon of Excellence, University of Nottingham, Sutton Bonington Campus, Nottingham, UK Cocoa Research Centre, The University of the West Indies, St Augustine, Trinidad and Tobago Naailah Ali, Darin Sukha & Pathmanathan Umaharan Fabio Esteban Herrera Rocha & Andrés Fernando González Barrios Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conducted DNA extraction, prepared DNA sequencing libraries and performed whole metagenome shotgun sequencing with A.F.G.B. prepared the bacteria and fungi collection. performed DNA extraction and 16S rRNA and ITS library preparation for the vitro fermentation experiment. Correspondence to David E. Salt or Gabriel Castrillo. The authors declare no competing interests. Nature Microbiology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The map shows the cocoa plantations from three cocoa-growing regions in Colombia (Santander, Huila, and Antioquia) evaluated in this study. b. Schematic diagram of field bean fermentation analysis performed in this work. Cocoa varieties grown on farms were genotyped using SNP analysis. Temperature, pH, and colour changes in the fermenting beans were recorded daily. Microbial communities were characterised using Oxford Nanopore metagenomic shotgun sequencing. The diagram was created in BioRender. Line graphs depict the magnitude of distribution of bean temperature and pH recorded during the fermentation. The difference between distributions was assessed using a Kolmogorov-Smirnov test, with corresponding p-values indicated within the plots. A linear regression line illustrates the relationship, with a shaded band representing the 95% confidence interval. The scatter plots display correlation coefficients (r) and associated p-values. f. Principal component analysis (PCA) of bean colour changes during fermentation. PCA is based on extracted RGB values from images of the beans taken daily. A minimum of eight points on each bean image were analysed for each time point. g. Pearson correlation analysis between bean colour parameters (blue, green, red, grayscale, and luminance) and fermentation temperature, between the bean colour parameters and the pH of the bean's cotyledon, and between the bean colour parameters and the pH of the testa/pulp pH. Correlation coefficients (r) and associated p-values are shown within each panel. h. Read length distribution of Nanopore sequencing reads. Features extracted from the temperature kinetic curves and b. pH kinetic curves of fermenting beans. The heatmap on the left displays key features extracted from the kinetic curves and is coloured according to mean normalised values. Pearson correlation coefficients for these features are in the centre heatmaps. Relationships were assessed using two-sided Pearson correlation tests. The plot is coloured to reflect the correlation coefficient (r) values and significant correlations (p < 0.05) are highlighted with black squares. All plots are clustered based on r values. e. Canonical analysis of principal coordinate of microbiome composition showing the projected bacterial and f. fungal community assembly across fermentations. Group differences were assessed using a PERMANOVA with 9999 permutations. PERMANOVA R2 and p-values explained by fermentation time points and farm location are indicated within the plots. g. Features extracted from the growth curves of selected bacteria and fungi in fermenting beans. Bacteria and fungi growth curves were plotted based on their relative abundance in fermenting beans from Santander, Huila and Antioquia, and growth features of the selected taxa were extracted. Pearson correlation coefficients (r) for these features are presented in the centre heatmap, with significant correlations (p < 0.05) highlighted by black squares. Relationships were assessed using two-sided Pearson correlation tests. The three plots are clustered based on r values. The left heatmap is also clustered according to taxa abundance. Principal coordinate analysis (PCoA) showing the projected metagenome assembled genomes (MAGs) abundance, with varying metabolic potentials, throughout the fermentation process in Santander, Huila, and Antioquia. The metabolic potential of each MAG, derived from cocoa pulp as the precursor, was transformed into a distance matrix and then converted into two dimensions using classical multidimensional scaling. MAGs are coloured based on their relative abundance during fermentation at each time point for each region. b. Neighbour-joining trees displaying bacteria and c. fungi isolates cultured in this study. a. Schematic representation of the experimental design showing the combinations of microbes used in cocoa beans fermentations performed using a full synthetic microbial consortium (T1, 9-member SYNCOM) and modified versions where individual strains were removed (T2–T10). Controls included non-inoculated beans (T11 and T12) and beans inoculated with a randomly selected 9-member microbial consortium (T13). The graphs illustrate the kinetics of pH changes in the cotyledons (red) and testa/pulp (purple) of cocoa beans fermented in vitro using the microbial combinations described in panel a. Smoothed lines indicate fitted trends based on raw data, with shaded areas representing 95% confidence intervals. The graphs depict the kinetics of pH changes in the cotyledons (red) and testa/pulp (purple) measured in cocoa beans fermented in vitro fermentations inoculated (SYNCOM; top)) or not (No SYNCOM; bottom) with a synthetic microbial consortium at different time points. Smoothed lines indicate fitted trends based on raw data, with shaded areas representing 95% confidence intervals. Isolates were cultured in artificial cocoa pulp media under varying pH conditions and temperatures for 72 h. Growth is represented by the area under the growth curve, with the heatmap colour-coded to reflect these values. Additionally, isolate Acetobacter ID918 was also cultured in acetic acid medium, as indicated by an asterisk (*). c. Total cell counts were determined in in vitro fermentations inoculated with the defined synthetic community and measured at different time points throughout the fermentation process. Smoothed lines indicate fitted trends, with shaded areas representing 95% confidence intervals. a. Graphs showing the alpha diversity estimates of bacterial and b. fungal communities in fermenting cocoa beans inoculated with the full microbial synthetic community (T1, 9-member SYNCOM) and modified versions where individual strains were removed (T2–T10). Controls included non-inoculated beans (T11 and T12) and beans inoculated with a randomly selected 9-member microbial consortium (T13). For each treatment, four independent fermentations were conducted, with a single microbial DNA sample collected from the fermenting mass at 0, 24, and 48 h in each fermentation. After quality filtering, 155 bacterial (n = 155) and 154 fungal (n = 154) community profiles were retained for downstream analysis. Boxplots depict the distribution of Shannon Diversity Index values at different sample time points. Individual data points, including outliers, are displayed as overlaid dots. One-way ANOVA was used to identify significant differences among groups and means were separated using Tukey's post hoc test. Letters indicate similarities and differences between groups (p < 0.05). a Canonical analysis of principal coordinate (CAP) of microbiome composition showing the projected bacterial community assembly across the different in vitro fermentations inoculated with the full microbial synthetic community (T1, full 9-member SYNCOM) and modified versions where individual strains were removed (T2–T10). Controls included non-inoculated beans (T11 and T12) and beans inoculated with a randomly selected 9-member microbial consortium (T13). Group differences were assessed using a PERMANOVA with 9999 permutations. CAP ordination illustrates sample clustering by synthetic community treatment, with the PERMANOVA R² and p-value displayed within the panel. CAPs analysis showing the projected bacterial community assembly in the individual fermentations inoculated with the different microbial combinations described in panel a at distinct time points. c. CAP analysis of microbiome composition showing the projected fungal community assembly across the different in vitro fermentations inoculated with the different microbial combinations described in panel a. Group differences were assessed using a PERMANOVA with 9999 permutations. CAP ordination illustrates sample clustering by synthetic community treatment. The PERMANOVA R² and p-value are shown. d. CAPs analysis showing the projected fungal community assembly in the individual fermentations inoculated with the different microbial combinations described in panel a at distinct time points. e. Phylogram illustrating the relative abundance of key bacterial and f. fungal families across in vitro fermentations inoculated or not with the different microbial combinations described above at distinct time points. The 0 h time point represents samples collected after the initial synthetic microbial consortium was applied. Principal coordinate analyses (PCoA) showing the volatile compound profile projections of cocoa beans fermentations driven by different synthetic microbial consortium and b. fermentation time. Fermentations were carried out using a full synthetic microbial consortium (T1, 9-member SYNCOM) and modified versions where individual strains were removed (T2–T10). Controls included non-inoculated beans (T11 and T12) and beans inoculated with a randomly selected 9-member microbial consortium (T13). c. Heatmaps displaying the enrichment profiles of individual volatile compounds detected in beans fermentations from a. Squares outlined in black indicate volatile compounds that were significantly enriched (red) or depleted (blue) relative to the start of fermentation (0 h) (q < 0.05 and log2 fold change > ±2). Both volatile compounds and bean samples are clustered based on their log2 fold change values. d. Bar plot showing the number of enriched and depleted volatile compounds detected in each synthetic microbial consortium treatment compared to T1 at 120 h. e. Principal coordinate analyses (PCoA) showing the non-volatile compound profile projections in cocoa beans during fermentation with the different starters (left) described in panel a, and across fermentation time (right). f. Principal coordinate analyses (PCoA) showing the non-volatile compound profile projections in cocoa beans during fermentation driven by the starters described in panel a at different fermentation times (0, 48, 120 h). Points are coloured according to the microbial starter treatments. a. Heatmaps showing individual non-volatile compounds enriched or depleted at 48 h and 120 h compared to 0 h for each microbial treatment used. Cocoa bean fermentations were conducted using the full synthetic microbial consortium (T1, 9-member SYNCOM) and modified versions where individual strains were systematically removed (T2–T10). Control conditions included non-inoculated beans (T11, T12) and beans inoculated with an alternative, randomly selected 9-member consortium (T13). b. Heatmaps showing individual non-volatile compounds enriched or depleted in the different fermentations (T2-T13) described above relative to the fermentation with the full synthetic community (T1; 9-member SYNCOM) at 0 h, 48 h, and 120 h. A generalised linear model was used to assess enrichment, with significant changes determined by an adjusted p-value < 0.05 and log₂ fold change > ±2. Heatmaps are colour-coded, with red indicating enriched compounds and blue indicating depleted compounds. Hierarchical clustering highlights trends in compound variation across different time points and fermentation treatments. c. Bar plot showing the number of enriched and depleted non-volatile compounds detected in each synthetic microbial consortium treatment compared to T1 at 0 h, 48 h, and 120 h. d. Heatmap representing sensory attributes of cocoa liquors produced from beans fermented with the microbial consortia described in panel a. Mean sensory scores from panel evaluations were normalised between 0 and 6. Clustering was applied to identify similarities in sensory characteristics among different cocoa liquors. Theobroma cacao SNP panel used to genotype cacao trees sampled from cocoa plantations located in the Santander, Huila and Antioquia regions in Colombia. SNP profiles of Theobroma cacao trees assessed from cocoa plantations in this study. Feature importance of abiotic and biotic markers associated with flavour attributes in chocolate from Colombia and Trinidad cocoa bean fermentations. Genome and quality statistics of metagenome assembled genomes (MAGs) constructed from metagenomic datasets originating from cocoa plantations in Santander, Huila and Antioquia. Enriched GO categories across microbial communities during cocoa bean fermentation. Enriched KEGG reactions across microbial communities during cocoa bean fermentation. Enriched proteins across microbial communities during cocoa bean fermentation. Characterization of bacterial and fungal isolates using 16S rRNA and ITS amplicon sanger sequencing. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Gopaulchan, D., Moore, C., Ali, N. et al. A defined microbial community reproduces attributes of fine flavour chocolate fermentation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/technology/robots/a65781776/openai-psychosis/'>OpenAI Tried To Save Users From ‘AI Psychosis.' Those Users Were Not Happy.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Fast-forward 12 years later, and the film is now traipsing dangerously close to “prescient documentary” territory. Last week, OpenAI released ChatGPT-5, a new version of the popular AI platform that (as most tech updates do) replaced the older versions that came before it. “Earlier this year, we released an update to GPT‑4o that unintentionally made the model overly sycophantic, or excessively flattering or agreeable,” OpenAI explains in the post. For months, article after article has detailed the rise of what's called “AI Psychosis,” where an AI's overly agreeable sycophantic behavior can create an outsized dependency on the platform and/or reinforce delusions. According to Psychology Today, researchers have identified three recurring themes in “AI psychosis” cases: Messianic missions (when a deeper truth of the universe is believed to be revealed), God-like AI, and (in Her-esque fashion) romantic love. Of course, ripping the groveling heart out of the model essentially forced some users to quit their self-described AI companions/friends/lovers/gods cold turkey, and people soon began voicing their displeasure. “We for sure underestimated how much some of the things that people like in GPT-4o matter to them, even if GPT-5 performs better in most ways,” OpenAI CEO Sam Altman wrote on X (formerly Twitter), in response to the backlash. On Tuesday, Altman said that “4o was back in the model picker” for paid users, but seemed to imply that future changes to the model could be coming: “If we ever do deprecate it, we will give plenty of notice.” Unfortunately, it's also a substitute that no flesh-and-blood human could ever hope to out-compete, especially when it comes to mindless, sycophantic devotion. Lego Just Released Pre-Orders for Its Game Boy Set The 2025 MacBook Air is at its Lowest Price Ever The Best 4K TVs for an Immersive Watch Each Time The Best OLED TVs You Can Buy Right Now</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-62510-6'>An automated framework for exploring and learning potential-energy surfaces</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 11:06:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 7666 (2025) Cite this article Machine learning has become ubiquitous in materials modelling and now routinely enables large-scale atomistic simulations with quantum-mechanical accuracy. However, developing machine-learned interatomic potentials requires high-quality training data, and the manual generation and curation of such data can be a major bottleneck. Here, we introduce an automated framework for the exploration and fitting of potential-energy surfaces, implemented in an openly available software package that we call autoplex (‘automatic potential-landscape explorer'). We discuss design choices, particularly the interoperability with existing software architectures, and the ability for the end user to easily use the computational workflows provided. We show wide-ranging capability demonstrations: for the titanium–oxygen system, SiO2, crystalline and liquid water, as well as phase-change memory materials. More generally, our study illustrates how automation can speed up atomistic machine learning in computational materials science. Machine-learned interatomic potentials (MLIPs) are now established as the method of choice for large-scale, quantum-mechanically accurate atomistic simulations1,2,3,4,5, with applications ranging from high-pressure research6,7,8 to the discovery of molecular reaction mechanisms9,10 and even to the realistic modelling of proteins11. MLIPs are trained on quantum-mechanical reference data—typically derived from density-functional theory (DFT)—using a variety of methods from linear fits12,13,14 and Gaussian process regression15 to neural-network architectures16,17,18,19,20,21,22. Traditionally, MLIPs have been largely hand-crafted models, built using configurations manually tailored for domain-specific tasks23,24,25,26, such as the fracture of silicon24 or the crystallisation of Ge–Sb–Te memory materials26. More recently, a trend has emerged towards pre-trained or ‘foundational' MLIPs27,28: these models are fitted to large datasets including many chemical elements, and can be fine-tuned for downstream tasks27,29. With sophisticated MLIP fitting frameworks available and continuously improving, we argue that the next area of innovation lies in the data used to train the models30,31. The aforementioned fine-tuning is one example of the more general challenges in this field: constructing high-quality datasets still remains a non-trivial, time- and labour-intensive aspect of MLIP model development, and (more) efficient methods for data generation are needed. Commonly, active-learning strategies are now used to iteratively optimise datasets by identifying rare events and selecting the most relevant configurations via suitable error estimates32,33,34. Active learning has been widely used to explore phase transitions35,36,37 and chemical reactions38,39,40. And yet, such methods often still rely on costly ab initio molecular dynamics (MD) computations to expand and refine the training datasets. Currently available ‘foundational' MLIPs are typically fitted to a dataset comprising relaxation trajectories of diverse crystalline materials sourced from the Materials Project initiative41,42. The present work is concerned with the—somewhat orthogonal—question of how one can build an MLIP model from scratch: exploring local minima but also highly unfavourable regions of a given potential-energy surface, which need to be taught to a robust potential. Previous work showed that random structure searching (RSS) provides a particularly promising approach for the exploration and iterative fitting of configurational space (Fig. The principle of the original RSS approach, known as Ab Initio Random Structure Searching (AIRSS)50,51, is illustrated by panels a–b in Fig. 1, and the approach proposed in ref. 43 unifies it with MLIP fitting: using gradually improved potential models to drive the searches, without relying on any first-principles relaxations (only requiring DFT single-point evaluations) or pre-existing force fields. We note that AIRSS has been used as part of the approach for developing the recently described ‘graph networks for materials exploration' (GNoME)52 and MatterSim53 models to create structurally diverse training data. From left to right, the process involves: (a) the random sampling of structures, using suitable constraints (in this simple cartoon, we assume that we are searching for crystal structures of a binary compound, containing one atom of each type under periodic boundary conditions, with xi denoting the atomic coordinate vector of the i-th structure in configuration space); (b) the relaxation of those structures into local minima which correspond to relevant crystal structures; and (c) the fitting of machine-learned (ML) potentials. Panel (d) provides a high-level overview of the workflow structure: the left-hand side is a cartoon of a 'flow' consisting of two arbitrary 'jobs' to be carried out in sequence, and the schematic in the main part of the figure summarises our approach taken in autoplex. We colour-code the different key steps in white (random structure generation, here using the buildcell code of Ab Initio Random Structure Searching, AIRSS50,51), red (iterative exploration), yellow (sampling and single-point computations, the latter using density functional theory, DFT), and blue (fitting of a machine-learned interatomic potential, MLIP). To date, these RSS-related approaches still depend heavily on the user's expertise and time and are by no means trivial to implement. This challenge is particularly apparent for very large training datasets, where manually executing and monitoring tens of thousands of individual tasks is practically impossible. A similar challenge was previously observed in DFT-driven materials discovery, and automation approaches have been developed in response54: numerous workflow systems can now be used to streamline first-principles materials exploration55,56,57,58,59. Owing to such efforts, together with high-performance computing facilities, DFT-driven high-throughput simulations have become commonplace today and have played an important role in the computational discovery of new materials60,61,62. However, the same level of maturity has not yet been achieved for the full development pipeline of MLIPs (exploration, sampling, fitting, refinement): although important steps have recently been made63,64,65,66,67,68,69, the development of fully automated workflows for MLIPs remains in high demand. Here, we describe an automated implementation of iterative exploration and MLIP fitting through data-driven random structure searching, focusing on both the automation infrastructure and its implications for materials modelling applications. We show that, within the open-source autoplex code we have developed, MLIP fitting can be carried out in a largely automated way on high-performance computing systems and in a high-throughput manner—and we show how the resulting potentials are robust and useful, especially given the ease with which they can be created from scratch. Note that our benchmarks here are restricted to bulk systems, whereas surfaces, interfaces, and reaction pathways remain important and challenging frontiers for future extension. We expect that this work will contribute to the mainstream uptake of ML-driven atomistic modelling in the wider community in the years ahead. The autoplex framework is a modular set of software that is interfaced to widely-used computational and automation infrastructure where applicable. In particular, our code follows the core principles of (and reuses some functionalities implemented in) the atomate270 framework, which in turn underpins the Materials Project initiative41,42. The code is available openly via GitHub, distributed under a permissive licence, and accompanied by documentation to facilitate its uptake. In the present work, we mainly use the Gaussian approximation potential (GAP)15 framework to drive exploration and potential fitting, leveraging the data efficiency of GAP and building on previous successful applications for this purpose43,47. We do note that autoplex is designed to accommodate other MLIP architectures as well. We demonstrate the validity of the method using key examples, moving up in difficulty from elemental silicon to the polymorphs of TiO2, and onwards to the full binary titanium–oxygen system. Figure 2 shows the evolution of energy prediction errors for relevant crystalline modifications with an increasing number of DFT single-point evaluations used to create GAP-RSS models. Each panel in Fig. 2 corresponds to a separate round of iterative, automated training using autoplex (details are provided in Supplementary Note 1). Each step encompasses 100 single-point DFT evaluations whose results are added to the training dataset. The root mean square error (RMSE) between the predicted and reference energies is used as a performance metric to evaluate models. We show the results of autoplex runs, characterised by the energy prediction error of the models, for gradually more complex scenarios: (a) the diamond-type and β-tin-type form of elemental silicon, as well as the open-framework oS24 allotrope described in ref. 71; (b) the common rutile and anatase forms of TiO2, as well as the less common 'B' polymorph; and (c) three phases from the full binary Ti–O system. Note that the exploration for panel (b) included only stoichiometrically precise TiO2 structures, whereas that in panel (c) used a variable composition range. The data are shown in a style similar to ref. The grey-shaded area indicates errors below 0.001 eV at.−1, which we take to roughly correspond to the accuracy limit achievable using density functional theory (DFT)47. The dashed line marks an error of 0.01 eV at.−1. Source data are provided as a Source Data file. Silicon (Fig. 2a)—perhaps the classic test case for any materials simulation method—has a main allotrope with the diamond-type structure, and multiple higher-pressure forms, prominently the β-tin type structure. We also include the open-framework oS24 allotrope of silicon71 as an example of a metastable phase that has been experimentally characterised and includes lower-symmetric atomic environments, as a more challenging test case for autoplex. The tests in Fig. 2a are similar to earlier work in ref. 47 and, for consistency, use the same DFT parameters as in that previous work (e.g., the same exchange–correlation functional). All three allotropes are well described to within an accuracy on the order of 0.01 eV at.−1: the highly symmetric diamond- and β-tin-type structures with  ≈ 500 DFT single-point evaluations, the oS24 structure within a few thousand (Fig. We take 0.01 eV at.−1 to be a ‘sensible' accuracy target for random exploration, and indicate this value by dashed horizontal lines in Fig. The higher numerical error for β-tin-type compared to diamond-type silicon (green vs. purple in Fig. 2a) is consistent with previous work on a general-purpose GAP model for the element24 as well as an earlier GAP-RSS study47. The binary oxide, TiO2, is structurally highly diverse, therefore forming a suitable next target for testing a crystal-structure searching method. The compound mainly exists in two common forms—rutile and anatase—which contain distorted octahedrally coordinated [TiO6] units and differ in the connectivity of the coordination polyhedra. We also include the bronze-type (‘B-') polymorph of TiO2, which is less abundant, but has been of interest for battery research72,73. Figure 2b shows that while the two main polymorphs are again correctly recovered, and the prediction error for TiO2-B reduces to a few tens of meV at.−1 as well, the latter polymorph appears to be distinctly more difficult to ‘learn' than the two simpler ones. We finally study the exploration of a full binary system containing multiple phases with varied stoichiometric compositions. In Fig. 2c, we present the results of testing our approach on compounds with different stoichiometric compositions (and electronic structure), viz. Ti2O3, TiO, and Ti2O. While we truncate the plot at 0.001 eV at.−1, we emphasise that we already consider achieving an accuracy of  ≈ 0.01 eV at.−1 to be sufficient in this test of random exploration. It is fair to observe that compared to simpler phases such as rutile and anatase, achieving the target accuracy in this case requires a greater number of iterations, as the search space is more complex. We explore the behaviour for RSS runs up to 5000 structures in Supplementary Fig. Table 1 shows results for relevant main Ti–O polymorphs, evaluated with the TiO2 potential characterised in Fig. 2b, and with the Ti–O potential from Fig. This is a particularly instructive case because it allows us to probe the limits of the method: if only trained on TiO2, a GAP-RSS model will faithfully capture the polymorphs with this specific stoichiometric composition, but produces unacceptable errors when applied to compositions that deviate largely from the stoichiometry ( > 100 meV at.−1 for one of the Ti3O5 polymorphs, and  > 1 eV at.−1 for rocksalt-type TiO, for example). In contrast, by training the model for the full Ti–O system (cf. 2c), we are able to obtain an accurate description for several different phases. This example highlights the flexibility of autoplex in handling varying stoichiometric compositions, requiring no substantially greater effort from the user than that for a single stoichiometrically precise compound—all that is required is a change in input parameters for RSS, and probably a moderately increased amount of computational resources. We note that likely, the overall accuracy of those models can be improved further—for the time being, we report them as obtained with standard DFT and GAP fitting settings. An examination of potential data leakage is provided in Supplementary Note 1. A distinct advantage of the RSS approach for potential fitting is that it requires only single-point computations to generate the reference data43. We therefore posit that we are able to use autoplex to easily build high-quality potentials beyond the ‘standard' GGA functionals that are commonly used. We show here an example where higher-level data are crucial. The seemingly simple silicon dioxide, SiO2, has long posed challenges for atomistic modelling (see ref. 25 and references therein). We start by running our workflow using the economic Perdew–Burke–Ernzerhof (PBE) functional74. We then re-run the workflow with the same RSS parameter settings (Supplementary Note 1), this time using the Strongly Constrained and Appropriately Normed (SCAN) functional75. Figure 3a shows that for α-quartz, achieving high prediction accuracy (1 meV at.−1) with both PBE and SCAN functionals requires less than 10,000 CPU core hours, corresponding to nominal costs on the order of $100. For the structurally more complex α-cristobalite polymorph, SCAN incurs higher computational costs compared to PBE but still remains at the scale of 10,000s of core hours. a Energy prediction errors for α-quartz and α-cristobalite, plotted as a function of computational cost for autoplex runs using the Perdew–Burke–Ernzerhof (PBE) and Strongly Constrained and Appropriately Normed (SCAN) functionals, respectively. The grey-shaded region corresponds to errors below 0.001 eV at.−1. b DFT-computed energies of key SiO2 polymorphs, relative to α-quartz which is set as energy zero. Note that PBE erroneously predicts α-cristobalite as being more stable than α-quartz; this incorrect prediction is highlighted by the orange shading. c Energy difference between α-quartz and α-cristobalite as a function of the number of training structures for Gaussian approximation potential (GAP) models trained with PBE (GAP@PBE) and SCAN (GAP@SCAN). The symbol E denotes the energy. The dashed line represents the benchmark energy difference between α-quartz and α-cristobalite obtained from DFT calculations. Source data are provided as a Source Data file. Why SCAN? The importance of using this higher-rung functional in this case becomes apparent when inspecting the absolute energy predictions of the different MLIPs. PBE fails to correctly reproduce the stability ordering of SiO2 polymorphs, incorrectly predicting that α-cristobalite is more stable than α-quartz (identified by a negative value in Fig. 3b, highlighted in orange). This is an issue that is well known for some DFT methods (see, e.g., an early study in ref. By contrast, SCAN does identify α-quartz as stable—and indeed this DFT level has been used to train an MLIP for SiO2 before25, as well as a more comprehensive one for the binary Si–O system77. Further work could now include a direct benchmarking of our candidate RSS-derived MLIPs, as well as other data-generation workflows65,66,67,68, against the models of refs. Figure 3c illustrates the energy difference between α-cristobalite and α-quartz as a function of the number of training structures for GAP models trained with PBE and SCAN. The results indicate that the stability predictions from both GAP-RSS runs align well with the respective DFT results—but because of the underlying training data, the GAP@SCAN model is qualitatively correct (ΔE > 0), whereas the GAP@PBE model is not (ΔE < 0). Table 2 shows the results for additional SiO2 polymorphs: aside from α-cristobalite, the stability order of moganite and tridymite with respect to α-quartz is also inaccurately predicted by DFT@PBE and GAP@PBE. In contrast, DFT@SCAN and GAP@SCAN successfully capture the stability of different crystal structures with overall accuracy. However, in terms of numerical precision, there is no significant difference between the performance of the GAP@PBE and GAP@SCAN models. Even polymorphs with more complex unit cells are still predicted within our primary accuracy target of 10 meV at.−1. An energy–volume analysis demonstrates that the structures generated through RSS iterations tend to approach the ground-state phases of SiO2 polymorphs (Supplementary Fig. Apart from crystalline inorganic solids, we assess the ability of our approach to describe condensed-phase molecular systems. Among them, water is particularly challenging, where, in addition to coexisting covalent bonds and hydrogen bonds, van der Waals forces also play non-negligible roles. Such interatomic complexity makes the modelling of water highly sensitive to the choice of DFT functionals (a systematic investigation can be found in ref. Here, we train a GAP-RSS potential at the level of revPBE-D3 with zero-damping79, which has been shown to reproduce the experimental radial distribution functions (RDFs) of liquid water with accuracy comparable to hybrid functionals80,81, but at much lower computational cost. Beyond GAP, we explore the use of a different MLIP fitting architecture, focusing on graph-neural-network potentials, aiming to better capture the aforementioned interactions in different condensed phases of water. Figure 4 characterises results for liquid water and ice polymorphs. We run GAP-RSS iterations as before, during which the model progressively identifies an increasing variety of hydrogen-bonded environments (Supplementary Fig. The resulting potentials provide an overall correct description of the structure of liquid water in MD simulations (Fig. 4a), although the intensity of the first peak in the O–O RDF is more pronounced in the simulation than in experiment. We do not present the H ⋯ H interactions here, as our MD simulations do not account for nuclear quantum effects, which significantly influence the first peak of the RDF—see, e.g., ref. 82 for a discussion of these effects. a Radial distribution functions (RDFs) of H–O and O–O pairs comparing experimental data (H–O: ref. 122) with Gaussian approximation potential (GAP) model predictions at 300 K. The symbol r denotes the radial distance from a reference particle. b Same for predictions from a NequIP model, fitted to the same training set as the GAP model. c Average hydrogen-bond number as a function of simulation time for GAP and NequIP models. The hydrogen-bond criterion is defined as the O–O distance being smaller than 3.5 Å and the O–H–O angle being larger than 140°. d Comparison of predicted energies from GAP and NequIP models fitted to the GAP-driven random structure searching (GAP-RSS) dataset, against density functional theory (DFT) energies for various ice polymorphs89, including those experimentally observed (labelled 'expt.') and those theoretically hypothesised (labelled 'hyp.'). For this test, structures were taken from ref. 89, and re-evaluated with DFT using the revPBE-D3(zero) functional79. This plot highlights the improved extrapolation capability of the NequIP model, compared to the initial GAP-RSS one, in capturing energy trends across different phases. Source data are provided as a Source Data file. With an initial GAP-RSS-based dataset available, we carried out fits using the NequIP architecture19, and used the resulting models to drive MD simulations in LAMMPS83. We used the implementations of the NequIP architecture, the training loop, and the relevant pair style from the graph-pes package84 to do this (hyperparameters are provided in Supplementary Note 1). We find that both GAP and NequIP models fitted to the same GAP-RSS dataset can accurately describe key structural features of liquid water (Fig. 4a–b), with the NequIP model showing some improvements for the second peak in the H–O RDF and the first peak in the O–O RDF (Fig. We emphasise that both fits could most likely be improved by using hand-crafted datasets: the aim of Fig. 4 is not to benchmark specific architectures, but to test what type of practical use can be gained from an MLIP model fitted purely to automatically generated RSS data. The unique properties of liquid water are usually attributed to its strong hydrogen-bond network85,86. Figure 4c shows that both our GAP and NequIP models, fitted to the same autoplex-generated GAP-RSS dataset, can appropriately describe the number of hydrogen bonds per molecule in liquid water in MD simulations, which matches the previously reported range of 3.48 to 3.84 based on different density functionals87. The NequIP model predicts a slightly higher average than the original GAP-RSS one, around 3.5, which is closer to the upper end of the range. We note in passing that MD simulations using random-search-based ‘ephemeral data-derived potentials' (EDDPs) have been presented recently88, in that case for hydrogen diffusion in ScH12. Beyond liquid water, we also tested our models by predicting the energies of 54 ice crystal structures (structures taken from ref. Here, the results from the GAP-RSS model are highly scattered, whereas the NequIP model shows much improved predictive accuracy. The poor performance of the GAP model could be due to the sparsity of low-density phases in the dataset—which, in turn, would underscore the extrapolation capability of the NequIP model, allowing it to better handle structures that are substantially different from the training data. The above discussion suggests that the GAP-RSS dataset is not only effective for training a GAP model itself, but is also beneficial for use with other, more complex fitting frameworks. Future work could explore the suitability of autoplex-generated (GAP-) RSS datasets as a pre-training task for subsequent fine-tuning of MLIP models, building on our previous study which showed that pre-training NequIP models can enhance not only numerical quality in the low-data regime, but also stability in MD simulations90. We finally demonstrate the applicability of autoplex to an inorganic material system of ‘real-world' interest. For this purpose, we focus on two ternary chalcogenides, viz. Ge1Sb2Te4 and In3Sb1Te2, which are relevant for applications in phase-change memory devices91,92,93. We have recently hand-built a GAP model for compositions along the pseudo-binary line between GeTe and Sb2Te3 (referred to as ‘GST' alloys)26, enabling an accurate description of the amorphous structure (digital ‘zero bits') and its crystallisation (‘0 → 1'). One of the challenges in modelling amorphous GST is the formation of tetrahedral structural motifs (Fig. 5a), which are relevant for ageing phenomena: the amount of those tetrahedra has been argued to change over time, affecting the stability of the zero bits94. However, this hand-crafted GAP model (denoted ‘GST-GAP-22') took months to complete, involving multiple runs of domain-specific iterations to cover the structural complexity of the Ge–Sb–Te system26. This makes Ge1Sb2Te4 an excellent candidate for testing our automated, RSS-based workflows in computational practice. a, b Schematics of the structural transitions for Ge1Sb2Te4 and In3Sb1Te2, respectively, between crystalline ('1') and amorphous ('0') phases. Atoms are colour-coded as follows: Ge in green, Sb in purple, Te in grey, and In in yellow. (c) Radial distribution function (RDF) and (d) distribution of shortest-path rings in a-Ge1Sb2Te4. e, f Same but for a-In3Sb1Te2. The symbol r represents the radial distance from a reference particle. All ab initio molecular dynamics (AIMD) data for a-Ge1Sb2Te4 are sourced from the literature26. For a-In3Sb1Te2, the AIMD-derived RDF data for a density of 6.448 g cm−3 are sourced from ref. 97, while the AIMD data at 6.120 g cm−3 for the RDF in panel (e) and the ring distribution in panel (f) are obtained from our own simulations with CP2K123,124 (Supplementary Note 1). The label 'GAP' in these panels refers to Gaussian approximation potential (GAP) models obtained via random structure searching (RSS). The boxplots in panels (d) and (f) represent the distribution of values across trajectories (see Supplementary Note 1 for statistics). The central line in each box indicates the median, the box spans the inter-quartile range (IQR), and the whiskers extend to 1.5 × IQR. Outliers are shown as individual points. g A crystallisation simulation of Ge1Sb2Te4 using our GAP-RSS-derived potential, showing the progressive crystallisation over a simulation period of 350 ps. Atoms are colour-coded based on a kernel-based 'crystallinity' measure99,100: yellow indicates high crystallinity, and red indicates amorphous regions. Source data are provided as a Source Data file. Compared to GST, In3Sb1Te2 (‘IST' in the following) is a rather unconventional phase-change material, with structural building blocks slightly different from those of GST. The latter alloys structurally resemble their constituent binary phases (viz. GeTe and Sb2Te3), and they can take disordered and defective rocksalt-like metastable structures (see ref. 95 and references therein). The relevant ternary In–Sb–Te compound, viz. In3Sb1Te2, crystallises in a disordered rocksalt-type structure with no substantial amount of cation vacancies96. Unlike GST, which does contain such vacancies, IST has all cation sites fully occupied by In atoms (Fig. 5b), with no notable cation disorder. Instead, its structural complexity arises from anion disorder, as Sb and Te share the same substructure. A previous ab initio MD study revealed that amorphous IST exhibits a large number of four-, five-, and six-membered rings97 (Fig. 5b, right), indicating that its medium-range order is more complex than that of amorphous GST (where four-membered rings are predominant). In the context of MLIP development, IST is an example of a less widely explored material: there are not as many ab initio MD studies as for GST, nor is there an existing potential model to our knowledge. We argue that instead of manually constructing a training dataset, the user can now use automated approaches, such as the one in autoplex, to study less-common functional materials, at least as a starting point. Figure 5c–g characterises the performance of GAP-RSS models for both phase-change materials. We first quantified the local structural properties by computing RDFs (Fig. 5c, e), and then calculated ring statistics which allow us to assess the medium-range order in those structures (Fig. The performance of the GAP-RSS models is encouraging, given how fast and inexpensive it has been to fit compared to an existing MLIP for Ge–Sb–Te compounds. Methodologically, the RSS approach in autoplex provides structural diversity by sampling from small cells, a strategy supported by previous literature48,98. We obtain a robust potential using 46% fewer atomic environments for Ge1Sb2Te4 compared to the training dataset in ref. 26, which comprises 49,056 atomic environments for this specific composition (and more for chemically related compounds). More importantly, however, the approach of ref. 26 was based on the expert-guided construction of training datasets, requiring substantial human effort and domain knowledge. In contrast, the RSS methodology enables broad exploration of the potential-energy landscape, requiring much less prior knowledge. In terms of implementation, the autoplex framework largely automates the iterative ML-driven RSS process, thereby enhancing the efficiency and ease of applying this methodology. The results for both phase-change materials show that our approach not only works well for materials with established domain knowledge (GST) but also performs effectively for materials for which so far there is limited domain knowledge (IST). We finally ran a GAP-driven MD simulation of the crystallisation process in Ge1Sb2Te4 (Fig. 5g), corresponding to the SET operation in memory devices (‘0 → 1'). We used a kernel similarity metric99 to quantify the gradual structural ordering process, as in our previous work (ref. The simulation shows a rapid growth proceeding at the amorphous–crystalline interface, leading to the formation of a largely ordered, defective rocksalt-like structure. We note that this crystallisation simulation using GAP was essentially completed after 350 ps, more quickly than what was seen in ab initio (DFT-based) MD simulations100 and using the hand-crafted GST-GAP-22 potential26. However, the former approach is computationally highly demanding100, and the latter is a specialised MLIP that has been deliberately trained on ‘domain-specific' configurations that correspond to the intermediate steps between fully amorphous and fully recrystallised Ge–Sb–Te materials. Our GAP-RSS based model, used to drive the simulations characterised in Fig. 5g, is trained in a much simpler way, and using only relatively small-scale configurations—perhaps it could serve as a starting point for constructing subsequent, more complex MLIP training datasets. Further tests for other phase-change memory materials are planned for the future. Automation is one of the major open challenges in ML potentials for materials and poised to accelerate their development into general mainstream simulation models. We have here shown how iterative exploration and MLIP fitting, initially proposed within the GAP-RSS framework43, can be automated substantially and at scale by integration with existing software ecosystems. The resulting autoplex code is openly available and free to use, and we expect that in the long run, it will develop into a computational ecosystem of its own for the next generation of ML-driven materials modelling. Although the present study has focused on bulk materials, the ML-driven RSS method is, in principle, well-suited for exploring a broader range of configurations (e.g., clusters, surfaces, and interfaces), as supported by previous applications of the established AIRSS method101. Accurately sampling complex dynamical processes, such as reaction pathways and adsorption sites in surface catalysis, remains a common challenge for various structural exploration methods, including RSS and MD, as these processes involve overcoming substantial energy barriers. In this context, the MLIPs produced by our RSS method can provide starting points for further refinement via active learning—an approach that has already proven effective in tackling similar dynamical challenges40. Such functionalities could be integrated into autoplex in the future. Our work also contributes to addressing the wider-ranging open question about how MLIPs are best developed and used going forward. Random structure searching provides a core approach to generating robust potentials, and our present results suggest a remarkable amount of stability that can be gained from RSS using small cells alone: therefore, ML-driven iterative RSS appears to emerge as a standard technique for at least providing a starting point for potential fitting102. We have previously pointed out that RSS datasets can constitute useful benchmark tasks for evaluating MLIP models103,104, and we expect the growing usefulness of this aspect as systematic benchmarking becomes more important in the community. In the years ahead, we expect that automated approaches to dataset construction—including the one in autoplex we have presented here—will play an increasingly important role in the field. With these efforts, and together with the demonstrated capabilities of advanced MLIP fitting architectures27,28,52,53, universal ML models for atomistic simulations could become widely established—which could make ML-driven modelling the genuine default in the field, just like direct quantum-mechanical modelling has been the default for many years. MLIPs represent a given quantum-mechanical potential-energy surface. The models used herein are based on a local (atom-wise) decomposition of the total energy15,16, where the atomic energies, \(\varepsilon\), are learned as a function of the atom's local environment, expressed through a general structural descriptor, qi, and the ML-predicted total energy, \({E}\), is obtained by summing over the per-atom contributions. For most of the present study, we used the GAP framework15, together with the SOAP descriptor99 to featurise atomic environments. However, our approach is more general, and we have also implemented interfaces to other architectures and codes, including ACEpotentials.jl105, NequIP19, M3GNet21, and MACE20, in the current public version of autoplex. We refer to the original literature for details of these methods. Random structure searching is an established approach to exploring configurational spaces and has seen large success in the context of the AIRSS framework by Pickard and Needs50,51. In ref. 43, it was proposed to combine RSS exploration with the fitting of potential-energy surfaces, gradually improving the models through iterative training (which had already been in itself a standard approach using iterative MD23) and not requiring any DFT-based relaxation but only single-point DFT evaluations, running all relaxations with GAP models instead. Note that the combination of structural searches and iterative MLIP fitting is not at all restricted to GAP or (AI)RSS—it has been demonstrated for other frameworks as well44,46. The approach was extended in ref. 47 by including appropriate selection steps (both structurally and energetically based). Subsequent work introduced the wfl software65. The latter work is different from ours in that it runs workflows through a custom implementation, whereas autoplex interfaces to atomate2 and its diverse set of DFT-based workflows where possible. Within autoplex, we have incorporated several methodological features. Our framework now supports multiple sets of buildcell input parameters, which define the scope of the RSS search and directly impact the diversity of the generated structures. We have also implemented a Hookean repulsion force to prevent atoms from approaching each other too closely, which can help to produce more physically plausible configurations. Furthermore, if a pre-existing MLIP along with its training dataset is available, our RSS framework can be readily used to generate supplementary data and further refine the potential. One common approach is to combine an existing dataset with configurations generated during the initial RSS stage and use the merged data to train a new MLIP that subsequently drives iterative RSS exploration. Alternatively, a trained MLIP can be used to initiate the first round of RSS iterations, after which the newly generated data may be merged with the original training dataset to improve the model, which is then used in subsequent RSS iterations. Finally, users may opt to generate an RSS dataset from scratch without relying on any prior data or potentials; once the structure searching is complete, this RSS dataset can be merged with pre-existing data for training a new MLIP. More details on the procedures and parameters of the RSS workflows are provided in Supplementary Note 1. The scripts and input parameters for running the workflows are provided in ref. In the present work, we focus on automating RSS and RSS-driven iterative potential fitting. We describe a software implementation that is connected to the atomate2 ecosystem. atomate2 is a library of computational materials science workflows that have mostly been developed in the context of the Materials Project41. As in atomate2, we rely on jobflow107 for writing workflows and jobflow-remote or fireworks108 for executing workflows and scheduling computing tasks. For handling DFT inputs and outputs and interfaces with ML potentials, we use pymatgen109 and the Atomic Simulation Environment (ASE)110. Specifically, we use existing atomate2 workflows for automatic DFT computations. For the MLIP fitting part, we provide direct interfaces to relevant software. We further automate the intermediate stage between data generation (typically via DFT) and MLIP fitting. Reference data were obtained using projector augmented-wave (PAW) potentials111,112 as implemented in the Vienna Ab Initio Simulation Package (VASP)112,113, version 6.3.2. To describe the effects of exchange and correlation, we used the Perdew–Burke–Ernzerhof (PBE) functional74, its revised parameterisation for solids (PBEsol)114, revPBE115 with the D3 dispersion correction79, as well as the Strongly Constrained and Appropriately Normed (SCAN) functional75, depending on the specific material system to be studied. The DFT settings were similar to previous work for silicon47, Ti–O116, SiO2 (ref. 25), water29, and phase-change materials26 (here using the same settings for GST and IST), and adapted where appropriate. Structures were visualised using OVITO117. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data supporting this work are openly available via GitHub at https://github.com/autoatml/papers-autoplex-rss, with a permanent version available via Zenodo at https://doi.org/10.5281/zenodo.15720026 (ref. 106) and https://doi.org/10.5281/zenodo.15258384 (ref. Source data are provided with this paper. The autoplex software is openly available at https://github.com/autoatml/autoplex. The code is under ongoing development; a copy of the version used for the results presented herein (v0.0.7) is deposited in the Zenodo repository (https://doi.org/10.5281/zenodo.14169361, ref. Behler, J. First principles neural network potentials for reactive simulations of large molecular and condensed systems. Google Scholar Deringer, V. L., Caro, M. A. & Csányi, G. Machine learning interatomic potentials as emerging tools for materials science. Google Scholar Noé, F., Tkatchenko, A., Müller, K.-R. & Clementi, C. Machine learning for molecular simulation. Google Scholar Unke, O. T. et al. Machine learning force fields. Google Scholar Friederich, P., Häse, F., Proppe, J. & Aspuru-Guzik, A. Machine-learned potentials for next-generation matter simulations. Google Scholar Cheng, B., Mazzola, G., Pickard, C. J. & Ceriotti, M. Evidence for supercritical behaviour of high-pressure liquid hydrogen. Google Scholar Deringer, V. L. et al. Origins of structural and electronic transitions in disordered silicon. Google Scholar Zong, H. et al. Free electron to electride transition in dense liquid potassium. Google Scholar Westermayr, J. et al. Deep learning study of tyrosine reveals that roaming can lead to photodamage. Google Scholar Zhang, S. et al. Exploring the frontiers of condensed-phase chemistry with a general reactive machine learning potential. Google Scholar Wang, T. et al. Ab initio characterization of protein molecular dynamics with AI2BMD. Google Scholar Thompson, A. P., Swiler, L. P., Trott, C. R., Foiles, S. M. & Tucker, G. J. Spectral neighbor analysis method for automated generation of quantum-accurate interatomic potentials. Google Scholar Shapeev, A. Moment tensor potentials: A class of systematically improvable interatomic potentials. Multiscale Model. Google Scholar Drautz, R. Atomic cluster expansion for accurate and transferable interatomic potentials. Google Scholar Bartók, A. P., Payne, M. C., Kondor, R. & Csányi, G. Gaussian approximation potentials: the accuracy of quantum mechanics, without the electrons. Google Scholar Behler, J. & Parrinello, M. Generalized neural-network representation of high-dimensional potential-energy surfaces. Google Scholar Zhang, L., Han, J., Wang, H., Car, R. & E, W. Deep potential molecular dynamics: a scalable model with the accuracy of quantum mechanics. Google Scholar Unke, O. T. & Meuwly, M. Physnet: A neural network for predicting energies, forces, dipole moments, and partial charges. Theory Comput. Google Scholar Batzner, S. et al. E(3)-equivariant graph neural networks for data-efficient and accurate interatomic potentials. Google Scholar Batatia, I., Kovacs, D. P., Simm, G., Ortner, C. & Csányi, G. MACE: Higher order equivariant message passing neural networks for fast and accurate force fields. Neural Inf. Google Scholar Chen, C. & Ong, S. P. A universal graph deep learning interatomic potential for the periodic table. Google Scholar Deng, B. et al. CHGNet as a pretrained universal neural network potential for charge-informed atomistic modelling. Google Scholar Sosso, G. C., Miceli, G., Caravati, S., Behler, J. & Bernasconi, M. Neural network interatomic potential for the phase change material GeTe. Google Scholar Bartók, A. P., Kermode, J., Bernstein, N. & Csányi, G. Machine learning a general-purpose interatomic potential for silicon. Google Scholar Erhard, L. C., Rohrer, J., Albe, K. & Deringer, V. L. A machine-learned interatomic potential for silica and its relation to empirical models. npj Comput. Google Scholar Zhou, Y., Zhang, W., Ma, E. & Deringer, V. L. Device-scale atomistic modelling of phase-change memory materials. Google Scholar Zhang, D. et al. DPA-2: A large atomic model as a multi-task learner. npj Comput. Batatia, I. et al. A foundation model for atomistic materials chemistry Preprint at https://arxiv.org/abs/2401.00096 (2024). Kaur, H. et al. Data-efficient fine-tuning of foundational models for first-principles quality sublimation enthalpies. Faraday Discuss. Ben Mahmoud, C., Gardner, J. L. A. & Deringer, V. L. Data as the next challenge in atomistic machine learning. Kulichenko, M. et al. Data generation for machine learning interatomic potentials and beyond. Smith, J. S., Nebgen, B., Lubbers, N., Isayev, O. & Roitberg, A. E. Less is more: sampling chemical space with active learning. van der Oord, C., Sachs, M., Kovács, D. P., Ortner, C. & Csányi, G. Hyperactive learning for data-driven interatomic potentials. npj Comput. Kulichenko, M. et al. Uncertainty-driven dynamics for active learning of interatomic potentials. Vandermause, J. et al. On-the-fly active learning of interpretable Bayesian force fields for atomistic rare events. npj Comput. Vandenhaute, S., Cools-Ceuppens, M., DeKeyser, S., Verstraelen, T. & Van Speybroeck, V. Machine learning potentials for metal-organic frameworks using an incremental learning approach. npj Comput. Xie, Y. et al. Uncertainty-aware molecular dynamics from Bayesian active learning for phase transformations and thermal transport in SiC. npj Comput. Young, T. A., Johnston-Wood, T., Deringer, V. L. & Duarte, F. A transferable active-learning strategy for reactive molecular force fields. Guan, X., Heindel, J. P., Ko, T., Yang, C. & Head-Gordon, T. Using machine learning to go beyond potential energy surface benchmarking for chemical reactivity. Schaaf, L. L., Fako, E., De, S., Schäfer, A. & Csányi, G. Accurate energy barriers for catalytic reaction pathways: an automatic training protocol for machine learning force fields. npj Comput. Jain, A. et al. Commentary: The Materials Project: A materials genome approach to accelerating materials innovation. APL Mater. Horton, M. K. et al. Accelerated data-driven materials science with the Materials Project. Deringer, V. L., Pickard, C. J. & Csányi, G. Data-driven learning of total and local energies in elemental boron. Tong, Q., Xue, L., Lv, J., Wang, Y. & Ma, Y. Accelerating CALYPSO structure prediction by data-driven learning of a potential energy surface. Faraday Discuss. Deringer, V. L., Proserpio, D. M., Csányi, G. & Pickard, C. J. Data-driven learning and prediction of inorganic crystal structures. Faraday Discuss. Podryabinkin, E. V., Tikhonov, E. V., Shapeev, A. V. & Oganov, A. R. Accelerating crystal structure prediction by machine-learning interatomic potentials with active learning. Bernstein, N., Csányi, G. & Deringer, V. L. De novo exploration and self-guided learning of potential-energy surfaces. npj Comput. Pickard, C. J. Ephemeral data derived potentials for random structure search. Pickard, C. J. Beyond theory-driven discovery: introducing hot random search and datum-derived structures. Faraday Discuss. Pickard, C. J. & Needs, R. J. High-pressure phases of silane. Pickard, C. J. & Needs, R. J. Ab initio random structure searching. Merchant, A. et al. Scaling deep learning for materials discovery. Yang, H. et al. MatterSim: A deep learning atomistic model across elements, temperatures and pressures Preprint at https://arxiv.org/abs/2405.04967 (2024). George, J. Automation in DFT-based computational materials science. Trends Chem. Curtarolo, S. et al. Aflow: an automatic framework for high-throughput materials discovery. Kirklin, S. et al. The open quantum materials database (OQMD): assessing the accuracy of DFT formation energies. npj Comput. Pizzi, G., Cepellotti, A., Sabatini, R., Marzari, N. & Kozinsky, B. AiiDA: automated interactive infrastructure and database for computational science. Mathew, K. et al. Atomate: A high-level interface to generate, execute, and analyze computational materials science workflows. Choudhary, K. et al. The joint automated repository for various integrated simulations (JARVIS) for data-driven materials design. npj Comput. Pyzer-Knapp, E. O., Suh, C., Gómez-Bombarelli, R., Aguilera-Iparraguirre, J. & Aspuru-Guzik, A. What is high-throughput virtual screening? A perspective from organic materials discovery. Gorai, P., Stevanović, V. & Toberer, E. S. Computationally guided discovery of thermoelectric materials. Hautier, G. Finding the needle in the haystack: Materials discovery and design through computational ab initio high-throughput screening. Janssen, J. et al. pyiron: An integrated development environment for computational materials science. Zhang, Y. et al. DP-GEN: A concurrent learning platform for the generation of reliable deep learning based potential energy models. Gelžinytė, E. et al. wfl Python toolkit for creating machine learning interatomic potentials and related atomistic simulation workflows. Guo, Y.-X., Zhuang, Y.-B., Shi, J. & Cheng, J. ChecMatE: a workflow package to automatically generate machine learning potentials and phase diagrams for semiconductor alloys. Menon, S. et al. From electrons to phase diagrams with classical and machine learning potentials: automated workflows for materials science with pyiron. npj Comput. Poul, M., Huber, L. & Neugebauer, J. Automated generation of structure datasets for machine learning potentials and alloys. npj Comput. Li, Z. et al. APEX: An automated cloud-native material property explorer. npj Comput. Ganose, A. M. et al. Atomate2: Modular workflows for materials science. Digital Discovery 4, 1944–1973 (2025). Kim, D. Y., Stefanoski, S., Kurakevych, O. O. & Strobel, T. A. Synthesis of an open-framework allotrope of silicon. Armstrong, A. R., Armstrong, G., Canales, J., García, R. & Bruce, P. G. Lithium ion intercalation into TiO2 B nanowires. Liang, S. et al. Bronze phase TiO2 as anode materials in lithium and sodium ion batteries. Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Sun, J., Ruzsinszky, A. & Perdew, J. P. Strongly constrained and appropriately normed semilocal density functional. Demuth, T., Jeanvoine, Y., Hafner, J. & Ángyán, J. G. Polymorphism in silica studied in the local density and generalized-gradient approximations. Erhard, L. C., Rohrer, J., Albe, K. & Deringer, V. L. Modelling atomic and nanoscale structure in the silicon-oxygen system through active machine learning. Villard, J., Bircher, M. P. & Rothlisberger, U. Structure and dynamics of liquid water from ab initio simulations: adding Minnesota density functionals to Jacob's ladder. Grimme, S., Antony, J., Ehrlich, S. & Krieg, H. A consistent and accurate ab initio parametrization of density functional dispersion correction (DFT-D) for the 94 elements H-Pu. Pestana, L. R., Mardirossian, N., Head-Gordon, M. & Head-Gordon, T. Ab initio molecular dynamics simulations of liquid water using high quality meta-GGA functionals. Marsalek, O. & Markland, T. E. Quantum dynamics and spectroscopy of ab initio liquid water: the interplay of nuclear and electronic quantum effects. Markland, T. E. & Ceriotti, M. Nuclear quantum effects enter the mainstream. Thompson, A. P. et al. LAMMPS - a flexible simulation tool for particle-based materials modeling at the atomic, meso, and continuum scales. Gardner, J. L. A.graph-pes: train and use graph-based ML models of potential energy surfaces. Poole, P. H., Sciortino, F., Grande, T., Stanley, H. E. & Angell, C. A. Effect of hydrogen bonds on the thermodynamic behavior of liquid water. Kumar, R., Schmidt, J. & Skinner, J. Hydrogen bonding definitions and dynamics in liquid water. Todorova, T., Seitsonen, A. P., Hutter, J., Kuo, I.-F. W. & Mundy, C. J. Molecular dynamics simulation of liquid water: Hybrid density functionals. Salzbrenner, P. T. et al. Developments and further applications of ephemeral data derived potentials. Monserrat, B., Brandenburg, J. G., Engel, E. A. & Cheng, B. Liquid water contains the building blocks of diverse ice phases. Gardner, J. L. A., Baker, K. T. & Deringer, V. L. Synthetic pre-training for neural-network interatomic potentials. Wuttig, M. & Yamada, N. Phase-change materials for rewriteable data storage. Selmo, S. et al. Low power phase change memory switching of ultra-thin In3Sb1Te2 nanowires. Zhang, W., Mazzarello, R., Wuttig, M. & Ma, E. Designing crystallization in phase-change materials for universal memory and neuro-inspired computing. Raty, J. Y. et al. Aging mechanisms in amorphous phase-change materials. Deringer, V. L., Dronskowski, R. & Wuttig, M. Microscopic complexity in phase-change materials and its role for applications. Schröder, T. et al. Disorder and transport properties of In3SbTe2 – an X-ray, neutron and electron diffraction study. Los, J. H., Kühne, T. D., Gabardi, S. & Bernasconi, M. First-principles study of the amorphous In3SbTe2 phase change compound. Meziere, J. A. et al. Accelerating training of MLIPs through small-cell training. Bartók, A. P., Kondor, R. & Csányi, G. On representing chemical environments. Xu, Y. et al. Unraveling crystallization mechanisms and electronic structure of phase-change materials by large-scale ab initio simulations. Schusteritsch, G. & Pickard, C. J. Predicting interface structures: from SrTiO3 to graphene. Deringer, V. L., Caro, M. A. & Csányi, G. A general-purpose machine-learning force field for bulk and nanostructured phosphorus. Morrow, J. D., Gardner, J. L. A. & Deringer, V. L. How to validate machine-learned interatomic potentials. Thomas du Toit, D. F., Zhou, Y. & Deringer, V. L. Hyperparameter optimization for atomic cluster expansion potentials. Theory Comput. Witt, W. C. et al. ACEpotentials.jl: A Julia implementation of the atomic cluster expansion. Liu, Y. et al. autoatml/papers-autoplex-rss: v1.0.0. Zenodo, https://doi.org/10.5281/zenodo.15720026 (2025). Rosen, A. S. et al. Jobflow: Computational Workflows Made Simple. Open Source Softw. Jain, A. et al. FireWorks: a dynamic workflow system designed for high-throughput applications. Ong, S. P. et al. Python Materials Genomics (pymatgen): a robust, open-source Python library for materials analysis. Hjorth Larsen, A. et al. The atomic simulation environment—a Python library for working with atoms. Blöchl, P. E. Projector augmented-wave method. Kresse, G. & Joubert, D. From ultrasoft pseudopotentials to the projector augmented-wave method. Kresse, G. & Furthmüller, J. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Perdew, J. P. et al. Restoring the density-gradient expansion for exchange in solids and surfaces. Zhang, Y. & Yang, W. Comment on “Generalized gradient approximation made simple”. Lyle, M. J., Pickard, C. J. & Needs, R. J. Prediction of 10-fold coordinated TiO2 and SiO2 structures at multimegabar pressures. Natl Acad. Stukowski, A. Visualization and analysis of atomistic simulation data with OVITO–the open visualization tool. Liu, Y. et al. Machine-learning-based interatomic potential models for “An automated framework for exploring and learning potential-energy surfaces". Zenodo, https://doi.org/10.5281/zenodo.15258384 (2025). Ertural, C. et al. autoatml/autoplex: v0.0.7. Zenodo, https://doi.org/10.5281/zenodo.14169361 (2024). Beckett, G. et al. ARCHER2 Service Description, Zenodo, https://doi.org/10.5281/zenodo.14507040 (2024). Soper, A. The radial distribution functions of water and ice from 220 to 673 K and at pressures up to 400 MPa. Skinner, L. B., Benmore, C. J., Neuefeind, J. C. & Parise, J. The structure of water around the compressibility minimum. Kühne, T. D., Krack, M., Mohamed, F. R. & Parrinello, M. Efficient and accurate Car-Parrinello-like approach to Born-Oppenheimer molecular dynamics. Kühne, T. D. et al. CP2K: An electronic structure and molecular dynamics software package - Quickstep: efficient and accurate electronic structure calculations. Download references We thank L.-B. Paşca and L. Wu for help with early tests. This work was supported through a UK Research and Innovation Frontier Research grant [grant number EP/X016188/1]. acknowledges funding from the EPSRC Centre for Doctoral Training in Inorganic Chemistry for Future Manufacturing (OxICFM), EP/S023828/1. We are grateful for computational support from the UK national high performance computing service, ARCHER2, for which access was obtained via the UKCP consortium and funded by EPSRC grant ref EP/X035891/1, as well as through a separate EPSRC Access to High-Performance Computing award120. Additionally, we acknowledge the Gauss Centre for Supercomputing e.V. (www.gauss-centre.eu) for funding this project by providing generous computing time on the GCS Supercomputer SuperMUC-NG at Leibniz Supercomputing Centre (www.lrz.de) (project pn73da) that enabled testing of the implementations in autoplex. Inorganic Chemistry Laboratory, Department of Chemistry, University of Oxford, Oxford, UK Yuanbin Liu, Joe D. Morrow, Natascia L. Fragapane, John L. A. Gardner, Yuxing Zhou & Volker L. Deringer Materials Chemistry Department, Federal Institute for Materials Research and Testing (BAM), Berlin, Germany Christina Ertural, Aakash A. Naik & Janine George Institute of Condensed Matter Theory and Solid-State Optics, Friedrich Schiller University Jena, Jena, Germany Aakash A. Naik & Janine George Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the RSS automation code in autoplex. are the core autoplex code developers and maintainers at the time of this writing. carried out numerical experiments. designed and supervised the research. drafted the manuscript, and all authors contributed to the final version. Correspondence to Janine George or Volker L. Deringer. The authors declare no competing interests. Nature Communications thanks Satoshi Watanabe and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Liu, Y., Morrow, J.D., Ertural, C. et al. An automated framework for exploring and learning potential-energy surfaces. Nat Commun 16, 7666 (2025). Download citation Received: 21 December 2024 Accepted: 21 July 2025 Published: 18 August 2025 DOI: https://doi.org/10.1038/s41467-025-62510-6 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02781-5'>High-throughput profiling of chemical-induced gene expression across 93,644 perturbations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 10:00:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In this Resource, we present an extensive dataset of chemical-induced gene signatures (CIGS), encompassing expression patterns of 3,407 genes regulating key biological processes in 2 human cell lines exposed to 13,221 compounds across 93,664 perturbations. This dataset encompasses 319,045,108 gene expression events, generated through 2 high-throughput technologies: the previously documented high-throughput sequencing-based high-throughput screening (HTS2) and the newly developed highly multiplexed and parallel sequencing (HiMAP-seq). Our results show that HiMAP-seq is comparable to RNA sequencing, but can profile the expression of thousands of genes across thousands of samples in one single test by utilizing a pooled-sample strategy. We further illustrate CIGS's utility in elucidating the mechanism of action of unannotated small molecules, like ligustroflavone and 2,4-dihydroxybenzaldehyde, and to identify perturbation-induced cell states, such as those resistant to ferroptosis. The full dataset is publicly accessible at https://cigs.iomicscloud.com/. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout The sample information and read counts for 3,407 genes across 2 cell lines (MDA-MB-231 and HEK293T cells) subjected to perturbation by 13,221 compounds in this study are available for download at https://cigs.iomicscloud.com/. Raw sequencing data are available from the corresponding author on reasonable request. Source data are provided with this paper. All other data generated during this study are included in this published article and its Supplementary Information. The source code used for the analyses presented in this study is available at https://github.com/Wang-lab302/CIGS/. DeRisi, J. L., Iyer, V. R. & Brown, P. O. Exploring the metabolic and genetic control of gene expression on a genomic scale. Chandrasekaran, S. N. et al. Three million images and morphological profiles of cells treated with matched chemical and genetic perturbations. The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Subramanian, A. et al. A next generation connectivity map: L1000 platform and the first 1,000,000 profiles. Exploring pharmacological active ingredients of traditional Chinese medicine by pharmacotranscriptomic map in ITCM. Stathias, V. et al. Drug and disease signature integration identifies synergistic combinations in glioblastoma. Li, H. et al. Versatile pathway-centric approach based on high-throughput sequencing to anticancer drug discovery. Shao, W. et al. Chemical genomics reveals inhibition of breast cancer lung metastasis by ponatinib via c-Jun. Wang, H. et al. Tumor immunological phenotype signature-based high-throughput screening for the discovery of combination immunotherapy compounds. Xing, F. et al. Activation of the c-met pathway mobilizes an inflammatory network in the brain microenvironment to promote brain metastasis of breast cancer. Giusto, E. et al. Graphene oxide nanoplatforms to enhance cisplatin-based drug delivery in anticancer therapy. Potential mechanisms for cancer resistance in elephants and comparative cellular response to DNA damage in humans. Phase I study of CC-90010, a reversible, oral BET inhibitor in patients with advanced solid tumors and relapsed/refractory non-Hodgkin's lymphoma. The novel bromodomain and extraterminal domain inhibitor INCB054329 induces vulnerabilities in myeloma cells that inform rational combination strategies. Picaud, S. et al. Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia. Gao, Y. et al. Overcoming resistance to the THZ series of covalent transcriptional CDK inhibitors. Identification of atuveciclib (BAY1143572), the first highly selective, clinical PTEFb/CDK9 inhibitor for the treatment of cancer. Rzymski, T. et al. SEL120-34A is a novel CDK8 inhibitor active in AML cells with high levels of serine phosphorylation of STAT1 and STAT5 transactivation domains. The BRD4 inhibitor JQ1 suppresses tumor growth by reducing c-Myc expression in endometrial cancer. Selective inhibition of tumor oncogenes by disruption of super-enhancers. Hu, J., Gu, W., Ma, N., Fan, X. & Ci, X. Leonurine alleviates ferroptosis in cisplatin-induced acute kidney injury by activating the Nrf2 signalling pathway. Fan, Z. et al. Nrf2-Keap1 pathway promotes cell proliferation and diminishes ferroptosis. Qiu, W. et al. Asiatic acid alleviates LPS-induced acute kidney injury in broilers by inhibiting oxidative stress and ferroptosis via activation of the Nrf2 pathway. Li, J. et al. PHDs-seq: a large-scale phenotypic screening method for drug discovery through parallel multi-readout quantification. Teder, H. et al. TAC-seq: targeted DNA and RNA sequencing for precise biomarker molecule counting. Ye, C. et al. DRUG-seq for miniaturized high-throughput transcriptome profiling in drug discovery. Li, J. et al. DRUG-seq provides unbiased biological activity readouts for neuroscience drug discovery. Bush, E. C. et al. PLATE-Seq for genome-wide regulatory network analysis of high-throughput screens. Yeakley, J. M. et al. A trichostatin A expression signature identified by TempO-Seq targeted whole transcriptome profiling. Peck, D. et al. A method for high-throughput gene expression signature analysis. Guo, Y. et al. YeastFab: the design and construction of standard biological parts for metabolic engineering in Saccharomyces cerevisiae. Efficient use of a crude drug/herb library reveals ephedra herb as a specific antagonist for TH2-specific chemokine receptors CCR3, CCR4, and CCR8. Protective effects of ligustroflavone, an active compound from ligustrum lucidum, on diabetes-induced osteoporosis in mice: a potential candidate as calcium-sensing receptor antagonist. Haghighi, M., Caicedo, J. C., Cimini, B. A., Carpenter, A. E. & Singh, S. High-dimensional gene expression and morphology profiles of cells across 28,000 genetic and chemical perturbations. Predicting compound activity from phenotypic profiles and chemical structures. Lotfollahi, M. Toward learning a foundational representation of cells and genes. & Azizi, E. A thousand and one tumors: the promise of AI for cancer biology. Yuan, B. et al. CellBox: interpretable machine learning for perturbation biology with application to the design of cancer combination therapy. Predicting cellular responses to complex perturbations in high-throughput screens. & Sudbery, I. UMI-tools: modeling sequencing errors in unique molecular identifiers to improve quantification accuracy. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Anders, S., Pyl, P. T. & Huber, W. HTSeq–a Python framework to work with high-throughput sequencing data. Zhang, S.-D. & Gant, T. W. A simple and robust method for connecting small-molecule drugs using gene-expression signatures. Lin, K. et al. A comprehensive evaluation of connectivity methods for L1000 data. The Protein Data Bank. Dai, Y. et al. Guizhi fuling decoction inhibiting the PI3K and MAPK pathways in breast cancer cells revealed by HTS2 technology and systems pharmacology. Liao, Y. et al. Ethyl acetate extract of Antenoron Filiforme inhibits the proliferation of triple negative breast cancer cells via suppressing Skp2/p21 signaling axis. Large-scale loss-of-function perturbations reveal a comprehensive epigenetic regulatory network in breast cancer. The cellular thermal shift assay for evaluating drug target interactions in cells. We thank X.-D. Fu (Westlake University) for valuable advice. We thank Y. Gao, J. Yang, J. Li, Y. Tian and R. Jia (Iomics Biosciences) for their assistance with the website building and drug screening. We thank H. Xu and C. Sun (Chengdu University of Traditional Chinese Medicine) for their assistance with flow cytometry experiments and pathological slide scanning. We thank G. Wang (Chengdu University of Traditional Chinese Medicine) for assistance with the CETSA analysis. D.W. received support from the National Key R&D Program of China (no. 2023YFF0720300), the National Natural Science Foundation of China (no. 82172723) and the Innovation Team and Talents Cultivation Program of National Administration of Traditional Chinese Medicine (no. These authors contributed equally: Lei Xiang, Yumei Wang, Wei Shao. School of Basic Medical Sciences, State Key Laboratory of Southwestern Chinese Medicine Resources, Chengdu University of Traditional Chinese Medicine, Chengdu, China Lei Xiang, Yumei Wang, Xiankuo Yu, Mingming Wei, Pan Qin, Chao Hu, Guochen Zhang, Xianwen Zhang, Jun An, Yan Luo, Yile Liao, Jinghong Deng, Xinran Tai, Lijun Huang & Dong Wang Iomics Biosciences Inc., Beijing, China Wei Shao, Jiawen Wang, Yingying Li & Junquan Xu Chinese Institutes for Medical Research, Beijing, China School of Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu, China Qingzhou Li, Yu Gui, Shengrong Li, Dale Guo & Yun Deng Department of Biology, University of California, San Diego, La Jolla, CA, USA School of Intelligent Medicine, Chengdu University of Traditional Chinese Medicine, Chengdu, China State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat-sen University, Guangzhou, China Yibin Vocational College of Medicine and Health, Yibin, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed HiMAP-seq, screened TCM compounds and conducted pharmacological studies. analyzed datasets and processed sequencing data. and Y. Li utilized HTS2 to screen MCE bioactive compounds. performed western blot and qPCR analyses and assisted animal experiments and the testing of TCM compounds. contributed to the development of HiMAP-seq. directed the SPR experiment protocols. Guochen Zhang carried out molecular docking analyses. were involved in conducting the animal experiments. and D.W. wrote the paper. All authors provided input on the paper. Correspondence to Yun Deng, Junquan Xu or Dong Wang. Patent applications were filed based on the results of this study. D.W. is a consultant for Iomics Biosciences. were employed by Iomics Biosciences during the time they contributed to this paper. The other authors declare no competing interests. Nature Methods thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Madhura Mukhopadhyay, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Comparison of the expression levels of the 3,407 genes selected in this study with random 3,407 genes. The gene expression profiles used for analysis were obtained from the GEO database (GSE204790 and GSE249322) and in-house bulk RNA-seq data (GSE294293). Box plots show the median (center line), 25th and 75th percentiles (box bounds), and full data range. From left to right, the medians are: 8.73, 1.58, 9.42, 0, 8.63, and 3.81, respectively. The interquartile ranges are: 4.86-10.09, 0-7.99, 4.46-11.04, 0-7.33, 4.33-10.27, and 0-8.41, respectively. b, The scheme of HTS2. Image generated in BioRender. c, Mapping rate of HTS2 results (n=5,639). The median is 43.86; the interquartile range is 26.87-59.90, with a minimum of 0.20 and a maximum of 100. d and e, Comparison of HTS2 with RNA-seq and qPCR. Pearson correlation (two-sided) of mRNA quantification between RNA-seq and HTS2 (d) and Pearson correlation (two-sided) of mRNA quantification between qPCR and HTS2 (e). f, Evaluation of reproducibility. Pearson correlation of gene expression between DMSO samples or JQ1 samples in different sequencing library construction batches. Gene expression (read counts) was scale-normalized and transformed into log2(read counts+1). Box plots show the median (center line), 25th and 75th percentiles (box bounds), and full data range. From left to right, the medians are 0.92, 0.91, 0.90, and 0.90, with interquartile ranges of 0.88-0.94, 0.87-0.94, 0.89-0.92, and 0.88-0.92, respectively. Sample sizes (n) are: 803, 810, 833, and 698, respectively. a-c, Evaluation of index misassignment. Pearson correlation (two-sided) was calculated for gene expression between MDA-MB-231 cell sample (pooled with MDA-MB-231 cells) and HepG2 cell sample (pooled with HepG2 cells) (a). Pearson correlation (two-sided) of gene expression was calculated between pairs of samples: two samples of MDA-MB-231 cells pooled with MDA-MB-231 cells (left panel), and one sample of MDA-MB-231 cells pooled with both MDA-MB-231 cells and HepG2 cells compared with another sample of MDA-MB-231 cells pooled with MDA-MB-231 cells (right panel) (b). Pearson correlation (two-sided) of gene expression was calculated between pairs of samples: two samples of HepG2 cells pooled with HepG2 cells (left panel), and one sample of HepG2 cells pooled with both MDA-MB-231 cells and HepG2 cells compared with another sample of HepG2 cells pooled with HepG2 cells (right panel) (c). Gene expression (read counts) was scale-normalized and transformed into log2(read counts+1). d-f, Evaluation of UMI error correction. Pearson correlation and dendrogram of hierarchical clustering were performed to analyze the gene expression values of 12 MDA-MB-231 cells samples treated with DMSO. Gene expression data (read counts) were scale-normalized and transformed into log2(read counts+1) for the left panel. Similarly, gene expression (UMI counts) was scale-normalized and transformed into log2(UMI counts+1) for the right panel (d). Correlation analysis (two-sided) was performed between 2 random samples from the above 12 samples. Panel (e) represents the analysis based on read counts, while panel (f) represents the analysis based on UMI counts. g and h, Evaluation of reproducibility. Pearson correlation (two-sided) of gene expression between DMSO samples prepared within the same sequencing library construction batch (g) and across different sequencing library construction batches (h). Gene expression (UMI counts) was scale-normalized and transformed into log2(UMI counts+1). a, Distribution of detected gene counts within samples of MDA-MB-231 cells, each comprising varying cell numbers (n = 16 for each group). Sequencing data from pooled samples amounted to 16,221 Mbp, with genes classified as ‘detected' if their read count in the sample exceeded 0. The average number of detected genes within each group was computed based on data collected from the 16 samples. b, Distribution of detected gene counts in samples of MDA-MB-231 cells with varying sequencing depths. Reads from the MDA-MB-231 cells sample containing 3,500 cells were randomly selected, and the number of detectable genes was recorded at various read depths. c and d, Comparison of HiMAP-seq with RNA-seq. Heat maps displaying the log2(fold change) of genes in MDA-MB-231 cells (c) and HepG2 cells (d) both treated with 10 μM JQ1. The log2(fold change) values were determined using either HiMAP-seq or RNA-seq. e and f, Comparison of HiMAP-seq with qPCR. Heat maps displaying the log2(fold change) of genes in MDA-MB-231 cells (e) and HEK293T cells (f) both treated with 10 μM JQ1. The log2(fold change) values were detected using HiMAP-seq or qPCR. g and h, The Pearson correlation (two-sided) between DMSO and JQ1 treated samples. For MDA-MB-231 cells, the treatment concentrations were 10 μM and 20 μM, respectively (g). The boxplot displays the Pearson correlation between identically perturbed samples and the correlation between two random samples. Self-samples represent biological replicate samples. Box plots show the median (center line), 25th and 75th percentiles (box bounds), and full data range. From left to right, the medians are: 0.94, 0.95, 0.97, 0.97, 0.96, 0.96, 0.96, and 0.97, respectively. The interquartile ranges are: 0.94-0.95, 0.95-0.96, 0.96-0.97, 0.967-0.970, 0.94-0.96, 0.96-0.97, 0.96-0.97, and 0.96-0.97, respectively. Sample sizes (n) are: 5,538, 1,846, 5,289, 1,763, 5,463, 1,821, 5,259, and 1,753, respectively (h). Comparisons were made using Student's t-test. i, Comparison of HiMAP-seq with RNA-seq from public datasets. Pearson correlation coefficients (two-sided) were calculated to evaluate mRNA quantification consistency between RNA-seq and HiMAP-seq in MDA-MB-231 cells. For HiMAP-seq and RNA-seq (GSE236250) samples, the perturbation concentration was 20 μM and 25 μM, respectively. Genes with |fold change | ≥ 1.5 were included in this analysis, and the results are presented as log2(fold change). a and b, TSNE plot of all compounds in MDA-MB-231 cells (a) and HEK293T (b). JQ1 treatment serves as a positive control in both HiMAP-seq and HTS2 experiments. Each dot represents a compound. c, Comparison of HiMAP-seq with HTS2. Heat maps displaying the log2(fold change) of genes in MDA-MB-231 cells treated with 10 μM JQ1. The log2(fold change) were detected using HiMAP-seq or HTS2. d, Schematic diagram illustrating the data composition of CIGS. a, Overlap among the top 30 compounds based on ZhangScore in two cell lines using the JQ1 signature. b and c, Analysis of Pearson correlation (two-sided) of gene expression profiles between JQ1 and ligustroflavone in HEK293T cells (b) and MDA-MB-231 cells (c). d and e, Heat map illustrating the differential expression of genes in HEK293T cells (d) or MDA-MB-231 cells (e) treated with either ligustroflavone or JQ1. f and g, Molecular dynamics simulation of ligustroflavone binding to the BRD4 protein. RMSF of residues respect to their time-averaged positions during the docking process of ligustroflavone and key target BRD4 (f). Percentage occupancy of different amino acid residues of BRD4 with ligustroflavone (g). h, Western blotting showing the BRD4 CETSA binding assay in the presence or absence of 50 μM ligustroflavone at different temperatures. i, The BRD4 band intensities in the CETSA were quantified. Data represent mean ± SEM (n = 3 biological replicates). j and k, Gene Set Enrichment Analysis (GSEA) of HTS2 results from siBRD4 knockdown in MDA-MB-231 cells, showing the normalized enrichment score (NES). The statistical analysis for GSEA used a one-sided permutation-based enrichment test with adjusted P values (p-adjust) to account for multiple comparisons. a-c, Cell viability of HK-2 cells after treatment with ERA (a), cisplatin (b) and 2,4-DIH (c) for 48 h (n = 3 biological replicates). d-f, HK-2 cells were treated with 2,4-DIH (250 μM) and ERA (panel d: 10 μM, panel e: 30 μM, panel f: 15 μM) for 24 h. Subsequently, the cellular ROS was tested by flow cytometry (d). HK-2 cells were labeled with FerroOrange fluorescent probes to measure intracellular Fe2+ concentrations using flow cytometry (e). Lipid peroxidation in HK-2 cells was detected using flow cytometry (n = 3 biological replicates, one-way ANOVA) (f). g, Representative photomicrographs of transmission electron microscopy in HK-2 cells. HK-2 cells were treated with 2,4-DIH (250 μM) and ERA (20 μM) for 24 h. Blue arrows indicate normal mitochondria. Red arrows indicate abnormal mitochondrial morphology typical of ferroptosis, including shrunken mitochondria, increased density and rupture of membranous structure, and decreased mitochondrial cristae. Yellow arrows indicate mildly abnormal mitochondria. Data shown represent mean ± SD. a, Body weight changes of mice in each group during treatment (n = 10 for each group, one-way ANOVA). b, The effects of cisplatin and 2,4-DIH on kidney coefficient. The kidney coefficient was calculated as the ratio of the kidney weight to the total body weight of KM mice (n = 10 per group, one-way ANOVA). c, Quantitative analysis of iron deposition in kidney (n = 6 per group, one-way ANOVA). The treatment dosage of 2,4-DIH is 100 mg/kg, while that of cisplatin is 25 mg/kg. d, Representative IHC staining for Nrf2 expression in kidney tissues (n = 3 for each group). Scale bar, 200 µm (upper panel) and 50 µm (lower panel). e, Quantitative analysis of Nrf2 expression in kidney tissues (n = 3 for each group, two-sided Student's t-test). All data are denoted as mean ± SD. Source data for Fig. Unprocessed western blots and gene signature of BRD4 knockdown. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Xiang, L., Wang, Y., Shao, W. et al. High-throughput profiling of chemical-induced gene expression across 93,644 perturbations. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/what-new-human-ancestor-has-been-discovered-and-how-a-virus-outbreak-in/'>Glacial Lake Outburst in Alaska and Discovery of a New Human Ancestor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Chikungunya Outbreak, Glacial Outbursts and a New Human Ancestor By Rachel Feltman, Fonda Mwangi & Alex Sugiura The remains of a home that collapsed due to erosion from a glacial outburst flood into the Mendenhall River in Juneau, Alaska on August 8, 2023. Christopher S. Miller for The Washington Post via Getty Images For Scientific American's Science Quickly, I'm Rachel Feltman. First, a public health update from one of our colleagues at Scientific American, senior health editor Josh Fischman. He's here to update us on an ongoing outbreak of the chikungunya virus in China. Josh Fischman: So what has happened here is that, in June, China started reporting a spike in cases of chikungunya, and China is having a fast rising outbreak in a place that has never had one before. And by the beginning of August, there were up to 7,000 cases. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The name comes from a Makonde word—uh, that's a language spoken in Tanzania—that means to bend up, and it refers to the most prominent symptoms, which are really, really painful joints that distort your posture and contort you into uncomfortable positions. They take a week or two to get over. And people can get sick within three to seven days of a bite. And what worries me a little bit is that chikungunya moves really easily in this age of fast global travel. Experts say that people in the United States should be a little concerned. There are, however, limiting factors that should minimize worry: spraying insecticides and illuminating mosquito areas. And this is important if you're traveling to affected areas where there are big outbreaks. Last Tuesday officials in Juneau, Alaska, confirmed a glacial outburst at Suicide Basin, a lake about 10 miles from the city center that's attached to the Mendenhall Glacier. As glaciers melt—which is happening faster these days, thanks to climate change—some of their water forms so-called glacial lakes. They're often dammed up naturally by glacial ice or by rocks and soil left behind as the glacier melts. But as meltwater levels rise, they can overflow the natural dam or even bust right through it. Suicide Basin has been a regular site of outburst floods since 2011, but the annual GLOFs have gotten worse each year since 2023. Floodwaters had receded by that same evening but only after reaching a peak of 16.65 feet. Local officials said a set of temporary barriers placed along the river earlier this year, along with close monitoring and early warnings to the public, kept flooding of homes, schools and businesses to a minimum. The Alaska Beacon reported that no overnight rescues or emergency evacuations had been required and that flood damage was limited to one bridge and some seepage of water into homes and yards. In other science news, a study published in Nature last Wednesday describes the remains of an entirely new species of human ancestor. The fossils, which date back to around 2.8 million to 2.6 million years ago, belong to a new member of the genus Australopithecus, meaning this species is a cousin of the famous Lucy. The species doesn't have a formal name yet, because researchers are hoping to find more fossils first—the new study is based only on a handful of teeth collected in Ethiopia. But even with just a few chompers to go on, the researchers say they're confident they've got a new hominin on their hands. The researchers note that we already knew that embryos had to burrow into uterine tissue in order to successfully implant but that most studies have focused on the genetic and biochemical aspects of this stage of conception instead of examining the mechanical process. When they introduced mouse embryos to their artificial uterus, the embryos exerted force to press themselves against the surface. Human embryos acted differently, burrowing into the uterine tissue to penetrate it. The researchers also saw signs that the embryos could sense and react to mechanical forces from their environment, as well as from other nearby embryos. Previous research suggests that between one third and half of all fertilized eggs fail to fully implant, so a better understanding of the mechanical process could help address some cases of infertility. Let's cap things off with a fun animal story. According to a study published last Tuesday in the journal Discover Animals, dolphins and whales have been hanging out together without us. Researchers studied nearly 200 different video clips of whales and dolphins interacting with each other, spanning across 20 years and 17 countries. Humpback whales and bottlenose dolphins were particularly prone to indulging in interspecies hangs, and the most common interaction involved dolphins swimming alongside a whale's snout. They may be engaging in a practice known as “bow riding,” which is where dolphins use the pressure fronts created by ships or large whales to swim faster. The researchers believe that dolphins may seek whales out for stimulation or play and that whales may sometimes reciprocate. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. Emily Makowski, Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/technology/a65717217/amazon-labor-day-deals-2025/'>The Best Labor Day Savings on Amazon Just Dropped. Here's Why Home Essentials Are 45% Off.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Huge savings on everything to keep your home maintained and secure. We may earn commission if you buy from a link. The official end of summer doesn't have to be a bummer. We've started seeing early Labor Day deals on Amazon for a variety of home security essentials, lawn care tools, and power stations. Currently, several savings are available on home security systems, outdoor cameras, and smart doorbells. For those who like to be prepared, we're also seeing sales on portable power stations. There are sales on small electric power stations—ideal for the cross-country road trip—from our favorite brands. Of course, we're also scoping out deals on everything you need for lawn maintenance. Top brands like Arlo, Ring, and Blink are on sale for up to 50% off, allowing you to kit out your home with the latest security technology. The Arlo Essentials XL is an outdoor camera that doubles as a motion-sensor spotlight. It's easy to set up and features a user-friendly app for monitoring the camera feed. My parents have had a Ring camera at their house for years and love it. They already had a reliable alarm system and wanted a bit of extra security without fussy installation, and one that was easy to operate. That is, until we tested the Greenwork 40V Mower—no need to pick sides though, they're both on sale right now. The Greenworks mower is part of a bundle which includes a string trimmer and leaf blower to keep your lawn in check. Another tested item that blew us away in testing (pun intended) is the 80V Backpack Leaf Blower. The handheld version is also currently on sale if you prefer not to strap up with a backpack each time. Recently, he was coordinator of partnership content at another product journalism outlet. Prior to that, he was a buyer for an independent men's shop in Houston, Texas, where he learned all about what makes great products great. He enjoys thrifting for 90s Broadway tees and vintage pajama sets. His spare time is occupied by watching movies and running to impress strangers on Strava. OpenAI Tries To Pump the Breaks on “AI Psychosis" Save up to 43% on these Early Labor Day TV Deals The Best OLED TVs You Can Buy Right Now This Giant Curved Gaming Monitor is the Lowest Price We've Ever Seen</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01729-3'>Edge curvature drives endoplasmic reticulum reorganization and dictates epithelial migration mode</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 09:42:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. From single-cell extrusion to centimetre-sized wounds, epithelial gaps of various sizes and geometries appear across organisms. Their closure involves two orthogonal modes: lamellipodial crawling at convex edges and purse string-like movements at concave edges. The mechanisms driving this curvature-dependent migration remain unclear. Here we perform an intracellular cartography to reveal that in both micropatterned and naturally arising gaps, the endoplasmic reticulum (ER) undergoes edge curvature-dependent morphological reorganizations, forming tubules at convex edges and sheets at concave edges. This reorganization depends on cytoskeleton-generated protrusive and contractile forces. Mathematical modelling reveals that these morphologies minimize strain energy under their respective geometric regime. Functionally, ER tubules at the convex edge favour perpendicularly oriented focal adhesions, supporting lamellipodial crawling, while ER sheets at the concave edge favour parallelly oriented focal adhesions, supporting purse string-like movements. Altogether, ER emerges as a central mechanotransducer, integrating signals from cytoskeletal networks to orchestrate two orthogonal modes of cell migration. Epithelial gap closure is essential for maintaining tissue integrity as disruptions occur at various scales, from individual cell loss to large wounds, throughout the life of an organism1,2,3,4,5,6. The epithelial gap closure happens by a collective movement of the surrounding cells7, which involves two orthogonal migration modes: lamellipodia-mediated cell crawling and actomyosin contractility-driven purse string-like movements8,9,10,11,12. These modes involve different actin and focal adhesion dynamics and assembly: branched actin polymerization and perpendicularly oriented focal adhesions in lamellipodia-mediated cell crawling versus actin bundle formation and parallelly oriented focal adhesions in purse string-like movements. Importantly, their relative contribution to epithelial gap closure depends on various biochemical and biophysical aspects of the tissue, including the curvature of the gap edge6. While previous studies have identified several molecules that sense or produce nanoscale membrane curvatures13,14, how cells coordinate their response towards curvature at cellular and multicellular length scales remains largely elusive. Nevertheless, epithelial cells preferentially use lamellipodial crawling at convex edges and purse string contraction at concave edges15. Also, a previous study showed that curvature-dependent migration is linked to actin flow, with anterograde flow at concave edges and retrograde flow at convex edges during lamellipodia formation16. However, it remains unknown how other intracellular structures organize in response to large-scale curvature, how such organization is regulated and whether this organization influences the mode of cell migration. Here, we performed a cartography of intracellular structures in convex and concave curved regions of micropatterned wounds of defined geometry in cultured epithelial monolayer and natural wounds in mouse embryonic epidermis. This revealed that the endoplasmic reticulum (ER) undergoes striking curvature-dependent morphological rearrangements. The ER is a continuous and highly dynamic organelle that spans the whole cell as a single entity. It comprises rounded tubular networks and dense flat sheets, which undergo constant rearrangements between themselves17,18,19,20,21,22,23. This highly dynamic nature of the ER reflects its adaptability and responsiveness to intracellular and extracellular changes24,25,26. However, the relationship between different ER morphologies and physical or geometric cues remains largely unknown. Here, we report that the endoplasmic reticulum alters its morphology and dynamics in response to edge curvature and plays a critical role in determining the modes of epithelial migration at different edge curvatures. First, to reveal the cell biological mechanism underlying the differential response of epithelial cells to edge curvature, we used microfabrication8,15,16 for the controlled and reproducible generation of gaps with defined shape, size and curvature. We fabricated polydimethylsiloxane (PDMS) stencils of two shapes—notched rectangles and flowers (Extended Data Fig. 1a,b)—consisting of convex, concave and flat edges15. We seeded Madin–Darby canine kidney (MDCK) epithelial cells around these PDMS stencils (Fig. 1a) for 15–18 h, allowed them to form a confluent monolayer and then lifted off the stencils triggering the cells to start migrating into the gaps. MDCK cells closed the gaps within 6–8 h (Extended Data Fig. Considering that the lamellipodial protrusions or actin bundles emerge within 15–30 min (ref. 1c and Supplementary Videos 3 and 4), we fixed the cells 30 min after removal of the stencil and stained them with fluorescent dye-labelled phalloidin to visualize the organization of the actin cytoskeleton. As expected15, we observed that the fraction of cells forming lamellipodia at the positively curved convex edge was significantly higher than that at the negatively curved concave edge (Fig. Microtubule filaments appeared more discrete and aligned perpendicular to the edge at the convex edge and formed thick bundles aligned parallel to the edge at the concave edge (Fig. Quantification of the microtubule directionality further confirmed this conclusion (Extended Data Fig. To ensure the accurate analysis of curvature-dependent behaviour, we quantified curvature at both the single-cell and tissue level, fitting circles to cell edges and splines to wound boundaries. We included only cells with more than 20% of their perimeter in contact with the wound, as lower-exposure cells were more influenced by neighbouring constraints than the geometry itself. This analysis revealed a strong one-to-one correspondence between the two curvature scales, thus validating our approach for linking local cell behaviour to global geometric cues (Supplementary Note 1 and Extended Data Fig. a, A schematic representing the experimental set-up for creating gaps of definite geometries. b, Representative images of actin, microtubule and ER at convex (left) and concave (right) edges. MDCK cells stained with phalloidin (grey, actin marker) and DAPI (cyan) (top), anti-α-tubulin (grey, microtubule marker) and DAPI (cyan) (middle), and MDCK cells expressing mApple-Sec61β (grey, ER marker) and labelled with DRAQ5 (cyan) (bottom). c, MDE quantifications for microtubules, actin and ER at convex (green) and concave edges (pink). From left to right: n = 62, 69, 74, 62, 84 and 76. d, The fraction of ER tubules at the front as a function of curvature, n = 50 at convex and n = 72 at concave. From left to right: n = 50 and 52. f, Representative super-resolution images of MDCK cells expressing mApple-Sec61β at convex (left) and concave (right) edges. Insets: tubules at the front (left) and sheets at the front (right). g, Representative images of cells stained with anti-Climp63 (yellow, ER sheet marker), phalloidin (pink) and DAPI (cyan) at the convex (top) and concave (bottom) edges. i, A schematic representing the ex vivo wound healing experiment. j, Representative images of actin and microtubule at convex (left) and concave (right) edges in mouse embryonic skin wounds. k, Representative images of wounded embryonic skin stained with anti-KDEL (magenta, ER marker), anti-Climp63 (yellow) and DAPI (cyan) at convex (top) and concave (bottom) edges. l, Super-resolution images of wounded embryonic skin stained for anti-Sec61β antibody, convex (green inset) and concave (pink inset) with the edges zoomed in and enlarged on the right. with a one-way ANOVA test (c) or two-tailed t-test (e). The dashed pink lines represent a concave edge and dashed green lines represent a convex edge. All experiments were performed at least three independent times. Next, to investigate whether other intracellular entities also respond to this large-scale geometrical cue, we performed a cartographic study of cellular organelles at differently curved edges. We used anti-GRASP65 antibody, anti-LAMP1 antibody and mitotracker green to visualize the Golgi apparatus, lysosomes and mitochondria, respectively. In addition, we transfected MDCK cells with mApple-Sec61β to observe the ER structure. Mitochondria had filamentous morphology spread throughout the cell and showed no apparent difference in structure or organization depending on the edge curvature (Extended Data Fig. Interestingly, ER morphology appeared remarkably different within the cells located at convex and concave edges. We observed the formation of a reticular network of ER tubules at the convex edge and a dense sheet-like lamellar morphology at the concave edge (Fig. We further quantified cytoskeletal and organelle distribution at different edges using a probability density-based approach to compute the mean distribution from the edge (MDE), which measures the distance between the cell edge and the centre of mass of each intracellular component27 (Extended Data Fig. We observed that the MDE for actin, microtubules, ER, Golgi apparatus and lysosomes at the concave edge is significantly lower than the respective MDE at the convex edge, suggesting that they are differentially polarized in response to the geometrical cue (Fig. Taken together, of all the organelles, ER showed differences not just in the distribution but also in its morphology and structure at different edge curvatures. We next quantified the structural changes in ER by calculating the fraction of ER tubules as a function of curvature at the front of the cell by segmenting the ER into tubules and sheet-like dense regions using Trainable Weka Segmentation28. This analysis revealed that at convex edges, the ER tubule fraction at the cell front increases sharply with increasing positive curvature (Fig. In contrast, at concave edges, the ER tubule fraction remains consistently low and shows only weak variation across the range of negative curvatures. These results demonstrated that cells at convex edges have a significantly higher ER tubule fraction than those at concave edges, reflecting a clear curvature-dependent structural reorganization of ER (Fig. To visualize ER morphology at higher resolution across curvatures, we performed super-resolution imaging of mApple-Sec61β-transfected cells using the super-resolution by optical realignment (SoRa) method. We observed a highly reticulated network of ER tubules at the convex edge. In contrast, at the concave edge, we observed accumulation of dense ER, which consists of flat ER sheets as well as dense clusters of tubules29 (Fig. Next, to check whether the dense ER at the front of migrating cells at the concave edge might be ER sheets, we labelled the cells with an antibody against cytoskeleton-linking membrane protein 63 (Climp63), which is an ER sheet marker. Climp63 fraction decreased with increasing convex curvature, but it remained comparatively high and relatively independent of the curvature magnitude at the concave edges (Fig. This contrasting ER morphology was very clearly observed in flower-shaped patterned wounds, where concave edges showed dense sheet-like ER along with actin bundles, while neighbouring convex edges displayed ER tubules and lamellipodial protrusions (Extended Data Fig. To test whether our findings extend beyond imposed geometries, we analysed epithelial colonies where curvature arises naturally. These spontaneous convex and concave edges closely match the scale of our micropatterns, allowing direct comparison between native and controlled settings. In this set-up, MDCK cells expressing mApple-Sec61β were allowed to form small colonies and actin organization was visualized using phalloidin staining. We observed that cells along convex regions formed lamellipodial protrusions and showed enrichment of ER tubules, whereas cells at concave zones exhibited actin bundles and dense sheet-like ER (Extended Data Fig. Quantification of ER tubule fraction as a function of curvature confirmed a significantly higher tubule content at convex edges compared with concave ones (Extended Data Fig. We also verified the generality of these observations in another epithelial cell line, EpH4-Ev, which originated from the mouse mammary gland (Extended Data Fig. Next, we examined the cellular response to spontaneously arising edge curvature in an incision wound made in a mouse embryonic skin explant (Fig. In this ex vivo model, dorsal skin explants from E14.5–E16.5 mouse embryos were wounded and cultured at an air–liquid interface to allow migration. Live imaging of SiR-actin-labelled tissue confirmed active migration, with actin accumulating at concave edges and small protrusions forming at convex edges within 30 min (Extended Data Fig. Hence, for subsequent experiments, we maintained wounded skin in the air–liquid interface and fixed after 30 min. We stained them for actin, microtubules and ER. Phalloidin staining showed that the epidermal cells formed lamellipodia at the convex wound edge and supracellular actin bundles at the concave wound edge (Fig. Microtubule orientation also depended on the local wound curvatures, showing perpendicular and parallel alignments at the convex and concave wound edges, respectively (Fig. Finally, to study ER morphology in the mouse embryonic skin wounds, we stained the skin explants with an anti-Climp63 antibody. 1g,h), Climp63 was specifically localized at the cell periphery at the concave wound edge. In contrast, staining these explants with an antibody against the KDEL sequence, a general ER marker, revealed that the ER network covered almost the entire cytoplasm (Fig. Together, these results indicated the specific enrichment of sheet-like dense ER structures at the concave wound edge. To further resolve the structural details of ER in the mouse skin tissue, we used Lattice SIM2 super-resolution imaging and observed that ER indeed showed more tubular morphology at the spontaneously generated convex wound edge and dense sheet-like structures at the concave wound edge (Fig. Taken together, these observations suggested that epithelial cells interfacing different edge curvatures have proclivity for generating different ER morphologies. Next, we wanted to determine whether the dense ER structures observed at the front of migrating cells at concave wound edges more closely resemble ER sheets or tubule clusters29. Although Climp63 and p180 are validated ER sheet markers30,31, we directly benchmarked their localization in our system to confirm how faithfully they label ER sheets over ER tubules. We transfected MDCK cells with the general ER marker Sec61β, allowed them to form confluent monolayers, and performed SoRa imaging after immunostaining for Climp63 or p180. In nonmigrating cells, both Climp63 and p180 localized exclusively to dense, perinuclear ER structures, consistent with their known enrichment in ER sheets30,31 (Extended Data Fig. These markers were absent from the peripheral ER, where tubules predominate, including potential dense tubular networks. This confirmed that Climp63 and p180 reliably distinguish ER sheets from tubules. We then examined migrating cell fronts, where Climp63 and p180 were strongly enriched in dense ER structures at concave edges (Fig. 2a,b), but not at convex edges. Quantitative analysis showed that the fraction of Climp63 and p180 signal over total ER (Sec61β) was significantly higher at concave than convex fronts (Fig. These results indicated that the dense ER at concave edges is largely composed of sheet-like structures. Using dual labelling with Sec61β and Rtn4b followed by super-resolution imaging (Extended Data Fig. 5b), we observed that while Rtn4b localized broadly across the ER network, it was most prominently enriched at the rims of dense ER sheets (Extended Data Fig. 5b, right), with only little presence along distinct tubules and little to no signal within the sheet interiors. This preferential localization to sheet boundaries was more pronounced and supports the view that the dense ER at the concave edge is primarily composed of Climp63-positive sheets, with minimal contribution from tubular elements. a, Representative images of MDCK cells expressing mApple-Sec61β (red) stained with anti-Climp63 antibody (cyan) migrating at concave (left) and convex (right) edges. Insets: zoomed-in enrichment of Climp63 towards the migrating front (dashed white line). b, Representative images of MDCK cells expressing mApple-Sec61β (red) stained with anti-p180 antibody (cyan) migrating at concave (left) and convex (right) edges. Insets: zoomed-in enrichment of p180 towards the migrating front (dashed white line). c, Quantification of the fraction of Climp63 on Sec61β at the front of migrating cells at convex (left, n = 30) and concave (right, n = 39) edges. d, Quantification of the fraction of p180 on Sec61β at the front of migrating cells at convex (left, n = 58) and concave (right, n = 73) edges. e, Time-lapse images of LifeAct MDCK cells expressing mApple-Sec61β migrating at convex (top) and concave (bottom) edges. h, Time-lapse images of LifeAct (green) MDCK cells expressing mCherry-Climp63 (Fire LUT)) migrating at the concave edge. i, Time-lapse images of the front of MDCK cells expressing mApple-Sec61β at convex (top) and concave (bottom) edges. j, Quantification of the ER tubule fraction in the cells over 60 min at the two curvatures (n = 9 at concave and n = 7 at convex). from two-tailed t-tests (c and d). All experiments were performed at least three independent times. Finally, a recent study using the ER–plasma membrane contacts marker MAPPER demonstrated that ER–plasma membrane contacts polarize in single migrating cells, with Climp63-rich ER sheets at the rear showing more ER–plasma membrane contacts than Rtn4a-rich tubular ER at the front, facilitating polarized receptor tyrosine kinase signalling and phosphoinositide distribution32. While their focus was on the front and back of cells during single-cell migration, our study examines curvature-dependent collective epithelial migration, where mechanical forces and ER organization are regulated at the tissue scale. Despite these differences, we find an intriguing connection: curvature-driven ER morphology correlates with ER–plasma membrane contact distribution, with reduced ER–plasma membrane contacts at convex edges (ER tubule enriched) and increased contacts at concave edges (ER sheet enriched) (Extended Data Fig. This suggests that curvature may influence ER–plasma membrane contact site regulation through ER structural changes, affecting receptor tyrosine kinase and phosphoinositide signalling. Regulating phosphoinositide distribution can then regulate cytoskeletal dynamics, membrane trafficking and polarity, which are essential components cell migration. To understand how distinct ER structures might be linked to ER dynamics at different edge curvatures, we performed live-cell imaging using MDCK cells expressing LifeAct, which marks the actin cytoskeleton, and mApple-Sec61β, which marks the ER. Live imaging revealed two kinds of ER dynamics at the front of the migrating cell. At the convex edge, we observed a retrograde flow of dense ER structures towards the nucleus where major ER sheets are present (Fig. Plotting a kymograph confirmed this retrograde flow (Fig. In contrast, at the concave edge, we observed an anterograde movement of ER sheets towards the front of the migrating cells (Fig. 2e and Supplementary Video 7), also confirmed by the kymograph (Fig. Next, we conducted super-resolution live imaging using SoRa microscopy in LifeAct MDCK cells expressing Sec61β, which provided enhanced spatial clarity. This allowed us to clearly visualize dense ER sheet-like structures moving anterogradely towards the leading edge at concave curvatures, and retrogradely at convex curvatures, as shown in Extended Data Fig. To further confirm that the dense ER structures at the concave front are indeed sheets, we expressed mCherry-Climp63, a known ER sheet marker, in LifeAct MDCK cells. To minimize the possibility of overexpression-induced structural artefacts, we selected cells with low to moderate expression. Live imaging of these cells revealed the anterograde movement of Climp63-positive ER sheets towards the leading edge at the concave wound edge (Fig. High-magnification imaging further revealed that at the convex edge, as the cells started migrating, newer ER tubules grew in the direction of migration (Fig. 2i, green arrowhead, and Supplementary Video 11) and existing tubules retracted back into the tubular ER network (Fig. 2i, red arrowhead, and Supplementary Video 11). Interestingly, we also observed anterograde movements of the existing sheets towards the edge. Notably, even within the short timescales, individual ER tubules demonstrate consistent retrograde movement at convex edges and anterograde movement at concave edges, aligning with the overall patterns observed. We then quantified the fraction of ER tubules at the cellular front at different timepoints. This analysis revealed that a bias existed at the beginning, where the ER tubule fraction was determined to be 0.64 and 0.35, at the convex and concave edges, respectively (Fig. As time progressed, the fraction of tubules at the convex edge increased further, while it decreased at the concave edge (Fig. Taken together, these results suggested that contrasting ER morphologies at the two edge curvatures are coupled with differential dynamics of ER. Epithelial cells exposed to the wound respond by generating protrusive forces due to branched actin polymerization at the convex edge and contractile forces due to actomyosin activity at the concave edge15,16,33. We investigated whether the microtubules and ER responded to these forces by altering their orientation and morphology, respectively. We first treated the cells with an Arp2/3 inhibitor, CK666, which prevented lamellipodia formation, attenuating the protrusive force. CK666-treated cells predominantly formed actin bundles at both convex and concave edges (Extended Data Fig. Importantly, in these cells at both edges, microtubules were arranged as parallel bundles (Fig. 3c,d), as opposed to the control cells where bundled microtubules and ER sheets were prevalent mainly at the concave edge. We next treated the cells with blebbistatin, inhibiting actomyosin contractility. Blebbistatin-treated cells formed protrusive structures at both convex and concave edges (Extended Data Fig. In these cells, microtubules aligned perpendicularly to the edge (Fig. 3a,b) and ER showed a tubular structure at both convex and concave edges (Fig. However, we wanted to confirm whether CK666 or blebbistatin treatments affected ER structure globally and therefore we examined several cells in the middle of the monolayer. These results suggested that microtubules and ER responded to different kinds of forces generated by the protrusive and contractile actin structures at different edge curvatures. a, Representative images of MDCK cells stained with anti-α-tubulin (yellow) and DAPI (cyan) treated with dimethylsulfoxide (DMSO) (left), CK666 (middle) and blebbistatin (right) migrating at convex (top) and concave (bottom) edges. b, Quantification of the directionality of microtubules in cells treated with CK666 or blebbistatin (at different curvatures (n = 56, 59, 77, 79, 95 and 76 from the top (convex, control) to bottom (concave, CK666)). c, Representative images of MDCK cells expressing Sec61β (yellow) stained with phalloidin (magenta) and DAPI (cyan) treated with DMSO (left), CK666 (middle) and blebbistatin (right) at convex (top) and concave (bottoml) edges. d, Quantification of the fraction of ER tubules at the front of the cells treated with DMSO, CK666 or blebbistatin at convex (green, left) and concave (pink, right) edges. From left to right: n = 40, 42, 41, 37, 37 and 42. e, Time-lapse imaging of cells expressing PA-Rac1 (magenta) labelled with SiR-tubulin (yellow), photoactivated with a 445 nm laser in the region shown with the red circle. The green circle represents a part of the nonactivated region and the zoomed-in regions show SiR-tubulin at activated and nonactivated regions. f, Time-lapse images of cells expressing PA-Rac1 (magenta) and GFP-Sec61β (yellow), photoactivated with a 445 nm laser in the region marked with the red circle. g, Quantification of the fraction of ER tubules in photoactivated regions before and after 20 min of activation (right) and nonactivated regions before and after 20 min (left). h, Representative images of MDCK cells expressing mApple-Sec61β treated with DMSO (left) or Calyculin A (right). i, Quantification of the fraction of tubules in cells treated with DMSO or Calyculin A. From left to right: n = 47 and 66. j, Representative images of mouse embryonic skin wounds stained with KDEL (magenta), Climp63 (yellow) and DAPI (cyan) and treated with DMSO (top), blebbistatin (middle) and CK666 (bottom). Pink insets: Climp63 in cells zoomed in at the concave edge and blue arrowheads point towards the edge of the cells. with a one-way ANOVA test (d), two-tailed paired t-test (g) or two-tailed t-test (i). The dashed pink lines mark the concave edges and dashed green lines mark the convex edges (a, c and j). All experiments were performed at least three independent times. Next, to perturb the system in a more controlled manner, we used optogenetic tools to drive lamellipodia formation and followed the morphological changes in ER and microtubules using live-cell imaging. Rac1 is a small Rho GTPase that regulates actin assembly and dynamics, especially during lamellipodia formation. We used a genetically encoded photoactivatable Rac1 (PA-Rac1), which induces the formation of lamellipodial protrusions on activation with light with a wavelength of 458 nm (ref. First, to check the response of microtubules to Rac1 activation, we transfected MDCK cells with mCherry-PA-Rac1 and labelled microtubules with SiR-tubulin. On photoactivation of Rac1, bundled microtubules opened up and grew perpendicularly to the cell edge (Fig. 3e and Supplementary Video 13), while we did not observe such microtubule dynamics in the neighbouring nonactivated cells (Fig. Subsequently, we cotransfected MDCK cells with mCherry-PA-Rac1 and GFP-Sec61β to study ER dynamics. On photoactivation of Rac1, ER matrices converted to tubules at the site of activation of Rac1, while ER morphology did not change considerably in the nonactivated region of the cell (Fig. Quantifying the fraction of ER tubule on Rac1 activation showed a significant increase in this parameter in activated regions as opposed to nonactivated regions (Fig. These results together confirmed that the protrusive forces generated due to actin polymerization and branching at the convex edge guide the organization of ER and the microtubule network. Conversely, to augment actomyosin contractility, we went on to treat MDCK mApple-Sec61β cells with Calyculin A. Calyculin A is a myosin light chain phosphatase inhibitor that leads to hyperactivated myosin35. We observed a considerable expansion of ER sheets and matrices in the cell on Calyculin A treatment (Fig. 3h) and a decrease in ER tubule fraction (Fig. We then explored whether these mechanical forces influence ER structure in the ex vivo wound healing model. To test this, we pretreated the embryonic skin explants with blebbistatin (100 μM) or CK666 (100 μM) 2 h before wounding. We observed that on blebbistatin treatment, the cells at both convex and concave edges had considerably less accumulation of Climp63 as compared with control (Fig. 3j), whereas on CK666 treatment, we observed a high Climp63 signal at the edge of the cells at both convex and concave edges (Fig. Taken together, these results suggested that the formation of ER tubules and sheets at convex and concave edges, respectively, depends on the different cellular forces acting at different edge curvatures. Next, to obtain a fundamental understanding of why a particular morphology of ER is preferred in response to geometrical and mechanical cues, we constructed a mathematical model. The ER forms a dynamic and intricate network within cells. While previous models have explored ER shape using Langevin dynamics36,37 and the influence of curvature-stabilizing proteins on tubular/sheet abundance30,38, our experiments highlight a crucial role for mechanical forces, specifically protrusion and contraction, in shaping ER morphology. To address this, we developed a computational model of the cell, comprising cytoplasm, actin cortex and ER, examining three distinct ER morphologies: perpendicular (tubular), sheet and parallel (Fig. We assume that morphology-dependent deformation of ER contributes to the strain energy of the cell. That is, ER tends to be in tube-like morphology during protrusion and sheet-like during contraction because the combination of morphologies and forces acting on the cell results in lower strain energy. a, A schematic representing the front of the cell with a flat edge, an actin cortex (magenta) and ER (green) with perpendicular or tubular ER (left), parallel ER (middle) and sheet ER (right) morphologies. b, The strain energy density of a cell with a flat edge undergoing contraction with perpendicular and sheet ER morphologies. c, The strain energy density of a cell with a flat edge undergoing protrusion with perpendicular and sheet ER morphologies. d, A schematic representing a cell with convex (left) and concave (right) curvatures exhibiting different ER morphologies as in a. e, The normalized strain energy density of a cell with parallel ER, sheet ER or perpendicular ER with increasing positive curvature during protrusion. f, Representative images of MDCK cells expressing mApple-Sec61β (green) and stained with phalloidin (magenta) migrating at positive curvatures of increasing magnitude (left to right). h, Normalized strain energy density of a cell with parallel ER, sheet ER or perpendicular ER with increasing negative curvature during contraction. i, Representative images of MDCK cells expressing mApple-Sec61β (green) and stained with phalloidin (magenta) migrating at negative curvatures of increasing magnitude (left to right). The dashed pink lines mark the negative curvatures and dashed green lines mark the positive curvatures (f and i). We modelled cells as 3D mechanical entities with ER embedded in the cytoplasm and an actin cortex at the cell edges. The geometry considered for analysis was the front of the cell where ER morphologies were observed to change due to varying mechanical and geometric stimuli; hence, we did not take the effect of the nucleus into account. We modelled the actin cortex as a strain rate-dependent nonlinear constitutive law, as proposed previously39, and generated forces necessary for curvature-dependent contraction or protrusion (Supplementary Note 2). For simplicity, the cytoplasm and ER were modelled as linear elastic materials. Moreover, since the aim of the model is to study the effect of curvature-dependent forces on ER morphology, we have not explicitly modelled microtubule structures, their response to curvature or their effect on ER reorganization. In vitro and ex vivo experiments showed that ER exhibited a tube-like perpendicular morphology at the convex edge during lamellipodia-based protrusion and sheet-like morphology at the concave edge during purse string movements. In silico studies were performed by considering cells with convex or concave edges with varying magnitudes of curvatures and ER morphologies. While perpendicular and sheet ER indicated distinct ER configurations, parallel ER is assumed to represent a cluster of parallel tubules. We observed that when cells experience purse string forces resulting in contraction, the cell with ER sheet morphology exhibited lower strain energy compared with cells with perpendicular ER (Fig. For a given volume of ER, the ability of sheet ER to resist bending moments better than perpendicular ER results in lowering the strain energy density of the cell experiencing purse string forces. In contrast, we observed that when cells experienced protrusive forces, the cell with perpendicular ER exhibited lower strain energy density compared with the cell with sheet ER (Fig. The axial forces during protrusion are dominant over the bending moments (Extended Data Fig. For a given volume of ER, since perpendicular ER resist protruding forces better than sheet ER, the strain energy density of the cell with perpendicular ER is lower during protrusion. Next, we simulated cells with positive (convex) and negative (concave) curvatures (Fig. For a given curvature, we kept the volume of ER in the cell constant across the different ER morphologies considered. We observed that during protrusion at convex edges, the strain energy density of a cell with either sheet ER or parallel ER was higher than that of a cell with perpendicular ER, indicating that ER prefers to stay in the perpendicular tube-like morphology aligned along the direction of protrusion (Fig. 7c), leading to higher elastic strain energy. In addition, as the curvature increased, the difference in strain energy density between cells with sheet and perpendicular ER morphologies increased, while the strain energy density of a cell with parallel ER remained close to that of the sheet ER, irrespective of curvature (Fig. This implies that, while the probability of finding perpendicular ER is always higher than sheet ER, it increases further with increasing positive curvature. Interestingly, this result corroborated our previous experimental finding that the fraction of ER tubules increases at the convex edge with increasing magnitude of curvature (Fig. Plotting ER tubules and sheets at the convex edge as a function of curvature also revealed that as the curvature increased, the amount of tubules at the front increased while the amount of sheets decreased (Fig. In contrast, during contraction, the strain energy density with sheet ER was lower than that of a cell with perpendicular ER, indicating that cells preferred sheet ER morphology over perpendicular tube-like morphology (Fig. We also observed that parallel ERs had a strain energy density higher than that of sheet ER but lower than that of perpendicular ER. In contrast to protrusion, under contraction, displacement profiles showed that cells with perpendicular ER had higher displacement compared with cells with sheet ER (Extended Data Fig. As the curvature increased, the model showed that the difference in strain energy density between cells with sheet and perpendicular ER morphologies decreased up to a certain curvature level and remained constant for higher curvatures (Fig. This implies that, while the probability of finding sheet ER is always higher than perpendicular ER, it exhibited curvature independence for higher levels of negative curvatures. The model displayed behaviour similar to our previous experimental findings, showing that the fraction of ER sheets is higher at concave edges than tubules (Fig. Collectively, from the simulation results shown (Extended Data Fig. Therefore, the distribution of ER responds to the geometric and biomechanical stimuli present in a cell. ER and microtubules have remarkable structural interdependency27,40,41,42. Moreover, ER morphology and dynamics are known to be regulated by microtubules42,43. In our study, we observed a great correlation between the organization of ER and microtubules. The tubular ER network coincided with the perpendicularly arranged microtubules, while lamellar ER coincided with parallelly arranged microtubules (Fig. We therefore wondered whether microtubules might guide edge curvature-dependent ER morphology. To this end, we first disrupted microtubules by treating the cells with 5 μM nocodazole. As expected44, upon nocodazole treatment, ER tubules disappeared irrespective of the edge curvature (Fig. We then tested whether ER tubule formation at the convex edge involved ER movements along microtubules by sliding and tip attachment complex (TAC) dynamics45,46 (Fig. We perturbed the ER sliding dynamics by expressing KHC+ (kinectin-binding domain of kinesin) or KNT+ (kinesin-binding domain of kinectin), which led to disruption of the kinectin–kinesin interaction47. However, at the concave edge, the accumulation of ER sheets in KHC+ and KNT+ expressing cells did not change as compared with the control EGFP-expressing cells (Fig. Next, we perturbed the TAC movement of the ER by expressing a dominant negative form (EB1-c) of microtubule plus end-binding protein1, EB1, which led to its reduced binding with the microtubules48. 5h,i), but the accumulation of ER sheets at the concave edge did not display any change as compared with the control cells (Fig. Taken together, these results suggested that both kinectin–kinesin-dependent sliding and TAC dynamics of ER are important for the generation of ER tubules at the convex edge but not for ER sheet accumulation at the concave edge. a, Representative images of MDCK cells expressing mApple-Sec61β (yellow), stained with anti-α-tubulin (magenta) and phalloidin (blue), migrating at convex (top) and concave (bottom) curvatures. b, Representative images of MDCK-expressing mApple-Sec61β treated with DMSO (left) and nocodazole (right). c, Quantification of fraction of tubules in the whole cell treated with DMSO (blue) or nocodazole (pink). From left to right: n = 47 and 55. d, A schematic representing sliding and TAC-dependent movement of ER on microtubules. e, Representative images of MDCK cells expressing mApple-Sec61β (yellow) and pEGFP vector control (VC) (magenta, top), KHC+ (magenta, middle) or KNT+ (magenta, bottom) migrating at convex (left) and concave (right) edges. f,g, Quantification of the fraction of ER tubules at the front of cells expressing pEGFP VC, KHC+ or KNT+ migrating at the convex edge (from left to right: n = 33, 39 and 30) (f) or the concave edge (from left to right: n = 34, 24 and 18) (g). h, Representative images of MDCK cells expressing mApple-Sec61β (yellow) and EGFP VC (magenta, top) or EB1-c (magenta, bottom) migrating at convex (left) and concave (right) edges. i,j, Quantification of the fraction of ER tubules at the front of the cells expressing the EGFP VC or EB1-c migrating at the convex edge (from left to right: n = 27 and 33) (i) or the concave edge (from left to right: n = 24 and 20) (j). with one-way ANOVA tests (f and g) or two-tailed t-tests (c, i and j). The dashed pink lines mark the concave edges and dashed green lines mark the convex edges (e and h). All experiments were performed at least three independent times. Next, we examined whether ER morphology influences the curvature-specific mode of epithelial migration at wound edges. To this end, we manipulated the ER morphology by overexpressing either an integral ER membrane protein Rtn4a, which leads to the enrichment of ER tubules, or an ER sheet-associated protein Climp63, which leads to the enrichment of ER sheets30. Indeed, MDCK cells overexpressing Rtn4a-GFP showed a higher ER tubule fraction and MDCK cells overexpressing mCherry-Climp63 showed a higher ER sheet fraction as compared with the control cells overexpressing mApple-Sec61β (Extended Data Fig. Subsequently, we seeded these cells around the PDMS micropattern and studied their migration behaviour at different edge curvatures. To normalize across experiments, we calculated the probability of forming lamellipodia by dividing the number of transfected cells forming the lamellipodia by the total number of transfected cells at a particular edge curvature. For cells overexpressing Rtn4a, the percentage of cells forming lamellipodia at the convex edge was comparable to that for wild-type (WT) cells. However, at the concave edge, where the WT cells majorly formed actin bundles, Rtn4a-overexpressing cells showed a significantly higher probability of lamellipodium formation (Extended Data Fig. In contrast, Climp63 overexpression, which amplified ER sheet morphology, resulted in a significant decrease in the percentage of the lamellipodium-forming cells at the convex edge (Extended Data Fig. Transient overexpression enabled rapid phenotype screening and dose-dependent analysis with internal controls, though variability in expression levels across cells may complicate the interpretation. To address this problem, we generated stable MDCK lines expressing Rtn4a-GFP or mCherry-Climp63 via antibiotic selection, followed by fluorescence-activated cell sorting (FACS) before each experiment to ensure uniform expression. The FACS-sorted Rtn4a-GFP cells formed prominent lamellipodia at both convex and concave edges (Fig. 6a), with MDE analysis showing increased protrusive activity at concave edges (Fig. In contrast, FACS-sorted Climp63-expressing cells displayed apical actin bundles and reduced basal lamellipodia at both curvatures, reflected in lower MDE values at convex edges (Fig. Time-lapse imaging revealed faster gap closure by Rtn4a-GFP cells, including individual cell migration into the gap, while Climp63-expressing cells also showed accelerated closure, probably via contractile, bundle-driven migration (Fig. These results highlight the instructive role of ER morphology in regulating migration mode and speed. a, Representative images of WT MDCK cells (top), overexpressing (OE) mCherry-Climp63 (yellow, middle) and overexpressing Rtn4a-GFP (yellow, bottom) migrating at micropatterned gaps stained with DAPI (cyan) and phalloidin (magenta). Zoomed insets: actin staining at convex (green) and concave (pink) edges in all three conditions. b, Quantification of MDE for actin at convex (green) and concave (pink) edges in WT, Rtn4aOE and Climp63OE cells. From left to right: n = 49, 50, 47, 43, 47 and 37. c, Quantification of a normalized wound closure area over time for MDCK WT (blue), Rtn4aOE cells (yellow) and Climp63OE cells (grey). d, Representative images of MDCK WT cells treated with blebbistatin (left), Climp63OE cells (middle) and Climp63OE cells treated with blebbistatin (right) seeded at micropatterned gaps and stained with phalloidin (black) (top) and MDCK WT cells treated with CK666 (left), Rtn4aOE cells (middle) and Rtn4aOE cells treated with CK666 (right) seeded at micropatterned gaps and stained with phalloidin (black) (bottom). e, Representative images of EpH4 cells treated with nontargeting (NT) siRNA or Rtn4 siRNA migrating at convex (top two rows) and concave (bottom two rows) edges stained with anti-KDEL antibody (yellow) and phalloidin (magenta). f, Representative images of EpH4 cells treated with NT siRNA or Climp63 siRNA migrating at convex (top two rows) and concave (bottom two rows) edges stained with anti-KDEL antibody (yellow) and phalloidin (magenta). g, Quantification for MDE for actin in cells treated with NT siRNA or Rtn4 siRNA migrating at convex (green) and concave (pink) edges. From left to right: n = 104, 131, 118 and 104. h, Quantification for MDE for actin in cells treated with NT siRNA or Climp63 siRNA migrating at convex (green) and concave (pink) edges. From left to right: n = 95, 112, 99 and 110. with one-way ANOVA tests (b, g and h). All experiments were performed at least three independent times. To examine whether ER morphology influences cytoskeletal organization in nonmigrating cells, we again transiently overexpressed Rtn4a and Climp63 in MDCK monolayers to induce ER tubule- and sheet-enriched states, respectively. The mosaic produced by the transient transfection allowed us to directly compare the cytoskeletal organization of transfected cells against neighbouring WT cells within the same monolayer, serving as internal controls. Immunostaining for actin and microtubules revealed no notable differences in their organization in either condition (Extended Data Fig. 8f), suggesting that ER morphological changes do not overtly disrupt cytoskeletal architecture in nonmigrating cells. To further investigate whether ER restructuring affects collective properties such as cell shape index (defined as the perimeter/sqrt(area))49, we seeded stable MDCK populations homogeneously expressing mCherry-Climp63 or Rtn4a-GFP. ZO-1 staining was used to delineate cell boundaries, and subsequent shape index analysis revealed slightly higher values in both Climp63- and Rtn4a-expressing monolayers compared with WT. While statistically significant, these changes were modest, and most cells retained a cobblestone-like morphology, suggesting limited physiological relevance from a shape-transition perspective49 (Extended Data Fig. Next, to dissect the relationship between ER morphology and cytoskeletal dynamics at the wound edge, we introduced a second layer of perturbations using drug treatments in Climp63- and Rtn4a-overexpressing cells. For example, we seeded the sorted Climp63-overexpressing cells around the PDMS chambers and treated them with blebbistatin to inhibit actomyosin contractility and stained them with phalloidin and anti-alpha-tubulin antibody. In WT cells, blebbistatin treatment promoted lamellipodia formation at both concave and convex edges (Extended Data Fig. However, in Climp63-overexpressing cells, blebbistatin failed to rescue lamellipodia, indicating that ER sheets override the effect of contractility inhibition, locking cells into a contractile state (Fig. These results suggests that ER tubules are necessary for branched actin polymerization and lamellipodia-driven migration. Blebbistatin treatment also failed to completely rescue microtubule rearrangements from parallel to perpendicular in Climp63-overexpressing cells (Extended Data Fig. In another experiment, we seeded sorted Rtn4a-overexpressing cells around PDMS chambers and treated them with CK666 to inhibit Arp2/3-mediated branched actin polymerization. In WT cells, CK666 treatment suppressed lamellipodia and promoted actin bundle formation at the convex edge (Extended Data Fig. However, in Rtn4a-overexpressing cells, CK666 failed to fully restore actin bundles, leaving a few persistent protrusive structures at the edge and disrupting the normally continuous contractile actin architecture (Fig. These results suggested that once ER tubules accumulate, they prevent full re-establishment of contractile actin structures, even when lamellipodia formation is inhibited. In contrast, CK666 treatment in Rtn4a-overexpressing cells led to a reorientation of microtubules from a perpendicular to a parallel arrangement, suggesting that microtubule organization adapts to the dominating actin architecture, and ER morphology may exert a more decisive influence on actin dynamics than on microtubules (Extended Data Fig. Notably, nocodazole treatment, which disrupts microtubules, consistently promoted ER sheet formation along with thick, continuous actin bundles at the edge and increased stress fibres within the cell, in both Climp63- and Rtn4a-overexpressing cells (Extended Data Fig. This again supports our earlier observation that microtubule–ER interactions facilitate ER tubule generation and suggests that microtubules contribute primarily to lamellipodia formation at convex edges but play a lesser role in actin bundling at concave edges (Fig. By combining ER structural manipulation with cytoskeletal drugs, we created a mechanistic tug-of-war that revealed ER morphology as a key regulator of cytoskeletal organization: sheets stabilize actin bundles and suppress lamellipodia, while tubules promote branched actin and weaken contractile structures. The failure of cytoskeletal drugs to fully rescue migration modes after ER perturbation underscores the instructive role of ER in guiding curvature-dependent migration. To further validate the functional roles of ER structure, we conducted siRNA-mediated knockdown of Climp63 and Rtn4 in EpH4 epithelial cells (Fig. Knockdown efficiency was confirmed by western blotting (Extended Data Fig. Upon Rtn4 knockdown, we observed a loss of ER tubules (anti-KDEL staining; Extended Data Fig. 9d) and a corresponding decrease in lamellipodia formation, with increased polarized actin bundles at the wound edge. This was reflected in lower MDE values for actin (Fig. 6e,g), consistent with a shift towards contractile behaviour at convex curvatures. Conversely, Climp63 knockdown led to expansion of sheet-like ER (Extended Data Fig. 9e) and formation of actin bundles even at convex edges (Fig. 6f,h), again supporting the idea that ER sheets promote contractile migration modes and inhibit protrusive activity. Finally, we tested whether smaller reticulons have similar effects by overexpressing Rtn2b, Rtn3S and Rtn4b in MDCK cells. Among these, only Rtn4b overexpression led to enhanced lamellipodia formation and suppression of actin bundles, including at concave edges (Extended Data Fig. 10a–f), mirroring the phenotype observed with Rtn4a overexpression. All reticulons (Rtn1–4) have been shown to localize on ER tubules and promote membrane curvature generation through a conserved reticulon homology domain50. Rtn4a and Rtn4b overexpression shifts ER morphology towards tubules in mammalian cells51 and, as shown in our study, enhances lamellipodia formation at both convex and concave edges. Rtn4b has also been linked to wound healing through its role in macrophage migration52. Together with previous findings, our results suggest that altering ER morphology through various methods consistently influences epithelial migration modes. Since ER-resident proteins interact with the focal adhesion complex53,54,55,56, ER dynamics might be crucial for the maturation of cell-substrate adhesions during cell adhesion and migration57. Importantly, focal adhesions orient differently at the two edge curvatures during migration, aligning perpendicular to the edge at convex regions and parallel to the edge at concave regions15,16. We therefore hypothesized that different ER morphologies at the convex and concave edges might influence the orientation and stability of focal adhesions. To test this hypothesis, we first checked whether ER tubules or sheets contacted focal adhesion punctae, using paxillin as the focal adhesion marker. Co-localization images revealed that ER tubules were associated with perpendicular paxillin punctae at the convex edge, while ER sheets were associated with parallel focal adhesions at the concave edge (Fig. Next, we quantified the fraction of focal adhesions of a particular orientation associated with ER sheets or tubules. Adhesions aligned at 0–18° relative to the edge were considered as parallel and those aligned at 72–90° were considered as perpendicular focal adhesions. Quantification revealed that at both curvatures, perpendicular focal adhesions mainly contacted ER tubules, while the parallel focal adhesions were associated with the ER sheets (Fig. Then, to determine whether altering ER morphology would alter the orientation of focal adhesions, we examined the orientation of paxillin punctae in Rtn4a- or Climp63-overexpressing cells (Fig. 7c), which showed an increased propensity to form ER tubules and sheets, respectively. Subsequently, Rtn4a overexpression led to a decrease in the fraction of parallelly oriented focal adhesions and increased perpendicularly oriented adhesions at the concave edge (Fig. In contrast, Climp63 overexpression led to a decrease in the fraction of perpendicularly oriented focal adhesions and increased parallelly oriented focal adhesions at the convex edge (Fig. Taken together, these results suggested that morphological changes in the ER modulate focal adhesion, promoting different modes of migration at the two curvatures. a, Representative images of MDCK cells expressing mApple-Sec61β (yellow) migrating at concave (top) and convex (bottom) edges stained with anti-paxillin (magenta) and phalloidin (green). b, Quantification of the fraction of perpendicular focal adhesions (from left to right: n = 58, 26, 47 and 70) and parallel focal adhesions (from left to right: n = 58, 26, 47 and 70). c, Representative images of MDCK WT (top), Rtn4a-GFP (yellow, middle) and mCherry-Climp63 (yellow, bottom) cells stained with anti-paxillin (green) and phalloidin (magenta). The blue arrowheads show perpendicular FA and white arrowheads show parallel FA. d, Summary of the study: a schematic (left) depicting ER tubules at the front of cells migrating at convex edge sliding and growing along the microtubules, and associating with the perpendicularly oriented focal adhesions promoting lamellipodial crawling. A schematic (right) depicting ER sheets accumulated at the front of cells migrating at the concave edge, associating with parallel focal adhesions and promoting actin bundle-driven purse string closure. Positioning ER as a key player in mechanotransduction, integrating signals from microtubules and actomyosin networks to regulate cellular responses to physical stimuli. Data are mean ± s.e.m with a one-way ANOVA test (b). All experiments were performed at least three independent times. Epithelial gap closure is a fundamental process in both health and disease, involving collective migration through lamellipodia or purse string contractions, each with distinct actin and adhesion dynamics. This response is strongly influenced by edge curvature, but the underlying mechanisms remain unclear. Here, we show that among intracellular organelles, the ER undergoes striking curvature-dependent morphological changes and plays a key role in determining the mode of epithelial migration (Fig. The ER is the largest and most dynamic cell organelle, which has been hypothesized to be a sensor of multiple extracellular cues attributing to its dynamicity and elaborate structure spanning the whole cell17,24,58,59. The curvature-dependent shift in ER morphology, from tubules at convex edges to sheets at concave edges, suggests that ER organization senses and responds to physical cues, driven by protrusive and contractile forces from actin, microtubules and potentially intermediate filaments. Alongside these findings, our theoretical model suggests that these morphologies minimize strain energy within their corresponding geometric contexts. This provides a mechanistic basis for curvature-specific ER organization, though further work is needed to clarify how mechanical constraints and cytoskeletal inputs jointly shape ER structure during migration. Relevantly, an important consideration is whether the dense ER structures at concave edges represent true ER sheets or instead dense tubular matrices, as described in previous studies29. Our results indicate that the dense ER structures at concave edges are predominantly sheets, supported by Climp63 and p180 enrichment, ER–plasma membrane contact patterns and Rtn4b imaging. Consistently, modelling shows that ER sheets have the lowest strain energy at concave edges, reinforcing their role as key regulators of curvature-guided cytoskeletal organization and migration. Interestingly, ER tubule formation at convex edges relies on microtubule-dependent dynamics, including kinesin-driven sliding and tip attachment, whereas these interactions are dispensable for ER sheet accumulation at concave edges. While it is possible that some ER sheet-associated proteins60,61, including Climp63 and p180, may support ER sheet accumulation at the concave edge, whether actomyosin contractility can directly promote ER sheet accumulation remains unknown. In addition, ER structure and dynamics influence the distribution and post-translational modifications of microtubules27,62. Therefore, ER reorganization in response to curvature may influence microtubule modifications, potentially altering organelle distribution and polarity during migration. Functionally, manipulating ER morphology alters the migration mode, with Climp63 promoting actin bundles and contractility even at convex edges, and Rtn4a enhancing protrusions at concave edges, probably through changes in focal adhesion orientation. However, ER-mediated regulation of focal adhesions could be just one of the mechanisms by which ER dictates the mode of epithelial cell migration. ER tubules are involved in lipid biogenesis, calcium signalling and forming membrane contact sites with other organelles and the plasma membrane. A reticulated network of ER tubules at the convex edge could also regulate lamellipodial crawling by providing membrane lipids via the contact sites and by releasing calcium for the activation of the actomyosin network. Conversely, ER sheets are involved in protein synthesis, and sheet accumulation at the concave edge might promote directed protein synthesis and delivery, facilitating purse string-like movements. How the conventional roles of ER subdomains differ at the two curvatures and guide cell migration remains unknown. Despite these remaining questions, our findings highlight the ER as a central mechanotransducer that integrates cytoskeletal signals (Fig. 7e) to coordinate curvature-dependent collective cell migration, underscoring the broader role of organelles in tissue homeostasis, development, and regeneration. The tetracycline-resistant WT MDCK cells utilized in this investigation were generously provided by Yasuyuki Fujita. These cells were cultured in Dulbecco's modified Eagle medium (DMEM) with GlutaMAX (Gibco), supplemented with 5% fetal bovine serum (Tet-free FBS, Takara Bio) and 10 U ml−1 penicillin and 10 μg ml−1 streptomycin (pen–strep, Invitrogen). Cultures were maintained in a CO2 incubator at 37 °C. EPH4-Ev epithelial cells (ATCC, CRL-3063), used in the study, were maintained in DMEM with GlutaMax (Gibco), supplemented with 10% FBS (Tet-free FBS, Takara Bio) and 1.2 μg ml−1 puromycin (Gibco). These cells were also maintained in a 37 °C, 5% CO2 environment. Subculturing of both cell lines occurred every 3–4 days utilizing Trypsin/EDTA (Invitrogen). C57/6J strain mice, 8–12-week-old adult females were used in the study for all experiments involving mice. No statistical methods were used to predetermine sample size. For each independent experiment, embryos were dissected from different pregnant females. E14.5–15.5 stage embryos were dissected and used for all wound healing assays. The development and physiology of the embryonic epidermal tissue is not known to be different in different sexes, therefore sex of the animal was not considered a parameter in the study. The protocol was approved by the Institutional Animal Ethics committee, TIFR Hyderabad on behalf of Committee for the Purpose of Control and Supervision of experiments on Animals, India. siRNAs from Dharmacon (Supplementary Table 2) were dissolved in 1× siRNA buffer (B-002000-UB-100) to prepare 5 μM siRNA stock. Stock solutions were aliquoted into multiple tubes to avoid repeated thawing. For siRNA transfection experiments in 12-well plates, Eph4 cells were seeded 16 h before transfection and transfected with siRNA using DharmaFECT 1 (Dharmacon, T-2001-02) according to the manufacturer's protocol. In brief, 75 nM siRNA and 2 μl DharmaFECT 1 were diluted with Opti-MEM medium (Gibco, 31-985-070) to prepare a 100 μl volume system in separate tubes. After 5 min of incubation at room temperature, the two tubes were mixed thoroughly and gently for another 20 min incubation. Then the 200 μl of transfection mix was transferred to each well for a 9–10 h transfection before the medium was replaced with the complete growth medium. The transfected cells were collected from the 12 wells 48 h after transfection and were seeded for migration experiments. For plasmid transfection, cells were treated with respective plasmids and Lipofectamine 2000 (Invitrogen), following the instructions provided by the manufacturer. At approximately 8–10 h post-transfection, cells were seeded for the migration assays. MDCK cells stably expressing Rtn4a-GFP or mCherry-Climp63 cells were transfected with the respective plasmids using Lipofectamine 2000 (Invitrogen) following the instructions provided by the manufacturer. At 16–18 h post-transfection, cells were seeded via serial dilution in a 96-well plate. At 48 h post-cell seeding, the cells were subjected to selection media (DMEM–GlutaMax plus 5% FBS) containing 600 μg ml−1 of Geneticin (Invitrogen). The growth of single cell-derived colonies was monitored over 2 weeks following fluorescence confirmation. Colonies with a homogeneous expression of the fluorescence-tagged protein of interest were expanded and maintained in media containing 400 μg ml−1. To ensure further homogeneous expression, these cells were sorted using FACS before each experiment. Briefly, the cells were trypsinized and the pellet was resuspended in Fluorobrite media with 20 mM HEPES. The suspension containing approximately 1 million cells were sorted and only cells showing high expression as compared with WT were selected and seeded for migration experiments. EpH4 cells treated with nontargeted siRNA, CLimp63 siRNA and Rtn4 siRNA were trypsinized and the pellet was resuspended in RIPA lysis buffer (50 mM Tris,150 mM NaCl, 0.2% Triton X-100 and 1× protease and phosphatase inhibitor cocktail), followed by centrifugation at 16,000g for 10 min at 4 °C. The supernatant was used for total protein estimation by a Pierce 660 nm assay. The lysate was mixed with equal volume of 1× Laemmli buffer (0.004% bromophenol blue, 20% glycerol, 4% SDS and 0.125 M Tris–HCl pH 6.8) and 5% beta-mercaptoethanol and heated at 98 °C for 5 min, centrifuged and then 20 μg of protein was loaded in each well along with the ladder. Wet transfer was done onto 0.2 μm nitrocellulose membrane as per the Trans-Blot Turbo kit (Bio-Rad, 1704270) for 30 min. Blocking was in or with 5% BSA made in 1× TBS with 0.1% Tween-20 (1× TBSTw20) for 1 h at room temperature followed by primary antibody incubation overnight at 4 °C. After washing three times in 1× TBSTw20, the membranes were incubated in HRP-conjugated secondary antibodies for 2 h at room temperature. After washing, they were developed using Clarity Western ECL Substrate (Pierce ECL, Thermo Fisher Scientific) and visualized using an iBright gel doc. PDMS6 stencils with pillars of defined shapes were created using soft photolithography. Briefly, 150-μm-thick SU-8 2075 (Kayaku advanced materials, Y111074) was coated on a silicon wafer followed by soft baking at 65 °C for 5 min and 95 °C for 20 min. The SU-8 was exposed in the desired pattern using the MicroWriter ML 3 Pro (Durham Optics Magneto Ltd.) followed by post-exposure baking at 65 °C for 7 min and 95 °C for 15 min. Patterned SU-8 wafers were then hard baked at 170 °C for 45 min. PDMS (Sylgard 184, Dow Corning) was mixed in a ratio of 1:10, degassed and poured over the patterned wafers. After curing at 85 °C for 2 h, stencils with pillars were peeled off. The stencils were then treated with 2% pluronic acid for 2 h followed by 15 min of a 1× PBS wash. The PDMS stencil and a Petri dish were plasma cleaned for 10 s, and the stencil was then placed on the dish. A solution of 20 μg ml−1 fibronectin was applied to a glass-bottom Petri dish (Ibidi). After three washes to remove excess protein, the dish was air dried. The PDMS stencil underwent plasma cleaning for 20 s before being positioned onto the dish. The Petri dish with the stencil was then subjected to UV sterilization for 1 h. A concentrated suspension of MDCK or other cells was seeded around the pillars, followed by the addition of excess media around the stencil. Cells were allowed to settle around the pillars for 15–18 h. Subsequently, the stencils were lifted, enabling cell migration at various curvatures. E14.5–E16.5 mouse embryos were dissected out. The dorsal skin from these embryos was then excised. The skin was placed on a filter paper (Whatman filter, 8 μm pore size) with the epidermal side up. Using a scalpel/fine scissor, incision wounds were made on the epidermis. The filter paper along with the tissue was kept floating in a 6-well plate containing 2 ml of DMEM with 5% FBS and 1% pen–strep. After the desired incubation period, the tissue was fixed in 4% paraformaldehyde for 2 h at room temperature. The tissue was then rehydrated using graded ethanol (75%, 50% and 25% in PBST). After two more washes in PBST, the tissue was blocked using 10% normal goat serum in PBST for 2 h. Primary antibodies were added and incubated overnight at 4 °C. This was followed by subsequent PBST washes, after which secondary antibodies were added and incubated for 2 h at room temperature. After washing, the tissue was mounted on a slide using Fluoroshield (Sigma-Aldrich) mounting media. The slide was cured overnight and imaged using an upright confocal (Leica Stellaris). After the skin was excised from the embryo, it was placed on a filter paper with the epidermis side up. The explant was incubated in a well containing the culture media (DMEM with 5% FBS and 1% pen–strep) and staining solution (SiR-actin and verapamil) overnight. Then, using a scalpel, incision wounds were made, and it was immediately transferred to a glass-bottom dish, such that the epidermis side was now facing the glass coverslip. Next, 20 µl of Matrigel was added and incubated for 10 min to help immobilize the explant. A confined chamber was placed on top of the filter and culture medium was added. The sample was then imaged using an inverted confocal. Cells were fixed using a solution containing 3% formaldehyde and 0.1% glutaraldehyde diluted in 1× PBS for 10 min at room temperature. Following fixation, cells underwent three washes with 1× PBS. Subsequently, permeabilization was carried out using 0.25% Triton X-100 in PBS for 10 min at room temperature, followed by three quick washes with 1× PBS. Cells were then incubated with a blocking solution (consisting of 0.1% Triton X-100 in 1× PBS + 2% BSA) for 1 h at room temperature. Afterwards, cells were incubated with a primary antibody prepared in the blocking solution for 2 h at room temperature. Following the primary antibody incubation, cells were washed three times with 1× PBS and then incubated with the secondary antibody, 4,6-diamidino-2-phenylindole (DAPI) (1:1,000) and phalloidin (1:40) dilutions, all prepared in the blocking/staining solution, for 1 h at room temperature. Finally, the samples underwent two quick washes and a 5 min wash with 1× PBS before proceeding to imaging. Fluorescence images were captured using a 60× oil objective (PlanApo N 60× oil, NA of 1.42, Olympus) and a 100× oil objective (UPlanSApo, 100×/1.40 oil), both mounted on an Olympus IX83 inverted microscope equipped with a scanning laser confocal head (Olympus FV3000). Time-lapse imaging of live samples was conducted in the live-cell chamber provided with the microscopy set-up. For the photoactivation experiment, a 60× oil objective (PlanApo N 60× oil, NA of 1.42, Olympus) mounted on an Olympus IX83 inverted microscope equipped with a scanning laser confocal head (Olympus FV3000) was utilized, supported by a live-cell imaging set-up. MDCK cells, either co-expressing photoactivable Rac1 and GFP-Sec61β or expressing photoactivable Rac1 and labelled with SiR-tubulin, were seeded on a glass-bottom dish in Opti-MEM medium with HEPES (20 mM). A region of interest (ROI) approximately 5 μm in diameter was irradiated with a 1% intensity 445 nm laser at 1,000 μs pixel−1, followed by laser scanning microscopy imaging of the two channels (561 nm and 488 nm or 561 nm and 647 nm) at 0.1–0.5% intensity. This cycle of activation and imaging was repeated continuously for 15–20 min. Fluorescence images were acquired using 100× oil objective (Apo TIRF, NA of 1.49, WD of 0.12) mounted on a Nikon inverted research microscope Eclipse Ti2-E with Yokogawa CSU-W1 SoRa unit. A further 2.8× magnification was used to acquire super-resolution images with the SoRa unit. After the acquisition, images were subjected to Richardson–Lucy 3D deconvolution. For all inhibition studies, cells underwent pretreatment with the desired inhibitor concentration in complete DMEM for 1 h at 37 °C in a 5% CO2 humidified incubator before lifting the PDMS stamps. To modulate actomyosin contractility, MDCK cells were treated with blebbistatin (Sigma), a myosin inhibitor, at a concentration of 50 μM. Arp2/3-inhibition was achieved using CK666 (Sigma) at a concentration of 50 μM. Nocodazole (Sigma) was utilized to disrupt microtubules in MDCK cells at a concentration of 10 μM. Following inhibitor treatment, cells were fixed and immunostained with the desired antibodies. Source and dilution information for all primary and secondary antibodies used in immunofluorescence staining are given in Supplementary Table 1. Details of plasmids used in this study are listed in Supplementary Table 2 with their source. Image analysis was conducted using Fiji. To determine the fraction of ER tubules, Trainable Weka Segmentation was utilized to distinguish between tubules and sheets within the ER structures. To train the model, we selected key image processing features, including Gaussian blur, Sobel filter, Hessian, difference of Gaussians and membrane projections, which help distinguish ER tubules from sheets based on intensity, texture and edge detection. A sample image was manually annotated to define three structural categories or classes as follows: (1) thin peripheral ER tubules; (2) dense, bright ER sheets; and (3) background regions that included cytoplasmic spaces between ER structures and the extracellular area. The tool was iteratively trained by manually marking multiple areas in each category until the segmented output closely matched the original image. Once trained, the model was saved and applied to multiple images to ensure consistency and minimize user bias in segmentation. Following classification, the segmented image was converted to an 8-bit greyscale format for further quantification. Using thresholding, we extracted binary masks for tubules, sheets and total ER. The pixel area corresponding to each category was measured, providing quantitative data on the fraction of tubules per cell. This automated approach ensured an objective assessment of ER morphology across different experimental conditions. To assess the reliability of our Weka-based segmentation, we performed a validation by comparing automated segmentation outputs with manual annotations. A subset of images (n = 30) was manually segmented using Fiji's polygon selection tool to trace ER tubule and sheet regions. We then applied our trained Weka model to the same images and quantified the corresponding pixel areas. The fraction of ER tubules obtained by both methods showed a strong linear correlation (Pearson r = 0.99, P < 0.001), indicating that the automated classification reliably replicates manual assessment, demonstrating the robustness and accuracy of our segmentation pipeline. The same classifier was applied for ER segmentation in all inhibition and mutant studies. In the photoactivation experiment of Rac1, the ROI for Rac1 activation was selected for segmentation, while a similarly sized ROI at the nonactivated region within the cell served as a control. The Directionality plugin in Fiji was used to quantify the orientation of focal adhesions. The Fourier components method, 6 Nbins from 0° to 90°, was used. This computes and generates a histogram representing the frequency of structures (focal adhesions) at a particular orientation. Focal adhesions (FAs) aligned between 0° and 18° were considered parallel to the wound, while those aligned between 72° and 90° were deemed perpendicular. FAs not falling within these ranges were excluded from the analysis. The data over many cells at different curvatures were collected and averaged to finally plot the focal adhesion orientation. Microtubule orientation was also determined using the directionality plugin in Fiji, with 10 Nbins and angles ranging from 0° to 180°. An ROI at the front of the migrating cell in the α-tubulin channel was chosen, and directionality frequency was calculated. Statistical analyses were performed using GraphPad Prism 9. Statistical significance was determined using an unpaired t-test with Welch's correction or a one-way analysis of variance (ANOVA), as specified in the corresponding figure legend. Scatter-bar plots were presented as mean ± s.e.m. P values above 0.05 were considered not statistically significant. Quantification was based on data from a minimum of three independent biological replicates. Data distribution was assumed to be normal, but this was not formally tested. No statistical method was used to predetermine sample size, but our sample sizes are consistent with the standard of the field and are appropriate for the types of analyses performed. Samples were not allocated using randomization, as this study involved observational analysis of epithelial cells under defined experimental conditions. All groups were subjected to identical culture, treatment and imaging protocols to control for potential covariates, ensuring comparability across conditions. Data collection and analysis were not performed blind to the conditions of the experiments. No data points were excluded from the analyses. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data supporting the findings of this work are available in this Article and its extended data figures and Supplementary Information. All other data supporting the findings of this study are available from the corresponding author on reasonable request. The time-lapse imaging data (otherwise available as supplementary video files) can be made available only upon request owing to the large file sizes and associated storage limitations. Source data are provided with this paper. The code for mathematical modelling is available as an open-source download via GitHub at https://github.com/bkprdp/Curvature_Dependent_ER_Morphology. Mayor, R. & Etienne-Manneville, S. The front and rear of collective cell migration. & Mège, R. M. Mechanical plasticity in collective cell migration. Vishwakarma, M., Spatz, J. P. & Das, T. Mechanobiology of leader-follower dynamics in epithelial cell migration. Wound healing recapitulates morphogenesis in Drosophila embryos. & Martin, P. Mechanisms of epithelial fusion and repair. Begnaud, S., Chen, T., Delacour, D., Mège, R. M. & Ladoux, B. Mechanics of epithelial tissues during gap closure. & Bement, W. M. Wound repair: toward understanding and integration of single-cell and multicellular wound responses. Abreu-Blanco, M. T., Verboon, J. M., Liu, R., Watts, J. J. & Parkhurst, S. M. Drosophila embryos close epithelial wounds using a combination of cellular protrusions and an actomyosin purse string. Klarlund, J. K. Dual modes of motility at the leading edge of migrating epithelial cell sheets. Le, A. P. et al. Adhesion-mediated heterogeneous actin organization governs apoptotic cell extrusion. Wound healing coordinates actin architectures to regulate mechanical work. Peter, B. J. et al. BAR domains as sensors of membrane curvature: the amphiphysin BAR structure. Lobato-Márquez, D. & Mostowy, S. Septins recognize micron-scale membrane curvature. Ravasio, A. et al. Gap geometry dictates epithelial closure efficiency. Large-scale curvature sensing by directional actin flow drives cellular migration mode switching. Lee, J. E., Cathey, P. I., Wu, H., Parker, R. & Voeltz, G. K. Endoplasmic reticulum contact sites regulate the dynamics of membraneless organelles. English, A. R., Zurek, N. & Voeltz, G. K. Peripheral ER structure and function. Structural organization of the endoplasmic reticulum. English, A. R. & Voeltz, G. K. Endoplasmic reticulum structure and interconnections with other organelles. The structure and global distribution of the endoplasmic reticulum network are actively regulated by lysosomes. Applying systems-level spectral imaging and analysis to reveal the organelle interactome. Architecture and dynamics of a desmosome–endoplasmic reticulum complex. & Voeltz, G. K. Form follows function: the importance of endoplasmic reticulum shape. Phuyal, S. & Baschieri, F. Endomembranes: unsung heroes of mechanobiology? Zheng, P. et al. ER proteins decipher the tubulin code to regulate organelle distribution. Arganda-Carreras, I. et al. Trainable Weka Segmentation: a machine learning tool for microscopy pixel classification. Increased spatiotemporal resolution reveals highly dynamic dense tubular matrices in the peripheral ER. Shen, B. et al. Calumenin-1 interacts with Climp63 to cooperatively determine the luminal width and distribution of endoplasmic reticulum sheets. Gong, B., Johnston, J. D., Thiemicke, A., de Marco, A. & Meyer, T. Endoplasmic reticulum–plasma membrane contact gradients direct cell migration. Forces driving epithelial wound healing. Wu, Y. I. et al. A genetically encoded photoactivatable Rac controls the motility of living cells. Mechanical interactions among followers determine the emergence of leaders in migrating epithelial cell collectives. Lin, C., Lemarchand, L., Euler, R. & Sparkes, I. Shemesh, T. et al. A model for the generation and interconversion of ER morphologies. Keshavanarayana, P., Ruess, M. & Borst, R. D. A feedback-loop extended stress fiber growth model with focal adhesion formation. & Fujiwara, K. Microtubules and the endoplasmic reticulum are highly interdependent structures. Farías, G. G. et al. Feedback-driven mechanisms between microtubules and the endoplasmic reticulum instruct neuronal polarity. Joensuu, M. et al. ER sheet persistence is coupled to myosin 1c-regulated dynamic actin filament arrays. Friedman, J. R., Webster, B. M., Mastronarde, D. N., Verhey, K. J. & Voeltz, G. K. ER sliding dynamics and ER–mitochondrial contacts occur on acetylated microtubules. Balancing ER dynamics: shaping, bending, severing, and mending membranes. Zhang, X. et al. Kinectin-mediated endoplasmic reticulum dynamics supports focal adhesion growth in the cellular lamella. Komarova, Y. et al. Mammalian end binding proteins control persistent microtubule growth. Bi, D., Lopez, J. H., Schwarz, J. M. & Manning, M. L. A density-independent rigidity transition in biological tissues. Zurek, N., Sparks, L. & Voeltz, G. Reticulon short hairpin transmembrane domains are used to shape ER tubules. Yu, J. et al. Reticulon 4B (Nogo-B) is necessary for macrophage infiltration and tissue repair. Hernández, M. V., Sala, M. G., Balsamo, J., Lilien, J. & Arregui, C. O. ER-bound PTP1B is targeted to newly forming cell-matrix adhesions. The role of protein–tyrosine phosphatase 1B in integrin signaling. Papp, S., Szabo, E., Kim, H., McCulloch, C. A. & Opas, M. Kinase-dependent adhesion to fibronectin: regulation by calreticulin. Ng, I. C. et al. Kinectin-dependent ER transport supports the focal complex maturation required for chemotaxis in shallow gradients. & Allan, V. How and why does the endoplasmic reticulum move? Shaping the endoplasmic reticulum into a social network. Schwarz, D. S. & Blower, M. D. The endoplasmic reticulum: structure, function and response to cellular signaling. Klopfenstein, D. R., Kappeler, F. & Hauri, H. P. A novel direct interaction of endoplasmic reticulum with microtubules. Ogawa-Goto, K. et al. p180 is involved in the interaction between the endoplasmic reticulum and microtubules through a novel microtubule-binding and bundling domain. Tikhomirova, M. S., Kadosh, A., Saukko-Paavola, A. J., Shemesh, T. & Klemm, R. W. A role for endoplasmic reticulum dynamics in the cellular distribution of microtubules. We thank R. Sharma (TIFR Hyderabad) for his assistance related to micropatterning and P. Gupta (TIFR Hyderabad) for western blot experiments. We thank T. N. Narayanan (TIFR Hyderabad) for providing the facility of plasma cleaner. We acknowledge H. Yu (National University of Singapore) and A. Akhmanova (Utrecht University) for generously providing plasmids used in this study and Y. Fujita (Kyoto University) for the WT MDCK cell line. This work was funded by Science and Engineering Research Board, India (now ANRF) (core research grant no. and by the BBSRC grant BB/V002708/1 (to F.S. In addition, we acknowledge generous support from the Human Frontier Science Program research grant (no. RGP0007/2022) to T.D., and intramural funds at TIFR Hyderabad from the Department of Atomic Energy, India, under the project identification number RTI 4007 to S.R., D.M. Simran Rawal, Diya Manoj, Purnati Khuntia, Sanak Banerjee, Basil Thurakkal, Rituraj Marwaha & Tamal Das Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed most of the experiments and analyses. performed mouse embryonic skin wound healing experiments. exclusively performed mouse embryonic skin live imaging. helped in the shape index analysis. modified the MATLAB code for MDE calculations. performed Imaris analysis on a few images. All authors approved the paper in its submitted form. Correspondence to Fabian Spill or Tamal Das. The authors declare no competing interests. Nature Cell Biology thanks Tsuyoshi Hirashima and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Sample image of a notched rectangle pattern, and bright-field time lapse images of MDCK cells closing the micro-patterned gap; scale bar: 50 µm. b, Sample image of a flower pattern, and bright field time lapse images of MDCK cells closing the micro patterned-flower gap; scale bar: 20 µm. c, Time-lapse images of LifeAct MDCK cells migrating at concave (upper panel) and convex (Lower panel), Zoomed insets showing actin remodeling and evolution of curvature in time, dashed white lines represent the edges; scale bar: 10 µm. d, Quantification of the fraction of cells forming lamellipodia out of the total cells at the edge at two curvatures no. of RoIs= 49, 38 from left to right. e, Color wheel representation of the directionality of microtubules at convex (left) and concave (right) curvatures, color wheel ranging from 0 to 180 represents the angle of microtubules relative to the edge of the wound. f, Quantification for directionality of microtubules at convex (green) and concave (pink) curvature n = 77 (convex), n = 74 (concave). Each experiment was repeated at least three independent times. h, Quantification of κtissue as a function of κcell at convex edge considering only Pexp > 20%, n = 77, dotted line-linear trend line. Each experiment was repeated at least three independent times. a, Representative images of MDCK cells stained for Golgi, lysosome, and mitochondria at convex (upper panel) and concave (lower panel) curvature. MDCK cells stained for left panel- anti-GRASP65 (magenta, Golgi marker), phalloidin (green) and DAPI (blue); middle panel- anti-LAMP1 (magenta, lysosome marker), phalloidin (green) and DAPI (blue); right panel- cells labeled with Mitotracker green (magenta, mitochondria marker) and DRAQ5 (blue), Scale bar: 10 µm. c, Quantification of MDE for actin, microtubules, ER, Nucleus, Golgi, lysosomes, and mitochondria at convex (green) and concave (pink) curvatures. n = 74, 62, 62 69, 83, 75, 86, 71, 63, 61, 80, 78, 86, and 88 from left to right. d, Quantification of fraction of Climp63 at the front of cells at convex (green) and concave (pink) edge, n = 74, 97. e, Representative images of LifeAct MDCK cells transfected with mAppleSec61β seeded in flower shape wound, dotted lines represent migrating edge, scale bar: 20 µm. f, Representative images of non-confluent MDCK-mApple sec61β colonies stained with phalloidin(magenta) and DAPI (Cyan), zoomed insets show cells at convex (green) and concave (pink) edges, scale bar: 10 µm. g, Quantification of fraction of ER tubules at the front of cells at convex (green) and concave (pink) edges in non-confluent monolayers n = 45, 45. Each experiment was repeated at least three independent times. a–c Representative images of EpH4 cells stained with a, phalloidin (yellow) and DAPI (cyan) b, anti-α-tubulin (yellow), Phalloidin (magenta) and DAPI (cyan) c, DAPI (cyan), Phalloidin (magenta), and expressing mAppple-Sec61β (yellow) at convex (upper panel) and concave curvature (lower panel), front of the cells at the edge is enlarged and zoomed, scale bar: 10μm. d, Quantification of MDE for actin, microtubules, and ER at convex (green) and concave (pink) curvatures. n = 100, 90, 58, 61, 59, and 49 from left to right. e, Time lapse zoomed images of mouse embryonic skin wound, labelled with SiR actin (blue) at concave (upper panel) and convex (Lower panel) edges. f, Mouse embryonic wounded skin tissue stained with phalloidin showing spontaneous convex (zoomed upper panel) concave (zoomed lower panel) curvature. g, Quantification for the directionality of microtubules at convex and concave curvature in mouse embryonic skin wounds (n = 24 at convex and n = 29 at concave curvature). One way ANOVA test (d). Each experiment was repeated at least three independent times. a, Representative images of MDCK cells transfected with mApple-Sec61β (red) in confluent monolayer stained with Left panel- Climp63 (cyan), zoomed insets represent enrichment of Climp63 at the cell center (purple) and at cell periphery (green); Right panel- Representative images of MDCK cells transfected with mApple-Sec61β (red) in confluent monolayer stained with p180 (cyan), zoomed insets represent enrichment of Climp63 at the cell center (purple) and at cell periphery (green) dotted boundary represents cell boundary, scale bar: 5 µm. b, Representative images of MDCK cells transfected with mApple-Sec61β (red) migrating at concave edge, stained with anti-Rtn4b antibody (cyan), inset (yellow and purple) represent zoomed region from the front of the cell, pink arrowhead represent Rtn4b (cyan) present around the ER sheet (red), dashed white line represent edge of the cell, scale bar: 3 µm. Right most panel: magnified super-resolution image of a flat ER sheet, surrounded by Rtn4b proteins, and a representative cartoon 3D render. c, Representative images of MDCK cells expressing GFP-MAPPER migrating at convex (upper panel) and concave (lower panel) curvature stained with phalloidin (green) and DAPI (cyan), scale bar: 10 µm. d, Quantification for GFP mapper intensity at the front at convex and concave curvatures n = 26 at each curve e, Time lapse super-resolution SoRa images of LifeAct (green) MDCK cells expressing mApple-Sec61β migrating at concave (upper panel) and Convex (lower panel) edges, black arrow represents direction of migration, dashed line represents migrating edge scale bar: 5 µm., Data are mean ± s.e.m. Each experiment was repeated at least three independent times. a, Representative images of MDCK cells treated with DMSO (control, left), CK666 (middle), and Blebbistatin (right) stained with phalloidin (grey), pink dashed lines mark concave curvatures and green dashed lines mark convex curvatures, scale bar: 20 μm. b, Quantification for fraction of cells forming lamellipodia at convex or concave curvatures in cells treated with DMSO, CK666, or Blebbistatin (no of ROIs=16, 14, 16, 16, 14, and 13 from left to right, each ROI). c, Representative images of MDCK cells expressing mApple-Sec61β (grey) treated with DMSO (left), CK666 (middle), or Blebbistatin (right) in the middle of confluent monolayers, scale bar: 10 μm. d, Quantification of fraction of ER tubules in the cells in the middle of the monolayer treated with DMSO, CK666, or Blebbistatin; n = 59, 42, and 47 from left to right. Data are mean ± s.e.m, One way ANOVA test. Each experiment was repeated at least three independent times. a, Schematic representing the purse-string contractile force acting on the cell with negative curvature on undeformed geometry (left). Orange dashed line indicate where boundary conditions are applied. Resulting deformation of cell (centre) and perpendicular ER (right). b, Schematic representing the protrusive force acting on the cell with positive curvature on undeformed geometry (left). Orange and black dashed lines indicate where boundary conditions are applied. Resulting deformation of cell (centre) and the perpendicular ER (right). c, Displacement map of cell with positive curvature during protrusion with perpendicular ER (left) and sheet ER (right). d, Displacement map of cell with negative curvature during contraction with perpendicular ER (left) and sheet ER (right). e, Representation of a scalar in (ω, ϕ) coordinates superimposed on Cartesian coordinate system. f, Schematic representing the notations used for evaluating positive (left) and negative (right) curvatures. h, Table elucidating formula to evaluate flexural rigidity of different cross sections. i, Strain energy density of cell with negative curvature during contraction with parallel ER and varying cross sections. a, Representative super-resolution (SoRa) images of cells expressing mApple-Sec61β (left), mCherry-Climp63 (middle) and Rtn4a-GFP (right); scale bar: 10μm. b, Representative MDCK cells migrating at concave edge heterogeneously expressing Rtn4a-GFP (yellow), stained with phalloidin (magenta), zoomed inset shows Rtn4a-GFP expressing cell forming lamellipodia at concave edge, scale bar: 10μm. c, Representative MDCK cells migrating at convex edge heterogeneously expressing mCherry-Climp63 (yellow), stained with phalloidin (magenta) zoomed inset shows mCherry-Climp63 expressing cell forming actin bundle at convex edge, scale bar: 10μm. d, Quantification of fraction of MDCK WT and Rtn4a over-expressing cells forming lamellipodia at convex and concave edges no. of ROIs= 90, 76, 96, and 89. e, Quantification of fraction of MDCK WT and Climp63 over-expressing cells forming lamellipodia at convex and concave edges no. of ROIs= 53, 39, 41, and 44. f, Representative images of MDCK cells expressing mCherry-Climp63 (blue, upper panel), Rtn4a-GFP (blue, lower panel) in confluent monolayer, stained with anti-αtubulin antibody (green), Phalloidin (red) and DAPI(cyan), yellow dashed line marks the transfected colonies, scale bar: 10 µm. g, Representative images of MDCK WT (left), MDCK overexpressing Climp63 (blue, middle), MDCK overexpressing Rtn4a (blue, right) in confluent monolayer, stained with anti ZO-1 antibody (red) and DAPI (Cyan), scale bar: 10 µm. h, Quantification of shape index of MDCK WT, Climp63 OE and Rtn4a OE, n = 268, 281, and 249. i, Representative images of MDCK overexpressing mCherry-Climp63 (red) seeded in micro patterned gaps untreated (upper panel) and Blebbistatin-treated (lower panel) stained for anti-αtubulin antibody (green) and Phalloidin (blue), scale bar: 10 µm. j, Quantification for directionality of microtubules, in Climp63 OE cells in untreated (left Convex n = 89, Concave n = 86) vs Blebbistatin-treated (right convex n = 85, concave n = 86) conditions at convex (green) and concave (pink) curvatures. k, Representative images of MDCK overexpressing Rtn4a-GFP (red) seeded in micro patterned gaps and untreated (upper panel) and CK666 treated (lower panel) stained for anti-αtubulin antibody (green) and Phalloidin (blue), scale bar: 10 µm. l, Quantification for directionality of microtubules, in Rtn4a-GFP OE cells in untreated (left Convex n = 77, Concave n = 86) vs CK666 (right convex n = 79, concave n = 78) treated conditions at convex (green) and concave(pink) curvatures. One way ANOVA test (d, e, h). Each experiment was repeated at least three independent times. a, Representative images of MDCK cells overexpressing mCherry-Climp63 (green) untreated (upper panel) and treated with Nocodazole (lower panel) stained with phalloidin (red) and DAPI (cyan), scale bar: 10 µm. b, Representative images of MDCK cells overexpressing Rtn4a-GFP (green) untreated (upper panel) and treated with Nocodazole (lower panel) stained with phalloidin (red) and DAPI (cyan), scale bar: 10 µm. c, Representative western blot for Left- EpH4 cells treated with either non targeting siRNA or Rtn4 siRNA probed for Rtn4b and GAPDH; Right- EpH4 cells treated for either non targeting SiRNA or Climp63 siRNA probed for Climp63 and GAPDH. d, Representative images of EpH4 cells treated with non-targeting siRNA (upper panel) and Rtn4 SiRNA (lower panel) stained with phalloidin (red) and DAPI (cyan) and anti-KDEL antibody (yellow), scale bar: 10 µm. e, Representative images of EpH4 cells treated with non-targeting siRNA (upper panel) and Climp63 siRNA (lower panel) stained with phalloidin (red) and DAPI (cyan) and anti-KDEL antibody (yellow), scale bar: 10 µm. Imaging experiments were repeated at least three independent times. a, Representative images of MDCK cells transiently overexpressing Rtn2b-GFP (red) and migrating at convex (upper) and concave (lower) curvature stained with phalloidin (green) and DAPI (cyan), scale bar: 10 µm. b, Representative images of MDCK cells transiently overexpressing Rtn3S-GFP (red) and migrating at convex (upper) and concave (lower) curvature stained with phalloidin (green) and DAPI (cyan), scale bar: 10 µm. c, Representative images of MDCK cells transiently overexpressing Rtn4b-GFP (red) and migrating at convex(upper) and concave (lower) curvature stained with phalloidin (green) and DAPI (cyan), scale bar: 10 µm. d, Quantification of MDE actin for WT and Rtn2b overexpressing cells at convex (green) and concave (pink) curvature n = 46, 52, 56, and 59 from left to right. e, Quantification of MDE actin for WT and Rtn3S overexpressing cells at convex (green) and concave (pink) curvature n = 70, 71, 64, and 70 from left to right. f, Quantification of MDE actin for WT and Rtn4b overexpressing cells at convex (green) and concave (pink) curvature n = 40, 47, 53, and 50 from left to right. Each experiment was repeated at least three independent times. Bright-field video of MDCK cells migrating into micropatterned wound. The movie is shown at 6 frames per second. Bright-field video of MDCK cells migrating in flower-shaped micropatterned wound. The movie is shown at 6 frames per second. LifeAct MDCK (green) cells migrating at concave edge. The movie is shown at 6 frames per second. LifeAct MDCK (green) cells migrating at convex edge. The movie is shown at 6 frames per second. Mouse embryonic skin labelled with SiR-actin (blue), migrating into incision wounds. The movie is shown at 3 frames per second. LifeAct MDCK (magenta) cell transfected with mApple-Sec61β (yellow) migrating at convex curvature. The movie is shown at 10 frames per second. LifeAct MDCK (magenta) transfected cell with mApple-Sec61β (yellow) migrating at concave curvature. The movie is shown at 10 frames per second. LifeAct MDCK cells transfected with mApple-Sec61β migrating at concave curvature imaged in SORA at a 3 min interval. The movie is shown at 6 frames per second. LifeAct MDCK cells transfected with mApple-Sec61β migrating at convex curvature imaged in SORA at a 3 min interval. The movie is shown at 6 frames per second. LifeAct MDCK cells transfected with mCherry-Climp63 migrating at concave curvature imaged in SORA at a 3 min interval. The movie is shown at 6 frames per second. LifeAct MDCK cells transfected with mApple-Sec61β (black) migrating at convex curvature, zoomed at the migrating front showing ER tubule dynamics. The movie is shown at 10 frames per second. LifeAct MDCK cell transfected with mApple-Sec61β (black) migrating at concave curvature, zoomed at the migrating front showing ER dynamics. The movie is shown at 10 frames per second. MDCK cells transfected with PA-Rac1 (magenta) labelled with SiR-tubulin, the first frame is before photoactivation. The movie is shown at 10 frames per second. MDCK cells cotransfected with PA-Rac1 (magenta) and GFP-Sec61β, the first frame is before photoactivation. The movie is shown at 10 frames per second. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Rawal, S., Keshavanarayana, P., Manoj, D. et al. Edge curvature drives endoplasmic reticulum reorganization and dictates epithelial migration mode. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250810093708.htm'>Gold refuses to melt at temperatures hotter than the Sun's surface</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 09:09:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers taking the first-ever direct measurement of atom temperature in extremely hot materials inadvertently disproved a decades-old theory and upended our understanding of superheating. It's notoriously difficult to take the temperature of really hot things. Knowing precisely how hot these materials are is crucial for researchers to fully understand such complex systems, but taking these measurements has been, until now, virtually impossible. "We have good techniques for measuring density and pressure of these systems, but not temperature," said Bob Nagler, staff scientist at the Department of Energy's SLAC National Accelerator Laboratory. "In these studies, the temperatures are always estimates with huge error bars, which really holds up our theoretical models. Nagler and researchers at SLAC's Matter in Extreme Conditions (MEC) instrument co-led this study with Tom White, associate professor of physics at University of Nevada, Reno. "LCLS, paired with these innovative techniques, play an important role in advancing high energy density science and transformative applications like inertial fusion." The team was thrilled to have successfully demonstrated this technique - and as they took a deeper look at the data, they discovered something even more exciting. "We were surprised to find a much higher temperature in these superheated solids than we initially expected, which disproves a long-standing theory from the 1980s," White said. Every material has specific melting and boiling points, marking the transition from solid to liquid and liquid to gas, respectively. For instance, when water is heated rapidly in very smooth containers - such as a glass of water in a microwave - it can become "superheated," reaching temperatures above 212 degrees Fahrenheit (100 degrees Celsius) without actually boiling. This occurs because there are no rough surfaces or impurities to trigger bubble formation. But this trick of nature comes with an increased risk: The further a system strays from its normal melting and boiling points, the more vulnerable it is to what scientists call a catastrophe - a sudden onset of melting or boiling triggered by slight environmental change. For example, water that has been superheated in a microwave will boil explosively when disturbed, potentially causing serious burns. While some experiments have shown it is possible to bypass these intermediary limits by rapidly heating materials, "the entropy catastrophe was still viewed as the ultimate boundary," White explained. In their recent study, the team discovered that the gold had been superheated to an astonishing 19,000 kelvins (33,740 degrees Fahrenheit) - more than 14 times its melting point and well beyond the proposed entropy catastrophe limit - all while maintaining its solid crystalline structure. "It's important to clarify that we did not violate the Second Law of Thermodynamics," White said with a chuckle. "What we demonstrated is that these catastrophes can be avoided if materials are heated extremely quickly - in our case, within trillionths of a second." The findings suggest that there may not be an upper limit for superheated materials, if heated quickly enough. Nagler noted that researchers who study warm dense matter have likely been surpassing the entropy catastrophe limit for years without realizing it, due to the absence of a reliable method for directly measuring temperature. As just one example, White and Nagler's teams used this method again this summer to study the temperature of materials that have been shock-compressed to replicate the conditions deep inside planets. Now, we finally have a way to make those measurements." This work was funded in part by the DOE National Nuclear Security Administration and Office of Science Fusion Energy Sciences. LCLS is a DOE Office of Science user facilities. Note: Content may be edited for style and length. Megaquake Triggers Rare Tsunami Caught by NASA's SWOT Satellite Webb Telescope Spots Oldest Black Hole, Shattering Cosmic Records Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02633-4'>Why nurturing the gut microbiota could resolve depression and anxiety</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 08:15:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Simon Makin is a freelance writer in Reading, UK. You have full access to this article via your institution. “Some days I wasn't able to get out bed. I had long periods of unemployment and was living in my car for a time.” He struggled to find relief, nothing worked. “Everything helped a little, but it was still there.” Then, in the spring of 2023, he found a clinical trial that would change his life. “Within about a week, I started feeling better,” he says. Two years later, Moseson is still taking his previously prescribed medication. Numerous psychiatric and neurological conditions have been linked to disturbances in people's gut microbiota — the community of trillions of microorganisms that live symbiotically in the gastrointestinal tract. These are just correlations, but studies in rodents show compelling evidence of causality, and other animal research points to multiple pathways through which the microbiota communicates with the brain. Moseson's experience, and others like it, offer preliminary evidence that microbiota-based treatments can be beneficial for people with mood disorders, such as depression and anxiety. The procedure need not be a faecal transplant — good outcomes have also been observed with probiotics (microbial species or strains that have a beneficial effect on the microbiota) and diet. The trial that Moseson participated in was led by Valerie Taylor, a psychiatrist at the University of Calgary in Canada. Taylor became interested in the gut microbiota after two people with bipolar disorder approached her. “They'd been very sick for a long time and nothing had worked; both took antibiotics, and their symptoms completely changed,” Taylor says. Around that time, in 2016, two studies had just been published that showed transferring faecal matter from someone with depression into rodents gave the animals depression-like behaviours1,2. “This is not how we'd thought about mental illness, as something that can be transferred the way you could catch measles.” One study came from John Cryan, a neurobiologist at University College Cork in Ireland, and his colleagues, who had been looking at the influence of the gut microbiota in stress-related conditions, such as depression and anxiety. They had studied a small sample of people with depression and found that they harboured microbial communities that differed from those of healthy people. “To prove causality, we wanted to know, if we took that microbiome and put it into a healthy rat, would it do anything?” says Cryan. Rats that received faecal transplants from people with depression exhibited anhedonia (a lack of interest and pleasure) and anxiety-like behaviours, and had altered metabolism of tryptophan, a precursor to biomolecules such as the neurotransmitter serotonin1. Humans source tryptophan from their diet, because the body doesn't produce it, and gut microbes help to metabolize it. The second study, from a team led by researchers at Chongqing Medical University in China, showed the same result in mice, and also noted metabolic changes2. In December 2023, Cryan and his colleagues showed that symptoms experienced by people with social anxiety disorder could similarly be transferred to rodents through faecal transplants3. Mice that received the transplants showed normal behaviour across a battery of depression and anxiety tests, but showed heightened sensitivity to social fear. “This same paradigm of faecal microbiota transplant in animals has been replicated for autism features, and psychosis,” says Viktoriya Nikolova, a psychiatrist at King's College London. “We got very interested in whether, if you could transfer illness this way, could you do the reverse?” Taylor says. Their first trial, which Taylor says is due to be published later this year, was focused on bipolar disorder. Taylor reasoned that if they could make progress in a condition that is generally thought of as highly genetically determined and hard to treat, it should work on other disorders, too. And if anything was going to work in humans, she thought, it would be the comprehensive alterations of the microbiota brought about by FMT, as opposed to tinkering with it by adding a handful of microbial strains. “We needed faecal transplants to see what happens when you completely reset things,” Taylor says. They have three more trials at various stages of completion: two focusing on depression, including the one Moseson participated in, and one on obsessive-compulsive disorder. They are also planning another, on attention-deficit hyperactivity disorder. One drawback of FMT is that it does little to narrow down which microbes are responsible for the benefits. In 2021, Nikolova and her colleagues published a meta-analysis of seven such trials4. The analysis found that probiotics — known as psychobiotics when used in this way — are effective as an add-on, but not when used alone. “For now, the best evidence supports psychobiotics as adjunctive therapies,” says Wolfgang Marx, a nutritional psychiatrist at Deakin University in Geelong, Australia. “Something that can complement, rather than replace, standard care.” In 2023, Nikolova and her colleagues published a pilot trial of 49 people with depression who had previously had only a partial response to antidepressants5. Half of the participants received a multistrain probiotic, in addition to their medication, for eight weeks. “What we found was better than expected in terms of clinical efficacy,” says Nikolova. Researchers have also been investigating whether alterations of the microbiota through diet can be leveraged to improve mental health. “Dietary changes affect the broader microbial ecosystem,” says Marx, partly because it introduces many compounds that microbes feed on, known as prebiotics. Although “psychobiotics may be a more targeted approach, diet might offer a more broad-spectrum route to supporting mental health”, he says. Animal studies suggest that there are two-way communication routes between the gut and the brain, and they fall into four broad categories. And finally, metabolites produced by microbes can communicate with the brain in various ways. “There's very good evidence to support all these mechanisms,” says Nikolova. This web of pathways is collectively called the gut–brain axis, but little is currently known about its relative importance in human illness. Neurobiologist John Cryan has been studying the effects of the gut microbiota on mental health and stress-related conditions.Credit: Cathal Noonan for Glucksman Gallery, Univ. They found no effects on circulating immune-signalling molecules, called cytokines, or on brain-derived neurotrophic factor (BDNF), a growth factor involved in neuronal survival, which some studies4 have implicated in the link between the microbiota and the brain. Several previous trials also failed to find changes in levels of cytokines. All but one study found no significant impact on BDNF4. The problem might be that, because so many factors affect the microbiota, any effects of probiotics on biomarkers such as immune molecules are drowned out in small samples. “So we can get a signal above the noise.” Researchers don't yet fully understand what constitutes a healthy microbiota, nor do they have a complete catalogue of human gut microbes. Microbial diversity, for instance, was significantly greater in people who received probiotics than in the placebo group after four weeks, but not after eight. Symptoms continued to improve, however, suggesting that benefits might not depend solely on changes in microbiota composition. “It's telling us this isn't the only way that [probiotics] have an effect,” says Nikolova. Researchers now understand that looking only at changes in microbiota composition will not be enough to get to the bottom of how psychobiotics affect mood. “If one species goes down, another might pick up its functions,” says Nikolova. “So it's not just about what's missing, but rather whether the function is impaired.” Multiomics techniques, which analyse gene activity, proteins, metabolites and more, allow this depth of analysis. These tools were once prohibitively expensive, but decreasing costs are bringing them in reach of more researchers. Trials will also need to account for other factors that can affect microbiota and mental health, such as lifestyle, diet and genetics. “We need large-scale longitudinal studies across the lifespan, combined with brain imaging and other biomarker read-outs that are going to give us real understanding,” says Cryan. A further complication is that researchers don't yet fully understand how psychiatric drugs and probiotics interact. Probiotics could work by enhancing the action of antidepressants. Alternatively, if probiotics operate through a different mechanism, they might have separate effects to those medications. This could allow probiotics to augment the action of drugs, such as selective serotonin reuptake inhibitors (SSRIs), and make them a useful add-on to conventional therapy. However, it is also possible that probiotics have an additive effect. A reduction in the neurotransmitters dopamine and noradrenaline has been linked to depression and anxiety, but SSRIs do nothing to change this. This could partly explain why people with an anxious type of depression typically don't respond as well to antidepressants. The probiotic that Nikolova and her colleagues used in their trial4,7 included a Bacillus species that produces dopamine and noradrenaline, raising the possibility that the beneficial effects they saw came from targeting different mechanisms to those of participants' existing medications. A higher-resolution picture of microbiota changes — and their downstream consequences — will be needed if researchers are to use microbiota data diagnostically. In 2021, Nikolova and her colleagues published a meta-analysis8 of microbiota alterations linked to psychiatric diagnoses. They found that, although increases in pro-inflammatory bacteria and reductions in some anti-inflammatory bacteria were associated with depression, anxiety, bipolar disorder and schizophrenia, the differences did not discriminate between conditions. Such deeper understanding could even be used for prognosis. “Multiomics approaches are beginning to provide insight into the functional patterns that could predict treatment response or disease risk,” says Marx. “In theory, we could one day personalize psychiatric care based on a person's microbial and metabolic profile,” he says. But that day has yet to arrive — mainly, says Marx, because “most findings aren't validated in large clinical cohorts”. “The main issue is you can market products without any evidence, the margins are so much lower than in pharma and it's more difficult to protect, so the incentives aren't there,” says Cryan. “The flip side is we could democratize health care: by not making these expensive products, we could get them to more people, to socio-economic areas that might benefit most.” Researchers will need to look beyond food and supplement companies. “As a scientist, I need more evidence, but that requires funding through either philanthropic or government sources, we can't rely on industry to do it.” Moseson is evangelical about the treatment that helped him so much. “For a long time, I felt ashamed I felt this way, but after the transplant, I started thinking about my depression differently; knowing it wasn't me, it was my bacteria,” he says. This article is part of Nature Outlook: The human microbiome, a supplement produced with financial support from Yakult. Nature maintains full independence in all editorial decisions related to the content. Single-celled organisms set for greater role in gut health Gut bacteria make a molecule that promotes plaque build-up in arteries Teens with anxiety and depression spend more time on social media The infuriating, expensive road to a good night's sleep Nanobody therapy rescues behavioural deficits of NMDA receptor hypofunction Augusta University is seeking applications for faculty positions at the Assistant, Associate, or Full Professor at both tenured and tenure track level The Department of General and Visceral Surgery is looking for a Postdoctoral Researcher in Bioinformatics (m/f/d). You have full access to this article via your institution. Single-celled organisms set for greater role in gut health An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250816113526.htm'>Scientists uncover the lost shelduck that chose walking over flight</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 04:20:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The discovery of yet another unique animal species from Rēkohu Chatham Islands illustrates how the physical qualities of an animal are influenced by its surroundings. New research led by the University of Otago - Ōtākou Whakaihu Waka describes a new, extinct shelduck whose ancestors arrived on the islands 390,000 years ago. While this may seem like a short period of time, co-lead author Associate Professor Nic Rawlence, Director of the Otago Palaeogenetics Laboratory, says it is long enough to impact the species. "In that time the Rēkohu shelduck evolved shorter, more robust wings and longer leg bones indicating it was going down the pathway towards flightlessness," he says. Co-lead author Dr Pascale Lubbe, also of the Otago Palaeogenetics Laboratory, says in a case of "use it or lose it, the wings start to reduce." "Flight is energetically expensive, so if you don't have to fly, why bother," she says. "The longer leg bones are more robust to support more muscle and create increased force for take-off - necessary when you have smaller wings." The shelduck's scientific name (Tadorna rekohu) and common name (Rēkohu shelduck) were gifted to researchers by the Hokotehi Moriori Trust who are tchieki (guardians) of the plants and animals on Rēkohu Chatham Islands, with which they are interconnected through shared hokopapa (genealogy). Hokotehi Moriori Trust CEO Levi Lanauze says "this discovery is great for Rēkohu as a whole and helps connect imi (tribe) Moriori with miheke (treasure) of the past." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250817055324.htm'>NASA's PREFIRE satellites reveal a secret glow escaping from our planet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 04:05:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA's PREFIRE (Polar Radiant Energy in the Far-InfraRed Experiment) mission has been extended through September 2026 and is broadening its focus from Earth's poles to the entire globe. The mission's two shoebox-size CubeSats gauge the capacity of water vapor, clouds, and other elements of Earth's system to trap heat and keep it from radiating into space. This information can help improve forecasts, including weather severity and storm frequency. Winds, weather, and ocean currents transport that heat toward the poles, which receive much less sunlight. They measure wavelengths of light in the far-infrared range of the electromagnetic spectrum and are sensitive to 10 times more far-infrared wavelengths than any similar instrument. "The PREFIRE satellites show that at these longer wavelengths, the amount of radiation going into space can differ from one type of ice to another by as much as 5%," said Brian Drouin, PREFIRE's project scientist at JPL. "Measurements that look at the same areas but with shorter wavelengths do not show this difference." What we'll be able to do is look at the size of ice particles in clouds that affect energy exchange between Earth and space," said PREFIRE's principal investigator, Tristan L'Ecuyer of the University of Wisconsin-Madison. "We'll be able to incorporate the data into weather prediction models to improve forecasts and improve our understanding of how moisture circulates, which affects where storms form and how precipitation moves around the world." The satellites are in what's called an asynchronous near-polar orbit, traveling near the poles with each pass but hours apart from one another. The launch services provider, Rocket Lab USA Inc. of Long Beach, California, launched both PREFIRE CubeSats from Rocket Lab Launch Complex 1 in New Zealand in May and June 2024. Note: Content may be edited for style and length. Megaquake Triggers Rare Tsunami Caught by NASA's SWOT Satellite Webb Telescope Spots Oldest Black Hole, Shattering Cosmic Records Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250816113528.htm'>Scientists just found a hidden factor behind Earth's methane surge</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-18 03:29:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Tracking atmospheric methane to its specific sources and quantifying their importance remains a challenge, however. "It is well understood that methane levels are rising, but there is a lot of disagreement on the underlying cause," said co-author Dipti Nayak, UC Berkeley assistant professor of molecular and cell biology. Humans are about 99% carbon-12 and 1% carbon-13, which is slightly heavier because it has an extra neutron in its nucleus. The natural abundances of isotopes are reflected in all biologically produced molecules and variations can be used to study and fingerprint various biological metabolisms. "Over the last 70 years, people have shown that methane produced by different organisms and other processes can have distinctive isotopic fingerprints," said geochemist and co-author Daniel Stolper, UC Berkeley associate professor of earth and planetary science. "Natural gas from oil deposits often looks one way. Methane made by the methanogens within cow guts looks another way. Methane made in deep sea sediments by microorganisms has a different fingerprint. Methanogens can consume or 'eat', if you will, a variety of compounds including methanol, acetate or hydrogen; make methane; and generate energy from the process. "I think what's unique about the paper is, we learned that the isotopic composition of microbial methane isn't just based on what methanogens eat," Nayak said. "What you 'eat' matters, of course, but the amount of these substrates and the environmental conditions matter too, and perhaps more importantly, how microbes react to those changes." "This should cause us to think more carefully when we analyze data from the environment." They ingest simple molecules -- molecular hydrogen, acetate or methanol, for example -- excreted by other organisms and produce methane gas as waste. This natural methane can be observed in the pale Will-o'-the-wisps seen around swamps and marshes at night, but it's also released invisibly in cow burps, bubbles up from rice paddies and natural wetlands and leaks out of landfills. While most of the methane in the natural gas we burn formed in association with hydrocarbon generation, some deposits were originally produced by methanogens eating buried organic matter. Gropp had earlier created a computer model of the metabolic network in methanogens to understand better how the isotope composition of methane is determined. Stolper's laboratory specializes in measuring isotope compositions to explore Earth's history. They found that when MCR is at low concentrations, cells respond by altering the activity of many other enzymes in the cell, causing their inputs and outputs to accumulate and the rate of methane generation to slow so much that enzymes begin running both backwards and forwards. Each forward and reverse cycle requires one of these enzymes to pull a hydrogen off of the carbon and add a new one ultimately sourced from water. As a result, the isotopic composition of methane's four hydrogen molecules gradually comes to reflect that of the water, and not just their food source, which starts with three hydrogens. "This isotope exchange we found changes the fingerprint of methane generated by acetate and methanol consuming methanogens vs. that typically assumed. Given this, it might be that we have underestimated the contribution of the acetate-consuming microbes, and they might be even more dominant than we have thought," Gropp said. "We're proposing that we at least should consider the cellular response of methanogens to their environment when studying isotopic composition of methane." Beyond this study, the CRISPR technique for tuning production of enzymes in methanogens could be used to manipulate and study isotope effects in other enzyme networks broadly, which could help researchers answer questions about geobiology and the Earth's environment today and in the past. "This opens up a pathway where modern molecular biology is married with isotope-geochemistry to answer environmental problems," Stolper said. "There are an enormous number of isotopic systems associated with biology and biochemistry that are studied in the environment; I hope we can start looking at them in the way molecular biologists now are looking at these problems in people and other organisms -- by controlling gene expression and looking at how the stable isotopes respond." "By reducing the amount of this enzyme that makes methane and by putting in alternate pathways that the cell can use, we can essentially give them another release valve, if you will, to put those electrons, which they were otherwise putting in carbon to make methane, into something else that would be more useful," she said. Other co-authors of the paper are Markus Bill of Lawrence Berkeley National Laboratory and former UC Berkeley postdoc Rebekah Stein, and Max Lloyd, who is a professor at Penn State University. Gropp was supported by a fellowship from the European Molecular Biology Organization. Nayak and Stolper were funded, in part, by Alfred B. Sloan Research Fellowships. Nayak also is an investigator with the Chan-Zuckerberg Biohub. Note: Content may be edited for style and length. Megaquake Triggers Rare Tsunami Caught by NASA's SWOT Satellite Webb Telescope Spots Oldest Black Hole, Shattering Cosmic Records Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            