
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09757-7'>Search for light sterile neutrinos with two neutrino beams at MicroBooNE</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 16:35:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 648, pages 64–69 (2025)Cite this article The existence of three distinct neutrino flavours, νe, νμ and ντ, is a central tenet of the Standard Model of particle physics1,2. Quantum-mechanical interference can allow a neutrino of one initial flavour to be detected sometime later as a different flavour, a process called neutrino oscillation. Several anomalous observations inconsistent with this three-flavour picture have motivated the hypothesis that an additional neutrino state exists, which does not interact directly with matter, termed as ‘sterile' neutrino, νs (refs. 3,4,5,6,7,8,9). This includes anomalous observations from the Liquid Scintillator Neutrino Detector (LSND)3 experiment and Mini-Booster Neutrino Experiment (MiniBooNE)4,5, consistent with νμ → νe transitions at a distance inconsistent with the three-neutrino picture. Here we use data obtained from the MicroBooNE liquid-argon time projection chamber10 in two accelerator neutrino beams to exclude the single light sterile neutrino interpretation of the LSND and MiniBooNE anomalies at the 95% confidence level (CL). Moreover, we rule out a notable portion of the parameter space that could explain the gallium anomaly6,7,8. This is one of the first measurements to use two accelerator neutrino beams to break a degeneracy between νe appearance and disappearance, which would otherwise weaken the sensitivity to the sterile neutrino hypothesis. We find no evidence for either νμ → νe flavour transitions or νe disappearance that would indicate non-standard flavour oscillations. Our results indicate that previous anomalous observations consistent with νμ → νe transitions cannot be explained by introducing a single sterile neutrino state. A broad experimental programme has shown that the three quantum-mechanical eigenstates of neutrino flavour, νe, νμ and ντ, are related to the three eigenstates of neutrino mass, ν1, ν2 and ν3, by the unitary Pontecorvo–Maki–Nakagawa–Sakata (PMNS) matrix11,12. This mixing between flavour and mass states gives rise to the phenomenon of neutrino oscillation, in which neutrinos transition between flavour eigenstates with a characteristic wavelength in \(L/{E}_{\nu }\propto {(\Delta {m}_{ji}^{2})}^{-1}\), where L is the distance travelled by the neutrino, Eν is the neutrino energy and \(\Delta {m}_{ji}^{2}={m}_{j}^{2}-{m}_{i}^{2}\) is the difference between the squared masses of the mass eigenstates νi and νj. The three known neutrino mass states give rise to two independent mass-squared differences and thus to two characteristic oscillation frequencies that have been well measured with neutrinos from nuclear reactors13,14, the Sun15, the atmosphere of Earth16,17 and particle accelerators18,19,20. In apparent conflict with the three-neutrino model, several experiments during the past three decades have made observations that can be interpreted as neutrino flavour change with a wavelength much shorter than is possible given only the two measured mass-squared differences3,4,5,6,7,8,9. These observations are often explained as neutrino oscillations caused by at least one additional mass state, ν4, corresponding to a mass-squared splitting of \(\Delta {m}_{41}^{2}\gtrsim 1{0}^{-2}\,{{\rm{eV}}}^{2}\), which is much greater than the measured \(\Delta {m}_{21}^{2}\) and \(\Delta {m}_{32}^{2}\). New mass states would require the addition of an equivalent number of new flavour states, in conflict with measurements of the Z-boson decay width21, which have definitively shown that only three light neutrino flavour states couple to the Z boson of the weak interaction. Therefore, these additional neutrino flavour states must be unable to interact through the weak interaction and are thus referred to as ‘sterile' neutrinos. In this analysis, we focus specifically on light sterile neutrinos—those with masses below at least half the mass of the Z boson. It should be noted that the term ‘sterile neutrino' has also been used to describe new particles, such as heavy right-handed lepton partners, that are potentially more massive than the Z boson. However, our study does not directly test these scenarios. The discovery of additional neutrino states would have profound implications across particle physics and cosmology, for example, on our understanding of the origin of neutrino mass, the nature of dark matter and the number of relativistic degrees of freedom in the early universe. With the addition of a single new mass state ν4 and a single sterile flavour state νs, the PMNS matrix becomes a 4 × 4 unitary matrix described by six real mixing angles θij (1 ≤ i < j ≤ 4). Oscillations driven by the two measured mass-squared splittings have not had time to evolve for small values of L/Eν. The νμ to νe flavour-change probability, \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{e}}}}\), and the νe and νμ survival probabilities, \({P}_{{\nu }_{{\rm{e}}}\to {\nu }_{{\rm{e}}}}\) and \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{mu}}}}\), can then, to a very good approximation, be described by where θee ≡ θ14, \({\sin }^{2}(2{\theta }_{{\rm{\mu \mu }}})\equiv 4{\cos }^{2}{\theta }_{14}{\sin }^{2}{\theta }_{24}(1-{\cos }^{2}{\theta }_{14}{\sin }^{2}{\theta }_{24})\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\equiv {\sin }^{2}(2{\theta }_{14}){\sin }^{2}{\theta }_{24}\), following the common parameterization22. Flavour transitions due to these new oscillation parameters are experimentally probed by observing unexpected deficits or excesses in charged current (CC) νe and νμ interactions in a flavour-sensitive neutrino detector from a source of well-defined neutrino flavour content. Observations compatible with a fourth neutrino mass state have been made in measurements of intense electron-capture decay sources6,7,8, in which a deficit in detected νe rates implies non-unity \({P}_{{\nu }_{{\rm{e}}}\to {\nu }_{{\rm{e}}}}\) from a \(\Delta {m}_{41}^{2} > {\mathcal{O}}(1\,{{\rm{eV}}}^{2})\). Although a hint of non-unity \({P}_{{\overline{\nu }}_{{\rm{e}}}\to {\overline{\nu }}_{{\rm{e}}}}\) is provided by the nuclear-reactor-based Neutrino-4 experiment9, this result is in conflict with other reactor-based observations from DANSS, NEOS, PROSPECT and STEREO, which see no evidence for L/Eν-dependent \({\overline{\nu }}_{{\rm{e}}}\) disappearance23,24,25,26. Two accelerator-based experiments, LSND and MiniBooNE, have observed potential evidence of non-zero \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{e}}}}\) associated with large mass splittings of \(\Delta {m}_{41}^{2} > {\mathcal{O}}(1{0}^{-2}\,{{\rm{e}}{\rm{V}}}^{2})\). The LSND experiment observed an anomalous excess of \({\overline{\nu }}_{{\rm{e}}}\) interactions in a π+ decay-at-rest beam3. The MiniBooNE experiment, situated downstream from the Booster Neutrino Beam (BNB) proton target facility generating a beam of GeV-scale νμ and \({\overline{\nu }}_{{\rm{\mu }}}\) from decays of boosted π+ and π−, observed an excess of electromagnetic showers indicative of νe interactions that would imply a non-zero \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{e}}}}\) (refs. 4,5). Observations of νe disappearance and νe appearance should be accompanied by νμ disappearance (non-unity \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{\mu }}}}\)) if the PMNS matrix is unitary. No conclusive observation of this νμ disappearance has been reported27,28,29. The overall picture of the existence and phenomenology of sterile neutrino states thus remains inconclusive. In this article, we present new results on sterile neutrino oscillations from the MicroBooNE liquid-argon time projection chamber (LArTPC) experiment at Fermilab10. Situated along the same BNB beamline hosting the MiniBooNE experiment, MicroBooNE was conceived to directly test the non-zero \({P}_{{\nu }_{{\rm{\mu }}}\to {\nu }_{{\rm{e}}}}\) observation of MiniBooNE. By supplanting the Cherenkov detection technology of MiniBooNE with the precise imaging and calorimetric capabilities of a LArTPC, MicroBooNE can reduce backgrounds and select a high-purity sample of true νe-generated final-state electrons. The first νe measurement results of MicroBooNE using differing final-state topologies showed no evidence for an excess of νe-generated electrons from the BNB30,31,32,33. These results were used to set limits on νμ → νe flavour transitions, excluding sections of the region in \((\Delta {m}_{41}^{2},{\sin }^{2}(2{\theta }_{{\rm{\mu e}}}))\) space favoured by LSND and MiniBooNE data34. As the BNB has an intrinsic contamination of electron neutrinos, the disappearance of electron neutrinos can cancel the appearance of electron neutrinos from νμ → νe oscillations35. This effect leads to a degeneracy between the impact of the mixing angles θμe and θee of equations (1) and (2) that weakens the sensitivity to the parameters of the expanded 4 × 4 PMNS matrix. We overcome the limitations of the degeneracy between νe appearance and νe disappearance by performing one of the first oscillation searches using two accelerator neutrino beams: the BNB and the Neutrinos at the Main Injector (NuMI) beam. The MicroBooNE detector is aligned with the direction of BNB and is at an angle of about 8° relative to the NuMI beam. Beam timing information is used to distinguish and record events from each beam separately. This configuration results in two neutrino datasets differing in the intrinsic electron-flavour fraction. The electron-flavour content of BNB is 0.57% and that of the NuMI beam is 4.6%. These two independent sets of data, with substantially different electron-flavour contents, break the degeneracy between νe appearance and disappearance. We show the impact of using two beams in Fig. 1a,b, in which we compare simulated νe energy spectra from the BNB and the NuMI beam for the three-flavour (3ν) hypothesis and for two sets of parameters of the expanded four-flavour (4ν) PMNS model with \(\Delta {m}_{41}^{2}=1.2\,{{\rm{e}}{\rm{V}}}^{2}\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})=0.003\). For \({\sin }^{2}{\theta }_{24}=0.0045\) the νe appearance and disappearance cancel in the BNB, leaving a νe spectrum that is almost identical to the 3ν case, whereas the NuMI beam shows an indication of νe disappearance. The appearance and disappearance effects almost fully cancel in the NuMI beam for \({\sin }^{2}{\theta }_{24}=0.018\), whereas the BNB shows a clear indication of νe appearance. In the Methods, we provide further discussion of this degeneracy over a broader range of mass-squared splittings and mixing angles. a,b, Simulated reconstructed energy spectra of FC CC νe interactions in MicroBooNE from the BNB (a) and the NuMI beam (b). The light blue and red histograms show expectations for two sets of parameters of the 4ν model, both with \(\Delta {m}_{41}^{2}=1.2\,{{\rm{e}}{\rm{V}}}^{2}\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})=0.003\). Note that these parameters were chosen specifically to highlight differences in the oscillated spectra between BNB and NuMI and do not imply that parameter spaces associated with these values are newly excluded by this result. Using the two-beam technique, this new MicroBooNE analysis achieves marked improvements in sensitivity to the parameters \({\sin }^{2}(2{\theta }_{{\rm{ee}}})\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\) relative to MicroBooNE's prior sterile neutrino analysis over a broad range of \(\Delta {m}_{41}^{2}\) values. These improvements are shown by the sensitivities presented in Extended Data Fig. The results presented here using two neutrino beams place robust new constraints on the validity of the sterile neutrino hypothesis in explaining existing short-baseline anomalies in neutrino physics. This analysis strengthens the direct test of the sterile neutrino interpretation of the MiniBooNE anomaly and allows MicroBooNE to probe the \({\sin }^{2}(2{{\theta }}_{{\rm{\mu }}{\rm{e}}})\) parameter space favoured by LSND. We also constrain \({\sin }^{2}(2{\theta }_{{\rm{ee}}})\), complementing existing exclusions from reactor, solar36,37 and β-decay38 experiments, thereby further restricting the sterile neutrino parameter space relevant to the gallium anomaly. We use data corresponding to 6.369 × 1020 protons on target (POT) in the BNB, with magnetic van der Meer horns configured to focus positively charged hadrons, leading to a νμ-dominated beam with a 5.9% \({\overline{\nu }}_{{\rm{\mu }}}\) component and a 0.57% \({\nu }_{{\rm{e}}}+{\overline{\nu }}_{{\rm{e}}}\) component. From the NuMI beam, a total of 10.54 × 1020 POT are used, in which 30.8% were taken with horns configured to focus positively charged hadrons and the remainder with horns focusing negatively charged hadrons. The NuMI flux observed in the MicroBooNE detector, with both horn configurations combined, is νμ dominated with a 42.1% \({\overline{\nu }}_{{\rm{\mu }}}\) component and a 4.6% \({\nu }_{{\rm{e}}}+{\overline{\nu }}_{{\rm{e}}}\) component. In the rest of this paper, we do not discriminate between neutrinos and antineutrinos and refer to the \({\nu }_{{\rm{\mu }}}+{\overline{\nu }}_{{\rm{\mu }}}\) and \({\nu }_{{\rm{e}}}+{\overline{\nu }}_{{\rm{e}}}\) samples as νμ and νe samples for brevity. For both BNB and NuMI, the POT used in this analysis represent roughly half of the total data collected by the MicroBooNE detector; additional data remain available for future studies. The LArTPC detector of MicroBooNE has an active volume of 10.4 × 2.6 × 2.3 m3 containing 85 tonnes of liquid argon. Charged particles passing through the argon create ionization trails. A 273 V cm−1 electric field drifts the ionization electrons towards an anode plane consisting of three layers of wires separated by 3 mm and each with a 3-mm wire pitch that collects the electrons and enables three-dimensional imaging of the neutrino interactions. The passage of charged particles through the argon also produces scintillation light that is collected by a system of photomultiplier tubes to provide timing information. Signal processing and calibrations of MicroBooNE data are described in refs. Neutrino interactions in the LArTPC are reconstructed with the Wire-Cell analysis framework45. The techniques for identifying and reconstructing neutrino interactions and their energies have been described elsewhere33. We select a sample of CC νe interactions from the BNB (NuMI beam) with 82% (91%) purity and 46% (42%) efficiency, and a sample of CC νμ interactions with 92% (78%) purity and 68% (62%) efficiency. The CC νe and CC νμ samples are divided into fully contained (FC) and partially contained (PC) samples, depending on whether all charge depositions are contained in a fiducial volume 3 cm within the TPC boundary. The CC νμ events that contain a reconstructed π0 are separated into two additional FC and PC samples per beam. Neutral current (NC) interactions that produce a π0 are distinguished by the absence of a long muon-like track and the presence of detached reconstructed electromagnetic showers. These form an additional sample. In total, we define 14 distinct event categories, seven for each beam. We produce a Monte Carlo prediction of our 14 samples, to which we compare the data. There is substantial systematic uncertainty creating this Monte Carlo simulation. The uncertainty on the predicted rates of the 14 samples is given in Table 1 and is referred to as the unconstrained systematic uncertainty. The largest uncertainties come from neutrino interaction modelling for the BNB samples and from a combination of neutrino flux and interaction uncertainties for the NuMI samples. Many of these uncertainties are highly correlated. Thus, a combined fit of all samples effectively constrains the uncertainties on the CC νe prediction and at the same time allows the CC νe prediction to be modified, as can be seen from Table 1. The pionless samples constrain uncertainties on CC νe signal events, whereas the π0 samples constrain uncertainties on the dominant background. Uncertainties on the neutrino flux prediction arise from uncertainties in the production of charged pions and kaons in the BNB and NuMI targets and the material around the target halls and hadron-decay volumes. These uncertainties are evaluated through comparison with external hadron production data46,47,48, following a procedure similar to that described in ref. 49. The νe flux from three-body K and μ decays is highly correlated with the νμ flux from two-body π and K decays, allowing our νμ samples to effectively constrain the uncertainties on the νe flux predictions. The neutrino interaction model is tuned using datasets of pionless CC interactions from the T2K experiment50. Uncertainties on this neutrino interaction model are evaluated by varying the input parameters within their allowed uncertainties. These uncertainties are correlated between the BNB and NuMI datasets and between the CC νμ and νe samples because of the lepton universality of the weak interaction. Uncertainties on the simulation of the detector include uncertainties on the response of the detector to ionization, uncertainties on the amount of ionization charge freed by passing charged particles through the detector, uncertainties on the electric field map of the TPC, uncertainties on the production and propagation of scintillation light, uncertainties on backgrounds from interactions occurring outside the cryostat, and uncertainties on finite statistics of the simulation samples used for predictions. The simultaneous fit to the 14 samples from the BNB and the NuMI beam incorporates all sources of systematic uncertainty through a covariance matrix. We allow \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\), \({\sin }^{2}(2{\theta }_{{\rm{ee}}})\) and \(\Delta {m}_{41}^{2}\) complete freedom within unitarity bounds as parameters of the fit. The constrained predictions shown in Fig. 2 assume the 3ν hypothesis of \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})={\sin }^{2}(2{\theta }_{{\rm{ee}}})=0\). They agree well with the data, with a P-value of 0.92. The best-fit values for the oscillation parameters in the 4ν hypothesis are \(\Delta {m}_{41}^{2}=1.30\times 1{0}^{-2}\,{{\rm{e}}{\rm{V}}}^{2}\), \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})=0.999\), and \({\sin }^{2}(2{\theta }_{{\rm{ee}}})=0.999\), with a χ2 difference with respect to the 3ν hypothesis of We observe no marked preference for the existence of a sterile neutrino with a P-value of 0.96 evaluated using the Feldman–Cousins procedure. a–d, Reconstructed energy spectra of events selected as FC CC νe candidates in the BNB (a), PC CC νe candidates in the BNB (b), FC νe candidates in the NuMI beam (c) and PC νe candidates in the NuMI beam (d). The data points are shown with statistical error bars. The background category contains CC νμ interactions, NC neutrino interactions, cosmic rays and interactions occurring outside the fiducial volume of the detector. Exclusion contours are calculated using the frequentist CLs (confidence level as a function of s) method51. The exclusion contour in any two-dimensional parameter space is obtained by profiling the third free parameter. Figure 3a shows the 95% CLs exclusion contour in the \((\Delta {m}_{41}^{2},{\sin }^{2}(2{\theta }_{{\rm{\mu e}}}))\) parameter space. Figure 3b shows the 95% CLs exclusion contour in the \((\Delta {m}_{41}^{2},{\sin }^{2}(2{\theta }_{{\rm{ee}}}))\) parameter space. A notable portion of the region allowed by gallium measurements and part of the region derived from the Neutrino-4 measurement are excluded. In the Methods and Extended Data Fig. 2, we compare our exclusions with the expected median sensitivities. a,b, The red lines show exclusion limits at the 95% CLs level in the plane of \(\Delta {m}_{41}^{2}\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu }}{\rm{e}}})\) (a) or \({\sin }^{2}(2{\theta }_{{\rm{e}}{\rm{e}}})\) (b). All the regions to the right of these lines are excluded by the MicroBooNE data. In a, the yellow shaded area is the LSND 99% CL allowed regions3, which neglects the degeneracy between νe disappearance and appearance. The light blue area is the MiniBooNE 95% CL allowed region58, considering both νe disappearance and appearance. For context, note that the stronger-than-expected constraint on \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\), driven by the deficit observed in the BNB νe CC FC sample and the excess in the NuMI νμ CC sample, is discussed in detail in the Methods and Extended Data Fig. In summary, using data from the MicroBooNE detector, we report one of the first searches for a sterile neutrino using two accelerator neutrino beams. The oscillation fit to the 4ν model using a total of 14 CC νe, CC νμ and NC π0 samples from the BNB and the NuMI beam in a single detector achieves a marked reduction of systematic uncertainties and a powerful mitigation of degeneracies between νe appearance and disappearance. The result shows no evidence of oscillations induced by a single sterile neutrino and is consistent with the 3ν hypothesis with a P-value of 0.96. We comprehensively exclude at a 95% CL the 4ν parameter space that would explain the LSND and MiniBooNE anomalies through the existence of a light sterile neutrino in a model with an extended 4 × 4 PMNS matrix. Our result expands the diverse range of experimental approaches, excluding regions that would explain the gallium anomaly and the Neutrino-4 observation with a light sterile neutrino. This work, therefore, provides a robust exclusion of a single light sterile neutrino as an explanation for the array of short-baseline neutrino anomalies observed over the past three decades, representing the strongest constraint from a short-baseline experiment using accelerator-produced neutrinos. Expanded models, including several light sterile neutrinos52, neutrino decay effects53,54 or production and decay of new particles connected with the dark sector55,56 might explain the anomalies. The Short Baseline Neutrino (SBN) Programme57 at Fermilab adds two new LArTPC detectors in the BNB, at different distances from the proton target. Future measurements by MicroBooNE and the broader SBN Programme can shed light on this expanded model space, with future comprehensive insights provided by near-term short-baseline measurements from diverse flavour channels and energy regimes. The BNB and the NuMI are conventional neutrino beamlines that use intense proton beam pulses to generate on-axis neutrino fluxes peaking in the neutrino-energy range 0.5–5 GeV. Boosted charged mesons escaping the target are focused into a decay pipe using magnetic horns, allowing decaying mesons to impart much of their kinetic energy directly to their neutrino product. The detector is about 8° off-axis and 679 m from the proton target of the NuMI beam. The muon flavour components of these beams are mostly generated via the primary decay channel of the dominant π mesons, \({\pi }^{\pm }\to {\mu }^{\pm }+{\nu }_{{\rm{\mu }}}({\overline{\nu }}_{{\rm{\mu }}})\), whereas the electron-flavour component is generated by decay of the pion's boosted μ± daughter, by \({\mu }^{\pm }\to {e}^{\pm }+{\overline{\nu }}_{{\rm{\mu }}}({\nu }_{{\rm{\mu }}})+{\nu }_{{\rm{e}}}({\overline{\nu }}_{{\rm{e}}})\), and by semileptonic decays of sub-dominant K mesons, specifically \({K}^{\pm }\to {\pi }^{0}+{e}^{\pm }+{\nu }_{{\rm{e}}}({\overline{\nu }}_{{\rm{e}}})\) and \({K}_{{\rm{L}}}^{{\rm{0}}}\to {\pi }^{\mp }+{e}^{\pm }+{\nu }_{{\rm{e}}}({\overline{\nu }}_{{\rm{e}}})\). Neutrinos in the BNB are created by colliding protons with a kinetic energy of 8 GeV on a beryllium target, whereas in the NuMI beamline, 120 GeV protons collide with a carbon target. These differences serve to generate a higher on-axis beam energy in the NuMI beam, as well as a greater proportion of K meson production, leading to a higher νe content at the highly off-axis position of MicroBooNE. The NuMI beam also incorporates a longer charged-meson decay pipe (675 m) than the BNB (50 m), which increases its electron-flavour content by facilitating a higher proportion of decays of secondary μ+ from upstream π+ decay. Although the latter effect drives higher electron-flavour content on-axis for NuMI relative to the BNB, it is the larger proportion of unfocused or poorly focused K mesons of NuMI that drives its elevated electron-flavour content at the off-axis angle of MicroBooNE relative to the on-axis flux of BNB. Hadron production cross-sections are constrained by dedicated external measurements where available, tailored to the specific beam parameters, including target differences and initial proton beam energy. The BNB simulation, identical to that used at MiniBooNE61, uses Geant v.4.10.4 with a custom physics list and constrains π± yields with data from the HARP experiment62, along with an updated K+ production constraint from SciBooNE63,64. The NuMI simulation has been updated to Geant v.4.10.4 with the FTFP-BERT physics list65. The constraints on π± and K± yields from the NA49 experiment at CERN46,47 are implemented using the PPFX toolkit49, which has been updated to use the new Geant version. Uncertainties are estimated for each process and various components of the beamline geometry, resulting in a combined systematic uncertainty of approximately 13% for the BNB and 26% for the NuMI beam on the integrated flux. These uncertainties on the fluxes are different from the uncertainties quoted in Table 1, which are on the overall event rates. These uncertainties are dominated by hadron production rates and are larger for the NuMI beam because of the lack of constraints at large off-axis angles from dedicated hadron production experiments. As the nature of neutrino production in the two beams is different, the uncertainties are considered uncorrelated across the respective beams. The interplay between oscillation of intrinsic νμ and νe components in the BNB and the NuMI beam is shown in Extended Data Fig. For various combinations of the expanded 4ν PMNS mixing angles, and assuming \(\Delta {m}_{41}^{2}=1.4\,{{\rm{e}}{\rm{V}}}^{2}\), the ratio of predicted νe signal events with 0 < Eν < 2.5 GeV in MicroBooNE relative to the 3ν prediction is shown for the BNB on the x-axis and for the NuMI beam on the y-axis. By tracing vertically along x = 1, we observe that a MicroBooNE BNB νe measurement could be consistent with the 3ν hypothesis of θee = θμe = 0 as well as with non-zero mixing angles in the alternate 4ν case. The addition of a NuMI νe measurement enables a much clearer interpretation of the allowed oscillation behaviour, while also strengthening the constraining power of the analysis. Specifically, perfect agreement between data and the 3ν prediction for both BNB and NuMI would favour the null oscillation case, whereas a large deficit in the high νe-content NuMI beam would indicate competing appearance and disappearance effects in the BNB νe sample. Moreover, we can see that the range of allowed 4ν predictions in NuMI and BNB when taken together is quite restricted, allowing us to set tighter limits on this oscillation model based on the observed CC νe event rates after constraints in Table 1 (x ≈ 0.9, y ≈ 1). The dashed lines show the exclusion regions of the previous BNB-only analysis of MicroBooNE34, compared with the solid red lines, which show the exclusions obtained by this analysis when including the NuMI beam data. The combined statistical and systematic uncertainties on the 14 event samples are described by the covariance matrix where the Ci,j are the bin-by-bin covariance matrices between the ith and jth event samples. These covariance matrices are the sums of the covariance matrices arising from statistical uncertainties and from each source of systematic uncertainty, where the sum runs over the k sources of systematic uncertainty. To produce the constrained predictions shown in Fig. 2 and the constrained systematic uncertainties in Table 1, a conditional constraint formalism66 is used, which uses the 14 event samples simultaneously to constrain the systematic uncertainties and to provide updated predictions for each event sample. To understand how the constraint is applied to the ith event sample, the full unconstrained covariance matrix can be written as where elements with a subscript x represent the remaining 13 blocks of the full matrix. An updated, constrained covariance matrix for the ith event sample is obtained as Given the unconstrained binned prediction for the ith event sample, μi, and the remaining binned prediction and data samples, μx and nx, a constrained binned prediction can also be formed as To perform the oscillation fit, a χ2 test statistic, is formed using the 14 binned event samples from the data, N = (n1, …, n14), where ni is the ith event sample from the data, and the corresponding unconstrained binned predictions for the 14 event samples, M = (μ1, …, μ14). The oscillation parameters used to produce the prediction are varied until χ2 is minimized. As χ2 is minimized, the absolute systematic uncertainty varies as the number of oscillated neutrino interactions within the fiducial volume changes, whereas the fractional systematic uncertainty remains constant. By contrast, the absolute systematic uncertainty related to non-neutrino backgrounds and out-of-fiducial-volume neutrino interactions remains unchanged. Consequently, the total covariance matrix is updated in accordance with the oscillation parameters of interest. Exclusion limits on the oscillation parameters are calculated using the frequentist-motivated CLs method51, which is commonly used for determining exclusion limits in high-energy physics. The CLs test statistic is defined as CLs = P4ν/P3ν, where P4ν and P3ν are the one-sided P-values of \(\Delta {\chi }_{{\rm{C}}{{\rm{L}}}_{{\rm{s}}},{\rm{d}}{\rm{a}}{\rm{t}}{\rm{a}}}^{2}\) under the 4ν and the null 3ν hypotheses, respectively, where at a given point in the 4ν parameter space. The P-values are determined using a frequentist approach by generating pseudo-experiments with the full covariance matrix, assuming the respective hypothesis is true. The region in which CLs ≤ 1 − α is excluded at the confidence level α. By generating pseudo-experiments under the null 3ν hypothesis, expected exclusion limits are calculated across the 2D parameter spaces of \(\Delta {m}_{41}^{2}\) and \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\) or \({\sin }^{2}(2{\theta }_{{\rm{ee}}})\). To quantify the expected sensitivity of the analysis, the median \({\sin }^{2}(2{\theta }_{{\rm{\mu e(ee)}}})\) value from all expected exclusion limits is determined for each \(\Delta {m}_{41}^{2}\) value. These median sensitivities are shown in Extended Data Fig. 2 and are compared with the exclusions set using the data. To show the expected level of fluctuations of the measured limit from the median sensitivity, 1σ and 2σ bands are also shown in Extended Data Fig. 2, which encompass the central 68.3% and 95.5% of exclusions from the pseudo-experiments. In \({\sin }^{2}(2{\theta }_{{\rm{\mu e}}})\) space, our exclusion is stronger than our median expected sensitivity. Two factors contribute to this stronger exclusion. First, a deficit in the BNB CC νe sample more strongly disfavours νe appearance. Second, the excess in the NuMI CC νμ sample leads to a reduction in the constrained fractional uncertainty on the NuMI νe prediction through the joint fit procedure, which in turn further strengthens the exclusion limit. In \({\sin }^{2}(2{\theta }_{{\rm{ee}}})\) space, the deficit in the BNB CC νe FC sample plays the opposite role, slightly favouring νe disappearance and making the exclusion contour weaker than the median sensitivity. As shown in Table 1, a combined fit of the 14 reconstructed samples constrains the signal CC νe prediction and its uncertainties due to the correlations between the sideband and the signal channels. For the NuMI CC νe signal sample in particular, a crucial driver of the constraint is the corresponding CC νμ sideband, shown in Extended Data Fig. Before the fit, the normalization difference between data and the prediction is 24.5% with an overall uncertainty of 21.1% on the prediction. To evaluate the impact of this difference on the combined fit, we can extend the covariance matrix used in the analysis (364 × 364, corresponding to the energy spectra of the 14 channels) by adding a bin representing the overall NuMI CC νμ normalization (combining FC and PC) and computing the respective covariances with the other 364 analysis bins, resulting in a 365 × 365 matrix. We can then obtain a post-fit mean and error on this normalization parameter by constraining the 364 × 364 block to the data using equations (8) and (9). This gives us an estimate of how much this parameter is effectively being pulled in the combined fit. The post-fit value of this parameter is 1.28 ± 0.058, indicating consistency with the corresponding observed value as well as a modest pull of  about 1.3σ. The measured data, predicted signal and background, along with their complete systematic uncertainties for the corresponding reconstructed neutrino energy bins in the νe channels, are publicly accessible on HEPData (https://doi.org/10.17182/hepdata.166435.v1) and Zenodo (https://doi.org/10.5281/zenodo.17161263). Moreover, Δχ2 values for each 4ν hypothesis across the three-dimensional grid of oscillation parameters are provided. The MicroBooNE Collaboration is responsible for developing and maintaining the code used for simulating and analysing the raw data that support this result. This code is accessible to the Collaboration members but is not publicly available. Questions about the algorithms and methods used in this analysis can be directed to the corresponding author. Glashow, S. L. Partial symmetries of weak interactions. Weinberg, S. A model of leptons. Evidence for neutrino oscillations from the observation of \({\overline{\nu }}_{e}\) appearance in a \({\overline{\nu }}_{e}\) beam. Improved search for \({\overline{\nu }}_{\mu }\to {\overline{\nu }}_{e}\) oscillations in the MiniBooNE experiment. Updated MiniBooNE neutrino oscillation results with increased data and new background studies. Kaether, F., Hampel, W., Heusser, G., Kiko, J. & Kirsten, T. Reanalysis of the GALLEX solar neutrino flux and source experiments. Measurement of the solar neutrino capture rate with gallium metal. III: results for the 2002–2007 data-taking period. Barinov, V. V. et al. A search for electron neutrino transitions to sterile states in the BEST experiment. Serebrov, A. P. et al. Search for sterile neutrinos with the Neutrino-4 experiment and measurement results. Acciarri, R. et al. Design and construction of the MicroBooNE detector. Pontecorvo, B. Neutrino experiments and the problem of conservation of leptonic charge. Maki, Z., Nakagawa, M. & Sakata, S. Remarks on the unified model of elementary particles. Precision measurement of neutrino oscillation parameters with KamLAND. Precision measurement of reactor antineutrino oscillation at kilometer-scale baselines by Daya Bay. Combined analysis of all three phases of solar neutrino data from the Sudbury Neutrino Observatory. Evidence for oscillation of atmospheric neutrinos. Measurement of atmospheric neutrino oscillation parameters using convolutional neural networks with 9.3 years of data in IceCube DeepCore. Precision constraints for three-flavor neutrino oscillations from the Full MINOS+ and MINOS dataset. Improved measurement of neutrino oscillation parameters by the NOvA experiment. Abe, K. et al. Measurements of neutrino oscillation parameters from the T2K experiment using 3.6 × 1021 protons on target. The ALEPH Collaboration Precision electroweak measurements on the Z resonance. Giunti, C. & Lasserre, T. eV-scale sterile neutrinos. Danilov, M. & Skrobova, N. New results from the DANSS experiment. Atif, Z. et al. Search for sterile neutrino oscillations using RENO and NEOS data. Final search for short-baseline neutrino oscillations with the PROSPECT-I detector at HFIR. Improved sterile neutrino constraints from the STEREO experiment with 179 days of reactor-on data. Adamson, P. et al. Search for sterile neutrinos in MINOS and MINOS+ using a two-detector fit. Abbasi, R. et al. A search for an eV-scale sterile neutrino using improved high-energy νμ event reconstruction in IceCube. Acero, M. A. et al. Dual-baseline search for active-to-sterile neutrino oscillations in NOVA. Abratenko, P. et al. Search for an excess of electron neutrino interactions in MicroBooNE using multiple final-state topologies. Abratenko, P. et al. Search for an anomalous excess of charged-current quasielastic νe interactions with the MicroBooNE experiment using deep-learning-based reconstruction. Abratenko, P. et al. Search for an anomalous excess of charged-current νe interactions without pions in the final state with the MicroBooNE experiment. Abratenko, P. et al. Search for an anomalous excess of inclusive charged-current νe interactions in the MicroBooNE experiment using wire-cell reconstruction. Abratenko, P. et al. First constraints on light sterile neutrino oscillations from combined appearance and disappearance searches with the MicroBooNE detector. Argüelles, C. A. et al. MicroBooNE and the νe interpretation of the MiniBooNE low-energy excess. Goldhagen, K., Maltoni, M., Reichard, S. E. & Schwetz, T. Testing sterile neutrino mixing with present and future solar neutrino data. Berryman, J. M., Coloma, P., Huber, P., Schwetz, T. & Zhou, A. Statistical significance of the sterile-neutrino hypothesis in the context of reactor and gallium data. Improved eV-scale sterile-neutrino constraints from the second KATRIN measurement campaign. Noise characterization and filtering in the MicroBooNE liquid argon TPC. Adams, C. et al. Ionization electron signal processing in single phase LArTPCs. Part I. Algorithm description and quantitative evaluation with MicroBooNE simulation. Adams, C. et al. Ionization electron signal processing in single phase LArTPCs. Data/simulation comparison and performance in MicroBooNE. Adams, C. et al. Calibration of the charge and energy loss per unit length of the MicroBooNE liquid argon time projection chamber using muons and protons. Adams, C. et al. A method to determine the electric field of liquid argon time projection chambers using a UV laser system and its application in MicroBooNE. Measurement of space charge effects in the MicroBooNE LArTPC using cosmic muons. Abratenko, P. et al. Wire-cell 3D pattern recognition techniques for neutrino event reconstruction in large LArTPCs: algorithm description and quantitative evaluation with MicroBooNE simulation. Tinti, G. M. Sterile neutrino oscillations in MINOS and hadron production in pC collisions. Alt, C. et al. Inclusive production of charged pions in p+C collisions at 158-GeV/c beam momentum. Measurements of π±, K±, \({{\rm{K}}}_{{\rm{S}}}^{0}\), Λ and proton production in proton–carbon interactions at 31 GeV/c with the NA61/SHINE spectrometer at the CERN SPS. Aliaga, L. et al. Neutrino flux predictions for the NuMI beam. Abratenko, P. et al. New CC0π GENIE model tune for MicroBooNE. Read, A. L. Presentation of search results: the CLs technique. Fong, C. S., Minakata, H. & Nunokawa, H. Non-unitary evolution of neutrinos in matter and the leptonic unitarity test. de Gouvêa, A., Peres, O. L. G., Prakash, S. & Stenico, G. V. On the decaying-sterile neutrino solution to the electron (anti)neutrino appearance anomalies. & Zhou, T. Decaying sterile neutrinos at short baselines. Explaining the MiniBooNE anomalous excess via a leptophilic ALP-sterile neutrino coupling. Dutta, B., Kim, D., Thompson, A., Thornton, R. T. & Van de Water, R. G. Solutions to the MiniBooNE anomaly from new physics in charged meson decays. & Schmitz, D. W. The short-baseline neutrino program at Fermilab. A. et al. MiniBooNE and MicroBooNE combined fit to a 3 + 1 sterile neutrino scenario. Results from the Baksan Experiment on Sterile Transitions (BEST). Agostinelli, S. et al. Geant4—a simulation toolkit. A. et al. Neutrino flux prediction at MiniBooNE. Measurement of the production cross-section of positive pions in the collision of 8.9-GeV/c protons on beryllium. Measurement of K+ production cross section by 8 GeV protons using high energy neutrino interactions in the SciBooNE detector. Mariani, C., Cheng, G., Conrad, J. M. & Shaevitz, M. H. Improved parameterization of K+ production in p-Be collisions at low energy using Feynman scaling. Bertini, H. W. Intranuclear-cascade calculation of the secondary nucleon spectra from nucleon-nucleus interactions in the energy range 340 to 2900 MeV and comparisons with experiment. Eaton, M. L. Multivariate Statistics: A Vector Space Approach (Wiley, 1983). This document was prepared by the MicroBooNE Collaboration using the resources of the Fermi National Accelerator Laboratory (Fermilab), a US Department of Energy, Office of Science, HEP User Facility. Fermilab is managed by Fermi Research Alliance (FRA), acting under contract no. MicroBooNE is supported by the following: the US Department of Energy, Office of Science, Offices of High Energy Physics and Nuclear Physics; the US National Science Foundation; the Swiss National Science Foundation; the Science and Technology Facilities Council (STFC), part of the UK Research and Innovation; the Royal Society (the United Kingdom); the UK Research and Innovation (UKRI) Future Leaders Fellowship; and the NSF AI Institute for Artificial Intelligence and Fundamental Interactions. Additional support for the laser calibration system and cosmic ray tagger was provided by the Albert Einstein Center for Fundamental Physics, Bern, Switzerland. We also acknowledge the contributions of technical and scientific staff to the design, construction and operation of the MicroBooNE detector as well as the contributions of past collaborators to the development of MicroBooNE analyses, without whom this work would not have been possible. All transfer of copyright agreements must be signed by the Fermilab Technical Publications Office, including Creative Commons Licenses (CC-BY). This is required by our contract with DOE and gives us the right to make the information contained in the Scientific/Technical Publication freely available on the web pages of Fermilab Technical Publications without breaking US copyright laws. Department of Physics & Astronomy, Tufts University, Medford, MA, USA P. Abratenko, J. Micallef, M. Rosenberg & T. Wongjirad Department of Physics, Illinois Institute of Technology (IIT), Chicago, IL, USA D. Andrade Aldana, O. Benevides Rodrigues, W. Foreman & B. R. Littlejohn Department of Physics & Astronomy, The University of Manchester, Manchester, UK L. Arellano, J. Bateman, J. J. Evans, E. Gramellini, R. Guenette, P. Guzowski, N. Lane, D. Marsden, L. Mora Lepin, M. M. Moudgalya, C. Thorpe & A. Trettin School of Physics & Astronomy, Tel Aviv University, Tel Aviv, Israel Fermi National Accelerator Laboratory (FNAL), Batavia, IL, USA S. Balasubramanian, B. Baller, V. Basque, M. Bhattacharya, F. Cavanna, G. Cerati, M. Del Tutto, A. Ereditato, S. Gardiner, H. Greenlee, C. James, W. Ketchum, T. Kobilarcik, L. Liu, C. D. Moore, O. Palamara, Z. Pavlovic, J. L. Raaf, A. Schukraft, R. Sharankova, E. L. Snider, M. Stancari, J. St. John, T. Strauss, M. Toups, S. Wolbers, T. Yang, L. E. Yates, G. P. Zeller & J. Zennamo A. Barnard, G. Barr, D. Barrow, K. Duffy & P. Green School of Physics & Astronomy, University of Minnesota, Minneapolis, MN, USA J. Barrow, A. P. Furmanski, C. Hilgenberg, B. Irwin & N. Pallat J. Bateman, N. Lane, A. Navrer-Agasson & S. Söldner-Rembold Department of Physics & Astronomy, Michigan State University, East Lansing, MI, USA Department of Physics, University of Chicago, Chicago, IL, USA A. Bhat, A. Ereditato, B. T. Fleming, D. Franco, L. Hagaman, D. W. Schmitz & A. J. Brookhaven National Laboratory (BNL), Upton, NY, USA M. Bishai, W. Gu, J. H. Jo, M. Kirby, Y. Li, S. Martynenko, N. Nayak, X. Qian, V. Radeka, D. Torbunov, B. Viren, H. W. Yu & C. Zhang A. Blake, L. Gu, T. Mahmud, I. Mawby, J. Nowak, N. Patel & I. Pophale Department of Physics, University of Michigan, Ann Arbor, MI, USA Department of Physics, Kansas State University (KSU), Manhattan, KS, USA T. Bolton, G. A. Horton-Smith, A. Hussain, N. Martinez & J. Tyler Department of Physics & Astronomy, The University of Kansas, Lawrence, KS, USA M. B. Brunetti, A. Chappell, R. Cross & J. Marshall Department of Physics, Columbia University, New York, NY, USA Jwa, D. Kalra, G. Karagiorgi, N. Oza, I. Safa, W. Seligman & M. H. Shaevitz Department of Physics, University of California, Santa Barbara, CA, USA D. Caratelli, C. Fang, F. Gao, X. Luo, D. Totani & E. Yandel SLAC National Accelerator Laboratory, Menlo Park, CA, USA Y. Chen, M. Convery, K. Terao, Y.-T. Tsai & T. Usher Department of Physics, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA Department of Physics & Astronomy, University of Pittsburgh, Pittsburgh, PA, USA L. Cooper-Troendle, S. Dytman, M. S. Ismail, D. Naples, V. Paolone & W. Wu Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas (CIEMAT), Madrid, Spain S. R. Dennis, P. Detje, M. D. Handley, A. F. Moor, J. Shi, N. Taniuchi, M. A. Uchida & K. Wresilo Argonne National Laboratory (ANL), Lemont, IL, USA Department of Physics, University of Southern Maine, Portland, ME, USA Department of Physics & Astronomy, Rutgers University, Piscataway, NJ, USA P. Englezos, K. Lin, A. Mastbaum & C. Nguyen Los Alamos National Laboratory (LANL), Los Alamos, NM, USA W. Foreman, S. Gollapinni, W. C. Louis, M. Ross-Lonergan & E. Yandel Departamento de Física Teórica y del Cosmos, Universidad de Granada, Granada, Spain School of Physics, Nankai University, Tianjin, China Department of Physics, University of Cincinnati, Cincinnati, OH, USA School of Physics & Astronomy, University of Edinburgh, Edinburgh, United Kingdom Li, M. Nebot-Guinot, H. B. Parkinson, M. Reggiani-Guzzo & A. M. Szelc Center for Neutrino Physics, Virginia Tech, Blacksburg, VA, USA Department of Physics, South Dakota School of Mines and Technology (SDSMT), Rapid City, SD, USA D. A. Martinez Caicedo & J. Rodriguez Rondon Department of Physics & Astronomy, Queen Mary University of London, London, UK Department of Physics & Astronomy, Louisiana State University, Baton Rouge, LA, USA Department of Physics, Colorado State University, Fort Collins, CO, USA Department of Physics, Indiana University, Bloomington, IN, USA Department of Physics, New Mexico State University (NMSU), Las Cruces, NM, USA The MicroBooNE Collaboration contributed collectively to this publication through the design, construction and installation of the detector, its operation and data acquisition, and the development of simulation and analysis tools. The scientific results presented here were reviewed and approved by the entire Collaboration. The authors declare no competing interests. Nature thanks Thomas Schwetz-Mangold, Alexandre Sousa and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The lines indicate how these ratios depend on the oscillation parameters of the expanded 4ν PMNS matrix for \(\Delta {m}_{41}^{2}=1.4\,{{\rm{e}}{\rm{V}}}^{2}\). The red solid lines show the measured exclusion lines at the 95% CLs level in the plane of \(\Delta {m}_{41}^{2}\) and (a) \({\sin }^{2}(2{\theta }_{{\rm{\mu }}{\rm{e}}})\) or (b) \({\sin }^{2}(2{\theta }_{{\rm{e}}{\rm{e}}})\). The blue dashed lines show the median expected sensitivities. The green and yellow bands show the 1σ and 2σ expected fluctuations around the median sensitivities at each \(\Delta {m}_{41}^{2}\) value. The red dashed lines show the previous MicroBooNE BNB-only result34 and the blue solid lines shows the measured NuMI-only exclusions at 95% CLs. Reconstructed energy spectra of events selected as (a) fully contained CC νμ candidates in the NuMI beam, and (b) partially contained CC νμ candidates in the NuMI beam. The data points are shown with statistical error bars. The background category contains NC neutrino interactions, cosmic rays, CC νe interactions, and interactions occurring outside the fiducial volume of the detector. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Search for light sterile neutrinos with two neutrino beams at MicroBooNE. Version of record: 03 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251203004721.htm'>A routine shingles shot may offer powerful defense against dementia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 15:23:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An unusual vaccination rule in Wales has given scientists some of the clearest evidence so far that a vaccine might help protect against dementia. In a new study led by Stanford Medicine, researchers examined health records from older adults in Wales and found that people who received the shingles vaccine were 20% less likely to be diagnosed with dementia over the following seven years than those who did not receive the shot. If these findings continue to hold up in future work, they suggest that a practical way to help prevent dementia could already exist. A second analysis from the same team, published Dec. 2 in Cell, pointed to another potential benefit. It is triggered by the same virus that leads to chicken pox -- varicella-zoster. Instead, it stays inactive inside nerve cells for life. For many years, most dementia research has centered on abnormal protein build-up in the brain, including plaques and tangles that are characteristic of Alzheimer's disease, the most common type of dementia. However, these efforts have not yet produced successful ways to prevent or halt the disease, so some scientists have turned their attention to other possible drivers, including infections by specific viruses that may damage the brain over time. Earlier observational studies using health records had hinted that people who received the shingles vaccine were less likely to develop dementia. "All these associational studies suffer from the basic problem that people who go get vaccinated have different health behaviors than those who don't," said Pascal Geldsetzer, MD, PhD, assistant professor of medicine and senior author of the new study. "In general, they're seen as not being solid enough evidence to make any recommendations on." About two years ago, Geldsetzer noticed an unusual opportunity in the way Wales had rolled out its shingles vaccination program. The setup functioned as what researchers call a "natural experiment," and it appeared to avoid much of the bias seen in earlier work. Because eligibility depended only on age at a specific cut-off date, the difference between being just under or just over the age threshold had a major impact on who could get the shot. That allowed researchers to compare people who turned 80 shortly before Sept. 1, 2013, with those who turned 80 shortly after, and to see how being eligible for the vaccine changed long-term outcomes. According to Geldsetzer, the detailed health records available in Wales made these circumstances about as close as possible to a randomized controlled trial without actually running one. They then concentrated their analysis on people whose birthdays placed them just on either side of the eligibility line, comparing those who turned 80 in the week before Sept. 1 to those who turned 80 in the week after. "We know that if you take a thousand people at random born in one week and a thousand people at random, born a week later, there shouldn't be anything different about them on average," Geldsetzer said. "They are similar to each other apart from this tiny difference in age." "What makes the study so powerful is that it's essentially like a randomized trial with a control group -- those a little bit too old to be eligible for the vaccine -- and an intervention group -- those just young enough to be eligible," Geldsetzer said. About half of the people who were eligible went on to be vaccinated, while almost none of those who were not eligible received it. By 2020, when the individuals being studied were around 86 and 87 years old, one in eight had developed dementia. Among those who received the shingles shot, however, the likelihood of a dementia diagnosis was 20% lower compared with those who did not receive it. "It was a really striking finding," Geldsetzer said. "This huge protective signal was there, any which way you looked at the data." Education levels were the same for eligible and ineligible people. Those who were eligible for the shingles vaccine were not more likely to receive other vaccines or preventive therapies, and they were not less likely to have common illnesses such as diabetes, heart disease or cancer. Even so, the team tested the data in a variety of alternative ways, such as examining different age windows or focusing only on deaths that listed dementia as a cause. Using the same natural experiment structure, they examined a broader range of outcomes, from mild cognitive changes to late-stage dementia. Many cases of dementia are preceded by a period of mild cognitive impairment -- characterized by deficits in memory and cognitive skills that do not interfere with independent living, Geldsetzer said. The team observed that people who had received the shingles vaccine were less likely to receive a diagnosis of mild cognitive impairment during a nine-year follow-up period than those who remained unvaccinated. In this group, the results were especially striking. In total, nearly half of the 7,049 Welsh seniors who had dementia when the program began died from dementia during follow-up. "The most exciting part is that this really suggests the shingles vaccine doesn't have only preventive, delaying benefits for dementia, but also therapeutic potential for those who already have dementia," Geldsetzer said. Another notable pattern emerged when the researchers compared outcomes by sex. On average, women tend to develop higher antibody responses after vaccination, and shingles occurs more often in women than in men. At this point, scientists still do not know exactly how the vaccine might be providing protection. It is not yet clear whether it works by broadly stimulating the immune system, by reducing how often the varicella-zoster virus reactivates, or through another pathway entirely. It is also unknown whether a newer shingles vaccine, which uses only certain proteins from the virus and is more effective at preventing shingles, would have a similar or even stronger effect on dementia risk. Global data and the push for a randomized trial Geldsetzer hopes these findings will encourage more investment in this line of research. "At least investing a subset of our resources into investigating these pathways could lead to breakthroughs in terms of treatment and prevention," he said. Over the past two years, his team has checked health records from other countries, including England, Australia, New Zealand and Canada, where similar shingles vaccine rollouts took place. In such a study, participants would be randomly assigned to receive either the live-attenuated shingles vaccine or a placebo injection. "It would be a very simple, pragmatic trial because we have a one-off intervention that we know is safe," he said. Geldsetzer is seeking philanthropic support to fund this work, in part because the live-attenuated vaccine is now off-patent, even though it is the vaccine type for which he has gathered strong evidence from natural experiments. He also pointed out that such a trial might show meaningful results relatively quickly. In the Wales data, when researchers plotted dementia rates for people who were eligible versus ineligible for the vaccine, the two curves began to move apart after about a year and a half. Note: Content may be edited for style and length. Scientists Close In on a Universal Cancer Vaccine Quantum Spin Ice Is Real: Physicists Confirm Exotic State of Matter Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69596658/light-magnetic-secret-faraday-effect/'>Scientists Just Discovered the Secret Magnetic Behaviors of Light</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This discovery goes against nearly two centuries of scientific assumptions. We may earn commission if you buy from a link. In the scientific history of the exploration of electricity and light, some pretty big names come to mind: Newton, Franklin, Maxwell, Edison, and Tesla, among others. Born to a poor family, Faraday became a self-taught scientist and eventually the protégé of British chemist and inventor Humphrey Davy. His discoveries of electromagnetic induction (the basic principle behind all modern turbines) and the Faraday Effect (which describes how electromagnetism and light are related) eventually earned him a knighthood—which he promptly turned down, wanting to remain “plain Mr. Faraday to the end.” In a new study published in the journal Scientific Reports, physicists Benjamin Assouline and Amir Capua from the Hebrew University of Jerusalem announce the finding that light can exhibit a magnetic influence in addition to an electric one. “What we've found is that the magnetic part of light has a first-order effect, it's surprisingly active in this process.” Using the Landau-Lifshitz-Gilbert (LLG) equation, which typically describes spin behavior in materials, Assouline and Capua demonstrated how light can create “magnetic torque” that is actually similar to a static magnetic field. When applying this theoretical model to Terbium Gallium Garnet (TGG)—a material commonly used to test the Faraday Effect—the magnetic component of light accounted for 17 percent of atomic rotation in the visible spectrum. “Our results show that light ‘talks' to matter not only through its electric field, but also through its magnetic field, a component that has been largely overlooked until now,' Assouline said. According to Igor Rozhansky, a physicist at the University of Manchester who spoke with New Scientist, a re-evaluation of the magnetic component of light could give scientists a new way to manipulate atomic spins, though it's uncertain how strong this effect will be in certain materials. Faraday's experimental foray into the world of electromagnetism eventually inspired James Clerk Maxwell's famous equations that, effectively, laid the foundation of our modern technological society. But even 180 years after one of Faraday's most famous observations, it turns out there's still a lot more to learn. AI Might Need Emotion to Reach True Consciousness This Ancient Larva Still Has Its Brain And Guts This New Lifeform Looks Like a Living Sun A Toxic Flood Crisis Is Threatening Our Coastlines</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251203004731.htm'>A common constipation drug shows a surprising ability to protect kidneys</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 13:49:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Although the condition is widespread and serious, there are still no approved medications that can actively restore kidney function. A team led by Professor Takaaki Abe at the Tohoku University Graduate School of Medicine has uncovered an unexpected approach by repurposing a constipation medication. "Essentially, constipation disrupts the intestinal microbiota, which worsens kidney function. To test this idea, the research team organized a multicenter Phase II clinical study (LUBI-CKD TRIAL) across nine medical facilities in Japan. The trial enrolled 150 individuals with moderate CKD and examined how lubiprostone affected kidney health. This finding was based on changes in the estimated glomerular filtration rate (eGFR), a standard measure used to evaluate kidney performance. They determined that lubiprostone boosts the production of spermidine, a compound that enhances mitochondrial activity by encouraging the growth of beneficial gut bacteria. Improved mitochondrial function was linked to a renoprotective effect that helped limit additional kidney damage. This approach represents a significant shift from current CKD therapies, which focus mainly on lowering uremic toxins. This concept could also open doors to new treatments for conditions involving mitochondrial dysfunction. New Images Reveal Young Solar Systems Filled With Secret Planets NASA Satellite Captures First-Ever High-Res View of a Giant Pacific Tsunami Scientists Uncover New Evidence of Ancient Airbursts That Ravaged Earth Without Leaving Craters Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a69520328/surprise-coronal-mass-ejection/'>Earth Just Got Slammed By a Surprise Solar Flare. Why Didn't Scientists See It Coming?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists can usually spot coronal mass ejections before they affect Earth—but this one snuck past coronagraphs and satellites. When the Sun erupts in a tantrum, charged particles expelled into space can seriously disrupt our electrical infrastructure—not to mention mess with GPS, cut off radio communications, trigger power outages, and cause satellites to glitch. Most of these solar storms come in the form of either flares or coronal mass ejections (CMEs) that can be predicted, but some arrive in stealth mode. Coronal holes have open magnetic field lines that allow plasma and solar wind to escape, potentially affecting operations on Earth, and this is in line with the 11-year solar cycle—our star has passed peak activity at solar maximum and is now headed towards solar minimum, which is the period of low activity during which coronal holes are most common. But this was only the prelude to a coronal mass ejection. Warped magnetic fields in the lower corona are what directly give rise to coronal mass ejections. When these stressed, unstable magnetic fields twist and realign to relieve some of that stress (a process known as magnetic reconnection), they also unleash an immense amount of electromagnetic energy, and gobs of coronal material and magnetic field are vomited into space. Stealth CMEs tend to start brewing at higher altitudes within the corona (where there are weaker magnetic fields), and only appear to be slightly brighter or dimmer spots on the Sun. Even when CMEs are detected by coronagraphs, it is impossible to tell what region of the Sun they emerged from, so their trajectories remain in the dark. There might be a way to catch these stealth storms before they come crashing into Earth's magnetic field in the future. It is only because of NASA's STEREO spacecraft duo that their beginnings were caught (at a different angle than observers can see them from on Earth). The new study makes recommendations for both identifying stealth CMEs and tracing their origins. These include looking at images taken at different times to see whether there were any obvious changes in coronal activity, using intensity data to interpret these images and gauge how intense that activity is, and using advanced processing techniques for images of phenomena (such as sudden brightening) that might signal a disturbance in the corona. Swaths of the northern sky lit up in brilliant purples and reds that night. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/scientific-americans-best-fiction-and-nonfiction-picks-for-science-minded/'>From Climate Catastrophes to Space Romance––Scientific American's Favorite Science Books of the Year</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientific American Shares Its First-Ever Best Fiction and Nonfiction Books of the Year Scientific American unveils its first-ever best fiction and nonfiction books of the year, spotlighting stories that blend science, imagination and unforgettable voices. By Kendra Pierre-Louis, Brianne Kane, Fonda Mwangi, Alex Sugiura & Jeffery DelViscio Ecco; Crown; Doubleday; Ballantine Books; Scientific American Illustration One way that we here at Scientific American stay on top of what's happening in science is not just by examining research papers and studies but also by reading books. For the past two years we've shared our staff favorites, but this year we've decided to expand into two new lists: Scientific American's first-ever best nonfiction books and fiction books of the year. To take us through the selections we have Bri Kane, SciAm's associate books editor. Thanks so much for joining us today, Bri. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Can you tell us a little bit about the process of getting [these book lists] together? What are some of the key things that you looked at? Kane: Yeah, I mean, this is Scientific American's first year doing a best fiction and best nonfiction of the year list; we've had a very popular staff-favorites books list the last two years. Kane: So in trying to find the best fiction and the best nonfiction of a year it means doing a ton of research, which, thankfully, we're a bunch of good-natured nerds, and we love any excuse to research and any excuse to read more books. I mean, it was a great self-assigned homework project this year. So some of the most important things that we were looking at for every book is it had to have an exceptional voice in writing and an incredible story. Kane: Even nonfiction needed to really catch our attention in the narrator's voice and the story that they were trying to tell us. And science is a really broad umbrella, and we tried to have a really broad view of what we considered science, but every single book on our list has something that can appeal to a more science-minded reader, so I'm really excited to showcase those books. And then every book has a little special je ne sais quoi—that little sparkle that just catches your eye—is the book that you just keep telling people about, even though you read it months and months ago, maybe. Those are the books that we really, like, found ourselves drawn to and were talking about with each other the most, and we wanted to talk to our audience about them. Pierre-Louis: That makes total sense, and I totally know how you feel. I think one of the books that I suggested was exactly in that [category] of just books that I was obsessed with and couldn't stop talking to people about. So can you tell us, starting with the nonfiction side, what are a couple of books that really topped your list? Kane: Yeah, I mean, one book that we have been talking about in our books group in our newsroom for a while now is Human Nature by Kate Marvel. I mean, I have been following climate science and the changes to our environment for a long time now ... Kane: But the way that Kate connects our current climate catastrophes with our history of literature and history of archaeology and just the whole world—it was really beautiful to bring everything all the way back down to our feelings and being pissed off about climate change, being sad, trying to find hope in all of this darkness and also some humor along the way. I mean, it was a really, really special read for a lot of us, and it had that sparkle from the first page. Pierre-Louis: It feels like a lot of climate books are often kind of depressing, and it—the way you're talking about Human Nature is—it wasn't necessarily just, like, a downer. Kane: No, and I think you're right: a lot of books trying to cover this issue really bum you out, and it can be kind of a slog to get through. But I found Human Nature to be, like, a wild ride. Pierre-Louis: That's awesome, and I know that there's another book kind of in a similar vein in terms of, like, really loving it and not being able to put it down within the nonfiction category. Kane: Yeah, They Poisoned the World by Mariah Blake is exceptional. Kane: Yeah, Andrea Gawrylewski, our chief newsletter editor, called it “an epic piece of science writing,” and that is not a compliment that gets thrown around lightly around here. Kane: And the companies that are poisoning our communities around this country and what that means for our understanding of our place in this fight and the legacy that we're leaving. I mean, Mariah Blake interviewed hundreds of people in this small town and discussed their personal medical histories ... Pierre-Louis: And they're chemicals that are pretty ubiquitous, right? They're often in the material that we use to make, like, jackets waterproof. They linger in the environment for so long that, by the perspective of human lifespans, they linger forever. Kane: But everyone needs to be looking into PFAS and forever chemicals a bit more than they are. And Mariah Blake does a really incredible job of bringing that all on the page. But those books are very much on the nonfiction side. Kane: Yeah, I mean, it was really fun this year to have an excuse to read some alien stories ... So one book that really got my attention this year is Daniel H. Wilson's Hole in the Sky ... Kane: It's a “first-contact” alien story, which you can't see my air quotes there, but it kind of plays with this idea of first contact because the aliens land on a Cherokee reservation in Oklahoma. And the Native people's relationship to what we would refer to as “first contact” is very different than the Western understanding of that term and science fiction as a genre's general use of that term. And Daniel is a brilliant thriller writer who can terrify you but hold your hand through this story so that you learn from different perspectives and to have a different understanding of a genre that maybe you're very well familiar with and you've read all the classics, or maybe you're still just dipping your toe in. Daniel Wilson's definitely one that caught my attention this year, and my conversation with him earlier this year was exceptional. Pierre-Louis: And it feels like, very often, when we encounter first-contact stories they are often kind of through a very white, male, Western lens, and it seems like he's playing with that a little bit. And they have a bit of an estranged, traumatic experience that created some distance between them—I don't wanna spoil it too much for you, Kendra ... Kane: But it's really, really beautiful, and it really made me think about Scientific American and science fiction, generally, [their] relationship to what we would refer to as “first contact” ... I think the specificity of our language is really important, and novels like Hole in the Sky are a novel that can remind you of these really intellectually rigorous topics. Kane: Yeah, I mean, we, along with everyone else, have been obsessed with Atmosphere from Taylor Jenkins Reid this year—lesbians falling in love in NASA? Former space reporter and current senior news editor here at SciAm, Sarah Lewin Frasier, reviewed it for us. And again, if a former space reporter thinks that your fictional version of NASA is real, relatable and interesting to be in, that is a very high bar to reach around here. I mean, I've been joking with Clara Moskowitz, our chief of reporters, who was also a space and physics editor here, about sobbing your eyes out at the end of this book in the best way possible, which all romance novels really should be doing. And it's fun to think about space romance. Astronauts kiss [Laughs], and, like, they want to have adventures that are not just taking place on Mars or on the moon. Kane: And I want to read every single story about that. Kane: Yeah, you can find all three of our lists—our staff favorites, best fiction and best nonfiction of the year—online later this week. Pierre-Louis: Thank you so much for coming today, and I can't wait to check out the full lists. Kane: I can't wait for everyone to see these lists and to see what you read this year, too, Kendra. Pierre-Louis: And don't forget to listen on Friday for our episode that looks at innovations in treating postpartum depression. Science Quickly is produced by me, Kendra Pierre-Louis, along with Fonda Mwangi and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. She has worked for Gimlet, the Bloomberg and Popular Science. Brianne Kane is associate editor for books and rights manager at Scientific American. After honing her reviewing skills at BUST Magazine and Electric Literature, Kane quickly made her mark at Scientific American by launching the first-ever end-of-year compilation of its staff's favorite books and spearheading its lists of best fiction and best nonfiction of the year. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, the Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02811-9'>Standardized metrics for assessment and reproducibility of imaging-based spatial transcriptomics datasets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 10:56:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Spatial transcriptomics lacks standardized metrics for evaluating imaging-based in situ hybridization technologies across sites. In this study, we generated the Spatial Touchstone (ST) dataset from six tissue types across several global sites with centralized sectioning, analyzed on both Xenium and CosMx platforms. These platforms were selected for their widespread use and distinct chemistries. We assessed reproducibility, sensitivity, dynamic ranges, signal-to-noise ratio, false discovery rates, cell type annotation and congruence with single-cell profiling. This study offers ST standardized operating procedures (STSOPs) and an open-source software, SpatialQM, enabling evaluation of samples across all technical metrics and direct imputation of cell annotations. The generated imaging-based spatial transcriptomics data repository comprises 254 spatial profiles, incorporating both public and newly generated ST datasets in a web-based application, which enables analysis and comparison of user data against an extensive collection of imaging-based datasets. Finally, we establish best practices and metrics to evaluate and integrate imaging-based multi-omics data from single cells into spatial transcriptomics to spatial proteomics. Spatial omics technologies provide insights into the molecular and cellular architecture of tissues, with a level of detail that is now as low as 5 nm1,2,3. However, there is a need for improved accessibility, scalable analytical tools and standardized practices across experimental design, data generation and interpretation4,5. The absence of broadly adopted standards and the variability in data quality across platforms complicate efforts to compare results across studies, thus obstructing the development of a unified framework for evaluating spatial data6,7. Benchmarking studies thus far have laid important groundwork but often rely on public datasets not optimized for cross-comparisons8,9, with a limited number of samples10, focused on a restricted range of tissue types8,11 or lacking replicates10,12,13,14 and matched single-cell references. The absence of standardized methodologies across all samples and of cross-institutional, multi-site comparisons in these studies limit their broader applicability and impact, similar to development and benchmarking of microarrays and sequencing methods (for example, MAQC/SEQC)15,16,17. These limitations underscore a need for more systematic strategies to assess reproducibility and performance in spatial omics. To address these gaps, the ST project was established as a multi-site, multi-platform initiative to evaluate imaging-based spatial transcriptomics technologies. Important components of this effort include the use of harmonized procedures developed collaboratively and implemented across institutions, which we term STSOPs; the establishment of quality metrics through an accessible open-source software package, SpatialQM; and the development of a web-friendly data repository, the Spatial Touchstone Portal (STP), for users to assess in real time the quality of their experimental run. SpatialQM, an open-source software package for metrics that users can adapt into their quality control analysis locally, enables researchers to evaluate their own experiments using technical benchmarks derived from the ST dataset. STP is an interactive resource that hosts, to our knowledge, at present the largest curated repository of imaging-based spatial transcriptomics datasets and generates metrics using SpatialQM. The STP encompasses comprehensive technical, biological and reproducibility metrics and provides immediate access to quality assessments for approximately 33 million cells and approximately 7 billion high-quality transcripts, drawn from both publicly available (PUB) datasets and newly generated ST datasets. STP also allows users to upload and compare their own datasets against ST and PUB datasets, contributing to a growing, community-driven repository of quality metrics. Through STSOPs, SpatialQM and STP, we aim not only to address technical variabilities inherent in spatial omics but also to promote standardization of experimental and analytical practices4,18,19. This initiative is poised to bridge gaps in data comparability and platform evaluation, ultimately advancing reproducibility and accelerating progress in the field. To evaluate technical performance, we first focused on data from the Xenium (10x Genomics) and CosMx SMI (Bruker Spatial Biology) platforms—the two commercial spatial imaging systems with subcellular resolution (50–100 nm) and high-depth multiplexing for RNA and proteins available across all three sites. These metrics were evaluated through serial sections from six tissue types (across two conditions: normal and cancerous), using predesigned panels for RNA and protein (Fig. The ST dataset profiles more than 8 million cells (with counts ranging from 3,624 to 321,482,112 per sample and a median of 71,296 cells). The ST dataset includes profiles from tissue sections comprising tissue microarrays (TMAs) and whole sections (Supplementary Figs. Additionally, we curated all PUB spatial imaging data on Gene Expression Omnibus (GEO) datasets, accounting for the Xenium, CosMx and MERSCOPE platforms, totaling approximately 33 million cells and approximately 7 billion transcripts. Except for breast cancer samples, which were serially sectioned and profiled in duplicates at a single site (University of Adelaide (UoA)), all other tissue samples in the ST project were serially sectioned at their respective originating institutions. This centralized sectioning approach ensured consistency in techniques and minimized variability that could arise from multiple handling procedures. Samples were subsequently profiled across multiple institutions. To maintain sample integrity and reduce potential degradation, all slides were processed within a 1−3-week window. All samples were analyzed on both Xenium and CosMx imaging-based platforms, generating 77 profiles, and subjected to FFPE single-nuclei RNA sequencing (snRNA-seq) (snPATHO-seq, n = 6) and hematoxylin and eosin (H&E) staining (n = 44) (n = 126 total). The PUB dataset includes both fresh-frozen and FFPE tissue samples from the brain, kidney, lymph node, skin, pancreas, lung, breast, colon, larynx, liver, ovary, prostate, uterus and nerve (n = 177; 131 human and 46 mouse) that have been profiled in Xenium, CosMx or MERSCOPE platforms. Despite the PUB dataset containing a larger number of tissue profiling experiments, samples had fewer cells profiled compared to those from the ST dataset. The STP comprises both the ST and PUB datasets, featuring human cells (n = 22,963,452) and mouse cells (n = 10,572,065) containing both FFPE cells (n = 23,852,502) across 14 different tissue types and fresh-frozen cells (n = 9,693,015) across 3 different tissue types. The STP provides access to approximately 13.833 million cells and a total of approximately 2.3 billion high-quality transcripts (Fig. All graphics within the STP web interface allow users to visualize data comparisons across three categories: ALL (combined PUB and ST), PUB and ST. These options enable detailed cross-comparisons, allowing researchers to probe their sample data against different tissue types or platforms, including Xenium, CosMx and MERSCOPE. a, Schematic overview of the ST workflow. Serial sections from the same FFPE blocks were processed and analyzed using two imaging-based spatial transcriptomics platforms, Xenium and CosMx, across three sites—Site1 (UoA), Site2 (WCM) and Site3 (SJH)—with an additional experimental run at 10x Genomics headquarters (Site 4; 10x Genomics). b, Representation of ST and PUB datasets and tissue types, including breast (BR), brain, pancreas (Panc. To establish the technical performance metrics across assays, various parameters were first assessed at the transcript level. In the ST dataset, transcript level varied from 0.07 to 0.95 normalized TPC with a mean of 0.29 ± 0.24 (95% confidence interval (CI): 0.24−0.35) across all tissue types. The PUB dataset, however, recorded up to 2.81 TPC with a mean of 0.82 ± 0.55 (95% CI: 0.72−0.90). For ST cancer samples, breast tissue showed the highest mean at 0.33 normalized TPC. In the PUB datasets, pancreas tissue again had the highest mean number of TPC (266), consistent with high overall gene expression20. To ascertain transcript distribution across samples, the total number of transcripts per area (TPA, μm2) was calculated, normalizing the mean number of transcripts in a given cell area to the target gene panel size. Cell size and cell segmentation can significantly impact these above metrics; therefore, normalized transcripts per nucleus (TPN) calculations were introduced to account for signals within the nucleus, independent of cell segmentation (Fig. TPN is calculated by dividing the observed number of transcripts within nuclei by the number of cells. An average TPN of 20 indicates that 20 transcripts were detected within the nuclei segmentation mask of a given imaging-based platform. The ST dataset showed a TPN range from 18 to 257, with an average of 73 ± 52.86 (95% CI: 61.02−85.01), whereas the PUB datasets ranged from 3.4 to 779, with an average of 91.43 ± 100 (95% CI: 75−108). Notably, high TPN, TPC and TPA values in the PUB dataset were primarily driven by a single sample with whole-transcriptome profiling, which highlights the ability of these metrics to find outlier samples. Within the ST dataset, which consists of serial sections and likely exhibits less variability than the PUB dataset, the CosMx platform demonstrated higher TPN (108.6 ± 60.9 (95% CI: 87.71−129.61)) and TPA (mean = 1.93 ± 0.66 (95% CI: 1.70−2.16)), whereas the Xenium platform showed superior TPC (mean = 0.4 ± 0.28 (95% CI: 0.30−0.48)). Variations in tissue size—from large sections to partial sections and TMAs—and differences in tissue type, such as healthy versus diseased states (for example, normal versus cancerous tissues and non-infected versus infected tissues), are both critical factors influencing the number of detectable transcripts and cell counts in spatial transcriptomics. For example, smaller tissue sections or TMAs, which are often used for high-throughput or targeted analyses, may yield fewer total detectable transcripts and lower cell counts due to the constrained sample size and limited area for capturing cellular and molecular heterogeneity. These factors highlight the critical importance of both tissue size and type when designing spatial transcriptomics experiments, and careful consideration of these variables is essential to ensure the robustness, accuracy and interpretability of the resulting data, as each contributes uniquely to the challenges and opportunities in spatial transcriptomics analyses. A notable example of how sample type and size impact data outcomes is evident in a large COVID lung study within the ST dataset, which predominantly used TMA cores from biopsied patients. These cores recorded the lowest overall cell counts (mean = 1,117.2 ± 598.01 (95% CI: 617.3−1,617.2)), likely due to the constrained sample size and specific pathological conditions of the tissue, affecting cell integrity and transcript detection efficiency. This highlights the crucial role of sample selection and preparation in spatial transcriptomics, which substantially influences the robustness and interpretability of the resulting data, highly affected by the selected field of view (FOV) and surface area. Probe specificity was then evaluated using the ‘specificityFDR' metric, where a value of 0.05 indicates that, on average, 5% of transcripts per gene are false positives. Our results show that, generally, lower specificityFDR values corresponded to higher experiment or sample quality. In the ST dataset, specificityFDR values ranged from 5.5 × 10−5 to 0.069 (mean = 0.05 ± 0.02 (95% CI: 0.41−0.49)), whereas, in the PUB dataset, values ranged from 8.2 × 10−5 to 0.69 (mean = 0.23 ± 0.21 (95% CI: 0.19−0.26)) (Supplementary Fig. 2), notably highlighting a very broad error range for public datasets. Next, sample noise levels were evaluated using the signal-to-noise ratio (SNR) and dynamic range. The SNR measures the distribution of mean transcript detection versus mean control probes, assessing the extent to which transcripts are detected above noise levels. The SNR for the ST dataset ranged from 0.12 to 0.53 (mean = 0.28 ± 0.08 (95% CI: 0.26−0.30)), and the PUB dataset ranged from 0.01 to 0.90 (mean = 0.33 ± 0.19 (95% CI: 0.30−0.36)) (Supplementary Fig. Dynamic range reflects the range of expression, with a dynamic range of 3 representing a thousand-fold increase in expression between the largest average of every gene compared to the average of negative control probes; higher values indicate less noise. The dynamic range for the ST dataset ranged from 2.14 to 5.85 (mean = 4.11 ± 1.03 (95% CI: 3.88−4.35)) and ranged from 1.13 to 5.06 (mean = 2.87 ± 1.25 (95% CI: 2.66−3.07)) for the PUB dataset (Supplementary Fig. Notably, the PUB dataset exhibited a narrower overall range for these metrics, whereas the ST dataset showed lower variance, suggesting greater stability across different tissues and assays. Tissue-specific trends were also observed: the highest SNR in the ST dataset was recorded in breast tissue (mean = 0.37 ± 0.11 (95% CI: 0.27−0.46)), whereas the highest SNR in the PUB dataset was recorded in brain tissue (mean = 0.51 ± 0.23 (95% CI: 0.43−0.59)). For dynamic range, both datasets showed the highest values in pancreas samples (ST: mean = 4.68 ± 0.86, 95% CI: 4.35−5.02; PUB: mean = 4.61 ± 0.44, 95% CI: 2.65−3.01). Variability in SNR and dynamic range was tissue dependent within the ST dataset, with normal tissues such as the appendix showing a narrower range (SNR = 0.11−0.37; dynamic range = 2.14−4.75) compared to cancerous tissues such as breast cancer (SNR = 0.22−0.52; dynamic range = 2.70−5.15). The fraction of transcripts assigned to cells (FTC) was calculated to determine the percentage of transcripts accounted for across the tissue sample after cell segmentation. For instance, an FTC value of 0.8 indicates that 80% of the total transcripts present in the sample are assigned to cells. The FTC for the ST dataset ranged from 0.20 to 1.0 (mean = 0.85 ± 0.20 (95% CI: 0.81−0.90)), whereas, for the PUB dataset, it ranged from 0.02 to 1.0 (mean = 0.87 ± 0.15 (95% CI: 0.85−0.90)) (Supplementary Fig. These results suggest that the FTC was consistent across datasets. As the STP continues to grow with user-generated datasets, the scale and robustness of spatial transcriptomics will improve, potentially leading to more consistent SNR, dynamic range and FTC values across a wider variety of tissue types and conditions. The mutually exclusive correlation ratio (MECR) measures the quality of cell segmentation by assessing the rate at which co-expressed genes, which should be exclusive to a specific cell type, are expressed together9. Using a series of marker genes in pairs of cell types, MECR measures how often these genes are co-expressed above a threshold of zero transcripts in the same assigned cell. Of note, this metric, primarily focused on general immuno-oncology profiling, indicates greater specificity of transcript discrimination with lower values. For example, an MECR value of 0.01 indicates that, on average, 1% of all cells co-express genes that should be mutually exclusive. The MECR for the ST dataset ranged from 0.02 to 0.12 (mean = 0.05 ± 0.02 (95% CI: 0.05−0.06)), whereas it ranged from 0.01 to 0.96 (mean = 0.13 ± 0.18 (95% CI: 0.10−0.16)) for the PUB dataset (Fig. Notably, the variability of MECR was higher in the PUB dataset, likely due to differences in tissue types examined, assay consistency, platforms used and procedural variations across different sites. In terms of sample type, MECR variability was more pronounced in normal samples than in cancer samples, with normal sample values ranging from 0.01 to 0.16 (mean = 0.06 ± 0.04 (95% CI: 0.055−0.071)) compared to 0.01 to 0.10 (mean = 0.04 ± 0.02 (95% CI: 0.035−0.046)) for cancer samples. MECR performance also varies with cancer type; it was more effective in breast cancer samples (range, 0.034−0.047) compared to prostate cancer samples (range, 0.028−0.059), where the presence of more infiltrating normal cells could impact the metric (see the ‘Biological quality metrics' subsection). We then examined several other key metrics for assessing samples. For the ST dataset, sparsity values ranged from 0.90 to 0.96 (mean = 0.94 ± 0.01 (95% CI: 0.93−0.94)), whereas, in the PUB dataset (Supplementary Fig. Second, we defined the ‘entropy' value, which quantifies uncertainty or randomness in gene expression, with higher values indicating greater variability and lower predictability. For the ST dataset, entropy values ranged from 0.30 to 0.68 (mean = 0.46 ± 0.08 (95% CI: 0.44−0.48)), whereas, in the PUB dataset, they ranged from 0.17 to 2.31 (mean = 0.88 ± 0.38 (95% CI: 0.82−0.95)) (Fig. Third, we quantified ‘complexity' as a measurement of how many genes account for explaining 50% of total counts in the sample, such that a complexity value of 30 means that 30 genes are responsible for half of all the transcripts in the sample. Notably, a normalization by panel size makes the complexity metric (‘normComplexity') comparable betweeen platforms with high variance in panel size. normComplexity metrics for the ST dataset ranged from 0.09 to 0.77 (mean = 0.5 ± 0.15 (95% CI: 0.46−0.53)), whereas, within the PUB dataset (Supplementary Fig. This metric was comparable across different tissue types within ST, except for pancreas, which had a minimum normComplexity range of 0.09 compared to all other tissue types (minimum range of 0.39). This is related to previous observation that Xenium has higher TPN despite the high SNR (Fig. For the ST samples analyzed with the CosMx platform, complexity and entropy values were consistently higher than those processed on the Xenium platform, specifically with these probe sets. Specifically, normComplexity values ranged from 0.49 to 0.77 (mean = 0.63 ± 0.10 (95% CI: 0.60−0.67)) on CosMx and from 0.09 to 0.49 (mean = 0.39 ± 0.09 (95% CI: 0.36−0.41)) on Xenium. To perform a comprehensive analysis of reproducibility across both ST and PUB datasets, we scaled all technical metrics for each sample and conducted a principal component analysis (PCA) (Supplementary Fig. Specifically, the Xenium and CosMx samples were distinctly separated by principal component 1 (PC1), which accounted for 34.39% of the total explained variance. Additionally, an outlier was identified in the PUB dataset, which, upon further examination, was attributed to a limited selection of FOVs for the CosMx platform. A comparison of cell counts across each tissue type revealed significant variability between standard H&E counts and those from both imaging platforms. Although there are differences in cell segmentation algorithms, resulting in varying cell counts, the ability of the platforms to detect broad cell types remains largely unaffected. This consistency is particularly evident because many of these signals concentrate in the nuclear area (Fig. H&E staining generally provides a more accurate cell count because it does not depend on various combinations of cell membrane segmentation markers. We found the cell counts through cell segmentation to be consistent between H&E and Xenium, whereas CosMx cell counts were consistent with the other platforms depending on the tissue type. This variability underscores the importance of careful interpretation of any metrics derived from imaging-based spatial transcriptomics platforms, as they heavily rely on the accuracy of the cell membrane markers (Supplementary Fig. Metrics include fraction of Tx per cell (CellTxFraction; orange), DynamicRange (yellow), SNR (SigNoiseRatio; green), TxPerCellNorm (purple), specificityFDR (gray) and Panel size (dark blue). FDR, false discovery rate; Norm, normalized; Tx, transcripts. The ST project provided a unique opportunity to delve deeper into reproducibility between the same tissue type (in this case, breast cancer) from different studies and datasets (ST and PUB) across imaging-based transcriptomics platforms (Fig. A total of six breast cancer samples were used for comparison: four sequential sections from two samples in the ST dataset (profiled in duplicates, with two sections from two samples on Xenium and two sections from two samples on CosMx) and four samples from the PUB dataset (three on Xenium and one on CosMx). For the ST samples, a total of 3,016,015 cells were detected across all spatial profiles, with an average of 377,000 ± 83,807 cells per section (95% CI: 306,937−447,066). By contrast, the PUB samples had a total of 1,393,890 cells across all spatial profiles, with an average of 348,472 ± 213,783 (95% CI: 8,295−688,649) cells per section profiled. Although the average number of cells profiled was similar between the two groups, more PUB samples were processed on the Xenium platform than on CosMx. The FTC for the ST dataset ranged from 0.20 to 1.0 (mean = 0.85 ± 0.20 (95% CI: 0.80−0.90)), whereas it ranged from 0.02 to 1.0 (mean = 0.87 ± 0.15 (95% CI: 0.85−0.90)) for the PUB dataset. The mean FTC for the ST dataset is likely more representative of metrics commonly observed by general users. The specificityFDR for the ST dataset ranged from 0.045 to 0.060 (mean = 0.05 ± 0.02 (95% CI: 0.041−0.048)), and it ranged from 8.24 × 10−5 to 0.7 (mean = 0.23 ± 0.21 (95% CI: 0.19−0.26)) for the PUB dataset. These values indicate that the specificityFDR was similar across breast tissue for both datasets (Supplementary Fig. The variance in entropy was much higher for the PUB dataset, ranging from 0.58 to 1.03 (mean = 0.75 ± 0.2 (95% CI: 0.43−1.05)) compared to the ST dataset, which ranged from 0.40 to 0.68 (mean = 0.55 ± 0.1 (95% CI: 0.46−0.62)), showing lower variability (Supplementary Fig. To establish the variability across the data, spatial autocorrelation was calculated for every sample using Moran's I statistic21. Only the common genes between the two platforms and all samples (n = 203) were selected (Fig. The standardized effect size for each metric across ST and PUB datasets is shown (Supplementary Fig. 3a) to understand the technical variability between cohorts and tissue types profiled in both. These observations highlight the reproducibility and consistency of metrics in spatial transcriptomics, which likely result from more controlled sampling and experimental design in the ST dataset. Overall, the ST breast cancer dataset demonstrated greater consistency across all metrics compared to the PUB breast dataset, yet their mean values remained similar (Fig. This underscores the necessity of comprehensive technical evaluations to assess the accuracy of experimental assays. This unique experimental design allowed us to evaluate the reproducibility of these assays. Serial sections sequentially processed within the same institute highlight the consistency of both Xenium and CosMx platforms. Data from both prostate (n = 1) and breast (n = 2) tumors had negligible variation in mean transcript quantifications across the entire range of detection (r = 1.00, n = 3 total) (Fig. Adjacent tissue sections, shipped and processed at independent sites, yielded highly consistent data, with correlation coefficients typically exceeding 0.95 (mean r = 0.97) (Supplementary Fig. a, Standardized effect size difference of technical metrics between tissue types present in ST and PUB datasets. b, Scatter plot of autocorrelation (Moran's I) for all breast cancer samples present in ST and PUB datasets for the Xenium and CosMx platforms. c, UMAP representation of reference scRNA breast cancer used for cell type annotation of spatial datasets40. Each cell is color coded based on its type. d, Variations in cell type proportions across different breast cancer samples. e, Comparative visualization of H&E with scale bar, 2,000 µm, and Xenium (XR) cluster plots for breast tissues in both ST and PUB datasets using cell type annotation from the same reference (using color code from c and d). (i) H&E image of an ST breast cancer sample profiled at Site 1 (UoA). (ii) Cell type plot of an ST breast cancer sample profiled at Site 1. (iii) H&E image of a PUB breast cancer tissue. (iv) Cell type plot of a PUB breast cancer tissue. f, Cell type plots from cell annotation transfer from snPATHO-seq reference for a prostate cancer sample tissue profiled on both XR and CosMx (CR) platforms across different sites. (vii) Sample from Site 1, profiled first with XR and then followed by CR Protein (same section); dual assay conducted at Site 1. BR, breast cancer; CAF, cancer-associated fibroblast; CO, colon; PA, pancreas; PVL, perivascular-like. To further validate the biological relevance of these metrics, we referenced snPATHO-seq data from the prostate samples as a cell-specific benchmark (Fig. 3) and annotated cell types for all 12 assays. Our analysis confirmed that 100% of cell types (n = 12) identified in the reference dataset were represented in all assays, with very similar proportions (Fig. This consistency in cell type identification across assays underscores the accuracy and utility of the ST dataset in capturing the cellular landscapes of tissues, exemplified here by prostate cancer. To evaluate the accuracy of transcript quantifications from ST platforms, we performed snRNA-seq (snPATHO-seq; Methods) from all the FFPE tissue assessed in this study (Supplementary Fig. Although the snPATHO-seq quantifications are not considered an absolute ground truth, the technology has proven effective in providing robust transcript detection across a broad dynamic range22,23. The isolation of individual cells or nuclei also provides relatively pure reference expression profiles for the cell types assessed, which is crucial for evaluating the extent of segmentation error that can result in mixed transcript profiles. Several criteria were used to assess the accuracy of ST platforms. Initially, the average transcript abundance for each gene between the snPATHO-seq and ST data for each tissue was compared. Xenium samples consistently showed good correlation (Spearman's ρ = 0.78, r = 0.64–0.94) with snPATHO-seq data across its entire dynamic range (Fig. By contrast, CosMx samples had lower correlation (Spearman's ρ = 0.60, r = 0.35–0.80) and displayed ubiquitously inflated detection of lowly expressed genes and variably reduced sensitivity for highly abundant transcripts (Fig. 3a,b), aligning with previously noted variation in dynamic range (Supplementary Fig. However, comparing mean expression across a tissue can be misleading because although noise tends to distribute evenly throughout the tissue, marker transcripts for rare cell types may be highly abundant in individual cells and yet have low mean counts across the tissue. To address this, the consistency of detected expression profiles within individual cell types was evaluated using matched snPATHO-seq data as a reference for supervised cell type annotation using InSituType24. Further supporting the reproducibility of these platforms, we note that replicate samples frequently exhibit similar cell type proportions with these predictions (Supplementary Figs. The correlation of cell type expression patterns was consistent with those of the bulk tissue. Xenium performed consistently well with a mean cell type correlation coefficient of 0.78 (r = 0.48–0.95). Although some CosMx samples, such as breast cancer sample 2 (BR_2: replicate 1, Spearman's ρ = 0.80; replicate 2, Spearman's ρ = 0.76), correlated reasonably well, the variability across samples was notably higher (mean = 0.57, r = 0.25–0.80) (Fig. Cell segmentation poses another major challenge that impacts the accuracy of transcript quantifications—imprecise cell boundaries can lead to transcript misassignment from adjacent cells, producing ‘mixed' transcriptomic profiles. Although metrics such as MECR can detect co-detection of mutually exclusive transcripts, they do not fully account for transcript misassignment, which can distort detected expression. To better understand the impact of segmentation errors, each cell was treated as a mix of pure cell types, and their profiles were decomposed using robust cell type decomposition (RCTD)25 with the snPATHO-seq data as a reference. In samples with high levels of noise, these weights may not solely reflect segmentation error (Fig. The prostate cancer sample was also subjected to Xenium In Situ Gene Expression (Human Multi-Tissue and Cancer Panel) with cell segmentation staining-based nuclear (DAPI), cytoplasmic (18S rRNA, αSMA/vimentin) and membrane (TP1A1/E-cadherin/CD45) staining. For this sample, three segmentation approaches were compared: nuclear expansion at varying distances, staining-based morphology segmentation and Proseg26, which infers plausible cell boundaries based on the spatial distribution of transcripts. However, increased expansion distances can result in transcript misassignment, as evidenced by elevated MECR values and decreased cell type purity after cell type decomposition (Fig. Tissue stroma is particularly problematic as it comprises diverse cell types—for example, fibroblast transcripts often contaminated annotated immune cells (Fig. Despite these challenges, the extent of transcript misassignment suggested by cell type decomposition remains modest: if we assume that decomposition weights purely reflect transcript assignment due to segmentation, the median extent of error (1 − dominant annotation weight) is only 14% for the common 15-µm expansion for Xenium data. Cell type annotation is also robust against these errors, producing effectively identical annotations for all segmentation strategies (Supplementary Fig. Nuclear segmentation alone provided higher cell type purity but at the cost of a dramatic reduction in transcript assignment (43.3 versus 81.6 TPC with 5-µm expansion). The staining-based morphology segmentation, or ‘multimodal' approach, struck a balance by producing high-purity profiles while retaining higher TPC (74.6 per cell) (Fig. Proseg demonstrated that high-quality segmentation could be performed without reliance on staining strategies to detect cellular compartments. Its segmentation resulted in the highest purity (MECR: 0.029; median maximum decomposition weight: 0.97) and 1.7× more TPC than multimodal segmentation (128.3 versus 74.6) (Fig. Clustering the resulting expression profiles also produced clusters that were more distinct than those from other segmentation approaches, with the highest mean silhouette width and cluster neighborhood purity (Fig. Although clustered data from each segmentation approach reasonably reflected cell type annotations, Proseg proved ideal for data where cell typing is performed by clustering rather than label transfer approaches. Together, this supports that segmentation approaches informed by regional transcript patterns may ultimately outperform those based on protein stains and morphological heuristics, such as nuclear expansion. To investigate this, we evaluated the ability of these platforms to perform several common tasks across datasets with varying quality characteristics. Specifically, we assessed the detection of diverse cell types throughout tissues, determined the extent to which both coarse and fine tissue organization can be resolved and evaluated the potential to predict intercellular signaling based on localized expression of ligands and their cognate receptors. For coarse cell typing, which minimally requires confident detection of cell-type-specific marker transcripts, it is plausible to assume that this task can tolerate some level of noise from off-target probe hybridization and segmentation error, provided that the dynamic range of marker transcripts is sufficiently high27,28. Our initial analysis focused on cell type marker detection within tissue structures with well-defined cellular organization. This included histologically confirmed blood vessels in a prostate adenocarcinoma sample, the mucosa of the appendix and the interface of malignant cells with healthy hepatocytes in a metastatic breast cancer sample (Supplementary Fig. The high sensitivity of the Xenium platform facilitated robust transcript detection of all marker genes assessed, clearly demarcating relevant tissue structures (Fig. Despite lower sensitivity in the CosMx data, highly expressed genes such as ACTA2 in fibroblasts from prostate tumors and APOC1 in hepatocytes from breast cancer metastasis still resolved tissue structure. Cell types defined by markers with inherently lower expression, such as LGR5 in intestinal stem cells or PECAM1 in endothelial cells, were not clearly identifiable (Fig. a, Left: visualization of tissue structure from adjacent prostate tumor samples with each cell type uniquely colored by technology (Xenium and CosMx). This provides a direct view of the cellular composition and spatial organization within the tissue. Right: expression of PECAM1 (endothelial) and ACTA2 (smooth muscle) in a region with defined blood vessels (marked by black box on left). Each colored point corresponds to an individual transcript. b, Abundance of cell-type-specific transcripts (ACTA2, smooth muscle; EPCAM, epithelial). Each point reflects an individual cell, colored by transcript abundance. c, Left: localization of annotated luminal epithelial cells. Spatial clustering was performed independently on each sample. d, Purity of annotated cells in gene expression (principal component) space. e, Localization of annotated T/NK cells (left, tissue plots) and T/NK-specific transcripts (right, cropped regions). We demonstrated that data of varying quality can yield similar proportions of predicted cell types using automated annotation methods but that the correlation of annotated cells with single-cell nuclei sequencing references can vary (Fig. Generally, these annotations capture similar high-level tissue organization across all datasets despite quality differences (Supplementary Figs. To further explore whether technical variation introduces more subtle errors in cell type annotation, we quantified the purity of cell type annotations within neighborhoods of the PCA embedding from transcript quantifications. Accurately annotated cells of a given type are assumed to have a high degree of transcriptional similarity, whereas ambiguous annotations will not be transcriptionally distinct from other cell types. The average neighborhood purity correlated directly with quality metrics, indicating that these metrics influence the ability to distinctly resolve expected cell types from the reference annotations (Fig. For instance, various cell types, such as lymphocytes, consistently exhibit lower purity in CosMx samples. This suggests that these annotated cells are not transcriptionally consistent, potentially indicating misannotation and needed improvements. However, this reduced purity may also reflect the ability of the platform to capture heterogeneous cell types and detect diverse cellular states, characterized by an increased diversity of transcripts, even if it compromises sensitivity. We next sought to address an important unanswered question in the field: ‘How does spatial transcriptomics correlate with spatial proteomics?' For this analysis, the Human Multi-Tissue and Cancer Panel (377-plex, Xenium) was used for transcriptomics, and the Human Immuno-Oncology Panel (64-plex, CosMx) was used for proteomics. Like our approach in transcriptomics, technical metrics were derived to assess multi-omic features from RNA to protein. For this study, a total of 24 samples were used, comprising 23 normal tissues and one cancerous tissue. For the statistical measurements, we assessed total nuclei count (TNC), fluorescence per cell (FPC), complexity, entropy, sparsity and SNR (Fig. a, Technical metrics comparison across different tissues for CosMx protein samples, with boxes color coded by tissue type: appendix (AP), green; colon (CO), orange; ileum (IL), blue; pancreas (PA), pink; prostate (PR). b, UMAP representation of cell type annotation for single-cell prostate cancer (Leiden clustering, resolution = 1, number of neighbors = 30, number of components = 30) with each cell colored according to its type. c, Analysis of the differences in cell type proportions across different sections of the prostate cancer sample, conducted across multiple sites; each bar represents an individual dataset that was cell phenotyped using the same reference snPATHO-seq PR sample. All cell type color codes are from b and c. e, UMAP representations of both the Xenium RNA and the CosMx Protein profiles, examining batch effects (left) and differences in clustering/cell annotations (right). Although TNC is not a direct measurement of sample quality, it can be a quick and useful metric to assess sample representation. However, the other metrics demonstrate more accurately sample quality. For spatial proteomic profiling, TNC ranged almost eight-fold, from 11,661 to 82,123 (mean = 49,579.58, s.d. = 18,764.04). Similar to spatial transcriptomics, TNC was most dependent on nuclei identification, tissue and FOV size, with TMAs having smaller TNC than whole sections. We then used FPC to measure the efficiency of antibodiesʼ ability to bind to cells across all markers, leveraging the total unfiltered fluorescence expressed within each cell. In the ST dataset, FPC ranged from 2,379 to 7,649 (mean = 4,109 ± 1,188). The FPC varied with tissue type; differences were observed among colon (mean = 4,440, s.d. Low values for this metric may indicate a lower expression level of the target fluorescent marker in the cells of that tissue type, which is useful for highlighting targets that might require orthogonal methods of quantification, differential labeling or tissues with unusual density profiles (Fig. The multi-omics integrative analysis to interrogate RNA and protein was conducted in two distinct stages. Initially, cell phenotyping was performed using the in situ RNA spatial transcriptomics breast cancer sample with corresponding reference data obtained via snPATHO-seq (Fig. Concurrently, the same tissue sample analyzed for protein expression using the CosMx platform was subjected to normalization using a centered log-ratio approach, and cells exhibiting low expression across all proteins (n = 62) were systematically filtered out (Fig. After normalization, PCA was performed on the top 50 principal components, with those 50 principal components used to calculate the relationships between cells, setting the number of neighbors at 30. Subsequently, cells were clustered using the Leiden algorithm with a resolution of 0.5, resulting in the identification of 17 distinct clusters. To integrate both modalities, MaxFuse, a method that leverages corresponding and weakly linked features from each modality, was used. Of the features identified, 27 corresponding to both RNA and protein were retained and used for integration analysis. Throughout the model training stage, labels previously established for each modality (cell annotation RNA and Leiden cluster protein) were used to maintain an agnostic perspective and prevent overfitting, particularly by avoiding reliance on unverified protein annotations. The integration results revealed a significant overlap between RNA and protein modalities in the derived embedded space (Fig. Notably, smooth muscle cells were overrepresented in the RNA data (Fig. 5e), whereas epithelial cell types showed good alignment across multiple clusters originating from the protein modality. This alignment suggests that these cell types were successfully profiled in both assays and can be distinctly identified through clustering. Based on these findings, we propose a downstream approach for cell annotation that uses this joint embedding, which annotates cells within a shared space between RNA and protein, notably enhancing the accuracy and tracing of cell type identification and localization (Fig. Although spatial transcriptomics has the capacity to profile a larger array of genes, proteomic profiling remains a valuable orthogonal method for validation, capable of resolving several cell types independently. This dual approach underscores the complementary nature of RNA and protein analyses in enhancing the understanding of cellular functions and interactions within complex tissue environments. In this context, the ST dataset demonstrates a narrower technical variability than the more variable PUB dataset. Although the PUB dataset reflects real-world variability, including data from commercial vendors with generally lower specificityFDR values, the ST dataset provides a controlled framework to assess baseline reproducibility in large-scale, standardized studies. These standardized protocols thus enhance reproducibility and comparability across institutions, establishing a unified field standard. Together, the ST and PUB datasets provide a balanced view, combining controlled reproducibility with practical, real-world applicability. Multi-institutional reproducibility was achieved by the consistent application of the STSOPs, which harmonized sample handling, quality control and data processing workflows (detailed in the Methods). The STSOPs were crucial in mitigating variability and enabling the reliable integration and comparison of data generated across different institutions. However, the centralization of tissue sectioning and initial FFPE processing limited the variability arising from pre-analytical steps, restricting our ability to fully evaluate inter-site differences in tissue handling and sectioning consistency. Future studies incorporating decentralized pre-analytical processing are necessary to capture the full spectrum of inter-institutional variability, including these critical upstream factors. Key metrics such as FTC and TPC depend heavily on factors such as site, panel size, tissue and platform and require accurate cell segmentation because they do not account for transcripts misassigned outside the tissue regions. Similarly, sparsity and entropy metrics are sensitive to panel specificity and segmentation quality; improper segmentation inflates these values by including transcripts outside cells. By contrast, metrics such as MECR, designed to detect misassignment within epithelial, immune and stromal compartments, may not be robust across different tissue types. Therefore, interpreting these metrics requires careful consideration of tissue type, probe panel and platform. In the ST dataset, cell segmentation was performed using instrument output files to ensure consistent analysis and community benchmarking. Nonetheless, ongoing development aims to improve segmentation algorithms26,29,30,31,32,33, with promising advances in pixel-based, segmentation-free models34,35, that learn tissue features directly from tissue images, bypassing traditional reliance on cell membrane or nuclei markers34,36,37,38,39. These methods could facilitate universal comparison across platforms and tissue types by eliminating platform-specific cell identification dependencies. Gene panel design strongly influences data quality and interpretability. For example, breast cancer samples analyzed with both multi-tissue and a tissue-specific panel showed that the latter yielded lower entropy and higher normalized TPC, demonstrating improved focus and efficiency. Tailoring panels by considering prevalent cell types, gene expression ranges and relevant biological pathways enhances both resource use and analytical clarity. Validating larger plex panels across diverse settings, including academia, contract research organizations, pharmaceutical companies and core facilities, is crucial for establishing reliable spatial omics standards. However, sample quality remains a critical factor; degraded or poorly preserved tissues can undermine even optimally designed panels, producing incomplete or biased data that obscure biological insights. The present study represents an initial effort to define foundational metrics to guide users in troubleshooting assay performance, tissue quality, instrumentation and probe design specificity challenges. These guidelines are not rigid thresholds but are expected to evolve as the portal incorporates more data and, hence, becomes more refined, as spatial technology applications expand to include a broader range of tissue types, platforms and experimental contexts. This study presents several caveats that are important for understanding its context and implications. Despite the relatively large number of samples, we recognize that it is not exhaustive. Further research with even larger datasets will be necessary to fully validate our findings and methods. Additionally, we selected two leading technologies specifically as examples of representative, cutting-edge commercial platforms. All these considerations were aimed at demonstrating best practices. We are confident that, as the field continues to evolve and more comprehensive data become available, the methodologies and metrics developed through this study will be expanded to offer deeper insights into the capabilities of multiple platforms. Comprehensive documentation of all experimental procedures and protocols (STSOPs) has been deposited at protocols.io (https://doi.org/10.17504/protocols.io.261ge54qdg47/v1). Normal tissues were collected as part of the Spatial Atlas of Human Anatomy project under strict ethical guidelines, with FFPE blocks prepared by pathologists at Weill Cornell Medicine (WCM) and New York Presbyterian Hospital. Institutional review board approval oversaw their collection. For prostate and breast cancer samples, H&E images were submitted to Histowiz (https://home.histowiz.com/) for evaluation. The prostate cancer sample (PR_1) contained prostatic adenocarcinoma (grade 3 cribriform, focal grade 4, total score grade 7), with surrounding normal tissues, focal inflammation and seminal vesicle involvement. This sample was obtained from a patient who underwent a radical prostatectomy and was provided by the Australian Prostate Cancer BioResource with written consent, complying with ethical protocols approved by the UoA and St. Andrew's Hospital (H-2018-222, no. Breast cancer sample BR_1 displays in situ ductal carcinoma and invasive ductal carcinoma, with focal inflammation and necrosis. BR_2 was a high-grade metastatic ductal carcinoma infiltrating the liver. Both breast cancer samples were collected with written informed consent under SVH 17/173 protocol (St. Vincent's Hospital Ethics Committee) and X19-0496 for autopsy samples, respectively. Both breast cancer samples were sectioned at the Garvan Institute of Medical Research and sent to the UoA for snPATHO-seq and spatial profiling. Tissues were immediately placed on ice, fixed in 10% neutral buffered formalin (NBF) for 24 hours and then paraffin embedded room temperature. RNA quality was confirmed using a Norgen Biotek FFPE RNA Purification Kit (product ID 25300) and a 5200 Fragment Analyzer with the High Sensitivity RNA Assay Kit (Agilent Technologies, DNF-472). Two 30−50-µm-thick scrolls per sample, adjacent to sections for CosMx, Xenium and H&E staining, were dewaxed with xylene (MilliporeSigma, 214736) and rehydrated with sequential ethanol (MilliporeSigma, E7023) immersions. After a wash in RPMI 1640 media (Gibco, 11875093), 50–100 µl of digestion mix (RPMI 1640, 1 mg ml−1 Liberase TH; MilliporeSigma, 5401151001) and 1 U µl−1 RNAse Inhibitor (RiboLock RNase Inhibitor; Thermo Fisher Scientific, EO0382) was added, and tissue was gently broken into smaller fragments with a 1.5-ml pellet pestle before topping up to 1 ml. After digestion, 400 µl of Ez Lysis Buffer (Sigma-Aldrich, NUC-101) was added, mixed and centrifuged. The pellet was resuspended in 250 µl of Ez Lysis Buffer with 2% BSA (MACS BSA stock solution; Miltenyi Biotec, 130-091-376) and 1 U µl−1 RNAse Inhibitor, homogeneized and supplemented with 750 µl of the same lysis mix. Samples were filtered through a 70-µm strainer (pluriStrainer Mini 70 µm; pluriSelect, 43-10070-50) and centrifuged and then washed again three times in 1× PBS (Gibco, 70011044) with 1% BSA, followed by two washes in 0.5× PBS (Gibco, 70011044) with 0.02% BSA. Nuclei were resuspended in 500–1,000 µl of the same buffer, filtered through a 40-µm strainer (pluriStrainer Mini 40 µm; pluriSelect, 43-10040-50) and counted using a Luna-FX7 using Acridine Orange/Propidium Iodide Stain (Logos). Immediate processing used the Chromium X controller and Chromium Fixed RNA Profiling Reagent Kits for multiplexed samples (Chromium Fixed RNA Kit, Human Transcriptome, PN1000475; Chromium Next GEM Chip Q Single Cell Kit, PN-1000418; Dual Index Kit TS Set A, PN-1000251; 10x Genomics) following user guide CG000527, Rev E. For storage, nuclei suspensions were supplemented with 10% glycerol and 0.1× Enhancer (10x Genomics, PN-2000482) and cryopreserved at −80 °C. Hybridization step used 500,000−1.5 million nuclei for 20 hours at 42 °C. Post-hybridization washes were pool-wash setting with an additional wash. Approximately 10,000 nuclei per sample were targeted for capture with an extra pre-amplification polymerase chain reaction (PCR) cycle. Indexing PCR conditions were tailored to 50–200 nM library concentration. Libraries were quality controlled and quantified using a Fragment Analyzer 5200 (Agilent Technologies) and a Qbit Flex Fluorometer (Thermo Fisher Scientific). CosMx SMI (Bruker Spatial Biology) and the Xenium Analyzer (10x Genomics) platforms were used. Samples were serially sectioned at curation sites (Site 1 and Site 2) and shipped promptly to minimize variations. The prescriptive details and step by step for each assay are outlined in the proceeding sections and can be found in the STSOPs. Samples were processed per CosMx slide preparation manuals MAN-10159-02 (prostate cancer, Adelaide) and MAN-10184-01-1 (all other tissues). Then, 5-µm sections were placed onto Superfrost Plus Micro Slides (VWR, 48311-703), dried, stored and then incubated overnight at 60 °C. Sections were dewaxed with xylene washes (or CitroSolv for normal tissues) and rehydrated through graded ethanol–water washes, followed by a final rinse in 1× PBS. Target Retrieval solution (CosMx FFPE slide preparation kit, RNA) was used in a pressure cooker at 100 °C for 15 minutes, followed by water, ethanol and air drying. The slides were then rinsed, fiducials (0.001%) applied and fixed in 10% NBF washed with NBF stop buffer and 1× PBS. Next, 100 mM NHS acetate was applied for 15 minutes, followed by 2× saline-sodium citrate (SSC) washes. The CosMx Human Universal Cell Characterization RNA Panel (CMX-H-USCP-1KP-R) targeting 950 human genes and a 50-target add-on panel set was denatured and cooled. A probe mix with RNase Inhibitor, Buffer R and nuclease-free water was applied for 17 hours at 37 °C. Colon, hepatocellular carcinoma (HCC) TMA, pancreas and prostate cancer samples received 4 µl of CD298/B2M antibody (MAN-10184-01-1), whereas appendix, ileum and prostate cancer received 8 µl of CD298/B2M antibody (MAN-10159-01-1). Samples were processed per CosMx SMI slide preparation manual MAN-10185-01-1 (Bruker Spatial Biology). Then, 5-µm FFPE sections were placed on Superfrost Plus Micro Slides, dried and stored. Slides were baked at 65 °C for 2 hours and then dewaxed with xylene (or CitriSolv) and rehydrated through graded ethanol–water washes, followed by a final rinse in 1× PBS. Pre-heated 1× Target Retrieval solution was used in a pressure cooker at 100 °C for 15 minutes, followed by 1× PBS washes. Incubation frames were attached, and Buffer W was applied for 1 hour in the dark. The Bruker Spatial Biology 64-plex human protein antibody panel was combined with CD298/B2M and PanCK/CD45 segmentation markers in Buffer W. Appendix, ileum and pancreas also received the CD68 segmentation marker. The primary antibody mix was incubated at 4 °C for 17 hours, followed by 1× TBS-T and 1× PBS washes. Sections were stained with nuclei stain, washed and incubated with 100 mM NHS acetate and a final 1× PBS wash. Pre-bleaching followed Configuration C and cell segmentation Configuration A. These procedures were uniformly followed at STJ, WCM and UoA. Breast cancer, normal appendix and normal ileum samples processed as part of Bruker Spatial Biology's Technology Access Program followed previously described methods41. Setup and scan acquisition followed the CosMx SMI Instrument User Manual (MAN-10161-02-1 or MAN-10161-03-2, software version 1.1; Bruker Spatial Biology). A new acquisition process was initiated via the CosMx SMI Control Center web interface. Slides were dried, placed in the Flow Cell Assembly Tool and secured. A new flow cell coverslip was aligned and affixed. Then, 2× SSC was introduced to hydrate the tissue. Flow cell configuration data (barcode and slide ID) were entered. Prostate sections were scanned with Configuration C for pre-bleaching and Configuration A for cell segmentation. Sections for Xenium were prepared according to the Xenium In Situ for FFPE − Tissue Preparation Guide (CG000578 Rev C; 10x Genomics) protocol. In brief, 5-µm sections were cut, floated in an RNAse-free water bath and carefully placed onto the 10.45-mm × 22.45-mm sample area of Xenium slides (10x Genomics, PN-3000941). The slides were then placed on a drying rack at room temperature for 30 minutes to remove excess water. This was followed by a 3-hour incubation at 42 °C on a Xenium Thermocycler Adapter plate, positioned atop the 96-well block of a C1000 Touch Thermal Cycler (Bio-Rad) with the lid open. Subsequently, the slides were stored overnight at room temperature with a desiccant to facilitate further drying. The slides were then processed following the Xenium In Situ for FFPE Deparaffinization and Decrosslinking protocol (CG000580 Rev C; 10x Genomics). The slides were placed again on a Xenium Thermocycler Adapter plate atop the 96-well block of a thermocycler with the lid open and incubated at 60 °C for 2 hours. The tissue sections were reverse crosslinked using a decrosslinking buffer containing tissue enhancer, urea and Perm Enzyme B at 80 °C for 30 minutes, followed by three washes with PBS-T. Slides were then immediately processed following the Xenium In Situ Gene Expression User Guide (CG000582 Rev E; 10x Genomics) for the remaining slide preparation steps. In brief, the predesigned gene expression probe set, Xenium Human Multi-Tissue and Cancer Panel (10X-1000626; 10x Genomics), targeting 377 human genes, was used for prostate cancer, ileum, appendix, pancreas and colon samples. For breast cancer samples, the Xenium Human Breast Panel (10X-1000463; 10x Genomics), targeting 380 human genes, was used. Panels were denatured at 95 °C for 2 minutes, crash cooled on ice for 1 minute and then equilibrated to room temperature before being added to the probe hybridization buffer and TE buffer to make the probe hybridization mix. Slides were washed three times with PBS-T for 1 minute each and then incubated with a post-hybridization wash buffer for 30 minutes at 37 °C. Slides were washed three times with PBS-T for 1 minute each and incubated with ligation mix for 2 hours at 37 °C. Three 1-minute PBS-T washes were followed by a 2-hour incubation at 30 °C with an amplification mix. Slides were then washed three times with TE buffer and then three washes with 1× PBS for 1 minute each, followed by incubation with Reducing Agent B for 10 minutes, washed once in 70% ethanol for 1 minute, washed twice in 100% ethanol and then treated with autofluorescence solution for 10 minutes in the dark. Slides were washed three times with 100% ethanol for 2 minutes each and then dried for 5 minutes at 37 °C before rehydrating with 1× PBS and then PBS-T for 2 minutes. Slides were stained with nuclei staining buffer for 1 minute at room temperature, followed by three washes with PBS-T for 1 minute each in the dark. The slides were then assembled into Xenium cassettes and loaded into a Xenium Analyzer for imaging and decoding. Breast cancer samples processed as part of the 10x Genomics Catalyst Program followed the methods previously described22; these experiments were carried out at 10x Genomics headquarters in Pleasanton, California. Sections from the prostate cancer samples were also subjected to Xenium multimodal segmentation through an early access program and processed at the 10x Genomics headquarters. In brief, the procedure for in situ gene expression with cell segmentation staining was conducted following the Demonstrated Protocol for the Xenium In Situ for FFPE Tissue Preparation Guide (CG000578 Rev C; 10x Genomics), the Xenium In Situ Gene Expression User Guide (CG000582 Rev F; 10x Genomics) and the Xenium In Situ Gene Expression Cell Segmentation User Guide (CG000749 Rev A; 10x Genomics). The combined protocol used the Xenium Human Multi-Tissue and Cancer Panel for in situ gene expression and the Xenium Cell Segmentation Staining Reagents add-on (10x Genomics, PN-1000661) for segmentation staining. This add-on kit contains a single cocktail of Xenium Multi-Tissue Stain Mix (10x Genomics, PN-2000991), which contains a cocktail of antibodies labeling the membranes (anti-ATP1A1/CD45/E-cadherin), antibodies labeling the cell interior (anti-αSMA/vimentin) and a universal interior label against ribosomal RNA (18S rRNA). Stain was then enhanced by the addition of Xenium Staining Enhancer Reagent, followed by treatment of the sections with Xenium Autofluorescence Mix (10x Genomics, PN-2000753) to diminish unwanted autofluorescence and enhance the SNR in the tissue sections. Subsequently, Xenium Nuclei Staining Buffer (10x Genomics, PN-2000762) was used to help the identification of tissues or regions of interest during the instrument's overview scan. Xenium slides, assembled within Xenium cassettes, were imaged using the Xenium Analyzer, in accordance with the guidelines specified in the Xenium Analyzer User Guide (CG000584 Rev D and Rev E for multimodal segmentation-stained slides; 10x Genomics). The Xenium Decoding Consumables Kit (10x Genomics, PN-1000487) was used for instrument loading. In brief, Xenium slide ID numbers and information on predesigned gene expression probes were input into the Analyzer. The necessary consumables and reagents were loaded into the instrument: reagent module B, thawed at 4 °C overnight and equilibrated to room temperature for 30 minutes prior to loading into the instrument; reagent module A, stored at 4 °C until loading in the instrument; instrument wash buffer (100% Milli-Q water); sample wash buffer A (1× PBS and 0.05% Tween 20); sample wash buffer B (100% Milli-Q water); and probe removal buffer (50% DMSO, 50 mM KCl and 0.1% Tween 20) were mixed prior to loading into the instrument with buffer caps along with pipette tip rack, extraction tip and objective wetting consumable. After loading, the sample was scanned to generate images of the fluorescently labeled nuclei in each section, which were used for FOV selection prior to scanning. Fresh PBS-T was applied to each slide/cassette, which was then covered and stored in the dark at 4 °C until post-run H&E staining. All procedures were standardized across three sites. The data were then acquired using the Xenium In Situ software suite (version 1.9, 10x Genomics), which is a set of software applications for analyzing and visualizing in situ gene expression data produced by the Xenium Analyzer. For prostate samples stained with multimodal cell segmentation reagents (10x Genomics headquarters, Site 4), both in situ gene expression and segmentation data were simultaneously collected and processed using the Xenium In Situ software suite version 2.0. After storage for up to 3 days, slides were washed three times in PBS-T for 1 minute each as recommended in the Technical Note: Post-Xenium In Situ Applications: Immunofluorescence, H&E and Visium CytAssist Spatial Gene Expression (CG000709 Rev A; 10x Genomics), removed from the Xenium cassette and placed into a wash jar of 1× PBS for 5 minutes as described in CosMx SMI Manual Slide Preparation for Protein Assays (MAN-10185-01-1; Bruker Spatial Biology). Then, incubation frames were attached to the Xenium slide, and tissue sections were covered with Buffer W and incubated at room temperature for 1 hour in the dark. The slides were dried for 15 minutes and then coverslipped using 1.5-mm-thick cover glass and Dako toluene-free mounting media (CS705). Slides were then washed by dipping into water several times to remove any glass shards. Next, the Demonstrated Protocol Xenium HE Staining (CG000613 Rev B; 10X Genomics) was followed from step 1.4 as described in post-Xenium H&E staining. Stained sections were covered using coverslips custom cut to fit inside the flow cell adhesive edges, using the glass scribe. All parameters were exact across three sites. To effectively search for datasets related to MERSCOPE, Xenium and CosMx, we used the GEO DataSets repository with tailored search terms for each platform within the specified date range. The data curation process for PUB datasets followed a structured pipeline to ensure completeness, consistency and usability for spatial transcriptomics analysis, comprising four key steps: We systematically identified and downloaded PUB datasets that met our inclusion criteria. To ensure a comprehensive dataset collection, specific search queries were used for each platform: These queries ensured that the dataset collection was comprehensive, platform specific and up to date. These queries ensured that the dataset collection was comprehensive, platform specific and up to date. To maintain data integrity and traceability, all datasets were structured into a one-folder-per-sample format. This organizational framework enabled efficient dataset comparison and computational processing. Rigorous validation was performed to ensure that only complete and high-quality datasets were retained. Datasets missing any of the required files were excluded from further processing. Additional data integrity checks were applied to filter out corrupted, incomplete or low-quality datasets that could affect reproducibility and technical comparability. To ensure uniform analysis across different datasets and technologies, all curated datasets were processed into a standardized format. This involved resolving annotation discrepancies, file format inconsistencies and structural variations. The resulting dataset repository was fully compatible with SpatialQM and the STP, facilitating seamless benchmarking and cross-platform comparisons. By implementing this structured four-step curation pipeline, we ensured that PUB datasets were accurate, well annotated and suitable for spatial transcriptomics analysis. The STP now provides researchers access to the largest curated public repository of imaging-based spatial transcriptomics datasets, enabling comparative analyses and benchmarking against the ST dataset. All datasets are accompanied by detailed metadata on sample origins, technical parameters and quality assessments, accessible via STP (https://spatialtouchstone.org/data-table) and documented in Supplementary Table 4. The transcript count matrices were imported from Cell Ranger version 8.0.0 outputs into Seurat version 5.0.120 (ref. Initial analysis involved identifying doublets using scDblFinder version 1.14.0 (ref. 43), with a preset doublet rate of 10% to approximate the expected doublet frequency when using the Single Cell Gene Expression Flex technology in a multiplexing setup (10x Genomics). Analysis of unique molecular identifiers (UMIs) and features counts distributions identified the presence of droplets with low counts. These populations, confirmed to lack distinct expression patterns, were excluded from subsequent analyses. Further quality control steps removed nuclei either marked as doublets or having fewer than 1,000 genes detected, ensuring that key cell types were retained as evidenced by comparison with the original dataset. Normalization and feature selection was performed using SCTransform44 employing default parameters. The dataset was then subjected to dimensionality reduction through a PCA by selecting the first 30 principal components to guide the generation of a uniform manifold approximation and projection (UMAP) embedding (RunUMAP) and facilitate Louvain clustering. Cell type annotations were transferred from a reference single-cell RNA sequencing (scRNA-seq) dataset of human prostate tissue45 using Seurat's label transfer function. This involved identifying dataset anchors based on similarity in the first 30 principal components of the reference's PCA embedding. To ensure a fair comparison, cells without detected transcripts were excluded from the analysis following the Xenium Onboard Analysis or the AtoMx processing workflow. Subsequent analyses, including cell type annotation, were conducted using raw transcript counts. For visualization, data were normalized using SCTransform, followed by PCA and subsequent UMAP embedding generation using the first 30 principal components. Visualization of transcript localization and segmentation boundaries was achieved using Seurat's ImageDimPlot() function. This methodical approach ensures a robust and comparative analysis of data across different platforms. All code employed for generating these metrics is contained within an open-source software, SpatialQM. Below are detailed descriptions of each metric: This metric is generally stable due to the robustness of DAPI staining for nuclear segmentation, although delineation issues can arise in densely nucleated areas. Established to determine assay specificity, this is calculated as a percentage: the product of the sum of all non-specific probes relative to the total expression of each target probe, and the sum of non-specific probes is divided by the number of genes and non-specific probes and then multiplied by a scaling factor of 0.01. This specificity metric is computed using the following formula: It is a metric calculated to evaluate the robustness of gene expression against background noise. This involves two matrices: ‘noise', which contains expression data from negative control probes across all cells, and ‘exp', which includes expression data for all other genes. This method isolates the signal contribution of each gene, distinguishing it from background variations. This metric is computed using the following formula: This metric represents the percentage of total transcripts assigned to cells after segmentation. An FTC value of 0.8 indicates that 80% of the transcripts in the samples are assigned to cells. This is a measure of cell segmentation quality, assessing the rate at which genes, which are expected to be exclusive, occur within the same cell9. For example, a MECR value of 0.01 means that, on average, 1% of cells co-express genes that should be mutually exclusive. This metric quantifies the uncertainty or randomness in an experiment using Shannon entropy46. This metric measures how many genes account for at least 50% of total counts in the sample. It is calculated by first summing the entire expression matrix to obtain a variable total_sum. Next, each gene per cell is summed to create another variable, row_sums. The cumulative sum of row_sums is then computed. The ‘complexity' value is normalized by the panel size to ensure comparability across different platforms, which may vary significantly in panel size. Libraries were demultiplexed with Cell Ranger's (version 7.2.0) ‘cellranger multi' function with default parameters aligning to the Chromium Human Transcriptome Probe Set version 1.0.1. UMI count matrices for each sample were minimally preprocessed with Seurat (version 5.1.0)42. A nearest neighbor graph was constructed using FindNeighbors() on the first 30 principal components with the dims = 1:30 parameter. Louvain clustering was then performed with a varying resolution depending on the segregation of distinct cell type expression patterns. Annotations were also informed and cross-referenced to public reference data where available. To provide additional confidence of annotations, top markers of each annotated cluster are shown in Supplementary Fig. Spatial transcriptomics data were annotated using the supervised annotation method implemented by InSituType24 with the matched snPATHO-seq data as a reference. Specifically, cell type references were generated from the pseudobulk of UMI counts from each annotated cluster of the snPATHO-seq data. Annotation was then performed with the insitutypeML() function. Then, log-transformed transcript counts and tissue coordinates for cells were processed with the Seurat wrapper function RunBanksy() with lambda = 0.8 to allow for spatial clustering and k_geom = 30. RunPCA() was run on the resulting data to generate an embedding for FindNeighbors (dims = 1:30) and FindClusters (resolution = 0.15). Protein flat files were exported using the Bruker Spatial Biology CosMx Flat File Export Custom Module version 1.1.1. The expression matrix was then filtered to remove cells where total fluorescence expression was more than 2 s.d. below the mean, identifying these as noise. The remaining data are then normalized through a centered log-ratio transformation, where x represents the expression matrix, and n represents the number of cells. For downstream analysis, rapids_singlecell version 0.9.2, which leverages GPU acceleration, is employed. The analysis includes PCA using 50 principal components, followed by a nearest neighbor search with 30 neighbors and 50 principal components. Dimensionality reduction was performed using UMAP, and clustering was conducted using the Leiden clustering method with a resolution of 1.0, resulting in 14 different clusters. 48) and plotted using Scanpy's dot plot function. This metric measures the average fluorescence level and reflects the relative amount of protein being detected within the cell, with a value of 5,000 indicating that each cell has a fluorescence measured at this level. It is calculated by counting the number of values 2 s.d. A sparsity value of 1 means that all the values in the matrix are equal to 0. Conversely, a sparsity value of 0 means that there are no values 2 s.d. This can help understand the overall coverage of the target panel in the tissue type used and on algorithms to use when performing downstream analysis. This metric quantifies how many genes account for 50% of the total fluorescence in the sample. A complexity value of 30 indicates that 30 genes are responsible for half of all the transcripts in the sample. A normalization by panel size makes this metric comparable between platforms with high variance in panel sizes. Available on GitHub at https://github.com/Center-for-Spatial-OMICs/SpatialQM, the package includes detailed vignettes, markdowns, predefined input file structures and complete source code, enabling users to replicate analyses. Additionally, the software supports seamless integration of single-cell annotations with spatial transcriptomic data outputs. STP is a user-friendly interface for researchers to compare their samples against the SpatialQM repository (n = 254). Within the STP, users can assess the spatial quality metrics derived from this study through visualizations of plots filtered by preferred parameters, including tissue type (for example, breast and brain), platform (for example, CosMx, Xenium and MERSCOPE) and others. For a more focused exploration of the database, we included a search function under the ‘Query by ID' tab on STP that allows users to examine how the queried sample performs against the database. All sample IDs from our ST database as well as general metadata can be accessed on the ‘Dataset' page under the ‘Database Explorer' tab. To enable researchers to compare their samples against both our technical metric database and the dynamic user-submitted technical metrics database, users can upload their CosMx or Xenium flat files through the ‘Submit Sample' tab on STP. We also included a video tutorial to guide users through the usage of our website as they explore our database. The tutorial can be accessed under the ‘Tutorial' tab on STP or on YouTube (https://www.youtube.com/watch?v=I_ooOqggoUg). Detailed metadata for the ST dataset used in this study can be found in Supplementary Table 4, which includes relevant identifiers such as GEO accession numbers. Users can register to receive a detailed HTML report via email, comparing the technical metrics from this study to their sample of interest. The portal, designed for academic use, includes necessary disclaimers regarding its usage. These tools aim to enhance the accessibility and replicability of spatial transcriptomics analyses, facilitating robust comparative studies across a wide array of samples. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw data files and processed data matrix can be accessed via the GEO (GSE277080)64 and via Zenodo (https://zenodo.org/communities/saha)65. Source data are provided with this paper. The inclusion of all necessary metadata and code to understand, evaluate and replicate this study can be found at GitHub (https://github.com/Center-for-Spatial-OMICs/SpatialQM)66 and in Supplementary Tables 1–5. Jackson, K. C. & Pachter, L. A standard for sharing spatial transcriptomics data. Covert, I. et al. Predictive and robust gene selection for spatial transcriptomics. Du, J. et al. Advances in spatial transcriptomics and related data analysis strategies. Systematic comparison of sequencing-based spatial transcriptomic methods. & Satija, R. Comparative analysis of multiplexed in situ gene expression profiling technologies. Cook, D. P. et al. A comparative analysis of imaging-based spatial transcriptomics platforms. Comparison of spatial transcriptomics technologies using tumor cryosections. Comparison of imaging based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed paraffin-embedded tumor samples. Comparison of spatial transcriptomics technologies across six cancer types. Yeo, Y. Y. et al. Same-slide spatial multi-omics integration reveals tumor virus-linked spatial reorganization of the tumor microenvironment. Salit, M. & Woodcock, J. MAQC and the era of genomic medicine. The MicroArray Quality Control (MAQC) project shows inter- and intraplatform reproducibility of gene expression measurements. Chen, W. et al. A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples. Heumos, L. et al. Best practices for single-cell analysis across modalities. Negi, S. K. & Guda, C. Global gene expression profiling of healthy human brain and its application in studying neurological disorders. Moses, L. et al. Voyager: exploratory single-cell genomics data analysis with geospatial statistics. Janesick, A. et al. High resolution mapping of the tumor microenvironment using integrated single-cell, spatial and in situ analysis. Wang, T. et al. snPATHO-seq, a versatile FFPE single-nucleus RNA sequencing method to unlock pathology archives. Robust decomposition of cell type mixtures in spatial transcriptomics. Nguyen, V. & Griss, J. scAnnotatR: framework to accurately classify cell types in single-cell RNA-sequencing data. Yang, S. & Zhou, X. SRT-Server: powering the analysis of spatial transcriptomic data. Cutler, K. J. et al. Omnipose: a high-precision morphology-independent solution for bacterial cell segmentation. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. Schmidt, U., Weigert, M., Broaddus, C. & Myers, G. Cell detection with star-convex polygons. In Medical Image Computing and Computer Assisted Intervention – MICCAI 2018: 21st International Conference, Proceedings, Part II (eds Frangi, A. F. et al.) 11071, 265–273 (Springer International Publishing, 2018). Nuclei instance segmentation and classification in histopathology images with Stardist. Si, Y. et al. FICTURE: scalable segmentation-free analysis of submicron resolution spatial transcriptomics. Dimitrov, D. et al. LIANA+ provides an all-in-one framework for cell–cell communication inference. Chen, X. et al. Benchmarking algorithms for spatially variable gene identification in spatial transcriptomics. Yang, A. et al. starmapVR: immersive visualisation of single cell spatial omic data. Amblard, E. et al. A robust workflow to benchmark deconvolution of multi-omic data. Wu, S. Z. et al. A single-cell and spatially resolved atlas of human breast cancers. STAMP: single-cell transcriptomics analysis and multimodal profiling through imaging. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Choudhary, S. & Satija, R. Comparison and evaluation of statistical error models for scRNA-seq. Resolving the immune landscape of human prostate at a single-cell level in health and cancer. & Reyes-Valdés, M. H. Defining diversity, specialization, and gene specificity in transcriptomes through information theory. Singhal, V. et al. BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Epstein, A. et al. Subventricular zone stem cell niche injury is associated with intestinal perforation in preterm infants and predicts future motor impairment. Androvic, P. et al. Spatial Transcriptomics-correlated Electron Microscopy maps transcriptional and ultrastructural responses to brain injury. Benjamin, K. et al. Multiscale topology classifies cells in subcellular spatial transcriptomics. Dennis, D., Wang, B. S., Karamboulas, K., Kaplan, D. R. & Miller, F. D. Single-cell approaches define two groups of mammalian oligodendrocyte precursor cells and their evolution over developmental time. Kedia, S. & Ji, H. T cell-mediated microglial activation triggers myelin pathology in a mouse model of amyloidosis. Mallach, A. et al. Microglia-astrocyte crosstalk in the amyloid plaque niche of an Alzheimer's disease mouse model, as revealed by spatial transcriptomics. Mangoli, A. et al. Disruption of ataxia telangiectasia–mutated kinase enhances radiation therapy efficacy in spatially directed diffuse midline glioma models. Schott, M. et al. Open-ST: high-resolution spatial transcriptomics in 3D. Tahmasian, N. et al. Neonatal brain injury triggers niche-specific changes to cellular biogeography. In situ single-cell profiling sheds light on IFI27 localisation during SARS-CoV-2 infection. Trigo, A. et al. Macrophage and neutrophil heterogeneity at single-cell spatial resolution in human inflammatory bowel disease. Vannan, A. et al. Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis. Watson, S. et al. Fibrotic response to anti-CSF-1R therapy potentiates glioblastoma recurrence. Ziegler, K. et al. Immune-mediated denervation of the pineal gland underlies sleep disturbance in cardiac disease. Dezem, F. et al. Standardized metrics for assessment and reproducibility of imaging-based spatial transcriptomics datasets. Dezem, F. et al. Standardized metrics for assessment and reproducibility of imaging-based spatial transcriptomics datasets. Dezem, F. et al. Standardized metrics for assessment and reproducibility of imaging-based spatial transcriptomics datasets. L.G.M., via the South Australian immune GENomics Cancer Institute (SAiGENCI), received grant funding from the Australian Government. A.S. is supported by the generosity of Deborah and John McMurtrie and the Petre Foundation and grants from the Breast Cancer Research Foundation (BCRF-23-209), the National Health and Medical Research Council (APP2018440), the National Breast Cancer Foundation (IIRS-23-074) and Cancer Council NSW (RG 20-09). thanks WorldQuant and GI Research Foundation (GIRF), NASA (80NSSC22K0254, 80NSSC24K0728, 80NSSC24K1052), the National Institutes of Health (P01CA272295, R01CA266279, U54AG089334), the UK Cancer Grand Challenges (SAMBAI), Blood Cancer United (MCL7001-18, LLS 9238-16, 7029-23/22, 7037-25), Boryung and Bumrungrad International Hospital. These authors contributed equally: Jasmine T. Plummer, Felipe Segato Dezem, David P. Cook, Jiwoon Park. These authors jointly supervised this work: Jasmine T. Plummer, Christopher E. Mason, Luciano G. Martelotto. Center for Spatial Omics, St. Jude Children's Research Hospital, Memphis, TN, USA Jasmine T. Plummer, Felipe Segato Dezem, Luke Zhang, Yutian Liu, Maycon Marção, Hannah DuBose, Arjumand Wani & Natalia Morosini Department of Developmental Neurobiology, St. Jude Children's Research Hospital, Memphis, TN, USA Jasmine T. Plummer, Felipe Segato Dezem, Luke Zhang, Yutian Liu, Maycon Marção, Hannah DuBose, Arjumand Wani & Natalia Morosini Department of Cellular and Molecular Biology, St. Jude Children's Research Hospital, Memphis, TN, USA Comprehensive Cancer Center, St. Jude Children's Research Hospital, Memphis, TN, USA Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medicine, New York, NY, USA The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA, USA Kellie Wise, Michael Roach, Kirk B. Jensen & Luciano G. Martelotto Kellie Wise, Michael Roach, Kirk B. Jensen, Natalie Ryan, Lisa M. Butler & Luciano G. Martelotto Kellie Wise, Michael Roach, Kirk B. Jensen & Luciano G. Martelotto Cancer Ecosystems Program, Garvan Institute of Medical Research, Darlinghurst, New South Wales, Australia Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY, USA Roberto De Gregorio, Alicia Alonso & Shauna Lee Houlihan Department of Pathology, Weill Cornell Medicine, New York, NY, USA Single-cell Analytics Innovation Lab, Memorial Sloan Kettering Cancer Center, New York, NY, USA Solid Tumour Program, Precision Cancer Medicine Theme, South Australian Health and Medical Research Institute, Adelaide, South Australia, Australia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and D.C. led the computational analyses in this study, together with M.M., L.Z. also coordinated the metadata and repository upload. Correspondence to Jasmine T. Plummer, Christopher E. Mason or Luciano G. Martelotto. serves on the Scientific Advisory Board of Cellanome, Inc. Nature Biotechnology thanks Andrea Radtke and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. All dataset metrics hosted within the STP. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Standardized metrics for assessment and reproducibility of imaging-based spatial transcriptomics datasets. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03912-w'>The ‘silent' brain cells that shape our behaviour, memory and health</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 09:35:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Alison Abbott is a writer in Munich, Germany. Under a light microscope, brain cells called astrocytes look star-shaped. Neurons were the main players, they thought, and everything else was made up of uninteresting support systems. By the 2010s, memory researcher Inbal Goshen was beginning to question that assumption. She was inspired by innovative molecular tools that would allow her to investigate the contributions of another, more mysterious group of cells called astrocytes. What she discovered about their role in learning and memory excited her even more. The computers that run on human brain cells The computers that run on human brain cells She imagined colleagues thinking, “Oh, that's the weird one who works on astrocytes,” says Goshen, whose laboratory is at the Hebrew University of Jerusalem. A lot of people were sceptical, she says. A rush of studies from labs in many subfields are revealing just how important these cells are in shaping our behaviour, mood and memory. “Neurons and neural circuits are the main computing units of the brain, but it's now clear just how much astrocytes shape that computation,” says neurobiologist Nicola Allen at the Salk Institute for Biological Studies in La Jolla, California, who has spent her career researching astrocytes and other non-neuronal cells, collectively called glial cells. As far back as the nineteenth century, scientists could see with their simple microscopes that mammalian brains included two major types of cell — neurons and glia — in roughly equal numbers. Twentieth-century technologies generated most of the excitement around neurons. Researchers studying the cells' electrical activity showed how they create the complex networks that underlie all brain functions. When neurons are activated, electrical signals zap down their length at lightning speed, causing their synapses to release chemical neurotransmitters. Some of these, such as glutamate, excite neighbouring neurons, whereas others, such as GABA (γ-aminobutyric acid), inhibit them. The development of a technique called patch clamping in the 1970s and 1980s, in which electrodes are inserted into individual cells to measure the flow of ions across membranes, let researchers probe this neurotransmission in unprecedented detail. Inside the push to build better brain models Inside the push to build better brain models Some glia, called oligodendrocytes, coat and insulate neurons. The many functions of astrocytes proved more elusive. Although few brain scientists focused on astrocytes until well into the 2000s, some fundamental discoveries were made before that1. They clear up waste products from around the synapses and regulate ion levels there. Those measurements revealed that astrocytes use fluctuations in their calcium levels to signal to each other and to other cells in response to certain molecules in their environment, such as an excess of neurotransmitters (see ‘Signals from the stars'). These calcium signals, which move slowly compared with those that pass between neurons, turn out to have big consequences. They drive many activities, including the release of further signalling molecules, ions, metabolites and other factors that affect how neurons, other glial cells and blood vessels behave. Because many of the signalling molecules are neurotransmitters, a small group of researchers in the 2010s began to wonder whether astrocytes might contribute to ultrafast electrical transmission in neurons. But over the past decade or so, ever-more-precise research tools have revealed a different, more complex story. Many scientists who had previously focused exclusively on neurons are now using these tools to investigate how astrocytes contribute to animal physiology and behaviour. No evidence emerged to support a role for astrocytes in ultrafast neurotransmission. Instead, it is becoming ever clearer that astrocytes orchestrate the molecular mix in the environment around synapses on timescales much slower than neuronal signalling, varying that mix according to brain state — how alert or awake the brain is, for example. Advances in microscopy have shown just how closely astrocytes are associated with neurons. Beyond their star-shaped core, astrocytes have an intricate architecture with many branches that extend into tiny, delicate structures, called leaflets, that are only tens of nanometres wide. At this resolution, astrocytes, which make up one-quarter of all brain cells, look less like stars and more like bushy spheres, and they fill all the available space between neurons without overlapping with each other2. In the human brain, each astrocyte can contact up to two million synapses. First-ever atlas of brain development shows how stem cells turn into neurons First-ever atlas of brain development shows how stem cells turn into neurons This vast reach of astrocytes has profound consequences in the brain, says neuroscientist Baljit Khakh at the University of California, Los Angeles. “Form follows function in biology.” His lab has developed many of the molecular and genetic tools that allow particular calcium signalling pathways in astrocytes to be switched on or off, letting scientists see what each pathway does. Neurons might transmit the signals that drive brain function, but it is now clear that astrocytes fine-tune those signals by altering the environment around the synapses, says neuroscientist Hongkui Zeng, director of the Allen Institute for Brain Science in Seattle, Washington. As just one example, astrocytes resolved an open question in the field of circadian biology. How does the body's master clock, a cluster of cells in the brain called the suprachiasmatic nucleus (SCN) that regulates biological cycles in a roughly 24-hour rhythm, keep its circular loop? “It was a mystery,” says circadian biologist Michael Hastings at the Laboratory of Molecular Biology in Cambridge, UK. The master-clock system is made up almost exclusively of neurons that release the inhibitory neurotransmitter GABA, and no one could see how a system that apparently does nothing but suppress neuronal activity could create a daily cycle. Biological clocks need a feedback mechanism so that they can generate rhythmic cycles autonomously, without needing an external signal every day to restart them. “I told Marco it would be a waste of time, because there wouldn't be any,” recalls Hastings. Brancaccio, who is now at Imperial College London, found plenty of glutamate — and to general astonishment, found that its levels were as rhythmic as those of GABA. Where could that glutamate be coming from?” says Hastings. It didn't take them long to discover through a literature search that the source must be astrocytes, a cell type they hadn't thought much about before. This set the researchers on a series of ever-more sophisticated experiments. They ultimately concluded that astrocytes support the SCN clock by switching on their uptake systems for GABA by day and turning them off at night4,5. Different astrocyte types, with their thousands of branches and leaflets, pack together to fill the brain.Credit: B. Khakh Laboratory In one of her first key experiments with the new tools, Goshen and her team were amazed to discover how astrocytes in the brains of mice encode information about the spatial location of rewards. When a mouse had already learnt where to find a water reward, the calcium activity in its astrocytes gradually increased as it approached the reward, the scientists found. Earlier this year, groups in Japan and the United States reported that astrocytes help with the stabilization and recall of memories primed by fear7,8. Because the astrocytes' signals are much slower than the electrical signals of neurons — sometimes developing over hours or even days rather than milliseconds — they are well suited to bridging the time gap between learning and remembering, says neuroscientist Jun Nagai at the RIKEN Centre for Brain Science in Wako, Japan, who led one of the studies. “Think of them as the brain's long-exposure camera: they capture the trace of meaningful events that might otherwise fade too fast.” Because neurons and astrocytes work together to process information, researchers have started to wonder whether astrocytes drive or exacerbate diseases that have been generally considered to be neuronal. The computers that run on human brain cells Inside the push to build better brain models First-ever atlas of brain development shows how stem cells turn into neurons A brain implant that could rival Neuralink's enters clinical trials TSC tunes progenitor balance and upper-layer neuron generation in neocortex Progressive coevolution of the yeast centromere and kinetochore Ethylene modulates cell wall mechanics for root responses to compaction Inhibitors supercharge kinase turnover through native proteolytic circuits The Intramural Research Program (IRP), National Institute on Drug Abuse (NIDA) Postdoctoral fellows sought to study neurodegenerative diseases, cancer, protein quality control, aging, metabolism, autophagy, and stem cells. We welcome applicants with research experience in academia, industry, or government who can bring all perspectives and expertise to our team. The computers that run on human brain cells Inside the push to build better brain models First-ever atlas of brain development shows how stem cells turn into neurons An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251203010207.htm'>Early Earth's sky may have created the first ingredients for life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 06:51:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Earth's ancient sky may have played a larger role in the beginnings of life than scientists once believed. According to a study published Dec. 1 in the Proceedings of the National Academy of Sciences, researchers from CU Boulder and their collaborators report that billions of years ago, the young planet's atmosphere may have been generating sulfur-based molecules that are known today as important components for life. This discovery challenges the long-standing idea that these sulfur molecules formed only after life had already taken hold on Earth. Sulfur, much like carbon, is a vital element found in every form of life, from bacteria to humans. It appears in certain amino acids, which serve as the basic building blocks of proteins. Although sulfur was present in the early atmosphere, most scientists believed that organic sulfur molecules, such as amino acids, arose only after living organisms were already present and producing them. When these molecules did appear, they formed only under unusual or highly specific conditions that were unlikely to have been common across the planet. This indicated that the molecule might appear even on worlds without life. They illuminated a mixture of methane, carbon dioxide, hydrogen sulfide, and nitrogen to recreate atmospheric conditions from before life emerged. Using a very sensitive mass spectrometer to identify and measure chemical compounds, the researchers discovered that their early Earth simulation produced a wide range of sulfur biomolecules. These included the amino acids cysteine and taurine, along with coenzyme M, which plays a key role in metabolism. The team then estimated how much cysteine an entire ancient atmosphere might generate. It might be enough for a budding global ecosystem, where life is just getting started," Reed said. "Life probably required some very specialized conditions to get started, like near volcanoes or hydrothermal vents with complex chemistry," Browne said. "We used to think life had to start completely from scratch, but our results suggest some of these more complex molecules were already widespread under non-specialized conditions, which might have made it a little easier for life to get going." Scientists Spot Hidden Water Ice on Mars: A Game-Changer for Human Missions New Theory Suggests We've Been Wrong About Black Holes for 60 Years Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-66803-8'>Non-small cell lung cancer molecular subtypes and vulnerability to immunotherapy treatment combinations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 04:53:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. The phase 3 IMpower150 trial in treatment-naïve patients with metastatic non-small-cell lung cancer (NSCLC) demonstrates significantly longer progression-free (PFS) and overall survival (OS) with first-line atezolizumab (anti-PD-L1)-bevacizumab (anti-VEGF)-carboplatin-paclitaxel (ABCP) than with bevacizumab-carboplatin-paclitaxel (BCP). We characterise four molecular NSCLC subtypes identified by unsupervised clustering of transcriptomes of 564 pre-treatment primary tumour samples from IMpower150 using non-negative matrix factorization (NMF1-4). Each subtype has distinct tumour PD-L1 expression levels, epithelial characteristics, immune composition, and treatment outcomes. Both NMF2 (enriched in tumour proliferation signal, macrophages, and monocytes) and NMF4 (enriched in B cells and T cells) have elevated tumour PD-L1 expression. Of these two, only NMF4 demonstrates PFS and OS benefits with ABCP versus either BCP or atezolizumab-carboplatin-paclitaxel (ACP). Patients with NMF1 (enriched in basal and squamous-like cells) have improved outcomes on ABCP compared with ACP or BCP; those with NMF3 (enriched in adenocarcinoma signatures) show similar outcomes among treatments. These insights could help inform individualised first-line treatment for metastatic NSCLC. The processed RNA-seq data and associated patient-level clinical data, are available in the EGA database under accession code [EGAS50000001272, https://ega-archive.org/studies/EGAS50000001272]. Data will be made available to qualified researchers via the European Genome-Phenome Archive (EGA). Access requires submission of a request through the standard EGA process and approval by the Genentech Data Access Committee. Approved researchers must enter into a Data Access Agreement, which enforces terms on patient privacy, data security, and use consistent with informed consent and applicable data protection laws. Under the current agreement, qualified researchers may access the data for one year for purposes aligned with these terms. Agreement provisions may evolve over time in line with changes in privacy regulations and technology. The remaining data are available within the Article, Supplementary Information or Source Data. Source data are provided with this paper. For up-to-date details on Roche's Global Policy on the Sharing of Clinical Information and how to request access to related clinical study documents, see https://go.roche.com/data_sharing. Anonymised records for individual patients across more than one data source external to Roche cannot, and should not, be linked due to a potential increase in risk of patient re-identification. Source data are provided with this paper. Siegel, R. L., Miller, K. D., Wagle, N. S. & Jemal, A. Sung, H. et al. GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. Molina, J. R., Yang, P., Cassivi, S. D., Schild, S. E. & Adjei, A. A. Non-small cell lung cancer: epidemiology, risk factors, treatment, and survivorship. Tang, S. et al. Immune checkpoint inhibitors in non-small cell lung cancer: progress, challenges, and prospects. & Hodi, F. S. Molecular pathways: next-generation immunotherapy—inhibiting programmed death-ligand 1 and programmed death-1. Sun, C., Mezzadra, R. & Schumacher, T. N. Regulation and function of the PD-L1 checkpoint. Herbst, R. S. et al. Atezolizumab for first-line treatment of PD-L1–selected patients with NSCLC. Mazieres, J. et al. Atezolizumab versus docetaxel in pretreated patients with NSCLC: final results from the randomized phase 2 POPLAR and phase 3 OAK clinical trials. Socinski, M. A. et al. Atezolizumab for first-line treatment of metastatic nonsquamous NSCLC. Socinski, M. A. et al. IMpower150 final overall survival analyses for atezolizumab plus bevacizumab and chemotherapy in first-line metastatic nonsquamous NSCLC. Brueckl, W. M., Ficker, J. H. & Zeitler, G. Clinically relevant prognostic and predictive markers for immune-checkpoint-inhibitor (ICI) therapy in non-small cell lung cancer (NSCLC). Gandara, D. R. et al. Blood-based tumor mutational burden as a predictor of clinical benefit in non-small-cell lung cancer patients treated with atezolizumab. Patil, N. S. et al. Intratumoral plasma cells predict outcomes to PD-L1 blockade in non-small cell lung cancer. Genova, C. et al. Therapeutic implications of tumor microenvironment in lung cancer: focus on immune checkpoint blockade. Rakaee, M. et al. Association of machine learning-based assessment of tumor-infiltrating lymphocytes on standard histologic images with outcomes of immunotherapy in patients with NSCLC. Cancer Genome Atlas Research Network Comprehensive molecular profiling of lung adenocarcinoma. Roh, W. et al. High-resolution profiling of lung adenocarcinoma identifies expression subtypes with specific biomarkers and clinically relevant vulnerabilities. Gavish, A. et al. Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours. Huang, R. S. P. et al. Pan-cancer landscape of CD274 (PD-L1) copy number changes in 244 584 patient samples and the correlation with PD-L1 protein expression. Murugesan, K. et al. Association of CD274 (PD-L1) copy number changes with immune checkpoint inhibitor clinical benefit in non-squamous non-small cell lung cancer. Finn, R. S. et al. Atezolizumab plus bevacizumab in unresectable hepatocellular carcinoma. Skoulidis, F. et al. STK11/LKB1 mutations and PD-1 inhibitor resistance in KRAS-mutant lung adenocarcinoma. Ren, F. et al. Tertiary lymphoid structures in lung adenocarcinoma: characteristics and related factors. Tertiary lymphoid structures as a predictive biomarker of response to cancer immunotherapies. Horvath, L. et al. Beyond binary: bridging neutrophil diversity to new therapeutic approaches in NSCLC. Mariathasan, S. et al. TGFβ attenuates tumour response to PD-L1 blockade by contributing to exclusion of T cells. McDermott, D. F. et al. Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma. Zhu, A. X. et al. Molecular correlates of clinical response and resistance to atezolizumab in combination with bevacizumab in advanced hepatocellular carcinoma. Motzer, R. J. et al. Molecular subsets in renal cancer determine outcome to checkpoint and angiogenesis blockade. Hendriks, L. E. et al. Non-oncogene-addicted metastatic non-small-cell lung cancer: ESMO Clinical Practice Guideline for diagnosis, treatment and follow-up. Mo, D.-C. et al. The role of PD-L1 in patients with non-small cell lung cancer receiving neoadjuvant immune checkpoint inhibitor plus chemotherapy: a meta-analysis. Barry, S. T., Gabrilovich, D. I., Sansom, O. J., Campbell, A. D. & Morton, J. P. Therapeutic targeting of tumour myeloid cells. Brunet, J.-P., Tamayo, P., Golub, T. R. & Mesirov, J. P. Metagenes and molecular pattern discovery using matrix factorization. Wilkerson, M. D. et al. Differential pathogenesis of lung adenocarcinoma subtypes involving sequence mutations, copy number, chromosomal instability, and methylation. This work was supported by F. Hoffmann-La Roche Ltd /Genentech, Inc, a member of the Roche Group. We thank the patients and their families and the IMpower150 study team and investigators. Medical writing assistance with preparation of the manuscript was provided by Samantha Santangelo, PhD, of Nucleus Global, an Inizio Company, and funded by Roche. These authors contributed equally: Minu K. Srivastava, Barzin Y. Nabet. Genentech Inc, South San Francisco, CA, USA Tianshi Lu, Habib Hamidi, Sören Müller, Assaf Amitai, Xiangnan Guan, Eloisa Fuentes, Hartmut Koeppen, Jennifer M. Giltnane, David S. Shames, Marcus Ballinger, Meng Xiao He, Yulei Wang, Minu K. Srivastava & Barzin Y. Nabet AdventHealth, Orlando, FL, USA Airway Research Center North, German Center for Lung Research, LungenClinic Grosshansdorf, Grosshansdorf, Germany Istituto Nazionale Tumori Regina Elena, Rome, Italy Aix-Marseille University, APHM, Marseille, France Rocky Mountain Cancer Center, Lone Tree, CO, USA F. Hoffmann-La Roche Ltd, Mississauga, ON, Canada Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Writing – Original Draft: T.L., D.S.S., M.K.S., and B.Y.N. Writing – Review and Editing: All authors. Correspondence to Minu K. Srivastava or Barzin Y. Nabet. declares grants or contracts from AstraZeneca, BeOne, Enliven, Genentech, and Lilly; consulting fees from BMS, Lilly, Spectrum, and Summit; payment or honoraria from AstraZeneca, Guardant, Janssen, Jazz, and Merck/MSD; and participation on a data safety monitoring board or advisory board for BMS and Summit. declares consulting fees from Amgen, AstraZeneca, Beigene, Boehringer-Ingelheim, BMS, Daiichi-Sankyo, GSK, Janssen, Lilly, Merck/MSD, Mirati, Novartis, Pfizer, Regeneron, Roche, and Sanofi; payment or honoraria from Amgen, AstraZeneca, Beigene, Boehringer-Ingelheim, BMS, Daiichi-Sankyo, GSK, Janssen, Lilly, Merck/MSD, Mirati, Novartis, Pfizer, Regeneron, Roche, and Sanofi; support for attending meetings and/or travel from Amgen, AstraZeneca, Beigene, Boehringer-Ingelheim, and BMS; and participation on a data safety monitoring board or advisory board for Daiichi-Sankyo and Sanofi. declares grants or contracts from AbbVie, Amgen, AstraZeneca, Bayer, Beigene, BMS, Galecto, Illumina, Lilly, Merck/MSD, Mirati, Novocure, OSE, Pfizer, Pharmamar, Regeneron, Roche, Sanofi, Summit Therapeutics, Takeda, and Thermofisher; consulting fees from AbbVie, Amgen, AstraZeneca, Bayer, Beigene, BMS, Galecto, Illumina, Lilly, Merck/MSD, Mirati, Novocure, OSE, Pfizer, Pharmamar, Regeneron, Roche, Sanofi, Summit Therapeutics, Takeda, and Thermofisher; payment or honoraria from AbbVie, Amgen, AstraZeneca, Bayer, Beigene, BMS, Galecto, Illumina, Lilly, Merck/MSD, Mirati, Novocure, OSE, Pfizer, Pharmamar, Regeneron, Roche, Sanofi, Summit Therapeutics, Takeda, and Thermofisher; and participation on a data safety monitoring board or advisory board for AbbVie, Amgen, AstraZeneca, Bayer, Beigene, BMS, Galecto, Illumina, Lilly, Merck/MSD, Mirati, Novocure, OSE, Pfizer, Pharmamar, Regeneron, Roche, Sanofi, Summit Therapeutics, Takeda, and Thermofisher. declares participation on a data safety monitoring board or advisory board for AbbVie, ACEA, Amgen, AstraZeneca, Bayer, Boehringer-Ingelheim, BMS, Eli Lilly Oncology, Eisai, F. Hoffmann–La Roche Ltd, Genentech, Ignyta, Innate Pharma, Ipsen, Loxo, MedImmune, Merck/MSD, Novartis, Pfizer, Pierre Fabre, Sanofi-Aventis, Summit Therapeutics, and Takeda; payments made to their institution. declares receiving personal fees and advisory board participation with Mirati Therapeutics, Inc. E.F. declares consulting fees from Genentech. declares royalties or licenses from UTSW Medical Center, stock in BeOne Medicines, and was an employee of Genentech at the relevant time of this study. declare they are employees of Genentech/Roche and hold Roche stock. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Non-small cell lung cancer molecular subtypes and vulnerability to immunotherapy treatment combinations. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251202052215.htm'>Is your gut being poisoned? Scientists reveal the hidden impact of everyday chemicals</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 04:51:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These chemicals interfere with the growth of microbes considered essential for maintaining good health. The gut microbiome contains an estimated 4,500 different types of bacteria that work together to support bodily functions. When this system becomes imbalanced, it can contribute to a wide range of health problems involving digestion, weight regulation, the immune system, and mental health. Standard safety evaluations for chemicals do not account for effects on the gut microbiome because most chemicals are created to act on specific targets. For instance, insecticides are meant to affect insects, not humans or their resident microbes. New Machine Learning Tool Predicts Chemical Risks to Gut Health We were surprised that some of these chemicals had such strong effects. For example, many industrial chemicals like flame retardants and plasticizers -- that we are regularly in contact with -- weren't thought to affect living organisms at all, but they do." Dr. Stephan Kamrad, also part of the research team, added: "Safety assessments of new chemicals for human use must ensure they are also safe for our gut bacteria, which could be exposed to the chemicals through our food and water." There is still limited knowledge about how environmental chemicals directly influence the gut microbiome and human health. Patil said: "Now we've started discovering these interactions in a laboratory setting it's important to start collecting more real-world chemical exposure data, to see if there are similar effects in our bodies." Scientists Spot Hidden Water Ice on Mars: A Game-Changer for Human Missions New Theory Suggests We've Been Wrong About Black Holes for 60 Years Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/12/251202052226.htm'>Scientists boost lifespan by 70% in elderly male mice using simple drug combo</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-03 03:44:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The work, led by first author Cameron Kato along with corresponding author and Aging-US Editorial Board Member Irina M. Conboy at the University of California, Berkeley, reveals that combining oxytocin with an Alk5 inhibitor (OT+A5i) greatly improved both lifespan and overall health in frail, elderly, male mice. The team used a two-part treatment designed to address major biological changes that occur with age. Oxytocin, a hormone known to support tissue repair and naturally decline over time, was paired with an Alk5 inhibitor that blocks the TGF-beta pathway. In this research, frail mice aged 25 months, which is roughly equivalent to 75 human years, received regular OT+A5i treatment. Female mice showed no major improvements in lifespan or sustained health measures, although middle-aged females did experience increased fertility. These outcomes highlight how strongly sex-specific biology can influence the effectiveness of aging interventions. The exact reasons behind these differences are still unclear, yet the study establishes a useful model for understanding how therapies may work differently across sexes. Oxytocin already has FDA approval, and Alk5 inhibitors are being evaluated in clinical trials, which raises the possibility that this combined approach could eventually be adapted for humans. Based on the robust improvements seen in frail elderly male mice, OT+A5i may hold significant promise for enhancing late-life health and survival in the future. Scientists Spot Hidden Water Ice on Mars: A Game-Changer for Human Missions New Theory Suggests We've Been Wrong About Black Holes for 60 Years Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            