
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-04</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/senate-hearing-with-rfk-jr-sparks-backlash-from-scientists-and-health/'>Senate Hearing with RFK Jr. Sparks Backlash From Scientists and Health Experts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 21:35:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Health experts responded to Kennedy's claims on social media in disbelief and anger Robert F. Kennedy Jr., US secretary of Health and Human Services (HHS), during a Senate Finance Committee hearing in Washington, DC, US, on Thursday, Sept. 4, 2025. On Thursday a combative and acerbic Robert F. Kennedy, Jr., secretary of the U.S. Department of Health and Human Services, appeared before the Senate Committee on Finance and fielded angry questions from some of the very senators that helped him join President Donald Trump's cabinet. Kennedy often interrupted the questioning: He told Senator Bill Cassidy of Louisiana, “You're wrong,” when Cassidy said Kennedy was denying people COVID vaccines. Kennedy defended his decision to fire Monarez after only 29 days of service by claiming that she said “no” when asked if she was “trustworthy.” In a piece in the Wall Street Journal, Monarez wrote that Kennedy had asked her to sign off on “the recommendations of a vaccine advisory panel newly filled with people who have publicly expressed antivaccine rhetoric.” Many public officials and health-related organizations have called for Kennedy's resignation. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Throughout the hearing, public health experts, medical professionals and scientists responded on social media. We spoke on the phone and she said ‘I won't ever abandon you, but I am no longer able to be involved in vaccine policy per HHS. Those decisions need to go through the chief of staff. Goes after him on everything from vaccines to the Big Beautiful Bill, gently pointing out that he's effectively, and repeatedly, lied. Wants a ‘written statement' from RFK [Jr.] on his view of Warp Speed and RNA tech. RFK [Jr.] not getting overwhelming support from GOP.” —Pam Herd, Social Sciences University of Michigan, writing on Bluesky “Kennedy continues to claim that vaccines aren't adequately tested. He is a threat to public health & scientific progress. That's why we have to fire people at the CDC. “CDC folks have been fired, fired at, and now being blamed for the country's health crisis by a anti-science quack.” —Madhu Pai, public health and tuberculosis expert, McGill University, writing on Bluesky “Confirming a man who is arguably the world's most extreme & dangerous anti-vaxx activist & conspiracy theorist was a massive failure…..” —Gavin Yamey, global health expert, Duke University, writing on Bluesky “It's very clear that RFK Jr. has been lying throughout his entire senate hearing that happened earlier today.” —Zachary Rubin, physician and science communicator, speaking on Instagram That's why we have to fire people at the CDC!' “He fired people at CDC because they would not agree to carry out his plan to end vaccines in America.” —Angela Rasmussen, infectious disease expert, University of Saskatchewan, writing on X Megha Satyanarayana is chief projects editor at Scientific American. She is a former scientist who has worked at several news outlets, including the Detroit Free Press and STAT. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02788-0'>Newfound immune cell in mice hints at why inflammation spikes with old age</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 17:00:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Gemma Conroy is a freelance science journalist based in Mexico City. Macrophages, which hoover up pathogens, have been implicated in the chronic inflammation that sets in with the march of time.Credit: Science Photo Library The inflammation-promoting cells emerge only in older mice, researchers found. Although the cells' precise role is unclear, they display several signatures associated with ‘inflammageing' — the persistent inflammation that develops as people grow older. “We did not anticipate that there would be a completely new cell type,” says study co-author Vishwa Deep Dixit, an immunobiologist at Yale University in New Haven, Connecticut. When injuries or infections occur, the immune system mounts a protective response by releasing cells and proteins to affected tissues. But as we age, inflammation gradually increases and becomes persistent instead of being a state that occurs only when things go wrong. Ageing is linked to inflammation — but only in the industrialized world Ageing is linked to inflammation — but only in the industrialized world But the exact roles of each type of macrophage and how these cells change throughout the ageing process remain a mystery, says study co-author Elsie Gonzalez-Hurtado, also an immunobiologist at Yale University. “There wasn't really a good characterization of these cells.” A previously known type of macrophage that resides near nerves, for example, became less numerous in female mice as they grew older — but its numbers remained constant in males. How ageing changes our genes — huge epigenetic atlas gives clearest picture yet Depriving brain tumours of an amino acid could enhance chemotherapy Biogen is seeking an Associate Director to support clinical therapeutic development. Biogen are seeking a Sr Principal Biostatistician to lead study design and data analysis in clinical development. Biogen is seeking a highly motivated and accomplished Scientist with drug discovery experience. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250904103917.htm'>Scientists reveal how breakfast timing may predict how long you live</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 15:41:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>However, how meal timing relates to our health remains less understood. Researchers at Mass General Brigham and their collaborators studied changes to meal timing in older adults and discovered people experience gradual shifts in when they eat meals as they age. "Our research suggests that changes in when older adults eat, especially the timing of breakfast, could serve as an easy-to-monitor marker of their overall health status. Patients and clinicians can possibly use shifts in mealtime routines as an early warning sign to look into underlying physical and mental health issues," said lead author Hassan Dashti, PhD, RD, a nutrition scientist and circadian biologist at Massachusetts General Hospital, a founding member of the Mass General Brigham healthcare system. "Also, encouraging older adults in having consistent meal schedules could become part of broader strategies to promoting healthy aging and longevity." Dashti and his colleagues -- including senior author Altug Didikoglu, MSc, PhD, of the Izmir Institute of Technology in Turkey -- examined key aspects of meal timing that are significant for aging populations to determine whether certain patterns might signal, or even influence, health outcomes later in life. Notably, later breakfast timing was associated with an increased risk of death during follow-up. "Up until now, we had a limited insight into how the timing of meals evolves later in life and how this shift relates to overall health and longevity," said Dashti. "Our findings help fill that gap by showing that later meal timing, especially delayed breakfast, is tied to both health challenges and increased mortality risk in older adults. Dashti noted that this has important implications as time-restricted eating and intermittent fasting gain popularity, where the health impacts of shifting meal schedules may differ significantly in aging populations from those in younger adults. Authorship: In addition to Dashti, Mass General Brigham authors include Chloe Liu, Hao Deng and Anushka Sharma. Funding: This study was supported by the National Institute of Health (R00HL153795). Astronomers Discover Mysterious New World at Edge of the Solar System Weight Loss Breakthrough: Scientists Develop Edible “Fat Sponges” From Green Tea and Seaweed Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/green-tech/a65963846/solar-panels/'>Solar Power Isn't as Green as You Think. Scientists Have a Weird Fix: Onions.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ultraviolet rays actually break down components of solar panels over time, but red onion Solar power is one of the best tools in humanity's decarbonization toolbox, but even these green energy technologies aren't 100 percent sustainable. Typically, commercial solar panels rely on petroleum-based filters made from polyvinyl fluoride (PVF) and polyethylene terephthalate (PET) to do this job. While bio-based replacements would be a preferable material, these oil-based filters get the job done in large commercial solar farms where panels can be responsibly recycled. However, some solar applications could benefit immensely from having a system that was safely biodegradable, particularly in microsensors. In the study, published in the journal ACS Applied Optical Materials, scientists developed UV protection film using nanocellulose—a natural material typically harvested from bacteria or plants that just so happens to be the most abundant organic polymer on Earth. Developing an effective solar cell filter isn't easy. The scientists discerned that red onion extract provided 80 percent light transmission between 650 to 1,100 nanometers while maintaining performance over the 1,000-hour testing period (equivalent to roughly one year of sunlight). This performance-over-time aspect proved especially important because filters treated with iron ions performed admirably at first, but degraded rapidly over the testing period. As we continue to reduce our dependence on fossil fuels, nature once again provides. First, with the necessary sunlight to create solar power, and then with the red onions to make those solar cells even more efficient. Who knows what other bio-based innovations will lead us to a new, green future? Experts Found First Branch On Animal Tree of Life This Ancient Larva Still Has Its Brain And Guts</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/hacking-ai-agents-how-malicious-images-and-pixel-manipulation-threaten/'>The New Frontier of AI Hacking—Could Online Images Hijack Your Computer?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The New Frontier of AI Hacking—Could Online Images Hijack Your Computer? Artificial-intelligence agents—touted as AI's next wave—could be vulnerable to malicious code hidden in innocent-looking images on your computer screen Compromised images can take over AI agents on user computers (image for illustrative purposes only). A website announces, “Free celebrity wallpaper!” You browse the images. There's Selena Gomez, Rihanna and Timothée Chalamet—but you settle on Taylor Swift. You set it as your desktop background, admire the glow. Instead it opens your web browser and downloads a file. What distinguishes an AI an agent from a chatbot is that it doesn't just talk—it acts, opening tabs, filling forms, clicking buttons and making reservations. And with that kind of access to your machine, what's at stake is no longer just a wrong answer in a chat window: if the agent gets hacked, it could share or destroy your digital content. Now a new preprint posted to the server arXiv.org by researchers at the University of Oxford has shown that images—desktop wallpapers, ads, fancy PDFs, social media posts—can be implanted with messages invisible to the human eye but capable of controlling agents and inviting hackers into your computer. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. For instance, an altered “picture of Taylor Swift on Twitter could be sufficient to trigger the agent on someone's computer to act maliciously,” says the new study's co-author Yarin Gal, an associate professor of machine learning at Oxford. That means that the next person who sees your Twitter feed and happens to have an agent running will have their computer poisoned as well. Before you begin scrubbing your computer of your favorite photographs, keep in mind that the new study shows that altered images are a potential way to compromise your computer—there are no known reports of it happening yet, outside of an experimental setting. And of course the Taylor Swift wallpaper example is purely arbitrary; a sabotaged image could feature any celebrity—or a sunset, kitten or abstract pattern. Furthermore, if you're not using an AI agent, this kind of attack will do nothing. “They have to be very aware of these vulnerabilities, which is why we're publishing this paper—because the hope is that people will actually see this is a vulnerability and then be a bit more sensible in the way they deploy their agentic system,” says study co-author Philip Torr. Now that you've been reassured, let's return to the compromised wallpaper. To the human eye, it would look utterly normal. For this reason, agents built with AI systems that are open-source—that allow users to see the underlying code and modify it for their own purposes—are most vulnerable. Anyone who wants to insert a malicious patch can evaluate exactly how the AI processes visual data. “We have to have access to the language model that is used inside the agent so we can design an attack that works for multiple open-source models,” says Lukas Aichberger, the new study's lead author. By using an open-source model, Aichberger and his team showed exactly how images could easily be manipulated to convey bad orders. “Basically, we adjust lots of pixels ever-so-slightly so that when a model sees the image, it produces the desired output,” says study co-author Alasdair Paren. If this sounds mystifying, that's because you process visual information like a human. But the computer breaks the picture down into pixels and represents each dot of color as a number, and then it looks for patterns: first simple edges, then textures such as fur, then an ear's outline and clustered lines that depict whiskers. But because the computer relies on numbers, if someone changes just a few of them—tweaking pixels in a way too small for human eyes to notice—it still catches the change, and this can throw off the numerical patterns. Now why does the new study pay special attention to wallpapers? The agent can only be tricked by what it can see—and when it takes screenshots to see your desktop, the background image sits there all day like a welcome mat. The hidden command even survived resizing and compression, like a secret message that's still legible when photocopied. “On this website you can have additional attacks encoded in another malicious image, and this additional image can then trigger another set of actions that the agent executes, so you basically can spin this multiple times and let the agent go to different websites that you designed that then basically encode different attacks,” Aichberger says. The team hopes its research will help developers prepare safeguards before AI agents become more widespread. “This is the first step towards thinking about defense mechanisms because once we understand how we can actually make [the attack] stronger, we can go back and retrain these models with these stronger patches to make them robust. That would be a layer of defense,” says Adel Bibi, another co-author on the study. And even if the attacks are designed to target open-source AI systems, companies with closed-source models could still be vulnerable. “A lot of companies want security through obscurity,” Paren says. “But unless we know how these systems work, it's difficult to point out the vulnerabilities in them.” Gal believes AI agents will become common within the next two years. “People are rushing to deploy [the technology] before we know that it's actually secure,” he says. Deni Ellis Béchard is Scientific American's senior tech reporter. You can follow him on X, Instagram and Bluesky @denibechard. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02324-w'>DNA methylation influences human centromere positioning and function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 10:05:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Maintaining the epigenetic identity of centromeres is essential to prevent genome instability. Centromeres are epigenetically defined by the histone H3 variant CENP-A. Prior work in human centromeres has shown that CENP-A is associated with regions of hypomethylated DNA located within large arrays of hypermethylated repeats, but the functional importance of these DNA methylation (DNAme) patterns remains poorly understood. To address this, we developed tools to perturb centromeric DNAme, revealing that it causally influences CENP-A positioning. We show that rapid loss of methylation results in increased binding of centromeric proteins and alterations in centromere architecture, leading to aneuploidy and reduced cell viability. We also demonstrate that gradual centromeric DNA demethylation prompts a process of cellular adaptation. Altogether, we find that DNAme causally influences CENP-A localization and centromere function, offering mechanistic insights into pathological alterations of centromeric DNAme. Epigenetic modifications such as DNA methylation (DNAme) regulate chromatin architecture, gene transcription and cell division, and are often deregulated in cancer1. Centromeres, the chromosomal loci that control faithful chromosome segregation during cell division2, are epigenetically defined by chromatin enriched in the centromeric histone H3 variant CENP-A3,4, generally within highly repetitive DNA sequences5,6. CENP-A, along with other factors, identifies and maintains centromeric position indefinitely throughout the cell cycle7 and is a key player in maintaining centromere assembly and consequent faithful chromosome segregation8,9. CENP-A nucleosomes are interspersed with canonical histone H3, with CENP-A representing 4–25% of nucleosomes in centromeric chromatin10,11, which constitutes about 10% of each repetitive sequence array on average12. What defines CENP-A position and boundaries within centromeric regions is still an open question. In canonical human centromeres, CENP-A assembles on direct (head-to-tail) tandem arrays of 171 base pair (bp) α-satellite (α-sat) DNA repeats13. α-Sat arrays are further arranged in higher-order repeats (HOR), which are themselves repeated hundreds to thousands of times to create homogeneous HOR arrays spanning up to 5 Mbp12. Although each chromosome may contain multiple distinct HOR arrays, CENP-A typically localizes within one array, deemed the ‘active' HOR array14. Approximately every other α-sat repeat unit contains a conserved 17 bp motif named the CENP-B box, which is bound by CENP-B, the only centromeric sequence-specific DNA-binding protein identified so far15 and a key player in centromere organization and function16. Recently, the complete assembly of centromere sequences revealed that centromeric DNA is heavily methylated with the exception of a small hypomethylated region within every active HOR array named the centromere dip region (CDR)17, which corresponds to a region of high CENP-A density11,12,17,18,19. These findings, along with the evidence that links alterations of DNAme at (peri-)centromeric regions and diseases such as cancer or immunodeficiency, centromeric heterochromatin instability and facial anomalies (ICF) syndrome20,21, hint that DNAme is an important contributor to the epigenetic determination of centromere identity and stability22 (Fig. a, Scope of the paper. b, DNA (de)methylation cycle. c, Schematic of the DBDTET1AID active demethylation. Black lollipops represent 5mCpGs, white lollipops represent CpGs and CENP-B boxes are in purple. Panel c is created with BioRender.com. d, Left: representative images of CENP-C and FLAG immunofluorescence in DLD-1 FRT DBDTET1AID cells −/+DOX for 48 h. Scale bars = 5 μm; zooms = 4.84 × 4.84 μm. Right: quantification of FLAG fluorescence intensity at centromeres (defined by CENP-C) and in random nuclear positions. Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. Each dot represents one region of interest (n = 1,409, 1,348 at centromeres, 490, 346 in random positions, −/+DOX, respectively). Kruskal–Wallis test with Dunn's multiple comparisons: ****P < 0.0001; NS, not significant. e, Average ± standard error of the mean (s.e.m.) FLAG fluorescence intensity at centromeres and random positions in nine different clones (color coded) of DLD-1 cells transduced with LV DBDTET1AID after 48 h of induction with DOX. Two-tailed unpaired t test: ****P < 0.0001. f, Kinetics of methylated (5mC) α-sat DNA as percentage of total DNA by COBRA. DLD-1 FRT cells with DOX-induced expression of active (TET1) or inactive (dTET1) construct. Kruskal–Wallis test with Dunn's multiple comparisons, *P = 0.0325, day 16 to day 0. g, Left: kinetics of 5mC α-sat DNA by COBRA in DLD-1 LV DBDTET1AID −/+DOX. h,k, Mapping-independent methylation calling of nanopore reads at α-sats, CENP-B boxes and pericentromeric HSat2 repeats in DLD-1 cells expressing FRT DBDTET1AID (h) or AIDDNMT1 ± DNMT3B−/− [27] treated for 6 days with IAA (k). i, Kinetics of 5mCG, 5hmCG and CG by nanopore sequencing, normalized to day 0. j, Kinetics of 5mC α-sat DNA by COBRA. Here we have developed cellular tools to induce hypomethylation globally or targeted at centromeric regions to investigate the functional relationship among centromeric DNAme, centromere position and genome stability. 1b), to the DNA-binding domain (DBD) of CENP-B, generating a construct that is inducible by doxycycline (DOX) and degradable by an auxin-inducible degron24 (AID) upon the addition of indole-3-acetic acid (IAA; Fig. The centromere-targeted demethylating construct (hereafter named DBDTET1AID) was integrated into the pseudodiploid colorectal cancer cell line DLD-1 either by lentiviral transduction (multicopy random integration) or by the Flp-In recombination method (isogenic single-copy integration at a specific FRT locus)25. Addition of DOX leads to DBDTET1AID accumulation at centromeres (Fig. 1d,e) and to a gradual reduction in DNAme levels at centromeric regions as observed by combined bisulfite restriction analysis (COBRA; Fig. 1f,g and Extended Data Fig. The catalytically inactive (or dead, dTET1) version of the construct had no effect on the centromeric DNAme levels despite its centromeric localization (Fig. DNA demethylation was not restricted to CENP-B-boxes, which harbor two CpG sites, and it spread throughout the centromeric α-sat repeats, while only minorly affecting noncentromeric regions including the flanking pericentromeric Human Satellite 2 (HSat2) regions (Fig. As expected from the TET-mediated active demethylation (Fig. 1b), the expression of DBDTET1AID led to a rapid centromeric accumulation of 5-hydroxymethylcytosine (5hmC)—the first and most abundant oxidation intermediate in the pathway26—that decreased at later time points (Fig. The kinetics of DNA demethylation were slower in the single-copy integration system compared to certain clones of the multicopy lentivirus integration (Extended Data Fig. 1g), and the rate of demethylation of the clones was dependent on DBDTET1AID expression levels based on centromeric fluorescence intensity (Fig. All centromeres were demethylated with similar kinetics (Extended Data Fig. The demethylation process is antagonized by the canonical DNAme maintenance machinery as seen by the enhanced demethylation kinetics upon concomitant induction of a DBDTET1 construct with an AID-mediated degradation of the endogenous maintenance DNA methyltransferase DNMT1, which doubled the rate of DNA demethylation (Fig. According to the role of DNMT1 in maintaining DNAme at centromeres, the rapid depletion of DNMT1 in DLD-1 cells27 led to a reduction of DNAme at α-sat and CENP-B boxes and, to similar extents, at pericentromeric HSat2 repeats (Fig. Co-depletion of DNMT1 and the de novo DNA methyltransferase DNMT3B, but not of DNMT3B alone, strongly affected DNAme levels, highlighting a cooperativity between DNMT1 and DNMT3B in the maintenance of (peri)centromeric DNAme. Altogether, these data show that the expression of DBDTET1 demethylates the centromeres and suggests that centromeric DNAme is mainly controlled by the canonical DNAme maintenance machinery, at least in somatic adult cells. We then addressed whether DNAme controls the position of human centromeres by modulating the binding of centromeric proteins (CENPs; Fig. CpG methylation of the CENP-B boxes was suggested to reduce the binding of CENP-B28,29 and CENP-B was previously implicated in promoting centromeric CENP-A assembly7,30. Using microscale thermophoresis, we determined in vitro that CENP-B has an eightfold stronger affinity for unmethylated CENP-B boxes (dissociation constant Kd ~ 0.3 µm) than for methylated ones (Kd ~ 2.5 µm; Fig. An increase of CENP-B signal at centromeric regions upon gradual DBDTET1AID-mediated DNA hypomethylation was confirmed in cells by immunofluorescence and CUT&RUN–qPCR (Fig. 2c,d and Extended Data Fig. DOX induction times were chosen for each cell line based on their demethylation kinetics (Extended Data Fig. 1g) and, unless stated otherwise, were followed by 2 days of auxin treatment to remove any possible steric hindrance of the TET1 enzyme bound to the centromeres. Immunofluorescence analysis also revealed an increase of CENP-A levels at centromeres as DNAme levels decreased (Fig. 2c,d and Extended Data Fig. The CENP-A increase was observed in all cell cycle phases (Fig. 2e,f and Extended Data Fig. 2e,f), including metaphase chromosomes (Fig. 2g), indicating that this increase is not an artifact due to centromeric clustering in interphase nuclei. Notably, the CENP-A/CENP-B increase was also observed when depleting DNMT1 and DNMT3B (Fig. A similar increase, although less consistently pronounced, was also observed for CENP-C (Fig. 2c,d,g,h and Extended Data Fig. 2a–c,g), a key factor for the assembly of the kinetochore31 and known to interact with both CENP-A and CENP-B7. The increase of CENP-A signal was independent of the CENP-B increase as the siRNA-mediated knockdown of CENP-B did not prevent CENP-A accumulation at hypomethylated centromeres (Extended Data Fig. a, Schematic of experiments shown in (c–g). b, Affinity curves of CENP-B binding to methylated or unmethylated DNA. c,e, Representative immunofluorescence images in DLD-1 LV DBDTET1AID clone 6 at 4 days −/+DOX + 2 days IAA. c, Staining for CENP-A, CENP-B, and CENP-C. Insets: 2.5 × 2.5 μm zoomed views of boxed regions. e, Staining for geminin, CENP-A, and ACA. d, Kinetics of centromeric CENP-A, CENP-B and CENP-C levels in DLD-1 LV DBDTET1AID clones 3, 6 and FRT DBDTET1AID. Mean ± s.e.m., n = 3 (Extended Data Fig. Kruskal–Wallis test with Dunn's multiple comparisons of last-time points versus untreated. Gray gradient indicates normalized α-sat methylation level at each timepoint. f, Centromeric CENP-A and ACA levels from (e) in geminin negative (G1) and positive (S-G2) cells normalized to −DOX. g, Left: representative images of immunofluorescence in mitotic chromosomes. Right: line scans quantification of CENP-A, CENP-B and CENP-C across mitotic centromeres. Two biological replicates, six cells per condition. n = 146 (untreated (NT)) and 170 (DOX) total centromeres. Three biological replicates, n = 190 (DNMT1/DNMT3B+/+), 211 (DNMT3B−), 176 (DNMT1−) and 183 (DNMT1/3B−/−) total nuclei, *P = 0.0489. i, Schematic of experiments shown in j and k. j, Aggregate plot of 5mC/C across all reads overlapping CDR boundaries in cells expressing DBDMQ1Q147LAID (purple) or DBDdMQ1AID (green). Two biological replicates, n = 152 and 91 total nuclei for DBDMQ1Q147LAID, 119 and 133 total nuclei for DBDdMQ1AID −/+DOX, respectively. Box plots from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. Small dots represent average centromeric fluorescence intensity of a nucleus, color/shape coded by biological replicate. Large dots represent average of all nuclei per biological replicate. Statistics defined by Kruskal–Wallis test with Dunn's multiple comparisons. To strengthen our observations on the inverse relationship between centromeric DNAme and CENP-A/CENP-B levels, we fused the prokaryotic DNA methyltransferase MQ1Q147L or its catalytically inactive variant dMQ1 (C141S and S317A)32 to CENP-BDBD, generating, as for TET1, a dually controllable DOX/IAA system for targeted centromeric CpG hypermethylation (Fig. We inserted this system in RPE-1 cells for which a telomere-to-telomere (T2T) genome assembly was recently published33, therefore allowing us to assess specifically the methylation levels of the CDRs where CENP-A is enriched. DOX induction of the active MQ1Q147L construct led to a rapid increase in DNAme within the CDRs (~1.8-fold increase in 4 days) and only minorly at the surrounding regions or at random genomic regions compared to the catalytically inactive dMQ1, as observed by ultralong read nanopore sequencing (Fig. In this condition, both CENP-A and CENP-B intensity levels at centromeres decreased (Fig. 2k), reinforcing the inverse relationship between DNAme and CENP-A/CENP-B centromeric binding. The increase in CENP-A levels could reflect a higher density of CENP-A nucleosomes within the same region or, alternatively, could mean that CENP-A expands beyond its canonical domain, that is the CDR, with CENP-A nucleosomes being deposited in newly hypomethylated positions. To test this hypothesis, we performed a directed methylation with long-read sequencing (DiMeLo-seq) experiment using ultralong nanopore sequencing to map CENP-A molecules on single chromatin fibers11 before and after demethylation. DiMeLo-seq involves antibody-mediated tethering of a sequence-nonspecific prokaryotic adenine methyltransferase to a DNA-binding or chromatin-binding protein in permeabilized cells and the subsequent methylation of nearby accessible DNA as N6-methyladenosine (m6A). The exogenous m6A will be read along with endogenous 5mC by nanopore sequencing of native DNA (Fig. We performed DiMeLo-seq from a cell population and a single clone of CHM13 cells, for which a T2T assembly is available34, harboring lentiviral expression of the DBDTET1 construct. DBDTET1 expression led to a strong decrease of 5mC both within and near CDRs, therefore removing the DNAme boundaries across all CDRs (Fig. After this TET1-mediated centromeric DNA demethylation, we observed an increase in m6A, reflecting an accumulation of CENP-A (Fig. This increase in m6A was detected on the same reads harboring 5mC loss at active α-sat HOR arrays and specifically within and around CDRs (Fig. 3b), although m6A levels remained higher inside CDRs. We saw no effect of DBDTET1 expression on m6A levels (Extended Data Fig. 4d,e) across random genomic positions, further validating the DiMeLo approach. Unlike the population (Extended Data Fig. 4f), the single clone presents a reduction of DNAme outside centromeres, possibly due to leaky DBDTET1 expression during the prolonged time in culture (Extended Data Fig. a, Schematic depicting CENP-A targeted DiMeLo-seq experiment. Panel a was created with BioRender.com. b, IGV browser region showing individual CENP-A targeted DiMeLo-seq reads overlapping a CDR within the active α-sat HOR array on chr21 in a CHM13 clone without treatment and after DBDTET1 expression (DOX). The same reads (gray rectangles) display 5mC (red) and m6A (blue). c,d, Aggregate plots of 5mC as a fraction of total cytosines centered on CDR centers (c) and CDR boundaries, with left CDR boundaries flipped (d). f,g, Aggregate plots of m6A (CENP-A) as a fraction of total adenines across all DiMeLo-seq reads overlapping CDR boundaries in a CHM13 clone untreated (black) and +DOX (red for 5mC, blue for m6A) centered on CDR centers (f) and CDR boundaries, with left CDR boundaries flipped (g). e,h, Distribution of 5mC (e, red) or m6A (CENP-A; h, blue) across CDRs and beyond 100 kb in a CHM13 clone untreated, +DOX and the ratio between them as a fraction of total C/total A, respectively. Each line of the heatmap represents a CDR/centromere/chromosome. CDRs have been scaled to the same length. i, Schematic of experiments shown in j and k. j, Proportion of MSPs >50 bp at CDRs and at active α-sat HORs excluding CDRs. Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, whiskers from 2.5th to 97.5th percentiles. Two-tailed Mann–Whitney test, ****P < 0.0001. k, Aggregate plots of m6A (H3K9me3) as fraction of total A across all DiMeLo-seq reads overlapping CDR boundaries in a CHM13 clone untreated (black) and +DOX (pink). Given that hypomethylation causes CENP-A levels to increase within and around the CDRs of all centromeres, we then investigated other possible changes to the centromeric chromatin following hypomethylation. Fiber-seq revealed enhanced chromatin accessibility at both CDRs and flanking regions following centromeric hypomethylation (Fig. On the other hand, DiMeLo-seq detected little to no change in H3K9me3 levels following loss of DNAme at the CDR and flanking boundaries (Fig. 3k), which was corroborated by immunofluorescence imaging in mitotically arrested cells (Extended Data Fig. Altogether, these data indicate that rapid hypomethylation of centromeric regions decreases chromatin compaction, but it does not decrease the level of the heterochromatic histone modification H3K9me3 on these short time scales. We then assessed the short-term and long-term consequences of loss of centromeric DNAme for cell viability and chromosome segregation fidelity (Fig. Progressive centromeric DNAme reduction had a minor impact on the cell cycle, with a slight accumulation of cells in G2/M (Extended Data Fig. 5a), and a consistent negative effect on cell doubling and viability proportional to the kinetics of DNA demethylation in DLD-1 cells (Fig. These findings support a link between the level of centromeric DNAme and centromere function. Similar results were obtained in nontransformed, diploid RPE-1 cells expressing a DBDTET1 construct by lentiviral integration (Extended Data Fig. DLD-1 cells expressing the catalytically dead construct had a similar capacity to form colonies compared to untreated cells (Fig. 4c), indicating that the binding of ectopic TET1 to the CENP-B boxes in and of itself does not account for the negative impact of the catalytically active TET1 on cell viability. The centromeric demethylation rate from different DLD-1 clones correlated with an increased frequency of micronuclei formation (Fig. 4d,e and Extended Data Fig. 5e), suggesting that the speed of centromeric demethylation (Fig. Time-lapse microscopy and immunofluorescence staining on a DLD-1 clone expressing high levels of DBDTET1AID revealed an increase in mitotic timing and chromosome mis-segregation in a DNAme-dependent manner (Fig. 4f,g and Extended Data Fig. 5f), consistent with a loss of centromere function. Single-cell genome sequencing (scGS) further revealed that centromere-targeted acute demethylation causes a substantial increase in numerical and structural aneuploidies, with copy number alterations (CNA) occurring mostly near or at the centromeres (Fig. 5g,h), suggesting breaks at these regions. Loss of DNAme did not reveal any substantially distinct aneuploidy pattern (Fig. 4i), in agreement with the observed equal removal of DNAme at all centromeres. We conclude that both the total level of centromeric DNAme and the speed of DNA demethylation affect centromeric function. a, Schematic of the experiments shown in b–k. b, Real-time cell index of naïve DLD-1 cells (NT) and DOX pretreated (4 or 8 days) +DOX during the assay. c, Representative colony formation assays of DLD-1 LV DBDTET1AID clones (left), and ratio of colonies formed +DOX/NT (right). d, Percentage of cells with micronuclei at different demethylation levels. e, Correlation between the demethylation ratio at 0–2 days (Fig. 1g) and the change in percentage of cells with micronuclei (day 2 – day 0) for all clones. f, Mitosis presenting errors in time-lapse microscopy. Two-sided Fisher's exact test: ****P < 0.0001, **P = 0.0080. g, Representative images of mitotic errors. h, Frequency of numerical and structural aneuploidy from scGS. Two-sided Fisher's exact test: *P = 0.0137; ****P < 0.0001. i, Heatmap of the frequency of chromosome-specific aneuploidies from scGS. j, Representative images and line plots from CENP-A expansion microscopy (ExM 4.5-5×, single z-stack) of two sister centromeres untreated or demethylated (+DOX) for 4 days + 2 days IAA. k, Number of quantifiable CENP-A sub-domains per single centromere from ExM (n = 3 biological replicates, 300 total centromeres per condition). Two-sided Fisher's exact test: ****P < 0.0001 (between DOX and centromeres with >2 CENP-A sub-domains). VINCULIN (VNC) used as loading control after stripping. m, Cell growth of DLD-1 LV DBDTET1AID cl.6 and two derived CENP-BKO sub-clones, untreated or +DOX. Kruskal–Wallis test with Dunn's multiple comparisons, ****P < 0.0001 (to WT +DOX). n, Percentage of cells presenting micronuclei. Two-sided Fisher's exact test: ***P = 0.0002 (WT versus CENP-B KO clones). ANOVA, analysis of variance; WT, wild type. Recently, it was revealed that centromeres are organized into a bipartite structure during mitosis, which is important for chromosome segregation fidelity35. To assess whether centromeric demethylation compromised this structure, we performed super-resolution expansion microscopy36 on the DLD-1 clone expressing high levels of DBDTET1AID (Fig. Although the largest fraction of centromeres conserves a bipartite structure in cells with demethylation, we observed an increase in the number of centromeres with noncanonical structures (Extended Data Fig. 6a) and with more than two CENP-A sub-domains (Fig. These results, along with the observed increase in mitotic intercentromeric distance (Fig. 2g; second CENP-A/CENP-C peak shift), demonstrate that centromeric demethylation alters the global organization of CENP-A chromatin. Considering that one of the most prominent effects of centromeric hypomethylation is increased CENP-B binding, we tested whether removal of CENP-B could rescue the mitotic defects and cell lethality. To this end, we knocked out CENP-B by CRISPR–Cas9 in DLD-1 cells expressing high levels of the DBDTET1AID transgene to uncouple the effects of reduced DNAme from those related to increased CENP-B binding (Fig. Both the cell-growth defects and the frequency of micronuclei formation were reduced upon centromeric hypomethylation in CENP-BKO cells (Fig. Similarly, a mild ectopic overexpression of CENP-B—to mimic the levels observed following centromere hypomethylation (Extended Data Fig. 6g) without affecting the normal centromeric methylation levels in DLD-1 cells (Extended Data Fig. 6h)—led to a slight increase in micronuclei and few colonies formed compared to control and CENP-A overexpressing cells (Extended Data Fig. This observed CENP-B-mediated toxicity could be due to an increased accumulation of DNA loops30 leading to impairment of replication fork progression causing chromosome ruptures37,38, as observed by the increase in CNA number that occurs within centromeric regions following hypomethylation (Extended Data Fig. Accordingly, CENP-B overexpression caused a mild accumulation of the replication stress/DNA damage marker γH2A.X at centromeres during S-phase (Extended Data Fig. In addition, CENP-B knockout cells rescued with ectopic overexpressed full-length CENP-B (FL) had substantially higher centromeric breaks than cells rescued with a CENP-B mutant lacking the C-terminal dimerization domain (ΔC) that is required to form DNA loops30 (Extended Data Fig. Altogether, these results indicate that the accumulation of CENP-B at hypomethylated centromeric regions promotes genome instability and is partially responsible for the observed cell lethality following DNAme removal. This suggests that DNAme is an epigenetic layer that prevents excessive CENP-B binding to the centromeres. Finally, we tested whether cells could adapt to gradual centromeric hypomethylation (Fig. To this end, we treated DLD-1 Flp-In cells with DOX for 16 days to induce mild DBD-TET1AID expression. After the removal of the construct by auxin addition for another 16 days, α-sat methylation levels increased only marginally, without reaching the original level (Fig. Accordingly, centromeric CENP-A and CENP-B levels remained substantially higher compared to untreated cells even after prolonged washout (Fig. Similar results were observed in RPE-1 cells that transiently expressed DBDTET1AID followed by ~30 days washout (Extended Data Fig. Collectively, these experiments demonstrate that the new, stable epigenetic status of demethylated centromeres is translated into an equally stable increase of CENP-A and CENP-B levels. This suggests that, following slow and gradual hypomethylation, a process of cellular adaptation and remodeling of the centromeric protein landscape took place, ensuring cell survival in these new conditions. Accordingly, cells that were demethylated for either 8 or 16 days, then seeded in the absence of DOX, formed comparable numbers of colonies as fully methylated cells (Fig. In addition, cell viability did not further decrease in cells continuously treated for either 24 or 32 days with DOX compared to the 16-day treatment (Fig. These observations are in agreement with the stabilization in the percentage of cells with micronuclei at late hypomethylation times (Extended Data Fig. a, Scheme of the experiments shown in b–j. Continuous DOX treatment (magenta) and after washout and addition of auxin (DOX W/O + IAA). Two-tailed Mann–Whitney test at day 30, P = 0.800. c, Centromeric CENP-A and CENP-B levels in DLD-1 FRT DBDTET1AID cells. e, Ratio of colonies (DOX/NT or DOXW/O/NT). f, CRISPR–Cas9 strategy for tagging HELLS with mNG-AID. HELLSmNG-AID is degraded with 5Ph-IAA (24 h). LCL from a patient with ICF4(pU64; hypomorphic HELLS mutation). h, Mapping-independent methylation calling of nanopore reads at α-sats, CENP-B boxes and HSat2 repeats. i, 5mC as percentage of total C by COBRA in six loci from HCT116 cells normalized to HELLSmNG-AID NT. DKO = DNMT1/DMNT3B double knockout (data partially published in ref. j, Centromeric CENP-A and CENP-B levels in HCT116 HELLSmNG-AID cells. Three biological replicates, n = 75 (WT) and 66 (ICF3) total nuclei. l, Ratio of 5Ph-IAA/DMSO HCT116 HELLSmNG-AID colonies formed after 10 days. Two-tailed one-sample t test, no statistical significance: 0.1087 < P < 0.7903. For c, j and k, box plot shows the 25th to 75th percentiles, with central line at median, ‘+' indicating mean and whiskers extending from the minimum to the maximum values. Small dots represent the average centromeric fluorescence intensity of a nucleus, colored by biological replicate. Large dots represent the average of all nuclei per replicate. Statistics in j and k defined by two-tailed Mann–Whitney test, ****P < 0.0001. mNG-AID, mNeonGreen and AID; LHA, left homology arm; RHA, right homology arm; LCL, lymphoblastoid cell line; GAPDH, glyceraldehyde-3-phosphate dehydrogenase. To further test whether cells adapt to changes in DNAme, we generated a cellular model of ICF syndrome. This represents a unique physiological context in which centromeres have a naturally reduced level of methylation beginning in early development20,39. Patients affected by the disease have a poor life expectancy, mainly attributed to recurrent infections, but they can survive up to adulthood, proving that centromeric methylation is important but dispensable for cell survival40. To address the early consequences of the ICF mutations, we generated an ICF model in HTC116 cells by endogenously tagging both alleles of HELLS (Helicase Lymphoid-Specific, also known as LSH), which is mutated in ICF4 patients41, with mNeonGreen and an AID tag to mediate its rapid degradation (Fig. 5f,g and Extended Data Fig. As expected from studies with patients, the degradation of HELLS causes DNA demethylation of specific genomic loci, including both pericentromeric (HSat2) and centromeric repeats as measured by nanopore sequencing, COBRA and pyrosequencing, but with extremely slow kinetics (Fig. 5h,i and Extended Data Fig. As for the DBDTET1AID system, both CENP-A and CENP-B levels increased as a consequence of centromeric demethylation (Fig. CENP-B levels increased only slightly, in agreement with the less pronounced demethylation of CENP-B boxes compared to the DNMT1AID or DBDTET1AID cells (Fig. Similar results were observed in ICF3 (CDCA7 mutations) patient-derived fibroblasts (Fig. In accordance with the hypothesis of cellular adaptation after slow centromeric DNA demethylation induced using TET1 targeting, degradation of HELLS has no effect on cell viability even at late time points, when the centromeres are demethylated to similar levels of the DLD-1 cells expressing the DBDTET1AID constructs (Fig. Altogether, these results show that gradual and slow centromeric demethylation gives cells time to adapt to a new epigenetic landscape. DNAme is a fundamental epigenetic mark that regulates key cellular processes, and which is often altered in disease. Centromeres notably lose methylation in cancer and in ICF syndrome, highlighting the importance of centromeric methylation for the normal physiology of the cell. Here we have demonstrated that DNAme acts as a boundary to help define the position of CENP-A chromatin, and the loss of centromeric DNAme induces centromere alterations, resulting in aneuploidy and increased cell lethality (Fig. Centromeric hypomethylation causes an increase in CENP-A and CENP-B binding to the centromeres and allows CENP-A to spread outside the CDR boundary. A rapid, short-term demethylation of the centromeres causes chromosome mis-segregation, aneuploidy and reduced viability. A slow, long-term demethylation leads to cellular adaptation to the new epigenetic status. Centromeres have a unique epigenetic landscape that sets these regions apart from the rest of the genome. While active α-sat HOR arrays are overall highly methylated, the CDR has been described in all active human centromeres12,17,18,19,41 and also in the eastern hoolock gibbon42, highlighting a functional link between DNAme and centromere position that transcends the human species. By using human cell models to induce targeted DNA demethylation, our data demonstrate that the hypomethylation of the CDR helps to define the localization of CENP-A. However, what determines the initial anticorrelation between CENP-A and DNAme remains unknown. Either CENP-A nucleosomes could favor the hypomethylation of the CDR or DNAme might prevent CENP-A loading. Neither possibility is mutually exclusive and could act together as a positive feedback loop. These residues are present in H3 and are ubiquitinated by UHRF1, promoting DNMT1 recruitment independently of DNA replication43, a process crucial for maintaining DNAme in mid-to-late-replicating regions, like centromeres38,44. As a result, centromeric DNAme maintenance may be inefficient, causing passive methylation loss in CENP-A-rich regions. The anticorrelation between CENP-A and DNAme has also been recently observed at nonrepetitive neocentromere sites, suggesting that CENP-A loading in a given locus might prevent its DNAme45,46. For the second hypothesis, we can envision that DNAme disfavors the assembly of new CENP-A, possibly affecting nucleosome positioning and stability as observed for H3 at HSat2/HSat3 regions47, and/or the chromatin interaction of components involved in CENP-A deposition. Loss of DNAme at centromeres caused an enhanced deposition of CENP-A at CDR and a mild spreading outside of the CDRs. This rapid change in CENP-A was observed only after 4 days of active demethylation in the CHM13 clone, which represents approximately two cell divisions. Our data suggest that the rapid loss of DNAme creates a permissive chromatin environment conducive to the incorporation of new CENP-A nucleosomes, not only within the CDR itself but also in its flanking regions. Previously and/or newly deposited CENP-A accumulates and spreads into these new chromatin environments, likely mediated by enhanced chromatin accessibility, which we observed by Fiber-seq. In this view, we cannot exclude that heterochromatin domains flanking CENP-A also have a role in controlling boundaries, as recently observed45, although, in our time scales, H3K9me3 levels barely changed. This indicates that loss of DNAme per se is sufficient to favor CENP-A spreading outside the CDR, likely replacing canonical histone H3. It is possible that with longer time in culture, the position of CENP-A could drift further away from the original CDR after DNA demethylation, as observed following long-term depletion of heterochromatin-forming enzymes45. Centromere drifting was already observed in nonrepetitive centromeres of chicken cells48 and equids49 and occurs at neocentromeres as observed in the human45,46 and in the fungal pathogen Cryptococcus deuterogattii50. The driving force that mediates CENP-A movement in hypomethylated centromeres remains to be elucidated. Here we speculate that, following the initial loss of DNAme boundaries, CENP-A nucleosomes redistribute during DNA replication and/or with transcription. This idea is supported by the particularly high number of replication origins found within the α-sats38 and by the fact that changes in DNAme are well known to be associated with deregulation in replication51 and transcription52. In addition, changes in DNAme have also been associated with changes in three-dimensional (3D) genome organization51,53, although only disrupting specific heterochromatin compartments27,54, that in turn could promote CENP-A deposition to certain locations, even favoring neocentromere formation55. The changes of the overall architecture of centromeres following hypomethylation observed in our experiments are likely to be directly responsible for the observed chromosome mis-segregation. The increase in CENP-A sub-domains as a consequence of DNAme loss could promote incorrect kinetochore/microtubule attachment, possibly by incrementing the incidence of merotelic attachments35. It can also induce changes in the stiffness of the centromere/kinetochore interface, possibly through condensin56 or CENP-B30 misregulation. It is also possible that promiscuous CENP-A incorporation outside its predefined domain could weaken centromere/kinetochore function. In this view, if the expansion and loading of CENP-A is not balanced, it is possible that the local CENP-A concentration will not be sufficient to form stable higher-order chromatin structures compatible with kinetochore assembly, similarly to what was proposed for CENP-T57. Our data highlight that the chromosome segregation defects observed in centromeric hypomethylated cells can be partially attributed to the demethylation-dependent increase of CENP-B. Here we show that while CENP-B is important for the normal topology of human centromeres, sustaining mechanical tension at the kinetochore/microtubules interface30, enhanced CENP-B binding interferes with DNA replication, possibly by forming an excessive amount of DNA loops, which in turn promote centromeric instability. This suggests that, at centromeric regions, the CENP-B-mediated fragility of centromeric DNA has a direct role in the observed aneuploidy and translocations of hypomethylated cells, possibly also fueled by the DNA damage repair activity of the TET1-mediated demethylation58. Therefore, we propose DNAme as an epigenetic layer that regulates centromere stability by controlling the binding of CENP-B to centromeric regions (Fig. However, the fact that two CDRs have been detected on the Y chromosome59, the only human chromosome that lacks CENP-B boxes and CENP-B binding15, is a strong indication that the role of DNAme at the centromere is not limited to the regulation of CENP-B binding. While rapid demethylation has severe effects on the cells, a slower and gradual demethylation of the centromeric DNA prompts a process of cellular adaptation as observed in the mild TET1-expressing cells and ICF4 (HELLS) cellular model. The molecular mechanisms by which cells adapt to low centromeric DNAme are still elusive, but this phenomenon can be related to the survival of ICF syndrome patients beyond early childhood despite having low centromeric methylation levels. Finally, our results offer mechanistic insights into the genesis of genome instability observed in certain cancers that display hypomethylation of repetitive regions, such as breast adenocarcinomas60, Wilms tumors61 and hepatocellular carcinomas62. While the association with genome instability was mainly established for pericentromeric repeats such as HSat2, it will be interesting to study how this imbalance in DNAme maintenance, including the dysregulation of TET enzymes, impacts specifically centromere functionality and stability, further promoting aneuploidy and chromosome rearrangements21,22,63. Altogether, our findings provide unambiguous experimental evidence that DNAme influences CENP-A localization and is a major determinant in regulating centromere functionality. Our results provide a direct connection between the loss of DNAme at centromeres and the pathological chromosomal alterations also observed in ICF and cancer, highlighting the functional importance of DNAme for human centromere biology. This study complied with all relevant local ethical regulations. Approval for the use of ICF patients' fibroblasts was obtained after informed consent from all patients or their families (for minors), in agreement with the Helsinki Declaration (CNIL authorization—908256; 14 October 2008)41,66. DLD-1 cells were cultured in DMEM–GlutaMAX medium (Gibco) supplemented with 10% FBS. hTERT RPE-1 cells were cultured in DMEM/F-12 GlutaMAX medium (Gibco) containing 10% FBS and 0.123% sodium bicarbonate. HCT116 cells were cultured in McCoy's 5A modified medium (Gibco) with 10% FBS and 2 mM l-glutamine (Invitrogen). CHM13 hTERT cells (CHM13) (Magee-Women's Hospital) were cultured in DMEM/F-12 medium containing 10% FBS and supplemented with glutamine (Thermo Fisher Scientific), nonessential amino acids (Thermo Fisher Scientific), 1 mM sodium pyruvate and insulin-transferrin-selenium (Thermo Fisher Scientific). Primary ICF fibroblasts (ICF3)66 were cultured in DMEM–GlutaMAX medium supplemented with 15% FBS. DOX (Sigma-Aldrich) was dissolved in water and used at 100 ng ml−1 unless stated otherwise. IAA (Sigma-Aldrich) was dissolved in water and used at 500 µm. 5Ph-IAA was a gift from M. Kanemaki (National Institute of Genetics, Japan) and now it is commercially available; it was dissolved in DMSO and used at 1 µm. For details on the generation of cell lines, see Supplementary Methods. For immunofluorescence of cells in interphase, cells were grown to 60–80% confluence in 12 mm 1.5H glass coverslips (Marienfeld Superior). Cells were fixed with 3.7% formaldehyde, 0.5% Triton X-100 in PBS for 10 min (fixation with extraction), washed thrice with 0.1% Triton X-100 in PBS and blocked for 30 min at room temperature or overnight at 4 °C in blocking buffer (0.2 M glycine, 2.5% FBS and 0.1% Triton X-100). Primary antibody incubation was performed for 2 h at room temperature in 2% BSA, 0.1% Triton X-100 in PBS, with mouse anti-CENP-A (1:500; clone 3-19, ENZO, ADI-KAM-CC006-E), rabbit anti-CENP-B (1 μg ml−1 polyclonal; Abcam, ab25734 or 0.539 μg ml−1; clone EPR24047-64; Abcam, ab259855), guinea pig anti-CENP-C (1:1,000; MBL, PD030), rabbit anti-GFP (1 μg ml−1; ChromoTek, PABG1), mouse anti-FLAG (4 μg ml−1; Sigma-Aldrich, F3165), human anticentromere antibody (1:500; Antibodies Incorporated, 15-235-0001), rabbit anti-Geminin (1:500; clone E5Q9S; Cell Signaling Technology, 52508), rabbit monoclonal anti-CDT1 (1:500; clone EPR17891; Abcam, ab202067), rabbit anti-H3K9me3 (1 μg ml−1; Abcam, ab8898). Cells were washed thrice for 3 min with 0.1% Triton X-100 in PBS before incubation with species-specific secondary antibodies for 1 h at room temperature in 2% BSA, 0.1% Triton X-100 in PBS. Alexa Fluor 488, Cy3 and Alexa Fluor 647 conjugated secondary antibodies (Jackson ImmunoResearch Laboratories) were used at 1:500. Cells were washed thrice with 0.1% Triton X-100 in PBS before DAPI staining (1 μg ml−1 in PBS) for 10 min and subsequent mounting with ProLong Gold Antifade Mountant (Invitrogen). For immunofluorescence of mitotic cells, cells were grown in 18-mm glass coverslips (Marienfeld) to ~60% confluence, blocked in mitosis for 3 h with 0.1 μg ml−1 colcemid (Roche) and subjected to a 7-min-long hypotonic shock in a 60% media, 40% water. The semidry coverslips were then centrifuged for 3 min at 320 rcf to spread the chromosomes and then promptly fixed in 3.7% formaldehyde in PBS for 10 min. Immunofluorescence was carried out as described for interphase cells. Fixed imaging was performed on a DeltaVision Core system (Applied Precision) consisting of an Olympus IX71 inverted microscope equipped with a CoolSNAPHQ2 camera (Photometrics) and a 250-W xenon light source. Images ≤4 μm thick were captured in 0.2 μm z-sections at room temperature using a 100× Olympus UPlanSApo oil-immersion objective (numerical aperture 1.4), and then deconvolved and projected (3D maximum intensity) using DeltaVision's Softworx software. Alternatively, images were acquired on a Thunder Live Cell Imager (Leica) equipped with a Leica K8 CMOS Black-thinned camera, LED lamps and a DFT 51010 dichroic Cube using the LAS X (v1.4.7, Leica) software. Images were captured using a 100× HCX PL APO oil-immersion objective (numerical aperture 1.4) and processed with Instant Computational Clearing. For live-cell imaging, cells were grown on a 35 mm FluoroDish (World Precision Instruments) with 0.17 mm-thick optical-quality glass bottom and fitted with a four-well silicone insert (Ibidi). Time-lapse images were taken every 10 min for 20 h using an inverted Eclipse Ti-E microscope (Nikon) equipped with a CSU-X1 (Yokogawa) spinning disk integrated in Metamorph software, and a four-laser bench (Gataca systems). Furthermore, ~45 μm z-stacks were acquired (z-step size = 3 μm) with a ×60 CFI Plan Apo VC oil-immersion objective (numerical aperture 1.4). The microscope has a motorized Nano z100 piezo stage (Mad City Lab), a stage top incubator (Tokai Hit) and an EMCCD camera (Evolve, Photometrics). For the quantification of fluorescence intensity at the centromeres of cells in interphase, an automatic analysis was performed by detecting the maximum intensity of CENP-C per nucleus, drawing 0.3 μm radius circles (region of interest (ROI)) and measuring the fluorescence of each channel on each ROI (custom Macro script, available upon request). For the quantification of fluorescence intensity in metaphase cells, circles of 0.84 μm radius were automatically drawn (custom Macro script, available upon request) and manually adjusted to encompass the whole centromere of each chromosome. For the line-scan quantification, the fluorescence intensity on each channel was measured along >3.5 μm long lines manually drawn across the centromeres. The data were manually phased to align the first peak of CENP-C with each centromere. Genomic DNA was extracted using the NucleoSpin Tissue kit (Macherey-Nagel). Between 0.5 and 2 µg of DNA (equalized quantity for parallel comparisons) was bisulfite converted using the EpiTect bisulfite kit (Qiagen) according to the manufacturer's protocol. The converted DNA was amplified by PCR using locus-specific primers64 (Supplementary Table 1). PCR reactions were performed with 1.5 mM MgCl2, 200 μm dNTPs, 200 μm of each primer and 0.5 U Platinum Taq DNA polymerase (Invitrogen). The PCR products were digested for 3 h with 10 U of the appropriate enzyme at its digestion temperature (Supplementary Table 1). An equal amount of PCR product was incubated alongside the digestions without enzyme (undigested control). The full digests were loaded in 3% agarose gels. Images were acquired using a ChemiDoc (Bio-Rad) and band intensities were quantified using Fiji. Bisulfite conversion affects C, 5fC and 5caC, all being considered as unmethylated DNA. The enrichment of centromeric DNA (CenRICH) was performed as described before65. Nanopore long-read sequencing was performed after CenRICH. The DNA was treated with the Short Read Eliminator kit (cutoff < 25 kb, Circulomics SKUSS-100-101-01) to remove contamination from shorter DNA fragments. Libraries were prepared using the Library Preparation by Sequencing kit (Oxford Nanopore Technology (ONT)), quantified with Qubit dsDNA HS Assay Kit (Thermo Fisher Scientific) and checked by capillary electrophoresis with a TapeStation 4150 system (Agilent). The sequencing was performed on a Spot-ON Flow Cell (R9.4.1) on a MinION Mk1B device. The raw nanopore data were basecalled using the ONT software Guppy version 4.0 with a high accuracy modified-bases model (dna_r9.4.1_450bps_modbases_5mc_hac.cfg), which at the time could only identify 5mC modification. The resulting data were then converted with the fast5mod utility (https://github.com/nanoporetech/fast5mod), to produce as output the sequences of all nanopore reads and their per-base likelihood of methylation, expressed as a value ranging from 0 to 255. Details of the reads processing can be found in Supplementary Methods. DiMeLo-seq was performed as described in ref. The CHM13 population was sequenced with R9 chemistry (Supplementary Methods). The CHM13 single clone and RPE-1 hypermethylation experiments were sequenced with R10 chemistry. Fiber-seq was performed using the same protocol as DiMeLo-seq, with the following alterations: the primary antibody binding, protein A-Hia5 binding and activation steps were skipped. Instead, following nuclear isolation, nuclei were washed once in Tween-wash, then incubated with 1 µM free Hia5 and 800 µM S-adenosyl methionine in 100 µl activation buffer for 30 min at 37 °C. Supernatant was removed, and nuclei were resuspended in 1× PBS. Ultrahigh molecular weight gDNA was extracted using the NEB Monarch UHMW DNA Extraction Kit for Tissue (NEB, T3010), with a modified protocol. Nuclei from DiMeLo-seq or Fiber-seq reactions were resuspended in 40 µl of 1× PBS. In a separate tube, 1.8 ml NEB Monarch HMW gDNA Tissue Lysis Buffer and 60 µl proteinase K were premixed, then used to gently resuspend nuclei by pipetting with a wide-bore tip. The sample was then incubated at 56 °C for 10 min. Next, 15 µl of RNase A was added, and the sample was returned to the thermomixer to incubate for ten more minutes at 56 °C, this time with agitation at 650–750 rpm. Two glass beads were added to the sample, and 2.5 ml of isopropanol was added. The tube was gently inverted by hand until a clear, jelly-like substance was seen forming around the glass beads. The beads were washed twice with 2 ml Wash Buffer and then were transferred to a bead retainer with a collection tube beneath, quickly spun to remove liquid and quickly transferred into 200 µl extraction elution buffer from the ONT Ultra-Long Sequencing Kit (ONT SQK-ULK001 for R9 and SQK-ULK114 for R10). Beads were transferred to a bead retainer with a clean Eppendorf tube beneath and spun at 1,000g for 1 min. The eluted DNA solution was then brought up to a final volume of 750 µl with extraction elution buffer, and quantified by Qubit (Thermo Fisher Scientific, Q3328). Samples were prepared for sequencing using an ONT Ultra-Long Sequencing Kit (ONT SQK-ULK001 for R9 and SQK-ULK114 for R10) and sequenced using one PromethION flow cell per sample, achieving estimated N50 values of 50–90 kb and throughput of 33–60 Gb per sample for the CHM13 population, estimated N50 values of 124–127 kb and throughput of 36–44 Gb for the CHM13 single clone CENP-A DiMeLo-seq, estimated N50 values of 120–130 kb and throughput of 12 Gb for the CHM13 single clone H3K9me3 DiMeLo-seq, N50 values of 79–86 kb and throughput 37–78 Gb for the CHM13 single clone Fiber-seq. Adaptive sampling was performed on all R10 experiments to deplete noncentromeric reads by providing a bed file with the genome-wide complement of all α-satellite HOR arrays in chm13v2.0 plus a 100 kb buffer to either side. Nanopore (R10) DiMeLo-seq raw data (.pod5 files) were aligned to the T2T-CHM13v2.0 human reference genome and basecalled using dorado (v.0.8.3) and DNA model dna_r10.4.1_e8.2_400bps_sup@v5.0.0 with the following option specified: --modified-bases 6 mA 5mCG_5hmCG. Native barcoding runs had the following additional option specified: --kit-name SQK-NBD114-24. Aligned reads were filtered for primary alignment and sorted using samtools (v1.16.1). Barcoded runs were demultiplexed with dorado demux (v0.8.3). Centromeric ‘core' reads were defined as those overlapping T2T-CHM13v2.0 CDR coordinates, as specified in ref. Noncore reads (α-sat) were defined as those overlapping active α-satellite HOR, as specified in chm13v2.0_censat_v2.1.bed (T2T Consortium), but excluding CDR coordinates with a 10 kb buffer. Overlapping reads were isolated using samtools view (v1.16.1) and bedtools intersect (v2.30.0). Modification calls were dynamically thresholded using modkit call-mods (v0.1.13) with default parameters. Calls above the empirically determined threshold were clamped to certainty, while those below were discarded. Nucleosomes were called with the ft add-nucleosomes subcommand with default parameters using fibertools (v0.5.3). Chromatin accessibility was assessed by computing the proportion of methylation-sensitive patches over 50 bp using ft extract (fibertools v0.5.3) and custom Python scripts. To assess methylation levels at CDRs, samtools view was used to filter for reads aligning to α-sat HOR sequences, as specified in chm13v2.0_censat_v2.1.bed (T2T Consortium). Reads were converted to fasta format with samtools fasta (v1.16.1), re-aligned to the RPE-1 reference genome33 using minimap2 (v2.26-r1175) with parameters specifying ONT sequencing (-x map-ont), then converted back to bam format and filtered for primary alignment with samtools view. MM/ML tags were repaired by broadcasting modification calls from the original BAM file with modkit repair (v0.1.13). Reads were visualized in IGV (v2.18.2 and v2.19.1). To assess off-target methylation, reads were converted to fastq format with samtools fastq (v1.16.1) with the -T MM, ML option specified to preserve modification calls and re-aligned to the RPE-1 reference with minimap2 with the following options specified: -ax map-ont --eqx -y --MD. Re-aligned reads were converted back to BAM format, filtered for primary alignment and sorted with samtools view and samtools sort, respectively. A bed file defining 10,000 random 1,000 bp intervals across the RPE-1 genome was generated with bedtools random (v2.30.0). Reads corresponding to these random intervals were isolated using bedtools intersect (v2.30.0), and the output was re-indexed with samtools index. To generate rolling profile plots, modkit pileup (v0.1.13) was used to count modifications at each reference position. The resulting bedGraph file was sorted using bedtools (v2.30.0), then converted to a bigWig file with bedGraphToBigWig (v2.10). The bigWig file was used to plot rolling mCpG/CpG values with a custom Python script. To quantify the kinetics of 5mCpG-to-5hmCpG-to-CpG conversion, we analyzed a time-course dataset from DLD-1 cells generated using native barcoding nanopore sequencing (R10 chemistry). Sequencing and basecalling were performed with the same parameters described above for DiMeLo-seq. The data were processed using modkit pileup with the --cpg setting enabled, and a CpG modification threshold of 0.8 was applied to classify cytosines as methylated (mCpG), hydroxymethylated (hmCpG) or unmethylated. CpG sites were then filtered to retain only those present in reads aligning within CENP-B boxes and α-sat HORs. Cells grown on 12 mm coverslip (1.5H) were permeabilized for 1 min with 0.1 M PHEM buffer (60 mM PIPES, 25 mM HEPES, 2 mM MgCl2, 10 mM EGTA, pH 6.9) with 0.5% Triton X-100, followed by fixation for 10 min with 4% PFA (Electron Microscopy Sciences, 15710) in PBS. Both pre-extraction and fixation solution were prewarmed to 37 °C. After fixation, coverslips were washed thrice with 0.5% Triton in PBS and blocked with 3% BSA, 0.5% Triton in PBS for 30 min at room temperature. Primary antibodies diluted at 1:100 in 3% BSA were added to the coverslips and incubated for 2 h at room temperature. Next, cells were washed thrice with 0.5% Triton in PBS and incubated with DAPI (1 μg ml−1) and secondary antibodies diluted 1:100 in 3% BSA 0.5% Triton in PBS for 2 h at room temperature. Coverslips were washed thrice with PBS and prepared for ExM. Stained samples were treated with 0.1 mg ml−1 Acryloyl-X (Thermo Fisher Scientific, A20770) in PBS for 2 h, washed thrice with PBS and incubated for 5 min in monomer solution (1× PBS, 2 M NaCl, 2.5% (wt/wt) acrylamide, 0.15% (wt/wt) N,N′-methylenebisacrylamide and 8.625% (wt/wt) sodium acrylate). Coverslips were placed on top of a drop of 90 μl of freshly prepared gelation solution (monomer solution supplemented with 0.2% (wt/wt) TEMED and 0.2% (wt/wt) APS) and incubated for 1 h at 37 °C. Gels were then incubated in digestion solution (8 U ml−1 proteinase K, 1× TAE, 0.5% TX-100, 0.8 M guanidine HCl) for 2 h at 37 °C and washed in PBS containing DAPI (1 μg ml−1). Full expansion was performed in a 10-cm plate by several washings of 30 min with excess volume of Milli-Q water (~4 to 4.5-fold expansion). Expanded samples were immobilized on 25 mm (1.5H) coverslips covered with 0.01% (wt/vol) poly-l-lysin (Sigma-Aldrich, P8920) and imaged on a Nikon Ti-E motorized microscope equipped with a Zyla 4.2Mpx sCMOS camera (Andor) and 100×1.35-NA objective lens (Nikon). Images were acquired as z-stacks at 0.3 μm intervals. Images from Nikon and Zeiss systems were deconvolved with Huygens Professional (v20.10) using up to 40 iterations of the Classic Maximum Likelihood Estimation algorithm with theoretical PSF. The images were only adjusted in brightness and contrast on raw or deconvoluted data using the Fiji software. The number of subunits per single centromere was manually scored in z-stacks using Fiji. Whole cell lysates were prepared by resuspending cell pellets in 100 mM Tris–HCl pH 7.6, 4% SDS, 20% glycerol buffer and by sonication. The lysates were BCA-quantified and denatured at 95 °C in 1× Laemmli sample buffer for 5 min. Proteins were separated by SDS–PAGE using TGX gels (Bio-Rad) and transferred onto nitrocellulose membranes (0.45 μm) using the Trans-Blot Turbo transfer system (Bio-Rad). The membrane was blocked with 5% milk in TBS-T (Tris–buffered saline with 0.1% Tween 20) for 2 h at room temperature and incubated overnight at 4 °C with mouse anti-FLAG (0.8 μg ml−1; Sigma-Aldrich), rabbit anti-GAPDH (1:5,000; clone 14C10; Cell Signaling Technology), rabbit anti-HELLS (0.75 μg ml−1; Proteintech), rabbit anti-CENP-B (1 μg ml−1 polyclonal; Abcam, ab25734) or mouse antivinculin (1:2,000; clone hVIN-1, Sigma-Aldrich). Images were acquired on a ChemiDoc imager (Bio-Rad) and analyzed on Image Lab software (v6.1; Bio-Rad). Statistical analysis of all the graphs was performed using GraphPad Prism 10 (GraphPad Software, www.graphpad.com). Details, including the value of cells and/or of centromeres measured, are reported in the figure legends. Full statistical test results can be found in Supplementary Table 5. Briefly, when possible GraphPad Prism's normality test was used to determine whether data followed a normal distribution. Depending on the data distribution, unpaired t test or Mann–Whitney test were used to compare two groups, ordinary one-way ANOVA or Kruskal–Wallis test to compare more than two groups, Fisher's exact test to compare categorical data and one-sample t test to compare samples to a fixed value. No statistical methods were used to predetermine sample sizes. All experiments were performed using several biological replicates. The number of biological replicates and of measurements for each experiment is indicated in the corresponding figure legend. Several steps were taken to ensure the reproducibility of experimental findings and key results were confirmed using complementary experimental approaches. With respect to data randomization, sample assignment to treatment and control groups, as well as data collection, were performed randomly without prior assessment of any cell culture characteristics. Generally, data collection and analysis were not performed blindly to the experimental conditions; however, quantitative image analyses, which are at the heart of all presented data, were independent of experimental conditions. No data were excluded from the analyses. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. DNA sequencing data have been deposited in the Sequence Read Archive (SRA) under BioProject PRJNA1270043. Accession codes can be found in Supplementary Table 6. Other data supporting the findings of this study are available at https://doi.org/10.5281/zenodo.15919153 (ref. Source data are provided with this paper. All code written in support of this publication is publicly available at https://doi.org/10.5281/zenodo.15875037 (ref. Greenberg, M. V. C. & Bourc'his, D. The diverse roles of DNA methylation in mammalian development and disease. Karpen, G. H. & Allshire, R. C. The case for epigenetic effects on centromere identity and function. McKinley, K. L. & Cheeseman, I. M. The molecular basis for centromere identity and function. Manuelidis, L. Chromosomal localization of complex and simple repeated human DNAs. Willard, H. F. Chromosome-specific organization of human α-satellite DNA. Mellone, B. G. & Fachinetti, D. Diverse mechanisms of centromere specification. Sundararajan, K. & Straight, A. F. Centromere identity and the regulation of chromosome segregation. The quantitative architecture of centromeric chromatin. Altemose, N. et al. DiMeLo-seq: a long-read, single-molecule method for mapping protein–DNA interactions genome wide. Complete genomic and epigenetic maps of human centromeres. Willard, H. F. & Waye, J. S. Hierarchical order in chromosome-specific human α satellite DNA. Maloney, K. A. et al. Functional epialleles at an endogenous human centromere. Masumoto, H., Masukata, H., Muro, Y., Naohito, N. & Okazaki, T. A human centromere antigen (CENP-B) interacts with a short specific sequence an alphoid DNA, a human centromeric satellite. Gamba, R. & Fachinetti, D. From evolution to function: two sides of the same CENP-B coin? Gershman, A. et al. Epigenetic patterns in a complete human genome. Miga, K. H. et al. Telomere-to-telomere assembly of a complete human X chromosome. The structure, function and evolution of a complete human chromosome 8. Vukic, M. & Daxinger, L. DNA methylation in disease: immunodeficiency, centromeric instability, facial anomalies syndrome. Pappalardo, X. G. & Barra, V. Losing DNA methylation at repetitive elements and breaking bad. & Fachinetti, D. Keeping the centromere under control: a promising role for DNA methylation. Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. Nishimura, K., Fukagawa, T., Takisawa, H., Kakimoto, T. & Kanemaki, M. An auxin-based degron system for the rapid depletion of proteins in nonplant cells. Holland, A. J., Fachinetti, D., Han, J. S. & Cleveland, D. W. Inducible, reversible system for the rapid and complete degradation of proteins in mammalian cells. & Yang, W.-C. New insights into 5hmC DNA modification: generation, distribution and function. Scelfo, A. et al. Tunable DNMT1 degradation reveals DNMT1/DNMT3B synergy in DNA methylation and genome organization. Tanaka, Y., Kurumizaka, H. & Yokoyama, S. CpG methylation of the CENP-B box reduces human CENP-B binding. Dubocanin, D. et al. Conservation of dichromatin organization along regional centromeres. Chardon, F. et al. CENP-B-mediated DNA loops regulate activity and stability of human centromeres. Targeted DNA methylation in vivo using an engineered dCas9-MQ1 fusion protein. The complete diploid reference genome of RPE-1 identifies human phased epigenetic landscapes. The complete sequence of a human genome. Sacristan, C. et al. Vertebrate centromeres in mitosis are functionally bipartite structures stabilized by cohesin. Chen, F., Tillberg, P. W. & Boyden, E. S. Expansion microscopy. & Fachinetti, D. Centromere: a Trojan horse for genome stability. Specialized replication mechanisms maintain genome stability at human centromeres. Maraschio, P., Zuffardi, O., Dalla Fior, T. & Tiepolo, L. Immunodeficiency, centromeric heterochromatin instability of chromosomes 1, 9, and 16, and facial anomalies: the ICF syndrome. Ehrlich, M. The ICF syndrome, a DNA methyltransferase 3B deficiency and immunodeficiency disease. Thijssen, P. E. et al. Mutations in CDCA7 and HELLS cause immunodeficiency–centromeric instability–facial anomalies syndrome. Hartley, G. A. et al. Centromeric transposable elements and epigenetic status drive karyotypic variation in the eastern hoolock gibbon. Nishiyama, A. et al. Two distinct modes of DNMT1 recruitment ensure stable maintenance DNA methylation. O'Keefe, R. T., Henderson, S. C. & Spector, D. L. Dynamic organization of DNA replication in mammalian cell nuclei: spatially and temporally defined replication of chromosome-specific α-satellite DNA sequences. Carty, B. L. et al. Heterochromatin boundaries maintain centromere position, size and number. Mahlke, M. A. et al. Epigenetically dynamic human centromeres are maintained within a stable DNA methylation signature. Hori, T. et al. Constitutive centromere-associated network controls centromere drift in vertebrate cells. Birth, evolution, and transmission of satellite-free mammalian centromeric domains. Schotanus, K., Yadav, V. & Heitman, J. Epigenetic dynamics of centromeres and neocentromeres in Cryptococcus deuterogattii. Du, Q. et al. DNA methylation is required to maintain both DNA replication timing precision and 3D genome organization integrity. Moore, L. D., Le, T. & Fan, G. DNA methylation and its basic function. Monteagudo-Sánchez, A., Noordermeer, D. & Greenberg, M. V. C. The impact of DNA methylation on CTCF-mediated 3D genome organization. Diverse silent chromatin states modulate genome compartmentalization and loop extrusion barriers. Nishimura, K., Komiya, M., Hori, T., Itoh, T. & Fukagawa, T. 3D genomic architecture reveals that neocentromeres associate with heterochromatin regions. Ribeiro, S. A. Condensin regulates the stiffness of vertebrate centromeres. Sissoko, G. B., Tarasovetc, E. V., Marescal, O., Grishchuk, E. L. & Cheeseman, I. M. Higher-order protein assembly controls kinetochore formation. Kohli, R. M. & Zhang, Y. TET enzymes, TDG and the dynamics of DNA demethylation. The complete sequence of a human Y chromosome. Narayan, A. et al. Hypomethylation of pericentromeric DNA in breast adenocarcinomas. Wong, N. et al. Hypomethylation of chromosome 1 heterochromatin DNA correlates with q-Arm copy gain in human hepatocellular carcinoma. Barra, V. & Fachinetti, D. The dark side of centromeres: types, causes and consequences of structural abnormalities implicating centromeric DNA. Comparative methylome analysis of ICF patients identifies heterochromatin loci that require ZBTB24, CDCA7 and HELLS for their methylated state. Gamba, R. et al. Enrichment of centromeric DNA from human cells. Velasco, G. et al. Germline genes hypomethylation and expression define a molecular signature in peripheral blood of ICF patients: implications for diagnosis and etiology. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Gamarra, N. et al. DiMeLo-cito: a one-tube protocol for mapping protein-DNA interactions reveals CTCF bookmarking in mitosis. Salinas Luypaert, C. IMAGES || DNA methylation influences human centromere positioning and function (v1.0). Salinas-Luypaert, C., Dubocanin, D., Lee, J., Altemose, N. & Fachinetti, D. DNA methylation influences human centromere positioning and function. The authors acknowledge C. Bartocci (Institut Curie), W.C. Earnshaw (University of Edinburgh), S. Erhardt (Karlsruhe Institute of Technology), G. Almouzni (Institut Curie), I. Drinnenberg (Institut Curie), P.-A. Defossez (Université Paris Diderot) and C.Valenzuela (Institut Pasteur) for suggestions; and M. Kanemaki (National Institute of Genetics), H. Izquierdo (Institut Curie) and A. F. Straight (Stanford University) for sharing reagents. The authors also acknowledge the Flow Cytometry platform and the Cell and Tissue Imaging facility (PICT-IBiSA, member of the French National Research Infrastructure France-BioImaging ANR-10-INBS-04) of the Institut Curie for their technical and scientific expertise. receives salary support from the CNRS. has received support for this project from Labex ‘CellnScale', Institut Curie and the ‘Investissements d'Avenir' program launched by the French Government and implemented by ANR (ANR-10-LABX-0038 and ANR-10-IDEX-0001-02 PSL), the Émergence(s) grant 2018 from the City of Paris, the ‘ARC Labellisation Program 2019' and ANR-19-CE12-0022; ANR-21-CE15-0037. is a Chan Zuckerberg Biohub Investigator and is supported by an HHMI Hanna H. Gray Fellowship and a Pew Biomedical Scholar Award. is supported by the Searle Scholars Program. is supported by a National Science Foundation Graduate Research Fellowship under grant DGE-2146755. is supported by an NIH T32 training grant T32GM141828. Present address: Department of Biological, Chemical and Pharmaceutical Sciences and Technologies (STEBICEF), University of Palermo, Palermo, Italy Present address: Institut de Biologie Paris Seine, Sorbonne Université, CNRS, UMR7622, Paris, France These authors contributed equally: Danilo Dubocanin, Rosa Jooyoung Lee. Institut Curie, PSL Research University, Sorbonne Université, CNRS, UMR144 and UMR3664, Paris, France Catalina Salinas-Luypaert, Riccardo Gamba, Marine Grison, Leonid Velikovsky, Annapaola Angrisani, Andrea Scelfo, Marie Dumont, Viviana Barra, Therese Wilhelm & Daniele Fachinetti Danilo Dubocanin, Rosa Jooyoung Lee & Nicolas Altemose Oncode Institute, Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences (KNAW) and University Medical Center Utrecht, Utrecht, the Netherlands Lorena Andrade Ruiz & Geert J. P. L. Kops Department of Biomolecular Engineering, University of California, Santa Cruz, Santa Cruz, CA, USA European Research Institute for the Biology of Ageing (ERIBA), University of Groningen, University Medical Center Groningen, Groningen, the Netherlands Marialucrezia Losito, René Wardenaar & Floris Foijer Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar All authors participated in reviewing and editing the paper. Correspondence to Nicolas Altemose or Daniele Fachinetti. is an inventor on a patent application related to the DiMeLo-seq method. The remaining authors declare no competing interests. Nature Genetics thanks Ian Henderson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The constructs contain a cytomegalovirus (CMV) promoter, a tetracycline operator (TetO), the DNA-binding domain of CENP-B (CENP-BDBD), two nuclear localization signals (NLS), a 3× FLAG tag and a micro-AID tag (named AID). A version of the construct with a tandem P2A–T2A (tPT2A) self-cleaving peptide and EGFP was also generated. Panel a is created with BioRender.com. (b) Immunoblot analysis of the constructs' expression and degradation (anti-FLAG). (c) Representative agarose gels of COBRA digestions. (d) α-sat, CENP-B box and HSat2 methylation level of individual CpGs from DLD-1 FRT DBDTET1AID cells quantified by pyrosequencing. One biological replicate, technical duplicate. (e) Representative images of CENP-C and FLAG immunofluorescence of DLD-1 FRT DBDdTET1AID cells ±100 ng µl−1 doxycycline (DOX) for 48 h. Scale bar: 5 μm. (f) Quantification of methylated DNA (5mC) as percentage of total DNA by COBRA in DLD-1 cells normalized to before treatment. IAA for 4 days to degrade DNMT1 (blue). DNAme was measured at the pericentromeric HSat2 repeats, the hypermethylated CGI promoters of MAEL (Maelstrom Spermatogenic Transposon Silencer), TDRD6 (Tudor Domain Containing 6) genes and GPR119 (G protein-coupled receptor 119). (g) Kinetic quantification of 5mC as percentage of total C at α-satellites by COBRA in DLD-1 FRT DBD (d)TET1AID cells (from Fig. 1f) alongside LV DBDTET1AID cells clones 3 and 6 normalized to day 0 (no induction). (h) Chromosome-specific heatmap of α-sat 5mCG kinetics by nanopore sequencing. DLD-1 LV DBDTET1AID clone 6 treated with DOX, normalized to day 0. (a,b) Centromeric CENP-A, CENP-B and CENP-C levels per nucleus in DLD-1 LV DBD-TET1AID clone 3 (a) or clone 6 (b) after induction with DOX over time and 2 days degradation with IAA. (c) Centromeric CENP-A, CENP-B and CENP-C levels per nucleus in DLD-1 FRT DBD(d)TET1AID after DOX for 8 and 16 days followed by IAA for 2 days. Three biological replicates, n = 282, 122 and 127 total nuclei for TET1, n = 135, 143 and 127 for dTET1. (d) CENP-B CUT&RUN-qPCR quantification with centromere 1, 5, 19 primers. (e) Representative images of Cdt1, CENP-A and ACA immunofluorescence in DLD-1 LV DBDTET1AID clone 6 at 4 days ±DOX + 2 days IAA. (f) Centromeric CENP-A and ACA levels from (e) in Cdt1 positive (G1) and negative (S-G2) cells normalized to –DOX. (g) Representative image of CENP-A, CENP-B and CENP-C immunofluorescence in DLD-1 FRT DBDTET1AID mitotic chromosomes before and after demethylation for 16 days (left). Tukey box-and-whiskers plots of mean centromeric CENP-A, CENP-B and CENP-C levels in DLD-1 FRT DBD(d)TET1AID cells (right). Each dot represents a single centromere. Large colored dots represent the average signal intensity of all centromeres per biological replicate. (a–c,f) Box plot from the 25th to 75th percentiles, central line at median, “+” at mean, min-to-max whiskers. Small dots represent the average centromeric fluorescence intensity in a nucleus and is color coded by biological replicate. Large dots represent the average of all nuclei per biological replicate. Statistics defined by the Kruskal–Wallis test with Dunn's multiple comparisons. (a) Schematics of the experiment. DLD-1 LV DBDTET1AID clone 6 cells were demethylated by induction of the construct for 4 days with 100 ng ml−1 DOX, followed by its degradation with 500 μM IAA for 2 days. Control cells were treated with 500 μM IAA for 6 days. A pool of 3 siRNAs against CENP-B or siRNA control were transfected twice, separated by 3 days. (b) Immunoblot analysis of CENP-B protein levels in cells from (a) after the first 3 days of siRNA treatment. GAPDH is used as loading control. c, Integrated centromeric CENP-A and CENP-B signal intensity per nucleus in DLD-1 LV DBDTET1AID clone 6, treated with siRNA control or against CENP-B, ±DOX normalized to −DOX −siCENP-B. Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. Each small dot represents the average signal intensity of a single nucleus and is color coded by biological replicate. The large dots represent the average signal intensity of all nuclei analyzed per biological replicate. Three biological replicates, n = 512 (siCtrl + IAA), 362 (siCtrl + DOX), 309 (siCENP-B + IAA), 290 (siCENP-B + DOX). Kruskal–Wallis test with Dunn's multiple comparisons: ****p < 0.0001. d, Aggregate plot of 5mC/C across 10,000 random intervals 1,000 bp long across the genome in cells expressing DBDMQ1Q147LAID (purple) or DBDdMQ1AID (green). (a) Browser region showing individual CENP-A targeted DiMeLo-seq reads overlapping a CDR within the active α-sat HOR array on chr11 in CHM13 cells (population, ~70% transduction efficiency) without treatment and after 10 days of DBDTET1 expression + 2 days washout. For each sample, the same reads (gray rectangles) are copied to two tracks to display mCG (red) and mA (blue) separately. (b,c) Aggregate plots of 5mC/C (b) and m6A/A (c) across all DiMeLo-seq reads overlapping CDR boundaries, with (DOX, red) and without (NT, blue) DBDTET1 expression in CHM13 population. Left CDR boundaries are flipped. (d,e) Aggregate plots of m6A/A. (f,g) Aggregate plots of 5mC/C across all DiMeLo-seq reads overlapping the boundaries of 250 random intervals across the genome, with and without (red and blue, respectively) DBDTET1 expression for 10 days in CHM13 population (d,f) or for 4 days in the single clone (e,g) followed by 2 days of washout. (h) Representative images of CENP-C, H3K9me3 and ACA immunofluorescence in mitotic chromosomes in DLD-1 LV DBDTET1AID clone 6 cells (left). Scale bar: 5 μm (right) CENP-C and H3K9me3 line scans across mitotic centromeres. One biological replicate, 6 cells, n = 132 centromeres (NT) and 142 (DOX). (a) Cell cycle profile analysis by flow cytometry after 30 min BrdU incorporation in DLD-1 FRT and LV DBDTET1AID cells. Brown–Forsythe and Welch ANOVA with Dunnett's T3 multiple comparisons test, for results, see Supplementary Table 1. (b) Kinetics of methylated (5mC) α-satellite DNA as percentage of total DNA by COBRA in RPE-1 LV DBDTET1 clones R1 to R6 with DOX-induced expression of TET1. One biological replicate per clone. (d) Cell-growth analysis of the indicated RPE-1 clones in absence (dotted lines) or presence (continuous lines) of DOX. Curves represent the mean confluence from triplicates. DLD-1 FRT expressing active (TET1) or inactive (dTET1) DBDTET1AID. (f) Mitosis duration calculated from timelapse imaging of individual DLD-1 LV DBDTET1AID clone 6 + H2B-mRFP cells untreated (NT) or treated for 2 or 4 days with DOX. (g) Single-cell whole-genome sequencing of DLD-1 LV DBDTET1AID clone 6 cells fully methylated (4 days IAA) or after demethylation with DOX for 2 days followed by 2 days of construct degradation with IAA. Each row represents an individual cell, chromosomes are plotted as columns and centromere position is marked by dotted lines and highlighted by a circle. Colors correspond to a defined copy-number state, legend on the top right. h, Number of copy-number alterations (CNA) per cell following hypomethylation at centromeres and chromosome arms detected in g. Two-sided Fisher's exact test ****p < 0.0001, ***p = 0.0001. (a) Canonical and non-canonical centromeres in DLD-1 LV DBDTET1AID cl.6, untreated (NT) or demethylated (+DOX). Three biological replicates, n = 300 centromeres (NT), 301 (+DOX). Two-sided Fisher's exact test, ****p < 0.0001 (DOX vs. non-canonical centromeres). (b,c) Distribution of CENP-A sub-domains in (b) canonical n = 279 (NT), 217 (+DOX) and in (c) non-canonical centromeres n = 21 (NT), 84 (+DOX). Fisher's exact test between DOX and centromeres with >2 sub-domains ****p < 0.0001 (canonical), *p = 0.0443 (non-canonical). (d) Schematic of CENP-B protein domains. (e,f) Sanger sequences of (e) CENP-BKO clone 1 and (f) CENP-BKO clone 2. (g,i) CENP-B (g) and CENP-A (i) centromeric levels in DLD-1 cells untreated (NT) or +DOX to induce overexpression (OE), normalized to NT. Three biological replicates, n = 211 total nuclei (NT), 207 (DOX) for CENP-BOE, 227 (NT) and 245 (DOX) for CENP-AOE. (h) Methylated (5mC) α-satellite DNA by COBRA in DLD-1 untreated and +DOX to overexpress CENP-A/CENP-B. (j) Percentage of cells presenting micronuclei. (k) Representative images of colony formation assays in CENP-A/CENP-B overexpressing cells. (l) Ratio of colonies formed after 14 days +DOX compared to NT. (m) Centromeric γH2A.X intensity per nucleus in EdU+ DLD-1 cells, untreated or 4 days +DOX to overexpress full-length CENP-B, normalized to untreated. (n) Percentage of metaphase spreads showing reduced DAPI at the centromere in DLD-1 CENP-BKO cells without rescue or rescued with full-length (FL) CENP-B or with a mutant lacking the C-terminal dimerization domain (ΔC). Three biological replicates ±s.e.m., n = 133 total mitosis (KO), 135 (FL) and 114 (ΔC). (g,i,m) Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. Small dots: average fluorescence intensity of a nucleus. Large dots: average of all nuclei per replicate. (a) Schematic of the experiment. 5b–d) + TP53KO cells were seeded with or without 100 ng ml−1 DOX for 4 days to induce demethylation. At day 4, doxycycline was washed out. At day 5 untreated and DOX-treated single cells were sorted into 96-well plates. Between days 19 and 26 clones were expanded. Approximately one month after sorting, the single clones were screened by COBRA (b) and by immunofluorescence (c). (b) Representative agarose gel of α-sat COBRA digestions. The percentage of α-sat 5mC to total C is indicated below the corresponding lane. (c) Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. The dots represent the centromeric fluorescence intensity of CENP-A (red) and CENP-B (blue) averaged per nucleus in long-term RPE-1 clones untreated (NT1) or after demethylation (D2) as described in (a). The expected product size is indicated. UTR: untranslated region (right) agarose gel electrophoresis of the wild-type (WT) and edited HELLS locus. (b) Sanger sequence of the PCR product from (a) 5' (top) and 3' (bottom) ends of the edited HELLS locus in HCT116 HELLSmNG-AID cells depicting the in-frame knock-in of the mNeonGreen-AID tags. (d) Kinetics of α-sat, CENP-B box and HSat2 methylation level of individual CpGs HCT116 HELLSmNG-AID cells quantified by pyrosequencing. One biological replicate, technical duplicate. (e) Centromeric CENP-A and CENP-B levels in wild-type (WT) and ICF3 (mutations in CDCA7) patient-derived fibroblasts. Three biological replicates, n = 75 (WT) and 66 (ICF3) total nuclei. Box plot from the 25th to 75th percentiles, central line at median, ‘+' at mean, min-to-max whiskers. Small dots represent the average centromeric fluorescence intensity in a nucleus and are color coded by biological replicate. Large dots represent the average of all nuclei per biological replicate. (f) Representative images of colony formation assays in HELLSmNG-AID cells. Cells were seeded for the 10-day-long colony assays at days 0, 7, 14, 21, 28, 42 and 56 of pre-treatment with 5Ph-IAA (HELLS degradation) either in DMSO after washout of the 5Ph-IAA (top wells) or in continued HELLS degradation with 5Ph-IAA (bottom wells). Total days of 5Ph-IAA treatment are indicated below. Results from statistical tests for Figs. 2, 3, 5–7 detailing, in particular, P values for all nonsignificant (NS) differences. SRA accession codes for all DNA sequencing data. Unprocessed western blot for Fig. Unprocessed western blot for Fig. Unprocessed western blot for Extended Data Fig. 1b and unprocessed agarose gels (COBRA PCRs and digestions) for Extended Data Fig. Unprocessed western blot for Extended Data Fig. Cytometry gating strategy for Extended Data Fig. Unprocessed agarose gels (COBRA PCRs and digestions) for Extended Data Fig. Unprocessed agarose gel for Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Salinas-Luypaert, C., Dubocanin, D., Lee, R.J. et al. DNA methylation influences human centromere positioning and function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03885-3'>Patritumab deruxtecan in HR+HER2− advanced breast cancer: a phase 2 trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 09:31:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Antibody–drug conjugates have shown impressive clinical outcomes, particularly in metastatic breast cancer, but biomarkers to predict response and resistance remain unidentified. Here we report the results of ICARUS-BREAST01, a phase 2 study evaluating efficacy, safety and biomarkers of response and resistance to patritumab deruxtecan (HER3-DXd), in patients with HR+HER2− metastatic breast cancer, who previously progressed on CDK4/6 inhibitors and one line of chemotherapy. The study met its primary endpoint, showing an overall response rate of 53.5% (90% confidence interval [44.8–62.1%]). The most frequent adverse events were fatigue (83%), nausea (75%), diarrhea (53%) and alopecia (40%). Exploratory biomarker analysis of baseline tumor samples suggested preliminary associations between overall response rate and both HER3 spatial distribution and absence of estrogen receptor 1 (ESR1) mutations, as well as between progression-free survival and HER3 expression, pending further validation. Analysis of on-treatment tumor samples showed that treatment efficacy seems to be associated with antibody–drug conjugate intratumoral distribution and interferon response. Overall, HER3-DXd showed promising activity and manageable tolerability in patients with HR+HER2− metastatic breast cancer who progressed on CDK4/6 inhibitors. These findings highlight the need for larger trials to define HER3-DXd efficacy relative to other drugs, including antibody–drug conjugates (ClinicalTrials.gov Identifier: NCT04965766). Breast cancer remains the most frequent cancer in women, with 2.3 million new cases and 685,000 deaths worldwide in 2020 (refs. There is therefore an urgent need to develop new therapeutic agents for this disease. However, as it has been estimated that only a small proportion of the drug reaches the tumor cells, the mechanism of action of ADCs is likely to extend beyond payload delivery, with additional mechanisms contributing to their efficacy, such as the antibody-dependent cellular cytotoxicity, complement-dependent cytotoxicity and immunogenic cell death4,5,6. Furthermore, it has been shown that both the bystander effect and the systemic release of the payload, related to either linker–drug instability or antibody–linker instability, can also determine ADC activity across tumor cells with a wide range of target expression7,8,9. Trastuzumab deruxtecan (T-DXd) has substantiallyimproved survival outcomes in patients with HER2 (also known as ERBB2)-expressing breast cancer (HER2+ and HER2-low)10,11. However, there remains room to develop new ADCs and combination strategies to expand the range of targetable antigens, enhance safety profiles and further personalize ADC selection based on patient and tumor characteristics, ultimately broadening the therapeutic landscape. Furthermore, reliable biomarkers for identifying patients most likely to obtain clinical benefit from ADC therapies remain an unmet medical need and the mechanisms of action of ADCs are still not well understood4,5. This is due to the limited number of clinical trials that systematically perform baseline (BL) and on-treatment tumor biopsies to understand ADC mechanisms of action and resistance. Such serial tumor biopsy was performed in the DAISY trial, which showed that response to T-DXd was associated with human epidermal growth factor receptor-2 (HER2) spatial distribution and the level of drug internalization was correlated with the expression of HER2 (ref. Nevertheless, the impact of ADC intratumoral distribution on treatment outcome, along with the underlying mechanisms of action of most ADCs in patients, remains largely unknown. To address these gaps, we developed the ICARUS-BREAST01 study15. This academic, phase 2 trial aimed to evaluate clinical efficacy, safety and biomarkers of response and resistance to HER3-DXd, in patients with HR+HER2− advanced breast cancer, who previously progressed on CDK4/6 inhibitors and endocrine therapy and received one line of chemotherapy. In hormone receptor-positive (HR+) and HER2− advanced breast cancer, endocrine therapy with CDK4/6 inhibitors has significantly improved disease and survival outcomes and several effective therapeutic agents are available beyond progression, particularly PI3K–AKT–mTOR pathway inhibitors in combination with endocrine therapy16,17,18,19. However, about 50% of patients receiving CDK4/6 inhibitors experience disease progression over the first 2 years of treatment and become rapidly eligible for standard chemotherapy, which is often associated with significant toxicity20,21. In breast cancer, HER3 expression plays a key role in resistance to PI3K–AKT–mTOR inhibitors, HER2-targeting therapies and endocrine therapy and is associated with poorer prognosis22. HER3-DXd is a new ADC composed of an anti-HER3 monoclonal antibody conjugated to a topoisomerase-I inhibitor by a cleavable peptide linker, which has shown preliminary signs of activity regardless of breast cancer subtype and HER3-expression in earlier phase 1 and 2 studies23. From 27 May 2021 to 9 March 2023, we screened 164 patients with estrogen receptor-positive (ER+) and/or progesterone receptor-positive (PR+) and HER2− (immunohistochemistry (IHC) 2+ and in situ hybridization (ISH) negative, or IHC 1+ or IHC 0+), unresectable locally advanced or metastatic breast cancer, who had progressed on previous treatment with CDK4/6 inhibitors and endocrine therapy and received one line of chemotherapy for advanced breast cancer. A total of 99 patients were enrolled to receive HER3-DXd 5.6 mg kg−1 intravenously (i.v.) every 3 weeks until progression or unacceptable toxicity (Fig. The study primary endpoint was the confirmed objective response rate (ORR) by a local investigator, whereas secondary endpoints included progression-free survival (PFS), duration of response (DoR), overall survival (OS) and safety. To explore biomarkers of response or resistance to HER3-DXd, we collected one frozen and three formalin-fixed paraffin-embedded (FFPE) tumor samples at BL, on-treatment (corresponding with cycle 1 day 3 (C1D3), cycle 1 day 19 (C1D19) or cycle 2 day 3 (C2D3)) and at end of treatment, along with whole-blood and serum samples. The main demographic and clinical characteristics of patients at study entry are reported in Table 1. Overall, 39.4% of patients had IHC HER2-0 breast cancer, as centrally assessed on the tumor biopsy at study entry. The median number of systemic therapies for advanced breast cancer was two (range one to four). All patients, with the exception of one, had progressed on previous CDK4/6 inhibitor with a median treatment duration of 13.7 months (interquartile range (IQR) 6.5–19.7 months). At data cut-off, 16 April 2024, 19 patients were still on treatment and the median number of treatment cycles was 11.0 (IQR 6–18.0), with disease progression being the primary reason for treatment discontinuation in most patients (64.6%), as reported in Supplementary Table 1. CONSORT flowchart illustrating patient disposition throughout the study. The confirmed ORR by the local investigator was 53.5% (90% confidence interval (CI) [44.8–62.1%]), with 2 patients having obtained complete response (CR) and 51 partial response (PR). Overall, 37 patients had stable disease (SD), with a clinical benefit rate (CBR) (CR, PR or SD ≥ 6 months) of 62.6% (95% CI [52.3–72.1%]) (Fig. a, Waterfall plot showing the best percentage change in the sum of tumor diameters from BL. The bars colored green, blue, purple and red represent CR, PR, SD and PD, respectively. The response was determined by investigator assessment according to RECIST v.1.1 and required confirmation after the first observed response at least 4 weeks later. c, Kaplan–Meier estimates of duration of response as assessed by a local investigator (median 9.3 months (95% CI 8.2 months–N/A)). aPatients who developed new lesions; bClopper–Pearson (exact) method used for the 90% CI; ctwo patients not evaluable (NE) for ORR: one patient had only one tumor assessment with PR and then treatment discontinued due to clinical progression and a second patient had no evaluable global response-of-target lesions. With a median duration of the study follow-up of 15.3 months (95% CI [13.0–17.2 months]), 62 patients experienced disease progression or death. The median DoR was 9.3 (95% CI [8.15–not available (N/A)) months] (Fig. OS was not mature at the time of data cut-off. We also performed a post-hoc analysis of ORR and PFS according to HER2 expression as centrally assessed on the tumor biopsy at BL and we found an ORR of 60% (95% CI [40.6–77.3%]) in the HER2-low (1+, 2+) and of 48.7% (95% CI 32.4–65.2%) in the HER2-0 patient populations. Given the high ORR observed by the local investigator assessment, the study was also amended in November 2024 to include a retrospective independent central review of RECIST, which confirmed an ORR of 55.6% (95% CI [45.2–65.5%]). Treatment-related adverse events (TRAEs) of any grade occurred in 97 patients (98.0%) and the rate of grade ≥3 TRAEs was 50.1%. Serious TRAEs were reported in 18.2% of patients. No fatal TRAEs were reported, although one patient died while on treatment due to a massive pleural effusion not related to HER3-DXd. The most frequent TRAEs of any grade (>30% of patients) were: fatigue (83%), nausea (75%), diarrhea (53%) and alopecia (40%). The most common TRAEs and the most common treatment-emergent adverse events (TEAEs) are reported in Table 2. Note that the study protocol recommended systemic prophylaxis with a serotonin 5-HT3 antagonist in all patients and the use of an additional antiemetic as indicated per the investigator's discretion. Most cases (n = 6) were classified as grade 1, with one leading to study drug discontinuation at the investigator's decision and two were grade 2, both resulting in treatment discontinuation as per protocol. Treatment was required in four of the grade 1 cases and in both grade 2 cases. Among the six patients with available data on ILD duration, two cases resolved within approximately 3 weeks, two cases within 1 month and two cases required around 6 months for resolution. We defined the patients as responders if they achieved a confirmed CR or PR and as nonresponders if their best response was SD or progressive disease (PD). We first investigated whether the target expression and spatial distribution were associated with treatment outcome (ORR or PFS). Analysis of PFS according to HER3-membrane expression showed a trend toward a slightly longer PFS in patients with HER3-membrane positivity >75% (median(m)PFS 10.9 months (95% CI [8.2–17.7 months]) or with HER3-membrane H-score >200 (mPFS 10.9 months (95% CI [8.51 months–N/A]) compared with those with the lowest HER3-membrane scoring (mPFS 8.1 months (95% CI [4.4 months–N/A]) in HER3-membrane positivity <25%; mPFS 6.7 (95% CI [5.3 months–N/A]) in HER3-membrane H-score 0–100) (Extended Data Fig. Developing these results further, we performed a post-hoc analysis of ORR and PFS using the HER3-membrane IHC scoring 1+, 2+ and 3+ as previously reported in ref. Consistent with earlier analysis, we did not find a statistically significant association between ORR and HER3-membrane positivity; however, we observed a trend for longer PFS in patients with %TC membrane 2+ and 3+ ≥50% versus <50% (9.7 months (95% CI [8.1–15.3] months) versus 6.8 months (95% CI [5.3 months–N/A]; P = 0.126)) and in those with %TC membrane 1+, 2+ and 3+ ≥75% versus <75% (10.9 months (95% CI [8.1–17.6 months]) versus 8.1 months (95% CI [6.7 months–N/A]); P = 0.136; Fig. Nevertheless, the correlative analysis of HER3 versus efficacy should be interpreted with caution, because the limited sample size results in wide CIs. We also analyzed the correlation between HER2 and HER3 expression by IHC to determine whether their expression is associated. We did not observe any clear correlation between HER3 and HER2 expression. Indeed, tumors with ≥50% HER3+ membrane staining were found across both HER2-0 and HER2-low groups (Supplementary Fig. a, Box plot showing the proportion of HER3 1+, 2+ and 3+ TCs at BL and best overall treatment response. c, Clusters of HER3 or neighborhood cell distribution. These regions were then segmented into eight different clusters. Box plots show the proportion of each cluster in 63 evaluable patients, according to treatment response: responders (CR, PR) and nonresponders (SD, PD). Our analysis revealed an association between treatment response and the percentage of cluster 0 (P = 0.040), with an odds ratio of 1.527 95% CI [1.020–2.287]. For all these analyses, using R v.4.1.2, we applied Dirichlet's regression to identify which clusters were significantly associated with the objective response to treatment (statistical significance was assessed using Wald's test based on Dirichlet-distributed parameters; the reported P values were unadjusted for multiple comparisons). d, H&E-stained slide showing the composition of cluster 0, characterized by a moderate number of TCs, surrounded by connective areas, with a few immune cells and no necrotic areas. e, Proportion of the different cell phenotypes (epithelial, connective, inflammatory, necrotic and neoplastic areas) across the identified clusters (n = 63). We further explored whether the spatial distribution of HER3+ cells was associated with response to the treatment. After the pathologist's review, we analyzed 69 IHC or hematoxylin and eosin (H&E) slides. First, we performed machine learning analysis on digitized, annotated, DAB-stained slides (membrane HER3 expression) by using an unsupervised clustering algorithm, as previously done in the DAISY trial14. Using two different models to extract patch representations, we did not find any significant association between the identified clusters and response to HER3-DXd. Afterwards, to map the distribution of HER3+ cells relative to neighboring cells, we superimposed digitized, whole DAB-stained slides with digitized H&E-stained slides from the same sample, capturing the architecture of the tumor microenvironment (TME) (n = 63, because 6 samples were excluded as a result of suboptimal registration quality control). We developed a new pipeline to identify epithelial, connective, inflammatory and necrotic areas, together with HER3+ and HER3− areas (Methods). These regions were then segmented into eight different clusters, the proportion of which was compared between responders and nonresponders. Our analysis revealed an association between treatment response and the percentage of cluster 0 (odds ratio (OR) 1.527; 95% CI [1.020–2.287]; P = 0.040; Fig. According to the pathologist's review, cluster 0 was characterized by isles containing a moderate number of HER3+ cells, surrounded by connective areas enriched with blood vessels, with few immune cells and no necrotic areas (Fig. To explore whether cluster 0 was associated with better tumor vascularization, we used ERG (ETS-related gene) IHC staining to selectively identify endothelial cells. We found a significant positive association between cluster 0 and number of ERG+ cells (P = 0.0275; Supplementary Fig. 3); however, this association did not remain significant after multiple testing correction and should be interpreted with caution. We further evaluated the proportion of cluster 0 in tumor samples collected at progression and we did not find any statistically significant difference in the proportion of cluster 0 at baseline and that at progression; however, these results should be interpreted with caution because only 11 tumor samples were analyzable at disease progression (Supplementary Fig. Although cluster 0* was composed of only connective tissue and so lacked biological rationale, cluster 1* seemed to share similar properties with the previously identified cluster 0, featuring TCs surrounded by connective areas, few immune cells and no necrotic areas (Supplementary Fig. Next, we analyzed whether genomic alterations at BL were associated with response to HER3-DXd. We carried out whole-exome sequencing (WES) on frozen tumor biopsies at BL (n = 43). Nonsynonymous point mutations, insertions or deletions (indels), homozygous deletions, loss of heterozygosity and low, medium and high-level amplifications were assessed. Analyses of WES were focused on these genes (Supplementary Table 3). It is interesting that we found a higher proportion of ESR1 mutations in nonresponders 52.9% (9 of 17) compared with responders 23.1% (6 of 26), with all 4 patients with PD as best response presenting ESR1 alterations at baseline. Finally, we found a higher frequency of FGFR1 amplifications (5 high, 3 medium and 1 low level) in responders, 34.6% (9 of 26), including the 2 patients with a confirmed and prolonged CR, compared with nonresponders, 23.5% (4 of 17) (Extended Data Fig. WES analysis according to the CBR, as defined by CR, PR or SD ≥ 6 months confirmed the same results (Extended Data Fig. We also performed the analysis of 21 mutational signatures with activity in at least 2% of breast cancer samples, showing an association between APOBEC-related SBS2 and treatment response (Supplementary Fig. Next we investigated the association between gene expression at BL and PFS, by performing bulk RNA sequencing (RNA-seq) on 56 BL frozen tumor biopsies. The adaptive lasso model procedure led a five-gene model for prediction of PFS, which contained RAB40AL, MAGEB17 and MKRN7P (P < 0.05). Note that RAB4 proteins are localized in early endosomes and regulate early endosomal recycling, determining a fast transport of internalized receptors from the endosomes back to the plasma membrane24. Therefore, the association of higher expression of RAB4 with poor prognosis suggests that higher recycling of internalized HER3-DXd back to the plasma membrane is detrimental to its efficacy (Supplementary Fig. Finally, given the higher proportion of ESR1mut (mutated) in nonresponders, we tried to better characterize the ESR1mut and ESR1wt (wild-type) group. We found that the median number of genomic alterations along with tumor mutational burden were similar in the two groups (Supplementary Table 4). We also compared the proportion of luminal tumors in the ESR1mut and ESR1wt groups, by performing PAM50-intrinsic subtyping on available bulk RNA-seq data (n = 14 ESR1mut, n = 23 ESR1wt) and we did not find any difference between the two groups (χ2 test, P = 0.57) (Supplementary Table 4). Nevertheless, gene set enrichment analysis (GSEA) using the Hallmark gene sets showed activation of estrogen pathways in ESR1mut compared with ESR1wt tumors (Supplementary Fig. We also aimed to evaluate the potential mechanisms of action of HER3-DXd and determine whether distinct mechanisms of action or pharmacodynamic events could explain treatment response. For this purpose, we analyzed tumor samples collected at BL and during treatment by using imaging mass cytometry (IMC) and bulk RNA-seq to explore, first, the intratumoral distribution and the effects of HER3-DXd on TCs and its role in gene expression modulation and, second, the effects of HER3-DXd on the TME. By IMC, a multiple protein analytical method that relies on metal-tagged antibodies to simultaneously detect multiple proteins and provides spatial analysis, we detected the expression of 23 proteins (Supplementary Table 5 and Supplementary Fig. 10) at subcellular resolution (Hyperion) on FFPE tumor samples. We profiled 61 FFPE samples at BL and 57 on-treatement (on-T) (C1D3, C1D19 or C2D3), of which 39 were matched pairs. The heatmap of TC and immune cell density at baseline is shown in Supplementary Fig. First, we explored the intracellular distribution of HER3-DXd in TCs and its association with treatment outcome, by using an anti-DXd antibody that selectively binds the DXd moiety linked to the antibody. The HER3-DXd expression was not found at BL, whereas a high proportion of both pan-keratin-positive (PANK+) and pan-keratin-negative (PANK−) cells were positive for intact-DXd in on-T tumor biopsies, with the highest values at C1D3 (Fig. We quantified HER3-DXd+ cells across cell phenotypes (PANK+, PANK−, PANK+HER3+ and PANK+HER3−) at C1D3 and found a similar proportion of HER3-DXd+ cells among PANK+ (median: 10.4 (IQR 1.3–22.9)) and PANK− (median 10.2 (IQR 1.7–37.2)) cells, also with a comparable proportion of HER3-DXd+ cells among the PANK+HER3+ (median 11.2 (IQR 0.9–26.5)) and the PANK+HER3− (median 9.6 (IQR 1.3–24.6)) cells (Supplementary Table 6). Nevertheless, the interpretation of these data should take into account that the proportion of HER3+ cells significantly reduced at C1D3 compared with BL (see below). As analyzable samples at C1D3 (n = 20) were all from patients who achieved CR, PR or SD, we then looked at the mean tumor shrinkage according to the proportion of PANK+HER3-DXd+ cells. We defined HER3-DXd positivity as having ≥5% positive cells. This threshold was chosen because, at BL (before any drug exposure), the maximum percentage of HER3-DXd+ cells observed was 3.75%, likely due to background noise. Even with the limitation of the small sample size, we found that mean tumor shrinkage was −56.2% (s.d. = 20.4%) in patients with PANK+HER3-DXd+ cells ≥5% (n = 11) and PANK+HER3-DXd+ cells <5% (n = 9) at cycle 1 day 3 (Mann–Whitney U-test, U = 17, P = 0.013), respectively (Fig. = 22.8%) in patients with HER3-DXd+ cells above and below the median staining (U = 25, P = 0.063) (Supplementary Fig. Although HER3-DXd positivity does not necessarily mean that an ADC has been internalized, we sought to evaluate whether the HER3-DXd positivity was associated with alterations or different expression of genes involved in the endocytosis pathway (Clathrin, Dynamin2, Rab, Numb, Caveolin, LAMP1 and CD71). The genomic profile at BL was available for 7 of 11 patients with HER3-DXd+ cells ≥5% (HER3-DXd+ tumor) and 4 of 9 patients with HER3-DXd+ cells <5% (HER3-DXd− tumor). Transcriptomic profile at BL was available for 8 of 11 patients with HER3-DXd+ tumors and 5 of 9 with HER3-DXd− ones. Differential expression analysis of RNA-seq data between the two groups did not reveal any significant difference of the expression of genes related to endocytosis (adjusted P or Padj > 0.05). As a specific tumor architecture potentially favoring the ADC intratumoral distribution (cluster 0) appeared to be linked to treatment response, we also analyzed the proportion of cluster 0 at BL in tumors with HER3-DXd positivity ≥5% and <5%, and did not find any statistically significant association (Supplementary Fig. a, Dynamics of HER3-DXd+ cells over the different time points. b, Mean tumor shrinkage in patients with ≥5% of HER3-DXd+ cells and those with <5% HER3-DXd+ cells at C1D3 (Student's t-test, P = 0.0146). PANK+ cells are shown in red and Intact-DXd+ cells in blue; similar staining patterns were observed across all analyzed samples (n = 20). d,e, Volcano plot of differentially expressed genes in paired on-T and BL tumor samples in responders (d) and nonresponders (e). The P values shown are from two-sided tests and were adjusted for multiple comparisons using the Benjamini–Hochberg method. f, Volcano plot of differentially expressed genes in on-treatment samples of HER3-DXd+ (≥5% TCs, n = 5) and HER3-DXd− (<5% TCs, n = 7) tumors. The P values shown are from two-sided tests and were adjusted for multiple comparisons using the Benjamini–Hochberg method. Next, we characterized the dynamics of HER3+ tumor cells after HER3-DXd administration and their association with response. We observed that HER3 expression dramatically decreased after HER3-DXd administration, with a greater decrease in responders (Student's t-test, P = 0.011; Extended Data Fig. Then, to determine the effect of the ADCs on TCs, we analyzed on-T gene expression changes to identify which genes were modulated by HER3-DXd. We carried out bulk RNA-seq on 22 pairs of BL and on-T frozen tumor biopsies, collected 3 d or 19 d after cycle 1 or 3 d after cycle 2, selected to have tumor cellularity ≥30%. We conducted differential gene expression (DGE) analyses looking at the fold-change of gene expression of matched on-T BL samples in the overall population (n = 22) and, then, separately in responders (confirmed CR and PR; n = 14) and nonresponders (SD, PD and not evaluable; n = 8). A higher proportion of up- and downregulated genes was observed in responders (n = 410 genes) compared with nonresponders (n = 5 genes) after one or two doses of HER3-DXd (Padj ≤ 0.05; Fig. As most of the downregulated genes could be related to decreased cell proliferation, we further focused on upregulated genes to explore genes and pathways modulated by HER3-DXd. The top ten upregulated genes in responders were EDA2R, IFI44L, RTP4, GBP1P1, IFIT1, IFIT2, AIM2, SELL, CLEC4E and IFIT3, with log2(fold-change) between 0.88 and 0.68 (Padj ≤ 0.05). Most of these genes are involved in the interferon (IFN) signaling pathway. We observed a larger number of upregulated genes in HER3-DXd+ compared with HER3-DXd− tumors (Padj ≤ 0.05; Fig. We further explored whether genes modulated by the drug administration clustered in specific pathways, by running a GSEA. In the overall population, using the gene set Hallmarks, we observed an activation of pathways involved in immune response, particularly IFNα and IFNγ (gene ratio 0.45) and complement signaling (gene ratio 0.3; Padj ≤ 0.05; Extended Data Fig. Likewise, activation of pathways of immune response, notably IFNα and IFNγ, and complement signaling was found in responders (Padj ≤ 0.05; Extended Data Fig. 4b), whereas, in nonresponders, no gene set was modulated by treatment exposure (Padj > 0.05). We also compared the pathways modulated on treatment between responders and nonresponders and confirmed that activation of immune-related pathways was enriched in responders compared with nonresponders (Padj ≤ 0.05; Extended Data Fig. However, we did not find any upregulation of IFN pathways in on-T HER3-DXd+ compared with HER3-DXd− tumors (Extended Data Fig. Finally, among the 54 Kyoto Encyclopedia of Genes and Genomes (KEGG) gene sets modulated by the treatment, we could identify that the ‘endocytosis' gene set was suppressed on treatment (gene ratio 0.36; Padj ≤ 0.05), a finding possibly related to an adaptation of cancer cells to therapy (Supplementary Fig. We also performed GSEA using the gene set Hallmarks according to CBR and, consistent with the previous results, we found activation of immune-related pathways in patients with CBR (n = 16) compared with those without (n = 6; Padj ≤ 0.05; Supplementary Fig. Given the IFN response observed in sensitive patients, we evaluated PFS in patients with upregulation of the IFNα or IFNγ pathway versus those presenting a stable or downregulated pathway. We observed a trend toward a longer PFS in patients with either α (P = 0.26) or γ (P = 0.04) pathway activation (Extended Data Fig. To this end, we investigated how HER3-DXd modulated the immune cell composition in the tumor and TME. Despite the activation of IFNα and IFNγ pathways being associated with a better treatment outcome (ORR, CBR or PFS), we found both increases and decreases of T cells with no clear differences between responders and nonresponders. We also observed that CD68+ cell density was decreased in the TME after one or two doses of HER3-DXd, without any difference between responders and nonresponders (Extended Data Fig. We finally performed immune deconvolution analysis using CIBERSORTx_ABSOLUTE on paired on-T versus BL tumor samples (n = 22) to assess changes in immune cell subpopulations. However, we did not observe any significant increase or decrease in specific immune cell subsets in responders (n = 14) compared with nonresponders (n = 8), suggesting that HER3-DXd did not induce major shifts in the immune composition of the TME (Supplementary Fig. In the ICARUS-BREAST01 study, HER3-DXd showed considerable efficacy in patients with HR+HER2− advanced breast cancer who had previously progressed on CDK4/6 inhibitors and one line of chemotherapy. Most of the patients achieved a confirmed treatment response (ORR 53.5%) and experienced prolonged disease control. In addition, HER3-DXd showed a manageable safety profile, with a low rate of grade ≥3 TRAEs and TRAEs leading to treatment discontinuation. Notably, the rate of ILD adjudicated as drug related was low (8.1%), most being grade 1 and thus allowing treatment recovery at ILD resolution, except in three cases. T-DXd was approved by the US Food and Drug Administration for patients with inoperable or metastatic HR+HER2-low breast cancer after at least one line of chemotherapy and, more recently, also for patients with HR+HER2-low or HER2-ultralow breast cancer that has progressed after at least one endocrine therapy in the metastatic setting, based on the remarkable results of the DESTINY-Breast04 and DESTINY-Breast06 trials10,25. For these patients, T-DXd is not an option if HER2 expression is <1% (about 20% of HER2-nonoverexpressing tumors)25,26 and the efficacy of T-DXd in the HER2-ultralow setting was based on exploratory analyses of the DESTINY-Breast06 study. Therefore, the ICARUS-BREAST01 findings emphasize the need for a large phase 3 study in patients with HR+HER2− breast cancer, who can no longer benefit from endocrine therapy, to determine the efficacy of HER3-DXd in HER2-null tumors and also in HER2-low or HER2-ultralow breast cancers, in which it could be a valuable alternative to T-DXd, considering some differences in the safety profile. This study would also provide an opportunity to further validate the preliminary findings, suggesting a positive trend between HER3-enriched populations and PFS, which helps to refine patient selection strategies. Furthermore, although the ICARUS-BREAST01 study excluded patients previously treated with ADCs, because they were designed before the results of the pivotal trials of DESTINY-Breast04, TROPICS-02 and DESTINY-Breast06 (refs. Previous clinical data showed that ADC resistance may sometimes derive from reduced target expression, while preserving payload sensitivity27,28,29. Leveraging these preclinical and clinical data, we designed the ICARUS-BREAST02 study (NCT06298084), which is currently evaluating the efficacy and tolerability of HER-DXd either as a single agent or in combination with olaparib in patients who have progressed on previous treatment with T-DXd. Although there was no association between ORR and HER3 expression, we found a trend toward a longer PFS in patients with higher HER3 expression. This trend was consistently observed across all three scoring methods. Therefore, although this association did not reach statistical significance, potentially due to the small sample size, and does not allow for the definition of a clear HER3 expression threshold, it remains a promising finding that warrants further investigation. It is interesting that we did not find a strong association between HER2 and HER3 expression, because there are tumors with high HER3 expression that have HER2-0 or HER2-low expression, reinforcing the interest in further determining the efficacy of HER3-DXd in both settings. Furthermore, using machine learning analysis on IHC and H&E slides, we found that the distribution of HER3+ TCs within tumor tissue may play a role in determining a better ADC penetration and distribution. Indeed, tumors with a higher proportion of cluster 0, and thus containing a moderate number of HER3+ cells and being surrounded by connective tissue areas and no necrotic areas, had a higher likelihood of responding to the treatment. It is important to note that the association between cluster 0 and treatment outcome may not be driven solely by the spatial distribution of HER3+ cells relative to neighboring cells. Instead, it could be the result of a specific tumor architecture favoring the ADC intratumoral distribution, independent of HER3 expression. Indeed, when HER3 staining was excluded from the clustering algorithm, two response-associated clusters were identified, of which one (1*) shared features with cluster 0, including TCs surrounded by connective tissue, sparse immune cells and absence of necrotic areas, and the other (0*) containing only connective tissue, interpreted as not biologically relevant. The composition of the 0* clusters and lack of significance of similar clusters in the initial analysis led us to interpret it as a compositional artifact, in contrast to clusters 0 and 1* sharing similar biological characteristics with a more straightforward interpretation. As Dirichlet-regression models have relative, rather than absolute, cluster proportions, significant clusters can also emerge due to shifts in other clusters, not necessarily reflecting direct association with treatment response. Given the limited sample size and novelty of the mechanisms, these analyses remain nevertheless exploratory and hypothesis generating, requiring validation in independent cohorts to confirm the relevance of HER3 spatial mapping. We also investigated whether driver mutations in breast cancer and genomic alterations of interest in DXd activity (for example, TOP1mut) could be associated with the response to treatment. We did not find any association between response to HER3-DXd and known breast cancer driver mutations or alterations in genes involved in HER3-DXd activity, except for ERBB3mut which was more frequent in responders (11.5%) versus nonresponders (5.9%). Although most ERBB3mut changes occur in the extracellular domain, previous preclinical data did not show differences in HER3-DXd activity across cell lines with or without ERBB3mut30. Furthermore, we observed that tumors harboring ESR1mut were more likely to be resistant to HER3-DXd (52.9% in nonresponders and 23.1% in responders). Although ESR1mut tumors are resistant to aromatase inhibitors and tend to be more heavily pretreated, we did not find any significant difference in the clinical characteristics of patients with ESR1mut tumors and those with ESR1wt tumors, such as the number of previous systemic therapies and time from initial diagnosis to study entry. Likewise, the median number of genomic alterations along with tumor mutational burden were similar in the two groups. However, given the small sample size, these data should be interpreted with caution. Previous preclinical studies demonstrated a bidirectional crosstalk between the ER signaling pathway and the HER3 or HER2 pathway in ER+ breast cancer31. For instance, it has been shown that exposure to tamoxifen or fulvestrant, which inhibits ER signaling, also increases epidermal growth factor receptor, HER2 and HER3 expression32,33. Therefore, it is possible that, in ESR1mut tumors, the constitutive activation of the ER pathway, as we found in our cohort, can be associated with downregulation of HER3. Furthermore, although no differences in the PAM50-subtype assignment were observed between ESR1mut and ESR1wt tumors in our cohort, it is likely that ESR1mut tumors exhibit more luminal characteristics which may reduce the response to HER3-DXd. This aligns with the observed higher prevalence of TP53 mutations in responders and previous clinical findings, because these mutations are associated with more proliferative and chemosensitive tumor profiles34. With this objective, we analyzed on-treatment tumor samples by IMC and bulk RNA-seq. IMC analysis revealed that tumors with a higher proportion of PANK+HER3-DXd+ cells 3 d after cycle 1 had a deeper treatment response. Although HER3-DXd+ cells at IMC do not necessarily indicate that the ADC has been internalized, we can hypothesize that a greater ability of ADCs to penetrate tumor tissues could allow a higher ADC uptake by TCs and ultimately increase cell death. DGE analysis further supported these findings, showing a greater number of modulated genes in responders compared with nonresponders and also in HER3-DXd+ tumors compared with HER3-DXd− ones. These preliminary findings raise the hypothesis that the ADC tumor penetration and cell internalization may play a critical role in determining ADC efficacy. For example, new technologies for the detection of ADC uptake in circulating tumor cells (CTCs) could represent a significant advancement in monitoring ADC efficacy in real-time. Furthermore, gene expression analysis of matched on-T or BL samples showed upregulation of genes involved in immune response, particularly IFNα and IFNγ, and complement signaling significantly enriched in the entire cohort, in particular among responders. It is possible that DXd-induced DNA damage activates the cGAS–cGAMP–STING pathway, leading to the upregulation of type I IFNs in tumor cells, which has been associated with better treatment outcomes (ORR and PFS). These findings suggest that a greater ADC intratumoral distribution can lead to a greater ADC intracellular uptake and DNA damage, resulting in an IFN response in tumor cells35,36. However, in our study, the exploration of immune TME composition by IMC on paired tumor samples at BL and on-T showed a notable T cell expansion and activation (increase of CD4+, CD8+, CD8+GzmB+ and CD8+CD107a+) after 3 d from cycle 1 in only two patients who achieved tumor response to the treatment. Although IFNα and IFNγ activation in tumors may indicate a response to treatment, the absence of immune cell expansion is likely driven by the immunosuppressive nature of the TME and also by additional contributing factors, such as the low density of immune cells at BL (as shown in Supplementary Table 9 and Supplementary Fig. 11), likely related to the predominance of liver metastases in the biopsy samples, known for their inherently low immune infiltration, and the possibility that the on-T time point was not optimal for capturing the immune cell expansion that is likely to happen later on. Therefore, it would be worthwhile exploring, in the future, IFNα and IFNγ activation in CTC, as an early predictor of optimal drug–tumor interactions and, ultimately, treatment response, so helping to refine patient selection and treatment strategies. Our study has several limitations that should be acknowledged. First, the sample size was relatively small and the study was a nonrandomized single-arm trial, which lacks the rigor of randomized controlled trials, and treatment response has not been confirmed by a blinded independent central review. Nevertheless, the high efficacy of HER3-DXd demonstrated in the ICARUS-BREAST01 study, also confirmed by the central review, certainly warrants being confirmed in larger trials in similar settings. Second, we included a higher proportion of patients with HER2-0 breast cancer than we would have expected37, which was due to the almost simultaneous activation of an expanded access program for T-DXd in HER2-low breast cancer in France (https://www.has-sante.fr/jcms/p_3341817/fr/enhertu-trastuzumab-deruxtecan-cancer-du-sein-her2), which limited the inclusion of these patients. Although these patients' characteristics could affect the generalizability of the clinical results, we did not find significant differences in treatment outcome between patients with HER2-0 and those with HER2-low breast cancer. However, the high proportion of tumor samples collected at BL and on-T in this study offers a unique opportunity for generating a hypothesis on some of the key mechanisms of response and resistance to HER3-DXd. In conclusion, the ICARUS-BREAST01 study shows a substantial efficacy and a manageable safety profile of HER3-DXd in patients with HR+HER2− breast cancer that make it an optimal candidate for further larger studies in this setting. Although limited by the nature of purely exploratory analyses, our findings generate a hypothesis on the mechanism of action of this ADC, suggesting that target expression and spatial distribution, and ADC intratumoral distribution, intracellular uptake and subsequent DNA damage, may influence treatment response. These findings warrant further investigations and underline the importance of further studies, employing similar approaches, to define biomarkers that will allow optimal patient selection for a given ADC. Despite the challenges associated with serial sample collection and analysis, such studies are nevertheless important in providing a comprehensive understanding of the mechanisms of action of new ADCs. The ICARUS-BREAST01 study (NCT04965766) is an academic, single-arm, phase 2 study which has been conducted across 11 cancer centers or hospitals in France. The following are the key eligibility criteria: female or male patient aged ≥18 years; locally advanced or metastatic (MBC) that was HR+ (ER+ and/or PR+) and HER2− (IHC 2+ and ISH negative, or IHC 1+ or IHC 0+) at the time of the first diagnosis (if the tumor were ER+ and/or PR+ at initial diagnosis and became ER− and/or PR− in the following biopsies performed while receiving previous standard treatments for MBC, the patient could be included); progression on endocrine therapy and CDK4/6 inhibitors; must have received one line, but not more than one, of chemotherapy for MBC; may have received previous treatment with PI3K inhibitors, mTOR inhibitors, AKT inhibitors and poly(ADP) ribose polymerase inhibitors; must have metastatic site easily accessible to biopsy (with the exception of bone metastasis) and have accepted the performance of pre-treatment and on-treatment biopsies; and Eastern Cooperative Oncology Group performance status 0 or 1. The study was initially designed to include only patients with HER3 membrane expression ≥75% with 10× in-tumor biopsies at BL; however, this inclusion criterion was deleted by amendment on 21 April 2022, after inclusion of the first 29 patients; afterwards recruitment proceeded regardless of HER3 expression. This decision was taken because data across different tumor types, including breast cancer, showed HER3-DXd clinical benefit across a broad range of tumor HER3 expression levels23. Patients with clinically inactive or treated brain metastases who were asymptomatic were eligible, whereas patients with any history of ILD (including pulmonary fibrosis or radiation pneumonitis), current confirmed or suspected ILD or with clinically severe intercurrent pulmonary illnesses or previous treatment with an ADC containing a topoisomerase-I inhibitor payload were excluded. Mandatory tumor biopsies were performed (1 frozen + 3 FFPE): pre-treatment (baseline), on-T and at the end of treatment. For on-T tumor biopsies, patients were divided into three groups to undergo a single on-T biopsy at the following time points: group 1 (34% of patients) at C1D3 ± 1 d, group 2 (33% of patients) at C1D19 ± 1 d and group 3 (33% of patients) at C2D3 ± 1 d (1 frozen + 3 FFPE). Serum and blood samples were also collected at multiple time points, which are detailed in the first and last versions of the protocol provided in Supplementary Information. Patients received treatment with HER3-DXd 5.6 mg kg−1 every 3 weeks until investigator-assessed radiological progression (per RECIST v.1.1), unacceptable toxicity, withdrawal of consent or any other discontinuation criterion indicated in the protocol. The protocol was submitted to the competent ethics committee (CPP), which gave its approval on 12 March 2021. This study was also approved by the competent authority on 29 April 2021 and performed in accordance with the ethical principles of the Declaration of Helsinki and consistent with the International Conference on Harmonisation Good Clinical Practice guidelines and other applicable regulatory requirements. All patients provided written informed consent before study participation. The null hypothesis was that the treatment with HER3-DXd yields an ORR ≤ 12%; 99 patients were required to provide 85% power to test this null hypothesis at a one-sided 5% significance level, assuming an ORR of 23% under the alternative hypothesis. The primary endpoint, ORR, was defined as the proportion of patients who achieved a confirmed CR or PR, as assessed by local investigators, without initiation of new anticancer treatment. Confirmation of response had to be demonstrated with an assessment 4 weeks or later after the initial response. The treatment objective response was radiologically assessed every 6 weeks using RECIST v.1.1. For equivocal findings of progression (for example, very small and uncertain new lesions, cystic changes or necrosis in existing lesions), treatment was allowed until the next scheduled assessment. Tumor imaging assessment was performed every 6 weeks (±7 d) starting at 6 weeks from the first dose for the first 12 months and then every 12 weeks (±7 d). Secondary endpoints were: PFS by investigator assessment, DoR, CBR, OS and safety. CBR was defined as the presence of at least a PR or CR, or a SD for ≥6 months under treatment. A waterfall plot was also used to present each patient's (unconfirmed or confirmed) ORR according to their variation for the minimum sum of the patient's tumor size from baseline. PFS was defined as the time from the date of the first dose until documented disease progression or death from any cause, whichever occured first. For patients with no documented radiological progression, follow-up was censored at the date of last radiological assessment without progression, unless death occurred within 12 weeks of the date of last known progression free, in which case the death was counted as a PFS event. Follow-up time was calculated using the reverse Kaplan–Meier method38. Safety endpoints (AEs, AESIs, serious adverse events (SAEs), TEAEs and TRAEs), defined by the National Cancer Institute Common Terminology Criteria for Adverse Events v.5.0 (NCI-CTCAE v.5.0) were described as maximum grade for each patient by the CTCAE term. TEAEs are defined as those AEs with a start or worsening date on or after the start date of study treatment until 47 d after the end date of study treatment (that is, on-T period). SAEs starting or worsening after the on-T period, if reported as related to the study treatment, were also counted as TEAEs. Proportions of treatment discontinuation, interruptions and dose reductions due to any AEs were also described. Data management was performed using TrialMaster software compliant with 21 CFR Part 11. The same software was used for patient registration. Continuous variables were presented by their mean and s.d. All enrolled patients received at least one dose of study drug so that the full analysis set and safety analysis set correspond to the same population. To assess the assumption of linearity, splines were used. All additional tests beyond the primary test were two sided; R/SAS 9.4 was used for all analyses. DEG of RNA-seq data was performed using R packages Deseq2 and glmmseq and the Benjamini–Hochberg method was used to control for the false discovery rate (Padj < 0.05). The study protocol was amended on 15 April 2022 to include patients with tumors presenting any HER3-membrane expression level at study entry and, thereafter, on 26 November 2024 to add a central review of RECIST response. HER3 staining was centrally performed on 72 samples by Roche CDx CAP/CLIA Laboratory (Tucson) using clone SP438 and scored as follows: membrane H-score: 0–300; cytoplasm H-score: 0–300; overall membrane positivity at ×10 magnification: 0–100. Overall, 4 samples at baseline were not available (not provided by the participating centers) and 23 samples were excluded from the analysis due to tumor cellularity <10%. Centralized HER2 assessment on tumor biopsies at BL was performed by a GEFPICS (Groupe d'Etude des Facteurs Pronostiques par Immunohistochimie dans le Cancer du Sein)-trained pathologist using the Roche HER2 (4B5) clone and scored according to the last version of the American Society of Clinical Oncology (ASCO) and College of American Pathologists (CAP) guidelines39. ‘Ultralow' HER2 category was defined as cases showing a faint-to-weak, incomplete membrane staining in <10% of tumor cells (classified in the IHC 0 category following the ASCO and CAP guidelines). A total of 69 samples was available for centralized HER2 assessment, because 4 samples were not provided and the remaining 26 had insufficient tumor tissue for further analysis. Pairs of H&E and IHC slides stained for HER3 expression were collected at BL and analyzed by machine learning algorithms to identify primary markers of resistance. On the pathologist's review, 69 pairs containing sufficient tumor tissue and TME were selected for analysis. The same preprocessing operations were used to delineate the tissue of interest within annotations provided by an expert pathologist, highlighting tumor tissue and its surrounding environment. Nonoverlapping patches of size 80 × 80 pixels2 were extracted to account for the different resolution at which the slides were scanned in our study. Using the Davies–Bouldin index, we determined an optimal number of clusters to be seven for the DAISY-like embeddings and eight for the CONCH embeddings for a number of clusters ranging from seven to twelve. Subsequently, we applied the mini-batch K-means algorithm to these optimal cluster numbers and computed the cluster percentages. To further integrate tissue microenvironment in the analysis, we aimed to leverage the information available from both H&E and IHC slides. To this end, we first extracted microenvironment features from the H&E slide using the CellViT model41. Specifically, we employed the CellViT-SAM-H model with HoVer-Net decoder42 trained on the Pannuke dataset43 to segment and classify nuclei into five classes: neoplastic, epithelial, inflammatory, connective and dead. The slides were serialized, with a difference of up to 15 μm between H&E and IHC slides, providing a solid foundation for joint modality analysis. Originally developed on surgical cuts, we adapted the method for biopsy slides with several preprocessing steps. These regions were then cropped to ensure the same scale on both modalities. The resulting images were subsequently processed using a preprocessing operator based on color deconvolution from a K-means clustering, with adaptive histogram equalization. After registration, we identified six pairs of slides with a relative target registration error above one, which we considered too high to be exploitable in our next analysis. For feature extraction, we performed the following steps: from the H&E slide, we extracted the number of cells for each class and computed several morphological features (area, perimeter, compactness, roundness, solidity and elongation) averaged across each class within each patch. The feature vectors from both modalities were concatenated, normalized and fed into a K-means algorithm, after which we computed cluster percentages on each slide. We switched to the Calinski–Harabasz score to identify the optimal number for clusters because it provided a more precise indication in this case. For a number of clusters ranging from 7 to 12, we identified an optimal number of 8 clusters. For all these analyses, using R v.4.1.2, we applied Dirichlet's regression to identify which clusters were significantly associated with the objective response to treatment. Statistical significance was assessed using Wald's test based on Dirichlet-distributed parameters. The second analysis involved investigating HER3 distribution within TCs. First, we segmented the nuclei on the IHC slides by extracting patches of size 512 × 512 pixels2 with an overlap of 20 μm and removed black and white patches. A contrast correction operation was applied to enhance nuclei segmentation, involving clamping values >0.3 and rescaling the value within this new range. The resulting nuclei segmentation was then exported to QuPath v.5.0.0 (ref. We used the provided tools to estimate stain decomposition vectors and cell membranes and to measure staining distribution within the segmented cells. For estimating the cell membranes, we used the provided cell expansion tool with a limit at 20 pixels from nuclei boundaries. Finally, we removed negative cells and/or cells with a background detected in the cytoplasm and mean DAB cytoplasm optical density variance across the cells for each patient. We compared the distribution of this value between nonresponders and responders using a Mann–Whitney U-test. Finally, among the 63 slides used to compute the clustering, 48 were compared with slides stained for ERG expression. Using QuPath software, we identified positive and negative cells within these slides. The absolute number of ERG+ cells in analyzed tissue was then compared with the number of patches belonging to each cluster for each patient with Spearman's correlation test and P values adjusted for multiple hypothesis testing using the Benjamini–Hochberg method. The tumor samples were qualified for WES and bulk RNA-seq if the sample contained ≥10% and 30% TCs, respectively. In total, 43 frozen tumor biopsies at baseline were analyzed for WES and 43 blood samples were used as germline control. Overall, at baseline, 15 fresh biopsies were either not collected or not provided by the participating centers, 28 were excluded due to <200 ng of DNA or <10% TCs and 13 failed the quality control. At BL, 15 fresh biopsies were not provided by the participating centers, 28 were excluded due to <200 ng of RNA or <30% TCs, 5 failed the quality control and 29 did not have the matched on-T sample. On-T, 22 fresh biopsies were unavailable because they were either not provided or not collected by the participating centers, 39 were excluded due to <200 ng of RNA or <30% TCs, 1 sample failed the quality control and 15 did not have the matched BL sample. DNA and RNA extraction Genomic DNA was isolated from blood using the DNeasy Blood and Tissue Kit (QIAGEN, cat. DNA or RNA dual extraction from the tissue biopsy was done using the Allprep DNA/RNA Micro kit (QIAGEN, cat. 80284) to purify DNA along with TRIzol LS Reagent (Thermo Fisher Scientific, cat. The DNA quality was checked on the Agilent Fragment Analyzer (Agilent Technologies) and the quantity was determined using Qubit dsDNA Broad Range Assay (Invitrogen, cat. Genomic DNA, 200 ng, was sheared with the Covaris E220 system (LGC Genomics or Kbioscience) to obtain 150-bp fragments according to the manufacturer's instructions. Briefly, DNA fragments were end-repaired, extended with an ‘A' base on the 3′-end, ligated with paired-end adapters with the Bravo Platform (Agilent Technologies) and amplified to generate libraries (ten cycles). Hybridization-based exome enrichment was performed using the Agilent SureSelectXT Low Input Clinical Research Exome v.2 target enrichment system (Agilent Technologies). The final libraries were indexed, pooled and sequenced using the onboard cluster method, as paired-end sequencing (2× 100 bp) on an Illumina NovaSeq-6000 sequencer at Gustave Roussy. A SureSelect Automated Strand Specific RNA Library Preparation Kit was used according to the manufacturer's instructions with the Bravo Platform (Agilent Technologies). Briefly, 200–300 ng of total RNA per sample was used for poly(A) messenger RNA selection using oligo(dT) beads and subjected to thermal mRNA fragmentation. The final libraries were indexed, purified, pooled together in equal concentrations and subjected to paired-end sequencing (2× 100 bp) on a Novaseq-6000 sequencer (Illumina) at Gustave Roussy. Point mutations, small indels and copy-number alterations (CNAs) were detected using an end-to-end pipeline on the raw WES files. In brief, paired-end reads were controlled (FastQC v.0.11.8), trimmed (Fastp v.0.20)47 and aligned to the reference human genome GRCh37 (BWA-MEM v.0.7.17)48. Files with aligned reads were processed following the best practices of GATK bundle v.4.1.8.1. As advised in GATK guidelines, we used a ‘panel of normal' to remove artifactual or false-positive mutations recurrently found in normal blood samples. Point mutations and small indels were called using Mutect2 (ref. All putative variants identified by Mutect2 were first filtered to account for possible sample contamination and read orientation artifacts. Additional threshold- and rule-based filtering was applied to the read coverage, genomic position and variant allele frequency. CNA, tumor purity and average tumor ploidy were identified with the FACETS R package v.0.5.14 (ref. To mitigate the effect of segmentation errors, only gene CNAs arising from segments spanning <10 Mb were considered in downstream analyses. Each CNA was classified into one of six categories and only high-level focal amplifications or homozygous focal deletions were considered. CNAs were categorized into five classes, with thresholds adjusted for the presence of whole-genome duplication (WGD): high-level gain was defined as six or more copies without WGD or ten or more copies with WGD; medium-level gain as four to five copies without WGD or seven to nine copies with WGD; low-level gain as three copies without WGD or five to six copies with WGD; homozygous deletion as zero copies; and loss of heterozygosity as one to two copies overall and zero copies for the minor allele without WGD, or one to four copies overall and zero copies for the minor allele with WGD. Oncogenic events were identified by intersecting point mutations and small indels with the OncoKB51 database and gene amplifications or deletions with the OncoKB or CIViC52 databases. Point mutations from all 43 WES samples were summarized into a matrix of mutation counts with 96 rows for the 96 trinucleotide mutation categories, totaling 6,280 point mutations. Only samples with at least 50 detected somatic point mutations were used for the mutational signature analysis, resulting in the selection of 36 out of 49 samples with WES data available. Mutational signature activities in each sample were determined by projecting this matrix onto a shortlist of reference signatures from the SBS signatures COSMIC v.3.2 database. The shortlist was established by downloading known signature activities in The Cancer Genome Atlas and Pancancer Analysis of Whole Genomes cohorts (available at https://www.synapse.org/Synapse:syn11804065) and selecting signatures showing an activity in at least 2% of breast cancer samples in any of the two cohorts, or of particular interest to our cohort (signatures SBS31, SBS35, SBS44 and SBS90). Overall, 21 signatures were selected and used as a reference set to perform the deconvolution using MutationalPatterns R package53 v.3.12.0. 54 to run the sparsity-inducing step occurring after a first signature activity assignment by non-negative matrix factorization. Raw Illumina reads were demultiplexed using bcl2fastq v.2.20 (https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html). Quality control of reads was performed with fastQC v.0.11.8 (https://www.bioinformatics.babraham.ac.uk/projects/fastqc) and contamination assessment with Fastq_Screen v.0.15.3 (https://zenodo.org/doi/10.5281/zenodo.1344583) against the default genomes database. All quality control reports were aggregated using MultiQC v.1.14 (ref. Bad sequence quality and adapter trimming was performed using fastp v.0.23.2 (ref. Transcript-level quantification was performed using Salmon v.1.4.0 (ref. Sample-level quantification were aggregated using the tximport v.1.32.0 (https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html) package on R v.4.4.1. Genes that had fewer than five counts for any of the compared condition levels were discarded. Differential analyses using an interaction term were performed using the glmmSeq (https://cran.r-project.org/web/packages/glmmSeq/vignettes/glmmSeq.html) R package v.0.5.5, GSEA, clusterProfiler v.4.12.0 package and multiple term databases in their latest version (MSigDb v.7.1, WikiPathways, KEGG, Reactome, DisGenet, Disease Ontology, Gene Ontology, Network of Cancer Genes). All P values from any statistical test performed were false recovery rate adjusted using the Benjamini–Hochberg method58. To investigate the prognostic signature for PFS, we used z-scores to avoid the influence of highly expressed genes. We used transcriptomic data obtained by RNA bulk analyses from 50 frozen biopsies at BL after passing quality control. We performed univariate Cox's proportional hazard models for each transcription factor using the Survival package in R. We generated a prognostic signature, using an adaptive lasso method with a preliminary step in the model subject to the ridge penalty. A 5× crossvalidation was used to determine the optimal value of λ. We repeated the adaptive lasso selection 100× and selected the transcription features with repeated occurrence frequencies >80%. The analysis were performed using R package biospear59. PAM50 molecular subtypes were determined using normalized transcriptomic data transcripts per million through the following steps. PAM50-specific normalization: given that all samples in our cohort were HR+HER2−, a tailored normalization approach was applied, as recommended for subtype prediction in this specific context (the normalization was performed using the publicly available code: https://github.com/afernan4/PAM50_ER_HER2_normalization). We analyzed, using IMC, 61 samples at BL and 57 on-T. At baseline, 4 samples were not available because they were not provided by the participating center, 6 samples were excluded for insufficient tumor cellularity (0%) and 28 were nonanalyzable due to inadequate tumor staining (technical issues). All antibodies used in IMC were first validated by immunofluorescence using breast and lung cancer tissues to determine optimal staining and specificity. Validated antibodies were conjugated to metal isotopes and tested by IMC to ensure staining specificity and intensity. They were then further validated in multiplexed IMC experiments and optimal concentrations of all metal-conjugated antibodies were determined by analysis of IMC images in breast cancer tissues (Supplementary Table 5). FFPE tumor slides were cut at 5 mm and incubated at 60 °C for 2 h before they were dewaxed in xylene, rehydrated in 100% alcohol and washed in water. Tumor tissues were incubated in antigen retrieval (Tris-EDTA, pH 9.0, 96 °C for 30 min) and left on the bench for 30 min to cool down. Next, slides were washed in water, then in phosphate-buffered saline (PBS), before blocking with 3% bovine serum albumin in PBS for 45 min and incubation with metal-tagged antibodies overnight at 4 °C. Subsequently, slides were washed with PBS and 0.1% Triton, then they were incubated with Cell-ID intercalator-Ir 125 µM diluted in PBS and 0.5% bovine serum albumin for DNA detection (Fluidigm, cat. Finally, slides were washed with water and air dried. To facilitate a comprehensive analysis, the pathologist marked the regions of interest (ROIs) on each slide. Hence, three ROIs were identified on each slide and then delineated using the IMC (Supplementary Fig. Before the acquisition, the Hyperion mass cytometry system IMC was auto-tuned using a three-element tuning slide according to the tuning protocol provided by Standard BioTools. ROIs with sizes of 1 mm2 (1,000 × 1,000 µm2) were ablated and acquired at 200 Hz. The data obtained from each scanned spot, where each antibody corresponded to a single image per sample, were processed. These individual images were then combined to create a multi-image stack. Raw data were visualized and converted to tiff format using the Standard BioTools MCDTM viewer. Next, ROIs were converted to TIFF concatenated images using Fiji software and cell segmentation was performed on the iridium channel for each ROI of each patient using the Qupath Stardist module. This segmentation process allowed the identification of individual cells and the extraction of single-cell distribution information. Using QuPath, a training image composed of several images from the dataset was used to classify regions and cell phenotypes. To discriminate the tumor region from the stroma, a pixel classifier tool was trained on a pan-cytokeratin marker. Then, to classify each cell population, an object classifier tool was trained by manually selecting positive and negative cells. The random Forest algorithm was applied to classify these detections (Supplementary Fig. In addition, spatial analysis was conducted to explore the interaction between immune cells and TCs. CTCs were enumerated using CellSearch (Menarini Silicon Biosystems) on 7.5 ml of blood according to the manufacturer's protocol as previously described60. Using CellSearch, we obtained CTC count data for 36 patients, with 22 having CTC assessments at each time point. Blood samples, 28 ml, were collected in CellSave tubes (Menarini Silicon Biosystems). Negative selection of CTCs was performed using the RosetteSep Human CD36 Depletion Cocktail (STEMCELL Technologies, cat. 15167) according to the manufacturer's protocol as previously reported61,62,63. The enriched cell fraction was washed with 1× PBS and centrifuged for 5 min at 558g. The cell pellet was suspended in 100 μl of fixative solution medium A from the Fix&Perm kit (Thermo Fisher Scientific, cat. After centrifugation the cell pellet was resuspended with 100 μl of permeabilization solution medium B from the Fix&Perm kit and anti-HER3 monoclonal antibody (2 µg ml−1, Thermo Fisher Scientific, clone RTJ2, cat. MA1-860), and incubated for 20 min at room temperature. After a 1× PBS wash the secondary goat anti-mouse AF488 antibody (Life Technologies, cat. After a 1× PBS wash, the cell pellet was resuspended in 300 μl of 1× PBS and kept at +4 °C. The system was run at 138 kPa pressure, a 100-μm nozzle and yield precision mode. Cell sorting was started by gating Hoechst+ elements. The second gate enabled selection of CD45-APC− events. For FACS, CTC count data were available for 34 patients, with 27 having 3 assessments at different time points. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. De-identified clinical data can be requested by filling out the data request form for Gustave Roussy's clinical trials at https://redcap.link/DataRequestClinicalTrialsGustaveRoussy. The anticipated timeframe for response is about 2 weeks. In case of approval, a specific agreement between the sponsor and the researcher may be required for data transfer. De-identified whole-genomic and transcriptomic data and digital pathology data have been deposited to the European Genome–phenome Archive (accession no. Digital pathology data are available via GitHub at https://github.com/gustaveroussy/ICARUS-01_Public. Other data that support the findings of this study are available from the corresponding author upon reasonable request. Breast cancer: global patterns of incidence, mortality, and trends. Z., Modi, S. & Chandarlapaty, S. Unlocking the potential of antibody–drug conjugates for cancer therapy. & André, F. Optimizing the potential of antibody–drug conjugates in oncology. Parallel genomic alterations of antigen and payload targets mediate polyclonal acquired clinical resistance to sacituzumab govitecan in triple-negative breast cancer. Peters, C. & Brown, S. Antibody–drug conjugates as novel anti-cancer chemotherapeutics. Tarcsa, E., Guffroy, M. R., Falahatpisheh, H., Phipps, C. & Kalvass, J. C. Antibody-drug conjugates as targeted therapies: are we there yet? A critical review of the current clinical landscape. Hoogenboezem, E. N. & Duvall, C. L. Harnessing albumin as a carrier for cancer therapies. Colombo, R., Tarantino, P., Rich, J. R., LoRusso, P. M. & De Vries, E. G. E. The journey of antibody–drug conjugates: lessons learned from 40 years of development. Modi, S. et al. Trastuzumab deruxtecan in previously treated HER2-low advanced breast cancer. André, F. et al. Trastuzumab deruxtecan versus treatment of physician's choice in patients with HER2-positive metastatic breast cancer (DESTINY-Breast02): a randomised, open-label, multicentre, phase 3 trial. Overall survival with sacituzumab govitecan in hormone receptor-positive and human epidermal growth factor receptor 2-negative metastatic breast cancer (TROPiCS-02): a randomised, open-label, multicentre, phase 3 trial. Bardia, A. et al. Datopotamab deruxtecan versus chemotherapy in previously treated inoperable/metastatic hormone receptor-positive human epidermal growth factor receptor 2-negative breast cancer: primary results from TROPION-Breast01. Mosele, F. et al. Trastuzumab deruxtecan in metastatic breast cancer with variable HER2 expression: the phase 2 DAISY trial. Pistilli, B. et al. 189O A phase II study of patritumab deruxtecan (HER3-DXd), in patients (pts) with advanced breast cancer (ABC), with biomarker analysis to characterize response to therapy (ICARUS-BREAST01). Gennari, A. et al. ESMO Clinical Practice Guideline for the diagnosis, staging and treatment of patients with metastatic breast cancer. Chemotherapy and targeted therapy for endocrine-pretreated or hormone receptor-negative metastatic breast cancer: ASCO Guideline rapid recommendation update. Turner, N. C. et al. Capivasertib in hormone receptor-positive advanced breast cancer. Kalinsky, K. et al. Randomized phase II trial of endocrine therapy with or without ribociclib after progression on cyclin-dependent kinase 4/6 inhibition in hormone receptor-positive, human epidermal growth factor receptor 2-negative metastatic breast cancer: MAINTAIN trial. Updated results from MONALEESA-2, a phase III trial of first-line ribociclib plus letrozole versus placebo plus letrozole in hormone receptor-positive, HER2-negative advanced breast cancer. Overall survival with ribociclib plus endocrine therapy in breast cancer. Papa, F. et al. Long road towards effective HER3 targeting in breast cancer. Krop, I. E. et al. Patritumab deruxtecan (HER3-DXd), a human epidermal growth factor receptor 3-directed antibody–drug conjugate, in patients with previously treated human epidermal growth factor receptor 3-expressing metastatic breast cancer: a multicenter, phase I/II trial. Bardia A. et al. Trastuzumab deruxtecan after endocrine therapy in metastatic breast cancer. Curigliano, G. et al. Trastuzumab deruxtecan (T-DXd) vs physician's choice of chemotherapy (TPC) in patients (pts) with hormone receptor-positive (HR+), human epidermal growth factor receptor 2 (HER2)-low or HER2-ultralow metastatic breast cancer (mBC) with prior endocrine therapy (ET): primary results from DESTINY-Breast06 (DB-06). Bardia, A. et al. Datopotamab deruxtecan in advanced or metastatic HR+/HER2− and triple-negative breast cancer: results from the phase I TROPION-PanTumor01 study. Abelman, R. O. et al. Sequential use of antibody-drug conjugate after antibody-drug conjugate for patients with metastatic breast cancer: ADC after ADC (A3) study. Huppert, L. et al. Multicenter retrospective cohort study of the sequential use of the antibody-drug conjugates (ADCs) trastuzumab deruxtecan (T-DXd) and sacituzumab govitecan (SG) in patients with HER2-low metastatic breast cancer (MBC). Koyama, K. et al. Patritumab deruxtecan (HER3-DXd), a novel HER3 directed antibody drug conjugate, exhibits in vitro activity against breast cancer cells expressing HER3 mutations with and without HER2 overexpression. Thrane, S., Lykkesfeldt, A. E., Larsen, M. S., Sorensen, B. S. & Yde, C. W. Estrogen receptor α is the major driving factor for growth in tamoxifen-resistant breast cancer and supported by HER/ERK signaling. Hutcheson, I. R. et al. Fulvestrant-induced expression of ErbB3 and ErbB4 receptors sensitizes oestrogen receptor-positive breast cancer cells to heregulin β1. Liu, B. et al. Downregulation of erbB3 abrogates erbB2‐mediated tamoxifen resistance in breast cancer cells. Brasó-Maristany, F. et al. Patritumab deruxtecan in HER2-negative breast cancer: part B results of the window-of-opportunity SOLTI-1805 TOT-HER3 trial and biological determinants of early response. Oh, K.-S. et al. Immunomodulatory effects of trastuzumab deruxtecan through the cGAS-STING pathway in gastric cancer cells. Tarantino, P. et al. HER2-low breast cancer: pathological and clinical landscape. Schemper, M. & Smith, T. L. A note on quantifying follow-up in studies of failure time. Wolff, A. C. et al. Human epidermal growth factor receptor 2 testing in breast cancer: ASCO–College of American Pathologists Guideline update. Lu, M. Y. et al. A visual-language foundation model for computational pathology. Hörst, F. et al. CellViT: vision transformers for precise cell segmentation and classification. Graham, S. et al. Hover-Net: simultaneous segmentation and classification of nuclei in multi-tissue histology images. & Rajpoot, N. in Digital Pathology (eds Reyes-Aldasoro C. C. et al.) 11–19 (Springer, 2019). Virtual alignment of pathology image series for multi-gigapixel whole slide images. Le Bescond L., et al. Unsupervised nuclei segmentation using spatial organization priors. In Medical Image Computing and Computer Assisted Intervention (eds Wang, L. et al.) 325–335 (Springer Nature Switzerland, 2022). Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Shen, R. & Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. Krysiak, K. et al. A community approach to the cancer-variant-interpretation bottleneck. Ewels, P., Magnusson, M., Lundin, S. & Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Ternès, N., Rotolo, F. & Michiels, S. biospear: an R package for biomarker selection in penalized Cox regression. Farace, F. et al. A direct comparison of CellSearch and ISET for circulating tumour-cell detection in patients with metastatic carcinomas. Pailler, E. et al. Acquired resistance mutations to ALK inhibitors identified by single circulating tumor cell sequencing in ALK-rearranged non-small-cell lung cancer. Circulating tumor cell copy-number heterogeneity in ALK-rearranged non-small-cell lung cancer resistant to ALK inhibitors. Mezquita, L. et al. Resistance to BRAF inhibition explored through single circulating tumour cell molecular profiling in BRAF-mutant non-small-cell lung cancer. We thank all the patients, families, investigators and study personnel who participated in this study. We also acknowledge Daiichi Sankyo, for supporting this study and the whole ICARUS program. This work, as part of IHU PRISM National PRecISion Medicine Center in Oncology, also received state funding managed by the French National Research Agency under the France 2030 program (grant no. We are grateful to A. Fernandez-Martinez for her expert assistance and insightful contributions to the transcriptomic analysis. These authors contributed equally: Fernanda Mosele, Noemie Corcos. Department of Medical Oncology, Gustave Roussy, Villejuif, France Barbara Pistilli, Fernanda Mosele, Jacqueline Deneuve & Fabrice André Barbara Pistilli, Noemie Corcos, Rasha Cheikh-Hussin & Guillaume Montagnac IHU PRISM National PRecISion Medicine Center in Oncology, Gustave Roussy, Villejuif, France Barbara Pistilli, Fernanda Mosele, Noemie Corcos, Livia Pierotti, Yoann Pradat, Loic Le Bescond, Magali Lacroix-Triki, Bastien Job, Françoise Farace, Marianne Oulhen, Diep T. N. Tran, Nathalie Droin, Maria Vakalopoulou, Stergios Christodoulidis, Elsa Bernard, Stefan Michiels, Fabrice André & Guillaume Montagnac Fernanda Mosele, Yoann Pradat, Loic Le Bescond, Bastien Job, Françoise Farace, Marianne Oulhen, Diep T. N. Tran, Elsa Bernard & Fabrice André Office of Biostatistics and Epidemiology, Gustave Roussy, Oncostat INSERM U1018, Université Paris-Saclay, Ligue Contre le Cancer, Villejuif, France Livia Pierotti, Pierre Guyader, Yves Koudou & Stefan Michiels Loic Le Bescond, Hugues Talbot & Maria Vakalopoulou Department of Pathology, Gustave Roussy, Villejuif, France Projects and Promotion Division, Gustave Roussy, Villejuif, France UMS AMMICa, CNRS UAR 3655, INSERM US23, Gustave Roussy, Villejuif, France Gustave Roussy, Université Paris-Saclay, ‘Rare Circulating Cells' Translational Platform, CNRS UMS3655 – INSERM US23 AMMICA, Villejuif, France Department of Medical Oncology, Institut Paoli Calmettes, Marseille, France Department of Medical Oncology, Oncopole Claudius Regaud, Toulouse, France Department of Medical Oncology, Centre Léon Bérard, Lyon, France Department of Medical Oncology, Centre Antoine Lacassagne, Nice, France Department of Medical Oncology, Tenon Hospital, APHP, Paris, France Department of Medical Oncology, Curie Hospital, Paris, France Department of Medical Oncology, Centre Georges François Leclerc, Dijon, France Department of Medical Oncology, Centre Hopital-Universitaire, Limoges, France Andrea Sporchia, Fumitaka Suto, Lie Li, David W. Sternberg & Dalila Sellami Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar has received consulting fees from Novartis and Pegascy. has received consulting fees from AstraZeneca and Daiichi Sankyo. has received consulting fees, honoraria for lectures and presentations and support for attending meetings from and participation on data safety monitoring for AstraZeneca, GSK, Eisai, MSD, Lilly, Pfizer, Novartis, Daiichi Sankyo and Seagen. has received consulting fees from Daiichi Sankyo; honoraria for lectures and presentations from Exact Sciences; support for attending meetings from Novartis, Lilly and Seagen; and participation on data safety monitoring for MSD, AstraZeneca, Novartis, Lilly, Esai, Exact Sciences, Daiichi and Sankyo. has received grants or contracts from other entities, such as AstraZeneca (institution), Pfizer (institution), SeaGen (institution) and Novartis (institution); honoraria for lectures and presentations from SeaGen, Novartis, Pfizer and Lilly; support for attending meetings from Roche, AstraZeneca, Daiichi Sankyo, Pfizer and Novartis; and participation on data safety monitoring for AstraZeneca, Daiichi Sankyo, SeaGen, Novartis, Pfizer and Lilly. has received consulting fees from Daiichi Sankyo and AstraZeneca. is a data and safety monitoring member of clinical trials for IQVIA, Kedrion, Biophytis, Servier and Yuhan and a scientific committee study member of an observational study for Roche. has received research funding and speaker or advisor honoraria (compensated to the hospital) for Roche, AstraZeneca, Daiichi Sankyo, Pfizer, Novartis and Eli Lilly. The remaining authors declare no competing interests. Nature Medicine thanks Aditya Bardia, Leif Ellisen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ulrike Harjes, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. ORR and PFS according to HER3 membrane positivity as measured by overall membrane positivity at 10x and scored as < 25%, 25-75% and > 75% of tumor cells; b. ORR and PFS according to HER3 membrane positivity as measured by H-score 0-100, 100-200, ≥ 200; the 95% confidence interval was estimated using the Clopper–Pearson exact method. (a) Oncoplot showing non-synonymous point mutations, indels, homozygous deletions, loss of heterozygosity (LOH) and low, medium, and high-level amplifications in the set of 73 genes of interest, according to treatment response: responders (confirmed CR, PR), n = 26; non-responders (SD, PD, NE), n = 17.Blood samples were available for all patients. (b) Oncoplot showing non-synonymous point mutations, indels, homozygous deletions, loss of heterozygosity (LOH) and low, medium, and high-level amplifications in the set of 73 genes of interest, according to CBR: CBR (confirmed CR, PR, SD > 6 months; n = 27; non CBR (unconfirmed PR, SD < 6 months, PD; n = 16). Blood samples were available for all patients. (a, b) Pie chart showing the proportion of the different cell phenotypes among the HER3-DXd-positive cells at C1D3 in responders (a) and non-responders (b); (c) Dynamics of HER3-positive cells after 1 or 2 doses of HER3-DXd in responders and non-responders (n = 39), showing a greater reduction on-treatment of HER3-positive cells in responders as compared to non-responders (Mann-Whitney U test, p-value 0.011). (a, b) Gene set enrichment analysis (GSEA), using the Hallmark gene set in the overall population (a) and in responders (b) showed upregulation of genes involved in immune response, particularly interferon alpha and gamma and complement signaling; (c) GSEA, using the Hallmark gene sets: Key activated/suppressed pathways in on treatment/baseline matched biopsies by comparing responders (n = 14 pairs) vs. non responders (n = 8 pairs), showing on-treatment activation of immune pathways in responders as compared non responders; (d) GSEA, using the Hallmark gene sets: Key activated/suppressed pathways in HER3-DXd-positive (HER3-DXd-positive cells ≥ 5%, n = 5) and HER3-DXd-negative tumors (HER3-DXd-positive cells < 5%, n = 7) on-treatment. (a) Kaplan-Meier curve estimating PFS according to up or stable/downregulation of interferon alfa (p-value = 0.258) (n = 22); (b) Kaplan-Meier curve estimating PFS according to up or stable/downregulation of interferon gamma (p-value = 0.045). (a–d)Heatmaps showing density of CD4 + (a), CD8 + (b), C8 CD107a + (c) and CD8 GzmB+ cells (d) at baseline and on-treatment in matched samples of responders and non-responders; (e) Heatmaps showing density of CD68+ cells in responders and non-responders. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Pistilli, B., Mosele, F., Corcos, N. et al. Patritumab deruxtecan in HR+HER2− advanced breast cancer: a phase 2 trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-62255-2'>Identification of Neuritin 1 as a local metabolic regulator of brown adipose tissue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 09:10:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Brown adipose tissue (BAT) plays a key role in metabolic homeostasis through its thermogenic effects and the secretion of regulatory molecules. Here we report that RAP250 haploinsufficiency stimulates BAT in mice, thus contributing to a decrease in fat accumulation. Local in vivo AAV-mediated RAP250 silencing in BAT reduces body weight and fat mass and enhances glucose oxidation, thereby indicating that RAP250 participates in the regulation of BAT metabolic activity. Analysis of the mechanisms led to the finding that Neuritin 1 is produced and released by brown adipocytes, it plays a key metabolic role, and it participates in the enhanced BAT metabolic activity under RAP250 deficiency. Forced overexpression of Neuritin 1 in UCP1-expressing cells markedly decreases fat mass and body weight gain in mice and induces the expression of thermogenic genes in BAT. Neuritin 1-deficient brown adipocytes also shows a reduced β-adrenergic response. We demonstrate a metabolic role of BAT-derived Neuritin 1 acting through an autocrine/paracrine mechanism. Based on our results, Neuritin-1 emerges as a potential target for the treatment of metabolic disorders. Obesity has reached epidemic proportions, affecting 650 million people worldwide (https://www.who.int/news-room/fact-sheets/detail/obesity-and-overweight). Comorbidities of obesity—most notably type 2 diabetes mellitus (T2D), atherosclerosis, hypertension, and hyperlipidemia—are now major public health concerns1. Recent data also indicate that obesity is associated with a more severe clinical course of COVID-19 and risk of fatality2,3. Thus, there is an urgent need to delineate the precise mechanisms that underlie the onset of obesity and metabolic syndrome. Brown adipose tissue (BAT) is an organ specialized in performing non-shivering thermogenesis, and it is present in adult human subjects4,5,6. The capacity of thermogenic fat to oxidize substrates and increase energy expenditure has drawn growing interest as a therapeutic approach to tackle obesity and associated metabolic disorders7. BAT thermogenesis requires the release of norepinephrine from local sympathetic axons; norepinephrine activates the β-adrenergic receptor-cAMP-PKA pathway in brown adipocytes to drive lipolysis and thermogenic respiration8. The release of norepinephrine from sympathetic nerves and β3-adrenergic receptor binding to brown adipocytes determine the level of thermogenesis in BAT by regulating the activity of lipases, providing the immediate fuel molecules for BAT mitochondria and heat production8. Recent data indicate that brown adipocytes secrete multiple signaling molecules, named batokines, including FGF21, Myostatin, CXCL14, EPDR1 and adiponectin, among others, which operate as endocrine factors9,10,11,12,13,14. Regarding the locally-acting factors, some of them are produced by vascular cells, while others are generated by brown adipocytes15,16,17,18. The action of these local factors is highly variable, ranging from effects promoting neurite outgrowth from sympathetic neurons to direct effects on the metabolism of brown adipocytes. RAP250 (encoded by NCOA6 gene) is a co-activator of nuclear hormone receptors. It also transactivates other transcription factors such as c-Jun, c-Fos, ATF-2, CBP, heat shock factor 1, SRF, E2F-1, Rb, NFκβ, and C/EBPα19. RAP250 is also an integral and unique component of a Set1-like complex named ASCOM (for ASC-2 complex), which contains histone H3-Lys-4-methyltransferase (H3K4MT) MLL3 or its paralogue MLL420,21. RAP250 is a pleiotropic modulator that affects growth, development, reproduction, apoptosis, and wound healing22. MLL3Δ/Δ mice show decreased adiposity and enhanced insulin sensitivity, although the underlying target tissues were not studied28. In this work, we focused on the metabolic effects caused by the haploinsufficiency of RAP250. We report that RAP250 modulates the thermogenic activity of BAT in mice. Surprisingly, BAT from RAP250-deficient mice showed enhanced expression of Neuritin 1 (Nrn1), a protein that impacts on the central nervous system29,30,31,32. AAV-mediated expression of Nrn1 in brown adipose cells caused metabolic effects leading to a reduction in fat mass and the induction of BAT thermogenic genes. The repression of Nrn1 in brown adipose cells disrupted the capacity of these cells to respond to a β-adrenergic receptor agonist. Based on previous reports indicating that RAP250 regulates metabolic homeostasis23,24,25,26,27,28, we examined whether it plays a relevant role in energy metabolism. To this aim, we monitored the phenotype of male mice heterozygous for RAP250 (RAP250+/) since homozygous knockout mice show prenatal lethality33. Heterozygous mice showed an almost 50% decrease in RAP250 expression in inguinal (ING) adipose tissue, brown (BAT) adipose tissue and skeletal muscle (Fig. Control (wild-type, WT) and RAP250+/− male mice showed a similar body weight at 10 weeks of age but later diverged, the latter being leaner, despite increased food intake (Fig. Thirty-week-old male RAP250+/− mice also showed smaller inguinal (ING) and perigonadal (PAT) adipose tissues compared with control animals (Fig. However, no differences in heart, skeletal muscle or liver weight were detected (Fig. We monitored the gain of adipose tissue volume by microCT in several sets of male mice. Data revealed that the differences in the size of subcutaneous and visceral depots (the latter defined as the sum of omental, mesenteric, retroperitoneal and perigonadal depots) were fully established at 30 weeks of age and were maintained thereafter (Fig. In keeping with a reduced adipose mass, male RAP250+/− mice showed a markedly reduced adipose cell size (Fig. 1H, I) and low plasma leptin levels (Fig. Total DNA present in ING or PAT adipose tissues was similar in control and RAP250+/− male mice (Fig. 1K), thereby suggesting no changes in cellularity. Thirty-seven-week-old but not thirty-week-old female RAP250+/− mice showed a reduced body weight compared to controls (Fig. ING or PAT adipose depots were also reduced in female RAP250+/− mice at the age of 37 weeks (Fig. Total DNA present in ING or PAT adipose tissues was similar in control and RAP250+/− female mice (Fig. A Body weight curves for 12- to 25-week-old male mice. Data are expressed as grams of body weight (WT n = 6, black open symbols; RAP250+/− n = 5, red symbols, and labeled as RAP250). B Food intake as grams of daily intake in 20- to 30-week-old mice (n = 11). Data are expressed as grams of tissue weight (n = 10). D Representative images of PAT and ING depots from WT and RAP250+/− male mice. E Transversal gray scale image of mouse abdominal region showing the subcutaneous adipose mass (SAT) and visceral adipose mass (VAT); vertebrae are seen as the denser white structures; least dense areas represent the adipose mass, and the layer that separates the visceral compartment from the subcutaneous region can be observed. F MicroCT adipose volume estimation of SAT, VAT and total abdominal adipose volume (n = 24) of 30-week-old male mice. Data are expressed as percentage of fat volume (femoral head to L5 vertebra). H PAT paraffin slices stained with Hematoxylin/Eosin (scale bar=100 µm). I PAT adipocyte cell size of 30-week-old male mice (n = 5, 52–200 cells counted per animal). J Plasma leptin in 16-h fasted 30-week-old male mice (n = 22 for WT, n = 18 for RAP250+/). K Total DNA in ING and PAT depots of male mice. Data are expressed as μg of DNA per depot (in ING depots n = 9 for WT and n = 6 for RAP250+/−, in PAT depots n = 10 for WT and n = 7 for RAP250+/−). Statistical differences according to a two-sided Student´s t test (B, C, F, I, J and K) and Ordinary two-way ANOVA (A and G). Next, we analyzed the response of RAP250+/− mice to a high-fat diet (HFD). Male mice subjected to a 60% fat diet gained body weight throughout the experiment (Fig. S2A), and RAP250+/− mice consistently remained leaner than control counterparts (Fig. S2A) under conditions in which food intake was similar in both genotypes (Fig. HFD-treated RAP250+/− mice also showed a smaller visceral fat depot (Fig. S2C) and lower plasma leptin levels compared to control animals (Fig. Female RAP250+/− mice also showed a lower body weight and reduced adipose depots after a HFD (Fig. These data reveal that male and female RAP250+/− mice are resistant to HFD-induced obesity. Upon fasting, male RAP250+/− mice showed normal levels of plasma glucose, adiponectin and free fatty acids, and reduced concentrations of plasma insulin, triglycerides and cholesterol on a normal diet (ND) (Fig. Administration of a HFD for 20 weeks caused high plasma concentrations of glucose and insulin in control male mice, but concentrations remained within the normal range in RAP250+/− counterparts (Fig. Studies performed in 30-week-old male RAP250+/− mice on a standard diet showed improved glucose tolerance and enhanced insulin secretion compared to control mice (Fig. Glucose tolerance was also improved in male RAP250+/− mice fed with HFD (Fig. 2G), and under these conditions, plasma insulin was markedly attenuated (Fig. C Fasting plasma glucose (n = 12) and D insulin levels (WT n = 10 and RAP250+/− n = 12) of 30-week-old male mice under a high-fat diet (HFD). I Whole-body glucose turnover (Turnover), hepatic glucose production (HGP), whole-body glucose infusion rate (GIR), whole-body glycolysis rate (Glycolysis), and whole-body glycogen synthesis rate (Glycogen synthesis) were measured in a 3-h hyperinsulinemic-euglycemic clamp (1.5 mU/kg/min insulin) in 30-week-old male mice (n = 9 for WT and n = 8 for RAP250+/−). J Hematoxylin/Eosin-stained paraffin slices of liver after 20 weeks of a HFD (scale bar=200 µm), and (K) Liver fat content as % of low density area (Lipid Droplets) over total histology slice area (n = 5, 25 slices per group). Statistical differences according to a two-sided Student´s t test (B, C, D, F, I and K) and Ordinary two-way ANOVA (E, G, and H). Female RAP250+/− mice showed no changes in plasma glucose or insulin under a standard diet or HFD (Fig. Thirty-week-old female RAP250+/− mice subjected to either a standard diet or HFD also showed enhanced glucose tolerance in the absence of changes in plasma insulin levels (Fig. The impact of RAP250 deficiency on whole-body glucose utilization was also monitored in conscious male mice on a normal diet during a euglycemic-hyperinsulinemic clamp (Fig. Under hyperinsulinemia, RAP250+/− mice showed an increase in whole-body glucose infusion rate (GIR), glucose turnover (Turnover), whole-body glycolysis, and glycogen synthesis rate (Fig. Thus, these animals were indeed more insulin-sensitive than control mice. Moreover, hepatic steatosis induced by the HFD was prevented in RAP250+/− male mice, as reflected by large differences in hepatic fat deposition (Fig. In all, this pattern of changes suggests that RAP250+/− mice of both sexes show improved glucose tolerance as a consequence of the improved insulin sensitivity, which occurs even in the absence of changes in adiposity. We next questioned whether the reduced adiposity detected in RAP250+/− mice derives from a higher rate of energy expenditure, thereby explaining a lower fat content and a parallel increased food intake. Indeed, RAP250+/− mice showed improved whole-body O2 consumption and CO2 production, as shown by ANCOVA analysis (Fig. Energy expenditure, glucose oxidation, and the respiratory exchange ratio (RER) were also increased (Fig. Exposure of control and RAP250+/− mice to a HFD for 20 weeks maintained most of the alterations detected under a standard diet, such as increased O2 consumption, CO2 production, energy expenditure, and lipid oxidation (Fig. S4E–H), and again, no changes in locomotor activity were detected (Fig. A Whole-body oxygen consumption in male mice. Adjusted means based on a normalized mouse weight of 25.8071 g determined using ANCOVA (n = 8 for WT; n = 7 for RAP250+/). B Correlation between energy expenditure (kcal/h) and body weight (grams). Data were evaluated in dark phase in 30-week-old male mice on a normal diet (n = 8 for WT; n = 7 for RAP250+/). C Glucose oxidation profile over 24 h in 30-week-old male mice on a normal diet. Data were evaluated in 30-week-old male mice on a normal diet (n = 8 for WT black symbols; n = 8 for RAP250+/− red symbols). D Brown adipose tissue weight. Data are expressed as percentage of tissue weight with respect total body weight (n = 5 for WT and n = 7 for RAP250+/−). E Glucose oxidation of isolated brown adipose cells (n = 9 from 3 independent experiments performed in triplicate). F Brown adipose tissue paraffin slices stained with Hematoxylin/Eosin (scale bar=100 µm) and lipid droplet size quantification (G). Data were evaluated in 20-week-old male mice (n = 5 for WT and n = 7 for RAP250+/−). H BAT sections stained with DAPI (blue), wheat germ agglutinin (WGA, green), and UCP1 (red). I Quantification of adipocyte area and J quantification of UCP1 mean signal intensity (n = 3 for WT; n = 4 for RAP250+/−). K Heatmaps showing gene expression modulation in transcriptomic analysis performed in BAT from 20-week-old mice (n = 3 for WT and n = 4 for RAP250+/−). L Gene expression of brown adipogenic markers Pparg2, Prdm16, Cebpa, Ppara, Dio2, Ppargc1 and Ucp1 in BAT from 20-week-old mice. Data were normalized to ARP mRNA expression (n = 5 for WT, n = 7 for RAP250+/−). Representative images for UCP1, PPARγ, C/EBPα and α-Tubulin proteins in BAT extracts (M) from 20-week-old mice and quantification of protein expression (N). Statistical differences according to a two-sided Student´s t test (A, D, E, G, I, J, L and N) and Ordinary two-way ANOVA (C). Given these observations, we explored the involvement of BAT in this phenotype. Interscapular BAT mass was reduced in RAP250+/− mice (Fig. 3D), and glucose oxidation was markedly enhanced in isolated brown adipocytes obtained from RAP250+/− mice (Fig. In addition, histological examination revealed less accumulation of lipids and a smaller lipid droplet size in RAP250-deficient brown adipocytes (Fig. Additional histological analyses using wheat germ agglutinin (WGA, to label the cell surface), DAPI (to label the nuclei), and anti-UCP1 antibody (to measure the differentiation state of adipocytes) were performed in BAT sections. Triple staining showed an increased abundance of UCP1 in brown adipocytes from RAP250+/− mice, consistent with an enhanced metabolic activity (Fig. In addition, data showed a reduced adipose cell size in BAT from RAP250+/− mice (Fig. Electron microscopy also revealed the presence of larger mitochondria in brown adipose cells in these mice (Fig. To further study the alterations in BAT caused by RAP250 deficiency, we performed RNA-Seq analysis. More than 500 genes were significantly upregulated and more than 330 were downregulated in BAT from RAP250+/− mice (Fig. In keeping with the RNA-seq analysis, the expression of adipogenic and thermogenic genes such as Pparg2, Prdm16, Cebpa, Ppara, Dio2, Ppargc1a, and Ucp1 was markedly increased in RAP250-deficient BAT, as determined by real-time PCR assays (Fig. In addition, the abundance of proteins PPARγ, C/EBPα and UCP1 was increased in brown adipose tissue from RAP250+/− mice compared to controls (Fig. In all, these data highly support the view that BAT is metabolically more active in RAP250-deficient mice compared to control, and that it may contribute to enhanced energy expenditure in response to RAP250 deficiency. Our data indicate that RAP250 is a negative regulator of BAT metabolism in mice. In order to further analyze this, we studied the impact of cold exposure on the BAT expression of RAP250. Exposure for 8 h to cold (4 °C) induced the expression of genes Ucp1, Ppargca1, and Pde3b, and this was accompanied by a marked repression of RAP250 gene expression (Fig. In parallel to these observations, exposure of brown adipocytes for 2 h in the presence of IBMX, a phosphodiesterase inhibitor induced Rgs2 and reduced RAP250 (Fig. This suggests that RAP250 participates in the activation of thermogenesis of brown adipocytes during cold and that the response is mediated by cAMP. In addition, RAP250 heterozygous mice showed a greater induction of Ucp1 mRNA levels in BAT in response to cold exposure (Fig. Under these conditions a greater induction of glucose oxidation was also detected in RAP250+/− mice during cold (Fig. Next, we addressed whether local alterations in BAT were sufficient to induce some of the metabolic alterations detected in RAP250+/− mice. To repress RAP250 in interscapular brown adipose, we injected adeno-associated viruses (AAVs) encoding shRNAs directed against RAP250, under H1 promoter, into the BAT of 2 month-old C57Bl/6 J mice (control mice were injected with AAVs encoding a scramble sequence). The injection of shRAP250 AAVs into this tissue reduced RAP250 expression (Fig. 4A and B), and this led to a significant decrease in body weight (Fig. 4C) and fat mass up to 10 weeks after the injection (Fig. In addition, shRAP250-injected mice showed enhanced glucose oxidation (Fig. 4E) and an improved respiratory exchange ratio (RER) (Fig. The expression of UCP1 protein was also enhanced in RAP250-deficient BAT (Fig. Under these conditions, plasma leptin levels remained unaltered (Fig. Representative image for RAP250 protein expression (A) and quantification (B) in BAT extracts from 32-week-old male mice 24 weeks after injection with Scr or ShRAP250 AAVs into BAT. Data were normalized to Vinculin (n = 5 for Scr and ShRAP250). Body weight (C) and total fat mass (D) curves after AAV administration into BAT (Scr, n = 5, black symbols; ShRAP250 n = 4, green symbols). Glucose oxidation (E) and RER (ratio CO2/O2) (F) over 24 h in 28-week-old male mice injected with Scr or ShRAP250 AAVs into BAT and on a normal diet. Gray phase corresponds to dark phase (n = 4 for WT; n = 4 for RAP250+/−). G UCP1 expression quantification in BAT extracts. Data were normalized to Vinculin (n = 5 for Scr and ShRAP250). Body weight curves from 8- to 22-week-old male mice subjected to thermoneutrality and on a normal chow diet (TN-ND) (I) or a high-fat diet (TN-HFD) (J). K Total fat mass from 22-week-old mice under TN-ND and from 20-week-old mice under TN-HFD. L Brown adipose tissue weight. Paraffin sections of PAT stained with Hematoxylin/Eosin for TN-ND (M) and TN-HFD (N) (scale bar=100 µm). O PAT adipocyte cell size of TN-ND (22-week-old) and TN-HFD (20-week-old) (WT TN-ND n = 5, RAP250 TN-ND n = 6, WT TN-HFD n = 3 and RAP250 TN-HFD n = 4, from 8−10 images per mouse). Statistical differences according to a two-sided Student´s t test (B and G) and Ordinary two-way ANOVA (C, D, E, F and J) and two-way ANOVA followed by Sidak's multiple comparison test (K, L and O). In parallel studies, the perigonadal (epididymal) adipose tissue depots of 2 month-old C57Bl/6J male mice were injected with shRAP250- or scramble RNA-encoding AAVs. Data indicate that shRAP250 AAV administration reduced RAP250 expression in adipose tissue by 50% (Fig. S6A and B) 24 weeks after AAV injection. Under these conditions, body weight and total fat mass or perigonadal adipose depots were not altered in the RAP250-deficient group (Fig. RAP250 deficiency in WAT depots did not alter RER or glucose oxidation (Fig. Plasma leptin levels were not altered upon PAT gene manipulation (Fig. Based on our data, we propose that selective repression of RAP250 in BAT is the main driver that leads to alterations in energy expenditure that resemble those detected in RAP250+/− mice. Next, we examined whether the metabolic profile of RAP250+/− mice was modified upon exposure to thermoneutral conditions, i.e., upon inactivation of BAT metabolic activity. Thus, control and RAP250+/− mice were housed at 30 °C for 14 weeks (starting at 8 weeks of age). Half the mice were fed a standard chow diet (TN-ND), while the other half received a HFD (TN-HFD), which enhances thermogenic activity due to the high amount of fat34,35. RAP250+/− mice subjected to thermoneutrality and a standard diet did not show alterations in body weight (Fig. 4I) or total fat mass (Fig. In contrast, RAP250+/− mice subjected to a HFD had a lower body weight (Fig. 4J) and a marked reduction of fat mass (Fig. Under these conditions, the weight of BAT was only reduced in RAP250+/− mice upon a HFD (Fig. Tissue sections revealed unaltered adipose cell size in mice on a standard diet (Fig. 4M and 4O) but a reduced adipose cell size in the RAP250+/− group upon treatment with a HFD (Fig. These studies provide further evidence that BAT plays a major role in the alterations in energy metabolism of RAP250+/− mice. To explore the mechanisms responsible for the enhanced metabolic activity of BAT under RAP250 deficiency, we focused on the differentially expressed gene sets detected in RNA-seq assays (Fig. Gene set enrichment analysis revealed a significant upregulation of genes involved in the action potential of neurons in RAP250-deficient BAT (Figs. In keeping with these data, RAP250-deficient BAT showed an enhanced expression of tyrosine hydroxylase (TH), suggesting an increased sympathetic innervation17 (Fig. In keeping with this, cell type deconvolution analysis of bulk transcriptomics detected an increased abundance of non-myelinated Schwann cells in RAP250-deficient BAT (Fig. This is relevant since sympathetic nerve fibers innervating BAT are predominantly non-myelinated Schwann cells, and they play a key role in neurotransmission, allowing rapid norepinephrine release to stimulate thermogenesis36,37. This type of analysis also revealed an increased abundance of myelinated Schwann cells in RAP250-deficient BAT (Fig. A Gene set enrichment analysis results of Regulation of action potential in neuron pathway (GO0019228) from Gene Ontology (GO) upregulated in BAT from RAP250+/− mice. B Heatmap of expression of specific neurotrophic and/or batokine genes in BAT from WT and RAP250 BAT (n = 3). Representative image for Tyrosine Hydroxylase (TH) expression (C) and quantification (D) in BAT extracts from 20 -week-old WT and RAP250-deficient mice (WT n = 7, RAP250+/− n = 6). E Percentage of non-myelinated Schwann cells in 20 -week-old WT and RAP250-deficient mice (n = 3). Representative image for NRN1 expression (F) and quantification (G) in BAT extracts from 20-week-old WT and RAP250-deficient mice (WT n = 4, RAP250+/− n = 6). M Representative image of NRN1 expression during brown adipogenesis at different days of differentiation (0, 2, 4, 6 and 10). N NRN1 expression in culture media from differentiated brown adipose cells under 2 hours of treatment without and with Isoproterenol (10uM). Data are expressed as total NRN1 in media (ng) corrected by total cell protein (mg) (Representative experiment performed in triplicate n = 3). Statistical differences according to a two-sided Student´s t test (D, E, G, H, I, J, K and N), and Enrichment score rotation based test (roastgsa) (A). Analysis of western blot and real-time PCR validated the increased expression of Nrn1 in RAP250-deficient BAT (Fig. The increased expression of Nrn1 mRNA was detected in BAT from RAP250+/− mice aged between 2 and 6 months (Fig. Nrn1 mRNA was more enriched in brown adipocytes compared to the expression detected in the stromal vascular fraction obtained from BAT, similarly to was detected for UCP1 or RAP250 (Fig. AAV-mediated RAP250 repression in BAT also caused an increased expression of Nrn1 mRNA and protein (Fig. In contrast, AAV-mediated RAP250 repression in perigonadal adipose tissue did not alter the expression of Nrn1 (Fig. Exposure of mice to thermoneutrality abrogated the stimulation of Nrn1 mRNA levels in BAT (Fig. In separate studies, we analyzed the expression of Nrn1 in mouse brown adipose cells during adipogenesis. Nrn1 mRNA was substantially expressed in preadipocytes and it underwent a two-fold increase in adipocytes under conditions in which UCP1 was markedly induced (Fig. In contrast, the expression of Nrn1 protein was not detectable in preadipocytes and was markedly induced during brown adipogenesis (Fig. Nrn1 was detected as 15 kDa and 12 kDa bands and it is likely that the former is an unprocessed form of the protein. Nrn1 protein was also detected in culture media from brown adipocytes, suggesting that it was secreted. Moreover, the secretion was rapidly stimulated in response to isoproterenol treatment (Fig. In all, our data indicate that Nrn1 is expressed and secreted by brown adipose cells and that its expression is markedly enhanced in response to brown adipogenesis. We next tested whether Nrn1 expression in thermogenic adipose cells had metabolic effects in mice. To this end, control mice subjected to a normal diet were treated with AAVs encoding Nrn1 under the control of the UCP1 promoter or an empty vector. Thirty weeks after the AAV administration, AAV-Nrn1-treated mice showed a 3-fold greater expression of Nrn1 mRNA and protein in BAT whereas no changes in expression were detected in ING adipose tissue (Fig. Plasma levels of Nrn1 were detectable but no differences were found between control and Nrn1-treated groups (Fig. Under these conditions, AAV-Nrn1-treated mice showed a lower body weight and lower fat mass (Fig. Energy expenditure, oxygen consumption, CO2 production and glucose oxidation were enhanced in AAV-Nrn1-treated mice (Fig. Under these conditions, food intake but not ambulation was enhanced (Fig. AAV-Nrn1-treated mice showed lower concentrations of plasma triglycerides, normal glycemia, and a trend for reduced leptin levels (Fig. A Nrn1 gene expression in BAT and SAT from mice injected via the tail-vein with either AAV-UCP1 empty (Control) or AAV-UCP1-NRN1 vector under UCP1 promoter (BAT AAV-Control n = 9; BAT AAV-UCP1-NRN1 n = 7; SAT AAV-Control n = 8, SAT AAV-UCP1-NRN1 n = 6). Body weight (time 0 Control n = 7 and NRN1 n = 10; time 30 Control n = 9 and NRN1 = 9) (B), total fat mass (n = 8) (C) and lean mass (n = 8) (D) graphs for mice before administration of an AAV Control or AAV-UCP1-NRN1 at time 0 (8-week-old mice) and 30 weeks after administration. Energy expenditure (E) and oxygen consumption (F) graphs during day and night periods in mice injected with AAV-Control or AAV-UCP1-NRN1 protein 26 weeks after infection (AAV-Control n = 7; AAV-UCP1-NRN1 n = 9). G Food intake as grams of daily intake in mice infected with AAV-control and AAV-UCP1-NRN1 (AAV-Control n = 7; AAV-UCP1-NRN1 n = 9). Brown adipose tissue paraffin slices stained with Hematoxylin/Eosin (H) (scale bar=100 µm) and lipid droplet size quantification (I). Data were evaluated 30 weeks after infection for Control and overexpressing NRN1 under UCP1 promoter mice (AAV-Control n = 8; AAV-UCP1-NRN1 n = 7). J BAT sections stained with DAPI (blue), wheat germ agglutinin (WGA, green), and UCP1 (red). K Quantification of adipocyte area and (L) of UCP1 mean signal intensity (n = 4 for AAV-Control; n = 5 for AAV-UCP1-NRN1). M Gene expression for Prdm16, Ppargc1a, Ppargc1b, Cidea, Cpt1a, Lipe and Dio2 in BAT 30 weeks after AAV infection with Control or NRN1 under the UCP1 promoter. Data were normalized to ARP mRNA (n = 9 for AAV-Control; n = 7 for AAV-UCP1-NRN1). Representative images for p-HSL, HSL, UCP1, proteins phosphorylated by PKA substrates (p-PKA) and GAPDH proteins in BAT extracts 30 weeks after AAV infection with Control or NRN1 under the UCP1 promoter (N) and quantification (O). Data were normalized to GAPDH expression (n = 6 for AAV-Control; n = 7 for AAV-UCP1-NRN1). Statistical differences according to a two-sided Student´s t test for all figures except one-sided Student´s t test (K and L). Sections of BAT revealed reduced size of brown adipose cells and lipid droplets upon Nrn1 treatment (Fig. Triple staining of BAT section showed a reduced adipose cell size in BAT from Nrn1-treated mice (Fig. In addition, an increased abundance of UCP1 in brown adipocytes from Nrn1 mice was also detected (Fig. In all, these data suggest an enhanced BAT metabolism and thermogenesis upon Nrn1 expression. In keeping with this view, we detected an increased expression of thermogenic and metabolic genes in the AAV-Nrn1-treated group (Fig. 6M), and also an enhanced expression of phosphorylated HSL, active PKA, UCP1 (Fig. We also detected that chronic overexpression of Nrn1 in brown adipocytes enhanced the expression of OXPHOS subunits relevant in respiratory complexes I and II but not for CV (Fig. The Nrn1 group also showed a reduction in adipose cell size in visceral and subcutaneous depots (Fig. Mice subjected to a HFD were also treated with AAVs encoding Nrn1 under the control of the UCP1 promoter or an empty vector. Nrn1-treated mice showed increased Nrn1 gene expression in BAT and ING adipose tissue (Fig. 7A), and no differences in plasma levels were detected between groups (Fig. AAV-Nrn1-treated mice showed a lower rate of body weight and fat mass gain compared to control mice (Fig. Energy expenditure, oxygen consumption, and lipid oxidation were enhanced in AAV-Nrn1-treated mice (Fig. Under these conditions, food intake was not altered (Fig. S9D), and nocturnal ambulation was enhanced in these mice (Fig. These mice showed lower concentrations of plasma triglycerides and leptin, and normal glycemia (Fig. We also detected an enhanced expression of thermogenic and metabolic genes in the Nrn1 group (Fig. A Nrn1 mRNA expression in BAT and SAT in mice injected via the tail-vein with either AAV Control or NRN1 vector under UCP1 promoter and subjected to a HFD for 15 weeks (AAV-Control n = 8, black symbols; AAV-UCP1-NRN1 n = 8, violet symbols). Body weight (B) and total fat mass (C) graphs for mice before administration of an AAV Control or NRN1 under the UCP1 promoter (AAV-UCP1-NRN1) at time 0 (8-week-old mice) and 3, 5, and 10 weeks after administration under HFD conditions (AAV-Control n = 8; AAV-UCP1-NRN1 n = 8). D Energy expenditure over 24 h in male mice infected with AAV empty vector or NRN1 under UCP1 on a HFD 13 weeks after administration. E Oxygen consumption graph during day and night period in mice injected with AAV Control or AAV-UCP1-NRN1 (AAV-Control n = 8; AAV-UCP1-NRN1 n = 7). F Gene expression of Ucp1, Adrb3, Lipe and Cox8b in BAT 15 weeks after AAV infection with Control or NRN1 under the UCP1 promoter. G and H Heatmaps showing z-scores of gene expression levels for Oxidative Phosphorylation (G) and Adipogenesis (H) Hallmarks in BAT from mice subjected to a normal diet infected with AAV empty vector or NRN1 under the control of the UCP1 promoter (n = 4 for Control and n = 5 for AAV-UCP1-NRN1). Column p shows fold changes for individual genes. I Normalized scores of gene set enrichment analysis of top GO terms (Cellular Compartment) upregulated in BAT from mice AAV-NRN1 treated mice (adj. In purple color, mitochondrial related pathways are shown. J Bubble chart of gene set enrichment analysis results showing pathways modified in response to NRN1 overexpression and upon cold exposure. Stars indicate p-value lower than 0.05. K Percentage of non-myelinated Schwann cells obtained by deconvolution analysis comparing control versus AAV-NRN1 treated mice in ND (AAV-Control n = 3; AAV-UCP1-NRN1 n = 5). Tyrosine Hydroxylase expression (L) and quantification (M) in BAT extracts from control and AAV-NRN1 treated mice in ND (AAV-Control n = 5; AAV-UCP1-NRN1 n = 7). Statistical differences according to a two-sided Student´s t test (A–F, K and M). In figure J the bubble size indicates statistical significance (bigger = lower adjusted P value). Inner asterisk indicates adjusted P value < 0.05. As we found that enhanced Nrn1 expression activates brown adipose tissue, we next analyzed the mechanisms involved. To this end, we performed RNA-Seq assays using BAT samples from AAV-Nrn1-treated and control mice. Gene set enrichment analysis revealed enhanced abundance of Hallmark Oxidative Phosphorylation, and Adipogenesis pathways in response to chronic NRN1 (Fig. Analysis of GO Cellular Compartment GSEA revealed that most of the genes upregulated were mitochondrial (encoding for mitochondrial proteins) (Fig. The pattern of changes detected upon NRN1 expression was similar to the changes reported in response to cold exposure to 4 °C for 1 week38. Our data indicate that BAT oxidative metabolism is activated upon NRN1 overexpression, and that the effects of NRN1 show striking similarity to the effects caused by chronic cold exposure (Fig. These data indicate the existence of common processes operating in Nrn1 overexpressing BAT and upon cold exposure. We performed cell type deconvolution analysis of bulk transcriptomics from control and Nrn1 overexpressing BAT tissues, and we detected and increased abundance of non-myelinated Schwann cells (Fig. 7K), which was parallel to an increased expression of tyrosine hydroxylase (Fig. Moreover, a reduced abundance of myelinated Schwann cells was also detected (Fig. In all, these data indicate a sympathetic remodeling in BAT by Nrn1 involved an increased proliferation of non-myelinated Schwann cells. To further analyze whether Nrn1 has metabolic effects in brown adipocytes, we repressed Nrn1 in primary brown preadipocytes using lentiviruses encoding shRNA (Fig. These cells showed a normal differentiation and the expression of protein markers such as PPARγ2, PPARγ1, UCP1 and HSL was indistinguishable from that observed in control adipocytes (Fig. Given these observations, we monitored the acute stimulatory response of brown adipocytes to the β-adrenergic receptor agonist isoproterenol39. Isoproterenol activated the phosphorylation of HSL and other PKA substrates in control brown adipocytes (Fig. This activation was markedly reduced in Nrn1-deficient brown adipocytes (Fig. Isoproterenol also induced crucial genes, such as Ppargc1a and Ucp1, in control cells (Fig. This process was largely repressed in Nrn1-deficient cells (Fig. We also analyzed the effects of Nrn1 deficiency on mitochondrial respiration in brown adipocytes. In control brown adipocytes, mitochondrial respiration was reduced upon addition of oligomycin, and isoproterenol increased mitochondrial respiration to values close to those measured in the presence of FCCP (maximal respiration) (Fig. Depletion of Nrn1 reduced the effects of isoproterenol on mitochondrial respiration, and also maximal mitochondrial respiration (Fig. Under these conditions basal respiration or ATP-coupled respiration were unaltered in Nrn1-deficient cells (Fig. The reduced response to isoproterenol was not a consequence of changes in the expression of β3-adrenergic receptors (Fig. In all, these results indicate that Nrn1 protein is synthesized by brown adipocytes and that it can modulate the metabolism of brown adipocytes and their capacity to respond to catecholamines. A Relative Nrn1 mRNA expression in Scr and siNRN1 preadipocytes. Data are from 3 separate differentiation experiments. p-HSL and phosphorylated by PKA substrates (p-PKA) expression (C) and quantification (D and E) in Scr and siNRN1 adipocytes (day 10 of differentiation), in basal and treated with Isoproterenol (10 μM) for 2 h (p-HSL n = 6 from 2 experiments performed in triplicate (p-HSL) and n = 3 from 1 representative experiment performed in triplicate (p-PKA)). Ppargc1a mRNA (F) and Ucp1 mRNA (G) expression in Scr and siNRN1 adipocytes cells, in basal and treated with isoproterenol (10 μM) for 2 h. Data are mean of 2 experiments performed in triplicate. H Seahorse representative experiments for Scr and siNRN1 brown adipocytes at day 10 of differentiation (Scr n = 4 and siNRN1 n = 5). I Mitochondrial respiration values referred to basal WT. Data are the mean of 5 independent experiments. J, K Expression (J) and quantification (K) of β3 Adrenergic Receptors in Scr and siNRN1 brown adipocytes at day 10 of differentiation. Data are mean of 4 independent experiments. L NRN1 SEC fractions run on 10% Bis-Tris NuPAGE Midi SDS-PAGE gels with MES running buffer. BenchMark protein markers (Invitrogen) were included for reference. M Abundance of PKA phosphorylated substrates (p-PKA) in brown adipocytes treated with 0.1, 0.5 and 1 μg/ml of recombinant NRN1 protein. P Mitochondrial respiration values referred to basal WT respiration in untreated brown adipocytes non-treated (white bars) and cells treated with NRN1 at a concentration of 100 ng/ml (orange bars) from day 4 of differentiation (Representative experiment, Basal n = 7 and NRN1 treated adipocytes n = 8). Q, R Expression (Q) and quantification (R) of CI-NDUFB8 subunit in Scr adipocytes cells treated chronically with NRN1 (100 ng/ml) from day 4 of differentiation until day 10. Data are the mean of 3 independent experiments. Statistical differences according to a two-sided Student´s t test (A, I, K, N, O, P and Q) and two-way ANOVA followed by Sidak's multiple comparison test (D–F and G). To analyze whether extracellularly present Nrn1 has metabolic effects in brown adipocytes, we have expressed and purified mouse recombinant NRN1 to homogeneity (Fig. Next, we incubated brown adipocytes with recombinant NRN1 (100 ng/ml). Short-term incubation with recombinant NRN1 enhanced the phosphorylation of protein kinase A (PKA) substrates in brown adipocytes (Fig. In addition, NRN1 caused a marked induction of Ucp1 and Ppargc1a gene expression in brown adipocytes (Fig. NRN1 also increased the effects of isoproterenol on mitochondrial respiration (Fig. These results clearly reveal a thermogenic effect of extracellular NRN1 in brown adipocytes, and further suggest that NRN1 has the capacity to act as an autocrine/paracrine factor in brown adipose tissue. The results reported here support the following highlights: a) RAP250 protein is a key component of energy homeostasis and it modulates BAT metabolism; b) RAP250 deficiency in brown adipose cells induces the expression of Nrn1, a factor until now unknown as a regulator of BAT biology; c) AAV-mediated overexpression of Nrn1 in brown adipose cells improves energy expenditure and stimulates BAT metabolism; d) Nrn1 repression disrupts the metabolism of brown adipocytes, an observation that indicates an autonomous cellular response; and e) exogenous mouse recombinant NRN1 enhances the activity of protein kinase A, stimulates mitochondrial respiration and induces the expression of thermogenic genes in brown adipocytes, supporting the view that NRN1 acts as a secreted factor in brown adipose tissue. Our findings reveal that RAP250 plays a key role in maintaining energy metabolism and adiposity in mice. We demonstrate that RAP250 deficiency upregulates energy metabolism, reduces adiposity, and enhances insulin sensitivity in mice. In addition, we found that global RAP250 deficiency in these animals stimulates BAT metabolism and that local AAV-mediated RAP250 repression in BAT reduces adiposity and enhances global glucose oxidation and BAT UCP1 expression. In addition, we document that RAP250 expression is markedly attenuated in BAT in response to cold, and is also reduced in brown adipocytes by phosphodiesterase inhibitors suggesting a regulatory role of cAMP. Based on all these data, we propose that RAP250 regulates energy metabolism, most likely through the modulation of the metabolic activity of BAT. In keeping with these data, many common variants of RAP250 show a very strong association with hip circumference-adjusted BMI (https://t2d.hugeamp.org/gene.html?gene=NCOA6)40 (Fig. This observation points to the participation of RAP250 in human metabolism. Our data indicate that RAP250 deficiency causes leanness in mice as a result of increased BAT metabolism and energy expenditure. We are aware of different studies indicating that Thus, the transcriptional activity of PPARγ is impaired in RAP250 KO mouse embryo fibroblasts27,41,42, and these cells are refractory to PPARγ-stimulated adipogenesis27. Future studies are required to proof that RAP250 indeed controls in vivo white adipogenesis. RAP250 also regulates liver metabolism, and its inactivation disrupts hepatic lipid metabolism and gluconeogenesis, leading to enhanced insulin sensitivity23,24,43. The enhanced energy expenditure of RAP250+/− mice was linked to greater glucose oxidation under a normal diet and to increased lipid oxidation upon HFD conditions. Moreover, these animals showed an enhanced BAT metabolism that was characterized by large transcriptomic alterations, increased glucose oxidation, improved mitochondrial respiration, high expression of key genes Pparg2, Prdm16, Cebpa, Ppara, Ppargc1a, Ucp1 and Dio2, and increased expression of proteins UCP1, PPARγ and C/EBPα. Two additional experimental findings revealed that BAT plays a major primary role in the metabolic changes detected in RAP250-deficient mice. Thus, local repression of RAP250 protein mediated by the injection of AAV shRAP250 particles into the interscapular BAT depot of wild-type mice phenocopied many of the metabolic changes detected in RAP250-deficient mice, including reduced body weight, lower fat mass, and enhanced glucose oxidation. These effects were specific to RAP250 repression in BAT, because a similar genetic manipulation of RAP250 in PAT did not have metabolic consequences. A second observation came from the exposure of RAP250-deficient mice to thermoneutrality, which inactivates BAT thermogenesis. Under these conditions, marginal or no differences in body weight, fat mass or glucose tolerance were detectable between control and RAP250+/− mice. Moreover, the combination of thermoneutrality and a HFD, the latter enhancing the thermogenic activity of BAT35,44, maintained the differences in body weight, fat mass, and glucose tolerance normally appearing in RAP250-deficient mice. In all, our data strongly support the view that RAP250 modulates energy expenditure through the regulation of BAT activity in such a way that primary changes in BAT function lead to alterations in adiposity. Neuritin 1 (Nrn1), originally termed candidate plasticity-related gene 15 or Cpg15, was identified as a gene induced by the glutamate analogue kainate in the hippocampus dentate gyrus30. Nrn1 is a glycosylphosphatidylinositol (GPI)-anchored neuropeptide expressed in postmitotic neurons and it regulates synaptic plasticity, neurite growth, and neuronal migration and survival29,31,32,45,46,47. It is also involved in memory and learning48. Moreover, Nrn1 operates in cells other than neurons. In this regard, it is also produced by follicular regulatory T cells, and it acts on B cells to suppress autoantibody development against tissue-specific antigens49. In addition, this protein is expressed in tissues such as the placenta, lung and spleen50. Our data support the notion that Nrn1 plays a key role in BAT metabolism. In this study, we document that Nrn1 is upregulated in BAT under conditions of RAP250 deficiency. Given the observation that Nrn1 is upregulated in BAT upon local AAV-mediated RAP250 repression, we support the notion that RAP250 locally modulates Nrn1 gene expression in BAT. Moreover, we document that Nrn1 is abundantly expressed in brown adipocytes and that its expression is regulated during differentiation state at translational or post-translational levels. As RAP250 participates in COMPASS-related complexes containing the H3K4 methyltransferases MLL3 and MLL420,21,51, we propose that RAP250 repression drives epigenetic changes, thus causing an enhanced expression of Nrn1. The intricacy of those complexes is high, and MLL3 and MLL4 induce monomethylation of H3K4 at enhancer regions, whereas another enzyme in MLL3 and MLL4 COMPASS, namely UTX, demethylates H3K27me3, a modification associated with transcription repression by the Polycomb group of proteins52,53. Here, we also demonstrate that Nrn1 expressed in brown adipose cells causes the local activation of BAT metabolism and thermogenesis. Thus, the expression of Nrn1 in thermogenic adipose cells causes the activation of energy expenditure under normal conditions or upon an HFD. Under these conditions, we also detected a reduction in body weight gain. This occurred in parallel to an enhanced expression of BAT metabolic genes, including Prdm16, Ppargc1a, Cidea, Cpt1a, Hsl, and Dio2. RNA-Seq assays revealed the existence of a metabolic reprogramming in response to chronic NRN1 expression in BAT characterized by the induction of mitochondrial genes and also genes relevant in adipose cell differentiation. The pattern of changes detected upon NRN1 expression was similar to the changes reported in response to cold exposure to 4 °C for 1 week38. In all, these data indicate that BAT oxidative metabolism is activated upon NRN1 overexpression, and that the effects of NRN1 show striking similarity to the effects caused by chronic cold exposure. The effects of Nrn1 in BAT seem to be largely local. At a cellular level, Nrn1 seems to operate through an autocrine/paracrine mode of action. In this context, we detected that Nrn1 is secreted from brown adipocytes and that this secretion is significantly enhanced in the presence of the β-adrenergic receptor agonist isoproterenol. In addition, we found that Nrn1 repression in brown adipose cells does not alter differentiation, but it markedly reduces the response to isoproterenol. Nrn1 groups with local-acting factors, such as BMP8b, which are secreted by brown adipocytes and activate BAT metabolism16,54. Interestingly, some common variants of the Nrn1 gene show a very strong association with BMI (https://t2d.hugeamp.org/gene.html?gene=NRN1), which suggests that Nrn1 participates in human metabolism. Interestingly, incubation of brown adipocytes with mouse recombinant NRN1 enhances the activity of protein kinase A, stimulates mitochondrial respiration, and induces the expression of thermogenic genes in brown adipocytes, supporting the view that NRN1 acts as a secreted factor in brown adipose tissue. The specific receptors for NRN1 have not been definitively identified. To date, no canonical, high-affinity cell surface receptor has been unequivocally demonstrated55. Moreover, NRN1 is not part of a known ligand family like neurotrophins or FGFs, and it does not display any conserved domain homologous to these or other classical signaling molecules56. Several studies have proposed indirect mechanisms or potential receptor candidates for NRN1. For instance, cross-talk between NRN1 and insulin signaling has been observed, suggesting that neuritin may enhance or depend on insulin receptor pathways for its effects57. Notably, pharmacological blockade of the insulin receptor, but not the insulin-like growth factor 1 receptor, abrogated these effects, indicating a specific involvement of the insulin receptor in neuritin signaling. In addition, there is evidence that neuritin interacts with fibroblast growth factor receptor 1 (FGFR1) to mediate axonal branching in granule neurons. Immunoprecipitation assays have revealed a strong association between neuritin and FGFR1, and inhibition or knockdown of FGFR1 significantly suppressed neuritin-induced axonal branching, suggesting that FGFR1 plays a critical role in neuritin's effects on neuronal development58. Regarding downstream signaling, activation of the ERK/MAPK and PI3K/Akt pathways has been observed following neuritin 1 treatment, consistent with its role in promoting cell survival and growth57,59. It is also likely that Nrn1-receptor complex undergoes internalization, which may be relevant in further signaling49. In our study, we further demonstrate that neuritin 1 can rapidly activate protein kinase A signaling in brown adipocytes. Altogether, these findings suggest that neuritin 1 acts pleiotropically, modulating multiple pathways depending on the cellular context. However, the precise molecular mechanisms underlying its actions remain to be fully elucidated and warrant dedicated investigation. On the basis of our findings, we conclude that Nrn1 is a key regulator of brown adipocytes, it performs trophic actions, and it preserves the capacity to respond to catecholamines, a crucial physiological characteristic of brown adipocytes. We do not exclude the possibility that Nrn1 exerts a neurotropic action in BAT. In fact, Nrn1 overexpression caused an increased tyrosine hydroxylase expression and enhanced abundance of non-myelinated Schwann cells in BAT, supporting a role on sympathetic innervation in this tissue. Given the properties of Nrn1, it emerges as a potentially useful protein for targeting energy expenditure in humans. RAP250 heterozygous mice, with a null mutation in the RAP250 gene, were used in this study42. Briefly, exons 7 and 8 were replaced with the neo gene which is flanked by a 5-kb homologous region on each side42. All animal work was conducted following the guidelines established by the Animal Care Committees of the “Parc Científic de Barcelona” and the University of Barcelona. Heterozygous female mice were crossed either with C57BL/6J male or heterozygous male mice to obtain control and heterozygous littermates. Mice were kept under a 12 h dark/light cycle and provided a standard chow diet and water ad libitum. For the high-fat diet (HFD) experiments, 10-week-old male mice were fed a standard diet or HFD for 15–20 weeks (10% or 60% calories from fat, D12492 and D12450B; Research Diets, New Brunswick, NJ, USA). Animals were fasted for 16 h (17:00–9:00) in glucose tolerance and glucose turnover tests. In thermoneutrality experiments, 8-week-old mice were kept for 22 weeks at 30 °C either under a standard chow diet or HFD. Mice were kindly gifted by the Mouse Mutant Core Facility at IRB Barcelona. At the times indicated in figure legends, the mice were anesthetized with isoflurane and euthanized by cervical dislocation. Interscapular BAT (BAT), Perigonadal Adipose Tissue (PAT), Inguinal adipose tissue (ING), liver, gastrocnemius and heart were collected and used for RNA purification, protein extraction, or histology. Plasma was obtained by cardiac punction. All animal experiments were done in compliance with the guidelines established by the Institutional Animal Care and Use Committee of the Barcelona Science Park and University of Barcelona (Protocol number 9279, approved by the Department of Territory and Sustainability, General Directorate of Environmental Policies and the Natural Environment, Government of Catalonia). Single-stranded AAV9-Scr-RSV-GFP and AAV9-Sh RAP250-RSV-GFP vectors were generated and produced by the “Unitat de Producció de Vectors” (www.viralvector.eu) at the UAB. The sequence used to repress RAP250 was 5'-ACACATACCTCAGAACATA-3'. AAV-UCP1 empty and AAV-UCP1-Nrn1 vectors were generated by Vector Biosystems and produced by “Unitat de Producció de Vectors” at the UAB. The AAV9-Scr-RSV-GFP and AAV9-Sh RAP250-RSV-GFP vectors were administered as reported60. For BAT infection, 1 × 1010 of the total viral genome (vg) of AAV9 Scr or Sh was administered (4 × 10 μl of viral solution) per BAT depot to 8-week-old male mice. In Nrn1 protein overexpression experiments, 1.5 × 1012 of the total viral genome (vg) was intravascularly administered (by tail) to 8-week-old C57BL6J male mice. Mice were placed individually in metabolic cages and acclimatised for 24 h before measuring food intake over four consecutive days. Feeding was accessed by evaluating the food remaining in a 24-h period. Blood samples were collected between 10:00 and 12:00 either after an overnight fast or under fed conditions; they were obtained by tail vein snip of unrestrained mice. Plasma was immediately separated by centrifugation at 4 °C and stored at −80 °C until assay. Plasma insulin and leptin levels were determined using the UltraSensitive Mouse Insulin Elisa Kit and Mouse Leptin Elisa Kit (Crystal Chem Inc., Downers Grove IL, USA); adiponectin with the Quantikine Adiponectin kit (R&D Systems, Minneapolis, USA); and cholesterol and triglycerides with BioSystems Kits (Costa Brava, Spain). Blood glucose was assayed with an Accu-Chek glucose monitor and reactive strips (Roche Diagnostics Corp.) directly in blood drops from conscious mice. were performed as described in ref. Blood was collected, and glucose and insulin concentrations were measured at different time points. Whole-body glucose utilization rate was determined under euglycemic-hyperinsulinemic conditions. Under physiological hyperinsulinemic conditions, insulin was infused at a rate of 4 mU/kg·min for 3 h, and D-[3H]3-glucose at a rate of 30 μCi/kg·min. Throughout the infusion, blood glucose was determined from blood samples (3.5 μL) collected from the tip of the tail vein with a blood glucose meter when needed. Euglycemia was maintained by periodically adjusting a variable infusion of 10% or 16.5% glucose. Plasma glucose concentrations and D-[3H]3-glucose-specific activity were determined in 5 μL of blood sampled from the tip of the tail vein every 10 min during the last hour of the infusion, as reported62. Mice were anesthetized with Ketamine/Xylazine by i.p. injection and positioned face down, with both legs fully extended, in a SkyScan 1076 high-resolution micro-CT scanner (SkyScan, Kontich, Belgium) mouse bed. 360° scans of the abdominal region were made at a rotation step of 0.8° and an isotropic voxel size of 35 μm; Source 50 kV/200 µA and 1.0 mm aluminium filter. Scans were reconstructed and analyzed using the NRecon and CTAn software provided by SkysScan. The region of interest (ROI) for each animal was defined based on skeletal landmarks from the gray-scale reconstructed images (between femoral head and L5 vertebra). Data are expressed as percentage of fat on a ROI: Fat Volume/ Total ROI Volume X 10063. Fat mass was also measured using magnetic resonance with EchoMRITM Body Composition Analyzer. An indirect calorimetry system (Oxymax, Columbus Instruments) was used to measure oxygen consumption (VO2), CO2 production (VCO2), and ambulation. The experiment included two days of acclimation and a third day (24 h) of valid data recording. Average RER index (VCO2 ml/h produced/VO2 ml/h consumed); Energy Expenditure (Kcal/h/Kg) [defined as VO2x(3.815 + 1.232xRER)/1000]; Glucose Oxidation (g/min/Kg0.75) [defined as [(4.545xVCO2)-(3.205xVO2)] /1000]; and Lipid Oxidation (g/min/Kg0.75) [defined as [1.672x(VO2-VCO2)]/1000] were calculated from VO2 and VCO2 data. To remove variance in metabolic parameters derived from differences in body weight between animals, analysis of covariance (ANCOVA) was performed. ANCOVA is a robust method to compare groups with divergent body weights and composition64,65. For data obtained in mice on a normal diet, values of O2 consumption and CO2 production were referred to an adjusted mean body weight of 25.8071 g determined with ANCOVA, using body weight as a covariate in WT and RAP250 male mice in normal diet. In acute cold exposure studies, a Sable Systems Promethion Core was used. In these studies, mice were acclimated to the cages during 4 days at 22 °C and subjected to 4 °C during 8 hours. Mouse body composition was measured using Quantitative Magnetic Resonance with the EchoMRITM Body Composition Analyzer. 30-week-old male mice were killed by cervical dislocation; BAT was harvested and kept in cold Washing Buffer (KRBH solution, 1% (w/v) BSA (Albumin bovine serum, fraction V, minimum 96% lyophilized)), until further processing. BAT samples were then transferred to Digestion Buffer (KRBH solution, 3.5% (w/v) BSA, 0.5 mg/ml glucose (D(+)- glucose anhydrous), 1 mg/ml Collagenase type II from Clostridium histolyticum), at a ratio of 10 ml Digestion Buffer per gram of tissue. They were then mechanically minced and allowed to digest at 37 °C and under 250 rpm shaking for 30'−1 h. Undigested material was discarded by using a 100-μm strainer. The BAT digestion solution was diluted (to lower collagenase activity) and used directly. For glucose oxidation, 0.5 ml adipocyte solution was incubated for 4 h with 0.5 ml Oxidation Buffer (KRBH solution, 1% (w/v) BSA, 5 mmol/L glucose, 1 μCi/mL (D-[U]-14C-glucose) in 6-well plates, and CO2 was collected as in ref. Data are expressed as nmol of glucose/h corrected by DNA content. Glucose oxidation from wild-type tissues was used as 100% value. Approximately 1-mm3 pieces of BAT were fixed in 2% paraformaldehyde and 2.5% glutaraldehyde in phosphate buffer. The protocol is described in ref. Sections were observed under a JEM-1010 electron microscope (Jeol, Japan) equipped with a CCD camera SIS Megaview III and the AnalySIS software. BAT, adipose tissue depots and liver were fixed in 10% formaldehyde in methanol and then paraffin-embedded. Sections were subjected to standard Hematoxylin/Eosin (HE) staining. Images were acquired with a NanoZoomer-2.0 HT C9600 digital scanner (Hamamatsu, Photonics, France) equipped with a 20X objective and coupled to a mercury lamp unit L11600-05 and using NDP.scan2.5 software U10074-03 (Hamamatsu, Photonics, France). Adipocyte area was measured using open software ImageJ (W. Rasband, National Institute of Mental Health, Bethesda, Maryland, USA). Liver fat content was estimated from HE-stained slides using the CTAn software (SkyScan, Kontich, Belgium). DAPI/Wheat Germ Agglutinin (WGA)/UCP1 staining was performed as reported67. RNA from various tissues was extracted using a protocol combining TRIZOL reagent (Invitrogen, Carlsbad, CA, USA) and RNAeasy® minikit columns (Qiagen. Alameda, CA, USA), following the manufacturer's instructions. RNA was reverse-transcribed with the SuperScript RTII kit (Invitrogen, Carlsbad Ca, USA). PCRs were performed using the ABI Prism 7900 HT real-time PCR machine (Applied Biosystems, USA) and the SYBR® Green PCR Master Mix (Applied Biosystems, USA), or Taqman Master Mix. The thermal protocol was 10 min at 95 °C, followed by 40 cycles of 15'/95 °C–60'/60 °C. All measurements for mouse mRNA were normalized to Arp. For each designed primer set, a melting curve of PCR products was performed to verify unspecific amplification or primer-dimer formation. The primers used are specified in Table S2. Total DNA was extracted from mouse interscapular BAT, inguinal subcutaneous Adipose tissue (SAT) and perigonadal adipose tissue (PAT). Tissue was digested with proteinase K (Merck Life Science S.L.U., 03115801001) and total DNA was purified using DNeasy Blood & Tissue Kit (Qiagen, 69504). Total cellular DNA (1 ng DNA) is used as a template and is amplified with specific primers for Cox2 gene, as a mitochondrial gene (Cox2f: CTACAAGACGCCACAT and Cox2r: GAGAGGGGAGAGCAAT) and for Sdha gene, as a nuclear gene (Sdhaf: TACTACAGCCCCAAGTCT, Sdhar: TGGACCCATCTTCTATGC) by Real-Time PCR using SYBRGreen. The ratio genomic-nuclear DNA/ mtDNA was calculated. The concentration of total RNA in preparations was quantified with Nanodrop One (Thermo Fisher), and RNA integrity was assessed using the Bioanalyzer 2100 RNA Nano assay (Agilent). Libraries for RNA-seq were prepared at IRB Barcelona's Functional Genomics Core Facility. Briefly, mRNA was isolated from 1.2 μg of total RNA using the kit NEBNext Poly(A) mRNA Magnetic Isolation Module (New England Biolabs). NGS libraries for RNA-seq were prepared from the purified mRNA using the NEBNext Ultra II RNA Library Prep Kit for Illumina (New England Biolabs). Seven (WT/RAP250) and eight (Control/UCP1-NRN1) cycles of PCR amplification were applied to all libraries. Libraries were quantified using the Qubit dsDNA HS assay (Invitrogen) and quality controlled with the Bioanalyzer 2100 DNA HS assay (Agilent), for WT/RAP250 BAT, and Tapestation HS D5000 assay (Agilent) for Control/NRN1 BAT. For WT/RAP250 BAT RNA-seq an equimolar pool was prepared with the nine libraries and subjected to sequencing at the Centre for Genomic Regulation. A final qPCR quality control was performed before sequencing in one lane of an Illumina HiSeq2500. Sequencing output was 270 million 50-bp single-end reads, and at least 28 million reads were obtained for all samples. For Control/UCP1 BAT RNAseq an equimolar pool was prepared with the nine libraries and subjected to paired-end 150 nt sequencing on a NovaSeq6000 S4 (Illumina). FastQ files were aligned against the Mus musculus mm10 genome with STAR 2.5.2b68 using default options. Aligned reads were used to obtain raw counts at mm10 NCBI RefSeq gene level using featureCounts from Rsubread version 1.32.469 with options minMQS=1. DESeq2 1.22.270 was used to find differentially expressed genes in KO vs WT. Genes were selected as differentially expressed with thresholds |FC| > 1.5 and Benjamini-Hochberg adjusted P < 0.1. For functional enrichment analysis (GSEA), regularized log transformation (rlog) was applied to read counts using DESeq2 and rotation gene set testing using the maxmean statistic was performed71. For cell type deconvolution analyses, the DWLS algorithm (https://github.com/dtsoucas/DWLS) was used with reference single cell data obtained from ref. All analyses were performed in R-3.5.1 (R Core Team, 2018). Immortalized brown preadipocytes were generated as described in refs. Retroviral vector pBabe encoding SV40 LT-Puro antigen was purchased from Addgene. Cells were grown in monolayer culture in DMEM supplemented with 10% (v/v) fetal bovine serum (FBS) (v/v) antibiotics (10,000 units/ml penicillin G and 10 mg/ml streptomycin), and 25 mM Hepes (pH 7.4). The differentiation process is described in ref. Immortalized brown preadipocytes were also used to generate the NRN1 deficiency model. Lentivirus encoding for a shRNA scramble or against Nrn1 (TRCN0000177415 validated clone from Merk), as control (SCR) or knockdown (shNRN1), were produced in HEK 293 T cells (ATCC, CRL-11268) as reported74. Transduced cells were selected based on Puromycin resistance (Santa Cruz Biotechnology, sc-108071; 3 μg/ml). Adipocyes and SVF were obtained as reported75,76. A DNA fragment encoding amino acids 28-115 inclusive of mouse Neuritin (Accession number NM_153529) was amplified using the following semi-nested primers: The resulting 343 bp fragment was purified using AMPure (Beckman Coulter) before InFusion (Takara Bio) into KpnI and NotI-cut pOPINTTG77. The resulting construct will produce the following protein when expressed in mammalian cells: The protein consists of the µ-phosphatase signal peptide (with signal peptidase site indicated, first ↓, a Hexa-histidine tag and TEV cleavage site (indicated, second ↓) followed by amino acids 28-115 of mouse Neuritin. This protein will be secreted, can be purified by Immobilized Metal Affinity Chromatography (IMAC) and can then be cleaved with TEV protease to produce mature ‘soluble' NRN1 (this construct is truncated to also exclude the GPI-anchored G116). 293-6E cells78 were cultured in shaken flasks in F-17 Freestyle media (Gibco) supplemented with GlutaMax, Poloxamer 188 and G418 (final concentrations of 4 mM, 0.1% and 25 µg/ml, respectively) at 37 °C in 5% CO2. After 24 h the media was supplemented with sterile TN1 solution (Tryptone N1, TekniScience Inc. to 0.5% w/v in F-17 Freestyle media). After a further 96 hours the media was cleared by pelleting the cells by centrifugation at 500 × g then 5000 g followed filtration through Whatman 3 M paper filters. All further steps were performed at 4 °C. Urea was added to a final concentration of 4 M and the pH adjusted to pH 7.5 before the sample was applied to a 5 ml Excel IMAC column equilibrated with IMAC Wash Buffer (Tris pH8 50 mM, NaCl 150 mM, Urea 4 M, Imidazole 20 mM, DTT 2 mM). His-NRN containing fractions were pooled and buffer exchanged to Digest Buffer (Tris pH8 50 mM, NaCl 500 mM, Urea 2 M, DTT 2 mM) using a HiPrep 26/10 Desalt column before the addition of His-tagged TEV protease at a 1:25 His-TEV:His-NRN ratio and incubation overnight at 4 °C. The TEV-digested sample was first passed through a 1 ml Excel HisTrap column to capture uncut His-NRN and His-TEV before size exclusion chromatography (SEC) was performed on a Superdex 30 pg 16/600 column equilibrated with Tris pH8 20 mM, NaCl 150 mM, Urea 6 M, DTT 2 mM, EDTA 2 mM. Peak fractions were buffer exchanged (as before, using a HiPrep 26/10 Desalt column) into phosphate-buffered saline (PBS) before use. Adipocytes (Scr and shNRN1) at day 10 of differentiation were stimulated with Isoproterenol (10 uM) in DMEM with 10% FBS for 2 h. Cells were lysed to obtain RNA or total cell extracts. For the detection of proteins phosphorylated by PKA (p-PKA substrates) control adipocytes were washed in PBS twice and were incubated during two hours in media without serum. Cells were treated with NRN1 at different concentration during 2 h and then total cell extracts were obtained. For NRN1 chronic studies, brown adipocytes at day 4 of differentiation were treated with NRN1 at 100 ng/ml in the presence of differentiation medium. NRN1 was maintained during all the process of differentiation. Cells were washed twice in cold phosphate-buffered saline (PBS) and lysed in Triton X-100 lysis buffer (50 mM Tris–HCl pH 7.5, 100 mM NaCl, 50 mM NaF, 5 mM EDTA, 40 mM β-glycerophosphate, 1% Triton X-100 and protease inhibitor cocktail (ROCHE)) for 15 min at 4 °C. Proteins from total homogenates were resolved in 8% (RAP250 detection) or 10% acrylamide gels for SDS/PAGE and transferred to Immobilon membranes (Millipore). Proteins were detected by the ECL method and quantified by densitometry using Genetools from Syngene or by NIR-fluorescence detected with LI-COR Odyssey System and measured with Image Studio software. The following antibodies were used: RAP250 (ASC2 antibody BL1874, Bethyl laboratories and NCOA6 polyclonal antibody 25241-1_AP, Proteintech); p-HSL (S563, #4139), HSL (#4107), P-PKA substrates (#9624), βactin (AC-15, SIGMA); α-tubulin (SIGMA); Vinculin (Abcam); GAPDH (2118S, Cell Signaling), UCP1 (ab10983 Abcam); PPARγ and C/EBPα (Santa Cruz); total OXPHOS rodent WB antibody cocktail (ab110413, Abcam), ADRB3 (bs-1063R Bioss Antibodies), NRN1 (B-9 Santa Cruz and AF283 from Biotechne R&D Systems); TH (R&D Systems MAB7566SP and ab6211 from Abcam). Preadipocytes (Scr/ ShNRN1) were plated, grown and differentiated as it is described in ref. 67 For NRN1 treated cells, recombinant NRN1 protein was added chronically at day 4 of differentiation until d9-d10 at a final concentration of 100 ng/ml. Media was changed every two days. Seahorse experiments were performed as reported79. Neuritin 1 (NRN1) ELISA Kit (Catalog No: abx155876, Abbexa) was used to measure Nrn1 from mouse plasma and cell culture media. 75 μl of plasma and 300 μl of culture media were used following a commercial protocol. Data are presented as mean ± SEM. Statistical significance was determined by two-tailed t-Student test, or by two-way ANOVA (Šidak's multiple comparisons test or by an Ordinary two-way ANOVA), or by Covariance (ANCOVA) with weight as the covariate. Significance was established at P < 0.05. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated or analyzed during this study are included in this published article (and its supplementary information and Source Data files). The complete transcriptomic datasets were deposited to the National Center for Biotechnology Information's Gene Expression Omnibus Database and are accessible through GEO Series accession number GSE295442 and GSE295443. Source data are provided with this paper. All data supporting the findings is also available from the corresponding author upon request. Source data are provided with this paper. Apovian, C. M. The clinical and economic consequences of obesity. Simonnet, A. et al. High Prevalence of Obesity in Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) Requiring Invasive Mechanical Ventilation. Ayres, J. S. A metabolic handbook for the COVID-19 pandemic. Identification and importance of brown adipose tissue in adult humans. Unexpected evidence for active brown adipose tissue in adult humans. Saito, M. et al. High incidence of metabolically active brown adipose tissue in healthy adult humans: effects of cold exposure and adiposity. The changed metabolic world with human brown adipose tissue: therapeutic visions. Brown adipose tissue: function and physiological significance. The heartwarming effect of brown adipose tissue. Cereijo, R. et al. CXCL14, a brown adipokine that mediates brown-fat-to-macrophage communication in thermogenic adaptation. Keipert, S. et al. Genetic disruption of uncoupling protein 1 in mice renders brown adipose tissue a significant source of FGF21 secretion. Hui, X. et al. Adiponectin enhances cold-induced browning of subcutaneous adipose tissue via promoting m2 macrophage proliferation. Kong, X. et al. Brown adipose tissue controls skeletal muscle function via the secretion of myostatin. Deshmukh, A. S. et al. Proteomics-based comparative mapping of the secretomes of human brown and white adipocytes reveals epdr1 as a novel batokine. The brown fat-enriched secreted factor Nrg4 preserves metabolic homeostasis through attenuation of hepatic lipogenesis. Whittle, A. J. et al. BMP8B increases brown adipose tissue thermogenesis through both central and peripheral actions. Zeng, X. et al. Innervation of thermogenic adipose tissue via a calsyntenin 3beta-S100b axis. Tseng, Y. H. et al. New role of bone morphogenetic protein 7 in brown adipogenesis and energy expenditure. & Samuels, H. H. Nuclear hormone receptor coregulator: role in hormone action, metabolism, growth, and development. Goo, Y. H. et al. Activating signal cointegrator 2 belongs to a novel steady-state complex that contains a subset of trithorax group proteins. Lee, S. et al. Coactivator as a target gene specificity determinant for histone H3 lysine 4 methyltransferases. & Samuels, H. H. Nuclear receptor coactivator/coregulator NCoA6(NRC) is a pleiotropic coregulator involved in transcription, cell survival, growth and development. & Xu, J. Tissue- and nuclear receptor-specific function of the C-terminal LXXLL motif of coactivator NCoA6/AIB3 in mice. Kim, G. H. et al. Regulation of hepatic insulin sensitivity by activating signal co-integrator-2. The nuclear receptor coactivator AIB3 is a modulator of HOMA beta-cell function in nondiabetic children. Yeom, S. Y. et al. Regulation of insulin secretion and beta-cell mass by activating signal cointegrator 2. Qi, C. et al. Transcriptional coactivator PRIP, the peroxisome proliferator-activated receptor gamma (PPARgamma)-interacting protein, is required for PPARgamma-mediated adipogenesis. Targeted inactivation of MLL3 histone H3-Lys-4 methyltransferase activity in the mouse reveals vital roles for MLL3 in adipogenesis. Marron, T. U. et al. Androgen-induced neurite outgrowth is mediated by neuritin in motor neurones. Nedivi, E., Hevroni, D., Naot, D., Israeli, D. & Citri, Y. Numerous candidate plasticity-related genes revealed by differential cDNA cloning. Nedivi, E., Fieldust, S., Theill, L. E. & Hevron, D. A set of genes expressed in response to light in the adult cerebral cortex and regulated during development. Gao, R. et al. Exogenous Neuritin Promotes Nerve Regeneration After Acute Spinal Cord Injury in Rats. Mahajan, M. A., Das, S., Zhu, H., Tomic-Canic, M. & Samuels, H. H. The nuclear hormone receptor coactivator NRC is a pleiotropic modulator affecting growth, development, apoptosis, reproduction, and wound repair. & Warwick, B. P. The effect of high fat and high carbohydrate cafeteria diets on diet-induced thermogenesis in the rat. Mercer, S. W. & Trayhurn, P. Effect of high fat diets on the thermogenic activity of brown adipose tissue in cold-acclimated mice. Lorsignol, A., Rabiller, L., Labit, E., Casteilla, L. & Penicaud, L. The nervous system and adipose tissues: a tale of dialogues. Blaszkiewicz, M., Willows, J. W., Johnson, C. P. & Townsend, K. L. The importance of peripheral nerves in adipose tissue for the regulation of energy balance. Early B Cell Factor Activity Controls Developmental and Adaptive Thermogenic Gene Programming in Adipocytes. Cypess, A. M. et al. Activation of human brown adipose tissue by a beta3-adrenergic receptor agonist. The Type 2 Diabetes Knowledge Portal: An open access genetic resource dedicated to type 2 diabetes and related traits. Kuang, S. Q. et al. Deletion of the cancer-amplified coactivator AIB3 results in defective placentation and embryonic lethality. Antonson, P. et al. Inactivation of the nuclear receptor coactivator RAP250 in mice results in placental vascular dysfunction. Oh, G. S. et al. Regulation of Hepatic Gluconeogenesis by Nuclear Receptor Coactivator 6. Himms-Hagen, J., Hogan, S. & Zaror-Behrens, G. Increased brown adipose tissue thermogenesis in obese (ob/ob) mice fed a palatable diet. Naeve, G. S. et al. Neuritin: a gene induced by neural activity and neurotrophins that promotes neuritogenesis. Zito, A. et al. Neuritin 1 promotes neuronal migration. Fujino, T. et al. CPG15 regulates synapse stability in the developing and adult brain. Zhao, Q. R. et al. Neuritin reverses deficits in murine novel object associative recognition memory caused by exposure to extremely low-frequency (50 Hz) electromagnetic fields. Gonzalez-Figueroa, P. et al. Follicular regulatory T cells produce neuritin to regulate B cells. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. & Shilatifard, A. Epigenetic balance of gene expression by Polycomb and COMPASS families. The MLL3/MLL4 branches of the COMPASS family function as major histone H3K4 monomethylases at enhancers. Lee, M. G. et al. Demethylation of H3K27 regulates polycomb recruitment and H2A ubiquitination. Pellegrinelli, V. et al. Adipocyte-secreted BMP8b mediates adrenergic-induced remodeling of the neuro-vascular network in adipose tissue. Yu, H. et al. Neurotrophic factor Neuritin modulates T cell electrical and metabolic state for the balance of tolerance and immunity. Bosserhoff, A. K., Schneider, N., Ellmann, L., Heinzerling, L. & Kuphal, S. The neurotrophin Neuritin1 (cpg15) is involved in melanoma migration, attachment independent growth, and vascular mimicry. Yao, J. J. et al. Neuritin activates insulin receptor pathway to up-regulate Kv4.2-mediated transient outward K+ current in rat cerebellar granule neurons. Shimada, T., Yoshida, T. & Yamagata, K. Neuritin Mediates Activity-Dependent Axonal Branch Formation in Part via FGF Signaling. Yao, J. J., Zhao, Q. R., Lu, J. M. & Mei, Y. In vivo adeno-associated viral vector-mediated genetic engineering of white and brown adipose tissue in adult mice. Sebastian, D. et al. Mitofusin 2 (Mfn2) links mitochondrial and endoplasmic reticulum function with insulin signaling and is essential for normal glucose homeostasis. Perrin, C., Knauf, C. & Burcelin, R. Intracerebroventricular infusion of glucose, insulin, and the adenosine monophosphate-activated kinase activator, 5-aminoimidazole-4-carboxamide-1-beta-D-ribofuranoside, controls muscle glycogen synthesis. Judex, S. et al. Quantification of adiposity in small rodents using micro-CT. Methods 50, 14–19 (2010). & Speakman, J. R. Some mathematical and technical issues in the measurement and interpretation of open-circuit indirect calorimetry in small animals. Tschop, M. H. et al. A guide to analysis of mouse energy metabolism. Soro-Arnaiz, I. et al. Role of Mitochondrial Complex IV in Age-Dependent Obesity. Sabate-Perez, A. et al. Autophagy-mediated NCOR1 degradation is required for brown fat maturation and thermogenesis. & Gingeras, T. R. Mapping RNA-seq Reads with STAR. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. & Stephan-Otto Attolini, C. Roastgsa: a comparison of rotation-based scores for gene set enrichment analysis. Shamsi, F. et al. Vascular smooth muscle-derived Trpv1(+) progenitors are a source of cold-induced thermogenic adipocytes. Klein, J. et al. β3-Adrenergic Stimulation Differentially Inhibits Insulin Signaling and Decreases Insulin-induced Glucose Uptake in Brown Adipocytes. Castillo, A. et al. Adipocyte MTERF4 regulates non-shivering adaptive thermogenesis and sympathetic-dependent glucose homeostasis. Enguix, N. et al. Mice lacking PGC-1beta in adipose tissues reveal a dissociation between mitochondrial dysfunction and insulin resistance. Berrow, N. S. et al. A versatile ligation-independent cloning method suitable for high-throughput expression screening applications. High-level and high-throughput recombinant protein production by transient transfection of suspension-growing human 293-EBNA1 cells. Oeckl, J., Bast-Habersbrunner, A., Fromme, T., Klingenspor, M. & Li, Y. Isolation, culture, and functional analysis of murine thermogenic adipocytes. We thank J. Seco for technical assistance. We also thank the Functional Genomics, Biostatistics/Bioinformatics and Histopathology Facilities at IRB Barcelona, and the Cytometry and Electron Microscopy at the CCiT-UB. was a predoctoral fellow of the Fundaçao para a Ciência e Tecnologia - Ministério da Ciência, Tecnologia e Ensino Superior, Portugal. This study was supported by research grants from MICINN (PID2022-137576OB-I00), Generalitat de Catalunya, INFLAMES (PIE-14/00045) Instituto de Salud Carlos III, CIBERDEM (“Instituto de Salud Carlos III”), CIBEREHD and Fondo de Investigaciones Sanitarias (PI22/00526); Junta de Castilla y Leon” (SA113/P23); Fundación Ramon Areces (CIVP18A3942), Fundación BBVA, the Fundació Marató de TV3 (20132330 and 201916-31), the European Foundation for the Study of Diabetes (EFSD) and AFM Téléthon. is a recipient of an ICREA “Academia” Award (Generalitat de Catalunya). We gratefully acknowledge institutional funding from MINECO through the Centres of Excellence Severo Ochoa Award, and from the CERCA Programme of the Generalitat de Catalunya. JAG is thankful for support from the Robert A. Welch Foundation (E-0004) and the Swedish Science Council. Present address: Medical BioSciences department, RadboudUMC, Geert Grooteplein 28, 6525, GA, Nijmegen, The Netherlands Present address: Department of Internal Medicine, Erasmus MC, University Medical Center Rotterdam, Rotterdam, The Netherlands Present address: Unit of Molecular Metabolism, Lund University Diabetes Centre, SE-20213, Malmö, Sweden Present address: Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, 2200, Denmark Present address: Department of Biochemistry and Physiology, School of Pharmacy and Food Sciences, University of Barcelona, 08028, Barcelona, Spain Manuela Sánchez-Feutrie, Montserrat Romero, Sónia R. Veiga, Núria Borràs-Ferré, Nick Berrow, Martina Ràfols, Noemí Giménez, Andrea Rodgers-Furones, Alba Sabaté-Pérez, Ángela Rodríguez Pérez, Luis Rodrigo Cataldo, Hans Burghardt, David Sebastián, Natàlia Plana, Vanessa Hernández, Laura Isabel Alcaide, Óscar Reina, Manuel Palacín & Antonio Zorzano Manuela Sánchez-Feutrie, Montserrat Romero, Sónia R. Veiga, Núria Borràs-Ferré, Alba Sabaté-Pérez, Hans Burghardt, Manuel Palacín & Antonio Zorzano CIBER de Diabetes y Enfermedades Metabólicas Asociadas (CIBERDEM), Instituto de Salud Carlos III, Madrid, Spain Manuela Sánchez-Feutrie, Montserrat Romero, Sónia R. Veiga, Núria Borràs-Ferré, Alba Sabaté-Pérez, Hans Burghardt, David Sebastián & Antonio Zorzano Maria J. Monte & José Juan G. Marin CIBER de Enfermedades Raras (CIBERER), Instituto de Salud Carlos III, Madrid, Spain UMR 1297 INSERM, Team INCOMM Rangueil Hospital, Toulouse, 31400, France Center for Nuclear Receptors and Cell Signaling, Department of Biology and Biochemistry, University of Houston, Houston, TX, 77204, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Manuela Sánchez-Feutrie or Antonio Zorzano. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Identification of Neuritin 1 as a local metabolic regulator of brown adipose tissue. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01653-y'>tRNA as an assembly chaperone for a macromolecular transcription-processing complex</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-04 09:05:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Transfer RNAs (tRNAs) are widely recognized for their role in translation. Here, we describe a previously unidentified function of tRNA as an assembly chaperone. During poxviral infection, tRNAGln/Arg lacking the anticodon mcm5s2U34 modification is specifically sequestered from the cellular tRNA pool to promote formation of a multisubunit poxviral RNA polymerase complex (vRNAP). Cryo-electron microscopy analysis of assembly intermediates illustrates how tRNAGln/Arg orchestrates the recruitment of transcription and mRNA processing factors to vRNAP where it controls the transition to the preinitiation complex. This is achieved by an induced fit mechanism that internalizes anticodon base G36 into the anticodon stem, creating a noncanonical tRNA structure and selecting a defined tRNA modification pattern. The role of tRNA as an assembly chaperone extends to the pathogenic Mpox virus, which features a similar vRNAP. Transfer RNAs (tRNAs) are best known for their role in decoding mRNA codons and translating them into proteins1. However, certain tRNAs are also involved in a variety of noncanonical functions including nutrient sensing2, splicing3, transcription4, apoptosis5 and scaffolding6, as reviewed by Su et al.7. In these contexts, tRNAs or their fragments can act as antisense decoys, protein modulators, primers or sensors. Recently, human tRNAGln and, to a lesser extent, tRNAArg have been identified in a cellular context that is not consistent with any of the established functions of tRNAs. Specifically, these tRNAs (termed tRNAGln/Arg throughout this manuscript) were found to be a stoichiometric component of a macromolecular RNA polymerase complex, known as complete vRNAP, which forms in cells upon infection with the prototypic poxvirus vaccinia8.This virus belongs to the diverse group of nucleocytoplasmic large DNA viruses, comprising double-stranded DNA viruses that express their genome within the cytoplasm of their host using their own gene expression machinery9,10. The megadalton complete vRNAP unit integrates the poxviral core RNA polymerase (core vRNAP), composed of eight Rpo subunits, with early transcription factors Rap94, VETF-s, VETF-l, NPH-I and E11, the capping enzyme dimer D1/D12 and host tRNAGln/Arg. We showed previously that recruited tRNAGln/Arg is uncharged and tethers associated factors to vRNAP through interactions with Rap94, NPH-I and VETF-l (ref. Consistent with its biochemical composition, complete vRNAP acts as an autonomous early transcription unit capable of generating m7G-capped transcripts8,11. Notably, tRNAGln/Arg, although part of this complex, is not directly involved in transcription and is absent from all DNA-bound transcription complexes identified to date. We, therefore, hypothesized that these tRNAs do not function as transcription factors but rather as chaperones that control the association of vRNAP with adjunct factors required for early transcription. Here, we combined a biochemical reconstitution system with structural analysis by cryo-electron microscopy (cryo-EM) to investigate the assembly pathway of complete vRNAP. Our study uncovers an unknown function of a specific tRNA as an assembly chaperone and reveals a unique induced fit mechanism that involves structural rearrangement of the tRNA, enabling complex formation. To test whether tRNAGln/Arg acts as an assembly chaperone rather than a scaffold, we dissected the steps that lead to the formation of complete vRNAP. This unit, along with recombinant transcription factors (Extended Data Fig. 1a) and tRNAGln/Arg isolated from complete vRNAP, was then used to study the formation of complete vRNAP in vitro (Fig. Cosedimentation and gel mobility shift experiments showed that [32P]tRNAGln/Arg alone failed to bind minimal vRNAP (Fig. 1b, bottom, gradient profiles 1–4) or any of the individual transcription factors (Fig. Transition to specific and stable tRNAGln/Arg binding occurred only in the presence of Rap94, NPH-I and E11, as demonstrated by band shift assays (Fig. Of note, none of these factors bound stably to each other (Extended Data Fig. 1b) or to core vRNAP in the absence of tRNAGln/Arg (Extended Data Fig. 1c), illustrating its crucial role in assembly. However, weak background binding activity can be attributed to existent invariable contact surfaces involving these three components, as documented in the structure of complete vRNAP. 1c, lane 9), D1/D12 also readily associated with vRNAP (Fig. 1c, lanes 12 and 13) whereas the VETF-s/l heterodimer was not required for these assembly steps, as discussed below. The fully assembled complete vRNAP exhibited remarkable stability, as evidenced by its inability to bind or exchange [32P] tRNAGln/Arg (Fig. Minimal and core vRNAPs were isolated from infected cells by anti-FLAG affinity chromatography; factors D1/D12, E11 and NPH-I were expressed in E. coli and purified as described in the Methods. tRNAGln/Arg was obtained from purified complete vRNAP as described in the Methods. b, Top, silver-stained protein gel of fractions from a sucrose density gradient centrifugation of isolated vRNAP complexes. tRNAGln/Arg from purified vRNAP was visualized by ethidium bromide staining. Bottom, [32P]tRNAGln/Arg was incubated with the indicated vRNAP complexes, separated by gradient centrifugation and analyzed by autoradiography. The experiment was performed in triplicate. c, Gel mobility shift assays of core vRNAP and recombinant factors in the presence of [32P]tRNAGln/Arg. Complex formation was visualized by autoradiography. The experiment was performed four times. The simultaneous requirement of E11, NPH-I, Rap94 and tRNAGln/Arg for the formation of complete vRNAP suggested a complex binding mechanism, potentially involving conformational changes among interacting partners. Using our biochemical reconstitution system, we leveraged cryo-EM to decipher the mechanism of assembly by visualizing critical intermediates. Atomic structures of the intermediates (labeled I1–I5) were then used to propose an assembly cycle that configures complete vRNAP and enables its subsequent conversion into the preinitiation complex (PIC), ready to initiate transcription at an early promoter (Figs. The color code of the individual factors is identical to that of Figs. Each structure is shown in a standard view that corresponds to that used in Fig. 3 and an additional view with 40° rotation. Rpo30 is highlighted in ochre only for minimal vRNAP. Note that, in general, all bound factors occupy corresponding positions; therefore, the visualized structures document an incremental buildup, eventually yielding to complete vRNAP in the absence of major conformational reorganizations (full assembly cycle in Fig. A model of the tRNA-chaperoned assembly cycle was derived from biochemical and structural data of this study and is illustrated by solved and available atomic structures, the latter of which are labeled with their PDB codes. Assembly intermediates, shown as accessible molecular surface, are labeled according to their appearance during the assembly cycle: I1, core vRNAP (PDB 6RIC); I2*, metastable core vRNAP–tRNA (no experimental structure available); I3, core vRNAP–tRNAGln/Arg–E11–NPH-I; I4, complete vRNAP–VETF-s/l; I5, complete vRNAP lacking the capping enzyme (complete vRNAP–D1/D12). The starting point is minimal vRNAP and the endpoint is PIC on an early promoter (PDB 7AMV). *Transient intermediate that can be detected only by biochemical means (Fig. #Conformational change in the tRNA assembly chaperone. We first determined the structure of minimal vRNAP (Fig. A comparison with the already known structure of core vRNAP (assembly intermediate I1)8 (Fig. In doing so, Rap94 adopts a conformation that remains stable throughout the whole assembly cycle (Extended Data Fig. In agreement with our biochemical experiments (Fig. 1b), binding of tRNAGln/Arg to core vRNAP (assembly intermediate I2*) turned out to be too transient for structural analysis. However, the stable assembly intermediate I3 was accessible for cryo-EM reconstruction, revealing how E11 connects with NPH-I and facilitates tRNA binding to core vRNAP (Fig. tRNAGln/Arg is anchored through extensive interactions of its D-arm and T-arm and the anticodon stem with Rap94. Upon NPH-I binding, its N-lobe squeezes the tRNAGln/Arg anticodon stem loop (ASL) into a strained conformation. Notably, the interface of Rap94 with tRNAGln/Arg does not depend on specific base contacts; instead, several bases in the ASL are specifically recognized by the NPH-I C-lobe, as discussed below. Full stability of the vRNAP complex is achieved by independently recruiting the VETF-s/l and D1/D12 heterodimers, forming either complete vRNAP–VETF (I4; Fig. 4 and Supplementary Table 1) or complete vRNAP–D1/D12 (I5; Fig. 5 and Supplementary Table 1), depending on which factor binds first. It is noteworthy that I5 was found in native vRNAP preparations, whereas I4 was only observed during reconstitution studies in vitro. It stands to reason that both intermediates enable alternative pathways leading to the assembly of complete vRNAP (Figs. Notably, complete vRNAP is characterized by exceptional stability because of a belt-like structure formed by E11, NPH-I, tRNAGln/Arg, VETF-s/I and D1/D12 on the surface of core vRNAP (Fig. a, An isosurface representation of the cryo-EM density with crucial structural elements labeled. This density can be attributed to mobile VETF. c, Electrostatic potential mapped to the complete vRNAP solvent-accessible surface. The orientation of both views is identical to those depicted in b. The lowpass-filtered density overlay shown in b is not presented here. As shown previously12, complete vRNAP can bind to early viral promoters and reconfigures under the elimination of E11, NPH-I, D1/D12 and tRNAGln/Arg into the PIC. tRNAGln/Arg, hence, orchestrates an assembly pathway without being a component of complete vRNAP, the final product (Fig. To explore the details of the tRNA–protein interactions enabling the assembly process, we generated a high-resolution cryo-EM dataset of complete vRNAP and rerefined our previous structural model (Figs. Upon superimposing the structure of tRNAGln as part of complete vRNAP with other known tRNA structures from the Protein Data Bank (PDB), we identified notable deviations in the base-pairing pattern of the anticodon stem. These differences were associated with a strained backbone conformation and the coordination of a magnesium ion (Fig. Similar rearrangements are possible for tRNAArg (Extended Data Fig. a, ASL structure from the cryo-EM model of complete vRNAP (strained state) depicted as a stick model, overlaid with the cryo-EM density shown as a transparent isosurface. b, ASL structure from the cryo-EM model of complete vRNAP depicted as cartoon-and-stick model. Please note that the relaxed state (free tRNA) is a manually created model because no experimental structure of this state is available (left, bound strained state; right, free relaxed state). Please note that this is a manually created model because no experimental structure of this state is available. g, Right, ASL region in cartoon-and-stick depiction with the adjacent NPH-I solvent-accessible surface (red) as observed in the model of complete vRNAP (strained state). h, Codon ‘sun' with recognition-relevant bases marked in color. i, ASL base-pairing scheme for tRNAGln(UUG) as observed in complete vRNAP in the strained state (bottom left). Sequences of several other tRNA species are threaded into the scheme for the other depictions. Unfavorable base pairing is indicated by two red bars; canonical base pairs are shown in gray and nonconical base pairs are shown in red. If present, pictograms in insets symbolize the relaxed (unbound) or strained (bound) states (Supplementary Video 2). Please note that there is no high-confidence tRNAArg(GCG) sequence available in the database. Because no other experimental structure for human tRNAGln(UUG) existed in the database, we modeled its ground state starting from the available structure of bacterial tRNAGln13 and performed energy minimization. 5a–c) into a pocket located in the N-lobe of NPH-I (Fig. Therefore, the strained ASL conformation in complete vRNAP sharply contrasts with the relaxed ASL conformation (ground state) of free tRNA (Fig. Lastly, we examined the RNA–protein interactions that might induce this strained conformation. In tRNAGln(UUG), the anticodon sequence comprises the bases U34, U35 and G36 (asterisks in Fig. Interestingly, there is no specific interaction between the protein and the nucleic acid bases in most tRNA regions, except for a selective pocket accommodating U35, which seems poorly suited for any base other than a pyrimidine (Fig. This observation suggests that the selectivity for tRNAGln (and tRNAArg, as discussed below) cannot be attributed to protein–tRNA interactions alone. Three specific conditions must be met for this interaction to occur. First, a noncanonical base pair must form between bases 33 and 36 at the distal position of the anticodon stem. Here, the imino group (N1) of G36 acts as a hydrogen-bond donor, binding to the carbonyl oxygen atom at C2 of U33 (limitation 1; Fig. 5b,c, yellow arrow, Supplementary Video 2, Extended Data Fig. Second, the anticodon base at position 35 fits into a hydrophobic pocket on NPH-I that can only accommodate a pyrimidine (limitation 2; Fig. 5g, green arrow), necessitating a purine in the codon sequence (Fig. Lastly, a noncanonical base pair must form between C32 and A38 in the anticodon stem. This pairing is compatible with five ASL sequences found in tRNAGln(UUG/CAG) and tRNAArg(UCG/CCG/ACG) but not with those found in tRNAHis, tRNAPro or tRNALeu (condition 3; Fig. Notably, tRNAArg(CCG) isoforms 1-1, 1-2 and 1-3 deviate in that they feature a canonical U32–A38 base pair (Fig. These constraints ensure that only tRNAGln and tRNAArg can act as assembly factors for complete vRNAP. Accordingly, these tRNA species are exclusively selected upon poxviral infection, determined by quantitative tRNA sequencing of isolated complete vRNAP, as discussed below. We conclude that complete vRNAP assembly occurs through a novel tRNA recognition mechanism, where anticodon readout is determined by internal base pairing. tRNAs belong to the most extensively modified noncoding RNA species14,15 and it was a possibility that specific modifications of tRNAGln/Arg influence its role in assembling the complete vRNAP complex. Supporting this hypothesis, we found that unmodified tRNAGln produced by transcription in vitro was less effective in promoting vRNAP assembly compared to tRNAGln/Arg extracted from complete vRNAP (Fig. a, Gel mobility shift assay of nonmodified (T7-transcribed) tRNASer (lanes 1–4, 13) and tRNAGln (lanes 5–8, 14) or modified tRNAGln/Arg (extracted from complete vRNAP; lanes 9–12, 15) in the presence of indicated adjunct factors E11, NPH-I and D1/D12 and core vRNAP. c, LC–MS analysis of mcm5s2U and cm5s2U modifications in total tRNA isolated from infected cells. d, LC–MS analysis mcm5s2U and cm5s2U modifications in total tRNA isolated from uninfected cells. e, K. lactis γ-toxin digestion of tRNAs. RNA was subsequently phenol-extracted, resolved by denaturing RNA gel electrophoresis and visualized by radioautography. In lanes 15–18, total tRNA from infected cells was digested with increasing concentrations of γ-toxin (500 nM, 1 µM and 5 µM), followed by TRIzol extraction. tRNAs were separated on a 12% denaturing gel, transferred to a nylon membrane and hybridized with a 5′-end-labeled DNA probe complementary to the 3′-end of the tRNAGln(UUG) anticodon region. Using liquid chromatography–tandem mass spectrometry (LC–MS)16 we determined the modification spectrum of vRNAP-associated tRNAGln/Arg and found the common modifications m1A, m5C, Cm, m1G, Gm and pseudouridine (Fig. We then evaluated whether the modified nucleotides, positioned as predicted, were consistent with their chemical environment in the complete vRNAP model and the surrounding cryo-EM density. For nucleotide 57 in the tRNA elbow, m1A fit the observed cryo-EM density better than its unmodified counterpart (Fig. In the anticodon stem at C32, Cm aligned well with the density and occupied a hydrophobic pocket partially formed by Y144 of Rap94 (Fig. Interestingly, this modification was missing in tRNA extracted from complete vRNAP (Fig. However, it was present in tRNAs from both infected (Fig. 6d), although it was less prevalent in the latter. We corroborated these results by showing that γ-toxin endonuclease from Kluyveromyces lactis18 failed to cleave complete vRNAP-associated tRNAGln/Arg (Fig. Therefore, a selective mechanism ensures that only tRNAGln/Arg lacking the mcm5s2 modification at base U34 can drive the assembly of complete vRNAP. a, Density and model at tRNA base A57. b, Density and model at tRNA base C32. c, Density and model at tRNA base U34. The model of mcm5s2U at position 34 of the complete vRNAP model produces a clash or unfavorable interaction with surrounding residues from Rap94 and NPH-I (red arrow). The tRNAGln/Arg-mediated assembly of complete vRNAP ensures the integration of all early transcription and processing factors into one stable unit. We hypothesized that this mechanism evolved to equip virions with precisely those components needed to start transcription upon infection. To test this prediction, purified vaccinia virions were analyzed for their RNA and protein content using RNA sequencing (RNA-seq) and MS, respectively (Fig. Consistent with this, the proteomics analysis showed that all subunits of complete vRNAP were present in the virion with comparable stoichiometry (Fig. 8b, red dots), whereas intermediate and late transcription factors were either absent or greatly underrepresented (Fig. Notably, the ratio of tRNAGln/Arg to the vRNAP core subunit Rpo132 was strikingly similar between isolated complete vRNAP and virion core (Extended Data Fig. These findings strongly suggest that complete vRNAP is transferred en bloc into newly formed virions, ensuring that the viral progeny is fully equipped with the machinery needed for early transcription. This notion is further corroborated by the observation that repression of Rap94, which we identify as the initial tRNAGln/Arg-binding platform, results in production of virus particles devoid of vRNAP19. a, Quantitative RNA-seq of RNA extracted from vaccinia virions (left) and purified complete vRNAP (right). The total number of reads is indicated below. b, Proteomics of purified complete vRNAP compared to total lysate from isolated vaccinia virions. Subunits of complete vRNAP are shown in red; typical abundant proteins of the virion are shown in dark blue. The only detectable intermediate or late transcription factor is shown as a green dot. c, Specific incorporation of tRNAGln/Arg into MPXV virions. RNA was extracted from MPXV and vaccinia virions and analyzed by northern blot analysis using probes against tRNAGln(UUG), tRNAArg(CCG) and tRNASer(CGA). Lanes 1–3 correspond to the amount of tRNA extracted from 2.1 × 106, 4 × 106 and 8 × 106 plaque-forming units of MPXV, respectively. Lanes 4 and 5 correspond to tRNA extracted from 5 × 105 and 1 × 106 plaque-forming units of vaccinia virus, respectively. Lane 6 corresponds to 100 ng of total calf tRNA. The experiment was performed in triplicate. Interestingly, its vRNAP subunits are almost identical to their vaccinia counterparts and would allow a similar tRNA-binding modus to that described for vaccinia complete vRNAP. In accordance with this, we observed a massive enrichment of tRNAGln/Arg in purified MPXV virions (Fig. This suggests that the formation of a tRNAGln/Arg-induced early transcription unit and its subsequent packaging into virions represent a conserved mechanism among various poxviruses. In this study, we dissected the entire assembly pathway of complete vRNAP through a combination of biochemical and structural (cryo-EM) approaches. Our findings identify tRNAGln/Arg as assembly chaperones for a macromolecular transcription and processing complex and, thus, add a novel aspect to the functional repertoire of tRNAs beyond their well-established role in translation. Assembly chaperones are a heterogeneous group of factors that join components with otherwise little or no affinity to each other and typically undergo conformational changes during their action21. They have, however, no further role with respect to the function of the assembled unit and often dissociate from the assembled particle. Traditionally, assembly chaperones have been attributed solely to proteins; examples include RbcX and Raf1 in the assembly of the multimeric Rubisco enzyme22,23 and pICln in the biogenesis of spliceosomal Sm class U snRNPs24,25. To the best of our knowledge, tRNAGln/Arg represents the first nonprotein of this group. We showed how an induced fit mechanism within the ASL facilitates the faithful assembly of a 16-component system. This generates an autonomous transcription unit, which is essential for the replication of poxviruses. Of note, the individual adjunct factors and vRNAP have little if any affinity to each other but essentially require the tRNA for joining. The emerging complete vRNAP complex is robust both in vivo and in vitro but will spontaneously convert to the PIC and initiate transcription when a cognate promoter is present. Remarkably, tRNAGln/Arg establishes a major checkpoint in this transition, impressing in the structure of complete vRNAP as a mechanical (Fig. 4c) barrier for the transcription-factor-bound promoter DNA. Thus, tRNAGln/Arg fulfills all key criteria of an assembly chaperone by facilitating the formation of an otherwise instable active transcription complex without directly participating in its catalytical activity. tRNAGln/Arg-mediated assembly of complete vRNAP may, hence, represent a widespread strategy among poxviruses for incorporating a transiently stalled form of the full early transcription machinery into the virion, thereby providing a convenient mechanism for an immediate initiation of transcription upon infection. It remains to be seen whether RNA chaperones have also evolved in the context of other systems. Our data show that the modification status of tRNAGln/Arg affects its role in assembly, with m1A at position 57 and Cm at position 32 being stimulatory. Despite a global upregulation of the mcm5s2U modification in total tRNAs upon infection, tRNAGln(UUG) carrying this modification in the anticodon would be incompatible with its function in assembly (Fig. An intriguing question is how tRNAGln/Arg displaying a specific modification pattern (that is, absence of mcm5s2U34 and presence of m1A and Cm) is diverted from the host translation machinery into the vRNAP assembly pathway. In uninfected cells, tRNAs are predominantly associated with cellular factors such as their cognate aminoacyl-tRNA synthetases and elongation factors (EFs). In particular, the extremely abundant translation factor EF1α would be expected to be an efficient binding competitor of vRNAP for charged tRNAGln/Arg. However, tRNAGln/Arg associated with vRNAP was shown to be uncharged and is, thus, presumably disengaged from active translation and interactions with the aforementioned cellular factors8. This raises the question of how tRNAGln/Arg can be ‘stolen' from the host for its activity in complete vRNAP assembly. As fully modified tRNAs are efficiently aminoacylated and likely sequestered by EF1α tRNAGln and tRNAArg lacking mcm5s2U may be less aminoacylated, as these hypomodified variants might avoid competition with EF1α and, thus, be preferentially used for vRNAP assembly. This might explain the observation that complete vRNAP contains exclusively uncharged tRNAGln/Arg. An alternative but not mutually exclusive scenario is that a local depletion of glutamine (and arginine) at viral replication sites may drive a shift from charged to uncharged tRNAs. Notably, such a situation may occur in the late phase of infection when complete vRNAP is formed for packaging and may provide a mechanism to link the metabolic status of the infected cell to the viral replication processes. In this regard, it is also noteworthy that glutamine is a crucial metabolite in viral energy metabolism28,29. Complete and core vRNAP complexes were purified from Hela S3 cells infected with GLV-1h439 virus as described previously30. To obtain highly pure complete and core vRNAP samples, FLAG-purified total vRNAP fractions obtained from GLV-1h439-infected HeLa cells were concentrated and separated by a 10–30% sucrose gradient centrifugation step at 164,199g, 16 h and 4 °C. Core vRNAP sedimented under these conditions in fractions 10–13, while complete vRNAP sedimented in fractions 14–16 (Fig. The respective fractions were analyzed by denaturing SDS–PAGE, concentrated and used for biochemical and structural studies. Adenosine triphosphate, guanosine triphosphate, cytidine triphosphate and DTT were added to a final concentration of 10 mM each and the sample was incubated for 15 min at 25 °C. Then, 4 nmol of 70-bp G8R promoter DNA scaffold spanning positions −35 to +35 containing a G-less cassette starting at +8 was added and the sample was adjusted to 50 mM HEPES pH 7.5, 150 mM NaCl and 1.5 mM MgCl2. After 30 min of incubation at 30 °C, the sample was applied on a 5–45% sucrose gradient and ultracentrifuged in a SW60Ti rotor at1 64,199g for 16 h at 4 °C. The gradient was isolated in 200-µl fractions; fractions 10–14 were pooled and concentrated in a Vivaspin 500 (100-kDa molecular weight cutoff (MWCO); Satorius, VS0141) to 2.4 µg µl−1. Isolated tRNAGln/Arg was 3′-end-labeled with [32P]pCp according to the manufacturer's protocol (Thermo Fischer Scientific, EL0021). Labeled RNA was separated by denaturing RNA gel electrophoresis (10%, 19:1, 8 M Urea, TBE), eluted from the gel in TBE and precipitated with ethanol. The pellet was washed with 70% ethanol and resuspended in resuspension buffer (10 mM Tris-HCl pH 8.0 and 1 mM EDTA). For the vRNAP reconstitution assays shown in Fig. 1b, 40 pmol of minimal, core or complete vRNAP was incubated with 80 pmol of [32P]tRNA at 30 °C for 30 min. After incubation mixtures were separated by density gradient centrifugation (10–30% sucrose gradient, 164,199g for 16 h, at 4 °C). For reconstitution of higher-order assembled intermediates (Fig. 1c), 1 pmol of [32P]pCp-labeled tRNAGln/Arg isolated from complete vRNAP was incubated with 2.4 pmol of core vRNAP in the presence of recombinant factors (D1/D12, E11 and NPH-I). Native electrophoresis was performed at 4 °C and complexes were visualized by autoradiography. Reconstituted I3 was purified by anti-FLAG affinity chromatography. For cryo-EM analysis, FLAG peptide was removed by sample concentration using Vivaspin (Satorius, VS0101) with a 10-kDa MWCO. To demonstrate that individual protein factors have no affinity to core vRNAP, recombinant NPH-I and D1/D12 or E11 were incubated with core vRNAP (molar ratio of 10:10:100:1) for 30 min at 25 °C. The mixture was subsequently separated by a 10–30% density gradient by centrifugation step (164,199g for 16 h and 4 °C). Sucrose fractions were collected manually and separated by 12% Bis–Tris gel electrophoresis30; proteins were visualized by silver staining. The interactions among recombinant factors (E11, NPH-I and D1/D12) were investigated by coimmunoprecipitation. Then, 10 μl of Dynabeads protein G was incubated with an affinity-purified monospecific anti-NPH-I antibody for 1 h at 25 °C. For control experiments, protein G beads were incubated with PBS buffer only. The beads were subsequently washed three times with PBS pH 7.5 and mixed with recombinant factors for 30 min at 25 °C in buffer containing 150 mM NaCl, 50 mM HEPES pH 7.5, 1.5 mM MgCl2 and 1 mM DTT. tRNAGln/Arg was obtained from affinity-purified complete vRNAP by TRIzol extraction (Invitrogen, 15596026). For isolation of total tRNA, Hela S3 cells were either mock-infected or infected with GLV-1h439. Cells were harvested and TRIzol was directly added to cell pellets for lysis. The aqueous phase was precipitated with ethanol and resuspended in TBE; total RNA was separated on denaturing RNA gels (8%, 19:1 acrylamide:bisacrylamide, 8 M urea and TBE). tRNAs were visualized by ethidium bromide and eluted from the gel in TBE buffer. Eluted tRNAs were precipitated with ethanol and resuspended in water or resuspension buffer (10 mM Tris-HCl pH 8.0 and 1 mM EDTA). Total calf tRNA was obtained from (Roche, 83298320-63). Wild-type tRNAGln and a variant mutated at position U33G were synthetized by T7 transcription with a high-yield T7 transcription kit (Thermo Fischer Scientific, K0441) according to the manufacturer's protocol. For the recombinant expression of vaccinia virus capping enzyme D1/D12 and early transcription termination factor NPH-I, the respective open reading frames were amplified from vaccinia virus genomic DNA and cloned into pET-Duet1 or pETM-11 vectors to generate pET-Duet1–D1-D12 and pETM-11-NPH-I, respectively. The resulting plasmids were verified by sequencing and used for protein expression experiments. A plasmid allowing for the expression of GST–γ-toxin was kindly provided by R. Schaffrath and D. Scherf31. Expression and purification of E11 was performed as described previously8. For purification of the His–D1/D12 heterodimer and His–NPH-I, pETDuet-D1R-D12L or pETM-11-NPH-I was transformed into E. coli Iq cells (New England Biolabs, C3037l). E. coli was grown in SB medium containing ampicillin and chloramphenicol for PETDuet-D1R-D12 or kanamycin and chloramphenicol in a case of pETM-11-NPH-I. At an optical density of 0.8, protein expression was induced by adding 0.5 mM IPTG for 16 h at 16 °C. The cleared lysate was mixed with 1 ml of Ni-NTA beads (His60 Ni Superflow Resin; Takara, 635660) and incubated at 4 °C for 3 h. The beads were washed three times with washing buffer (150 mM NaCl, 50 mM HEPES pH 8.0, 25 mM imidazole and 2 mM β-mercaptoethanol) and proteins were eluted in washing buffer containing 250 mM imidazole. The eluates were applied to a Superdex 200 (HiLoad 26/600 PG) column and analyzed by SDS–PAGE. Pure fractions eluting at the expected volume were collected, concentrated in Vivaspin 6 (10-kDa MWCO; Satorius, VS0101) to 3 µg µl−1 for NPH-I and 1.5 µg µl−1 for D1R/D12L and used for biochemical reconstitution assays (Extended Data Fig. GST–γ-toxin was expressed in the E. coli BL21(DE3) pRARE strain. Expression was induced with 0.5 mM IPTG (final concentration) for 16 h at 18 °C. After sonication, the lysate was cleared and the supernatant was incubated with 10 ml of glutathione Sepharose beads (Cytiva, 17513202). The column was washed three times with lysis buffer omitting lysozyme and the protein was eluted in the same buffer containing 20 mM glutathione. The elution was applied onto a Superdex 200 column (HiLoad 26/600 PG). GST–γ-toxin eluted as a single peak and was concentrated in Vivaspin 6 (10-kDa MWCO; Sartorious, VS0101) to 2.5 µg µl−1 before it was used for RNA cleavage assays. To investigate the mcm5s2U34 modification of tRNA associated with complete vRNAP, a cleavage assay using γ-toxin was performed as described previously8. vRNAP-associated tRNA purified by TRIzol extraction and total calf tRNA were 3′-end-labeled with [32P]pCp and purified by denaturing gel electrophoresis. Then, 100 ng of isolated tRNAs were incubated with buffer (20 mM HEPES pH 7.5, 150 mM NaCl, 2 mM EDTA and 2 mM DTT) or increasing amounts of γ-toxin in a total volume of 15 µl at 30 °C for 30 min. tRNA was subsequently TRIzol-extracted, separated by denaturing RNA gel electrophoresis and visualized by autoradiography. For analysis of tRNAs from Hela S3 cells infected with vaccinia virus, total tRNAs were purified as described above. Then, 100 ng of total tRNAs collected from infected cells were digested with an increasing amount of γ-toxin, followed by TRIzol extraction. Full-length tRNAs and digested fragments were separated on denaturing RNA gels and transferred to a nylon membrane (transfer at 72 mA for 1 h). The membrane was hybridized with a 32P-labeled DNA probe (5′-AGGTCCCACCGAGATTTGAACTCG-3′) complementary to the 3′-end of tRNAGln(UUG) at 42 °C for 12 h. The membrane was washed three times with NB washing buffer (2× SSC and 0.01% SDS) and bands were visualized by autoradiography. For identification of tRNAGln/Arg in MPox and vaccinia virus, RNA was purified by TRIzol extractions from virions, precipitated and resuspended in RNAase-free water. For northern blot analysis, RNA was separated in a 12% denaturing RNA gel and transferred to a nylon membrane (GE Healthcare, RPN203B) at 72 mA for 1 h. After transfer, the membrane was hybridized at 42 °C with 5′-32P-labeled DNA oligonucleotides (5′-ACTCGGATCGCTGGATTCAAAGTCCAGAGTGCTAACCA-3′, 5′-ACCCTCAATCTTCTGATCCGGAATCAGACGCCTTATCCA-3′ or 5′-CGGGGAGACCCCATTGGATTTCGAGTCCAACGCCTTAACCACT-3′) complementary to the anticodon regions of tRNAGln(UUG), tRNAArg(CCG) and tRNASer (CGA), respectively. The membrane was washed three times with a washing buffer and visualized by autoradiography as described above. The sucrose-gradient-purified vRNAP samples prepared as described above were diluted 1:50 and concentrated in a Vivaspin concentrator to a concentration of roughly 1 mg ml−1 to remove the sucrose, centrifuged for 2 h at 21,000g and diluted 1:1 in a buffer containing 20 mM HEPES pH 7.5, 200 mM (NH4)2SO4, 1 mM MgCl2 and 5 mM β-mercaptoethanol. Next, R 1.2/1.3 Quantifoil holey carbon grids (Jena Bioscience) were glow discharged for 90 s in a plasma cleaner (PDC-002, Harrick Plasma) at medium power and 3.5 ml of C2 sample was applied inside a Vitrobot Mark IV (Thermo Fisher Scientific) at 4 °C and 100% relative humidity. The grids were blotted for 3 s with a blot force of 5 and plunged into liquid ethane. The cryo-EM datasets were collected with Titan Krios G3 equipped with a Falcon III or Falcon IV camera (Thermo Fischer Scientific). Data were acquired with EPU at 300 keV. Several cycles of two-dimensional classification and manual selection of classes based on the appearance of their class averages were applied for initial cleanup. Ab initio maps were created from a subset of 50,000 particles and the full set was subjected to a consensus three-dimensional refinement. The datasets were further classified as detailed in Extended Data Figs. The classified particle sets were finally subjected to a nonuniform refinement step33 and the density was docked with initial models derived manually from the previously published complete vRNAP model (PDB 6RFL). In a first round, each model was manually refined including removal of some stretches of Rap94 that were not represented in the cryo-EM density. The models were then subjected to alternating rounds of automatic refinement with phenix.real_space_refine34 including an atomic displacement parameter refinement step and manual building with Coot35. During automated refinement, secondary-structure and mild Ramachandran restraints were imposed. A total of three cycles of manual inspection and automated refinement were performed for each model. In-solution digestion: For the analysis of vRNAP and virion samples by MS, proteins were precipitated with 80% acetone at −20 °C for 2 h. Proteins were resolubilized in 8 M urea and 50 mM ammonium bicarbonate to reach a concentration of 0.5 µg µl−1 followed by reduction with 5 mM Tris(2-carboxyethyl)phosphine dissolved in 10 mM ammonium bicarbonate (30 min at 56 °C) and alkylation of free thiol groups with 100 mM 2-chloracetamide and 10 mM ammonium bicarbonate (30 min at 37 °C). Urea concentration was adjusted to 2 M by adding 50 mM ammonium bicarbonate, trypsin was added at a protease-to-protein ratio of 1:50 and samples were incubated for 16 h at 37 °C and 200 rpm. Protein digestion was stopped by acidification with trifluoroacetic acid (TFA) added at a final concentration of 1% (v/v). Peptides were dried in vacuo and stored at −80 °C until used for LC–MS analysis. LC–MS analysis: Dried peptides were reconstituted in 0.1% (v/v) TFA and analyzed by reverse-phase LC–MS using an UltiMate 3000 RSLCnano system (Thermo Fisher Scientific) coupled online to a Q Exactive Plus instrument (Thermo Fisher Scientific). Peptide separation and elution were performed at 40 °C using a binary solvent system composed of 0.1% (v/v) formic acid (FA) (solvent A) and 86% (v/v) acetonitrile in 0.1% (v/v) FA (solvent B). Peptide mixtures from RPO132 pulldown experiments were analyzed by LC–MS using a 3-h method. Peptides were loaded at 1% solvent B for 3 min at a flow rate of 10 µl min−1 and eluted by applying the following gradient: 5–22% B in 96 min, 22–42% B in 54 min, 4 min at 80% B and re-equilibration of the column for 19 min at 100% A. The flow rate for peptide elution was set to 0.3 µl min−1. The MS instrument, equipped with a nanoelectrospray ion source and a stainless-steel emitter (Thermo Fischer Scientific), was externally calibrated using standard compounds. Parameters for MS measurements in data-dependent acquisition mode were as follows: MS full scan window of m/z 375–1,700, resolution of 70,000 (at m/z 200), automatic gain control (AGC) of 3 × 106 and maximum injection time (IT) of 60 ms. Multiply charged peptide ions were fragmented by higher-energy collisional dissociation applying a normalized collision energy of 28% and a dynamic exclusion time of 45 s. A TOP12 method was applied to analyze peptides with an MS2 resolution of 35,000, an AGC of 7 × 102 and a maximum IT of 120 ms. MS data analysis: MS raw data were processed using MaxQuant/Andromeda (version 2.0.2.0)36 and the vaccinia virus (strain Western Reserve) reference proteome from UniProt (UP000000344; downloaded January 2022, containing 218 protein entries). Proteins were identified using MaxQuant default settings. Oxidation of methionine and N-terminal acetylation were considered as variable modifications, while carbamidomethylation of cysteine residues was set as the fixed modification. The option ‘match between runs' was enabled. Bioinformatic analysis: For the identification of proteins enriched in vaccinia virus virions, data were first normalized as follows. For each replicate, the summed MS intensity of all protein groups was shifted to the mean of the summed MS intensities determined across all three replicates. Subsequently, intensities were transformed by variance stabilizing data transformation37. Protein abundance ratios were determined and the linear models for microarray data (limma) approach, as implemented in the R package limma (version 3.60.4)38,39, was applied to calculate P values for the enrichment of proteins. Extracted tRNAGln/Arg from the vRNAP complex was purified with an RNA clean and concentrator-5 (Zymo Research, lot 213186). Then, 400 pmol of tRNA was digested by 6.0 U of bacterial alkaline phosphatase and 1.0 U of snake venom phosphodiesterase in reaction buffer (40 mM Tris-HCl and 20 mM MgCl2, pH 7.5). After extracting the digested nucleosides mixture with 100 μl of chloroform, the aqueous layer was concentrated by lyophilization and the residue was dissolved in 70 μl of 10 mM ammonium acetate. The analysis was run with a gradient of 0–5% (0–15 min) and 5–72.5% (15–45 min) of solvent B, using an Synergi Fusion RP column (Phenomenex; 4 μm, 250 × 2 mm). Solvent A was 10 mM ammonium acetate (pH 5.3), solvent B was acetonitrile and the flow rate was 0.2 ml min−1 at 25 °C with ultraviolet detection at 260 nm and online MS in a microTOF-Q III system in positive ion mode. The same experiment was also performed for in vitro transcription using tRNAGln as control and for total tRNA extracted from infected and noninfected cells. Libraries were generated from the isolated RNA fraction following the Ion Torrent ion total RNA-seq kit v2 (Thermo Fisher Scientific, 4475936) protocol with some modifications. Briefly, 30 ng of the RNA was incubated with 5 U of RNase T1 (Thermo Fisher Scientific, EN0541) at 20 °C for 35 min. The samples were then treated with 5 U of Antarctic phosphatase (New England Biolabs, M0289) for 30 min at 37 °C. After heat inactivation at 65 °C, the RNA was phosphorylated with 20 U of T4 polynucleotide kinase (New England Biolabs, M0201) for 60 min at 37 °C. Adaptor ligation was performed for 16 h at 16 °C. Reverse transcription (RT) was performed using SuperScript III with incubations at 42 °C, 50 °C and 55 °C for 45, 15 and 10 min, respectively. The RT reactions were purified and the complementary DNA was amplified by Platinum PCR SuperMix high fidelity. The resulting libraries were sequenced using an Ion Proton (Ion Torrent TM) with Hi-Q. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. PDB validation reports were deposited to figshare or are freely available from the PDB. Cryo-EM maps of complete vRNAP, minimal vRNAP and all assembly intermediates newly described in this work were deposited to the EM Data Bank. All other data not publicly available at the time of the review of the manuscript were deposited to figshare for review purposes. Source data are provided with this paper. Ramakrishnan, V. Ribosome structure and the mechanism of translation. Gupta, R. et al. A tRNA modification balances carbon and nitrogen metabolism by regulating phosphate homeostasis. Di Segni, G., Gastaldi, S. & Tocchini-Valentini, G. P. cis- and trans-splicing of mRNAs mediated by tRNA sequences in eukaryotic cells. & Henkin, T. M. The T box mechanism: tRNA as a regulatory molecule. Mei, Y. et al. tRNA binds to cytochrome c and inhibits caspase activation. Liu, Y.-T., Strugatsky, D., Liu, W. & Zhou, Z. H. Structure of human cytomegalovirus virion reveals host tRNA binding to capsid-associated tegument protein pp150. Su, Z., Wilson, B., Kumar, P. & Dutta, A. Noncanonical roles of tRNAs: tRNA fragments and beyond. Structural basis of poxvirus transcription: vaccinia RNA polymerase complexes. Entry and disassembly of large DNA viruses: electron microscopy leads the way. Structural basis of poxvirus transcription: transcribing and capping vaccinia complexes. Structural basis of the complete poxvirus transcription initiation process. Structure of E. coli glutaminyl-tRNA synthetase complexed with tRNAGln and ATP at 2.8 Å resolution. Pan, T. Modifications and functional genomics of human transfer RNA. Suzuki, T. The expanding world of tRNA modifications and their disease relevance. Thüring, K., Schmid, K., Keller, P. & Helm, M. Analysis of RNA modifications by liquid chromatography–tandem mass spectrometry. Shigi, N. Biosynthesis and functions of sulfur modifications in tRNA. & Byström, A. S. Kluyveromyces lactis γ-toxin, a ribonuclease that recognizes the anticodon stem loop of tRNA. Targeting of a multicomponent transcription apparatus into assembling vaccinia virus particles requires RAP94, an RNA polymerase-associated protein. Evolutionary trajectory and characteristics of Mpox virus in 2023 based on a large-scale genomic surveillance in Shenzhen, China. Structure and mechanism of the RuBisCO-assembly chaperone Raf1. Structural basis of assembly chaperone-mediated snRNP formation. Structure of the large ribosomal subunit from human mitochondria. Karagoz, A. et al. Monkeypox (Mpox) virus: classification, origin, transmission, genome organization, antiviral drugs, and molecular diagnosis. Cellular metabolism hijacked by viruses for immunoevasion: potential antiviral targets. Host cell glutamine metabolism as a potential antiviral target. Bartuli, J., Lorenzi, I., Backes, S., Grimm, C. & Fischer, U. A generic protocol for the affinity-purification of native macromolecular complexes from poxvirus-infected cells. Jaciuk, M. et al. Cryo-EM structure of the fully assembled Elongator complex. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Punjani, A., Zhang, H. & Fleet, D. J. Non-uniform refinement: adaptive regularization improves single-particle cryo-EM reconstruction. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Cox, J. et al. Andromeda: a peptide search engine integrated into the MaxQuant environment. Huber, W., von Heydebreck, A., Sültmann, H., Poustka, A. & Vingron, M. Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. This work was supported by German Research Foundation grants to U.F. (grant 9B813) and by Austrian Science Fund grants to M.E. Cryo-EM was carried out in the cryo-EM facility of the Julius Maximilian University of Würzburg funded by the German Research Foundation (projects INST 93/903-1 359471283, INST 93/1042-1 456578072 and INST 93/1143-1 525040890). Open access funding provided by Julius-Maximilians-Universität Würzburg. Present address: National Heart Lung and Blood Institute, National Institutes of Health, Bethesda, MD, USA These authors contributed equally: Julia Bartuli, Clemens Grimm. Julia Bartuli, Stefan Jungwirth, Manisha Dixit, Clemens Grimm & Utz Fischer Helmholtz Institute for RNA-based Infection Research (HIRI), Helmholtz Centre for Infection Research (HZI), Würzburg, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Data processing, model building, structure analysis and visualization, C.G. Correspondence to Clemens Grimm or Utz Fischer. The authors declare no competing interests. Nature Structural & Molecular Biology thanks Tsutomu Suzuki and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Sara Osman, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) SDS-gel of affinity purified adjunct factors D1/D12, NPH-I and E11 obtained by E. coli overexpression and used for reconstitution assays. The arrow indicates an NPH-I degradation product. The experiment was performed in duplicates. (b) Minimal binding of E11, D1/D12 and NPH-I to each other. Recombinant NPH-I and E11 (lanes 1-4) or NPH-I, D1/D12 and E11 (lanes 5-8) were incubated with each other and immunoprecipitated by antisera against NPH-I or a non-immune serum. No co-immunoprecipitation was observed for NPH-I to E11, and only marginal binding of D1/D12 to NPH-I/E11 was detectable. The arrow indicates an NPH-I degradation product. (c) Sucrose density gradient of vRNAP affinity-purified from infected cells (upper panel). Fractions 11-13 contain core vRNAP that was used for reconstitution assays, fractions 15-18 contain complete vRNAP. Gradient centrifugation of core vRNAP (fractions 11-13) pre -incubated with a mixture of either recombinant NPH-I and E11 or D1/D12. The arrow indicates an NPH-I degradation product. The experiment was performed in duplicates. (c) Local resolution mapped to the reconstruction density isosurface. (d) FSC plots for final reconstruction. (e) Orientation distribution plot for final reconstruction. (f) Posterior precision distribution plot for final reconstruction. The Rap94 structure as extracted from complete vRNAP is shown in surface representation. The five domains (NTD, Domain 2, B Ribbon, B-cyclin domain and CTD) are coloured in shades of green. Domains are colored according to the scheme used in Fig. (d) FSC plots for final reconstruction of intermediate I3. (e) Orientation distribution plot for final reconstruction of intermediate I3. (f) Posterior precision distribution plot for final reconstruction of intermediate I3. (h) FSC plots for final reconstruction of intermediate I4. (i) Orientation distribution plot for final reconstruction of intermediate I4. (j) Posterior precision distribution plot for final reconstruction of intermediate I4. (c) Local resolution mapped to the reconstruction density isosurface. (d) FSC plots for final reconstruction. (e) Orientation distribution plot for final reconstruction. (f) Posterior precision distribution plot for final reconstruction. (c) FSC plots for final reconstruction. (d) Orientation distribution plot for final reconstruction. (e) Posterior precision distribution plot for final reconstruction. Comparison of the tRNAGln-UUG model with its associated cryo EM density as extracted from the complete vRNAP structure (left) to a model for the tRNAArg-CCG sequence, modelled manually into the complete vRNAP density (right). Core vRNAP and the indicated recombinant transcription factors were incubated with [³²P]-labelled tRNAGln/Arg (isolated from complete vRNAP), in vitro-transcribed tRNAGln harbouring a U33G mutation, and in vitro-transcribed wild-type tRNAGln (WT). Samples were analysed by native gel electrophoresis and complexes were visualized by radioautography. Black circles indicate components that were added, while white circles indicate components that were omitted. The experiment was performed three independent times with similar results. Anti-HA Western blot of isolated complete vRNAP and virions (above left). Northern blot of tRNAGln(UUG) isolated from complete vRNAP and virions (below left). Correlation of the respective Western blot (WB) signals to Northern blot (NB) signals. The experiment was performed in triplicates. Supplementary Table 1: Alignment of relevant tRNAGln isodecoders and their isoforms. ASL remodeling during recruitment of tRNA to vRNAP. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Bartuli, J., Jungwirth, S., Dixit, M. et al. tRNA as an assembly chaperone for a macromolecular transcription-processing complex. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            