
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64514529/maya-kingdom-collapse-burning-event/'>Archaeologists Found a Smoking Gun Behind the End of the Maya Kingdom's Reign</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 17:33:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Archaeologists discovered clues to a fire in Guatemala from between 733 and 881 AD that they say represents a key turning point in Maya rule—a very public turning point. “Rather than examine this fire-burning event as a bookend to Maya history, we view it as a pivot point around which the K'anwitznal polity reinvented itself and the city of Ucanal went on to a flourishing of activities.” The new leadership regime welcomed a non-royal leader called Papmalil, and there is little in the written record indicating how he came to power. The study's authors, led by Christina Halperin at the University of Montreal, state that Papmalil appears to have ushered in an era of prosperity. That new era may have had a dramatic beginning. Included with the bodies were 1,470 fragments of greenstone pendants, beads, plaques, and mosaics, along with large blades—all representing a “single burning event.” The quantity and quality of the burnt and broken ornaments indicate they came from a royal tomb, likely belonging to multiple individuals. The authors state that the event “appears to have bene an act of desecration: it was dumped at the edge of a crude wall used as a construction pen and no effort was made to protect the fragmented bones and ornaments from the tomb blocks deposited on top of them as construction fill.” It all likely made for a “dramatic public affair” meant to be charged with emotion. “It could dramatically mark,” they wrote, “the dismantling of an ancient regime.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Ancient Plants May Show the Site of Jesus's Tomb</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01198-6'>Invasion of the ‘journal snatchers': the firms that buy science publications and turn them rogue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 15:53:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Research-integrity analysts are warning that ‘journal snatchers' — companies that acquire scholarly journals from reputable publishers — are turning legitimate titles into predatory, low-quality publications with questionable practices. In an analysis published on the preprint repository Zenodo in January1, researchers identified three dozen journals that have been caught in this predicament after being bought by what they describe as a network of recently established international companies with no track record in the publishing industry. “We found at least 36 journals but we think that there may be more,” says study co-author Alberto Martín-Martín, an information scientist at the University of Granada in Spain. The titles were previously indexed by databases such as Scopus and Web of Science, and were owned by various institutions, including the Dutch publishing giant Elsevier; the academic publisher Palgrave Macmillan, based in London; Indiana University Northwest, in Gary, Indiana; and the University of São Paulo, in Brazil. According to emails evaluated by Martín-Martín and his co-author, Emilio Delgado López-Cózar, also an information scientist at Granada, publishers are being offered hundreds of thousands of euros in exchange for each journal. “For small journals, this is a very attractive offer,” Martín-Martín says. These practices are typically associated with predatory publishing, where questionable publishers cut corners to generate low-quality or fraudulent research papers in exchange for high publication fees. David Radhor, relationship manager at Oxbridge Publishing House, says the company is “not a publisher” and does not directly own all of the titles in Martín-Martín's preprint. (The study claims that Oxbridge has directors in common with some of the other companies, and suggests that it is part of a network of linked firms that are actively acquiring journals.) Those decisions are made independently by each journal's editorial board, he says. Nature also attempted to contact all 36 journals listed in the study. Amjid Iqbal, managing editor of the American Journal of Health Behavior (AJHB) in Los Angeles, California, says his journal increased their publication fees (from US$1,595 to £2,000 according to the study) because of inflation and because its vendors increased their charges. The remaining journals did not respond to requests for comment. In two other cases, researchers contacted by Nature said they had been falsely listed as editors on journal websites. “One of the problems of these companies is that when they buy journals, they are not very open about this, and in many cases, the journals don't give any information about this new owner,” Martín-Martín says. Hundreds of scientists have peer-reviewed for predatory journals These are the most-cited research papers of all time Richard Fortey obituary: palaeontologist, author and TV presenter who traced continents through fossils The Medical Faculty Mannheim of Heidelberg University offers the position of a   Full Professorship (W3) for Special Hematology (f/m/d)     to be f... Full Professorship (W3) for “Dermatology and Venereology” (f/m/d)     to be filled as of 01.10.2027. Job Title: Senior Publisher, Applied Science Journals  Location(s): Paris or London  Application Deadline: 1st May    About Springer Nature Group  ... Hundreds of scientists have peer-reviewed for predatory journals An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64491730/earthquake-ancient-structure/'>A 7.7-Magnitude Earthquake Split the Ground—And Brought an Ancient Structure to the Surface</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>According to experts, the building may have once been a royal “water palace.” A recent natural disaster may give researchers a rare look into remnants of the empire, or at least one remnant in particular. On March 28, a 7.7 magnitude earthquake revealed an ancient structure in Tada-U Township, Myanmar. But the recent earthquake caused a fissure that revealed more parts of the structure that were previously hidden beneath layers of soil. Researchers say that additional ruins, including a handrail, brick platforms, and an 18-inch riser step, are now visible. According to the report, some of these features resemble sketches from ancient palm-leaf manuscripts called “Pura-pike.” Historically, water has been a religious symbol in Myanmar culture, so water palaces served as important places for rituals, the Department of Archaeology and National Museum reports. Though the prospect of finding a royal water palace is exciting, it's much more likely the structure is something simpler. Some commenters are critical of researchers not unearthing the structure when it was initially discovered in 2009. Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. Ancient Plants May Show the Site of Jesus's Tomb Experts Found an Ancient Altar in the Wrong City</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01191-z'>25 million deaths: what could happen if the US ends global health funding</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 13:01:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. The United States spent roughly US$12 billion on global health in 2024. Without that yearly spending, roughly 25 million people could die in the next 15 years, according to models that have estimated the impact of such cuts on programmes for tuberculosis, HIV, family planning and maternal and child health. The United States has long been the largest donor for health initiatives in poor countries, accounting for almost one-quarter of all global health assistance from donors. These investments have contributed to consistent public-health gains for more than a decade. John Stover, an infectious-diseases modeller at Avenir Health, a global-health organization in Glastonbury, Connecticut, and his colleagues used mathematical models to estimate health outcomes, should all US funding for global health be cut and not replaced, compared with outcomes if funding provided in 2024 were to continue through to 2040. “Their findings are devastating to read” and “a wake-up call for all of us working in global health.” James Trauer, an infectious-disease modeller at Monash University in Melbourne, Australia, adds: “These models are probably as good as we have available at the moment for predicting the direct effects of the funding cuts on these various programmes.” More than 60% of those deaths would take place in six African countries, including Mozambique, Nigeria and Uganda. Roughly 14 million extra children would become orphans as a result of those AIDS deaths — a trend that had been expected to decrease over the next 15 years. And 26 million more people could become infected with HIV without PEPFAR. These estimates are broadly consistent with other efforts to assess the impact, says Trauer. Trump team guts AIDS-eradication programme and slashes HIV research grants Trump team guts AIDS-eradication programme and slashes HIV research grants Antibody prophylaxis may mask subclinical SIV infections in macaques Why a fortunate few don't get ill after HIV infection Faster, cheaper, better: the rise of blood tests for Alzheimer's Full Professorship (W3) for “Dermatology and Venereology” (f/m/d)     to be filled as of 01.10.2027. Job Title: Senior Publisher, Applied Science Journals  Location(s): Paris or London  Application Deadline: 1st May    About Springer Nature Group  ... Trump team guts AIDS-eradication programme and slashes HIV research grants An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64500206/archaeologists-found-317-skeletons-buried-under-a-department-store/'>Archaeologists Found 317 Skeletons Buried Under a Department Store</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The remains of 317 individuals from medieval and post-medieval burial grounds were discovered by archaeologists from Cotswold Archaeology in Kings Square, Gloucester, as the site was being redeveloped into the University of Gloucester's City Campus. “Every time we work in Gloucester, we make new discoveries,” Cliff Bateman, Cotswold Archaeology Senior Project Officer, said in a press release. Many earlier artifacts from the Roman period also surfaced, which makes sense, considering that what is now King's Square is thought to have once been the northeast quadrant of an ancient Roman town. Other recent finds include brick burial vaults and a crypt from St. Aldate's Church—the external wall and porch of which appeared when remodeling efforts started. The later St. Aldate's stood until 1960, but has long since been demolished. Evidence of its medieval predecessor has not yet appeared, but is thought to be in the area. The bones of twelve individuals were only exhumed for research before being reinterred in their original graves. Bateman also said that he is just about positive “there will be Roman buildings in situ” beneath the post-medieval necropolis. This isn't surprising to archaeologists, who first started finding mosaics and ruins of Roman buildings in the basement of the empty Debenham's. Preliminary studies on the teeth of skeletons have found that the people they came from likely consumed a diet high in sugar. Excavations in another part of the city previously revealed even more skeletons, this time from the Late Roman period, which were buried both on their backs and facedown. “These objects have been retained on site, following archaeological recording, and will be displayed on site for students, staff and visitors to City Campus to appreciate once the site is fully operational,” Steve Sheldon, Acting Principal Manager of Cotswold Archaeology, said in a more recent press release. There could be no more epic way to kick off an ancient history class. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. The Maya Kingdom Collapsed Due to Burning Events Ancient Plants May Show the Site of Jesus's Tomb Experts Found an Ancient Altar in the Wrong City</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64478332/dacian-silver/'>Metal Detectorists Found a Jackpot Stash of Ancient Silver Treasure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Experts aren't sure why it was buried in the first place. The discovery of a silver treasure hoard—this one unearthed by metal detectorists—has shed a sparkling light on just how richly adorned the Dacian elite once were. What is now modern-day Romania was once ruled by a people group known as the Dacians, who were prominent from around 500 B.C. This spring (according to a translated statement from the Museum of Mures County posted by the Breaza Mures Municipality City Hall), metal detectorists Moldovan Dionisie-Aurel and Zahan Sebastian-Adrian scoured an area around the town of Breaza in central Romania, and uncovered a cache of six silver ornamental pieces that date to the Dacian people. This is the first find of Dacian treasure in Breaza, and it doesn't disappoint. He added that the items would have been worn by a prominent member of the Dacian aristocracy on various special occasions, but it's “difficult to say whether it was a man or a woman.” Either way, they would have shined in silver. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Ancient Plants May Show the Site of Jesus's Tomb Experts Found an Ancient Altar in the Wrong City</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02676-5'>A highly stable monomeric red fluorescent protein for advanced microscopy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 09:48:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The stability of fluorescent proteins (FPs) is crucial for imaging techniques such as live-cell imaging, super-resolution microscopy and correlative light and electron microscopy. Although stable green and yellow FPs are available, stable monomeric red FPs (RFPs) remain limited. Here we develop an extremely stable monomeric RFP named mScarlet3-H and determine its structure at a 1.5 Å resolution. mScarlet3-H exhibits remarkable resistance to high temperature, chaotropic conditions and oxidative environments, enabling efficient correlative light and electron microscopy imaging and rapid (less than 1 day) whole-organ tissue clearing. In addition, its high photostability allows long-term three-dimensional structured illumination microscopy imaging of mitochondrial dynamics with minimal photobleaching. It also facilitates dual-color live-cell stimulated emission depletion imaging with a high signal-to-noise ratio and strong specificity. Systematic benchmarking against high-performing RFPs established mScarlet3-H as a highly stable RFP for multimodality microscopy in cell cultures and model organisms, complementing green FPs for multiplexed imaging in zebrafish, mice and Nicotiana benthamiana. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The coordinates and structure factors for mScarlet-H and mScarlet3-H have been deposited in the Protein Data Bank with accession numbers 8ZXO and 8ZXH, respectively. The most essential raw datasets, including source files for supplementary figures and raw unprocessed images, are available on figshare at https://doi.org/10.6084/m9.figshare.28398170.v1 (ref. The remaining files are available from the corresponding author upon request. All plasmids used in this study are available on WeKwikGene at https://wekwikgene.wllsb.edu.cn/. Source data are provided with this paper. Watanabe, S. et al. Protein localization in electron micrographs using fluorescence nanoscopy. Fu, Z. et al. mEosEM withstands osmium staining and Epon embedding for super-resolution CLEM. Campbell, B. C., Paez-Segala, M. G., Looger, L. L., Petsko, G. A. & Liu, C. F. Chemically stable fluorescent proteins for advanced microscopy. Tanida, I., Kakuta, S., Trejo, J. & Uchiyama, Y. Visualization of cytoplasmic organelles via in-resin CLEM using an osmium-resistant far-red protein. Tanida, I. et al. Two-color in-resin CLEM of Epon-embedded cells using osmium resistant green and red fluorescent proteins. Peng, D. et al. Improved fluorescent proteins for dual-colour post-embedding CLEM. Tainaka, K., Kuno, A., Kubota, S. I., Murakami, T. & Ueda, H. R. Chemical principles in tissue clearing and staining protocols for whole-body cell profiling. Hirano, M. et al. A highly photostable and bright green fluorescent protein. Bright and stable monomeric green fluorescent protein derived from StayGold. Ando, R. et al. StayGold variants for molecular fusion and membrane-targeting applications. Ivorra-Molla, E. et al. A monomeric StayGold fluorescent protein. Shcherbakova, D. M., Subach, O. M. & Verkhusha, V. V. Red fluorescent proteins: advanced imaging applications and future design. Bindels, D. S. et al. mScarlet: a bright monomeric red fluorescent protein for cellular imaging. Gadella, T. W. J. et al. mScarlet3: a brilliant and fast-maturing red fluorescent protein. Ai, H., Olenych, S. G., Wong, P., Davidson, M. W. & Campbell, R. E. Hue-shifted monomeric variants of Clavulariacyan fluorescent protein: identification of the molecular determinants of color and applications in fluorescence imaging. Cranfill, P. J. et al. Quantitative assessment of fluorescent proteins. Qiao, C. et al. Rationalized deep learning super-resolution microscopy for sustained live imaging of rapid subcellular processes. Fenno, L. E. et al. Comprehensive dual- and triple-feature intersectional single-vector delivery of diverse functional payloads to cells of behaving mammals. Paez-Segala, M. G. et al. Fixation-resistant photoactivatable fluorescent proteins for CLEM. Improving the photostability of bright monomeric orange and red fluorescent proteins. Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. red fluorescent protein. Shen, Y., Chen, Y., Wu, J., Shaner, N. C. & Campbell, R. E. Engineering of mCherry variants with long Stokes shift, red-shifted fluorescence, and low cytotoxicity. Chu, J. et al. Non-invasive intravital imaging of cellular differentiation with a bright red-excitable fluorescent protein. Paul-Gilloteaux, P. et al. eC-CLEM: flexible multidimensional registration software for correlative microscopies. Yi, Y. et al. Mapping of individual sensory nerve axons from digits to spinal cord with the transparent embedding solvent system. Tian, T., Yang, Z. & Li, X. Tissue clearing technique: recent progress and biomedical applications. Hell, S. W. & Wichmann, J. Breaking the diffraction resolution limit by stimulated emission: stimulated-emission-depletion fluorescence microscopy. Hense, A. et al. Monomeric Garnet, a far-red fluorescent protein for live-cell STED imaging. Wegner, W. et al. In vivo mouse and live cell STED microscopy of neuronal actin plasticity using far-red emitting fluorescent proteins. Matela, G. et al. A far-red emitting fluorescent marker protein, mGarnet2, for microscopy and STED nanoscopy. Verma, V. & Aggarwal, R. K. A comparative analysis of similarity measures akin to the Jaccard index in collaborative recommendations: empirical and theoretical perspective. Dynamic nanoscale morphology of the ER surveyed by STED microscopy. Glogger, M. et al. Synergizing exchangeable fluorophore labels for multitarget STED microscopy. Dynamic constriction and fission of endoplasmic reticulum membranes by reticulon. Lin, C., White, R. R., Sparkes, I. & Ashwin, P. Modeling endoplasmic reticulum network maintenance in a plant cell. Ren, W. et al. Visualization of cristae and mtDNA interactions via STED nanoscopy using a low saturation power probe. & Voeltz, G. K. Endoplasmic reticulum–mitochondria contacts: function of the junction. Ai, H., Henderson, J. N., Remington, S. J. & Campbell, R. E. Directed evolution of a monomeric, bright and photostable version of Clavularia cyan fluorescent protein: structural characterization and applications in fluorescence imaging. & Noirclerc-Savoye, M. Stabilizing role of glutamic acid 222 in the structure of enhanced green fluorescent protein. Scott, D. J. et al. A novel ultra-stable, monomeric green fluorescent protein for direct volumetric imaging of whole organs using CLARITY. Thermal green protein, an extremely stable, nonaggregating fluorescent protein created by structure‐guided surface engineering. Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. & MacKerell, A. D. CHARMM36 all-atom additive protein force field: validation based on comparison to NMR data. Aho, N. et al. Scalable constant pH molecular dynamics in GROMACS. Jansen, A., Aho, N., Groenhof, G., Buslaev, P. & Hess, B. phbuilder: a tool for efficiently setting up constant pH molecular dynamics simulations in GROMACS. Humphrey, W., Dalke, A. & Schulten, K. VMD: visual molecular dynamics. Wang, S. et al. Epon post embedding correlative light and electron microscopy. Demmerle, J., Wegel, E., Schermelleh, L. & Dobbie, I. M. Assessing resolution in super-resolution imaging. Chmyrov, A. et al. Nanoscopy with more than 100,000 ‘doughnuts'. Xiong, H. Dataset title: A highly stable monomeric red fluorescent protein for advanced microscopy. We thank S. Papadaki from Westlake Laboratory for verifying all plasmid sequences and depositing them to WeKwikGene. We thank E. Snapp from Janelia Research campus for the help with the interpretation of OSER imaging results. This project was supported by the National Natural Science Foundation of China (grant no. ), the Natural Science Foundation of Fujian Province, China (grant nos. ), the Research Foundation for Advanced Talents at Fujian Medical University, China (grant nos. ), the Finance Special Science Foundation of Fujian Province, China (grant no. ), Foundation of NHC Key Laboratory of Technical Evaluation of Fertility Regulation for Non-human Primate, Fujian Maternity and Child Health Hospital (grant no. ), Open Project Fund of Fujian Key Laboratory of Drug Target Discovery and Structural and Functional Research (grant no. ), Foundation of Westlake University, Westlake Laboratory of Life Sciences and Biomedicine, National Natural Science Foundation of China (grant no. ), and ‘Pioneer' and ‘Leading Goose' R&D Program of Zhejiang (grant no. We thank L. Zhou, M. Wu and X. Lin at the Public Technology Service Center, Fujian Medical University for support with EM sample preparation and EM imaging. These authors contributed equally: Haiyan Xiong, Qiyuan Chang, Jiayi Ding, Shuyuan Wang, Wenhao Zhang, Yu Li, Yaochen Wu. Key Laboratory of Clinical Laboratory Technology for Precision Medicine, Institute of Neuroscience, and Fujian Key Laboratory of Molecular Neurology, Public Technology Service Center, Fujian Medical University, Fuzhou, China Haiyan Xiong, Qiyuan Chang, Shuyuan Wang, Yaochen Wu, Pengyan Lin, Yiming Chen, Congxian Wu & Zhifei Fu The School of Basic Medical Sciences, Fujian Medical University, Fuzhou, China Haiyan Xiong, Qiyuan Chang, Yaochen Wu & Miaoxing Liu Academy for Advanced Interdisciplinary Studies, Peking University, Beijing, China Chinese Institute for Brain Research, Beijing, China Jiayi Ding & Hu Zhao School of Life Sciences, Westlake University, Hangzhou, China Westlake Laboratory of Life Sciences and Biomedicine, Hangzhou, China Wenhao Zhang & Kiryl D. Piatkevich Institute of Basic Medical Sciences, Westlake Institute for Advanced Study, Hangzhou, China Wenhao Zhang & Kiryl D. Piatkevich College of Life Sciences, Zhejiang University, Hangzhou, China Wenhao Zhang & Kiryl D. Piatkevich State Key Laboratory of Vaccines for Infectious Diseases, National Institute of Diagnostics and Vaccine Development in Infectious Diseases, School of Public Health, Xiamen University, Xiamen, China Yu Li, Chengyu Yang & Qingbing Zheng Microscopy core facility of Westlake University, Hangzhou, China Guicun Fang & Jiongfang Xie Institute of Life Sciences, Fuzhou University, Fuzhou, China Optofem Technology Company, Beijing, China National Laboratory of Biomacromolecules, New Cornerstone Science Laboratory, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China Wenfeng Fu & Dong Li Key Laboratory of Ministry of Education for Gastrointestinal Cancer, School of Basic Medical Sciences, Fujian Medical University, Fuzhou, China Fen Hu & Fenghua Zhang Fujian Key Laboratory of Drug Target Discovery and Structural and Functional Research, School of Pharmacy, Fujian Medical University, Fuzhou, China School of Pharmacy, Center of Translational Hematology, Fujian Medical University, Fuzhou, China Rongcai Yue & Yunlu Xu State Key Laboratory of Cellular Stress Biology, School of Life Sciences, Faculty of Medicine and Life Sciences, Xiamen University, Xiamen, China Yanbin Li & Yong Cui X-ray crystallography platform of National Protein Science Facility, Tsinghua University, Beijing, China Min Li & Shilong Fan You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and supervised the whole project. engineered mScarlet3-H and measured its properties. and M. Liu did SIM imaging. performed rapid tissue clearing. did CLEM imaging. tested the photostability of mScarlet3-H. P.L. measured the pKa of mScarlet3-H. Y. Cui and Yanbin Li performed experiments on mScarlet3-H's performance in plants. did experiments on mScarlet3-H's performance in zebrafish. and G.F. helped to do EM sample preparation. Yiwei Yang and Y. Chen cultured cells. conducted the MD simulation. analyzed the images from rapid tissue clearing. helped with SIM imaging and analyzing the images of 3D-SIM imaging. helped to purify mScarlet3-H. Q.Z., S.F., M. Li, Yu Li and Yufeng Yang solved the crystal structures of mScarlet-H and mScarlet3-H. All authors reviewed the paper. Correspondence to Congxian Wu, Qingbing Zheng, Kiryl D. Piatkevich or Zhifei Fu. A Chinese patent application (no. 202410568362.4) covering the use of mScarlet3-H for CLEM, rapid tissue clearing, expansion microscopy and fluorescent microscopy has been filed in which the Fujian Medical University is the applicant and Z.F., Y.W., H.X., Q.C., C.W., S.W. and Yiwei Yang are the inventors. The other authors declare no competing interests. Nature Methods thanks Benjamin Campbel, Takeharu Nagai and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Rita Strack, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Long-term super-resolution imaging of the ER and microtubule dynamics in live HeLa cells acquired with CSU-W1 SoRa imaging setup (×100 NA 1.41, total duration 01:18 h:min; plays at 100 f.p.s.). ER: mScarlet3-H; EMTB: mBaoJin. Long-term super-resolution imaging of mitochondria and EB3 dynamics in live HeLa cells acquired with CSU-W1 SoRa imaging setup (×100 NA 1.41, total duration 50:45 m:s; plays at 100 f.p.s.). Long-term super-resolution imaging of mitochondria and lifeact labeled actin dynamics in live HeLa cells acquired with CSU-W1 SoRa imaging setup (x100 NA 1.41, total duration 58:29 m:s; plays at 20 f.p.s.). Long-term super-resolution imaging of ER and mitochondria dynamics in live HeLa cells acquired with CSU-W1 SoRa imaging setup (×100 NA 1.41, total duration 08:15 m:s; plays at 10 f.p.s.). ER: mScarlet3-H; Mito: mBaoJin. Long-term imaging of cell mitosis of zebrafish larva labeled by mScarlet3-H using a STELLARIS 8 FALCON confocal microscope. Long-term imaging of H2B-mScarlet3-H in a developing zebrafish larva using a STELLARIS 8 FALCON confocal microscope. Video showing the fluorescence dynamics of HeLa cells expressing H2B-mScarlet3-H alternately treated with PBS and 5 M GdnHCl. Long-term super-resolution imaging of the dynamics of mitochondria labeled by mScarlet3-H in live COS-7 cells using a 3D-SIM imaging setup (×100 NA 1.49, total duration 2 h). Long-term super-resolution imaging of the dynamics of ER sheet labeled by mScarlet3-H in live COS-7 cells using a STED imaging setup (×100 NA 1.45, total duration 09:00 min:s). Long-term super-resolution imaging of the dynamics of ER sheet fusion labeled by mScarlet3-H in live COS-7 cells using a STED imaging setup (×100 NA 1.45, total duration 10.8 s). Long-term super-resolution imaging of the dynamics of ER sheet fission labeled by mScarlet3-H in live COS-7 cells using a STED imaging setup (×100 NA 1.45, total duration 3:18 m:s). Long-term super-resolution imaging of the dynamics of ER ball labeled by mScarlet3-H in live COS-7 cells using a STED imaging setup (×100 NA 1.45, total duration 14.4 s). Long-term super-resolution imaging of the dynamics of ER and mitochondria in live COS-7 cells using a STED imaging setup (×100 NA 1.45, total duration 5:32 m:s). ER: mScarlet3-H; Mito: HBmito Crimson. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Xiong, H., Chang, Q., Ding, J. et al. A highly stable monomeric red fluorescent protein for advanced microscopy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-58538-3'>Mitochondrial complexity is regulated at ER-mitochondria contact sites via PDZD8-FKBP8 tethering</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 09:39:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mitochondria-ER membrane contact sites (MERCS) represent a fundamental ultrastructural feature underlying unique biochemistry and physiology in eukaryotic cells. The ER protein PDZD8 is required for the formation of MERCS in many cell types, however, its tethering partner on the outer mitochondrial membrane (OMM) is currently unknown. Here we identify the OMM protein FKBP8 as the tethering partner of PDZD8 using a combination of unbiased proximity proteomics, CRISPR-Cas9 endogenous protein tagging, Cryo-electron tomography, and correlative light-electron microscopy. Single molecule tracking reveals highly dynamic diffusion properties of PDZD8 along the ER membrane with significant pauses and captures at MERCS. Overexpression of FKBP8 is sufficient to narrow the ER-OMM distance, whereas independent versus combined deletions of these two proteins demonstrate their interdependence for MERCS formation. Furthermore, PDZD8 enhances mitochondrial complexity in a FKBP8-dependent manner. Our results identify a novel ER-mitochondria tethering complex that regulates mitochondrial morphology in mammalian cells. Mitochondria and the endoplasmic reticulum (ER) form contact sites (mitochondria–ER contact sites: MERCS), where the two membranes are juxtaposed within 10–50 nm, an ultrastructural feature conserved in unicellular eukaryotes and metazoans. MERCS are the most abundant membrane contact sites (MCS) between organelles in many cell types and serve as a unique subcellular signaling platform for exchanging metabolites such as Ca2+ and glycerophospholipids. In addition to these critical biochemical reactions, key physiological and cell biological events essential for the maintenance of cellular homeostasis, including mitochondrial fission, mitochondrial DNA replication, and autophagosome biogenesis occur at these contact sites1,2,3. Observations using electron microscopy (EM) have demonstrated that mitochondria and ER membranes are closely apposed at MCS, requiring proteins able to tether these two membranes within tens of nanometers of one another4. Intensive screening studies have identified multiple proteins localizing at MERCS in mammalian cells1,5,6,7. Among those, the ER-resident protein PDZD8 was identified as a paralog of yeast Mmm1, a component of the ER–mitochondria encounter structure (ERMES)8,9. Although the ERMES as a full complex formed by four proteins is lost in mammals, PDZD8 is required for forming the majority (~40–80%) of MERCS in various cell types, and its deletion in cell lines and in mammalian neurons results in the disruption of intracellular Ca2+ dynamics by decreasing the fraction of Ca2+ released from the ER that can be imported directly into mitochondria8,10,11,12,13,14,15. Consistent with its role in neurons of central nervous system (CNS), PDZD8 regulates dendritic Ca2+ dynamics in hippocampal CA1 and underlies their response properties in vivo16. In addition, expression quantitative trait loci (eQTL) mapping identified a single-nucleotide polymorphism affecting the expression of PDZD8 in the dorsolateral prefrontal cortex in a population of patients with high risk for post-traumatic stress disorder (PTSD)18. In addition to MERCS, the ER forms various MCSs with other organelles such as lysosomes, endosomes, Golgi apparatus, lipid droplets, and the plasma membrane (PM)2,19,20,21,22. We and other groups have previously shown that PDZD8 localizes at MERCS in various cell types8,23,24. However, it has also been reported recently that overexpression of Rab7 or LAMP1 can recruit PDZD8 to the ER–late endosome or ER–lysosome contact sites, respectively23,25,26. In addition, overexpression of PDZD8 and Rab7 recruits the mitochondria to ER–endosome contact sites and was proposed to lead to the formation of three-way MCS23. Therefore, PDZD8 might participate in the formation of MCS networks besides tethering MERCS. As such, we hypothesized the existence of a currently unknown molecular effector required to recruit PDZD8 specifically to MERCS. To elucidate the molecular mechanisms underlying PDZD8-dependent MERCS formation, we used multiple independent proximity-based proteomic approaches relying on endogenous protein tagging. Since overexpression of PDZD8 can alter its subcellular distribution8. we implemented CRISPR–Cas9 technology to generate knock-in cell lines where endogenous PDZD8 is tagged with various epitopes, fluorescent proteins or catalytic enzymes, allowing its localization by microscopy or proximity-based proteomic screens. We demonstrate that the mitochondrial LC3 receptor FK506 binding protein 8 (FKBP8 also known as FKBP38) is a novel, direct PDZD8-interacting protein, and that the PDZD8–FKBP8 complex is required for MERCS formation in metazoan cells. Using combinations of Cryo-EM tomography and correlative light-electron microscopy (CLEM), we revealed the ultrastructural features of MERCS mediated by the PDZD8–FKBP8 tethering complex. Finally, our serial scanning electron microscopy demonstrated that PDZD8 regulates mitochondrial complexity through inhibition of FKBP8 function. While we previously reported that a significant fraction of PDZD8 localizes at MERCS8, PDZD8 was recently shown to localize to the ER–late endosome and ER–lysosome contact sites11,23,25,26. Some of these studies failed to detect the enrichment of PDZD8 at MERCS, however, in the absence of reliable antibodies detecting endogenous PDZD8 protein by immunofluorescence, these studies often relied on overexpression of tagged forms of PDZD8 which disrupts both its subcellular localization and can generate gain-of-function phenotypes, for instance by increasing the number and size of MERCS or other MCSs where it is localized. Therefore, to determine the subcellular distribution of endogenous PDZD8 protein at MCS formed by the ER, we developed a knock-in mouse embryonic fibroblast NIH3T3 cell line fusing the fluorescent protein Venus sequence to the C-terminus of the Pdzd8 coding sequence (Supplementary Fig. To avoid an artifactual increase in size and/or biogenesis of late endosome/lysosome due to overexpression of key effector proteins Rab7 or LAMP1, colocalization analyses were performed by detecting these proteins at endogenous levels with antibodies against endogenous markers: LAMP1 for lysosomes, Rab7 for the late endosomes, and Tomm20 and OXPHOS proteins for mitochondria. In agreement with previous studies11,25, confocal microscopy imaging showed that 14.8% of PDZD8-Venus visualized by an enhancement with anti-GFP antibody staining overlapped with LAMP1 staining, and under these endogenous expression conditions 7.7% overlapped with Rab7 staining. However, a significantly larger fraction overlapped mitochondria labeled either with Tomm20 (25.0%) or OXPHOS staining (22.1%), suggesting that endogenous PDZD8 is present at multiple MCS but is most abundant at MERCS (Supplementary Fig. We next investigated the dynamics of endogenously expressed PDZD8 using time-lapse imaging in live cells. Because native signal of Venus was undetectable in PDZD8-Venus KI NIH3T3 cells, we established the PDZD8-HaloTag KI HeLa cell line and transiently transfected the ER-localized reporter (BiP-mTagBFP2-KDEL) and an outer mitochondrial membrane (OMM)-localized reporter (YFP-ActA27,28) (Supplementary Fig. The PDZD8-HaloTag was labeled with Janelia Fluor (JF) 549 dye. Triple-color time-lapse imaging using confocal microscopy demonstrated that PDZD8-Halotag puncta can be stably localized at MERCS despite significant dynamics of both ER and mitochondria, suggesting a direct association of PDZD8 with mitochondria may be present (Supplementary Fig. Recent work demonstrated that the ER-resident MERCS forming protein VAPB exhibits transient but highly frequent visits to MERCS29. Thus, to determine the localization and molecular dynamics of PDZD8 along the ER membrane relative to MERCS, we performed single particle tracking-photoactivation localization microscopy (sptPALM)30. Single PDZD8 molecules were visualized by labeling overexpressed PDZD8-HaloTag with a photoactivatable version of JF646 in COS7 cells (Supplementary Movie 2). Analysis of localization probabilities using a spatially defined probability function29 revealed PDZD8 localization was entirely restricted to the ER (Fig. Strikingly, we observed regions along the ER where the probability was significantly higher (hotspots), presumably as a result of tethering and engagement with interacting proteins at contact sites with other organelles. In agreement with our endogenous labeling, ~47% of these hotspots were in close proximity with mitochondria (Fig. By following the trajectories of single PDZD8 molecules outside and within these mitochondria-associated PDZD8 hotspots (MitoHS), we found that PDZD8 can dynamically enter and exit these hotspots in seconds (Fig. Importantly, the effective diffusion (Deff) of single PDZD8 molecules within MitoHS was significantly reduced compared the rest of the ER (0.22 ± 0.0025 μm2/s in MitoHS, mean ± SEM, n = 90; Fig. 1e) suggesting that PDZD8 is captured at MERCS but still remains mobile at these contact sites. Consistent with this, PDZD8 single particles dwelled at the hotspots for a median time of just 1.1 s per each visit (Fig. In addition to MitoHS, we also observed spots with high probability of PDZD8 that were not mitochondria-associated (OtherHS: Other hotspots, Fig. The Deff and dwell time of PDZD8 in the OtherHS is similar to those in MitoHS, but the mean of individual HS area was significantly larger at MitoHS than at the OtherHS (Fig. a Diffraction-limited imaging of the ER (cyan) and the mitochondria (red) in the periphery of a representative COS7 cell with the simultaneously measured likelihood of finding a PDZD8 molecule in a 1-min window. Boxes correspond to the mitochondria-associated hotspots (MitoHS, magenta) or non-mitochondria-associated hotspots (OtherHS, green) in b and c. Data are representative of 14 cells from two independent experiments. b Zooms of MitoHS in a showing individual PDZD8 trajectories engaging with the hotspots and the associated PDZD8 probability density. Dotted lines indicate hotspot boundaries as used for subsequent analysis. Dotted lines indicate hotspot boundaries as used for subsequent analysis. The shape and size are consistent with endosomal contact sites, as described in Obara et al.29. e PDZD8 shows reduced diffusion within both classes of hotspots as compared to freely diffusing in the surrounding ER. Statistical analysis was performed using two-sided Mann–Whitney test for comparing % reduction in 2D Deff in MitoHS between OtherHS and one sample Wilcoxon signed-rank test for comparing the hypothetical median (0) and the median of % reduction in 2D Deff within MitoHS or OtherHS. Whiskers extend to the minimum and maximum values. f Sizes of MitoHS are significantly larger than those of OtherHS in the same cells. Whiskers extend to the minimum and maximum values.Statistical analysis was performed using two-sided Mann–Whitney test. **p = 0.0065. g PDZD8 dwell times in individual MitoHS and OtherHS. Inset shows the leaving frequency (kout) of individual PDZD8 molecules from probability hotspots associated or unassociated with mitochondria. Scale bar: a 5 µm, b, c 100 nm. We note that the size of the hotspots observed with PDZD8 is significantly larger and the mean dwell time of PDZD8 at the hotspots was significantly longer compared to those reported with VAPB (Fig. Taken together, these data suggest that PDZD8 is highly dynamic along the ER but drastically slows down at contacts between ER and mitochondria as well as other potential MCS between ER and other organelles. These results strongly suggest the existence of an unknown tethering partner for PDZD8 along the OMM. To identify the tethering partner of PDZD8 facilitating this behavior at MERCS, we designed unbiased proteomic screens using endogenous PDZD8 protein immunoprecipitation coupled with mass spectrometry (IP–MS) (Fig. To avoid artifacts due to PDZD8 overexpression, we established a mouse line engineered with a 3× HA tag fused to the endogenous PDZD8 protein using CRISPR–Cas9-mediated genomic knock-in (Pdzd8-3× HA KI mouse line) (Fig. Since PDZD8 is expressed at high levels in neurons, protein complexes containing PDZD8 were isolated from the neocortex of either Pdzd8-3× HA KI mice or control littermates at postnatal day 10 by IP using anti-HA antibody. Identification of the corresponding proteins immunoprecipitated in a complex with PDZD8-3× HA by LC–MS/MS revealed that, in addition to previously identified PDZD8 interactors such as Protrudin, VAPA and VAPB11,31, proteins known to localize at MERCS and/or mitochondria were significantly enriched in the immunoprecipitates from the KI mice compared to the control mice (Fig. b Diagram describing the genomic sequence of Pdzd8-3× HA KI mice. c Volcano plot of proteins differentially binding to the PDZD8-3× HA. Protrudin and VAPA, which have been previously reported to interact with PDZD8, are labeled in blue. The plot represents data from three biological replicates. The p-value was calculated using an unadjusted two-tailed Student's t-test. The biotin–5′-AMP can covalently bind to proteins located within about 20 nm of endogenously expressed PDZD8-TurboID. f Volcano plot of proteins differentially biotinylated with biotin in the PDZD8-TurboID KI HeLa cell. Protrudin and VAPA, which have been previously reported to interact with PDZD8, are labeled in blue. The volcano plot represents three biological replicates. The p-value was calculated using an unadjusted two-tailed Student's t-test. g Numbers of proteins highly enriched in the IP–MS (c) and TurboID-MS (f) are shown in a Venn diagram. Twelve proteins are commonly found in the two proteomes. Note that FKBP8 is the only protein annotated with mitochondrial localization. Extracts from neocortex in Pdzd8-3×HA KI mouse (h) or Pdzd8-Venus KI NIH3T3 cells (i) were subjected to immunoprecipitation (IP) with antibodies to HA or GFP respectively. The resulting precipitates as well as the original tissue extracts (Total) were subjected to immunoblot analysis with antibodies to FKBP8, VAPA, MFN2, HA (h), GFP (i), and β-actin (i). Data are representative of three independent experiments. Next, in order to narrow down the protein list to only proteins in close proximity to PDZD8, we employed an independent approach, a proximity-based labeling screen using a biotin ligase TurboID (Fig. Again, to avoid overexpression-induced artifacts, we established a PDZD8-TurboID KI HeLa cell line using CRISPR–Cas9 knock-in technology (Fig. These PDZD8-TurboID KI HeLa cells were treated with biotin for 6 h and biotinylated peptides were isolated using tamavidin 2-REV beads and identified by LC–MS/MS (Fig. Among 166 proteins identified by this screening approach, 12 proteins were also identified by the IP–MS-based screen (Fig. Among these candidate interactors, the only protein previously shown to localize at the outer mitochondrial membrane (OMM) was FKBP8 (Fig. Finally, we also performed a proteomic screen using TurboID in a mouse neuroblastoma cell line (Neuro2a) and again identified FKBP8 in the list of biotinylated proteins (Supplementary Fig. Specific co-immunoprecipitation of FKBP8 and PDZD8 was confirmed by Western blotting using the Pdzd8-3× HA knock-in mouse (Fig. These three independent proteomic approaches converge to strongly suggest that PDZD8 and FKBP8 reside in the same protein complex. To test if the interaction between PDZD8 and FKBP8 is direct, we measured the binding affinity of PDZD8–FKBP8 interaction in vitro. We used surface plasmon resonance (SPR) with purified recombinant cytosolic portions of both FKBP8 and PDZD8 (Supplementary Fig. Recombinant PDZD8 proteins without their transmembrane domain (∆TM) were immobilized on a sensor chip and changes of the surface resonance upon recombinant FKBP8∆TM injection were measured. As expected, the SPR responded in a FKBP8 dose-dependent manner in the 2–90 µM range (Fig. Even though the titration did not reach a plateau, by assuming a monovalent binding of FKBP8 and PDZD8, fitting Req (SPR responses in equilibrium) and FKBP8 concentration to the titration curve provided a KD value of 142 µM (74–447 µM, 95% confidence interval) (Fig. Thus, the affinity between recombinant PDZD8 and FKBP8 is in the same range as other previously reported VAPB-PTPIP51 MERCS tethering complex and agrees with the unexpectedly rapid exchange observed by sptPALM33. Recombinant human PDZD8 (1, 28–)—FLAG was immobilized on the sensor chip and FKBP8 (1–380)—Histag with indicated concentrations were injected. b SPR responses at equilibrium (Req) were plotted against FKBP8 concentration. The plot of Req versus FKBP8 concentration was fitted to a monovalent binding model to determine KD values. d Pdzd8f/f::CreERT2 MEFs expressing a series of deletion mutants of PDZD8-3× FLAG shown in (c) and HA-FKBP8 were treated with 1 μM 4-hydroxy tamoxifen (4-OHT) and cell extracts were immunoprecipitated with anti-HA antibody. Data are representative of three independent experiments. e GST-Pulldown assay from the mixture of recombinant GST—Thrombin cleavage site—human PDZD8 (1, 28–506)—HA and recombinant human FKBP8 (1–380)—Histag in vitro. FKBP8—Histag was eluted only from the GST beads incubated with GST- PDZD8 (1, 28–506)—HA. Data are representative of two independent experiments. The FKBP8-Histag was enriched when incubated with hPDZD8 (1, 28–506)—HA, compared to the negative controls (buffer or buffer with BSA). Data are representative of two independent experiments. g Immunofluorescence analysis of Pdzd8-Venus KI NIH3T3 cells knocking out endogenous FKBP8 by confocal microscopy with a Nikon Spatial Array Confocal (NSPARC) detector. The cells were transfected with the control gRNA (upper two rows) or three gRNAs against FKBP8 (bottom two rows), Cas9, and transfection marker mtagBFP2, and stained with antibodies to GFP, and Tomm20 for visualizing endogenous PDZD8-Venus (green) and mitochondrial outer membrane (magenta), respectively. Whiskers extend to the minimum and maximum values. Statistical analysis was performed using two-sided Mann–Whitney U test. Next, to identify the protein domains of PDZD8 required to mediate interaction with FKBP8, we conducted co-IP experiments by expressing a series of 3× FLAG-tagged PDZD8 deletion mutants together with HA-tagged FKBP8 (Fig. Based on the previous reports suggesting that PDZD8 can homodimerize, endogenous full-length PDZD8 can act as a bridge between exogenously expressed truncated forms of PDZD8 and FKBP8, even in the absence of direct binding26. To avoid this, we established a tamoxifen-inducible Pdzd8 conditional KO mouse embryonic fibroblast cell line (Pdzd8f/f::CreERT2 MEFs) (Supplementary Fig. Using a time-course analysis, we determined that PDZD8 was undetectable 45 h after CreERT2-mediated deletion of the floxed allele by treatment with 4-hydroxytamoxifen (4-OHT; Supplementary Fig. Whereas truncated forms of PDZD8 including TM-SMP domains co-precipitated FKBP8 efficiently, none of the other domains showed strong binding to overexpressed FKBP8 (Fig. These results suggest that TM-SMP domains of PDZD8 represent the minimal domain mediating interaction with FKBP8. Next, we tested if SMP-C2n-PDZ domain of PDZD8 directly binds to FKBP8 using purified recombinant proteins. Recombinant glutathione S-transferase (GST)—Thrombin cleavage site - human PDZD8 (1, 28–506) - HA and human FKBP8 (1–380)—Histag were expressed in E. coli and purified with GST-binding beads and TALON affinity columns, respectively. These purified proteins were mixed in vitro, applied to a column with GST-binding beads and eluted by cleaving the thrombin cleavage site. Western blotting analysis revealed that FKBP8 was isolated only when it was incubated with GST-PDZD8 (1, 28–506)—HA (Fig. This binding of FKBP8 and PDZD8 was confirmed by a pull-down assay using the same set of purified proteins and anti-HA antibodies (Fig. Collectively, these results suggest that the SMP domain of PDZD8 plays a dominant role in interacting with FKBP8 while the TM domain is necessary for the interaction in cellulo, presumably for recruiting PDZD8 to the ER membrane. Given that protein binding and late endosome/lysosome recruitment functions of PDZD8 are suggested to be independent of the SMP domain23,25,34, the SMP domain of PDZD8 may represent a unique binding interface with FKBP8. Our live imaging of endogenously expressed PDZD8 and the single molecule tracking of PDZD8 showed that PDZD8 is highly mobile throughout the ER but shows distinct interactions (confined diffusion) where the ER is contacting mitochondria (Supplementary Fig. Therefore, the direct binding of PDZD8 and FKBP8 prompted us to examine whether FKBP8 is required for capturing of PDZD8 to mitochondria. To achieve this, Cas9 and guide RNAs targeting Fkbp8 gene locus were transiently expressed in the PDZD8-Venus KI NIH3T3 cell line. Immunocytochemistry confirmed that FKBP8 was not detectable in more than 81% of transfected cells (labeled with mTagBFP2; Supplementary Fig. By quantifying the ratio of PDZD8 closely associated with mitochondria (stained by the anti-Tomm20 antibody) using a confocal microscopy equipped with a super resolution Nikon Spatial Array Confocal (NSPARC) detector, we found that colocalization of PDZD8 with mitochondria was significantly reduced in the FKBP8-depleted cells (Fig. This demonstrates that FKBP8 is required for recruiting the ER protein PDZD8 to mitochondria. Our results demonstrate that a direct binding between FKBP8 and PDZD8 and also that FKBP8 is required for PDZD8 recruitment to mitochondria. Thus, we investigated if the interaction between PDZD8 and FKBP8 is critical for MERCS formation. As previously reported in HeLa cells constitutively deleted with PDZD88, conditional deletion of PDZD8 induced by a treatment with 4-OHT to Pdzd8f/f::CreERT2 MEFs (Pdzd8 cKO) significantly decreased the size of MERCS, defined as the fraction of OMM membranes associated (≤3 pixels: 23.4 nm) with ER, compared to the vehicle-treated control isogenic MEFs (Fig. Strikingly, shRNA mediated knock-down (KD) of Fkbp8 (validated in Supplementary Fig. Importantly, Fkbp8 KD in Pdzd8 cKO MEFs did not further reduce the fraction of MERCS compared to Pdzd8 KO MEF only (Fig. Two-way ANOVA analysis shows that there is a strong functional interaction between the effects of FKBP8 and PDZD8 loss of function regarding the size of MERCS (Fig. We also observed the same trend when MERCS were defined as OMM membranes associated with the ER at a longer distance, specifically 32.4–54.6 nm (4–7 pixels) (Supplementary Fig. a Representative electron micrographs of Pdzd8f/f::CreERT2 MEFs infected with lentivirus carrying shControl or shFKBP8, and treated with or without 0.5 µM 4-OHT. MERCS (yellow arrowheads) were more frequently observed in the Control cells than in Pdzd8 cKO, Fkbp8 KD, and Pdzd8 cKO + Fkbp8 KD cells. Whiskers extend to the minimum and maximum values. n = 33, 29, 39, 34 cells from two independent experiments for the control, Pdzd8 cKO, Fkbp8 KD, and Pdzd8 cKO + Fkbp8 KD cells, respectively. Statistical analysis was performed using one-way ANOVA and Fisher's LSD test. ****p < 0.0001, ***p = 0.0003. c The interaction plot corresponding to b. Dots show the mean of each condition. Although the vast majority of FKBP8 localizes at the mitochondria, an escape of FKBP8 from mitochondria to the ER has been reported upon mitophagy induction35. Therefore, to determine if PDZD8 colocalizes with the ER-resident FKBP8 (cis-interaction) or with the mitochondrial FKBP8 (trans-interaction), we determined the subcellular compartments where PDZD8 and FKBP8 colocalize. Immunostaining using anti-FKBP8 antibodies showed a puncta-like distribution of endogenous FKBP8 and revealed that 88.2% of FKBP8 is localized at mitochondria in HeLa cells (Fig. This juxtaposition of PDZD8 and FKBP8 was not observed outside the mitochondria (‘off mito regions') (Fig. These suggest that FKBP8 localizes near PDZD8 within mitochondria but not in other cytoplasmic region. Additionally, we found that the ratio of PDZD8 intensity overlapped with FKBP8 on mitochondria was significantly reduced by the FKBP8 randomizing (Fig. Moreover, PDZD8 puncta were significantly enriched in the FKBP8-positive area of mitochondria (Fig. 5e), indicating that PDZD8 colocalizes with FKBP8 more frequently than random occurrences on mitochondria. To independently confirm these results in cells derived from a different species and using endogenous tagging of FKBP8 and PDZD8 simultaneously, we developed a dual KI strategy (Supplementary Fig. 7b), whereby an HA-tag was knocked-in at the Fkbp8 genomic locus to express HA-FKBP8 in the Pdzd8-Venus KI NIH3T3 cell line. Consistent with the localization in HeLa cells, PDZD8 significantly accumulated in FKBP8-present regions on mitochondria compared to FKBP8-absent regions (Supplementary Fig. Taken together, these results strongly suggest that an ER-resident protein PDZD8 colocalizes with FKBP8 specifically on the mitochondria. a Immunofluorescence analysis of PDZD8-Halotag KI HeLa cells. The cells were treated with 200 nM of Janelia Fluor 549 for 20 h and then stained with antibodies to FKBP8 and to Tomm20. Arrowheads indicate PDZD8 colocalized both with FKBP8 and Tomm20. b The ratios of FKBP8 intensity on or outside (off) the mitochondria were determined for images obtained as described in a. of nine cells from two independent experiments. c Distribution of PDZD8 puncta with the indicated distance to the nearest FKBP8 puncta was determined for images obtained as described in a. The distance from centroids of each PDZD8 punctum to the nearest FKBP8 centroids was calculated within mitochondria (on mito) or outside of the mitochondria (off mito) respectively. Nine cells from two independent experiments were used in the calculation. ****P < 0.0001, *P = 0.0254. d The ratios of PDZD8 intensity overlapped with FKBP8 on mitochondria (Mander's coefficients) were determined for images as described in a. Data are representative of two independent experiments (9 cells). A two-sided paired t-test was used to test statistical significance. *P = 0.0172. e The means of PDZD8 intensity in the FKBP8-present or FKBP8-absent area on mitochondria were determined for images as in a. Data are representative of two independent experiments (nine cells). A two-sided paired t-test was used to test statistical significance. We next tested if overexpression of the mitochondrial FKBP8 is sufficient for recruiting endogenous ER-localized PDZD8 to mitochondria. We overexpressed a mutated form of FKBP8 previously shown to lock its localization at the OMM (FKBP8N403K)35 in the PDZD8-Halotag KI HeLa cells. Strikingly, the overlap of PDZD8 with an OMM-marker YFP-ActA in HA-FKBP8N403K overexpressing cells was significantly increased (Fig. This suggests that the mitochondrial FKBP8 binds to PDZD8. Then, we examined if overexpression of FKBP8N403K recruits the ER together with PDZD8 by a correlative light-electron microscopy (CLEM) analysis (Supplementary Fig. Endogenous PDZD8 was labeled with JF549 dye in the PDZD8-HaloTag KI HeLa cell expressing with Venus-FKBP8N403K or YFP-ActA (OMM marker). Confocal microscopy with an NSPARC detector was used to visualize JF549 labeled PDZD8-HaloTag and Venus-FKBP8N403K or YFP-ActA signals within fixed cells, and the area imaged by confocal microscopy was subsequently re-identified in EM images (Supplementary Fig. OMM and the ER membrane within 25 nm of each other (MERCS) were segmented in the EM images and then 3D-reconstructed from 8 slices with 50 nm thickness (total 400 nm thick in z-axis) (Fig. The 3D-reconstructed mitochondria and MERCS was aligned to the confocal microscopy image using the FKBP8 signals or ActA signals as landmarks for mitochondria (Fig. Notably, PDZD8 puncta observed near mitochondria were highly accumulated in MERCS of the FKBP8N403K-overexpressing cell (arrowheads in Fig. Taken together, using multiple independent approaches, our results demonstrate that PDZD8 and FKBP8 form a complex between the ER and mitochondria and the overexpression of FKBP8 at OMM increases the abundance of this protein complex at MERCS. a Immunofluorescence analysis of PDZD8-Halotag KI HeLa cells overexpressing HA-FKBP8N403K. Cells transfected with either the control plasmid (upper two rows) or the plasmid encoding HA-FKBP8N403K (bottom two rows), along with the mitochondrial marker YFP-ActA, were treated with 200 nM of JF549 for 20 h, and subsequently, the fixed cells were observed using a confocal microscope equipped with a Nikon Spatial Array Confocal (NSPARC) detector. Whiskers extend to the minimum and maximum values. Statistical analysis was performed using two-sided Student's t-test. ***P = 0.0007. c–e Correlative light and electron microscopy (CLEM) analysis in a PDZD8-HaloTag KI HeLa cell. Cells overexpressing with Venus-FKBP8N403K or YFP-ActA (for the control) were treated with 200 nM of JF549 for 20 h and then fixed cells were observed by a confocal microscope. After that, ultra-thin sections (50 nm thick) were created and observed in a field emission scanning electron microscope (FE–SEM). Electron micrographs of the serial 8 slices were corresponding to an optical section of fluorescence images. Segmentations and 3-demensional (3D) reconstructions of mitochondria and the ER within 25 nm of mitochondria (MERCS) in electron micrographs were shown in c. 3D reconstruction from electron micrographs (shown as “EM”) were merged with fluorescence images (shown as “LM”) in d. The z projection of mitochondria and MERCS in EM was overlaid with fluorescence images in e. Arrowheads indicate PDZD8 puncta that localize to MERCS. Cryo-electron tomography (cryo-ET) provides a resolution range of 3–50 Å that is not accessible with other techniques and, importantly, allows the quantification of ultrastructural features of MERCS in situ under native conditions. Using correlative cryo-light microscopy and cryo-ET, we studied the in situ topology of mammalian mitochondria and mitochondria-associated membrane (MAM). First, we overexpressed FKBP8N403K in Pdzd8-Venus KI NIH3T3 cells and replicated the effect of recruiting and stabilizing PDZD8 in cells grown on cryo-EM grids (Fig. We then used cryo-focused ion beam (cryo-FIB) milling to generate lamellae (<200 nm-thick slice per cell) from PDZD8-HaloTag KI HeLa cells overexpressing mScarlet-FKBP8N403K (Fig. Since mScarlet-FKBP8N403K overexpression significantly increased the number of associations between mitochondria and MAM in each lamella compared to control cells, we fully segmented and labeled 20 or 6 membrane structures at OMM within 50 nm of their associated membranes in the FKBP8N403K overexpression condition or in the control cells, respectively. Using surface morphometrics analysis36, we quantified MAM–OMM distances at the level of a fraction of MAM. Our cryo-ET analyses demonstrate that MAM–OMM distances at any given interface are quite heterogeneous ranging from 10 to 50 nm (Fig. Moreover, an aggregate analysis for both the overexpressed and control conditions showed that overexpression of FKBP8N403K significantly (p < 0.0005, Kolmogorov–Smirnov test) shifted MAM–OMM distances to shorter values with a weighted median value at 25.7 nm compared to 30.1 nm in control cells (Fig. These results suggest that overexpression of FKBP8N403K, which efficiently recruits ER-localized PDZD8 to MERCS, imposes distances shorter than 25 nm between the MAM and OMM. The images represent two stacks (field of view: 638.9 µm2) for the control and four stacks (field of view: 638.9 µm2) for FKBP8N403K OE, respectively. b–e For the cryo-ET analysis, mScarlet-FKBP8N403K overexpression (OE) was used to increase the number of associations between mitochondria and mitochondria-associated membrane (MAM) captured in cryo-FIB milled lamellae. An SEM image of a target cell before Cryo-FIB milling is shown (b). Note that the apparent mismatch between the cryo-fluorescence image and cryo-TEM is due to factors including autofluorescence, differences in resolution, registration errors, distortions in the imaging plane, and ice-crystal contaminations (c). Using medium-mag high-resolution TEM montages of the lamellae, mitochondria with MAM were targeted for high-resolution tilt series acquisition (d). Eighty tomograms containing MAM were obtained for the OE condition (of which 20 were fully segmented and labeled), and 10 tomograms containing MAM were obtained for the control (of which 6 were fully segmented and labeled). Two representative tomograms from the OE condition corresponding to the arrows in panel (d) are shown (e). The image of b represents more than 35 cells. The image of c represents 13 lamellae. f A surface morphometrics analysis was used to calculate the MAM-outer mitochondrial membrane (OMM) distance. The distances are shown as a heatmap. Mammalian OMM and MAM show a great deal of heterogeneity in their membrane ultra-structure. g Aggregate analysis of the area-weighted MAM–OMM distance histogram shows a shift to smaller distances in the overexpression condition compared to the control. MERCS represent hotspots for both fission and fusion of mitochondria37,38,39. Importantly, previous reports suggested that FKBP8 promotes mitochondrial fission40,41. Thus, we decided to examine the role of PDZD8–FKBP8 complex in the regulation of mitochondrial morphology. Quantitative volume analyses of mitochondria reconstructed from serial electron microscopy images revealed that mitochondria in PDZD8 cKO cells were significantly spherical represented by the smaller mitochondria complexity index (MCI42) compared to the control (Fig. These data suggest that PDZD8 increases but FKBP8 decreases the mitochondrial complexity. Interestingly, PDZD8 KO did not reduce the MCI in FKBP8 KD background (Fig. This indicates that PDZD8 suppresses the function of FKBP8 in regulating mitochondrial structure. a Schematic of mitochondrial morphology analysis by the volume EM. b Formula of calculating MCI (mitochondrial complexity index)42. MCI is calculated as SA3/(16π2V2), where SA is the surface area and V is the volume of each mitochondrion (see  details in the Methods section). c Representative 3D reconstruction of mitochondria extracted from serial EM images acquired by array tomography in Pdzd8f/f::CreERT2 MEFs infected with lentivirus carrying shControl or shFKBP8, and treated with or without 0.5 µM 4-OHT. d Quantification of MCI (mitochondrial complexity index)42. Whiskers extend to the minimum and maximum values. Statistical analysis was performed using one-way ANOVA and Fisher's LSD test. The biology of organelle contacts has emerged as molecularly complex and highly dynamic but mediating many crucial aspects of cell physiology. In this study, we investigated the molecular mechanisms of MERCS formation and its role in regulating mitochondrial morphology by analyzing the dynamics of the ER–mitochondria tethering protein PDZD8 using endogenous tagging, single particle tracking, identifying the partner protein on the mitochondrial side, and investigating mitochondrial morphology at a nanometer scale. Our results show (1) specific PDZD8 recruitment at MERCS under endogenous conditions, (2) PDZD8 moving dynamically along the ER membrane and exhibits a significant increase in dwell time or transient ‘capture' near points of contacts with mitochondria, and (3) that FKBP8, identified using a battery of endogenous tags and biotin ligase mediated proteomics, is a novel PDZD8 binding partner mediating its tethering function at MERCS across multiple systems and species. Super-resolution optical imaging and CLEM analysis revealed that FKBP8 is necessary and sufficient for recruiting PDZD8 to MERCS. Furthermore, our ultrastructural analysis suggested that the binding between FKBP8 and PDZD8 is necessary for the formation of a significant fraction of MERCS. We took advantage of a cryo-ET pipeline for characterizing mitochondria and their associated membranes at sub-nanometer resolution in native conditions and discovered that the MAM–OMM distances are highly variable with a range of 10–50 nm and narrowed by overexpressing FKBP8N403K. Taken together, our results revealed a novel molecular mechanism underlying the formation of contacts between the ER and mitochondria. Given that our screenings identified VAPA and VAPB (VAPs), known participants in the MERCS formation, as candidate proteins interacting with PDZD8, it would be intriguing to explore whether PDZD8 and these proteins form a complex and how they regulate MERCS. The ER spreads throughout the cell and works as a hub exchanging a wide variety of molecules with other organelles especially through membrane contact sites. It has been shown that PDZD8 is an ER protein required for the formation of MERCS, but also localizes at MCS between ER–lysosome, ER–late endosome, and at the ER–late endosome–mitochondria tripartite contacts, all of which we can observe with some frequency in our data11,23,25. The results of sptPALM showing the dynamic exchange of PDZD8 inside and outside hotspots at ER-mitochondrial contacts suggest that PDZD8 may be able to move rapidly between different types of MCS, as suggested for VAPB. Thus, it is possible that FKBP8 and Rab7 are competing for sequestration of PDZD8 and therefore might control the balance between the areas of MERCS and ER–lysosome contacts. Although the absence of Rab7 and other late endosomal/lysosomal proteins in the protein list from our screening results suggests that our current experimental system may not be ideal for investigating this hypothesis, single-particle tracking analyses exploring the sequestration of PDZD8 in cells where it binds with both FKBP8 and Rab7 would be of great interest. Previous studies revealed that FKBP8 expression induces mitochondrial fragmentation and mitophagy through its LC3-interacting region motif-like sequence (LIRL) and LC3-interacting region (LIR), respectively40. In agreement with this, our high-throughput volume EM analysis demonstrated that FKBP8 limits mitochondrial volume possibly by limited fusion or promoting fission (Supplementary Fig. Considering that PDZD8 is required to suppress mitophagy in Drosophila neurons12, one potential function of PDZD8 binding to FKBP8 could be to arrest the progression of mitophagy by inhibiting FKBP8-dependent modulation of mitochondrial shape after initiating the formation of the isolation membrane. Future studies using high-resolution live imaging will be required to clarify how the localization of this complex affects the morphological changes of mitochondria. Cryo-ET pipelines will pave the way for sub-nanometer analysis of intact mammalian MCS in their native state. However, cryo-ET has limitations: (1) it is restricted to imaging lamellae thinner than 200 nm, which makes it challenging to capturing the entire ultrastructure, including whole mitochondria and MERCS, and (2) its technically demanding sample preparation limits its use for large datasets and broader fields of view. To compensate these limitations, we performed a 3D ultrastructural analysis using array tomography on chemically fixed cells using wide-field SEM. Although our cryo-ET analysis demonstrated FKBP8's role in controlling distances between membranes at mitochondrial contact sites in a near-native state (Supplementary Fig. 9d), fixation-induced artifacts remain a potential limitation, regardless of whether chemical or cryo-fixation methods are used. Thus, studying the PDZD8–FKBP8 complex's role in MERCS formation within living cells, especially in relation to ER and mitochondrial dynamics, is an important direction for future research. Furthermore, protein localization analysis using confocal microscopy revealed that FKBP8 is essential for MERCS formation by recruiting PDZD8 to the vicinity of mitochondria. Although FKBP8 knockout did not completely block PDZD8 recruitment to mitochondria, it is important to note that the limited resolution of the z-axis in fluorescence microscopy may have added background signals to overestimate the localization of PDZD8 near mitochondria. While this limitation makes it difficult to precisely define MERCS using fluorescence microscopy alone, the combination of fluorescence and electron microscopy (3D-CLEM) ensured that PDZD8 is indeed localized within MERCS. These analyses were performed using endogenous protein observation, as overexpression can obscure true localization due to excessive protein levels. Although the low expression levels of endogenous PDZD8 prevented us from observing its dynamics using sptPALM, photoactivation enabled us to track the dynamics of individual molecules in overexpressed proteins. Future studies are needed to investigate the dynamics of endogenous PDZD8. This study elucidates a molecular pathway that regulates mitochondrial morphology at the interface with the ER. Given the dynamic nature of MERCS, revealing how cellular conditions utilize this pathway to modulate mitochondria will provide novel insights into cellular homeostasis. D6429) supplemented with 10% FBS (MP Biomedicals, catalog No. COS7 cells were maintained in phenol red-free DMEM (Corning, catalog No. 25200114) supplemented with 10% FBS (Corning, catalog No. HeLa cells were transfected with plasmids by Lipofectamine LTX reagent with Plus reagent (Thermo Fisher) and Lipofectamine 2000 transfection reagent (Thermo Fisher), or with siRNAs by Lipofectamine RNAiMAX transfection reagent (Thermo Fisher). All animals were maintained and studied according to protocols approved by the Animal Care and Use Committee of The University of Tokyo. Mice were housed under a 12-h light/dark cycle at an ambient temperature of 23 °C and a humidity of >40%. For CRISPR–Cas9 plasmid, CRISPR guide RNA that targets the region prior to Fkbp8 start codon was designed using CRISPR Design tool (Horizon Discovery Ltd.) and cloned into pSpCas9 (BB)-2A-Puro (PX459) V2.0 (Addgene plasmid # 62988)46. The donor oligonucleotides containing the 5′ arm sequence, the sequence of HA tag and the 3′ arm sequence (5′-TCC CCG AGC CGC AGG GCC AGT TCC TGA TCC CAG CAG CAT GTA CCC ATA CGA TGT TCC AGA TTA CGC TGC GTC TTG GGC TGA GCC CTC TGA GCC TGC TGC CCT-3′) were obtained from Eurofins Genomics. Pdzd8-Venus KI NIH3T3 cells were transfected with CRISPR–Cas9 plasmid and the donor oligonucleotides by polyethylenimine. Pdzd8f/f mouse embryos were dissected from anesthetized females at embryonic day 13.516. The embryos were minced, and after treatment with 0.25% trypsin (Gibco), 50 μg/mL of DNaseI (Merck) and 0.67 mg/mL of Hyaluronidase (Merck) in PBS for 20 min, the cells of the resulting suspension were plated onto 100-mm culture dishes and maintained in culture medium for 4 days. The cells were then immortalized by transfecting with plasmids encoding simian virus 40 (SV40) large T antigen (pMK16_SV40 T ori (−)47). After that, the cells were infected with lentivirus carrying Cre-ERT2 and single cell clones were obtained using a limiting dilution in 96-well plates. 5e, cells were boiled with a solution containing 50 mM Tris-HCl (pH 7.5), 100 mM NaCl, 1 mM EDTA, 0.5% Tween20, and 0.5 mg/mL Proteinase K (Nacalai) at 55 °C for 1 h. Using the resulting supernatant, the target DNA sequence was amplified with KOD FX Neo (TOYOBO). CRISPR/Cas9-mediated genome editing was performed using the iGONAD method48. Two- to three-month old female mice (Jcl:ICR, CLEA Japan) were mated with male mice the day before electroporation. The female mice with virginal plug were used for iGONAD at embryonic day 0.75. Genome editing solution was prepared with 1 mg/ml Cas9 protein (IDT, 1081059), 30 mM crRNA (annealed with tracrRNA, IDT, 1072534), 2 mg/ml ssODN (IDT, Ultramer DNA Oligo, standard desalting), and FastGreen (Fujifilm Wako, 061-00031) in OPTI-MEM (Thermo Fisher Scientific, 11058021). Oviduct electroporation was performed using NEPA21 and CUY652P2.5 × 4 (NEPA gene) with the following protocol: three poring pulses (50 V, 5 ms, 50 ms interval, and 10% decay [±pulse orientation]) and three transfer pulses (10 V, 50 ms, 50 ms interval, and 40% decay [±pulse orientation]). After electroporation, the oviducts were returned to their original position. The sequences of crRNA and ssODN were as follows: crRNA, 5′-ATT GAT TAC ACT GAC TCA GA-3′ and ssODN, 5′-AGC CAT TCA GCA ACA TTT CCG ATG ACT TGT TCG GCC CAT CTG AGT CAG TGT ACC CAT ACG ATG TTC CAG ATT ACG CTG GCT ATC CCT ATG ACG TCC CGG ACT ATG CAG GAT CCT ATC CAT ATG ACG TTC CAG ATT ACG CTG TTT AAT CAA TAA GCT ATT TCA ACT TTC ACA TGG ATG GAG GGG ACA AGA CGT A-3′. 4b, mouse tail fragments were boiled with a solution containing 50 mM Tris-HCl (pH 7.5), 100 mM NaCl, 1 mM EDTA, 0.5% Tween20, and 0.5 mg /mL Proteinase K (Nacalai) at 55 °C for 20 h. Using the resulting supernatant, the target DNA sequence was amplified with KOD FX Neo (TOYOBO). Primary Abs for immunostaining; anti-Tomm20 (Abcam, ab78547; 1:500), anti-LAMP1 (BD Bioscience, 553792; 1:500), rat anti-GFP (Nacalai, 04404-84; 1:200–1:500), mouse anti-GFP (Invitrogen, A-11120; 1:1000), anti-Rab7 (Cell Signaling Technology, 9367; 1:100), anti-OXPHOS complex (Invitrogen, 45-8099; 1:200), anti-FKBP8 (R and D systems, MAB3580; 1:500), anti-HA-tag (Cell Signaling Technology, 3724; 1:200), and anti-HA-tag (BioLegend, 16B12; 1:500). Primary Abs for immunoblotting; anti-PDZD8 (Hirabayashi et al.8; 1:500), anti-HA-tag (BioLegend, 901501; 1:2000), anti-FKBP8 (R and D systems, MAB3580; 1:500), anti-VAPA (Bethyl laboratories, A304-366A; 1:1000), anti-Mfn2 (Abcam, ab56889; 1:1000), anti-β-actin (Cell Signaling Technology, 4967; 1:500), anti-FLAG (M2) (Sigma-Aldrich, F1804; 1:1000), anti-α-tubulin (Sigma-Aldrich, T6188; 1:1000), anti-GFP (Medical & Biological Laboratories, 598; 1:1000), anti-His-tag (Medical & Biological Laboratories, D291-3S; 1:1000), and anti-v5 (Abcam, ab27671; 1:500). The supernatants were boiled with 1× Laemmli's sample buffer containing 10% mercaptoethanol at 98 °C for 5 min. The cell lysates were fractionated by SDS-PAGE on a 10% gel or a 4–15% gradient gel (Bio-Rad) and the separated proteins were transferred to a polyvinylidene difluoride membrane (Merck). The membrane was incubated first with primary Abs for 24 h at 4 °C and then with HRP–conjugated secondary Abs (GE Healthcare) for 1 h at room temperature. After a wash with TBS-T (50 mM Tris-HCl (pH 8), 150 mM NaCl, and 0.05% Tween 20), the membrane was processed for detection of peroxidase activity with chemiluminescence reagents (100 mM Tris-HCl (pH 8.5), 1.25 mM Luminol, 0.2 mM P-coumaric acid, 0.01% H2O2) and the signals were detected by Image Quant LAS4000 instrument (GE Healthcare). Pdzd8-3× HA knock-in mice and control littermates at postnatal 10 days were put to sleep using medetomidine hydrochloride (Domitor, Nippon zenyaku kogyo, 0.75 mg/kg), midazolam (Sandoz, 4 mg/kg) and butorphanol (Vetorphale, Meiji Seika Pharma Co., Ltd., 5 mg/kg). Pups were then put on the ice for 5 min and exsanguinated by terminal intracardial perfusion with ice-cold 2% paraformaldehyde (Merck) in phosphate-buffered saline (PBS). The neocortex was then removed and sonicated five times for 30 s with ice-cold lysis buffer (50 mM Tris-HCl (pH 7.5), 1 mM EDTA, 0.2% Triton-X100, PhosSTOP phosphatase inhibitor (Roche) and cOmplete protease inhibitor cocktail (Roche)). The supernatants were incubated in rotation at 4 °C for 20 h with a protein complex of anti-HA antibody (Cell signaling technology, C29F4) and Sera-Mag SpeedBeads Protein A/G (Cytiva). After the rotation, beads were washed three times with TBS buffer. The immunoprecipitates were eluted from beads by incubating in 2× Laemmli's sample buffer containing 10% mercaptoethanol at 98 °C for 10 min and then subjected to immunoblotting. Total fraction samples were prepared using 2% of the cell extracts. Cells were fixed with 0.1% PFA for 10 min at room temperature, and 100 mM glycine–NaOH was treated for 4 min at RT. Cell extracts were incubated on ice for 15 min, then insoluble pellets and supernatants were separated by centrifugation at 15,000 × g at 4 °C for 15 min. The supernatants were incubated in rotation at 4 °C for 20 h with a protein complex of anti-GFP antibody (MBL) and Dynabeads Protein A (Thermo Fisher Scientific). After the rotation, beads were washed three times with TBS buffer. The immunoprecipitates were eluted from beads by incubating in 2× Laemmli's sample buffer, then mercaptoethanol was added at the final concentration of 9%. Samples were boiled at 98 °C for 5 min and then subjected to immunoblotting. Total fraction samples were prepared using 1.5% of the cell extracts. Pdzd8f/f::CreERT2 MEFs were treated with 1 μM 4-OH tamoxifen for 24 h and then transfected with plasmids encoding 3× FLAG-tagged full-length PDZD8/deletion mutants and HA-tagged FKBP8. Twenty-four hours post transfection cells were lysed with ice-cold lysis buffer (50 mM Tris-HCl (pH 7.5), 1 mM EDTA, 0.2% Triton-X100, 1 mM Na3VO4 and cOmplete protease inhibitor cocktail (Roche)), and insoluble pellets and supernatants were separated by centrifugation at 15,000 × g at 4 °C for 15 min. The supernatants were incubated in rotation at 4 °C for more than 3 h with a protein complex of anti-HA antibody (Cell signaling technology) and Dynabeads Protein A (Thermo Fisher Scientific). After the rotation, beads were washed twice with TBS-T buffer and once with TBS buffer. The immunoprecipitates were eluted from beads by incubating in 2× Laemmli's sample buffer, then mercaptoethanol was added at the final concentration of 9%. Samples were boiled at 98 °C for 5 min and then subjected to immunoblotting. Total fraction samples were prepared using 20% of the cell extracts. The pCAX–Cas9 and gRNA backbone vector (YT210) were generously provided by Matsuzaki49. To enhance knockout (KO) efficiency, three gRNAs targeting different exons were designed for the CRISPR/Cas9-based KO system50. Target sequences were amplified using forward and reverse oligonucleotides through PCR and subsequently cloned into the gRNA backbone vector at the AflII restriction sites49. Cells were fixed with 4% paraformaldehyde for 15 min at 37 °C, permeabilized with 90% methanol in PBS for 20 min under −20 °C and incubated for 20 h (Supplementary Fig. 7a) in PBS containing 2% FBS and 2% BSA (blocking buffer) at room temperature. They were then exposed at room temperature first for 1 h to primary Abs in blocking buffer and then for 30 min to Alexa Fluor-conjugated secondary Abs (Thermo Fisher Scientific) in blocking buffer. ProLong Gold (Thermo Fisher Scientific) was used as a mounting medium. All equipment was controlled via NIS Elements software (Nikon). Optical sectioning was performed at Nyquist for the longest wavelength. The resulting images were deconvoluted with NIS-elements (Nikon) and processed with NIS-elements (Nikon) or ImageJ (NIH). S7c, the images were taken as z-stack images (interval; 100 nm) and then 3D-deconvoluted with NIS-elements (Nikon) to enhance resolution. All analyses of PDZD8 or FKBP8 localization in fluorescent images were conducted using homemade programs written with Python, as detailed below. All binarized images were created using OpenCV's threshold function. 1c, d, mitochondrial area, lysosomal area, or late endosomal area were defined by binarizing signals of Tomm20/OXPHOS, Lamp1, or Rab7, respectively, and then the percentages of PDZD8 intensity on mitochondria, lysosome or late endosome were calculated. 3g, h, the mitochondrial areas were defined by binarizing signals of Tomm20 with global thresholding and then the percentage of PDZD8 intensity on mitochondria (Mander's coefficient, M1) was calculated. 5b–e, mitochondrial areas were defined by binarizing signals of Tomm20. 5b, to calculate Mander's coefficient between FKBP8 and Tomm20, the sum of FKBP8 intensity on mitochondria divided by total FKBP8 intensity was calculated. 5c, using OpenCV's connectedComponentsWithStats function, the puncta of PDZD8 and FKBP8 were segmented in the binarized images of PDZD8 and FKBP8, respectively, and then obtained the centroids of individual puncta. To obtain scrambled images of FKBP8, the pixels of the image mapping FKBP8 centroids on mito or off mito regions were shuffled in the corresponding area using the random module of Python. 5d, to calculate Mander's coefficient between PDZD8 and FKBP8 on mitochondria, the sum of PDZD8 intensity on FKBP8-present mitochondrial regions divided by total PDZD8 intensity on mitochondria was calculated. The scrambled images of FKBP8 were created by shuffling the pixels of FKBP8 channel within mitochondrial area using the random module of Python. 5e, FKBP8-present or FKBP8-absent mitochondrial areas were defined as ROIs using binarized images of Tomm20 and FKBP8, and then the sum of PDZD8 intensity at ROIs divided by the area of ROIs was calculated. S7d, e were performed using the same methods as in Fig. The mitochondrial area was defined as binarized images of YFP-ActA with Otsu's method and then the percentage of PDZD8 intensity on mitochondria (Mander's coefficient, M1) was calculated. Cells were fixed with 4% paraformaldehyde for 15 min at 37 °C. The fixed cells were mounted by ProLong Gold (Thermo Fisher Scientific). Images were acquired on a Nikon Ti2 Eclipse microscope with a Nikon AX confocal microscopy with a Nikon Spatial Array Confocal (NSPARC) detector and a CFI Plan Apochromat Lambda D 100× Oil (NA 1.45). For the expression of FLAG-tagged human PDZD8 (1, 28–), human PDZD8 sequences were cloned into the pCAG vector. Recombinant human PDZD8 (1, 28–)—FLAG was expressed in Expi293 Cells (Thermo Fisher Scientific) using ExpiFectamine 293 Transfection Kit (Thermo Fisher Scientific) according to the manufacturer's protocol. The cells were cultured for 4 days after transfection at 37 °C and 8% CO2. The Expi293 cell pellets were homogenized with lysis buffer (25 mM Tris-HCl (pH 8.0) 150 mM NaCl) and centrifuged at 40,000 × g for 30 min at 4 °C. The supernatant was filtered through a 0.8-μm pore-size filter and subsequently applied to a DDDDK-tagged protein purification gel (MBL) equilibrated with the lysis buffer. After washing with the lysis buffer once, the FLAG-tagged proteins were eluted with 1 M l-arginine-HCl (pH 4.4). The dialyzed fraction was subjected to size-exclusion chromatography in a HiLoad 16/600 Superdex 200 pg column equilibrated with the SEC buffer in an AKTA system (GE Healthcare). The purified fractions were concentrated using Amicon Ultra-15 (Cut off: 100 kDa) Centrifugal Filter Units (MERCK). For expressing GST-tagged human PDZD8 (1, 28–506)—HA, human PDZD8 sequences were cloned in pGEX4-T-1 vector (Cytiva) and transformed into Escherichia coli BL21 (DE3) cells. Then 0.5 mM of IPTG was added into the LB medium and incubated at 20 °C. 16–20 h after IPTG induction, the cells were collected by centrifugation (8000 × g 10 min 4 °C), frozen by liquid nitrogen, and stored at −30 °C. The cell lysate was centrifuged (40,000 × g for 30 min) at 4 °C. For the elution of hPDZD8 (1, 28–506)—HA, the Glutathione Sepharose beads were treated with a buffer (20 mM Tris pH 7.5 containing 150 mM NaCl, 5 mM DTT) supplemented with 0.04 U/µL Thrombin (Cytiva) for 2 days. For the expression of His-tagged human FKBP8 (1–380), pRSETA-hFKBP8 (1–380)—Histag (a kind gift from Dr. Chrisostomos Prodromou53) was transformed into Escherichia C43 (DE3) cells. After culturing for 24 h at 37 °C, the cells were incubated at 28 °C until the OD600 reached 0.6–1.0. Then, 0.5 mM of IPTG was added to the LB medium and incubated at 20 °C. 16–20 h after IPTG induction, the cells were collected by centrifugation (8000 × g for 10 min at 4 °C), frozen in liquid nitrogen, and stored at −30 °C. The frozen pellet was homogenized with lysis buffer (20 mM Tris-HCl pH 7.5, 100 mM NaCl, 0.5 mM imidazole, Benzonase diluted at 1:10,000) and centrifuged at 40,000 × g for 30 min at 4 °C. The supernatant was filtered through a 0.8-μm pore-size filter and subsequently loaded onto a TALON Metal Affinity Resin (Clontech) equilibrated with the lysis buffer. The dialyzed fraction was subjected to size-exclusion chromatography in a HiLoad 16/600 Superdex 200 pg column equilibrated with the SEC buffer in an AKTA system (GE Healthcare). The interactions of hPDZD8 (1, 28−)—FLAG with hFKBP8 (1–380)—Histag were analyzed using SPR in a Biacore T200 instrument (Cytiva). A Series S CM5 Biacore sensor chip (Cytiva) was activated with N-hydroxysuccinimide/N-ethyl-N′-(3-dimethylaminopropyl) carbodiimide hydrochloride, followed by immobilization of hPDZD8 (1, 28–)—FLAG at 618 resonance units. Binding analysis was performed at 25 °C in a running buffer of HBS-T (10 mM HEPES-NaOH, pH 7.4, 150 mM NaCl, and 0.005% (v/v) Tween-20). A series of five 2.5-fold dilutions of the FKBP8 solution was injected into the sensor chip at 30 μL/min, with a contact time of 120 s and a dissociation time of 120 s. The KD values were calculated with the Steady State Affinity model on Biacore T200 Evaluation Software, version 3.2 (Cytiva). The 95% CI of the KD value was calculated with Saturation binding analysis on Prism 10 (GraphPad Software). The eluate was subjected to SDS-PAGE and processed for Western blotting with anti-Histag and anti-HA antibodies. As a result, the IP fraction primarily contained hPDZD8 (1, 28–506)—HA, which has an estimated protein size of 55 kDa based on its amino acid sequence, rather than GST-hPDZD8 (1, 28–506)—HA, which has an estimated size of 81 kDa. Anti-HA antibodies (C29F4, from Cell Signaling Technology, Cat# 3724) were mixed with Dynabeads protein A (Thermo Fisher Scientific, Cat# 10001D, Lot#2791319) in a buffer (20 mM Tris pH 7.5 containing 150 mM NaCl, 1 mM EDTA, 5 mM DTT) with 100 mg/mL BSA and 0.4% Triton-X100 overnight at 4 °C. Finally, mercaptoethanol was added at a final concentration of 9%. 4 and 8, the cells were fixed with 2.5% glutaraldehyde (Electron Microscopy Sciences) in DMEM for 1 h at 37 °C. After being washed with 0.1 M phosphate buffer (0.02 M sodium dihydrogenphosphate dihydrate, 0.08 M disodium hydrogenphosphate), the cells were scraped and collected with 0.2% BSA/0.1 M phosphate buffer followed by centrifugation at 820 × g. After being embedded in low melting agarose (2% in 0.1 M phosphate buffer, MP Biomedicals), cell pellets were sectioned at 150-µm thickness with a Leica VT1000S vibratome. The sections were post-fixed with 1% OsO4 (Electron Microscopy Sciences) and 1.5% potassium ferrocyanide (FUJIFILM Wako Pure Chemical Corporation) in a 0.05 M phosphate buffer for 30 min. After being rinsed for 3 times with H2O, the cells were stained with 1% thiocarbohydrazide (Sigma-Aldrich) for 5 min. After being rinsed with H2O for 3 times, cells were stained with 1% OsO4 in H2O for 30 min. After being rinsed twice with H2O at room temperature and 3 times with H2O at 50 °C, the cells were treated with Walton's lead aspartate (0.635% lead nitrate (Sigma-Aldrich), 0.4% aspartic acid (pH 5.2, Sigma-Aldrich)) at 50 °C for 20 min. The sections were followed by incubations in an ascending ethanol series (15 min each in 50% on ice, 70% on ice, and 10 min each in 90%, 95% ethanol/H2O at room temperature), 10 min in 100% ethanol 4 times and 60 min in butyl 2,3-epoxypropyl ether (Fujifilm Wako pure chemical corporation). This was followed by infiltration of Epok812 resin-butyl 2,3-epoxypropyl ether for 24 h at a 1:1 dilution. After incubating with 100% Epok812 resin for 4 h, followed by 2 h, the resin was cured at 40 °C for 12 h, followed by 60 °C for 48 h. Epok812 resin was made by mixing 7.5 g of MNA (Oken), 13.7 g of Epok812 (Oken), 3.8 g of DDSA (Oken), and 0.2 g of DMP-30 (Oken). Resin blocks were trimmed with a TrimTool diamond knife (Trim 45; DiATOME). 4, 50–80 nm thick ultra-thin sections made with a diamond knife (Ultra 45; DiATOME) were collected on a cleaned silicon wafer strip in a Leica Ultramicrotome (UC7). The ultra-thin sections were imaged with a scanning electron microscope (JSM7100F; JEOL). Imaging was done at 5 kV accelerating voltage, probe current setting 12, 1280 × 960 frame size, and 7.4-mm working distance, using the Backscattered Electron Detector. The final pixel size was a 7.8 nm square. 8, in the array tomography analysis, the resin blocks created above were serially sectioned further at a 50-nm thickness with a diamond knife (Ultra JUMBO, 45°; DiATOME) fitted in a Leica Ultramicrotome (UC7) to obtain a ribbon of 70–200 serial sections. The serial sections were imaged by a field emission scanning electron microscope (JSM-IT800SHL; JEOL) with the Array Tomography Supporter software (System in Frontier). Imaging was done at 3 kV accelerating voltage, 268 pA beam current, 2560 × 1920 frame size, 6.5 mm working distance, 32.0 × 24.0 µm field of view (pixel size is 12.5 nm) and 2.67 µs dwell time, using the scintillator backscattered electron detector. 4, electron micrographs were manually annotated using PHILOW software54. Mitochondria and ER in the vicinity of mitochondria were annotated and ER regions within 3 pixels (23.4 nm) of the mitochondrial periphery were defined as MERCS. One cell in the Pdzd8 cKO condition was excluded from the statistical analysis because it was considered an outlier in the ROUT test (Q = 0.1%). The numbers of analyzed cells are 33, 29, 39, and 34 cells from two independent experiments for the control, Pdzd8 cKO, Fkbp8 KD, and Pdzd8 cKO + Fkbp8 KD cells, respectively. 6d, the same analysis was performed, defining MERCS as ER regions located within 4, 5, 6, and 7 pixels (32.4, 39.0, 46.8, and 54.6 nm) from the mitochondrial periphery. The electron micrographs were aligned using the linear stack alignment with scale invariant feature transform (SIFT) Plugin, implemented in ImageJ (NIH). Mitochondria, whose entire volume is within the imaging volume, were semi-automatically annotated using Empanada software55 and PHILOW software54. 4, 9, 2, and 2 mitochondria in the control, Pdzd8 cKO, Fkbp8 KD, and Pdzd8 cKO + Fkbp8 KD condition were excluded from the statistical analysis because it was considered outliers in the ROUT test (Q = 0.1%) (Supplementary Data 5). 3D visualization of the mitochondrial structure was performed using Imaris version 9.6.0 (Bitplane). 293 T cells (BRC) were co-transfected with shuttle vectors (FUW-CreERT2-P2A-NeoR), HIV-1 packaging vectors Delta8.9 and VSV-G envelope glycoproteins, or shuttle vectors (pLKO-shFKBP8 or pLKO-scramble), LP1, LP2, and VSV-G using FuGENE transfection reagent (Promega, catalog no. Twenty-four hours after transfection, the media were exchanged with 8 mL of fresh DMEM supplemented with 10% FBS, and 1% penicillin–streptomycin, and 24 h later, supernatants were harvested, spun at 500 × g to remove debris and filtered through a 0.45 μm filter (Sartorius). The filtered supernatant was concentrated to 125 μL using an Amicon Ultra-15 (molecular weight cut-off 100 kDa) centrifugal filter device (Merck Millipore), which was centrifuged at 4000 × g for 60 min at 4 °C. Then, 100 μL of viral supernatants was added to each 6-well dish containing MEFs. Prior to imaging, cells were washed twice with PBS and then incubated in phenol red-free full DMEM supplemented with 10% FBS and 1% P/S for approximately 30 min. During imaging, cells were maintained at 37 °C in an incubation chamber (Tokai Hit). All equipment was controlled via NIS Elements software (Nikon). Optical sectioning was performed at Nyquist for the longest wavelength. The resulting images were deconvoluted with NIS-elements (Nikon). The resulting images were processed with NIS-elements (Nikon) or ImageJ (NIH). PDZD8-HaloTag knocked-in HeLa cells were incubated with 200 nM JF549 dye (Promega, catalog No. Three hours before the confocal imaging, three times PBS washes were performed and mediums were changed to JF549 free DMEM, then incubated at 37 °C. 1.5, high tolerance, Warner Scientific) were cleaned according to a previously described protocol58 and stored in dry, sterile, 35 mm tissue culture dishes sealed with parafilm until use within 3 months. Immediately before plating, coverslips were coated with 500 μM phenol red-free Matrigel (Corning) for 1 h at 37 °C. Coverslips were then washed once with sterile PBS before being overlaid with 2 mL of complete, phenol red-free DMEM. Simultaneously, 1 × 106 COS7 cells per sample were trypsinized and resuspended directly into a transfection cocktail made of 750 ng PrSS-mEmerald-KDEL, 500 ng mTagRFP-T2-Mito-7, and 250 ng of msPDZD8-HaloTag-N1 mixed with Fugene HD (Promega) according to the manufacturer's specifications. Cells were incubated in the suspension for 15 min at 37°, and then the entire mixture was plated onto the coverslip and incubated for 18–22 h until imaging. Immediately prior to imaging, coverslips were loaded into a custom imaging chamber, labeled for 1 min with 10 nM PA-JF64659 in OptiMEM (Gibco), and washed excessively (at least 5 times) with 10 mL of sterile PBS. Cells were then washed once with 10 mL of complete, phenol red-free medium and allowed to recover in 1 mL of complete, phenol red-free DMEM for 15 min before imaging. Single molecule imaging was performed using a custom inverted Nikon TiE scope outfitted with a stage top incubation system to maintain cells at 37 °C with 5% CO2 and appropriate humidity during imaging (Tokai Hit). Regions amenable to sptPALM (primarily the flat lamella of cells, 500 nm thick or less) were located by eye using the fluorescence of the ER label. Once a region with sufficiently flat ER was chosen, excitation was achieved using three fiber couples solid state laser lines (488 nm, 561 nm, 642 nm, Agilent Technologies) to illuminate the sample. The illumination by the 488 nm and 561 nm lines were adjusted for each sample to minimize the bleed through into the single molecule channel, but both were always kept beneath 50 μW (488 nm) and 150 μW (561 nm) total power into the back aperture. The single molecule channel was always collected with a constant 11.5 mW of 642 nm light introduced into the back aperture. Emission light was collected using a 100× α-plan apochromat 1.49 NA oil immersion objective (Nikon Instruments). The collected light was focused onto three simultaneously running, electronically synchronized iXon3 electron multiplying charged coupled device cameras (EM-CCD, DU-897; Andor Technology), using a MultiCam optical splitter (Cairn Research) and sequential 565LP and 647LP dichroic mirrors (Chroma) within the optical path. The three emission paths were additionally cleaned up with a 525/50 BP, 605/70 BP, and 705/60 BP filter (Chroma) to filter extra light in the system. The microscope was operated in sptPALM mode using only 128 × 128 pixel region on the camera (20.48 μm × 20.48 μm) to drive the system quickly enough to unambiguously track single proteins. Imaging was performed with 5 ms exposure times, and the final speed was monitored using an oscilloscope directly coupled to the system (mean frame rate ~95 Hz). Single molecule localizations were linked to form trajectories using the TrackMate plugin in Fiji. Linking parameters were experimentally selected for each data set to minimize visible linkage artifacts and identified by eye. The resulting trajectories were then projected on to the ER network and manually curated for linkages that are close in 2D space but prohibitively far in the underlying ER structure itself. The resulting trajectories were exported from TrackMate and analyzed in Matlab for subsequent analysis, as described elsewhere29. Probability analysis was performed largely as described elsewhere30, where a spatially-defined probability mass function was derived from the total number of localizations observed within a 30 s time window. Note that while enrichments in the probability can correspond to regions of slowed movement, there are several other potential explanations for this phenomenon. Thus, an orthogonal analysis approach was also used where localizations are grouped into Voronoi tessellations and a simple Langevin motion model was applied. The resulting best fit values for diffusion coefficients for each tessellation were then mapped, and in this context they were directly compared to the mean diffusion coefficient of PDZD8 in regions of ER far from the putative contact sites. PDZD8-Halotag knocked-in HeLa cells were overexpressed with plasmids encoding Venus-FKBP8N403K or YFP-ActA and then treated with 200 nM JF549 dye (Promega) for 20 h at 37 °C. After that, cells were washed with PBS twice and plated in No. 1S gridded coverslip-bottom dishes (custom made, based on IWAKI 3922-035; coverslips were attached inversed side), precoated with carbon by a vacuum coater and then coated with poly-d-lysine (Merck, catalog No. The cells were fixed with 2% paraformaldehyde (Electron Microscopy Sciences) in PBS at room temperature for 10 min and then washed with PBS. Fluorescence imaging was conducted using Nikon AX confocal microscopy with a Nikon Spatial Array Confocal (NSPARC) detector and a CFI Plan Apochromat Lambda D 100× Oil (NA 1.45). The cells were then fixed with 2.5% glutaraldehyde in 0.1 M phosphate buffer (pH 7.4) for 2 days at 4 °C. After washing with 0.1 M phosphate buffer, the cells were post-fixed with 1% OsO4 (Electron Microscopy Sciences), 1.5% potassium ferrocyanide (Fujifilm Wako pure chemical corporation, catalog No. 161-03742) in a 0.05 M phosphate buffer for 30 min. After being rinsed 3 times with H2O, the cells were stained with 1% thiocarbohydrazide (Sigma-Aldrich) for 5 min. After being rinsed with H2O for three times, the cells were stained with 1% OsO4 in H2O for 30 min. After being rinsed with H2O for two times at room temperature and three times with H2O at 50 °C, the cells were treated with Walton's lead aspartate (0.635% lead nitrate (Sigma-Aldrich), 0.4% aspartic acid (pH 5.2, Sigma-Aldrich)) at 50 °C for 20 min. The cells dehydrated with an ascending series of ethanol (15 min each in 50% on ice, 70% on ice, and 10 min each in 90%, 95% ethanol/H2O at room temperature, 10 min in 100% ethanol 4 times at room temperature) were embedded in epoxy resin (LX-112) by covering the gridded glass with a resin-filled beam capsule. LX-112 resin was made by mixing 4.85 g of NMA (Ladd Research Industries), 7.8 g of LX-112 (Ladd Research Industries), 2.35 g of DDSA (Ladd Research Industries), and 0.3 g of BDMA (Ladd Research Industries). Polymerization was carried out at 42 °C for 12 h and 60 °C for 72 h. After polymerization, the gridded coverslip was removed and the resin block was trimmed to a square of about 150–250 μm. The block was sectioned using an ultramicrotome (EM UC7, Leica) equipped with a diamond knife (Ultra JUMBO 45°, DiATOME) to cut 50 nm thick sections. The serial ultra-thin sections were collected on the cleaned silicon wafer strip and imaged with a scanning electron microscope (JSM-IT800SHL; JEOL). Imaging of the Venus-FKBP8N403K overexpressing cell was done at 2 kV accelerating voltage, 34.1 pA beam current, 5120 × 3840 frame size, 6.5 mm working distance, 12.8 × 9.6 µm field of view (pixel size is 2.5 nm) and 1.33 µs dwell time, using the Scintillator Backscattered Electron Detector. Imaging of the YFP-ActA-overexpressing cell was done at 2 kV accelerating voltage, 48.8 pA beam current, 2560 × 1920 frame size, 6.9 mm working distance, 12.8 × 9.6 µm field of view (pixel size is 5 nm) and 14.1 µs dwell time, using the Scintillator Backscattered Electron Detector. The images taken by confocal microscopy were processed with ImageJ (NIH). The electron micrographs were stitched by Stitch Sequence of Grids of Images Plugin and aligned using the Linear Stack Alignment with Scale Invariant Feature Transform (SIFT) Plugin, implemented in ImageJ (NIH). After that, pixel size of images from the Venus-FKBP8-overexpressing cell was converted 2.5 nm to 5 nm by OpenCV's resize function. Mitochondria and ERs in the vicinity of mitochondria in electron micrographs were semi-automatically annotated using Empanada software55 and PHILOW software54. ER regions within 5 pixels (=25 nm) of the mitochondrial periphery were defined as MERCS. Reconstructing segmented images of the electron micrographs to three-dementional images and overlaying it with fluorescence images were conducted using Imaris software (Bitplane). The z projection of electron micrographs was created using ImageJ (NIH). For the analysis of mitochondrial membrane curvature at PDZD8-localized MERCS, the arbitrary area was extracted from the z-projection images of 3D-reconstructed electron micrographs merged with fluorescence images and counted the number of PDZD8-localized MERCS with positive, negative, and neutral OMM using ImageJ. Cells were transiently transfected with FKBP8N403K-mScarlet plasmid for 24 h and then were transferred to cryo-EM grids (Quantifoil, 200-Au-mesh with carbon film) (pre-treated with 50 µg/ml fibronectin for at least 15 min) and incubated for 3 h. The cells were incubated with 5% glycerol in the media for 15 min right before freezing after which the media was exchanged with PBS and 5% glycerol, and the grids were plunge-frozen. A Leica GP2 was used for plunge freezing with double blotting each for 7 s. The grids were stored in liquid nitrogen for the following steps. First, the grids were imaged using a Zeiss LSM900 AiryScan and the Linkam cryo-stage to screen for overall quality and transfection efficiency. Second, 4 grids were milled in one session using an Aquilos2 FIB-SEM equipped with the Delmic IceShield. Thirty-five lamellae were generated with a thickness of around 170 nm and varying surface areas. These lamellae were loaded on a Titan Krios equipped with a K3 detector and BioQuantum energy filter. High-resolution medium-mag montages of lamellae were collected and manually inspected for MAM. Mitochondria and MAM were then targeted for high-resolution tilt series acquisition at a pixel size of 2.07 Å/pixel. Grids were loaded into the cassette with a lamella pre-tilt of −9°, thus the tilt series acquisition started at 9 degrees, with a target range of −42 to +60° on the grid (−51 to +51 on the lamellae) acquired in 3° increments in a dose-symmetric fashion using SerialEM61. The data collection was monitored live using Warp62. After cryo-ET data collection, the lamellae were imaged in the Zeiss AiryScan with a 100× objective. This was feasible since we discovered that the mScarlet tag survives TEM radiation. Tilt series alignment was done using AreTomo. Weighted back projection tomogram reconstructions were CTF-deconvolved using ISONET63. Initial Segmentation was done using TomoSegMemTV64 on the deconvoluted tomograms. The segmentations were corrected and labeled manually using Amira. In the OE condition, 73 out of 116 collected tomograms (63%) contained at least one contact site, while in the control, only 19 out of 49 collected tomograms (38%) contained at least one contact site. Full analysis was on 20 mitochondria–MAM associations for the OE condition, and 6 for the control. Pdzd8-3× HA knock-in mice and control littermates at postnatal 10 days were put to sleep using medetomidine hydrochloride (Domitor, Nippon zenyaku kogyo, 0.75 mg/kg), midazolam (Sandoz, 4 mg/kg) and butorphanol (Vetorphale, Meiji Seika Pharma Co., Ltd., 5 mg/kg). Pups were then put on the ice for 5 min and exsanguinated by terminal intracardial perfusion with ice-cold 2% paraformaldehyde (Merck) in phosphate-buffered saline (PBS). The neocortex was then removed and sonicated five times for 30 s with ice-cold lysis buffer (20 mM Hepes-NaOH pH7.5, 1 mM EGTA, 1 mM MgCl2, 150 mM NaCl, 0.25% Na-deoxycholate, 0.05% SDS, 1% NP40, Benzonase (Merck), PhosSTOP phosphatase inhibitor (Roche) and cOmplete protease inhibitor cocktail (Roche)). After the lysates were centrifuged at 20,000 × g for 15 min at 4 °C, the resulting supernatants were incubated for 3 h at 4 °C with a 2.5 µL slurry of Sera-Mag SpeedBeads Protein A/G (Cytiva) pre-incubated with 2.5 µL of anti-HA-tag rabbit mAb (Cell signaling technology, C29F4). The beads were washed four times with the lysis buffer and then twice with 50 mM ammonium bicarbonate. Proteins on the beads were digested by adding 200 ng trypsin/Lys-C mix (Promega) at 37 °C overnight. The resulting digests were reduced, alkylated, acidified, and desalted using GL-Tip SDB (GL Sciences). LC–MS/MS analysis of the resultant peptides was performed on an EASY-nLC 1200 UHPLC connected to an Orbitrap Fusion mass spectrometer through a nanoelectrospray ion source (Thermo Fisher Scientific). The peptides were separated on a C18 reversed-phase column (75 mm [inner diameter] x 150 mm; Nikkyo Technos) with a linear 4%–32% ACN gradient for 0–100 min, followed by an increase to 80% ACN for 10 min and final hold at 80% ACN for 10 min. The mass spectrometer was operated in data-dependent acquisition mode with a maximum duty cycle of 3 s. MS1 spectra were measured with a resolution of 120,000, an automatic gain control (AGC) target of 4e5, and a mass range of 375–1500 m/z. HCD MS/MS spectra were acquired in the linear ion trap with an AGC target of 1e4, an isolation window of 1.6 m/z, a maximum injection time of 35 ms, and a normalized collision energy of 30. Dynamic exclusion was set to 20 s. Raw data were directly analyzed against the SwissProt database restricted to Mus musculus using Proteome Discoverer version 2.5 (Thermo Fisher Scientific) with Sequest HT search engine for identification and label-free precursor ion quantification. The search parameters were as follows: (i) trypsin as an enzyme with up to two missed cleavages; (ii) precursor mass tolerance of 10 ppm; (iii) fragment mass tolerance of 0.6 Da; (iv) carbamidomethylation of cysteine as a fixed modification; and (v) acetylation of the protein N-terminus and oxidation of methionine as variable modifications. Peptides and proteins were filtered at a false discovery rate (FDR) of 1% using the Percolator node and Protein FDR Validator node, respectively. Normalization was performed such that the total sum of abundance values for each sample over all peptides was the same. PDZD8-TurboID knocked-in HeLa cells were plated into a 15 cm dish at the density of 2 × 106 cells/dish and cultured two overnight. The cells were treated with Biotin at the final concentration of 50 μM, and incubated for 6 h. The cells were washed twice with ice-cold Hepes-saline and lysed in 6 M guanidine-HCl (Wako) containing 100 mM HEPES-NaOH (pH7.5), 10 mM TCEP (Nacalai), and 40 mM chloroacetamide (Sigma). After heating and sonication, proteins (1.3 mg each) were purified by methanol–chloroform precipitation and resuspended in 200 μL of PTS buffer (12 mM SDC, 12 mM SLS, 100 mM Tris-HCl, pH 8.0)65. After sonication and heating at 95 °C for 10 min, the protein solutions were diluted 5-fold with 100 mM Tris-HCl, pH8.0 and digested with 13 μg of trypsin (Pierce) at 37 °C overnight. After heating at 95 °C for 10 min, the digested peptides were incubated with the ACN-prewashed Tamavidin 2-REV beads (FUJIFILM Wako) for 3 h at 4 °C. After washing five times with TBS (50 mM Tris-HCl, pH 7.5, 150 mM NaCl), biotinylated peptides were eluted for 15 min at 37 °C twice with 100 µL of 1 mM biotin in TBS. The combined eluates were desalted using GL-Tip SDB, evaporated, and redissolved in 0.1% TFA and 3% ACN. LC–MS/MS analysis of the resultant peptides was performed on an EASY-nLC 1200 UHPLC connected to an Orbitrap Fusion mass spectrometer. The peptides were separated on the C18 reversed-phase column with a linear 4–32% ACN gradient for 0–60 min, followed by an increase to 80% ACN for 10 min and final hold at 80% ACN for 10 min. The mass spectrometer was operated in data-dependent acquisition mode with a maximum duty cycle of 3 s. MS1 spectra were measured with a resolution of 120,000, an AGC target of 4e5, and a mass range of 375–1500 m/z. Dynamic exclusion was set to 10 s. Raw data were directly analyzed against the SwissProt database restricted to Mus musculus using Proteome Discoverer version 2.5 with Sequest HT search engine for identification and label-free precursor ion quantification. The search parameters were as follows: (i) trypsin as an enzyme with up to two missed cleavages; (ii) precursor mass tolerance of 10 ppm; (iii) fragment mass tolerance of 0.6 Da; (iv) carbamidomethylation of cysteine as a fixed modification; and (v) acetylation of the protein N-terminus, oxidation of methionine, and biotinylation of lysine as variable modifications. Peptides and proteins were filtered at a FDR of 1% using the Percolator node and Protein FDR Validator node, respectively. Normalization was performed such that the total sum of abundance values for each sample over all peptides was the same. 2g, the highly enriched proteins were selected according to the following criteria: log2 (fold change) > 1 and −log10 (p-value) > 1 for IP–MS and log2 (fold change) > 3, −log10 (p-value) > 1.25 for TurboID-MS. One flask per each triplicate samples were prepared for MS analysis. The cells were then treated with biotin (50 μM) in complete DMEM medium for 30 min at 37 °C. The labeled cells were washed three times with cold DPBS 30 min after the in situ biotinylation, following next cell lysis. The cells were lysed with 750 μl of 1× TBS (25 mM Tris, 0.15 M NaCl, pH 7.2) containing 2% SDS and 1× protease inhibitor cocktail. Lysates were clarified by ultrasonication (Bioruptor, diagenode) to physically break down nucleic acid such as DNA for 5 min three times with iced water bath. The 4 ml of cold acetone were added to the lysates and incubated at −20 °C for at least 2 h to 16 h. After first precipitation with acetone, the samples were centrifuged at 13,000 × g for 10 min at 4 °C and the supernatant was gently discarded. The pellets were reconstituted with 500 μl of 8 M urea containing 50 mM ABC, followed by measuring protein concentration using BCA assay. Samples were reduced by reducing agent, 10 mM DTT and alkylated with 40 mM IAA at 650 rpm for 1 h at 37 °C, respectively. Insoluble fractions were removed by centrifugation for 10 min at 10,000 × g. The 150 μl of Streptavidin magnetic beads were firstly washed with 1× TBS containing 2 M urea four times and then added to the digested solution. The binding with the beads were for 1 h room temperature, followed by washing the beads two times with 50 mM ABC containing 2 M urea. The washed beads were washed with pure water compatible with LC–MS and transferred to new protein lobind tubes (Eppendorf). For elution of biotinylated peptides from the beads, 150 μl of elution solution (80% ACN, 20% pure water, 0.2% TFA, and 0.1% formic acid) were incubated at 60 °C three times repeatedly. Total elution fractions were completely dried using a speedvac. The resulting peptides were analyzed by Q Exactive Plus orbitrap mass spectrometer (Thermo Fisher Scientific, MA, USA) equipped with a nanoelectrospray ion source. To separate the peptide mixture, we used a C18 reverse-phase HPLC column (500 mm × 75 μm ID) using an acetonitrile/0.1% formic acid gradient from 4 to 32.5% for 120 min at a flow rate of 300 nL/min. For MS/MS analysis, the precursor ion scan MS spectra (m/z 400–2000) were acquired in the Orbitrap at a resolution of 70,000 at m/z 400 with an internal lock mass. The 15 most intensive ions were isolated and fragmented by High-energy collision induced dissociation (HCD). All MS/MS samples were analyzed using the Sequest Sorcerer platform (Sagen-N Research, San Jose, CA, USA Sequest was set up to search the Mus musculus protein sequence database (86320 entries, UniProt, http://www.uniprot.org/) assuming the digestion enzyme stricttrypsin. Sequest was searched with a fragment ion mass tolerance of 1.00 Da and a parent ion tolerance of 10.0 ppm. Carbamidomethylation of cysteine was specified in Sequest as a fixed modification. Scaffold (Version 4.11.0, Proteome Software Inc., Portland, OR, USA) was used to validate MS/MS-based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 95.0% probability by the Scaffold Local FDR algorithm. Protein identifications were accepted if they could be established at greater than 99.0% probability and contained at least 2 identified peptide. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony. Proteins were annotated with Gene Ontology (GO) terms from the National Center of Biotechnology Information database (NCBI; downloaded November 1, 2019)67. All statistical analyses were performed in Prism 10 (GraphPad Software). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The MS proteomics data of Pdzd8-3× HA mice and PDZD8-TurboID HeLa cells generated in this study have been deposited in the ProteomeXchange Consortium via the jPOST partner repository with the dataset identifiers PXD052134 and PXD052135. The MS proteomics data of Pdzd8-v5-TurboID KI Neuro2a have been deposited to the PRIDE (Project accession: PXD052694). 8 have been deposited to the figshare website (https://doi.org/10.6084/m9.figshare.26501122) and EMPIAR, respectively. Source data are provided with this paper. Aoyama-Ishiwatari, S. & Hirabayashi, Y. Endoplasmic reticulum-mitochondria contact sites-emerging intracellular signaling hubs. Wu, H., Carvalho, P. & Voeltz, G. K. Here, there, and everywhere: the importance of ER membrane contact sites. & Balla, T. The functional universe of membrane contact sites. Hung, V. et al. Proteomic mapping of cytosol-facing outer mitochondrial and ER membranes in living human cells by proximity biotinylation. Cho, K. F. et al. Split-TurboID enables contact-dependent proximity labeling in cells. Kwak, C. et al. Contact-ID, a tool for profiling organelle contact sites, reveals regulatory proteins of mitochondrial-associated membrane formation. Hirabayashi, Y. et al. ER-mitochondria tethering by PDZD8 regulates Ca. Kornmann, B. et al. An ER-mitochondria tethering complex revealed by a synthetic biology screen. Hertlein, V. et al. MERLIN: a novel BRET-based proximity biosensor for studying mitochondria-ER contact sites. Shirane, M. et al. Protrudin and PDZD8 contribute to neuronal integrity by promoting lipid extraction required for endosome maturation. Hewitt, V. L. et al. Decreasing pdzd8-mediated mito-ER contacts improves organismal fitness and mitigates Aβ. Liu, Y. et al. PDZD8 augments endoplasmic reticulum-mitochondria contact and regulates Ca2+ dynamics and cypd expression to induce pancreatic β-cell death during diabetes. Liu, T. et al. PDZD8-mediated endoplasmic reticulum-mitochondria associations regulate sympathetic drive and blood pressure through the intervention of neuronal mitochondrial homeostasis in stress-induced hypertension. Hasegawa, S. et al. Organelle communication maintains mitochondrial and endosomal homeostasis during podocyte lipotoxicity. O'Hare, J. K. et al. Compartment-specific tuning of dendritic feature selectivity by intracellular Ca. Al-Amri, A. H. et al. PDZD8 disruption causes cognitive impairment in humans, mice, and fruit flies. Bharadwaj, R. A. et al. Genetic risk mechanisms of posttraumatic stress disorder in the human brain. Applying systems-level spectral imaging and analysis to reveal the organelle interactome. Helle, S. C. et al. Organization and function of membrane contact sites. & Voeltz, G. K. Structure and function of ER membrane contact sites with other organelles. Elbaz-Alon, Y. et al. PDZD8 interacts with Protrudin and Rab7 at ER-late endosome membrane contact sites associated with mitochondria. Distinct fission signatures predict mitochondrial degradation or biogenesis. & De Camilli, P. PDZD8 mediates a Rab7-dependent interaction of the ER with late endosomes and lysosomes. & Ji, W. K. PDZD8-mediated lipid transfer at contacts between the ER and late endosomes/lysosomes is required for neurite outgrowth. Pistor, S., Chakraborty, T., Niebuhr, K., Domann, E. & Wehland, J. The ActA protein of Listeria monocytogenes acts as a nucleator inducing reorganization of the actin cytoskeleton. de Brito, O. M. & Scorrano, L. Mitofusin 2 tethers endoplasmic reticulum to mitochondria. Obara, C. J. et al. Motion of VAPB molecules reveals ER-mitochondria contact site subdomains. High-density mapping of single-molecule trajectories with photoactivated localization microscopy. Cabukusta, B. et al. Human VAPome Analysis Reveals MOSPD1 and MOSPD3 as membrane contact site proteins interacting with FFAT-related FFNT motifs. Efficient proximity labeling in living cells and organisms with TurboID. Yeo, H. K. et al. Phospholipid transfer function of PTPIP51 at mitochondria-associated ER membranes. Structural basis of human PDZD8-Rab7 interaction for the ER-late endosome tethering. Selective escape of proteins from the mitochondria during mitophagy. A., Medina, M., Fuentes, D., Wiseman, R. L. & Grotjahn, D. A. Quantifying organellar ultrastructure in cryo-electron tomography using a surface morphometrics pipeline. Friedman, J. R. et al. ER tubules mark sites of mitochondrial division. Abrisch, R. G., Gumbin, S. C., Wisniewski, B. T., Lackner, L. L. & Voeltz, G. K. Fission and fusion machineries converge at ER contact sites to regulate mitochondrial morphology. Guo, Y. et al. Visualizing intracellular organelle and cytoskeletal interactions at nanoscale resolution on millisecond timescales. Yoo, S. M. et al. FKBP8 LIRL-dependent mitochondrial fragmentation facilitates mitophagy under stress conditions. Bhujabal, Z. et al. FKBP8 recruits LC3A to mediate Parkin-independent mitophagy. Vincent, A. E. et al. Quantitative 3D mapping of the human skeletal muscle mitochondrial network. Osman, C., Voelker, D. R. & Langer, T. Making heads or tails of phospholipids in mitochondria. John Peter, A. T., Petrungaro, C., Peter, M. & Kornmann, B. METALIC reveals interorganelle lipid flux in live cells by enzymatic mass tagging. Hamasaki, M. et al. Autophagosomes form at ER-mitochondria contact sites. Ran, F. A. et al. Genome engineering using the CRISPR-Cas9 system. Tanaka, H. et al. Peroxisomes control mitochondrial dynamics and the mitochondrion-dependent apoptosis pathway. Ohtsuka, M. et al. i-GONAD: a robust method for in situ germline genome engineering using CRISPR nucleases. Developing a de novo targeted knock-in method based on in utero electroporation into the mammalian brain. Sunagawa, G. A. et al. Mammalian reverse genetics without crossing reveals Nr3a as a short-sleeper gene. Concordet, J. P. & Haeussler, M. CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Naito, Y., Hino, K., Bono, H. & Ui-Tei, K. CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites. Blundell, K. L., Pal, M., Roe, S. M., Pearl, L. H. & Prodromou, C. The structure of FKBP38 in complex with the MEEVD tetratricopeptide binding-motif of Hsp90. Suga, S. et al. An interactive deep learning-based approach reveals mitochondrial cristae topologies. Conrad, R. & Narayan, K. Instance segmentation of mitochondria in electron microscopy images with a generalist deep learning model trained on a diverse dataset. Vincent, A. E. et al. Quantitative 3D mapping of the human skeletal muscle mitochondrial network. Kwon, S. K. et al. LKB1 regulates mitochondria-dependent presynaptic calcium clearance and neurotransmitter release properties at excitatory synapses along cortical axons. Unraveling trajectories of diffusive particles on networks. Bright photoactivatable fluorophores for single-molecule imaging. & Mizushima, N. Quantitative 3D correlative light and electron microscopy of organelle association during autophagy. Mastronarde, D. N. SerialEM: a program for automated tilt series acquisition on tecnai microscopes using prediction of specimen position. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Liu, Y. T. et al. Isotropic reconstruction for electron tomography with deep learning. Martinez-Sanchez, A., Garcia, I., Asano, S., Lucic, V. & Fernandez, J. J. Robust membrane detection based on tensor voting for electron tomography. Nishino, K., Yoshikawa, H., Motani, K. & Kosako, H. Optimized workflow for enrichment and identification of biotinylated peptides using tamavidin 2-REV for BioID and cell surface proteomics. Nesvizhskii, A. I., Keller, A., Kolker, E. & Aebersold, R. A statistical model for identifying proteins by tandem mass spectrometry. Ashburner, M. The Gene Ontology Consortium et al. Gene ontology: tool for the unification of biology. Heike Blockus, Tommy Lewis, and Seok-Kyu Kwon for their critical reading of the paper and members of the Hirabayashi lab for constructive discussions. We thank Chenxing Jiang and Machiko Tsumura for their technical support. Yoshibumi Yamaguchi (Hokkaido University), Masato Ohtsuka (Tokai University), Masayuki Miura (The University of Tokyo), and Makoto Matsuyama (Shigei Medical Research Institute) for the kind instruction of the iGONAD method, Dr. Luke Hammond (Columbia University) for the kind instruction of fluorescence image analysis, and Drs. Satoru Takahashi, Yoko Ishida, Chieko Saito, Ikuko Koyama-Honda, and Noboru Mizushima (The University of Tokyo) for the kind instruction of the CLEM method. We thank Yuki Umeda, Ryotaro Yamamoto, Taiki Uno, Dr. Satoshi Yamaguchi, and Dr. Akimitsu Okamoto (The University of Tokyo) for their assistance in the synthesis of Halotag ligands. Shigeo Okabe and Yuka Sato for the support in FE–SEM imaging. We thank Dr. Jonathon Nixon-Abell for performing pilot experiments with sptPALM of PDZD8 and assisting with establishing imaging and tracking conditions. This work was supported by JSPS KAKENHI under Grant Number JP20H04898 (Y.H. ), SECOM Science and Technology Foundation Research grant (Y.H. ), the Uehara memorial foundation research grant (Y.H. ), Joint Usage and Joint Research Programs by the Institute of Advanced Medical Sciences of Tokushima University (K.N., T.N., Y.S-S., Y.H., and H.K. We thank Dr. Chrisostomos Prodromou (University of Sussex) for providing the pRSETA-hFKBP8 (1–380)—Histag. Yuji Tsunekawa and Fumio Matsuzaki (RIKEN) for providing the YT210 plasmid. pENTR4-HaloTag (w876-1) was a gift from Dr. Eric Campeau (Addgene plasmid #29644; http://n2t.net/addgene:29644; RRID: Addgene_29644). Mohammadreza Paaran, Clint Potter & Bridget Carragher Present address: Chan Zuckerberg Imaging Institute, Redwood City, CA, USA Present address: Laboratory of Molecular Neurobiology, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, 113-0032, Japan Present address: Department of Neurosurgery, Stanford University School of Medicine, Stanford, CA, 94304, USA These authors contributed equally: Koki Nakamura, Saeko Aoyama-Ishiwatari, Takahiro Nagao. Koki Nakamura, Saeko Aoyama-Ishiwatari, Takahiro Nagao, Yudan Du, Shogo Suga, Masafumi Tsuboi, Makoto Nakakido, Kouhei Tsumoto & Yusuke Hirabayashi Simons Electron Microscopy Center, New York Structural Biology Center, New York, NY, 10028, USA Mohammadreza Paaran, Jake Johnston, Clint Potter & Bridget Carragher Janelia Research Campus, Howard Hughes Medical Institute, Ashburn, VA, 20147, USA Yui Sakurai-Saito, Makoto Nakakido, Kouhei Tsumoto & Yusuke Hirabayashi Columbia University Medical Center, New York, NY, 10032, USA Medical Proteomics Laboratory, The Institute of Medical Science, The University of Tokyo, Tokyo, 108-8639, Japan UNIST Central Research Facilities (UCRF), Ulsan National Institute of Science and Technology (UNIST), Ulsan, 44919, Korea Department of Neuroscience, Columbia University Medical Center, New York, NY, 10032, USA Mortimer B. Zuckerman Mind Brain Behavior Institute, New York, NY, 10027, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed cryo-ET analysis advised by C.P., B.C., and F.P. J.J. helped with the cryo-ET grid preparation and data processing. performed in vitro analysis under the supervision of M.N. acquired and analyzed the MS data. performed proximity labeling-MS experiment of Pdzd8-TurboID KI Neuro2a. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Nakamura, K., Aoyama-Ishiwatari, S., Nagao, T. et al. Mitochondrial complexity is regulated at ER-mitochondria contact sites via PDZD8-FKBP8 tethering. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250416204034.htm'>Strongest hints yet of biological activity outside the solar system</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 04:15:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Astronomers have detected the most promising signs yet of a possible biosignature outside the solar system, although they remain cautious. The researchers say between 16 and 24 hours of follow-up observation time with JWST may help them reach the all-important five-sigma significance. Their results are reported in The Astrophysical Journal Letters. Earlier observations of K2-18b -- which is 8.6 times as massive and 2.6 times as large as Earth, and lies 124 light years away in the constellation of Leo -- identified methane and carbon dioxide in its atmosphere. However, another, weaker signal hinted at the possibility of something else happening on K2-18b. "We didn't know for sure whether the signal we saw last time was due to DMS, but just the hint of it was exciting enough for us to have another look with JWST using a different instrument," said Professor Nikku Madhusudhan from Cambridge's Institute of Astronomy, who led the research. As K2-18b transits, JWST can detect a drop in stellar brightness, and a tiny fraction of starlight passes through the planet's atmosphere before reaching Earth. "This is an independent line of evidence, using a different instrument than we did before and a different wavelength range of light, where there is no overlap with the previous observations," said Madhusudhan. "It was an incredible realisation seeing the results emerge and remain consistent throughout the extensive independent analyses and robustness tests," said co-author Måns Holmberg, a researcher at the Space Telescope Science Institute in Baltimore, USA. Both molecules have overlapping spectral features in the observed wavelength range, although further observations will help differentiate between the two molecules. However, the concentrations of DMS and DMDS in K2-18b's atmosphere are very different than on Earth, where they are generally below one part per billion by volume. On K2-18b, they are estimated to be thousands of times stronger -- over ten parts per million. "Earlier theoretical work had predicted that high levels of sulfur-based gases like DMS and DMDS are possible on Hycean worlds," said Madhusudhan. "And now we've observed it, in line with what was predicted. Given everything we know about this planet, a Hycean world with an ocean that is teeming with life is the scenario that best fits the data we have." Madhusudhan says that while the results are exciting, it's vital to obtain more data before claiming that life has been found on another world. He says that while he is cautiously optimistic, there could be previously unknown chemical processes at work on K2-18b that may account for the observations. Working with colleagues, he is hoping to conduct further theoretical and experimental work to determine whether DMS and DMDS can be produced non-biologically at the level currently inferred. "The inference of these biosignature molecules poses profound questions concerning the processes that might be producing them" said co-author Subhajit Sarkar of Cardiff University. "Our work is the starting point for all the investigations that are now needed to confirm and understand the implications of these exciting findings," said co-author Savvas Constantinou, also from Cambridge's Institute of Astronomy. "It's important that we're deeply sceptical of our own results, because it's only by testing and testing again that we will be able to reach the point where we're confident in them," Madhusudhan said. While he is not yet claiming a definitive discovery, Madhusudhan says that with powerful tools like JWST and future planned telescopes, humanity is taking new steps toward answering that most essential of questions: are we alone? "Decades from now, we may look back at this point in time and recognise it was when the living universe came within reach," said Madhusudhan. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-58959-0'>Direct observation and control of non-classical crystallization pathways in binary colloidal systems</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 02:33:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Crystallization stands as a prime example of self-assembly. Elementary building blocks converge, seemingly adhering to an intricate blueprint, orchestrating order from chaos. While classical theories describe crystallization as a monomer-by-monomer addition, non-classical pathways introduce complexity. Using microscopic charged particles as monomers, we uncover the mechanisms governing the formation of ionic colloidal crystals. Our findings reveal a two-step process, wherein metastable amorphous blobs condense from the gas phase, before evolving into small binary crystals. These small crystals then grow into large faceted structures via three simultaneous processes: addition of free monomers from bulk, capture and absorption of surrounding blobs, and oriented attachment of other crystals. These complex crystallization pathways occur both in bulk and on surfaces across a range of particle sizes and interaction strengths, resulting in a diverse array of crystal types and morphologies. Harnessing our ability to tune the interaction potential through small changes in salt concentration, we developed a continuous dialysis approach that allows fine control over the interaction strength in both time and space. This method enables us to discover and characterize various crystal structures in a single experiment, including a previously unreported low-density hollow structure and the heteroepitaxial formation of composite crystal structures. Charged colloidal particles are well-known model systems for studying self-assembly, as their interactions can be finely tuned through surface charge density and surrounding ionic conditions. When oppositely charged colloidal particles interact, they can form ordered binary structures akin to atomic crystals1,2. While such electrostatically driven assemblies have been extensively investigated in the context of simple crystalline lattices3, the detailed mechanisms governing their nucleation and growth remain largely unexplored. Previous studies on binary colloidal crystals have primarily focused on equilibrium structures and phase behavior, often assuming direct monomer-by-monomer attachment4. However, accumulating evidence suggests that electrostatically driven crystallization follows more complex pathways, involving intermediate amorphous phases, aggregation-driven nucleation, and particle reorganization5,6,7,8,9,10,11,12. Despite theoretical predictions and computational models supporting these non-classical pathways8,13,14,15, direct observation has remained limited, particularly in systems where interaction potentials can be dynamically controlled16,17,18,19,20,21. Here, we investigate the formation of ionic binary colloidal crystals under conditions where electrostatic interactions dictate crystallization pathways. We use binary mixtures of oppositely-charged colloids coated by a neutral polymer brush as model ions, carefully observing their assembly behavior as they transition from homogeneous suspensions to fully-formed macroscopic crystals. The particles interact via a superposition of a screened Coulomb interaction between their surface charges and the steric repulsion arising from overlap of their associated polymer coatings2. The pair potential for these model ions features an attractive minimum whose depth increases monotonically with decreasing salt concentration in the system. This relationship is key to understanding how variations in salt concentration influence the assembly process from dispersed particles to structured macroscopic crystals, and has been validated through the close correspondence of crystallization behavior observed within simulations accounting for these effects2,3. To initiate crystallization, we first prepare both positively and negatively charged particles in the same, precisely controlled, salt solution. We then mix the two groups of particles in an approximately 1:1 stoichiometric ratio and immediately transfer the resulting mixture into an observation cell. The time-lapse images captured through bright-field microscopy in Fig. 1a clearly show a two-step crystallization pathway. This process, which is believed to play significant roles in a range of systems13,22,23,24,25,26, begins with the rapid formation of particle blobs - a condensed liquid-like phase containing both positive and negative colloidal ions. Crystal nucleation starts within these blobs, and, as shown in Supplementary Movie 1, the crystallization front becomes visibly distinguishable as it progresses through the blob, eventually rendering it fully crystalline. Using 3D confocal microscopy to image refractive index-matched particles3, we thoroughly characterize the distinct phases present in the samples over time, confirming the amorphous nature of the blobs and pinpointing the precise locations of nucleation events within them (Fig. The transient phases were further validated by imaging quenched samples with SEM (Fig. Crystal (C), liquid- (L), and gas-like (G) phases are labeled. The red dashed area highlights small blobs dissolving through Ostwald ripening. b Laser-scanning confocal microscopy images showing volumetric scans of a similar two-step nucleation process for CsCl-like colloidal crystals formed from fluorescently-labeled PFPMA colloids3. (right) Crystalline regions can be seen nucleating and growing within the liquid phase. c SEM image of a quenched sample, capturing the initial transient state during the crystallization process, where both blobs and crystallites are present. All scale bars are 10 μm. Following nucleation, the newly-formed crystallites transition into the growth phase, where they mature into faceted macroscopic crystals. This growth unfolds through a series of simultaneous mechanisms which are illustrated in detail in Fig. Sequential time-lapse microscopy images capturing the growth of CsCl-like colloidal crystals. a Well-defined faceted crystals formed via a classical nucleation pathway and growing through direct monomer-by-monomer addition from the gas phase (see also Supplementary Movie 2). b Similar crystals formed via a non-classical crystallization pathway, growing through the attachment and absorption of amorphous blobs (see also Supplementary Movie 3). c–d SEM imaging of fixed samples reveals detailed features during the blob absorption process such as well-developed CsCl crystals with particle blobs in the midst of absorption (c) and crystal facets with distinct step-edges (d), which correspond to the surface waves observed in growing crystals (Supplementary Movie 3). Certain crystals and blobs have been colored yellow and blue, respectively, for enhanced clarity. Similarly, in (d) some of the growing crystal planes have been false-colored, with white arrows indicating their growth direction. e Time-lapse optical microscopy (from Supplementary Movie 4) captures the oriented attachment of two CsCl-like colloidal crystals. The white arrow points to the contact region where the crystals locally melt and recrystallize, resulting in a single, seemingly defect-free crystal. All scale bars are 10 μm. The most intuitive growth mechanism is monomer-by-monomer addition, where individual particles from the gas phase add to the crystal one-by-one. 2a and Supplementary Movie 2, showing isolated crystals in contact with the gas phase growing at similar rates. Ostwald ripening serves as a second key mechanism. In this process, the exchange of particles between the blobs and the bulk solution leads to net growth of nearby crystals through monomer addition, without necessitating direct contact between the crystals and the blobs (Fig. A third mechanism comes into play when crystals do make physical contact with blobs. This process manifests in experiments through the rapid deflation of the connected blob (Fig. These waves are reminiscent of wetted step-edge crystal growth, where particles from a liquid phase deposit onto the edges of crystal steps and then integrate into the lattice27. Bright-field microscopy provides a comprehensive view of the entire process dynamics, while SEM images—captured from quenched samples—provide snapshots at single-particle resolution (Fig. The fourth mechanism involves instances where small crystals come together and fuse in a specific, orientation-dependent manner to form a larger crystal (Supplementary Movie 4). This process differs from the more common random aggregation leading to polycrystalline structures; here, the particles align according to their crystallographic axes before merging, thereby maintaining a common crystallographic orientation across the newly formed, larger crystal28,29,30. During these oriented attachment events, we frequently observe the melting and subsequent re-crystallization of the contact region between the two crystals. While our colloidal model system allows us to observe these processes with high precision, capturing the very earliest steps of crystal formation remains challenging. We therefore employ our previously developed computational model2,3 to confirm that these mechanisms emerge naturally from our proposed interaction potential. We perform Molecular Dynamics (MD) simulations of 210 nm and 170 nm particles interacting via a pair potential that accounts for the combination of surface charge and polymer brush coating on the particles used in the experiment (details in Methods). As described throughout the rest of this article, these simulations recapitulate crystallization via a two-step process and subsequent crystal growth through Ostwald ripening and blob absorption, both on surfaces and in bulk, depending on the particle concentration and solution salt concentration. A representative example of the formation of CsCl-like crystals can be seen in Supplementary Movie 1. Here, nucleation occurs from within these blobs, often starting from the surface where interfacial fluctuations more easily allow for transient changes in density. For lower interaction strengths, particles can rearrange resulting in blobs that are spherical to minimize surface tension, but for higher interaction strengths we also observe more dendridic gel structures14, which then crystallize through nucleation within the dense phase, followed by propagation as in Fig. Absorption of neighboring blobs is common, especially at higher density where many small blobs nucleate initially. For some of these cases, we tracked the particles in the blob being absorbed and found that the vast majority of those in the final crystal arrived via direct flow rather than through evaporation from the blob then subsequent deposition. To assess the generality of this two-step crystallization process, we conducted crystallization experiments varying the size ratio (β) between positive and negative particles. As shown in Supplementary Fig. 1a, we observed no substantial differences in the crystallization mechanism, although the resulting crystal structures varied as expected31. Specifically, we examined systems with β = 0.36, 0.43, 0.50, 0.74, and 0.81, leading to the formation of Cs6C60–, NaCl–, K4C60–, Th3P4–, and CsCl–like structures, respectively. We also investigated if our results were influenced by gravitational settling by conducting crystallization experiments under density-matching conditions. This was achieved by using a H2O/D2O mixture as the solvent, which made the polystyrene particles buoyancy-neutral. As we show in Supplementary Fig. 1b, we again observed no substantial differences in the crystallization behavior compared to the density-mismatched samples, which was predicted based on the observation of non-classical crystallization mechanisms in simulations of bulk assembly. We hypothesized that the strength and range of particle interactions plays a crucial role in modulating the crystal formation pathways we observed, a phenomenon recognized in a growing number of particle systems11,32,33,34,35,36. To investigate this, we first conducted a series of simulations and experiments in which crystals were allowed to form in sealed capillaries at different fixed values of Debye length (λD). Both approaches revealed similar qualitative trends: at short λD, the samples remained in a gaseous state. As λD increased, we observed a narrow window where classical crystallization occurred, followed by a regime of two-step crystallization, and finally, at even higher λD, random aggregation (Fig. a Experimental behavior of oppositely charged 210 nm (+) and 190 nm (−) diameter PS particles mixed at various volume fractions (ϕp) and interaction energy minima (Eb). As the salt concentration increases (decreasing Eb), the assembly behavior transitions from forming amorphous aggregates (blue triangles) to polycrystal networks (green squares) and finally to well-separated crystals (green/yellow diamonds) of increasing sizes. Further increasing the salt concentration results in a stable gas phase (red circles). Most samples crystallize via a non-classical pathway (green area), but a small window of Eb and ϕp leads to classical crystallization behavior (yellow area). Simulations performed with settings mirroring experimental conditions show the same qualitative trend as experiment, albeit requiring slightly stronger interactions to observe nucleation within the fixed amount of available simulation time. Experimentally, we observe that with increasing ϕp, the boundaries between the different observed final states and assembly processes shift toward less negative Eb values. The initial states were captured approximately 5 minutes after sample preparation and the final states were captured after 48 hours. c Representative snapshots from simulations of systems with ϕc=1.0% and Eb = −6.00, −6.30, −6.50, −6.70, −7.20, and −68.52 kBT, showing similar trend as in experiments. Next, we concentrated on designing experiments that allowed us to gradually, and in a controlled fashion, scan different particle interaction strengths over time. Because the interaction strength is highly sensitive to salt concentration, we modified our experimental setup to allow for gradual and continuous variation of the salt concentration. These experiments begin with a high salt concentration (short λD), maintaining the particles in a stable gas-like state. We subsequently reduce the salt concentration by connecting the observation cell to a deionized water reservoir (Fig. 5a, Methods), and allows us to monitor and capture images of the crystallization front (Fig. The gradual decrease of salt concentration enables us to pinpoint the relatively small window of interaction strengths where the formation of amorphous blobs is completely suppressed and where instead classical crystallization takes place. a Schematic of the experimental setup. A 50 mm long capillary, sealed at one end, is immersed in a 100 mm diameter Petri dish filled with deionized water. The Petri dish is mounted on an inverted microscope and monitored with air objectives. b Representative bright field images taken at regular intervals along the capillary at the end of the crystallization experiment, assessing crystal types, sizes, and quality as a function of distance from the open end. The zoomed-in images (bottom) show aggregates and small crystallites near the open end, progressively larger single crystals toward the middle, and a mixed region of different crystal types near the sealed end. a Overlay of observed crystallization behavior on top of predicted salt concentration within a capillary as a function of time and distance to the open end, as computed using COMSOL51 (see Supplementary Movie 5). The experimental data points represent the positions of the crystallization front over time, determined both macroscopically (black circles) and microscopically (white circles), as well as the end of crystal growth (gray circles). The solid lines represent concentration contours for 3.55 mM (white) and 3.40 mM (gray), corresponding to approximate bond energies of 6.5 and 7.0 kBT, respectively (see “Methods). d–f Bright-field (left) and SEM (right) images showing regions where one type of crystal dominates. These regions are typically found in the central part of the capillary (indicated by a star symbol). For this sample with a size ratio of 0.81, we observed and characterized CsCl–like (d), Th3P4–like crystals (e), and a third crystal form termed L3S4 (f) which is studied further in Fig. 8. g Representative microscopy image showing small crystallites and disordered aggregates typically found near the open end of the capillary (indicated by the square symbol). h A microscopy image shows various crystal structures coexisting near the closed end of the capillary (indicated by the circle symbol), including CsCl (I), Th3P4 (II), L3S4 (III), as well as two examples of heteroepitaxial crystals (IV, V), with one still unidentified (V) (Supplementary Movie 7). All scale bars in the bright field images are 50 μm, and in the SEM images, they are 10 μm. Continuous dialysis allows us to create well-defined, spatially and temporally variable interaction potentials between particles, enabling the identification of optimal conditions for crystal growth in a single experiment. For instance, in regions where the interaction strength increases very rapidly—akin to supercooling—crystal formation is interrupted, with crystalline assemblies showing small, defective, and irregular structures, along with visible presence of disordered aggregates. In contrast, in areas of the same sample where the interaction strength increases more gradually, crystals develop with well-defined polyhedral habits (Figs. Figure 5 provides a comprehensive experimental overview, where experimental observations are superimposed onto a simulated spatiotemporal map of interaction energies in the experiments. Continuous dialysis allows us to discover and spatially resolve crystal structures with similar nucleation energy barriers, which might be overlooked in experiments using static interaction potentials. For example, in sealed capillaries, binary mixtures with β = 0.81 primarily form CsCl–like crystals that take on macroscopic rhombic dodecahedral shapes (Figs. However, under continuous dialysis, we also observe crystals structurally similar to Thorium Phosphide (Th3P4) exhibiting distinct triakis tetrahedral habits, and this is also observed in simulation for certain values of λD and volume fraction (ϕp) (Figs. a Bright-field microscopy image of CsCl–like colloidal crystals, displaying rhombic dodecahedral habits. The crystals were assembled from a 1:1 mixture of 170 nm (+) PS and 210 nm (−) particles. b SEM images and the accompanying schematics (below) display crystals in various orientations, all exhibiting rhombic dodecahedral habits bounded by {110} planes. c SEM image (bottom) and rendering (top) of the crystal surface showing the same motif, with rows of negative particles (yellow lines) converging at the intersection vertices of three equivalent {110} planes (yellow points in b). d Potential energy per particle of the entire system and representative snapshots from MD simulations display classical (purple) and non-classical (green) crystallization of CsCl–like rhombic dodecahedral crystals from bulk under two different conditions. Classical crystallization (ϕp=0.01, λ=5.41 nm, ψ±=+70/−30 mV) exhibits as a sudden decrease in energy as a crystal rapidly grows after initial nucleation, whereas two-step nucleation through a liquid phase (ϕp=0.027, λ=5.15 nm, ψ±=+50/−50 mV) is characterized by a large dwell time in the condensed state before a crystal nucleus forms within the blob. e Distribution of Steinhardt-Dellago order parameter (\(\overline{{q}_{6}}\))52 for CsCl–like crystals (orange), amorphous blobs (blue), and for an intermediate structure (red). \({\overline{q}}_{6}\) in this case can clearly distinguish the state of each particle. f Volumetric confocal scan (left) and corresponding three-dimensional reconstruction (right) of a typical CsCl–like crystal. g Comparison between the theoretical X-ray diffraction pattern of a CsCl crystal and the patterns calculated from the particle coordinates extracted from confocal scans and simulations following Ref. a SEM images and the accompanying schematics (below) showing the characteristic truncated triakis tetrahedron habit of Th3P4–like crystals. All faces of the crystals consist of equivalent {112} planes, displaying the same particle arrangement. The crystals were assembled from a 1:1 mixture of 170 nm (+) PS and 210 nm (−) particles. b A zoomed-in SEM image of one crystal face, showing the complex particle motif with yellow lines highlighting the characteristic zig-zag pattern. d Time-lapse of snapshots from MD simulation showing formation of Th3P4–like crystals in bulk via a non-classical crystallization pathway, whereby particles condense into a blob and subsequently the crystal nucleates and grows within the blob. In the main panels, particles are displayed with diameters scaled by 20% to allow the crystalline order to be observable, while insets show the true particle sizes. Particles are colored according to their similarity to a Th3P4 neighborhood as in (f). e Reference distance distribution functions calculated for individual particles within a blob and Th3P4–like crystal formed and empirically identified in MD simulations. f Calculating the Kullback-Leibler divergence (KLD)53 for a particle relative to these two reference distributions allows us to classify and highlight particles as amorphous or Th3P4–like depending on if a particle is more similar (lower KLD) to amorphous or Th3P4. g Diffraction patterns calculated from coordinates of the final structure of in (d) and a natural Th3P4 crystal allow us to confirm the identity of this structure, following Ref. In addition to the Th3P4 crystals, we have discovered a previously unreported crystal structure that, surprisingly, forms in the same sample from the same building blocks and exhibits a distinctive needle-like morphology (Fig. 8a,b, this crystal features an unusual open structure with empty channels running through its entire length. By analyzing the distribution of distances between particles, we constructed the crystal's unit cell, which has a 3:4 ratio of large to small particles and a remarkably low volume fraction of 56%. Hereafter, we refer to this new crystal as L3S4. Unlike CsCl and Th3P4, L3S4 seems to nucleate only heterogeneously on the charged surface of the crystallization chamber, and simulations show that for sufficiently large surface charge the nucleation of L3S4 can be greatly enhanced relative to CsCl despite it less favorable bulk energy (Figs. So far, we have only observed L3S4 formation under dialysis conditions, whereas simulations reproducibly assembled L3S4 under static conditions, suggesting that its formation might occur only within a very narrow range of λD values. a (left) Bright-field microscopy image displaying the rod-like habit of the type (III) crystals shown in Fig. (right) SEM images showing particle arrangements in two characteristic planes of this crystal. b (top) MD simulations depict similar rod-like crystals nucleating on a negatively charged substrate. Cross-sections along the (100) and (010) planes reveal an unusual open structure with empty channels running through the entire length of the crystal. (bottom) The distribution of pairwise distances from simulations allows us to construct a detailed crystal unit cell with dimensions a = 254 nm, b = 509 nm, c = 923 nm, and a Cmmm (No. 65) space group symmetry with an L3S4 stoichiometry (where L stands for large and S for small particles). The particle coordinates within the unit cell, labeled from 1 to 5, are (0, 0, 0.28), (0, 0.5, 0.22), (0, 0.5, 0.5), (0.5, 0.21, 0.14), and (0.5, 0.29, 0.36), respectively. c Bright-field (left) and SEM (right) images illustrating the heteroepitaxial growth of thin triangular crystals on L3S4 substrates. d Similar heteroepitaxial growth is observed in simulations when the L3S4 crystal shown in (b) is used as a seed and allowed to grow in bulk. In this case, two types of epitaxial crystals are clearly observed: the same thin triangular crystals seen in (c) and a CsCl-type crystal. Within the L3S4 crystals, negative and positive particles are colored soft pink and lavender, respectively. To enhance visualization, the color scheme shifts to pink and azure for epitaxial crystal A and red and blue for epitaxial crystal B. All scale bars in the bright field images are 50 μm, and in the SEM images, they are 2 μm. a SEM images and MD simulation snapshot showing the coexistence of L3S4– and CsCl–like crystals nucleated on a negatively charged substrate using particles with size ratio of 0.81. b The potential energy per particle for the entire system, along with key snapshots, shows that high surface potentials (-50 mV) tend to favor the nucleation of L3S4 through a classical-like behavior, while lower surface potentials (−45 mV) promote the nucleation of CsCl-like structures via a two-step process. Key snapshots are highlighted with a bottom view of layers attached to the charged wall. c Final frames for the two simulations in (b) showing the resulting single crystals with L3S4 (top) and CsCl–like (bottom) structures. The surface potentials of -50 mV and -45 mV correspond to attractions between the wall and positive particles of 2.9 and 2.6 kBT, respectively. Both simulations used λD = 5.15 nm and similar ϕp. d The interaction between the first layer of L3S4 and the surface is more attractive than for CsCl–like structure, confirming that a charged substrate is crucial for L3S4 nucleation. This interaction considers the total energy of attraction between the wall and first-layer particles, attraction between the first and second layer particles, and repulsion between like-charged particles in the first layer. The reduction in repulsion is dominant due to the arrangement of positive particles. e L3S4 unit cell highlighting two characteristic distances between the small positive particles: long (red line) and short (yellow line). Also shown is a characteristic angle between three large negative particles (purple line). By examining the (100) plane of a larger crystal, these characteristic distances and angles form easily distinguishable patterns, revealing a ∣ABBA∣ stacking sequence of small positive particles with hollow channels between A − A or B − B layers and a zig-zag pattern of negative particles between A − B layers (purple line). While in the middle of our dialysis setup we observe regions where each one of these structures seem to be individually dominant (Fig. Since these structures had the most time available to nucleate at low salt concentration, we infer that the barriers to nucleation for these structures must be similar. So far, we have described how our dialysis setup allowed us to observe formation of numerous large binary single crystals with an array of forms for a system prepared with a fixed ratio of particle size, surface charge, and number ratio. Within our setup, we are also able to observe more complex hierarchical assemblies, in particular crystals formed through heteroepitaxial growth37 of one structure templated by another (e.g., Fig. 5h structures IV and V, Supplementary Movie 6). SEM images of these flags show that they have a sharp 80 degree corner, and a surface arrangement of particles that does not belong to the structure we determined for L3S4. To inspect this behavior in more detail, we seeded MD simulations with the large L3S4 crystal from Fig. After several rounds of continued addition of particles, we observed not one but two simultaneous instances of heteroepitaxial growth, one of which produced a structure commensurate with the flags observed in SEM, and one of which we identified as CsCl–like (Supplementary Movie 7). Using our full knowledge of particle positions in the MD simulation, we are able to demonstrate that the nucleation of these secondary crystals occurs due to commensurate arrangements of particles within the two different unit cells. This investigation shows that the (210) face of CsCl and (022) face of L3S4 are able to interlock, and that the flag structure seen in simulation can be formed by extending a rhombic subset of particles from the L3S4 unit cell (Supplementary Fig. The flag formed in MD simulations has a surface structure that closely matches the one observed in experiments. The compatibility of the CsCl unit cell with L3S4 may also explain why they are often observed in contact within the experiments. Another example of hetero-epitaxial growth can be observed in Fig. 5h (structure V) and Supplementary Fig. 3, where anisotropic crystals stem from a central seed, forming a repetitive branched structure. Further work is required to fully characterize these structures, as their extreme thinness and fragility make them prone to damage during the fixing process required for SEM imaging. We believe this previously unobserved variety of crystal structures, habits, and crystallization behaviors was obscured by our inability to precisely modulate particle interaction potentials in time and space. Our continuous dialysis setup addresses this limitation, unveiling a spectrum of crystallization phenomena not readily apparent under static conditions. These capabilities reveal the underlying mechanisms governing colloidal crystallization pathways and underscore the intricate interplay between particle interactions and crystallization outcomes. This approach lays the groundwork for developing precisely tunable control over colloidal crystallization, enabling the creation of materials with tailored microstructures. A particularly promising outcome of this work is the potential to control the formation of specific structures or polymorphs by creating charged patterns on the substrate. Low-density (ρ = 1.05g/cm3) polystyrene (PS) and low-refractive-index (n = 1.39) pentafluoropropyl methacrylate (PFPMA) colloids were synthesized via surfactant-free emulsion polymerization for bright-field microscopy and laser scanning confocal microscopy experiments, respectively3,38. For the synthesis of 200 nm diameter positively charged PS particles, a mixture of 550 mL deionized water and 3 mL styrene monomer (≥99% from MilliporeSigma) was stirred at 330 rpm in a 1 L three-neck round-bottom flask. After purging with nitrogen for 1 hour, 0.5 g of 2,2'-azobis(2-methylpropionamidine) dihydrochloride (AIBA) (97% from MilliporeSigma), dissolved in 10 mL deionized water, was injected into the mixture. The components were then heated to 60 °C and stirred at 330 rpm under nitrogen overnight. The particles were stabilized by adding 5 mL of 5 wt% Pluronic F108 solution and washed via repeated sedimentation and resuspension cycles. Finally, the particle suspensions were dialyzed against deionized water for one week using 50 kD Spectra/Por dialysis tubing, with daily water changes to ensure complete removal of salt. Negatively charged PS particles were synthesized similarly, replacing AIBA with an equivalent weight of potassium persulfate (KPS) (≥99% from MilliporeSigma). To prepare 375 nm diameter positively charged PFPMA particles, a mixture of 100 mL deionized water, 8 mL 2,2,3,3,3-pentafluoropropyl methacrylate monomer (SynQuest Labs), and 20 μL of (2-(methacryloyloxy)ethyl)trimethylammonium chloride comonomer (75 wt% in H2O from MilliporeSigma) was stirred at 330 rpm in a 250 mL three-neck round-bottom flask. After purging with nitrogen for 1 hour, 0.1 g AIBA, dissolved in 5 mL deionized water, was injected. The mixture was heated to 60 °C and stirred at 330 rpm under nitrogen for 12–16 hours. The particles were filtered (Whatman, pore size 20 μm) to remove aggregates, stabilized by adding 1 mL of 5 wt% Pluronic F108 solution, washed via sedimentation and resuspension, and dialyzed as for the PS particles. Negatively charged PFPMA particles were synthesized using the same procedure, excluding the comonomer and replacing AIBA with an equivalent weight of KPS. The sizes of PS particles and negatively charged PFPMA particles were controlled by varying the monomer concentration, achieving diameters ranging from 100 nm to 700 nm. For positively charged PFPMA particles, diameters from 150 to 400 nm were obtained by adjusting the comonomer-to-monomer ratio. Larger sizes required repeated seeded growth. Fluorescent labeling of PFPMA particles was done using the swell-deswell method39. BODIPY FL NHS ester and BODIPY 581/591 NHS ester dyes (ThermoFisher) were used for positively and negatively charged particles, respectively. Typically, 100 μL dye solution (1 mg/mL in toluene) mixed in 9 mL 60 vol% THF was added to 10 mL of particle suspension stabilized with 2.5 wt% Pluronic F108, resulting in a final concentration of 28.5 vol% THF. The mixture was swirled for 15 minutes and then rapidly diluted by a factor of 15. The dyed particles were transferred into deionized water through repeated sedimentation and resuspension cycles. The surface potential of each particle system was measured using a Malvern Zetasizer Nano ZS with DTS1070 folded capillary cells. Measurements were performed on highly dilute samples equilibrated in 10 mM NaCl. Zeta potentials were calculated using the Smoluchowski approximation (Henry's function f(κa) = 1.5), with refractive indices of 1.39 for PFPMA particles and 1.59 for PS particles. Oppositely charged PS particles were separately equilibrated with 0.1 wt% Pluronic F108 and 2–5 mM NaCl for 30 minutes. They were then mixed together while vortexing. The resulting mixture was promptly transferred into hydrophobized borosilicate glass capillaries (VitroCom, model 3520), which were then sealed on both ends with wax (Hampton Research) or epoxy resin (Norland 81). Capillaries were pre-cleaned in a Jelight Model 18 ultraviolet ozone (UVO) cleaner for 20 minutes. Subsequently, they were exposed to methyltrichlorosilane (99%, MilliporeSigma) vapor inside a moisture-free sealed chamber for 1 hour to hydrophobize the surface. After this treatment, the capillaries were washed with water and ethanol for three cycles and finally dried in an oven. This hydrophobization pre-treatment facilitates the formation of a polymer brush on the glass surface when in contact with a Pluronic F108 solution. Density-matched samples were prepared similarly, using 57% deuterium oxide (D2O) instead of pure water. Refractive index-matched samples were prepared using PFPMA particles suspended in 40% DMSO solutions. These samples were allowed to crystallize either in micro-wells (Thomas Scientific) or micro-channels (Ibidi USA). To determine the optimal crystallization conditions for each particle system, series of samples with varying NaCl concentrations were prepared, typically ranging from 2 to 5 mM in increments of 0.1 mM. The samples were monitored over several days to identify the ideal crystallization conditions. 221) were formed using particle size ratios ranging from 0.8 to 1.0 and a number ratio of 1:1. The Wulff shape of these crystals is a rhombic dodecahedron, bound by {110} planes, reflecting their equilibrium shape as determined by surface energy minimization (Fig. 220) were formed using 170 nm (+) PS and 210 nm (−) PS particles (β = 0.81), as well as 255 nm (+) PS and 190 nm (−) PS particles (β = 0.74), both at a number ratio of 1:1. The Wulff shape of these crystals is a triakis tetrahedron, bound by {112} planes (Fig. 225) were formed using 300 nm (+) PS and 130 nm (−) PS particles (β = 0.43) at a 1:1 number ratio. A precise size ratio of approximately \(\sqrt{2}-1\) is required to form NaCl crystals; otherwise, K4C60 is typically observed. The Wulff shape of these crystals is a cube, bound by {100} planes (Fig. a Bright-field microscopy and SEM images of K4C60–like crystals, along with accompanying schematics, display two orientations of rhombic dodecahedral habits. Zoomed-in SEM images show these two orientations, exhibiting (110) and (101) top surface planes. b MD simulation of the same particle system as in (a) at λD = 5.9 nm with a negatively charged substrate and high density (see “Methods”) produces crystals with these two orientations and characteristic top surface planes within the same simulation box. Distance distributions of these two crystals confirm an identical K4C60–like unit cell. c Example MD simulation in bulk yielding a K4C60–like crystal via a two-step crystallization mechanism at low density and λD = 5.3 nm (see “Methods”). e Bright-field microscopy and SEM images of NaCl–like crystals displaying cubic habits bounded by {100} planes. f Cs6C60–like crystals displaying rhombic dodecahedral habits bounded by {110} planes. 139) were formed using 350 nm (+) PS and 160 nm (−) PS particles (β = 0.46) at a 1:4 number ratio, as well as 200 nm (+) PS and 400 nm (−) PS particles (β = 0.50) at the same number ratio. The Wulff shape of these crystals is a rhombic dodecahedron, bound by distinct {110} and {101} planes, resulting in two types of orientations observed (Fig. 204) were formed using 525 nm (+) PS and 190 nm (−) PS particles (β = 0.36) at a 1:6 number ratio. In the unit cell of this cubic lattice, the larger particles are positioned at the body-centered cubic (bcc) lattice sites, specifically at coordinates (0, 0, 0) and (0.5, 0.5, 0.5). The smaller particles occupy all tetrahedral interstitial sites, located on each face, such as in the (001) plane at coordinates (0.5, 0.25, 0), (0.5, 0.75, 0), (0.25, 0.5, 0), and (0.75, 0.5, 0), displaced by  ±0.25 along the x and y axes from the face center. The Wulff shape of this crystal is a rhombic dodecahedron, bound by {110} planes (Fig. To perform experiments at variable interaction potentials, mixtures of oppositely charged PS particles with 4.5 mM NaCl and 0.1 wt% F108 were introduced into 50-mm-long glass capillaries (inner dimensions: 2.0 mm  × 0.2 mm, VitroCom) and sealed at one end. The capillaries were then immersed in 100 mm diameter Petri dishes filled with deionized water (Fig. During crystallization, each capillary was imaged using a Luxonis OAK-D S2 camera, recording a time-lapse with a 10-minute delay between frames (Supplementary Movie 5). Simultaneously, we tracked the crystallization front by taking microscopy images at regular intervals using a bright-field microscope equipped with a 10X air objective. Bright-field images and movies were acquired using a Leica DMI3000 inverted microscope equipped with differential interference contrast optics and high-resolution cameras: a grayscale Jenoptik Gryphax Rigel and a color FLIR Grasshopper3. Crystal nucleation was observed within hours using a 100X oil immersion objective. For continuous imaging of crystal growth over several days, sealed capillary samples were glued to a 100 mm Petri dish (FisherScientific), immersed in water at room temperature, and monitored using a 10X air objective with minimal light intensity to reduce heating. Time-lapse z-stacks of scanning fluorescent images were acquired using a Leica SP8 confocal microscope equipped with a 100X oil objective, recording the crystallization process of positively charged 375 nm and negatively charged 440 nm PFPMA particles in 40% DMSO at 30-minute intervals. The z-step size was set to 0.07 μm, allowing for 5 scans per particle to achieve precise tracking. To prevent dye bleaching during extended scanning, the lowest possible laser power (typically around 5% of the maximum) was used. The coordinates of oppositely charged particles were estimated from grayscale images of separated channels in the z-stack data using the TrackPy Python package40. These coordinates were imported into Blender, where sizes based on scanning electron microscopy were assigned to recreate the structure in 3D. X-ray diffraction (XRD) patterns shown in Fig. 6 were subsequently generated using Mercury software from the particle coordinates. Electron Microscopy imaging was performed on fixed samples. Crystals formed in sealed capillaries were fixed by immersing the capillaries in a deionized water bath and carefully removing the sealant (wax or epoxy). After a few days, ion exchange resin (AmberLite MB from MilliporeSigma) was added to the samples to remove all remaining ions from the system. This ion removal process lasted for another 3–4 days, during which the water bath was changed daily. To recover the fixed crystals, the capillaries were scored with a glass cutter and carefully broken underwater. The broken capillaries were taped onto SEM stubs (Ted Pella) using conductive carbon adhesive tabs (Electron Microscopy Science), air-dried, and coated with 3-5 nm of iridium using a Cressington 208HR high-resolution sputter coater. The samples were then imaged using a MERLIN field emission scanning electron microscope (Carl Zeiss). We estimated the temporal and spatial variations in salt concentration within the capillary using COMSOL Multiphysics. Based on the physical dimensions of the capillaries used in experiments (50 mm × 2 mm × 0.2 mm), we simulated the diffusion of NaCl ions using Fick's law: where Jj is the flux of salt, D is the diffusion coefficient of salt (1.61 × 10−9m2/s)41, and cj is the concentration of component j. The initial NaCl concentration throughout the capillary was 4.5 mM, with boundary conditions applied such that the interface at one end was fixed at 0 mM, and there was no flux across any other interfaces. The numerical solutions were calculated for 96 hours at 15-minute intervals. 5 were computed by converting the salt concentration at a given position and time to a λD in meters using the formula, To model the assembly of charged colloids, we performed Langevin dynamics simulations of binary mixtures of colloidal particles using the HOOMD-Blue software43, as previously implemented2,3 (see Code Availability Statement for run and analysis scripts). A total 6750 number of charged particles were placed in the simulation boxes in a simple cubic arrangement in a 1:1 ratio at varying ϕp. Unless otherwise indicated, the radii of the positive- and negatively charged particles were taken to be 85 nm and 105 nm, respectively. The temperature was maintained by a Langevin thermostat at 1 kBT. The drag coefficient, γ was set to 0.001. The screened Coulombic interaction between two particles is computed by the Derjaguin-Landau-Verwey-Overbeek (DLVO)42 theory, expressed as \({V}_{{{{\rm{E}}}}}({h}_{ij})=2\pi {k}_{{{{\rm{B}}}}}T\epsilon {a}_{ij}{\psi }_{i}{\psi }_{j}\exp (-{h}_{ij}/{\lambda }_{{{{\rm{D}}}}})\). Here, ϵ is the solvent permittivity which was set to 80, ψi and ψj are the surface potentials for the two particles which unless otherwise indicated were set to +50 mV and −50 mV for positive and negative particles, respectively, and λD is the Debye length. The repulsive interactions generated from the polymer brushes between two particles was computed by the Alexander-de Gennes model44,45,46,47 which is given by- L, the polymer brush length, is set to 10 nm for all simulations48. The simulations were performed at six different values of ϕp, and for each, unless otherwise mentioned, λD was varied from 5.1 to 5.3 nm. We performed 6 simulations in each condition. Repulsive walls, which interact with the particles by a shifted Lennard-Jones potential, were used to produce a simulation box, which is convenient for analysis but is not expected to effect results given we are studying assembly from a dilute gas phase. Simulations were carried out for 4 x 109 number of steps using a dimensionless time step of 0.005 unless otherwise specified. 3 used sizes and charges set to match experiments performed specifically for that figure. A 1:1 mixture of positive and negative particles was placed in a closed cubic box with ϕp=1.0%, and were simulated for 109 steps. The diameters of the positive and negative particles were 210 nm and 190 nm with surface potentials of +44 mV and −54 mV, respectively. λD was chosen to give attractive well depths from −5.6 kBT to −8.77 kBT, as well as −68.5 kBT, corresponding to λD values between 5.0 and 13.8 nm. All other settings were the same as described above. To study the effect of a charged substrate on the crystallization, we included a negatively charged wall at position \({{{{\rm{Z}}}}}_{\min }\) in the simulation box. The surface potential of the wall particles was varied from  − 45 mV to  −50 mV. In addition to simulating the size ratio of 0.81, we performed simulations with size ratio of 0.45 (rP = 176.5 nm and rN = 80 nm) and number ratio (positive to negative particles) of 1:2 to produce K4C60 crystals (see Fig. Simulations using experimentally realistic surface potentials of positive and negative particle, +30 mV and −50 mV, formed crystals on a surface with wall particles having a charge of −50 mV. 10b formed with λD=5.9 nm and volume fraction 0.0345. 10c was observed for λD=5.3 nm with surface charges of +/− 50 mV at volume fraction 0.005. All other parameters were same as described above. Simulation trajectories were visualized using Ovito software50. Source data are provided with this paper. Simulations used the publicly available code HOOMD-2.943. All scripts to run and analyze simulations have been deposited in the Zenodo database under acession code https://zenodo.org/records/15225217. All raw simulation data are available from an NYU resource linked to from our project's GitHub https://github.com/hocky-research-group/Zang-NonClassical-2025. Leunissen, M. E. et al. Ionic colloidal crystals of oppositely charged particles. & Sacanna, S. Ionic solids from common colloids. Zang, S., Hauser, A. W., Paul, S., Hocky, G. M. & Sacanna, S. Enabling three-dimensional real-space analysis of ionic colloidal crystallization. Hensley, A., Jacobs, W. M. & Rogers, W. B. Self-assembly of photonic crystals by controlling the nucleation and growth of dna-coated colloids. Bian, T. et al. Electrostatic co-assembly of nanoparticles with oppositely charged small molecules into static and dynamic superstructures. Martirossyan, M. M., Spellings, M., Pan, H. & Dshemuchadse, J. Local structural features elucidate crystallization of complex structures. Sanz, E., Valeriani, C., Frenkel, D. & Dijkstra, M. Evidence for out-of-equilibrium crystal nucleation in suspensions of oppositely charged colloids. Gispen, W. & Dijkstra, M. Kinetic phase diagram for nucleation and growth of competing crystal polymorphs in charged colloids. Bian, T. et al. Catalan solids from superionic nanoparticles. Coropceanu, I. et al. Self-assembly of nanocrystals into strongly electronically coupled all-inorganic supercrystals. Enhancing nanocrystal superlattice self-assembly near a metastable liquid binodal. ten Wolde, P. R. & Frenkel, D. Enhancement of protein crystal nucleation by critical density fluctuations. Sanz, E., Leunissen, M. E., Fortini, A., van Blaaderen, A. & Dijkstra, M. Gel formation in suspensions of oppositely charged colloids: Mechanism and relation to the equilibrium phase diagram. Fortini, A., Sanz, E. & Dijkstra, M. Crystallization and gelation in colloidal systems with short-ranged attractive interactions. Yoreo, J. J. D. et al. Crystallization by particle attachment in synthetic, biogenic, and geologic environments. & Cölfen, H. New horizons of nonclassical crystallization. Tsarfati, Y. et al. Crystallization of organic molecules: Nonclassical mechanism revealed by direct imaging. & De Yoreo, J. J. Non-classical crystallization in soft and organic materials. Dachraoui, W., Henninen, T. R., Keller, D. & Erni, R. Multi-step atomic mechanism of platinum nanocrystals nucleation and growth revealed by in-situ liquid cell stem. Ou, Z., Wang, Z., Luo, B., Luijten, E. & Chen, Q. Kinetic pathways of crystallization at the nanoscale. & Myerson, A. S. Nucleation of crystals from solution: classical and two-step models. Sanz, E. et al. Out-of-equilibrium processes in suspensions of oppositely charged colloids: liquid-to-crystal nucleation and gel formation. Lutsko, J. F. & Nicolis, G. Theoretical evidence for a dense fluid precursor to crystallization. Houben, L., Weissman, H., Wolf, S. G. & Rybtchinski, B. A mechanism of ferritin crystallization revealed by cryo-STEM tomography. Ma, W. et al. Nonclassical mechanisms to irreversibly suppress ß-hematin crystal growth. Direct measurements of island growth and step-edge barriers in colloidal epitaxy. Zhu, C. et al. In-situ liquid cell transmission electron microscopy investigation on oriented attachment of gold nanoparticles. Oriented attachment induces fivefold twins by forming and decomposing high-energy grain boundaries. B. V., van der Sluijs, M. M., Soligno, G. & Vanmaekelbergh, D. Oriented attachment: from natural crystal growth to a materials engineering tool. Shevchenko, E. V., Talapin, D. V., Kotov, N. A., O'Brien, S. & Murray, C. B. Structural diversity in binary nanoparticle superlattices. Marino, E. et al. Crystallization of binary nanocrystal superlattices and the relevance of short-range attraction. Hagen, M., Meijer, E., Mooij, G., Frenkel, D. & Lekkerkerker, H. Does c60 have a liquid phase? de Hoog, E. H. & Lekkerkerker, H. N. Measurement of the interfacial tension of a phase-separated colloid- polymer suspension. Dang, M. T., Verde, A. V., Nguyen, V. D., Bolhuis, P. G. & Schall, P. Temperature-sensitive colloidal phase behavior induced by critical Casimir forces. Fang, H., Hagan, M. F. & Rogers, W. B. Two-step crystallization and solid–solid transitions in binary colloidal mixtures. & Fujiwara, K. Heteroepitaxial growth of colloidal crystals: dependence of the growth mode on the interparticle interactions and lattice spacing. Goodwin, J. W., Hearn, J., Ho, C. C. & Ottewill, R. H. Studies on the preparation and characterisation of monodisperse polystyrene latices. Kim, A. J., Manoharan, V. N. & Crocker, J. C. Swelling-based method for preparing stable, functionalized polymer colloids. Crocker, J. C. & Grier, D. G. Methods of digital video microscopy for colloidal studies. The differential diffusion coefficients of lithium and sodium chlorides in dilute aqueous solution at 25°. Hunter, R. J.Foundations of colloid science (Oxford university press, 2001). & Glotzer, S. C. Hoomd-blue: a python package for high-performance molecular dynamics and hard particle monte carlo simulations. Alexander, S. Polymer adsorption on small spheres. Kleshchanok, D., Tuinier, R. & Lang, P. R. Direct measurements of polymer-induced forces. Likos, C., Vaynberg, K., Löwen, H. & Wagner, N. Colloidal stabilization by adsorbed gelatin. Youssef, M., Morin, A., Aubret, A., Sacanna, S. & Palacci, J. Rapid characterization of neutral polymer brush with a conventional zetameter and a variable pinch of salt. Stenkamp, V. S. & Berg, J. C. The role of long tails in steric stabilization and hydrodynamic layer thickness. Lechner, W. & Dellago, C. Accurate determination of crystal structures based on averaged local bond order parameters. Bishop, C. M. & Nasrabadi, N. M.Pattern recognition and machine learning. This research was primarily supported by the US Army Research Office under award number W911NF-21-1-0011 to S.S. and G.M.H., which also supported S.Z., C.W.L., and S.P., M.S.C. was supported as a fellow of the Simons Center for Computational Physical Chemistry at NYU (SCCPC, Simons Foundation Grant No 839534). were partially funded by NIH award R35GM138312. Computational work was supported in part through the NYU IT High Performance Computing resources, services, and staff expertize, and simulations were partially executed on resources purchased by the SCCPC. We thank Bart Kahr and Steven van Kesteren for insightful discussions. Present address: Department of Materials Science and Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA These authors contributed equally: Sanjib Paul, Cheuk W. Leung. Shihao Zang, Sanjib Paul, Cheuk W. Leung, Michael S. Chen, Theodore Hueckel, Glen M. Hocky & Stefano Sacanna Simons Center for Computational Physical Chemistry, New York University, New York, NY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed and performed experiments with the assistance of C.W.L. Correspondence to Glen M. Hocky or Stefano Sacanna. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Direct observation and control of non-classical crystallization pathways in binary colloidal systems. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-58827-x'>Coupling of ribosome biogenesis and translation initiation in human mitochondria</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-17 00:33:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Biogenesis of mitoribosomes requires dedicated chaperones, RNA-modifying enzymes, and GTPases, and defects in mitoribosome assembly lead to severe mitochondriopathies in humans. Here, we characterize late-step assembly states of the small mitoribosomal subunit (mtSSU) by combining genetic perturbation and mutagenesis analysis with biochemical and structural approaches. Isolation of native mtSSU biogenesis intermediates via a FLAG-tagged variant of the GTPase MTG3 reveals three distinct assembly states, which show how factors cooperate to mature the 12S rRNA. In addition, we observe four distinct primed initiation mtSSU states with an incompletely matured rRNA, suggesting that biogenesis and translation initiation are not mutually exclusive processes but can occur simultaneously. Together, these results provide insights into mtSSU biogenesis and suggest a functional coupling between ribosome biogenesis and translation initiation in human mitochondria. Ribosomes are large macromolecular RNA-protein complexes that synthesize proteins in a highly efficient and accurate manner. Human cells contain ribosomes in the cytosol, but also within mitochondria, where they synthesize the essential core subunits of the oxidative phosphorylation (OXPHOS) system. Defective mitochondrial ribosomes (mitoribosomes) lead to OXPHOS deficiency and thus to a decline in cellular energy production, ultimately leading to severe mitochondrial disorders (mitochondriopathies)1. To form functional ribosomes, the assistance of auxiliary factors is required that mediate RNA processing, modification and folding, recruit and guide ribosomal proteins, and that facilitate molecular switches by releasing other factors. Mitoribosomes and their assembly factors are evolutionarily related to the prokaryotic translation system. However, their structures and assembly pathways differ substantially2,3,4,5. The 55S human mitoribosome has a higher protein and reduced RNA content compared to its bacterial counterpart. It is composed of a 39S large subunit (mtLSU) containing the 16S rRNA, tRNAVal and 52 proteins (MRPs), and a 28S small subunit (mtSSU) comprising the 12S rRNA and 30 MRPs. While all MRPs are encoded in the nucleus, synthesized in the cytosol and imported into mitochondria, the rRNA is encoded by the mitochondrial genome (mtDNA), transcribed as part of a polycistronic transcript and processed by mitochondrial RNase P and Z6,7,8,9. Due to these differences in composition and its dual genetic nature, mitoribosomes follow different biogenesis routes by forming protein-only submodules during early assembly, which are RNA-independent5. These steps depend on biogenesis factors including RNA modifying enzymes like methyltransferases, helicases, chaperones and a conserved group of GTPases10. In particular, PTC folding depends on MRM2, MTERF4-NSUN4 and the GTPases GTPBP5, −6, −7, and −10, as recently revealed by high-resolution cryo-EM structures11,12,13,14,15,16,17,18. Loss of MTG3 (also called NOA1 or C4ORF14) causes mitochondrial translation deficiency in different model systems; embryonic lethality in mice and reduced cell viability in isolated MEFs suggest a crucial role of MTG3 in mitoribosome assembly, however the molecular consequences of MTG3 ablation have remained to be addressed21,22. Initial studies have shown that MTG3 interacts specifically with the mtSSU and not with the mtLSU or 55S, indicating a function during mtSSU assembly, similar to its bacterial homolog YqeH10,22. A previous study has shown that multiple mutations within the GTPase domain impair the ability of MTG3 to bind mtSSU particles, thus indicating the requirement of GTP-binding or -hydrolysis for mtSSU biogenesis22. Recent structural data have revealed its binding site at the subunit interface of the maturing mtSSU and suggest that MTG3 is bound very early in the assembly pathway, when the head is still immature20. The release of MTG3 from the mtSSU has been suggested to occur prior to binding of mS38 and mtRBFA20. However, the precise role of MTG3 during mtSSU biogenesis and how its release from the maturing subunit is triggered remain unknown. Here, we combine genetic perturbation with biochemical and structural analyses to elucidate the role of MTG3 during mtSSU biogenesis. We show that loss of MTG3 leads to a substantial decrease in mitochondrial translation due to a disturbed mtSSU assembly. Surprisingly, immunoprecipitation experiments of MTG3-containing mtSSU complexes reveal the co-isolation of translation initiation factors. Cryo-EM structures of MTG3-bound mtSSU particles show that MTG3 remains bound to late maturing particles and even to initiation complexes via its N-terminal domain (NTD), preventing the docking of an rRNA helix (h44) and therefore subunit joining. Taken together, these data suggest that MTG3 may act as a quality control factor that couples late mtSSU maturation with the formation of primed translation initiation complexes. To study the role of MTG3 in mitoribosome biogenesis, we generated MTG3 knockout cell lines using CRISPR/Cas9 technology with guide RNAs targeting its first exon. Two knockout clones were isolated for which MTG3 is not detectable via western blotting (Fig. Genomic DNA sequencing shows that both clones contain premature stop codons, leading to truncated and presumably unstable variants of MTG3 (Fig. As both clones show a similar behavior, we proceeded with only one of them for further downstream approaches. The loss of MTG3 significantly affects cell growth and is accompanied by rapid acidification of the media even with high-glucose, indicating a mitochondrial dysfunction (Fig. Indeed, oxygen consumption rate (OCR) is strongly reduced in Mtg3−/− while extracellular acidification rate (ECAR) is elevated (Supplementary Fig. In agreement with decreased OXPHOS capacity we observed reduced in gel activity for respiratory chain complexes I and IV (Supplementary Fig. To understand the reason of the OXPHOS deficiency, we monitored mitochondrial translation by [35S]Methionine incorporation into newly synthesized mtDNA-encoded proteins, which is strongly impaired, but not completely abolished (Fig. Mitochondrial protein synthesis is restored upon expression of a FLAG-tagged variant of MTG3 in the knockout background, excluding possible off-target effects in the knockout cell line and confirming that tagged MTG3 is physiologically functional (Fig. To dissect the underlying basis of this translation defect, we analyzed the protein steady state levels of multiple nucleus-encoded components of the mitochondrial gene expression machinery, in particular ribosomal proteins, assembly- and translation factors (Fig. Interestingly, we observe a differential reduction in multiple MRPs of the mtSSU. The mt-rRNA-dependent MRPs uS14m and uS15m and the late binding protein mS37 are drastically decreased to 20–30%, whereas other proteins remain more stable, indicating a role of MTG3 in late maturation steps. In contrast, assembly factors such as the methyltransferases TFB1M and NSUN4 or the GTPase ERAL1 are slightly elevated. Components of the mtLSU are not affected or slightly increased, consistent with a role of MTG3 in mtSSU- but not mtLSU biogenesis. In agreement with the reduced protein steady state levels of mtSSU MRPs, we observe a significant reduction of 12S mt-rRNA steady state level to 40%, whereas the 16S mt-rRNA remains stable (Fig. The level of mRNA encoding for COX1 (MT-CO1) does not differ, suggesting that the observed defect in mitochondrial translation is not caused by a decrease in mitochondrial transcripts. To investigate the consequences of MTG3 ablation on mtSSU biogenesis in more detail, we separated mitoribosomal complexes from both wildtype and Mtg3−/− cells by sucrose density gradient centrifugation (Fig. MTG3 is only detectable in less dense (2-5) and in mtSSU-corresponding fractions (6/7) in the wildtype sample, consistent with a role of MTG3 during mtSSU biogenesis as previously suggested22. The mtSSU in fractions 6/7 is also drastically reduced, as monitored with individual mtSSU components. However, mtSSU proteins such as mS40 or mS27 are unaffected in less dense fractions 2 to 4, indicating that intermediate complexes of the mtSSU can be stably formed independently of MTG3. Interestingly, assembly factors ERAL1 and TFB1M remain detectable or are slightly increased in mtSSU-corresponding fractions 6/7 although the overall level of mtSSU is decreased, indicating a stalling in the mtSSU assembly pathway. In contrast, mtRBFA is significantly decreased in fraction 6/7, suggesting that MTG3 action is required upstream of mtRBFA. Components of the mtLSU accumulate in fractions 8/9, as they cannot proceed to form functional 55S mitoribosomes due to the absence of sufficient matured mtSSU in Mtg3−/−. In conclusion, MTG3 ablation affects mtSSU biogenesis, leading to a reduced pool of translationally active mitoribosomes. a Confirmation of MTG3 knock out in two cell lines generated using CRISPR/Cas9 technology. Isolated mitochondria (10 µg) from wildtype and Mtg3−/− cell lines (cl.1 and cl.2) were analyzed by western blotting with antibodies as indicated. Similar results were obtained in n ≥ 3 biologically independent experiments. The guide RNA targets exon 1 of the MTG3 gene, which encodes for a 648 aa protein. A two bp deletion and a four bp insertion in the two alleles of the Mtg3−/− cl.1 lead to premature stop codons and truncated proteins (65 aa). c Ablation of MTG3 reduces growth rate. Equal amounts of wild type and Mtg3−/− cells were seeded in three biologically independent experiments on day 0 and counted at the indicated time points (n = 3; mean ± SEM). Significance was calculated by two-sample one-tailed Student's t-test and defined as **p ≤ 0.01. d Translation of mtDNA-encoded proteins is disturbed upon loss of MTG3. Mitochondrial translation in wild type, Mtg3−/− and Mtg3−/− cells inducibly expressing MTG3FLAG was analyzed via [35S]Methionine de novo incorporation and subsequently visualized via autoradiography and with indicated antibodies. The signal in Mtg3−/− using MTG3 antibody represents unspecific binding of the antibody in whole cell lysates as we confirmed several times the loss of MTG3 in isolated mitochondria. Similar results were obtained in n ≥ 3 biologically independent experiments. e, f MTG3 loss leads to reduced mtSSU MRP level. Steady state analysis of MRPs, assembly factors, and translation-related proteins in the Mtg3−/− cells in comparison to wild type cells. Isolated mitochondria were analyzed via western blotting with indicated antibodies (e) and protein levels in Mtg3−/− were quantified relative to wild type control (f). SDHA was used as a loading control. Statistical analysis was performed as two-sample one-tailed Student's t-test with n ≥ 3 biologically independent samples shown as mean ± SEM (individual data points are shown as circles). g RNA isolated from Mtg3−/− and wild type cells was subjected to northern blotting using indicated probes (MT-RNR1: 12S rRNA; MT-RNR2: 16S rRNA; MT-CO1: mRNA encoding for COX1). h Quantification of RNA signals in Mtg3−/− from (g) relative to wild type signals. Statistical analysis was performed as two-sample one-tailed Student's t-test with n = 3 biologically independent samples shown as mean ± SEM (individual data points are shown as circles). Significance was defined as ***p ≤ 0.001. i mtSSU and monosome levels are severely reduced in Mtg3−/− cells. Isolated mitoplasts (500 µg) were lysed and subjected to sucrose density gradient centrifugation. Fractions (1-16) were collected and analyzed via western blotting with antibodies against MRPs and assembly factors as indicated. Similar results were obtained in n ≥ 3 biologically independent experiments. To reveal the precise role of MTG3 during mtSSU biogenesis, we inducibly expressed a FLAG tagged variant of MTG3 in HEK293 cells and purified endogenous MTG3-containing complexes via MTG3FLAG-co-immunoprecipitation (Fig. ERAL1 and TFB1M, but not NSUN4 co-purify with MTG3-containing mtSSU particles, suggesting that MTG3 is part of an assembly intermediate with these and other factors, consistent with previous reports20. A reciprocal experiment using ERAL1FLAG in an Eral1−/− background as a bait likewise results in the co-isolation of MTG3 and TFB1M, thus supporting this hypothesis (Supplementary Fig. In contrast to previous observations, however, MTG3 also associates with factors characteristic of late mtSSU biogenesis steps. For example, mtRBFA, one of the final mtSSU assembly factors19, can be also co-purified with MTG3. Likewise, mS37, which is the last MRP that joins the maturing mtSSU19, is also detectable, although to a slightly lesser extent compared to other mtSSU constituents. This suggests that MTG3 remains bound to very late assembly states of the mtSSU. FLAG-immunoprecipitation was performed with lysed mitochondria from HEK293 wild type (WT) cells and a stable HEK293 cell line inducibly expressing MTG3FLAG, and subsequently analyzed via western blotting with indicated antibodies (total = 3%, eluate = 100%). Similar results were obtained in n ≥ 3 biologically independent experiments. b Schematic depiction of the mature 12S rRNA secondary structure. Regions are depicted by their level of maturation in each state, with M corresponding to the mature mtSSU (PDB: 3J9M2;). c Cryo-EM structures of the MTG3-TFB1M-mtRBFA(in)-bound small mitoribosomal subunit (mtSSU) intermediate (state A), METTL15-mtRBFA(in)-bound mtSSU (state B), and METTL15-mtRBFA(out)-bound mtSSU (state C). The 12S rRNA (red) and indicated biogenesis factors are shown as cartoon and the remaining mitoribosomal proteins (MRPs) are indicated as white transparent surface. MTG3: light blue, TFB1M: lime green, mtRBFA: indigo, METTL15: turquoise, mS38: beige2. Close-up views of the immature decoding center (d) and of the foot region (e) in state A. Coloring as in (c), with cryo-EM densities (from map A3) of immature rRNA helices shown as red surface. Close-up views of the immature decoding center with cryo-EM densities (from maps B-C3) (f, h) and of the foot region (g, i) in state B and C, respectively. (e, g, i) Close-up views of the foot region in each state, showing the densities for MTG3 (state A, map A3), or MTG3NTD and h44 with altered trajectory (state B-C, 15 Å low-pass filtered maps B1 and C1). To further investigate the role of MTG3 during late maturation of the mtSSU, we analyzed the endogenous MTG3-containing ribosomal complexes isolated via co-immunoprecipitation with MTG3FLAG as the bait by single-particle cryo-electron microscopy (cryo-EM). After extensive particle classification, we obtained reconstructions for three distinct mtSSU assembly intermediates with immaturely folded 12S rRNA (States A-C) at overall resolutions ranging from 3.1 to 3.4 Å (Fig. Compared to the mature mtSSU2, we observe three additional densities which could be unambiguously assigned as assembly factors MTG3, TFB1M and mtRBFA, together constituting a combination of assembly factors which has not been described to date. Consistent with recently reported structural data20, MTG3 consists of a C-terminal globular GTPase domain, which resides on the face of the 12S rRNA, and a N-terminal domain (NTD), which occupies the binding site of an extended lasso structure in the mature h44 (1501-1549)19,20 (Fig. MTG3 binding to the mtSSU thus prevents folding of h44 into its mature state via its globular domain as well as its NTD (Supplementary Fig. TFB1M binds next to MTG3 and assists in maturation of the DC by carrying out a di-methylation reaction in h4523. The rRNA adopts a premature conformation in this state, with a completely disordered h44 (1481-1572), partially disordered h45, and a misplaced h24 in the DC due to interactions with TFB1M and mtRBFA (Fig. State B and C represent later assembly intermediates in which the 12S rRNA is partially matured (Fig. They both do not show densities for the MTG3 globular domain or TFB1M, but contain mtRBFA and an additional density, which corresponds to the methyltransferase METTL15 (Fig. mtRBFA has previously been shown to adopt two different states on the mtSSU, termed “in” and “out”19. While states A and B contain mtRBFA in the “in” conformation, it adopts the “out” conformation in state C, which has been reported to be a hallmark of very late-stage assembly steps19. METTL15 has been shown to interact with h24 and h44 to methylate residue C1486 in h4424 (Supplementary Fig. Consistent with this, h24 and the region of h44 containing C1486 are moved towards METTL15 compared to their locations in the mature mtSSU (Fig. These observations suggest that MTG3 may remain associated to the mtSSU even during late assembly steps via its NTD, thereby preventing docking of h44 and in turn binding of the mtLSU. A recent study provided structural snapshots of MTG3 bound to the mtSSU together with the assembly factors TFB1M, MCAT, METTL17, and ERAL120 (Fig. Recruitment of mtRBFA and docking of the bS21m-uS11m-h23 module causes platform compaction and was suggested to occur after MTG3 dissociation and subsequent h44 docking. Recruitment of this module further induces di-methylation of two residues in the hairpin loop in h45 (A1583/A1584) by TFB1M. a Table of six described assembly intermediates each compromising at least one of the assembly factors MTG3, TFB1M, or mtRBFA, as well as the mature mtSSU (PDB: 3J9M2;). Maturation of 12S rRNA modules is shown for each state (matured: “+”, unmatured: “−”). b The previously described assembly state (PDB 8CSP20) (left) as well as state A (this study) (right) are shown side-by-side, with additional modules highlighted (right). The 12S rRNA and indicated biogenesis factors are shown as cartoon and the remaining mitoribosomal proteins (MRPs) are indicated as white transparent surface. Depiction as follows (state A vs PDB 8CSP20): 12S rRNA: red vs peach, MTG3: light blue vs dark blue, TFB1M: lime green vs dark green, mtRBFA: indigo, bS21m, uS21m, uS15m: beige, METTL17: yellow, MCAT: light blue, ERAL1: dark purple. c Zoomed-in view of h18 and TFB1M in state A d Zoomed-in view of MTG3 (light blue) and uS15m (beige) in state A vs additional ordered regions of MTG3 in a previous state (PDB 8CSP20) (c, d) Densities from map A3 (c) and A2 (d) are depicted as transparent surfaces. e Side-by-side view of state B (right) and C (left) with coloring as followed: 12S rRNA: red vs peach, METTL15: dark vs light turquoise, mtRBFA: indigo vs dark violet. f Different routes of the rRNA content in state B vs state C. The model of state C is shown superimposed to the model of state B, with 12S rRNA density from state B (map B3) depicted transparent. g Close-up view of METTL15-mtRBFA(in) interaction in state B superimposed with METTL15 from state C and depicted mito-specific extension from of METTL15. The 12S rRNA and MRPs are indicated as transparent surface. The three states we observe differ from previously reported states both in their composition and their rRNA maturation states (Fig. In state A, the head is already fully matured but METTL17 and MCAT are not present (Fig. In addition, ERAL1 is replaced by mtRBFA(in) and the bS21m–uS11m–h23 module, causing a rotation of the head by 25 Å (Fig. This also moves the NTD of TFB1M away from the bS21m-uS11m-h23 module to a location that differs from a previously reported late assembly intermediate containing mtRBFA(in) and TFB1M19 (Fig. In addition, we observe a density that appears to correspond to a flexible helix at the N-terminus of TFB1M (Fig. Close to this region, METTL17 was previously observed to form contacts with h44 together with the flexible helix of TFB1M (Supplementary Fig. In contrast, METTL17 is absent in state A and TFB1M may instead form contacts with the partially disordered h18 (Fig. Within the rRNA, h44 is disordered to a larger extent than previously observed and does not form any contacts with TFB1M or h45 (Supplementary Fig. In addition, h18 in the shoulder region is partially disordered and oriented towards TFB1M19,20 (Fig. 5c), and h27 is in its mature conformation, although it has been predicted that this would lead to clashes with MTG3 and TFB1M (Supplementary Fig. This is possible because both assembly factors adopt distinct arrangements in state A compared to previous structures. In the observed conformation, MTG3 would clash with mS38, which locks h27 and h44 in their final conformation in the mature mtSSU, but mS38 is not present in our state A and replaced by interactions between TFB1M, MTG3, and the rRNA helices. 5d), which is a prerequisite for bS21m-uS11m-h23-module recruitment, platform compaction and TFB1M-NTD re-orientation. This is in contrast to the previous structure of a MTG3- and TFB1M-bound intermediate, in which h27 adopts a more immature conformation (Supplementary Fig. 5d) and which suggested that platform compaction and TFB1M catalytic activity is precluded by MTG3-bound h27 until h44 docking has occurred. Taken together, state A differs substantially from previously reported MTG3-containing mtSSU intermediates as the rRNA fold in the head and platform more closely resembles later stages of mtSSU maturation19, even though h44 docking and h18 maturation has not yet occurred. We also observed differences in MTG3 compared to previously reported data. The NTD is connected via a linker to a helical insertion (residues: 179-195) at the GTPase domain of MTG3. This helical insertion was previously observed to partially occupy the empty GTP-binding site, preventing switch I loop from entering a GTP-binding competent conformation. Based on the bacterial homolog YqeH, it has been suggested that the helical insertion and switch I loop rearrange during the GTPase cycle, which triggers h27 and h44 maturation21,25. In state A, the long α-helix and the helical insertion of MTG3 are disordered and their previously observed position is occupied by uS15m (Fig. In contrast to previous MTG3-bound structures, the N-terminal helix of uS15m is ordered and occupies the position of the long α-helix (Fig. In this conformation of MTG3, the nucleotide-binding site is accessible and the switch I loop may adopt a conformation competent for GTP-binding. In agreement with this, we observe a density in the nucleotide binding pocket which would be consistent with a bound nucleotide (Fig. However, we refrained from modeling a ligand due to the limited resolution of the map in this region. Taken together, these data indicate that MTG3 can adopt multiple conformations while bound to the mtSSU and that its GTP-bound conformation is compatible with mature folding of h27 and binding of uS15m. This links MTG3 GTPase function to docking of its own NTD into the foot region and re-organization of rRNA content, but also to re-orientation of an MRP. The structural data also provide insights into the role of METTL15 during mtSSU biogenesis. METTL15 interacts with h44 to methylate m4C1486, but this residue is ~40 Å away from its catalytic center in its mature conformation in state C (Supplementary Fig. In contrast, while the overall fold of the rRNA in states B and C resembles previously observed conformations, the segment containing m4C1486 appears less well ordered in state B, suggesting that it may represent an intermediate state previously not observed (Fig. Previous studies have suggested that METTL15 plays an important structural role during mtSSU biogenesis in addition to its catalytic activity as a methyltransferase19. Consistent with this previous report, we observe METTL15 together with mtRBFA in its “out” conformation (mtRBFA(out)) in state C (METTL15C). By contrast, state B contains mtRBFA in its “in” conformation together with METTL15B, a combination that has not been previously described (Fig. This is possible because a mitochondria-specific extension of METTL15, which would clash with mtRBFA(in), is disordered in METTL15B (Fig. In previous mtRBFA(in) containing structures, its C-terminal extension (mtRBFA-CTE) was observed to be disordered. This suggests that in state B, the mitochondrial specific extension of METTL15 is disordered, but folds upon m4C1486 methylation and maturation of this part of h44 upon transition to state C, thereby inducing a conformational switch of mtRBFA from “in” to “out”. The mtRBFA(in) conformation has been associated with earlier mtSSU intermediates than mtRBFA(out), and the transition from “in” to “out” was suggested to cause the head to rotate closer to its mature position, constituting a checkpoint in the mtSSU assembly pathway19. Our structural data suggest how conformational changes of METTL15 and mtRBFA may be coupled, and thus provide a molecular rationale for this checkpoint. In all previously reported METTL15 and mtRBFA containing states19,20 the lasso of h44 adopts its mature conformation. It has thus been suggested that these two factors are hallmarks of the final steps of mtSSU maturation, and are released by mS37 as well as the initiation factor mtIF3 during the transition from assembly to initiation19. However, state B and C both contain a disordered h44 lasso, which is not yet docked into the foot region. Instead, its location is occupied by the MTG3NTD (Fig. This suggests that docking of h44 and MTG3 dissociation is not necessarily required for late stage mtSSU assembly steps, and that initiation complex assembly could potentially commence already prior to completed mtSSU maturation. Consistent with a potential link between mtSSU assembly and initiation complex formation, we noticed accumulation of mtSSU initiation factors, namely mtIF2 and mtIF3, together with mtSSU proteins during immunoprecipitation via FLAG-tagged MTG3 (Fig. Reciprocally, using FLAG-tagged mtIF3 to isolate native mtSSU complexes accumulated mtIF2 as well as the assembly factors mtRBFA and MTG3 (Fig. Thus, our data suggest that IC intermediates co-purify with the assembly factor MTG3 and vice versa. In agreement with this, particle classifications showed that a substantial proportion of mtSSU particles in our cryo-EM dataset indeed had translation initiation factors bound (Supplementary Fig. This enabled us to resolve four distinct (pre-) translation initiation complexes ((P)-IC) (states D-G) at overall resolutions ranging from 3.1 to 3.6 Å (Fig. a Western blot analysis of mitoribosome complexes co-purified via mtIF3FLAG. FLAG-immunoprecipitation was performed with lysed mitochondria from wild type cells and cells inducibly expressing mtIF3FLAG. Samples (total = 3%, eluate = 100%) were subsequently analyzed via western blotting with indicated antibodies. Similar results were obtained in n ≥ 3 biologically independent experiments. The 12S rRNA and indicated initiation factors are shown as cartoon and the remaining mitoribosomal proteins (MRPs) are indicated as white transparent surface. c Zoomed-in view of the codon-anticodon interaction in state G. Coloring as in (b) with mRNA backbone being depicted in purple and nucleotides being highlighted in red (A), blue (U), green (G), and yellow (C). The 12S rRNA and MRPs are indicated as transparent surface. The trajectory of the mRNA is shown by superimposing with a known IC state (7PO219;) and depicted as transparent cartoon. d–g Views of the foot region in each state, showing the densities for MTG3NTD and for h44 from the 15 Å low-pass filtered maps D1-G1. Mature h44 is depicted by red dashed lines and would clash with MTGNTD. State D resembles a previously described structure of the PIC with mtIF3 bound19. State E additionally contains mtIF2, thus resembling another recently reported PIC26. In this state, we observe clear density for the codon-anticodon interaction, consistent with a published IC state primed for translation initiation (Fig. In all four (P-)IC states, most of the mt-rRNA is matured and mtRBFA is replaced by mS37, consistent with previous data (Fig. This suggests that h44 maturation is not required to start IC formation and that translation initiation factors can be recruited before mtSSU maturation is finalized. However, the association of MTG3 during first steps of IC formation prevents docking of h44 and thus mtLSU recruitment to the mtSSU. The N-terminal region of MTG3 displaces h44 from its mature conformation and is highly conserved across vertebrates (Supplementary Fig. Our data indicate that it may remain stably bound to the mtSSU, even when mS38 binds and the globular domain of MTG3 is displaced (Fig. To further dissect the impact of this domain on MTG3 function, we expressed a FLAG-tagged mutant variant of MTG3 lacking the highly conserved 10 aa (69-83) which interact with mS27 and displace h44 (ΔN-MTG3FLAG) in the Mtg3−/− cell line (Supplementary Fig. Deletion of these 10 aa does not interfere with the import of ΔN-MTG3 into mitochondria, as the upstream N-terminal pre-sequence remains preserved and ΔN-MTG3FLAG is comparably detectable in isolated mitochondria like the wildtype variant of MTG3FLAG. To dissect the role of this N-terminal helix, we monitored mitochondrial translation by [35S]Methionine de novo synthesis in Mtg3−/− expressing MTG3FLAG wildtype or ΔN-MTG3FLAG mutant. Although mitochondrial translation seems to be slightly increased upon ΔN-MTG3FLAG expression in comparison to the full knockout, in contrast to full-length MTG3, the ΔN-MTG3FLAG can only partially restore mitochondrial translation (Fig. Interestingly, we did not observe a uniform restored translation pattern. While translation of e.g., ATP6 and ATP8 seems to be completely rescued in the mutant cell line, the levels of newly synthesized COX1 and corresponding protein steady state are only marginally increased in comparison to the full knockout, but by far not restored as by expressing MTG3FLAG. This indicates that the N-terminal region of MTG3 is essential for its full functionality. This is further supported by the fact that the mutated ΔN-MTG3FLAG cannot co-immunoisolate any mtSSU proteins, assembly factors or translation initiation factors, indicating that the N-terminal region is responsible for stable binding of MTG3 to the mtSSU (Fig. However, as indicated by the partially restored translation phenotype, translationally active mitoribosomes must be formed in Mtg3−/−+ΔN-MTG3FLAG cells. a ΔN-MTG3FLAG lacking 10 conserved aa in the N-terminus can only partially restore mitochondrial translation. Translation of mtDNA-encoded proteins in wild type, Mtg3−/− and Mtg3−/− cell lines expressing full length (MTG3FLAG) or mutated MTG3 (ΔN-MTG3FLAG), respectively, were analyzed using [35S]Methionine de novo incorporation and visualized via autoradiography and western blotting. GAPDH was used as a loading control. Similar results were obtained in n ≥ 3 biologically independent experiments. b ΔN-MTG3FLAG does not co-purify any MRPs. Samples were analyzed using western blotting and antibodies as indicated (total = 3%, eluate = 100%). Similar results were obtained in n ≥ 3 biologically independent experiments. c, d Monosome level can be restored upon ΔN-MTG3FLAG expression. Mitoplasts (500 µg) were isolated from wild type, Mtg3−/− and Mtg3−/− cell lines expressing full length (MTG3FLAG) or ΔN-MTG3FLAG, respectively. Mitoribosomal complexes were separated via sucrose density gradient centrifugation and collected fractions (1-16) were analyzed via western blotting with indicated antibodies against MRPs and assembly factors (c) or northern blotting with probes against 12S rRNA (MT-RNR1) and 16S rRNA (MT-RNR2) (d). Similar results were obtained in n ≥ 3 biologically independent experiments. e Abundance of mt-mRNAs bound to monosomes is similar in the Mtg3−/− + ΔN-MTG3FLAG cell line in comparison to wild type MTG3FLAG. Fraction 11 from sucrose gradients from (c) were used to isolate RNA and perform NanoString analysis. Level of mt-mRNAs were normalized to 16S-rRNA and RNA abundance bound to monosomes in the Mtg3−/− + ΔN-MTG3FLAG cell line was calculated relative to Mtg3−/− + MTG3FLAG cell line (n = 3 biologically independent samples shown as mean ± SEM; individual data points are shown as circles). Statistical analysis was performed as two-sample one-tailed Student's t-test. Interestingly, sucrose gradient analysis revealed similar levels of 55S mitoribosomes in fraction 11 comparing Mtg3−/−+MTG3FLAG and Mtg3−/−+ΔN-MTG3FLAG, but different levels of assembled mtSSU in fractions 6/7 (Fig. While expressing MTG3FLAG in Mtg3−/− completely rescues the mtSSU assembly defect and leads to comparable levels of assembled mtSSU as the wildtype control, ΔN-MTG3FLAG appears incapable of restoring the pool of assembled mtSSU. The relative amount of assembly factors such as ERAL1, TFB1M, and mtRBFA co-migrating with the mtSSU is increased, considering the decreased amount of mtSSUs in fraction 6/7 in the ΔN-MTG3FLAG cell line. This suggests that the majority of mtSSU in ΔN-MTG3FLAG-expressing cells represent immature biogenesis intermediates. Nevertheless, some assembled mtSSUs seem to be able to bind the mtLSU and are thus directly transferred to the monosome pool in fraction 11, but the mitochondrial translation assay indicates that those particles are not as active as when expressing MTG3FLAG wildtype. However, as the mutant variant is not able to stably interact with the mtSSU to prevent subunit joining, some 55S particles might be formed containing defective mtSSU. To elaborate the nature of these particles further, we asked whether they have mt-mRNA bound and whether MTG3 plays an active role in mt-mRNA loading during translation initiation. Thus, we measured the abundance of mt-mRNAs in fraction 11 relative to the 16S mt-rRNA and did not observe a strong reduction of mitoribosome-associated mt-mRNAs (Fig. In fact, some mt-mRNAs, including MTCO1, appear to be elevated in Mtg3−/−+ΔN-MTG3FLAG compared to Mtg3−/−+MTG3FLAG cells, which might indicate a compensatory mechanism to counteract the inefficient translation of COX1. In order to study the role of the GTPase domain of MTG3 during mtSSU maturation similar mutational analyses were performed. A mutant variant of MTG3 with a glycine to proline substitution (G499P) in the G3 motif (switch II) was expressed in the Mtg3−/− background (MTG3-G499PFLAG). The G3 motif interacts with the γ-phosphate of a bound GTP and the G499 was shown to be essential for GTP hydrolysis in other GTPases10,27,28. Indeed, the MTG3-G499PFLAG cannot restore mitochondrial translation (Supplementary Fig. 6c), indicating that GTPase activity is abolished and mtSSU biogenesis impaired. As the ΔN-MTG3FLAG mutant was at least partially able to restore translation (Fig. To further compare and dissect the roles of these two domains, mitoribosome assembly was investigated by sucrose gradient centrifugation with cell lines expressing the mutated variants of MTG3 in comparison to Mtg3−/− and Mtg3−/−+MTG3FLAG rescue (Supplementary Fig. The MTG3-G499PFLAG mutant reveals mtSSU levels in fractions 6/7 and monosomes in fraction 11 comparable to those of Mtg3−/−, explaining the inability to restore mitochondrial translation. In contrast to the ΔN-MTG3FLAG mutant, almost no monosomes are formed in MTG3-G499PFLAG mutant. FLAG-immunoprecipitation reveals that the MTG3-G499PFLAG can co-isolate mtSSU particles, however it seems that these only resemble early-stage mtSSU intermediates (Supplementary Fig. This indicates that mtSSU biogenesis gets stalled at an early assembly step when the GTPase hydrolysis function of MTG3 is abolished. Taken together, these data suggest that the GTPase activity of MTG3 is essential for mtSSU assembly. Our structural and biochemical data suggest that mtSSU assembly and translation initiation are two processes that do not stringently occur sequentially after each other, and allow us to deduce an alternative model of mtSSU biogenesis (Fig. First, the assembly factors TFB1M, MTG3 and mtRBFA(in) associate with the mtSSU to facilitate maturation of h44, h45, h44-h45-linker and parts of h27 (Supplementary Table 3). The globular CTD of MTG3 then dissociates, but MTG3 remains associated with the mtSSU via its N-terminal region thereby preventing h44 maturation. The late-step assembly protein METTL15 is then recruited, and its catalytic activity triggers the maturation of h24 and the h44-h45 linker, thereby inducing a conformational switch of mtRBFA from “in” to “out”. Although h44 is still immature and MTG3 remains bound via its NTD, translation initiation factors can be recruited to this mtSSU assembly intermediate. From state F onwards, fMet-tRNAMet and mt-mRNA associate with the mtSSU generating state G, while h44 remains immature. After priming of the IC by codon-anticodon formation, MTG3 must be released to allow docking and maturation of h44 at the foot and subsequent binding of mtLSU. What triggers the dissociation of MTG3 remains to be addressed. Taken together, our data suggest a mitoribosome assembly and translation initiation pathway in which MTG3 is the last factor to be released from the mtSSU before the complete mitoribosome is formed. Intermediate states of the mtSSU are depicted as surface and factors and rRNA content is colored as in Figs. Models for mature mtSSU IC (PDB: 7PO219), mtLSU and mature IC (PDB: 6GAW53) have been reported previously. Ribosome biogenesis is an energetically expensive process that requires multiple auxiliary factors to mediate rRNA and protein folding and to coordinate the sequential binding of ribosomal proteins. Thus, multiple biogenesis factors bind to the intersubunit interface of the mtLSU and mtSSU and thus prevent immature subunit joining. It was assumed for a long time that ribosome biogenesis and translation initiation are separated processes that follow a single stringent route and occur sequentially after each other. Using a combination of biochemical and structural analysis, we here show that the mtSSU biogenesis factor MTG3 plays a critical role during late-stage maturation of the mtSSU. We describe several previously unobserved mtSSU assembly intermediates, which provide molecular insights into the role of the biogenesis factors MTG3, METTL15, and mtRBFA. Surprisingly, we were not able to resolve an MTG3-bound mtSSU with associated ERAL1. The reason for this could be that such particles represent a minor fraction of the sample or due to flexibility of the GTPase ERAL1. The challenge of solving GTPases co-purified with assembling mitoribosomal subunits is a common phenomenon in the field14. Nevertheless, together with previous data, these structural snapshots enable us to propose a unified model for mtSSU maturation. In addition, we show that MTG3 can remain bound to the mtSSU via its NTD until the final maturation steps and even during initiation complex assembly. Surprisingly, our data suggest that the latter can commence before the rRNA, and in particular h44, is completely matured. This suggests the existence of several mtSSU biogenesis pathways and, for at least one of them, a coupling of mtSSU maturation and translation initiation by the action of MTG3, which may act as a quality control factor. In particular, the differential effects of mutations within the NTD and the GTPase domain suggest a potential dual role of MTG3 during mtSSU assembly and maturation: First, MTG3 is bound to the mtSSU with its N-terminus and the globular GTPase domain. GTP hydrolysis might lead to rearrangements in the mtSSU such as h27 maturation, allowing mS38 to bind and displace the globular domain of MTG3. The GTPase activity of MTG3 seems to be essential for continuing mtSSU maturation, as the MTG3-G499PFLAG is unable to form mature mtSSUs. The N-terminus of MTG3 remains stably bound to the mtSSU during late maturation steps and even during the formation of initiation complexes, potentially acting as a quality control mechanism, preventing docking of h44 and therefore premature mtLSU binding. The N-terminal region is not conserved in yeast or bacterial MTG3 homologs, but is highly conserved in other vertebrates, suggesting its function was newly acquired in the course of evolution. Our data do not give insights into what triggers the dissociation of MTG3, we can only speculate that other, unidentified factors might be involved. In the ΔN-MTG3FLAG cell line, the globular GTPase domain can fulfill its function, so mtSSU maturation can continue and monosomes can be formed. However, without the N-terminus, MTG3 might dissociate after mS38 has displaced the globular domain, and cannot act as a quality control factor. Thus, immature mtSSU particles bypass this critical quality checkpoint leading to premature subunit joining and thus to the formation of translational inactive or less-active mitoribosomes. The presence of a quality control check point for proper ribosome maturation coupled to translation initiation is reminiscent to the systems described in bacteria, the eukaryotic cytosol, and also in Trypanosoma brucei mitochondria. However, the detailed molecular mechanisms differ. The initiator tRNA has been ascribed a direct role during late maturation steps by licensing RNases to perform the final processing steps of the bacterial 16S rRNA31. Final maturation, including the final processing of the rRNA, occurs within these 80S-like complexes. The recruitment of the mature 60S LSU to the immature 40S depends on eIF5B, a translation factor with GTPase activity akin to bacterial IF2. These 80S-like ribosomes do not represent translation initiation intermediates as they lack initiator tRNA, mRNA and eIF2. In Trypanosoma brucei mitochondria, mtIF2 acts as a quality control factor monitoring the proper folding of the DC while other late assembly factors remain bound to the mtSSU33. Together with an mtIF3-like factor, mtIF2 interacts with elements of the DC and blocks the mRNA channel preventing mRNA binding and therefore the formation of a canonical initiation complex. In contrast, we report for human mitochondria an mtSSU complex with bound initiator tRNA, mRNA and mtIF2, but with immature h44 due to the association of MTG3NTD. Thus, our data suggest that MTG3 acts as a late quality control factor which remains to be bound until all elements for translation initiation are in place. A common principle in all systems seems to be that domains close to the 3' end, such as h44, are the last regions that are matured31,33,34,35. This also includes RNA modifications such as 12S methylation within h44 and h45 catalyzed by METTL15, NSUN4, and TFB1M23,24,36. However, the 12S mt-rRNA requires no further trimming of the 5' or 3' ends after its release from the polycistronic transcript by RNase P and—Z and its assembly with MRPs, which again is distinct from the maturation process described for bacterial and eukaryotic cytosolic SSU. The binding of MTG3 to IC particles might also indicate a function as a splitting factor that dissociates 55S complexes reminiscent to the dual function of GTPBP6, which is required for PTC maturation, but which can also actively dissociate 55S mitoribosomes11,27. However, overexpression of MTG3 has no negative impact on mitochondrial translation, which is in contrast to elevated levels of GTPBP6 and thus argues against a function as a ribosome dissociation factor. In contrast to a previous report, which suggests the dissociation of MTG3 from the mtSSU prior to binding of mS38 and mtRBFA20, our data indicate that MTG3 remains bound via its NTD to the mtSSU during final maturation steps and even during IC formation. As alternative assembly pathways might occur especially during late assembly steps, these observations are not necessarily in conflict. While it cannot be excluded that ICs with immature rRNA represent an unproductive OFF-pathway, it appears to be a common principle that ribosome maturation is a dynamic process with alternative routes to ensure efficient proper maturation of ribosomes35,37. For bacterial ribosomes, for example, it was shown that assembly blocks can join the assembling 50S LSU in a flexible order37,38. Under optimal conditions, the most effective pathway is preferred. However, the flexibility ensures that ribosome biogenesis can still continue under non-optimal conditions, such as when a ribosomal protein is depleted, through re-routing of the process. Similarly, alternative pathways that do not require MTG3 might also explain the appearance of pre-SSU states during late assembly and translation initiation, but with folded h4419,20 and also explain why translation-competent ribosomes can still be formed in Mtg3−/−, although less efficiently (Fig. In addition, different isolation strategies may lead to the enrichment of different particle populations. While Harper et al.20 enriched for METTL17-containing complexes via affinity purification of tagged METTL17, and Itoh et al.19 isolated mitoribosome population from cells deficient in TRMT2B (another known assembly factor of mtSSU), our approach followed the immunoisolation of MTG3-containing complexes. Thus, depending on the bait and background, different populations from earlier or later maturation steps will be enriched. Taken together, our data suggest an alternative pathway for mitoribosome biogenesis in which MTG3 can remain bound to the mtSSU during the first steps of translation initiation. Whether and under which conditions this pathway is physiologically relevant remains to be determined. In summary, our work suggests the presence of an alternative mtSSU maturation pathway, where MTG3 has a quality control function during the final maturation steps of the mtSSU by preventing the docking of h44 during the IC formation primed for translation initiation. HEK293 cell lines (Human Embryonic Kidney 293-Flp-In T-Rex, Thermo Fisher Scientific; R78007) were cultured in DMEM (Dulbecco's Modified Eagle's Medium) supplemented with 10% FCS (Fetal Calf Serum), 2 mM L-glutamine, 1 mM sodium pyruvate and 50 µg/ml uridine under standard cultivation conditions (37 °C under 5% CO2 humidified atmosphere). Cells were regularly tested to be free of Mycoplasma contaminations by GATC Biotech. To monitor cell growth 7.5 × 104 cells were seeded into 6-wells plates on day 0 and cell numbers were counted after 1, 2, and 3 days. HEK293 Mtg3−/− and Eral1−/− cell lines were generated by using the Alt-R CRISPR/Cas9 system (Integrated DNA Technologies) according to the manufacturer's instructions. Briefly, cells were co-transfected with Cas9 enzyme and a crRNA-tracrRNA duplex carrying a fluorescent dye to select successfully transfected cells. Knockout clones were first tested via immunoblotting and further confirmed via TOPO sequencing. Sequencing of the Eral1−/− clone revealed premature stop codons in the first exon (Supplementary Fig. 1d), leading to truncated and probably unstable variants of ERAL1. Stable inducible HEK293 cells lines expressing C-terminal FLAG-tagged MTG3, mutant versions of MTG3 (ΔN-MTG3FLAG lacking aa 69-78 and MTG3-G499PFLAG), mtIF3 or ERAL1 were generated as described previously39,40. Briefly, cells were transfected with pOG44 and pcDNA5/FRT/TO plasmids containing the FLAG construct using Lipofectamine 3000 (Invitrogen) as transfection reagent according to the manufacturer's instructions. Two days after the transfection selection of cells carrying the FLAG construct was started using 100 µg/ml Hygromycin B and 5 µg/ml Blasticidin S. For measuring oxygen consumption rates (OCR) and extracellular acidification rates (ECAR), a Seahorse XFe96 Extracellular Flux Analyzer (Agilent) was used according to the manufacturer's instructions. Cells (5 × 104 per well) were seeded into a Seahorse XF cell culture plate. For OCR analysis, basal respiration was determined, and subsequently different metabolic conditions were analyzed by adding 3 µM oligomycin, 1.5 µM CCCP, and 0.5 µM antimycin A and rotenone each. For ECAR analysis, acidification of the media was measured under basal conditions and after the addition of 25 mM glucose, 3 µM Oligomycin, and 25 mM 2-deoxy-D-glucose. For in gel activity measurements of complex I and IV, first a BN-PAGE was conducted. After centrifugation (20,000 × g, 10 min, 4 °C), BN loading buffer (100 mM Bis-Tris pH 7, 500 mM amino caproic acid, 5% Serva Blue G250) was added to the samples, which were subsequently loaded onto a 3–10% polyacrylamide gradient gel and run at 4 °C. Gels were incubated in complex I in gel activity solution (2 mM Tris-HCl pH 7.4, 0.1 mg/ml NADH, 2.5 mg/ml nitro tetrazolium blue) or complex IV solution (0.5 mg/ml diaminobenzidine, 20 µg/ml catalase, 1 mg/ml reduced cytochrome c, 75 mg/ml sucrose, 50 mM KPi pH 7.4), respectively. For isolation of mitochondria, cells were harvested and resuspended in trehalose buffer (300 mM trehalose, 10 mM HEPES-KOH pH 7.4, 10 mM KCl, 1 mM PMSF, and 0.2% BSA) and homogenized with a Homogenplus Homogenizer (Schuett-Biotech). After removing cell debris (centrifugation at 400 × g, 10 min), mitochondria were pelleted at 11,000 × g for 10 min. To generate mitoplasts, isolated mitochondria were incubated in trehalose buffer with 0.1% digitonin for 30 min on ice following Proteinase K treatment (0.5 µg Proteinase K per 100 µg mitochondria) for 15 min and blocking of the reaction with 2 mM PMSF for 10 min. Lysed mitoplasts (500 µg in 3% sucrose, 20 mM HEPES-KOH pH 7.4, 100 mM KCl, 20 mM MgCl2, 1x PI-Mix, 0.08 U/µl RiboLock RNase inhibitor (Thermo Fisher) and 1% digitonin) were separated by sucrose gradient ultracentrifugation (5–30% sucrose (w/v) in 20 mM HEPES-KOH pH 7.4, 100 mM KCl, 20 mM MgCl2, 1x PI-Mix) at 79,000 × g for 15 h at 4 °C using a SW41Ti rotor (Beckman Coulter). Fractions (1-16) were collected with a BioComp fractionator, precipitated with 2.5 volumes of ethanol and 1/3 volume of 3 M sodium acetate pH 6.5. Samples were subsequently used for western blotting or RNA was isolated for northern blotting or NanoString analysis. Immunoprecipitation of FLAG-tagged proteins was performed as described previously with some modifications40. After centrifugation at 16,000 × g at 4 °C for 10 min, the lysate was incubated with anti-FLAG M2 Affinity Gel (Sigma) for 1 h. For the elution of co-purified proteins, FLAG peptides were added. Eluate was either analyzed via western blotting, or used for cryo-EM. [35S]Methionine labeling of newly synthesized mitochondrial proteins was performed as described previously40,41. Cells were incubated in methionine-free media without FCS for 10 min, followed by 10 min incubation in methionine-free media containing 10% FCS and 100 µg/ml emetine to block cytosolic translation. Then 100 µCi/ml [35S]Methionine was added and cells were incubated for 1 h. After harvesting, cells were lysed using nonionic lysis buffer (50 mM Tris-HCl pH 7.4, 130 mM NaCl, 2 mM MgCl2, 1% NP-40, 1 mM PMSF and 1x Protease Inhibitor Cocktail (PI-Mix, Roche)) and centrifuged for 2 min at 600 × g. Supernatants were collected and protein concentration was determined by Bradford. Radioactive labeled mitochondrial products were visualized with Typhoon imaging system (GE healthcare). Total RNA or RNA from sucrose gradient fractions was isolated using TRIzol reagent (Life Technologies) or the PureLink RNA Mini Kit (Invitrogen), both according to the manufacturer's instructions. RNA was separated on a denaturing formaldehyde/formamide 1.2% agarose gel and transferred to an Amersham HybondTM-N membrane (GE healthcare) or GeneScreen Plus membrane (PerkinElmer). [32P]-radiolabeled probes targeting mitochondrial RNAs were used for visualization with Typhoon imaging system (GE healthcare) (Supplementary Table 4). RNA was isolated from sucrose gradient fractions and total mitochondrial lysate using TRIzol reagent (Life Technologies) and RNA Clean and Concentrator kit (Zymo Research). RNA was further processed following the manual (NanoString Technologies). In brief, isolated RNA pools were hybridized with TagSet (nCounter Elements XT Reagents, Nanostring Technologies) and specific probes targeting mitochondrial transcripts (Supplementary Table 4)39,42. Samples were analyzed with an nCounter MAX system (nanoString Technologies) and data were processed with nSolver software. Raw data of mt-mRNAs were normalized to the abundance of 16S rRNA in the respective fractions. Cell lysates, mitochondria samples, or samples recovered from sucrose gradient fractions were separated on 10–18% Tris-Tricine gels and transferred onto nitrocellulose membranes (Cytiva) via semi-dry western blotting. For detection of specific proteins, primary antibodies were incubated overnight at 4 °C following incubation with secondary antibodies coupled to HRP or with fluorescence labeled antibodies for 1 h at room temperature (Supplementary Table 4). Signals were detected using ECL western blotting solution (ThermoFisher Scientific) or LI-COR Odyssey CLx system, and analyzed using Fiji Image J43. FLAG-immunoprecipitation was conducted with lysed mitoplasts from a stable HEK293 cell line inducibly expressing MTG3FLAG. The eluate was crosslinked with 0.15% glutaraldehyde for 10 min on ice and the reaction was stopped by adding 50 mM lysine pH 7.5 and 50 mM aspartate pH 7.5. The sample was then desalted using Zeba Spin Desalting columns 7 K MWCO (Thermo Scientific) according to the manufacturer's instructions. For grid preparation, 4 µl of the sample were applied to freshly glow-discharged R 3.5/1 holey carbon grids (Quantifoil) that were precoated with a 2–3 nm carbon layer using a Leica EM ACE600 coater, at 4 °C and 95% humidity in a Vitrobot (FEI). Grids were blotted for 5 s with a blot force of 0 and 60 s before plunge-freezing in liquid ethane. Cryo-EM data collection was performed with SerialEM using a Titan Krios transmission electron microscope (Thermo Fisher Scientific) operated at 300 keV44. Images were acquired in EFTEM mode using a GIF quantum energy filter set to a slit width of 20 eV and a K3 direct electron detector (Gatan) at a nominal magnification of 81,000x corresponding to a calibrated pixel size of 1.05 Å/pixel. Exposures were recorded in counting mode for 3 s with a dose rate of 14.82 e−/px/s resulting in a total dose of 40.33 e−/Å2 that was fractionated into 40 movie frames. Images were acquired in 14 by 1 hole per stage movement. Motion correction, CTF-estimation, particle picking, and extraction were performed using Warp45. Further processing was carried out using Relion 3.1.046 and final non-uniform refinement and local refinement in cryoSPARC47. A representative micrograph and the cryo-EM processing workflow are depicted in Supplementary Fig. For initial processing steps, the dataset was split into four batches containing 1,521,397, 1,592,662, 1,569,593 and 1,046,895 4-times binned particles, respectively. From initial 2D classification in Relion, 2,456,047 particles were selected and joined together. From a second round of 2D classification, 663,593 “SSU candidates” and 811,261 “ambiguous SSU” particles were selected. The ambiguous set was re-classified in 2D revealing 242,768 “rescued SSU (1)” particles. The “SSU candidates” were unbinned to a factor of 2 and used for initial 3D auto-refinement and subsequent local 3D classification sorting them into 544,822 good classes (“SSU candidates (1)”) and 118,771 “ambiguous SSU (2)”. All “rescued SSU” sets were joined and unbinned to a factor of 2 followed by 3D auto-refinement and local 3D classification. 78,528 “SSU candidates (2)” were rescued from those and merged with “SSU candidates (1)” ending in 623,350 good SSU particles (10.9% of initially extracted particles) for further processing. The particles were unbinned and subjected to 3D auto-refinement with a mask around the entire SSU leading to a consensus refinement of 3.0 Å resolution. Two rounds of Bayesian polishing and CTF refinements particles followed by 3D refinement led to a “shiny” reconstruction at 2.8 Å resolution (Supplementary Fig. 2, “Head”) at 2.7 Å resolution was generated by subsequent focused 3D refinement with a mask encompassing the head. Signal subtraction with the same mask followed by 3D refinement with a mask around the SSU body generated a body-focused reconstruction without the SSU head (Supplementary Fig. A second round of signal subtraction with a mask around the factor binding site followed by focused 3D classification without image alignment using the same mask revealed subsets containing densities for METTL15, mtRBFA, TFB1M, MTG3, mtIF2, fMet-tRNAMet, or empty classes. Re-classification of METTL15 in combination with either mtRBFA(in) or mtRBFA(out) with a mask around METTL15 was performed to increase the signal for METTL15, which was reduced due to its intrinsic flexibility on top of the SSU. All three empty classes were joined together, and all classes were backprojected using the angles from the shiny refinement followed by filtering by local resolution. The joined empty classes and mtIF2-only classes were again subjected to double signal subtraction and local refinements to subtract the SSU head and everything outside a mask encompassing the mtIF3 binding site, revealing three additional classes containing only mtIF3 or mtIF2 or both factors together (Supplementary Fig. Subsequent non-uniform refinement of these subsets in cryoSPARC47 led to improved maps of the respective regions (Supplementary Fig. Local refinements using masks encompassing the head, the body, METTL15, mtIF2 (for state F) or mtIF2 and fMet-tRNAMet (for state G) were generated in cryoSPARC (Supplementary Fig. For final model building and refinements, composite maps were generated in ChimeraX48,49 (Supplementary Fig. For states A-C, composite maps were generated using unsharpened local refinement maps to improve visibility of the factors, while the composite maps of state D to G were generated using sharpened local refinement maps. In order to generate an initial model of the SSU, we rigid-body fitted all residues belonging to the SSU head into the head-focus map (Supplementary Fig. 3, “Head”) and all residues belonging to body in the global mtSSU reconstruction (Supplementary Fig. 3, “Shiny”) in ChimeraX, using a published structure from19 (PDB: 7PO1) as starting model. For state C, which mainly resembles pre-SSU-3a from19 (PDB: 7PNX), we used this model as a starting point. For state G, which mainly resembles the IC state from19 (PDB: 7PO2), we used this model as a starting point. The head and body models were then rigid-body fitted into each of the composite maps (Supplementary Fig. In all other states, published structures of factors were used as initial models and could be unambiguously fit into our densities. All residues within missing or weak density were deleted, and differently folded rRNA content was adjusted to some extent or deleted due to weak densities. Residues in h44 that differ from its mature orientation in states B and C were modeled by rigid-body fit these residues from (PDB: 7PO2) followed by manual adjustments in Coot. In order to obtain stereochemically sound models, the models for states A-C were interactively re-build and refined using molecular dynamics force fields in ISOLDE52 within ChimeraX followed by real-space refinements against the composite maps (map A-C3) in PHENIX, while the models for states D-G were real-space refined against the composite maps (map D-G3) in PHENIX. This resulted in models with excellent stereochemistry. To obtain B-factors that adequately represent the conformational flexibility of head towards body, as well as weaker signal of the factors at the periphery of the whole mtSSU body (Supplementary Tables 1, 2), the models were refined by a final real-space refinement in Phenix against the local B-factor filtered maps (map A-G2). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Material will be available upon request. Source data are provided with this paper. Structure of the large ribosomal subunit from human mitochondria.Science 346, 718–722 (2014). & Ban, N. Structure and function of the mitochondrial ribosome. Lavdovskaia, E. et al. A roadmap for ribosome assembly in human mitochondria. & Hillen, H. S. Structural and mechanistic basis of RNA processing by protein-only ribonuclease P enzymes. Bhatta, A., Dienemann, C., Cramer, P. & Hillen, H. S. Structural basis of RNA processing by human mitochondrial RNase P. Nat. Kummer, E. & Ban, N. Mechanisms and regulation of protein synthesis in mitochondria. Rackham, O. et al. Hierarchical RNA processing is required for mitochondrial ribosome assembly. & Richter-Dennerlein, R. Role of GTPases in driving mitoribosome assembly. Structural basis of GTPase-mediated mitochondrial ribosome biogenesis and recycling. Lavdovskaia, E., Hillen, H. S. & Richter-Dennerlein, R. Hierarchical folding of the catalytic core during mitochondrial ribosome biogenesis. Cipullo, M., Gesé, G. V., Khawaja, A., Hällberg, B. M. & Rorbach, J. Structural basis for late maturation steps of the human mitoribosomal large subunit. Lenarčič, T. et al. Stepwise maturation of the peptidyl transferase region of human mitoribosomes. Chandrasekaran, V. et al. Visualizing formation of the active site in the mitochondrial ribosome. Rebelo-Guiomar, P. et al. A late-stage assembly checkpoint of the human mitochondrial ribosome large subunit. & Beckmann, R. A distinct assembly pathway of the human 39S late pre-mitoribosome. Mechanism of mitoribosomal small subunit biogenesis and preinitiation. Harper, N. J., Burnside, C. & Klinge, S. Principles of mitoribosomal small subunit assembly in eukaryotes. Kolanczyk, M. et al. NOA1 is an essential GTPase required for mitochondrial protein synthesis. He, J. et al. Human C4orf14 interacts with the mitochondrial nucleoid and is involved in the biogenesis of the small mitochondrial ribosomal subunit. Metodiev, M. D. et al. Methylation of 12S rRNA is necessary for in vivo stability of the small subunit of the mammalian mitochondrial Ribosome. Haute, L. V. et al. METTL15 introduces N4-methylcytidine into human mitochondrial 12S rRNA and is required for mitoribosome biogenesis. Sudhamsu, J., Lee, G. I., Klessig, D. F. & Crane, B. R. The structure of YqeH. An AtNOS1/AtNOA1 ortholog that couples GTP hydrolysis to molecular recognition. Distinct pre-initiation steps in human mitochondrial translation. Lavdovskaia, E. et al. Dual function of GTPBP6 in biogenesis and recycling of human mitochondrial ribosomes. Huang, B. et al. Functional study on GTP hydrolysis by the GTP-binding protein from Sulfolobus solfataricus, a member of the HflX family. Stokes, J. M., Davis, J. H., Mangat, C. S., Williamson, J. R. & Brown, E. D. Discovery of a small molecule that inhibits bacterial ribosome biogenesis. Translation initiation factor IF2 contributes to ribosome assembly and maturation during cold adaptation. An evolutionarily conserved element in initiator tRNAs prompts ultimate steps in ribosome maturation. Strunk, B. S., Novak, M. N., Young, C. L. & Karbstein, K. A translation-like cycle is a quality control checkpoint for maturing 40S ribosome subunits. Lenarčič, T. et al. Mitoribosomal small subunit maturation involves formation of initiation-like complexes. Klinge, S. & Woolford, J. L. Ribosome assembly coming into focus. Shajani, Z., Sykes, M. T. & Williamson, J. R. Assembly of bacterial ribosomes. Metodiev, M. D. et al. NSUN4 is a dual function mitochondrial protein required for both methylation of 12S rRNA and coordination of mitoribosomal assembly. Davis, J. H. et al. Modular assembly of the bacterial large ribosomal subunit. Dong, X. et al. Near-physiological in vitro assembly of 50S ribosomes involves parallel pathways. The human Obg protein GTPBP10 is involved in mitoribosomal biogenesis. In vivo labeling and analysis of human mitochondrial translation products. Nadler, F. et al. Human mtRF1 terminates COX1 translation and its ablation induces mitochondrial ribosome-associated quality control. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Mastronarde, D. N. SerialEM: a program for automated tilt series acquisition on Tecnai microscopes using prediction of specimen position. Tegunov, D. & Cramer, P. Real-time cryo-electron microscopy data preprocessing with Warp. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Croll, T. I. ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Unique features of mammalian mitochondrial translation initiation revealed by cryo-EM. We thank Christian Dienemann and Ulrich Steuerwald (MPI-NAT cryo-EM facility) for technical assistance during cryo-EM sample preparation and data collection, and Tanja Gall for technical assistance during Seahorse experiments. were supported by the German Research Foundation (DFG, Deutsche Forschungsgemeinschaft) under Germany's Excellence Strategy - EXC 2067/1- 390729940 (to R.R.-D., H.S.H. and ERC Advanced Grant MiXpress, ERCAdG no. Views and opinions expressed are however those of the author(s) only and do not necessarily reflect those of the European Union or the European Research Council Executive Agency. Neither the European Union nor the granting authority can be held responsible for them. was supported by a Boehringer Ingelheim Fonds Ph.D. fellowship. Funding for open access charge: SFB1565 (Project number 469281184) and Open Access Publication Funds of the Göttingen University. Open Access funding enabled and organized by Projekt DEAL. These authors contributed equally: Marleen Heinrichs, Anna Franziska Finke. Marleen Heinrichs, Angelique Krempler & Ricarda Richter-Dennerlein Marleen Heinrichs, Peter Rehling, Hauke S. Hillen & Ricarda Richter-Dennerlein Anna Franziska Finke, Angela Boshnakovska, Peter Rehling & Hauke S. Hillen Research Group Structure and Function of Molecular Machines, Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany Department of Molecular Biology, Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany Max Planck Institute for Multidisciplinary Sciences, Göttingen, Germany Fraunhofer Institute for Translational Medicine and Pharmacology ITMP, Translational Neuroinflammation and Automated Microscopy, Göttingen, Germany Peter Rehling, Hauke S. Hillen & Ricarda Richter-Dennerlein You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar ; Investigation—cell culture, biochemical approaches and sample preparation, M.H., A.K., A.B. Correspondence to Hauke S. Hillen or Ricarda Richter-Dennerlein. The authors declare no competing interests. Nature Communications thanks Yaser Hashem and the other, anonymous, reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Heinrichs, M., Finke, A.F., Aibara, S. et al. Coupling of ribosome biogenesis and translation initiation in human mitochondria. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            