
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-18</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65104608/light-37-dimensions-quantum-mechanics/'>Scientists Produced a Particle of Light That Simultaneously Accessed 37 Different Dimensions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 18:49:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This strange fact flies in the face of classical physics, and even led Albert Einstein to famously describe this quantum quirk as “spooky action at a distance.” This is what is known as “quantum nonlocality,” where objects are influenced across distances (seeming beyond the speed of light) whereas classical physics follows local theory, the idea that objects are influenced by their immediate surroundings. This is a pretty sharp divide as explained by the famous no-go theorem known as the Greenberger–Horne–Zeilinger (GHZ) paradox, which essentially details how quantum theory cannot be described by local realistic description. Named for the physicists who described the paradox in 1989, GHZ-type paradoxes show that when particles can only be influenced by proximity they produce mathematical impossibilities. As New Scientist reports, the paradox can even be expressed through a calculation where 1 equals -1. This paradox is useful in showing how quantum properties can not be described using classical means, but a new paper published in the journal Science Advances, decided to see just how strange these paradoxes could get. Essentially, an international team of scientists wanted to see how un-classical particles of light could get—and the results were maybe stranger than the authors originally anticipated. This extremely technical experiment produced photons, or particles of light, that existed in 37 dimensions. Just as you and I exist in three dimension—plus an additional temporal dimension—these photons required 37 similar reference points. Pulling this off is not an easy thing to do as Liu and his team needed to feed a version of the GHZ paradox into coherent light—even in color and wavelength—so that they could easily manipulate the photons. “We believe that this work has opened several avenues for future research,” the authors write. “We hope our findings can be used to build even stronger quantum advantages in high-dimensional systems.”</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65104210/han-dynasty-family-tombs-china/'>Archaeologists Found a Family's 3 Ancient Tombs Filled to the Brim With Treasure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 18:01:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Archaeologists discovered three remarkable 1,800-year-old, residential-style tombs featuring rooms and windows, where an ancient Chinese family was laid to rest alongside their treasured possessions. While preparing for a local park expansion in Rizhao, China, archaeologists stumbled upon the tombs, which date back to the Han dynasty. Experts say the tombs, accessed through passageways, share a similar design. The most intact tomb, known as M3, includes interconnected rooms with wooden windows and doors, indicating it may have been the final resting place of a married couple. The excavation in Rizhao, which is about 400 miles from Beijing, yielded more than 70 artifacts, ranging from an iron sword to bronze mirrors and a variety of pottery. In a distinctly unusual discovery, the team also located pieces of a coffin carriage, which was used to transport coffins into the tomb. The archaeologists described it as “exquisitely crafted and an uncommon find.” A recent find from the second Jin Dynasty, approximately 800 years old, provides evidence of non-Chinese influence in the cultural tapestry of Shanxi province. That discovery, which was also a three-tomb set, showed each had its own distinct characteristic and style, from architectural design to murals. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. An Ancient Coin May Mark the Rise of Christianity Archaeologists Found an Odd Skull With a Big Wound</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09190-w'>Kupffer cell programming by maternal obesity triggers fatty liver disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 15:50:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Kupffer cells (KCs) are tissue-resident macrophages that colonize the liver early during embryogenesis1. Upon liver colonization, KCs rapidly acquire a tissue-specific transcriptional signature, mature alongside the developing liver and adapt to its functions1,2,3. Throughout development and adulthood, KCs perform distinct core functions that are essential for liver and organismal homeostasis, including supporting fetal erythropoiesis, postnatal erythrocyte recycling and liver metabolism4. However, whether perturbations of macrophage core functions during development contribute to or cause disease at postnatal stages is poorly understood. Here, we utilize a mouse model of maternal obesity to perturb KC functions during gestation. We show that offspring exposed to maternal obesity develop fatty liver disease, driven by aberrant developmental programming of KCs that persists into adulthood. Programmed KCs promote lipid uptake by hepatocytes through apolipoprotein secretion. KC depletion in neonate mice born to obese mothers, followed by replenishment with naive monocytes, rescues fatty liver disease. Furthermore, genetic ablation of the gene encoding hypoxia-inducible factor-α (HIF1α) in macrophages during gestation prevents the metabolic programming of KCs from oxidative phosphorylation to glycolysis, thereby averting the development of fatty liver disease. These results establish developmental perturbation of KC functions as a causal factor in fatty liver disease in adulthood and position fetal-derived macrophages as critical intergenerational messengers within the concept of developmental origins of health and diseases5. Early-life environmental factors strongly influence the risk of developing non-communicable diseases in adulthood, a concept known as developmental origins of health and diseases (DOHaDs)5. Maternal malnutrition, obesity, infection, environmental pollutants and psychosocial stress have been associated with metabolic disorders, neurological diseases, cancer and dysregulated immune responses in offspring6,7,8,9,10,11. At the cellular level, maternal immune activation can alter fetal haematopoietic stem and progenitor cells (HSPCs), contributing to neonatal sepsis12. A maternal high-fat diet (HFD) or viral infection can also program microglia, the brain-resident macrophages, leading to neuronal disorders9,13. However, it remains elusive whether macrophages outside the brain can act as intergenerational messengers and causally contribute to disease onset, serving as cellular components in DOHaDs7,8. Tissue-resident macrophages, including KCs in the liver, are long-lived, yolk sac-derived cells that support tissue development and homeostasis via production of growth factors, signalling molecules, and phagocytosis of waste products and apoptotic cells4. Whether maternal environmental factors, particularly diet, influence KC development and functions, thereby leading to liver disease, remains unknown. Metabolic dysfunction-associated fatty liver disease (MAFLD) is rising with global obesity15. MAFLD can progress to metabolic dysfunction-associated steatohepatitis, fibrosis and hepatocellular carcinoma. In diet-induced MAFLD, yolk sac-derived KCs can lose self-renewal and are, at least partially, replaced by monocyte-derived KC-like cells, diversifying the macrophage pool16,17,18,19. These macrophages adapt to the fatty liver environment, altering their epigenome and transcriptome, and contribute to disease progression through the secretion of inflammatory and non-inflammatory mediators19,20. Although our understanding of MAFLD and the role of KCs in disease progression to metabolic dysfunction-associated steatohepatitis is growing, it remains unclear how KCs respond to maternal obesity-induced fatty liver disease (FLD) in the offspring and whether they drive disease onset. Here we show that maternal obesity induces a HIF1α-dependent metabolic reprogramming of yolk sac-derived KCs, persisting into adulthood and driving FLD. Using fate-mapping, depletion and conditional knockout (KO) models, combined with multi-omic profiling, we identified KCs as intergenerational messengers that translate maternal nutritional cues into long-lasting hepatic dysfunction. Our study uncovers a causal link between developmental KC perturbation and adult-onset FLD, positioning yolk sac-derived macrophages as central players in DOHaDs. Maternal obesity is associated with obesity and liver disease in offspring in humans and animal models21, yet mechanisms remain unclear. We hypothesized that KCs may contribute to FLD due to their early liver colonization and metabolic roles4. To test this hypothesis, C57BL/6Jrcc female mice were fed a HFD for 8 weeks, leading to increased body weight and homeostatic model for insulin resistance (HOMA-IR; Extended Data Fig. These diet conditions did not lead to maternal inflammation as assessed by serum cytokine and chemokine measurements, with only interleukin-1α (IL-1α), C-C motif chemokine ligand 7 (CCL7) and leptin being upregulated (Extended Data Fig. Females were then bred to control diet (CD)-fed males. To dissect gestational versus lactational effects, newborn mice were cross-fostered and/or weaned onto a CD or HFD (Fig. As maternal lean control groups, we analysed offspring born to CD females and fed with a CD throughout life or weaned on a HFD (Fig. At 11 weeks, the maternal obese groups fed with a CD after birth (HFDMCDLCD; M denotes maternal diet during gestation and L refers to foster mother diet during lactating phase) or weaning (HFDMHFDLCD) showed no differences in body weight or white adipose tissue (WAT) weight compared with the maternal lean group on a CD (CDMCDLCD; Extended Data Fig. Only groups on a HFD postnatally (CDMCDLHFD, HFDMCDLHFD and HFDMHFDLHFD) developed obesity (Extended Data Fig. 1e), indicating that a post-weaning diet, not maternal obesity alone, determines overall body and WAT weight. a, Generation of the maternal obesity mouse model. b, ORO staining of offspring livers. Representative for n = 11, 7, 7, 9, 7 and 9 mice for CDMCDLCD, CDMCDLHFD, HFDMCDLCD, HFDMHFDLCD, HFDMCDLHFD and HFDMHFDLHFD groups, respectively, on 11 experimental days. c, ORO staining quantification of panel b by QuPath. Violin plots were created from 2–10 images per mouse, and the median and quartile of all images are shown. One-way analysis of variance (ANOVA) with Tukey's multiple comparison test, comparing the mean of all biological samples and showing significances of only CDMCDLCD, CDMCDLHFD and HFDMCDLCD groups, was used. d, Liver lipidomics of all experimental groups visualized by PCA. e, Bulk RNA-seq analysis of sorted KCs visualized by PCA. Each circle or triangle represents one mouse. f, Horizontal co-expression network analysis of KC RNA-seq data with the 5,000 most variable genes across all samples (left). The network was clustered, and group fold change (GFC) across all conditions per identified cluster is depicted. The selected enriched pathways of each horizontal co-expression network analysis module by over-representation analysis are also shown (right). By contrast, liver lipid content increased in all maternal obese groups, as shown in the Oil-red-O (ORO) and haematoxylin and eosin (H&E) stainings, although the liver weight was similar in all conditions (Fig. Both principal component analysis (PCA; Fig. 1d) and hierarchical clustering of lipidomics analyses (Extended Data Fig. 1g) indicated that maternal obese groups with a CD as their final diet (HFD>CD) were distinct from all animals fed a HFD as their final diet (CD>HFD and HFD>HFD), all of which diverge from CDMCDLCD (CD>CD). Indeed, the HFDMCDLCD and HFDMHFDLCD groups showed a significant increase of lipid species, such as saturated triacylglycerols, diacylglycerols and cholesterol esters, whereas maternal lean CDMCDLHFD as well as maternal obese HFDMCDLHFD and HFDMHFDLHFD groups showed only a moderate increase of lipid accumulation compared with CDMCDLCD livers. We next performed serum metabolomics profiling of more than 900 metabolites and lipids to test whether the observed FLD phenotype of offspring born to obese mothers is also associated with systemic alterations, which might even serve as specific biomarkers. Group differences primarily reflected final dietary status (CD versus HFD; Extended Data Fig. Comparing CDMCDLCD with HFDMCDLCD offspring, in which body composition was similar but the FLD phenotype was most prominent in HFDMCDLCD livers, revealed 129 significantly altered metabolites (Supplementary Table 2), which were enriched in the metabolic pathways ‘taurine and hypotaurine metabolism', ‘arginine biosynthesis' and ‘histidine metabolism' (Extended Data Fig. In particular, the glutamine:glutamate ratio, two amino acids that feed into arginine biosynthesis, was significantly increased in HFDMCDLCD mice (Supplementary Table 2). Although generally decreased in FLD22, a maternal obesity model has reported an increased ratio correlating with liver metabolic state23, suggesting a biomarker for maternal obesity-driven FLD. In summary, maternal obesity induces a distinct FLD phenotype in offspring compared with postnatal diet-induced FLD. We next characterized the maternal obesity-induced FLD phenotype focusing on hepatic myeloid cells, which have a critical role in FLD initiation and propagation24. Quantification confirmed increased neutrophils, monocytes and classical dendritic cells (cDC1 and cDC2) in HFDMCDLCD offspring (Extended Data Fig. KC numbers were increased across maternal obese groups, whereas liver capsular macrophages were unaffected. Post-weaning HFD in maternal lean controls (CDMCDLHFD) did not induce myeloid cell influx (Extended Data Fig. RNA sequencing (RNA-seq) on sorted KCs (Extended Data Fig. 2c) revealed clustering according to their in utero exposure to either lean (CD>) or obese (HFD>) mothers (Fig. Co-expression network analysis (hCo-Cena)25 identified five modules (Fig. The steelblue module represented genes upregulated by maternal obesity. Pathway enrichment analyses using Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG) and HALLMARK databases revealed an upregulation of genes related to ‘cell activation involved in immune response'. Among these genes was Trem2, which has been previously described to be upregulated in KCs during FLD17, and which was accompanied by its downstream signalling cascade molecules Tyrobp and Syk in the same module (Fig. In addition, Myd88, Gpnmb and the monocyte-specific genes Ccr2 and Cx3cr1 were found in the steelblue module with the highest expression in HFDMCDLCD KCs (Fig. By contrast, the turquoise and lightgreen modules indicated that metabolic processes such as ‘oxidative phosphorylation', ‘cellular respiration' and ‘xenobiotic metabolism' were downregulated upon maternal obesity (Fig. Thus, maternal obesity causes an inflammatory response of KCs, and leads to an alteration of their metabolic status on a transcriptional level. To validate the transcriptional changes of KCs at the protein level, we performed spectral flow cytometry-based metabolic profiling26 of KCs from CDMCDLCD and HFDMCDLCD offspring (Extended Data Fig. Although most nutrient transporters and metabolic enzymes involved in the glucose and lipid metabolic pathways remained unchanged, the enzyme succinate dehydrogenase (SDHA), a key component of the tricarboxylic acid cycle, and glucose transporter 1 (GLUT1) were reduced in HFDMCDLCD KCs compared with CDMCDLCD (Extended Data Fig. These findings indicate that maternal obesity impairs the metabolic capacity of offspring KCs. To determine systemic effects, we assessed serum cytokines and bone marrow compartment cellularity. No systemic cytokine elevation or progenitor changes were observed (Extended Data Fig. Together, maternal obesity-induced FLD is accompanied by local inflammation and a switch of KCs from oxidative phosphorylation to glycolysis. Diet-induced FLD models show increased monocyte influx and replacement of yolk sac-derived KCs16,17,18. Our RNA-seq data hinted at a monocyte signature in KCs from maternal obesity offspring. To address KC ontogeny, we generated Tnfrsf11aCre;Rosa26LSL-YFP;Ms4a3FlpO;Rosa26FSF-tdTomato mice. This double fate mapper allows to follow ontogeny of yolk sac-derived pre-macrophages and bone marrow-derived monocytes simultaneously using a combination of conventional recombinase approaches (Fig. We have previously shown that Tnfrsf11aCre mice label pre-macrophages and their progeny efficiently1. However, Tnfrsf11a is a core macrophage gene1 whose expression is induced once monocytes differentiate into macrophages27,28 whereby fetal-derived and bone marrow-derived macrophages remain undistinguishable. Thus, we additionally took advantage of the newly developed Ms4a3FlpO mice (Extended Data Fig. 4a), as we previously have shown that the Ms4a3 locus specifically targets monocytes and, thus, all monocyte-derived macrophages29. First, we confirmed that the fate-mapping efficiency of Ms4a3FlpO mice was as efficient as in Ms4a3Cre mice, with more than 95% Ly6C+ blood monocytes being labelled with tdTomato (tdT; Extended Data Fig. Among these monocytes, we observed a small fraction (approximately 13%) that already expressed YFP, suggesting that these cells already start to express a core macrophage program. However, none of the monocytes was only YFP+, demonstrating that the double fate mapper can unequivocally distinguish between the yolk sac-derived and adult bone marrow-derived waves of haematopoiesis (Extended Data Fig. High YFP labelling was also confirmed in microglia that are yolk sac derived30,31 (Extended Data Fig. In 11–12-week-old CDMCDLCD livers, more than 96% of KCs were YFP+, with minimal tdT labelling (less than 1% for YFP+tdT+ and approximately 3% for tdT+; Fig. Similarly, HFDMCDLCD KCs retained high YFP labelling (approximately 96%), despite the FLD phenotype and monocyte gene upregulation. Thus, KCs remain of yolk sac origin despite experiencing maternal obesity and a FLD environment during adulthood, and therefore can serve as intergenerational messengers conveying information that they experience in utero to adult life. a, Breeding scheme to generate Tnfrsf11aCre;Rosa26LSL-YFP;Ms4a3FlpO;Rosa26FSF-tdTomato double fate mapper mice. c, Labelling efficiency of the double fate mapper model in KCs isolated from 11–12-week-old CDMCDLCD (n = 4 on 2 experimental days) and HFDMCDLCD (n = 3 on 2 experimental days) mice. d, Schematic of hepatocyte culture isolated from chow diet-fed mice with KC addition isolated from CDMCDLCD or HFDMCDLCD mice, respectively, which were recorded for lipid accumulation for a 4-h time course. e, Lipid accumulation shown by normalized LD540 intensity through live imaging in ex vivo-cultured hepatocytes co-cultured for 4 h with KCs from CDMCDLCD or HFDMCDLCD mice. n = 3 for CDMCDLCD or HFDMCDLCD mice with technical duplicates. Two-way ANOVA with Sidak's multiple comparison test on biological samples was used. f, Generation of KC depletion and cell transfer maternal obesity model. Diphtheria toxin (DT) indicated by the skull symbol. g, Liver ORO staining in HFDMCDLCD offspring generated as shown in panel f. n = 4, 5 and 9 mice for Clec4f+/+ with DT and Mono/HSPCs, Clec4fDTR/+ with DT, Clec4fDTR/+ with DT and Mono/HSPCs, respectively, on 7 experimental days. h, ORO staining quantification of panel g by QuPath. Violin plots were created from n = 10 images per mouse, and the median and quartile of all images are shown. Each triangle represents the mean of all images per mouse. A two-tailed Mann–Whitney test was used, comparing the mean of all biological samples. We hypothesized that developmental programming of KCs by maternal obesity impairs their homeostatic function and promotes FLD via paracrine signalling. To test this hypothesis, we co-cultured hepatocytes with KCs from CDMCDLCD or HFDMCDLCD livers without direct cell contact, ensuring that only paracrine signalling occurred (Fig. Real-time lipid accumulation in hepatocytes was monitored using LD540 labelling to visualize neutral lipids via live-cell imaging. Hepatocytes co-cultured with HFDMCDLCD KCs exhibited significantly greater lipid accumulation, as indicated by increased LD540 intensity after 4 h, compared with those co-cultured with CDMCDLCD KCs (Fig. Thus, KCs that have experienced dietary factors from obese mothers during embryogenesis exert their FLD-promoting function via paracrine signalling. To further validate KC-dependent lipid accumulation in hepatocytes in vivo, we utilized the KC–diphtheria toxin receptor (DTR) mouse model32, in which human DTR is expressed under the control of the KC-specific Clec4f promoter, enabling targeted depletion of KCs within 24 h following a single dose of diphtheria toxin (Extended Data Fig. We generated HFDMCDLCD offspring with either the Clec4fDTR/+ or Clec4f+/+ genotype. At postnatal day 0 (P0), diphtheria toxin was injected to deplete KCs in Clec4fDTR/+, followed by transplantation of tdT+ bone marrow monocytes and HSPCs from naive mice at P1 to replenish the KC niche (Fig. We confirmed the presence of tdT+ KC-like cells in the adult liver of Clec4fDTR/+ mice, despite a large fraction of KC-like cells originating also from endogenous tdT− progenitors (Extended Data Fig. This treatment effectively rescued the fatty liver phenotype in Clec4fDTR/+ HFDMCDLCD mice, as evidenced by reduced lipid droplet accumulation in hepatocytes, demonstrated by ORO and H&E staining (Fig. By contrast, Clec4f+/+ littermates receiving the same diphtheria toxin injection and cell transfer retained lipid accumulation, consistent with observations in C57BL/6Jrcc mice (Fig. Moreover, replenishment of the empty KC niche solely by endogenous monocytes or HSPCs and residual KCs (Clec4fDTR/+ mice injected with diphtheria toxin but without any cell transfer from a naive source), did not rescue the fatty liver phenotype (Fig. 5e), indicating that HSPCs themselves may be developmentally programmed, as previously reported33,34. As HIF1α regulates the metabolic switch of macrophages from oxidative phosphorylation to glycolysis35,36, we tested whether preventing HIF1α-dependent developmental programming of KCs by maternal obesity would rescue the FLD phenotype. As maternal diet during lactation showed minimal impact (Fig. 1), we focused on the dietary switch after birth in the maternal obese groups and used HFD-fed animals after weaning as control groups (Fig. Both Hif1af/f (wild type (WT)) and LysMCre;Hif1af/f (KO) littermates gained body weight and WAT weight when the end diet was a HFD (CDMHFD and HFDMHFD), indicating that myeloid-specific loss of HIF1α does not prevent diet-induced obesity (Extended Data Fig. a, Generation of the maternal obese model with myeloid Hif1a ablation. b, Representative liver ORO staining of n = 8, 5, 8, 6, 9, 7, 6 and 9 mice for CDMCD WT, CDMCD KO, CDMHFD WT, CDMHFD KO, HFDMCD WT, HFDMCD KO, HFDMHFD WT and HFDMHFD KO groups, respectively, on 15 experimental days. c, ORO staining quantification of panel b by QuPath. Violin plots were from n = 1–10 images per mouse, and the median and quartile of all images are shown. Each circle or triangle represents the mean of all images per mouse. One-way ANOVA with Tukey's multiple comparison test was used. d, Liver lipidomics visualized by PCA. e, Bulk RNA-seq analysis of sorted KCs visualized by PCA. f, Chord diagram showing KC ligands (in orange; DEGs between HFDMCD WT and HFDMCD KO in panel e) and their respective receptors (in brown) expressed by hepatocytes. g, Venn diagram showing differentially expressed proteins from KCs comparing HFDMCD WT versus CDMCD WT (light blue), HFDMCD WT versus HFDMCD KO (light orange) mice, and overlapping proteins of the two comparisons (‘Cross'). n = 3, 3, 4 and 3 mice for CDMCD WT, CDMCD KO, HFDMCD WT and HFDMCD KO groups, respectively, on 2 experimental days. ANOVA multiple-sample testing (S0 = 0.5, false discovery rate < 0.1 with 250 randomizations) was used. Next, we focused on the liver phenotype. 6a), and Hif1af/f offspring born to obese mothers (HFDMCD WT) developed a FLD, as observed by ORO staining (Fig. Similar to the lipidomics results in C57BL/6Jrcc mice, HFDMCD WT offspring showed an increase of saturated triacylglycerols and cholesterol esters. However, HFDMCD KO offspring was rescued from developing a FLD (Fig. 6b,c), indicating that depletion of HIF1α in KCs during embryogenesis is sufficient to prevent their metabolic programming, which ultimately drives lipid accumulation in hepatocytes. This effect was diminished when KO offspring was kept on a HFD after birth (HFDMHFD KO), as they showed a similar development of FLD as HFDMHFD WT with high lipid accumulation (Fig. 3b,c) and significant increase of triacylglycerols, diacylglycerols, monoacylglycerol and cholesterol esters (Extended Data Fig. Hepatocyte transcriptomes reflected these differences (Extended Data Fig. To assess a possible transient activation of HIF1α by maternal obesity specifically during gestation that may have long-lasting effects, we performed RNA-seq of KCs isolated from C57BL/6Jrcc mice at P0 born to CD-fed or HFD-fed mothers. Comparing differentially expressed genes (DEGs) from CD and HFD conditions resulted in 54 regulated genes (Extended Data Fig. To investigate gene regulation patterns, we inferred transcription factor activity on the basis of all DEGs and a collection of transcriptional regulatory interactions38, indicating that DEGs were downstream of HIF1α, among other transcription factors (Extended Data Fig. Indeed, a total of five HIF1α target genes (Hif3a, S100a8, S100a9, Tgfb3 and Vegfa) were among the 29 upregulated DEGs (Extended Data Fig. 7c), and HALLMARK analysis of these 29 upregulated DEGs revealed that they fall into the term ‘hypoxia' (Extended Data Fig. Of note, only Hif3a expression was regulated, whereas genes encoding other HIF proteins were not transcriptionally regulated (Extended Data Fig. To validate that HIF1α is transiently active in KCs, we evaluated HIF1α protein expression in KCs of P0 neonate mice born to either HFD or CD mothers (Extended Data Fig. Quantification of immunofluorescence staining in the nucleus compared with the cytoplasm showed an increase of HIF1α translocation to the nucleus in maternal obese neonate mice when compared with maternal lean neonate mice (Extended Data Fig. These data indicate that KCs react to maternal obesity by increased HIF1α translocation to the nucleus, leading to the upregulation of HIF1α target genes at birth. Together, our results show that the development of KCs from the offspring is perturbed by maternal obesity in a HIF1α-dependent manner, and that ablation of HIF1α function in fetal KCs rescues the development of FLD during adulthood caused by maternal obesity. As FLD induced by maternal obesity was only rescued by Hif1a deletion if the offspring was kept on a CD, we asked whether additional factors, such as inflammation, may add to the hepatic lipid accumulation at postnatal stages when the offspring is fed with a HFD. To address whether hepatocytes may attract immune cells and cause a slowly increasing local inflammation in the liver when experiencing a HFD throughout life, we used the hepatocyte transcriptome data and investigated expression of chemoattractant factors. Indeed, we found Agt (encoding angiotensin), C3 (encoding complement C3), Lect2 (encoding leukocyte cell-derived chemotaxin 2) and Cxcl12 (encoding C-X-C motif chemokine 12) to be particularly upregulated in hepatocytes isolated from HFDMHFD WT and/or HFDMHFD KO groups, which are involved in attracting monocytes and neutrophils39,40,41 (Extended Data Fig. In line with these findings, flow cytometry analyses showed an, albeit not always significant, increase in the number of recruited myeloid cells (monocytes, neutrophils, cDC1 and cDC2; Extended Data Fig. Of note, deletion of Hif1a did not decrease myeloid cell numbers compared with WT littermates in any of the dietary conditions (Extended Data Fig. To investigate how HIF1α-dependent developmental programming in KCs causes FLD, we analysed the transcriptomes of WT and KO KCs from maternal lean and maternal obese conditions. Only KCs from HFDMCD KO mice clustered with maternal lean conditions (Fig. 3e), demonstrating that developmental programming events caused by maternal obesity are HIF1α dependent. Focusing specifically on HIF1α-dependent gene regulation, comparing HFDMCD WT versus HFDMCD KO KCs, yielded 409 DEGs (adjusted P value of 0.1; Supplementary Table 4). GO term analysis of DEGs downregulated in HFDMCD KO KCs revealed terms such as ‘fatty acid metabolic process', ‘cellular response to xenobiotic metabolism' and ‘blood coagulation, fibrin clot formation' (Extended Data Fig. As ex vivo culture data indicated that KCs cause lipid accumulation via paracrine factors (Fig. 2d,e), we searched for secreted ligands with corresponding hepatocyte receptors. Among 25 differentially expressed ligands, the majority were upregulated in HFDMCD WT compared with HFDMCD KO KCs (Fig. To validate HIF1α-dependent upregulation of KC-derived ligands at the protein level, we performed proteomics on isolated KCs from CDMCD WT, CDMCD KO, HFDMCD WT and HFDMCD KO offspring. These analyses identified 66 and 59 differentially expressed proteins, respectively, as well as 147 proteins that were both HIF1α dependent and induced by maternal obesity (Cross list; Fig. GO, HALLMARK and KEGG analyses of these proteins revealed pathways such as ‘triglyceride catabolic process', ‘complement and coagulation cascades', ‘insulin-like growth factor receptor signalling pathway' and ‘ECM receptor interaction' among the Cross proteins. In addition, lipid transport pathways were enriched in the WT and HFDMCD lists, whereas immune-related terms were specifically enriched in the HFDMCD list (Extended Data Fig. A heatmap visualizing Cross proteins (Fig. 3h) showed that HIF1α deficiency in KCs prevented the upregulation of many proteins, including apolipoproteins and coagulation factors, in response to maternal obesity. Together, these transcriptional and proteomic analyses identified several HIF1α-dependent candidates induced by maternal obesity, potentially acting as KC-derived paracrine signals driving lipid accumulation in hepatocytes. To determine whether maternal obesity-induced changes persist into adulthood and to avoid potential contamination from sorted KCs42, we performed single-nucleus RNA (snRNA)-seq–assay for transposase-accessible chromatin using sequencing (ATAC-seq) on nuclei isolated from 12-week-old C57BL/6Jrcc HFDMCDLCD and CDMCDLCD offspring. This allowed for the epigenetic and transcriptomic analysis of individual cells. Integration of RNA-seq and ATAC-seq data resulted in various hepatic cell types, including KCs and hepatocytes (Extended Data Fig. Subclustering the KCs revealed five distinct KC states (Fig. Increased MHC class II expression was validated in the HFDMCDLCD liver parenchyma, whereas CDMCDLCD mice showed restricted MHC class II expression around the periportal and central veins of the liver (Extended Data Fig. a, snRNA/ATAC-seq of CDMCDLCD and HFDMCDLCD mice visualized by uniform manifold approximation and projection (UMAP) showing clusters depicting five different KC states. b, Dotplot showing genes specifically expressed by each KC state. c, Heatmap showing predicted transcription factor activity in each cluster of both CDMCDLCD and HFDMCDLCD groups. d, Violin plots comparing Apoe (left) and Apoa1 (right) expression in all KC states between CDMCDLCD and HFDMCDLCD groups. Two-tailed Wilcoxon rank-sum test was used. e, ATAC-seq peak coverage plot of the Apoe locus comparing CDMCDLCD and HFDMCDLCD groups for each KC state. A possible differentially exploited regulatory region controlling Apoe expression is highlighted in red. Transcription factor-binding sites of PPARγ–RXRα are highlighted with a blue vertical line and of PPARα with green vertical lines. f, Lipid accumulation shown by normalized LD540 intensity through live imaging in ex vivo-cultured hepatocytes supplemented without any factor (control), with TNF, APOE or APOA1. n = 4 mice per group with technical triplicates. One-tailed ratio paired Student's t-test on biological samples comparing treated groups to control was used. Comparisons of ATAC-seq peak-related genes between HFDMCDLCD and CDMCDLCD KCs identified 28 downregulated and 60 upregulated gene loci (Extended Data Fig. Despite the relatively low numbers, GO term analyses of downregulated genes indicated enrichment of lipid metabolic processes (Extended Data Fig. Of note, ATAC-seq peaks were differentially present within the Pck1 locus, a gene encoding phosphoenolpyruvate carboxykinase 1, which is an enzyme that typically has an antagonistic role to glycolysis. Plotting the gene expression validated the loss of Pck1 expression across all HFDMCDLCD KC states (Extended Data Fig. 9f), aligning with the metabolic switch described earlier (Fig. Furthermore, we observed epigenetic changes within the Gpnmb locus where ATAC-seq peaks increased significantly. Of note, Gpnmb has been previously described as a biomarker for steatohepatitis43. Consistent with bulk RNA-seq results, this epigenetic change was accompanied by upregulation of Gpnmb gene expression (Extended Data Fig. However, this upregulation was restricted only to clusters 2 and 3, which expand proportionally under maternal obesity conditions. To investigate whether these distinct KC states are differentially regulated at the transcriptional level, we performed DEG analysis comparing the clusters (Supplementary Table 7), combined with DecoupleR transcription factor activity analysis. This revealed that KC states clustered by dietary condition rather than identity, indicating that persistent changes in transcription factor activity were driving the observed DEGs in HFDMCDLCD KCs (Fig. Transcription factor activity analysis identified HIF1α to be mainly active in clusters 1, 2 and 3 in the HFDMCDLCD condition. Transcription factors, such as Jun, Ets1 and Esr2, were more active in most HFDMCDLCD clusters, with PPARα and PPARγ showing increased activity across all HFDMCDLCD clusters (Fig. These findings suggest that, in addition to HIF1α, other persistent transcriptional changes stem from altered transcription factor activity in offspring born to obese mothers. Next, we leveraged the snRNA-seq data to assess signalling pathways that are activated in HFDMCDLCD KCs using CellChat analysis. In line with the bulk RNA-seq data, KCs demonstrated increased intercellular communication via APOE and APOA1 signalling pathways under maternal obesity conditions (Extended Data Fig. Gene expression analyses confirmed upregulation of Apoe and Apoa1 in all HFDMCDLCD states (Fig. 4d), consistent with increased transcription factor activity of PPARα and PPARγ, which are known regulators of apolipoproteins44. In addition, ATAC-seq analysis revealed an increase in peaks at putative regulatory elements located upstream of the Apoe promoter in HFDMCDLCD cluster 3 compared with CDMCDLCD, which harbour PPARα and PPARγ–RXRα transcription factor-binding sides (Fig. Together, our data show that maternal obesity induces persistent changes in KCs. Of note, we identified new KC states (clusters 2 and 3) that probably account for the immune-related signature observed in the bulk RNA-seq data. These clusters expand in response to maternal obesity and specifically upregulate inflammatory genes. Furthermore, the snRNA–ATAC-seq approach corroborates our findings from bulk RNA-seq and proteomics approaches, highlighting the significant upregulation of apolipoproteins in KCs. To investigate whether apolipoproteins contribute to lipid accumulation in the liver, we utilized the hepatocyte live-imaging cell culture approach, adding APOE or APOA1 to the medium. As a positive control, we used tumour necrosis factor (TNF), which has been previously identified as a KC-derived driver of lipid storage in hepatocytes45. In all conditions, the LD540 signal intensity increased, with APOE inducing the highest lipid accumulation compared with control hepatocytes (Fig. In summary, our findings provide evidence that HIF1α-dependent developmental programming by maternal obesity leads to the upregulation of apolipoproteins in adult KCs, which can induce lipid accumulation in hepatocytes, thereby contributing to the onset of FLD. This work supports the hypothesis that yolk sac-derived tissue-resident macrophages are cellular components within the concept of DOHaDs, as developmental perturbations of hepatic macrophages result in postnatal liver pathophysiology. This is in line with previous studies showing that microglia can contribute to neurodevelopmental and behavioural disorders8,9,46,47,48. Our data indicate that continuous exposure to a maternal HFD during gestation induces a HIF1α-dependent metabolic switch in KCs from oxidative phosphorylation to glycolysis, detrimentally affecting adult hepatic lipid metabolism. Unlike most maternal immune activation studies, in which cytokines such as IL-6 and IL-17a influence offspring development49,50, our model does not indicate a maternal immune activation phenotype or systemic inflammation. For instance, stearic acid, enriched in obese mice, has been proposed to reprogram macrophages via Toll-like receptor 4 (TLR4)9,51. An intriguing question for future research is to administer defined fatty acids to pregnant dams and trace their effect on fetal KC development in a HIF1α-dependent manner. Using the LysMCre mouse model to delete Hif1a specifically in myeloid cells excludes developmental programming of oocytes, hepatocytes and their progenitors as primary driver of FLD, as the promotor controlling expression of the Cre recombinase only becomes active during early fetal macrophage differentiation1,52. However, indirect effects via inter-organ communication, such as adipose-derived signals, cannot be ruled out. Our double fate mapper model showed that KCs remain of yolk sac origin upon maternal obesity, contrasting with diet-induced FLD models in which bone marrow-derived cells replace KCs16,17,18. Thus, in accordance with DOHaDs, KCs can serve as intergenerational messengers that sense maternal-derived factors during gestation and convey this information into a long-lasting transcriptional program during adulthood. Our transcriptomic, epigenetic and metabolic analyses demonstrate that maternal obesity functionally programs KCs, promoting lipid accumulation in neighbouring hepatocytes via persistent transcriptional changes. KC depletion experiments further confirmed this: replenishment with monocyte-derived cells from animals that never experienced a HFD rescues the developmental programming effects, providing a potential cellular target for treating FLD caused by maternal obesity. Replenishment with monocytes from maternal obesity-experienced animals did not rescue the phenotype, suggesting that neonatal monocytes or HSPCs undergo developmental programming33,34. A previous study has shown that upon a postnatal HFD, KCs secrete pro-inflammatory factors, such as IL-1β, IL-6 and TNF, with TNF inducing lipid accumulation in hepatocytes45. By contrast, in maternal obesity-induced FLD, we observed no upregulation of these particular factors. Instead, genes associated with lipid metabolism (for example, Trem2, Tyrobp, Syk and Lpr1) were upregulated, partially resembling the phenotype of hepatic lipid-associated macrophages (LAMs)17. Recent work has also shown that TREM2+ LAMs expand during injury and can originate from both resident KCs and monocyte-derived macrophages53. In our model, however, LAM-like KCs remain of yolk sac origin, suggesting that functional adaptation to lipid-rich environments, rather than a change in cellular origin, drives their phenotype. This distinction highlights the unique nature of maternal obesity-induced FLD compared with postnatal HFD models. Notably, maternal obesity upregulates apolipoproteins such as APOE and APOA1 in a HIF1α-dependent manner, aligning with previous findings linking apolipoproteins with FLD54,55. In summary, our study establishes KCs as active players in liver metabolism, demonstrating distinct responses to a maternal-derived versus postnatal diet-derived HFD. We showed that a maternal HFD during gestation programs KCs metabolically and transcriptionally in a HIF1α-dependent manner, leading to persistent changes that promote FLD. These findings offer a therapeutic time window to manipulate the metabolic status of KCs, and thereby prevent the development of FLD in the offspring. All investigations concerning mouse work had local approval and all procedures conformed to the guidelines from Directive 2010/63/EU of the European Parliament on protecting animals used for scientific purposes. In detail, all animal experiments were conducted according to the German law of animal protection and in agreement with the approval of the local institutional animal care committee (Landesamt für Natur, Umwelt und Verbraucherschutz (LANUV), North Rhine-Westphalia, Az 81-02.04.2019.A146 and Az 2024-A314). Mice were housed under specific pathogen-free conditions with 12-h light–dark cycle, at 21 °C, 55% relative humidity, and with food and water provided ad libitum. Animals were euthanized using liver or heart perfusion after anaesthesia injection. C57BL/6JRcc was used as the WT strain. To generate the double fate mapper Tnfrsf11aCre (ref. 56); Rosa26LSL-YFP (JAX stock #006148)57; Ms4a3FlpO;Rosa26FSF-tdT (JAX stock #032864)58, we bred Tnfrsf11aCre/+;Rosa26FSF-tdT/FSF-tdT animals with Ms4a3FlpO;Rosa26LSL-YFP/LSL-YFP animals (see Extended Data Fig. All mice were originally of the C57BL/6J background and crossed to C57BL/6Jrcc for 2–5 generations. To generate Hif1aflox/flox;LysMCre/+, the Hif1aflox/flox (JAX stock #007561)59 and LysMCre/+ (JAX stock #004781)37 were used (both kept on the C57BL/6J background after import). All lines were back-crossed to their respective WT line (C57BL/6J for LysMCre/+;Hif1aflox/flox and C57BL/6Jrcc for all other lines) once per year. Mice were genotyped according to protocols provided by JAX or donating researchers. For all experiments, male mice were analysed with the exception of the double fate mapper, where both male and female mice were used, and ex vivo-isolated hepatocytes, where only young female mice were used to assure a lean and metabolically healthy state. Subsequently, female mice were separated into two groups and received either a HFD (sniff, E15742-347) or a CD for 8 weeks. For metabolic assessment, the HOMA-IR was used60. Glucose was measured using GlucoCheck GOLD with stripes (11864873 and 11864933), and insulin was measured via ELISA (EMINS, Thermo Fisher). Only mice with significantly higher HOMA-IR were used to generate offspring. Females were always placed back in their cages with a HFD after overnight mating. All offspring were cross-fostered to females on the respective diet. The litter size was standardized to a maximum of six pups per foster mother on the day of birth. In addition, whenever possible, litters from different biological mothers were cross-fostered to the same foster mother to ensure consistency and minimize maternal variability. Cross-fostering took place also between mothers on the same diet to exclude stress-mediated effects of cross-fostering. After 3 weeks of lactation at postnatal day 21 (P21), offspring were weaned into their post-weaning cage (CD or HFD). Here offspring from maternal lean and maternal obese groups were mixed to minimize effects caused by cage-specific microbiome differences. In addition, during the post-weaning diet, bedding stemming from cages with the same diet was mixed weekly between cages (typically 10–20 cages at any time) to further reduce cage-specific effects related to the microbiome. Mice were euthanized and used in the experiment at weeks 11–13, unless otherwise stated. Maternal obese KC-DTR male offspring were injected at P0 with 50 ng diphtheria toxin and injected at P1 through the temporal vein under ice-induced anaesthesia with 106 bone marrow monocytes and HSPCs isolated from Rosa26mTmG mice. Bone marrow monocytes and HSPCs were isolated as following: Rosa26mTmG mice were killed through cervical dislocation, and their tibias and femurs from the hindlimbs were freed. Bone marrow was flushed out with DMEM (Pan Biotech) and filtered through a 70-μm strainer on ice. Cells were then blocked with FACS buffer (0.5% BSA and 2 mM EDTA in PBS) containing 2% rat serum and stained with biotinylated antibodies (anti-Ly6G, CD19, CD3, Ter119, Nkp45 and F4/80) for 30 min at 4 °C. Monocytes and HSPCs were then negatively selected using MojoSort Streptavidin Nanobeads (BioLegend) following the manufacturer's manual. Adult mice were anaesthetized and perfused with ice-cold PBS. For flow cytometry analysis of hepatic myeloid cells, 200–300 mg of adult or half of a P0 liver was collected, cut into small pieces and incubated in a digestion mix (PBS containing 1 mg ml−1 collagenase D (11088858001, Roche), 100 U ml−1 DNase I (DN25, Sigma-Aldrich), 2.4 mg ml−1 of dispase (17105041, Gibco) and 3% fetal calf serum (FCS; Invitrogen) for 30 min at 37 °C before mechanical disruption through a 100-μm filter. The cell suspension was diluted in 3 ml of FACS buffer (0.5% BSA and 2 mM EDTA in PBS) and centrifuged at 50g for 3 min to remove hepatocytes. Then, the supernatant containing myeloid cells was collected and centrifuged at 370g for 7 min at 4 °C. For bone marrow, one leg was isolated, and both the tibia and the femur were cleaned from surrounding tissues and cut at the ends. The bones were flushed with 5 ml of ice-cold FACS buffer. Then, 5 ml FACS buffer was added, cells were resuspended and centrifuged at 370g for 7 min at 4 °C. Cell pellet was resuspended in FACS buffer containing purified anti-CD16/32 and 2% rat serum (liver) or 2% rat serum only (bone marrow) and incubated for 15 min at 4 °C. Samples were immune-stained with antibody mixes for 30 min at 4 °C. The complete list of antibodies used can be found in Supplementary Table 8. Data were acquired with FACSymphony A5 (BD Biosciences) or LSRII and analysed in FlowJo Software. For sorting of KCs and hepatocytes, liver cell suspension was prepared as described above with the following modifications: first, 1 ml twice-concentrated digestion mix was used for tissue digestion; second, digestion was performed at room temperature and centrifugation at 50g for 3 min was omitted to retain hepatocytes; and last, all steps were performed with buffers containing 1 mM of flavopiridol (F3055, Sigma-Aldrich). Cells were stained by antibody mixes and sorted using FACS ARIA III cell sorter (BD Biosciences) directly into 500 μl lysis buffer (79306, Qiagen). For KC metabolic flow cytometry assays, single-cell suspensions were blocked with anti-CD16/32 (1%) and rat serum (2%) in FACS buffer, followed by surface marker staining for 30 min at 4 °C. Cells were washed and fixed in 1% paraformaldehyde for 5 min and then permeabilized for 15 min with PBS supplemented with 0.4% Triton X-100 (X100, Sigma-Aldrich). Intracellular staining was performed for 1 h at 4 °C. Intracellular antibodies were conjugated in-house using lightning-link conjugation kits (Abcam) as previously described26. Cells were analysed on an ID7000 7-laser Spectral Cell Analyzer (Sony Cooperation). For neutral lipid staining, cryo-preserved liver blocks were cut at 10 µm thickness and dried for 1 h at room temperature. Lipid staining was performed using an Oil-red-O Stain Kit (O0625, Sigma-Aldrich) according to the manufacturer's instructions. The sections were then stained with haematoxylin for 5–10 min to visualize nuclei, rinsed with distilled water and mounted with Kaiser's glycerol gelatine (pre-heated at 55 °C; 6474.1, Carl Roth). Images were taken with an Axio Lab.A1 microscope (Zeiss). For ORO staining quantification, 2–10 images of each sample with a ×40 objective were taken and quantification was performed using QuPath software (v0.5.1). For haematoxylin and eosin (H&E) staining, paraffin-embedded liver blocks were cut at 5 µm thickness and heated for 1 h at 65 °C before staining. Sections were deparaffinized by two steps of xylol and descending series of alcohol and rinsed in distilled water. Next, sections were stained with haematoxylin solution for 1.5 min and neutralized by running tap water, followed by alcoholic eosin staining for 2 min. Sections were then dehydrated by an ascending series of alcohol and two steps of xylol and mounted with Entellan (1.07961.0100, Millipore). Images were taken with the Axio Lab.A1 microscope. Quantification of droplet adiposity in liver cells in H&E-stained histological sections was performed manually using a light microscope without the support of digital imaging or software tools. From each section, 10 random high-power fields were selected at ×10 magnification to ensure a representative selection of liver parenchyma while avoiding areas of artefacts or tissue damage. Hepatocytes were analysed for the presence of lipid droplets. All hepatocytes in each selected field were counted and classified as either normal (without lipid droplets) or lipid laden (with lipid droplets recognizable as intracellular vacuoles). The percentages of all analysed fields were averaged to obtain a representative value for each sample. For immunofluorescence staining, cryo-preserved liver blocks were cut at 10 µm thickness, dried for 1 h at room temperature and permeabilized with PBS-T (0.4% Triton X-100; X100, Sigma-Aldrich) for 1 h at room temperature. Thereafter, tissue was blocked with PBS-T containing 2% BSA (37525, Thermo Scientific) and 2% donkey serum (D9663, Sigma-Aldrich) for 2 h at room temperature, followed by primary antibody staining overnight at 4 °C, and secondary antibodies hosted in donkey for 2 h at room temperature. Nuclei were stained with DAPI (42281, BioLegend). Immunofluorescence images were acquired with a Zeiss LSM Airyscan 880 microscope (Zeiss) and processed with Fiji61. Livers were isolated from ice-cold PBS-perfused mice and snap-frozen in liquid nitrogen. Ten milligrams were homogenized in ddH2O using a Precellys homogenizer (Peqlab Biotechnology). The supernatant was collected into a new tube, and 200 µl chloroform and 800 µl 1% AcOH water were added. The sample was shaken and centrifuged for 2 min at 20,000g. The upper aqueous phase was discarded and the lower phase was transferred into a new tube and evaporated in a speed vacuum concentrator (45 °C at 10 min). Spray buffer (500 µl of 8:5:1 2-propanol:MeOH:H2O and 10 mM ammonium acetate) was added to the sample and sonicated for 5 min, infused at 10 µl min−1 into a Thermo Q Exactive Plus spectrometer equipped with the heated electrospray ionization II ion source for direct infusion lipidomics. MS1 spectra (resolution of 280,000) were recorded in 100-m/z windows from 250 to 1,200 m/z (positive) and 200 to 1,700 m/z (negative) followed by recording tandem mass spectrometry spectra (resolution of 70,000) by data-independent acquisition in 1-m/z windows from 200 to 1,200 (positive) and 200 to 1,700 (negative) m/z. Raw files were converted to .mzml files and imported into and analysed by LipidXplorer software (v1.2.8.1)62 using custom mfql files to identify sample lipids and internal standards. Raw measurements of the lipidome were quantified in relation to internal standards measured simultaneously on the same instrument. The resulting values were normalized by the amount of tissue to obtain measurements in pmol mg−1. All lipid species that remained constant across all samples were removed from the dataset. In addition, species belonging to the triacylglycerol class with odd numbers of double bonds were excluded, as they were unlikely to originate from mice. This resulted in a set of 19 lipid classes: cholesteryl ester, ceramide, diacylglycerol, dihexosylceramide, hexosylceramide, LPC, ether-linked LPC, lysophosphatidylethanolamine, monoacylglycerol, phosphatidic acid, PC, ether-linked PC, PE, ether-linked PE, phosphatidylglycerol, phosphatidylserine, sphingomyelin, unsaturated fatty acyl tails of triacylglycerol (unsat) and saturated fatty acyl tails of triacylglycerol (sat). For each lipid class, the conditions were assessed for significant deviations from the control condition using the lm function in R. Subsequently, a hierarchical clustering analysis was performed on the resulting dataset, both row wise and column wise, based on the Euclidean distance with complete linkage. This deviation arose from significant changes with low-effect sizes. The Immune Monitoring 48-Plex Mouse ProcartaPlex Panel (EPX480-20834-901, Invitrogen) was used. Of sera, 25 µl was thawed and analytes were evaluated using Luminex xMAP system. All reagents were prepared and used according to the manufacturer's instructions. Cytokines that were below the detection threshold in more than 80% of the samples were not plotted. The values displayed within the heatmap are log10 + 1 transformed and ordered by decreasing overall mean. cDNA library for sequencing was prepared as described in the SMART-Seq2 protocol63. The mRNA was extracted and primed utilizing poly-T oligonucleotides and converted into cDNA by SMART reverse transcription. Pre-amplification was performed by SMART ISPCR, followed by fragmentation using the Nextera XT DNA Library Preparation kit (Illumina), amplification and indexing. Library fragments were then selected by size (300–400 bp) and purified using SPRIBeads (Beckman-Coulter). The Agilent high-sensitivity D5000 assay was used to measure the size distribution of cDNA libraries on a Tapestation 4200 system (Agilent). Quantifying cDNA libraries was performed using a Qubit high-sensitivity dsDNA assay (Thermo Fisher). Sequencing was performed using a 75-bp single-end setup on the NextSeq500 system (Illumina), applying NextSeq 500/550 High Output Kit (v2.5; Illumina). Library fragments were then selected by size (300–400 bp) and purified using SPRIBeads (Beckman-Coulter). The Agilent high-sensitivity D5000 assay was used to measure the size distribution of cDNA libraries on a Tapestation 4200 system (Agilent). The cDNA libraries were quantified using a high-sensitivity dsDNA assay (Qubit). To quantify the abundances of transcripts from the bulk RNA-seq data, Kallisto pseudo alignment was applied64. The raw transcriptome files were prepared for Kallisto import to DESeq2 using the genecode annotation M16 (https://www.gencodegenes.org/mouse/release_M16.html) to correct for library size based on the provided average transcript length. Outliers (eight from the C57BL/6Jrcc dataset and three from the Hif1a dataset) were identified on the basis quality assessment and filtering of low-quality samples. The set of genes was further refined to include only protein-coding genes that were non-constant across all present samples. Low-expressed genes, defined as those with less than 10 counts in 25% of the samples, were also removed. PCA was performed on the variance-stabilized counts. 65) analysis was conducted to identify DEGs within each cell type. The default settings in DESeq2 were used. A gene was called differentially expressed if its corresponding Benjamini–Hochberg adjusted P value was less than 0.1 for the tested contrast. For the KC P0 DEGs subjected to transcription factor inference analysis, the absolute LFC threshold was set to 2. Each set of DEGs was subjected to an over-representation analysis to identify enriched GO terms, HALLMARK pathways and KEGG pathways using the respective functions in clusterProfiler66. To reduce observed redundancy in the GO terms, the ‘simplify' function in clusterProfiler was used, which calculates the similarity among enriched terms based on their information content and selects a representative term with the lowest adjusted P value67. For the ligand–receptor analysis, DEGs with an absolute LFC greater than 2 in the KC set were subsetted to include ligands recorded in the CellTalk database68. For the hepatocyte dataset, all expressed genes (more than 10 counts in Hif1a-WT or Hif1a-KO HFDMCD conditions; not necessarily differentially expressed) were subsetted to include recorded receptors within the same database. The order of ligands was based on their LFC from lowest to highest. A co-expression network analysis using hCoCena25 was conducted on the filtered and DESeq2 normalized counts, following the provided showcase Notebook25. hCoCena calculates pairwise correlations among the top most varying genes (the number needs to be user-specified) and constructs a network where vertices represent genes with at least one edge characterized by a correlation above a specified threshold. Here hCoCena was performed using the top 5,000 genes as input and all edges were kept with a correlation above 0.7. Each node that was advised to more than three clusters during the process was disregarded in the proceeding cluster analysis. The identified gene sets were summarized using Group Fold Change25 for each condition and displayed on a heatmap. Each cluster set was subjected to an over-representation analysis, with the hCoCena-input genes specified as the universe for testing. Manually selected terms from GO, KEGG and HALLMARK sets are displayed for each cluster to avoid term cluttering. For selected enriched terms, their intersection with the cluster genes was recovered, namely, ‘oxidative phosphorylation' with turquoise and lightgreen, ‘glycolysis' and ‘cell activation involved in immune response' with steelblue. The resulting set of genes was further manually reduced. The mean expression values for those intersectional genes were displayed and scaled across each condition. Transcription factor inference analysis was conducted using the CollecTRI38 framework, which integrates information from multiple sources, including RegNetwork70, ChEA3 (ref. Within this analysis, a linear model was used to estimate the transcription factor activity. A positive value indicates an active transcription factor. This analysis was conducted using decoupler's74 functionality. Nuclei were isolated from approximately 50 mg of snap-frozen adult liver tissue to enable simultaneous snRNA-seq and snATAC-seq, capturing both transcriptional profiles and chromatin accessibility. Nuclei were prepared using the Chromium Next GEM Single Cell Multiome ATAC + Gene Expression Kit (1000283, 10X Genomics). Diluted homogenate was slowly laid atop a cushion (75% of the volume of diluted homogenate) containing 29% OptiPrep, 77.5 mM KCl, 15.5 mM MgCl2, 31 mM Tris-HCl (pH 8) and 129.2 mM sucrose in UltraPure H2O and centrifuged with low acceleration and maximum deceleration at 10,000g at 4 °C for 30 min. The pellet containing nuclei was resuspended in 1X PBS containing 1% BSA, 0.2 U μl−1 RNase inhibitor (New England Biolabs) and 1 mM flavopiridol, stained with 7-AAD and purified by fluorescence-activated cell sorting using FACS ARIA III (BD Biosciences) for high-quality, monoploid and 7-AAD nuclei. For nuclei permeabilization, approximately 500,000 sorted nuclei were incubated for 2 min in a 0.1× lysis buffer (10 mM Tris-HCl (pH 7.4), 10 mM NaCl, 3 mM MgCl2, 0.1% IGEPAL CA-630, 0.1% Tween-20, 0.01% digitonin, 1 mM dithiothreitol, 1% BSA and 1 U μl−1 RNase inhibitor, with 1 μM flavopiridol in nuclease-free water). Following permeabilization and washing steps, nuclei concentration was adjusted to 5,000 nuclei per microlitre in ice-cold 1× nuclei buffer (2000153/2000207, 10X Genomics) and were incubated in a transposition mix containing a transposase, allowing adapter sequences to be added to the ends of DNA fragments. Furthermore, 16,000 nuclei were used for analysis with the Next GEM Single Cell Multiome ATAC + Gene Expression Kit (PN-1000230, 10X Genomics) and Gel Beads in Emulsion (GEMs) using the Chromium Next GEM Chip J Single Cell Kit. After GEM reverse transcription (GEM-RT) cleanup, pre-amplification of the sample was performed, producing material for both ATAC library construction and cDNA amplification for gene expression library construction. During the reverse transcription reaction, cDNA tagging was achieved with 16-nucleotide barcodes and 10-nucleotide molecular identifiers. The pre-amplified product was split into two aliquots: 40 μl for ATAC library construction and 35 μl for further cDNA amplification. For snRNA-seq library preparation, 25% of the total cDNA was used to generate gene expression (GEX) libraries. Libraries were prepared and sequenced according to the manufacturer's protocols: snATAC libraries used the 10X Genomics Single Index N Set, and snRNA-seq libraries used the Dual Index TT Set A. Pooled and barcoded libraries were sequenced on an Illumina NextSeq 2000 system with P3 flow cells, achieving an average depth of 30,000 reads per nucleus for both snRNA-seq and snATAC-seq libraries. The sequencing quality was assessed using FastQC (v0.12.1) executed within a containerized environment provided by biocontainers using Podman. All files passed initial quality assessment. Following assessment, Cell Ranger ARC (v2.0.0; 10X Genomics) was used for simultaneous alignment and quantification of both modalities. The automatically obtained quality assessment alerted for both conditions that the amount of valid ATAC barcodes was at 54% (ideally more than 80%). Despite the low proportion, both datasets were used because other critical ATAC quality metrics, such as fragment count and enrichment scores, met acceptable thresholds. Single-cell RNA-seq and ATAC-seq data were processed using mainly the Seurat and Signac packages in R. Both modalities were extracted from the 10X Genomics ‘filtered feature matrix file', in which count data were used for additional filtering of low-quality cells, before using MACS3 to perform the peak calling within the remaining cells to reduce computational time. Applied thresholds were: cells with RNA counts below 500, number of genes less than 200 or more than 2,500, and mitochondrial RNA content above 1% (as for single-nucleus sequencing, we expected none). Resulting peaks were also used for filtering of low-quality cells. Applied thresholds were: ATAC counts below 1,000, nucleosome signal more than 2 and transcription start site enrichment less than 1. In addition, peaks were removed that overlapped with non-standard chromosomes: genomic blacklist regions. Each RNA-seq dataset was pre-processed within each condition before integration using SCTransform, regressing out the percentage of mitochondrial DNA. The integration of RNA and ATAC data across conditions followed the outline from Kim et al.75. Accordingly, RNA data integration began with the selection of 3,000 integration features using SelectIntegrationFeatures in Seurat. The datasets were prepared for integration with PrepSCTIntegration, and integration anchors were identified using FindIntegrationAnchors with SCTransform (SCT) normalization. ATAC count integration started by creating a feature matrix for each condition based on a unified peak set that was shared across conditions. Dimensionality reduction for ATAC data relied on latent semantic indexing, which included feature selection of most variable peaks, transformation of peak counts and singular value decomposition. To improve the integration, Harmony was applied (RunHarmony). Harmony aligns datasets by projecting them into a shared low-dimensional space while minimizing the effects of batch-specific biases. For the final integration across modalities, multimodal neighbours were identified by combining RNA-based PCA and ATAC-based Harmony embeddings (FindMultiModalNeighbors) across 50 dimensions for each modality, without the first dimension for Harmony, as commonly done, aiming for additional batch correction. Clusters were identified using the weighted nearest neighbour similarity graph (FindClusters) with a resolution of 0.5. This approach allowed the simultaneous integration of RNA and ATAC data for both conditions while leveraging the complementary information from both modalities. Cluster-specific marker genes were identified by calculating differential expression for each cluster using FindAllMarkers in Seurat. Markers were filtered based on minimum expression percentage (0.10), minimum differential expression percentage (0.20) and log fold-change threshold (0.20), retaining only positive markers. Cell-type annotation was based on the expression of a curated gene set within the identified markers. Again, multimodal neighbours were identified (FindMultiModalNeighbors) using PCA for RNA and Harmony for ATAC, followed by UMAP for dimensionality reduction (RunUMAP) and clustering (FindClusters), as done for the dataset using all cell types. Subclusters with fewer than 50 cells were excluded due to low numbers, and markers for each remaining cluster were characterized using the FindAllMarkers function in Seurat with the number of neighbours set to 60. Subcluster characterization was based on the expression of a curated gene set within the identified markers. To analyse intercellular communication involving the differing KC subclusters to hepatocytes, a CellChat analysis was performed. Normalized RNA expression data were used to define communication networks between KC subclusters and other cell populations. Overexpressed genes and interactions were identified for each condition, identifying 1,366 (CDMCDLCD) and 1,458 (HFDMCDLCD) highly variable ligand–receptor pairs subsequently used for signalling inference. Communication probabilities were computed using the 10% truncated mean for calculating the gene expression per cell group. Interactions with fewer than ten participating cells were filtered out to ensure robustness. The KC-specific communication networks were compared across conditions using rankNet function. A paired Wilcoxon test was performed to determine whether there was a significant difference of the signalling information flow between two conditions. Potential activated transcription factors were investigated using the package decouplR. The resulting set was subjected to the unified linear model to identify potential regulators (transcription factors). Only regulators were tested, which had at least ten known targets among the union differentially expressed set. According to decouplR's vignette, the resulting score matrix, representing regulations strength, was scaled and centred. The 25% trimmed mean activity per cluster for each condition is displayed to provide a robust central tendency measure. Metabolomics analysis was conducted using data obtained from Metabolon. The Metabolon platform performed preprocessing steps on the data, including imputation of missing values and natural log transformation. Missing values for each metabolite were imputed with the minimum value observed across all samples. Finally, a natural logarithm transformation was applied to the metabolomic data, as it typically exhibited a log-normal distribution. The corrected data were then used for PCA analysis. To identify metabolites with significant effects of diet, genotypes or their interaction on their respective levels, an ANOVA was performed. Specifically, a contrast77 was applied within the ANOVA to compare metabolite levels between maternal obese (HFDMCDLCD) and maternal lean (CDMCDLCD) conditions. The resulting set of 129 metabolites with significant raw P values was further analysed using Metaboanalyst (https://www.metaboanalyst.ca/)78 for pathway over-representation analysis. The HMDB IDs provided by Metabolon were used as input, resulting in a reduction to 111 metabolites due to missing labels. Furthermore, mapping to Metaboanalyst failed for ten additional metabolites. Default parameters were used for all remaining settings, except for choosing Mus musculus (KEGG) as the pathway library and applying out-degree centrality as the topology analysis method. Eight-to-ten-week-old chow diet-fed C57BL/6JRcc female mice were used to isolate hepatocytes. Therefore, a two-step collagenase liver perfusion was performed. First, 7 min after injecting heparin-sodium (Ratiopharm; 30 units per gram body weight), mice were anaesthetized with ketamine–xylazine. Next, a 26 G indwelling venous catheter (0.62 mm × 19 mm; 391379, BD Neoflon Pro) was inserted into the portal vein. Perfusion was initiated with 25 ml Hank's balanced salt solution containing 0.5 mM EGTA (pH 7.4) at a rate of 2.5 ml min−1 using a peristaltic pump, and subsequently with 25 ml collagenase buffer (William's Medium E, P04-29510, Pan-Biotech) supplemented with 3.6 mM CaCl2, 0.1 U ml−1 collagenase NB46 (17465, Serva Electrophoresis) and 10 mM HEPES at 37 °C at a rate of 1.25 ml min−1. Liver cells were released by gentle swirling with forceps into 40 ml suspension medium (William's Medium E supplemented with 10% FCS, 2 mM l-glutamine, 100 units per millilitre penicillin and 100 µg ml−1 streptomycin). The pellet containing the hepatocytes was resuspended in fresh suspension medium and plated on collagen-coated 24-well plates for co-culture experiments with KCs at a density of 30,000 cells per well or on collagen-coated 96-well plates for factor screening experiments at a density of 5,000 cells per well. To ensure a viable hepatocyte culture, cells were pre-incubated for 3 h at 37 °C and 5% CO2 to allow for attachment. Thereafter, the medium was replaced with fresh suspension medium containing LD540 (0.02 µg ml−1) and vitamin C (1 µg ml−1) for live-cell imaging using the IncuCyte SX5 (Sartorius). To isolate KCs, 11–15-week-old mice were perfused with the same method as isolating hepatocytes but without the in vivo digestion step. After perfusion, liver was minced and digested in a digestion mix (PBS containing 1 mg ml−1 collagenase D, 100 U ml−1 DNase I, 2.4 mg ml−1 dispase and 3% FCS) for 30 min at 37 °C without agitation. Digested liver was further disrupted with gentle pipetting, filtered through a 100-µm cell strainer and centrifuged at 400g for 10 min at 4 °C. Pellet was washed with ice-cold FACS buffer (PBS containing 0.5% BSA and 2 mM EDTA) and centrifuged at 50g for 3 min at 4 °C to remove hepatocytes. The supernatant was then further filtered and washed with FACS buffer to obtain a clean, single-cell suspension and centrifuged at 400g for 10 min at 4 °C. The resulting pellet was then blocked with 1% anti-mouse CD16/32 (101302, BioLegend) and 5% rat serum (C13SDZ, Bio-Rad) for 10 min at 4 °C followed by incubation with biotinylated CD11b antibody (101204, BioLegend) for 30 min at 4 °C. Cells were then washed, incubated with pre-washed streptavidin-conjugated magnetic beads (480016, BioLegend), washed and enriched in a 12-well untreated cell-culture dish mounted atop a magnet. Enriched cells were abundantly washed in FACS buffer and subsequently counted using a Neubauer chamber. Cells (3 × 105) were plated onto a 13-mm diameter, untreated glass coverslip (VWR; co-culture experiments) or 2–4 × 106 cells onto an untreated glass slide (Fisher Scientific; proteomics experiments). Cells were attached for 2 h in macrophage medium (DMEM, containing 1% penicillin–streptomycin, 10% FBS and 1 mM sodium pyruvate) in a cell culture incubator at 37 °C with 5% CO2. After attachment, cells were abundantly washed with PBS. Macrophages on coverslips were used for co-culture experiments as described below. Macrophages for proteomics experiments were scraped off in PBS, centrifuged at 400g at 4 °C for 5 min to obtain pellets, snap-frozen in liquid nitrogen and further processed for proteomics as described below. For hepatocyte–KC co-culture experiments, a plastic ring (8 mm in height × 15 mm in diameter) was placed in each well of hepatocytes seeded as described above. A glass coverslip attached with approximately 3 × 104 KCs (resulted from initially seeded 3 × 105 CD11b magnet-enriched cells), isolated from either CDMCDLCD or HFDMCDLCD mice, was positioned upside-down on the plastic ring. LD540 was then added, and signal was recorded by live imaging hourly for 4 h in the IncuCyte SX5, with each mouse represented by multiple replicates and nine fields of view (FOVs) per well. For treatment of hepatocytes with APOE or APOA1, APOE (1 µg ml−1 recombinant mouse APOE; ab226314, Abcam) or APOA1 (2 µg ml−1 recombinant mouse APOA1; 50918-M08H, Sino Biological) was pre-incubated with hepatocyte medium at 37 °C for 30 min. The pre-incubated medium was then added to hepatocytes already containing 1 volume of hepatocyte medium. Non-treated hepatocytes served as negative controls. For each mouse, three replicates were prepared, LD540 was then added and the signal was recorded by live imaging hourly for 4 h in the IncuCyte SX5, with four FOVs imaged per well. A cell area filter was applied to ensure single-cell detection. Out-of-focus FOVs were excluded from analysis, as indicated by red labelling in the raw data spreadsheet. For co-culture data, the mean integrated LD540 signal per mouse was calculated following outlier removal by the interquartile range method. Statistical tests included the Shapiro–Wilk test for normality, Levene's test for variance homogeneity, and two-sample t-tests to compare CD versus HFD conditions at each time point. For screening data, mean LD540 measurements were averaged across replicates and normalized to the control measurement mean for each mouse after 3 h and 4 h in the co-culture experiment, and after 2 h in the screening assay. KCs were isolated and pelleted as described above. Non-dissolved cellular debris was removed by centrifugation at 15,000g for 10 min, and clean supernatants were transferred to fresh tubes. Protein concentrations were determined with the Pierce 660 nm Protein-Assay-kit (Thermo). Of protein input, 25 µg was used for digestion in 2 M urea (diluted with 50 mM Tris (pH 8)) with a trypsin–LysC mixture (1:50 (enzyme:protein)) at 800 rpm for 16 h. Protein digestion was stopped with 1% formic acid, peptides were desalted on in-house-produced SDB-RPS Stage Tips and loaded onto EvoTip Pure trap columns following the manufacturers' protocol. A liquid chromatography–tandem mass spectrometry system consisting of an Evosep One liquid chromatograph (Evosep) and a timsTOF Pro 2 mass spectrometer (Bruker) was used for peptide separation and acquisition. Reversed-phase separation was performed on an Aurora Elite analytical column (IonOpticks) using the 40 samples-per-day separation method with a 31-min gradient preprogrammed on the liquid chromatography system. Eluting peptides were on-line transferred into the mass spectrometry via a CaptiveSpray ionisation source operated at a constant voltage of 1.5 kV. We used the data-independent acquisition parallel accumulation-serial fragmentation mode with the TIMS analyser operating with 100-ms accumulation and ramping times and a 100% duty cycle. Peptide ranges were limited to a mass-to-charge range of 350–1,100 m/z and an ion mobility range of 0.63–1.45 Vs cm−2 with 12 pydiAID-optimized data-independent acquisition parallel accumulation-serial fragmentation windows of variable widths79. We increased collisional energies stepwise from 24 eV at 0.70 Vs cm−2 and 49 eV at 1.35 Vs cm−2 for peptide fragmentation. Spectral library prediction, peptide identification and protein quantification were performed in library-free mode using DIA-NN (v1.8.1)81. The SwissProt Mus musculus database was used for predicted spectral library (downloaded from UniProt: 2022-12-28). In silico digestion was set to trypsin as the digestion enzyme with a maximal one miss-cleavage and cysteine carbamidomethylation set as fixed modifications. Precursor peptides were filtered at a false discovery rate (FDR) < 1%. Only proteotypic peptides were considered for quantification. Statistical analysis was performed in the Perseus software suite (v1.6.15)83. Protein group label-free quantification intensities were log2 transformed and filtered for protein groups with data completeness in at least one condition. Missing values were replaced by random value drawing from 1.8 standard deviations downshifted and 0.3 standard deviations broad normal distributions. Significantly regulated proteins between any conditions were identified by ANOVA multiple-sample testing (S0 = 0.5, FDR < 0.1 with 250 randomizations) and all proteins were z-score normalized. For visualization, the dataset was summarized by taking the mean over all samples per group, namely, WT and KO in maternal lean, as well as WT and KO in maternal obese. The summarized data were objected to k-means clustering using k = 6 to obtain groups of proteins with varying expression patterns across samples from which several interesting representatives were chosen to be highlighted. All bar graphs represent mean values. In all of the aforementioned analyses, PCA was performed using the prcomp function in R on centred data. Significance levels were consistently set at P < 0.05. Raw P values were adjusted for multiple testing using the false discovery rate method and considered significant if the adjusted P < 0.1. The analysis was implemented using R (v4.2.0)84 and Bioconductor (v3.15)85. Detailed information about every package used, including all dependencies, along with their respective versions, can be found in the renv.lock file located in the GitHub repository (https://github.com/LeaSeep/MaternalObesity) generated with the renv package86. The main analysis and visualization packages used can be found in Supplementary Table 9 that contains all references. Furthermore, a database file has been provided, which presents all the statistical figures and serves as a comprehensive resource for quick reference and independent verification of our own hypothesis or results. These resources have been made available to facilitate reproducibility and enable researchers to conduct their own investigations with ease. A snapshot of the GitHub repository can be found on Zenodo87 (https://doi.org/10.5281/zenodo.14287647). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The raw transcriptome files and the count data have been deposited in NCBI's Gene Expression Omnibus (GEO)88 and are accessible through GEO Series accession number GSE237408 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE237408). Raw and processed lipidomics data are available at Metabolomics Workbench89 where it has been assigned study IDs ST002781 and ST002782 within the project PR001715. All imaging data used for quantification are available at Bonndata (https://doi.org/10.60507/FK2/PP2MPB). Source data are provided with this paper. Mass, E. et al. Specification of tissue-resident macrophages during organogenesis. Environment drives selection and function of enhancers controlling tissue-specific macrophage identities. Mass, E., Nimmerjahn, F., Kierdorf, K. & Schlitzer, A. Tissue-specific macrophages: how they develop and choreograph tissue biology. Barker, D. J. P. The developmental origins of chronic adult disease. Barker, D. J. P., Osmond, C., Kajantie, E. & Eriksson, J. G. Growth and chronic disease: findings in the Helsinki birth cohort. Mass, E. & Gentek, R. Fetal-derived immune cells at the roots of lifelong pathophysiology. Viola, M. F., Franco Taveras, E. & Mass, E. Developmental programming of tissue-resident macrophages. Ceasrine, A. M. et al. Maternal diet disrupts the placenta–brain axis in a sex-specific manner. Effects of prenatal exposure to the Dutch famine on adult disease in later life: an overview. Collins, A. et al. Maternal inflammation regulates fetal emergency myelopoiesis. Knuesel, I. et al. Maternal immune activation and abnormal brain development across CNS disorders. Aplin, J. D., Myers, J. E., Timms, K. & Westwood, M. Tracking placental development in health and disease. Global incidence and prevalence of nonalcoholic fatty liver disease. Impaired Kupffer cell self-renewal alters the liver response to lipid overload during non-alcoholic steatohepatitis. Remmerie, A. et al. Osteopontin expression identifies a subset of recruited macrophages distinct from Kupffer cells in the fatty liver. Seidman, J. S. et al. Niche-specific reprogramming of epigenetic landscapes drives myeloid cell diversity in nonalcoholic steatohepatitis. Guilliams, M. & Scott, C. L. Liver macrophages in health and disease. Morgantini, C. et al. Liver macrophages regulate systemic metabolism through non-inflammatory factors. Brumbaugh, D. E. & Friedman, J. E. Developmental origins of nonalcoholic fatty liver disease. & Martínez-Chantar, M. L. Understanding gut–liver axis nitrogen metabolism in fatty liver disease. & Thompson, M. D. Maternal exercise protects male offspring from maternal diet programmed non-alcoholic fatty liver disease progression. Huby, T. & Gautier, E. L. Immune cell-mediated features of non-alcoholic steatohepatitis. Oestreich, M. et al. hCoCena: horizontal integration and analysis of transcriptomics datasets. Heieis, G. A. et al. Metabolic heterogeneity of tissue-resident macrophages in homeostasis and during helminth infection. Hume, D. A., Summers, K. M. & Rehli, M. Transcriptional regulation and macrophage differentiation. Expression of RANK is dependent upon differentiation into the macrophage/osteoclast lineage: induction by 1α,25-dihydroxyvitamin D3 and TPA in a human myelomonocytic cell line, HL60. Fate mapping via Ms4a3-expression history traces monocyte-derived cells. & Pessac, B. Microglia derive from progenitors, originating from the yolk sac, and which proliferate in the brain. Fate mapping analysis reveals that adult microglia derive from primitive macrophages. Bone marrow-derived monocytes give rise to self-renewing and fully differentiated Kupffer cells. Kamimae-Lanning, A. N. et al. Maternal high-fat diet and obesity compromise fetal hematopoiesis. Sureshchandra, S. et al. Maternal Western-style diet remodels the transcriptional landscape of fetal hematopoietic stem and progenitor cells in rhesus macaques. Taylor, C. T. & Scholz, C. C. The effect of HIF on metabolism and immunity. Cramer, T. et al. HIF-1α is essential for myeloid cell-mediated inflammation. Clausen, B. E., Burkhardt, C., Reith, W., Renkawitz, R. & Förster, I. Conditional gene targeting in macrophages and granulocytes using LysMcre mice. Expanding the coverage of regulons from high-confidence prior knowledge for accurate estimation of transcription factor activities. Yamagoe, S. et al. Purification and primary amino acid sequence of a novel neutrophil chemotactic factor LECT2. Ruiz-Ortega, M. et al. Angiotensin II participates in mononuclear cell recruitment in experimental immune complex nephritis through nuclear factor-κB activation and monocyte chemoattractant protein-1 synthesis. Metzemaekers, M., Gouwy, M. & Proost, P. Neutrophil chemoattractant receptors in health and disease: double-edged swords. Lynch, R. W. et al. An efficient method to isolate Kupffer cells eliminating endothelial cell contamination and selective bias. Beneficial impact of Gpnmb and its significance as a biomarker in nonalcoholic steatohepatitis. Fuior, E. V. et al. Peroxisome proliferator-activated receptor α in lipoprotein metabolism and atherosclerotic cardiovascular disease. Diehl, K. L. et al. Kupffer cells sense free fatty acids and regulate hepatic lipid metabolism in high-fat diet and inflammation. Block, C. L. et al. Prenatal environmental stressors impair postnatal microglia function and adult behavior in males. Han, V. X., Patel, S., Jones, H. F. & Dale, R. C. Maternal immune activation and neuroinflammation in human neurodevelopmental disorders. Bilbo, S. D. & Schwarz, J. M. Early-life programming of later-life brain and behavior: a critical role for the immune system. & Patterson, P. H. Activation of the maternal immune system induces endocrine changes in the placenta via IL-6. Otero, A. M. & Antonson, A. M. At the crux of maternal immune activation: viruses, microglia, microbes, and IL-17A. Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. Kierdorf, K. et al. Microglia emerge from erythromyeloid precursors via Pu.1- and Irf8-dependent pathways. De Ponti, F. F. et al. Spatially restricted and ontogenically distinct hepatic macrophages are required for tissue repair. Jiang, Z. G., Robson, S. C. & Yao, Z. Lipoprotein metabolism in nonalcoholic fatty liver disease. Nascimento, J. C. R. et al. Impact of apolipoprotein E genetic polymorphisms on liver disease: an essential review. Maeda, K. et al. Wnt5a–Ror2 signaling between osteoblast-lineage cells and osteoclast precursors enhances osteoclastogenesis. Srinivas, S. et al. Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus. Daigle, T. L. et al. A suite of transgenic driver and reporter mouse lines with enhanced brain-cell-type targeting and functionality. Ryan, H. E. et al. Hypoxia-inducible factor-1α is a positive factor in solid tumor growth. Matthews, D. R. et al. Homeostasis model assessment: insulin resistance and β-cell function from fasting plasma glucose and insulin concentrations in man. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Hoffmann, N., Ackerman, M. J., Machot, A. F., Schwudke, D. & Shevchenko, A. LipidXplorer 1.2.8.1. Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Shao, X. et al. CellTalkDB: a manually curated database of ligand–receptor interactions in humans and mice. From Louvain to Leiden: guaranteeing well-connected communities. Liu, Z. P., Wu, C., Miao, H. & Wu, H. RegNetwork: an integrated database of transcriptional and post-transcriptional regulatory networks in human and mouse. B. et al. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Garcia-Alonso, L., Holland, C. H., Ibrahim, M. M., Turei, D. & Saez-Rodriguez, J. Badia-I-Mompel, P. et al. decoupleR: ensemble of computational methods to infer biological activities from omics data. Kim, Y. I. L., O'Rourke, R. & Sagerström, C. G. scMultiome analysis identifies embryonic hindbrain progenitors with mixed rhombomere identities. Leek, J. T., Johnson, W. E., Parker, H. S., Jaffe, A. E. & Storey, J. D. The SVA package for removing batch effects and other unwanted variation in high-throughput experiments. O'Callaghan, A., Kuhn, M., Weston, S., Wing, J., Forester, J. Rapid and in-depth coverage of the (phospho-)proteome with deep libraries and optimal window design for dia-PASEF. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. The Perseus computational platform for comprehensive analysis of (prote)omics data. R: A Language and Environment for Statistical Computing. https://www.R-project.org/ (R Foundation for Statistical Computing, 2022). Huber, W. et al. Orchestrating high-throughput genomic analysis with Bioconductor. Seep, L. Bioinformatic analysis for developmental programming of Kupffer cells by maternal obesity causes fatty liver disease in the offspring (2.0.0). Edgar, R., Domrachev, M. & Lash, A. E. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. We thank C. Cygon, P. Kern and J. Rollheiser for their support in the laboratory; and Y. Kobayashi for providing the Tnfrsf11aCre mice. and TRR333/1 (project-ID 450149205 to D.W.); the European Research Council under the European Union's Horizon 2020 research and innovation program (grant agreement no. ); the PhD fellowship from Boehringer Ingelheim Fonds (to K.M. ); and the EMBO fellowship (ATLF 873-2023 to M.F.V.). The Metabolomics Workbench used to deposit metabolomic and lipidomics data is supported by US National Institutes of Health (NIH) grants U2C-DK119886 and OT2-OD030544. acknowledges support from MicroONE, a COMET Modul (CBmed GmbH) funded by BMK, BMDW, Styria, Vienna, and managed by FFG within the COMET program. The Austrian Federal Ministry of Science, Research and Economy, the National Foundation for Research, Technology and Development, the Christian Doppler Research Association, Siemens Healthineers, EU Horizon 2020 Marie Sklodowska-Curie Doctoral Network grants FANTOM (P101072735) and eRaDicate (101119427), the Christian-Doppler Lab for Applied Metabolomics, the Austrian Science Fund (P26011, P29251, P34781 and IPPTO doc.funds) and the Vienna Science and Technology Fund (LS19-018) provided additional support to L.K. is a member of the European Research Initiative for ALK-Related Malignancies (www.erialcl.net). This work was supported by the Open Access Publication Fund of the University of Bonn. We thank the Flow Cytometry Core Facility of the Mathematical and Natural Sciences Faculty and Flow the Cytometry Core Facility Medical Faculty at the University of Bonn for providing support and instrumentation funded by the DFG (German Research Foundation) — project numbers 341039622, 144734146, 471514137 and 01EO2107 (BMBF). These authors contributed equally: Hao Huang, Nora R. Balzer Developmental Biology of the Immune System, Life and Medical Sciences (LIMES) Institute, University of Bonn, Bonn, Germany Hao Huang, Nora R. Balzer, Iva Splichalova, Nelli Blank-Stein, Maria Francesca Viola, Eliana Franco Taveras, Kerim Acil, Franzisca Petrovic, Nikola Makdissi, Seyhmus Bayar, Katharina Mauel, Carolin Radwaniak, Amir H. Kayvanjoo, Malin Jessen & Elvira Mass Computational Life Sciences, Life and Medical Sciences (LIMES) Institute and Bonn Center for Mathematical Life Sciences, University of Bonn, Bonn, Germany Lea Seep, Stephan Grein & Jan Hasenauer Biochemistry and Cell Biology of Lipids, Life and Medical Sciences (LIMES) Institute, University of Bonn, Bonn, Germany Jelena Zurkovic, Klaus Wunderling, Mohamed H. Yaghmour & Christoph Thiele Clinical Institute of Pathology, Department for Experimental and Laboratory Animal Pathology, Medical University of Vienna, Vienna, Austria Center for Biomarker Research in Medicine (CBmed), Graz, Austria Systems Medicine, Deutsches Zentrum für Neurodegenerative Erkrankungen (DZNE) e.V., Bonn, Germany Thomas Ulas, Joachim L. Schultze & Marc D. Beyer Genomics and Immunoregulation, Life and Medical Sciences (LIMES) Institute, University of Bonn, Bonn, Germany Thomas Ulas, Joachim L. Schultze & Marc D. Beyer Laboratory of Myeloid Cell Biology in Tissue Damage and Inflammation, VIB-UGent Center for Inflammation Research, Ghent, Belgium Laboratory of Myeloid Cell Biology in Tissue Homeostasis and Regeneration, VIB-UGent Center for Inflammation Research, Ghent, Belgium INSERM U1015, Gustave Roussy Cancer Campus, Villejuif, France Immunogenomics and Neurodegeneration, Deutsches Zentrum für Neurodegenerative Erkrankungen (DZNE) e.V., Bonn, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and M.J. helped with experimental procedures and analyses, as well as production of mouse cohorts. supported the lipidomics and ex vivo hepatocyte culture analyses. performed and analysed the proteomics experiments. All authors contributed to and reviewed the manuscript. The authors declare no competing interests. Nature thanks Rodrigo Carlessi, Janina Tirnitz-Parker and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Body weight of mothers on control diet (CD, grey) or high-fat diet (HFD, green). b, Homeostatic Model Assessment for Insulin Resistance (HOMA-IR) of mothers on CD or HFD performed after 8 weeks on respective diet. c, Heatmap showing log-transformed values of detected cytokines and chemokines in the sera of mothers on CD and HFD. Two-tailed unpaired Student's t-test on the individual coefficients after linear fitting model and multiple testing correction using false discovery rate <0.05. d, Plots of results from c that show significance when using two-tailed unpaired Student's t-test. e, Body, white adipose tissue (WAT) and liver weights of offspring at 11 weeks of age. n = 8, 8, 7, 6, 7, 7 mice for CDMCDLCD, CDMCDLHFD, HFDMCDLCD, HFDMHFDLCD, HFDMCDLHFD, HFDMHFDLHFD groups, respectively, on 11 experimental days. Two-tailed unpaired Student's t-test on the individual coefficients after linear fitting model and multiple testing correction using false discovery rate <0.05 (* = p < 0.05, ** = p < 0.01, *** = p < 0.001). h, Metabolomics of serum from all experimental groups visualized by PCA. i, Enriched metabolic pathways in HFDMCDLCD compared to CDMCDLCD based on serum metabolomics data. One-way ANOVA – Tukey's multiple comparison test. a, Flow cytometry gating strategy for KCs, Liver Capsular Macrophages (LCM), classical Dendritic Cells 1 and 2 (cDC1, cDC2), monocytes (Mono) and neutrophils (Neu). b, Cell number of different myeloid populations per gram of liver tissue quantified by flow cytometry. n = 4, 5, 4, 6, 4, 4 mice for CDMCDLCD, CDMCDLHFD, HFDMCDLCD, HFDMHFDLCD, HFDMCDLHFD, HFDMHFDLHFD groups, respectively, on 8 experimental days. One-way ANOVA – Tukey's multiple comparison test, comparing and showing significances of only CDMCDLCD, CDMCDLHFD and HFDMCDLCD groups. d, Schematic illustration of metabolite transporters and enzymes assessed for the metabolic state of KCs. The schematic was created in BioRender. e, Mean Fluorescence Intensity (MFI) of GLUT1, PKM, SDHA, CytC, ATP5A, G6PD, CD36, CPT1a and ACC1 in KCs isolated from 11–13 weeks CDMCDLCD and HFDMCDLCD mice. a, Heatmap showing log-transformed values of detected cytokines and chemokines in the offspring sera. n = 5, 7, 6, 6, 5, 6 mice for CDMCDLCD, CDMCDLHFD, HFDMCDLCD, HFDMHFDLCD, HFDMCDLHFD, HFDMHFDLHFD groups, respectively, on 11 experimental days. No significance across conditions using Kruskal-Wallis test. b, Flow cytometry gating strategy for analysis of hematopoietic stem and progenitor cells in the bone marrow. c, Cell number of different hematopoietic stem and progenitor cells quantified by flow cytometry. n = 6, 7, 3, 6, 4, 4 mice for CDMCDLCD, CDMCDLHFD, HFDMCDLCD, HFDMHFDLCD, HFDMCDLHFD, HFDMHFDLHFD groups, respectively, on 11 experimental days. No significance across conditions using One-way ANOVA – Tukey's multiple comparison test. b, Flow cytometry gating strategy for blood monocytes and microglia. c, Quantification of labelling efficiency of blood monocytes and microglia in 11–12 weeks old maternal lean (CDMCDLCD, n = 4 on 2 experimental days) and maternal obese (HFDMCDLCD, n = 3 on 2 experimental days) mice. Bar graphs show the mean ± SD. d, Flow cytometry gating strategy for KCs. Bar graphs show the mean ± SD. Two-tailed unpaired Student's t-test when sample size is no less than 3. c, Flow cytometry analysis of isolated bone marrow monocytes and hematopoietic stem and progenitor cells (HSPCs) from Rosa26mTmG mice used for transplanting Clec4f+/+ and Clec4fDTR/+ mice 15 h after DT injection. d, Immunofluorescence staining of HFDMCDLCD Clec4fDTR/+ liver at the age of 13 weeks showing the partial replacement of endogenous KCs by tdT+ bone marrow monocytes and HSPCs from Rosa26mTmG mice. Representative image of n = 9 mice on 4 experimental days. The illustrations were created in BioRender. a, Body, WAT and liver weight of 11-week-old offspring. One-way ANOVA – Tukey's multiple comparison test. b, Heatmap of liver lipidomics in Fig. Two-tailed unpaired Student's t-test on the individual coefficients after linear fitting model and multiple testing correction using false discovery rate <0.05 (* = p < 0.05, ** = p < 0.01, *** = p < 0.001). c, Pathological scoring of liver steatosis performed on HE stainings. One-way ANOVA – Tukey's multiple comparison test. d, FACS-gating strategy to sort KCs and hepatocytes for RNA-seq. e, RNA-seq of FACS-sorted hepatocytes from all diet groups of Hif1α-WT and Hif1α-KO mice visualized by PCA. a, Heatmap showing scaled expression of DEGs of KCs isolated from pups born to CD-fed (CDM) or HFD-fed (HFDM) mothers. b, Transcription factor (TF) activity analysis with genes from a. as input. c. Volcano plot of DEGs (blue: down, red: up) between P0 KCs born to HFD- and CD-fed mothers. d. HALLMARK pathway overrepresentation-analysis of all upregulated DEGs shown in a. e, Variant supporting counts (vsc) of all Hif genes. Box plots show median with interquartile range and min/max. f, Representative images of immunofluorescence staining of HIF1α in F4/80+ KCs in P0 livers born to CD- and HFD-fed mothers. n = 4, 3 P0 pups born to CD-, HFD-fed mothers, respectively, on 3 experimental days. g, Quantification of HIF1α intensity in nuclei in % to cytoplasm in F4/80+ KCs comparing P0 pups born to CD- and HFD-fed mothers in f. n = 9, 12 cells from P0 pups born to CD-, HFD-fed mothers, respectively. Bar graphs show the mean ± SD. a, Expression of selected chemo-attractants expressed by hepatocytes across diet groups of HIF1α mouse model. Box plots show medians with interquartile range and min/max. b, Cell number of different myeloid populations per gram of liver tissue quantified by flow cytometry. One-way ANOVA – Tukey's multiple comparison test. c, Dot plot showing ORA analysis of DEGs comparing HFDMCD WT versus HFDMCD KO conditions. d, Heatmap showing the relative mean expression and hierarchical clustering of differentially expressed proteins from isolated KCs comparing HFDMCD WT vs. CDMCD WT mice, HFDMCD WT vs. HFDMCD KO mice, and the overlapping proteins of the two comparisons. 3h, where selected proteins from the different clusters are displayed. e, Dot plot showing enriched pathways of the differentially expressed proteins from Fig. a, snRNA/ATAC-seq of CDMCDLCD and HFDMCDLCD livers visualized by Uniform Manifold Approximation (UMAP) showing all liver cell types. b, Dot plot showing the genes determining the cell type annotation of identified clusters shown in a. c, Representative immunofluorescence (IF) staining showing MHC-II expression in the liver of CDMCDLCD and HFDMCDLCD mice. All KCs and macrophages are labelled with Iba1. d, Volcano plot showing assigned genes to the differentially detected snATAC-seq peaks between KCs from CDMCDLCD and HFDMCDLCD livers. e, GO term analysis of downregulated genes associated with snATAC-seq peaks. f, Violin plots comparing Gpnmb (upper) and Pck1 (lower) expression within all KC states comparing CDMCDLCD and HFDMCDLCD groups. g, CellChat analysis using differentially expressed genes between CDMCDLCD and HFDMCDLCD KCs as sender molecules and all cell types as receivers. Coloured genes indicate significant (p < 0.05) differences of information flow between the conditions tested with Two-tailed Wilcoxon Rank Sum test. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Huang, H., Balzer, N.R., Seep, L. et al. Kupffer cell programming by maternal obesity triggers fatty liver disease. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09171-z'>Bimodal centromeres in pentaploid dogroses shed light on their unique meiosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 15:50:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Sexual reproduction relies on meiotic chromosome pairing to form bivalents, a process that is complicated in polyploids owing to the presence of multiple subgenomes1. Uneven ploidy mostly results in sterility due to unbalanced chromosome pairing and segregation during meiosis. However, pentaploid dogroses (Rosa sect. Caninae; 2n = 5x = 35) achieve stable sexual reproduction through a unique mechanism: 14 chromosomes form bivalents and are transmitted biparentally, while the remaining 21 chromosomes are maternally inherited as univalents2,3. Despite being studied for over a century, the role of centromeres in this process has remained unclear. Here we analyse haplotype-resolved chromosome-level genome assemblies for three pentaploid dogroses. Subgenome phasing revealed a bivalent-forming subgenome with two highly homozygous chromosome sets and three divergent subgenomes lacking homologous partners, therefore explaining their meiotic behaviour. Comparative analyses of chromosome synteny, phylogenetic relationships and centromere composition indicate that the subgenomes originated from two divergent clades of the genus Rosa. Pollen genome analysis shows that subgenomes from different evolutionary origins form bivalents, supporting multiple origins of dogroses and highlighting variation in subgenome contributions. We reveal that bivalent-forming centromeres are enriched with ATHILA retrotransposons, contrasting with larger tandem-repeat-based centromeres mainly found in univalents. This centromere structural bimodality possibly contributes to univalent drive during female meiosis. Our findings provide insights into the unique reproductive strategies of dogroses, advancing our understanding of genome evolution, centromere diversity and meiotic mechanisms in organisms with asymmetrical inheritance systems. Whole-genome duplication or polyploidy is a frequent phenomenon across the phylogeny of land plants4. Meiosis is essential for sexual reproduction, ensuring the reduction in genomic content in gametes through chromosome pairing and exchanges between non-sister chromatids, that is, crossovers5,6. Polyploidy often results from meiotic failure, that is, the generation of unreduced gametes, which poses challenges to meiotic chromosome pairing and the maintenance of sexual reproduction1,7. Thus, polyploids often skip sexual reproduction by promoting vegetative propagation8 or apomixis9. However, in many allopolyploids, in which distinct subgenomes come into contact through hybridization, recombination partners from homologous chromosomes (same parental subgenome) are preferred, while recombination between homoeologous chromosomes (different parental subgenomes) is suppressed10,11. The genus Rosa, which comprises approximately 150 species, is a typical example of evolution through frequent polyploidy and hybridization events12, which is also reflected by the large variety of cultivated roses with a long breeding history that includes both processes. The genus comprises two major clades, the Rosa and allies clade and the Synstylae and allies clade with subg. Available genomes from diploid roses of sect. Rosa17 revealed high levels of synteny, enabling comparative studies in this taxonomically difficult genus. Studies on tetraploid cut roses (Rosa hybrida) undergoing regular meiosis have shown that most genomic markers were recombined freely from all four chromosome sets, but preferential recombination between chromosomes and even chromosome arms vary18,19. Within the Synstylae clade, allopolyploid dogroses (Rosa sect. exhibit a unique reproductive strategy known as Canina meiosis, in which the selective chromosome pairing results in a mixed mode of inheritance—combining biparental transmission of bivalents and uniparental transmission of univalents within the same nucleus20,21. First observed in the early twentieth century, this mechanism is most common in pentaploid dogroses (2n = 5x = 35), in which the male and female parents contribute 7 and 28 chromosomes, respectively, to the zygote2,3,22,23. During meiosis, 14 chromosomes form 7 bivalents, while the other 21 chromosomes remain as univalents. Bivalent-forming chromosomes from two highly homozygous sets are transmitted to both sperm and egg cells, whereas univalents are inherited exclusively through the egg cell and excluded from pollen grains, restoring pentaploidy in the offspring through the fusion of haploid male and tetraploid female gametes24,25,26,27 (Fig. Despite extensive study, the precise cellular mechanisms underlying this asymmetric inheritance remain poorly understood. During Canina meiosis2,3,22,23, two chromosome sets form bivalents (dark grey and red asterisks) during meiosis and are transmitted through both pollen and egg cells24,25,26. The remaining three sets form univalents (lighter grey) and are transmitted through the egg cell only24,25,26. Diakinesis of male meiosis I (n = 15) of R. canina is shown on the left. b, GENESPACE synteny and phylogenetic relationships of the five chromosome sets of R. canina and their close diploid relatives R. chinensis (sect. Rosa) with a dated phylogenetic tree constructed using 16,372 orthologous genes on the left. Each colour indicates synteny to each R. chinensis chromosome, which was used as reference to name the R. canina chromosomes. c,d, Unrooted maximum-likelihood phylogenies of the homoeologous R. canina chromosomes and chromosomes from the respective linkage groups of the diploid species R. chinensis (sect. Rosa) based on alignments of whole-chromosome sequences. The phylogeny of synteny group 1 chromosomes is shown in c. Synteny groups 2–7 are shown in d. Filled chromosomes refer to subgenomes of R. canina belonging to the Synstylae clade (violet/light blue) and the Rosa clade (dark/light orange). Chromosomes from the diploid roses are indicated by hatching. e, Synteny and rearrangement analyses (SyRI) of the R. canina genome assembly. Pairwise comparisons of the synteny of all R. canina subgenomes (S1_h1/S1_h2, S2, R3 and R4) are juxtaposed against the corresponding chromosomes (chr.) Only synteny blocks and rearrangement blocks greater than 50 kb in length are shown. Bivalent-forming and univalent chromosomes are thought to have originated from multiple ancient hybridization events28,29,30. Phylogenetic studies based on maternally inherited plastids suggest that dogroses are polyphyletic within the Synstylae clade, with subsects. Rubigineae and Caninae are separated by species exhibiting regular meiosis13,31,32,33. Cytogenetic evidence shows that bivalent-forming chromosomes in subsect. Caninae become univalents in subsect. The rose-specific (peri)centromeric satellite repeat (CANR4) is notably enriched in dogrose univalents, possibly linking centromere expansion to their drive during female meiosis36,37. Although other uneven polyploid systems with hemisexual reproduction exist38,39,40, the meiosis observed in dogroses is unique among eukaryotes. However, the lack of genomic studies has hindered understanding of how centromere properties contribute to this enigmatic reproductive behaviour. Here we present a high-quality haplotype-phased chromosome-scale assembly of the pentaploid genome of Rosa canina (subsect. Caninae) and compare it with another R. canina individual and R. agrestis (subsect. Rubigineae), both from the Darwin Tree of Life (DToL; https://www.darwintreeoflife.org/). Subgenome-aware analyses revealed that dogroses are composed of four subgenomes with one subgenome being present in two highly homozygous haplotypes and the other three in only one. Targeted sequencing of rose-specific single-copy orthologues (SCOs) from pollen DNA, together with the analysis of synthetic F1 hybrids, confirmed that the two-copy subgenome is biparentally inherited and therefore forming bivalents, while 21 non-recombining univalents from three distinct subgenomes are exclusively inherited through the female germline. Our SCO-based phylogenetic analysis supports the multiple-origin nature of dogroses, as subgenomes from different evolutionary origins were found in pollen, that is, forming bivalents. We also identified a bimodal centromere architecture with small (retrotransposon ATHILA-based) and large (tandem repeat CANR4-based) centromeres. Notably, CANR4-based centromeres were prevalent in univalents, possibly contributing to their drive in asymmetric female meiosis. Our results therefore provide a valuable basis for studying the trade-offs between sexual and asexual reproduction within a single genome. Although R. canina has been recognized as a pentaploid species (2n = 5x = 35) for decades, its genome has remained unresolved owing to its complex polyploid structure and hybrid origin. To address this, we assembled a de novo haplotype-resolved, chromosome-level genome using PacBio HiFi sequencing (23× coverage) and chromatin conformation capture (Hi-C) data (Supplementary Table 1). The total size of the assembled 35 pseudochromosomes is about 2.4 Gb, achieving 99.2% completeness in terms of gene content (Extended Data Fig. This high-quality reference genome provides a critical resource for understanding R. canina's genetic features, asymmetric meiosis and hybridization. All-to-all chromosome alignments revealed seven syntenic groups in R. canina, each consisting of five chromosomes (Supplementary Fig. In each group, two chromosome sets consistently exhibited 99–100% similarity, indicating two haplotypes of the same subgenome, while the remaining three chromosome sets showed lower similarities (95–98%; Supplementary Fig. 1 and Supplementary Data 1), which may derive from three different subgenomes. Phylogenetic analyses based on gene and chromosome data, using the diploid rose genomes of Rosa chinensis (sect. Synstylae)16 and Rosa rugosa (sect. Rosa; https://www.darwintreeoflife.org/) as references, revealed that two subgenomes are closely related to R. chinensis and were therefore designated ‘S', while the other two are more similar to R. rugosa (Fig. 1b–e) and were therefore designated ‘R'. The two highly similar Synstylae-like haplotypes were named S1_h1 and S1_h2, while the more divergent Synstylae-like chromosome set was named S2. The Rosa-like subgenomes were named R3 and R4, respectively. Moreover, using subgenome-specific k-mers41, we observed that chromosomes assigned to the same subgenome clustered together both in the k-mer heat map and the principal component analysis plot, confirming the correct assignment of the four primary subgenomes (S1, S2, R3 and R4; Extended Data Fig. These findings resolve the long-standing question regarding the identity of the five homoeologous chromosomes within each syntenic group. To further validate the evolutionary relationships of the subgenomes in pentaploid R. canina, we conducted orthologous cluster analysis and genome-wide comparisons of synonymous substitution rates (Ks) between R. canina, R. chinensis and R. rugosa. These results were consistent with previous phasing assignments and confirmed the allopolyploid origin of R. canina (Supplementary Figs. Structure-based pairwise chromosomal analysis across the subgenomes of R. canina and genomes of R. chinensis and R. rugosa revealed a strong conservation of synteny between the two haplotypes of the homozygous S1 subgenome (S1_h1/h2). By contrast, the S1 subgenome showed much lower synteny with the other three subgenomes (S2, R3, R4), which were characterized by large inversions, duplicated regions and translocations (Fig. Notably, the R3 and R4 subgenomes of R. canina exhibit greater synteny to the R. rugosa genome than to the other subgenomes of R. canina (Fig. 1e), supporting their origination from sect. Despite the distinct divergence and origins of four subgenomes, a comparison of R. canina chromosomes of all subgenomes against R. rugosa syntenic chromosomes revealed no evidence of differential fractionation (loss of one or the other copy of a duplicated gene; Supplementary Fig. To detect differential evolutionary rates between subgenomes, we analysed the ratio of nonsynonymous versus synonymous substitution rates and revealed, besides a few outliers, strong purifying selection across orthologous genes in all subgenomes (Supplementary Fig. Together, these results suggest an absence of large-scale subgenome dominance. We next took advantage of the recent HiFi and Hi-C sequencing datasets from the DToL (https://www.darwintreeoflife.org/) for another R. canina accession (European Nucleotide Archive (ENA): PRJEB79802) and from Rosa agrestis (subsect. Rubigineae; ENA: PRJEB79880) to generate de novo pseudochromosomes (Supplementary Table 2). Comparative analysis revealed a high degree of synteny between our R. canina S27 genome and the DToL R. canina, both sharing the same subgenome composition (Extended Data Fig. The R. agrestis genome, also pentaploid with 35 pseudochromosomes, displayed a different subgenome composition, with two highly similar haplotypes for the R4 subgenome (R4_h1/h2) and only one copy of the S1 subgenome (Extended Data Fig. Comparative subgenome phasing revealed a gradient of differentiation between the subgenomes of R. canina and R. agrestis. The S1 subgenomes of both species were the least differentiated and clustered together (Supplementary Fig. This was followed by the R4 subgenomes, which exhibited a slightly higher degree of differentiation compared with S1 (Supplementary Fig. The most pronounced differentiation was seen in the S2 and R3 subgenomes, which were distinctly separated in both species (Supplementary Fig. This pattern suggests that the subgenomes S2 and R3 are accumulating more divergence over time. To trace the hybridization history of dogroses, we identified subgenome-specific long terminal repeat retrotransposons (LTR-RTs) and estimated their insertion times in R. canina and R. agrestis to determine the timing of subgenome differentiation before hybridization. Notably, we observed a distinction in the median insertion times of LTR-RTs: the S1, S2 and R3 subgenomes in R. canina were estimated to have diverged around 0.7 million years ago (Ma), while the R4 subgenome was older, at approximately 1.2 Ma (median values, 95% confidence intervals; Extended Data Fig. In R. agrestis, the median insertion times for S1 and S2 (~0.7 Ma) and for R4 (~1.2 Ma) were the same as for R. canina, whereas a slightly older median insertion time was detected for the R3 subgenome (around 0.9 Ma; Extended Data Fig. These results suggest that the combination of S1, S2 and R3 subgenomes arose at different timepoints in R. canina and R. agrestis. This is further supported by the Ks-based divergence time estimation obtained from SCOs (Supplementary Fig. 6) and comparable findings from the high differentiation between their R3 subgenomes (Supplementary Fig. Together, these findings suggest that modern dogroses originated through independent, stepwise hybridization events. Only bivalent-forming chromosomes are able to segregate properly and produce viable haploid (1x) pollen in dogroses. We therefore used flow sorting to isolate pollen nuclei as a proxy to confirm which subgenomes are exclusively pollen-inherited and form bivalents in dogroses. We successfully collected around 200,000 generative nuclei from pollen samples (Fig. Caninae; and 3 of R. rubiginosa, subsect. Rubigineae) and analysed them using single-copy orthologous nuclear locus target enrichment (Methods). This enabled us to create sample-specific reference sequences for each SCO locus. We mapped a total of 5,794 SCO sequences to the R. canina S27 genome and identified 7 major chromosome pairs, with most hits in R. canina pollen located on the S1_h1/h2 chromosomes (Fig. By contrast, R. rubiginosa pollen showed hits primarily on the R3 and R4 chromosomes (Fig. By leveraging dogrose pollen SCO mappings on the R. canina genome, we unambiguously identified seven pairs of bivalent-forming chromosomes across several dogrose species. Moreover, mapping SCO loci to the R. agrestis genome revealed that the R4_h1/h2 subgenome forms bivalents in this species (Extended Data Fig. These results confirm that bivalent-forming chromosomes in subsect. Caninae form univalents in subsect. a, Flow cytometry analysis of R. canina isolated pollen nuclei. Vegetative (veg) and generative (gen) nuclei differ in their DNA content in the binucleate pollen grains. Inset: an intact pollen grain after DAPI staining of both nucleus types. b, Genome-wide pollen SCO mapping of eight dogrose species (subsect. Caninae: three samples of R. canina and two samples of R. corymbifera; subsect. Rubigineae: three samples of R. rubiginosa) to the R. canina S27 genome. The bubble map represents chromosomal hits, which were selectively filtered to display loci with a single alternative hit. The size of the symbols corresponds to the mean counts of pollen SCOs mapped to each chromosomal pair, identifying seven pollen-inherited chromosomes from the S1 subgenome within the R. canina (subsect. Rubigineae), pollen SCO mapped preferentially to the R3 and R4 subgenomes. c, Maximum-likelihood phylogeny of the genus Rosa based on SCO loci including those retrieved from chromosome-scale assemblies and all pollen samples. Nodes with less than 100% bootstrap are indicated by dashed lines. d, Flow cytometry analysis of nuclei isolated from nutlets of R. canina, showing an endosperm/embryo ratio of 1.8, corresponding to the expected 9C/9x endosperm and 5C/5x embryo ratio (in which C denotes unreplicated haploid DNA content; x is the basic chromosome number), confirming sexual reproduction and endosperm fertilization. e, Assessment of the parental genomes contribution of a synthetic hybrid between R. canina (female donor) and R. rubiginosa (male donor). The x axis shows the coverage histogram of the short reads from this species mapped to each R. canina subgenome. The hybrid revealed doubled coverage for R4, indicating the presence of two sets of R4 copies, biparentally inherited, while only one set of maternally inherited S1 was detected, confirming sexual reproduction and the subgenome's inheritance through male and female meiosis. Next, we aligned SCO loci obtained from the R. canina S27, R. canina DToL and R. agrestis DToL subgenomes, along with pollen DNA from section Caninae and different diploid rose samples and outgroups. This resulted in 58 sequences, totalling 642,158 positions derived from 1,904 concatenated SCO loci. Subgenome-wise as well as chromosome-wise phylogenetic analysis delineated two large clades within the genus Rosa: the Synstylae clade and the Rosa clade, as well as the earlier splits of subg. Hesperhodos (Rosa stellata) and sect. All of the subgenomes of R. agrestis grouped with the corresponding R. canina subgenome samples, supporting a common origin of individual subgenomes despite the high differentiation observed in S2 and R3 subgenomes (Fig. However, the pollen SCO data from all R. rubiginosa samples grouped as sister to both R3 and R4 subgenomes from R. canina and R. agrestis, respectively, implying multiple origins of the bivalent-forming chromosomes in subsect. Notably, the pollen SCO data from R. canina ‘CAN2' was sister to the univalent-forming subgenome S2, suggesting some intraspecific variation in the bivalent-forming subgenomes within R. canina (Fig. 2c and Supplementary Data 3), as indicated by variation in microsatellite alleles from bivalent-forming chromosomes in R. canina26. Supported by the respective clustering of bivalent-bearing pollen data from subsect. Rubigineae, our data demonstrate an independent origin for the bivalent-forming subgenomes of dogroses and, consequently, the independent origins of asymmetric meiosis34,35. Furthermore, the finding that representatives of subsect. Rubigineae (including the newly generated plastome assemblies for R. canina and R. agrestis) contain phylogenetically distant plastids from the Synstylae clade13,31,32,33 (Extended Data Fig. 5b) supports the hypothesis that two progenitors from the Synstylae clade formed reciprocal hybrids, which subsequently incorporated R genomes through pollen donors. While haploid (1x) pollen nuclei are clearly a product of Canina meiosis (Fig. 2a,b), tetraploid (4x) sexually derived egg cells were inferred by the respected embryo/endosperm ratio in seeds (Fig. In sexually reproducing diploids with double fertilization of the egg cell and the polar nuclei, the endosperm/embryo ratio is 1.5 (3x endosperm/2x embryo; Extended Data Fig. However, in sexually reproducing 5x dogroses, the ratio was found to be 1.8, indicating a 9x endosperm and a 5x embryo (Fig. To further check the reproduction mode of dogroses, we have investigated the genome composition of two synthetic hybrids obtained from controlled crossing experiments43. In the first cross, the female gamete came from R. canina (subsect. Caninae) and the male donor was R. rubiginosa (subsect. As anticipated from the result of the Canina meiosis, the subgenome contribution in the hybrid was consistent with the expected 4x egg cell containing one copy of each S1/S2/R3/R4 subgenome from R. canina and a 1x pollen nucleus with the R4 subgenome from R. rubiginosa (Fig. In the second case, the female gamete came from R. rubiginosa and the male donor was R. corymbifera—a very close relative of R. canina44. Again, the hybrid showed the expected subgenome composition, consisting of a male haploid S1 subgenome and a female tetraploid S1/S2/R3/R4 subgenome (Extended Data Fig. These results are in agreement with S1 and R4 being bivalent-forming subgenomes and confirm the 1x male versus 4x female gamete composition. Our findings further suggest that different subgenomes are potentially interexchangeable in hybridization events; however, hybrids in extant populations originated mostly from unreduced eggs suggesting some subsection-specific differentiation subgenomes, which might impact bivalent formation45. To gain further insights into the subgenome differentiation of R. canina, we aimed to characterize its global repeat composition, both genome-wide and specifically at centromeres. The R. canina genome exhibited a very high content of LTR Ty1/Copia elements, which made up 40% of the total repeat content, compared with 23% of Ty3/Gypsy elements. Among the Ty1/Copia elements, the BIANCA family accounted for more than 45% of all annotated full-length LTR-RTs while, among the Ty3/Gypsy elements, RETAND and ATHILA were the largest classes found, comprising 10% and 8% of all annotated full-length LTR-RTs, respectively. Tandem repeats, that is, satellite DNA, were mainly composed of the (peri)centromeric CANR4 repeats15,20 and rDNA sequences (Fig. The repeat profile across the 35 chromosomes revealed prominent 2–3 Mb peaks of highly dense repeats probably corresponding to the centromeres (Fig. a, The global distribution of the main types of repeats identified across all chromosomes. Subgenomes (top right) and sequence tracks (bottom left) are assigned by coloured names. b,c, Centromere analysis of tandem-repeat structures in the chromosomes of synteny group 6 (additional detailed plots for the other chromosomes are provided in Supplementary Fig. The sequence structure of bivalent (b) and univalent (c) centromeres was visualized using ModDotPlot (top triangles). The colour-intensity histograms (top right) show the number of alignments versus pairwise sequence similarity. The sequence tracks plotted below highlight the main classes of repeats identified and the respective association with CENH3 and DNA methylation. The window size in b and c is 10 kb. For the y axes in a–c, all features were scaled [0, 1]; the original values are provided in Supplementary Fig. 10. d, CENH3 ChIP–seq enrichment (log2[CENH3/H3]) compared with the densities of the centromeric elements ATHILA and CANR4 in 50 kb windows, and DNA methylation in the CpG, CHG and CHH sequence contexts, for each subgenome. Only the centromere proximity regions are shown—10% of the centromere-to-telomere distance. Centromeres (CEN) were defined by the maximum CENH3 enrichment. All signal values (y axis) were scaled from 0 to 1 based on the global minimum to global maximum, except for DNA methylation, for which the original percentage values were retained. e, Linear regression of CANR4 size and CENH3 abundance on the centromere across all chromosomes. Each dot coloured by its subgenome presents a centromere. The abundance of CENH3 was calculated by the sum of CENH3 ChIP–seq (log2[CENH3/H3]) signals on centromeres normalized to coverage. The Spearman's rank correlation is 0.93. Note the high CENH3 enrichment for the cenLTR1-based centromere in Rca1_R4, which lacks CANR4 repeats. To validate the DNA sequences associated with functional centromeres, we developed an R. canina centromeric histone H3 (CENH3)-specific antibody and performed chromatin immunoprecipitation followed by sequencing (ChIP–seq). Notably, our results revealed two main types of centromere composition—Ty3/Gypsy ATHILA and CANR4 satellite-based centromeres (Fig. Analysis of the centromere-wide repeat structures revealed that ATHILA-based centromeres were most frequent in the chromosomes of the S1 and R4 subgenomes, while larger CANR4-based centromeres were found across all chromosomes in the S2 and R3 subgenomes (Fig. CANR4 centromeric arrays were also found in 3 out of 7 syntenic groups (2, 4 and 5) of the S1_h1/h2 bivalent-forming chromosomes and in two R4 chromosomes, in which these arrays were frequently interrupted by ATHILA elements (Fig. A similar centromeric sequence composition was observed in both R. canina and R. agrestis from DToL, despite considerable variation in sequence length (Supplementary Figs. Further structural sequence analysis of the diploid relatives R. chinensis and R. rugosa revealed that CANR4 repeats are present in only four and three centromeric regions, respectively, while centrophilic ATHILA elements were found in all centromeres (Supplementary Data 9–11). Together, our results confirm the expansion and predominance of CANR4-based centromeres in exclusively maternally inherited univalent chromosomes in dogroses20. Moreover, we identified two centromeres of R. canina S27 from the R4 subgenome (Rca1_R4 and Rca4_R4) that lack CANR4 repeats but exhibited high affinity for CENH3 in regions other than ATHILA elements. These two centromeres were characterized by the presence of several tandem-repeat sequences with very long monomers ranging from 1,425 to 2,596 bp. Detailed characterization of these tandem-repeat arrays has revealed that all of these sequences identified are probably derived from different centrophilic ATHILA elements, as they all share over 75% similarity with their LTR sequences and are therefore referred to cenLTR1–4 (Extended Data Fig. Notably, the cenLTR arrays showed significantly higher CENH3 enrichment compared with neighbouring ATHILA elements, with the most pronounced enrichment observed in Rca1_R4, which contained a large array of cenLTR1 (235 kb; Extended Data Fig. Although cenLTR arrays found in Rca4_R4 were shorter and characterized by less CENH3 enrichment compared with cenLTR1 in Rca1_R4, we found two different arrays of cenLTR2 and cenLTR3 with higher enrichment than neighbouring ATHILAs. Furthermore, the cenLTR1 monomer sequence showed over 85% similarity to the LTR sequences of ATHILAs in Rca4_R4; however, it was not found in tandem arrays in this chromosome (Supplementary Fig. Notably, these cenLTR arrays were not detected in either of the DToL genome assemblies of R. canina or R. agrestis (Supplementary Figs. 11 and 12), suggesting that the formation of these centromeric tandem repeat arrays is a very recent evolutionary event. To investigate the epigenetic organization within centromeres, we analysed DNA methylation patterns across chromosome arms, scaling from the telomeres to the centromere midpoints. Methylation levels were generally elevated at centromeres across CpG, CHG and CHH contexts. However, DNA methylation was slightly reduced, particularly in CANR4-based centromeres (Fig. These findings suggest distinct methylation patterns in CANR4 centromeres compared with ATHILA-based centromeres. Indeed, ATHILA accumulation was less pronounced in the centromeres of univalents of S2 and R3 subgenomes, despite a few insertions being found within CANR4 centromeric arrays (Fig. Notably, ATHILA insertions in CANR4 arrays disrupted CENH3 binding, while ATHILA-based centromeres were smaller and showed a lower level of CENH3 association compared with CANR4-based centromeres (Fig. Frequent insertion of ATHILA elements into CANR4 arrays was also observed in R. agrestis and the diploid roses R. chinensis and R. rugosa (Supplementary Data 7–11). A similar disruption of centromere activity by centrophilic ATHILA has been recently found within Arabidopsis centromeres46,47. We further observed a positive correlation between the amount of CANR4 repeats and CENH3 abundance along R. canina S27 centromeres (Fig. In fact, the total centromere length, as the measurement of the CENH3-binding regions per subgenome, confirmed that the S2 and R3 centromeres were larger compared with those of S1 and R4 (Fig. Most CANR4-based centromeres showed increased levels of CENH3 accumulation compared with the CANR4-less ones. Notably, the centromere on Rca1_R4, which is mainly based on a cenLTR1 array (Extended Data Fig. 6), showed one of the highest enrichments for CENH3 among CANR4-less centromeres (Fig. Thus, tandem repeats bearing different monomer composition seem to accumulate high CENH3 levels in the R. canina centromeres. CENH3 immunostaining of R. canina chromosomes revealed differences in the size of individual centromeres (Fig. 4a and Supplementary Video 1; n = 12). The size difference was further confirmed by immunostaining analysis of the kinetochore component KNL148 (Extended Data Fig. Furthermore, in situ hybridization with probes for CANR4 and ATHILA on mitotic (n = 15) and meiotic (n = 18) chromosomes of R. canina confirmed the predominance of large CANR4 signals at the centromeres of univalents, while all bivalent and almost all univalents showed ATHILA centromeric signals (Extended Data Fig. In the diploids R. chinensis and R. rugosa, CANR4 centromeric signals were observed in only three pairs of chromosomes while, again, ATHILA was found in all centromeric regions (Extended Data Fig. Thus, the observed difference in the size of CENH3 centromeric signals is probably associated with the accumulation of CANR4 satellite repeats, supporting our ChIP–seq analysis (Fig. These results confirm the bimodal architecture of R. canina centromeres, which are preferentially ATHILA-based in bivalents and CANR4-based in univalents, with the caveat that our ChIP–seq experiment was performed using leaf tissue. a, Centromeres on all chromosomes were detected by CENH3 protein in mitotic metaphase. Note the size difference in CENH3 signals among different chromosomes. b,c, Centromeric organization during early prophase I. d, The orientation of centromeres during the diakinesis stage. e, Metaphase I with univalents (U) surrounding the bivalents (B); the dashed line highlights the typical clustering of bivalents in the middle. f, Metaphase I, with all chromosomes organized in the equatorial plate initiating the segregation of homologous pairs (in the case of bivalents) and separation of sister chromatids (in the case of univalents). g,h, Early (g) and late (h) anaphase I shows early separation of bivalents, while sister chromatids of univalents usually lagging behind. i,j, During metaphase II (i) and anaphase II (j), two main spindles are formed, while lagged chromosomes are still attached to additional abnormal spindles (arrowheads). At anaphase II (j), sister centromeres of the bivalent-forming chromosomes finally segregate to form haploid gametes, while single chromatids from univalents lag behind and are eliminated. Experiments were independently repeated at least ten times with similar results to track all meiotic stages represented here. Next, we investigated the behaviour of centromeres and spindle dynamics during male meiosis of R. canina. Our immunostaining analysis using antibodies against CENH3 and alpha-tubulin clarified the asymmetric distribution of chromosomes during meiosis. In early stages of male meiosis, both small and large centromeres were visible (Fig. At onset of metaphase I, we observed seven bivalents organized at the centre of the cell surrounded by 21 univalents (Fig. 4e and Supplementary Video 5; n = 31), a configuration that was first proposed over a century ago2,3,22,23. During this stage, microtubules facilitated the separation of the homologous pairs through bipolar attachment, while univalents also showed bipolar attachment (Fig. 4f and Supplementary Video 6; n = 31). In early and late anaphase I, homologous chromosomes migrated first, while single chromatids derived from univalents lagged behind (Fig. Notably, we observed two groups of univalents exhibiting different timing in sister-chromatid separation (Fig. During metaphase and anaphase II, we frequently observed both normal and abnormal spindles, resulting from single chromatids lagging behind from anaphase I (Fig. Homologous pairs derived from bivalents separated normally at end of anaphase II, forming haploid nuclei with seven chromosomes, while single chromatids from univalents lagged behind and were probably eliminated (Fig. Despite the apparent irregularities in male meiosis, viable pollen grains in R. canina (S27) were produced at a rate of approximately 20% (Supplementary Fig. Notably, while at the end of meiosis polyads are formed showing nuclei with varying number of centromere foci (Extended Data Fig. 9 and Supplementary Video 11), the mature binucleate pollen grains seem to contain only a haploid vegetative (1n = 1x = 7 chromosomes) and generative (2n) nucleus, as confirmed by their genome size and composition (Fig. This suggests that, despite meiotic irregularities, a selective mechanism ensures the formation of haploid pollen. Our findings highlight how the atypical centromere behaviour and spindle dynamics during male meiosis in R. canina deviates from canonical meiotic processes but can still result in viable pollen production. By leveraging newly generated genome assemblies of dogroses, we shed light onto the long-standing century-old mystery of the unique Canina meiosis2,3,22,23. Through structural analysis of subgenomes and centromeres, combined with pollen-derived genomics and hybridization experiments, we demonstrate that the bivalent-forming subgenomes in dogroses evolved independently and exhibit distinct interaction patterns during meiosis (Fig. a, Reproduction mode and centromere evolution of the pentaploid dogroses. Caninae have two copies of the S1 subgenome, while dogroses of subsect. Rubigineae have two copies of the R4 subgenome. The S2 and R3 subgenomes are found as a single copy only. Caninae, the S1_h1/S1_h2 chromosomes form bivalents and are transmitted to both pollen and egg cells. The remaining univalent subgenomes (S2, R3 and R4) are transmitted by the egg cell only. Rubigineae, the R4_h1/R4_h2 chromosomes form bivalents and are transmitted to both pollen and egg cells, while the remaining univalent subgenomes S1, S2 and R3 are transmitted only by the egg cell. The analyses of centromeric sequence composition revealed a dominance in ATHILA LTRs in most of the bivalent-forming centromeres of S1 and R4 subgenomes. By contrast, larger CANR4-based centromeres were found in all chromosomes of the permanently univalent S2 and R3 subgenomes. On the basis of the findings that representatives of subsect. Rubigineae contain phylogenetic distant plastids from the Synstylae clade13,31,32,33 (Extended Data Fig. 5b), we propose that two progenitors of the Synstylae clade formed reciprocal hybrids and additionally incorporated R genomes through pollen donors. Subgenomes are represented by one chromosome. Synstylae subgenomes S1 (violet) and S2 (light blue) and Rosa clade subgenomes R3 (light orange) and R4 (dark orange) are shown. The bimodal architecture of centromeres in R. canina is particularly intriguing when considered alongside its asymmetric female meiosis, in which univalent chromosomes are obligatorily transmitted through the egg cell3,49. The prevalence of CANR4 repeats in univalent centromeres could possibly link centromere expansion with their drive in female meiosis (Fig. 5a), a phenomenon in which larger centromeres are preferentially transmitted during meiosis37,50,51. The structural divergence and selective enrichment of CANR4 repeats in the centromeres of univalents may underpin their larger size, ensuring the preferential transmission of univalents through the egg cell and, therefore, maintaining the pentaploid genome structure. This may represent a rare case of an obligate drive mechanism, functioning in a ‘drive or die' manner to maximize the transmission of univalents. However, the occasional presence of ATHILA-based centromeres in some univalent chromosomes suggests that CANR4 expansion alone does not fully explain univalent drive. Furthermore, the structural divergence of centromeres seems also to influence their behaviour in male meiosis, as large CANR4-based centromeres in univalents could possibly promote bipolar orientation and premature chromatid separation in male meiosis. Notably, in Arabidopsis thaliana, bipolar orientation of univalents happens only when sister chromatid cohesion is defective52, but it appears to be more common in wheat univalents53. This observation contrasts with that of female meiosis, in which univalents seem to have monopolar orientation in dogroses3 (Supplementary Fig. Thus, a potential role for sexual dimorphism in sister chromatid cohesion regulation could be part of the adaptations enabling Canina meiosis. It is possible that the absence of a homologous pair (and therefore a competing centromere) in the obligatory univalents (S2 and R3 subgenomes) may facilitate the expansion of CANR4 repeats. By contrast, the S1 and R4 centromeres, which exist within a competitive pairing environment, experience counterbalancing forces that limit CANR4 accumulation (Fig. This is further supported by the absence of solely CANR4-based centromeres in diploid roses R. chinensis and R. rugosa. Furthermore, we provide strong evidence for the emergence of few tandem repeats originating from LTR sequences, which outcompete neighbouring ATHILA elements for CENH3 binding in two R4 univalent chromosomes (Rca1_R4 and Rca4_R4). These findings highlight the higher affinity of tandem repeats for centromere function in dogroses, further emphasizing the role of centromere composition in shaping meiotic behaviour. However, while the observed correlation is intriguing, we acknowledge that future studies will be essential to confirm whether the expansion of CANR4 in univalent centromeres is directly linked to their drive during female meiosis. Despite the lack of recombination, univalent chromosomes retain functional protein-coding genes, as evidenced by high BUSCO completeness and the absence of differential selection pressures. This supports a relatively recent origin of modern dogroses and highlights the resilience of their polyploid genome. Phylogenetic analyses using pollen SCOs and maternally inherited plastid markers further corroborate the polyphyletic origin of dogroses13,31,32,33 with multiple hybridization events contributing to their evolutionary history (Fig. The distinct subgenome ratios observed in R. canina (3:2 Synstylae to Rosa) and R. agrestis (2:3 Synstylae to Rosa) underscore the complexity of their hybrid origins30,32. Pollen SCO data align with cytogenetic studies, indicating that bivalent-forming subgenomes in the subsections Caninae and Rubigineae are phylogenetically distant34,35. Moreover, multiple origins for R. canina are suggested, as the S1 subgenome clusters with five R. canina individuals, while the S2 subgenome clustered with another individual. This model aligns with reports of Canina-like meiosis arising spontaneously in hybrids of diploid sexual Synstylae species54, further highlighting the complex hybrid origin and evolutionary dynamics of dogroses. For genome sequencing, we used the same individual of R. canina (S27), which has already been cytogenetically analysed34 (voucher: GLM12396) from a natural stand (WGS84: 51.1732° N; 14.6271° E; Weißenberg, Saxony, Germany). A vegetative runner was dug on 28 March 2022 and planted in a pot. Clones of the collected plant specimen were cultivated in a greenhouse at the Max Planck Institute for Plant Breeding Research, Cologne, Germany. High molecular mass genomic DNA was isolated from leaves using the NucleoBond HMW DNA Kit (Macherey Nagel). A HiFi library was prepared according to the manual of the HiFi SMRTbell Libraries using SMRTbell Express Template Prep Kit 2.0 (Pacific Biosciences) with initial DNA fragmentation using Diagenode Megaruptor 3 and final library size binning into defined fractions by Blue Pippin with 0.75% agarose cassettes (Sage Science). The size distribution was again controlled by a Femto pulse system (Agilent). Size-selected libraries were then sequenced on the Sequel II device with a Binding Kit 2.0 and Sequel II Sequencing Kit 2.0 for 30 h using two SMRT cells (Pacific Biosciences). Moreover, a chromatin-capture library was prepared from 0.5 g of fresh-weight leaf material input. All treatments were performed according to the recommendations of the Dovetail Omni-C kit for plants (Dovetail Genomics). As a final step, an Illumina-compatible library was prepared (Dovetail) and paired-end 2 × 150 bp deep-sequenced on the HiSeq 3000 (Illumina) device. All libraries were sequenced at the Max Planck Genome Centre Cologne at the Max Planck Institute for Plant Breeding Research. A phased chromosome-level genome was assembled using the generated PacBio HiFi and Hi-C data. First, a phased primary assembly was obtained by running Hifiasm55 using 50 Gb of PacBio HiFi reads in combination with Dovetail Omni-C reads with the following command: hifiasm -o out.phased.asm.hic --h1 hic.R1.fastq.gz --h2 hic.R2.fastq.gz hifi.reads.fastq.gz. In the default diploid mode, we generated two sets of phased contigs. Each set was further scaffolded to the chromosome scale using Salsa256, followed by successive rounds of manual curation and rescaffolding. We then identified 14 and 21 pseudochromosomes, respectively. We used Benchmarking Universal Single-Copy Orthologues (BUSCO, v.5.4.0)57 to evaluate the completeness of 35 chromosome-level scaffolds and for each of the four subgenomes. The lineage database used for running BUSCO was eudicots_odb10. The protein sequences were converted from the assembly using the GFF analysis toolkit AGAT: agat_sp_extract_sequences.pl -g annotation.gff -f genome.fasta -p --type cds -o protein.fasta. The k-mer-based tool Merqury (v.1.3)58 was used to estimate both the completeness and base quality of the chromosome assembly. The quality value of the chromosome assembly was greater than 66.6, and the quality value of each chromosome was at least 62. Read k-mers were built from HiFi sequences by Meryl (v1.3) with a k-mer size of 31 bp. During the scaffolding step, we noted the absence of approximately 20 Mb (including the centromere) on chromosome Rca2_S1_h2. Further validation using fluorescence in situ hybridization (FISH) showed that this chromosome should indeed have a large array of CANR4, as found in its homologous chromosome Rca2_S1_h1 (Supplementary Fig. We noted that this assembly error was probably generated by the presence of a small translocation found at the start of the missing region in Rca_S1_h2 compared with Rca_S1_h1. We further mapped the HiFi reads to the region present in Rca_S1_h1 and found robust evidence for the presence of five copies of this region in the genome. We then concluded that this region was incorrectly missing from the Hifiasm assembly due to its high degree of similarity between the S1_h1 and S1_h2 haplotypes. We therefore duplicated the Rca2_S1_h1 region (37265454–54065178 bp) and manually assigned it to the expected position on Rca_S1_h2 (from Rca2_S1_h2: 33518152 bp). We also downloaded available data from the Darwin Tree of Life (DToL) project for another accession of R. canina (PRJEB79801) and for R. agrestis (PRJEB79880) and performed phased chromosome-level genome assemblies as described above. R. agrestis from DToL revealed two copies of the R4 subgenome; our previous studies suggested that some accessions of R. agrestis were of hybrid origin, which should then have two copies of S1 subgenome45,59. k-mer analysis to estimate genome size was performed using jellyfish (v.2.3.0)60 and Genomescope (v.2.0)61. The pentaploidy of R. canina was further confirmed and analysed using Smudgeplot (v.0.2.5)61. GetOrganelle (v.1.7.7.0)62 was used to de novo assemble the first draft of the plastid genome using 2× 150 bp Illumina short-read data (Sequence Read Archive (SRA): ERS1370372). This toolkit implements Bowtie 263 to initially find reads mapped to a plant chloroplast database and SPAdes64 for de novo assembly and iterative extension. During the assembly and iteration process, BLAST+65 was used to identify off-target contigs, which were then removed or trimmed. The resulting plastid genome was then used as a reference for mapping the original reads back using Geneious Prime v.2023.2.1 (Biomatters), allowing only mapping of paired reads mapped nearby with a minimum overlap of 75 bp and a minimum overlap identity of 98%. The results were manually examined and corrected where necessary. The initial annotation of the chloroplast genome was performed using GeSeq (v.2.03)66. The annotation included the chloroplast inverted repeats (IRs), rps12 interspersed gene, protein-coding sequences, tRNAs and rRNAs using 55% identity as thresholds for annotation of proteins and 90% for DNA as well as RNAs. Furthermore, tRNAscan-SE (v.2.0.7)67 and Chloë (v.0.1.0)68 were used as additional annotators within GeSeq. The annotations were manually edited using Geneious Prime v.2023.2.1 (Biomatters). The presence of chloroplast genomes differing in the orientation of the single-copy units (large single-copy (LSC) region, small single-copy (SSC) region) was checked by selecting motifs from the border region of the IR and the single-copy units (LSC-trnH-GUG 5′-GGTTCAATTCCCGTCGTTC-3′ or LSC-rps19 5′-GTGACACGTTCACTGAAAAAA-3′ and IRb-rps19-rpl2-IGS 5′-AGACGAAGAAACAAATTCTAT-3′; SSC-ndhF 5′-TGTAATAATATAATAATTGAA-3′ or SSC-ycf1 5′-CGACCCTAAACGATGGAATCG-3′ and IRa-ycf1 5′-TTGAAAAACCCGTTGTAACTAT-3′), noting their relative orientation to each other on the same reads using SeqKit (v.2.6.1)69. The assembled R. canina chloroplast genome had a length of 156,650 bp and a classical quadripartite structure (Supplementary Fig. Different isomers were found to differ in the orientation of the SSC and LSC (flip-flop configuration). We computed a chloroplast phylogeny using 37 samples, including sequences downloaded from GenBank and newly assembled data (Supplementary Data 17). The alignment was performed with MAFFT70, and the phylogenetic tree was calculated using IQ-TREE71 with the following settings: -m MFP --con-tree --burnin 250 -B 1000 -T 36 --wbtl. The assembled chromosomes were subjected to pairwise comparisons presented as dot plots using the Synteny Mapping and Analysis (SyMAP) tool72. Multiple alignments within the synteny groups comprising five R. canina chromosomes plus R. chinensis and R. rugosa assemblies were carried out in the CLC Genomics Workbench (CLC) using the ‘whole genome alignment' plugin with the following parameters: minimum initial seed length of 250; minimum alignment block length of 250. The aligned chromosomes were subjected to pairwise comparisons. Multifasta files containing assembled short-read sequences of pollen SCO loci from eight different dogrose individuals (three samples of R. canina, two samples of R. corymbifera (subsect. Caninae) and three samples of R. rubiginosa (subsect. Rubigineae)) that were not sequenced by long reads (Supplementary Data 17) were used as queries to map them with the software BWA73 with the aln command to the R. canina chromosome assembly. From the sequence alignment map (.sam) file, those chromosome hits with only one alternative were filtered according to the ‘XA:Z:' flag using a Python script written by GPT-4 (ChatGPT Plus, OpenAI). A bubble map displaying the mean counts of chromosome pairs within different subsections was drawn with ggplot274. Chromosome synteny was analysed with the Synteny and Rearrangement Identifier (SyRI)75. For this purpose, chromosomes of subgenomes S1_h1, S1_h2, S3, R3 and R4 were aligned against each other within each linkage group (Rca1–Rca7) by minimap276,77 using the following command: minimap2 -ax asm5 --eqx -t 16 genome1.fa genome2.fa | samtools sort -@8 > aln.sorted.bam. Moreover, subgenome S1 was also aligned with R. chinensis (NCBI: GCA_041222415)16 and R. canina subgenome R4 was aligned with R. rugosa (NCBI: GCA_958449725.1; https://www.darwintreeoflife.org/) to analyse its synteny. To keep all of the chromosomes arranged in the same order as R. canina and for better visualization, chromosomes 2, 5 and 7 of R. chinensis were inverted, and the chromosomes of R. rugosa were reordered to 6, 1 (inverted), 7, 5 (inverted), 2, 3 and 4, corresponding to chromosomes 1–7 in R. canina. SyRI was implemented for all of the aligned genome pairs using the following command: syri -c aln.sorted.bam -r genome1.fa -q genome2.fa -k -F B --nc 16. Syntenic orthologues among the primary annotations of diploid strawberry Fragaria nilgerrensis78, R. chinensis16, R. rugosa (GCA_958449725.1, DToL; https://www.darwintreeoflife.org/) and the five sets of chromosomes of R. canina were inferred using the GENESPACE (v.1.2.3)79 pipeline with the default parameters. In brief, GENESPACE compares protein similarity scores into syntenic blocks using MCScanX80 and Orthofinder (v.2.5.4)81 to search for orthologues/paralogues within synteny constrained blocks. Syntenic blocks were used to query pairwise peptide differences among progenitor alleles, determine divergence among progenitor orthologues using R. chinensis syntenic anchors and search for specific orthogroups. Synteny information was obtained using the SynMap tool on the CoGe platform82,83. Only genes within synteny blocks were considered, including not only gene pairs but also singleton genes in each genome that lost their counterpart in the other genome due to fractionation or other gene loss. The identification of syntelogues between species was performed using SynMap2 (https://genomevolution.org/wiki/index.php/SynMap2), which internally uses LAST for sequence alignments84, and then fractionation bias was analysed with FractBias85. Protein-coding sequences (CDSs) were extracted for each R. canina subgenome according to coordinates from the gene structural annotation file using GffRead (v.0.12.6)86 and translated into amino acid sequences using the transeq command from EMBOSS87. Additional amino acid sequences and CDSs of R. rugosa (BioProject: PRJNA1061178; https://www.darwintreeoflife.org/), R. chinensis (NCBI: GCF_002994745.2)14 and Fragaria vesca (NCBI: GCF_000184155.1)88 were downloaded. The CDS and amino acid sequences were validated, for example, for correct start codons or methionine as the first amino acid in the proteins, using Python scripts. The confirmed proteomes were subsequently analysed using OrthoFinder81 to identify common single-copy orthologues. According to the protein IDs, FASTA files for each orthologue gene containing five proteins of R. canina together with the three of outgroups were aligned with MAFFT (v.7.490)70. On the basis of the aligned proteins, corresponding CDSs were codon based aligned using PAL2NAL89 and DNA alignments were transformed into PHYLIP format. The PAML pipeline90 with yn00 was used in a looped pairwise mode over all PHYLIP files for each subgenome and outgroup to estimate the nonsynonymous (dN) and synonymous (dS) substitution rates, as well as their ratio (dN/dS = ω). The results based on the Yang–Nielsen91 method were extracted from PAML output files and combined using a Python script and graphical visualized with ggplot274 in the R environment92. To covert the relative evolutionary time (Timet) from yn00 into absolute divergence time (TMa) in millions of years ago (Ma), we used F. vesca as a fixed calibration point with its fossil record of 2.96 Ma (refs. The relative divergence time Timet for each pairwise compared gene was multiplied with a scaling factor as follows: All scripts were developed with the help of ChatGPT-4o (ChatGPT Plus, OpenAI). We first generated whole-chromosome multiple alignments in synteny groups 1–7 using the Whole Genome Alignment tool in ClC workbench (Qiagen). The algorithm identifies seeds, that is, short stretches of nucleotide sequences that are shared between multiple genomes but not present multiple times in the same genome. These seeds were then extended using a HOXD scoring matrix, and the HOXD substitution score was combined with an adjustment term based on k-mer frequency to avoid spurious matches to repetitive regions in the genome95. The program parameters were as follows: minimum initial seed length, 250; minimum alignment blocks, 250; and mismatches in seeds, allowed. The chromosome phylogenies were constructed from multiple alignments using RAxML (v.8.2.12)96 with the GTRGAMMAI model. The diploid accessions were chromosome assemblies of R. chinensis16 and R. rugosa. For R. rugosa, the original chromosomes were renamed to fit the R. chinensis synteny. We used SubPhaser41 (default parameters) to phase and partition the subgenomes of the pentaploid R. canina and R. agrestis by assigning chromosomes to subgenomes based on differential repetitive k-mers. These were assumed to have expanded during the period of independent evolution after divergence from the nearest common ancestor and before the stepwise hybridization events (the divergence–hybridization period). A subgenome is considered to be well phased when it displays distinct patterns of both differential k-mers and homoeologous chromosomes, confirming the presence of subgenome-specific features, as expected. LTR insertion times were calculated by Subphaser as follows: LTR-TRs were de novo detected using LTRharvest (v.1.6.1)97 and LTRfinder (v.1.07)98. To reduce false positives, TEsorter (v.1.3.0)99 was used to reconstruct the classification of LTR-RTs and further refine this classification. The subgenome-specific k-mer sequences were mapped to the LTR-RT sequences using a substring match procedure to identify the subgenome-specific LTR-RTs using the Fisher's exact test. Two LTRs of each subgenome-specific LTR-RT were retrieved and the nucleotide divergence was estimated using the Jukes–Cantor 1969 model. To isolate the nuclei from embryo and endosperm tissue, nutlets from fruits of the sequenced individual of R. canina S27 (voucher: GLM12396) were first cracked with pliers. The embryo and endosperm were then carefully transferred into a droplet of nuclei isolation buffer (CyStain PI Absolute P; Sysmex-Partec) in a Petri dish and chopped with a sharp razorblade. After adding additional nuclei isolation buffer to a final volume of 500 µl, the nuclei suspensions were filtered through 50 µm disposable filters (CellTrics, Sysmex-Partec), stained with 4′,6-diamidino-2-phenylindole (DAPI) to a final concentration of 1.5 µg ml−1 and stored on ice until use. The measurements were performed on a CyFlow Space flow cytometer (Sysmex-Partec) equipped with a high-power UV LED (365 nm). To analyse the phylogenetic origin of the subgenomes of allopentaploid R. canina, we sampled 30 rose individuals of 24 diploid species across the genus Rosa. Hesperhodos), 10 species from sect. Synstylae, seven from sect. Rosa (Cinnamomeae, including sect. Carolinae), four from sect. Pimpinellifoliae and one individual of R. chinensis (sect. Chinensis) were sampled from the living collection of the Europa-Rosarium Sangerhausen. Moreover, one accession of R. majalis (sect. Rosa) was collected from the Botanical Garden Würzburg, and R. persica (subg. Hulthemia) from Botanical Garden Jena. Species were rechecked using their respective floras101,102,103, and the material was compared with available herbarium specimens available online (JSTOR Global Plants, https://plants.jstor.org/; Moscow Digital Herbarium, https://plant.depo.msu.ru). Herbarium vouchers were deposited in the Herbarium Senckenbergianum Görlitz (GLM; Supplementary Data 17). To determine bivalent-forming genomes, we sampled pollen from several individuals of Rosa sect. Caninae: three 5x R. canina, two 5x R. corymbifera; subsect. Rubigineae: three 5x R. rubiginosa; Supplementary Data 17). We collected anthers from 50 to 100 freshly opened flowers under dry weather conditions in early May 2021 in the field, stored them in open glass for 1 day to allow the anthers to open and subsequently transferred them to a 50 ml tube. Anthers were carefully removed, and the pollen powder was collected at the bottom of the tube by gentle centrifugation. The pollen powder was then tapped out over clean paper and transferred to tubes with the help of a spatula. This procedure was repeated three times. Pollen grains were stored in a refrigerator until use. Nuclei of mature pollen grains were isolated by the filter bursting method104 using the nuclear isolation buffer as described previously105. Pollen grains were burst on the surface of a 20 μm disposable CellTrics filter (Sysmex-Partec). The resulting nuclear suspension was stained with propidium iodide (50 µg ml−1, PI) and run on a BD Influx cell sorter (BD Biosciences). Nuclear populations were identified in a dot plot showing the PI fluorescence signal (log scale) versus side scatter signal (SSC, log-scale). A sort gate was defined based on the corresponding fluorescence intensity (lin-scale) histogram. A total of 200,000 individual generative nuclei (volume, around 400 µl) were collected into a 1.5 ml reaction tube using the ‘1.0 Drop Pure' sorting mode of the BD FACS software (BD Biosciences). After adding 50 µl of 1× TE and 50 µl of NaN3, the nuclei were sedimented by centrifugation (1,000g for 10 min at 4 °C). Next, 300 µl of the supernatant was removed, and the nuclei with the remaining liquid were stored at −20 °C. The gating strategy to isolate generative nuclei of R. canina is presented in Supplementary Fig. DNA from diploid rose species was first extracted from 20 mg of silica-dried leaf tissue according to the ATMAB protocol106 and subsequently purified using the Mag-Bind Total Pure NGS Kit (Omega Bio-Tek, Nocross) according to the manufacturer's manual. DNA from flow-sorted pollen nuclei was extracted using the Mag-Bind Plant DNA DS Kit with the modification that permanent but careful mixing was performed during binding and elution because the DNA quantities ranged from 37 ng to 236 ng. The DNA yield was quantified using the Qubit 4 Fluorometer (Thermo Fisher Scientific). To analyse nuclear single-copy regions in rose genomes, we used published SCO tags107. The SCO tags were originally developed to be amplifiable by PCR and covered coding as well as non-coding regions. 107, which consisted of SCO tags of 17 rose species and seven outgroup species of the Rosaceae family. These sequences were filtered for uniqueness so that duplicates were removed and searched with BLAST in the R. chinensis haploid line genome (v.1.0)15. Owing to the structural gene model annotation of the R. chinensis genome, we were able to identify 923 full-length nuclear genes with single-copy characteristics. The target-capturing baits were designed by the Agilent bioinformatics service (I. Kisakesen, Agilent Technologies) and covered exons + UTRs with flanking regions and small introns of the selected genes in the R. chinensis genome. Finally, the target consisted of 5,794 sequences of different lengths (the shortest at 179 bp and the longest at 6,544 bp) named according to R. chinensis gene prediction and had a total size of 2 Mb (Supplementary Data 12). All target sequences were covered by 2× tiling with a total of 85,670 specific baits. For target enrichment, we used the SureSelect XT HS2 DNA system with precapture pooling (Agilent Technologies) and target design as described above. For diploid roses, 200 ng of input DNA was used, and for pollen DNA, 36–200 ng of input DNA was sheared with a Bioruptor Pico sonication device (Diagenode) to a recommended fragment size of 180–250 bp. The Illumina short-read libraries were amplified for 9 cycles after adapter ligation, pooled for precapture to 16 samples and then postcapture library pools were amplified again with 12 cycles of PCR amplification. The library pools were sequenced in 150 bp paired-end mode on the Illumina NovaSeq 6000 system by Novogene with approximately 1 GB of data output per sample. To analyse the ploidy of the samples, in vitro flow cytometry was performed on silica-dried leaflets according to a protocol described previously45 using R. arvensis (2n = 2x = 14) as an internal standard. The fluorescence intensity was measured using the CyFlow Ploidy Analyser (Sysmex Partec), and the data were analysed using Flowing Software v.2.5.1 (Turku Bioscience Centre). Each sample was measured three times with a minimum of 3,000 particles. To estimate the ploidy of the samples in silico, we used K-Mer Counter (KMC) (v.3.1.1)108,109 to generate a k-mer database from FASTQ sequence files containing short-read data covering SCOs. The setting was a k-mer size of 21, a minimum count for a k-mer to be included of 1 and an upper limit for k-mer counts of 5,000. To avoid noise, KMC database reduction was performed using the transform operation with the L 30 and U 5000 settings. With smudgeplot61 analysis and its hetkzer operation, the coverages of the identified k-mer pairs were written to a ‘_coverages.tsv' file. A custom R script with ggplot274 and data.table packages92 was used to plot the distribution of frequencies of different SNP ratio classes. For each sample, the ploidy level was then estimated by visual inspection of the plots. The raw SCO reads were trimmed using Trimmomatic (v.0.39)110 with the following settings: 2:30:8 LEADING:13 TRAILING:13 SLIDINGWINDOW:4:19 MINLEN:36. We updated the script from a previous study111 to run it with current package versions and used it for mapping, variant calling and sample-specific reference building. Using SAMtools (v.1.16.1)112, the reads were sorted and indexed, and duplicates were removed. Notably, approximately 98% of trimmed reads were successfully mapped to the target. Hits with exactly one alternative mapping position were subsequently filtered. After mapping, the Genome Analysis Toolkit (GATK) (v.4.1.9.0)113 was used with the operation HaplotypeCaller114 for variant calling, BaseRecalibrator and ApplyBQSR were used to realign around SNPs and indels, and FastaAlternateReferenceMaker was used to create a sample-specific consensus sequence as a reference for each SCO locus in each sample. The provided ploidy level for the HaplotypeCaller was 2 (diploid) for both the pollen and diploid roses with regular meiosis, and the --max-alternate-alleles flag was set to 6, so that although the pollen is monoploid, it would be possible to call potential variances. The SCO target was used as a query for a local search with BLAST+65 in our R. canina S27 genome assembly with a customized output table (-outfmt 6 qseqid sseqid pident length qstart qend sstart send evalue bitscore) and additional in the DToL R. canina (PRJEB79801) and R. agrestis (PRJEB79880; https://www.darwintreeoflife.org/) genomes also assembled by us (see below). Those SCO loci that had only five hits, one each on subgenomes S1_h1, S1_h2, S2, R3, R4 and R4_h1, R4_h2, R3, S2 and S1 for R. agrestis, respectively and within the same linkage group, were filtered and considered single copies. A main list of common single-copy loci for all three genomes was created to preserve the correct order and used to extract the filtered loci from the BLAST outputs with the grep command. The filtered BLAST output was then converted into a BED file containing the sequence coordinates using a bash script written with the help of GPT-4 (ChatGPT Plus, OpenAI). Using the BEDtools (v.2.30.0)115 command getfasta, sequences for each SCO locus were extracted from the R. canina genome assembly and written into a multifasta file. To obtain sequences with the same strand orientation, two locus lists were also created: one of the loci with a positive strand orientation and one with a negative-strand orientation. Loci with negative-strand orientation were identified by calculating the end coordinates minus the start coordinates and filtering according to negative values. According to both lists, the sequences were extracted and stored in two separate multi-FASTA files. Sequences with negative-strand orientation were reversed and complemented with SeqKit116 and combined with the positive strand-oriented SCO sequences in one fasta file. Finally, for each subgenome (S1_h1/S1_h2, S2, R3, R4 for R. canina and R4_h1/h2, R3, S2, S1 for R. agrestis), the extracted SCO sequences were concatenated in the same order according to the main locus list and written to subgenome-specific fasta sequences. The same procedure was used for the haploid genome assemblies of Rubus ideaus (GenBank: GCA_030142095.1)117 and three strawberry species, F. vesca subsp. vesca (GCA_000184155.1)88, Fragaria iinumae (GCA_009720345.1)118 and F. nilgerrensis (GCA_010134655.1)78 as outgroups. Moreover, the same single-copy loci considered in the genome assembly were extracted and concatenated in the same order with target enrichment samples from nine pollen samples and 30 leaf samples of 26 diploid rose species. The concatenated multilocus sequences were aligned using MAFFT (v.7.490)70. Finally, a maximum-likelihood phylogenetic tree was generated by applying IQTREE71 with ModelFinder using the following settings: iqtree2 -s -m TEST --con-tree --burnin 250 -B 1000 -T 12 --wbtl. The tree figures were graphically finalized with MEGA X119 and Inkscape v.0.92.3 (2405546, 2018-03-11) software. Synthetic hybrid R. canina (seed parent) × R. rubiginosa (pollen parent) (sample ID, D62b_2; SRA: SRR15033882) was a cross between R. canina (sample ID, D3b_2; SRA: SRR15033883) and R. rubiginosa (sample ID, D145b_2; SRA: SRR15033877), and the second synthetic hybrid R. rubiginosa (seed parent) × R. corymbifera (pollen parent) (sample ID, D166b_2; SRA: SRR15033879) was a cross between R. rubiginosa (D145b_2; SRA: SRR15033877) and R. corymbifera (sample ID, D81b_2; SRA: SRR15033881). These hybrids were originally produced by Wissemann and Hellwig43 and kept as a living plant in the Botanical Garden Gießen, Germany. Whole-genome short-read sequencing was performed for both hybrids and their parental plants. The mean coverage of the maternal plant (sample ID, D3b_2) is ~27×, and the paternal plant (sample ID, D145_b2) is ~27×. The hybrid's (sample ID, D62b_2) coverage is ~29×. The reciprocal hybrid (sample ID, D166_b2) has an average of ~30× coverage, whereas its paternal plant (sample ID, D81_b2) is ~19×. The reads from these six samples were mapped to the S1 subgenome of our assembled R. canina, respectively, using bowtie2 (v.2.5.1)63 with the default parameters. Filtering was applied for all alignments with the same setup ‘samtools view -F 3340 --min-MQ 1'. The coverage of each sample was calculated by ‘bedtools coverage' (v.2.30.0) with a 100 kb window size. SNPs were called with the filtered alignments by bcftools (v.1.9)112. Specifically, ‘bcftools mpileup' ran first with the minimum mapping quality 1, then ‘bcftools call' ran with flags ‘--keep-alts --variants-only --multiallelic-caller'. Structural analysis of DNA sequences of whole chromosomes and centromere cuts were performed with ModDotPlot (v.0.9.0)120 using the default parameters. ModDotPlot is a dot plot visualization tool designed for large sequences and whole genomes. The method outputs an identity heat map by rapidly approximating the average nucleotide identity between pairwise combinations of genomic intervals. The predicted gene model structures in the nuclear genome were annotated by applying the full-length chromosome sequences to Helixer121. Moreover, complete LTR retrotransposons were annotated with the DANTE and DANTE-LTR tools implemented in RepeatExplorer2122,123. R. canina short-read data (SRA: ERR1662939) were subjected to clustering analysis using the RepeatExplorer2 pipeline, and the output library of repeats was subsequently used to annotate the genome with the implemented RepeatMasker124. Tandem-repeat annotation and genome abundance estimation were performed using TAREAN and TideCluster implemented in RepeatExplorer2122. Total mRNA was extracted from the leaf tissue of R. canina S27 using the Spectrum Plant Total RNA-Kit (Sigma-Aldrich). The RNA-sequencing library was prepared with poly(A) enrichment and then sent for sequencing on the NextSeq 2000 platform with 2 × 150 bp mode, resulting in 33,594,132 reads. For a more accurate mapping of RNA sequences, the annotated tandem repeats and transposable elements were hard-masked from the genome. RNA alignment was done using hisat2 (v.2.1.0)125 with the flag --no-mixed. The output was then filtered by only allowing for tag NM:0 and minimum mapping quality 2. To count the transcripts number for each gene, we converted the masked genome to protein sequences based on Helixer121 structural annotation, and then functionally annotated the protein sequences by Mercator4 (v.7.0)126 with both Prot-scriber and Swissprot databases, then htseq-count (v.2.0.1)127 was applied to count the transcripts for all annotated proteins. The gene expression was analysed by DESeq2128. As the high homozygosity between the haplotypes of S1 subgenome, RNA reads were aligned to S1_h1, S2, S3, S4 genome, and the expression level of S1 was then halved (Supplementary Fig. For detecting the functional centromeres of R. canina S27, we designed a specific polyclonal antibody against its CENH3 protein (ARVKHTAARKDRIKTARRQP-C, AB016310), synthetized by LifeTein with immunization in rabbits. The ChIP experiment was performed as described previously129 with a few modifications. Young leaves (around 2–5 g) of R. canina S27 were collected and cross-linked in 4% formaldehyde in 1× PBS on ice with vacuum infiltration applied for 30 min. The quenching was performed applying 1 M glycine in each sample followed by vacuum infiltration at room temperature for 15 min. The material was then macerated in liquid nitrogen and the chromatin was extracted. After extraction, the chromatin was sonicated for 30 min on a Bioruptor (Diagenode) until fragments of around 200–600 bp length were achieved (30 s on; 30 s off; in high mode). The sonicated chromatin was incubated over night at 4 °C with 1 µg of each polyclonal antibodies (anti-CENH3 specific for R. canina (LifeTein, AB016310) raised in rabbit and anti-histone H3 (Active Motif, 39064) raised in mouse). Samples with no addition of primary antibodies were also incubated as input control samples and at least two experimental replications were used for each ChIP combination. After incubation, protein beads (anti-rabbit: rProtein A Sepharose FastFlow 50% slurry; anti-mouse: rProtein G Sepharose FastFlow 50% slurry (GE Healthcare)) were washed and added to each complex protein–antibody and incubated for at least 2 h at 4 °C in slow rotation. The final recovered chromatin was eluted from the beads, followed by a de-cross-linking step and final DNA extraction. After quality control using the 4200 TapeStation System (Agilent Technologies), the samples were forwarded for 150 bp paired-end Illumina sequencing. For the analysis, the raw 150 bp paired-end reads were quality checked and then mapped to the R. canina haplotype phased reference genome using the default parameters in bowtie263. The BAM file was converted to bigwig using the bamCompare tool from deeptools2130, and then normalized to reads per kilobase of transcript per million reads mapped. The plots showing the distribution of different genomic features per chromosome or specific region were constructed using pyGenomeTracks132. The ChIP–seq signals in metaplots to compare chromosome (Extended Data Fig. 3e) were calculated by bamCompare with parameters ‘--ignoreDuplicates --scaleFactorsMethod readCount --operation log2' to normalize the CENH3/H3 by read coverage. Functional centromere regions in the genome assembly of R. canina S27 were annotated based on the detection of CENH3 peaks with MACS3 (see above). After alignment to the annotated functional centromeres in R. canina S27, comparable centromeric regions were extracted from DToL R. canina and R. agrestis (https://www.darwintreeoflife.org). The repeat abundance of CANR4 satellite repeats and Ty3/Gypsy ATHILA retrotransposons in the predefined centromeric regions were determined in base pairs for each chromosome of the three investigated Rosa genomes (R. canina S27, R. canina DToL and R. agrestis DToL). A Shapiro–Wilk normality test was used to check normal distribution of the data with R (v.4.3.3)92 (29 February 2024). A bivariate Bayesian generalized linear mixed model was implemented using the MCMCglmm package133. The model included pairing type (bivalent B, univalent U and univalent in R. canina but bivalent in R. agrestis Ub) as a fixed effect, while subgenome, genome, and synteny group were random effects, with an unstructured covariance structure (us(trait):random_effect) to account for correlations between response variables. MCMC settings included 100,000 iterations, with a 50,000 burn-in and a thinning interval of 50 and the family parameter was set according to ‘gaussian'. Data visualization was performed using the ggplot274, patchwork134, tidyr135 and dplyr136 packages (Supplementary Fig. The correlation of CANR4 size with CENH3 abundance (Fig. 3e) was calculated by Spearman's rank correlation as the Shapiro–Wilk normality test resulted in P ≪ 0.5. Linear regression model was fitted using the lm function in R (v.4.4.0), with multiple R2 value as 0.842 and adjusted R2 value as 0.836. cenLTR sequences were primarily annotated as tandem repeats using TAREAN and TideCluster implemented in RepeatExplorer2122. Further sequence similarity with LTR retrotransposons was performed using the transfer annotation tool of Geneious Prime v.2025.0.2 (https://www.geneious.com) with a minimum sequence similarity threshold of 75%. Using a Geneious Prime plugin for ClustalO137, we performed alignments of consensus cenLTR sequences against the regions with the highest similarity found in the R. canina S27 genome, which all corresponded to different ATHILA elements on chromosomes Rca1_R4 and Rca4_R4. Consensus sequences of cenLTR1–4 are available in Supplementary Dataset 14. To investigate the methylome of R. canina, we performed enzymatic methyl-sequencing (EM-seq). For this, we extracted genomic DNA from young leaves and the samples were then prepared for an Illumina-compatible library using the NEBNext Enzymatic Methyl-seq Kit and further sequenced on the HiSeq 3000 device with paired-end orientation. We ended up with 68,632,618 pairs of 150 bp reads. EM-seq data were first aligned to the S1_h1, S2, R3, R4 combined subgenomes with Bismark (v.0.23.0) with the flag ‘--local' and duplications were removed by deduplicate_bismark. CpG-, CHG- and CHH-context methylations were then extracted by bismark_methylation_extractor (v.0.23.0). The output was converted to bedgraph by bismark2bedGraph (v.0.23.0) with the flag ‘-CX' activated for CHG and CHH contexts to visualize the methylations chromosome-wide and on the centromeres. 14), all signals were smoothed by the spline.smooth function with spar 0.3 in R (v.4.4.0). CENH3 enrichment, DNA methylations, ATHILA density and CANR4 were calculated in 50 kb adjacent windows and averaged by all chromosomes of the corresponding subgenome. All chromosome coordinates were scaled on the basis of their distance to centromere against the distance of centromere to telomere. Centromere position was defined on the basis of where the maximum CENH3 enrichment was located. Mitochondrial sequences were masked when computing the CENH3 enrichment. All signal values (y axis of metaplots) were scaled from 0 to 1 based on the global minimum to global maximum except for DNA methylations, for which the original percentage values were retained. For immunodetecting the centromeres of R. canina S27, we used polyclonal antibodies against CENH3 protein (see above) and kinetochore protein KNL1 (C-EDHFFGPVSPSFIRPGRLSD, AB015677-3) described previously48, also synthetized by LifeTein and raised in rabbits. To identify the microtubules, we used a commercial antibody against alpha-tubulin (Sigma-Aldrich, T6199) with immunization in mouse. For analysing the distribution of these markers in mitotic cells, root tips were fixed after a pretreatment in 0.2 mM 8-hydroxyquinoline for 4 h at 18 °C. For meiotic stages, the young anthers were directly fixed with no previous antimitotic pretreatment. The immunodetection experiment was performed according to a previously published protocol138 with modifications to R. canina material. After the fixation, the samples were washed three times with buffer A and then digested in enzymatic solution containing 1% cellulase-onozuka, 1% cellulase, 1% pectolyase Y23, 1% cytohelicase, 1% macerozyme and 10% pectinase in citrate buffer for 1 h in a humid chamber at 37 °C. To remove the excess of enzymatic solution, the material was gently washed with buffer A and left on ice until the preparation of the slides. A couple of anthers were placed and dissected in a drop of buffer A on a 18 × 18 mm high-precision coverslips, a few μl of polyacrylamide solution (25 µl 15% polyacrylamide (Sigma-Aldrich, A3574) in buffer A plus 1.25 µl of 20% sodium sulfite and 1.25 µl of 20% ammonium persulfate) were added to the dissected anthers, quickly mixed and a second coverslip was put above the first making a sandwich gently squeezing the anthers with a needle to liberate the meiocytes. The sandwiches were allowed to dry for up to 1 h until complete polymerization. After this, the coverslips were carefully separated and incubated in PBS with 1% Triton X-100 and 1 mM EDTA for at least 1 h, then more 2 h in blocking solution containing 3% BSA in PBS with 0.1% Tween-20. After this period, the primary antibodies were diluted in 1:500 (CENH3 and KNL1) and 1:200 (alpha-tubulin) ratios in blocking solution and applied on each sample, which were sequentially incubated at 4 °C for 48 h. After primary antibody incubation, primary antibodies were detected using secondary antibodies conjugated with specific fluorophores (Alexa Fluor 488 and Abberior StarRed and STAROrange for STED microscopy, also diluted in blocking solution in a proportion of 1:250) and incubated in a dark humid chamber at room temperature for at least 2 h. The material was then washed four to five times for 20 min each in 1× PBS + 0.1% Triton X-100 and then mounted in SlowFade Gold medium containing DAPI. The slides were photographed using a super-resolution STED microscope (Abberior instrument facility line; https://abberior-instruments.com/) and posterior brightness and contrast adjustments were done in Photoshop. For mitotic chromosome preparations, root tips and young flower buds from R. canina S27 plants cultivated in the greenhouse were collected and then fixed in methanol:acetic acid solution (3:1 (v/v)) for 2–24 h at room temperature and then kept at −20 °C until use. After fixation, the root tips were pretreated with an enzymatic solution of 2% cellulase R10-onozuka (Duchefa Bioquemie)/20% pectinase (Sigma-Aldrich) in 0.1 M citric acid for 40 min at 37 °C in a humid chamber and then squashed in a drop of LB01 buffer (15 mM Tris, 2 mM Na2EDTA, 80 mM KCl, 20 mM NaCl, 0.5 mM spermine, 15 mM β-mercaptoethanol, 0.1% Triton X-100 (pH 7.5)) and, after frozen in liquid nitrogen, the coverslips were removed. For meiotic chromosome preparations, the anthers of R. canina C1 (GLM-P-0181117) were dissected from fixed flower buds around 0.5 cm in length. Anthers were washed with 1% (w/v) polyvinylpyrrolidone 40 (PVP-40; Sigma-Aldrich Chemie) and 0.5% (v/v) Triton X-100 for 15–20 min, followed by enzymatic digestion overnight in a humid chamber at 4 °C in 1% (w/v) cellulase Onozuka R-10 (Serva), 0.2% (w/v) pectolyase Y-23 (Sigma-Aldrich), 0.5% (w/v) hemicellulose (Sigma-Aldrich) and 0.5% (w/v) macerozyme R-10 (Duchefa Biochemie) dissolved in citric buffer (0.04 M citric acid and 0.06 M sodium citrate). The anthers were macerated on a slide, squashed in a drop of 70% acetic acid and fixed by freezing in liquid nitrogen. For FISH experiments, a 22 bp oligo probe directly labelled with a Cy3 fluorophore at the 5′ terminus was designed, synthesized by Sigma-Aldrich and then used to detect the CANR4 satellite repeat (Cy3-5′-ACCCTAGAAGCAAGAAGTTTGG-3′) or an insert of the plasmid carrying the CANR4 dimer (GenBank MK069593) was used as a FISH probe20. For detection of the centromeric LTR ATHILA retrotransposon sequences, we designed a probe based on clustering analysis of Illumina reads (SRA: ERR1662939) using the RepeatExplorer2 pipeline. It was revealed that cluster 5 (CL5) contig contained Ty3/Gypsy/ATHILA sequences. The CL5 contig was used to design PCR primers amplifying a 180 bp product from R. canina genomic DNA. The primers were as follows: Rcan_centr_CL5_for: 5′- GCAAGCGCATAATTTAACC-3′ and Rcan-centr_CL5_rev: 5′-CAATCAAAAATATCCCCCC-3′. The PCR product was purified and cloned into the pDrive vector (Qiagen) and sequenced by the Sanger dideoxy method using the SP6 primer (Micosynth). Clone 11 was submitted to GenBank (PV030978). The inserts of plasmids were directly labelled in a nick translation reaction with Cy5 d-UTP or Cy3 d-UTP fluorochromes (Jena Bioscience) and used for FISH. To detect the 5S and 35S rDNA loci, the full-length 18S rRNA gene from tomato (GenBank: X51576.1) and the Pta71 clone from Triticum aestivum were used to detect the 35S rDNA region, while a 5S rDNA unit (B variant) from R. canina35 and the D2 clone from Lotus japonicus were used to detect the 5S rDNA locus. rDNA robes were directly labelled by Nick translation using Cy5 d-UTP (Jena Bioscience). The slides were prepared in accordance with the protocols described previously35,139. In brief, the slides were treated with pepsin solution (1 mg ml−1 diluted in 0.01 N HCl) for 30 min at 37 °C in a humid chamber, washed with 2× SSC (saline sodium citrate, pH 7.0) solution, post-fixed with 4% paraformaldehyde for 10 min at room temperature, washed again with 2× SSC and then dried in 70% and 100% ethanol. After air drying for at least 30 min, the slides were denatured with hybridization mix (50% formamide, 2× SSC, 10% dextran sulfate and ~50 ng of each probe (15 μl per slide)) for 5 min at 75 °C and then incubated for at least 18 h at 37 °C. After hybridization, stringency washes were performed with 2× and 0.1× SSC solutions at 42 °C, achieving around 76% stringency. The slides were then washed at room temperature with 2 × SSC solution and mounted with DAPI in the antifade mounting medium Vectashield (Vector Laboratories). Five mature and well open flowers were collected from the plant in the greenhouse. They were shaken above a microscope slide and their pollen was released on top of the slide. Then, 20 µl of Alexander staining solution (Morphisto, 13441.00250) was added and briefly mixed with the pollen by stirring with the pipette tip. A coverslip 24 × 40 mm was put on top of the mix. Pictures were taken with a Labscope microscope by Zeiss, using 10 × magnification. Five snapshots were counted with the help of the ZEN software. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All raw sequencing data (HiFi, Hi-C, RNA, CENH3-ChIP, DNA methylation, SCO of pollens) and genome assembly of R. canina S27 isolate are available under NCBI BioProject: PRJNA1111045. The chloroplast genome of R. canina S27 isolate available under GeneBank accession number PV550499. Raw sequencing data of R. canina DToL and R. agrestis DToL are available from Darwin Tree of Life (DToL) data portal (https://portal.darwintreeoflife.org/). The corresponding NCBI BioProject accession numbers are PRJEB79802 and PRJEB79880, respectively. Genome assemblies, the sample-specific SCO reference sequences, variant calling format files, annotations and alignments presented in this work are also available for download at Dryad140 (https://doi.org/10.5061/dryad.cc2fqz6fh). The REXdb database Viridiplantae v.3.0 (http://repeatexplorer.org/?page_id=918) is publicly available. All other data needed to evaluate the conclusions in the paper are provided in the Article and its Supplementary Information. Bomblies, K. Learning to tango with four (or more): the molecular basis of adaptation to polyploid meiosis. & Harrison, J. H. W. The status of the British rose forms as determined by their cytological behaviour. Täckholm, G. Zytologische studien über die gattung Rosa. Van de Peer, Y., Maere, S. & Meyer, A. The evolutionary significance of ancient genome duplications. Mercier, R., Mezard, C., Jenczewski, E., Macaisne, N. & Grelon, M. The molecular biology of meiosis in plants. & Copenhaver, G. P. in Annual Review of Plant Biology Vol. Pelé, A., Rousseau-Gueutin, M. & Chevre, A. M. Speciation success of polyploid plants closely relates to the regulation of meiotic recombination. & Klimešová, J. Polyploid species rely on vegetative reproduction more than diploids: a re-examination of the old hypothesis. Deb, S. K., Edger, P. P., Pires, J. C. & McKain, M. R. Patterns, mechanisms, and consequences of homoeologous exchange in allopolyploid angiosperms: a genomic and epigenomic perspective. Moore, G. Meiosis in allopolyploids—the importance of ‘Teflon' chromosomes. Debray, K. et al. Unveiling the patterns of reticulated evolutionary processes with phylogenomics: hybridization and polyploidy in the genus Rosa. Fougere-Danezan, M., Joly, S., Bruneau, A., Gao, X. F. & Zhang, L. B. Phylogeny and biogeography of wild roses with specific attention to polyploids. The Rosa genome provides new insights into the domestication of modern roses. Saint-Oyant, L. H. et al. A high-quality genome sequence of Rosa chinensis to elucidate ornamental traits. Zhang, X. et al. Haplotype-resolved genome assembly of the diploid Rosa chinensis provides insight into the mechanisms underlying key ornamental traits. Chen, F. et al. A chromosome-level genome assembly of rugged rose (Rosa rugosa) provides insights into its evolution, ecology, and floral characteristics. Partial preferential chromosome pairing is genotype dependent in tetraploid rose. Zhang, Z. et al. Haplotype-resolved genome assembly and resequencing provide insights into the origin and breeding of modern rose. Asymmetrical canina meiosis is accompanied by the expansion of a pericentromeric satellite in non-recombining univalent chromosomes. Evolutionary implications of permanent odd polyploidy in the stable sexual, pentaploid of Rosa canina L. Heredity 94, 501–506 (2005). Blackburn, K. B. Chromosomes and classification in the genus Rosa. Nybom, H., Esselink, G. D., Werlemark, G., Leus, L. & Vosman, B. Unique genomic configuration revealed by microsatellite DNA in polyploid dogroses, Rosa sect. Nybom, H., Esselink, G. D., Werlemark, G. & Vosman, B. Microsatellite DNA marker inheritance indicates preferential pairing between two highly homologous genomes in polyploid and hemisexual dog-roses, Rosa L. sect. Ritz, C. M. & Wissemann, V. Microsatellite analyses of artificial and spontaneous dogrose hybrids reveal the hybridogenic origin of Rosa micrantha by the contribution of unreduced gametes. Ritz, C. M., Köhnen, I., Groth, M., Theißen, G. & Wissemann, V. To be or not to be the odd one out—allele-specific transcription in pentaploid dogroses (Rosa L. sect. Ritz, C. M., Schmuths, H. & Wissemann, V. Evolution by reticulation: European dogroses originated by multiple hybridization across the genus Rosa. Wissemann, V. Molecular evidence for allopolyploid origin in the Rosa canina-complex (Rosaceae, Rosoideae). Ballmer, D. Dogrose Evolution and its Implications for Conservation. Bruneau, A., Starr, J. R. & Joly, S. Phylogenetic relationships in the genus Rosa: new evidence from chloroplast DNA sequences and an appraisal of current knowledge. Debray, K. et al. Unveiling the patterns of reticulated evolutionary processes with phylogenomics: hybridization and polyploidy in the genus Rosa. Wissemann, V. & Ritz, C. M. The genus Rosa (Rosoideae, Rosaceae) revisited: molecular analysis of nrITS-1 and atpB-rbcL intergenic spacer (IGS) versus conventional taxonomy. The fate of ribosomal RNA genes in spontaneous polyploid dogrose hybrids (Rosa L. sect. Ancient origin of two 5S rDNA families dominating in the genus Rosa and their behaviour in the canina-type meiosis. Henikoff, S., Ahmad, K. & Malik, H. S. The centromere paradox: stable inheritance with rapidly evolving DNA. Henikoff, S. & Malik, H. S. Centromeres: selfish drivers. Kolarčik, V., Zozomová-Lihová, J., Ducár, E. & Martonfi, P. Evolutionary significance of hybridization in Onosma (Boraginaceae): analyses of stabilized hemisexual odd polyploids and recent sterile hybrids. Dedukh, D. et al. A cyclical switch of gametogenic pathways in hybrids depends on the ploidy level. Maintenance of pure hybridogenetic water frog populations: genotypic variability in progeny of diploid and triploid parents. Jia, K. H. et al. SubPhaser: a robust allopolyploid subgenome phasing method based on subgenome-specific k-mers. Kolarčik, V., Kocova, V. & Vaskova, D. Flow cytometric seed screen data are consistent with models of chromosome inheritance in asymmetrically compensating allopolyploids. Wissemann, V. & Hellwig, F. H. Reproduction and hybridisation in the genus Rosa, section Caninae (Ser) Rehd. Reichel, K. et al. Untangling the hedge: genetic diversity in clonally and sexually transmitted genomes of European wild roses, Rosa L. PLoS ONE 18, e0292634 (2023). Herklotz, V. & Ritz, C. M. Multiple and asymmetrical origin of polyploid dog rose hybrids (Rosa L. sect. Cycles of satellite and transposon evolution in Arabidopsis centromeres. Oliveira, L. et al. KNL1 and NDC80 represent new universal markers for the detection of functional centromeres in plants. Blackhurst, H. T. Cytogenetic studies on Rosa rubiginosa and its hybrids. Malik, H. S. & Henikoff, S. Major evolutionary transitions in centromere complexity. Fishman, L. & Saunders, A. Centromere-associated female meiotic drive entails male fitness costs in monkeyflowers. Turning meiosis into mitosis. Behavior of centromeres in univalents and centric misdivision in wheat. Wulff, H. D. Über das spontane Auftreten einer Canina-Meiosis bei der Mikrosporogenese der diploiden Rosa ruga Lindl. Cheng, H. Y., Concepcion, G. T., Feng, X. W., Zhang, H. W. & Li, H. Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm. Manni, M., Berkeley, M. R., Seppey, M., Simao, F. A. & Zdobnov, E. M. BUSCO update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. Rhie, A., Walenz, B. P., Koren, S. & Phillippy, A. M. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. Herklotz, V. & Ritz, C. M. Spontane Hybridisierung von Hundsrosen (Rosa L. sect. an einem natürlichen Vorkommen in der Oberlausitz (Sachsen, Deutschland). Marcais, G. & Kingsford, C. A fast, lock-free approach for efficient parallel counting of occurrences of k-mers. Ranallo-Benavidez, T. R., Jaron, K. S. & Schatz, M. C. GenomeScope 2.0 and Smudgeplot for reference-free profiling of polyploid genomes. Jin, J. J. et al. GetOrganelle: a fast and versatile toolkit for accurate de novo assembly of organelle genomes. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Bankevich, A. et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. Camacho, C. et al. BLAST+: architecture and applications. Tillich, M. et al. GeSeq—versatile and accurate annotation of organelle genomes. Chan, P. P., Lin, B. Y., Mak, A. J. & Lowe, T. M. tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes. & Castleden, I. Chloë v0.1.0 (GitHub); https://github.com/ian-small/chloe (2020). Shen, W., Le, S., Li, Y. & Hu, F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Nguyen, L. T., Schmidt, H. A., von Haeseler, A. & Minh, B. Q. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Soderlund, C., Bomhoff, M. & Nelson, W. SyMAP v3.4: a turnkey synteny system with application to plant genomes. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer, 2016). & Schneeberger, K. SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies. Li, H. Minimap2: pairwise alignment for nucleotide sequences. Li, H. New strategies to improve minimap2 alignment accuracy. The high-quality genome of diploid strawberry (Fragaria nilgerrensis) provides new insights into anthocyanin accumulation. Lovell, J. T. et al. GENESPACE tracks regions of interest and gene copy number variation across multiple genomes. Wang, Y. P. et al. MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity. Emms, D. M. & Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. Lyons, E. & Freeling, M. How to usefully compare homologous plant genes and chromosomes as DNA sequences. Lyons, E. et al. Finding and comparing syntenic regions among Arabidopsis and the outgroups papaya, poplar, and grape: CoGe with Rosids. Kielbasa, S. M., Wan, R., Sato, K., Horton, P. & Frith, M. C. Adaptive seeds tame genomic sequence comparison. Joyce, B. L. et al. FractBias: a graphical tool for assessing fractionation bias following polyploidy. & Bleasby, A. EMBOSS: the European Molecular Biology Open Software Suite. The genome of woodland strawberry (Fragaria vesca). Suyama, M., Torrents, D. & Bork, P. PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments. Yang, Z. H. PAML: a program package for phylogenetic analysis by maximum likelihood. Yang, Z. H. & Nielsen, R. Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2024). A., Ovenden, L., Carter, L. D. & Fouch, T. Stratigraphy, fossils, and age of sediments at the upper pit of the Lost Chicken gold mine: new information on the late Pliocene environment of east central Alaska. Matthews, J. V. & Ovenden, L. E. Late Tertiary plant macrofossils from localities in Arctic/Subarctic North America: a review of the data. & Perna, N. T. progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement. Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Ellinghaus, D., Kurtz, S. & Willhoeft, U. LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons. & Wang, H. LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons. Zhang, R. G. et al. TEsorter: an accurate and fast method to classify LTR-retrotransposons in plant genomes. & Quesneville, H. LTR retrotransposons in rice (Oryza sativa, L.): recent burst amplifications followed by rapid DNA loss. Cuizhi, G. & Robertson, K. in Flora of China Vol. 9 (eds Wu, Z. Y. et al.) 339–381 (Science Press and Missouri Botanical Garden Press, 2003). Henker, H. in Gustav Hegi—Illustrierte Flora von Mitteleuropa Vol. IV/2 C (eds H. J. Conert et al.) 1–108 (Parey, 2000). & Bruneau, A. in Flora of North America North of Mexico Vol. Kron, P. & Husband, B. C. Using flow cytometry to estimate pollen DNA content: improved methodology and applications. Rapid flow cytometric analysis of the cell cycle in intact plant tissues. & Doyle, J. L. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Identification and assessment of variable single-copy orthologous (SCO) nuclear loci for low-level phylogenomics: a case study in the genus Rosa (Rosaceae). Deorowicz, S., Kokot, M., Grabowski, S. & Debudaj-Grabysz, A. KMC 2: fast and resource-frugal k-mer counting. Kokot, M., Dlugosz, M. & Deorowicz, S. KMC 3: counting and manipulating k-mer statistics. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Evaluating allopolyploid origins in strawberries (Fragaria) using haplotypes generated from target capture sequencing. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Van der Auwera, G. & O'Connor, B. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at bioRxiv https://doi.org/10.1101/201178 (2017). Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. & Hu, F. Q. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. Price, R. J. et al. Chromosome-scale genome sequence assemblies of the ‘Autumn Bliss' and ‘Malling Jewel' cultivars of the highly heterozygous red raspberry Rubus idaeus L.) derived from long-read Oxford Nanopore sequence data. Reply to: Revisiting the origin of octoploid strawberry. Kumar, S., Stecher, G., Li, M., Knyaz, C. & Tamura, K. MEGA X: molecular evolutionary genetics analysis across computing platforms. Sweeten, A. P., Schatz, M. C. & Phillippy, A. M. ModDotPlot—rapid and interactive visualization of tandem repeats. Stiehler, F. et al. Helixer: cross-species gene annotation of large eukaryotic genomes using deep learning. Novák, P., Neumann, P. & Macas, J. Global analysis of repetitive DNA from unassembled sequence reads using RepeatExplorer2. Novák, P., Hoštáková, N., Neumann, P. & Macas, J. DANTE and DANTE_LTR: lineage-centric annotation pipelines for long terminal repeat retrotransposons in plant genomes. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Bolger, M., Schwacke, R. & Usadel, B. MapMan visualization of RNA-Seq data using Mercator4 functional annotations. Putri, G. H., Anders, S., Pyl, P. T., Pimanda, J. E. & Zanini, F. Analysing high-throughput sequencing data in Python with HTSeq 2.0. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. & Turck, F. Genome-wide mapping of protein-DNA interaction by chromatin immunoprecipitation and DNA microarray hybridization (ChIP-chip). Part A: ChIP-chip molecular methods. Ramirez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Lopez-Delisle, L. et al. pyGenomeTracks: reproducible plots for multivariate genomic datasets. Hadfield, J. D. MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Pedersen, T. L. patchwork: the composer of plots. Wickham, H., Vaughan, D., Girlich, M. & Ushey, K. tidyr: tidy messy data. Wickham, H., François, R., Henry, L., Müller, K. & Vaughan, D. dplyr: a grammar of data manipulation. Sievers, F. et al. Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Hurel, A. et al. A cytological approach to studying meiotic recombination and chromosome dynamics in Arabidopsis thaliana male meiocytes in three dimensions. High rates of structural rearrangements have shaped the chromosome evolution in dysploid beans. Herklotz, V. et al. Three haplotype-resolved pentaploid Rosa assemblies with assembled and extracted single copy orthologue (SCO) sequences from Rosa canina genome, diploid Rosa species, and sect. The authors thank A. Houben, H. Bruelheide, B. Usadel, M. Thines, M. Bagdevi and K. Wesche for discussions on the research; B. Schlitt, M. Schwager, S. H. Sourav, J. Wesenberg, A. Smolka and S. Stegmann for their technical assistance in the laboratory; colleagues F. Erdogan, I. Kisakesen and U. Abraham for their help with target enrichment; T. Schell, C. Sinai, C. Greve, L. Schardt, D. Baranski and A. Ben Hamadou for technical support; J. Bauernfeind for help with IT facilities; T. Hawel, G. Schulz, S. Arndt, G. Vogg, S. Ruge and Oberlausitz-Stiftung for providing access to their collections and plant material; M. Simon for providing rose photos; the staff at the Darwin Tree of Life Project at the Wellcome Sanger Institute for making the data of the additional accession of R. canina and R. agrestis available (https://www.darwintreeoflife.org/project-resources); and the staff at the Institute of Biophysics AS CR for internal support. This study was funded by the German Research Foundation (DFG Ri 2090/4-1, MA 9363/2-1 and MA 9363/3-1), the Czech Science Foundation (GAČR 22-16826S), the Max Planck Society (core funding to A.M.), and the European Union (European Research Council Starting Grant, HoloRECOMB, grant no. The DFG also funded this work under Germany's Excellence Strategy—EXC 493 2048/1–390686111 (to A.M.). is financially supported by the DFG (grant no. Open access funding provided by Max Planck Society. These authors contributed equally: V. Herklotz, M. Zhang, T. Nascimento Senckenberg Museum for Natural History Görlitz, Senckenberg–Leibniz Institution for Biodiversity and Earth System Research, Görlitz, Germany Department of Chromosome Biology, Max Planck Institute for Plant Breeding Research, Cologne, Germany Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, Seeland, Germany Max Planck Genome Centre Cologne, Max Planck Institute for Plant Breeding Research, Cologne, Germany Cluster of Excellence on Plant Sciences (CEPLAS), Heinrich-Heine University, Düsseldorf, Germany International Institute (IHI) Zittau, TUD Dresden University of Technology, Zittau, Germany German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and performed the SCO experiments and processed the data. D.H. performed the plastome assembly and annotation. performed all nuclear sequencing libraries. performed the chromosome preparations, immunodetection and FISH. performed the pollen viability assay. created and curated the synthetic hybrids. Correspondence to A. Kovařík, A. Marques or C. M. Ritz. The authors declare no competing interests. Nature thanks Ian Henderson, Damon Lisch and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Hi-C map of 35 chromosome-level scaffolds. Each synteny group contains five chromosomes, which were ordered as S1_h1, S1_h2, S2, R3, and R4 in sequence. The vanished Hi-C signals of the first two chromosomes in each linkage group suggest the high homozygosity of S1_h1 and S1_h2 haplotypes. (b) GenomeScope2 21-mer distribution from HiFi sequences and genome size estimation confirmed the expected R. canina genome size, i.e., 5× 490 Mbp. The 4x peak at coverage 94 is less invisible, indicating two out of five haplotypes are highly similar. (c) Ploidy and genome structure inference based on 21-mer Smudgeplot analysis. A and B represent the number of heterozygous k-mers identified. Note the high amount of pentaploid heterozygous k-mer combinations AAAAB and AAABB. (d) BUSCO (Universal Single-Copy Orthologues) genes assessment results based on the annotated protein sequences of the pentaploid genome and each individual subgenome set. (e) HiFi k-mer multiplicity frequency by Merqury. (f) and (g) are Mercator4 statistics of gene contents based on its functional annotation on Rosa canina protein sequences. (f) Percentage of genes occupying a certain gene category. (g) Protein length distribution based on the deviation to category-specific reference lengths. Each bar represents the number of proteins having a certain length difference to the reference length of the corresponding Mercator4 category. The overall small deviation of gene lengths in R. canina comparing to Mercator reference genes indicate most annotated genes in R. canina are complete. (f) and (g) show the results for the subgenome S1 haplotype h1 only since the other three subgenomes show highly similar distribution. The larger the Z score is, the greater the relative abundance of a k-mer), and (b) Principal Component Analysis (PCA) of differential 15-mers confirmed that the genome was successfully phased into four subgenomes based on clearly distinct patterns of both differential k-mers and homoeologous chromosomes. From the outer to inner circles (1–9): (1) subgenome assignments based on the k-means algorithm; (2) significant enrichment of subgenome-specific k-mers − the same colour as the subgenome indicates significant enrichment for those subgenome-specific k-mers − white areas are not significantly enriched; (3) normalized proportion (relative) of subgenome-specific k-mers; (4–7) count (absolute) of each subgenome-specific k-mer set; (8) density of long terminal repeat retrotransposons (LTR-RTs) − if the colour is consistent with the subgenome, it indicates that LTR-RTs are significantly enriched in those subgenome-specific k-mers; grey indicates nonspecific LTR-RTs; and (9) homoeologous blocks. All statistics (2–7) are computed in sliding windows of 1 Mb. Notably, the fraction of subgenome differentiation between the S1 and S2 subgenomes was restricted to proximal regions, likely representing their different centromere compositions. Whereas R3 and R4 subgenomes are clearly distinct from each other in their specific k-mer spectrum. (a) Synteny and rearrangement analyses (SyRI) between our and the R. canina genome assembly from DToL. (b) Synteny and rearrangement analyses (SyRI) between our R. canina and the R. agrestis genome assembly from DToL. Pairwise comparisons of synteny of all R. canina subgenomes (S1_h1/S1_h2, S2, R3, and R4) and all R. agrestis subgenomes (S1, S2, R3, and R4_h1/R4_h2) juxtaposed against the corresponding chromosomes of R. chinensis (sect. Note that only synteny blocks and rearrangement blocks greater than 50 kb in length are shown here. (c) GENESPACE synteny and phylogenetic relationships of the five chromosome sets of our R. canina assembly, R. agrestis from DToL and their close relatives R. chinensis and R. rugosa. Chromosomes are normalized by number of genes. (d–e) Unrooted Maximum Likelihood phylogenies of the homoeologous R. canina and R. agrestis chromosomes and chromosomes from the respective syntenic groups of the diploid species R. chinensis and R. rugosa based on multiple alignments of whole chromosome sequences. The upper panel (d) exemplarily depicts the phylogeny of chromosomes of synteny group 1. Synteny groups 2 − 7 are presented in (e). Filled chromosomes refer to subgenomes of R. canina belonging to the Synstylae clade (violet/light blue) and Rosa clade (dark/light orange). Chromosomes from R. agrestis are marked with hatched white filling, while the diploid roses are marked with black hatched filling. The 95% confidence intervals (CIs) are marked in the upper right corner to predict the insertion time boundaries of LTR-RTs on the subgenome for R. canina (a), R. canina DToL (b) and R. agrestis DToL (c). (a) Genome-wide distribution of pollen SCOs from pollen of eigth dogrose species (subsect. Caninae: three samples of R. canina, two samples of R. corymbifera; subsect. Rubigineae three samples of R. rubiginosa) from two subsections in the R. agrestis genome assembly from DToL. The bubble map represents chromosomal hits from the SAM file output, which were selectively filtered to display loci with a single alternative hit. The size of the symbols corresponds to the mean count of pollen SCOs mapped to each chromosomal pair, identifying seven bivalent chromosome pairs within the R. agrestis (subsect. Rubigineae) genome assembly and seven different pairs in which R. rubiginosa (subsect. (b) Plastome phylogeny of the genus Rosa. Maximum Likelihood phylogeny of the genus Rosa retrieved from plastome sequences obtained in this study and those available in Genbank (Supplementary Table 4). All nodes which were not supported by 100% bootstrap are indicated by dashed lines. (c) Representative histogram of flow cytometric measurements of nuclei isolated from nutlets of the sexual diploid R. rugosa to determine the endosperm/embryo ratios. The observed endosperm/embryo ratio 1.5, fits to the expectation of 3x endosperm and a 2x embryo originated by sexual reproduction including endosperm fertilization. Please note that the gain of the fluorescence was adjusted to position the embryo peak at channel 100. For a better visualization of the endosperm peak the scale of the Y-axis was manually adjusted to 200 counts (red rectangles). (d) The parental SNP ratios detected in each subgenome of the hybrid R. canina × R. rubiginosa (Sample ID: D62b_2). SNPs were called by aligning short reads to our assembled R. canina (S27) S1_h1. The y-axis displays the proportion of SNPs in each subgenome. For instance, ~62% SNPs detected in the alignments to S1 subgenome are the same as the maternal R. canina (Sample ID: D3b_2), while only around 5% SNPs are from the paternal R. rubiginosa (Sample ID: D145_b2), indicating the S1 subgenome of the hybrid was supposed to be contributed by the maternal parent. Overall, the hybrid subgenome S1, S2, and R3 are all from maternal parent R. canina, while one R4 subgenome is from maternal R. canina and the other R4 is contributed by paternal R. rubiginosa. (e) Assessment of the parental genomes contribution of a synthetic hybrid between R. rubiginosa (female donor) and R. corymbifera (male donor), confirming the sexual reproduction and subgenome's inheritance trough male and female meiosis. Note, that R. corymbifera belongs to subsect. Caninae and is a very close relative of R. canina and so it has two copies of S1 subgenome. (f) The parental SNP ratios detected in each subgenome of the hybrid R. rubiginosa × R. corymbifera (Sample ID: D166b_2). (a) Structural visualization of the chromosome-wide (left) and centromere (right) close up. (b) Close up visualization of the cenLTR array and insertion of an ATHILA inside the cenLTR array. (c) Alignment of the cenLTR consensus sequence with the LTR sequence from the inserted ATHILA element showing over 75% sequence similarity. CENH3 enrichment marked in dark red was calculated by CENH3 ChIP-seq (log2 CENH3/H3) signal normalized by coverage. DNA methylations have three contexts with CpG marked in black, CHG maked in dark grey, and CHH marked in light grey. ATHILA (yolk yellow) and CANR4 (magenta) were presented by their density. All signals were calculated in 50 kb adjacent windows. All chromosome coordinates were scaled based on their distance to centromere against the distance of centromere (CEN) to telomere (TEL). Centromere position (CEN) was defined by where maximum CENH3 enrichment was located. All signal values (y-axis) were scaled from 0 to 1 by global minimum to global maximum except for DNA methylations, which retained the original percentage values. Please note the higher CENH3 association with CANR4-based centromeres compared with ATHILA-based centromeres. (a–b) Immunodetection of the KNL1 protein (in red) at all centromeres of R. canina revealed a differential distribution, with the presence of large and small centromeres, as shown in more detail in the upper and lower insets (a). (c–d) Distribution of the centromeric satellite repeats CANR4 (in magenta) and 5S rDNA (in cyan) in the R. canina miotic chromosomes, while the CANR4 repeat is present at the centromeres of a set of chromosomes. The 5S rDNA is located at the pericentromeric regions of chr1, insets in c and d highlight the pericentromeric location of 5S loci (right) and a weak CANR4 signal (left). (e–h) Distribution of the CANR4 repeat in meiotic diakinesis, revealing its distribution mostly in univalents according (e and f), while the pattern of 5S and 35S rDNA (g and h) (cyan and yellow, respectively) hybridized to three sites, two of which were located near 35S sites on a bivalent (inset in h) corresponding to the Rca3_S1_h1/h2 chromosomes as found in our assembly annotation. Week signals are highlighted in the insets present in g and h. (i–m) FISH with CANR4 and ATHILA on a diakinesis of R. canina (i–k) and mitotic cells of R. chinensis (l) and R. rugosa (m). Asterisks in i–k point to bivalent-forming chromosomes. The experiments for this set of data were repeated at least 10 times and independently presenting similar results. (a) A single polyad showing at least two nuclei with seven CENH3 signals (arrowheads), while several other nuclei show irregular number of centromeric foci. (b–d) Different Z-stacks of three polyads showing few nuclei with seven centromeric foci, while several other smaller nuclei are formed with irregular number of centromeres. Immunostaining experiments were repeated independently at least ten times outputting similar results. R. canina (S27) mitotic metaphase immunolabelled with antibodies against CENH3. Note the size difference of CENH3 signals among different chromosomes (Fig. STED-imaged early prophase I. CENH3 immunostaining during early prophase I of R. canina (S27) (Fig. STED-imaged early prophase I. CENH3 immunostaining during early prophase I of R. canina (S27) (Fig. CENH3 immunostaining in diakinesis of R. canina (S27) (Fig. STED-imaged metaphase I. CENH3 and alpha-tubulin immunostaining during metaphase I of R. canina (S27) (Fig. STED-imaged early anaphase I. CENH3 and alpha-tubulin immunostaining during anaphase I of R. canina (S27) (Fig. STED-imaged late anaphase I. CENH3 and alpha-tubulin immunostaining during anaphase I of R. canina (S27) (Fig. CENH3 and alpha-tubulin immunostaining during metaphase II of R. canina (S27) (Fig. CENH3 and alpha-tubulin immunostaining during anaphase II of R. canina (S27) (Fig. CENH3 immunostaining in a polyad of R. canina (S27) showing different nuclei with varying number of centromeric foci. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Herklotz, V., Zhang, M., Nascimento, T. et al. Bimodal centromeres in pentaploid dogroses shed light on their unique meiosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09135-3'>Bogong moths use a stellar compass for long-distance navigation at night</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 15:50:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Each spring, billions of Bogong moths escape hot conditions across southeast Australia by migrating up to 1,000 km to a place that they have never previously visited—a limited number of cool caves in the Australian Alps, historically used for aestivating over summer1,2. At the beginning of autumn, the same individuals make a return migration to their breeding grounds to reproduce and die. Here we show that Bogong moths use the starry night sky as a compass to distinguish between specific geographical directions, thereby navigating in their inherited migratory direction towards their distant goal. By tethering spring and autumn migratory moths in a flight simulator3,4,5, we found that, under naturalistic moonless night skies and in a nulled geomagnetic field (disabling the moth's known magnetic sense4), moths flew in their seasonally appropriate migratory directions. Visual interneurons in different regions of the moth's brain responded specifically to rotations of the night sky and were tuned to a common sky orientation, firing maximally when the moth was headed southwards. Our results suggest that Bogong moths use stellar cues and the Earth's magnetic field to create a robust compass system for long-distance nocturnal navigation towards a specific destination. The Bogong moth (Agrotis infusa; Fig. 1a) is an iconic and threatened migratory insect endemic to Australia. During the austral spring, newly eclosed Bogong moths make a highly directed migration of up to 1,000 km to a geographically restricted assemblage of high alpine caves in the extreme southeast of the country (Fig. 1b), a place that they have never previously visited. After 3–4 months of dormancy (aestivation; Fig. 1c), the same individuals return to their breeding grounds during autumn to reproduce, after which they die1,2. We have recently shown4 that these moths steer flight during the long-distance phase6 of their migration by sensing the Earth's magnetic field and correlating its inherent directional information with one or more visual landmarks, potentially using these landmarks as temporary orientation beacons along the route. However, exactly which natural visual features are used by Bogong moths for navigation remains unclear. A dark feature on the nocturnal horizon, one or more stars, or the moon are all possible cues. In the austral night sky, the constellations of stars and the bright Milky Way are particularly prominent and have excellent potential as reliable navigational cues7,8. However, to our knowledge, no invertebrate is currently known to use the stars for discerning specific geographical directions (that is, a direction relative to north) for directed long-range navigation to a distant goal. Here we show that moths have this ability and use the stars as a compass during the long-distance phase6 of their migration to determine and follow an inherited migratory bearing, reversing this bearing when the season changes. Moreover, we show that visual interneurons in the moth's brain are tuned to specific orientations of the night sky, suggesting their involvement in a stellar compass network. 2. b, Adult moths migrate from their breeding grounds in various regions of southeast Australia to the Australian Alps during spring (green arrows), where they aestivate in cool alpine caves over summer, and return to the breeding grounds in autumn (purple arrows). At the breeding grounds, they mate, lay eggs and die. Immature stages develop underground during the winter. c, Around 16,000 moths per m2 aestivate on the walls of specific caves in the Australian Alps for up to 4 months before making the return migration. Inset: close-up image of the moths. Using a light trap located at Mount Selwyn, about 70 km north-northeast of the main alpine aestivation sites in southeast New South Wales, we captured migrating Bogong moths departing northwards during their autumn return migration (Fig. Captured moths were transferred to our nearby experimental site and tethered and flown within a modified Mouritsen–Frost flight simulator3,4,5 that continuously measured the instantaneous orientations of vigorously flying moths that were free to turn in any azimuthal direction (details of the simulator, the tethering and the criteria used to select moths for analysis are provided in the Methods and Extended Data Fig. To test whether Bogong moths fly in their inherited migratory directions under natural night skies, two transparent ultraviolet (UV)-transmissive non-magnetic cylindrical flight simulator arenas were placed on an open hilltop (Methods and Extended Data Fig. They also experienced the natural geomagnetic field. Such movements substantially shift and rotate the stripe of the Milky Way and the positions of the stars as the night progresses, therefore creating a potential problem for migrating animals that rely on them to navigate a straight course. Early during the night (20:49 ± 17 min), under a cloudless autumn starry sky with a waxing gibbous half-full moon, moths flown in the two arenas were significantly oriented northward, in their expected migratory direction (Fig. When tested 3 h later (23:42 ± 17 min), when the stars and the moon had shifted substantially (and the sky was still cloudless), the same cohort of moths (which had been swapped between arenas to eliminate landmarks as possible cues) retained their northward orientation (Fig. 2b; n = 95, α = 355°), with their earlier and later mean orientation directions being indistinguishable (likelihood-ratio test on population means for all 95 moths, early versus late: χ2 = 2.663, d.f. If one exclusively considers moths that were highly oriented both earlier and later in the evening (r > 0.8 in both trials: 35 out of 95 moths), this finding is only reinforced (Extended Data Fig. Thus, Bogong moths not only orient in their expected migratory direction under natural night skies, but they also do so despite the nightly movements of the stars and moon. a,b, The orientation directions of 95 migratory moths (blue vectors) under a cloudless autumn night sky (Adaminaby, over 3 nights at the end of March 2023), early in the evening (a; 20:49 ± 17 min, α = 5°, 95% confidence interval = 22°, R* = 3.981, P = 4.23 × 10−20) and again 3 h later when the half-moon and stars had substantially shifted their positions (b; 23:42 ± 17 min, α = 355°, 95% confidence interval = 42°, R* = 2.677, P = 5.76 × 10−9). Moths were tethered in a clear cylindrical UV-transmissive Perspex arena with a full view of the sky and the surrounding landscape. c, The orientation directions of 44 migratory moths (blue vectors) under a completely overcast autumn night sky (Adaminaby, end of March 2023) at 21.30 ± 18 min (α = 13°, 95% confidence interval = 30°, R* = 2.828, P = 7.56 × 10−10). Each red mean vector (MV) in a–c results from weighting the mean directions and mean directedness (vector lengths) of all of the individual moths. Statistical analysis was performed using one-sided Moore's modified Rayleigh test with Bonferroni correction for multiple comparisons; corrected P = 4.23 × 10−20 (a), 5.76 × 10−9 (b) and 7.56 × 10−10 (c) (Methods). The direction and directedness of mean vectors is given by α and R*, respectively. Sky images in a and b, Stellarium. The dim pattern of polarized light18 associated with the moon is also a potential compass cue. Lunar polarized light is used by nocturnal dung beetles for straight-line orientation19 but, when the moon is absent, they instead revert to the Milky Way, and are disoriented when the sky is obscured by clouds9. However, substantial migrations of Bogong moths also occur around the new moon (E.W. and B.F., unpublished data), implying that, if polarized light is used as a compass cue, it is not essential. Notably, on nights on which the moon and stars were obscured by heavy cloud, Bogong moths were still oriented in a northward direction (Fig. 2c; n = 44, α = 13°), with a mean orientation direction that was not significantly different to either of the two mean directions (Fig. 2a,b) recorded on clear nights (likelihood-ratio tests comparing mean orientation direction in Fig. Thus, even when the stars and moon are unavailable, Bogong moths can still navigate in their inherited migratory direction, strongly suggesting reliance on the only known remaining compass cue—the Earth's magnetic field. The most parsimonious explanation for these results is that, in overcast conditions, Bogong moths rely on the geomagnetic field as their compass for navigation, which raises the critical question of whether Bogong moths use a stellar compass at all. To investigate whether Bogong moths use a stellar compass, we captured Bogong moths at Mount Selwyn that were migrating roughly southwards into the Australian Alps in spring and back northward during autumn (Fig. Moths were transferred to a purpose-built dark ferromagnetic-free laboratory at our field site and tethered and flown in a flight simulator (details of the laboratory are provided in the Methods and Extended Data Fig. Moths were subjected to a natural clear moonless austral night sky (matching the date and time of the experiment) projected at natural intensity onto a flat screen placed on top of the dark-walled arena (Fig. 3a) and their flights were recorded for 5 min. a,b, Images of a laboratory-projected natural night sky during spring and autumn (a), and an autumn sky with its stars randomly arranged (b). c, The orientation directions of migratory moths under naturally oriented night skies, and in a nulled magnetic field, during spring 2018 and 2019 (green vectors, left; n = 70, α = 168°, 95% confidence interval = 58°, R* = 2.014, P = 6.3 × 10−5) and autumn (purple vectors, right; n = 54, α = 341°, 95% confidence interval = 41°, R* = 2.512, P = 9.0 × 10−9). d, As for c, but for night skies rotated by 180° (spring: n = 68, α = 355°, 95% confidence interval = 53°, R* = 2.085, P = 2.7 × 10−5; autumn: n = 56, α = 167°, 95% confidence interval = 69°, R* = 1.716, P = 1.8 × 10−3). e,f, The orientation directions of migratory moths under randomized starry skies during spring (e; n = 58, R* ≈ 0.598, P = 0.35) and autumn (f; n = 49, R* ≈ 0.016, P = 0.91) in 2018 and 2019. All other definitions and statistics are as described in Fig. Bonferroni correction for multiple comparisons was used; corrected P = 6.3 × 10−5 (spring, c) and 9.0 × 10−9 (autumn, c), 2.7 × 10−5 (spring, d) and 1.8 × 10−3 (autumn, d). Under naturally oriented night skies (geographical north 0°; Fig. 3a) and in the absence of a magnetic field, moths flew in their seasonally appropriate migratory directions4 during two springs and two autumns in 2018 and 2019 (Fig. 3c): roughly southwards in spring (n = 70, mean vector direction α = 168°) and north-northwest in autumn (n = 54, α = 341°). Under the natural sky stimulus rotated by 180° (arena-centric geographical north now at 180°), the moths flew in almost exactly the opposite direction (Fig. 3d), turning by 187° in spring (n = 68, α = 355°) and by 174° in autumn (n = 56, α = 167°). When the stars of the natural sky were randomly distributed (identical light intensity, no directional information: Fig. 3b), moths were disoriented in both seasons (spring: Fig. Full statistics are provided in Extended Data Fig. Moth flight directions during spring and autumn, and under rotated and unrotated skies in either season, were significantly different (Mardia–Watson–Wheeler tests; Extended Data Fig. These results show that Bogong moths can use the distribution of stars within the austral night sky alone as a navigational compass to distinguish between specific geographical directions, thereby determining their required migratory direction, during both their southward spring migration and their northward autumn migration. When these stars were rearranged randomly, this ability disappears. Apart from humans15,16 and several night-migratory songbirds6,13,14, to our knowledge, no other animal is known to use the stars to navigate in a specific geographical direction. Birds use the centre of stellar rotation to determine north or south6,13,14 and, if Bogong moths do the same, this might explain their ability to hold an inherited migratory course despite the nightly movements of the stars (Fig. Alternatively, Bogong moths might possess a time-compensated star compass that can account for the predictable movements of the stars, analogous to the time-compensated sun compass of Monarch butterflies3. However, exactly which aspects of the starry sky are used by Bogong moths remains to be determined. In summary, our behavioural results demonstrate that Bogong moths, like night-migratory birds20, use a stellar compass and a geomagnetic compass and, like birds6,21, remain oriented in their inherited migratory direction when at least one is available, either the natural stars alone (Fig. 3c,d) or the Earth's magnetic field alone (Fig. When neither cue is available the moths are disoriented (Fig. We measured neural responses to the same stimuli that induced behavioural responses in our laboratory flight experiments (dorsal starry sky, nulled magnetic field). Restrained moths were mounted in the apparatus with an eastward orientation relative to the initial starry sky. During experiments, we rotated the sky through 360° from this starting point. Impaled neurons (n = 28) revealed a peak spiking frequency at a specific sky rotation angle φmax (that is, at a specific compass direction of the moth; Fig. Their responses fell into four physiological categories: (1) unimodally excited; (2) unimodally inhibited; (3) rotation-direction-selective unimodal; or (4) rotation-direction-selective bimodal (Fig. While unimodal neurons showed a single activity peak (maximal excitation or maximal inhibition), bimodal neurons showed two peaks separated by approximately 180°. Responses of neurons in categories 1 and 2 were largely insensitive to movement direction, with clockwise and anticlockwise sky rotations leading to similar responses. Despite the diversity of response types, φmax was consistent across repetitions, with the majority of cells having a tuning variability of 20° or less (Fig. 3b) were rotated as a control, cells failed to show tuning (Figs. As the control shares local contrast and motion cues, its inability to drive the recorded neurons suggests that these neurons are not simply movement detectors but are tuned to features present in the natural starry sky. a, Examples of general response categories (1–4) for 360° clockwise (CW; grey curves) and anticlockwise (CCW; purple curves) night sky stimulus rotations (mean ± s.d. tuning curves show spike impulses (imp) per s). In the moth's reference frame, flying under a natural (stationary) sky, 0° = 360° indicates that it is heading north; 90° indicates that it is heading east; 180° indicates that it is heading south; and 270° indicates that it is heading west. φmax is defined at the top (red dashed line). b, The φmax for all cells (note that bimodal cells have two φmax ≈ 180° apart; separate plots for all four cell categories are shown in Extended Data Fig. The red vector gives the mean φmax (α) and the vector length (r) for all 18 unimodal cells (one-sided Rayleigh test, spring and autumn pooled, P = 0.00319; 95% confidence limits (black straight lines), ±35°). The dashed circles show the required r value for statistical significance: P < 0.05 and P < 0.01, respectively for increasing radius. Unimodal cells fire maximally when moths head southwards (171° ± 35°) under natural skies (irrespective of season). c, The variability in φmax for successive clockwise (average 9°) or anticlockwise (average 10°) sky rotations in individual cells. d, φmax for dot and bar control stimuli in a subset of unimodal and bimodal cells (autumn 2020). The bottom row shows the angular difference between the cell φmax for dot/bar stimuli and its starry sky φmax (quoted ± angular limits, dashed lines). e, The response SNR for different stimuli (grey dots, included data; black dots, outliers). The SNR is the maximum response during sky rotation (that is at φ = φmax) divided by the standard error before rotation. a, Neuron morphologies for a neuron interconnecting both optic lobes and innervating the LAL of the central brain (medulla (ME) to medulla, left), a central-complex neuron innervating the fan-shaped body (FB; middle) and an LX neuron (right). b, Raster plots of spike data shown with the mean tuning curves for clockwise and anticlockwise rotating starry skies (blue/green curves) and rotating randomized stars (control, grey curves). Other conventions are as described in Fig. c, Peak tuning directions (φmax) of individual cells following clockwise and anticlockwise sky rotations (the rotation angle when cellular firing frequency was maximal, derived from spike train raster plots in b). To further test the selectivity of these responses, a subset of six cells was also stimulated with artificial compass cues (of natural intensity) that mimicked aspects of the Milky Way (Fig. 4d): its brightest region around the Carina nebula (revolving dot) and its stripe-like shape (rotating bar). All but one of the cells responded to both cues (dot, n = 6; bar, n = 5) and these qualitatively resembled their response to the starry sky, with similar azimuthal tuning (deviations from starry sky φmax were clustered around 0° across neurons; Fig. This suggests that these neurons are suited to encode at least two features of the starry sky: the extended shape of the Milky Way and its brightest region around the Carina nebula. Notably, across the entire population of unimodal cells, φmax values consistently clustered in the Southern Hemisphere, independent of recording season or response type (Fig. Thus, in the reference frame of a Bogong moth flying under a natural (stationary) sky, unimodal cells fire maximally when moths head southwards (irrespective of season: a Mardia–Watson–Wheeler test indicates that the spring and autumn mean φmax values do not significantly differ: P > 0.2). Notably, head direction cells recorded in the medial pallium of fledgling shearwater chicks show a similar behaviour—all fire maximally when the chick's head is directed towards magnetic north (even though the chick will eventually fly due south on its inaugural migration)26. In fixed quiescent Monarch butterflies, CX neurons recorded extracellularly in response to a rotating ‘sun' stimulus (a bright green LED at 30° elevation), also revealed narrow φmax tuning27. However, when allowed to fly and receive self-motion cues this tuning disappeared, revealing dependence on behavioural state27. As our Bogong moths were likewise fixed and quiescent, we cannot rule out that the narrow φmax tuning we report is also due to behavioural state. Of the 28 neurons that responded to our stimuli, we anatomically identified 11 cells by intracellular tracer injections (Fig. While the optic lobe is responsible for primary visual processing, the CX is a higher-order integration centre for spatial orientation, navigation and action selection28,29. For visually guided migration in Bogong moths, all three brain regions are therefore expected to have key roles in processing visual compass cues and transforming these signals into steering commands. While the optic lobe neurons showed sharply tuned responses with a high signal-to-noise ratio (SNR), the responses of the two CX neurons that we held long enough to record from were less pronounced (Fig. 4 and 7; cell 181122_003), indicating that visual stimulation alone is insufficient to fully drive these cells. The three anatomically identified category 4 (bimodal) cells localized to the LAL and the ventrolateral and ventromedial protocerebrum—their direction-selective responses are suggestive of a potential role in steering. The six identified category 1 and 2 (unimodal) cells localized to the optic lobe and CX, locations that are more concerned with early-stage sensory processing (optic lobe) and heading encoding (CX). Although we obtained only a sparse sample of neurons, and despite the constraints of in vivo intracellular electrophysiology, our data nonetheless demonstrate that information from the starry sky is encoded in the moth's brain. While these neurons are located in brain regions that are known to analyse multiple navigational cues in other insects24, surprisingly, our identified cell types have not previously been described in analyses of insect compass pathways24 or head-direction circuits30 (Fig. Further work is clearly required to understand these neural circuits in detail, and whether their properties vary with behavioural state. Our results indicate that a stellar compass is sufficient on its own to guide Bogong moths during their migration, and this may become useful if the geomagnetic field becomes perturbed by a magnetic storm or by local magnetic anomalies31 (which are admittedly not strong in southeast Australia, and typically alter field strength by only a few per cent). Under natural conditions, a geomagnetic compass is also sufficient if the stars become obscured by cloud. When neither cue is available, moths are disoriented. Moreover, we have also shown that Bogong moths possess previously undescribed neurons in the navigational centres of the brain that respond to a specific orientation of the night sky. Notably, these neurons are all tuned to a common azimuth, firing maximally when the moth is heading southwards, irrespective of season (Fig. Newly eclosed Bogong moths therefore rely on stellar and geomagnetic compass cues to target a distant and restricted geographical region they have never previously visited (during spring), relying on the same global cues for the reverse migration (during autumn). For Bogong moths, how these cues are interpreted to determine the desired migratory heading depends on three spatiotemporal factors: the moth's geographical origin (different breeding populations across southeast Australia migrate in different directions to reach the same target), the season (moths migrate towards the mountains in spring and return to the breeding grounds in autumn) and the time of night (stellar positions change due to Earth's rotation). As in other insects24, to move in a desired (migratory) heading direction, Bogong moths must first establish their current heading (for example, using global compass cues), compute the angular difference between their current and desired headings and, depending on geographical location, season and time of night, generate a steering command that turns the moth to its required migratory direction. While head direction is well understood to be encoded by the CX24,30,32, our understanding of how desired heading is encoded is only now emerging33,34. However, consistent with data from flies28,34,35,36,37, bees38 and Monarch butterflies33, the computation of desired migratory heading in Bogong moths could well take place in the fan-shaped body of the CX. Indeed, our identification of starry sky responses in a fan-shaped body input neuron (Fig. However, exactly how the fan-shaped body of Bogong moths accounts for geographical location, season and time of night to generate a desired migratory heading remains unclear. The question remains of what aspects of a moonless austral night sky might be useful as compass cues for Bogong moths. Humans15,16 and birds6,13,14 can use individual stars, or constellations of stars, as compass cues for distinguishing specific geographical directions, but it remains unclear how many of the brightest stars are discernible by the moth's small compound eyes. However, the Milky Way—which, in the Southern Hemisphere, is a bright extended stripe of light that becomes brightest somewhere in the southern half of the sky7,10—is very likely visible to Bogong moths39, as also suggested by our physiological results (Fig. Its shape and celestial position at any time of night are predictable from one night to the next throughout each migratory season, and its rotation around the South Celestial Pole over any single night retains the brightest parts of the Milky Way (including the Carina nebula) somewhere in the southern half of the sky, but not consistently due south. Exactly where the brightest parts are located depends on season and time of night and can vary from southeast (Figs. 2a and 3b) to almost due west (Fig. Thus, a simple phototactic mechanism, with Bogong moths orienting towards the brightest parts of the sky in spring and away from them in autumn, seems unlikely. Furthermore, the presence of a bright moon in different parts of the sky would severely disrupt such a mechanism. However, Bogong moths orient in their inherited migratory direction even when the moon is present (Fig. Notably, the Southern Hemisphere Milky Way is used to determine an arbitrary compass direction for short-range orientation in ball-rolling dung beetles7,9,10, not for distinguishing a specific geographical direction to travel towards a distant goal, but as a visual guide to keep a straight bearing for a few minutes in any random (goalless) direction. By contrast, Bogong moths, navigate long distances over many nights towards a distant destination they have never previously visited and therefore require a global compass to distinguish and hold their specific, inherited, geographical migratory heading. While the Earth's magnetic field certainly acts as one such compass4 (Fig. 2c), the southern night sky clearly acts as another, with each of these compasses probably taking over from the other when the salience of either diminishes or fails (Figs. However, exactly which celestial features are used for the stellar compass, whether the common directional tuning of stellar compass neurons has any role in the computation of desired heading, and how stellar, magnetic and any other hitherto unknown sensory cues en route are behaviourally and neurally integrated for robust navigation (including how and whether they are calibrated against each other40), all remain enticing topics for future research. Bogong moths (A. infusa) of both sexes were caught in the wild during their autumn and spring migrations (2019 and 2018) using a LepiLED insect light (www.gunnarbrehm.de), or a vertical beam search light (model GT175, Ammon Luminaire Company), placed in front of a white sheet suspended between two trees. Almost all of the animals were caught near the Mount Selwyn Snowfields (southeast New South Wales, Australia: 35.914° S, 148.444° E; elevation, 1,600 m), which is approximately 70 km north-northeast of the nearest aestivation cave in the Main Range of the New South Wales Alps. Thus, to reach these caves in spring, these moths (a tiny subset of all moths travelling to the mountains in a multitude of directions from across southeast Australia) would be expected to fly south-southwest in spring, and returning moths might be expected to travel north-northeast in autumn (which agrees with our behavioural results). A few animals were also caught near Thredbo (Dead Horse Gap, southeast New South Wales, Australia: 36.524° S, 148.260° E, elevation 1,580 m). These moths were used for electrophysiology only. Each captured moth was transferred to its own plastic container to isolate it from influence by other moths. After capture, moths were transported to the testing site Glenhare, a rural property near Adaminaby New South Wales (36.040° S, 148.864° E; elevation, 1,250 m), fed with 20% honey solution (in water) and stored in a cool and sheltered place (exposed to the natural light cycle) to recover from stress induced by capture. A purpose-built ferromagnetic-free laboratory located at Glenhare, Adaminaby (built on a concrete slab reinforced with fibreglass and constructed entirely from non-magnetic materials) housed the indoor behavioural and electrophysiological experiments (Extended Data Fig. Each experimental apparatus (behaviour and electrophysiology) has its own dedicated earth separated from the mains earth (through a 6-mm thick, 30-mm wide and 12-m long copper strap dug into the ground below the concrete slab). Background levels of radio-frequency disturbances at this rural site are extraordinarily low4. All of the experiments were performed on dark-adapted moths in darkness at night (beginning at least 1 h after sunset). Darkness was achieved with black-out blinds (to remove residual starlight and moonlight from outside) and dark cloth around the experimental apparatus (to shield from the minimal stray light emitted from the equipment). The non-magnetic electrophysiological apparatus (such as table and animal mounts; Extended Data Fig. 1f,g) was constructed from Thorlabs aluminium optomechanical components using high-grade stainless-steel fasteners. Vibration isolation between the aluminium pillar legs and the aluminium bread board table (on which the moth and manipulators were mounted) was provided by four high-grade stainless steel Stillpoints Ultra 6 (with Ultra base) isolators (Stillpoints). The moth was mounted (see below) onto a pillar attached to the bread board table, and a custom-built non-magnetic Sensapex piezo micromanipulator (Sensapex Oy, Oulu), also attached to the pillar, was used to move and advance a glass microelectrode. A removable circular UV-transmissive Perspex disc (diameter, 250 mm; thickness, 5 mm), covered in a layer of UV-transmissive diffusing paper (Lee Filters 251 1/4 white diffuser) and mounted 127 mm above the moth, was used for projection of celestial visual stimuli (see below). The electrophysiological apparatus was placed at the centre of a computer-controlled, double-wrapped41 three-axis (3D) Helmholtz coil system custom built in aluminium and copper (University of Oldenburg workshop; outer coil diameters; x, 900 mm; y, 835 mm; z, 775 mm) to create a nulled magnetic field (Extended Data Fig. The coil systems were powered by constant-current power supplies (Kepco, BOP 50-2M) and the current running through the coil systems was controlled through High-Speed USB Carriers (USB-9162, National Instruments) and custom-written codes in MATLAB (v.2019a and 2022b, MathWorks). Before each experimental session, Meda FVM-400 magnetometer measurements ensured that the magnetic field was nulled within the apparatus (Extended Data Fig. The non-magnetic behavioural apparatus (Extended Data Fig. 1h,i) consisted of a modified Mouritsen–Frost flight simulator3,4,5,42 used to record the virtual flight path of tethered migratory Bogong moths. In brief, each flight simulator consisted of a cylindrical Perspex arena (diameter, 50 cm; height, 35 cm) placed vertically onto an aluminium table with a clear Perspex top within a 3D Helmholtz coil system (as described above, but with coil outer dimensions: x, 1,245 mm; y, 1,300 mm; z, 780 mm). The arena walls were covered with two layers of black felt. An optical encoder (E4T Miniature Optical Kit, US Digital) was mounted in the middle of a UV-transmissive Perspex disc (diameter, 50 cm; thickness, 0.5 cm), which was placed on top of the arena like a lid. A layer of UV-transmissive diffusing paper (Lee Filters 251 1/4 white diffuser) was placed on top of the disc (and served as a screen for dorsal projection of celestial stimuli; see below). We used the encoder manufacturer's software (USB1 Digital Explorer 1.07, US Digital) to continuously record the moth's heading relative to geographical north (gN), therefore allowing us to reconstruct the moth's virtual flight path in the presence of celestial visual cues. An LED projector (ASUS S1 Mobile), neutral density (ND) filters (optical density between 4 and 5 log units) and a mirror placed at 45° under the Perspex tabletop were used to project a dim moving (10 mm s−1) pattern of optic flow onto a screen (Lee Filters 251 1/4 white diffusing paper) placed beneath the arena and therefore below the tethered flying moth (Extended Data Fig. The direction of the optic flow was controlled by custom written software (M. York), which coupled the encoder system (USB1 or USB4 encoder data acquisition USB device, US Digital) through a feedback loop. Thus, the optic flow would always move from head to tail below the moth, instantaneously changing direction as the moth changed direction. A natural moonless starry austral night sky, in both the electrophysiological and the behavioural rigs, was projected using a downward pointing projector (behavioural rig: LED projector ASUS S1 Mobile; electrophysiological rig: Sony MP-CL1A laser projector; spectra of both projectors are shown in Extended Data Fig. Each projector was mounted sufficiently high above the moth in each rig to provide clear and correctly sized dorsal images of the night sky. To avoid unwanted stray light, each projector was housed in a custom-built 3D-printed plastic box featuring an opening in front of the lens and ventilation slits above the projector. The free software Stellarium43 was used to simulate the moonless starry night sky over Canberra (about 80 km from Adaminaby as the crow flies) at 21:30 on four respective dates: 1 October 2018 (spring 2018), 21 March 2018 (autumn 2018), 21 October 2019 (spring 2019) and 27 February 2019 (autumn 2019). Screenshots (screen resolution 7,480 × 720 pixels) of these simulated starry skies were taken, cut in a circular shape using CorelDRAW X5 and saved as PNG files (300 dpi) to create the stimulus images (Fig. Subtended at the moth, celestial images projected onto the circular screens in each rig (see above) provided 160° (behavioural rig) and 100° (electrophysiological rig) fields of view of the starry night sky centred on the zenith. Celestial images contained grey level values ranging from 4 (darkest) to 255 (brightest, on a scale of 0–255), with an average grey level of 62. The quality of the night sky provided by these images was comparable to that provided by the natural rural night sky at Glenhare (as measured with a Unihedron Sky Quality Metre; Extended Data Table 1). Before each experiment, the PNG files were opened using IrfanView64 on a PC using a screen resolution of 1,280 × 720 pixels. Before each experiment, we ensured that the projection was centred accurately and that its light intensity was reduced to starlight levels using ND filters (the behavioural rig is shown in Extended Data Fig. The projection of the starry sky was initially set to its natural orientation relative to geographical north or flipped along its vertical and horizontal axes to test the moths under a 180° rotated sky. As the projectors did not emit UV light, we installed a custom-made LED-ring (built by T. McIntyre; diameter, 120 mm; inner diameter, 50 mm) featuring eight UV LEDs (LED370E Ultra Bright Deep Violet LED, Thorlabs) in front of the projector. The brightness of the LED-ring was controlled using custom written software using MATLAB (v.2019a and 2022b, MathWorks) and several layers of ND filters that were fixed in front of the LED-ring to bring the UV intensity into a quasi-natural range (the behavioural rig is shown in Extended Data Fig. To create randomized starry skies for experiments in autumn 2018 and spring 2018, the positions of all individual pixels of the natural night sky stimulus image were reassigned randomly to new positions, and the resulting randomized images were likewise saved as PNG files (Extended Data Fig. These featureless stellar conditions provided an identical stimulation intensity but provided no celestial spatial information. The randomized starry skies used during spring and autumn 2019 were improved by randomizing groups of pixels containing individual stars, therefore retaining the stars but removing spatial variations in the night sky (such as the Milky Way) that could be used for orientation (again maintaining identical intensity). Here, the stimulus image (of autumn 2019) was subdivided into squares with a size of 13 × 13 pixels, as the brightest star in the image had roughly these dimensions. The positions of these squares were now randomly reassigned and the resulting image was saved as a PNG file. A final improved randomized stimulus (used during spring 2019) was generated from the test stimulus by randomizing the positions of the individual visible stars. This was achieved by first detecting the position and size of each star in the test stimulus using a multiscale Laplacian of a Gaussian convolution of a greyscale version of the test stimulus, followed by local maxima detection. The resulting spatial information was then used to extract and save the image of each star from the natural night sky stimulus, before replacing them on a new background image, with a uniform colour and intensity equal to that of the mean of all pixels in the test stimulus that were not part of a star. To test the selectivity of central brain visual neurons to various parts of the Milky Way during electrophysiological experiments, we stimulated cells with artificial compass cues that, like the starry sky stimulus, filled the projection screen above the moth (Fig. These cues mimicked aspects of the autumn Milky Way: its brightest region around the Carina nebula (revolving dot) and its stripe-like shape (rotating bar). The bar length and width were chosen to mimic the main stripe of our natural Milky Way stimulus. The intensities of these artificial stimuli were also adjusted to provide a good mimic of the intensities of the stars and the dark background sky in our natural stimulus (see above) and had the same mean grey level (61 ± 1 for values between 0 to 255, darkest to brightest). To achieve this, each flight simulator arena consisted of a transparent UV-transmissive Perspex cylinder (again with diameter, 50 cm; height, 35 cm), placed vertically onto an aluminium table (Extended Data Fig. The table top was also made of transparent Perspex. The two flight simulators (and their tables) were placed around 15 m apart. A tethered moth was connected to an optical encoder suspended at the centre of the open top end of each arena by a thin horizontal UV-transmissive Perspex arm. The two arenas were controlled from computers that were operated from within a black light-tight cubical tent that was placed midway between the two arenas, approximately 7 m from each. The tent was therefore a landscape feature that the moths could potentially see during their tethered flights. Most behavioural procedures used in this study have been previously described4,5. Before attachment of tethering stalks, moths were chilled in a freezer for 5–10 min to immobilize them. The scales on the moth's dorsal thorax were removed by suction using a micro-vacuum pump (custom built by B.F.). Afterwards a thin vertical tungsten stalk (which is ferromagnetic free), fashioned at its end to create a small circular footplate, was glued to the dorsal thorax using contact cement while being restrained by a weighted-down plastic mesh. Moths were tested on the day of stalk attachment. Shortly before sunset, UV-transmissive Perspex boxes holding individual stalked moths were placed onto an elevated outdoor location and provided with a clear view of the western sky and the setting sun (and the skylight polarization pattern), in case these cues were important for calibrating compass mechanisms (as found for the magnetic compass of birds44,45,46). For tethering within the behavioural arena (performed in dim red light to maintain a dark-adapted visual state), the arena lid holding the optical encoder was lifted and the tungsten stalk of a vigorously flying moth (held with medical forceps) was attached to the bottom end of the encoder shaft through a 1.5 cm length of thin rubber intravenous medical tubing that connected the stalk to the shaft (Extended Data Fig. Once the arena lid was returned to the arena, this coupling enabled the moth to rotate freely around its yaw axis and choose any flight direction. Once the moth was mounted in the arena, it was gently pointed manually towards geographical north. The heading direction count was then reset, the moth was released and the optical encoder was enabled to register the flight heading direction of the moth under a given night sky condition (projected on the arena lid) at a sampling rate of 5 Hz (and a horizontal resolution of 3°). Moths chosen for analysis were required to fulfil three ante hoc criteria, two before the experiment and one during the experiment: (1) the tethering stalk was perfectly vertical; (2) wing flapping was vigorous and its amplitude was large and equal for both wings (indicating that the contact cement had not interfered with the wings); and (3) that the moth flew continuously for the full 5 min. For the last criterion, if a moth stopped flying, the arena was gently tapped in order to stimulate the moth to continue flight behaviour. A moth that stopped flying four times was rejected and the recording aborted. For the field seasons of spring 2018 and 2019, as well for Autumn 2019, the percentages of moths that were aborted due to failing to meet these criteria were 2 out of 59 or 3.4% (Spring 2018), 25 out of 137 or 18.3% (Spring 2019), and 35 out of 76 or 46.1% (Autumn 2019). The high rejection rate during Autumn 2019 may have been due to the unusually wet and cold weather that occurred during this season. Moths often behave erratically on rainy or stormy nights, and even on days before and after such nights. The same behavioural methods (and criteria) that were used indoors were also used outdoors. The goal of these experiments was to understand how migratory Bogong moths deal with natural skies (in particular the nightly movements of the stars and moon, as well as cloud cover) while experiencing a normal geomagnetic field and the local surrounding landscape under natural illumination. Experiments were performed under clear starry skies during autumn 2023 (over three nights during the last week of March), as well as at two times of night, to test whether the migratory orientations of moths were affected by the nightly movements of the stars and moon (which was approximately half-full): (1) between 20:32 and 21:06 (about 1.5 h after sunset), and (2) between 23:25 and 23:59. The same moths were used for orientation measurements at both times, and moths that were flown in one arena at the earlier time (and saw the black tent on their eastern side) were flown in the other arena at the later time (and now saw the black tent on their western side) and vice versa. Thus, other panoramic landmarks (apart from the tent) differed markedly in their spatial positions from within the two arenas. Between earlier and later experiments, tethered moths were kept isolated and in the dark in a suitcase that was warmed with hot water bottles. Experiments were also carried out on a fourth completely overcast night that totally covered the stars and moon (between 21:12 and 21:48). For afternoon experiments, we removed the two 1.2 log unit ND filters in front of the projector lens (as described above) to generate a starry sky projection around 250 times brighter than the one used at night (to account for the circadian-rhythm-induced light-adapted state of the moths). For night experiments, the ND filters were reinserted. The moths were mounted onto a custom-made 3D-printed animal holder and immobilized using wax. The neural sheath was digested with Pronase (Sigma-Aldrich) for about 30 s and then carefully washed. It was then removed using a pair of fine forceps. A second small hole was cut into the cuticle above the proboscis muscle and a chlorinated silver wire was inserted into this muscle to serve as reference electrode. Glass electrodes were pulled from borosilicate glass on a P-97 Flaming-Brown micropipette puller (Sutter Instrument), and had a typical resistance of 50–100 MΩ. The electrode tip was filled with Neurobiotin solution (4% Neurobiotin in 1 M KCl, Vector Laboratories), and the remainder of the electrode was filled with 1 M KCl. Electrodes were moved into position with a non-magnetic Sensapex micromanipulator. Signals were amplified using a BA-03X intracellular amplifier and headstage (NPI Electronic), and were then digitized using a CED Micro 1401-3 (Cambridge Electronic Design) and recorded with Spike2 software (v.8.03, Cambridge Electronic Design). Stimulus control signals from MATLAB (v.2019a and 2022b, MathWorks) were simultaneously recorded in Spike2. During the recording, the brain was kept hydrated through regular application of moth ringer solution47. Moths were mounted under starry skies that were naturally oriented before sky rotation (that is, had the same orientation as the stars outside the laboratory). In earlier experiments, the initial heading orientation of the mounted moth was north relative to the stars and, in later experiments, the initial heading orientation south; however, greatest ease of access to the brain was eventually found for an eastward heading. The action potential spike trains obtained for a 360° rotation of the sky were corrected according to the initial heading orientation of the moth so that spike trains obtained across all experiments were comparable (with a correction angle of 0° applied for an initial northward orientation, 90° for an eastward orientation and 180° for southward orientation). Impaled cells were stimulated with dorsally projected images of the natural starry night sky, or control images of randomized stars (see above). Cells were also stimulated with bars and dots that mimicked different parts of the Milky Way (see above). All of these projected images (natural and randomized stars, artificial Milky Way stimuli) were rotated 360° at 30° s−1 to 45° s−1, with a 2 s break between clockwise and anticlockwise rotations, using custom-written MATLAB code (v.2019a and 2022b, MathWorks). Between rotations, a neutral grey background image was presented that had the same average grey value as the starry sky/control images. We targeted an area of the central brain in which we expected to find both CX and lateral complex neurons, as well as optic lobe neurons traversing the brain in the posterior optic tract. Neurons that clearly did not respond to an initial sky rotation were immediately discarded and no recording was saved. 79 neurons were assessed as potentially responding to the stimulus and, of these, 28 (35%) met the inclusion criteria of a unimodal or bimodal response profile. The remaining 51 neurons were classified as uniform in their response to stellar rotation (that is, showed no obvious response) and were therefore excluded from the analysis. After recording from a suitable cell, a positive current (range: 1–3 nA for 3 min) was applied to the electrode to inject Neurobiotin into the cell. During washing, the retinas were removed. Brains were then incubated with streptavidin–Cy5 (Jackson Immuno Research, 1:1,000 in PBS with 0.3% Triton X-100) at 4 °C for 3 days and kept in the dark from this point onwards. After incubation, the brains were washed in PBS-Triton X-100 (6 times for 20 min) and PBS (2 times for 20 min) and then dehydrated in an increasing ethanol series (50%, 70%, 90%, 95% and twice at 100%). Brains were then transferred to a fresh mixture of methylsalicylate and ethanol (1:1) and, after 15 min, were left to clear in 100% methylsalicylate for 75 min. The cleared brains were mounted in Permount (Thermo Fisher Scientific) mounting media between two coverslips and left to dry for at least 2 days. Brain samples were scanned with the 633 nm laser of a Leica SP8 confocal microscope and viewed with a ×20 oil-immersion objective (Leica Microsystems). For optimal resolution, the scan settings were set to 1,024 × 1,024 pixels, 12-bit pixel depth, 3 times line accumulation and 400 lines per s in the photon-counting mode of the hybrid detector. Neurons and relevant neuropils were then reconstructed in Amira v.5.3 (Thermo Fisher Scientific) and registered into the Bogong moth standard brain22,23. As there were no statistical differences in results obtained from male and female moths, the results were pooled. The behavioural analysis used in this study has been previously described4,5. As mentioned above, the encoder software (USB1 Digital Explorer v.1.07, US Digital) recorded the instantaneous heading directions of a tethered flying moth every 200 ms (5 Hz) and saved these values in a text file. We used custom-written MATLAB code (v.2019a and 2022b, MathWorks) to visualize the virtual flight paths of all tested moths and calculated a mean orientation vector based on each virtual flight path. 3c–f) encodes the mean orientation direction of a moth's individual recorded flight path as well as its r value (that is, length, or directedness, of the flight path vector). To take advantage of the extra information in our data arising from the fact that the flight trajectories of moths not only had a mean direction (as used for a classic Rayleigh test49) but also a mean directedness (vector length), we used the circular statistics software Oriana (v.4 (2011), KCS) and Excel (Microsoft Office 2019, Microsoft) to apply a one-sided Moore's modified Rayleigh test4,50,51 with Bonferroni correction for multiple comparisons (Supplementary Table 1). The R* value encodes the directedness of a population of tested moths and reveals the likelihood that the combined flight direction of these moths—each with its own direction and directedness—differs significantly from random. To confirm that the two distributions of moth flight directions were significantly different between naturally oriented and 180°-rotated night sky conditions, as well as significantly different between spring and autumn for any single sky condition (Fig. 3c–f), a Mardia–Watson–Wheeler test49 (Oriana) was used (Extended Data Fig. The statistical procedures used for data collected outdoors were the same as those used for data collected indoors, with the exception that a likelihood-ratio test52 was used to test whether mean orientation directions for moth populations flown under two different clear natural sky conditions (earlier and later in the evening) were significantly different from each other (Fig. We did this by fitting maximum-likelihood distributions to the mean orientation directions for each trial, using the circular statistics package circular53 in R v.3.6.1 (www.r-project.org). In a similar manner, we determined the likelihood that the moth population flown under overcast conditions (Fig. 2c) had the same mean direction as either of the two populations flown under clear skies, in this case testing whether the means of the two different populations differed significantly. Moreover, for the cohort of 95 moths flown under clear skies earlier and later in the evening (Fig. 2a,b), we used the mean orientation direction of each moth flown earlier (Fig. 2b), and tested whether the mean angular differences were significantly different from zero (no change in heading). As the moth cohort was clearly less oriented later in the evening (possibly due to fatigue), we made this analysis by separating moths into three groups (Extended Data Fig. 2): (1) those that were well-oriented throughout the night (r > 0.8, both earlier and later in the evening, 35 out of 95 moths); (2) those that were well-oriented in only one trial (typically r > 0.8 earlier and r < 0.8 later, 42 out of 95 moths), and those that were less well-oriented throughout the evening (0.2 < r < 0.8 in both trials, 18 out of 95 moths). This analysis revealed that moths that were well oriented throughout the night (group 1) showed no change in heading, while those that were less-well oriented (groups 2 and 3) showed a westward drift later in the evening (Extended Data Fig. However, this drift was not sufficient (in either direction or directedness) to significantly alter the mean direction of the population as a whole from earlier to later in the evening (Fig. Spike train data were analysed using custom-written code in MATLAB (v.2019a and 2022b, MathWorks). Responses were classified as unimodal (models M2A, M2B or M2C in the R-library CircMLE) or bimodal (models M4A or M4B) based on the Akaike information criterion. For cells that responded to night sky rotation (rotation angle φ) with an increase in spiking activity (which became maximal at the sky rotation angle φmax), the bootstrapped interquartile range of the unimodal part of the response distribution was taken to be an estimator of the half-width of the rotation tuning curve (φ range for 50% maximal response or greater). For cells that responded with a decrease in spiking activity, von Mises and mixed-von Mises distributions did not appropriately reflect this inhibition. Instead, the responses of these neurons were binned at 1° intervals and low-pass filtered, and the width at half-maximum was then determined in MATLAB (v.2019a and 2022b, MathWorks). before the night sky rotation started. This was calculated for each stimulus rotation separately. To obtain an estimate of the reliability of a given cell's response to repeated rotations of the night sky, the mean preferred stimulus rotation angle (φmax) was calculated as the circular mean for each individual night sky rotation. Behavioural data under natural clear starry skies (Fig. 2a,b) were obtained in independent experiments over three separate nights (with new moths used each night). Behavioural data under natural overcast skies (Fig. 2c) were obtained during only a single night. Behavioural data under projected starry skies generated using Stellarium (Fig. 3c–f) were obtained in independent experiments run over several nights with each night's cohort of moths used only once. These experiments were repeated with the same results over two separate spring and autumn seasons (2018 and 2019). Thus, all of these experiments deal with biological replicates. Sample sizes were based on the availability of moths, with at least 40 moths (and as many as 70) being used for each experiment. Experiments were conducted without blinding and without a specific randomization protocol (apart from the random selection of moths used as replicates). 5) were obtained from single visual cells in the brain that were subjected to rotations of a projected starry sky (generated using Stellarium). Since each recording (and subsequent dye injection) was a unique observation, replication was not possible. This is due to the stochastic nature of intracellular recordings—single neurons were penetrated randomly from target brain regions that contain many thousands of cells. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Source data are provided with this paper. The code used to generate stimuli and to collect and analyse data in this study are available at GitHub (https://github.com/stanleyheinze/Starry_sky_code). A study of the ecology of the adult Bogong moth Agrotis infusa (Boisd.) (Lepidoptera: Noctuidae), with special reference to its behaviour during migration and aestivation. The Australian Bogong moth Agrotis infusa: a long-distance nocturnal navigator. Virtual migration in tethered flying Monarch butterflies reveals their orientation mechanisms. The Earth's magnetic field and visual landmarks steer migratory flight behavior in the nocturnal Australian Bogong moth. Dreyer, D. et al. A guide for using flight simulators to study the sensory basis of long-distance migration in insects. Mouritsen, H. Long-distance navigation and magnetoreception in migratory animals. Foster, J. J., Smolka, J., Nilsson, D.-E. & Dacke, M. How animals follow the stars. Sotthibandhu, S. & Baker, R. R. Celestial orientation by the large yellow underwing moth Noctua pronuba L. Anim. Dacke, M., Baird, E., Byrne, M., Scholtz, C. & Warrant, E. J. Dung beetles use the Milky Way for orientation. Foster, J. J. et al. Stellar performance: mechanisms underlying Milky Way orientation in dung beetles. Smolka, J. et al. Night sky orientation with diurnal and nocturnal eyes: dim-light adaptations are critical when the moon is out of sight. Ferguson, D. E., Landreth, H. F. & Turnipseed, M. R. Astronomical orientation of the southern cricket frog, Acris gryllus. Emlen, S. T. The stellar-orientation system of a migratory bird. Mouritsen, H. & Larsen, O. N. Migrating songbirds tested in computer-controlled Emlen funnels use stellar cues for a time-independent compass. Mauck, B., Gläser, N., Schlosser, W. & Dehnhardt, G. Harbour seals (Phoca vitulina) can steer by the stars. & Wehner, R. Polarization of the moonlit clear night sky measured by full-sky imaging polarimetry at full moon: comparison of the polarization of moonlit and sunlit skies. Dacke, M., Nilsson, D.-E., Scholtz, C. H., Byrne, M. & Warrant, E. J. Insect orientation to polarized moonlight. Comparison of navigation-related brain regions in migratory versus non-migratory noctuid moths. & Heinze, S. The Bogong moth brain: anatomical basis of nocturnal migration. Honkanen, A., Adden, A., da Silva Freitas, J. & Graham, P. Connecting brain to behaviour: a role for general purpose steering circuits in insect orientation? Takahashi, S., Hombe, T., Matsumoto, S., Ide, K. & Yoda, K. Head direction cells in a migratory bird prefer north. Beetz, M. J. et al. Flight-induced compass representation in the monarch butterfly heading network. Hulse, B. K. et al. A connectome of the Drosophila central complex reveals network motifs suitable for flexible navigation and context-dependent action selection. Hulse, B. K. & Jayaraman, V. Mechanisms underlying the neural computation of head direction. Milligan, P. R., Franklin, R., Minty, B. R. S., Richardson, L. M. & Percival, P. J. Heinze, S. & Homberg, U. Maplike representation of celestial E-vector orientations in the brain of an insect. Beetz, M. J., Kraus, C. & el Jundi, B. Neural representation of goal direction in the monarch butterfly brain. Mussells Pires, P., Zhang, L., Parache, V., Abbott, L. F. & Maimon, G. Converting an allocentric goal into an egocentric steering signal. Rayshubskiy, A. et al. Neural circuit mechanisms for steering control in walking Drosophila. Lyu, C., Abbott, L. F. & Maimon, G. A neuronal circuit for vector computation builds an allocentric traveling-direction signal in the Drosophila fan-shaped body. Stone, T. et al. An anatomically constrained model for path integration in the bee brain. The remarkable visual capacities of nocturnal insects: vision at the limits with small eyes and tiny brains. Alerstam, T. Conflicting evidence about long-distance animal navigation. Kirschvink, J. L. Uniform magnetic fields and double-wrapped coil systems: improved techniques for the design of bioelectromagnetic experiments. Evidence for a southward autumn migration of nocturnal noctuid moths in central Europe. Cochran, W. W., Mouritsen, H. & Wikelski, M. Migrating songbirds recalibrate their magnetic compass daily from twilight cues. & Åkesson, S. Polarized light cues underlie compass calibration in migratory songbirds. Sjöberg, S. & Muheim, R. A new view on an old debate: type of cue conflict manipulation and availability of stars can explain the discrepancies between cue-calibration experiments with migratory songbirds. Heinze, S. & Reppert, S. M. Sun compass integration of skylight cues in migratory Monarch butterflies. Dreyer, D. et al. Data for ‘Bogong moths use a stellar compass for long-distance navigation at night'. Batschelet, E. Circular Statistics in Biology (Academic, 1981). Moore, B. R. A modification of the Rayleigh test for vector data. Zar, J. H. Biostatistical Analysis 4th edn (Prentice Hall, 1999). The log likelihood ratio test (the G-test); methods and tables for tests of heterogeneity in contingency tables. Fitak, R. R. & Johnsen, S. Bringing the analysis of animal orientation data full circle: model-based approaches with maximum likelihood. Mardia, K. Statistics of Directional Data (Academic, 1972). Controlling the false discovery rate: a practical and powerful approach to multiple testing. This paper is dedicated to the memory of our friend and collaborator Barrie Frost who passed away in 2018. This research made use of the Stellarium planetarium43. We acknowledge the European Research Council (advanced grant number 741298 to E.W., starting grant number 714599 to S.H., synergy grant 810002 to H.M.), US Air Force Office of Scientific Research (grant number FA9550-14-1-0242 to E.W. ), the Royal Physiographic Society of Lund (to E.W. ), the Swedish Research Council (Vetenskapsrådet, grant number 621-2012-2205 to E.W. ), the German Research Council (Deutsche Forschungsgemeinschaft, grant number 395940726, SFB 1372 ‘Magnetoreception and navigation in vertebrates' to H.M.) and the Natural Sciences and Engineering Research Council of Canada (NSERC, grant number 353-2009 to B.F.) for their ongoing support. The research was performed under a Scientific Licence issued by the New South Wales National Parks and Wildlife Service (no. SL100806) and took place on the traditional lands of the Walgalu and Ngarigo peoples, to whose elders—past, present and future—the authors pay their respect. Open access funding provided by Lund University. These authors contributed equally: David Dreyer, Andrea Adden David Dreyer, Andrea Adden, Hui Chen, Jesse Wallace, James Foster, Stanley Heinze & Eric Warrant Neural Circuits and Evolution Laboratory, The Francis Crick Institute, London, UK Department of Psychology, Queens University, Kingston, Ontario, Canada College of Asia and the Pacific, Australian National University, Canberra, Australian Capital Territory, Australia Department of Applied Biosciences, Macquarie University, Sydney, New South Wales, Australia National Collections & Marine Infrastructure, CSIRO, Canberra, Australian Capital Territory, Australia Centre for the Advanced Study of Collective Behaviour, University of Konstanz, Konstanz, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed detailed knowledge of Bogong moth ecology and aestivation sites and provided critical logistical support in the Australian Alps. Correspondence to David Dreyer, Andrea Adden or Eric Warrant. The authors declare no competing interests. Nature thanks Roy Ritzmann and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. c, Our new Australian lab, built of entirely non-magnetic materials (to eliminate magnetic artefacts), at Adaminaby. The laboratory occupies 50 m2, has two separate rooms (one for the electrophysiology rig, and one for the behavioural rig) and sits on a concrete slab. Each rig has its own earth separated from the mains earth (provided by a 6 mm thick, 30 mm wide and 12 m long copper strap dug into the ground below the concrete slab). d, The flight arena, with a flying moth (circled) tethered to an optical encoder that tracks changes in the moth's heading direction (relative to North). The flying moth is free to turn in any direction, and the encoder thus records its virtual flight trajectory over time. The floor of the arena displays an optic flow pattern of starlight-intensity (here brightly shown in order to see it) which always moves from head to tail as the moth turns, inducing it to fly. f,g, The non-magnetic electrophysiology rig for recording the responses of compass cells in the moth's brain in response to precise magnetic and visual stimulation. The moth is mounted horizontally on a stand beneath the projection screen and an electrode is inserted into its central brain. h,i, The non-magnetic behavioural rig for monitoring the trajectories of tethered flying moths. The flight arena, again enclosed within Helmholtz coils, receives an image of the starry night sky on a projection screen installed above the moth (i), from a projector mounted above the coils (h). a–c, Orientation directions under a cloudless autumn night sky earlier and later in the evening (data from Fig. a,d, Individual moths that were well-oriented in both trials (r > 0.8 in both trials, n = 35) retained the same heading (a), exhibiting no change in direction between trials. At a population level, the pattern of changes (d) was not significantly different from 0° (χ2 = 0.428, d.f. b,e, Moths that were well-oriented in only one of the two trials (r > 0.8 in either trial, n = 42), typically earlier in the evening, started the evening with a population mean heading similar to those that were well-oriented throughout the night (left panels in a and b), but drifted to the West later in the evening (b, right panel), a drift (e) that was significantly different from 0° (χ2 = 7.454, d.f. c,f, For moths less well-oriented throughout the night (r < 0.8 in both trials, n = 18) there was no significant population level orientation (c), and no significant population-level difference (f) from a 0° heading change (χ2 = 1.769, d.f. Nonetheless, the mean headings, despite lacking significance, were similar to those of the moths in b. a–c, Direction and directedness of MV given by α and R* value, respectively. Inner dashed circle: required R* value for statistical significance: p < 0.05. One-sided Moore's modified Rayleigh tests with Bonferroni correction for multiple comparisons – corrected p-values: 1.72 × 10−11 (earlier) and 2.76 × 10−8 (later) in a; 2.46 × 10−12 (earlier) and 5.57 × 10−3 (later) in b; 0.426 (earlier) and 1.000 (later) in c. d–f, Red arrow: the bias-corrected mean vector for the maximum-likelihood distribution of heading changes; Dashed circle: required r value for statistical significance: p < 0.05; Red arc: 95% confidence intervals (CI95) of the mean angle across heading changes (numerical simulation). One-sided likelihood-ratio tests52 adjusted within each dataset (from a, b and c) using the Benjamini-Hochberg method56. g, Mardia-Watson-Wheeler tests for determining whether populations of moth directions (n) in different seasons and under different skies (as presented in Fig. 3c,d) were significantly different from each other (if so, p is small). 0° is the natural starry sky orientation and 180° is the starry sky rotated by 180°. φmax values were derived from neural responses to a rotating starry sky stimulus, subdivided into response type and seasons. Frequency plots and circular φmax diagrams, separated into clockwise (CW, left panels) and counter-clockwise (CCW, right panels) stimulus rotations, for every cell recorded in spring 2018 (cell ID, red text). Responses to the starry sky stimulus are shown in blue, responses to the control stimulus are shown in grey. Where the cell's anatomy was identified, branching regions are also given (shown as green text). For detailed anatomies, see Extended Data Fig. Frequency plots and circular φmax diagrams, separated into clockwise (CW, left panels) and counter-clockwise (CCW, right panels) stimulus rotations, for every cell recorded in autumn 2019 (cell ID, red text). Responses to the starry sky stimulus are shown in blue, responses to the control stimulus are shown in grey. Where the cell's anatomy was identified, branching regions are also given (shown as green text). For detailed anatomies, see Extended Data Fig. Frequency plots and circular φmax diagrams, separated into clockwise (CW, left panels) and counter-clockwise (CCW, right panels) stimulus rotations, for every cell recorded in autumn 2020 (cell ID, red text). Responses to the starry sky stimulus are shown in blue, responses to the control stimulus are shown in grey. Where the cell's anatomy was identified, branching regions are also given (shown as green text). For detailed anatomies, see Extended Data Fig. Abbreviations: OL = optic lobe, CX = central complex, LX = lateral complex, ME = medulla, LO = lobula, LOP = lobula plate, PC = protocerebrum, LAL = lateral accessory lobes, FB = fan-shaped body, EB = ellipsoid body, PB = protocerebral bridge, GA = gall, VLP = ventrolateral protocerebrum, VMP = ventromedial protocerebrum. All cells will be published on insectbraindb.org, preliminary links to a subset of these cells are given below. a,b, The Helmholtz coil system was used to compensate the local geomagnetic field and create a “nulled” or near-zero magnetic field (NZMF) within the two experimental rigs. The typical variation of the magnetic field strength, shown as nanotesla (nT) over time (s), for the duration of a typical behavioural experiment (300 s). This measurement was made at the position of the moth (position 0 in d). The three-component vector (X, Y, Z: red, green and purple lines respectively) of the field was measured 6 times over 300 s to obtain X, Y and Z traces. These six traces were averaged to give the mean magnetic field strength for each component (X, Y and Z) and the standard deviation (shaded areas around the mean). c,d, To control that the central area of the arena (where tethered moths flew) had a nulled magnetic field, the magnetometer probe was positioned at the arena centre (position 0 in c) and moved outwards in 1 cm steps along 4 directions (coloured arrows in d) relative to geographic North (gN) and the strengths of the X, Y, and Z field components were measured at each position. These field strengths were less than 100 nT a radius of 4 cm from the coil centre (dashed yellow line and circle in c and d) in all four directions (bar colours in c correspond to direction colours in d). Since the wingspan radius of a Bogong moth is about 2.5 cm (dashed brown line and circle in c and d), this indicates that the entire moth experienced a magnetic field strength substantially less than 100 nT, effectively a “nulled field” (which can be compared to the external local magnetic field strength of around 58,500 nT). a, Stellarium images of the austral starry night sky (and randomized sky) used during the spring and autumn migratory seasons in 2018 and 2019. Geographic North (gN) is upwards in each image. Randomized skies in 2018 were created by randomizing pixels (insets), whereas those in 2019 were improved by randomizing stars (see Methods). b,c, Brightness distributions of natural starry skies and randomized skies projected onto a screen above the moth in the behavioural (b) and electrophysiological rigs (c). The probe of a spectrometer (Ocean Optics QE65 PRO, see Methods), equipped with a collimating lens (25° field of view), was positioned at the same location as the moth and systematically pointed at the projected sky in different directions (0–360° in 30° steps) relative to geographic North. The radiance (photons/cm²/sec/sr) was measured from the screen in each direction in (5 measurements per position): mean radiance is shown as a function of measurement direction (green squares = spring projection, magenta squares = autumn projection, blue squares = randomized projection). Note that the brightest areas of the autumn and spring projections were respectively located south-easterly and westerly. For the projector in the behavioural rig, a ring of UV LEDs was used to create a quasi-realistic spectrum (see Methods). Note that for this spectrum the noise level is relatively high since we used all the neutral density filters normally used during an experiment. The underlying population-level statistics for each population of moths tested in each season during 2018 and 2019 under seasonally appropriate naturalistic projected night skies at 0° (natural orientation) or rotated 180°, or under randomized starry night skies (which form the basis of the results shown in Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Dreyer, D., Adden, A., Chen, H. et al. Bogong moths use a stellar compass for long-distance navigation at night. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09085-w'>Impacts of climate change on global agriculture accounting for adaptation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 15:24:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 642, pages 644–652 (2025)Cite this article Climate change threatens global food systems1, but the extent to which adaptation will reduce losses remains unknown and controversial2. Even within the well-studied context of US agriculture, some analyses argue that adaptation will be widespread and climate damages small3,4, whereas others conclude that adaptation will be limited and losses severe5,6. Scenario-based analyses indicate that adaptation should have notable consequences on global agricultural productivity7,8,9, but there has been no systematic study of how extensively real-world producers actually adapt at the global scale. Here we empirically estimate the impact of global producer adaptations using longitudinal data on six staple crops spanning 12,658 regions, capturing two-thirds of global crop calories. We estimate that global production declines 5.5 × 1014 kcal annually per 1 °C global mean surface temperature (GMST) rise (120 kcal per person per day or 4.4% of recommended consumption per 1 °C; P < 0.001). We project that adaptation and income growth alleviate 23% of global losses in 2050 and 34% at the end of the century (6% and 12%, respectively; moderate-emissions scenario), but substantial residual losses remain for all staples except rice. In contrast to analyses of other outcomes that project the greatest damages to the global poor10,11, we find that global impacts are dominated by losses to modern-day breadbaskets with favourable climates and limited present adaptation, although losses in low-income regions losses are also substantial. These results indicate a scale of innovation, cropland expansion or further adaptation that might be necessary to ensure food security in a changing climate. Disruptions of the global food system owing to climate change threaten human well-being12,13,14 and social stability10,15, but researchers lack a complete understanding of the magnitude and structure of potential impacts on food systems globally2,16. It is known that changes to the climate will alter the distribution of weather experienced across the planet17 and that biophysical processes in agricultural systems will respond5,7,8. However, the degree to which humans around the world will effectively adapt their agricultural practices in reaction to these changes remains unknown2,3,8,9. For this reason, existing global projections have been unable to account for the adoption rates and efficacy of producer adaptations7,8,18 and it remains unresolved whether the compensatory responses of producers are likely to overcome the challenges posed by climate change4,19. Here we develop a unified empirical approach to measure the effect of climate change on staple crop production, accounting for the costs, benefits and adoption rates of producer adaptations as they are observed in practice around the world. We study one of the most comprehensive samples of subnational crop yields ever assembled, representing two-thirds of global cropped calorie production, which allows our results to be globally representative. Using high-resolution data from populations across diverse contexts allows us to understand the real-world response of producers to weather events, changes in climate and economic development. We then apply our empirical results to project probabilistic global climate change impacts on yields that account for environmental changes, biophysical processes and the compensatory responses of producers. Previous analyses using process-based models have provided crucial insights about the potential impacts of climate change on global food systems and have enabled decades of progress in agronomic modelling7,8,9,18. These models explicitly characterize the biophysical processes that generate yields (for example, root depth, evapotranspiration, light utilization) and are calibrated to precisely managed experimental fields7,18, thereby providing detailed insight into the agronomic mechanisms by which climatic changes influence yields. However, these models generally assume that producers optimize yields subject to decision rules formed through modeller expert judgement (for example, ‘no adaptation' or ‘optimal varietal switching'7,18) in contrast to observing the decisions made by producers on the ground16,20,21. These decision rules describe what is theoretically possible but might not reflect actual conditions across diverse socioeconomic contexts, which are influenced by financial constraints, market failures and human error, along with other factors (Methods). Furthermore, because global process-based models are parameterized on the basis of data from scientifically managed experimental fields, concerns have been raised about their ability to represent actual producer decisions in diverse and resource-constrained contexts16,20,21. To address these challenges, we develop an econometric approach that simultaneously captures the combined impact of biophysical crop responses and producer decision-making. In place of modeller-prescribed adaptation scenarios, our approach empirically recovers the effect of actual adaptations undertaken in response to diverse climatic and economic conditions faced by producers. This reduced-form approach does not attempt the difficult task of identifying and modelling each individual mechanism through which these adaptations occur. Instead, we measure the total impact of adaptive adjustments in response to the climate (for example, changing varietals, altering cultivation windows, adjusting fertilizer) without modelling intermediary processes explicitly (Methods and Supplementary Information, section B). We apply these measurements—which capture the extent, intensity and efficacy of real-world producer adaptations as a function of local climates and economic conditions—in simulations of future crop production under climate change and economic development. The key assumption underlying our approach is that producers facing similar climates, incomes and irrigation infrastructure make similar management decisions, which allows us to estimate the effects of producer decisions, which we do not observe. The weakness of our approach is that, if producers facing similar contexts are not comparable, then our estimates of producer adaptation may be biased (Methods and Supplementary Information, section B). Previous empirical work has demonstrated how reduced-form econometric approaches can be calibrated to producer behaviour in specific regions2,5,11,22,23, such as maize producers in the USA5. Follow-up work demonstrated that this approach is complementary to, rather than a replacement for, detailed process-based models8,9,20,23,24,25. We build on the literature analysing the variation of weather responses with local climate as an indicator of adaptation4,19,26,27, making use of the method for capturing the benefits of adaptation in ref. 28 and accounting for adaptation costs based on ref. 29. Our work here represents, to the best of our knowledge, the first global analysis of staple crops that: (1) accounts for observed adaptation behaviour and how it varies globally; (2) uses these results to generate global projections of climate impacts on yields over the twenty-first century; and (3) transforms these projections into an empirically derived ‘damage function' that links changes in global calorie production to changes in GMST (ΔGMST). The scale and scope of our projections are comparable with global gridded process-based models, such as those underlying the Agricultural Model Intercomparison and Improvement Project (AgMIP)7. However, unlike those models, our analysis is based on globally representative data and accounts for the adoption rates and efficacy of adaptive behaviours that producers choose to undertake in practice across diverse environmental and economic contexts (for example, varietal switching and optimization of inputs such as fertilizer or irrigation intensity; see Supplementary Information, section B). We make four contributions that address key challenges when projecting global impacts of climate change on agriculture. First, we assemble one of the largest datasets of subnational crop production available (Ray et al.30 analyse the only comparable dataset we are aware of, containing “~13,500 political units” and four crops, but have not made their data public), covering 12,658 subnational administrative units from 54 countries for six staple crops spanning diverse local climates and socioeconomic contexts (Fig. Second, we systematically select from many correlated aspects of weather affecting crop biophysical processes which have been separately studied2. In our models of crop biophysical processes, we endeavour to ‘let the data speak' and select from this broad set of weather variables using cross-validation over our rich panel data, an approach commonly used in machine learning (Supplementary Information, section C, Extended Data Figs. Third, we estimate the degree of producer adaptation, which has been widely debated2,3,4,5,6,19,22,31. Following previous research, we use climate summary statistics (such as average temperature, average rainfall; see bars in Fig. 1d,f) to identify similarly adapted subpopulations4,5,26,28,29,31,32 (Extended Data Figs. However, we select crop-specific measures of climate by means of cross-validation (Extended Data Fig. Further, we account for both the costs29 and the benefits of local adaptations (Methods and Supplementary Information, section H); failing to account for costs overstates the value of adaptation19. Fourth, our model allows for differences in resource access (for example, fertilizer, technology, insurance and credit markets or labour markets) to influence patterns of food production (Methods). This is particularly important in projections, as future economic development is likely to alter agricultural practices in ways that could make agriculture more resilient or vulnerable to weather33. Region-specific and crop-specific impacts are projected to evolve in the future based on changes to economic conditions (for example, GDP per capita) and climate (for example, average temperature), reflecting economic development and adaptation. a, Fraction of cropped area observed in our subnational administrative data, for example, regions spanning North, Central and South America. b, Example inset regions surrounding Lucas do Rio Verde municipality (black outline) in Mato Grosso state, Brazil. c, Time series of maize yield administrative data from Lucas do Rio Verde. d, Empirically estimated relationship between daily temperature and log maize yields (shaded region: 95% confidence interval) conditional on other factors that include rainfall, climate and access to inputs and technology (Methods). Bars indicate county-specific values for variables used to predict the local, county-level effects of temperature and rainfall (red is average temperature, blue is average precipitation, orange is GDP per capita and green is area equipped for irrigation). Example regions depicted from top to bottom are: Iroquois County, Illinois, USA; La Barca municipality in Jalisco, Mexico; and Lucas do Rio Verde in Mato Grosso, Brazil. e, As in a but for an example region of Asia. f, As in d but for the response of rice to daily temperature in Dongbao district in Hubei, China, the cities of Thanjavur and Tiruchirappalli in Tamil Nadu, India, and Sulawesi Utara, Indonesia. g, Expected exceedance degree days over 30 °C projected for each of 24,378 regions in 2090 in a high-emissions scenario (RCP 8.5). Exceedance degree day projections (and other variables) from each model are combined with econometric results, such as d and f, to project crop-specific climate impacts that account for adaptation. Because our data span climatic zones around the world, we are able to empirically measure how adaptive practices in different climates mediate the influence of environmental conditions (Fig. 1d,f and Extended Data Fig. This ‘reduced-form' approach does not provide the granular detail contained in process-based models but it does account for the net consequence of nearly all climate-associated adaptive actions available to producers, without requiring that each is explicitly modelled or observed22. For example, our approach accounts for adaptive actions such as selecting a varietal with an earlier harvest date to avoid late-season heat exposure, adjusting typical fertilizer use conditional on socioeconomic conditions and adjusting typical irrigation water use conditional on access to irrigation, along with many other available adaptive measures. Notably, our approach does not account for altering the area planted to a given crop (that is, crop switching, although we do account for this in our valuation of climate impacts) and shifting the planting date outside present growing season windows (although we do account for varietal selections with earlier harvest dates). Our final global model for each crop is high dimensional, nonlinear and dependent on the underlying economic and climatic characteristics of each location—but nonetheless fully interpretable. Each crop model includes several measures of weather that are all interacted with several measures of climate, as well as income and possibly irrigation (Extended Data Fig. This approach allows yields in a given year and location to be computed by (1) weather in that location–year, which is an argument to (2) a location–year function determined by crop-specific measures of climate, income and irrigation. The resulting models are relatively parsimonious compared with global process-based models7,8 because the influence of many other factors not explicitly included in the model (for example, soil type and quality) is captured non-parametrically (Methods and Supplementary Information, section B). However, these parsimonious models are skilful. Our model outperforms process-based model benchmarks globally across crops and regionally over 81% of crop–country pairs (Supplementary Information, section F and Extended Data Tables 1 and 2). Furthermore, in a ground-truth exercise, our model reproduces variation in local yields (R2 values from 0.63 (cassava) to 0.88 (rice); Extended Data Fig. 5), including over localized regions in our data representing both high and low values of average yields and cropping intensity (Extended Data Fig. We project climate impacts to yields across 24,378 global administrative regions until the end of the century. Projections include a high-emissions scenario (Representative Concentration Pathway (RCP) 8.5; Figs. 7), each matched to two socioeconomic scenarios, in each of 33 climate models and model surrogates that represent the full range of climate sensitivities (Methods). We account for statistical uncertainty through 1,920 Monte Carlo simulations per crop, following refs. a–f, Colours indicate central estimate in a high-emissions scenario (RCP 8.5), net of adaptation costs and benefits, for maize (a), soybean (b), rice (c), wheat (d), cassava (e) and sorghum (f) for 2089–2098. Projections computed for 24,378 subnational units relative to counterfactual yields, uncropped regions are shaded in grey. Wheat shows winter wheat and spring wheat projections combined, weighted by their area share in each region. Estimates in each location are ensemble means across climate and statistical uncertainty. 7 for a moderate-emissions scenario (RCP 4.5) and Supplementary Information, section J for results adjusted by CO2 fertilization. We report results based on where crops are cultivated today, noting that our approach also allows us to estimate theoretical impacts for a crop in locations in which it is not cultivated at present (Supplementary Fig. The projected effects of climate change on crop yields vary around the world (Fig. Regions exhibit different initial climates and socioeconomic conditions and they experience different changes in temperature, precipitation and incomes. These in turn alter the environmental factors crops are exposed to and the adaptive decisions of producers. Our projections are deviations from local baseline yield trends, which have historically been positive30,35 and will probably remain generally positive (see the ‘Conclusions and discussion' section). First, for all crops, temperature changes (through degree days and minimum temperature for rice and wheat) generally dominate the sign of local projected impacts. Precipitation strongly influences inter-annual variability in yields—which is important to producers, consumers and government planners—but does not generally drive overall trends. Global patterns in yield losses reflect the nonlinear response of crops to temperatures, with increasing extreme heat depressing yields and reductions in cold days increasing yields. Adaptive responses to rising average temperature moderate losses to extreme heat (for example, Extended Data Fig. 3a), consistent with producers taking protective measures in response to temperature extremes that are expected19. However, benefits from these protective measures are partially offset by decreased yield gains during moderate temperatures. The trade-off between temperature resilience and average yield has been documented and is understood to reflect physiological compromises across varietals19,36, although its potential global impact on future food production has not been previously demonstrated or quantified. A second general pattern across several crops applies to equatorial regions of the world with high rainfall, including central Africa, Southeast Asia and South America. In these regions, the benefits of moderate temperatures are amplified by increasing levels of precipitation, driven by an intensifying water cycle37. Although rising temperatures may ultimately reduce yields in these regions, the benefits of high rainfall in these regions partially offset those effects. Under a high-emissions scenario, our projected end-of-century maize yield losses are severe (about −40%) in the grain belt of the USA, Eastern China, Central Asia, Southern Africa and the Middle East (Fig. Losses in South America and Central Africa are more moderate (about −15%), mitigated in part by high levels of precipitation and increasing long-run precipitation (Extended Data Fig. Impacts in Europe vary with latitude, from +10% gains in the north to −40% losses along the Mediterranean. Gains in theoretical yield potentials occur in many northern regions in which maize is not widely grown (Supplementary Fig. The spatial distribution of soybean yield impacts is similar in structure to maize, although magnitudes are accentuated (Fig. 10 and 11); for example, about −50% in the USA and about +20% in wet regions of Brazil under a high-emissions scenario. High-emissions rice yield impacts are mixed in India and Southeast Asia, which lead global rice production, with small gains and losses throughout these regions. This regional result is broadly consistent with earlier work1. In the remaining rice-growing regions, central estimates are generally negative, with magnitudes in Sub-Saharan Africa, Europe and Central Asia exceeding −50% (Fig. Wheat losses are notably consistent across the main wheat-growing regions, with high-emissions yield losses of −15% to −25% in Eastern Europe, Western Europe, Africa and South America and −30% to −40% in China, Russia, the USA and Canada (Fig. There are notable exceptions to these global patterns: wheat-growing regions of Western China exhibit both gains and losses, whereas wheat-growing regions of Northern India exhibit some of the most severe projected losses across the globe. Cassava is projected to have uniformly negative projected impacts in nearly all regions in which it is grown at present, with the largest losses in Sub-Saharan Africa (−40% on average under a high-emissions scenario). Although cassava does not make up a large portion of global agricultural revenues, it is an important subsistence crop in low-income and middle-income countries. Thus, these yield losses may be a substantial future threat to the nutritional intake of the global poor (Fig. Sorghum losses are widespread in almost all of the main regions in which it is grown at present: North America (−40%), South Asia (including India) (−10%) and Sub-Saharan Africa (−25%). Projected gains emerge in Western Europe (+28%) and Northern China (+3%) (Fig. We compute aggregated global yield impacts based on the present distribution of global croplands, accounting for adaptation benefits and costs (Fig. For all crops except rice, we estimate that warming will likely reduce global yields by 2050 (probability of loss ranges from 0.701 (sorghum) to 0.946 (wheat), high-emissions scenario) after accounting for both statistical and climate model uncertainty. End-of-century uncertainty ranges may be substantial, with the largest 90% credible range for rice (q5 = −58.6%; q95 = 109.4%; RCP 8.5) and the narrowest range for wheat (q5 = −20.7%; q95 = −10.4%; RCP 4.5). a, Time series of projected climate change impacts on global average yields (area-weighted) in a high-emissions scenario (RCP 8.5), accounting for producer adaptations and adaptation costs. Box and whisker plots show total distribution of end-of-century projections accounting for joint statistical and climate model uncertainty (in each subplot: boxes, 25th to 75th percentiles; whiskers, 5th to 95th percentiles). To the right are projections adjusted for the estimated effect of CO2 fertilization from ref. 9 (shaded) and projections for moderate emissions (RCP 4.5) without CO2 fertilization (unshaded). See full results for moderate emissions in Supplementary Fig. 8, CO2 fertilization in Supplementary Information, section J, and uncertainty in Supplementary Figs. b, Projected end-of-century yield impacts, by deciles of present-day average temperature over 24,378 regions. Impacts across crops weighted within region by cropped area and caloric content; distributions across regions weighted by total cropped area. c, As in b but by decile of present-day income. d, Empirical end-of-century global damage function describing calories (kcal) lost as a quadratic function of global mean surface temperature anomaly (ΔGMST). Each point represents a single climate-model-by-Monte Carlo run for RCP 4.5 (blue) or RCP 8.5 (red) in 2093–2097, including gains from CO2 fertilization. Bottom panel shows the distribution of warming under each RCP across 33 climate models and model surrogates (Supplementary Information, section G). Box plots to the right show distribution of damages collapsed to RCP. Right axis describes calorie losses normalized by 2015 global calorie production for the six crops studied here. Projected log(yield) impacts for panels a–c winsorized at the top and bottom 1% of the impacts distribution over all region–GCM–years, by RCP–crop, then converted to percentages. Although the impact of heat on yields is nonlinear and pronounced for many crops (Extended Data Figs. 3 and 4), global aggregate yields are not driven downward most strongly by the hottest regions of the world. Accounting for adaptation, we find that the middle 50% of regions with moderate average temperatures tend to suffer the largest yield losses (Fig. This is largely because hotter locations are already more adapted to heat, so further warming has a reduced impact, whereas cold locations benefit from warming. Furthermore, high average rainfall, prevalent in many hot locations, is linked to increased gains from moderate warming (Extended Data Figs. This finding underscores the importance of accounting for real-world patterns of differentiated adaptation in global-scale analyses. Our projections suggest that changes in global yields affect populations around the world unequally. We estimate that total calorie production is generally affected more heavily by climate change in regions that are richer today (Fig. 3c), along with the lowest-income decile owing to its reliance on cassava. We estimate average losses of 28% in the lowest-income decile but more moderated losses of roughly 18% across deciles 2–8. In the highest-income deciles, average losses increase to 29% (ninth) and 41% (top). This result is partially because lower-income populations tend to live in hotter climates, in which present adaptation rates are higher, and in the tropics, in which high average precipitation reduces warming impacts (Fig. This has important implications for global damages, as high-income regions include many of the world's breadbaskets. Because relative yield losses are greatest in regions in which modern agriculture is concentrated, they have amplified influence on global caloric production under climate change. We combine our projections across all six crops to estimate the total potential impact of warming on global caloric production (Methods and Supplementary Information, sections B and K). Following the approach in refs. 28,29,34, we index lost calorie production against ΔGMST for each climate model. We recover a damage function describing the joint distribution of ΔGMST and lost calorie production for these six staple crops in each 5-year moving window of the twenty-first century (Fig. We estimate that the magnitude of impacts to present croplands increases nearly linearly at a rate of −5.54 × 1014 kcal (P < 0.001, 95% credible interval −5.63 to −5.44 × 1014 kcal) of calorie production per +1 °C in GMST in 2100 (see Supplementary Fig. 21 for other decades), amounting to approximately −121 kcal per day per 1 °C per person based on Shared Socioeconomic Pathway 3 (SSP3) 2100 global population projections, or −4.4% of present per capita recommended consumption per 1 °C. (For example, warming of 4 °C in GMST in 2100 would correspond to a loss of about 17.6% of present per capita recommended consumption). In contrast to previous global analyses7,8, projections here account for producer adaptation rates that reflect observed behaviour. We find that accounting for climate adaptation, its costs and rising incomes have substantial impact on high-emissions projections for all crops and regions (Table 1). We estimate that development and adaptive adjustments reduce global calorie impacts (relative to a ‘no adaptation' scenario) by roughly 23% in 2050 (column 1a versus 1b, Table 1, 6% for moderate emissions) and 34% at the end of the century (column 2a versus 2b, 12% for moderate emissions), owing to the growing extent of adjustment under greater climatic change. The relative impact of adaptation is largest for rice (79% reduction of end-of-century impacts, 86% for moderate emissions) and smallest for wheat (statistically unchanged) (Supplementary Tables 10 and 11). Adaptation and development exacerbate average wheat losses from climate change (Supplementary Figs. 4 and 5 and Supplementary Tables 10 and 11) because wheat producers are observed to take on further weather-related risk as GDP per capita rises (Supplementary Information, section G), resulting in small losses from adaptation and development for Europe and Oceania (Table 1). By contrast, the largest regional benefits of adaptation accrue to South America (61% reduction of end-of-century impacts, 23% for moderate emissions), mostly from maize and soybean adaptation gains. Our results indicate that producer adaptations are likely to have strong influence over climate change impacts to agriculture, highlighting the critical importance of accounting for these adaptations; however, our central estimates for the overall impact of climate change on potential production of staples remain negative and meaningful, even accounting for the net effects of adaptation. Substantial uncertainty persists in these estimates, originating from both econometric uncertainty and climate model uncertainty, such that overall risks should be assessed across the full range of projected impacts. The above results do not account for CO2 fertilization, which is challenging to measure empirically. However, we adjust our results post-estimation to incorporate CO2 fertilization using previous estimates9 (Methods and Supplementary Information, section J). Adjusting for CO2 fertilization does not qualitatively alter the structure of our findings (Supplementary Figs. 13–18) but it does reduce the central estimate for end-of-century yield losses by 5.0– 9.5 percentage points (Fig. 3a) and increases the likelihood of positive aggregate effects. We do not account for the decreased crop nutrient content that might result from CO2 fertilization38. Losses to future global yields can be incorporated into present-day climate policy, for example, by computing the net present value of future economic harms that result from the present emission of an extra ton of CO2, a value known as the social cost of carbon (SCC). We compute a component of the SCC—a ‘partial SCC'—that results from changes in global yields by integrating these projections into the Data-driven Spatial Climate Impact Model (DSCIM) developed in refs. 28,29 (Methods and Supplementary Information, section K). We assign country-specific average prices to total calorie production (Supplementary Fig. 20), allowing producers to intensify/reduce production to meet demand in their market at increasing marginal cost (Methods, Supplementary Information, section K, and Supplementary Fig. In general, aggregate yield losses under climate change harm consumers as caloric consumption is lower and prices are higher; producers are also harmed by production losses, but these may be offset by gains from higher prices. In all scenarios, our approach implicitly allows the spatial distribution of within-crop varietals to adjust optimally within present cropped locations (Extended Data Fig. 1, Methods and Supplementary Information, section B). In some further scenarios, we deflate these estimates to account for the roles of crop switching and international trade, which might cause the spatial distribution of planting areas to shift (Methods and Supplementary Information, section K). Across different modelling assumptions, the partial SCC for the crops we study ranges from $0.99 per ton of CO2 (5% constant discounting; adjusted for flexible crop switching, movement of cropped land and international trade; moderately elastic demand and supply; frictionless trade within countries; RCP 8.5) to $49.48 per ton (Ramsey discounting39; crop type and land distributions mirror present day; moderately elastic demand and supply; frictionless trade within countries; RCP 8.5). See Extended Data Table 3 and Supplementary Table 13 for all partial SCC estimates. Our analysis of staple crops is, to the best of our knowledge, the first to quantify the adaptations of real-world producers to climate around the world and to apply these measurements to projections of global agricultural damages from climate change. Our econometric results complement a well-developed literature from process-based models that resolve the underlying mechanics of crop production and some important forms of adaptation but which take producer decision rules as exogenous7,8,18,20. Our results indicate a substantial and statistically significant rate at which ΔGMST reduces the ability of present global food systems to produce calories, net of adaptation. However, in the absence of adaptation, we project that agricultural outcomes would be materially worse. Our findings indicate that projected consumption losses tend not to be evenly distributed across global populations. Wealthy regions of the world more easily absorb grain price shocks. In poor regions of the world, food shortages and associated price shocks may be more destabilizing. This suggests that future modelling efforts would benefit from representing realistic costs and benefits of adaptation, imperfect information and other aspects of producer decision-making. We expect that these changes would lead to projections of climate impacts to agriculture that are less optimistic than those assuming agronomically optimal management7 and more optimistic than models that do not model adaptation at all2. A key finding is that global populations exhibit extensive adaptation to climate already, especially in relatively low income and hot regions of the world—with the exception of the world's poorest, who depend heavily on cassava and face higher potential losses (Fig. We also observe that breadbaskets of the world, in which the climate is moderate, exhibit more limited adaptation at present. Because such a large fraction of agricultural production is concentrated in these wealthy-but-low-adaption regions, they dominate projections of global calorie production, generating much of the global food security risk we document. This result is consistent with earlier regional findings that US crop systems are optimized for high average yields but not robustness to climatic changes36 and that projected maize losses in Sub-Saharan Africa are largest in the most productive and climatically moderate countries, whereas countries with hotter present-day climates have lower historical yields and smaller projected yield losses40. Our projection of productivity declines under climate change contrasts with process-based model results that are calibrated to experimental farms, use researcher-defined adaptation rules and generally tend to indicate productivity gains globally (ref. 8 is an important exception). For example, ensemble mean projections of AgMIP process-based models indicate end-of-century productivity gains for maize (+1.3%, RCP 8.5), wheat (+9.9%), soybean (+15.3%) and rice (+23.3%)7. These differ substantially from the generally negative impacts we predict, with central end-of-century estimates that range from −35.6% (soybean) to −6.0% (rice) (Supplementary Table 10; RCP 8.5), although both sets of projections exhibit large uncertainties. A comparison of model skill over historical data suggests that the statistical approach of this paper outperforms the AgMIP ensemble across crops and locations (Extended Data Tables 1 and 2 and Supplementary Information, section F) and accurately captures observed variation in local yields (Extended Data Figs. These differences highlight the importance of using globally representative data and accounting for empirically measured rates of adaptive behaviours in projections of climate change damages. Unlike process-based crop models, our estimates do not provide insight about the agronomic mechanisms through which climatic changes influence future yields. Such insight is useful in guiding agricultural innovation and, indeed, process-based models are used at present in a Genotype by Environment by Management (GxExM) framework to target crop science research about future climate stresses41,42,43,44. The work we present underscores the urgency and importance of such crop technology developments and the importance of ensuring access to adaptive technologies by global producers broadly. Future producer adaptation to climate change may differ from the historically observed patterns we recover. For example, technological breakthroughs, such as the industrial fixation of nitrogen, transformed what is understood about the limit of global productivity. However, the repeatability and penetration of these innovations has been uneven and perhaps should be interpreted cautiously—for example, synthetic fertilizers remain incompletely deployed despite their invention long ago45 and progress towards heat tolerance for some staple crops has stalled46. Furthermore, to the extent that these advances are correlated with income, our projections implicitly account for them. Notably, innovation has contributed positively to average yield trends in many regions30,35, which will probably continue into the future in some form. Our projected yield impacts should therefore be interpreted as deviations from a future trend in average yields that is driven by other factors, including innovation. Indeed, our findings provide a sense of the scale required of such innovations to maintain global food security. Finally, our monetization of agricultural damages from climate change is limited, as it does not capture the impact of all future changes, such as linkages between climate change, food availability and social instability10,15. Nonetheless, synthesizing the complex global response of agriculture into a partial SCC is an important step towards representing these damages in climate policies47. Comparison of our partial SCC with existing estimates provides context for these findings. The Framework for Uncertainty, Negotiation and Distribution (FUND) model reports a partial SCC for agriculture of −$2.70 per ton CO2 (ref. 48) using a 3% discount rate (that is, warming improves global welfare through agriculture). Because this value relies on older (pre-2000) studies for calibration49, Moore et al.9 construct a revised estimate based on a meta-analysis of previous estimates8, mostly from process-based models. Inserting these projections into a computable general equilibrium model for 16 regions, Moore et al.9 estimate a partial SCC of $13.05 per ton (adjusted to the USD in 2023), opposite in sign from the original FUND partial SCC. Because this estimate is derived from process-based models, it does not account for observed rates of producer-level adaptation and it is two to five times larger than the range of our comparable estimates, $3.08–$6.84 (3% discount rate; Extended Data Table 3), that do account for autonomous adaptation. We note that, in a recent paper50, researchers incorporate the Moore et al.9 results directly into the Greenhouse Gas Impact Value Estimator (GIVE) integrated assessment model and find that climate damages to the agricultural sector increase to $84. The discussion of that paper focuses on other aspects of the analysis and does not detail why this estimate changed. Also, neither Moore et al.9 nor this study account for non-staple crops or livestock, an important area for future work, and neither estimate accounts for the unequal impact of crop losses on the global poor (Fig. 3c), for whom lost agricultural revenues and/or consumption may be particularly damaging10,14,15. In continuing work51, we seek to more fairly represent these unequal costs in global estimates of monetized damages. This section provides an overview of the data and methods used to estimate the relation between agricultural yields and variation in weather, accounting for adaptation to climate and economic development. Subnational yields, production and harvested area, primarily at the second administrative level, were collected from the statistical offices of 54 countries (Supplementary Fig. This includes data that were generously shared for Burma, Cambodia, Laos, Malaysia, Thailand52 and Zambia53. This yields a dataset for 6 crops in 12,658 locations (41,186 location–crop pairs) spanning up to 137 years. Wheat varietals, when not identified in the data, were manually assigned to spring or winter wheat based on ancillary country-specific data (Supplementary Table 6). All growing season definitions are from ref. The main weather dataset used in this analysis is the Global Meteorological Forcing Dataset (GMFD v1)55. Data are available on a 0.25° × 0.25°-resolution grid from 1948 to 2010. We obtain daily maximum and minimum temperatures and daily total precipitation for all grid cells globally. These data provide surface temperature and precipitation information using a combination of observations and reanalysis, which is downscaled and bias-corrected using several station-based observational datasets to remove biases in monthly temperature and precipitation. We use the NASA Earth Exchange (NEX) Global Daily Downscaled Projections (GDDP) dataset for future projections of climate change. This comprises 21 climate projections, which are downscaled to 0.25° × 0.25° resolution56 from global climate model (GCM) runs in the Coupled Model Intercomparison Project Phase 5 (CMIP5) archive57. NEX-GDDP uses GMFD55 and quantile mapping to adjust the GCM outputs (means and variances) in historical and future time periods so that the systemic bias of the GCMs is removed. In contrast to other approaches58, NEX-GDDP does not assume a common distribution shape between observed (GMFD) and GCM data. However, NEX-GDDP only assumes that GMFD (versus GCM) variances are correct, which other approaches58 do not require. Climate is projected under emissions from Representative Concentration Pathways 4.5 and 8.5 (RCP 4.5 and RCP 8.5) up to 2100 (refs. 59,60), although the NEX-GDDP outputs of some models lack precipitation data past 2098, so that the ‘end of the century' in this paper is the year 2098. The CMIP5 ensemble of GCMs is not a systematic sample of possible futures. To provide an ensemble of climate projections with a probability distribution of GMSTs consistent with that estimated by a probabilistic simple climate model, we use the method outlined in ref. A full list of models and their weights is given in Supplementary Table 9. For a more complete description, see ref. We obtain national and subnational income data for 1,503 administrative regions from 83 countries from ref. 62. Data are provided by ref. 62 at the state/province level for each country. We use these subnational data to allocate national GDP data from the Penn World Tables (PWT) database (https://www.rug.nl/ggdc/productivity/pwt/) under the assumption that the within-country distributions of GDP recorded in ref. 62 are accurate, but the exact levels may not be. Using these data, we construct a consistent panel of subnational incomes for all areas in our crop dataset that sum to the national GDP from the PWT database for all countries in the sample. Future projections of national incomes are derived from the Organization for Economic Co-operation and Development (OECD) ENV-Growth model63 and the International Institute for Applied Systems Analysis (IIASA) GDP model64, as part of the ‘socioeconomic conditions' of the Shared Socioeconomic Pathways (SSPs)65. These are the only models that provide GDP per capita projections for a wide range of countries in the SSP database. The SSPs propose a set of plausible scenarios of socioeconomic development over the twenty-first century in the absence of climate impacts and policy. Data are interpolated to annual frequency and downscaled following ref. We obtain data on the area equipped for irrigation from AQUASTAT of the Food and Agriculture Organization of the United Nations (FAO)66. These data are a single cross-section centred around the year 2000 at 0.08° × 0.08° resolution and we hold the values fixed in climate projections. (Increases in incomes and heat exposure may drive increases in future irrigation, whereas aquifer and reservoir drawdown may drive decreases in future irrigation). We develop a reduced-form approach to modelling crop yields and producer adaptation, allowing us to recover plausibly causal effects of weather on yields based on data from real-world farms, without having to observe all of the biophysical processes in a given crop that mediate these effects. The approach allows us to account for the total costs and benefits of numerous real-world adaptive adjustments that producers undertake, without requiring that we observe or model each adjustment explicitly. This approach builds on ideas and techniques developed in refs. 4,19,26,27, using the method for capturing the benefits of adaptation in ref. 28 and accounting for its costs based on the revealed preference approach in ref. 29. Following common practice in the literature2, our implementation also allows our model to recover local, micro level and instantaneous nonlinear effects of environmental conditions, even though outcome data are only observed at more aggregate spatial scales and temporal frequencies28,31. We apply cross-validation techniques common in machine learning but adapted for model selection in a causal inference context. Our approach has three stages: (1) we remove variation that is associated with non-parametric controls to condition out the influence of unobservables; (2) we apply cross-validation to this residualized data to select weather measures; and (3) we again apply cross-validation, modifying the model chosen in the previous step to select dimensions of climate that matter for adaptation. We perform the sequence (1)–(3) for each of the six crops separately to identify the most suitable model for each crop. Having selected and fitted an empirical model that accounts for weather and adaptation, we then apply the model in projections of climate change impacts. Each of these steps is detailed below. A large number of candidate weather variables are constructed from raw weather data based on previous literature5,67,68,69,70,71. We assemble variables that previous analyses report were influential for at least one crop, although individual variables might not ultimately be included in the specification for all or any crops. All weather transformations were conducted at the pixel level using daily data before aggregation, unless otherwise specified. Encoding pixel-by-day weather data after nonlinear transformations allows a high degree of flexibility in how local daily weather conditions can have an instantaneous and nonlinear impact on crops28,31. This means that even short periods of environmental stress are resolved and captured by these variables and the model. Degree day terms were constructed following ref. 5, with kink points selected for each crop by a tenfold cross-validation search over potential kinks. Growing-season-phase precipitation was summed within month by grid cell and then transformed70. Construction of vapour pressure deficit followed ref. 69. The count of rain days was calculated by grid cell as days with positive precipitation68. Extreme rain was identified by constructing the 1980–2010 rainfall distribution for each grid cell in the GMFD daily precipitation data and aggregating the amount of rainfall or number of days of rainfall in excess of the 95th percentile of this distribution68. Count of extreme rain days was measured following extreme rain except grid cell days were counted instead of summed within grid cell68. Drought was defined as a binary indicator over administrative unit growing season precipitation, using a tenfold cross-validation search over percentile cut-offs in the administrative unit growing season total precipitation distribution, largely following ref. 71 (final cut-offs ranged from the 8th to 25th percentiles). Minimum temperatures were calculated as the average of daily grid cell minimum temperatures within each month, following ref. 67. For all weather terms, weighting and aggregation to administrative units were done as described in ref. 29 with cropped area weights from ref. 54. See Supplementary Information, section D and Supplementary Tables 2–5 for full details on the construction of each weather variable. We first select non-parametric controls for unobservable factors to isolate plausibly random variation in our set of weather measures5,22,31. Specifically, we use administrative unit i (typically county), fixed effects \({\mu }_{i}\), absorbing time-invariant confounds such as soil quality; country–year fixed effects \({\rho }_{ct}\), absorbing country-by-year price and trade policy confounds; and state or province quadratic time trends \({h}_{s}(t)\), absorbing slow-moving, local confounds such as the diffusion of new technologies or local variation in prices (equation (1), Supplementary equations (B.12) and (D.1)–(D.6)). We separately partial out the variation in yields yit and the vector of regressors xit on these non-parametric controls (Supplementary Information, section B), leaving us with residual variation \({\widetilde{y}}_{it}\) and \({\widetilde{{\bf{x}}}}_{it}\) for our cross-validation procedures. We implement cross-validation over ten random folds to systematically evaluate the importance for yields of the broad range of weather measures previously described in the literature. Each potential model is required to have at least one temperature term and one moisture term. (Temperature and precipitation are known to be correlated, so causal effects estimation of one without controlling for the other is vulnerable to bias5. Vapour pressure deficit was treated as a moisture term. Note: these terms may affect moisture demand and/or soil moisture supply). Weather terms are each interacted with a fixed set of covariates: growing season average daily maximum temperature (‘long-run temperature', \(\overline{T}\)), growing season average monthly precipitation (‘long-run precipitation', \(\bar{P}\)), log(GDPpc) (‘income', \(\bar{I}\)) and the share of cropped area equipped for irrigation (‘irrigation', \(R\)). This ensures that weather variable selection occurs using models that are at least as flexible as those selected after the second cross-validation step, without having to simultaneously explore the full set of potential combinations of weather measures and covariate interactions (which would number 103,904 separate models). In selecting a candidate set of weather terms over which to study adaptation, we seek to optimize over parsimony and explanatory power, selecting a high-performing model in which additional weather terms do not individually contribute substantial fit gains (see Supplementary Tables 7 and 8). Not all weather terms can be reliably projected by climate models72,73,74 and models containing these candidate variables were explored in the cross-validation procedure (Extended Data Fig. 2) but ultimately excluded in projections of climate change because they could not be projected with reasonably high confidence. Overall, this procedure evaluates the out-of-sample (OOS) performance of candidate models that include all combinations of weather variables that we draw from the literature and allows for varying degrees of nonlinearity (Fig. In our second cross-validation step, we select a set of covariate interactions for which covariates reflect the factors that mediate producer adaptation actions4,19,26,27,75,76 (equation (B.7)). We take as given the set of weather parameters selected from the first step, allowing for interactions between weather variables and income, irrigation, \(\bar{T}\), \(\bar{P}\) and/or \(\bar{T}\times \bar{P}\). Further, we allow for the nonlinear precipitation response to vary over phases of the growing season or to be constant over the growing season, with ‘phases' of the growing season defined by a systematic model search (Supplementary Table 4). Finally, we also allow for interaction terms between degree day responses and monthly precipitation. Accounting for mediating effects of income and irrigation is important because access to resources alters producer actions through several channels—such as access to technology77,78,79,80,81, insurance and credit markets77,80,82,83, inputs such as fertilizer77,84 or labour markets77,79,81 or information85. Notably, this minimizes confounding in our estimates, as resources and climate tend to be correlated (for example, poorer nations tend to be located in the tropics86) and it allows our model to capture that climate and income jointly determine yields87. Further, this allows us to project future changes associated with economic development. In this step, we use cross-validation with ten folds over random blocks of data grouped by state/province. Here the test of model performance is the ability to model yields for a set of states or provinces that are held out of the data. This matches one of the intended uses of our interaction surface, which is to model the heterogeneous response of yields to weather for locations with no yield data available. In our climate projections, long-run average precipitation (\(\bar{P}\)) values range more than 17 standard deviations outside the mean in our historical data. To prevent extrapolating adaptation behaviour to \(\bar{P}\) values far outside our data, we imposed a single-knot linear spline in \(\bar{P}\) for each weather measure (\({z}_{a}(\bar{{P}_{i}})\), \({z}_{b}(\bar{{P}_{i}})\) and \({z}_{c}(\bar{{P}_{i}})\) below). These knots were chosen using a within-R2 search over knot locations (Supplementary Information, section D). See Supplementary Information, sections B–D for further estimation details. The final maize specification is: in which yit is log(yields) in administrative unit i in year t, \({{\rm{DD}}}_{it}^{8-31}\) represents degree days from 8 °C to 31 °C, which is interacted with each of average growing season daily maximum temperature (\({\bar{T}}_{i}\)), splines in average growing season total precipitation (\({z}_{[\cdot ]}({\bar{P}}_{i})\)), the interaction between \({\bar{T}}_{i}\) and \({z}_{[\cdot ]}({\bar{P}}_{i})\), average log(GDPpc) (Ii) and area equipped for irrigation (Ri). The remaining weather terms have similar interactions. μi, hs(t) and ρct are as described previously. Standard errors of all estimated parameters allow for the correlated nature of yields (and weather) across space and over time. Specifically, we estimate cluster-robust standard errors88, which allow arbitrary correlations across observations over time within first-level administrative units (that is, autocorrelations within and between counties within a given state) and across all observations within a given country–year. Note that the terms in equation (1) reflect rows (weather variables) and columns (mediating variables) illustrated in Extended Data Fig. See Supplementary Information, section D, for full details of the final model specification for each crop. We note that the overall fit of the above empirical model to the historical data is high, with an R2 of 0.85 for maize. For reference, the seminal analysis of maize by Schlenker and Roberts5 report a model R2 of 0.77 when modelling yields for only the Eastern United States, for which data quality is high relative to much of the globe and the underlying patterns of producer behaviour are probably more homogeneous across locations. We are able to empirically observe the benefits of adaptive management among producing regions because adaptation decisions have costs19,26. If adaptations were free, then we would expect all regions to use uniform management practices and varieties using only the most climate-resilient technology available, as this strategy would convey benefits at no cost19,29. To capture the full impact of climate change on agricultural systems, it is therefore essential that these costs of adaptation are accounted for, as well as adaptation benefits, as these costs would not be incurred in the absence of climate change. We cannot observe these costs directly but they can be inferred under the assumption that profit-maximizing producers undertake adaptations for which the benefits exceed costs, but not those for which costs exceed benefits29. This is referred to as a ‘revealed preference' approach and is widely used in economics research89. Specifically, we follow the approach developed in ref. 29 to track these costs for each crop and in each region until 2100 (see Supplementary Information, section H, for a derivation, Supplementary Fig. 6 for intuition and Supplementry equations (H.6) and (H.7) for a detailed example). 19 and deduct these costs from yield changes to develop a complete measure of climate change impacts29. Thus, our projections of climate change impacts on yields should be interpreted as capturing both the costs and the benefits of voluntary producer adaptations in each location. Intuitively, consider the term \({\gamma }_{6}{\bar{T}}_{i}\) in equation (1), through which long-run average temperature in region i (\({\bar{T}}_{i}\)) modulates the negative effect of extreme heat shocks (\({{\rm{DD}}}_{it}^{31+}\)) on yields. A 1 °C increase in \({\bar{T}}_{i}\) flattens the negative extreme heat slope by γ6 log points per extreme heat degree day—a benefit of adaptation in hotter climates. If a producer in this example expects ten extreme heat degree days during the growing season, then the expected adaptation benefit for a 1 °C increase in \({\bar{T}}_{i}\) would be 10 × γ6 log points of yields. This benefit must not be costless, otherwise producers would have already adapted away all extreme heat yield losses. We thus infer marginal adaptation costs as equal to the marginal adaptation benefits that we estimate (an implication of assuming that producers maximize profits; see Supplementary Information, section H for details). Many correlated aspects of weather affecting crop biophysical processes have been separately studied2,5,36,67,68,69,70,71,101,102,103 and we evaluate which are important drivers of the biophysical processes affecting yields of each crop. We use cross-validation to evaluate more than 8,000 candidate empirical models across all six crops, for which each model predicts residual changes in historical crop yields using a different combination of weather variables proposed by previous studies5,67,68,69,70,71. On the basis of OOS performance, we find that nonlinear degree days are the most important determinant of yields, on average across crops, by a considerable margin (Extended Data Figs. Degree days increases model OOS fit (Supplementary Table 7) across crops by 51.7% on average (relative to the second-best model), with the largest gain for sorghum (101.6%, a doubling of model OOS fit) and the smallest gains for rice and cassava (9.1% and 9.4%). In general, exposure to cool daily temperatures (<9 °C for wheat and <29–31 °C for all other crops) modestly increase yields, whereas daily temperatures above these thresholds cause sharp declines in yields; for example, lowering end-of-season maize yields by −5% for each day that shifts from a 25 °C to a 40 °C day (Fig. This finding generalizes previous results from the USA (for example, see ref. Seasonal precipitation is the second most important determinant of yields across all crops, to the exclusion of a wide range of other candidate weather variables put forward in the literature (Extended Data Figs. Nonlinear measures of seasonal precipitation increase model OOS fit across crops by 14.5%, with the largest gains for rice and maize (22.8% and 22.2%) and the smallest gains for wheat, soybean and sorghum (7.8%, 9.2% and 9.4%; Supplementary Table 8). Across crops, increasing seasonal monthly precipitation raises yields on average 0.2% per mm at low rainfall levels (<100 mm per month), has little effect at intermediate levels (250–300 mm per month) and reduces yields −0.6% per mm at higher levels (>400 mm per month; Extended Data Figs. In contrast to reports from individual empirical studies (for example, refs. 67,68,69,71), we find that monthly average minimum temperatures (tmin), vapour pressure deficit, drought, the count of rain days, the count of extreme rain days and the amount of extreme rain generally only contribute modest gains to yield model fits (average % gains in OOS fit range from 1.2% to 3.6% excluding drought for cassava, discussed below). Minimum temperatures are important predictors for rice and wheat yields but not other crops (Extended Data Fig. There are exceptions for individual crops. Cassava yields are generally insensitive to weather overall (the maximum OOS variance predicted is 2.3%), motivating its cultivation in contexts in which the climate is not conducive to other staple crops that are more sensitive104. We find that growing season drought is the most important factor for predicting cassava yields (24.0% gain in variance predicted; Supplementary Table 7). Drought is also important for rice yields, as is the count of rain days (6.4% and 8.0% gain in variance predicted; Supplementary Table 7). The count of rain days is also moderately important for maize yields (6.6% gain in variance predicted; Supplementary Table 7). Taken together, these results indicate the importance of nuanced projections of future precipitation distributions for a restricted set of global staple crops. We find that producers adapt to higher temperatures such that, across crops and around the world, yields are systematically less sensitive to daily temperature and generally more sensitive to seasonal precipitation in locations that are hotter on average (Fig. Similarly, yields are less responsive to temperature and more responsive to precipitation in locations that are drier on average (Extended Data Figs. Because climate-associated adaptations influence numerous—possibly correlated—aspects of weather–crop responses, we estimate the mediating influence of all dimensions of climate simultaneously on all dimensions of weather for each crop, generating a multidimensional function describing these interacting relationships (Extended Data Figs. This function serves as the basis for projecting adaptation to climate change (Methods and Supplementary Figs. 4 and 5) and has not been previously represented in global process-based models7,8. We also find that rising incomes enable producers to respond to environmental conditions in important but varied ways, thereby mediating the effects of different climates. For example, maize and sorghum yields become more sensitive to temperature as incomes rise, whereas rice and cassava become less sensitive (Extended Data Figs. These differences could be because of differences in how access to improved technologies, infrastructure, banking and credit or insurance77,78,80,82 influence producer strategies77,83. Notably, however, we find that higher incomes and access to irrigation are associated with greater resilience to precipitation extremes across all crops (Extended Data Figs. 1b, 3g and 4g), consistent with access to improved irrigation technologies increasing with income77,80. We integrate our econometric results with the DSCIM28,29 to develop projections and partial SCC estimates (Supplementary Information, section K.5). Calculations were conducted as described in refs. 28,29, over 24,378 geographically granular regions, 33 climate models and 1,920 Monte Carlo draws of statistical uncertainty per crop. 29, we project the impacts of climate change net of adaptation and its costs, relative to a counterfactual in which incomes grow but the climate is held fixed. Projections in percentage terms represent weighted averages of log yield estimates, converted to percentages. Crop impacts are aggregated over space using cropped area weights (Supplementary Information, section A), over climate models using surrogate model mixed ensemble61 weights (Supplementary Table 9) and over crops using calorie weights90. Yield gains from CO2 fertilization are difficult to empirically quantify91,92,93,94 but we adjust our projections to incorporate CO2 fertilization using previous estimates9. Yield projections are robust to restricting future weather projections to be bound within the support of the historical data (Supplementary Table 12 and Supplementary Fig. See Supplementary Information, section G for full details of the projection methodology. To calculate a partial SCC, we must value climate-induced changes in yields for both producers and consumers (Supplementary Figs. Our approach to this valuation follows ref. 95 and models consumers and producers of calories with constant elasticity of demand and supply, drawing elasticities from the literature95,96,97. We allow for frictionless crop trade within country but do not allow for trade between countries. We test sensitivity to this assumption by allowing frictionless trade between countries within continents and throughout the globe. We consider the potential impacts of crop switching, shifting of cropped areas and international trade by drawing on the previous model-based literature75,98,99,100. In calculating the SCC, we follow ref. 28 to calculate results for constant discount rates (2%, 2.5%, 3%, 5%) and refs. 28,47 for Ramsey discounting (ρ = 0, η = 2). See Supplementary Information, section K, for full details of the valuation methodology. A challenge to this analysis is the limited way in which future resource access has been projected105, that is, relying on standardized scenarios contained in the SSPs65. Numerous types of resources probably influence how staple crops are managed. However, we restricted our analysis to consider only the mediating effects of income and irrigation. We treat income as a proxy measure for access to resources associated with economic development, such as fertilizer or extension services. These variables are intentionally omitted from regression models so that their influence will be captured by the income variable, allowing them to be represented in standard socioeconomic projections for which income is projected and available to us, but these other variables are not65. Mediating effects of irrigation are accounted for explicitly because they are influential and irrigation presence is dependent on groundwater geology, local topography and surface water flows, which are not well captured by income nor climate measures. An important area for future work is to develop projections of irrigation for use in crop yield projections. Another challenge is that our measures of climate used to estimate adaptation are cross-sectional; thus, correlations between climate and omitted variables that determine the shape of the yield–weather response could potentially bias our estimates of adaptation. If such correlations exist, they might be regional in nature or otherwise only affect limited subsamples of our data. As a partial test of this possibility, we examine non-parametric estimates of the association between climate and temperature sensitivity and find that the associations we rely on are systematic throughout the data (Supplementary Fig. This suggests that any unknown omitted variable would have to be consistently correlated with climate throughout our sample to influence our analysis. We are able to account for some, but not all, potential shifts in crop growing seasons under future climate change. Our parameterization of adaptation implicitly captures effects of shifting the harvest date earlier in the growing season (for example, to avoid late-season heat exposure; see Supplementary Information, section B). However, we cannot capture the effects of planting dates that may shift to a period before the start of our growing season windows nor can we capture crop switching in our yield models (although we do capture it in the valuation of impacts; see Supplementary Information, section K). We note that such shifts may not depend only on temperatures but also on rainfall quantity and timing, which are not yet well modelled by existing GCMs72,73,74. These are forms of adaptation that some process-based models explicitly include7,8,9,18,106,107,108. Furthermore, unlike process-based models, our reduced-form approach by design does not capture the agronomic mechanisms by which climatic changes influence future yields. Such mechanistic insights can be used to guide crop research in expectation of future climate stresses41,42,43,44, highlighting an important trade-off in the two methodological approaches (Supplementary Information, section B.1). For precipitation effects, we do not project some extreme precipitation effects because these precipitation extremes are not well captured by the existing suite of GCMs72,73,74. We highlight this as an area of need for future research from the climate modelling community, with potentially important implications for our further understanding of climate change impacts to global food security. Also, we note that our valuation of agricultural damages from climate change and the partial SCC only accounts for the crops in our sample, which account for a large share of global calories but a smaller share of total agricultural revenues. The present analysis does not include effects of climate change on other sources of agricultural revenue, such as livestock, citruses and other specialty crops. We look to future work to incorporate these output categories. Replication data (including historical yield and weather data) and code for the main paper figures and table are available on our Zenodo repository at https://zenodo.org/records/14511340 (ref. 110). Instructions for downloading and running the replication code are provided on the repository website. Lobell, D. B. et al. Prioritizing climate change adaptation needs for food security in 2030. Auffhammer, M. & Schlenker, W. Empirical studies on agricultural impacts and adaptation. Mendelsohn, R., Nordhaus, W. D. & Shaw, D. The impact of global warming on agriculture: a Ricardian analysis. Butler, E. & Huybers, P. Adaptation of us maize to temperature variations. Schlenker, W. & Roberts, M. J. Nonlinear temperature effects indicate severe damages to US crop yields under climate change. Burke, M. & Emerick, K. Adaptation to climate change: Evidence from us agriculture. Rosenzweig, C. et al. Assessing agricultural risks of climate change in the 21st century in a global gridded crop model intercomparison. Challinor, A. J. et al. A meta-analysis of crop yield under climate change and adaptation. Moore, F. C., Baldos, U., Hertel, T. & Diaz, D. New science of climate change impacts on agriculture implies higher social cost of carbon. Burke, M., Hsiang, S. M. & Miguel, E. Climate and conflict. & Hsiang, S. M. Social and economic impacts of climate. Barros, V. R. et al. (eds) Climate Change 2014: Impacts, Adaptation, and Vulnerability. Part B: Regional Aspects. Contribution of Working Group II to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change (Cambridge Univ. Climate change and global food systems: potential impacts on food security and undernutrition. Baker, R. E. & Anttila-Hughes, J. Characterizing the contribution of high temperatures to child undernourishment in Sub-Saharan Africa. & Schlenker, W. Asylum applications respond to temperature fluctuations. Agricultural adaptation to climate change in rich and poor countries: current modeling practice and potential for empirical contributions. Stocker, T. F. et al. (eds) Climate Change 2013: The Physical Science Basis. Contribution of Working Group I to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change (Cambridge Univ. The Global Gridded Crop Model Intercomparison phase 1 simulation dataset. Roberts, M. J., Braun, N. O., Sinclair, T. R., Lobell, D. B. & Schlenker, W. Comparing and combining process-based crop models and statistical models with some implications for climate change. & Asseng, S. Comparing estimates of climate change impacts from process-based and statistical crop models. & Greenstone, M. The economic impacts of climate change: evidence from agricultural output and random fluctuations in weather. On the use of statistical models to predict crop yield responses to climate change. The critical role of extreme heat for maize production in the United States. Consistent negative response of US crops to high temperatures in observations and crop models. Hsiang, S. M. & Narita, D. Adaptation to cyclone risk: evidence from the global cross-section. Deryugina, T. & Hsiang, S. The marginal product of climate. National Bureau of Economic Research (NBER) Working Paper 24072. https://www.nber.org/papers/w24072 (2017). Rode, A. et al. Estimating a social cost of carbon for global energy consumption. Valuing the global mortality consequences of climate change accounting for adaptation costs and benefits. Ray, D. K., Ramankutty, N., Mueller, N. D., West, P. C. & Foley, J. Recent patterns of crop yield growth and stagnation. Mérel, P., Paroissien, E. & Gammans, M. Sufficient statistics for climate change counterfactuals. Porter, J. R. et al. in Climate Change 2014: Impacts, Adaptation, and Vulnerability. Part A: Global and Sectoral Aspects. Contribution of Working Group II to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change (eds Field, C. B. et al.) 485–533 (Cambridge Univ. Hsiang, S. et al. Estimating economic damage from climate change in the United States. Global spatially explicit yield gap time trends reveal regions at risk of future crop yield stagnation. Greater sensitivity to drought accompanies maize yield increase in the US Midwest. Robust responses of the hydrological cycle to global warming. Myers, S. S. et al. Increasing CO2 threatens human nutrition. Valuing Climate Damages: Updating Estimation of the Social Cost of Carbon Dioxide (National Academies Press, 2017). Robust negative impacts of climate change on African agriculture. & Challinor, A. J. Identifying traits for genotypic adaptation using crop models. Global wheat production could benefit from closing the genetic yield gap. Messina, C. D., Gho, C., Hammer, G. L. & Cooper, M. Two decades of harnessing standing genetic variation for physiological traits to improve drought tolerance in maize. Interdisciplinary Plant Science Consortium. Inclusive collaboration across plant physiology and genomics: now is the time! Closing yield gaps through nutrient and water management. & Schlenker, W. in The Economics of Climate Change: Adaptations Past and Present (eds Libecap, G. D. & Steckel, R. H.) 225–251 (Univ. Carleton, T. & Greenstone, M. Updating the United States government's social cost of carbon. University of Chicago, Becker Friedman Institute for Economics Working Paper no. Evaluating the key drivers of the US government's social cost of carbon: a model diagnostic and inter-comparison study of climate impacts in DICE, FUND, and PAGE. Diaz, D. & Moore, F. Quantifying the economic risks of climate change. Rennert, K. et al. Comprehensive evidence implies a higher social cost of CO2. The welfare economics of a data-driven social cost of carbon. Heft-Neal, S. C. The Non-linear Relationship Between Agricultural Productivity and the Environment and its Effects on Human Health and the Economy. Gatti, N., Baylis, K. & Crost, B. The effects of market access on the market structure in Zambia. Ramankutty, N., Eva, A. T., Monfreda, C. & Foley, J. Farming the planet: 1. geographic distribution of global agricultural lands in the year 2000. Sheffield, J., Goteti, G. & Wood, E. F. Development of a 50-year high-resolution global dataset of meteorological forcings for land surface modeling. Thrasher, B., Maurer, E. P., McKellar, C. & Duffy, P. Technical note: Bias correcting climate model simulated daily temperature extremes with quantile mapping. An overview of CMIP5 and the experiment design. Hawkins, E., Osborne, T. M., Ho, C. K. & Challinor, A. J. Calibration and bias correction of climate projections for crop modelling: an idealised case study over Europe. Riahi, K. et al. RCP 8.5—a scenario of comparatively high greenhouse gas emissions. van Vuuren, D. P. et al. The representative concentration pathways: an overview. Rasmussen, D. J., Meinshausen, M. & Kopp, R. E. Probability-weighted ensembles of U.S. county-level climate projections for climate risk analysis. Gennaioli, N., Porta, R. L., Silanes, F. L. D. & Shleifer, A. Dellink, R., Chateau, J., Lanzi, E. & Magné, B. Long-term economic growth projections in the shared socioeconomic pathways. Samir, K. & Lutz, W. The human core of the shared socioeconomic pathways: population scenarios by age, sex and level of education for all countries to 2100. The shared socioeconomic pathways and their energy, land use, and greenhouse gas emissions implications: an overview. Food and Agriculture Organization of the United Nations (FAO). AQUASTAT - FAO's global information system on water and agriculture. Welch, J. R. et al. Rice yields in tropical/subtropical Asia exhibit large but opposing sensitivities to minimum and maximum temperatures. Fishman, R. More uneven distributions overturn benefits of higher precipitation for crop yields. Roberts, M. J., Schlenker, W. & Eyer, J. Agronomic weather measures in econometric models of crop yield with implications for climate change. Lobell, D. B., Schlenker, W. & Costa-Roberts, J. Climate trends and global crop production since 1980. Auffhammer, M., Ramanathan, V. & Vincent, J. R. Climate change, the monsoon, and rice yield in India. Kharin, V. V., Zwiers, F. W., Zhang, X. & Wehner, M. Changes in temperature and precipitation extremes in the CMIP5 ensemble. Seasonal and regional biases in CMIP5 precipitation simulations. Evaluation of CMIP5 continental precipitation simulations relative to satellite-based gauge-adjusted observations. & Devineni, N. Crop switching reduces agricultural losses from climate change in the United States by half under RCP 8.5. Climate impacts and adaptation in US dairy systems 1981–2018. Mwangi, M. & Kariuki, S. Factors determining adoption of new agricultural technology by smallholder farmers in developing countries. Rural livelihoods and poverty reduction strategies in four African countries. Reardon, T., Stamoulis, K. & Pingali, P. Rural nonfarm employment in developing countries in an era of globalization. Muzari, W., Gatsi, W. & Muvhunzi, S. The impacts of technology adoption on smallholder agricultural productivity in sub-Saharan Africa: a review. Doss, C. R. & Morris, M. L. How does gender affect the adoption of agricultural innovations? The case of improved maize technology in Ghana. Mohamed, K. S. & Temu, A. E. Access to credit and its effect on the adoption of agricultural technologies: the case of Zanzibar. Annan, F. & Schlenker, W. Federal crop insurance and the disincentive to adapt to extreme heat. Makokha, S., Kimani, S., Mwangi, W., Verkuijl, H. & Musembi, F. Determinants of Fertilizer and Manure Use for Maize Production in Kiambu District, Kenya (CIMMYT, 2001). Kelly, D. L., Kolstad, C. D. & Mitchell, G. T. Adjustment costs from environmental change. The socioeconomics of food crop production and climate change vulnerability: a global scale quantitative analysis of how grain crops are sensitive to drought. Wooldridge, J. M. Econometric Analysis of Cross Section and Panel Data (MIT Press, 2010). & De Rock, B. Empirical revealed preference. Cassidy, E. S., West, P. C., Gerber, J. S. & Foley, J. A. Redefining agricultural yields: from tonnes to people nourished per hectare. Long, S. P., Ainsworth, E. A., Leakey, A. D. & Morgan, P. B. Treatment of major food crops with elevated carbon dioxide or ozone under large-scale fully open-air conditions suggests recent models may have overestimated future yields. Long, S. P., Ainsworth, E. A., Leakey, A. D., Nösberger, J. & Ort, D. R. Food for thought: lower-than-expected crop yield stimulation with rising CO2 concentrations. Rice production in a changing climate: a meta-analysis of responses to elevated carbon dioxide and elevated ozone concentration. & Schlenker, W. Environmental drivers of agricultural productivity growth: CO2 fertilization of US field crops. National Bureau of Economic Research (NBER) Working Paper 29320. https://www.nber.org/papers/w29320 (2021). & Schlenker, W. Identifying supply and demand elasticities of agricultural commodities: implications for the U ethanol mandate. Berry, S. & Schlenker, W. Technical report for the ICCT: empirical evidence on crop yield elasticities. Scott, P. T. Indirect estimation of yield-price elasticities. Unpublished manuscript, New York University (2013). Costinot, A., Donaldson, D. & Smith, C. Evolving comparative advantage and the impact of climate change in agricultural markets: evidence from 1.7 million fields around the world. Baldos, U. L., Hertel, T. W. & Moore, F. C. Understanding the spatial distribution of welfare impacts of global warming on agriculture and its drivers. Gouel, C. & Laborde, D. The crucial role of domestic and international market-mediated adaptation to climate change. Why are agricultural impacts of climate change so uncertain? The importance of temperature relative to precipitation. Lobell, D. B., Bänziger, M., Magorokosho, C. & Vivek, B. Nonlinear heat effects on African maize as evidenced by historical yield trials. Dong, W. et al. Impacts of nighttime post-anthesis warming on rice productivity and grain quality in East China. Burns, A., Gleadow, R., Cliff, J., Zacarias, A. & Cavagnaro, T. Cassava: the drought, war and famine crop in a changing world. Hsiang, S. & Kopp, R. E. An economist's guide to climate change science. Osborne, T., Rose, G. & Wheeler, T. Variation in the global-scale impacts of climate change on crop productivity due to climate model uncertainty and adaptation. The effect of climate change on global potato production. Global potato yields increase under climate change with adaptation and CO2 fertilisation. The GGCMI Phase 2 experiment: global gridded crop model simulations under uniform changes in CO2, temperature, water, and nitrogen levels (protocol version 1.0). Hultgren, A. Replication data and code for Hultgren et al. 2025 “Climate Change Impacts on Global Agriculture Accounting for Adaptation”. This project is an output of the Climate Impact Lab consortium that gratefully acknowledges funding from the Carnegie Corporation; Energy Policy Institute at the University of Chicago (EPIC); International Growth Centre; National Science Foundation (SES1463644); Sloan Foundation; Tata Centre for Development; the Skoll Global Threats Fund; King Philanthropies; the Alfred P. Sloan Foundation; the Heising-Simons Family Fund; the Ray and Dagmar Dolby Fund; The University of Chicago Booth School of Business; Mark Heising and Liz Simons. gratefully acknowledges support by the National Key Research and Development Program of China under grant no. 2020YFA0608602 and the National Natural Science Foundation of China under grant no. We thank L. Alcocer, T. Bearpark, S. Cadavid-Sánchez, K. Champion, T. Chong, Z. Delgerjargal, G. Dobbels, R. Du, R. Frost, J. Gilbert, R. Goyal, S. Greenhill, E. Grenier, I. H. D. Hogan, A. Hussain, T. Kulczycki, R. Li, M. Norman, O. Nwabuikwu, S. Phan, Y. Song, K. Schwarz, N. Sharma, E. Tenezakis and J. Wang for invaluable research assistance during all stages of this project and S. Anderson, J. Chang and M. Landín for excellent project management. We acknowledge the World Climate Research Programme's Working Group on Coupled Modeling, which is responsible for CMIP, and we thank the climate modelling groups (listed in Supplementary Table 9) for producing and making available their model output. For CMIP, the US Department of Energy's Program for Climate Model Diagnosis and Intercomparison provides coordinating support and led development of software infrastructure in partnership with the Global Organization for Earth System Science Portals. We would like to thank K. Baylis and S. Heft-Neal for generously sharing subnational yield data for this project. We thank the Database of Global Administrative Areas (GADM; https://gadm.org) for making freely available under the GADM license the global administrative maps as used in Figs. Any views expressed in this paper are those of the authors and do not necessarily represent the views of the Federal Reserve System or its staff. Department of Agricultural and Consumer Economics, University of Illinois Urbana-Champaign, Urbana, IL, USA Department of Agricultural and Resource Economics, University of California, Berkeley, Berkeley, CA, USA National Bureau of Economic Research (NBER), Cambridge, MA, USA Tamma Carleton, Michael Greenstone, Solomon Hsiang & Amir Jina Rhodium Group, Oakland, CA, USA Michael Delgado, Trevor Houser, Steven B. Malevich & Kelly E. McCusker BlackRock, New York, NY, USA Department of Economics, The University of Chicago, Chicago, IL, USA Michael Greenstone & Ashwin Rode Harris School of Public Policy, The University of Chicago, Chicago, IL, USA Department of Earth and Planetary Sciences, Rutgers University, New Brunswick, NJ, USA Hubert H. Humphrey School of Public Affairs, University of Minnesota, Minneapolis, MN, USA Federal Reserve Bank of San Francisco, San Francisco, CA, USA School of Marine Science and Policy, University of Delaware, Newark, DE, USA Fudan University, Shanghai, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and planned the study. computed projected impacts of future climate change. and A.H. developed methods for the valuation of impacts. and J.R. constructed damage functions and computed the partial social cost of carbon. wrote the supplementary materials and all authors reviewed and edited the text. Correspondence to Andrew Hultgren or Solomon Hsiang. The authors declare no competing interests. Nature thanks Andy Challinor and Nathan Mueller for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. For each staple crop, cross-validation is used to first select key weather variables that influence yields (rows) and then applied to select mediating variables that capture adaptation or resource access (columns). Each selected mediating variable interacts with each selected weather variable. All observations are residualized before model selection, to remove cross-sectional and time series confounders. All candidates models are tested for each crop; a and b illustrate the process only for maize. a, First step: cross-validation is used to select weather variables that influence maize (see Extended Data Figs. Top panel (blue) shows the unconditional distribution of OOS root-mean-squared error (RMSE) for all candidate models. Subsequent rows show pairs of conditional distributions that partition the sample. Green distributions depict model performance for all model permutations that include the indicated weather variable; grey distributions represent all model permutations that exclude the weather variable. Vertical lines indicate average values for each distribution. Strongly non-overlapping distributions indicate that models including the weather variable systematically tend to outperform those that exclude it. Fully overlapping distributions indicate that the indicated weather variable used to stratify models does not improve model fit. Weather terms with black stars were retained in the maize model. b, Second step: using cross-validation to select mediating variables that capture adaptation to climate (for example, average temperature) or resource access (for example, GDP per capita). Candidate mediating variables are all interacted with weather variables from a. Sub-panels depict the mediating influence of the these factors (columns) on the response to weather variables (rows). Grey panels indicate pairs of variables that are not selected for the final model. Colours indicate a weather response that is empirically recovered for a lower (blue) or upper (red) tercile, although actual model specifications are continuous interactions. All coefficients and interactions are estimated simultaneously from a single joint model. Standard errors robust88 to autocorrelation over time within first-level administrative units (for example, states) and across all units within country–year. c, Similar to b but indicating weather-mediating variable pairs selected in the final model (white squares) for each crop. The top row shows the kernel density of model OOS root-mean-squared error values for each crop; subsequent rows show conditional densities for each weather term. Pink represents all model permutations including the given weather term; grey represents all model permutations excluding the given term. Black bars indicate the average fit difference from including the term and red stars indicate significance (P < 0.05, t-test). Weather terms with black stars were carried forward to the next cross-validation procedure. The bottom four rows represent weather terms that cannot be reliably projected by existing climate models72,73,74. Empirically estimated temperature and precipitation response surfaces, by crop and by adaptation parameter. Yellow: all covariates evaluated at the sample mean; blue: covariate of interest evaluated at its lower tercile; red: covariate of interest evaluated at its upper tercile. Note that the top panel includes an extra column for rice (tmin). Also note that cassava does not include irrigation in the model, thus no irrigation interaction effects are plotted for this crop. Standard errors robust88 to correlated shocks within state over space and time and within country–year. Empirically estimated temperature and precipitation response surface for wheat, by adaptation parameter. Yellow: all covariates evaluated at the sample mean; blue: covariate of interest evaluated at its lower tercile; red: covariate of interest evaluated at its upper tercile. Standard errors robust88 to correlated shocks within state over space and time and within country–year. Crop yields observed in our data (‘Observed') versus those predicted by our model when dropping country–year fixed effects from estimation (‘Fitted'). The dotted red line is the y = x line; perfect fitted values would lie on this line. The distribution of fitted values around this line indicates that our modelled yields are, on average, unbiased. Fitted yields in levels converted from logs accounting for the non-zero expectation of the log error term. Data winsorized at the 99th percentile for visual clarity. Horizontal banding in some plots associated with known data issues in some countries. Comparison of modelled versus observed yields for many regions of the world. Red dots are observed values, blue dots are modelled values. Column 1 plots data for the region with average yield at the 95th percentile of yields in the data (by crop), column 2 the 95th percentile of cropped area, column 3 the 5th percentile of yields and column 4 the 5th percentile of cropped area. Yields originally modelled in logs and converted to levels accounting for the non-zero expectation of the exponentiated error term. 95% confidence intervals are asymmetric owing to conversion to levels and top-coded in some cases for visual clarity. The y-axis scale is consistent within crop (rows). Wheat is a combination of winter wheat and spring wheat projections, weighted by their area share in each region. Incomes taken from SSP3 as modelled by the OECD, emissions from the RCP 4.5 warming scenario. Results for RCP 8.5 are shown in Fig. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Hultgren, A., Carleton, T., Delgado, M. et al. Impacts of climate change on global agriculture accounting for adaptation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a65089793/mysterious-signals-antarctica-physics/'>A Baffling Signal Shot Up From Deep Within Earth's Ice—and No One Knows What Caused It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Could the signals be coming from particles previously unknown to physics? Two decades ago, far above the vast expanse of ice and snow, instruments aboard the Antarctic Impulsive Transient Antenna (ANITA) experiment (now retired) were picking up radio waves from cosmic rays shooting through the atmosphere. When those rays collide with particles in said atmosphere, they scatter into secondary particles—a hundred of which pass right through our bodies every second without our realizing it. By detecting these particles, ANITA was giving insight into distant cosmic events. What appeared to be radio pulses had to have somehow penetrated thousands of miles of rock that they should have been absorbed by. How these ultrahigh-energy cosmic ray (UHECR) pulses could have been detected at all was a mystery. Physicist Stephanie Wissel of Penn State University, who had been scouring ANITA data for signals from neutrinos (which are already mysterious subatomic particles) had never seen anything remotely like it. “Theoretical interpretations involving [exotic physics] have been put forward invoking new particles that induce upward-going showers in the atmosphere.” The researchers thought it was possible that the signal could have been coming from those elusive neutrinos, which are blasted out by objects and phenomena that generate extreme amounts of energy. This flavor of neutrino is associated with the tau lepton, another subatomic particle that starts to decay after it is released from ice, losing energy as it breaks down. Tau neutrino decay ends up giving off signals known as air showers, and the electromagnetic radiation released by these showers is detectable by a radio antenna such as ANITA's. The signals didn't even fit into the Standard Model of Particle Physics, which describes how matter is made up of subatomic particles. Could the signals be coming from particles previously unknown to physics? They might also, however, be from neutrinos that went undetected. Wissel is now working on ANITA's successor—the larger and more sensitive PUEO neutrino detector. Maybe this anomaly will tell us something we would have never otherwise imagined. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. This ‘Fingerprint' May Explain the Ring of Fire</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250617014222.htm'>Forever chemicals' toxic cousin: MCCPs detected in U. S. air for first time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 12:45:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Once in a while, scientific research resembles detective work. That was the case for the University of Colorado Boulder researchers who led a field campaign in an agricultural region of Oklahoma. Using a high-tech instrument to measure how aerosol particles form and grow in the atmosphere, they stumbled upon something unexpected: the first-ever airborne measurements of Medium Chain Chlorinated Paraffins (MCCPs), a kind of toxic organic pollutant, in the Western Hemisphere. "It's very exciting as a scientist to find something unexpected like this that we weren't looking for," said Daniel Katz, CU Boulder chemistry PhD student and lead author of the study. MCCPs are currently under consideration for regulation by the Stockholm Convention, a global treaty to protect human health from long-standing and widespread chemicals. While the toxic pollutants have been measured in Antarctica and Asia, researchers haven't been sure how to document them in the Western Hemisphere's atmosphere until now. They are often found in wastewater and as a result, can end up in biosolid fertilizer, also called sewage sludge, which is created when liquid is removed from wastewater in a treatment plant. In Oklahoma, researchers suspect the MCCPs they identified came from biosolid fertilizer in the fields near where they set up their instrument. Sewage sludge fertilizers have been shown to release similar compounds." Regulation came after studies found the toxic pollutants, which travel far and last a long time in the atmosphere, were harmful to human health. "We always have these unintended consequences of regulation, where you regulate something, and then there's still a need for the products that those were in," said Ellie Browne, CU Boulder chemistry professor, CIRES Fellow, and co-author of the study. Measurement of aerosols led to a new and surprising discovery With some additional research, he identified them as chlorinated paraffins found in MCCPs. Katz says the makeup of MCCPs are similar to PFAS, long-lasting toxic chemicals that break down slowly over time. Known as "forever chemicals," their presence in soils recently led the Oklahoma Senate to ban biosolid fertilizer. Many unknowns surrounding MCCPs remain, and there's much more to learn about their environmental impacts. "I think it's important that we continue to have governmental agencies that are capable of evaluating the science and regulating these chemicals as necessary for public health and safety." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250617014219.htm'>This tiny patch could replace biopsies—and revolutionize how we detect cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 12:45:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Biopsies are among the most common diagnostic procedures worldwide, performed millions of times every year to detect diseases. However, they are invasive, can cause pain and complications, and can deter patients from seeking early diagnosis or follow-up tests. Traditional biopsies also remove small pieces of tissue, limiting how often and how comprehensively doctors can analyse diseased organs like the brain. Now, scientists at King's College London have developed a nanoneedle patch that painlessly collects molecular information from tissues without removing or damaging them. This could allow healthcare teams to monitor disease in real time and perform multiple, repeatable tests from the same area - something impossible with standard biopsies. Because the nanoneedles are 1,000 times thinner than a human hair and do not remove tissue, they cause no pain or damage, making the process less painful for patients compared to standard biopsies. Dr Ciro Chiappini, who led the research published today in Nature Nanotechnology, said: "We have been working on nanoneedles for twelve years, but this is our most exciting development yet. It will allow scientists - and eventually clinicians - to study disease in real time like never before." Dr Chiappini said: "This approach provides multidimensional molecular information from different types of cells within the same tissue. This technology could be used during brain surgery to help surgeons make faster, more precise decisions. For example, by applying the patch to a suspicious area, results could be obtained within 20 minutes and guide real-time decisions about removing cancerous tissue. Made using the same manufacturing techniques as computer chips, the nanoneedles can be integrated into common medical devices such as bandages, endoscopes and contact lenses. The breakthrough was possible through close collaboration across nanoengineering, clinical oncology, cell biology, and artificial intelligence -- each field bringing essential tools and perspectives that, together, unlocked a new approach to non-invasive diagnostics. The study was supported by the European Research Council through its flagship Starting Grant programme, Wellcome Leap, and UKRI's EPSRC and MRC, which enabled acquisition of key analytical instrumentation. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/mysterious-link-between-earths-magnetism-and-oxygen-baffles-scientists/'>Mysterious Link between Earth's Magnetism and Oxygen Baffles Scientists</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Mysterious Link between Earth's Magnetism and Oxygen Baffles Scientists The strength of Earth's magnetic field and the amount of oxygen in its atmosphere seem to be correlated—and scientists want to know why Explaining the link could help to reveal fundamental trends in the evolution of life on Earth — and could show astronomers the most promising places to look for signs of complex life on other planets. But it is so far unclear whether Earth's magnetism plays direct a role in keeping oxygen levels high — and sustaining animal life — or whether both are influenced by a third, unidentified mechanism. Nature; Source: “Strong Link between Earth's Oxygen Level and Geomagnetic Dipole Revealed since the Last 540 Million Years,” by Weija Kuang et al., in Science Advances, Vol. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “We don't really have a good explanation for it,” says Benjamin Mills, a biogeochemist at the University of Leeds, UK, and co-author of the study, published in Science Advances on 13 June. But the study suggests “some potential causes that are exciting and potentially testable”, says Aubrey Zerkle, a biogeochemist at the University of St Andrews, UK. Knowing how Earth's deep interior could influence the evolution of the atmosphere is “critical to understanding what makes our planet habitable”, says Richard Bono, a geophysicist at Florida State University in Tallahassee who has helped to compile long-term records of geomagnetism. Oxygen is the main component of Earth's crust and mantle. But molecular oxygen only began to slowly accumulate in the atmosphere after organisms that produce oxygen through photosynthesis began to evolve, around 2.5 billion years ago. And only in the current aeon, covering the last 540 million years or so, has it reached concentrations that are breathable by most animals. For example, oxygen concentration “has a strong relationship with how easy it is to start and maintain wildfires,” says Mills, and the frequency of large wildfires can be worked out by looking at ancient charcoal deposits, among other factors. Geophysicists can also reconstruct how the strength and direction of the geomagnetic field has varied across even longer stretches of Earth's history, by studying rocks produced by ancient volcanic eruptions. That's because magnetic crystals that form in the solidifying lava align themselves with the field, acting like tiny compasses frozen in time. To put these two long records side-by-side and compare them, Mills teamed up with geophysicists Weijia Kuang and Ravi Kopparapu, both at NASA Goddard Space Flight Center, Greenbelt, Maryland, and with exobiologist Joshua Krissansen-Totton at the University of Washington in Seattle. Earth's magnetic field is known to have a protective effect on the upper atmosphere because it deflects solar wind, a stream of charged particles from the Sun that would otherwise cause oxygen and other gases to slowly escape into space. “We don't necessarily think that the magnetic field impacts cycling directly, but it may be a result of the same processes,” says Mills. Sanja Panovska, a geophysicist at the Helmholtz Centre for Geoscience in Potsdam, Germany, says that the study is convincing, but it introduces more questions than it answers. This article is reproduced with permission and was first published on June 13, 2025. Davide Castelvecchi is a staff reporter at Nature who has been obsessed with quantum spin for essentially his entire life. First published in 1869, Nature is the world's leading multidisciplinary science journal.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250617014206.htm'>Biggest boom since the Big Bang? Astronomers record 25x supernova brightness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 09:08:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These extraordinary phenomena occur when massive stars—at least three times heavier than our Sun—are torn apart after wandering too close to a supermassive black hole. Their disruption releases vast amounts of energy visible across enormous distances. "We've observed stars getting ripped apart as tidal disruption events for over a decade, but these ENTs are different beasts, reaching brightnesses nearly ten times more than what we typically see," said Jason Hinkle, who led the study as the final piece of his doctoral research at IfA. “Not only are ENTs far brighter than normal tidal disruption events, but they remain luminous for years, far surpassing the energy output of even the brightest known supernova explosions.” While typical supernovae emit as much energy in just one year as the Sun does in its 10 billion-year lifetime, ENTs radiate the energy of 100 Suns over a single year. He identified two unusual flares in data from the European Space Agency's Gaia mission that brightened over a timescale much longer than known transients and without characteristics common to known transients. "Gaia doesn't tell you what a transient is, just that something changed in brightness," said Hinkle. The discovery launched a multi-year follow-up campaign to figure out what these sources were. Because ENTs evolve slowly over several years, capturing their full story took patience and persistence. Recently, a third event with similar properties was discovered by the Zwicky Transient Facility and reported independently by two teams, adding strong support that ENTs are a distinct new class of extreme astrophysical events. The authors determined these extraordinary events could not be supernovae because they release far more energy than any known stellar explosion. The sheer energy budget, combined with their smooth and prolonged light curves, firmly pointed to an alternative mechanism: accretion onto a supermassive black hole. However, ENTs differ significantly from normal black hole accretion which typically shows irregular and unpredictable changes in brightness. Benjamin Shappee, Associate Professor at IfA and study co-author, emphasized the implications: "ENTs provide a valuable new tool for studying massive black holes in distant galaxies. By observing these prolonged flares, we gain insights into black hole growth when the universe was half its current age when galaxies were happening places—forming stars and feeding their supermassive black holes 10 times more vigorously than they do today." Future observatories like the Vera C. Rubin Observatory and NASA's Roman Space Telescope promise to uncover many more of these spectacular events, revolutionizing our understanding of black hole activity in the distant, early universe. "These ENTs don't just mark the dramatic end of a massive star's life. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250618033727.htm'>From shortage to supremacy: How Sandia and the CHIPS Act aim to reboot US chip power</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 08:22:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Sandia National Laboratories has joined a new partnership aimed at helping the United States regain its leadership in semiconductor manufacturing. Today, the U.S. manufactures none of the world's most advanced chips which power technologies like smartphones, owned by 71% of the world's population, as well as self-driving cars, quantum computers, and artificial intelligence-powered devices and programs. "We have pioneered the way for other labs to join," said Mary Monson, Sandia's senior manager of Technology Partnerships and Business Development. By including the national labs, U.S. companies, and academia, it's really a force multiplier." Sandia has a long history of contributing to the semiconductor industry through research and development partnerships, its Microsystems Engineering, Science and Applications facility known as MESA, and its advanced cleanrooms for developing next-generation technologies. Industry is moving fast, so we are keeping abreast of everything happening and incorporating what will help us deliver more efficiently on our national security mission. It's about looking at innovative ways of partnering and expediting the process," Monson said. The pandemic provided a perfect example, as car lots were left bare and manufacturers sat idle, waiting for chips to be produced to build new vehicles. "An average car contains 1,400 chips and electric vehicles use more than 3,000," said Rick McCormick, Sandia's senior scientist for semiconductor technology strategy. "Other nations around the globe are investing more than $300 billion to be leaders in semiconductor manufacturing. "More than $12 billion in research and development spending is planned under CHIPS, including a $3 billion program to create an ecosystem for packaging assemblies of chiplets," McCormick said. "These chiplets communicate at low energy and high speed as if they were a large expensive chip." MESA already fabricates high-reliability chiplet assembly products for the stockpile and nonproliferation applications. The government has invested billions of dollars in new factories, all of which need to be staffed by STEM students. "There is a potential crisis looming," McCormick said. That effort will also help Sandia bolster its staffing pipeline." As part of its membership, Sandia will offer access to some of its facilities to other NSTC members, fostering collaboration and partnerships. Tech transfer is a core part of Sandia's missions, and this initiative will build on that by helping private partners increase their stake in the industry while enabling Sandia to build on its own mission. "We will be helping develop suppliers and strengthen our capabilities," Monson said. "We are a government resource for semiconductor knowledge. We are in this evolving landscape and have a front row seat to what it will look like over the next 20 years. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250617014203.htm'>Monster salamander with powerful jaws unearthed in Tennessee fossil find</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-18 06:03:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A giant, strong-jawed salamander once tunneled through ancient Tennessee soil. And thanks to a fossil unearthed near East Tennessee State University, scientists now better understand how it helped shape Appalachian amphibian diversity. The findings appeared in the journal Historical Biology, authored by a team of researchers from the Gray Fossil Site & Museum and ETSU: Assistant Collections Manager Davis Gunnin, Director and Professor of Geosciences Dr. Blaine Schubert, Head Curator and Associate Professor of Geosciences Dr. Joshua Samuels, Museum Specialist Keila Bredehoeft and Assistant Collections Manager Shay Maden. "This exciting find underscores the vital role our university plays in preserving and exploring Appalachia's deep natural history." Tennessee alone is home to more than 50 different salamanders - one in eight of all living salamander species. Dusky salamanders, common in Appalachian Mountain streams, likely evolved from burrowing ancestors, relatives of Alabama's Red Hills salamander, a large, underground-dwelling species with a worm-like body and small limbs. Their explosive diversification began around 12 million years ago, shaping much of the region's salamander diversity today. "Today they're only found in a few counties in southern Alabama, and researchers thought of them as a highly specialized dead-end lineage not particularly relevant to the evolution of the dusky salamanders. Discovery of Dynamognathus robertsoni here in Southern Appalachia shows that these types of relatively large, burrowing salamanders were once more widespread in eastern North America and may have had a profound impact on the evolution of Appalachian salamander communities." Dusky salamanders in the Appalachians today reach only seven inches long at their largest. Researchers believe predators like this one may have driven the rapid evolution of Appalachian stream-dwelling salamanders, highlighting the region's key role in salamander diversification. "The warmer climate in Tennessee 5 million years ago, followed by cooling during the Pleistocene Ice Ages, may have restricted large, burrowing salamanders to lower latitudes, like southern Alabama, where the Red Hills salamander lives today," said Samuels. "This group of salamanders has unusual cranial anatomy that gives them a strong bite force, so the genus name - Dynamognathus - Greek for 'powerful jaw,' is given to highlight the great size and power of the salamander compared to its living relatives," said Maden. From volunteers and students to staff to faculty, the ETSU Gray Fossil Site & Museum is represented by a dynamic team of lifelong learners and is one of the many reasons ETSU is the flagship institution of Appalachia. "The latest salamander publication is a testament to this teamwork and search for answers," said Schubert. "When Davis Gunnin, the lead author, began volunteering at the museum as a teenager with an interest in fossil salamanders, I was thrilled, because this region is known for its salamander diversity today, and we know so little about their fossil record. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            