
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-10</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64713874/ancient-chinese-tomb-reveals-blonde-man/'>Archaeologists Found Someone They Never Expected in an Ancient Chinese Tomb: a Blonde Man</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-10 12:18:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Long Zhen, director of the Jinyang Ancient City Research Institute of the Taiyuan Cultural Relics Protection Research Institute, says, according to a translation from the state-run news outlet Xinhua, that the tomb has pairs of figures painted on both sides of the tomb door and that paintings of people continue throughout the tomb, including everything from women pushing stones to grind flour to men making pasta, women using oranges to fetch water, and men stepping on hammers to help pound rice. Experts believe the tomb was owned by a man who died in 736 AD at age 63, during the middle of the Tang dynasty, which ran from 618 to 907 AD. He was buried in the tomb along with his wife. The scenes offer a fresh view on what daily life and responsibilities were during the Tang dynasty. Victor Xiong, professor of history at Western Michigan University, who wasn't involved in the discovery, told Live Science the “Westerner,” painted in the tomb leading camels, likely represented someone from Central Asia. “Based on his facial features and outfit style,” Xiong said, “we can identify him as a ‘Westerner,' likely a Sogdian from Central Asia.” The Sogdian people lived in what is now modern-day Tajikistan and Uzbekistan and were active along Silk Road trading routes. The South China Morning Post wrote that the artwork is styled similarly to other Tang-era murals with strong outlines, simple shading, and two-dimensional artwork. Figures painted at the gate may represent guardians, with yellow robes and swords tied around their waist. The paintings also feature plenty of trees, flowers, mountains, and livestock. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Archaeologists Find Third Ancient Fort on One Site This Tooth Changes Part of Christianity's History Scientists Spotted the Milky Way on a Sarcophagus Hikers Found an Old Can Filled With Gold Coins</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59627-z'>BiDAC-dependent degradation of plasma membrane proteins by the endolysosomal system</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-10 03:36:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. BiDAC-induced ubiquitination is generally believed to direct cytosolic and nuclear proteins to proteolytic destruction by proteasomes. However, pathways that govern the degradation of other classes of BiDAC targets, such as integral membrane and intraorganellar proteins, have not been investigated in depth. In this study we use morphological profiling and CRISPR/Cas9 genetic screens to investigate the mechanisms by which BiDACs induce the degradation of plasma membrane receptor tyrosine kinases (RTKs) EGFR and Her2. We find that BiDAC-dependent ubiquitination triggers the trafficking of RTKs from the plasma membrane to lysosomes for degradation. Additionally, our screen uncovers a non-canonical function of the lysosome-associated arginine/lysine transporter PQLC2 in EGFR degradation. Our data show that BiDACs can target proteins to proteolytic machinery other than the proteasome and motivate further investigation of mechanisms that govern the degradation of diverse classes of BiDAC targets. A mechanistic understanding of the relationship between extracellular signaling and cell proliferation has led to the development of pharmacological agents that mitigate the progression of human cancers. This has been exemplified by the development of antibodies and small molecules that block aberrant pro-growth signals generated by overexpressed receptor tyrosine kinase (RTK) proteins at the cell surface. Antibodies that target the extracellular domains of oncogenic RTKs, such as those found in the epidermal growth factor (EGF) receptor family members EGFR (also known as Her1) and Her2, inhibit RTK signaling by competing with extracellular RTK ligands for receptor binding, by inhibiting RTK dimerization, and by inducing RTK internalization and degradation1. More recently, cell-permeable small-molecule kinase inhibitors have been utilized to directly inhibit RTK activity and potentiate the anti-tumor effects of antibodies2,3. Extracellular ligand binding induces dimerization of RTK monomers and subsequent cross-phosphorylation of their intracellular domains. This mechanism has motivated the development of therapeutics that inhibit RTK dimerization. In particular, Her2 lacks a ligand binding domain and thus can be activated through dimerization with Her3 or EGFR bound to their respective ligands, neuregulin and EGF4. The Her2 antibody pertuzumab, unlike the original therapeutic Her2 antibody trastuzumab, inhibits Her2/Her3 heterodimerization and is used in cases where trastuzumab and kinase inhibitor therapies have lost efficacy5. Similarly, the covalent kinase inhibitor neratinib also triggers Her2 degradation and may prevent dimerization of Her2 with other Her receptors6. Thus, therapeutic modalities that inhibit the scaffolding functions of Her2 may overcome the resistance that develops to other classes of Her2 inhibitors. Recent advances in the development of bifunctional degradation activating compounds (BiDACs) have enabled the conversion of existing small molecules into potent degraders. These molecules recruit E3 ubiquitin ligase complexes to target proteins, resulting in their polyubiquitination and proteolysis by ubiquitin-dependent degradation pathways7. This mechanism presents opportunities to target proteins that also have essential scaffolding functions, since in theory, appending polyubiquitin chains on the target protein may be sufficient to trigger its degradation. In agreement with this idea, BiDACs targeting BRD4, an epigenetic regulator that recruits transcriptional machinery to histones, triggered BRD4 degradation and exhibited increased potency relative to BRD4 inhibitors in hematological cancer models8,9. This concept was investigated with respect to RTKs using the small molecule lapatinib, which inhibits the kinase domains of Her2 and EGFR. A BiDAC consisting of lapatinib conjugated to a molecule that binds the E3 substrate adapter VHL triggered degradation of Her2 and EGFR. As with BRD4, degradation of these RTKs resulted in a more sustained inhibition of downstream signaling and a greater impact on cell viability than inhibition alone10. However, an unbiased comparison of the effects of RTK inhibition and degradation on cell physiology has not been performed. The consequences of cellular perturbations can be explored using morphological profiling, a method that utilizes high-throughput microscopy and machine learning-based image analysis to establish morphological phenotypes representing cellular states induced by chemical, genetic, or environmental perturbations. Prevalent techniques such as cell painting utilize a multiplexed panel of dyes to broadly label organelles and other cellular structures11,12. Machine learning-based image analysis tools are used to extract and quantitatively compare information from microscopy images, and morphological profiles derived from such data have been shown to be expressive, high-dimensional representations of cell state analogous to gene expression profiles13. Due to its integrative and information-rich nature, morphological profiling presents a promising approach for unbiased discovery of new phenotypes, accessing signals in the data that lie below the threshold of human perception14,15. In this study, we ask if BiDAC-dependent degradation of Her2 produces distinct cellular morphologies when compared to other modalities that inhibit Her2 signaling. We find that Her2 BiDACs induce distinctive changes in Her2 localization and levels. Functional genomic analysis of BiDAC-dependent degradation reveals that the localization changes reflect RTK internalization and degradation in lysosomes. In addition, we show that degradation of EGFR partially depends on a non-canonical function of the lysosome-associated amino acid transporter, PQLC2. These data suggest that BiDACs can induce degradation of their targets through multiple ubiquitin-dependent pathways. To investigate the phenotypic consequences of Her2 degradation, we used a previously reported BiDAC that consists of the Her2/EGFR inhibitor lapatinib linked to a VHL binding group (Lap-VHL) (Supplementary Fig. We also synthesized a Her2-specific BiDAC by replacing lapatinib with a tucatinib warhead linked to a VHL binding group by an azetidinyl piperidine linker (Tuc-VHL) (Supplementary Fig. Exposure to Tuc-VHL resulted in degradation of Her2 (Fig. 1A, B) and inhibition of Her2 kinase activity (Supplementary Fig. Lap-VHL and Tuc-VHL containing a non-functional VHL binding group (VHLInactive) inhibited Her2 kinase activity but failed to induce Her2 degradation (Fig. Western blot analysis of Her2 and EGFR levels in SKBR3 (A) or SKOV3 cell lysates (B) from cells treated with active or VHLInactive BiDACs for 24 h. Panels are representative of three independent experiments. CellTiter-Glo viability analysis of SKBR3 (C) or SKOV3 (D) cells treated for 5 days with the indicated drugs. Data points show mean ± SD of three technical replicates. SKOV3, **p = 0.0059, ****p < 0.0001 by two-way ANOVA, adjusted for multiple comparisons using Tukey's multiple comparison test. E Representative fluorescence microscopy images of SKOV3 cells treated for 24 h with 1 µM of the indicated drugs. F Quantification of Her2 intensity in SKOV3 cells treated with Tuc-VHL at the indicated timepoints. ****p < 0.0001 by one-way ANOVA adjusted for multiple comparisons using Bonferroni correction. T-SNE cell morphology plots of SKOV3 (G) or SKBR3 (H) cells treated for 48 h with the indicated drugs. For T-SNE plots, dose order represents increasing drug concentrations. For (E, F), n = 45,000 cells were imaged at each timepoint and for each treatment condition. Active BiDACs reduced SKBR3 and SKOV3 viability (Fig. 3a), and induced apoptosis in SKBR3 cells (Supplementary Fig. 3b–d) more potently than their inactive counterparts. This difference was most pronounced when cells were treated at doses that induced Her2 degradation (Fig. Thus, active and inactive Her2 BiDACs can be used to compare the consequences of Her2 degradation and inhibition. To create unbiased morphological profiles of Her2 loss of function, we utilized a modified version of the cell painting protocol12, combining probes for the nucleus, Golgi, mitochondria, and actin with an immunofluorescence probe for Her2 (Fig. As expected, Tuc-VHL induced a dose and time-dependent decrease in Her2 signal (Fig. This panel included Her2-binding antibodies (Trastuzumab, Pertuzumab), covalent and allosteric inhibitors (Neratinib, BDTX-189), inhibitors of downstream kinases PI3K (Alpelisib) and AKT (Ipatasertib), in addition to Tucatinib, Lapatinib, and the VHLInactive compounds (Supplementary Fig. Given the time-dependency of BiDAC-induced Her2 degradation, cells were imaged across multiple time points and treatment doses. The averaged vector representations for each treatment condition were clustered and visualized on a two-dimensional projection (Fig. Points in proximity to each other suggest treatment conditions that produce similar morphological phenotypes. For SKOV3 cells treated for 48 h, DMSO controls were localized to the center of the projection (Fig. 1G, gray outline) while inhibitors of downstream PI3K and AKT kinases produced distinctive, dose-dependent phenotypic trajectories in the lower right quadrant (Fig. In contrast, the BiDAC warheads Lapatinib and Tucatinib induced phenotypes that clustered together at the lower left quadrant, suggesting that these non-covalent Her2 inhibitors do not produce distinct cell states despite their differential specificity for EGFR (Fig. The active BiDACs formed a distinct cluster at the top of the projection which was adjacent to the cluster formed by Neratinib, a Her2 covalent inhibitor previously shown to both inhibit Her2 and trigger its degradation6 (Fig. Active BiDACs administered at higher doses were dispersed away from the primary cluster, suggesting that they induced distinct degradation-dependent phenotypes. Lastly, we observed that Pertuzumab and Trastuzumab induced their own phenotypes that were not adjacent to any of the small molecule inhibitors (Fig. 1G, orange and yellow outlines), consistent with the distinct modes by which these antibodies disrupt Her2 activation. Morphological profiles generated for SKBR3 cells, which are more sensitive to Her2 inhibitors, resembled those for SKOV3, indicating that the clustering patterns were not dependent on degree of dependency on Her2 signaling (Fig. Distinct, time-dependent clustering of active BiDAC-treated wells was correlated with the time required to achieve Her2 degradation (Fig. Indeed, distinct active BiDAC phenotypes were evident at some doses after only 6 h of treatment (Fig. To determine the extent to which these phenotypes depend on Her2 staining, we trained models lacking Her2 imaging information. These models showed that active BiDAC clusters were less distinctive and were closer to the VHLInactive BiDACs in the latent space of the model (Fig. 2C–F), a result that was observed in both cell lines at early and late time points (Supplementary Figs. Upon manual examination of the images, we observed that BiDACs induced formation of Her2-postive puncta (Fig. These results suggest that changes in Her2 levels and localization are the primary features that account for distinct morphological phenotypes induced by active BiDACs. T-SNE cell morphology plots of SKOV3 (A) or SKBR3 (B) cells treated for 6 h with the indicated drugs. T-SNE cell morphology plots generated without Her2 staining data for SKOV3 (C) or SKBR3 (D) cells treated for 6 h with the indicated drugs. Graphical representation of the degree of clustering between active and VHLInactive BiDAC-dependent phenotypes in the presence (E) or absence (F) of Her2 staining information. G Representative fluorescence microscopy images of SKOV3 cells treated with the indicated drugs for 6 h. n = 45,000 cells were imaged for each treatment condition. For T-SNE plots, dose order represents increasing concentrations of drugs. The unexpected relocalization of Her2 observed after treatment with functional BiDACs suggested that this phenotype is mechanistically related to RTK degradation. To investigate the mechanisms that govern BiDAC-induced RTK degradation in depth, we used a functional genomics approach that employed SKOV3 cells expressing Cas9 and GFP/RFP knocked-in at one or more Her2 loci and at both EGFR loci (Supplementary Fig. 8a), yielding expression of Her2-GFP and EGFR-RFP (RTK-FP). Treatment of RTK-FP cells with Lap-VHL or Tuc-VHL induced the degradation of the expected RTKs (Fig. Treatment with EGF, which binds to and downregulates expression of EGFR, resulted in degradation of EGFR-RFP but not Her2-GFP (Fig. Lap-VHL-induced degradation of both RTKs was partially blocked by the addition of a competing VHL-binding molecule (Fig. 3D), consistent with dependence of RTK-FP degradation on VHL recruitment. Thus, RTK-FPs, like untagged RTKs, are amenable to degradation by BiDACs. A–D Flow cytometric analysis of RTK-FP levels in cells treated for 24 h with the indicated compounds. Cells in (D) were concurrently treated with a fixed dose of 1 µM Lap-VHL. Data points show mean ± SD of three technical replicates, and each panel is representative of two independent experiments. E Dot plot shows RTK-FP levels in cells treated for 24 h with 1 µM Lap-VHL. F Pseudo-color flow cytometric plots show the gates used to sort RTK-FP-positive cells treated with Lap-VHL. I, J Graphical representations of protein degradation machinery identified across both RTK-FP screens. To perform the genetic screen, we infected RTK-FP cells with a whole genome CRISPR library containing six single guide RNAs (sgRNAs) per gene. Ten days after infection, cells were treated with 1 µM Lap-VHL, which resulted in a 70% or greater decrease in RTK-FP levels after 24 hs (Fig. Cells that failed to degrade RTK-FPs, defined as those having the 0.5-1% highest RFP or GFP fluorescence levels, were enriched by cell sorting in two separate screens (Fig. To determine the degree of sgRNA enrichment in the sorted populations, libraries from the sorted and the bulk unsorted populations were prepared from extracted genomic DNA, analyzed by next generation sequencing and compared using MAGeCK analysis (Supplementary Data 1 and  2)16. Cells that failed to degrade RTK-FPs expressed sgRNAs targeting key E3 complex components, including VHL, the Cullin scaffolding protein CUL2, and the elongin B/C adapters TCEB2 and TCEB117 (Fig. In addition, we detected enrichment of sgRNAs targeting ubiquitin, the ubiquitin-activating enzyme UBA1 (E1), and the Cullin-associated ubiquitin-conjugating enzyme (E2) UBE2R2. We also identified core neddylation machinery necessary for activation of cullin E3s, which consists of NEDD8, the heterodimeric NAE1-UBA3 NEDD8 activating enzyme and the UBE2M NEDD8 conjugating enzyme19. Interestingly, we identified all nine subunits of the signalosome, which functions in the removal of NEDD8 from Cullin scaffolds (Fig. This finding is consistent with a previous study which demonstrated that perturbation of the signalosome complex prevents BiDAC-mediated degradation by triggering the autodegradation of E3 substrate receptors20. Instead, we found enrichment of guides targeting multiple Vo and V1 subunits of the lysosomal ATPase (V-ATPase) and its accessory subunit ATP6AP121 (Fig. 3J), suggesting that BiDAC-induced degradation of RTKs is mediated by the endolysosomal system. Lysosomal proteolysis requires acidification of lysosomes by the V-ATPase. To investigate the role of lysosomes in BiDAC-induced RTK degradation, we treated RTK-FP cells with the V-ATPase inhibitor, Bafilomycin-A1 (Baf-A1). Baf-A1 inhibited RTK degradation by Lap-VHL (Fig. Treatment with the E1 inhibitor MLN-7243 and the proteasome inhibitor Carfilzomib also blocked RTK-FP degradation (Fig. Hydroxychloroquine (HCQ), which also disrupts lysosomal pH, inhibited RTK-FP degradation to a similar extent as Baf-A1 (Fig. To determine if BiDACs trigger trafficking of RTKs to lysosomes, we imaged RTK-FP cells expressing the lysosomal marker LAMP1-SNAP. In untreated cells, RTKs were present at plasma membrane projections and colocalized with LAMP1-SNAP-positive lysosomes (Fig. Lap-VHL induced relocalization of RTKs to lysosomes, consistent with the BiDAC-dependent changes in Her2 localization observed by morphological profiling (Fig. Concurrent treatment with Baf-A1 resulted in formation of enlarged LAMP1-SNAP-positive lysosomes containing undegraded RTK-FP (Fig. Internalized EGFR-RFP also colocalized with the late endosome marker SNAP-RAB7A, confirming that BiDACs trigger delivery of RTK-FPs to lysosomes through the canonical endocytic pathway (Supplementary Fig. We asked if inhibition of EGFR-RFP degradation by inhibitors of the lysosome and the ubiquitin-proteasome system was common to EGF, which binds to EGFR and triggers its degradation by the endolysosomal pathway22. EGFR-RFP levels in cells treated with EGF were partially stabilized by Baf-A1 but completely stabilized by HCQ and Carfilzomib (Fig. 4L), suggesting that BiDACs and EGF induce EGFR degradation by similar mechanisms. Western blot analysis of lysates (A) and quantification (B) of untagged Her2 and EGFR-RFP levels in cells treated for 24 h with active and VHLInactive BiDAC in combination with protein degradation inhibitors (Carfilzomib, 1 µM; MLN-7243, 200 nM; Bafilomycin-A1, 100 nM). (A) is representative of three biological replicates and (B) shows the mean ± SD of the band intensities. C Western blot analysis of endogenous RTK levels in lysates from SKBR3 cells treated for 24 h with the indicated BiDACs and Bafilomycin-A1. D–I Flow cytometric analysis of RTK-FP levels in cells treated for 8 h with the indicated BiDACs and protein degradation inhibitors (HCQ hydroxychloroquine, 30 µM). Data points show mean ± SD of three technical replicates, and each panel is representative of two independent experiments. J Confocal microscopy of RTK-FP cells expressing LAMP1-SNAP treated for 8 h with 1 µM Lap-VHL alone or in combination with 100 nM Bafilomycin-A1. Images are representative of n = 100 cells. n = 10,000 lysosomes were identified for colocalization analysis of each treatment condition. L Flow cytometric analysis of RTK-FP levels in cells treated for 8 h with epidermal growth factor (EGF) and protein degradation inhibitors, as in (D–I). Data points show mean ± SD of three technical replicates. For all combination treatments, cells were pretreated for 30 min with degradation inhibitors prior to addition of BiDACs. P-values calculated by ANOVA were adjusted using Tukey's multiple comparisons test. Our results raised the possibility that BiDACs trigger RTK degradation by both lysosome and proteasome-dependent pathways. Alternatively, inhibition of one pathway may impact RTK degradation by the other. To distinguish between these possibilities, we examined how the localization of RTK-FPs changes in response to BiDACs when used in combination with degradation inhibitors. First, we measured surface RTK levels by staining cells with antibodies against EGFR and Her2 (Fig. This decrease was not blocked by Baf-A1, consistent with internalization of RTK-FPs and delivery to lysosomes under these conditions (Fig. Similar effects on EGFR-RFP levels were observed after treatment with EGF, though in agreement with prior studies23, these inhibitors did not completely block EGFR-RFP internalization (Fig. To track changes in RTK-FP localization, we imaged cells treated with BiDACs and degradation inhibitors by time lapse microscopy. Lap-VHL, but not Lap-VHLInactive, triggered appearance of RTK-FP puncta within 2 h of treatment, which decreased in abundance over the next 8 h (Fig. Cotreatment with Carfilzomib or MLN-7243 prevented RTK-FP clearance and puncta formation, resulting in retention of RTK-FPs at the plasma membrane. In contrast, cotreatment with Baf-A1 did not inhibit puncta formation over time (Fig. Appearance of Her2-GFP-positive puncta was most evident after Baf-A1 treatment, likely due to dequenching of GFP fluorescence associated with the Baf-A1-induced increase in lysosomal pH (Fig. Together, these data show that proteasome inhibition prevents degradation of RTKs by blocking BiDAC-induced internalization. A Flow cytometric analysis of surface RTK levels in cells treated for 24 h with 1 µM Lap-VHL. Flow cytometric analysis of surface EGFR (B, E) or surface Her2 (C, D) levels in cells treated for 24 h with the indicated BiDACs or EGF in combination with protein degradation inhibitors (Carfilzomib, 1 µM; MLN-7243, 200 nM; Bafilomycin-A1, 100 nM). Data points show mean ± SD of three technical replicates, and each panel is representative of two independent experiments. ****p < 0.0001; ns not significant by two-way ANOVA. F Confocal timelapse microscopy of RTK-FP cells treated with 1 µM Lap-VHL or Lap-VHLInactive in combination with Bafiloymcin-A1 or Carfilzomib. Images at T = 0 h were captured immediately after addition of BiDACs. Montages are representative of 28 fields imaged for each condition. Quantification of the number of Her2-GFP-positive (G) or EGFR-RFP-positive (H) puncta in cells treated with Lap-VHL and the indicated protein degradation inhibitors. Each data point shows the mean ± SD of three replicate wells, and the results are representative of two independent experiments. I, J RTK-FP cells were treated with 1 μM BiDACs alone or together with 1 μM Carfilzomib for 3 h. Her2-GFP was immunopurified and levels of Her2-GFP-associated K48-ubiquitin were analyzed by western blot. The data was obtained from three independent experiments. P-values calculated by ANOVA were adjusted using Tukey's multiple comparisons test. We reasoned that this decrease in free ubiquitin may prevent BiDAC-induced RTK ubiquitination and subsequent internalization. To assess the ubiquitination status of Her2, we immunoprecipitated Her2-GFP and determined the levels of Her2-associated K48-linked ubiquitin. Treatment with active Lap-VHL resulted in an increase in K48 ubiquitin chains that were copurified with Her2-GFP (Fig. Cotreatment with Carfilzomib blocked the BiDAC-induced increase in Her2-associated polyubiquitin chains, despite causing the expected accumulation of total polyubiquitin chains (Fig. We found that sgRNAs targeting the amino acid transporter PQLC2 (encoded by the SLC66A1 gene) were enriched in cells that failed to degrade EGFR-RFP (Fig. Deletion of PQLC2 in RTK-FP cells (PQLC2KO), confirmed by sequencing its genomic locus (Fig. 6B), did not strongly impact basal EGFR-RFP expression levels (Fig. However, EGFR-RFP degradation by Lap-VHL was reduced in PQLC2KO relative to control cells (Fig. Deletion of PQLC2 also inhibited EGFR-RFP degradation in response to EGF (Fig. We reasoned that loss of PQLC2 may block EGFR-RFP internalization, trafficking, or proteolysis in lysosomes. Imaging analysis revealed that PQLC2KO cells had more internalized EGFR-RFP and had more residual EGFR-RFP in lysosomes following treatment with Lap-VHL (Fig. A Control and PQLC2-targeting sgRNA counts from Lap-VHL-treated RTK-FP cells sorted for high EGFR-RFP levels, compared to those from unsorted cells. B Genomic PCR analysis of the fraction of frameshifted PQLC2-coding alleles in cell pools expressing Cas9 and sgRNAs targeting PQLC2. Histograms (1 µM Lap-VHL) (C) and quantification (D) of EGFR-RFP levels in control or clonal PQLC2KO cells treated for 24 h with Lap-VHL. Control vs. Clone #2, ***p = 0.0003. ns, not significant by two-way ANOVA adjusted using Tukey's multiple comparison test. Each data point shows a single measurement, and the panel is representative of two independent experiments. Histograms (10 ng/ml EGF) (E) and quantification (F) of EGFR-RFP levels in control or PQLC2KO cells treated for 24 h with EGF. Data points in (F) show the mean ± SD of three technical replicates, and the panel is representative of two independent experiments. G Confocal microscopy of control or PQLC2KO cells expressing LAMP1-SNAP, treated for 24 h with 1 μM Lap-VHL or Lap-VHLInactive. I RTK-FP cells were pre-treated with 100 nM Bafilomycin-A1 for 30 min and subsequently treated with 1 μM Lap-VHL. The number of Her2-GFP puncta at each time point, shown as mean ± SD, was quantified using time lapse microscopy, and the rate of accumulation of Her2-GFP puncta was determined by fitting the data using linear regression analysis. ns, slopes not significantly different (p = 0.1019) using multiple regression analysis with dummy coded variables (GraphPad Prism). n = 18 fields were analyzed for each condition. For (C–I), RTK-FP control cells express an sgRNA targeting the cell surface protein CD81. PQLC2 contains two proline-glutamine (PQ) loop domains that are critical for its CAA transport function (Fig. PQLC2 also recruits the C9ORF72-SMCR8-WDR41 (CSW) complex to lysosomes by binding to WDR41, an interaction that is enhanced when cells are starved of CAA28. Two dileucine motifs mediate the localization of PQLC2 to lysosomes29. To determine which functions of PQLC2 are necessary for EGFR-RFP degradation, we expressed doxycycline (dox)-inducible PQLC2-Flag variants in PQLC2KO cells (Fig. PQLC2-Flag mutants were localized to puncta that were morphologically similar to those formed by wild type (WT) PQLC2 with the exception of PQLC2 (L288A/L289A)-Flag, which was present at peripheral structures consistent with the plasma membrane (Fig. In the absence of dox, EGFR-RFP degradation by Lap-VHL was attenuated in most cells, but a small population of cells exhibited more EGFR-RFP degradation due to leaky PQLC2 expression from the dox-inducible promoter (Fig. Dox-dependent induction of WT PQLC2 expression in all cells fully restored EGFR-RFP degradation to that of control cells (Fig. Surprisingly, expression of PQLC2 variants that contain inactivating mutations in their PQ domains (P55L and P201L)27 resulted in complete rescue of EGFR-RFP degradation (Fig. Similarly, we observed rescue by PQLC2 variants containing mutations that disrupt binding to WDR41 (F49A, Y105A, and Y195A)29, and the CSW complex (Fig. Lastly, partial mis-localization of PQLC2 to the plasma membrane through mutation of its dileucine motifs (L127A/L128A and L288A/L289A) resulted in complete rescue (Fig. 7H), though a fraction of dileucine-mutant PQLC2 was still present at lysosomes (see Fig. Together, these results indicate that PQLC2 regulates EGFR-RFP degradation by a mechanism that is not governed by its known biological functions. A Schematic showing PQLC2 mutations generated for this study. Asterisks indicate the functional consequences of each mutation. B Western blot analysis of lysates from RTK-FP PQLC2KO cells expressing PQLC2-Flag mutants. C Confocal immunofluorescence microscopy analysis of PQLC2-Flag localization in PQLC2KO cells. Images are representative of n = 10 cells. Histograms (1 μM Lap-VHL) (D) and quantification (E) of EGFR-RFP levels in PQLC2KO cells expressing dox-inducible PQLC2-Flag, treated for 24 h with Lap-VHL. Each data point shows mean ± SD of three technical replicates. Histograms of EGFR-RFP levels in PQLC2KO cells induced with 500 ng/mL dox to express PQLC2-Flag mutants defective in CAA transport (F), WDR41 binding (G), or lysosomal localization (H), treated for 24 h with 1 μM of the indicated BiDACs. I, J Western blot analysis of lysates from PQLC2KO cells expressing Myc-PQLC2-Flag or Flag-PQLC2-Myc. Membranes were probed with anti-Flag or anti-Myc antibodies. K Western blot analysis of lysates from PQLC2KO cells expressing a C-terminal PQLC2-Flag fragment (∆1–171) or mutant PQLC2-Flag containing deletion mutations within a predicted intralysosomal loop. L Histogram of EGFR-RFP levels in BiDAC-treated (1 μM) PQLC2KO cells induced with dox to express a C-terminal PQLC2-Flag fragment or mutant PQLC2-Flag containing a ∆162-180 deletion. All western blot and histogram panels are representative of two independent experiments. For (D, E), control cells express an sgRNA targeting the cell surface protein CD81. A Representative fluorescence lifetime (FLIM) images of PQLC2KO-pH cells incubated in buffers with the indicated pH and ionophores to equalize pH across membranes. Distribution of mScarlet-LAMP1 lifetimes across individual lysosomes (B) and the median lifetimes across all lysosomes (C) quantified from the images in (A). n = 2000 lysosomes were analyzed at each pH across two independent experiments for PQLC2KO-pH and in a single experiment for control-pH cells. D Representative fluorescence lifetime (FLIM) images of PQLC2KO-pH or control-pH cells. Distributions of lifetimes (E) and areas (F) of individual lysosomes quantified from images in (D). n = 25,000 lysosomes were analyzed across six (for PQLC2KO-pH cells) or four (for control-pH cells) independent experiments. Western blot analysis of LC3 (G) and quantification of LC3-II levels (H) in PQLC2KO or control RTK-FP treated with 1 µM Torin-1 for the indicated times. The data points in (H) show mean ± SD of three independent experiments. I Confocal microscopy of mScarlet-LAMP1 and PQLC2-SNAP in PQLC2KO-pH cells. J Distribution of PQLC2-SNAP intensities and mScarlet-LAMP1 lifetimes across lysosomes imaged in (I). K Representative FLIM image of PQLC2KO-pH cells expressing PQLC2-SNAP. The white inset shows a cell containing lysosomes labeled by PQLC2-SNAP, and the blue inset shows a cell containing lysosomes negative for PQLC2-SNAP. Median pH (L) and area (M) of lysosomes in PQLC2KO-pH expressing the indicated PQLC2-SNAP mutants. Each bar shows the median ± SD values of n = 5000 analyzed lysosomes in n ≥ 3 independent experiments, shown as unfilled symbols. For (C–H), control-pH cells express an sgRNA targeting the cell surface protein CD81. Images in (I, K) are representative of n = 15 imaged fields. P-values calculated by ANOVA were adjusted using Tukey's multiple comparisons test. We observed that WT and mutant PQLC2 migrated at a size of ~15–18 kD by SDS-PAGE, at half of its predicted molecular weight of 32 kD (Fig. We reasoned that PQLC2-Flag undergoes cleavage, which yields a C-terminal fragment that is detected by the anti-Flag antibody. To investigate this possibility, we generated PQLC2KO cells that expressed Myc and Flag tags at the C or N-terminus (Myc-PQLC2-Flag or Flag-PQLC2-Myc) and analyzed expression of the resulting N and C-terminally tagged versions. C-terminally Flag or Myc-tagged PQLC2 migrated at the reduced size (Fig. However, we failed to detect the N-terminal tags (Fig. To determine the putative cleavage site, PQLC2-Flag was purified by immunoprecipitation and analyzed using Edman sequencing (Supplementary Fig. Ten amino acids corresponding to residues 172-181 were identified at the N-terminus, indicating potential cleavage after a glutamic acid residue that resides within a predicted intralysosomal loop (Supplementary Fig. Mutation of residues directly adjacent to the putative cleavage site did not impact the size of PQLC2-Flag, and a fragment containing only the residues C-terminal to the cleavage site was not expressed (Fig. Mutation of a larger segment of the intralysosomal loop produced a protein that migrated at ~30 kD (Fig. 7K), confirming that this region contains the cleavage site. Expression of the cleavage-deficient mutant in PQLC2KO cells rescued BiDAC-dependent EGFR-RFP degradation (Fig. We were surprised that the C-terminal cleavage product of full-length PQLC2 was sufficient to rescue lysosomal degradation of EGFR-RFP. To confirm that the cleaved form behaves like the putative full-length protein in cells, we determined the identity of its binding partners by co-immunoprecipitation followed by mass spectrometric analysis. 11a) and other components of the CSW complex (Supplementary Fig. However, both WT and mutant PQLC2 co-purified with V0 subunits of the V-ATPase, including AT6V0A1 and ATP6BV0D1 (Supplementary Fig. These data indicate that the cleaved form of PQLC2 can still bind its partners and suggest that the C-terminal product alone can rescue EGFR-RFP degradation, albeit by an unknown mechanism. We exploited the pH-sensitive fluorescence lifetime properties of mScarlet fluorescent protein30 by generating PQLC2 null cells expressing mScarlet-LAMP1 (PQLC2KO-pH), which localizes mScarlet to the lysosomal lumen. When cells were cultured in complete media, mScarlet-LAMP1 lifetime was longer in PQLC2KO-pH than control-pH cells, (Fig. 8D), which indicated that deletion of PQLC2 resulted in a higher lysosomal pH (Fig. PQLC2KO-pH cells also contained significantly enlarged lysosomes (Fig. To determine if the elevated pH and size of PQLC2-deficient lysosomes is associated with a general defect in lysosomal degradation, we tracked lysosome-dependent turnover of LC3 in cells treated with the autophagy inducer Torin-1. Autophagy induction in control cells was accompanied by conversion of LC3-I to LC3-II and degradation of LC3-II over time (Fig. 8H), suggesting that LC3-II was not efficiently turned over. Thus, loss of PQLC2 impairs lysosomal degradative functions. We posited that the incomplete EGFR-RFP degradation in PQLC2KO cells was a consequence of dysregulated lysosomal pH. We therefore asked if expression of the PQLC2 mutants that rescued EGFR-RFP degradation in RTK-FP cells also restored lysosomal pH in PQLC2KO-pH cells. The tag did not interfere with PQLC2 localization, as WT and cleavage deficient PQLC2-SNAP were localized to lysosomes, and the dileucine motif mutant partitioned between lysosomes and the plasma membrane (Fig. PQLC2-SNAP expression levels were variable between cells, with some cells expressing only mScarlet-LAMP1 and others expressing both mScarlet-LAMP1 and PQLC2-SNAP (Fig. We exploited this heterogeneity to determine if re-expression of PQLC2-SNAP restored the pH of individual lysosomes (Fig. Lysosomes negative for PQLC2-SNAP were of the same pH as those in PQLC2KO-pH cells (Fig. These data suggest that maintenance of lysosomal pH, a function not previously attributed to PQLC2, may be necessary for degradation of lysosomal substrates such as EGFR. Whereas the downstream consequences of blocking RTK-dependent signaling have been studied in detail, the effects of BiDAC-dependent RTK degradation have not been previously explored. In this study, we compared the consequences of inhibiting Her2 using RTK inhibitors, BiDACs, and antibodies. By using unbiased morphological profiling, we were able to identify phenotypic states associated with these perturbations. After selectively removing Her2 staining information from the morphological profile, we observed that BiDACs and their inactive counterparts induced similar phenotypes, suggesting that they have limited additional effects on cell state beyond inhibiting Her2. Thus, the greater potency of BiDACs may be due to a more sustained inhibition of canonical Her2 signaling rather than through loss of Her2 kinase-independent functions10. The distinctiveness of the AKT and PI3K inhibitor phenotypes was expected, as these kinases integrate many different growth signals, and their inhibition impacts pathways beyond those regulated by Her231. Surprisingly, our data indicate that anti-Her2 antibodies have an impact on cell physiology that is distinct from other inhibitory modalities, potentially due to their ability to simultaneously alter Her2 dimerization, localization, and expression1. A potential limitation to using pharmacological perturbations was that our BiDACs also inhibited Her2 kinase activity and triggered degradation of EGFR. Future studies can address this limitation by inducing Her2 degradation with genetically encoded degrons. Taken together, our unbiased profiling of mechanistically distinct Her2 inhibitors reveals a diverse spectrum of phenotypes associated with agents notionally known to target the same protein. Furthermore, the morphological profiling approach can be used to provide mechanistic insight into how BiDACs trigger degradation of their targets. The finding that BiDAC-mediated ubiquitination targeted RTKs to the endolysosomal pathway was surprising given our current understanding of ubiquitin-dependent trafficking of RTKs to lysosomes. Lysosomal EGFR degradation triggered by binding to EGF is thought to require modification of EGFR by monoubiquitin32 or lysine 63 (K63)-linked ubiquitin chains33. Our functional genomics screen identified UBE2R2 as an E2 that functions in BiDAC-dependent degradation of Her2-GFP. UBE2R2 has been shown to ubiquitinate its substrates with lysine 48 (K48)-linked ubiquitin chains in vitro34, a linkage which canonically targets ubiquitinated proteins for degradation by the proteasome. Our data suggest that BiDAC-dependent K48 ubiquitination is also sufficient to trigger internalization and delivery of RTKs to lysosomes. A recent study used a genetically encoded split ubiquitin system to label EGFR with K63 or K48 ubiquitin chains. K63 ubiquitination resulted in the expected targeting of EGFR to lysosomes35. Modification by K48 chains triggered EGFR endocytosis but resulted in mistargeting of EGFR to the endoplasmic reticulum. A direct comparison between this study and BiDAC-induced EGFR ubiquitination is not possible since the split ubiquitin system does not allow attachment of linkage-specific ubiquitin chains to lysines within EGFR. However, these results raise the possibility that BiDAC-induced K48 chains may recruit machinery that further ubiquitinates EGFR with K63-linked chains, thus promoting its trafficking to lysosomes36. Another possibility is that the sustained K48 ubiquitination induced by BiDACs is sufficient to promote internalization and sorting of RTKs to lysosomes, even in the absence of K63 ubiquitination. Unexpectedly, we found that inhibition of the proteasome blocked RTK degradation by interfering with BiDAC-induced internalization of RTKs from the plasma membrane, an explanation for why Lap-VHL-dependent Her2 degradation was previously reported to require active proteasomes10. We showed that inhibition of the proteasome partially blocked ubiquitination of Her2-GFP, potentially by impacting free ubiquitin pools. Interestingly, inhibition of the proteasome has been shown to inhibit EGF-induced delivery of EGFR to multivesicular bodies but not EGFR endocytosis23. This is consistent with our data showing that Carfilzomib does not fully block the EGF-induced reduction in surface EGFR despite potently inhibiting EGFR-RFP degradation. Further studies are needed to determine the identity and dynamics of ubiquitin chain linkages that govern BiDAC-induced internalization, trafficking, and degradation of plasma membrane proteins. Our screen uncovered a potential mechanism by which PQLC2 regulates lysosomal homeostasis and, consequently, degradation of lysosomal substrates. PQLC2 functions both as a CAA transporter and as a sensor of CAA abundance within lysosomes. Reduction of CAA availability stabilizes PQLC2 in a conformation that binds WDR41, which in turn recruits other members of the CSW complex to lysosomes28. The role of the CSW complex is poorly understood, but it has been proposed to activate GTPase activity in proteins that function in vesicular trafficking and autophagy37. By reintroducing PQLC2 mutants into PQLC2-null cells, we found that CAA transport and CSW binding were dispensable for rescue of EGFR-RFP degradation, lysosome size, and pH. These results agree with a study using C. elegans, which examined the impact of deleting the PQLC2 homolog LAAT-1 on lysosome function27. Loss of LAAT-1 resulted in lysosome enlargement, impaired lysosomal degradation, and delayed development of the mutant worms. Supplementation of LAAT-1-null worms with CAA rescued development but did not impact lysosomal phenotypes, suggesting that while LAAT-1-dependent CAA transport was sufficient to maintain cytosolic CAA levels to support organismal growth, LAAT-1 regulated lysosomal homeostasis through a different mechanism. Our results indicate that PQLC2 regulates lysosomal pH, potentially through its interaction with the V-ATPase at the lysosomal membrane. Interestingly, PQLC2 has optimal CAA transport activity within the pH range found in lysosomes25. Thus, regulation of V-ATPase activity by PQLC2 may be required for lysosomal acidification, which in turn sets the conditions under which PQLC2 optimally transports CAA. We also found that the reintroduced PQLC2 undergoes cleavage within an intra-lysosomal loop, yielding a C-terminal fragment. While we were unable to obtain an antibody to confirm the size of endogenously expressed PQLC2, previous reports support our findings that the dominant form of PQLC2 is expressed as a cleaved fragment of the full-length protein27,28, including a study in which PQLC2 was fused to a hemagglutinin tag at its genomic locus28. Our results do not rule out the possibility that a minor, undetectable fraction of PQLC2 does not undergo cleavage, as reintroduction of a non-cleaved PQLC2 mutant was sufficient to rescue lysosomal phenotypes and restore EGFR degradation. Additional studies are needed to investigate the relationship between PQLC2 cleavage and its CAA transport and sensing functions. SKOV3 and SKBR3 cell lines were cultured at 37 °C with 5% CO2 in RPMI 1640 medium supplemented with 10% fetal bovine serum (FBS) and Glutamax (Gibco). Parental SKOV3 and SKBR3 lines were obtained from ATCC. RTK-FP SKOV3 cells (Sigma-Aldrich) were subcloned using FACS to yield a homogenously expressing clonal line. RTK-FP PQLC2KO cells were generated by infection of RTK-FP cells with lentivirus containing LentiCRISPR v2 encoding Cas9 and sgRNAs targeting exon 3 or exon 2 of the SLC66A1 gene. Virus-containing media from HEK293T cells was added at a 1:1 v/v ratio to RTK-FP cells with 5 μg/mL Polybrene infection reagent. The medium was changed after 24 h. After an additional 24 h of incubation, infected cells were selected by treatment with 2 μg/mL puromycin for 72 h to generate a knockout pool. Knockout of PQLC2 in pools was confirmed using genomic sequencing. Clonal lines expressing sgRNA #1 or sgRNA #2 were isolated by limited dilution cloning, and knockout of PQLC2 in individual clones was confirmed by their inability to fully degrade EGFR-RFP. SKOV3 PQLC2KO cells were similarly generated by infection of SKOV3 cells with LentiCRISPR v2 encoding PQLC2 sgRNA #1, selection with puromycin, and subcloning. Control cells were generated by infection of RTK-FP or SKOV3 cells with lentivirus containing LentiCRISPR v2 encoding an sgRNA targeting the cell surface protein CD81. RTK-FP LAMP1-SNAP and RAB7A-SNAP cells were generated by transient transfection of RTK-FP cells with plasmids encoding LAMP1-SNAP or RAB7A-SNAP, followed by selection in 50 µg/ml Geneticin (Thermo Fisher Scientific). RTK-FP PQLC2KO cells expressing PQLC2 rescue constructs or LAMP1-SNAP were generated by infection of PQLC2KO clonal cells expressing PQLC2 sgRNA #1 with a dilution series of lentivirus containing a construct encoding PQLC2-Flag or LAMP1-SNAP under control of a dox-inducible promoter. Infected cells were selected by treatment with 10 μg/mL blasticidin for 7 days and the multiplicity of infection (MOI) was determined by CellTiter-Glo assay (Promega). Lines infected with a dilution that yielded an MOI ≤ 0.3 were used for downstream experiments. LAMP1-SNAP-expressing cells were further enriched by FACS. Prior, expression of LAMP1-SNAP was induced by treatment with 500 ng/mL doxycycline for 48 h, and LAMP1-SNAP was fluorescently labeled by treating cells with 1 µM SNAP-Cell 647-SiR (New England Biolabs) for 30 min at 37 °C. Cells were pelleted and washed twice prior to sorting of the 647-SiR-positive population. PQLC2KO-pH cells were generated by infection of SKOV3 PQLC2KO cells expressing PQLC2 sgRNA #1 with lentivirus containing a construct encoding mScarlet-LAMP1 under control of a dox-inducible promoter, followed by selection with 10 µg/mL blasticidin. PQLC2KO-pH expressing PQLC2 rescue constructs were generated by infection with virus containing a construct encoding PQLC2-SNAP under dox control. UBE2R2KO cells were generated by electroporation of 200 × 103 RTK-FP cells with a mixture of three sgRNAs, 5′ CGAGCUGGUCAUCUGCUGCU 3′, 5′ GGAGGGCUUCCGGAUCACCC 3′, and 5′ CCGCCUUCGUAGAGGGUGUU 3′ (EditCo Bio, Inc.). A single sgRNA targeting CD81, 5′ GTTGGCTTCCTGGGCTGCTA 3′ was used to generate control cells. 120 pmol of sgRNA was combined with 104 pmol of Alt-R Cas9 Nuclease V3 (IDT) in a total volume of 5 µL and incubated at room temperature for 15 min. A master mix from SF Cell Line 4D-Nucleofector X Kit S (Lonza Bioscience) was prepared according to manufacturer's instructions. Immediately after addition of the sgRNA/Cas9 complex to the master mix, cells were resuspended and electroporated using a 4D-Nucleofector X System (Lonza Biosciences) using program no. The oligonucleotide sequence preceding the protospacer is 5′ caccgCATCTCTTACAGACCTACA 3' for sgRNA #1 and 5′ caccgGAGACTCCTGCAACCTCAT 3′ for sgRNA #2. The dox-inducible PQLC2 lentiviral expression construct was generated by inserting PQLC2-myc-flag (OriGene Technologies) between the NheI and NotI sites of the custom dox-inducible lentiviral expression vector pSHUSHV2, encoding a blasticidin resistance cassette and the reverse tetracycline-controlled transactivator. PQLC2 constructs containing N and C-terminal Myc or Flag tags were generated by incorporating sequences encoding these tags into the PCR amplification primers. LAMP1-SNAP and SNAP-RAB7A were generated by amplification of the LAMP1 or RAB7A cDNA sequences (Origene Technologies) and the SNAP DNA sequence (New England Biolabs), followed by joining using Gibson assembly and insertion into pSHUSHV2 or a vector containing a G418 resistance cassette. mScarlet-LAMP, previously described30, was amplified and inserted into pSHUSHV2. Cells were harvested by scraping or trypsinization and washed with PBS after pelleting by centrifugation. Cell pellets were lysed in 1% SDS/water and solubilized by sonication. Cell lysates were boiled at 100 °C for 10 min, and protein concentrations were determined using Bicinchoninic acid (BCA) assay (Thermo Fisher Scientific). 20–40 μg of lysate was combined with Laemmli buffer containing 2.5% v/v ß-Mercaptoethanol (BME), heated at 65 °C for 10 min, and separated by SDS-PAGE on 4–20% Criterion TGX gels (Bio-Rad) at 150 V for 75 min. For the Torin-1 time course, proteins were separated on 4–12% Criterion gels (Bio-Rad) in MES buffer to achieve better separation between LC3-I and LC3-II. The proteins were transferred to 0.22 μm nitrocellulose membranes (Bio-Rad) using the Trans-Blot Turbo transfer system (Bio-Rad). The following day, membranes were washed three times for 5 min in PBST and incubated in secondary anti-mouse IRDye 680RD or anti-rabbit IRDye 800CW antibodies (LI-COR) for 30 mins at room temperature. After three washes in PBST, the membranes were imaged on an Odyssey Imaging System (LI-COR). The following primary antibodies were diluted 1:1000 v/v, unless otherwise noted, in 5% BSA/PBST and used for immunoblots in this study: anti-Her2 (Cell Signaling Technology cat no. D38B1), anti-GAPDH diluted 1:5000 (Cell Signaling Technology no. ab13970), anti-K48-linkage Specific Polyubiquitin Antibody (Cell Signaling Technology no. Uncropped blots are provided within the Source Data file. Six 15-cm plates of RTK-FP cells were cultured until confluence for each treatment condition. Cells were treated with BiDACs with or without Carfilzomib for 60 min, scraped into the treatment media, and pelleted at 4 °C for 5 min. After two washes with cold PBS, cells were lysed in cold RIPA buffer containing 50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 mM EDTA, 0.1% Sodium dodecyl sulfate (SDS), 1% Triton X-100 and 1% Sodium deoxycholate, supplemented with cOmplete EDTA-free Protease Inhibitor Cocktail (Thermo Fisher Scientific) and 5 mg/mL N-ethylmaleimide (NEM, Sigma). The lysates were rotated for 30 min at 4 °C and centrifuged for 15 min at 20 K × g. The protein concentration of the supernatant was quantified using BCA assay. Twenty five microliters of Chromotek GFP-Trap agarose beads (Proteintech Group) were washed twice with RIPA buffer, combined with 8 mg of lysate per sample and rotated at 4 °C for 2 h. The beads were pelleted by centrifugation at 2000 × g, and washed 4× with cold RIPA buffer lacking NEM or protease inhibitors, followed by 2× washes with cold buffer containing 50 mM Tris-HCl pH 7.5, 150 mM NaCl, and 0.5 mM EDTA. Bound proteins were eluted by boiling beads in 2× Laemmli buffer. The supernatant was combined with 2.5% BME, heated at 65 °C for 10 min, and separated as described above. Twenty micrograms of input and flowthrough was loaded to assess efficiency of the affinity purification. Proteins were transferred to 0.45-μm PVDF membranes (Bio-Rad) using the high MW transfer protocol, blotted using anti-GFP and anti-K48-linkage antibodies, and visualized using anti-Rabbit HRP and anti-Chicken HRP antibodies (Jackson Immuno Research) and the SuperSignal West Dura detection reagent (Thermo Fisher Scientific). Live RTK-FP cells expressing LAMP1-SNAP were imaged using a spinning disk confocal microscope (Nikon) equipped with a 60× 1.4 na oil objective, in an environmental chamber set to 37 °C and 5% CO2. Cells were seeded at a density of 10,000 cells per well in a glass bottom #1.5H-N 96-well plate (Cellvis) containing 100 μL of complete phenol-red free RPMI 1640 per well. Five hundred nanograms per milliliter of dox was added to the media to induce expression of LAMP1-SNAP. After a 48 h incubation, cells were treated with 1 μM Lap-VHL or Lap-VHLInactive and 100 nM Bafilomycin-A1 (Selleck Chemicals). To label LAMP1-SNAP or SNAP-RAB7A immediately prior to imaging, cells were treated with 1 µM SNAP-Cell 647-SiR for 30 min at 37 °C, and subsequently washed three times with complete media while adhered to the imaging chamber. After an additional 30 min incubation, the media was replaced with fresh media containing 1 µg/mL Hoechst 33342 Trihydrochloride Trihydrate (Invitrogen) to stain nuclei. Colocalization analysis and lysosome quantification was performed using CellProfiler (Broad Institute). Confocal image slices for the RFP and SNAP channels were combined into a single max intensity projection. CellProfiler was used to calculate and apply an illumination function to each image and align the images. Lysosomes were identified using the SNAP channel and a Pearson correlation coefficient between the RFP and SNAP signals was calculated on a per lysosome basis. Time-lapse imaging of live RTK-FP cells was performed using a Opera Phenix Plus System (Revvity Health Sciences) set to 37 °C and 5% CO2. Cells were seeded as above and treated with 1 µM Lap-VHL in combination with degradation inhibitors. To label nuclei, cells were treated with 50 nM SiR-DNA (Cytoskeleton, Inc.), and single plane images across 20 fields per well were acquired in confocal mode every 30 min. At each time point, the number of puncta in each well was determined using Harmony imaging and analysis software (Revvity) and normalized by the number of SiR-DNA-positive nuclei in the well. For immunofluorescence microscopy analysis of PQLC2-Flag expression, RTK-FP cells were seeded as above and incubated for 48 h with 500 ng/mL dox. Cells were washed three times with PBS and fixed with 4% paraformaldehyde (PFA) in PBS for 15 min at room temperature. Following fixation, cells were washed three times with PBS and permeabilized with 0.1% Triton X-100 in PBS for 5 min at room temperature. Cells were blocked in 1% BSA in PBS for 30 min and stained for 2 h with a 1:500 v/v dilution of anti-Flag monoclonal antibody (M2, Sigma-Aldrich) in blocking solution. After three washes with blocking solution, cells were stained for 1 h with anti-Mouse IgG Fragment conjugated with Alexa Fluor 647 (Cell Signaling Technologies) diluted 1:5000 v/v in blocking solution, washed 3 times, and imaged on the Opera Phenix Plus System. Analysis of SKBR3 cell proliferation was performed using an Incucyte SX5 live-cell imaging and analysis instrument (Sartorius). Five thousand cells per well were seeded in a 96-well tissue culture plate and incubated for 24 h prior to drug treatment and start of imaging. Images were taken in the phase channel every 6 h using the 10× objective over a total period of 120 h, and percent confluence over time was determined using the Incucyte image analysis software. Flow cytometric analysis was performed using the LSRFortessa X-20 cell analyzer (BD Biosciences). For analysis of total RTK-FP levels, cells were collected from 96-well plates by trypsinization, transferred to V-bottom plates, and analyzed using the high throughput sampler. For staining of surface RTKs, trypsinized cells were transferred to V-bottom plates and pelleted at 300 × g for 5 min. Cells were resuspended in 50 µL of a solution containing 0.5% v/v BSA in PBS, 2 µg/mL anti-EGFR Alexa Fluor 405-conjugated antibody (R&D Systems), and 2 µg/mL anti-Her2 Alexa Fluor 647-conjugated antibody (R&D Systems). After a 30 min incubation at room temperature, 100 µL of 0.5% BSA/PBS was added to each well, and the plates were centrifuged at 300 × g for 5 min. Plates were washed once with 150 µL of 0.5% BSA/PBS, and the cells were resuspended in FACS buffer containing 2% FBS in PBS for analysis. Each MFI value was normalized by subtracting the background MFI determined from analysis of parental SKOV3 cells and dividing by the value obtained from untreated cells within each treatment condition. For analysis of apoptosis induction, 25,000 cells per well were seeded in a 96-well plate and treated with BiDACs for 24–72 h. Prior to analysis, cells were incubated with 0.1% Caspase-3/7 Green Flow Cytometry Assay reagent (Thermo Fisher Scientific) for 25 min. Cells were washed with PBS and transferred to a U-bottom plate in PBS containing 2% FBS prior to analysis. RTK-FP PQLC2KO cells expressing PQLC2-Flag were treated with 500 ng/mL dox for 48 h, and lysates were prepared as described above. Sixty milligrams of lysate in RIPA buffer was combined with 150 µL of anti-Flag M2 magnetic bead slurry (Sigma-Aldrich) and rotated for 2 h at 4 °C. Beads were separated from lysates using a magnetic tube rack, washed five times with cold RIPA buffer, and bound proteins were eluted by boiling beads in 2× Laemmli buffer for 10 min. The supernatant was transferred to a new tube, combined with 2.5% v/v ß-Mercaptoethanol, and heated at 65 °C. Proteins were separated by SDS-PAGE using a 12.5% Bis-Tris gel (Bio-rad) at 150 V for 75 min. The proteins were transferred to 0.22 μm PVDF membranes (Bio-Rad) using the Trans-Blot Turbo Transfer System (Bio-Rad). The membranes were stained with Coomassie Gel Code Blue at room temperature for 10 min, and destained several times with 50% methanol. A PVDF band corresponding to the expected size of PQLC2-Flag was cut from the membrane and further destained with 50% methanol in a tube, followed by two washes with water. Ten cycles of Edman sequencing analysis were performed (Creative Proteomics) using an ABI Procise 494HT protein sequencing system (Thermo Fisher Scientific). Cells were seeded in 384-well PhenoPlates (Perkin Elmer) using a Fluent Liquid Handler (Tecan). Prior to seeding, cells were counted using a Vi-Cell XR cell counter (Perkin Elmer), and their concentration was adjusted to generate the required stock concentrations that varied depending on the final time point of analysis. Fifty microliters of cell suspension was added to each well at a density of 2500 cells per well for the 2 h and 6 h time points, 1500 cells per well for the 24 h time point, 1000 cells per well for the 48 h time point, and 500 cells per well for the 72 h time point. After seeding, the plates were left at room temperature for 30 min to allow uniform attachment of cells to the plate. The plates were subsequently transferred to an incubator for 16 h. Cells were treated with experimental compounds using the Echo 555 Acoustic Liquid Handler (Beckman Coulter). Specifically, low dead volume Echo source plates containing 10 μL of compound per well were prepared and stored at −80 °C. Source plates were limited to a maximum of three thaws at room temperature and centrifuged at 1000 rpm for 5 min prior to dosing. After dosing, phenoplates were returned to an incubator for the duration of the 2–72 h treatments. After incubation with drugs, the cells were stained with 0.5 μM of MitoTracker Deep Red FM (Invitrogen) in cell culture media by adding 10 μL of a 3 μM MitoTracker solution to each well using a Multidrop Combi Reagent Dispenser (Thermo Fisher). The cells were then incubated at 37 °C for 30 min. During this time, the cell staining solution was prepared as follows: 1% BSA solution was made by dissolving fresh Bovine Serum Albumin (BSA) powder in Hank's Balanced Salt Solution (HBSS), followed by filtration through a 0.22 μm filter. Next, a 0.1% v/v Triton X-100 solution was prepared by mixing 1% BSA/HBSS with a 10% Triton X-100 stock solution. Dyes from the Cell Painting panel were manually added at the following concentrations to generate the post-fixation permeabilization and staining solution containing 1 μg/mL Phalloidin Conjugates CF430 (Biotium), 1 μg/mL Hoechst 33342 Trihydrochloride Trihydrate (Invitrogen) and 1.5 μg/mL Wheat Germ Agglutinin (WGA) Alexa Fluor 555 (Invitrogen). To stain Her2, Trastuzumab (Herceptin, Genentech) and Pertuzumab (Perjeta, Genentech) were labeled using the DyLight 488 Antibody Labeling Kit (Life Technologies Corporation) and added to the staining solution at a concentration of 1 μg/mL. After staining mitochondria, the plates were transferred with lids removed to a BioTek EL406 Microplate Washer Dispenser with BioStack (Agilent). Twenty microliters of a 16% PFA solution was added to each well using a syringe pump to yield a final concentration of 4% PFA. Cells were fixed for 20 min at room temperature, and PFA was removed using a wash manifold on the EL406. Excess PFA was removed by washing each well twice with 60 μL of HBSS. Cells were subsequently incubated in 20 μL of the permeabilization and staining solution for 30 min at room temperature followed by two washes with HBSS. Upon completion of staining, all wells were filled with 60 μL of an HBSS solution containing 0.05% v/v sodium azide, and the plates were sealed before imaging. Each stained 384-well plate was imaged using an Opera Phenix Plus System (Revvity Health Sciences) by acquiring 5-channel fluorescence and digital phase contrast images across 9 fields in the center of each well with a 20× objective. For generation of deep learning image embeddings, images were down sampled to 512 × 512 × 6, with each channel corresponding to an image acquisition channel (see “Morphological profiling” methods). Separate deep convolutional neural networks were trained using all images for each cell type. All networks had a modified ResNet38 architecture and were trained using ArcFace loss39, with each image's class label corresponding to a combination of its timepoint × dose × perturbation. This training objective encourages the network to minimize differences in representation due to batch effects between technical replicates. All image embeddings were further normalized by subtracting the mean of all DMSO embeddings per plate to mitigate plate-level batch effects. For cell segmentation and image intensity analysis, all channels other than the Hoechst nuclei channel were grouped as “cyto” channels. Cyto channels were combined by first linearly rescaling each individual channel image to have its 3rd to 97th percentile pixel intensity values fall in the range [0, 1], with all outliers clipped to 0 or 1. The channels were then combined by taking the per pixel maximum value across all rescaled cyto channel images. The nuclei channel and composite cyto channel were then down-sampled to 512 × 512 and processed using the Cellpose model40 to derive individual cell segmentation masks. For each channel, the median intensity value per cell was then computed. t-SNE plots were generated by averaging the image embeddings across the timepoint × dose × perturbation groups for each plate. The two-dimensional t-SNE embeddings were generated using a multicore implementation of the Barnes-Hut t-SNE algorithm41,42. The perplexity value was manually tuned for each dataset. Cells were seeded at a density of 10,000 cells per well in a Cellvis 8-well chambered coverglass and treated with 500 ng/ml dox for 48 h. To image SNAP-tagged PQLC2, cells were treated with 1 μM SNAP-Cell 647 SiR for 30 min and washed 3 times with complete RPMI 1640 medium. The medium was replaced again following a 30 min incubation with complete RPMI 1640 lacking phenol red. During imaging, cells were incubated at 36 °C and 5% CO2 using a stage top incubator (Okolab). Ionophores consisting of 10 μM nigericin and 2 μM monensin were also added to the imaging buffer to equalize proton gradients across cellular membranes. Fluorescence lifetime imaging was performed on an SP8 point scanning confocal microscope (Leica) equipped with a time-correlated single photon counting (TCSPC) add-on (PicoQuant GmbH, Berlin, Germany). mScarlet was excited with the 561 nm output of the white light laser at an average power of ~10–20 μW and a repetition rate of 40 MHz. Emission was collected with a 100×/1.4 NA oil objective (HC PL APO CS2, Leica), and photon arrival times were determined with the built-in HyD detectors and the PicoHarp 300 TCSPC module. Scanning was performed with the built-in Leica non-resonant scanner with a total time per field of view of 20 s. Images were binned to pixel sizes of 200 nm × 200 nm to obtain adequate photons for lifetime determination. All imaging was performed with the confocal pinhole set to 1 Airy unit at 580 nm. mScarlet was imaged with the 561 nm output of the white light laser, and emissions between 587 and 625 nm were collected. SiR647 SNAP was imaged with the 640 nm output of the white light laser (average power 0.5 μW) and emissions between 660 and 788 nm were collected. To reduce motion artifacts from lysosome movement, image capture was alternated between the two illumination colors between lines. Because the PicoQuant software did not support between-line switching, a custom parser was used to process the raw TCSPC output and reconstruct an image. Parser code is available on GitHub (https://github.com/AndrewGYork/tools/blob/master/picoquant_tttr.py). For two color images, the total acquisition time per field of view was ~40 s. TCSPC images were used to calculate mean arrival time at each pixel either within the PicoQuant SymPhoTime software (one color imaging) or with custom parser code (two color imaging). Fluorescence intensity images were generated by summing the photons at each pixel over the fluorescence lifetime decay. To segment fluorescence lifetimes and intensity images, instance segmentation of lysosomes was performed using the mScarlet-LAMP1 emission channel and the “StarDist2D” model trained on the “2D_versatile_fluo” dataset44. The model distinguishes individual lysosomes within 2D images by recognizing their star-convex shape. These quantifications were performed using Scikit Image's region properties function45. For fluorescence lifetime quantification, only those pixels with a photon count above five were used to calculate the mean fluorescence lifetime value. pH calibration (ionophore) data were fit to a four-parameter logistic function to extract a calibration curve, where tau is the lifetime, min and max are the minimum and maximum lifetime values observed (at low and high pH respectively), and b is the slope factor (similar to the Hill coefficient) (Eq. Median lifetime of all lysosomes in each condition was converted to pH using the corresponding calibration curves. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The genomic screen data discussed in this publication have been deposited in NCBI's Gene Expression Omnibus and are accessible through GEO Series accession number GSE291219. Precomputed image embeddings are available on Zenodo [https://doi.org/10.5281/zenodo.15104083]. Raw images for morphological profiling experiments are available upon request. Code for analysis of morphological profiling data is available on Zenodo [https://doi.org/10.5281/zenodo.15104083]. & Yasri, A. Antibodies directed against receptor tyrosine kinases. Huang, L., Jiang, S. & Shi, Y. Tyrosine kinase inhibitors for solid tumors in the past 20 years (2001–2020). Pottier, C. et al. Tyrosine kinase inhibitors in cancer: breakthrough and challenges of targeted therapy. Structures of the HER2-HER3-NRG1β complex reveal a dynamic dimer interface. Targeting ligand-activated ErbB2 signaling inhibits breast and prostate tumor growth. Zhang, Y. et al. Neratinib induces ErbB2 ubiquitylation and endocytic degradation via HSP90 dissociation in breast cancer cells. Békés, M., Langley, D. R. & Crews, C. M. PROTAC targeted protein degraders: the past is prologue. Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4. Winter, G. E. et al. Phthalimide conjugation as a strategy for in vivo target protein degradation. The advantages of targeted protein degradation over inhibition: a RTK case study. Cell painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes. A. et al. Optimizing the cell painting assay for image-based profiling. A., Carpenter, A. E. & Singh, S. High-dimensional gene expression and morphology profiles of cells across 28,000 genetic and chemical perturbations. Chandrasekaran, S. N., Ceulemans, H., Boyd, J. D. & Carpenter, A. E. Image-based profiling for drug discovery: due for a machine-learning upgrade? Eulenberg, P. et al. Reconstructing cell cycle and disease progression using deep learning. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Crystal structure of the Cul2-Rbx1-EloBC-VHL ubiquitin ligase complex. Li, J. et al. Cullin-RING ligases employ geometrically optimized catalytic partners for substrate targeting. A., Cowley, G. S., Fischer, E. S. & Ebert, B. L. Genome-wide screen identifies cullin-RING ligase machinery required for lenalidomide-dependent CRL4CRBN activity. Tomas, A., Futter, C. E. & Eden, E. R. EGF receptor trafficking: consequences for signaling and cancer. Longva, K. E. et al. Ubiquitination and proteasomal activity is required for transport of the EGF receptor to inner membranes of multivesicular bodies. Mimnaugh, E. G., Chen, H. Y., Davie, J. R., Celis, J. E. & Neckers, L. Rapid deubiquitination of nucleosomal histones in human tumor cells caused by proteasome inhibitors and stress response inducers: effects on replication, transcription, translation, and the cellular stress response. Jézégou, A. et al. Heptahelical protein PQLC2 is a lysosomal cationic amino acid exporter underlying the action of cysteamine in cystinosis therapy. & Zoncu, R. Picking the arginine lock on PQLC2 cycling. & Wang, X. LAAT-1 is the lysosomal lysine/arginine transporter that maintains amino acid homeostasis. Amick, J., Tharkeshwar, A. K., Talaia, G. & Ferguson, S. M. PQLC2 recruits the C9orf72 complex to lysosomes in response to cationic amino acid starvation. & Ferguson, S. M. Receptor-like role for PQLC2 amino acid transporter in the lysosomal sensing of cationic amino acids. & Ingaramo, M. mScarlet fluorescence lifetime reports lysosomal pH quantitatively. Multiple monoubiquitination of RTKs is sufficient for their endocytosis and degradation. Huang, F. et al. Lysine 63-linked polyubiquitination is required for EGF receptor degradation. Mechanism of millisecond Lys48-linked poly-ubiquitin chain formation by cullin-RING ligases. Ubiquiton-An inducible, linkage-specific polyubiquitylation tool - PubMed. Tang, D. et al. Cryo-EM structure of C9ORF72–SMCR8–WDR41 reveals the role as a GAP for Rab8a and Rab11a. Deep Residual Learning for Image Recognition. in 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR) 770–778 (2016). Deng, J. et al. ArcFace: additive angular margin loss for deep face recognition. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Maaten, L.van der Accelerating t-SNE using tree-based algorithms. Carmody, W. R. Easily prepared wide range buffer series. Schmidt, U., Weigert, M., Broaddus, C. & Myers, G. Cell detection with star-convex polygons. Medical Image Computing and Computer Assisted Intervention – MICCAI 2018 Vol. van der Walt, S. et al. scikit-image: image processing in Python. We would like to thank Dr. Stewart Fisher (C4 Therapeutics, Inc.) for his expertise regarding generation of BiDAC degraders and Dr. Isabel Chiu (C4 Therapeutics, Inc.) for supporting collaborations for the manuscript. We thank Dr. Daniel Gottschling and Dr. Hannah Baddock (Calico Life Sciences, LLC) for providing comments and suggestions for the manuscript. Calico Life Sciences LLC, South San Francisco, CA, USA Sammy Villa, Qumber Jafri, Julia R. Lazzari-Dean, Manjot Sangha, Niclas Olsson, Austin E. Y. T. Lefebvre, Zhenghao Chen, Brian Y. Feng, Aaron H. Nile, David Stokoe & Kirill Bersuker You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Morphological profiling experiments and data modeling were performed by Q.J. Lifetime microscopy experiments and image processing were conducted by J.L.R. designed and synthesized the BiDAC molecules. generated RTK-FP cells and oversaw collaborations with C4 Therapeutics. contributed to discussions and supervised the project. The authors declare no competing interests. Nature Communications thanks Chang Hoon Ji and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Villa, S., Jafri, Q., Lazzari-Dean, J.R. et al. BiDAC-dependent degradation of plasma membrane proteins by the endolysosomal system. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            