
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-18</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41596-025-01313-x'>Harnessing intermediate-scale bioreactors for next-generation macrophage production and application</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 17:10:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Macrophages are crucial in immune responses, tissue repair and homeostasis, making them prime candidates for translational applications. Induced pluripotent stem cell (iPS cell)-derived macrophages hold considerable promise for regenerative medicine, cancer therapy, inflammatory disease treatment and in vitro bioassays. However, cost-effective, standardized intermediate-scale bioreactor systems tailored for early-stage research and drug discovery in academia remain limited. Here, we present an extension of our previously published protocol that is feeder free, semi-defined and user friendly, enabling the standardized production of iPS cell-derived macrophages in an intermediate (10–50 mL)-scale benchtop bioreactor. This Protocol can be implemented by users with basic iPS cell culture experience without requiring advanced bioprocessing expertise. This method consists of two primary endpoints: the generation of mesoderm-primed aggregates with hematopoietic potential, termed hemanoids, and the standardized production of iPS cell-derived macrophages that are ready for downstream applications. This Protocol enables continuous macrophage generation in long-term cultures, with a minimum of five consecutive collections, yielding an average of 2–3 × 107 cells per collection per vessel. Four vessels operate independently, each with a maximum culture volume of up to 50 mL, while critical process parameters (CO2, temperature and pH) are monitored. This semi-automated platform and in-process monitoring improve process control, leading to higher yields, reproducibility and cell quality compared with other systems. The simplified process spans 24 d, starting from single-cell iPS cells to ready-to-use macrophages. By bridging the gap between small- and large-scale systems, this approach provides scalable, standardized manufacturing of iPS cell-derived macrophages, making it a valuable tool for academics focused on human immune cells such as macrophages. This Protocol Extension details the standardized production of iPS cell-derived macrophages in an intermediate-scale benchtop bioreactor. This method is divided into two stages: the generation of mesoderm-primed aggregates with hematopoietic potential, termed hemanoids, and the standardized production of iPS cell-derived macrophages that are ready for downstream applications. The use of standardized intermediate-scale bioreactor systems is tailored for early-stage research and drug discovery in academic and industrial settings. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout We declare that the data supporting the findings of this study are available within the supporting Protocol13 and its Supplementary information. The scRNA sequencing datasets are deposited in the NCBI GEO repository under accession number GSE268458. Should any raw data files be needed in another format, they are available from the corresponding author upon reasonable request. Ng, E. S. et al. Long-term engrafting multilineage hematopoietic cells differentiated from human induced pluripotent stem cells. Frenz-Wiessner, S. et al. Generation of complex bone marrow organoids from human induced pluripotent stem cells. Leung, C. M. et al. A guide to the organ-on-a-chip. Rowe, R. G. & Daley, G. Q. Induced pluripotent stem cells in disease modelling and drug discovery. Kyrysyuk, O. & Wucherpfennig, K. W. Designing cancer immunotherapies that engage T cells and NK cells. Park, M. D., Silvin, A., Ginhoux, F. & Merad, M. Macrophages in health and disease. Alsinet, C. et al. Robust temporal map of human in vitro myelopoiesis using single-cell genomics. Lee, B. et al. Cell culture process scale-up challenges for commercial-scale manufacturing of allogeneic pluripotent stem cell products. Godoy, P. et al. Recent advances in 2D and 3D in vitro systems using primary hepatocytes, alternative hepatocyte sources and non-parenchymal liver cells and their use in investigating mechanisms of hepatotoxicity, cell signaling and ADME. Ackermann, M. et al. Bioreactor-based mass production of human iPSC-derived macrophages enables immunotherapies against bacterial airway infections. Ackermann, M. et al. Continuous human iPSC-macrophage mass production by suspension culture in stirred tank bioreactors. Abdin, S. M. et al. Scalable generation of functional human iPSC-derived CAR-macrophages that efficiently eradicate CD19-positive leukemia. Ackermann, M. et al. Standardized generation of human iPSC-derived hematopoietic organoids and macrophages utilizing a benchtop bioreactor platform under fully defined conditions. Stem Cell Res. Lachmann, N. et al. Large-scale hematopoietic differentiation of human induced pluripotent stem cells provides granulocytes or macrophages for cell replacement therapies. Stem Cell Rep. 4, 282–296 (2015). Karlsson, K. R. et al. Homogeneous monocytes and macrophages from human embryonic stem cells following coculture-free differentiation in M-CSF and IL-3. Borys, B. S. et al. Optimized serial expansion of human induced pluripotent stem cells using low-density inoculation to generate clinically relevant quantities in vertical-wheel bioreactors. Stem Cells Transl. Cuesta-Gomez, N. et al. Suspension culture improves iPSC expansion and pluripotency phenotype. Stem Cell Res. Kwok, C. K. et al. Scalable expansion of iPSC and their derivatives across multiple lineages. Abdin, S. M. et al. Sensor macrophages derived from human induced pluripotent stem cells to assess pyrogenic contaminations in parenteral drugs. Pouyanfard, S. et al. Human induced pluripotent stem cell-derived macrophages ameliorate liver fibrosis. Nenasheva, T. et al. Macrophages derived from human induced pluripotent stem cells are low-activated “naive-like” cells capable of restricting mycobacteria growth. Schinke, M. Polarization of human iPSC-derived macrophages directs their immunological response to secondary pro-inflammatory stimuli. Cao, X. et al. Differentiation and functional comparison of monocytes and macrophages from hiPSCs with peripheral blood derivatives. Stem Cell Rep. 12, 1282–1297 (2019). Monkley, S. et al. Optimised generation of iPSC-derived macrophages and dendritic cells that are functionally and transcriptionally similar to their primary counterparts. Mass, E., Nimmerjahn, F., Kierdorf, K. & Schlitzer, A. Tissue-specific macrophages: how they develop and choreograph tissue biology. Kulle, A., Thanabalasuriar, A., Cohen, T. S. & Szydlowska, M. Resident macrophages of the lung and liver: the guardians of our tissues. Gomez Perdiguero, E. et al. Tissue-resident macrophages originate from yolk-sac-derived erythro-myeloid progenitors. Article PubMed Williamson, A. E. et al. Discovery of an embryonically derived bipotent population of endothelial-macrophage progenitor cells in postnatal aorta. Hoeffel, G. et al. C-Myb+ erythro-myeloid progenitor-derived fetal monocytes give rise to adult tissue-resident macrophages. Kent, G. M. et al. Human liver sinusoidal endothelial cells support the development of functional human pluripotent stem cell-derived Kupffer cells. Buchrieser, J., James, W. & Moore, M. D. Human induced pluripotent stem cell-derived macrophages share ontogeny with MYB-independent tissue-resident macrophages. Stem Cell Rep. 8, 334–345 (2017). Lock, R. I. et al. Macrophages enhance contractile force in iPSC-derived human engineered cardiac tissue. Landau, S. et al. Primitive macrophages enable long-term vascularization of human heart-on-a-chip platforms. Long, C. et al. Effects of macrophages on the proliferation and cardiac differentiation of human induced pluripotent stem cells. Qi, L. et al. Human iPSC-derived proinflammatory macrophages cause insulin resistance in an isogenic white adipose tissue microphysiological system. Park, D. S. et al. iPS-cell-derived microglia promote brain organoid maturation via cholesterol transfer. Zhang, W. et al. Microglia-containing human brain organoids for the study of brain development and pathology. Article PubMed Schafer, S. T. et al. An in vivo neuroimmune organoid model to study human microglia phenotypes. Maxwell, K. G. & Millman, J. R. Applications of iPSC-derived beta cells from patients with diabetes. Cheng, W., Zeng, Y. & Wang, D. Stem cell-based therapy for pulmonary fibrosis. Stem Cell Res. Gazdhar, A. et al. The secretome of induced pluripotent stem cells reduces lung fibrosis in part by hepatocyte growth factor. Stem Cell Res. Tuberculosis in otherwise healthy adults with inherited TNF deficiency. Neehus, A. L. et al. Human inherited CCR2 deficiency underlies progressive polycystic lung disease. Rosain, J. et al. Human IRF1 governs macrophagic IFN-gamma immunity to mycobacteria. Forde, A. J. et al. Metabolic rewiring tunes dermal macrophages in staphylococcal skin infection. Hartung, T. Pyrogen testing revisited on occasion of the 25th anniversary of the whole blood monocyte activation test. Article PubMed Senju, S. et al. Generation of dendritic cells and macrophages from human induced pluripotent stem cells aiming at cell therapy. Gene Ther. Kambal, A. et al. Generation of HIV-1 resistant and functional macrophages from hematopoietic stem cell-derived induced pluripotent stem cells. Choi, K. D. et al. Hematopoietic and endothelial differentiation of human induced pluripotent stem cells. Bitzer, S. et al. Application of human iPSC-derived macrophages in a miniaturized high-content-imaging-based efferocytosis assay. van Wilgenburg, B., Browne, C., Vowles, J. & Cowley, S. A. Efficient, long term production of monocyte-derived macrophages from human pluripotent stem cells under partly-defined and fully-defined conditions. Gutbier, S. et al. Large-scale production of human iPSC-derived macrophages for drug screening. Toda, S. et al. Characteristics of macrophage aggregates prepared by rotation culture and their response to polymeric materials. Fridley, K. M., Kinney, M. A. & McDevitt, T. C. Hydrodynamic modulation of pluripotent stem cells. Stem Cell Res. Klepikova, A. et al. iPSC-derived macrophages: the differentiation protocol affects cell immune characteristics and differentiation trajectories. Scott, C. L. et al. The transcription factor ZEB2 is required to maintain the tissue-specific identities of macrophages. This work was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy (EXC 2155; RESIST; project no. 390874280 and DFG support LA 3680/9-1 and 10-1) (N.L. ); the European Research Council (ERC) under the European Union (EU)'s Horizon 2020 research and innovation program (grant agreement 852178); and the EU (grant agreements 101100859 and 101158172) (N.L.). Additional funding was provided by the German Center of Lung Research (DZL) and the Federal Ministry of Research, Technology and Space (BMFTR, SMARTibone project). This work was supported by the Fraunhofer Internal Programs under grant no. The work also received funding by the SPARK BIH (01BIHTP2521B) funding scheme within the National Strategy for Gene- and Cell-based Therapies and by the program ‘zukunft.niedersachsen' of Lower Saxony, Germany for the project ‘MacroAB-Delivery'. The views and opinions expressed are, however, those of the authors only and do not necessarily reflect those of the EU or the ERC. Neither the EU nor the granting authority can be held responsible for them. The project was additionally supported by zukunft.niedersachsen (Federal State of Lower Saxony), R2N.Micro-Replace-Systems. The EBiSC Bank acknowledges Bioneer A/S as the source of the human induced pluripotent cell line BIONi010-C, which was generated with support from the EBiSC project. The EBiSC has received support from the Innovative Medicines Initiative (IMI) Joint Undertaking (JU) under grant agreement no. 115582 and from the IMI-2 JU under grant agreement no. 821362, resources of which are composed of financial contributions from the European Union's Seventh Framework Programme (FP7/2007–2013), the European Union's Horizon 2020 research and innovation programme, and EFPIA. Department for Pediatric Pneumology, Allergology and Neonatology, Hannover Medical School, Hannover, Germany Fawaz Saleh, Edwin Emilio Valdivia Malqui, Malene Kappelhøj, Eirini Nikolouli, Ariane Hai Ha Nguyen, Mi-Sun Jang, Débora Basílio-Queirós & Nico Lachmann Fraunhofer Institute for Toxicology and Experimental Medicine ITEM, Hannover, Germany Ingrid Gensch, Maximilian Schinke & Nico Lachmann Biomedical Research in Endstage and Obstructive Lung Disease Hannover (BREATH), German Center for Lung Research (DZL), Hannover, Germany Cluster of Excellence RESIST (EXC 2155), Hannover Medical School, Hannover, Germany Research Center for Translational Regenerative Medicine, Hannover Medical School, Hannover, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Nico Lachmann. is an author of the patent application (European patent application number PCT/EP2018/061574) entitled ‘Stem-cell derived myeloid cells, generation and use thereof'. The priority date of the application is 4 May 2017. is an author on the patent application (European patent application number PCT/EP2021/083371) entitled ‘Application of stem cell derived monocytes in a monocyte activation test for the assessment of pyrogenicity and inflammatory potential'. The priority date of the application is 29 November 2021. receives research funding from Novo Nordisk and holds a consultancy agreement with Evotec (scope outside the manuscript). All other authors declare no competing interests. Nature Protocols thanks Megumu Saito and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Ackermann, M. et al. Stem Cell Res. Abdin, S. M. et al. J. Immunother. Cancer 11, e007705 (2023): https://doi.org/10.1136/jitc-2023-007705 Ackermann, M. et al. Nat. Ackermann, M. et al. Nat. This Protocol is an extension to Nat. a) Representative graphs of bioreactor process parameters (CO2, pH, and temperature) during Mesoderm priming. b) Representative graphs of bioreactor process parameters (CO2, pH, and Temperature) during macrophage production. c) IL-6 secretion in naïve macrophages (iPSC lines: 1, 2, and 3). d) Phagocytosis of macrophages across different harvests 1b, 2, 3, 4, 5, 6, 7 (89.7 ± 7.3%, 84.8 ± 6.68%, 80.8 ± 6.03%, 81.2 ± 15.6%, 92.4 ± 7.97%, 95.2 ± 5.09%, SD +/- mean) (iPSC lines 1, 2, 3, 4, and 5, mean ± SD, n=10). Representative of image of iPSC-derived macrophages produced in the benchtop bioreactor for iPSC 1, 2, 3, 4, and 5 at harvests 1a/1b, 3, 4 and 6. Top: brightfield, (magnification 10x, scale bare 100 µm) bottom: cytospin (magnification 20x) stained with May-Grünwald-Giemsa. Cytospin images were taken using Keyence BZ-X800 (Keyence, Japan) with 20x plan Achromat objective. a) UMAP representation of dataset13 from three independent iMac harvests generated from CERO benchtop bioreactor, the cells are grouped by cell line. (iPSC lines 1, 2, and 3 were used). b) UMAP representation split by cell line and grouped by cluster identity. c) UMAP representation with normalized expression of hematopoietic/myeloid lineage marker genes (PTPRC, ITGAM, CD33, SPI1). d) Dot plot displaying the normalized expression of myeloid progenitor, macrophage, mast cell (MC), granulocyte (Gran), lymphoid lineage and fibroblast marker genes grouped by cell line (iPSC lines: 1-3). Panels a and b adapted from ref. 13, CC BY 4.0. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Saleh, F., Valdivia Malqui, E.E., Gensch, I. et al. Harnessing intermediate-scale bioreactors for next-generation macrophage production and application. Nat Protoc  (2026). Accepted: 21 November 2025 Version of record: 18 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1750-2799 (online) ISSN 1754-2189 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-026-10135-0'>Nucleotide signals coordinate activation and inhibition of bacterial immunity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 17:02:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The cellular nucleotide pool is a major focal point of the host immune response to viral infection. Immune effector proteins that disrupt the nucleotide pool enable animal and bacterial cells to broadly restrict diverse viruses, but reduced nucleotide availability induces cellular toxicity and can limit host fitness1,2,3,4,5. Here we identify Clover, a bacterial anti-phage defence system that overcomes this trade-off by encoding a deoxynucleoside triphosphohydrolase enzyme (CloA) that dynamically responds to both an activating phage cue and an inhibitory nucleotide immune signal produced by a partnering regulatory enzyme (CloB). Analysis of phage restriction by Clover in cells and reconstitution of enzymatic function in vitro demonstrate that CloA is a dGTPase that responds to viral enzymes that increase cellular levels of dTTP. To restrain CloA activation in the absence of infection, we show that CloB synthesizes a dTTP-related inhibitory nucleotide signal, p3diT (5′-triphosphothymidyl-3′5′-thymidine), that binds to CloA and suppresses activation. Cryo-electron microscopy structures of CloA in activated and suppressed states reveal how dTTP and p3diT control distinct allosteric sites and regulate effector function. Our results define how nucleotide signals coordinate both activation and inhibition of antiviral immunity and explain how cells balance defence and immune-mediated toxicity. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout Coordinates and density maps of the Salmonella CloA apo, CloA co-expressed with CloB, CloA(HEAA)–dGTP–p3diT and CloA(HEAA)–dGTP–dTTP complex have been deposited in the Protein Data Bank (PDB) and under accession codes 9P8S, 9P8T, 9P8U and 9P8V, and in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-71386, EMD-71388, EMD-71389 and EMD-71390. Coordinates and structure factors of the S. putrefaciens dGTPase have been deposited in the PDB under the accession code 9P8W. Source data are provided with this paper. Goldstone, D. C. et al. HIV-1 restriction factor SAMHD1 is a deoxynucleoside triphosphate triphosphohydrolase. Tal, N. et al. Bacteria deplete deoxynucleotides to defend against bacteriophage infection. Hsueh, B. Y. et al. Phage defence by deaminase-mediated depletion of deoxynucleotides in bacteria. Itsko, M. & Schaaper, R. M. dGTP starvation in Escherichia coli provides new insights into the thymineless-death phenomenon. Ahmad, S. I., Kirk, S. H. & Eisenstark, A. Thymine metabolism and thymineless death in prokaryotes and eukaryotes. Systematic and quantitative view of the antiviral arsenal of prokaryotes. Nicastro, G. G., Burroughs, A. M., Iyer, L. M. & Aravind, L. Functionally comparable but evolutionarily distinct nucleotide-targeting effectors help identify conserved paradigms across diverse immune systems. Nucleic Acids Res. Aravind, L. & Koonin, E. V. The HD domain defines a new superfamily of metal-dependent phosphohydrolases. Ji, X., Tang, C., Zhao, Q., Wang, W. & Xiong, Y. Structural basis of cellular dNTP regulation by SAMHD1. Characterization of the deoxynucleotide triphosphate triphosphohydrolase (dNTPase) activity of the EF1143 protein from Enterococcus faecalis and crystal structure of the activator-substrate complex. Whiteley, A. T. et al. Bacterial cGAS-like enzymes synthesize diverse nucleotide signals. Cohen, D. et al. Cyclic GMP-AMP signalling protects bacteria against viral infection. Tan, J. M. J. et al. A DNA-gated molecular guard controls bacterial Hailong anti-phage defence. Systematic exploration of Escherichia coli phage–host interactions with the BASEL phage collection. Lopatina, A., Tal, N. & Sorek, R. Abortive infection: bacterial suicide as an antiviral immune strategy. The crystal structure of dGTPase reveals the molecular basis of dGTP selectivity. & Warner, H. R. Properties of deoxynucleoside 5′-monophosphatase induced by bacteriophage T5 after infection of Escherichia coli. Eriksson, S. & Berglund, O. Bacteriophage-induced ribonucleotide reductase systems. Warner, H. R., Drong, R. F. & Berget, S. M. Early events after infection of Escherichia coli by bacteriophage T5. Induction of a 5′-nucleotidase activity and excretion of free bases. Kang, G., Taguchi, A. T., Stubbe, J. & Drennan, C. L. Structure of a trapped radical transfer pathway within a ribonucleotide reductase holocomplex. Mozer, T. J., Thompson, R. B., Berget, S. M. & Warner, H. R. Isolation and characterization of a bacteriophage T5 mutant deficient in deoxynucleoside 5′-monophosphatase activity. & Schaaper, R. M. Novel mutator mutants of E. coli nrdAB ribonucleotide reductase: insight into allosteric regulation and control of mutation rates. Direct activation of a bacterial innate immune system by a viral capsid protein. Gao, L. A. et al. Prokaryotic innate immunity through pattern recognition of conserved viral proteins. Multiple phage resistance systems inhibit infection via SIR2-dependent NAD+ depletion. Burman, N. et al. A virally encoded tRNA neutralizes the PARIS antiviral defence system. Deep, A., Liang, Q., Enustun, E., Pogliano, J. & Corbett, K. D. Architecture and activation mechanism of the bacterial PARIS defence system. Kibby, E. M. et al. A bacterial NLR-related protein recognizes multiple unrelated phage triggers to sense infection. Preprint at bioRxiv https://doi.org/10.1101/2024.12.17.629029 (2024). Loeff, L., Walter, A., Rosalen, G. T. & Jinek, M. DNA end sensing and cleavage by the Shedu anti-phage defense system. Roisné-Hamelin, F., Liu, H. W., Taschner, M., Li, Y. & Gruber, S. Structural basis for plasmid restriction by SMC JET nuclease. Jaskólska, M., Adams, D. W. & Blokesch, M. Two defence systems eliminate plasmids from seventh pandemic Vibrio cholerae. Robins, W. P., Meader, B. T., Toska, J. & Mekalanos, J. J. DdmABC-dependent death triggered by viral palindromic DNA sequences. Pradhan, B. et al. Loop-extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex. Banh, D. V. et al. Bacterial cGAS senses a viral RNA to initiate immunity. Structural basis for Lamassu-based antiviral immunity and its evolution from DNA repair machinery. & Kranzusch, P. J. Nucleotide immune signaling in CBASS, Pycsar, Thoeris, and CRISPR antiphage defense. Athukoralage, J. S. & White, M. F. Cyclic nucleotide signaling in phage defense and counter-defense. Maruta, N. et al. TIR domain-associated nucleotides with functions in plant immunity and beyond. Slavik, K. M. & Kranzusch, P. J. CBASS to cGAS–STING: the origins and mechanisms of nucleotide second messenger immune signaling. The Panoptes system uses decoy cyclic nucleotides to defend against phage. Doherty, E. E. et al. A miniature CRISPR–Cas10 enzyme confers immunity by inhibitory signalling. LeRoux, M. & Laub, M. T. Toxin–antitoxin systems as phage defense elements. Kelly, A., Arrowsmith, T. J., Went, S. C. & Blower, T. R. Toxin–antitoxin systems as mediators of phage defence and the implications for abortive infection. Structure of the human cGAS–DNA complex reveals enhanced control of immune surveillance. Hobbs, S. J. et al. Phage anti-CBASS and anti-Pycsar nucleases subvert bacterial immunity. Collaboration gets the most out of software. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Evans, P. R. & Murshudov, G. N. How good are my data and what is the resolution? McCoy, A. J. et al. Phaser crystallographic software. Accurate structure prediction of biomolecular interactions with AlphaFold 3. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Li, H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Purhonen, J., Banerjee, R., McDonald, A. E., Fellman, V. & Kallijärvi, J. A sensitive assay for dNTPs based on long synthetic oligonucleotides, EvaGreen dye and inhibitor-resistant high-fidelity DNA polymerase. Nucleic Acids Res. Adams, K. J. et al. Skyline for small molecules: a unifying software package for quantitative metabolomics. The authors are grateful to members of the Kranzusch laboratory for helpful comments and discussion. The work was funded by grants to P.J.K. from the Pew Biomedical Scholars programme, the Burroughs Wellcome Fund PATH programme, The G. Harold and Leila Y. Mathers Charitable Foundation, The Mark Foundation for Cancer Research, the Cancer Research Institute, the Parker Institute for Cancer Immunotherapy, the Massachusetts Consortium on Pathogen Readiness (MassCPR), and the National Institutes of Health (1DP2GM146250-01), S.Y. is supported by a JSPS Overseas Research Fellowships (202360072) and a Human Frontiers Science Program Long-Term Fellowship (LT0051). is supported through a Helen Hay Whitney Foundation postdoctoral fellowship. is supported through a Cancer Research Institute Irvington Postdoctoral Fellowship (CRI14458). X-ray data were collected through support by an agreement between the Advanced Photon Source, a US Department of Energy (DOE) Office of Science user facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357, and the Diamond Light Source, the national synchrotron science facility of the UK, located at the Harwell Science and Innovation Campus in Oxfordshire; at the Advanced Photon Source through the Northeastern Collaborative Access Team beamlines, which are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P30 GM124165) and a NIH-ORIP HEI grant (S10OD021527); and at The Center for Bio-Molecular Structure (CBMS) that is primarily supported by the NIH-NIGMS through a Center Core P30 Grant (P30GM133893), and by the DOE Office of Biological and Environmental Research (KP1607011). NSLS2 is a US DOE Office of Science User Facility operated under Contract No. Cryo-EM data were collected at the Harvard Cryo-EM Center for Structural Biology at Harvard Medical School. ITC data were collected at the Center for Macromolecular Interactions at Harvard Medical School CMI (RRID: SCR_018270). Department of Microbiology, Harvard Medical School, Boston, MA, USA Sonomi Yamaguchi, Samantha G. Fernandez, Douglas R. Wassarman & Philip J. Kranzusch Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA, USA Sonomi Yamaguchi, Samantha G. Fernandez, Douglas R. Wassarman & Philip J. Kranzusch Biolog Life Science Institute, Bremen, Germany Marlen Lüders & Frank Schwede Parker Institute for Cancer Immunotherapy at Dana-Farber Cancer Institute, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The study was designed and conceived by S.Y. Phage defence, biochemical experiments, crystallography and cryo-EM structural biology experiments and modelling were performed by S.Y. Phage escape mutant analysis was performed by S.Y. Nucleotide product LC–MS analysis was performed by S.Y. with assistance from D.R.W. Synthetic nucleotide product synthesis and characterization experiments were performed by M.L. The manuscript was written by S.Y. All authors contributed to editing the manuscript and support the conclusions. Correspondence to Philip J. Kranzusch. are employed at Biolog Life Science Institute GmbH & Co. KG, which sells p3diT and related compounds as research tools. The other authors declare no competing interests. Nature thanks Luciano A. Marraffini and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Phylogenetic analysis of ~650 CloB homologs identified in the IMG database. The inner segments are colored according to identity of the operon encoded adjacent effector protein and the surrounding rings depict the genera of bacteria encoding CloB. b, Heatmap illustrating fold defense of E. coli expressing Clover system from Salmonella enterica and Escherichia coli H5. Reduction in plaque forming units (PFU) of wildtype CloA or CloA H116A/D117A mutants compared to bacteria expressing a GFP control vector (n = 2 biologically independent experiments). c, Representative plaque assays of Clover, CloA catalytic mutant, or CloB catalytic mutant operons from E. coli cells expressing Clover from Salmonella enterica SA20044414 or Escherichia coli H5 (n = 3 biologically independent experiments). d, Growth curves of Clover-expressing cells (green) and control (gray) cultures with and without infection by phage T5 at an MOI of 5 or 0.5. Curves show the mean ± s.d. (n = 3 biologically independent experiments; technical triplicates). e, Cartoon representation of Salmonella CloA octameric assembly formation (top). Overall structure of the octameric CloA assembly (bottom, left) and an example CloA tetrameric unit showing the interface between two dimeric units (bottom, right). f, g, and h, Overview and detailed views of interacting residues of the CloAA-CloAB, CloAA-CloAc, CloAA-CloAD interfaces that mediate octameric assembly. i, Cartoon representation of Shewanella putrefaciens CN-32 hexameric assembly (top). Overall structure of the hexameric SpdGTPase assembly (bottom, left) and an example SpdGTPase tetrameric unit showing the interface between two dimeric units (bottom, right). j, Cryo-EM structure of a canonical dGTPase E. coli Dgt (PDB id: 6OIY) and view of Dgt dimer. k, Cartoon representation of Salmonella CloA (left) and SpdGTPase monomers (middle) with the helices forming the catalytic active site depicted in yellow. A superposition of the Salmonella CloA and SpdGTPase monomers with the C-terminal helices highlighted as opaque (right). l, Overview of canonical dGTPase E. coli Dgt active site including the metal binding HD motif and conserved H126/E129 residues. a, SeCloA cryo-EM particle picking and classification strategy. b, Example motion-corrected micrograph, subjected to particle picking and further analysis. c, Example 2D class averages from the particle curation stage. d, Gold-standard Fourier shell correlation (GSFSC) curves after FSC-mask auto-tightening, as produced by CryoSPARC. e, Local resolution of the final SeCloA map. a, AlphaFold3 modelled structures of phage Bas28 dNMP monophosphatase (top) and Bas28 dNMP monophosphatase escape mutant with an S201 nonsense mutation (bottom). b, Cryo-EM structure and magnified view of the active site of the E. coli ribonucleotide diphosphate reductase subunit A (NrdA) dimer (PDB id: 6W4X) compared to an AlphaFold3 modelled structure and magnified view of the active site of phage T5 NrdA (bottom). c, Bacterial growth assay of a 10-fold dilution series of E. coli transformed with two separate plasmids expressing E. coli Clover system or GFP, and enzymes reported to be involved in regulating dTTP levels. Note that T4 nrdAB exhibited toxicity when it expressed at high-levels with GFP (n = 3 biologically independent experiments). d, e, and f, LC-MS analysis of purified Escherichia coli H5 or Salmonella enterica CloA incubated with dNTPs demonstrates that CloA is a dGTPase activated by the presence of dTTP (n = 2 independent experiments). g, LC-MS analysis of purified Escherichia coli H5 or Salmonella enterica CloA incubated with dNTPs demonstrates that EcCloA exhibits more potent dGTPase activity compared to SeCloA (n = 2 independent experiments). a, SeCloA–p3diT complex particle picking and classification strategy. b, Gold-standard Fourier shell correlation (GSFSC) curves after FSC-mask auto-tightening, as produced by CryoSPARC. c, Local resolution of the final CloA–p3diT map. d, A260 chromatograph of the nucleotide signal released upon heat denaturation of CloA purified from cells co-expressing CloB or CloBDDAA (left). The purified CloB nucleotide signal was further treated with apyrase or a combination of phosphatase (CIP) and nuclease P1. e, f, and g, LC-MS analysis run in negative mode of the purified CloB nucleotide signal alone or treated with apyrase or a combination of CIP and nuclease P1. Formate and chloride ions formed the major adducts [dT+formate]− and [dT+Cl]− respectively observed with deoxythymidine. g, Left, comparison between chemically synthesized p3diT, p2diT, p1diT, and the purified CloB nucleotide immune signal. Right, LC-MS analysis run in negative mode of chemically synthesized p3diT, p2diT, and p1diT mixed with purified CloB nucleotide signal. h, MS/MS analysis of the nucleotide signal released from denatured CloA co-expressed with CloB compared with synthetic standards, confirms the CloA bound signal is mixture of p3diT, p2diT, and p1diT. a, Phylogenetic analysis of ~650 CloB homologs identified in the IMG database and schematics of CloB proteins used for the experiment b, AlphaFold3 predicted structures of CloB from Salmonella, Limnohabitans and Xanthomonas. c, Bacterial growth assay of a 10-fold dilution series of E. coli containing arabinose-inducible plasmids expressing Escherichia coli Clover wildtype operons, CloA HD motif mutant, CloB inactive mutant (D71A/D73A), wildtype Escherichia coli H5 CloA with CloB core from Limnohabitans and Xanthomonas (n = 2 biologically independent experiments). d, A260 chromatograph of the nucleotide signal released upon heat denaturation of CloA purified from cells co-expressing Salmonella, Limnohabitans, and Xanthomonas CloB (n = 3 independent experiments). e, LC-MS analysis of the purified LpCloB NTase core incubated with indicated dNTPs and deoxynucleotides demonstrates that CloB synthesizes a nucleotide product in a thymidine-dependent manner (n = 2 independent experiments). a, SeCloA–dGTP–p3diT complex particle picking and classification strategy. b, Gold-standard Fourier shell correlation (GSFSC) curves after FSC-mask auto-tightening, as produced by CryoSPARC. c, Local resolution of the final SeCloA–dGTP–p3diT complex map. d, SeCloA–dGTP–dTTP complex particle picking and classification strategy. e, Gold-standard Fourier shell correlation (GSFSC) curves after FSC-mask auto-tightening, as produced by CryoSPARC. f, Local resolution of the final SeCloA–dGTP–dTTP complex map. a, Top, contact helices of p3diT in the SeCloA dimer revealing that α2A, α4B, and α14A helices form the p3diT binding pocket. Bottom, surface carving and electrostatic potential of the p3diT binding pocket. b, Top, contact helices of dTTP in the SeCloA dimer revealing that α3A, α4 A, and α18A–α21A helices and loop α3B–α4B form the dTTP binding pocket. Bottom 180° rotated view of the top figure showing surface carving and electrostatic potential of the dTTP binding pocket. c, Top cartoon view of CloAA-CloAB dimer in the p3diT bound suppressed state structure. Bottom detailed view of α3A, α4A, and the dGTPase active site in the p3diT bound CloA suppressed state structure. α17AB is omitted for clarity. d, Top cartoon view of CloAA-CloAB dimer in the dTTP bound active state structure. Bottom detailed view of α3A, α4A, and the dGTPase active site in the dTTP bound CloA active state structure revealing that dTTP-binding results in a conformational change that stabilizes loop α3B–α4B and completes the dGTPase active site. α17AB is omitted for clarity. e, Left, superposition of CloA p3diT and dTTP binding states. Middle, 90° rotated view of the left figure of the CloA p3diT binding state. Right, 90° rotated view of the left figure of the CloA dTTP binding state, highlighting additional interactions between loop α3B–α4B and Q127A with substrate dGTP. f, Rotated view of the superposition of the CloA p3diT and dTTP binding states demonstrates that dTTP binding induces a conformational change that positions the substrate dGTP α-phosphate is proximal to the dGTPase active site residues. g, Phosphate release assay measuring dGTP hydrolysis by wildtype SeCloA and SeCloA mutants (Y452A and 3RA) in the presence dTTP and demonstrates that of 200 µM dTTP is sufficient to overcome inhibition induced by 20 µM p3diT. Bars show the mean ± s.d. (n = 2 biologically independent experiments; technical duplicates). h, Effect of p3diT and dTTP binding pocket mutations on Clover anti-phage defense tested at at 30 °C. Data represent plaque-forming units per mL (PFU mL−1) of phage T5. Bar graph depicts the mean, with error bars representing the mean ± s.e.m. (n = 2 biologically independent experiments; technical triplicates). i, Bacterial growth assay of a 10-fold dilution series of E. coli containing arabinose-inducible plasmids expressing Escherichia coli H5 Clover wildtype operons, CloA mutants with wildtype CloB or CloB inactive mutant (D71A/D73A) (n = 2 biologically independent experiments). j, ITC experiments measuring the binding of p3diT (left) or dTTP (right) to Salmonella CloA H126A/E129A mutant protein (n = 2 independent experiments). Sequence information of CloB homologues. List of IMG Gene IDs and amino acid sequences of SeCloB homologues obtained from IMG/MER. Structural basis of mutually exclusive ligand binding. Structural superposition shows loop α8–9 repositioning, with CloA L174 sterically blocking p3diT. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Yamaguchi, S., Fernandez, S.G., Wassarman, D.R. et al. Nucleotide signals coordinate activation and inhibition of bacterial immunity. Version of record: 18 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-026-10174-7'>Practical lithium–organic batteries enabled by an n-type conducting polymer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 16:34:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Organic batteries using abundant and recyclable organic electrode materials provide a sustainable and environmentally friendly alternative to commercial lithium-ion batteries1,2,3,4,5, which rely on resource-limited mineral-derived inorganic electrode materials6,7,8. However, the practical use of organic batteries has been severely hindered by the intrinsic insulation and dissolution of organic electrode materials9,10. Here we report practical organic batteries using an n-type conducting polymer cathode, poly(benzodifurandione) (PBFDO), which exhibits excellent mixed ionic and electronic transport and low solubility. The PBFDO cathode maintains its n-doped state throughout the electrochemical processes and exhibits stable and reversible redox characteristics, high electrical conductivities and significant lithium-ion diffusion coefficients, without the need for additional conductive additives. Consequently, ultrahigh-mass-loading polymer cathodes, with mass loadings up to 206 mg cm−2, are realized, delivering a high areal capacity of 42 mAh cm−2 and demonstrating robust cycling stability. Furthermore, practical 2.5 Ah lithium–organic pouch cells were fabricated, achieving an impressive energy density of 255 Wh kg−1. Notably, the conducting polymer cathode operates efficiently over a wide temperature range from −70 °C to 80 °C and demonstrates excellent flexibility and safety, marking considerable potential for applications in extreme conditions and wearable electronics. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access $199.00 per year only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data supporting the findings of this study are available in the paper and its Supplementary Information. The data of this study are available from the corresponding authors upon reasonable request. Nguyen, T. P. et al. Polypeptide organic radical batteries. Lu, Y. & Chen, J. Prospects of organic electrode materials for practical lithium batteries. Kim, J. et al. Organic batteries for a greener rechargeable world. Dai, H., Guan, L., Mao, M. & Wang, C. J. Evaluating the present and future of organic batteries. Clean Technol. Li, M. et al. Electrolytes in organic batteries. Li, M. & Lu, J. Cobalt in lithium-ion batteries. Deng, T. et al. Designing in-situ-formed interphases enables highly reversible cobalt-free LiNiO2 cathode for Li-ion and Li-metal batteries. Ogihara, N. et al. Direct capacity regeneration for spent Li-ion batteries. Bai, S. et al. Permselective metal–organic framework gel membrane enables long-life cycling of rechargeable organic batteries. Li, M. et al. Soluble organic cathodes enable long cycle life, high rate, and wide-temperature lithium-ion batteries. Chen, Z. et al. A nitroaromatic cathode with an ultrahigh energy density based on six-electron reaction per nitro group for lithium batteries. Natl Acad. Schön, T. B., McAllister, B. T., Li, P.-F. & Seferos, D. S. The rise of organic electrode materials for energy storage. Article PubMed Lee, M. et al. High-performance sodium–organic battery by realizing four-sodium storage in disodium rhodizonate. Luo, C. et al. Azo compounds derived from electrochemical reduction of nitro compounds for high performance Li-ion batteries. Sang, P., Chen, Q., Wang, D.-Y., Guo, W. & Fu, Y. Organosulfur materials for rechargeable batteries: structure, mechanism, and application. Xiong, P. et al. Thiourea-based polyimide/RGO composite cathode: a comprehensive study of storage mechanism with alkali metal ions. China Mater. Guo, J. et al. Revealing hydrogen bond effect in rechargeable aqueous zinc-organic batteries. Cong, G., Wang, W., Lai, N.-C., Liang, Z. & Lu, Y.-C. A high-rate and long-life organic-oxygen battery. Chen, Z. et al. Anion chemistry enabled positive valence conversion to achieve a record high-voltage organic cathode for zinc batteries. Wang, J. et al. Conjugated sulfonamides as a class of organic lithium-ion positive electrodes. Suga, T., Ohshiro, H., Sugita, S., Oyaizu, K. & Nishide, H. Emerging n-type redox-active radical polymer for a totally organic polymer-based rechargeable battery. Li, Z. et al. A small molecular symmetric all-organic lithium-ion battery. Zhao, C. et al. In situ electropolymerization enables ultrafast long cycle life and high-voltage organic cathodes for lithium batteries. Yu, Z. et al. Redox-active donor-acceptor conjugated microporous polymer for high-voltage and high-rate symmetric all-organic lithium-ion battery. Song, Z. et al. Polyanthraquinone as a reliable organic electrode for stable and fast lithium storage. Deng, X. et al. Ultrafast charging of two-dimensional polymer cathodes enabled by cross-flow structure design. Luo, L. et al. A redox-active conjugated microporous polymer cathode for high-performance lithium/potassium-organic batteries. China Chem. Kolek, M. et al. Ultra-high cycling stability of poly(vinylphenothiazine) as a battery cathode material resulting from π–π interactions. Liang, Y. et al. Heavily n-dopable π-conjugated redox polymers with ultrafast energy storage capability. Peng, C. et al. Reversible multi-electron redox chemistry of π-conjugated N-containing heteroaromatic molecule-based organic cathodes. Lu, D. et al. Ligand-channel-enabled ultrafast Li-ion conduction. Tang, H. et al. A solution-processed n-type conducting polymer with ultrahigh conductivity. Jin, Z. et al. Iterative synthesis of contorted macromolecular ladders for fast-charging and long-life lithium batteries. Qin, J. et al. A metal-free battery with pure ionic liquid electrolyte. Ke, Z. et al. Controlled de-doping and redoping of n-doped poly(benzodifurandione) (n-PBDF). Li, Z. et al. Electrolyte design enables rechargeable LiFePO4/graphite batteries from −80 °C to 80 °C. Dong, X., Guo, Z., Guo, Z., Wang, Y. & Xia, Y. Organic batteries operated at −70 °C. Asl, H. Y. Reining in dissolved transition-metal ions. Feng, X., Ren, D., He, X. & Ouyang, M. Mitigating thermal runaway of lithium-ion batteries. Liu, D. et al. Controlled large-area lithium deposition to reduce swelling of high-energy lithium metal pouch cells in liquid electrolytes. Muench, S. et al. Polymer-based organic batteries. Tang, H. et al. Highly conductive alcohol-processable n-type conducting polymer enabled by finely tuned electrostatic interactions for green organic electronics. Neese, F. Software update: the ORCA program system—version 5.0. Wiley Interdiscip. Lu, T. & Chen, F. Multiwfn: a multifunctional wavefunction analyzer. Wang, B. et al. Diffusion coefficients during regenerated cellulose fibers formation using ionic liquids as solvents: experimental investigation and molecular dynamics simulation. Download references This work was financially supported by the Fundamental and Interdisciplinary Disciplines Breakthrough Plan of the Ministry of Education of China (JYB2025XDXM410), the National Natural Science Foundation of China (22579126, 22179092, 52433012 and 52303227), the Fundamental Research Funds for the Central Universities (2024ZYGXZR076) and the China Postdoctoral Science Foundation (2024T170286 and 2023M741201). acknowledges support from the New Cornerstone Science Foundation through the XPLORER PRIZE. These authors contributed equally: Zhenfei Li, Haoran Tang School of Materials Science and Engineering, State Key Laboratory of Advanced Materials for Intelligent Sensing, National Industry-Education Platform for Energy Storage, Tianjin University, Tianjin, China Zhenfei Li, Yuansheng Liu, Mengjie Li, Lanhua Ma, Hongpeng Chen, Yanhou Geng & Yunhua Xu Institute of Polymer Optoelectronic Materials and Devices, Guangdong Basic Research Center of Excellence for Energy & Information Polymer Materials, State Key Laboratory of Luminescent Materials and Devices, South China University of Technology, Guangzhou, China Haoran Tang, Yuanying Liang, Yining Wang, Shaohua Tong, Qinglin Jiang, Yuguang Ma, Yong Cao & Fei Huang Guangdong Artificial Intelligence and Digital Economy Laboratory (Guangzhou), Guangzhou, China Yuanying Liang School of Materials Science and Engineering, Zhejiang University, Hangzhou, China Xiaoyu Zhai & Jiangwei Wang Department of Materials Science and Engineering, Southern University of Science and Technology (SUSTech), Shenzhen, China Eastern Institute for Advanced Study, Eastern Institute of Technology, Ningbo, China Meng Danny Gu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project under the supervision of Y.C., Y.X. synthesized materials, performed characterizations and assembled batteries. discussed and analysed the data. Y. Liu and H.C. conducted the DFT and molecular dynamics calculations. assisted with data curation and manuscript revision. assisted in materials synthesis and characterizations. performed the Hall effect measurements. carried out the HRTEM characterization. designed and executed the cryo-TEM experiments. wrote and revised the paper, and all authors read and approved the paper. Correspondence to Yunhua Xu or Fei Huang. The authors declare no competing interests. Nature thanks Matthieu Becuwe and Nagaraj Patil for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Flexibility test of the self-supporting PBFDO cathode. b, Digital microscopy and SEM images of the flexible PBFDO cathode at different bending states. c-d, Bending test of the flexible PBFDO cathode: photographs of the test process (c) and bending cycle life of 75,000 cycles (d). The cycle life was indicated by the stress variation with test time. e-f, Photographs of the flexible Li ||PBFDO pouch cell at different bending states. g, Cycling stability of the flexible Li ||PBFDO pouch cell under different bending conditions. a, Schematical illustration of the lithium storage processes. Owing to the complexity of conducting polymer resonance and lithium-ion storage mechanisms, coupled with the uncertainty regarding the positions of counter-ions, only a schematic representation of one possible resonance pathway was provided here. The pristine PBFDO, with an n-doping level of around 90%, was first discharged to 1.5 V by lithium-ion uptake. During the subsequent charge process, lithium ions were extracted, while approximately half of protons were also removed, leaving the remaining protons preserved within PBFDO, which also contributed to PBFDO maintaining its n-doped state. Afterward, PBFDO experienced reversible electrochemical reactions, with some protonated carbonyl groups persisting, leading to a reversible capacity of 230.4 mAh g−1. b, XPS spectra of the PBFDO cathode at different charge/discharge states. c, Voltage profiles of the PBFDO cathode at 50 mA g−1 with marked voltages for the FTIR tests. d, FTIR spectra of the PBFDO cathode at different charge/discharge states as indicated in c. e, In situ Raman spectra of the PBFDO cathodes during charge/discharge processes. This file contains Supplementary Sections A–X, including Supplementary Figs. 1–53 and Supplementary Tables 1–9. Live demonstration of the bending endurance test of the PBFDO cathode. The PBFDO cathode can withstand 75,000 stretch cycles. This video shows the remarkable mechanical stability of the PBFDO cathode, highlighting its potential for applications requiring high durability under repeated mechanical deformation. Live demonstration showcasing the flexibility of the PBFDO cathode in comparison to a commercial inorganic cathode. The flexibility test of commercial inorganic cathode is shown from 36″ to 4′13″, and PBFDO cathode is shown from 4′24″ to 7′16″. The video highlights the superior mechanical flexibility of the PBFDO cathode, highlighting its durability and potential for flexible batteries. Live footage of the nail penetration test conducted on a 2.5 Ah PBFDO pouch cell. The test demonstrates the safety and robustness of the PBFDO pouch cell. No deformation, gas production or explosion was observed during or after the penetration, highlighting the exceptional stability of the PBFDO pouch cell under extreme mechanical stress. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Li, Z., Tang, H., Liang, Y. et al. Practical lithium–organic batteries enabled by an n-type conducting polymer. Download citation Received: 20 November 2024 Accepted: 22 January 2026 Version of record: 18 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1476-4687 (online) ISSN 0028-0836 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09967-z'>Ancestry and somatic profile indicate acral melanoma origin and prognosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 16:12:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Acral melanoma, which is not ultraviolet-associated, is the type of melanoma reported most commonly in several non-European-descent populations1,2,3, including in Mexican people4. Latin American samples are substantially under-represented in global cancer genomics studies5, which directly affects patients in these regions as it is known that cancer risk and incidence may be influenced by ancestry and environmental exposures6,7,8. To address this, we characterized the genome and transcriptome of 123 acral melanoma tumours from 92 Mexican patients—a population notable because of its genetic admixture9. Compared with other studies of melanoma, we found fewer mutations in classical driver genes such as BRAF, NRAS or NF1. Although most patients had predominantly Amerindian genetic ancestry, those with higher European ancestry had increased frequency of BRAF mutations. The tumours with activating BRAF mutations had a transcriptional profile more similar to cutaneous non-volar melanocytes, indicating that acral melanomas in these patients may arise from a distinct cell of origin compared with other tumours arising in these locations. Transcriptional profiling defined three expression clusters; these characteristics were associated with recurrence-free and overall survival. Our study enhances knowledge of this understudied disease and underscores the importance of including samples from diverse ancestries in cancer genomics studies. Melanoma is classified into several clinicopathological subtypes on the basis of tumour site of presentation and histopathological features. Acral melanoma is an understudied melanoma subtype due to its low incidence globally, and because it represents a small proportion of melanoma cases in European-descent populations2,10; however, acral melanoma represents the vast majority of melanoma cases in some Latin American, African and Asian countries due to the lower incidences of ultraviolet-induced melanoma subtypes11. Furthermore, the causes of this type of disease are unknown, with patients managed in a similar way to ultraviolet-associated cutaneous melanoma. However, its site of presentation and genomic characteristics are vastly different12. Acral melanoma arises on the glabrous (non-haired) skin of soles, palms and in the nail unit (subungual location), and its genome differs substantially from other cutaneous melanoma subtypes13. In contrast to ultraviolet-linked subtypes, acral melanoma has a lower burden of single nucleotide variants (SNVs), a higher burden of structural variants and a low prevalence of mutational signatures SBS7a/b/c/d, which are associated with ultraviolet irradiation14,15,16,17,18. Genes that are mutated frequently in cutaneous melanoma such as BRAF, the RAS genes and NF1, are reported to be altered at a significantly lower frequency in acral melanoma. This, coupled with the comparatively lower number of studies of acral melanoma when compared with other cutaneous melanoma subtypes, has translated into limited available therapies for acral melanoma management. It is known that cancer risk and incidence, as well as tumour genomic profiles, vary with ancestry and geographical location6,19,20. As most genomic studies on acral melanoma have been performed on patients of European or Asian ancestry, we considered it necessary to examine the genomics of this subtype of melanoma in Latin American people. Specifically, Latin American populations have been substantially under-represented in cancer genomic studies, with only about 1% of all samples in cohorts such as the Pan-Cancer Analysis of Whole Genomes, The Cancer Genome Atlas (TCGA) and other repositories, and those contributing to cancer genome-wide association studies, being of Latin American origin5,21,22. Identification of differences in the genomic profile among populations can potentially aid the discovery of germline/inherited or environmental factors related to acral melanoma aetiology, as well as identify optimal therapeutic strategies for all patients. In this study, we analysed 123 acral melanoma samples from 92 Mexican patients through genotyping, exome sequencing, SNV and insertion/deletion (indel) variant calling, copy number estimation and gene expression profiling, and examined the correlation of these molecular characteristics with clinical variables. We reveal a significant correlation between genetic ancestry and BRAF somatic mutations, as well as a distinct transcriptomic profile in tumours with BRAF-activating mutations compared with samples without activating mutations in BRAF. We also identify significant differences in recurrence-free and overall survival among patients with tumours with distinct gene expression profiles. A total of 123 uniformly ascertained samples from 92 patients from a large Mexican tertiary referral hospital were analysed in this study (Supplementary Table 1; Methods). Latin American genomes are generally a mixture of European, African and Amerindian ancestry. Of note, 90% of genotyped samples (n = 80) in this study had predominantly Amerindian ancestry (median 81%) (Supplementary Fig. Most patients were stage III (American Joint Committee on Cancer, 8th edn)23 at diagnosis, and the most common primary site was the foot—most frequently the sole. The median Breslow thickness was 4.0 mm and most primary tumours were ulcerated (68%) (Table 1). It should be noted that only four patients received immune checkpoint inhibitors or targeted therapy, due to lack of access. Considering all 123 samples, acral melanoma tumours showed a SNV+indel (hereafter referred to as tumour mutational burden (TMB)) mean of 0.95 mutations per megabase and a median of 0.87 mutations per megabase (range, 0–3.49 mutations per megabase). When including only one sample per patient, with primaries being selected preferentially, the most frequently mutated genes were NRAS (14% of samples; Q-value < 4.97 × 10−10), KIT (14% of samples not counting deletions, as they are unlikely to be activating; Q-value = 4.97 × 10−10), BRAF (13%, Q-value = 3.86 × 10−7) and NF1 (9%, Q-value = 0.0001) (Fig. Two of these samples had homozygous NF1 deletions, in addition to a further two secondary samples from other patients (Extended Data Fig. These genes, which represent known drivers, were identified as being under positive selection (Methods) and exhibit mutual exclusivity (only one patient has tumours with mutations in more than one of these genes), which reflects their functional redundancy in activating the MAPK pathway. Separate capillary sequencing of the TERT promoter in 76 samples belonging to 64 patients identified that six carry the −124 promoter mutation (9.3%) and two out of 59 patients for which the −146 position was amplified successfully carry a mutation in this position (3.4%) (Supplementary Table 4). In total, we estimate that about 10.5% of patients have an activating TERT promoter mutation, which is similar to estimates in other studies15,24. All samples from all patients that had several samples sequenced and that could be assessed had a concordant TERT genotype, in agreement with an early emergence of this mutation during tumour evolution24. In summary, the ‘classic' melanoma driver genes (N/H/KRAS, BRAF and NF1) are mutated in 40% of Mexican acral melanoma samples, with most of the samples in this cohort therefore being classified as ‘triple wild-type' melanomas. Apart from the known HRAS, SPRED1, TP53 and KRAS driver genes, we also find mutations in PTPRJ, ATM, NF2 and RDH5 (Extended Data Fig. Specifically, in those tumours without mutations in any of the abovementioned four driver genes (BRAF, NRAS, KIT, NF1, ‘quadruple wild type' (QWT)), we find two tumours each from different patients with deleterious mutations in ATM and RDH5 (Extended Data Fig. The mutations in these genes are all protein-changing and deleterious. All these genes have been linked previously to tumour suppressor activities in either acral or mucosal melanomas25,26,27,28,29,30, as well as other cancer types, and may represent low-frequency drivers. We also observe a significantly higher proportion of women versus men carrying mutations in driver genes (two-tailed Fisher's test P value = 0.003) (Supplementary Table 5). After adjusting for date of diagnosis, age at diagnosis, ancestry and tumour stage, the odds ratio of having a mutation in a driver gene in female patients (compared with men) was estimated to be 3.83 (95% confidence interval, 1.32, 11.03) (multivariate logistic regression, P value = 0.013). a, Oncoplot depicting the seven most mutated genes according to dNdScv and their status in the samples with mutations in these (52 samples out of 92, one per patient). Mutational classification, sample type, tumour stage, sex, age at diagnosis, ulceration status, tumour site and mutational spectra are shown by sample. In the mutational spectra plot, asterisks indicate that these mutations occurred in the same sample. b, Mutations found in KIT, NRAS, BRAF and NF1, which are the most significantly mutated genes. c, A logistic regression model controlling for age, sex and total TMB was fitted to predict the presence or absence of a mutation on the acral melanoma samples using the inferred ADMIXTURE cluster related to the European ancestry component. d, Barplot depicting the number and mutational classification of samples in different acral and cutaneous melanoma studies14,15,18,31,32,33,34. When examining the relationship between ancestry and somatic profile, we identified significantly higher odds (P value = 0.02) of carrying a BRAF somatic mutation with increasing European ancestry in a linear model controlling for age at diagnosis, sex and total TMB (Fig. Patients with mutations in KIT showed a tendency for higher Amerindian ancestry (Supplementary Fig. We also found that patients with NRAS mutations were younger at diagnosis (median and mean age of diagnosis for patients with NRAS mutations was 49 years and 50.84 years versus 63 years and 62.9 years without, respectively), but this effect is probably mediated by ulceration status, as patients with NRAS mutations have a significantly lower rate of ulceration (two-tailed Fisher's exact test P value = 0.016). Out of 22 patients for whom we sequenced at least two different samples (for example, a primary and a metastasis), 13 were classified as NRAS/KIT/BRAF/NF1-mutated. For BRAF, all four patients have mutations across all samples and, for NRAS, three out of four patients have a NRAS mutation in both the primary and lymph node metastasis. These data agree with those of Wang et al.24, and indicate that metastases are seeded before the appearance of these mutations (Supplementary Table 6). Collectively, these results are similar to those reported in other acral melanoma studies14,15,18,31,32,33,34, with some important differences: first, the genetic composition of the patients in our study includes a high proportion of Native American ancestry, which is severely under-represented in already published cancer genomics studies and permits the identification of relationships of specific ancestries with somatic characteristics. In addition, the fraction of activating BRAF mutations is lower than in the studies with predominantly European-descent patients, and more similar to those with Asian patients, probably due to the positive relationship between BRAF mutation and European ancestry (Fig. Somatic copy number alteration (SCNA) analysis across all samples showed a higher burden of amplifications than deletions (Fig. Examination of 47 samples, one per patient, that passed our stringent quality filtering for this type of analysis (Methods), showed that 18 regions were significantly amplified, and six regions were frequently deleted (Supplementary Table 7). About a quarter (11, 23%) of these 47 samples had whole genome duplication events (Supplementary Table 1). Potential driver genes in frequently amplified regions include TERT (43% of samples), CRKL (36%), GAB2 (30%) and CCND1 (28%) (Supplementary Tables 4 and 7–9). Regions that showed recurrent deletions contained genes such as CDKN2A, CDKN2B, ATM and TP53. No association was found between ancestry and any of the significantly altered CNA regions. SCNA profiles varied depending on whether samples had mutations in driver genes or were QWT. Specifically, samples with mutations in driver genes (n = 23) showed preferential amplification of NOTCH2 (P value = 0.036, two-tailed exact Fisher test) and 1q21.3, containing several genes (P value = 0.02), whereas CCND1 (P value = 0.049), and ARF6 and SOS2 (both in same amplification peak, P value = 0.048) were preferentially amplified in QWT tumours (n = 24). The 8p12 region, containing genes such as FGFR1 and TACC1, was found amplified only in five QWT samples, whereas several regions were found altered only in mutated tumours (Supplementary Tables 10–15 and Extended Data Figs. When stratifying samples by mutational status (considering BRAF-, NRAS-, NF1-, KIT-mutated and multi-hit, which included one sample with mutations in more than one of these drivers), we did not observe any significant differences in SCNA among groups (measured by global copy number alteration score (GCS); Methods) (Fig. Considering all samples, those with NRAS, BRAF and NF1 mutations had the lowest median total TMB, whereas KIT-mutated and multi-hit tumours had the highest median total TMB (Supplementary Fig. We found a significant correlation between GCS score and total TMB (Pearson's product-moment correlation coefficient = 0.72; P value < 0.0001) (Fig. Tumours from the subungual region also had a higher median GCS score than those found on the feet (Fig. a, Regions of amplification (red) and deletion (blue) in 47 acral melanoma samples, one per patient, as identified by GISTIC2. Known drivers, or the chromosomal regions, are shown. This analysis shows alterations with respect to the normal sample, that is, with respect to a ploidy of two. b, Heatmap showing regions of amplification (red) and deletion (blue) by sample and chromosomal arm in all 60 samples classified into genomic subgroups. This analysis shows alterations with respect to the estimated tumour sample ploidy. d, Scatter plot of total TMB (referring to total number of mutations, x axis) and GCS (y axis) for 47 samples, one per patient. Dots represent samples, coloured by genomic subtype. GCS scores are calculated with respect to tumour sample ploidy. Statistical significance was assessed using two-sided Wilcoxon–Mann–Whitney tests. Single-base substitution mutational signature analysis across 116 samples that carried at least one SNV revealed previously reported COSMICv.3.4 signatures SBS1, SBS5 and SBS40a (Extended Data Fig. SBS40a is of unknown origin but was identified originally in kidney cancer and is present in many cancer types36. Copy number signature analysis identified a number of previously reported signatures across different samples (n = 60 samples; Methods)37,38. CN1, which has been associated with a diploid state and CN9, which is potentially caused by local loss of heterozygosity on a diploid background, dominated the copy number landscape (Extended Data Fig. As expected, signatures related to chromothripsis (CN7, CN8) were also found in several samples across the cohort. The number of indels in the samples was too low to add meaningful information (average, 2.52 indels per sample), so signature analysis for indels was not performed. It has been postulated previously that BRAF-mutated acral melanomas might be more biologically similar to melanomas from non-acral sites than to other acral melanomas18,39; because of the observed correlation of European ancestry with BRAF mutation rate, we decided to investigate this hypothesis. We successfully extracted and sequenced RNA from 77 primary tumours from different patients (Supplementary Table 1; Methods). We then generated a gene signature-based score for identifying acral- versus cutaneous-derived melanomas. For this, we sourced a list of candidate genes from acral melanoma and cutaneous melanoma datasets (Supplementary Table 16; Methods) and identified 20 genes with high classification accuracy in a training cohort of ten primary acral melanomas (used to derive a v-mel score, or ‘A' for acral) and ten primary cutaneous melanomas (used to derive a c-mel score, or ‘C' for cutaneous) recruited at the University of Utah (Fig. We then obtained scores (v-mel/c-mel, or A:C) for samples in our dataset of acral melanomas, separating primary tumours with BRAF-activating mutations (n = 10) versus BRAF-wild-type (n = 67) tumours. We observed a difference between BRAF-activating and BRAF-wild-type tumours (P value = 0.045), with BRAF-activating tumours having a score closer to cutaneous melanomas (Fig. We then replicated this analysis in an independent cohort of 63 acral melanomas from Newell et al.15 (BRAF-activating n = 10, wild type n = 53), which further confirmed these results (P value = 0.039) (Fig. Therefore, we explored the possibility that this difference could be due to downstream mutated BRAF signalling. First, we replicated this analysis in the TCGA cutaneous melanoma data, finding no significant differences among BRAF and non-BRAF-mutated samples (Extended Data Fig. We then examined datasets in which mutant BRAF was introduced into primary melanocytes in a doxycycline-inducible manner40. We found that the c-mel signature genes were not activated downstream of mutant BRAF, further indicating that the classifier does not simply reflect BRAF-driven transcriptional changes (Fig. Using a recently developed method for assessing gene signature similarity41, we compared the c-mel gene signature from our classifier to a previously published set of genes directly activated by mutant BRAF in melanoma cells42. We found no significant correlation between these signatures (Extended Data Fig. a, Elucidation of genes used to classify acral melanoma (AM) versus cutaneous melanoma (CM) samples. Right, loadings on PC2 were used to identify the top differentially expressed genes contributing to the variance between acral melanomas and cutaneous melanomas. b, Scatter plot showing the distribution of the acral:cutaneous (A:C) gene expression ratios between test acral and cutaneous melanoma samples. Acral melanoma samples (n = 10) are represented by blue dots and cutaneous melanoma samples (n = 10) are represented by purple dots (P = 0.00018, two-sided Wilcoxon–Mann–Whitney test). c, Left, comparison of A:C gene expression ratio in acral melanoma samples with different mutation status. Box and whiskers plot comparing two groups: BRAF-wild-type (BRAF-WT; n = 67) and BRAF-activating mutated tumours (n = 10). Right, comparison of A:C gene expression ratio in acral melanoma samples with BRAF-activating mutations (n = 10) and BRAF-WT tumours (n = 53) from Newell et al.15. Individual data points are plotted as dots. Statistical significance was assessed using individual one-sided Wilcoxon–Mann–Whitney tests. Each dot is an individual biological replicate (n = 3) with horizontal lines indicating median values. Right, relative expression levels of cutaneous genes across individual normal human melanocytes. Each point represents a biological replicate (n = 3 per condition) with horizontal lines indicating median values. Expression data for d are derived from McNeal et al.40. This result indicates that BRAF-mutated melanomas that occur at acral sites are transcriptionally closer to non-acral cutaneous melanomas, a transcriptional program that is not explained by BRAF downstream signalling, and are associated with increasing European genetic ancestry. We then applied a more stringent quality filter, including coverage and alignment features, to primary tumours in this collection with 44 samples remaining for further analyses (Supplementary Table 1; Methods). Cluster 2 expressed higher levels of a mitotic/proliferative-related signature, with high expression of genes such as MITF and TYR, and processes such as chromosome segregation, nuclear division and mitochondrial translation (Fig. Cluster 3 showed expression of a gene module characterized by respiration and oxidative phosphorylation-related genes (Fig. Deconvolution of gene expression profiles also indicated differences in immune cell infiltration composition, with cluster 1 having a higher proportion of CD4+ T cells and cancer-associated fibroblasts (CAFs) and cluster 2 having a higher proportion of B cell infiltration (Fig. a, Gene expression heatmap showing the 3,870 genes identified as differentially expressed among sample clusters; x axis, samples; y axis, genes. Mutational status and clinical covariates by sample are shown above the heatmap. b, Scaled mean expression patterns per cluster for the three gene modules defining each cluster. c, Box plot of mitotic index (y axis) per sample classified by transcriptional cluster. d, Box plot of B cell proportion (y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. e, Box plot of CD4+ T cell proportion (y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. f, Box plot of CAFs (y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. Individual data points are plotted as dots. For c–f, two-sided Wilcoxon–Mann–Whitney paired tests were performed. Next, we evaluated whether the genomic and transcriptomic characteristics had any impact on patient overall or recurrence-free survival. We included in the analysis those participants whose primary could be analysed (n = 85; Methods). The mean time between diagnosis and recruitment was 2.01 years, including 21 participants recruited within 6 months; the range was from a few days to more than 10 years. Carrying any driver mutation was not associated with age at diagnosis or tumour stage (data not shown). Having a tumour with a driver mutation was, however, associated with a reported recurrence, with 66.7% of mutated tumours having a recurrence as compared with 37.2% of QWT tumours (Pearson two-tailed Chi-squared test P value = 0.007). After adjusting for date of diagnosis, sex, age at diagnosis, ancestry and tumour stage (n = 73, primaries with ancestry information available), the odds ratio for a mutated tumour having a recurrence compared with QWT tumours was 5.31 (95% confidence interval, 1.56, 18.12), (multivariate logistic regression, P value = 0.008) (Fig. Notably, among the mutated tumours, for each different gene, tumour recurrence was increased over QWT tumours (Fig. 5a), most notably for NF1, where all seven of the mutated tumours recurred. a, Recurrence-free survival of patients (n = 85, all those participants whose primary could be analysed) with and without driver mutations, depicted by each category of the mutational classification. b, Recurrence-free survival for patients with tumours in each of the three transcriptional clusters (n = 44). c, Overall survival of patients with and without driver mutations, depicted by each category of the mutational classification (n = 85). d, Overall survival for patients with tumours in each of the three transcriptional clusters (n = 44). Each panel indicates the crude survival curves as indicated. The centre of each error bar is the estimated cumulative survival (recurrence-free or alive proportion) to that time point, and the bars represent its 95% confidence interval (CI). All reported P values are two-sided. There was no association between tumour driver mutation and transcriptomic cluster (data not shown). There was, however, evidence of differences in recurrence frequency by cluster, with 35.7% of cluster 1 tumours, 81.2% of cluster 2 tumours and 57.1% of cluster 3 tumours having a recurrence (Fisher's exact test, P value = 0.04 for homogeneity). Logistic regression adjusting for age at diagnosis, sex, diagnosis date and stage at presentation showed that those tumours from cluster 2 had a higher rate of recurrence as compared with cluster 1 (odds ratio = 6.68; 95% confidence interval, 0.97, 46.27), multivariate logistic regression, P value = 0.054; Supplementary Table 23), whereas cluster 3 had intermediate rates of recurrence (Fig. Fifteen participants (17.6%) died during the study period; 9% of participants with QWT tumours and 26.2% of participants with tumours with driver mutations died (P value = 0.042 for homogeneity). Log rank analysis of time to death from diagnosis showed a tendency for an increased risk of death among those with any mutation (P value = 0.077) (Supplementary Table 24), whereas similar analysis by specific mutation showed more extreme significance (P value = 0.0006) (Supplementary Table 25). Cox proportional hazards analysis adjusting for age, sex, tumour stage and ancestry indicated participants whose tumour had any mutation in a driver gene had a tendency for an increased risk of death (hazard ratio = 3.19, 95% confidence interval, 0.8, 12.74; P value = 0.1), although this was not conventionally significant (Supplementary Table 26) (Fig. Finally, survival analysis based on the 44 tumours with transcriptomic classification showed significant variation between the clusters, with 0% of cluster 1, 43.7% of cluster 2 and 21.4% of cluster 3 having died (log-rank P value = 0.011, Pearson two-tailed Chi-squared P value = 0.017), again in keeping with the analysis of recurrence, a known main risk factor for survival (Fig. Cox proportional hazards analysis adjusting for age at diagnosis, sex, stage at presentation and ancestry did not provide significant evidence of differences between the clusters in terms of mortality rates, in keeping with the limited sample size. In our view, this study helps address several research gaps: (1) the underrepresentation of samples of Latin American ancestry in cancer sample repositories5: as shown previously, genetic ancestry and environment influence the somatic profile of tumours, with potential impacts on patient management and treatment6,19,20; (2) the relative lack of studies of acral melanoma, when compared with other types of the disease, as this type of melanoma constitutes most reported cases in some non-European populations11, and (3) the relative paucity of genomic studies performed and directed from low- and middle-income countries such as Mexico. Most patients in this study had predominantly Amerindian genetic ancestry, which allowed us to perform an analysis of genetic ancestry correlates with somatic mutation profile. We identified a positive correlation between European ancestry and BRAF mutation rate (Fig. A possible link between European ancestry and BRAFV600E mutation had been described previously18, and this study provides further confirmatory evidence. Other similar correlations have been described recently for other types of cancer, such as a positive relationship between Native American ancestry and EGFR mutation rate in lung cancer20, and an increased rate of somatic FBXW7 in African patients compared with European patients6. In accordance with this observation, other cohorts of acral melanoma, which studied patients with predominantly European ancestry, have a higher BRAF mutation rate than that in this study (for example, 23% in Australian patients with predominantly European ancestry15). These observations should provide the basis for future studies exploring the relationships between ancestry and somatic mutation rate. We were intrigued to discover that acral melanomas with BRAF-activating mutations exhibit a more ‘cutaneous melanoma-like' transcriptome than other genetic subtypes of acral melanoma. One possible explanation is that this gene signature is uniquely downstream of a BRAF missense mutation. However, in further analyses, we do not see evidence for this explanation (Fig. An alternative explanation involves the distinct origins of acral melanomas with BRAF-activating mutations compared with other acral melanomas. In our previous work43, we identified distinct subclasses of human epidermal melanocytes: a common type enriched in limbs (c-type) and a rare type enriched in volar regions (v-type). We observed that most acral melanomas generally retained a transcriptional signature such as v-type melanocytes, whereas a subset seemed more akin to c-type melanocytes43. The current work indicates that these tumours are more likely to belong to the BRAF-activating genetic subtype, indicating that a subset of volar melanomas might be classified more accurately by cell of origin and/or genetic profile as non-acral cutaneous melanoma, rather than bona fide acral melanomas. It is important to clarify that the hypothesis that acral melanomas may arise from different cells of origin is not based solely on this study but is also supported by previous work. Our previous research has demonstrated transcriptional diversity among melanocytes in different anatomic locations, including distinct populations of epidermal melanocytes in the palms and soles43. Furthermore, our zebrafish model studies have shown that acral melanoma-associated drivers preferentially (although not exclusively) induce tumours in the limbs (fins), whereas cutaneous melanoma-associated BRAF mutations lead preferentially (but not exclusively) to tumours in the trunk44. Furthermore, we have demonstrated that BRAF mutations selectively drive hyperproliferation in less-pigmented primary human melanocytes40. Therefore, although our additional analyses do not strongly support an oncogene signature as the explanation for the differences in transcriptional scores, thus favouring the cell-of-origin hypothesis, it is possible that, in some cases, these two phenomena could be intertwined. For example, recent data have shown that some acral melanomas harbouring amplifications of the CRKL oncogene depend on HOX13 positional identity programs already present in the cell of origin, indicating that oncogenes and cell-of-origin programs can synergize44. Future studies could explore the diagnosis of cutaneous melanoma as acral versus non-acral based on molecular signatures rather than solely on anatomic location. The fact that BRAF-mutated tumours occur less frequently on patients of non-European ancestry highlights the need to study a diverse set of samples to maximize clinical benefit to all patients. Patients with cluster 1 tumours showed better prognosis than other patients, which is not surprising given their associated clinical characteristics (lower Breslow thickness, a tendency for earlier stages at diagnosis and lower mitotic indexes). However, what may be surprising is the gene expression profile characteristic of this cluster. More CAFs and CD4+ T cells were found by deconvolution to be associated to cluster 1, signatures commonly associated with immunosuppression. A possible explanation is that early-stage tumours are associated with immunosuppressive microenvironments—a balance which, in later tumours, may have been tilted in favour of tumour cell growth. Another potential explanation may involve the recently described roles of CAFs in immunostimulation45. Patients with cluster 2 tumours, with a proliferative signature and expression of pigmentation genes showed the worst survival. It has been observed previously in a zebrafish model and in TCGA samples that a pigmentation signature also predicts worse survival46 and, in a recent report by Liu and collaborators31, acral melanoma tumours with a proliferative signature also were associated with worse survival than other tumours. This study both extends and replicates these findings in acral melanoma. This study was also done using whole-exome data, which limits our ability to call mutations in non-coding regions of the genome. There are other challenges of setting up such a study. For instance, the fact that the year of diagnosis preceded the date of recruitment by up to 10 years means that somatic mutations associated with higher mortality rates would be under-represented among those recruited, whereas survival is probably extended over a similarly sized cohort of prospectively recruited cases. To assess the impact of any biases on our interpretation of the impact of mutations, we performed an analysis with only those tumours diagnosed after mid-2016, that is, those closest to the time of recruitment (Methods), obtaining similar results. All tumours included in this study were confirmed as melanomas arising from glabrous skin. Our data indicate that tumours harbouring BRAF mutations may constitute a distinct subtype, sharing characteristics with superficial spreading and acral melanoma. These findings would have implications for patient selection in clinical trials evaluating new therapies for acral melanoma. Overall, we were able to identify new associations of the germline and somatic profile in acral melanoma, genomic-clinical correlates of overall and recurrence-free survival, as well as transcriptional differences in BRAF-mutated acral melanomas. This study shows the value of studying diverse populations, allowing us to uncover previously unreported relationships and better understand tumour evolution. A flowchart describing the analyses, steps and number of samples used in each individual section can be found in Supplementary Fig. The protocol for sample collection was approved by the Mexican National Cancer Institute's (Instituto Nacional de Cancerología, INCan, México) Ethics and Research committees (017/041/PBI;CEI/1209/17) and the United Kingdom's National Health Service (18/EE/00076). Patient samples collected for the Utah cohort analysis were derived as described previously47. Recruitment of patients and sample collection took place from 2017 to 2019. Patients attending follow-up appointments at INCan who had previously been diagnosed with acral melanoma were offered the chance to participate in this study and, upon signing a written consent form, were asked to provide access to a FFPE sample of their tumour tissue that had been kept at the INCan tumour bank, as well as a saliva or normal adjacent tissue sample. Patients provided samples and their clinical data in Excel format with written informed consent. FFPE samples underwent inspection by a medical pathologist to establish whether sufficient tumour tissue was available for exome sequencing. Saliva samples were collected using the oragene DNA kit (DNAGenotek, catalogue no. DNA extraction from all saliva samples was performed at the International Laboratory for Human Genome Research from the National Autonomous University of México (LIIGH-UNAM) using the reagent prepITL2P (DNAGenotek, catalogue no. PT-L2P) and the All-Prep DNA/RNA/miRNA Universal Kit (Qiagen, catalogue no. DNA and RNA extraction from FFPE samples was performed at the Wellcome Sanger Institute (UK) using the All-prep DNA/RNA FFPE Qiagen kit. Samples with >0.1 ng μl−1 were sequenced through the Sanger Institute's standard pipeline. Saliva and adjacent tissue samples were used for whole-exome sequencing, and only saliva samples were used for genotyping. Genotyping was performed using Illumina's Infinium Multi-Ethnic AMR/AFR-8 v.1.0 array at King's College London and Infinium Global Screening Array v.3.0 at University College London. Ancestry estimation was performed using PLINK v.1.9, and ADMIXTURE48 v.1.3.0 for unsupervised analysis together with the superpopulations of the 1000 Genomes dataset49. Five superpopulations were identified, corresponding to AFR (Q1), AMR (Q2), SAS (Q3), EAS (Q4) and EUR (Q5) (Supplementary Table 2 and Supplementary Fig. FFPE samples, saliva and normal adjacent tissue underwent whole-exome sequencing as follows: Exome capture was performed using Agilent SureSelect AllExon v.5 probes and paired-end sequencing was performed at the Wellcome Sanger Institute (UK) in Illumina HiSeq 4000 machines. Control and tumour samples were sequenced to a mean coverage of 43.72×. Sequencing quality filters were performed using Samtools v.1.9 stats51 and fastqc v.0.11.3 (ref. Sample contamination was estimated using the GATK v.4.2.3.0 tool CalculateContamination53. Concordance between sample pairs was estimated using the Conpair v.0.2 tool54. Samples that had less than 90% similarity with their pair (tumour-normal) or showed a level of contamination above 5% were excluded from the study. After this step, 123 samples remained for further analysis. The nature of our samples (FFPE) may introduce artefacts that affect our ability to identify SNVs and indels accurately. Therefore, to mitigate this risk and increase specificity, we used three different variant calling tools, albeit at the cost of reduced sensitivity. As formalin fixation can generate DNA fragmentation, this may also affect copy number estimation analyses and, consequently, copy number mutational signature analysis. To mitigate this, we stringently filtered the samples used for this analysis, which affected our statistical power due to a reduced sample size. Somatic variant calling was done using three different tools (cgpCaVEMan55 v.1.15.2, Mutect2 (ref. 57) v.2.3.9), keeping only the variants identified by a minimum of two out of the three tools. 58) was used for variant pairs phasing. VCF handling was done using bcftools v.1.9 (ref. For BRAFV600E mutations, we kept these variants even if they were identified by only one of the tools as its oncogenicity and relevance in melanoma is well known. When available within the variant calling tool, strand bias filters were applied. A minimum base quality score of 30 on the Phred scale was used. When selecting one sample per patient, preference was given to primaries, and metastases or recurrences were chosen only when a primary had not been collected. Significantly mutated genes were identified using the tool dNdScv62 v.0.0.1.0 with default parameters using SNVs identified by two of the three tools used for variant calling and indels identified by pindel as input data. Positive selection was considered for genes that had global Q values below 0.1 according to the dNdScv tool recommendations. Visualization of variants was done using Maftools v.2.2.10 (ref. Two lymph node metastasis samples (one from a patient that had a BRAFV600E mutation and another one with an NRAS mutation) and their primaries were annotated as having the same mutation for follow-up analysis after manual inspection using IGV64. Statistical tests were performed to identify potential clinical and ancestry covariates that correlated with driver mutational status. For tumour stage, sex, ulceration status and tumour site, which are discrete variables, association was tested with contingency Chi-squared tests. For each of the four driver genes, a logistic regression model was fitted to predict the presence or absence of a mutation on the acral melanoma samples using the inferred ADMIXTURE48 cluster related to the European ancestry component from the 1000 Genomes Project, correcting for age, sex and total TMB (total TMB, SNVs + indels), as such: driver gene status ~ EUR related cluster proportion + age + sex + total TMB. The log odds related to the EUR cluster were then plotted with their respective confidence intervals. The models were constructed using 80 samples out of the 92, which were those with available genotyping information and with all tested covariables available. These values per sample were compared between the two tools, and samples that had a high discrepancy in their purity estimates (less than 0.15 versus 1, respectively) were filtered out (Supplementary Table 28). Samples with an estimated goodness of fit below 95 were also filtered out. Subsequently, copy number, cellularity and ploidy values estimated by ASCAT were used in follow-up analyses. Whole genome duplication events were assigned as reported by ASCAT. Regions significantly affected by CNAs were identified using GISTIC2 (ref. Amplifications were classified as low-level amplifications when regions had a copy number gain above 0.25 and below 0.9, and as high-level amplifications when regions had a copy number gain above 0.9 according to GISTIC2 values; deletions were classified as low-level deletions when regions had a copy number change between −0.25 and −1.3 and as high-level deletions when regions had a copy number change lower than −1.3. Only peaks with residual Q values < 0.1 were considered as significantly altered. For the analyses of differences in CNA burden by sample group (mutational status or site of presentation), we used the CNApp tool68 to generate GCS, focal copy number alteration scores and broad copy number alteration scores, calculating segment means (seg.mean) as log2(cn/ploidy) and using default parameters. GCS is a number quantifying the copy number aberration level in each sample provided by the CNApp tool68. Higher GCS scores indicate a higher burden of copy number aberrations compared with all other samples in the cohort. GCS is the sum of the normalized broad copy number alteration score and focal copy number alteration score, which are calculated considering broad (chromosome and arm-level) and focal (weighted focal CNAs corrected by the amplitude and length of the segment) aberrations per sample. These values are calculated using as input the number of DNA copies normalized by sample ploidy. A more detailed explanation can be found in the original publication68. GCS values were used for comparisons between sample groups. All paired comparisons between groups were evaluated with a Mann–Whitney test. To further scrutinise the presence of deletions in tumour suppressors NF1 and CDKN2A, we used CNVkit69 v.0.9.10. We called copy number alterations against a pooled reference generated from the highest quality normal samples, and generated bin-level and segmented level log2 ratios. We calculated the log2 ratio estimated for homozygous deletions for each sample based on ASCAT's estimation of ploidy and purity as published previously25. For NF1—a large gene—we considered homozygous focal deletions when at least two contiguous bins had log2 ratios at or below the calculated threshold for that sample, or at least one full exon has read coverage equal to zero. For CDKN2A—a small gene—we considered a sample as having a homozygous deletion if it had at least one bin below the threshold, at least two other bins close to the threshold and a noticeable difference in log2 ratios for bins falling in CDKN2A in comparison with its neighbours by manual scrutiny. These matrices, with single nucleotide mutations found by at least two of the three variant callers and all insertions and deletions identified by cgpPindel, were used as input for mutational signature extraction using SigProfilerExtractor71 v.1.1.23 and decomposition to COSMICv.3.4 (ref. For single base substitutions, the standard SBS-96 mutational context was used, with default parameters and a minimum and maximum number of output signatures being set as one and five, respectively. A total of 116 samples with an SNV count > 0, were used for this analysis. For copy number signature analysis, all 60 samples with available copy number data were used with default parameters, and using the standard CN-48 context from COSMICv.3.4. Total RNA library preparation followed by exome capture using Agilent SureSelect AllExon v.5 was performed on Illumina HiSeq 4000 machines. Of these, we focused on the 77 samples that came from different patients, that had matching DNA and were primaries for the score analysis (see below). We then applied further quality control filters for the consensus clustering analysis: samples were excluded if total read counts were fewer than 25 million, or if the sum of ambiguous reads and no feature counts was greater than the sum of all gene read pair counts. Patient samples collected for the Utah cohort analysis were derived as described previously47. Invasive acral and non-acral cutaneous melanomas were identified and collected as part of the University of Utah IRB umbrella protocol no. A custom NanoString nCounter XT CodeSet (NanoString Technologies) was designed to include genes differentially expressed between glabrous and non-glabrous melanocytes43,44. Sample hybridization and processing were performed in the Molecular Diagnostics core facility at Huntsman Cancer Institute. Data were collected using the nCounter Digital Analyzer. Normalization was carried out using the geometric mean of housekeeping genes included in the panel (Supplementary Table 16). Background thresholding was performed using a threshold count value of 20. The log2 normalized gene expression data were subjected to principal component analysis (PCA) using the PCA function in Prism v.10.2.1 (GraphPad Software). PCA was performed to identify the main sources of variability in the data and to distinguish between acral and cutaneous samples. Genes with the highest positive and negative loadings on PC2 were selected as the top ten and bottom ten genes, respectively; log2 expression values of these genes were used to generate a multiplicative score, producing the ratio of acral to cutaneous melanocyte genes. Statistical analyses were performed using Prism v.10.2.1 (GraphPad Software). Differences in acral to cutaneous ratios were assessed using the Mann–Whitney U test. The acral:cutaneous (A:C) ratio was calculated for each of the 77 primary acral tumours using the method described above after batch correction (limma v.3.64.1, ref. 75) on normalized and transformed expression data processed by the R package DESeq2 v.1.48.1 (ref. Differences in the A:C gene expression ratio scores between BRAF-activating mutation-positive and BRAF-wild-type acral melanoma samples were assessed using a Mann–Whitney U test. The same normalization, scoring method and statistical testing was applied to the 63 transcriptomes from acral melanoma tumours considering BRAF-activating (n = 10) and wild type (n = 53) in Newell et al.15. All available samples in this cohort were used, as only one primary had a BRAF mutation. Only samples with BRAF-activating mutations (V600E and L597R for the Mexican acral melanoma set) were included in the BRAF group. To determine whether the cutaneous melanoma classifier genes are induced by BRAFV600E signalling in melanocytes, we analysed RNA sequencing (RNA-seq) data from McNeal et al.40, which consisted of bulk-RNA-seq of primary human melanocytes transduced with BRAFV600E and cultured under two conditions: phorbol 12-myristate 13-acetate (PMA) and endothelin-1 (ET1). We extracted normalized expression values for cutaneous melanoma classifier genes across four conditions: PMA, PMA + BRAFV600E, ET1 and ET1 + BRAFV600E. Normalized expression levels were compared using the Mann–Whitney U test in Prism v.10.2.1 (GraphPad Software). We evaluated the A:C classifier in clinical melanoma samples using RNA-seq data from TCGA Skin Cutaneous Melanoma Firehose Legacy cohort. Normalized gene expression data were downloaded from cBioPortal77. Samples were classified as BRAF-activating or BRAF-wild type in the same way as for Fig. We calculated the product of the expression of cutaneous melanoma classifier genes for each category. Differences were assessed using the Mann–Whitney U test. We used an interactive Shiny application, What Is My Melanocytic Signature (WIMMS; https://wimms.tanlab.org)41 to compare transcriptional programs associated with distinct melanocytic cell states. WIMMS classifies melanocytic gene expression profiles by aggregating previously published gene expression signatures and clusters them into seven principal cell state categories. We input our gene signature into WIMMS to assess correlation with these reference states. 6b) represents the relationship between our classifier-derived cutaneous melanoma genes and known signatures. To identify molecular subgroups based on transcriptome data, we performed consensus clustering using the Cola R v.2.10.1 package78. Consensus clustering was performed using several algorithms (k-means, partitioning around medoids, hierarchical clustering) and feature selection methods (s.d., median absolute deviation, coefficient of variation) to ensure robust partition identification. The optimal number of clusters was determined using several stability metrics including 1-PAC (Proportion of Ambiguous Clustering) score, concordance and index, Jaccard index, coefficient and visual inspection of the consensus matrix through heatmaps visualizations. :partitioning around medoids with k = 3) was selected based on highest consensus scores and biological interpretability. Following sample clustering, we performed a two-level signature analysis to characterize both sample clusters and gene co-expression modules. Characterization of sample clusters was performed by identifying genes with significantly different expression across the three identified sample groups using F-tests with false discovery rate correction (P < 0.05). For each differentially expressed gene, we determined the sample cluster with highest mean expression to characterize the molecular profiles of patient subgroups. To understand co-expression patterns within signature genes, we applied k-means clustering (k = 3) to group signature genes based on their scaled expression profiles across the three sample clusters. This identified gene modules (M1, M2, M3) representing distinct biological programs that may be co-ordinately regulated across different sample subtypes (Supplementary Tables 17–20). Functional enrichment was performed separately for each gene module using over-representation analysis with the clusterProfiler R package v.4.12.6 (ref. Gene Ontology Biological Process terms were tested using the hypergeometric test with Benjamini–Hochberg false discovery rate correction (Q < 0.05). 80) annotation package before enrichment analysis (Supplementary Tables 17–20). The EPIC algorithm81 v.1.1.7 was used in the R programming environment to perform deconvolution to infer immune and stromal cell fractions within acral melanoma tumours. We used the TRef signature method with default parameters, which includes gene expression reference profiles from tumour-infiltrating cells. The algorithm generated an absolute score that could be interpreted as a cell fraction. Because of the challenges of recruiting sufficient numbers of participants with acral melanoma, patients diagnosed in earlier years who were still attending follow-up clinics for primary or recurrent disease were recruited. To ensue data consistency, only participants with a primary available for analysis were the subject of focus on analyses of recurrence and/or death (n = 85 patients). In total, 73 participants were recruited whose primary and ancestry data were available for analysis on driver mutations. A total of 44 patients with primary and RNA cluster data available were used for analysis on clusters regardless of their ancestry data availability. For recurrence, we examined time to recurrence using a life-table approach from date of primary diagnosis onwards as a descriptor, and based analyses of differences between mutations (and clusters) on logistical regression adjusting for date of diagnosis, age at diagnosis, sex, stage at diagnosis (advanced/early), ancestry (only for mutations) and time from diagnosis through either death or last known to be alive. Primary tumours were either classified as QWT or mutated for known drivers; tumours with several driver mutations were classified as ‘multi-hit'. We also conducted analyses based on a binary exposure of ‘QWT' or ‘Mutated' tumour based on the existence of one or more mutations in a known driver gene. For survival analysis, we also included a life-table approach, again as a descriptor, based on time from diagnosis through to date of death or date last known to be alive. Statistical assessment of the effect of each mutation and/or cluster were based on Cox proportional hazard analysis with follow-up starting from date of recruitment through to date of death (or last date alive) adjusting for date of diagnosis, age at diagnosis, sex and ancestry (as a sensitivity analysis). Analysis of survival gave qualitatively and quantitatively similar results to those reported above; in total, 7 of 23 (30.4%) of cases with mutated tumours had died, as opposed to 1 of 17 among cases with QWT tumours (P value = 0.055). Results for the RNA clusters were similar to the results quoted above for both survival and recurrence. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequencing data are available at the European Genome-Phenome Archive (EGA). DNA sequencing data are available under ENA accession number EGAD00001015755 and RNA-seq data under ENA accession number EGAD00001015756. The 1000 Genomes Project datasets can be downloaded from https://www.internationalgenome.org/data. The GRCh38 reference genome can be downloaded from https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.40/. Sequencing data for the Newell et al.15 study is available from the EGA under study accession EGAS00001001552 and dataset accession EGAD00001005500. Information on how to apply for access is available at the EGA dataset link: https://ega-archive.org/datasets/EGAD00001005500. TCGA Skin Cutaneous Melanoma Firehose Legacy cohort data can be downloaded from cBioPortal (https://www.cbioportal.org/). Code for reproducing the analyses in this paper is available at https://github.com/CGBio-Lab/Mex-acral-exomes-transcriptomes. Fujisawa, Y. et al. Clinical and histopathological characteristics and survival analysis of 4594 Japanese patients with melanoma. Swan, M. C. & Hudson, D. A. Malignant melanoma in South Africans of mixed ancestry: a retrospective analysis. Lino-Silva, L. S. et al. Melanoma in Mexico: clinicopathologic features in a population with predominance of acral lentiginous subtype. Molina-Aguilar, C. & Robles-Espinoza, C. D. Tackling the lack of diversity in cancer research. Carrot-Zhang, J. et al. Comprehensive analysis of genetic ancestry and its molecular correlates in cancer. Amuzu, S. et al. Meta-analysis reveals differences in somatic alterations by genetic ancestry across common cancers. The genetics of Mexico recapitulates Native American substructure and affects biomedical traits. & Robles-Espinoza, C. D. Melanoma subtypes: genomic profiles, prognostic molecular markers and therapeutic possibilities. Basurto-Lozada, P. et al. Acral lentiginous melanoma: basic facts, biological characteristics and research perspectives of an understudied disease. Distinct sets of genetic alterations in melanoma. Hayward, N. K. et al. Whole-genome landscapes of major melanoma subtypes. Newell, F. et al. Whole-genome sequencing of acral melanoma reveals genomic complexity and diversity. Distinct genomic features in a retrospective cohort of mucosal, acral and vulvovaginal melanomas. Whole genome sequencing of matched primary and metastatic acral melanomas. Nassar, A. H. et al. Ancestry-driven recalibration of tumor mutational burden and disparate clinical outcomes in response to immune checkpoint inhibitors. Carrot-Zhang, J. et al. Genetic ancestry contributes to somatic mutations in lung cancers from admixed Latin American populations. & Haiman, C. A. Genome-wide association studies of cancer in diverse populations. Campbell, P. J. et al. Pan-cancer analysis of whole genomes. The eighth edition AJCC cancer staging manual: continuing to build a bridge from a population-based to a more ‘personalized' approach to cancer staging. & Bastian, B. C. Integrated genomic analyses of acral and mucosal melanomas nominate novel driver genes. Wong, K. et al. Cross-species genomic landscape comparison of human mucosal melanoma with canine oral and equine melanoma. Hendricks, W. P. D. et al. Somatic inactivating PTPRJ mutations and dysregulated pathways identified in canine malignant melanoma by integrated comparative genomic analysis. Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Expanding the landscape of oncogenic drivers and treatment options in acral and mucosal melanomas by targeted genomic profiling. The tumor suppressor adenomatous polyposis coli and caudal related homeodomain protein regulate expression of retinol dehydrogenase L. J. Biol. Farshidfar, F. et al. Integrative molecular and clinical profiling of acral melanoma links focal amplification of 22q11.21 to metastasis. Krauthammer, M. et al. Exome sequencing identifies recurrent somatic RAC1 mutations in melanoma. Alexandrov, L. B. et al. Clock-like mutational processes in human somatic cells. Geographic variation of mutagenic exposures in kidney cancer genomes. Steele, C. D. et al. Signatures of copy number alterations in human cancer. Everall, A. et al. Comprehensive repertoire of the chromosomal alteration and mutational signatures across 16 cancer types from 10,983 cancer patients. McNeal, A. S. et al. BRAFV600E induces reversible mitotic arrest in human melanocytes via microrna-mediated suppression of AURKB. Hu, M., Coleman, S., Judson-Torres, R. L. & Tan, A. C. The classification of melanocytic gene signatures. Global analysis of BRAFV600E target genes in human melanocytes identifies matrix metalloproteinase-1 as a critical mediator of melanoma growth. Belote, R. L. et al. Human melanocyte development and melanoma dedifferentiation at single-cell resolution. Weiss, J. M. et al. Anatomic position determines oncogenic specificity in melanoma. Tsoumakidou, M. The advent of immune stimulating CAFs in cancer. Kim, I. S. et al. Microenvironment-derived factors driving metastatic plasticity in melanoma. Deacon, D. C. Classification of cutaneous melanoma and melanocytic nevi with microrna ratios is preserved in the acral melanoma subtype. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Auton, A. et al. A global reference for human genetic variation. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Andrews, S. FastQC: a quality control tool for high throughput sequence data. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Bergmann, E. A., Chen, B.-J., Arora, K., Vacic, V. & Zody, M. C. Conpair: concordance and contamination estimator for matched tumor–normal pairs. Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Hager, P., Mewes, H.-W., Rohlfs, M., Klein, C. & Jeske, T. SmartPhase: accurate and fast phasing of heterozygous variant pairs for genetic diagnosis of rare diseases. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Raine, K. M. et al. cgpPindel: identifying somatically acquired insertion and deletion events from paired end sequencing. Ye, K., Schulz, M. H., Long, Q., Apweiler, R. & Ning, Z. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. Martincorena, I. et al. Universal patterns of selection in cancer and somatic tissues. Mayakonda, A., Lin, D.-C., Assenov, Y., Plass, C. & Koeffler, H. P. Maftools: efficient and comprehensive analysis of somatic variants in cancer. Robinson, J. T. et al. Integrative genomics viewer. Favero, F. et al. Sequenza: allele-specific copy number and mutation profiles from tumor sequencing data. Ross, E. M., Haase, K., Van Loo, P. & Markowetz, F. Allele-specific multi-sample copy number segmentation in ASCAT. Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Franch-Expósito, S. et al. CNApp, a tool for the quantification of copy number alterations and integrative analysis revealing clinical implications. Talevich, E., Shain, A. H., Botton, T. & Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. Bergstrom, E. N. et al. SigProfilerMatrixGenerator: a tool for visualizing and exploring patterns of small mutational events. Islam, S. M. A. et al. Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. Sondka, Z. et al. COSMIC: a curated database of somatic variants and clinical data for cancer. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Gu, Z., Schlesner, M. & Hübschmann, D. cola: an R/Bioconductor package for consensus partitioning through a general framework. clusterProfiler: an R package for comparing biological themes among gene clusters. Genome wide annotation for human v.3.19.1 (Bioconductor, 2025). Racle, J., de Jonge, K., Baumgaertner, P., Speiser, D. E. & Gfeller, D. Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data. We are also thankful to members of the CGBio laboratory team at LIIGH-UNAM for valuable discussions regarding the findings in this article. We wish to thank L. A. Aguilar, A. de León and A. Avalos from the Laboratorio Nacional de Visualización Científica Avanzada and J. S. García Sotelo, A. Hernández, E. Lomelín, I. Martínez, R. Muciño, M. A. Ávila, A. Castillo and C. Uribe Díaz from the International Laboratory for Human Genome Research, National Autonomous University of Mexico. We are grateful to the International Cancer Genome Consortium Data Access Committee for granting access to ICGC controlled data. We are also thankful to K. Wong and D. Desposorio for useful discussions. Work included in this paper has been funded by Wellcome Trust (204562/Z/16/Z and 227228/Z/23/Z to C.D.R.-E.), the Melanoma Research Alliance (Pilot Award #825924, to C.D.R.-E.), the Mexican National Council of Humanities, Science and Technology (CONAHCYT/SECIHTI, FOSISS A3-S-31603, to C.D.R.-E.), Programa de Apoyo a Proyectos de Investigación e Innovación Tecnológica (PAPIIT UNAM) (IN209422 to C.D.R.-E.), the Academy of Medical Sciences through a Newton Advanced Fellowship (NAF/R2/180782) and the Wellcome Sanger Institute through an International Fellowship. C.D.R.-E. is grateful to the William Guy Forbeck Research Foundation for their generous support and for promoting a collaborative and rich environment that helped advance the ideas underlying this study. This work was funded in part by the Melanoma Research Alliance Dermatology Fellows award to D.C.D., the Harry J Lloyd Charitable Trust Melanoma Research Grant to R.L.J.-T., a National Cancer Institute R01 (R01CA229896) to R.L.J.-T. and pilot funds from the Huntsman Cancer Institute Melanoma Center. We used the Shared Resources for Research Informatics and High-Throughput Genomics and Bioinformatics Analysis, each supported by the National Cancer Institute of the National Institutes of Health under Award Number P30CA042014. M.D.-G. and P.G.-G. were awarded fellowships within the ‘Generación D' initiative, Red.es, Ministerio para la Transformación Digital y de la Función Pública, for talent attraction (C005/24-ED CV1), funded by the European Union NextGenerationEU funds, through PRTR. This work was in part supported by the US National Institute of Health grants R01ES032547, R01ES036931, R01CA269919, R01CA296974, P01CA281819 and U01CA290479 to L.B.A. 's Packard Fellowship for Science and Engineering and the UC San Diego Sanford Stem Cell Institute. P.B.-L. is a PhD student from Programa de Doctorado en Ciencias Biológicas, Universidad Nacional Autónoma de México (UNAM), and was supported by Consejo Nacional de Humanidades, Ciencia y Tecnología (CONAHCyT, now known as SECIHTI) (holder no. P.B.-L. is grateful to the Posgrado en Ciencias Biológicas for the support received during her doctoral studies. This paper is part of P.B.-L.'s requirements for obtaining a Doctoral degree at the Posgrado en Ciencias Biológicas, UNAM. Patricia Basurto-Lozada, Martha Estefania Vázquez-Cruz, Christian Molina-Aguilar, Irving Simonin-Wilmer, Fernanda G. Arriaga-González, Kenya L. Contreras-Ramírez, Emiliano Ferro-Rodríguez, J. Rene C. Wong-Ramirez, Johana Itzel Ramos-Galguera, O. Isaac García-Salinas, Rebeca Olvera-León & Carla Daniela Robles-Espinoza Huntsman Cancer Institute, University of Utah Health Sciences Center, Salt Lake City, UT, USA Amanda Jiang, Dekker C. Deacon & Robert L. Judson-Torres Amanda Jiang, Dekker C. Deacon & Robert L. Judson-Torres Fernanda G. Arriaga-González, Jamie Billington, O. Isaac García-Salinas, Ingrid Ferreira, Rebeca Olvera-León, Louise van der Weyden, Martín del Castillo Velasco-Herrera, Patrícia A. Possik, David J. Adams & Carla Daniela Robles-Espinoza Research Programs Unit, Research Program in Systems Oncology, Faculty of Medicine, University of Helsinki, Helsinki, Finland Surgical Oncology, Skin, Soft Tissue and Bone Tumors Department, National Cancer Institute, Mexico City, Mexico Laboratorio de Genómica del Cáncer, Instituto Nacional de Medicina Genómica (INMEGEN), Mexico City, Mexico Department of Histopathology, University Hospitals Sussex, St Richard Hospital, Chichester, UK Dermato-Oncology Clinic, Research Division, Faculty of Medicine, Universidad Nacional Autónoma de México, Mexico City, Mexico Pediatric Dermatology Service, Hospital General de México Dr. Eduardo Liceaga, Ministry of Health, Mexico City, Mexico Surgical Oncology, Bajio Regional High Specialty Hospital, Leon, Mexico Division of Surgery, Instituto Nacional de Ciencias Médicas y Nutrición Salvador Zubirán, Mexico City, Mexico Department of Molecular Genetics, The Ohio State University, Columbus, OH, USA Department of Dermatology, The Ohio State University, Columbus, OH, USA Department of Cellular and Molecular Medicine, University of California San Diego, La Jolla, CA, USA Moores Cancer Center, University of California San Diego, La Jolla, CA, USA Division of Basic and Experimental Research, Brazilian National Cancer Institute, Rio de Janeiro, Brazil Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar did sample cataloguing and nucleic acid extraction. provided computational resources and advice on statistical analyses. assessed patients and provided access to biological samples. A.H.-M. provided facilities for sample processing and supervised that part of the work. Y.S.-P. provided access to patient clinical information and supervised that part of the work. provided data and information that crucially helped the interpretation of the results in this manuscript. C.D.R.-E. wrote the manuscript with assistance from P.B.-L., P.A.P., R.L.J.-T. and D.J.A. is a co-founder, CSO, scientific advisory member and consultant for io9 (now Acurion), has equity and receives income. The terms of this arrangement have been reviewed and approved by the University of California, San Diego in accordance with its conflict-of-interest policies. is also an inventor of a US Patent 10,776,718 for source identification by non-negative matrix factorization. All other authors declare no competing interests. Nature thanks Hunter Shain and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Red dots indicate bins falling in the gene region. PD40986d and PD41020d are secondary samples and a different sample was selected as representative for these patients. a) Oncoplot depicting the seven most mutated genes according to dNdScv and five selected genes based on mutational frequency and biological function. Mutational classification, sample type, tumour stage, sex, age at diagnosis, ulceration status, tumour site and mutational spectra are shown by sample. Primary samples were selected preferentially for this analysis. One sample where no mutations were detected is not depicted in the oncoplot. b) Mutations found in PTPRJ, ATM, NF2 and RDH5, for which all mutations are deleterious and are found altered each in two samples. All depicted regions have been identified by GISTIC2 analysis per group of samples. Differences are determined first by assessing the global GISTIC2 output and determining differences between groups by one-sided Fisher's exact test (P-value < 0.05). If a region is not found in the global GISTIC2 output, but it is found only in the analysis per group, we have indicated it as statistically different. Number of significant regions altered by sample. Binary heatmap showing the significantly altered regions identified by GISTIC2 per sample. One sample per patient is shown. Heatmap is ordered on the X axis by mutational classification and on the Y axis by frequency of alterations per region. As a note, sample PD41002a does not show a deletion in CDKN2A (Deletion peak 3) by GISTIC2 analysis, but a deletion was detected by CNVkit (Methods, Extended Data Fig. a) The SNV component of tumour mutational burden per sample. b-c) Proportions of mutational signatures per sample are shown in stacked bars for single base substitutions (b), and copy-number aberrations (c). In b) and c), samples with a light gray background did not have data available. Genomic subtypes and clinical characteristics are plotted at the bottom. As a note, mutational signature CN48F in c) is a de novo mutational signature that was not successfully reconstructed by COSMIC reference mutational signatures, and was therefore considered as novel. A) Scatter plot comparison of Acral:Cutaneous gene expression ratio in cutaneous melanoma samples from The Cancer Genome Atlas (TCGA) stratified by activating mutation status. Statistical significance was assessed using individual Mann-Whitney U test. B) Hierarchical clustering dendrogram generated using the WIMMS platform to compare the cutaneous melanoma classifier genes to other published molecular signatures, including a signature of genes activated by mutant BRAFV600E in melanoma cells (Ryu_2011_BRAFV600E_Targets)42. P-values were estimated with Kruskal-Wallis tests. P-values were estimated with Kruskal-Wallis tests. P-values were estimated with Kruskal-Wallis tests. A) Box plot of macrophage proportion (Y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. B) Box plot of endothelial cell proportion (Y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. C) Box plot of other cells (non-immune), as calculated by deconvolution, per sample classified by transcriptional cluster. D) Box plot of CD8 T cell proportion (Y axis), as calculated by deconvolution, per sample classified by transcriptional cluster. Individual data points are plotted as dots. This zipped file contains Supplementary Tables 1–4, 7–15, 17–21 and 28. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Basurto-Lozada, P., Vázquez-Cruz, M.E., Molina-Aguilar, C. et al. Ancestry and somatic profile indicate acral melanoma origin and prognosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a70405991/singularity-moores-law-race/'>Inside the Race to Achieve the Singularity—Before Moore's Law Runs Out</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 15:47:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>AI optimists envision a future where artificial general intelligence (AGI) surpasses human intelligence, but the path remains riddled with scientific and logistical hurdles. Once upon a time, the world seemed like it was made of larger building blocks. Rain fell from what looked like opaque, puffy masses that also blocked the Sun. Even when alchemists were melting pieces of ore, they thought mercury must be related to silver. Higher resolutions, more powerful zoom, electron microscopes, particle accelerators, nuclear energy. For technology, powering these levels of detail has fallen into the broad purview of computer chips—and computer chips have also gotten higher-resolution, in a sense. Early punchcards (predecessors to the chips we have now) had openings so that portions of circuitry could literally form a connection, like playing certain notes on the piano using certain fingers, or connecting phone lines on an old switchboard. But our knowledge of electronics has grown so minuscule that it's almost hard to fathom. Moore was a cofounder of computer chip pioneer Intel, and it was there that he noticed that transistors—switches used to direct the current within electrical devices—were shrinking at a pretty consistent rate. This led to the 1958 invention of the integrated circuit, which could be installed in devices that were previously built one transistor at a time. But Moore's Law, as it was later called, has held for decades past this initial forecast. But for the last several years, those in the computing industry (and those who study it) have started to discuss the “end of Moore's Law.” There's a point at which transistors simply can't get any smaller because of the basics of physics itself—these tiny transistors must still be able to communicate with the rest of what's required to build an integrated circuit, be widely manufacturable, and stay cost effective. The slowdown of Moore's Law has been notable for a while. From MIT's Computer Science and Artificial Intelligence Lab (CSAIL): Companies like OpenAI make opaque promises about how generative AI will change all of our lives, save us hours a week, and make many sectors of human labor obsolete. Manufacturers and generative AI companies are already doing this. But that's not a long-term solution to the growing demand for this amount of computing. Like leadership of the late Roman Empire or the icing on a dry cake, our computing components can't be spread too thin. However, if you don't like the idea of a limit on computing, you can turn to futurism, longtermism, or “AI optimism,” depending on your favorite flavor. The goal of these AI boosters is known as artificial general intelligence, or AGI. They theorize, or even hope for, an AI so powerful that it thinks like... well... a human mind whose ability is enhanced by a billion computers. AI optimists want to accelerate the singularity and usher in this “godlike” AGI. One of the key facts of computer logic is that, if you can slow the processes down enough and look at it in enough detail, you can track and predict every single thing that a program will do. Algorithms (and not the opaque AI kind) guide everything within a computer. Over the decades, experts have written the exact ways information can be sent, one bit—one minuscule electrical zap—at a time through a central processing unit (CPU). From there, those bits are assembled into a slightly more concrete format as another type of code. Networks work the same way, with your video or document broken into pieces, then broken down further and further until tiny packets of data can be carted back and forth as electrical zaps over lengths of wire. The human brain is, in some ways, another piece of electrical machinery. The National Institute of Standards and Technology (NIST) quantifies it as an exaflop caliber computer: “a billion-billion (1 followed by 18 zeros) mathematical operations per second—with just 20 watts of power.” By this standard, you power dozens of human brains by plugging them into a single U.S. household outlet. It's possible that the human brain is also predictable when you understand all of its parts and influences enough. And despite an iconic, massively influential paper stating otherwise, the cell is not like a machine. There are countless ways the human brain could be boosted or hindered by factors we can't even measure yet. That seems especially true when aspiring “AI caretaker” engineers want their AIs to know everything from all of human history. But let's say that efficiency or quantity of information isn't an issue. Let's say we can build one-million-exaflop computers to run advanced AIs that will mimic human think tanks. How does the end of Moore's Law affect scientists who work toward that technological singularity? That's both the size of electrical energy required and the physical size associated with storage, processing, cooling, and everything else required to keep a computer running. There are a few directions we could go to solve the size problem, but none of them are easy to achieve.AI boosters push nuclear fusion (another technology that is still far away) as a cure-all for the energy problems associated with large AI computing. But no one knows for sure when (or if) nuclear fusion will produce more energy than what is required to run nuclear fusion facilities. The Kardashev scale is a thought exercise about Solar System- or galaxy-scale civilizations. But while Moore's Law was a forecast based on expertise in both technology and global supply chains, the Kardashev scale and Dyson spheres are thought exercises with no real-life analog at all. On a more grounded level, quantum computing has been touted as an advance toward the realm of AI, ultimately leading into the singularity. But quantum computing is in its infancy, to say the least. It currently requires extreme cooling unlike anything in today's traditional computer realm. There is no usable consumer version of a quantum computer, and we're not even close to one. All of that means we have a lot of options that are at least 10 years away—or even as much as 100 or 1,000 years away. Venture capitalists today are selling a vision of the future. In the huge field of artificial intelligence, there are countless ways to define and work toward goals like finding new prescription drugs or faraway galaxies. AGI is a separate, specific idea, but even within that there are variations. I personally believe that AGI is very far away—though some very smart people, like Google DeepMind and Imperial College London computer scientist Murray Shanahan, believe it's closer than I think. (Shanahan's book for MIT Press about the technological singularity is a great introduction.) Then there's OpenAI's Sam Altman, who has suggested a Dyson Sphere that encloses our Solar System, for example, as a back-of-the-napkin solution to the rising energy costs of AI. In 2019, over 750 million people on Earth still didn't have access to electricity, an additional over 400 million aren't able to use local available electricity, and both numbers are subject to stagnation or even worsening in the wake of the global COVID-19 pandemic. We would need to drain the entire Solar System (and more!) of certain elements to even build what Altman suggests. While Moore's Law is real, many factors of the singularity are not—at least, not this decade. This is one of its best use cases, because the human mind is just not good at this kind of work. The same way we can look around a room and categorize and remember many details at a glance, computers can plug away at enormous lists of ingredients without missing a beat or losing their place. In 2024, Oregon State University chemist Mas Subramanian (the creator of the novel pigment YInMn Blue) told Popular Mechanics that algorithms to discover new molecules are difficult to work with because of factors that the public doesn't really understand. It's just not that easy to find a new pigment, for example—YInMn blue has an unusual crystal structure. The chemical reaction that makes the color is found in a bipyramidal shape, Subramanian explains, rather than a tetrahedral or octohedral network. (Bipyramidal is like two tetrahedrons, or “D4” shapes, glued together. The octohedron has eight faces in a different form.) Graphite and diamond are different crystalline forms of the same element. That need for context is a major limitation of algorithms as we know them. So, Subramanian explains, the machine learning algorithm suggests a long list that must be vetted by a human, and many suggestions don't work in real life right off the bat. And because these models are trained on what already exists, they can't innovate, in the most literal sense. The landmarks of expert-level artificial intelligence studies don't sound like sales pitches or soundbytes—they sound more like Shanahan's clarifying note, written after he used some imprecise language in a paper that escaped containment and entered the mainstream press: He concludes: “The aim, rather, was to remind readers of how unlike humans LLM-based systems are, how very differently they operate at a fundamental, mechanistic level, and to urge caution when using anthropomorphic language to talk about them.” It's very different than Altman's public comment that he might need to Dyson-Shere the entire Solar System. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. A Man and His Daughter Discovered Ancient Rock Art</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/fda-agrees-to-review-moderna-mrna-flu-vaccine-in-dramatic-reversal/'>FDA agrees to review Moderna mRNA flu vaccine in dramatic reversal</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 14:50:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>FDA agrees to review Moderna mRNA flu vaccine in dramatic reversal The U.S. Food and Drug Administration will review a messenger RNA (mRNA) flu vaccine for approval, according to its maker, Moderna. On Wednesday, Moderna said it had made modifications to its application. Secretary of Health and Human Services Robert F. Kennedy, Jr., whose department has jurisdiction over the FDA, is a noted vaccine skeptic who has repeatedly criticized mRNA COVID vaccines. HHS spokesperson Andrew Nixon said in a statement that the FDA had held discussions with the company, leading to “a revised regulatory approach and an amended application, which FDA accepted.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “FDA will maintain its high standards during review and potential licensure stages as it does with all products,” Nixon said. William Schaffner, an infectious disease physician and a professor at Vanderbilt University Medical Center, says the FDA's decision to backtrack is “good news.” “It is important to give all candidate new vaccines a fair equitable assessment. This is especially true for new mRNA-based vaccines as this technology currently is being applied to create vaccines against a variety of illnesses, including cancers," he says. Modern's mRNA flu shot is based on the same technology as its COVID vaccine. The mRNA COVID shots have been credited with saving millions of lives. In these kinds of shots, mRNA—essentially the instruction manual for genes to make proteins—is injected into the body, where it teaches cells to recognize and attack viral proteins. Vaccines that use mRNA are attractive prospects for protecting against flu and a host of other diseases, including cancer. Having such a vaccine available for flu would “potentially be a major step forward in efforts to protect the health of individuals from severe influenza,” says Robert Hopkins, medical director of the National Foundation for Infectious Diseases. Editor's Note (2/18/26): This article was updated after posting to include comments from Andrew Nixon, Angie Rasmussen and Robert Hopkins. This is a breaking news story and may be updated further. Originally from Scotland, she moved to New York City in 2012. Her work has appeared in National Geographic, Slate, Inc. Magazine, Nautilus, Semafor, and elsewhere. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a70328740/ai-fatal-flaw/'>Scientists Found AI's Fatal Flaw—The Most Advanced Models Are Failing Basic Logic Tests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Identifying vulnerabilities is good for public safety, industry, and the scientists making these models. In a new paper that's making waves, scientists from Stanford, Cal Tech, and Carleton College have combined existing research with new ideas to look at the reasoning failures of large language models (LLMs) like ChatGPT and Claude. Those who rely on LLMs for intellectual labor often cite the models' reasoning ability as a major draw, despite the evidence that this ability is limited, even when dealing with simple questions. One of the major lines of criticism leveled by today's AI skeptics goes something like this: large language models work much like your phone's autocomplete—spicy autocomplete, so to speak. Huge portions of the public internet, books, magazines, academic journals—whatever is most relevant to a particular model—are transformed into code that then organizes everything into complicated lists. Furthermore, while computing in general is not like the human brain almost at all, LLMs have something in common with the way we humans think. We may think of computers as doing binary arithmetic, but LLMs start with college-level linear and matrix algebra, and only get more complicated from there. All of this math behind the curtain can give the impression that an LLM is thinking or sentient, but it is not. An LLM is, however, capable of certain types of associative reasoning—a technical and philosophical term meaning that it can consider information and apply logic to draw a conclusion. Yet, as the new research paper's authors make clear, there are limits. “Despite these advances, significant reasoning failures persist, occurring even in seemingly simple scenarios.” In their review—which is available now on the preprint site arXiv, as well as through the online resource Transactions on Machine Learning Research—the scientists categorized LLM reasoning failures and picked out common categories of errors, some of which are listed below. (You can also find a link to their repository of compiled references and research here.) The news sounds bad (and it is), but identifying weaknesses and working to mitigate them is key to developing any model or product. For example, the scientists pointed out architecture and training as an area for feasible large improvements: “[R]oot cause analyses in those categories are particularly rich, suggesting meaningful methods not only for mitigating the specific failures, but for generally improving the architecture and our understanding of it.” In other words, large language models are great for lots of things, but they're not the path to artificial general intelligence. Root cause analysis across all the types of reasoning failures that LLMs display. Failure-injection principles, applied “by adding adversarial sections, multi-level task difficulty, or cross-domain compositions designed to trigger known weaknesses.” Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. Are There More Species on Earth Than We Thought?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260217005759.htm'>Viagra and shingles vaccine show surprising promise against Alzheimer's</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 12:04:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have pinpointed three already approved medications that may be repurposed to treat or prevent Alzheimer's disease. Instead of starting from scratch, scientists examined medicines that are currently used for other conditions to see whether any could help protect the brain. Viagra (sildenafil) and a medication used to treat motor neurone disease (riluzole) also showed strong potential. Their goal was to identify which ones showed the greatest promise for treating or preventing Alzheimer's disease, which accounts for more than half of all dementia diagnoses. After multiple rounds of review, the panel agreed on three 'priority candidates' for further research. Each drug was selected because it targets biological processes linked to Alzheimer's, has shown encouraging results in cell and animal studies, and is considered safe for use in older adults. Experts are now calling for clinical trials to determine whether these medications truly benefit people who have Alzheimer's or are at risk of developing it. It requires no more than two doses and has a long record of safety. Previous research suggests that people who received the vaccine were about 16% less likely to develop dementia. PROTECT is an online registry in which volunteers complete annual questionnaires about their health and lifestyle and take part in brain health research. Five additional medications were shortlisted but did not meet the criteria to be named 'priority candidates.' Dr. Anne Corbett, Professor of Dementia Research at the University of Exeter, said: "Beating dementia will take every avenue of research -- from using what we already know, to discovering new drugs to treat and prevent the condition. "It's important to stress that these drugs need further investigation before we will know whether they can be used to treat or prevent Alzheimer's. This is what we want to see in the field of dementia, and why we believe drug repurposing is one of the most exciting frontiers in dementia research." Note: Content may be edited for style and length. Scientists Say Your Fingers Hold a Secret of Brain Evolution Scientists Discover How To “Switch Off” Cancer Genes for Good Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2026/02/260217005717.htm'>Breakthrough CRISPR system could reverse antibiotic resistance crisis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 08:10:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Antibiotic resistance (AR) has escalated rapidly in recent years, growing into a serious global health emergency. Disease-causing bacteria are continually adapting, finding new ways to survive treatments that once eliminated them. As a result, more drug resistant "superbugs" are spreading, and projections suggest that by 2050 they could be responsible for more than 10 million deaths worldwide each year. These dangerous bacteria often thrive in hospitals, wastewater treatment facilities, livestock operations, and fish farms. Researchers at the University of California San Diego are now using powerful new gene editing tools to directly counter antibiotic resistance. Professors Ethan Bier and Justin Meyer of the UC San Diego School of Biological Sciences have teamed up to create a new way to remove resistance traits from bacterial populations. Their approach builds on CRISPR gene editing and borrows concepts from gene drives, which are used in insects to block the spread of harmful traits such as malaria carrying parasites. The team developed a second generation Pro-Active Genetics (Pro-AG) system called pPro-MobV. "With pPro-MobV we have brought gene-drive thinking from insects to bacteria as a population engineering tool," said Bier, a faculty member in the Department of Cell and Developmental Biology. The foundation for this work began in 2019, when Bier's lab partnered with Professor Victor Nizet's team (UC San Diego School of Medicine) to design the original Pro-AG system. That earlier version introduced a genetic cassette into bacteria, allowing it to copy itself between bacterial genomes and shut down antibiotic resistance genes. This cassette specifically targets resistance genes carried on plasmids, which are small circular DNA molecules that replicate inside bacterial cells. Importantly, the team showed that this method works inside biofilms. They are involved in most serious infections and help bacteria survive antibiotic treatment by forming a protective barrier that limits how easily drugs can penetrate. Because of this, the new approach could have important applications in hospitals, environmental cleanup efforts, and microbiome engineering. "The biofilm context for combating antibiotic resistance is particularly important since this is one of the most challenging forms of bacterial growth to overcome in the clinic or in enclosed environments such as aquafarm ponds and sewage treatment plants," said Bier. Phage are already being engineered to fight antibiotic resistance by slipping past bacterial defenses and delivering disruptive genetic material into cells. As an added safeguard, the platform can include a process known as homology-based deletion, which allows scientists to remove the inserted genetic cassette if necessary. Scientists Discover How To “Switch Off” Cancer Genes for Good Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260217005738.htm'>NASA fired three rockets into the northern lights and the results are stunning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 04:21:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA has successfully launched two sounding rocket missions from Alaska to investigate the powerful electrical forces behind the northern lights. Principal investigator Marilia Samara reported that all instruments, including technology demonstrations, performed as planned and that the mission returned high-quality data. When the aurora lights up the night sky, it is powered by electrons streaming down from space into Earth's upper atmosphere. These charged particles energize atmospheric gases, causing them to glow. The incoming particle beams are relatively focused, like current flowing through a cord. After igniting the aurora, electrons spread out in many directions. Eventually, they make their way back to space, but only after weaving through the constantly changing upper atmosphere. "We're not just interested in where the rocket flies," said Kristina Lynch, principal investigator for GNEISS and a professor at Dartmouth College in New Hampshire. Using two rockets and a coordinated network of ground receivers, the mission builds a three-dimensional picture of the aurora's electrical environment. Once inside, each rocket released four subpayloads to take measurements at multiple points within the glowing region. The plasma changed those signals as they passed through it, much like body tissues alter X rays during a medical CT scan. By analyzing those changes, scientists can determine plasma density and identify where electrical currents are able to flow. Understanding these electrical currents is not just about solving a physics puzzle. Auroral currents control how energy from space is distributed through Earth's upper atmosphere. When currents spread out, they heat the atmosphere, stir up winds, and create turbulence that can affect satellites traveling through that region. Researchers have long relied on ground-based instruments to study auroras. NASA's EZIE satellite mission, launched in March 2025, measures auroral electrical currents from orbit. By combining satellite observations, ground imagery, and direct measurements from sounding rockets, scientists can examine the system from multiple angles at once. The GNEISS rockets were not alone during this launch campaign. These blank spots may mark areas where electrical currents suddenly reverse direction. The mission marked its second attempt at flight after a 2025 effort was postponed due to weather and scientific conditions. With this successful launch, researchers now have new data to examine how these mysterious dark patches fit into the broader auroral circuit. Electric currents, streams of charged particles, and countless collisions combine to create these glowing displays. Sounding rockets provide a rare opportunity to fly directly through them, placing instruments exactly where the action unfolds. Through brief but precisely timed missions, NASA is turning fleeting flashes of light into deeper insight about how space weather shapes our planet's upper atmosphere. Note: Content may be edited for style and length. Scientists Say Your Fingers Hold a Secret of Brain Evolution Scientists Discover How To “Switch Off” Cancer Genes for Good This Simple Exercise Habit May Keep Your Brain Younger Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225600.htm'>One stem cell generates 14 million tumor-killing NK cells in major cancer breakthrough</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-18 03:33:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers in China have created a more efficient strategy for producing natural killer (NK) cells for use in cancer immunotherapy. NK cells play a critical role in the body's early defense against viruses and cancer, along with other immune functions. Because of their natural ability to detect and destroy abnormal cells, they are an attractive tool for cancer treatment. This method presents several obstacles, including wide variability between cells, limited efficiency during genetic modification, high production costs, and lengthy preparation times. Earlier efforts to produce NK cells from cord blood-derived CD34+ HSPCs struggled with low efficiency and immature cell function. This strategy combined CAR transduction, strong expansion of progenitor cells, and guided commitment to the NK lineage. Next, the expanded cells were cultured with OP9 feeder cells to create artificial hematopoietic organoid aggregates, structures that support efficient NK lineage commitment and development. This process produced highly pure iNK or CAR-iNK cells that expressed endogenous CD16. Another major improvement was the sharp reduction in viral vector needed for CAR engineering. Compared with the amount usually required to modify mature NK cells, this method used only about ~1/140,000 (by Day 42 of culture) to ~1/600,000 (by Day 49) as much viral vector. In laboratory testing, both iNK and CAR-iNK cells demonstrated powerful tumor-killing ability. In cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) mouse models of human B-cell acute lymphoblastic leukemia (B-ALL), CD19 CAR-iNK cells reduced tumor growth and extended the animals' survival. Note: Content may be edited for style and length. Scientists Say Your Fingers Hold a Secret of Brain Evolution Scientists Discover How To “Switch Off” Cancer Genes for Good This Simple Exercise Habit May Keep Your Brain Younger Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            