
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/what-is-chronic-venous-insufficiency-trumps-blood-vessel-condition/'>What Is Chronic Venous Insufficiency, Trump's Blood Vessel Condition?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 21:18:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>After photographs showed President Donald Trump with swollen ankles and bruised hands, the White House revealed he has chronic venous insufficiency—a blood vessel disease that affects circulation in the legs Editor's Note (7/17/25): This story has been updated with additional reporting and expert commentary. White House Press Secretary Karoline Leavitt disclosed the diagnosis during a press conference on July 17 in response to public concern raised by photographs of the president with significant swelling around his ankles and bruises on his hands. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. A July 17 letter from Sean Barbabella, the president's physician, notes that the Trump underwent several diagnostic tests, including bloodwork and an echocardiogram, which did not identify any signs of systemic illness or heart failure. Chronic venous insufficiency is “a very, very common thing, and for [President Trump's] age it's totally understandable that he has it,” says Monara Dini, a treating physician at the University of California, San Francisco Center for Limb Preservation. Chronic venous insufficiency—a subset of a larger category of conditions called venous disorders—predominantly affects the legs and causes only pain rather than more serious harm or systemic issues. The condition is quite common—affecting perhaps 5 percent of U.S. adults, according to Cleveland Clinic—and the risk of developing it increases as people age. Chronic venous insufficiency is characterized by weakened valves in leg veins allowing blood to flow backward and pool under the force of gravity, according to Johns Hopkins Medicine. These valves leak more often as people age, Dini says. “The worst outcome of having this condition is ulcers that can develop,” Dini says. “The skin is retaining so much fluid that it can at some point burst and break the skin, and you develop ulcerations. Makeup covers a bruise on the back of Trump's hand as he hosts Macron for meetings at the White House on February 24, 2025. In his letter about Trump, Barbabella attributes this to “minor soft tissue irritation from frequent handshaking and the use of aspirin, which is taken as part of a standard cardiovascular prevention regimen.” Other risk factors can include smoking and insufficient exercise. Typically, management of chronic venous insufficiency relies on keeping the legs elevated, increasing exercise and reducing weight. In certain cases, doctors may recommend minor surgical interventions to either repair or remove damaged tissue. Chronic venous insufficiency is a progressive condition that cannot be healed or reversed, however. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65426000/lost-site-george-washington-friendly-fire/'>Archaeologists Found the Lost Site Where George Washington's Men Accidentally Shot Each Other</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Today, George Washington is remembered as one of America's greatest military leaders. But decades before the American Revolution, when Washington was just a 26-year-old colonel in the Virginia Regiment, his military career was jeopardized by one big mistake. That an instance of friendly fire occurred under Washington's supervision shouldn't impugn his leadership abilities too greatly in our minds today. Serving under General John Forbes, troops led by Colonel George Mercer were sent to try and thwart a raid on what is now known as Fort Ligonier. Though the friendly fire incident claimed 40 lives, and Washington in his own account would claim he was “never was in more imminent danger by being between two fires, knocking up with [my] sword the presented pieces,” it did yield a positive result for the British forces: French prisoners were able to provide information that Fort Duquesne was not well defended, and General Forbes was able to seize the opportunity and strike it. During the four years of fieldwork that transpired at the site (which is roughly two miles away from the fort), the team recovered “over 350 artifacts” ranging from musket balls and uniform buttons to a French officer's pocket watch key “adorned with a fleur-de-lis.” Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. His stories for the Enthusiast Group have involved coordinating with organizations like the National Parks Service and the Secret Service, and travelling to notable historical sites and archaeological digs, from excavations of America' earliest colonies to the former homes of Edgar Allan Poe. Scientists Found Life After Death—as a Mysterious Third State Humans Could Develop a Sixth Sense, Scientists Say Is This How to Unlock Human Intelligence in AI? Just Before Death, the Brain May Try to Stay Alive Your Nightmares Might Slowly Be Scaring You to Death Archaeologists Are Saving Ruins of a World Wonder</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65438356/restoring-eyesight-breakthrough/'>Scientists Found the Staggering Natural Switch That Could Bring Back Your Eyesight</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 16:32:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Millions of people live with vision loss that we thought was permanent. Researchers in South Korea found a protein, PROX1, that acts sort of like a molecular gatekeeper, suppressing your body's natural ability to regrow damaged retinal cells. When the scientists blocked PROX1 in mice, something incredible happened: they regenerated essential vision cells for a whopping six months. Watch Pop Mech editors Andrew Daniels and Manasee Wagh discuss the new discovery that could reshape how we treat blindness. Andrew and Manasee break down why this is such a groundbreaking moment in regenerative medicine, how retinal cells are basically extensions of your brain, and what this new study could mean for millions of people (and Andrew's dog) suffering from degenerative eye diseases. They also explore other mind-blowing methods for restoring eyesight, including one that uses gold nanoparticles and lasers—yes, really. The Third State of Life That Begins After Death One Single Ancestor Gave Rise to All Life on Earth We're Already Fighting the World's First AI War How to Pick the Right Motor Oil for Your Car</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250717013857.htm'>Cognitive collapse and the nuclear codes: When leaders lose control</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 14:37:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Many former leaders of the world's nine nuclear-armed nations were impaired by health conditions while in office, raising concerns over their decision-making abilities while they had access to nuclear weapon launch codes, a study from the University of Otago, New Zealand, has found. Professor Wilson says that of the leaders who left office while still alive, 15 had confirmed or possible health issues which likely hastened their departure. In some cases, the degree of impairment was profound, such as in the case of two former Israeli Prime Ministers: Ariel Sharon, who became comatose after suffering a stroke in office, and Menachem Begin, whose depression was so severe he spent his last year as leader isolated in his home. Impairment during crises was also seen in the case of Richard Nixon's bouts of heavy drinking - including during a nuclear crisis involving the Middle East. This was the case for multiple US presidents, including Dwight D Eisenhower, whose doctor described his 1955 heart attack as a digestive upset; John F Kennedy, whose aides lied about him having Addison's disease, a serious, chronic condition; and Ronald Reagan, whose administration hid the extent of his injuries after he was shot in 1981, and the likely signs of his dementia near the end of his term. Professor Wilson says Kennedy was in poor health during his first two years in office in 1961 and 1962, with his performance likely impaired from Addison's disease, back pain, and his use of anabolic steroids and amphetamines. It was in 1961 that he authorized the failed CIA-backed Bay of Pigs invasion of Cuba and that his poor performance at a Cold War summit with Soviet leader Nikita Khrushchev in Vienna was noted. This latest study follows previous research involving Professor Wilson on the health of former New Zealand Prime Ministers. He says there are a range of measures which could reduce global security risks from leaders whose judgement is in question. They include removing nuclear weapons from 'high alert' status, adopting 'no first use' policies where nations refrain from using nuclear weapons except as a retaliatory second strike, ensuring any weapon launches need authorization by multiple people, and progressing nuclear disarmament treaties. Professor Wilson says democracies could consider introducing term limits for their leaders, as well as recall systems, so voters could petition for politicians to step down. "Maintaining a strong media with investigative journalists can also help expose impairment in leaders." Professor Wilson says politicians in general are exposed to high levels of stress, which can affect their mental well-being. A study of UK Members of Parliament has found they were 34 per cent more likely to experience mental health problems than other high-income earners. "Finding ways to reduce stress on politicians and better address their mental health needs is another way global security risks can be reduced." Earth Could Be in a Massive Cosmic Bubble That's Warping the Universe NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65436974/pyramids-anomaly-portal/'>Archaeologists Found an ‘Anomaly' Near the Pyramids That May Reveal an Ancient Portal</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 14:17:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New findings beneath the desert floor hint at entrances to long-lost chambers. This geophysical technique uses radar pulses to image the subsurface. Similar techniques have uncovered Viking longships in Norway, revealed lost civilizations in the Amazonian jungle, and even entire Roman cities without ever putting a shovel into dirt. Researchers led by Tohoku University's Motoyuki Sato used GPR—along with a method known as electrical resistivity tomography (ERT), which uses electrical resistance to map underground structures—to discover what's being described as an “L-shaped anomaly” in the western cemetery near the world-renowned pyramids. Below this L-shaped structure was an anomaly, lying 16 to 33 feet down, that the researchers described as “highly electrically resistive.” Such an anomaly could have a few explanations, but the team identified two main possibilities—a mixture of sand and gravel, or “sparse spacing with air voids.” While we know that the surrounding area (built roughly 4,500 years ago, around the same time as its adjacent pyramids) is filled with flat-roofed tombs known in Arabic as mastaba, the stretch of sand where the anomaly was found has not been nearly as intensely excavated, largely because the area sported no impressive structures to warrant a thorough investigation. So, what exactly could this L-shape structure and its lower anomaly represent? That deeper structure sounds suspiciously like a tomb. Archaeologists Found a Lost Site of Friendly Fire Knight Found Buried Under Former Ice Cream Parlor Archaeologists May Have Found a Maya King's Tomb Someone Just Found a ‘Holy Grail' Scroll From 1847</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/new-interstellar-object-3i-atlass-biggest-mysteries-explained/'>New Interstellar Object Is Full of Mysteries. Here's What Scientists Most Want to Learn</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Future observations should reveal much more about this mysterious object, the third known visitor from beyond our solar system. Hawaii) Image Processing: Jen Miller & Mahdi Zamani (NSF NOIRLab) (CC BY 4.0) Earlier this month astronomers were thrilled to discover only the third known interstellar object ever seen in our solar system. Now dubbed 3I/ATLAS, the suspected comet has just zoomed past the orbit of Jupiter, traveling so fast that it's bound to slip through our sun's gravitational grip. The high speed and hyperbolic trajectory of 3I/ATLAS means it must have come from another star and was cast adrift in the Milky Way by some unknown process before it eventually, by chance, briefly swooped by our sun. It will reach about the orbit of Mars before it boomerangs back toward interstellar space, never to be seen again, at the end of this year. “I didn't get any sleep for like 35 hours,” says Bryce Bolin of Eureka Scientific in California, who rushed to release a preprint paper and arrange additional observations following 3I/ATLAS's discovery. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. Stefanie Milam of NASA's Goddard Space Flight Center is part of a group that had reserved time on JWST to observe an interstellar object—if the researchers were fortunate enough for one to be discovered. “When he finally got back, his phone just blew up. I said, ‘You're never allowed to go on vacation again! The first major question to answer about 3I/ATLAS is its origin. Tracing it back to an individual star is likely impossible, given the mixing of myriad stars in their orbits around our galaxy across billions of years. But we might be able to work out roughly the region it came from. One team of astronomers has already begun doing just that, using the high velocity of the object with respect to our sun—60 kilometers (37 miles) a second—to argue that it might have come from the vicinity of our galaxy's thick disk. This is a puffy torus of older stars moving at high velocities above and below the main flat plane of the Milky Way—which is where our sun serenely orbits. A thick-disk origin might mean that 3I/ATLAS is extremely ancient, more than eight billion years old. “It's from a star that's potentially not even there anymore,” says Michele Bannister of the University of Canterbury in New Zealand, a co-author on the work. Such estimates might soon be revised if subsequent observations can show just how much space weathering the object has endured during its interstellar sojourn. This image shows the observation of interstellar object 3I/ATLAS when it was discovered on July 1, 2025. Although early observations have led astronomers to categorize 3I/ATLAS as a comet, at the moment, it's not behaving exactly like one. The object doesn't display a large tail or enveloping coma of cast-off gas, only a hint of dust—but that is expected to change soon. As it traverses the asteroid belt between Mars and Jupiter and basks in the sun's radiance, its surface should warm enough to sublimate ice, venting sufficient material to form a large coma and perhaps a prominent tail. A substantial coma would be like a curtain drawn over astronomers' eyes, obscuring their view of the object and complicating efforts to gauge its dimensions. Before that happens, a team led by David Jewitt at the University of California, Los Angeles, is hoping to pin down its size with Hubble in August. (Other telescopes might be able to determine the size of 3I/ATLAS, too.) Initial estimates suggested 3I/ATLAS might be up 20 kilometers (12 miles) across—very big for a comet—but most astronomers now think it is much smaller. If 3I/ATLAS turns out to be much bigger, 10 kilometers (six miles) or more, this would pose problems for preexisting estimates of how many big interstellar objects reside in the galaxy. “It's statistically extremely unlikely we should ever see something that size,” Noonan says. But as an observer, I would love to see a really weird, big object.” “Certain ways of kicking these objects out tend to make them spin up,” Taylor says. A close pass of a gas giant planet, for instance, could easily set the object twirling while hurling it away from its home star. Those objects become unbound and just flow out into the galaxy.” The rotation period can also tell us more about the shape of 3I/ATLAS—a steady rotation suggests a fairly spherical form, whereas a fluctuating rotation speed might suggest a “wonky shape,” Taylor says, like that of ‘Oumuamua, which was estimated to be cigar- or pancake-shaped. If 3I/ATLAS really is an ancient cometary castaway that has been drifting through the galaxy for eons, it might be full of ice that has never been heated by a star. If so, then as it gets closer, the object might suddenly erupt into activity. While that could be bad news for measuring its size, it would aid efforts to determine 3I/ATLAS's chemical composition. In November, post-perihelion, Noonan will use Hubble to study 3I/ATLAS and its emissions, looking for signs of substances such as hydroxide and hydrogen that can help clarify its composition. The image is composed of exposures taken through three filters, shown here as red, green and blue. Hawaii) Image Processing: Jen Miller & Mahdi Zamani (NSF NOIRLab) (CC BY 4.0) If the object is several billion years old, as predicted, then it might be rich in water because of the suspected formation environment around older stars. Thanks to its keen infrared vision, JWST is better suited for teasing out the presence of molecules such as water, carbon monoxide, carbon dioxide and ammonia. “We can really home in and see what this thing looks like,” she says. “Borisov had a pretty boring chemistry, but it wasn't like any object in our solar system—there was hardly any water at all but a lot of carbon monoxide and hydrogen cyanide. An unlikely additional capability will be at Mars, where spacecraft such as NASA's Mars Atmosphere and Volatile Evolution (MAVEN) orbiter may be able to see 3I/ATLAS as it passes about 30 million kilometers (19 million miles) from the planet. “They'll be able to see the coma,” giving us an insight into 3I/ATLAS's activity near the sun that would otherwise be impossible to see from Earth. A big unknown about 3I/ATLAS is whether it will actually survive its close encounter with our sun. While ‘Oumuamua did so, Comet Borisov was not so fortunate, with the object appearing to split and break apart on its way out of our solar system. “Borisov fragmented, which is pretty usual for comets,” Bannister says. All eyes will be on our latest visitor to see if the same thing happens again. By chance, the object is entering our solar system at quite a shallow angle, much flatter than that of most comets and thus much closer to Earth's orbital plane, allowing careful comparisons between the solar wind in both places. Yet an upcoming European Space Agency (ESA) mission called Comet Interceptor, set to launch in 2029, might attempt to visit another interstellar object, if we find one within its reach. “It is possible we could get an interstellar object, but we have to be really lucky,” says Colin Snodgrass, an astronomer at the University of Edinburgh, who is deputy lead on the mission. One of our biggest outstanding questions about interstellar objects concerns their unknown abundance. The object 3I/ATLAS is our third interstellar visitor in eight years—a real but weak hint of how many are out there, waiting to be found. Predictions estimate there are trillions upon trillions of interstellar objects drifting around our galaxy, and perhaps one in our solar system at any given time—but they're typically just so faint that they're unlikely to be found by most telescopes. This is expected to change when a new telescope called the Vera C. Rubin Observatory begins a 10-year survey of the sky later this year. Rubin is expected to see somewhere between six and 51 interstellar objects in its 10-year survey. Seeing such a population will tell us “how unique, or varied, planetesimal formation is across different parts of the galaxy,” Bannister says, referring to kilometer-scale objects thought to coalesce around newborn stars that become the feedstock for planets—and, when kicked to a system's hinterlands, become a reservoir of comets. One puzzling question is why we haven't seen much smaller interstellar objects, Moskowitz says. If smaller objects are more plentiful than larger objects, as scientists expect, then we should have seen some small interstellar objects entering our atmosphere, appearing as meteors streaking across Earth's skies at speeds and trajectories that clearly convey their interstellar origins. Detections of such objects have been claimed, but the evidence behind them has failed to convince most experts. “I think that's going to be one of the major questions: Why are we seeing these big cometlike things coming through the solar system, but we're not seeing things that are smaller? Jonathan O'Callaghan is an award-winning freelance journalist covering astronomy, astrophysics, commercial spaceflight and space exploration.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a65417916/black-goo-new-species/'>A Marine Researcher Found Black Goo Inside a Ship. Turns Out It's a New Lifeform.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Typically in Hollywood movies—no matter the genre—mysterious black goo isn't a good thing. Basically the entire 2018 Marvel film Venom is about evil black goo. The trope is everywhere, and it almost always portends ill fortune. Luckily, when Doug Ricketts—Marine Superintendent at the Large Lake Observatory (LLO)—found a mysterious black goo tucked away in the R/V Blue Heron's rudder shaft during some routine maintenance, he disregarded these cultural warnings and brought a cup of the substance to researchers at the University of Minnesota Duluth (UMD). “The biggest surprise was that the ship goo had life in it at all,” Sheik said in a press statement. But surprisingly, we found DNA and it wasn't too destroyed, nor was the biomass too low.” It turns out that the rudder shaft of the Blue Heron provided the perfect habitat for a particular kind of archaea that prefers a semi-warm, stable environment with no oxygen. They then compared those genomes to a database of previously identified organisms, and according to Popular Science, one of the species represents an entire new order of archaea, while another might represent a new bacterial phylum. This isn't just a novel discovery. Some of the organisms appear to produce methane, which could be used in the production of biofuels. Sheik even mentions that he's explored extreme environments like hydrothermal vents or hot springs, but not so much ship rudder housings and other artificial structures. “ShipGoo001” is also a living, anaerobic testament to what can happen when scientists are allowed to explore their inherent curiosities. “Scientists don't often have time to be playful—we're focused and have projects to complete,” Sheik said in a press statement. “Time and resources for exploratory work can be daunting. These 4 New ‘Dragon Pseudoscorpions' Are So Metal This Giant Bird Went Extinct 600 Years Ago. How the Dogs of Chernobyl Kept Evolving Our Body's Blueprint Can Be Found in... Anemones? A Colossal Squid Might Have Shown Its Secret Face The Rodents of Chicago Are Evolving in Real Time</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65416985/grow-neurons-adult-brain/'>Scientists Found a Cell in the Adult Brain That Was Only Supposed to Exist in Childhood</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It's well-understood that brain plasticity in children—including a process known as “neurogenesis,” which describes the creation of new neurons—is significantly greater than adults. And it doesn't take too much imagination to understand why evolution took this course. After all, children need to learn a lot of things really quickly, so forming new neurons (particularly in the hippocampus where learning, memory, and emotional regulation are located) makes a lot of sense. However, scientists have long wondered whether neurogenesis is an exclusive feature of growing up, or if adults continue to generate new neurons as well (albeit at lower rates). In 2013, a team of scientists led by the Karolinska Institute in Stockholm, Sweden, found evidence of neurogenesis in the hippocampus, and other studies discovered the presence of stem cells and immature neurons. Identifying these well-hidden cells was far from a simple process. Frisén and his team first created a machine learning algorithm that could accurately identify neural progenitor cells by using the hippocampus samples from the brains of children donated for research after death. Because these cells look remarkably similar to progenitor cells found in mice, the team developed an AI that can spot the “molecular fingerprints of childhood progenitors and use that to identify these cells in adults,” Frisén told New Scientist. To help things along, they used an antibody that identified dividing cells at the time of death, which helped to narrow things down by excluding mature neural cells. Once the samples had been run through their AI models, the team found that nine of those samples contained progenitor cells. Because they weren't universally present, and concentrations of the cells varied between individuals, scientists theorized that the presence of these cells in adults could be influenced by genetics or environment. “Our research may also have implications for the development of regenerative treatments that stimulate neurogenesis in neurodegenerative and psychiatric disorders,” Frisén said in a press statement. Turns out it's never too late to learn something new—our biology tells us so. Just Before Death, the Brain May Try to Stay Alive Magic Mushrooms Might Be the Key to Longer Lives Your Nightmares Might Slowly Be Scaring You to Death</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02264-9'>‘Controlled chaos': how to create an environment that fosters genius</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 11:26:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). On a Monday afternoon in April, the Szkocka Restaurant and Bar in Lviv, Ukraine, seems empty, except for two waiters, a maintenance worker on a ladder and the ghost of Frank Sinatra, whose song asks us to hold him in our arms. But it was in this restaurant — known in the 1930s as the Scottish Cafe — that Ulam, along with other prominent mathematical figures such as Stefan Banach and Hugo Steinhaus, sat for hours, bandying about ideas. They would write their calculations in pencil on the marble tabletops. But Banach's wife, Łucja Braus, dismayed that the work was getting wiped away by waiters, bought them a present: a lined notebook that would eventually be filled with mathematical challenges. In its pages are workings that led Ulam to later co-author a paper elaborating on the ‘hairy ball theorem', which shows that, on any sphere in n dimensions, if there is a hair-like substance or current on it that is ‘combed' in one direction, there will always be a place where the directional combing cannot continue — a cowlick, in other words. Far from being a barbering quirk, the theorem has had applications in fields such as meteorology and fusion energy. Since then, universities, cities and companies have been trying, with varying success, to replicate this environment of a loose, productive exchange of ideas. When I returned to the cafe the next day, Iryna Banakh (no relation), a mathematician at the Ivan Franko National University of Lviv, a few blocks away, told me a little bit about what went right. There, over a glass of beer, he would think about his problems.” If this emphasis on controlled chaos, social interaction and free-wheeling exploration sounds familiar, it's because it is the basis for everything from child-led learning at Montessori schools to some courses at the University of Kansas in Lawrence, in which unconventional lecture-delivery models have been implemented. But this is not necessarily a recipe for success in everyone's hands. Although Ulam had wonderful things to say about environments full of genial geniuses in his autobiography, Adventures of a Mathematician (1976), he was scathing about the 1930s mathematical community at Harvard University, in Cambridge, Massachusetts. He found it full of overdone formality, bloated egos and people performing the act of work, rather than working productively and informally. There are some steps to nudge an academic community towards this type of collaboration. Richard Florida's 2002 urban-construction manual, Rise of the Creative Class, says that well-built communities — such as the walkable square around the Scottish Cafe — are more likely to provide people with places to exchange ideas than are automotive exurbs that encourage isolation. US college towns are increasingly working to replicate the pedestrian-accessible cafe environments found in Lviv and other European cities, such as Vienna and Prague. And some large institutions in small US cities, such as the University of Utah in Salt Lake City and the University of Maryland, College Park, have been encouraging the construction of tram and light-rail systems, viewing them as providing a competitive advantage. Even in modern-day Lviv, which is already vibrant, there have been moves — in the middle of war, no less — to enhance that vibrancy with even better urban-rail infrastructure. However, cities need to be positive, liveable places for everyone, not just those who can afford the latest Macbook. Those have moved to a cheap pizza place elsewhere in the city, among other places, with Szkocka filling a more ceremonial niche for formal dinners in recent years. AI and misinformation are supercharging the risk of nuclear war Inventions that made the United States a powerhouse of innovation The breakthrough proof bringing mathematics closer to a grand unified theory This AI ‘thinks' like a human — after training on 160 psychology studies How to avoid nuclear war in an era of AI and misinformation AI and misinformation are supercharging the risk of nuclear war The breakthrough proof bringing mathematics closer to a grand unified theory Interested applicants can contact with the relevant department/school and submit their CV directly. State Key Laboratory of Materials-Oriented Chemical Engineering, Nanjing Tech University An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41929-025-01370-1'>Controlling hydrocarbon chain growth and degree of branching in CO</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 09:53:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Catalysis (2025)Cite this article Nickel-based materials can facilitate the electrocatalytic CO2 reduction (CO2R) reaction to generate hydrocarbons up to C6. Here we show that fluorine doping alters the nature of the Ni active sites, which proves instrumental in tuning the selectivity of the CO2R. We interrogate the CO2R reaction mechanism using intermediate surrogates, including aldehydes, alkyl iodides and acetylene. Aldehydes are electroreduced to alcohols and deoxygenated intermediates. Among the latter, unsaturated hydrocarbon intermediates (RCH2−x*, where the asterisk represents surface-bound species and x = 1 or 2) reacting with *CO dictate chain propagation, modulated by competitive C–C coupling and C–H hydrogenation reactions. Compound branching in the hydrocarbons initiates from *CO coupling with two *CH2 species, and the branch-to-linear hydrocarbon ratio can be doubled using a pulsed potential strategy. An inverse H/D kinetic isotope effect promotes deuterated hydrocarbon formation with a Faradaic efficiency of 22.2%. This work reveals mechanisms and strategies for the conversion of CO2 into linear and branched hydrocarbons, thus advancing electrosynthetic fuel development. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data supporting the results of this study are available within the Article and Supplementary Information and via Zenodo at https://doi.org/10.5281/zenodo.15300306 (ref. All of the DFT simulations are available via ioChem-BD76 at https://doi.org/10.19061/iochem-bd-1-326. Ozden, A. et al. Carbon-efficient carbon dioxide electrolysers. Google Scholar Jordaan, S. M. & Wang, C. Electrocatalytic conversion of carbon dioxide for the Paris goals. Google Scholar Weekes, D. M., Salvatore, D. A., Reyes, A., Huang, A. & Berlinguette, C. P. Electrolytic CO2 reduction in a flow cell. Google Scholar Higgins, D., Hahn, C., Xiang, C., Jaramillo, T. F. & Weber, A. Z. Gas-diffusion electrodes for carbon dioxide reduction: a new paradigm. ACS Energy Lett. Google Scholar Romiluyi, O., Danilovic, N., Bell, A. T. & Weber, A. Z. Membrane‐electrode assembly design parameters for optimal CO2 reduction. Google Scholar Lees, E. W., Mowbray, B. A. W., Parlane, F. G. L. & Berlinguette, C. P. Gas diffusion electrodes and membranes for CO2 reduction electrolysers. Google Scholar Ge, L. et al. Electrochemical CO2 reduction in membrane-electrode assemblies. Google Scholar Esmaeilirad, M. et al. Imidazolium-functionalized Mo3P nanoparticles with an ionomer coating for electrocatalytic reduction of CO2 to propane. Google Scholar Ding, J. et al. A tin-based tandem electrocatalyst for CO2 reduction to ethanol with 80% selectivity. Google Scholar Nitopi, S. et al. Progress and perspectives of electrochemical CO2 reduction on copper in aqueous electrolyte. Google Scholar Wang, X. et al. Mechanistic reaction pathways of enhanced ethylene yields during electroreduction of CO2–CO co-feeds on Cu and Cu-tandem electrocatalysts. Google Scholar Kwon, S., Zhang, J., Ganganahalli, R., Verma, S. & Yeo, B. S. Enhanced carbon monoxide electroreduction to >1 A cm−2 C2+ products using copper catalysts dispersed on MgAl layered double hydroxide nanosheet house-of-cards scaffolds. Google Scholar Timoshenko, J. et al. Steering the structure and selectivity of CO2 electroreduction catalysts by potential pulses. Google Scholar Xia, W. et al. Adjacent copper single atoms promote C–C coupling in electrochemical CO2 reduction for the efficient conversion of ethanol. Google Scholar Kim, Y. J. et al. Unlocking long-chain hydrocarbons (C2–7) via direct electrochemical CO2 and CO reduction on balanced Au/Ni electrodes. Google Scholar Preikschas, P., Martin, A. J., Yeo, B. S. & Perez-Ramirez, J. NMR-based quantification of liquid products in CO2 electroreduction on phosphate-derived nickel catalysts. Google Scholar Zhou, Y. et al. Long-chain hydrocarbons by CO2 electroreduction using polarized nickel catalysts. Google Scholar Vos, R. E. & Koper, M. T. M. Nickel as electrocatalyst for CO(2) reduction: effect of temperature, potential, partial pressure, and electrolyte composition. Google Scholar Lin, T. et al. Advances in selectivity control for Fischer–Tropsch synthesis to fuels and chemicals with high carbon efficiency. Google Scholar Friedel, R. A. & Anderson, R. B. Composition of synthetic liquid fuels. Product distribution and analysis of C5–C8 paraffin isomers from cobalt catalyst. Google Scholar Schulz, H. & Claeys, M. Kinetic modelling of Fischer–Tropsch product distributions. Google Scholar Yang, J. H. et al. Opening direct electrochemical Fischer–Tropsch synthesis path by interfacial engineering of Cu electrode with p-block elements. ACS Appl. Google Scholar Kortlever, R. et al. Palladium–gold catalyst for the electrochemical reduction of CO2 to C1–C5 hydrocarbons. Google Scholar Xiao, H., Cheng, T. & Goddard, W. A. III Atomistic mechanisms underlying selectivities in C1 and C2 products from electrochemical reduction of CO on Cu(111). Google Scholar Liu, X. et al. pH effects on the electrochemical reduction of CO(2) towards C2 products on stepped copper. Google Scholar Ding, Y. et al. Enhanced formation of multi-branched isoparaffins in syngas conversion by ZnCrOx-MCM-22 composites. Google Scholar Demirbas, A., Balubaid, M. A., Basahel, A. M., Ahmad, W. & Sheikh, M. H. Octane rating of gasoline and octane booster additives. Google Scholar Ali, S. A. et al. Molecular-level kinetic modeling of catalytic naphtha reforming. Google Scholar Aitani, A. et al. Catalytic upgrading of light naphtha to gasoline blending components: a mini review. Google Scholar Hussain, N. et al. Ultrathin mesoporous F-doped α-Ni(OH)2 nanosheets as an efficient electrode material for water splitting and supercapacitors. Google Scholar Wang, T., Chen, H., Yang, Z., Liang, J. & Dai, S. High-entropy perovskite fluorides: a new platform for oxygen evolution catalysis. Google Scholar Tatara, R. et al. The effect of electrode-electrolyte interface on the electrochemical impedance spectra for positive electrode in Li-ion battery. Google Scholar Wygant, B. R. et al. The role of electrolyte composition in enabling Li metal-iron fluoride full-cell batteries. Google Scholar Friebel, D. et al. Identification of highly active Fe sites in (Ni,Fe)OOH for electrocatalytic water splitting. Google Scholar Ko, Y. J. et al. Exploring dopant effects in stannic oxide nanoparticles for CO2 electro-reduction to formate. Google Scholar Ma, W. et al. Electrocatalytic reduction of CO2 to ethylene and ethanol through hydrogen-assisted C–C coupling over fluorine-modified copper. Google Scholar Bertheussen, E. et al. Acetaldehyde as an intermediate in the electroreduction of carbon monoxide to ethanol on oxide-derived copper. Google Scholar Wang, L. et al. Selective reduction of CO to acetaldehyde with CuAg electrocatalysts. Google Scholar Ledezma-Yanez, I., Gallent, E. P., Koper, M. T. M. & Calle-Vallejo, F. Structure-sensitive electroreduction of acetaldehyde to ethanol on copper and its mechanistic implications for CO and CO2 reduction. Google Scholar Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Google Scholar Grimme, S., Ehrlich, S. & Goerigk, L. Effect of the damping function in dispersion corrected density functional theory. Google Scholar Almora-Barrios, N., Carchini, G., Blonski, P. & Lopez, N. Costless derivation of dispersion coefficients for metal surfaces. Theory Comput. Google Scholar Anisimov, V. V., Zaanen, J. & Anderson, O. K. Band theory and Mott insulators: Hubbard U instead of Stoner I. Phys. Google Scholar Smidstrup, S., Pedersen, A., Stokbro, K. & Jonsson, H. Improved initial guess for minimum energy path calculations. Google Scholar Cullity, B. D. & Stock, S. R. Elements of X-ray Diffraction 3rd edn (Prentice-Hall, 2001). Bagger, A., Ju, W., Varela, A. S., Strasser, P. & Rossmeisl, J. Electrochemical CO2 reduction: a classification problem. Google Scholar Mavrikakis, M., Hammer, B. & Nørskov, J. K. Effect of strain on the reactivity of metal surfaces. Google Scholar Li, J. et al. Intercepting elusive intermediates in Cu-mediated CO electrochemical reduction with alkyl species. Google Scholar Xu, B., Madix, R. J. & Friend, C. M. Alkyl groups as synthetic vehicles in gold-mediated oxidative coupling reactions. Google Scholar Wang, S. et al. Highly efficient ethylene production via electrocatalytic hydrogenation of acetylene under mild conditions. Google Scholar Xue, W. et al. Electrosynthesis of polymer-grade ethylene via acetylene semihydrogenation over undercoordinated Cu nanodots. Google Scholar Shi, R. et al. Room-temperature electrochemical acetylene reduction to ethylene with high conversion and selectivity. Google Scholar Teh, W. J. et al. Selective electroreduction of acetylene to 1,3-butadiene on iodide-induced Cuδ+–Cu0 sites. Google Scholar Cheng, T., Xiao, H. & Goddard, W. A. III Full atomistic reaction mechanism with kinetics for CO reduction on Cu(100) from ab initio molecular dynamics free-energy calculations at 298 K. Proc. Google Scholar Wu, Y. et al. Enhancing CO2 electroreduction to CH4 over Cu nanoparticles supported on N-doped carbon. Google Scholar Zhao, J. et al. Modulation of *CHxO adsorption to facilitate electrocatalytic reduction of CO2 to CH4 over Cu-based catalysts. Google Scholar Dunwell, M., Luc, W., Yan, Y., Jiao, F. & Xu, B. Understanding surface-mediated electrochemical reactions: CO2 reduction and beyond. Google Scholar Shao, F. et al. Surface water as an initial proton source for the electrochemical CO reduction reaction on copper surfaces. Google Scholar Pablo-García, S. et al. Mechanistic routes toward C3 products in copper-catalysed CO2 electroreduction. Google Scholar Kresse, G. & Hafner, J. Ab initio molecular dynamics for liquid metals. Google Scholar Kresse, G. & Hafner, J. Ab initio molecular-dynamics simulation of the liquid-metal–amorphous-semiconductor transition in germanium. Google Scholar Kresse, G. & Furthmuller, J. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Google Scholar Kresse, G. & Furthmüller, J. Efficiency of ab-initio total energy calculations for metals and semiconductors using a plane-wave basis set. Google Scholar Blochl, P. E. Projector augmented-wave method. Google Scholar Kresse, G. & Joubert, D. From ultrasoft pseudopotentials to the projector augmented-wave method. Google Scholar Tkalych, A. J., Yu, K. & Carter, E. A. Structural and electronic features of β-Ni(OH)2 and β-NiOOH from first principles. Google Scholar Hjorth Larsen, A. et al. The atomic simulation environment—a Python library for working with atoms. Sawant, K. J., Zeng, Z. & Greeley, J. P. Universal properties of metal-supported oxide films from linear scaling relationships: elucidation of mechanistic origins of strong metal–support interactions. Wang, V., Xu, N., Liu, J.-C., Tang, G. & Geng, W.-T. VASPKIT: a user-friendly interface facilitating high-throughput computing and analysis using VASP code. Neugebauer, J. & Scheffler, M. Adsorbate-substrate and adsorbate-adsorbate interactions of Na and K adlayers on Al(111). Nørskov, J. K. et al. Origin of the overpotential for oxygen reduction at a fuel-cell cathode. Mathew, K., Kolluru, V. S. C., Mula, S., Steinmann, S. N. & Hennig, R. G. Implicit self-consistent electrolyte model in plane-wave density-functional theory. Mathew, K., Sundararaman, R., Letchworth-Weaver, K., Arias, T. A. & Hennig, R. G. Implicit solvation model for density-functional study of nanocrystal surfaces and reaction pathways. Henkelman, G., Uberuaga, B. P. & Jónsson, H. A climbing image nudged elastic band method for finding saddle points and minimum energy paths. Ou, Y. et al. Controlling hydrocarbon chain growth and degree of branching in CO2 electroreduction on fluorine-doped nickel catalysts. Alvarez-Moreno, M. et al. Managing the computational chemistry big data problem: The ioChem-BD platform. Download references This work was supported by the National Research Foundation of Singapore (Urban Solutions and Sustainability, Industry Alignment Fund (Pre-Positioning) Programme, A-0004543-00-00) and the Ministry of Education, Singapore (A-8001571-00-00). acknowledges funding from the European Union's Horizon 2020 research and innovation programme under Marie Skłodowska-Curie grant agreement no. thank the Spanish Ministry of Science and Innovation (PID2021-122516OBI00) for financial support. J.P.-R. acknowledges NCCR Catalysis (grant no. 225147) funded by the Swiss National Science Foundation. We acknowledge the EuroHPC Joint Undertaking for awarding us access to MareNostrum5 at the Barcelona Supercomputing Center, Spain. The Barcelona Supercomputing Center (BSC-RES) is further acknowledged for providing generous computational resources and technical support. We thank S. Xi from Singapore Synchrotron Light Source (SSLS) for assisting with X-ray spectroscopy measurements. We thank P. Sanz Berman for his valuable assistance in preparing the Table of Contents graphic and W. J. Teh from the National University of Singapore (NUS) for his feedback in this project. These authors contributed equally: Yingqing Ou, Lu Liu. Department of Chemistry, Faculty of Science, National University of Singapore, Singapore, Singapore Yingqing Ou, Futian You, Haibin Ma & Boon Siang Yeo Institute of Sustainability for Chemicals, Energy and Environment (ISCE2), Agency for Science Technology and Research in Singapore (A*STAR), Singapore, Singapore Lu Liu Institute of Chemical Research of Catalonia (ICIQ-CERCA), The Barcelona Institute of Science and Technology (BIST), Tarragona, Spain Ranga Rohit Seemakurthi & Núria López Institute for Chemical and Bioengineering, Department of Chemistry and Applied Biosciences, ETH Zürich, Zürich, Switzerland NCCR Catalysis, Zürich, Switzerland Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the research. directed the research project. designed most of the experiments and analysed the results. carried out the computational simulations. conducted the XAS-based experiments and analysed the results. assisted with the XPS, SEM and TEM characterizations. assisted with the synthesis of the catalysts. H.M. helped with the isotopic CO2 reduction experiments and analysed the data. wrote the paper. All authors discussed the experimental and simulation results and read and commented on the paper. Correspondence to Núria López or Boon Siang Yeo. The authors declare no competing interests. Nature Catalysis thanks Joel Ager III, Yingju Yang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Notes 1 and 2, Figs. 1–79, Tables 1–16 and refs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Ou, Y., Liu, L., Seemakurthi, R.R. et al. Controlling hydrocarbon chain growth and degree of branching in CO2 electroreduction on fluorine-doped nickel catalysts. Nat Catal  (2025). Download citation Received: 06 September 2024 DOI: https://doi.org/10.1038/s41929-025-01370-1 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Catalysis (Nat Catal) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61355-3'>AI-guided patient stratification improves outcomes and efficiency in the AMARANTH Alzheimer's Disease clinical trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 09:51:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Alzheimer's Disease (AD) drug discovery has been hampered by patient heterogeneity, and the lack of sensitive tools for precise stratification. Here, we demonstrate that our robust and interpretable AI-guided tool (predictive prognostic model, PPM) enhances precision in patient stratification, improving outcomes and decreasing sample size for a AD clinical trial. The AMARANTH trial of lanabecestat, a BACE1 inhibitor, was deemed futile, as treatment did not change cognitive outcomes, despite reducing β-amyloid. Employing the PPM, we re-stratify patients precisely using baseline data and demonstrate significant treatment effects; that is, 46% slowing of cognitive decline for slow progressive patients at earlier stages of neurodegeneration. Our results provide evidence for AI-guided patient stratification that is more precise than standard patient selection approaches (e.g. β-amyloid positivity) and has strong potential to enhance efficiency and efficacy of future AD trials. Dementia presents a major global healthcare challenge, affecting more than 55 million individuals around the world, with a projected threefold increase over the next 50 years1. Despite decades of research and development, clinical trials of potential disease-modifying treatments for dementia have been largely unsuccessful. Cumulative expenditure on clinical-stage AD research and development is estimated to have reached $42.5 billion since 19954,5, and the cost to develop a treatment for AD from the preclinical stage to FDA approval is estimated to be $5.7 billion6. Recruitment and patient selection from diverse and qualified pools of volunteers7,8 often cause significant delays and contribute to the high failure rates of these trials4,5. Recent positive phase three clinical trial results (i.e., lecanemab, donanemab)9,10 highlight the need for interventions earlier in the progression of disease when treatments may be maximally effective11,12 and have the potential to enhance patient outcomes. Yet, we still lack effective tools for precise stratification of patients at risk or early disease stages for inclusion in clinical trials. In particular, patient selection often relies on biomarkers (e.g., β-Amyloid) that are limited in predicting AD progression and treatment outcomes due to variability in amyloid binding and immune activation3,4,13,14. Further, up to a third of patients at early Mild Cognitive Impairment (MCI) stages may be misdiagnosed due to a lack of sensitive tools for early diagnosis. Including patients with symptoms due to comorbidities (e.g., anxiety or mood-related disorders) rather than dementia pathology in clinical trials may impact trial efficiency and costs (i.e., larger numbers of patients and longer recruitment are necessary), as well as confound efficacy signals due to increased heterogeneity in the patient sample. Further, including patients who have progressed to advanced disease stages may reduce the potential of targets (e.g., amyloid removal targets) to be effective3,15,16,17. Recent developments in Artificial Intelligence (AI) based on machine learning (ML) algorithms provide a turning point in precise patient stratification in early dementia stages. We have developed and validated a robust and interpretable predictive prognostic model (PPM) that extends beyond binary patient classification approaches and predicts not only whether but also how fast individuals at early stages of the disease (MCI) or even pre-symptomatic stages (Cognitive Normal, CN) may progress to AD18,19. PPM has been tested on independent real-world patient data from memory clinics and validated against longitudinal clinical outcomes20. PPM delivers an AI-guided marker of future cognitive health that predicts progression to AD more precisely than standard clinical assessments (i.e., cognitive data, MRI scan)20 and biomarkers typically used for patient inclusion in clinical trials (i.e., β-Amyloid positivity)19, offering potential to reduce misdiagnosis and optimize patient stratification. Here, we test whether employing the PPM to improve patient stratification may change the outcome and efficiency of a randomized phase 2/3 clinical trial (AMARANTH, NCT02245737). AMARANTH tested the effect of lanabecestat (AZD3293, LY3314814), a brain-permeable inhibitor of human beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1/β-secretase)21. BACE1 was considered to be a promising therapeutic target for slowing disease progression in AD by preventing the generation of Aβ peptides, reducing the effects of Aβ toxicity and the formation of amyloid plaques in the brain. The objective of the trial was to test the efficacy of lanabecestat in slowing cognitive decline in patients diagnosed with MCI due to AD or mild AD. Despite lanabecestat reducing β-amyloid, the readout of the trial was deemed unsuccessful, and the trial was terminated early due to a lack of significant changes in primary cognitive outcomes. In contrast to the futility assessment, we demonstrate significant treatment effects on primary trial outcomes. In particular, we test the effect of lanabecestat (20 mg vs. 50 mg) on outcomes (β-amyloid, cognition: CDR-SOB, ADAS-Cog13) separately for slow vs. rapid progressive individuals. Our results demonstrate 46% slowing of cognitive decline (as measured by CDR-SOB) for the slow rather than the rapid progressive group following treatment with lanabecestat 50 mg compared to placebo. That is, patients stratified by the PPM as slow progressive at earlier stages of neurodegeneration showed slowing of cognitive decline related to β-amyloid reduction due to treatment. Further, we show that using PPM for patient stratification substantially decreases the sample size necessary for identifying significant changes in cognitive outcomes. Our results suggest that using PPM to stratify patients for clinical trials has strong potential to enhance their efficiency (faster and cheaper) and efficacy (more reliable outcomes), as the right patients are included in the trials at the right time. PPM adopts a trajectory modeling approach based on Generalized Metric Learning Vector Quantization (GMLVQ)18,19 that leverages multimodal data to make predictions about future cognitive decline at early dementia stages by iteratively adjusting class-specific prototypes and learning class boundaries (Supplementary Material: Predictive prognostic model). GMLVQ incorporates a full metric tensor to provide a robust distance measure (metric) tuned to the classification task. We trained the PPM on baseline data from ADNI (n = 256) to discriminate Clinically Stable (n = 100) from Clinically Declining (n = 156) patients, using β-Amyloid, APOE4, and medial temporal lobe (MTL) GM density. Employing ensemble learning with cross-validation and majority voting showed 91.1% classification accuracy (0.94 AUC: Area Under Curve) with sensitivity of 87.5% and specificity of 94.2% (Supplementary Table S1; Precision: 93.8%, F1 score: 90.5%). Note that discriminating Clinically Stable vs. declining individuals comprises a finer classification task compared to previous work focusing on patient (AD) vs. healthy (cognitive normal) classifications22,23, where signals are more discriminable and may result in higher model performance. Thus, achieving higher than 90% accuracy and precision provides evidence for PPM robustness. Further, the PPM architecture is transparent and interpretable. In particular, the metric tensors indicate that β-amyloid burden is the most discriminative feature compared to MTL GM density and APOE4. This is consistent with the role of β-amyloid and MTL atrophy as markers of Alzheimer's pathology, consistent with the NIA-AA 2018 diagnostic framework of AD24 A PPM metric tensor based on training the PPM on MTL Gray Matter (GM) Density, β-Amyloid, and APOE 4 for model training. The color scale represents values for each cell in the metric tensor, with diagonal terms summing to 1. The diagonal terms show strong contribution of β-amyloid burden (weight: 0. 51) compared to Gray matter density (weight: 0. B PPM-derived prognostic index for an independent ADNI validation dataset (n = 419): Box plots of PPM-derived prognostic index showing significant differences between Cognitive Normal: CN, Mild Cognitive Impairment: MCI, Alzheimer's Disease: AD (Kruskal-Wallis H two-sided tests, p < 0.001, Bonferroni corrected). Dashed lines indicate boundaries between stable vs. slow progressive (red) and rapid progressive (green) based on a multinomial logistic regression testing the relationship of the PPM-derived prognostic index to the rate of future tau accumulation. In particular, we observed a positive interaction between baseline β-amyloid burden and APOE 4, while a negative interaction between baseline β-amyloid burden and MTL GM density, consistent with the role of β-amyloid and APOE 4 as risk factors for progression to AD. Third, the PPM prototypes (one per class: clinically stable, progressive) indicate the most discriminative class representatives and allow us to predict an individual's trajectory. That is, using the GMLVQ-Scalar Projection method, we estimate the distance (based on the learnt metric tensor) of an independent test dataset from the Clinically Stable prototype and determine the PPM-derived prognostic index for each individual, allowing individualized prognosis beyond binary clinical labels (Fig. In particular, we extracted the PPM-derived prognostic index for each individual in an independent ADNI sample (out-of-sample validation, n = 419; cognitive normal individuals n = 119, patients with MCI n = 150, patients with AD n = 150). We then used multinomial logistic regression to capture the relationship of the PPM-derived prognostic index to the rate of future tau accumulation and determine boundaries for quartile classes that differ in likelihood of disease progression. We scaled the boundaries so that PPM-derived prognostic index indicates individuals who are more likely to: (1) remain stable (PPM index values below 0 fall within the 20th percentile of the future tau accumulation slope), (2) experience rapid progression (PPM index values higher than 1 fall above the 60th percentile of future tau accumulation), (3) experience slower progression (PPM index between 0 and 1)18,19. 1B) showed that the PPM-derived prognostic index was significantly different across groups (Kruskal-Wallis H test χ(2) = 121.46, p < 0.001), with a significantly higher index (Bonferroni corrected) for AD vs. MCI and CN (p < 0.001), MCI vs. CN (p < 0.001). This validation against clinical outcomes (i.e., diagnosis) provides evidence that the PPM-derived prognostic score is clinically relevant. We used the PPM trained on ADNI data to extract the PPM-derived prognostic index from AMARANTH patients (Supplementary Tables S2, S3 for sample sizes) with: APOE4 at week 1, structural MRI (MTL GM density), florbetapir PET (β-amyloid) scans at three time points (week 1, 52, 104), and cognitive measures (CDR-SOB, ADAS-Cog13) at three time points (week 1, 52, 104). In particular, using the GMLVQ-Scalar Projection method, we estimated the distance of each patient in the AMARANTH dataset at baseline (week 1) from the Clinically Stable prototype, determined the PPM-derived prognostic index for each individual in the AMARANTH sample and stratified individuals as slow vs. rapid progressive based on baseline (week 1) data (Fig. 2A; the sample size for stable was small (n = 5) and these data were excluded from further analysis). There were no significant differences in the PPM-derived prognostic index between treatment groups at baseline (Kruskal-Wallis H test χ(2) = 2.9733, p = 0.2261; Fig. Interestingly, individuals in the slow progressive group showed lower β-amyloid burden (t (330.22) = 11.833, p < 0.001), higher MTL GM density (t (326.33) = −17.351, p < 0.001) and better performance in cognitive tests (i.e., lower CDR-SOB (t(911.03) = 6.38, p < 0.001) and ADAS-Cog13 (t (806.77) = 5.23, p < 0.001)) compared to the rapid progressive group at baseline. These results suggest that individuals stratified as slow progressive by the PPM were at earlier stages of neurodegeneration and cognitive decline compared to individuals stratified as rapid progressive, corroborating the link of the PPM-derived prognostic score to cognitive decline and disease progression. Note that only 7.5% of the patients included in the trial had β-amyloid less than 50 centiloid, indicating intermediate or high likelihood of AD, and making it harder to stratify into subgroups based on β-amyloid alone. However, training the PPM on multimodal data (rather than β-amyloid alone) allowed a more precise stratification to subgroups at earlier vs. later disease progression stages. A Scatter plot of β-Amyloid against MTL GM density for AMARANTH baseline data (week 1) for placebo, lanabecestat 20 mg, lanabecestat 50 mg. Red dots indicate data for Slow progressive; green dots indicate data for Rapid progressive individuals. Individuals with higher β-Amyloid and lower GM density are stratified as Rapid progressive by the PPM model. Individuals with lower β-Amyloid and higher GM density are stratified as Slow progressive by the PPM model. B PPM-derived prognostic index for AMARANTH data at baseline (week 1): Box plots showing no significant differences in the PPM-derived prognostic index between treatment groups (placebo, lanabecestat 20 mg, lanabecestat 50 mg) at baseline (sample size: Supplementary Table S2, S3). Dashed lines indicate boundaries between stable vs. slow progressive (red) and rapid progressive (green) based on a multinomial logistic regression testing the relationship of the PPM-derived prognostic index to the rate of future tau accumulation. We tested whether stratifying the AMARANTH dataset based on the PPM-derived index at baseline (week 1) shows treatment effects on β-Amyloid. In particular, we used a mixed model for repeated measures (MMRM; Supplementary Table S4), to test Treatment (placebo, 20 mg, 50 mg) effects across timepoints (week 1, 52, 104) for each PPM-stratified group (Slow vs. 3A) a significant decrease in β-Amyloid for lanabecestat 20 mg and lanabecestat 50 mg treatment compared to placebo over time (week 104 compared to week 1). This decrease in β-Amyloid was observed for both PPM-stratified groups (Slow, Rapid progressive). In particular, MMRM analysis including fixed effects for treatment, timepoint, PPM-stratified group showed significant main effect of Timepoint (F(2, 485.43) = 37.01, p < 0.001) and PPM-stratified group (F(1, 489.19) = 84.04, p < 0.001), significant interactions for Treatment × Timepoint (F(4, 484.87) = 10.27, p < 0.001) and PPM-stratified group x Timepoint (F(2, 485.46) = 8.38, p < 0.001). Post-hoc comparisons showed that this PPM-stratified group x Timepoint interaction was significant for the lanabecestat 20 mg (F(2, 184.22) = 5.36, p < 0.01) and lanabecestat 50 mg (F(2, 146.80) = 3.47, p < 0.05) but not the placebo (F(2,152.76) = 1.20, p = 0.3033) group. A Mean β-Amyloid levels over time for Slow, Rapid, and All Progressive individuals (sample size: Supplementary Table S2) in the placebo (gray dashed), lanabecestat 20 mg (blue), and lanabecestat 50 mg (purple). Box plots of change in β-Amyloid levels (week 104 minus week1) for Slow, Rapid, and All Progressive (sample size: Supplementary Table S2). Asterisks indicate significant differences between treatment groups and placebo. Further, computing change in β-Amyloid burden over time (week 104 minus week 1) corroborated these results showing significantly higher reduction in β-Amyloid for lanabecestat 20 mg and lanabecestat 50 mg compared to placebo for both the slow and rapid progressive group (Fig. In particular, we observed significantly higher reduction of β-Amyloid for a) lanabecestat 20 mg vs. placebo (Welch's Two Sample t test: Slow progressive group: t(55.289) = 2.95, p < 0.001; Rapid progressive group: t(154.29) = 4.10, p < 0 0.001) b) lanabecestat 50 mg vs. placebo (Welch's Two Sample t test: Slow progressive group: t(62.362) = 3.86, p < 0.001, Rapid progressive group: t(112.53) = 5.00, p < 0.001). Further, we observed stronger reduction in β-Amyloid due to treatment for the rapid (21.91 % change) than slow (17.14 % change) progressive group (Welch's Two Sample t test: lanabecestat 20 mg: t(73.062) = 3.31, p < 0.01; lanabecestat 50 mg: t(94.62) = 2.54, p = 0.013). This result was potentially due to the higher β-Amyloid burden for the rapid compared to the slow progressive group at baseline (week 1; t(330.22) = 11.83, p < 0.001). Finally, similar analyses in the All Progressive group (n = 434, including individuals from the slow and rapid progressive groups) showed a significant reduction in β-Amyloid due to treatment over time compared to the placebo group. In particular, MMRM analysis showed a significant main effect of timepoint (F(2,482.33) = 55.65, p < 0.001) and significant interactions of treatment x timepoint (F(4,481.95) = 12.87, p < 0.001). Comparing Least-square means (LSM) for treatment effects over time (week 104 vs. week 1) across progressive groups showed lower LSM for the Rapid progressive than the All Progressive group for treatment (lanabecestat 20 mg, lanabecestat 50 mg) compared to placebo (Supplementary Table S5). In particular, we observed the lowest LSM for lanabecestat 50 mg for the Rapid Progressive group (LSM = − 25.38, SE = 3.28; placebo group: LSM = − 3.71, SE = 2.90) compared to the Slow Progressive group (LSM = −14.04, SE = 3.10; placebo group: LSM = 3.48, SE = 3.21) and All Progressive group (LSM = − 19.674, SE = 2.43), consistent with the result previously reported for the AMARANTH trial21 (LSM = − 19.74, SE = 1.97). Taken together, these results suggest that PPM-derived stratification provides a more sensitive tool for assessing treatment effects, showing stronger β-Amyloid reduction due to lanabecestat treatment in the rapid progressive group than across all progressive individuals or the whole sample considered in the AMARANTH trial. We tested whether stratifying the AMARANTH dataset based on the PPM-derived index at baseline (week 1) shows treatment effects on cognitive outcomes (CDR-SOB, ADAS-Cog13). In particular, we used a mixed model for repeated measures (MMRM; Supplementary Table S4), to test Treatment (placebo, 20 mg, 50 mg) effects across timepoints (week 1, 52, 104) for each PPM-stratified group (Slow vs. We observed an overall increase in CDR-SOB scores over time (week 104 vs. week 1), suggesting progression in dementia symptoms (Fig. However, for the Slow progressive group, we observed a significant decrease in CDR-SOB scores for lanabecestat 50 mg compared to placebo at week 104, suggesting slowing down of dementia progression (Fig. In particular, MMRM analysis showed: a) significant interactions: PPM-stratified group x Treatment x Timepoint (F(4, 2173.3) = 2.62, p < 0.05), Treatment × Timepoint (F(4, 2172.8) = 2.20, p = 0.067), PPM-stratified group x Treatment (F(2, 1866.4) = 2.51, p = 0.08), PPM-stratified group x Timepoint (F(2, 2163.7) = 14.72, p < 0.001); b) significant main effects of PPM-stratified group (F(1, 1809.1) = 35.70, p < 0.001), Treatment (F(2, 1870.5) = 3.00., p = 0.05), Timepoint (F(2, 2190.7) = 24.08, p < 0.001). Post-hoc comparisons showed that PPM-stratified group x Timepoint interaction was significant for the lanabecestat 20 mg (F(2, 716.64) = 4.301, p = 0.014) and lanabecestat 50 mg (F(2, 709.67) = 15.28, p < 0.001) but not the placebo (F(2, 720.26) = 0.96, p = 0.38) group. A Mean CDR-SOB over time for Slow, Rapid, and All Progressive individuals (sample size: Supplementary Table S3) in the placebo (gray dashed), lanabecestat 20 mg (blue), and lanabecestat 50 mg (purple). B Box plots of change in CDR-SOB (week 104 minus week1) for Slow, Rapid, and All Progressive (sample size: Supplementary Table S3). Asterisks indicate significant differences between slow progressive individuals in the 50 mg treatment group vs. placebo. Further, computing change in CDR-SOB scores over time (week 104 minus week 1) corroborated these results showing significant reduction in CDR-SOB scores (Welch's Two Sample t test, t (84.762) = 2.4475, p = 0.016) for lanabecestat 50 mg compared to placebo for the slow progressive group (Fig. In particular, the slow progressive group showed 33.64 % change in CDR-SOB scores over time compared to higher change in the a) Rapid progressive (80.84%) in the lanabecestat 50 mg group, b) the Slow (77.13%) and Rapid (70.41 %) in the placebo group. That is, the slow progressive group treated with lanabecestat 50 mg showed 46% reduction in cognitive decline compared to the placebo group, suggesting slowing of dementia progression for individuals stratified by the PPM as slow progressive. In contrast, no significant differences in CDR-SOB scores were observed for a) lanabecestat 50 mg compared to placebo in the rapid progressive group t(153.96) = − 0.26, p = 0.80, b) lanabecestat 20 mg compared to placebo in the slow (t(66.073) = 0.41, p = 0.68) or rapid (t (186.19) = − 0.53, p = 0.60) progressive group. Similar MMRM analyses for the All Progressive group (n = 1354, including individuals in the slow and rapid progressive groups) showed significant main effects of Timepoint (F(2,2195.0) = 23.68, p < 0.001) and Treatment (F(2,1883.7) = 3.68, p = 0.03), but no significant Treatment x Timepoint interaction (F(4, 2176.9) = 2.00, p = 0.09). Further, we did not observe significant changes in CDR-SOB scores over time (week 104 minus week 1) for lanabecestat 20 mg (Welch's Two Sample t test, t (266.93) = − 0.35, p = 0.72) nor lanabecestat 50 mg (Welch's Two Sample t test, t (242.09) = 1.16, p = 0.25) compared to placebo. These results are consistent with the lack of significant slowing of cognitive decline due to treatment, as previously reported for the AMARANTH trial21. Comparing Least-square means (LSM) for treatment effects over time (week 104 vs. week 1) across progressive groups showed lower LSM for the Slow progressive than the Rapid and All Progressive group for treatment compared to placebo (Supplementary Table S5). In particular, we observed the lowest LSM for lanabecestat 50 mg in the Slow Progressive group (LSM = 1.03, SE = 0.25; placebo group LSM = 2.21, SE = 0.24), compared to the Rapid Progressive group (LSM = 2.86, SE = 0.20); placebo group LSM = 2.74, SE = 0.18) and the All Progressive group (LSM = 1.97, SE = 0.16), consistent with the result observed when the whole clinical trial sample was previously analyzed (LSM = 3.17, SE = 0.18). Finally, MMRM analyses showed a similar reduction (39%) in ADAS-Cog13 scores for the slow progressive group compared to placebo (Supplementary Fig. In particular, we observed a significant PPM-stratified group x Timepoint interaction (F(2, 2162.8) = 10.37, p < 0.001) but no other significant interactions (Supplementary Table S4). Post-hoc comparisons showed that PPM-stratified group x Timepoint interaction was significant across all groups (lanabecestat 20 mg: F(2, 719.95) = 3.31, p = 0.03, lanabecestat 50 mg: F(2, 706.74) = 6.83, p < 0.001, placebo: F(2, 720.14) = 4.53, p = 0.011), suggesting that the lack of significant treatment effect may be due to differences over time in ADAS-Cog13 in the placebo group. For lanabecestat 50 mg, computing change in ADAS-Cog13 scores over time (week 104 minus week 1) showed lower change (18.62 %) for the Slow progressive group than the Rapid progressive (42.99 %) and for Slow (30.77%) and Rapid (33.62%) in the placebo group; that is, for the Slow progressive group we observed similar treatment effects as for CDR-SOB. However, this reduction in ADAS-Cog13 scores (39% reduction compared to placebo) was not statistically significant (Welch's Two Sample t test, t (78.854) = 0.90, p = 0.37) for lanabecestat 50 mg compared to placebo for the slow progressive group (Supplementary Fig. Considering LSM for treatment effects over time (week 104 vs. week 1) showed similar results with the analysis of CDR-SOB. That is, LSM was lower for the Slow progressive than the Rapid progressive, All Progressive group and the whole clinical trial sample for lanabecestat 50 mg compared to placebo (Supplementary Table S5). Taken together, these results suggest that PPM-derived stratification provides a more sensitive tool for assessing treatment effects, providing evidence for slowing of cognitive decline in the slow rather than the rapid progressive group; that is, lanabecestat 50 mg may slow disease progression at earlier stages of neurodegeneration. We asked whether treatment has an effect on PPM-guided stratification; that is, whether patients stratified as slow progressive using the PPM-derived prognostic score at baseline (week 1) remain in the slow progressive or transition to the rapid progressive group when stratified based on the PPM-derived prognostic score at week 104. We reasoned that treatment may slow dementia progression, resulting in a lower number of individuals in the slow group transitioning to rapid progression. Our results showed that treatment decreased the PPM-derived score compared to placebo, suggesting that treatment slowed cognitive decline (Fig. In particular, a three-way ANOVA on the PPM-derived scores showed a significant Treatment x Timepoint interaction (F(2, 562) = 3.16, p < 0.05) but not a significant PPM-stratified group x Treatment x Timepoint x (F(2, 562) = 0.08, p = 0.92). That is, we observed a higher increase in the PPM-derived scores in the placebo group rather than the treatment (lanabecestat 20 mg, lanabecestat 50 mg) groups over time. In particular, for the placebo group, 60% of individuals in the slow progressive group transitioned to rapid progressive, consistent with increased neurodegeneration over time. In contrast, individuals in the placebo group stratified as rapid progressive at baseline (week 1) remained mostly in this group rather than transitioning to the slow progressive group; that is, the percentage of individuals that remained in the rapid progressive group across weeks (week 1, 104) were 98.5% for the placebo, 95.8% for the lanabecestat 20 mg and 96.2% for the lanabecestat 50 mg group. Taken together, these results provide additional evidence that lanabecestat 50 mg may slow dementia progression at earlier stages of neurodegeneration, that is for individuals stratified by the PPM at baseline as slow rather than rapid progressive. Percentage of patients in each treatment group (placebo, lanabecestat 20 mg, lanabecestat 50 mg) transitioning between PPM-stratified groups (slow vs. rapid progressive) from baseline to week 104. There is a reduction in the percentage of patients transitioning from the slow progressive group (red) at baseline to the rapid progressive group (green) at week 104 compared to placebo. Conversely, there is an increase in the percentage of patients transitioning from the rapid progressive group at baseline to the slow progressive at week 104 compared to placebo. In light of our findings showing that lanabecestat 50 mg may slow dementia progression as measured by a decrease in CDR-SOB scores in the slow progressive group, we asked whether PPM-guided stratification reduces the sample size necessary for future clinical trials. We conducted power calculations to estimate the sample size needed when comparing lanabecestat 50 mg to placebo (i.e., decrease in CDR-SOB change between week 104 and week 1). Figure 6 shows that including slow progressive individuals based on PPM-guided stratification, reduces drastically the sample size necessary. Power (1-beta) of detecting a significant treatment effect (lanabecestat 50 mg vs. placebo) for CDR-SOB change (week 104 minus week 1) at different sample sizes and alpha level of 0.05. Smaller sample size (90% redcution) is needed to detect treatment effects on cognitive outcomes for the Slow (red; effect size: Cohen's d = 0.51) compared to the All Progressive (blue; effect size: Cohen's d = 0.15) group. In particular, for the slow progressive group, we observed a significant moderate effect size (Cohen's d = 0.51) for CDR-SOB change in lanabecestat 50 mg vs. placebo (t(84.76) = 2.45; p = 0.016). We estimated that for this effect size, a sample size of n = 82 per group (lanabecestat 50 mg vs. placebo) would be required for 90% power at alpha 0.05, (n = 117 at alpha = 0.01, n = 164 at alpha 0.001). In contrast, power calculations for all progressive group (i.e., including both slow and rapid progressive individuals) showed that a sample size of n = 762 per group would be required for a small effect size (Cohen's d = 0.15); i.e., significant change in lanabecestat 50 mg vs. placebo and 90% power at alpha 0.05 (n = 1198 at alpha = 0.01, n = 1760 at alpha 0.001). This is consistent with the lack of significant treatment effect on cognitive outcomes that was observed in the AMARANTH trial (n = 1380 for both lanabecestat 50 mg and placebo). Redesigning the AMARANTH trial to include only individuals stratified by the PPM at baseline as slow progressive would result to 90.00% reduction in sample size at alpha 0.01 (lanabecestat 50 mg vs. placebo at week 104), compared to the sample size for the all progressive group included in the AMARANTH trial. Recruiting the right patients at early disease stages is key to efficient clinical trials, maximizing the potential to reveal treatment effects. Patient heterogeneity and lack of sensitive tools for stratification at early stages of dementia pose a major challenge for AD drug discovery. To address this challenge, we built a robust and interpretable clinical-AI tool (PPM) based on a multimodal machine learning approach that supports feature extraction, precise patient classification to clinically stable vs. progressive and trajectory modeling to derive individualized patient prognosis. We have previously shown that the PPM predicts progression to AD at early dementia stages more precisely than standard clinical markers (i.e., gray matter atrophy, cognitive decline, β-amyloid positivity) or clinical diagnosis18,19,20. Here, we demonstrate that the PPM provides a robust tool for patient stratification and inclusion/exclusion in clinical trials with strong potential impact for drug discovery in the following key respects. First, we demonstrate that the PPM provides a more sensitive tool for patient stratification than standard approaches used for patient selection in clinical trials (e.g., β-amyloid positivity). It is important to know that the PPM was pretrained on research data (ADNI) and tested on the AMARANTH data (independent sample), supporting its utility as a patient stratification tool for clinical trials. In particular, the PPM predicted whether patients would progress to AD slowly vs. rapidly based on baseline data (i.e., week 1 before treatment). Testing the effect of treatment over 24 months (lanabecestat 20 mg vs. lanabecestat 50 mg) provides evidence that the slow progressive group showed not only a reduction in β-amyloid but, importantly 46% reduction in cognitive decline (i.e., decrease in CDR-SOB) compared to placebo. This is in contrast to the rapid progressive group that showed higher reduction in β-amyloid but no significant difference in cognitive outcomes compared to placebo. We observed a similar slowing of cognitive decline (39%) for ADASCog-13, consistent with previous trials suggesting that ADAS-Cog may be less sensitive in capturing cognitive decline at early dementia stages25. Our results are consistent with the findings from—recently FDA and MHRA approved—anti-amyloid antibody target (lecanemab, donanemab) phase 3 trials9,10; e.g., lecanemab was shown to slow the rate of cognitive decline by 27%9. Although these are independent trials that cannot be directly compared, our results suggest that our AI-guided patient stratification has strong potential to enhance the efficacy of trials and aid the discovery of new treatments. Second, individuals stratified by the PPM as slow progressive were shown to be at earlier stages of neurodegeneration based on baseline measurements (i.e., higher gray matter density, lower β-amyloid and better cognitive scores) compared to individuals in the rapid progressive group. AD develops gradually, involving a cascade of pathophysiological events beginning with the deposition of β-amyloid that may promote widespread pathological tau protein accumulation, leading to neurodegeneration and cognitive impairment26. It is likely that individuals in the rapid progressive group had progressed to later stages of neurocognitive decline, when it was too late for the lanabecestat treatment to be effective in slowing cognitive decline, despite the fact that it resulted in β-amyloid reduction. Interestingly, most patients in the trial were stratified by the PPM as rapid progressive; therefore, when all individuals in the trial were considered, the results were similar to the rapid progressive group; that is, reduction in β-amyloid but no significant changes in cognitive decline were observed. This is consistent with recent work suggesting that the timing of β-amyloid removal during a clinical trial is key for the trial success. If patients are too advanced at the start of the trial, or the underlying disease has progressed to a more advanced stage during the course of treatment because β-amyloid was not removed early enough, treatments may be less effective3. Evidence from previous trials of both successful and unsuccessful anti-β-amyloid therapies suggests that there is little clinical effect of amyloid removal if patients progressed past the mild dementia phase16,27. Further, recent clinical trial results (i.e., lecanemab, donanemab)9,10 showing decline in cognitive symptoms highlight the importance for treatments earlier in disease progression before damage has settled in the brain and when treatments may be more likely to be effective11,12. Finally, previous phase II and III trials of BACE inhibitors have shown a robust relationship between dose and degree of β-amyloid reduction, suggesting that a lower dose may be more effective at early stages of the disease28. Our results showing slowing of cognitive decline for the higher lanabecestat dose (50 mg vs. 20 mg) for the slow progressive group suggest that higher dose of BACE inhibition may be more effective over a 24-month treatment period, as in the AMARANTH trial. Our modeling approach has the potential to redefine inclusion/exclusion criteria for clinical trials, reducing patient heterogeneity that is known to hamper statistical power29. We have previously shown20 that our multimodal clinical AI marker (i.e., PPM-derived prognostic index) that captures the multivariate relationships across predictors is more sensitive in early prediction and prognosis of AD than standard clinical markers (β-amyloid, gray matter atrophy, cognitive scores). This is consistent with previous work demonstrating the benefit of integrating multimodal biomarkers for predicting future changes in cognition23,30,31,32,33,34. Further, PPM implements a trajectory modeling approach extending beyond binary classifications22 based on clinical labels (e.g., CN vs AD) that are poorly constrained; that is, individual patients at the class boundary that differ only slightly in their trajectory may be misclassified35. In contrast, our PPM-derived clinical AI marker provides a continuous index of future cognitive health from baseline data, reducing misdiagnosis associated with clinical labels and aiding patient stratification based on prognosis (i.e., predicted progression to AD). Third, the GMLVQ framework with ensemble learning that we adopt in our PPM enables us to develop: a) robust models36,37by combining data from multiple disease-relevant modalities, rather than considering single data types, b) interpretable models for early dementia prediction and prognosis, that is key for trusted clinical-AI solutions. In particular, interrogating the model metric tensors allows us to assess the contribution of different features (i.e., predictors and their interactions) to patient stratification and determine the most predictive data types to include in clinical trials. Further, estimating the distance (based on the trained PPM metric tensors) of an independent test dataset (AMARANTH sample) from the Clinically Stable prototype allows us to predict individual patient trajectories, re-stratify the trial sample and test treatment effects in slow vs. rapid progressive groups. This has the potential to accelerate clinical trials and reduce the costs associated with data collection by (a) focusing on key data types to be collected for patient stratification and treatment outcome assessment, (b) tailoring treatment targets to the right patient groups at different stages of disease progression. Integrating novel markers (e.g., blood biomarkers, digital markers from wearable technologies) for dementia diagnosis into our multimodal modeling approach provides the potential to move to less-invasive and more cost-effective clinical trials. Finally, we show that using PPM for patient stratification substantially decreases the sample size necessary for identifying significant changes in treatment outcomes. Recruitment and retention of a large number of qualified, diverse volunteers to participate in clinical research studies remain key barriers to the successful completion of AD clinical trials7,8. Thus, identifying the right patients at earlier stages for neurodegeneration to include in clinical trials decreases sample heterogeneity and size with strong potential to enhance trial efficiency (faster and cheaper). A possible limitation is that the PPM was trained with biomarker data (β-amyloid, gray matter density) from PET and MRI scans. We have previously shown that PPM reliably predicts cognitive decline when trained with cognitive data alone; yet, adding biomarker data enhances the precision of patient18,19,20. Future work is needed to test whether training the PPM with less-invasive and cost-effective data types (e.g., blood markers, cognitive tests) would provide a reliable stratification tool, reducing further costs and patient burden, and enhancing the efficiency of AD clinical trials. In sum, there is increasing interest in adopting AI tools for clinical trial optimization and drug discovery13,38,39,40,41,42. With trials lasting around 18 months for dementia that typically spans decades, enriching clinical trials with the right patients for specific targets is fundamental to clinical trial outcomes, may account for past failures43 and inform go / no-go decisions44. Clinical AI tools have the potential to play a key role in improving trial design by assisting trial enrichment with the right patients classified prior to enrollment based on AI-guided stratification. Further, including data from underrepresented groups that may be disproportionately affected by dementia is key for tackling the global dementia challenge and developing precision medicine interventions. Our vision is to scale up our predictive prognostic modeling approach to a responsible AI-guided stratification system that will support smarter multi-arm multi-stage trials, accelerating new target discovery for dementia treatment. We used data from: (1) a research cohort (the Alzheimer's Disease Neuroimaging Initiative, ADNI) for PPM training with within-sample cross-validation (n = 256) and independent test (n = 419), (2) a Randomized Clinical Trial cohort (AMARANTH, n = 1354), as independent test dataset for out-of-sample validation (see Supplementary Material for more information on ADNI and AMARANTH samples, including Patients, Randomization, and Blinding). All data were collected in accordance with ethics approvals at each site and following ethical guidelines (Declaration of Helsinki), including informed consent from participants and approved by the ethics committees at each site. AMARANTH is a phase 2/3, multicenter, randomized, double-blind, placebo-controlled, global clinical trial of Drug Substance: lanabecestat21. Lanabecestat is a brain-permeable inhibitor of human Beta-site amyloid precursor protein-cleaving enzyme 1 (BRACE1/β-secretase). Patients enrolled in the trial were diagnosed with MCI due to AD or mild AD. AD decline was measured by changes in cognition (primary outcome: ADAS-Cog13: 13-item Alzheimer Disease Assessment Scale–cognitive subscale; secondary outcomes: CDR-SB: Clinical Dementia Rating–sum of boxes, MMSE: Mini-Mental State Examination). AMARANTH was terminated early (approximately 16 months prior to planned completion) due to futility analysis. Because of this early termination, we included data from the placebo-controlled periods of the study (weeks: 1, 52, 104; Supplementary Table S2, S3) from patients with: APOE4 at week 1, structural MRI, florbetapir PET (β-amyloid) scans at three time points (week 1, 52, 104), and cognitive measures (CDR-SOB, ADAS-Cog13) at three time points (week 1, 52, 104). ADNI and AMARANTH datasets differ in patient demographics and data collection tools, allowing us to test PPM interoperability across research cohorts and clinical trial data. In particular, for ADNI, patients were selected with specific criteria related to amnestic MCI and Alzheimer's disease and MRI data were collected across MRI acquisition sites in the US. For AMARANTH, data were collected from patients diagnosed with early AD (i.e., patients with MCI; MCI due to AD) and patients diagnosed with mild dementia of the Alzheimer's type in Australia, Belgium, Canada, the USA, France, Germany, the United Kingdom, Italy, Japan, and Poland. We have previously demonstrated PPM interpretability across diverse research and clinical cohort data collected across sites and countries20. We have developed a trajectory modeling approach based on Generalized Metric Learning Vector Quantization (GMLVQ)18,19,20 that leverages multimodal data to make predictions about future cognitive decline at early dementia stages by iteratively adjusting class-specific prototypes and learning class boundaries (Supplementary Material: Predictive Prognostic Model). Generalized Learning Vector Quantization (GLVQ) is a supervised classification method that iteratively modifies class-specific prototypes to identify boundaries between discrete classes. For each training example, the GLVQ classifier determines the closest prototype for each class. This process helps to form class boundaries with large classification margins. Once the training is completed, the GLVQ classifier can be used for classifying test data. GMLVQ is an extension of the GLVQ algorithm that learns the metric to be used in the input space that enhances class separation. GMLVQ incorporates a full metric tensor to provide a robust distance measure (metric) tuned to the classification task. This metric defines a distance that naturally groups together members of the same class while separating the different classes away from each other. Mathematically, it provides specific feature scaling and quantifies pairwise task-conditional dependencies of the input features. Diagonal elements of the metric tensor identify key predictors, while the off-diagonal terms reveal pairwise feature interactions contributing to the classification task. Following our previous work18,19, we trained GMLVQ models to discriminate Clinically Stable (CN individuals who remain stable for 4+ years following baseline; n = 100) vs. Clinically Declining (individuals have a baseline diagnosis (at date of FBP scan) of either CN (n = 17) or MCI (n = 139) but received a diagnosis of dementia in future clinical evaluation (i.e., progressed to dementia (n = 75)), or had been diagnosed with dementia in a clinical evaluation prior to baseline (i.e., reverted (n = 81); n = 156) using ADNI data at baseline: medial temporal lobe gray matter (MTL GM) density18,19 (see Supplementary Material: MRI analysis: extracting medial temporal gray matter density), β-Amyloid and APOE4. All data were adjusted by regressing out potential confounding covariates (i.e., age, sex, and education). Following our previous work,18,19, we used the steepest descent method to minimize the cost function through online learning and performed hyperparameter tuning for the model using a nested cross-validation approach45, considering two hyperparameters. To evaluate the model's performance, we employed 10 iterations of a 10-fold cross-validation45. To mitigate any potential biases due to class imbalance in the dataset (Clinically Stable, n = 100; Clinically Declining, n = 156), we resampled the data to generate balanced classes. For each training fold, we repeatedly (n = 400) randomly down-sampled the majority class (i.e., Clinically Declining) to match the size of the minority class (i.e., Clinically Stable). Further, we used ensemble learning46, combining multiple models (n = 400) for robust learning of unbalanced classes. This ensemble learning approach with cross-validation helps mitigate potential individual model biases, resulting in more robust and accurate predictions36,37. Moving beyond binary classifications, we extended the GMLVQ framework to generate continuous predictions for each individual in the test dataset. In particular, we extracted a PPM-derived prognostic index employing a GMLVQ-Scalar Projection18,19 that extracts distance information (based on the learnt metric tensor) between the sample vector and the learnt class prototypes (representing Clinically stable vs. Declining). The scalar projection yields a large positive value for Clinically Declining individuals and a zero or negative value for Clinically Stable individuals. That is, the scalar projection index captures information about how far an individual is from the Clinically Stable prototype, serving as an individualized PPM-derived prognostic index. We have previously shown that this index relates significantly to the rate of memory decline18,20and future tau accumulation19, allowing us to estimate how fast an individual progresses from MCI to AD. Following our previous work 18we next used multinomial logistic regression to test the relationship of the PPM-derived prognostic index to the rate of future tau accumulation (e.g., future Tau slope in fusiform gyrus that showed significant tau accumulation for Clinically Declining individuals) and determine quartile classes (based on boundaries) that represent different levels of progression. We estimated the probabilities of each quartile class for a range of boundary values and identified the boundaries based on the quartile class with the highest probability at each value. The lower boundary (at the 20th percentile of the future tau accumulation slope) indicates individuals who are more likely to experience slower progression (slow progressive), while the higher boundary (at the 60th percentile of future tau accumulation) indicates individuals who are more likely to experience faster progression (rapid progressive). This multinomial logistic regression model allows us to stratify individuals based on their PPM-derived prognostic index (i.e., scalar projection score) and future tau accumulation slope. All available data for the PPM input features were used (Supplementary Table S2, S3 for sample size); no statistical method was used to pre-determine the PPM test sample size. We used the Kruskal-Wallis non-parametric test (SPSS) for comparisons of the PPM-derived prognostic index across groups, as the index data were not normally distributed (Shapiro-Wilk test). To ensure direct comparison to the analysis previously performed on the AMARANTH data21, we used a mixed model of repeated measures (MMRM), that is typically used in individual-randomized trials with longitudinal continuous outcomes and missing data. We tested the efficacy of lanabecestat 20 mg and lanabecestat 50 mg (compared to placebo) across time points (week 1: baseline, week 52, week 104) on: (a) biomarker outcome: β-amyloid, (b) cognitive outcomes: CDR-SOB, ADAS-Cog13. We conducted this analysis separately for each group (slow progressive, rapid progressive) and for the full sample (All progressive: slow and rapid progressive), including fixed effects for treatment, timepoint, PPM-stratified group. We calculated Least-square means (LSM; emmeans R package) from the MMRM model (mmrm, lme4 R packages) that represent model-adjusted means, including fixed effects for treatment, timepoint and their interactions, while accounting for covariates and unbalanced data. Pairwise comparisons of LSM across treatment groups and timepoints were performed using post-hoc tests, correcting for multiple comparisons. This approach ensures that reported means accurately reflect treatment effects while accounting for within-subject correlations and repeated measurements over time. We repeated these analyses with non-parametric ANCOVA tests (same fixed effects and covariates as for MMRM analyses) to account for deviations from normality (Supplementary Fig. S2, Supplementary Material: Non-parametric Statistical analysis), with similar results as the MMRM analyses. Further, for each group (slow progressive, rapid progressive) we computed change in outcomes (β-amyloid, cognitive data) at week 104 from baseline (i.e., week 104 minus week 1). We used Welch's t test to test for differences in outcomes across groups (i.e., lanabecestat 20 mg, lanabecestat 50 mg vs. placebo), as it does not assume equal variances between the groups and allows adjusting the degrees of freedom used in the test to accommodate differences in variance. MMRM, post hoc comparisons and Welch's t tests were conducted in R. Finally, we conducted power analysis, using the ‘pwr.t.test‘ function (pwr package in R), to estimate sample size for change in CDR-SOB (week 104 minus week 1) for lanabecestat 50 mg vs. placebo. A power level of 0.90 (1-β) was chosen to minimize the risk of Type II error, ensuring a high probability of detecting a true treatment effect. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data may be obtained in accordance with AstraZeneca's data sharing policy described at: https://astrazenecagrouptrials.pharmacm.com/ST/Submission/Disclosure. The raw trial data are protected and are not available due to data privacy laws. Anonymized processed data for studies directly listed on Vivli can be requested through Vivli at www.vivli.org. AstraZeneca Vivli member page is available outlining further details: https://vivli.org/ourmember/astrazeneca/. Source data for the figures are provided with this paper and are available at the Cambridge University repository: https://doi.org/10.17863/CAM.117732 Source data are provided in this paper. Code for figures is available at the Cambridge University repository: https://doi.org/10.17863/CAM.117732. Epidemiology of Alzheimer's disease and other dementias. Boxer, A. L. & Sperling, R. Accelerating Alzheimer's therapeutic development: The past and future of clinical trials. Cummings, J. L., Goldman, D. P., Simmons-Stern, N. R. & Ponton, E. The costs of developing treatments for Alzheimer's disease: A retrospective exploration. New approaches to symptomatic treatments for Alzheimer's disease. Economic analysis of opportunities to accelerate Alzheimer's disease research and development. Watson, J. L., Ryan, L., Silverberg, N., Cahan, V. & Bernard, M. A. Obstacles and opportunities in Alzheimer's clinical trial recruitment. The crisis in recruitment for clinical trials in Alzheimer's and dementia: An action plan for solutions. Dyck et al. Lecanemab in early Alzheimer's disease. Sims, J. R. et al. Donanemab in early symptomatic Alzheimer disease: The TRAILBLAZER-ALZ 2 randomized clinical trial. & Dell'Agnello, G. Timely diagnosis for Alzheimer's disease: a literature review on benefits and challenges. Artificial intelligence for dementia drug discovery and trials optimization. Cummings, J. et al. Alzheimer's disease drug development pipeline: 2023. Bateman, R. J. et al. Two phase 3 trials of gantenerumab in early Alzheimer's disease. Salloway, S. et al. A phase 2 multiple ascending dose trial of bapineuzumab in mild to moderate Alzheimer disease. Edmonds, E. C. et al. Susceptibility of the conventional criteria for mild cognitive impairment to false-positive diagnostic errors. Giorgio, J., Landau, S. M., Jagust, W. J., Tino, P. & Kourtzi, Z. Modelling prognostic trajectories of cognitive decline due to Alzheimer's disease. Giorgio, J. et al. A robust and interpretable machine learning approach using multimodal biological data to predict future pathological tau accumulation. Robust and interpretable AI-guided marker for early dementia prediction in real-world clinical settings. Wessels, A. M. et al. Efficacy and safety of lanabecestat for treatment of early and mild Alzheimer disease: The AMARANTH and DAYBREAK-ALZ randomized clinical trials. & Davatzikos, C. A review on neuroimaging-based classification studies and associated feature extraction methods for Alzheimer's disease and its prodromal stages. Insel, P. S. et al. Biomarkers and cognitive endpoints to optimize trials in Alzheimer's disease. Jack, C. R. et al. NIA-AA Research Framework: Toward a biological definition of Alzheimer's disease. Kueper, J. K., Speechley, M. & Montero-Odasso, M. The Alzheimer's disease assessment scale-cognitive subscale (ADAS-Cog): Modifications and responsiveness in pre-dementia populations. Sperling, R. A. et al. Trial of solanezumab in preclinical Alzheimer's disease. Jutten, R. J. et al. Finding treatment effects in Alzheimer trials in the face of disease progression heterogeneity. Dumurgier, J. et al. Alzheimer's disease biomarkers and future decline in cognitive normal older adults. Burnham, S. C. et al. Clinical and cognitive trajectories in cognitively healthy elderly individuals with suspected non-Alzheimer's disease pathophysiology (SNAP) or Alzheimer's disease pathology: a longitudinal study. Jansen, W. J. et al. Association of cerebral amyloid-β aggregation with cognitive functioning in persons without dementia. Comparison of different MRI-based morphometric estimates for defining neurodegeneration across the Alzheimer's disease continuum. Effects of amyloid pathology and neurodegeneration on cognitive change in cognitively normal adults. Artificial intelligence for diagnostic and prognostic neuroimaging in dementia: A systematic review. Schapire, R. E. The strength of weak learnability. Deep learning in drug discovery: An integrative review and future challenges. Artificial intelligence in pharmaceutical technology and drug delivery design. Deep learning tools for advancing drug discovery and development. Artificial intelligence in drug discovery and development. Cummings, J., Feldman, H. H. & Scheltens, P. The ‘rights' of precision drug development for Alzheimer's disease. Wessels, A. M. et al. Cognitive Go/No-Go decision-making criteria in Alzheimer's disease drug development. Application of majority voting to pattern recognition: an analysis of its behavior and performance. Combining Pattern Classifiers: Methods and Algorithms. We would like to thank Ali Aghelmaleki for data preparation, Avraam Papadopoulos and Anthony Fernandes for technical support. Data used in the preparation of this article were obtained from the Alzheimer's Disease Neuroimaging Initiative (ADNI) database (adni.loni.usc.edu). A complete listing of ADNI investigators can be found at: http://adni.loni.usc.edu/wp-content/uploads/how_to_apply/ADNI_Acknowledgement_List.pdf. This work was supported by grants to Z.K. from the Royal Society (INF\R2\202107), Alan Turing Institute (TU/B/000095), Wellcome Trust (221633/Z/20/Z). For the purpose of open access, the authors have applied for a CC BY public copyright license to any Author Accepted Manuscript version arising from this submission. Data collection and sharing for the Alzheimer's Disease Neuroimaging Initiative (ADNI) is funded by the National Institute on Aging (National Institutes of Health Grant U19 AG024904). In the past, ADNI has also received funding from the National Institute of Biomedical Imaging and Bioengineering, the Canadian Institutes of Health Research, and private sector contributions through the Foundation for the National Institutes of Health (FNIH) including generous contributions from the following: AbbVie, Alzheimer's Association; Alzheimer's Drug Discovery Foundation; Araclon Biotech; BioClinica, Inc.; Biogen; Bristol-Myers Squibb Company; CereSpir, Inc.; Cogstate; Eisai Inc.; Elan Pharmaceuticals, Inc.; Eli Lilly and Company; EuroImmun; F. Hoffmann-La Roche Ltd and its affiliated company Genentech, Inc.; Fujirebio; GE Healthcare; IXICO Ltd.; Janssen Alzheimer Immunotherapy Research & Development, LLC. ; Lumosity; Lundbeck; Merck & Co., Inc.; Meso Scale Diagnostics, LLC. ; NeuroRx Research; Neurotrack Technologies; Novartis Pharmaceuticals Corporation; Pfizer Inc.; Piramal Imaging; Servier; Takeda Pharmaceutical Company; and Transition Therapeutics. These authors contributed equally: Delshad Vaghari, Gayathri Mohankumar. Centre for AI, Data Science & Artificial Intelligence, BioPharmaceuticals R&D, AstraZeneca, Gaithersburg, USA Neuroscience, BioPharmaceuticals R&D, AstraZeneca, Cambridge, UK Neuroscience, BioPharmaceuticals R&D, AstraZeneca, Boston, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The remaining authors declare no competing interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Vaghari, D., Mohankumar, G., Tan, K. et al. AI-guided patient stratification improves outcomes and efficiency in the AMARANTH Alzheimer's Disease clinical trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61152-y'>Global dissemination of npmA mediated pan-aminoglycoside resistance via a mobile genetic element in Gram-positive bacteria</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 09:41:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The npmA gene, encoding a 16S rRNA methyltransferase, confers resistance to all clinically available aminoglycosides, posing a significant threat to effective antibiotic therapy. We analyze 1,932,812 bacterial genomes to investigate the distribution and mobilization of npmA variants. npmA is not found in Gram-negative bacteria, where it was originally described, but is identified among Gram-positive bacteria, predominantly as the npmA2 variant in the globally distributed Clostridioides difficile ST11 lineage. We also detect npmA2 in two vancomycin-resistant Enterococcus faecium isolates from a Dutch hospital. Upon sequencing and phenotypic analysis, we determine that E. faecium isolates are pan-resistant to aminoglycosides. Genomic characterization links npmA2 to a composite transposon, Tn7734, which is integrated within a previously uncharacterized Integrative and Conjugative Element (ICE) Tn7740, present in both npmA2-carrying C. difficile and E. faecium clinical isolates. Tn7740-like, but not npmA2, appears across diverse taxa, including human microbiome members. Here, we show that Tn7740 likely facilitates cross-species npmA2 mobilization between these Gram-positive bacteria and emphasize the risk of mobile genetic elements transferring pan-aminoglycoside resistance between clinically important bacterial pathogens. Aminoglycosides represent an essential family of antibiotics possessing wide activity spectra against both Gram-positive and Gram-negative bacteria1. Their introduction marked a significant advancement in antimicrobial therapy and made aminoglycosides a first line of defense against bacterial infections. However, in the 1980s, aminoglycosides lost their preferential status to alternative classes of antibiotics such as cephalosporins, carbapenems, and fluoroquinolones due to concerns about their associated toxicity and the development of resistance1. The persistent and escalating threat of multidrug-resistant bacterial pathogens has recently renewed the interest in aminoglycosides2,3. Nevertheless, the effectiveness of aminoglycosides is compromised by multiple aminoglycoside-resistance mechanisms, particularly the 16S rRNA methyltransferases (16S-RMTases). These enzymes target the 30S ribosomal subunit by methylating the 16S rRNA to prevent aminoglycoside binding, conferring a high level of resistance to multiple antibiotics of this class, including novel agents like plazomicin4,5. npmA is of particular concern as it provides high-level cross-resistance to nearly all clinically available aminoglycosides4,7. This gene was originally identified in an Escherichia coli isolate from Japan in 2003, located on a transferable IncF plasmid (pARS3) and flanked by IS26 elements7. Despite the impact it may have on aminoglycoside therapy, reports of npmA were scarce in the following years. This could be explained by the fitness costs npmA may impose to the cell, as it significantly inhibits ribosome assembly and translation, suggesting an evolutionary trade-off between resistance and cellular fitness8. The presence and expression of the 16S-RMTases was thought to be limited to Gram-negative bacteria9. However, npmA, and its variant npmA2, differing from npmA (re-designated as npmA1) by only 3 nucleotides, resulting in a single amino acid change, were discovered in Clostridioides difficile clinical isolates from the USA in 201810. This finding has raised several questions that remain unresolved, including the possibility that the acquired npmA genes emerged from the chromosomally encoded npmA genes carried by some C. difficile strains. The limited data available suggest a notable difference between the genomic context of npmA1 in E. coli and npmA1 and npmA2 in C. difficile. However, the high genetic similarity among C. difficile isolates in the regions flanking npmA2 suggests a conserved genomic environment10. The increasing availability of bacterial genomes from genomic surveillance studies opens new opportunities to study the immediate genetic context of antimicrobial resistance (AMR) genes systematically at an unprecedented scale and resolution across bacterial populations. Bioinformatic analyses of bacterial genomes can be used to identify the mobile genetic elements (MGEs) carrying AMR genes and to reconstruct the mobilization events that led to the acquisition, maintenance and spread of such genes11,12. The recent identification of npmA1 and npmA2 in C. difficile highlights the need for the characterization of their genomic context and the identification of the mechanisms of acquisition and mobilization of these genes across bacterial species and populations. In this work, by systematically identifying and analyzing the whole-genome sequences of all available npmA-carrying strains, we demonstrate that npmA2 is widespread across diverse hosts and geographic regions in C. difficile. We discover that npmA2 is integrated within a conserved composite transposon, Tn7734, which in most isolates is, in turn, embedded in a previously unreported integrative and conjugative element (ICE) designated ICE Tn7740. We also report npmA2 in Enterococcus faecium isolates from a Dutch hospital linked to ICE Tn7740, underscoring the potential for cross-species mobilization of this aminoglycoside resistance gene among Gram-positive bacteria. We identified 103 genomes harboring the npmA gene from a combined total of 1,932,812 bacterial genomes in the AllTheBacteria dataset and our dataset of 33,253 non-overlapping C. difficile genomes (Supplementary Fig. Following QC filtering, 71 of these genomes met the high-quality criteria (see Methods). These were predominantly C. difficile isolates (n = 69), as detailed in Supplementary Data 1, alongside a minority of E. faecium (n = 2). We did not identify any npmA-carrying Gram-negative isolates. To date, the sole npmA1-carrying isolate is the E. coli plasmid pARS3 (AB261016.2), absent from our initial dataset. A striking observation was the predominance of the npmA2 variant, present in 66 of the C. difficile genomes, in contrast to the npmA1 variant, differing by 3 nucleotides and a single amino acid, found in only three genomes. The origin of these isolates was diverse, mainly sourced from humans (n = 42), pigs (n = 19), and environmental sources (n = 8), which underscores the cross-niche distribution of the gene. Geographically, npmA was globally distributed and isolated from a total of six countries (Fig. The isolates were collected over a prolonged period of 20 years (2001-2021), with a notable majority (n = 57/69) obtained between 2008 and 2021. a Global map of npmA-carrying C. difficile isolates included. b Minimum spanning tree (MST) representing the genetic relationships among npmA-carrying C. difficile isolates based on core-genome multilocus sequence typing (cgMLST). Tips are collapsed if there are not allelic differences and colored according to the sequence type (ST), and sizes correspond to the number of isolates. Numbers on the connecting lines indicate allelic differences between them. Colors correspond to sequence types as shown in the panel legend. c Bar graphs showing the distribution of isolates according to their source (human, livestock, or environmental), geographic origin, and the year of isolation, categorized by ST. The MLST and cgMLST analyses of npmA-carrying C. difficile genomes uncovered limited genetic diversity. This was evident from the few predominant clusters among the 69 genomes analyzed (Fig. ST11 emerged as the most common ST, since 69.5% (n = 48) of the isolates belonged to this ST. We further demonstrated that the npmA2 gene significantly increases resistance to gentamicin (>256 mg/L) in ST11 strains (SAMEA5846557 and SAMEA5846561), relative to the npmA-negative lab strain 630Δerm (Supplementary Table 1A). Its broad geographical and ecological presence make ST11 as a pivotal carrier of npmA across niches, with the earliest npmA2-carrying ST11 isolate dating back to 2001 from a pig sample (Fig. Conversely, ST161 (n = 8 isolates) and ST36 (n = 4) exhibited a more localized distribution, as all these isolates were collected from pigs in Germany within a narrow time frame (Fig. The first identification of a C. difficile npmA-carrying ST54 isolate was in 2016 (Fig. There was also a single detection of one ST109 isolate from an Australian pig in 2008. To contextualize these findings, we queried the C. difficile EnteroBase database (30,415 genomes, accessed January 15, 2025). Ribotype (RT) 078, the dominant RT within ST11, was among the most prevalent lineages (7.85%, n = 2387), whereas ST36, ST54, and ST161 represented < 1% of the dataset (Supplementary Fig. npmA was not detected in other well-represented lineages (e.g., RT027, RT014, RT001, RT002), underscoring the particular significance of ST11. In contrast, ST161 and ST36 originated from a single German study13, likely representing a single farm population, suggesting potential under-sampling of these lineages (Supplementary Data 1). To determine the genomic context of npmA and its potential events of integration and mobilization of this gene, we incorporated contextualized genomes lacking npmA in our analysis that shared identical core-genome alleles with npmA-carrying isolates and created maximum likelihood phylogenies for each ST (see Methods). In ST11, our study of 48 npmA-carrying isolates and 520 related genomes revealed extensive dissemination of this lineage, especially in humans and pigs, across 19 countries, particularly in the United Kingdom and Germany. Applying a 10-SNP threshold commonly used to infer recent transmission14, we identified clonal expansions within certain npmA-positive clades, indicating genetic relatedness and likely recent transmission events (Supplementary Data 1). a Maximum likelihood phylogeny of 568 ST11 isolates (48 npmA-positive and 520 contextual genomes) based on non-recombinant core single-nucleotide polymorphisms (SNPs) across 1954 core-genome alleles. b Maximum likelihood phylogeny of 105 ST54 isolates (8 npmA-positive and 97 contextual genomes) based on non-recombinant core SNPs across 2096 core-genome alleles. Metadata columns show isolation niche, country of origin, npmA variant and presence of mobile genetic elements Tn7734 and ICE Tn7740. ST54 isolates, which were primarily sourced from human clinical specimens, were mainly restricted to China (n = 41), the United Kingdom (n = 34), and the United States (n = 21). A distinct clade of npmA2-carrying isolates in the USA, separate from sporadic npmA1 and npmA2-carrying isolates in Australia and China, indicates local circulation of npmA2 within a defined timeframe (Fig. Contrarily, ST161 and ST36 clusters exhibited a restricted ecogeographic distribution linked to pigs from Germany. The ST161 cluster was particularly restricted, with no additional related genomes identified, revealing a highly exclusive lineage. The ST36 cluster, albeit slightly more expanded, included just two additional genomes. Despite their geographical and ecological diversity, these ST36 genomes showed remarkable genetic similarity, with only 7-8 SNPs differentiating them, indicating a close genetic relatedness (Supplementary Fig. There is a notable scarcity of information regarding these STs, and existing literature lacks evidence of their association with either pigs or humans. We conducted an in-depth analysis of the genomic context of npmA2 to identify the MGEs carrying and mobilizing this gene (see Methods). We found an identical ~ 5 kb genomic region across all isolates, irrespective of ST, source, or geographic origin, flanked by two identical sequences of a newly characterized IS30 family member, named ISCld1. This arrangement suggested mobilization through a putative composite transposon, which we termed Tn7734 (Fig. Most npmA2-carrying C. difficile isolates (64 out of 66) also shared a common and broader mobilizable entity of ~ 33 kb that contained Tn7734, annotated as a complete Integrative and Conjugative Element (ICE) by ICEscreen15, designated ICE Tn7740. Intriguingly, ICE Tn7740 often co-occurred with the remnants of an Integrative and Mobilizable Element (IME) harboring aadE, an aminoglycoside nucleotidyltransferase, indicating a putative point of integration of ICE Tn7740 (Fig. b Genetic context in the 30 kb flanking regions of npmA2. The top panel displays the inverse cumulative distribution function (cdf), which represents the decay of structural similarity with distance from npmA2, helping to identify common positions where non-homologous structural variation is introduced. Each block is colored according to homology, and the gray blocks indicate regions of non-homology. c Genetic structure and comparison of the ICE Tn7740 variants. d Schematic representation of the npmA2 integration sites using as a reference C. difficile 630 (GenBank accession AM180355.1) by mapping the npmA2-carrying contigs. Key integration sites include a helix-turn-helix transcriptional regulator gene (CD630_15730, coordinates: 1,821,726-1,822,940) and a distinct site in German ST11 isolates (CD630_19010, 2,203,327-2,203,836). In addition, an integration into a metallophosphoesterase gene (CD630_06890, 833,702-835,627) in an ST11 isolate and within a helix-turn-helix transcriptional regulator gene (CD630_02920, 353,054-354,148) for ST161 and ST54 clusters, along with a unique site in the ST36 cluster within a fic family protein gene (CD630_06470, 769,216-769,530). The structure of ICE Tn7740 varied slightly among isolates, with additional insertions of IS110, disruptions by ltrA gene or both, giving rise to diverse ICE Tn7740 variants (Fig. Core components to all ICE Tn7740 included a serine recombinase, a mobP family relaxase, and the conjugative system proteins virB4 and virD4, which allowed its classification within the Tn5252 superfamily and Tn1549 family. Adjacent to the ICE, the IME featured a mobV family relaxase, and notably, a serine recombinase, often compromised structurally due to proximity and interaction with ICE Tn7740. Using the complete ICE Tn7740 as a reference, we compared and annotated isolates with shorter contigs harboring incomplete ICE sequences, thereby assigning them to specific ICE Tn7740 variants. The analysis revealed a consistent integration pattern of ICE Tn7740 variants and associated IMEs across various clusters, despite the diverse phylogenetic distribution of npmA2 (Fig. Notably, ST11 isolates frequently showed integration into genes encoding a helix-turn-helix transcriptional regulator. A subset of German ST11 environmental isolates exhibited a distinct integration site, while a complex genetic interplay was observed with ICE Tn7740 integrated into another ICE within a metallophosphoesterase gene in one ST11 isolate. Similarly, ST161 and ST54 clusters shared an integration site within a helix-turn-helix transcriptional regulator gene, whereas the ST36 cluster presented a unique site within a fic family protein gene, highlighting the specificity of ICE integration patterns across different clusters (Fig. The analysis of the integration sites showed low information content across all positions (mean < 0.5 bits) and only a modest A/T bias at a few sites, with no discernible core motif (Supplementary Fig. In the C. difficile M120 chromosome (FN665653.1), we observed a helix-turn-helix transcriptional regulator protein (CDM120_RS08225, 1,726,966-1,728,177) which may have served as the original integration site of the IME carrying aadE (SAMN25516600). Subsequently, ICE Tn7740 seems to have been integrated into the serine recombinase of this IME, leading to the consistent co-localization of these two elements (Fig. The phylogenetic analysis of the ST11 cluster revealed that npmA2-negative isolates carrying the same ICE variants but lacking Tn7734 and npmA2 yet retained a copy of ISCld1 (Fig. This pattern is consistent with the behavior of IS30 family members, where npmA2 could potentially be mobilized via a circular intermediate, leaving a remnant ISCld1 from where Tn7734 was initially located (Fig. a Genetic comparison of the ICE Tn7740 integration site using npmA2-carrying and npmA-negative ST11 contextual isolates. b Schematic diagram of the potential of ISCld1 to retain a copy at the integration site through the generation of a putative circular intermediate. We conducted a thorough analysis of the genomic surroundings flanking the npmA1 gene variant, too, identified as a rarer variant in C. difficile (n = 3, two ST11 and one ST54). The same genetic context was found in all npmA1-carrying isolates (Supplementary Fig. The npmA1 gene was found integrated within a putative complete ICE, belonging to the Tn5252 superfamily and Tn1549 family, as well as the ICE Tn7740. This putative ICE exhibited structural variations primarily due to the insertion of the ltrA gene, expanding its size to an estimated 40,000-45,000 bp (Supplementary Fig. This ICE, while sharing the same family as the ICE Tn7740 carrying npmA2, presented a divergent structural composition beyond its similar conjugative machinery (Supplementary Fig. A comparative analysis with the initially described genetic context of npmA1 in the pARS3 plasmid of E. coli (AB261016.2) revealed a distinct genomic environment, suggesting disparate origins of npmA1 in C. difficile and E. coli (Supplementary Fig. To investigate the capacity of npmA for cross-species transmission, we investigated the genetic context of npmA2 in the two vancomycin-resistant, clonal E. faecium genomes identified (out of the 71 npmA-carrying isolates identified), both isolated from a single Dutch hospital16. The isolates were provided by the Microbiology Department at Onze Lieve Vrouwe Gasthuis, Amsterdam. These strains were derived from two patients who had undergone heart valve replacements. Resequencing both isolates using Oxford Nanopore technology, we found that npmA2 also appeared within the same Tn7734 and ICE Tn7740 structure observed in C. difficile isolates with a 99.9% identity (Fig. 5), demonstrating a common MGE responsible for the genetic mobilization of npmA2 across these two Gram-positive bacterial species. ICE Tn7740 was integrated into the chromosomal dinB gene (E6A31_14200, coordinates: 2,753,333-2,753,857; reference E. faecium strain SRR24, GenBank accession CP038996.1). Despite this shared genomic architecture, conjugation assays failed to detect transfer of ICE Tn7740 from C. difficile to either C. difficile or E. faecalis recipients. In contrast, control transfer of Tn6194 from C. difficile to E. faecalis occurred at a similar frequency to previous studies17. Similarly, intra-enterococcal mating experiments resulted in vanA-positive plasmid transfer at 1.82 ± 0.016×10⁻² per donor, but no npmA2-positive ICE Tn7740 transconjugants were recovered under the same conditions. Comparison of the genetic environment surrounding npmA2 in C. difficile isolate SAMEA1710827 and E. faecium isolate SAMEA4885232, showing conserved synteny and shared mobile genetic elements. To further characterize the clinical isolates recovered in the Netherlands, we determined the antimicrobial susceptibility profile of the two E. faecium isolates in vitro, summarized in Supplementary Table 1B, which revealed high minimum inhibitory concentrations (MICs) (≥ 1024 mg/L) for all the aminoglycosides tested. High levels of resistance to gentamicin, kanamycin and streptomycin can be explained in these E. faecium isolates by the presence of other aminoglycoside-modifying enzymes (Supplementary Tables 2 and 3). However, the high MIC values observed in the npmA2-carrying E. faecium isolates against apramycin, not explained by these enzymes, suggest an association between the presence of the npmA2 gene and aminoglycoside resistance (see Discussion). ISCld1, a component of Tn7734, was found exclusively in association with C. difficile ST11, either co-existing with ICE Tn7740 (n = 8) or in ST11 genomes lacking the ICE (n = 8). Conversely, ICE Tn7740 exhibited a broader presence, being in various C. difficile STs (detailed in the Supplementary Table 4) and in genomes from the Lachnospiraceae family. We queried a comprehensive dataset of 147,851 high-quality metagenome-assembled and isolate genomes from the Unified Human Gastrointestinal Genome (UHGG) catalog18 for the presence of ISCld1 and ICE Tn7740. While ISCld1 was only discovered in a Roseburia hominis genome, ICE Tn7740 showcased extensive dissemination across multiple Lachnospiraceae family genomes (n = 14), including species such as Sellimonas intestinalis, Clostridium bolteae, and Dorea spp., and even Ruminococcus bromii genomes, belonging to the Acutalibacteraceae family (Supplementary Table 5). To evaluate the intrinsic stability of npmA2 without selective pressure, we conducted serial passaging experiments in various host backgrounds. In its native context within ICE Tn7740, C. difficile ST11 (SAMEA5846557) retained full gentamicin resistance in 100% checked colonies across six consecutive daily passages (~200 generations) (Supplementary Fig. Similarly, the chromosomally integrated ICE in E. faecium E7474 remained stable over ~ 200 generations, with no loss of apramycin resistance. To assess the stability of npmA2 with its native promoter in a different mobile element, the gene was cloned into the shuttle vector pTG262. In E. coli HST08 and MG1655, plasmid maintenance remained complete (100%) over ~ 60 generations. In E. faecalis JH2-2, plasmid stability was slightly more variable. One biological replicate showed a marked loss (~50%) on day 2, while the other two remained > 90% stable. This single event reduced the day-2 group average to 83 ± 17%, but plasmid carriage recovered by day 6 (94 ± 4%), consistent with transient instability rather than sustained plasmid loss (Supplementary Fig. These results indicate that npmA2 imposes minimal or no detectable fitness cost in Gram-positive hosts and remains fully stable in standard E. coli strains under laboratory conditions. This study represents a comprehensive analysis of the occurrence, distribution, and mobilization dynamics of npmA in C. difficile populations and in one E. faecium clone. Through a systematic and in-depth genomic analysis, we discovered a predominant presence of the npmA2 variant and demonstrated the within- and cross-species mobilization of this gene through a previously unreported and common complex MGE (Tn7734 and ICE Tn7740), as shown by its conserved genomic structure across diverse bacterial species, lineages, niches and geographical regions. Although 16S-RMTases have been described as exclusive to Gram-negative bacteria9, the detection of npmA variants in C. difficile and in E. faecium reported in this study challenges this prior assumption. This wider distribution of npmA across various nosocomial pathogens represents a significant threat in clinical settings10. The analysis of over 30,000 C. difficile genomes show that npmA is still rare, detected in only 101 isolates (0.34%). When determining the population structure of npmA-carrying C. difficile, we identified C. difficile ST11 as a predominant group (Fig. ST11 comprises multiple ribotypes (078, 126, 127, 033, 288), it is well-established in livestock populations worldwide and a key contributor to the incidence of community-acquired C. difficile infection in humans, suggesting zoonotic and anthroponotic transmission pathways19. Notably, the earliest npmA2-carrying ST11 isolate traced back to 2001 and was sampled from a pig in the USA. This evidence further supports the hypothesis that animals could play a very important role in harboring and disseminating this gene. However, in the phylogenetic analysis focused on the ST11 cluster, we observed different evolutionary events of npmA acquisition (Fig. 2a), without identifying any clonal transmission of npmA-carrying C. difficile lineages from pigs to humans. Intriguingly, some npmA2-carrying ST11 isolates originated from the same study as those in the ST161 and ST36 clusters13, both of which are rare STs, representing < 1% of C. difficile genomes in EnteroBase, compared to ST11, which accounts for 7.85% (Supplementary Fig. This might imply that the npmA2 origin in these clusters (ST161 and ST36), which appears to be restricted to pigs in a single geographic location (Germany) in this dataset, could be attributed to the extensively detected and disseminated ST11 cluster. The limited detection of npmA2 in these rare STs could reflect under-sampling of these lineages in public genomic databases, which are predominantly enriched for clinical isolates, leading to a relative underrepresentation of genomes from animal and environmental sources20. All pig-derived isolates in these clusters were collected as part of a single study involving multiple farms in Germany13. Contrary to what is observed in the ST11 lineage, our analysis of the ST54 cluster revealed a strong link with human-derived isolates, predominantly restricted to specific geographic regions (Fig. Although not widely studied, ST54 has emerged as a predominant ST in China21,22. In contrast, we identified a subclade within the USA isolates, characterized by the presence of npmA2 (Fig. Since no npmA-carrying isolates were detected in other high-prevalence C. difficile lineages, including RT027, RT014, RT001 and RT002, this emphasizes the specific role of ST11 (RT078), and, to a lesser extent, ST54 (RT012), as key carriers of the npmA gene. Our stability assays demonstrate that npmA2 persistence is influenced more by the nature of the genetic vehicle and host-specific factors than by a Gram-positive versus Gram-negative classification. The gene remained stably maintained on its native ICE in both C. difficile (Supplementary Fig. The transient decrease observed in E. faecalis (Supplementary Fig. 9) likely reflects host-specific differences in pTG262 plasmid replication and maintenance mechanisms rather than a direct fitness burden from the npmA2 gene. These results contrast with previous reports of rapid npmA1 inactivation in IS-rich E. coli DH5α strains8, suggesting that npmA2, potentially due to its single amino acid substitution, the pTG262 plasmid biology, or both, may exert a lower fitness cost. Further experiments will be needed to quantify potential fitness effects and to better understand the factors influencing npmA2 stability across different bacterial hosts. The characterization of the genetic context of AMR genes and their phylogenetic distribution offers fundamental insights into their mobilization capacity, host range and epidemiology, facilitated by the increasingly large datasets of bacterial genomes available and recent advancements in bioinformatics tools11,12. These approaches have recently been applied, for example, to the study of beta-lactamase12 and the critical carbapenemase gene blaNDM-123. Through a standardized bioinformatic pipeline, our study identified the integration of npmA2 within an uncharacterized composite transposon Tn7734. This previously unreported ICE Tn7740, belonging to the Tn5252 superfamily and Tn1549 family, contained a complete set of genes for recombination and conjugation. Tn1549-type elements represent the major transposon carrying vanB genes among enterococci24, and these elements have also been identified in C. difficile25,26. In our study, the widespread presence of Tn7734 and ICE Tn7740 across C. difficile isolates, irrespective of ST, geographical origin, or source, underscores their pivotal role in both the mobilization and persistence of the npmA2 gene. We determined that ICE Tn7740 integrates into specific chromosomal loci, typically in association with IMEs carrying the aadE gene. Intriguingly, the ST11 phylogenetic analysis revealed subclades where ICE Tn7740 was present without the npmA2 gene, suggesting an ongoing dynamic of npmA2 gene acquisition and loss (Fig. This was further evidenced by the detection of ISCld1 remnants in isolates lacking npmA2, indicative of a behavior typical of the IS30 family IS elements. This mechanism mirrors that observed with ISApl1 and Tn6630, known for mobilizing the colistin resistance gene mcr-1, through a copy-out-paste-in process in which the element transiently forms a circular intermediate that can integrate in a new location while partial or complete copies remain at the original site27,28, suggesting a common strategy for gene mobilization and the potential of npmA2-ISCld1-Tn7734-ICE Tn7740 association to become an emerging health concern. The genomic context of npmA1 diverged significantly from that found in a previously characterized E. coli plasmid7. This observation suggests that the hypothesis proposing the chromosomal integration of npmA1 in C. difficile as a precursor to its presence in E. coli is less likely. The identification of these same MGEs (Tn7734 and ICE Tn7740) in two E. faecium isolates (Fig. 5) demonstrates the capacity of npmA2 to disseminate beyond C. difficile. E. faecium may have acquired ICE Tn7740 by conjugation from C. difficile, potentially shared within a niche like the gut microbiome. Although evidence of ICE conjugation between C. difficile and E. faecalis exists30, our study suggests that a similar phenomenon could have taken place between C. difficile and E. faecium. Despite the transfer of control elements under identical conditions, no ICE Tn7740 transconjugants were recovered. These results support the inference from our genomic survey that horizontal dissemination of ICE Tn7740 carrying npmA2 is an infrequent event. Its observed distribution across species and lineages may thus reflect rare transfer events followed by clonal expansion, rather than ongoing active spread. Moreover, our analysis revealed the presence of ICE Tn7740 across various C. difficile STs and within members of the Lachnospiraceae family (Supplementary Table 4), which are one of the core families of the human gut microbiota31. Notably, this occurs without any observed association between ICE Tn7740 and Tn7734 or npmA2. This observation suggests these genetic elements might be originating from anaerobic Gram-positive gut commensals, reflecting the distribution patterns observed in other Tn1549-type ICEs32. In addition, the identification of ISCld1 within the genome of R. hominis, also a member of the Lachnospiraceae family, further supports the hypothesis that these elements may originate from such bacteria. Despite these insights, our search has not yet identified npmA2-like genes within these bacterial populations, indicating a need for further research to clarify the precise origins and mechanisms of spread for this resistance determinant. Phenotypic analysis of two ST11 isolates showed that npmA2 presence in C. difficile increased resistance to aminoglycosides like gentamicin (MICs > 256 mg/L) compared to non-npmA isolates, as it was already reported10, demonstrating the role of npmA in conferring a high level of gentamicin resistance. Notably, high levels of aminoglycoside resistance in enterococci (defined by MICs ≥ 512 mg/L) are also linked to aminoglycoside-modifying enzymes33. Our genotypic predictions, based on WGS of E. faecium34, were consistent with phenotypic resistance patterns to gentamicin, kanamycin, and streptomycin, attributable to varied aminoglycoside-modifying enzymes (Supplementary Table 2). These findings suggest that resistance to apramycin cannot be solely attributed to the action of aminoglycoside-modifying enzymes, as none of the four known resistance determinants of apramycin (aac(3)-IVa, apmA, kamB and npmA)35 were detected in the enterococci isolates here analyzed. Therefore, we hypothesize that the npmA2 gene plays a significant role in conferring high pan-aminoglycoside resistance levels. This study, while large and comprehensive, is subject to several limitations. Firstly, the reliance on public databases for genome sequences, which may suffer from selection biases and not fully represent global microbial diversity, might lead to uneven distribution of species, strains, carrying hosts and countries among npmA-positive isolates. Secondly, although our conjugation assays provided preliminary insights, experiments under conditions other than standard in vitro assays would be needed to better study these rare transfer events. Further work demonstrating direct gene transfer and assessing the phenotypic impact of npmA2 in the absence of aminoglycoside-modifying enzymes in E. faecium would markedly strengthen the conclusions drawn from genomic analyses. Our study provides significant insights into the global distribution and spread, genetic integration, and the capacity for intraspecies and interspecies mobilization of npmA. This study emphasizes the risk of MGEs in propagating AMR genes within and between clinically important bacterial pathogens, and the public health value of standard bioinformatics pipelines to analyze available bacterial genomes and metagenomes to systematically assess that risk. Future research should focus on the experimental verification of gene transfer mechanisms and the examination of the ecological and evolutionary dynamics of AMR genes in complex microbial communities. To investigate the presence of the npmA gene, the nucleotide sequences of npmA1 (GenBank accession number NG_048018.1) and npmA2 (NZ_JFOF01000002.1, region:1860-2519) were used as queries for a Compact Bit-sliced Signature (COBS)36 v0.3.0 index search. This search was performed on the AllTheBacteria dataset, which includes 1,932,812 curated draft genomes37 (combining 661,405 genomes from the Blackwell dataset38 with 1,271,428 new assemblies up to May 2023), applying a k-mer similarity threshold of 95%, which allowed us to distinguish between npmA1 and npmA2 variants. This search approach was chosen for its efficiency and ability to process large-scale genomic data. Upon identifying that npmA was predominantly associated with C. difficile (see Results), we expanded our search to compile a dataset of 33,253 non-overlapping C. difficile genomes. These genomes were collected from multiple databases, ensuring no overlap by using unique BioSample identifiers. Specifically, 28,923 C. difficile genomes were obtained from EnteroBase39 (accessed on 19 September 2023), 17,680 from NCBI Pathogen Detection (https://www.ncbi.nlm.nih.gov/pathogens/, accessed on 16 November 2023), and 13,706 from the Blackwell dataset38 (November 2018). A diagram showing the databases used for this study can be seen in Supplementary Fig. For a uniform analysis approach, raw Illumina reads from genomes containing npmA1 and npmA2 sequences were assembled using Shovill v1.1.0 (https://github.com/tseemann/shovill) with default settings. Available assemblies derived from long-read sequencing technologies, specifically Oxford Nanopore (n = 1) and PacBio (n = 8), were also downloaded and included in our analysis. To ensure a high and comparable quality of genomic data across genomes sequenced from multiple laboratories, stringent quality control (QC) measures were applied. We used Quast v5.0.240 and CheckM v1.2.141 for the quality assessment. De novo assemblies were checked for total assembly length to exclude those shorter than 3.7 Mb or longer than 4.7 Mb that may represent low-coverage or contaminated samples. In addition, assemblies with more than 200 contigs were excluded to prevent the inclusion of highly fragmented assemblies. Genomes with <90% completeness and >5% contamination were excluded, as recommended by CheckM41. We included only publicly available assemblies that could be linked to a published study so that isolate metadata on source (human, animal or environmental) and country could be extracted. The genome accessions and related metadata of curated npmA-harboring assemblies are detailed in Supplementary Data 1. We employed in silico MLST to determine sequence types (STs) from npmA-carrying C. difficile genome assemblies. This was performed using the MLST tool v2.23.0 (https://github.com/tseemann/mlst) and pubMLST database42 for C. difficile. For a higher resolution genotyping, core genome MLST (cgMLST) was determined using chewBBACA v3.3.143. This tool uses a gene-by-gene approach to analyze coding sequences, assigning alleles based on a BLAST Score Ratio (BSR). First, Ridom C. difficile cgMLST schema v2 (encompassing 2147 genes, https://www.cgmlst.org/ncs/schema/Cdifficile971/) was adapted using the PrepExternalSchema module for chewBBACA. Second, the AlleleCall module was used to determine allelic profiles of npmA-positive genomes, applying the default BSR threshold of 0.6. Following allele identification, the ExtractCgMLST module was applied to create a comprehensive cgMLST retaining all loci present in 100% of the strains (--t 1). This resulted in a robust cgMLST profile of 1796 loci. Finally, we constructed a Minimum Spanning Tree (MST) using GrapeTree v1.5.044 and the MSTree method, ignoring missing data (--missing 0), to visualize the genetic relationships among the isolates. After establishing the population structure of npmA-carrying C. difficile isolates, we identified ST clusters as distinct groups of genetically related isolates in the MST assigned to the same ST. To extend our search and systematically identify closely related C. difficile genomes without the npmA gene from the public genome databases originally queried, we identified the common cgMLST alleles shared by all isolates within each ST cluster. Using nucleotide sequences of these shared alleles, we conducted a COBS36 index search of the entire C. difficile dataset, applying a stringent k-mer similarity threshold of 97% to ensure high specificity and recover genomes most similar to the npmA-carrying isolates. Afterwards, we applied the same genomic QC and metadata completeness criteria as for npmA-carrying genomes (see section De novo assembly and Quality Control), ensuring that selected contextual genomes met the same quality standards. The genomes selected through this process served as contextual genomes for the two major clusters, ST11 and ST54 (Fig. The information of contextualized C. difficile assemblies is detailed in Supplementary Data 2. We used the snippy-multi from the Snippy package v4.6.0 (https://github.com/tseemann/snippy) to identify Single Nucleotide Polymorphisms (SNPs) within the assembled C. difficile genomes against specific reference genomes for each cluster: C. difficile M120 (GenBank accession FN665653.1) for ST11 and C. difficile 630 (AM180355.1) for ST54. We then used snippy-core to construct a genome-wide core alignment. ClonalFrameML v1.1245 was applied to this alignment to identify and remove recombinant regions, using the parameter -output_filtered to generate a recombination-masked alignment, which includes only non-recombinant sites. This recombination-free alignment was processed using SNP-sites v2.5.146 to produce an alignment with SNP sites only. This alignment was used as input by IQ-TREE v2.2.647 to construct a maximum-likelihood phylogenetic tree with a general time-reversible (GTR) and gamma substitution model (-m GTR + G), applying 1000 ultra-fast bootstrap replicates (-B 1000). The resulting phylogenetic trees were midpoint rooted and processed with ape v5.748 and visualized with ggtree v3.8.049 packages on R v4.2.2. To further delineate the genomic relationships, pairwise SNP distances between isolate genomes were calculated using snp-dists v0.8.2 (https://github.com/tseemann/snp-dists) using the recombination-free alignment obtained from ClonalFrameML. The identification and extraction of contigs carrying npmA were performed using a custom Python script that searches for a specific DNA sequence in a set of bacterial assembly files. Then, we employed a pipeline designated for the in-depth analysis, contextualization and visualization of genetic flanking regions, targeting npmA (https://github.com/liampshaw/mobile-gene-regions)12. Briefly, this approach used pangraph50 to identify blocks of homologous sequences by iteratively aligning subsets of sequences with minimap2 or mmseqs2. The algorithm identifies stretches of homologous sequences, referred to as pancontigs or homologous blocks, which represent the consensus sequence of the aligned regions. These blocks are then connected to form a graph structure that visualizes the structural diversity of the flanking regions. To further explore the immediate genetic context of npmA, we analyzed a window of 5 kb both upstream and downstream of the gene. In case of longer contigs, we expanded this window to 30 kb, enabling a comprehensive view of the surrounding genetic context. Selected contigs were also annotated using Bakta v1.8.251. Annotated contigs served as the input for subsequent analysis with ICEscreen v1.2.015, which facilitated the detection and annotation of Integrative and Conjugative Elements (ICEs) and Integrative and Mobilizable Elements (IMEs). This comparison was performed using NUCmer v3.152, applying stringent criteria for ICE identification: only those assemblies aligning for ≥98% of their length and demonstrating > 98% identity with a complete ICE were considered as matches. Visual representation of these genetic structures was performed through Clinker on the CAGECAT web server (https://cagecat.bioinformatics.nl/), offering a clear and detailed view of their organization. Afterwards, we identified elements with characteristics similar to known insertion sequences (ISs). To confirm their identity, we conducted a blastp analysis against the ISFinder database53 (https://isfinder.biotoul.fr/, last update: 2023-11-27). This analysis revealed that these sequences did not match any known IS in the database, leading to their recognition as novel IS sequences. Subsequently, ISFinder was used to assign a name to these previously unreported ISs, and their sequences were uploaded to the ISFinder database. New composite elements were named and registered in the Transposon Registry (https://transposon.lstmed.ac.uk/)54. To assess target-site specificity of ICE Tn7740, we extracted 15 bp of chromosomal sequence immediately flanking each of the five C. difficile and the single E. faecium integration sites and aligned the 30 bp windows using MUSCLE v5.3. Then, we visualized the alignment with WebLogo (https://weblogo.berkeley.edu/logo.cgi). Antimicrobial susceptibility testing (AST) for the two npmA2-carrying E. faecium isolates was performed to determine their minimum inhibitory concentrations (MICs) for various aminoglycosides, following the European Committee on Antimicrobial Susceptibility Testing (EUCAST) standards. The selected aminoglycosides included amikacin, apramycin, gentamicin, kanamycin, and neomycin, with concentrations ranging from 1 mg/L to 1024 mg/L in cation-adjusted Mueller-Hinton Broth. E. faecium ATCC 19434 was used as a quality control reference. After incubating at 37 °C for 18 h in an aerobic condition, MICs were assessed. On the other hand, AMR predictions were generated using raw Illumina reads from the two E. faecium isolates and a streamlined WGS data pipeline34 (https://github.com/francesccoll/amr_efm_wgs). Briefly, the pipeline employs ARIBA55 to detect and annotate mutations and resistance genes against a curated database of AMR determinants for E. faecium. In addition, the pipeline uses Snippy for reference-mapped SNP calling to identify nucleotide polymorphisms and complex mutations that ARIBA might miss. This process enables precise resistance prediction by correlating the presence of these specific genes or mutations with known resistance phenotypes. This tool incorporates a curated database of 228 genetic markers involved in resistance to 12 antibiotics in E. faecium, which was evaluated against a dataset of over 4000 E. faecium genomes with phenotypic AST data34. The antimicrobial susceptibility of two npmA2-carrying C. difficile strains was evaluated by ETEST® (BioMérieux) for amikacin, gentamicin and streptomycin, with concentrations ranging from 0.016 mg/L to 256 mg/L (for amikacin and gentamycin) and 1 mg/L to 1024 mg/L (for Streptomycin), using Columbia blood agar plates (BioMérieux). C. difficile 630Δerm was included as a quality control reference. One hundred colonies were streaked on BHI agar, with and without gentamicin (250 mg/L). Colonies from antibiotic-free plates were replicated daily onto both media types for six days. Growth on gentamicin was used to qualitatively indicate ICE retention. The gentamicin-susceptible strain C. difficile 630Δerm was used as a negative control. Triplicate 5 ml BHI cultures, initiated from single colonies, were serially diluted 1:1000 every 24 h. At generations 0, 100, 200, and 300, 100 colonies from BHI plates were screened for apramycin resistance (200 mg/L) to quantify ICE maintenance. To assess npmA2 stability outside of chromosomal context in Gram-positive and Gram-negative bacteria, the npmA2 gene with its native promoter was cloned into shuttle vector pTG262 (removing kanamycin cassette) by overlapping PCR amplification and subsequent assembly using the In-Fusion HD Cloning Kit (Takara). Three different transformant colonies per strain were grown in BHI apramycin (200 mg/L) cultures for 24 h and passaged daily in antibiotic-free BHI for six days (~ 60 generations in E. coli and ~ 40 in E. faecalis). Each day, cultures were plated, and 100 colonies were replicated to determine the percentage retaining apramycin resistance. Single colonies of donor strains (C. difficile SAMEA5846557 and CII7) and recipient strains (C. difficile 630Δerm and E. faecalis JH2-2) were used to inoculate 20 ml BHIS broth and cultured to mid-exponential phase (OD₆₀₀ up to 0.6). Cells were harvested and resuspended in 1 ml BHIS. 100 µl of both the donor and recipient suspensions were mixed and loaded onto 0.45 µm pore size nitrocellulose filters placed on BHIS agar. After 24 h of incubation at 37 °C, filters were removed, washed with 1 ml BHIS, and 100 µl aliquots were plated on BHIS agar supplemented with selective antibiotics. Transconjugants were recovered after 48 h. C. difficile transconjugants were selected anaerobically on plates containing gentamicin and rifampicin. E. faecalis transconjugants were selected aerobically on erythromycin-supplemented plates (method adapted from refs. For intra-enterococcal conjugation assays, E. faecium BM4105RF (rifampicin- and fusidic acid-resistant) was used as the recipient. Donor strains included the apramycin-resistant E. faecium E7474 harboring ICE Tn7740/npmA2 and a vancomycin-resistant E. faecium strain CCP129 harboring plasmid-encoded vanA. Donor and recipient cells were mixed at a 1:1 ratio and incubated on 0.45 μm cellulose nitrate filters for 24 h. Filters were then washed, and serial dilutions were plated on selective media. Transconjugants were selected on BHI agar supplemented with rifampicin (30 mg/L), fusidic acid (25 mg/L), and either apramycin (100 mg/L) or vancomycin (6 mg/L), depending on the transferred resistance marker. All assays were conducted in triplicate. For long-read sequencing, the two npmA2-carrying E. faecium were grown overnight in LB broth at 37 °C. Genomic DNA extraction and purification was conducted using the Wizard Genomic DNA Purification Kit (Promega Corp.), following the Isolating Genomic DNA from Gram-positive Bacteria protocol. Subsequently, DNA concentration was quantified with Qubit (Invitrogen). We prepared the genomic library employing the 1D Genomic DNA by Ligation sequencing kit (SQK-LSK109 and EXP-NBD104). Sequencing run was executed on a FLO-MIN106 R9.4.1 flow cell and run for 36 h using a MinKNOW v4.0.5. The Fast5 files generated were basecalled using Guppy v4.0.9. Hybrid assemblies were performed using Unicycler v0.4.757, applying default parameters. The quality of genomes was assessed with Quast v5.0.240, and complete genomes were annotated with Bakta v1.8.251. To extend our investigation to a broader ecological context, in addition to scanning our initial dataset for the previously described MGEs, we analyzed 147,851 high-quality metagenome-assembled genomes (>90% completeness, <5% contamination) from the Unified Human Gastrointestinal Genome (UHGG) catalog v1.050. We employed BLAST, focusing on ISCld1 (a component of Tn7734) and ICE Tn7740 in the absence of Tn7734 as our primary queries. The detection of ISCld1 was confirmed through strict parameters of 100% coverage and identity, whereas for ICE Tn7740, a slightly more flexible criterion of a minimum 90% coverage and 98% sequence identity was applied. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The Nanopore sequencing reads generated in this study have been deposited in the European Nucleotide Archive (ENA) under BioProject PRJEB75521. All genome assemblies analyzed are available in ENA, NCBI and EnteroBase. The full list of accession numbers is provided in Supplementary Data 1 and 2. The Compact Bit-Sliced Signature Index (COBS) built from 33,253 C. difficile assemblies is available from Zenodo [https://doi.org/10.5281/zenodo.15619548]. Source data are provided in this paper. Krause, K. M., Serio, A. W., Kane, T. R. & Connolly, L. E. Aminoglycosides: An Overview. Comparison of the next-generation aminoglycoside plazomicin to gentamicin, tobramycin and amikacin. Tevyashova, A. N. & Shapovalova, K. S. Potential for the development of a new generation of aminoglycoside antibiotics. & Arakawa, Y. Aminoglycoside resistance: Updates with a focus on acquired 16S ribosomal RNA methyltransferases. Comparative in vitro activity of plazomicin and older aminoglycosides against Enterobacterales isolates; prevalence of aminoglycoside modifying enzymes and 16S rRNA methyltransferases. & Cundliffe, E. Sites of action of two ribosomal RNA methylases responsible for resistance to aminoglycosides. Novel plasmid-mediated 16S rRNA m1A1408 methyltransferase, NpmA, found in a clinically isolated Escherichia coli strain resistant to structurally diverse aminoglycosides. Instability of the 16S rRNA methyltransferase-encoding npmA gene: why have bacterial cells possessing npmA not spread despite their high and broad resistance to aminoglycosides?. & Arakawa, Y. Exogenously acquired 16S rRNA methyltransferases found in aminoglycoside-resistant pathogenic Gram-negative bacteria: an update. Marsh, J. W. et al. Clostridioides difficile: a potential source of NpmA in the clinical environment. Matlock, W. et al. Flanker: a tool for comparative genomics of gene flanking regions. Shaw, L. P. & Neher, R. A. Visualizing and quantifying structural diversity around mobile resistance genes. Diverse sources of C. difficile infection identified on whole-genome sequencing. Lao, J. et al. ICEscreen: a tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures. Arredondo-Alonso, S. et al. Plasmids shaped the recent emergence of the major nosocomial pathogen Enterococcus faecium. Wasels, F. et al. Inter- and intraspecies transfer of a Clostridium difficile conjugative transposon conferring resistance to MLSB. Almeida, A. et al. A unified catalog of 204,938 reference genomes from the human gut microbiome. Evolutionary and genomic insights into clostridioides difficile sequence type 11: A diverse zoonotic and antimicrobial-resistant lineage of global one health importance. Molecular characterization of Clostridium difficile isolates in China From 2010 to 2015. Wen, X. et al. Whole-genome sequencing reveals the high nosocomial transmission and antimicrobial resistance of clostridioides difficile in a single center in China, a four-year retrospective study. Acman, M. et al. Role of mobile genetic elements in the global dissemination of the carbapenem resistance gene blaNDM. Freitas, A. R. et al. Multilevel population genetic analysis of vanA and vanB Enterococcus faecium causing nosocomial outbreaks in 27 countries (1986–2012). Knight, D. R., Androga, G. O., Ballard, S. A., Howden, B. P. & Riley, T. V. A phenotypically silent vanB2 operon carried on a Tn1549-like element in Clostridium difficile. Amy, J., Johanesen, P. & Lyras, D. Extrachromosomal and integrated genetic elements in Clostridium difficile. Snesrud, E. et al. A model for transposition of the colistin resistance gene mcr-1 by ISApl1. Li, R., Chen, K., Chan, E. W.-C. & Chen, S. Characterization of the stability and dynamics of Tn6330 in an Escherichia coli strain by nanopore long reads. A framework for identifying the recent origins of mobile antibiotic resistance genes. Jasni, A. S., Mullany, P., Hussain, H. & Roberts, A. P. Demonstration of conjugative transposon (Tn5397)-mediated horizontal gene transfer between Clostridium difficile and Enterococcus faecalis. The controversial role of human gut Lachnospiraceae. Howden, B. P. et al. Genomic insights to control the emergence of vancomycin-resistant enterococci. Ramin, B., Asadpour, L., Forouhesh Tehrani, H. & Amirmozafari, N. Detection and distribution of various HLAR Gene in Enterococcus faecalis and Enterococcus faecium by Multiplex-PCR. Antibiotic resistance determination using Enterococcus faecium whole-genome sequences: a diagnostic accuracy study using genotypic and phenotypic data. Bordeleau, E. et al. ApmA is a unique aminoglycoside antibiotic acetyltransferase that inactivates apramycin. Bingmann, T., Bradley, P., Gauger, F. & Iqbal, Z. COBS: A compact bit-sliced signature index. In String Processing and Information Retrieval 285–303 (Springer International Publishing, 2019). & Iqbal, Z. AllTheBacteria - all bacterial genomes assembled, available and searchable. Exploring bacterial diversity via a curated and searchable snapshot of archived DNA sequences. & Achtman, M. The EnteroBase user's guide, with case studies on Salmonella transmissions, Yersinia pestis phylogeny, and Escherichia core genomic diversity. Gurevich, A., Saveliev, V., Vyahhi, N. & Tesler, G. QUAST: quality assessment tool for genome assemblies. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. & Maiden, M. C. BIGSdb: Scalable analysis of bacterial genome variation at the population level. Silva, M. et al. chewBBACA: A complete suite for gene-by-gene schema creation and strain identification. Zhou, Z. et al. GrapeTree: visualization of core genomic relationships among 100,000 bacterial pathogens. & Wilson, D. J. ClonalFrameML: Efficient inference of recombination in whole bacterial genomes. Page, A. J. et al. SNP-sites: rapid efficient extraction of SNPs from multi-FASTA alignments. Minh, B. Q. et al. IQ-TREE 2: New models and efficient methods for phylogenetic inference in the genomic Era. Paradis, E. & Schliep, K. ape 5.0: An environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019). ggtree: an r package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Noll, N., Molari, M., Shaw, L. P. & Neher, R. A. PanGraph: scalable bacterial pan-genome graph construction. Schwengers, O. et al. Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification. Marçais, G. et al. MUMmer4: A fast and versatile genome alignment system. & Chandler, M. ISfinder: the reference centre for bacterial insertion sequences. Hunt, M. et al. ARIBA: rapid antimicrobial resistance genotyping directly from sequencing reads. Farrow, K. A., Lyras, D. & Rood, J. I. Genomic analysis of the erythromycin resistance element Tn5398 from Clostridium difficile. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads. Carlos Serna is supported by a position as Graduate Teaching Assistant at Complutense University of Madrid (0312/AYU/021). We thank all members of the Antimicrobial Resistance Unit (ARU) for their contributions to the discussion of our results. We thank Frederic Barbut of the National Reference Laboratory for C. difficile, Hôpital Saint-Antoine, Paris, France, and Julie Germond and Jean Sautereau of the National Reference Laboratory of anaerobes and botulism for their contribution to determining CMI of aminoglycosides, and Pascale Serror for E. faecalis JH2-2 strain used in this study. We are grateful to Luísa Peixe, Ana R. Freitas, Bárbara Duarte and Carla Novais from UCIBIO, Porto, Portugal, and to the Culture Collection of Porto (CCP, https://ccp.ff.up.pt/) for providing the E. faecium CCP129 strain carrying a conjugative vanA plasmid used as a positive control in conjugation experiments. We also thank Dr. Christian Seyboldt from the Institute for Bacterial Infections and Zoonoses (IBIZ), Jena, Germany, for providing the C. difficile strains carrying npmA for use in our experiments, and Juan Miguel Rodríguez, from the Department of Nutrition and Food Science of the Universidad Complutense de Madrid, Spain, for lending us different Enterococcus strains for the stability and conjugation experiments. The funder had no role in study design, data collection, interpretation of results, or the decision to submit the work for publication. Present address: Biodiversity and Epidemiology of Bacterial Pathogens, Institut Pasteur, Université Paris-Cité, Paris, France These authors contributed equally: Bosco R. Matamoros, Mario Pulido-Vadillo. Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty of Veterinary Medicine and VISAVET Health Surveillance Centre, Complutense University of Madrid, Madrid, Spain Carlos Serna, Bosco R. Matamoros, Mario Pulido-Vadillo, Jose F. Delgado-Blas & Bruno Gonzalez-Zorn Department of Medical Microbiology, Onze Lieve Vrouwe Gasthuis, Amsterdam, Netherlands Parasites and Microbes Programme, Wellcome Sanger Institute, Hinxton, UK Pathogenesis of Bacterial Anaerobes Laboratory, Department of Microbiology, Institut Pasteur, Université Paris-Cité, Paris, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to conceptualization, methodology, software, validation, formal analysis, investigation, data curation, writing the original draft and visualization. carried out the phenotypic experiments and was involved in the investigation and methodology. performed the stability and conjugation experiments. aided in conceptualization, drafted and reviewed the manuscript, and engaged in scientific discussions. also providing critical feedback on the manuscript draft. A.A. performed the metagenomic analyses and contributed to writing, review & editing. provided supervision, discussed results, and contributed to writing, review & editing. was involved in conceptualization, resources, data curation, supervision, methodology, and writing, review & editing. provided supervision, contributed to writing, review & editing, project administration and funding acquisition. All authors read and commented on successive drafts and approved the final version of the manuscript. Correspondence to Francesc Coll or Bruno Gonzalez-Zorn. The authors declare no competing interests. Nature Communications thanks Mathew Beale, and the other anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Global dissemination of npmA mediated pan-aminoglycoside resistance via a mobile genetic element in Gram-positive bacteria. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250716000854.htm'>It's not that you look—it's when: The hidden power of eye contact</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 06:21:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For the first time, a new study has revealed how and when we make eye contact -- not just the act itself -- plays a crucial role in how we understand and respond to others, including robots. They discovered that the most effective way to signal a request was through a specific gaze sequence: looking at an object, making eye contact, then looking back at the same object. This timing made people most likely to interpret the gaze as a call for help. Dr Caruana says that identifying these key patterns in eye contact offers new insights into how we process social cues in face-to-face interactions, paving the way for smarter, more human-centered technology. "We found that it's not just how often someone looks at you, or if they look at you last in a sequence of eye movements but the context of their eye movements that makes that behavior appear communicative and relevant," says Dr Caruana, from the College of Education, Psychology and Social Work. The authors say the research can directly inform how we build social robots and virtual assistants that are becoming ever more ubiquitous in our schools, workplaces and homes, while also having broader implications beyond tech. "Understanding how eye contact works could improve non-verbal communication training in high-pressure settings like sports, defense, and noisy workplaces," says Dr Caruana. "These subtle signals are the building blocks of social connection," says Dr Caruana. Acknowledgements: Authors were supported by an Experimental Psychology Society small grant. Earth Could Be in a Massive Cosmic Bubble That's Warping the Universe NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250716000851.htm'>Corals in crisis: A hidden chemical shift is reshaping Hawaiian reefs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 03:53:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The study helps researchers, conservationists and policymakers understand the future challenges facing Hawaiian coral reefs and provides information for preserving these critical ecosystems for future generations. In the high‐emission scenario, the team found that ocean chemistry will become dramatically different from what corals have experienced historically, potentially posing challenges to their ability to adapt. The team calculated the difference between projected ocean acidification and acidification that corals in a given location have experienced in recent history. They refer to this as 'novelty' and discovered that various areas of the Hawaiian Islands may experience acidification differently. Windward coastlines consistently exhibited higher novelty, that is, future conditions deviate more dramatically from what coral reefs have experienced in recent history. "We did not expect future levels of ocean acidification to be so far outside the envelope of natural variations in ocean chemistry that an ecosystem is used to," said Tobias Friedrich, study co-author and research scientist in the Department of Oceanography. "This is the first ocean acidification projection specifically for Hawaiian waters to document that." Previous studies have shown that a coral that is exposed to slightly elevated ocean acidity can acclimatize to those conditions, thereby enhancing the coral's adaptability. This is why we continue new research to examine the combined effects of stresses on corals," said Powell. "This study is a big first step to examine the totality of changes that will impact corals and other marine organisms and how it varies around the islands." The research team will continue to investigate the future changes in Hawaiian waters, specifically, heat stress, locations of possible refugia for coral reefs, and changes to Hawai'i's fisheries. Earth Could Be in a Massive Cosmic Bubble That's Warping the Universe NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-62015-2'>Small RNA in sperm–Paternal contributions to human embryo development</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-17 00:37:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Sperm not only delivers the paternal genome to the oocyte but also regulatory small RNA (sRNA). However, the role of sRNA in fertilisation and human embryo development remains poorly understood. Here, couples undergoing IVF are recruited, and sperm sRNA analysed to investigate their role in IVF treatment. Differential expression of mitochondrial sRNA and Y-RNA are observed in relation to sperm concentration. For fertilisation rate, sRNA from a single locus are significantly changed. Expression of microRNA (miRNA) and ribosomal sRNA correlates positively and negatively, respectively, to high-quality embryos. Notably, the top miRNA have an area under ROC of >0.8. Predicted targets of these miRNA are relevant for development, suggesting a role for sperm-borne miRNA in embryo development. In conclusion, sperm-borne sRNA are biomarkers for sperm concentration and embryo quality in IVF. These findings may contribute to clinical strategies improving embryo quality, lowering costs and reducing the need for additional treatment cycles. Infertility is increasing globally, affecting one in six couples of reproductive age1. To address this issue, researchers have developed assisted reproductive technologies, such as In Vitro Fertilisation (IVF), resulting in about 12 million babies being born2. Despite being well-established, IVF is only successful in ~30% of cases3. One major limiting factor in modern IVF treatments is embryo development. IVF clinics often aim to have 30% of fertilised oocytes develop into high-quality embryos, although this success rate is not reached consistently4. Often, couples need several rounds of treatments to receive a baby5. In this context, the potential for male contributing factors to increase IVF success has remained unexplored. Accumulating evidence suggests that the sperm delivers not only genetic material but also small RNA (sRNA) to the oocyte. By definition, sRNA are short RNA molecules ranging from tens to a few hundreds of nucleotides. sRNA include microRNA (miRNA), originally discovered for their regulatory role in embryogenesis, as well as tRNA-derived fragments (tsRNA), ribosomal RNA-derived fragments (rsRNA), mitochondrial-derived RNA (mitosRNA), Piwi-interacting RNA (piRNA), and fragments from ribonucleoprotein associated RNA (ribonucleoprotein associated sRNA) such as Y-RNA and small nucleolar RNA6. Here, we show that by analysing sRNA in sperm samples retrieved during ongoing IVF treatment, we find subpopulations of sRNA to be biomarkers for sperm concentration, fertilisation rate, and embryo quality. Our work highlights the paternal role in IVF success and establishes a foundation for a new era in reproductive medicine. Couples undergoing IVF were recruited between the 18 November 2022 and 8 June 2023 at the Centre of Reproductive Medicine at University Hospital in Linköping, Sweden. Of the 84 couples assessed for eligibility, 72 were recruited and pseudonymized. One couple had an inadequate sperm count for sRNA sequencing and was excluded after sample collection. Two couples did not retrieve any oocytes and were excluded from further analysis. One couple provided two samples because they underwent two IVF treatments; both samples were included. Thus, 70 treatments from 69 couples were studied, with sperm samples sequenced from each of these treatments (Fig. The median age of the men was 34 years, and their median BMI was 27. The median age of the women was 33, and their median BMI was 25. To investigate whether there are molecular markers capable of predicting IVF outcome, leftover sperm was collected and sequenced. In total, 70 samples from 69 couples were sequenced for sRNA. Analysis of this data confirms previous findings that sperm contains a large and diverse repertoire of sRNA7,8 (Supplementary Data 1, and Supplementary Fig. We performed differential expression analysis on sequenced sRNA and identified sRNA differing by sperm concentration, fertility rate, and rate of high-quality embryos. Specifically, differential expression analysis comparing samples with high ( >16 million sperm/mL) and low ( ≤16 million sperm/mL) sperm concentration (Supplementary Data 2) identified 563 (1.89%) significantly upregulated sRNA in samples of high concentration (Fig. Conversely, 640 (2.15%) sRNA were significantly downregulated. As sperm concentration and motility are used to determine sperm quality, we performed differential expression analysis of these two parameters. We identified a great overlap between significant sRNA in sperm concentration and sperm motility ( ≥5 million progressively motile sperm as high and <5 million sperm as low motility), where 48% of all differentially expressed sRNA were identical between the two parameters (Supplementary Fig. a Comparison of high and low sperm concentration (>/≤16 million sperm/mL). Pie charts represent up and downregulated sequences respectively, where the percentage of normalised sequences per biotype is shown. b Comparison of high and low fertilisation rate (≥/< 70%). Pie charts represent up and downregulated sequences respectively, where the percentage of normalised sequences per biotype is shown. Pie charts represent up and downregulated sequences respectively, where the percentage of normalised sequences per biotype is shown. Ribonucleoproteins include SRPs, vault-RNA, snRNA, snoRNA, scaRNA and Y-RNA. Upregulated are sequences with an adjusted P value of <0.1 and a log2 fold change above 1, whereas downregulated are sequences with an adjusted P value of <0.1 and a log2 fold change below -1. Adjusted P values and log2 fold changes shown in a, b and c were all produced by differential expression analysis with DESeq2 as described in the “Methods section”. Up and downregulated RNA are marked blue in volcano plots. Next, comparing sRNA from sperm associated with high ( ≥70%) and low ( <70%) fertilisation rates (Supplementary Data 4), we identified 34 sequences (0.11%) to be downregulated in samples of high fertilisation rate (Fig. Of these, 39% were piRNA, 34% unannotated, and 27% tsRNA (Fig. We found that no sRNA were significantly associated with high fertilisation rate. Lastly, when comparing sperm producing high ( ≥20%) respectively low ( <20%) rates of high-quality embryos (Supplementary Data 6), 60 sRNA (0.20%) were upregulated and 104 sRNA (0.35%) were downregulated (Fig. Upregulated sRNA consisted of 66% miRNA, and downregulated sRNA consisted of 73% rsRNA (Fig. Thus, differential expression analysis of sperm-borne sRNA identified unique RNA profiles associated with sperm concentration, fertilisation, and embryo quality. Next, we investigated sRNA differentially expressed in sperm concentration. Therefore, we mapped differentially expressed sRNA to their genes of origin and will hereafter name sRNA according to their annotation. Since the most prominent sRNA biotype upregulated in sperm concentration originated from the mitochondrial genome (Fig. 2a), we investigated these mitosRNA further. All upregulated mitosRNA originated from mitochondrial tRNA genes (Supplementary Fig. Of these genes, the three most significant were MT-TQ-Glu, MT-TH-His, and MT-TS1-Ser1 (Fig. Further examination with linear regression of sperm concentration to levels of sRNA mapping to MT-TS1-Ser1 showed a positive significant correlation (R2 = 0.208, P ≤ 0.0001) (Fig. Additionally, the potential of MT-TS1-Ser1 to differentiate a low or high sperm concentration was assessed by performing a receiver operating characteristic (ROC) analysis, where the area under curve (AUC) was 0.891 (Fig. a MitosRNA with highest significance between high and low sperm concentration (>/≤16 million sperm/mL). P values were produced by Wilcoxon nonparametric test (two-sided). b Linear regression of MT-TS1-Ser1 comparing sum cpm and sperm concentration (R² = 0.208, P value = <0.0001, equation= Y = 0,01862*X + 15,83). c ROC of sum cpm of MT-TS1-Ser1 to differentiate between high and low sperm concentration. d Ribonucleoprotein associated sRNA with highest significance between high and low sperm concentration. P values were produced by Wilcoxon nonparametric test (two-sided). e Linear regression comparing RNY4 and sperm concentration (R² = 0.238, P value = <0.0001, equation= Y = −0,0004823*X + 45,23). f ROC of sum cpm of RNY4 to differentiate between high and low sperm concentration. Data is presented as sum of normalised sequences (cpm) for each sperm sample by indicated genomic origin. P values for Wilcoxon nonparametric t-test, *≤0.05, **≤0.001, ***≤0.0001. Line in box plot represents the median, hinges show the first and third quartiles and whiskers extend to the largest value unless values are above 1.5 times the inter-quartile range. In linear regression, coloured area represents 95% confidence interval. The top three most significant all mapped to Y-RNA (Fig. Of these, sRNA mapping to RNY4 was the most significantly changed. Linear regression of sperm concentration and sRNA mapping to RNY4 showed a significant negative correlation (R2 = 0.238, P ≤ 0.0001, Fig. The AUC was 0.845 in this ROC analysis (Fig. Thus, we identified mitosRNA and ribonucleoprotein associated sRNA from Y-RNA as biomarkers that differentiate between sperm samples with high or low sperm concentrations. We identified 34 sequences of sperm-borne sRNA to be associated with low fertilisation rate (Fig. A closer analysis of these sequences revealed that they all originate from the same genomic region (Fig. Sequences from this region can be annotated as both tRNA and piRNA depending on database. It is therefore not possible to precisely classify the differentially expressed sequences found here. Comparing the sum of normalised sequences in samples with high or low fertilisation rate did not show a statistically significant difference (Fig. In addition, ROC analysis showed an AUC of 0.5817 (Fig. In all, our data suggests that sequences from this specified region may be relevant for the sperm's capability of fertilisation, but more data would be needed to strengthen this finding. a Mean cpm of high and low fertilisation rate (≥/< 70%) of differentially expressed sequences from Fig. Genomic region of chromosome 1, 146,035,680−146,035,770 from reference GRCh38. Annotation was extracted from IGV with RNAcentral genomic coordinates data version 24.0 (lower panel). b Sum cpm of all differentially expressed sequences in high and low fertilisation rate. P values as shown in figure were produced by Wilcoxon nonparametric test (two-sided). c Linear regression comparing sum cpm of differentially expressed sequences and fertilisation rate (R² = 0.01918, P value = 0.2529, equation= Y = −0,0003943*X + 65,25). d ROC of sum cpm of all differentially expressed sequences to differentiate between low and high fertilisation rate. Data is presented as sum of normalised sequences (cpm) for each sperm sample by indicated genomic origin. Line in box plot represents the median, hinges show the first and third quartiles and whiskers extend to the largest value unless values are above 1.5 times the inter-quartile range. In linear regression, coloured area represents 95% confidence interval. Concerning the rate of high-quality embryos, miRNA was the most prominent biotype of upregulated sRNA. Specifically, 16 miRNA were found (Supplementary Fig. 4a); the three most statistically significant were hsa-let-7g, hsa-miR-30d, and hsa-miR-320b/a (Fig. Gene Ontology (GO) term analysis of the predicted targets for these three miRNAs showed several biological processes to be involved. Interestingly, nine of the top GO terms were related to embryogenesis, development, or cell proliferation (Supplementary Fig. This finding suggests that these miRNA play important roles in developmental processes. Further investigation of the miRNA revealed that hsa-let-7g had the lowest P value (P = 0.000000005.4, Fig. 5a), making it an interesting candidate for further investigation. Comparing hsa-let-7g expression against the rate of high-quality embryos showed no significant regression (R2 = 0.001, p = 0.79, Fig. However, exclusion of one sample with an expression 12 times above mean, resulted in a significant regression (Supplementary Fig. Moreover, an AUC of 0.812 for the ROC curve (Fig. 5c) supports the notion that hsa-let-7g affects embryo quality outcomes. Additionally, we studied linear regression and ROC analysis for hsa-miR-30d (Supplementary Fig. 4d) to address the sensitivity of miRNA for embryo quality. Regression was significantly positive (R2 = 0.065, p = 0.04), and AUC was 0.712, suggesting that these miRNA are robust biomarkers of the ability of the sperm to produce high-quality embryos. a MiRNA with highest significance between high and low rate of high-quality embryos (≥/< 20%). P values as shown in figure were produced by Wilcoxon nonparametric test (two-sided). b Linear regression comparing hsa-let-7g in sum cpm and rate of high-quality embryos (R² = 0.001, P value = 0.7903, equation= Y = -0,1167*X + 33,97). d RsRNAs with highest significance between high and low rate of high-quality embryos. P values as shown in figure were produced by Wilcoxon nonparametric test (two-sided). e Linear regression comparing 28 s rRNA in sum cpm and rate of high-quality embryos (R² = 0.0837, P value = 0.0194, equation=Y = −0,009374*X + 39,65). f ROC of sum cpm of 28 s rRNA to differentiate between high and low rate of high-quality embryos. P values for Wilcoxon nonparametric t-test, *≤0.05, **≤0.001, ***≤0.0001. Data is presented as sum of normalised sequences (cpm) for each sperm sample by indicated genomic origin. High rate of high-quality embryos n = 47, low rate of high-quality embryos n = 17. Line in box plot represents the median, hinges show the first and third quartiles and whiskers extend to the largest value unless values are above 1.5 times the inter-quartile range. In linear regression, coloured area represents 95% confidence interval. Additionally, downregulated sRNA were identified as mostly rsRNA. RsRNA originating from ribosomal genes 28S, 5S, 5.8S, and 12S were associated with sperm giving rise to low rates of high-quality embryos (Fig. Of these, rsRNA from the 28S gene were the most significantly changed between the high and low rate of high-quality embryos. Linear regression against sRNA mapping to 28S rRNA and rate of high-quality embryos showed a significant negative correlation (R2 = 0.084, P = 0.02), and AUC was 0.79 (Fig. We found that miRNAs and sRNA from 28S were sperm-borne biomarkers that predict embryo quality in IVF treatment. From the 65 treatments with successful fertilisation, 28 children were born (Fig. To investigate whether there are molecular markers capable of predicting live birth, gestational age, and size for gestational age, we performed differential expression of sperm sRNA and these parameters (Supplementary Fig. We identified a few sRNA to be upregulated in sperm associated with live birth (Supplementary Fig. These sequences originated from the snRNA U6 (Supplementary Fig. Regarding gestational age, there was no association with sperm-borne sRNA (Supplementary Fig. Next, looking at size for gestational age, we identified a subset of tsRNA (Supplementary Fig. These sequences mapped to four tRNA, Arg-CCT-3−1, iMet-CAT−1−1, Glu-TTC−14−1 and Lys-CTT-2−1 (Supplementary Fig. Sequences aligning to the same tRNA was summed and plotted as a function of size for gestational age. Three of these tRNA showed a negative correlation to size for gestational age (R2 between 0.05811 and 0.1879), suggesting that these sequences relate to foetal growth (Supplementary Fig. When grouping the samples in sperm giving rise to babies small for gestational age (SGA), average for gestational age (AGA), large for gestational age (LGA) and performing a non-parametric Kruskal-Wallis, there was no significant difference between the groups (Supplementary Fig. While the negative correlation between tsRNA and size for gestational age is an interesting observation, the few data points in SGA (n = 2) and LGA (n = 5), indicate that more data is needed to support this observation. We provide evidence that sperm-borne sRNA are biomarkers for sperm quality, fertility rate, as well as embryo quality in an IVF setting. Our data reveal specific subpopulations that are relevant throughout IVF treatment. These findings provide intriguing new insights into the contributions of sperm in embryo development. We identified mitosRNA to be markers of sperm quality in couples undergoing IVF (Fig. In a previous study, we found that sperm-borne mitosRNA is affected by a short-term diet intervention7 and directly related to human sperm quality9. Thus, there is accumulating evidence that mitosRNA are markers for high sperm concentration and motility. A recent investigation has shown that sperm-borne mitosRNA in mice, as in humans, are sensitive to a short-term diet intervention, suggesting that there are evolutionary conserved mechanisms in nutrient sensing capabilities10. Further prospective studies, such as IVF cohorts, are needed to investigate whether a similar mechanism is involved in humans. Notably, we found that high levels of mitosRNA correlate with sperm concentration but not with fertilisation rate and embryo quality. This finding suggests that in an IVF setting sperm-borne mitosRNA are not important for embryonic development or fertilisation. However, sperm-borne mitosRNA may have functional relevance in in vivo fertilisation, where progressive motility in sperm is required to fertilise the egg. Examining the role of sperm-borne sRNA in embryo development, we found 16 miRNA to be relevant (Supplementary Data 7, 8, and Supplementary Fig. GO term analysis of predicted targets from the top three miRNA suggests a role for these miRNA in embryogenesis, development, and cell proliferation (Supplementary Fig. Previous studies have reported that several sperm-borne miRNA are important for male reproductive health11,12,13,14,15,16, embryo quality17,18, as well as likelihood of live birth19. In a recent meta-analysis, Joshi et al.20 concluded that three miRNA are relevant biomarkers for male infertility. Here, we found one of these to be a predictor of low rates of high-quality embryos, supporting the relevance of this miRNA for embryo quality (Supplementary Data 11). Similarly, Hamilton et al.21 found sperm-borne miRNA to be related to the rate of blastocyst formation. Of these, we identified miR-200b and let-7a to be associated with low sperm concentration and miR-200b to be associated with low fertility rates. Of the 16 miRNA identified in this study, three are members of the let-7 family (Supplementary Data 8). This miRNA family is well-established as a conserved family that is crucial for diverse aspects of embryogenesis. Previous work suggests that let-7 regulates embryonic and postnatal growth in mice via the insulin-PI3K-mTOR pathway22. Huang et al.23 found that rodent sperm-borne let-7 dictates the metabolic phenotype in offspring. Furthermore, they found that weight loss in humans reduces levels of let-7d/e in sperm. Thus, our results combined with mentioned findings support a functional role for sperm-borne miRNA. From our data, however, we cannot infer whether the level of miRNA in human sperm is causal to embryo quality or is a consequence of other processes in sperm. In mice, it is clear that sperm-borne miRNA are pivotal for preimplantation embryonic development24. In humans, however, functional validation studies would be required to provide further evidence. It is evident that sperm miRNA, such as let-7g and mir-30d, are promising biomarkers that could predict whether a sperm sample will result in high-quality embryos in IVF treatment (Fig. Notably, the let-7 family holds prognostic value in other contexts. In terms of fertility, let-7c is important for blastocyst formation25, pregnancy loss26, and azoospermia27 and let-7i is relevant for PCOS28, IVF success rate29, endometriosis30, and male subfertility31. Lastly, let-7g has been associated with ovarian response to hormonal stimuli29 and foetal alcohol spectrum disorders32. In addition to mitosRNA and miRNA, we identified several other relevant subtypes of sRNA for IVF. We found that sRNA originating from Y-RNA – particularly RNY1, RNY3, and RNY4—are associated with low-quality sperm. While sRNA from Y-RNA has previously been reported to be present in human sperm33, we found that sRNA from Y-RNA is also relevant for male reproductive health. Full-length Y-RNA are components of the Ro60 ribonucleoprotein particle, involved in DNA replication and RNA quality control34. In addition, our analysis supports the relevance of sperm-borne rsRNA for embryo quality, where specific sequences of 28S rRNA exhibit a lower expression in sperm samples producing high-quality embryos (Fig. In mice, sperm-borne rsRNA have been found to share biogenesis pathways with tsRNA via Dnmt235 as well as being phthalate-sensitive36, but so far their roles in fertility have not been evaluated. Regarding fertilisation rate, we find sequences of piRNA or tsRNA originating from the same locus to be differentially expressed, suggesting a potential intriguing role for this locus in male fertility. We also find other sperm-borne tsRNA, namely from Arg-CCT-3−1, iMet-CAT-1-1, Glu-TTC-14-1 and Lys-CTT-2-1 to have a trend of negative correlation with size for gestational age. In mouse, sperm-borne tsRNA has been related to offspring body weight37. However, in a human setting, these findings need to be further explored in larger cohorts since foetal growth as well as fertilisation is heavily dependent on female factors before and after conception. Nonetheless, Oluwayiose et al.38 identified two sperm-borne piRNAs to be relevant for the chance of live birth. Additionally, Chen et al. and Hua. et al. also showed sperm-borne tsRNA to be relevant for early embryo cleavage events and blastocyst formation39,40, highlighting that both piRNA and tsRNA are highly relevant and interesting in the case of human reproduction. One limitation in our analysis is the difficulty of examining the isolated effects of sperm sRNA on clinical outcome. Specifically, in analyses concerning rate of high-quality embryos, female factors influence the outcome. Additionally, within the studied population, positive correlations exist between females and males in both age and BMI. However, despite the female influence, we are still able to associate sperm-borne sRNA with embryonic quality, supporting their robustness in this system. As embryo quality is one of the major limiting factors of modern IVF success, finding sperm-borne biomarkers capable of predicting embryo quality holds great potential. Couples often undergo several demanding IVF treatments before a successful pregnancy. Additionally, some couples are left without a diagnosis or explanation after failed IVF attempts, causing emotional distress. A more efficient selection of sperm can attain more high-quality embryos, alleviating the number of hormonal treatments needed, aiding in closing the inequality gap in reproductive medicine. In conclusion, we identified different subpopulations of sRNA in sperm that exhibit distinct profiles in clinical reproductive measurements, such as sperm concentration, fertilisation rate, and the rate of high-quality embryos. Furthermore, our analysis identifies specific sRNA as highly promising biomarkers. These biomarkers are clinically applicable for the examination of sperm samples while providing insight into possible molecular mechanisms of male fertility and paternal contribution to embryonic development. Couples were recruited at the Centre for Reproductive Medicine at the University Hospital in Linköping, Sweden, on the day of ovum pick-up (OPU), the same day as semen sample collection and IVF fertilisation. As couples share an IVF record, written consent was obtained from both partners. The sex of participants was collected from their medical records. Couples did not receive any compensation for enroling. Male participants had to meet the following criteria: the presence of mature sperm in the ejaculate and proficiency in Swedish. Males were excluded if they had a known malignant disease, a history of radiation in the pelvic area, previous chemotherapy, or a known genetic anomaly. Importantly, only males with over 600,000 leftover sperm were sequenced due to a restriction in input RNA level. Therefore, males with low sperm count requiring intra-cytoplasmic sperm injection could not be included. Patients received standard care for IVF, where the only laboratory operation different from non-recruited couples was the retrieval of leftover sperm not used in the fertilisation process. On the day of OPU, the semen samples were collected (at home or the clinic between 6:30–9:30) in sterile 50-mL non-spermotoxic polypropylene Falcon tubes (Corning Science, Mexico) after recommendation of ejaculation every other day during hormonal stimulation (8–14 days) and a maximum of 3 days abstinence. After liquefaction for 15–30 min at room temperature on a Nutating Mixer (VWR, Pennsylvania, USA), standard semen parameters were examined manually by embryologists. Parameters for concentration of sperm, concentration of progressively motile sperm and percentage of progressive motile and motile sperm in the ejaculate were evaluated according to WHO criteria41. Internal controls are performed every six months. Most samples were examined within 60 min post ejaculation and none later than 120 min. Spermatozoa were prepared for IVF using PureSperm gradient (1.5 mL 80%/1.5 mL 40%) (Nidacon, Gothenburg, Sweden). Semen was layered on top of the gradient and centrifuged at 300 g for 20 min, followed by resuspension of the pellet (with PureSperm Wash; Nidacon). Subsequently, sperm was centrifuged at 500 g for 10 min, the pellet was resuspended, and the number and concentration of progressive motile sperm post gradient were estimated (Makler counting chamber, Cellvision, Heerhugowaard, Netherlands). From this post-gradient suspension, the sperm sample was prepared and diluted with G-IVF (VitroLife, Gothenburg, Sweden) for fertilisation. Leftover sperm from the sample was used for sRNA sequencing. After fertilisation on the day of OPU (between 13:30 to 15:30), leftover sperm was diluted with G-IVF (VitroLife) or concentrated (centrifugation on 500 g for 10 min and then fluid removal) to a concentration of 12 million sperm/mL. Aliquots of 50 µl (i.e., 600,000 sperm) were put in RNAse-free 2-mL Eppendorf tubes (Eppendorf, Germany), followed by immediate freezing in liquid nitrogen. RNA extraction was performed with miRNeasy Micro kit (Qiagen; 217084) according to the manufacturer's instructions. Prechilled Steel beads (stainless steel bead 5 mm, Qiagen; 69989) were added to frozen samples, followed by the addition of 1 mL of prechilled Qiazol (Qiagen). Samples were instantly processed in a Tissue Lyser LT (Qiagen) for 2 + 2 min at 40 oscillations/s upon defrosting. RNA concentration was determined with a Qubit microRNA assay kit (Q32880, Life Technologies) using Quantus Fluorometer (E6150; Promega, Madison, WI). RNA quality was controlled by BioAnalyzer 2100 (Agilent RNA 6000 Nano kit; 5067-1511), both by extracting RIN values and by visual inspection of the curves. The RIN values were low (2,3 -5,4), with no visible rRNA peaks indicative of pure sperm samples42,43,44. Library synthesis used 4 ng of RNA; if lower, full volume was used. Small RNA libraries were produced with NEBNext Multiplex SmallRNA Library Prep Kit for Illumina (E7300S, E7330S, E7580S; New England Biolabs), with the customisation of a dilution of primers 1:8. The 3´adaptor ligation reaction was carried out at 16 °C overnight. Libraries were amplified for 15 cycles and cleaned using Agencourt AMPure XP (Beckman Coulter, Brea, CA). The quality of cDNA libraries was studied using BioAnalyzer 2100 with Agilent High Sensitivity DNA kit, 5067-4626. Amplified libraries were pooled in relation to peak size before size selection. Size selection was done using TBE gel (EC6265BOX; Invitrogen) of 130–165 nt length. The extraction of cDNA from gel was performed with Gel Breaker Tubes (3388-100, IST Engineering) by incubation with buffer included in the NEBNext kit and incubated on a shaker for 1 h at 37 °C, flash frozen for 15 min, and again incubated on a shaker. Gel debris was removed by Spin-X 0.45 µm centrifuge tubes (8162, Corning, Inc.). Precipitation was done using GlycoBlue (Invitrogen), 0.1 times the volume of Acetate 3 M (pH5.5) and three times the volume of 100% ethanol at –70 °C overnight. The final cDNA concentration was determined using a Quantus Fluorometer (E6150; Promega, Madison, WI) and QuantiFluor ONE ds DNA system (E4870, Promega). The quality of pooled and size selected cDNA libraries was studied again with BioAnalyzer and sequenced on NextSeq 550 with NextSeq 500/550 High Output kit version 2.5, 75 cycles (Illumina, San Diego, CA). All libraries passed Illumina's default quality control. Data analysis was performed within R version 4.3.2 with Seqpac version 1.2.01, mapping to human genome GRCh38. Any sequence that did not match the human genome with a maximum of 3 mismatches was discarded. Primary filters of 1 read in 90% of samples and a length of 16–75 nt were performed before normalisation by counts per million (cpm). Small RNA was annotated against ensemble ncRNA GRCh38.ncrna, piRNA from piRbase 2.0, protein coding from GRCh38 from NIH, and ribosomal RNA from RNAcentral, and searching for rRNA in Homo sapiens. Biotypes were annotated in a hierarchy as follows: rRNA, miRNA, mitochondrial RNA, tRNA, ribonucleoproteins (including SRPs, vaultRNA, snRNA, snoRNA, scaRNA, and Y-RNA), lncRNA, miscRNA, and piRNA. Biotypes were mapped with a mismatch of 1, where sequences failing to annotate to any of the previously defined databases were classified as “unannotated”. Data visualisation was performed with ggplot2 version 3.5.0. Differential expression analysis was performed with DESeq22 version 1.42.1, with the model accounting for flow cell (technical batch effect), as this is the major driver for variation in data when explored for with R package variancePartition version 1.32.5 and Principal Component Analysis performed with Seqpac (Supplementary Fig. In gestational age and size for gestational age, differential expression analysis was performed as continuous variables. Wald test and local regression were used for differential expression. Linear regression and normality check of cpm data were performed GraphPad Prism ver. Results from tests for each RNA shown in boxplots may be found in Supplementary Data 12. Non-parametric Wilcoxon Rank Sum and Signed Rank Tests were two-sided and were performed with R function wilcox.test in stats version 4.3.2. Clinical parameters from the couple's IVF records were retrieved, forming groups for analysis. Males with ≤16 million sperm/mL were classified as having low sperm concentration. This cut-off was based on WHO guidelines41. 2 and 7, ≥5 million progressively motile sperm in total amount after preparation with density gradient was considered as high sperm motility, <5 million sperm as low motility. 7,3 30% motile sperm before density gradient was considered high motility percentage and <30% as low motility. In line with The Vienna Consensus4 for fertilisation rate (n fertilised oocytes/n oocytes retrieved) and benchmark values for the Centre of Reproductive Medicine Linkoping, Sweden, the ratio of ≥70% was set as a high fertilisation rate. Embryologists examined the embryos through microscope with criteria mentioned below. For cleavage stage embryos (days 2 and 3), the guidelines from the Istanbul consensus were used45. At day 2, High-quality embryos were defined as embryos with 4–6 evenly sized blastomeres with a maximum of 20% fragmentation and a smooth cytoplasmic appearance. At day 3, embryos with 6–10 evenly sized blastomeres with a maximum of 20% fragmentation and a smooth cytoplasmic appearance were defined as high-quality. High-quality embryos were considered to have a grade of 3BB or higher46. Couples with a high rate of high-quality embryos had a ratio of ≥20% (n high-quality embryos/ n fertilised oocytes*100). 6, size of gestational age is calculated by neonatal weight charts resulting in z-scores corresponding to centiles47. Small for gestational age (SGA) is defined as <10th centile. Average for gestational age (AGA) is defined as 10-90th centile. Large for gestational age (LGA) is defined as <90th centile. These calculations were performed with web tool from fetalmedicine.com. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. This study involves human participants and is covered by an ethical permit granted by the Swedish Ethics Board under number 2022-00244-01. The sequencing data are available under restricted access for ethical permit and data privacy law reasons, access can be obtained by extension of ethical permit by contacting anita.ost@liu.se. Response time after mail contact is expected within weeks and data will be available as long as needed for purpose specified in extended ethical permit. The processed data generated in this study are provided in the Supplementary material. The code used in this study is available at github.com/signeisacson/KIPF. Cox, C. M. et al. Infertility prevalence and the methods of estimation from 1990 to 2021: a systematic review and meta-analysis. Adamson, G. D. et al. How many infants have been born with the help of assisted reproductive technology? Kupka, M. S. et al. International Committee for Monitoring Assisted Reproductive Technology world report: assisted reproductive technology, 2015 and 2016. ESHRE Special Interest Group of Embryology and Alpha Scientists in Reproductive Medicine. The Vienna consensus report of an expert meeting on the development of ART laboratory performance indicators. Smith, A. D. A. C., Tilling, K., Nelson, S. M. & Lawlor, D. A. Live-birth rate associated with repeat in vitro fertilisation treatment cycles. Male reproductive health and intergenerational metabolic responses from a small RNA perspective. Nätt, D. et al. Human sperm displays rapid responses to diet. Repeated sampling facilitates within- and between-subject modeling of the human sperm transcriptome to identify dynamic and stress-responsive sncRNAs. Ramesh, R. et al. Dietary sugar shifts mitochondrial metabolism and small RNA biogenesis in sperm. Tomar, A. et al. Epigenetic inheritance of diet-induced and sperm-borne mitochondrial RNAs. Systematic characterization of seminal plasma piRNAs as molecular biomarkers for male infertility. Barceló, M., Mata, A., Bassas, L. & Larriba, S. Exosomal microRNAs in seminal plasma are markers of the origin of azoospermia and can predict the presence of sperm in testicular tissue. Xu, Y., Zhang, Y., Yang, Y., Liu, X. & Chen, Y. Seminal plasma miR-210-3p is a biomarker for screening dyszoospermia caused by varicocele. Wu, W. et al. Genome-wide microRNA expression profiling in idiopathic non-obstructive azoospermia: significant up-regulation of miR-141, miR-429 and miR-7-1-3p. Wang, C. et al. Altered profile of seminal plasma microRNAs in the molecular diagnosis of male infertility. Larriba, S. et al. Seminal extracellular vesicle sncRNA sequencing reveals altered miRNA/isomiR profiles as sperm retrieval biomarkers for azoospermia. Xu, H. et al. MicroRNA expression profile analysis in sperm reveals hsa-mir-191 as an auspicious omen of in vitro fertilization. Denomme, M. M., McCallie, B. R., Parks, J. C., Schoolcraft, W. B. & Katz-Jaffe, M. G. Alterations in the sperm histone-retained epigenome are associated with unexplained male factor infertility and poor blastocyst development in donor oocyte IVF cycles. A. et al. Non-coding RNAs from seminal plasma extracellular vesicles and success of live birth among couples undergoing fertility treatment. Joshi, M. et al. Qualitative and quantitative assessment of sperm miRNAs identifies hsa-miR-9-3p, hsa-miR-30b-5p and hsa-miR-122-5p as potential biomarkers of male infertility and sperm quality. Hamilton, M., Russell, S., Menezes, K., Moskovtsev, S. I. & Librach, C. Assessing spermatozoal small ribonucleic acids and their relationship to blastocyst development in idiopathic infertile males. Shinoda, G. et al. Fetal deficiency of Lin28 programs life-long aberrations in growth and glucose metabolism. Huang, C. et al. A multi-omics census reveals obesity-associated microRNA miR-let-7 as novel instigator of adipose mitochondrial dysfunction and of intergenerational metabolic decline. Yuan, S. et al. Sperm-borne miRNAs and endo-siRNAs are important for fertilization and preimplantation embryonic development. Expression of microRNA let-7 in cleavage embryos modulates cell fate determination and formation of mouse blastocysts. Tutunfroush, M., Ghorbian, S., Mohseni, J., Danaii, S. & Rad, M. G. Down-regulation of circulating miR-23a-3p, miR-101-3p, and miR-let-7c in women with idiopathic recurrent pregnancy loss. The association between Bisphenol A, steroid hormones, and selected microRNAs levels in seminal plasma of men with infertility. Xu, X., Shen, H. R., Yu, M., Du, M. R. & Li, X. L. MicroRNA let-7i inhibits granulosa-luteal cell proliferation and oestradiol biosynthesis by directly targeting IMP2. Circulating microRNAs as candidate biomarkers for the ovarian response during in vitro fertilization. Identification of MicroRNAs as potential biomarkers in ovarian endometriosis. Kumar, K., Trzybulska, D., Tsatsanis, C., Giwercman, A. & Almstrup, K. Identification of circulating small non-coding RNAs in relation to male subfertility and reproductive hormones. Tavanasefat, H. et al. Molecular consequences of fetal alcohol exposure on amniotic exosomal miRNAs with functional implications for stem cell potency and differentiation. Elzer, D., Bremser, M. & Zischler, H. Human sperm heads harbor modified YsRNA as transgenerationally inherited non-coding RNAs. Valkov, N. & Das, S. Y. RNAs: Biogenesis, Function and Implications for the Cardiovascular System. Zhang, Y. Y. et al. Dnmt2 mediates intergenerational transmission of paternally acquired metabolic disorders through sperm small non-coding RNAs. Liu, J. et al. Paternal phthalate exposure-elicited offspring metabolic disorders are associated with altered sperm small RNAs in mice. Zhang, Y. et al. Angiogenin mediates paternal inflammation-induced metabolic disorders in offspring through sperm tsRNAs. A. et al. Urinary phthalate metabolites and small non-coding RNAs from seminal plasma extracellular vesicles among men undergoing infertility treatment. Chen, X. et al. Human sperm tsRNA as potential biomarker and therapy target for male fertility. Identification of small non-coding RNAs as sperm quality biomarkers for in vitro fertilization. Cooper, T. G. et al. World Health Organization reference values for human semen characteristics. High-quality human and rat spermatozoal RNA isolation for functional genomic studies. Roszkowski, M. & Mansuy, I. M. High efficiency RNA extraction from sperm cells using guanidinium thiocyanate supplemented with tris(2-Carboxyethyl)phosphine. Gòdia, M. et al. A technical assessment of the porcine ejaculated spermatozoa for a sperm-specific RNA-seq analysis. The Istanbul consensus workshop on embryo assessment: proceedings of an expert meeting. Gardner, D. K., Lane, M., Stevens, J., Schlenker, T. & Schoolcraft, W. B. Blastocyst score affects implantation and pregnancy outcome: Towards a single blastocyst transfer. We wish to thank the personnel at the Centre for Reproductive Medicine at the University Hospital at Region Östergötland for their patience and assistance while setting up this study in parallel with their standard work. We thank collaborators and colleagues for their evaluation and feedback on this work. We acknowledge the Core Facility at the Faculty of Medicine and Health Sciences, Linköping University for providing assistance in sequencing. Financial support was provided by The Swedish Research Council (2015-03141, A.Ö), The Swedish Research Council (2020-00577, A.Ö), Ragnar Soderberg's foundation, Knut and Alice Wallenberg foundation (2015.0165, A.Ö), ALF Grants Region Östergötland (RÖ-995139, A.Ö), ALF Grants Region Östergötland (RÖ-975378, A.Ö). Open access funding provided by Linköping University. Signe Isacson, Kajsa Karlsson, Anna Asratian, Unn Kugelberg & Anita Öst Kajsa Karlsson, Stefan Zalavary & Susanne Liffner Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed bioinformatic analysis, data curation, and manuscript writing. performed sample collection, data curation, and manuscript writing. A.A. conceptualised and performed manuscript revision. supervised clinical sampling of material and data and performed manuscript revision. conceptualised, provided funding, performed manuscript writing, and supervised the project. The authors declare no competing interests. Nature Communications thanks Raffaele Teperino and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Isacson, S., Karlsson, K., Zalavary, S. et al. Small RNA in sperm–Paternal contributions to human embryo development. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            