
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-30</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63417-y'>Monumental rock art illustrates that humans thrived in the Arabian Desert during the Pleistocene-Holocene transition</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 18:54:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Dated archaeological sites are absent in northern Arabia between the Last Glacial Maximum (LGM) and 10,000 years ago (ka), signifying potential population abandonment prior to the onset of the Holocene humid period. Here we present evidence that playas became established in the Nefud desert of northern Arabia between ~16 and ~13 ka, the earliest reported presence of surface water following the hyper-aridity of the LGM. These fresh water sources facilitated human expansions into arid landscapes as shown by new excavations of stratified archaeological sites dating to between 12.8 and 11.4 ka. During the Pleistocene-Holocene transition, human populations exploited a network of seasonal water bodies - marking locations and access routes with monumental rock engravings of camels, ibex, wild equids, gazelles, and aurochs. These communities made distinctive stone tool types showing ongoing connections to the late Epipalaeolithic and Pre-Pottery Neolithic populations of the Levant. At the end of the last Ice Age, the Last Glacial Maximum (LGM, ~25–20 ka) ushered in harsher, cooler climatic conditions that resulted in extreme aridity across the Middle East, causing widespread dune mobilisation and de-population on the Arabian peninsula1,2,3,4. In the “Fertile Crescent”, the Pleistocene-Holocene transition brought a return to more favourable climatic conditions. The Natufian (~14.6 − 11.5 ka) is associated with intensive plant exploitation, and the widespread adoption of sedentism and food storage1,2,5. The domestication of plants and animals commenced in the Pre-Pottery Neolithic (PPN) A and B, respectively (~11.7–10.5 ka and 10.5–8.25 ka)1,6,7,8. These pronounced socio-economic shifts went hand in hand with the emergence of a rich symbology expressed in carvings, figurines, and architecture1,9. Notably, distinct geographic differences formed across the late Epipalaeolithic and the emerging Neolithic. In the Mediterranean woodland zones of the Levant, populations were mostly sedentary, while in the more arid regions of the Negev, Sinai, and eastern Jordan, populations were more mobile and reliant on hunting5,6. How climate and population dynamics during this crucial period in human history unfolded further south, on the Arabian Peninsula, is poorly understood. The presence of human populations in northern Arabia has been primarily linked with episodic humid phases10,11. Several sites document human occupation of the region between 10 and 8 ka, such as in the Umm Jirsan lava tube in the Harrat Khaybar17, and the sites of Al Rabyah12, Jebel Qattar18, and Jebel Oraf19, all located in the Jubbah oasis. Radiocarbon ages from Neolithic sites across north-western Arabia suggest a peak in human activity between 7.6 and 6.8 ka, which corresponds with the end of the Holocene humid period19,20,21,22,23. Neolithic communities built large stone structures, including large hunting traps known as desert kites, and large rectangular ritual structures called mustatils, which stretch up to 620 m in length21,23,24. Pastoral communities also created a rich record of rock art, with thousands of depictions of wild animals such as ibex and livestock such as cattle. However, superimpositions in the rock art record appear to show earlier phases of human activity, prior to the arrival of domesticated livestock25. In 2022, a rock art site was discovered at Sahout (SAU), an area south of the Nefud desert. Here, sandstone outcrops form low-lying mountain ranges that are separated by scattered sand dunes and rocky pavements (Fig. Survey documented 18 life-sized engravings of camels, ibex, and equids, with stratigraphic analysis suggesting that multiple large camel engravings may predate Neolithic engravings and the Holocene humid period. However, test excavations did not permit a correlation between dated deposits and rock art27. Three further panels with large camels were recently documented by members of the public in localities ca. Their coordinates were compiled by the Saudi Heritage Commission and shared with our team. This information provided an opportunity to document additional panels belonging to this possible early Holocene rock art tradition, and to survey and excavate any associated archaeological deposits. White dots: locations of rock art panels; ARN: 46 panels with 64 life-sized animals, 15 human figures and 22 small or partial figures; MLH: 1 panel with 1 life-sized camel, JMI: 14 panels with 48 life-sized animals (23 on a single panel – JMI18), 1 human figure and 4 small or partial figures, SAU: 18 life-sized animal engravings recorded in 202227, 3 human figures and 1 small figure. Blue triangles: archaeological excavations; yellow squares: playa excavations with sufficient sediment for analysis. Bing Virtual Earth imagery as basemap in QGIS. Here we report the results of archaeological surveys and excavations at rock art sites with life-sized camel engravings south of the Nefud desert, as well as analyses of spatially associated playa deposits. Three archaeologically unexplored areas were visited during our fieldwork in 2023: Jebel Arnaan (ARN), Jebel Mleiha (MLH), and Jebel Misma (JMI). To acquire paleoenvironmental context for the region, playa sediments were excavated and dated at ARN and JMI (Fig. The primary aim of this research was to test if the rock art panels and archaeological deposits represent earlier occupations during the Holocene humid period, resulting in a condensed chronology of rock art production, or if they belong to earlier periods and represent a longer period of human presence in northern Arabia. To evaluate the terminal Pleistocene and early Holocene environments of the region, trenches were excavated at four playas in the vicinity of the archaeological sites, two of which contained sufficient sediment for palaeoenvironmental analysis and luminescence dating: ARN (Site 1) and JMI (Site 4), (Fig. As local centres of deposition, and areas where water accumulated in the past, these playas provide archives of hydroclimate. Gravels found at the base of both 2 m deep trenches were poorly sorted, containing pebbles of up to 5 cm, and were interpreted as alluvial fan deposits. A luminescence age of 68.8 ± 5.0 ka (MIS-B-1) from JMI indicates that these sediments were deposited at the Marine Isotope Stage 5a to 4 transition (Table 1). Palaeohydrological activation at this time is consistent with both broader regional records28 and a surface find of a Middle Palaeolithic Levallois core from JMI (Supplementary Fig. At both excavated paleoenvironmental sites, the overlying sediments consist of a thin interval of alternating much finer-grained and well-sorted aeolian sands and playa deposits. These are followed by a prolonged sequence of playa deposits consisting of quartz sand, clays (illite and kaolinite), and calcite, indicating more humid conditions with increased water accumulation in these local depressions. We interpret the onset of playa sediment accretion as representing the change in the balance between aeolian erosion and fluvial sedimentation. During the LGM, the hyper-arid environment meant that any fine-grained sediments deposited in ephemeral floods were subsequently eroded by the wind. As the climate became less arid, fluvial sedimentation increased, becoming greater than the aeolian erosion rate and sediments started to accumulate. However, the subordinate carbonate concentration and the absence of root traces and organic-rich layers suggest that conditions remained too dry for the establishment of more permanent water bodies, indicating that a dryland environment persisted around these ephemeral, and probably seasonal, lakes. Luminescence ages constrain the onset of surface water to between 17.1 ± 1.7 ka (MIS-B-3) and 15.5 ± 1.5 ka (MIS-B-2) at JMI (Site 4) and to 12.7 ± 2.1 ka (SAH-L3-1) at ARN (Site 1) (Table 1). The onset of sedimentation after a long hiatus, coupled with an increase in calcite noted in both sedimentary records, indicates that local conditions gradually became more humid after ~16 ka at JMI to ~13 ka at ARN, with these site-specific differences in timing likely due to differences in catchment hydrology (Table 1). These sites thus represent the earliest evidence from northern Arabia of increased humidity following the hyper-arid LGM. However, the lakes were ephemeral, indicating an arid or semi-arid climate prevailed in the region. Surveys at ARN and JMI identified previously unknown rock art landscapes with life-sized depictions of wild mammals and human figures, and an individual panel at MLH. Across the three areas 62 rock art panels were recorded, containing 176 engravings. Of these, 130 were life-sized and naturalistic engravings depicting camels (90), ibex (17), equids (15), gazelles (7), and aurochs (1), with individual representations frequently measuring up to 2.5–3.0 m in length and 1.8–2.2 m in height. In addition, we identified 2 camel footprints, 15 smaller scale naturalistic depictions of camels, 19 human figures, 4 human faces or masks, and 6 unidentified, partial engravings (Supplementary Data 1). This includes the frequent depiction of a bulging neckline, indicating they represent male camels in rut29,30 (Fig. Rock art scale is 10 cm wide. A Panel ARN21A: Several small, stylised depictions of women (phase 1, traced in green), superimposed with a large human figure (phase 2, traced in yellow). Large, kneeling camel engraved over the top (phase 3, traced in white). B Panel ARN22A : naturalistic camel (phase 3, traced in white), superimposed by a camel with stylised, rounded eye and standardised outline (phase 4, traced in blue); original and unfinished camel head (traced in dark blue); three gazelle engravings were added during a later part of phase 4 (traced in light blue). Additional examples are provided in Supplementary Fig. C Panel ARN3B : ibex with cartoon-like eye and horn (phase 4, traced in blue), superimposed with life-sized aurochs (phase 4, traced in lighter blue). D Panel ARN 39 : equid with cartoon-like eye, and with a young (phase 4, traced in blue). Untraced photos are provided in Supplementary Figs. The depictions span multiple engraving phases, with images often overlapping on rock surfaces. Sometimes this was done to update an existing representation (Fig. 2B) or to depict a different animal species (Fig. We distinguish four phases here. Two early rock art phases: small, stylised depictions of women (phase 1, traced in green), followed by large human figures in frontal view (phase 2, traced in yellow). These human figures were always noted to be older than, i.e. underneath, the recorded life-sized animal representations (Fig. 2A), and they make up a much smaller proportion of motifs. The third phase shows detailed, extremely naturalistic representations of animals, where each depiction has individual characteristics (traced in white) (Fig. A later, fourth phase (traced in blue) shows more stylised depictions of animals with cartoonish features, including rounded eyes and horn ridges, and more standardised, near-identical depictions of animals (Figs. An OSL date of 12.0 ± 1.8 was obtained from JMI8 (Table 1: JMI8-T1-4). Excavated playa deposits are located directly behind this spur (see also Figs. B narrow, downward sloping ledges in front of the panel viewed from above. C orthophoto generated via a high-resolution 3D model of the panel, with tracings showing 19 life-sized camels and 3 equids (one further camel was documented on a collapsed fragment left of the image). Naturalistic animals belonging to phase 3 traced in white. More stylised and standardised depictions of phase 4 traced in blue, including two engravings of equids traced in dark blue, and superimposed stylised camels traced in light blue. Unidentified lines traced in black. A human figure was added on the far left for scale (1.7 m). Unlike the SAU site, where many of the engravings were found inside narrow gaps between boulders27, engravings at ARN and JMI were found on prominent locations on boulders or cliff surfaces, facing into the landscape (Supplementary Figs. Some of these panels were etched onto cliff surfaces in inaccessible but highly visible locations (Fig. The difficulty in getting to and engraving these rock surfaces, and their enhanced visibility by height were clearly attractive for the engravers. On the ground, panel JMI18 is today only visible in optimal light conditions for about 1.5 h in the morning, due to its elevated location and the varnish build-up on the engravings. This panel would have been accessed by climbing up a cliff and then engraved while standing on a downward sloping ledge, only ~30–50 cm in width (Fig. Engraving at close range would have required them to use direct percussion, while also preventing them from being able to see the complete image. Twenty-three life-sized camels and equids, each with an individual length of ~1.7–2.6 m were engraved on this surface (Fig. 3A), giving this rock art a monumental scale. All recorded rock art panels show a thick coating of dark rock varnish on the natural sandstone surface and inside the engraved lines (Fig. 2), although on some panels this varnish has partially eroded and is now only visible in areas that are less exposed (Fig. The appearance of multiple life-sized animal engravings is impressive today. Freshly engraved against the varnish, the images would have had considerable visual impact. The durability of the images may have facilitated the remembrance of meaning and symbolism across generations of people using these sites. The large naturalistic engravings, therefore, align with the definition of monumentality, which references great size and effort, but also longevity and remembrance within a community33,34. Four trenches revealed stratified archaeological deposits. 1), yielding artefacts including over 1200 lithics (Supplementary Note 2), and 16 bone fragments (Supplementary Note 4). Optically stimulated luminescence (OSL) and radiocarbon dating (Supplementary Note 6) samples from the archaeological deposits attest to human occupation during the terminal Pleistocene and earliest Holocene. In front of panel ARN3, a 2 × 1 m trench (T1) was excavated directly below two life-sized engravings of camels, one superimposed on the other (Fig. Below recently disturbed sediments, the excavation revealed intact archaeological deposits with layer 5 in particular containing dozens of flaked stone artefacts. This layer was sealed by several large sandstone fragments in layer 4 (Fig. 4), likely the result of a colluvial event. In ARN3 T1, the legs of the engraved camels were buried at a depth indicating that they were engraved no later than when layer 4 was deposited (Fig. Layer 5 included a stone tool that may have been used for engraving the rock art (discussed below) (Fig. Two luminescence samples obtained from layer 5, dating to 12.2 ± 1.4 ka, and 12.8 ± 1.1 ka (Table 1), directly date the burial of the artefacts and pecking tool, and provide an indirect date for the engraving above. Analysis of the engraved lines shows that all images were made using a pecking stone. In some engravings, pecked lines were smoothed in a second step (Fig. South-western and north-western sections, highlighting the extent of the disturbance: original height of deposit (pink dotted line), and extent of disturbed sand (pink line). Colour grading of layers reflects artefact density. Location of OSL samples shown in black, location of engraving tool projected onto section from a plotted position 72 cm south-east. Earlier camel engraving (phase 3) traced in grey, later camel engraving (phase 4) traced in blue, dashed lines indicated parts of the engraving that were covered with sediment. Note that the legs of the earlier (grey) camel have partially eroded and are no longer visible. Remains of dark rock varnish29,31,47 can be seen in several patches on both lines. A second trench (T2) was excavated at ARN3 1.2 m to the east, beneath two more camel engravings at the other end of the same boulder surface (Supplementary Fig. 13) with three small hearths in layer 7, one of which was radiocarbon dated to 11.44 ± 0.18 ka (cal. Refits from layers 8 and 9 in T2 testify to the high integrity of the archaeological contexts (Supplementary Note 3). At JMI, two trenches were excavated beneath boulders featuring rock art panels. At JMI7 a 2 × 1 m trench was excavated beneath life-sized engravings of two camels and an equid, which revealed a single layer with high artefact concentrations (layer 4; Supplementary Fig. Here moderate artefact concentrations were interspersed throughout the stratigraphic sequence, with the highest artefact concentration in layer 2, near the surface (Supplementary Fig. A luminescence sample was obtained from layer 6, which provided an age of 12.0 ± 1.8 ka (JMI8-T1-4, Table 1), consistent with the luminescence and 14C ages at ARN. The ARN3 excavations revealed a rich assemblage of occupation debris. 4), while in T2, 532 lithics were recovered, with over 95% from layers 5–9 (Supplementary Fig. Chert was the main material used, constituting 36% of the pieces, with obsidian and crystal quartz making up another 23% and 15% respectively (Supplementary Data 2). Lithics showed a diversity of reduction strategies, including small bipolar cores occurring alongside chert bladelets/blades with lamellar scar patterns and often ground platforms (Supplementary Note 3.1). A total of 30 retouched pieces were recovered. Among these, notches were common with eight notched pieces and two opposed notch blades (Supplementary Table 1). Two chert drills were recovered in T2, as well as a single endscraper from the disturbed upper part of T1. The largest retouched artefacts were two tongue-shaped scrapers from T2 layer 8 (Supplementary Fig. T2 produced seven marginally retouched convergent points made on chert bladelets, while the tanged butt of a point on an obsidian blade was recovered from T1 (Supplementary Figs. The most distinctive artefacts were a single crystal quartz Helwan bladelet from T1 layer 4 (Fig. 5B), and three chert El Khiam points, a broken butt from T1 layer 5, and both a broken butt and a complete El Khiam point from T2 layer 5 (Fig. Quartz was the most commonly used material comprising 65% of pieces, with an additional 20% made from silcrete (Supplementary Data 3). The single core was a silcrete naviform piece from layer 4 in JMI7 (Supplementary Fig. The JMI7 and JMI8 excavations produced nine retouched artefacts (Supplementary Table 2). An opposed-notch chert blade was recovered in layer 5 (Supplementary Fig. 31), with two other chert notched blades recovered from JMI8 layer 4, but these did not have the distinctive opposing notches. This same layer also produced a chert drill with a distal break. A broken Helwan point was found on the surface between JMI7 and JMI8 (Supplementary Fig. 5B) and opposed notch blades are known from Natufian sites; El Khiam points (Fig. 5A) are an artefact type characteristic of the PPNA, whereas Helwan points and naviform cores are typical for the PPNB (Supplementary Note 3). These artefact types accord with the chronometric dating at ARN and JMI (Tables 1 and 2). Four grinding stones were recovered from the ARN3 excavations and six from the JMI excavations, along with a facetted stone ball (Supplementary Fig. 34) and two stone platters in JMI7 layer 4 (Supplementary Fig. Two mortars were documented on an exposed rock surface between JMI7 and JMI8 (Supplementary Fig. Mortars are typical of the PPNA35 and have not previously been documented among grindstone assemblages of Neolithic northern Arabia36. A total of five ground stone beads were recovered, four green pieces from ARN T2 layers 7 and 9, and one from JMI8 layer 2. The recovery of two broken beads and one unfinished bead, together with drills at both sites, suggests on-site manufacture. Two marine dentalium (tooth-shell) beads were recovered from ARN3 T2, layers 7 and 8 (Fig. Among mostly red pigment, noteworthy was a crayon of green copper ore pigment from ARN3 T2 layer 7. Green pigment and ground stone beads are characteristic of the earlier PPN37,38. Three tools with clear battering marks were recovered from directly below engraved panels. One is from a stratified and dated archaeological context underneath a life-sized camel engraving at ARN3 T1 (Figs. 4 and 6), immediately below OSL sample ARN-T1-2, dated to 12.2 ± 1.4 ka, and above sample ARN-T1-3 dated 12.8 ± 1.1 ka (Table 1). Two additional tools were recovered on the surface beneath panels ARN30 and JMI8 (Supplementary Fig. The ARN tools are made on clasts of ferruginous sandstone, a tough material not immediately available at the rock art localities, while the JMI8 tool is made of silcrete, a material available at SAU. The tools are all wedge shaped, and the two made on ferruginous sandstone were flaked to sharpen the pecking edge. The tools comfortably fit into a medium-sized adult hand, with the battered edges protruding (Fig. Battering is evident on multiple edges of all three tools, indicating they were used extensively. Due to their angular form, these tools are not viable as lithic hammerstones in the arced free-hand percussion motion; however, the tapered ends would make good surfaces for the direct end-on percussion required to peck petroglyphs (Supplementary Note 3.3). The tool recovered from ARN3 T1 was used to peck and could thus have been involved in the production of either of the two camel engravings above (Fig. In Arabia, this type of tool was first noted at the Camel Site, where traceological analysis on silcrete pieces showed damage consistent with sandstone engraving40. Comparable tools and patterns of wear have also been observed at petroglyph sites in Europe41 and South America42,43. A Two sides of the pecking tool. B, C pecking tool held in a hand. For additional photos see Supplementary Figs. Archaeological and palaeoenvironmental research provides new insights into the human occupation history of northern Arabia (Fig. An absence of sites dating between the LGM and ~12 ka has been observed across the Arabian Peninsula and was likely exacerbated by rising sea levels, which would have submerged any sites associated with coastal refugia44,45. Our evidence for the onset of playa sedimentation indicates that as the hyper-arid climate of the LGM gave way to gradually increasing humidity, ephemeral water bodies were established after ~16 ka at JMI and from ~13 ka at ARN. Human groups in northern Arabia began to exploit these newly forming seasonal water bodies in dryland ecosystems during the Pleistocene-Holocene transition. 2σ 14 C age ranges are presented in calibrated years BP, while luminescence age ranges are shown at 1σ in ka prior to the measurement datum of 2023. Evidence for human activity and environments listed in grey refer to published research11,19,22,57, evidence listed in black is presented in this paper. Artefacts listed under human activity were recovered during excavations at ARN and JMI and are dated based on OSL and 14 C ages recovered at ARN and JMI, as well as typological comparisons with the Levant. The timing of rock art phases reflect the interpretation of our evidence presented in the text: Rock Art phases 3 and 4 are based on OSL dates from a layer containing a pecking tool at ARN T1, a radiocarbon age from ARN3 T2, an OSL date from JMI8 T1, and the overall characteristics of artefact assemblages at ARN and JMI. The timing of Rock Art phases 1 and 2 is estimated based on superimpositions. The environmental phases are based on a change in the balance between aeolian erosion and fluvial sedimentation that is visible in the sedimentary record, with increasing fluvial sedimentation indicating a reduction in aridity. Sediment analyses indicate seasonal waterbodies in a landscape that was likely slightly wetter than it is today but too arid to allow the establishment of more permanent lakes (Supplementary Note 5). The fauna represented in the rock art supports this evidence with the identifiable engravings dominated by arid-adapted species, i.e., camels (72%), ibex (12%), gazelles (5%), and wild equids (10%) (Supplementary Data 1). In this context a single depiction of an aurochs at ARN3B (Fig. 2C) stands out and suggests that either these obligate drinkers were able to penetrate the Arabian interior during exceptionally wet seasons or that aurochs were seen elsewhere, perhaps when human populations retreated to wetter areas during the dry season. Observations of modern feral camels indicate their mating season correlates with the wet season46. The depiction of male camels in rut29,30 and with visible winter fur in these engravings27 thus indicates rainfall may have also coincided with the winter months around 12 ka. Two independent lines of evidence provide a chronology for the rock art at Jebel Arnaan. Firstly, life-sized engravings of animals are the principal rock art tradition at the ARN localities, with no later or superimposed images present until the introduction of firearms (Supplementary Fig. The ferruginous pecking stone in layer 5 would be effective in engraving sandstone. In the absence of other rock art within 200 m of the site, our parsimonious assumption is that the pecking stone was used to produce one of the engraved figures at ARN3, and that corresponding chronometric dates can thus be used to date the rock art. Nevertheless, the possibility that the engraving tool was transported from a different panel at Jebel Arnaan cannot be excluded entirely. Luminescence and radiocarbon ages obtained at ARN3 indicate that the pecking stone was deposited at the Pleistocene-Holocene transition, and the main phase of occupation is contemporary with the PPNA. Our resulting correlation is that at least one of these large animal engravings was produced during the PPNA. This is supported by analyses of rock varnish which have shown that complete re-varnishing of engraved lines (Figs. 2 and 3) requires ca 8 ka31,32,47, and match previous observations at SAU and at the Camel Site that this type of engraving does not include any depictions of livestock and thus likely pre-dates the emergence of pastoralism in northern Arabia27,29,48. Two earlier rock art phases were recorded with repeated superimposition by life-sized camels, but these do not have direct associations with archaeological deposits. Small, stylised depictions of women with accentuated curves (rock art phase 1), and subsequent depictions of large, stylised human figures (rock art phase 2, Fig. These human figures consequently pre-date engravings such as those at ARN3, and are thus likely older than 12 ka (Fig. Similarly, an Epipalaeolithic date had previously been tentatively suggested for stylistically similar figures in the Jubbah oasis based on a relative chronology of rock art superimpositions25. Re-engraving events such as the one documented at JMI18 (Fig. 3C) indicate that the tradition of large, naturalistic engravings was long-lived. For new lines to be visible, previous representations must have been sufficiently faded through varnish accumulation, weathering or erosion. This pattern was also observed in the rock art of SAU27 and appears to be typical for this monumental rock art tradition. Engravings of phase 4, which are often found superimposed over earlier naturalistic depictions of phase 3 (Fig. 2B and 3C), are more cartoonish and follow a more standardised ideal of beauty, reflecting a stylistic evolution over time (Supplementary Note 1 and Supplementary Fig. During the terminal Pleistocene and earliest Holocene, human populations appear to have moved along established routes determined by a network of water sources, which were marked with monumental rock art. Such ‘freshwater corridors' also connected the Nefud and Levant during the much wetter Pleistocene humid events when large permanent lakes formed in the region10. At JMI all panels flank the edge of a playa (Fig. At ARN, all life-sized rock art panels follow a gulley up the slope in which water pools even today after infrequent rains (Fig. This route provided convenient access to ephemeral water pools and it would also have acted as a shortcut across the mountain to reach two playas on the western side (one of which is Site 1, ARN), where water would likely have been available for longer periods after infrequent flood events (Fig. All panels with life-sized animals overlook this route and would have been highly visible when freshly engraved (Supplementary Fig. A Palaeolakes on the western slope of Jebel Arnaan. Rock art panels follow a gully up the mountain that connect to a valley which leads to two palaeolakes. Excavation of lake sediments marked in yellow, archaeological excavation marked with a blue triangle. Inset: close up of the rock art distribution at ARN, showing all panels with large animal engravings along a gulley in which water pools today. In the foreground water can be seen pooling in a rock pool. C Palaeolake and life-sized rock art at Jebel Misma. A water hole directly east of the mapped panels held water during excavations in May 2023 (Supplementary Fig. Bing Virtual Earth imagery as basemap in QGIS. The engravings express a symbolism that relates to desert animals and to seasonality, namely the depiction of male camels in rut, which may have referenced the mating season of these animals after the winter rains46, and potentially also their extraordinary resilience in these arid landscapes. In the drylands of the Pleistocene-Holocene transition, hunter-gatherer groups were likely highly mobile. Depictions of wild animals and the marking of natural landscapes may have been a mechanism to mark routes and perhaps also served as territorial markers documenting access rights via impressive images that would have “guarded” locations in a group's absence. The engravings, which may have been created over a time span of millennia, would have reminded people of ancient symbolisms and beliefs of their group, which likely structured their highly seasonal lives and thus enhanced their ability to thrive in these marginal landscapes. Mounting archaeological evidence suggests repeated contact between human populations in northern Arabia and the Levant, throughout the terminal Pleistocene and the Holocene. Arrowheads associated with the Late Neolithic in the Levant, such as Ha-Parsa, Nizzanim, and Herzliya types, as well as decorative artefacts such as shell beads and stone bracelets, have been identified at Jebel Oraf19 and at multiple sites in the AlUla region49. PPNB technologies such as naviform cores are well documented from the Jawf region north of the Nefud desert50, with the discovery of Helwan points at Jebel Oraf51 extending these links into the southern Nefud desert, where PPNA technology in the form of El Khiam points is also known from Jebel Qattar18 in the Jubbah oasis. The recent discovery of a Helwan bladelet at SAU, alongside a radiocarbon age of 13.4 ka indicates that these connections may reach back to the terminal Pleistocene27. The artefact assemblages presented here include Levantine stone tool types and decorative forms of material culture (green ground stone disc and dentalium beads), suggesting regular and repeated cultural contact across long distances. Luminescence and radiocarbon ages obtained from stratigraphic contexts indicate that the use of these artefacts was contemporaneous with the Levant (Fig. To our knowledge, this is the first time PPN artefacts have been found in a stratified and dated context in Arabia, and the first time they have been associated with this style of monumental naturalistic art. The distribution of the monumental rock art along routes connecting seasonal water bodies suggests high mobility, and regular movements across considerable distances may have been a key adaptation to the cyclical availability of water. In one case this may even have been captured in three rock art panels north and south of the Nefud desert, which appear to have been re-worked multiple times by the same succession of artists48. Connectivity is also a key feature of Epipalaeolithic occupations in the Levant and Jordan52, with such connectivity increasing in scale in the PPN53,54. Population movements in both northern Arabia and regions further north may therefore have facilitated the transfer of knowledge while also accommodating distinct cultural expressions. These pioneers were able to thrive in the arid conditions of the terminal Pleistocene and earliest Holocene due to seasonal water bodies. The presence of key lithic artefacts such as El Khiam and Helwan points, as well as decorative artefacts such as green pigment and dentalium beads, suggests these human groups maintained contact with their Levantine neighbours during the PPN, travelling across vast distances. However, the engravers of Jebel Arnaan and Jebel Misma had their own, distinct cultural and symbolic identity. Their adaptation to an environment where water was only available temporarily, involved complex mobility along routes connecting different water sources. Unlike their Levantine neighbours, they produced monumental rock art that centres around a desert animal symbolism: the camel. These monumental images were used to mark water sources and the routes between them, perhaps providing impressive visual reminders of access rights, while also commemorating these extraordinary desert-adapted groups over millennia. Permission for excavations in Saudi Arabia, sampling, export of samples and artefacts to the United Kingdom, and analysis was granted by the Heritage Commission, Ministry of Culture, Saudi Arabia. Rock art panels were recorded using pro-forma record sheets, with content, size and GPS location noted, and documented using high-resolution digital photographs as raw files and JPGs. To enhance faded details on the rock art, some photos were enhanced using Adobe Lightroom. Key panels were photographed systematically to allow generation of high-resolution 3D models and orthophotos using Metashape photogrammetry software. One exceptionally large panel was located on a cliff 34–39 m above the ground (Fig. 3C) and had to be documented using a DJI Mavic Mini 2 UAV due to its height and inaccessibility, with the resulting photographs combined into an orthophoto to facilitate analysis. Rock art surveys were carried out systematically, with survey teams following the base of jebels and exploring all boulders and rock surfaces along the base and within heights that could safely be reached. At Jebel Arnaan, rock art survey followed a gully that leads up the mountain slope and connects to a wadi above (Fig. Distinct surfaces of boulders were defined as separate panels, with multiple surfaces on the same boulder labelled with the same number, distinguished by a letter suffix (e.g. ARN3_A, ARN3_B). All trenches were hand-dug by trowel using the single context method55. Thicker deposits were sometimes divided into multiple contexts for extra stratigraphic control. Contexts were recorded using project-specific pro forma record sheets. Depths were recorded with a line level, with five points taken for each context. Excavated sediment was sieved through a 3 mm mesh with all artefacts and faunal remains retained. The position of key finds was manually measured using a plumb bob from an origin in one corner of each trench. All lithics were classified by material, counted, and weighed. Any retouched pieces and cores were assigned to types. Key types were photographed and measured. Grinding stones were assigned to types, their ground surface form characterised, and any complete dimensions measured. Pigment was weighed and pieces with striations visible at low magnifications designated as crayons. Bead diameter, aperture width, and thickness were measured. Luminescence dates presented in this study were determined using optically stimulated luminescence techniques applied to quartz. These measure the time elapsed since sediments were last exposed to sunlight. Opaque metal tubes were hammered into cleaned sections, transported to the UK and analysed as described in Supplementary Note 6.1. Environmental dose rates were calculated using location and overburden density (cosmic rays), field gamma spectrometry (gamma), and thick-source beta counting (beta). Luminescence analyses were carried out by Simon Armitage at the School of Life Sciences and the Environment, Royal Holloway University of London, United Kingdom. Samples suitable for radiocarbon dating were collected from stratified archaeological deposits during excavation and included charcoal-rich sediment samples from hearth deposits, ostrich egg shell, micro charcoal, and one piece of shell. The latter was identified as a pearl oyster (Supplementary Note 4). Samples for radiocarbon dating were sent to the Centre for Applied Isotope Studies (CAIS) at the University of Georgia and analysed as described in Supplementary Note 6.2. Trenches were dug in the middle of the playa settings, up to 2 m deep. The sections were described sedimentologically and sampled for geochemical lab analysis and luminescence dating. The XRD semiquantitative analysis of the major mineral components was carried out using a D8 Bruker machine, at the laboratory in KAUST, with a chosen angle (2θ) between the incoming and outcoming X-ray beam between 5 and 80°. The mineral phase interpretation was done using DIFFRAC.EVA software and the results are presented as a proportional percentage of the mineral phase present in the analysed samples (Supplementary Note 5). The samples were powdered using a handheld mortar. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data used in this study are available in the Supplementary Information files. All data used to support our results were generated in our research. Sediment samples from playa deposits are stored at King Abdullah University for Science and Technology, Physical Science and Engineering Division. Artefacts recovered during excavations will be returned to and stored by the Heritage Commission, Ministry of Culture, Saudi Arabia, in 2025. Access can be negotiated with the Heritage Commission (faljibrin@moc.gov.sa). Parker, A. G. in The Evolution of Human Populations in Arabia (eds Petraglia M. D. & Jeffrey I. Multiple phases of human occupation in Southeast Arabia between 210,000 and 120,000 years ago. Richter, T., Arranz-Otaegui, A., Yeomans, L. & Boaretto, E. High Resolution AMS Dates from Shubayqa 1, northeast Jordan Reveal Complex Origins of Late Epipalaeolithic Natufian in the Levant. Martin, L. & Edwards, Y. in Origins and Spread of Domestic Animals in Southwest Asia and Europe (eds Sue Colledge, James Conolly, Keith Dobney, & Katie Manning) 49–82 (Left Coast Press, 2013). From foraging to farming in the southern Levant: the development of Epipalaeolithic and Pre-pottery Neolithic plant management strategies. & Lucas, L. Contrasting pathways to domestication and agriculture around Southwest Asia. Breeze, P. S. et al. Palaeohydrological corridors for hominin dispersals in the Middle East ∼250–70,000 years ago. Petraglia, M. D., Groucutt, H. S., Guagnin, M., Breeze, P. S. & Boivin, N. Human responses to climate and ecosystem change in ancient Arabia. Hilbert, Y. H. et al. Epipalaeolithic occupation and palaeoenvironments of the southern Nefud desert, Saudi Arabia, during the Terminal Pleistocene and Early Holocene. Preston, G. W. et al. A multi-proxy analysis of the Holocene humid phase from the United Arab Emirates and its implications for southeast Arabia's Neolithic populations. A comment on ‘The middle Holocene climatic records from Arabia: reassessing lacustrine environments, shift of ITCZ in Arabian Sea, and impacts of the southwest Indian and African monsoons' by Enzel et al. Glob. The unexpectedly short Holocene Humid Period in Northern Arabia. Dinies, M., Plessen, B., Neef, R. & Kürschner, H. When the desert was green: Grassland expansion during the early Holocene in northwestern Arabia. Stewart, M. et al. First evidence for human occupation of a lava tube in Arabia: the archaeology of Umm Jirsan Cave and its surroundings, northern Saudi Arabia. Beyond the Levant: first evidence of a pre-pottery Neolithic incursion into the Nefud Desert, Saudi Arabia. Guagnin, M. et al. A tale of two hearth sites: Neolithic and intermittent mid to late Holocene occupations in the Jubbah oasis, northern Saudi Arabia. Abu-Azizeh, W. et al. in Revealing Cultural Landscapes in North-west Arabia (Supplement to Volume 51 of the Proceedings of the Seminar for Arabian Studies) (eds Rebecca, F., Maria, G., Ingrid, P., & Karacic, S.) 133-156 (Archaeopress, 2022). Groucutt, H. S. et al. Monumental landscapes of the Holocene humid period in Northern Arabia: the mustatil phenomenon. Kennedy, M. et al. Cult, herding, and ‘pilgrimage' in the Late Neolithic of north-west Arabia: Excavations at a mustatil east of AlUla. Thomas, H. et al. Monumentality, social memory, and territoriality in Neolithic–Chalcolithic Northwestern Arabia. Fradley, M., Simi, F. & Guagnin, M. Following the herds? A new distribution of hunting kites in Southwest Asia. Guagnin, M. et al. An illustrated prehistory of the Jubbah oasis: Reconstructing Holocene occupation patterns in north-western Saudi Arabia from rock art and inscriptions. Guagnin, M. et al. Hunters and herders: exploring the Neolithic transition in the rock art of Shuwaymis, Saudi Arabia. Guagnin, M. et al. Before the Holocene humid period: Life-sized camel engravings and early occupations on the southern edge of the Nefud Desert. Multiple hominin dispersals into Southwest Asia over the past 400,000 years. Life-sized Neolithic camel sculptures in Arabia: a scientific assessment of the craftsmanship and age of the Camel Site reliefs. Guagnin, M. et al. in Revealing Cultural Landscapes in North-west Arabia (Supplement to Volume 51 of the Proceedings of the Seminar for Arabian Studies) (eds Rebecca, F., Maria, G., Ingrid, P., & Karacic, S.) 110–132 (Archaeopress, 2022). Macholdt, D. S., Al-Amri, A. M., Tuffaha, H. T., Jochum, K. P. & Andreae, M. O. Growth of desert varnish on petroglyphs from Jubbah and Shuwaymis, Ha'il region, Saudi Arabia. Zerboni, A. Holocene rock varnish on the Messak plateau (Libyan Sahara): Chronology of weathering processes. Scarre, C. in The Oxford Handbook of the Archaeology of Ritual and Religion (ed Timothy I.) Osborne, J. F. in Approaching Monumentality in Archaeology IEMA Proceedings (ed James F. Osborne) Ch. 1, 1–19 (State University of New York Press, 2014). Wright, K. I. Ground-stone tools and hunter-gatherer subsistence in southwest Asia: implications for the transition to farming. Lucarini, G. et al. Plant, pigment, and bone processing in the Neolithic of northern Arabia – New Evidence from Use-Wear Analysis of Grinding Tools at Jebel Oraf. Application of XRD and digital optical microscopy to investigate lapidary technologies in Pre-Pottery Neolithic societies. Groman-Yaroslavski, I., Porat, N. & Bar-Yosef Mayer, D. E. A Neolithic technological approach to the production of stone beads: insights from the analysis of the Nahal Hemar cave assemblage. Bar-Yosef Mayer, D. E. Towards a typology of stone beads in the Neolithic Levant. Hilbert, Y. H. et al. Traceological analysis of lithics from the Camel Site, al-Jawf, Saudi Arabia: an experimental approach to identifying mineral processing activities using silcrete tools. Guéret, C. & Bénard, A. Fontainebleau rock art” (Ile-de-France, France), an exceptional rock art group dated to the Mesolithic? Critical return on the lithic material discovered in three decorated rock shelters. Alvarez, M., Fiore, D., Favret, E. & Guerra, R. C. The use of lithic artefacts for making rock art engravings: observation and analysis of use-wear traces in experimental tools through optical microscopy and SEM. The study of rock art engravings through lithic technologies: the Villavil 2 Site (Hualfin Valley, Catamarca, Argentina). Jennings, R. P. et al. Human occupation of the northern Arabian interior during early Marine Isotope Stage 3. Rose, J. I. in An Introduction to Human Prehistory in Arabia: The Lost World of the Southern Crescent (ed Jeffrey I. Its Evolution, Ecology, Behaviour, and Relationship to Man (The University of Chicago Press, 1981). Andreae, M. O. et al. Archaeometric studies on pretroglyphs and rock varnish at Kilwa and Sakaka, northern Saudi Arabia. & Petraglia, M. D. A Rock Art Tradition of Life-sized Naturalistic Engravings of Camels in Northern Arabia: New Insights on the Mobility of Neolithic populations in the Nafud Desert. McMahon, J. et al. New evidence for Neolithic occupation in north-west Arabia: standing stone circles on the Harrat ‘Uwayrid. Crassard, R. & Hilbert, Y. H. Bidirectional blade technology on naviform cores from northern Arabia: new evidence of Arabian-Levantine interactions in the Neolithic. Richter, T., Garrard, A. N., Allock, S. & Maher, L. A. Interaction before agriculture: exchanging material and sharing knowledge in the final Pleistocene Levant. Ibáñez, J. J., Ortega, D., Campos, D., Khalidi, L. & Méndez, V. Testing complex networks of interaction at the onset of the Near Eastern Neolithic using modelling of obsidian exchange. The flow of ideas: shared symbolism during the Neolithic emergence in Southwest Asia: WF16 and Göbekli Tepe. Westman, A. Archaeological Site Manual, 3rd edn (Museum of London, 1994). The IntCal20 Northern Hemisphere Radiocarbon Age Calibration Curve (0–55 cal kBP). Guagnin, M., Perri, A. R. & Petraglia, M. D. Pre-Neolithic evidence for dog-assisted hunting strategies in Arabia. We thank HH Prince Badr Bin Abdullah Bin Farhan Al-Saud, Saudi Minister of Culture, for permission to conduct research at Sahout. We also thank Dr. Jasir Alherbish, CEO of the Saudi Heritage Commission, and Dr. Abdullah al-Zahrani, General Director for Archaeological Excavations. Financial and logistic support in the field was provided by the Saudi Heritage Commission. Funding for fieldwork and research was provided by a British Academy/Leverhulme Small Research Grant (SRG2223\231473 to M.G. We acknowledge funding from King Abdullah University of Science and Technology (KAUST) from baseline support of F.v.B. contribution to this work was partly supported by the Research Council of Norway, through its Centres of Excellence funding scheme, SFF Centre for Early Sapiens Behaviour (SapienCE), project number 262618. acknowledges the support of the Researchers Supporting Project number (RSP-2025/126), King Saud University, Riyadh, Saudi Arabia. We also thank Saleh Idris, a labourer who assisted with project excavations, and who discovered panel JMI18. Open Access funding enabled and organized by Projekt DEAL. College of Asia and the Pacific, Australian National University, Canberra, ACT, Australia Heritage Commission, Ministry of Culture, Riyadh, Saudi Arabia Faisal Al-Jibreen, Fahad Al-Tamimi, Mohammed Al-Shamry, Ahmed Al-Shammari & Jaber Al-Wadani Physical Science and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia Giacomo Losi, Amir Kalifi & Frans van Buchem SFF Centre for Early Sapiens Behaviour (SapienCE), University of Bergen, Bergen, Norway Australian Research Centre for Human Evolution, Griffith University, Brisbane, QLD, Australia Department of Archaeology, College of Tourism and Archaeology, King Saud University, Riyadh, Saudi Arabia Human Origins Program, Smithsonian Institution, Washington, DC, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar : conceptualisation, data curation, funding acquisition, investigation, methodology, visualisation, writing—original draft preparation, writing—review and editing. : conceptualisation, data curation, funding acquisition, investigation, methodology, visualisation, writing—original draft preparation, writing—review and editing. : Data curation, formal analysis, investigation, methodology, visualisation, writing—original draft preparation. : data curation, formal analysis, investigation, methodology. : data curation, formal analysis, investigation, methodology, writing—review and editing. : data curation, formal analysis, investigation, methodology, writing—review and editing. : data curation, formal analysis, investigation, methodology. : data curation, funding acquisition, formal analysis, investigation, methodology, writing—review and editing. N.D.: data curation, investigation, methodology, writing —original draft preparation, writing—review and editing. MP: investigation, writing—original draft preparation, writing—review and editing. Correspondence to Maria Guagnin or Ceri Shipton. The authors declare no competing interests. Nature Communications thanks Joy McCorriston and the other anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Guagnin, M., Shipton, C., Al-Jibreen, F. et al. Monumental rock art illustrates that humans thrived in the Arabian Desert during the Pleistocene-Holocene transition. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44220-025-00508-1'>Macroeconomic income inequality, brain structure and function, and mental health</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 18:09:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Mental Health (2025)Cite this article Income inequality, a structural property of societies characterized by the unequal distribution of resources, is associated with adverse mental health outcomes during adolescence, which is a sensitive period of neurodevelopment. While previous research has explored the impact of individual-level socioeconomic factors on brain structure and function, the neurobiological mechanisms linking structural inequality to mental health disparities remain poorly understood. Here, using data from the Adolescent Brain Cognitive Development study, we investigated the associations between state-level income inequality, indexed by the Gini coefficient, and brain structure and functional connectivity in over 8,000 children aged 9–10 years (from 17 states in the USA). We analyzed whole-brain cortical thickness and surface area, and volume and region-specific measures of thickness and surface area, as well as functional connectivity within and between 12 brain networks, controlling for several individual-level and state-level confounders (for example, income, educational attainment, state-level incarceration rate and Medicaid expansion status). Mediation analyses were conducted to test whether brain metrics linked income inequality to mental health outcomes at 6-month and 18-month follow-ups. Higher income inequality was associated with reduced cortical thickness and surface area across widespread brain regions, as well as altered functional connectivity between multiple brain networks. Lower cortical volume and surface area, as well as connectivity between the default mode and dorsal attention networks, mediated the association between higher structural income inequality and greater mental health problems. Our findings reveal income inequality as a unique societal-level determinant of neurodevelopment and mental health, independent of individual socioeconomic status. Policies aimed at reducing inequality and strengthening social cohesion to mitigate its neurobiological and mental health impacts are needed. Economic inequality, characterized by the unequal distribution of income and wealth in a society, is associated with mental health and wellbeing outcomes beyond the influence of household income1,2,3. This may be particularly relevant during adolescence—a dynamic period of neurodevelopment when the brain undergoes rapid maturation and many mental health disorders begin to emerge4,5. Research increasingly underscores the link between higher income inequality and a range of adverse mental health outcomes in young people, including elevated rates of depression, anxiety and behavioral disorders6,7,8,9. Inequality-related stressors may contribute to heightened mental health vulnerabilities by shaping neurodevelopmental pathways, particularly those involved in emotion regulation and social cognition. Although numerous studies have examined how individual-level income is associated with brain development10,11, few have considered the broader role of structural income inequality in brain development and mental health. Income inequality has risen worldwide in recent decades12. Investigating whether and how inequality becomes neurobiologically embedded and heightens the risk of psychopathology during adolescence could provide valuable insights. Absolute poverty—typically defined as the inability to meet basic needs and operationalized based on thresholds set by the government and/or economists—has known effects on neurobiology10. If structural inequality has similar measurable associations, addressing it may gain broader acceptance across political ideologies. Income inequality, which refers to the disproportionate concentration of income within a small fraction of the population, can fundamentally alter the social environment and is a distinct construct from poverty. While income inequality can exacerbate poverty by concentrating resources among a small proportion of the population, it is a structural characteristic of a society that can influence mental health through unique pathways—such as increased social comparison, which is a fundamental process through which individuals evaluate themselves by comparing their attributes, abilities and achievements to those of others13, and reduced social cohesion—not captured by individual income or poverty alone1,3. Inequality fosters relative deprivation, social comparison and perceptions of limited social mobility1,3. This creates conditions that induce chronic stress, a well-established risk factor for psychopathology14. Although evidence from adolescent samples is limited, biological evidence is aligned with the idea that inequality contributes to increased stress. A study of over 19,000 individuals across four European countries found that C-reactive protein levels, a marker of inflammation, were lowest in more equal societies (for example, Switzerland) and highest in more unequal ones (for example, Portugal)15. Experimental findings are also consistent with this: Shapiro et al.16 showed that short-term exposure to inequality triggers physiological stress responses. Increased stress levels could explain the well-established links between inequality and mental health1,3,17. Stress neurobiology research shows that chronic stress exposure is associated with neural pathways involved in emotion regulation and cognitive control18, which are relevant for mental health. Extensive research has linked low individual income to altered brain structure and function, such as reduced cortical thickness and surface area and changes in functional connectivity19,20, factors closely associated with mental health21. Importantly, the association of individual socioeconomic status (SES) with brain volume has been shown to vary across European countries, with associations being stronger in some countries than others22—potentially reflecting broader structural factors such as inequality—a hypothesis that remains untested. Apart from a study examining electrophysiological brain dynamics in adults23 and a few studies examining links of local income disparity with gray matter24,25, the role of structural income inequality in shaping brain structure and function and, crucially, behavior outcomes in youth, remains unexplored. This gap underscores the critical need to explore how income inequality may influence neurobiology and mental health, independent of individual income. In this study, we leverage population-based data from the Adolescent Brain Cognitive Development (ABCD) study to assess associations between state-level income inequality, indexed by the Gini coefficient, and brain structure and functional connectivity in children aged 9–10 years. Specifically, we examine the associations of inequality with cortical thickness and surface area, analyzing both whole-brain and region-specific measures, as well as functional connectivity within and between 12 brain networks. Finally, we evaluated whether any implicated brain structure or connectivity metrics mediated the relationship between state-level inequality and mental health 6 and 18 months later, aiming to elucidate the pathways through which structural inequality contributes to mental health disparities. Overall, the present study aims to examine the neural mechanisms through which income inequality is associated with adolescent mental health. By examining neurobiological pathways, we aim to better understand how inequality contributes to mental health disparities and inform potential interventions. The sample consisted of 10,071 children (4,871 female; 56% non-Hispanic white; Table 1) from across 17 states in the USA (N = 8,412 for functional connectivity after excluding unusable scans). We used linear mixed-effects models to examine associations of income inequality (operationalized as the Gini coefficient) with average cortical thickness, total cortical volume, total surface area, regional cortical thickness (n = 34 variables) and surface area (n = 34 variables), and within- and between-network connectivity (n = 78 variables). Our analyses control for multiple key covariates, including participant age and sex assigned at birth, scanner model, in-scanner motion (for connectivity variables), household income-to-needs ratio, parental educational attainment, parent mental health and state-level variables including incarceration rate, Temporary Assistance for Needy Families (TANF) and Medicaid expansion status. We also accounted for the nested family structure. These covariates were included to isolate the unique contribution of structural inequality to brain and mental health outcomes. False discovery rate (FDR) corrections (PFDR < 0.05) were applied within cortical thickness, surface area and functional connectivity analyses. Higher income inequality was significantly associated with lower overall cortical volume (β = −2.93, s.e.m 0.49, t = −6.04, P < 0.001), average cortical thickness (β = −1.33, s.e.m. 0.55, t = −2.41, P = 0.016) and total surface area (β = −2.99, s.e.m. 0.49, t = −6.06, P < 0.001; Fig. Further, higher income inequality was associated with widespread patterns of lower cortical thickness (Fig. 1e and Table 2) across frontal, temporal, parietal and occipital regions (n = 48 variables). For a few regions, inequality was associated with higher thickness and surface area, including for the bank of superior temporal sulcus (STS) and supramarginal thickness, and parahippocampal area. The complete model output for all associations is available in Supplementary Table 2. a–c, The association between Gini coefficient and average cortical thickness (a), total surface area (b) and total cortical volume (c). d,e, The t-statistic values from linear mixed-effects models for significant associations between Gini coefficient and regional cortical thickness variables (d) and surface area variables (e). Only areas with significant associations are depicted in the figure. In sensitivity analyses, we adjusted for the average cortical thickness for thickness models and the total surface area in surface area models (Supplementary Table 4). Briefly, thickness results remained largely unchanged (22 out of 24 variables were still significant), whereas some surface area results were no longer significant (12 out of 22 variables remained significant). This suggests that inequality is associated with regional structure, over and above global effects. Similarly, we found a significant relationship between inequality and several (n = 46) connections between and within functional networks. Higher inequality was associated with alterations in connectivity within and between networks involved in higher-order cognition and attention, such as the cingulo-opercular network (CON), dorsal attention network (DAN), default mode network (DMN), frontoparietal network (FPN) and ventral attention network (VAN). Inequality was also linked to connectivity in networks related to sensory and motor functions, including the auditory network (AN), sensorimotor network (SMN) and visual network (VN) (Fig. The complete model output for all associations is available in Supplementary Table 3. a, A heat map of the t-statistic values (from linear mixed models) of the relationship between Gini coefficient and significant connectivity variables after correction for multiple comparisons (n = 46). Only the bottom half of the matrix has been displayed for ease of readability. b, A chord diagram for significant connections. Red and blue chords represent positive and negative associations, respectively. Additionally, to directly assess whether income inequality explained additional variance beyond individual-level SES, we compared models with and without the Gini coefficient for a few representative models (global brain measures and the first two significant connectivity variables). We found that including income inequality significantly improved the model fit for all brain outcomes tested and resulted in small but consistent increases in marginal R2 when the Gini coefficient was added to the models (Supplementary Table 5). This suggests that the Gini coefficient explains unique variance above individual-level SES and other covariates in the model. The main effects of parent income to needs and educational attainment are presented in Supplementary Tables 11 and 12. We then leveraged structural equation modeling, including the same covariates as the linear mixed-effects models, to test the role of the implicated brain variables as mediators of the association of inequality with total mental health problems at 6 months and 18 months later. The FDR was used to adjust for multiple comparisons within each set of analyses. Fit indices indicated acceptable to excellent fit for the structure models (root mean square error of approximation: mean 0.013, s.d. 0.004; comparative fit index: mean 0.997, s.d. 0.023) and connectivity models (root mean square error of approximation: mean 0.016, s.d. 0.003; comparative fit index: mean 0.993, s.d. There was a significant total effect of inequality on mental health at the 18-month follow-up but not the 6-month follow-up (Supplementary Tables 6–10). We found a significant indirect effect for total surface area and cortical volume, and connectivity between the DAN and DMN for the association between inequality and total mental health problems at both 6 months and 18 months later. Specifically, higher inequality was associated with lower surface area and volume and higher (that is, less negative) DMN–DAN connectivity, which were in turn associated with higher mental health problems (Fig. 3 and results for 6 months provided in the Supplementary Information). These results were significant even when total mental health problems at baseline were accounted for in sensitivity analyses (see Supplementary Information for the model output). a–c, The neural mediators between income inequality and total mental health problems for total surface area (a), total cortical volume (b) and DMN to DAN connectivity (c). The present study provides new evidence on the neural mechanisms through which structural income inequality contributes to psychopathology. Our findings show that structural inequality is associated with measurable differences in brain structure and functional connectivity, which have implications for subsequent mental health outcomes during early adolescence. Higher income inequality was associated with reduced cortical thickness and surface area across widespread brain regions spanning all four lobes of the brain, which is in line with our whole-brain findings. The present study also revealed that structural income inequality was associated with alterations in functional connectivity within and between multiple cortical brain networks, including both higher-order cognitive and sensorimotor systems. These findings of widespread associations between inequality and brain structure and connectivity extend prior research on the impact of individual socioeconomic factors10,11,19,20,26,27,28,29,30,31,32 on brain development by highlighting the role of structural income inequality as a unique societal-level determinant that operates independently of individual-level SES. Importantly, total brain volume and surface area, as well as connectivity between the DMN and DAN, mediated links between inequality and mental health 6 months and 18 months later. Given the evidence showing that inequality fundamentally alters the social environment—resulting in lower levels of social cohesion, trust and social capital, as well as higher levels of social comparison13—and the crucial role that social relationships play in our health and wellbeing33, it is unsurprising that inequality is associated with higher stress levels34. While not directly measured in our study, we speculate that these social changes and increased stress levels probably contribute to the profound impacts on the brain outcomes in children observed in our study. We discuss these ideas further below. A key finding of this study is that youth living in states with higher income inequality have, on average, lower surface area and thickness across widespread regions in the brain, spanning the frontal, parietal, temporal and occipital lobes. The implicated regions support a range of functions including executive function, emotion regulation, reward processing, attention and sensory integration, socioemotional processing and visual processing35,36,37,38,39,40. Inequality was also associated with differences in functional connectivity within and between numerous functional systems involved in both sensorimotor and higher-order cognitive functions. Together, these findings indicate that income inequality may shape brain development in a diffuse and pervasive manner, potentially influencing a wide range of cognitive, emotional and behavioral outcomes in children and adolescents. Importantly, our findings are consistent with and extend prior work on individual SES showing similarly widespread associations of SES with brain structure and connectivity19,20,29 and associations between SES and a range of outcomes in children41,42,43,44. We speculate that such widespread patterns may be a consequence of inequality-related chronic stress. While both household income and inequality generate chronic stress, with potential biological consequences, they may do so via different psychosocial pathways. For instance, absolute poverty may trigger stress via material deprivation and income insecurity, while inequality may amplify status anxiety and social comparison as inequality heightens the salience of socioeconomic hierarchies, leading to increased upward social comparisons and feelings of inadequacy3,45,46. Social comparison—a psychological process through which individuals evaluate themselves relative to others13—is heightened in unequal societies where status hierarchies are more salient. This fosters upward comparison and contributes to status anxiety, a persistent worry about one's social standing47. According to the status anxiety hypothesis, individuals in more unequal societies may experience greater stress about their social rank—even if they are not economically deprived. Supporting this, relative income (for example, income rank) is often a stronger predictor of psychological distress and life satisfaction than absolute income, particularly in high-inequality areas48,49,50,51. Experimental and observational studies have also linked higher inequality to increased materialism, status seeking and conspicuous consumption52. For children and adolescents, who are particularly sensitive to social comparison53, this may translate into a heightened preoccupation with peer status, social rejection and negative self-evaluation54, all of which lead to higher stress levels55. Growing up in the context of higher income inequality is also associated with greater victimization in adolescence56. Further, inequality may also contribute to higher levels of adolescent stress indirectly. For example, ample evidence shows that inequality degrades mental health in adults1,3,17, and poor parent mental health is associated with greater family conflict and reduced parental support57, both of which may contribute to adolescent stress. Biological and experimental evidence is in line with the idea that inequality contributes to stress15,16. Experiencing chronic stress may disrupt the hypothalamic–pituitary–adrenal axis and elevate cortisol levels, norepinephrine and adrenaline58. While short-term stress responses may be protective, chronic stress places strain on the brain and other organ systems58. In the brain, chronic stress may disrupt synaptic homeostasis by impairing the balance between synapse formation and pruning, driven in part by prolonged glucocorticoid exposure59. This can lead to widespread synaptic loss and reduced synaptic density59, which may contribute to the widespread connectivity alterations in large-scale functional networks observed in the present study. Additionally, chronic stress can reduce dendritic arborization complexity, potentially driving reductions in cortical volume and thickness60,61. Further research is needed to test these pathways directly. Income inequality not only exacerbates chronic stress but may also deprive adolescents of the protective effects of strong social ties, such as emotional support and shared resources, which may buffer these associations. For example, income inequality erodes social capital—defined as trust, networks and norms of reciprocity within communities—which is critical for fostering social cohesion and mental wellbeing62. Communities with high levels of inequality tend to experience greater segregation, reduced civic participation and weaker social networks, all of which can amplify feelings of isolation and mistrust63, contributing to the maintenance or exacerbation of chronic stress. This chronic stress may ultimately influence brain structure and function as described earlier. Another central finding of our study is that whole-brain cortical surface area and cortical volume, and connectivity between the DMN and DAN, mediated the association between income inequality and mental health problems. These results are consistent with prior work showing that the DMN and DAN are involved in emotion regulation, and disruptions in their connectivity have been associated with various mental health disorders64,65,66. Furthermore, our findings build on prior research showing associations between reduced cortical volume and surface area and various forms of psychopathology (for example, depression, anxiety and externalizing disorders)67,68,69,70,71. Our results suggest that income inequality may contribute to these associations by shaping neurodevelopment, although further research is needed to disentangle the mechanisms underlying these links. Notably, only a small subset of brain variables associated with income inequality mediated these links, leaving the exact contribution of these neural alterations to behavioral outcomes uncertain. We speculate that it is possible such effects would be evident in outcomes not examined in the present study—such as emotion regulation and cognitive function—or over a longer developmental timespan, such as late adolescence or young adulthood. Our findings suggest that structural income inequality is associated with neurobiological differences, even after accounting for absolute income and poverty. These brain differences, in turn, help explain links to adverse mental health outcomes. As such, structural inequality should be considered a harmful social determinant with clear biological sequalae, meriting attention in efforts to improve population mental health, regardless of political ideology. In other words, addressing absolute poverty alone is likely to be inadequate and policies aimed at reducing inequality, improving social capital and cohesion, and mitigating the effects of social comparison are needed. We have discussed possible interventions in depth in a recent paper (see ref. Briefly, economic reform including progressive taxation, increased social safety nets and universal healthcare are needed to alleviate the stressors that disproportionately affect children in more unequal societies3. Further, efforts to promote social capital, such as community-building initiatives and investments in public infrastructure, could mitigate the detrimental effects of inequality on mental health by fostering trust and social cohesion1,2,3. Interventions targeting status anxiety, such as programs promoting emotion regulation, social connectedness and feelings of self-worth, may help reduce the salience of social comparison among adolescents. In particular, schools could play a critical role by creating inclusive environments that minimize socioeconomic disparities and emphasize belonging and connectedness to promote mental health in young people72. Given our findings of links between structural inequality and brain structure and function, and evidence suggesting that inequality is associated with worse outcomes for everyone, even those not struggling financially2,17, such measures are urgently needed. Despite its strengths, this study has several limitations: (1) while the Gini coefficient we used provides a robust measure of income inequality, it did not capture other inequities, such as wealth disparities or access to healthy nutrition, that may also influence brain development. Although we relied on the Gini coefficient, other indices, such as the proportion of total income earned, Atkinson index, income ratios or the Robin Hood index may offer complementary insights73. For instance, the Theil index allows for the decomposition of inequality into between-group and within-group components74,75 (for example, between race versus within race), offering a more granular understanding of how different dimensions of inequality may relate to brain development. Future work could benefit from incorporating multiple measures of inequality to provide a more nuanced understanding of how different aspects of income distribution may differentially relate to brain outcomes. (2) Our findings are correlational. Experimental studies (for example, from behavioral economics) and longitudinal neuroimaging research, ideally grounded in causal inference frameworks, are needed to establish causal pathways. (3) While we speculate that the brain differences observed in our study may be due to the chronic stress associated with inequality, we did not measure perceived stress or biological markers of stress (for example, cortisol levels) directly. As such, any interpretations should be made with caution. Future work should test whether both self-reported and physiological indicators of stress mediate links between inequality and brain structure and function. Relatedly, stress and its neurodevelopmental consequences caused by structural factors such as inequality, and individual-level factors including poverty are not easily dissociable. (4) We chose to focus on state-level income inequality, which captures broader macrosocial factors and has been underexplored in developmental cognitive neuroscience. Future work should test the unique, distinct and joint effects of local inequality versus state inequality to better understand how different social contexts contribute to developmental outcomes76. (5) In our dataset, 21 sites span only 17 states, with 15 out of 17 states represented by a single site. This precluded us from including site as a random effect without compromising our ability to estimate state-level associations. While we controlled for scanner manufacturer and model, future research with broader geographic sampling and multiple sites per state could better disentangle site-specific variability from state-level predictors of brain structure and function. (6) The ABCD study does not include data from states with the highest poverty rates in the USA (for example, Mississippi, Louisiana and New Mexico). These findings therefore may not be generalizable to structurally disadvantaged states. However, our findings of inequality being associated with brain structure and function even in states with comparatively lower levels of poverty highlights the relevance of state-level contextual factors beyond absolute deprivation. (7) Averaging left and right hemisphere values may have masked potential hemispheric differences. This could be explored in future work with specific hypotheses. (8) This study examined whole-brain patterns and overall psychopathology and cannot comment on associations relevant for specific mental health domains. Future work should investigate more specific pathways to delineate risk and resilience mechanisms for distinct mental health domains. (9) We did not examine the role of residential segregation and homogeneity, which may influence children's development through shared pathways with inequality such as low social capital, fragmentation, exclusion and lack of belonging76,77,78. Future work that examines the extent to which these exposures share common versus distinct pathways is needed. Further, it would also be valuable to examine whether racial homogeneity within a state or region influences the strength of the association between inequality and children's outcomes in future work. Future research should examine the environmental and biological mechanisms linking inequality to brain structure and connectivity, such as neighborhood cohesion, prosocial behavior, parental stress, family conflict and stress pathways. Finally, while effect sizes are small, small effects can accumulate and have meaningful effects at the population level79. This study advances our understanding of how income inequality shapes child brain structure and function and mental health. Our findings highlight how structural inequality becomes biologically embedded to influence mental health. Addressing inequality at the societal level is essential to fostering environments that support healthy neurodevelopment and mental wellbeing for all children. This study utilized data from the ABCD study (release 5.1), an ongoing large-scale, multisite, longitudinal research study from the USA. Baseline data were collected from September 2017 to August 2018. The present study used neuroimaging data from the baseline time point and self-reported mental health from 6-month (when self-reported mental health was first assessed80) and 18-month (the last time point before the COVID-19 pandemic) follow-ups. The ABCD study aims to comprehensively track psychological and neurobiological development from late childhood to late adolescence. The study is conducted across 21 research sites (from 17 states). Schools within a 50-mile radius of each site—across public, charter and private institutions—were selected using probability sampling within the 21 defined catchment areas (see ref. Participants took part in clinical interviews, neuroimaging sessions, neurocognitive testing and completed various surveys. Written informed consent was obtained from parents or caregivers, and children provided assent. All participant rights were safeguarded by local institutional review boards. After excluding participants based on imaging quality and missing data on covariates, 10,071 children remained in the final analysis sample for brain structure and 8,412 for functional connectivity. Detailed demographic information for the sample has been provided below. We focused on structural inequality at the state level, which is intended to capture broader, system-level socioeconomic stratification—the policies, norms and institutional frameworks that shape income distribution at a population level. Inequality at the state level was assessed using the Gini coefficient, one of the most widely used measures of income inequality2,34,73, making it comparable with previous research. The Gini coefficient quantifies the extent to which income distribution among individuals or households within a population deviates from perfect equality. Values range from 0 (complete equality, where everyone has the same income) to 1 (complete inequality, where one person has all the income and everyone else has none). It is typically calculated from the Lorenz curve, which plots the cumulative proportion of income earned against the cumulative proportion of the population, ranked from poorest to richest. The coefficient represents the area between the Lorenz curve and the line of perfect equality, divided by the total area under the line of perfect equality. State-level Gini coefficient values were acquired for the year 2017 from the US Census Bureau. Importantly, larger geographic units, such as states or nations, offer more consistent and robust findings on the relationship between inequality and various outcomes than inequality at a smaller geographical scale such as the neighborhood34,82. For example, Wilkinson and Pickett82 found that the proportion of analyses reporting a supportive association between income inequality and health outcomes was 83% among international studies, 73% in studies using large subnational units (for example, states) and only 45% in studies using small areas such as neighborhoods. Similarly, Hsieh and Pugh83 concluded that consistent associations between income inequality and violent crime were found in studies using national or state-level data, but not in studies using smaller geographic units34. This pattern has been echoed in later multilevel meta-analyses34,84, supporting the idea that state-level measures more reliably capture meaningful variation in inequality. Further, inequality is a relatively stable construct and can be assumed to precede neurobiological changes temporally. More importantly, brain structure and function cannot influence state-level income inequality, making reverse causality a non-issue in this context. We report associations between the state-level Gini coefficient and state-level median income and racial homogeneity (percentage white) and income-to-needs ratio (averaged for individuals in the state) in Supplementary Table 1. We utilized youth-reported symptoms of psychopathology on the Brief Problem Monitor, which assesses symptoms over the past week using 19 items drawn from the Child Behavior Checklist, Teacher's Report Form and Youth Self-Report85,86. Items are rated as 0 (‘not true'), 1 (‘somewhat true') or 2 (‘very true') and are categorized into three domains (attention, internalizing and externalizing). We utilized the total problems raw score, which included scores on internalizing, externalizing and attention symptoms. Internal consistency in our sample was high (Cronbach's α = 0.85), indicating good reliability. The present study used data from two time points: the 6-month (the time point at which self-reported mental health was first assessed) and 18-month follow-up (which was the last timepoint of complete data before the pandemic). This was done to examine whether brain structure and connectivity mediated associations between inequality and mental health both in the short term and over a longer period, thereby testing the robustness and persistence of any observed effects. Race and ethnicity data were collected through two questions: (1) “What race do you consider the child to be?” (Asian, Black or African American, white) and (2) “Is the child of Hispanic/Latino/Latina descent?” (yes or no). Responses were categorized into five groups: Asian, Hispanic, non-Hispanic Black, non-Hispanic white and multiracial/multiethnic, and this classification is used in the study's data releases. This variable was binarized into non-Hispanic white and non-white. Models were adjusted for a range of theoretically motivated covariates87. The mean educational attainment of parents/caregivers (in years) was calculated by averaging the education levels of both parents/caregivers. In cases where data for both were unavailable, the information for the available parent or caregiver was used. The income-to-needs ratio was determined by dividing the median value of the household's income band (as defined by the ABCD study) by the federal poverty line for the respective household size. Parental mental health was the (responding) parent's total raw score on the self-reported Adult Self Report instrument of broad psychopathology. Additionally, we accounted for the average TANF benefit at the state level88,89 as low assistance for needy families can exacerbate inequality as well as increase individual-level stress, which in turn can influence neurodevelopment. Further, we covaried for incarceration rate90 (that is, the number of individuals incarcerated in the state out of every 100,000 in population), as more unequal states may have higher levels of crime, which can influence brain development and mental health, as well as Medicaid expansion (binary variable) as an indicator of whether the state had expanded Medicaid eligibility91. In sensitivity analyses for our mediation models, we accounted for parent-reported youth mental health problems (total problems subscale) based on the Child Behavior Checklist92. The imaging procedures used here have been comprehensively outlined by Casey et al.93. Neuroimaging was performed following standardized protocols across multiple sites using 3T magnetic resonance imaging scanners from Siemens, Phillips or General Electric, equipped with 32-channel head coils. A high-resolution 3D T1-weighted image with 1 mm voxel size was obtained for all participants. Further, four or five 5 min resting-state scans (eyes open) were performed to ensure a minimum of 8 min of relatively low-motion data. For further details, refer to Hagler et al.94. The preprocessing was carried out by the ABCD data analysis and informatics core, utilizing a uniform pipeline (for detailed procedures and quality control, refer to Hagler et al.94). Motion correction was implemented in real-time at the Siemens and GE sites using dedicated software tools. Both automated and manual methods were employed to assess data quality, and all images were reviewed by trained professionals to detect artifacts or abnormalities. Artifact severity in the cortical reconstructions of the postprocessed images was rated on a scale from 0 to 3, with 3 indicating the highest level of artifact. Based on these ratings, technicians recommended the inclusion of only those images rated 0 for further analysis. Signal-to-noise ratios and head motion were assessed using automated tools. FreeSurfer version 7.1.1 was used for cortical surface reconstruction and parcellation (Desikan–Killiany atlas). In this study, we focused on cortical thickness and surface area (34 variables each). Since no lateralized effects were hypothesized and prior work on individual-level environmental factors and brain structure has not yielded consistent results for lateralized effects10, values for the left and right hemispheres were averaged for the analysis. This is consistent with ours and others' prior work on this sample19,95, and a study by Taylor et al.96 showed that SES is largely consistently associated with volume in the left and right hemisphere in the ABCD sample. Scans were excluded if they did not meet the quality control inclusion criteria outlined by the data analysis, informatics and resource center of ABCD (see release notes for details). Functional magnetic resonance imaging time courses were projected onto FreeSurfer's cortical surface. Using these time courses, within- and between-network connectivity (Pearson correlation) was calculated based on the Gordon parcellation scheme (36) for 12 predefined resting-state networks: AN, CON, cinguloparietal network, DAN, DMN, FPN, retrosplenial temporal network (RTN), SMN (hand (H)), SMN (mouth (M)), salience network (SN), VAN and VN. These connectivity measures were then Fisher z-transformed, resulting in 78 dependent variables, 66 between-network connectivity variables and 12 within-network connectivity variables (with within-network connectivity representing the average correlation across all pairs of regions within each network). Scans were excluded if they did not meet the quality control inclusion criteria outlined by the ABCD data analysis, informatics and resource center (see release notes for details). To examine the effects of state-level income inequality on brain structure and functional connectivity, we conducted linear mixed-effects models (using the lme4 package in R). State-level income inequality, indexed by the Gini coefficient, was included as the main predictor, with brain structure (cortical thickness and surface area, 34 variables each) and functional connectivity (across 12 brain networks, 78 variables in total) as the dependent variables (with separate models for each outcome). We corrected for multiple comparisons using the FDR within each set of analyses for thickness, area and connectivity (PFDR < 0.05). P values were obtained using the lmerTest package. We ran three additional models for total cortical volume, total surface area and average cortical thickness. We included participant age at baseline, sex, scanner type and average framewise displacement (for connectivity measures), income-to-needs ratio at baseline, parent educational attainment at baseline and parent mental health problems at baseline as covariates to control for potential confounding factors. We additionally adjusted for TANF, incarceration rate and Medicaid expansion status (from 2016–2017) at the state level and included a random effect for family ID. All numerical variables were standardized for analyses. Semi-partial R2 values were obtained using the partR2 package in R. Given the small number of states (n = 17) and adjustments for key individual-level covariates (for example, age, sex, income-to-needs ratio, parent educational attainment and parent mental health) and site and state-level variables (for example, scanner manufacturer and model, state-level policies and cash assistance, and incarceration rates), clustering or random effects at the state or site level were deemed redundant as the included covariates adequately account for potential confounding and site-related variability. Since some covariates might arguably lie on the causal pathway from inequality to youth mental health (for example, parent mental health issues), the estimates obtained from our model are conservative. However, including this covariate adjusts for other environmental factors that may be associated with parent mental health issues—such as neglect, inconsistent caregiving and substance abuse—as well as genetic propensity to mental health problems. This approach allowed us to isolate effects associated with inequality to a greater extent. Further, to tease apart regional effects from global effects, we adjusted for average thickness (for regional thickness models) and total surface area (for regional surface area models). Model output for this additional analysis has been provided in Supplementary Table 4. Additionally, sensitivity analyses adjusting for racial homogeneity at the state-level (that is, percentage white) have been provided in Supplementary Tables 18–21. On secondary analyses, race/ethnicity was modeled as a moderator of the association between inequality and brain structure and connectivity to account for intersectional influences. As above, we corrected for multiple comparisons using the FDR within each set of analyses for thickness, area and connectivity (PFDR < 0.05). These results have been reported and are discussed in the Supplementary Information. For the significant variables identified in the linear mixed-effects models, we performed mediation analysis using structural equation modeling in the lavaan package in R. The analysis aimed to investigate whether brain structure or functional connectivity metrics mediated the relationship between state-level income inequality and total mental health symptoms assessed both 6 months and 18 months later. Scanner manufacturer/model, family ID (to account for family structure) and mean framewise displacement (for functional connectivity variables) were regressed out of the brain measures (that is, the mediator). All covariates (for example, sex, baseline income-to-needs ratio, baseline parental education and baseline parent psychopathology, 2016–2017 Medicaid, TANF and incarceration rate) were included as predictors in both the brain and outcome equations of the mediation models to ensure effects of interest were adjusted for these variables. Additionally, interview age at baseline was modeled in the ‘a' path and interview age at the respective mental health assessment (either the 6-month or 18-month follow-up) was modeled as a covariate in the ‘b' path. The structural models were not saturated and included only theoretically driven paths. To assess the significance of indirect effects, we employed a bootstrapping procedure with 5,000 resamples to obtain bias-corrected confidence intervals (CIs) and P values for the indirect pathways. As before, we corrected for multiple comparisons within each set of analyses (that is, thickness, surface area and connectivity; PFDR < 0.05). A moderated mediation model was conducted for variables where race/ethnicity moderates the association between inequality and brain variables (results reported in the Supplementary Information). In a sensitivity analysis, we tested whether results were robust to the inclusion of total mental health problems at baseline as a covariate. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The ABCD study data used in this project are publicly available (https://abcdstudy.org/). Access to the data is granted to qualified researchers via a data use agreement. For further information on how to obtain access to this dataset, visit the NIH Brain Development Cohorts data sharing platform (https://www.nbdc-datahub.org/). All data analyses used readily available programs (for example, open-source R code). No custom code was used. Patel, V. et al. Income inequality and depression: a systematic review and meta-analysis of the association and a scoping review of mechanisms. Wilkinson, R. G. & Pickett, K. The Spirit Level: Why Equality Is Better for Everyone (Penguin, 2010). Rakesh, D. et al. Economic inequality and mental health: causality, mechanisms, and interventions. Solmi, M. et al. Age at onset of mental disorders worldwide: large-scale meta-analysis of 192 epidemiological studies. Rakesh, D., Dehestani, N. & Whittle, S. in Encyclopedia of Adolescence 2nd edn (eds Troop-Gordon, W. & Neblett, E. W.) (Academic, 2024); https://www.sciencedirect.com/science/article/pii/B978032396023600124X Dierckens, M. et al. National-level wealth inequality and socioeconomic inequality in adolescent mental well-being: a time series analysis of 17 countries. Pabayo, R., Dunn, E. C., Gilman, S. E., Kawachi, I. & Molnar, B. E. Income inequality within urban settings and depressive symptoms among adolescents. Elgar, F. J., Gariépy, G., Torsheim, T. & Currie, C. Early-life income inequality and adolescent health and well-being. Farmer, G., MacDonald, S. W., Yamamoto, S. S., Wilkes, C. & Pabayo, R. Neighbourhood income inequality and general psychopathology at 3-years of age. Rakesh, D. & Whittle, S. Socioeconomic status and the developing brain—a systematic review of neuroimaging findings in youth. Rakesh, D., Whittle, S., Sheridan, M. A. & McLaughlin, K. A. Childhood socioeconomic status and the pace of structural neurodevelopment: accelerated, delayed, or simply different? Horowitz, J. M., Igielnik, R. & Kochhar, R. Trends in income and wealth inequality. Pew Research Center https://www.pewresearch.org/social-trends/2020/01/09/trends-in-income-and-wealth-inequality/ (2020). Wilkinson, R. & Pickett, K. The enemy between us: the psychological and social costs of inequality. Lupien, S. J., Juster, R.-P., Raymond, C. & Marin, M.-F. The effects of chronic stress on the human brain: from neurotoxicity, to vulnerability, to opportunity. Layte, R. et al. A comparative analysis of the status anxiety hypothesis of socio-economic inequalities in health based on 18,349 individuals in four countries and five cohort studies. Shapiro, M. S., Rylant, R., de Lima, A., Vidaurri, A. & van de Werfhorst, H. Playing a rigged game: inequality's effect on physiological stress responses. Wilkinson, R. & Pickett, K. The Inner Level: How More Equal Societies Reduce Stress, Restore Sanity and Improve Everyone's Well-Being (Penguin, 2018). McEwen, B. S. Allostasis and allostatic load: Implications for neuropsychopharmacology. Rakesh, D., Zalesky, A. & Whittle, S. Assessment of parent income and education, neighborhood disadvantage, and child brain structure. Rakesh, D., Zalesky, A. & Whittle, S. Similar but distinct—effects of different socioeconomic indicators on resting state functional connectivity: findings from the Adolescent Brain Cognitive Development (ABCD) study. Whittle, S., Zhang, L. & Rakesh, D. Environmental and neurodevelopmental contributors to youth mental illness. Walhovd, K. B. et al. Education and income show heterogeneous relationships to lifespan brain and cognitive differences across European and US cohorts. Article PubMed Central Baez, S. et al. Structural inequality and temporal brain dynamics across diverse samples. Vargas, T. G., Damme, K. S. F. & Mittal, V. A. Differentiating distinct and converging neural correlates of types of systemic environmental exposures. Parker, N. et al. Income inequality, gene expression, and brain maturation during adolescence. Farah, M. J. Socioeconomic status and the brain: prospects for neuroscience-informed policy. The neuroscience of socioeconomic status: correlates, causes, and consequences. Rakesh, D., Seguin, C., Zalesky, A., Cropley, V. & Whittle, S. Associations between neighborhood disadvantage, resting-state functional connectivity, and behavior in the adolescent brain cognitive development study: the moderating role of positive family and school environments. Sripada, C. et al. Socioeconomic resources are associated with distributed alterations of the brain's intrinsic functional architecture in youth. Rakesh, D., Sadikova, E. & McLaughlin, K. A. Associations among socioeconomic disadvantage, longitudinal changes in within-network connectivity, and academic outcomes in the ABCD study. Jednoróg, K. et al. The influence of socioeconomic status on children's brain structure. Noble, K. G. et al. Family income, parental education and brain structure in children and adolescents. Cohen, S. Social relationships and health. Pickett, K. E. & Wilkinson, R. G. Income inequality and health: a causal review. & Emory, E. Executive function and the frontal lobes: a meta-analytic review. Stuss, D. T. Functions of the frontal lobes: relation to executive functions. Grill-Spector, K. & Malach, R. The human visual cortex. Pozzi, E., Vijayakumar, N., Rakesh, D. & Whittle, S. Neural correlates of emotion regulation in adolescents and emerging adults: a meta-analytic study. & Petersen, S. E. The attention system of the human brain. Olson, I. R., Plotzker, A. The enigmatic temporal pole: a review of findings on social and emotional processing. Rakesh, D., Lee, P. A., Gaikwad, A. Annual research review: associations of socioeconomic status with cognitive function, language ability, and academic achievement in youth: a systematic review of mechanisms and protective factors. Lawson, G. M., Hook, C. J. A meta-analysis of the relationship between socioeconomic status and executive function performance among children. Bradley, R. H. & Corwyn, R. F. Socioeconomic status and child development. Reiss, F. Socioeconomic inequalities and mental health problems in children and adolescents: a systematic review. Cheung, F. & Lucas, R. E. Income inequality is associated with stronger social comparison effects: the effect of relative income on life satisfaction. Income inequality and status seeking: searching for positional goods in unequal US states. Delhey, J., Schneickert, C. & Steckermeier, L. C. Sociocultural inequalities and status anxiety: redirecting the Spirit Level Theory. Wood, A. M., Boyce, C. J., Moore, S. C. & Brown, G. D. A. An evolutionary based social rank explanation of why low income predicts mental distress: a 17 year cohort study of 30,000 people. Osafo Hounkpatin, H., Wood, A. M., Brown, G. D. A. & Dunn, G. Why does income relate to depressive symptoms? Testing the income rank hypothesis longitudinally. & Moore, S. C. Money and happiness: rank of income, not income, affects life satisfaction. Boyce, W. T., Sokolowski, M. B. & Robinson, G. E. Toward a new biology of social adversity. Pybus, K., Power, M., Pickett, K. E. & Wilkinson, R. Income inequality, status consumption and status anxiety: an exploratory review of implications for sustainability and directions for future research. Harter, S. The Construction of the Self, Second Edition: Developmental and Sociocultural Foundations (Guilford Publications, 2015). & Hawk, S. The mediating roles of upward social comparison and self-esteem and the moderating role of social comparison orientation in the association between social networking site usage and subjective well-being. He, D., Shen, X. The relationship between upward social comparison on SNSs and excessive smartphone use: a moderated mediation analysis. Elgar, F. J. et al. Association of early-life exposure to income inequality with bullying in adolescence in 40 countries. Van Loon, L. M. A., Van de Ven, M. O. M., Van Doesum, K. T. M., Witteman, C. L. M. & Hosman, C. M. H. The relation between parental mental illness and adolescent mental health: the role of family factors. Shonkoff, J. P. et al. The lifelong effects of early childhood adversity and toxic stress. Hall, B. S., Moda, R. N. & Liston, C. Glucocorticoid mechanisms of functional connectivity changes in stress-related neuropsychiatric disorders. Eiland, L. & Romeo, R. D. Stress and the developing adolescent brain. Glucocorticoids are critical regulators of dendritic spine development and plasticity in vivo. Ehsan, A., Klaas, H. S., Bastianen, A. & Spini, D. Social capital and health: a systematic review of systematic reviews. Kawachi, I., Kennedy, B. P., Lochner, K. & Prothrow-Stith, D. Social capital, income inequality, and mortality. Owens, M. M. et al. Investigation of psychiatric and neuropsychological correlates of default mode network and dorsal attention network anticorrelation in children. Jirsaraie, R. J. et al. Mapping the neurodevelopmental predictors of psychopathology. Xu, Z., Zhao, W., Wang, H., Tian, Y. & Lei, X. Functional connectivity between dorsal attention and default mode networks mediates subjective sleep duration and depression in young females. Regional brain volume in depression and anxiety disorders. Romer, A. L., Ren, B. Brain structure relations with psychopathology trajectories in the ABCD study. Brain structural associations with depression in a large early adolescent sample (the ABCD study). Shad, M. U., Muddasani, S. & Rao, U. Gray matter differences between healthy and depressed adolescents: a voxel-based morphometry study. Li, Q., Whittle, S. & Rakesh, D. (2025). Longitudinal associations between greenspace exposure, structural brain development, and mental health and academic performance during early adolescence. Raniti, M., Rakesh, D., Patton, G. C. & Sawyer, S. M. The role of school connectedness in the prevention of youth depression and anxiety: a systematic review with youth consultation. BMC Public Health 22, 2152 (2022). Maio, F. G. D. Income inequality measures. Hero, R. E. & Levy, M. E. The racial structure of inequality: consequences for welfare policy in the United States. Hero, R. E. & Levy, M. E. The racial structure of economic inequality in the United States: understanding change and continuity in an era of “great divergence”. Vargas, T., Conley, R. E. & Mittal, V. A. Chronic stress, structural exposures and neurobiological mechanisms: a stimulation, discrepancy and deprivation model of psychosis. Erdem, Ö., Prins, R. G., Voorham, T. A. J. J., van Lenthe, F. J. Structural neighbourhood conditions, social cohesion and psychological distress in the Netherlands. & Guhn, M. Ethnic density of regions and psychiatric disorders among ethnic minority individuals. Funder, D. C. & Ozer, D. J. Evaluating effect size in psychological research: sense and nonsense. Barch, D. M. et al. Demographic and mental health assessments in the adolescent brain and cognitive development study: updates and age-related trajectories. Garavan, H. et al. Recruiting the ABCD sample: design considerations and procedures. Wilkinson, R. G. & Pickett, K. E. Income inequality and population health: a review and explanation of the evidence. Hsieh, C.-C. & Pugh, M. D. Poverty, income inequality, and violent crime: a meta-analysis of recent aggregate data studies. Kondo, N. et al. Income inequality and health: the role of population size, inequality threshold, period effects and lag effects. Achenbach, T. M., McConaughy, S. H., Ivanova, M. Y. & Rescorla, L. A. Manual for the ASEBA Brief Problem Monitor for Ages 6–18 (ASEBA) Achenbach, T. M. & Rescorla, L. Manual for the ASEBA School-Age Forms and Profiles: An Integrated System of Multi-Informant Assessment (ASEBA, 2001). Saragosa-Harris, N. M. et al. A practical guide for researchers and reviewers using the ABCD study and other large longitudinal datasets. Increases in TANF cash benefit levels are critical to help families meet rising costs. Center on Budget and Policy Priorities https://www.cbpp.org/research/income-security/increases-in-tanf-cash-benefit-levels-are-critical-to-help-families-meet-0 (2014). Weissman, D. G., Hatzenbuehler, M. L., Cikara, M., Barch, D. M. & McLaughlin, K. A. State-level macro-economic factors moderate the association of low income with brain structure and mental health in US children. Maruschak, L. Correctional populations in the United States, 2017−2018. Centers for Medicare and Medicaid Services. Achenbach, T. M. in Encyclopedia of Clinical Neuropsychology (Springer, 2018); https://doi.org/10.1007/978-3-319-56782-2_1529-3 Casey, B. J. et al. The Adolescent Brain Cognitive Development (ABCD) study: imaging acquisition across 21 sites. Hagler, D. J. et al. Image processing and analysis methods for the Adolescent Brain Cognitive Development study. Hackman, D. A. et al. Association of local variation in neighborhood disadvantage in metropolitan areas with youth neurocognition and brain structure. Taylor, R. L., Cooper, S. R., Jackson, J. J. & Barch, D. M. Assessment of neighborhood poverty, cognitive function, and prefrontal and hippocampal volumes in children. acknowledges support from a Young Investigator Grant from the Brain and Behaviour Research Foundation (no. 32908) and a New Investigator Research Grant from the UKRI Medical Research Council (no. acknowledges funding from the National Institute of Mental Health (NIMH) (no. Data used in the preparation of this article were obtained from the ABCD study (https://abcdstudy.org) held in the NIMH Data Archive. This is a multisite, longitudinal study designed to recruit more than 11,000 children ages 9–10 years old and follow them over 10 years into early adulthood. The ABCD study is supported by the National Institutes of Health and additional federal partners under award numbers U01DA041048, U01DA050989, U01DA051016, U01DA041022, U01DA051018, U01DA051037, U01DA050987, U01DA041174, U01DA041106, U01DA041117, U01DA041028, U01DA041134, U01DA050988, U01DA051039, U01DA041156, U01DA041025, U01DA041120, U01DA051038, U01DA041148, U01DA041093, U01DA041089, U24DA041123 and U24DA041147. A full list of supporters is available at https://abcdstudy.org/federal-partners.html. A listing of participating sites and a complete listing of the study investigators can be found at https://abcdstudy.org/consortium_members/. ABCD consortium investigators designed and implemented the study and/or provided data but did not necessarily participate in the analysis or writing of this report. The views expressed in this paper are those of the authors and do not necessarily reflect the official views of the National Institutes of Health, the Department of Health and Human Services, the US federal government or ABCD consortium investigators. Department of Neuroimaging, Institute of Psychology, Psychiatry and Neuroscience, King's College London, London, UK Divyangana Rakesh & Dimitris I. Tsomokos UCL Institute of Education, Department of Psychology and Human Development, University College London, London, UK Department of Psychology, Harvard University, Cambridge, MA, USA Department of Health Sciences, University of York, York, UK Department of Global Health and Social Medicine, Harvard Medical School, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the study and research design, conducted analyses and wrote and revised the first and subsequent drafts of the paper. consulted on the analysis plan and provided additional feedback on the paper. provided feedback on the analysis plan, critical feedback on the paper and expert guidance and support. All authors approved the final version before submission. All authors have read and agreed to the published version of the paper. Correspondence to Divyangana Rakesh. The authors declare no competing interests. Nature Mental Health thanks Pilyoung Kim, Genevieve Patterson, Xueyi Shen and the other, anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Tables 1–23, Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Rakesh, D., Tsomokos, D.I., Vargas, T. et al. Macroeconomic income inequality, brain structure and function, and mental health. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63454-7'>Induction of experimental cell division to generate cells with reduced chromosome ploidy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 17:34:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Somatic cell nuclear transfer (SCNT) enables the direct reprogramming of somatic cells into functional oocytes, albeit with a diploid genome. To address ploidy reduction, we investigated an experimental reductive cell division process, termed mitomeiosis, wherein non-replicated (2n2c) somatic genomes are prematurely forced to divide following transplantation into the metaphase cytoplasm of enucleated human oocytes. However, despite fertilization with sperm, SCNT oocytes remained arrested at the metaphase stage, indicating activation failure. Fertilized human SCNT oocytes progressed through normal embryonic cell divisions, ultimately developing into embryos with integrated somatic and sperm-derived chromosomes. While our study demonstrates the potential of mitomeiosis for in vitro gametogenesis, at this stage it remains just a proof of concept and further research is required to ensure efficacy and safety before future clinical applications. In women, age-related decline in oocyte quality and quantity has been recognized as a primary contributing factor to infertility, particularly after their mid-thirties1. In vitro fertilization (IVF) is ineffective for patients lacking functional gametes, leaving them with no option to have genetically related children except through the use of donor gametes. Reprogramming a patient's somatic cells into induced pluripotent stem cells (iPSCs), followed by differentiation into functional sperm or oocytes—a concept known as in vitro gametogenesis (IVG)—holds immense therapeutic potential, especially for patients without viable gametes. Proof-of-concept for female IVG has been established in mouse models2; however, due to significant differences in germ cell development between mice and humans, the translation of IVG to human oocytes remains elusive. Unlike the differentiation of iPSCs into adult somatic cells, IVG-derived oocytes must acquire maternal cytoplasmic factors essential for inducing totipotency after fertilization. They must also undergo proper meiotic recombination, chromosome ploidy reduction, and imprint reprogramming to meet the requirements for future clinical applications. Notably, the natural development of human oocytes from pluripotent progenitors within an embryo is a lengthy process, taking more than a decade to reach functional maturity. For clinical applications, IVG must substantially shorten this timeline to a few weeks. Functional oocytes can also be generated through direct reprogramming of somatic cells via donor oocyte cytoplasm following somatic cell nuclear transfer (SCNT)3,4. SCNT-based IVG could offer several advantages, including reprogramming of transplanted somatic cell genomes directly to the oocyte-like epigenome by natural oocyte cytoplasmic factors5,6. Furthermore, the cytoplasm of in vivo matured donor oocytes provides essential maternal factors and germline mitochondria critical for subsequent induction of totipotency and full-term development of SCNT embryos. Since the cytoplasm of enucleated metaphase II (MII)-arrested oocytes retains metaphase activity, implanted somatic chromatin rapidly condenses to form a metaphase-like spindle, resulting in the establishment of an MII-arrested SCNT oocyte7,8. Given that SCNT oocytes are diploid, fertilization with sperm has not been considered due to the risk of triploidy. Conventional SCNT applications instead utilize artificial activation to induce exit from metaphase arrest while preventing cytokinesis and chromosome loss9,10. Simultaneously, artificial activation triggers the release of maternal cytoplasmic factors responsible for inducing totipotency and initiating embryonic mitotic cycles. The developmental potential of SCNT oocytes has been well demonstrated by the successful production of live cloned mammals and the generation of functional human blastocysts and embryonic stem cells (SCNT-ESCs)8. With the ultimate goal of developing infertility treatments for individuals without functional oocytes, we recently demonstrated that ploidy reduction can be experimentally achieved in mouse SCNT oocytes following fertilization with sperm11. Our findings indicated that premature metaphase spindles, comprising single-chromatid chromosomes formed after transplanting non-replicated, G0/G1-arrested somatic cell nuclei into enucleated MII oocytes, are capable of discarding one set of parental homologs into a pseudo polar body (PB) upon fertilization12. SCNT zygotes consisting of haploid somatic and sperm pronuclei can develop into diploid embryos and produce live mouse offspring11,12. In this study, we adapted mitomeiosis for human SCNT oocytes and investigated whether metaphase spindles are functional and capable of discarding somatic chromosomes into the PB upon fertilization or artificial activation. Lastly, we evaluated the integration of somatic and sperm chromosomes in preimplantation embryos derived from fertilized human SCNT oocytes. Typical mitotic and meiotic cell divisions are preceded by a single round of DNA replication during the S-phase of the cell cycle resulting in duplication of chromosomes to two sister chromatids (2n4c; Fig. In the mitotic metaphase, sister chromatids from each chromosome are pulled to opposite poles of the spindle and segregated upon cytokinesis into two genetically identical daughter cells with a diploid number of chromosomes (2n2c). In contrast, during the female meiotic metaphase I, homologous chromosomes are paired with each other within a spindle and separated upon reductional division to haploid daughter cells, i.e. an oocyte and a polar body 1 (PB1), each containing one copy of each homolog (1n2c). Since chromosomes still contain two sister chromatids, the second meiotic division bypasses the S-phase and directly enters metaphase II (MII) and remains arrested at MII until fertilization with sperm. Upon fertilization, an MII oocyte exits from metaphase arrest and similar to mitosis, segregates sister chromatids to the female pronucleus (PN) and the PB2 (1n1c; Fig. A schematic cell cycle and chromosome segregation mechanisms to daughter cells during canonical mitosis, meiosis I and meiosis II vs. mitomeiosis, an experimental reductive cell division induced in non-duplicated (2n2c) somatic cell genomes. a Chromosome dynamics and segregation during two consecutive meiotic cell divisions. Left panel shows the reductive meiosis I, where the duplicated homologous chromosomes consisting of sister chromatids pair, synapse and crossover followed by their segregation to the MII oocyte and its PB1. Right panel depicts the second meiotic division, where, similar to mitosis, sister chromatids segregate to the zygote and its PB2. b Left panel presents the canonical mitotic cell cycle, where following chromosome duplication, sister chromatids of each chromosome are segregated into two daughter cells maintaining the diploid chromosome content during cell divisions. Depending on chromosome positioning and attachment to the spindle, subsequent segregation of chromosomes to the zygote and the PB can result in either randomly reduced ploidy or haploidy in daughter cells. We recently modified the conventional mouse SCNT and induced an experimental reductive cell division by mitomeiosis (Fig. Progression through various cell cycle stages is strictly regulated by DNA integrity checkpoint regulatory mechanisms including S/G2 and G2/M checkpoints. These checkpoints play a crucial role in ensuring that the metaphase does not commence if DNA replication is incomplete13,14. In meiosis, the cell cycle checkpoint blocks progression from metaphase to anaphase if positions of chromosomes and formation of a proper bipolar spindle are compromised15. Therefore, we initially investigated if nonduplicated G0/G1 (2n2c) genomes of human fibroblasts can prematurely enter the metaphase upon nuclear transfer into enucleated, MII-arrested cytoplasm of human oocytes. In vivo matured oocytes were collected following routine clinical ovarian stimulation and retrieval from young, healthy egg donor volunteers after obtaining informed consent8. Spindle-chromosomal complexes from 270 mature, MII-arrested oocytes containing the PB1 were removed shortly after retrieval and cytoplasts were fused with G0/G1-arrested fibroblasts following our previously developed methods8. Premature onset of metaphase was monitored by observation of de novo spindle formation in SCNT oocytes using noninvasive imaging under polarized microscope equipped with the Oosight system8. Visible spindles, similar to those seen in control intact MII oocytes, became first detectable in SCNT oocytes 1 h after fusion, but the majority of spindles (77.5% ± 4.8) appeared after 2 h (Fig. SCNT spindles were elongated in shape and measurements of length and width were similar to spindles in intact MII oocytes (Fig. a De novo spindles formed in SCNT oocytes become clearly visible after 2 h post fusion. Black dots in the graph indicate individual values. 1 h group: 28 SCNT oocytes examined over 2 independent experiments. 2 h group: 215 SCNT oocytes were examined over 17 independent experiments. b Spindle size measurements in control MII and SCNT oocytes using Oosight imaging. Bar plots show average length and width of spindles measured in control MII (n = 4) and SCNT (n = 21) oocytes. Black and white dots in the graph indicate individual values. c Spindle morphology visualized with Oosight live imaging. Upper-left, control MII oocyte with typical spindle image. Upper-right, SCNT oocyte spindle with similar shape formed 2 h after fusion. d SCNT oocyte spindle components immunolabeled with antibodies detecting microtubules (α-tubulin-green), centromeres (kinetochore-red) and chromosomes (DAPI-blue) and visualized under fluorescence microscope. e Representative images of abnormal spindles observed in 60% of SCNT oocytes 2 h post fusion, including small, round, and multipolar configurations Scale bar 2 μm. f Failed activation after fertilization of SCNT oocytes, as indicated by reduced PB extrusion, PN formation, cleavage and high embryonic arrest compared to controls. Bar plots show the mean of 8 independent replicate experiments. Two-tailed unpaired student's t test was used for comparison. Black and white dots in the graph indicate individual values. Selected SCNT oocytes were fixed at several critical time points and immunolabeled to visualize spindle microtubules (α-tubulin), chromosome centromeres (kinetochore), and stained with DAPI (for DNA). These oocytes were then imaged under fluorescence microscopy to monitor spindle formation dynamics. Somatic cell nuclei in the interphase stage persisted in SCNT oocytes for up to 30 min post-fusion (Fig. By 1 h, SCNT oocyte nuclei underwent nuclear envelope breakdown (NEBD), premature chromatin condensation (PCC), and initiated microtubule formation (Fig. Bipolar SCNT spindles, with chromosome centromeres positioned at the equatorial region similar to control MII oocytes, were observed 2 h post-fusion (Fig. However, abnormal spindle morphology was also noted in majority of SCNT oocytes including small, round, or multipolar configurations (Fig. These findings provide important insights into spindle formation dynamics during mitomeiosis in human oocytes and support our conclusion that, following fusion with enucleated MII oocyte cytoplasm, somatic 2n2c chromatin directly enters metaphase, bypassing the natural S-phase. Next, we asked if human SCNT oocytes can exit from the MII-arrest and segregate non-duplicated chromosomes consisting of single chromatids to daughter cells. Naturally, mature MII oocytes remain arrested at the metaphase until fertilization with sperm which triggers exit from metaphase and sister chromatid segregation into the PB2 and the female PN, a signaling mechanism commonly referred to as oocyte activation. Intact control MII (n = 30) and SCNT oocytes (n = 41) with visible spindles were fertilized with sperm donated by a healthy research donor using intracytoplasmic sperm injection (ICSI). In the control group, 82.9% ± 9.5 fertilized oocytes extruded the PB2 and 79.8% ± 9.2 formed two PNs (female and male) visible after 6 h post fertilization (Fig. In contrast, only 23.4% ± 8.7 of sperm injected SCNT oocytes extruded the PB and 17.4% ± 9.3 formed PNs but PN formation was delayed up to 12 h post ICSI. While all control zygotes cleaved and 59% ± 16.1 reached blastocysts, only 25% of SCNT zygotes cultured cleaved but arrested at the 2-cell stage (Fig. It is important to note that the 25% activation rate refers specifically to PB extrusion, representing completion of the second division in mitomeiosis, rather than failure of subsequent mitotic cleavage. These results indicate that a nonduplicated (2n2c) genome can enter prematurely into metaphase when nuclei of G0/G1 somatic cells are directly transplanted into enucleated MII-arrested oocyte cytoplasm. However, natural, sperm-induced activation is insufficient to release SCNT oocytes from the MII arrest. As the percentage of abnormal spindles did not directly correlate with the 77% that failed polar body extrusion upon sperm fertilization, it may be reasonable to suggest that additional factors beyond spindle morphology contribute to activation failure. While the exact mechanisms responsible for failed fertilization of SCNT oocytes remain unclear, we reasoned that the persistent metaphase arrest could be due to failed activation. MII arrest is maintained by high activity of Maturation Promoting Factor (MPF) consisting of cyclin-dependent protein kinase 1 (Cdk1) and cyclin B (Fig. Sperm entry during fertilization triggers Ca2+ oscillations leading to degradation of cyclin B and subsequently inactivation of Cdk1 and exit from the MII arrest18. Alternatively, exit from the MII arrest can be induced experimentally, following so called artificial oocyte activation approaches10. Artificial activation is a critical step for conventional SCNT allowing exit from the MII arrest in SCNT oocytes without fertilization with sperm and thus avoiding possible triploidy. In addition to exit from the MII arrest, conventional activation procedures also block cytokinesis in SCNT oocytes preventing chromosome segregation into PBs10. We previously established sequential artificial activation stimuli critical for human SCNT oocytes to exit from the MII arrest and develop to blastocysts and SCNT-ESCs8. However, implementation of 6-dimethylaminopurine (6-DMAP), a broad protein phosphorylation inhibitor for conventional SCNT activation also prevents cytokinesis and PB extrusion10. a Schematic of the MII arrest mechanism and rescue of SCNT oocyte fertilization by supplemental activation with electroporation+roscovitine. Two tail unpaired student's t test was used for comparison. c Bright-field (BF) image of an SCNT zygote featuring a single PN and a PB (arrows) following activation with electroporation+roscovitine. Note that unfertilized but artificially activated SCNT zygotes predominantly display a single pronucleus (1 PN). In contrast, activated and sperm fertilized SCNT zygotes contain two pronuclei originating from somatic and sperm genomes. Bar plots show the mean of 8 independent experiments. Two tail unpaired student's t test was used for comparison. Black and white dots in the graph indicate individual values. e BF image of preimplantation embryo development following SCNT oocyte fertilization supplemented with electroporation+roscovitine treatment. Note: All SCNT zygotes that developed to blastocysts had successfully extruded polar bodies, indicating completion of mitomeiosis. Therefore, we tested an alternative activation for the ability to induce exit from the metaphase arrest without interfering with cytokinesis and PB extrusion. We initially mimicked calcium elevations by electroporation of MII and SCNT oocytes in Ca2+-containing D-sorbitol buffer8 and monitored release from the MII arrest. Electroporation alone induced robust cytokinesis and PB2 extrusion in the control MII oocyte group, but pronuclear formation was significantly delayed (10–12 h) compared to that seen in sperm fertilized zygotes (6 h). Moreover, all zygotes failed to cleave and remained arrested at the 1-cell stage (Fig. Upon electroporation, only 16.7% ± 10.5 of SCNT oocytes extruded the PB and 13.3% ± 9.7 formed one PN also visible late, 10–12 h after electroporation. Generally, calcium elevations alone are not sufficient to induce release from the MII arrest and initiate embryonic mitotic cycles10. Additional stimuli are required to inactivate MPF in human or monkey MII oocytes. Therefore, we treated control intact MII (n = 15) and SCNT (n = 74) oocytes with electroporation followed by 4-h incubation with roscovitine, a selective cyclin-dependent kinase inhibitor, in order to downregulate MPF activity while also allowing cytokinesis and PB extrusion (Fig. In the control MII group, 83.3% ± 10.5 oocytes extruded the PB2 and formed a single PN, 6–7 h after electroporation (Fig. In SCNT group treated with this sequential activation, 68.6% ± 7 of oocytes extruded the PB and 57.7% ± 6.2 formed a single PN, visible within 6–7 h after electroporation (Fig. Imaging of immunolabelled SCNT oocytes 1 h after activation demonstrated an anaphase-like microtubule arrangement with somatic chromosomes situated at both poles of the spindle (Fig. The majority of zygotes in both control and SCNT groups cleaved (91.7% ± 4.6 and 93.8% ± 6.3, respectively) and while 45.8% ± 16.2 of control (parthenogenetic) embryos reached the blastocyst stage, SCNT embryos arrested at the 4-10-cell stage. To determine if this artificial activation can rescue SCNT oocytes from the MII arrest upon fertilization with sperm. Control MII (n = 30) and SCNT (n = 82) oocytes were fertilized with sperm by ICSI and then SCNT oocytes activated with electroporation+roscovitine. All control and most SCNT zygotes with 2 PNs cleaved (100% and 83.3% ± 16.7, respectively), while 59% ± 16.1 of control embryos reached the blastocyst stage, only 8.8% ± 5.2 fertilized SCNT embryos developed to blastocysts (Fig. Our results showed that supplemental activation with electroporation and roscovitine can rescue the fertilization by sperm and release SCNT oocytes from the MII arrest. Moreover, this artificial activation does not affect normal fertilization nor development of control, intact MII oocytes suggesting its potential applications for assisted activation for IVF cases with fertilization failures. In summary, a diploid but non-replicated somatic genome (2n2c) can be forced to enter prematurely into a metaphase of mitomeiosis by SCNT. Moreover, upon assisted activation, SCNT oocytes consisting of 2n2c somatic chromosomes can bypass spindle checkpoint arrest and resume embryonic cell divisions. Next, we asked how 2n2c somatic chromosomes in human SCNT oocytes are segregated to daughter cells (i.e. PN and PB) upon exit from metaphase of mitomeiosis. To identify all human chromosomes and distinguish somatic parental homologs from each other and from sperm (total of 69 chromosomes), we developed a custom, whole chromosome tracing approach using an amplicon sequencing assay (Ampliseq). A family consisting of parents from diverse ethnic backgrounds and their daughter were recruited to participate in this research, as was an unrelated research sperm donor who also provided blood DNA and skin biopsies for this study. Initially, WGS was performed on genomic DNA from all four study subjects and screened for presence of unique SNVs discriminating maternal and paternal somatic chromosomes from sperm homologs. WGS data were then analyzed to identify informative triallelic loci on all autosomal and X chromosomes that would allow distinction between maternal and paternal chromosomes in SCNT oocytes generated from daughter's skin fibroblasts from sperm homologs in fertilized SCNT embryos. We selected a total of 630 highly confident loci covering 584 triallelic and 47 biallelic genomic variants in autosomal and X chromosomes and used this database to build the targeted AmpliSeq sequencing platform. We then validated the AmpliSeq on DNA samples from the study subjects confirming consistent capture between 92 and 95% of targeted genomic loci (min. 598 of 630) necessary for determining homologous chromosome origin in SCNT oocytes and embryos (Supplementary Fig. We then used daughter's skin fibroblasts arrested at the G0/G1 of the cell cycle for transfer into enucleated MII oocyte cytoplasm. Human SCNT oocytes containing visible spindles were either activated with electroporation+roscovitine or fertilized with sperm and then activated with electroporation+roscovitine and PBs and PNs or individual blastomeres of cleaving embryos were collected from each of 113 SCNT embryos. Only SCNT embryos that successfully extruded polar bodies were selected for AmpliSeq analysis. The DNA samples were processed by WGA, sequenced by AmpliSeq and the chromosomal content of 90 embryos was suitable for analysis while the remaining samples were lost during collection or sequencing. Paired analysis of both PN or blastomeres  and PB revealed that in 41 (45.6%) SCNT zygotes segregation of somatic chromosomes did not occur (Fig. Among the remaining 49 SCNT embryos, segregation patterns of 46 chromosomes to the daughter cells ranged from 3:43 to 23:23 (Fig. We removed 8 SCNT embryos with partial segregation patterns (10 or fewer chromosomes) from the subsequent analyses to avoid artificial biases during computation of segregation patterns (Fig. Of the remaining 41 embryos, 22 provided paired, readable somatic chromosome sequences for both PN and PB compartments, while in the remaining samples clear sequencing reads were available only for PN or PB. The mean somatic chromosome count in PNs (N = 38) was 22.8 ± 1 while in PBs (N = 25) the mean count was 23.3 ± 1.3 (Fig. While similar to meiosis, mitomeiosis achieves numerical chromosome reduction, the random segregation pattern and absence of recombination represent fundamental differences from conventional meiotic mechanisms. a Percentage of SCNT zygotes with segregated and non-segregated somatic chromosomes (N = 90). b Somatic chromosome counts in individual SCNT zygotes (N = 49). Each dot represents an individual SCNT zygote. Red dots indicate zygotes removed from the subsequent segregation pattern analyzes. More detailed segregation results for each zygote are presented in Supplementary data 1, 2, 3, 4. c Mean number of chromosomes counts in PNs and PBs of SCNT zygotes. Black and white dots in the graph indicate individual values. d Mean number of segregated homologous chromosomes in PNs and PBs. Black and white dots in the graph indicate individual values. e Comparison of homolog segregation pattern observed in PNs (N = 38) with random segregation pattern derived by the Monte Carlo simulation. Horizontal lines represent the mean; (p = 0.93, Wilcoxon rank sum test). f Comparation of homolog segregation pattern observed in PBs (N = 25) with random segregation pattern derived by the Monte Carlo simulation. Horizontal lines represent the mean; (p = 0.76, Wilcoxon rank sum test). g No linear relationship was found between chromosome length and homolog segregation frequency; R2 = 0.02. h Random distribution of maternal and paternal homologs to PNs and PBs for all chromosomes except the chromosome 8 that showed preferential segregation of maternal homologs to PNs and paternal homologs to PBs. Note: only SCNT zygotes that extrude polar bodies were included for analyses. Note: analysis focuses on somatic-origin chromosomes only; sperm chromosomes in fertilized samples are present but excluded from this segregation analysis to enable aggregation between fertilized and non-fertilized SCNT oocytes. Next, we examined the segregation pattern of each 23 pairs of maternal and paternal somatic homologous chromosomes during mitomeiosis in SCNT zygotes. The mean number of properly segregated somatic homologs in PNs was 10.9 ± 2.5 and 11.1 ± 2.4 in PBs, ranging in individual SCNT zygotes from a minimum of 6 to a maximum of 16 (Fig. To evaluate the homolog segregation mechanism, we compared the distribution seen in human SCNT zygotes to a Monte Carlo simulation test calculated for random segregation of 23 pairs. The mean number of segregated homologs expected by random distribution in this simulation was 11/11.5, ranging from a minimum of 6/7 to a maximum of 16 homologous chromosomes segregated properly (Fig. This simulated segregation was statistically similar to that seen in SCNT zygotes, suggesting that in contrast to canonical meiosis, segregation of parental somatic homologs during mitomeiosis is random. We also analyzed segregation dynamics of individual chromosomes based on their relative length but did not observe any correlation with homolog segregation frequency (Fig. We also analyzed if chromosome morphology (metacentric vs. acrocentric) influenced segregation patterns, as recent studies have suggested differential behavior of acrocentric chromosomes in the meiotic context. However, we observed no significant correlation between chromosome morphology and segregation frequency during mitomeiosis. We next analyzed distribution of paternal vs. maternal homologs to PNs and PBs and concluded random distribution of all pairs except for chromosome 8 that showed preferential segregation of maternal homologs to PNs and paternal homologs to PBs (Fig. The bias observed for this particular chromosome segregation rates is unexpected considering that chromosome 8 is relatively moderate in size and aneuploidy rate in human IVF embryos is relatively low compared to some other chromosomes. The underlying mechanism behind this intriguing exception to the otherwise random segregation pattern for other chromosomes remains unclear and warrants further investigations. We then analyzed all segregated somatic homologous chromosomes containing AmpliSeq reads for the entire length of chromosome for crossover recombinations. The AmpliSeq profiles of all segregated chromosomes in SCNT zygotes were identical to the original fibroblasts used for SCNT, indicating absence of recombination after mitomeiosis (Fig. We next asked if SCNT zygotes produced by artificial activation can resume subsequent mitotic cell divisions and replicate somatic chromosomes. DNA isolated from individual blastomeres of nine (2–10 cell stage) cleaving SCNT embryos and DNA from pooled cells of two embryos (6–8 cell stage), were processed for sequencing. These early stage embryos were chosen to ensure accurate representation of the initial chromosome segregation patterns before additional chromosomal abnormalities potentially arising after embryonic genome activation. Single-cell chromosome analysis indicated that all 9 embryos were uniform with every sister blastomere carrying the same number and origin of somatic chromosomes (Supplementary data 3, 5). Three embryos carried the full diploid set of 46 somatic chromosomes indicating that segregation of chromosomes to the PB did not occur in these embryos (Supplementary data 3,5). The remaining 6 embryos carried a reduced number of chromosomes ranging from N = 14 to N = 23, consisting of mixture of various maternal, paternal or both parental chromosomes while some chromosome pairs were absent altogether (Fig. The total number of chromosomes in two SCNT embryos from pooled cells was also reduced (N = 34 and N = 41; Supplementary data 3). These results suggest that SCNT zygotes can resume embryonic mitotic cell divisions and replicate and distribute reduced number of somatic chromosomes. a Uniform 4-cell stage SCNT embryo #89 produced by artificial activation alone. AmpliSeq detected presence of 23 somatic chromosomes in every blastomere with random homolog segregation. Individual sister blastomeres carried various number of sperm and somatic chromosomes. Analysis of pooled cell DNA from three SCNT blastocysts detected the presence of a total N = 49, N = 28, and N = 27 chromosomes in each embryo, respectively (Supplementary data 4 and 6). All 23 sperm homologs were present in the blastocysts, while the number and origin of somatic chromosomes varied. To determine mosaicism within sister blastomeres, we analyzed additional nine SCNT embryos at the 2–6-cell stage by single-cell sequencing of individual blastomeres. All 23 sperm homologs were present in all blastomeres of these embryos, while number and origin of somatic chromosomes varied. As expected, some somatic chromosomes were of paternal or maternal origin while for other chromosomes both homologs were present or absent (Fig. Mosaic embryos consisted of blastomeres carrying either sperm or somatic chromosomes only or mixture of somatic and sperm chromosomes (Fig. All 23 sperm chromosomes were present in most mosaic embryos except one embryo where not all sperm chromosomes were detected. To assess genomic stability following mitomeiosis, we performed detailed chromosome analyses of polar bodies and embryos derived from fertilized or activated SCNT oocytes (Fig. Our results indicate that mitotic cycles after mitomeiosis proceed normally, with both somatic and sperm chromosomes replicating faithfully despite initial aneuploidy. In uniform embryos, chromosome numbers and composition were consistent among all sister blastomeres, suggesting stable maintenance of the initial chromosomal content throughout early embryonic divisions. Overall, our results suggest that somatic and sperm chromosomes can integrate into a joint genome in some SCNT embryos, while in others the two genomes segregated to different sister blastomeres and formed a mosaic embryo. Reverse diploidy reduction to haploidy occurs naturally only in gamete precursors following the reductive meiosis I division. In prophase of female meiosis I, duplicated chromosomes, each consisting of two sister chromatids, undergo extraordinary rearrangements by sensing, recognizing and pairing with their homologs followed by crossover recombination21. These homolog alignment and crossover events collaborate with sister chromatid cohesion to create temporary physical links, called chiasmata, that connect homologs and allow them to orient and subsequently segregate one parental homolog into the MII oocyte and another into the PB1, thereby achieving haploidy. The reductional meiosis I is followed by an equational meiotic II division where, similar to mitosis, sister chromatids are segregated to opposite spindle poles eliminating one set into the PB2. Human oocyte haploidization is also an extremely long process initiating meiosis I in the first trimester of fetal development and completing the meiosis II roughly 12 or more years later, after the fertilization. Prophase I is considered the longest stage compared to other meiotic stages22. Mirroring these mechanistically complex and lifelong meiotic processes of natural oogenesis for human IVG remains challenging. We have successfully utilized the residual metaphase activity and meiotic machinery in the cytoplasm of enucleated human MII oocytes to implant donor MII spindles23 (known as spindle transfer) or PB1 genomes24 (PB1 transfer) to induce de novo spindle formation. Moreover, such reconstructed human MII oocytes (1n2c) can readily be fertilized with sperm followed by exit from the MII arrest, segregation of sister chromatids and formation of normal diploid zygotes containing 1n1c sperm and 1n1c oocyte chromosomes23,24,25,26. Applications of the spindle transfer, generally known as cytoplasmic/mitochondrial replacement therapy or MRT, has been already tested in clinical IVF for treatment of female infertility and resulted in birth of healthy children27. Here, we explored the metaphase machinery of enucleated MII oocytes to induce reductive cell division and chromosome count reduction in the transplanted somatic genome. As indicated above, transition through different phases of the cell cycle is closely coordinated with DNA replication, chromosome duplication and distribution of chromosomes to daughter cells. This coordination is controlled by a system of cell cycle checkpoints including in G2 that prevents entry into the M-phase if DNA replication is incomplete and chromosomes are not duplicated28. Direct transfer of somatic cell nuclei arrested at the G0/G1 phase into the MII-arrested oocyte cytoplasm bypasses this checkpoint and induces formation of MII-like bipolar spindles in SCNT oocytes. However, human SCNT oocytes did not respond to the conventional activation stimulus induced by fertilization with sperm, failing to form zygotes. While the precise nature of this persistent metaphase arrest requires further investigation, we investigated artificial activation as supplement for fertilization with sperm. We previously showed that fertilization of mouse SCNT oocytes provides sufficient activation signaling for release from the metaphase arrest and formation of SCNT zygotes. This suggests that interspecies differences in SCNT spindles exist between mouse and human oocytes. Therefore, we investigated supplemental activation stimuli to bypass the MII arrest in human SCNT oocytes. We demonstrate that activation by electroporation followed by incubation with cyclin-dependent kinase inhibitor roscovitine alone or in combination with sperm fertilization is required to circumvent the MII arrest and induce reductive cell division in human SCNT oocytes. Based on our recent mouse study, the sequence differences between homologs from the divergent mouse strains correlated with random segregation while proper segregation of homologs was only observed in mouse SCNT oocytes generated from inbred somatic cells12. Fertilized human SCNT embryos were diploid (one somatic and one sperm chromosome) for approximately half the homologs while other chromosome pairs were either triploid (two somatic and one sperm) or monosomic (sperm only). Consequently, substantial further experimental modifications of the mitomeiosis will be required to achieve reductive cell division outcomes similar to the meiosis I. As mentioned above, crossover recombination and haploidization are preceded by homologous chromosomes pairing and alignment during the meiotic prophase I21. However, mechanisms controlling this critical and fundamental stage of meiosis are poorly understood29. It is believed that in most species including mammals, mechanisms responsible for crossover recombination also promote homolog pairing by delivering the machinery required for reading and detecting DNA sequence homology29,30. Meiotic recombination, in turn, is dependent on programmed DNA double-strand breaks (DSBs) induced across all chromosomes at multiple loci and catalyzed by the Spo11 protein31. Meiotic DSBs recruit high-fidelity DNA repair machinery by homologous recombination including the ubiquitously expressed RAD51 DNA recombinase and the meiosis-specific DMC1 DNA recombinase which search for a homologous repair template34. The subsequent repair of DSBs results in either crossover or no non-crossover (gene conversion) recombination between homologous chromosomes30. It is likely that many of these critical components of meiosis I are absent in the MII oocyte cytoplasm thus precluding somatic homolog pairing and accurate segregation in human SCNT oocytes. Additionally, the structural complexity hypothesis suggests that the condensed, differentiated state of somatic chromatin may prevent homolog recognition mechanisms that operate during natural meiosis. As somatic genome carries complex epigenetic modifications and higher-order chromatin organization, this could impede the molecular machinery required for homolog pairing and synapsis. Moreover, meiosis naturally operates with duplicated homologs each consisting of two sister chromatids, while SCNT spindles consist of non-duplicated chromosomes. Translation of the SCNT technology for IVG applications will require addressing these and other important characteristics of gamete biology and development of novel methods for segregating somatic cell chromosomes. Despite incomplete haploidy, somatic chromosomes retained within human SCNT zygotes enter the S-phase, duplicate and resume typical embryonic mitotic cycles. Remarkably, a semi-haploid somatic chromosome set is accurately replicated and segregated to daughter blastomeres in SCNT embryos evading further cell cycle checkpoint control mechanisms. Moreover, the somatic genome can functionally integrate with sperm chromosomes after fertilization of human SCNT oocytes and form uniform embryos consisting of somatic and sperm chromosomes. An important limitation of our current study is the inability to distinguish between developmental arrest caused by chromosomal abnormalities versus incomplete epigenetic reprogramming of somatic chromosomes. Given that all analyzed embryos exhibited chromosomal imbalances, separating aneuploidy effects from reprogramming defects remains challenging. The modest 8.8% blastocyst development rate likely reflects contribution from both factors. The OHSU Institutional Review Board (IRB) reviewed, approved and provided regular oversight of this research study. In addition, a Data Safety Monitoring Committee (DSMC) was formed that provided bi-annually review of all gamete and tissue donations, use of samples, and any adverse events in study participants. During this study, 270 mature MII oocytes were utilized to generate 155 research embryos for developmental analysis and genetic characterization. No embryos were cultured beyond day 6 post-fertilization (blastocyst stage), which represents the standard timepoint for clinical embryo transfer and preimplantation genetic testing. These guidelines also regulate procurement, fertilization and genetic manipulation of human gametes, embryos and embryonic stem cells for research. Study subjects were provided a protocol specific consent form which included a lay language summary of the research design, approved by the OHSU IRB, prior to obtaining informed consent. The study volunteers were informed of risks associated with participation in this study, including both medical and loss of confidentiality risks. Study subjects were also provided the option to consent to be notified of any clinically relevant findings related to their genetic information, as well as whether they would allow genetic material from their donation to be shared with outside researchers. In accordance with Oregon regulations and ethical guidelines, research oocyte donors received compensation ranging from $7000 to $8000 per donation cycle, reflecting the time, effort, and potential discomfort involved in the donation process. This compensation range reflects changes over the 3–4 year study period and is consistent with amounts offered for egg donations in fertility treatments in Portland, Oregon. Ovarian stimulation protocols and procedures were as described previously25. Ovarian stimulation and oocyte retrievals were managed by OHSU fertility physicians and recovered gametes were transferred to the research laboratory. All study-related procedures were conducted at the OHSU Center for Embryonic Cell and Gene Therapy. Cumulus-oocyte complexes (COCs) were treated with hyaluronidase to disaggregate and remove cumulus cells. Mature metaphase II (MII) oocytes were placed in Global Medium (Life Global) supplemented with 10% serum substitute supplement (Global 10% medium) at 37 °C in 6% CO2 and covered with tissue culture oil (Sage IVF, Cooper Surgical). Skin biopsy and blood draws were performed at OHSU fertility clinic by the study physicians. Skin punches of 1–3 mm were placed into sterile saline solution and processed to establish primary culture of fibroblasts as described earlier35. Oocyte enucleations and fusion with fibroblasts were performed as described previously8. Briefly, mature MII oocytes were place into a 30μl manipulation droplet of HTF w/HEPES (Life Global) supplemented with 10% serum substitute supplement (HTF w/HEPES 10%), containing 5 μg/ml cytochalasin B and 1.25 mM caffeine in a glass-bottom dish. Droplet was covered with tissue culture oil (Sage IVF, Cooper Surgical) and oocytes were incubated at 37 °C for 5 min before spindle removals. Dish was mounted on the stage of an inverted microscope (Olympus IX71) equipped with a stage warmer (Tokai Hit), Narishige micromanipulators, Oosight spindle imaging system and a laser objective (HamiltonThorne). An oocyte was held by a holding pipette and positioned so that the spindle was situated close to the 3 o'clock position. A spindle with small amount of cytoplasm surrounded by a membrane was aspirated into the pipette and discarded. Fused SCNT oocytes were incubated for two hours to allow spindle formation. SCNT and MII oocytes were placed into a micromanipulation dish containing 30 µL droplet of HTF w/HEPES 10% covered with tissue culture oil and mounted on the stage of an inverted microscope (Olympus IX71) equipped with a stage warmer and Narishige micromanipulators. SCNT and MII oocytes were initially subjected to Ca2+ treatment by electroporation (Electro Square Porator T-820, BTX) in 0.25 M D-sorbitol buffer containing 0.1 mM calcium acetate, 0.5 mM magnesium acetate, 0.5 mM HEPES, and 1 mg/ml fatty-acid-free BSA as described before8. Briefly, SCNT oocytes are place between the electrodes of the BTX apparatus in D-Sorbitol buffer and subjected to two pulses (HV 3.0 kV, Pulse length 50 and 0.13 charging voltage). Activated SCNT oocytes were rinsed with HTF w/HEPES 10% and cultured in Global medium supplemented with 10% FBS, 12 mM b-mercaptoethanol (BME), and 10 nM Trichostatin A (TSA, Sigma) at 37 °C in 6% CO2, 5% O2, and 89% N2 for 15 h in an embryoscope time-lapse incubator (Vitrolife). Zona pellucidae of SCNT and control MII oocytes was removed using a laser and oocytes were fixed in 4% paraformaldehyde for 20 min at room temperature, followed by rinsing in a blocking solution (0.3% BSA in PBS) three times for 15 min each. Fixed oocytes were permeabilized with 0.1% Triton X-100 for 20 min at room temperature and rinsed in blocking solution for 20 min at room temperature to reduce non-specific binding during subsequent antibody staining. Oocytes were incubated with antibodies for centromeres (human anti-centromere and anti-kinetochore, 1:30, Antibodies incorporated, cat#15-234) and microtubules (α-tubulin rabbit mAb AlexaFluor 488, 1:100, Cell Signaling Technology, cat#5063) washed in blocking solution and then placed on glass slides into a micro drop of mounting medium containing DAPI. A coverslip was carefully placed on top of the drop and images were analyzed with super-resolution epifluorescent microscope Olympus IX83 with inbuilt cellSens Dimension software. Pictures acquired with Olympus cellSens Dimension software were used to produce the rotating 3D videos using ImageJ Fiji software. Three to four hours before filming, culture dishes were prepared with pre-equilibrated medium at 37 °C in 6% CO2, 5% O2, and 89% N2 to ensure optimal medium conditions and pH. Embryos were placed individually into each well, and dishes were loaded into the EmbryoScope time-lapse incubator (Vitrolife, Sweden). Embryo development was continuously monitored using integrated EmbryoViewer software v.5.4. The system captures high-resolution images of each embryo every 15-min intervals in various focal planes, allowing non-invasive, non-disturbed observation throughout the preimplantation period. Zona pellucidae from zygotes and cleaved embryos were removed by brief exposure to acidic Tyrode solution (NaCl 8 mg/ml, KCl 0.2 mg/ml, CaCl2.2H2O 2.4 mg/ml, MgCl2.6 H2O 0.1 mg/ml, glucose 1 mg/ml, PVP 0.04 mg/ml). Zygotes were placed in a droplet of HTF w/HEPES 10% containing 5 μg/ml cytochalasin B and 10 μg/ml nocodazole. Cleaved embryos were manually disaggregated into single blastomeres with a small-bore pipette. Whole-genome amplification was performed employing multiple displacement amplification using the REPLI-g Single Cell Kit (Qiagen), following the manufacturer's manual. Whole genome sequencing and AmpliSeq design were conducted to identify genomic loci capable of distinguishing between paternal and maternal homologous chromosomes of fibroblasts as nuclear donors for SCNT and from the sperm chromosomes. DNA extracted from nuclear donor fibroblasts and blood samples from both proband's parents and the sperm donor, was normalized to 500 ng in 25 μl of resuspension buffer RSB. Paired-end sequencing was performed at 2 × 150 bp with an average autosome coverage depth of 27X using the Illumina NovaSeq 6000 platform. This involved several steps: generating unaligned BAM files using FastqToSam (Picard tools v2.26.9), marking adapters with MarkIlluminaAdapters, converting unaligned BAM to fastq using SamToFastq, mapping to the GRCh38 genome assembly with BWA-MEM (v0.7.17), and merging BAM and unaligned BAM with MergeBamAlignment. The aligned reads were then coordinate-sorted using SortSamSpark (GATK v4.2.6.1), and duplicate reads were marked with MarkDuplicatesSpark (GATK v4.2.6.1). Additional filters (QUAL  >  1 & QUAL/AO  >  10 & SAF  >  0 & SAR  >  0 & RPR  >  1 & RPL  >  1) were applied to remove low-confidence calls. The resulting variant sets were used for downstream analysis and identification of unique triallelic loci in R (v4.0.3). The genomic positions of these loci were used to design a custom targeted sequencing (AmpliSeq) panel. This panel targeted 651 genomic regions across all chromosomes. Among these, 583 targeted genomic regions contained either single nucleotide variants differentiating maternal, paternal, and sperm chromosomes, or a defined set of adjacent biallelic variants that, when phased, determined the haplotype of maternal, paternal, or sperm origin. Due to the low number of informative triallelic haplotypes, chromosomes 19, 21, and 22 were supplemented with 47 biallelic variants differentiating maternal and paternal homologs only. The average amplicon length was 260 bp, and the total length of all targeted regions was approximately 39 Kb. To analyze the chromosome content of collected PNs, PBs and blastomeres, whole genome amplified DNA concentrations were normalized to 10 ng/μl, and 50 ng input was utilized for library preparation with the AmpliSeq panel and the AmpliSeq Library PLUS kit (Illumina), following the manufacturer's protocols. Paired-end sequencing was conducted on either the Illumina MiSeq platform as 2 × 200 bp or on the Nextseq1000 as 2  × 150 bp to achieve a median coverage depth of 95X. Preprocessing steps were applied to the raw sequencing reads using GATK to generate BAM files suitable for further analysis. These steps included generating unaligned BAM files via FastqToSam (Picard tools v2.26.9), marking adapters using MarkIlluminaAdapters, converting unaligned BAM to fastq via SamToFastq, mapping to the GRCh38 genome assembly with BWA-MEM (v0.7.17), and merging BAM and unaligned BAM with MergeBamAlignment. Genomic variants were called using FreeBayes (v1.3.5), followed by normalization with bcftools (v1.14). The resulting variant calling data were then employed for downstream analysis of chromosome segregation in R (v4.3.0). Genomic loci with variant coverage depth of less than 22× were considered not amplified and were therefore excluded from the inference of chromosome origin. Statistical analyses were performed using GraphPad Prism. Chi-square test was used for the comparison between maternal and paternal segregation. Monte Carlo Simulation was use for random segregation. Unpaired t test was used for comparison of mean length and width of the novo spindles, PB2 extrusion, PN formation, cleavage and blastocyst development, which were presented as mean ± SEM. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data supporting the findings of this study are included in this published article and its supplementary information files. Due to OHSU IRB retsrictions and Oregon laws as well as terms of the consents signed by research participants, we are unable to publicly upload and share whole-genome and amplicon sequencing data outside of the OHSU network as these data sets may be used to reveal the genetic identity of the study participants. Nevertheless, we are willing to share this information on a case-by-case basis, onsite at OHSU if approval is granted by OHSU Research Integrity and the IRB (Kara Drolet, Associate VP, ORIO irb@ohsu.edu). These measures are intended to ensure the confidentiality of our research participants while also striving for research transparency and reproducibility. Upon successful approvals the requestor will be escorted by a team member at all times and granted access to an OHSU computer in a shared office where they can access and review the data sets. All requests should be initiated with OHSU Research Integrity and will move through their process, which may take upwards of 3–6 months to gain full access to all genome sequencing data that was generated in this work but not shared publicly. Source data are provided with this paper. & Campbell, K. H. Viable offspring derived from fetal and adult mammalian cells. & Wilmut, I. Sheep cloned by nuclear transfer from a cultured cell line. Chen, M. et al. Chromatin architecture reorganization in murine somatic cell nuclear transfer embryos. Rapid H1 linker histone transitions following fertilization or somatic cell nuclear transfer: evidence for a uniform developmental program in mice. Szollosi, D., Czolowska, R., Szollosi, M. S. & Tarkowski, A. K. Remodeling of mouse thymocyte nuclei depends on the time of their transfer into activated, homologous oocytes. Tachibana, M. et al. Human embryonic stem cells derived by somatic cell nuclear transfer. Concise Review: Embryonic Stem Cells Derived by Somatic Cell Nuclear Transfer: A Horse in the Race?. Mitalipov, S. M., Nusser, K. D. & Wolf, D. P. Parthenogenetic activation of rhesus monkey oocytes and reconstructed embryos. & Sacristan, M. P. Working on genomic stability: From the S-phase to mitosis. Corellou, F. et al. A S/M DNA replication checkpoint prevents nuclear and cytoplasmic events of cell division including centrosomal axis alignment and inhibits activation of cyclin-dependent kinase-like proteins in fucoid zygotes. The spindle checkpoint and chromosome segregation in meiosis. The metaphase II arrest in mouse oocytes is controlled through microtubule-dependent destruction of cyclin B in the presence of CSF. Nurse, P. Universal control mechanism regulating onset of M-phase. Madgwick, S. & Jones, K. T. How eggs arrest at metaphase II: MPF stabilisation plus APC/C inhibition equals Cytostatic Factor. Zickler, D. & Kleckner, N. Recombination, Pairing, and Synapsis of Homologs during Meiosis. Cold Spring Harbor perspectives in biology 7 https://doi.org/10.1101/cshperspect.a016626 (2015). Schvarzstein, M., Wignall, S. M. & Villeneuve, A. M. Coordinating cohesion, co-orientation, and congression during meiosis: Lessons from holocentric chromosomes. Bennett, M. D. The time and duration of meiosis. Tachibana, M. et al. Mitochondrial gene replacement in primate offspring and embryonic stem cells. Ma, H. et al. Functional human oocytes generated by transfer of polar body genomes. Towards germline gene therapy of inherited mitochondrial diseases. Costa-Borges, N. et al. First pilot study of maternal spindle transfer for the treatment of repeated in vitro fertilization failures in couples with idiopathic infertility. & Holzen, T. M. Cell cycle regulation of DNA replication. & Mohibullah, N. Self-organization of meiotic recombination initiation: general principles and molecular pathways. Keeney, S., Giroux, C. N. & Kleckner, N. Meiosis-specific DNA double-strand breaks are catalyzed by Spo11, a member of a widely conserved protein family. Grey, C., Baudat, F. & de Massy, B. PRDM9, a driver of the genetic map. Lange, J. et al. ATM controls meiotic double-strand-break formation. & Keeney, S. Clarifying the mechanics of DNA strand exchange in meiotic recombination. Kang, E. et al. Age-Related Accumulation of Somatic Mitochondrial DNA Mutations in Adult-Derived Human iPSCs. We would like to thank the OHSU Institutional Review Board (IRB) and members of the Data Safety Monitoring Committee (DSMC) for oversight and guidance of this project. We are grateful to the study participants for gamete and tissue donations and staff of the Women's Health Research Unit, IVF laboratory, University Fertility Consultants, the Reproductive Endocrinology and Infertility Division in the Department of Obstetrics and Gynecology, OHSU and Spring Fertility for their assistance in procurement of human gametes. We would like to thank Evident Scientific for providing their fluorescence microscope. We are grateful to Rebecca Tippner-Hedges, former member of our laboratory, for her dedicated efforts and assistance during early stages of this research project. The studies were supported by grants from Open Philanthropy, Haploid Gamete Research Foundation, Longevity Impetus and OHSU institutional funds. Present address: Department of Obstetrics and Gynecology, the First Affiliated Hospital of Anhui Medical University, Hefei, Anhui, China Center for Embryonic Cell and Gene Therapy, Oregon Health & Science University, Portland, OR, 97239, USA Nuria Marti Gutierrez, Aleksei Mikhalchenko, Maria Shishimorova, Daniel Frana, Crystal Van Dyken, Ying Li, Hong Ma, Amy Koski, Dan Liang, Sang-Goo Lee, Daniel Eyberg, Zahra Safaei, Paula Amato & Shoukhrat Mitalipov Division of Reproductive Endocrinology and Infertility, Department of Obstetrics and Gynecology, Oregon Health & Science University, Portland, OR, 97239, USA Thomas O'Leary, David Lee, Sacha Krieg, Diana Wu, Elizabeth Rubin & Paula Amato coordinated recruitment and consenting of gamete donors. performed SCNT, fertilization, activation, embryo culture, blastomere and ESC isolations. performed DNA extractions, WGS and library preparation. acquired research funding and supervised this project. Correspondence to Paula Amato or Shoukhrat Mitalipov. Nature Communications thanks Pasqualino Loi, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Induction of experimental cell division to generate cells with reduced chromosome ploidy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a68133877/human-minds-slow-time-reality/'>You Can Trigger 'Time Expansion'—Meaning You Can Stretch Seconds and Warp Reality, Scientists Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 17:09:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ever felt the world slow down around you? Some cosmologists even wonder if entire pockets of the universe—whether filled with galaxies or endless nothing—experience time at different rates. And it's this mental realm that Steve Taylor—a psychologist at Leeds Beckett University—is exploring in a new paper and a new book called Time Expansion Experiences. “Everything went into slow motion,” Taylor describes in a post on the Leeds Beckett University website. “I looked behind, and the other cars seemed to be moving incredibly slowly, almost as if they were frozen. In the following decade, Taylor began investigating how and why these moments of ultra-slowness manifest, and his research has led him to some pretty interesting conclusions. It's possible that this perception is an evolutionary hack for surviving intense situations, but that still doesn't explain how they occur outside of life-or-death events. “[Another] theory is that TEEs aren't real experiences, but illusions of recollection. In emergency situations, so this theory goes, our awareness becomes acute, so that we take in more perceptions than normal,” Taylor wrote. However, Taylor's findings from people who've experienced TEEs indicate that they were able to process thoughts and information much faster than what would be possible under normal circumstances. Instead, Taylor advocates for an idea that these events shift the human mind into an altered state of consciousness. In these moments, we step outside our normal consciousness into what Taylor calls a different “time-world.” In sport, intense altered states occur due to what I call ‘super-absorption,'” Taylor wrote. While the flow of time remains a delicate dance between gravity and space-time, research shows that time can also very much be a state of mind. Fire Patrol Finds Ancient Face Carving in Boulder Experts Find Missing Piece of Ramesses II Statue Scientists Say Daylight Savings Time is Hurting Us These Edge State Atoms Showcase a Quantum Mystery</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02071-5'>Facial expressions in mice reveal latent cognitive variables and their neural correlates</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 16:58:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Brain activity controls adaptive behavior but also drives unintentional incidental movements. Such movements could potentially be used to read out internal cognitive variables that are also neurally computed. Establishing this would require ruling out that incidental movements reflect cognition merely because they are coupled with task-related responses through the biomechanics of the body. Here we addressed this issue in a foraging task for mice, where multiple decision variables are simultaneously encoded even if, at any given time, only one of them is used. We found that characteristic features of the face simultaneously encode not only the currently used decision variables but also independent and unexpressed ones, and we show that these features partially originate from neural activity in the secondary motor cortex. Our results suggest that facial movements reflect ongoing computations above and beyond those related to task demands and demonstrate the ability of noninvasive monitoring to expose otherwise latent cognitive states. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout The behavioral and electrophysiological data used in this study are available on Figshare at https://figshare.com/s/924af1de619f4597f37a (ref. Raw videos and electrophysiological data are too large to be shared on a public repository and are therefore available from the authors upon request. All analyses were performed using custom code written in MATLAB that is available upon request. The code used to process the videos is publicly available at https://github.com/MouseLand/facemap. The code used for the central GLM analyses is publicly available at https://hastie.su.domains/glmnet_matlab/. The code developed for the LM-HMM can be accessed at https://github.com/mazzulab/ssm/blob/master/notebooks/2c%20Input-driven%20linear%20model%20(LM-HMM).ipynb. Cowen, A. S. et al. Sixteen facial expressions occur in similar contexts worldwide. New tests, new tools: mobile and connected technologies in advancing psychiatric diagnosis. Euston, D. R. & McNaughton, B. L. Apparent encoding of sequential context in rat medial prefrontal cortex is accounted for by behavioral variability. Cowen, S. L. & McNaughton, B. L. Selective delay activity in the medial prefrontal cortex of the rat: contribution of sensorimotor information and contingency. Selen, L. P. J., Shadlen, M. N. & Wolpert, D. M. Deliberation in the motor system: reflex gains track evolving evidence leading to a decision. Flavell, S. W., Gogolla, N., Lovett-Barron, M. & Zelikowsky, M. The emergence and influence of internal states. Barrett, L. F., Adolphs, R., Marsella, S., Martinez, A. & Pollak, S. D. Emotional expressions reconsidered: challenges to inferring emotion from human facial movements. LeDoux, J. E. What emotions might be like in other animals. Dolensek, N., Gehrlach, D. A., Klein, A. S. & Gogolla, N. Facial expressions of emotion states and their neuronal correlates in mice. Vinck, M., Batista-Brito, R., Knoblich, U. A. Arousal and locomotion make distinct contributions to cortical activity patterns and visual encoding. McGinley, M. J., David, S. V. & McCormick, D. A. Cortical membrane potential signature of optimal states for sensory signal detection. Cazettes, F., Reato, D., Morais, J. P., Renart, A. & Mainen, Z. F. Phasic activation of dorsal raphe serotonergic neurons increases pupil size. Bimbard, C. et al. Behavioral origin of sound-evoked activity in mouse visual cortex. Clayton, K. K. et al. Sound elicits stereotyped facial movements that provide a sensitive index of hearing abilities in mice. Musall, S., Kaufman, M. T., Juavinett, A. L., Gluf, S. & Churchland, A. K. Single-trial neural dynamics are dominated by richly varied movements. Yin, C. et al. Spontaneous movements and their relationship to neural activity fluctuate with latent engagement states. Salkoff, D. B., Zagha, E., McCarthy, E. & McCormick, D. A. Movement and performance explain widespread cortical activity in a visual detection task. Tremblay, S., Testard, C., DiTullio, R. W., Inchauspé, J. & Petrides, M. Neural cognitive signals during spontaneous movements in the macaque. Syeda, A. et al. Facemap: a framework for modeling neural activity based on orofacial tracking. Talluri, B. C. et al. Activity in primate visual cortex is minimally driven by spontaneous movements. Separating cognitive and motor processes in the behaving mouse. Reato, D., Steinfeld, R., Tacão-Monteiro, A. Response outcome gates the effect of spontaneous cortical state fluctuations on perceptual decisions. Hulsey, D., Zumwalt, K., Mazzucato, L., McCormick, D. A. & Jaramillo, S. Decision-making dynamics are predicted by arousal and uninstructed movements. Cazettes, F. et al. A reservoir of foraging decision variables in the mouse brain. Vertechi, P. et al. Inference-based decisions in a hidden state foraging task: differential contributions of prefrontal cortical areas. Stringer, C. et al. Spontaneous behaviors drive multidimensional, brainwide activity. Cowen, A. S. & Keltner, D. Self-report captures 27 distinct categories of emotion bridged by continuous gradients. Langford, D. J. et al. Coding of facial expressions of pain in the laboratory mouse. & Li, N. Behavioral measurements of motor readiness in mice. Lottem, E. et al. Activation of serotonin neurons promotes active persistence in a probabilistic foraging task. Dotan, D., Meyniel, F. & Dehaene, S. On-line confidence monitoring during decision making. Cisek, P. & Kalaska, J. F. Neural mechanisms for interacting with a world full of action choices. Steinmetz, N. A., Zatka-Haas, P., Carandini, M. & Harris, K. D. Distributed coding of choice, action, and engagement across the mouse brain. The importance of accounting for movement when relating neuronal activity to sensory and cognitive processes. Miller, M. R., Herrera, F., Jun, H., Landay, J. & Bailenson, J. N. Personal identifiability of user tracking data during observation of 360° VR video. Asish, S. M., Kulshreshth, A. K. & Borst, C. W. User identification utilizing minimal eye-gaze features in virtual reality applications. Smith, M. & Miller, S. The ethical application of biometric facial recognition technology. Pereira, R. S. in The Legal Challenges of the Fourth Industrial Revolution (eds Moura Vicente, D., de Vasconcelos Casimiro, S. & Chen, C.) 193–209 (Springer International Publishing, 2023). Faraldo Cabana, P. in Artificial Intelligence, Social Harms and Human Rights (eds Završnik, A. & Simončič, K.) 35–54 (Springer International Publishing, 2023). Lopes, G. et al. Bonsai: an event-based framework for processing and controlling data streams. Shamash, P., Carandini, M., Harris, K. & Steinmetz, N. A tool for analyzing electrode tracks from slice histology. Preprint at bioRxiv https://doi.org/10.1101/447995 (2018). Cazettes, F. Facial expressions in mice and neural correlates. We thank J. P. Morais for support with behavioral training. This work was funded by CNRS (F.C. : HR23-00799), the European Research Council Advanced Grant (Z.F.M. This work was also supported by Portuguese national funds through Fundação para a Ciência e a Tecnologia in the context of the project UIDB/04443/2020 and by the research infrastructure CONGENTO, cofinanced by Lisboa Regional Operational Programme (Lisboa2020), under the PORTUGAL 2020 Partnership Agreement, through the European Regional Development Fund and Fundação para a Ciência e a Tecnologia (Portugal) under the projects LISBOA-01-0145-FEDER-02217 and LISBOA-01-0145-FEDER-022122. These authors contributed equally: Fanny Cazettes, Davide Reato. These authors jointly supervised this work: Alfonso Renart, Zachary F. Mainen. Fanny Cazettes, Davide Reato, Elisabete Augusto, Raphael Steinfeld, Alfonso Renart & Zachary F. Mainen Institut de Neurosciences de la Timone, CNRS and Aix Marseille Université, Marseille, France Departement BEL, Centre CMP, Mines Saint-Etienne, Gardanne, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed behavioral and optogenetics experiments. performed electrophysiological experiments and curated the data. designed and performed the analyses. All authors reviewed the paper. The authors declare no competing interests. Nature Neuroscience thanks Nadine Gogolla, Jeffrey Markowitz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Decision variables (for example, consecutive failures) evolve after each lick outcome. Therefore, to predict these variables from movement PCs, we aligned the PCs to lick events using a 200 ms window. b, Decoding accuracy of various decision variables from movement PCs was estimated using a 200 ms sliding window (75% overlap) at different time lags relative to the lick event (example session shown). c, Peak decoding accuracy (grey dots) for each decision variable occurred between 100 ms and 350 ms after the lick. Error bars represent the median and m.a.d. d, This schematic describes the method used to partial out the linear relationship between latent variables. This approach allowed us to decompose DV1 (for example, consecutive failures) into the sum of two time series: one proportional to DV0 (for example, action outcomes) and another orthogonal (uncorrelated) to DV0, which we denote as unique DV1 (for example, unique consecutive failures). The unique consecutive failures residual (pink) is orthogonal to the action outcomes (black). Subsequently, the same procedure generated the unique negative value residual, orthogonal to both action outcomes and unique consecutive failures. The three resulting orthogonal time series (action outcomes, unique consecutive failures, and unique negative value) were then fit using movement PCs. a, Changes in lick rate across 4 example bouts. b, Correlation (Pearson's coefficient) between lick rate and the two decision variables (median across 10 sessions with 25th and 75th percentiles; whiskers represent minimum and maximum values). A small negative correlation exists between lick rate and the decision variables, suggesting that mice tend to lick slightly faster during reward consumption and slow down towards the end of a lick bout. c, Decoding accuracy (cross-validated R²) for action outcome, unique aspects of the two decision variables, and arbitrary signals in each region of interest (median across 10 sessions with 25th and 75th percentiles; whiskers represent minimum and maximum values). The movement PCs are used as predictors in multivariate regression models to predict the action outcome and the decision variables with lick rate variance removed (partialed out). The variance in action outcome, consecutive failures and negative value that is not explained by lick rate remains highly decodable. This suggests that the relationship between facial expressions and latent variables is not solely explained by licking behavior. d, Decoding of bout number from facial movement PCs in an example session. The actual bout number is shown as a thick light orange line, and the decoded projection is shown as a thin dark orange line. e, Decoding accuracy (cross-validated R²) across all sessions (N = 10, orange dots). Boxplots show the median (center line), 25th and 75th percentiles (box edges), and minimum and maximum values (whiskers). a, Mice (N = 5 for a total of 20 behavioral sessions) were presented with single tones (150 ms) of varying frequencies (low: 9.9, 12, and 13 kHz; high: 15, 16.3, and 20 kHz) and, after a delay period (500 ms after stimulus offset), reported their perceived frequency (high or low) by licking one of two water spouts (left or right) to receive a water reward if the response was correct. Simultaneously videos were recorded at 60 fps. b, Task schematics and time intervals analyzed. Video analysis focused on three periods, color-coded in the schematic: “pre-stimulus” (before tone onset; 1 s), “pre-response” (including stimulus presentation and part of the delay period; 500 ms), and “response” (around the time of the lick response; 200 ms). c, Representative frame from the co-registered video (generated by combining the 20 sessions), along with the first four eigenfaces. Error bars represent the median and m.a.d. d, Average motion energy across task periods. Motion energy (relative to the video average) is shown for each analysis period. Low motion is observed during the pre-stimulus period (top), increasing during stimulus presentation (middle), and peaking during the response (bottom), reflecting movement associated with licking. e, Decoding accuracy of task variables from facial movement PCs. Decoding accuracy from facial movement PCs for previous trial outcome (rewarded/unrewarded) and trial number. Each dot represents the average decoding accuracy across sessions for a single mouse; error bars indicate the median and m.a.d. The task variables, especially the slow latent variable ‘Trial number', can be decoded with relatively high accuracy from facial movement PCs. f, Weighted masks (that is, facial representation of the decoded task variables in panel e) for the three different time intervals. The expression of the task variables on the face is highly consistent across the different time intervals, suggesting that this representation is independent of the animal's overall movement. a, Eight facial landmarks (red) were manually identified on average frames from each video. An affine transformation determined by using these landmarks and MATLAB's fitgeotrans function, co-registered frames from each video to a reference frame (orange, bottom). This transformation facilitated comparisons and averaging of weighted masks (Fig. 3), and definition of an average facial silhouette (Fig. b, Improvement in pairwise 2D cross-correlation between average video frames before and after co-registration (N = 10 videos; error bars represent the median and m.a.d. c, Example traces of the first five principal components (PCs) derived from motion energy analysis of three co-registered videos. Singular value decomposition (SVD) was applied to the merged video data. d, t-SNE visualization of the first 100 PCs of the merged video. Points are color-coded by video identity, revealing substantial overlap between videos. a, Weighted masks for a single example session (top) of each wild-type mouse (N = 3) and averaged across all sessions (N = 24, bottom) from all mice during the laser OFF condition. b, Same as in panel (a) but for a single example session (top) of each VGAT (N = 5) and averaged across all sessions (N = 24, bottom) from all mice during the laser OFF condition. c, Inter-animal similarity of facial expression of decision variable (Action outcome: OUT; Unique consecutive failure: CF; Unique negative value: VAL; Arbitrary signal: ARB). Colors represent the normalized 2D cross-correlation at zero lag between the mean weighted masks of two mice (for each mouse the mean weighted mask was the average across sessions, N = 8 mice, 6 ± 2.7 sessions per mouse). d, Mean weighted mask similarity for each mouse and decision variable. Each gray dot represents the average pairwise correlation of the mean weighted mask of a mouse for a given decision variable with the mean weighted masks of all the other mice for the same decision variable. Each gray diamond represents the average pairwise correlation of the mean weighted mask of a mouse for a given decision variable with the mean weighted masks of the same mouse for all the other decision variables. Color errorbars represent median and m.a.d across mice (N = 10, 8 sessions from distinct animals, two from the same animal). e, Distribution of average pairwise 2D correlations at zero lag for electrophysiology (data in Figs. 2–4) and downsampled optogenetics (data in Fig. Pairwise correlations of facial expression of decision variables were calculated for all sessions in the electrophysiology dataset (black dots; median and median absolute deviation (m.a.d.) For comparison, 10 sessions were randomly sampled from the optogenetics dataset multiple times, and average correlations were calculated for each sample (gray dots). The sampling procedure was repeated twenty times, each represented as a row of gray dots. Note that correlations are overall smaller than the ones estimated using averages across sessions (rather than single ones) as in (d). f, Within- and across-mice correlation of decision variable facial expression. Each point represents the average correlation of a single behavioral session with all other sessions, either from the same mouse (within) or different mice (across). across sessions per mouse (N = 8 mice, 6 ± 3 sessions per mouse). a, We recorded with Neuropixels probes in multiple regions of the frontal cortex. Schematic target location of the neuropixels probe insertion. Vertical insertions were performed within a 1 mm diameter craniotomy centered around +2.5 mm anterior and +1.5 mm lateral from Bregma. b, An example of histology with the electrode track. We painted the probe with a red, fluorescent dye to recover the probe's location post-hoc. c, To decode the instantaneous value of multiple decision variables (pink & blue traces, right), we used regression models taking as predictors the activity of simultaneously recorded neurons in each brain region (black traces, left, example activity from M2). The model predictions (the weighted sums of neural activity, black trace right) overlap with the decision variables. d, Neural vs. facial movement PC decoding latencies for different decision variables (N = 10 sessions). Each session contributes three points (gray, pink, blue), one per decision variable. Points above the identity line indicate later facial movement representation relative to neural representation. e, Predicting facial movements from neural activity in M2, OFC, and OC. GLMs were trained to predict facial movement PCs using a 50 ms non-overlapping sliding window of lagged neural activity from M2, OFC, and OC. Facial movement PCs were derived from concatenated videos to enable cross-session comparisons. f, Relationship between decoding accuracy and the time of peak decoding accuracy for facial movements. Peak times (median of cross-validated R² across sessions, N = 10) are shown for the 25 PCs of facial movement with the largest variance, as a function of decoding accuracy from neurons in M2, OFC, and OC. Each dot represents one facial movement PC. Negative values, particularly in M2 (M2 = −0.05 ± 0.06, p = 0.002; OFC = −0.05 ± 0.10, p = 0.194; OC = 0.05 ± 0.09, p = 0.0194; median ± m.a.d, N = 10 sessions, Wilcoxon signed rank test, Holm-Bonferroni corrected), indicate that neural activity preceding the facial movement is most predictive. a, Laser-induced changes in facial movement patterns. 2D masks show the difference in average facial motion (calculated from movement PCs) between laser ON and OFF conditions, across mice and at different lick numbers (decision time points). b, Variability of laser-induced facial motion changes. The variance of the difference in motion energy between laser ON and OFF conditions (normalized by the variance in laser OFF) is shown as a function of lick number for inactivated (green, N = 5 mice) and control (black, N = 3 mice) groups. Thick lines represent group means. c, Decoding accuracy after removing the effect of lick rate (top & middle): Comparison of laser ON vs. laser OFF conditions. Dots below the unity line indicate that representations of decision variables derived from facial movement PCs were decoded less accurately during laser ON than laser OFF. Difference in decoding accuracy (bottom): Laser ON minus laser OFF (mean across sessions for each mouse in the inactivated and control groups). Individual mice are indicated by color. d, Same as in (c) but for decoding latency. Dots above the unity line indicate that representations of decision variables derived from facial movement PCs were decoded later during laser ON than laser OFF. Partial silencing of M2 reduced the accuracy and increased the latency with which facial movement PCs predicted decision variables, even after controlling for lick rate. This suggests that the latent variable represented in facial movements is not simply a consequence of the relationship between M2 activity and lick rate. A detailed example of the dynamic relationship between facial movements and an evolving decision variable (that is, consecutive failures). It simultaneously displays raw video of a representative mouse (top left), calculated facial motion energy (top middle), and example weighted eigenfaces by their related movement PCs changing frame-by-frame (top right). Focusing on two example behavioral bouts, the video synchronizes these facial dynamics with the complete sequence of action outcomes (rewards/failures; green and black dots, respectively), the actual evolving decision variable (pink trace), and its prediction derived from facial movement PCs (gray trace). This video offers a visual illustration of the temporal relation between these elements. Comparison of video processing stages for three example mice. Top row: raw videos display variations in the original field of view. Middle row: corresponding facial ROIs considered for analysis, aligned to a standard orientation. Bottom row: videos after co-registration. Note that co-registration primarily rescales facial dimensions to correct for differing initial viewing angles (for example, Mouse 1). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Cazettes, F., Reato, D., Augusto, E. et al. Facial expressions in mice reveal latent cognitive variables and their neural correlates. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02342-8'>Limited overlap between genetic effects on disease susceptibility and disease survival</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 15:58:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Understanding disease progression is of high biological and clinical interest. Unlike disease susceptibility, whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering nine common diseases (ncases ranging from 11,980 to 124,682) across seven biobanks, we systematically compared genetic architectures of susceptibility and progression, defined as disease-specific mortality. We identified only one locus substantially associated with disease-specific mortality and showed that, at a similar sample size, more genome-wide significant loci can be identified in a genome-wide association study of disease susceptibility. Variants substantially affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictors of disease-specific mortality, while a PGS for general lifespan was substantially associated with disease-specific mortality for seven of nine diseases. We explored alternative definitions of disease progression and found that genetic signals for macrovascular complications in type 2 diabetes overlap with similar phenotypes in the general population; however, these effects are attenuated. Overall, our findings indicate limited similarity in genetic effects between disease susceptibility and disease-specific mortality, suggesting that larger sample sizes, different measures of progression, or the integration of related phenotypes from the general population is needed to identify the genetic underpinnings of disease progression. Genome-wide association studies (GWASs) have been successful in uncovering the genetic basis of human diseases by using a relatively straightforward study design that compares individuals with the disease to controls1,2,3. This approach is well-suited to identify loci associated with disease susceptibility, but it remains unclear whether these results can also inform on the biology of disease progression. Studying the genetic basis of disease progression is relevant for at least two reasons. First, biological insights from the study of disease progression can be more relevant for drug target discovery since many medicines are developed to cure a disease rather than prevent its occurrence. Second, most individuals approach the healthcare system once they develop a disease or its symptoms, and predicting disease progression is, in most diseases, an important clinical challenge. In the past years, several GWASs of disease progression have been performed (see Supplementary Table 1 for a detailed review), but the number of progression-specific loci discovered has been limited. In cancer, GWASs have primarily focused on disease survival and have generally been unsuccessful in identifying genome-wide significant signals. For example, a GWAS of breast cancer survival in over 96,000 patients did not identify any robust association4 and failed to replicate two loci found in the previous largest GWAS of breast cancer survival5. Among neurological conditions, GWASs have focused on disease survival as well as cognitive or motor decline. In one of the largest studies, researchers identified three new loci associated with the progression of Parkinson's disease6. A recent study on multiple sclerosis progression identified a locus indicating involvement of the central nervous system in disease outcome, as opposed to the enrichment of immunological-related signals observed for disease susceptibility7. However, it is worth noting that older studies of multiple sclerosis outcomes have failed to replicate in larger ones8,9. In cardiovascular diseases, studies have focused on disease recurrence, and initial results from the Genetics of Subsequent Coronary Heart Disease (GENIUS-CHD) consortium showed the strongest GWAS signal for coronary artery disease was not associated with subsequent events10. Apart from single-variant-level effects, some studies examined the aggregate effect of many genetic variants. Most of them suggested that polygenic scores (PGSs) for disease susceptibility do not transfer well to disease progression11,12,13, although they might outperform other disease-specific biomarkers in the case of cardiovascular diseases14. Some authors have highlighted the challenges in interpreting results from genetic studies of disease progression due to the bias induced when individuals are selected based on their disease status. If common causes of susceptibility and progression are not accounted for, association results can be unreliable due to what is called an index event bias15, and several approaches to detect and correct for index event bias have been proposed16,17. Large-scale biobanks linked with longitudinal electronic health records have accelerated research into the genetic basis of disease progression and provide sufficient sample size to answer the following two key questions: (1) do genetic predictors that influence disease susceptibility have a similar impact on disease progression? In this study, we aim to provide empirical answers to these two questions by focusing on a specific but commonly used definition of disease progression: disease-specific mortality (Fig. Through an international collaboration across multiple large-scale biobanks, we systematically compared genetic architecture of disease susceptibility and mortality for nine common diseases, focusing on both single variant and aggregated polygenic effects. Using data from seven biobanks, we investigated the genetic similarity between disease susceptibility and disease progression, defined as disease-specific mortality. We selected nine diseases and ran GWASs of disease-specific mortality among disease individuals. We then compared the genetic architecture of disease susceptibility and mortality, focusing on both single variant and aggregated polygenic effects. We considered nine common diseases that substantially increase mortality risk in the general population and have a large public-health impact (Table 1). We confirmed disease association with mortality using nation-wide Finnish data and observed a hazard ratio (HR) for 20-year mortality ranging from 1.31 for type 2 diabetes to 3.61 for chronic kidney disease in females18 (Supplementary Table 2). We identified diseased individuals based on consistent disease definitions captured via electronic health records or registry data across eight longitudinal studies—FinnGen19, UK Biobank20, Estonian Biobank21, Generation Scotland22, Genomics England23, Genes & Health24, Dana–Farber Cancer Institute25 and BioMe. All individuals with a disease were followed up for at least three months. We defined disease-specific mortality based on death certificates in which the disease of interest was listed as the primary or secondary cause of death. One participating biobank did not have information on causes of death, and we used overall death instead (Supplementary Table 2). For each disease, we carried out a GWAS of disease-specific mortality among disease individuals using Cox proportional hazard model as implemented in GATE26 or SPACox27 (Supplementary Figs. On top of all common GWAS covariates, all analyses were also adjusted for age at disease diagnosis for the following two reasons: (1) age is a strong predictor of mortality; (2) age of onset has a nontrivial genetic contribution partially overlapping with disease susceptibility28, and we are instead interested in genetic effects on disease-specific mortality. Of all nine diseases studied, we identified only one locus associated with disease-specific mortality at genome-wide significance (P < 5 × 10−8). The locus (rs7360523) on chromosome 20, close to SULF2, was associated with disease-specific mortality among patients with heart failure. We asked whether well-established signals for disease susceptibility were associated with disease-specific mortality (Fig. In total, 804 lead variants were reported from all susceptibility GWASs. None of them were substantially associated with disease-specific mortality after multiple testing correction (P < 0.05/804 = 6.22 × 10−5), while 392 showed the same effect direction, which is no more than expected by chance (probability of observing same direction of effect direction = 0.49 (95% confidence interval (CI) = 0.45–0.52); binomial test against 0.5, P = 0.5028). Variants were selected either because of genome-wide significance for susceptibility in the largest disease-specific GWAS or because of genome-wide significance for disease-specific mortality in the current study, indicated by color of the dot (red, significant in susceptibility GWAS; blue, significant in mortality GWAS). Data are presented as 95% CI for GWAS effect sizes (β ± 1.96 s.e. See also Supplementary Table 3 for quantitative results and sources of susceptibility GWAS summary statistics. The only disease-specific mortality locus identified for heart failure also did not show comparable effect on heart failure susceptibility (P = 0.87 in susceptibility GWAS with opposite direction of effect). The low number of genome-wide signals for disease-specific mortality was consistent with the lower estimated heritability compared to the GWAS of disease susceptibility (Supplementary Table 2). To assess whether the overall lack of substantial genetic signals for disease-specific mortality was simply due to smaller sample sizes compared to the GWASs of disease susceptibility, we performed a down-sampling experiment in FinnGen and UK Biobank by imposing the same effective sample size for both analyses. To further make the two analyses comparable, the GWASs of disease susceptibility were conducted using survival analysis with age as time scale and disease diagnosis as outcome. The GWASs of disease susceptibility returned 33 genome-wide significant loci across five of the nine tested diseases, while the GWASs of disease-specific mortality returned no genome-wide significant results (Table 2). We investigated the joint effects of genetic variants associated with disease susceptibility in predicting disease-specific mortality. For each disease, we constructed a PGS using results from the largest GWAS of disease susceptibility. All the PGSs were strongly associated with disease susceptibility. in the PGS ranged from 1.17 (1.15–1.18) for stroke to 1.90 (1.88–1.93) for prostate cancer (dashed line in Fig. On the contrary, the same PGSs were weakly or not associated with disease-specific mortality (orange dots in Fig. For example, although strongly associated with disease susceptibility, a PGS for breast cancer showed no association with breast cancer mortality (HR = 0.98 (0.93–1.02)). The most substantial association was observed between the heart failure PGS and heart failure mortality (HR = 1.09 (1.07–1.12)), while the PGSs for chronic kidney disease and prostate cancer trend towards having a protective effect on mortality (HR = 0.96 (0.91–1.02) and HR = 0.96 (0.93–1.00), respectively). PGS associations with both disease susceptibility and disease-specific mortality were examined using a Cox proportional hazards model. Horizontal solid lines represent 95% CI for PGS association with disease-specific mortality HR (exp(β ± 1.96 s.e.)). The vertical dashed lines in black and gray represent association with disease susceptibility HR and 95% CI, respectively, (exp(β ± 1.96 s.e. First, we assessed whether using a less specific definition of disease progression, namely all-cause mortality, would impact the observed results. We observed significantly larger correlation coefficients of susceptibility PGSs on disease-specific mortality than on all-cause mortality in five of nine diseases (Supplementary Fig. Second, we only considered individuals who developed the disease after study enrollment (Supplementary Fig. 11a and Supplementary Table 8) as a way to account for survival bias, which might explain some of the negative associations between PGSs and cause-specific mortality. Nonetheless, results were consistent (correlation coefficient r between effect sizes β in the main analysis and sensitivity analysis = 0.96), and we continued to observe a negative association between a PGS for prostate cancer and prostate cancer mortality. Third, we considered different maximum follow-up lengths (2, 5 and 10 years) because we reasoned that deaths occurring shortly after disease diagnosis were more likely to be caused by the disease. However, results were overall comparable across follow-up lengths (correlation coefficient r between effect sizes in main analysis and sensitivity analysis = 0.67, 0.78 and 0.92 for 2, 5 and 10 years, respectively; Supplementary Fig. Fourth, we evaluated whether adjusting the analyses for age at diagnosis could mask an age-specific effect of PGS on cause-specific mortality, for example, because such effect was only observed among young or old patients. We observed the largest change in z score for PGS effect on disease-specific mortality between lower and upper 50% quantile diagnosed age groups only for Alzheimer's disease (Δz = 3.51; Supplementary Fig. That is, the association between Alzheimer's disease PGS and mortality was substantial only among younger but not older patients. Finally, we tested the effect of using only unrelated individuals in FinnGen and found the result to be robust (Supplementary Fig. We also carried out the same analyses using non-European individuals from Genes & Health. However, due to limited power, no conclusion could be drawn (Supplementary Fig. Forest plots of effects from each participating European biobank are shown in Supplementary Fig. Having established that susceptibility PGSs are weakly associated with disease-specific mortality, we reasoned that other PGSs that are better proxies of disease-specific mortality could show stronger associations. First, we considered PGSs constructed directly from our GWASs of disease-specific mortality. For diseases where power allowed, we derived PGSs using weights from the meta-analyzed GWAS results from all biobanks except for FinnGen and tested the association between PGS and disease-specific mortality within FinnGen. Surprisingly, of all four diseases, only colorectal has its PGS nominally associated with disease-specific mortality, with a rather marginal effect (β = 0.0523, P = 3.58 × 10−2; Supplementary Fig. Second, we considered a PGS for general longevity derived from the largest lifespan GWAS29 under the assumption that it might capture some of the genetic effects related to disease survival. The longevity PGS was substantially associated with disease-specific mortality for seven of nine diseases (P < 0.005, accounting for the number of diseases tested), and for seven diseases, it shows larger HR than a PGS for susceptibility (Fig. 4 and Supplementary Table 5), including the three diseases where we observed protective effects of susceptibility PGSs. Horizontal solid lines represent 95% CI for PGS association with disease-specific mortality HR (exp(β ± 1.96 s.e.)). We also tested another composite mortality PGS developed from the genetics of multiple risk factors30 in FinnGen. The composite mortality PGS was substantially associated with disease-specific mortality for four of nine diseases, and outperformed longevity PGS in predicting type 2 diabetes and coronary artery disease mortality (Supplementary Fig. Disease-specific mortality is widely accessible across biobanks but can be an inaccurate proxy of progression for some diseases. For type 2 diabetes, macrovascular complications (defined as coronary artery disease, stroke or peripheral arterial disease) and microvascular complications (defined as diabetic retinopathy, nephropathy, and neuropathy) are more clinically relevant measures of disease progression that are captured in electronic health records. We ran time-to-event progression GWAS among type 2 diabetes patients for both macro (ncases = 17,332 of 85,188 eligible type 2 diabetes patients) and microvascular (ncases = 5,798 of 103,185 eligible type 2 diabetes patients) complications in FinnGen and UK Biobank, and meta-analyzed the results. We identified one genome-wide significant locus on chromosome 9 for macrovascular complications, but none for microvascular complications (Fig. The locus was not associated with type 2 diabetes susceptibility (P = 0.2898 from UK Biobank and FinnGen meta-analysis), but it was associated with type 2 diabetes mortality (P = 5.6 × 10−4), highlighting how disease-specific mortality can capture, albeit imprecisely, disease progression. We wondered whether this genetic signal was shared with cardiovascular conditions in individuals without type 2 diabetes and performed a GWAS for age of onset for coronary artery disease, stroke, or peripheral arterial disease in the UK Biobank and FinnGen. For this experiment, we performed random down-sampling to match the available sample sizes for the progression GWAS. We observed a strong overlap in the signals from the GWAS of diabetic macrovascular complications and the GWAS of cardiovascular conditions in individuals without type 2 diabetes (Fig. In other words, the locus detected for type 2 diabetes macrovascular complications was also detected by a GWAS of similar phenotypes in individuals without type 2 diabetes. However, the effect was overall reduced among individuals with type 2 diabetes (leading SNP—β = 0.09, P = 3.62 × 10−16 versus β = 0.11, P = 1.36 × 10−23). The top Manhattan plot displays the results of a GWAS carried out in individuals with type 2 diabetes. The bottom Manhattan plots display the results of GWAS carried out in individuals without type 2 diabetes, where proxy phenotypes for cardiovascular conditions were used instead. Two GWASs were matched to the same number of cases and controls to ensure similar power (ncases = 17,332 of 85,188 eligible type 2 diabetes patients/type 2 diabetes-free individuals). As we have previously shown that a longevity PGS created in the general population was predictive of disease-specific mortality, we wondered whether a similar observation would hold for these refined definitions of type 2 diabetes progression. To test this, we considered PGSs for coronary artery disease31 and stroke32 as predictors of type 2 diabetes macrovascular complication, and PGSs for age-related macular degeneration33 and chronic kidney disease34 as predictors of type 2 diabetes macrovascular complication. A PGS for coronary artery disease was a stronger predictor of macrovascular complications among type 2 diabetes patients than the PGS of type 2 diabetes susceptibility (HR = 1.19 (1.17–1.21) versus HR = 1.04 (1.03–1.06)). For microvascular complications, we did observe a nominally significant association for the PGS of age-related macular degeneration (HR = 1.03 (1.01–1.06)), but not chronic kidney disease (HR = 1.00 (0.98–1.03); Supplementary Fig. Noticing an attenuation of variant effects in progression GWASs with the same sample sizes, we suspected that index event bias could have a role. Therefore, we carried out simulation under a simple liability threshold model and explored the impact of index event bias by introducing a shared nongenetic risk factor accounting for various proportions of the liability in disease susceptibility and progression. We compared the simulated effect of causal genetic variants on progression with the observed effect from the progression GWASs and found larger differences when the shared nongenetic component accounted for higher liability variance, indicating higher impact of index event bias (Supplementary Fig. However, in the scenario of a low progression heritability, which is consistent with our empirical findings for disease mortality, index event bias correction showed limited impact as we observed no genetic variants substantially associated with disease progression before or after bias correction (Supplementary Table 11). In this study, we systematically explored the overlap of genetic effects on disease susceptibility and a common measure of disease progression, disease-specific mortality, for nine common diseases. By conducting the largest within-patient GWAS of disease-specific mortality to date, we found that (1) lead variants affecting disease susceptibility do not have comparable effect sizes on disease mortality; rather, they show little effect and no significant association with disease-specific mortality in GWAS; (2) at a similar sample size, GWASs of disease-specific mortality identified fewer genome-wide significant loci than GWASs of disease susceptibility, suggesting that GWASs of disease progression might require larger sample size or more refined phenotypes than GWAS of disease susceptibility; (3) disease susceptibility PGSs do not transfer well on disease-specific mortality, suggesting that current PGSs are more suitable for identify individuals at high risk of developing a disease rather than those more likely to suffer from the worst consequences and (4) if available, traits measured in a general population, but related to disease progression, can inform the genetic underpinning of disease progression, and PGSs derived from these related traits can predict progression better than PGSs for disease susceptibility. The limited overlap between genetic effects on disease susceptibility and disease progression may have several explanations. First, genetic influences on disease progression might be too small to detect. External environmental effects such as treatment choice, treatment response, quality and access to care might have a disproportionate impact on disease progression as compared to disease susceptibility, thus limiting the genetic influence. Heterogeneity in patients and their treatments has a substantial role in disease progression, and we are currently unable to account for all this heterogeneity. Using data from clinical trials rather than observational studies and including finer measurements, such as disease-relevant biomarkers, could obviate these shortcomings. We also noticed that adjusting for age at disease diagnosis reduces the overlap between susceptibility and progression because variants that increase disease susceptibility are often associated with earlier disease diagnosis28. Previous studies have demonstrated impact of adjusting for age in disease progression analyses35 and suggested that association between PGSs and measurement of disease progression may be mediated by age. Second, our definition of disease progression may be a poor proxy for the biological mechanisms that impact disease progression. Our approach aims to compare progression across multiple diseases, but this comes at the expense of a tailored definition of progression for each disease. Nonetheless, disease-specific mortality has been widely used as a measure of progression6,36,37,38. Association between composite genetic score of various risk factors, including several diseases considered in our study, and all-cause mortality has also been reported30. We tested this score in FinnGen and observed better prediction performance for some diseases of interest. In practice, biobank-based studies of disease progression often require simplification of definitions to maximize sample size, and disease-specific mortality information is typically available across biobanks. However, a definition that maximizes data availability might not be one that best reflects the genetic etiology of a specific disease. Third, as a common concern for all studies on disease progression, we examined the impact of index event bias resulting from conditioning on individuals with the disease. While this is not the main focus of this study, we found that index event bias alone does not fully explain the lack of concordance between genetic effects on susceptibility and progression observed in our study. Our empirical observations, in comparison to various simulations, indicate a relatively low heritability for disease progression, as defined by disease-specific mortality. Furthermore, heterogeneous phenotypes, such as mortality, although disease-specific, can be highly polygenic. In this case, even a perfect correction for index event bias will only be able to recover effect sizes that are not likely to be detected from a progression GWAS. The fact that only one genome-wide significant locus was detected from our progression GWASs, indicating low signal-to-noise ratio in the progression GWASs, might be a bigger concern than index event bias. Furthermore, most methods to correct for index event bias rely on fitting the relationships between variant effects on susceptibility and progression. In our case, this relationship is close to zero; thus, the correction will be small and insignificant. Lastly, even with all three aspects taken into account, there is still the possibility that we do not observe comparable genetic effects on disease susceptibility and progression simply because their underlying biological mechanisms are truly distinct. Given the aforementioned challenges in conducting GWAS of disease progression, one practical alternative would be to study genetic signals for disease progression in a general population and subsequently adapt them for within-patient prognostic prediction. For example, PGSs for autoimmune conditions derived from the general population are correlated with immune-related adverse events among cancer patients treated with immune checkpoint inhibitors39,40,41,42, and a PGS for ulcerative colitis was associated with immune checkpoint inhibitor-mediated colitis42. In our analysis, a longevity PGS derived from a GWAS of lifespan was substantially associated with survival for seven of nine diseases, suggesting that survival may be more affected by general factors related to mortality than by disease-specific factors. Similarly, the substantial findings for macrovascular complications in type 2 diabetes were recapitulated in the general population when looking at similar phenotypes, even among individuals without type 2 diabetes. In fact, the signal was stronger among individuals without type 2 diabetes, possibly reflecting index event bias. Notably, PGSs for cardiovascular conditions in the general population were better predictors of type 2 diabetes macrovascular complications than a PGS for type 2 diabetes susceptibility. Methods for cross-trait PGS43 might be leveraged to obtain progression PGS based on existing GWAS results in the general population. This relationship, however, is not always obvious. For diabetic microvascular complications, it is not as straightforward to find any population equivalent measurement, although it was interesting to observe a nominally substantial but weak effect of a PGS for age-related macular degeneration. On the other hand, this could indicate that such a progression definition is more unique to the diseased cohort and has the potential to yield truly progression-specific genetic signals, if sufficient power is available. First, while we explored the similarity in genetic effects between disease susceptibility and disease-specific mortality, we cannot decisively conclude that the biological underpinnings of susceptibility and progression are distinct. For example, a phenotype that serves as poor proxy for disease progression will result in attenuated effect sizes, despite genetic variants being causally associated with both susceptibility and progression. Nonetheless, the poor replication rate and opposite direction of effect observed for susceptibility signals on disease-specific mortality are consistent with a scenario where at least some variants have no shared effect on both susceptibility and progression. Second, our findings do not necessarily extend outside the diseases explored in this study, and further work is needed to confirm the observed trends across more disease categories. Third, whether death certificates accurately capture primary or contributing causes of death depends on the biobank and healthcare system. We tried to address these concerns by restricting the follow-up duration in sensitivity analyses, reasoning that deaths occurring shortly after disease diagnosis were more likely to be caused by the disease. In conclusion, our current results suggest that there is a limited overlap in genetic effects on disease susceptibility and progression, as defined by patients' mortality. Further refinement in inclusion criteria among the patient population and in the definitions of disease progression can be considered in future studies to robustly identify the genetic underpinnings of disease progression. This study was conducted in compliance with the relevant ethical guidelines and approved by the appropriate ethics committees. We selected nine common complex diseases spanning various disease categories for the analyses. The diseases are selected to meet following criteria: (1) have high epidemiological HR on mortality, so that mortality can be viewed as a reasonable prognosis; (2) constitute high global disease burden in terms of disability adjusted life years44; (3) be relatively common ( > 1% prevalence) in population and have reasonable patient bodies in all biobanks and (4) be heritable and have large-scale GWAS available to construct PGSs. All disease endpoints were defined as a composition of ICD-10 codes curated by the clinical expert groups from FinnGen, Institute for Molecular Medicine Finland and Finnish Institute for Health and Welfare19. The same disease definitions, in terms of ICD-10 codes, were adopted by all participating biobanks to the maximum possible extent. See Supplementary Table 2 for a list of diseases and relevant descriptive statistics. For all selected diseases, we defined mortality as our outcome. Precisely, we were interested in both all-cause mortalities, namely simple death status of the patient regardless of relevance to the disease, and disease-specific mortalities, meaning the death caused directly or indirectly by disease of interest specifically. Disease progression was evaluated as patients' survival from each type of mortality after being diagnosed with the disease. For all mortality GWASs, we consider only disease-specific mortality whenever possible for each participating biobank, whereas for the PGS analysis, both all-cause and disease-specific mortalities were evaluated. Similar to the disease endpoints, cause of death linked to each disease was also curated by clinical expert groups and defined in terms of ICD-10 codes45. See Supplementary Table 2 for definitions of cause-specific mortality for each disease of interest and available sample sizes from each biobank. To achieve variant-level effect comparison, a within-patient mortality GWAS was carried out for each selected disease using GATE26 for all biobanks, except Generation Scotland, which used SPACox27 as an alternative. The event of interest in this GWAS was patients' survival after disease diagnosis. For each disease of interest, GWAS was carried out separately within each ancestry group for biobanks that have a cause-specific mortality event count of 50 at minimum after quality control. Eligible individuals were restricted to patients having a follow-up time after diagnosis of three months (0.25 years) at minimum. We used the model below to examine SNP association with patients' survival: surv(duration of follow-up after diagnosis | disease-specific mortality) ~ SNP + patient's age of diagnosis + patient's birth year + sex + PCs + study-specific covariates, where study-specific covariates included other available nonheritable biobank-specific covariates, such as genotyping chip or batch. For analyses in the UK Biobank, to minimize potential impact of survivor bias, only patients with disease diagnosed after enrollment were considered. After conducting mortality GWAS for selected diseases within each contributing biobank, we then filtered the resulting summary statistics by imputation INFO scores and minor allele counts. We retained only variants with an imputation INFO score >0.7 and at least 20 minor allele counts for each summary statistic. For GWAS summary statistics with a different human genome build, we used the UCSC LiftOver tool46 to convert their genome coordinates into the hg38 assembly. Subsequently, for each disease, we meta-analyzed GWAS results from each biobank using fixed-effect meta-analysis implemented in METAL47, with which we also scanned for heterogeneity in effect sizes across different biobanks using Cochran's Q test. We applied an inverse-variance weighted meta-analysis scheme whenever possible. However, since SPACox does not have effect size or s.e. output, in Generation Scotland, we estimated direction of effect under a logistic regression model using PLINK48 and subsequently proceeded with a sample-size weighted meta-analysis using the Z-scores. This was done for four of the nine diseases for which Generation Scotland was one of the data sources: atrial fibrillation, breast cancer, coronary artery disease and type 2 diabetes. We compared our mortality GWAS results for each disease of interest with large-scale published GWAS on diagnosis of the same disease. For disease diagnosis GWAS, we extracted SNP effects of reported genome-wide significant leading SNPs at independently associated loci from each study. For chronic kidney disease, a large GWAS on estimated glomerular filtration rate was considered34. Specifically, we examined the effect sizes of independent lead SNPs on the binary diagnosis of chronic kidney disease reported in the study, ensuring a more comparable scale of measurement. For our meta-analyzed mortality GWAS, we identify independent genome-wide loci using summary statistics based on conditional analysis implemented in GCTA-COJO. We merged 5,000 Finnish genomes, which is one of the largest GWAS cohorts in this study, with EUR from Human Genome Diversity Project as linkage disequilibrium (LD) reference for this step. To carry out the effect size comparison for all diseases, we reran the meta-analysis of mortality GWAS, excluding results from Generation Scotland due to the use of an incomparable GWAS approach for the cohort. We compared genetic architectures between disease diagnosis and mortality in terms of SNP heritability estimated from the meta-analyzed mortality GWAS summary statistics using LD score regression49. For eligible traits, that is, traits with nonzero estimated SNP heritability, we further analyzed genetic correlation across disease diagnosis, mortality, and general longevity GWAS using the same tool. To ensure heritability comparison between disease susceptibility and progression endpoints not being subject to power issues resulting from difference in sample sizes and GWAS models, for each disease of interest, we also ran time-to-event GWAS to find SNP association with age of diagnosis using a randomly down-sampled cohort which had comparable number of total individuals and event counts as what was available for the within-patient mortality GWAS. surv(follow-up from birth until diagnosis | disease diagnosis) ~ SNP + patient's birth year + sex + PCs + study-specific covariates. This analysis was also carried out using GATE26 but in FinnGen and UK Biobank only, which are two of the largest participating biobanks in this study (see Supplementary Table 2 for sample sizes). For each selected disease, we derived variant weights for PGSs from GWAS summary statistics listed in Supplementary Table 2 using MegaPRS50. Heritability contributed by each variant was estimated under the BLD-LDAK model as recommended. Since we studied mortality, apart from the nine selected diseases, we also computed PGS weights for general longevity using the largest GWAS on lifespan29. Due to the heterogeneous and polygenic nature of lifespan, we used the LDAK-Thin model for SNP-level heritability estimation for this trait instead. Unlike the BLD-LDAK model used in variant weighting for other diseases, LDAK-Thin model does not take functional annotations into account but estimates SNP heritability only as function of SNP allele frequencies and local linkage structures. Variant weights were derived for 1,330,820 common SNPs (minor allele frequency > 0.1) lying in the intersection of HapMap3 (ref. 51) and 1000 Genomes52 that are available for each GWAS summary statistic. Once the SNP weights were derived, individual-level PGSs for each disease and general longevity were subsequently computed as a weighted sum of effect allele counts using PLINK48. Scores were standardized to have 0 mean and 1 as variance within each ancestry group. For the composite mortality PGS, we used sex-stratified SNP weights developed by ref. Scores for males and females were computed separately and subsequently combined during the association step to obtain a population effect estimate. As a baseline, we first examined whether the disease PGSs were associated with their diagnoses. For each selected disease, the association was first tested using a general linear model on case–control status as below: logit(Pr(Individual is diagnosed)) ~ disease PGS + birth year + sex + PC1-10. To achieve a fairer comparison with the other experiments, we also evaluated such relationship using a survival model on the age of diagnosis as below: To reduce noise in measurements, we limited these within-patient analyses to individuals having a follow-up time of at least three months (0.25 years) after the diagnosis. We tested the association of disease PGSs with our defined prognosis, namely patient survival, using the model below: surv(duration of follow-up after diagnosis | mortality) ~ disease PGS + birth year + sex + PC1-10 + age of diagnosis, surv(duration of follow-up after diagnosis | mortality) ~ general longevity PGS + birth year + sex + PC1-10 + age of diagnosis. For both associations, we examined both all-cause mortality and cause-specific mortality within the patient group. All analyses were corrected for sex, except in analyses for breast cancer and prostate cancer, where only female/male individuals were used. We only included biobanks where the count of events of interest in the analyzed ancestry group was 50 or more. We subsequently meta-analyzed effect sizes for the same ancestry group across biobanks using the inverse-variance weighted approach. For diseases with sufficient power, we derived mortality PGS weights using meta-analyzed mortality GWAS results of European populations from all available biobanks, except for FinnGen or Generation Scotland. Apart from FinnGen, which was used as a test cohort, we also left out results from Generation Scotland for this analysis because their summary statistics did not have effect size or s.e. and therefore cannot be used for inverse-variance weighted meta-analysis, which returns necessary statistics for weight derivation. After deriving PGS weights using MegaPRS50, we subsequently computed individual-level disease-mortality PGS for patients of each corresponding disease within FinnGen cohort. We evaluated the effects of these scores on predicting patients' disease mortality in FinnGen using the model below: surv(duration of follow-up after diagnosis | mortality) ~ disease-mortality PGS + birth year + sex + PC1-10 + age of diagnosis We ran a series of sensitivity analyses in eligible biobanks to ensure our observations on the PGSs association were robust, under considerations listed below. First, to demonstrate the impact of relevance between disease progression and susceptibility as shown in our theories, we examined the association between susceptibility PGS and all-cause mortality and compared the results with disease-specific mortality in FinnGen (see Supplementary Fig. We then considered other factors that may bias the results. Depending on each biobank's recruitment scheme, some patients were diagnosed before the start of their follow-up, which may lead to biased results due to the survivor effect. Therefore, we also ran these analyses for each disease using only samples from individuals enrolled before their first onset of the disease of interest (see Supplementary Fig. In this study, we aimed to define disease progression as accurately as possible by focusing our analysis on disease-caused mortality. However, some national death registries may not precisely capture the immediate cause of death, and some mortalities, while documented with the disease as one of the causes, may not be truly relevant to the diagnosed disease. To address this concern, we ran the same analysis using only patients with a restricted maximum follow-up length, since death taking place reasonably soon after being diagnosed might have more to do with the diagnosis, compared to death taking place decades after. Under this consideration, we varied the maximum duration of follow-up after diagnosis by 2, 5 or 10 years. The minimum is still 0.25 years for this analysis (see Supplementary Fig. 11b and Supplementary Table 8 for these results; see also Supplementary Table 2 for sample size breakdown by duration of follow-up in each biobank). To facilitate comparability between results, we reported the regression coefficients for PGS effect sizes on nine diseases for each sensitivity analysis and the main results. As shown above, we included the age of diagnosis as one of the covariates in all within-patient main analysis models to specifically investigate PGSs' unique genetic effect on disease progression by correcting for the diagnosis. As part of our sensitivity analysis, we also examined the role of these diagnosed ages in more detail. We repeated all the within-patient analyses for each disease by stratifying patients into early onset and late onset groups using 50% age of diagnosis quantile as a cutoff and compared the PGS effects across the two groups (see Supplementary Fig. We included all eligible individuals of each biobank in our main analysis, and one may argue that this could impact our effect size estimates. Therefore, we ran the same analysis in FinnGen with up to second-degree relatives removed (see Supplementary Fig. Since only patients were considered for most of our analyses, although some of the biobanks (for example, UK Biobank and BioMe) were known to be rather diverse, we ended up with enough power for the main results only for the European super-population. Nevertheless, comparison of results with other less powered but available populations can be found in Supplementary Fig. Forest plot for effects from each biobank is presented in Supplementary Fig. For type 2 diabetes, we explored the genetics of two additional widely considered progressions—macrovascular and microvascular complications. For macrovascular complications, we only consider patients who did not have any coronary artery disease, stroke or peripheral arterial disease incidents before the onset of type 2 diabetes. Among those, we define the ones having at least one of the aforementioned diagnoses after type 2 diabetes as cases for macrovascular complications. Event time is defined as the duration from a patient's diagnosis of type 2 diabetes to the earliest diagnosis of a macrovascular complication. Similarly, for microvascular complications, we consider onset of diabetic retinopathy, nephropathy and neuropathy after the patients' diagnosis of type 2 diabetes. For macrovascular complications, for which we identified genome-wide significant signals among diabetic patients, we further carried out a down-sampled time-to-event GWAS on population-comparable phenotypes, matching the case–control count in the progression GWAS. For this down-sampled GWAS, we considered onset of coronary artery disease, stroke, or peripheral arterial disease in nondiabetic population. Please see section ‘Simulation to explore the impact of index event bias' from Supplementary Note for details. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Summary statistics of meta-analyzed disease-mortality GWASs for nine diseases of interest can be found from figshare public project https://figshare.com/projects/Progression_GWAS/252002 (ref. Weights used to compute susceptibility and longevity PGSs can be downloaded from https://github.com/Zhiyu-9668/ProgressAnalysis/. Code used for within-biobank PGS association analyses, results meta-analysis and plotting can be found on public GitHub repo https://github.com/Zhiyu-9668/ProgressAnalysis/ and Zenodo under https://doi.org/10.5281/zenodo.16946874 (ref. Large-scale genome-wide association study of coronary artery disease in genetically diverse populations. Zhang, H. et al. Genome-wide association study identifies 32 novel breast cancer susceptibility loci from overall and subtype-specific analyses. Wightman, D. P. et al. A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer's disease. Escala-Garcia, M. et al. Genome-wide association study of germline variants and breast cancer-specific mortality. Tan, M. M. X. et al. Genome-wide determinants of mortality and motor progression in Parkinson's disease. Harroud, A. et al. Locus for severity implicates CNS resilience in progression of multiple sclerosis. Pan, G. et al. Role of genetic susceptibility variants in predicting clinical course in multiple sclerosis: a cohort study. Patel, R. S. et al. Association of chromosome 9p21 with subsequent coronary heart disease events: a GENIUS-CHD study of individual participant data. Lee, J. C. et al. Genome-wide association study identifies distinct genetic contributions to prognosis and susceptibility in Crohn's disease. Barbieux, P. et al. No prognostic role of a GWAS-derived genetic risk score in renal outcomes for patients from French cohorts with type 1 and type 2 diabetes. Liu, G. et al. Genome-wide survival study identifies a novel synaptic locus and polygenic score for cognitive progression in Parkinson's disease. Cho, S. M. J. et al. Genetic, sociodemographic, lifestyle, and clinical risk factors of recurrent coronary artery disease events: a population-based cohort study. Yaghootkar, H. et al. Quantifying the extent to which index event biases influence large genetic association studies. Mahmoud, O., Dudbridge, F., Davey Smith, G., Munafo, M. & Tilling, K. A robust method for collider bias correction in conditional genome-wide association studies. Adjustment for index event bias in genome-wide association studies of subsequent events. Viippola, E. et al. Data Resource Profile: nationwide registry data for high-throughput epidemiology and machine learning (FinRegistry). Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. The UK Biobank resource with deep phenotyping and genomic data. Leitsalu, L. et al. Cohort Profile: Estonian Biobank of the Estonian Genome Center, University of Tartu. Smith, B. H. et al. Cohort Profile: Generation Scotland: Scottish Family Health Study (GS:SFHS). Turnbull, C. Introducing whole-genome sequencing into routine cancer care: the Genomics England 100 000 Genomes Project. Finer, S. et al. Cohort Profile: East London Genes & Health (ELGH), a community-based population genomics and health study in British Bangladeshi and British Pakistani people. Gusev, A., Groha, S., Taraszka, K., Semenov, Y. R. & Zaitlen, N. Constructing germline research cohorts from the discarded reads of clinical tumor sequences. Efficient and accurate frailty model approach for genome-wide survival association analysis in large-scale biobanks. Bi, W., Fritsche, L. G., Mukherjee, B., Kim, S. & Lee, S. A fast and accurate method for genome-wide time-to-event data analysis and its application to UK Biobank. Findings and insights from the genetic investigation of age of first reported occurrence for complex disorders in the UK Biobank and FinnGen. Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. Nelson, C. P. et al. Association analyses based on false discovery rate implicate new loci for coronary artery disease. Malik, R. et al. Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes. Winkler, T. W. et al. Genome-wide association meta-analysis for early age-related macular degeneration highlights novel loci and insights for advanced disease. Wuttke, M. et al. A catalog of genetic loci associated with kidney function from analyses of a million individuals. Houlahan, K. E. et al. A polygenic two-hit hypothesis for prostate cancer. Jabbari, E. et al. Genetic determinants of survival in progressive supranuclear palsy: a genome-wide association study. Wu, C. et al. Genome-wide association study of survival in patients with pancreatic adenocarcinoma. Groha, S. et al. Germline variants associated with toxicity to immune checkpoint blockade. Khan, Z. et al. Polygenic risk for skin autoimmunity impacts immune checkpoint blockade in bladder cancer. Khan, Z. et al. Genetic variation associated with thyroid autoimmunity shapes the systemic immune response to PD-1 checkpoint blockade. Kember, R. L. et al. Polygenic risk of psychiatric disorders exhibits cross-trait associations in electronic health record data from European ancestry individuals. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. The UCSC genome browser and associated tools. & Abecasis, G. METAL: fast and efficient meta-analysis of genomewide association scans. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Bulik-Sullivan, B. K. et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. & Speed, D. Improved genetic prediction of complex traits from individual-level data or summary statistics. Integrating common and rare genetic variation in diverse human populations. 1000 Genomes Project Consortium A global reference for human genetic variation. This study received funding from the European Union's Horizon 2020 research and innovation program (grant agreement 101016775 to S.R. and A.G.), from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation program (grant 945733 to A.G.) and from Academy of Finland fellowship (grant 323116 to A.G.). We thank George Davey Smith for the invaluable comments on this manuscript. We thank all those who contributed samples and data for the FinnGen scientific project and P. VandeHaar for technical consultation on PheWeb. The FinnGen project is funded by two grants from Business Finland (HUS 4685/31/2016 and UH 4386/31/2016) and the following industry partners: AbbVie, AstraZeneca UK, Biogen, Bristol Myers Squibb (and Celgene Corporation & Celgene International II), Genentech, Merck Sharp & Dohme, a subsidiary of Merck & Co., Pfizer, GlaxoSmithKline Intellectual Property Development, Sanofi US Services, Maze Therapeutics, Janssen Biotech, Novartis and Boehringer Ingelheim. All Finnish biobanks are members of the BBMRI.fi infrastructure (https://www.bbmri.fi). The Finnish biobank data can be accessed through the Fingenious services (https://site.fingenious.fi/en/) managed by FINBB. Ethics approval for the UK Biobank study was obtained from the North West Centre for Research Ethics Committee (11/NW/0382). UK Biobank data used in this study were obtained under application 78537. We also acknowledge the EGCUT technical personnel. Other contributors were A. Allik, T. Annilo, M. Hass, A.-H. Jõks, A.-T. Kaasik, A. Keis, E. Leego, M. Leego, K. Lilienthal, K. Metsalu, E. Mihailov, K. Mikkel, E. Mölder, H. Niinemäe, T. Nikopensius, M. Puusepp, S. Smit, V. Soo, R. Tamm, M. Teder-Laving and M. Väli-Täht. This research was made possible through access to data in the National Genomic Research Library, which is managed by Genomics England Limited (a wholly owned company of the Department of Health and Social Care). The National Genomic Research Library holds data provided by patients and collected by the National Health Service (NHS) as part of their care and data collected as part of their participation in research. The National Genomic Research Library is funded by the National Institute for Health Research and NHS England. The members of this consortium are J. C. Ambrose, P. Arumugam, R. Bevers, M. Bleda, F. Boardman-Pretty, C. R. Boustred, H. Brittain, M. J. Brown, M. J. Caulfield, G. C. Chan, A. Giess, A. Hamblin, S. Henderson, T. J. P. Hubbard, R. Jackson, L. J. Jones, D. Kasperaviciute, M. Kayikci, A. Kousathanas, L. Lahnstein, S. E. A. Leigh, I. U. S. Leong, J. F. Lopez, F. Maleady-Crowe, M. McEntagart, F. Minneci, J. Mitchell, L. Moutsianas, M. Mueller, N. Murugaesu, A. C. Need, P. O'Donovan, C. A. Odhams, C. Patch, D. Perez-Gil, M. B. Pereira, J. Pullinger, T. Rahim, A. Rendon, T. Rogers, K. Savage, K. Sawant, R. H. Scott, A. Siddiq, A. Sieghart, S. C. Smith, A. Sosinsky, A. Stuckey, M. Tanguy, A. L. T. Tavares, E. R. A. Thomas, S. R. Thompson, A. Tucci, M. J. Welland, E. Williams, K. Witkowska, S. M. Wood and M. Zarowiecki. Genotyping of the GS:SFHS samples was carried out by the Genetics Core Laboratory at the Wellcome Trust Clinical Research Facility, Edinburgh, Scotland, and was funded by the Medical Research Council, UK, and the Wellcome Trust (Wellcome Trust Strategic Award ‘Stratifying Resilience and Depression Longitudinally' (STRADL) reference 104036/Z/14/Z). We are grateful to all the families who took part, the general practitioners and the Scottish School of Primary Care for their help in recruiting them and the whole Generation Scotland team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists, healthcare assistants and nurses. Ethical approval for the GS:SFHS study was obtained from the Tayside Committee on Medical Research Ethics (on behalf of the National Health Service). Genes & Health is/has recently been core-funded by Wellcome (WT102627 and WT210561), the Medical Research Council (UK; M009017, MR/X009777/1 and MR/X009920/1), Higher Education Funding Council for England Catalyst, Barts Charity (845/1796), Health Data Research UK (for London substantive site) and research delivery support from the NHS National Institute for Health Research Clinical Research Network (North Thames). Genes & Health is/has recently been funded by Alnylam Pharmaceuticals, Genomics, and a Life Sciences Industry Consortium of AstraZeneca, Bristol Myers Squibb Company, GlaxoSmithKline Research and Development, Maze Therapeutics, Merck Sharp & Dohme, Novo Nordisk A/S, Pfizer and Takeda Development Center Americas. We thank Social Action for Health, Centre of The Cell, members of our Community Advisory Group and staff who have recruited and collected data from volunteers. We thank the NIHR National Biosample Centre (UK Biocentre); the Social, Genetic & Developmental Psychiatry Centre (King's College London); Wellcome Sanger Institute; and Broad Institute for sample processing, genotyping, sequencing and variant annotation. We thank Barts Health NHS Trust, NHS Clinical Commissioning Groups (City and Hackney, Waltham Forest, Tower Hamlets, Newham, Redbridge, Havering, Barking and Dagenham), East London NHS Foundation Trust, Bradford Teaching Hospitals NHS Foundation Trust, Public Health England (especially David Wyllie), Discovery Data Service/Endeavour Health Charitable Trust (especially David Stables), Voror Health Technologies (especially Sophie Don) and NHS England (for what was NHS Digital)—for General Data Protection Regulation-compliant data sharing backed by individual written informed consent. This work was supported in part by the computational and data resources, and staff expertise provided by Scientific Computing and Data at the Icahn School of Medicine at Mount Sinai and supported by the Clinical and Translational Science Awards (CTSA; grant UL1TR004419) from the National Center for Advancing Translational Sciences. Additionally, this work was supported by the Office of Research Infrastructure of the National Institutes of Health (award S10OD026880), which allowed us to use Mount Sinai Data Warehouse (MSDW) data. Regarding HPI.MS resources, funding was provided by the Hasso Plattner Foundation (HPF). We thank all participants in the Mount Sinai BioMe Biobank. We also thank all of our recruiters who have assisted in data collection and management and are grateful for the computational resources and staff expertise provided by Scientific Computing at the Icahn School of Medicine at Mount Sinai. Zhiyu Yang, Julian Wanner, Bradley Jermy, Joel Rämö, Henrike Heyne, Samuli Ripatti, Nina Mars & Andrea Ganna Med Biotech Hub and Competence Center, Department of Medical Biotechnologies, University of Siena, Siena, Italy Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK Yipeng Cheng, Caroline Hayward, Riccardo E. Marioni & Daniel L. McCartney Hasso Plattner Institute, Mount Sinai School of Medicine, New York City, NY, USA Genetica Medica, Azienda Ospedaliera Universitaria Senese, Siena, Italy Department of Computer Science, Purdue University, West Lafayette, IN, USA Department of Biological Sciences, Purdue University, West Lafayette, IN, USA Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar J.R. provided valuable clinical input for type 2 diabetes analyses. supervised and led GWAS analysis in Genes & Health. provided valuable input on the theoretical framework and clinical impact. All authors contributed to revising the manuscript draft before submission. A.G. is the founder of Real World Genetics Oy. The other authors declare no competing interests. Nature Genetics thanks Peter Kraft and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Yang, Z., Pajuste, FD., Zguro, K. et al. Limited overlap between genetic effects on disease susceptibility and disease survival. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250929055003.htm'>First living cochlea outside the body shows how hearing really works</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 15:10:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Their new device allowed them to capture the live biomechanics of the cochlea's remarkable auditory powers, including exceptional sensitivity, sharp frequency tuning, and the ability to encode a broad range of sound intensities. Described in two recent papers (in PNAS and Hearing Research, respectively), the innovation is a product of Hudspeth's five decades of work illuminating the molecular and neural mechanisms of hearing -- insights that have illuminated new paths to preventing or reversing hearing loss. These challenges have long frustrated hearing researchers, because most hearing loss results from damage to sensory receptors called hair cells that line the cochlea. The organ has some 16,000 of these hair cells, so-called because each one is topped by a few hundred fine "feelers," or stereocilia, that early microscopists likened to hair. Each bundle is a tuned machine that amplifies and converts sound vibrations into electrical responses that the brain can then interpret. It's well documented that in insects and non-vertebrate animals -- such as the bullfrogs studied in Hudspeth's lab -- a biophysical phenomenon known as a Hopf bifurcation is key to the hearing process. At this knife-edge, even the faintest sound tips the system into movement, allowing it to amplify weak signals far beyond what would otherwise register. To answer that question, Hudspeth's team decided they needed to observe the active process in a mammalian cochlea in real time and at a greater level of detail than ever before. To do so, the researchers turned to the cochlea of gerbils, whose hearing falls in a similar range as humans. They placed a sliver of tissue within a chamber designed to reproduce the living environment of the sensory tissue, including continuously bathing it in nutrient-rich liquids called endolymph and perilymph and maintaining its native temperature and voltage. Key to the development of this custom device were Brian Fabella, a research specialist in the Hudspeth lab, and instrumentation engineer Nicholas Belenko, from Rockefeller's Gruss Lipper Precision Instrumentation Technologies Resource Center. They then began to play sounds via a tiny speaker and observed the response. "This experiment required an extraordinarily high level of precision and delicacy," notes Magnasco. "There's both mechanical fragility and electrochemical vulnerability at stake." Importantly, they observed that key to the active process was indeed a Hopf bifurcation -- the tipping point that turned mechanical instability into sound amplification. The scientists anticipate that experimentation using the ex vivo cochlea will improve their understanding of hearing and hopefully point to better therapies. "For example, we will now be able to pharmacologically perturb the system in a very targeted way that has never been possible before, such as by focusing on specific cells or cell interactions," says Alonso. There's a great need in the field for new potential therapies. "So far, no drug has been approved to restore hearing in sensorineural loss, and one reason for that is that we still have an incomplete mechanistic understanding of the active process of hearing," Gianoli says. "But now we have a tool that we can use to understand how the system works, and how and when it breaks -- and hopefully think of ways to intervene before it's too late." Scientists Found a Major Problem With Vitamin B12 Guidelines, and Your Brain Might Be at Risk Scientists Warn: Common Vitamin D Supplement Has “Previously Unknown” Negative Effect Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/are-we-alone-nasas-habitable-worlds-observatory-aims-to-find-out/'>NASA's First-Ever Alien-Hunting Space Telescope Could Enlighten Our New Dark Age</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA's First-Ever Alien-Hunting Space Telescope Could Enlighten Our New Dark Age An animated rendering of several engineering architecture concepts for NASA's Habitable Worlds Observatory, a next-generation flagship-class space telescope designed to find and study Earth-like planets around nearby sunlike stars. It's a sweltering Tuesday in Washington, D.C., the kind of day that stretches the definition of Earth as a “habitable” planet. But on an eighth-floor terrace near the U.S. Capitol building, dozens of people are outside anyway, talking and watching as passersby dart between pools of shade on the sticky streets below. Hundreds of scientists and engineers, NASA leaders, lobbyists, congressional staffers and Senator Mark Kelly of Arizona are also discussing the search for ET. It's the reason why everyone is here for a four-day science conference in late July, at the height of D.C.'s arguably habitable oppressive summer heat. They are prepping for NASA's next flagship space telescope, a multibillion-dollar machine that could launch as early as the mid-late 2030s and reveal whether Earths are as universal as sand on a beach or if our watery world is instead a lonely island in a quiet sea. Called the Habitable Worlds Observatory, the telescope's core mission is to search for signs of alien biospheres on Earth-size planets orbiting sunlike stars—worlds, at least in theory, that could be twins of our own. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. This search demands something conceptually simple but breathtakingly difficult: we need to actually see those faraway planets. “HabWorlds is not out to answer your typical science question,” said astrophysicist Chris Stark of NASA's Goddard Space Flight Center (GSFC), using what's become the preferred nickname for the mission, during a presentation the day before. As Stark spoke, “Are we alone?”—in huge letters—loomed over the audience from one of his slides, almost daring anyone to deny the question's importance. Absent any answer, it's entirely reasonable to think that our planet is, and that we are, either as common as dirt or astronomically implausible enough to represent life's sole spark throughout the entire observable universe. Some of those obstacles are technical, such as choosing the right targets to investigate and developing innovative but affordable instruments to get the job done. For NASA, those plans would lower the curtain on dozens of its iconic space missions and make it very difficult (if not impossible) to launch new ones. But space science is just one small stitch in the fabric of life for most Americans, and its potential ruin pales against this country's rising tide of socioeconomic disruption, civil unrest and political violence. So for HabWorlds, another, more pressing question looms: Can a nation so busy ripping itself apart come together to achieve any big, bold aspiration—especially one as lofty as solving life's cosmic mystery? Normally, city lights—and stray moonlight—made it difficult to see the twinkling tapestry overhead. This night was different because a total lunar eclipse brought deeper darkness to the sky. With the moon in Earth's shadow, Domagal-Goldman and his younger brother found themselves on their front lawn, gazing up at the stars. “I remember freezing for a solid couple of minutes,” Domagal-Goldman recalls. That's a really tough question, but it seems like one worth thinking about. Fast-forward more than a quarter-century, and Domagal-Goldman is now an astrobiologist in charge of NASA's entire astrophysics division, still transfixed by the abiding mystery of our apparent solitude. It's his job to manage and oversee NASA's astrophysics fleet, including such flagship missions as the venerable (and beloved) Hubble Space Telescope, the James Webb Space Telescope (JWST) and the upcoming Nancy Grace Roman Space Telescope. Each of these projects has a special place in his heart, but the Habitable Worlds Observatory gets him particularly excited. “I cannot hide my passion for this mission,” he admits. “In any field, there aren't many opportunities where you get to be a part of something that can literally change the world—and that's not hyperbole. I think about those pictures of Earth from the moon and what it meant to see ourselves that way, and I think this is a similar chance to change our view.” NASA's existing great observatories—like Hubble, JWST and their siblings—have helped us to read our cosmic origin story, revealing how the big bang's blaze of energy infused the newborn universe with the physics needed to forge atoms, stars, galaxies, planets and eventually life. The Habitable Worlds Observatory could tell us how often that story unfolds beyond Earth. It could be very common,” says Evgenya Shkolnik of Arizona State University, an astrophysicist and co-chair of HabWorlds' Community Science and Instrument Team. One of the mission's primary tasks will be to find a few dozen of those worlds that are ripe for in-depth study. As the telescope dissects those almost inscrutably faint planetary gleams, it will look for the spectral signatures of molecules that, here on Earth, are intimately associated with life. Short of receiving an interstellar radio transmission, finding wriggling Martian microbes or Europan cephalopods or watching a starship touch down next to the U.S. Capitol, a HabWorlds detection of gases such as oxygen, ozone and methane in the skies of a warm, wet, rocky planet would be the best evidence for alien life we could get. Biology's atmospheric fingerprints are of course still somewhat of an open question on alien worlds, but the data harvested by HabWorlds should at least give scientists something to debate. Such capabilities are crucial for getting broader buy-in from the entire astronomical community, not all of whom see a search for aliens as their guiding star. (Technically, much of that buy-in already occurred years ago, when a once-every-10-year Decadal Survey of the U.S. space science community anointed HabWorlds as the nation's highest new priority in astrophysics.) “This is the strongest science case of any flagship I've ever been a part of, whether I've helped build it, fix it or review it,” says Lee Feinberg, HabWorlds' principal architect and an engineer at GSFC. In March 1930, almost immediately after the stock market crash that rang in the Great Depression, crews began building a hulking skyscraper on Manhattan, N.Y.'s Fifth Avenue. Each week, those workers added more than four floors to the rising structure of glass, concrete and steel until it soared to be superlative: this was the world's tallest building, the first to top 100 stories. A mere 21 months had elapsed since architects drew up the first plans for what's now known as the Empire State Building—an art deco edifice that dramatically changed New York City's skyline and, in many ways, Americans' conceptions about what we as a nation could achieve. It demonstrates, he says, that with careful planning and concerted action, we can achieve great things in almost no time at all—great things such as building the first space telescope to find alien biospheres. “If we could match their level of planning and thought, I think this could be done a lot faster and for a lot less money. But we also need to solve hard technical problems,” Feinberg says. Impatience is not the main reason to move quickly; the best reason to go fast is that speed can save money. (Delays during the development of JWST cost NASA more than $1 million a day.) Launched in 1990, Hubble was designed to be serviceable, with instruments that could be swapped out and tinkered with by human hands. Between 1993 and 2009, astronauts visited Hubble five times to repair, upgrade or replace components, each time greatly improving the telescope and extending its life. Had Hubble's planners instead waited for technology to catch up with all their aspirations, the observatory might never have launched in the first place. “When we sent up Hubble, it was not a great observatory. It was an okay observatory,” says John Grunsfeld, a veteran astronaut, former NASA science chief and self-proclaimed Hubble hugger, who flew on three of those servicing missions. Astronauts John M. Grunsfeld (right) and Richard M. Linnehan (left) service the Hubble Space Telescope in low-Earth orbit on March 8, 2002. As it stands, some of the concepts Feinberg and his colleagues are considering even include an empty instrument bay for future filling-in. And the instruments that launch with HabWorlds don't need to be the fanciest, most sophisticated things we can build; they only need to be good enough until we replace them with more capable, next-generation hardware. In many ways, making the observatory serviceable could pay for itself. Beyond the possibility of answering existential questions about extraterrestrial life, all the scientific excitement really boils down to one big thing: HabWorlds is likely to boast the largest, most stable starlight-gathering mirror ever sent into space. But size doesn't always play nicely with other considerations, including cost. A bigger mirror can also mean fewer tough engineering challenges down the road. “There's an obvious trade that exists between how many technologies you want to advance [to support high-contrast coronagraphy] and how large your aperture is,” Stark said in his June presentation. And in theory, it's easy to make a mirror very big. You can break it into segments, as engineers do with ground-based telescopes, which then can dynamically adjust to function like a single, solid reflective surface. But with space telescopes, the crucial question is: Just how big of a mirror can we reasonably launch? “We are ultimately limited by our launch vehicles,” says Breann Sitarski, an optical engineer at GSFC and deputy principal architect of HabWorlds. Right now that means teams are looking at rockets with payload fairings between seven and nine meters wide—SpaceX's Starship, Blue Origin's New Glenn and NASA's Space Launch System. Given those constraints, one option is to design a segmented mirror to fit as-is within the fairings; another is to make a mirror that, like JWST's, folds up for launch and then unfurls in space. One includes a mirror that would be at least 6.5 meters in diameter, and the other uses a foldable mirror that would be at least eight meters across. “You don't want too many potential points of failure, but we do have heritage from JWST on folding mirrors, so that's one of the reasons we're looking at it,” Sitarski says. Additionally, for HabWorlds to image exo-Earths, the telescope must be ultrastable. “We're on the verge of being there,” Sitarski says. Ultimately, size is not something that scientists are keen to compromise on. Many are pushing for the larger option of an eight-meter-class mirror. Such a big, starlight-catching beast would make it easier to characterize the atmospheres of Earth-size worlds and, in particular, to search for telltale molecules in the near-infrared. In some ways, a larger mirror means the difference between finding potential biosignatures and simply finding life—having more light and more resolution makes ruling out potential “false positives” much easier. And we are learning that studying Earth-size worlds is hard to do with anything smaller. Even the mighty JWST is having trouble meeting that challenge as it scrutinizes rocky planets orbiting smaller, dimmer red dwarf stars. “Nothing would be more depressing than only finding one Earth-like planet where you're not 100 percent sure whether it has life and you don't have the sensitivity to do any more,” Mountain says. Avoiding that frustrating result is one argument for building a bigger mirror. “The other big argument for aperture is: What if you find nothing? What does it tell you about the abundance of life?” Feinberg says. Bigger mirrors are more expensive, however, and in NASA's politically charged project planning, even marginal budgetary fluctuations can mean the difference between enduring support and abrupt cancellation. “I'd rather have a HabWorlds that looks at 15 exoplanets than one that looks at zero because I couldn't afford it.” There's more behind such caution than just the lessons of JWST. NASA has pursued a HabWorlds-style mission before—a 2000s-era effort called the Terrestrial Planet Finder (TPF) that was ultimately abandoned after astronomers, policymakers and agency officials failed to win broader support from the astrophysics community and couldn't entirely agree on a preferred design, let alone a realistic price tag. The challenges for HabWorlds, at least right now, seem to be more political and less technological. No shortage of dire portents exist to make the sunny optimism of this summer's conference seem like little more than futile coping. The fate of federally funded U.S. science has yet to be decided, but if the Trump administration gets its way, the national research enterprise will be transformed into a flimsy husk of its former self, scarcely capable of supporting HabWorlds and other visionary projects. Macroeconomic effects from other policy decisions—such as runaway deficit spending and the White House's tariff-fueled trade wars—could further accelerate and amplify the downward spiral. In their proposed science appropriations bills, both congressional chambers largely rejected Vought's machinations, although the OMB is pursuing ways to evade congressional oversight and set U.S. science on a collision course with mediocrity. In truth, just as the scientific process can't prove a negative statement, it's hard to argue that a world without HabWorlds would be materially worse off. But history tells us, time and time again, how even seemingly small acts can have outsize effects that profoundly shape reality and our place within it. If nothing else, the Apollo lunar missions gave Americans a reason to look up again in unison, as enormous sociopolitical unrest wracked the U.S. The Empire State Building's spire became a towering beacon of progress on New York City's skyline, even as almost unbearable economic hardships unfolded at its base. Such grand projects expand the horizons of our hopes and dreams. “I think it's that kind of enthusiasm which allows us to carry these projects through all of the ups and downs that we're going to face between now and actually getting this launched,” Mountain says, “because it's going to be a journey.” She is a former contributing writer with National Geographic and was the interim physics editor at Quanta magazine. Her work has appeared in, among other publications, the New York Times, the Atlantic and Scientific American, for which she covered NASA's Artemis I mission. Lee Billings is a science journalist specializing in astronomy, physics, planetary science, and spaceflight, and is a senior editor at Scientific American. Billings joined Scientific American in 2014, and previously worked as a staff editor at SEED magazine. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a68120495/easter-island-rapamycin/'>Scientists Found a Miracle Drug in Easter Island's Soil. The Natives Never Saw a Dime.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Although rapamycin eventually became a billion-dollar drug, many of the people instrumental to its discovery—most importantly, the people of Rapa Nui—have been largely forgotten in both a monetary and historical sense. Because of this isolated nature, the history of the island is famously mysterious, and is openly debated to this day. However, Rapa Nui is also home to another famous export, this one medical in nature—an immunosuppressant drug called (fittingly) rapamycin. “This protein acts as a master regulator of cell growth and metabolism,” Ted Powers, a biologist from UC Davis, wrote in an article for The Conversation. “Together with other partner proteins, TOR controls how cells respond to nutrients, stress and environmental signals, thereby influencing major processes such as protein synthesis and immune function.” This makes rapamycin a critical tool for combatting organ transplant rejection and increasing the usefulness of medical interventions like stents. Powers also mentions that this molecule has now found its way into treatment for certain cancers, and could prove useful in areas like diabetes, neurodegenerative disease, and aging as well. Despite its origin being baked into its name, few know exactly how this isolated island in the Pacific provided the world with such a powerful, billion-dollar drug. But in Powers' new Conversation piece, he attempts to bring to light the indigenous origins of rapamycin and its controversial past. “As a scientist who built my career around the effects of rapamycin on cells, I felt compelled to understand and share the human story underlying its origin,” Powers wrote. Who officially earns recognition for scientific discoveries is a complex subject, and sometimes those who are the most deserving can be unfortunately forgotten (or even purposefully neglected). Sadly, rapamycin's origins come from an unfortunate example of scientific colonialism. When Medical Expedition to Easter Island (METEI), led by a Canadian research team, arrived on the shores of Easter Island in December 1964 with an honorable goal—to understand what could happen to the island's health with the arrival of its first international airport. During this expedition, scientists retrieved 200 soil samples, one of which contained the Streptomyces bacterium that produces rapamycin. But though it was well-intentioned, the original study (according to Powers) contained many instances of inherent bias. In June of 2024, members of the World Intellectual Property Organisation agreed to a treaty that would protect “traditional knowledge” from being patented by for-profit companies. With many areas of the natural world still unexplored, it's only a matter of time before the Earth offers humanity another life-changing remedy. Hopefully when that happens, all of the rightful participants in its discovery will be fairly recognized and compensated.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/six-new-gecko-species-discovered-by-loud-barking-mating-calls/'>Mildly Menacing Mating Calls Lead to Discovery of New Gecko Species</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Mildly Menacing Mating Calls Lead to Discovery of New Gecko Species Scientists found new gecko species hidden in plain sight in pristine deserts of southern Africa, thanks to their loud, barking mating calls On a cool, starry night six years ago, a young scientist crept barefoot across dunes in the Namib Desert in southern Africa. These ultra-shy lizards are only about the size of a stick of string cheese, but they possess powerful vocal cords to sing their nightly love songs. These tunes would eventually lead Becker to a startling discovery about the species that make them. Males issue the calls, which sound like a cartoon villain's evil laugh, from the entrance of their burrows to attract mates (and sometimes frighten tourists, who can be surprised that such small creatures can make such a loud noise). But because the animals' calls vary slightly in pitch and frequency, they also intrigued Becker, who was doing research at Gobabeb Namib Research Institute. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Becker had previously worked with frogs and knew that different species have unique mating calls to prevent interbreeding. He sent DNA samples from the geckos out for genetic sequencing. He spent several years scooping up even more geckos from other areas of varied terrain in the Namib and Kalahari deserts, working under the hypothesis that separate species might prefer different soils for their burrows. A male gecko (P maculatus) calls from the entrance of his burrow. This month Becker—now the chief curator of natural science at the National Museum of Namibia—published a paper in Vertebrate Zoology that revealed that the three species of barking geckos known to inhabit southern Africa are actually nine species. “Some of these species that were previously put together as one species are actually separated by 25 million years of evolution,” Becker says. But it's little wonder that these lizards hid under the same species name for so long. Barking geckos leave their burrows so rarely that even farmers who have heard the creatures on their land for decades likely have never seen one. The geckos look so similar that the fleeting glimpses scientists usually get of them are insufficient to tell them apart. Some of them look almost identical, like vermillion-speckled twins. That's why zoologists have shifted away from identifying species based mainly on physical traits and toward using what is called integrative taxonomy, where several pieces of evidence are used to decide on classifying new species, says Aaron Bauer, a gecko expert at Villanova University, who was not involved in the research but was a reviewer for the paper and an examiner on Becker's Ph.D. committee. “This paper is an excellent example of this, with morphology, color, calls and DNA sequence data all being used to support the authors' decisions,” Bauer says. Now that the geckos are properly classified, scientists can more accurately assess their distribution patterns, habitat preferences and physical traits. “I think they deserve a bit of extra attention. Ashley Balzer Vigil writes about astrophysics for NASA's Goddard Space Flight Center by day and moonlights as a freelance environmental writer. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250929054931.htm'>This tiny butterfly has the most chromosomes of any animal on Earth</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 03:43:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at the Wellcome Sanger Institute and the Institute of Evolutionary Biology (IBE: CSIC-UPF), Barcelona, have revealed that these chromosomes have been broken up over time, instead of duplicated. The first genomic study of this butterfly, published on September 10 in Current Biology, allows experts to begin to explore the evolutionary reasons behind this extreme number of chromosomes. Chromosome changes are also seen in human cancer cells, and therefore, understanding this process in different species could help inform cancer research. From this, experts have produced a gold-standard reference genome for this species, allowing researchers to compare this extreme genome to other butterflies and moths to understand more about how species form and change over time. For example, we might be able to understand how a species could respond to the increasing global temperature and if they have any genes or mechanisms that might protect them. The group to which the Atlas blue butterfly belongs contains many closely related species that evolved over a short period of time. Usually, it is assumed that this kind of extreme chromosomal change is negative; however, the Atlas blue butterfly has evolved and survived for millions of years. It is only now, due to climate change and human impacts on the environment, such as the destruction of cedar forests and overgrazing, that its populations are under threat. This research raises multiple questions that can now be addressed in the future. Splitting up the chromosomes could help give greater genetic diversity by allowing more frequent shuffling of genome parts or give other unknown benefits. While this may help butterflies to rapidly adapt, species with many chromosomes may also face challenges due to the extra complexity of this, potentially making them more vulnerable to extinction over time. Further investigations and comparisons with other butterflies could highlight whether any genes have been lost or preserved, giving us more information on the biology of the butterflies, but also a deeper understanding of evolution. Dr Roger Vila, senior author at the Institute of Evolutionary Biology, said: "Breaking down chromosomes has been seen in other species of butterflies, but not on this level, suggesting that there are important reasons for this process which we can now start to explore. Additionally, as chromosomes hold all the secrets of a species, investigating whether these changes impact a butterfly's behaviour could help form a full picture of how and why new species occur." Dr Charlotte Wright, first author at the Wellcome Sanger Institute, said: "When we set out to start to understand evolution in butterflies, we knew we had to sequence the most extreme, and somewhat mysterious, Atlas blue butterfly. Thanks to Roger Vila, who had previously worked with his colleague to find and identify this elusive butterfly, we were able to sequence this species, highlighting the collaborative nature of science. Being able to see, in detail, how the Atlas blue butterfly chromosomes have been split over time in specific places, we can start to investigate what benefits this might have, how it impacts their ability to adapt to their environment, and whether there are any lessons we can learn from their DNA that might aid conservation in the future." Professor Mark Blaxter, senior author at the Wellcome Sanger Institute, said: "Genomes hold the key to how a creature came to be, but also, where it might go in the future. To be able to tell the story of our planet, we must have the story of each species and see where they overlap and interact with each other. A full acknowledgement list can be found in the publication. Note: Content may be edited for style and length. Scientists Found a Major Problem With Vitamin B12 Guidelines, and Your Brain Might Be at Risk Scientists Warn: Common Vitamin D Supplement Has “Previously Unknown” Negative Effect Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250929054923.htm'>Scientists finally explain the real reason pregnant women get morning sickness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 03:03:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>UCLA researchers have uncovered a link between "morning sickness" symptoms and the body's natural, but complex, inflammatory response to biological and bodily changes during pregnancy. While uncomfortable, these symptoms are not typically a sign that anything is wrong with the health of the mother or the developing fetus, but rather an indication of a delicate balance unique to pregnant women. "During pregnancy, a mother's immune system faces a tricky challenge: it has to protect both her and the fetus from infection, but without accidentally attacking the fetus, whose genetic identity is half-foreign because it is half derived from the father. Normally, the immune system attacks anything that seems foreign, so in pregnancy, it has to carefully adjust to keep the fetus safe while still defending against infection," said UCLA anthropology professor Molly Fox. The researchers believe that this delicate balance, which protects mother and fetus, is achieved by a unique mix of inflammatory responses. For the study, the UCLA-led team of anthropologists and epidemiologists collected and analyzed blood samples to measure immune system molecules called cytokines. Cytokines are proteins that send signals to help the body launch a quick defense against sickness and regulate inflammation. Participants also filled out questionnaires that asked about morning sickness-related symptoms and food and smell aversions during the early stages of pregnancy. Sixty-four percent of study participants experienced odor or food aversions, primarily to tobacco smoke and meat. Food aversions, nausea and vomiting were also associated with a more pro-inflammatory immune balance. The correlation is consistent with researchers' theory that these symptoms may be part of an evolutionary adaptation that helps pregnant mothers' bodies minimize exposure to harmful substances, though the study's authors caution that the evidence is not definitive and more research is needed. Humans have the most invasive of all placentas, burrowing deep into maternal tissue. These immunological changes may induce nausea, which in turn encourages food avoidance that might act as an additional layer of protection, according to the researchers The researchers, including first author Dayoon Kwon, who just completed her Ph.D. in epidemiology at UCLA (and is now a postdoctoral fellow at Stanford), said that the study could help bolster recognition that nausea and vomiting are normal symptoms with biological underpinnings associated with healthy pregnancies. The study's results could help in paving the way for common-sense workplace accommodations, such as more efficient deployment of health care benefits and other helpful resources to reduce stigma, excessive absences and lost productivity. Funding for this study was provided by the National Institute of Health. Note: Content may be edited for style and length. Scientists Found a Major Problem With Vitamin B12 Guidelines, and Your Brain Might Be at Risk Scientists Warn: Common Vitamin D Supplement Has “Previously Unknown” Negative Effect Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250929054927.htm'>Astronomers stunned as fiery auroras blaze on a planet without a star</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-30 01:31:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Strong Northern Lights-like activity is the standout feature of today's weather report, which is coming at you from a strange, extrasolar world, instead of a standard TV studio. That is thanks to astronomers from Trinity College Dublin, who used the NASA/ESA/CSA James Webb Space Telescope to take a close look at the weather of a toasty nearby rogue planet, SIMP-0136. Surprisingly, these observations also illuminated SIMP-0136's strong auroral activity, similar to the Northern Lights here on Earth or the powerful aurora on Jupiter, which heat up its upper atmosphere. “And at over 1,500 °C, SIMP-0136 makes this summer's heat wave look mild,” he continued. These changes in temperature were related to subtle changes in the chemical composition of this free-floating planet, which is suggestive of storms – similar to Jupiter's Great Red Spot – rotating into view.” This is the first publication from the new ‘Exo-Aimsir' group led by Prof. Johanna Vos in Trinity's School of Physics, and includes contributions from all the group members, including PhD candidates Merle Schrader, Madeline Lam and Cian O'Toole. These data were initially published by a similar team led by Allison McCarthy at Boston University, but the new analysis has revealed more details about the atmosphere. Prof. Vos said: “This work is exciting because it shows that by applying our state-of-the-art modelling techniques to cutting-edge datasets from JWST, we can begin to piece together the processes that drive weather in worlds beyond our solar system. Understanding these weather processes will be crucial as we continue to discover and characterize exoplanets in the future.” Scientists Found a Major Problem With Vitamin B12 Guidelines, and Your Brain Might Be at Risk Scientists Warn: Common Vitamin D Supplement Has “Previously Unknown” Negative Effect Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            