
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-09</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02962-7'>Nonlinearity in marine fish populations is amplified by temperature variation and fast life histories</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 16:30:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Analysis of marine fish populations reveals that nonlinear dynamics are widespread and that the degree of nonlinearity is elevated by high temperature variation and for species with fast life histories. These findings support the nonlinear amplification hypothesis and challenge assumptions of stable equilibrium dynamics commonly used in ecology studies and in fisheries management. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 digital issues and online access to articles Prices may be subject to local taxes which are calculated during checkout & Luis, A. D. Nonlinear population dynamics are ubiquitous in animals. A broad analysis that reveals widespread nonlinear dynamics in animal populations. Chaos is not rare in natural ecosystems. This article reveals chaos, a type of nonlinear and unstable behaviour, to be readily displayed in animals. Anderson, C. N. K. et al. Why fishing magnifies fluctuations in fish abundance. This article identifies elevated nonlinearity as the cause of more variable fluctuations in exploited compared to unexploited fishes. Munch, S. B., Rogers, T. L. & Sugihara, G. Recent developments in empirical dynamic modelling. This review presents an accessible introduction to nonlinear dynamics and the empirical dynamic modelling framework. Ye, H. et al. Equation-free mechanistic ecosystem forecasting using empirical dynamic modeling. This article demonstrates improved prediction skill of empirical dynamic models compared with conventional fisheries models, for sockeye salmon. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. This is a summary of: Hechler, R. M. & Krkosek, M. Temperature variation and life history mediate nonlinearity in fluctuations of marine fish populations worldwide. Nonlinearity in marine fish populations is amplified by temperature variation and fast life histories. Version of record: 09 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a70178967/hydrophobic-metal-unsinkable-ship/'>Engineers Just Developed a Perfect Material for a Truly Unsinkable Ship</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'popularmechanics.com'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For years researchers have tried to unlock the impressive underwater abilities of the diving bell spider, and new study details their best attempt yet. Here's what you'll learn when reading this story: The natural world is humanity's greatest inspiration, and its influence can be seen all throughout our modern society. Honeycomb-inspired architecture maximizes storage while minimizing building materials. So, it comes as no surprise that in the pursuit of creating an unsinkable ship, scientists have once again ripped a page from nature's playbook. However, it still does have to breathe air, so it creates a dome-shaped web known as a “diving bell.” Then, using its superhydrophobic (SH) arms and abdomen, it fills those webs with breathable air bubbles carried in from elsewhere in the environment. For more than a decade, Guo has investigated ways to leverage the superhydrophobic abilities of the diving bell spider to create a material that's essentially unsinkable. In 2019, the team outlined a technique that etched complicated nano- and microscale patterns into metal so that they trapped air and created a superhydrophobic surface. Two treated parallel plates were then arranged facing inward, trapping enough air to maintain flotation while creating a waterproof compartment. After submerging this material under load for two months, the structure immediately resurfaced when the load was removed. While this design proved promising, it wasn't perfect, as the structures lost buoyancy when turned at extreme angles. “We tested them in some really rough environments for weeks at a time and found no degradation to their buoyancy. It doesn't take much imagination to understand why “unsinkable” materials could be of significant benefit. Guo said that these superhydrophobic tubes can be rafted together to handle large loads or serve as the flotation bedrock of emerging green energy technologies like wave energy generators. Are There More Humans on Earth Than We Thought? We May Have Been Wrong About the Origin of Life Did Scientists Find the Key to Staying Young?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-026-69344-w'>Assessing HPAI-H5 transmission risk across wild bird migratory flyways in the United States</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 14:29:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. As natural hosts of avian influenza viruses, wild birds pose an increasing threat to public health. Here, using surveillance data from wild bird infections across the United States (2022-2025), we show that HPAI-H5 transmission exhibits strong interspecific variation, seasonality, and spatial heterogeneity linked to migratory flyways. Phylogeographic analysis reveals that viral genotypes evolve from early, limited transmission along single migratory routes to a nationwide dispersal pattern spanning multiple migratory flyways. Anseriformes exhibits the highest number of infections but the lowest transmission risk, whereas Strigiformes demonstrates the greatest transmission risk. The HPAI-H5 transmission in wild birds along migratory flyways exhibits significant spatial heterogeneity and is associated with bird migration. Meteorological conditions are correlated with outbreak timing and may inform early warning efforts; however, these relationships are nonlinear. These findings provide a foundation for risk assessment, early warning systems, and integrated management of avian influenza in wild bird populations. The authors declare that the data supporting the findings of this study are available in the paper and the Supplementary Information file. The raw data and code have been uploaded to GitHub (https://github.com/kfang0/wild-bird-analysis-and-risk-assessment). Details on how to access the previously reported H5 sequencing data analyzed in this study (including GISAID accession numbers) are provided in the Source Data file accompanying this manuscript. For further information, please contact the corresponding author. Source data are provided with this paper. Xiong, W. & Zhang, Z. Influenza Virus Genomic Mutations, Host Barrier and Cross-species Transmission. Klim, H. et al. Serological analysis in humans in Malaysian Borneo suggests prior exposure to H5 avian influenza near migratory shorebird habitats. Fang, K. et al. Detection and prevalence of avian influenza epidemic in the southwest of Poyang Lake and analysis of the influence of meteorological factors. Dokter, A. M. et al. Seasonal abundance and survival of North America's migratory avifauna determined by weather radar. Emergence and interstate spread of highly pathogenic avian influenza A(H5N1) in dairy cattle in the United States. Taylor, L. First ever human case of H5N5 avian flu confirmed in the US. The episodic resurgence of highly pathogenic avian influenza H5 virus. Tian, H. et al. Avian influenza H5N1 viral and bird migration networks in Asia. Olsen, B. et al. Global patterns of influenza A virus in wild birds. Leguia, M. et al. Highly pathogenic avian influenza A (H5N1) in marine mammals and seabirds in Peru. Youk, S. et al. H5N1 highly pathogenic avian influenza clade 2.3.4.4b in wild and domestic birds: introductions into the United States and reassortments, December 2021-April 2022. Alcayna, T., Rao, V. B. & Lowe, R. Identifying the climate sensitivity of infectious diseases: a conceptual framework. Lancet Planet Health 9, 101291 (2025). Zhang, L. B., Ma, Z. Biological traits and biome features mediate responses of terrestrial bird demography to droughts. Koopmans, M. P. G. et al. The panzootic spread of highly pathogenic avian influenza H5N1 sublineage 2.3.4.4b: a critical appraisal of One Health preparedness and prevention. Cui, P. et al. Global dissemination of H5N1 influenza viruses bearing the clade 2.3.4.4b HA gene and biologic analysis of the ones detected in China. Damodaran, L., Jaeger, A. S. & Moncla, L. H. Ecology and spread of the North American H5N1 epizootic. Potential risk zones and climatic factors influencing the occurrence and persistence of avian influenza viruses in the environment of live bird markets in Bangladesh. & Haga, T. Association between highly pathogenic avian influenza outbreaks and weather conditions in Japan. Fourment, M., Darling, A. E. & Holmes, E. C. The impact of migratory flyways on the spread of avian influenza virus in North America. BMC Evol Biol 17, 118 (2017). The role of birds of prey in West Nile virus epidemiology. Grear, D. A., Hall, J. S., Dusek, R. J. & Ip, H. S. Inferring epidemiologic dynamics from viral evolution: 2014–2015 Eurasian/North American highly pathogenic avian influenza viruses exceed transmission threshold, R0 = 1, in wild birds and poultry in North America. McKenna, O. P., Mushet, D. M., Kucia, S. R. & McCulloch-Huseby, E. C. Limited shifts in the distribution of migratory bird breeding habitat density in response to future changes in climate. Prosser, D. J., Teitelbaum, C. S., Yin, S., Hill, N. J. Climate change impacts on bird migration and highly pathogenic avian influenza. Khare, S. et al. GISAID's Role in Pandemic Response. China CDC Wkly 3, 1049–1051 (2021). Animal and Plant Health Inspection Service. National Centers for Environmental Information (NCEI). Birds of the World - Comprehensive life histories for all bird species and families. Migratory Bird Program Administrative Flyways | U.S. Fish & Wildlife Service. Kim, W.-H. & Cho, S. Estimation of the basic reproduction numbers of the subtypes H5N1, H5N8, and H5N6 during the highly pathogenic avian influenza epidemic spread between farms. & Lipsitch, M. How generation intervals shape the relationship between growth rates and reproductive numbers. Wand, H., Dassaye, R., Reddy, T., Yssel, J. & Ramjee, G. Geographical-level contributions of risk factors for HIV infections using generalized additive models: results from a cohort of South African women. Zimbalist, A. et al. Utilization of locally estimated scatterplot smoothing (LOESS) regression to estimate missing weights in a longitudinal cohort of breast cancer patients. Lin, L. et al. Spatiotemporal distribution of migraine in China: analyses based on Baidu index. Zhang, C., Luo, L., Xu, W. & Ledwith, V. Use of local Moran's I and GIS to identify pollution hotspots of Pb in urban soils of Galway, Ireland. We would like to express our sincere gratitude to the Global Initiative on Sharing All Influenza Data (GISAID), the United States Department of Agriculture (USDA), and the National Centers for Environmental Information (NCEI) for providing open-access data. We thank Dr. Song Chao's team from the West China School of Public Health, Sichuan University, for their assistance with model development in the early stages. We also thank Edward Hill from the University of Liverpool for offering crucial insights into the manuscript. This research was financially supported by the Self-supporting Program of Guangzhou Laboratory (GZNL2024A01004 and SRPG22-007), the National Key Research and Development Program of China (2024YFC2311404), the Fujian Guiding Science and Technology Plan Project (2023Y0004), the Science and Technology Project of Jiangxi Provincial Health Commission (202410978), the Major Science and Technological Project of Jiangxi Province (20201BBG71010), the China Postdoctoral Science Foundation (K2825002) and the Postdoctoral Fellowship Program of CPSF (GZC20250516). These authors jointly supervised this work: Zeyu Zhao, Wentao Song, Tianmu Chen. State Key Laboratory of Vaccines for Infectious Diseases, Xiang An Biomedicine Laboratory, National Innovation Platform for Industry-Education Integration in Vaccine Research, School of Public Health, Xiamen University, Xiamen, Fujian, China Kang Fang, Jiahui Li, Hongfeng Zhao, Jiangshaya Bahati, Zeyu Zhao & Tianmu Chen Jiangxi Provincial Key Laboratory of Prevention and Treatment of Infectious Diseases, Jiangxi Medical Center for Critical Public Health Events, the First Affiliated Hospital, Jiangxi Medical College, Nanchang University, Nanchang, Jiangxi, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar : conceptualization, data analysis, visualization, writing—original draft; J.H.L. : data analysis, visualization, and writing—review; H.F.Z. : conceptualization, project administration, writing—review and editing. All the authors read and approved the final manuscript. Correspondence to Zeyu Zhao, Wentao Song or Tianmu Chen. The authors declare no competing interests. Nature Communications thanks Dae-Sung Yoo, and the other, anonymous, reviewer for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Fang, K., Li, J., Zhao, H. et al. Assessing HPAI-H5 transmission risk across wild bird migratory flyways in the United States. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-026-69289-0'>Recombinant zoster vaccine is associated with a reduced risk of dementia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 12:49:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. , Article number:  (2026) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Vaccines to prevent herpes zoster have been associated with reduced dementia risk. We conducted a retrospective matched cohort study of Kaiser Permanente Southern California members aged ≥65 years who received two doses of RZV 4 weeks–6 months apart between 01 April 2018 and 31 December 2020, with no dementia diagnoses or dementia medications prior to or within 6 months of their second RZV dose. Cox regression with inverse probability of treatment weighting was used to estimate adjusted hazard ratios (aHRs). The study included 65,800 RZV-vaccinated individuals and 263,200 unvaccinated matches. Vaccination with two doses of RZV was associated with a 51% lower risk of dementia (aHR: 0.49 [95% confidence interval (CI): 0.46–0.51]); aHRs were comparable across age, and racial and ethnic groups, but the risk reduction was stronger in females compared to males. In an evaluation of potential healthy vaccinee bias, the aHR of dementia for RZV compared to Tdap was 0.73 (95% CI: 0.67–0.79). Vaccination with two doses of RZV is associated with a statistically significant reduction in the risk of dementia in adults aged ≥65 years. After accounting for healthy vaccinee bias, RZV vaccination remains associated with a statistically significant lower risk of dementia. Individual-level data are not publicly available due to privacy concerns and protection of patient identities. Requests for aggregate-level data may be submitted to KPSC and are subject to review. De-identified aggregate-level data that support the findings of this study may be shared upon approval of a proposal and a signed data access agreement. Epidemiological analyses were conducted using standard commands in SAS 9.4 (SAS Institute; Cary, North Carolina, USA). World Health Organization. 2024 Alzheimer's disease facts and figures. GBD 2019 Dementia Forecasting Collaborators. Estimation of the global prevalence of dementia in 2019 and forecasted prevalence in 2050: an analysis for the Global Burden of Disease Study 2019. Lancet Public Health 7, e105–e125 (2022). Gao, J. et al. The association between varicella zoster virus and dementia: a systematic review and meta-analysis of observational studies. Wainberg, M. et al. The viral hypothesis: how herpesviruses may contribute to Alzheimer's disease. Devanand, D. P. Viral hypothesis and antiviral treatment in Alzheimer's disease. Moir, R. D., Lathe, R. & Tanzi, R. E. The antimicrobial protection hypothesis of Alzheimer's disease. Itzhaki, R. F. et al. Microbes and Alzheimer's disease. J. Alzheimers Dis. Hyde, V. R. et al. Anti-herpetic tau preserves neurons via the cGAS-STING-TBK1 pathway in Alzheimer's disease. A. et al. Varicella zoster virus infection. Steiner, I., Kennedy, P. G. & Pachner, A. R. The neurotropic herpes viruses: herpes simplex and varicella-zoster. Lancet Neurol. Thapa, S. et al. Risk of dementia following herpes zoster infection among patients undertreatment versus those not: a systematic review and meta-analysis. Health Sci. Chen, V. C. et al. Herpes zoster and dementia: a nationwide population-based cohort study. Bae, S. et al. Association of herpes zoster with dementia and effect of antiviral therapy on dementia: a population-based cohort study. Zhang, Y., Liu, W. & Xu, Y. Association between herpes zoster and Parkinson's disease and dementia: a systematic review and meta-analysis. Levine, K. S. et al. Virus exposure and neurodegenerative disease risk across national biobanks. Cairns, D. M., Itzhaki, R. F. & Kaplan, D. L. Potential involvement of varicella zoster virus in Alzheimer's disease via reactivation of quiescent herpes simplex virus type 1. Anwar, M. M. The emerging mechanism behind viral infections and extracellular vesicles hypotheses leading to neuroinflammation and Alzheimer's disease pathology. Bronzuoli, M. R., Iacomino, A., Steardo, L. & Scuderi, C. Targeting neuroinflammation in Alzheimer's disease. Gilden, D., Cohrs, R. J., Mahalingam, R. & Nagel, M. A. Varicella zoster virus vasculopathies: diverse clinical manifestations, laboratory features, pathogenesis, and treatment. Lehrer, S. & Rheinstein, P. H. Herpes zoster vaccination reduces risk of dementia. Lophatananon, A. et al. Shingles, Zostavax vaccination and risk of developing dementia: a nested case-control study-results from the UK Biobank cohort. Scherrer, J. F. et al. Impact of herpes zoster vaccination on incident dementia: a retrospective study in two patient cohorts. Schnier, C., Janbek, J., Lathe, R. & Haas, J. Reduced dementia incidence after varicella zoster vaccination in Wales 2013-2020. Remschmidt, C., Wichmann, O. & Harder, T. Frequency and impact of confounding by indication and healthy vaccinee bias in observational studies assessing influenza vaccine effectiveness: a systematic review. Ford, D. V. et al. The SAIL Databank: building a national architecture for e-health research and evaluation. BMC Health Serv. Eyting, M. et al. A natural experiment on the effect of herpes zoster vaccination on dementia. The Medical Letter. Herpes zoster vaccine (Zostavax). Centers for Disease Control and Prevention. What Everyone Should Know about Zostavax (Centers for Disease Control and Prevention, 2020). U.S. Food and Drug Administration. Approval letter - SHINGRIX. Lal, H. et al. Efficacy of an adjuvanted herpes zoster subunit vaccine in older adults. Cunningham, A. L. et al. Efficacy of the herpes zoster subunit vaccine in adults 70 years of age or older. Dooling, K. L. et al. Recommendations of the advisory committee on immunization practices for use of herpes zoster vaccines. Taquet, M., Dercon, Q., Todd, J. & Harrison, P. J. The recombinant shingles vaccine is associated with lower risk of dementia. Tang, E., Ray, I., Arnold, B. F. & Acharya, N. R. Recombinant zoster vaccine and the risk of dementia. Liu, Y., Jun, H., Becker, A., Wallick, C. & Mattke, S. Detection rates of mild cognitive impairment in primary care for the United States Medicare population. Mattke, S. et al. Expected and diagnosed rates of mild cognitive impairment and dementia in the U.S. Medicare population: observational analysis. Alzheimers Res Ther. Kaiser Permanente Southern California Department of Research & Evaluation. Chen, W. et al. Research data warehouse: using electronic health records to conduct population-based observational studies. Koebnick, C. et al. Sociodemographic characteristics of members of a large, integrated health care system: comparison with US Census Bureau data. International Society for Pharmacoepidemiology. Guidelines for Good Pharmacoepidemiology Practices (GPP) (Revision 3). Austin, P. C. An introduction to propensity score methods for reducing the effects of confounding in observational studies. Tartof, S. Y. et al. Safety of seasonal influenza vaccination in hospitalized surgical patients: a cohort study. Intern Med. Salmon, D. A., Black, S., Didierlaurent, A. M. & Moulton, L. H. Commentary on “common vaccines and the risk of dementia: a population-based cohort study”: science can be messy but eventually leads to truths. VanderWeele, T. J. & Ding, P. Sensitivity analysis in observational research: introducing the E-value. Intern Med. Quan, H. et al. Updating and validating the Charlson comorbidity index and score for risk adjustment in hospital discharge abstracts using data from 6 countries. Kim, D. H. et al. Measuring frailty in medicare data: development and validation of a claims-based frailty index. This work was supported and funded by GSK. Medical writing assistance for the manuscript and coordination were provided by Akkodis Belgium c/o GSK. The authors would like to thank the patients of Kaiser Permanente for their partnership. Their information, collected through our electronic health record system, leads to findings that help us improve care for our patients and can be shared with the larger community. Department of Research & Evaluation, Kaiser Permanente Southern California, Pasadena, CA, USA Emily Rayens, Lina S. Sy, Lei Qian, Bradley K. Ackerson, Julia Tubert, Yi Luo, Punam P. Modha, Raul O. Calderon & Hung Fu Tseng GSK, Rockville, MD, USA Elizabeth Chmielewski-Yee, Driss Oraichi & Huifeng Yun GSK, Collegeville, PA, USA Department of Pharmaceutical Health Services Research, School of Pharmacy, University of Maryland, Baltimore, MD, USA Department of Health Systems Science, Kaiser Permanente Bernard J. Tyson School of Medicine, Pasadena, CA, USA Hung Fu Tseng Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar is the guarantor of the work and accepts full responsibility for the integrity of the study and the decision to publish. designed and conducted this study, including interpretation of data. performed statistical analysis. provided administrative, technical, and material support. drafted the manuscript. All authors critically revised the manuscript and contributed to the final drafting. The corresponding author attests that all listed authors meet authorship criteria and that no others meeting the criteria have been omitted. Correspondence to Emily Rayens. The authors declare the following competing interests: E.R., L.S.S., L.Q., B.K.A., Y.L., P.P.M., and H.F.T. received research funding from AstraZeneca. received research funding from Moderna. ER and BKA received research funding from F2G, Inc. received research funding from Dynavax. received research funding from Pfizer. received research funding from Genentech. received research funding from GSK (unrelated to this study). are employed by GSK and hold financial equities in GSK. The authors declare no other financial or non-financial interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Findings from this study were presented at the Alzheimer's Association International Conference 2025. Shingrix is a trademark owned by or licensed to GSK. Zostavax is a trademark of Merck. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Rayens, E., Sy, L.S., Qian, L. et al. Recombinant zoster vaccine is associated with a reduced risk of dementia. Received: 15 July 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/quad-god-ilia-malinin-and-the-science-of-figure-skatings-near-impossible/'>'Quad God' Ilia Malinin and the science of figure skating's near-impossible jumps</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ilia Malinin competes in the Championship Men Free Skating during the 2026 United States Figure Skating Championships at Enterprise Center on January 10, 2026, in St Louis, Mo. For at least a decade, the quadruple axel jump was figure skating's white whale. “It's been this unreachable thing, like the four-minute mile” once was, says Matthew Lind, a technical specialist for U.S. But at 4.5 rotations, the quad axel is a special case, and it remained incredibly risky to attempt, let alone to perfect. He calls himself the Quad God, and it's hard to disagree with him. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “He's a phenom,” says figure skating coach and former Olympian Karen Preston. Malinin represents the direction in which figure skating has been moving for at least 20 years, rewarding harder and flashier jumps. I spoke with figure skating coaches and biomechanics researchers to learn how these jumps became possible, what makes Malinin special, and whether we're headed toward the era of quintuples. Skaters glide along the ice to build momentum, then twist themselves up like springs and push off with explosive muscle movements. During takeoff, skaters push off the ice at an angle, which lets them maximize angular momentum, or the ability to rotate quickly. The axel is the only jump in which skaters take off while facing forward, which is part of what makes it so difficult—because it is landed backward, skaters must rotate an extra half turn before they land. The five other jumps take off backward and can be launched from the figure skate blade's distinctive toe pick or from either of its two edges. “You're really putting your body at risk,” Lind says. For elite skaters, landing these harder jumps requires strength and conditioning, innate talent, mental focus, great coaches—and a slight body, explains biomechanics researcher Lee Cabell, who coaches figure skating at IceWorks Skating Club in Pennsylvania. The importance of a narrow body comes down to physics. Because angular momentum must be conserved, skaters can't change their rotation potential once they're in the air. Narrower bodies, then, have the capacity to spin faster. “These very slight but muscular athletes really have the advantage for rotating,” says Sarah T. Ridge, a biomechanics researcher at the University of Hartford, who studies figure skating. With his slight body, immense talent, and parents who are former Olympians and double as his coaches, Malinin is a rare skater with the whole package, Cabell says. For a while, it seemed like quads were taking over women's skating, too. In the early 2020s the field was dominated by young, predominantly Russian teenagers who could land quads, a feat that is easier in narrower, prepubescent bodies. Now the quad's relevance in women's skating has faded. All eyes there are on the triple axel, an element that was once very risky but which skaters are now landing with apparent ease, says Deborah King, who studies biomechanics at Ithaca College. This development is less likely to grab headlines, but it's another example of figure skating moving toward harder and harder jumps. At the time, judges awarded skaters a maximum of six points for both artistry and technique. In response to “skategate,” the ISU created a new scoring system that capped artistry scores at 10 points per judge but had no ceiling for technical scores, which are awarded for the difficulty of jumps, spins, and more. This means that, theoretically, skaters can always do more and harder jumps to surpass their competition, Lind says. To be competitive, skaters have kept learning harder skills, and both they and their coaches have accumulated expertise along the way. “I think just small increments in all of those things can add up” to a skater that has enough strength and skill to land a quad comfortably, King says. Technology has also given coaches more resources to teach harder jumps. Pole harnesses, for example, attach around skaters' torsos and connect to a rod their coach holds like a fishing pole. This allows coaches to support some of their students' weight in midair so they can try jumps they couldn't yet land on their own. These harnesses allow skaters to develop muscle memory and also protect them from injury. “We didn't have this when I was a skater. I look at these and I'm like, ‘Oh, that would have been nice,'” says Lind, who progressed to a triple axel before retiring in 2004. But perhaps the most important technology is video analysis. About 20 years ago coaches started using a software called Dartfish that allows them to play back and overlay videos of their students' jumps in real time, Lind explains. “We're getting more people from different places doing harder skills, just because the information is more readily available and analyzable” for coaches, he says. Preston specializes in Dartfish video analysis at the Toronto Cricket Skating and Curling Club. Now we can break [the jump] down into extremely tangible moments.” The Skating Club of Boston, where Lind coaches, is also experimenting with three-dimensional cameras to give coaches even more valuable information about their skaters' technique. Scrutinizing every moment of a jump has also allowed coaches to re-think and sometimes change how they teach jump technique. For example, skaters were once told to look in the direction of their jump. “Nowadays, that is forbidden,” Preston says—it throws the head off axis, which decreases rotation speed. “There's a lot of variables” in any given jump, Preston says. And perfection doesn't really exist—unless you're looking at Ilia Malinin.” Every expert I spoke with gushed about Malinin's technique. To Lind's expert eye, what makes Malinin's technique unique is his patience. He doesn't immediately rush to rotate his quad axel upon takeoff. At this point, it seems inevitable that Malinin will land a quintuple jump. If history is any clue, we'll see other skaters catch up soon enough, as they eventually did with quadruple jumps. But so long as skate boots and blades do not change, we'll eventually reach a point where more rotations are impossible. Whether this movement toward harder jumps is the right direction for skating's future is fiercely debated, both among the figure skating community and fans. “I do think that quads are here to stay, permanently, in all levels,” Preston says. Brown, a 31-year-old U.S. skater, is renowned for his artistry but has struggled to land quads. “Having been on Jason Brown's coaching team and knowing the magic that he has brought to the sport, I would hate to lose that,” she says. Allison Parshall is associate editor for mind and brain at Scientific American and she writes the weekly online Science Quizzes. As a multimedia journalist, she contributes to Scientific American's podcast Science Quickly. Parshall's work has also appeared in Quanta Magazine and Inverse. She graduated from New York University's Arthur L. Carter Journalism Institute with a master's degree in science, health and environmental reporting. She has a bachelor's degree in psychology from Georgetown University. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-026-03003-9'>Sensitive detection of cancer antigens enabled by user-defined peptide libraries</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 10:31:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Human leukocyte antigen (HLA)-bound tumor peptides can be routinely isolated from cancer samples and identified using mass spectrometry (MS). However, MS approaches can be stochastic or rely on spectral libraries, which are not customarily available for individual-specific peptides, thus limiting the ability to discover novel peptides. Here, we introduce Pepyrus, which generates user-defined, individual-specific or disease-specific peptide libraries in Escherichia coli to improve the sensitivity and confidence of MS peptide identification, including lowly abundant neoantigens. Using Pepyrus-generated peptide libraries paired with an HLA-specific data-independent acquisition strategy, we recover >75% of the expected sequences per single injection for libraries of >10,000 peptides and identify 0.1 fmol of spiked-in peptides in a complex background. We apply Pepyrus to create personalized libraries, facilitating identification of clinically relevant HLA peptides, including several novel peptides from cell lines derived from persons with melanoma and renal cell carcinoma. Pepyrus enables identification of rare HLA-bound peptides and provides the ability to generate large training datasets to improve spectra, retention time and ion mobility prediction tools. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Plasmids generated in this study are available upon reasonable request and are subject to a Materials Transfer Agreement. Raw data pertaining to primary participant samples were deposited to the Database of Genotypes and Phenotypes under the following accession numbers: Ribo-seq data for MEL2 (Pt-A) and MEL11 (Pt-C) and matching RNA-seq data for MEL11, phs001451.v5.p1; melanoma RNA-seq data from previous study4, phs001451.v1.p1; raw DNA RNA sequencing files for RCC-102 tumor sample from previous study, phs003710.v1.p1. The mass spectra, PSM results and the protein sequence databases used for searches were deposited to the public proteomics repository MassIVE (http://massive.ucsd.edu) under accession number MSV000096555. Transcript and protein sequences for nuORFdb version 1.0 (hg19) and version 1.2 (hg38) can be obtained online (https://proteomics.broadinstitute.org/nuORFdb/). Generated plasmid sequencing data were deposited to the Gene Expression Omnibus under accession number GSE309497. All other data are available from the corresponding author on reasonable request. Source data are provided with this paper. Haen, S. P., Löffler, M. W., Rammensee, H.-G. & Brossart, P. Towards new horizons: characterization, classification and implications of the tumour antigenic repertoire. Sensi, M. & Anichini, A. Unique tumor antigens: evidence for immune control of genome integrity and immunogenic targets for T cell-mediated patient-specific immunotherapy. Sellars, M. C., Wu, C. J. & Fritsch, E. F. Cancer vaccines: building a bridge over troubled waters. Ott, P. A. et al. An immunogenic personal neoantigen vaccine for patients with melanoma. Sahin, U. et al. Personalized RNA mutanome vaccines mobilize poly-specific therapeutic immunity against cancer. Keskin, D. B. et al. Neoantigen vaccine generates intratumoral T cell responses in phase Ib glioblastoma trial. Rojas, L. A. et al. Personalized RNA neoantigen vaccines stimulate T cells in pancreatic cancer. Hong, D. S. et al. Autologous T cell therapy for MAGE-A4+ solid cancers in HLA-A*02+ patients: a phase 1 trial. Parkhurst, M. et al. Adoptive transfer of personalized neoantigen-reactive TCR-transduced T cells in metastatic colorectal cancer: phase 2 trial interim results. Chong, C., Coukos, G. & Bassani-Sternberg, M. Identification of tumor antigens with immunopeptidomics. Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Michalski, A., Cox, J. & Mann, M. More than 100,000 detectable peptide species elute in single shotgun proteomics runs but the majority is inaccessible to data-dependent LC–MS/MS. J. Proteome Res. Granados, D. P. et al. Impact of genomic polymorphisms on the repertoire of human MHC class I-associated peptides. Article PubMed van der Lee, D. I. et al. Mutated nucleophosmin 1 as immunotherapy target in acute myeloid leukemia. Pak, H. et al. Sensitive immunopeptidomics by leveraging available large-scale multi-HLA spectral libraries, data-independent acquisition, and MS/MS prediction. Gabriel, W., Picciani, M., The, M. & Wilhelm, M. Deep learning-assisted analysis of immunopeptidomics data. Adams, C. et al. Fragment ion intensity prediction improves the identification rate of non-tryptic peptides in timsTOF. Oliinyk, D. et al. diaPASEF analysis for HLA-I peptides enables quantification of common cancer neoantigens. Cell Proteomics 24, 100938 (2025). Gessulat, S. et al. Prosit: proteome-wide prediction of peptide tandem mass spectra by deep learning. Bouwmeester, R., Gabriels, R., Hulstaert, N., Martens, L. & Degroeve, S. DeepLC can predict retention times for peptides that carry as-yet unseen modifications. Zeng, W.-F. et al. AlphaPeptDeep: a modular deep learning framework to predict peptide properties for proteomics. C Silva, A. S., Bouwmeester, R., Martens, L. & Degroeve, S. Accurate peptide fragmentation predictions allow data driven approaches to replace and improve upon proteomics search engine scoring functions. Degroeve, S. & Martens, L. MS2PIP: a tool for MS/MS peak intensity prediction. Wilhelm, M. et al. Deep learning boosts sensitivity of mass spectrometry-based immunopeptidomics. Declercq, A. et al. TIMS2Rescore: a data dependent acquisition-parallel accumulation and serial fragmentation-optimized data-driven rescoring pipeline based on MS2Rescore. J. Proteome Res. Strauss, M. T. et al. AlphaPept: a modern and open framework for MS-based proteomics. Lee, C.-D. et al. An improved SUMO fusion protein system for effective production of native proteins. Aebersold, R. & Mann, M. Mass spectrometry-based proteomics. Roepstorff, P. & Fohlman, J. Proposal for a common nomenclature for sequence ions in mass spectra of peptides. Zubarev, R. A. & Makarov, A. Orbitrap mass spectrometry. Prensner, J. R. et al. What can Ribo-Seq, immunopeptidomics, and proteomics tell us about the noncanonical proteome? Skiadopoulou, D. et al. Retention time and fragmentation predictors increase confidence in identification of common variant peptides. J. Proteome Res. Frewen, B. E., Merrihew, G. E., Wu, C. C., Noble, W. S. & MacCoss, M. J. Analysis of peptide MS/MS spectra from large-scale proteomics experiments using spectrum libraries. Lam, H. et al. Development and validation of a spectral library searching method for peptide identification from MS/MS. Yang, X. et al. Autoimmunity-associated T cell receptors recognize HLA-B*27-bound peptides. Phulphagar, K. M. et al. Sensitive, high-throughput HLA-I and HLA-II immunopeptidomics using parallel accumulation–serial fragmentation mass spectrometry. Malakhov, M. P. et al. SUMO fusions and SUMO-specific protease for efficient expression and purification of proteins. Kapust, R. B., Tözsér, J., Copeland, T. D. & Waugh, D. S. The P1′ specificity of tobacco etch virus protease. Chang, J. Y. Thrombin specificity. Requirement for apolar amino acids adjacent to the thrombin cleavage site of polypeptide substrate. Gomez-Zepeda, D. et al. Thunder-DDA-PASEF enables high-coverage immunopeptidomics and is boosted by MS2Rescore with MS2PIP timsTOF fragmentation prediction model. Käll, L., Storey, J. D., MacCoss, M. J. & Noble, W. S. Posterior error probabilities and false discovery rates: two sides of the same coin. J. Proteome Res. Oliveira, G. et al. Phenotype, specificity and avidity of antitumour CD8+ T cells in melanoma. Oliveira, G. et al. Landscape of helper and regulatory antitumour CD4+ T cells in melanoma. Klaeger, S. et al. Optimized liquid and gas phase fractionation increases HLA-peptidome coverage for primary cell and tissue samples. Hodis, E. et al. A landscape of driver mutations in melanoma. Ouspenskaia, T. et al. Unannotated proteins expand the MHC-I-restricted immunopeptidome in cancer. Eisenberg, G. et al. Transcutaneous immunization with hydrophilic recombinant gp100 protein induces antigen-specific cellular immune response. Kawakami, Y. et al. Recognition of multiple epitopes in the human melanoma antigen gp100 by tumor-infiltrating T lymphocytes associated with in vivo tumor regression. Salgaller, M. L. et al. Recognition of multiple epitopes in the human melanoma antigen gp100 by peripheral blood lymphocytes stimulated in vitro with synthetic peptides. Bruderer, R. et al. Extending the limits of quantitative proteome profiling with data-independent acquisition and application to acetaminophen-treated three-dimensional liver microtissues. Kawakami, Y. et al. Identification of the immunodominant peptides of the MART-1 human melanoma antigen recognized by the majority of HLA-A2-restricted tumor infiltrating lymphocytes. Braun, D. A. et al. A neoantigen vaccine generates antitumour immunity in renal cell carcinoma. Jiang, Q. et al. HIF regulates multiple translated endogenous retroviruses: implications for cancer immunotherapy. Takahashi, Y. et al. Regression of human kidney cancer following allogeneic stem cell transplantation is associated with recognition of an HERV-E antigen by T cells. Bonaventura, P. et al. Identification of shared tumor epitopes from endogenous retroviruses inducing high-avidity cytotoxic T cells for cancer immunotherapy. Saini, S. K. et al. Human endogenous retroviruses form a reservoir of T cell targets in hematological cancers. Smith, C. C. et al. Endogenous retroviral signatures predict immunotherapy response in clear cell renal cell carcinoma. Bassani-Sternberg, M. et al. Direct identification of clinically relevant neoepitopes presented on native human melanoma tissue by mass spectrometry. Yi, X. et al. caAtlas: an immunopeptidome atlas of human cancer. Kraemer, A. I. et al. The immunopeptidome landscape associated with T cell infiltration, inflammation and immune editing in lung cancer. Ferreira, H. J. et al. Immunopeptidomics-based identification of naturally presented non-canonical circRNA-derived peptides. Liao, H. et al. MARS an improved de novo peptide candidate selection method for non-canonical antigen target discovery in cancer. Minegishi, Y. et al. Differential ion mobility mass spectrometry in immunopeptidomics identifies neoantigens carrying colorectal cancer driver mutations. Hoenisch Gravel, N. et al. TOFIMS mass spectrometry-based immunopeptidomics refines tumor antigen identification. Wahle, M. et al. IMBAS-MS discovers organ-specific HLA peptide patterns in plasma. & Türeci, Ö Personalized vaccines for cancer immunotherapy. Huber, F. et al. A comprehensive proteogenomic pipeline for neoantigen discovery to advance personalized cancer immunotherapy. Bruno, P. M. et al. High-throughput, targeted MHC class I immunopeptidomics using a functional genetics screening platform. Chong, C. et al. Integrated proteogenomic deep sequencing and analytics accurately identify non-canonical peptides in tumor immunopeptidomes. A., Soh, W. T., Mishto, M. & Liepe, J. iBench: a ground truth approach for advanced validation of mass spectrometry identification method. Parker, R. et al. The choice of search engine affects sequencing depth and HLA class I allele-specific peptide repertoires. Harper, S. & Speicher, D. W. Purification of proteins fused to glutathione S-transferase. Zhu, K. et al. Cleavage of fusion proteins on the affinity resins using the TEV protease variant. Protein Expr. Lau, Y.-T. K. et al. Discovery and engineering of enhanced SUMO protease enzymes. Alspach, E. et al. MHC-II neoantigens shape tumour immunity and response to immunotherapy. Abelin, J. G. et al. Mass spectrometry profiling of HLA-associated peptidomes in mono-allelic cells enables more accurate epitope prediction. Chen, B. et al. Predicting HLA class II antigen presentation through integrated deep learning. Deutsch, E. W. et al. High-quality peptide evidence for annotating non-canonical open reading frames as human proteins. Preprint at bioRxiv https://doi.org/10.1101/2024.09.09.612016 (2024). Soufi, B. & Macek, B. Stable isotope labeling by amino acids applied to bacterial cell culture. Park, H.-S. et al. Expanding the genetic code of Escherichia coli with phosphoserine. Fan, C., Ip, K. & Söll, D. Expanding the genetic code of Escherichia coli with phosphotyrosine. FEBS Lett. Ojima-Kato, T., Nagai, S. & Nakano, H. N-terminal SKIK peptide tag markedly improves expression of difficult-to-express proteins in Escherichia coli and Saccharomyces cerevisiae. Berman, H. M. The Protein Data Bank. Nucleic Acids Res. Yu, F. et al. Fast quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. da Veiga Leprevost, F. et al. Philosopher: a versatile toolkit for shotgun proteomics data analysis. Yang, K. L. et al. MSBooster: improving peptide identification rates using deep learning-based features. Yu, F., Haynes, S. E. & Nesvizhskii, A. I. IonQuant enables accurate and sensitive label-free quantification with FDR-controlled match-between-runs. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Kong, A. T., Leprevost, F. V., Avtonomov, D. M., Mellacheruvu, D. & Nesvizhskii, A. I. MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics. Li, K., Teo, G. C., Yang, K. L., Yu, F. & Nesvizhskii, A. I. diaTracer enables spectrum-centric analysis of diaPASEF proteomics data. Tareen, A. B. Logomaker: beautiful sequence logos in Python. Henikoff, S. & Henikoff, J. G. Amino acid substitution matrices from protein blocks. Bremel, R. D. & Homan, E. J. An integrated approach to epitope analysis I: dimensional reduction, visualization and prediction of MHC binding using amino acid principal components and regression approaches. Immunome Res. McInnes, L., Healy, J. & Melville, J. UMAP: Uniform Manifold Approximation and Projection. Open Source Softw. Ester, M. Density-based clustering. In Data Clustering (eds Aggarwal, C. C. & Reddy, C. K.) (Chapman and Hall/CRC, 2018). & Doolittle, R. F. A simple method for displaying the hydropathic character of a protein. We thank M. Wilhelm and B. Kuster for access to HLA chemically synthesized HLA peptide libraries24. We also thank D. Braun and Q. Jiang for help in designing ERV and RCC TAA peptide libraries. This work was supported in part by grants P01CA206978 to C.J.W. and N.H., and from the Dr. Miriam and Sheldon G. Adelson Medical Research Foundation to S.A.C. acknowledges support from the Claudia Adams Barr Program for Innovative Cancer Research. was supported by Harvard Medical School's Landry Cancer Biology Research Fellowship and Fujifilm Fellowship Program. and N.H. acknowledge support from the Mark Foundation Endeavor Award. N.H. is the David P. Ryan, MD Endowed Chair in Cancer Research MGH, and C.J.W. is the Lavine Family Chair for Preventative Cancer Therapies at the Dana-Farber Cancer Institute (DFCI), is a member of, and her work is supported in part by, the Parker Institute of Cancer Immunotherapy at DFCI. is supported by R01CA229261. is supported by U24CA271037. is supported by U54CA272688, R01HL157174, R01CA285308, R01CA279391 and R01NS140967. This project was supported in part by the Emerson Collective. These authors contributed equally: Kasidet Manakongtreecheep, Claudia Ctortecka. Broad Institute of MIT and Harvard, Cambridge, MA, USA Kasidet Manakongtreecheep, Claudia Ctortecka, Luis O. Correa-Medero, Timothy Zhu, Isabelle Lippincott, Griffin M. Lawrence, Gabrielle M. Hernandez, Emma C. Duggan, Marta A. Wilbrink, Eva K. Verzani, Derin B. Keskin, Patrick A. Ott, Karl R. Clauser, Matthew Bakalar, Siranush Sarkizova, Nir Hacohen, Steven A. Carr, Jennifer G. Abelin & Catherine J. Wu Harvard Medical School, Boston, MA, USA Kasidet Manakongtreecheep, Alexander B. Afeyan, Derin B. Keskin, Patrick A. Ott, Nir Hacohen & Catherine J. Wu Dana Farber Cancer Institute, Boston, MA, USA Kasidet Manakongtreecheep, Timothy Zhu, Alexis Howard, Cleo Forman, Alexander B. Afeyan, Jiaxun Li, Giacomo Oliveira, Derin B. Keskin, Patrick A. Ott, Jennifer G. Abelin & Catherine J. Wu Department of Pathology, University of Michigan, Ann Arbor, MI, USA Department of Computer Science, Metropolitan College, Boston University, Boston, MA, USA Technical University of Denmark, Lyngby, Denmark Massachusetts General Hospital, Krantz Family Center for Cancer Research, Boston, MA, USA Nir Hacohen Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and N.H. conceptualized the project. and S.S. developed the methodology. K.M., C.C., J.G.A., K.R.C., A.I.N., L.O.C.-M. and C.F., participated in the analysis. participated in the sample preparation. participated in the investigation of the results. participated in visualization of the figures. The original draft was written by K.M., C.C., J.G.A. All authors participated in writing, review and editing. The work was supervised by C.J.W., J.G.A., S.A.C. Correspondence to Nir Hacohen, Steven A. Carr, Jennifer G. Abelin or Catherine J. Wu. has received research funding from and/or has advised Agenus, Arcturus, Amgen, Armo BioSciences, Array, AstraZeneca/MedImmune, Bristol-Meyers Squibb, Celldex, CytomX, Evaxion, Immunetune, Imunon, LGChem, Merck, Neon Therapeutics (now BioNTechUS), Novartis, Pharmajet, Phio, Pfizer, Oncorus, Roche/Genentech, Servier and Xencor. is a paid consultant for Moderna. is a member of the scientific advisory boards of Kymera, PTM BioLabs, Seer, PrognomIQ and Mobilion. is an equity holder of BioNtech, is a member of the scientific advisory board of Adventris, Aethon Therapeutics, Nature's Toolbox and Repertoire and receives research funding from Pharmacyclics. is a scientific advisor for Immunitrack, a wholly owned subsidiary of Eli Lilly and Company and Breakbio. owns equity in Affimed, Agenus, Armata Pharmaceuticals, Beam Therapeutics, Breakbio, BioMarin Pharmaceutical, Celldex Therapeutics, Editas Medicine, Immunitybio, Lexicon Pharmaceuticals, Moderna, Prime Medicine, Sana Biotechnology and Summit Therapeutics. N.H. holds equity in and advises Danger Bio/Related Sciences, Repertoire Immune Medicines and CytoReason, owns equity and has licensed patents to BioNtech and receives research funding from Bristol Myers Squibb, Moderna, ResolveM/JJDC, Takeda and Calico Life Sciences. is the founder of Fragmatics, serves on the scientific advisory boards of Protai Bio, Infinitopes and Mobilion Systems and has financial interest because of the licensing of MSFragger to commercial entities. is a consultant for Bicycle Therapeutics. The other authors declare no competing interests. Nature Biotechnology thanks Stefan Tenzer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Workflow of Golden Gate cloning. (b) Detected peptides from each Pepyrus peptide library's MS injections, based on three biological replicates, with one to three MS injections done per replicate. (c) Detected peptides for each MS injections of the matched synthetic peptide pool, based on 3 replicates with varying library concentration and two MS injections done per replicate. (d) Log10 MS1 intensity ranking of peptides from A375 cell line HLA-IP (1×107 cells) (Left/red) and from 10 K Pepyrus library-generated peptides (right/blue) in DDA. The density distribution of HLA-I peptide intensities is overlaid on the right. Source data (a) Data filtering for Pepyrus libraries and Pepyrus-enabled DIA datasets. (b) Unique Pepyrus peptide elution times are binned per minute across the 100 min active gradient, comparing the number of peptides eluting per minute or (b) summarized to 10 min intervals, directly comparing the unique peptide counts per unit time across each of the seven Pepyrus complexity libraries (100-20 K). Pepyrus DDA and Pepyrus DIA include sample preparation specific modifications and truncations. Source data (a) Unique peptide count and (b) log10 intensity of Pepyrus library peptides across all complexities. Colors indicate the occurrence of a N-term K or RP or an oxidized methionine. Peptides are split by their charge state and respective peptide counts are displayed on top. The box represents the IQR, the line marks the median, and the whiskers extend to the most extreme data points within 1.5×IQR. n = 2 technical per 3 biological replicates for all Pepyrus library complexities (total n = 42) (c) Peptide Logo plot showing motifs of detected and missed peptides in each NMDS cluster. NMDS clustering is performed on the 10 K Pepyrus library. Source data Peptide logo plot showing motifs of detected and missed peptides in each Pepyrus library and the synthetic peptide pool20, broken into different peptide amino acid lengths. N represents unique peptides contributing to the respective motif. Source data (a) MiSeq read count of the peptide-encoding oligo sequences, ranked from peptide with the highest read count to peptide with the zero read count (top), or (b) as frequency distribution (bottom) of the peptide-encoding oligo sequences in the 10 K (left) and 20 K (right) Pepyrus plasmid libraries. The red dotted lines and red bars denote peptide-encoding oligos with zero read count. (c) Representation of codons in Pepyrus peptides split by LC-MS/MS recovery grouped by individual amino acids. (d) Log10 MS1 intensity of peptides identified in 10 K Pepyrus library and their log(CPM + 1) read count for cloning batches 1 (left) and 2 (right). log(CPM + 1) = 0 denotes peptide sequences that were not identified by MiSeq in the cloning library. (e) Percent DDA Pepyrus peptide recovery for each read counts decile for 10 K Pepyrus library cloning batches 1 (left) and 2 (right). (f) Probability of Pepyrus 10 K peptides ionizing using electrospray ionization (denoted here as ‘flyer') predicted using pfly and categorized by their DDA detection. Source data (a) top: percent recovery of peptides by their first position amino acids (P1' site for Ulp1 protease cleavage), from 10 K (left) and 20 K (right) Pepyrus library; bottom: total count of detected peptides (dark blue) by their first position amino acids, from 10 K (left) and 20 K (right) Pepyrus library. (b) Percent recovery of peptides by their second position amino acids, from 10 K (left) and 20 K (right) Pepyrus library. Grey denotes the number of undetected peptides that were encoded in the library. (c) Percent detection of P1' glycine (right) and glycine-truncated peptides among the P1' glycine (left) in Pt-C NeoAg nuORF library. (d) Kernel density distribution of hydrophobicity scores of peptides detected (blue) and missed (orange) in the synthetic-matched Pepyrus library (top), synthetic peptide pool (middle), and the 10 K Pepyrus library (bottom), using Kyte-Doolittle hydrophobicity scale96. (e) Kernel density distribution of isoelectric points of peptides detected (blue) and missed (orange) in the synthetic-matched Pepyrus library (top), synthetic peptide pool (middle), and the 10 K Pepyrus library (bottom)3. (f) Subclustering and two-dimensional projection (left) of peptide sequences in cluster 0 of the 10 K Pepyrus peptide library shown in Fig. 2e, highlighting three submotifs with the lowest rate of detection on MS (bottom) and counts (right) of MS-detected and missed peptides. (g) Subclustering and two-dimensional projection (left) of peptide sequences in cluster 1 of the 10 K Pepyrus peptide library shown in Fig. 2e, highlighting three submotifs with the lowest rate of detection on MS (bottom) and counts (right) of MS-detected and missed peptides. Source data (a) Upset plot of all unique HLA-I peptide sequences from 1e7 A375 cell input acquired with standard HLA-I DDA or HLA-DIA method iterations shown in Fig. (b) HLAthena predictions for HLA-I peptides identified with standard DDA or HLA-DIA method iterations. Colors indicate rank cutoffs < 0.5 strong binder; 0.5-1 binder; 1-2 weak binder; >2 unknown binding. For individual datapoints please refer to Fig. Source data (a) Percent peptide recovery per single injection of Pepyrus libraries at increasing complexities (that is 100-20,000 peptides per library) colored by injection replicate and separated by DDA or DIA. (b) Unique peptide counts of Pepyrus complexity DIA searches using the experimental-specific Pepyrus-DDA spectral libraries generated with MSFragger (Pepyrus; purple), in silico predicted spectral library using APD (APD; pastel) or Prosit (Prosit; pink) with instrument specific calibrations according to the developers recommendations. Bar height indicates median and error bars the MAD across 3 biological replicates for all Pepyrus library complexities (total n = 42) (c) Spike-in HLA-DIA analyzed library free directDIATM within Spectronaut or (d) diaTracer-based workflow in FragPipe. (e) Log10 peptide spike-in abundances across the titration curve illustrated in Fig. 3d acquired in DIA or (f) DDA analyzed within FragPipe. Peptide abundances are normalized to 1 fmol level, every point represents one peptide. Horizontal lines indicate expected ratios relative to the 1 fmol spike-in level. (a) PEP score and (b) S/N distribution of target (forward - green), decoy (red) and alternative decoy (rank2 - blue) peptides. (c) Detailed peptide elution of forward (blue) versus decoy peptides (red) with their (d) respective delta. In c, the box represents the IQR, the line marks the median, and the whiskers extend to the most extreme data points within 1.5×IQR. Duplicate injections from biological triplicates are shown. (e) Delta IM of forward versus their respective decoy peptides. (f) Unique peptide identifications colored according to their peptide status (Peptide target - red; Peptide decoy - green; alternative peptide decoy - blue) across the spike-in titration. Identifications are separated by standard spectral library search within Spectronaut using the FragPipe generated spectral library pre- (left) and post-filtering using experimentally generated Pepyrus peptide libraries (right). (a) Allele assignment via HLAthena predictions (rank cutoff <2) to TAA peptides displayed in Fig. 4d separated by cell input and previously identified or novel DIA peptides. (b) Schematic overview of a low mutation burden RCC participant selected for this study and acquired with HLA-I DDA or DIA methods at 1e6, 5e6 and 1e7 cell input. Data analysis overview of Pepyrus libraries based on WES, previously published MS and RNAseq data4, and recent ERV evaluation for RCC in immunotherapy5 and prioritization using HLAthena-based binding prediction (Methods). HLA-DDA data was analyzed with SM as described in the Methods section. (c) Unique HLA peptide sequences and (d) HLAthena based binding predictions for 1×106, 5×106, 1×107 RCC participant-derived cells analyzed with SM for DDA (left) or directDIA for DIA (right). Bar height represents median and error bars MAD from duplicates. (e) Log10 MS2 Intensity of TAA peptides identified from 1×106, 5×106, 1×107 RCC cell input using DDA, Pepyrus-DIA or directDIA with their respective NetMHC-based binding prediction. Colors indicate respective source protein and light background represents strong binders, while darker reflects weak binders. (f) Extracted ion chromatogram for the Pepyrus-enabled DIA identified neoantigen peptide IYVPITGRL at 1×106 cell input. Supplementary Tables 1, 2, 3 and 8. Pepyrus complexity library oligo specifications. Pepyrus complexity library spectra versus synthetic-matched and predicted spectra. Method optimization identifiers and HLAthena filter. Pepyrus complexity library DIA-MS. Spike-In identifiers DDA-MS. Pepyrus library composition and oligo specifications related to cell lines derived from person with melanoma. Pt-A TAA peptide identifiers from DIA-MS and DDA-MS. Pt-A neoantigen and nuORF peptide identifiers. Pepyrus library composition and oligo specifications related to RCC cell line. Pt-102 TAAs, participant-specific neoantigens and ERV peptide identifiers. Statistical source data. Unprocessed western blots. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Manakongtreecheep, K., Ctortecka, C., Correa-Medero, L.O. Sensitive detection of cancer antigens enabled by user-defined peptide libraries. Nat Biotechnol  (2026). Received: 29 January 2025 Published: 09 February 2026 Version of record: 09 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Biotechnol) ISSN 1546-1696 (online) ISSN 1087-0156 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-026-68994-0'>) activates HDAC3 epigenetic axis to maintain intestinal barrier function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 07:20:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. HDAC inhibition shows promise in cancer treatment but pan-HDAC inhibitors cause gastrointestinal issues in 48% of patients. Understanding HDAC activation mechanisms is crucial to treating diverse diseases beyond cancer. Our study reveals that inositol polyphosphate multikinase (IPMK) and inositol hexakisphosphate (InsP6 or phytic acid), enriched in vegan diets, play essential roles in activating the HDAC3 epigenetic axis and maintaining intestinal barrier integrity. IPMK binds to HDAC3 and drives InsP6 synthesis, which selectively activates HDAC3 at a 10 nM concentration by recruiting the DAD domain of its corepressor protein. IPMK deletion diminishes HDAC3 activation, leading to histone hyperacetylation and MMP gene transcription that compromise intestinal barrier integrity. InsP6 treatment is sufficient to rescue these effects. In inflammatory bowel disease, diminished IPMK levels exacerbate intestinal permeability, while oral InsP6 treatment mitigates leaky gut effects by restoring the HDAC3 epigenetic axis, highlighting the clinical significance of the IPMK-HDAC3 pathway and the therapeutic potential of phytic acid. The RNA-sequencing data generated in this study have been deposited in the SRA database under accession code PRJNA1354901. Source data are provided with this paper. Watson, P. J., Fairall, L., Santos, G. M. & Schwabe, J. W. R. Structure of HDAC3 bound to co-repressor and inositol tetraphosphate. & Seto, E. HDACs and HDAC inhibitors in cancer development and therapy. Cold Spring Harb. Subramanian, S., Bates, S. E., Wright, J. J., Espinoza-Delgado, I. & Piekarz, R. L. Clinical toxicities of histone deacetylase inhibitors. Feng, D. et al. A circadian rhythm orchestrated by histone deacetylase 3 controls hepatic lipid metabolism. Alenghat, T. et al. Histone deacetylase 3 coordinates commensal-bacteria-dependent intestinal homeostasis. Watson, P. J. et al. Insights into the activation mechanism of class I HDAC complexes by inositol phosphates. Guha, P. et al. IPMK mediates activation of ULK signaling and transcriptional regulation of autophagy linked to liver inflammation and regeneration. Kim, S., Bhandari, R., Brearley, C. A. The inositol phosphate signalling network in physiology and disease. Millard, C. J. et al. Class I HDACs share a common mechanism of regulation by inositol phosphates. The role of HDAC3 and its inhibitors in regulation of oxidative stress and chronic diseases. Cell Death Discov. Kumar, A. et al. Phytic acid: blessing in disguise, a prime compound required for both plant and human nutrition. O'Sullivan, S., Gilmer, J. F. & Medina, C. Matrix metalloproteinases in inflammatory bowel disease: an update. Reilly, L. et al. Loss of PI3k activity of inositol polyphosphate multikinase impairs PDK1-mediated AKT activation, cell migration, and intestinal homeostasis. & Yoshikawa, T. Role of matrix metalloproteinases in inflammatory bowel disease. Rumbaugh, G. et al. Pharmacological selectivity within class I histone deacetylases predicts effects on synaptic function and memory rescue. Nguyen, H. C. B., Adlanmerini, M., Hauck, A. K. & Lazar, M. A. Dichotomous engagement of HDAC3 activity governs inflammatory responses. Pavlovic, I. et al. Prometabolites of 5-diphospho-myo-inositol pentakisphosphate. Ning, L., Rui, X., Bo, W. & Qing, G. The critical roles of histone deacetylase 3 in the pathogenesis of solid organ injury. Massimino, L. et al. The inflammatory bowel disease transcriptome and metatranscriptome meta-analysis (IBD TaMMA) framework. Barberio, B. et al. Matrix metalloproteinase 3 predicts therapeutic response in inflammatory bowel disease patients treated with infliximab. Iaia, N. et al. The CaCo-2 cell junction derangement exerted by the single addition of oxysterols commonly detected in foods is markedly quenched when they are in mixture. Horowitz, A., Chanez-Paredes, S. D., Haest, X. & Turner, J. R. Paracellular permeability and tight junction regulation in gut health and disease. Tsai, P. et al. IL-22 upregulates epithelial claudin-2 to drive diarrhea and enteric pathogen clearance. Cell Host Microbe 21, 671–681.e4 (2017). & Turner, J. R. The intestinal epithelial barrier: a therapeutic target?. Grases, F. et al. Absorption and excretion of orally administered inositol hexaphosphate (IP(6) or phytate) in humans. & Shamsuddin, A. M. 3H]inositol hexaphosphate (phytic acid) is rapidly absorbed and metabolized by murine and human malignant cells in vitro. Grases, F. et al. Effects of exogenous inositol hexakisphosphate (InsP(6)) on the levels of InsP(6) and of inositol trisphosphate (InsP(3)) in malignant cells, tissues and biological fluids. Badodi, S. et al. Inositol treatment inhibits medulloblastoma through suppression of epigenetic-driven metabolic adaptation. Wu, S. et al. Microbiota-derived metabolite promotes HDAC3 activity in the gut. Williams, K. L. et al. Enhanced survival and mucosal repair after dextran sodium sulfate-induced colitis in transgenic mice that overexpress growth hormone. Aouameur R., Da Cal S., Bissonnette P., Coady M. J., Lapointe J. Y. SMIT2 mediates all myo-inositol uptake in apical membranes of rat small intestine. & Shamsuddin, A. M. Cancer inhibition by inositol hexaphosphate (IP6) and inositol: From laboratory to clinic. Ge, Y., Zadeh, M. & Mohamadzadeh, M. Dissociation and flow cytometric isolation of murine intestinal epithelial cells for multi-omic profiling. Chetterjee, S. et al. Phytic Acid (InsP6) Activates HDAC3 Epigenetic Axis to Maintain Intestinal Barrier Function. Wilson M. S., Saiardi A. Inositol phosphates purification using titanium dioxide beads. Harmel R. K. et al. Harnessing 13C-labeled myo-inositol to interrogate inositol phosphate messengers by NMR. Yu G., Wang L. G., Han Y. & He Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Liberzon A. et al. Molecular signatures database (MSigDB) 3.0. This work was supported by NIH R16GM154726 and 5P20GM121325 COBRE grant and University of Nevada, Las Vegas start-up funds to Prasun Guha. This work was supported by UK Research and Innovation (UKRI)'s Medical Research Council (MRC) grant MR/T028904/1. The partial publication fee covered by GM103440. We would like to thank NIPM's Genomic Core for assisting with instruments and experiments. The specimen collection and phenotyping were made possible by the Washington University (WU) DDRCC (NIDDK P30 DK052574). Parakkal Deepak was supported by the Helmsley Charitable Trust, the IBD Plexus of the Crohn's and Colitis Foundation, and the Leo & Carean Goss Crohn's Disease Research Fund. The publication fees for this article were supported by the UNLV University Libraries Open Article Fund. We would like to thank Marc C Jhonson, from University of Missouri-School of Medicine for generously providing IPPK Knock out cells. We would like to thank Mark Donowitz from Johns Hopkins Medicine, Mark Donowitz from Johns Hopkins Subrata H Mishra from NIST and Brian Hedlund from UNLV for reading and editing the paper and giving suggestions. Nevada Institute of Personalized Medicine, University of Nevada, Las Vegas, NV, USA Sujan Chatterjee, Zachary Sin, Nguyen Tran, Loretta Vierra, Tam Tran, George Koshkaryan, Saharat Jolak Ragsac, Qian Liu & Prasun Guha School of Life Sciences, University of Nevada, Las Vegas, NV, USA Loretta Vierra, Qian Liu, Richard Van, Mira V. Han & Prasun Guha Institute of Organic Chemistry, Faculty of Chemistry and Pharmacy, University of Freiburg, Freiburg, Germany Anuj Shukla, Kevin Ritter & Henning J. Jessen Medical Research Council Laboratory for Molecular Cell Biology, University College London, London, UK Xue Bessie Su & Adolfo Saiardi Department of Mechanical Engineering, University of Nevada, Las Vegas, Las Vegas, NV, USA Division of Gastroenterology, Washington University School of Medicine in St. Louis, St. Louis, MO, USA Katherine Huang, Kayci Huff-Hardy, Richard Rood, Anas Gremida, Martin Gregory, Chien-Huan Chen & Parakkal Deepak Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the study. and S.C. designed the experiments. S.C. performed most experiments. validated major biochemical experiments. S.C. performed wet-lab experiments related to NGS studies. performed most of the NGS data analysis. generated the cell-permeable InsP6. and A.S. analyzed intercellular inositol content. provided and analyzed Human patient samples. performed NGS data validation. maintained animal colonies and isolated animal samples for genotyping. performed mathematical analysis of transwell assay. and S.C. wrote the manuscript takes responsibility of all wet lab data. Correspondence to Prasun Guha. Parakkal Deepak, MBBS MS has received research support under a sponsored research agreement unrelated to the data in the study and/or consulting from Johnson and Johnson, Pfizer, AbbVie, Arena Pharmaceuticals, Bristol Myers Squibb, CorEvitas LLC, Sandoz, Takeda Pharmaceuticals, Direct Biologics, Prometheus Biosciences, Lilly, Teva Pharmaceuticals, Merck, ExeGI Pharmaceuticals, AGMB, Landos Pharmaceuticals, Tr1X, and Boehringer Ingelheim. The remaining authors declare no conflict of interest. Nature Communications thanks Chunfang Gu and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Chatterjee, S., Sin, Z., Tran, N. et al. Phytic acid (InsP6) activates HDAC3 epigenetic axis to maintain intestinal barrier function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41598-026-38842-8'>Eco-friendly synthesis of star-shaped Zn nanoparticles using Beta vulgaris peel extract and evaluation of their antibacterial, photocatalytic, and cytotoxic activities</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-09 02:05:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Scientific Reports , Article number:  (2026) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. In this study, for the first time, Zn nanoparticles were created using the root peel of Beta vulgaris by a green method. Synthesized Zn nanoparticles have been characterized via UV–Vis spectroscopy, Zeta potential analysis, XRD, FTIR, TEM, and FESEM-EDX techniques. The synthesis of zinc-based nanoparticles was confirmed by spectroscopic techniques. UV-Vis analysis presented a broad absorption peak in the UV region, centered around 300 nm. Furthermore, a zeta potential of − 21.6 mV characterized the particles' negative surface charge and suggested moderate colloidal stability. The XRD pattern of Zn nanoparticles revealed crystalline structures, and FESEM and TEM images of Zn nanoparticles displayed a star-shaped morphology. The photocatalytic activity of NPs for the color degradation of Rhodamine B (RhB) and Methylene Blue (MB) dyes has been investigated under UV light, and the results show 97% and 98% degradation, respectively. The antibacterial activity of the Zn nanoparticles was evaluated at a concentration of 1 mg/mL against the gram-positive bacterium Staphylococcus aureus (ATCC 1112 and clinical isolates), Enterococcus faecalis (ATCC and clinical isolates), and gram-negative bacterium Escherichia coli (ATCC and clinical isolates) and Pseudomonas aeruginosa (PAO1 and S7). The cytotoxic effects of Zn nanoparticles were assessed over a concentration range of 7 to 500 µg/mL on both normal L929 fibroblast cells and MCF-7 breast cancer cells. After 24 h of incubation, the IC₅₀ value for the MCF-7 cell line was determined to be 284 µg/mL, while no significant cytotoxicity was observed on L929 cells. To our understanding, this is the initial report synthesizing star-shaped Zn nanoparticles utilizing B. vulgaris root peel as an eco-friendly reducing and stabilizing agent, highlighting their potential for environmental and biomedical applications. The datasets generated and/or analysed during the current study are not publicly available due to the sensitive nature of the data and to protect participant confidentiality, but are available from the corresponding author on reasonable request.The data that support the findings of this study are available from the corresponding author, [Majid Darroudi, Email: majiddarroudi@gmail.com], upon reasonable request. Ajaz, M., Rasool, W. & Mahmood, A. Comprehensive review of nanotechnology: innovations and multidisciplinary applications: comprehensive review of nanotechnology. Poonam, D., Honey, S. & Indu, J. Green and facile approach for the synthesis of ZnO/MgO nanocomposite using Azadirachta indica leaf extract: characterization and photocatalytic activity. Google Scholar Dwivedi, P., Jatrana, I., Satiya, H. & Lokhande, R. S. Understanding the synergistic effect of bimetallic Cu2O-ZnO nanocomposites for the antimicrobial activity and facile synthesis of phenyl xanthenedione: A comparative study. Google Scholar Prakash, M., Kavitha, H. P., Abinaya, S., Vennila, J. P. & Lohita, D. Green synthesis of bismuth based nanoparticles and its applications - A review. Sustainable Chem. Google Scholar Paramesh, C. C. et al. Silver nanoparticles synthesized using saponin extract of Simarouba glauca oil seed meal as effective, recoverable and reusable catalyst for reduction of organic dyes. Results Surf. Google Scholar Elattar, K. M. et al. Phytogenic synthesis and characterization of silver Metallic/Bimetallic nanoparticles using beta vulgaris L. Extract and assessments of their potential biological activities. Google Scholar Mudike, R. et al. Copper zinc Tin sulfide and multi-walled carbon nanotubes nanocomposite for visible-light-driven photocatalytic applications. Google Scholar Gengatharan, A., Dykes, G. A. & Choo, W. S. Betalains: natural plant pigments with potential application in functional foods. LWT-Food Sci. Google Scholar Wybraniec, S. Formation of decarboxylated betacyanins in heated purified betacyanin fractions from red beet root (Beta vulgaris L.) monitored by LC – MS/MS. Food Chem. Google Scholar Chaari, M. et al. Potentials of beetroot (Beta vulgaris L.) Peel extract for quality enhancement of refrigerated beef meat. Google Scholar Tzanova, M. T. et al. Green solvents for extraction of natural food colorants from plants: selectivity and stability issues. Google Scholar Adeyemi, J. O., Oriola, A. O., Onwudiwe, D. C. & Oyedeji, A. O. Plant extracts mediated metal-based nanoparticles: synthesis and biological applications. Google Scholar Elattar, K. M. et al. Phytogenic synthesis and characterization of silver metallic/bimetallic nanoparticles using beta vulgaris L. extract and assessments of their potential biological activities. Google Scholar Dwivedi, P. et al. Plant-mediated synthesis, characterization, and evaluation of a copper oxide/silicon dioxide nanocomposite by an antimicrobial study. Google Scholar Costa, M. I., Sarmento-Ribeiro, A. & Gonçalves, A. C. Zinc: from biological functions to therapeutic potential. Google Scholar Maret, W. Zinc biochemistry: from a single zinc enzyme to a key element of life. Google Scholar Wessels, I., Maywald, M. & Rink, L. Zinc as a gatekeeper of immune function. Google Scholar Haase, H. & Rink, L. Zinc signals and immune function. Google Scholar Kim, B. & Lee, W. W. Regulatory role of zinc in immune cell signaling. Google Scholar Prasad, A. S. Zinc in human health: effect of zinc on immune cells. Google Scholar Lebaka, V. R., Ravi, P., Reddy, M. C., Thummala, C. & Mandal, T. K. Zinc oxide nanoparticles in modern science and technology: multifunctional roles in Healthcare, environmental Remediation, and industry. Nanomaterials (Basel Switzerland). Google Scholar Jayakodi, S. et al. Controlling pore size during the synthesis of hydroxyapatite nanoparticles using CTAB by the sol–gel hydrothermal method and their biological activities. Google Scholar AlSalhi, M. S. et al. Therapeutic potential assessment of green synthesized zinc oxide nanoparticles derived from fennel seeds extract. Google Scholar Riss, T. L. et al. Cell viability assays. In (eds Markossian, S., Grossman, A., Baskir, H., Arkin, M., Auld, D. & Austin, C.) Assay Guidance Manual. Bethesda (MD): Eli Lilly & Company and the National Center for Advancing Translational Sciences. Google Scholar Mehdipour, A. et al. Green synthesis of zinc nanoparticles by hydroalcoholic extract of lavender (Lavandula Stoechas L.), characterization, and cytotoxic effects on human breast and colon cancer. Google Scholar Salari, Z. et al. Microwave-assisted biosynthesis of zinc nanoparticles and their cytotoxic and antioxidant activity. Google Scholar Singh, J., Kaur, S., Kaur, G., Basu, S. & Rawat, M. Biogenic ZnO nps: A study of blueshift of optical band gap and photocatalytic degradation of reactive yellow 186 dye under direct sunlight. Google Scholar Sharma, S., Singh, A. K., Singh, S. P. & Singh, R. K. Synthesis and characterization of ZnO nanoparticles prepared by green routes: controlling morphologies by maintaining pH. Google Scholar Sharma, S. S., Palaty, S. & John, A. K. Band gap modified zinc oxide nanoparticles: an efficient visible light active catalyst for wastewater treatment. Google Scholar Stroyuk, O. L., Dzhagan, V. M., Shvalagin, V. V. & Kuchmiy, S. Y. Size-Dependent optical properties of colloidal ZnO nanoparticles charged by photoexcitation. Google Scholar Thyr, J., Montero, J., Österlund, L. & Edvinsson, T. Energy alignment of Quantum-Confined ZnO particles with copper oxides for heterojunctions with improved photocatalytic performance. Google Scholar Vinay, S. P., Nagaraju, G., Chandrappa, C. & Chandrasekhar, N. Novel Gomutra (cow urine) mediated synthesis of silver oxide nanoparticles and their enhanced photocatalytic, photoluminescence and antibacterial studies. Science: Adv. Google Scholar Bisht, G. & Rayamajhi, S. ZnO nanoparticles: A promising anticancer agent. Google Scholar Dwivedi, P., Satiya, H. & Sharma, S. K. Novel green approach for the synthesis of Co3O4/ZnO nanocomposite, characterization and antimicrobial activity. Today: Proc. Google Scholar Ouf, M. S., Duab, M. E., Abdel-Meguid, D. I., El-Sharouny, E. E. & Soliman, N. A. Biogenic zinc nanoparticles: green approach to synthesis, characterization, and antimicrobial applications. Google Scholar Shalini, K. et al. Hydrothermal route alignment of Ag/rGO@ Curcumin nanocomposite: photocatalyst degrading action and antibacterial effectiveness. Journal Indian Chem. Degen, T., Sadki, M., Bron, E., König, U. & Nénert, G. The highscore suite. Google Scholar Al-Kordy, H. M. H., Sabry, S. A. & Mabrouk, M. E. M. Statistical optimization of experimental parameters for extracellular synthesis of zinc oxide nanoparticles by a novel haloalaliphilic Alkalibacillus sp.W7. Google Scholar Dwivedi, P. et al. Eco-Friendly CuO/Fe3O4 nanocomposite synthesis, characterization, and cytotoxicity study. Jayakodi, S. & Shanmugam, V. K. Statistical optimization of copper oxide nanoparticles using response surface methodology and Box–Behnken design towards in vitro and in vivo toxicity assessment. Biointerface Res. Google Scholar El Faroudi, L. et al. Facile and sustainable synthesis of ZnO nanoparticles: effect of gelling agents on ZnO shapes and their photocatalytic performance. ACS Omega. Google Scholar M, A. M., Ashwin, D. U., Yardily, B. M., Dennison, M. S. & A., & Microwave-assisted green synthesized ZnO nanoparticles: an experimental and computational investigation. Google Scholar Jatrana, I., Satiya, H. & Dwivedi Novel green synthesis of ZnO/SiO2 nanocomposite: characterization and biocidal activity. Today: Proc. Google Scholar Mohammed, N. & Al Khazraji, A. Synthesis and study of zinc oxide nanoparticles and their nanocomposites. Google Scholar Pasieczna-Patkowska, S., Cichy, M. & Flieger, J. Application of fourier transform infrared (FTIR) spectroscopy in characterization of green synthesized nanoparticles. Molecules 30 (3), 684. https://doi.org/10.3390/molecules30030684 (2025). Google Scholar Gomez-Zavaglia, A., Cassani, L., Hebert, E. M. & Gerbino, E. Green synthesis, characterization and applications of iron and zinc nanoparticles by probiotics. Google Scholar Al-Harbi, H. F. et al. Green synthesis of zinc oxide nanoparticles: physicochemical Characterization, photocatalytic Performance, and evaluation of their impact on seed germination parameters in crops. Catalysts 15 (10), 924. https://doi.org/10.3390/catal15100924 (2025). Google Scholar Ogunyemi, S. O. et al. Green synthesis of zinc oxide nanoparticles using different plant extracts and their antibacterial activity against Xanthomonas oryzae pv. Google Scholar Mustapha, S. et al. Comparative study of crystallite size using Williamson-Hall and Debye-Scherrer plots for ZnO nanoparticles. Google Scholar Shaik, K., Cole, S. & K, V., & Comparative study of crystallite size from XRD and TEM results for pure and V2O5 doped CdO-FePO4 composite nanopowders. Google Scholar Vinay, S. P., Udayabhanu, Nagaraju, G., Chandrappa, C. & Chandrasekhar, N. A novel, green, rapid, nonchemical route hydrothermal assisted biosynthesis of ag nanomaterial by Blushwood berry extract and evaluation of its diverse applications. Google Scholar Paramesh, C. C. et al. Silver nanoparticles anchored TiO2 nanotubes prepared using saponin extract as heterogeneous and recyclable catalysts for reduction of dyes. Google Scholar Porous covalent organic frameworks in photocatalytic ROS-mediated processes. Google Scholar Hassaan, M. A. et al. Principles of photocatalysts and their different applications: A review. Google Scholar Dwivedi, P. et al. Photoremediation of methylene blue by biosynthesized ZnO/Fe3O4 nanocomposites using callistemon viminalis leaves aqueous extract: a comparative study. Google Scholar Supin, K. K., Vasundhara, M. & PM, P. N., & Enhanced photocatalytic activity in ZnO nanoparticles developed using novel lepidagathis ananthapuramensis leaf extract. Google Scholar Kahsay, M. H., Tadesse, A., RamaDevi, D., Belachew, N. & Basavaiah, K. Green synthesis of zinc oxide nanostructures and investigation of their photocatalytic and bactericidal applications. Google Scholar Kumar, M. S., Suprajaa, N. & David, E. Photocatalytic activity of methylene blue using zinc nanoparticles synthesized from Gymnema sylvestre and antimicrobial assay. Google Scholar Ghasemian Yadegari, J. et al. Antileishmanial, cellular mechanisms, and cytotoxic effects of green synthesized zinc nanoparticles alone and in combined with glucantime against Leishmania major infection. Google Scholar Zhang, C. et al. Biosynthesis of zinc nanoparticles using allium saralicum RM Fritsch leaf extract; chemical characterization and analysis of their cytotoxicity, antioxidant, antibacterial, antifungal, and cutaneous wound healing properties. Google Scholar Rajeshkumar, S., Jayakodi, S., Tharani, M., Alharbi, N. S. & Thiruvengadam, M. Antimicrobial activity of probiotic bacteria-mediated cadmium oxide nanoparticles against fish pathogens. Google Scholar Holmes, A. H. et al. Understanding the mechanisms and drivers of antimicrobial resistance. Lancet (London England). Google Scholar Jayakodi, S. et al. Preparation of novel nanoformulation to enhance efficacy in the treatment of cardiovascular disease. Halligudra, G., K, V., Rangappa, M. N. R., Shivaramu, P. D. & D., & Antibacterial and antiproliferative effect of ZnO nanoparticles prepared using Origanum Marjorana plant and garcinia indica fruit extracts. Sirelkhatim, A. et al. Review on zinc oxide nanoparticles: antibacterial activity and toxicity mechanism. Krishnamoorthy, R. et al. Antibacterial mechanisms of zinc oxide nanoparticle against bacterial food pathogens resistant to Beta-Lactam antibiotics. Molecules (Basel Switzerland). Majeed, H., Iftikhar, T., Nadeem, A., Altaf Nazir, M. & M., & Green synthesis of Eucalyptus globulus zinc nanoparticles and its use in antimicrobial insect repellent paint formulation in bulk industrial production. This project was financially supported by the Vice-Chancellor for Research (Grant no 4040328), Mashhad University of Medical Sciences. This study is based on the MS thesis of Ms. F. S. Mousavi Khatat. This project was financially supported by the Vice-Chancellor for Research (Grant no 4040328), Mashhad University of Medical Sciences. Department of Medical Biotechnology & Nanotechnology, Faculty of Medicine, Mashhad University of Medical Sciences, Mashhad, Iran Faeghe Sadat  Mousavi Khatat & Zahra Sabouri Student Research Committee, Mashhad University of Medical Sciences, Mashhad, Iran Zahra Sabouri Nuclear Medicine Research Center, Mashhad University of Medical Sciences, Mashhad, Iran Majid Darroudi Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Faeghe Sadat Mousavi Khatat: Data acquisition, Analysis, Software, Investigation, Methodology, writing original draft. Zahra Sabouri: Data curation, Software, Investigation, Methodology, Writing review & editing. Majid Darroudi: Supervision, Project administration, Funding, Methodology, Writing review & editing. Correspondence to Majid Darroudi. The authors declare no competing interests. Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Mousavi Khatat, F.S., Sabouri, Z. & Darroudi, M. Eco-friendly synthesis of star-shaped Zn nanoparticles using Beta vulgaris peel extract and evaluation of their antibacterial, photocatalytic, and cytotoxic activities. Published: 16 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Scientific Reports © 2026 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            