
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-01</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09325-z'>Architecture, dynamics and biogenesis of GluA3 AMPA glutamate receptors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 16:06:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Please note there may be errors present which affect the content, and all legal disclaimers apply. Assembled from combinations of four core subunits, GluA1-4, and ~20 auxiliary subunits, their molecular diversity tunes information transfer and storage in a brain circuit-specific manner. Here, we present cryo-EM structures of the CP GluA3 homomer, which substantially diverge from other AMPARs. The GluA3 extracellular domain tiers (NTD and LBD) are closely coupled throughout gating states, creating previously unseen interfaces that impact signalling and harbour human disease mutations. Central to this architecture is a stacking interaction between two arginine residues (Arg163) in the NTD dimer interface, trapping a unique NTD dimer conformation that enables close contacts with the LBD. We further show that a mammalian-specific GluA3 trafficking checkpoint determines conformational stability of the LBD tier. Hence, specific design features define communication and biogenesis of GluA3, offering a framework to interrogate this disease-prone glutamate receptor. Neurobiology Division, Medical Research Council (MRC) Laboratory of Molecular Biology, Cambridge, UK Aditya Pokharna, Imogen Stockwell, Josip Ivica, Bishal Singh, Ondrej Cais & Ingo H. Greger Structural Studies Division, Medical Research Council (MRC) Laboratory of Molecular Biology, Cambridge, UK Supplementary Figure 1: Uncropped image of the gel presented in Extended Data Fig. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250701020649.htm'>Synthetic storm: What's really in your teen's vape — and why scientists are alarmed</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 15:01:51
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Also, adolescents are increasingly unsure about the substances they vaped in their e-cigarettes. A new study appearing in the American Journal of Preventive Medicine, published by Elsevier, sheds light on this alarming trend and contributes to informing evidence-based public health policies and harm reduction strategies aimed at protecting youth from the long-term consequences of vaping. Vaping of substances other than nicotine has become increasingly popular among adolescents in the United States. The current study analyzed national trends of adolescent cannabis vaping from the National Youth Tobacco Survey for 2021, 2022, and 2023, comprising a total of 69,899 US middle and high school students (aged 11 to 18 years). THC vaping peaked in 2022 while the use of SCs continued to increase. Adolescents increasingly expressed uncertainty about the substances they were vaping; for example, uncertain respondents answering 'don't know' if they have vaped SCs tripled across the years. Our results also showed that females had a higher prevalence of THC, CBD, and SCs vaping compared to males." This trend is particularly alarming given that these substances are often accessed through unregulated, illicit markets, where there are no safety standards or quality controls. The growing popularity of SCs among youth raises serious concerns about potential health risks and highlights the urgent need for targeted public health interventions and regulatory oversight. Chan, PhD, National Centre for Youth Substance Use Research, Faculty of Health and Behavioural Sciences, The University of Queensland, adds, "We still know very little about the long-term health effects of cannabis vaping, which makes it even more important to understand what's in your vape." This study is one of the first to track national adolescent vaping prevalence of THC, CBD, and SCs independently, given that most recent studies categorized various cannabinoids vaping under the umbrella term "cannabis vaping," despite their vastly different psychological and health effects. Mr. Chung concludes, "Experimentation with substance use among teenagers is often driven by peer influence, curiosity, and a desire for social acceptance. I hope this study will raise awareness of youth cannabis vaping and divert public health resources into better psychoeducation on adolescent vaping, as well as tailored harm reduction interventions to protect our young generations." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250701020645.htm'>Is that really ADHD? Why flawed trials may be misleading millions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 14:14:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Originally developed for children, the diagnosis of ADHD is often difficult to make in adults. This is partly because the diagnostic criteria are based on behaviour in children. "The rising number of adults diagnosed with ADHD raises important questions about diagnostic validity -- especially since many were never identified in childhood and are now seeking help, sometimes prompted by ADHD content on social media. That made us curious: how have randomized controlled trials on ADHD dealt with this diagnostic challenge?" Moreover, ADHD shares its symptoms with a number of other mental disorders such as depression, schizophrenia, and bipolar disorder, making it crucial to exclude these disorders when diagnosing ADHD. This requires a thorough diagnostic assessment by an experienced psychologist or psychiatrist. A new study from the University of Copenhagen and the University of Sao Paulo in Brazil now shows that even psychiatric research into ADHD often neglects this fundamental work. "We have examined how 292 of the most credible studies in evidence-based medicine -- the so-called randomised controlled trials -- diagnosed their adult subjects," says Professor of Psychiatry and Consultant Psychiatrist Julie Nordgaard, who conducted the study together with Associate Professor and Senior Researcher Mads Gram Henriksen and Dr. Igor Studart. This means that they can't actually know, if their subjects have other mental disorders such as depression or schizophrenia. More than half of the studies included subjects, who have also been diagnosed with other mental disorders, making the diagnosis even more difficult to allocate," Julie Nordgaard explains. According to the researchers, these methodological shortcomings are problematic, because they imply that it is impossible to know which disorders and symptoms the treatment investigated in these trials potentially had an effect on. "This makes the research results from many of these clinical trials difficult to utilise. Yet, the results of randomised controlled trials are considered particularly trustworthy, and they may inform the guidelines we use to treat adult ADHD patients, even though the results from many of these trials should be assessed very carefully," says Mads Gram Henriksen. But diagnostic assessment should always be performed by an experienced professional with the necessary training to ensure that the diagnosis is made correctly, and this should be stated in the studies' method section," explains Mads Gram Henriksen. "In psychiatry, we really need that all diagnoses, not just ADHD, are made with the same uniform criteria and by trained professionals. Otherwise, we cannot rely on the results or compare them across studies," says Julie Nordgaard and concludes: Otherwise, we risk too many people getting a wrong diagnosis and not being able to give them the most effective treatment. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65206364/captain-cook-hms-endeavor-rhode-island/'>Archaeologists Have Confirmed the Shipwreck of Captain Cook's Long-Lost Endeavour</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Though controversy erupted when a shipwreck was declared to be the HMS Endeavour back in 2022, a new published report officially confirms it to be. Now, the wreckage once labeled RI 2394 has been confirmed to be that same ship, once made famous by Captain James Cook. Then, in December of 2023, the Australian National Maritime Museum reasserted their declaration, pointing to further evidence gleaned from the wreckage's pump well and bow. Yet, only now, in this new final report for 2025, has the Rhode Island Maritime Archeology Project come to see eye to eye with the Australian National Maritime Museum. In the report's executive summary, written by Kieran Hosty & James Hunter and published by the Australian National Maritime Museum, they state that: The long-sought Endeavour wreckage has officially (or officially-officially-officially) been identified. As the report writes, “His Majesty's Bark (HMB) Endeavour is a significant vessel in Australian maritime history,” though they concede that it is “one that elicits mixed opinions.”: They note that, even for those in Australia deeply aware of Captain Cook, they might not know the other role the Endeavour played, that of a “British troop transport and prison ship caught up in the American War of Independence.” Hence why the then-named Lord Sandwich was sunk near Rhode Island, only just now to be officially, formally found. Their efforts resumed in 2015, and by 2016 they had narrowed down where in the harbor the ship had been scuttled. In investigating RI 2394, however, the team found a number of characteristics that looked promising, including “dimensions of a range of structural timbers” which “compare favourably with measurements taken when Endeavour was surveyed by the Royal Navy in 1768,” and procured timber samples which suggest repairs made with European timber, much in the same fashion as the Endeavour/Sandwich, “which underwent significant repairs in 1776, shortly after being sold out of naval service.” So, after much, much analysis, all parties involved can now confidently conclude that RI 2394 is, in fact, Captain Cook's long lost HMS Endeavour/HMB Endeavour/Lord Sandwich. Now, should anyone else have any reason why this wreckage should not be identified as such, speak now or forever hold your peace. Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. The Physicist That Wants to Build a Time Machine Tons of Coal Ash Holds a Vast Treasure in the U.S.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/see-earths-forests-as-never-before-in-biomass-satellites-first-images/'>See Earth's Forests as Never Before in Biomass Satellite's First Images</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>See Earth's Forests as Never Before in Biomass Satellite's First Images New images from the European Space Agency's Biomass mission show how the satellite uses advanced radar to map flows of carbon through our planet's most precious and remote ecosystems This image features a section of the Bolivian rainforest. Each color represents a distinct ecological feature sensed by the radar, including forests, bodies of water, wetlands, and grasslands. Just two months after its launch, the European Space Agency (ESA) Biomass satellite is already showing us Earth in a whole new light. On June 23 the ESA released the mission's first images, which showcase its unique capability to peer through dense vegetation to study our planet's forests and other ecosystems as never before. Using cutting-edge radar technologies, the satellite is on a five-year mission to survey forests across sprawling swaths of the globe, looking beneath their canopies to accurately assess their total woody biomass—trunks, limbs, stems, and so on. By measuring all that material and how it's changing over time, scientists can better monitor deforestation trends and track the flow of carbon through ecosystems to enhance our understanding of Earth's rapidly changing climate. The satellite is still in its six-month-long commissioning phase, but now we have a taste of what is to come. During this time, Bolivia lost the equivalent of 15 percent of the total forest cover it had in 2000, in large part to encroaching agricultural plots and cattle ranches. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The new pictures represent a significant increase in detail that is primarily owed to Biomass's main instrument: the P-band synthetic aperture radar. P-band radio waves are longer than others more typically used in such radar systems, allowing them to pass through layers of foliage, soil and other materials to create accurate three-dimensional maps of otherwise hidden landscape features. Because of engineering challenges as well as diplomatic ones, this is the first time P-band radar has been used in orbital Earth observations—the P-band radio frequency is typically reserved for American and European missile-detection systems. (In order to ensure it will not interfere with the U.S. Department of Defense's space-object-tracking radar systems, Biomass will not observe North and Central America, Europe, and some Arctic regions.) Pictured above is Indonesia's mountainous Halmahera rainforest, featuring rugged topography shaped by outpourings from the numerous volcanoes that pockmark the region. Though the main goal of the Biomass satellite is to catalog forests, it will also study other environments. Able to map features as far as five meters beneath dry sand, the satellite can reveal ancient geologic details, such as dried-up riverbeds and lakes, now buried below the Sahara's ever-shifting dunes. These data can help researchers understand past ecological changes in such regions—and could even help unearth hidden pockets of precious groundwater in arid, remote deserts. Biomass can also penetrate into ice, pictured below, and collect information about ice-sheet structure and movement that could be used to better understand what controls the slide of glaciers into the sea. ESA Biomass satellite image of a portion of the Antarctic Transantarctic Mountains with the Nimrod Glacier flowing into Ross Ice Shelf. Below, another image features jungle-covered terrain in Gabon and prominently displays the Ivindo River, which nurtures the surrounding rainforest. Biomass captured this image over Gabon in Africa. Biomass satellite image of the Amazon rainforest in northern Brazil. This is the very first image that Biomass returned. Innovative satellites like Biomass could greatly increase our knowledge of these overlooked places and similar ones around the world, helping researchers better understand—and protect—our changing planet and its myriad beautiful ecosystems. Emma R. Hasson is a PhD candidate in mathematics at the City University of New York Graduate Center with expertise in math education and communication. Hasson is also a 2025 AAAS Mass Media Fellow at Scientific American. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a65225588/vera-rubin-images/'>A Colossal Telescope in the Desert Just Captured Galaxies We've Never Seen Before</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>On August 1, 2014, workers began construction of the Large Synoptic Survey Telescope (LSST) perched atop Cerro Pachón in Chile's Atacama Desert. Its main mission will be the Legacy Survey of Space and Time (confusingly also abbreviated LSST), which will ceaselessly scan the night sky continuously for 10 years, noting every visible change along the way. Vera Rubin is particularly well-suited for this task—with a 28-foot-wide primary mirror, and 11-foot-wide secondary mirror, and the largest digital camera ever constructed, this terrestrial telescope can move its 300-ton bulk one full rotation in a half a minute at full speed, according to The New York Times. The amount of data Rubin will collect in the next year alone will be more than all other optical observatories combined. A combination of 678 different images, the photo above shows that brilliant Trifid nebula (top right) with the Lagoon nebula (bottom left), which is located more than 4,000 light-years from Earth. The lead image of this article (an in-depth look at the Virgo cluster) is only 2 percent of a full Rubin image, which would actually require 400 ultra high-definition TVs to display. “Rubin is truly transformative, thanks to its unique combination of sensitivity, vast sky area coverage and exceptional image quality.” It's a big moment for ground-based astronomy, which has often been eclipsed by groundbreaking discoveries made by space-based telescopes like Hubble and JWST. While it's true that these floating telescopes offer a ton of benefits—chief among them being that they circumvent atmospheric interference and the increasing annoyance of satellite disruption (think Starlink)—the benefits of ground-based observatories far outweigh these visual costs. For one, these telescopes can be easily maintained and—crucially—upgraded with the latest and greatest technologies because... you know... they're on the ground. That means it can simply see way more sky than its space-based cousins. Rubin will also be able to produce incredibly high-definition photographs in just a few days, rather than the few years needed by other telescopes. At the end of its 10-year LSST mission, Rubin will provide invaluable data for investigating dark energy and dark matter, as well as information regarding some 20 billion galaxies. Especially excitingly, it will capture asteroids as they move through the Solar System. After decades of waiting, it's time for the science to finally commence. There's a Wild New Theory for Our Universe's Start Some Black Holes May Be Portals Through Spacetime What If Aliens Are Talking, We Just Can't Hear?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-025-00897-z'>DunedinPACNI estimates the longitudinal Pace of Aging from a single brain image to track health and disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 11:21:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. To understand how aging affects functional decline and increases disease risk, it is necessary to develop measures of how fast a person is aging. Using data from the Dunedin Study, we introduce an accurate and reliable measure for the rate of longitudinal aging derived from cross-sectional brain magnetic resonance imaging, that is, the Dunedin Pace of Aging Calculated from NeuroImaging (DunedinPACNI). Exporting this measure to the Alzheimer's Disease Neuroimaging Initiative, UK Biobank and BrainLat datasets revealed that faster DunedinPACNI predicted cognitive impairment, accelerated brain atrophy and conversion to diagnosed dementia. Faster DunedinPACNI also predicted physical frailty, poor health, future chronic diseases and mortality in older adults. When compared to brain age gap, DunedinPACNI was similarly or more strongly related to clinical outcomes. DunedinPACNI is a next-generation brain magnetic resonance imaging biomarker that can help researchers explore aging effects on health outcomes and evaluate the effectiveness of antiaging strategies. Aging is the gradual, progressive and correlated decline of multiple organ systems over decades. Longitudinal studies provide evidence for substantial individual variation in the rate of aging; people born in the same year can age slower or faster than their peers1,2,3. Furthermore, aging itself is increasingly regarded as a potentially preventable cause of chronic disease. Accordingly, accurate and reliable measures of how fast a person is aging are needed to effectively study how individual variation in the rate of aging contributes to disease risk and to evaluate interventions intended to slow aging before irreversible decline4,5,6,7,8. Age-sensitive alterations in DNA methylation, referred to as epigenetic clocks, are currently the most widely used measures for estimating individual differences in aging4,9,10. First-generation epigenetic clocks were trained on chronological age11,12, but the more precisely they predicted chronological age, the less well they predicted clinical outcomes13,14. In response, second-generation clocks were trained on measures of health that predict mortality in older people15,16,17. However, these clocks were trained on cross-sectional phenotypes in multiage samples, not on longitudinal observations of the same person as recommended in geroscience5,18. This limitation led to the development of a third-generation longitudinal approach to measuring aging. We previously adopted this longitudinal approach in the Dunedin Study, which has followed a population representative sample of 1,037 people born in the same year (1972–1973) from birth to age 45 (ref. Across two decades (ages = 26, 32, 38 and 45 years), we repeatedly measured 19 biomarkers of cardiovascular, metabolic, renal, immune, dental and pulmonary functioning. By averaging the decline in the trajectories of these biomarkers, we operationalized the theoretical construct of biological aging into a specific measure that we called the Pace of Aging2. We subsequently developed an epigenetic clock that accurately and reliably estimates the Pace of Aging: the Dunedin Pace of Aging Calculated from the Epigenome (DunedinPACE)20. Because DunedinPACE is calculated from a single time point measurement of DNA methylation, it has been rapidly adopted by studies on aging, where it has been associated with signs of accelerated brain aging, morbidity and mortality20,21,22,23,24,25. However, it has not been possible to export DunedinPACE or other epigenetic clocks to studies lacking DNA methylation data. This includes many neuroimaging studies of brain aging and neurodegenerative diseases such as Alzheimer's disease (AD). Current neuroimaging-based approaches to measure aging, akin to first-generation epigenetic clocks, involve training models to predict chronological age from variability in magnetic resonance imaging (MRI) measures of brain structure in large multiage samples26,27,28,29,30. Researchers then typically quantify a brain age gap, which reflects the difference between a participant's predicted and actual chronological age. A positive brain age gap is interpreted as evidence of accelerated brain aging. As with first-generation epigenetic clocks, these age deviation approaches unavoidably mix model error (for example, historical differences in environmental exposures, survivor bias, disease effects, measurement bias) with a person's true rate of biological aging31,32,33. In this study, using a single T1-weighted MRI scan collected at age 45 in the Dunedin Study, we describe the development and validation of a brain MRI measure for the Pace of Aging (Fig. We call this measure Dunedin Pace of Aging Calculated from NeuroImaging (DunedinPACNI). Using data from the Human Connectome Project (HCP), we evaluated the test–retest reliability of DunedinPACNI. Exporting the measure to the Alzheimer's Disease Neuroimaging Initiative (ADNI), UK Biobank (UKB) and Latin American Brain Health Institute (BrainLat) datasets, we conducted a series of preregistered analyses (https://rb.gy/b9x4u6) designed to evaluate the utility of DunedinPACNI for predicting multiple aging-related health outcomes (Fig. To benchmark our findings, we compared the effect sizes for DunedinPACNI to those for brain age gap34. DunedinPACNI is a brain-based measure trained to directly estimate longitudinal aging of non-brain organ systems. Therefore, if DunedinPACNI indeed estimates individual differences in the rate of aging, it would add evidence for close links between brain integrity and whole-body aging and establish neuroimaging as a powerful tool for measuring aging, that is, not just of the brain but of the entire body35. a, Plot of mean scores for all 19 biomarkers comprising the Pace of Aging across four waves of observation at ages 26, 32, 38 and 45 years in the Dunedin Study. Hypothetical individual trajectories are shown for people with relatively slow, average and fast Pace of Aging from ages 26 to 45 years. b, Distribution of Pace of Aging scores in Dunedin Study members at age 45. Warmer colors represent a faster Pace of Aging; cooler colors represent a slower Pace of Aging. c, A single T1-weighted MRI scan collected from 860 Dunedin Study members at age 45 years was used to train an elastic net regression model to predict the Pace of Aging. We call the resulting measure DunedinPACNI. d, Regression weights from the DunedinPACNI model developed in the Dunedin Study were applied to T1-weighted MRI scans collected in the ADNI and UKB datasets to derive DunedinPACNI scores. Those scores were then related to aging-related phenotypes. AL, attachment loss; Apo, apolipoprotein; BMI, body mass index; FEV1, forced expiratory volume in 1 s; eGFR, estimated glomerular filtration rate; HbA1c, glycated hemoglobin; HDL, high-density lipoprotein; hsCRP, high sensitivity C-reactive protein; VO2max, maximal oxygen uptake. We trained an elastic net regression model to predict the longitudinal Pace of Aging measure using T1-weighted MRI scans collected in a subsample of 860 Dunedin Study members when they were 45 years old. This subsample maintains the population representativeness of the full cohort (Extended Data Figs. Specifically, the elastic net regression model used 315 MRI-derived structural measures for each study member, including regional cortical thickness (CT), surface area (SA), gray matter volume (GMV) and gray:white signal intensity ratio (GWR), as well as subcortical gray matter and ventricular volumes36. We performed tenfold cross-validation to identify the optimal tuning parameters37. This optimized model was used to create DunedinPACNI. The in-sample correlation between DunedinPACNI and the longitudinal Pace of Aging was r = 0.60 (Fig. We performed a cross-validation analysis by splitting the sample into training and testing subsets 100 different times. Each time, we used 90% of the sample for training and held out the remaining 10% for testing. Across all 100 different splits, the average correlation between DunedinPACNI and Pace of Aging in the testing sample was r = 0.42. This prediction accuracy is in line with next-generation epigenetic biomarkers of aging15,20,38. For both DunedinPACNI and the longitudinal Pace of Aging, higher scores indicate faster aging. Note that DunedinPACNI is a quantitative variable that indexes relative differences between people without clear units. For this reason, we restricted our interpretation to directional comparisons within studies. Associations between faster DunedinPACNI scores and measures of physical functioning, cognitive functioning and facial aging were similar to those previously observed with Pace of Aging2. In these analyses, we controlled for sex but not age because all Dunedin Study members have the same chronological age. 2b; full results in Supplementary Table 1). This was expected given the high internal correlation between DunedinPACNI and Pace of Aging. Dunedin Study members with faster DunedinPACNI scores had worse balance, slower gait, weaker lower-body and upper-body strength, and poorer coordination; they also reported worse health and more physical limitations; performed more poorly on tests of cognitive functioning; experienced greater childhood-to-adulthood cognitive decline; and looked older. These results indicate that DunedinPACNI accurately estimates the longitudinal Pace of Aging in the Dunedin Study dataset. a, In-sample correlation between Pace of Aging and DunedinPACNI. Warmer colors represent a faster Pace of Aging and cooler colors represent a slower Pace of Aging. b, Comparison of absolute effect sizes for associations between DunedinPACNI and Pace of Aging with physical functioning, cognition and subjective aging measures in 860 members of the Dunedin Study. Effects are presented as standardized β coefficients with the error bars as the 95% CIs. c, Covariance between MRI-derived brain features and Pace of Aging. Of the 315 brain features used in model training, 216 were set equal to zero because of the high correlation between brain measures and to reduce overfitting. The 99 features included in the final model are visualized in Supplementary Fig. Warmer colors represent features that positively covaried with DunedinPACNI scores (that is, a larger value indicates faster aging), while cooler colors represent features that negatively covaried with DunedinPACNI scores (that is, a larger value indicates slower aging). Features that did not contribute to the estimation of DunedinPACNI predictions are shown in gray. CC, corpus callosum; DC, diencephalon; IQ, intelligence quotient; L, left; R, right. The optimized model used to derive DunedinPACNI included 99 regional brain measures. Because of difficulties in interpreting multivariable model coefficients, we used the Haufe transformation to estimate feature importance scores from the covariance between each brain measure and Pace of Aging39. Given that many of the MRI-derived measures are highly correlated, our elastic net model reduced overfitting by setting the weights for many of them to zero (visualized in Supplementary Fig. Faster Pace of Aging covaried with thinner cortex, smaller cortical SA, smaller cortical GMV, lower cortical GWR, smaller subcortical GMV and larger ventricular volumes (Fig. We also observed positive covariance between calcarine CT and GWR, although not calcarine GMV. This is probably due to known aging-related effects on gray and white matter signal intensity that have been demonstrated previously40,41,42,43. These structural features overlap with the MRI signatures of both normal brain aging and neurodegenerative diseases44,45,46,47,48, suggesting that faster DunedinPACNI reflects, at least in part, canonical patterns of brain aging. If DunedinPACNI is to be used as a measure of aging, it must exhibit sufficient measurement reliability when exported to new datasets. We used test–retest MRI data (n = 45) from the HCP49 to estimate the reliability of DunedinPACNI. The test–retest reliability was excellent (intraclass correlation coefficient = 0.94, 95% CI = 0.89 to 0.97; Supplementary Fig. Having established both internal validity and test–retest reliability, we sought to examine whether DunedinPACNI generalizes to new datasets to detect aging-related outcomes. Specifically, we first tested for associations with cognitive impairment in ADNI and cognitive functioning in UKB. We generated DunedinPACNI scores from T1-weighted MRI scans collected in 1,737 ADNI participants (mean age = 74.3 years, s.d. In ADNI, participants with faster DunedinPACNI showed greater impairment on mental status examinations used to screen for dementia, as well as tests of memory, psychomotor speed and executive functioning. They also reported more impairment in cognitively demanding activities of daily living (ADLs), such as maintaining finances or preparing a meal (Fig. Absolute standardized effect sizes across all cognitive measures in ADNI ranged from β = 0.18 to 0.39 (all P < 0.001; full results in Supplementary Table 2). Similarly, UKB participants with faster DunedinPACNI performed more poorly on tests of executive functioning and psychomotor speed (Fig. Absolute standardized effect sizes across all cognitive measures in UKB ranged from β = 0.05 to 0.12 (all P < 0.001; full results in Supplementary Table 3). Associations in UKB were not driven by individuals with early cognitive decline or APOE ε4 homozygotes, who have heightened genetic risk for Alzheimer's disease and show earlier onset of cognitive decline50 (Extended Data Fig. These, and all subsequent analyses presented in this article, control for sex and chronological age. a,b, Cross-sectional associations between DunedinPACNI and cognitive test scores in ADNI (a) and UKB (b). a,b, Effects are presented as standardized β coefficients with the error bars as the 95% CIs. We visualized the absolute effect sizes to aid visual comparison and clarity (see Supplementary Tables 2 and 3 for the raw effect sizes). The exact sample sizes for each test in a and b are reported in Supplementary Tables 2 and 3. c, Group differences in DunedinPACNI scores in 1,737 ADNI participants according to cognitive status at scanning. Data beyond the whiskers are plotted as individual outliers. d, Survival curve of the relative proportion of CN ADNI participants at baseline who remained CN during the follow-up window, grouped according to slow, average and fast baseline DunedinPACNI scores. Note that although the maximum follow-up length is 16 years, we chose to visualize only 9 years of follow-up because of high amounts of censoring after 9 years. A plot with the full 16 years of follow-up and points marking censoring is presented in Extended Data Fig. ADAS-Cog, Alzheimer's Disease Assessment Scale-Cognitive Subscale 13; DSST, Digit Symbol Substitution Task; FAQ, Functional Activities Questionnaire; LogMemory, Logical Memory test; Matrix, Matrix Pattern Completion; MMSE, Mini-Mental State Examination; RAVLT, Rey Auditory Visual Learning Test; Tower, Tower Rearranging; TrailsA, Trail Making Test Part A; TrailsB, Trail Making Test Part B; VM, visual memory; WM, working memory. We next tested if DunedinPACNI differentiates between normal and clinically impaired cognitive functioning in ADNI (Fig. Participants with mild cognitive impairment (MCI) had faster DunedinPACNI compared to cognitively normal (CN) participants (β = 0.27, P < 0.001, 95% CI = 0.18 to 0.35). Participants with dementia had faster DunedinPACNI than both participants with MCI (β = 0.54, P < 0.001, 95% CI = 0.43 to 0.65) and CN participants (β = 0.81, P < 0.001, 95% CI = 0.69 to 0.92). We further tested whether DunedinPACNI predicts future cognitive decline in people without cognitive impairment. Specifically, we analyzed a subsample of 624 ADNI participants who were CN at the time of their first scan (mean age = 72.4 years, s.d. CN participants with faster DunedinPACNI at baseline were more likely to develop MCI or dementia and to do so earlier during the follow-up window (hazard ratio (HR) = 1.49, P = 0.005, 95% CI = 1.12 to 1.97; Fig. 3d), meaning that those in the top 10% had a 61% increased risk of developing MCI or dementia compared to participants with an average DunedinPACNI. We conducted a similar analysis in the 701 participants who were diagnosed with MCI at the time of their first scan (mean age = 72.8 years, s.d. Participants with MCI with faster DunedinPACNI at baseline were more likely to convert to dementia (HR = 1.44, P < 0.001, 95% CI = 1.26 to 1.65). These effect sizes were similar when controlling for the number of APOE ε4 alleles, a well-established genetic risk allele for sporadic, late-onset AD (baseline CN: HR = 1.49, P = 0.005, 95% CI = 1.13 to 1.96; baseline MCI: HR = 1.42, P < 0.001, 95% CI = 1.23 to 1.62). Because only a very small number of UKB participants with MRI data received diagnoses of dementia during the follow-up observation (n = 73), we were underpowered to report parallel results in this dataset. As an estimate of how fast a person is aging, DunedinPACNI should reflect longitudinal trajectories of brain decline33. We tested whether faster baseline DunedinPACNI predicted accelerated hippocampal atrophy, which is an established risk factor for cognitive decline and dementia onset in older adults51. Specifically, we computed longitudinal trajectories of hippocampal atrophy among 1,302 ADNI participants who had MRI data at multiple time points (average number of scans = 4.4, range = 2–13 scans) as well as 4,601 UKB participants who had MRI data at two time points (Fig. Participants with faster baseline DunedinPACNI exhibited accelerated hippocampal atrophy in both ADNI (β = −0.15, P < 0.001, 95% CI = −0.21 to −0.10; Fig. 4b) and UKB (β = −0.09, P < 0.001, 95% CI = −0.12 to −0.05; Fig. This result was consistent while controlling for number of APOE ε4 alleles (Supplementary Table 5). a, Individualized trajectories of hippocampal atrophy in ADNI (left) and UKB (right). Warmer colors represent accelerated atrophy. b, Forest plot of associations between baseline DunedinPACNI scores and accelerated hippocampal atrophy in 1,302 ADNI participants and 4,601 UKB participants. Effects are presented as standardized β coefficients with the error bars as the 95% CIs. As a measure of aging derived from longitudinal assessments of multiple biomarkers, DunedinPACNI should capture instances of declining health across all organ systems, not just the brain. To test this hypothesis, we used UKB to map DunedinPACNI scores onto measures of frailty, subjective overall health, incident aging-related chronic diseases and all-cause mortality. We used the Fried Frailty Index to quantify the degree of vulnerability to common stressors associated with aging-related decline in energy reserves and functioning. When treating index scores as a continuous measure ranging from 0 to 5, with higher scores indicating greater frailty52,53, we found that participants with faster DunedinPACNI were frailer (n = 42,583; β = 0.17, P < 0.001, 95% CI = 0.16 to 0.18). Participants with faster DunedinPACNI also self-reported poorer overall health (n = 42,235; β = −0.17, P < 0.001, 95% CI = −0.18 to −0.16; Fig. 5a), which predicts mortality even independently of objective health measures54. These associations were not driven by individuals with early cognitive decline or high genetic risk for AD (Extended Data Fig. a, Forest plot of absolute associations between DunedinPACNI and frailty (n = 42,583) and self-rated health (n = 42,235) in UKB. Effects are presented as standardized β coefficients with the error bars as the 95% CIs. b, Group differences in DunedinPACNI scores according to the lifetime number of aging-related chronic disease diagnoses, including myocardial infarction, chronic obstructive pulmonary disease, dementia and stroke in 42,583 UKB participants. c, Survival curve of the relative proportion of disease-free UKB participants at the time of MRI who remained disease-free during the follow-up window, grouped according to slow, average and fast baseline DunedinPACNI scores. We excluded participants who had chronic disease before scanning from this analysis. d, Survival curve of the relative proportion of UKB participants who remained alive during the follow-up window grouped according to baseline DunedinPACNI scores. e, Group differences in DunedinPACNI according to education level in 1,734 ADNI participants. f, Group differences in DunedinPACNI according to education level in 38,297 UKB participants. Data beyond the whiskers are plotted as individual outliers. Similar patterns emerged when considering clinical diagnoses of chronic aging-related diseases, including myocardial infarction, chronic obstructive pulmonary disease, dementia and stroke. Participants with a lifetime prevalence of one of these chronic diseases had faster DunedinPACNI compared to those with no diagnoses (β = 0.19, P < 0.001, 95% CI = 0.16 to 0.23). Participants with a lifetime prevalence of two or more chronic diseases had faster DunedinPACNI than those with a single chronic disease (β = 0.25, P < 0.001, 95% CI = 0.12 to 0.38) and those with no chronic disease (β = 0.44, P < 0.001, 95% CI = 0.31 to 0.57; Fig. Extending beyond contemporaneous associations, we assessed whether faster DunedinPACNI at baseline predicted future myocardial infarction, chronic obstructive pulmonary disease, dementia or stroke in UKB participants who were diagnosis-free at the time of scanning (n = 40,753). A total of 827 participants reported a new diagnosis of at least one of these aging-related chronic diseases over a maximum follow-up period of 9.7 years after scanning (that is, baseline). Consistent with the contemporaneous associations, healthy participants with faster DunedinPACNI at baseline were more likely to be later diagnosed with chronic aging-related diseases (HR = 1.14, P < 0.001, 95% CI = 1.05 to 1.23; Fig. 5c), that is, those in the top 10% had an 18% or greater increased risk of developing a chronic disease compared to participants with average DunedinPACNI. These associations were not driven by individuals with early cognitive decline or high genetic risk for AD (Extended Data Fig. Given the increased mortality rates among people with chronic aging-related diseases, we asked if baseline DunedinPACNI scores predicted all-cause mortality. Of the 42,583 UKB participants included in our dataset, 757 died over the follow-up period after their baseline MRI scan. Participants with faster baseline DunedinPACNI scores died earlier (HR = 1.32, P < 0.001, 95% CI = 1.22 to 1.43; Fig. 5d), that is, those in the top 10% were at least 41% more likely to die compared to participants with average DunedinPACNI. These associations were not driven by individuals with early cognitive decline or high genetic risk for AD (Extended Data Fig. Taken together, these findings suggest that DunedinPACNI is useful for gauging general physical health and assessing the risk for future chronic disease and death. People who are less advantaged in their socioeconomic position experience a wide range of chronic diseases and earlier mortality55,56,57; DunedinPACNI should reflect such gradients of health inequities. We used information about educational attainment and income to test this prediction. Faster DunedinPACNI was observed for participants who either had fewer years of formal education (ADNI: β = −0.10, P < 0.001, 95% CI = −0.15 to −0.05]; UK Biobank: β = −0.09, P < 0.001, 95% CI = −0.10 to −0.08) or lower income (UKB: β = −0.06, P < 0.001, 95% CI = −0.07 to −0.05), reflecting the expected socioeconomic health gradient (Fig. These associations were not driven by individuals with early cognitive decline or high genetic risk for AD (Extended Data Fig. Brain-based predictive algorithms often fail when applied to groups that demographically differ from the training sample58. Most cognitive and brain aging studies are collected in high-income countries in North America and Europe; this may limit the generalizability of brain-based models to people in low-income and middle-income countries59,60,61. Latin Americans are underrepresented in biomedical research60,62 and may experience distinct social and environmental influences on brain aging compared to people in high-income countries in North America and Europe60. To test the generalizability of DunedinPACNI in Latin Americans, we calculated DunedinPACNI scores in a sample of 369 adults from Argentina, Chile, Colombia, Mexico and Peru from the BrainLat dataset63. A total of 162 BrainLat participants were diagnosed with AD, 84 were diagnosed with behavioral variant frontotemporal dementia (FTD) and 123 were healthy controls. When controlling for age and sex, BrainLat participants with AD and FTD had faster DunedinPACNI scores compared to healthy controls (AD: β = 0.70, P < 0.001, 95% CI = 0.48 to 0.91; FTD: β = 0.79, P < 0.001, 95% CI = 0.55 to 1.04; Fig. Notably, these effect sizes are comparable to the difference between participants with dementia and CN participants in ADNI (Fig. In addition, 191 BrainLat participants also completed the Montreal Cognitive Assessment (MoCA). In this subset, BrainLat participants with faster DunedinPACNI scores had poorer scores on the MoCA (β = −0.35, P < 0.001, 95% CI = −0.49 to −0.20). Notably, this effect was similar to the association between DunedinPACNI and MoCA scores in ADNI participants (β = −0.32, P < 0.001, 95% CI = −0.38 to −0.25; Fig. a, Group differences in DunedinPACNI scores according to cognitive diagnosis in 369 BrainLat participants. Data beyond the whiskers are plotted as individual outliers. b, Forest plot of standardized mean differences in DunedinPACNI between participants with dementia and CN controls in BrainLat (orange; n = 369) and ADNI (dark green; n = 1,201) while controlling for age and sex. DunedinPACNI was similarly accelerated in dementia in a sample of Latin Americans (BrainLat) and North Americans (ADNI). Effects are presented as standardized β coefficients with the error bars as the 95% CIs. c, Scatter plot of associations between MoCA scores and DunedinPACNI in BrainLat (orange; n = 191) and ADNI (dark green, n = 1,206) participants. DunedinPACNI scores were residualized for age and sex. The linear associations between MoCA scores and DunedinPACNI scores were similar in a sample of Latin Americans (BrainLat) and North Americans (ADNI). Lastly, we compared DunedinPACNI with existing approaches for measuring aging using brain MRI data. Specifically, we compared the effect sizes for DunedinPACNI from all of the aforementioned analyses in ADNI, UKB and BrainLat with brain age gap generated using brainageR34. We selected this algorithm because of its high accuracy and test–retest reliability compared to other brain age gap algorithms64. DunedinPACNI and brain age gap were only modestly correlated (ADNI: r = 0.17, P < 0.001; UKB: r = 0.31, P < 0.001; BrainLat: r = 0.32, P < 0.001; Supplementary Fig. Compared to brain age gap, the effect sizes for DunedinPACNI were similar or larger across measures of cognitive function, cognitive decline, brain atrophy, frailty, disease risk, mortality and socioeconomic health gradients (Fig. Commensurate with the low correlation between these measures, when we included both DunedinPACNI and brain age gap in a single model, each measure explained unique variance in clinical outcomes, with only minor reductions in effect sizes. Moreover, using both DunedinPACNI and brain age gap together in a single model generally increased the prediction of these outcomes (Extended Data Figs. For example, the combined HR of DunedinPACNI and brain age gap predicting mortality risk was 1.50 (95% CI = 1.36 to 1.65), compared to the independent HRs of 1.32 for DunedinPACNI and 1.24 for brain age gap. a, Forest plots of DunedinPACNI and brain age gap absolute effect sizes in ADNI (left) and UKB (right). Effects are presented as standardized β coefficients with the error bars as the 95% CIs. Note that for visualization, the signs of some outcomes were flipped, such that higher scores for all outcomes reflected worse performance or health. Raw effect sizes are presented in Supplementary Tables 2, 3, 7 and 9. b, Forest plots of DunedinPACNI and brain age gap HRs in ADNI and UKB. Effects are presented as HRs with the error bars as the 95% CIs. a,b, Exact sample sizes for each test are reported in Supplementary Tables 2, 3 and 7–9). Lighter shades represent the effect size for each measure while controlling for the other measure (that is, the effect of DunedinPACNI when controlling for brain age gap and vice versa). ADAS-Cog, Alzheimer's Disease Assessment Scale-Cognitive Subscale 13; DSST, Digit Symbol Substitution Task; FAQ, Functional Activities Questionnaire; LogMemory, Logical Memory test; Matrix, Matrix Pattern Completion; MMSE, Mini-Mental State Examination; RAVLT, Rey Auditory Visual Learning Test; Tower, Tower Rearranging; TrailsA, Trail Making Test Part A; TrailsB, Trail Making Test Part B; VM, visual memory; WM, working memory. In addition, we investigated how DunedinPACNI differs from commonly reported MRI-based measures of brain aging, namely hippocampal volume and ventricular volume. To test this, we compared the effect sizes for DunedinPACNI with the effect sizes for bilateral hippocampal volume and bilateral ventricular volume in UKB and in CN participants in ADNI. In both datasets, we observed that a faster DunedinPACNI was generally more strongly and more consistently associated with poor cognition, poor health and frailty, as well as risk of dementia, disease and mortality. Furthermore, DunedinPACNI explained incremental variance in these outcomes over and above hippocampal volume and ventricular volume alone (Extended Data Figs. DunedinPACNI is an accurate and reliable measure of how fast a person is aging derived from a single brain MRI scan. Using 50,106 scans from people ranging from 22 to 98 years old across the HCP, ADNI, UKB and BrainLat datasets, we demonstrate that people with faster DunedinPACNI had not only worse cognitive and brain health (that is, poorer cognition, faster hippocampal atrophy and greater dementia risk) but also worse general health (that is, greater frailty, poorer self-reported health, greater risk of chronic disease and mortality). This indicates that patterns of aging detected during midlife are clinically useful among people in advanced age, including people with neurodegenerative disease. Furthermore, DunedinPACNI showed evidence of generalization to a sample of Latin American adults with and without dementia. Across all analyses, the effect sizes for DunedinPACNI were similar or larger than the effect sizes for brain age gap, an existing age deviation measure estimated using the same structural MRI data. Moreover, DunedinPACNI and brain age gap were only weakly correlated, and DunedinPACNI accounted for incremental variance in aging-related health outcomes. While weak correlations between neuroimaging-based measures of aging may appear surprising, they mirror findings that different epigenetic clocks are also weakly correlated, and that multiple clocks are useful for predicting disease and death65,66. It is possible that biomarkers of aging trained in midlife samples more closely detect the earlier stages of aging compared to biomarkers of aging trained in samples of advanced age. This may partially explain the modest correlations observed between aging biomarkers. Aging remains a construct in search of measurement tools4,5, and DunedinPACNI represents a next-generation measure of aging that is distinct from existing approaches. DunedinPACNI is not without limitations. First, the Dunedin Study, ADNI and UKB consist of data collected primarily from participants of European ancestry. Furthermore, ADNI and UKB oversample participants from higher socioeconomic backgrounds67. There is growing awareness that lack of representativeness in neuroimaging research may hinder clinical translation68,69, including clinical translation of brain-based predictive models58. To address this, we replicated associations with DunedinPACNI in a sample of Latin Americans, as well as low-income and non-White UKB participants. Additionally, our findings in ADNI, UKB and BrainLat demonstrate that DunedinPACNI generalizes well to older adults. This suggests that early differences in aging can be detected in brain MRI measures collected at age 45 years that are clinically useful in people who are decades older. At the same time, it is also possible that an analogous aging biomarker trained in older adults may generalize more readily to other samples of older adults. A priority for future work is to further evaluate the generalizability of DunedinPACNI to people of diverse demographics and backgrounds. Second, DunedinPACNI only uses structural brain measures derived from a T1-weighted MRI scan. We chose this strategy because these scans are collected in nearly every MRI study, thereby maximizing the potential adoption of DunedinPACNI. It is possible that the performance accuracy reported in this study could be improved by including additional structural or functional MRI measures (for example, white matter microstructural integrity from diffusion-weighted images, blood-oxygen-level-dependent signal from T2*-weighted images). Relatedly, while elastic net is widely used to measure biological age11,12,15,16,20, future research should evaluate whether predictive performance accuracy could be increased by more complex, nonlinear models. Third, by design, DunedinPACNI is a measure of the longitudinal rate of the aging of the body derived from a single MRI scan and is not designed to replace longitudinal measurement of brain aging through repeated MRI assessments. Fourth, DunedinPACNI is estimated through observed correlations between measures of brain structure and longitudinal aging, which could reflect multiple causal pathways. For example, faster aging of non-brain organs might cause poorer brain health or vice versa. Alternatively, both may be driven by a third factor. Fifth, although we found robust associations with aging phenotypes across both ADNI and UKB, we generally observed larger effect sizes in ADNI. This could suggest that DunedinPACNI is sensitive to dysfunction in individuals with neurodegenerative diseases. Sixth, DunedinPACNI is currently a quantitative tool for comparison of individuals within datasets but not between datasets. Future research, potentially through data harmonization and normative modeling approaches, should establish normative reference values and ranges that can aid in the clinical interpretation of DunedinPACNI scores. Seventh, as is true for all MRI-based measurements, the derivation of DunedinPACNI may be compromised in low-quality scans, such as those with high levels of head motion. In the analyses presented in this study, we followed standard practice by excluding low-quality scans (Supplementary Figs. Future research should adopt methods to improve data quality in high-motion participants to help maximize generalizability70,71. Currently, DunedinPACNI and general aging biomarkers are research tools that require further validation before potential translation to the clinic5. Nonetheless, we believe that DunedinPACNI and other biomarkers of general aging have several promising uses that differ from, and are complementary to, biomarkers of specific diseases (for example, hippocampal atrophy and AD72 or high blood pressure and stroke73). For example, biomarkers of general aging could help identify a broad risk for many age-related diseases earlier in the lifespan, when individuals may benefit most from health and antiaging intervention23. Biomarkers of general aging could also help establish a person's prognosis after the onset of a specific disease or establish how interventions or insults may change how fast a person is aging25,74,75. Lastly, biomarkers of general aging are necessary to mechanistically link exposures (for example, low socioeconomic status) to aging or to link the rate of aging to outcomes such as increased disease risk4. While detecting or diagnosing a specific disease requires a proximal, specific biomarker of that disease, biomarkers of general aging could be used to measure the distal risk for a broad array of diseases. This difference allows for clinical applications of biomarkers of aging such as DunedinPACNI alongside biomarkers of specific diseases. Several unique features of the Dunedin Study contribute advantages to DunedinPACNI compared to other biomarkers of aging. First, DunedinPACNI was developed in a cohort of people all born in the same year and studied at the same ages throughout their lives, thereby avoiding biases that are introduced by differences in historical exposures across generations and across time. Second, DunedinPACNI was trained on 19 biomarkers that were each assessed over nearly two decades and thus are not influenced by short-term illnesses that can cause aberrant biomarker signals at a single assessment. Third, DunedinPACNI was derived from participants followed from birth to age 45 years—before the onset of chronic, aging-related diseases that cause divergence from typical trajectories of aging. Other biomarkers of aging are typically derived from multiage samples in which many of the older members already have chronic diseases that have altered their body and brain, so those measures may inadvertently repackage disease instead of aging. Fourth, because the Dunedin Study cohort is a population representative cohort with very low attrition and mortality rates, DunedinPACNI does not suffer from oversampling of healthy volunteers, attrition bias (that is, people with worse health being more likely to drop out) or survivor bias (that is, people with worse health dying earlier). Indeed, our results align with prior research showing that DunedinPACE, which like DunedinPACNI was trained on the longitudinal Pace of Aging, is associated with dementia, morbidity and mortality20,22,23. Our results, alongside the fast-growing literature on DunedinPACE, suggest that these unique design characteristics of the Dunedin Study make it a powerful training sample for longitudinal biomarkers of aging. The scope of geroscience has rapidly expanded with the proliferation of omic clocks that can measure how fast people age10. DunedinPACNI is poised to further this growth by allowing individual differences in the rate of longitudinal aging to be estimated from a single noninvasive MRI scan that can be collected in just a few minutes. Indeed, the requisite MRI data to estimate DunedinPACNI have already been collected in many psychiatric, neurological and brain health cohorts, from tens of thousands of research participants across the lifespan and around the world. DunedinPACNI offers an opportunity to enrich such studies and deepen our understanding of the causes of individual differences in the rate of longitudinal aging, including genetics76, childhood adversities77, environmental exposures (for example, lead78,79) and lifestyle factors (for example, physical inactivity, social isolation80). DunedinPACNI may also be adopted as a surrogate endpoint to accelerate our ability to develop, prioritize and evaluate potential antiaging interventions that slow aging and prevent disease81,82. The algorithm for DunedinPACNI is publicly available to the research community to facilitate these and other future research directions (https://github.com/etw11/DunedinPACNI). This research complies with all relevant ethical regulations; all study protocols were approved by the relevant ethical review boards. The specific ethical review boards are detailed in the description of each dataset. The premise and analysis plan for this study were preregistered (https://rb.gy/b9x4u6). All analyses and code were checked for accuracy by an independent analyst. Analyses were conducted on data collected through the Dunedin Study, HCP, ADNI, UKB and BrainLat. The details of each study and dataset are described below. Participants are members of the Dunedin Study, a longitudinal investigation of health and behavior in a population-representative birth cohort. The 1,037 participants (91% of eligible births, 48% female) were all people born between April 1972 and March 1973 in Dunedin, New Zealand, who were residents in the province and who participated in the first assessment at age 3 years19. The cohort represented the full range of socioeconomic status in the general population of New Zealand's South Island and, as adults, matched the New Zealand National Health and Nutrition Survey on key adult health indicators (for example, BMI, smoking and general practitioner visits) and the New Zealand Census of citizens of the same age on educational attainment19,83. Study members are primarily of New Zealand European ethnicity; 8.6% reported Māori ethnicity at age 45 years. General assessments were performed at birth as well as ages 3, 5, 7, 9, 11, 13, 15, 18, 21, 26, 32 and 38 years, and most recently (completed April 2019) at age 45 years, when 938 of the 997 living study members (94.1%) participated. At each assessment, study members were brought to the Dunedin Study Research Unit at the University of Otago for interviews and examinations. In addition, staff provided standardized ratings; informant questionnaires were sent to people who the study members nominated as people who knew them well, and administrative records were searched. The Dunedin Study was approved by the University of Otago Ethics Committee and study members gave written informed consent before participating. As a component of the assessments at age 45 years, study members were scanned using a Siemens MAGNETOM Skyra (Siemens Healthineers) 3T scanner equipped with a 64-channel head/neck coil at the Pacific Radiology Group imaging center in Dunedin, New Zealand. High-resolution T1-weighted images were obtained using an MP-RAGE sequence with the following parameters: repetition time (TR) = 2,400 ms; echo time (TE) = 1.98 ms; 208 sagittal slices; flip angle, 9°; field of view (FOV), 224 mm; matrix = 256 × 256 px; slice thickness = 0.9 mm with no gap (voxel size 0.9 × 0.875 × 0.875 mm); and total scan time = 6 min and 52 s. Three-dimensional (3D) fluid-attenuated inversion recovery (FLAIR) images were obtained with the following parameters: TR = 8,000 ms; TE = 399 ms; 160 sagittal slices; FOV = 240 mm; matrix = 232 × 256 px; slice thickness = 1.2 mm (voxel size 0.9 × 0.9 × 1.2 mm); and total scan time = 5 min and 38 s. Additionally, a gradient echo field map was acquired with the following parameters: TR = 712 ms; TE = 4.92 and 7.38 ms; 72 axial slices; FOV = 200 mm; matrix = 100 × 100 px; slice thickness = 2.0 mm (voxel size 2 mm isotropic); and total scan time = 2 min and 25 s. Of the 938 study members seen at phase 45, 63 declined to participate in MRI scanning, that is, 875 study members completed the MRI scanning protocol. Scanned study members did not differ from other living participants in terms of childhood neurocognitive functioning or childhood socioeconomic status (see the attrition analysis in Extended Data Figs. Of these 875 study members for whom data was available, four were excluded because of major incidental findings or previous injuries (for example, large tumors or extensive damage to the brain or skull), nine because of missing FLAIR or field map scans, one because of poor surface mapping yielding and one because of missing the Pace of Aging variable. This yielded a final training sample of 860 study members (see Supplementary Fig. Structural MRI data were processed using FreeSurfer v.6.0 (ref. Specifically, T1-weighted images were processed and refined with 3D FLAIR images using the recon-all pipeline. Participants' pace of biological aging was measured as changes in 19 biomarkers of study members' cardiovascular, metabolic, pulmonary, kidney, immune and dental systems across ages 26, 32, 38 and 45 years. This measure quantifies participants' rate of aging in year-equivalent units of physiological decline per chronological year. The average participant experienced 1 year of physiological decline per year, that is, a mean (s.d.) See the ‘Statistical analysis' section for more details. One-legged balance was measured using the unipedal stance test as the maximum time achieved across three trials of the test with eyes closed84,85,86. Gait speed (meters per second) was assessed with the 6-m-long GAITRite Electronic Walkway (CIR Systems) with 2-m acceleration and 2-m deceleration before and after the walkway, respectively. Gait speed was assessed under three walking conditions: usual gait speed (walk at a normal pace from a standing start, measured as a mean of two walks) and two challenge paradigms, dual-task gait speed (walk at a normal pace while reciting alternate letters of the alphabet out loud, starting with the letter ‘A', measured as a mean of two walks) and maximum gait speed (walk as fast as safely possible, measured as a mean of three walks). Gait speed was correlated across the three walk conditions87. To increase reliability and take advantage of the variation in all three walk conditions (usual gait and the two challenge paradigms), we calculated the mean of the three highly correlated individual walk conditions to generate our primary measure of composite gait speed. Chair rises were measured as the number of stands with no hands completed in 30 s from a seated position88,90. Handgrip strength was measured for each hand (elbow held at 90°, upper arm held tight against the trunk) as the maximum value achieved across three trials using a Jamar digital dynamometer91,92. Analyses using handgrip strength controlled for BMI. Visuomotor coordination was measured as the time to completion of the grooved pegboard test. Scores were reversed so that higher values corresponded to better performance. Physical limitations were measured with the RAND 36-Item Health Survey 1.0 physical functioning scale. Responses (‘limited a lot', ‘limited a little', ‘not limited at all') assessed difficulty with completing several activities (for example, climbing several flights of stairs, walking more than 1 km, participating in strenuous sports). Scores were reversed to reflect physical limitations so that a high score indicates more limitations. We obtained reports about study members' health and age appearance from three sources: self-reports; informant impressions; and staff impressions. For self-reports, we asked study members about their own impressions of how old they looked: ‘Do you think you look older, younger, or about your actual age?' We also asked study members to rate their age perceptions in years: ‘How old do you feel?' For informant impressions, informants who knew a study member well (94% response rate) were asked: ‘Compared to others their age, do you think they (the study member) look younger or older than others their age? Response options were: ‘much younger', ‘a bit younger', ‘about the same', ‘a bit older' or ‘much older'. For staff impressions, four members of the Dunedin Study unit staff completed a brief questionnaire describing each study member. To assess age appearance, staff used a seven-item scale to assign a ‘relative age' to each study member (1 = young-looking, 7 = old-looking). All reporters rated the study member's general health using the following response options: excellent, very good, good, fair or poor. Correlations between self-ratings, informant ratings and staff ratings ranged from r = 0.48 to r = 0.55. The Wechsler Adult Intelligence Scale, Fourth Edition was administered at age 45 years, yielding the adult IQ. In addition to full-scale IQ, the Wechsler Adult Intelligence Scale, Fourth Edition yields indexes of four specific cognitive functional domains: processing speed; working memory; perceptual reasoning; and verbal comprehension. The Wechsler Intelligence Scale for Children-Revised was administered at ages 7, 9 and 11 years. We measured cognitive decline by studying adult IQ scores after controlling for childhood IQ scores. We focused on change in overall IQ given evidence that aging-related slopes are correlated across all cognitive functions, indicating that research on cognitive decline may be best focused on a highly reliable summary index, rather than focused on individual functions93. Facial age was based on two measurements of perceived age by an independent panel of eight people. First, age range was assessed by an independent panel of four raters, who were presented with standardized (non-smiling) digital facial photographs of study members when they were 45 years old. Raters, who were kept blind to the actual age of study members, used a Likert scale to categorize each study member into a 5-year age range (that is, from 20 to 24 years old and up to 70+ years old). The scores for each study member were averaged across all raters. Second, relative age was assessed by a different panel of four raters, who were told that all photos were of people aged 45 years old. These raters then used a 7-item Likert scale to assign a ‘relative age' to each participant (that is, 1 = ‘young-looking' to 7 = ‘old-looking'). The HCP is a publicly available dataset that includes 1,206 participants with extensive MRI data49. All participants provided written informed consent. Specifically, we used data from 45 participants who completed the scan protocol a second time (with a mean interval between scans of approximately 140 days) allowing for the calculation of test–retest reliability. All participants were free of current psychiatric or neurological illness and were between 25 and 35 years of age. Structural MRI data were analyzed using the HCP minimal preprocessing pipeline94. Briefly, T1-weighted images were processed using a custom FreeSurfer recon-all pipeline optimized for structural MRI with a higher resolution than 1 mm isotropic. Details on HCP MRI data acquisition have been described elsewhere94. The primary goal of ADNI is to test whether serial MRI, positron emission tomography, other biological markers and clinical and neuropsychological assessments can be combined to measure the progression of neurodegeneration in participants with MCI, AD and CN older adults (adni.loni.usc.edu)95. Cognitive and diagnostic data were downloaded on 12 June 2022. MRI data curated from the Alzheimer's Disease Sequencing Project collection were downloaded on 7 December 2023. ADNI was approved by the institutional review boards of all the participating institutions. All participants provided written informed consent. The ADNI sample demographic information can be found in Supplementary Table 17. T1-weighted scans were collected using either 1.5T or 3T scanners. MRI acquisition parameters varied across ADNI sites and waves; however, the targets for acquisition were isotropic 1-mm3 voxels96. Raw T1-weighted images were processed using longitudinal FreeSurfer v.6.0. Scans were excluded for low quality if they did not have a quality control rating of ‘pass' from the ADNI investigators or if segmentation failed visual inspection. Scans were also excluded if participants were missing demographic data, such as age, sex or diagnosis (Supplementary Fig. Further details on the MRI methods in ADNI can be found at adni.loni.usc.edu. ADNI participants completed several cognitive and behavioral assessments at the time of scanning. The ADAS-Cog is a structured scale that evaluates memory, reasoning, language, orientation, ideational praxis and constructional praxis97. Delayed word recall and number cancellation are included in addition to the 11 standard ADAS items98. The test is scored for errors, ranging from 0 (best performance) to 85 (worst performance). The MMSE is a screening instrument that evaluates orientation, memory, attention, concentration, naming, repetition and comprehension, and ability to create a sentence and to copy two overlapping pentagons99. The MMSE is scored as the number of correctly completed items ranging from 0 (worst performance) to 30 (best performance). The MoCA is designed to detect people at the MCI stage of cognitive dysfunction100. The RAVLT is a list learning task that assesses learning and memory. On each of five learning trials, 15 unrelated nouns are presented orally at the rate of one word per second and immediate free recall of the words is elicited. After a 30-min delay filled with unrelated testing, free recall of the original 15-word list is elicited. Both immediate recall and percentage forgotten are used. The LogMem tests I and II (delayed paragraph recall) are from the Wechsler Memory Scale-Revised. Free recall of one short story is elicited immediately after being read aloud to the participant and again after a 30-min delay. The total bits of information recalled after the delay interval (maximum score = 25) are analyzed. The Trail Making Test, Part B, consists of 25 circles, either numbered (one through 13) or containing letters (A through L). Time to complete (300 s maximum) is the primary measure of interest. The FAQ is a self-report measure of instrumental ADLs, such as preparing meals, performing chores, keeping a schedule and traveling outside of one's neighborhood101. Each unique cognitive testing measure was paired with the participant's most temporally proximate brain scan within 6 months of cognitive testing. ADNI participants were classified into CN, MCI or dementia groups by ADNI study physicians based on subjective memory complaints, multiple neurocognitive and behavioral assessment scores, and level of impairment in ADLs. Complete diagnostic criteria can be found at adni.loni.usc.edu. Each individual scan was categorized according to the most temporally proximate cognitive diagnosis received by that participant. For the purposes of visualization in Fig. UKB is a UK population-based prospective study of 502,486 participants between the ages of 40 and 69 at baseline assessment102. We analyzed data from 42,583 participants who underwent brain MRI. UKB was approved by the North West Multi-centre for Research Ethics Committee. All participants provided written informed consent. UKB sample demographic information can be found in Supplementary Table 17. MRI methods for UKB have been described in detail elsewhere103. Briefly, MRI data were collected using three identical 3T Siemens Skyra scanners with a 32-channel Siemens head coil. T1-weighted images were obtained using a 3D MP-RAGE with the following parameters: TR = 2,000 ms; inversion time = 880 ms; 208 sagittal slices, matrix = 256 × 256 px; slice thickness = 1 mm with no gap; and total scan time = 4 min and 52 s. Our study made use of imaging-derived phenotypes generated using an image-processing pipeline developed and run on behalf of UKB103. As part of this pipeline, raw T1-weighted images were processed using the cross-sectional FreeSurfer v.6.0. All brain measures used in the cross-sectional analyses presented in this study were derived from the outputs of this FreeSurfer pipeline. We excluded UKB participants with a very low signal-to-noise ratio and highly unusual summary morphometrics indicative of low-quality reconstruction (Supplementary Fig. To measure change in hippocampal volume in the subset of UKB participants with longitudinal MRI data, we reprocessed all T1-weighted images for this subset of participants using the longitudinal FreeSurfer v.6.0 pipeline104. This allowed us to avoid the known biases that can be introduced by different processing stages of the longitudinal pipeline in different hardware and software environments. Specifically, we reprocessed both time points of each participant's T1-weighted scans with the cross-sectional recon-all pipeline105. Then, we built an unbiased within-participant template106 using robust, inverse, consistent registration107 and reprocessed each T1-weighted scan through the automated longitudinal pipeline104. UKB participants completed a battery of cognitive tests at the time of MRI. The details of these cognitive tests have been described elsewhere108. To further investigate aging-related health, we used the Fried Frailty Index53. Briefly, the Fried Frailty Index is based on meeting the criteria for declining functioning across five domains: unintentional weight loss; exhaustion; weakness; physical inactivity; and slow walking speed. During their imaging visit, UKB participants were also asked to rate their overall health as ‘poor', ‘fair', ‘good' or ‘excellent'. We used these ratings to investigate self-reported overall health (field ID = 2178). To assess the influence of DunedinPACNI on aging-related disease and mortality risk in UKB, we used variables from algorithmically defined health outcomes. Briefly, algorithmically defined outcomes are generated by combining information from baseline assessments (self-reported medical conditions, operations and medications) with linked data from hospital admissions and death registries. Because of the relatively small number of aging-related disease diagnoses at the follow-up, we defined aging-related morbidity as being diagnosed with myocardial infarction (field ID = 42000), chronic obstructive pulmonary disease (field ID = 42016), dementia (field ID = 42018) or stroke (field ID = 42006). Furthermore, we defined the risk of chronic disease as the emergence of one or more of these diagnoses among participants who were healthy at the time of scanning (that is, baseline). Mortality was quantified during the follow-up from death records (field ID = 40000). To test the association between DunedinPACNI and socioeconomic gradients of health, we tested whether UKB participants differed in DunedinPACNI scores as a function of educational attainment and household income. We grouped participants into three categories according to their self-reported educational qualifications (field ID = 6138) following prior work110. We also tested whether UKB participants differed in DunedinPACNI scores as a function of household income (field ID = 738). We also conducted sensitivity analyses while restricting the UKB sample to either only low-income or only non-White participants. We considered participants as having a low income if they reported making less than £18,000 per year in household income. We considered participants to be non-White if they did not report their ethnic background (field ID = 21000) as ‘any other White background', ‘British', ‘do not know', ‘Irish', ‘prefer not to answer' or ‘White'. BrainLat is a multimodal neuroimaging dataset of patients with neurodegenerative diseases and healthy adult controls collected in Argentina, Chile, Colombia, Mexico and Peru63. We analyzed neuroimaging data from 368 individuals who were either cognitively healthy or diagnosed with AD or behavioral variant FTD. The BrainLat study was approved by the institutional ethical boards of each recruitment site. All participants, or their legal representatives, provided written informed consent. The BrainLat demographic data can be found in Supplementary Table 17. The MRI methods for BrainLat have been described in detail elsewhere63. Briefly, T1-weighted MP-RAGE scans were collected on either 1.5 or 3T scanners. Acquisition parameters varied across sites, but scans most frequently had isometric 1-mm3 voxels. Scans were downloaded and then processed using FreeSurfer v.6.0. Participants were excluded if they failed the automated FreeSurfer quality metrics or a visual quality check of segmentation output (Supplementary Fig. All participants included could speak fluent Spanish and had adequate visual and auditory capacity for testing. Participants were classified as CN if they had a modified clinical dementia rating of 0, an MMSE score above 25 and lacked a history of substance abuse, and neurological or psychiatric disorders. Patients were classified into the AD or FTD groups according to the National Institute of Neurological Disorders and Stroke–Alzheimer Disease and Related Disorders working group for probable AD or probable behavioral variant FTD. Diagnosis was supported using appropriate MRI or positron emission tomography imaging when needed63. BrainLat participants were evaluated with the MoCA. The MoCA is designed to detect people at the MCI stage of cognitive dysfunction100. Briefly, we measured a panel of the following 19 biomarkers (Fig. 1a) at ages 26, 32, 38, and 45 years: BMI, waist/hip ratio, HbA1c, leptin, blood pressure (mean arterial pressure), cardiorespiratory fitness (VO2max), forced vital capacity ratio (FEV1/FVC), FEV1, total cholesterol, triglycerides, HDL, lipoprotein(a), apolipoprotein B100/A1 ratio, eGFR, blood urea nitrogen, hsCRP, white blood cell count, mean periodontal AL and the number of dental-caries-affected tooth surfaces (tooth decay). To calculate each study member's Pace of Aging, we first transformed the biomarker values to a standardized scale. For each biomarker at each wave, we standardized values according to the age 26 distribution. Next, we calculated each study member's slope for each of the 19 biomarkers using a mixed-effects growth model that regressed the biomarker's level on age. Finally, we combined information from the 19 slopes of the biomarkers using a unit-weighting scheme. We calculated each study member's Pace of Aging as the sum of age-dependent annual changes in biomarker z-scores. Biomarker standardization was performed separately for men and women. A schematic of DunedinPACNI model development can be found in Fig. We trained an elastic net regression model to estimate the Pace of Aging from structural neuroimaging phenotypes in 860 Dunedin Study members at age 45 years (for the attrition analysis and inclusion criteria see Extended Data Figs. We selected 315 FreeSurfer measures as predictors from the following categories: regional CT, regional cortical SA, regional cortical GMV, regional cortical GWR and ‘ASEG' volumes (that is, regional subcortical GMV, ventricular volumes and bilateral volume of white matter hypointensities). All cortical data were parcellated according to the Desikan–Killiany Atlas111. Note that although many ADNI scans do not pass quality control (Supplementary Fig. 5), FreeSurfer is a robust segmentation method, especially in healthy individuals112. Four phenotypes from the ‘ASEG' volumes were excluded because of insufficient variance in the Dunedin Study (left and right white matter hypointensities, left and right non-white matter hypointensities). Model training was performed using the caret package in R. We conducted a grid search across a range of α and λ values. We used 100 repetitions of tenfold cross-validation to estimate model performance in held-out participants. The effect of sex was regressed from the Pace of Aging before model training. To prevent information leak during cross-validation, we regressed sex from each training set and applied the resulting β weights to each test set. This approach ensured that our model only used information from the training set, including covariate regression, when calculating predictions in each test set. We selected optimal tuning parameters according to the highest variance explained and lowest mean absolute error. The optimal tuning parameters were α = 0.214 and λ = 0.100. Using these parameters, we fitted the model to the entire n = 860 sample. The raw elastic net regression model weights can be found in Supplementary Table 18. To generate DunedinPACNI scores in HCP, ADNI, UKB and BrainLat participants, we applied the regression weights from the DunedinPACNI model to FreeSurfer-derived phenotypes in each dataset and summed the products and model intercept. In ADNI, UKB and BrainLat, DunedinPACNI scores were correlated with chronological age (ADNI: r = 0.37; UKB: r = 0.50; BrainLat: r = 0.37; Supplementary Fig. In addition, we conducted the same procedure again without GWR because this measure is not always distributed in public datasets. We observed slightly reduced model accuracy when GWR was not included. DunedinPACNI estimates without GWR phenotypes showed excellent test–retest reliability in HCP. DunedinPACNI estimates were similar with and without GWR phenotypes in ADNI, UKB and BrainLat (see Supplementary Figs. We submitted raw T1-weighted images from ADNI, UKB and BrainLat to the publicly available brainageR algorithm (v.2.1). This model, which has been described in detail elsewhere113, is trained to predict chronological age in a sample of healthy, cognitively unimpaired individuals aged 18–92 years. This algorithm was selected because it generates highly reliable estimates among published algorithms64. Briefly, brainageR is estimated by first segmenting and normalizing T1-weighted images using SPM12. Next, coefficients derived from a Gaussian process regression model predicting chronological age in a training dataset (n = 2,001) are applied to morphometric features from brain segmentations to predict participants' chronological age. Brain age gap was subsequently estimated by subtracting actual chronological age from predicted age113. Notably, 15 ADNI scans failed the brain age gap pipeline (14 failed visual inspection of segmentation, one error when computing the predicted age). These scans were excluded from all brain age gap analyses, including comparative analyses with DunedinPACNI. To first test the validity of DunedinPACNI in the Dunedin Study training sample, we tested for linear associations between DunedinPACNI scores and one-legged balance, gait speed, step in place, chair stands, grip strength, visuomotor coordination, subjective physical limitations, subjective health, cognitive function, child-to-adult cognitive decline and facial aging while controlling for sex. We compared these effect sizes to associations between each of these measures and the original, longitudinal Pace of Aging. Reliability was quantified using a two-way mixed-effects intraclass correlation coefficient (3,1) with session modeled as a fixed effect, participant as a random effect and the test–retest interval as an effect of no interest114. We first used linear regression models to test for associations between DunedinPACNI and scores on tests of cognition, physical function and health in ADNI and UKB. All analyses controlled for age and sex. In ADNI, we calculated robust standard errors to account for nonindependence from repeated observations. We also tested the standardized differences in DunedinPACNI scores between three groups based on cognitive status: CN, MCI and dementia. All group difference comparisons controlled for age and sex. We again calculated robust standard errors to account for nonindependence and conduced a sensitivity analysis while controlling for APOE ε4 carriership. We repeated these analyses with brain age gap. Notably, when conducting analyses on the combined effects of DunedinPACNI and brain age gap on cognitive outcomes in ADNI, we restricted the sample to the first time point of each measure. These analyses included only one observation per participant, allowing us to more easily combine effect sizes and CIs. We conducted a Cox proportional hazards regression using the baseline DunedinPACNI scores of ADNI participants to predict their probability of cognitive decline or clinical conversion to dementia during the follow-up window. Conversion in CN participants was defined as having a diagnosis of CN at baseline but a diagnosis of MCI or dementia at the end of the follow-up. Conversion in participants with MCI was defined as having a diagnosis of MCI at baseline and a diagnosis of dementia by the end of the follow-up. Participants who had a baseline diagnosis of dementia or transitioned from MCI to CN were not included in this analysis. The analysis controlled for sex, age at baseline and length of the observation window. We investigated the influence of AD genetic risk on these results by conducting all analyses while additionally controlling for APOE ε4 carriership. We repeated these analyses for brain age gap. We used repeated MRI measurements from ADNI (n = 1,302) and UKB (n = 4,601) to generate estimates of change in hippocampal GMV. We used longitudinal ComBat on ADNI MRI data to remove differential scanner effects115. Next, using all available time points for each participant, we generated multilevel linear models for bilateral hippocampal volume with random effects for both participant and age. Using these models, we derived trajectories to track change in hippocampal GMV for each participant. We then tested whether each participant's baseline DunedinPACNI scores could predict their subsequent rate of hippocampal atrophy. These analyses controlled for age, sex and length of the observation period. We investigated the influence of AD genetic risk on these results by conducting these analyses while additionally controlling for APOE ε4 carriership. We repeated these analyses for brain age gap. To investigate the association between DunedinPACNI and morbidity, we used UKB data to calculate the standardized differences in DunedinPACNI scores between three groups based on the number of lifetime chronic disease diagnosis (0, 1, 2+). Next, we conducted a Cox proportional hazards regression using UKB participants' baseline DunedinPACNI scores to predict the onset of a chronic aging-related disease (n = 827 emergent diagnoses: myocardial infarction, chronic obstructive pulmonary disease, dementia or stroke) in participants who had never previously received any of these diagnoses at the time of scanning (n = 40,753). Similarly, to investigate the association between DunedinPACNI and mortality, we conducted a Cox proportional hazards regression using UKB participants' baseline DunedinPACNI scores to predict death (n = 757 deaths). Both models controlled for baseline age, time to onset and sex. We repeated these analyses for brain age gap. To investigate whether DunedinPACNI reflected gradients of socioeconomic inequality57, we first tested for linear relationships between DunedinPACNI and years of education in ADNI and UKB. We also tested for a linear relationship between DunedinPACNI and household income in UKB. These analyses controlled for sex and age. In ADNI, we included only the first MRI observation per participant. To investigate whether DunedinPACNI generalizes to samples of individuals who are underrepresented in neuroimaging research59, we tested whether the degree of acceleration in DunedinPACNI among ADNI participants with dementia was similar in BrainLat participants with dementia. We tested for standardized differences by comparing the AD and FTD groups to the CN group, respectively. We also tested for linear associations between DunedinPACNI and MoCA scores. All analyses in this sample controlled for age and sex. We then compared the magnitude of acceleration among BrainLat participants with dementia to the previously identified acceleration among ADNI participants with dementia. Lastly, we compared the strength of the linear association between DunedinPACNI and MoCA scores in BrainLat participants to the previously identified association in ADNI participants. We investigated how DunedinPACNI differs from two commonly used MRI-based measures of brain aging: hippocampal volume and ventricular volume. We calculated hippocampal volume as the sum of left and right hippocampal GMV measures derived from FreeSurfer. Likewise, we calculated ventricular volume as the sum of the left and right lateral ventricular volume measures. We first repeated cross-sectional associations with cognition, frailty and poor health in UKB, substituting DunedinPACNI with hippocampal volume or ventricular volume. Next, we conducted this same procedure with our Cox proportional hazards regression models of chronic disease and mortality risk in UKB, and cognitive decline risk among CN ADNI participants. Lastly, we compared coefficients for all analyses while including DunedinPACNI and either hippocampal volume or ventricular volume in the respective models. All analyses controlled for age and sex. All visualizations were generated using the R package ggplot2 (ref. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The Dunedin Study data are available via managed access. Researchers who wish to use the Dunedin Study data are invited to submit a concept paper proposing the data analysis project they wish to carry out, subject to the approval of the Dunedin Study investigators. Complete instructions on accessing the Dunedin Study data can be found at https://sites.duke.edu/moffittcaspiprojects/data-use-guidelines/. The HCP data are publicly available at www.humanconnectomeproject.org/data/. The ADNI data are publicly available at https://adni.loni.usc.edu/. Researchers can apply to access all UKB data at https://ams.ukbiobank.ac.uk/ams/. The BrainLat data are publicly available at www.synapse.org/Synapse:syn51549340/wiki/624187. 1–3 and 5–9) are published with this article. The DunedinPACNI algorithm is publicly available at GitHub (https://github.com/etw11/DunedinPACNI). All scripts used in the analyses presented in this article are publicly available at GitHub (https://github.com/etw11/WhitmanElliott_2024). Belsky, D. W. et al. Quantification of biological aging in young adults. Disparities in the pace of biological aging among midlife adults of the same chronological age have implications for future frailty risk and policy. Kuo, P.-L. et al. Longitudinal phenotypic aging metrics in the Baltimore Longitudinal Study of Aging. Moqri, M. et al. Biomarkers of aging for the identification and evaluation of longevity interventions. Putting epigenetic biomarkers to the test for clinical trials. Melton, L. Scientists hone tools to measure aging and rejuvenation interventions. Blood biomarkers for healthy aging. Horvath, S. & Raj, K. DNA methylation-based biomarkers and the epigenetic clock theory of ageing. Rutledge, J., Oh, H. & Wyss-Coray, T. Measuring biological age using omics data. Hannum, G. et al. Genome-wide methylation profiles reveal quantitative views of human aging rates. Horvath, S. DNA methylation age of human tissues and cell types. Improved precision of epigenetic clock estimates across tissues and its implication for biological ageing. Raffington, L. & Belsky, D. W. Integrating DNA methylation measures of biological aging into social determinants of health research. Lu, A. T. et al. DNA methylation GrimAge strongly predicts lifespan and healthspan. Levine, M. E. et al. An epigenetic biomarker of aging for lifespan and healthspan. Sehgal, R. et al. Systems Age: a single blood methylation test to quantify aging heterogeneity across 11 physiological systems. Moffitt, T. E. Behavioral and social research to accelerate the geroscience translation agenda. Poulton, R., Guiney, H., Ramrakha, S. & Moffitt, T. E. The Dunedin study after half a century: reflections on the past, and course for the future. Belsky, D. W. et al. DunedinPACE, a DNA methylation biomarker of the pace of aging. Whitman, E. T. et al. A blood biomarker of the pace of aging is associated with brain structure: replication across three cohorts. Sugden, K. et al. Association of pace of aging measured by blood-based DNA methylation with age-related cognitive impairment and dementia. Faul, J. D. et al. Epigenetic-based age acceleration in a representative sample of older Americans: associations with aging-related morbidity and mortality. Harris, K. M. et al. Sociodemographic and lifestyle factors and epigenetic aging in US young adults: NIMHD social epigenomics program. Associations of seven measures of biological age acceleration with frailty and all-cause mortality among adult survivors of childhood cancer in the St. Jude Lifetime Cohor. Predicting brain age with deep learning from raw imaging data results in a reliable and heritable biomarker. Bashyam, V. M. et al. MRI signatures of brain age and disease over the lifespan based on a deep brain network and 14 468 individuals worldwide. Yin, C. et al. Anatomically interpretable deep learning of brain age captures domain-specific cognitive impairment. Brain aging in major depressive disorder: results from the ENIGMA major depressive disorder working group. Sluiskes, M. H. et al. Clarifying the biological and statistical assumptions of cross-sectional biological age predictors: an elaborate illustration using synthetic and real data. Butler, E. R. et al. Pitfalls in brain age analyses. Individual variations in ‘brain age' relate to early-life factors more than to longitudinal brain change. Biondo, F. et al. Brain-age is associated with progression to dementia in memory clinic patients. Brain and body are more intertwined than we knew. A Practical Guide to Support Vector Classification (National Taiwan Univ., 2016); www.csie.ntu.edu.tw/~cjlin/papers/guide/guide.pdf Belsky, D. W. et al. Quantification of the pace of biological aging in humans through a blood test, the DunedinPoAm DNA methylation algorithm. Haufe, S. et al. On the interpretation of weight vectors of linear models in multivariate neuroimaging. Accelerated longitudinal gray/white matter contrast decline in aging in lightly myelinated cortical regions. Grydeland, H., Walhovd, K. B., Tamnes, C. K., Westlye, L. T. & Fjell, A. M. Intracortical myelin links with performance variability across the human lifespan: results from T1- and T2-weighted MRI myelin mapping and diffusion tensor imaging. Differential longitudinal changes in cortical thickness, surface area and volume across the adult life span: regions of accelerating and decelerating change. Apparent thinning of human visual cortex during childhood is associated with myelination. Regional brain changes in aging healthy adults: general trends, individual differences and modifiers. Salat, D. H. et al. Thinning of the cerebral cortex in aging. Structural progression of Alzheimer's disease over decades: the MRI staging scheme. Salat, D. H. et al. Age-associated alterations in cortical gray and white matter signal intensity and gray to white matter contrast. Effects of age on volumes of cortex, white matter and subcortical structures. The WU-Minn Human Connectome Project: an overview. Fortea, J. et al. APOE4 homozygozity represents a distinct genetic form of Alzheimer's disease. Jack, C. R. Jr et al. Rates of hippocampal atrophy correlate with change in clinical status in aging and AD. Associations of physical frailty with health outcomes and brain structure in 483 033 middle-aged and older adults: a population-based study from the UK Biobank. Fried, L. P. et al. Frailty in older adults: evidence for a phenotype. Idler, E. L. & Benyamini, Y. Self-rated health and mortality: a review of twenty-seven community studies. Chen-Xu, J. et al. Subnational inequalities in years of life lost and associations with socioeconomic factors in pre-pandemic Europe, 2009–19: an ecological study. Balaj, M. et al. Parental education and inequalities in child mortality: a global systematic review and meta-analysis. Marmot, M. G. et al. Health inequalities among British civil servants: the Whitehall II study. Greene, A. S. et al. Brain–phenotype models fail for individuals who defy sample stereotypes. Global South research is critical for understanding brain health, ageing and dementia. Brain clocks capture diversity and disparities in aging and dementia across geographically diverse populations. Prediction of dementia risk in low-income and middle-income countries (the 10/66 Study): an independent external validation of existing models. Mills, M. C. & Rahal, C. The GWAS Diversity Monitor tracks diversity by disease in real time. The BrainLat project, a multimodal neuroimaging dataset of neurodegeneration from underrepresented backgrounds. Prediction of brain age using structural magnetic resonance imaging: a comparison of accuracy and test–retest reliability of publicly available software packages. Belsky, D. W. et al. Eleven telomere, epigenetic clock, and biomarker-composite quantifications of biological aging: do they measure the same thing?. Moqri, M. et al. A unified framework for systematic curation and evaluation of aging biomarkers. Brayne, C. & Moffitt, T. E. The limitations of large-scale volunteer databases to address inequalities and global challenges in health and aging. Participant diversity is necessary to advance brain aging research. Falk, E. B. et al. What is a representative brain? Brain morphometry in older adults with and without dementia using extremely rapid structural scans. Greene, D. J. et al. Behavioral interventions for reducing head motion during MRI scans in children. Jack, C. R. Jr, Petersen, R. C., O'Brien, P. C. & Tangalos, E. G. MR-based hippocampal volumetry in the diagnosis of Alzheimer's disease. Sehl, M. E., Carroll, J. E., Horvath, S. & Bower, J. E. The acute effects of adjuvant radiation and chemotherapy on peripheral blood epigenetic age in early stage breast cancer patients. Sehgal, R. et al. DNAm aging biomarkers are responsive: insights from 51 longevity interventional studies in humans. Melzer, D., Pilling, L. C. & Ferrucci, L. The genetics of human ageing. Kim, K. et al. Association of adverse childhood experiences with accelerated epigenetic aging in midlife. Reuben, A. et al. Association of childhood lead exposure with MRI measurements of structural brain integrity in midlife. Lee, H., Lee, M. W., Warren, J. R. & Ferrie, J. Childhood lead exposure is associated with lower cognitive functioning at older ages. Livingston, G. et al. Dementia prevention, intervention, and care: 2020 report of the Lancet Commission. & Espeland, M. A. Metformin as a tool to target aging. Kraus, W. E. et al. 2 years of calorie restriction and cardiometabolic risk (CALERIE): exploratory outcomes of a multicentre, phase 2, randomised controlled trial. Richmond-Rakerd, L. S. et al. Clustering of health, crime and social-welfare inequality in 4 million citizens from two nations. A., Marin, R., Cyhan, T., Roberts, H. & Gill, N. W. Normative values for the unipedal stance test with eyes open and closed. Bohannon, R. W., Larkin, P. A., Cook, A. C., Gear, J. Decrease in timed balance test scores with aging. Vereeck, L., Wuyts, F., Truijen, S. & Van de Heyning, P. Clinical assessment of balance: normative data, and gender and age effects. Rasmussen, L. J. H. et al. Association of neurocognitive and physical function with gait speed in midlife. & Rikli, R. E. Measuring functional fitness of older adults. Rikli, R. E. & Jones, C. J. Functional fitness normative scores for community-residing older adults, ages 60–94. Jones, C. J., Rikli, R. E. & Beam, W. C. A 30-s chair-stand test as a measure of lower body strength in community-residing older adults. Grip and pinch strength: normative data for adults. Rantanen, T. et al. Midlife hand grip strength as a predictor of old age disability. Coupled cognitive changes in adulthood: a meta-analysis. The Alzheimer's Disease Neuroimaging Initiative in the era of Alzheimer's disease treatment: a review of ADNI studies from 2021 to 2022. The Alzheimer's Disease Neuroimaging Initiative (ADNI): MRI methods. Rosen, W. G., Mohs, R. C. & Davis, K. L. A new rating scale for Alzheimer's disease. Mohs, R. C. et al. Development of cognitive instruments for use in clinical trials of antidementia drugs: additions to the Alzheimer's Disease Assessment Scale that broaden its scope. The Alzheimer's Disease Cooperative Study. A practical method for grading the cognitive state of patients for the clinician. The Montreal Cognitive Assessment, MoCA: a brief screening tool for mild cognitive impairment. Pfeffer, R. I., Kurosaki, T. T., Harrah, C. H. Jr, Chance, J. M. & Filos, S. Measurement of functional activities in older adults in the community. Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Reuter, M., Schmansky, N. J., Rosas, H. D. & Fischl, B. Within-subject template estimation for unbiased longitudinal image analysis. & Dale, A. M. Measuring the thickness of the human cerebral cortex from magnetic resonance images. Avoiding asymmetry-induced bias in longitudinal image processing. Highly accurate inverse consistent registration: a robust approach. Reliability and validity of the UK Biobank cognitive tests. Williams, D. M., Jylhävä, J., Pedersen, N. L. & Hägg, S. A frailty index for UK biobank participants. Chadeau-Hyam, M. et al. Education, biological ageing, all-cause and cause-specific mortality and morbidity: UK biobank cohort study. Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. Alfaro-Almagro, F. et al. Confound modelling in UK Biobank brain imaging. Shrout, P. E. & Fleiss, J. L. Intraclass correlations: uses in assessing rater reliability. Beer, J. C. et al. Longitudinal ComBat: a method for harmonizing longitudinal multi-scanner imaging data. Wilkinson, L. Ggplot2: elegant graphics for data analysis by WICKHAM, H. Biometrics 67, 678–679 (2011). This research received support from the National Institute on Aging (grant no. and the UK Medical Research Council (grant no. The Dunedin Multidisciplinary Health and Development Research Unit is supported by the New Zealand Health Research Council (program grant no. We thank the Dunedin Study members, Unit research staff, the previous Study Director, Emeritus Distinguished Professor, the late R. Poulton, for his leadership during the study's research transition from young adulthood to aging (2000–2023), and Study founder P. A. Silva. The Dunedin Unit is located within the Ngāi Tahu tribal area who we acknowledge as first peoples, tangata whenua (people of this land). This research has been conducted using the UKB Resource under application no. Data collection and sharing for ADNI is funded by the National Institute on Aging (National Institutes of Health grant no. The grantee organization is the Northern California Institute for Research and Education. In the past, ADNI has also received funding from the National Institute of Biomedical Imaging and Bioengineering, the Canadian Institutes of Health Research and private sector contributions through the Foundation for the NIH, including generous contributions from the following: AbbVie, Alzheimer's Association, Alzheimer's Drug Discovery Foundation, Araclon Biotech, BioClinica, Biogen, Bristol Myers Squibb, CereSpir, Cogstate, Eisai, Elan Pharmaceuticals, Eli Lilly and Company, EuroImmun, F. Hoffmann-La Roche and its affiliated company Genentech, Fujirebio, GE Healthcare, IXICO, Janssen Alzheimer Immunotherapy Research & Development, Johnson & Johnson Pharmaceutical Research & Development, Lumosity, Lundbeck, Merck & Co., Meso Scale Diagnostics, NeuroRx Research, Neurotrack Technologies, Novartis Pharmaceuticals Corporation, Pfizer, Piramal Imaging, Servier, Takeda Pharmaceutical Company and Transition Therapeutics. These authors contributed equally: Ethan T. Whitman, Terrie E. Moffitt. Department of Psychology and Neuroscience, Duke University, Durham, NC, USA Ethan T. Whitman, Annchen R. Knodt, Karen Sugden, Benjamin S. Williams, Avshalom Caspi, Terrie E. Moffitt & Ahmad R. Hariri Department of Psychology, Center for Brain Science, Harvard University, Cambridge, MA, USA Department of Neurology, Christchurch Hospital, Waitaha Canterbury, Te Whatu Ora–Health New Zealand, Christchurch, New Zealand Dunedin Multidisciplinary Health and Development Research Unit, Department of Psychology, University of Otago, Dunedin, New Zealand Sean Hogan, David Ireland, Sandhya Ramrakha & Reremoana Theodore Te Kura Mahi ā-Hirikapo, School of Psychology, Speech and Hearing, University of Canterbury, Christchurch, New Zealand Pacific Radiology Canterbury, Christchurch, New Zealand King's College London, Social, Genetic, and Developmental Psychiatry Centre, Institute of Psychiatry, Psychology, & Neuroscience, London, UK Department of Psychiatry and Behavioral Sciences, Duke University, Durham, NC, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Ethan T. Whitman or Terrie E. Moffitt. are listed as inventors of DunedinPACE, a Duke University and University of Otago invention licensed to TruDiagnostic for commercial uses; however, the DunedinPACE algorithm is open access for research purposes. The other authors declare no competing interests. Nature Aging thanks Denise C. Park and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. No significant differences in childhood IQ were found between the full cohort, those still alive, those seen at Phase 45, or those scanned at Phase 45. Those who were deceased by the Phase 45 data collection had significantly lower childhood IQ's than those who were still alive (t = 2.09, p = 0.04). Center lines of boxes represent the median. We report childhood IQ because it is known to be a strong predictor of late-life health outcomes, as shown by many cohort studies from many nations. Childhood IQ predicts health and social outcomes in adulthood, and these outcomes include physical functions, cognitive decline, mental health, inflammation, metabolic syndrome, disease incidence, dementia, mortality, and also neuroimaging-based, genomic, and epigenetic indicators of health. Based on the literature, we report three groups: study members who died before age 45, and thus could not have taken part in data collection, study members who were alive and thus could take part, and study members who actually did take part. We compared these three groups to the original birth cohort. The figure shows that the small group of study members who had died before age 45 had significantly lower mean childhood IQ as a group. Some of the early deaths were Dunedin Study members who had more disadvantages in their lives leading to poorer health and increased risk of early mortality. Study members who died of childhood diseases may have been already unwell at the time of IQ testing, which could have lowered their scores. However, cohort members who are still alive and cohort members who took part in data collection did not differ from the full original cohort on their mean childhood IQ; they still represent population variation on this key health risk factor. No significant differences were found between the full cohort, those deceased, those alive, those seen at Phase 45, or those scanned at Phase 45 on childhood SES. Center lines of boxes represent the median. We report childhood SES because it is known to be a strong predictor of late-life health outcomes, as shown by many cohort studies from many nations. Childhood SES separately predicts health and social outcomes in adulthood, and these outcomes include physical functions, cognitive decline, mental health, inflammation, metabolic syndrome, disease incidence, dementia, mortality, and also neuroimaging-based, genomic, and epigenetic indicators of health. Based on the literature, we report three groups: study members who died before age 45, and thus could not have taken part in data collection, study members who were alive and thus could take part, and study members who actually did take part. We compared these three groups to the original birth cohort. The figure shows that the small group of study members who had died before age 45 had somewhat lower mean childhood SES as a group. Some of the early deaths were Dunedin Study members who had more disadvantages in their lives leading to poorer health and increased risk of early mortality. However, cohort members who are still alive and cohort members who took part in data collection did not differ from the full original cohort on their mean childhood SES; they still represent population variation on this key health risk factor. Forest plot of associations between cognitive, frailty, health, and social class measures and DunedinPACNI with and without inclusion of participants who later develop dementia or who carry two APOE E4 risk alleles. Effects are presented as standardized beta coefficients with error bars as 95% confidence intervals. Note that for visualization, the signs of some outcomes were flipped, such that higher scores for all outcomes reflected worse performance or health. Forest plot of DunedinPACNI hazard ratios for chronic disease and mortality with and without inclusion of participants who develop dementia or carry two APOE E4 risk alleles. Effects are presented as hazard ratios with error bars as 95% confidence intervals. Exact sample sizes for all tests are presented in Supplementary Table 4. Abbreviations: DSST = Digit Symbol Substitution Task, HR = hazard ratio, IQ = intelligence quotient, Matrix = Matrix Pattern Completion, RT = Reaction Time, Tower = Tower Rearranging, TrailsA = Trail Making Test Part A, TrailsB = Trail Making Test Part B, VM = Visual Memory, WM = Working Memory. Survival curve of the relative proportion of 624 cognitively normal ADNI participants at baseline who remained cognitively normal during the follow-up window grouped by slow, average, and fast baseline DunedinPACNI scores. Censored timepoints (that is, points when participants either converted to MCI/dementia or were lost to follow-up) are shown with cross marks. Note that relatively few participants have >9 years of follow-up. Abbreviations: ADNI = Alzheimer's Disease Neuroimaging Initiative, SD = standard deviation. Forest plots of DunedinPACNI and brain age gap standardized mean differences between dementia groups and healthy controls in BrainLat. Error bars represent 95% confidence intervals. Lighter shades represent the effect size for each measure while controlling for the other measure (that is, effect of DunedinPACNI when controlling for brain age gap, and vice versa). Forest plots of DunedinPACNI and brain age gap absolute standardized association effect sizes with the MoCA. Error bars represent 95% confidence intervals. Lighter shades represent the effect size for each measure while controlling for the other measure (that is, effect of DunedinPACNI when controlling for brain age gap, and vice versa). c. Forest plot of absolute standardized associations between DunedinPACNI alone, brain age gap alone, and their combined associations with MoCA scores in BrainLat. Error bars represent 95% confidence intervals. Exact sample sizes for all tests are presented in Supplementary Tables 10, 11). Abbreviations: AD = Alzheimer's dementia, FTD = frontotemporal dementia, MoCA = Montreal Cognitive Assessment, SDs = standard deviations. Forest plots for effect sizes for DunedinPACNI alone, brain age gap alone, and their combined associations with clinical outcomes. Combined effects were considered to be the sum of beta coefficients when both DunedinPACNI and brain age gap were included in a model while controlling for age and sex. Error bars represent 95% confidence intervals. Note that for visualization, the signs of some outcomes were flipped, such that higher scores for all outcomes reflected worse performance or health. Forest plots of hazard ratios for risk of cognitive decline, dementia, chronic disease, and death. Combined effects were considered to be the products of respective hazard ratios for DunedinPACNI and brain age gap included in a model while controling for age and sex. Error bars represent 95% confidence intervals. Exact sample sizes for each test in a. and b. are reported in Supplementary Tables 2, 3, 7–9). Note that for cognitive tests in ADNI, only baseline observations were used in this analysis to facilitate combination of effects. Thus, sample sizes for these tests are reflected by the number of individuals, not observations, in the sample. This information is provided in Supplementary Table 2. Abbreviations: ADAS-Cog = Alzheimer's Disease Assessment Scale – Cognitive Subscale 13, ADLs = activities of daily living, ADNI = Alzheimer's Disease Neuroimaging Initiative, CI = confidence interval, CN = cognitively normal, Cog. = cognitive, DSST = Digit Symbol Substitution Task, FAQ = Functional Activities Questionnaire, Hipp. = hippocampal, HR = hazard ratio, IQ = intelligence quotient, LogMemory = Logical Memory, Matrix = Matrix Pattern Completion, MCI = mild cognitive impairment, MMSE = Mini-Mental State Exam, MoCA = Montreal Cognitive Assessment, RT = Reaction Time, RAVLT = Rey Auditory Visual Learning Test, SD = standard deviation, Tower = Tower Rearranging, TrailsA = Trail Making Test Part A, TrailsB = Trail Making Test Part B, VM = Visual Memory, WM = Working Memory. Forest plot of associations between cognitive measures and DunedinPACNI and hippocampal volume. Effects are presented as standardized beta coefficients with error bars as 95% confidence intervals. Note that we flipped the direction of hippocampal associations to match the direction of DunedinPACNI. Forest plot of associations between cognitive measures and DunedinPACNI and ventricular volume. Effects are presented as standardized beta coefficients with error bars as 95% confidence intervals. Lighter shades represent the effect size for each measure while controlling for the other measure (that is, effect of DunedinPACNI when controlling for hippocampal or ventricular volume, and vice versa). Note that for visualization, the signs of some outcomes were flipped, such that higher scores for all outcomes reflected worse performance or health. Exact effect sizes and sample sizes for each test are presented in Supplementary Table 12, 13). Abbreviations: DSST = Digit Symbol Substitution Task, Hipp. = hippocampus, IQ = intelligence quotient, Matrix = Matrix Pattern Completion, RT = Reaction Time, Tower = Tower Rearranging, TrailsA = Trail Making Test Part A, TrailsB = Trail Making Test Part B, Vent. Effects are presented as standardized beta coefficients with error bars as 95% confidence intervals. Note that we flipped the direction of hippocampal associations to match the direction of DunedinPACNI. Effects are presented as standardized beta coefficients with error bars as 95% confidence intervals. Lighter shades represent the effect size for each measure while controlling for the other measure (that is, effect of DunedinPACNI when controlling for hippocampal or ventricular volume, and vice versa). Exact sample sizes for each test are presented in Supplementary Table 12, 13). a. Cognitive decline prediction among cognitively normal ADNI participants, b. Chronic disease prediction in UK Biobank. C. Mortality prediction in UK Biobank. Gray points indicate hazard ratios for each brain measure while covarying for sex and age at scan. The blue point indicates the DunedinPACNI hazard ratio while also covarying for hippocampal volume. The pink point indicates the DunedinPACNI hazard ratio while covarying for ventricular volume. The orange points indicate the hippocampal and ventricular volume hazard ratios while covarying for DunedinPACNI. Note that we flipped the direction of hippocampal associations to match the direction of DunedinPACNI. Error bars on all plots represent 95% confidence intervals. Exact sample sizes for each test are presented in Supplementary Table 14). Abbreviations: ADNI = Alzheimer's Disease Neuroimaging Initiative, Hipp. = hippocampus, HR = hazard ratio, UKB = UK Biobank, Vent. Raw elastic net regression weights from the DunedinPACNI algorithm. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. DunedinPACNI estimates the longitudinal Pace of Aging from a single brain image to track health and disease. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/rubin-observatory-data-flood-will-let-the-universe-alert-astronomers-10/'>Astronomers Strategize for the Best Problem—More Data Than They Can Handle</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Astronomers Brace for 10 Million Alerts a Night from Rubin Observatory Made from over 1,100 images captured by NSF-DOE Vera C. Rubin Observatory, the video begins with a close-up of two galaxies then zooms out to reveal about 10 million galaxies. Those 10 million galaxies are roughly .05% of the approximately 20 billion galaxies Rubin Observatory will capture during its 10-year Legacy Survey of Space and Time. The universe is a happening place—full of exploding stars, erupting black holes, zipping asteroids, and much more. And astronomers have a brand-new, superpowerful eye with which to see the changing cosmos: the Vera C. Rubin Observatory in Chile. The Rubin Observatory released its first images last week, and they're stunning—vast, glittering star fields that show off the telescope's massive field of view and spectacularly deep vision. But two of the endeavor's most compelling aspects are difficult to convey in any individual image, no matter how spectacular: the sheer amount of data Rubin will produce and the speed with which those data will flood into astronomers' work. “We can detect everything that changes, moves and appears,” says Yusra AlSayyad, an astronomer at Princeton University and Rubin's deputy associate director for data management. Any time something happens in Rubin's expansive view, the observatory will automatically alert scientists who may be interested in taking a closer look. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “It's way too much for one person to manually sift through and filter and monitor themselves.” AlSayyad compares Rubin's data stream to a dashcam or a video doorbell that constantly films everything in its view. “You can't just sit there and watch it,” she says. “In order to make use of that video feed, you need data management.” For Rubin, that means building a static image of the sky—a background template, so to speak—against which any changes will be easy to spot. As the telescope snaps its gigantic photographs, algorithms will first automatically correct for effects such as stray light from the sky and image-blurring atmospheric turbulence. Astronomers love data, but no one has that kind of time in a day. So each individual scientist (amateurs can sign up, too) must first enroll with the Rubin Observatory's so-called alert brokers. Users can request alerts about supernovae or asteroids, for example, then set constraints on just how interesting an event should be to trigger a notification. Such limitations are important because, again, fielding 10 million alerts per night is an untenable prospect for anyone. And that flood will continue for 10 years straight as the Rubin Observatory executes its signature project, dubbed the Legacy Survey of Space and Time (LSST). During its primary mission the Rubin Observatory will take about a thousand images every night, allowing it to scan the entire visible Southern Hemisphere sky every three to four nights. “Ten years ago we were not really seriously thinking of gravitational-wave counterparts, which is all the rage today,” Bianco says. “We truly believe that LSST itself will discover new things, will transform the way in which we think about the universe,” she adds. “If that is true, then we need to enable changes that allow us to capture these new physics, these new phenomena.” But a particularly intriguing case comes from planetary science within our own solar system. All that effort has put more than 1.3 million asteroids in our catalogs, but astronomers expect Rubin to identify perhaps three times that many new objects—practically without trying. When the LSST survey is running at full capacity, alerts for potential newfound asteroids will be sent straight to an international group called the Minor Planet Center, which tends a database of all such space rocks. “We just sort of sit back and these objects will be discovered and reported to us,” says Meg Schwamb, an astronomer at Queen's University Belfast. Schwamb co-chairs the LSST Solar System Science Collaboration and has worked to estimate what the telescope will find in our cosmic neighborhood. And because these space rocks are already out there, rattling through the solar system, Rubin will rack up discoveries quickly, Schwamb and her colleagues predict—with some 70 percent of new objects discovered during the survey's first two years. Not everything Rubin will study is so speedy and unsubtle; the observatory will also be an astonishingly powerful tool for probing the enigmatic dark matter that produces no light yet holds galaxies together and outweighs the normal, familiar matter we know in our daily lives. One way astronomers study this lightless stuff is to measure how dark matter gravitationally warps light from more distant objects. Decades ago Anthony Tyson, now an astrophysicist at the University of California Davis, wanted to do just that. “I proposed a project to [what was then] the biggest telescope, the biggest camera that was in existence, and got turned down,” he recalls. In the short run, however, he took an approach that now seems prophetic. “I decided maybe I should make another application to take the same data but for a different purpose,” he says. He and his colleagues wrote up a different proposal for the same telescope, this time pitching a study of radio-bright plasma jets emanating from around the supermassive black holes at the core of galaxies. And while it won't observe radio light, it certainly will observe oodles of active galactic nuclei—by the tens of millions, in fact, repaying Tyson's slyly earned telescope time many times over. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02012-9'>In situ structure of a bacterial flagellar motor at subnanometre resolution reveals adaptations for increased torque</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 10:03:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The bacterial flagellar motor, which spins a helical propeller for propulsion, has undergone evolutionary diversification across bacterial species, often involving the addition of structures associated with increasing torque for motility in viscous environments. Understanding how such structures function and have evolved is hampered by challenges in visualizing motors in situ. Here we developed a Campylobacter jejuni minicell system for in situ cryogenic electron microscopy imaging and single-particle analysis of its motor, one of the most complex flagellar motors known, to subnanometre resolution. Focusing on the large periplasmic structures which are essential for increasing torque, our structural data, interpreted with molecular models, show that the basal disk comprises concentric rings of FlgP. The medial disk is a lattice of PflC with PflD, while the proximal disk is a rim of PflB attached to spokes of PflA. PflAB dimerization is essential for proximal disk assembly, recruiting FliL to scaffold more stator complexes at a wider radius which increases torque. We also acquired insights into universal principles of flagellar torque generation. This in situ approach is broadly applicable to other membrane-residing bacterial molecular machines. How evolution innovates remains a fundamental question. While innovations in eukaryotes often arise by rewiring existing gene transcriptional netwoks1, examples of the emergence of evolutionary novelty at the molecular-scale focus on small protein complexes2,3,4,5. What is needed is a comprehensive case study of a family of molecular machines that have diversified, so we can infer how their diversity evolved. Bacterial flagella, helical propellers rotated by cell-envelope-embedded rotary motors, are icons of the evolution of molecular novelty6 (Fig. The flagellum, best studied in model organisms Salmonella enterica serovar Typhimurium and Escherichia coli, is composed of a ring of inner-membrane motor proteins (‘stator complexes') that harness ion flux to rotate a large cytoplasmic rotor ring (the ‘C-ring'). Torque is transmitted through a chassis (the ‘MS-ring') and periplasm-spanning axial driveshaft (the ‘rod') to an extracellular propeller structure that generates thrust. Structures of the C-ring7,8,9, parts of stator complexes10,11, MS-ring and rod12,13, and axial structures14,15 have recently been determined from purified subcomplexes. Proton flux through the stator complexes drives rotation of the C-ring, MS-ring, rod and hook/filament. In C. jejuni and other Campylobacterota, a basal disk and periplasmic scaffold have evolved that scaffold a wider ring of additional stator complexes thought to increase motor torque17. b, Wildtype C. jejuni cells used in previous studies17 typically provide 1 flagellar motor per field of view (arrowhead) as compared with c, many motors per field of view in our minicell strain (arrowheads), greatly increasing throughput and reducing sample thickness for higher-quality electron micrograph acquisition. Note that curvature of minicells is comparable to wildtype cells. The 51781 raw micrographs are available from EMPIAR90 (https://doi.org/10.6019/EMPIAR-11580). d, Periplasmic and cytoplasmic features are evident in single-particle analysis 2D classes of manually picked motors. e, Cross-section through an isosurface rendering of a C17 whole-motor 3D reconstruction (deposited as EMD-16723). f, Map from e segmented and exploded along the z axis to highlight component substructures. Because flagella may pre-date the emergence of the bacteria, their age renders understanding how these core structures first evolved challenging. More tractable is the promise of understanding how recent diversifications evolved. Many flagellar motors have recruited additional proteins that scaffold more stator complexes than the ~11 seen in model motors16,17,18. By increasing the number of stator complexes and increasing the radius at which they exert leverage to rotate the C-ring, these adaptations increase the magnitude of the motor's maximum torque output17: while Salmonella and E. coli motors deliver torque up to ~1,300 pN nm, spirochaete motors (with 16 stator complexes and substantially wider C-ring) deliver up to 4,000 pN nm19, and Campylobacterota motors (with 17 or 18 stator complexes and wide C-ring) up to 3,600 pN nm20. Such adaptations probably facilitated species with high-torque motors to inhabit viscous environments such as mucous21. Understanding the architecture of these relatively recently evolved adaptations is a prerequisite to understanding how they evolved. The Campylobacterota, a phylum that includes gastrointestinal pathogens Campylobacter jejuni and Helicobacter pylori17,21, feature some of the most complex motors known, incorporating additional proteins that scaffold their wider ring of stator complexes to generate greater torque than E. coli and Salmonella18. Comparisons of diverse Campylobacterota motors and contextualization against the C. jejuni motor, the best-studied Campylobacterota motor, have suggested a stepwise evolutionary recruitment of these scaffolding proteins18. First, inner-membrane-associated periplasmic proteins (PflA and PflB, from ‘paralyzed flagellum' proteins A and B) were recruited to template assembly of a wider ring of the stator complex motor proteins17. This wider ring could exert greater leverage on a correspondingly wider C-ring to increase the motor's torque, an adaptation that may have enabled lifestyles in the highly viscous gut mucous that many contemporary Campylobacterota species inhabit17,21. Independently, a large (~100-nm-wide) outer membrane-associated ‘basal disk' of FlgP evolved17,18,22, which we recently found is needed for buttressing the motor while the flagellar filament wraps and unwraps from the cell23. We have speculated that these structures were initially independent but subsequently became co-dependent for assembly18. The physical basis for this and the origins of these new proteins remain unclear. Understanding how and why these additional proteins were incorporated into the motor and how they contribute to function requires molecular-scale models, but their size, intimate membrane association and multiple moving parts have hampered structural determination. Cryogenic electron microscopy (cryoEM) subtomogram average structures at 16–18 Å resolution12,24 and structures of purified flagellar motors by single-particle analysis cryoEM lacking dynamic components7,8,9,12,13 do not study the molecular architecture of the additional structures. To address these challenges, we engineered C. jejuni minicells for subnanometre-resolution structure determination of the flagellar motor in situ. The quality of our map enabled us to assemble a molecular model of its evolutionary elaborations. Our model provides insights into the contributions of new proteins, reveals distant homologies, identifies previously unknown components, contextualizes adaptations of pre-existing core machinery and acquires observations informative of how flagellar motors generate rotation. To increase particle number for high-resolution structure determination, we exploited the multiply flagellated, still-motile minicell phenotype produced by a ΔflhG mutation27, but removed the flagellar filament by deleting filament-encoding genes flaA and flaB for centrifugal purification (Fig. The resulting ~200-nm-diameter minicells are more homogeneous than ~350-nm-diameter minicells from Enterobacteriaceae such as E. coli, Shigella and Salmonella28. The polar curvature remains comparable to wildtype cells, meaning motor structures are unperturbed (Fig. We acquired micrographs of minicells for in situ single-particle analysis. Initial two-dimensional (2D) classification revealed features including the stator complexes and their periplasmic scaffold, basal disk, C-ring, MS-ring and rod (Fig. Classification and refinement applying the dominant C17 symmetry17 yielded a reconstruction to 9.4 Å resolution by gold-standard Fourier shell correlation (FSC) using 32,790 particles (Fig. The majority of the periplasmic structures exhibited 17-fold symmetric features consistent with discrete proteins (Fig. The basal disk is composed of concentric rings, but while the first ring featured 17-fold symmetry, subsequent rings were cylindrical averages, indicating that they do not share the 17-fold symmetry of the other periplasmic structures. The symmetry-mismatched core structures were also cylindrical averages. Given that these symmetry-mismatched regions comprise universally conserved structures (such as the P-, L-, MS- and C-rings) and the stator complexes which have already been structurally characterized7,8,9,10,11,12,13, we focused on C. jejuni's relatively recently evolved 17-fold symmetric periplasmic scaffold structures. Focused refinement resolved this region to 7.9 Å (Fig. 2a,b, and Extended Data Figs. Right: 1 asymmetric unit of the periplasmic scaffold. Asterisks denote densities beneath the first 5 rings. Only the innermost ring (dashed box) is 17-fold symmetric; subsequent rings were not part of the focused refinement. 29) structures of FlgP oligomers reveal that the innermost basal disk ring is composed of 51 FlgP monomers as 17 trimers. c, Fit of a FlgP trimer and interaction with the 17-fold symmetric density of the medial disk (asterisks). e, Density of 10 concentric basal disk rings (left) and fitted FlgP models (right). f, The SHS2-like fold of FlgP (dashed box) is shared with dodecin30 (PDBID: 1MOG), Helicobacter pylori Lpp20 (ref. FlgP uniquely features the ring-forming β-hairpin insert. 29) model of FlgQ structurally resembles a 2-FlgP repeat. h, 100 × 100 nm slice through the subtomogram average structure of a motor in which FlgQ was fused to mCherry is indistinguishable from the WT motor (deposited as EMD-17419). i, The C. jejuni LP-rings have comparable diameters to the 26-fold symmetric Salmonella LP-rings. The density map of the Salmonella LP-rings12 (EMD-12183) was low-pass filtered to 15 Å resolution and cylindrically averaged for comparison with the C. jejuni LP-rings (Additional data for EMD-16723). The C. jejuni L- and P-rings are of comparable diameters to those from Salmonella. j, Support for the C. jejuni LP-rings having comparable stoichiometry to E. coli and Salmonella from ~26 steps in flagellar rotation. Bottom: weighted power spectrum of angular position; grey dashed line highlights 26. Inset: x,y-position histogram with density represented by darkness of coloration (Extended Data Fig. The basal disk forms a concave cup that pushes the outer membrane away at greater disk radii (Fig. We focused on the first, 17-fold symmetric ring. These 17 repeats were composed of 3-repeat units, making a ring of 51 similar subunits in 17 trimeric repeats. Knowing that the disk is composed of FlgP17,22, we predicted monomeric and multimeric structures of FlgP, excluding the N-terminal signal sequence and linker22 to the membrane23, using AlphaFold2 (ref. Oligomers laterally associated to form a continuous β-sheet, with interaction of 1 FlgP with the next 2 protomers. Bending the arc of these oligomers demonstrated that 17 trimeric repeats, that is, 51 protomers, fit into the 51 periodic densities of the innermost ring of the basal disk, with a map-model FSC of 9.9 Å at FSC = 0.5 (Fig. Each trimeric repeat also featured an additional medial disk-facing density (Fig. Although we could not discern the symmetries of rings beyond the first, assembly of the disk is entirely reliant upon flgP17. Subsequent rings all have comparable cross-section densities, making them unlikely to be composed of another protein. On the basis of ratios of circumferences, we predict that each subsequent ring adds 11 protomers, explaining the symmetry mismatch with the 17-fold symmetric structures (Fig. Attempts at focused refinement of discrete rings failed, possibly due to insufficient signal from the 11.5 kDa FlgP subunits that lack prominent features to assist alignment. FlgP features a modified SHS2 domain fold from InterPro family IPR024952, with a three-stranded β-sheet and an α-helix spanning one face30 (Fig. Structural similarity searches revealed that this fold is also present in dodecin30, Helicobacter Lpp20 (ref. FlgP, however, features an additional C-terminal helix and long β-hairpin insert between the SHS2 α-helix and the second SHS2 β-strand. This β-hairpin insert extends 35 Å at a ~42° angle to the vertical axis and is the basis of the continuous β-sheet in the inner face of each FlgP concentric ring. We have previously found that FlgQ is required for basal disk assembly, so wondered whether it is also a basal disk component22. We predicted its structure using AlphaFold2 (ref. 29), finding that it resembles a 2-protomer FlgP repeat (Fig. To clarify its location, we fused an mCherry tag to FlgQ and determined a subtomogram average structure. We conclude that FlgQ is a low-abundance or irregular component, or an assembly chaperone. While ascertaining the symmetry of the L- and P-rings was beyond the resolution of our map, cross-sections through L- and P-ring densities showed comparable radii to Salmonella12,13 (Fig. We cylindrically averaged our map by applying arbitrary high-order symmetry during focused refinement and compared these to low-pass filtered structures of similarly cylindrically averaged Salmonella LP-rings. The C. jejuni L- and P-rings have similar width to Salmonella (103% and 104% as wide, respectively) (Fig. To experimentally probe the C. jejuni LP-ring stoichiometries, we looked for steps in flagellar rotation—in Salmonella and E. coli, interactions between the asymmetric rod and 26-fold symmetric LP-rings impose 26 steps on flagellar rotation12. We attached beads to truncated flagella, and upon slowing rotation by de-energizing cells with carbonyl cyanide 3-chlorophenylhydrazone (CCCP), observed 25 to 27 phase-invariant dwell positions (Fig. Taking these structural and biophysical observations together, we conclude that the C. jejuni LP-rings have not expanded to match the symmetry of the basal disk or other more recently evolved components and have a symmetry similar to the 26-fold symmetry of the Salmonella LP-rings. Together with their 17:26 symmetric mismatch, this indicates that the basal disk and P-ring do not form a stable complex, despite their proximity. We neither saw evidence of YecR, which in Salmonella forms a belt around the LP-rings12, nor found a homologue in the C. jejuni genome, although an unidentified adjacent density hints at an as-yet-unidentified analogue (Fig. We speculated that our map might help us understand the medial disk, a lattice between the basal and proximal disks of unknown composition (Fig. Our unpublished transposon insertion sequencing-based infection screen using C. jejuni NCTC11168 (ref. 32) revealed that polarly localized Cj1643 and Cj0892c are required for motility but not flagellar filament assembly, suggesting them as potential candidates. Co-immunoprecipitation with proximal disk component PflA in C. jejuni 81–176 recovered orthologues CJJ81176_1634 and CJJ81176_0901, respectively (Supplementary Table 1), which we renamed PflC and PflD (Paralyzed flagellum C and D) after subtomogram average structures revealed destablization of the proximal and medial disks upon deletion of pflC, and loss of a peripheral cage-like structure between the medial and proximal disks upon deletion of pflD (Fig. Dashed box regions are enlarged in c, d and f. b, Comparing the WT subtomogram average motor structure (100 × 100 nm cross-section of EMD-3150 (ref. 17)) to a pflC deletion (100 × 100 nm cross-section, deposited as EMD-17415) reveals loss of the medial disk (open arrowhead; filled on WT structure), while pflD deletion (deposited as EMD-17416) abolishes assembly of a peripheral post-like density (open arrowhead; filled on WT structure). c, Close-up below the innermost basal disk ring shows a ring of 17 domain-swapped PflC protomers attached to FlgP trimeric repeats. A single PflC (dashed outline) contributes domains to 2 protomeric units. Asterisks denote the interdomain linker. d, View of the medial disk from outside the cell depicting 17 asymmetric units (dashed box highlights 1 asymmetric unit) of 7 PflC protomers and 1 PflD. Inset: differential oligomerization interfaces of PflC protomers. Twofold symmetry axis symbols highlight symmetric dimerization; empty circles represent asymmetric interfaces. See focus in f. e, PflC and PflD interact with known flagellar components. Detected heavy (HC) and light (LC) antibody chains are indicated. Sn1/2, supernatant 1/2; W, wash; E, eluate. f, Top: molecular model of the PflC lattice denoting symmetry elements, enlarged from red box in d. Densities adjacent to every Asn239 denoted by asterisks correspond to a glycosylation site of PflC from a related species. Symmetry elements as in d. Bottom: side view of PflD beneath PflC4,5. g, The predicted structure of PflC (top) highlights common fold with HtrA (bottom), a periplasmic protease (PDB 6Z05 (ref. Left panel aligned to PflC1 in c. Protease and 2 PDZ domains labelled for comparison. PflC is a 364-residue periplasmic protein featuring an N-terminal serine protease domain followed by two PDZ-like domains. 29) model revealed 2 folding regions, PflCN (protease and PDZ1, residues 16–252) and PflCC (PDZ2, residues 265–364), connected by a proline-rich linker. By separately rigid-body docking PflCN and PflCC, we located 17 copies of PflC in the densities projecting from each third FlgP subunit of the first basal disk ring, yielding a map-model FSC of 9.3 Å at FSC = 0.5 (Fig. PflCC docks to the underside of the first ring of FlgP, while PflCN forms the inner band of the medial disk; a density corresponding to the linker bridges adjacent PflC protomers, indicating that the PflCC domain of one chain assembles with the PflCN domain of another in a daisy chain of domain-swapped protomers (Fig. PDZ domains usually interface with binding partners via a hydrophobic pocket, but PflCC interactions are unlikely to be mediated by canonical binding33 as its ligand-binding groove is oriented away from FlgP, lacks the conserved binding loop34, and is predicted to be occupied by residues from its own chain. The remainder of the medial disk is a lattice of α-helices that we could interpret with 17 asymmetric units of 6 additional PflCN protomers in a lattice (Fig. PflCN protomers could be docked into our density map with a map-model FSC of 9.7 Å at FSC = 0.5 (Extended Data Figs. We refer to the subunits in one asymmetric unit as PflC2–7; PflC2–6 form radial spokes with a slight twist, while PflC7 connects adjacent spokes. Pulldowns using FLAG-tagged PflC verified interactions with FlgP, while coIPs of FLAG-tagged proximal disk components PflA and PflB co-purified PflD (Fig. Supporting our modelling of PflC, we noted an additional rodlike density protruding from the location of N239 in all PflC protomers, which in a homolog from a closely related species is glycosylated with an N-linked heptasaccharide35 (red asterisks, Fig. The PflC lattice consists of diverse oligomerization interfaces (Fig. Three types of symmetric dimerization interface have rotational axes perpendicular to the plane of the lattice: one in PflC2:PflC3 and PflC4:PflC5; one in PflC3:PflC4 and PflC5:PflC6; and a third in PflC3:PflC7, although the latter is substantially distorted. 2a, opaque teal in 3a), although the lack of C17 symmetry renders linker helices invisible. These promiscuous self–self interfaces predicted that PflC oligomerizes ex situ. Curiously, however, size exclusion chromatography (Extended Data Fig. 6a,b) and mass photometry (MP) (Extended Data Fig. 6c) of purified PflC revealed mainly monomers. This suggested that the domains of discrete PflC polypeptides might self-associate until the context of the motor facilitates correct assembly. To test this, we removed PflCC, which might inhibit oligomerization. Consistent with our speculation, PflC∆236–349 formed substantially more dimers than full-length PflC (Extended Data Fig. We speculate that interaction of PflCN and PflCC from the same polypeptide prevents cytoplasmic oligomerization, and binding to the basal disk facilitates medial disk assembly only in the context of the assembled motor. This would explain why we have never seen isolated PflC lattices in C. jejuni tomograms. To better understand the origin of the PflC-based medial disk, which is absent from other Campylobacterota species, we performed a DALI search of PflC's predicted structure against the Protein Databank. Our search revealed that its protease:PDZ:PDZ domain architecture is similar to trypsin-like HtrA serine proteases36 (Fig. Indeed, despite 20% and 27% sequence identity between the N- and C-terminal regions of PflC and HtrA, respectively, PSI-BLAST of PflC against the non-redundant protein sequence database returned hits to serine proteases from the first iteration, and a hidden Markov model derived from PSI-BLAST alignments returned four significant hits in the C. jejuni proteome, with the highest confidence to PflC and HtrA (e-values of 2.4 × 10−108 and 3 × 10−57, respectively; HtrA is CJJ81176_1242). The two other hits were also to proteases (CJJ81176_1086 and CJJ81176_0539, with e-values of 6.7 × 10−8 and 3.2 × 10−6, respectively). Because PflCC is connected to PflCN by a flexible linker, we aligned it separately to the C-terminal PDZ domain of HtrA with RMSD between 42 pruned residue pairs (of 64) of 1.425 Å. The region of PflCN corresponding to the active site cleft of the serine protease domain is most divergent from HtrA (Extended Data Fig. 7c) and PflC lacks the catalytic triad residues His119, Asp150 and Ser224: the large loop that His119 occupies has been lost in PflC, Asp150 is replaced by Asn65, and although Ser224 aligns with PflC's Ser122, these residues do not align structurally (Extended Data Fig. The second candidate medial disk component, PflD, is a 162-residue periplasmic protein that pulls down with PflA of the proximal disk (Supplementary Table 1). We inspected the peripheral part of the medial disk adjacent to PflC4 which disappeared when we deleted pflD, and found that a model of PflD was consistent with this density, despite the lower resolution of this area of our map yielding only a modest mean main chain correlation coefficient of 0.63 (Fig. This location is consistent with our co-purification of PflD with both PflA and PflB (Fig. Finally, we sought to interpret our density map of the proximal disk, which contains PflA, PflB and stator complex protein MotB17. Consistent with this, the proximal disk features the extensive short antiparallel α-helical motifs characteristic of the repetitive TPR motifs37,38 predicted for PflA and PflB (Fig. a, Side view of an asymmetric unit of the periplasmic scaffold highlighting the proximal disk (dashed red line). Asterisks denote unassigned PflD-adjacent density (*), E-ring (**) and peripheral cage (***). b, Top view of the proximal disk. Dashed red box denotes the asymmetric unit in c. c, Every asymmetric unit features 1 PflA, 1 PflB, 4 FliL and 1 stator complex (composed of 5 copies of MotA and 2 copies of MotB). PflA (light green) forms spokes whose N-terminal domain interacts with a rim of PflB (dark green) at the periphery of the scaffold. An arc of FliL (red) and periplasmic domain of MotB (pink, residues 68–247) are also evident at lower confidence. Inset: focus on FliL at lower threshold to demonstrate match of 4 FliL models into 4 periodic densities. d, A representative motility agar plate stabbed with WT and fliL::cat demonstrates that fliL knockout has only a minor effect on motility. e, Mass photometry measurements confirm that the PflAB dimer (167 kDa peak) forms in vitro. Inset: 100 × 100 nm cross-section through the subtomogram average density map of the WT motor exhibits PflAC (filled red arrowhead) and PflB densities (filled blue arrowhead). f, Mass photometry shows that deleting the PflA β-sandwich and linker abolishes dimerization with PflB. Inset: 100 × 100 nm cross-section through a density map of the subtomogram average of this mutant reveals a vestigial PflAC density (filled red arrowhead) and loss of PflB (open blue arrowhead) (deposited as EMD-17417), whereas g, a 100 × 100 nm cross-section through a density map of the subtomogram average of a pflA deletion further lacks the vestigial PflAC density (open red arrowhead) (structure from EMD-3160 (ref. PflA was predicted to form an elongated superhelix consisting of 16 TPR motifs connected to an N-terminal β-sandwich domain by an unstructured linker, while PflB was mainly α-helical except two 5-residue β-strands. The predicted structure of a PflAB heterodimer (Extended Data Fig. 4d) indicated that PflA's N-terminal domain and linker wraps around PflB. We flexibly fitted these structures into our map, resulting in a structure in which 17 radial spokes of PflA position a continuous rim of 17 PflBs (Fig. PflA fitted the map with a map-model FSC of 9.1 Å at FSC = 0.5 (Extended Data Fig. Similar to PflC, PflA is a glycoprotein, with N-linked glycans attached at N458 and N497 (ref. 39), and this modelling is validated by densities consistent with N-linked heptasaccharides emanating from our modelled locations of both residues (Fig. The resolution of PflB was lower, presumably because it is more peripheral and flexible, with map-model mean main chain correlation coefficient of 0.71 (Extended Data Fig. To test the prediction that PflAB heterodimer formation is mediated by the linker between the N- and C-terminal domains of PflA binding a TPR-superhelical groove in PflB, with the N-terminal β-sandwich domain wrapping around PflB and the C-terminal α-helical PflA spoke pointing toward the motor axis (Fig. 4c), we measured their interaction using mass photometry. PflAB heterodimerized even at nanomolar concentrations (Fig. 4e), while deleting the linker and β-sandwich domain of PflA (residues 16–168) abolished dimer formation (Fig. A subtomogram average structure of the motor in a PflAΔ16–168 mutant confirmed that PflB was unable to assemble into the motor (Fig. 4e), although the C-terminal end of PflA remained evident, unlike a full pflA deletion17 (Fig. Together with PflAB interaction seen in pulldowns40, we conclude that PflAB dimerization is essential for completion of proximal disk assembly. Although our previous work and the topology of the globular periplasmic MotBC dimer make its location unambiguous17, the resolution of this region was low; this is presumably due to flexibility between this domain and the periplasmic portion of the helical linker between globular MotBC and its N-terminal transmembrane helix, meaning we could only crudely position a MotB model. This is consistent with the MotBC domain not being resolved in recent structures of purified MotA5B2 complexes10,11. The linker itself, however, was evident (Fig. An arc of density partially encircling the periplasmic portion of the helical MotBN linker had similar radius and location as the tertiary structures of complete circles of FliL in other motors41,42; we found that a curved tetrameric homology model of FliL fitted into this arc with a mean main chain correlation coefficient of 0.61 (Fig. Co-immunoprecipitation assays confirmed that FliL is found in pulldowns of PflA and PflB (Supplementary Table 1), suggesting that the FliL arc is augmented by PflB and PflA to scaffold MotB, and explains why PflA and PflB are both required for the high occupancy or static anchoring of stator complexes into the C. jejuni motor17. Indeed, we found that deletion of fliL in C. jejuni had only a modest impact on motility (Fig. 4d) in contrast to H. pylori where FliL is essential, reinforcing that FliL's role is partially fulfilled by PflA and PflB in C. jejuni. The presence of the stator complexes in the C. jejuni structure, in contrast to their absence in Salmonella, indicates either high occupancy or static anchoring, probably mediated by their interactions with PflA, PflB, FliL and unidentified cytoplasmic proteins. We could not assign identities to three remaining densities in the scaffold: the so-called E-ring that spaces the MS-ring from PflA, a cage previously observed in H. pylori41 on the periphery of the PflB rim that extends through the membrane to wrap around the stator complexes, and a small density adjacent to PflD (Fig. 4a, opaque regions, single, double and triple asterisks, respectively). To better understand the conserved torque-generation machinery, we focused on the stator complexes, C-ring and MS-ring. By symmetry expansion and classifying stator complexes from our whole-motor map, we observed a pentameric structure in contact with the C-ring that is directly beneath the periplasmic peptidoglycan-binding domain of MotB (Fig. In cross-section, the dimensions and shape of this density match MotA5 from the purified C. jejuni stator complex11 (Fig. The consistent rotational register of this pentameric density even after symmetry expansion and classification, with a pentameric corner pointing toward the C-ring, indicates that stator complexes are most frequently in this rotational register. a, A 100 × 100 nm cross-section through the subtomogram average of the wildtype C. jejuni motor (from EMD-3150 (ref. 17)) depicting locations of stator and rotor cross-sections illustrated in b and d. b, Cross-section through the whole-motor map just beneath the outer membrane shows 17 circular densities at the expected location of MotA. c, Top: focused refinement of the stator complexes reveals pentameric densities that, in cross-section (bottom), have the distinctive thimble-like shape of a MotA pentamer (from PDB 6YKM11, with N-terminal helices of MotB removed). d, Cross-section through the C. jejuni C-ring showing 38-fold periodic structure (deposited as EMD-19642). Labels E and F denote cross-sections depicted in respective panels. f, Cross-section through the edge of the C-ring showing post-like densities corresponding to the periodicity shown in d (arrowheads) as have been reported in the Salmonella C-ring. Top: cross-section through a composite map of the Salmonella MS-ring (middle top dashed box, from EMD-12195 (ref. 12)) and C-ring (two lower dashed boxes, from EMD-42439 (ref. 8)) rotor components depicting the 51-Å-radius MS-ring β-collar. Both maps were low-pass filtered to 15 Å resolution and cylindrically averaged for like-for-like comparison with Bottom: cross-section through a composite map of the whole-motor C. jejuni map, with superimposed corresponding cross-sections through focused, cylindrically averaged C. jejuni MS-ring and C-ring maps (deposited as Additional data for EMD-16723), highlighting the wider 62-Å-radius β-collar. We wondered how the C. jejuni rotor components, that is, the C-ring, MS-ring and rod, have adapted to interact with a wider ring of stator complexes. The C. jejuni C-ring is wider than that of Salmonella17; to ascertain its stoichiometry, we determined its structure by subtomogram averaging. To remove the strong 17-fold signal of MotA, we used a motile C. jejuni mutant whose stator complexes have lower occupancy than WT motors. Our subtomogram average revealed a 38-fold periodic structure whose vertical post-like architecture resembles that of the near-atomic-resolution Salmonella C-ring7,8,9 (Fig. C-ring diameter is reliant on templating on the C terminus of the MS-ring protein FliF8,43. To examine possible mechanisms behind templating a wider C-ring, we compared the C. jejuni MS-ring to that from Salmonella. The distinctive FliF β-collar has a radius of ~51 Å in Salmonella (Fig. Focused refinement of the C. jejuni MS-ring with imposed C38 symmetry expected from 1:1 stoichiometry of FliF and FliG43,44 demonstrated a 62-Å-radius FliF β-collar (Fig. We could not resolve azimuthal features corresponding to discrete FliF subunits, but imposition of other arbitrary high-order symmetries did not alter the radius of FliF's β-collar. We conclude that the wider C-ring is achieved by C. jejuni assembling a wider MS-ring, which in turn templates the wider C-ring. This study provides a near-complete structural inventory of the proteins incorporated into the C. jejuni motor during evolution of higher-torque output and adaptations of pre-existing components, providing insights into evolutionary origins and the universal principles of flagellar rotation (Fig. This work lays the foundation for understanding where these new proteins came from and how they became incorporated into a pre-existing machine. A partial cut-away schematic of the structure of the C. jejuni flagellar motor contextualizing the protein components modelled in this study. The basal disk is formed of FlgP; the medial disk of PflC and PflD; and the proximal disk of PflA, PflB and FliL together with stator complex components MotA and MotB. We previously inferred from E. coli and Salmonella that a single stator complex exerts a force of 7.3 pN17. Given the approximation that torque from multiple stator complexes is additive45, the number of stator complexes, together with our force estimate and the radius at which stator complexes exert force on the C-ring make predictions that nicely explain observations of 3,600 pN nm torque for Campylobacterota flagellar motors (our structure-based prediction is 3,288 pN nm) versus observations ranging from 1,260 to 2,000 pN nm45,46 for E. coli and Salmonella flagellar motors (our structure-based prediction is 1,606 pN nm). For the C. jejuni motor to build this wider ring of additional stator complexes to produce its higher torque, the spoke-and-rim scaffold of PflA and PflB, anchored by PflC and PlflD, positions stator complexes in their wider ring. This scaffold facilitates the static positioning (or high occupancy) of the C. jejuni stator complexes17; the radius of the MS-ring is correspondingly larger than that of Salmonella to template a wider C-ring to maintain contact with the larger stator ring. The structures of PflA and PflB hint at their possible origins. The widespread nature of TPR motifs would have made them available for co-option to form PflA or PflB. Intriguingly, other flagellar proteins are homologous to parts of the widespread Tol/Pal cell-envelope maintenance system6, and the Tol/Pal component YbgF features TPR motifs47, making it tempting to contemplate a scenario in which PflA and PflB originated from another Tol/Pal co-option. Our structures support our previous speculation that PflA may be widespread among other species such as Bdellovibrio18. In C. jejuni, the PflAB-containing proximal disk is linked to the basal disk by the PflC-composed medial disk, while other species lack a medial disk and have separate proximal and basal disks. PflC's shared domain organization and structure with HtrA-family enzymes suggests that the medial disk originated as a co-option of an ancestral member of the HtrA family48. Indeed, HtrA family proteins are periplasmic, meaning that they already co-localized with the flagellar motor. Curiously, HtrA also forms higher-order oligomers36, suggesting that this oligomerization tendency may have been a pre-existing property easily exapted for its role as a structural scaffold. The basal disk is comparatively ancient. While the basal disk protein FlgP comes from a broad protein family, ring oligomerization in the family correlates with the subset of sequences that feature a sequence insertion between the SHS2 α-helix and the second β-strand, as seen with outer membrane-associated rings formed by FlgT49. Curiously, the basal disk from Wolinella has been proposed to form a spiral, not concentric rings50, raising the possibility that FlgP forms concentric rings only in the presence of a medial disk. Our study also provides clues to the universal mechanism of flagellar rotation. In our averaged structure, MotA5 pentamers all have similar rotational orientations relative to the C-ring, meaning that the majority of MotA pentamers are in this orientation, as averaging an unbiased range of orientations would yield a circle. This, on face value, is inconsistent with the dominant ‘cogwheel' model of the flagellar rotation mechanism, which posits that rotation of a MotA5 pentamer around its MotB anchor enables the pentamers to act as cogs that drive rotation of the larger C-ring cogwheel. This meshing means that MotA5 pentamers would always be engaged with the C-ring, that is, they would have a ‘high duty ratio'. Because the number of small cogs (17 in C. jejuni, 11 in E. coli and Salmonella) has a symmetry mismatch with the number of teeth in the big cog (38 in C. jejuni, 34 in E. coli and Salmonella), once assembled, each of the small cogs, which are evenly distributed around the circumference of the larger cog, would need to be rotationally orientated differently to enable all to mesh simultaneously with the symmetry-mismatched teeth of the larger cog. Thus in an averaged structure, MotA5 would appear as smooth circles, inconsistent with our observation. We can imagine two explanations for this inconsistency. First, the symmetry mismatch might be capacitated by structural tolerances, such as flexibility in the so-called FliGC ‘torque helix' (a ‘tooth' of the large cog) that interacts with MotA, which would enable all pentamers to retain the same orientation while the FliGC cogteeth distort to facilitate maintaining the interface. Previous models of high-duty-ratio stator complexes cite tolerance from an elastic linker from peptidoglycan through MotB to MotA51, although this could not be universal, as species such as C. jejuni have stator complexes held rigidly in proteinaceous cages. With such tolerances, all cogs would rotate in unison to drive a single rotational step of the C-ring, all starting and finishing a work cycle in the orientation that we observe. Alternatively, the tolerances might be insufficiently broad to enable engagement of all stator complexes, in which case the duty ratio might be lower than a naïve interpretation of the cogwheel model suggests. A second explanation is that the cogwheel model does not adequately describe the mechanism underlying flagellar rotation. Indeed, a recent study hints that previous results suggesting a high stator complex duty ratio derived from artefacts from stator complex dissociation at low load, and that stator complexes actually have a low duty ratio52. While our single-particle analysis approach achieves substantially higher resolution than previous studies, our approach is not without limitations. Our study focused on the 17-fold rotationally symmetric periplasmic scaffold, but flagella feature diverse symmetry-mismatched subcomponents. The approach can in principle identify arbitrary symmetries, but we did not resolve other substructures such as the C-ring, which instead required a subtomogram averaging approach, probably due to excess noise owing to the size, heterogeneity and the in situ context. Our approach builds a foundation to study molecular evolution mechanisms and provides information on how torque is generated. Our approach confirms that imaging protein machines in situ can provide subnanometre resolution without sectioning, and lays a foundation for future studies expanding to other symmetry elements and structures. Plasmids and strains generated in this study (see Supplementary Table 3) are available from the lead author. C. jejuni 81–176 or NCTC11168 were cultured from frozen stocks on Mueller–Hinton (MH) agar (1.5% w/v) supplemented with trimethoprim (10 μg ml−1) (MHT) for 1–2 days at 37 °C under microaerophilic conditions (5% O2, 10% CO2, 85% N2) in a Heracell 150i trigas incubator (Thermo Fisher). Additional antibiotics were added to the agar medium when required: kanamycin at 50 μg ml−1, streptomycin at 2 mg ml−1. All 81–176 mutants were constructed in DRH212 (ref. 60), a streptomycin-resistant derivative of C. jejuni 81–176, which is the reference wildtype strain in this work unless otherwise stated. When working with E. coli, cultures were grown at 37 °C on Luria-Bertani (LB) agar plates (1.5% w/v) or in LB medium with agitation, both supplemented with carbenicillin at 100 μg ml−1. Please see Supplementary Table 3 for details of strains and plasmids used in this study. The minicell (∆flhG ∆flaAB) and PflA truncation (pflAΔ18–168) strains were constructed as described previously60,61. Briefly, aphA-rpsLWT cassettes flanked by ~500 bp overhangs with homology to the targeted chromosomal loci and EcoRI sites at the 5' and 3' termini were synthesized by ‘splicing by overlap extension' PCR. Linear DNA fragments were methylated at their EcoRI sites with EcoRI methyltransferase (New England Biolabs) and transformed into C. jejuni using the biphasic method62. Transformants were selected for on MH agar supplemented with 50 µg ml−1 kanamycin. Replacement of the aphA-rpsLWT with the desired mutation was achieved using the same method, but with transformants being selected for on MH agar supplemented with 2 mg ml−1 streptomycin sulphate. Kanamycin-sensitive, streptomycin-resistant transformants were single-colony purified and checked by Sanger sequencing (Source Bioscience). For the minicell background, in-frame deletion of flhG left the first and last 20 codons intact, while the ∆flaAB allele spanned from 20 base pairs upstream of the flaA translational start site to codon 548 of flaB. To construct the C. jejuni fliL mutant, we made a cat insertional knockout and confirmed absence of polar effects. To preserve expression of the essential acpS gene downstream of fliL, we constructed a fliL mutant that disrupted fliL with an antibiotic-resistance cassette containing an intact flaA promoter positioned to maintain expression of acpS. First, the fliL locus from C. jejuni 81–176 was PCR amplified with an HpaI site engineered within the fliL coding sequence. This fragment was then cloned into the BamHI site of pUC19 to create pDAR1712. The flaA promoter and start codon were PCR amplified from C. jejuni 81–176 and cloned into the XbaI and BamHI sites of pUC19 to create pDAR2039. The chloramphenicol-resistance cassette containing cat was digested as a PstI fragment from pRY109 and cloned into PstI-digested pDAR2039 to create pDAR2045. The cat-flaA promoter was then digested from pDAR2045 as an EcoRI-BamHI fragment, treated with T4 DNA polymerase to create blunt ends and cloned into the HpaI site of pDAR1712 to create pDAR2072. pDAR2072 was verified to contain the cat-flaA promoter in the correct orientation to maintain expression of acpS. DRH212 was then electroporated with pDAR2072 and chloramphenicol-resistance transformants were recovered. Colony PCR verified creation of a fliL mutant (DAR2076). Deletion mutants of C. jejuni NCTC11168 were constructed by double-crossover homologous recombination with an antibiotic-resistance cassette to remove most of the coding sequence using overlap PCR products. As an example, deletion of cj1643 (pflC) is described. First, ~500 bp upstream of the cj1643 start codon was amplified using CSO-3359/3360 and ~500 bp downstream of the cj1643 stop codon was amplified with CSO-3361/-3362 from genomic DNA (gDNA) of the wildtype strain (CSS-0032). A non-polar kanamycin-resistance cassette (aphA-3, KanR) was amplified from pGG1 (ref. To fuse the up- and downstream regions of cj1643 with the resistance cassette, the three fragments were mixed and subjected to overlap-extension PCR with CSO-3359/3362. PCR products were electroporated into the WT strain as previously described64. The final deletion strain (CSS-4087; NCTC11168 ∆cj1643) was verified by colony PCR with CSO-3363/HPK2. To fuse sfgfp to the penultimate codon of cj0892c (pflD), its coding sequence was first amplified with CSO-3611/3612, digested with NseI/ClaI and inserted into similarly digested pSE59.1 (ref. 65), which had been amplified with CSO-0347/CSO-0760, to generate pSSv106.5, where cj0892c transcription was driven from the metK promoter. The fusion of rdxA::PmetK-cj0892c-sfGFP was amplified from pSSv114.1 with CSO-2276/2277 and introduced into the rdxA locus of ∆cj0892c (CSS-4081) by electroporation. Colony PCR was also used to confirm retention of the original deletion with CSO-3343 and HPK2. Similar to construction of deletion mutants, C-terminal epitope-tagged strains were generated by homologous recombination at the native locus by electroporation of a DNA fragment. As an example, 3×FLAG tagging of PflA (CSS-5714) is described. The upstream fragment was amplified with CSO-4224 and CSO-4225 from C. jejuni NCTC11168 WT gDNA. The downstream fragment was amplified using CSO-4226 and CSO-4227. Next, a three-fragment overlap PCR using CSO-4224 and CSO-4227 was performed and the resulting PCR product was electroporated into CSS-4666. The obtained clones were validated by PCR using CSO-4223 and HPK2 and by sequencing using CSO-4223. The 3×FLAG with a non-polar kanamycin-resistance cassette was amplified from csrA::3×FLAG-aphA-3 (ref. To construct a FlgQ-mcherry fusion protein for expression, a 76-bp DNA fragment containing the cat promoter and start codon with an in-frame BamHI site from pRY109 was amplified by PCR and cloned into the XbaI and XmaI sites of pRY112 to create pDAR1003. PCR was then used to amplify mcherry from codon to the stop codon, which was then inserted into XmaI and EcoRV sites of pDAR1003 to create pDAR1006. This plasmid contains a start codon that is in-frame with DNA for BamHI and XmaI sites followed by the mcherry coding sequence. Primers were then designed and used for PCR to amplify flgQ from codon 2 to the stop codon from C. jejuni 81–176. This fragment was inserted in-frame into the BamHI site of pDARH1006 to create pDRH7476, which was then conjugated into DRH2071 to result in DRH7516. The minicell-enriched supernatant was removed and spun in a tabletop microcentrifuge at 21,000 × g for 5 min to pellet the minicells. The pellet was then resuspended to a theoretical optical density (OD)600 of ~15. Minicells were vitrified on QUANTIFOIL R0.6/1 or R1.2/1.3 holey carbon grids (Quantifoil Micro Tools) using a Vitrobot Mark IV (Thermo Fisher). For electron cryo-tomography and subtomogram averaging, whole cells were grown on MHT agar, restreaked on fresh plates and grown overnight before use. Freshly grown cells were suspended into ~1.5 ml of PBS buffer and concentrated to an approximate theoretical OD600 of 10 by pelleting at 1,500 × g for 5 min on a tabletop microcentrifuge and resuspending appropriately. Of the concentrated cell sample, 30 μl was mixed with 10 nm gold fiducial beads coated with bovine serum albumin (BSA). Of this mixture, 3 μl was applied to freshly glow-discharged QUANTIFOIL R2/2, 300 mesh grids. Grids were plunge frozen in liquified ethane–propane using a Vitrobot mark IV. Micrographs of the minicell sample were collected using 300 keV Thermo Fisher Titan Krios TEMs across two sessions using EPU acquisition software. The first dataset was collected on a microscope with a Falcon III direct electron detector (Thermo Fisher), the second dataset using a K2 direct electron detector equipped with a GIF energy filter (Gatan), using a slit width of 20 eV. Due to our large particle size relative to that of the holes, we collected one shot per hole. Gain correction was performed on the fly. Details of data-collection parameters are described in Extended Data Table 1. Tilt series of motors in pflAΔ18–168 were collected using a 300 keV Titan Krios TEM (Thermo Fisher) equipped with a K2 direct electron detector and a GIF energy filter (Gatan) using a slit width of 20 eV. Data were collected in Tomography 5 (Thermo Fisher) using a dose-symmetric tilt scheme across ±57° in 3° increments. We used a dose of 3 e− Å−2 per tilt, distributed across 4 movie frames. Pixel size was 2.2 Å and defocus range from −4.0 to −5.0 μm. To determine the C. jejuni C-ring architecture, tilt series of 194 motors in DRH8754 were collected using a 200 keV Glacios TEM (Thermo Fisher) equipped with a Falcon 4 direct electron detector and a Selectris energy filter (Thermo Fisher) using a slit width of 10 eV. Data were collected in Leginon automated data-collection software using a dose-symmetric tilt scheme across ±51° in 3° increments. We used a dose of 2.7 e− Å−2 per tilt, distributed across 5 movie frames. All other tilt-series datasets were acquired on a 200 keV FEI Tecnai TF20 FEG transmission electron microscope (FEI) equipped with a Falcon II direct electron detector camera (FEI) using Gatan 914 or 626 cryoholders. Tilt series were recorded from −57° to +57° with an increment of 3° collected defocus of approximately −4 μm using Leginon automated data-collection software at a nominal magnification of ×25,000 and were binned two times. Cumulative doses of ∼120 e− Å−2 were used. Overnight data collection was facilitated by the addition of a 3 litre cold-trap Dewar flask and automated refilling of the Dewar cryoholder triggered by a custom-written Leginon node interfaced with a computer-controlled liquid nitrogen pump (Norhof LN2 Systems). Movie frames were aligned and dose weighted according to exposure, as implemented in MotionCor2 (v.2.1)67. All subsequent processing was done in RELION (3.1)68,69. Contrast transfer function (CTF) correction was performed using CTFFIND4 (ref. Flagellar motor positions were picked manually, yielding 79,287 particle coordinates for the K2 dataset and 14,605 particle coordinates for the F3 dataset. The two datasets were first processed separately in RELION 3.1 before merging for a final round of refinement. For the K2 dataset, 79,287 particles were extracted at a box size of 800 px. A round of 2D classification removed junk and membrane particles, and an initial model was created using these particles with imposed C17 symmetry, which is known from past structural characterization of the motor by subtomogram averaging17. A round of mask-free 3D classification and refinement with applied C17 symmetry produced the first consensus refinement. A total of 27,164 particles were then re-extracted, centring on the periplasmic structures. After another round of 3D classification, 19,736 particles were refined in C17 symmetry to produce a whole-motor reconstruction at 9.88 Å using gold-standard refinement. For the F3 data, 14,605 particles were extracted at a box size of 1,000 px rescaled to 500 px. They underwent 2D classification to remove junk, 3D classification and refinement to arrive at an initial consensus 3D structure. The particles were again recentred on the periplasmic structures and underwent another round of refinement. Finally, the 13,054 particles were re-extracted at an unbinned 1,000 px box size for a final round of refinement. The two recentred refined datasets were merged, assigning them different RELION 3.1 optics groups, and refined to a global resolution of 9.36 Å (32,790 total particles). Signal subtraction was used to further refine the structure of the periplasmic scaffold. A mask encompassing the regions of interest was made by segmenting and smoothing the whole-motor map using UCSF Chimera (1.16)71,72 and its Segger plugin73, binarizing and adding a soft-edge in RELION 3.1. This mask was used to computationally remove signal outside of the periplasmic regions of interest, as implemented in RELION 3.1. The signal subtraction and subsequent masked classification and refinement were conducted for the combined dataset, as well as K2 and F3 datasets separately. The periplasmic scaffold map was post processed using LAFTER74 as implemented in the CCP-EM 1.6.0 software suite75 to suppress noise and enhance signal between the half maps. The LAFTER-filtered map of the scaffold was used for docking and modelling of periplasmic regions. After convergence of the full-motor refinement in C17 symmetry, focused refinement of the MS-, LP- and C-rings was carried out using a lathed map lacking azimuthal features. The C17 map was lathed by applying C360 symmetry using EMAN2, producing a map with axial and radial features but no azimuthal features. This map was used as a reference for asymmetric refinement, so that no bias towards a particular cyclic symmetry could be imposed. For the LP-ring, a tubular mask covering the LP-ring was used for signal subtraction and focused refinement. Subtracted particles were recentred on the LP-ring in a smaller 256-pixel box. A recentred subvolume of the full lathed map was used for the initial reference. Refinement with imposition of C26 symmetry produced the LP-ring localized reconstruction. For the MS-ring, a tubular mask covering the MS-ring was used for signal subtraction and focused refinement. Subtracted particles were recentred on the MS-ring in a smaller 256-pixel box. A recentred subvolume of the full lathed map was used for the initial reference. Refinement with imposition of C38 symmetry produced the MS-ring localized reconstruction. For the C-ring, a toroidal mask covering the C-ring was used for signal subtraction and focused refinement against the full-size lathed motor map. Refinement with imposition of C38 symmetry produced the C-ring localized reconstruction. This map was then post processed and sharpened with an empirically determined B factor of −1,200 Å−2. From the full-motor refinement in C17 symmetry, particles were symmetry expanded and rewindowed into a smaller 360-pixel box centred on a stator complex, with subtraction of the surrounding signal using a spherical mask encompassing a triplet of neighbouring stator complexes. Classification without alignment produced the stator maps shown. Fiducial models were generated and tilt series were aligned for tomogram reconstruction using the IMOD package76. Tomo3D77 was used to reconstruct tomograms with the SIRT (Simultaneous Iterative Reconstruction Technique) method due to approximately 1 particle per tomogram. All steps were automated by in-house custom scripts. Subtomogram averaging was performed using the Dynamo package78 unless otherwise stated. Motors were picked using 3dmod from the IMOD suite and imported into Dynamo as ‘oriented particles' using an in-house script, and subtomograms were extracted for averaging. For each structure, an initial model was obtained by reference-free averaging of the oriented particles, with randomized Z-axis rotation to alleviate missing wedge artefacts. This initial model was used for a first round of alignment and averaging steps, implementing an angular search and translational shifts, with cone diameter and shift limits becoming more stringent across iterations. The resulting average was used as a starting model for a round of masked alignment and averaging. In this round, custom alignment masks were implemented, focusing on the periplasmic and inner-membrane-associated parts of the motor. This excluded dominant features that would otherwise drive the alignment, most prominently the outer membrane and extracellular hook. A 17-fold rotational averaging was applied. Tomograms were CTF corrected using IMOD, motors picked using 3dmod and subtomograms extracted for averaging in PEET. After an initial C1 whole-motor alignment, a subsequent alignment recentred on the tightly masked C-ring was performed. To capitalize on the redundancy of C-ring archecture and to increase the number of particles, each motor was then symmetry expanded so that the C-ring beneath each of the 17 asymmetric units could be treated as a separate particle. Subsequent alignment and classification to remove poorly aligned particles yielded the final structure. 29), was used to create structural models of PflA, PflB, a dimer of PflA-PflB, PflC, PflD, FlgP (as a monomer, trimer and heptamer), FlgQ, FliL and a dimer of MotBC (with transmembrane residues 1–67 removed). The PflAB dimer model was created by merging two predicted structures: that of a dimer of PflA residues 16–455 and PflB residues 113–820, and full-length PflA. For regions where α-helices and β-sheets were resolved, we refined our AlphaFold-predicted protein models into the scaffold map using ISOLDE82 in UCSF Chimera X 1.6. We imposed torsion, secondary structure and reference distance restraints in our modelling due to resolution limitations. Models of FlgP multimers showed that each subunit interacts with subunits i − 1 and i − 2. To avoid artefacts due to this, a heptamer of FlgP was docked into the innermost basal disk ring and then refined into the map. Two subunits were then removed from each end, resulting in a fitted FlgP trimer. The PflC model was first separated into three domains, PflCN (residues 16–252), linker (253–263) and PflCC (264–364). Each domain was rigidly docked into its appropriate density, the three chains merged and the resulting protein was fitted using ISOLDE to resolve clashes or poor geometry at merging points. Six copies of PflCN were rigidly docked into an asymmetric unit of PflC lattice in the medial disk, forming PflC2–7. PflD, MotB and an arc of FliL were rigidly docked into the LAFTER-filtered scaffold map. We used ‘phenix.validation_cryoem' of the Phenix package83 to evaluate map and model quality for PflA, PflB, PflC, PflC2–7, PflD, FlgP trimer, MotB dimer and FliL. For each model, a soft mask of the region was first created and applied to the map. The resulting masked volume and corresponding structural model were input to ‘phenix.validation_cryoem' which generated map-model FSC plots or cross-correlation scores. WT C. jejuni and DAR2076 were grown from freezer stocks on MH agar containing trimethoprim for 48 h in microaerobic conditions at 37 °C. Strains were restreaked on MH agar containing trimethoprim and grown for 16 h at 37 °C in microaerobic conditions. After growth, strains were resuspended from plates and diluted to an OD600 of 0.8. Strains were then stabbed in MH motility agar (0.4% agar), incubated at 37 °C in microaerobic conditions for 30 h and assessed for migration from the point of inoculation. The C. jejuni bead assay strain was constructed by deleting flaA (to shorten the flagellar filament), making a cysteine substitution in flaB (to allow for filament biotinylation via a malemide-PEG2-biotin linker (Thermo Scientific)) making an alanine substitution in flhF (which moves motors to a subpolar position permitting better bead coupling), and deleting cheY (to prevent motor switching). From frozen stocks, this strain was cultured on MH agar (1.4% w/v) supplemented with trimethoprim (10 μg ml−1) overnight at 37 °C in a pouch with a gas-controlling sachet (EZ CampyPouch System BD, 260685) and a damp paper towel. The growth from this culture was restreaked onto another plate and grown in the same conditions overnight. Passages to new plates on the following 2 days were optionally made to propagate the culture for multiple days of experiments. After growth, bacteria were washed off the plate with MH broth and washed twice with PBS (centrifuged for 2 min at 8,900 × g) to create a 500 μl suspension of OD600 = 2. Next, 25 μl of 20 mM malemide-PEG2-biotin linker in PBS was added and left to react for 10 min. The bacteria were then washed twice more in PBS. A custom tunnel slide composed of a slide with two drilled holes, a parafilm spacer and another glass slide was constructed and connected to a peristaltic pump (Lambda Multiflow) to draw flow through the slide. Poly-l-lysine solution (Sigma-Aldrich, P4707) was flowed into the slide channel and left for 5 min to coat the glass surface. Excess poly-l-lysine was flushed with PBS. Bacteria were added and left for 5 min to settle on the surface. Unattached bacteria were flushed sequentially with PBS and MH broth. Beads (1,300 nm diameter, streptavidin-coated) (Sigma-Aldrich, 49532) were washed once in PBS and dispersed in MH broth. Excess beads were flushed away with MH broth. Spinning beads were observed with a bright-field microscope with a high numerical aperture (NA) objective (×100, 1.46 NA, Zeiss, 420792-9800-720) and recorded with a high-speed scientific complementary metal-oxide semiconductor camera (Optronis CL600X2-FULL-M-FM). Positions of slowly rotating beads attached to cells treated with CCCP at concentrations of 0–5 μM in MH broth revealed preferred phase-invariant dwell positions. C. jejuni proteins for recombinant expression in E. coli were cloned into the pLIC plasmid backbone, which confers resistance to ampicillin and places the gene of interest under an isopropyl β-d-1-thiogalactopyranoside (IPTG)-inducible T7 promoter for high levels of controlled expression. We used WT C. jejuni genomic DNA as template for gene amplification (extracted using the Wizard genomic DNA purification kit by Promega), and the Gibson Assembly method84 to seamlessly assemble all plasmid constructs. For all constructs, primer pairs were designed to amplify (1) the pLIC backbone and (2) the gene to be expressed, while also introducing a 25~30 bp complementary overlap between the two fragments. The pLIC plasmid primers also introduced an N-terminal hexahistidine tag. After vector linearization and purification of PCR product, it was digested with DpnI (New England Biolabs) to remove template vector. The resulting linear DNA fragments were assembled using the Gibson Assembly master mix (New England Biolabs). Of the mix, 5 μl was added to 15–20 fmol of linearized vector and 4× excess of insert, and topped up to 10 μl with double-distilled water (ddH2O). The tube was incubated at 50 °C for 15 min and kept on ice until transformation. Of the diluted Gibson mix, 2 μl was added to 25 μl of chemically competent E. coli DH5α and transformed using the heat shock method85. The entire volume of the tube was then plated onto a carbenicillin-supplemented LB agar plate. After confirmation by Sanger sequencing (Source Bioscience), each assembled construct was isolated from the cloning strain (QIAprep Spin Miniprep kit, QIAGEN) and transformed into E. coli BL21(DE3) for recombinant overexpression. All proteins encoded on pLIC expression vectors were purified using the same protocol. A small (5 ml) overnight liquid culture of E. coli BL21(DE3) carrying the appropriate expression vector was prepared and diluted 1:50 in 1,000 ml of LB medium. Shaking at 37 °C, the culture was grown to OD600 of 0.4–0.6, after which protein expression was induced by addition of 0.5 mM IPTG. All subsequent steps were done on ice using buffers chilled to 4 °C. Cells were lysed using a LM10 Microfluidizer Processor cell disrupter (Analytik) at 15,000 psi. The resulting supernatant was filtered through a 0.45 μm syringe filter (Whatman). A 5 ml HisTrap HP affinity chromatography nickel column (Cytiva) was first equilibrated with wash buffer. Supernatant was loaded onto the column with a peristaltic pump at a flow rate of 3 ml min−1 The column was washed with 50 ml of wash buffer and then transferred onto a Fast protein liquid chromatography system (Bio-Rad). Then, protein was eluted from the column using a high-imidazole buffer (50 mM Tris-HCl, 100 mM NaCl, 500 mM imidazole, pH 7.5) at a flow rate of 2 ml min−1 using ‘reverse flow'. The purified protein was kept at 4 °C or flash frozen in liquid nitrogen for longer-term storage before characterizing them by mass photometry. Analytical size exclusion of PflC and PflCN (∆236–349) was performed with an ENrich SEC 650 column (Bio-Rad), equilibrated with 1× PBS at a flow rate of 0.1 ml min−1 and a total sample injection volume of 400 µl. Absorption was recorded at 280, 220 and 495 nm to follow elution profiles and plotted using GraphPad Prism. C. jejuni PflC (CJJ81176_1634) was used as seed for a PSI-BLAST search86 against the NCBI clustered non-redundant protein sequence database (nr_clustered, 5 December 2024) using default settings. After four iterations using default settings, the top 500 hits were used to build an HMM using hmmbuild from the hmmer 3.3 package87. This HMM was then used with hmmsearch to search predicted proteins from C. jejuni 81–176 (ASM1552v1). Microscope coverslips (24 × 60 mm, Carl Roth) and CultureWell Gaskets (CW- 50R-1.0, 50-3 mm diameter × 1 mm depth) were cleaned with alternating ddH2O and 100% isopropanol washes, then dried roughly with pressurized air and left to dry further overnight at room temperature. Before use, gaskets were assembled onto coverslips and placed on the lens of a OneMP mass photometer (Refeyn) with immersion oil. For each measurement, a gasket was first filled with 18 μl of PBS buffer and the instrument was focused. Then, 2 μl of sample was added to the droplet and rapidly mixed by pipetting. Measurements were then started using AcquireMP v.1.2.1 (Refeyn). Mass photometry data were processed and analyzed in DiscoverMP software v.1.2.3 (Refeyn). Measurements were conducted using affinity chromatography-purified proteins diluted to 200–800 nM, calculated from absorption at 280 nm. For measurements of hetero-oligomers, the different proteins were first combined and mixed in a separate tube and subsequently applied to the PBS droplet. MP measurements were calibrated against molecular masses of commercial NativeMark unstained protein standards (Thermo Fisher). A volume of 1 μl of NativeMark was diluted 30-fold in PBS and 2 μl of this solution was added to 18 μl PBS for measurement. Co-purification of FlgP or PflD-sfGFP was investigated by western blot (WB) analysis using FlgP-specific antisera22 or an anti-GFP antibody (Roche, 11814460001, RRID: AB_390913), respectively. In brief, strains were grown to an OD600 of 0.6, and 60 OD600 of cells were collected (4,600 × g, 20 min, 4 °C) and washed in buffer A (20 mM Tris-HCl pH 8, 1 mM MgCl2, 150 mM KCl, 1 mM dithiothreitol). In parallel, 1 OD600 of cells was collected as ‘culture' control and boiled in 1× protein loading buffer (PL; 62.5 mM Tris-HCl, pH 6.8, 100 mM dithiothreitol, 10% (v/v) glycerol, 2% (w/v) SDS, 0.01% (w/v) bromophenol blue; 8 min at 95 °C, shaking at 1,000 r.p.m.). Next, 60 OD600 cell pellets were lysed with a FastPrep system (MP Biomedical, matrix B, 1 × 4 m s−1, 10 s) in 1 ml lysis buffer [buffer A including 1 mM phenylmethylsulfonyl fluoride (Roche), 20 U DNase I (Thermo Fisher), 200 U RNase Inhibitor (moloX and Triton X-100, 2 µl ml−1 lysis buffer)]. Cleared lysates (14,500 × g, 10 min, 4 °C) were incubated with 35 µl anti-FLAG antibody (Sigma-Aldrich, F1804-1MG, RRID: AB_262044) for 30 min at 4 °C with rotation. Before and after incubation, a 1 OD600 aliquot was taken aside as lysate and supernatant 1 samples. Lysates with anti-FLAG antibody were then incubated for an additional 30 min (4 °C, rotating) with 75 µl per sample pre-washed (3× in buffer A) Protein A-Sepharose beads (Sigma-Aldrich, P6649). Afterwards, the supernatant/unbound fraction was removed after centrifugation (15,000 × g, 1 min, 4 °C; supernatant 2) and Protein A-Sepharose beads with bound proteins were washed 5× with buffer A. Elution of the bound proteins was performed with boiling of the beads in 400 µl 1× PL (8 min at 95 °C, 1,000 r.p.m.). Six volumes of acetone were used to precipitate eluted proteins overnight at −20 °C. Next, precipitated proteins were collected by centrifugation (21,000 × g, 1 h, 4 °C), air-dried and resuspended in 1× PL. Culture, lysate, supernatants 1 and 2, wash (aliquots corresponding to 0.1 OD600) and eluate samples (corresponding to 10 OD600) were analysed by WB. Western blots were performed as described previously63 and probed with the appropriate primary antibodies (anti-FLAG, anti-GFP (1:1,000 in 3% BSA/TBS-T)) or FlgP antisera (1:20,000 in 3% BSA/TBS-T) and secondary antibodies (anti-mouse or anti-rabbit IgG, HRP-conjugate (1:10,000) in 3% BSA/TBS-T; GE Healthcare, RPN4201 and RPN4301, respectively). Plasmids were constructed with specific promoters for expression of FLAG-tagged proteins in C. jejuni mutants for co-immunoprecipitation experiments. To express a C-terminal FLAG-tagged PflA protein, a 206-base-pair DNA fragment from C. jejuni 81–176 (containing the promoter for flaA encoding the major flagellin with its start codon and an in-frame SpeI restriction site followed by an in-frame BamHI restriction site) was amplified by PCR. This fragment was cloned into the XbaI and BamHI sites of pRY108 to result in pDAR1425. Primers were then constructed to amplify DNA from codon 2 to the penultimate codon of pflA from C. jejuni 81–176 with an in-frame C-terminal FLAG tag epitope and stop codon. This DNA fragment was then cloned into the BamHI site of pDAR1425 so that pflA-FLAG was expressed from the flaA promoter to create pDAR3417. As a control, a 229-base-pair DNA fragment from C. jejuni 81–176 (containing the promoter for flaA encoding the major flagellin with its start codon and DNA encoding an in-frame FLAG tag epitope followed by an in-frame BamHI restriction site) was cloned into the XbaI and BamHI sites of pRY108 to create pDAR1604. Transconjugants were selected for on media with kanamycin and verified to contain the correct plasmids to result in DAR3447 and DAR3477. To express a C-terminal FLAG-tagged PflB protein, primers were constructed to amplify DNA from codon 2 to the penultimate codon of pflB from C. jejuni 81–176 with an in-frame C-terminal FLAG tag epitope and stop codon. This DNA fragment was then cloned into the BamHI site of pECO102 so that pflB-FLAG was expressed from the cat promoter to create pDAR3414. Transconjugants were selected for on media containing chloramphenicol and verified to contain the correct plasmids to result in DAR3451 and DAR3479. C. jejuni ΔpflA and ΔpflB mutants containing plasmids to express a FLAG-tag alone or C-terminal FLAG-tagged PflA or PflB proteins were grown from freezer stocks on MH agar containing chloramphenicol for 48 h in microaerobic conditions at 37 °C. Each strain was restreaked onto two MH agar plates containing chloramphenicol and grown for 16 h at 37 °C in microaerobic conditions. After growth, strains were resuspended from plates in PBS and centrifuged for 10 min at 5,210 × g. Each cell pellet was resuspended in 2 ml of PBS. Formaldehyde was added to a final concentration of 0.1% and suspensions were gently mixed for 30 min at room temperature to crosslink proteins. After crosslinking, 0.4 ml of 1 M glycine was added to each sample and suspensions were then gently mixed for 10 min at room temperature to quench the crosslinking reaction. Bacterial cells were collected by centrifugation for 10 min at 5,210 × g. Cells were then disrupted by osmotic lysis and FLAG-tagged proteins with associated interacting proteins were immunoprecipitated with α-FLAG M2 affinity resin as previously described88,89. To identify potential proteins interacting with PflA and PflB, resin with immunoprecipitated proteins were resuspended in SDS-loading buffer and electrophoresed on a 4–20% TGX stain-free gel (Bio-Rad) for 10 min. The gel was then stained with Coomassie blue for 30 min and then destained overnight. After equilibration of the gel in dH2O for 30 min, a 1 cm region of the gel containing a majority of the co-immunoprecipitated proteins was excised and diced into 1 mm pieces and then submitted for analysis by LC–MS/MS. The top 20 proteins that co-immunoprecipitated only with the FLAG-tagged bait proteins and were not detected in the negative control samples are also reported with their respective raw abundance counts. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. CryoEM maps have been deposited at the Electron Microscopy Data Bank (EMDB) and are publicly available as of the date of publication. The whole-motor map has been deposited with accession code EMD-16723 together with the original micrographs deposited to the EMPIAR repository with public accession code EMPIAR-11580 (https://doi.org/10.6019/EMPIAR-11580). The refined periplasmic scaffold map has been deposited in the EMDB with accession code EMD-16724. Subtomogram average maps were deposited as follows: ∆pflC - EMD-17415; ∆pflD - EMD-17416; pflA∆16-168 - EMD-17417; FlgQ-mCherry - EMD-17419. Source data are provided with this paper. Johnson, A. D. The rewiring of transcription circuits in evolution. Bridgham, J. T., Carroll, S. M. & Thornton, J. W. Evolution of hormone-receptor complexity by molecular exploitation. Finnigan, G. C., Hanson-Smith, V., Stevens, T. H. & Thornton, J. W. Evolution of increased complexity in a molecular machine. Origin of complexity in haemoglobin evolution. Schulz, L. et al. Evolution of increased complexity and specificity at the dawn of form I Rubiscos. Beeby, M., Ferreira, J. L., Tripp, P., Albers, S.-V. & Mitchell, D. R. Propulsive nanomachines: the convergent evolution of archaella, flagella and cilia. Singh, P. K. et al. CryoEM structures reveal how the bacterial flagellum rotates and switches direction. Structural basis of directional switching by the bacterial flagellum. Structural basis of the bacterial flagellar motor rotational switching. Structures of the stator complex that drives rotation of the bacterial flagellum. Johnson, S. et al. Molecular structure of the intact bacterial flagellar basal body. & Wolf, M. Torque transmission mechanism of the curved bacterial flagellar hook revealed by cryo-EM. B. et al. Convergent evolution in the supercoiling of prokaryotic flagellar filaments. Terashima, H., Fukuoka, H., Yakushi, T., Kojima, S. & Homma, M. The Vibrio motor proteins, MotX and MotY, are associated with the basal body of Na+-driven flagella and required for stator formation. Diverse high-torque bacterial flagellar motors assemble wider stator rings using a conserved protein scaffold. & Beeby, M. Evolution of higher torque in Campylobacter-type bacterial flagellar motors. Nakamura, S., Leshansky, A., Magariyama, Y., Namba, K. & Kudo, S. Direct measurement of helical cell motion of the spirochete Leptospira. Celli, J. P. et al. Helicobacter pylori moves through mucus by reducing mucin viscoelasticity. Beeby, M. Motility in the epsilon-proteobacteria. Cohen, E. J. et al. Evolution of a large periplasmic disk in Campylobacterota flagella enables both efficient motility and autoagglutination. Chang, Y. et al. Molecular mechanism for rotational switching of the bacterial flagellar motor. Automatic cryo-EM particle selection for membrane proteins in spherical liposomes. Tonggu, L. & Wang, L. Structure of the human BK ion channel in lipid environment. Balaban, M. & Hendrixson, D. R. Polar flagellar biosynthesis and a regulator of flagellar number influence spatial parameters of cell division in Campylobacter jejuni. Butan, C., Lara-Tejero, M., Li, W., Liu, J. & Galán, J. E. High-resolution view of the type III secretion export apparatus in situ reveals membrane remodeling and a secretion pathway. Highly accurate protein structure prediction with AlphaFold. & Oesterhelt, D. Crystal structure of halophilic dodecin: a novel, dodecameric flavin binding protein from Halobacterium salinarum. Vallese, F. et al. Helicobacter pylori antigenic Lpp20 is a structural homologue of Tipα and promotes epithelial-mesenchymal transition. Alzheimer, M. et al. Functional genomics of Campylobacter–host interactions in an intestinal tissue model reveals a small lipoprotein essential for flagellar assembly. Mechanism and role of PDZ domains in signaling complex assembly. B. et al. Characterisation of N-linked protein glycosylation in the bacterial pathogen Campylobacter hepaticus. Zarzecka, U. et al. Functional analysis and cryo-electron microscopy of Campylobacter jejuni serine protease HtrA. Blatch, G. L. & Lässle, M. The tetratricopeptide repeat: a structural motif mediating protein–protein interactions. D'Andrea, L. D. & Regan, L. TPR proteins: the versatile helix. Simultaneous glycan-peptide characterization using hydrophilic interaction chromatography and parallel fragmentation by CID, higher energy collisional dissociation, and electron transfer dissociation MS applied to the N-linked glycoproteome of Campylobacter jejuni. Gao, B., Lara-Tejero, M., Lefebre, M., Goodman, A. L. & Galán, J. E. Novel components of the flagellar system in epsilonproteobacteria. The flagellar motor protein FliL forms a scaffold of circumferentially positioned rings required for stator activation. Guo, S., Xu, H., Chang, Y., Motaleb, M. A. & Liu, J. FliL ring enhances the function of periplasmic flagella. Lynch, M. J. et al. Co-folding of a FliF-FliG split domain forms the basis of the MS:C ring interface within the bacterial flagellar motor. Lee, L. K., Ginsburg, M. A., Crovace, C., Donohoe, M. & Stock, D. Structure of the torque ring of the flagellar motor and the molecular basis for rotational switching. The maximum number of torque-generating units in the flagellar motor of Escherichia coli is at least 11. Che, Y.-S. et al. Load-sensitive coupling of proton translocation and torque generation in the bacterial flagellar motor. Krachler, A. M., Sharma, A., Cauldwell, A., Papadakos, G. & Kleanthous, C. TolA modulates the oligomeric status of YbgF in the bacterial periplasm. Skórko-Glonek, J., Wawrzynów, A., Krzewski, K., Kurpierz, K. & Lipińiska, B. Site-directed mutagenesis of the HtrA(DegP) serine protease, whose proteolytic activity is indispensable for Escherichia coli survival at elevated temperatures. Engelhardt, H., Schuster, S. & Baeuerlein, E. An Archimedian spiral: the basal disk of the Wolinella flagellar motor. Sato, K., Nakamura, S., Kudo, S. & Toyabe, S. Evaluation of the duty ratio of the bacterial flagellar motor by dynamic load control. Sangermani, M., Hug, I., Sauter, N., Pfohl, T. & Jenal, U. Tad pili play a dynamic role in Caulobacter crescentus surface colonization. Toro, E., Hong, S.-H., McAdams, H. H. & Shapiro, L. Caulobacter requires a dedicated mechanism to initiate chromosome segregation. & Koster, M. Polar secretion of proteins via the Xcp type II secretion system in Pseudomonas aeruginosa. Kozaeva, E., Nieto-Domínguez, M., Tang, K. K. Y., Stammnitz, M. & Nikel, P. I. Leveraging engineered Pseudomonas putida minicells for bioconversion of organic acids into short-chain methyl ketones. Müller, A. et al. Ultrastructure and complex polar architecture of the human pathogen Campylobacter jejuni. Structural diversity of bacterial flagellar motors. Schuhmacher, J. S., Thormann, K. M. & Bange, G. How bacteria maintain location and number of flagella? & DiRita, V. J. Transposon mutagenesis of Campylobacter jejuni identifies a bipartite energy taxis system required for motility. Cohen, E. J. et al. Campylobacter jejuni motility integrates specialized cell shape, flagellar filament, and motor, to coordinate action of its opposed flagella. 27 (eds Williams, P. et al.) 407–419 (Academic Press, 1998). The CsrA-FliW network controls polar localization of the dual-function flagellin mRNA in Campylobacter jejuni. Alzheimer, M. et al. A three-dimensional intestinal tissue model reveals factors and small regulatory RNAs important for colonization with Campylobacter jejuni. Dugar, G. et al. CRISPR RNA-dependent binding and cleavage of endogenous RNAs by the Campylobacter jejuni Cas9. Corcoran, C. P. et al. Superfolder GFP reporters validate diverse new mRNA targets of the classic porin regulator, MicF RNA. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Scheres, S. H. W. RELION: implementation of a Bayesian approach to cryo-EM structure determination. Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. Goddard, T. D., Huang, C. C. & Ferrin, T. E. Visualizing density maps with UCSF Chimera. Pintilie, G. D., Zhang, J., Goddard, T. D., Chiu, W. & Gossard, D. C. Quantitative analysis of cryo-EM density map segmentation by watershed and scale-space filtering, and fitting of structures by alignment to regions. Ramlaul, K., Palmer, C. M. & Aylett, C. H. S. A local agreement filtering algorithm for transmission EM reconstructions. Burnley, T., Palmer, C. M. & Winn, M. Recent developments in the CCP-EM software suite. Computer visualization of three-dimensional image data using IMOD. Tomo3D 2.0 – exploitation of Advanced Vector eXtensions (AVX) for 3D reconstruction. Castaño-Díez, D., Kudryashev, M., Arheit, M. & Stahlberg, H. Dynamo: a flexible, user-friendly development tool for subtomogram averaging of cryo-EM data in high-performance computing environments. The molecular architecture of axonemes revealed by cryoelectron tomography. Teufel, F. et al. SignalP 6.0 predicts all five types of signal peptides using protein language models. Mirdita, M. et al. ColabFold: making protein folding accessible to all. Croll, T. I. ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Afonine, P. V. et al. New tools for the analysis and validation of cryo-EM maps and atomic models. Gibson, D. G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. & Hall, J. E. Transformation of plasmid DNA into E. coli using the heat shock method. Altschul, S. F. et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. HMMER: biosequence analysis using profile hidden Markov models. Boll, J. M. & Hendrixson, D. R. A regulatory checkpoint during flagellar biogenesis in Campylobacter jejuni initiates signal transduction to activate transcription of flagellar genes. Johnson, T. L., Scott, M. E. & Sandkvist, M. Mapping critical interactive sites within the periplasmic domain of the Vibrio cholerae type II secretion protein EpsM. Iudin, A., Korir, P. K., Salavert-Torres, J., Kleywegt, G. J. & Patwardhan, A. EMPIAR: a public archive for raw electron microscopy image data. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Laskowski, R. A., Jabłońska, J., Pravda, L., Vařeková, R. S. & Thornton, J. M. PDBsum: structural summaries of PDB entries. We thank P. Simpson at the Imperial College London Electron Microscopy Centre and the Francis Crick Institute Structural Biology science technology platform for electron microscopy assistance. This work was supported by Medical Research Council grant MR/V000799/1 to E.J.C. and M.B., Human Frontier Science Program grant RGP0028/2021-HOCHBERG to F.P., M.B. were supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (CC2106), the UK Medical Research Council (CC2106), and the Wellcome Trust (CC2106). was supported by research grants Sh580/7-2 and Sh580/8-2 in the framework of the DFG (Deutsche Forschungsgemeinschaft) priority program SPP2002 (Small proteins in prokaryotes, an unexplored world), the DFG Research Training Group GRK2157 ‘3D-Infect', and a ‘CampyRNA' junior consortium grant within the 2nd call of Infect-ERA (ERA-NET; www.infect-era.eu)/Bundesministerium für Bildung und Forschung (BMBF; www.bmbf.de). was supported by two French National Research Agency (ANR) grants (PHY-BABIFO ANR-22-CE30-0034 and PHYBION ANR-23-ERCB-0005-01). The CBS is a member of France-BioImaging (FBI) and the French Infrastructure for Integrated Structural Biology (FRISBI), two national infrastructures supported by the ANR (ANR-10-INBS-04-01 and ANR-10-INBS-05, respectively). For the purpose of open access, the authors have applied a Creative Commons Attribution (CC BY) licence to any author-accepted manuscript version arising from this submission. Present address: MRC Laboratory of Molecular Biology, Francis Crick Avenue, Cambridge, UK Present address: The Center for Microbes, Development and Health, CAS Key Laboratory of Molecular Virology and Immunology, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, China These authors contributed equally: Tina Drobnič, Eli J. Cohen. Tina Drobnič, Eli J. Cohen, Nanki Singh, Louie D. Henderson, Trishant R. Umrekar & Morgan Beeby Structural Biology of Cells and Viruses Laboratory, The Francis Crick Institute, London, UK Mona Alzheimer, Kathrin Froschauer, Sarah Svensson & Cynthia M. Sharma William H. Hoffman, Francesco Pedaci & Ashley L. Nord Max Planck Institute for Terrestrial Microbiology, Marburg, Germany Structural Biology Science Technology Platform, The Francis Crick Institute, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed sample preparation, image processing, analysis, molecular modelling, mass photometry, and wrote the paper. conducted strain construction, sample preparation, image processing and analysis. performed sample preparation, data acquisition and image processing. conducted data acquisition and image processing. performed flgQ-mCherry strain construction, pulldowns and fliL knockout. carried out strain construction, pulldowns and funding acquisition. performed conceptualization, funding acquisition, supervision, and wrote the paper. The authors declare no competing interests. Nature Microbiology thanks Tina Iverson, Susan Lea and Keiichi Namba for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Simplified flowchart showing the generation of cryoEM volumes. (b) Central slice through the refined whole-motor structure. (c) FSC curve for B. (d) and (e) show slices through the volume of the refined, signal-subtracted periplasmic scaffold. (f) FSC curve for D, E. (b) Structure of the periplasmic scaffold density in isolation. (c) Location of one asymmetric unit from the 17-fold-symmetric periplasmic scaffold in pink as depicted in Figs. (d) Innermost basal disk ring of 51 FlgP as depicted in Fig. (e4) Innermost basal disk ring of 51 FlgP and innermost medial disk ring of 17 PflC1 as depicted in Fig. (f) Innermost basal disk ring of 51 FlgP and medial disk lattice of 17 PflC1–7 as depicted in Fig. 3d, f. (g) Proximal disk of 17 PflAB FliL4 as depicted in Fig. (a) Map-model FSC curves for protein models refined into the scaffold map: FlgP, PflA, PflC1, and PflC2–7 calculated in Phenix83, alongside images of excised pieces of the density map corresponding to individual proteins. High and low isosurface thresholds are denoted by solid and mesh surfaces, respectively, to illustrate fit of models into secondary structure densities. (b) Cross-correlation per residue plots of proteins docked into the map: PflB, FliL, MotB, PflD, calculated using Phenix. Mean CC values are shown with dashed line and in text inset. (b) AlphaFold2 prediction of PflC1 required independent rigid-body docking of the N- and C-terminal domains (magenta). (c) The resulting N-terminal domain of PflC was rigid-body docked into density multiple times for PflC2–7. C-terminal domains were not modelled. (d) PflA and PflB AlphaFold2 models (dark grey) required bending to fit into density maps (pale and dark green for PflA and PflB, respectively). (a) Kernel density estimation bead position plots over many rotations for 11 cells. Measurements were of duration 20 to 240 s, substantially de-energized by CCCP to slow rotation. Their (x,y) histograms, a kernel density of the angular position, and the weighted power spectrum are depicted. (b) Weighted power spectrum of all 11 traces. (a) Size Exclusion Chromatography of PflC (red) along with protein standards (grey) showing elution volume. (b) Calibration graph showing a dimer of PflC (red) corresponding to retention times during elution. (c) Mass photometry measurements of purified PflC (replicates). (d) Size Exclusion Chromatography of PflCN (∆236–349, green) along with protein standards (grey) showing elution volume. (e) Calibration graph of PflCN (green) corresponding to retention times during elution. (f) Mass photometry measurements of purified PflCN. Theoretical masses are shown in the inserts. The instrument's limit of detection is 35 kDA, meaning the monomer mass is larger than expected. (a) Needleman-Wunsch global sequence alignments of Campylobacter jejuni PflCN and PflCC with C. jejuni HtrA from ChimeraX MatchMaker91 using default parameters. Asterisks highlight HtrA His-Asp-Ser catalytic triad not conserved in PflC. Coloured boxes indicate pruned residue pairs between PflCN:HtrAN (red) and PflCC:HtrAC (green) from MatchMaker algorithm using a 4 Å RMSD cutoff (b) Core folds of PflC and HtrA from ChimeraX structural alignment using pruned residue pairs of PflCN and PflCN (left) to HtrA (right) (PDBID: 6Z0536) guided by the global alignment in panel A. Pruned core residues are depicted in colour, while non-core residues are depicted in grey. (c) RMSD of PflC alignment to HtrA depicted on the structure of PflC as worm diameter and colour (blue: 0 Å, white: 4 Å, red: 8 Å). (d) Left panel shows PflC and HtrA oriented as with the PflC protomer in Fig. 3c Right panels depict PflC and HtrA rotated similarly to reveal the HtrA active site cleft. Close-up boxes include stick representation of residues, highlighting HtrA's H119:D150:S224 catalytic triad, and PflC's N65 and S122 residues that sequence align to D150 and S224, revealing divergence and atrophy of the active site region in PflC. (a) Automated secondary structure annotation of HtrA using the ChimeraX implementation of the Defining the Secondary Structure of Proteins algorithm (top); and PflC (bottom) with HtrA nomenclature transferred based on structural correspondence. (b) Topology diagram of HtrA and PflC generated using PDBsum92. Note that due to algorithmic nuances, PDBsum's secondary structure definitions do not exactly correspond to those of DSSP. Locations of the three catalytic triad residues H119, D150 and S224 are labelled. (a–d) Mass photometry measurements of purified PflA and PflB constructs show the proteins are mainly monodisperse. There is a dimer peak present for the PflAΔCter construct, likely due to a reduction in stability and solubility. Dimer peaks appear only when β-sandwich and linker domain of PflA is present. In panels E and G, monomer peaks of PflA and PflB are not resolved due to their similar molecular weights. Structural components of the Campylobacter jejuni flagellar motor. Antisera, strains and recombinant DNA used in this study Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Drobnič, T., Cohen, E.J., Calcraft, T. et al. In situ structure of a bacterial flagellar motor at subnanometre resolution reveals adaptations for increased torque. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02372-4'>Extreme weather event attribution predicts climate policy support across the world</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 09:14:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Extreme weather events are becoming more frequent and intense due to climate change. Yet, little is known about the relationship between exposure to extreme events, subjective attribution of these events to climate change, and climate policy support, especially in the Global South. Combining large-scale natural and social science data from 68 countries (N = 71,922), we develop a measure of exposed population to extreme weather events and investigate whether exposure to extreme weather and subjective attribution of extreme weather to climate change predict climate policy support. We find that most people support climate policies and link extreme weather events to climate change. Subjective attribution of extreme weather was positively associated with policy support for five widely discussed climate policies. However, exposure to most types of extreme weather event did not predict policy support. Overall, these results suggest that subjective attribution could facilitate climate policy support. Climate change is increasing the frequency and intensity of extreme weather events (defined as an event that is rare at a particular place and time of year1), which puts a substantial proportion of the global population at physical and economic risk1. The cost of extreme weather events attributable to climate change is estimated at US$143 billion per year2. The impacts of extreme weather events are disproportionately felt in countries in the Global South3. Even though the Global South is at greater risk, attribution studies and social science research on human responses to such events overwhelmingly focus on countries and populations in the Global North4,5,6. Mitigative action is needed to slow climate change and mitigate the impacts of extreme weather events. So far, global efforts have been insufficient, which calls for more stringent climate policies. Public support for climate policies is important because such support can drive governmental policy outputs7 and policymakers often respond to public demand for climate policies8. The psychological distance of climate change (that is, the perception that climate change is spatially, temporally and socially distant) may help explain societal inaction on this issue9. If so, public awareness and understanding of climate change may increase as more people experience extreme weather events for themselves10,11,12,13,14,15. However, previous studies on the relationship between experiencing extreme weather events and climate change action and beliefs have produced inconsistent findings. In particular, some studies have found that experiencing extreme weather events increases climate change belief16, concern11,17,18,19, support for climate policies and green parties17,20,21,22,23, and climate change adaptation24, while other studies found no relationship6,25,26,27. Studies using aggregate objective measures of exposure to and impacts of extreme weather events often find no effect of extreme weather experience on climate change attitudes25,26,28. For example, one US study found that living in an area with higher fatalities from extreme weather events was associated with perceiving more climate risks29, while another US study found that fatalities from extreme weather events were not associated with opinions about climate change30. However, these studies used different definitions and measurements of extreme weather events, and these extreme weather events were compared with different psychological and behavioural outcomes27. Further, most studies have focused on a single country31 or a single type of extreme weather event (for example, heatwaves), which limits the comparability of the impacts of different types of extreme weather event. This limitation is considerable, as a meta-analysis found notable differences in effect sizes depending on the type of extreme weather event32. The inconsistency of previous studies might also be explained by another important factor: whether people attribute the extreme weather event to climate change6,11,31,33,34,35. Recent studies support this hypothesis: people who attribute extreme weather events to climate change are more likely to perceive climate change as a risk and to report engaging in mitigation behaviour36,37. For example, a study in the United Kingdom found that the subjective attribution of floods to climate change is a necessary condition for the experience of floods to translate into climate change threat perception36. However, no cross-country evidence exists on the subjective attribution of extreme weather events to climate change. We combined natural and social science approaches to examine how extreme weather events and their attribution to climate change relate to support for widely discussed climate change mitigation policies across 68 countries (N = 71,922). This study employed an interdisciplinary design by triangulating data on exposed populations computed using the probabilistic CLIMADA risk modelling platform38,39 with global survey data on subjective attribution of extreme weather events and support for climate policies collected in the Trust in Science and Science-related Populism (TISP) study40. We used a standardized metric to comparatively assess the relationship between the size of exposed populations to several extreme weather events—river floods, heatwaves, European winter storms, tropical cyclones, wildfires, heavy precipitation and droughts—and climate policy support. Specifically, we modelled how many people in a country were exposed to extreme weather events over the past few decades relative to the total population. We referred to this as the ‘exposed population' (see Online Methods). Our preregistered study addressed the following research questions: (1) Does exposure to extreme weather events on the population level relate to climate policy support? (2) Do subjective attribution and exposed population have an interactive effect on policy support? In addition, we addressed the following non-preregistered questions: (1) What is the level of public support for five climate policies across countries? (2) To what degree do people attribute extreme weather events to climate change across countries (subjective attribution) and is subjective attribution related to policy support? We hypothesized that people who live in countries with higher exposure would show stronger support for mitigative climate policies, and that the relationship between exposed population and policy support would be stronger for individuals with higher subjective attribution. We also hypothesized that the relationship between exposed population and policy support is associated with people's income and residence area (urban vs rural), which might relate to their adaptation potential to extreme events. Note that not all preregistered questions are addressed in this paper. We assessed support for the following five climate policies with a 3-point scale (1 = not at all, 2 = moderately, 3 = very much): Increasing taxes on carbon-intense foods, raising taxes on fossil fuels, expanding infrastructure for public transportation, increasing the use of sustainable energy, and protecting forested and land areas. In line with previous research, increasing carbon taxes received the lowest support41,42, with only 22% and 29% of people, respectively, indicating they very much support increased taxes on carbon-intensive foods and fossil fuels (Fig. Protecting forested and land areas, by contrast, was a popular policy option, with 82% supporting it very much and only 3% not supporting it at all. The second most-supported policy was increasing the use of sustainable energy, with 75% supporting it very much, and only 5% not supporting it at all. For further analyses, we combined responses to the five policy options into an index (α = 0.61; see factor analysis in Supplementary Table 12 and non-preregistered analyses with policy subscales in Supplementary Fig. b, Mean support for climate policies in 66 countries (climate policy support was not measured in Argentina and Malaysia). Participants were asked: “Please indicate your level of support for the following policies.” Response option ‘not applicable' is not shown. No data were available for countries shaded in light grey. A clear majority supported climate policies in all countries (global mean (M) = 2.37, s.d. = 0.43 on a scale from 1 = Not at all, 2 = Moderately and 3 = Very much). These findings are in line with a previous study showing that 89% of participants demand intensified political action on climate change43. We calculated mean support by averaging participants' support for five policies (see Online Methods and Fig. This mean value is representative in terms of gender, age and education due to post-stratification weighting (see Online Methods). We found strong differences in support across countries and policies (Fig. Support for climate policies was particularly high in African and Asian countries, average in Australia, Costa Rica and the United Kingdom, and below the global average in several European countries, such as Czechia, Finland and Norway (Supplementary Figs. Non-preregistered analyses comparing our aggregate measure with policy support subscales (that is, support for taxes, support for green transition) can be found in Supplementary Fig. Our results for the aggregate measure and policy subscales were mostly consistent. Participants who identified as men, were younger, more religious, had higher education, higher income, left-leaning politics and who lived in urban areas were more likely to support climate policies (Supplementary Tables 1–7 and Fig. Participants indicated subjective attribution by rating the degree to which they believed that climate change has increased the impact of six extreme weather events—droughts, heatwaves, wildfires, heavy rain, floods, heavy storms—in their country over the past decades (1 = Not at all, 5 = Very much). Responses to the six items were mean averaged (α = 0.92). Globally, subjective attribution of extreme weather events to climate change was well above the scale midpoint in all countries (M = 3.80, s.d. = 1.02). In line with a previous study36, non-preregistered analyses showed that subjective attribution was positively related to identifying as a woman, being older, more religious, having higher education and higher income, living in an urban (vs rural) area and self-identifying as politically liberal and left-leaning (Supplementary Table 8). There was little variation in subjective attribution across extreme event types. Subjective attribution appeared relatively lower for wildfires (M = 3.67, s.d. However, subjective attribution varied across global regions (Fig. Participants in South American countries most strongly agreed that the occurrence of extreme weather events has been affected by climate change over the past decades, especially in Brazil and Colombia (Supplementary Fig. Subjective attribution was lowest in Northern European and African countries (Supplementary Fig. Lower subjective attribution in African countries could be explained by the fact that climate change awareness and belief in human-caused climate change are still relatively low across African countries46. Subjective attribution was not assessed in Albania. No data were available for countries shaded in light grey. While almost all the sampled populations were exposed to heatwaves and heavy precipitation over the past decades at least once, fewer populations had been exposed to droughts, wildfires and floods. Our fully anonymous data did not allow geospatially matching participants to certain areas where extreme events occurred; we therefore do not know whether participants were personally exposed to those events and cannot test whether exposure at the individual level relates to policy support. However, we can reliably estimate whether exposure at the population level relates to policy support. Exposed population refers to the average annual proportion of a country's total population exposed to a specific weather-related hazard and averaged over the past few decades. The exact time frame varies slightly across events. b, Exposed population to European winter storms. d, Exposed population to heavy precipitation. e, Exposed population to river floods. f, Exposed population to tropical cyclones. No data were available for countries shaded in light grey. We investigated whether exposure at the country level and subjective attribution of extreme events at the individual level were associated with stronger climate policy support. Since we were interested in studying how the relationships vary between different types of extreme weather event and policy support, we ran seven blockwise multilevel regression models—one for each type of extreme weather event—predicting an index of climate policy support. Step 1 of the blockwise regression included socio-demographic variables and exposed population. In Step 2, we added subjective attribution for the specific event and three interaction terms: exposed population × subjective attribution, exposed population × income and exposed population × residence area. Belief that climate change has impacted local extreme weather events predicted support for climate policy (Fig. Random effects models show that the relationship between subjective attribution and policy support was significantly stronger in North America, Australia and in several European countries than the mean global effect, and significantly weaker in Peru and South Africa (Supplementary Figs. Summary of seven multilevel models, one for each type of extreme weather event, with random intercepts across countries predicting climate policy support and controlling for socio-demographic variables and two additional interaction terms. Filled circles denote significant effects at P < 0.05. Full models for each event type can be found in Supplementary Tables 1–7. For five out of the seven extreme weather events, exposed population size did not predict policy support (Fig. However, people in countries more exposed to wildfires were more supportive of climate policies (Supplementary Table 5). Conversely, people in countries more exposed to heavy precipitation were less supportive of climate policies (Supplementary Table 3). We conducted additional exploratory, non-preregistered robustness checks to investigate whether exposed population and land area, as well as exposed population and climate change belief at the country level had an interactive effect on policy support. Since climate change belief was not assessed in this study, we relied on country-level data from another study47, available for 48 countries included in this study. The relationship between exposure to heavy precipitation/wildfires and policy support was no longer statistically significant when controlling for beliefs and land area, while the relationship between subjective attribution and policy support remained significant (Supplementary Fig. Therefore, the relationship between exposure to wildfires/heavy precipitation and policy support should be interpreted with caution. We tested whether exposed population size and subjective attribution interacted to predict policy support, as investigated in previous studies33,36,37. We found that the relationship between exposed population and policy support was stronger for participants with higher attribution of heatwaves and tropical cyclones, whereas the relationship between exposed population and policy support was weaker for participants with higher attribution of heavy precipitation and European winter storms. However, we found the opposite interaction effect for river floods, droughts and wildfires: as subjective attribution increases, the relationship between exposed population and policy support weakens. In other words, for individuals with high subjective attribution, support for policies is already high and less dependent on exposure to these extreme events. In contrast, for individuals with low subjective attribution, support for policies increases with higher exposure to droughts, floods and wildfires (Fig. The lines represent varying levels of subjective attribution at −1s.d, the mean and +1 s.d., with shaded regions indicating 95% confidence intervals. These findings are in tension with the results of previous studies, which reported a positive moderation effect for flooding36, a negative moderation effect for hurricanes33 and no moderation effect for wildfires37. Our seven multilevel models each included interaction effects for exposed population × income and exposed population × residence area. We found significant interactions with small effect sizes for river floods and wildfires, but not for any other events. For river floods, we found a negative interaction effect with income and a positive interaction with urban areas (Supplementary Table 4). This indicates that the relationship between exposed population size and policy support was stronger for individuals with lower income as well as for individuals who live in urban areas. For wildfires, we found a positive statistical effect for income, meaning that the relationship between exposed population and policy support was stronger for richer individuals (Supplementary Fig. This study provides global evidence that subjective attribution of extreme weather events to climate change is associated with greater policy support for climate mitigation. Overall, different extreme weather events appear to have different relationships with climate policy support. This pattern highlights the importance of comparative analyses that consider different types of event. We additionally provide evidence that subjective attribution is high, and particularly so in Latin America. This might be explained by the fact that belief in human-caused climate change and self-reported personal experience of extreme weather events are high in Latin America48, and that people in Latin American countries were among the most likely to report that climate change will harm them and future generations a great deal and that climate change should be a high priority for their government49. The finding that the relationship between subjective attribution and policy support was weaker in some Latin American countries might therefore be due to a ceiling effect In line with previous studies36, we also found that subjective attribution interacts with exposure to European winter storms, heatwaves, heavy precipitation and tropical cyclones to predict climate policy support. While larger exposure to extreme events was not found to be related to policy support (except for wildfires), we cannot rule out that changes in the frequency of extreme weather events over time might be sufficient to shift support. Nevertheless, our data suggest that if individuals attribute extreme weather events to climate change, support for climate policies is higher regardless of whether the events are more frequent. The reverse causal relationship is also possible: people who are supportive of climate policies are more likely to attribute extreme weather to climate change. Longitudinal panel studies are needed to investigate the nature and direction of this relationship. These findings might also help explain previous inconsistent results on the relationship between extreme weather event experience and mitigation behaviour. Few of these studies assessed whether participants linked these events to climate change, therefore missing a key controlling variable. Consequently, we strongly recommend that future studies assess subjective attribution. We found a negative relationship between exposed population to heavy precipitation and policy support in our preregistered model. Subjective attribution was relatively low for heavy precipitation. This corroborates previous findings that people often fail to link extreme rainfall with climate change10. In line with this argument, a media analysis that investigated themes in climate change coverage in 10 countries (2006–2018) found that media reporting on extreme weather events mostly focused on weather anomalies, as well as fires, hurricanes and storms50. Our moderation analyses show that the negative effect of heavy precipitation exposure on policy support is strongest for people with low subjective attribution. This further highlights the need for more research on climate change communication on types of extreme weather event that are not typically associated with climate change, such as heavy precipitation, as these events might serve as ‘teachable moments'15. However, it should be noted that the relationship between exposure to heavy precipitation and policy support was no longer significant in our exploratory analyses that included the interactions of exposed population with land area and climate change belief. This finding should therefore be interpreted with caution. Wildfires are the only type of extreme weather event that positively predicts climate policy support when controlling for subjective attribution, although this effect was no longer significant in models that included interaction effects for exposure with land area and climate change belief. Several previous studies similarly reported a positive relationship between wildfire exposure and climate policy support23,37,51,52. This positive relationship could be explained by the fact that wildfires often result in extensive and visible damage51, and are linked to personal health concerns due to smoke exposure53. Another study found that among Australian adults who directly experienced wildfires, 45% increased individual climate activism, providing further evidence of the effects of wildfires on behavioural intentions54. Contrary to our hypothesis, the relationship between exposed population and policy support was weaker for individuals with higher subjective attribution of droughts, floods and wildfires. One possible explanation is that these three types of extreme weather event allow for management strategies that can directly reduce the hazard itself, such as man-made flood protections, irrigation systems, prescribed burn-offs and land-use policies. Therefore, people may be more likely to support policies pertaining to law enforcement or economic regulations instead of climate change mitigation55,56. In contrast, although heavy precipitation, storms and heatwaves are exacerbated by climate change and can be mitigated by addressing it, once they occur, we can only manage their impacts, not prevent their occurrence. Our measure of exposed population has strengths and limitations. While the standardized metric of exposed population allows the comparison of the impacts of different events across countries, it is a relative measure (that is, to a country's total population) and does not reflect the severity of exposure or the potential for individuals to be repeatedly exposed to different events. Further, the measure does not consider the exposure to compound events57, that is, when two or more events occur in an interacting combination. No conclusions can be drawn as to whether the participants in the study were directly exposed to these events. This measure therefore reflects the broader population-level exposure to these events, rather than individual-level exposure. The data cannot speak to whether exposure at the individual level relates to policy support. However, it can be reliably concluded that exposure at the population level did not relate to policy support. Some extreme weather events are less likely to be experienced directly (for example, floods or hurricanes), but they still receive widespread media coverage. The approach of analysing exposure at the population level therefore allows the study of effects that go beyond individual exposure to events. It should be noted that for some extreme weather events (for example, heatwaves and heavy precipitation), variance was very low, given that most people were affected by these events at some points over the past few decades (Supplementary Table 9). Researchers have found that temporal proximity of an event matters for climate change concern: the more recent an event, the larger the impact on climate change concern18. Since some of these events occur infrequently (for example, tropical cyclones), longer time frames such as in this study have the advantage that they allow the comparison of the effects of several different events in a global context58. With the use of a measure of exposure to extreme weather events at the population level, this article finds that subjective attribution predicts climate policy support, while exposure to five out of the seven extreme events considered in this study does not predict policy support. Overall, ensuring subjective attribution might be an important way to increase support for climate policies37. Experimental research could focus on finding effective communication strategies to increase subjective attribution among the public to help develop causal models (for example, ref. Extreme weather events are increasingly linked to climate change in news and social media50,60,61,62,63, but more research is needed to study communication of extreme weather events and their attribution in the Global South62,64. Detailed information on the data collection strategy can be found in ref. Participants were asked to carefully read a consent form (approved under IRB protocol number IRB22-1046), which included some general information about the study and the anonymity of the data. Only participants who consented to participating in the study were allowed to proceed with the study. To generate models with parameters that are representative for target populations in terms of gender, age and education, and have more precise standard errors, we used post-stratification weights. Participants were asked: “Many countries have introduced policies to reduce carbon emissions and mitigate climate change. This can include the implementation of laws aiming to reduce greenhouse gases, for example. Please indicate your level of support for the following policies: 1) Raising carbon taxes on gas and fossil fuels or coal, 2) Expanding infrastructure for public transportation, 3) Increasing the use of sustainable energy such as wind and solar energy, 4) Protecting forested and land areas, 5) Increasing taxes on carbon intense foods (for example, beef and dairy products).” Response options ranged from 1 = Not at all, 2 = Moderately, 3 = Very much, and 4 = Not applicable. Response option 4 was coded as missing for the analyses. Participants were asked: “The next questions are about climate change and weather events. When you answer them, please think about your country. 1) Floods, 2) Heatwaves, 3) Heavy storms, 4) Wildfires, 5) Heavy rain, 6) Droughts.” Response options ranged from 1 = Not at all, to 5 = Very much. 65 for a detailed overview of the other measures. We submitted a detailed preregistration including research questions, hypotheses and an analysis plan to OSF (https://doi.org/10.17605/OSF.IO/G23A7) before data collection on 15 November 2022. To estimate the relationships between subjective attribution, exposed population and three interaction terms (exposed population × subjective attribution; exposed population × income log (US$); exposed population × residence area (urban vs rural)), we used blockwise multilevel regression models with random intercepts across countries. In addition, we computed models with random effects to estimate how the effects of subjective attribution on climate policy support varied across countries. We scaled all independent variables by country means and country s.d.s, except for the country-level variable ‘exposed population', which we scaled with grand means and grand s.d.s. We estimated the reliability of our two scales: subjective attribution and climate policy support. Scale reliability of subjective attribution in the global sample was very high, with Cronbach's alpha = 0.92 and omega = 0.92. An overview of the reliability of subjective attribution across 67 countries (ranging from omega = 0.74 to omega = 0.95) can be found in Supplementary Table 10. Scale reliability of climate policy support in the global sample was acceptable, with Cronbach's alpha = 0.61 and omega = 0.62. An overview of the reliability of climate policy support across 66 countries (ranging from omega = 0.40 to omega = 0.75) can be found in Supplementary Table 11. To further assess the robustness of our policy support scale, we ran a polychoric parallel analysis with principal axis factoring to inspect how many factors should be retained for an exploratory factor analysis (EFA). The parallel analysis determined that two factors should be kept for an EFA. We therefore ran an EFA with unweighted least squares factoring and promax oblique rotation to inspect two factor loadings (Supplementary Table 12). Our items clearly loaded on two factors, with items relating to the expansion of public transport, protected areas and increasing renewable energy loading on Factor 1 (labelled as ‘Green transition') and the two items related to increasing taxes on meat and dairy and fossil fuels loading on Factor 2 (labelled as ‘Taxes'). The Taxes subscale had good internal reliability (omega = 0.73). We further conducted three non-preregistered robustness checks. Specifically, we examined whether our results are robust to the inclusion of an interaction between land area of countries (in square kilometres) and exposed population, an interaction between country-level climate change belief and exposed population, and across the two climate policy support subscales (Taxes and Green Transition). Data on climate change belief were retrieved from the Climate Many Labs study as processed by Our World in Data66, while data on land area were retrieved from multiple sources compiled by World Bank (2024) and processed by Our World in Data67. Data on land area for Taiwan was retrieved from ref. The term ‘country' in this Article refers to both sovereign states and territories not recognized as such. In this study, we used the open-source, probabilistic CLIMADA (CLIMate ADAptation) risk modelling platform38,39 for the spatially explicit computation of exposed population from different hazards on a grid at 150 arc-seconds (~4.5 km at the equator) resolution. CLIMADA was designed to simulate the interaction of climate and weather-related hazards, the exposure of assets or populations to this hazard, and the specific vulnerability of exposed infrastructure and people in a globally consistent fashion. The GPW dataset was chosen for its high spatial resolution and its comprehensive and consistent coverage, providing population count estimates at a granularity of 30 arc-seconds (~1 km at the equator), which we aggregated to match the 150-arc-second resolution used in our risk model. Seven types of extreme weather event were analysed in this study: droughts, river floods, heatwaves, heavy precipitation, tropical cyclones, wildfires and European winter storms, which form the input hazard layer in our risk model. We computed the exposed population to these events. Detailed information on the definition of each event, data sources, the years covered and other relevant details for each type of extreme weather event are provided in Supplementary Table 13. Each hazard in this study was defined on the basis of its unique characteristics and the potential impact it has on the exposed population, with the chosen underlying datasets ensuring consistent coverage across all countries involved. Some of these hazards were evaluated in an event-based perspective (for example, tropical cyclones, wildfires), while others were assessed as annually aggregated measures (for example, river floods, heatwaves). Hazards were inferred either from historical records (tropical cyclones, European winter storms, wildfires), climate reanalyses of a reference period (heatwaves, heavy precipitation) or historical climate modelling (droughts, river floods). For drought, we utilized a ‘long-term' definition based on soil moisture70, a methodology that primarily captures agricultural impacts, potentially leading to indirect effects on populations. Furthermore, the dataset provides annual maxima, without representing single drought events, which potentially limits the depth of our risk analysis for certain areas. In the case of river floods, the datasets used in this study represent large rivers and fluvial floods, while coastal or pluvial floods are not included70,71. We note that ‘heavy precipitation' as a different hazard may serve as a proxy for pluvial or flash floods. Besides, there was a potential overestimation of affected areas due to the methodology of considering full grid cells as affected. We utilized ERA-5 reanalysis data to display observed trends as changes between the reference period and the more recent 20-year period 2000–201972. Finally, changes were displayed as the multimodel median. Wildfires of the historical period 2000–2019 were assessed using satellite imagery to derive thermal anomalies. A grid cell was considered affected if the temperature exceeded 300 K73. The approach does not distinguish between intentional and unintentional fires, and the dataset captures gridpoint-specific annual maxima only. Finally, in our preregistration, we broadly categorized tropical cyclones and European winter storms under the umbrella term ‘storms'. Typically, tropical cyclones prevail in tropical and subtropical regions, while our modelled winter storms are predominantly observed in Europe. Given their distinct geographical occurrences, the impacts of these two storm types can be considered additive or complementary. While this classification ensured consistency with our framework, modelling these exposures carries higher uncertainty compared with the tropics and subtropics. In addition, storm impacts are expressed relative to population size, which may lead to disproportionately high exposure percentages in regions with low population density compared with densely populated areas experiencing similar storm frequencies. We relied on historical records to assess the impacts of both storm hazards74,75, and readers should interpret the results for higher latitudes with these considerations in mind. In this study, we defined ‘exposed population' as the average annual proportion of a country's total population exposed to a specific weather-related hazard within a given time period. An overview of time periods can be found in Supplementary Table 13. Therefore, this metric is relative and does not reflect the severity of exposure or the potential for individuals to be repeatedly impacted by different events. In addition, in large countries such as the United States, different hazards may affect different regional populations (for example, wildfires on the West Coast versus tropical cyclones in the East) which, unfortunately, is not captured in our country-level aggregation. The exposed population is presented as a percentage of the total population, providing a standardized measure for comparative analysis across the 68 countries included in our study. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The dataset on subjective attribution and policy support analysed during the current study is available in the Open Science Framework (OSF) repository at https://doi.org/10.17605/OSF.IO/5C3QD (ref. The dataset on exposed populations to extreme weather events generated and analysed during the current study is available in OSF at https://doi.org/10.17605/OSF.IO/G23A7 (ref. The analysis code is available in OSF at https://doi.org/10.17605/OSF.IO/G23A7 (ref. Seneviratne, S. I. et al. in Climate Change 2021: The Physical Science Basis (eds Masson-Delmotte, V. et al.) 1513–1766 (Cambridge Univ. The global costs of extreme weather that are attributable to climate change. Howe, P. D. Perceptions of seasonal weather are linked to beliefs about global climate change: evidence from Norway. Challenges to understanding extreme weather changes in lower income countries. Sisco, M. R. The effects of weather experiences on climate change attitudes and behaviors. Anderson, B., Böhmelt, T. & Ward, H. Public opinion and environmental policy output: a cross-national analysis of energy policies in Europe. & Bernauer, T. Are policymakers responsive to public demand in climate politics? Hot dry days increase perceived experience with global warming. Ogunbode, C. A., Doran, R. & Böhm, G. Individual and local flooding experiences are differentially associated with subjective attribution and climate change concern. Spence, A., Poortinga, W., Butler, C. & Pidgeon, N. F. Perceptions of climate change and willingness to save energy related to flood experience. What shapes perceptions of climate change? Global perceptions of local temperature change. & Otto, F. E. L. Extreme weather events as teachable moments: catalyzing climate change learning and action through conversation. Hornsey, M. J., Harris, E. A., Bain, P. G. & Fielding, K. S. Meta-analyses of the determinants and outcomes of belief in climate change. & Stanig, P. Climate change experiences raise environmental concerns and promote Green voting. Konisky, D. M., Hughes, L. & Kaylor, C. H. Extreme weather events and climate change concern. Bergquist, P. & Warshaw, C. Does global warming increase public concern about climate change? Garside, S. & Zhai, H. If not now, when? Local temperature anomalies increase climate policy interest and support: analysis of internet searches and US congressional vote shares. Hazlett, C. & Mildenberger, M. Wildfire exposure increases pro-environment voting within Democratic but not Republican areas. van Valkengoed, A. M. & Steg, L. Meta-analyses of factors motivating climate change adaptation behaviour. & Jenkins, J. C. Shifting public opinion on climate change: an empirical assessment of factors influencing concern over climate change in the U.S., 2002–2010. & Grover, H. Examining the relationship between physical vulnerability and public perceptions of global climate change in the United States. Marlon, J. R. et al. Detecting local environmental change: the role of experience in shaping risk judgments about global warming. Xia, Z. et al. A meta-analysis of the relationship between climate change experience and climate change perception. Wong-Parodi, G. & Garfin, D. R. Hurricane adaptation behaviors in Texas and Florida: exploring the roles of negative personal experience and subjective attribution to climate change. Whitmarsh, L. Are flood victims more concerned about climate change than other people? The role of direct experience in risk perception and behavioural response. van der Linden, S. On the relationship between personal experience, affect and risk perception: the case of climate change. & Sposato, R. G. Attribution matters: revisiting the link between extreme weather experience and climate change mitigation responses. Wong-Parodi, G. & Berlin Rubin, N. Exploring how climate change subjective attribution, personal experience with extremes, concern, and subjective knowledge relate to pro-environmental attitudes and behavioral intentions in the United States. Bresch, D. N. & Aznar-Siguan, G. CLIMADA v1.4.1: towards a globally consistent adaptation options appraisal tool. Aznar-Siguan, G. & Bresch, D. N. CLIMADA v1: a global weather and climate risk assessment platform. Fairbrother, M. Public opinion about climate policies: a review and call for more studies of what people want. Douenne, T. & Fabre, A. Yellow vests, pessimistic beliefs, and carbon tax aversion. Andre, P., Boneva, T., Chopra, F. & Falk, A. Globally representative evidence on the actual and perceived support for climate action. Bergquist, M., Nilsson, A., Harring, N. & Jagers, S. C. Meta-analyses of fifteen determinants of public opinion about climate change taxes and laws. Nielsen, K. S. et al. Underestimation of personal carbon footprint inequality in four diverse countries. Selormey, E. E., Dome, M. Z., Osse, L. & Logan, C. Change Ahead: Experience and Awareness of Climate Change in Africa Report No. Addressing climate change with behavioral science: a global intervention tournament in 63 countries. Climate change beliefs and their correlates in Latin America. Leiserowitz, A. et al. International Public Opinion on Climate Change, 2023 (Yale Program on Climate Change Communication, 2023). Hase, V., Mahl, D., Schäfer, M. S. & Keller, T. R. Climate change in news media across the globe: an automated analysis of issue attention and themes in climate change coverage in 10 countries (2006–2018). Gould, R. K., Shrum, T. R., Ramirez Harrington, D. & Iglesias, V. Experience with extreme weather events increases willingness-to-pay for climate mitigation policy. Visconti, G. & Young, K. The effect of different extreme weather events on attitudes toward climate change. Burke, M. et al. Exposures and behavioural responses to wildfire smoke. Examining contrasting influences of extreme weather experiences on individual climate activism. Moral Hazard, Wildfires, and the Economic Incidence of Natural Disasters Working Paper 26550 (NBER, 2019). Calviño-Cancela, M. & Cañizo-Novelle, N. Human dimensions of wildfires in NW Spain: causes, value of the burned vegetation and administrative measures. Zscheischler, J. et al. Future climate risk from compound events. Meiler, S. et al. Intercomparison of regional loss estimates from global synthetic tropical cyclone models. Communicating the links between climate change and heat waves with the Climate Shift Index. Berglez, P. & Al-Saqaf, W. Extreme weather and climate change: social media results, 2008–2017. How Indian mainstream media covered two extreme weather events in 2015. Painter, J. et al. Is it climate change? Thaker, J., Painter, J., Borwankar, V., Jain, G. & Negi, K. English and regional media coverage of the 2022 heatwave in India. Mede, N. G. et al. Perceptions of science, science communication, and climate change attitudes in 68 countries – the TISP dataset. Share of People Who Believe in Climate Change and Think It's a Serious Threat to Humanity, 2023 (Our World in Data, 2024). Gridded population of the world, version 4 (GPWv4): basic demographic characteristics, revision 11. Projecting exposure to extreme climate impact events across six event categories and three spatial scales. Climate signals in river flood damages emerge under sound regional disaggregation. The International Best Track Archive for Climate Stewardship (IBTrACS) (NOAA National Centers for Environmental Information, 2010); https://doi.org/10.1175/2009BAMS2755.1 Röösli, T. The Impact of Winter Storms in Switzerland—Prototyping Decision-Support Tools. Cologna, V. Extreme weather event attribution predicts climate policy support across the world. We thank H. Karami (University of Zurich) for managing the author list. and C.T.-E. were supported by ANR PICS; A.F.-B. was supported by Aarhus University Research Foundation grant AUFF-E-2019-9-13; P.M. was supported by Aarhus University Research Foundation grant AUFF-E-2019-9-2; R. Bardhan was supported by Africa Albarado Fund, Cambridge Africa ESRC GCRF, and UKRI ODA International Partnership Fund; J.P. Reynolds was supported by Aston University, and UK Research and Innovation (UKRI) under the UK government's Horizon Europe funding guarantee EP/X042758/1; N.L. were supported by Australian Research Council grant DP180102384, and John Templeton Foundation grant number 62631; O. Ghasemi was supported by Australian Research Council grant DP190101675; U.K.H.E. was supported by Australian Research Council grant FT190100708; D.D., A.G., D.G. were supported by the Basic Research Program at the National Research University Higher School of Economics (HSE University); R.D. was supported by Bill and Melinda Gates Foundation grant OPP1144, a Cambridge Humanities Research Grant, CRASSH grant fund for climaTRACES lab, the Keynes Fund, the UKRI ODA International Partnership Fund, and the Quadrature Climate Foundation; T.C. were supported by Boston University (Startup Funds); F.A. was supported by CNPq - INCT (National Institute of Science and Technology on Social and Affective Neuroscience, grant number 406463/2022-0); K.C.D. was supported by a COVID-19 Rapid Response grant from the University of Vienna, and Austrian Science Fund grant FWF I3381; C.L., J.P.N., E.P. were supported by a COVID-19 Rapid Response grant from the University of Vienna, and Austrian Science Fund grants FWF I3381 and W1262-B29; R.M.A. and S. Kristiansen were supported by the Center for Climate and Energy Transformation, University of Bergen, Norway; C.G.B. and A.C.H.-M. were supported by Conacyt grant A1S9013; O.K. was supported by a Concerted Research Action grant from the Fédération Wallonie-Bruxelles (Belgium) (‘The Socio-Cognitive Impact of Literacy'); J.S. was supported by Core ETHZ funding and Swiss Agency for Development and Cooperation (SDC) grant 7F09521; E.A. was supported by the EDCTP2 Programme (TMA2020CDF-3171), and BMGF (INV075699); K.M.D. was supported by European Research Council Advanced Grant ‘Consequences of conspiracy theories - CONSPIRACY_FX' grant 101018262; J.R. was supported by European Union's Horizon 2020 Research and Innovation Programme under grant agreement number 101006436 (GlobalSCAPE); S. Meiler, C.M.K. were supported by European Union's Horizon 2020 research and innovation program grant agreement numbers 820712 (PROVIDE), 101073978 (DIRECTED) and 101081369 (SPARCCLE); G.H. were supported by Fundação para a Ciência e a Tecnologia, UIDB/04295/2020 and UIDP/04295/2020; E.G. was supported by Government of Alberta Major Innovation Fund grant RES0049213; J.N. was supported by HELTS Foundation (USA); K. Breeden was supported by Harvey Mudd College; T.K.R. and K. Pštross were supported by the Institute of Communication Studies and Journalism, Charles University; H.F. was supported by Internal project costs IWM; M. Tanaka was supported by JST-RISTEX ELSI grant number JPMJRX20J3, and the Hitachi Fund Support for Research Related to Infectious Diseases; G.C. and E. Szumowska were supported by Jagiellonian University; M. Alfano and M. Ferreira were supported by John Templeton Foundation number 61378, John Templeton Foundation grant number 62631, and Australian Research Council DP1901015077; A. Krouwel was supported by Kieskompas.nl; M. Tsakiris was supported by the NOMIS Foundation; R.M. was supported by NOMIS Foundation/Leverhulme International Professorship Grant LIP-2022-001; T.K., K. Petkanopoulou and J.v.N. were supported by the NORFACE Joint Research Programme on Democratic Governance in a Turbulent Age, NWO, and European Commission through Horizon 2020 grant 822166; A.R. were supported by Nicolaus Copernicus University; N.I. was supported by a Research grant from the College of Social Sciences, Kimep University; E.B., P.K. were supported by SNSF (VAR-EXP); O. Białobrzeska and M. Parzuchowski were supported by SWPS University; M.E. were supported by the School of Geography, Planning and Spatial Sciences, University of Tasmania; and the Centre for Marine Socioecology, University of Tasmania; E.J.N. were supported by the School of Medicine and Psychology, Australian National University; M.D.M. was supported by School of Psychology and Public Health Internal Grant Scheme 2022; I.A. was supported by the School of Psychology, University of Sheffield; Beasiswa Pendidikan Indonesia Kemendikbudristek - LPDP provided by Balai Pembiayaan Pendidikan Tinggi (BPPT) Kemdikbudristek and LPDP Indonesia; R. Bhui was supported by the Sloan School of Management, Massachusetts Institute of Technology; O. Buchel was supported by Slovak Research and Development Agency (APVV), contract number APVV-22-0242; N.M.L. was supported by Social Sciences and Humanities Research Council grant number 430-2022-00711; M.P.-C. was supported by Statutory Funds from University of Silesia in Katowice; A.C.V. was supported by Swedish Research Council grant 2020-02584; L.S. was supported by Swiss Agency for Development and Cooperation (SDC) grant 7F09521; S.B. was supported by the Swiss Federal Office of Energy (SI/502093–01); J.L.G. was supported by Swiss National Science Foundation PRIMA Grant PR00P1_193128; V.C. was supported by Swiss National Science Foundation Postdoc Mobility Fellowship P500PS_202935, Harvard University Faculty Development Fund, and SPEED2ZERO Joint Initiative that received support from the ETH Board under the Joint Initiatives scheme; E.W.M. was supported by The HELTS Foundation; G.R. was supported by Universitas Islam Negeri Sunan Kalijaga; S.J. were supported by the University of Bamberg; J.M.M. was supported by the University of Delaware; M.D. were supported by the University of Lodz; A. Koivula and P.R. were supported by the University of Turku; M.B. were supported by the University of Warsaw; A.P. was supported by the University of Zurich/IMKZ; T. Ostermann and J.P. Röer were supported by the University research budget; A. Bajrami and R.T. were supported by University ‘Aleksandër Moisiu', Durrës; S. Schulreich was supported by Universität Hamburg; L.S.K. was supported by the Victoria University of Wellington; H.K. Open access funding provided by Swiss Federal Institute of Technology Zurich. Viktoria Cologna, Simona Meiler, Chahan M. Kropf, David N. Bresch, Johan Six & Leonhard Späth Federal Office of Meteorology and Climatology MeteoSwiss, Zurich-Airport, Switzerland Simona Meiler, Chahan M. Kropf & David N. Bresch Department of Psychobiology and Methodology, Faculty of Psychology, Universidad Complutense de Madrid, Madrid, Spain Department of Advertising and Public Relations, Michigan State University, East Lansing, MI, USA Centre for Climate Change Communication, George Mason University, Fairfax, VA, USA Centre for Research in Communication and Culture, Department of Communication and Media, Loughborough University, Loughborough, UK School of Collective Intelligence, Mohammed VI Polytechnic University, Ben Guerir, Morocco Department of Science and Technology Studies, Faculty of Science, Universiti Malaya, Kuala Lumpur, Malaysia ELTE Institute of Psychology, Eotvos Lorand University, Budapest, Hungary Department of Philosophy, Macquarie University, Sydney, Australia Linde Center for Science, Society, and Policy, Division of Humanities and Social Science, California Institute of Technology, Pasadena, CA, USA Department of Communication, George Mason University, Fairfax, VA, USA Department of Psychology, Universitas Islam Negeri Sunan Kalijaga, Yogyakarta, Indonesia National Institute of Science and Technology on Social and Affective Neuroscience, São Paulo, Brazil Eri Bertsou, Pradeep Krishnan & Amber Zenklusen Institute of Psychology, SWPS University, Warsaw, Poland Cedars-Sinai Medical Center, Los Angeles, CA, USA Computer Science Department, Harvey Mudd College, Claremont, CA, USA Department of Scientific and Innovation Culture, Spanish Foundation for Science and Technology, Madrid, Spain Federica Cagnoli, Lucia Di Stefano & Paolo Parra Saiani Department of Health Law, Policy, and Management, Boston University School of Public Health, Boston, MA, USA Institute for Sociology, Slovak Academy of Sciences, Staré Mesto, Slovakia Network for Economic and Social Trends, Western University, London, Ontario, Canada Behavior in Crisis Lab, Institute of Psychology, Jagiellonian University, Cracow, Poland Gabriela Czarnek, Theofilos Gkinopoulos & Ewa Szumowska Kimberly C. Doell, Claus Lamm, Jonas P. Nitschke, Ekaterina Pronizius & Boryana Todorova Center for Sociocultural Research, HSE University, Moscow, Russia Dmitrii Dubrov, Albina Gallyamova, Dmitry Grigoryev & Elizaveta Komyaginskaya Department of Economics, Harvard University, Cambridge, MA, USA Department of Computer Science and Engineering, University of Notre Dame, Notre Dame, IN, USA Department of Political Science and International Relations, Carleton College, Northfield, MN, USA Department of Philosophy, Macquarie University, Sydney, New South Wales, Australia Marinus Ferreira, Neil Levy & Robert M. Ross Leibniz Institut für Wissensmedien, Tübingen, Germany Department of Psychology, Saarland University, Saarbrücken, Germany Institute for Management and Organization, Leuphana University, Lueneburg, Germany School of Psychology, University of New South Wales, Sydney, New South Wales, Australia Institute for Climate Risk and Response, University of New South Wales, Sydney, New South Wales, Australia Research Institute for Responsible Innovation, School of Management, University of St Gallen, St Gallen, Switzerland Centro de Investigación y de Estudios Avanzados del Instituto Politícnico Nacional, Mexico City, Mexico Department of Media and Communication, City University of Hong Kong, Kowloon, Hong Kong Department of Political Science and International Relations, KIMEP University, Almaty, Kazakhstan Department of Government and Politics, Jahangirnagar University, Dhaka, Bangladesh Doctoral School of Social Sciences, Nicolaus Copernicus University, Toruń, Poland Hixon Center for Climate and the Environment, Harvey Mudd College, Claremont, CA, USA Institute of Communication Studies and Journalism, Charles University, Prague, Czech Republic Center for Social and Cultural Psychology, Université Libre de Bruxelles, Brussels, Belgium Department of Political Science and International Relations, School of Sciences and Humanities, Nazarbayev University, Astana, Kazakhstan Department of Psychology, Ural Federal University, Yekaterinburg, Russia Anthony Lantian, Jean-Baptiste Légal & Claudia Teran-Escobar Departamento de Psicología, Universidad Peruana Cayetano Heredia, La Molina, Peru Network Science Institute, Northeastern University, Boston, MA, USA School of Psychology and Public Health, La Trobe University, Melbourne, Victoria, Australia Office for Quality Assurance, Analyses and Reporting, Project EUTOPIA, University of Ljubljana, Ljubljana, Slovenia Department of Management and Supply Chain Studies, Nkumba University, Entebbe, Uganda Communication Arts Programme, Bowen University, Iwo, Nigeria Department of Psychology and Psychotherapy, Witten/Herdecke University, Witten, Germany Department of Clinical Neuroscience, Karolinska Institutet, Stockholm, Sweden Penn Center for Neuroaesthetics, University of Pennsylvania, Philadelphia, PA, USA Department of Psychological Science, Pomona College, Claremont, CA, USA School of Psychology, Aston University, Birmingham, UK Max Planck Institute for Human Development, Berlin, Germany Social and Cognitive Neuroscience Laboratory, Mackenzie Presbyterian University, São Paulo, Brazil Psychology Department, TED University, Ankara, Turkey Sociology Department, Vrije Universiteit Brussel, Brussels, Belgium School of Sustainability, Arizona State University, Tempe, AZ, USA Faculty of Political Science and Economics, Waseda University, Tokyo, Japan Faculty of Technology and Bionics, Rhine-Waal University, Kleve, Germany Faculty of Philosophy and Social Science, Nicolaus Copernicus University, Toruń, Poland School of Economics and Management, Tongji University, Shanghai, China School of Environment, Tsinghua University, Beijing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and the TISP Consortium acquired funding. discussed the design, methods and results. and the TISP Consortium collected data. The authors declare no competing interests. The questionnaire used for this study was considered exempt from full IRB review by the Harvard University Area Committee on the Use of Human Subjects in November 2022 (protocol number IRB22-1046). Nature Climate Change thanks Miaomiao Liu, Matto Mildenberger and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Extreme weather event attribution predicts climate policy support across the world. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02032-5'>Human gut bacteria bioaccumulate per- and polyfluoroalkyl substances</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 09:09:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Per- and polyfluoroalkyl substances (PFAS) are persistent pollutants that pose major environmental and health concerns. While few environmental bacteria have been reported to bind PFAS, the interaction of PFAS with human-associated gut bacteria is unclear. Here we report the bioaccumulation of PFAS by 38 gut bacterial strains ranging in concentration from nanomolar to 500 μM. Bacteroides uniformis showed notable PFAS accumulation resulting in millimolar intracellular concentrations while retaining growth. In Escherichia coli, bioaccumulation increased in the absence of the TolC efflux pump, indicating active transmembrane transport. Cryogenic focused ion beam secondary-ion mass spectrometry confirmed intracellular localization of the PFAS perfluorononanoic acid (PFNA) in E. coli. Proteomic and metabolomic analysis of PFNA-treated cells, and the mutations identified following laboratory evolution, support these findings. Finally, mice colonized with human gut bacteria showed higher PFNA levels in excreted faeces than germ-free controls or those colonized with low-bioaccumulating bacteria. Together, our findings uncover the high PFAS bioaccumulation capacity of gut bacteria. Environmental contamination by manufactured chemicals has, by some estimates, exceeded the safe planetary boundary1,2. Due to the widespread contamination of water and agricultural systems, a vast number of chemical pollutants make it into food and, hence, into the human body3,4,5,6,7,8. The gut microbiota is particularly susceptible to exposure, and adverse interactions therein could cause systemic effects owing to the critical role of the microbiota in host physiology9,10,11. While the effect of chemical pollutants on bacterial growth is being mapped12, the impact of gut bacteria on chemicals remains largely an open question. Among the chemicals of greatest concern are per- and polyfluoroalkyl substances (PFAS), which are often referred to as ‘forever chemicals'. This group of chemicals includes >4,700 compounds13 that are used in a wide range of industrial and consumer products, including firefighting foams, waterproof clothes and nonstick cookware. The widespread use of PFAS owes to their exceptional stability resulting from the strength of C–F bonds14, and surfactant-like properties due to the presence of strong hydrophilic and hydrophobic groups. Yet, these same properties have made PFAS a concern for environmental and human health1,4,6,15,16,17,18,19. The annual health-related cost of PFAS exposure is estimated to be 50–80 billion Euros across Europe20. Studies in Europe and the United States have found high prevalence of PFAS in blood7,21. Legislative actions are therefore planned to control PFAS levels in drinking water and to restrict usage22,23,24,25. However, such efforts are not global and, with long environmental half-lives and no efficient route for their removal, PFAS pose a challenge for human and environmental health. Chemical methods to degrade PFAS have had limited success. Most methods show slow kinetics, require multiple processing steps and focus on non-perfluorinated compounds that have some carbon atoms linked to, for example sulfur26,27. For reducing PFAS in the human body, ion-exchange resins have been shown to be effective but also exhibited side effects28. Environmental bacteria, such as Pseudomonas sp. strains isolated from PFAS-contaminated sites, have been reported to bioaccumulate a sulfur-containing PFAS (perfluorohexane sulfonate, PFHxS). Furthermore, due to the surfactant-like properties of PFAS, the molecules are believed to interact with and accumulate mainly in the lipid bilayers30,31,32. Similarly, the interaction of perfluorooctane sulfonate (PFOS)—an 8-C PFAS with a sulfonic acid functional group—with Lactobacillus strains was noted as ‘bio-binding'33 suggesting binding to the cell surface. Intracellular localization thus remained an open question. As the gut microbiota is a critical interface between exposure and the human body, we investigated how gut bacteria interact with PFAS. To assess the potential impact of food-borne pollutants on commensal gut bacteria, we started with a community-based screening approach. We assessed the ability of two mixtures (synthetic communities) of gut bacterial strains to sequester pollutant compounds during 4 h of exposure (Extended Data Fig. Each synthetic community consisted of 10 human gut bacterial strains (Supplementary Table 1). Allocation of strains to the communities was based on similarity in growth rates. In total, 42 common pollutants were selected for testing against these communities based on reported occurrence in food and for representation of different classes including pesticides, food contact materials and industrial chemicals (Supplementary Table 2). A total of 13 pollutants were found to be depleted by more than 20% by one or both synthetic communities (Extended Data Fig. Of these compounds, 10 were then tested for depletion by 14 individual strains during a 24-h growth period (Extended Data Fig. These strains are a subset of the 20 strains forming the two synthetic communities and were selected for their prevalence and abundance in a healthy population, and for phylogenetic and metabolic representation34,35 (Supplementary Table 1). In the test with individual strains, 7 pollutants were found to be depleted to more than 20% by at least one of the bacterial strains (Fig. a, Specificity of human gut bacteria to sequester (bioaccumulate and biotransform) chemical pollutants during a 24-h growth period as identified using mass spectrometry. Links between bacterial species and pollutant denote >20% depletion. The link thickness is proportional to the median depletion from 6 replicates (3 biological, 2 technical; initial pollutant concentration = 20 μM) (Supplementary Tables 7 and 8). C. comes, Coprococcus comes; E. rectale, Eubacterium rectale; P. merdae, Parabacteroides merdae; R. intestinalis, Roseburium intestinalis. b, PFNA bioaccumulation in 89 strains spanning major bacterial phyla, and three yeasts. OD600 = 3.75; initial PFNA concentration = 20 μM (9.3 mg l−1); n = 3 technical replicates (Supplementary Tables 3 and 9). c, PFNA depletion by B. uniformis cultures at different OD600 values in PBS buffer and PFNA exposure concentration of 20 μM. P values are based on two-sided t-test; ***P < 0.001; n = 4 technical replicates (Supplementary Table 10). d, Kinetics of PFNA depletion during B. uniformis growth starting with low cell density and 20 μM PFNA. e, Kinetics of PFNA depletion by B. uniformis at high cell density in PBS over 7 days (OD600 = 3.75; initial PFNA concentration = 20 μM). Two-sided t-test; **P value < 0.01; ***P value < 0.001 (supernatant compared with the compound control; pellet compared with 0); n = 3 biological replicates (Supplementary Table 13). Exact P values in Supplementary Table 48. f, PFNA is significantly bioaccumulated by B. uniformis grown in mGAM at a range of initial concentrations (initial OD600 = 0.05; initial PFNA concentrations = 0.01–100 μM) compared with the compound control. Two-sided t-test; P value FDR corrected for number of concentrations tested; **adjusted (adj.) P value < 0.001; n = 4 technical replicates (Supplementary Table 14). h, Growth sensitivity of gut bacteria to PFAS is independent of bioaccumulation (n = 3 technical replicates). Asterisks denote bioaccumulating bacteria (Supplementary Tables 16 and 17). By comparing the whole culture and supernatant concentrations to compound controls, we were able to distinguish the observed sequestration between bioaccumulation and biotransformation. Bioaccumulation—storage of the pollutant without modification—was defined as compound sequestration to at least 20% from the supernatant but recovery from the whole culture sample, whereas biotransformation was defined as both supernatant and whole culture sample showing more than 20% depletion. The pollutants bioaccumulated by gut bacteria included perfluorooctanoic acid (PFOA) and perfluorononanoic acid (PFNA), which belong to the chemical group of PFAS. The degree of bioaccumulation over 24 h for 20 μM (9.3 mg l−1) PFNA exposure varied from 25% (P. distasonis) to 74% (O. splanchnicus) and for 20 μM (8.3 mg l−1) PFOA from 23% (P. merdae) to 58% (O. splanchnicus). Given the widespread presence of PFAS in the environment, we further investigated PFAS bioaccumulation in gut bacteria to understand its underlying mechanisms. Human gut bacteria are known to harbour extensive intraspecies genomic and functional variability across individuals, including for bioaccumulation of drugs36. To assess the differences in intra- and interspecies bioaccumulation capabilities, we tested 89 microbial strains for PFNA sequestration (Supplementary Table 3). We selected PFNA as it is one of the long-chain PFAS that are excreted via faeces and thus are more likely to be encountered by the gut bacteria than short-chain PFAS that are excreted mainly via urine37. PFNA accumulation showed distinct grouping by phylum, with Bacteroidota showing the highest accumulation (Fig. 1b), and a bimodal distribution that could be divided into two groups using a Gaussian mixture model: low accumulating (51 strains) and high accumulating (38 strains) (Extended Data Fig. Low-accumulating strains were predominantly Gram positive (43 of 51; 84%), and high-accumulating strains, Gram negative (29 of 38; 76%) (Extended Data Fig. Gram-negative bacteria generally have a higher lipid content38, which could be a factor underlying PFAS accumulation31. However, the divide between the bioaccumulation capacity of Gram-negative bacteria and Gram-positive bacteria is not definitive, with notable exceptions including Escherichia coli. To further test the influence of lipid content, we tested three yeast species (Rhodotorula mucilaginosa, Candida californica and Kluyveromyces marxianus) that have a high lipid content39 comparable to Gram-negative bacteria. The tested yeasts showed only about 5% sequestration (Fig. 1b), suggesting that additional factors beyond lipid content influence bioaccumulation. To characterize the kinetics of PFAS bioaccumulation, we selected high-accumulating B. uniformis, a prevalent gut bacterium34,35. PFNA accumulation scaled with cell density, both in non-growing suspension cultures and during bacterial growth (Fig. We next probed the kinetics of PFNA uptake using stationary-phase cultures or cells suspended in phosphate-buffered saline (PBS). In both cases, bioaccumulation occurred within the timescale of sampling (<3 min) (Fig. Notably, for B. uniformis, no PFNA was released back into the supernatant over the course of 7 days (Fig. We next asked whether the fraction of PFAS sequestered was dependent on the exposure concentration. For this, we tested sequestration of PFNA by B. uniformis over a broad range of concentrations (0.01–500 μM (4.64 µg l−1 to 232 mg l−1)). The degree of sequestration remained constant around 50%, both in growing cultures and resting cells (Fig. Furthermore, the degree of bioaccumulation by B. uniformis increased with increasing PFAS chain length, from none for perfluoroheptanoic acid (PFHpA, 7C) to 60% in the case of perfluorodecanoic acid (PFDA, 10C) (Fig. Despite this high degree of bioaccumulation, B. uniformis as well as other abundant gut bacterial strains grew well even at high micromolar concentrations (Fig. Thus, PFAS bioaccumulation occurs within 3 min without inhibiting bacterial growth up to concentrations orders of magnitude higher than known contamination levels of circa 1 nM (refs. To investigate how PFAS is bioaccumulated, we first tested whether inactive cell mass (that is, dead, lysed cells) could bioaccumulate PFAS. By contrast, live E. coli bioaccumulated much lower levels of PFOA, PFNA and PFDA (~5%, ~25% and ~40%, respectively), while inactivated cells bioaccumulated to a similar degree as B. uniformis and O. splanchnicus (Fig. This suggests that PFAS bioaccumulation is not solely a passive phenomenon driven by attachment to membrane lipid bilayers. To test this, we measured bioaccumulation in E. coli mutants that lacked one or more genes coding for efflux pump proteins (∆acrA-acrB, ∆tolC) or altered permeability (imp4213) (Fig. Efflux pumps are a common mechanism used by several bacterial species to reduce intracellular concentration of toxic compounds41,42. We reasoned that E. coli did not bioaccumulate PFNA to the same extent as other tested Gram-negative bacteria because it could, at least to some extent, pump out the PFAS. As gene deletions can exhibit different phenotypes in different strain backgrounds43, we used two E. coli strains, viz. Mutants lacking TolC showed circa 1.5-fold increase in PFDA and circa 5-fold increase in PFNA bioaccumulation (Fig. Consistent with the increased bioaccumulation, these mutants showed increased growth inhibition at higher PFAS exposure levels (Extended Data Fig. These results show that E. coli strains use a TolC-dependent mechanism to limit PFAS bioaccumulation. a, PFAS bioaccumulation by live, dead (heat inactivated) and lysed (heat inactivated, freeze-thawed and sonicated) B. uniformis, E. coli and O. splanchnicus cultures (OD600 = 3.75) in PBS buffer. Two-sided t-test; P value FDR corrected for number of strains and compounds tested; *adj. P value < 0.001 and >20% reduction compared with the compound control; n = 3 technical replicates (Supplementary Table 18). TolC can work in combination with other pumps41,81,82. Efflux mutants E. coli BW25113 ∆tolC and E. coli C43 (DE3) ∆acrAB-tolC showed a ~1.5-fold increase in PFDA and ~5-fold increase in PFNA bioaccumulation. OD600 = 3.75; exposure concentration = 20 µM (PFNA, 9.3 mg l−1; PFDA, 10.3 mg l−1); two-sided t-test; *P value < 0.05; **P value < 0.01 compared with the corresponding wild type; n = 3 technical replicates (Supplementary Table 19). d, PFNA bioaccumulation by B. uniformis at 0.34 nM (160 ng l−1) exposure. Two-sided t-test (supernatant compared with the compound control; pellet compared with the pellet control); *P value < 0.05; **P value < 0.01; n = 3 biological replicates (Supplementary Tables 20 and 21). e, Capacity of gut bacteria to concentrate PFAS from the media into the bacterial pellet in a growth assay in mGAM or resting assay in PBS at 5 µM PFAS exposure (growth assay: initial OD600 = 0.05, 24-h incubation; resting assay: OD600 = 3.75, 4-h incubation). f, PFAS recovery from the bacterial pellets after 1 h of exposure in PBS (OD600 = 3.75, PFAS mix of 14 compounds each at a concentration of 1 mg l−1). The bars depict the median concentration based on pellet weight; the error bars show standard error; n = 3 technical replicates shown as circles (Supplementary Table 23). We next tested whether bacterial cells could bioaccumulate PFAS at very low exposure levels. B. uniformis cells were exposed to 0.34 nM (160 ng l−1) PFNA, a PFAS concentration observed in water samples across Europe and the United States15,40 and below average blood levels6. We validated the applicability of our liquid chromatography tandem mass spectrometry (LC–MS/MS) method for low-concentration samples using the US Environmental Protection Agency (US EPA) method44,45 and using another, independently developed, LC–MS/MS method (Methods). At this low exposure level, B. uniformis bioaccumulated 37% of PFNA into the bacterial pellet, concordant with depletion from the supernatant (Fig. The estimated concentration within the bacterial pellet (wet biomass) was circa 17.7 nM (8,200 ng l−1), a 50-fold increase (Extended Data Fig. As these estimates are based on wet pellet weight, intracellular concentrations will be higher. Considering the estimated number of cells per optical density (OD) unit (0.6–2 × 109 cells ml−1 OD600−1 (ref. 38)) and 1 µm3 cell volume38, the enrichment of PFAS in bacterial cells would be 1.2–7 times higher than in the wet pellet. These results show the large capacity of bacterial cells to bioaccumulate and concentrate PFAS from their surrounding environment. Environmental contamination often contains a mixture of multiple PFAS compounds. We therefore investigated the bioaccumulation capacity of bacteria exposed to a mixture of 14 PFAS (C4–C14, and three sulfonated variants), each at a concentration of 1 mg l−1 (circa 2 µM for PFNA). Many of these compounds, such as PFNA, PFOA, PFOS, PFDA and perfluoroundecanoic acid (PFUnDA), are among the chemicals of high concern as per the National Report on Human Exposure to Environmental Chemicals of the Centers for Disease Control and Prevention. While the E. coli wild type showed minimal bioaccumulation, E. coli ∆tolC and B. uniformis bioaccumulated almost 100% of PFAS with chain length above 10C. The percentage of bioaccumulated PFAS for individual compounds is comparable to a single compound exposure. Thus, at 1 mg l−1 per compound, PFAS bioaccumulation is additive across different molecules with minimal mixture effects. Intracellular PFNA concentration in B. uniformis at 250 µM (116 mg l−1) exposure level would be circa 5–10 mM (2.3–4.6 g l−1), well above that of most native metabolites46. At these concentrations, the observed degree of bioaccumulation, if considered to be membrane associated only, would imply one molecule of PFNA per two lipid molecules in the cell membrane, which is physiologically improbable. How do cells cope with such high intracellular levels of highly effective surfactants such as PFAS and maintain their growth? Consistent with their observed levels of bioaccumulation, all cells except E. coli wild type showed disruption and condensation of cytoplasmic content (Extended Data Fig. Similar morphological features have previously been observed in Clostridium perfringens exposed to lauric acid, which also has surfactant properties47. We quantified cytoplasmic condensation using automated image processing (Methods). However, TEM imaging can show artefacts due to extensive sample processing, including resin embedding. Therefore, to confirm PFNA localization inside the bacterial cells, we performed cryogenic focused ion beam time-of-flight secondary ion mass spectrometry (FIB-SIMS) imaging of E. coli ∆tolC cells exposed to 250 µM PFNA. This method circumvents the need to embed the specimen in resin, preserving the cells in a near-native state in vitreous ice. Voxel-by-voxel three-dimensional analysis of the bacterial cell composition uncovered a strong fluorine signal localized inside the bacterial cells, corresponding to intracellular PFNA bioaccumulation (Fig. In total, 2 biological replicates were analysed, comprising 120 E. coli ∆tolC cells exposed to 250 µM PFNA and 44 control cells. All 120 cells exposed to PFNA showed a clear fluorine signal within the bacterial cell, while 43 control cells showed no fluorine signal, and one control cell showed a very low fluorine signal. Furthermore, intracellular PFNA was unevenly distributed with a distinct tendency for aggregation. The intracellular aggregation suggests either interaction of PFAS with cytosolic contents, such as proteins and other macromolecules, or phase separation of PFNA and cytosolic contents within the bacterial cell. A given area of the sample is imaged by scanning over it repeatedly with a gallium focused ion beam and analysing the chemical composition of the ablated material using FIB-SIMS. Shown is one of 120 cells imaged (additional images in Extended Data Fig. a, Secondary electron images (formed by secondary electrons resulting from the FIB scanning) of three different Z-frames of the sample, that is, at three different positions along the z-stack, provide spatial images of the imaged cells. b–e, Top (b) and side (c) views of the three-dimensional stack of SIMS data for a mass-to-charge ratio of 19, corresponding to fluorine (F−), in which the colour scale represents the ion count per extraction. We next reasoned that if the cells had mechanisms to modulate bioaccumulation and/or to cope with high intracellular levels, resistance to inhibitory PFAS exposure would rapidly evolve under natural selection. We therefore evolved, through serial transfer, B. uniformis, B. thetaiotamicron, P. merdae, C. difficile and E. coli ΔtolC in a medium with 500 μM (182 mg l−1) PFHpA, 500 μM (207 mg l−1) PFOA, 250 μM (116 mg l−1) PFNA or 125 μM (64 mg l−1) PFDA (Extended Data Fig. merdae (PFDA), B. uniformis (PFNA, PFDA) and E. coli ΔtolC (all tested PFAS) showed between 1.3- and 46-fold increase in growth (Extended Data Fig. Genome sequencing of evolved B. uniformis populations identified 55 variants linked to PFNA and PFOA exposure (Supplementary Table 4). More than half of the variants are in non-coding regions, suggesting that changes in gene expression contribute to the adaptation of B. uniformis to PFAS. To characterize how intracellular localization of PFNA affects the molecular physiology of the bioaccumulating cells, we measured their proteomic and metabolic response. Most PFNA-impacted proteins are either membrane- or stress response-related (Extended Data Fig. B. uniformis cells exposed to PFNA also showed changes in membrane-related proteins, with the top three being efflux pumps, viz. Missense variants of the latter RND transporter (R9I2L8) were also identified in all four PFNA-evolved B. uniformis populations with high (>0.92) allele frequencies. The proteomic and genomic changes together show the involvement of efflux pumps in B. uniformis in cellular interaction with PFAS, as we also found for E. coli. To identify proteins interacting with PFNA, we used thermal proteome profiling (TPP), which allows proteome-wide assessment of structural changes (stabilization or destablization) upon ligand binding48,49. We thus compared E. coli wild type and ΔtolC mutant exposed to PFNA, with the exposure before or after cell lysis, with the latter removing the membrane barrier for PFNA–protein interactions. The bioaccumulating ΔtolC mutant featured 10-fold more PFNA-interacting proteins (Supplementary Table 5). a, Proteomics analysis shows proteins that are differentially abundant between B. uniformis treated with 20 µM PFNA and those treated with DMSO. The red and green dots mark proteins with a log2(abundance ratio) > 1 or < −1 (that is, twofold increase or decrease) and a multiple-testing corrected P value of less than 0.05; n = 6 biological replicates (Supplementary Table 24). The circle marks a protein from the RND efflux system (gene ID 962 corresponds to protein R9I2L8), for which nine missense variants within the coding region of the gene were identified in populations evolved under high PFAS concentrations (Supplementary Table 4). P values were calculated using analysis of variance followed by Benjamini–Hochberg correction for multiple testing. b,c, TPP analysis of E. coli BW25113 wt (wild type; low-PFNA bioaccumulating) (b) and E. coli BW25113 ΔtolC (high-PFNA bioaccumulating) (c). Lysate and live cells incubated with PFNA look more similar for E. coli BW25113 ΔtolC mutant compared with the wild type, supporting increased bioaccumulation in ΔtolC mutants. Each data point represents the summed log2(FC) across all temperatures for a specific protein. Black dashed line, diagonal; blue line, linear regression with 95% confidence interval (Supplementary Table 25). d, Principal component (PC) analysis shows a clear distinction between B. uniformis pellet samples treated with 20 µM PFNA and the control; n = 6 biological replicates (Supplementary Table 26). e, Aspartic acid, glutamic acid and glutamine concentrations in B. uniformis pellet and supernatant samples; n = 6 biological replicates (Supplementary Table 26). P values were calculated using two-sided t-test and corrected for multiple testing using the Benjamini–Hochberg method. The proteomic changes in efflux pump and other membrane-related proteins could be expected to result in altered levels of cellular metabolites. To test this, we used a targeted metabolomics method aimed at broadly conserved metabolites, including amino acids and vitamins. Metabolic changes were observed only for the bioaccumulating strains. B. uniformis, one of the highest bioaccumulators, showed a distinct metabolic response to PFNA exposure (Fig. Metabolites with altered levels include amino acids aspartate, glutamate and glutamine, and kynurenine, a metabolite implicated in the gut–brain axis50,51. Increased cadaverine levels, along with decreased glutamate and glutamine levels, indicate B. uniformis response like that observed in acid stress52,53,54. To determine whether bacterial PFAS bioaccumulation occurs in an in vivo context, we tested C57BL/6J mice colonized with a community of 20 human gut bacterial strains (Com20)55 (Supplementary Table 6) against germ-free (GF) controls. The animals were administered a one-time oral dose of PFNA (10 mg kg−1 body weight) by gavage. This initial dose was selected based on previous reports indicating it causes no adverse effects in mice56, while still allowing for reliable detection in their faeces. Faecal samples were collected over the following 2 days, and on day 3, colon and small intestine content samples were taken post-mortem (Fig. Faecal microbiota analysis using 16S rRNA sequencing showed that 17 of the 20 strains colonized the mice and PFNA treatment had no effect on the microbiota composition (Extended Data Fig. Colonized mice showed a significantly higher faecal PFNA concentration at all follow-up time points (3 h, P = 0.009, fold change (FC) = 7.9; 1 day, P = 0.001, FC = 2.6; 2 days, P < 0.0001, FC = 2.8; 3 days (colon), P = 0.0004, FC = 3.1; 3 days (small intestine), P = 0.007, FC = 1.9) (Fig. Next, we tested whether microbial communities containing high- versus low-accumulating species would cause differences in faecal PFNA excretion. GF mice were colonized with a community of either five high- or five low-PFNA-accumulating strains (LC: Akkermansia muciniphila, Collinsella aerofaciens, Enterocloster bolteae, E. coli and Ruminococcus gnavus; HC: Bacteroides fragilis, B. thetaiotaomicron, B. uniformis, Lacrimispora saccharolytica and Phocaeicola vulgatus; Supplementary Table 6). PFNA treatment had no effect on gut microbiota composition (Extended Data Fig. 10c); however, LC-colonized mice showed circa twofold higher colonization compared with HC-colonized mice (Extended Data Fig. HC-colonized mice showed increased faecal excretion of PFNA, normalized by bacterial biomass, on days 1 and 2 (1 day, P = 0.02, FC = 2.4; 2 days, P = 0.009, FC = 3.8) (Fig. We also carried out a GF versus colonized comparison using a 100× lower PFNA dose (0.1 mg kg−1 body weight PFNA), which showed the same trend of increased excretion for the colonized mice (Extended Data Fig. Together, the mouse experiments show the contribution of gut bacterial bioaccumulation to faecal PFAS excretion. a, Experimental setup for comparison of faecal excretion between GF mice and mice colonized with human gut bacteria (Com20) or comparison between mice colonized with high-PFNA bioaccumulating gut bacteria (HC) and mice colonized with low-PFNA bioaccumulating gut bacteria (LC). b, Mice colonized with a community of 20 human gut bacterial strains (Com20) show higher PFNA excretion after 10 mg kg−1 body weight PFNA exposure compared with GF controls. Box plot: centre = 50th percentile, bounds of box = 25th and 75th percentiles, lower and upper whiskers = lower and upper hinges ± 1.5 × interquartile range; two-sided t-test; P values are FDR corrected; n = 9 mice per group (Supplementary Table 27). c, Mice colonized with a community of HC show higher PFNA excretion after 10 mg kg−1 body weight PFNA exposure compared with mice colonized with LC. Box plot: centre = 50th percentile, bounds of box = 25th and 75th percentiles, lower and upper whiskers = lower and upper hinges ± 1.5 × interquartile range; two-sided t-test; P values are FDR corrected; n = 9 mice per group (Supplementary Table 28). Our study reveals the capacity of gut bacteria to bioaccumulate PFAS and has fundamental implications for understanding how PFAS interact with biological systems. We find that bioaccumulation is characterized by high capacity and rapid kinetics. We observe >50-fold enrichment of PFAS within bacterial pellets implying intracellular concentration in the mM range. The internalization of PFAS is supported both by analytical and biological evidence. The latter includes multi-omics analyses, imaging, phylogenetic grouping of bioaccumulators and increased bioaccumulation in the E. coli efflux-pump mutant. How do the bioaccumulating cells maintain their viability and growth given the surfactant-like properties of PFAS? Cryogenic FIB-SIMS imaging shows that the bioaccumulated molecules aggregate in dense clusters (Fig. Sequestration of PFAS into dense intracellular aggregates appears to minimize interference with vital cellular processes, which may explain the bacterial viability and growth. This is supported by the relatively limited impact of PFAS on the bacterial proteome and metabolome. Yet, these changes may have implications in a community or host context. We observe changes in the expression of membrane proteins, including transporters, and reduced the secretion of amino acids that can reduce cross-feeding opportunities in microbial communities57. Studies in synthetic and ex vivo communities would be needed to investigate how PFAS-induced changes in bacterial physiology impact fellow community members. Redistribution of bioaccumulated PFAS following cell division also remains an open question. It is unlikely that bacteria have evolved a specific tolerance mechanism for PFAS given the relatively low fitness cost and complex selection pressures in the gut environment. We postulate that the cellular entry and cytosolic aggregation are consequences of the physicochemical properties of PFAS allowing interaction with non-membrane cellular components. The interaction of PFAS with proteins has been noted in the case of human serum58. Furthermore, studies in aqueous solution have shown self-aggregation of PFAS molecules59. Our conjecture is further supported by the observation that bioaccumulation capacity is phylogenetically segregated and could be genetically modulated in E. coli. In support, a recent study investigating the kinetics of PFAS compounds ingested by a volunteer found that long-chain PFAS bioaccumulate most in the human body37. Together, the biophysical considerations and our data suggest that interactions with transporters and self-aggregation in the cytosol, possibly facilitated by binding to proteins, underlie the observed PFAS condensates inside the bacterial cells. Thus, PFAS bioaccumulation inside the cells is likely to be a consequence of biophysical properties of PFAS rather than an active, regulated, bacterial process. A key open question is the mechanism of PFAS uptake. Given their surfactant-like properties as well as our data showing active export in E. coli, passive diffusion across the membrane appears unlikely. There is no correlation between PFAS bioaccumulation by gut bacteria and the previously observed drug bioaccumulation data36, indicating distinct import mechanisms. Indeed, higher PFAS bioaccumulation by Gram-negative bacteria is in contrast with their generally lower susceptibility to drug and xenobiotic uptake60,61. The rapid kinetics of PFAS uptake by gut bacteria (timescale of minutes; Extended Data Fig. 2f) contrasts with the slower accumulation reported in environmental bacteria, which occurs over days29. This difference may reflect differences in bacterial membrane composition, surface properties, or transporter properties and expression. Thus, identifying gut bacterial transporters involved in PFAS import through, for example, functional genomic approaches, will be critical to fully elucidate microbial PFAS sequestration. By uncovering microbial PFAS bioaccumulation in molecular detail, this study provides a framework for investigating microbial contribution to PFAS toxicokinetics. Our gnotobiotic mouse studies show that gut bacteria bioaccumulate PFAS in an in vivo context. However, these experiments were done using a one-time dosage while most populations experience low chronic exposure. Therefore, cohort studies tracking PFAS intake, blood levels, urine and faecal excretion, and microbiota composition over a prolonged period will be an important next step. Strains were selected to represent prevalent and abundant members of the healthy human gut microbiota34,35 (Supplementary Tables 1 and 3). E. coli mutants were obtained from the Typas laboratory (EMBL Heidelberg, BW25113 wild type, BW25113 ∆tolC, BW25113 imp4213::FRT) and Luisi laboratory (University of Cambridge, C43 (DE3) wild type, C43 (DE3) ∆acrAB-tolC, C43 (DE3) ∆acrAB). All bacterial experiments were performed in an anaerobic chamber (Coy Laboratory Products) filled with 2% hydrogen and 12% carbon dioxide in nitrogen. Bacteria were grown at 37 °C in modified Gifu anaerobic medium (mGAM, HyServe, produced by Nissui Pharmaceuticals), prepared according to the instructions from the manufacturer and sterilized by autoclaving. Bacteria for starting cultures were grown for 1 or 2 days (depending on the growth rate) in 10 ml of media in 15-ml plastic tubes, which were inoculated directly from frozen glycerol stocks. Unless otherwise specified, the screening plates and tubes with cultivation medium were prepared the day before at 2× compound concentration (2% DMSO) and placed into the chamber overnight to ensure anaerobic conditions for inoculation. Inoculation was performed 1:1 with a bacterial culture, and plates were sealed with AlumaSeal II film (A2350-100EA) to avoid evaporation during incubation. For all measured compounds, including PFHpA, PFOA, PFNA and PFDA, pure standards were obtained from Sigma-Aldrich (Merck KGaA). All compounds were dissolved in DMSO, with the exception of NMOR, MDMA and methamphetamine, which came dissolved in methanol, and cocaine and heroin, which came dissolved in acetonitrile. A mix containing 14 PFAS compounds was purchased from Agilent (ITA-70). Labelled standards of PFNA and a mix of 13 PFAS standards were purchased from Cambridge Isotope Laboratories and Greyhound Chromatography (CLM-8060-1.2, MPFAC-C-ES). On the day of the screen, communities were assembled by pooling together second passages of individual strains according to their OD600 values (Supplementary Table 1). Assembled communities were then centrifuged at 25 °C and 3,202 × g for 15 min, and the pellet was resuspended in PBS buffer to create a community with OD600 of 7.5 (Extended Data Fig. Each well was inoculated 1:1 with the community in PBS to reach a starting OD600 of 3.75 and a compound concentration of 20 µM (1% DMSO). Plates were incubated at 37 °C for 4 h, after which they were centrifuged at 21 °C and 3,202 × g for 15 min. The supernatant and compound controls were transferred to fresh 96-well plates and stored at −80 °C until extraction. Ten compounds selected based on the ‘Community-based screening approach' were tested for sequestration by individual strains (Supplementary Table 7). On the day of the screen, each well was inoculated 1:1 with a second-passage culture to reach a starting OD600 of 0.05 and a compound concentration of 20 µM (1% DMSO) (Fig. Plates were incubated at 37 °C for 24 h, after which they were removed from the anaerobic chamber for sample collection. Whole culture, supernatant and compound control samples were collected and stored at −80 °C until extraction. Each well was inoculated 1:1 with a culture in PBS to reach the desired starting OD600 of 3.75, unless otherwise specified (Figs. Samples were incubated at 37 °C for 4 h, after which they were removed from the anaerobic chamber for sample collection. Whole culture, supernatant and compound control samples were collected and stored at −80 °C until extraction. On the day of the screen, each well was inoculated 1:1 with a second-passage culture to reach a starting OD600 of 0.05 (Fig. Samples were incubated at 37 °C for 24 h, after which they were removed from the anaerobic chamber for sample collection. Whole culture, supernatant and compound control samples were collected and stored at −80 °C until extraction. On the day of the screen, each tube was inoculated 1:1 with a second-passage culture to reach a starting OD600 of 0.05 and a concentration of 20 µM (9.3 mg l−1) PFNA (Fig. Samples were incubated at 37 °C for 11 h. Bacteria-free mGAM was added to compound control samples. OD600 was measured every hour, and supernatant, whole culture and compound control samples for PFNA analysis were collected also every hour and stored at −80 °C until extraction. On the day of the screen, each well was inoculated 1:1 with a culture in PBS to reach a starting OD600 of 3.75 and a concentration of 20 µM (9.3 mg l−1) PFNA (Fig. Bacteria-free PBS was added to compound control samples. Supernatant, pellet, whole culture and compound control samples for PFNA analysis were collected at 0 h, 4 h, 1 day, 2 days, 4 days and 7 days and stored at −80 °C until extraction. The aim of this experiment was to see whether B. uniformis would release PFAS after a prolonged period irrespective of the viability or cell death. A total of 1.5 ml of stationary second-passage cultures of B. unifomis or pure mGAM was spiked with 15 μl of 2 mM PFNA in DMSO (final concentration 20 µM (9.3 mg l−1) PFNA, 1% DMSO) (Extended Data Fig. On the day of the screen, each well was inoculated 1:1 with a live, heat-inactivated or lysed culture in PBS to reach a starting OD600 of 3.75 (Fig. Second-passage cultures were spun down, and the pellet was resuspended in PBS to an OD600 of 7.5. Each culture was split up into three aliquots: live, heat inactivated and lysed cultures. Live cultures were used as is. Bacteria were heat inactivated at 70 °C for 40 min, and lysed cultures were additionally freeze-thawed three times and sonicated for 3 min. After 4 h, whole culture, supernatant and compound control samples were collected and stored at −80 °C until extraction. Three biological replicates of B. uniformis were grown over the course of 2 days (Fig. One pellet per biological replicate was kept as negative control and stored at −80 °C until extraction. A total of 3 × 0.5 l of 160 ng l−1 PFNA in PBS without a bacterial pellet was used as the compound control and incubated for the same amount of time. For collection, samples were removed from the anaerobic chamber and centrifuged at 3,202 × g for 15 min. Supernatant and pellet samples were stored separately at −80 °C until extraction. To determine pellet concentration based on pellet weight across four PFAS compounds (PFHpA, PFOA, PFNA, PFDA), a resting cell assay and growth assay were conducted at an exposure concentration of 5 µM in 0.5 ml volume (Fig. On the day of the screen, each well was inoculated 1:1 with a second-passage culture to reach a starting OD600 of 0.05 in mGAM or a starting OD600 of 3.75 in PBS and a starting concentration of 5 µM of the respective PFAS compound (0.5 ml total volume). Samples were incubated at 37 °C for 24 h (mGAM) or 37 °C for 4 h (PBS), after which they were removed from the anaerobic chamber for sample collection. Supernatant was collected for LC–MS analysis, and the pellets were weighed and also collected for LC–MS analysis. In the resting assay, B. uniformis and E. coli pellets weighed ~8 mg and ~5 mg, respectively, meaning bacterial cells contribute only circa 1–2% to total culture weight and volume. On the day of the screen, each well was inoculated 1:1 with a culture in PBS to reach a starting OD600 of 3.75 (1 mg l−1 for each PFAS compound: perfluorobutanoic acid (PFBA), perfluorobutanesulfonic acid (PFBS), perfluoropentanoic acid (PFPeA), perfluorohexanoic acid (PFHxA), PFHxS, PFHpA, PFOA, PFOS, PFNA, PFDA, PFUnDA, perfluorododecanoic acid (PFDoDA), perfluorotridecanoic acid (PFTrDA) and perfluorotetradecanoic acid (PFTA)) (Fig. Bacteria-free PBS was added to the respective compound control wells. Samples were incubated at 37 °C for 4 h, after which they were removed from the anaerobic chamber for sample collection. Whole culture, supernatant, pellet and compound control samples were collected and stored at −80 °C until extraction. A total of 20 µl of a second-passage culture was added to 380 µl of PFAS in mGAM to reach a starting concentration of 20 µM PFNA or PFDA (Extended Data Fig. Bacteria-free mGAM was added to compound control samples. Samples were incubated at 37 °C for 24 h, after which whole culture, compound control and supernatant samples were collected for LC–MS analysis. For all PFAS assays, either mGAM or PBS with 1% DMSO was used. To test whether the solubility of PFAS compounds could affect our assays, we measured the solubility of PFNA and PFDA in mGAM (1% DMSO), PBS (1% DMSO) and 80% methanol (1% DMSO). PFNA was soluble in all conditions up to 500 µM, while PFDA was soluble up to 500 µM in 80% methanol (1% DMSO) and mGAM (1% DMSO), and up to 100 µM in PBS (1% DMSO) (Supplementary Fig. For the ‘Community-based screening approach', 70 μl of supernatant was extracted with 140 μl of ice-cold methanol:acetonitrile (1:1) containing the internal standard (IS; 20 µM caffeine, 60 µM ibuprofen) and incubated at 4 °C for 30 min. For the ‘Single strain-xenobiotic screen', a 50-μl sample was extracted with 200 μl of ice-cold methanol:acetonitrile (1:1) containing the IS (100 µM caffeine, 60 µM ibuprofen) and incubated at 4 °C for 15 min. For other follow-up in vitro assays, a 50-μl sample was extracted with 200 μl of ice-cold methanol:acetonitrile (1:1) containing the IS (60 µM ibuprofen or 20 µM caffeine) and incubated at 4 °C for 15 min. For the PFAS-mix assay, all samples were diluted 1:1 with water and 10 µl of diluted sample was extracted with 90 µl of ice-cold methanol:acetonitrile (1:1) containing the IS (final IS concentration 20 µg l−1 for each IS) at 4 °C for 15 min. Sample plates were then centrifuged at 4 °C and 3,202 × g for 10 min. Supernatants were transferred to 96-well plates for LC–MS analysis. Samples for concentration calibration and bacteria-free compound controls were processed in the same way. Supernatant and pellet samples were extracted according to the EPA 3rd Draft Method 1633 for analysis of PFAS in aqueous, solid, biosolid and tissue samples44 and the corresponding Agilent application note45 (Fig. In short, 500 ml of supernatant or compound control samples were weighed and spiked with a known concentration of an IS (1 ml of 50 µg l−1 13C9-PFNA). Pellet samples underwent three freeze–thaw cycles to ensure bacterial lysis. Samples were weighed, resuspended in 20 ml of 0.3% methanolic ammonium hydroxide, spiked with a known concentration of an IS (1 ml of 50 µg l−1 13C9-PFNA) and shaken for 30 min. Samples were then centrifuged at 1,569 × g for 10 min, and the supernatant was transferred to a fresh sample tube. The remaining pellet was resuspended in 15 ml of 0.3% methanolic ammonium hydroxide, shaken for 30 min and centrifuged again. To the combined supernatants from each pellet, circa 10 mg carbon (Agilent 5982-4482) was added. Samples were handshaken for up to 5 min and then centrifuged for 10 min at 1,569 × g. The supernatants were collected in a fresh sample tube. Agilent solid-phase extraction (SPE) cartridges (Agilent 5610-2150) were loaded with silanized glass wool (Agilent 8500-1572) fitted with large volume adaptors (Agilent 12131012 and 12131001) and conditioned with 15 ml of 1% methanolic ammonium hydroxide followed by 5 ml of 0.3 M formic acid. Samples were then loaded into SPE cartridges and set to a low flow rate of circa 5 ml min−1, followed by a rinse with 10 ml reagent water and 5 ml of 1:1 0.1 M aqueous formic acid:methanol, before being dried under the vacuum. Sample bottles were rinsed and eluted with 5 ml of 1% methanolic ammonium hydroxide. Then, 25 µl of acetic acid was added to each sample and each sample was vortexed. To each supernatant and compound control eluate, 10 mg carbon (Agilent 5982-4482) was added, and samples were handshaken for up to 5 min and then centrifuged for 10 min at 1,569 × g. All samples were filtered through a nylon syringe filter (Agilent 9301-6476, 5190-5092) into a collection tube for LC–MS analysis. Frozen faecal samples were weighed out into tubes with beads, and 250 μl extraction buffer (methanol + 0.05 KOH + 15 μM caffeine (Fig. Tubes were then homogenized at 1,500 rpm (SPEX SamplePrep LLC) for 10 min followed by centrifugation at 16,900 × g and 4 °C for 5 min. Then, 20 μl supernatant was added to 80 μl water + 0.1% formic acid, and the mixture was vortexed, incubated at 4 °C for 15 min and centrifuged at 16,900 × g and 4 °C for 5 min. The supernatant was transferred to LC–MS vials with inserts. Samples for concentration calibration were processed in the same way. LC–MS analysis was performed on an Agilent 1290 Infinity II LC system coupled with an Agilent 6546 LC/Q-TOF. The quadrupole time-of-flight (QTOF) MS scan was operated in positive or negative MS mode using four different collision energies (0 V, 10 V, 20 V, 40 V) (30–1,500 m/z), depending on the xenobiotic targeted for measurement (Supplementary Table 2). The source parameters were as follows: gas temperature, 200 °C; drying gas, 9 l min−1; nebulizer, 20 psi; sheath gas temperature, 400 °C, sheath gas flow, 12 l min−1; VCap, 3,000 V; nozzle voltage, 0 V; fragmentor, 110 V; skimmer, 45 V; and Oct RF Vpp, 750 V. The online mass calibration was performed using a reference solution (positive: 121.05 and 922.01 m/z; negative: 112.99 and 1033.99 m/z). The compounds were identified based on their retention time, accurate mass and fragmentation patterns. For all measured compounds, pure standards were used for method development, compound identification and calibration. Five different LC methods were applied. The separation was performed using a ZORBAX RRHD Eclipse Plus column (C18, 2.1 × 100 mm, 1.8 μm; Agilent 858700-902) with a ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard column at 40 °C (Community-Based Screening Approach and Extended Data Fig. The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of A: water + 0.1% formic acid + 5 mM ammonium formate, and B: methanol + 0.1% formic acid + 5 mM ammonium formate. The 15-min gradient started with 5% solvent B, which was increased to 30% by 1 min and then further increased to 100% by 7 min and held for 3 min, before returning to 5% solvent B for a 5-min re-equilibration. The separation was performed using two ZORBAX RRHD Eclipse Plus columns (C18, 2.1 × 100 mm, 1.8 μm; Agilent 858700-902) with the ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard columns at 40 °C (Fig. The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of A: water + 0.1% formic acid + 5 mM ammonium formate and B: methanol + 0.1% formic acid + 5 mM ammonium formate. The re-equilibration gradient started with 5% solvent B, which was then ramped up to 100% solvent B at 0.1 min and held until 4 min before returning to the starting condition of 5% solvent B at 4.1 min. The separation was performed using a ZORBAX RRHD Eclipse Plus column (C18, 2.1 × 100 mm, 1.8 μm; Agilent 858700-902) with ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard columns at 40 °C (Community-Based Screening Approach and Extended Data Fig. The multisampler was kept at a temperature of 4 °C. The 13-min gradient started with 35% solvent B, which was increased to 100% by 9 min and held for 1 min, before returning to 35% solvent B for a 3-min re-equilibration. Mobile phases for BPA, BPB, BPF and catechol analysis consisted of A: water and B: methanol; mobile phases for BPAF, BPS, PFNA, PFOA, patulin and 2-nitrofluorene analysis consisted of A: water + 5 mM ammonium acetate + 0.03% acetic acid and B: methanol + 5 mM ammonium acetate + 0.03% acetic acid. The separation was performed using two ZORBAX RRHD Eclipse Plus columns (C18, 2.1 × 100 mm, 1.8 μm; Agilent 858700-902) with the ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard columns at 40 °C (Figs. The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of A: water + 5 mM ammonium acetate + 0.03% acetic acid and B: methanol + 5 mM ammonium acetate + 0.03% acetic acid. The 10-min gradient started with 35% solvent B, which was increased to 100% by 7 min and held for 1.7 min, before returning to 35% solvent B at 8.8 min, which was held until 10 min. The re-equilibration gradient started with 35% solvent B, which was then ramped up to 95% solvent B at 0.1 min and held until 4 min before returning to the starting condition of 35% solvent B at 4.1 min. The separation was performed using a ZORBAX RRHD Eclipse Plus column (C18, 3.0 × 50 mm, 1.8 μm; Agilent 959757-302) at 40 °C (Extended Data Fig. The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of A: water + 5 mM ammonium acetate + 0.03% acetic acid and B: methanol + 5 mM ammonium acetate + 0.03% acetic acid. LC–MS/MS analysis was performed on an Agilent 1290 Infinity II LC system coupled with an Agilent 6470 triple quadrupole. The triple quadrupole (QQQ) was operated in dynamic multiple reaction monitoring (dMRM) mode. The source parameters were as follows: gas temperature, 300 °C; gas flow, 10 l min−1; nebulizer, 50 psi; sheath gas temperature, 300 °C; sheath gas flow, 11 l min−1; VCap, 3,500 V (positive) or 3,000 V (negative); and nozzle voltage, 2,000 V (positive) or 500 V (negative). The transitions for PFAS can be found in Supplementary Table 29. Pure standards were used for method development, compound identification and calibration. The separation was performed using a ZORBAX RRHD Eclipse Plus (C18, 3 × 50 mm, 1.8 μm; Agilent 959757-302) or a Poroshell 120 EC-C18, 1.9 μm, 2.1 × 50 mm (Agilent 699675-902) at 40 °C (Figs. The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of either A: water + 5 mM ammonium acetate + 0.03% acetic acid and B: methanol + 5 mM ammonium acetate + 0.03% acetic acid, or A: water + 0.1% formic acid and B: acetonitrile + 0.1% formic acid. The separation was performed using a ZORBAX RRHD Eclipse Plus column (C18, 2.1 × 100 mm, 1.8 μm; Agilent 858700-902) with a ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard column at 40 °C (Fig. The multisampler was kept at a temperature of 4 °C. The injection volume was 2–10 μl (Fig. 10f: 5 µl) and the flow rate was 0.4 ml min−1. The mobile phases consisted of A: water + 0.1% formic acid and B: acetonitrile + 0.1% formic acid. The 10-min gradient started with 5% solvent B, which was increased to 90% by 5 min and further increased to 100% solvent B by 7 min, before returning to 5% solvent B at 7.1 min and held until 10 min (Supplementary Fig. Samples were analysed according to the EPA 3rd Draft Method 1633 (ref. 44) and the corresponding Agilent application note45 (Fig. In short, separation was performed using a ZORBAX Eclipse Plus column (C18, 2.1 × 100 mm, 1.8 μm; Agilent 959758-902) with a ZOBRAX Eclipse Plus (C18, 2.1 × 5 mm, 1.8 μm; Agilent 821725-901) guard column at 40 °C. A PFC delay column was used (Agilent 5062-8100, 4.6 × 30 mm). The multisampler was kept at a temperature of 4 °C. The mobile phases consisted of A: 2 mM ammonium acetate in 95% water + 5% acetonitrile and B: acetonitrile. The 10-min gradient started with 2% solvent B, which was increased to 95% by 10 min, before returning to 2% solvent B for a 2-min re-equilibration. LC–MS/MS analysis was performed on an Agilent 1290 Infinity II LC system coupled with an Agilent 6470 triple quadrupole. The QQQ was operated in dMRM mode. The source parameters were as follows: gas temperature, 230 °C; gas flow, 6 l min−1; nebulizer, 20 psi; sheath gas temperature, 355 °C; sheath gas flow, 10 l min−1; VCap, 3,500 V (positive) or 2,500 V (negative); and nozzle voltage, 2,000 V (positive) or 0 V (negative). Pure standards (Agilent ITA-70) and labelled standards (Wellington Laboratories MPFAC-C-ES) were used for method development, compound identification and calibration (Supplementary Fig. Analysis was performed (Imperial College London; Extended Data Fig. 3c) using a Shimdazu Nexera X2 LC-system coupled to an LCMS-8060 triple quadrupole system with an electrospray ionization source operated in negative ionization mode (Shimadzu). Separations were performed at 0.5 ml min−1 using a Force C18 column (50 × 2.1 mm, 3 µm, Thames Restek) fitted with a matching guard column (Force C18, 5 × 2.1 mm, 5 µm), and a delay column (50 × 2.1 mm, 5 µm, Thames Restek) was installed between the mobile phase mixer and the autosampler. Sample order was randomized throughout the batch with an injection volume of 10 µl, and the autosampler was held at 4 °C for the entire analysis. The elution programme is as follows: an initial hold at 30% mobile phase B (MPB, MeOH; MPA—10 mM ammonium acetate (aq)) for 0.51 min, then an increase to 60% MPB by 1.5 min, followed by another increase to 90% MPB at 5 min, a hold at 95% MPB from 6 to 7 min, followed by a 3-min re-equilibration period at initial conditions. A 90:10 (MeOH:H2O) needle wash was used to rinse the outside of the needle before and after sample aspiration. Refer to Supplementary Table 30 for the voltages and collision energies (optimized) used in each of the quadruples for each transition of PFNA and M9PFNA. Analysis of all extracts was accompanied by quality control samples (0, 1 and 10 µg l−1 in methanol, 1% ammonium hydroxide, 0.5% acetic acid) and mobile phase blanks to account for instrumental contamination and to mitigate carryover. A deep-well plate containing 1,180 µl PFNA in mGAM was inoculated with 10 µl second-passage culture to create a final concentration of 20 µM PFNA (Fig. Samples were incubated at 37 °C for 24 h, after which 1 ml bacterial culture was transferred to a fresh tube and centrifuged at 4 °C and 3,202 × g for 3 min. Pellet samples and 100 µl of supernatant per sample were extracted with 800 µl of 1:1 methanol:water and vortexed; 200 µl chloroform was added to each sample and samples were incubated at −20 °C for 1 h for protein precipitation. Samples were then centrifuged at 4 °C and 3,202 × g for 2 min, and the water phase was transferred to LC–MS vials for analysis. Amino acids and vitamins were quantified using liquid chromatography–tandem mass spectrometry as described previously62. On an Agilent 1290 Infinity II system, analytes were separated using hydrophilic interaction liquid chromatography with a Waters Acquity BEH Amide 1.7 µm, 2.1 mm × 100 mm column operated at 35 °C. A binary buffer system of buffer A (50% acetonitrile, 10 mM ammonium formate, 0.176% formic acid) and buffer B (95:5:5 acetonitrile:methanol:water, 10 mM ammonium formate, 0.176% formic acid) was used at a constant flow rate of 0.9 ml min−1 and the following gradient: 0 min: 85% B, 0.7 min: 85% B, 2.55 min: 5% B, 2.9 min: 5% B, 2.91 min: 85% B and 3.5 min: stop time. The Agilent triple quadrupole 6470 instrument with JetStream ion source (AJS-ESI) was used in dMRM mode with a cycle time of 320 ms. The source parameters were as follows: gas temperature, 325 °C; gas flow, 10 l min−1; nebulizer, 40 psi; sheath gas temperature, 350 °C; sheath gas flow, 11 l min−1; capillary (positive), 3,500 V; capillary (negative), 3,500 V; nozzle voltage (positive), 1,000 V; and nozzle voltage (negative), 1,000 V. The injection volume was 0.25 µl. A serially diluted external calibration standard, blanks and a pooled quality control sample were injected at regular intervals between samples. Data were analysed using MassHunter Workstation Quantitative Analysis for QQQ v10.1. The Agilent MassHunter Qualitative Analysis 10.0 software was used to qualify the selected xenobiotic standards. Total ion chromatogram (TIC), extracted ion chromatogram (EIC) and EIC-fragment graphs were extracted for each compound. The Agilent MassHunter TOF Quantitative Analysis (version 10.1) or Agilent MassHunter QQQ Quantitative Analysis software (version 10.1) was used to quantify the xenobiotic compounds in each sample. Calibration curves based on pure compound standards were used to estimate the concentrations of target compounds. 2i, low concentration experiment per EPA Draft Method 1633, PFAS-mix experiment and mouse faecal analysis, in which the ratio of the compound response to internal standard response was used. Data analysis was performed in RStudio Version 1.3.1093. The median of each sample group was compared with the median of the compound controls, and an appropriate reduction of more than 20% was chosen as cut-off, to ensure relevant reduction compared with the compound control distribution. In addition, statistical comparison was performed using t-test (two sided), and P values were false discovery rate (FDR) corrected (when a t-test was performed, the adjusted P values are given in the respective figures). An adjusted P value of less than 0.05 was considered significant. In the ‘Single strain-xenobiotic screen', bioaccumulation was defined as compound sequestration to at least 20% from the supernatant but not from the whole culture sample, whereas biodegradation was defined as both supernatant and whole culture sample showing more than 20% compound sequestration. Concentration in samples extracted according to EPA Draft Method 1633 was calculated based on the ratio with the labelled IS, which was spiked into the sample at a known concentration of 100 ng l−1. Plates (Corning 3795) containing 50 μl mGAM with 2× PFAS (2% DMSO) concentration were prepared the evening before and placed into the anaerobic chamber overnight to ensure anaerobic conditions for inoculation (Fig. On the day of the screen, each well was inoculated with 50 μl of a second-passage culture to reach a starting OD600 of 0.05. The plates were sealed with a gas-permeable membrane (Breath-Easy, Merck, catalogue number Z380059), which was additionally pierced with a syringe to prevent gas build-up. The plates were stacked without lids and incubated at 37 °C for 24 h in a stacker–incubator system (Biostack 4, Agilent BioTek) connected to a plate reader (Epoch 2, Agilent BioTek) to record the OD600 (ref. Growth curve analysis was performed in RStudio version 1.3.1093. First, for each growth curve, the minimum OD value was set to 0. Then, the raw area under the curve (AUC) was calculated for each well using ‘bayestestR' package and area_under_curve() function. Further processing of growth curves was done by plate. AUC values were normalized by median AUC of all control wells (DMSO controls) on the respective plate to determine percentage growth inhibition. On the day of the experiment, each tube containing 2× concentration of PFAS was inoculated 1:1 with a second-passage culture to reach a starting OD600 of 0.05 (Extended Data Fig. Samples were incubated at 37 °C for 24 h, after which bacterial cultures were spun down and the supernatant was removed. The bacteria were then fixed with a half Karnovsky fixative as 2.5% glutaraldehyde and 2% paraformaldehyde in 0.1 M sodium cacodylate buffer (pH 7.4 with NaOH) for a few hours at room temperature. For conventional TEM64, the post-fixation was performed with a mixture of 1% osmium tetroxide and 1% potassium ferrocyanide in the cacodylate buffer. The bloc was stained with 5% aqueous uranyl acetate solution. The dehydration with a series of ethanol and the resin infiltration were completed for the plastic embedding in TAAB epoxy resin. After the polymerization at 65 °C for a few days, the ultrathin sections (∼60 nm) were cut using an ultramicrotome (Leica EM UCT/UC7/Artos-3D), mounted on formvar-carbon films on EM copper grids and stained with lead citrate. The bacterial ultrastructure was observed using a FEI Talos F200C 200 kV transmission electron microscope (Thermo Fisher Scientific) with a Ceta-16M CMOS-based camera (4k × 4k pixels under a 16-bit dynamic range) and a JEM-1400 Flash TMP (JEOL) with TVIPS TemCam-XF416 CMOS (Tietz Video and Image Processing Systems) as described previously65. Electron microscopy images were converted to an 8-bit format using FIJI66 (Extended Data Fig. Only images with the same magnification level were analysed. The segmentation and quantification of these images were performed using CellProfiler 4.2.6 (Stirling, DR, BMC Bioinformatics, 202167). Individual cells were segmented as primary objects based on their intensity and size after the application of a Gaussian blur filter. Within each cell, condensates were also segmented based on intensity and size. Mean pixel intensity and number of condensates were quantified for each cell and their corresponding condensates, and the results were exported as CSV files. Cells touching the edges of the images were not considered for this quantification. Images showing the segmented areas were saved as TIFF files. All sectioned cells and aggregates were manually checked, and wrong sectioned cells and cells that were identified in duplicate were excluded from further analysis. Cryogenic FIB-SIMS imaging68 was performed on a focused ion beam scanning electron microscope (Zeiss Crossbeam 550) equipped with a time-of-flight mass spectrometer (Tofwerk; Fig. During FIB-SIMS imaging, the sample is scanned by a gallium focused ion beam, which ablates material at each pixel. The secondary ions resulting from the interaction of the ion beam and sample are extracted at each scanning position and are analysed by time-of-flight secondary electron mass spectrometry. The simultaneous detection of secondary electrons yields spatial images of the sample (FIB images, Fig. As material is continuously removed during the imaging process, repeated scanning of a given sample area provides three-dimensional information regarding both spatial features and chemical composition of the sample, thus creating a volume with a mass spectrum for each voxel. This not only allows this method to show the lateral localization of elements of interest, but also provides depth information: each two-dimensional image resulting from a full scan of the observed area can be considered as a slice of the sample (frame) within a Z-stack (Fig. To retain a near-native state of the sample, E. coli ∆tolC cells were plunge-frozen in liquid ethane using a Vitrobot Mark IV (Thermo Fisher) after applying 2.5 µl of sample to a cryo-EM grid (Quantifoil Cu/Rh R3.5/1, UltrAuFoil R 1.2/1.3) as described previously69. A deep-well plate containing 1,180 µl PFNA in mGAM was inoculated with 10 µl second-passage culture to create a final concentration of 20 µM PFNA (Fig. Samples were incubated at 37 °C for 24 h, after which bacterial cultures were spun down and the supernatant was removed. The bacterial cell pellets were lysed in 100 mM triethyl ammonium bicarbonate containing 0.1% RapiGest surfactant by sonication on ice (50J ×5, 30% amplitude) after denaturing by two rounds of incubation at 80 °C for 10 min with intermittent cooling. The solubilized protein content was estimated using Pierce 660 nm assay as per the manufacturer's instructions. Then, 50 µg of protein from each sample was reduced with 4 mM dithiothreitol, alkylated with 14 mM iodoacetamide and further digested with trypsin at 1:50 protease to protein ratio for 16 h at 37 °C. The peptide content was estimated at 1:10 dilution using Pierce quantitative colorimetric assay as per the manufacturer's instructions. Equal amounts of peptides from respective bacterial samples were labelled with TMT in a randomized fashion to ensure that the reporter ion isotopic distribution is spread across the replicates following the manufacturer's guidelines (Thermo Fisher Scientific). An equal volume of the respective tandem mass tag (TMT)-labelled peptides was pooled together and desalted to remove unbound labels. The desalted peptides were further subjected to fractionation based on their reverse-phase chromatographic properties under basic pH using an analytical high-performance liquid chromatography (HPLC). The 12 concatenated fractions collected were vacuum dried, resuspended in 0.1% trifluoroacetic acid containing 3% acetonitrile and taken for LC–MS/MS analysis. LC–MS/MS analysis was carried out in RTS-SPS-MS3 mode for TMT reporter ion quantification. An equal amount of predetermined sample load from each of the 12 fractions was analysed using an Orbitrap Eclipse mass spectrometer with an Ultimate 3000 RSLC nano chromatography system coupled in-line. The peptides were loaded onto the trapping column (Thermo Fisher Scientific, PepMap100, C18, 300 μm × 5 mm), using partial loop injection, washed for 3 min at a flow rate of 15 μl min−1 with 0.1% formic acid (FA). The peptides were resolved on an analytical column (Easy-Spray C18 75 µm × 500 mm; 2 µm particle size) at a flow rate of 300 nl min−1 using a gradient in which the percentage of B is raised from 3–25 over 140 min and then to 40% for an additional 13 min. The column was washed in 90% B for 12 min and re-equilibrated in 3% B for 15 min before next injection. Subsequently, 0.1% FA in water was used as mobile phase A and 0.1% FA in 80% acetonitrile was used as mobile phase B. For each FAIMS experiment with a maximum cycle time of 2 s, mass spectrum acquisition was carried out in RTS-SPS-MS3 mode. In this mode, full-scan MS was acquired in the mass range of 415–1,500 m/z at 120,000 resolution with a maximum ion injection time of 30 ms (AGC target 2e5 ions). Precursors selected for MS/MS were isolated using an isolation width of 0.7 m/z and fragmented by HCD using collision energy of 32%. An MS/MS scan was performed on the ion trap at a turbo scan rate (AGC 5e4 ions) with a maximum ion injection time of 22 ms. To avoid repeated selection of peptides for MS/MS, the programme used a 40-s dynamic exclusion window. Real-time search parameters were set as follows using respective fasta databases: Uniprot B. uniformis database, UP000014212 (downloaded on 21 June 2023, contains 3,957 sequences); Uniprot O. splanchnicus database, UP000006657 (downloaded on 21 June 2023, contains 3,479 sequences); and Uniprot E. coli K12 database, UP000000625 (downloaded on 21 June 2023, contains 4,362 sequences). Protease: trypsin, static modifications: cysteine carbamidomethylation (+57.0215) and TMT (+229.163) or TMTPro (+304.207) on lysine and N-terminus, variable modification: methionine oxidation (+15.99491). Only 5 peptides per protein were allowed per basic reverse-phase fraction (using the close-out option), and the maximum search time for RTS was set to 35 ms. Ten of the most abundant peptide fragments were selected for SPS-MS3, and the MS3 spectrum was acquired at 120,000 resolution on the mass range 100–500 m/z with an AGC target of 500% and a maximum injection time of 246 ms. The spectrum identification was performed with the following parameters: MS accuracy, 10 ppm; MS/MS accuracy, 0.6 Da for spectra acquired in an ion trap mass analyser; missed cleavage, up to 2; fixed modifications, carbamidomethylation of cysteine and TMT/TMTpro on lysine and on peptide N-terminus; and variable modifications, oxidation of methionine. An intensity-based rescoring of PSMs identified in SequestHT node was performed by Inferys before false discovery rate estimation using Percolator. Only rank 1 peptide identifications with high confidence (FDR < 1%) were accepted for identification and quantification reporting. The normalized abundance values are reported for comparison across datasets. TPP was performed as previously described36,70 (Fig. In brief, cells were grown anaerobically at 37 °C for 48 h. Cells were then washed twice with anaerobic PBS. For the live cell experiments, cells were treated with eight different concentrations of anaerobic PFNA (twofold dilutions starting at 10 µM including DMSO control) for 30 min at 37 °C in the anaerobic chamber. Aliquots of treated cells were then incubated in 10 different temperatures for 3 min. Cells were resuspended in lysis buffer (50 µg ml−1 lysozyme, 1 mM MgCl2, complete protease inhibitors, 0.25 U µl−1 benzonase in PBS) with 0.8% NP-40 and lysed with 5 freeze–thaw cycles. For the lysate experiments, cells were lysed as above but without the presence of NP-40. Eluted peptides were labelled with TMT16plex, pooled (all treatments of each two adjacent temperatures per experiment) and pre-fractionated into six fractions under high-pH conditions. Samples were then analysed with liquid chromatography coupled to tandem mass spectrometry, as previously described70. All raw files were converted to mzmL format using MSConvert from Proteowizard, using peak picking from the vendor algorithm. Files were then searched using MSFragger v3.7 (ref. 73) in Fragpipe v19.0 against the Uniprot proteome UP000000625 containing common contaminants and reversed sequences. Data analysis was performed using R. Only proteins with at least two identified peptides were kept for the analysis, and outlier conditions were removed altogether. For the correlation analysis, per experiment, protein intensities across all concentrations within each temperature were normalized using variance stabilization normalization74 and the cumulative fold change across temperatures and concentrations per protein was calculated. DMSO was used as a control. For each PFAS compound 4 and for the DMSO controls 8 replicate lineages were evolved in parallel. Subsequently, 2-ml deep-well stock plates containing a 100× stock of each PFAS in DMSO were prepared before the experiment and stored at −80 °C until use. On day 0, each well was inoculated with a second-passage culture to reach a starting OD600 of 0.05. On each of the following days, 50 μl of grown culture was transferred to a fresh compound plate containing PFAS–DMSO in mGAM. Every 5 days, the growth of the strains in the presence of PFAS was measured by transferring 100 μl of starting culture to a clear-bottom plate, measuring and analysing it as described in the section ‘PFAS–Bacteria Growth Screens'. On day 20, glycerol stocks were prepared from each lineage and stored at −80 °C. Genomic DNA of B. uniformis after evolving in PFNA, PFOA and DMSO control was extracted with the QIAamp PowerFecal Pro DNA kit (Qiagen; 51804) and quantified using Qubit 1X dsDNA HS Assay Kits (Q33230). On average, 74.38 ng µl−1 of DNA was obtained and sequenced by Illumina NovaSeq 6000 with a library insert size of 350 bp, resulting in 6.50 to 9.95 million paired reads (mean = 8.68 million) with a length of 150 bp. Sequencing reads were then filtered with fastp (v 0.32.2)76 using default parameters. We called the variants in evolved strains using snippy pipeline (https://github.com/tseemann/snippy). Filtered reads from PFNA-, PFOA- and DMSO-evolved populations were aligned to the reference genome with BWA, and variants were called by freebayes77 using parameters ‘–mincov 10–basequal 30–fbopt '-C 3 -p 1 -F 0.01'' defined through snippy. Afterwards, those variants appearing in both PFNA–PFOA and DMSO control samples were removed. Animal experiments were approved by the local authorities (Regierungspräsidium Tübingen, H 02/20 G) (Fig. GF C57BL/6J mice were bred in house (Gnotobiotic Mouse Facility). Mice were housed under GF conditions in flexible film isolators (Zoonlab) and transferred to the Isocage P system (Tecniplast) to perform the experiments. The housing conditions were 12:12-h light–dark cycles; temperature, 22 °C ± 2 °C; and humidity, 50–56%. Mice were supplied with autoclaved drinking water and γ-irradiated maintenance chow for mice (Altromin) ad libitum. Female and male mice between 5 and 6 weeks old were randomly assigned to experimental groups. Mice were kept in groups of three per cage during the experiment. All animals were scored daily for their health status. Consumables, glassware and media were pre-reduced at least 2 days before inoculation of bacteria. Each strain was grown in monoculture overnight in 5 ml of their respective growth medium at 37 °C. The next day, bacteria were subcultured (1:100) in 5 ml fresh medium and incubated for 16 h at 37 °C, except Eggerthella lenta, which was grown for 2 days. OD at 578 nm was determined, and bacteria were mixed together in equal ratios to a total OD of 0.5 in a final volume of 10 ml. After 2.5 ml of 50% glycerol (with a few crystals of palladium black (Sigma-Aldrich)) was added, 200-µl aliquots were prepared in glass vials (2 ml; Supelco, ref. Frozen vials were used within 3 months. Inoculation of GF mice was performed according to a previous study55. In short, cages were transferred to an ISOcage Biosafety Station (IBS) (Tecniplast) through a 2% Virkon S disinfectant solution (Lanxess) dipping bath. Glycerol stocks of the frozen communities (one per mouse) were kept on dry ice before being thawed during transfer into the IBS. Mixtures were used directly after thawing with a minimal exposure time to oxygen of maximum 3 min. Mice were inoculated by oral gavage (50 µl) and inoculation was repeated after 48 h using the same protocol. The GF control group was left untreated. The IBS was sterilized with 3% perchloracetic acid (Wofasteril, Kesla Hygiene AG). Mice were orally gavaged with PFNA (10 mg kg−1 or 0.1 mg kg−1 in 25% DMSO) in a volume of 50 µl 10 days after the second inoculation with Com20, or high- or low-accumulating communities. The human equivalent dose of 0.1 mg kg−1 is 8.1 µg kg−1 (ref. 78), which amounts to 486 µg PFNA for a 60-kg person. Given the EU drinking water limit for individual PFAS at 0.1 µg l−1 and an estimated daily water intake of 4 l, this dose corresponds to approximately 3 years of exposure. Fresh faecal samples were collected before treatment and 3 h, 1 day and 2 days after treatment in sterile 1.5-ml Eppendorf tubes and immediately frozen at −80 °C. DNA extraction from faecal samples was performed in house using a MagMAX Microbiome Ultra Nucleic Acid Isolation Kit in bead tubes (Thermo Fisher Scientific, A42358) and the KingFisher Flex (MAN0018071) according to the manufacturer's instructions. Subsequently, DNA integrity was verified using agarose gel electrophoresis and the DNA concentration was determined using a Qubit dsDNA BR assay kit (Thermo Fisher Scientific, Q32853) in combination with the Varioskan LUX plate reader (Thermo Fisher Scientific). A two-step PCR protocol was used to prepare 16S amplicons. A single amplicon was generated using V4 primer pairs (515F/806R) for the first mouse experiment (10 mg kg−1 PFNA, Com20 versus GF) and V4 and V5 primer pairs (515F/926R) for the second mouse experiment (10 mg kg−1 PFNA, high- versus low-accumulating community). In the first step PCR, primers with overhang adapter sequences were added; afterwards, in the second PCR, Illumina sequencing adaptors and dual‐index barcodes were added to the amplicon target for pooling. Pooled samples are sequenced on an Illumina MiSeq instrument using a 2 × 250 PE protocol at the Genomics Core Facility (EMBL Heidelberg). In total, 1,086,628 pair-end reads for the first mouse experiment and 6,531,459 for the second mouse experiment (an average of 10,061 pairs per sample in the first experiment and 56,306 pairs per sample in the second experiment) with 250 bp were generated. Raw sequencing reads were truncated and filtered using the DADA2 pipeline (v1.26.0)79 with the following parameters: ‘truncLen=c(230,210), maxN=0, maxEE=c(2,2), truncQ=2, rm.phix=TRUE'. Afterwards, error rates were learned from filtered reads, and corrected reads were merged as ASVs. A self-defined reference was used for alignments of ASVs and calculation of relative abundances. All data analysis was performed using open-source packages accessed from RStudio (version 1.3.1093). The Agilent MassHunter TOF Quantitative Analysis (version 10.1) or Agilent MassHunter QQQ Quantitative Analysis software (version 10.1) was used to quantify the xenobiotic compounds in each sample. All P values are FDR corrected. Biological replicates refer to different inoculation cultures, while technical replicates refer to experiments starting with the same inoculation culture. None of the data points correspond to repeated measurements of the same sample. Fold changes refer to the ratio of medians. No randomization was used in microbiological experiments. In animal experiments, female and male mice were housed in separate cages, and subsequently, cages were randomly assigned to experimental groups. Each experimental group included mice from both sexes. Data collection and analysis were not performed blind to the conditions of the experiments. No statistical methods were used to predetermine sample sizes. Sample sizes were chosen on previous experience and according to those reported in previous publications36,56. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data that support the findings of this study are included in Supplementary Information. TEM images are available at the EBI Bioimage Archive (S-BIAD1892). The raw proteomics and TPP data have been deposited in the ProteomeXchange Consortium via the PRIDE80 partner repository with dataset identifiers PXD050999 and PXD062206. Raw sequencing reads of the evolved B. uniformis and 16S rRNA genotyping were uploaded to the European Nucleotide Archive (PRJEB72794 and PRJEB75767). No custom algorithms were used for data analysis. The code used is available via GitHub at https://github.com/AnnaELindell/PFAS-paper. & Scheringer, M. Outside the safe operating space of a new planetary boundary for per- and polyfluoroalkyl substances (PFAS). Planetary boundaries: guiding human development on a changing planet. The C8 health project: design, methods, and participants. Caporale, N. et al. From cohorts to molecules: adverse impacts of endocrine disrupting mixtures. Calafat, A. M. et al. Serum concentrations of 11 polyfluoroalkyl compounds in the U.S. population: data from the National Health and Nutrition Examination Survey (NHANES). & Needham, L. L. Polyfluoroalkyl chemicals in the U.S. population: data from the National Health and Nutrition Examination Survey (NHANES) 2003–2004 and comparisons with NHANES 1999–2000. Lewis, R. C., Johns, L. E. & Meeker, J. D. Serum biomarkers of exposure to perfluoroalkyl substances in relation to serum testosterone and measures of thyroid function among adults and adolescents from NHANES 2011–2012. Gut microbiota in human metabolic health and disease. Lindell, A. E., Zimmermann-Kogadeeva, M. & Patil, K. R. Multimodal interactions of drugs, natural compounds and pollutants with the gut microbiota. Lindell, A. E. et al. Off-purpose activity of industrial and agricultural chemicals against human gut bacteria. Summary Report on the New Comprehensive Global Database of Per- and Polyfluoroalkyl Substances (PFASs). OECD Series on Risk Management of Chemicals (2018); https://doi.org/10.1787/1a14ad6c-en (2018). Gluge, J. et al. An overview of the uses of per- and polyfluoroalkyl substances (PFAS). Horel, S., Martinon, L. & Pilz, S. The Forever Pollution Project https://foreverpollution.eu/ (2023). Emerging Chemical Risks in Europe—‘PFAS' (European Environment Agency, 2019); https://www.eea.europa.eu/publications/emerging-chemical-risks-in-europe Schrenk, D. et al. Risk to human health related to the presence of perfluoroalkyl substances in food. Salvatore, D. et al. Presumptive contamination: a new approach to PFAS contamination based on likely sources. Risks of PFAS for Human Health in Europe (Signal) (European Environment Agency, 2025); https://www.eea.europa.eu/en/european-zero-pollution-dashboards/indicators/risk-of-pfas-in-humans?activeTab=265e2bee-7de3-46e8-b6ee-76005f3f434f Official Journal of the European Union (2020); https://eur-lex.europa.eu/eli/reg_del/2020/784/oj Official Journal of the European Union (2020); https://eur-lex.europa.eu/eli/reg/2022/2388/oj ECHA publishes PFAS restriction proposal https://echa.europa.eu/-/echa-publishes-pfas-restriction-proposal (2023). Verma, S., Lee, T., Sahle-Demessie, E., Ateia, M. & Nadagouda, M. N. Recent advances on PFAS degradation via thermal and nonthermal methods. Substantial decrease of PFAS with anion exchange resin treatment—a clinical cross-over trial. Presentato, A. et al. On the ability of perfluorohexane sulfonate (PFHxS) bioaccumulation by two Pseudomonas sp. strains isolated from PFAS-contaminated environmental matrices. Shen, Z., Ge, J., Ye, H., Tang, S. & Li, Y. Cholesterol-like condensing effect of perfluoroalkyl substances on a phospholipid bilayer. Fitzgerald, N. J. M. et al. Partitioning and accumulation of perfluoroalkyl substances in model lipid bilayers and bacteria. Sobolewski, T. N. T. et al. Nanomolar PFOA concentrations affect lipid membrane structure: consequences for bioconcentration mechanisms. Capabilities of bio-binding, antioxidant and intestinal environmental repair jointly determine the ability of lactic acid bacteria to mitigate perfluorooctane sulfonate toxicity. Dai, D. et al. GMrepo v2: a curated human gut microbiome database with special focus on disease markers and cross-dataset comparison. Tramontano, M. et al. Nutritional preferences of human gut bacteria reveal their metabolic idiosyncrasies. Klunemann, M. et al. Bioaccumulation of therapeutic drugs by human gut bacteria. Abraham, K. et al. Kinetics of 15 per- and polyfluoroalkyl substances (PFAS) after single oral application as a mixture—a pilot investigation in a male volunteer. Milo, R., Jorgensen, P., Moran, U., Weber, G. & Springer, M. BioNumbers—the database of key numbers in molecular and cell biology. & Branduardi, P. The plasma membrane at the cornerstone between flexibility and adaptability: implications for Saccharomyces cerevisiae as a cell factory. Smalling, K. L. et al. Per- and polyfluoroalkyl substances (PFAS) in United States tapwater: comparison of underserved private-well and public-supply exposures and associated health implications. Du, D. et al. Multidrug efflux pumps: structure, function and regulation. Maier, L. et al. Extensive impact of non-antibiotic drugs on human gut bacteria. The impact of the genetic background on gene deletion phenotypes in Saccharomyces cerevisiae. 3rd Draft Method 1633 Analysis of Per- and Polyfluoroalkyl Substances (PFAS) in Aqueous, Solid, Biosolids, and Tissue Samples by LC-MS/MS (United States Environmental Protection Agency, 2022); https://www.epa.gov/system/files/documents/2024-12/method-1633a-december-5-2024-508-compliant.pdf Hunt, K., Hindle, R., Anumol, T., Giardina, M., Juck, M. & Parry E. Analysis of Per- and Polyfluoroalkyl Substances (PFAS) in Aqueous Samples Per EPA Draft Method 1633. Bennett, B. D. et al. Absolute metabolite concentrations and implied enzyme active site occupancy in Escherichia coli. A., Valle-González, E. R. & Cho, N. J. Antibacterial free fatty acids and monoglycerides: biological activities, experimental testing, and therapeutic applications. Thermal profiling reveals phenylalanine hydroxylase as an off-target of panobinostat. Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry. Kennedy, P. J., Cryan, J. F., Dinan, T. G. & Clarke, G. Kynurenine pathway metabolism and the microbiota–gut–brain axis. Targeted modification of gut microbiota and related metabolites via dietary fiber. Feehily, C. & Karatzas, K. A. G. Role of glutamate metabolism in bacterial responses towards acid and other stresses. Otaru, N. et al. GABA production by human intestinal Bacteroides spp. : prevalence, regulation, and role in acid stress tolerance. & Lee, J. K. Physiological polyamines: simple primordial stress molecules. Grießhammer, A., et al. Non-antibiotic Drugs Break Colonization Resistance Against Pathogenic Gammaproteobacteria (Cold Spring Harbor Laboratory, 2023). Comparative pharmacokinetics of perfluorononanoic acid in rat and mouse. Kost, C., Patil, K. R., Friedman, J., Garcia, S. L. & Ralser, M. Metabolic exchanges are ubiquitous in natural microbial communities. Binding of per- and polyfluoroalkyl substances (PFAS) to serum proteins: implications for toxicokinetics in humans. Sobolewski, T. N., Findlay, J. L., Hemphill, J. E. & Walker, R. A. Aggregation, not micellization: perfluorooctanoic acid, perfluorobutanesulfonic acid, and potassium perfluorooctanesulfonate behavior in aqueous solution. & Gnanakaran, S. Permeability barrier of Gram-negative cell envelopes and approaches to bypass it. Defining new chemical space for drug penetration into Gram-negative bacteria. Mülleder, M. et al. Functional metabolomics describes the yeast biosynthetic regulome. Müller, P. et al. High-throughput anaerobic screening for identifying compounds acting against gut bacteria in monocultures or communities. Minami, I. et al. A small molecule that promotes cardiac differentiation of human pluripotent stem cells under defined, cytokine- and xeno-free conditions. The C terminus of p73 is essential for hippocampal development. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. The 3D OrbiSIMS—label-free metabolic imaging with subcellular lateral resolution and high mass-resolving power. Structure of the Pseudomonas aeruginosa PAO1 Type IV Pilus (Cold Spring Harbor Laboratory, 2024). The functional proteome landscape of Escherichia coli. Hughes, C. S. et al. Single-pot, solid-phase-enhanced sample preparation for proteomics experiments. Kong, A. T., Leprevost, F. V., Avtonomov, D. M., Mellacheruvu, D. & Nesvizhskii, A. I. MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics. Huber, W., von Heydebreck, A., Sultmann, H., Poustka, A. & Vingron, M. Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Kurzawa, N. et al. A computational method for detection of ligand-binding proteins from dose range thermal proteome profiles. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Garrison, E. & Marth, G. Haplotype-based variant detection from short-read sequencing. & Jacob, S. A simple practice guide for dose conversion between animals and human. Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. In situ structure and assembly of the multidrug efflux pump AcrAB-TolC. Blanco, P. et al. Bacterial multidrug efflux pumps: much more than antibiotic resistance determinants. Gonzalez (MRC Toxicology Unit, University of Cambridge, UK) for helpful discussions on TEM imaging, E. Kafkia (University of Copenhagen, Denmark) for advice on mass spectrometry, and A. Typas (EMBL Heidelberg, Germany) and B. F. Luisi (Department of Biochemistry, University of Cambridge, UK) for providing the E. coli efflux pump mutants. acknowledge funding from the UK Medical Research Council (project number MC_UU_00025/11). L. Maier acknowledges funding from the German Research Council (DFG) (grant EXC2124). was supported by the EIPOD4 programme from the European Union's Horizon 2020 research and innovation programme under Marie Skłodowska-Curie (grant 847543). was supported by BBSRC Responsive Mode award BB/Y000730/1. Anna E. Lindell, Stephan Kamrad, Rui Guan, Sonja Blasche, Bini Ramachandran, Nonantzin Beristain-Covarrubias, Juan Carlos Yam-Puc, Indra Roux, Maria Guerra Martin, Nobuhiro Morone, James E. D. Thaventhiran & Kiran R. Patil Interfaculty Institute of Microbiology and Infection Medicine, M3 Research Center, Cluster of Excellence ‘Controlling Microbes to Fight Infection', University of Tübingen, Tübingen, Germany Anne Grießhammer, Lena Michaelis & Lisa Maier European Molecular Biology Laboratory (EMBL), Heidelberg, Germany Dimitrios Papagiannidis, Hilal Ozgur, Vladimir Benes & Mikhail M. Savitski Structural Studies Division, MRC Laboratory for Molecular Biology, Cambridge, UK Medical Research Council Centre for Environment and Health, Environmental Research Group, School of Public Health, Imperial College London, London, UK NIHR Health Protection Research Unit in Environmental Exposures and Health, Imperial College London, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and planned the overall study. contributed to mass spectrometry experiments, mouse faecal sample processing and data analysis. advised on and contributed to microbiological experiments. performed an independent LC–MS analysis of the 160 ng l−1 PFNA experiment samples. performed and analysed the proteomics experiment. performed and analysed the TPP experiments. advised on the design of the animal experiments and contributed to animal sample processing. carried out the cryogenic FIB-SIMS analysis. All authors commented on the paper. are inventors in a patent application related to the findings presented in the paper (UK patent application number 2215307.6). The other authors declare no conflicts of interest. Nature Microbiology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Method workflow for the artificial community experiment. animalis, Odoribacter splanchnicus, Parabacteroides distastonis, Ruminococcus bromii). 13 out of 42 tested pollutants were sequestered by at least one synthetic gut bacterial community. Coloured squares denote sequestration, that is, >20% reduction from the bacteria-free supernatant. n = 6 (3 biological and 2 technical replicates) (Supplementary Table 31). c. Method workflow for the single strain experiment. Distribution of PFNA accumulation across all 89 strains (Supplementary Tables 3,9). b. Gram-negative strains show on average higher PFNA accumulation compared to gram-positive strains (Supplementary Tables 3,9). c. Comparison of PFNA accumulation from the resting assay in PBS compared to the growth assay in mGAM (Fig. 1c including compound control and whole culture samples. PFNA depletion by B. uniformis cultures of varying OD600 in PBS buffer. All ODs show significant PFNA accumulation compared to the compound control. OD600 = 1–8; 20 μM initial PFNA concentration; two-sided t-test; *** p < 0.001; n = 4 technical replicates (Supplementary Table 10). 1d including compound control and whole culture samples. Kinetics of PFNA depletion during B. uniformis growth starting with low cell density. A significant amount of PFNA was sequestered from the media after 8 h of growth and onwards. Initial OD600 = 0.05; 20 μM initial PFNA concentration; * p<0.05 and >20% PFNA sequestration from the media (Supplementary Tables 11,12). p = 0.015 (8h), 0.005 (9h), 0.012 (10h), and 0.014 (11h) (two-sided t-test; n = 3 biological replicates). f. Kinetics of PFNA depletion by B. uniformis when starting with high cell density in mGAM (OD600 = 4). 50% PFNA happens within the time frame of sample collection (ca. 1g including compound control and whole culture samples. Bioaccumulation of PFAS compounds with varying chain length by B. uniformis. PFNA and PFDA are significantly accumulated compared to the compound control. OD600 = 3.75; 20 μM initial concentration for all compounds; two-sided t-test; ** p-value < 0.01; *** p-value < 0.001; n = 3 technical replicates (Supplementary Table 15). 1f including compound control and whole culture samples. PFNA is bioaccumulated by B. uniformis grown in mGAM at a range of initial concentrations compared to the compound control. initial OD600 = 0.05; initial PFNA concentrations = 0.01 to 100 μM; Two-sided t-test; p-value FDR corrected for number of concentrations tested; ** adj. p-value < 0.001; n = 4 technical replicates (Supplementary Table 14). i. PFNA and PFOA are bioaccumulated by B. uniformis in PBS at a range of initial concentrations. OD600 = 3.75; initial PFAS concentrations = 0.78 to 500 μM; Two-sided t-test; p-value FDR corrected for number of concentrations tested; ** adj. p-value < 0.001; n = 3 technical replicates (Supplementary Table 33). 2c including compound control and whole culture samples. Accumulation of PFDA and PFNA by wild-type E. coli strains and corresponding efflux mutants. OD600 = 3.75; exposure concentration = 20 µM; two-sided t-test; ** p-value < 0.01 and >20% sequestration compared to the compound control; *** p-value < 0.001 and >20% sequestration compared to the compound control; n = 3 technical replicates (Supplementary Table 19). b. Efflux mutants show increased PFAS sensitivity. c. Independent measurement of samples from Fig. Two-sided t-test (supernatant compared to the compound control; pellet compared to pellet control); * p-value < 0.05; ** p-value < 0.01; n = 3 biological replicates (Supplementary Table 34). 2d displayed in a different way: based on wet pellet weight, concentration within the bacterial pellet was a median of circa 8200 ng/l, which is a 50-fold increase compared to the exposure concentration. Two-sided t-test (supernatant compared to the compound control; pellet compared to pellet control); * p-value < 0.05; ** p-value < 0.01; n = 3 biological replicates (Supplementary Tables 20, 21). 2e: PFAS recovery from pellet and supernatant after exposure to 5 µM PFAS in mGAM (initial OD600 = 0.05, 24 h incubation) or PBS (OD600 = 3.75, 4 h incubation). Bars depict median percent recovery from the bacterial pellet, supernatant or compound control; error bars depict standard error; overlaid points depict single data points; n = 3 technical replicates; (Supplementary Table 35). 2f including compound control, whole culture and supernatant samples. PFAS recovery after 1 h exposure in PBS (OD600 = 3.75, PFAS mix of 14 compounds each at a concentration of 1 mg/l). For PFAS compounds with a chain length of more than ten carbon atoms, the solubility is limited at this concentration; the low accumulation in E. coli BW25113 wild-type pellets acts as a biological control in this case. Bars depict median percent recovery from the compound control, whole culture, supernatant or bacterial pellet; error bars depict standard error; overlaid points depict single data points; n = 3 technical replicates; (Supplementary Table 36). a-f. TEM of B. uniformis cells grown in mGAM + DMSO (59 images from 3 biological replicate) (a), 5 μM PFNA (31 images from 1 biological replicate) (b), 50 μM PFNA (40 images from 1 biological replicate) (c), 125 μM PFNA (51 images from 1 biological replicate) (d), 250 μM PFNA (79 images from 3 biological replicate) (e), or 125 μM PFDA (46 images from 1 biological replicate) (f) for 24 h. g,h. BW25113 wild-type cells grown in mGAM + DMSO (40 images from 1 biological replicate) (g) or 250 μM PFNA (45 images from 1 biological replicate) (h) for 24 h. i,j. BW25113 ∆tolC cells grown in mGAM + DMSO (31 images from 1 biological replicate) (i) or 250 μM PFNA (50 images from 1 biological replicate) (j) for 24 h. k,l. TEM of O. splanchnicus cells grown in mGAM + DMSO (37 images from 1 biological replicate) (k) or 250 μM PFNA (38 images from 1 biological replicate) (l) for 24 h. A tomogram for B. uniformis cells exposed to 250 µM PFNA confirming that the structures are inside the cells is available as a supplementary video. a-f. Bacteria show distinct morphological features in transmission electron microscopy (TEM). Automatic identification of bacterial cells (a) and condensates (b) for B. uniformis cells grown in mGAM + 250 µM (116 mg/l) PFNA. 119 images analysed in total including all strains and exposures. c. Mean pixel intensity of bacterial cells show significant differences between DMSO and PFNA treated cells for B. uniformis, E. coli BW25113 and O. splanchnicus; two-sided t-test; p-values FDR corrected; Boxplot: centre = 50th percentile, bounds of box = 25th and 75th percentile, lower/upper whisker = lower/upper hinge −/+ 1.5 * inter quartile range; Number of cells analysed: B. uniformis: DMSO = 141, PFNA_250µM = 178; E. coli (BW25113): DMSO = 115, PFNA_250µM = 61; E. coli (BW25113) ∆tolC: DMSO = 56, PFNA_250µM = 14; O. splanchnicus: DMSO = 76, PFNA_250µM = 57 (Supplementary Table 37). d. Condensate count per bacterial cell show significant increase in condensates for B. uniformis and O. splanchnicus, supporting morphological changes through PFNA exposure; two-sided t-test; p-values FDR corrected; Number of cells analysed: B. uniformis: DMSO = 141, PFNA_250µM = 178; E. coli (BW25113): DMSO = 115, PFNA_250µM = 61; E. coli (BW25113) ∆tolC: DMSO = 56, PFNA_250µM = 14; O. splanchnicus: DMSO = 76, PFNA_250µM = 57 (Supplementary Table 37). Mean intensity of bacterial cells show significant differences between DMSO and PFNA or PFDA treated cells; two-sided t-test; p-values FDR corrected; Boxplot: centre = 50th percentile, bounds of box = 25th and 75th percentile, lower/upper whisker = lower/upper hinge −/+ 1.5 * inter quartile range; Number of cells analysed: DMSO = 141, PFNA_5µM = 34, PFNA_50µM = 29, PFNA_125µM = 26, PFNA_250µM = 178, PFDA_125µM = 38 (Supplementary Table 37. f. B. uniformis data from d including data for 5, 50, 125 µM PFNA and 125 µM PFDA. Condensate count per bacterial cell show significant increase in condensates for B. uniformis exposed to 50, 125 and 250 µM PFNA, supporting morphological changes through PFNA exposure; two-sided t-test; p-values FDR corrected; Number of cells analysed: DMSO = 141, PFNA_5µM = 34, PFNA_50µM = 29, PFNA_125µM = 26, PFNA_250µM = 178, PFDA_125µM = 38 (Supplementary Table 37). During cryogenic FIB-SIMS imaging, the vitrified sample is scanned by a Gallium focused ion beam (FIB) and time-of-flight secondary electron mass spectrometry (ToF-SIMS) is performed on the secondary ions created by the interaction of ion beam and sample at each scanning position. For both cases, a spatial image of a cell resulting from the secondary electrons created during FIB imaging is shown on the left (a,d). Please note that because the specimens were frozen on a holey cryo-EM grid, round holes in the film are additionally observed, in addition to the elongated cells. On the right (b,c,e,f), ToF-SIMS images of the same cell are shown for mass-to-charge ratios 19 and 63, corresponding to F− and PO2−, respectively. The colour bars indicate ion count, that is, regions shown in red are those where the most ions of the particular species were detected. F− signal indicates the presence of PFNA, PO2− was chosen as a comparison as it occurs in most bacterial cells and hence is a reliable marker for cells in negative SIMS imaging mode. In E. coli ∆tolC cells exposed to PFNA, a strong F− signal is observed within the cell (b), which is not the case for the DMSO treated cells (e), in which a faint background signal can only be observed when adjusting the colour scale by several orders of magnitude. Improved growth of adapted P. merdae in presence of 125 μM PFDA at days 5, 10, 15 and 20 compared to day 0 (day 20: 1.3-fold change, p-value = 0.001). n = 4 independent populations per compound (Supplementary Tables 38,39). c. Improved growth of adapted B. uniformis population in presence of 250 μM PFNA and 125 μM PFDA after 5, 10, 15 and 20 days compared to day 0 (day20 PFNA 7-fold change, p-value = 0.0004; day 20 PFDA 46-fold change, p-value = 0.00003). n = 4 independent populations per compound (Supplementary Tables 38,39). d. Improved growth of adapted E. coli BW25113 ΔTolC population in presence of 500 µM PFHpA, 500 µM PFOA, 250 μM PFNA and 125 μM PFDA at day 20 compared to day 0 (PFHpA 1.3-fold change, p-value = 0.002; PFOA 1.7-fold change, p-value = 0.0006; PFNA 2.3-fold change, p-value = 0.0006; PFDA 1.6-fold change, p-value = 0.00005). n = 4 independent populations per compound (Supplementary Tables 38,39). e. Adapted populations retain PFDA and PFNA bioaccumulation capability. Mean of n = 4 independent populations per compound (Supplementary Table 40). Results showing the proteins that are differentially abundant between (a) E. coli BW25113, (b) E. coli BW25113 ΔTolC, and (c) O. splanchnicus treated with 20 µM PFNA in comparison to DMSO. The green and the blue dots mark proteins with a log2 abundance ratio >1 or <−1 (that is, two-fold increase or decrease) and a multiple-testing corrected p-value of less than 0.05. n = 6 biological replicates (Supplementary Table 24). a. PCA analysis shows clear distinction between bacterial strains, with E. coli wild-type and TolC mutant overlapping. b. PCA analysis of pellet and supernatant samples shows clear distinction of PFNA and DMSO treated cells only for B. uniformis pellet samples (B. uniformis pellet PCA is a replicate of Fig.4d, added for completion). c. Metabolomics analysis show change in intra- and extracellular metabolites. Fold change of analysed metabolites in supernatant and pellet samples of 20 µM PFNA exposed cells compared to control (DMSO) samples. P-values calculated using two-sided t-test and Benjamini-Hochberg correction for multiple testing. n = 6 biological replicates (Data and p-values provided in Supplementary Table 41). Gut microbiota composition of mice colonised with 20 human gut bacterial strains (Com20) and exposed to 10 mg/kg body weight by oral gavage on day 0. 17 out of 20 gut bacteria colonised the mice with 7 strains making up 90% of bacterial composition (P. vulgatus, R. gnavus, B. thetaiotaomicron, B. uniformis, A. rectalis, Veillonella parvula, Dorea formicigenerans). In the colon and fecal samples 70% of bacteria were classified as high-PFNA accumulating strains, while in the small intestine only 20% of bacteria were classified as high-PFNA accumulating (Supplementary Table 43). c. Gut microbiota composition of mice colonised with a community of five high-PFNA accumulating strains (HC) and five low-PFNA accumulating strains (LC) exposed to 10 mg/kg body weight by oral gavage on day 0. All gut bacteria colonised the mice, but A. muciniphila was present in both HC and LC colonised mice (Supplementary Table 44). d. LC (low-PFNA accumulating community) colonised mice showed higher DNA yields after DNA extraction compared to HC (high-PFNA accumulating community) colonised mice (1.96-fold change, p-value = 0.002), indicating higher bacterial content in LC colonised mice; two-sided t-test; Boxplot: centre = 50th percentile, bounds of box = 25th and 75th percentile, lower/upper whisker = lower/upper hinge −/+ 1.5 * inter quartile range; n = 54 samples per group (Supplementary Table 28). e. HC colonised mice showed higher relative abundance of Acetobacter fabarum spike-in compared to LC colonised mice (2-fold change, p-value = 0.0009), indicating higher bacterial content in LC colonised mice. Two-sided t-test; Boxplot: centre = 50th percentile, bounds of box = 25th and 75th percentile, lower/upper whisker = lower/upper hinge −/+ 1.5 * inter quartile range; Subset of 76 samples (Supplementary Table 45). f. Mice colonised with a community of 20 human gut bacterial strains (Com20) show higher PFNA excretion after 0.1 mg/kg body weight PFNA exposure compared to germfree (GF) controls; Boxplot: centre = 50th percentile, bounds of box = 25th and 75th percentile, lower/upper whisker = lower/upper hinge −/+ 1.5 * inter quartile range; two-sided t-test; p-values are FDR corrected; n = 6 mice per group; y-axes are on log10 scale (Supplementary Table 46). Supplementary Tables 1–48 containing source data for Figs. Additional images from cryogenic FIB-SIMS imaging. A tomogram for B. uniformis cells exposed to 250 µM PFNA. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Lindell, A.E., Grießhammer, A., Michaelis, L. et al. Human gut bacteria bioaccumulate per- and polyfluoroalkyl substances. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250630073445.htm'>Scientists just mapped how the body rejects pig organs—and how to stop it</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 05:14:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A pioneering study has provided unprecedented insights into the immune response following pig-to-human kidney xenotransplantation.1 Using cutting-edge spatial molecular imaging, researchers mapped how human immune cells interact with pig kidney tissue in transplanted organs, revealing critical early markers of rejection and potential intervention strategies. The study, led by Dr. Valentin Goutaudier and a collaborative international research team (Paris Institute for Transplantation and Organ Regeneration & NYU Langone Transplant Institute), highlights key molecular mechanisms that could shape the future of xenotransplantation. Researchers observed early molecular signs of antibody-mediated rejection as soon as Day 10 and peaking at Day 33, reinforcing previous findings that rejection begins rapidly but progresses over time.2 By tracking these immune responses for up to 61 days, the team identified a crucial window for targeted therapeutic intervention. Notably, macrophages and myeloid cells were the most prevalent immune cell types across all time points, further confirming their role as key mediators in xenograft rejection. Combined with novel spatial insights into how immune cells interact with pig kidney tissue, this marks a major breakthrough -- paving the way for more refined anti-rejection strategies. These advances come at a pivotal time as the first US-based clinical trials of pig kidney transplantation into living human recipients begin in 2025. With xenotransplantation poised to address the global organ shortage crisis, these findings bring researchers one step closer to making genetically modified pig kidneys a viable long-term solution. "Understanding the specific immune interactions at a molecular level allows us to develop targeted interventions that can prevent rejection before it escalates," explained Dr. Goutaudier. As scientific progress accelerates, researchers remain cautiously optimistic that genetically modified pig kidneys could become a routine transplant option within the next decade. However, regulatory approvals will require consistent demonstration of safety and efficacy in diverse patient populations. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250630073443.htm'>This virus infects millions—and we just discovered its secret weapon</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-01 03:42:17
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research from the University of Pittsburgh School of Medicine and La Jolla Institute for Immunology, published today (June 30) in Nature Microbiology, reveals an opportunity for developing a therapy against cytomegalovirus (CMV), the leading infectious cause of birth defects in the United States. In addition to using molecular machinery that is shared by all herpes viruses, CMV employs another molecular "key" that allows the virus to sneak through a side door and evade the body's natural immune defenses. The finding might explain why efforts to develop prophylactic treatments against CMV have, so far, been unsuccessful. This research also highlights a new potential avenue for the development of future antiviral drugs and suggests that other viruses of the herpes family, such as Epstein-Barr and chickenpox, could use similar molecular structures to spread from one infected cell to the next while avoiding immune detection. "If we don't know what weapons the enemy is using, it is hard to protect against it," said senior author Jeremy Kamil, Ph.D., associate professor of microbiology and molecular genetics at Pitt. "We found a missing puzzle piece that represents one possible reason why immunization efforts against CMV have been unsuccessful." Of the babies infected, one in five will have birth defects, such as hearing loss, or go on to have long-term health challenges. But a CMV infection during pregnancy presents significant health risks to the unborn child and could be deadly for people who are immunosuppressed, including organ transplant recipients. The newly discovered GATE could become a potential vaccine target for CMV and other herpes viruses. "Previous attempts to generate a CMV vaccine have failed, but that was before we identified the GATE complex. We hope that new strategies targeting GATE will improve our chances to combat CMV infection, and also perhaps cleanse our bodies of this lifelong infection," said Chris Benedict, Ph.D., associate professor at La Jolla Institute for Immunology and co-senior author of the study with Kamil and LJI professor, president & CEO Erica Ollmann Saphire, Ph.D., MBA. "If we can develop antiviral drugs or vaccines that inhibit CMV entry, this will allow us to combat the many diseases this virus causes in developing babies and immune-compromised people." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            