
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-21</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250521161106.htm'>Scientists discover class of crystals with properties that may prove revolutionary</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 20:27:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Rutgers University-New Brunswick researchers have discovered a new class of materials -- called intercrystals -- with unique electronic properties that could power future technologies. Intercrystals exhibit newly discovered forms of electronic properties that could pave the way for advancements in more efficient electronic components, quantum computing and environmentally friendly materials, the scientists said. A subtle misalignment between the layers that formed moiré patterns -- patterns similar to those seen when two fine mesh screens are overlaid -- significantly altered how electrons moved through the material, they found. By understanding and controlling the unique properties of electrons in intercrystals, scientists can use them to develop technologies such as more efficient transistors and sensors that previously required a more complex mix of materials and processing, the researchers said. "You can imagine designing an entire electronic circuit where every function -- switching, sensing, signal propagation -- is controlled by tuning geometry at the atomic level," said Jedediah Pixley, an associate professor of physics and a co-author of the study. "Intercrystals could be the building blocks of such future technologies. "The discovery hinges on a rising technique in modern physics called "twistronics," where layers of materials are contorted at specific angles to create moiré patterns. The foundational idea was first demonstrated by Andrei and her team in 2009, when they showed that moiré patterns in twisted graphene dramatically reshape its electronic structure. In regular crystals, which possess a repeating pattern of atoms forming a perfectly arranged grid, the way electrons move is well understood and predictable. If a crystal is rotated or shifted by certain angles or distances, it looks the same because of an intrinsic characteristic known as symmetry. This variability can lead to new and unusual behaviors, such as superconductivity and magnetism, which aren't typically found in regular crystals. Superconducting materials offer the promise of continuously flowing electrical current because they conduct electricity with zero resistance. Intercrystals could be a part of the new circuitry for low loss electronics and atomic sensors that could play a part in the making of quantum computers and power new forms of consumer technologies, the scientists said. "Because these structures can be made out of abundant, non-toxic elements such as carbon, boron and nitrogen, rather than rare earth elements, they also offer a more sustainable and scalable pathway for future technologies," Andrei said. They also are different from quasicrystals, a special type of crystal discovered in 1982 with an ordered structure but without the repeating pattern found in regular crystals. Research team members named their discovery "intercrystals" because they are a mix between crystals and quasicrystals: they have non-repeating patterns like quasicrystals but share symmetries in common with regular crystals. "With intercrystals, we go a step further, showing that materials can be engineered to access new phases of matter by exploiting geometric frustration at the smallest scale." "We are excited to see where this discovery will lead us and how it will impact technology and science in the years to come." Other Rutgers researchers who contributed to the study included research associates Xinyuan Lai, Guohong Li and Angela Coe of the Department of Physics and Astronomy. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a64784652/copper-filo-del-sol-argentina/'>A Copper Treasure Lies Hidden Beneath the Andes. Getting It Out May Not Be Worth the Danger.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 19:02:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The Filo del Sol deposit high in the Argentine Andes likely contains five times more metal than previously believed, but many environmental concerns remain. However, a new initial mineral resource estimate completed earlier this month suggests that the companies in charge of mining this area—the U.S.-based Lundin Mining and BHP—may have stumbled upon five times more metal than they bargained for. This new update, gathered from data collected from 400 additional exploration holes, came from the discovery that deeper mineralization of copper far exceeded the estimates that were closer to the surface. According to AFP, Filo del Sol could prove to be richer still, as experts dig deeper and explore the resource's northern and southern boundaries. “Filo del Sol has been one of the most significant greenfield discoveries in the last 30 years and an amazing journey for all those that have been involved,” Jack Lundin, CEO of Lundin Mining, said in a press statement. This mine is particularly lucrative, as many of the metals found there will be vital to both the green energy revolution and other industries (such as aerospace and telecommunications) that need precious metals like gold. Of course, discovering the existence of these resources is one thing, and extracting them is something else entirely—especially due to this open pit mine's particular location. Located at roughly 5,000 meters (16,400 feet) above sea level, the mine and its high elevation (not to mention the overall punishing environment) can take a toll on workers and even induce altitude sickness, according to AFP. As with mining any hard-to-reach location, the logistics of moving equipment up to those altitudes will also be difficult. Mining operations on both sides of the border have tried to clean up their act, and mines in other areas of Chile's Atacama Desert are working toward transitioning to mining powered by renewable energy, according to Mining Technology. In December of 2024, Argentina's largest private electricity generator Central Puerto began a feasibility study to figure out the best way to build transmission lines capable of ferrying renewable energy to mining sites in the northwest. Some 70 percent of the country's drinking water comes from glaciers. It also doesn't help that mining itself requires lots of water. For example, one of the country's biggest mining sites—La Alumbrera—uses 25 billion liters of water every year, which is equal to 34 percent of the water consumed by the region's nearly half a million inhabitants in a year, according to FARN. It's no secret that humanity needs minerals hiding within the Earth if it has any hope of transitioning away from fossil fuels. Chinese Tomb Mural from 8th Century Has Blonde Man</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64841409/silver-amulet-christianity-history/'>Archaeologists Found a Skeleton Wearing an Amulet That May Change the History of Christianity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 17:34:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A 1.37-inch inscription could upend our understanding of the religion's spread. An 1,800-year-old silver amulet discovered buried in a Frankfurt, Germany grave, still next to the chin of the man who wore it, has 18 lines of text written in Latin on just 1.37 inches of silver foil. Every other link to reliable evidence of Christian life in the northern Alpine area of the Roman Empire is at least 50 years younger, all coming from the fourth century A.D. “This extraordinary find affects many areas of research and will keep science busy for a long time,” Ina Hartwig, Frankfurt's head of culture and science, said in a translated statement. Such a significant find here in Frankfurt is truly something extraordinary.” The amulet was found in what was once the Roman city of Nida at an archaeological site outside of Frankfurt in 2018. The wafer-thin silver foil was too brittle to roll out. In May 2024, a breakthrough came when using a state-of-the-art computer tomograph at the Leibniz Center for Archaeology in Mainz. Markus Scholz from Frankfurt's Goethe University was able to piece together the 18 lines. “Sometimes it took weeks, even months, until I had the next idea,” he said in a statement. “I called in experts from the history of theology, among others, and we approached the text together bit by bit and ultimately deciphered it.” Some edges were lost due to damage and some words remain open to discussion. The original inscription is entirely in Latin, unusual for a time that featured amulets written in Greek or Hebrew. During the third century A.D., association with Christianity was still dangerous, and identifying as Christian came with great personal risk, especially as Roman emperor Nero punished Christians with death or even a date in the Colosseum. That was no matter for this man in Frankfurt who took his allegiance to Jesus Christ to his grave. “As a result, the history of Christianity in Frankfurt and far beyond will have to be turned back by around 50 to 100 years. We can be proud of this, especially now, so close to Christmas.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Found an Ancient Trove of Unfinished Art Explorers Found Maya Sacrifices in a ‘Blood Cave'</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09069-w'>Stepwise ATP translocation into the endoplasmic reticulum by human SLC35B1</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 15:37:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The endoplasmic reticulum (ER) cannot synthesize ATP but must import cytoplasmic ATP to energize protein folding, quality control and trafficking2,3. It was recently proposed that a member of the nucleotide sugar transporter family, termed SLC35B1 (also known as AXER), is not a nucleotide sugar transporter but a long-sought-after ER importer of ATP4. Here we report that human SLC35B1 does not bind nucleotide sugars but indeed executes strict ATP/ADP exchange with uptake kinetics consistent with the import of ATP into crude ER microsomes. A CRISPR–Cas9 cell-line knockout demonstrated that SLC35B1 clusters with the most essential SLC transporters for cell growth, consistent with its proposed physiological function. We have further determined seven cryogenic electron microscopy structures of human SLC35B1 in complex with an Fv fragment and either bound to an ATP analogue or ADP in all major conformations of the transport cycle. We observed that nucleotides were vertically repositioned up to approximately 6.5 Å during translocation while retaining key interactions with a flexible substrate-binding site. We conclude that SLC35B1 operates by a stepwise ATP translocation mechanism, which is a previously undescribed model for substrate translocation by an SLC transporter. ATP is the most critical and universal fuel currency of cells across all kingdoms of life2. In eukaryotes, ATP is regenerated in the mitochondria by rotary F0F1-ATP synthase and exported to the cytoplasm by mitochondrial ADP/ATP carriers belonging to the SLC25 family1. Owing to our high energy requirements, mitochondrial carriers move our own body weight in ATP every day1. One of the most demanding organelles for ATP consumption is the endoplasmic reticulum (ER)3, which occupies a large portion of eukaryotic cell and has no endogenous production. Therefore, it must import ATP from the cytoplasm. ER luminal ATP is required for the calreticulin-dependent trafficking of glycosylated major histocompatibility complex class I molecules from the peptide-loading complex8, an important process in the adaptive immune system. Over the years, different proteins have been proposed and disregarded as routes for ATP entry into the ER3,9,10,11, with a recent proposal for SLC35B1 considered the most convincing4; SLC35B1 has also been referred to as AXER for ATP/ADP exchanger in the ER membrane4 (Fig. 1a). SLC35B1 belongs to the nucleotide sugar transporter (NST) family12, which mainly transports cytoplasmic nucleoside phosphate sugars into the ER and Golgi for the counter-transport of luminal mononucleotides, wherein the nucleotide sugars are used for glycosylation (for example, uridine diphosphate (UDP)-glucose is exchanged for uridine monophosphate (UMP)13) (Extended Data Fig. 1a). SLC35B1 localizes to the ER4, and it has been shown that short interfering RNA (siRNA)-mediated depletion of SLC35B1 in HeLa cells reduces ER ATP levels4. BiP-dependent protein import was also lowered in SLC35B1 knockdown, and ATP/ADP exchange was demonstrated in Escherichia coli upon overexpression of human SLC35B1 (ref. 4). A follow-up study confirmed that SLC35B1 knockdown in Chinese hamster ovary cells substantially decreased ER ATP levels14. Nevertheless, the ATP/ADP exchange kinetics of human SLC35B1 in E. coli are poor and lack validation with purified components4. SLC35B1 and yeast orthologues have also been proposed as UDP-galactose/glucuronic acid transporters15,16,17, and NST members adopt the drug–metabolite transporter (DMT)-fold18,19, which is a different transporter-fold than that used by the mitochondrial ADP/ATP carriers1. Here we have validated the physiological function, structure and transport mechanism of human SLC35B1. a, Schematic highlighting the proposed uptake of cytoplasmic ATP into the ER in exchange for luminal ADP by SLC35B1 and ATP exported from the mitochondria in exchange for cytoplasmic ADP by SLC25A4. b, Thermal stabilization of the purified SLC35B1–GFP in the presence of 1 mM ATP (green), ADP (black), AMP (yellow), UDP-galactose (red) or buffer (black; empty circle). c, Left, comparison of ATP and UDP-galactose interactions by STD NMR (light brown) to SLC35B1 proteoliposomes and off-resonance 1H NMR spectra (black). Right, total STD amplification factor for either ATP, AMP–PNP, ADP, AMP or UDP-galactose interaction with SLC35B1. d, One-minute uptake of [3H]ATP by SLC35B1 proteoliposomes (black bars) preloaded with either a buffer or 1 mM cold ADP and compared to empty protein-free liposomes (white bars). e, Time-course uptake of [3H]ADP by SLC35B1 proteoliposomes (black circles) or empty liposomes (non-filled circles) both preloaded with 1 mM ATP. Inset: enlarged image of the uptake from 0 to 4 min to highlight the initial near-linear rates. f, Normalized SLC35B1-mediated uptake of [3H]ADP in competition with either a buffer (white bar) or cold ATP, ADP, AMP or UDP-galactose (black bars) into proteoliposomes preloaded with cold 1 mM ATP. of six independent experiments from two separate reconstitutions. The Km and kcat parameters from these fits are shown. Graphic in a was created using BioRender (https://biorender.com). Thermal shift assays have shown to be a powerful approach for detecting nucleotide binding to the mitochondrial ADP/ATP carrier SLC25A4 (refs. 20,21) and were therefore used for human SLC35B1. We observed that the addition of either 1 mM ATP or ADP to purified SLC35B1 increased its resistance to heat denaturation, with an average melting temperature (ΔTm) increase of 5.2 °C and 5.3 °C, respectively (Fig. The degree of nucleotide thermostabilization was similar to that measured for the mitochondrial carrier SLC25A4 (ref. 20). By contrast, adenosine monophosphate (AMP) and UDP-galactose only showed a minimal ΔTm increase for SLC35B1 by 1.1 °C and 0.3 °C, respectively (Fig. We then reconstituted purified SLC35B1 into liposomes and assessed nucleotide binding using saturation transfer difference (STD) nuclear magnetic resonance (NMR) spectroscopy22. Briefly, ligands will only produce STD NMR signals if their non-exchangeable protons interact specifically with SLC35B1, receiving magnetization transfer during on-pulses. Consistent with the thermal shift analysis, the addition of ATP and ADP resulted in strong STD NMR signals, whereas the addition of UDP-galactose and AMP resulted in only weak signals (Fig. To assess transport function, SLC35B1 proteoliposomes were preloaded with 1 mM ADP, and the uptake of externally added [3H]ATP measured after 1 min. Robust uptake of [3H]ATP was apparent, with little uptake in either unloaded or protein-free liposomes (Fig. 1d). Comparable results were obtained when proteoliposomes were preloaded with 1 mM ATP, followed by uptake of [3H]ADP (Fig. The addition of external cold ATP or ADP showed strong competition for [3H]ADP uptake, whereas AMP or UDP-galactose addition showed only weak competition, which is consistent with the thermal shift assays and STD NMR analysis (Fig. 1f). The half-maximal inhibitory concentration (IC50) values were determined to be 10.6 μM for ATP, 1.6 μM for ADP and 3.7 mM for AMP (Extended Data Fig. 1f). Under symmetric pH conditions and with no membrane potential (ψ) applied, a Michaelis–Menten constant (Km) of 3.4 μM for ATP was measured, similar to the Km of 4 μM reported for ATP import into crude ER microsomes23 (Fig. 1g). The turnover (kcat) of 12 ATP min−1 was calculated and was found to be higher than the rates for the major ER folding chaperone BiP, with a kcat of 0.013 min−1 (ref. To support the physiological role of human SLC35B1, cell growth in an SLC35B1 knockout was compared against all other SLC transporters in a CRISPR–Cas9 knockout screen in HCT 116 cells (Methods). Together with SLC transporters known to be essential for cell survival, the SLC35B1 protein emerged as one of the top five most essential SLCs, indicating a critical housekeeping role (Extended Data Fig. 2a). Taken together, our biochemical, kinetic and genetic analyses confirmed that SLC35B1 is a bona fide ATP/ADP exchanger for the ER. Detergent-purified human SLC35B1, with a molecular mass of approximately 35 kDa, was considered too small for structural determination by cryo-electron microscopy (cryo-EM) (Extended Data Fig. 1c). Therefore, a monoclonal antibody was raised against human SLC35B1, and a recombinant Fv-maltose-binding protein (MBP) fiducial marker was constructed for structural studies (Methods and Extended Data Fig. 2b). Purified SLC35B1 formed a homogeneous complex with the Fv–MBP fusion protein (Methods, Extended Data Fig. Although the SLC35B1–Fv–MBP fusion protein complex showed a stronger STD NMR signal for ATP than SLC35B1 alone (Fig. 2d), the ADP/ATP kinetics and IC50 for ADP with and without the Fv–MBP fusion were similar; therefore, the antibody fragment had not overly perturbed SLC35B1 transport activity (Fig. a, Left, normalized STD effect measured for ATP interaction with either wild-type (WT) SLC35B1 or SLC35B1–Fv–MBP in proteoliposomes. Right, 1-min uptake of [3H]ADP into SLC35B1, SLC35B1–Fv–MBP or empty liposomes preloaded with 1 mM ATP. b, Cartoon representation of SLC35B1 harbouring the expected DMT-fold. TM5 and TM10 (grey) are positioned peripheral to the V-type helices, which in some DMT-fold members mediate homodimerization with TM5 and TM10 of the neighbouring protomer. Access to the ER lumen is closed (obstructing helices boxed) but open to the cytoplasm. c, Top, cryo-EM maps of SLC35B1–Fv complex. d, Cartoon representation of the ER lumen cavity-closing contacts (sticks; labelled). For comparison, the thermal shift of SLC35B1–GFP with 1 mM AMP is shown (red bar). f, Total STD amplification factor after ATP addition to wild-type SLC35B1 proteoliposomes and Q113F mutant. For comparison, the STD signal of SLC35B1–GFP with 1 mM AMP is shown (red bar). The SLC35B1–Fv–MBP fusion complex was optimized for cryo-EM with or without the addition of the non-hydrolysable ATP analogue adenylyl imidodiphosphate (AMP–PNP), which was first confirmed to produce equivalent STD NMR signals and transport activity as ATP (Fig. After processing and refinement, the SLC35B1 structures showed good fitting into the cryo-EM maps reconstructed to a resolution of approximately 3.4 Å (Extended Data Figs. The SLC35B1 structure had the expected DMT-fold, which comprised two structurally similar four-transmembrane (4-TM) helix bundles made up from two overlocking V-shaped transmembrane helical pairs of TM1–TM2 with TM8–TM9 and TM3–TM4 with TM6–TM7 (Fig. 2b). The peripheral and shorter TM5 and TM10 are connected to the two 4-TM helix bundles by flexible loops that in other DMT-fold members can mediate the formation of a stable homodimer25,26,27,28, but this was not observed in SLC35B1. The middle of TM4 was found to be broken between the conserved K117 and P212 residues and was therefore designated TM4a–TM4b (Fig. A helix–break–helix in TM4 has not been observed in other NST structures for GDP-mannose and cytidine monophosphate (CMP)–sialic acid19,25,29, but it has been observed in a distantly related bacterial DMT-fold member acting on amino acids30. The SLC35B1 structure displayed a large cavity open towards the cytoplasm (inward-facing), which is also a conformation yet to be captured experimentally for the NST family19,25,29 (Fig. Bundle closure on the luminal side is formed by highly conserved polar residues located between the ends of TM1 and TM9 together with TM3–TM4a (Fig. SLC35B1 structures with and without AMP–PNP addition are mostly equivalent, apart from the extra cryo-EM map density for the nucleotide (Extended Data Fig. 4a–e). Unexpectedly, the ring-shaped map density implied a nucleotide conformation where the terminal γ-phosphate had arched back towards the adenine moiety (Extended Data Fig. 4e). However, this is an unusual configuration for the ATP analogue, and the symmetrical map density made it difficult to confidently assign its correct orientation. After screening various mutations, a Q113F variant was identified that increased the melting temperature (Tm) from 33.8 °C to 37.2 °C, and the ΔTm (ATP) shifted from 5.2 °C to 6.6 °C (Fig. The Q113F variant also displayed a stronger STD NMR signal for ATP than wild-type SLC35B1 (Fig. Although transport activity was reduced to approximately 25% of wild-type levels, the ATP IC50 at 2.7 μM was only somewhat lower (Extended Data Figs. Consistent with the increased stability, the cryo-EM structure of Fv–MBP(Q113F) was determined to have an improved resolution of approximately 3.0 Å (Supplementary Fig. a, Electrostatic surface representation of the cytoplasmic-facing SLC35B1(Q113F) structure in complex with AMP–PNP (sticks) highlighting the hydrophobic patch and positively charged surfaces that interact with nucleotide phosphates (red/orange sticks) and adenosine (cyan). b, Cartoon representation of SLC35B1, highlighting helices and residues (sticks) interacting with AMP–PNP. c, Electrostatic surface representation of the cytoplasmic-facing SLC35B1 structure in complex with ADP (sticks) highlighting the hydrophobic patch and positively charged surfaces that interact with nucleotide phosphates (red/orange) and adenosine (grey). d, Comparison of SLC35B1 helices harbouring residues interacting with ADP (grey) and AMP–PNP (teal, orange and cyan) in the cytoplasmic-facing states. The dashed box shows an enlarged view, with labelled residues interacting with ADP (dashed lines). e, Mutational analysis of single time point [3H]ADP/ATP uptake into SLC35B1 proteoliposomes. Data were normalized to the absolute signal of wild-type SLC35B1. of six independent experiments carried out from two separate reconstitutions. f, Comparison of the cytoplasmic-facing wild-type protein (bundles in orange and teal) and luminal-facing E33A variant (pink). Left, view from the cytosolic side with TM4b, TM6, TM8 and TM9 moving particularly inwards for gate closure. Right, view from the ER luminal side with TM1, TM3, TM4a and TM9 moving particularly outwards for gate opening. Matching the positively charged surface of the cytoplasmic-facing cavity, the negatively charged α- and β-phosphates in AMP–PNP were found to be coordinated by R276 and K277 in TM9 and by K120 in TM4b (Fig. 3a,b). The K120 residue also interacted with γ-phosphate, which had circled back to interact with the adenosine nitrogen. The T273 residue in TM9 also formed a polar interaction with the α-phosphate, and the ribose ring oxygen was stabilized by hydrogen bonding to Q254 in TM8 (Fig. 3b). Comparing the apo wild-type and AMP–PNP-bound Q113F structures, we found that K120 and K277 residues had repositioned to interact with the nucleotide, breaking polar interactions with D183 and S118, respectively (Extended Data Fig. 5h). Unexpectedly, there were no polar interactions coordinating the adenine moiety (Fig. 3a,b). Instead, I257 and V261 in TM8 formed a hydrophobic patch for adenine, together with the C269 residue in TM9. At this position, K117 in TM4a was too distant to directly interact with AMP–PNP (Fig. 3b). Although the modelled position for AMP–PNP was unexpected, adenine is fairly hydrophobic31 and, therefore, the amphipathic substrate was nevertheless matched by complementary charged and non-charged protein surfaces (Fig. 3a). However, given the atypical binding mode of AMP–PNP, we sought to determine whether ADP would also interact in a similar manner. Although the physiological substrate in the cytoplasmic-facing conformation is ATP, we observed that ADP/ATP versus ADP/ADP exchange activities were comparable and, as such, mechanistically, ADP and ATP are robustly transported in either direction (Extended Data Fig. 6a,b). Repeating the cryo-EM workflow for the SLC35B1–Fv–MBP fusion, we noticed that the particles were more homogeneous with ADP, and the map quality could be further improved to a resolution of 2.85 Å (Supplementary Fig. The wild-type structure with ADP was very similar to previous structures, and we were further able to model a peripheral lipid on the outside of the TM3 and TM6 helices (Extended Data Fig. 6c,d). Strong map density showed that ADP was also bound to the same location as AMP–PNP (Fig. ADP also adopted a toroidal shape conformation, and the adenine moiety was likewise positioned next to the hydrophobic patch formed by the I257 and V261 residues (Fig. 3d). The K277 residue in TM9 and the side chain of K120 in TM4b had further repositioned to maintain an interaction with both phosphates in ADP but R276 was no longer forming a direct interaction (Fig. Confirming their requirement for transport, all variants were inactive, including K117A in TM4a and Y25A in TM1, which interacts with a Q254 residue (TM8) that itself is interacting with the ribose moiety (Fig. 3b). To assess the role of the hydrophobic interactions, I257 and V261 residues were substituted with glutamate and threonine, respectively (Supplementary Fig. 3c). The transport activities of the I257E and V261T variants were also severely reduced, confirming their importance (Fig. 3e). Finally, C269, located at the beginning of TM9, was substituted with either alanine or serine residues. Yet, cysteine variants retained robust transport activity, and therefore the role of this residue is unclear (Fig. The lack of specific polar interactions with the adenine moiety suggests that other trinucleotide phosphates should also be able to compete for binding (Fig. 3b). Indeed, both cytidine triphosphate (CTP) and uridine triphosphate (UTP) nucleotides showed strong competition for [3H]ADP uptake, with IC50 values only approximately twofold higher than that of ATP at 21 and 29 μM, respectively (Extended Data Figs. SLC35B1 showed a stronger inhibition for ADP than ATP, and consistently, UDP and CDP were more effective at competition for [3H]ADP uptake than UTP and CTP (Extended Data Fig. 6f). Furthermore, preloading liposomes with either CTP or UTP confirmed that these nucleotides could also catalyse [3H]ADP import, although at approximately 30% of the levels measured for ATP preloading (Extended Data Fig. 6g). By contrast, the addition of GTP showed only weak competition with an IC50 of 3.5 mM, as well as poor thermostabilization and low STD NMR signals (Extended Data Figs. Similar to many transporters, DMT-fold members operate by means of a rocker-switch alternating-access mechanism27,28,30,32, in which structurally similar bundles move around a centrally located substrate. However, in SLC35B1, the adenine moiety binds at a position close to the cytoplasmic surface, which is incompatible with such a mode of transport. 29), we found that the position of adenine would physically clash with the predicted inward movement of the TM8–TM9 gating helices (Extended Data Fig. 6h). To determine how adenine nucleotides would be translocated, we examined whether the E33A variant might have shifted the SLC35B1 population to an ER lumen-facing conformation (Fig. 2d). The E33A variant was selected because it retained robust ATP binding and had a similar IC50 value to that of wild-type SLC35B1 at 11.7 μM (Fig. Because ADP was the physiological substrate from the ER luminal side, we collected cryo-EM datasets with either ADP or AMP–PNP and after refinement, cryo-EM maps could be reconstructed to a resolution of 3.2 and 3.1 Å, respectively (Methods, Supplementary Figs. Comparing cytoplasmic- and luminal-facing structures, we found that TM8–TM9 helices had moved inwards to form cavity-closing contacts with TM6 and TM7 on the cytoplasmic side (Fig. 3f). On the ER luminal side, the TM1, TM3, TM4a, TM6 and TM9 helices had most predominantly moved outwards (Fig. 3f). Notably, the cavity-closing contacts formed on the cytoplasmic side are comparatively weaker than those on the ER luminal side and are primarily made up of hydrophobic and main-chain interactions (Fig. 4a). Between cytoplasmic-facing and luminal-facing states, the adenine nucleobase had been vertically translocated approximately 6.5 Å by the closure of TM8–TM9 gating helices (Fig. The phosphate moieties extend vertically in both ADP and AMP–PNP nucleotides and maintained their interactions with the positively charged residues (Fig. a, Cartoon representation of the cytoplasmic cavity-closing contacts (sticks; labelled) for the luminal-facing SLC35B1(E33A) structure with ADP. b, Left, electrostatic surface representations of the luminal-facing SLC35B1(E33A) in complex with ADP (grey sticks). Right, as in the left panel, but with AMP–PNP (cyan sticks). c, Left, comparison between the cytoplasmic facing AMP-PNP bound (cyan; sticks) structure (orange and teal) with the ER luminal-facing AMP–PNP-bound (grey) structure (pink). Upon TM8–TM9 gate closure, the nucleotide was vertically displaced by approximately 6.5 Å (dark-blue arrow). Middle: cartoon highlighting helices and interacting residues (sticks; dashed lines) coordinating AMP–PNP in the luminal-facing SLC35B1(E33A). Right, as in the middle panel for ADP. d, Thermal stabilization of the SLC35B1 variants in the presence of 1 mM ATP, or 1 mM GTP (red bar). e, Cartoon highlighting the gating helix TM9 pivoting around the central R276 during nucleotide translocation by comparing the cytoplasmic-facing AMP–PNP-bound (cyan; sticks) structure (orange and teal) with the ER luminal-facing AMP–PNP-bound (grey; sticks) structure (pink). f, Structural superposition of the two 5-TM structural inverted repeats in SLC35B1 (teal, orange, grey) with functionally important residues highlighted (sticks). Specifically, although R276 and K277 in TM9 maintained their interaction with the α-phosphate of AMP–PNP in both cytoplasmic-facing and luminal-facing conformations, K120 had shifted from interacting with the β- and γ-phosphates to forming a π–cation interaction with adenine instead (Figs. K117 was still not required for ADP coordination and, instead forms a salt bridge to D183 (Fig. 4c). By contrast, R276 directly interacted with ADP in the luminal-facing state. These structures imply that although four positively charged residues are required for ATP4− translocation, only three are required for ADP3−, and K117 can interact with either a phosphate or D183, depending on the substrate. Finally, Q190 and Q254 formed polar interactions with the nucleobase, which contrasts with the cytoplasmic-facing state in which only hydrophobic contacts were observed (Fig. 4c). Similar to Q254A, the Q190A variant showed a comparatively poorer transport activity than the wild-type protein (Fig. Because the mutations of most nucleotide-coordinating residues were transport inactive, we used the observed ATP-induced thermostabilization as a proxy for probing the requirements for substrate binding (Methods and Supplementary Table 1). Upon increasing the nucleotide concentrations, we observed a maximal thermostabilization of wild-type SLC35B1 at 1 mM ATP (Fig. By contrast, alanine variants of the positively charged residues K117, K120, R276 and K277 showed no clear thermostabilization (Fig. Likewise, the hydrophobic residue variants I257E and V261T severely diminished ATP-induced thermostabilization, although C269A was still able to interact with ATP, consistent with the variant retaining robust transport activity (Figs. We further quantified the ΔTm shift at 1 mM ATP and found that only Q190A retained wild-type-like thermostabilization (Fig. 4d), indicating that the diminished transport of Q190A could be a result of perturbed cavity closing (Fig. 3e). Mutation of the more peripheral polar residues Y25, T273 and Q254 also retained 40–60% of the ATP-induced thermostabilization observed for wild-type protein (Fig. 4d). I257E and V261T displayed diminished ATP-induced thermostabilization to the same extent as the positively charged variants, reinforcing their importance (Fig. 4d). Furthermore, R276A was found to have the greatest impact on nucleotide binding. To analyse the conformational plasticity of SLC35B1, we performed 3D variability analysis (3DVA) (Methods). For the cytoplasmic-facing SLC35B1 structures, cryo-EM map distributions supported the mobility of the TM8–TM9 gating helices, consistent with their movement in transitioning to a luminal-facing state (Supplementary Videos 4 and 5). For the luminal-facing E33A variant with ADP, we unexpectedly observed map density that supported full gate closure of TM3–TM4a gating helices on the ER luminal side with the symmetry-related opening of TM8–TM9 gating helices on the cytoplasmic side (Fig. We were further able to extract E33A particles from which we could reconstruct cryo-EM maps of a cytoplasmic-facing conformation with ADP bound at a resolution of 3.1 Å, as well as improve the map quality of the previously modelled ADP-bound ER luminal-facing structure (Supplementary Fig. 6). From the E33A variant with ADP, we were therefore able to obtain a distribution of cryo-EM maps that covered the full transport cycle (Supplementary Videos 7 and 8). In the cytoplasmic-facing E33A structure, the map quality for ADP was clearly weaker than that in all other structures, and ADP was further positioned approximately 7 Å closer to the cytoplasmic entrance than its position in the wild-type structure (Extended Data Fig. Adenosine was positioned along the same TM8–TM9 gate and, although more poorly coordinated, the β-phosphate retained an interaction with R276 together with Y25 and Q254 (Extended Data Fig. 8e). There was a further potential π–cation interaction with R194 (Extended Data Fig. 8e), which in the ER luminal-facing state, may help stabilize the luminal-facing conformation (Fig. 4a). However, given that R194A retained robust ATP binding and displayed higher transport activity than the wild-type structure, mutagenesis reinforced polar interactions with adenine do not seem to be required for nucleotide binding (Extended Data Figs. Consistently, the E33A cytoplasmic-facing ADP-bound structure was more open than that observed for wild-type apo, wild-type ADP or Q113F AMP–PNP structures, with a larger outward displacement of TM8–TM9, TM1 and TM4b helices (Extended Data Fig. 8g). Together, we have been able to determine human SLC35B1 cryo-EM structures to reconstruct a comprehensive transport model for both ATP and ADP translocations (Fig. a, Cartoon representation of the transport cycle depicting the rearrangements of SLC35B1 bundles (teal and orange) to import ATP into the ER in exchange for ADP. AMP–PNP and ADP in the occluded state were positioned as in the experimental outward-facing (OF) states. The peripheral TM5 and TM10 helices were omitted for clarity. The insert (asterisk) illustrates that we postulated that ADP initially interacted with phosphate first from the ER luminal side but was then flipped in the binding pocket to the coordination observed by cryo-EM. b, Hydrophobic surface representation of the AMP–PNP-bound (cyan) luminal-facing structure (above) and cytoplasmic-facing structure (below), highlighting that adenine in AMP–PNP was accommodated by a hydrophobic patch of residues in an otherwise positively charged substrate-binding site. c, Top, surface representation of cytoplasmic- and luminal-facing structures. Bottom, as in the top panel with AMP–PNP (sticks; cyan). Bottom, in the rocker-switch mechanism seen here for SLC35B1, the substrate-binding site was asymmetric, and, as such, the substrate was vertically repositioned during substrate translocation. Graphic in a was created using BioRender (https://biorender.com). It was shown over 30 years ago that ATP is transported into crude ER microsomes with Km of 3–5 µM (ref. 23). Here we combined genetic, biochemical and structural analyses to confirm that SLC35B1 is not an NST but the route for ATP entry into the ER. A recent biochemical study has obtained robust ADP/ATP transport kinetics for SLC35B1 in proteoliposome assays33, further supporting our independent analysis. In fact, the closest isoform to SLC35B1 is SLC35B2, for which biochemical analysis has shown is not an NST but an importer of 3′-phosphoadenosine-5′-phosphosulfate34 — a molecule structurally similar to ADP (Supplementary Fig. 8a,b). The SLC35B2 transporter delivers 3′-phosphoadenosine-5′-phosphosulfate to the Golgi, where it is used as a donor for the sulfation of glycan sugars, such as heparan sulfate, which is a crucial post-translational modification in cellular physiology35. Although SLC35B1 and SLC35B2 share approximately 30% sequence identity, the similar substrates and conservation of substrate-binding residues (Supplementary Fig. 8a) indicate that they are likely to operate by means of a similar transport mechanism. Transporters must attract the right solutes to their cavity, and because ATP is highly negatively charged, a positively charged cavity was expected. Compared with the NSTs for GDP-mannose19 (Vrg4) and CMP–sialic acid29 (SLC35A1), however, the luminal- and cytoplasmic-facing cavities are narrower and more positively charged in SLC35B1 (Extended Data Fig. 8h). Furthermore, because both ADP and AMP–PNP adopted an unusual bent conformation in the cytoplasmic-facing state, adjusting to the position of the hydrophobic and positively charged surfaces, it is unlikely that the polar and bulkier nucleotide sugars could be accommodated in this substrate-binding site (Figs. The problem encountered is that ATP is now positioned only part-way down the cavity. At this position, the small protein cannot move around the nucleotide, as it typically would in a conventional rocker-switch mechanism32,36. It seems SLC35B1 has evolved a unique way to counter this problem. First, the adenine moiety is rather hydrophobic, and so becomes initially positioned in a hydrophobic patch (Fig. 5b,c). Rather than being restrained by polar residues, we postulate that hydrophobic interactions are important because they enable the nucleobase to easily readjust its position with the closure of TM8–TM9 gating helices (Fig. 5a–c). The trade-off for a less discriminative nucleobase binding site might be the lack of selectivity observed against other dinucleotide or trinucleotide phosphates. In this case, however, this might be tolerated because the cytoplasmic concentration of ATP is sixfold to tenfold higher than that of any other trinucleotide phosphates37. Specifically, TM9 undergoes a large rigid-body movement during translocation around R276, whereas TM4a–TM4b helices can independently adjusted their position to sustain coordination with the nucleotide (Fig. 4c). However, in the return step, we propose that it is unlikely that ADP would bind nucleobase first when entering from the ER luminal side because electrostatic interactions attract over a longer distance and adenine nucleotides bind phosphate first from the cytoplasmic side. Instead, we propose that ADP transiently binds phosphate first when entering from the ER luminal side but then flips around the positively charged residues to the position observed in the cryo-EM structures. Notably, the luminal-facing cavity is hydrophobic on one side, which would enable repositioning of the greasy adenine moiety (Extended Data Fig. 8i). In the return step, we think that ADP will first be repositioned approximately 6.5 Å to the state observed for ADP in the wild-type cytoplasmic-facing state, whereas the distal site observed in the E33A variant structure is likely to be only transiently occupied. Although there might be further en route intermediates, we conclude that because the substrate-binding site in the cytoplasmic-facing state is asymmetric compared to its location in the luminal-facing state, ATP import into the ER by SLC35B1 must use a stepwise mechanism, which further includes vertical repositioning of the substrate (Fig. An emerging theme in the transport of amphipathic substrates, such as lipids and nucleotides, is that they can populate several positions along the translocation pathway38,39,40. Most relevant to SLC35B1 is that exhaustive biochemical and structural analyses strongly indicate that a stepwise translocation mechanism is likely adopted by the mitochondrial ADP/ATP carrier SLC25A4, although nucleotide-bound states are yet to be obtained to confirm this21. Although each transporter is fine-tuned differently, evolution may have converged on similar solutions for the transport of large amphipathic substrates. Finally, given the high demand for ATP in the ER for protein folding and adaptive ER stresses41,42,43, it will be important to establish how and if ATP import by SLC35B1 is regulated by the levels of ATP produced and exported by the mitochondria14. In conclusion, our fundamental mechanistic insights into the import of ATP into the ER not only expand our understanding of SLC transport mechanisms, but solidifying its function also provides an important framework for exploring new avenues in cellular metabolism and physiology. Human SLC35B1 was previously cloned into the yeast GAL1 inducible vector pDDGFP2 (ref. SLC35B1 variants were constructed using polymerase chain reaction (PCR)-based mutagenesis and recloned, as previously described46. The resulting vectors were transformed into Saccharomyces cerevisiae strain FGY217 (MATα, ura3-52, lys2Δ201 and pep4Δ)47. Cells harbouring GFP-fused SLC35B1-expressing plasmids were incubated in 12 l of URA medium containing 0.1% (w/v) glucose at 30 °C in 2-l shaking flasks. Protein expression was induced by the addition of galactose to a final concentration of 2% (w/v) when the optical density at 600 nm reached 0.6. After a 22-h incubation at 30 °C, the cells were harvested, resuspended in a buffer (containing 50 mM Tris-HCl (pH 7.6), 1 mM EDTA and 0.6 M sorbitol) and lysed using a high-pressure cell disruption system (Constant Systems). Membranes were isolated by means of ultracentrifugation at 4 °C and 195,000g for 2 h, homogenized in a buffer (containing 20 mM Tris-HCl (pH 7.5), 0.3 M sucrose and 0.1 mM CaCl2), flash-frozen in liquid nitrogen and stored at −80 °C until use. SLC35B1 and its variants were purified, as previously described46. In brief, isolated membranes from S. cerevisiae cultures containing GFP-fused SLC35B1 were diluted to a total protein concentration of 3.5 mg ml−1 in a buffer containing 1× PBS, 150 mM NaCl, 10% (v/v) glycerol and 1% (w/v) n-dodecyl-β-d-maltoside (DDM). The membranes were solubilized by mild agitation at 4 °C for 1 h, followed by centrifugation at 120,000g at 4 °C for 45 min to remove the non-solubilized material. Imidazole was added to the supernatant to a final concentration of 10 mM, and the mixture was incubated with 10 ml of Ni2+-nitrilotriacetate affinity resin (Ni-NTA; QIAGEN) for 2 h at 4 °C. The resin was washed with 30 column volumes of a buffer containing 1× PBS, 150 mM NaCl, 10% (v/v) glycerol, 0.1% (w/v) DDM and 60 mM imidazole. The protein was eluted in 3 column volumes of 1× PBS, 150 mM NaCl, 0.03% (w/v) DDM and 250 mM imidazole and concentrated to 2 mg ml−1. The concentrate was applied to a PD-10 desalting column (Sephadex G-25; GE) pre-equilibrated in a buffer containing 1× PBS and 0.02% (w/v) DDM. The initial 1.6 ml of the flow-through was collected and concentrated using a 50-kDa molecular weight cut-off spin concentrator (Amicon; Merck Millipore). The same procedure was used to purify each mutant that was fused with GFP. Before large-scale culturing, the membranes of each respective mutant were assessed for monodispersity by fluorescence-detection size exclusion chromatography (FSEC) using a Shimadzu high-performance liquid chromatography (HPLC) LC-20AD/RF-20A (excitation at 488 nm and emission at 512 nm) and Enrich SEC 650 10 × 300 column (Bio-Rad) in 20 mM Tris-HCl (pH 7.5), 150 mM NaCl and 0.03% (w/v) DDM. GFP thermal shift experiments were performed, as previously described48,49. In brief, purified SLC35B1–GFP was diluted to 0.6 µM in a buffer containing 20 mM HEPES (pH 7.5), 150 mM NaCl and 1% (w/v) DDM. Substrates of interest were added to the purified SLC35B1–GFP to a final (v/v) concentration of 1 mM, and the resulting mixtures were incubated for 10 min on ice. After incubation, the detergent β-n-octyl-β-d-glucopyranoside was added to a final concentration of 1% (w/v). Subsequently, the samples were transferred to PCR tubes and heated for 10 min at 10, 20, 30, 40, 50, 60, 70 and 80 °C using a Veriti 96-well thermal cycler (Thermo Fisher Scientific). The samples were centrifuged at 5,000g for 45 min at 4 °C to pelletize the larger protein aggregates. The resulting supernatants were transferred to a 96-well black plate (Thermo Fisher Scientific), and GFP fluorescence (excitation at 488 nm and emission at 512 nm) was measured using a Fluoroskan microplate fluorometer (Thermo Fisher Scientific). The apparent Tm for each titration was calculated by plotting the average GFP fluorescence intensity from two technical repeats at each temperature and fitting the curves to a sigmoidal dose–response equation using GraphPad Prism v.8.4. ΔTm was calculated by subtracting the average Tm with nucleotide (calculated from three titrations) from the average Tm without nucleotide (calculated from three titrations). The relative difference in concentration-dependent ATP-induced thermostabilization was assessed from the final 1 µM to 5 mM, as previously described, but at a single temperature of 37 °C, which was based on wild-type Tm + 4.2 °C, because we previously found a 4–6 °C temperature increase from the Tm was optimal for monitoring ligand binding49. The normalized fluorescence at each concentration of ATP was calculated relative to the GFP fluorescence measured at the starting (lowest) concentration of ATP. Each titration was performed in triplicate. Purified SLC35B1 was reconstituted into liposomes following the freeze–thaw extrusion method. Lipid extract from bovine brain 7 (Sigma-Aldrich) and cholesteryl hemisuccinate (Sigma-Aldrich) were added at final concentrations of 30 and 5 mg ml−1, respectively, in a buffer containing 10 mM Tris-HCl (pH 7.5) and 2 mM MgSO4. To preload liposomes with nucleotides, 5 µl of a 0.1 M nucleotide stock in 0.5 M Tris (pH 7.5) was added to a 500-µl lipid mix, yielding a final nucleotide concentration of 1 mM. The mixture was flash-frozen and thawed at room temperature before sonication to make unilamellar liposomes. Then 10–20 µl of protein (20 µg total) was added to the liposomes, which were then extruded (LiposoFast; AVESTIN; 400-nm membrane pore size), resulting in large unilamellar proteoliposomes. Liposomes were then diluted in 25 ml of buffer containing 100 mM Tris-HCl (pH 7.5) and 2 mM MgSO4 (transport buffer) and pelleted at 250,000g for 45 min to remove free nucleotides. Finally, the proteoliposomes were resuspended in a transport buffer to a final concentration of approximately 60 mg ml−1. Liposomes without protein (protein-free liposomes) were prepared in the same way but with the addition of the same volume of buffer instead of protein. To calculate the protein reconstitution efficiency, 120 µl of proteoliposomes and protein-free liposomes at 60 mg ml−1 were solubilized in 1× PBS, 150 mM NaCl (pH 7.5) and 1% (w/v) DDM to a final volume of 300 µl for 1 h at 4 °C. Non-solubilized material was pelleted at 250,000g for 45 min at 4 °C, and the resulting supernatant was injected into an ENrich SEC 650 10 × 300 Column (Bio-Rad) pre-equilibrated with 20 mM Tris-HCl (pH 7.5), 150 mM NaCl and 0.03% (w/v) DDM and ran at 1 ml min−1 using a high-performance liquid chromatography system (Shimadzu) in the same buffer. In addition, 0.6 µg of purified SLC35B1 was injected onto the same column and run as stated previously. The AUC values from empty liposomes were subtracted from the AUC proteoliposomes and normalized to the control injection of 100% protein reconstitution. An estimated protein reconstitution of 11% was calculated and used for subsequent calculation of kinetic analysis. For uptake measurements, 5 µl of proteoliposomes was diluted into 45 µl of transport buffer with either [3H]ATP (0.14 μM) (American Radiolabeled Chemicals and Moravek Biochemicals) or [3H]ADP (0.3 μM) (American Radiolabeled Chemicals) and incubated at 25 °C. Transport was stopped by the addition of 1 ml of transport buffer and by rapid filtration through a 0.22-µm mixed cellulose hydrophilic filter (Millipore). Filters with liposomes were then washed with 6 ml of transport buffer, transferred to scintillation vials and emulsified in 5 ml of Ultima Gold scintillation liquid (PerkinElmer) before scintillation counting (TRI-CARB 4810TR 110 V; PerkinElmer). For the IC50 data acquisition, disintegrations per minute values recorded from protein-free liposomes after 2 min were used for baseline subtraction of the respective tested conditions, and the data were later internally normalized. The IC50 values were obtained by fitting a nonlinear regression of [inhibitor] versus the normalized response with a variable slope using GraphPad Prism v.8.4. For competitive uptake assays, the uptake of external [3H]ADP (0.14 μM) was monitored in the presence of 1 mM cold nucleotide in a transport buffer after 60 s. For kinetic analysis, the initial velocities were estimated from the initial 30 s of the time-course experiment. A mixture of radiolabelled ADP and ADP at a 1:18 molar ratio was used for the initial points of the curve (0.2–5 µM), and ratios of 1:36 and 1:64 were used for the last points (10 and 20 µM). These different ratios were later corrected when transforming raw radioactive ADP counts to the amount (pmol) of ADP transported. For each concentration of ADP, the disintegrations per minute values from protein-free liposomes were subtracted from their respective proteoliposome values. Final Km and Kcat values were obtained by fitting Michaelis–Menten kinetics using GraphPad Prism v.8.4. NMR samples were prepared as a mixture of 10 μM purified SLC35B1–GFP into proteoliposomes consisting of total bovine brain lipid extract 7 (Sigma-Aldrich), cholesteryl hemisuccinate (Sigma-Aldrich) and 500 μM of the respective substrate, which were pre-dissolved in a buffer in D2O containing 25 mM potassium phosphate (pH 8.2) and 50 mM NaCl. All NMR experiments were performed at 298 K on a Bruker 500 or 700 MHz spectrometer equipped with cryogenic probes. Proteins were saturated using a train of Gaussian-shaped 50-ms-long pulses. The total length of the saturation train was set to 2 s. All NMR spectra were acquired with 4,096 scans per dataset. where I0 are integrated peaks in off-resonance spectra, and I0 − Isat are integrated peaks of the STD spectra. HCT 116 (Research Resource Identifier: CVCL_0291) cells were transduced in triplicates with lentiviral particles containing a transporter-focused CRISPR–Cas9 library (Addgene, 213695) at a multiplicity of infection of 0.3. Cells were selected with blasticidin for 13 days to remove non-transduced cells and passaged in Roswell Park Memorial Institute 1640 (R8758, Sigma) supplemented with 10% fetal bovine serum (10270-106, lot 42F8381K, Gibco) and penicillin–streptomycin (15140-122, Gibco) for 5 weeks. We performed genomic DNA purification, PCR amplification of the single guide RNA (sgRNA) regions and Illumina sequencing, as previously described51. The sgRNA sequences were quantified using MAGeCK count v.0.5.9.2 (ref. 52). Only sgRNAs targeting SLC transporters and control genes were included in further analysis. Raw count tables were used to determine the significant depletion and enrichment of sgRNAs from the pool using the MAGeCK test v.0.5.9.2 with default parameters. Raw sequencing data were deposited in the Gene Expression Omnibus (GEO) (GSE277685). The HCT 116 (CCL-247) cell line was purchased from the American Type Culture Collection and authenticated by means of short tandem repeat profiling. PCR testing confirmed the absence of Mycoplasma infection. All animal experiments conformed to the guidelines of the Guide for the Care and Use of Laboratory Animals of Japan and were approved by the Kyoto University Animal Experimentation Committee. Full-length human SLC35B1 containing residues 1–322 (UniProt accession number P78383) was expressed in the Sf9-baculovirus system and purified. Mouse monoclonal antibodies against SLC35B1 were raised essentially, as previously described54. In brief, a proteoliposome antigen was prepared by reconstituting purified SLC35B1 at a high density into phospholipid vesicles consisting of a 10:1 mixture of chicken egg yolk phosphatidylcholine (egg PC; Avanti Polar Lipids) and adjuvant lipid A (Sigma-Aldrich) to facilitate an immune response. MRL/lpr mice were immunized with proteoliposome antigen using three injections at 2-week intervals. Antibody-producing hybridoma cell lines were generated by using a conventional fusion protocol. Biotinylated proteoliposomes were prepared by reconstituting SLC35B1 with a mixture of egg yolk phosphatidylcholine and 1,2-dipalmitoyl-sn-glycero-3-phosphoethanolamine-N-(cap biotinyl) (16:0 biotinyl Cap-PE; Avanti) and used as binding targets for conformation-specific antibody selection. The targets were immobilized on streptavidin-coated microplates (Nunc). Hybridoma clones producing antibodies recognizing conformational epitopes in human SLC35B1 were selected using an ELISA on immobilized biotinylated proteoliposomes (liposome enzyme-linked immunosorbent assay), allowing positive selection of antibodies that recognized the native conformation of SLC35B1. Further screening for reduced antibody binding to SDS-denatured SLC35B1 was performed for negative selection against linear epitope-recognizing antibodies. The stable complex formation between SLC35B1 and each antibody clone was checked using FSEC55. A monoclonal antibody (clone number YN4027) that specifically binds to and stabilizes the conformational epitopes in SLC35B1 was selected. The sequence of Fab YN4027 was determined by means of standard 5′-rapid amplification of cDNA ends using the total RNA isolated from hybridoma cells. The Fab molecules have a pseudo-symmetrical axis. To overcome this problem, we created an asymmetric fiducial marker with a single synthetic polyprotein consisting of the YN4027 variable-light domain, short linker, MBP, another linker and YN4027 variable-heavy domain. The resulting Fv–MBP fusion protein was used as a cryo-EM fiducial marker for SLC35B1. The linkers are underlined, and MBP is italicized. The DNA sequence of Fv–MBP was inserted downstream of and in frame with the secretion signal sequence of the plasmid pNY326 (Takara Bio/Clontech). To facilitate purification of the secreted proteins, the TEV protease cleavage site sequence, His6 tag and HA tag were added at the C-terminal. B. choshinensis cells harbouring the Fv–MBP expression plasmid were grown at 30 °C with shaking at 200 rpm in 2SY medium (40 g l−1 of soytone, 5 g l−1 of yeast extract, 20 g l−1 of glucose and 0.15 g l−1 of CaCl2) supplemented with 50 mg l−1 of neomycin for 65–70 h. The recovered culture supernatant was adjusted to a final ammonium sulfate concentration of 60% saturation. The resulting precipitate was pelleted, dissolved in Tris-buffered saline (TBS) buffer (10 mM Tris-HCl (pH 7.5) and 150 mM NaCl) and dialysed overnight against the same buffer. The dialysed sample was purified using Ni-NTA resin, mixed with TEV-His6 and dialysed overnight again against the TBS buffer. The cleaved His6-HA tag and TEV-His6 were removed using a HisTrap column. The flow-through fractions were further purified using a HiLoad 16/600 Superdex 75 pg column (Cytiva) equilibrated with the TBS buffer. The peak fractions were pooled, concentrated, flash-frozen in liquid nitrogen and stored at −80 °C. For cryo-EM sample preparation, the purified SLC35B1–GFP fusion was incubated at 4 °C overnight with equimolar TEV protease during dialysis in a 3-l buffer containing 20 mM HEPES (pH 7.5), 150 mM NaCl and 0.006% (w/v) glyco-diosgenin (GDN). The dialysed mixture was applied to a 5-ml HisTrap HP column pre-equilibrated with 20 mM HEPES (pH 7.5), 150 mM NaCl, 15 mM imidazole and 0.006% GDN, and the flow-through was collected, concentrated to around 2 mg ml−1, flash-frozen and stored at −80 °C. The purified SLC35B1 protein was incubated on ice with Fv–MBP antibody at a molar ratio of 1:1.2 for 30 min. The complex was isolated by size exclusion chromatography in a buffer containing 20 mM HEPES (pH 7.5), 150 mM NaCl and 0.006% (w/v) GDN detergent. For data collection with nucleotides, either 5 mM AMP–PNP or 5 mM ADP was added to the protein–Fv–MBP complex at 4 °C before blotting. The concentrated protein sample (3 µl) was applied to either QUANTIFOIL Cu R2/1 (wild type and Q113F) or QUANTIFOIL Cu R1.2/1.3 (E33A) grids and blotted for the optimal time for each construct (3 s for wild type and 1.5 s for E33A and Q113F) at 4 °C under 100% humidity and plunge frozen in liquid ethane using Vitrobot Mark IV (Thermo Fisher Scientific). Cryo-EM datasets were collected using a Titan Krios G3i microscope equipped with a Gatan BioQuantum K3 detector in the super-resolution hard-binned mode. The videos were collected at ×130,000 magnification with aberration-free image shift and fringe-free imaging using EPU (Thermo Fisher Scientific). Image processing for all datasets was performed using the CryoSPARC software56. For apo data, 31,550 videos were recorded. After the CTF estimation, micrographs with an estimated resolution worse than 5.5 Å were rejected. 2D templates were generated from a set of 1,000 micrographs by means of blob picking and 2D classification. Around 9.9 million particles were extracted after template-based picking from the entire dataset. These particles were then subjected to several rounds of 2D classification. Around 1.1 million particles belonging to good 2D classes were selected and subjected to multimodel ab initio reconstruction. A good class containing 443,730 particles was selected and subjected to another round of hetero-refinement and multimodel ab initio reconstruction to remove the particles corresponding to the junk classes. Finally, 180,530 particles were selected for the final round of non-uniform refinement, resulting in a 3D reconstruction with a gold-standard Fourier shell correlation (FSC) resolution of 3.7 Å. To improve the alignments, the flexible MBP domain was masked out and local refinements were performed, which gradually improved the gold-standard FSC resolution to 3.37 Å. For the SLC35B1 AMP–PNP-bound structure, 16,532 of 17,914 micrographs had an estimated CTF resolution better than 6 Å and were selected for further image processing. Around 8.7 million particles were extracted after template-based picking and subjected to several rounds of 2D classification. Around 420,000 particles were selected, and the initial 3D volumes were obtained using multi-class ab initio reconstruction. We further cleaned up 218,098 particles corresponding to a good 3D reconstruction using several rounds of hetero-refinement. Finally, 114,510 particles were selected and refined to a high resolution using non-uniform refinement. For the SLC35B1 ADP-bound structure, 25,806 of 28,798 micrographs had an estimated CTF resolution better than 6 Å and were selected for further image processing. Around 4.5 million particles were extracted after template-based picking and subjected to several rounds of 2D classification. We selected 1,082,175 particles and obtained the initial 3D volumes using multi-class ab initio reconstruction. We further cleaned up 665,896 particles corresponding to a good 3D reconstruction using several rounds of hetero-refinement. Finally, 323,707 particles were selected and refined to a high resolution using non-uniform refinement. To further improve the map features, the volume corresponding to MBP was masked before local refinement. For the SLC35B1(Q113F) AMP–PNP-bound structure, 33,714 of 34,801 micrographs had an estimated CTF resolution better than 6 Å and were selected for further image processing. Around 7.7 million particles were extracted after template-based picking and subjected to several rounds of 2D classification. Around 1.2 million particles were selected, and initial 3D volumes were obtained using multi-class ab initio reconstruction. We further cleaned up 590,082 particles corresponding to a good 3D reconstruction using several rounds of hetero-refinement. Finally, 332,121 particles were selected and refined to a high resolution using non-uniform refinement. To further improve the map features, reference-based motion correction was performed, and the volume corresponding to MBP was masked out before local refinements. For the SLC35B1(E33A) AMP–PNP-bound structure, 33,579 of 34,144 micrographs had an estimated CTF resolution better than 6 Å and were selected for further image processing. Around seven million particles were extracted after template-based picking and subjected to several rounds of 2D classification. We selected 658,505 particles and obtained the initial 3D volume using multi-class ab initio reconstruction. We selected 313,397 particles and refined them to a high resolution using non-uniform refinement. To further improve the map features, reference-based motion correction was performed, and the volume corresponding to MBP was masked out before local refinements. A round of hetero-refinement was performed to further clean up the data, and a high-resolution reconstruction containing 223,502 particles was obtained after another round of local refinement. A 3.16-Å resolution map was initially obtained after the local refinement from 306,109 particles with the transporter protein in an outward open conformation. A 3D variability analysis indicated the presence of several conformations; as such, 3D classification without alignment was performed using some of the frames from 3DVA as input volumes. MBP was masked, and local refinements were performed. The final reconstructions had overall resolutions of 3.15 and 3.12 Å. 44) model of human SLC35B1 was in the outward open conformation and showed poor side-chain fitting in the cryo-EM maps. To determine the protein conformational state, de novo model building into the apo SLC35B1 maps was instead performed using ModelAngelo software57. The output model was examined and manually adjusted using Coot58. The structure of the Fv fragment was also built de novo using ModelAngelo57 and manually examined and adjusted using Coot58. AMP–PNP was modelled into the extra non-proteinaceous density present at the binding site using Coot58, and the model was refined in Phenix59 using real-space refinement. Model building for the ADP-bound SLC35B1 and AMP–PNP-bound SLC35B1(Q113F) variants was also performed using the apo structure as a starting model, followed by manual adjustment in Coot58. Because MBP was masked during refinement, only Fv and the transporter domains were built. The models were refined in Phenix59 using real-space refinement. For model building of the luminal-facing AMP–PNP- and ADP-bound SLC35B1(E33A) reconstructions, a model obtained from AlphaFold 2 (ref. 44) was fitted to the maps using ChimeraX60. The models were manually examined and adjusted using Coot58 and refined using Phenix59 real-space refinement. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The coordinates and maps for SLC35B1 have been deposited in the Protein Data Bank (PDB) and Electron Microscopy Data Bank (EMDB) with entries 9GSL/EMD-51551 (wild-type apo), 9GRZ/EMD-51529 (wild-type AMP–PNP), 9I20/EMD-52578 (wild-type ADP), 9GRY/EMD-51528 (Q113F AMP–PNP), 9GS7/EMD-51541 (E33A AMP–PNP), 9GS5/EMD-51539 (E33A-ADP-OF) and 9GS3/EMD-51538 (E33A-ADP-IF), respectively. Correspondence and request for materials should be addressed to D.D. Source data are provided with this paper. & Kunji, E. R. S. The SLC25 mitochondrial carrier family: structure and mechanism. Hirschberg, C. B., Robbins, P. W. & Abeijon, C. Transporters of nucleotide sugars, ATP, and nucleotide sulfate in the endoplasmic reticulum and Golgi apparatus. Klein, M. C. et al. AXER is an ATP/ADP exchanger in the membrane of the endoplasmic reticulum. An ATP transporter is required for protein translocation into the yeast endoplasmic reticulum. & Hendershot, L. M. The endoplasmic reticulum (ER) chaperone BiP is a master regulator of ER functions: getting by with a little help from ERdj friends. Key steps in ERAD of luminal ER proteins reconstituted with purified components. Wijeyesakere, S. J., Gagnon, J. K., Arora, K., Brooks, C. L. 3rd & Raghavan, M. Regulation of calreticulin-major histocompatibility complex (MHC) class I interactions by ATP. & Meyer, D. I. Sac1p mediates the adenosine triphosphate transport into yeast endoplasmic reticulum that is required for protein translocation. & Park, J. S. Identification of the ATP transporter of rat liver rough endoplasmic reticulum via photoaffinity labeling and partial purification. & Mayinger, P. Sac1p plays a crucial role in microsomal ATP transport, which is distinct from its function in Golgi phospholipid metabolism. Hadley, B. et al. Nucleotide sugar transporter SLC35 family structure and function. Maszczak-Seneczko, D., Wiktor, M., Skurska, E., Wiertelak, W. & Olczak, M. Delivery of nucleotide sugars to the mammalian Golgi: a very well (un)explained story. Yong, J. et al. Mitochondria supply ATP to the ER through a mechanism antagonized by cytosolic Ca2+. Ondo, K., Arakawa, H., Nakano, M., Fukami, T. & Nakajima, M. SLC35B1 significantly contributes to the uptake of UDPGA into the endoplasmic reticulum for glucuronidation catalyzed by UDP-glucuronosyltransferases. Kainuma, M., Chiba, Y., Takeuchi, M. & Jigami, Y. Overexpression of HUT1 gene stimulates in vivo galactosylation by enhancing UDP–galactose transport activity in Saccharomyces cerevisiae. The ortholog of human solute carrier family 35 member B1 (UDP-galactose transporter-related protein 1) is involved in maintenance of ER homeostasis and essential for larval development in Caenorhabditis elegans. Parker, J. L. & Newstead, S. Structural basis of nucleotide sugar transport across the Golgi membrane. Mavridou, V. et al. Substrate binding in the mitochondrial ADP/ATP carrier is a step-wise process guiding the structural changes in the transport cycle. Characterization of ligand binding by saturation transfer difference NMR spectroscopy. & Hirschberg, C. B. Translocation of ATP into the lumen of rough endoplasmic reticulum-derived vesicles and its binding to luminal proteins including BiP (GRP 78) and GRP 94. Li, H. et al. A novel and unique ATP hydrolysis to AMP by a human Hsp70 binding immunoglobin protein (BiP). A., Coincon, M. & Drew, D. Structural basis for the delivery of activated sialic acid into Golgi for sialyation. & Newstead, S. Structural basis for substrate specificity and regulation of nucleotide sugar transporters in the lipid bilayer. Barland, N. et al. Mechanistic basis of choline import involved in teichoic acids and lipopolysaccharide modification. Ahuja, S. & Whorton, M. R. Structural basis for mammalian nucleotide sugar transport. Structural basis for amino acid export by DMT superfamily transporter YddG. Shih, P., Pedersen, L. G., Gibbs, P. R. & Wolfenden, R. Hydrophobicities of the nucleic acid bases: distribution coefficients from water to cyclohexane. Shared molecular mechanisms of membrane transporters. & Bredeston, L. M. The broad range di- and tri-nucleotide exchanger SLC35B1 displays asymmetrical affinities for ATP transport across the ER membrane. Kamiyama, S. et al. Molecular cloning and identification of 3′-phosphoadenosine 5′-phosphosulfate transporter. Kauffman, F. C. Sulfonation in pharmacology and toxicology. Drew, D. & Boudker, O. Ion and lipid orchestration of secondary active transport. Traut, T. W. Physiological concentrations of purines and pyrimidines. Sun, Y. et al. Molecular basis of cholesterol efflux via ABCG subfamily transporters. Nguyen, C. et al. Lipid flipping in the omega-3 fatty-acid transporter. Recognition of cyclic dinucleotides and folates by human SLC19A1. Chronic enrichment of hepatic endoplasmic reticulum–mitochondria contact leads to mitochondrial dysfunction in obesity. Tubbs, E. et al. Mitochondria-associated endoplasmic reticulum membrane (MAM) integrity is required for insulin signaling and is implicated in hepatic insulin resistance. Beaulant, A. et al. Endoplasmic reticulum-mitochondria miscommunication is an early and causal trigger of hepatic insulin resistance and steatosis. Highly accurate protein structure prediction with AlphaFold. Newstead, S., Kim, H., von Heijne, G., Iwata, S. & Drew, D. High-throughput fluorescent-based optimization of eukaryotic membrane protein overexpression and purification in Saccharomyces cerevisiae. Drew, D. et al. GFP-based optimization scheme for the overexpression and purification of eukaryotic membrane proteins in Saccharomyces cerevisiae. Kota, J., Gilstring, C. F. & Ljungdahl, P. O. Membrane chaperone Shr3 assists in folding amino acid permeases preventing precocious ERAD. Nji, E., Chatzikyriakidou, Y., Landreh, M. & Drew, D. An engineered thermal-shift screen reveals specific lipid preferences of eukaryotic and prokaryotic membrane proteins. Chatzikyriakidou, Y., Ahn, D. H., Nji, E. & Drew, D. The GFP thermal shift assay for screening ligand and lipid interactions to solute carrier transporters. Group epitope mapping by saturation transfer difference NMR to identify segments of a ligand in direct contact with a protein receptor. Ferdigg, A., Hopp, A. K., Wolf, G. & Superti-Furga, G. Membrane transporters modulating the toxicity of arsenic, cadmium, and mercury in human cells. Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2022 update. Structure of the bile acid transporter and HBV receptor NTCP. Kawate, T. & Gouaux, E. Fluorescence-detection size-exclusion chromatography for precrystallization screening of integral membrane proteins. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Automated model building and protein identification in cryo-EM maps. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. The EMBL-EBI Job Dispatcher sequence analysis tools framework in 2024. B., Martin, D. M., Clamp, M. & Barton, G. J. Jalview Version 2—a multiple sequence alignment editor and analysis workbench. We are grateful to M. Claesson for the critical reading of the paper and the Cryo-EM Swedish National Facility at SciLifeLab for cryo-EM data collection. This study was predominantly funded by the Knut and Alice Wallenberg Foundation (D.D.) This study was partially supported by JSPS KAKENHI (grant no. ), the Joint Usage/Research Center Program of the Institute for Life and Medical Sciences at Kyoto University (N.N.) and the AMED Basis for Supporting Innovative Drug Discovery and Life Science Research (BINDS; JP24ama121007 to N.N. Part of this study was conducted within the RESOLUTE project. RESOLUTE received funding from the Innovative Medicines Initiative 2 Joint Undertaking under grant agreement no. This Joint Undertaking received support from the European Union's Horizon 2020 research and innovation programme and EFPIA. This study reflects only the authors' views, and neither IMI nor the European Union and EFPIA are responsible for any use that may be made of the information contained therein. Open access funding provided by Stockholm University. These authors contributed equally: Ashutosh Gulati, Do-Hwan Ahn, Albert Suades Ashutosh Gulati, Do-Hwan Ahn, Albert Suades, Yurie Hult & David Drew CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Cloning, expression screening and sample preparation for cryo-EM were performed by D.-H.A., Y.H. and A.S. Fv–MBP generation was carried out by S.I. CRISPR–Cas9 knockout and cell growth assays were performed by G.W. and G.S.-F. Cryo-EM data collection and map reconstruction were performed by A.G. Model building was performed by A.G. with support from D.D. STD NMR and GFP thermal shift experiments were performed by D.-H.A. Experiments for transport assays were performed by A.S. All authors discussed the results and commented on this paper. G.S.-F. is co-founder and owns shares of Solgate GmbH, an SLC-focused company. The other authors declare no competing interests. Nature thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Phylogenetic tree depicting evolutionary relationship across all human SLC35 members. b, Thermal stabilization of purified SLC35B1-GFP WT in the presence of either 1 mM ATP (white bar) or other nucleotides, UDP-galactose (black bars). c, FSEC traces of SLC35B1-GFP in detergent solubilised membranes (grey) and after purification (bright green). d, Saturation-transfer difference (STD) NMR spectra (red) in response to the addition of various nucleotides to SLC35B1 proteoliposomes as labelled, as well as their respective off resonance 1H spectra (black). e, Single time point (1 min) uptake of [3H]-ADP by SLC35B1 in proteoliposomes (black bars) preloaded with either 1 mM ATP or 1 mM AMP-PNP compared to protein-free liposomes (white bars). Error bars are the mean ± s.e.m of n = 3 independent experiments. f, IC50 curves for the competitive inhibition of [3H]-ADP uptake by external cold ADP (black), ATP (green), AMP (ochre), UTP (red), CTP (cyan), GTP (purple) and AMP-PNP (pink) nucleotides in SLC35B1 proteoliposomes that were preloaded with 1 mM of the respective nucleotide. Activity was normalized after subtraction of the non-specific uptake as estimated from protein-free liposomes. Error bars are the mean ± s.e.m of n = 3 or 6 (AMP-PNP) independent experiments. a, Pooled transporter-focused CRISPR/Cas9 proliferation assay in HCT 116 cells. Log2-fold changes of sgRNA frequencies are plotted against the robust ranking aggregation (RRA) score as determined by MAGeCK. b, Structure of the cryo-EM fiducial marker Fv-MBP used for cryo-EM structural studies of human SLC35B1. c, Size exclusion chromatograph (SEC) of purified SLC35B1 mixed with Fv-MBP fusion protein at a molar ratio of 1:1.2, respectively. inset: SDS-PAGE of the first peak at 16 mL (1) and second 17 mL (2); uncropped gel is shown in Supplementary Fig. First peak corresponds to SLC35B1 in complex with Fv-MBP fusion protein and second peak is uncomplexed Fv-MBP. d, Saturation-transfer difference (STD) NMR spectrum (red) of ATP addition to the SLC35B-Fv-MBP fusion protein complex and its off-resonance 1H spectra (black) in proteoliposomes. a, Cryo-EM datasets of SLC35B1 in GDN detergent were processed using CryoSPARC56. Movie frames were aligned using the “Patch motion correction” and contrast transfer function was estimated using the “patch CTF” algorithms. Datasets were pruned using multiple rounds of 2D classifications, initial maps were generated using multiclass ab initio reconstruction and cleaned using heterogenous refinement. Final apo SLC35B1 cryo-EM maps were reconstructed from 180, 530 particles after local refinement with MBP masking, with an overall resolution of 3.37 Å resolution according to the FSC at 0.143. b, As in a, for SLC35B1 with AMP-PNP. The Fv-MBP fusion protein was masked during local refinement. The final reconstruction was obtained from 114,510 particles with an overall resolution of 3.41 Å according to the FSC at 0.143. a, Modelled structure with corresponding cryo-EM maps for all transmembrane helices in SLC35B1 apo (brown). The images were rendered with a map contour level of 0.12 using ChimeraX60. b, Modelled structure with corresponding cryo-EM maps for all transmembrane helices in SLC35B1 with AMP-PNP (teal). The images were rendered with a map contour level of 0.13 in ChimeraX60. c, Model and corresponding density for Fv fragment as observed in the apo SLC35B1 WT structure. The figure was rendered with a map contour level of 0.12 using ChimeraX60. d, left: Side-view of cartoon representation of apo (brown) and AMP-PNP bound (orange, teal and gray) SLC35B1 structures after superimposition. AMP-PNP is shown as sticks (cyan). Structures were aligned using PyMol61 with a Cα r.m.s.d of 1.0 Å. Both structures are in the cytoplasmic-facing conformation. right: as viewed from the cytoplasmic side. e, Cryo-EM maps for the AMP-PNP nucleotide (sticks) in WT SLC35B1 at a map contour level of 0.12 using ChimeraX60, which could be built in potentially two different conformations (orange, grey) due to the circular nature of the map density. a, STD NMR spectrum (red) in response to the addition of ATP to SLC35B1 Q113F proteoliposomes and the off resonance 1H spectrum (black). b, IC50 curves for external ATP competition of [3H]-ADP/ATP normalized transport activity by Q113F proteoliposomes. Error bars are the mean ± s.e.m of n = 6 independent experiments carried out from two separate reconstitutions. c, Mutant analysis by single time point uptake of [3H]-ADP/ATP exchange in proteoliposomes. Data was normalized to the absolute signal of SLC35B1 WT. Error bars are the mean ± s.e.m of n = 6 independent experiments carried out from two separate reconstitutions. d, Side view of the structural superimposition of WT SLC35B1 (orange, teal, grey) and Q113F (magenta) bound to AMP-PNP (sticks and cyan for WT, magenta for Q113F); structures were aligned using PyMol61 with a Cα r.m.s.d of 1.1 Å. e, Cartoon representation of the ER lumen cavity-closing contacts (sticks and labelled) for SLC35B1 WT (orange, teal, grey) and Q113F (magenta). Only minor conformational differences were observed with the introduced Q113F forming hydrophobic interactions with L105 and Y110. f, Cryo-EM map density for AMP-PNP (sticks, magenta/red) in the Q113F structure. g, left: Cryo-EM map density for AMP-PNP (teal) and neighbouring residues in SLC35B1 WT (teal). middle: As in the left panel for Q113F (purple) variant with AMP-PNP (teal). right: Cryo-EM map density and structure of apo SLC35B1 WT (brown). h, Comparison of the nucleotide binding residues in the apo WT SLC35B1 (brown) and AMP-PNP bound Q113F (orange and teal) structures. Both S118 and D183 are also highly conserved (see Supplementary Fig. a, [3H]-ADP uptake after 2 min for SLC35B1 proteoliposomes (black bars) pre-loaded with either ADP (homo-exchange) or ATP (hetero-exchange) and empty, protein free liposomes (white bars). Error bars are the mean ± s.e.m of n = 3 independent experiments. b, IC50 curves for external ATP competition of normalized transport activity by proteoliposomes for SLC35B1 under either homo-exchange [3H]-ADP/ADP (black-filled circles) or hetero-exchange [3H]-ADP /ATP (green-filled circles) conditions. Data was fitted using the non-linear function [Inhibitor] vs normalized response function in GraphPad prism. Error bars are the mean ± s.e.m of n = 3 independent experiments. c, Density of the peripheral lipid (lavender) observed in the cryo-EM map of SLC35B1 WT with ADP (transparent light grey). d, Superimposition of the AMP-PNP bound Q113F structure (grey) with the ADP bound WT structure (cyan). Lipid density (purple transparent) matching PE (cyan sticks) was observed peripheral to TM3 and TM6 helices in the WT structure with ADP. e, Cryo-EM map density (grey mesh) for ADP (sticks) in the cytoplasmic-facing WT structure (cartoon) and surrounding residues (sticks). f, Normalised SLC35B1-mediated uptake of [3H]-ADP in competition with either buffer (white-bar) or cold nucleotides (black-bars) into proteoliposomes preloaded with cold 1 mM ATP. External ATP is included from Fig. Error bars are the mean ± s.e.m of n = 6 independent experiments from two separate reconstitutions. g, Single time point of [3H]-ADP uptake by SLC35B1 in proteoliposomes preloaded with either 1 mM ATP (white bar), CTP, UTP (black bars) or nucleotide free (red bar). Error bars are the mean ± s.e.m of n = 6 independent experiments carried out from two separate reconstitutions. h, left: Cytoplasmic view of the structural superimposition of the cytoplasmic-facing AMP-PNP (cyan sticks) bound SLC35B1 structure (orange, teal and grey) with the occluded CMP-Sialic acid SLC35A1 (structure PDB: 6OH2, transparent pink). If the cytoplasmic-facing SLC35B1 protein would adopt a similar occluded conformation as SLC35A1, then the inward movement of TM8-TM9 gating helices (white arrow) would physically clash with the bound nucleotide. a, STD NMR spectrum (red) in response to the addition of ATP to SLC35B1 E33A proteoliposomes and the off resonance 1H spectrum (black). b, IC50 curves for external ATP competition of [3H]-ADP/ATP normalized transport activity by E33A proteoliposomes. Error bars are the mean ± s.e.m of n = 6 independent experiments carried out from two separate reconstitutions. c, Cryo-EM map density (grey mesh) for AMP-PNP (blue sticks) in the luminal-facing state E33A structure (cartoon). d, Cryo-EM map density (grey mesh) for ADP (mustard sticks) in the luminal-facing state E33A structure (mustard). e, Concentration dependent thermostabilization of purified SLC35B1 with ATP (black-filled circles) and AMP (white circles). The y-axis represents the relative fluorescent signal (see Methods) and dotted line highlights that the maximal stabilization was observed at 1 mM ATP. Error bars are the mean ± s.e.m of n = 6 independent experiments. Error bars are the mean ± s.e.m of n = 6 independent experiments. The WT is shown as a reference, as in panel e. Error bars are the mean ± s.e.m of n = 6 independent experiments. Error bars are the mean ± s.e.m of n = 6 independent experiments. a, Structural superimposition of cytoplasmic facing WT with ADP and the occluded CMP-Sialic acid (SLC35A1) transporter structure (PDB 6OH2), as viewed from the cytoplasm. b, Structural superimposition of ER luminal-facing E33A variant with AMP-PNP and the occluded CMP-Sialic acid (SLC35A1) transporter structure (PDB 6OH2), as viewed from the ER lumen. c, Structural superimposition of cytoplasmic facing WT with ADP, the occluded CMP-Sialic acid (SLC35A1) transporter structure (PDB 6OH2) and an occluded model of human SLC35B1 based on an AF2 worm model (Wuchereria bancrofti: AF-A0A3P7E1A7-F1-v4), as viewed from the ER lumen. d, Cryo-EM map density (grey mesh) for ADP (brown sticks) in the cytoplasmic-facing state E33A structure (cartoon). e, Structural superimposition of the AMP-PNP (grey sticks) bound cytoplasmic-facing SLC35B1 WT (grey cartoon) and ADP (brown sticks) bound cytoplasmic-facing E33A variant (cartoon, orange and teal). Dashed-box for zoomed in view, with the few ADP interacting residues (dashed lines) labelled. f, Comparison between the luminal-facing E33A variant structure (pink) with ADP (pink sticks) and the cytoplasmic-facing E33A variant structure (orange, teal) with ADP (brown/red sticks). Upon ER luminal gate closure (top black arrow), substrate is vertically displaced by ~12 Å (blue arrow). g, Comparison of the cytoplasmic-facing WT-apo (light blue), WT-ADP (salmon) and Q113F-AMP-PNP (grey) structures with the ADP bound (brown sticks) cytoplasmic-facing E33A variant structure (orange, teal) adopting a more open conformation. h, Electrostatic surface representations of the luminal-facing yeast GDP-mannose transporter Vrg4 (PDB: 5OGK) structure bound to GDP-mannose (pink), which adopts a more tilted position than AMP-PNP (grey) in the luminal-facing SLC35B1-E33A structure. The cavity is lined by positively charged residue on one side and hydrophobic residues on the other, which we propose would allow for flipping of the greasy adenine after interacting phosphate first from the ER luminal side. Map density was rendered at a threshold of 0.013 using ChimeraX. The conserved nucleotide-binding site residues are shown as sticks. In this morph, a fully occluded state was not formed because the inside-gating helices TM8–TM9 and the outside-gating helices TM3–TM4a moved at the same time, rather than sequentially. As in Supplementary Video 2 but viewed from the cytoplasm. View from the cytosolic side is shown. Movement towards the partial closing of cytosolic gate helices TM8 and TM9 is highlighted with blue arrows. View from the cytosolic side is shown. Minimal movement towards the partial closing of cytosolic gate helices TM8 and TM9 is highlighted with blue arrows. View from the side is shown. The opening of cytosolic gate helices TM1 and TM9 is highlighted with blue arrows. Same as Supplementary Video 6 but viewed from the ER luminal side. The TM1, TM4a and TM9 luminal gate closure is indicated by blue arrows. Map density was rendered at a threshold of 0.028 using ChimeraX. The conserved nucleotide-binding site residues are shown as sticks. In this morph, a fully occluded state was not formed because the inside-gating helices TM8–TM9 and the outside-gating helices TM3–TM4a moved at the same time, rather than sequentially. Same as Supplementary Video 9 but viewed from the cytoplasmic side. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Gulati, A., Ahn, DH., Suades, A. et al. Stepwise ATP translocation into the endoplasmic reticulum by human SLC35B1. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08986-0'>Multigenerational cell tracking of DNA replication and heritable DNA damage</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 15:37:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3,4,5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development. Cellular heterogeneity, including genetic and non-genetic heterogeneity, is pervasive in nature, yet insufficiently represented in ensemble behaviours of cell populations6,7,8. Stochastic processes in cells, such as the ones involved in transcription, translation, protein turnover and cell division, cause fluctuations in cellular components and in biochemical reactions that can drive phenotypic variability. Such stochastic variation may be buffered or amplified by deterministic factors, most of which are unknown9,10. Phenotypic plasticity is a hallmark of cancer, and most tumours show genetic and non-genetic spatial and temporal heterogeneity, which affects adaptability and resistance to cancer therapies11,12. Classical Darwinian somatic evolution driven by mutations, selection and clonal expansion appears to be insufficient to fully explain cancer progression and responses to therapies13,14. Moreover, genomic lesions that shape cancer genome evolution and adaptation are typically not resolved within a single cell cycle but, instead, segregate into subsequent cell generations15,16. How genetic and non-genetic heterogeneity are intertwined and the dynamics with which phenotypic heterogeneity emerges and is propagated across cell generations are poorly understood. Polyploidization by whole-genome duplication occurs frequently in cancer and is associated with enhanced phenotypic variability17,18. Polyploidization can lead to chromosomal instability and aneuploidy19,20,21, which correlate with therapy resistance and poor patient outcomes22,23. Aneuploidy can arise through erroneous cell division, and asymmetric cell divisions promote cell-to-cell variability, including both genetic and non-genetic heterogeneity24,25. Alternative routes to polyploidization exist, but how they affect genome integrity and cellular variability between cancer cells remains unclear. A main driver of carcinogenesis is oncogene-induced replication stress26. Endogenous and oncogene-induced replication stress give rise to heritable DNA lesions that are transmitted through cell division to daughter cells27. Such heritable lesions may be an inevitable consequence of stochastic replication origin activation, resulting in under-replicated DNA in large replicons of the human genome28. In daughter cells, inherited genomic lesions regulate G1 duration and determine the decision between quiescence and cell cycle commitment29,30,31,32,33. They are bound by the chromatin reader 53BP1, which protects them from nucleolytic degradation27 and regulates their replication timing in the next S phase34. While representing a paradigm for inheritance of genomic lesions, whether 53BP1-marked chromatin scars affect sister cell heterogeneity and are propagated to subsequent cell generations is unclear. Here, to overcome limitations of cell tracking and complement genomics-focused retrospective analyses of cancer cell evolution, we devised an approach for quantitative multigenerational single-cell tracking and lineage analysis. The approach is based on endogenous protein labelling through CRISPR–Cas9-mediated genome editing, tailored cell segmentation and tracking algorithms, and single-cell end-point measurements by iterative staining. Applied to various conditions of oncogenic replication stress and DNA damage, it provides time-resolved insights into damage inheritance and emergence of sister cell asymmetry, and sheds light on routes towards polyploidization and associated phenotypic variability that may contribute to cancer evolution. To develop an experimental framework enabling simultaneous tracking of individual cells and their fates across multiple cell generations, we first used cells stably expressing ectopic H2B–GFP. The bright nuclear H2B–GFP signal facilitated software-assisted image segmentation, enabling us to define conditions for multi-day live-cell imaging and design custom scripts for semi-automated single-cell tracking (Methods and Extended Data Fig. To test whether multigenerational single-cell tracking could be combined with tracking DNA lesions in individual cells, we turned to cells stably expressing ectopic mEGFP–53BP1. The nuclear mEGFP–53BP1 signal was bright enough for automated image segmentation and cell tracking (Fig. 1a), and the appearance and disappearance of nuclear 53BP1 bodies resembling replication-stress-associated DNA lesions could be quantified in a time-resolved manner in the daughter and granddaughter cell generation and could be depicted as cell lineage trees (Fig. a, Representative images from time-lapse microscopy with multigenerational cell tracking of mEGFP–53BP1 U-2 OS cells. The daughter cell generation (F1) and granddaughter cell generation (F2) generations with 53BP1 foci after cell division are highlighted. b, Example cell lineage depicting lineage relationships and DNA damage marked by 53BP1. Time-lapse microscopy for 65 h at 30 min intervals. c, High-content-microscopy-derived EdU profiles of the parental and the edited 53BP1–PCNA U-2 OS cell line. d, Representative images of endogenously tagged 53BP1 from 53BP1–PCNA U-2 OS cells treated as indicated and fixed after 45 min. e, Representative images from 53BP1–PCNA U-2 OS cells depicting endogenous PCNA patterns during S phase. f, Representative images of the cell-tracking procedure, depicting the smoothened endogenous PCNA signal for nuclei segmentation, the nuclear mask and the single-cell tracks applied on the non-smoothened PCNA signal. g, Single-cell lineages of the cells depicted in f. Endogenous PCNA patterns are depicted according to the colour code indicated. h, The same single-cell lineages as in g, with endogenous PCNA foci intensities depicted according to the colour code indicated. i, Single-cell lineages from endogenously tagged 53BP1–PCNA U-2 OS cells. PCNA foci are depicted according to the colour code indicated. j, The same single-cell lineages as in i, with cell cycle phase of the F1 generation, based on the endogenous PCNA signals, depicted according to the colour code indicated (G1, S, G2). k, The cell cycle distribution derived from the time spent in G1, S and G2 according to the endogenous PCNA foci pattern of the single-cell lineages shown in i and j. Representative single-cell images are shown below. Having established that endogenous DNA lesions marked by 53BP1 can be tracked quantitatively across multiple cell generations, we aimed to overcome limitations associated with ectopic protein overexpression by using CRISPR–Cas9-engineered cells containing the coding sequence for mScarlet-I in-frame with the last exon of TP53BP133. To combine time-resolved measurements of heritable DNA lesions with DNA replication patterns, we also tagged the replication factor PCNA by CRISPR–Cas9-mediated knock-in of mEmerald in frame with the last exon (Extended Data Fig. The engineered double-tagged cells showed normal proliferation, unperturbed cell cycle profiles and a normal response to DNA damage compared to the parental cells (Fig. 1c and Extended Data Fig. No detectable phototoxicity was observed in time-lapse microscopy experiments, as measured by cell cycle profiling, EdU incorporation and cell-cycle-resolved analysis of the DNA damage marker γH2AX (Extended Data Fig. Like mEGFP–53BP1 cells, endogenous DNA lesions could be measured with sufficient resolution using the endogenous 53BP1 protein signal (Extended Data Fig. 2a) and 53BP1–mScarlet formed nuclear repair condensates at endogenous DNA lesions and at sites of induced DNA damage (Fig. The endogenous 53BP1 foci detected through the mScarlet signal co-localized with foci from 53BP1 immunofluorescence, and their specificity was validated by 53BP1 knockdown (Extended Data Fig. The endogenous PCNA–mEmerald signal marked regions of active replication in S phase, which were confined to EdU-positive cells (Fig. 1e and Extended Data Fig. Segmentation of the PCNA replication pattern enabled cell cycle staging in single-cell lineages of asynchronously growing cells (Fig. 1f–h), and cell cycle distributions based on PCNA replication patterns from live-cell imaging closely matched cell cycle distributions based on 4′,6-diamidino-2-phenylindole dihydrochloride (DAPI)/EdU profiles from fixed cell populations (Fig. 1i–k and Extended Data Fig. Moreover, the endogenous PCNA signal in the double-tagged cells could be used for multigenerational cell tracking (Supplementary Video 1). Next, we tracked a cohort of asynchronously growing cells for 30 h and performed in silico alignment of the cell lineages according to cell cycle position based on the PCNA signal (Extended Data Fig. As expected, 53BP1 nuclear bodies marking regions of inherited endogenous replication stress appeared primarily after cell division in G1 and were cleared as cells entered S phase (Extended Data Fig. By contrast, when mild replication stress was induced by a low dose (200 nM) of the DNA polymerase inhibitor aphidicolin (APH), which did not induce measurable amounts of DNA damage as detected by western blot analysis of the DNA damage markers phosphorylated KAP1 (pKAP1) and pRPA (Extended Data Fig. 1), 53BP1 foci formation was slightly enhanced and occurred not only in G1 cells but also during S-phase progression (Extended Data Fig. Inhibition of the central checkpoint kinase ATR resulted in more severely perturbed replication patterns and in elevated 53BP1-marked genomic lesions after cell division (Extended Data Fig. ATR inhibition was also associated with DNA damage as detected by pKAP1 and pRPA (Extended Data Fig. Consistent with the perturbed replication patterns observed in single cells after APH and ATR inhibitor (ATRi) treatment (Extended Data Fig. 3e,g), cell population based EdU profiles were also altered (Supplementary Fig. Both treatments resulted in increased PCNA foci in cells, which, according to EdU and DNA content, would be classified as G2, indicating replication-stress-induced DNA synthesis beyond normal S phase (Supplementary Fig. To extend these analyses to additional stress markers, including post-translational protein modifications inaccessible for live-cell imaging, we combined multigenerational time-lapse microscopy for lineage analyses with sequential immunofluorescence staining35 of the tracked cells in a cell-cycle-resolved manner, that is, live + quantitative image-based cytometry (Live+QIBC); Extended Data Fig. For multiplexing after live imaging, we focused on γH2AX, a DNA damage-induced histone phosphorylation, on pRb as a marker of cell cycle commitment after cell division, on the tumour suppressor protein p53, which is stabilized after DNA damage, and on its downstream target, the CDK inhibitor p21. All markers showed nuclear signals that could be eluted efficiently (Extended Data Fig. No cross-talk between sequential stainings was observed, as demonstrated by the inversely correlated signals of pRb and p21 when imaged in the same colour channel (Extended Data Fig. Overall, APH and ATRi treatments increased the levels of γH2AX, p53 and p21, and reduced pRb (Extended Data Fig. 4f–i), and treatment effects on single cells could be analysed in a cell-cycle-resolved manner on the basis of the DAPI signal (Extended Data Fig. We combined single-cell tracks from multigenerational live-cell imaging for 55 h, covering up to three cell generations, with end-point measurements obtained by sequential staining. As before (Extended Data Fig. 3), unchallenged asynchronously growing cells had comparatively sharp cell cycle phase transitions, daughter cells showed little heterogeneity in G1 duration and S-phase onset after cell division, and the nuclear area increased gradually during the cell cycle until mitosis (Fig. DNA damage levels measured by 53BP1 were low and mostly confined to G1, and the granddaughter cell generation had background levels of γH2AX, p53 and p21, and high pRb. By contrast, asynchronously growing cells treated with a low dose of APH when they were in G2 showed perturbed S-phase entry in the daughter cell generation, with seemingly normal S-phase exit into G2 and normal cell division (Fig. However, compared with untreated cells, the granddaughter cells had elevated γH2AX and p53 levels, and individual granddaughter cells showed high p21 and low pRb, which was associated with defective S-phase commitment (Fig. When asynchronously growing cells were treated with ATRi in G2, the next cell cycle seemed prolonged, with sister cells showing perturbed DNA replication and heterogeneity in S-phase duration, as well as 53BP1 foci formation, and γH2AX and p21 induction (Fig. PCNA foci, 53BP1 foci and the area of the nucleus over 55 h of live imaging are shown. Sequential staining intensities from individual cells of the lineage (sc) and the mean intensities from the reference population (rp) are depicted for the markers pRb, γH2AX, p21 and p53. b, Representative video stills from the single-cell lineage in a and corresponding images from the sequential staining of cells for pRb, γH2AX, p21 and p53. c, Single-cell lineage from 53BP1–PCNA U-2 OS cells treated with 0.2 µM APH during the G2 phase of the cell cycle. PCNA foci, 53BP1 foci and area of the nucleus over 55 h of live imaging are shown. Sequential staining intensities from individual cells of the lineage and mean intensities from the corresponding reference populations are shown. d, Single-cell lineage from 53BP1–PCNA U-2 OS cells treated with 1 µM of ATRi during the G2 phase of the cell cycle. PCNA foci, 53BP1 foci and the area of the nucleus over 55 h of live imaging are shown. Sequential staining intensities from individual cells of the lineage and mean intensities from the corresponding reference populations are shown. For the box plots, the box limits show the interquartile range (IQR, 25th percentile (Q1) to 75th percentile (Q3)), the median (centre line) and the whiskers define the lower and upper adjacent value; the dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR; n > 35 end-point measurements per sample. Drugs in c and d were removed after 24 h. Cells that received APH during S-phase progression showed S-phase prolongation and signs of DNA synthesis in an extended G2 phase, yet their progeny entered S phase seemingly normally (Extended Data Fig. However, cells that received ATRi during S-phase showed G2 acceleration and increased sustained 53BP1 foci in the next cell generation, associated with heterogenous and strongly impaired S-phase entry (Extended Data Fig. Finally, cells that received APH before S-phase entry in G1 showed greatly prolonged cell cycle duration without further cell division during the 55 h of the experiment (Extended Data Fig. 5c), whereas sister cells that received ATRi in G1 exhibited asymmetric S-phase duration and DNA damage accumulation, which was associated with p21 induction and loss of pRb (Extended Data Fig. When scored across multiple sister cell pairs, sister cell heterogeneity was increased at the level of 53BP1 nuclear bodies, γH2AX levels, and p53 and p21 induction (Extended Data Fig. Other markers of impaired cell cycle progression and increased heterogeneity were also elevated after APH or ATRi treatment, including perturbed replication (assessed by PCNA patterns), periods with highly elevated nuclear 53BP1 foci and reduced numbers of cell division during the time-lapse imaging period (Supplementary Fig. Considering the observed p53–p21 induction and their tumour-suppressive functions, we performed multigenerational cell tracking after p53 or p21 depletion. Both knockdowns seemed to accelerate cell cycle progression and to synergize with APH and ATRi to cause perturbed replication, increased DNA damage and enhanced cellular heterogeneity (Supplementary Fig. Moreover, depletion of the tumour suppressor AMBRA1, which controls the G1/S transition through cyclin D regulation36,37,38, synergized with APH and especially ATRi (Supplementary Fig. Collectively, these results shed light onto how replication stress (here induced by APH or ATRi), defective checkpoint signalling (here induced by ATRi) or loss of tumour suppressor functions (here induced by p53, p21 or AMBRA1 depletion) induce cellular heterogeneity. Although analysed merely at the phenotypic level, it is conceivable that such changes may accelerate cellular transformation and evolution of drug resistance. To validate these findings in an independent cell system, we used CRISPR–Cas9-mediated genome engineering to fluorescently label endogenous 53BP1 and PCNA in hTERT-immortalized, non-cancer RPE-1 cells. Endogenous 53BP1–mScarlet foci were easily detectable, induced after ionizing radiation (IR), could be segmented, co-localized with antibody-stained 53BP1 and were specific as revealed by 53BP1 knockdown (Extended Data Fig. Endogenous PCNA–mEmerald patterns in RPE-1 cells were also detectable, could be segmented and marked EdU-positive S-phase cells (Extended Data Fig. Similar to U-2 OS cells, APH and ATRi increased γH2AX, p53 and p21 levels, and reduced pRb (Extended Data Fig. 6f–i), with APH hampering more the G2/M transition, as judged by the increase in 4N cells with low pRb and high p21, and ATRi causing cells to accumulate in G1 with low pRb and high p21 (Extended Data Fig. This is consistent with APH-mediated slowing of DNA replication and ATRi-mediated inhibition of the G2/M checkpoint. Similar to U-2 OS cells, APH caused mild and ATRi more severe DNA damage, as measured by KAP1 phosphorylation (Extended Data Fig. Tracking of endogenously tagged RPE-1 cells across multiple cell generations was possible yet more difficult owing to their high motility (Supplementary Fig. Moreover, iterative staining was complicated by their loose attachment, resulting in cell loss during sequential rounds of washing and signal elution. Nevertheless, single-cell lineages confirmed the main phenotypes observed in U-2 OS cells, including asymmetric S-phase onset and perturbed replication patterns in individual sister cells after APH, ATRi and IR (Supplementary Fig. To investigate potential sources of heterogeneity, we first performed bulk RNA-sequencing (RNA-seq) analysis, revealing differentially expressed genes after IR-induced DNA damage (Extended Data Fig. We found that DNA damage response and cell cycle regulating genes were enriched (Extended Data Fig. 7b), with CDKN1A (encoding p21), FAS, GADD45A and MDM2 being among the most highly induced genes (Fig. 3a and Extended Data Fig. By contrast, TP53BP1 expression was not significantly induced (Fig. 3a and Extended Data Fig. 7c), consistent with its constitutive expression and mechanism of action through recruitment to sites of DNA damage. Single-cell RNA-seq (scRNA-seq) analysis showed very similar (Extended Data Fig. 7d) and reproducible effects (Extended Data Fig. A clustering analysis of the scRNA-seq results revealed primarily genes involved in DNA replication, repair and cell division as main drivers of DNA-damage-induced cellular heterogeneity (Extended Data Fig. We next analysed highly variable genes in the unchallenged and IR-treated conditions by computing the mean and s.d. of the normalized expression values and performing a linear fit of the log2-transformed s.d. versus the log2-transformed mean (Extended Data Fig. Highly variable genes were identified as those of which the residual of the fit was above 0.5. Significantly more highly variable genes were found in the IR-treated condition compared with in the untreated condition, including some genes of the DNA damage response pathway, which are highlighted by closed circles (Fig. Further analysis showed that genes with high variability (residuals above 0.5) only in the IR condition were predominantly involved in cell cycle regulation (Fig. 3c), suggesting that DNA damage-induced increased cellular heterogeneity is linked to cell-to-cell differences in cell cycle control. a, Volcano plot showing differentially expressed genes in U-2 OS cells 24 h after IR, derived from bulk RNA-seq analysis. b, Residual variability of gene expression after a linear fit of the log2-transformed s.d. to the log2-transformed mean expression in U-2 OS cells 48 h after IR from scRNA-seq analysis. Positive residual values indicate a higher s.d. Genes of the DNA damage response pathway (GO: 0006974) are highlighted by closed circles. P = 2.45 × 10−12; odds ratio = 1.90 (Fisher's exact test on genes with residuals > 0.5 versus genes with residuals ≤ 0.5). c, GO analysis of genes with specifically increased residuals (>0.5) after IR treatment. d, U-2 OS cells were treated with the indicated doses of IR and fixed 45 min later for 53BP1 foci analysis by QIBC. The G1 populations were selected based on DAPI and EdU and categorized on the basis of the nuclear 53BP1 levels. The IR-induced 53BP1 foci formation in G1 cells was then analysed as a function of IR dose and 53BP1 expression. e, Single-cell lineage from 53BP1–PCNA U-2 OS cells treated with 4 Gy IR as indicated. (left) and 53BP1 foci formation (right) are depicted by the colour code. f, U-2 OS cells were treated as indicated (1 µM ATRi; 4 Gy IR) and γH2AX and 53BP1 foci in G1 cells were analysed by QIBC. Example images are shown to the right. Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. To investigate potential sources of non-genetic heterogeneity beyond induced gene expression changes, we turned to 53BP1, the expression of which was not significantly changed after DNA damage (Extended Data Fig. 53BP1 requires self-association through its oligomerization domain for foci formation at sites of DNA damage and shows features of phase separation, which is greatly affected by protein concentration33,39. Consistently, we found that nuclear 53BP1 concentration, at the single-cell level (analysed specifically in G1 cells to avoid interference from DNA replication), limits 53BP1 foci formation in response to increasing doses of IR (Fig. Consistently, we identified single-cell lineages, in which nuclear 53BP1 levels correlated with the strength of 53BP1 foci formation after IR in sister cells (Fig. Taken together, these results suggest that 53BP1 recruitment is limited by its nuclear concentration, and that heterogeneity in 53BP1 between individual cells impacts the response to DNA damage. Consistent with 53BP1 being limiting for the response to DNA damage at single-cell level, when cells were pretreated with ATRi to trigger replication stress-induced 53BP1 condensates in G1 cells, before acute DNA damage induction by IR, the formation of ATRi-induced 53BP1 condensates impaired the 53BP1 response to acute DNA damage (Fig. Different from γH2AX foci, which showed an additive increase after ATRi and IR, 53BP1 recruitment to IR-induced breaks was reduced due to its sequestration at replication stress-induced DNA lesions, which in turn resulted in IR-induced γH2AX foci not being served by 53BP1 (Fig. To investigate further how a singular event of acute DNA damage affects cellular heterogeneity, we focused on IR. As IR, in principle, induces DNA strand breaks at random sites in replicating genomes, which can lead to non-symmetric damage loads in duplicated DNA, we deliberately focused on cells that received IR in the G1 phase of the cell cycle. DNA breakage in G1 and their subsequent repair, indicated by the induction and disappearance of 53BP1 foci, was associated with pronounced heterogeneity in the next cell generation between sister cells (Extended Data Fig. Similar effects could be observed in RPE-1 cells (Supplementary Fig. To test whether we could recapitulate these findings in a larger cohort of cells, we sequentially pulse-labelled asynchronously growing cell populations with EdU and BrdU to identify cells that, at the time of irradiation, were in G1 and that had then, after transient cell cycle arrest, gone through a complete cell cycle until being analysed in their next G1 phase (Supplementary Fig. Notably, the irradiated cycled cells had more 53BP1 nuclear bodies in G1 compared with their controls (Extended Data Fig. 8c), consistent with the live-cell tracking data. As IR causes not only DNA double-stranded breaks (DSBs), but can, through direct and indirect routes, induce a variety of DNA lesions, we used DSB inducible via AsiSI (DIvA) cells for endonuclease-mediated DSB formation40. DSBs were induced by 4-hydroxytamoxifen (4-OHT) as measured by 53BP1 and γH2AX foci formation, and most of these foci disappeared after indole-3-acetic acid (IAA)-mediated degradation of the AsiSI endonuclease fused to an auxin-inducible degron, indicating repair (Extended Data Fig. 4-OHT-induced repairable DSBs were also detected by live imaging of DIvA cells, and they were associated with DNA lesions and sister cell heterogeneity in daughter and granddaughter cells (Extended Data Fig. 8f), similar to IR-induced DNA breaks. Likewise, cycled DIvA cells after transient AsiSI induction had more 53BP1 nuclear bodies in G1 compared with their uninduced controls (Extended Data Fig. Consistent effects were observed in RPE-1 cells, especially after p53 loss (Extended Data Fig. When we quantified cellular heterogeneity across multiple pairs of tracked sister cells of which the parental cells had received IR in G1, sister cell asymmetry was increased at several levels (53BP1, γH2AX, p53, p21) compared with the untreated control cells (Extended Data Fig. Thus, even a single, transient genotoxic event in the parental DNA may result in a persistent genomic scar that affects cell cycle commitment and replication timing in the next cell generation, leading to increased asymmetry between sister cells and increased phenotypic heterogeneity within a cell population. Alternatively, damaged DNA might sequester proteins with dual functions in DSB repair and replication fork protection away from replication factories, thereby affecting replication fidelity indirectly, or aberrant repair processes such as breakage–fusion–bridge events might induce sister cell heterogeneity. These mechanisms are not mutually exclusive, and they jointly imply that a single genotoxic event can induce heterogeneity and lasting phenotypic changes in a cell population. One way to rapidly increase phenotypic heterogeneity of cancer cells is through polyploidization17,18. Polyploidization can lead to chromosomal instability and aneuploidy, which are frequently observed in cancers19,20,21 and which correlate with therapy resistance and poor patient outcomes22,23. Using time-resolved single-cell tracking, based on the PCNA signal, we observed examples of notably asymmetric replication patterns in sister cells, such as two rounds of replication without cell division in one of the two sister cells, which in principle could cause polyploidization in this branch (Extended Data Fig. To assess potential routes towards polyploidization in more detail, we used the NEDD8-activating enzyme (NAE) inhibitor pevonedistat (MLN4924, TAK924), a first-in-class anticancer drug that induces DNA rereplication, that is, the replication of already replicated DNA resulting in replication bubbles within larger replicons, through CDT1 stabilization41,42,43. Consistently, pevonedistat caused polyploidization in multiple cancer and non-cancer cell lines, which was associated with replication stress and an increase in nuclear size (Fig. 4a and Extended Data Fig. Moreover, single-cell tracking combined with sequential staining and DAPI-based DNA content measurements confirmed continuous replication patterns and polyploidization, in agreement with rereplication (Fig. However, tracking of sister cells also revealed distinct, asymmetric replication patterns consistent with endoreplication (also referred to as endoreduplication or endocycling, meaning multiple rounds of DNA replication without cell division) (Fig. Thus, two mechanistically distinct routes may lead to pevonedistat-induced polyploidization, rereplication and endoreplication (Fig. 4e,f and Extended Data Fig. a, High-content-microscopy-derived cell cycle profiles of U-2 OS cells treated with increasing doses of pevonedistat (pevo. ; MLN4924) for 24 h and released into fresh medium for 42 h. The percentage of hyperploid cells (>4N DNA content, marked by the dashed vertical line) is indicated. b, A single-cell lineage from 53BP1–PCNA U-2 OS control cells and cells treated with 175 nM of pevonedistat. The total DAPI intensity of the nucleus from the corresponding reference populations (ctrl) is depicted as well as from the pevonedistat-treated single cell at the end of live imaging. c, A single-cell lineage from 53BP1–PCNA U-2 OS cells treated with 175 nM of pevonedistat. The total DAPI intensity of the nucleus from the corresponding reference populations is depicted as well as from the pevonedistat-treated single cells at the end of live imaging. d, A second example of a single cell lineage from 53BP1–PCNA U-2 OS cells (details as in c). e, Representative single-cell analyses of the PCNA patterns of cells that undergo endoreplication or rereplication, respectively. f, Representative video stills of 53BP1–PCNA U-2 OS cells undergoing either endoreplication or rereplication after being treated with pevonedistat. PCNA patterns are depicted. Schematics depicting differences between endoreplication and rereplication are shown on the right. g, The total DAPI intensity (a.u.) of normal G2 cells and cells that underwent either endoreplication (endo.) after being treated with pevonedistat. h, The mean nuclear intensity of γH2AX of normal G2 cells and cells that underwent either endoreplication or rereplication after being treated with pevonedistat. i, As in h for p53 intensity. j, As in h, but for p21 intensity. k, As in h, but for pRb intensity. Statistical analysis was performed using two-tailed unpaired t-tests between endoreplicating and rereplicating cells. The horizontal solid lines represent the mean and horizontal dashed lines represent s.d. The box plots in b–d show the IQR (box limits), with median (centre line) and the whiskers define the lower and upper adjacent value; dots show outliers greater than Q3 + 1.5 × IQR. To assess whether the route towards increased ploidy matters for genome stability, we compared the DNA damage markers obtained through sequential staining after cell tracking in polyploid cells that had gone through either rereplication or endoreplication. Despite analysing cells with comparable DNA content after rereplication versus endoreplication, rereplicating cells acquired consistently higher DNA damage markers than cells that had gone through endoreplication (Fig. Similar results were obtained when polyploidization was induced by depletion of the replication inhibitor and CDT1 antagonist geminin44,45 (Extended Data Fig. Next, to address whether activation of common oncogenes might induce similar patterns of phenotypic heterogeneity and polyploidization, we generated cell lines that, in addition to expressing endogenously tagged 53BP1 and PCNA, overexpress oncogenic H-RAS V12 (HRAS) or cyclin E1 (Extended Data Fig. As expected, overexpression of HRAS or cyclin E1 induced replication stress, which was associated with reduced replication fork speed and an increase in micronuclei formation (Extended Data Fig. Using these cell lines, we performed single-cell tracking and lineage analysis over 70 h, covering up to four cell generations (Extended Data Fig. Oncogene overexpression not only markedly increased cellular heterogeneity and sister cell asymmetry compared with the control, but also resulted in enhanced polyploidization (Fig. 5a and Extended Data Fig. Induction of polyploidy in HRAS and cyclin E1 cells was further increased by IR (Extended Data Fig. Supporting their proliferative potential, polyploid cells showed EdU incorporation (Extended Data Fig. 11d) and could be followed by live-cell microscopy through multiple rounds of cell division (Extended Data Fig. Notably, oncogene-induced polyploidization occurred by either rereplication or endoreplication, similar to pevonedistat-treated cells (Extended Data Fig. Consistently, polyploidization was observed in RPE-1 cells with inducible cyclin E1 (Extended Data Figs. Taken together, these results suggest that routes towards polyploidization in interphase involve both rereplication and endoreplication induced by the same oncogenic triggers. We noticed that endoreplication in HRAS- and cyclin-E1-overexpressing cells was associated with increased DNA damage, marked by elevated 53BP1 foci, after the first and before the second round of DNA replication (Fig. Similarly, cells that were irradiated in G2 showed more frequent endoreplication compared with cells that were irradiated in G1, whereas the opposite was true for replication patterns indicative of rereplication (Extended Data Fig. DNA damage experienced early in the cell cycle therefore increases the risk of replication stress and rereplication in the ensuing S phase, while DNA damage experienced late in the cell cycle increases the risk of endoreplication. a, Single-cell lineage from 53BP1–PCNA U-2 OS cells overexpressing HRAS. PCNA foci, 53BP1 foci and area of the nucleus over 70 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from the corresponding reference populations are depicted for the markers pRb, γH2AX, p21 and p53. The total DAPI intensity per nucleus from the corresponding reference populations is depicted as well as from the single cells at the end of live imaging. Representative images of the polyploid cells are included as well as their position within the DAPI scatter plot of the G1 and G2 populations. The box plots show the IQR (box limits), with median (centre line) and the whiskers define the lower and upper adjacent value; dots show outliers greater than Q3 + 1.5 × IQR. b, t-Distributed stochastic neighbour embedding analysis of scRNA-seq results. 2N–4N, cells with 2N–4N DNA content; polyploid, cells with a DNA content >4N; pevo., pevonedistat-treated; HRAS, HRAS-overexpressing cells. n > 300 cells per condition. c, Subclustering of the HRAS polyploid sample with a Louvain resolution of 0.5. d, GO analysis of the top 28 genes from the overlap between HRAS polyploid and pevonedistat polyploid subclusters. FDR < 0.00001, P < 0.00001, FC > 4. e, STRING functional protein association network analysis of the top 28 genes. f, Cell-cycle-resolved nuclear cyclin A levels in U-2 OS cells either untreated or treated with 175 nM of pevonedistat for 24 h and then released into fresh medium for 42 h. g, The cellular behaviour towards polyploidy from live imaging experiments of 53BP1–PCNA U-2 OS cells treated with 175 nM of pevonedistat, 20 μM of etoposide or 5 μM of RO3306 for 24 h and then released for 42 h. Mean and individual values are depicted from two biological replicates based on live imaging data. Having observed that both routes cause aberrant genome duplication yet by different mechanisms and at different costs for genome integrity, we investigated differential gene expression patterns among polyploid cells. To this end, we performed fluorescence-activated cell sorting (FACS) for scRNA-seq analysis of polyploid and non-polyploid (2N–4N) cells after pevonedistat treatment or HRAS overexpression, comparing them to non-polyploid empty vector (EV) control cells (Supplementary Fig. Reassuringly, the polyploid HRAS and pevonedistat-treated cells segregated from the EV and their non-polyploid (2N–4N) control populations (Fig. For both, polyploid HRAS and pevonedistat-treated cells, differentially regulated genes were identified (Supplementary Fig. Despite the difference in treatment, 150 differentially regulated genes were common between the HRAS and pevonedistat-treated polyploid cells (Supplementary Fig. We then performed a subcluster analysis of the HRAS polyploid cells (Fig. Pairwise analysis of differential gene expression between the subclusters followed by Gene Ontology (GO) analysis revealed cell cycle regulation and chromosome segregation during mitosis as well as DNA replication and DNA damage signalling consistently as the most highly enriched GO terms, in agreement with these processes being relevant for polyploidization (Supplementary Fig. Performing the same subcluster analysis for pevonedistat-treated polyploid cells yielded very similar results with the same highly enriched GO terms (Supplementary Fig. We therefore focused on the overlap between the HRAS and pevonedistat-treated polyploid samples and particularly on the four subclusters with the strongest differential gene expression (Supplementary Fig. Among those four subclusters, 148 differentially regulated genes were identified using a cut-off of 0.001 for false-discovery rate (FDR) and P value and a fold-change (FC) cut-off of 1.5 (Supplementary Fig. A GO analysis of this group of genes yielded terms related to cell cycle regulation and chromosome segregation as the most significantly enriched terms (Supplementary Fig. Similar results were obtained from an independent replicate scRNA-seq analysis (Supplementary Fig. Using more stringent criteria for FC, FDR and P values (FC > 4, FDR < 0.00001, P < 0.00001), we extracted 28 genes from the original list of 148 genes, which were most significantly deregulated among the subclusters (Supplementary Table 4). These 28 genes were still enriched for GO terms related to cell cycle and mitotic cell division (Fig. 5d) and showed high connectivity in functional network analysis (Fig. In total, 22 of these 28 genes were identified in the independent replicate scRNA-seq analysis (Supplementary Table 4). At the centre of this network are the cyclin-dependent kinase CDK1, cyclin A, aurora kinase B (AURKB) and topoisomerase II alpha (TOP2A) (Fig. To further explore the functional relevance of this network, we analysed nuclear cyclin A expression at protein level using QIBC and observed distinct subpopulations among polyploid cells with high/low nuclear cyclin A levels (Fig. Moreover, while pevonedistat induced both endoreplication and rereplication (with roughly identical shares), TOP2A inhibition by etoposide induced predominantly rereplication, whereas CDK1 inhibition by RO3306 caused mainly endoreplication (Fig. These differences were reflected by the cell cycle patterns of chromatin-bound MCM2, MCM4 and MCM7, constitutive components of the replicative helicase MCM2-7: while etoposide caused heterogenous MCM chromatin loading in S phase and G2, consistent with rereplication during S phase and endoreplication after S-phase completion, CDK1 inhibition by RO3306 primarily caused new loading of MCM in late S/G2, consistent with endocycles (Supplementary Fig. Thus, the identified gene network has functional implications for polyploidization by endoreplication versus rereplication. Cellular heterogeneity is widespread in nature, yet how it emerges remains poorly understood. Stochasticity is involved in replication origin activation, and multiprotein complexes like the replisome show inherent stochastic behaviour at the molecular level46. Moreover, sister cell heterogeneity after replication stress experienced in the previous cell cycle may result from mosaic inheritance of new DNA synthesized by leading- and lagging-strand synthesis, respectively47. Acute DNA damage experienced in G1 also induced sister cell heterogeneity in the next cell generation, including marked differences in S-phase commitment. Although strand-specific DNA lesion segregation, as previously observed for replication-associated damage, DNA single-strand breaks, UV-induced lesions and chemotherapy-induced DNA damage15,16,48,49,50,51,52, can contribute to sister cell heterogeneity, additional mechanisms probably exist. One possibility is the sequestration of proteins with dual functions in DNA damage repair and replication at DSBs, therefore limiting their availability at replication forks after S-phase entry, which in turn could trigger heritable strand-specific changes. Indeed, 53BP1 and its upstream regulators RNF8 and RNF168 are limiting for the response to DSBs53, and they are also involved in replication fork protection54,55. Conversely, sequestration of RNF8, RNF168 and 53BP1 at replication-stress-associated heritable DNA lesions in G1 can limit their ability to respond to acute DSBs, and asymmetric distribution of these proteins or their mRNAs during mitosis may amplify such limitations in individual sister cells. Moreover, DNA damage-induced chromatin changes may have a more long-lasting effect on genome organization. Although controlled DSB induction in primary mouse cells did not cause persistent transcriptional repression56, chromatin alterations after completion of DSB repair in HeLa cells were recently found to manifest as heritable impairments of gene expression57. Moreover, changes in gene expression after micronucleus formation can become heritable after reincorporation of micronuclear DNA into the main daughter cell nucleus58. Similarly, DNA-damage-induced changes in genome organization, beyond the process and duration of genome repair, might affect replication timing in the next cell generation. Considering that replication stress is a major source of endogenous DNA damage, a transformative loop from replication perturbations to DNA damage-induced stable chromatin changes (epigenetic scars) might be induced, which could alter transcription programs and replication timing in the next cell cycle, thereby promoting further replication stress and fuelling cellular plasticity. Possible implications are that transient DNA lesions may have long-lasting effects on genome function through inducing and amplifying cellular heterogeneity. All cell lines were grown at 37 °C under standard cell culture conditions (humidified atmosphere, 5% CO2) in Dulbecco's modified Eagle's medium (DMEM, Gibco) containing 10% FBS (Corning) and 1% penicillin–streptomycin antibiotics. Stable endogenously tagged U-2 OS PCNA–mEmerald 53BP1–mScarlet cells were maintained in the presence of 4 µg ml−1 blasticidin (InvivoGen) and 500 µg ml−1 geneticin (Gibco). Stable endogenously tagged U-2 OS 53BP1–mScarlet cells33 were maintained in the presence of 400 µg ml−1 geneticin (Gibco). Stable U-2 OS PCNA–mEmerald 53BP1–mScarlet cells with overexpression of HRAS or cyclin E1 and their empty vector controls were maintained in the presence of 4 µg ml−1 blasticidin (InvivoGen), 500 µg ml−1 geneticin (Gibco) and 100 µg ml−1 hygromycin B (Invitrogen). RPE-1 PCNA–mEmerald 53BP1–mScarlet cells were maintained in the presence of 4 µg ml−1 blasticidin (InvivoGen) and 500 µg ml−1 geneticin (Gibco). Inducible cyclin E1 RPE-1 cells expressing endogenously tagged PCNA–mEmerald and 53BP1–mScarlet were maintained in the presence of 4 µg ml−1 blasticidin (InvivoGen), 500 µg ml−1 geneticin (Gibco) and 0.5 µg ml−1 puromycin (Thermo Fisher Scientific). U-2 OS 53BP1-GFP AID-DIvA cells40,59 were maintained in the presence of 1 mM sodium pyruvate (Sigma-Aldrich) and 800 µg ml−1 geneticin (Gibco) and 1 µg ml−1 puromycin (Thermo Fisher Scientific). U-2 OS CDC45-GFP MCM4-Halo cells60 were maintained in the presence of 1 mM sodium pyruvate (Sigma-Aldrich) and the HaloTag ligand JF549 (Promega) was added at a concentration of 200 nM for a duration of 20 min before fixation. U-2 OS and RPE-1 cells were reauthenticated by short tandem repeat (STR) profiling in 2025 with 100% match and no detectable contamination (RRID: CVCL_0042 and CVCL_4388). All cell lines used in this study were grown in sterile conditions and routinely tested for mycoplasma. Transfections were performed with Ambion Silencer or Silencer Select siRNAs using Lipofectamine RNAiMAX (Thermo Fisher Scientific) at a final concentration of 25 nM. Negative Silencer Select control NEG1 from Ambion was used as a non-targeting control. The following siRNAs were used (5′–3′): TP53BP1 (s14313; GAAGGACGGAGUACUAAUATT); GMNN (134697; GGAGUCAUUUGAUCUUAUGTT); CDKN1A (s416; GCACCCUAGUUCUACCUCATT); TP53 (s606; GAAAUUUGCGUGUGGAGUATT); and AMBRA1 (s31112; GCUCAACAAUAACAUUGAATT). Cloning was done using chemically competent bacteria generated in-house, derived from Library Efficiency DH5a competent cells (Thermo Fisher Scientific). Primer sequences are provided separately (Supplementary Table 5). Correct cloning and integration into target vectors was confirmed by Sanger sequencing (Microsynth). The mEmerald-P2A-BlasticidinR pUC18 template was cloned by a three-piece Gibson assembly. The pUC18 vector was linearized by PCR using primers pUC18_lin_fwd and pUC18_lin_rev. The mEmerald gene was amplified by two rounds of PCR from the mEmerald-C1 plasmid template (Addgene, 53975) using first primers mEm_GA_fwd and mEm_P2A_rev and the product amplified with GA_pUC18_fwd and mEm_P2A_rev. The blasticidin gene was amplified with primers BlastR_P2A_fwd and BlastR_pUC18_rev. The individual pieces were purified by gel extraction and assembled with Gibson assembly followed by transformation of the product, isolation of plasmids and verification by sequencing. The sgRNA duplex to target Cas9 endonuclease in pX459 (Addgene, 48139) to the endogenous PCNA C-terminal near the stop codon was generated as described61. In brief, primers PCNA_sgRNA_top and PCNA_sgRNA_bot were phosphorylated with T4 phosphonucleotide kinase (NEB) at 37 °C for 30 min followed by a temperature gradient from 95 °C to 25 °C decreasing 5 °C min−1. The product was diluted 100-fold in water and assembled into the vector by Golden Gate assembly using BbsI (NEB) and T4 DNA ligase (NEB) through 12 cycles between 5 min at 37 °C and 5 min at 16 °C followed by transformation, plasmid isolation and sequencing verification of correct insertion. The repair template for tagging endogenous PCNA by homology-directed repair was amplified by PCR from the mEmerald-P2A-BlasticidinR pUC18 plasmid generated as described above using Q5 (NEB) polymerase amplification with primers PCNA_HDR_mEm_fwd and PCNA_mEm_blst_rev followed by PCR purification with a QIAquick PCR purification column according to the manufacturer's instructions. U-2 OS cells with endogenously tagged 53BP1–mScarlet were described previously33. RPE-1 cells with endogenously tagged 53BP1–mScarlet were generated in the same manner. These cell lines were then used as the starting point for endogenous tagging of PCNA with the monomeric green fluorescent protein mEmerald. A total of 40,000 cells was seeded into two wells of a six-well plate one day before transfection. The PCR amplified PCNA mEmerald-P2A-BlasticidinR HDR template and the pX459 Cas9 plasmid generated as described above were used for transfection. Then, 1 µg of the HDR template and 1 µg of the pX459 plasmid were diluted with 250 µl OptiMEM (Invitrogen) and mixed with 6 µl TransIT-LT1 transfection reagent (Mirus). pX459 without the HDR template was prepared similarly as a negative control. This was allowed to stand 15 min before adding it to the cells. Next, 24 h later, cells were transferred to 15 cm dishes. Then, 48 h after transfection, blasticidin (InvivoGen) was added to a final concentration of 4 µg ml−1 for selection the next 7 days. After selection and death of cells transfected without the HDR template, individual colonies were picked by trypsin detachment in cloning cylinders and transferred to a 96-well plate (Greiner µ-clear) for expansion and validation of the presence of green fluorescence. Clones with fluorescence were further expanded and cells collected for genomic PCR with the primers PCNA_genCterm_fwd and PCNA_genCterm_rev to confirm the insertion of DNA with a size corresponding to the mEmerald-P2A-BlasticidinR module by agarose gel electrophoreses imaged on Infinity ST5 Xpress v16.16d. To visualize endogenous 53BP1 and endogenous PCNA with minimal interference of their cellular functions, monoallelic targeting was considered sufficient. The mEmerald-C1 plasmid was a gift from M. Davidson (Addgene, 53975), pSpCas9(BB)-2A-Puro (PX459) was a gift from F. Zhang (Addgene, 48139)61, pmScarlet-i_C1 was a gift from D. Gadella (Addgene, 85044)62. For retroviral transduction, retroviral vectors pBABEneo-HRASV12, a gift from J. Debnath (Addgene, 71304)63 and pBABEneo were used. For them to be compatible with the stable PCNA–mEmerald 53BP1–mScarlet U-2 OS cells, the resistance of the pBABE plasmids was exchanged from neomycin to hygromycin. The pBABE_EV (EV = empty vector) and pBABE_HRAS V12 backbones were amplified by PCR amplification using primers Backbone_fwd and Backbone_rev while adding AscI and MfeI restriction sites. The HygroR insert was also generated by PCR amplification using primers Hygro_fwd and Hygro_rev, adding AscI and MfeI restriction sites. This was followed by incubation with DpnI (NEB) for 2.5 h at 37 °C. PCR-cleanup was done according to the manufacturer's instructions (QIAquick PCR Purification Kit, Qiagen). Both the backbones and HygroR inserts were incubated with AscI (NEB) for 10 h, followed by heat inactivation of the restriction enzyme at 80 °C for 20 min. This was followed by incubation with MfeI (NEB) for 10 h at 37 °C. Backbones were dephosphorylated using rSAP (NEB) for 1 h followed by gel purification (QIAquick Gel Extraction Kit, Qiagen). The ligation of the backbones and HygroR inserts was done at 16 °C for 16 h. Transformation was performed using chemically competent DH5α generated in house, derived from Library Efficiency DH5α competent cells (Thermo Fisher Scientific). Correct sequences were confirmed by control digestion and sequencing. The pBABEneo-cyclin E1 plasmid for retroviral transduction was a gift from P. Janscak (University of Zurich). To exchange the resistance from neomycin to hygromycin, the pBABE_cyclin E1 plasmid was linearized using primers pBABE_lin_fwd and pBABE_lin_rev. After gel extraction, Gibson assembly and transformation were performed. Correct sequences were confirmed by control digestion and sequencing. The pLVX-TetONE plasmid was linearized using the primers pLVX-TetONE_lin_rev and pLVX-TetONE_lin_fwd and the insert for cyclin E1 was amplified with primers pLVX-TetONE_to_cyclinE1_GA_amp_fwd and pLVX-TetONE_to_cyclinE1_GA_amp_rev. After gel extraction, Gibson assembly and transformation were performed. Correct insertion was confirmed by control digestion and sequencing. For the generation of U-2 OS PCNA 53BP1 cells with overexpression of HRAS V12 or cyclin E1, the U-2 OS PCNA 53BP1 cells were used for retroviral transduction. For this, HEK293T Phoenix retrovirus producer cells for transduction were prepared by plating them 48 h before infection in DMEM (Gibco) containing 10% FBS (Corning) and 1% penicillin–streptomycin antibiotics (Gibco). For transduction, chloroquine was added to each plate of HEK293T Phoenix cells at a final concentration of 20 µM 1 h before transduction. For the formation of the calcium-phosphate-DNA co-precipitate deionized water, 10 µg DNA and CaCl2 (Sigma-Aldrich) at a final concentration of 1.25 M were mixed for a final volume of 500 µl per 10 cm plate. After incubating for 5 min, the H2O/DNA/CaCl2/HBS mix was added dropwise to the HEK293T Phoenix cells and then gently distributed. After 6 h of incubation at 37 °C under standard cell culture conditions (humidified atmosphere, 5% CO2), the medium was exchanged to fresh growth medium. Then, 48 h after transduction, the supernatant from the transfected HEK293T Phoenix cells was collected and centrifuged and the supernatant transferred to a new tube. Polybrene (Sigma-Aldrich) was added to the supernatant at a final concentration of 8 µg ml−1. After 3 h, the infection was repeated with fresh viral supernatant. After 24 h, the viral supernatant was exchanged to selection medium (DMEM containing 10% FBS (Corning), 1% penicillin–streptomycin antibiotics (Gibco), 4 µg ml−1 blasticidin (InvivoGen), 500 µg ml−1 geneticin (Gibco) and 100 µg ml−1 hygromycin B (Invitrogen)). After selection, the expression levels and functionality were validated by immunofluorescence staining and western blot analysis. For the generation of RPE-1 cells with inducible overexpression of cyclin E1, RPE-1 cells expressing endogenously tagged PCNA–mEmerald and 53BP1–mScarlet were subjected to lentiviral transduction. For this, a DNA/CaCl2/H2O mix was prepared by mixing 6 µg of LTR plasmid and 6 µg of pCDN-LBH plasmid with 12 µg of oncogene-encoding plasmid with CaCl2 at a final concentration of 250 mM in H2O. After incubating the precipitate for 5 min at room temperature, it was added to HEK293T cells and left overnight at 37 °C and 5% CO2. The next day, the medium was replaced with fresh medium and cells were incubated again overnight. The virus was then collected on two consecutive days, the fractions were pooled and filtered through a 0.45 µm filter and stored at 4 °C. The viral supernatant was diluted with medium 1:2 and added together with 8 µg ml−1 polybrene to the target cells. After overnight incubation, the virus-containing cell supernatant was replaced with fresh medium and cells were incubated for 48 h. Then, puromycin was added at a final concentration of 0.5 µg ml−1 for selection. After selection, the expression levels and functionality were validated by immunofluorescence staining and western blot analysis. On the day before imaging, cells were seeded in Fluorobrite DMEM (Thermo Fisher Scientific, A1896701) supplemented with 10% FBS (Corning), penicillin and streptomycin (Gibco) and GlutaMAX (Gibco) into 96-well µ-plates (Ibidi, 89626). In the case of RPE-1 PCNA–mEmerald 53BP1–mScarlet cells, before cell seeding, the 96-well imaging plates were coated with collagen diluted in water (1:50) (PureCol-S, 5015-20ML, Advanced Biomatrix) according to the protocol provided by the company. The plate was sealed with a Breathe-Easy sealing membrane (Sigma-Aldrich, Z380059). Live-cell imaging of cells was done using a previously described automated widefield GE InCell Analyzer 2500HS high-content screening microscope33 with environmental control for gas (5% CO2 and 20% O2) and temperature (37 °C) using the GE InCell Analyzer 2500 V7.4 acquisition software. The system contains a seven-colour solid-state illuminator (SSI), a PCO-sCMOS camera system (16 bit, 2,048 × 2,048 pixel, pixel size 6.5 × 6.5 μm, readout speed: 272 MHz), two quad band-pass polychroic mirrors and single band-pass emission filters. Single-plane widefield images were acquired at 100 ms exposure for green and 300 ms exposure for orange; no binning was performed. Nine fields per well were acquired at 30 min intervals for up to 72 h, with typically 2–3 wells per experimental condition and at least 3,000 cells seeded per well. To irradiate the cells, the plate acquisition was temporarily paused for irradiation at 130 kvp for 33 s per 1 Gy of irradiation in a Faxitron Cabinet X-ray System Model RX-650. Unless indicated otherwise, irradiation was performed with 4 Gy. The addition of drugs was done likewise. At the end of the live imaging, cells were washed once with PBS followed by fixation with 4% formaldehyde (Sigma-Aldrich) in PBS for 18 min, before being washed once more with PBS followed by storage at 4 °C until multiplex staining. Cells fixed after live-cell imaging were processed for iterative indirect immunofluorescence imaging (4i) as described previously35. The cells were permeabilized with 0.5% Triton X-100 (Sigma-Aldrich) for 5 min followed by washing with PBS. Cells were then stained with DAPI (Thermo Fisher Scientific) for 10 min, washed with PBS and images of the fixed cells with endogenous labels and DAPI were acquired. The signals from the fluorescent proteins were removed by denaturation with elution buffer (0.5 M glycine (BioSolve Chemicals), 3 M urea (Eurobio Scientific), 3 M guanidine hydrochloride (Sigma-Aldrich), 50 mM TCEP (Sigma-Aldrich) in water, pH 2.5) for 10 min, 2 times, followed by three washes with double-distilled H2O. Images were acquired to control for the loss of signal. The cells were then blocked and free cysteines were conjugated by incubation with sBS buffer (1% bovine serum albumin (Sigma-Aldrich), 150 mM maleimide (Sigma-Aldrich) in PBS, pH 7.4) for 1 h. After two washes with PBS the cells were incubated with 100 μl cBS (1% BSA (Sigma-Aldrich) in PBS, pH 7.4) with primary antibodies anti-γH2AX (1:1,000, mouse, BioLegend 613402) and anti-pRb (1:500, rabbit, Cell Signaling Technologies, 8516S) for 2 h. After two PBS washes, cells were incubated with 100 µl cBS with secondary antibodies anti-mouse A488 (1:500, Thermo Fisher Scientific, A11029) and anti-rabbit A647 (1:500, Thermo Fisher Scientific, A21245) for 1 h. Finally, cells were stained with DAPI. Elution and acquisition of images after elution was repeated as above, followed by sBS blocking for 1 h and washing. Then cells were stained with anti-p21 (1:500, rabbit, Abcam, ab109520) and anti-p53 (1:500, mouse, Thermo Fisher Scientific, AHO0152) for 2 h. After two PBS washes, cells were incubated with anti-mouse A488 (1:500, Thermo Fisher Scientific, A11029) and anti-rabbit A647 (1:500, Thermo Fisher Scientific, A21245) for 1 h. For every staining or elution round, imaging buffer (700 mM N-acetyl-cysteine (Sigma-Aldrich) in double-distilled H2O, pH 7.4) was added before imaging. Imaging was performed using the automated widefield GE InCell Analyzer 2500HS high-content screening, which was also used for live-cell imaging (seven-colour SSI, PCO-sCMOS camera system (16 bit, 2,048 × 2,048 pixel, pixel size: 6.5 × 6.5 μm, readout speed: 272 MHz), two quad band-pass polychroic mirrors, single band-pass emission filters). The polychroic beam splitter BGOFR_1 (blue (excitation BP 390/22, emission BP 435/48), green (excitation BP 473/28, emission BP 511/23), orange (excitation BP 542/30, emission BP 597/45), far red (excitation BP 631/28, emission BP 684/24)) and a CFI Plan Apo Lambda ×20 Air objective (NA 0.75, WD 1.0 mm) were used, with hardware laser autofocus for acquisition of single plane images. No binning was performed. Nine fields per well were acquired, with typically 2–3 wells per experimental condition and at least 3,000 cells seeded per well. For all imaging and staining rounds the acquisition settings were kept constant. Images from the live imaging were processed using a time-lapse script (https://github.com/AltmeyerLab/SingleCellTracking_Timelapse), which involves generation of .tif stacks and generation of Olympus ScanR Analysis (v.3.0.1, 3.2 and 3.3.0) compatible images. This was followed by custom conversion in Olympus ScanR Analysis software (v.3.0.1, 3.2 and 3.3.0) to generate the required metadata and the experiment file. Segmentation of cells was performed on a smoothened mask of the PCNA channel. Parameters for total and mean intensities of the two channels, nuclear area (1 pixel ≅ 0.1056 µm2), circularity factor, perimeter, elongation factor, foci counts for PCNA and 53BP1, integrated intensities of foci in each channel, time slices, well number, position, centre x coordinate and centre y coordinate were extracted and tabulated for each cell in every timeframe. For alignment of 4i imaging data, an alignment script was used (https://github.com/AltmeyerLab/SingleCellTracking_Multiplex-Alignment), which renders images from multiple rounds of staining compatible with downstream analysis using the Olympus ScanR Analysis software (v.3.0.1, 3.2 and 3.3.0). Custom conversion and image analysis was performed with the Olympus ScanR Analysis software (v.3.0.1, 3.2 and 3.3.0). In addition to the parameters outlined above, the mean and total intensities for all stainings as well as γH2AX foci counts and integrated intensities of γH2AX foci were extracted and tabulated for each cell. Tables with data from a field of view from the post-fixation 4i acquisition were appended to data from the live-cell imaging for the same field of view. The frame numbers and x–y coordinates were used as the basis for a previously published MATLAB (MathWorks, MATLAB R2019b, R2020b, R2023a) script to automatically generate live-cell tracks for each cell. The same script along with the ImageJ/Fiji 64-bit (v.1.53f, 1.53t, 1.54f, 1.54m) plugin ‘Manual Tracking' was used together with the corresponding image stacks to visualize, correct and manually reassign cells for lineage-based analysis. After reassignment, MATLAB scripts (both scripts are available at GitHub; https://github.com/AltmeyerLab/MatlabTracking/; script 1, Cell_tracking.m; and script 2, KeyDataExtractions.m) were used to generate lineage assignments and data from curated and reassigned cells was written into .csv files and loaded into TIBCO Spotfire (v.7.9.1, 10.10.1) for visualization of the lineage trees together with the post-fixation 4i data. For U-2 OS cells, at least 20 individual cell lineages per condition, corresponding to up to 80 granddaughter cells, were tracked. For RPE-1 cells, at least ten individual cell lineages per condition, corresponding to up to 40 granddaughter cells, were tracked. For all extended live-cell imaging experiments with lineage tracking and multiplex staining (Live+QIBC), at least two independent biological replicate experiments were performed. In some instances, although the median and the complete Q1–Q3 range are shown for whole-population box plots associated with Live+QIBC results, not all individual values are displayed to allow for visualization of population effects with consistently scaled axes. After validating that PCNA foci were mostly confined to EdU-positive S-phase cells, thresholds were established for S-phase categorization as well as for G1–S and S–G2 transitions based on live-cell tracking of untreated control cells. In brief, unchallenged cells transitioned from G1 to S phase quickly (typically within a 30-min interval), indicated by a marked increase in PCNA foci (≥10 foci). Likewise, unchallenged cells transitioned from S phase to G2 quickly (typically within 1–2 30-min intervals), indicated by a marked decrease in PCNA foci (≤4 foci). Thresholds were therefore defined as follows: 0–4 PCNA foci for G1; ≥10 PCNA foci for ≥2 consecutive timepoints for S phase (G1–S transition); ≤4 PCNA foci for ≥2 consecutive timepoints for G2 phase (S–G2 transition). For the other markers, heterogeneity was evaluated from multiplexed end-point measurements. Three categories of sister cell heterogeneity were introduced based on the differences in foci counts or nuclear intensity levels; thresholds were dependent on the individual experiment: for Extended Data Fig. 5e, 53BP1 foci heterogeneity was categorized as follows: low (0–1 focus difference), medium (2–4 foci difference), high (>4 foci difference). γH2AX fluorescence intensity was categorized as follows: low (0–100 a.u. p53 fluorescence intensity was categorized as follows: low (0–50 a.u. p21 fluorescence intensity was categorized as follows: low (0–100 a.u. For Extended Data Fig. 8i, 53BP1 foci heterogeneity was categorized as follows: low (0–1 focus difference), medium (2–4 foci difference), high (>4 foci difference). For γH2AX fluorescence intensity, heterogeneity was categorized as follows: low (0–50 a.u. For p53 fluorescence intensity, heterogeneity was categorized as follows: low (0–200 a.u. For pRb fluorescence intensity, heterogeneity was categorized as follows: low (0–500 a.u. 8b, 53BP1 foci heterogeneity was categorized as follows: low (0–1 focus difference), medium (2–4 foci difference), high (>4 foci difference). Automated multichannel widefield microscopy for high-content imaging and QIBC was performed using the Olympus ScanR High-Content Screening System as described previously33,64. The system is equipped with an inverted motorized Olympus IX83 microscope, a motorized stage, IR-laser hardware autofocus, a fast emission filter wheel with one set of band-pass filters for multi-wavelength acquisition (DAPI (excitation BP 395/25 or BP 390/22, emission BP 435/26), FITC (excitation BP 470/24 or BP 475/28, emission BP 515/30), TRITC (excitation BP 550/15 or BP 555/28, emission BP 595/40), Cy5 (excitation BP 640/30 or BP 635/22, emission BP 705/72)), and a Hamamatsu ORCA-FLASH 4.0 V2 sCMOS camera (12 bit, 2,048 × 2,048 pixel, pixel size 6.5 × 6.5 μm) with a ×40 UPLSAPO (NA 0.9, WD 0.18 mm), a ×20 UPLSAPO (NA 0.75, WD 0.6 mm) and a ×10 UPLSAPO (NA 0.4, WD 3.1 mm) air objective. Images of cell populations were acquired under non-saturating conditions (Olympus ScanR Image Acquisition software (v.3.0.1, 3.2 and 3.3.0)), typically 25 (5 × 5) to 81 (9 × 9) images per well, depending on the objective and cell density, and identical settings were applied to all samples within one experiment. Hardware and software autofocus on the DAPI channel were used. No binning was performed. Images were analysed using the inbuilt Olympus ScanR Analysis software (v.3.0.1, 3.2 and 3.3.0), a dynamic background correction was applied and nucleus segmentation was performed using an integrated intensity-based object detection module based on the DAPI signal. Downstream analyses were focused on properly detected nuclei containing a 2N–4N DNA content as measured by total and mean DAPI intensities, unless increased ploidy in cells with >4N DNA content was also analysed. Fluorescence intensities were quantified and are depicted as arbitrary units. Colour-coded scatterplots of asynchronous cell populations were generated with TIBCO Spotfire (v.7.9.1, 10.10.1). Within one experiment, similar cell numbers were compared for the different conditions. For visualizing discrete data in scatterplots, mild jittering (random displacement of data points along discrete data axes) was applied to demerge overlapping datapoints. Representative scatterplots and quantifications of independent experiments, typically containing several thousand cells each, are shown. A Leica THUNDER (Las X 3.7.6.25997) Imager 3D Live Cell system was used for DNA fibre analysis. Dual-labelled DNA fibres (Alexa Fluor 488 and 555) were imaged with the 475 nm and 555 nm LEDs, the DFT5 Quad filter (excitation filters: 375–407, 462–496, 542–566, 622–654; main beam splitter: 415, 500, 572, 660; emission filters: 420–450, 506–532, 581–607, 666–724), additional emission filters 535/70 and 642/80 from an external clean-up filter wheel, and a Leica HC PL APO CS2 (NA 1.4, WD 0.14 mm) ×63 oil objective. For standard immunofluorescence staining, high-content microscopy and QIBC analyses, cells were seeded on sterile 12 mm glass coverslips or 96-well plates and were allowed to proliferate until they reached a cell density of 70–90%. Cells were then fixed in 3% formaldehyde (Sigma-Aldrich) for 15 min at room temperature, washed once in PBS, permeabilized for 5 min at room temperature in 0.2% Triton X-100 (Sigma-Aldrich) in PBS, washed twice in PBS and incubated in blocking solution (filtered DMEM containing 10% FBS (Corning) and 0.02% sodium azide (Merck)) for 15 min at room temperature. When the staining was combined with an EdU Click-iT reaction, the reaction was performed before the incubation with the primary antibody according to manufacturer's recommendations (Thermo Fisher Scientific). Where indicated, cells were pre-extracted in 0.2% Triton X-100 (Sigma-Aldrich) in PBS for 2 min on ice before formaldehyde fixation. Denaturing, where indicated, was performed in 2.5 M HCl for 10 min. All primary antibodies were diluted in blocking solution and incubated for 2 h at room temperature. Secondary antibodies (Alexa Fluor 488, 555, 568, 647 anti-mouse, anti-rabbit, anti-goat and anti-rat IgG, Thermo Fisher Scientific) were diluted 1:500 in blocking solution and incubated at room temperature for 1 h. Cells were washed once with PBS and incubated for 10 min with DAPI (0.5 mg ml−1) in PBS at room temperature. After three washing steps in PBS, the coverslips were briefly washed with distilled water and mounted onto 6 µl Mowiol-based mounting medium (Mowiol 4.88 (Calbiochem) in glycerol/Tris), whereas the wells of the 96-well plates were kept filled with PBS. Proteins were separated by standard SDS–PAGE and transferred onto PVDF membranes. Membranes were blocked with 5% milk in PBS-T (PBS + 0.1% Tween-20) for 1 h at room temperature and incubated with primary antibodies over night at 4 °C. The membranes were then washed three times with PBS-T and incubated with HRP-conjugated secondary antibodies for 1 h at room temperature, washed again three times with PBS-T and protein signals were detected using ECL Western Blotting Detection Reagent (Thermo Fisher Scientific) and an OPTIMAX X-Ray Film Processor (PROTEC Medizintechnik). To enable detection of multiple target proteins on the same membrane without stripping (typically 2–3 target proteins per membrane with sufficiently distinct molecular weight), membranes were cut horizontally, each piece was probed with specific primary and secondary antibodies, and the membrane was then reassembled for ECL detection. Original western blot scans are provided in Supplementary Fig. Cells transfected with the indicated siRNAs were seeded at single-cell density and exposed to the indicated drugs at the indicated final concentrations. All conditions were performed in triplicates. Cells were then incubated for 10 days and the number of colonies with more than 50 cells was counted after staining with crystal violet (0.5% crystal violet (Sigma-Aldrich) in 20% ethanol). GraphPad Prism (v.9 and 10) was used to display clonogenic survival data. For SORT-seq, cells were sorted on a FACS Aria III 5L (FACS Diva Software v.8.0.1) system equipped with a 488 nm laser line. For the polyploidy dataset, pevonedistat-treated cells were treated with 175 nM of pevonedistat for 24 h. Before single-cell sorting, all live cells were incubated for 1 h with a final concentration of 5 µg ml−1 Hoechst 33342 (Thermo Fisher Scientific). All reagents used until the moment of sorting also contained 5 µg ml−1 Hoechst 33342 (Thermo Fisher Scientific). For the scSeq dataset, where IR-treated cells were compared to untreated cells, U-2 OS cells were treated with 4 Gy of IR 48 h before sorting or left untreated. Cells were sorted into 384-well plates that were acquired from Single Cell Discoveries (Bio-Rad, HSP3801), each well containing 10 µl sterile mineral oil (Sigma-Aldrich, M5310) and 50 nl DNA oligo primer (Sigma-Aldrich, M8410). Cells with increased ploidy were gated according to their Hoechst signal and, as a reference for the whole population, the gate was adjusted in the same sample. After sorting, plates were immediately centrifuged and placed on dry ice. They were stored at -80 °C and were shipped on dry ice to Single Cell Discoveries, where scRNA-seq was performed according to an adapted version of the SORT-seq protocol65 with primers described previously66. Cells were heat-lysed at 65 °C followed by cDNA synthesis. After second-strand cDNA synthesis, all of the barcoded material from one plate was pooled into one library and amplified using in vitro transcription (IVT). After amplification, library preparation was performed according to the CEL-Seq2 protocol67 to prepare a cDNA library for sequencing using TruSeq small RNA primers (Illumina). The DNA library was paired-end sequenced on the Illumina NextSeq 500 system, high output, with the 1 × 75 bp Illumina kit (read 1: 26 cycles, index read: 6 cycles, read 2: 60 cycles). During sequencing, read 1 was assigned 26 bp and was used to identify the Illumina library barcode, cell barcode and UMI. Read 2 was assigned 60 bp and was used to map to the reference genome Homo sapiens GRCh38.p13 with STARSolo (v.2.7.10b)68. In brief, mapping and generation of count tables were automated using the STARSolo v.2.7.10b and aligner. For mapping, no UMI cut-off was used and intronic reads were not included. Multi-gene reads were not counted. Unsupervised clustering and differential gene expression analysis was performed with the Seurat R toolkit69. For this, logarithmic normalization was applied, which normalizes to the total RNA counts in each cell. This approach consists of dividing each raw UMI count by the total detected RNAs in that cell, multiplying by a scale factor (10,000), adding a pseudocount (1) and performing log transformation. For visualization of the t-SNE, the clustering algorithm used was the original Louvain algorithm where the first 19 principal components were used with a k parameter of 30 and a resolution of 0.5. Subclustering of the polyploid samples was performed with Louvain resolution of 0.5 for both samples with 50 principal components as identified by principal component analysis. Differential gene expression analysis for all comparisons was done using the Venice method, a nonparametric statistical test for single-cell data70 in BioTuring BBrowser X software (BioTuring). Tables were extracted and plotted in TIBCO Spotfire (v.7.9.1, v.10.10.1). For the AUCellScores the integrated tool in BioTuring BBrowser X was used. For GO-analysis, ShinyGO (v.0.77 and v.0.80) available online (https://bioinformatics.sdstate.edu/go/)71 was used. For generation of Venn diagrams, the tool InteractiVenn (https://www.interactivenn.net)72 was used. The data analysis for the replicate samples was done as described above, except that a different version of the mapper STARSolo (v.2.7.11b)68 was used. For subclustering of the polyploid cells of the replicate sample, a Louvain resolution of 0.5 was used for the HRAS polyploid sample and a Louvain resolution of 1.0 was used for the Pevo polyploid sample. Sequencing raw data and processed Poisson-corrected data files for both replicates of the polyploidy data set are accessible on GEO (GSE255874). Cell clustering analysis of the IR-treated samples was performed with the Seurat R Bioconductor package69, using the SC transformed counts generated using the ‘v2' vst flavour, with Louvain resolution of 0.6 and the first 20 principal components as identified by principal component analysis, for both samples. Dimension reduction was performed using the UMAP method, using the first 20 principal components. Differential expression was performed between treated versus untreated cells using the FindMarkers() function, using the option to return only positive genes. Enriched pathways per cluster were generated using the enrichGO function of the clusterProfiler Bioconductor R package73 or the Enrichr gene list enrichment analysis tool74, using the marker genes identified per cluster from the FindMarkers() function. All R functions were executed on R v.4.4.2 (https://www.R-project.org) and Bioconductor v.3.20. All scRNA-seq and SORT-seq analyses were performed on two independent sets of samples. Sequencing raw data and processed data files for the IR-treated versus UT dataset are accessible on GEO (GSE288487). Cells were treated for 24 h with 4 Gy of IR or left untreated. RNA was extracted from triplicate samples using the TRIzol RNA MiniPrep Plus Kit according to the manufacturer's protocol. Extracted RNA was prepared for sequencing by the Functional Genomics Center Zurich (FGCZ) using the Illumina TruSeq Total RNA Library Prep assay according to the manufacturer's protocol. Sequencing was performed on the Illumina NovaSeq 6000 system using the S1 Reagent Kit v1.5 (100 cycles) according to the manufacturer's protocol. Demultiplexing was performed using the Illumina bcl2fastq Conversion Software (v.2.20.0.422). RNA-seq analysis was performed using the SUSHI framework75, which encompassed the following steps: read quality was inspected using FastQC, and sequencing adaptors were removed using fastp76; pseudoalignment and transcriptomic counts of the RNA-seq reads was performed using the Kallisto Bioconductor R package77 with the GENCODE human genome build GRCh38.p13 (release 37)78; differential expression using the generalized linear model as implemented by the edgeR Bioconductor R package79; and Gene Ontology (GO) term pathway analysis using the hypergeometric over-representation test with the enrichGO function of the clusterProfiler Bioconductor R package73 or the Enrichr gene list enrichment analysis tool74. Additional figures were generated using the exploreDE Shiny app (https://doi.org/10.5281/zenodo.13927692). All R functions were executed on R v.4.4.2 (https://www.R-project.org) and Bioconductor v.3.20. Bulk sequencing data files are accessible on GEO (GSE288485). Alexa Fluor 647 goat anti-rabbit (Thermo Fisher Scientific, A21244, 1:500 for IF), Alexa Fluor 647 goat anti-mouse (Thermo Fisher Scientific, A21235, 1:500 for IF), Alexa Fluor 568 goat anti-rabbit (Thermo Fisher Scientific, A11036, 1:500 for IF), Alexa Fluor 568 goat anti-mouse (Thermo Fisher Scientific, A11031, 1:500 for IF), Alexa Fluor 555 goat anti-rat (Thermo Fisher Scientific, A21434, 1:250 for IF), Alexa Fluor 488 goat anti-rabbit (Thermo Fisher Scientific, A11034, 1:500 for IF), Alexa Fluor 488 goat anti-mouse (Thermo Fisher Scientific, A11029, 1:500 for IF), Alexa Fluor 488 rabbit anti-goat (Thermo Fisher Scientific, A11078, 1:500 for IF), goat anti-rabbit IgG antibody (H+L), peroxidase (Vector Laboratories, PI-1000-1, 1:10,000 for WB), goat anti-mouse IgG Antibody (H+L), peroxidase (Vector Laboratories, PI-2000-1, 1:10,000 for WB). Apart from time-lapse microscopy, no samples were measured repeatedly, and all other measurements were taken from distinct samples. Statistical analysis was performed in GraphPad Prism (v.9 and 10). Two-tailed unpaired t-tests, χ2 tests, Fisher's exact tests or one-way ANOVA followed by Tukey's post hoc test were performed as indicated in the figure legends. All extended time-lapse experiments were performed at least twice for each experimental condition (that is, independent biological replicates with different batches of cells and the live-cell microscopy performed in different weeks), each experiment with 1–5 wells per experimental condition (typically 2–3 wells per condition) and multiple images taken per well (typically 9 images per well). Bulk RNA-seq was performed in triplicates, scRNA-seq was performed in duplicates. Control western blots (Supplementary Figs. 3a,b and 6b) were performed once, control DNA fibre experiments (Extended Data Figs. 9c and 10c) were performed twice. All of the other experiments were performed at least three times, and the presented results were reliably reproduced. Sample numbers, numbers of experiments performed, statistical tests, exact P values, and definition of error bars and box plots are as follows: Fig. 1c, 3 independent experiments performed, n = 2,955 (parental) and n = 3,018 (edited) cells are shown. 1d, 3 independent experiments performed, representative images are shown. 1e, 3 independent experiments performed, representative images are shown. 1i, 3 independent experiments performed, n = 24 cell lineages shown. 1j, 3 independent experiments performed, n = 24 lineages shown. 2a, 3 independent live-cell experiments performed, n = 51 cells analysed; the box plot limits indicate the 25th percentile (Q1) and 75th percentile (Q3); the boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 2c, 3 independent experiments performed, n = 51 (UT), n = 43 (APH) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 2d, 3 independent experiments performed, n = 51 (UT), n = 38 (ATRi) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 3a, P values were derived from a differential gene expression test using the generalized linear model as implemented by edgeR Bioconductor R package with Benjamini–Hochberg multiple test correction, n = 14,664 unique genes. 3b, P value (two-sided Fisher's exact test on genes with residuals of >0.5 versus genes with residuals of ≤0.5): 2.45 × 10–12, odds ratio = 1.90, confidence interval (CI): 1.58 to 2.299, n = 9,797 unique genes. 3c, P values were derived through Enrichr by one-sided right-tailed Fisher's test with Benjamini–Hochberg multiple-test correction. 3f, 3 independent experiments performed, n = 855 (UT), n = 1,011 (ATRi), n = 1,205 (IR), n = 965 (ATRi→IR), n = 273 (UT in G1), n = 598 (ATRi in G1), n = 404 (IR in G1), n = 557 (ATRi→IR in G1) cells are shown; P values and CIs γH2AX foci (one-way ANOVA followed by Tukey's post hoc test): P = 0.0012 (CI: −3.886 to −0.7074) for UT versus ATRi, P < 0.0001 (CI: −5.710 to −2.865) for IR versus ATRi→IR; P values and CIs 53BP1 foci (one-way ANOVA followed by Tukey's post hoc test): P = 0.0072 (CI: 0.2131 to 1.908) for IR versus ATRi→IR; the solid line indicates the mean and the dashed lines indicate the s.d. 4b, 3 independent live-cell experiments performed, n = 557 (G1 peak, control), n = 170 (≥G2 peak, control), n = 221 (G1 peak, pevonedistat), n = 282 (≥G2 peak, pevonedistat) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 4c, 3 independent live-cell experiments performed, n = 557 (G1 peak, control), n = 170 (≥G2 peak, control), n = 221 (G1 peak, pevonedistat), n = 282 (≥G2 peak, pevonedistat) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 4d, 3 independent live-cell experiments performed, n = 557 (G1 peak, control), n = 170 (≥G2 peak, control), n = 221 (G1 peak, pevonedistat), n = 282 (≥G2 peak, pevonedistat) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 4e, 3 independent live-cell experiments performed, representative examples are shown. 4f, 3 independent live-cell experiments performed, representative images are shown. 4g, 2 independent live-cell experiments performed, n = 61 (normal), n = 94 (endoreplication), n = 84 (rereplication) cells analysed; where the solid line indicates the mean and the dashed lines indicate the s.d. 4h, 2 independent live-cell experiments performed, n = 61 (normal), n = 94 (endoreplication), n = 84 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d., P values and CI (two-tailed unpaired t-test): P < 0.0001 (CI: 423.0 to 673.3). 4i, 2 independent live-cell experiments performed, n = 61 (normal), n = 94 (endoreplication), n = 84 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d. ; P values and CI (two-tailed unpaired t-test): P < 0.0001 (CI: 186.4 to 372.6). 4j, 2 independent live-cell experiments performed, n = 61 (normal), n = 94 (endoreplication), n = 84 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d. ; P values and CI (two-tailed unpaired t-test): P < 0.0001 (CI: 738.3 to 2,175). 4k, 2 independent live-cell experiments performed, n = 61 (normal), n = 94 (endoreplication), n = 84 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d., P values and CI (two-tailed unpaired t-test): P = 0.0022 (CI: 536.2 to 2,404). 5a, 2 independent live-cell experiments performed, n = 88 (EV), n = 88 (HRAS), n = 489 (G1 peak, EV), n = 191 (≥G2 peak, EV), n = 411 (G1 peak, HRAS), n = 216 (≥G2 peak, HRAS) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5g, 2 independent live-cell experiments performed, n = 20 and 15 (pevonedistat), n = 4 and 5 (etoposide), n = 20 and 20 (RO3306) cells undergoing polyploidization analysed, where solid lines depict the mean and dots represent the cumulative percentage of each experiment. 1a, 2 independent live-cell experiments performed, representative images are shown. 1g, 2 independent live-cell experiments performed, n = 2,038 (imaged), n = 2,036 (not imaged) cells shown. 1h, 2 independent live-cell experiments performed, n = 2,038 (imaged), n = 2,036 (Not imaged) cells shown. 1i, 2 independent live-cell experiments performed, n = 2,051 (imaged), n = 2,034 (not imaged) cells shown. 2b, 3 independent experiments performed, n = 4,833 (siCtrl UT), n = 4,486 (siCtrl IR), n = 4,510 (si53BP1 UT), n = 4,407 (si53BP1 IR) cells analysed; where the solid line indicates the mean and the dashed lines indicate the s.d. ; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 (CI: −0.4840 to −0.4030) for UT siCtrl versus IR siCtrl, P < 0.0001 (CI: 0.5565 to 0.6393) for IR siCtrl versus IR si53BP1. 2d, 3 independent experiments performed, n = 4,453 cells shown. 2e, 3 independent experiments performed, n = 945 (G1), n = 2,963 (S), n = 545 (G2/M) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 (CI: −39.23 to −34.93) for G1 versus S, P < 0.0001 (CI: 31.86 to 37.23) for S versus G2/M; box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 3b, 3 independent live-cell experiments performed, n = 23 lineages shown. 3d, 3 independent experiments performed, representative results are shown. 3e, 3 independent live-cell experiments performed, n = 26 lineages shown. 3g, 3 independent live-cell experiments performed, n = 27 lineages shown. 4b, 3 independent experiments performed, representative images are shown. 4c, 3 independent experiments performed, n = 1,659 (control), n = 1,324 (replication stress) cells shown. 4d, 3 independent experiments performed, n = 1,659 (control), n = 1,324 (replication stress) cells shown. 4e, 3 independent experiments performed, n = 1,659 (control), n = 1,324 (replication stress) cells shown. 5a, 3 independent live-cell experiments performed, n = 51 (UT), n = 43 (APH) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5b, 3 independent live-cell experiments performed, n = 51 (UT), n = 38 (ATRi) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5c, 3 independent live-cell experiments performed, n = 51 (UT), n = 43 (APH) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5d, 3 independent live-cell experiments performed, n = 51 (UT), n = 38 (ATRi) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5e, 3 independent live-cell experiments performed, n = 29 (53BP1 foci) and n = 27 (γH2AX, p53, p21) sister cell pairs for UT, n = 49 (53BP1 foci), n = 33 (γH2AX), and n = 31 (p53, p21) sister cell pairs for APH, n = 36 (53BP1 foci) and n = 31 (γH2AX, p53, p21) sister cell pairs for ATRi; P values (χ2 test): P = 0.0004 for UT versus ATRi (53BP1 foci), P = 0.0012 for UT versus ATRi (γH2AX levels), P = 0.0284 for UT versus APH (p53 levels), P = 0.0031 for UT versus ATRi (p53 levels), P = 0.0131 for UT versus APH (p21 levels), P < 0.0001 for UT versus ATRi (p21 levels). 6b, 3 independent experiments performed, n = 3,592 (siCtrl UT), n = 4,824 (siCtrl IR), n = 3,267 (si53BP1 UT), n = 4,662 (si53BP1 IR) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 for UT siCtrl versus IR siCtrl (CI: −1.431 to −1.288), P < 0.0001 for IR siCtrl versus IR si53BP1 (CI: 1.413 to 1.547); the solid line indicates the mean and the dashed lines indicate the s.d. 6d, 3 independent experiments performed, n = 4,083 cells shown. 6e, 3 independent experiments performed, n = 1,972 (G1), n = 1,759 (S), n = 352 (G2) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 for G1 versus S (CI: −38.77 to −36.22), P < 0.0001 for S versus G2 (CI: 31.77 to 36.31); box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 6j, 3 independent experiments performed, representative results are shown. 7b, P values were derived through Enrichr using a one-sided right-tailed Fisher's test with Benjamini–Hochberg multiple-test correction. 7c, P values were derived from differential gene expression test using the generalized linear model as implemented by the edgeR Bioconductor R package with Benjamini–Hochberg multiple-test correction, boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 7h, P values were derived by one-sided right-tailed Fisher's test with Benjamini–Hochberg multiple-test correction through clusterProfiler. 7i, Pearson's correlation coefficient, n = 9,279 all genes, n = 418 DDR genes. 8a, 3 independent live-cell experiments performed. 8b, 3 independent live-cell experiments performed. 8c, 3 independent experiments performed, n = 423 (control), n = 158 (IR) cells shown; P values and CI (two-tailed unpaired t-test): P < 0.0001 (CI: 3.246 to 3.978); the solid line indicates the mean and the dashed lines indicate the s.d. 8d, 3 independent experiments performed, n = 4,049 (UT), n = 4,079 (4-OHT), n = 4,377 (4-OHT-IAA) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 for UT versus 4-OHT (CI: −2.414 to −2.093), P < 0.0001 for 4-OHT versus 4-OHT-IAA (CI: 1.669 to 1.984); the solid line indicates the mean and the dashed lines indicate the s.d. 8e, 3 independent experiments performed, n = 4,049 (UT), n = 4,079 (4-OHT), n = 4,377 (4-OHT-IAA) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 for UT versus 4-OHT (CI: −6.751 to −5.977), P < 0.0001 for 4-OHT versus 4-OHT-IAA (CI: 2.990 to 3.749); the solid line indicates the mean and the dashed lines indicate the s.d. 8f, 2 independent live-cell experiments performed. 8g, 3 independent experiments performed, n = 621 (−4-OHT), n = 639 (+4-OHT + IAA) cells shown; P values and CI (two-tailed unpaired t-test): P = 0.0003 (CI: 0.1963 to 0.6682); the solid line indicates the mean and the dashed lines indicate the s.d. 8h, 3 independent experiments performed, n = 2,428 (WT control), n = 614 (WT IR), n = 2,406 (p53KO control), n = 2,106 (p53-KO IR) cells shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 for WT control versus WT IR (CI: −0.5428 to −0.2159), P < 0.0001 for p53KO control versus p53KO IR (CI: −0.9969 to −0.7810); the solid line indicates the mean and the dashed lines indicate the s.d. 8i, 2 independent live-cell experiments performed, n = 32 (53BP1 foci) and n = 52 (γH2AX, p53, pRb) sister cell pairs for UT; n = 54 (53BP1 foci) and n = 68 (γH2AX, p53, pRb) sister cell pairs for 4 Gy; P values (χ2 test): P = 0.0088 for UT versus 4 Gy (53BP1 foci), P = 0.0080 for UT versus 4 Gy (γH2AX levels), P = 0.0397 for UT versus 4 Gy (p53 levels), P = 0.0176 for UT versus 4 Gy (pRb levels). 8j, 2 independent live-cell experiments performed. 9b, 3 independent experiments performed, representative images shown. 9c, 2 independent experiments performed, DNA fibre length from n = 100 (control), n = 100 (pevonedistat) fibres shown; P values and CI (two-tailed unpaired t-test): P < 0.0001 (CI: −11.40 to −9.096); the red solid line indicates the median. 9f, 3 independent live-cell experiments performed, representative examples are shown. 9g, 3 independent live-cell experiments performed, representative examples are shown. 9i, 2 independent live-cell experiments performed, representative examples are shown. 9j, 2 independent live-cell experiments performed, representative examples are shown. 9l, 2 independent live-cell experiments performed, n = 43 (normal), n = 15 (endoreplication), n = 37 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d. 9m, 2 independent live-cell experiments performed, n = 43 (normal), n = 15 (endoreplication), n = 37 (rereplication) cells analysed; P values and CI (two-tailed unpaired t-test): P = 0.0256 (CI: 6.720 to 98.88) for endoreplication versus rereplication; the solid line indicates the mean and the dashed lines indicate the s.d. 9n, 2 independent live-cell experiments performed, n = 43 (normal), n = 15 (endoreplication), n = 37 (rereplication) cells analysed; P values and CI (two-tailed unpaired t-test): P = 0.0214 (CI: 4.769 to 56.93) for endoreplication versus rereplication; the solid line indicates the mean and the dashed lines indicate the s.d. 9o, 2 independent live-cell experiments performed, n = 43 (normal), n = 15 (endoreplication), n = 37 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d. 9p, 2 independent live-cell experiments performed, n = 43 (normal), n = 15 (endoreplication), n = 37 (rereplication) cells analysed; the solid line indicates the mean and the dashed lines indicate the s.d. 10a, 3 independent experiments performed, representative results are shown. 10b, 3 independent experiments performed, representative results are shown. 10c, 2 independent experiments performed, DNA fibre length of n = 120 (EV), n = 120 (HRAS), n = 120 (cyclin E) fibres shown; P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 (CI: 1.699 to 5.797) for EV versus HRAS, P = 0.0003 (CI: 1.399 to 5.497) for EV versus cyclin E; the red solid line indicates the median. 10d, 4 independent experiments performed, n = 100 cells per replicate (EV), n = 100 cells per replicate (HRAS), n = 100 cells per replicate (cyclin E); P values and CIs (one-way ANOVA followed by Tukey's post hoc test): P = 0.0234 (CI: −25.86 to −3.645) for EV versus HRAS, P = 0.0078 (CI: −17.75 to −5.749) for EV versus cyclin E; data are mean ± s.d. 10e, 2 independent live-cell experiments performed, n = 88 (EV), n = 489 (G1 peak, EV), n = 191 (≥G2 peak, EV) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 10f, 2 independent live-cell experiments performed, n = 88 (EV), n = 88 (HRAS), n = 489 (G1 peak, EV), n = 191 (≥G2 peak, EV), n = 411 (G1 peak, HRAS), n = 216 (≥G2 peak, HRAS) cells analysed; box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 10g, 2 independent live-cell experiments performed, n = 88 (EV), n = 92 (cyclin E), n = 489 (G1 peak, EV), n = 191 (≥G2 peak, EV), n = 548 (G1 peak, cyclin E), n = 248 (≥G2 peak, cyclin E) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 10h, 2 independent live-cell experiments performed, n = 88 (EV), n = 92 (cyclin E), n = 489 (G1 peak, EV), n = 191 (≥G2 peak, EV), n = 548 (G1, cyclin E), n = 248 (≥G2 peak, cyclin E) cells analysed; box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 11a, 3 independent experiments performed, n = 3,097 (empty vector UT), n = 3,026 (HRAS UT), n = 3,144 (cyclin E UT), 3,082 (empty vector IR), n = 3,149 (HRAS IR), n = 3,121 (cyclin E IR) cells shown. 11b, 3 independent experiments performed, representative images are shown. 11f, 3 independent live-cell experiments performed, representative examples are shown. 11g, 2 independent experiments performed, representative results are shown. 12a, 2 independent live-cell experiments performed, n = 12 (−Dox), n = 414 (G1 peak, −Dox), n = 271 (≥G2 peak, −Dox) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 12b, 2 independent live-cell experiments performed, n = 12 (−Dox), n = 36 (+Dox), n = 414 (G1 peak, −Dox), n = 271 (≥G2 peak, −Dox), n = 286 (G1 peak, +Dox), n = 259 (≥G2 peak, +Dox) cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 12c, 2 independent live-cell experiments performed, n = 174 (experiment 1), n = 231 (experiment 2), mean (solid lines) and individual percentage values are depicted. 2d, 3 independent experiments performed, representative images are shown. 2e, 3 independent live-cell experiments performed, n = 10 lineages per condition; P values and CIs for perturbed replication (one-way ANOVA followed by Tukey's post hoc test): P < 0.0001 (CI: −2.148 to −0.6516) for UT versus APH, P < 0.0001 (CI: −2.648 to −1.152) for UT versus ATRi, P < 0.0001 (CI: −2.148 to −0.6516) for UT versus IR; P values and CIs for >10 53BP1 foci (one-way ANOVA followed by Tukey's post hoc test): P = 0.0003 (CI: −1.582 to −0.4182) for UT versus IR; P values and CIs for number of divisions (one-way ANOVA followed by Tukey's post hoc test): P = 0.0002 (CI: 0.6531 to 2.347) for UT versus APH, P < 0.0001 (CI: 0.7531 to 2.447) for UT versus ATRi, P = 0.0011 (CI: 0.4531 to 2.147) for UT versus IR. 3c, 2 independent live-cell experiments performed; an example cell lineage is shown. 3d, 2 independent live-cell experiments performed; an example cell lineage is shown. 3e, 2 independent live-cell experiments performed; an example cell lineage is shown. 3f, 2 independent live-cell experiments performed; an example cell lineage is shown. 3g, 2 independent live-cell experiments performed; an example cell lineage is shown. 3h, 2 independent live-cell experiments performed; an example cell lineage is shown. 3i, 2 independent live-cell experiments performed; an example cell lineage is shown. 3j, 2 independent live-cell experiments performed; an example cell lineage is shown. 3k, 2 independent live-cell experiments performed; an example cell lineage is shown. 4a, 2 independent live-cell experiments performed, n = 21 lineages shown. 4b, 2 independent live-cell experiments performed, n = 17 lineages shown. 4c, 2 independent live-cell experiments performed, n = 20 lineages shown. 4d, 2 independent live-cell experiments performed, n = 20 lineages shown. 4e, 2 independent live-cell experiments performed, n = 20 lineages shown. 4f, 2 independent live-cell experiments performed, n = 20 lineages shown. 4g, 2 independent live-cell experiments performed, n = 20 lineages shown. 4h, 2 independent live-cell experiments performed, n = 20 lineages shown. 4i, 2 independent live-cell experiments performed, n = 20 lineages shown. 5a, 2 independent live-cell experiments performed, n = 27 cells analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5b, 2 independent live-cell experiments performed, n = 27 cells (siCtrl), n = 47 cells (siAMBRA1) analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5c, 2 independent live-cell experiments performed, n = 27 cells (siCtrl), n = 47 cells (siAMBRA1), n = 38 cells (siAMBRA1 + APH) analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 5d, 2 independent live-cell experiments performed, n = 27 cells (siCtrl), n = 47 cells (siAMBRA1), n = 25 cells (siAMBRA1 + ATRi) analysed; the box plot limits indicate Q1 and Q3; boxes represent the IQR with the median value (solid lines); the whiskers define the lower and upper adjacent value; dots show outliers smaller than Q1 − 1.5 × IQR and greater than Q3 + 1.5 × IQR. 6a, 3 independent experiments performed; P values and CI (two-tailed unpaired t-test): P = 0.0006 (CI: 17.60 to 31.07) for 0.1 μM ATRi, P = 0.0114 (CI: 1.740 to 7.593) for 0.2 μM ATRi, P = 0.0111 (CI: 1.264 to 5.403) for 0.3 μM ATRi; data are mean ± s.d. 7b, 2 independent live-cell experiments performed, an example lineage is shown. 7c, 2 independent live-cell experiments performed, an example lineage is shown. 7d, 2 independent live-cell experiments performed, an example lineage is shown. 7e, 2 independent live-cell experiments performed, an example lineage is shown. 8b, 2 independent live-cell experiments performed, n = 60 sister cell pairs for UT, n = 60 sister cell pairs for IR; P value (Fisher's exact test): P = 0.0013. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. No restrictions apply to data availability. Additional data (original high-content microscopy, time-lapse microscopy and Live+QIBC raw data) are available on request. Gel source data are provided in Supplementary Fig. Single-cell sequencing data have been uploaded to the Gene Expression Omnibus (GEO) under accession numbers GSE255874, GSE288487 and GSE288485. GENCODE human genome build GRCh38.p13 (Release 37) was used as a reference genome77. Source data are provided with this paper. No restrictions apply to code availability. ImageJ/Fiji and MATLAB scripts that were used in this study are available at GitHub (https://github.com/AltmeyerLab/). Detailed instructions can be found at Zenodo80 (https://zenodo.org/records/14921691) in the file ‘Live+QIBC_guide.pdf', together with a test dataset, which contains data for pre-imaging (Pre-imaging.zip), post-imaging (Post-imaging.zip) and iterative staining (Iterative staining.zip), and a README file (README_TestData.pdf). Moreover, there are .txt files, which correspond to analyses performed with the test dataset and the Olympus ScanR Analysis software (v.3.0.1, 3.2 and 3.3.0; analysis_live.txt and analysis_iterative_staining.txt) to be used with the MATLAB scripts without having to perform the image analysis. Finally, there is a Tracking_fiji.xls file, which is used to copy the values generated from the ‘Coord' table in the MATLAB script to import into the tracking plugin in ImageJ/Fiji 64-bit (v.1.53f, 1.53t, 1.54f, 1.54m), so that the tracks can be overlayed with the stack of the corresponding cells. Buenrostro, J. D. et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Eling, N., Morgan, M. D. & Marioni, J. C. Challenges in measuring and understanding biological noise. The repertoire of mutational signatures in human cancer. Koh, G., Degasperi, A., Zou, X., Momen, S. & Nik-Zainal, S. Mutational signatures: emerging concepts, caveats and clinical applications. & Tan, P. Mapping genomic and epigenomic evolution in cancer ecosystems. & Wu, L. F. Cellular heterogeneity: do differences make a difference? Brock, A., Chang, H. & Huang, S. Non-genetic heterogeneity—a mutation-independent driving force for the somatic evolution of tumours. & Lahav, G. We are all individuals: causes and consequences of non-genetic heterogeneity in mammalian cells. Sandler, O. et al. Lineage correlations of single cell division time as a probe of cell-cycle dynamics. & Pelkmans, L. Origins of regulated cell-to-cell variability. & Shaw, A. T. Tumour heterogeneity and resistance to cancer therapies. & Zhao, K. The epigenetic basis of cellular heterogeneity. Huang, S. Reconciling non-genetic plasticity with somatic evolution in cancer. Vendramin, R., Litchfield, K. & Swanton, C. Cancer evolution: Darwin and beyond. Pervasive lesion segregation shapes cancer genome evolution. Anderson, C. J. et al. Strand-resolved mutagenicity of DNA damage and repair. Beach, R. R. et al. Aneuploidy causes non-genetic individuality. Lambuta, R. A. et al. Whole-genome doubling drives oncogenic loss of chromatin segregation. & Kuffer, C. The consequences of tetraploidy and aneuploidy. Davoli, T. & De Lange, T. The causes and consequences of polyploidy in normal development and cancer. Prasad, K. et al. Whole-genome duplication shapes the aneuploidy landscape of human cancers. & Sheltzer, J. M. Chromosomal instability and aneuploidy as causes of cancer drug resistance. The evolution of lung cancer and impact of subclonal selection in TRACERx. Berge, U. et al. Asymmetric division events promote variability in cell cycle duration in animal cells and Escherichia coli. Huh, D. & Paulsson, J. Non-genetic heterogeneity from stochastic partitioning at cell division. Halazonetis, T. D., Gorgoulis, V. G. & Bartek, J. An oncogene-induced DNA damage model for cancer development. & Altmeyer, M. Dealing with DNA lesions: when one cell cycle is not enough. Moreno, A. et al. Unreplicated DNA remaining from unperturbed S phases passes through mitosis for resolution in daughter cells. Arora, M., Moser, J., Phadke, H., Basha, A. & Spencer, S. L. Endogenous replication stress in mother cells leads to quiescence of daughter cells. Barr, A. R. et al. DNA damage during S-phase mediates the proliferation-quiescence decision in the subsequent G1 via p21 expression. & Altmeyer, M. Inherited DNA lesions determine G1 duration in the next cell cycle. A., Afifi, M. M., Paul, D. & Cappell, S. D. Cell cycle inertia underlies a bifurcation in cell fates after DNA damage. Phase separation of 53BP1 determines liquid-like behavior of DNA repair compartments. Spies, J. et al. 53BP1 nuclear bodies enforce replication timing at under-replicated DNA to limit heritable DNA damage. Gut, G., Herrmann, M. D. & Pelkmans, L. Multiplexed protein maps link subcellular organization to cellular states. Maiani, E. et al. AMBRA1 regulates cyclin D to guard S-phase entry and genomic integrity. Simoneschi, D. et al. CRL4(AMBRA1) is a master regulator of D-type cyclins. Chiolo, I., Altmeyer, M., Legube, G. & Mekhail, K. Nuclear and genome dynamics underlying DNA double-strand break repair. Arnould, C. et al. Chromatin compartmentalization regulates the response to DNA damage. Benamar, M. et al. Inactivation of the CRL4-CDT2-SET8/p21 ubiquitylation and degradation axis underlies the therapeutic efficacy of pevonedistat in melanoma. Lin, J. J., Milhollen, M. A., Smith, P. G., Narayanan, U. & Dutta, A. NEDD8-targeting drug MLN4924 elicits DNA rereplication by stabilizing Cdt1 in S phase, triggering checkpoint activation, apoptosis, and senescence in cancer cells. Petropoulos, M., Champeris Tsaniras, S., Taraviras, S. & Lygerou, Z. Replication licensing aberrations, replication stress, and genomic instability. Klotz-Noack, K., McIntosh, D., Schurch, N., Pratt, N. & Blow, J. J. Re-replication induced by geminin depletion occurs from G2 and is enhanced by checkpoint activation. Fragkos, M., Ganier, O., Coulombe, P. & Mechali, M. DNA replication origin activation in space and time. Lewis, J. S., van Oijen, A. M. & Spenkelink, L. M. Embracing heterogeneity: challenging the paradigm of replisomes as deterministic machines. Reijns, M. A. M. et al. Lagging-strand replication shapes the mutational landscape of the genome. Spencer Chapman, M. et al. Prolonged persistence of mutagenic DNA lesions in somatic cells. Seplyarskiy, V. B. et al. Error-prone bypass of DNA lesions during lagging-strand replication is a common source of germline and cancer mutations. Kimble, M. T. et al. Repair of replication-dependent double-strand breaks differs between the leading and lagging strands. Structure and repair of replication-coupled DNA breaks. Ferrand, J. et al. Mitotic chromatin marking governs the segregation of DNA damage. Gudjonsson, T. et al. TRIP12 and UBR5 suppress spreading of chromatin ubiquitylation at damaged chromosomes. A. et al. Histone ubiquitination by the DNA damage response is required for efficient DNA replication in unperturbed S phase. Kilgas, S., Swift, M. L. & Chowdhury, D. 53BP1-the ‘Pandora's box' of genome integrity. Kim, J., Sturgill, D., Tran, A. D., Sinclair, D. A. & Oberdoerffer, P. Controlled DNA double-strand break induction in mice reveals post-damage transcriptome stability. Bantele, S. et al. Repair of DNA double-strand breaks leaves heritable impairment to genome function. Preprint at bioRxiv https://doi.org/10.1101/2023.08.29.555258 (2023). Papathanasiou, S. et al. Heritable transcriptional defects from aberrations of nuclear architecture. Aymard, F. et al. Transcriptionally active chromatin recruits homologous recombination at DNA double-strand breaks. Polasek-Sedlackova, H., Miller, T. C. R., Krejci, J., Rask, M. B. Solving the MCM paradox by visualizing the scaffold of CMG helicase at active replisomes. Ran, F. A. et al. Genome engineering using the CRISPR-Cas9 system. Bindels, D. S. et al. mScarlet: a bright monomeric red fluorescent protein for cellular imaging. Lock, R. et al. Autophagy facilitates glycolysis during Ras-mediated oncogenic transformation. Phase separation properties of RPA combine high-affinity ssDNA binding with dynamic condensate functions at telomeres. Muraro, M. J. et al. A single-cell transcriptome atlas of the human pancreas. van den Brink, S. C. et al. Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. Hashimshony, T. et al. CEL-Seq2: sensitive highly-multiplexed single-cell RNA-Seq. Kaminow, B., Yunusov, D. & Dobin, A. STARsolo: accurate, fast and versatile mapping/quantification of single-cell and single-nucleus RNA-seq data. Preprint at bioRxiv https://doi.org/10.1101/2021.05.05.442755 (2021). Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Vuong, H., Truong, T., Phan, T. & Pham, S. Venice: a new algorithm for finding marker genes in single-cell transcriptomic data. Preprint at bioRxiv https://doi.org/10.1101/2020.11.16.384479 (2020). Ge, S. X., Jung, D. & Yao, R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. Heberle, H., Meirelles, G. V., da Silva, F. R., Telles, G. P. & Minghim, R. InteractiVenn: a web-based tool for the analysis of sets through Venn diagrams. Yu, G. C., Wang, L. G., Han, Y. Y. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Hatakeyama, M. et al. SUSHI: an exquisite recipe for fully documented, reproducible and reusable NGS data analysis. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. Frankish, A. et al. Gencode 2021. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Panagopoulos, A. et al. Code for ‘Multigenerational cell tracking of DNA replication and heritable DNA damage'. We acknowledge the staff at Functional Genomics Center Zurich, the Center for Microscopy and Image Analysis, the Flow Cytometry Facility of the University of Zurich and ETH ScopeM for technical support and Single Cell Discoveries for single-cell sequencing and data analysis. We thank P. Janscak, R. Santoro, M. Luijsterburg, G. Legube, H. Polasek-Sedlackova and D. Gerlich for providing reagents. This study received funding from the Swiss National Science Foundation (grants 179057, 197003 and 10000233 to M.A. ), the European Research Council under the European Union's Horizon 2020 research and innovation program (ERC-2016-STG 714326 to M.A.) and the Swiss State Secretariat for Education, Research and Innovation (SERI) in relation to the European Union's Horizon Europe research and innovation programme under the Marie Skłodowska-Curie grant agreement no. received additional financial support from a UZH Postdoc Grant and the Kurt & Senta Herrmann Foundation. Open access funding provided by University of Zurich. Present address: The Novo Nordisk Foundation Center for Protein Research, University of Copenhagen, Copenhagen, Denmark Present address: NEXUS Personalized Health, ETH Zurich, Schlieren, Switzerland These authors contributed equally: Andreas Panagopoulos, Merula Stout Andreas Panagopoulos, Merula Stout, Sinan Kilic, Julia Vornberger, Virginia Pasti, Antonio Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof & Matthias Altmeyer Peter Leary & Hubert Rehrauer Center for Microscopy and Image Analysis, University of Zurich, Zurich, Switzerland You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed RNA-seq and scRNA-seq experiments. performed colony-formation experiments, helped by A.G.; A.P., M.S. and R.I. performed western blot experiments. analysed Live+QIBC and QIBC data, helped by K.K., J.V. analysed RNA-seq and scRNA-seq data. Funding acquisition: project grants to M.A. Project administration and supervision: M.A. and M.S., with edits and help from all of the authors. Correspondence to Matthias Altmeyer. The authors declare no competing interests. Nature thanks Sarah Aitken and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Representative live imaging stills with the nuclear detection mask and the multigenerational single cell tracks from GFP-H2B U-2 OS cells. (b) Schematic representation of the strategy for tagging the endogenous PCNA gene locus at the last exon in U-2 OS cells. The arrowhead points to the band indicative of correct targeting. (d) Cell cycle phase distribution of the parental and the edited U-2 OS cell line based on DAPI and EdU. (e) Cell cycle resolved γH2AX mean intensity scatter plots of the parental and the edited 53BP1-PCNA U-2 OS cells either untreated or IR-treated (4 Gy, 2 h recovery). (f) Cell cycle resolved Cyclin A mean intensity scatter plots of the parental and the edited 53BP1-PCNA U-2 OS cells either untreated or IR-treated (4 Gy, 2 h recovery). (g) High-content microscopy derived cell cycle profiles of 53BP1-PCNA U-2 OS cells subjected or not to timelapse microscopy for 28 h at 30 min intervals. (h) Cell cycle resolved EdU mean intensity scatter plots of 53BP1-PCNA U-2 OS cells subjected or not to timelapse microscopy for 28 h at 30 min intervals. (i) Cell cycle resolved γH2AX mean intensity scatter plots of 53BP1-PCNA U-2 OS cells subjected or not to timelapse microscopy for 28 h at 30 min intervals. (a) 53BP1-PCNA U-2 OS cells were treated with siRNA for 72 h and IR (4 Gy, 2 h) as indicated and stained for 53BP1 (Cy5). The endogenous 53BP1-mScarlet and antibody-based signals are shown, together with their segmentation masks. (b) High-content microscopy-based quantification of 53BP1-mScarlet foci using the samples and foci segmentation mask from (a). Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (c) Video stills of 53BP1-PCNA U-2 OS cells at different cell cycle phases from G1 through S-phase and to G2. PCNA-mEmerald patterns and their segmentation are shown. (d) DAPI- and EdU-based cell cycle gating of 53BP1-PCNA U-2 OS cells and PCNA foci analysis in different cell cycle phases (colour-coded based on DAPI and EdU). (e) PCNA foci quantification from (d) in G1 versus S-phase versus G2 (based on DAPI and EdU). Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (a) Schematic representation of in silico single cell lineage sorting. (b) In silico sorted single cell lineages from untreated 53BP1-PCNA U-2 OS cells imaged for 30 h. Single cell lineages colour-coded for PCNA patterns (top) and 53BP1 foci (bottom) are shown. (c) Representative image stills from a single cell lineage from (b) depicting PCNA patterns and 53BP1 localization at the timepoints indicated. (d) Western blot analysis of DNA damage markers in untreated U-2 OS cells or U-2 OS cells treated with 0.2 μM APH or 1 μM ATRi for 24 h. (e) In silico sorted single cell lineages from 53BP1-PCNA U-2 OS cells imaged for 30 h. Cells were treated with 0.2 μM APH 4.5 h after the beginning of acquisition. Single cell lineages colour-coded for PCNA patterns (top) and 53BP1 foci (bottom) are shown. (f) Representative image stills from a single cell lineage from (e) depicting PCNA patterns and 53BP1 localization at the timepoints indicated. (g) In silico sorted single cell lineages from 53BP1-PCNA U-2 OS cells imaged for 30 h. Cells were treated with 1 μM ATRi 4.5 h after the beginning of acquisition. Single cell lineages colour-coded for PCNA patterns (top) and 53BP1 foci (bottom) are shown. (h) Representative image stills from a single cell lineage from (g) depicting PCNA patterns and 53BP1 localization at the timepoints indicated. For gel source data, see Supplementary Fig. (a) Schematic representation of the Live+QIBC image analysis pipeline. E, antibody elution; S, staining. (b) Representative images of sequential staining and antibody elution steps in the Live+QIBC image analysis pipeline with 53BP1-PCNA U-2 OS cells. (c, d) Scatter plots and representative images of control cells or cells undergoing replication stress sequentially stained for phospho-Rb and p21 in the same colour channel. The total DAPI intensity per nucleus indicates DNA content, with 2 N (G1) on the left and 4 N (G2) on the right. (e) Direct comparison of the pRb signals and the p21 signals after antibody elution and re-staining. (f) Cell cycle resolved scatter plots of γΗ2ΑΧ mean intensities from untreated U-2 OS cells or U-2 OS cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (g) Cell cycle resolved scatter plots of pRb mean intensities from untreated U-2 OS cells or U-2 OS cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (h) Cell cycle resolved scatter plots of p53 mean intensities from untreated U-2 OS cells or U-2 OS cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (i) Cell cycle resolved scatter plots of p21 mean intensities from untreated U-2 OS cells or U-2 OS cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (j) Heatmaps of cell cycle resolved analysis of asynchronous U-2 OS cell populations treated as indicated for 24 h with 0.2 μM APH, 1 μM ATRi, or with IR (1 Gy and 4 Gy) followed by recovery for 24 h. Cells were sequentially stained for multiple DNA damage and cell cycle markers. Nuclear intensities (in the case of 53BP1 accumulated foci intensities) per cell and morphological features are indicated by colour-code. Note that the colour-code was deliberately reversed for pRb. (a) Single cell lineage from 53BP1-PCNA U-2 OS cells treated with 0.2 µM aphidicolin during the S phase of the cell cycle. PCNA foci, 53BP1 foci and area of the nucleus over 55 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from corresponding reference populations are depicted for the markers pRB, γH2AX, p21 and p53. (b) Single cell lineage from 53BP1-PCNA U-2 OS cells treated with 1 µM ATR inhibitor during the S phase of the cell cycle. PCNA foci, 53BP1 foci and area of the nucleus over 55 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from corresponding reference populations are depicted for the markers pRB, γH2AX, p21 and p53. (c) Single cell lineage from 53BP1-PCNA U-2 OS cells treated with 0.2 µM aphidicolin during the G1 phase of the cell cycle. PCNA foci, 53BP1 foci and area of the nucleus over 55 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from corresponding reference populations are depicted for the markers pRB, γH2AX, p21 and p53. (d) Single cell lineage from 53BP1-PCNA U-2 OS cells treated with 1 µM ATR inhibitor during the G1 phase of the cell cycle. PCNA foci, 53BP1 foci and area of the nucleus over 55 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from corresponding reference populations are depicted for the markers pRB, γH2AX, p21 and p53. Drugs in (a-d) were removed after 24 h. (e) Sister cell heterogeneity analysis of 53BP1-PCNA U-2 OS cells treated either with 0.2 µM aphidicolin or with 1 µM ATR inhibitor. Heterogeneity was scored for 53BP1 foci between sister cells of the F1 generation in G1 phase, whereas γΗ2ΑΧ, p53 and p21 levels were scored at the end of 55 h of live imaging. Statistical analysis by Chi-Square test. Box plot limits indicate 25th percentile (Q1) and 75th percentile (Q3); boxes represent interquartile range (IQR, Q3-Q1) with medians (solid lines). Whiskers define lower and upper adjacent value; dots show outliers smaller than Q1 - 1.5 × IQR and greater than Q3 + 1.5 × IQR. (a) 53BP1-PCNA RPE-1 cells were treated with siRNA for 72 h and IR (4 Gy, 2 h) as indicated and stained for 53BP1 (Cy5). The endogenous 53BP1-mScarlet and antibody-based signals are shown, together with their segmentation masks. (b) High-content microscopy-based quantification of 53BP1-mScarlet foci using the samples and foci segmentation mask from (a). Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (c) Video stills of 53BP1-PCNA RPE-1 cells at different cell cycle phases from G1 through S-phase and to G2. PCNA-mEmerald patterns and their segmentation are shown. (d) DAPI- and EdU-based cell cycle gating of 53BP1-PCNA RPE-1 cells and PCNA foci analysis in different cell cycle phases (colour-coded based on DAPI and EdU). (e) PCNA foci quantification from (d) in G1 versus S-phase versus G2 (based on DAPI and EdU). Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (f) Cell cycle resolved scatter plots of γΗ2ΑΧ mean intensities from untreated RPE-1 cells or RPE-1 cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (g) Cell cycle resolved scatter plots of pRb mean intensities from untreated RPE-1 cells or RPE-1 cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (h) Cell cycle resolved scatter plots of p53 mean intensities from untreated RPE-1 cells or RPE-1 cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (i) Cell cycle resolved scatter plots of p21 mean intensities from untreated RPE-1 cells or RPE-1 cells treated with 0.2 µM aphidicolin or 1 µM ATR inhibitor for 48 h. (j) Western blot analysis of phospho-KAP1 in untreated RPE-1 cells or RPE-1 cells treated with 0.2 μM APH or 1 μM ATRi for 24 h. For gel source data, see Supplementary Fig. (a) Heatmap of the top 100 differentially expressed (DE) genes ranked by adjusted p-value upon IR treatment (4 Gy, 24 h) from bulk RNA-seq. (b) Gene ontology (GO) analysis of up-regulated DE genes upon IR treatment from bulk RNA-seq. (c) Example boxplots of DE genes after IR-induced DNA damage from bulk RNA-seq, including the CDK inhibitor and cell cycle regulator CDKN1A (p21), the pro-apoptotic cell death receptor FAS, the growth arrest and DNA damage-inducible protein GADD45A, and the p53 target gene and p53 regulator MDM2. TP53BP1 (53BP1) is shown as a control gene, the expression of which was not upregulated after IR. (d) Violin plots showing gene expression changes from scRNA-seq of the same genes depicted in (c). (e) Validation of the observed gene expression changes in a second set of cell samples subjected independently to scRNA-seq. (f) Heatmap of the top 10 marker genes identified from unsupervised clustering analysis of the scRNA-seq data from untreated and IR treated cells. (g) UMAPs (uniform manifold approximation and projection) of the scRNA-seq data colour-coded by the clusters identified in (f) and split by condition (UT, IR). (h) GO pathway enrichment in the single cell clusters identified in (f), excluding the smallest cluster k6, which did not yield any significantly enriched GO term. (i) Standard deviation (SD) versus Mean analysis of the scRNA-seq data. DNA damage response genes (GO: 0006974) highlighted in orange. (a, b) Single cell lineages from 53BP1-PCNA U-2 OS cells treated with 4 Gy IR during the G1 phase of the cell cycle. (c) QIBC-derived scatter plot of IR-induced 53BP1 foci in the next G1 phase. U-2 OS cells were pulsed with 10 µM EdU for 20 min before irradiation (4 Gy) and then allowed to cycle in the presence of 10 µM BrdU for 48 h to overcome the G2/M checkpoint. DNA was denatured for BrdU detection and cycled G1 cells negative for EdU and positive for BrdU were analysed. Horizontal solid lines represent the mean and horizontal dashed lines represent standard deviation. Example images are shown on the right. Statistical analysis by two-tailed unpaired t-test. (d) Endonuclease-induced DSBs in 53BP1-GFP DIvA-AID U-2 OS cells. AsiSI was induced by 4-OHT (300 nM) for 4 h, followed by fixation or release into medium without 4-OHT and containing auxin (IAA) (500 µM) for 20 h. 53BP1-GFP foci were analysed by QIBC. Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (e) In the same cells as in (d), γH2AX foci were analysed by QIBC. Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (f) Single cell lineages from 53BP1-GFP DIvA-AID U-2 OS cells treated with 4-OHT (300 nM) and IAA (500 µM) as indicated. 53BP1-GFP foci formation is depicted by the colour code. (g) As in (c), 53BP1 foci in cycled G1 53BP1-GFP DIvA-AID U-2 OS cells after transient AsiSI induction (4-OHT for 4 h, followed by release into medium with auxin (IAA) and without 4-OHT for 44 h). Statistical analysis by two-tailed unpaired t-test. (h) As in (c), 53BP1 foci in cycled G1 RPE-1 cells (p53 WT and p53 KO) after IR (4 Gy). Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (i) Sister cell heterogeneity in 53BP1-PCNA U-2 OS cells treated with IR (4 Gy) in G1. Heterogeneity was scored for 53BP1 foci between sister cells of the F1 generation in G1 phase, whereas p53, γH2AX and pRB levels were scored at the end of 55 h of live imaging. Statistical analysis by Chi-square test. (j) Single cell lineage of daughter cells irradiated with IR (4 Gy). Two rounds of replication associated with an increase in nuclear area can be seen for one of the two daughter cells. (a) Cell cycle resolved scatter plots of DNA content and area of the nucleus of cells shown in Fig. Hyperploid cells (> 4 N DNA content, marked by the dashed vertical line) are coloured in red. (b) Representative images of 53BP1-PCNA U-2 OS cells either untreated or treated with 175 nM of Pevonedistat for 24 h and allowed to recover for 42 h. Scale bar, 10 µm. (c) DNA fibre analysis of U-2 OS cells either untreated or treated with 175 nM of Pevonedistat for 24 h and then allowed to recover for 42 h. Statistical analysis by two-tailed unpaired t-test. (d) High-content microscopy derived cell cycle profiles of HCT 116 cells treated with 250 nM of Pevonedistat for 24 h. The percentage of hyperploid cells (> 4 N DNA content, marked by the dashed vertical line) is indicated. (e) High-content microscopy derived cell cycle profiles of RPE-1 cells treated with 175 nM of Pevonedistat for 24 h and then released into fresh medium for 42 h. The percentage of hyperploid cells (> 4 N DNA content, marked by the dashed vertical line) is indicated. (f) Representative single cell lineages depicting endoreplication after Pevonedistat exposure. (g) Representative single cell lineages depicting re-replication after Pevonedistat exposure. (h) Bar chart depicting the frequency of endo- and re-replication in 53BP1-PCNA U-2 OS cells treated either in G1 or S phase with 175 nM of Pevonedistat for 24 h and then released for 42 h. Mean ± SD are depicted. (i) Single cell lineage depicting endoreplication from live imaging of 53BP1-PCNA RPE-1 cells that were treated with Pevonedistat for 48 h. PCNA patterns and 53BP1 foci are shown. (j) Single cell lineage depicting re-replication from live imaging of 53BP1-PCNA RPE-1 cells that were treated with Pevonedistat for 48 h. PCNA patterns and 53BP1 foci are shown. (k) High-content microscopy derived cell cycle profiles of U-2 OS cells depleted of Geminin for 72 h. The percentage of hyperploid cells (> 4 N DNA content, marked by the dashed vertical line) is indicated. (l) Scatter plot of total DAPI intensity of normal G2 cells and Geminin depleted cells that underwent either endo- or re-replication. (m) Scatter plot of mean nuclear intensity of γH2AX of normal G2 cells and Geminin depleted cells that underwent either endo- or re-replication. (n) As in (m) for p53. (o) As in (m) for p21. (p) As in (m) for pRb. Statistical analysis was performed with two-tailed unpaired t-test between endo- and re-replication. Horizontal solid lines represent the mean and horizontal dashed lines represent standard deviation. (a) Western blot analysis of 53BP1-PCNA U-2 OS cells overexpressing HRAS. (b) Western blot analysis of 53BP1-PCNA U-2 OS cells overexpressing Cyclin E1. (c) DNA fibre analysis of 53BP1-PCNA U-2 OS cells overexpressing either HRAS or Cyclin E1. Statistical analysis by one-way ANOVA followed by Tukey's post hoc test. (d) Analysis of micronuclei induction in 53BP1-PCNA U-2 OS cells overexpressing either HRAS or Cyclin E1 compared to empty vector transfected control cells. Mean ± SD are depicted. Statistical analysis was performed with one-way ANOVA followed by Tukey's post hoc test. (e) Single cell lineage from empty vector control 53BP1-PCNA U-2 OS cells. PCNA foci, 53BP1 foci and area of the nucleus over 70 h of live imaging are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from the corresponding reference population are depicted for the markers pRB, γH2AX, p21 and p53. The total DAPI intensity of the nucleus from the corresponding reference population is depicted as well as from the single cells at the end of live imaging. (f) Single cell lineage from 53BP1-PCNA U-2 OS cells overexpressing HRAS. Live+QIBC data is shown as in (e). (g) Single cell lineage from 53BP1-PCNA U-2 OS cells overexpressing Cyclin E1. Live+QIBC data is shown as in (e). (h) Single cell lineage from 53BP1-PCNA cells overexpressing Cyclin E1. Live+QIBC data is shown as in (e). Representative images of the polyploid cells are included as well as their position within the DAPI scatter plot of the G1 and G2 populations. Box plot limits indicate 25th percentile (Q1) and 75th percentile (Q3); boxes represent interquartile range (IQR, Q3-Q1) with medians (solid lines). Whiskers define lower and upper adjacent value; dots show outliers greater than Q3 + 1.5xIQR. For gel source data, see Supplementary Fig. (a) High-content microscopy derived cell cycle profiles of 53BP1-PCNA U-2 OS cells overexpressing either HRAS or Cyclin E1 that were treated or not with 4 Gy of IR. The percentage of hyperploid cells (> 4 N DNA content, marked by the dashed vertical line) is indicated. (c) Bar chart depicting the percentage of polyploid cells overexpressing either HRAS or Cyclin E1 that were treated or not with 4 Gy of IR. Statistical analysis was performed with one-way ANOVA followed by Tukey's post hoc test. (d) Cell cycle resolved EdU profiles of 53BP1-PCNA U-2 OS cells overexpressing HRAS or Cyclin E1. Cells were either untreated or treated with 4 Gy of IR. (e) Video stills of 53BP1-PCNA U-2 OS cells overexpressing HRAS that are maintaining polyploidy through multiple cell generations. The polyploid granddaughter cells with 8 N DNA content are identified in a scatter plot of mean vs total DAPI intensity post live imaging. (f) Representative single cell lineages from live imaging of 53BP1-PCNA U-2 OS cells that overexpress HRAS. Lineages depict endo-replication and re-replication, respectively. (g) Western blot validation of inducible expression of Cyclin E1 in 53BP1-PCNA RPE-1 cells. Cells were induced with Doxycycline (100 ng/ml) for 24 h. (h) High-content microscopy derived cell cycle profiles of 53BP1-PCNA RPE-1 with inducible expression of Cyclin E1. The percentage of hyperploid cells (> 4 N DNA content, marked by the dashed red vertical line) is indicated. For gel source data, see Supplementary Fig. (a) Single cell lineage from control 53BP1-PCNA RPE-1 cells without Dox-induced Cyclin E1 expression. PCNA patterns and 53BP1 foci are depicted. Sequential staining intensities from individual cells of the lineage and mean intensities from the corresponding reference population are depicted for the markers pRB, γH2AX, p21 and p53. The total DAPI intensity of the nucleus from the corresponding reference population is depicted as well as from the single cells at the end of live imaging. (b) Single cell lineage from 53BP1-PCNA RPE-1 cells with Dox-induced Cyclin E1 expression (100 ng/ml Doxycycline for 24 h). Box plot limits indicate 25th percentile (Q1) and 75th percentile (Q3); boxes represent interquartile range (IQR, Q3-Q1) with medians (solid lines). Whiskers define lower and upper adjacent value; dots show outliers smaller than Q1 – 1.5 × IQR and greater than Q3 + 1.5 × IQR. (c) Analysis of endo- and re-replication frequency in asynchronously growing 53BP1-PCNA U-2 OS cells that received IR (4 Gy) either in G1 or in G2. Means and individual values from two independent live cell experiments are shown. A combined Supplementary Information PDF of 14 pages, containing Supplementary Figs. 1–12 and the corresponding legends. Differential gene expression from scRNA-seq of HRAS polyploid vs. EV cells, related to Supplementary Fig. Differential gene expression from scRNA-seq of Pevonedistat-treated polyploid vs. EV cells, related to Supplementary Fig. Differential gene expression from scRNA-seq of the polyploidy set sub-cluster analysis, related to Supplementary Fig. Differential gene expression from scRNA-seq of the polyploidy set sub-cluster analysis with more stringent cutoffs than in Supplementary Table 3, related to Supplementary Fig. Primer sequences used in this study. Cell tracking: live-cell imaging of U-2 OS cells expressing endogenous PCNA–mEmerald and 53BP1–mScarlet for 36 h at 30 min intervals. Shown is the PCNA–mEmerald signal together with the single-cell tracks after cell tracking. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Panagopoulos, A., Stout, M., Kilic, S. et al. Multigenerational cell tracking of DNA replication and heritable DNA damage. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09028-5'>Closed-loop vagus nerve stimulation aids recovery from spinal cord injury</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 15:08:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Decades of research have demonstrated that recovery from serious neurological injury will require synergistic therapeutic approaches. Rewiring spared neural circuits after injury is a long-standing goal of neurorehabilitation1,2. We hypothesized that combining intensive, progressive, task-focused training with real-time closed-loop vagus nerve stimulation (CLV) to enhance synaptic plasticity3 could increase strength, expand range of motion and improve hand function in people with chronic, incomplete cervical spinal cord injury. Here we report the results from a prospective, double-blinded, sham-controlled, randomized study combining gamified physical therapy using force and motion sensors to deliver sham or active CLV (ClinicalTrials.gov identifier NCT04288245). After 12 weeks of therapy composed of a miniaturized implant selectively activating the vagus nerve on successful movements, 19 people exhibited a significant beneficial effect on arm and hand strength and the ability to perform activities of daily living. CLV represents a promising therapeutic avenue for people with chronic, incomplete cervical spinal cord injury. Single therapies have demonstrated limited success in promoting recovery in people with chronic neurological injury. It has become increasingly apparent that the combination of complementary interventions represents a more promising strategy, such as recent applications that use electrical stimulation during rehabilitation to facilitate engagement of weakened networks4,5,6. CLV integrates targeted, intensive rehabilitative training with precisely timed vagus nerve stimulation (VNS) to promote recovery7,8. Rehabilitative training produces neural activation in spared pathways that control target musculature, which promotes plasticity through canonical mechanisms9,10, but is typically insufficient to support meaningful recovery on its own. CLV combines this training with VNS, which drives rapid, phasic release of acetylcholine, noradrenaline and serotonin throughout the central nervous system11,12,13,14,15. The resultant engagement of neuromodulatory networks within the seconds-long synaptic eligibility trace serves to enhance plasticity in networks activated by rehabilitation16. Conceptually, this approach of triggering stimulation to influence neuromodulatory networks during rehabilitation differentiates CLV from other combinatorial strategies that seek to directly facilitate the circuits engaged in movement6. In CLV, VNS and rehabilitation operate synergistically to mitigate the notoriously difficult credit assignment problem of identifying which synapses in a damaged network should be modified to produce more effective recovery of motor control10,17. CLV was initially developed and refined in a range of animal models, including traumatic incomplete spinal cord injury (SCI)3,18,19,20,21,22,23,24. These studies have revealed that the combination of high-intensity rehabilitative training with real-time VNS produces synaptic plasticity in motor control networks in the cortex, subcortical structures and spinal cord to engender functional recovery that is not possible with training alone. Concurrent implementation is a key component, as decoupling the delivery of VNS by even tens of seconds from the appropriate movements fails to promote recovery, consistent with the reliance on the synaptic eligibility trace3,25. Moreover, efficacy is dependent on hundreds of VNS–movement pairings per day for many weeks26,27. We sought to evaluate this approach in humans with chronic, incomplete SCI. To do so, we developed an integrated two-element system to govern both high-intensity rehabilitation and real-time neuromodulation (Fig. The first element utilizes devices instrumented to collect a range of movement parameters to allow each user to control an individualized set of video games incorporating dozens of different exercises to target specific muscles with limited function28. Difficulty was continuously adjusted so that the speed, force and range of motion needed to control gameplay were individualized to the peak performance abilities of each participant throughout the course of therapy. The second element, a miniaturized implanted VNS device, delivers concurrent real-time neuromodulation. The implanted stimulator29, which is 50 times smaller than existing systems and implanted via a simplified procedure, was wirelessly activated by a therapist or the gameplay software using a low-latency, adaptive real-time algorithm to deliver stimulation concurrent with movements that best approximate the desired outcome30 (Fig. After integrating these components and gaining regulatory approval, including investigational device exemption and designation as a breakthrough device from the FDA, we performed a double-blinded, sham-controlled clinical trial to test the hypothesis that CLV could enhance recovery of arm and hand function in 19 individuals with chronic, incomplete cervical SCI (Fig. a, CLV combines intensive, task-oriented rehabilitation and concurrent VNS delivered with a miniaturized implanted stimulator to promote adaptive changes in the central nervous system and facilitate recovery of function. b, CLV integrates external and implanted hardware to deliver high-intensity arm and hand therapy enhanced by real-time neuromodulation. The miniaturized implant is powered and controlled by an external device placed on the neck during rehabilitation sessions. A suite of sensors enables the rehabilitation software to provide continuous visual feedback of hand position and force production during individualized rehabilitative exercises and facilitate real-time delivery of VNS during above average movements. c, In this study, we observed that CLV (red) produced accumulating improvements in upper limb recovery exceeding those with intensive rehabilitation with sham stimulation (blue). d, CONSORT diagram summarizing the enrolment stages and the per protocol completion rate (detailed in Extended Data Fig. A description of reasons participants did not meet criteria can be found in Extended Data Table 2. All participants that met enrolment criteria were implanted and completed the study per protocol, which involved 42 visits total, including 36 physical therapy sessions. Participants were block randomized to receive 18 sessions of rehabilitation with either active CLV (1× CLV) or sham stimulation (1× rehab) during the RCT phase, followed by an additional 18 sessions of rehabilitation with active CLV regardless of previous group assignment. This design allows comparison with sham stimulation and of two dosing regimens of CLV. e, The proportion of injury type, ethnicity and age of individuals in this study are comparable with averages reported by the National Spinal Cord Injury Statistical Center. This trial followed the recommendations of the International Campaign for Cures of Spinal Cord Injury Paralysis panel33 and succeeded in enrolling a reasonably diverse set of participants whose injury type, ethnicity and age were generally consistent with the national averages reported by the National Spinal Cord Injury Statistical Center (Fig. This study had a somewhat higher proportion of female individuals and individuals within 5 years since injury. There is evidence that interest in clinical trials may lessen with time after injury34, which could explain why we did not recruit a larger proportion of individuals who were more than 5 years post-injury. The 19 people who met the study criteria ranged considerably in age (21–65 years of age, median 40), time since injury (13–541 months, median 54), injury severity classification (American Spinal Injury Association Impairment Scale (AIS) B, C and D) and resultant arm and hand deficits (33–101 GRASSP, median 64). This diverse study population allows for initial exploration of the impact of demographic factors that could influence response to CLV. Our baseline assessments confirmed earlier studies showing that clinical impairment in upper limb function as measured by the Graded and Redefined Assessment of Strength, Sensibility and Prehension (GRASSP) score is highly correlated with the peak pinch force that participants could produce between their thumb and index finger35 (Pearson correlation, R2 = 0.77, P = 1 × 10−6; Fig. GRASSP was also well correlated with the peak torque participants could produce with their wrist (Pearson correlation, R2 = 0.68, P = 1×10−5; Fig. The linear combination of these two measures of hand strength produced a model that explains 87% of the variance in disability in this population, with each factor providing a statistically significant contribution to the model (Pearson correlation, P = 7 × 10−9; Fig. The strong association of clinical impairment with finger and wrist strength provides a rigorous therapeutic rationale for the use of exercises designed to target these muscles. a,b, Clinical impairment as measured by GRASSP is well correlated with pinch force (a) and wrist torque (b). c, Both metrics make statistically significant contributions to a linear model that accounts for most of the variance in clinical impairment. The strong correlation between clinical impairment and distal strength supports the selection of rehabilitative exercises designed to target these muscles. d, Each participant displayed a unique set of impairments as measured by the GRASSP score. The Pearson correlation coefficient between individuals was 0.34 ± 0.35 (mean ± s.d.). This diversity in function motivated the creation of individualized rehabilitative regimens. Although some degree of finger and wrist weakness is ubiquitous in this patient population, there is a great deal of individual heterogeneity in the nature and magnitude of functional impairments. As expected, all participants in this study presented with a unique pattern of impairment, illustrated by variable deficits in GRASSP category scores (Fig. To ensure that each participant received exercises tailored to their specific impairment, the rehabilitative training regimen for each participant consisted of a personalized set of six to nine exercises and accompanying games (Fig. Exercises were further individualized by adjusting the degree to which the force, torque or range of motion produced on each exercise was multiplied by a linear assistance factor (that is, gain) to make the games challenging but playable. The average assistance factor applied to each participant during rehabilitation was well correlated with their GRASSP score (Pearson correlation, R2 = 0.63, P = 5 × 10−5; Fig. Therapists observed exercises and, when possible, progressively increased task difficulty by gradually reducing the assistance factor. Therapists also adjusted task difficulty by increasing the level of each game, which necessitated greater speed and precision (Fig. Over the course of the study, exercise difficulty was adjusted 154 ± 15 times per person to ensure that participants were maximally challenged and at the limits of their ability, as documented by a stable rate of errors during gameplay. The observations that game error rate (1.1 ± 0.3 errors per minute, R2 = 0.00, P = 0.9), time actively engaged with task training (38.6 ± 0.7 h, R2 = 0.03, P = 0.5) and number of exercise repetitions per day (1,537 ± 74 repetitions, R2 = 0.002, P = 0.8) were not correlated with GRASSP confirm that the therapy parameters were set to be highly challenging despite a wide range in impairment nature and severity. Stimulation triggering was governed by an algorithm that scaled adaptively based on performance measured with the sensorized controllers30, which resulted in stable delivery of VNS across a wide range of impairments and over the course of rehabilitation sessions (Extended Data Fig. The rehabilitative regimen used a range of sensors, including a wireless accelerometer, strain gauge, rotary encoder, keyboard, video camera and touchscreen. An algorithm utilized the sensor data to select above-threshold movements during exercises and deliver real-time VNS. 28, Sage (traffic racer and breakout); ref. Colours correspond to the key in panel b. b, Over 36 days of therapy, participants completed approximately 4,800 activities, individualized to their level of impairment and residual hand and arm function. Participants performed 6–9 different activities per session, with each activity lasting an average of 3.3 min (IDR: 0.7–5.5 min) and typically repeated twice per session. We observed a 100% compliance rate, with all participants completing 36 sessions of therapy. A supplementary interactive HTML version of this figure provides comprehensive information on each session for each participant, including the number of VNS events, the number of repetitions, the assistance factor, the difficulty level, motor performance, and whether active or sham VNS was delivered. c, A therapist guiding exercises increased the linear assistance factor (gain), which was multiplied by the force or range of motion for each exercise for each person until they could succeed in gameplay. The average value was highly correlated with baseline GRASSP scores, which confirms that exercises were individualized and challenging. d, In addition, the overseeing therapist increased game level to ensure that tasks were challenging at every stage of therapy. Higher game levels required greater speed and precision. The average game level increased steadily over the 36 days of therapy for all participants (Pearson correlation, P < 0.001). Isometric pinch and knob exercises were included in the therapy programme developed for every person, as distal hand function was consistently impaired (Fig. Consequently, we used these metrics to track progress in all participants and assessed whether a course of CLV would produce improvements in these commonly impaired measures. Thirty-six sessions of therapy produced statistically significant increases in pinch force for 18 out of 19 people. The average pinch force increased by 936 ± 247 g, a gain of 393 ± 102% over baseline (two-tailed, paired Student's t-test, n = 19, P = 5 × 10−8; Fig. Similarly, the wrist torque that participants could exert on a door knob increased significantly in 13 out of 19 people, with an average increase of 28.5 ± 16.8 N cm, representing a 152 ± 87% improvement (two-tailed, paired Student's t-test, n = 19, P = 0.007; Fig. Over the course of therapy, the force, speed and range of motion produced during various different exercises were approximately doubled (Fig. These gains exceed the preregistered outcomes. As expected from earlier studies in chronic cervical SCI, some participants failed to make statistically significant gains on some exercises36. These observations suggest that CLV can significantly improve hand strength in people with chronic cervical SCI, although rigorously validated methods are necessary to ensure that changes in force production are clinically meaningful. a,b, Pinch force (a) and knob torque (b), exercises performed by all participants, steadily increased over the course of therapy in the substantial majority of individuals. c–i, Similarly, strength, speed and range of motion (ROM) increased across a wide range of isometric and dynamic exercises. A linear mixed model was fitted to the data for each task, which was collected from daily rehabilitative training sessions. There was a significant fixed effect of therapy day for each task, as noted by the P value on each of the figure's panels. n denotes the number of participants that performed the task as part of their individualized therapy. The thick lines indicate participants who made statistically significant increases in task performance as a function of therapy day. We next evaluated whether these gains in hand and arm capabilities translated to improvements in clinical metrics. Collectively, we observed a statistically significant improvement in GRASSP score after CLV (two-tailed, paired Student's t-test, n = 19, 4.1 ± 1.5 points, P = 0.01; Fig. This improvement exceeds the preregistered outcome and casts doubt on the long-held notion that additional gains are not possible in people with traumatic SCI more than 1 year post-injury and highlights the potential of CLV as a novel therapy for SCI37,38,39,40,41. Thirty-six sessions of CLV produced accumulating gains approximately double that observed after 18 sessions (Fig. In addition, the cumulative gains are consistent with the hypothesized mechanism of action by which real-time VNS promotes synaptic plasticity in spared networks25. We also observed modest gains in the untrained arm (Extended Data Fig. a, In the double-blinded, sham-controlled phase of the study, GRASSP score was significantly improved compared with baseline in all participants after 18 sessions of CLV and in the subset of participants that received 36 sessions of CLV. No gains were observed in the group that received intensive rehabilitation with sham stimulation (stim). CLV produces a medium effect in improvement in upper limb function (Cohen's d effect size > 0.5). b, After the completion of 36 sessions of therapy (after therapy), 8 individuals made meaningful increases in the GRASSP score. This effect appears to be larger in individuals with motor incomplete injuries (AIS C and D). Dashed line indicates the cut-off for a meaningful increase, as defined by a 6-point or greater increase in GRASSP score. c, Meaningful improvements (denoted with bold lines) were observed across participants with severe, moderate and mild impairments in hand and arm function. d, Single characteristics, including the baseline GRASSP score, were not correlated with treatment response. Dashed line denotes no change in GRASSP score. e, However, a multiple linear regression model (detailed in Extended Data Table 3) with the AIS grade, GRASSP strength subscore, and GRASSP palmar and dorsal sensory subscores as inputs was highly correlated with the change in the GRASSP score from baseline to completion of 18 sessions of CLV. g, A range of the ten muscle groups evaluated during the GRASSP assessment improved with therapy. Muscles are colour coded to illustrate the percent of responders who exhibited measurable improvements in each muscle. Deltoid, pollicis longus, elbow extensors and wrist extensors were among the most commonly improved muscles. Panel g adapted using Sketchfab under a CC BY 4.0 license. Panels a,c,f used two-way paired Student's t-test versus baseline for CLV, and Wilcoxon signed-rank versus baseline for sham stimulation; *P < 0.05 and **P < 0.01. Group data are presented as mean ± s.e.m. The majority of people with motor incomplete injuries (7 of 13, AIS C or D) exhibited a meaningful response to therapy, as defined by a six-point or greater increase in GRASSP score (Fig. By contrast, only one of the six people with motor complete injuries (AIS B) made a meaningful improvement. The greater recovery observed in individuals with motor incomplete injuries may suggest that individuals with motor complete injuries do not benefit from CLV. Alternatively, the observation that this population exhibited half the benefit seen in those with motor incomplete injuries benefit (2.3 ± 1.3 versus 4.9 ± 2.1 points) could be explained if this population requires twice as much therapy to produce the same benefit. Future studies are needed to distinguish between these possibilities. In addition to GRASSP scores, we performed an exploratory analysis to evaluate whether CLV would enhance performance of common activities of daily living, as measured with the Jebsen–Taylor hand function test. At the conclusion of CLV, the Jebsen–Taylor hand function score increased by 7.5 ± 3.1 points (two-tailed, paired Student's t-test, n = 19, P = 0.03) compared with baseline (Extended Data Fig. An exploratory analysis revealed that the Spinal Cord Independence Measure Version 3 (SCIM-III) components that emphasize arm and hand use demonstrated significant improvement after CLV (Wilcoxon signed-rank test, +1.3 ± 0.7 points, n = 19, P = 0.04). Collectively, these results demonstrate that hand and arm function can be improved in people with chronic, incomplete cervical SCI. Given the heterogeneity along a range of participant characteristics, we sought to determine whether there were potential relationships between these baseline features and treatment response. Single characteristics, including baseline impairment severity (Fig. This observation is consistent with findings in stroke42 and suggests that this approach could be potentially valuable for a large number of people living with chronic, incomplete SCI. Multiple linear regression modelling indicates that treatment response is greatest in individuals classified as motor incomplete and that have less strength and poor palmar sensation with preserved dorsal sensation (Fig. This aligns with preclinical evidence showing that both lesion characteristics and pathological synaptic plasticity directed to non-functional skin surfaces shape the potential to benefit from CLV21. These initial findings highlight the importance of exploring predictors of response in a larger study. To estimate the effect size that VNS contributed to recovery, participants were randomized to receive either sham stimulation (n = 9) or active CLV (n = 10) during the first 18 sessions of high-intensity physical therapy (Extended Data Fig. 1; additional description of trial design, clinical protocol and consent procedures are available in Supplementary Information). Because all participants were implanted with the miniaturized device, received comparable high-intensity rehabilitation and stimulation was software controlled, therapists, assessors and participants were blinded to group assignment. Participant blinding was confirmed by questionnaires at the end of the randomized controlled trial (RCT) phase, which revealed that 58% of participants (11 of 19) were incorrect or uncertain about whether they received active VNS or sham stimulation, comparable with previous studies that used a similar blinding strategy43. The RCT phase of the study confirmed that adding VNS to rehabilitation provides a medium-sized effect size over equivalent rehabilitation with sham stimulation, although this study was not powered to reach statistical significance (Fig. The effect size observed in this study was comparable with that observed in people with ischaemic stroke, which is now an FDA-approved therapy43. These findings highlight the need for a prospectively powered study to evaluate the efficacy of CLV in individuals with chronic SCI. On average, the eight responders lost 23 ± 4% of their disability as measured by GRASSP. Future studies are needed to determine whether greater gains are possible with additional therapy sessions and whether some proportion of the non-responders can be converted to responders by adjusting VNS intensity. Although animal studies have consistently shown that the treatment effects of VNS during therapy are an inverted-U function of stimulation intensity44,45,46,47, this study did not seek to individualize VNS current. To clarify the degree to which the current set of rehabilitative tools and exercises was able to improve specific muscle groups, we evaluated the probability that each of the ten muscles tested in the GRASSP assessment were improved in responders (Fig. Improvements were observed in proximal muscles of the arm and intrinsic and extrinsic muscles of the hand (Fig. Additional work is needed to ensure that rehabilitative exercises are developed to target this muscle, as well as additional muscles that are not included in the GRASSP assessment but that may benefit from targeted exercises. The core premise of CLV is the coupling of task-specific training and real-time neuromodulation. On average, during sessions in which active VNS was paired with rehabilitative training, participants received 341 ± 15 half-second stimulation bursts to coincide with selected movements. We utilized two strategies to govern the triggering of stimulation during movement. Initially, for the first 96,000 stimulation events, triggering decisions were made by therapists observing rehabilitative exercises using a dedicated software application. Manual triggering resulted in VNS events coinciding with movements that were twice the average amplitude above a non-targeted periodic triggering scheme, confirming that the therapists successfully coupled stimulation with task-focused movements (Extended Data Fig. Using the therapists triggering scheme as a ground truth, we developed and validated an automated algorithm to monitor movement and trigger stimulation30. As expected, the automated triggering method significantly outperformed therapists (unpaired, two-tailed Student's t-test, P = 2 × 10−34). In conjunction with feasibility, a key consideration in the delivery of any therapy with an implanted device is safety. After 19 implant surgeries, 760 patient visits, 3.7 million total VNS pulses and a collective 42 patient-years of device contact, there were zero serious device-related adverse events and zero unexpected device-related adverse events. All devices performed to specification, and we observed no device complications or technical failures. These findings are consistent with the larger corpus of literature using standard VNS strategies and reinforce that this approach is safe and well tolerated. Because the CLV strategy tested in this study used a novel miniaturized VNS system, we assessed occurrence of adverse events. All study-related adverse events were classified as mild (Extended Data Table 4). As expected, the rate of post-surgical pain at the incision site was comparable with published rates using VNS in individuals with stroke48. By contrast, rates of voice alteration appear to have been reduced in this study. This probably reflects the reduced device size and simplified surgical procedure that obviates subcutaneous tunnelling in the neck, and the dramatically lower total charge delivery required for CLV relative to conventional VNS for epilepsy. Because the vagus nerve exerts control of autonomic function, it is conceivable that VNS could influence cardiovascular function and induce autonomic dysreflexia in individuals with SCI. Over the course of treatment, we observed no changes in heart rate or blood pressure between active and sham stimulation (Extended Data Fig. In addition, no instances of autonomic dysreflexia were observed with stimulation. Consistent with results in animal models, these findings indicate that the stimulation parameters used for CLV do not meaningfully alter autonomic function and reinforce the safety of this strategy in individuals with SCI49. Here we report the first-in-human use of CLV in 19 individuals with chronic, incomplete cervical SCI. Delivery of CLV using a novel, miniaturized VNS system was safe and feasible, with individuals receiving approximately 12,000 stimulations combined with weeks of intensive, personalized, task-focused rehabilitation. The majority of individuals exhibited meaningful improvements in multiple metrics of arm and hand function, and a longer course of therapy produced accumulating gains. The magnitude of upper limb recovery and proportion of responders was comparable with that observed using a similar approach in chronic stroke, which recently received the first-ever FDA approval for an intervention to improve recovery in the chronic phase. Collectively, these findings argue against the long-held belief that recovery after SCI is limited in the chronic phase and highlight the clinical potential of combinatorial treatment approaches. Some limitations of the present study merit consideration and should be addressed in future trials. Because there are no validated biomarkers of VNS, we did not collect direct evidence of neuromodulator engagement; such a metric, as well as direct measures of plasticity, could potentially be used to individualize stimulation parameters in future implementations. Personalization of neuromodulator release or plasticity represents a potential means to increase the proportion of individuals that exhibit a clinically meaningful response, a key consideration for an implanted device. Given the limited size of this first-in-human study, subsequent trials should evaluate a larger pool of participants to examine potential predictors of recovery, including demographic characteristics or the GRASSP predictor developed here, and expand evaluation of hand and arm metrics at longer times post-therapy. In summary, this study provides strong evidence that CLV is safe, feasible to deliver and provides initial evidence of robust improvements of arm and hand function that supports investigation in a larger trial. The trial was designed as a double-blinded, randomized, sham-controlled early feasibility study (Extended Data Fig. Individuals who indicated interest in participation and met general criteria in an initial screen underwent informed consent. After enrolment, all participants underwent implantation of the miniaturized VNS device. Approximately 4 weeks after implantation, all participants received a post-implantation baseline assessment. Participants then received 18 sessions of rehabilitation with active CLV or sham stimulation in accordance with their randomization. After the initial 18 sessions, participants received an additional 18 session of rehabilitation with active VNS, regardless of their previous group assignment. The primary objectives evaluated safety, feasibility and changes in measures of arm and hand function. The use of the miniaturized VNS device used in this study received investigational device exemption (approval from the FDA investigational device exemption approval ID G190032). In addition, all study procedures were approved by the Baylor Scott and White Research Institute Institutional Review Board, the University of Texas at Dallas Institutional Review Board, and the US Department of Defense Human Research Protections Office. The study was conducted in compliance with all relevant regulatory and ethical guidelines. Participants were recruited from 5 March 2021 to 30 June 2023. Patient characteristics are delineated in Extended Data Table 1. A total of 293 potential participants contacted us or were identified through an established referral network at Baylor Scott and White Spinal Cord Injury Model System, study flyers, local advertisements and online advertisement. Of these, 19 met criteria, elected to participate, were consented and enrolled, and ultimately underwent implantation. Informed consent was obtained in a private setting after consultation with study staff to answer any questions. The informed consent process followed our established procedure, which is included in the Supplementary Information. Participants met the following key inclusion criteria: (1) first time cervical SCI occurring at least 12 months before and resulting in AIS grade B, C or D (confirmed during the baseline visit by an experienced clinician); (2) residual movement in the upper limb and hand in either arm; (3) appropriate candidate for VNS implantation; (4) between 18 and 64 years of age; (5) a signed and dated informed consent form; and (6) willing to comply with all study procedures and were available for the duration of the study. Participants were excluded based on the following criteria: (1) SCIs by sharp objects, firearms, and non-traumatic or congenital causes; (2) evidence of recurrent laryngeal nerve injury; (3) excessive scar tissue in the neck; (4) concomitant clinically significant brain injuries; (5) previous injury to the vagus nerve; (6) previous or current treatment with VNS; (7) receiving any therapy that would interfere with VNS; (8) pregnant or lactating; (9) psychiatric disorders, psychosocial and/or cognitive impairment that would interfere with study participation; (10) abusive use of alcohol and/or illegal substances; (11) participation in other interventional clinical trials; (12) known immunodeficiency or receipt of chronic corticosteroids, immunosuppressants, immunostimulating agents or radiation therapy within 6 months; (13) significant comorbidities or conditions associated with high risk for surgical or anaesthetic survival; (14) active neoplastic disease; (15) significant local circulatory problems; (16) any medical condition or other circumstances that might interfere with their ability to receive rehabilitation or return for follow-up visits; (17) any condition that would preclude adequate evaluation of the safety and performance of the device; (18) aphasia and other cognitive deficits that interfere with provision of informed consent; (19) recent history of syncope; (20) recent history of dysphagia; (21) currently require, or are likely to require, diathermy; (22) significant respiratory issues that would interfere with participation; (23) non-English speaking; (24) acutely suicidal and/or have been admitted for a suicide attempt; and (25) incarceration or legal detention. Full eligibility criteria can be found on ClinicalTrials.gov (NCT04288245). No changes were made to eligibility criteria after trial commencement. All 19 enrolled participants were implanted with a next-generation VNS device29. This device is over 50 times smaller than conventional VNS systems, largely due to the offloading of the battery and elimination of the need for leads. Both of these changes simplify surgical implantation by necessitating only a single incision at the neck and obviating the need for tunnelling. After surgical preparation and induction of general anaesthesia, the skin and platysma overlying the left cervical vagus nerve were incised. The sternocleidomastoid muscle was mobilized laterally to reveal the carotid sheath, and the vagus nerve was dissected free circumferentially over a length of 3–4 cm. The IPG was then positioned superficially to facilitate alignment with the external components during stimulation. Before closure of the skin, wireless communication and power functions of the IPG were verified. After confirmation of device functionality, closure of the platysma and skin with absorbable sutures was performed, followed by a second communication verification. Impedance checks confirmed that all participants were within the acceptable range. Approximately 1 week later, participants returned to the clinic for a follow-up visit to assess recovery. Participants were randomized 1:1 to receive either 18 sessions of rehabilitation with sham stimulation followed by 18 sessions of rehabilitation with active VNS (n = 9) or 36 sessions of rehabilitation with active VNS (n = 10). Ten participants were randomized to receive active (0.8 mA, 30 Hz for 500 ms) VNS for all 36 sessions of VNS. Nine participants were randomized to receive sham VNS for the first 18 sessions and active VNS for the second 18 sessions (Extended Data Fig. Before the beginning of each rehabilitation session, vital signs were collected with a digital blood pressure cuff. Because stimulation was only delivered during rehabilitative training sessions, the PCM was only worn during these sessions. Each 0.5-s train of VNS was delivered concurrent with exercises during rehabilitative training (as detailed below), and comprised 0.8-mA 100-µs biphasic pulses at 30 Hz, as in previous studies. The concurrent timing of VNS and movement is based on the principle that precisely timed neuromodulatory feedback can direct specific synaptic changes to enhance recovery. After an SCI, a subset of neurons within the central nervous system is affected by the injury, and promoting long-term potentiation and depression within these networks has long been recognized as an approach to support recovery1,50,51. The ability to direct synapse-specific, adaptive plasticity within the affected networks while not influencing other networks is at the core of the credit assignment problem, which articulates the challenge of identifying which synapses should be changed to improve motor function17,52,53,54. CLV leverages the synaptic eligibility trace, a phenomenon in which the arrival of neuromodulatory reinforcement within seconds promotes plasticity in recently active networks and not in inactive networks16,55,56,57, to direct changes within specific synapses. In CLV, motor networks controlling the upper limb affected by the SCI are engaged by rehabilitative exercises, and closed-loop VNS delivered in response to a target movement triggers neuromodulatory release to produce therapeutic synaptic plasticity within these networks3,11. All participants underwent implantation, a key feature of blinding. In addition, all participants received equivalent individualized rehabilitation regimens and donned the external device during rehabilitation sessions, regardless of active or sham group assignment. The stimulation parameters were software controlled and were preset by a study staff member who was not involved in rehabilitation, data collection or analysis, allowing therapists and assessors to maintain blinding. Most participants did not perceive the stimulation or only felt the first few stimulations each day and rapidly adapted to the sensation. Preclinical and clinical studies indicated that this amount of stimulation was insufficient for therapeutic effect and sufficient to ensure blinding. Participants were instructed that they may initially perceive stimulation, but the perception may fade. Participant blinding was confirmed with a questionnaire at the end of phase 1 of the study, in which 11 of 19 participants were incorrect or uncertain about group assignment. Although therapists and assessors were blinded, blinding status was not directly surveyed in these individuals and should be assessed in future studies to confirm effective concealment. All 19 participants completed 36 sessions of intensive rehabilitation at a rate of approximately 3 per week. Each session was approximately 90 min long. Both groups followed the same visit schedule and received equivalent rehabilitation of one arm. Clinical judgement was used to select the arm that was most likely to benefit from therapy. Personalized exercise regimens were selected based on the ability profile of each participant. Each regimen incorporated conventional rehabilitative exercises and training on a computer-based rehabilitation system28. As appropriate for their level of ability and expressed interest, participants also performed functional tasks (labelled as object manipulation in Fig. 2 and ‘Individualized therapy sessions' in Supplementary Information), including jar opening, threading a nut on a bolt, lifting cans, writing, and inserting and turning keys. Participants generally cycled between relatively short (1–5 min) sets of each exercise to mitigate substantial fatigue. Difficulty was continuously monitored by licensed therapists to ensure that the therapeutic exercises were challenging. Assistance factor adjustments to the exercise regimens were made systematically, such that a typical progression for a single exercise included decreasing the input multiplier from the controller by 20%. The closed-loop triggering scheme to deliver VNS concurrent with movements used two strategies. In the first strategy, the therapist overseeing the rehabilitation session used a button press in a software app to trigger stimulation on movements identified as above-average attempts based on specific performance metrics, such as increased force, speed, accuracy or fluidity of motion. This is the stimulation approach that has been applied in conventional studies using paired VNS therapy43,58,59. In the second approach, triggering was automatically controlled by a software algorithm, as previously described30. This strategy used real-time movement signals collected from the sensors in the rehabilitative training devices and delivered stimulation on movements that exceeded a continuously updated threshold. The relevant aspect of the movement for triggering was dependent on the gameplay; for example, gameplay controlled by pinch force used the force signal to determine triggering, whereas gameplay controlled by wrist rotation used degree of rotation to determine triggering28,35. The algorithm initiated stimulation within 500 ms when movement exceeded the 95th percentile of previous repetitions. The algorithm produced stable triggering across a range of impairments and over the course of rehabilitative sessions (Extended Data Fig. Most participants received a combination of both stimulation-triggering strategies. The components were designed to emphasize adoption among individuals with SCI, maximize comfort and promote engagement with the therapy. Consistent with this, study participants indicated a high level of satisfaction with the therapy (4.6 ± 0.2 on a 5-point Likert scale surveyed at the end of the study), a critical consideration for eventual clinical adoption. The GRASSP (version 1) is a clinician-administered assessment quantifying three domains describing upper limb function and impairment60. This assessment was specifically designed to evaluate the effect of novel interventions on upper limb impairment in the traumatic tetraplegic population. The preregistered clinical end point was a greater than four-point increase in GRASSP. An improvement of six points or more in the trained limb was considered a meaningful difference. Higher scores are associated with improved arm and hand function. GRASSP scores are categorical and were collected at each assessment session. Percent disability, as reported in the main text, was derived from GRASSP scores. This metric was calculated as the number of GRASSP points gained during therapy as a proportion of the difference between baseline pre-therapy score and the total number of available points to reach a maximum score. A suite of rehabilitative tools was used to assess force and range of motion in the hand and wrist throughout the course of the study28,35. The system includes modules to quantify wrist rotational range of motion and torque on a doorknob and a revolving D-handle, wrist flexion–extension range of motion and torque, and finger flexion–extension force. These continuous values were collected at each rehabilitation session, as appropriate for the assigned exercises for each participant. The preregistered strength end points were greater than 10% increases in finger pinch and flexion force following active VNS, 10% increases in wrist flexion and extension force following active VNS, and 10% increases in wrist pronation and supination force following active VNS. The Jebsen Taylor Hand Function Test is a widely used standardized and objective measure of fine and gross motor hand functions that uses simulated activities of daily living61. This was an exploratory end point. As this study only performed hand and arm rehabilitation, we did not expect to see improvements in respiration, lower body function, bowel function, bladder function or mobility. We thus excluded these metrics from SCIM-III to produce an 18-point measure of independence associated with arm and hand function. This composite score includes feeding, grooming, toileting, upper body bathing and upper body dressing, all of which require arm and hand function. This was an exploratory end point. The AIS is the gold-standard assessment to evaluate level and completeness of injury in individuals with SCI. In this study, AIS was evaluated by R.G.H., a clinician with extensive expertise in SCI, at baseline and at the end of each phase. AIS grade was used to confirm study eligibility. The primary and secondary study outcomes were reviewed by the regulatory bodies and were preregistered on ClinicalTrials.gov (NCT04288245). Data in the figures and text are presented as mean ± standard error of the mean unless otherwise indicated. Comparisons across groups were made using unpaired Student's t-tests or Wilcoxon rank-sum tests. As appropriate, comparisons across time were made using paired Student's t-tests or Wilcoxon signed-rank tests. Before conducting any parametric test, the assumption of normality was first checked using the Jarque–Bera goodness-of-fit test. 3, a linear mixed model was fitted to the data from each task, and statistics were performed using that model. Pearson correlation tests were used to determine the correlation between a number of elements and GRASSP score changes. Standardized effect size was calculated using Cohen's d. As there was high test–retest reliability (Pearson correlation coefficient, R = 0.95, P < 0.00001) for the baseline assessments performed before and after implantation and no significant difference (P > 0.05), these two values were averaged to serve as the baseline assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The data collection code is available on GitHub (https://github.com/davepruitt/RePlay). All raw data have been included in the Supplementary Information and are available at Open Data Commons for Spinal Cord Injury (https://odc-sci.org/data/1315). & Schwab, M. E. Plasticity of motor systems after incomplete spinal cord injury. Murphy, T. H. & Corbett, D. Plasticity during stroke recovery: from synapse to behaviour. Ganzer, P. D. et al. Closed-loop neuromodulation restores network connectivity and motor control after spinal cord injury. Krucoff, M. O., Rahimpour, S., Slutzky, M. W., Edgerton, V. R. & Turner, D. A. Enhancing nervous system recovery through neurobiologics, neural interface training, and neurorehabilitation. Walking naturally after spinal cord injury using a brain–spine interface. Moritz, C. et al. Non-invasive spinal cord electrical stimulation for arm and hand function in chronic tetraplegia: a safety and efficacy trial. Hays, S. A., Rennaker, R. L. & Kilgard, M. P. Targeting plasticity with vagus nerve stimulation to treat neurological disease. Targeted vagus nerve stimulation for rehabilitation after stroke. Principles of experience-dependent neural plasticity: implications for rehabilitation after brain damage. Ganguly, K. & Poo, M. Activity-dependent neural plasticity from bench to bedside. Hulsey, D. R. et al. Parametric characterization of neural activity in the locus coeruleus in response to vagus nerve stimulation. Hulsey, D. R., Shedd, C. M., Sarker, S. F., Kilgard, M. P. & Hays, S. A. Norepinephrine and serotonin are required for vagus nerve stimulation directed cortical plasticity. Reorganization of motor cortex by vagus nerve stimulation requires cholinergic innervation. Bowles, S. et al. Vagus nerve stimulation drives selective circuit modulation through cholinergic reinforcement. & McCormick, D. Vagus nerve stimulation induces widespread cortical and behavioral activation. Distinct eligibility traces for LTP and LTD in cortical synapses. Targeting neuroplasticity to improve motor recovery after stroke: an artificial neural network model. Darrow, M. J. et al. Vagus nerve stimulation paired with rehabilitative training enhances motor recovery after bilateral spinal cord injury to cervical forelimb motor pools. Meyers, E. C. et al. Vagus nerve stimulation enhances stable plasticity and generalization of stroke recovery. Khodaparast, N. et al. Vagus nerve stimulation during rehabilitative training improves forelimb strength following ischemic stroke. Enhancing plasticity in central networks improves motor and sensory recovery after nerve damage. Pruitt, D. T. et al. Vagus nerve stimulation delivered with motor training enhances recovery of function after traumatic brain injury. Restoration of somatosensory function by pairing vagus nerve stimulation with tactile rehabilitation. Hays, S. A. et al. Vagus nerve stimulation during rehabilitative training improves functional recovery after intracerebral hemorrhage. Hays, S. A., Rennaker, R. L. & Kilgard, M. P. How to fail with paired VNS therapy. Ruiz, A. D. et al. Vagus nerve stimulation must occur during tactile rehabilitation to enhance somatosensory recovery. Effective delivery of vagus nerve stimulation requires many stimulations per session and many sessions per week over many weeks to improve recovery of somatosensation. Pruitt, D. T. et al. Usage of RePlay as a take-home system to support high-repetition motor rehabilitation after neurological injury. ReStore: a wireless peripheral nerve stimulation system. Characterization of an algorithm for autonomous, closed-loop neuromodulation during motor rehabilitation. Occa, A., Morgan, S. E. & Potter, J. N. E. Underrepresentation of Hispanics and other minorities in clinical trials: recruiters' perspectives. & Frontera, W. R. An analysis of the inclusion of women, older individuals, and racial/ethnic minorities in rehabilitation clinical trials. Guidelines for the conduct of clinical trials for spinal cord injury as developed by the ICCP Panel: clinical trial inclusion/exclusion criteria and ethics. Evaluating patient perspectives on participating in scientific research and clinical trials for the treatment of spinal cord injury. Grasse, K. M. et al. A suite of automated tools to quantify hand and wrist motor function after cervical spinal cord injury. Levi, A. D. et al. Clinical outcomes from a multi-center study of hman neural stem cell transplantation in chronic cervical spinal cord injury. Richard-Denis, A., Chatta, R., Thompson, C. & Mac-Thiong, J. M. Patterns and predictors of functional recovery from the subacute to the chronic phase following a traumatic spinal cord injury: a prospective study. Burns, A. S. & Ditunno, J. F. Establishing prognosis and maximizing functional outcomes after spinal cord injury: a review of current and future directions in rehabilitation management. Motor and sensory recovery following incomplete tetraplegia. Neurological recovery following traumatic spinal cord injury: a systematic review and meta-analysis. Kirshblum, S., Snider, B., Eren, F. & Guest, J. Characterizing natural recovery after traumatic spinal cord injury. Dawson, J. et al. Vagus nerve stimulation paired with rehabilitation for upper limb motor impairment and function after chronic ischemic stroke: subgroup analysis of the randomized, blinded, pivotal, VNS-REHAB Device Trial. Dawson, J. et al. Vagus nerve stimulation paired with rehabilitation for upper limb motor function after ischaemic stroke (VNS-REHAB): a randomised, blinded, pivotal, device trial. Morrison, R. A. et al. Vagus nerve stimulation intensity influences motor cortex plasticity. Morrison, R. A. et al. A limited range of vagus nerve stimulation intensities produce motor cortex reorganization when delivered during training. Borland, M. S. et al. Cortical map plasticity as a function of vagus nerve stimulation intensity. Pruitt, D. T. et al. Optimizing dosing of vagus nerve stimulation for stroke recovery. Liu, C. Y. et al. Vagus nerve stimulation paired with rehabilitation for stroke: implantation experience from the VNS-REHAB trial. Acute cardiovascular responses to vagus nerve stimulation after experimental spinal cord injury. Onifer, S. M., Smith, G. M. & Fouad, K. Plasticity after spinal cord injury: relevance to recovery and approaches to facilitate it. Edgerton, V. R., Tillakaratne, N. J. K., Bigbee, A. J., De Leon, R. D. & Roy, R. R. Plasticity of the spinal neural circuitry after injury. Sutton, R. S. & Barto, A. G. Reinforcement Learning: an Introduction (MIT Press, 2018). Frémaux, N., Sprekeler, H. & Gerstner, W. Functional requirements for reward-modulated spike-timing-dependent plasticity. & Lillicrap, T. P. Dendritic solutions to the credit assignment problem. Frémaux, N. & Gerstner, W. Neuromodulated spike-timing-dependent plasticity, and theory of three-factor learning rules. Hong, S. Z. et al. Norepinephrine potentiates and serotonin depresses visual cortical responses by transforming eligibility traces. & Kerr, J. N. D. Timing is not everything: neuromodulation opens the STDP gate. Dawson, J. et al. Safety, feasibility, and efficacy of vagus nerve stimulation paired with upper-limb rehabilitation after ischemic stroke. Kimberley, T. J. et al. Vagus nerve stimulation paired with upper limb rehabilitation after chronic. An objective and standardized test of hand function. We acknowledge D. Wakefield and J. French for valuable discussions and guidance on the perspective of people living with SCI; J. Weger, D. Powell, T. Odom and K. Travers for supervising physical therapy sessions; M. Ramsey, G. Cutter, S. Kirshblum and T. Chan-Leveno for assistance with trial monitoring; D. Rosenfield for assistance with experimental design and statistical analysis; N. Bleker, R. Verma, R. Chatterjee, J. Wright, P. Gonzalez, N. Friedman, I. Russell, D. Zondervan and R. Hudson for engineering support; A. Carrera, G. Chauvin, E. Turner, E. Sarker, H. Garcia, M. Adams, A. M. Warren, A. Salem, V. Warren, K. Raman, N. Rennaker, Z. Bynum, S. Kian, M. McAuliffe, S. Driver, A. Dwadasi, K. Wigginton, K. Malley, R. Hamlin and A. Muir for assistance with trial administration; and J. Hsieh and J. French for valuable comments on an earlier draft of the manuscript. This work was sponsored by Wings for Life through the Accelerated Translational Program and the Defense Advanced Research Projects Agency Biological Technologies Office TNT program under the auspices of D. Weber, T. McClure-Begley, M. Pava and J. Arthur through the Naval Information Warfare Center Pacific, Pacific grant/contract no. Michael P. Kilgard, Joseph D. Epperson, Emmanuel A. Adehunoluwa, Amy L. Porter, David T. Pruitt, Holle L. Gallaway, Jane G. Wigginton, Seth A. Hays & Robert L. Rennaker Michael P. Kilgard, Emmanuel A. Adehunoluwa & Robert L. Rennaker Baylor Scott and White Research Institute, Dallas, TX, USA Chad Swank, Jaime Gillespie, Dannae Arnold & Mark B. Baylor Scott and White Institute for Rehabilitation, Dallas, TX, USA Chad Swank, Christi Stevens, Jaime Gillespie, Dannae Arnold & Rita G. Hamilton Division of Acute Care Surgery, Baylor University Medical Center, Dallas, TX, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar coordinated randomization and programmed stimulation parameters. acquired funding to support the study. has a financial interest in MicroTransponder Inc., which markets VNS therapy for stroke. is the founder and CEO of XNerve, which developed the VNS device used in this study. The other authors declare no competing interests. Nature thanks James Guest, Teresa Kimberley and Nicolas Schweighofer for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) The study was designed to establish safety and feasibility as well as to estimate the effect of closed-loop delivery of VNS during high-intensity physical therapy compared to high-intensity physical therapy alone. After enrollment, all subjects were implanted with the VNS device. Participants were block randomized to receive 18 sessions of rehabilitation with either active CLV (1x CLV; n = 10 participants) or sham stimulation (1x Rehab; n = 9 participants) during the RCT phase, followed by an additional 18 sessions of rehabilitation with active CLV regardless of prior group assignment. This design allows comparison with sham stimulation and of two dosing regimens of CLV. (b) Depiction of change in GRASSP scores from baseline, the main functional outcome of the study, at each assessment during the RCT. Paired Student's t-test v. baseline; * indicates p < 0.05; data presented as mean ± SEM. The number of VNS events delivered during a rehabilitation sessions was not correlated with either baseline GRASSP score (a) or change in GRASSP score (b), indicating that the therapy was scaled by the level of impairment to ensure a consistent number of stimulations. (c) Additionally, stimulation rate was stable across the duration of rehabilitative sessions, indicating that the algorithm was robust against effects of fatigue or other sources of variability in performance. Data are presented as mean ± SD number of VNS events per 10-minute bin over the course of each rehabilitation session for each participant. a) Pinch force and b) knob torque, exercises performed by all participants, steadily increased over the course of therapy in the substantial majority of individuals. c-i) Similarly, strength, speed, and range of motion increased across a wide range of isometric and dynamic exercises. A linear mixed model was fitted to the data for each task. There was a significant fixed effect of therapy day for each task, as noted by the p-value on each of the figure's panels. n denotes the number of participants that performed the task as part of their individualized therapy. Performance metrics for each day are shown as dots. Thick lines indicate participants who made statistically significant increases in task performance as a function of therapy day. (j-r) Same as above, but plotted on a linear scale. (a) GRASSP score in the untrained arm was significantly improved compared to baseline in all participants after 18 sessions of CLV and in the subset of participants that received 36 sessions of CLV. No gains were observed in the group that received intensive rehabilitation with sham stimulation. Gains in the untrained arm were more modest than those observed in the trained arm. Improvement in the untrained arm was not expected because VNS-induced plasticity is specific to the paired training. However, these improvements may be explained by some amount of bilateral involvement in training, generalization, or VNS-dependent pruning of synaptic connectivity in hyperconnected networks, each of which could reduce spasticity or otherwise produce benefits to the untrained arm. Future physiological studies are needed to understand why small, but statistically significant, improvement was observed in the untrained arm. (b) Additionally, participants that receive 36 sessions of CLV exhibit improved Jebsen-Taylor Hand Function scores in the trained arm compared to baseline. (c) Though not significant in the whole population, an exploratory analysis of only motor incomplete participants demonstrates a significant correlation between change in GRASSP score and JTHF score at the end of therapy (R = 0.59; p = 0.034), potentially indicative of clinical and functional gains. Symbols (and n) indicate individual participants. For panels a and b: Paired Student's t-test v. baseline, * indicates p < 0.05. Group data are presented as mean +/− SEM. Closed-loop triggering produces bursts of VNS that coincide with the largest movements that occurred during therapy. Both therapist and an automated algorithm were able to outperform open-loop (e.g. periodic) triggering. Automated closed-loop triggering using real-time sensor data was able to outperform therapists using visual inspection to determine stimulation timing. Unpaired Student's t-test across conditions, *** indicates p < 1 × 10−17. To determine whether CLV would impact measures of cardiovascular function, we measured vital signs prior to the initiation of therapy (baseline) and after the completion of the first 18 sessions of CLV (n = 19 participants). Heart rate (a), systolic blood pressure (b), and respiratory rate (c) were not influenced by CLV. Data are presented as mean ± STD. Description of the statistical methods and outcomes from statistical testing regarding assumptions of normality and linear mixed modelling. Demographic data for each person, GRASSP subscores at 4 assessment time points, baseline forces, and forces recorded during sessions. Exercise type was customized for each participant based on their ability. Initial exercise selection was based on the GRASSP profile (Fig. Exercises were refined and progressed by licensed physical therapists to ensure tasks were consistently challenging. Participants completed computer-based exercises that were progressed by increasing the speed and precision required for success and by reducing the assistance factor so that more force or range of motion was needed. Every participant completed exercises with isometric pinch and knob inputs because all study participants exhibited reduced pinch force and knob torque. Each participant also completed exercises with real-world objects to ensure natural tactile stimulation and support skills needed for activities of daily living. During reach across exercises, participants pressed pucks in different locations while holding objects that required specific grips (e.g. key or cylindrical) to improve range of motion and grip strength. Other exercises included typing, touchscreen use, and stereognosis. Positive values reflect closing (flexion) of the index finger and thumb. Participants were given real-time feedback that was scaled to their maximum performance to enable them to see when they were producing greater than average forces. The video ends with an overlay of the forces produced before and after therapy. Positive values reflect pronation of the wrist. Negative values indicate supination of the wrist. The video ends with an overlay of the range of motion that was available before and after therapy. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Closed-loop vagus nerve stimulation aids recovery from spinal cord injury. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09041-8'>Clonal tracing with somatic epimutations reveals dynamics of blood ageing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 15:08:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Current approaches used to track stem cell clones through differentiation require genetic engineering1,2 or rely on sparse somatic DNA variants3,4, which limits their wide application. Here we discover that DNA methylation of a subset of CpG sites reflects cellular differentiation, whereas another subset undergoes stochastic epimutations and can serve as digital barcodes of clonal identity. We demonstrate that targeted single-cell profiling of DNA methylation5 at single-CpG resolution can accurately extract both layers of information. To that end, we develop EPI-Clone, a method for transgene-free lineage tracing at scale. Applied to mouse and human haematopoiesis, we capture hundreds of clonal differentiation trajectories across tens of individuals and 230,358 single cells. In mouse ageing, we demonstrate that myeloid bias and low output of old haematopoietic stem cells6 are restricted to a small number of expanded clones, whereas many functionally young-like clones persist in old age. In human ageing, clones with and without known driver mutations of clonal haematopoieis7 are part of a spectrum of age-related clonal expansions that display similar lineage biases. EPI-Clone enables accurate and transgene-free single-cell lineage tracing on hematopoietic cell state landscapes at scale. Lineage tracing using genetic or physical labels has been an important tool in developmental and stem cell biology for more than a century1,2. More recently, genetic barcoding compatible with single-cell RNA sequencing (scRNA-seq) has provided information on the cellular output of hundreds of stem cell clones together with cell-state information on the stem cell itself8,9,10,11,12. Such methods require complex genetic engineering and therefore have limited applications, for example, in humans or during native ageing. Thus, methods are needed that rely on endogenous clonal markers (for example, somatic mutations) and allow tracing of various stem cell clones in parallel. Whole-genome sequencing can reconstruct cellular phylogenies3 but has limited throughput. It also lacks information about cell states, which precludes clonal tracking across cellular differentiation landscapes. Conversely, spontaneous mitochondrial DNA (mtDNA) mutations can be captured together with cell-state information by scRNA-seq or ATAC–seq4,13,14. Although mtDNA variants can be clonally informative, it is unclear whether mtDNA variants can reconstruct cellular phylogenies15,16. Clonal signals have been identified in bulk DNA methylation data obtained from cancer and healthy tissues17,18. Somatic epimutations, defined as spontaneous but heritable losses and gains of DNA methylation, have been explored as a potential clonal label in cancer19,20. However, differentiation-associated changes in DNA methylation may mask clone-associated differences21,22. Furthermore, current single-cell DNA methylation methods23,24 suffer from data sparsity, which makes it challenging to exploit the stochasticity of epimutations at individual CpGs. A compelling case for the use of lineage tracing is haematopoiesis, whereby, in humans, 50,000–200,000 stem cell clones generate blood throughout life3. In mice, much of our understanding of clonal behaviour in ageing either comes from transplantation experiments25 or mathematical modelling26, which may not recapitulate steady-state haematopoiesis or lacks the resolution of single-cell lineage analysis. In humans, literature focuses on the role of driver mutations in clonal haematopoiesis (CH), but clonal expansions without (known) drivers are common with age and are associated with an increased all-mortality risk27. So far, the lineage output of clones with or without (known) CH driver mutations have not been compared because of a lack of suitable methods. Here we develop EPI-Clone, a method that exploits the targeted single-cell readout of DNA methylation at single-CpG resolution to track clones while providing detailed cell-state information. EPI-Clone builds on single-cell targeted analysis of the methylome (scTAM-seq), which is implemented on the Mission Bio Tapestri platform to read out methylation states of several hundred CpGs in thousands of single cells at a time, with a dropout rate of around 7%5. scTAM-seq uses a methylation-sensitive restriction enzyme to selectively digest unmethylated CpGs and thus generates sequencing reads only from methylated CpGs. We applied EPI-Clone to lineage-barcoded cells and in native human and mouse haematopoiesis to characterize the decline in clonal complexity and the functional properties of age-expanded clones in mouse and human ageing. We performed a series of experiments, which, for clarity, are defined as follows: scTAM-seq applied to eight different settings in mice (experiments M.1–M.8; Extended Data Fig. An overview of all data is provided in Supplementary Table 1. To create a ground-truth dataset of clonal identity and DNA methylation, we labelled mouse haematopoietic stem cells (HSCs) with lentiviral barcodes using the LARRY system8. Labelled HSCs were transplanted into lethally irradiated recipient mice and the mice were profiled 5 months later, a time point at which all blood populations should be reconstituted. Sorted haematopoietic stem and progenitor cells (HSPCs) from bone marrow (sorted as LIN−KIT+ (LK) cells with additional enrichment of LIN−SCA1+KIT+ (LSK) cells) were profiled by scTAM-seq (experiment M.1, the main LARRY experiment; Fig. Specifically, we analysed methylation of 453 CpGs that were selected as differentially or variably methylated from bulk HSPC DNA methylation data22 (Fig. The LARRY barcode was read out directly from the DNA by including a LARRY-specific amplicon in our targeting panel for scTAM-seq. Finally, the expression of 20 surface proteins (Supplementary Table 2) was simultaneously profiled using oligonucleotide-tagged antibodies to obtain independent information on cellular differentiation. In summary, for experiments M.1–M.3, we profiled DNA methylation at 453 CpGs and the expression of 20 surface proteins across HSPCs. In experiments M.1 and M.2, we also profiled LARRY barcodes from the same cells. b, Overview of the 453 CpGs covered by our scTAM-seq panel in mice. Variably methylated CpGs were selected from bulk whole-genome bisulfite sequencing data22. DMC, differentially methylated CpG; IMC, intermediately methylated CpG; WSH, within-sample heterogeneity (see Extended Data Fig. c, UMAP of DNA methylation data for HSPCs from experiments M.1–M.3. Colours highlight groups identified from unsupervised clustering. d, DNA methylation UMAP as in c, highlighting the average, relative methylation state of cells across all CpGs that are methylated in HSCs or MPP3/MPP4 cells in bulk-sequencing data22. e, Enrichment analysis of TFBSs near CpGs specifically unmethylated in a cell-type cluster. f, Normalized surface-protein expression of SCA1, KIT, CD135, CD201, CD48 and CD150. The CD135–CD201 and CD48–CD150 plots only show LSK cells. Colour indicates cell states per c. g, UMAP of DNA methylation data from HSPCs from experiment M.1. Colour indicates cell states per c. h, Same UMAP as in g, highlighting clones as defined from LARRY barcodes. LARRY barcodes were read out from DNA as part of scTAM-seq. The CpGs in the upper and lower central rectangle were defined as static or dynamic CpGs, respectively. j, Bar chart depicting the percentage of static and dynamic CpGs annotated as enhancer or heterochromatin. The scTAM-seq schematic in a was adapted from ref. 5 under a Creative Commons licence CC BY 4.0. We applied Seurat's default batch-correction method to integrate methylation data from 28,782 cells across the three experiments. We thereby obtained a low-dimensional embedding in which most variation was driven by differentiation along four trajectories (Fig. To annotate cell states from the DNA methylation data, we used three layers of information: (1) bulk methylation profiles (Fig. 2a); (2) the methylation states of important lineage-specific transcription-factor-binding sites (TFBSs; Fig. 2b); and (3) the expression of surface proteins (Fig. scTAM-seq data revealed a cluster of HSCs and early multipotent progenitors (MPP1, also called short-term or active HSCs), several additional MPP subsets (MPP2, MPP3 and MPP4), myeloid, erythroid and B cell progenitors, as well as two subsets of megakaryocyte progenitors (MKPs). As we also performed scRNA-seq on different cells obtained from the same samples, we could compare low-dimensional uniform manifold approximation and projection (UMAP) generated by DNA methylation with a UMAP generated from transcriptomic data (Extended Data Fig. We observed an overall similar topology (Extended Data Fig. Overall, through data integration of several experiments, we obtained a DNA-methylation-based map of mouse HSC differentiation at single-CpG resolution. This map contains two orders of magnitude more cells than two previous, single-cell bisulfite sequencing datasets of the haematopoietic system28,29. Computational batch-correction methods, by definition, remove most individual-specific signals (Extended Data Fig. As clonal information is individual-specific, we computed a UMAP display of the data from experiment M.1 only. This analysis revealed that DNA methylation jointly captures two layers of information: differentiation state and clonal identity. Specifically, although cells clustered according to differentiation states (Fig. 1g), they also clustered by their clonal identity as defined through LARRY barcodes (Fig. To use this clone-specific signal for lineage tracing, we sought to determine whether clonal identity and differentiation affect different subsets of CpGs. We tested for the association of every CpG with the expression of any surface protein and thereby identified differentiation-associated, dynamic CpGs. Performing dimensionality reduction using only these dynamic CpGs (Extended Data Fig. 2d) or only the expression of surface proteins (Extended Data Fig. 2e,f) resulted in a similar landscape to that obtained by batch correction. This finding indicates that dynamic CpGs and surface antigens independently capture differentiation state well. The remaining, static CpGs were frequently associated with clonal identity, as defined through LARRY barcodes (Fig. Dynamic CpGs were enriched in enhancer elements, whereas the static CpGs were preferentially located in heterochromatic regions (Fisher test P = 2.2 × 10−5; Fig. Moreover, static CpGs were enriched in late-replicating domains (Fisher test P = 0.001; Extended Data Fig. In summary, clonal identity and differentiation state affect the methylation of different sets of CpGs in haematopoietic cells, which creates a valuable tool to read out both processes simultaneously at the single-cell level. We focused on exploiting static CpGs to analyse clonal identity. To this end, we developed EPI-Clone, which is divided into three steps: (1) identification of static CpGs, as described above; (2) identification of cells from expanded clones by using cell density in the DNA methylation space defined by the static CpGs; and (3) clustering of cells from the expanded clones (Fig. b, UMAP of DNA methylation computed on static CpGs only for experiment M.1, which highlights clonal identity as defined by LARRY barcodes. Only cells carrying a LARRY barcode are shown and cells with a relative clone size (rel. size; defined using LARRY) less than 0.25% are shown in grey. c, Same UMAP as in b, but highlighting the cell states as defined in Fig. d, UMAP highlighting cells that were selected as part of expanded clones based on local density in PCA space. e, Receiver-operating characteristics curve visualizing the performance of classifying cells into expanded and non-expanded clones based on local density in PCA space spanned by the static CpGs. LARRY clone size was used as the ground truth, whereby clone sizes larger than 0.25% were considered expanded. f, Heatmap depicting the overlap between LARRY barcode and methylation-based clonal clusters identified by EPI-Clone. The row labelled with an asterisk contains all LARRY clones with a clone size less than 0.25%. h, UMAP of DNA methylation for cells from expanded clones in experiment M.5. The static CpGs identified from experiment M.1 were used. i, Same UMAP representation as in h, but highlighting the cell-state annotation as defined in Supplementary Fig. Of note, most of the clones identified using EPI-Clone were specific for T cells, B cells or myeloid cells, in line with the result from LARRY (Supplementary Fig. j, ARI values between the ground-truth clonal label (LARRY) and the clones identified by EPI-Clone stratified by cell type. Using this algorithm, expanded LARRY clones with relative clone sizes larger than 0.25% clustered separately, with no influence of cell state (Fig. By contrast, cells from small LARRY clones with relative sizes less than 0.25% were interspersed between clusters (Fig. EPI-Clone identified cells that belong to expanded clones on the basis of the high local density in principal component analysis (PCA) space spanned by the static CpGs (Fig. EPI-Clone correctly identified cells from expanded clones with an area under the receiver operating characteristic curve (AUC) of 0.79 when using the LARRY clone sizes as ground truth (Fig. Subsequently, EPI-Clone clustered cells from expanded clones by clonal identity, achieving an adjusted rand index (ARI) of 0.88 relative to ground-truth LARRY barcodes (Fig. Quantitatively and qualitatively similar results were obtained from a biological replicate that used the same parameters and cut-off values in the EPI-Clone analysis (Extended Data Fig. These results demonstrate that epimutational clonal signals are stably maintained in blood stem and progenitor cells over long periods of time (5 months from transplant to analysis). We next asked whether EPI-Clone can determine clonal identity in mature immune cells. To that end, we collected mature immune (lymphoid and myeloid) cells from bone marrow and spleen (experiment M.5; Fig. Using the static CpGs defined from experiment M.1, EPI-Clone again produced clonal clustering that recapitulated ground-truth clonal labels (Fig. We separately computed ARI values between EPI-Clone results and LARRY barcodes. ARI values were higher than 0.7 for monocytes, neutrophils, other myeloid cells, CD8+ T cells and one B cell subset, higher than 0.4 for CD4+ T cells and low for macrophages and a second B cell subset (Fig. 4d), which implicated an origin in lymphoid-biased or restricted progenitors30. In a separate experiment (M.4), we profiled mature myeloid cells from lung, bone marrow and peripheral blood, and found that myeloid cell types, except macrophages, retained this clonal mark also outside of the bone marrow (Extended Data Fig. These results show that clonal information encoded in the DNA methylation state is maintained in most lineages until terminal differentiation and 10 months after the lentiviral labelling event (Discussion). Finally, we asked whether EPI-Clone can be applied to tissues other than blood. We used the same CpG panel to sorted endothelial cells (ECs) from lung of an aged mouse. ECs share a common developmental origin with blood (experiment M.6; Extended Data Fig. Using the dynamic CpGs defined in haematopoiesis and CD31, SCA1 and podoplanin protein-expression information, we identified two previously described types of capillaries and lymphatic ECs31 (Extended Data Fig. Using the same set of static CpGs as in haematopoiesis, EPI-Clone revealed cell-state-independent, yet statistically supported, clusters containing all three cell types (Extended Data Fig. In summary, DNA methylation patterns at static CpGs constitute a broadly applicable clonal barcode. EPI-Clone can provide joint information on the cell state of progenitors, clonal identity and clonally derived progeny. Therefore, it is an ideal method to characterize the clonal dynamics of native (unperturbed) haematopoiesis. In contrast to the transplantation setting, native haematopoiesis has been described as polyclonal32,33, whereby several thousand clones contribute to blood formation. To investigate whether EPI-Clone also identifies clones in native haematopoiesis, we applied it to bone marrow samples from two untreated, young mice (experiment M.7, 12 weeks old; Supplementary Fig. These clone sizes are in line with a study that genetically barcoded adult haematopoietic clones in situ33 (Fig. The remaining cells were classified as belonging to small and non-expanded clones. A limitation of EPI-Clone is that only cells belonging to expanded clones can be assigned to their clone of origin. Cells belonging to very small clones (<0.25% of cells after transplant and <1% in native haematopoiesis) could be identified as not belonging to expanded clones, but their clonal identity could not be inferred with the cell numbers used here. a, DNA methylation UMAP based on the static CpGs for a native, young (12 weeks old) mouse from experiment M.7. b, DNA methylation UMAP based on the static CpGs for an old mouse (100 weeks old). In a and b, three outlier clusters with size <1% were removed to improve visualization. c, Comparison of clone sizes for old and young mice (two biological replicates), and a young mouse from a previous study33. Clones with a relative size less than 1% are shown in grey. e, Bubble plot visualizing the frequency of HSC/MPP1 cells per clone for old and young mice. f, Differentiation UMAP defined on the basis of dynamic CpGs, highlighting example clones with different behaviour for old and young mice. For a, b, e and f, data from replicate 1 is shown, see Supplementary Fig. 7 for replicate 2. g, Comparison of the ratio between lymphoid and myeloid output per clone identified using EPI-Clone. P values calculated using two-sided Wilcoxon tests. h, Experimental design for the transplantation experiment (M.8). i,j, Boxplots of post-transplant clone sizes, comparing clones with different pre-transplant differentiation bias calculated as the ratio of mature versus immature cells per clone (i) and different pre-transplant immature clone sizes (j). Tertile T1 has the lowest mature output (i) and smallest clone size (j). k, Boxplot showing the distribution of pairwise cosine observed (Obs.) distances (before and after transplant) computed using the cell-type distribution of each clone. Observed data are compared with a null model created by randomly shuffling the clonal identities of post-transplant clones (1,000 times). P values of i–k are from two-sided Wilcoxon tests. For d,e,g and i–k, see the section ‘Data visualization' in the Methods for a definition of boxplot elements and further detail. The scTAM-seq schematic in h was adapted from ref. 5 under a Creative Commons licence CC BY 4.0. We next applied EPI-Clone to study ageing by comparing the data from young mice (12 weeks old) to 100-week-old mice in two biological replicates (experiment M.7; Fig. We observed weak shifts in cell-type proportions between the young and the old mice, a result that confirmed previous observations34 (Supplementary Fig. Expanded clones in the old mice were individually also larger than in the young mice (Fig. This gradual loss of clonality with age resembles certain properties of human HSC ageing (see below). Next, we measured the distribution of cell types for each clone across the various stem and progenitor clusters. In the old mice, we observed several expanded clones that contained mostly HSCs across both of our replicates (Fig. 7d,e; Kolmogorov–Smirnov test P < 0.05), which were not present in the young mice. These HSC-expanded clones contained large numbers of stem cells apparently incapable of proceeding with differentiation and contained little progeny. Old mice showed a moderate increase in the number of myeloid-biased clones, which was in contrast to results from classical transplantation experiments35,36,37,38 (Fig. However, the rare HSC-expanded clones were mostly myeloid-biased (Fig. To determine the long-term stability of the HSC-expanded clonal behaviour, we performed a transplantation assay using an aged donor mouse. We used EPI-Clone to compare the clonal composition of the haematopoietic system in the native state (before transplant) and after transplant, and used LARRY barcoding as an additional control during transplantation (experiment M.8; Fig. Clonal identities defined using EPI-Clone remained stable during transplantation (Extended Data Fig. HSCs with abundant progeny before transplant showed poor engraftment, a result in line with serial transplantation studies using lentiviral barcoding8,33 (Fig. Notably, HSC-expanded clones also engrafted poorly, and we identified non-expanded HSCs as the major drivers of transplantation haematopoiesis (Fig. Clones with quantifiable output before and after transplant showed a stable lineage bias that was inherited after transplantation (Fig. In summary, our data demonstrate age-related loss of clonal complexity in mouse ageing that is accompanied by an emergence of HSC-expanded clones with low engraftment capacity. We propose that these rare but expanded clones drive the increase in stem cell number and decrease in output that had typically been associated with aged haematopoiesis in transplantation studies39,40,41 and in Cre-lox-based native lineage-tracing studies42. Our transplant data support the idea that HSCs that do not expand with age persist and drive regeneration. To relate these results to human ageing, we next adapted EPI-Clone for use on human samples. We designed a panel that targeted 448 CpGs with variable methylation between or within blood progenitor populations (Methods and Extended Data Fig. We also included 147 genomic regions commonly mutated in CH and 20 regions that targeted chromosome Y to serve as a partial ground truth for clones identified by EPI-Clone. We also assembled a dataset of CD34+ cells from bone marrow from nine donors (donors B.1–B.5 and X.1, and donors A.1, A.3 and A.4, for whom >1,000 CD34+ cells had been captured from TBM) (Fig. Three of the TBM donors had previously been characterized for CH mutations43, and we de novo identified CH mutations or loss of the Y chromosome (LoY) for four additional donors from scTAM-seq data (Methods). Samples were stained with an antibody panel targeting 45 surface proteins to provide phenotypic characterization. Across all donors, we profiled 135,432 single cells using scTAM-seq. a, Summary of donor characteristics (Supplementary Table 1). Dots connected by dashed lines denote samples that were analysed as part of the TBM and the CD34+ dataset. b, Integrated UMAP of dynamic CpG and surface-protein data for all donors from the TBM and CD34+ datasets. Cell states were annotated based on the expression of surface proteins (Extended Data Fig. c, UMAPs computed per donor on a consensus set of static CpGs, highlighting cells containing the specified CH mutations. 7f–h and Methods for how consensus static CpGs were identified. The donors are sorted by increasing age. d, UMAPs as in c, highlighting clones identified using EPI-Clone. The identified clones (x axis) are sorted by size. Dots in colours correspond to the clones dominated by a CH mutation, see c for colour scheme. f,g, Scatter plot relating donor age (f) and the presence of GMPs (g) to the number of clones identified by EPI-Clone in the TBM cohort and CD34+ cohort, respectively. P value calculated with a two-sided t-test computed from a generalized linear model of the Poisson family, using the number of cells observed as a weight. Dot size denotes the number of cells analysed (see b for a scale). h, Boxplot depicting clone sizes stratified into clones carrying CH mutations and clones for which no CH mutation was identified. We followed the same analytical strategy as for the mouse experiments, but with minor adaptations (Methods). Specifically, we detected expanded clones using a statistical criterion (CHOIR44; Extended Data Fig. We then used all myeloid cells to identify a consensus set of static CpGs across individuals (Extended Data Fig. To assess the fidelity of static CpGs to identify clones, we exploited the CH mutations and LoY events as a clonal ground-truth. Quantitatively, the epimutational clones dominated by CH mutant cells were on average 78.8% mutant and those dominated by wild-type cells were on average 95.4% wild-type (Fig. These numbers probably underestimate the true overlap between the identified clones and CH clones owing to allelic dropout of CH mutations. We observed a stronger separation of clones identified using our algorithm and better overlap with CH mutations in older donors than in young donors. This result suggests that EPI-Clone most accurately identifies clones in haematopoietic systems of reduced clonal complexity. Besides the CH clones, EPI-Clone identified a total of 67 other clonal expansions in the seven TBM donors, a result that highlights the capacity of this algorithm to recapitulate clonal expansions driven by known and unknown drivers. We included natural killer (NK) cells and immature B cells in our analysis and used CH mutations to validate that these cells also clustered by clone (Extended Data Fig. When T cells and mature B cells were included, they associated with lymphoid-dominant clusters, a finding in line with the results from mice (Fig. 8e) and indicating their distinct clonal origins compared with the other cells. In donor A.4, in whom a large CH clone contributed to T cells, mutant T cells clustered with the remaining CH-derived cells (Extended Data Fig. Together with the results from the mouse LARRY experiment, this finding constitutes evidence that the identified clones remain stable from HSCs to myeloid, T cells, NK cells and immature B cells. To establish a conservative estimate for a minimum clone size of EPI-Clone in humans, we determined the smallest CH clone identified using this method. The clone DNMT3A(C666Y) in donor A.4 had 145 cells or a relative size of 1% in the myeloid compartment. Furthermore, we observed that several large CH clones (for example, DNMT3A(R659H) in donor A.4; Fig. 8) had diversified into two clones with a similar but distinguishable static CpG profile. This result suggests that over decades, epimutations can continue to accrue phylogenetic information. In conclusion, these analyses demonstrate the ability of EPI-Clone to identify expanded haematopoietic clones of a wide range of sizes in human bone marrow and blood. We leveraged the ability of EPI-Clone to trace both CH clones, which are well characterized in humans28,43, and clones without known driver mutations (non-CH clones) to functionally compare these two types of clonal expansions in our TBM and CD34+ cohorts. Owing to their putatively distinct clonal origins, we excluded T cells and mature B cells from this analysis. As expected3, in the TBM cohort, we observed an age-dependent accumulation of expanded CH and non-CH clones (Fig. CH clones tended to be more expanded than non-CH clones, but were not always among the largest ones (Fig. Expanded clones were significantly depleted (compared with cells from non-expanded clones) from the B cell and erythroid lineages (Fig. 8f), which implicated a link between myelopoiesis and expansion even for non-CH clones. Compared with non-CH clones, CH clones were significantly enriched in HSCs and MPPs but depleted from the B cell and erythroid lineages (Fig. These results highlight a stem-cell bias in age-expanded clones that is conserved across mice and humans, and they support a model whereby CH clones are part of a spectrum of such age-expanded clones. a, Scatter plot depicting the fraction of immature B cells per clone relative to the fraction of immature B cells in non-expanded clones from the same patient. Grey dots are clones with no known driver mutation, dots in colour are clones with a CH mutation (see Fig. b, Dot plot depicting P values for enrichments and depletion of cell types in expanded versus non-expanded and CH versus non-CH clones. For this analysis, cell-type composition of clones (for example, the percentage of clone CD34+) were transformed using a logit transform and P values were computed using a mixed-effect model, using donor as a random effect and clone type (expanded or non-expanded or CH or non-CH) as a fixed effect (Extended Data Fig. d, Clones discovered using EPI-Clone were identified on CD34+ cells from donor X.1 using DNA methylation data. Subsequently, genes with differential expression between clones and correlation with the percentage of HSC/MPPs in the clone were identified. Adjusted P values were calculated using two-sided tests for Pearson correlation, adjusted for multiple testing. e, Schematic of experiment X.2 (scTAMito-seq; Extended Data Fig. f, Scatter plot depicting the presence of six mitochondrial variants in the different clones identified using EPI-Clone from X.2. Cells were scored as positive for the variant if at least 5% of reads supported the variant. The identified clones were classified as B cell, T cell or NK cell clones if at least 80% of cells were from a single lineage or as multilineage clones otherwise. To resolve transcriptional differences between clones in the HSC and MPP (HSC/MPP) compartment, we added targeted RNA-seq to the scTAM-seq protocol (single-cell targeted analysis of the methylome and RNA (scTAMARA-seq); Fig. To that end, we combined SDR-seq45, a recently described targeted RNA-seq protocol for Mission Bio Tapestri, with scTAM-seq. We profiled one of the CD34+ bone marrow samples (X.1) and obtained high-quality DNA methylation and RNA-seq data from 2,745 cells (Extended Data Fig. scRNA-seq data confirmed the accuracy of DNA-methylation-based cell-state annotation and showed an increased resolution of transcriptomic data at the level of erythromyeloid progenitors (Extended Data Fig. We then investigated the gene-expression pattern of distinct clones. HSC/MPP-biased clones expressed low levels of TAL1, SLC40A1 and CDC45 at the HSC/MPP level and high levels of CEBPA, which suggested that clonal fate biases are correlated with gene-expression changes at early stem and progenitor states (Fig. In the field, there is controversy regarding the potential of other somatic events, in particular low-heteroplasmy mtDNA variants, for lineage tracing14,15,16. To perform a direct experimental comparison, we used EPI-Clone to analyse peripheral blood from a 38-year-old healthy donor (X.2) that had previously been characterized by a state-of-the-art single-cell mitochondrial lineage tracing method, mt-scATAC-seq13,46. We identified 44 clones from this sample, which displayed prominent clonal expansions of NK cells and T cells (Extended Data Fig. By including a mitochondrial targeting panel into scTAM-seq, we achieved a median coverage of 176 reads per cell on the mitochondrial genome (Fig. Of the 23 mtDNA variants previously identified46 (Supplementary Table 3) in this donor and covered in scTAM-seq, 5 had clear phylogenetic relationships with the clones identified using EPI-Clone. That is, they were either subclones of single clones or were parental to several of the identified clones (Fig. 5f), and one variant was observed in two clones. A highly abundant variant (mt:7076A>G) was strongly enriched or depleted in 17 T cell or NK cell clones identified using EPI-Clone, but was observed in approximately 50% of cells of the remaining, mostly multilineage or B cell, clones identified (Fig. This variant was probably present before epimutational patterns were established and repeatedly underwent selection throughout development and adulthood. Therefore, T cell clones with a recent history of expansion may or may not carry the variant, whereas multilineage clones that expanded before selection of the variant contain a mix of mutant and wild-type cells. Finally, the remaining 16 low-heteroplasmy mitochondrial variants did not segregate with clones identified using EPI-Clone (Extended Data Fig. These findings are in line with a recent report15 observing that only some observed mitochondrial variants carry phylogenetic information, and illustrate the complexity of mitochondrial genetics, for which selection of variants can happen repeatedly during differentiation46. In summary, DNA methylation at a few hundred CpGs is sufficient to simultaneously identify clones and cell states of haematopoietic cells and ECs, whereas individual CpGs are either informative of cell states or clones. Somatic epimutations seem to be a stable, long-term lineage tracer. Indeed, 5–10 months had elapsed between introduction of the ground-truth clonal label and collection of cells after transplantation. In humans, previous studies have indicated that decades pass between the initial acquisition of CH or LoY and the observation of expanded clones in age3. This result raises the question of where and how clonal epimutations arise. We found that they randomly occur but remain stable over many cell divisions. Moreover, their numbers do not increase during differentiation (Supplementary Fig. We propose that some developmental events that are characterized by rapid cellular proliferation and/or a remodelling of the DNA methylome, such as the specification of HSCs47, essentially randomize the methylation state of CpGs in heterochromatic and late-replicating regions. A potential explanation of this effect is that in rapidly dividing cells, DNMT1 may not act sufficiently to copy the DNA methylation state to the nascent DNA strand (Supplementary Fig. Consistent with this idea, a recent study of bulk methylome profiles from blood cells in monozygotic twins suggested that clone-associated variation of the methylome may be established during embryonic development48. We therefore propose that variably methylated CpGs in non-regulatory genomic regions can act as a digital barcode of clonal origin. The digital and stochastic nature of epimutations makes single-cell methods that are capable of mapping the methylation state of single CpGs at high confidence, such as scTAM-seq, a powerful tool for lineage tracing. While this article was under review, a method termed MethylTree49 demonstrated identification of clonal identity from sparse whole-genome, single-cell DNA methylation data. Conversely, scTAM-seq requires the design of a species-specific targeting panel. The robustness of EPI-Clone is best evidenced by its capacity to identify high-resolution clonal patterns in native haematopoiesis. We demonstrated that both native human and mouse haematopoiesis shifts from highly polyclonal to oligoclonal blood production, and we investigated clone function in these two species using a coherent, unified method. Expanded clones in mice tended to be more numerous, but individually smaller, and poorly contribute to haematopoiesis in transplants. This observation seems to be in line with the larger and more polyclonal stem cell compartment in humans, but a much longer period of clonal selection and drift. In our human data, oligoclonal blood production became detectable at an age of around 50 years and manifested itself as an inevitable and potentially clock-like process after the age of 60 years. Our data further put CH mutations into a perspective with clonal expansions without known drivers. That is, CH clones are more strongly biased towards the myeloid lineage and towards an expansion of stem cells, but together with non-CH clones form part of a spectrum of age-related clonal expansions that display similar functional properties. In aged mice, we similarly detected large HSC-expanded clones that had reduced regenerative capacity. Together with recent transplantation studies of human HSCs50, this result suggests that there is conservation of the processes that drive haematopoietic ageing and decline in clonal complexity, and it highlights that CH mutations might not be the main driver of this process. Epidemiological studies have demonstrated an increased mortality risk in carriers of driver-free expanded clones27. These results call for a broader investigation of age-related decline in clonality instead of a strict focus on CH. An overview of all experiments performed for this study is included in Extended Data Fig. For the mouse experiments with an available ground truth from the LARRY lentiviral barcoding system (experiments M.1, M.2, M.4, M.5), LARRY barcoding vectors were constructed and lentiviruses were produced (see the section ‘Lentiviral barcoding using the LARRY system'). Stem cells were then collected from mice, transduced with the LARRY lentiviruses and transplanted, and different cellular compartments were collected 5–10 months later for profiling by scTAM-seq (see the section ‘Experimental procedures (mouse study)'). Additional experiments were performed on biological material from non-treated mice of different ages (experiments M.3 and M.6–M.8; see the section ‘Experimental procedures (mouse study)'). For the human study, primary bone marrow samples were analysed (see the section ‘Experimental procedures (human study)'). All biological material was analysed by scTAM-seq5 (see the section ‘Single-cell DNA methylation profiling with scTAM-seq'). scTAM-seq is a targeted method for DNA methylation analysis based on the Mission Bio Tapestri platform. Specifically, up to 1,000 amplicons 200–400 base pairs in length are amplified from the genomes of single cells. Before this amplification step, scTAM-seq includes a digestion step with a methylation-sensitive restriction enzyme, HhaI. Therefore, CpG dinucleotides in HhaI sites are only effectively amplified if methylated. The selection of the target amplicons comprising individual CpGs is a crucial step in this protocol (see the sections ‘Mouse panel design for scTAM-seq' and ‘Human panel design for scTAM-seq'). Relevant genetic information (LARRY barcodes or CH mutations) can be read out from gDNA by scTAM-seq in the same cells, specifically by covering these regions with amplicons not containing HhaI cut sites. We also included dedicated experiments demonstrating the combination of scTAM-seq with RNA-seq from the same single cell (experiment X.1, see the section ‘Combined profiling of DNA methylation and RNA in the same cell') or mitochondrial genome sequencing (experiment X.2, see the section ‘Combined profiling of DNA methylation and mitochondrial variants'). Key steps in the data analyses (see the section ‘Bioinformatic analysis (mouse)') were to define cell states through data integration and subsequently to identify clones using the EPI-Clone algorithm. This algorithm first identifies CpGs with no surface-antigen association as potentially clone-associated or ‘static' CpGs, and subsequently performs clustering and dimensionality reduction exclusively on these CpGs (see the section ‘EPI-Clone'). Additional steps and adjustments included mutation calling and definition of a consensus set of static CpGs across donors (see the section ‘Bioinformatic analysis (human)'). Barcode libraries were constructed according to a previously established protocol (https://www.protocols.io/view/barcode-plasmid-library-cloning-4hggt3w). First, the T-Sapphire or eGFP coding sequences and the EF1a promoter sequence were PCR-amplified from pEB1-T-Sapphire and pLARRY-eGFP with primers homologous to the vector insertion site in a custom lentiviral plasmid backbone (Vectorbuilder) using Gibson assembly (Gibson assembly master mix, NEB, E2611L). After magnetic bead purification, ligated vectors were transformed into NEB10-beta electroporation ultracompetent Escherichia coli cells (NEB 10-beta electrocompetent E. coli, NEB, C3020K) and grown overnight on LB plates supplemented with 50 μg ml–1 carbenicillin (carbenicillin disodium salt, Thermo Scientific Chemicals, 11568616). Colonies were scraped using LB medium and pelleted by centrifugation. Plasmid maxipreps were performed using an Endotoxin-Free Plasmid Maxi kit (Macheray Nagel), following the manufacturer's protocol. pEB1-T-Sapphire was a gift from P. Cluzel (Addgene plasmid 103977). pLARRY-eGFP was a gift from F. Camargo (Addgene plasmid 140025). To barcode pLARRY plasmids and generate a library, a spacer sequence flanked by EcoRV restriction sites was cloned into the plasmid after the WPRE element of the vector. Custom PAGE-purified single-strand oligonucleotides with a pattern of 20 random-bases (GTTCCANNNNTGNNNNCANNNNGTNNNNAGNNNN) and surrounded by 25 nucleotides homologous to the vector insertion site were synthesized by IDT DNA Technologies. Six electroporations of the bead-purified ligations were performed into NEB10-beta E. coli cells (NEB 10-beta electrocompetent E. coli, New England Biolabs, C3020K) using a Gene Pulser electroporator (Bio-Rad). After incubation, the transformed cells were plated in six large LB–ampicillin agar plates overnight at 30 °C. Colonies from all six plates were collected by scraping with LB–ampicillin and then grown for an additional 2 h at 225 r.p.m. Cultures were pelleted by centrifugation, and plasmids were isolated using an Endotoxin-Free Plasmid Maxi kit (Macheray-Nagel), following the manufacturer's protocol. For estimating diversity, LARRY barcode amplicon libraries were prepared by PCR amplification of the lentiviral library maxiprep using flanking oligonucleotides carrying TruSeq read1 and read2 adaptors using 10 ng of the library (Supplementary Table 4). We used the minimal number of cycles that we could detect by quantitative PCR to avoid PCR amplification bias (10–12 cycles). After bead purification, 10 ng of the first PCR product was used as a template for a second PCR to add Illumina P5 and P7 adaptors and indexes (Supplementary Table 4). Two independent PCRs were sequenced on an Illumina NovaSeq 6000 S4 platform (Novogene) to confirm diversity after correction of errors through collapsing with a Hamming distance of 4. After collapsing, libraries were confirmed to contain at least 50 million different barcodes, with enough diversity for uniquely labelling up to 100,000 HSCs with a minimal false-positive rate. Lentivirus production and HSPC transduction were performed as previously described8. All procedures involving animals adhered to the pertinent regulations and guidelines. The study followed all relevant ethical regulations. CD45.1 (CD45.1, B6.SJL-Ptprca Pep3b/BoyJ, 002014, The Jackson Laboratory) mice were used as transplantation recipients for CD45.2 (BL6/J) donor cells. Mice were kept under specific-pathogen-free conditions for all experiments. We used 12–100-week-old male and female mice for our experiments. Experiments were performed with one or two biological replicates of mice, and no statistical methods were used for sample size choice. To minimize distress, euthanasia was performed by administering isoflurane inhalation, followed by cervical dislocation to ensure the animals were fully deceased. The collected bone marrow cells were then sieved through a 40 μm strainer and cleansed with a cold ‘Easy Sep' buffer containing PBS, 2% FBS, 1 mM EDTA and penicillin–streptomycin followed by lysis of red blood cells using RBC lysis buffer (BioLegend, 420302). At first, mature lineage cells were selectively depleted using a Lineage Cell Depletion kit, mouse (Miltenyi Biotec, 130-110-470), and the resulting LIN– (lineage-negative) fraction was then enriched for KIT expression using CD117 MicroBeads (Miltenyi Biotec, 130-091-224). For transplants, EPCR+LIN–SCA1+KIT+CD48–CD150+ HSCs were sorted by FACS with a BD FACSAria Fusion with a 70 µm nozzle. In vitro cultures of HSCs were done under self-renewing F12-PVA-based conditions as previously described52. To culture HSCs, 96-well flat-bottom plates from Thermo Scientific were coated with a layer of 100 ng ml–1 fibronectin (bovine fibronectin protein, 1030-FN) for 30 min at room temperature. After the sorting process, HSCs were transferred into 200 µl complete HSC medium supplemented with 100 ng ml–1 recombinant mouse TPO (PeproTech Recombinant Murine TPO, 315-14) and 10 ng ml–1 recombinant mouse SCF (PeproTech Recombinant Murine SCF, 250-03) and grown at 37 °C with 5% CO2. Three days after labelling, the cultured HSCs were collected and subsequently transplanted into conditioned CD45.1 mice. The CD45.1 recipient mouse was preconditioned with a lethal X-ray radiation dose, administered as two separate sessions amounting to 5 Gy each, with a 4-h interval between them. All mice demonstrated stable long-term engraftment until the experimental end point. Engraftment analysis, along with the measurement of labelling frequency, was carried out using BD FACS Fusion. In all single-cell experiments, unless described otherwise in the subsequent sections, transplanted or untreated mice were euthanized at specified ages and time points after transplant, and a KIT-enriched cell fraction was isolated from the femur, tibia, pelvis and sternum, per the protocol described above. This KIT-enriched cell population was stained with FcX block to prevent nonspecific binding and subsequently stained again with the following panel of fluorescently labelled antibodies: APC anti-mouse CD117 (clone ACK2, BioLegend, 105812); PE/Cy7 anti-mouse Ly6a (SCA1) (BioLegend, 108114); and Pacific Blue anti-mouse Lineage cocktail (BioLegend, 133310). In all mouse experiments, cells were also labelled with a custom TotalSeq-B antibody cocktail (Supplementary Table 2). After staining, distinct cellular compartments were sorted as illustrated in Supplementary Fig. For M.1, two donor mice were killed, and HSCs were labelled with LARRY constructs containing a GFP label in one case and LARRY constructs containing a Sapphire label in the other case. Accordingly, the dataset contains cells from four mice that contain two sets of clones, labelled with GFP and Sapphire, respectively. GFP and Sapphire clones did not mix on EPI-Clone UMAPs (Extended Data Fig. 3f), which further demonstrates that clones identified using EPI-Clone are individual-specific. We profiled all four recipient mice after allowing full blood reconstitution over 5 months. We also repeated this experiment again for validating the computational method (experiment M.2) using only one donor mouse. For both experiment M.1 and experiment M.2, we collected LSK and LK cells from the bone marrow and mixed them at 60,000 (LK) plus 50,000 (LSK) before analysing the cells using the Tapestri platform (Supplementary Table 1). 1), we killed a 12-week-old wild-type BL6/J (CD45.2) mouse, extracted 120,000 LK cells and subjected them to scTAM-seq (Supplementary Table 1 and Supplementary Fig. For profiling tissue-resident myeloid cells (experiment M.4; Extended Data Fig. 4), a single LARRY-transplanted mouse was anaesthetized 10 months after transplantation and perfused. Subsequently, lungs were extracted from the chest cavity, and a single-cell suspension was prepared using a protease and DNAse solution from a Lung Dissociation kit (Miltenyi Biotech, 130–095-927) followed by mechanical dissociation using gentleMACS ‘C' columns (Miltenyi Biotech, 130–093-237) according to the manufacturer's instructions. The dissociated cells were filtered using a 70 μm strainer and centrifuged at 400g for 5 min at room temperature. The supernatant was removed by aspiration and red blood cell lysis was performed using RBC lysis buffer (BioLegend, 420302). The supernatant was removed, and the pellet was resuspended in FACS buffer before being passed through a 40 μm strainer and stained for the mature myeloid cell marker. Cells were stained with the following fluorescently labelled antibodies: PerCP/Cyanine5.5 anti-mouse/human CD11b (BioLegend, 101227; clone M1/70) and PE/Cyanine7 anti-mouse CD45.2 (BioLegend, 109829; clone 104). Cells were also labelled with TotalSeq-B antibody cocktail. We then sorted CD45.2+CD11b+LARRY(GFP)+ immune cells from lung. 4), a single LARRY-transplanted mouse was euthanized 5 months after transplantation, and cells from the spleen and bone marrow were collected as described above. After red blood cell lysis, equal amounts of cells from both organs were pooled, washed and blocked with FcX. For profiling ECs from 100-week-old mice (experiment M.6; Extended Data Fig. 5), dissociated lung cells were collected as described above. The resultant cell population was then enriched for CD31 expression using CD31 MicroBeads (mouse, 130-097-418, Miltenyi Biotec) per the manufacturer's guidelines. These CD31-enriched cells were then washed, blocked with FcX and stained with the following fluorescently labelled antibodies: PE anti-mouse CD31 (BioLegend, 102507; clone MEC13.3) and PE/Cyanine7 anti-mouse CD45.2 (BioLegend, 109829; clone 104). Following staining, CD31+ and CD45.2– cells were sorted as illustrated in Extended Data Fig. 5–7), the KIT-enriched cell fraction was stained and subsequently sorted to collect LSK and LK populations as described above. 6), half of the HSC population from a 100-week-old mouse that was profiled as part of experiment M.7 were labelled with LARRY lentivirus and transplanted into lethally irradiated mice. Six months after transplant, mice were euthanized, and a KIT-enriched cell fraction was isolated from the femur, tibia, pelvis and sternum, following the protocol outlined above. This KIT-enriched cell population was stained with FcX block to prevent nonspecific binding and subsequently stained again with the following panel of fluorescently labelled antibodies: APC anti-mouse CD117 (clone ACK2, BioLegend, 105812); PE/Cy7 anti-mouse Ly6a (SCA1) (BioLegend, 108114); and Pacific Blue anti-mouse Lineage cocktail (BioLegend, 133310). After staining, LK and LSK cells were sorted as described above. Bone marrow samples were obtained from different sources. Samples A.1, A.6 and A.7 were bone marrow aspirates from healthy volunteers collected at the Heidelberg University Hospital after informed written consent. This study was approved by the Ethics Committee of the Medical Faculty of Heidelberg University (S-480/2011). Sample A.4 was a TBM sample obtained through the Banc de Sang i Teixits (Barcelona, Spain) and approved by the Ethics Committee of the Hospital Clinic de Barcelona (HCB/2023/0367). No genomic characterization was performed on these samples before this study. Samples A.2, A.3 and A.5 were collected after informed written consent from individuals undergoing elective total hip replacement surgery at the Nuffield Orthopaedic Centre under the ‘Mechanisms of Age-Related Clonal Haematopoiesis' (MARCH) study. This study was approved by the Yorkshire and The Humber–Bradford Leeds Research Ethics Committee (NHS REC ref: 17/YH/0382). These samples were screened for somatic mutations with a variant allele frequency of ≥0.01 by targeted DNA sequencing of a panel covering 97 genes (347 kb) recurrently mutated in myeloid malignancies and CH, as previously described43. Samples with somatic mutations in DNMT3A and PPM1D were selected for analyses. was a peripheral blood sample collected and characterized by mt-scATAC–seq as previously described46. Informed consent was given and approved for genomics profiling by the Stanford Institutional Review Board (number 14734). All experiments involving human samples were approved by the corresponding ethics committees and were in accordance with the Declaration of Helsinki. Bone marrow samples were thawed and stained using CD34 and CD3 sorting antibodies (BioLegend, 343517) and a pool of oligonucleotide-conjugated antibodies from the TotalSeq-D Heme Oncology Cocktail from BioLegend (MB53-0053) as well as additional TotalSeq-D antibodies from BioLegend (Supplementary Table 6). For details on sorting, see Supplementary Table 1. Chromosome Y and pre-characterized somatic single-nucleotide variants (SNVs) were used as controls (Supplementary Fig. 10 and see the section on ‘Bioinformatic analysis (human)'). For profiling DNA methylation at single-cell resolution, we used scTAM-seq5, which leverages the Mission Bio Tapestri technology to investigate up to 1,000 CpGs in 1,000s of cells per experiment. In brief, we loaded 120,000–140,000 cells into the Tapestri machine and followed the default Mission Bio DNA+Protein protocol for V2/V3 chemistry for the experiments (v.2: https://missionbio.com/wp-content/uploads/2021/02/Tapestri-Single-Cell-DNA-Protein-Sequencing-V2-User-Guide-PN_3360A.pdf; v.3: https://missionbio.com/wp-content/uploads/2023/08/Tapestri-Single-Cell-DNA-Protein-Sequencing-v3-User-Guide_MB05-0018.pdf; see also Supplementary Table 1), but with the following modifications: (1) we added a DNA methylation-sensitive restriction enzyme (HhaI) to digest non-methylated targets (CpGs) before amplification; and (2) in the case of the mouse experiments, we used TotalSeq-B antibodies and different primers for the amplification of antibody oligonucleotide tags. The default Mission Bio protocol uses a different type of oligonucleotide tag, TotalSeq-D, which we used here for the experiments using human samples, but which are currently not available for mouse antigens. For the mouse samples stained with TotalSeq-B, we added 5 μl of highly concentrated HhaI (150,000 U ml–1, NEB) enzyme and 2 µl of 30 µM of a custom antibody tag primer specific for the amplification of the oligonucleotide tags of TotalSeq-B antibodies (ACTCGCAGTAGTCTTGCTAGGACCGGCCTTAAAG) to the Tapestri barcoding mix reagent. The use of TotalSeq-B antibodies primarily affected the ‘Protein Library Cleanup I' section of the protocol, for which we replaced the 2× binding and washing (B&W) buffer from the kit with the following buffer prepared with nuclease-free water: Tris-HCl (final concentration 10 mM, pH 7.5), EDTA (final concentration 1 mM) and NaCl (final concentration 2 M). Finally, each tube of streptavidin beads was resuspended in 45 µl of nuclease-free water then transferred and combined into a new tube for a total of 90 µl. To amplify the final protein target library, we used 5 µl of 4 µM of each custom indexed primers (forward: CAAGCAGAAGACGGCATACGAGAT[i7 index]GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT; reverse: AATGATACGGCGACCACCGAGATCTACAC[i5 index]TCGTCGGCAGCGTC). Typically, we performed twice as many reactions to amplify the DNA target library, but this may be increased to achieve sufficient yield. Using the stained cells that we used as input to scTAM-seq, we also performed 10x Genomics Chromium Single Cell 3′ for transcriptomic profiling of the cells, following the standard protocol. This step was exclusively performed for experiment M.1 (Supplementary Table 1). For the transcriptomic data, LARRY barcodes were later amplified using a modified version of the protocol8 (see Supplementary Table 4 for an updated list of primers). We aimed to design a panel with CpGs dynamically methylated in HSCs, as well as in more committed progenitors (MPPs). Using these data, we selected CpGs that were variably methylated in HSPCs (Extended Data Fig. 1b,c) using three criteria: (1) CpGs differentially methylated between the HSCs and the different MPP populations (DMCs); (2) CpGs intermediately methylated within HSCs (IMCs); and (3) CpGs harbouring within-sample heterogeneity in HSCs (WSHs). The code for selecting CpGs is available from GitHub (https://github.com/veltenlab/EPI-CloneSelection). For DMCs, we used RnBeads54 to determine CpGs that were specifically methylated in one of the HSPCs (that is, in HSC, MPP1, MPP2 or MPP3/MPP4) but not methylated in all the remaining HSPC populations. We only focused on CpGs that were covered by at least 10 sequencing reads in all samples and that had a methylation difference of at least 0.2 between the target cell type and the average of the remaining cell types. IMCs had to be non-overlapping with DMCs and were then defined by a DNA methylation level in the bulk samples (HSCs) between 0.25 and 0.75. Such CpGs may be differentially methylated between two sub-cell types of HSCs. IMCs were required to have a low proportion of discordant reads (PDRs)55 together with a high quantitative fraction of discordant read pairs (qFDRPs)56. PDR and qFDRP are measures of WSH in bulk bisulfite sequencing data and quantify the concordance of methylation states on the same sequencing read (PDR) or of multiple CpGs across different sequencing reads (qFDRP). These CpGs were therefore located in regions showing variable methylation profiles in bulk sequencing data and might represent regions with stochastic methylation in HSCs. After identifying all CpGs that fulfilled the above criteria, we excluded any CpGs that were not in the context of a HhaI cut site and enriched the selected CpGs for those located in the vicinity (100 bp) of at least one TFBS of an important haematopoietic transcription factor (Supplementary Table 5). We also included the following control amplicons: 20 constitutively methylated, 20 constitutively unmethylated and 50 amplicons without a HhaI cut site. Control amplicons were required to identify cells from the data because the remaining amplicons were digested depending on their methylation state. We uploaded this list to the Mission Bio Designer tool (https://designer.missionbio.com/) to receive a final list of 663 amplicons and corresponding primer sequences (Supplementary Table 5). The CpGs were further annotated according to their location in the genome with respect to chromatin states as previously defined57. From the 573 non-control amplicons, a subset of 453 amplicons with low dropout rate in an experiment without HhaI digest was used for analysis. For amplifying the LARRY barcodes, we spiked in an additional primer into the primer pool targeting the LARRY barcode sequence (forward: GCATCGGTTGCTAGGAGAGA; reverse: GGGAGTGAATTAGCCCTTCCA). Two previously published datasets21,58 were used to similarly profile DMCs, IMCs and, additionally, CpGs with interindividual heterogeneity (IIH). For DMCs, we considered peripheral blood and bone marrow samples from a previous study21. Samples with an average coverage across all CpGs below 1 were removed. CpGs with a mean methylation difference higher than 0.1 between the cell types were identified as DMCs. We performed IMC detection on HSC-enriched lineage-negative (LIN−CD34+CD38−) samples from eight male donors using a previously published dataset58. To deal with data sparsity, we set the maximum quantile of missing values per site to 0.005 and removed any sites that exceeded this threshold. IMCs were defined as CpGs with a DNA methylation level between 0.25 and 0.75 in at least 5 samples. When checking for a HhaI cut site, we allowed for a maximum of 25 CpG sites in the extended region around the IMC. For CpGs with IIH, we used the same dataset to identify CpGs with a variance higher than 0.1 across all individuals of the dataset from ref. We also created genotyping amplicons that cover mutations in ASXL1, DNMT3A, TET2, TP53, JAK2, IDH2, PPM1D, SF3B1, IDH1 and SRSF2. We used 62 amplicons covering these genes from the Tapestri single-cell DNA myeloid panel by Mission Bio (https://missionbio.com/products/panels/myeloid/) as a base panel, excluding amplicons with the HhaI restriction sequence GCGC. We designed further amplicons for exons in the aforementioned genes that had a coverage of less than 60% in the default myeloid panel. To prevent these amplicons from having a recognition site, we performed a virtual digestion of the exonic sequences using the HhaI cut sequence. We then uploaded a list containing the fragmented genomic regions to the Mission Bio Designer tool, which resulted in 82 additional amplicons. We uploaded the CpG targets and readily designed genotyping, chromosome Y, and control amplicons using the Mission Bio Designer tool. The final list comprises 665 amplicons and corresponding primer sequences. The resulting 448 CpG targeting amplicons are divided into 215 DMC, 145 IMC and 88 IIH amplicons (Supplementary Table 6). For an overview of the sequencing statistics, see Supplementary Table 7. To jointly profile DNA methylation and RNA in the same cell (scTAMARA-seq, experiment X.1), we took advantage of the recently published SDR-seq method45 and combined it with scTAM-seq. DNA methylation targets were a subset of the original set, excluding amplicons that were not identified as consensus static or dynamic CpGs in the total bone marrow original cohort and low-performing amplicons. RNA targets were selected from a RNA-seq reference map59 using LASSO regression to identify 120 RNAs most predictive of cell state in the CD34+ compartment. Once the cells had been fixed, permeabilized and reverse-transcribed, they were loaded onto the Mission Bio Tapestri platform and processed as for scTAM-seq. The final RNA and DNA sequencing libraries were individually generated as previously described45. For this experiment (scTAMito-seq, experiment X.2), we performed scTAM-seq using the same 367 DNA methylation and genotyping amplicons as for scTAMARA-seq. For processing of raw data, we used a modified pipeline that was based on the originally described pipeline for scTAM-seq5 (https://github.com/veltenlab/scTAM-seq-scripts), which is available from GitHub (https://github.com/veltenlab/EPICloneProcessing). Reads mapping to each of the amplicons were quantified to generate a count matrix, and DNA methylation states were determined using a cut-off value of one sequencing read as in the original scTAM-seq publication5. We used those cellular barcodes that had more than 10 sequencing reads in at least 70% of the control (non-HhaI) amplicons. Doublets were removed using the DoubletDetection tool (https://zenodo.org/record/2678042). To minimize the effect of dropout, we determined the primer combinations that reliably amplified in our panel using a single experiment without the restriction enzyme. For this experiment in mice, LIN−KIT+ cells from a young, wild-type mice (12 weeks) were used and we determined that 453 out of the 573 non-control amplicons (79%) amplified in more than 90% of the cells. These amplicons were used for subsequent analyses. For the surface-protein data, the Mission Bio pipeline was used to extract sequencing reads for a particular cell-barcode–antibody-barcode combination. We restricted analyses of the protein data to those cellular barcodes identified in the DNA methylation library. LARRY barcodes could be directly identified from the scTAM-seq sequencing library because an additional primer pair capturing the LARRY barcode was included (see the section ‘Mouse panel design for scTAM-seq'). Barcode extraction was performed using a modified version of the scripts provided in the original LARRY publication8 (https://github.com/AllonKleinLab/LARRY). Barcodes supported by fewer than five sequencing reads were discarded, and LARRY barcodes with a Hamming distance lower than three were merged for each of the experimental batches individually. Notably, each cell can have more than one unique LARRY barcode owing to multiple lentiviral infections. In these cases, groups of LARRY barcodes were jointly passed on to the progeny. To call clones in this setting, we computed for any pair of LARRY barcodes the extent to which these two barcodes were observed in an overlapping set of cells (formally a Jaccard index). LARRY barcodes were then clustered according to this distance metric. We used a permutation test to determine LARRY barcodes that are merged together to a clone. When LARRY barcodes were merged, cells were assigned to the merged clone if any constituent LARRY barcode was observed. We constructed Seurat61 objects for each of the scTAM-seq samples individually using the binary DNA methylation matrix. To integrate all the samples from experiments M.1–M.3, we used Seurat's IntegrateData62 function. Then we used Seurat's standard workflow without normalization to obtain a low-dimensional representation of our data using UMAP. We removed cells in low-density parts of the UMAP because we found that these cells were of lower quality using the non-digested control amplicons. To compare single-cell to bulk DNA methylation we computed the relative methylation state by dividing the average methylation state of all CpGs in the given group of bulk data (for example, HSC-specific) by the mean methylation state of all CpGs. To that end, we performed differential analysis for each cell-type cluster individually and selected CpGs with a log fold change larger than 1 for each cluster. For those sites, we investigated whether they are in the vicinity (100 bp) of any of the 39 transcription factors in Supplementary Table 5 and computed enrichment P values with the Fisher exact test. A full vignette is available from GitHub (https://github.com/veltenlab/EPI-clone). All remaining experiments (M.4–M.8) were analysed without batch correction, as the samples were processed as single batches. Annotation of the cell-type clusters was performed in dynamic CpG space (see below) using bulk methylation values, demethylation of TFBSs and surface-protein expression. In addition to this information, for experiments M.7 and M.8, we projected cell-type labels from the initial analysis (experiments M.1–M.3) using scmap63. EPI-Clone was then used with the standard parameters as described below. To generate a low-dimensional representation of the protein data only, we opted to use SCTransform64, which produced an improved cell-state resolution. For the scRNA-seq dataset, we used cellranger to generate transcriptomic and surface-protein count matrices, which were used as input to Seurat. Harmony65 was used for batch integration and the cell-type annotation was performed using known haematopoietic marker genes together with the expression of surface proteins. The EPI-Clone algorithm is divided into three steps: (1) identification of static CpGs, (2) identification of cells from expanded clones and (3) clustering of cells from expanded clones. A detailed, step-by-step vignette is available from GitHub (https://github.com/veltenlab/EPI-clone; v.2.0 used in this article). To identify static CpGs, for each combination of CpG and surface protein, EPI-Clone performs a Kolmogorov–Smirnov test to investigate whether cells with methylated CpG differ in surface-antigen expression relative to cells with unmethylated CpG. CpGs with no significant antigen association (determined by the lowest P value for any of the surface proteins) according to a Bonferroni criterion were then selected if their average methylation across all cells was less than 90% but higher than 25% in mouse and higher than 5% in human. To identify cells from expanded clones, cells stemming from an expanded clone should be in a higher density region of the space defined by the static CpGs than cells stemming from non-expanded clones. A density estimate was therefore computed as follows. PCA was performed on all static CpGs from step (1). Effects of cell state, batch and sequencing depth on this measure of local density were then removed by linear regression. We observed that smoothing the resulting quantity locally over 20 nearest neighbours additionally improved performance. Optimal parameters n and k of this step, as well as the density threshold for a cell to be classified as stemming from an expanded clone, were identified through a systematic grid search on experiment M.1, using LARRY barcodes as a ground truth. Here clones >0.25% in size were defined as expanded. To cluster expanded clones, cells from expanded clones were clustered using the standard Seurat workflow, again in a space spanned by n = 100 principal components. The parameters of steps (2) and (3) were established on the basis of the original LARRY experiment (M.1: LARRY main experiment) and used for all subsequent analyses of the mouse haematopoietic system (M.2–M.5, M.7 and M.8) without further adjustments. Static CpGs were defined in experiment M.1 and used for all remaining experiments. In particular, the performance on a replicate LARRY ground-truth experiment (M.2) is analysed in Extended Data Fig. In the native ageing experiment (M.7), we opted for a more conservative threshold for defining large, expanded clones (1%), as native haematopoiesis is more polyclonal than the transplantation setting. This threshold resembled what we found in human native haematopoiesis, using CH mutations and mitochondrial mutations as a partial ground truth (Fig. For when no or only a partial ground truth was available (mouse endothelia (M.6), see below for more details, and the human analysis), we opted instead for a parameter-free approach to identify expanded clones. We used a recently published clustering method, CHOIR44, which automatically determines clusters that have statistical support in the data. Unlike the density-based criterion, CHOIR does not have free parameters (for example, number of principal components, density threshold, number of nearest neighbours to consider). We confirmed that on the mouse LARRY experiment, CHOIR had a similar quantitative performance to the density-based criterion at optimal parameter values (Extended Data Fig. As the first step of this experiment (M.6), all CpGs were used for dimensionality reduction and clustering. We then used the dynamic CpGs defined in experiment M.1 to construct a cell-state map of ECs. The CLR-transformed protein levels enabled us to annotate ECs as capillary, Car4 or lymphatic, in concordance with transcriptomic references. Finally, the 110 static CpGs defined in experiment M.1 were used to identify clones in these lung ECs. Binary data were used as input for CHOIR44 using false-discovery rate adjustment. Only clones with a relative clone size greater than 1% are highlighted in the figures. For comparison with transcriptomic data, the Mouse LungMAP31 was downloaded from CELLxGENE Datasets (Mus musculus + Lung + 10×3′ v.2 + Smart-seq2) and subset for adult samples. The lung EC atlas67 was also downloaded (https://endotheliomics.shinyapps.io/ec_atlas/). To understand whether EPI-Clone robustly identifies clones before and after transplantation, we investigated replicate 2 (old mouse) of the M.7 experiment together with the transplanted mouse (experiment M.8). Notably, the HSCs that were barcoded with LARRY and used for transplantation were obtained from replicate 2 (old mouse) of experiment M.7 (donor mouse). Moreover, and to estimate the false-positive rate of this approach, we performed the same analysis using replicate 1 (old mouse) of experiment M.7. This mouse had no relationship with the transplanted mouse and we would not expect clonal clusters to have cells from both samples to appear from a joint analysis. Data processing followed the methods described for mice. SNVs were called with cellsnp-lite using a minimum allele frequency of 0.05 and a count threshold of 5. Donor assignments were validated by detecting the presence of the Y chromosome in cases when male and female donors had been multiplexed, and/or the presence of previously known donor-specific CH mutations (Supplementary Fig. For samples with previously characterized CH mutations, the mutational status of each cell was determined using a custom script written in pysam. Any cell for which more than 5% of reads covering the relevant genomic site displayed the CH mutation were classified as mutant. Cells with a low number of reads covering the site were excluded, using a read threshold that was determined as a function of total site coverage. Additional CH mutations were identified using SComatic68 based on the assumption that T cells and B cells are depleted from CH mutations. Base counts per cell type were calculated using BaseCellCounter with a minimum mapping quality of 30 and a maximum depth proportional to the number of cells in each group. Beta-binomial parameters were estimated across 35,000 genomic sites to model the distribution of reference and alternate alleles. Final mutation calling was performed using BaseCellCalling, considering all identified cell groups and estimated beta-binomial parameters. This strategy led to the identification of CH mutations in donors A.4 and A.7. Finally we repeated the same analysis for the TBM and CD34+ cohorts and comparing cells that EPI-Clone had annotated as expanded clones to cells annotated as stemming from non-expanded clones. This enabled us to identify the CH mutation in donor B.5, and the DNMT3A(C666Y) variant in donor A.4. Unlike in mouse data, data integration across all CpGs in the human dataset did not effectively remove interindividual differences (for example, large CH clones still clustered apart). However, a larger set of CITE-seq antibodies was included in the human cohort. We therefore identified surface-protein associated (‘dynamic') CpGs across all cells in the TBM cohort and performed data integration using three approaches: CITE-seq data alone, dynamic CpG data alone or a combination of both modalities concatenated into a single feature matrix. All three strategies produced similar results (Extended Data Fig. Notably, the inclusion of both modalities provided more consistency across donors than dynamic CpGs alone, and was less susceptible to technical variation within the donors than CITE-seq data alone. Data integration was performed using scanorama69, as it offered a higher biological resolution of cell types or cell states compared with Seurat integration. In the TBM cohort, we identified a cluster of overstained cells (positive for all antibodies) that were removed before further analyses. The same strategy was followed as for mouse; however, several adjustments were made as described below. EPI-Clone analyses were performed while excluding mature T cells and B cells, unless denoted otherwise. For identification of static CpGs, we proceeded as described above for each donor from cohort A (TBM) individually. We then defined consensus static CpGs as those CpGs that were identified as ‘static' in at least five donors. Eventually, the same set of 94 consensus static CpGs was used for EPI-Clone analysis in all samples. The use of consensus static CpGs in some donors led to substantial improvements in the performance of EPI-Clone with respect to the ground-truth clonal labels (for example, CH mutations). Moreover, it eliminated the need for a static CpG identification step in future studies, as it established a reference set of static CpGs. We used CHOIR44 with false-discovery rate adjustment for identifying expanded clones, see the section on EPI-Clone (above) For this experiment (X.1), DNA methylation data were projected to the CD34+ reference (Fig. EPI-Clone was used on the DNA methylation data with identical settings to all other human samples. For this experiment (X.2), cell types were identified by clustering on all surface antigens. EPI-Clone was then applied using consensus static CpGs. Heteroplasmies of mitochondrial mutations that had previously been identified for that sample using mt-scATACseq46 were called in single cells using pysam by dividing the number of reads supporting the mutant allele by the total number of reads covering the site. Cells with fewer than ten reads on the site were excluded as potential dropout. Plots were generated using the R packages ggplot2 (ref. Data beyond the end of the whiskers are called ‘outlying' points and are plotted individually. For computing lineage-specific output as shown in Fig. 3, we defined output as the fraction of all HSC/MPP1 or myeloid cells per EPI-Clone cluster compared with all HSC/MPP1 or myeloid cells per experiment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Count matrices are available from the Gene Expression Omnibus under accession number GSE282971. Raw reads for the mouse experiments are available from the NCBI Sequence Read Archive with BioProject number PRJNA1191391. Raw sequencing data for the human cohort has been deposited into the European Genome-phenome Archive (accession number EGAS00001008056). The lung EC atlas was downloaded from https://endotheliomics.shinyapps.io/ec_atlas/. Source data are provided with this paper. Release v.2.0 was used for the work included in this article. Cellular barcoding to decipher clonal dynamics in disease. Wagner, D. E. & Klein, A. M. Lineage tracing meets single-cell omics: opportunities and challenges. Mitchell, E. et al. Clonal dynamics of haematopoiesis across the human lifespan. Ludwig, L. S. et al. Lineage tracing in humans enabled by mitochondrial mutations and single-cell genomics. Bianchi, A. et al. scTAM-seq enables targeted high-confidence analysis of DNA methylation in single cells. & Passegué, E. Hematopoietic stem cells through the ages: a lifetime of adaptation to organismal demands. Jaiswal, S. & Ebert, B. L. Clonal hematopoiesis in human aging and disease. Weinreb, C., Rodriguez-Fraticelli, A., Camargo, F. D. & Klein, A. M. Lineage tracing on transcriptional landscapes links state to fate during differentiation. Diverse and heritable lineage imprinting of early haematopoietic progenitors. Chan, M. M. et al. Molecular recording of mammalian embryogenesis. Bowling, S. et al. An engineered CRISPR–Cas9 mouse line for simultaneous readout of lineage histories and gene expression profiles in single cells. Resolving fates and single-cell transcriptomes of hematopoietic stem cell clones by PolyloxExpress barcoding. Lareau, C. A. et al. Massively parallel single-cell mitochondrial DNA genotyping and chromatin profiling. Weng, C. et al. Deciphering cell states and genealogies of human haematopoiesis. Lareau, C. A. et al. Artifacts in single-cell mitochondrial DNA mutation analyses misinform phylogenetic inference. Gabbutt, C. et al. Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues. Li, L. et al. A mouse model with high clonal barcode diversity for joint lineage, transcriptomic, and epigenomic profiling in single cells. Gaiti, F. et al. Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. Chaligne, R. et al. Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states. Farlik, M. et al. DNA methylation dynamics of human hematopoietic stem cell differentiation. Identification of regulatory networks in HSCs and their immediate progeny via integrated proteome, transcriptome, and DNA methylome analysis. Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Nichols, R. V. et al. High-throughput robust single-cell DNA methylation profiling with sciMETv2. Heterogeneity of young and aged murine hematopoietic stem cells revealed by quantitative clonal analysis using cellular barcoding. Ashcroft, P., Manz, M. G. & Bonhoeffer, S. Clonal dominance and transplantation dynamics in hematopoietic stem cell compartments. Zink, F. et al. Clonal hematopoiesis, with and without candidate driver mutations, is common in the elderly. Nam, A. S. et al. Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A R882 mutations perturb early progenitor states through selective hypomethylation. Hui, T. et al. High-resolution single-cell DNA methylation measurements reveal epigenetically distinct hematopoietic stem cell subpopulations. Long-term propagation of distinct hematopoietic differentiation programs in vivo. Guided construction of single cell reference for human and mouse lung. Pei, W. et al. Polylox barcoding reveals haematopoietic stem cell fates realized in vivo. Rodriguez-Fraticelli, A. E. et al. Clonal analysis of lineage fate in native haematopoiesis. Beerman, I. et al. Functionally distinct hematopoietic stem cells modulate hematopoietic lineage potential during aging by a mechanism of clonal expansion. & Nakauchi, H. Age-associated characteristics of murine hematopoietic stem cells. Large-scale clonal analysis resolves aging of the mouse hematopoietic stem cell compartment. Dykstra, B., Olthof, S., Schreuder, J., Ritsema, M. & de Haan, G. Clonal analysis reveals multiple functional defects of aged murine hematopoietic stem cells. Rodriguez-Fraticelli, A. E. et al. Single-cell lineage tracing unveils a role for TCF15 in haematopoiesis. Kuribayashi, W. et al. Limited rejuvenation of aged hematopoietic stem cells in young bone marrow niche. Konturek-Ciesla, A. et al. Temporal multimodal single-cell profiling of native hematopoiesis illuminates altered differentiation trajectories with age. Säwen, P. et al. Murine HSCs contribute actively to native hematopoiesis but with reduced differentiation capacity upon aging. Selective advantage of mutant stem cells in human clonal hematopoiesis is associated with attenuated response to inflammation and aging. Mucke, L. & Ryan Corces, M. CHOIR improves significance-based detection of cell types and states from single-cell data. Lindenhofer, D. et al. Functional phenotyping of genomic variants using multiomic scDNA-scRNA-seq. Lareau, C. et al. Codon affinity in mitochondrial DNA shapes evolutionary and somatic fitness. The comprehensive DNA methylation landscape of hematopoietic stem cell development. A., Shibata, D. & MacLean, A. L. Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life. Chen, M., Fu, R., Chen, Y., Li, L. & Wang, S.-W. High-resolution, noninvasive single-cell lineage tracing in mice and humans based on DNA methylation epimutations. Aksöz, M. et al. Hematopoietic stem cell heterogeneity and age-associated platelet bias are evolutionarily conserved. Long-term ex vivo haematopoietic-stem-cell expansion allows nonconditioned transplantation. & Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. Müller, F. et al. RnBeads 2.0: comprehensive analysis of DNA methylation data. Locally disordered methylation forms the basis of intratumor methylome variation in chronic lymphocytic leukemia. Scherer, M. et al. Quantitative comparison of within-sample heterogeneity scores for DNA methylation data. Universal chromatin state annotation of the mouse genome. Aging human hematopoietic stem cells manifest profound epigenetic reprogramming of enhancers that may predispose to leukemia. Triana, S. et al. Single-cell proteo-genomic reference maps of the hematopoietic system enable the purification and massive profiling of precisely defined cell states. B. et al. Multiomic profiling of human clonal hematopoiesis reveals genotype and cell-specific inflammatory pathway activation. Stuart, T. et al. Comprehensive integration of single-cell data. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Marchal, C. et al. Genome-wide analysis of replication timing by next-generation sequencing with E/L Repli-seq. Kalucka, J. et al. Single-cell transcriptome atlas of murine endothelial cells. Muyas, F. et al. De novo detection of somatic mutations in high-throughput single-cell profiling data sets. Efficient integration of heterogeneous single-cell transcriptomes using Scanorama. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer, 2009). Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Velten, L. et al. EPI-Clone datasets M.1-M.3: Single cell targeted DNA methylation profiling of hematopoietic stem and progenitor cells. Scherer, M. et al. EPI-Clone dataset M.5: LARRY mature immune cells. Scherer, M. et al. EPI-Clone dataset M.6: endothelial cells (Lung). Velten, L. EPI-Clone dataset: Native hematopoiesis, old and young mouse. Scherer, M. et al. EPI-Clone dataset M.7: Native hematopoiesis: old and young mouse (replicate 2). Scherer, M. et al. EPI-Clone dataset M.8: Transplantation experiment. Velten, L. EPI-Clone dataset: Human total bone marrow(A.1-A.7). Velten, L. EPI-Clone dataset X.1 : Targeted DNAm+DNA+RNA-seqfrom CD34+ BM cells of a healthy donor. Velten, L. et al. EPI-Clone supplementary dataset: Single cell RNA-seq of clonally barcoded hematopoietic progenitors. We thank staff at Mission Bio for support and at the CRG Core Technologies Programme, specifically to the CRG Genomics Unit for assistance with sequencing and the CRG/UPF Flow Cytometry Unit for flow sorting. Funding for this project was provided to L.V. was supported through the Walter Benjamin Fellowship funded by Deutsche Forschungsgemeinschaft (DFG, German Research Foundation, reference 493935791) and a postdoctoral fellowship provided by the Dr. Rurainski Foundation for Cancer Research. was supported through the European Union's Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement no 945352. The project that gave rise to these results received the support of a fellowship to M.M.B. acknowledges supported by grants by the German Research Foundation (DFG), including an Emmy Noether fellowship (LU 2336/2-1), LU 2336/3-1, LU 2336/6-1, STA 1586/5-1, TRR241, SFB1588, and the Heinz Maier-Leibnitz Award. was supported by a Medical Research Council and Leukaemia UK Clinical Research Training Fellowship (MR/R002258/1) and MRC DTP Supplementary Funding 2021. acknowledges funding from the Medical Research Council Molecular Haematology Unit Programme Grant (MC_UU_00029/8), Blood Cancer UK Programme Continuity Grant 13008, NIHR Senior Fellowship, and the Oxford BRC Haematology Theme. These authors contributed equally: Michael Scherer, Indranil Singh, Martina Maria Braun, Chelsea Szu-Tu Computational Biology and Health Genomics, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology, Barcelona, Spain Michael Scherer, Martina Maria Braun, Chelsea Szu-Tu, Julia Rühle, Agostina Bianchi, Luca Cozzuto, Robert Frömel, Sergi Beneyto-Calabuig, Renée Beekman & Lars Velten Martina Maria Braun, Pedro Sanchez Sanchez, Julia Rühle, Agostina Bianchi, Robert Frömel, Sergi Beneyto-Calabuig, Renée Beekman & Lars Velten DZHK (German Centre for Cardiovascular Research), Partner site Heidelberg/Mannheim, Heidelberg, Germany MRC Molecular Haematology Unit, MRC Weatherall Institute of Molecular Medicine, University of Oxford, Oxford, UK Niels Asger Jakobsen, Verena Körber & Paresh Vyas Max-Delbrück-Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin Institute for Medical Systems Biology (BIMSB), Berlin, Germany Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain Stanford Genome Technology Center, Palo Alto, CA, USA Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar analysed the mouse data and developed EPI-Clone. analysed the human data with support from R.B., A.B. and A.R.-F. analysed the mouse aged endothelial data. supervised data analyses, with conceptual input from I.S. created the targeting panels with support from R.F. wrote the manuscript with input from all co-authors. Correspondence to Alejo Rodriguez-Fraticelli or Lars Velten. A.R.-F. serves as an advisor for Retro Bio. Parts of this study have been supported with reagents donated by Mission Bio. The other authors declare no competing interests. Nature thanks Elisa Laurenti, Shalin Naik and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Distribution of the CpGs covered by all 663 amplicons in our panel. From this set of amplicons, 453 WSH/DMC/IMR CpGs were selected based on a low dropout in a control experiment, see methods. c. Schematic overview of the CpG selection for scTAM-seq. Bulk DNA methylation data was collected from Cabezas-Wallscheid et al.22. We identified three classes of CpGs, which we included in the final panel design shown in Fig. DMCs are defined by comparisons between cell types, IMCs are regions with intermediate methylation in HSCs, and WSHs are regions with intermediate methylation in HSCs and a high degree of intra-molecule variability. The lines represent sequencing reads, where filled circles stand for methylated and unfilled circles for unmethylated CpGs, respectively. To compute the confusion matrix, a random forest classifier was trained to predict cell type from surface antigen expression data, using the scRNA-seq modality. c. Integrated UMAP of the LARRY main experiment, replicate, and native haematopoiesis (experiments M.1-M.3) as in Fig. 1c, highlighting the LARRY barcodes and donor mouse. d. UMAP defined only on the dynamic CpGs. e. Surface protein UMAP of experiment M.1 (13,885 cells) with the cell type labels obtained from the DNA methylation UMAP as shown in Fig. Protein data was normalized using SCTransform64 prior to generating a low-dimensional representation with PCA and UMAP. f. Expression of selected surface proteins in the protein UMAP. g. Bar chart depicting the percentage of static and dynamic CpGs within early/late replicating domains66, respectively. Clonal UMAP based on static CpGs as in Fig. C highlights cells that were selected as part of expanded clones, based on local density in PCA space. e. Overlap between clones defined using EPI-clone and ground truth labels for the biological replicate. 2b highlighting the LARRY donor labeled by two unique fluorophore sequences. For experiment M.1, two donor mice were sacrificed and HSCs were labeled with LARRY constructs containing a GFP label in one case, and LARRY constructs containing a Sapphire label in the other case. Accordingly, the data set contains cells from four mice that contain two sets of clones, labeled with GFP and Sapphire, respectively, see also methods. Precision and recall were calculated for the identification of cells from expanded (>0.25%) clones. a. Overview of the sorting scheme for experiment M.4: Mature myeloid cells. b. UMAP based on dynamic CpGs (defined from experiment M.1) showing the differentiation state of mature myeloid cells and their progenitors. c. Enrichment of CpGs specifically unmethylated in a cell-type cluster according to the vicinity to the annotated TFBS, see also main Fig. d. Expression of surface proteins in the different cell type clusters for stem-cell-specific markers (KIT, SCA1, CD201) and markers of mature myeloid cells (CD9, CD44). f. UMAP computed on static CpGs (defined from experiment M.1) with the LARRY barcodes indicated. h. UMAP representation as in F visualizing the different cellular compartments including progenitors (LSK, LK) and mature cells from lung and BM/PB. Overlap between clones defined using EPI-clone and ground truth clonal labels for the mature myeloid experiment. k. Adjusted rand indices quantifying the overlap between EPI-clone clusters and LARRY barcodes stratified by the different cell types identified in B. l. Cell type distribution and clone sizes in different clones identified by EPI-Clone and stratified by cellular compartment m. Number of unique LARRY barcodes per cell type cluster. The elevated number of LARRY barcodes per cell in the macrophage cluster suggests the presence of contaminant DNA from doublets or phagocytosis in this cluster. Lung cells were isolated from an old mouse, then purified and sorted to filter out CD45+ cells and enrich for CD31 + , before profiling with scTAM-seq. b. UMAP embedding and low-resolution clustering of endothelial cells using the dynamic CpGs identified in experiment M.1. c. Differential expression analysis of surface markers in the different clusters from panel B. d. CLR-normalized expression values of surface markers across the different clusters. e. Normalized expression of the corresponding genes (scRNA-seq) for endothelial cells from the Mouse LungMAP, only for adult samples31. g. UMAP computed on static CpGs (identified in experiment M.1). Colors highlight clones identified by EPI-Clone with a relative clone size greater than 1%. h. Barplot of endothelial cell types contributions across clones; again, only EPI-clones with a relative clone size greater than 1% are visualized; numbers in the top of the bars represent the absolute clone size, i.e. number of cells. Mutual information between methylation status of all CpGs and the EPI-clones for endothelial and haematopoietic cells. HSCs from an old donor mouse (100 weeks) were either LARRY-barcoded and transplanted into a recipient mouse or directly used for processing with scTAM-seq/EPI-clone. In the negative control, we performed EPI-clone analysis on a set of unrelated HSCs from an old mouse (100 weeks) and the transplanted mouse. Highlighted in red are HSCs from the donor mouse. Same EPI-clone UMAP as in B highlighting the sample origin (C) and the LARRY barcode (D). e. Quantification of the fraction of EPI-clone clones that have at least one HSC from the donor mouse. If a HSC successfully engrafts, it should keep its clonal DNA methylation pattern (i.e., EPI-clone identity) and pass it to all of its progeny. We observe that this is the case for the transplantation experiment, but not for clustering together the transplanted mouse with an unrelated, aged mouse (negative control). The asterisk indicated p-values below 0.1 from a correlation test. Scheme illustrating selection of target CpGs from bulk whole genome bisulfite sequencing data, see also Methods. DMCs are differentially methylated between cell types, IMCs display intermediate methylation levels in HSCs and IIH are variably methylated across individuals in HSCs. c. Cell state clustering for the TBM cohort using antibodies, DNA methylation or both modalities. Colors correspond to clustering on the DNA methylation (DNAm)+AB data, see main Fig. UMAPs were computed using data integration by scanorama across donors from the TBM cohort, using the indicated modality. f. Selection of static and dynamic CpGs for donor A.6, see also main Fig. g. Scatter plot depicting for all CpGs the average methylation across myeloid cells per donor, as well as the classification of the CpG as static or dynamic. h. CpGs that were classified as static in at least five donors were selected as consensus static CpG and used for the EPI-clone analysis. a. Static CpG UMAPs and EPI-clone clustering result for donor B.5. Left panel highlights a CH mutation identified in this donor, right panel highlights EPI-clone clusters. b. Scatter plot displaying the percentage of cells from each EPI-Clone displaying CH mutations, for the CD34+ cohort. Dots in colors correspond to EPI-clones dominated by a CH mutation, see Fig. All donors from the CD34+ cohort with a detected CH mutation are shown. c. Scatter plot displaying the percentage of cells from each EPI-Clone displaying CH mutations, for NK and immature B cells. EPI-Clone was run on all cells except T and mature B cells, but the overlap was computed on NK and immature B cells only. d. Static CpG UMAPs as in main Fig. 4c,d, highlighting NK and immature B cells classified according to CH status. Mature and immature B cells are also highlighted to demonstrate that mature B and T cells mostly cluster in lymphoid clusters. Barchart depicts precision and recall for the task of classifying T cells as CH or non-CH based on EPI-Clone labels. Grey dots correspond to EPI-clones with no known driver mutation. Dots in colors correspond to EPI-clones dominated by a CH mutation, see Fig. g. Same as F, for cell states within the CD34+ compartment. Composition of the panel used, see Supplementary Table 6. RNA-seq amplicons were selected using a scRNA-seq reference59 to identify the set of 120 genes with highest information on cell states in the CD34+ compartment by LASSO regression. c. Scatter plot depicting the number of RNA, DNA methylation (DNAm) and genotyping amplicons observed per cell. d. Boxplot comparing the number of features (RNA species) observed per cell in scTAMARA-seq to the number of features observed in whole transcriptome analysis (WTA) on CD34+ cells for the same 120 genes59. See methods, section Data visualization for a definition of boxplot elements. e. Heatmap depicting correlation in DNA methylation profiles between sample X.1 and the other CD34+ BM donors. g. Heatmap depicting scaled expression of marker genes for the different RNA-based cell states. a. Static CpG UMAP computed on all cells from the patient, highlighting cell types identified using surface antigen expression levels. c. Scatter plot comparing average heteroplasmies for these mutations, as determined by mt-scATAC-seq (reference46) or scTAMito-seq (this study). d. Scatter plot depicting, for all mitochondrial variants, the average heteroplasmy and the statistical association with EPI-Clone. Specifically, a linear model was trained on EPI-Clone clusters to predict heteroplasmy at the single cell level, and the p value from an F-test is shown. e. Heatmap relating the single-cell heteroplasmies of mitochondrial variants to EPI-Clones, for T cells only. The columns correspond to different T cells and the rows comprise mitochondrial mutations measured by scTAMito-seq. Overview of the experiments and cell pools used as input to scTAM-seq for the different experiments. This table details which cell types and pools of cell types were used as input to scTAM-seq across the different experiments, for both human and mouse samples. TotalSeq-B antibodies used for generating surface-protein-expression libraries. This table contains information on mitochondrial variants derived from ref. List of primers used for constructing and amplifying LARRY barcodes. These primers are used for amplifying LARRY barcodes from a cDNA library and enables the association between a lineage barcode and gene expression. List of all amplicons and CpGs covered by the scTAM-seq mouse panel with further annotations. This table comprises 663 amplicons comprising CpGs, their genomic location and further annotation such as vicinity to TFBSs and DNA methylation values in bulk data. List of all amplicons and CpGs covered by the scTAM-seq human panel with further annotations. This table comprises (1) a list of primers and corresponding genomic locations for the human scTAM-seq experiments, (2) a list of antibodies for assessing the expression of surface proteins for the human experiments, and (3) the gene-specific primers used for scTAMARA-seq. Sequencing statistics for scTAM-seq and scRNA-seq libraries. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Clonal tracing with somatic epimutations reveals dynamics of blood ageing. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08944-w'>The origin of vertebrate teeth and evolution of sensory exoskeletons</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 15:04:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The earliest record of tooth antecedents and the tissue dentine1,2, an early-vertebrate novelty, has been controversially represented by fragmentary Cambrian fossils identified as Anatolepis heintzi3,4,5. Anatolepis exoskeletons have the characteristic tubules of dentine that prompted their interpretation as the first precursors of teeth3, known as odontodes. Debates over whether Anatolepis is a legitimate vertebrate6,7,8 have arisen because of limitations in imaging and the lack of comparative exoskeletal tissues. Here, to resolve this controversy and understand the origin of dental tissues, we synchrotron-scanned diverse extinct and extant vertebrate and invertebrate exoskeletons. We find that the tubules of Anatolepis have been misidentified as dentine tubules and instead represent aglaspidid arthropod sensory sensilla structures9,10. Synchrotron scanning reveals that deep ultrastructural similarities between odontodes and sensory structures also extend to definitive vertebrate tissues. External odontodes of the Ordovician vertebrate Eriptychius11,12,13 feature large dentine tubules1 that are morphologically convergent with invertebrate sensilla. Immunofluorescence analysis shows that the external odontodes of extant chondrichthyans and teleosts retain extensive innervation suggestive of a sensory function akin to teeth14,15,16. These patterns of convergence and innervation reveal that dentine evolved as a sensory tissue in the exoskeleton of early vertebrates, a function retained in modern vertebrate teeth16. Middle-Ordovician fossils now represent the oldest known evidence for vertebrate dental tissues. The origin of vertebrate teeth has been a long-standing problem in palaeontology17,18,19,20,21,22,23,24. Although teeth evolved from structures in the dermal exoskeleton of jawless vertebrates known as odontodes25, their origin and function remains obscure. Odontodes are the direct evolutionary and developmental antecedents of diverse tooth-like structures, including mandibular and pharyngeal teeth, dermal scales and body denticles19,25,26,27. The odontode unit is characteristically made of dentine, a neural crest-derived vertebrate novelty2,26,28,29, which has diagnostic internal tubules made by odontoblast processes as they deposit tissue30. The presence of dentine in the exoskeleton of Palaeozoic stem-gnathostomes suggests that external odontodes were secondarily recruited into the oral epithelium to form teeth7,8,9. Although dentine seems to be a vertebrate apomorphy, its origins remain controversial owing to its variability, lack of a sufficient comparative dataset, and challenges identifying it in fossil forms. Cambrian conodonts are the earliest mineralizing vertebrates31, but remain problematic owing to their uncertain phylogenetic position17,32,33,34 and the uniqueness of the mineralized tissues in their pharyngeal elements. Critically, conodont elements lack true dentine17,34. The earliest putative occurrence of dentine and odontodes can be traced to fragmentary phosphatic fossils from late-Cambrian and Early-Ordovician deposits known as Anatolepis heintzi3,4,5. Anatolepis was first described as the earliest agnathan fish4, but its vertebrate affinity was challenged and instead suggested to correspond to an arthropod exoskeleton on the basis of morphological and histological grounds6,7,8. Researchers then revealed histological details of odontode-specific tissues in Anatolepis, such as tubular dentine, a pulp cavity and lamellar basal tissue, which reasserted its position as the first fish3. The conclusion that Anatolepis represents the earliest mineralizing stem-gnathostome has profound bearing on the initial evolution of vertebrate mineralization, and particularly on the evolution of teeth. If Anatolepis embodies the earliest odontodes this would mean dentine and acellular bone were the first vertebrate tissues and enameloid and enamel evolved at a later stage3,35,36. The ambiguity of the first vertebrate dental tissues, combined with the lack of comparative data from diverse taxa, restricts our ability to differentiate between several hypotheses on odontode origins including: protection—the dermal skeleton including odontodes evolved as a means against abrasion and predation25,37,38; locomotion—increased mineralization in the dermal skeleton acted to stiffen the body in the absence of the mineralized axial skeleton25,39; mineral storage—the mineralized skeleton evolved for calcium or phosphate storage25,40; and sensory—the dermal skeleton evolved as part of, or as a support for, sensory systems16,25. In this study we use high-resolution synchrotron computed microtomography to analyse a diverse representation of extant and extinct vertebrate and invertebrate exoskeletons to analyse the origin and distribution of dentine in Cambrian and Ordovician taxa. Together with tissue clearing and immunofluorescence analyses of the external odontodes of extant chondrichthyans and teleosts, our findings illuminate the deep evolutionary origins of dentine and the likely function of the earliest odontodes. To test the vertebrate affinity of Anatolepis, we first deployed high-resolution synchrotron tomography on late-Cambrian fossil fragments. Vertical and horizontal canals infiltrate the mineralized tissue (Fig. 1) and are connected to teardrop-shaped cavities within an otherwise homogeneous lamellar basal layer (Fig. The tubercles connect to a central cavity that extends to the ventral surface and dorsally attenuates into multiple large-calibre tubules (Fig. These features were previously interpreted as a pulp cavity and dentine tubules, respectively, in scanning electron micrographs3. Although Anatolepis' tubules were difficult to segment because of partial collapse due to acid preparation, tubules can be seen to have a distinct flared arrow-shaped end in the virtual transverse section (Fig. Tubules are mostly restricted to the centre with a few peripheral tubules that surround the edges of the tubercle (Fig. Notably, the peripheral tubules had been observed previously in scanning electron micrographs3 but interpreted as the natural odontode edge. Externally, Anatolepis fragments are marked by rounded tubercles that are interspersed and non-overlapping (Fig. Anatolepis' tubercles vary subtly in morphology and are situated within the thin basal tissue; the tubercles also extend above the basal tissue that is marked externally by several pore openings (Fig. We used this distinct internal and external morphology to match our samples with previously described specimens of Anatolepis3,5. a, Three-dimensional reconstruction of the late-Cambrian Anatolepis sp. fragment from the central Texas Wilberns Formation (TC-1021) with the slice indicating the location of b. b, Tomographic cross-section of Anatolepis. c, Translucent three-dimensional reconstruction of the Anatolepis fragment with a distribution of segmented cuticle organs in yellow. d, Segmented single tubercle of Anatolepis. e, Three-dimensional reconstruction of an indeterminate aglaspidid tail spine fragment taken from the complete specimen (Milwaukee Public Museum (MPM) 18572), from the late-Cambrian Saint Lawrence Formation of Wisconsin with the slice indicating the location of f. f, An artificially shaded tomographic cross-section of indeterminate aglaspidid gives contrast to the microanatomy of the tubricle. g, Translucent three-dimensional reconstruction of the cuticle of an indeterminate aglaspidid highlighting the distribution of segmented cuticular organs. h, Segmented single tubercle of an indeterminate aglaspidid with the central cavity in purple, peripheral tubules in orange, and central tubules in green. cc, central cavity; co, cuticular organ; ct, central tubules; lm, lamellar tissue; pt, peripheral tubule system; tb, tubules; tu, tubercle. The asterisk indicates arrow-shaped tubules in cross-section. High-resolution phase-contrast synchrotron scans of coeval aglaspidid arthropod cuticles—extracted from complete specimens—were compared to Anatolepis. Similarly to vertebrate skeletons, the exoskeleton of aglaspidids is also phosphatic41,42,43. Aglaspidid cuticles have a homogeneous lamellar structure that is interrupted by vertical pore canals like those described in Anatolepis3 (Fig. Some of these pore canals flare internally into teardrop-shaped cuticular organs, which vary in size through the specimen and are similar to the Anatolepis specimens (Fig. 1c,d and Extended Data Figs. franconensis, we resolve horizontal canals in addition to the vertical canals and teardrop-shaped cuticular organs. However, this feature was not seen in all scanned aglaspidid cuticle samples and is notably absent from thicker cuticular regions such as portions of the tail spine (Extended Data Fig. Virtual sections indicate partially infilled central cavities from which tubules emanate similar to the putative ‘pulp cavities' in Anatolepis. The central tubules flare dorsally with a distinctive arrow-shaped cavity, as seen in Anatolepis, and are capped by a mineralized cone that sits within a pore (Fig. Each central tubule is surrounded by a hypermineralized layer that is then surrounded by a hollow cavity, in a tube-in-tube formation that is not seen in vertebrate dentine (Fig. The tubules circumferentially diffuse from the central cavity, and the hollow cavities merge and form a honeycomb structure that is hollow (Fig. Additionally, there is a set of tubules that are peripheral to the central tubules; these are simpler in morphology, lacking both the flared end and mineralized coating (Figs. Externally, aglaspidid cuticles have diverse tubercles that vary in size and morphology but are generally rounded, with a corresponding dimple on the underside, the same characters that define Anatolepis (Fig. The external morphology, scans and subsequent segmentation of several aglaspidid cuticle fragments revealed microanatomy with all of the hallmarks of Anatolepis, in having a lamellar basal tissue perforated by vertical and horizontal canals, a central cavity, teardrop-shaped cuticular organs and characteristic arrow-shaped tubules. We conclude that Anatolepis is not a vertebrate but is most parsimoniously identified as an aglaspidid arthropod. a, Three-dimensional reconstruction of a single tubercle from the tail spine of an indeterminate late-Cambrian aglaspidid from the Saint Lawrence Formation of Wisconsin (MPM 18572) showing a dorsal cross-section indicating the relative region of c. b, Three-dimensional reconstruction of the claw sensilla of the extant anomuran Neopetrolisthes sp. showing the multiple tubercles with emanating setae and a dorsal cross-section indicating the relative region in d. c, Tomographic dorsal cross-section of indeterminate late-Cambrian aglaspidid (MPM 18572) tubercle. d, Tomographic dorsal cross-section of a single sensillum. e, A close-up of the dorsally oriented tubules within the tubercle that are composed of a hollow space with a mineralized sheath surrounding a hollow tubule. f, A close-up of the extant Neopetrolisthes sp. dorsolaterally oriented tubules, exhibiting a hollow tube with a mineralized sheath set into a hollow space. g,h, Three-dimensional reconstruction showing the internal tubule anatomy of the segmented tubercle and segmented sensilla of the extinct indeterminate aglaspidid (g) and extant Neopetrolisthes sp. To interrogate whether the histological similarities that led to the misidentification of Anatolepis reflect deeper similarities between invertebrate and vertebrate exoskeletons, we scanned extant invertebrate cuticle and vertebrate mineralized tissues of 35 extant and extinct genera (Extended Data Table 1). We found that the histology of aglaspidid and Anatolepis cuticles did not conform to any dental or osteological tissues seen in the sampled vertebrates but instead was most similar to those of sampled modern arthropod tissues. Specifically, the gnathobases of Limulus polyphemus (Atlantic horseshoe crab; Extended Data Fig. 4), the chelicerae of Hadrurus arizonensis (giant hairy scorpion; Extended Data Fig. 4), the dactyl sensilla of Planes mitutus (Columbus crab; Extended Data Fig. 4) and the dactyl sensilla of Petrolisthe galathinus (porcelain crab) have distinct similarities to aglaspidid and Anatolepis cuticles (Fig. Porcelain crab sensilla have the distinctive tube-in-tube morphology of their tubules that emanate from a central cavity and radiate dorsolaterally, terminating in setae (Fig. Overall, the histological microanatomy of the porcelain crab sensilla is most like that of aglaspidid and Anatolepis tubercles, with the main difference being a laterally versus circumferentially radiating arrangement of tubules, respectively. The claws and much of the body are covered with sensilla, which are rounded tubercules with seta stemming from one side. Each mineralized seta corresponds to a tube-in-tube structure internally and coalesces past the exocuticle (Fig. In the porcelain crab, the tubercle with setae probably represents a tricoid sensillum9,10, which is an innervated mechanosensitive receptor sensitive to deflection10. The strong similarity between cuticular structures of aglaspidids (including the fragments ascribed to Anatolepis) and the sensilla of extant invertebrates suggests that they had a sensory function. We next examined definitive samples of Middle-Ordovician vertebrate dentine and odontodes to compare them to these invertebrate structures. The microanatomy of the Middle-Ordovician vertebrates is reminiscent of, but distinct from, the invertebrate cuticular tissues described above. Vertebrates from the Middle Ordovician remain largely enigmatic owing to the paucity of specimens. The two most studied taxa are the co-occurring stem-gnathostomes Eriptychius and Astraspis, which are almost exclusively represented by their odontodes, with rare articulated material confirming stem-gnathostome affinity1,11,13,44,45. Eriptychius odontodes have a distinct elongate morphology, are made up entirely of dentine and lack a capping enameloid tissue (Fig. Remarkably similar to aglaspidids, Eriptychius odontodes have large dentine tubules stemming from a pulp cavity that repeatedly branch and attenuate to the surface as pores as seen also in another Ordovician vertebrate from the Harding sandstone46,47. This convergent exposure of tubules to the surface in aglaspidids and Eriptychius is suggestive of a sensory function, as is the fact that dentine exposure is the most common cause of painful tooth sensitivity in modern taxa2. Additionally, Eriptychius odontodes maintain an open pulp cavity that is continuous with the vascular network (Fig. This is a critical feature when considering the role of vasculature and nerves in the development and maintenance of the pulp cavity, and sensory function48. (Field Museum of Natural History (FMNH) PF 17901) showing the elongated odontodes and a transverse section indicating the relative region of c. b, Three-dimensional reconstruction of a fragment of dermal armour of the agnathan Astraspis sp. (FMNH PF 17898) showing the rounded and stellate-shaped odontodes and a transverse section indicating the relative region of d. c, A transverse tomographic section of Eriptychius sp. odontodes exhibiting a lack of enameloid, wide-calibre dentine tubules, pulp cavity and vascular canals. d, A transverse tomographic section of Astraspis sp. odontodes exhibiting a thick layer of enameloid, narrow-calibre dentine tubules, pulp cavity and vascular canals. e,f, Three-dimensional reconstruction showing the internal anatomy of Eriptychius sp. ; note the lack of enamel, wide-calibre dentine tubules and extensive vascularity of Eriptychius sp. dt, dentine tubule; en, enameloid; od, odontode; pc, pulp cavity; vc, vascular canal. To explore the extent to which the ancestral sensory function of odontodes persists in extant forms, we tested for innervation in the external odontodes from a diverse range of extant fishes. Dental innervation studies are typically based on oral teeth48,49,50 rather than external odontodes, limiting our inferences on the original function of these structures. Given that external odontodes are homologous to teeth, innervation probably also has a crucial role in the development and function of odontodes. We tested this hypothesis through tissue clearing and immunofluorescence analysis on tail odontodes from late developmental stages and juvenile catsharks (Scyliorhinus retifer; Fig. 4) and little skates (Leucoraja erinacea; Extended Data Fig. 5), and the pectoral fin odontodes of juvenile bristlenose catfish (Ancistrus sp. Three-dimensional segmentation of confocal stacks shows innervation associated with odontodes to be present in all taxa sampled, with nerves surrounding the base of odontodes in chondrichthyans and invading the pulp cavity in the pectoral fin in the catfish (Fig. These findings support the hypothesis that the innervation associated with the dentine of odontodes is an ancestral trait among extant gnathostomes. a, Adult bristlenose catfish (Ancistrus sp.) b, Juvenile catshark (S. retifer) showing odontodes along the entire body, with a close-up of the tail region. immunofluorescence confocal stack-based segmentation of fin odontodes with nerves branching and entering the pulp cavity of the odontodes. d, Dorsal view of S. rotifer immunofluorescence confocal stack-based segmentation of tail odontodes with nerves associated with odontodes. e, Immunostained cross-section of Ancistrus sp. f, Dorsal view of immunostained catshark tail tip odontodes, with nerves surrounding and associated with the forming odontode. This experiment was performed on one representative sample of the correct stage in each species, n = 1. nv, nerves; 3A10, neurofilament marker; DAPI, 4′,6-diamidino-2-phenylindole, dihydrochloride. Anatolepis' putative vertebrate affinities hinged on two main arguments, namely the absence of a complex pore canal system in arthropod cuticles, and the presence of dentine, a calcified tissue of neural crest origin unique to vertebrates. Our data show that late-Cambrian Anatolepis and aglaspidid arthropods and modern arthropods have a cuticle with a pore canal arrangement (Extended Data Fig. 1), which includes sensilla with tubules radiating from a central cavity (Figs. Our new high-resolution synchrotron scan data demonstrate that the seemingly simple dentine tubules in the tubercles of Anatolepis are in reality complex tubules with distinct morphology that more closely conform to arthropod sensilla found in Cambrian aglaspidids and modern arthropods9,10,51. Thus, the fossil tissues of Anatolepis are not dentine and lamellar bone, as they do not conform to any known variation found in extinct or extant vertebrates, but instead correspond to an arthropod exoskeleton as expressed in both extinct and extant species52,53. a, Artistic reconstruction of the Ordovician agnathan Astraspis sp. as a representative of an early mineralizing vertebrate. b, Schematic illustration of the elongate odontode system in the agnathan Eriptychius sp. exhibiting the wide-calibre dentine tubules and pulp cavity. e, Artistic reconstruction of Cambrian aglaspidid Aglaspis sp. f, Schematic illustration of the mineralized exoskeleton of an aglaspidid showing cuticle organization and tubercles. i, An artistic rendering of the extant anomuran Neopetrolisthes sp. j, Schematic illustration of Neopetrolisthes sp. k, Close-up of a single sensillum with tubules emanating dorsolaterally to the seta. l, Close-up of the Neopetrolisthes sp. bn, bone; ct, central tubule; end, endocuticle; ep, epidermis; ex, exocuticle; odb, odontoblast. The illustrations were created by A. Boersma under a Creative Commons licence CC BY-SA 4.0. The removal of Anatolepis from vertebrates means that the presence of enamel or enameloid at the origin of odontodes can be tracked to the Middle Ordovician22,28,45. The only other putative Cambrian armoured agnathan is an unnamed taxon originating from Australia with the fragmentary material being compared to the Ordovician pteraspidomorph agnathan Porophoraspis54. However, the described histology of this Cambrian Australian material also reveals more similarities to arthropod cuticle than to any known vertebrate. The horizontal and vertical canals are similar to those we describe in aglaspidids and other arthropods, and granular middle layers with distinct polygons54 are seen in many arthropods including the porcelain crab (P. galathinus) in this study (Extended Data Fig. The odontodes of early vertebrates such as Eriptychius have clear markers of sensory capability, such as an open pulp cavity, large dentine tubules and a lack enameloid. The lack of capping enameloid in Eriptychius would have allowed dentine tubules to be exposed to surface stimuli, which in modern taxa causes extreme sensitivity14,55. Additionally, the large-calibre dentine tubules in the odontodes of Eriptychius indicate the persistent presence of odontoblast processes, which have been shown to act as sensory receptors in teeth16. The open pulp cavity of Eriptychius odontodes implies innervation, as dental studies indicate critical cross-talk between the dental mesenchyme and the development and maintenance of innervation in open pulp cavity48 (see the discussion on dental sensitivity in the Supplementary Information). By contrast, co-occurring Astraspis odontodes have a thick layer of hypermineralized enameloid, blocking the small-calibre dentine tubuli from being exposed to the surface, and the pulp cavity is gradually infilled in older odontodes, probably leading to a diminished sensory function (Fig. On the basis of these observations, we reason that Eriptychius odontodes were more suited to sensory capability than those of Astraspis. Sensory capabilities are crucial to maintaining environmental awareness in an organism covered by external protective armour, whether in vertebrates or invertebrates. This biological demand led to the remarkable convergence of early-vertebrate dermal odontodes and arthropod cuticle sensilla, which along with the lateral lines in vertebrates contribute to ‘sensory armour' (Fig. Indeed, arthropod cuticles with pore canal systems are part of advanced and highly specialized sensory organs that have been documented in modern taxa10,51,52 but largely overlooked in the fossil record. An ancestral sensory function of dentine reveals that independent vertebrate dental specializations, or autapomorphies, reflect a shared history. Notably, there are numerous reports of modern vertebrates with sensitive external odontodes. Blind catfish have specialized dermal odontodes that have a purported sensory function56,57. Several mammals, such as narwhals, have specialized dentition that serves a sensory function58,59,60. Odontoblasts themselves are widely recognized to be sensory cells14,16,55. When viewed through this evolutionary lens, the fact that teeth in the mouth are extremely sensitive is less of a mystery, and more a reflection of their evolutionary origins within the sensory armour of early vertebrates. The original described material3,4,5 was not available or could not be located owing to collections moving from Denver to the Smithsonian National Museum of Natural History (USNM) and other non-returned loans. The Anatolepis material in this study is from Miller's sample TC-1021, from central Texas and from near the top of the Morgan Creek Limestone Member of the Wilberns Formation61. Limestone at that stratigraphic horizon is assigned to the trilobite Idahoia Zone. At the time of collection, a 2–3-kg limestone sample was dissolved in ≈15% glacial acetic acid. The dissolved material was wet-sieved, and the insoluble residue was dried. This residue was concentrated using a dense liquid, 1,1,2,2 tetrabromoethane, with a specific gravity of about 2.87. Conodonts, fragments of Anatolepis and other phosphatic fossils were concentrated in the dense fraction. The fossils were picked from the dense fraction with a fine artist's paintbrush under ×25 magnification with a binocular microscope. Additional ‘Anatolepis' was provided by the Geological Survey of Canada (GSC), for external examination and scanning but no destructive sampling was permitted. Additionally, two specimens of ‘undetermined fish A' (GSC 65598 and GSC 65599) and two specimens of ‘undetermined fish B' (GSC 65603 and GSC 65604) were examined. All GSC specimens are from the Cow Head Group, a deposition at the eastern edge of the North American continent spanning an interval extending from the middle Cambrian to the base of the Middle Ordovician, and have been previously published62. All Cow Head specimens appeared to correspond to arthropod invertebrate cuticle. MPM 18572 is a telson fragment removed from a mostly complete articulated individual, and is featured in Figs. However, we examined and scanned many more aglaspidids. All examined late-Cambrian aglaspidid material was from the Raasch63 collection, which is split between the USNM, Washington, DC and the MPM, Milwaukee, with some material collected by G. Gunderson and donated to the University of Wisconsin Geology Museum. All of the late-Cambrian aglaspidids sampled here come from the Upper Cambrian of the Saint Lawrence Formation, Sauk County, WI (see ref. franconensis holotype USNM PAL 98916, from the ‘Ptychaspis beds' of the Tunnel City Group at Hudson, Saint Croix County, WI41 (see ref. 63 for other locality details), was sampled. Permission for destructive sampling of the Aglaspis? franconensis USNM PAL 98916 holotype was obtained from the USNM, granted by C. Labanderia, to sample a small fragment of the exoskeleton for comparative purposes. The fragment was then mounted with reversible glue on a plastic toothpick and labelled for synchrotron scanning. FMNH PF 17898 is a fragment of dermal bone with several odontodes ankylosed. This was picked from loose dissolved material, and then mounted on a plastic toothpick with reversible glue for scanning. This specimen originates from the Middle-Ordovician Harding Sandstone of Wyoming11,12. FMNH PF 17901 is a fragment of dermal bone with several elongate odontodes ankylosed. This was picked from loose dissolved material, and then mounted on a plastic toothpick with reversible glue for scanning. The elongate odontode morphology and lack of dentine identify the fragment as Eryptychius sp.11,65. This specimen originates from the Middle-Ordovician Harding Sandstone of Wyoming11,12. and shed exoskeletons of Petrolisthes galathinus (porcelain crab) were donated by C. Ferret, from his own collection. Limulus  polyphemus (Atlantic horseshoe crab), Callinectes sapidus (Atlantic blue crab), Planorbarius corneus (ramshorn snail), Eupatorus gracilicornis (five-horned rhinoceros beetle) and Megabalanus tintinnabulum (giant purple barnacle) were purchased online as pinned or desiccated specimens. All other extant invertebrate cuticles came from the FMNH ethanol collections and were not from animals killed for this project (see Extended Data Table 1). Suckermouth armoured catfish (Ancistrus sp.) were purchased from a local aquarium store (NationWideAquaticsUSA) and were bred in aquarium tanks following a slightly modified version of the protocol described in ref. 66. In this study, we used the albino and ‘lemon' morphs for their reduced melanin; however, these lines have potentially been hybridized in the hobby, and therefore an exact species is uncertain. Water changes with cold water were increased when breeding behaviour was observed. The water was maintained at 27 °C and pH 6.5–7. The temperature was reduced by up to 3 °C by unplugging the heater and doing cold water changes to simulate the rainy season until eggs were laid. Each tank contained one male and two to three female fish, allowing natural breeding. Ceramic D-shaped caves were placed in the tanks to allow for the males to establish territories and the females to lay eggs in them. Once eggs were laid, they were removed from the cave and placed in a jar with a bubbler. Juveniles and adult fish were fed boiled vegetables and algae wafers. Guidelines for animal rearing were approved by The Institutional Animal Care and Use Committees (IACUC) of the University of Chicago who approved the care and breeding of Ancistrus sp. All of the procedures were performed at the University of Chicago. The hatchling Leucoraja erinacea (little skate) and hatchling Scyliorhinus retifer (catsharks) were obtained from the Marine Resources Center, Marine Biological Laboratory, Woods Hole, MA, USA. All animals were euthanized using 0.5% tricaine methanesulfonate (MS-222, Syndel) until the cessation of heartbeat and fixed in 4% paraformaldehyde (PFA; Acros Organics, item number EW-88353-82) overnight before transferring them to 1× phosphate-buffered saline (PBS; Fisher Bioreagents). The white X-ray beam from the Advanced Photon Source bending magnet was reflected from a Pt-coated mirror at a 3 mrad incidence angle, producing an energy spectrum centred at about 25 keV with approximately 10 keV bandwidth. A total of 3,600 projection views were acquired over 180° using a Point Grey Grasshopper 3 camera, with 1,920 (horizontal) × 1,200 (vertical) pixels, and a LuAG scintillator. The sample-to-scintillator distance was approximately 45 mm. Depending on the sample, the camera was equipped with either a 10× or 5× Mitutoyo long-working-distance objective lens. For both objectives, a 175-mm tube was used, resulting in 0.56-μm pixels in object space for the 10× lens and 1.09-μm pixels in object space for the 5× lens. The exposure time was 0.014 s per view for the 10× lens and 0.025 s per view for the 5× lens. Images were reconstructed using the GSECARS tomography processing software (https://cars-uchicago.github.io/IDL_Tomography/)67, which dark-current-corrects and white-field-normalizes the acquired data before performing gridding-based image reconstruction. The resulting image voxel sizes are 0.56 × 0.56 × 0.56 μm for the 10× configuration and 1.09 × 1.09 × 1.09 μm for the 5× configuration. The experiment was run at GSECARS; proposal GUP: 82458; proposal title: Characterizing the origin and development of the earliest mineralizing tissues in vertebrates using synchrotron micro-computed tomography; experiment dates: 21 March 2023 and 2 December 2022. Laboratory micro-computed tomography was performed on a Phoenix v|tome|x S 240 from GE (PaleoCT facility, RRID: SCR024763, University of Chicago) using the 180-kV nano-focus tube. The scanning parameters for S. retifer (stage 38 catshark) were 70-kV tube voltage and 400-µA tube current, with a timing of 100 ms, 2,000 projections, 3-frame averaging and a 0.1 Cu filter applied; the voxel size was 10.256 µm. For L. erinacea (stage 33 little skate), the parameters were 100 kV and 135 µA, with a timing of 150 ms, 3-frame averaging, resulting in a voxel size of 26.181. (bristlenose catfish), the parameters were 60 kV and 269 µA tube current, with a timing of 100 ms, 2,000 projections and 3-frame averaging; the voxel size was 35.062. Segmentation, reconstruction and visualization were performed on Amira 3D 2021.1 (Konrad-Zuse-Zentrum Berlin, 1995–2001 and FEI SAS, a part of Thermo Fisher Scientific, 1999–2021). Fertilized eggs of S. retifer and L. erinacea were obtained from the Marine Biological Laboratory facilities (https://www.mbl.edu/research/research-organisms). Embryos of S. retifer (stage 38, following ref. 69, in addition to 2-week-old hatchlings (n = 6)) were mechanically extracted and placed in 0.5% tricaine diluted in 1% PBS for 30 min. Embryos were then fixed in freshly made 4% PFA in PBS for 4 days on a rocker at 4 °C. The samples were then dehydrated in 25%, 50%, 75% and then 100% methanol and stored at −20 °C. To acquire whole-odontode images of immunofluorescently labelled tissue with a confocal microscope, samples were optically cleared using modified versions of previously established techniques70,71,72,73. Juvenile bristlenose catfish (Ancistrus sp.) were first rehydrated (100%, 75%, 50%, 25% and 0% methanol in 1× PBS). The rehydrated samples were then delipidated in a solution of 10 w/v% Triton X-100 (Fisher Scientific, BP151) and 10 w/v% N-butyldiethanolamine (Alfa Aesar, L09953.22) dissolved in distilled water (CUBIC-L)70 for 5 days, gently shaking at 37 °C, with the CUBIC-L solution changed daily. The samples were then washed in 1× PBS six times for 2 h each, with one wash left overnight. Samples were then blocked overnight at 4 °C while shaking in blocking buffer (5 v/v% goat serum (Gibco, 16210-072, lot 2285796), 1 w/v% bovine serum albumin (Fisher Scientific, BP9704, lot 222347) and 0.3 v/v% Triton X-100 in 1 × PBS) before immunostaining. The next day, samples were incubated in primary antibodies to neurofilament-associated antigens (Developmental Studies Hybridoma Bank, clone 3A10, lot 5/7/20; 1:100 in blocking buffer) for 7 days at 4 °C while shaking. Samples were then washed in blocking buffer three times for 2 h each, with one wash left overnight. Samples were then stained with secondary antibodies (Cy3 donkey anti-mouse IgG, Jackson Laboratories, 715-165-150, lot 163873; 1:300) and counterstained with DAPI (Biotium, 40009, lot 15D1117; 1:1,000) in blocking buffer for 7 days at 4 °C while shaking. Samples were then washed in 1 × PBS three times for 2 h each and postfixed overnight in 1 w/v% PFA in 1× PBS overnight at 4 °C while shaking. Samples were washed with 1 × PBS for 2 h before preparing them for confocal microscopy. We found that embryonic S. retifer and L. erinacea tissues were more difficult to image, probably owing to larger tissue sizes and increased extracellular matrix protein concentration; thus, additional steps were required to optically clear the tissue samples. Samples were first bleached to reduce autofluorescence and pigmentation into a solution of methanol, dimethylsulfoxide and 30% hydrogen peroxide (4:1:1) while exposed under constant light overnight. After this, samples were rehydrated (100%, 75%, 50%, 25% and 0% methanol in PBS 1%). The rehydrated samples were then delipidated in CUBIC-L as described above. To increase antibody penetration and optical transparency, samples were enzymatically digested with collagenase P. Samples were first immersed in reaction buffer for 2 h at 37 °C while shaking, and then digested in a solution of 1 mg ml−1 collagenase P in carbonate buffer (50 mM sodium carbonate (Sigma, 223530), 50 mM sodium bicarbonate (Sigma, 26014), 150 mM sodium chloride (Sigma, S3014) and 25 μM ethylenediaminetetraacetic acid (EDTA; Sigma, E5134), pH 10) for 24 h at 37 °C with gentle shaking. Samples were washed in carbonate buffer with 5 mM EDTA three times for 2 h at 37 °C with gentle shaking. Samples were incubated in blocking buffer overnight at 37 °C while shaking before incubating in primary antibodies to neurofilament (1:40 3A10 in blocking buffer) for 4 days at 37 °C while shaking. Embryos were washed six times (3 h each) in 0.1% Triton X-100 in 1 × PBS before staining with secondary antibodies (Biotium CF633 goat anti-mouse IgG, 20121, lot 23C1003; 1:300) and counterstained with DAPI (1:100) in blocking buffer for 4 days at 37 °C while shaking. Samples were then washed in 1 × PBS three times for 2 h each and postfixed overnight in 1 w/v% PFA in 1 × PBS overnight at 4 °C while shaking. Samples were washed with 1 × PBS for 2 h before preparing them for confocal microscopy. Before imaging, the samples were placed in a refractive-index matching solution (45 w/v% antipyrine (Thermo Scientific, A11089.22), 30 w/v% nicotinamide (Thermo Scientific, A128271000) and 0.5 v/v% N-butyldiethanolamine in distilled water (CUBIC-R70)) for 1–3 days (depending on the size of the embryonic sample), to render the tissues optically transparent for three-dimensional imaging. Samples were mounted on a 35-mm glass-bottom microscopy dish suspended in a solution of 0.5% low-melting-point agarose (Thermo Scientific, R0801) in CUBIC-R to avoid movement of the sample during the imaging process. The mounted samples were then covered with a glass coverslip. Samples were imaged with a Zeiss confocal laser scanning microscope (LSM 900) equipped with a Plan-Apochromat 25×/0.8 glycerol immersion objective and using a pinhole of 1 AU. Three-dimensional, whole-tissue volumes were acquired using a combination of Z-stacks and image tiling. Images were processed with Zen (Zeiss) and FIJI74, and VGStudio Max 3.3 was used for three-dimensional visualization of the scans. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All original data are available via MorphoSource, including micro-computed tomography, synchrotron and confocal scans. Data are publicly available, except in cases in which the museum holds the copyright, for which data are available upon reasonable request via https://www.morphosource.org/projects/000626244. For synchrotron reconstruction, we used GSECARS Tomography Software, which is free to download via https://cars-uchicago.github.io/IDL_Tomography/index.html. Smith, M. M. & Sansom, I. J. in Development, Function and Evolution of Teeth (eds Teaford M.F. Gans, C. & Northcutt, R. G. Neural crest and the origin of vertebrates: a new head. & Repetski, J. E. Histology of the first fish. Repetski, J. E. A fish from the Upper Cambrian of North America. The affinity of Anatolepis Bockelie & Fortey. Peel, J. S. & Higgins, A. K. Anatolepis - a problematic Ordovician vertebrate reinterpreted as an arthropod. Peel, J. S. Anatolepis from the Early Ordovician of East Greenland - not a fishy tail. The complex structure of the cuticle of Pseudagnostus (Agnostina, Trilobita?). Comparative morphogenesis of sensilla: a review. Bryant, W. L. A study of the oldest known vertebrates, Astraspis and Eriptychius. Sansom, I. J., Smith, M. P., Smith, M. M. & Turner, P. The Harding Sandstone revisited -a new look at some old bones. The oldest three-dimensionally preserved vertebrate neurocranium. Pashley, D. H. Mechanisms of dentin sensitivity. Byers, M. R. in International Review of Neurobiology Vol. Magloire, H., Couble, M.-L., Thivichon-Prince, B., Maurin, J.-C. & Bleicher, F. Odontoblast: a mechano-sensory cell. & Rücklin, M. The ins and outs of the evolutionary origin of teeth. Fraser, G. J., Cerny, R., Soukup, V., Bronner-Fraser, M. & Streelman, J. T. The odontode explosion: the origin of tooth-like structures in vertebrates. & Witten, P. E. Evolutionary and developmental origins of the vertebrate dentition. Origin and early evolution of vertebrate skeletonization. in eLS, https://doi.org/10.1002/9780470015902.a0026408 (John Wiley & Sons, 2025). Major Events in Early Vertebrate Evolution, 1st edn (CRC Press, 2001). Reif, W.-E. in Evolutionary Biology (eds Hecht, M. K. et al.) 287–368 (Springer, 1982). The homology of odontodes in gnathostomes: insights from Dlx gene expression in the dogfish, Scyliorhinus canicula. Haridy, Y., Gee, B. M., Witzmann, F., Bevitt, J. J. & Reisz, R. R. Retention of fish-like odontode overgrowth in Permian tetrapod dentition supports outside-in theory of tooth origins. Smith, M. M. Putative skeletal neural crest cells in early Late Ordovician vertebrates from Colorado. Ancient vertebrate dermal armor evolved from trunk neural crest. The amazing odontoblast: activity, autophagy, and aging. & Smith, M. M. Presence of the earliest vertebrate hard tissue in conodonts. Blieck, A. et al. Fossils, histology, and phylogeny: why conodonts are not vertebrates. Miyashita, T. et al. Hagfish from the Cretaceous Tethys Sea and a reconciliation of the morphological–molecular conflict in early vertebrate phylogeny. Turner, S. et al. False teeth: conodont-vertebrate phylogenetic relationships revisited. Qu, Q., Haitina, T., Zhu, M. & Ahlberg, P. New genomic and fossil data illuminate the origin of enamel. Sire, J.-Y., Davit-Béal, T., Delgado, S. & Gu, X. The origin and evolution of enamel mineralization genes. A., Mark-Kurik, E., Karatajūtė-Talimaa, V. N., Lukševičs, E. & Ivanov, A. Bite marks as evidence of predation in early vertebrates. Halstead, L. B. Vertebrate Hard Tissues (Wykeham, 1974). Wainwright, S. A., Vosburgh, F. & Hebrank, J. H. Shark skin: function in locomotion. Bone metabolism and evolutionary origin of osteocytes: novel application of FIB-SEM tomography. Hesselbo, S. P. Aglaspidida (Arthropoda) from the Upper Cambrian of Wisconsin. Lerosey-Aubril, R., Ortega-Hernández, J., Kier, C. & Bonino, E. Occurrence of the Ordovician-type aglaspidid Tremaglaspis in the Cambrian Weeks Formation (Utah, USA). Sansom, I. J., Smith, M. P., Smith, M. M. & Turner, P. Astraspis-the anatomy and histology of an Ordovician fish. Denison, R. H. Ordovician Vertebrates from Western United States (Field Museum of Natural History, 1967). Sansom, I. J., Smith, M. M. & Smith, M. P. Scales of thelodont and shark-like fishes from the Ordovician of Colorado. Johanson, Z., Tanaka, M., Harriman, N. & Meredith Smith, M. Early Palaeozoic dentine and patterned scales in the embryonic catshark tail. Fried, K. & Gibbs, J. in The Dental Pulp: Biology, Pathology, and Regenerative Therapies (ed. & Lumsden, A. G. Tooth morphogenesis: the role of the innervation during induction and pattern formation. A. G. An experimental study of timing and topography of early tooth development in the mouse embryo with an analysis of the role of innervation. & Wong, L. Structure, innervation, and distribution of sensilla on the wings of a grasshopper. Locke, M. Pore canals and related structures in insect cuticle. Young, G. C., Karatajute-Talimaa, V. N. & Smith, M. M. A possible Late Cambrian vertebrate from Australia. Kim, J. W. & Park, J.-C. Dentin hypersensitivity and emerging concepts for treatments. Haspel, G., Schwartz, A., Streets, A., Camacho, D. E. & Soares, D. By the teeth of their skin, cavefish find their way. A. Odontode morphology and skin surface features of Andean astroblepid catfishes (Siluriformes, Astroblepidae): Odontode morphology. Nweeia, M. T. et al. Sensory ability in the narwhal tooth organ system. Catania, K. C. & Remple, M. S. Somatosensory cortex dominated by the representation of teeth in the naked mole-rat brain. Weissengruber, G. E., Egerbacher, M. & Forstenpointner, G. Structure and innervation of the tusk pulp in the African elephant (Loxodonta africana). & Taylor, J. Biostratigraphy of Cambrian and Lower Ordovician strata in the Llano uplift, central Texas. Fortey, R., Landing, E. & Skevington, D. Cambrian-Ordovician boundary sections in the Cow Head Group, western Newfoundland. Raasch, G. O. Cambrian Merostomata (Geological Society of America, 1939). Houée, G., Bardin, J., Germain, D., Janvier, P. & Goudemand, N. Developmental models shed light on the earliest dental tissues, using Astraspis as an example. Smith, M. M. & Sansom, I. J. in Development, Function and Evolution of Teeth (eds. Teaford, M. F. et al.) 65–81 (Cambridge Univ. Mori, S. & Nakamura, T. Redeployment of odontode gene regulatory network underlies dermal denticle formation and evolution in suckermouth armored catfish. Onimaru, K., Motone, F., Kiyatake, I., Nishida, K. & Kuraku, S. A staging table for the embryonic development of the brownbanded bamboo shark (Chiloscyllium punctatum). A. et al. Big insight from the little skate: Leucoraja erinacea as a developmental model system. Susaki, E. A. et al. Versatile whole-organ/body staining and imaging based on electrolyte-gel properties of biological tissues. Structural and molecular interrogation of intact biological systems. Smith-Paredes, D. et al. Embryonic muscle splitting patterns reveal homologies of amniote forelimb muscles. The developing bird pelvis passes through ancestral dinosaurian conditions. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. We thank the Shubin Lab members S. Ali, E. Hillan, D. Jockel and S. Kult Perry for early discussion and support; J. Miller for segmentation assistance and animal care; I. Sansom, S. Giels, J. Repetski, M. Webster, S. Bensmaia, N. Hughes, M. Coates, M. Friedman, A. R. C. Millner, M. Bonilla, T. Stewart and B. Engh for assistance, discussions and thoughtful comments; J. A. Maisano, D. Edey, M. W. Colbert and all of the University of Texas Computed Tomography Facility for initial scans; K. Duncan, G. Olack and A. Neander for assistance with additional scanning of specimens; P. Coorough Burke, W. Simpson, C. A. Eaton, M. Coyne, M. Flourence and C. Labanderia for providing access to fossil specimens; R. Bieler, J. Voight, K. Griffin-Jakym and M. Pryzdia for specimen access; C. Ferret for donating invertebrates from his personal collection and J. Garcia for Ancistrus specimens and catfish knowledge; A. Boersma for work on Fig. 5; and M. Yakovlev for assistance with synchrotron scanning. Portions of this work were performed at GeoSoilEnviroCARS (The University of Chicago, Sector 13), Advanced Photon Source, Argonne National Laboratory. GeoSoilEnviroCARS was supported by the National Science Foundation – Earth Sciences (EAR – 1634415). Tomography capability developments were supported by the Department of Energy Office of Basic Sciences Geosciences Research Program (DE‐SC0020112). This research used resources of the Advanced Photon Source, a US DOE Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract number DE-AC02-06CH11357. Additional support was provided by the Brinson Family Foundation and the Biological Sciences Division of the University of Chicago. Department of Organismal Biology and Anatomy, The University of Chicago, Chicago, IL, USA Yara Haridy, Sam C. P. Norris, Matteo Fabbri, Neelima Sharma & Neil H. Shubin Center for Functional Anatomy and Evolution, Johns Hopkins University School of Medicine, Baltimore, MD, USA Museum of Comparative Zoology, Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, MA, USA Department of Ecology and Evolutionary Biology, University of Toronto, Toronto, Ontario, Canada Department of Earth Sciences, University of California, Riverside, Riverside, CA, USA School of Earth, Environment, and Sustainability, Missouri State University, Springfield, MO, USA Center for Advanced Radiation Sources, The University of Chicago, Chicago, IL, USA Patrick La Riviere & Phillip Vargas You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the project. Fossil sample acquisition was carried out by Y.H., N.S., K.N., N.H.S., J.F.M. and J.O.-H. Fossil sample preparation was carried out by Y.H. Synchrotron data collection was performed by Y.H., S.C.P.N., M.F., N.S., M.R., P.L.R. Synchrotron data analysis was carried out by Y.H., S.C.P.N., N.S. Sample preparation, immunostaining and imaging of Ancistrus, Scyliorhinus and Leucoraja was carried out by Y.H., S.C.P.N., M.F. All segmented images, visualizations and figures (except Fig. with input from all authors. The original draft was written by Y.H. Correspondence to Yara Haridy or Neil H. Shubin. The authors declare no competing interests. Nature thanks Philippe Janvier and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, The late Cambrian Anatolepis sp. fragment from central Texas Wilberns Formation (TC-1021) from which a fragment was removed for scanning. b, The holotype of Aglaspis? franconensis USNM PAL 98916 from the late Cambrian St. Lawrence Formation of Wisconsin. c, Dorsal view of a translucent three-dimensional reconstruction of Anatolepis fragment with a distribution of horizontal canals that surround the tubricles. d, Dorsal view of a translucent three-dimensional reconstruction of Aglaspis? franconensis fragment with a distribution of horizontal canals that surround the tubercles. e,f, Segmented horizontal canals showing how they sometimes are attached to the cuticular organs, in Anatolepis and Aglaspis? Abbreviations: co, cuticular organ; hc, horizontal canal; tb, tubercle. b, Magnification of the rendering of flat top tubercles, in purple are the segmented horizontal canals. c,d, Tomographic cross sections of cuticle showing that the horizontal canals run through the tubercles, unlike arthropods, but some tubercles have a central cavity similar to aglaspidid specimens. e, Three-dimensional rendering of a specimen of ‘undetermined fish B' (GCS65603) from Cow Head Formation. f, Magnification of the rendering of the pointed tubercles, in purple are the segmented horizontal canals. g, Tomographic cross sections of cuticle showing presence of horizontal canals and a cuticular organ similar to aglaspidid arthropods. h, Rendering of a segmented horizontal canal and attached cuticular organ similar to previously figured known aglaspidid cuticle. i, Three-dimensional rendering of a specimen of ‘Anatolepis' (GCS65600) from Cow Head formation. j, Magnification of the rendering of the flattened pointed tubercles, in purple are the segmented horizontal canals. k, Tomographic cross sections of cuticle showing presence of central cavity in the tubercles, and possible tubules in tubercles, similar to other figured aglaspidids. Abbreviations: cc, central canal; co, cuticle organ; hc horizontal canal; tb?, possible tubule; tu, tubercle. a, Tomographic cross section of Anatolepis cuticle showing the multiple layers at the base. b Tomographic cross section of Aglaspis? franconensis cuticle showing the multiple layers c, Tomographic cross section of Aglaspis sp. cuticle showing the multiple layers at the base, this specimen was taken from a body segment d, Tomographic cross section of Aglaspis sp. cuticle showing the multiple layers at the base, this specimen was taken from a tailspine segment. a-c, Atlantic horseshoe crab (Limulus polyphemus) gnathobase; Giant hairy scorpion (Hadrurus arizonensis) chelicerae; Columbus crab (Planes minutus) claw dactyl. d-f, Partially translucent renderings showing the prevalence of tubules in the cuticles of the aforementioned arthropods. g,i,j, Variation in tubule orientation when viewed dorsally. h, A cuticle organ like structure seen in the gnathobase of Limulus. Abbreviations: tb, tubule; co, cuticular organ. a, µCT rendering of a stage 33 little skate (Leucoraja erinacea) showing the distribution of odontodes across the body, the odontodes samples for clearing and immunofluorescence were the distal most tail odontodes. b, Immunofluorescence confocal stack-based segmentation of tail odontodes with nerves branching and surrounding the odontode, note that the nerve is at the base of the odontode where foramina are observed in older specimens. c, Immunostained cross section of little skate single tail odontode, showing the innervation of the base of the odontode, the epidermis of the little skate is bright either due to autofluorescence or nonspecific antibody binding. This experiment was performed on one representative sample of the correct stage, n = 1. a,b, Three-dimensional renderings of a portion of Neopetrolisthes sp. dactyl, showing the distribution of external sensilla with seta. c, Tomographic cross section of the cuticle showing the multiple layers and penetration of vertical canals. d, Tomographic dorsal cross section showing the distinctive granular polygonal layer described in (Young et al.54). e, Tomographic cross section of a single sensillum with vertical pores akin to those described in (Young et al.54). Abbreviations: ex, exocuticle; en, endocuticle; gpl, granular polygonal layer; hc horizontal canal; sn, sensillum; st, seta; vc, vertical canal. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Haridy, Y., Norris, S.C.P., Fabbri, M. et al. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/new-class-of-drugs-blocks-wakefulness-chemical-and-offers-relief-from-insomnia/'>People with Insomnia Find Relief in New Drugs, Promising Cannabinoids, and Wearable Devices</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-21 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Sleep Better with New Drugs, Select Cannabinoids and Wearable Devices This Nature Outlook is editorially independent, produced with financial support from Avadel. The 23 year old, who asked for her last name to be withheld, started struggling with sleep when she was a child. She takes “a myriad of medications” each night, she says, but usually still cannot fall asleep until the early hours of the morning. “I can't get up and be functional until halfway through the day,” she says. Her insomnia exacerbates other medical conditions as well, including migraines and the pain condition fibromyalgia. In the United States, about 12% of adults have been diagnosed with chronic insomnia — when a person struggles to sleep for more than three nights each week for at least three months, and experiences daytime distress as a result. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Fortunately for Miranda and millions of others with chronic insomnia, new treatments are arriving. The emergence of a class of pharmaceuticals that induces sleep through a different brain pathway from existing drugs is a welcome development, and molecules in cannabis and specialized medical devices to promote sleep are also showing potential as sleep aids. Soon, those struggling with sleep could have a range of new options available to help. Cognitive behavioural therapy for insomnia (CBT-I) is usually the recommended first treatment. But CBT-I is not covered by all health-care insurance plans in the United States. In the United Kingdom and parts of Europe, public health-care systems usually provide it, but waiting times can be long. This is because, around the world, there is a limited availability of therapists, says Andrew Krystal, a psychiatrist at the University of California, San Francisco. “We keep hiring new people, but almost immediately their schedules are completely filled and the wait list is a year.” Pharmacological interventions are the next line of defence, Krystal says. Benzodiazepines and a class of medicines called Z-drugs, which include zolpidem (Ambien), are among the most prescribed insomnia medications. But they can create a hangover effect and increase the risk of falls in older people. These drugs also have the potential for misuse and can cause dependence. Miranda tried Ambien, but says that she quickly became chemically dependent. Over-the-counter products such as antihistamines are also used for sleeplessness. None are ideal, however, because they have not been evaluated as sleep aids, says Emmanuel Mignot, a sleep-medicine researcher at Stanford University in California. When she first developed chronic insomnia as a child, her paediatrician recommended melatonin, which is available without a prescription in the United States. But they came with what she calls “torturous” side effects: she felt constantly anxious and exhausted during the day, and her memory became “incredibly foggy”. Mignot was studying narcolepsy, a chronic disorder that affects sleep–wake cycles and causes people to fall asleep suddenly, when he inadvertently helped to pave the way towards the latest means of treating insomnia. Mignot then found that people with narcolepsy lack orexin, confirming the chemical's main job: promoting wakefulness. If drugs could be developed to prevent orexin from binding to its receptors, Mignot thought, then people with insomnia would become “narcoleptic for one night”. In 2014, the biopharmaceutical company Merck, received US Food and Drug Administration (FDA) approval for the first dual orexin receptor antagonist (DORA) drug, suvorexant (Belsomra). Compared with benzodiazepines and Z-drugs, which inhibit activity all over the brain, DORA drugs affect only the neurons activated by orexins (see ‘Blocking wakefulness'). “The beauty of it is it does nothing but block the stimulation of wakefulness,” says neurologist Joe Herring, who heads neuroscience clinical research at Merck in Rahway, New Jersey. Daridorexant is the only DORA drug for which data are available about daytime functioning, says Antonio Olivieri, chief medical officer at Idorsia, which produces daridorexant. In clinical trials, Idorsia showed that, compared with those given a placebo, people who received daridorexant experienced significant improvements in daytime insomnia symptoms the following day. So far, there have been no one-to-one comparisons of DORA drugs. “But we rarely have such evidence, so instead, we have to rely on statistical techniques that allow you to make indirect comparisons.” It's also difficult to say definitively how DORA drugs compare with older treatments for insomnia, but Buysse says that drug registration trials suggest that DORA drugs have fewer adverse cognitive or hangover effects compared with benzodiazepines and Z-drugs, as well as less potential for dependence and misuse. The main drawback to DORA drugs, Buysse says, is not medical but financial: their high cost keeps them out of reach of many people who could benefit from them. “There are many patients I would like to prescribe these drugs for, but I know in order for them to get one of these medications we'll have to go through trials of several other drugs before the request will be considered,” Buysse says. DORA drugs are also available only in a few countries, so far. Without the drug, she says, “I'd probably be on a much higher benzodiazepine dose than I am.” She hopes her suvorexant dose can continue to increase, so that some of her other medications can be reduced. Seltorexant, for example, is being developed by the US pharmaceutical firm Johnson & Johnson for people with both major depressive disorder and insomnia. Around 70% of people with depression have insomnia, so having a medication that treats both of those disorders “has the potential to fill an important gap”, says Krystal, who has consulted for Johnson & Johnson on the drug. In a phase III trial, participants who took the drug experienced meaningful improvement in both sleep and depressive symptoms, with an antidepressant effect that seemed to be independent of the participants getting better sleep. Seltorexant might have an antidepressant effect because it is designed to block only one of the two types of orexin receptor, Krystal adds, whereas other DORA drugs block both receptor types. Investigations of already-approved DORA drugs are also expanding into other populations. Merck has sponsored investigator-led studies of suvorexant in people with insomnia as well as depression or substance-use disorders, and Idorsia is sponsoring studies of daridorexant's safety and efficacy in sub-groups of people who have insomnia and other conditions. Insomnia is often a precursor to and co-morbid with Alzheimer's, and the disease seems to manifest differently in people with the condition. In one study comparing older people with insomnia with those with both insomnia and Alzheimer's, people with both conditions had a number of extra changes to their sleep patterns, including less time spent in deep sleep — sometimes called slow-wave sleep because that describes the pattern of the brain's electrical activity during these intervals. Preliminary data suggest that suvorexant could also help to reduce toxic brain proteins. The results of a follow-up study testing that finding are expected in 2026. “It's definitely a key player in my sleep-medication arsenal,” she says. McGregor is investigating cannabinol (CBN), a molecule that develops in cannabis as the psychoactive component tetrahydrocannabinol (THC) oxidizes. His group reported that CBN increased sleep in rats to a similar degree as zolpidem, but without the drug's known negative side effect of suppressing rapid-eye-movement sleep. Unpublished data of a single-night trial with 20 people with insomnia disorder show that people fell asleep 7 minutes faster after taking 300 milligrams of CBN compared with those taking a placebo; participants also reported subjective improvements in sleep and mood. Although 7 minutes “doesn't sound like a lot”, it is on a par with what benzodiazepines and Z-drugs typically accomplish, says Camilla Hoyos, a sleep researcher at the Woolcock Institute of Medical Research in Sydney, who led the work. McGregor, Hoyos and their colleagues are aiming to follow up the work with a large, community-based trial in which people with insomnia take either CBN or a placebo for six weeks at home. Several small studies have failed to find a sleep benefit from taking CBD. In one experiment, researchers observed that participants in a study who received 10 milligrams of THC and 200 milligrams of CBD actually slept for 25 minutes less compared with when they received a placebo. Several other company-sponsored trials of low-dose CBD for insomnia were not published, McGregor adds, because they found no significant improvement. The search for more effective insomnia treatments continues in other realms, as well. Some research groups are experimenting with different receptors that they hope could lead to new classes of drugs. Gabriella Gobbi, a clinical psychiatrist and research neuroscientist at McGill University in Montreal, Canada, for example, has homed in on one of the brain's two melatonin receptors, MT2. “We want to find an alternative mechanism without any addiction liability and with fewer side effects, especially for use in children and elderly people,” she says. A few companies and health systems, including the US Department of Veterans Affairs and the Cleveland Clinic in Ohio, have also created or are developing digital platforms for delivering CBT-I. These apps take users through regimens that are tailored to their symptoms. SleepioRx, for example, is a 90-day digital programme that has been evaluated in more than two dozen clinical trials and has showed efficacy as high as 76%. Uptake among physicians has been slow so far, Krystal says. But once practitioners catch on, he adds, “I can imagine a world where you have digital care as your first stop, and if that's not successful, you see a therapist.” Some studies suggest that insomnia can stem from a high level of underlying brain activity during sleep. This raises the question of whether reducing this activity could treat insomnia, says Ruth Benca, a psychiatrist at Wake Forest School of Medicine in North Carolina. Companies and academic research groups are beginning to test this proposition with wearable devices that use auditory tones or mild electrical stimulation to increase slow-wave activity in the brain. Last June, for example, researchers at Elemind Technologies in Cambridge, Massachusetts, confirmed that auditory stimuli delivered in sync with specific brain-wave rhythms generated in a headband allowed people who usually struggle for more than 30 minutes to fall asleep to shave an average of 10.5 minutes off that time. Even after a lifetime of struggling to find safe and effective help, Miranda says that she still holds out hope that better treatments for insomnia are on the horizon. “I can't be on these medications forever,” she says. Her latest book is I Feel Love: MDMA and the Quest for Connection in a Fractured World (Bloomsbury, 2023).</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            