
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-12-20</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/u-s-plan-to-drop-some-childhood-vaccines-to-align-with-denmark-will-endanger/'>U.S. Plan to Drop Some Childhood Vaccines to Align with Denmark Will Endanger Children, Experts Say</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-20 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The move could set public health back decades, experts say The move, first reported by CNN, would change how many vaccines to protect against various diseases children get and when they receive those immunizations. Recommendations for several vaccines that are currently given routinely to children in the U.S.—including shots for rotavirus, varicella (chickenpox), hepatitis A, meningococcal bacteria, influenza and respiratory syncytial virus (RSV)—could be scrapped entirely under the plans, according to CNN. Childhood vaccines collectively protect children and the U.S. population as a whole against diseases, such as measles and hepatitis B, that once sickened, hospitalized or killed hundreds or even thousands every year. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Changing what vaccines kids get would be “a terrible mistake,” says Jessica Malaty Rivera, an infectious disease epidemiologist at Defend Public Health, an all-volunteer organization sponsored by a nonprofit. More children could get sick and die from preventable illnesses as a result. The two available shots, which are not technically vaccines but antibody drugs that protect against RSV, were approved in 2023 and 2025 and are more than 90 percent effective at protecting against hospitalization. The Trump administration has previously stated that it wants to model the U.S.'s vaccine policy after other developed countries and specifically Denmark, which recommends fewer vaccines than the U.S. does and recommends them at different times of life. But it doesn't make sense to compare the U.S. to countries, such as Denmark, that have a vastly different health care system. “We can learn a lot from some studies that come from other countries, but we have to use a critical mind to figure out what is applicable to our context and what isn't,” says Jennifer Nuzzo, an epidemiologist and director of the Pandemic Center at Brown University. A key difference between the U.S. and Denmark that Kennedy and other U.S. health officials seem to avoid is that the European country has a national health care system that covers everyone for free while the U.S. does not. “Denmark or other places have universal health coverage where people don't fall into health care gaps like they do in the United States. Whatever the CDC recommends influences what private health insurers will cover and what federal programs, such as the Vaccines for Children program, will subsidize. “When changes are made to the schedule, it will have consequences for who is able to get vaccines, whether or not you want them,” Nuzzo says. “This isn't about allowing you to opt out. This is about making it harder for you to opt in.” The Department of Health and Human Services had scheduled a press conference about children's health on Friday but has since pushed the announcement back until next year. If these further changes come to pass, they will chip away at the collective protection against deadly infectious diseases, Nuzzo says. Individual medical providers and states may step up to preserve access to vaccines, but people could still slip through the cracks of an increasingly patchwork public health system. “We have to make public health recommendations that work for all. There are clearly people who can't spend a bulk of their time trying to find the credible sources of information,” Nuzzo says. “I'm worried about people who just won't get the lifesaving protection that they need.” Young is associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. Tanya Lewis is senior desk editor for health and medicine at Scientific American. Previously, she has written for outlets that include Insider, Wired, Science News and others. She has a degree in biomedical engineering from Brown University and one in science communication from the University of California, Santa Cruz. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67609-4'>Generalizable and scalable protein stability prediction with rewired protein generative models</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-12-20 08:19:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. , Article number:  (2025) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Predicting changes in protein thermostability caused by amino acid substitutions is essential for understanding human diseases and engineering proteins for practical applications. While recent protein generative models demonstrate impressive zero-shot performance in predicting various protein properties without task-specific training, their strong unsupervised prediction ability remains underexploited to improve protein stability prediction. We present SPURS, a deep learning framework that rewires and integrates two complementary protein generative models–a protein language model and an inverse folding model–and reprograms this unified framework for stability prediction through supervised fine-tuning on mega-scale thermostability data. SPURS delivers accurate, efficient, and scalable stability predictions and generalizes to unseen proteins and mutations. Beyond stability prediction, SPURS enables broad applications in protein informatics, including zero-shot identification of functional residues, improved low-N protein fitness prediction, and systematic dissection of stability-pathogenicity for human diseases. Together, these capabilities establish SPURS as a versatile tool for advancing protein stability prediction and protein engineering at scale. Unless otherwise stated, all data supporting the results of this study can be found in the article, supplementary, and source data files. We used esm2_t33_650M_UR50D checkpoint of ESM2, v_48_020 checkpoint of ProteinMPNN [https://github.com/dauparas/ProteinMPNN/blob/main/vanilla_model_weights_48_020.pt], and esm1v_t33_650M_UR90S_1 checkpoint of ESM1v [https://dl.fbaipublicfiles.com/fair-esm/models/esm1v_t33_650M_UR90S_1.pt]. The Megascale dataset was downloaded from [https://zenodo.org/records/7844779]. Megascale split, Fireprot(HF), S669, Ssym-direct, and Ssym-inverse were downloaded from [https://github.com/Kuhlman-Lab/ThermoMPNN]. S783, S2648, S461, S8754, S4346 and S571 were downloaded from [https://github.com/Gonglab-THU/GeoStab]. Domainome data and the performance of baseline models were downloaded from [https://zenodo.org/records/11260616]. Clinvar annotations were collected from [https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/]86. Cancer-associated mutations were obtained from the Cancer Mutation Census v99 (GRCh38) dataset, specifically the file CancerMutationCensus_AllData_Tsv_v99_GRCh38, available at [https://cancer.sanger.ac.uk/cosmic/download/cancer-mutation-census/v99/alldata-cmc]. Source data are provided with this paper. SPURS was implemented in Python using the PyTorch (v1.12.0) library. The source code is available at [https://github.com/luo-group/SPURS]125. Mesbahuddin, M. S., Ganesan, A. & Kalyaanamoorthy, S. Engineering stable carbonic anhydrases for CO2 capture: a critical review. Google Scholar Wu, S., Snajdrova, R., Moore, J. C., Baldenius, K. & Bornscheuer, U. T. Biocatalysis: enzymatic synthesis for industrial applications. Google Scholar Shan, S. et al. Deep learning guided optimization of human antibody against SARS-CoV-2 variants with broad neutralization. Google Scholar Kuhlman, B. & Bradley, P. Advances in protein structure prediction and design. Cell Biol. Google Scholar Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Google Scholar Quan, L., Lv, Q. & Zhang, Y. STRUM: structure-based prediction of protein stability changes upon single-point mutation. Google Scholar Li, B., Yang, Y. T., Capra, J. Predicting changes in protein thermodynamic stability upon point mutation with deep 3D convolutional neural networks. Google Scholar Chen, Y. et al. PremPS: predicting the impact of missense mutations on protein stability. Google Scholar Pancotti, C. et al. Predicting protein stability changes upon single-point mutation: a thorough comparison of the available tools on a new dataset. Google Scholar Umerenkov, D. et al. PROSTATA: a framework for protein stability assessment using transformers. Google Scholar Li, G., Yao, S. & Fan, L. ProSTAGE: predicting effects of mutations on protein stability by using protein embeddings and graph convolutional networks. Google Scholar Benevenuta, S., Pancotti, C., Fariselli, P., Birolo, G. & Sanavia, T. An antisymmetric neural network to predict free energy changes in protein variants. D Appl. Google Scholar Pancotti, C. et al. A deep-learning sequence-based method to predict protein stability changes upon genetic variations. Google Scholar Montanucci, L., Capriotti, E., Frank, Y., Ben-Tal, N. & Fariselli, P. DDGun: an untrained method for the prediction of protein stability changes upon single and multiple point variations. BMC Bioinforma. Google Scholar Dehouck, Y. et al. Fast and accurate predictions of protein stability changes upon mutations using statistical potentials and neural networks: PoPMuSiC-2.0. Google Scholar Xavier, J. S. et al. ThermoMutDB: a thermodynamic database for missense mutations. Nucleic Acids Res. Google Scholar Nikam, R., Kulandaisamy, A., Harini, K., Sharma, D. & Michael Gromiha, M. ProThermDB: thermodynamic database for proteins and mutants revisited after 15 years. Nucleic Acids Res. Google Scholar Stourac, J. et al. FireProtDB: database of manually curated protein stability data. Nucleic Acids Res. Google Scholar Gromiha, M. M. et al. ProTherm, version 2.0: thermodynamic database for proteins and mutants. Nucleic Acids Res. Google Scholar Wang, C. Y. et al. ProtaBank: a repository for protein design and engineering data. Protein Sci. Google Scholar Tsuboyama, K. et al. Mega-scale experimental analysis of protein folding stability in biology and design. Google Scholar Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Google Scholar Elnaggar, A. et al. ProtTrans: toward understanding the language of life through self-supervised learning. Pattern Anal. Google Scholar Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Hayes, T. et al. Simulating 500 million years of evolution with a language model. Dauparas, J. et al. Robust deep learning–based protein sequence design using proteinmpnn. Google Scholar Hsu, C., Nisonoff, H., Fannjiang, C. & Listgarten, J. Learning protein fitness models from evolutionary and assay-labeled data. Google Scholar Meier, J., Rao, R., Verkuil, R., Liu, J., Sercu, T. & Rives, A. Language models enable zero-shot prediction of the effects of mutations on protein function. Neural Inf. Google Scholar Notin, P. et al. ProteinGym: large-scale benchmarks for protein fitness prediction and design. Neural Inf. Reeves, S. & Kalyaanamoorthy, S. Zero-shot transfer of protein sequence likelihood models to thermostability prediction. Google Scholar Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Google Scholar Hie, B. L. et al. Efficient evolution of human antibodies from general protein language models. Google Scholar Cagiada, M., Ovchinnikov, S. & Lindorff-Larsen, K. Predicting absolute protein folding stability using generative models. Protein Sci. Google Scholar Blaabjerg, L. M. et al. Rapid protein stability prediction using deep learning representations. Google Scholar Chu, S. K. S., Narang, K. & Siegel, J. Protein stability prediction by fine-tuning a protein language model on a mega-scale dataset. Google Scholar Zhao, J., Zhang, C. & Luo, Y. Contrastive fitness learning: Reprogramming protein language models for low-N learning of protein fitness landscape. In Proc. International Conference on Research in Computational Molecular Biology 470–474 (Springer, 2024). Hsu, C. et al. Learning inverse folding from millions of predicted structures. In Proc. International Conference on Machine Learning, 8946–8970 (PMLR, 2022). Dieckhaus, H., Brocidiacono, M., Randolph, N. Z. & Kuhlman, B. Transfer learning to leverage larger datasets for improved prediction of protein stability changes. Google Scholar Widatalla, T., Rafailov, R. & Hie, B. Aligning protein generative models with experimental fitness via direct preference optimization. Beltran, A., Jiang, X., Shen, Y. & Lehner, B. Site-saturation mutagenesis of 500 human protein domains. UniProt Consortium. UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Res. Zheng, Z. et al. Structure-informed language models are protein designers. In Proc. International Conference on Machine Learning 42317–42338 (PMLR, 2023). Houlsby, N. et al. Parameter-efficient transfer learning for NLP. In Proc. International Conference on Machine Learning 2790–2799 (PMLR, 2019). Sanavia, T. et al. Limitations and challenges in protein stability prediction upon genome variations: towards future applications in precision medicine. Google Scholar Xu, Y., Liu, D. & Gong, H. Improving the prediction of protein stability changes upon mutations by geometric learning and a pre-training strategy. Pak, M. A., Dovidchenko, N. V., Sharma, S. M. & Ivankov, D. N. New mega dataset combined with deep neural network makes a progress in predicting impact of mutation on protein stability. Pucci, F., Bernaerts, K. V., Kwasigroch, J. M. & Rooman, M. Quantification of biases in predictions of protein stability changes upon mutations. Google Scholar Schymkowitz, J. et al. The FoldX web server: an online force field. Nucleic Acids Res. Google Scholar Alford, R. F. et al. The Rosetta all-atom energy function for macromolecular modeling and design. Google Scholar Diaz, D. J. et al. Stability Oracle: a structure-based graph-transformer framework for identifying stabilizing mutations. Google Scholar Ouyang-Zhang, J., Diaz, D., Klivans, A., & Krähenbühl, P. Predicting a protein's stability under a million mutations. Benevenuta, S., Birolo, G., Sanavia, T., Capriotti, E. & Fariselli, P. Challenges in predicting stabilizing variations: an exploration. Google Scholar Capriotti, E., Fariselli, P., Rossi, I. & Casadio, R. A three-state prediction of single point mutations on protein stability changes. BMC Bioinforma. Google Scholar Cheng, J. et al. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Google Scholar Tokuriki, N. & Tawfik, D. S. Stability effects of mutations and protein evolvability. Google Scholar Faure, A. J. et al. Mapping the energetic and allosteric landscapes of protein binding domains. Google Scholar Høie, M. H., Cagiada, M., Frederiksen, A. H. B., Stein, A., & Lindorff-Larsen, K. Predicting and interpreting large-scale mutagenesis data using analyses of protein stability and conservation. Li, X. & Lehner, B. Biophysical ambiguities prevent accurate genetic prediction. Google Scholar Weng, C., Faure, A. J., Escobedo, A. & Lehner, B. The energetic and allosteric landscape for KRAS inhibition. Google Scholar Otwinowski, J. Biophysical inference of epistasis and the effects of mutations on protein stability and function. Google Scholar Faure, A. J. et al. The genetic architecture of protein stability. Faure, A. J. & Lehner, B. MoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data. Wang, J. et al. The conserved domain database in 2023. Nucleic Acids Res. Google Scholar Cagiada, M. et al. Discovering functionally important sites in proteins. Google Scholar Agu, P. C. et al. Molecular docking as a tool for the discovery of molecular targets of nutraceuticals in diseases management. Google Scholar Biswas, S., Khimulya, G., Alley, E. C., Esvelt, K. M. & Church, G. M. Low-N protein engineering with data-efficient deep learning. Google Scholar Riesselman, A. J., Ingraham, J. & Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. Google Scholar Hopf, T. A. et al. Mutation effects predicted from sequence co-variation. Google Scholar Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. & Church, G. M. Unified rational protein engineering with sequence-based deep representation learning. Google Scholar Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Google Scholar Stenson, P. D. et al. The human gene mutation database: towards a comprehensive repository of inherited mutation data for medical research, genetic diagnosis and next-generation sequencing studies. Google Scholar Pandey, P., Ghimire, S., Wu, B. & Alexov, E. On the linkage of thermodynamics and pathogenicity. Google Scholar Wang, Z. & Moult, J. SNPs, protein structure, and disease. Google Scholar Yue, P., Li, Z. Loss of protein structure stability as a major causative factor in monogenic disease. Google Scholar Casadio, R., Vassura, M., Tiwari, S., Fariselli, P. & Martelli, P. L. Correlating disease-related mutations to their effect on protein stability: a large-scale analysis of the human proteome. Google Scholar Gao, M., Zhou, H. & Skolnick, J. Insights into disease-associated mutations in the human proteome through protein structural analysis. Google Scholar Stein, A., Fowler, D. M., Hartmann-Petersen, R. & Lindorff-Larsen, K. Biophysical and mechanistic models for disease-causing protein variants. Trends Biochem. Google Scholar Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. Google Scholar Gerasimavicius, L., Livesey, B. J. & Marsh, J. A. Loss-of-function, gain-of-function and dominant-negative mutations have profoundly different effects on protein structure. Google Scholar Gudmundsson, S. et al. Variant interpretation using population databases: Lessons from gnomad. Google Scholar Tiemann, J. K. S., Zschach, H., Lindorff-Larsen, K. & Stein, A. Interpreting the molecular mechanisms of disease variants in human transmembrane proteins. Google Scholar Ahmed, S. S. et al. Characterization of intrinsically disordered regions in proteins informed by human genetic diversity. Google Scholar Ruff, K. M. & Pappu, R. V. AlphaFold and implications for intrinsically disordered proteins. Google Scholar Tesei, G. et al. Conformational ensembles of the human intrinsically disordered proteome. Google Scholar Savojardo, C., Manfredi, M., Martelli, P. L. & Casadio, R. Solvent accessibility of residues undergoing pathogenic variations in humans: from protein structures to protein sequences. Google Scholar Cagiada, M., Jonsson, N. & Lindorff-Larsen, K. Decoding molecular mechanisms for loss-of-function variants in the human proteome. Zhang, J. et al. Assessing predictions on fitness effects of missense variants in HMBS in CAGI6. Benegas, G., Albors, C., Aw, A. J., Ye, C. & Song, Y. S. A DNA language model based on multispecies alignment predicts the effects of genome-wide variants. Cagiada, M. et al. Understanding the origins of loss of protein function by analyzing the effects of thousands of variants on activity and abundance. Google Scholar Veitia, R. A., Caburet, S. & Birchler, J. A. Mechanisms of Mendelian dominance. Google Scholar McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University. Online Mendelian Inheritance in Man, OMIM® https://omim.org/ (2025). Tokuriki, N., Stricher, F., Schymkowitz, J., Serrano, L. & Tawfik, D. S. The stability effects of protein mutations appear to be universally distributed. Google Scholar Sahni, N. et al. Widespread macromolecular interaction perturbations in human genetic disorders. Google Scholar Jubb, H. C. et al. Mutations at protein-protein interfaces: small changes over big surfaces have large impacts on human health. Google Scholar Livesey, B. J. & Marsh, J. The properties of human disease mutations at protein interfaces. Google Scholar Kontomanolis, E. N. et al. Role of oncogenes and tumor-suppressor genes in carcinogenesis: a review. Anticancer Res. Google Scholar Chillón-Pino, D., Badonyi, M., Semple, C. A. & Marsh, J. Protein structural context of cancer mutations reveals molecular mechanisms and candidate driver genes. Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. Nucleic Acids Res. Google Scholar Sondka, Z. et al. The cosmic cancer gene census: describing genetic dysfunction across all human cancers. Google Scholar Gkouveris, I., Nikitakis, N. G., Aseervatham, J. & Ogbureke, K. U. E. The tumorigenic role of dspp and its potential regulation of the unfolded protein response and ER stress in oral cancer cells. Google Scholar Yoshida, T. et al. CLK2 is an oncogenic kinase and splicing regulator in breast cancer. Google Scholar Bordes, F. et al. An introduction to vision-language modeling. arXiv preprint https://doi.org/10.48550/arXiv.2405.17247 (2024). Qiu, J. et al. InstructPLM: aligning protein language models to follow protein structure instructions. A. et al. Adapting protein language models for structure-conditioned design. Dieckhaus, H. & Kuhlman, B. Protein stability models fail to capture epistatic interactions of double point mutations. Protein Sci. Ng, P. C. & Henikoff, S. SIFT: predicting amino acid changes that affect protein function. Nucleic Acids Res. Kircher, M. et al. A general framework for estimating the relative pathogenicity of human genetic variants. Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Li, G., Panday, S. K. & Alexov, E. SAAFEC-SEQ: a sequence-based method for predicting the effect of single point mutations on protein thermodynamic stability. Adzhubei, I., Jordan, D. M. & Sunyaev, S. R. Predicting functional effect of human missense mutations using PolyPhen-2. Pejaver, V. et al. Inferring the molecular and phenotypic impact of amino acid variants with MutPred2. Critical Assessment of Genome Interpretation Consortium. CAGI the critical assessment of genome interpretation, establishes progress and prospects for computational genetic variant interpretation methods. Genome Biol. Ingraham, J., Garg, V., Barzilay, R. & Jaakkola, T. Generative models for graph-based protein design. Jing, B., Eismann, S., Suriana, P., Townshend, R. J. L. & Dror, R. Learning from protein structure with geometric vector perceptrons. Vaswani, A. Attention is all you need. Ding, N. et al. Parameter-efficient fine-tuning of large-scale pre-trained language models. He, K., Zhang, X., Ren, S., & Sun, J. Deep residual learning for image recognition. In Proc. IEEE Conference on Computer Vision and Pattern Recognition 770–778, (IEEE, 2016). Hendrycks, D. & Gimpel, K. Gaussian error linear units (GELUs). arXiv preprint https://doi.org/10.48550/arXiv.1606.08415 (2016). & Hutter, F. Decoupled weight decay regularization. In International Conference on Learning Representations (2019). Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Kunzmann, P. & Hamacher, K. Biotite: a unifying open source computational biology framework in Python. BMC Bioinforma. Tien, M. Z., Meyer, A. G., Sydykova, D. K., Spielman, S. J. & Wilke, C. O. Maximum allowed solvent accessibilites of residues in proteins. PLoS ONE 8, e80635 (2013). Hernández, I., Dehouck, Y., Bastolla, U., Ramón López-Blanco, J. & Chacón, P. Predicting protein stability changes upon mutation using a simple orientational potential. Steinegger, M. & Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Code for “Generalizable and scalable protein stability prediction with rewired protein generative models”. This work is supported in part by the National Institute of General Medical Sciences of the National Institutes of Health under award R35GM150890 (Y.L.). The authors acknowledge the computational resources provided by the Delta GPU Supercomputer at NCSA of UIUC through allocation CIS230097 (Y.L.) from the Advanced Cyberinfrastructure Coordination Ecosystem: Services & Support (ACCESS) program, which is supported by NSF grants #2138259, #2138286, #2138307, #2137603, and #2138296 (Y.L. ), and the cloud computational resources provided by Microsoft Azure through the Cloud Hub program at GaTech IDEaS and the Microsoft Accelerate Foundation Models Research (AFMR) program (Y.L. School of Computational Science and Engineering, Georgia Institute of Technology, Atlanta, GA, USA Ziang Li & Yunan Luo Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and supervised the project. developed the computational framework and performed the evaluation analyses. wrote the manuscript. Correspondence to Yunan Luo. The authors declare no competing interests. Nature Communications thanks Emil Alexov, Arne Elofsson, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Li, Z., Luo, Y. Generalizable and scalable protein stability prediction with rewired protein generative models. Accepted: 03 December 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            