
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-30</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08930-2'>TIR domains produce histidine-ADPR as an immune signal in bacteria</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 16:19:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Toll/interleukin-1 receptor (TIR) domains are central components of pattern recognition immune proteins across all domains of life1,2. In bacteria and plants, TIR-domain proteins recognize pathogen invasion and then produce immune signalling molecules exclusively comprising nucleotide moieties2,3,4,5. Here we show that the TIR-domain protein of the type II Thoeris defence system in bacteria produces a unique signalling molecule comprising the amino acid histidine conjugated to ADP-ribose (His-ADPR). His-ADPR is generated in response to phage infection and activates the cognate Thoeris effector by binding a Macro domain located at the C terminus of the effector protein. By determining the crystal structure of a ligand-bound Macro domain, we describe the structural basis for His-ADPR and its recognition and show its role by biochemical and mutational analyses. Our analyses furthermore reveal a family of phage proteins that bind and sequester His-ADPR signalling molecules, enabling phages to evade TIR-mediated immunity. These data demonstrate diversity in bacterial TIR signalling and reveal a new class of TIR-derived immune signalling molecules that combine nucleotide and amino acid moieties. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access Buy this article Prices may be subject to local taxes which are calculated during checkout All data are available in the Article and the supplementary material. Strains of bacteria and phages and plasmid maps of the constructs used for the experiments are attached as Supplementary Tables 1 and 2. The atomic coordinates and structure factors have been deposited in the Protein Data Bank (PDB) under accession codes 8V3E (MoTad2), 9EIB (His-ADPR-bound MoTad2) and 8R66 (His-ADPR-bound BaMacro). & Kagan, J. C. Toll-like receptors and the control of immunity. Essuman, K., Milbrandt, J., Dangl, J. L. & Nishimura, M. T. Shared TIR enzymatic functions regulate cell death and immunity across the tree of life. Ofir, G. et al. Antiviral activity of bacterial TIR domains via immune signalling molecules. Bayless, A. M. et al. Plant and prokaryotic TIR domains generate distinct cyclic ADPR NADase products. Li, S., Manik, M. K., Shi, Y., Kobe, B. & Ve, T. Toll/interleukin-1 receptor domains in bacterial and plant immunity. Doron, S. et al. Systematic discovery of antiphage defense systems in the microbial pangenome. Ka, D., Oh, H., Park, E., Kim, J.-H. & Bae, E. Structural and functional evidence of bacterial antiphage protection by Thoeris defense system via NAD+ degradation. Manik, M. K. et al. Cyclic ADP ribose isomers: production, chemical structures, and immune signaling. Viruses inhibit TIR gcADPR signalling to overcome bacterial defence. Multiple phage resistance systems inhibit infection via SIR2-dependent NAD+ depletion. Zaremba, M. et al. Short prokaryotic Argonautes provide defence against incoming mobile genetic elements through NAD+ depletion. Karras, G. I. et al. The macro domain is an ADP-ribose binding module. Rack, J. G. M., Perina, D. & Ahel, I. Macrodomains: structure, function, evolution, and catalytic activities. Duncan-Lowey, B., McNamara-Bordewick, N. K., Tal, N., Sorek, R. & Kranzusch, P. J. Effector-mediated membrane disruption controls cell death in CBASS antiphage defense. Yirmiya, E. et al. Phages overcome bacterial immunity via diverse anti-defence proteins. Holm, L. Dali server: structural unification of protein families. Structural and functional analysis of Oceanobacillus iheyensis macrodomain reveals a network of waters involved in substrate binding and catalysis. Guberovic, I. et al. Evolution of a histone variant involved in compartmental regulation of NAD metabolism. Rack, J. G. M., Palazzo, L. & Ahel, I. (ADP-ribosyl)hydrolases: structure, function, and biology. Tamulaitiene, G. et al. Activation of Thoeris antiviral system via SIR2 effector filament assembly. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Li, D. et al. Single phage proteins sequester signals from TIR and cGAS-like enzymes. Jia, A. et al. TIR-catalyzed ADP-ribosylation reactions produce signaling molecules for plant immunity. Yu, D. et al. TIR domains of plant immune receptors are 2′,3′-cAMP/cGMP synthetases mediating cell death. Structural basis of SARM1 activation, substrate recognition, and inhibition by small molecules. Systematic and quantitative view of the antiviral arsenal of prokaryotes. Cohen, D. et al. Cyclic GMP–AMP signalling protects bacteria against viral infection. Whiteley, A. T. et al. Bacterial cGAS-like enzymes synthesize diverse nucleotide signals. & Kranzusch, P. J. CBASS phage defense and evolution of antiviral nucleotide signaling. Tal, N. et al. Cyclic CMP and cyclic UMP mediate bacterial immunity against phages. Type III CRISPR–Cas systems produce cyclic oligoadenylate second messengers. Kazlauskiene, M., Kostiuk, G., Venclovas, Č., Tamulaitis, G. & Siksnys, V. A cyclic oligonucleotide signaling pathway in type III CRISPR–Cas systems. Athukoralage, J. S. & White, M. F. Cyclic nucleotide signaling in phage defense and counter-defense. Chi, H. et al. Antiviral type III CRISPR signalling via conjugation of ATP and SAM. Hobbs, S. J. et al. Phage anti-CBASS and anti-Pycsar nucleases subvert bacterial immunity. Athukoralage, J. S. et al. An anti-CRISPR viral ring nuclease subverts type III CRISPR immunity. Huiting, E. et al. Bacteriophages inhibit and evade cGAS-like immune function in bacteria. Jenson, J. M., Li, T., Du, F., Ea, C.-K. & Chen, Z. J. Ubiquitin-like conjugation by bacterial cGAS enhances anti-phage defence. Spear, A. M., Loman, N. J., Atkins, H. S. & Pallen, M. J. Microbial TIR domains: not necessarily agents of subversion? Gerdts, J., Brace, E. J., Sasaki, Y., DiAntonio, A. & Milbrandt, J. SARM1 activation triggers axon degeneration locally via NAD+ destruction. The SARM1 Toll/Interleukin-1 receptor domain possesses intrinsic NAD+ cleavage activity that promotes pathological axonal degeneration. Horsefield, S. et al. NAD+ cleavage activity by animal and plant TIR domains in cell death pathways. Identification and receptor mechanism of TIR-catalyzed small molecules in plant immunity. Rousset, F. et al. TIR signaling activates caspase-like immunity in bacteria. Mazzocco, A., Waddell, T. E., Lingohr, E. & Johnson, R. P. in Methods in Molecular Biology, vol. 501 (eds Clokie, M. R. & Kropinski, A. M.) 81–85 (Humana Press, 2009). Zheng, L. et al. Fumarate induces redox-dependent senescence by modifying glutathione metabolism. Frey, S. & Görlich, D. A new set of highly efficient, tag-cleaving proteases for purifying recombinant proteins. 426 (eds Kobe, B., Guss, M. & Huber, T.) 197–208 (Humana Press, 2008). Unger, T., Jacobovitch, Y., Dantes, A., Bernheim, R. & Peleg, Y. Applications of the restriction free (RF) cloning procedure for molecular manipulations and protein expression. Collaborative Computational Project, No. The CCP4 suite: programs for protein crystallography. McCoy, A. J. et al. Phaser crystallographic software. Song, Y. et al. High-resolution comparative modeling with RosettaCM. Terwilliger, T. C. et al. Iterative model building, structure refinement and density modification with the PHENIX AutoBuild wizard. Towards automated crystallographic structure refinement with phenix.refine. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. Moriarty, N. W., Grosse-Kunstleve, R. W. & Adams, P. D. Electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation. Heyrovska, R. Structures of the molecular components in DNA and RNA with bond lengths interpreted as sums of atomic covalent radii. Mirdita, M. et al. ColabFold: making protein folding accessible to all. Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Vonrhein, C. et al. Data processing and analysis with the autoPROC toolbox. Evans, P. R. & Murshudov, G. N. How good are my data and what is the resolution? The authors thank O. Izsak for help with mass spectrometry analysis; M. Itkin and S. Malitsky for conducting the mass spectrometry experiments; Y. Peleg and S. Albeck for assistance with MoTad2 purification; M. Malikenas for assisting in His-ADPR purification procedure; and Sorek laboratory members for comments on earlier versions of this manuscript. X-ray diffraction data were collected at the beamline P13 operated by EMBL Hamburg at the PETRA III storage ring (DESY, Hamburg, Germany) and at the Northeastern Collaborative Access Team beamlines 24-ID-C and 24-ID-E (P30 GM124165). Access to the EMBL P13 beamline has been supported by iNEXT-Discovery, project number 871037, funded by the Horizon 2020 programme of the European Commission. NE-CAT data collection was supported by use of a Pilatus detector (S10RR029205), an Eiger detector (S10OD021527), and the Argonne National Laboratory Advanced Photon Source (DE-AC02-06CH11357). was supported, in part, by the European Research Council (grant ERC-AdG GA 101018520), Israel Science Foundation (MAPATS grant 2720/22), the Deutsche Forschungsgemeinschaft (SPP 2330, grant 464312965), the Ernest and Bonnie Beutler Research Program of Excellence in Genomic Medicine, Dr. Barry Sherman Institute for Medicinal Chemistry, Miel de Botton, the Andre Deloro Prize, and the Knell Family Center for Microbiology. was supported, in part, by the Pew Biomedical Scholars programme and The G. Harold and Leila Y. Mathers Charitable Foundation. was supported by the Research Council of Lithuania (LMTLT) agreement no. E.H. was supported by the Israel Cancer Research Fund Postdoctoral Fellowship. was supported by the Clore Scholars Program, and, in part, by the Israeli Council for Higher Education (CHE) via the Weizmann Data Science Research Center. These authors contributed equally: Dziugas Sabonis, Carmel Avraham, Renee B. Chang Institute of Biotechnology, Life Sciences Center, Vilnius University, Vilnius, Lithuania Dziugas Sabonis, Arunas Silanskas, Deividas Vilutis, Audrone Ruksenaite, Mindaugas Zaremba & Giedre Tamulaitiene Department of Molecular Genetics, Weizmann Institute of Science, Rehovot, Israel Carmel Avraham, Ehud Herbst, Azita Leavitt, Erez Yirmiya, Ilya Osterman, Gil Amitai & Rotem Sorek Department of Microbiology, Harvard Medical School, Boston, MA, USA Renee B. Chang, Allen Lu, Hunter C. Toyoda & Philip J. Kranzusch Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA, USA Renee B. Chang, Allen Lu, Hunter C. Toyoda & Philip J. Kranzusch You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed the research. carried out bacterial genetic and phage infection assays, phage infection dynamics, His-ADPR pulldown from lysates, biochemical experiments and mass spectrometry analyses. and E.H. carried out biochemical experiments and mass spectrometry analyses. carried out a phylogenetic analysis of Tad2. A. Leavitt carried out bacterial genetics and phage infection assays. performed cloning and mutagenesis of ThsA and its BaMacro domain. A.S. carried out the type II Thoeris proteins purifications. characterized BaMacro and its mutants, performed E. coli survivability assay, western blot analysis and biochemical experiments and analysed the results. carried out mass spectrometry analysis. and A.S purified His-ADPR. A. Lu crystallized apo MoTad2 and solved its structure. crystallized MoTad2–His-ADPR complex and solved its structure with assistance from H.C.T. wrote the manuscript with input from all authors. All authors reviewed and edited the manuscript and supported the conclusions. Correspondence to Philip J. Kranzusch, Rotem Sorek or Giedre Tamulaitiene. is a scientific cofounder and advisor of BiomX and Ecophage. The other authors declare no competing interests. Nature thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Growth curves of Thoeris-expressing and control cultures with and without infection by phage SBSphiJ at an MOI of 10. Data from three replicates are presented as individual curves. OD600, optical density at 600 nm. b, Same as in panel A, but with cells infected at MOI of 0.1. c, A representative plaque assay showing that both type I and type II Thoeris systems protect from phage SBSphiJ. Shown are tenfold serial dilution plaque assays with phage SBSphiJ. Representative of three replicates. Multiple sequence alignment was performed using the HHpred server59. a, b, Two repeats of the mass spectrometry (MS) analysis shown in Fig. 3a, analyzed in both positive and negative modes. Cells expressing BaThsB or control cells that express GFP instead were infected with phage SBSphiJ at MOI = 10. After 120 min the cells were lysed and lysates filtered, then incubated with purified MoTad2. The lysates prior to and post-incubation with MoTad2, or after denaturation of MoTad2, were analyzed by mass spectrometry (MS). Masses visualized in positive mode are in the m/z range of 697.1262 – 697.1462 with a retention time of 8.5 – 9.5 min. Masses visualized in a negative mode in S3 are in the m/z range of 695.1153 – 695.1353 with a retention time of 8.5 – 9.5 min. Extracted mass chromatograms of ions with an m/z value of 697.1374 and retention time of 9.04 min are presented in positive mode. In negative mode, extracted mass chromatograms of ions with an m/z value of 695.1251 are presented. Data in panel a are the same data presented in Fig. a, b Two repeats of the mass spectrometry analysis of the level of His-ADPR molecules produced by BaThsB before and after infection with phage SBSphiJ. A 50 mL culture of cells expressing BaThsB were collected before infection and 120 min after infection with phage SBSphiJ at MOI = 10. Expression of BaThsB was induced by 100 μM IPTG. The cells were lysed and lysates were filtered and analyzed by untargeted mass spectrometry. Shown are masses recorded in positive ionization mode in the m/z range of 3 PPM around the theoretical m/z of His-ADPR with a retention time of 8.5 – 9.5. a, LC-MS analysis of BaMacro protein purified from the cells expressing BaThsB protein or its active site E99A mutant. Shown are the extracted mass chromatograms (EIC) of ions with an m/z value of 697.1374 corresponding to His-ADPR. The mass spectrogram is shown for the gray-colored area in the EIC. Theoretical m/z of His-ADPR ions: [M + H]+ 697.1374, [M+Na]+ 719.1198. b, Topology diagram of BaMacro domain structure. Residues that make side-chain contacts to the ligand are marked by red circles. c, Superposition of the BaMacro domain (blue) with an ADPR-bound type OiMacroD domain of bacterium Oceanobacillus iheyensis (yellow, PDB ID 5LAU, Dali Z-score 15.3, r.m.s.d. 2.8 Å2 over 168 aligned atoms, 17% identity)18 and MacroH2A-like macrodomain from Capsaspora owczarzaki CoMacroH2A (green, PDB ID 7NY7, Dali Z-score 13.7, r.m.s.d. 2.7 Å2 over 152 aligned atoms, 13% identity)19. The BaMacro domain possesses longer loops near the binding pocket (colored purple). In b, c Unique BaThsA elements: a β hairpin (strands β4 and β5) and a small beta-sheet (strands β8, 9, 11) are marked. d, Conserved residues of the ligand binding pockets of the BaMacro (blue), OiMacroD (yellow) and CoMacroH2A (green). Ligand molecules bound in the binding pocket are colored respectively. OiMacroD catalytic aspartate D40 is not conserved in BaThsA and CoMacroH2A. e, Detailed view of the BaThsA residues interacting with His-ADPR. Yellow dashed lines denote hydrogen bonding interactions. A and N marks A- and N-ribose, respectively. f, Western blot analysis of cell lysates from cells expressing BaThsA and its mutants. M - PageRuler™ Prestained Protein Ladder, C41 - E. coli C41 cell lysate. The arrow indicates the bands containing BaThsA (Mw 36.558 kDa). Shown is a cropped image of one of two replicates (provided in Supplementary Fig. a, LC-MS analysis of ThsB (from B. amyloliquefaciens Y2 and E. rectale ATCC 33656) in vitro reactions. Mass spectrograms indicate the presence of His-ADPR ions (theoretical m/z values [M + H]+ 697.1374, [M+Na]+ 719.1198). No His-ADPR is present in the ErThsB-E99A sample. b, LC-MS analysis of the purified His-ADPR. His-ADPR molecule was obtained from the BaMacro domain preparation by denaturing the protein and purifying His-ADPR by HPLC. Shown is the UV chromatogram and mass spectrogram of the interval specified by the gray area. Theoretical masses of His-ADPR ions: [M + H]+ 697.1374, [M + 2H]+2 349.0726. a, SEC-MALS analysis of the purified BaMacro and its mutants. Blue lines indicate molecular weight estimates. b, Predicted oligomer contacts in the BaThsA AlphaFold 3 model. Subunits are shown in different colors, dimers are marked by dashed lines. Interface residues T222 (T) and S226 (S) are shown in sphere representation. c, Protein melting data showing Tm values of apo- and His-ADPR bound BaMacro domain and its T222R/S226R mutant. Tm value of apo T222R/S226R mutant could not be determined (n.d.). Bars represent an average of three replicates with individual data points overlaid. a, Structure of apo MoTad2 tetramer (top) and Bacillus phage SPO1 Tad223 (PDB ID 8SMF, bottom). b, Comparison of the canonical gcADPR binding site observed in the structures of MoTad2 and SPO1 Tad2, with a yellow sphere modeled in the cavity. c, Comparison of the putative second binding pocket observed in MoTad2 and SPO1 Tad2, with a red sphere modeled centrally in the cavity. Mutated MoTad2 pockets residues (see, Fig. 4d) are shown in stick representation in the inlets of b and c. Specified catalytic glutamate (E) residue corresponds to E99A in BaThsB and ErThsB. We propose that ThsB acts via a double-displacement catalytic mechanism suggested for the TIR domain of SARM125, in which a covalent intermediate of the glycosyl-enzyme complex with the catalytic glutamate residue is formed. A list of bacterial strains and phages used in this study. A list of plasmids and primers used in this study. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Sabonis, D., Avraham, C., Chang, R.B. et al. TIR domains produce histidine-ADPR as an immune signal in bacteria. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08809-2'>Serotonin and neurotensin inputs in the vCA1 dictate opposing social valence</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 16:19:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The ability to evaluate valence of a social agent based on social experience is essential for an animal's survival in its social group1. Although hippocampal circuits have been implicated in distinguishing novel and familiar conspecifics2,3,4,5,6,7, it remains unclear how social valence is constructed on the basis of social history and what mechanisms underlie the heightened valence versatility in dynamic relationships. Here we demonstrate that the ventral (v)CA1 integrates serotonin (5-HT) inputs from the dorsal raphe and neurotensin inputs from the paraventricular nucleus of the thalamus (PVT) to determine positive or negative valence of conspecific representations. Specifically, during an appetitive social interaction 5-HT is released into the vCA1 and disinhibits pyramidal neurons through 5-HT1B receptors, whereas neurotensin is released during an aversive social interaction and potentiates vCA1 neurons directly through NTR1s. Optogenetic silencing of dorsal raphe 5-HT and PVT neurotensin inputs into the vCA1 impairs positive and negative social valence, respectively, and excitation flexibly switches valence assignment. These results show how aversive and rewarding social experiences are linked to conspecific identity through converging dorsal raphe 5-HT and PVT neurotensin signals in the vCA1 that instruct opposing valence, and represent a synaptic switch for flexible social valence computation. Many mammals live in social groups and navigating a social environment requires the ability to flexibly update valence of a social agent, unlike the predetermined valence towards predator odour, foot shock or food reward1,8,9,10. Owing to the dynamic nature of social relationships, the temporal scales and plasticity may differ from learned valence of inanimate objects and contexts11,12,13,14. Indeed, it is poorly understood how social history influences social valence and how valence information can be flexibly updated. In a changing environment, how does the brain integrate different and even conflicting information to calculate new behavioural decisions? We reasoned that distinct neuromodulatory inputs from converging circuit motifs may represent a potential neural plasticity mechanism, which would allow flexible behavioural switch based on social history. To interrogate this hypothesis, we established new behavioural models to determine the causal relationship between neuromodulator release and social valence. Unlike innate valence, assessment of social valence has been challenging in rodents, as it is difficult to dissociate conditioning to a non-social stimulus (for example, foot shock pairing) from valence of the social interaction itself15,16. To address how social valence is regulated we developed new social cognitive models, which enabled us to assess valence association with a conspecific. We presented strong stimuli consisting of a potential mate or an aggressor before a three-chamber social memory assay to elicit rewarding and aversive social experiences (Fig. First, we evaluated whether C57BL/6J test mice show memory of aggressor identity using a standard social memory assay without valenced interactions, as subjects only encounter the aggressive mice under a cup. Indeed, C57BL/6J subjects can distinguish novel and familiar aggressive mice, as evidenced by their preference for the chamber containing the novel mouse (Extended Data Fig. It is important to dissociate innate valence, such as natural response to predators, from social valence, which can be flexibly assigned based on social history1. Thus, we examined whether subjects showed any avoidance behaviour towards aggressive mice under a cup and found that without previous direct interaction, test mice showed no preference (Extended Data Fig. To assess negative social valence assignment, the subject mouse sequentially interacts with a same-sex aggressor or a non-aggressor (male CD-1 or female CFW) for 5 min separated by 5-min intervals (Fig. After a 5-min interval, mice were tested for social valence assignment to the aggressor in a three-chamber box, where both aggressor and non-aggressor were confined under a cup. With previous direct aggressor and non-aggressor interaction, both male and female subjects preferred the chamber with the non-aggressor in the subsequent social valence test (Fig. 1c), thereby demonstrating negative social valence association with the conspecific based on the previously aversive interaction. a, Left, schematic of the negative social valence (sv) test. Right, duration subjects spent in chamber containing the non-aggressor with a previous more neutral (ntrl) encounter or in chamber containing the aggressor with a previous more negative (neg) encounter (t17 = 4.091, P = 0.0008, n = 18). b, Left, schematic of positive social valence test. Right, duration subjects spent in chamber containing the opposite-sex partner with a previous more neutral (ntrl) encounter or in chamber containing the potential with a previous more positive (pos) encounter (t25 = 5.98, P = 0.000003, n = 26). c,d, Schematic and representative image of injection site. Social memory test: duration in chamber with familiar mouse (fm) or novel mouse (nm) and discrimination (d) scores (c, t16 = 3.841, P = 0.0014, n = 17; d, t11 = 3.885, P = 0.0025, n = 12). Negative sv, duration in either chamber and d scores (c, t16 = 4.224, P = 0.0006, n = 17; d, t11 = 3.552, P = 0.0045, n = 12). Positive sv, duration in either chamber and d scores (c, t17 = 3.817, P = 0.0014, n = 18; d, t11 = 2.817, P = 0.0168, n = 12). Statistical tests: two-tailed paired Student's t-test (a–d); one-way ANOVA with Tukey's post hoc test, Kruskal–Wallis test was used for NDB and vCA1 (f). Scale bars, 200 μm (c,d). To test positive social valence association, C57BL/6J subjects undergo sequential 5-min direct interactions with a potential mate and a ‘neutral' partner followed by a 5-min three-chamber test, each separated by 5-min intervals (Fig. Without previous direct interaction, male subjects do not show any preference for females in proestrus or oestrus cycles over other cycles in the three-chamber assay (Extended Data Fig. Since female subjects may not experience the same level of reward during a male partner interaction, we constructed a barrier small enough that the female could escape to another compartment and thus control the timing of the interaction, which has been reported to be rewarding20,21. The neutral (less positive) experiences are without mating posture or barrier in a new cage. After a direct neutral and potential mate interaction, the male and female subjects spent more time in the chamber with the potential mate in the subsequent social valence test (Fig. 1e), thereby demonstrating positive social valence association with the conspecific based on the previously appetitive interaction. The dorsal (d)CA2 and vCA1 are critical for distinguishing novel and familiar conspecifics2,3,4,5,6,22, but it remains unclear whether they participate in valence association of social representations. To investigate the role of the dCA2 and vCA1 in social valence, we expressed an inhibitory designer receptor exclusively activated by designer drugs (DREADD) hM4Di fused to mCherry (mCh) in the dCA2 of Amigo2-Cre mice, a dCA2-specific Cre-driver line2, or vCA1 of C57BL/6J mice and administered the DREADD activator clozapine-N-oxide (CNO) or saline before three different social memory and valence tests. In the social memory test2,3,7, subjects encounter a novel and a familiar age-matched same-sex conspecific, which are relatively neutral interactions. After saline administration as a within animal control, subjects with hM4Di expression in the dCA2 (Fig. 1d) showed normal social memory as well as negative and positive social valence assignment, as they spent more time in the chamber with the novel mouse, the non-aggressor and the potential mate, respectively. However, the same cohorts of hM4Di-expressing mice receiving CNO showed impaired social memory and valence (Fig. The discrimination scores3,7, which allowed us to directly compare saline and CNO conditions, confirmed the disruption of both social memory and valence in mice in which dCA2 and vCA1 neuronal activity was inhibited (Fig. As an extra control, we expressed mCh alone in the dCA2 and the vCA1, which did not affect social memory nor valence after saline or CNO administration (Extended Data Fig. Baseline sociability, assessed by the initial phase of the neutral social memory test, was not affected by hM4Di-mediated silencing of the dCA2 and vCA1, nor by expression of mCh in the control cohort (Extended Data Fig. To test whether the positive social valence test is primarily driven by reproductive related behaviours, we assessed the time subjects spent in the chamber with a male or female mouse under a cup. Like saline administration, CNO administration in the hM4Di vCA1 cohort did not decrease the preference of subjects for the chamber with females over males (Extended Data Fig. Further, the number of times female subjects escaped through the barrier from male partners did not significantly differ after hM4Di-mediated vCA1 inhibition, suggesting that the positive social valence test does not affect innate sexual behaviours (Extended Data Fig. Although our results show that information encoded in the dCA2 and vCA1 is important for both positive and negative social valence association, it is possible that an impairment of conspecific identity memory also affects valence evaluation. To determine whether the dCA2 and vCA1 are indeed involved in social valence encoding, independent assessment of valence and identity is necessary. We postulated that different hippocampal inputs may send information mediating positive and negative association and thus be dissociable when manipulated separately. To uncover dCA2 and vCA1-projecting regions, involved in valence induction, we exposed TRAP2;Ai14 mice to a neutral social partner, a potential mate and an aggressor, and quantified the number of tdTomato-positive cells in the dCA2, vCA1 and all extra-hippocampal input regions (Fig. Increased numbers of tdTomato-positive cells were detected in the aPVT, reuniens and median raphe only after an aversive interaction and in the DR only after a rewarding interaction (Fig. To address whether distinct inputs can confer opposing social valence, we used a viral intersectional chemogenetic approach to achieve projection-specific manipulations. 3), we injected canine adenovirus type 2 expressing Cre (CAV2-Cre) into the vCA1 and Cre-dependent inhibitory DREADD AAV-DIO-hM4Di-mCh into the DR (Fig. In two control cohorts, mice expressing hM4Di with saline administration or expressing mCh with saline or CNO administration showed normal social memory (Fig. 2c) and positive social valence (Fig. On the contrary, hM4Di mice that received CNO showed impaired positive social valence, while social memory and negative social valence were unperturbed (Fig. Consistent with manipulations of the vCA1, hM4Di-mediated silencing of vCA1-projecting DR neurons did not alter baseline sociability (Extended Data Fig. Because the vCA1 is implicated in non-social valence association24,25,26,27,28, which could influence social valence, we tested whether inhibiting the DR to vCA1 projection induced conditioned place preference (CPP) or aversion. hM4Di-mediated silencing of the vCA1-projecting DR cells had no effect on non-social reward and aversion as test mice spent an equal amount of time on the surfaces paired with CNO or saline (Extended Data Fig. Neither sociability nor CPP were significantly affected by CNO administration in the mCh control cohort (Extended Data Fig. a, Schematic of injection and representative image of AAV-DIO-hM4Di-mCh expression in the DR. b, Left, duration in chamber with familiar mouse (fm) or novel mouse (nm). Right, discrimination (d) scores (F3,36 = 0.04147, P = 0.9886, n = 13). c, Left, duration in chamber containing non-aggressor with previous neutral encounter or aggressor with previous negative encounter. Right, d scores (F3,36 = 0.3357, P = 0.7996, n = 13). d, Left, duration in chamber containing partner with previous ntrl encounter or potential mate with previous pos encounter. Right, d scores (F3,36 = 3.992, P = 0.0149, n = 13). f–h, Left, duration that subjects spent in chamber containing fm or nm and d scores (F3,26 = 0.2621, P = 0.852, hM4Di n = 11; mCh n = 9) (f), non-aggressor with previous ntrl encounter or aggressor with previous neg encounter and d scores (F3,25 = 3.556, P = 0.0286, hM4Di n = 10; mCh n = 9) (g) and a partner with previous ntrl encounter or potential mate with previous positive encounter and d scores (hM4Di F3,26 = 0.7687, P = 0.522, n = 11; mCh n = 9) (h). Statistical tests: duration, two-tailed paired Student's t-test. d score, two-way ANOVA with Šidák's post hoc test. Wilcoxin rank test was only used for mCh saline (d) and hM4Di saline duration (h). Scale bars, 200 μm (a,e). To identify inputs mediating negative social valence, we used the same viral intersectional strategy to chemogenetically manipulate projections of the median raphe to the dCA2, reuniens to the vCA1 and aPVT to the vCA1 based on connectivity found in our tracing experiment. Inhibition of the median raphe to dCA2 projection impaired social memory but not sociability (Extended Data Fig. 4c,d), whereas inactivation of the reuniens to vCA1 projection had no effect on social memory, social valence nor sociability (Extended Data Fig. hM4Di-mediated silencing of the aPVT to vCA1 projection (Fig. Saline administration in the same hM4Di cohort or CNO administration in the mCh control cohort did not influence social memory nor valence (Fig. No manipulations of vCA1-projecting aPVT neurons in hM4Di or mCh-expressing cohorts affected baseline sociability and CPP (Extended Data Fig. Similarly to previously reported studies29,30, bath application of CNO significantly reduced the excitability of hM4Di-expressing DR and PVT cells compared to non-expressing control cells (Extended Data Fig. Together, these results indicate that positive and negative social valence information converge in the vCA1 through DR and PVT inputs. We questioned how vCA1 neurons dissociate opposing valence information. To explore the possibility that distinct neuromodulators facilitate memory valence encoding in the same brain region, we assessed differential neuromodulatory signals from the DR and PVT. As the DR and PVT are major release sites for 5-HT (refs. 23,31) and neurotensin (NT)9,32, respectively, we tested whether DR 5-HT and aPVT NT neurons send projections to the vCA1. Consistent with previous reports9,23,33, expressing Cre-dependent GFP in the DR of Sert-Cre mice (Extended Data Fig. 5a) and Cre-dependent mCh in the aPVT of NT-Cre mice (Extended Data Fig. 5b) revealed 5-HT and NT axon innervation in the vCA1. To assess the involvement of DR 5-HT neurons in positive social valence induction, we expressed Cre-dependent NpHR3.0eYFP or enhanced yellow fluorescent protein (eYFP) alone in the DR and implanted optical fibres adjacent to the vCA1 (Fig. Optogenetic inhibition of DR 5-HT inputs in the vCA1 did not alter social memory (Extended Data Fig. 5c) and negative social valence (Fig. 3b) but only blocked positive social valence assignment (Fig. The same manipulation did not affect baseline sociability (Extended Data Fig. 5d) nor induce real-time place preference (RTPP) (Extended Data Fig. 5e), suggesting no change in innate reward and aversion. Moreover, the NpHR3.0 cohort without light stimulation (light off) or the eYFP cohort with or without light stimulation (light on or off) showed no significant changes in social memory, social valence, sociability and RTPP (Fig. 3b,c and Extended Data Fig. a,d,j, Schematic and representative image of virus injection in the DR (a,j) and PVT (d). b,e, Left, duration in chamber with ntrl partner or chamber with neg valenced partner (Sert-Cre (b); NT-Cre (e)). c,f, Left, duration in chamber with ntrl partner or chamber with positive valenced partner. h,i, Left, time courses of average GRAB sensor transient z scores event-locked to direct interactions. k, Left, duration in chamber with neutral partner or chamber with negative valenced partner and d scores (t16 = 1.1, P = 0.2874, Tph2: n = 10; NTC: n = 8). Right, duration in chamber with neutral partner or chamber with positive valenced partner and d scores (t16 = 3.335, P = 0.0042, Tph2 n = 10; NTC n = 8). l, Representative images of virus expression in the vCA1 with TPH2 immunohistochemistry shown in magenta. Quantification of the percentage of the TPH2 area (t16 = 3.752, P = 0.0017, Tph2: n = 10; NTC: n = 8). Statistical tests: duration (b,c,e,f,k); two-tailed paired Student's t-test, Wilcoxin rank test was only used for YFP off duration (b,f); d score (b,c,e), two-way ANOVA with Šidák's post hoc test; d score (f), Kruskal–Wallis test with Dunn's post hoc test; one-way ANOVA with Tukey's post hoc test (h,i); d score (k,l), unpaired Student's t-test. To address whether PVT NT neurons exert opposing effects, we injected AAVs expressing Cre-dependent NpHR3.0 in the aPVT and implanted optic fibre cannulas above the vCA1 to silence PVT NT inputs in the vCA1 (Fig. Optogenetic inhibition during initial interaction and three-chamber testing specifically impaired negative social valence association (Fig. 3e) without affecting social memory (Extended Data Fig. 5f) and positive social valence (Fig. Again, light-off conditions in the NpHR3.0 cohort or light on and off conditions in the eYFP cohort did not alter social memory and valence (Fig. Baseline sociability and RTPP were not significantly affected by those manipulations (Extended Data Fig. To evaluate NpHR3.0-mediated silencing, we expressed Cre-dependent NpHR3.0eYFP and RCaMP2, a calcium indicator, in the DR or PVT of the respective Cre-driver lines and implanted optogenetics fibre cannulas above the injection sites as well as fibre photometry cannulas above the vCA1. Light stimulation of NpHR3.0 strongly diminished terminal activity during social interactions in both cohorts (Extended Data Fig. Our results predict enhanced 5-HT release into the vCA1 during an appetitive social interaction and increased NT release during an aversive one. Therefore, we expressed GRAB5HT3.5 or GRABNT1.0 (ref. 9) in the ventral hippocampus and placed fibre implants adjacent to the vCA1 to record 5-HT or NT activity while the subject was exposed to sequential interactions with a same-sex age-matched conspecific, a potential mate and an aggressor (Fig. Robustly elevated 5-HT activity was detected only during the appetitive interaction, whereas interactions with the neutral and aversive partners produced smaller increases in 5-HT activity (Fig. By contrast, NT signals showed a marked increase only during an aversive interaction, with neutral and rewarding social encounters only generating moderate NT activity (Fig. When we expressed a membrane-bound enhanced green fluorescent protein (EGFP) as a control, we did not detect any significant changes in fluorescence between different social encounters (Extended Data Fig. Because DR 5-HT neurons also corelease glutamate into the vCA1 (ref. 23), we used CRISPR interference (CRISPRi) combined with a viral intersectional strategy to specifically knockdown the 5-HT producing enzyme tryptophan hydroxylase 2 (encoded by Tph2) in vCA1-projecting DR neurons. Initially, we validated knockdown efficiency by expressing AAVs carrying three single-guide RNAs (sgRNAs) targeting Tph2 (AAV-sgTph2) or non-targeting control sgRNAs (AAV-sgNTC) along with AAV-Cre in the DR of dCas9-KRAB mice, where dCas9 expression can be turned on by Cre recombination. Immunostaining for TPH2 in the DR revealed a roughly 75% reduction in fluorescence intensity in sgTph2-expressing mice compared to non-targeting controls (Extended Data Fig. To further quantitatively assess knockdown efficiency, we performed quantitative PCR (qPCR) on DR primary cultured neurons infected with AAV-Cre and AAV-sgTph2 or AAV-sgNTC, which revealed a 98% reduction in Tph2 messenger RNA (mRNA) level after knockdown (Extended Data Fig. We then injected AAV-sgTph2 into the DR and CAV2-Cre into the vCA1 of dCas9-KRAB mice (Fig. Knockdown of Tph2 in vCA1-projecting DR neurons only ablated positive social valence but not negative social valence, social memory and sociability (Fig. No changes in those behaviours were detected in control mice expressing non-targeting control sgRNA (Fig. To further confirm knockdown, we assayed TPH2 expression in DR terminals in the vCA1 of the behavioural cohort and found a roughly 80% reduction in TPH2-positive terminals in sgTph2 mice compared to non-targeting controls (Fig. Our data indicate that 5-HT and NT promote opposing social valence. Since DR 5-HT and PVT NT neurons can corelease other neuromodulators9,23, we assessed whether perturbing 5-HT and NT action is sufficient to block social valence evaluation by directly inhibiting 5-HT and NT receptors in the vCA1. To uncover receptor subtypes involved, we locally infused receptor (R) antagonists (A) into the vCA1 20 min before social memory tests (Fig. NTR1-A (SR48692) but not NTR2-A (NTRC 824) micro-infusion blocked negative social valence (Fig. Although the NTR1-A (SR48692) did not influence positive social valence, a promiscuous 5-HTR-A (methiothepin) impaired it (Fig. Next, we targeted the 5-HT1BR, as it is known to be involved in sociability and social memory when released into the nucleus accumbens31 and medial septum7, respectively. Indeed, local infusion of the 5-HT1BR-A (NAS-181) blocked positive (Fig. 4c) but not negative social valence (Fig. To examine the specificity of this receptor, we also infused an antagonist against 5-HT1AR (NAD299). The results show a P value of 0.07, suggesting that 5-HT1AR is most probably not involved in positive social valence regulation (Fig. Neither NTR1-A nor 5-HT1BR-A infused into the vCA1 had detectable effects on social memory (Fig. When comparing discrimination scores, NTR1 antagonist micro-infusion into the vCA1 seem to specifically impair negative social valence, whereas 5-HT1BR antagonist infusion specifically blocked positive social valence (Extended Data Fig. a, Schematic and representative image. e–g, Time courses of GCaMP6f transient z scores event-locked to direct interactions. e, Left, schematic and representative image. h,i, Schematic of experiment and pie charts of cell responses (h, n = 10, i, n = 11) with representative traces. j, Left, sample traces of vCA1 pyramidal neuron spiking in response to drug. CP versus baseline results have an extra * at 60, 100–140 pA. k,l, Left, summary time course of EPSCs and IPSCs. Right, average PSC (percentage of control) with sample traces at time points 1 and 2 (k, t11 = 2.92, P = 0.0139, EPSC n = 6, IPSC n = 7; l, t14 = 3.076, P = 0.0082, EPSC, IPSC n = 8). Statistical tests: two-tailed paired Student's t-test (b–d); one-way ANOVA with Tukey's post hoc test (e–g); two-way ANOVA mixed-effects with Tukey's post hoc test (j); two-tailed unpaired Student's t-test (k,l); Wilcoxon rank test was only used for NAD299 (b) and saline (c). Our data indicate that NTR1-A and 5-HT1BR-A can regulate vCA1 neuronal activity during differently valenced social interactions. To evaluate this hypothesis, we expressed a calcium indicator GCaMP6f in the vCA1 and recorded vCA1 neuronal activity during sequential novel object, neutral, negatively and positively valenced social interactions. Like previous studies3,34, we observed a robust increase of vCA1 activity, during all types of social interaction compared to object interaction (Fig. Administration of NTR1-A specifically decreased the activity during an aversive social interaction (Fig. 4f), whereas 5-HT1BR-A administration only attenuated the activity during a rewarding one (Fig. To assess the effect of NTR1-A on PVT inputs, we expressed channelrhodopsin2 (ChR2) in the aPVT of C57BL/6J mice and performed ex vivo optogenetic recordings in the vCA1. Light stimulation evoked action potentials in half of vCA1 cells and bath application of the NTR1-A (SR48692) significantly reduced responses (Fig. To evaluate DR inputs, we used the same approach and expressed ChR2 in the DR. Light stimulation induced responses in more than 90% of vCA1 cells and 5-HT1BR-A (NAS-181) application strongly diminished responses (Fig. To interrogate how 5-HT and NT release may cause an increase in vCA1 activity during different social encounters, we performed ex vivo whole-cell recordings to assess the excitability of vCA1 cells in response to NTR1 (0.5 µM PD149163) and 5-HT1BR (5 µM CP93129) agonists. Bath application of both NTR1 (PD) and 5-HT1BR (CP) agonists increased vCA1 neuronal spiking in response to depolarizing current pulses (Fig. To address if both receptors work through the same or different synaptic mechanisms, we further assessed evoked and spontaneous postsynaptic currents. Because 5-HT1BRs are commonly presynaptic and inhibit neurotransmitter release35 but NTR1s have been reported to be postsynaptic and excitatory36,37, we predicted differential synaptic mechanisms. Consistent with our hypothesis, bath application of the specific NTR1 agonist, PD149163, increased evoked excitatory postsynaptic currents (EPSCs) and had minimal impact on evoked inhibitory postsynaptic currents (IPSCs) (Fig. Further, a higher percentage of cells showed a change in evoked EPSC compared to IPSC amplitude in response to PD149163 (Extended Data Fig. Likewise, bath application of the 5-HT1BR agonist, CP93129, decreased evoked IPSCs but had no significant effects on evoked EPSCs (Fig. 4l), which suggests that the increase in excitability is due to depression of inhibitory inputs onto the vCA1 pyramidal neurons. Consistently, a greater proportion of cells showed a change in IPSC compared to EPSC amplitude in response to CP93129 (Extended Data Fig. To assess pre- versus postsynaptic mechanisms, we measured spontaneous postsynaptic currents (sPSCs). Bath application of PD149163 increased sEPSC amplitude, but not frequency (Extended Data Fig. 6d) and had minimal effects on sIPSC amplitude and inter-event interval (Extended Data Fig. By contrast, CP93129 increased sIPSC frequency, but not amplitude (Extended Data Fig. The results show that NT increases excitability by directly exciting vCA1 pyramidal neurons, whereas 5-HT acts through disinhibition by decreasing the inhibitory input onto them. To assess whether inhibitory neurons in the vCA1 express 5-HT1BRs, we performed in situ hybridization with probes targeting the γ-aminobutyric acid-expressing (GABAergic) neuron marker, Gad2 and 5-HT1BR (Htr1b). Quantification of Gad2+ cells revealed that 44% express Htr1b (Extended Data Fig. Next, we tested whether positive and negative social valence flexibly mapped onto the same, overlapping or distinct populations of cells. Therefore, we tagged cells activated by a negative or positive social experience with tdTomato and used cFos immunostaining to label re-activated cells in the same or different social context (Extended Data Fig. Quantification of tdTomato-positive cells stained with cFos antibody revealed a 64–77% overlap when animals were exposed to the same social context, whereas the overlap decreased by roughly 40% when the second context had the opposite social valence (Extended Data Fig. We reasoned that these distinct cell populations encoding opposing social valence could be due to regionally separated innervation of differential neuromodulator inputs and/or heterogeneity in receptor expression. Hence, we examined DR and PVT innervation of the vCA1 by expressing GFP in the aPVT and mCh in the DR of the same brains and found overlapping but not identical areas of innervation (Extended Data Fig. Then, we evaluated receptor composition of cells encoding positive or negative social valence by using in situ hybridization with probes targeting NTR1 (Ntsr1) and 5-HT1BR (Htr1b) in TRAP2;Ai14 vCA1 cells that were labelled by tdTomato after a rewarding or aversive social experience (Extended Data Fig. Quantification of tdTomato cells positive for Ntsr1, revealed that roughly 80% of cells participating in negative social valence express Ntsr1, whereas only roughly 45% of cells participating in positive social valence showed expression (Extended Data Fig. There is also a trend towards a decrease, from roughly 40 to 30%, in the proportion Htr1b-expressing cells active during an aversive compared to an appetitive social interaction (Extended Data Fig. The data suggest that separate populations of vCA1 pyramidal neurons may be preferentially potentiated for distinct valence associations. Having established that social valence is encoded in cells that are potentiated by 5-HT and NT, we proposed that endogenous release of those neuromodulators can attribute valence to the memory of a conspecific. To determine whether we could invert the polarity of social valence, we expressed Cre-dependent ChR2-mCh in DR 5-HT neurons of Sert-Cre mice. Because NT is still released during the aversive social interaction, it is possible that release of both 5-HT and NT signals contradictory valence and causes the ‘impairment' phenotype. Blocking NT signalling might shift the balance in favour of positive social valence. Consistent with this prediction, simultaneous NTR1-A (SR48692) administration and 5-HT release during an aversive social interaction completely switched social valence, whereas administration of NTR1-A without light stimulation inhibited positive social valence (Fig. 5-HT release during the entire duration of the sociability and social memory tests caused no behavioural effects (Extended Data Fig. Likewise, this manipulation did not significantly influence RTPP (Extended Data Fig. 8c), nor male and female sexual behaviours (Extended Data Fig. a, Top, schematic of experiment. Bottom left, schematic and representative image of injection site. Bottom right, duration in chamber with ntrl or neg valenced partner and discrimination scores (F2,24 = 3.253, P = 0.0483, n = 11). b, Top, schematic of experiment. Bottom left, schematic and representative image of injection site. Bottom right, duration in chamber with ntrl or pos valenced partner and discrimination scores (F2,21 = 11.11, P = 0.0004, n = 10). d–g, Left, duration in chamber containing mouse (m) 1 or object (o) 1 without previous light pairing, or in chamber containing m2 or o2, with previous light stimulation during the 10-min social interaction phase. m1/o1 and m2/o2 are the same mice and objects in the same experiment, but not between experiments. i, Left, duration in chamber with ntrl or pos valenced partner. j, Left, sample traces of vCA1 pyramidal neuron spiking. Asterisks are depicted for CP versus pre-CP. Scale bars: 200 μm (a,b,h); 100 ms (x axis), 20 mV (y axis) (j). Statistical tests: duration (a,b,d,f,g,i), d scores; two-tailed paired Student's t-test (d–g); one-way ANOVA with Tukey's post hoc test (a,b,i); two-way ANOVA mixed-effects with Tukey's post hoc test (j); Wilcoxin rank test was used for NTR1-A off duration (a) and duration (e). To test bi-directionality, we expressed Cre-dependent ChR2-mCh in the aPVT of NT-Cre mice and placed optical fibres adjacent to the vCA1. When the light was off, saline administered mice showed normal positive social valence, whereas activation of NT inputs in the vCA1 during the appetitive social interaction impaired it (Fig. Further, 5-HT1BR-A administration with simultaneous NT release during the rewarding social interaction inversed social valence, whereas 5-HT1BR-A administration alone in the light-off condition merely disrupted positive social valence (Fig. Sociability, social memory, RTPP nor anxiety were influenced by NT release into the vCA1 (Extended Data Fig. By manipulating 5-HT and NT release in combination with receptor antagonists, we were able to invert the polarity of social valence with the same social partner during repeated exposure. Moreover, these results also indicate that 5-HT and NT induce opposite valence signals that are integrated for social valence calculation as presence of both ablated positive and negative social valence assignment. Next, we asked whether valence can be conferred to a neutral social stimulus by manipulating 5-HT and NT signalling. To test this, we allowed subjects to interact with two novel mice under a cup in separate chambers, where light stimulation was paired with one chamber only and memory was tested without stimulation after a 5-min interval (Fig. When the light was off in session 1, Sert-Cre mice did not show a preference for either of the familiar mice under a cup in session 2 (Fig. To address whether novelty, regardless of social interaction, drives the behaviour, we used the same experimental set-up replacing novel mice with novel objects (Fig. Light pairing with an object did not evoke preference in the subsequent memory test (Fig. To examine whether NT release can also convey negative social valence to an otherwise more neutral stimulus, we manipulated NT release in the same experimental set-up (Fig. Activation of NT inputs in the vCA1 caused avoidance of the previously light-paired mouse (Fig. 5f), but not object in the second session (Fig. Given social cognitive capabilities are commonly impaired in autism spectrum disorder (ASD)38, we asked whether our findings can be leveraged to specifically rescue positive social valence in heterozygous Shank3Δ4-22+/− mice. Because sociability is not impaired in Shank3Δ4-22−/− mice39, but Shank3 mutations disrupt social memory40,41, we surmised that Shank3Δ4-22+/− mice would show a similar deficit. Consistent with previous reports39, Shank3Δ4-22+/− and wild-type littermates showed normal sociability after saline micro-infusion into the vCA1, which was not altered by infusion of 5-HT1BR agonist, CP93129 (Extended Data Fig. By contrast, social memory was impaired in Shank3Δ4-22+/− mice compared to wild-type littermates and CP93129 infusion did not rescue the deficits (Extended Data Fig. To activate 5-HT1BRs during an appetitive but not neutral social interaction, we exposed the mice to the neutral social interaction immediately after CP93129 infusion, before the drug had time to diffuse. After a 15-min incubation, we exposed the subjects to an appetitive social interaction and then tested for social valence (Fig. Notably, 5-HT1BR agonist infusion rescued positive social valence in Shank3Δ4-22+/− mice to a level comparable to wild-type mice (Fig. To assess whether the 5-HT1BR agonist acts through increasing excitability of vCA1 cells, we performed current-clamp recordings in the vCA1 of Shank3Δ4-22+/− and wild-type littermates. Indeed, spike numbers were lower in Shank3Δ4-22+/− mice compared to wild-type littermates, but bath application of the 5-HT1BR agonist rescued the decrease in excitability in the mutant mice (Fig. Our results demonstrate that two neuromodulatory inputs, DR 5-HT and PVT NT, converge in a single brain region, the vCA1, to balance social valence by means of distinct mechanisms. 5-HT signals in the vCA1 were specifically elevated during a rewarding social interaction, whereas NT activity increased only during an aversive one. Our findings suggest that whereas 5-HT potentiates vCA1 pyramidal neurons by reducing the inhibitory inputs through presynaptic 5-HT1BRs, NT increases excitability by directedly activating NTR1 on the pyramidal neurons. Moreover, our data indicate that 5-HT and NT affect distinct or possibly overlapping populations of vCA1 neurons, as cells activated by rewarding and aversive social experiences differ in their 5-HT1BR and NTR1 composition in that they can be preferentially potentiated by 5-HT or NT (Extended Data Fig. These results not only extend previous findings suggesting distinct vCA1 cell populations are activated by appetitive and aversive stimuli24, but also reveal that neuromodulator receptor expression can be a strong determinant in valence information routing. In addition, the physiological and behavioural effects of 5-HT and NT contrast non-social valence assignment in the amygdala, where a single neuromodulator gates directionality independent of receptor expression9. Our experiments show that 5-HT conveys positive, and NT negative valence association with a conspecific, which can be flexibly switched within minutes. Thus, activation of 5-HT1BRs specifically restored positive social valence assignment as well as vCA1 excitability in an ASD mouse model haplo-sufficient for Shank3 Exon 4-22. This suggests that enhanced 5-HT1BR activity can compensate for the reduction in synaptic plasticity caused by mutation of Shank3, which might be a common mechanism applicable to many ASD-risk gene mutations affecting synaptic transmission. Given that information of conspecific identity is routed from the dCA2 to the vCA1 (refs. 4,42,43), we postulate that 5-HT and NT increase excitability of a subset of vCA1 neurons so that incoming identity information is preferentially mapped onto the potentiated population and thereby overlayed with valence. Since manipulation of 5-HT and NT release into the vCA1 lead to distinct behavioural outcomes, it is possible that information stored in the 5-HT and NT potentiated vCA1 neurons is differentially routed to promote appropriate action-selection. This represents a basic mechanism defining how the brain processes and updates many characteristics of a given entity. Further studies investigating the downstream target regions receiving inputs from those cells will thus be informative. None of our projection- and neuromodulator-specific manipulations affected sexual behaviours, anxiety, baseline sociability and RTPP, indicating that innate valence is not affected. As manipulations of 5-HT and NT in other brain regions have been found to influence one or many of those behaviours9,31,44,45, our findings extend growing evidence that behavioural effects of the neuromodulators are highly region specific. Assuming the high regional heterogeneity of vCA1 neurons along with expression of various neuromodulator receptors46,47, it is possible that further neuromodulatory signals such as oxytocin, norepinephrine or dopamine48,49, are multiplexed to promote more specific types of social representation. This converging neuromodulatory motif has the potential to integrate complex information into a behaviourally relevant outcome and may represent a shared principle used by behaviours including social dominance, cooperation and foraging. In sum, we have identified physiological correlates of a neural competition mechanism for flexible valence association, which is capable of integrating social history. Our findings indicate that further simultaneous exploration of several neuromodulators in converging circuit motifs will not only provide critical insights into complex social behaviours but also reveal potential therapeutic targets such as 5-HT1BR and NTR1, which can be leveraged to improve social cognitive deficits in neuropsychiatric disorders including ASD and schizophrenia. 50), Amigo2-Cre2 (B6.Cg-Tg(Amigo2-cre)1Sieg/J; Jackson Laboratory, 30215), Sert-Cre (Mouse Mutant Resource and Research Centers, stock number 017260-UCD, strain code: Tg(Slc6a4-cre)ET33Gsat/Mmucd), NT-Cre44 (a gift from S. Russo) and, Shank3Δ4-22 (ref. TRAP2;Ai14 mice were generated by crossing TRAP2 (STOCK Fostm2.1(icre/ERT2)Luo/J; Jackson Laboratory, 30323) and Ai14 (B6.Cg-Gt(ROSA)26Sortm14(CAG-tdTomato)Hze/J; Jackson Laboratory, 7914) mice. Male retired breeder CD-1 (Charles River, ICR), female Swiss-Webster (Charles River, CFW) as well as male and female C57BL/6J mice were used as partner mice. Except for singly housed aggressors, mice were housed with 2–5 mice per cage and weaned at 21 days old. All behavioural experiments were conducted with 7–14-week-old mice. Animals were maintained on a 12-h light–dark cycle at roughly 21 °C with 50% humidity and food and water ad libitum. Behavioural experiments were performed during the same circadian period (07:00–19:00). Experiments were conducted in accordance with the National Institutes of Health Guide for Care and approved by the Use of Laboratory Animals and the Icahn School of Medicine at Mount Sinai Institutional Animal Care and Use Committee. Sample sizes were not predetermined by statistical analysis but based on previous experience with the variance of the assays. The sequence of control and manipulated conditions were shuffled. Mice (4–8 weeks old) were anaesthetized with intramuscular injections of a drug mixture containing ketamine (100 mg kg−1) and xylazine (5 mg kg−1) and heads were then positioned on a stereotaxic apparatus (David Kopf Instruments) for virus injections (0.2–0.5 μl) at a flow rate of 0.1–0.25 μl min−1 with a microinjection pump (Harvard Apparatus). The following bregma coordinates from dura were used to target the virus solution filled glass cannula: dCA2 (bilateral, anteroposterior −1.6 mm; mediolateral ±1.6 mm; dorsoventral 1.7 mm), vCA1 (bilateral, anteroposterior −3.16; mediolateral ±3.1; dorsoventral −4.55), DR (unilateral, anteroposterior −4.36; mediolateral 0; dorsoventral −3.1), aPVT (unilateral, anteroposterior −0.4; mediolateral 0; dorsoventral −3.8), median raphe (unilateral, anteroposterior −4.4; mediolateral 0; dorsoventral −4) and reuniens (bilateral, anteroposterior −0.2; mediolateral ±0.85; dorsoventral −4.3). Adeno-associated viruses (AAVs) used for stereotaxic injections were purchased from Addgene, WZ Biosciences and the Stanford Neuroscience Gene Vector and Virus Core and included: AAVDJ-hSyn-EGFP, AAVDJ-hSyn-hM4Di-mCh, AAVDJ-hSyn-DIO-hM4D(Gi)-mCh, AAVDJ-hSyn-mCh, AAVDJ-EF1α-DIO-eYFP, AAVDJ-hSyn-ChR2-eYFP, AAVDJ-hSyn-ChR2-mCh, AAVDJ-hSyn-GCaMP6f, AAVDJ-EF1α-DIO-RCaMP2, AAVDJ-EF1α-DIO-NpHR3.0eYFP, AAVDJ-EF1α-DIO-ChR2-eYFP, AAVDJ-EF1α-DIO-ChR2-mCh, AAVDJ-CMV-DIO-EGFP, AAVDJ-EF1α-DIO-mCh, AAVDJ-EF1α mCh-IRES-Cre-WPRE, AAV9-hSyn-EGFP-CAAX, AAV9-hSyn-GRAB5-HT3.5 and AAV9-hSyn-GRABNT1.0. AAV titres ranged from 1 × 1012 to 2 × 1013 gc ml−1. Synapsin-driven lentiviral NLS-GFP-Cre, AAV-CAG-FLEx-TC, AAV-CAG-FLEx-G and rabies virus (Janelia Research Campus) were used for monosynaptic tracing. Behavioural experiments with cell body manipulations were performed 3–4 weeks after virus injections, whereas the viral particles were allowed to incubate 4–8 weeks for manipulation of axon terminals. Optic fibres (400-µm core, numerical aperture (NA) 0.5; RWD Life Science) for optogenetic light delivery were implanted in the vCA1 bilaterally at anteroposterior −3.16; mediolateral ±3.1; dorsoventral −4.4. A 26-gauge guide cannula (Plastics One) for drug micro-infusion was implanted at anteroposterior −3.16; mediolateral ±3.1, so that the 33-gauge infuser insert reached a depth of dorsoventral −4.5. Miniature screws (thread size 00–90 × 1/16; Antrin Miniature Specialties and McMaster-Carr), light-cured dental adhesive cement (C&B Metabond, Parkell) and resin (Geristore A&B paste, DenMat or Ortho-Jet powder&liquid, Lang Dental) were applied to secure implants to the skull. A small percentage (roughly 5%) of mice were excluded from behavioural analysis based on either off-target transgene expression or inaccurate implant placement. For this, 4-hydroxytamoxifen (4-OHT, Sigma H6278-50MG) was administered by means of intraperitoneal (i.p.) injection at 50 mg kg−1 for 15 min after pairing with partner mice. CNO (Tocris Biosciences 4936) was also delivered through i.p. injections at 10 mg kg−1 for 30 min before behavioural experiments. Drugs used for cannula microinjection experiments were infused in a total volume of 200–400 nl at a speed of 200 nl min−1 through an injector cannula connected to a micro-infusion pump (World Precision Instruments). The following drugs and concentrations were used: methiothepin mesylate salt (Millipore Sigma M149, 0.2 μg), NAS-181 (Tocris Biosciences 1413, 2 μg), NAD-229 (Tocris Biosciences 3282, 3.5 μg), SR48692 (Tocris Biosciences 3721, 5 pg), NTRC 824 (Tocris Biosciences 5438, 22 pg) and CP93129 dihydrochloride (Tocris Biosciences 1032, 0.5 μg). After completion of the infusion, injection infusers stayed for 2 min until removal and sociability and social memory assays were carried out 20 min later. injections 30 min before fibre photometry experiments. Three groups of TRAP2;Ai14 mice (6–8 weeks old) were habituated to i.p. saline injections and empty cages for two consecutive days. On test day, group 1 subjects were introduced to an age-matched same-sex conspecific in a new cage, group 2 to a sex-matched aggressive male CD-1 or female CFW mouse in the aggressor's home cage and group 3 to a potential mate (opposite sex) in either the female partner's home cage or the female subject's home cage. After 15 min of interaction, subjects were administered 4-OHT (50 mg kg−1) and interaction was continued for 1 h before subjects returned to their home cages. Ten days later, mice were either perfused with 10% neutral buffered formalin (NBF) for histological analysis or brains were removed to be fresh frozen for in situ hybridization experiments. Depending on the experiment, test mice were habituated for 2 days to i.p. injections, infuser inserts or the fibre optic patch cord as well as the three-chamber apparatus containing two empty inverted metal grid pencil cups (10 cm diameter) in the outer two chamber for 5 min. White opaque acrylic dividers were used to block entrance to the two outer chambers from the centre chamber. Partner mice were also habituated under the cup for 5 min for 2 days. On test day, subjects were placed into the centre chamber for 2 min before the barriers were lifted, whereafter the subjects were able to freely explore all three chambers. Four variations of this assay were performed to test social memory. The location of partner mice and objects in either chamber was counterbalanced between trials in all tests. In the neutral social memory test, the first 10-min session is also a sociability test, in which the time spent in the chamber with a same-sex age-matched novel mouse or an object is measured similarly to previously described methods3,5,7. After a 10-min interval, in which subjects were separated from the contents of the two outer chambers, the barriers were lifted and the subjects were allowed to freely explore the three chambers for 5 min in a second session. This time, the one of the outer two chambers contained a novel mouse and the other one a familiar mouse (same mouse from previous session) under a cup. In the negative social memory test, subjects interacted directly or 5 min with either a same-sex aggressor in the aggressor's home cage (similar to a resident–intruder test17,18) or a same-sex non-aggressor in a new cage. After a 5-min interval, those subjects that interacted with an aggressor were exposed to a non-aggressor for 5 min and vice versa. The sequence was counterbalanced between trials. Owing to the sharp drop in the discrimination scores after a short separation time3,7, a 5-min interval was chosen to shorten the separation time after the first interaction. Singly housed (at least 10 days) male retired breeder CD-1s were used as male aggressors and pair-housed female CFWs were used as female aggressors18. CD-1s and CFWs were screened for their level of aggression as previously described17 and only mice that attacked within 10 s and more than three times within 5 min qualified as aggressors. Retired breeder CD-1s and female CFW group-housed with same-sex conspecifics were used as non-aggressors. Only mice that did not attack in a 5-min interaction qualified as non-aggressors. In the positive social memory test, subjects interacted directly with either an opposite-sex conspecific potential mate in the potential mate's home cage or an opposite-sex conspecific neutral partner in a new cage for 5 min. After a 5-min interval, the subject that interacted with one type of partner was exposed to other for 5 min, whereby the sequence was counterbalanced between trials. After another 5-min interval, social memory was tested in a three-chamber apparatus for 5 min with one outer chamber containing the potential mate and the other containing the neutral partner under the cup. The interactions with the neutral partner were without lordosis posture and barrier in a new cage. It is important to note that the term neutral is used to describe a more neutral or less positive stimulus compared to the potential mate but not an absolute neutral stimulus. We used two variations of the RTPP test. The basic RTPP was performed as previously described31, in which light stimulation was paired with one of the two outer chambers while the subject was allowed to freely explore the three chambers for 15 min. Next, light stimulation was paired with the opposite outer chamber while the subject freely explored all three chambers again for 15 min. In variation 1, the two outer chambers each contained a new mouse and light stimulation was paired with one of the two chambers, while the subjects freely explored all three chambers for 10 min. After a 10-min interval, the barriers were lifted and the subjects were allowed to move freely between the three chambers, where the outer chambers contained the two previously acquainted mice (now familiar mice) under a cup. During this 5-min period, no light stimulation was applied. In variation 2 of this assay, two new objects instead of mice were used. Recorded videos in the three-chamber apparatus was analysed using a video tracking system (BIOBSERVE, v.3.01), which automatically tracked the location of the subject mouse. For all experiments, male and female subject mice were used in roughly equal numbers. Test mice were excluded from the analysis (less than 2% of the total), if they were not attacked by the aggressor, if the non-aggressor attacked more than three times in 5 min, or if they spent the entirety of the assay in one chamber only. Mice were habituated for 2 days to behavioural chambers with optic fibres connected to light-emitting diodes (LEDs) in off states. A 545-nm Dual-LED (Prizmatix) was connected to the optical implants via a rotary joint and a fibre optic patch cord (Prizmatix) to photostimulate NpHR3.0. A 450-nm Dual-LED (Prizmatix) was used to photostimulate ChR2. The LEDs were adjusted to around 15 mW for axon terminal stimulation using a digital power meter console (ThorLabs) and the frequency was controlled by Pulser Plus (Prizmatix). A cycle of 8 s on and 2 s off was used to stimulate NpHR3.0 to avoid tissue overheating. ChR2 was stimulated at 20 Hz with a 5-ms pulse duration. The ‘grid' floor was constructed of seven evenly spaced apart rails, which were 2.5 cm in width. The hole floor contained 4 × 9 round holes 8 mm in diameter, evenly spaced apart. On day 1, mice were either administered saline or CNO (10 mg kg−1) through i.p. On the following day, mice that received saline on the previous day, received CNO and vice versa. On day 3, test mice were placed into the chamber containing both types of flooring and time spent on either type was automatically scored with video tracking software (BIOBSERVE). Mice were placed into the centre of the elevated-plus maze (74 cm long by 74 cm wide by 94 cm high; Med Associates Inc.) with white floors (each arm 35 cm long by 6 cm wide with 19 cm high walls on closed arms) for 10 min and location of the mice were scored with video tracking software (BIOBSERVE). Male subjects were placed for 5 min into a female conspecific's home cage with one female present. Male mounting attempts during the 5 min of free interaction were manually quantified. As females do not show mounting behaviour, a different test was used to assess female receptiveness to males. Male partners were placed into the female subject's home cage, which was divided into two compartments by a barrier, small enough for the female to easily escape. Escape times and duration of the female in each compartment were manually scored. 9) was injected into the vCA1 (unilateral, anteroposterior −3.16; mediolateral ±3.1; dorsoventral −4.55), and fibre optic implants were secured above the injection site (unilateral, anteroposterior −3.16; mediolateral ±3.1; dorsoventral −4.4) in stereotaxic surgeries. Two to three weeks later, mice were habituated to the behavioural set-up and tested on the following day with simultaneous video and fibre photometry acquisition. Test mice interacted with a new object, a novel same-sex age-matched conspecific, a sex-matched aggressor or a potential mate for 5 min with an interval of 5 min, whereby the sequence of the interaction was counterbalanced between subjects. Female subjects interacted with the potential mate in their own home cage. Synapse software, which controlled an RZ10X LUX-I/O processor (Tucker-Davis Technologies), was used for data acquisition. To stimulate Ca2+-dependent and isosbestic emission, GCaMP6f was excited by frequency-modulated built-in 465- and 405-nm LEDs (RZ10X LUX-I/O processor), respectively. All optical signals were band-pass-filtered with a fluorescence mini cube (Doric), emission was measured with built-in photosensors in the RZ10X LUX-I/O processor and the signal was digitized at 6 kHz. A previously described custom MATLAB_R2022b (MathWorks) code7 was used for signal processing. To debleach, fibre photometry raw signals were fitted with a mono- or bi-exponential decay function, and the resulting fluorescence trace was z scored. Corresponding videos were manually analysed by frame in MATLAB to identify the time of physical contact between test mouse and partner mouse and object. Peristimulus time histograms were constructed by averaging 7 s of non-overlapping epochs from the z scored trace, where a time of 0 represents the time of contact. The maximal z score between 0 and 4 s was used as peak z scored fluorescence. We generated a multi-guide CRISPRi AAV vector starting with pX552 (Addgene, 60958) as the backbone, which we then digested with NotI-HF (NEB). Between the NotI sites, we inserted two gene fragments to generate the AAV backbone pAAV mU6-sgRNA-CR1 EF1α-EGFP-W3-SV40. The first fragment comprised a mouse U6 (mU6) promoter, multiple cloning site for protospacers, constant region (CR1) and an EF1α promoter derived from pU6-sgRNA EF1α-puro-T2A-BFP (Addgene, 60955) but engineered to include BbsI sites flanking the mU6-sgRNA-CR1 cassette. The second fragment contained EGFP, W3 terminator and SV40 poly(A) signal. Both were assembled into the NotI-digested pX552 backbone using a three-part Gibson assembly (HiFi NEB). Subsequently, sgRNA arrays comprising a mU6 promoter, sgRNA protospacer sequence of interest and CR1 cassette, synthesized as gene fragments, were inserted into the backbone with a Golden gate approach using the engineered BbsI (NEB) sites in accordance with the manufacturer's instructions. To ensure maximum knockdown efficiency, the vectors consist of three tandem sgRNA cassettes containing the protospacers predicted to have highest activity according to the Weissman laboratory's V2 CRISPR sgRNA algorithm54. AAV particles were produced using a previously described protocol with minor modifications55. One T175 flask of HEK293T cells (American Type Culture Collection) per construct was transfected with 87.6 µg PEI (1 µg ml−1; 3:1 PEI: DNA; PolySciences 24765) along with 15.4 µg of pAdΔF6, 7.3 µg of pAAV2/9n (pAdΔF6 and pAAV2/9n were gifts from J. M. Wilson; Addgene, catalogue nos. Viral supernatant was harvested 72 h posttransfection, along with cells and mixed thoroughly with 0.1 volumes of chloroform. NaCl was added to a final concentration of 1 M, mixed thoroughly, and the sample centrifuged for 5 mins at 3,000g, 4 °C. The aqueous phase was retained, mixed with 9.4 ml of 50% PEG-8000, incubated on ice for 1 h and centrifuged for 30 min at 3,000g, 4 °C. Pellets we resuspended in dPBS, mixed with 5 µl benzonase (EMD Millipore, 71205) and 1 mM MgCl2, and incubated at 37 °C for 30 min. Postbenzonase treatment, samples were mixed thoroughly with an equal volume of chloroform and centrifuged for 5 min at 3,000g, 4 °C. Samples were buffer exchanged into dPBS and concentrated to roughly 50 µl using 100-kDa cut-off cellulose centrifugal filters (Millipore Sigma, UFC8100). For the cFos immunohistochemistry experiment, mice were subjected to an interaction with an aggressor (CD-1/CFW) or a potential mate for 1.5 h and immediately perfused with chilled 10% NBF. After overnight incubation in 10% NBF, 40-μm slices were prepared on a Leica VT 1000S vibratome and collected in 24-well plates filled with phosphate buffered saline (PBS). After a 1.5-h blocking in 5% normal goat serum, slices were incubated with 1:1,000 diluted primary antibody and 5% normal goat serum at 4 °C for 12 h. After three 5-min washes with PBS, slices were incubated for 1.5 h in secondary antibody, subsequently washed three times in PBS for 15 min and Fluoromount-G mounting medium was used for 4,6-diamidino-2-phenylindole (DAPI) staining. The following primary antibodies were used: 1:1,000 diluted cFos antibody (Synaptic Systems, 226 008) and 1:500 diluted TPH2 antibody (Abcam, ab184505). AAVDJ-EF1α mCh-IRES-Cre-WPRE and AAV9-EF1α-sgTph2-EGFP or AAV9-EF1α-sgNTC-EGFP were injected into the DR (unilateral, anteroposterior −4.36; mediolateral 0; dorsoventral −3.1) of dCas9-KRAB mice. Three weeks later, immunohistochemistry was performed as described above. Imaging was conducted on a Zeiss LSM 780 confocal (×20 objective; NA 0.8; diode laser 405 nm, argon laser 458/488/514 nm, diode-pumped solid-state laser 561 nm and HeNe laser 594/633 nm; detectors 32 channel GaAsP, T-PMT (transmitted light), 2-PMT and Airyscan detector; bit depth of images was 8 bit). TPH2 channels were background subtracted, and the same cut-off threshold was applied for all replicates. Polygons were manually drawn around the DR area and particles within the area of interest were analysed. All infected DR cells received sgTph2 or non-targeting guides. To evaluate cell body TPH2 expression in the DR, we analysed fluorescence intensity. An average percentage fluorescence intensity was calculated from the sgNTC replicates and the remaining ratio was used for calculation of the percentage TPH2 fluorescence intensity and knockdown efficiency. Behavioural cohorts were injected with AAV9-EF1α-sgTph2-EGFP or AAV9-EF1α-sgNTC-EGFP into the DR and CAV2-Cre into the vCA1. Six weeks later, behavioural experiments were performed followed by perfusion and immunohistochemistry. Imaging and background subtraction was performed as described above. Polygons were manually drawn around the vCA1 and particles in within the area were analysed. An average percentage of TPH2+ area within the vCA1 area was calculated from the sgNTC replicates and the remaining ratio was used for calculation of percentage TPH2+ area. Primary neuronal culture was performed as previously described56. In brief, DR nuclei were dissected from P0 pups in ice-cold Hank's buffered saline solution (HBSS) buffer (Thermo Fisher Scientific, 88284) supplemented with 2 mM Ca2+ and 0.5 mM EGTA under a dissection hood and incubated in Neuronal Isolation Enzyme (Thermo Fisher Scientific, 88285) and HBSS for 20 min, washed twice with HBSS and dissociated in prewarmed Neurobasal-A Medium (Thermo Fisher Scientific, 10888022) with 2% FBS (Avantor, 97068-085) by gentle trituration. Cell yield and viability were determined by a hemocytometer and trypan blue (Thermo Fisher Scientific, 15250061) staining, respectively. Neurons were plated on a poly-d-lysine (Thermo Fisher Scientific, A3890401) precoated 24-well plate. After 24 h, half of the serum medium was replaced with an equivalent volume of Serum-free Neurobasal-A Medium supplemented with 0.5 mM l-glutamine (Thermo Fisher Scientific, A2916801) and 0.5% B-27 (Thermo Fisher Scientific, 17504044). On day 3, half of the old medium was replaced with fresh medium supplemented with 4 mM Ara-C (Sigma, C6645), and neurons were infected with AAV constructs expressing control sgRNAs or Tph2 sgRNAs. After infections, the primary neuronal cultures were maintained for 1 week with medium changes every 3 days. RNA extraction was performed using a Quick-RNA MicroPrep Kit (Zymo Research, R1050) according to the manufacturer's protocol. Total RNA was reverse transcribed using the High-Capacity RNA-to-cDNA Kit (Thermo Fisher Scientific, 4387406). The resulting complementary DNA (cDNA) was then used for qPCR using PowerUp SYBR Green Master Mix (Thermo Fisher Scientific, A25742) with gene-specific primers. Data were recorded using a QuantStudio 7 Flex Real-Time PCR system (Thermo Fisher Scientific). Tph2 expression was normalized to Actb. Primers used in the studies include: Tph2 (mouse): forward, 5′-GCAAGACAGCGGTAGTGTTCT-3′; reverse, 5′-CAGTCCACGAAGATTTCGACTT-3′; Actb (mouse): forward, 5′-GGCTGTATTCCCCTCCATCG-3′; reverse, 5′-CCAGTTGGTAACAATGCCATGT-3′. Here, 4–8-week-old C57BL/6J, Shank3Δ4-22+/− and wild-type littermates were used for whole-cell recordings. Mice were euthanized and brains were sliced in a sucrose cutting solution containing (in mM): 228 sucrose, 26 NaHCO3, 11 glucose, 2.5 KCl, 1.2 NaH2PO4, 7 MgCl2 and 0.5 CaCl2 on a vibratome (Leica VT1200 S). Coronal vCA1 slices (300 μm) were transferred to a slice holding chamber with artificial cerebrospinal fluid containing (in mM): 119 NaCl, 26 NaHCO3, 11 glucose, 2.5 KCl, 1.2 NaH2PO4, 1.3 MgCl2 and 2.5 CaCl2 (osmolarity 289–295) for 30 min at 32 °C and then further equilibrated for 30 min at room temperature. Next, brain slices were placed in a recording chamber perfused with 28–30 °C artificial cerebrospinal fluid and visualized with a ×40 water-immersion objective on an upright fluorescent microscope (BX51WI; Olympus) equipped with infrared-differential interference contrast video microscopy and epifluorescence (CoolLED). Whole-cell current-clamp recordings (pipette opening 4–6 MΩ) were performed with pipettes filled with (in mM): 130 C6H11KO7, 5 KCl, 10 HEPES, 0.6 EGTA, 2.5 MgCl2, 4 Mg2ATP, 0.4 Na3GTP, 10 phosphocreatine (pH 7.25; osmolarity 290). Whole-cell voltage-clamp recordings (pipette opening 3–4 MΩ) of IPSCs were performed with pipettes filled with (in mM): 80 CsCl, 65 CsMeSO4, 8 NaCl, 10 HEPES, 0.25 EGTA, 2 Mg2ATP, 0.3 Na3GTP, 0.1 spermine, 7 phosphocreatine (pH 7.34; osmolarity 300). No CsCl but 140 CsMeSO4 was used for whole-cell voltage-clamp recordings of EPSCs. Series and input resistance were monitored with −4 mV, 70-ms pulse delivered through the recording pipette and experiments were excluded from the analysis if series resistance varied by more than 15%. Current-clamp recordings were carried out to assess intrinsic cell excitability with a series of incremental rectangular depolarizing current pulses (20 pA, 500 ms) injected into vCA1 pyramidal cells. Next, the same procedure was performed in the same cell after either CP93129 dihydrochloride (Tocris Biosciences 1032, 5 µM) or PD149163 (Sigma, 0.5 µM) was bath applied for 5 min. Following 10 min of wash-out, recordings were carried out in the same cell in presence of the second drug, whereby the sequence of drug application was counterbalanced between cells. Only one cell was recorded per slice. Current-clamp recordings in the PVT was carried out with 20-pA depolarizing current pulses and in the DR with 25-pA depolarizing current pulses. Pipettes were filled with in (mM): 130 C6H11KO7, 10 KCl, 10 HEPES, 0.2 EGTA, 4 Mg2ATP, 0.5 Na3GTP, 10 phosphocreatine. CNO (Tocris Biosciences 4936, 5 µM) was bath applied for 5 min. Evoked EPSCs and IPSCs were recorded from vCA1 cells with a bipolar stimulating electrode (fabricated from platinum and iridium wire) placed near the recording pipette. Baseline evoked and spontaneous EPSCs were recorded in voltage-clamp settings at −70 mV the presence of picrotoxin (50 μM). Baseline evoked and spontaneous IPSCs were recorded at −70 mV in the presence of NBQX (10 μM) and D-AP5 (50 μM). CP93129 dihydrochloride (5 µM) or PD149163 (0.5 µM) were bath applied after an initial baseline recording. Summary graphs of the effects of CP93129 and PD149163 over time in evoked recordings were generated by averaging 1-min bins as a percentage of the averaged 4-min baseline. Cumulative probability graphs were generated from all spontaneous events before and after drug application. Synaptic responses were recorded in the vCA1 8 weeks after stereotaxic injections of AAVDJ-hSyn-ChR2-eYFP into the DR or PVT. To photostimulate ChR2 in vCA1 axons and/or terminals, 470-nm 0.1–5.0-ms light pulses from the pE-300ultra (CoolLED) were delivered by means of a ×40 water-immersion objective to the whole slice. Recordings were made using a MultiClamp 700B amplifier (Molecular Devices), digitized at 10 kHz with the Digidata 1320A or 1440A data acquisition system (Molecular Devices) and analysed with Clampfit v.10.7 software (Molecular Devices). Briefly, brains were fresh frozen on dry ice and stored at −80 °C until 15-μm slices were prepared and collected directly on Superfrost plus microscopy slides using a cryostat. Following fixation in prechilled 4% paraformaldehyde, slices were washed in PBS and dehydrated gradually in 50%, 70% and two times 100% ethanol (5 min each). After a barrier was created with a hydrophobic pen, slides were incubated with roughly five drops of RNAscope hydrogen peroxide for 10 min, subsequently washed with distilled water and incubated with Protease IV for 30 min, then washed twice with PBS. The probes mCh-O3 (513201), Mm-Slc17a6-C2 (319171-C2), Mm-Gad2-C2 (439371-C2) and Mm-Nts-C3 (420441-C3) were warmed up to 40 °C in a water bath, and 1 volume of C2 and C3 were diluted in 50 volumes of C1 to create the probe mixture. In the hybridization step, slices were incubated with the probe mixture for 2 h at 40 °C in a HyEZ Oven and rinsed twice at room temperature in Wash buffer (50× RNAscope Wash buffer diluted in distilled water). Next, slides were incubated with roughly six drops of RNAscope Multiplex FL v2 Amp1 for 30 min at 40 °C in the HyEZ Oven and washed twice with wash buffer at room temperature. This procedure was repeated with RNAscope Multiplex FL v2 Amp2 (30 min) and RNAscope Multiplex FL v2 Amp3 (15 min). Slides were then incubated with RNAscope Multiplex FL v2 HRP-C1 in the HyEZ Oven at 40 °C for 15 min, rinsed twice in wash buffer at room temperature and incubated in a 1:1,000 diluted Opal 520 dye for 30 min at 40 °C and subsequently rinsed twice in wash buffer. Finally, Fluoromount-G mounting medium was used for DAPI staining. Imaging was performed on a Zeiss LSM 780 confocal (×20 objective; NA 0.8; diode laser 405 nm, argon laser 458/488/514 nm, diode-pumped solid-state laser 561 nm and HeNe laser 594/633 nm; detectors 32 channel GaAsP, T-PMT (transmitted light), 2-PMT and Airyscan detector; bit depth of images, 8 bits). A Fiji (ImageJ v.1.52p) macro was used for automated image analysis, which was performed blinded. Based on pixel-intensity threshold, DAPI channels were background subtracted, binarized and converted to masks. The cut-off threshold was visually determined by comparing ascending intensity values with marked location in the images to eliminate background particles. Prism10 (GraphPad) was used for statistical analysis. Parametric statistical tests were only chosen for normally distributed samples. Otherwise, non-parametric statistical tests were performed. One-way analysis of variance (ANOVA) with Tukey's or Dunnett's multiple comparison post hoc test was used to determine significance for several treatment comparisons and two-way ANOVA with Tukey's or Sidak's multiple comparison post hoc test for several group comparisons and across many time points. P values found in figure legends refer to ANOVA results of the whole group, whereas the asterisks between individual samples represent results of the multiple comparison post hoc tests. A two-tailed paired Student's t-test was applied for within-group comparison of two treatments and unpaired test for comparison between two groups. When normality was not assumed, the Wilcoxon signed rank test was used for within-group comparison of two treatments, Mann–Whitney test for between group comparison and Kruskal–Wallis with post hoc Dunn's test for multiple comparisons. In all figures, data are shown as mean ± s.e.m. Representative data are one of: Fig. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequences of the custom vector and sgRNAs used for CRISPRi knockdown experiments, detailed statistical analysis between sexes and raw histology images are available at Zenodo (https://doi.org/10.5281/zenodo.14814653)58. All other raw and/or processed data generated during this study are available from the corresponding author upon reasonable request. Source data are provided with this paper. MATLAB codes used in this study are available at Zenodo (https://doi.org/10.5281/zenodo.14814674)59. Padilla-Coreano, N., Tye, K. M. & Zelikowsky, M. Dynamic influences on the neural encoding of social valence. The hippocampal CA2 region is essential for social memory. Okuyama, T., Kitamura, T., Roy, D. S., Itohara, S. & Tonegawa, S. Ventral CA1 neurons store social memory. Meira, T. et al. A hippocampal circuit linking dorsal CA2 to ventral CA1 critical for social memory dynamics. & Pozzo-Miller, L. Ventral hippocampal projections to the medial prefrontal cortex regulate social memory. Chen, S. et al. A hypothalamic novelty signal modulates hippocampal memory. Wu, X., Morishita, W., Beier, K. T., Heifets, B. D. & Malenka, R. C. 5-HT modulation of a medial septal circuit tunes social memory stability. Namburi, P. et al. A circuit mechanism for differentiating positive and negative associations. Li, H. et al. Neurotensin orchestrates valence assignment in the amygdala. Jhang, J. et al. Anterior cingulate cortex and its input to the basolateral amygdala control innate fear response. Tye, K. M. Neural circuit motifs in valence processing. Redondo, R. L. et al. Bidirectional switch of the valence associated with a hippocampal contextual memory engram. Jimenez, J. C. et al. Contextual fear memory retrieval by correlated ensembles of ventral CA1 neurons. Creating a false memory in the hippocampus. Social odor discrimination and its enhancement by associative learning in the hippocampal CA2 region. A standardized protocol for repeated social defeat stress in mice. Sex differences in appetitive and reactive aggression. Sexual behavior regulated (paced) by the female induces conditioned place preference. Characterization of copulatory behavior in female mice: evidence for paced mating. Lopez-Rojas, J., de Solis, C. A., Leroy, F., Kandel, E. R. & Siegelbaum, S. A. A direct lateral entorhinal cortex to hippocampal CA2 circuit conveys social information required for social memory. Ren, J. et al. Single-cell transcriptomes and whole-brain projections of serotonin neurons in the mouse dorsal and median raphe nuclei. Shpokayte, M. et al. Hippocampal cells segregate positive and negative engrams. Jimenez, J. C. et al. Anxiety cells in a hippocampal-hypothalamic circuit. Reduced fear expression after lesions of the ventral hippocampus. Direct ventral hippocampal-prefrontal input is required for anxiety-related neural activity and behavior. Sanchez-Bellot, C., AlSubaie, R., Mishchanchuk, K., Wee, R. W. S. & MacAskill, A. F. Two opposing hippocampus to prefrontal cortex pathways for the control of approach and avoidance behaviour. Li, L. et al. Dorsal raphe nucleus to anterior cingulate cortex 5-HTergic neural circuit modulates consolation and sociability. Keyes, P. C. et al. Orchestrating opiate-associated memories in thalamic circuits. Walsh, J. J. et al. 5-HT release in nucleus accumbens rescues social deficits in mouse autism model. Curtis, G. R., Oakes, K. & Barson, J. R. Expression and distribution of neuropeptide-expressing cells throughout the rodent paraventricular nucleus of the thalamus. Circuit and molecular architecture of a ventral hippocampal network. Rao, R. P., von Heimendahl, M., Bahr, V. & Brecht, M. Neuronal responses to conspecifics in the ventral CA1. Sari, Y. Serotonin1B receptors: from protein to physiological function and behavior. Chen, L., Yung, K. K. & Yung, W. H. Neurotensin depolarizes globus pallidus neurons in rats via neurotensin type-1 receptor. Xiao, Z. et al. Activation of neurotensin receptor 1 facilitates neuronal excitability and spatial learning and memory in the entorhinal cortex: beneficial actions in an Alzheimer's disease model. Banker, S. M., Gu, X., Schiller, D. & Foss-Feig, J. H. Hippocampal contributions to social and cognitive deficits in autism spectrum disorder. Peca, J. et al. Shank3 mutant mice display autistic-like behaviours and striatal dysfunction. Disrupted social memory ensembles in the ventral hippocampus underlie social amnesia in autism-associated Shank3 mutant mice. Boyle, L. M., Posani, L., Irfan, S., Siegelbaum, S. A. & Fusi, S. Tuned geometries of hippocampal representations meet the computational demands of social memory. Okuyama, T. Social memory engram in the hippocampus. Li, L. et al. Social trauma engages lateral septum circuitry to occlude social reward. Hull, E. M., Muschamp, J. W. & Sato, S. Dopamine and serotonin: influences on male sexual behavior. Cembrowski, M. S. et al. Spatial gene-expression gradients underlie prominent heterogeneity of CA1 pyramidal neurons. Shah, S., Lubeck, E., Zhou, W. & Cai, L. seqFISH accurately detects transcripts in single cells and reveals robust spatial organization in the hippocampus. Ferguson, J. N. et al. Social amnesia in mice lacking the oxytocin gene. Cheng, D. et al. Noradrenergic consolidation of social recognition memory is mediated by beta-arrestin-biased signaling in the mouse prefrontal cortex. Allen, W. E. et al. Thirst-associated preoptic neurons encode an aversive motivational drive. Kremer, E. J., Boutin, S., Chillon, M. & Danos, O. Canine adenovirus vectors: an alternative for adenovirus-mediated gene transfer. Cunningham, C. L., Gremel, C. M. & Groblewski, P. A. Drug-induced conditioned place preference and aversion in mice. Wan, J. et al. A genetically encoded sensor for measuring serotonin dynamics. Negrini, M., Wang, G., Heuer, A., Bjorklund, T. & Davidsson, M. AAV production everywhere: a simple, fast, and reliable protocol for in-house AAV vector production based on chloroform extraction. Wu, X. et al. Neuroligin-1 signaling controls LTP and NMDA receptors by distinct molecular pathways. Wang, F. et al. RNAscope: a novel in situ RNA analysis platform for formalin-fixed, paraffin-embedded tissues. Wu, X. MATLAB code [Data set]. This work was supported by funding from NIH K99/R00 Career Development Award (grant no. MH122697), NIMH BRAINS R01 Award (grant no. We thank S. J. Russo for providing mouse breeding pairs; K. Kelley for technical support and P. J. Kenny, S. J. Russo, J. J. Walsh and D. J. Christoffel for discussions. Department of Neuroscience, Icahn School of Medicine at Mount Sinai, New York, NY, USA Julia M. Zorab, Huanhuan Li, Richa Awasthi, Anna Schinasi, Yoonjeong Cho, Thomas O'Loughlin & Xiaoting Wu You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed the CRISPRi knockdown strategy. cloned the sgRNA vectors and made AAVs. performed CRISPRi validation and associated histology. performed ex vivo electrophysiology recordings. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Left, schematic of social memory test. Right, duration C57BL/6J test mice spent in chamber with familiar mouse (fm) or novel mouse (nm), which are both aggressive CD-1s under a cup with which the subjects had no prior direct interaction (t9 = 2.343, P = 0.0438, n = 10). b, Schematic of experiment and duration C57BL/6J test mice spent in chamber containing an aggressive and non-aggressive CD-1 under a cup without prior direct interaction (t7 = 0.6644, P = 0.5277, n = 8). c, Left, same data as Fig. 1a, but divided into males and females. d, Schematic of experiment and duration C57BL/6J male mice spent in chamber containing C57BL/6J female in pro/estrus cycles and not in those cycles without prior direct interaction (t9 = 0.1644, P = 0.8731, n = 10). e, Left, same data as Fig. 1b, but divided into males and females. Right, d scores (t24 = 0.5349, P = 0.5976, n = 13). Statistical tests: duration (a – e): two-tailed paired Student's t-test. d scores (c, e): two-tailed unpaired Student's t-test. a, b, social memory: duration subjects spent in chamber with familiar mouse (fm) or novel mouse (nm) and discrimination (d) scores (a: F3,34 = 6.693, P = 0.0011, hM4Di n = 17, mCh n = 10; b: F3,35 = 6.257, P = 0.0016, hM4Di n = 12, mCh n = 14). Negative social valence (sv): duration in chamber containing the non-aggressor with a previous ntrl encounter or in chamber containing the aggressor with a previous neg encounter and d scores (a: F3,34 = 8.018, P = 0.0004, hM4Di n = 17, mCh n = 10; b: F3,34 = 5.437, P = 0.0037, hM4Di n = 12, mCh n = 13). Positive sv: duration in chamber containing the partner with a previous ntrl encounter or in chamber containing the potential mate with a previous pos encounter and d scores (a: F3,35 = 4.622, P = 0.0079, hM4Di n = 17, mCh n = 10; b: F3,34 = 6.909, P = 0.0009, hM4Di n = 12, mCh n = 13). e, Left, Schematic of experiment and duration subjects spent in chamber containing a conspecific male or female. Right, d scores (t9 = 0.3229, P = 0.7541, n = 10). f, Schematic of experiment and the number of times female test mice escaped from males into the separate compartment of the cage within 5 min (t4 = 0.3203, P = 0.7648, n = 5). Statistical tests, duration (a – f) and d scores (e, f): two-tailed paired Student's t-test. a, Schematic of experimental set-up. b, c, Representative images (1 of 3 mice) of injection sites in the (b) vCA1 and (c) dCA2 and respective presynaptic labelling with GFP in the Medial septum (MS), nucleus of the diagonal band (NDB), the anterodorsal (AD), anteroventral (AV) and paraventricular (PVT) thamalamic nuclei, supramammillary nucleus (SUM), nucleus reuniens (RE), dorsal (DR) and median raphe (MR). a, Duration that subjects spent in chamber with a novel object (no) or nm and discrimination (d) scores (F3,36 = 0.617, P = 0.6083, n = 13). b, Duration spent on previously CNO or saline paired surfaces (F3,34 = 1.948, P = 0.1404, hM4Di: n = 13; mCh: n = 12). c, Duration that subjects spent in chamber with a nm or familiar mouse (fm) and d-scores (t14 = 4.034, P = 0.0012, n = 15). d, Duration that subjects spent in chamber with a novel object (no) or nm and d scores (t14 = 1.346, P = 0.1996, n = 15). e – g, Left, Duration that subjects spent in chamber with (e) fm or nm, (f) non-aggressor with previous ntrl encounter or aggressor with previous neg encounter and d scores, (g) partner with previous ntrl encounter or potential mate with previous pos encounter and d scores (e: t12 = 0.2963, P = 0.772, n = 13; f: t12 = 0.2874, P = 0.7787, saline n = 14, CNO n = 13; g: t13 = 0.9073, P = 0.3807, n = 14). h, Duration that subjects spent in chamber with a no or nm and d scores (t12 = 0.4988, P = 0.627, saline n = 14, CNO n = 13). i, Duration that subjects spent in chamber with a no or nm and d scores (F3,26 = 2.552, P = 0.0774, hM4Di: n = 11; mCh: n = 9). j, Duration spent on previously CNO or saline paired surfaces (F3,29 = 0.8432, P = 0.4814, hM4Di: n = 14; mCh: n = 9). k, l, Left, Sample traces of DR (k) and PVT (l) neuron spiking in response to CNO. Scale bars, x-axis = 100 ms, y-axis = 10 mV. Statistical tests, duration (a, c – i) and d scores (d – h, k): two-tailed paired Student's t-test. d scores (a, b, i, j): two-way ANOVA with Šidák's post-hoc test. a, Schematic of experiment and representative images of injection site (1 of 3 mice) in the DR of Sert-Cre animals and axon innervation in the vCA1, n = 3. b, Schematic of experiment and representative images of injection site (1 of 3 mice) in the aPVT of NT-Cre animals and axon innervation in the vCA1, n = 3. c, Left, duration mice spent in chamber with familiar mouse (fm) or novel mouse (nm). d, Duration that subjects spent in chamber with a novel object (no) or nm and d) scores (P = 0.4517, NpHR3.0: n = 13; eYFP: on n = 14). e, Average time spent in light paired or unpaired chambers in initial and reversal trials (F3,22 = 1.602, P = 0.2174, NpHR3.0: n = 13; eYFP: n = 6). f, Left, Duration that subjects spent in chamber with fm or nm encounter and d scores (F3,34 = 0.8096, P = 0.4974, NpHR3.0: n = 14; eYFP: off n = 12, on n = 11). g, Duration that subjects spent in chamber with a novel object (no) or nm and d scores (F3,34 = 0.1939, P = 0.8998, NpHR3.0: n = 14; eYFP: off n = 12, on n = 11). h, Average time spent in light paired or unpaired chambers in initial and reversal trials (F3,28 = 1.649, P = 0.2005, NpHR3.0: n = 10; eYFP: n = 11). i, Time courses of average RCaMP2 transient z scores event-locked to direct interactions and quantification of peak z score during interaction (Top: t2 = 30.31, P = 0.021, n = 2; Bottom: t2 = 24.95, P = 0.0016, n = 3). j, Left, time courses of average EGFP-CAAX transient z scores event-locked to direct interactions. Right, quantification of peak z score during interaction (F2,9 = 0.2688, P = 0.7702, n = 4). k, Left, representative images of AAV-sgRNA-EGFP expression in the DR with Tph2 IHC. Top right, schematic of virus injection. Bottom right, quantification of Tph2 fluorescence intensity (P = 0.0286, n = 4). l, Schematic of qPCR experiment. Fold change of Tph2 mRNA level in DR primary cultured neurons infected with AAV-Cre and AAV-sgTph2 or AAV-sgNTC, t4 = 5.027, P = 0.0073, n = 3, biological replicates. m, Left, duration that subjects spent in chamber with a novel object (no) or nm and d scores (t16 = 0.6332, P = 0.5355, Tph2: n = 10; NTC: n = 8). Right, Duration that subjects spent in chamber with a nm or familiar mouse (fm) and d scores (t16 = 1.171, P = 0.2587, Tph2: n = 10; NTC: n = 8). Statistical tests, duration (c, d, f, g, i, m): two-tailed paired Student's t-test. (c, f, g) d scores and e, h: two-way ANOVA with Šidák's post-hoc test. j, one-way ANOVA with Tukey's post-hoc test. l, (m) d score: two-tailed unpaired Student's t-test. a, Discrimination scores between different drug infusions into the vCA1 from Fig. b, c, Percentage of cells from Fig. 4k,l which showed a change or no change in their evoked EPSC/IPSC amplitudes after bath-application of PD149163 (PD) or CP93129 (CP). d, Left, recordings of spontaneous EPSCs (sEPSCs) before and after bath-application of PD and cumulative probability plot of sEPSC amplitudes with representative traces (1 of 10 cells) above (t9 = 2.628, P = 0.0274, n = 10). Scale bars, x-axis = 50 ms, y-axis = 10 pA. Right, cumulative probability plot of sEPSC inter-event intervals with representative traces (1 of 10 cells) above (t9 = 1.333, P = 0.2154, n = 10). Scale bars, x-axis = 0.50 s, y-axis=10 pA. e, Left, recordings of sIPSCs before and after bath-application of PD and cumulative probability plot of sIPSC amplitudes with representative traces (1 of 8 cells) above (t7 = 0.6492, P = 0.5369, n = 8). Right, cumulative probability plot of sIPSC inter-event intervals with representative traces (1 of 8 cells) above (t7 = 1.823, P = 0.111, n = 8). f, Left, recordings of spontaneous IPSCs (sIPSCs) before and after bath-application of CP and cumulative probability plot of sIPSC amplitudes with representative traces (1 of 9 cells) above (t8 = 0.1498, P = 0.8846, n = 9). Scale bars, x-axis = 50 ms, y-axis = 10 pA. Right, Cumulative probability plot of sIPSC inter-event intervals with representative traces (1 of 9 cells) above (t8 = 2.94, P = 0.0187, n = 9). Scale bars, x-axis = 0.50 s, y-axis = 10 pA. Scale bars, x-axis = 50 ms, y-axis = 10 pA. g, Left, recordings of sEPSCs before and after bath-application of CP and cumulative probability plot of sEPSC amplitudes with representative traces (1 of 10 cells) above (t9 = 0.4582, P = 0.6577, n = 10). Right, Cumulative probability plot of sEPSC inter-event intervals with representative traces (1 of 10 cells) above (t9 = 0.2687, P = 0.7942, n = 10). Scale bars, x-axis = 50 ms, y-axis = 10 pA. h, Left, Schematic and representative images (1 of 4 mice) of in-situ hybridization in the vCA1. Right, pie chart of cells positive and negative for Gad2 and Htr1b along with a smaller pie chart and bar graph showing the proportion of Gad2+ cells positive and negative for Htr1b. Statistical tests: a, one-way ANOVA with Dunnett's post-hoc test, d–g, two-tailed paired Student's t-test. a, Schematic of one experimental condition and representative images (1 of 2 for neg-neg; 1 of 3 for neg, neg-pos, pos) displayed as merge with DAPI, tdTomato and cFos (Alexa 488). b, percentage of tdTomato (tdT) and cFos double positive cells. F3,7 = 26.29, P = 0.0003, neg-neg: n = 2, neg-pos, pos-pos, pos-neg: n = 3. c, Left, schematic of experiment and representative images (1 of 2 mice) of dual injection sites in the DR and aPVT. Right, representative images of axon innervation in the vCA1. e, Left, representative images of in situ hybridization in the vCA1 (1 of 3 mice). Right, quantification of triple (above) and double (below) positive cells (F3,8 = 34.723, P = 0.00006, n = 3). a, Duration that subjects spent in chamber with a novel object (no) or novel mouse (nm) and discrimination (d) scores (t11 = 0.1971, P = 0.8474, n = 12). b, Duration that subjects spent in chamber with a nm or familiar mouse (fm) and d scores (t11 = 0.7047, P = 0.4956, n = 12). c, Average time spent in light paired or unpaired chambers in initial and reversal trials (t11 = 0.7037, P = 0.4963, n = 12). d, Number of times male subjects attempted to mount females in 5 min (t5 = 0.3492, P = 0.7412, n = 6). e, Female subjects interacted with male partners in their home cage. d score of duration female subjects spent in the compartment with male partners or in the empty compartment (t5 = 0.2499, P = 0.8126, n = 6). f, g, Duration that subjects spent in chamber with (f) no or nm (t9 = 0.5448, P = 0.5991, n = 10) and (g) fm or nm (t9 = 0.1835, P = 0.8584, n = 10) and d scores. h, Average time spent in light paired or unpaired chambers in initial and reversal trials (t9 = 0.7234, P = 0.4878, n = 10). i, Time subjects spent in open and closed arms of the elevated plus maze (F1.3,12.1 = 159.5, P = 8 × 10−9, n = 10). k, l, Duration in chamber with (k) no or nm (F2,24 = 0.607, P = 0.5531, n = 9) and (l) with nm or fm (F2,24 = 2.207, P = 0.1319, n = 9) and d scores. Statistical tests: a – h, two-tailed paired Student's t-test. i, (k, l) d scores, one-way ANOVA with Tukey's post-hoc test. DR 5-HT and PVT NT neurons innervate overlapping but not identical areas within the vCA1. During an aversive social interaction NT is released from PVT neurons into the vCA1 and binds to NTR1 receptors on vCA1 pyramidal neurons, which increases excitability. During an appetitive social interaction 5-HT is released from DR neurons into the vCA1 and binds onto 5-HT1BRs on GABAergic interneurons and inhibits their activity. As there is heterogeneity in the vCA1 cell population, a subset of cells can be preferentially potentiated by NT due to higher expression of NTR1s or preferentially potentiated by 5-HT as they receive inputs from interneurons with higher expression of 5-HT1BRs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Zorab, J.M., Li, H., Awasthi, R. et al. Serotonin and neurotensin inputs in the vCA1 dictate opposing social valence. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08791-9'>Picuris Pueblo oral history and genomics reveal continuity in US Southwest</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 15:49:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Indigenous groups often encounter significant challenges when asserting ancestral claims and cultural affiliations based on oral histories, particularly in the USA where such narratives have historically been undervalued. Although ancient DNA offers a tool to complement traditional knowledge and address gaps in oral history, longstanding disregard for Indigenous sovereignty and beliefs has understandably led many Indigenous communities to distrust DNA studies1,2,3,4. Earlier research often focused on repatriation claims5,6,7, whereas more recent work has increasingly moved towards enhancing Tribal histories8,9. Here we present a collaborative study initiated by a federally recognized Native American tribe, the sovereign nation of Picuris Pueblo in the Northern Rio Grande region of New Mexico, USA, to address gaps in traditional knowledge and further their understanding of their population history and ancestry. We generated genomes from 16 ancient Picuris individuals and 13 present-day members of Picuris Pueblo, providing genomic data spanning the last millennium. We show genetic continuity between ancient and present-day Picuris, and more broadly with Ancestral Puebloans from Pueblo Bonito in Chaco Canyon10, 275 km to the west. This suggests a firm spatiotemporal link among these Puebloan populations of the North American Southwest. Furthermore, we see no evidence of population decline before European arrival11,12,13, and no Athabascan ancestry in individuals predating 1500 ce, challenging earlier migration hypotheses14,15,16. This work prioritizes Indigenous control of genetic data and brings together oral tradition, archaeology, ethnography and genetics. Ancient pueblos created by the Ancestral Pueblo culture, a term indicative of their ancestral relationship to contemporary Native American Pueblo Tribes, are among the most recognizable archaeological culture of North America. Many ancient pueblos have become the focus of US National Parks, Historic Parks or Monuments, and several have become UNESCO World Heritage Sites, including the large, multi-story masonry ‘Great Houses' concentrated in Chaco Canyon within the San Juan River Basin on the Colorado Plateau in Northwestern New Mexico in the USA17. Chacoan Great Houses were constructed between roughly 850 and 1150 ce, and were central to a broader ritual landscape of agricultural communities across the northern Southwest (Fig. Chaco Canyon was the centre of an extensive political and social network of more than 200 Great House communities covering an area larger than modern England. However, after around 1150 ce major construction ceased in Chaco Canyon and the area was largely depopulated for reasons still debated, which include drought, overexploitation of available ecological resources or arable land, collapse of regional exchange and belief systems or a combination of those factors20,21. Chaco culture (Ancestral Puebloan) related sites are indicated with circles and present-day Pueblos are indicated with squares. Base elevation data are derived from the USGS EROS SRTM 1-arc second digital elevation model99, with smoothing applied to produce derivative hillshade and terrain-relief datasets. Where Chacoan populations resettled in roughly 1150 ce, and what subsequent migrations they might have undertaken, remains a debated topic within Southwestern Archaeology and Indigenous communities alike. One area that has gained significant interest for its connections to Chaco is the Northern Rio Grande (NRG) region. Dating to roughly 900 ce, the earliest NRG Puebloan centres were contemporaneous with early Chaco, whereas archaeological, historic and modern pueblos—such as Picuris—provide evidence of NRG cultural continuity since the depopulation of Chaco. Whereas the oral traditions of present-day Puebloans22,23,24,25,26,27,28,29,30 and many lines of scholarly evidence18,31,32,33 support some level of relationship between the inhabitants of Chaco Canyon and the NRG, the nature and depth of the relationship between these two distant regions is debated (see Supplementary Information 3.5 for a discussion on some archaeological models). This has inevitably led to scepticism about the strength of links between past and present Puebloan groups, which has negatively affected Tribal Nations seeking to protect and preserve Chaco Canyon and its ancestral role in the Puebloan world11,12,13. Traditional knowledge keepers at Picuris Pueblo assert ancestral and cultural connections to numerous Ancestral Pueblo and Chaco culture sites in the San Juan Basin. The specifics of these relationships cannot be discussed outside traditional Picuris contexts, but include architectural motifs, an overlap between their racetracks with the system of Chacoan roads—which connect present-day Picuris with the San Juan Basin—and shared religious symbols found in material culture (such as pottery and sacred figurines) (Supplementary Information 3.1). However, they also acknowledge uncertainties because of gaps in their oral history. Such gaps are inevitable in a centuries-old community that by our estimate (below) lost at least 85% of their members in the first several decades after European Contact. By 1700 ce, the community had shrunk to around 300 members, making them the smallest of the extant Pueblo nations. In addition, centuries of oppression and prohibition of Picuris religion and rituals, the loss of access to traditional and sacred landscapes, and the placement of Picuris children into Native American boarding schools inhibited the intergenerational transmission of cultural knowledge and history at Picuris. Cumulatively, these factors have often minimized Picuris' voice in continuing discussions regarding the status and future of Chaco Canyon. The Picuris community initiated this research to bridge and mend gaps in their traditional knowledge with appropriate scientific data, and to address the extent to which they might have ancestral ties to Chaco Canyon and the regional system it anchored centuries ago. Recognizing the potential of ancient DNA to help secure links to their past, the Governor and leadership of Picuris Pueblo requested some of their archaeological collaborators help them initiate conversations with the Centre for GeoGenetics (University of Copenhagen). In many consultations over a 2-year period, a mutually agreed-on collaborative research design was developed to investigate Picuris population, genomic and health history (Supplementary Information 2). A memorandum of agreement resulted between the sovereign nation of Picuris Pueblo and the University of Copenhagen for this collaborative study. Following that, Picuris approved the transfer of ancient individuals excavated from the pueblo in the 1960s34, that were on long-term loan to Southern Methodist University. For clarity, we have chosen to address both the Picuris Pueblo Tribal Nation and its members in the third person, even though some of its members are coauthors of this study. Indigenous peoples of the Americas have documented their history through rich oral traditions, encompassing narratives, storytelling, songs, poetry and music35, as well as through painted and sculpted art forms27,31,36. Whereas there is some debate whether those Indigenous histories can be co-interpreted alongside archaeological and/or genetic evidence and how37,38,39,40,41, there is a general agreement that Indigenous narratives possess both symbolic and literal meanings, recording historical events42,43 and probably others detected archaeologically29,44. However, it is vital to recognize that oral narratives have many functions, and they may concentrate on a cosmological dimension or a character's intentions instead of more quotidian questions such as when and where22,25. Furthermore, oral narratives thrive in their plurality, as alternative, complementary accounts, and do not seek consensus to establish their importance25,26,28. Picuris trace their origins to an underground realm inhabited by Pá à wíá è Páyó, the Creator, and other peoples. The community is divided in two halves (the ‘North' and ‘South' people), which have separate birthplaces in Picuris oral histories. These peoples were sent to the present, surface world by the Creator, some on clouds, others by watery routes, with the earliest peoples emerging from lakes near big white mountains. One of those lakes is called Phaxwii Oxwalna, and thought to be Serpent Lake, near Jicarita Peak, a mountain with perennial snow. It is located south of Picuris and, fittingly, the origin of the ‘South' people. The geographical correspondence of the ‘North' people's emergence location is less clear, with some traditions identifying it as a spring near Pikes Peak, Colorado. After migrating through the landscape for an unknown duration of time, these peoples settled at Picuris and Taos Pueblos, as well as Pot Creek Pueblo, which is located between the two. Archaeological estimates put the founding of these communities at roughly 900 ce. Although Pot Creek Pueblo was depopulated in roughly 1320 ce, Picuris and Taos, communities that speak closely related languages, were not, and represent some of the oldest, continuously occupied communities in the Americas. In dedicated ancient DNA clean laboratories at the Center for GeoGenetics, we extracted DNA45,46 and built 29 double-stranded libraries47,48 and 42 uracil-specific excision reagent (USER)-treated single-stranded libraries49,50 of 16 ancient individuals buried in Picuris Pueblo, dated between 500 and 700 years ago (median depth of coverage, 2.67×, ranging from 0.009× to 36.6×). All non-USER-treated libraries showed positive evidence of post mortem ancient DNA damage51, and we estimated present-day human contamination to be below 2% for all individuals52,53,54 (Supplementary Tables 1 and 2 and Supplementary Information 4 and 5). At the community request, we also sequenced 13 present-day genomes from enrolled members of the Picuris Nation (Supplementary Table 3). We co-analysed these data with 590 ancient and modern genomes from the Americas and Siberia (Methods and Supplementary Information 9.1). For all individuals, we obtained imputed diploid genotypes for the 44,136,290 variable sites in the 1000 Genomes high-coverage dataset, using GLIMPSE v.1.1.1 (refs. 55,56,57) (Supplementary Table 4 and Supplementary Information 9.1.2). As the genetic data of many Indigenous North American groups are available only as genome-wide arrays or hybridization methodologies, we also subset our dataset accordingly to allow a direct comparison between the newly generated data and the wealth of sequences in the literature58,59 (Methods, Supplementary Tables 5 and 6, and Supplementary Information 9.2). We used principal component analysis (PCA)60, f statistics61, patterns of sharing of identity-by-descent (IBD) segments62 and estimated pairwise branch lengths5 to explore the genetic affinities between Ancestral and present-day Picuris, and all other Indigenous Americans with available genome-wide data, including the published low-coverage data from elite burials in Pueblo Bonito10 (Supplementary Information 10, 11.1, 11.2, 14.2 and 16). We find that Ancestral and present-day Picuris are most closely related to each other, confirming the genetic continuity in Picuris Pueblo throughout the last millennium. Outgroup f3 statistics, D statistics, TreeMix63 and PCA results unambiguously show that no other sampled population, ancient or present-day, is more closely related to Ancestral Puebloans from Pueblo Bonito than the Picuris individuals are (Fig. 10–12 and 15–25 and Extended Data Fig. These results are supported by Y-chromosome64 and mitochondrial DNA65 haplogroup distributions (Extended Data Figs. We acknowledge, however, that DNA sampling in the region is very limited and includes no other Pueblo communities. Nonetheless, sampled individuals buried in Room 33 in Pueblo Bonito all carried the same very rare mitochondrial haplotype (B2y1), suggesting a matrilineal dynastic system. We found one modern Picuris individual to carry this same haplotype, and to be closer to the Pueblo Bonito individuals than to any other sampled population in both maximum-likelihood and Bayesian analyses (Extended Data Fig. 1 and Supplementary Fig. a, Outgroup f3 statistics in the format f3(PuebloBonitoRoom33, pop, Yorubas), in which we find present-day and ancient Picuris Pueblo to share the most shared genetic drift with ancient individuals from Pueblo Bonito. Error bars represent about 3.3 standard errors (P ≈ 0.001 in a Z test with 172,863 SNPs in 5-Mb jackknife blocks). b, Same outgroup f3 statistics results, plotted on a map of North America. Warmer colours indicate higher f3 values. Triangle and circle denote ancient and present-day individuals, respectively. c, Treemix maximum-likelihood tree of Indigenous North Americans, rooted using a West African population (Yorubas). Populations were grouped and branches were shortened (denoted with a *) for increased readability. Unedited results can be found in Extended Data Fig. Previous studies have shown that Native American genomic ancestry featured two early splits, taking place between 22 and 14.6 thousand years ago (ka)7,58,59,66,67, giving rise to three main lineages. The first group to diverge were Ancient Beringians, represented by 11.5–9-ka-old individuals from Alaska7,67. This is followed by a split that gave rise to two extra lineages; one that includes mainly speakers of Athabascan and Algic languages, as well as groups in the Pacific Northwest58,59, whereas the other comprises all other previously studied Native Americans, including all ancient South Americans, and are represented by a roughly 12.8 ka individual of the Clovis culture (‘Anzick1')5. However, many Indigenous groups have a complex population history that involves ancestry from above this split, from both lineages or ancestry related to the lineage represented by Anzick1, but not present in Anzick1 itself5,7,9,68,69,70,71 (Supplementary Information 11.2.2). Using D statistics of the form D(ancientPicurisPueblo, Karitiana, Anzick, Yorubas)5,61 we found a significant rejection (D = −0.017; Z = −4.19) of ancient individuals from Picuris forming a clade with Southern Amazonian Karitiana to the exclusion of Anzick1, indicating Picuris carries ancestry that is not entirely from the lineage represented by Anzick1, a similar signal found in some North Mexican populations7,71,72 (Supplementary Figs. However, when directly testing whether those populations carry instead ancestry from the other lineage in the form D(pop, Karitiana, test, USR1.11,500 bp) we found all to be consistent with zero (maximum D = 0.011, Z = 1.82 for LaPlaya.600 bp and ‘CanAmerindian_1'), suggesting our dataset does not include a suitable proxy for the excess ancestry not represented in Anzick1 found in Picuris and other populations. This probably represents either ancestry more basal than the one found in the Anzick1 genome, or an ancestry source very distantly related to the ones with genetic data available. To investigate whether the end of construction and occupation of Chacoan Great Houses in the North American Southwest roughly 1150–1200 ce was followed or preceded by a population bottleneck, we applied three different methods73,74,75 on the imputed diploid genotypes from ancient and modern Picuris separately to estimate founder effects and their effective population size (Ne) over time. We estimate a similar population size in Picuris before European arrival using both ancient and present-day individuals, while also detecting no signal of population decline in pre-Colonial time using the three methods (Fig. Left, the average shared IBD segments longer than 8 centiMorgans (cM) between Indigenous American populations and present-day individuals from Picuris Pueblo. Populations are ordered by latitude and coloured by major geographic areas. Right, the demographical history of present-day Picuris individuals inferred using HapNe-LD with imputed diploid genotypes. Shaded areas show 95 and 90% confidence intervals, and the dotted line the median estimate. The x axis is the time (measured in generations ago since the time of sampling), and the y axis the effective population size (Ne). Archaeological and ethnohistoric estimates put the population of Picuris Pueblo on the eve of European arrival at around 3,000 individuals76,77,78. Architectural room-count-based estimates for the Northern Tiwa area in general (which includes Picuris), suggest that around 10,000 individuals occupied the region roughly 1300–1350 ce (ref. Although it is challenging to translate effective population size estimates to census sizes in human populations, effective population sizes are conservatively estimated to roughly one-third of the census size79 or of a ratio between 0.21 and 0.65 (refs. Therefore our point estimates, calculated both with ancient (harmonic mean over the ten last generations: 3,190 using IBDNe (ref. 73) and 5,300 using HapNe-LD75) or present-day Picuris Pueblo members (harmonic mean between 20 and 30 generations, 5,480 using HapNe-LD) are consistent with estimates of around 10,000 individuals being part of the Picuris sphere of influence at the time. The estimate of around 3,000 people at Picuris Pueblo is supported by oral traditions at Picuris, as well as archaeological estimates based on ceramic frequencies and architectural space occupied at the site's maximal size. Using these high-quality genomic data, we sought to investigate the relationship of a Pueblo population with other populations, such as Athabascan-speaking groups. The latter occupy a large contiguous region in interior Alaska and western Canada, with two outlier populations residing along the Pacific Coast of South Oregon and Northern California, and the other in the US Southwest (the so-called Southern Athabascans)81. Whereas a north-to-south direction for their migration is attested in oral tradition82 and universally accepted among scholars83, its timing, causes, source population(s) and routes, and arrival times in those regions have been of longstanding academic debate. The oldest direct archaeological evidence of Southern Athabascan settlement in the Southwest is a proposed Navajo-affiliated site in the Gobernador area of northwestern New Mexico, with a tree ring date of 1541 ce (ref. 84), a date that is coincident with the oldest written colonial record of a Southern Athabascan population—the ‘Querechos', commonly identified as Apaches85—in the region. The archaeological Tierra Blanca complex has been tentatively suggested by some to be culturally associated with these groups, and would push the date of an Athabascan presence back to roughly 1450 ce (refs. One hypothesis suggests this movement took place due to the cold climate of the Little Ice Age (which peaked in North America roughly 1400 ce)86,87, the colder conditions leading some northern groups to move south. In contrast to this hypothesis, other researchers have proposed that there was a substantially earlier Southern Athabascan presence in the region, perhaps by roughly 800 ce. The lack of archaeological evidence in support of this hypothesis is attributed to the group's low census size and high mobility, which would have left relatively few archaeological traces14,15,16. To investigate whether it is possible to resolve the timing of Athabascan dispersal, we compared the Southern Athabascan individuals reported in ref. 59 with all individuals in our dataset. Using f statistics61,88, we found that Southern Athabascans can be modelled as receiving most of their ancestry from a Puebloan source (ancientPicurisPueblo) alongside a Northern Athabascan population (for example, Chipewyan) (P = 0.34, Fig. This is in line with the oral history of Southern Athabascans from the time of their arrival in the Southwest82, ethnographic evidence of many clan names being of ultimately Puebloan origin89,90,91, loanwords from Puebloan languages92 and, critically, the adoption of a more sedentary lifestyle and cultivation of maize, squash, beans, cotton and tobacco. a, D statistics in the format D(Southern_Athabascans_1, Chipewyan, pop, Yorubas), where warmer colours indicate higher values of D, therefore a stronger genetic similarity with Southern Athabascans. Circles and triangles denote present-day and ancient individuals, respectively. Ancestral Puebloans from Picuris Pueblo is the population that maximize the value of D. Full D statistics results can be found in Supplementary Fig. 36. b, D statistics in the format D(Southern_Athabascans_1, ancientPicurisPueblo, pop, Yorubas). Athabascan speakers from northern North America are the ones to maximize the value of D (for example, Chipewyan). Full D statistics results can be found in Supplementary Fig. For both plots, error bars represent about 3.3 standard errors (P ≈ 0.001 in a Z test with 172,863 SNPs in 5-Mb jackknife blocks). Southern Athabascan traditions link them to interactions with Ancestral Puebloan communities before sixteenth century European Contact, but we do not find evidence of Athabascan ancestry in any Ancestral Picuris individual predating 1535 ce; such ancestry only occurs among present-day Picuris individuals (Supplementary Figs. These results support a late arrival of Athabascan-related populations in the US Southwest, most likely after 1500 ce. Puebloan marriage preferences, burial practices, or admixture evident in Southern Athabascans could have been one-directional, as for example through Pueblo captives. Although this would explain the absence of an Athabascan ancestry signal in ancient Picuris, given the extensive interactions between those two groups, both historically as well as to this day, it is unlikely that such a scenario persisted over an extended period. In conclusion, we show that individuals from Picuris Pueblo are the closest sampled population, past or present, to ancient individuals from Chaco Canyon, challenging claims of depopulation or disappearance in the area, and establishing a genetic component to suspected cultural affiliation between a present-day group and Ancestral Puebloan heritage. We emphasize that this conclusion does not challenge or call into question the connections and relationships that more than two dozen federally recognized Tribes have to Chaco Canyon. It is, however, the only example of paleogenetic data supporting a federally recognized Tribe's affiliation with Chaco Canyon ancestors. Whereas Indigenous oral histories are increasingly being accepted and legislated as evidence of cultural affiliation, as they should be, caution is also needed to avoid unintentionally excluding groups such as Picuris Pueblo who have lineal and cultural affiliation, but who may have notable gaps in their histories due to centuries of continuing systemic injustices and colonial rule. Scientific tools, such as genetics, archaeology and linguistics, can assist in demonstrating local genetic continuity7,8,93,94,95,96 and provide opportunities to mend gaps in oral histories. Their application can have profound restorative value for present-day communities, discussions of their cultural heritage and ensure their federally recognized rights as stakeholders of ancient sites. Finally, a crucial aspect of this research has been the shared commitment to ensure data sovereignty for Picuris Pueblo throughout this collaboration. Tribal leadership was involved in both drafting the research agreement and crafting the key findings of the work. Furthermore, Picuris Pueblo retained the right to curtail research at any point in the collaboration, and publication of results continues to be the sole decision of Picuris leadership. These guidelines and policies were an essential aspect of this collaborative research between non-Indigenous researchers and an Indigenous stakeholder community3,97,98. We hope collaborative efforts such as this one serve as catalysts for meaningful action and policy consideration, greater respect for oral histories and traditions, and that these results can be incorporated into well informed decision-making processes affecting Tribal sovereignty and community identity. Following the decision of Picuris Pueblo members to study their population history using DNA, their archaeological collaborators initiated contact with the Centre for GeoGenetics in the University of Copenhagen. After 2 years of consultation and the signing of a research agreement, ancient individuals from Picuris Pueblo on long-term loan to Southern Methodist University were sent to the Centre for GeoGenetics. Saliva samples from present-day members of Picuris Pueblo were collected by members of the community following the signature of an informed consent that was explained by a Tribal liaison in both English and Northern Tiwa. Ethical oversight was provided by the Picuris Pueblo Tribal Council, which reviewed the study and granted permission for its conduct, with individual and tribal consent obtained from all participants. Furthermore, we consulted the National Committee on Health Research Ethics in Denmark, which had no comments on the study design H-16018872 (Supplementary Information 1–3). All ancient DNA work was performed in dedicated clean laboratories at the Centre for GeoGenetics, GLOBE institute, University of Copenhagen. We extracted DNA from between 50 and 100 mg of crushed bone powder obtained from tooth cementum and the pars petrosa of the temporal bone of 28 samples from 22 ancient individuals. We built 34 double-stranded and 54 single-stranded Illumina libraries from those extracts, the latter after USER treatment. DNA was extracted from present-day saliva samples using Qiagen DNeasy Blood & Tissue Kit and built into dual-indexed Illumina libraries. All libraries were paired-end sequenced (2 × 100 bp) on an Illumina NovaSeq 6000 platform at the GeoGenetics Sequencing Core (Supplementary Information 4). Accelerator mass spectrometry was performed on the University of California Irvine Keck-CCAMS facility to obtain direct radiocarbon dates. Calibrated dates were obtained with Bayesian modelling done using OxCal v.4.4 (ref. 100) software using the Northern Hemisphere curve101. Raw sequencing data were demultiplexed and had their adaptors trimmed using AdapterRemoval v.2.3.2 (ref. Collapsed and paired-end reads were mapped to GRCh38 using bwa aln v.0.7.17 (ref. 103), optical duplicates were marked using Picard v.2.25.0 and depth of coverage and average read length estimated using pysam v.0.22.1. Ancient DNA characteristic misincorporation patterns were measured using mapDamage v.2.0 (ref. 104), and present-day contamination using ContamMix52 and ANGSD v.0.931 (ref. For all analyses, only reads with mapping quality greater than or equal to 30 and bases with quality equal to or greater than 20 were considered. Chromosomal sex was estimated by estimating the ratio of reads mapping to the autosomal, X- and Y-chromosomes (Supplementary Information 5 and 6). Consensus sequences were obtained using samtools v.1.3.1 mpileup105), requiring a minimum of five reads and more than 70% frequency in a locus to call a base. Haplogroup assignment was done using Haplogrep v.3 (ref. Following sequence alignment using mafft107, haplogroup-specific phylogenetic trees were built using raxml-ng v.0.8.1 (ref. 108) and BEAST v.2.6 (ref. Genotypes for positions located in the single-copy short-read callable 10-Mb region of the Y-chromosome110 were called using bcftools v.1.17 mpileup, excluding triallelic locus, indels and variants called in less than 95% frequency in the locus. Haplogroups were called by matching ancestral and derived calls to ISOGG 2019–2020, private databases and a previously published curated database111 (Supplementary Information 8). We compared Picuris Pueblo present and ancient individuals with 835 published present-day and ancient genomes from the literature5,6,7,59,67,68,72,93,94,95,112,113,114,115,116,117,118,119,120,121,122,123, alongside the 2,503 individuals from the 1000 Genomes high-coverage dataset124. Furthermore, as data from some key population only exists as hybridization capture or single-nucleotide polymorphism (SNP) array10,58,69,71,125,126,127,128,129,130,131,132,133, we subset our whole genome dataset to those to allow a direct comparison. In total, we compared Picuris Pueblo data with more than 300 populations from the American continent and more than 5,500 individuals worldwide (Supplementary Information 9). We used GLIMPSE v.1.1.1 (ref. 55) to impute ancient and present-day genomes with depth of coverage above 0.3× on a set of 44,136,290 variable sites on the 1000 Genomes high-coverage dataset, which was used as a reference panel. We excluded individuals with average genotype probability below 0.98. For the remaining individuals, we excluded variants with an INFO score below 0.8, not present in the GRCh38 strict accessibility mask and with a reference panel allele frequency below 1% (Supplementary Information 9). We performed PCA on the dataset as implemented in plink v.1.90 (ref. 134), first including all individuals, then restricting to Far East Asia and the Americas and finally only including North American individuals. We used ADMIXTURE63 to investigate ancestry proportions and detect the presence of non-Indigenous American ancestry. We masked African and European tracts in present-day Indigenous Americans using RFMix v.2.03-r0 (ref. 135) (Supplementary Information 9 and 10). Using FrAnTK136, we computed f3 statistics to estimate shared drift between two populations and D statistics to formally test hypothesis of cladality and gene flow61. We used qpAdm88, as implemented in Admixtools 2 (ref. 137), to model individual and population ancestry proportions (Supplementary Information 11). We identified segments in IBD in the whole genome dataset using IBDseq138, but only segments with an LOD score higher than three and that were longer than two centiMorgans, and removing regions with excess IBD sharing following138. IBD sharing between individuals was averaged out per population (Supplementary Information 14). To test the hypothesis of a pre-Colonial depopulation event taking place in the North American Southwest, we estimated demographic trajectories using IBD segments with IBDNe (ref. 73) and using patterns of linkage disequilibrium using HapNe-LD75. Age of founder population and bottlenecks were estimated using ASCEND74, and timing for admixture events with DATES139 (Supplementary Information 12, 13 and 15). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All maps were made using freely available vector and raster data from Natural Earth or the US Geological Survey and plotted using ESRI ArcGIS Pro v.3.0 and cartopy v.0.24.1. Compressed sequence alignment map files (BAM) aligned using human reference genome GRCh38 for ancient and present-day individuals are available in the European Genome-Phenome Archive under accession numbers EGAD50000001245 and EGAD50000001246. Picuris Pueblo is the sole owner of this dataset. Data access is controlled by the Picuris Pueblo Tribal Council through standard signed agreements that regulate the use of the ancient and/or present-day data. Consent for access will be granted exclusively to universities and research institutions exclusively for studies in population genetics that focus on understanding population history. The agreement strictly prohibits the use of the data for commercial purposes, inclusion in any kind of private databases (including forensic, mitochondrial or Y-chromosome) or medical or natural selection studies. Data cannot be used for tribal enrolment purposes. Present-day Picuris genomes are available exclusively for replication purposes using exactly identical comparison datasets, parameters and software and are controlled under a different agreement. Requests for data access will be evaluated jointly by the authors and the currently serving Picuris Pueblo Tribal Council every 3 months during the first year since publication, and every 6 months thereafter. Legitimate medical applications will be considered by the Picuris Pueblo Tribal Council if they explicitly serve Picuris health interests and guarantee that Picuris Pueblo retains full ownership of the data at all times and the right to halt any project at its sole discretion. Furthermore, previously published genomic data used for comparison are detailed and referenced in Supplementary Tables 4–6. They were obtained from the following sources: access granted by authors58,120,122,131,132,133; custom repositories (https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/)120,121,124; European Nucleotide Archive: SRX381032 (ref. Analyses were done using freely available software, which have been fully referenced throughout the paper and Supplementary Information. Custom code can be found on https://github.com/abigailramsoe/lundbeck-pipeline. Claw, K. G. et al. Chaco Canyon dig unearths ethical concerns. Google Scholar Bardill, J. et al. Advancing the ethics of paleogenomics. Google Scholar Garrison, N. A. et al. Genomic research through an indigenous lens: understanding the expectations. Google Scholar Tsosie, K. S. et al. Ancient-DNA researchers write their own rules. Google Scholar Rasmussen, M. et al. The genome of a Late Pleistocene human from a Clovis burial site in western Montana. Google Scholar Rasmussen, M. et al. The ancestry and affiliations of Kennewick Man. Google Scholar Moreno-Mayar, J. V. et al. Early human dispersals within the Americas. Google Scholar Severson, A. L. et al. Ancient and modern genomics of the Ohlone Indigenous population of California. Google Scholar First Rider, D. et al. Genomic analyses correspond with deep persistence of peoples of Blackfoot Confederacy from glacial times. Google Scholar Kennett, D. J. et al. Archaeogenomic evidence reveals prehistoric matrilineal dynasty. Google Scholar Colton, H. S. The rise and fall of the prehistoric population of Northern Arizona. Google Scholar Collapse: How Societies Choose to Fail or Succeed (Penguin, 2005). Two views of collapse. Google Scholar Derry, D. E. Later Athapaskan prehistory: a migration hypothesis. Google Scholar Seymour, D. J. Gateways for Athabascan migration to the American Southwest. Google Scholar Ives, J. W. A Theory of Northern Athapaskan Prehistory (Routledge, 2019). World Heritage in the United States. National Park Service www.nps.gov/subjects/internationalcooperation/worldheritage.htm (2024). Ancient Puebloan Southwest (Cambridge Univ. Lekson, S. H. The Architecture of Chaco Canyon, New Mexico (Univ. Lekson, S. H. & Cameron, C. M. The abandonment of Chaco Canyon, the Mesa Verde Migrations, and the reorganization of the Pueblo World. Google Scholar Lentz, D. L. et al. Ecosystem impacts by the Ancestral Puebloans of Chaco Canyon, New Mexico, USA. Google Scholar The Tewa World: Space, Time, Being, and Becoming in a Pueblo Society (Univ. Sando, J. S. Nee Hemish, a History of Jemez Pueblo (Univ. New Mexico Press, 1982). Sando, J. S. Pueblo Nations: Eight Centuries of Pueblo Indian History (Clear Light, 1992). Naranjo, T. Thoughts on migration by Santa Clara Pueblo. Google Scholar Colwell-Chanthaphonh, C. Native American perspectives of the past in the San Pedro Valley of Southeastern Arizona. Google Scholar Sekaquaptewa, E. & Washburn, D. They go along singing: reconstructing the Hopi past from ritual metaphors in song and image. Google Scholar Colwell-Chanthaphonh, C. & Ferguson, T. J. Memory pieces and footprints: multivocality and the meanings of ancient times and ancestral places among the Zuni and Hopi. Google Scholar Bernardini, W. Identity as history: Hopi clans and the curation of oral tradition. Google Scholar Hays-Gilpin, K. & Lomatewama, R. Some contemporary pueblo perspectives on archaeology in the Pueblo world. Google Scholar Schaafsma, P. The Cave in the Kiva: the Kiva Niche and painted walls in the Rio Grande Valley. Google Scholar Washburn, D. K., Crowe, D. W. & Ahlstrom, R. V. N. A symmetry analysis of design structure: 1,000 years of continuity and change in Puebloan ceramic design. Google Scholar Fowles, S. M. An Archaeology of Doings: Secularism and the Study of Pueblo Religion (School for Advanced Research Press, 2013). Dick, H. W. Picuris Pueblo Excavations, November 1964 Document No. PB 177047 (Department of the Interior, National Park Service, National Technical Service Information, 1965). Colwell, C. in The Oxford Handbook of Southwest Archaeology (eds Mills, B. & Fowles, S. M.) (Oxford Univ. Signs from the Ancestors: Zuni Cultural Symbolism and Perceptions of Rock Art (Univ. Swanton, J. R. & Dixon, R. B. Primitive American history. Lowie, R. H. Oral tradition and history. Mason, R. J. Archaeology and Native North American oral traditions. Whiteley, P. M. Archaeology and oral tradition: the scientific importance of dialogue. Schaafsma, C. F. Truth dwells in the deeps: Southwestern oral traditions and archaeological interpretations. Truth and the Hopi: an historiographic study of documented oral tradition concerning the coming of the Spanish. Colwell-Chanthaphonh, C. Western Apache oral histories and traditions of the Camp Grant Massacre. American Indian Quarterly 27, 639–666 (2003). Echo-Hawk, R. C. Ancient history in the New World: integrating oral traditions and the archaeological record in deep time. Rohland, N., Glocke, I., Aximu-Petri, A. & Meyer, M. Extraction of highly degraded DNA from ancient bones, teeth and sediments for high-throughput sequencing. Damgaard, P. B. et al. Improving access to endogenous DNA in ancient bones and teeth. Meyer, M. & Kircher, M. Illumina sequencing library preparation for highly multiplexed target capture and sequencing. Cold Spring Harb. Margaryan, A. et al. Population genomics of the Viking world. Gansauge, M.-T., Aximu-Petri, A., Nagel, S. & Meyer, M. Manual and automated preparation of single-stranded DNA libraries for the sequencing of DNA from ancient biological remains and other sources of highly degraded DNA. Briggs, A. W. et al. Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA. Nucleic Acids Res. Briggs, A. W. et al. Patterns of damage in genomic DNA sequences from a Neandertal. Fu, Q. et al. A revised timescale for human evolution based on ancient mitochondrial genomes. Korneliussen, T. S., Albrechtsen, A. & Nielsen, R. ANGSD: analysis of next generation sequencing data. Moreno-Mayar, J. V. et al. A likelihood method for estimating present-day human contamination in ancient male samples using low-depth X-chromosome data. Rubinacci, S., Ribeiro, D. M., Hofmeister, R. J. Efficient phasing and imputation of low-coverage sequencing data using large reference panels. Sousa da Mota, B. et al. Imputation of ancient human genomes. Allentoft, M. E. et al. Population genomics of post-glacial western Eurasia. Reich, D. et al. Reconstructing Native American population history. Raghavan, M. et al. Genomic evidence for the Pleistocene and recent population history of Native Americans. Patterson, N., Price, A. L. & Reich, D. Population structure and eigenanalysis. Patterson, N. et al. Ancient admixture in human history. Browning, S. R. & Browning, B. L. Identity by descent between distant relatives: detection and applications. Pickrell, J. K. & Pritchard, J. K. Inference of population splits and mixtures from genome-wide allele frequency data. & Tyler-Smith, C. Human Y-chromosome variation in the genome-sequencing era. Kivisild, T. Maternal ancestry and population history from whole mitochondrial genomes. Willerslev, E. & Meltzer, D. J. Peopling of the Americas as inferred from ancient genomics. Moreno-Mayar, J. V. et al. Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans. Scheib, C. L. et al. Ancient human parallel lineages within North America contributed to a coastal expansion. Posth, C. et al. Reconstructing the deep population history of Central and South America. Castro e Silva, M. A., Ferraz, T., Bortolini, M. C., Comas, D. & Hünemeier, T. Deep genetic affinity between coastal Pacific and Amazonian natives evidenced by Australasian ancestry. Nakatsuka, N. et al. Genetic continuity and change among the Indigenous peoples of California. Villa-Islas, V. et al. Demographic history and genetic structure in pre-Hispanic Central Mexico. Browning, S. R. & Browning, B. L. Accurate non-parametric estimation of recent effective population size from segments of identity by descent. Tournebize, R., Chu, G. & Moorjani, P. Reconstructing the history of founder events using genome-wide patterns of allele sharing across individuals. Fournier, R., Tsangalidou, Z., Reich, D. & Palamara, P. F. Haplotype-based inference of recent effective population size in modern and ancient DNA samples. Ortiz, A. Handbook of North American Indians Vol. 9 Southwest (Smithsonian Institution, 1979). & Dick, H. W. Picuris Pueblo through Time: Eight Centuries of Change in a Northern Rio Grande Pueblo (William P. Clements Center for Southwest Studies, 1999). Schroeder, A. H. A Brief History of Picuris Pueblo: A Tiwa Indian Group in North Central New Mexico (Alamosa, 1974). Felsenstein, J. Inbreeding and variance effective numbers in populations with overlapping generations. Frankham, R. Effective population size/adult population size ratios in wildlife: a review. Mithun, M. The Languages of Native North America (Cambridge Univ. Sundberg, L. D. Dinétah: An Early History of the Navajo People (Sunstone, 1995). Magne, M. P. R. & Matson, R. G. Moving on: expanding perspectives on Athapaskan migration. Southwestern dated ruins, IV. A. Coronado's Querechos and Teyas in the archaeological record of the Texas Panhandle. Luckman, B. H. The Little Ice Age in the Canadian Rockies. Mann, M. E. et al. Global signatures and dynamical origins of the Little Ice Age and Medieval climate anomaly. Harney, É., Patterson, N., Reich, D. & Wakeley, J. Assessing the performance of qpAdm: a statistical tool for studying population admixture. Hodge, F. W. Pueblo Indian clans. Carr, M., Spencer, K. & Woolley, D. Navaho clans and marriage at Pueblo Alto. Social Life of the Navajo Indians: With Some Attention to Minor Ceremonies (Columbia Univ. De Reuse, W. J. in Studies in Language Companion Series Vol. 185 (eds Dakin, K. et al.) 301–318 (John Benjamins Publishing Company, 2017). de la Fuente, C. et al. Genomic insights into the origin and diversification of late maritime hunter-gatherers from the Chilean Patagonia. Schroeder, H. et al. Origins and genetic legacies of the Caribbean Taino. Capodiferro, M. R. et al. Archaeogenomic distinctiveness of the Isthmo-Colombian area. Barquera, R. et al. Ancient genomes reveal insights into ritual life at Chichén Itzá. Summer internship for INdigenous peoples in Genomics (SING) Consortium et al. A framework for enhancing ethical genomic research with Indigenous communities. Kowal, E. et al. Community partnerships are fundamental to ethical ancient DNA research. Earth Resources Observation and Science (EROS) Center. USGS EROS Archive—Digital Elevation—Shuttle Radar Topography Mission (SRTM) Void Filled. Ramsey, C. B. Bayesian analysis of radiocarbon dates. Reimer, P. J. et al. The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP). Schubert, M., Lindgreen, S. & Orlando, L. AdapterRemoval v2: rapid adapter trimming, identification, and read merging. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Jónsson, H., Ginolhac, A., Schubert, M., Johnson, P. L. F. & Orlando, L. mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters. Li, H. et al. The Sequence Alignment/Map format and SAMtools. Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic Acids Res. Katoh, K. & Standley, D. M. MAFFT Multiple Sequence Alignment Software Version 7: improvements in performance and usability. Kozlov, A. M., Darriba, D., Flouri, T., Morel, B. & Stamatakis, A. RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference. Bouckaert, R. et al. BEAST 2: a software platform for Bayesian evolutionary analysis. Poznik, G. D. et al. Sequencing Y chromosomes resolves discrepancy in time to common ancestor of males versus females. Pinotti, T. et al. Y Chromosome sequences reveal a short Beringian standstill, rapid expansion, and early population structure Native American Founders. Popović, D. et al. Ancient genomes reveal long-range influence of the pre-Columbian culture and site of Tiwanaku. Campelo dos Santos, A. L. et al. Genomic evidence for ancient human migration routes along South America's Atlantic coast. Rasmussen, M. et al. Ancient human genome sequence of an extinct Palaeo-Eskimo. Bongers, J. L. et al. Integration of ancient DNA with transdisciplinary dataset finds strong support for Inca resettlement in the south Peruvian coast. Lindo, J. et al. The genetic prehistory of the Andean highlands 7000 years BP through European Contact. Raghavan, M. et al. Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Sikora, M. et al. The population history of northeastern Siberia since the Pleistocene. Flegontov, P. et al. Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America. Mallick, S. et al. The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Bergström, A. et al. Insights into human genetic variation and population history from 929 diverse genomes. Raghavan, M. et al. The genetic prehistory of the New World Arctic. Joseph, S. K. et al. Genomic evidence for adaptation to tuberculosis in the Andes before European contact. Byrska-Bishop, M. et al. High-coverage whole-genome sequencing of the expanded 1000 Genomes Project cohort including 602 trios. Nakatsuka, N. et al. Ancient genomes in South Patagonia reveal population movements associated with technological shifts and geography. Fernandes, D. M. et al. A genetic history of the pre-contact Caribbean. Nägele, K. et al. Genomic insights into the early peopling of the Caribbean. Tiesler, V. et al. Life and death in early colonial Campeche: new insights from ancient DNA. Nakatsuka, N. et al. A Paleogenomic reconstruction of the deep population history of the Andes. Kennett, D. J. et al. South-to-north migration preceded the advent of intensive farming in the Maya region. Gnecchi-Ruscone, G. A. et al. Dissecting the pre-Columbian genomic ancestry of Native Americans along the Andes–Amazonia divide. Verdu, P. et al. Patterns of admixture and population structure in native populations of Northwest North America. Borda, V. et al. The genetic structure and adaptation of Andean highlanders and Amazonians are influenced by the interplay between geography and culture. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Maples, B. K., Gravel, S., Kenny, E. E. & Bustamante, C. D. RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference. Moreno-Mayar, J. V. FrAnTK: a Frequency-based Analysis ToolKit for efficient exploration of allele sharing patterns in present-day and ancient genomic datasets. Maier, R. et al. On the limits of fitting complex models of population history to f-statistics. Browning, B. L. & Browning, S. R. Detecting identity by descent and estimating genotype error rates in sequence data. Chintalapati, M., Patterson, N. & Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. We dedicate this paper to the memory of Richard Mermejo, former Picuris Pueblo governor, war chief and kiva leader, as well as the Tribal liaison of this project, who passed away on 27 January 2024. We thank Picuris Pueblo Tribal council for their support for the project, R. W. Hughes for legal assistance, G. Simbolo, L. Olsen and A. Razeto Richter for administrative and P. Selmer Olsen and T. Blumensaadt Brand for technical support. thanks St. John's College, Cambridge, for providing a stimulating environment of discussion and learning. The Lundbeck Foundation GeoGenetics Centre is supported by grants from the Lundbeck Foundation (grant nos. R302-2018-2155 and R155-2013-16338), the Novo Nordisk Foundation (grant no. NNF18SA0035006), the Wellcome Trust (grant no. 214300/A/18/Z), the Carlsberg Foundation (grant nos. CF18-0024, CF21-0135), the Danish National Research Foundation (grant nos. DNRF94, DNRF174), the University of Copenhagen (KU2016 programme) and Ferring Pharmaceuticals A/S to E.W. is funded by the National Science Foundation (grant no. is funded by the Queen Margrethe's and Vigdís Finnbogadóttir's Interdisciplinary Research Centre on Ocean, Climate and Society (grant no. is funded by the grant nos. CNPq 406864/2022-5 and FAPEMIG APQ-00220-22 from Brazil. is funded by the Quest Archaeological Research Fund and the Potts & Sibley Foundation. J.V.M.-M. is funded by the European Research Council (grant no. 101078151) and Villum Fonden (grant no. These authors contributed equally: Thomaz Pinotti, Michael A. Adler, Richard Mermejo These authors jointly supervised this work: David J. Meltzer, J. Víctor Moreno-Mayar, Craig Quanchello, Eske Willerslev Deceased: Richard Mermejo Lundbeck Foundation GeoGenetics Centre, GLOBE Institute, University of Copenhagen, Copenhagen, Denmark Thomaz Pinotti, Julie Bitz-Thorsen, Hugh McColl, Gabriele Scorrano, Motahareh Feizabadifarahani, Devlin Gandy, Charleen Gaunitz, Jesper Stenderup, Abigail Ramsøe, Thorfinn Korneliussen, Fabrice Demeter, Lasse Vinner, Martin Sikora, David J. Meltzer, J. Víctor Moreno-Mayar & Eske Willerslev Laboratório de Biodiversidade e Evolução Molecular (LBEM), Instituto de Ciências Biológicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil Thomaz Pinotti & Fabrício R. Santos Department of Anthropology, Southern Methodist University, Dallas, TX, USA Michael A. Adler, Matthew Boulanger & David J. Meltzer Picuris Pueblo, Peñasco, NM, USA Richard Mermejo & Craig Quanchello Center for Molecular Anthropology for the Study of Ancient DNA, Department of Biology, University of Rome ‘Tor Vergata', Rome, Italy Department of Archaeology and Classical Studies, Stockholm University, Stockholm, Sweden Department of Archaeology, University of Cambridge, Cambridge, UK Eco-anthropologie (EA), Departement ABBA, Muséum national d'Histoire naturelle, CNRS, Université Paris Cité, Musée de l'Homme, Paris, France Department of Genetics, University of Cambridge, Cambridge, UK MARUM Center for Marine Environmental Sciences, University of Bremen, Bremen, Germany You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar initiated the study. led the study. provided laboratory infrastructure. developed and maintained bioinformatic infrastructure. and J.B.-T. did laboratory work and data generation (with support from M.F. and supervision of G.S., H.M., C.G. analysed sequence data (with support from G.S. and supervision of J.V.M.-M.). led community engagement (with support from T.P., D.J.M., J.V.M.-M. and E.W.). compiled oral histories. provided archaeological curation and context. provided data visualization. were involved in conceptualizing the draft and final version. wrote the initial draft with significant input from D.G. All authors revised and agreed on the final submitted version. Correspondence to Craig Quanchello or Eske Willerslev. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The high branch lengths in some populations may reflect analytical artifacts: for ancient individuals, they arise from high error rates due to low coverage and DNA damage, while for present-day individuals, they result from missing data caused by masking non-Indigenous American segments. Present-day literature mitogenomes are in black, present-day Picuris in blue and all other ancient mitogenomes in gold. Near-zero length branches were collapsed for convenience. Ancient Picuris are in red and present-day Picuris in blue. Relevant literature individuals appear in italic. In accordance with Picuris wish to not have its data included in any private database, we do not describe any variant private to them. Supplementary Table 1: Per library sequencing summary statistics. Supplementary Table 2: Per individual sequencing summary statistics. Supplementary Table 3: Present-day Picuris individuals sequencing summary statistics. Supplementary Table 4: Reference genomic dataset summary statistics. Supplementary Table 5: Reference hybridization capture dataset. Supplementary Table 6: Reference array dataset. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Pinotti, T., Adler, M.A., Mermejo, R. et al. Picuris Pueblo oral history and genomics reveal continuity in US Southwest. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08888-1'>Adversarial testing of global neuronal workspace and integrated information theories of consciousness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 15:19:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Different theories explain how subjective experience arises from brain activity1,2. Here we present an open science adversarial collaboration directly juxtaposing integrated information theory (IIT)4,5 and global neuronal workspace theory (GNWT)6,7,8,9,10 via a theory-neutral consortium11,12,13. Human participants (n = 256) viewed suprathreshold stimuli for variable durations while neural activity was measured with functional magnetic resonance imaging, magnetoencephalography and intracranial electroencephalography. We found information about conscious content in visual, ventrotemporal and inferior frontal cortex, with sustained responses in occipital and lateral temporal cortex reflecting stimulus duration, and content-specific synchronization between frontal and early visual areas. These results align with some predictions of IIT and GNWT, while substantially challenging key tenets of both theories. For IIT, a lack of sustained synchronization within the posterior cortex contradicts the claim that network connectivity specifies consciousness. GNWT is challenged by the general lack of ignition at stimulus offset and limited representation of certain conscious dimensions in the prefrontal cortex. These challenges extend to other theories of consciousness that share some of the predictions tested here14,15,16,17. Beyond challenging the theories, we present an alternative approach to advance cognitive neuroscience through principled, theory-driven, collaborative research and highlight the need for a quantitative framework for systematic theory testing and building. Philosophers and scientists have sought to explain the subjective nature of consciousness (for example, the feeling of pain or of seeing a colourful rainbow) and how it relates to physical processes in the brain18,19. Furthermore, empirical support for a given theory is often highly dependent on methodological choices, pointing towards a confirmation bias in theory testing3. Convergence on a broadly accepted neuroscientific theory of consciousness will have profound medical, societal and ethical implications. To advance this goal, we tested two theories of consciousness, through a large-scale, open-science adversarial collaboration11,12,20,21,22 aimed at accelerating progress in consciousness research by building on constructive disagreement. We brought together proponents of IIT4,5 and GNWT6,23, in addition to theory-neutral researchers. The group identified differential existing and novel predictions of the two theories and developed an experimental design to test them (Fig. We preregistered these predictions, including pass or fail criteria, expected outcomes and their interpretation ex-ante11,12. We focus on IIT and GNWT, among other widely discussed theories (for example, recurrent processing theory and higher-order theories1,2), because they feature prominently in consciousness science, as demonstrated by a recent systematic review of the literature3. a, Predictions of IIT and GNWT. For prediction 1 (decoding of conscious content), IIT predicts maximal decoding of conscious content in posterior brain areas, whereas GNWT emphasizes a necessary role for the PFC. For prediction 2 (maintenance of conscious content), IIT posits that conscious content is actively maintained in the posterior cortex, whereas GNWT predicts brief content-specific ignition (approximately 0.3–0.5 s) in the PFC at stimulus onset and offset, with content stored in a non-conscious silent state between these events. Waveforms (left) and temporal generalization matrices (right) illustrate the predicted amplitude-based and information-based temporal profiles: coloured rectangles indicate the three stimulus durations for PFC (GNWT) and posterior cortex (IIT; left); the arrows mark stimulus onset (brown) and offset (red), whereas predicted temporal generalization is depicted in green (GNWT) and blue (IIT; right). For prediction 3 (interareal connectivity supporting consciousness), the stars and arrows on the brain diagram illustrate predicted synchrony patterns, with green representing GNWT and blue representing IIT. For instance, viewing the Mona Lisa involves experiencing it as occupying a specific spatial location, categorizing it as a face, recognizing an identity and noting its leftward orientation, with this complex experience maintained over time. c, To manipulate conscious content, stimuli varied across four dimensions: category (faces, objects, letters and false fonts), identity (different exemplars within each category), orientation (left, right and front views) and duration (0.5 s, 1.0 s and 1.5 s). Participants detected predefined targets (for example, a specific face and object or a letter and false font) in sequences of single, high-contrast stimuli. Each trial contained three stimulus types: targets (red; coloured frames for illustration only), task-relevant stimuli (orange-red; same categories as targets) and task-irrelevant stimuli (purple; other categories). Blank intervals between stimuli are not depicted. Object stimulus images in panels c,d are courtesy of Michael J. Tarr, Carnegie Mellon University, http://www.tarrlab.org/; face stimuli were created using FaceGen Modeler 3.1. IIT and GNWT explain consciousness differently: IIT proposes that consciousness is the intrinsic ability of a neuronal network to influence itself, as determined by the amount of maximally irreducible integrated information (phi) supported by a network in a state. On the basis of theoretical and neuroanatomical considerations, IIT suggests that a complex of maximum phi probably resides primarily in the posterior cerebral cortex, in a temporo–parietal–occipital ‘hot zone'4,5,24,25. GNWT instead posits that consciousness arises from global broadcasting and late amplification (or ‘ignition') of information across interconnected networks of higher-order sensory, parietal and especially prefrontal cortex (PFC)6,9,23. Both theories have a mathematical or computational core (integrated information for IIT and global workspace for GNWT) and proposed biological implementations (posterior cortex versus PFC and associated areas, respectively). Although it is difficult to test the mathematical or computational core of either theory directly, their competing biological implementations are empirically testable with current methodologies. Thus, our study focuses on brain regions where the predictions diverge most notably—posterior cortex for IIT and PFC for GNWT, rather than the associated areas in higher-order sensory or parietal cortex—to facilitate maximally diagnostic experiments. One consequence of this biological focus is that theorists could respond to challenging data by modifying the proposed biological implementation while retaining the mathematical or computational core of a theory. Another consequence is that some predictions (and their associated consequences) may overlap with other theories of consciousness that share similar biological bases, such as higher-order theories16,17 in the PFC and local recurrency theories14,26 in the visual cortex. Although these are inherent aspects of studying theoretical proposals about neural mechanisms of consciousness, the results are expected to help the community make more informed judgements about the tested theories (for rationale, see the preregistration document27). We tested three preregistered, peer-reviewed predictions of IIT and GNWT12 for how the brain enables conscious experience (Fig. Prediction 1 addresses the cortical areas holding information about different aspects of conscious content. IIT predicts that conscious content is maximal in posterior brain areas, whereas GNWT predicts a necessary role for PFC. Prediction 2 pertains to the maintenance of conscious percepts over time28,29,30. IIT predicts that conscious content is actively maintained by neural activity in the posterior ‘hot zone' throughout the duration of a conscious experience, whereas GNWT predicts ignition events in PFC at stimulus onset and offset, updating the global workspace, with activity-silent information maintenance in between31. Prediction 3 examines interareal connectivity during conscious perception. IIT predicts sustained short-range connectivity within the posterior cortex, linking low-level sensory (V1/V2) with high-level category-selective areas (for example, fusiform face area and lateral occipital cortex), whereas GNWT predicts long-range connectivity between high-level category-selective areas and PFC. The combination of predictions, tested through highly powered, multimodal studies, places a high bar for either theory to pass, rendering failures more informative. Predictions were differentially weighted on the basis of their centrality to the theory and methodological considerations (Extended Data Table 1; for an additional preregistered non-critical analysis, see section 8 in Supplementary Information). To empirically test these predictions, we investigated the content and temporal extent of conscious visual experiences, focusing on their phenomenological richness and multifaceted nature, even for a single stimulus. For instance, when viewing the Mona Lisa (Fig. To approximate such multifaceted experiences, we manipulated several attributes of conscious content by presenting suprathreshold visual stimuli across four different categories (faces, objects, letters and false fonts), each containing 20 unique identities shown in three orientations (front, left and right view) and for three durations (0.5, 1.0 and 1.5 s). In each block, participants were instructed to detect two infrequent target stimuli from either the pictorial (face–object) or symbolic (letter–false fonts) stimulus categories (for example, a specific face or object), making these categories task relevant for that block (Fig. First, it provides robust conditions to test the predictions of the theories by focusing on clearly experienced conscious content, studied through a high signal-to-noise, suprathreshold, fully attended single stimulus at fixation. This amplifies the significance of any challenges to the theories, as they cannot be explained by weak signals. Second, it minimizes task and report confounds, isolating neural activity specifically related to consciousness. Third, it allows testing of novel predictions to address previously unexplored questions, that is, how experience is maintained over time, refining theories and yielding new insights. All research was conducted by theory-neutral teams to minimize confirmatory bias. We evaluated the predictions of theories in 256 participants performing the same behavioural task in three neuroimaging modalities: functional magnetic resonance imaging (fMRI; n = 120), magnetoencephalography (MEG; n = 102) and intracranial electroencephalography (iEEG; n = 34). To overcome the spatial and temporal limitations of each modality, we combined whole-brain, non-invasive fMRI and MEG with invasive iEEG, ensuring methodological rigour. Combined with large sample sizes, this minimizes the likelihood that negative results are due to methodological or sensitivity issues. Data collection occurred in two (or three) independent laboratories for each modality to guarantee generalization across groups of participants, instruments and experimenters. To foster informativeness, reproducibility and robustness, we (1) separated theory proponents from data acquisition and analysis to minimize bias and post-hoc interpretation, (2) used a multimodal approach that maximizes spatiotemporal resolution and coverage for a stringent and comprehensive tests of the theories in humans, (3) predefined large samples to increase statistical power, (4) followed standardized32 and preregistered protocols12 to reduce setup differences and confirmatory bias22 (see sections 1 and 2 in Supplementary Information), and (5) implemented an analysis optimization phase (one-third of the sample) followed by a final testing phase (two-thirds of the sample) on independent data for result validation33. Consequently, this large-scale international effort aimed at implementing a rigorous adversarial collaboration framework, thereby establishing a precedent for an alternative scientific approach. According to IIT, PFC is not necessary for consciousness. Consequently, decoding conscious content should be most effective from the posterior cortex, and adding PFC activity as additional information should not improve decoding accuracy. This prediction was considered non-critical for testing IIT, as the theory focuses on the intrinsic, causal perspective of information within a neural substrate rather than the amount of information decodable from the perspective of an extrinsic observer5. By contrast, GNWT posits that conscious content can be decoded from PFC activity. Both theories predict that conscious content should be evident in theory-relevant areas independently of other cognitive processes (for example, report and task); thus, conscious content should be present irrespective of task manipulations34,35. This prediction was tested by evaluating the decoding accuracy of stimulus category (faces–objects (pictorial) and letters–false fonts (symbolic)) and orientation (left, right and front facing) in all theory-relevant areas. All stimulus categories alternated between being task-relevant and task-irrelevant across blocks (Fig. Stimulus orientation, being orthogonal to the task, remained task-irrelevant in all blocks. On the basis of our preregistered predictions and pre-approved interpretations27 (Extended Data Table 1), the theories would pass the test if decoding is possible for both category (in at least one category pairing) and orientation (in at least one category), but would fail otherwise. Testing both category and orientation decoding constitutes a stringent test, as it requires two conditions to be satisfied, increasing the likelihood of failures36, while capturing the critical multidimensionality of conscious content, that is, phenomenological richness (Fig. For decoding of category, we tested whether information is present in the relevant regions irrespective of the task, using a cross-task generalization approach (see Methods). Here we report the most robust results for decoding of category (faces–objects) and orientation (left, right and front views of faces). Qualitatively similar results were observed for decoding of letters–false fonts (Extended Data Fig. Results for orientation decoding were consistent across stimulus categories and data modalities in posterior cortex, but mostly absent in PFC (see section 5.1.2 in Supplementary Information). In the iEEG data, pattern classifiers were trained on high gamma frequency band (70–150 Hz), which correlates with spiking activity37,38, at each time point in the task-irrelevant condition, and tested in the task-relevant condition, for each stimulus duration and category, and across all electrodes within the theory-relevant region of interests (ROIs; see Fig. In the PFC ROIs, significant cross-task face–object decoding accuracy (approximately 70%) was also evident, but the temporal generalization of this decoding was restricted to approximately 0.2–0.4 s (Fig. 1e; within-task decoding is presented in Extended Data Fig. a, Spatial coverage of intracranial electrodes (npatients = 29) on a standard inflated cortical surface map (top), and within theory-defined ROIs (bottom): posterior (blue; nelectrodes = 583) and prefrontal (green; nelectrodes = 576). Pattern classifiers were trained to discriminate stimulus category (faces–objects) in the task-irrelevant condition at each time point and tested in the task-relevant condition across all time-points. Contoured red-shaded regions depict significant above-chance (50%) decoding. Here and below, significance was evaluated through a non-parametric cluster-based permutation test (P < 0.05; two-sided). c, MEG average cross-task decoding of stimulus category (n = 65) from task-relevant to task-irrelevant stimuli (purple) and vice versa (orange), separately for the posterior (top) and prefrontal (bottom) ROIs, depicted on inflated cortical surfaces (posterior in blue and prefrontal in green), across durations, using pseudotrial aggregation. The shading depicts 95% CI across participants. d, fMRI searchlight cross-task decoding of stimulus category (n = 73), collapsed across durations, from task-relevant stimuli to task-irrelevant stimuli (left; purple) or vice versa (right; orange). The outlined coloured regions on the inflated cortical surfaces (left–right lateral views; right–left medial views (bottom)) indicate significant above-chance decoding. e, iEEG ROIs significant cross-task decoding of stimulus category, collapsed across durations. f, iEEG average decoding of stimulus orientation (left, right and front) within posterior (top) and prefrontal (bottom) ROIs, collapsed across durations. The shading depicts 95% CI estimated across cross-validation folds. g, fMRI searchlight decoding of face orientation (left, right and front). Regions with significantly above-chance (33%) decoding accuracies are outlined in blue. h, iEEG ROIs decoding of face orientation (left, right and front). Conventions are as in panel g. i, MEG ROIs average decoding of face orientation (left, right and front). Conventions are as in panel f. Brain surfaces in panels a,c–e,g,h are from Freesurfer. Although electrode coverage across our sample of iEEG patients (n = 29 for decoding analyses) was exceptional in the relevant brain regions (Fig. 2a; PFC ROIs nelectrodes = 576, posterior ROIs nelectrodes = 583), we further analysed a larger population of healthy participants (n = 65) using MEG, focusing on theory-relevant ROIs (see Methods). Here too, cross-task generalization of face–object decoding was significant in both posterior and prefrontal ROIs (Fig. A searchlight approach (see Methods) revealed distributed, robust cross-task generalization (approximately 75%) in the striate and extrastriate, ventral temporal and intraparietal cortex (Fig. Generalization in PFC had lower accuracy (approximately 60%) and was spatially restricted to middle and inferior frontal cortex regions (Fig. Theory-relevant ROIs defined in the Destrieux atlas yielded comparable results (see section 5.1.1.3 in Supplementary Information). These results closely matched those from iEEG-restricted to theory-specified ROIs and time windows (Fig. Hence, across modalities, face–object decoding occurred in both posterior and prefrontal ROIs, consistent with IIT and GNWT predictions. Given the rich and multidimensional nature of conscious content, we assessed the decoding of stimulus orientation, which was always task-irrelevant. We obtained divergent results for IIT and GNWT: decoding of face orientation (left, right or front views) was achieved in posterior but not in prefrontal ROIs, both with iEEG (Fig. 2f,h, approximately 95% with pseudotrial aggregation; Extended Data Fig. 3a) and the fMRI searchlight approach (Fig. Decoding of face orientation was robust from MEG cortical time series in posterior ROIs (approximately 75% with pseudotrial aggregation), but was weaker, yet above chance (35%), in prefrontal ROIs (Fig. 2i), with a possibility of signal leakage from posterior regions (Extended Data Fig. Bayesian testing further validated these findings. For iEEG, Bayes factor (BF)01 values (5.11–8.65) supported the null hypothesis of no face orientation decoding in prefrontal regions. fMRI Bayesian analysis revealed substantial-to-very-strong support for the null hypothesis in 34–55% of prefrontal voxels (BF01: 3–71.5), with support for the alternative hypothesis in only 1–9% of voxels, whereas the rest remained inconclusive. Across all modalities, orientation decoding was observed for letters and false fonts—but not objects—in posterior, but not prefrontal, ROIs (see section 5.1.2 in Supplementary Information). Finally, we tested IIT's prediction that prefrontal regions do not contribute further information beyond that specified by posterior areas (or may even degrade performance by introducing noise)39. If PFC activity increased decoding accuracy, IIT would be challenged, whereas no improvement in decoding accuracy would align with both IIT and GNWT, as GNWT posits that PFC workspace neurons broadcast but do not add information. We compared the performance of decoders trained exclusively on posterior ROIs with those trained on posterior and prefrontal ROIs together (Extended Data Fig. Across critical time-resolved methods (iEEG and MEG) and various PFC ROI definitions, adding prefrontal ROIs did not improve—and in some cases reduced—category and orientation decoding (Extended Data Fig. 3d,e and see section 5.1.3 in Supplementary Information for non-critical fMRI results). Bayesian testing confirmed these findings: we found strong evidence against increased decoding accuracy when including PFC ROIs for category decoding (face–object: iEEG BF01 = 1.94 × 104 and MEG BF01 = 3.05; letter–false font: iEEG BF01 = 1.91 × 105 and MEG BF01 = 4.70) and face orientation (iEEG BF01 = 1,205 and MEG BF01 = 3.26). By contrast, GNWT predicts brief, content-specific ignition in PFC within 0.3–0.5 s after stimulus onset, as the workspace is updated12. Activity then decays back to baseline, with information maintained in a latent state, until another ignition marks the offset of the current percept and the onset of a new percept (for example, the fixation screen following stimulus offset). Thus, although the underlying brain responses (the workspace update) are temporally discrete (that is, an onset and an offset response), the conscious experience can be temporally continuous, spanning from one update to the next. Following the preregistration27 (Extended Data Table 1), IIT would be challenged if sustained content-specific information and activation tracking stimulus duration were absent in the posterior cortex. GNWT would be challenged if transient prefrontal activation (at stimulus onset and offset) was not observed. These patterns were expected for at least one conscious feature (category, identity or orientation). We assessed activation strength as a function of stimulus duration and the informational content of this activation within theory-relevant ROIs. For IIT, both activation and information content were critical predictions, jointly determining result interpretation. For GNWT, activation was the primary measure owing to the difficulty of reliably detecting content-specific reinstatement at stimulus offset. The temporal predictions were tested in time-resolved iEEG and MEG data. We focused on the task-irrelevant condition as it best isolates neural activity related to consciousness while minimizing confounds (see sections 6.1 and 6.2.9 in Supplementary Information for task-relevant results). First, we tested the predictions of the theories by investigating iEEG neural activation as a function of stimulus duration using linear mixed models (LMMs; see Methods) to model the time course of neural activity in the high gamma. In posterior cortex ROIs, 25 electrodes (out of 657) measured sustained activity tracking stimulus duration (Extended Data Table 3 for electrode localization and section 6.1.1 in Supplementary Information for results of the full model), consistent with IIT's (Fig. Of these, 12 electrodes tracked duration independent of stimulus category, primarily in early visual areas (for example, occipital pole; Fig. 3b), whereas 13 showed category-specific duration tracking (mostly for faces) in the ventral temporal cortex (for example, lateral fusiform gyrus; Fig. Overall, only a small proportion of electrodes exhibited both category selectivity and duration tracking—for example, just 15% (8 of 53) of face-selective electrodes showed sustained activity as predicted by IIT, suggesting a sparse neural substrate. These responses were mostly localized to the lateral fusiform gyrus, whereas the majority of face-selective electrodes displayed transient activations at stimulus onset across the striate, extrastriate and ventral areas (see section 6.1.2 in Supplementary Information). a, Intracranial electrode localization on the MNI template, for posterior (left; blue; npatients = 31 and nelectrodes = 657) and prefrontal (right; green; npatients = 31 and nelectrodes = 655) ROIs. Electrodes are colour coded by response type based on model comparison (see Methods): sustained non-category-selective activation (light blue; n = 12), sustained category-selective activation (dark blue; n = 5), sustained face-selective activation (purple; n = 8), biphasic onset–offset activation in posterior areas (green; n = 11) and in PFC (black; n = 1), and onset-responsive activation in PFC (grey; n = 99). Time-series plots depict average high gamma (HG), separated by stimulus duration (0.5 s (dark), 1 s (medium) and 1.5 s (light)) for non-selective (left) and face-selective (middle) electrodes. The shading here and in panel c depicts standard error of the mean across electrodes and trials. Raster plots show single-trial (n = 320) HG of individual electrodes during task-irrelevant trials: a sustained non-selective (left), sustained face-selective (middle) and onset–offset (right) electrode. The rows depict single trials, sorted per duration (from top: 0.5, 1.0 and 1.5 s), and then category (from top: false fonts, letters, objects and faces). Time-series plots (top left) depict the average HG response per stimulus duration (shades of grey) for onset-responsive electrodes (n = 99) in task-irrelevant trials (n = 320). Average HG response per stimulus duration for a single electrode exhibiting onset–offset responses, with an earlier-than-predicted offset (top right). Matrix values represent z-scored, within-class-corrected correlation distances derived from a label shuffle null distribution. Contours denote significance (cluster-based permutation tests, P < 0.05, upper tail). However, none of the 655 electrodes measured the temporal profile predicted by GNWT (that is, onset and offset). Bayesian analysis confirmed this result (BF01 > 3 for all electrodes in PFC ROIs), providing stronger evidence for either an intercept-only or a time-varying amplitude model over the GNWT model, with or without category interaction. This null result was not due to analysis limitations, as the LMM successfully detected the GNWT-predicted pattern in 10 electrodes in other ROIs (in the striate or extrastriate cortex; Fig. An exploratory decoding analysis of stimulus duration with unrestricted temporal profiles identified only one electrode, in the inferior frontal sulcus, showing the GNWT-predicted pattern, although with transient responses occurring earlier than expected (0.15 s post-onset and post-offset; Fig. Additional control analyses confirmed the IIT-predicted pattern in posterior ROIs and the absence of the GNWT-predicted pattern in PFC ROIs (see sections 6.2.1–6.2.3 in Supplementary Information). We used LMMs to track gamma frequency band (60–90 Hz) power changes from the MEG source time series across posterior (15 parcels) and PFC (11 parcels) ROIs. Although signals were strong in posterior areas, none of the theory-based models adequately fit the data (BF01 > 3 for all parcels; see section 6.1.3.1 in Supplementary Information). We also examined alpha band activity (8–13 Hz), which negatively correlates with neural spiking activity40,41. Validation of theoretical predictions from iEEG and MEG data was inconclusive: no prefrontal iEEG electrodes showed the GNWT-predicted combination of an onset and offset response (BF01 > 3 for all prefrontal electrodes); instead, this pattern appeared in posterior sites and in MEG data, including the anterior cingulate cortex. However, the MEG results were sensitive to parameter choices, and signal leakage from posterior sites could not be ruled out (see sections 6.1.1 and 6.1.3.2 in Supplementary Information). Next, we used cross-temporal representational similarity analysis (RSA) on both iEEG and MEG source data, within each theory-relevant ROI, to test the predictions of IIT and GNWT about the temporal profile of the maintenance of conscious content (Fig. This test was critical for IIT only. Results for faces and objects are presented below (see Extended Data Fig. In iEEG, we calculated the correlation distance between high gamma activity patterns across 583 electrodes in posterior (npatients = 28) and 576 electrodes in PFC (npatients = 28) ROIs separately (see Methods). In posterior cortex ROIs, cross-temporal RSA revealed sustained face–object categorical representation, with larger correlation distances between categories (face–objects) than within category (face, object; compare Fig. In PFC ROIs, cross-temporal RSA revealed transient face–object categorical representation at stimulus onset, but not at stimulus offset. Consequently, no significant correlation was found with the GNWT onset and offset model (compare Fig. 3d, right, with the predicted pattern in Fig. This pattern held even for the task-relevant condition, in which face–object information was stronger, more stable and longer lasting. Additional evidence for the absence of GNWT-predicted patterns in PFC ROIs emerged from three control analyses using (1) feature selection, which improved RSA in PFC; (2) modified time-windows to account for a potential earlier ignition at stimulus offset; and (3) a decoding analysis time-locked to stimulus offset to enhance sensitivity (see section 6.4 in Supplementary Information). These results align with two independent studies using comparable methods42,43. In posterior ROIs, iEEG revealed sustained object identity information, with smaller distances for same-identity objects than for different identities (Fig. Similar results were found for letter and false-font identity, but not for faces (Extended Data Fig. In PFC ROIs, identity information was absent for all categories across analysed time windows (Fig. Face orientation information appeared weakly in posterior ROIs at stimulus onset but was not sustained, decaying after 0.5 s, contrary to the predictions of IIT. No face orientation information was detected in PFC ROIs (Fig. Finally, the predictions of neither theory were supported for category, identity or orientation by the MEG data (see section 6.5 for Supplementary Information). IIT predicts sustained gamma-band connectivity within the posterior cortex, that is, between high-level and low-level sensory areas (V1/V2), throughout any conscious visual experience. By contrast, GNWT predicts brief, late-phase metastable connectivity (more than 0.25 s) with information sharing between the PFC and category-specific areas, manifested in long-range gamma-band or beta-band synchronization45. On the basis of our preregistration (see Extended Data Table 1), IIT would be challenged in the absence of sustained content-specific synchronization between face–object selective areas and V1/V2; whereas a challenge for GNWT would be a lack of phasic connectivity (0.3–0.5 s) between category-selective areas and PFC. Given the dynamical nature of these predictions, iEEG and MEG provided the most informative empirical test. These predictions were tested by computing pairwise phase consistency (PPC)46 between each category-selective time series (face-selective and object-selective nodes) and either the V1/V2 or the PFC time series in the intermediate (1.0 s) and long-stimulus-duration (1.5 s), task-irrelevant trials (see section 7.1.2 in Supplementary Information for task-relevant trials). For iEEG, we analysed connectivity between electrodes showing face and object selectivity, using a different subset of electrodes to test connectivity with V1/V2 and PFC (see Methods; Extended Data Fig. Given the sparse electrode coverage, the requirement to focus on ‘activated' electrodes (see Methods) was relaxed, although restricting it to only activated electrodes yielded similar results. These effects were early and brief (for example, less than 0.75 s), and restricted to low frequencies (2–25 Hz). This synchrony was mostly explained by the stimulus-evoked response (Extended Data Fig. These results fail to align with IIT's predictions: the activity was neither sustained nor observed in the gamma frequency band. Bayesian analysis further supported the null hypothesis (BF01 = 1.15–4.9). No content-selective PPC was found between face-selective and object-selective electrodes and PFC electrodes in the relevant time window, contradicting the prediction of GNWT (Extended Data Fig. These effects were again early, restricted to low frequencies (2–25 Hz), and mostly explained by stimulus-evoked responses (Extended Data Figs. PPC was chosen based on the mechanistic considerations of the theories because it assesses oscillatory phase. However, phase estimation from macroscopic recordings is susceptible to noise. We thus used dynamic functional connectivity (DFC; see Methods), a metric sensitive to co-modulations of signal amplitude, after removing stimulus-evoked responses (Extended Data Fig. In iEEG, we observed significant connectivity between object-selective electrodes and V1/V2 (Fig. Connectivity between face-selective electrodes and V1/V2 was scarce, further supported by Bayesian analysis (BF01 = 1.3 in favour of the null hypothesis). For MEG, brief DFC in the alpha–beta frequency bands was found between face-selective nodes and both PFC and V1/V2 (Fig. a, iEEG DFC analysis of task-irrelevant trials revealed significant content-selective synchrony only for object-selective electrodes in V1/V2 (for example, top row; face-selective: npatients = 4 and nelectrodes = 30; object-selective: npatients= 4 and nelectrodes= 21), while showing significant content-selective synchrony for both categories in the PFC ROI (bottom row; face selective: npatients = 19 and nelectrodes = 81; object selective: npatients = 14 and nelectrodes = 57). Here and in panel b, significance was assessed using a cluster-based permutation analysis (P < 0.05, two-sided) and the colour bars represent the average change in the DFC between conditions. b, MEG DFC analysis of task-irrelevant trials (n = 65) revealed significant content-selective synchrony below 25 Hz for the face-selective GED filter in both V1/V2 (top left) and PFC (bottom left), but not for the object-selective GED filter (right panels). c, fMRI generalized psychophysiological interaction (gPPI; n = 70) on task-relevant and task-irrelevant trials combined revealed significant content-selective connectivity when FFA is used as the analysis seed. Various significant regions showing task-related connectivity with the FFA seed were observed including V1/V2, right intraparietal sulcus (IPS) and right inferior frontal gyrus (IFG). d, Analysis of iEEG face-selective DFC synchrony across tasks is shown at the single-electrode level in PFC (top) and V1/V2 (bottom) ROIs. Electrodes showing significant synchrony (tested using a permutation test, FDR-corrected, P < 0.05) in relevant (orange-red), irrelevant (purple) or combined relevant and irrelevant (black) trials are shown (averaged over 70–120 Hz and 0–0.5-s time window). DFC synchrony was observed in both tasks, but restricted to IFG for the GNWT analysis and V2 regions for the IIT analysis, consistent with the fMRI gPPI analysis shown in panel c. Brain surfaces in panels c,d are from Freesurfer. The exploratory DFC results in iEEG were consistent with the predictions of GNWT while challenging the predictions of IIT, as connectivity with V1/V2 was not sustained. However, V1/V2 were sparsely sampled in our population, with only 12 electrodes localized to V1/V2 compared with 472 in PFC. Using fMRI, we evaluated connectivity across the entire cortex with homogeneous sampling. We computed the generalized psychophysiological interaction, defining the fusiform face area (FFA) and lateral occipital complex as seed regions (see Methods). Task-relevant and task-irrelevant trials were pooled to increase statistical power (see sections 7.1.1 and 12 in Supplementary Information for separate preregistered analyses). FFA showed content selective (face > object stimuli) connectivity with V1/V2, inferior frontal gyrus and intraparietal sulcus, consistent with predictions from both theories (Fig. No selective increase in interareal connectivity between object-selective nodes and PFC or V1/V2 was found in fMRI, even when separating task conditions (Extended Data Fig. Bayesian testing across prefrontal ROIs confirmed our findings with 62–94% of voxels across ROIs showing substantial evidence for the null hypothesis of no interareal connectivity (BF01 = 3–7.75). Support for the alternative hypothesis was observed only in 0–4% of voxels. The remaining voxels showed inconclusive evidence. To determine whether connectivity to PFC and V1/V2 was task driven in the generalized psychophysiological interaction, we explored the iEEG data, separating task-relevant and irrelevant trials. We found task-independent, selective DFC connectivity (face > objects) for face-selective electrodes in both inferior frontal gyrus and V1/V2 (Fig. This adversarial collaboration aimed to address confirmation biases by researchers, breaking theoretical echo chambers3 and identifying the strengths and weaknesses of theories2,48 by forcing them to be explicit and committal about their empirical predictions, rigorously testing them on common methodological grounds22,49, and providing the means for theorists to change their minds given conflicting results49. Doing so catalyses our ability to evaluate and arbitrate between theories of consciousness. Embracing this spirit, and adhering to guidelines for structuring adversarial collaborations21, we opted for a three-voice discussion format, acknowledging that despite stringent testing of incompatible theoretical views, different interpretations of the same evidence may persist. Then, adversaries offer their own interpretation of the findings and future directions (see sections 12 and 13 in Supplementary Information). This summary covers both central and peripheral findings related to theory evaluation. The consortium adopted Lakatos' sophisticated falsificationist approach to philosophy of science13,50, emphasizing that challenged predictions provide more valuable insights than those confirmed by the data (see section 11 in Supplementary Information). Outcomes are weighted differentially across predictions and with respect to the different brain imaging modalities (Extended Data Table 1). This approach ensures a nuanced evaluation of the theories, highlighting areas of strength and those requiring further refinement. For IIT, the lack of sustained synchronization within posterior cortex represents the most direct challenge, based on our preregistration. This is incompatible with IIT's claim that the state of the neural network, including its activity and connectivity, specifies the degree and content of consciousness5. Although this null result could stem from methodological limitations (for example, limited iEEG sampling of V1/V2 areas), our multimodal and highly powered study provided the best conditions so far for evaluating the prediction. We urge IIT proponents to direct future efforts to evaluate this prediction and determine the implications of this failure. More broadly, although IIT passed the predefined criteria for the duration prediction (number 2), there was no evidence for a sustained representation of orientation, despite being a property of the consciously perceived stimuli25. This is an informative challenge for IIT, as orientation decoding was robust across all three data modalities, leaving open the question of whether and how information about orientation is maintained over time. For GNWT, the most substantial challenge based on our preregistered criteria pertains to its account for the maintenance of a conscious percept over time and, in particular, the lack of ignition at stimulus offset. This result is unlikely to stem from sensitivity limitations, as offset responses were robustly found elsewhere (for example, visual areas); and in PFC, strong onset responses were found to the very same stimuli. This clear update to the content of consciousness should have been represented somehow by the global workspace12. Thus, that aspect of consciousness remains unexplained within the GNWT framework. This raises the question of whether PFC is involved in broadcasting all conscious content, as predicted by GNWT23, or only a subset (for example, abstract concepts and categories, rather than low-level details), in which case the role of PFC in consciousness might need to be redefined. Before this study, predictions from IIT and GNWT were typically tested with one data modality at a time23,24, leaving room for negative results to be easily attributed to the limitations of the chosen modality51. We combined multiple techniques (iEEG, MEG and fMRI) to mitigate these limitations, cross-compensating for their weaknesses. This methodological approach was mutually agreed upon by the theory leaders before data collection and results disclosure as the most powerful and conclusive approach, making both positive and negative findings more meaningful. Although this study was designed around IIT and GNWT, the results may have implications for other theories of consciousness. For example, the prediction of GNWT about PFC is shared by those higher-order theories of consciousness that hypothesize that PFC actually supplies the content of visual consciousness (for example, ref. 17), rather than those that take it to merely enable the consciousness of content that is located in posterior visual areas (for example, refs. As a result, the failures to confirm this prediction challenge not only GNWT but also those higher-order theories54. Predictions 2 and 3, about timing and connectivity, are more distinctive to GNWT but could also be shared by other theories. Likewise, the non-core prediction 1 about the posterior cortex by IIT is also shared by many theories (for example, recurrent processing theory14), and its prediction 2 about timing may be shared by some posterior theories of consciousness, such as the local recurrency theory15. Its prediction 3 about interareal connectivity is more distinctive to IIT (for example, it is not shared by synchrony theory55), so the challenge here is more specific as well. Our study focused on the contents of consciousness (for example, category, identity, orientation and duration), linking brain activity to subjective phenomenology. This departs from the traditional contrastive method, which compares the presence and absence of consciousness but conflates it with other cognitive processes (for example, decision-making or memory formation)56,57,58. Some might argue that our approach tracks stimulus processing rather than consciousness. Yet, our aim is to challenge and potentially falsify50,59 IIT and GNWT, by examining where their predictions differ, rather than to discover the neural correlates of consciousness. In this context, what might seem like a weakness—focusing on the presence of fully attended, consciously experienced stimuli—is actually beneficial for testing the primary positive predictions of the theories and their failures. Thus, our approach assesses whether the proposed neural mechanisms are truly necessary for consciousness. Our study, although comprehensive, is not without limitations. First, we cannot entirely rule out residual task engagement with respect to category, although our design ensured that orientation and duration remained task-irrelevant, so the results on these dimensions cannot be explained by task-related effects. Second, although we aimed to capture multiple aspects of consciousness, our approach still falls short of encompassing its full phenomenal richness. Third, despite the high spatial and temporal resolution of our data, it lacks single-unit recordings, which are typically restricted to patients with epilepsy and selected brain regions. Ongoing studies in animal models, as part of a separate adversarial collaboration, can accordingly complement our findings. Beyond directly challenging the theories, our study raises broader questions about theory testing and development across disciplines. We adopted a lenient falsificationist approach, in which evidence for any tested feature (for example, decoding of category or orientation) was sufficient to rule out failure, rather than requiring consistency across all features. However, a formal framework is urgently needed to quantitatively integrate evidence, accounting for prediction centrality, measurement error and cross-sample consistency. Such a framework would enhance systematic theory building in an era of accumulating results60. Scientific progress is rarely a matter of simple verdicts; evidence is filtered through previous beliefs and motivations61, making theory evaluation a dynamic process. By presenting results and adversarial responses transparently, we embrace the openness needed for science to converge on robust explanations of complex phenomena such as consciousness. The experiment was approved by the institutional ethics committees of each participating data-collecting laboratory, including the Science, Technology, Engineering and Mathematics Ethical Review Committee at the Centre for Human Brain Research, University of Birmingham (ERN_18-0226AP20); the Committee for Protecting Human and Animal Subjects at the School of Psychological and Cognitive Sciences, Peking University (2020-05-07e); the Commissie Mensgebonden Onderzoek Regio Arnhem-Nijmegen at the Centre for Cognitive Neuroimaging at Donders Institute (NL45659.091.14); the Human Research Protection Program Institutional Review Board at Yale School of Medicine (2000027591); the Office of Science and Research Institutional Review Board at New York University Langone Health (i14-02101_CR6); the Boston Children's Hospital Institutional Review Board at Children's Hospital Corporation d/b/a Boston Children's Hospital (04-05-065R); the Institutional Review Board at the University of Wisconsin-Madison (ID: 2017-1299); and the Ethics Council of the Max Planck Society at Max Planck Institute for Empirical Aesthetics (Nr. All participants and patients provided oral and written informed consent before participating in the study. All study procedures were carried out in accordance with the Declaration of Helsinki. Patients were also informed that clinical care was not affected by participation in the study. Healthy participants and patients with pharmaco-resistant focal epilepsy participated in this study. The datasets reported here consist of: (1) behaviour, eye tracking and iEEG data collected at the Comprehensive Epilepsy Center at New York University (NYU) Langone Health, the Brigham and Women's Hospital, the Boston Children's Hospital (Harvard), and the University of Wisconsin School of Medicine and Public Health (WU). (2) Behaviour, eye tracking, MEG and EEG data collected at the Centre for Human Brain Health (CHBH) of the University of Birmingham (UB), and at the Center for MRI Research of Peking University (PKU). (3) Behaviour, eye tracking and fMRI data collected at the Yale Magnetic Resonance Research Center (MRRC) and at the Donders Centre for Cognitive Neuroimaging (DCCN), of Radboud University Nijmegen. A total of 97 healthy participants were included in the MEG sample (mean age of 22.79 ± 3.59 years, 54 females, all right handed), 32 of those datasets were included in the optimization phase (mean age of 22.50 ± 3.43 years, 19 females, all right handed), and 65 in the replication sample (mean age of 22.93 ± 3.66, 35 females, all right handed). Five additional participants were excluded from the MEG dataset: two because of failure to meet predefined behavioural criteria (that is, Hits of less than 80% and/or False Alarms > 20%), two because of excessive noise from sensors, and one because of incorrect sensor reconstruction. A total of 108 healthy participants were included in the fMRI sample (mean age of 23.28 ± 3.46 years, 70 females, 105 right handed); 35 of those datasets were included in the optimization sample (mean age of 23.26  ±  3.64 years, 21 females, 34 right handed) and 73 in the replication sample (mean age of 23.29 ± 3.37, 49 females, 71 right handed). Twelve additional participants were excluded from the fMRI dataset: eight because of motion artefacts, two because of insufficient coverage and two because of incomplete data (with respect to these last two participants, see section 14 of the Supplementary Information for deviations from the preregistration document). Two patients were excluded owing to incomplete data. Demographic, medical and neuropsychological scores for each patient, when available, are reported in Supplementary Table 25. Three iEEG patients whose behaviour fell slightly short of the predefined behavioural criteria (that is Hits of less than 70%, FA > 30%) were nonetheless included given the difficulty of obtaining additional iEEG data (see section 14 in Supplementary Information for deviation from the preregistration). To test critical predictions of the theories, five experimental manipulations were included in the experimental design: (1) four stimulus categories (faces, objects, letters and false fonts), (2) 20 stimulus identities (20 different exemplars per stimulus category), (3) three stimulus orientations (front, left and right view), (4) three stimulus durations (0.5 s, 1.0 s and 1.5 s), and (5) task relevance (relevant targets, relevant non-targets and irrelevant). Stimulus category, stimulus identity and stimulus orientation served to test predictions about the representation of the content of consciousness in different brain areas by the theories. In addition, stimulus duration served to test predictions about the temporal dynamics of sustained conscious percepts and interareal synchronization between areas. Task relevance served to rule out the effect of task demands, as opposed to conscious perception per se, on the observed effects62. This aspect of the experimental design was inspired by ref. Four stimulus categories were used: faces, objects, letters and false fonts. These natural couplings were aimed at creating a clear difference between task-relevant and task-irrelevant stimuli in each trial block (see the section ‘Procedure'). All stimuli covered a squared aperture at an average visual angle of 6° by 6°. Face stimuli were created with FaceGen Modeler 3.1; letter and false font stimuli were generated with MAXON CINEMA 4D Studio (RC - R20) 20.059; object stimuli were taken from the Object Databank64. To facilitate face individuation, faces had different hairstyles and belonged to different ethnicities and genders. Equal proportions of male and female faces were presented. The orientation of the stimuli was manipulated, such that half of the stimuli from each category had a side view (30° and −30° horizontal viewing angle, left and right orientation) and the other half had a front view (0°). Participants performed a non-speeded target detection task (see Supplementary Video 1). The experiment was divided into runs, with four blocks in each run (see the section ‘Trial counts'). On a given block, participants viewed a sequence of single, supra-threshold, foveally presented stimuli belonging to one of four stimulus categories and presented for one of three stimulus durations onto a fixation cross that was present throughout the experiment. To manipulate task relevance, at the beginning of each block participants were instructed to detect the rare occurrences of two target stimulus identities, one from each relevant category (for pictures, face–object; for symbols, letter–false font), irrespective of their orientation. This was specified by presenting the instruction ‘detect face A and object B' or ‘detect letter C and false font D', accompanied by images for each target (see Fig. Targets did not repeat across blocks. Each run contained two blocks of the face–object task and two blocks of the letter–false font task, with block order counterbalanced across runs. Accordingly, each block contained three different trial types: (1) targets: the two stimuli being detected (for example, the specific face and object identities); (2) task-relevant stimuli: all other stimuli from the task-relevant categories (for example, the non-target faces–objects); and (3) task-irrelevant stimuli: all stimuli from the two other categories (for example, letters–false fonts). An advantage of this design is that the three trial types enabled a differentiation of neural responses related to task goal, task relevance and simply consciously seeing a stimulus. We confirmed that participants were conscious of the stimuli in both the task-relevant and task-irrelevant trials in a separate experiment, which included a surprise memory test (see section 3 in Supplementary Information). Stimuli were presented for one of three durations (0.5 s, 1.0 s or 1.5 s), followed by a blank period of a variable duration to complete an overall trial length fixed at 2.0 s. For the MEG and iEEG version, random jitter was added at the end of each trial (mean inter-trial interval of 0.4 s, jittered 0.2–2.0 s, truncated exponential distribution) to avoid periodic presentation of the stimuli. The mean trial length was 2.4 s. For the fMRI protocol, timing was adjusted as follows: the random jitter between trials was increased (mean inter-trial interval of 3 s, jittered 2.5–10 s, with truncated exponential distribution), with each trial lasting approximately 5.5 s. This modification helped with avoiding non-linearities in BOLD signal, which may affect fMRI decoding65. Second, to increase detection efficacy for amplitude-based analyses, three additional baseline periods (blank screen) of 12 s each were included per run (total of 24). Participants were further instructed to maintain central fixation on a black circle with a white cross and another black circle in the middle throughout each trial (see Supplementary Fig. 1d and Supplementary Video 1 for a demonstration of the experimental paradigm). The MEG study consisted of 10 runs containing 4 blocks each with 34–38 trials per block, 32 non-targets (8 per category) and 2–6 targets, for a total of 1,440 trials. The same design was used for iEEG, but with half the runs (5 runs total), resulting in a total of 720 trials. For fMRI, there were 8 runs containing 4 blocks each with 17–19 trials per block, 16 non-targets (4 per category) and 1–3 targets, for a total of 576 trials. Rest breaks between runs and blocks were included. The task was run on Matlab (PKU: R2018b; DCCN, UB and Yale: R2019b; Harvard: R2020b; NYU: R2020a, and WU: 2021a) using Psychtoolbox (v3)66. The iEEG version of the task was run on a Dell Precision 5540 laptop, with a 15.6′′ Ultrasharp screen at NYU and Harvard and on a Dell D29M PC with an Acer 19.1′′ screen in WU. Participants responded using an eight-button response box (Millikey LH-8; response hand (or hands) varied based on the setting in the patient's room). The MEG version was run on a custom PC at UB and a Dell XPS desktop PC on PKU. Stimuli were displayed on a screen placed in front of the participants with a PROPixx DLP LED projector (VPixx Technologies). Participants responded with both hands using two 5-button response boxes (NAtA or SINORAD). The fMRI version was run on an MSI laptop at Yale and a Dell Desktop PC at DCCN. In DCCN, stimuli were presented on an MRI compatible Cambridge Research Systems BOLD screen 32′′ IPS LCD monitor, and in Yale they were presented on a Psychology Software Tools Hyperion projection system to project stimuli on the mirror fixed to the head coil. Participants responded with their right hand using a 2 × 2 current designs response box at Yale and a 1 × 4 current designs response box at DCCN. For the iEEG setup, eye tracking and pupillometry data were collected using a EyeLink 1000 Plus in remote mode, sampled monocularly at 500 Hz (from the left eye at WU, and depending on the setup at Harvard), or on a Tobii-4C eye tracker, sampled binocularly at 90 Hz (NYU). The MEG and fMRI laboratories used the MEG-compatible and fMRI-compatible EyeLink 1000 Plus Eye-tracker system (SR Research) to collect data at 1,000 Hz. For MEG, eye tracking data were acquired binocularly. For fMRI, data were acquired monocularly from either the left or the right eye, in DCCN and Yale, respectively. For all recordings, a 9-point calibration was performed (besides Harvard, where a 13-point calibration was used) at the beginning of the experiment, and recalibration was carried out as needed at the beginning of each block or run. Brain activity was recorded with a combination of intracranial subdural platinum-iridium electrodes embedded in SILASTIC sheets (2.3-mm diameter contacts, Ad-Tech Medical Instrument and PMT Corporation) and/or depth stereo-electroencephalographic platinum-iridium electrodes (PMT Corporation; 0.8 mm in diameter, 2.0-mm length cylinders; separated from adjacent contacts by 1.5–2.43 mm), or depth stereo-electroencephalographic platinum-iridium electrodes (BF08R-SP21X-0C2, Ad-Tech Medical; 1.28 mm in diameter, 1.57 mm in length, 3–5.5-mm spacing). Electrodes were arranged as grid arrays (either 8 × 8 with 10-mm centre-to-centre spacing, 8 × 16 contacts with 3-mm spacing, or hybrid macro–micro 8 × 8 contacts with 10-mm spacing and 64 integrated microcontacts with 5-mm spacing), linear strips (1 × 8/12 contacts), depth electrodes (1 × 8/12 contacts) or a combination thereof. Recordings from grid, strip and depth electrode arrays were done using a Natus Quantum amplifier or a Neuralynx Atlas amplifier. A total of 4,057 electrodes (892 grids, 346 strips and 2,819 depths) were implanted across 32 patients with drug-resistant focal epilepsy undergoing clinically motivated invasive monitoring. A total of 3,512 electrodes (780 grids, 307 strips and 2,425 depths) that were unaffected by epileptic activity, artefacts or electrical noise were used in subsequent analyses. To determine the electrode localization for each patient, a post-operative computed tomography scan and a pre-operative T1 MRI were acquired and co-registered. MEG was acquired using a 306-sensor TRIUX MEGIN system, comprising 204 planar gradiometres and 102 magnetometres in a helmet-shaped array. The MEG gantry was positioned at 68° for optimal coverage of frontal and posterior brain areas. Simultaneous EEG was recorded using an integrated EEG system and a 64-channel electrode cap (EEG data are not reported here, but are included in the shared dataset). ECG was recorded with a set of bipolar electrodes placed on the chest of the participant. Two sets of bipolar electrodes were placed around the eyes (two at the outer canthi of the right and left eyes and two above and below the centre of the right eye) to record eye movements and blinks (EOG). The head position of participants on the MEG system was measured at the beginning and end of each run, and also before and after each resting period, using four HPI coils placed on the EEG cap, next to the left and right mastoids and over left and right frontal areas. For source localization of the MEG data with individual realistic head modelling, a high-resolution T1-weighted MRI volume (3 T Siemens MRI Prisma scanner) was acquired per participant. Anatomical scans were acquired either with a 32-channel coil (repetition time (TR)/echo time (TE) = 2,000/2.03 ms; inversion time (TI) = 880 ms; 8° flip angle; field of view = 256 × 256 × 208 mm; 208 slices; 1-mm isotropic voxels, UB) or a 64-channel coil (TR/TE = 2,530/2.98 ms; TI = 1,100 ms; 7° flip angle; field of view = 224 × 256 × 192 mm, 192 slice, 0.5 × 0.5 × 1 mm voxels, PKU). The FreeSurfer standard template was used (fsaverage) for participants lacking an anatomical scan (n = 5). A session included high-resolution anatomical T1-weighted MPRAGE images (GRAPPA acceleration factor = 2, TR/TE = 2,300/3.03 ms, 8° flip angle, 192 slices, 1-mm isotropic voxels), and a whole-brain T2*-weighted multiband-4 sequence (TR/TE = 1,500/39.6 ms, 75° flip angle, 68 slices, voxel size of 2 mm isotropic, anterior/posterior (A/P) phase-encoding direction, field of view = 210 mm, bandwith (BW) = 2,090 Hz px−1). A single-band reference image was acquired before each run. To correct for susceptibility distortions, additional scans using the same T2*-weighted sequence, but with inverted phase-encoding direction (inverted readout/phase-encoding (RO/PE) polarity), were collected while the participant was resting at multiple points throughout the experiment. For readability, we first detail the preprocessing protocols for each of the modalities (iEEG, MEG and fMRI) separately. Then, we describe the different analyses, combining information across the modalities, while noting any differences between them. As part of our testing framework, after excluding a limited number of participants due to data quality checks, we conducted an initial optimization phase on one-third of the MEG (n = 32) and fMRI (n = 35) datasets to evaluate data quality across sites and to optimize analysis pipelines. Following the optimization phase, pipelines were preregistered27 and applied to the novel datasets containing twice as much data (MEG n = 65 and fMRI n = 73). For iEEG, given the smaller sample, a different analysis strategy was implemented. We refer the reader to the iEEG methods section and text in the main paper for numbers of participants that were entered in each analysis. Results from the optimization phase are reported in section 4 of Supplementary Information. The results of the optimization phase and the preregistered replication phase were compared and deemed to be largely compatible, with some minor exceptions (section 4 of Supplementary Information). Preprocessing steps included downsampling to 512 Hz, detrending, bad channel rejection, line noise and harmonic removal, and re-referencing. Electrodes were re-referenced to a Laplacian scheme69, whereas bipolar referencing was used for electrodes at the edge of a strip, grid or stereo EEG, and the signal was localized at the midpoint (Euclidean distance) between the two electrodes. Electrodes with no direct neighbours were discarded. Seizure-onset zone electrodes, those localized outside the brain and/or containing no signal or high amplitude noise level were discarded. Line noise and harmonics were removed using a one-pass, zero-phase non-causal band-stop FIR filter. To produce a single high-gamma envelope time series, all frequency bands were averaged together70. Most analyses focused on the high-gamma power as it closely correlated with neural spiking activity71 and with the BOLD signal37. To obtain the event-related potentials (ERPs), the raw signal was low-pass-filtered at 30 Hz with a one-pass, zero-phase, non-causal low-pass FIR filter. Epochs were segmented between 1-s pre-stimulus until 2.5-s post-stimulus of interest. Electrode positions were determined based on a computed tomography scan coregistered with a pre-implant T1-weighted MRI. For visualization, the electrode positions for individual participants were converted to the Montreal Neurological Institute (MNI)152 space. As each theory specified a set of anatomical ROIs, after electrode localization, electrodes were labelled according to the Freesurfer-based Destrieux atlas segmentation72,73 and/or Wang atlas segmentation74. To identify task-responsive electrodes, we computed the area under the curve (AUC) for the baseline (−0.3 to 0 s) and the stimulus-evoked period (0.05–0.35 s) separately for the task-relevant and task-irrelevant conditions, and compared them per electrode using a Wilcoxon sign-rank test, corrected for false discovery rate (FDR)75. A Bayesian t-test76 was used to quantify evidence for non-responsiveness. To determine category selectivity for faces, objects, letters and false fonts in the high gamma, we followed the method of Kadipasaoglu and colleagues77. Per category, we computed a d′ (AUC of 0.05–0.4 s) comparing the activation between the category of interest (uj) and each of the other categories (ui), normalized by the standard deviation of each category: A permutation test (10,000 permutations) was used to evaluate significance. d′ was computed for the task-relevant and task-irrelevant conditions separately. An electrode was considered selective if it showed selectivity on both tasks. Owing to the sparse and highly variable coverage of iEEG data, all collected electrodes were combined into a ‘super participant' multivariate analyses (RSA and decoding). Participants that did not complete the full experiment were discarded (n = 3), resulting in a total of 29 participants with 583 electrodes in posterior ROIs and 576 electrodes in prefrontal ROIs. For analyses on stimuli identities, stimuli that were presented less than three times to any of the participants across intermediate and long trials in the task-relevant and task-irrelevant trials were discarded. The subsampling procedure was repeated 100 times to avoid random fluctuations induced by the subsampling. The analysis was computed for each repetition and average across repetitions. Preprocessing steps included MEG sensor reconstruction using a semi-automatic detection algorithm and signal-space separation81 to reduce environmental artefacts. FastICA82 was used to detect and remove cardiac and ocular components from the data for each participant (mean = 2.90 components, s.d. = 0.92). After preprocessing, data were epoched into 3.5-s segments (1-s pre-stimulus to 2.5-s post-stimulus onset). Trials in which gradiometre values exceeded 5,000 fT cm−1, magnetometres exceeded 5,000 fT and/or the trial contained muscle artefacts were rejected from the MEG dataset. Finally, to be included in the analyses, participants should have a minimum of 30 clean trials per condition. No participants were excluded because of not meeting this criterion. MEG source modelling was performed using the dynamic statistical parametric mapping method83, based on depth-weighted minimum-norm estimates (MNEs)84,85, on epoched and baseline (−0.5 s to 0 s before stimulus onset) corrected data. To build a forward model, the MRI images were manually aligned to the digitized head shape. A single shell boundary elements model was constructed in MNE-Python based on the inner skull surface derived from FreeSurfer72,73, to create a volumetric forwards model (5-mm grid) covering the full-brain volume. A noise covariance matrix for the baseline and a covariance matrix for the active time window were calculated and the combined (that is, sum) covariance matrix was used with the forwards model to create a common spatial filter. Data were spatially pre-whitened using the covariance matrix from the baseline interval to combine gradiometre and magnetometre data86. Source DICOM data were converted to BIDS using BIDScoin (v3.6.3)87. This includes converting DICOM data to NIfTI using dcm2niix88 and creating event files using custom Python codes. BIDS compliance of the resulting dataset was controlled using BIDS-Validator. Subsequently, MRI data quality control was performed using MRIQC (0.16.1)89 and custom scripts for data rejection. All (f)MRI data were preprocessed using fMRIPrep (20.2.3)90, based on Nipype (1.6.1)91. For further details on the fMRIprep pipeline, see preregistration. Custom scripts used NumPy (1.19.2)92 and Pandas (1.1.3)93. Additional, analysis-specific, fMRI data preprocessing was performed using FSL 6.0.2 (FMRIB Software Library)94, Statistical Parametric Mapping (SPM 12) software95, and custom Python scripts (using NiBabel (3.2.2)96 and SciPy (1.8.0)97 after the above-outlined general preprocessing. Functional data for univariate data analyses were spatially smoothed (Gaussian kernel with full-width at half-maximum of 5 mm), grand mean scaled and temporal high-pass filtered (128 s). No spatial smoothing was applied for multivariate analyses. We modelled BOLD signal responses to the experimental variables by fitting voxel-wise general linear model (GLM) to the data of each run using FSL FEAT. The following regressors were modelled in an event-related approach, with event duration corresponding to the stimulus duration (that is, 0.5 s, 1.0 s and 1.5 s), and convolved with a double gamma haemodynamic response function: 12 regressors of interest (targets, task-relevant and task-irrelevant stimuli per stimulus category, that is, faces, objects, letters and false fonts; and a regressors of no interest, that is, target screen display). We included the first-order temporal derivatives of the regressors of interest, and a set of nuisance regressors: 24 motion regressors (FMRIB Software Library (FSL)'s standard + extended set of motion parameters) plus a cerebrospinal fluid (CSF) and a white matter (WM) tissue regressor. Each of the 12 regressors of interest was contrasted against an implicit baseline (used in the putative Neural Correlates of Consciousness analysis; see below). In addition, we obtained contrast of parameter estimates for ‘relevant faces versus relevant objects', ‘relevant letters versus relevant false fonts', ‘irrelevant faces versus irrelevant objects', ‘irrelevant letters versus irrelevant false fonts' (used for the definition of decoding ROIs), ‘relevant and irrelevant faces versus relevant and irrelevant objects' and ‘all stimuli versus baseline' (used for the definition of seeds for the generalized psychophysiological interaction (gPPI) analysis). Data were averaged across runs per participant using FSL's fixed-effects analysis and subsequently averaged across participants using FSL's FLAME1 mixed-effect analysis. Gaussian random-field cluster thresholding was used to correct for multiple comparisons, using the default settings of FSL, with a cluster formation threshold of one-sided P < 0.001 (z ≥ 3.1) and a cluster significance threshold of P < 0.05. ROIs were defined a priori in consultation with the adversarial theories. They were determined per participant based on the Destrieux atlas73 including both hemispheres, and then resampled to standard MNI space (see Supplementary Table 26). For the connectivity analysis, areas V1/V2 (combining dorsal and ventral) were defined based on the Wang cortical parcellation74. All anatomical segmentations were performed using Freesurfer (6.0.1)72. These measures were compared with linear–logistic mixed models, where appropriate. For the former, we report analysis of variance omnibus F-tests, and for the latter, omnibus χ2 test from an analysis of deviance. We approximated degrees of freedom using the Satterthwaite method99. Pairwise t-tests following significant interactions were Bonferroni corrected. To estimate Bayesian information criterion (BIC) differences between the original and null logistic models, we used the P values and sample size100 (p_to_bf package in R). For Eyelink, gaze and pupil data were segmented, and trials with missing data were excluded. Blinks were detected using the Hershman algorithm101, and removed with 200-ms padding102. The Eyelink standard parser algorithm was used for saccade and fixation detection. Saccades were further corroborated using the Engbert and Kliegl103 algorithm. Mean fixation distance, mean blink rate, mean saccade amplitude and mean pupil size were compared in a LMM with category and task relevance as fixed effects, and participant and item as random effects. Of note, the coordinate data were not added to the LMMs due to its poorer quality with respect to the EyeLink data. All decoding analyses were performed using a linear support vector machine (SVM; scikit learn (0.23.2), https://scikit-learn.org/) classifier. Below, we explain how this was done for each one of the predictions. iEEG decoding was done on the high-gamma signal, averaged over non-overlapping windows of 0.02 s separately for electrodes located in the GNWT and IIT ROIs. The top 200 electrodes (selectKbest104), as determined by a F-test within a given set of electrodes from the theory ROIs, were used as features for the classifier. Two-hundred features were selected to provide a balance between model optimization (for example, feature selection) and participant representation (for example, electrodes or features coming from multiple participants). Statistical significance of decoding performance was assessed via permutation test, randomly permuting the sample labels and repeating the decoding analysis 1,000 times, corrected for multiple comparisons using a cluster-based correction (cluster mass inference with cluster forming threshold at P < 0.05)105,106. Also, to assess the decoding accuracy within unique ROIs (for example, S_temporal_sup of the Destrieux atlas), separate classifiers were trained using all electrodes in a given parcel. Each classifier was fitted using all electrodes in a parcel and time window (GNWT: 0.3–0.5 s, IIT: 0.3–1.5 s) as features, resulting in a single accuracy value per parcel. SelectKbest (200 features for iEEG) feature selection and fivefold cross-validation with three repetitions was used. To assess the statistical significance of the decoding accuracy within unique ROIs (so only one accuracy score is obtained per ROI), P values obtained via permutation tests were corrected for multiple comparisons across all ROIs using FDR correction (q ≤ 0.0575). To compute Bayes factors on the decoding accuracy values, we used a β-binomial approach that compares the marginal likelihood under a point-null hypothesis against a flat \(B(\alpha =1,\,\beta =1)\) alternative prior, yielding an analytic Bayes factor. We then derived the null hypothesis parameters from the empirical null distribution by updating a tight prior centred at chance level (\(B(\alpha =1,000,\,\beta =1,000)\)) with the shuffle-based accuracies, thereby incorporating any bias present in the null distribution. We applied temporal smoothing (0.05-s window, 0.01-s sliding window), computed pseudotrials107, normalized the data and selected the top 30 features within a given ROI as features for the different classifiers. A group-level one-sample t-test per time point was performed on the decoding accuracy results, corrected for multiple comparisons using a cluster-based correction106. The overall decoding strategy for fMRI was similar to that used on the iEEG and MEG data, yet with some differences. A multivariate pattern analysis approach was used on the pattern of BOLD activity over voxels. A non-spatially smoothed parameter estimate map was obtained by fitting a GLM per event with that event as the regressor of interest and all the other remaining events as one regressor of no interest108 as implemented in NiBetaSeries (0.6.0) package. Decoding was performed using whole-brain and ROI-based approaches. The whole-brain analysis was performed using a searchlight approach with 4-mm radius. For ROI-based decoding, decoding ROIs were defined based on functional fMRI contrasts (see the ‘fMRI preprocessing' section) and constrained with pre-defined anatomical ROIs (see Extended Data Table 2 on anatomical ROIs). A one-sample permutation test was used to determine whether decoding significantly exceeded chance level within each ROI. FDR was used to correct for multiple comparisons across ROIs. For whole-brain decoding, a cluster-based permutation test was used to evaluate the decoding statistical significance across participants (P < 0.05), complemented by Bayesian analysis. In addition, stimulus versus baseline searchlight decoding was performed using leave-one-run out cross-validation, and the resultant decoding accuracy maps were used as input for the multivariate putative NCC analysis (see below). To test GNWT and IIT decoding predictions, stimulus category (faces versus objects and letters versus false fonts) was decoded separately for the task-relevant and task-irrelevant conditions (within-task category decoding), whereas orientation (front view versus left view versus right view) was decoded on the combined data from the two task conditions. In addition, cross-task category decoding from the task-relevant to task-irrelevant condition and vice versa was performed to test generalization by training classifiers on one condition and testing on the other condition. Both within-task category and orientation decoding were performed in a leave-one-run-out cross-validation scheme for fMRI and in an k-fold cross-validation scheme for MEG and iEEG. For category decoding, trials from each task condition (that is, task relevant or task irrelevant) were extracted for each category comparison of interest: 160 face/160 objects classification, 160 letters/160 false-fonts classification within each task-relevant condition for MEG, and half the trials for iEEG. For orientation decoding, task-relevant and task-irrelevant trials were collapsed within category to increase the signal-to-noise ratio, resulting in 160 front, 80 left and 80 right trials per category for MEG, and half these numbers for iEEG. Decoding was evaluated using accuracy measures, tested against 50% chance level for category decoding (binary classification) and against 33% chance level for orientation decoding (three-class classification). For orientation decoding, balanced accuracy was used due to the unbalanced number of trials for the different orientations. For within-task decoding (for example, classification of categories across time), a classifier at each time point was trained and tested separately using a fivefold cross-validation (with three separate repeats of cross-validation). For cross-task decoding (task relevant → irrelevant and task irrelevant → relevant), each SVM model was trained on one task (for example, faces–objects in the task-relevant condition) and tested on the second task (for example, faces–objects in the task-irrelevant condition). As cross-decoding in iEEG data is performed across all pooled electrodes, an additional cross-validation step was performed on this modality data to provide a confidence metric (for example, confidence intervals) using a fivefold cross-validation with three repetitions (for example, train on 80% of task 1, and test on held-out 20% of task 2). Within-task temporal generalization was performed by training a classifier at each time point (using selectKbest feature selection) and testing its performance across all time points using the same set of selected features and three repetitions of fivefold cross-validation. To generalize from one task to another across all time points, cross-temporal generalization was used: a classifier was trained at each time point in task 1 (for example, task relevant) using selectKbest feature selection, and tested across all time points in task 2 (for example, task irrelevant) using the same set of selected features. For the latter analysis, all trials from different durations were aligned to the stimulus offset. To assess the prediction of IIT that included prefrontal regions along with posterior regions to the decoding of categories will not significantly affect decoding accuracy, we performed an additional decoding analysis in which the decoding performance of electrodes from the IIT region were compared with the decoding performance when electrodes from both the posterior + PFC ROIs are included. Posterior ROI included all IIT ROIs shown in Supplementary Table 26. The analysis compared the decoding accuracy for a model including all electrodes from posterior regions to a separate model in which electrodes (features) from posterior and PFC regions were combined (for example, feature combination). Training and testing of the individual models followed all previously described cross-validation procedures, and model comparison was performed using a variance-corrected paired t-test110 and complemented with Bayesian analysis. We also tested this prediction on the fMRI data. To select features to be used for both analyses, the face versus object contrast for each participant was masked by a predefined anatomical posterior ROIs as well as PFC anatomical ROIs, defined the same way as described above. The first analysis compared the decoding accuracies for a model that included 300 voxels from the posterior ROIs as features to another model that included 600 voxels (300 features from each ROI). In the second analysis, two separate models were constructed, calibrated and combined as described above. For the two analyses, model comparison was performed using a group-level one-sample permutation test to determine if accuracies obtained by combining posterior and PFC ROIs were significantly higher than the accuracies obtained based on posterior ROIs only. FDR was used to correct for multiple comparisons. Bayesian analysis was performed to quantify evidence for the null hypothesis that adding prefrontal ROIs will not improve decoding accuracy. Two theory models were fitted: the GNWT model predicts activation (ignition) following stimulus offset (0.3–0.5 s) independent of duration, with virtually no response in between. Both theoretical models were complemented with an interaction term between category (faces, objects, letters and false fonts) and the theories' predictors, to account for regions showing selective responses to categories. BIC was used to define the winning model and we computed Bayes factors based on the difference in BIC values, comparing the GNWT model (with or without interaction) against either the null model (intercept only) or the time-window model (capturing amplitude changes over time)111. Models for iEEG were fitted per electrode on the predefined ROIs, using the high-gamma (AUC), alpha (8–13 Hz, obtained through Morlet wavelets, f = 8–13 Hz, in 1-Hz steps; f/2 cycles, AUC),and ERPs (peak to peak) as signal, separately for task-relevant and task-irrelevant condition. MEG models were fitted to source data on the predefined ROIs, using the gamma (60–90 Hz) and alpha (8–13 Hz) bands as signal, separately for task-relevant and task-irrelevant conditions. Time-frequency analyses were performed on source-data using Morlet wavelets (f = 8–13 Hz, in 1-Hz steps; f/2 cycles; f = 60–90 Hz, in 2-Hz steps, f/4 cycles) and were baseline corrected. Spectral activity was computed for each vertex, baseline corrected and then averaged across trials within each parcel included in the ROIs, yielding a unique time course per ROI parcel. In addition, a single-source time course capturing the entire prefrontal ROI and the posterior ROI was computed by averaging the spectral activity within an ROI. Models were fitted on each parcel and ROI, as defined by the theories. To examine how the neural representations evolved over time in response to the different stimulus properties (that is, category, orientation and identity representation), we performed cross-temporal RSA on source-level MEG data and iEEG high-gamma power within each of the theory-defined ROIs, using all trials. Specifically, at each set of data points, we computed a representational dissimilarity matrix (RDM) by calculating the correlation distance (1 − Pearson's r, Fisher corrected) between all pairs of stimuli (the preregistration document described a different method that was however updated to optimize trial numbers; see section 14 in Supplementary Information for justification). Next, to quantify the representational space occupied by one class versus another, we computed the average within-class distances versus the average between-class distances. This analysis was performed in a cross-temporal manner, in which RDMs were computed between all stimuli at time point t1 and the corresponding set of stimuli at time points t1, t2,…tn. Long trials (1.5 s) were used to investigate category and orientation representation. As specific identities were repeated a limited number of times per duration, both intermediate (1.0 s) and long (1.5 s) trials were combined and equated in duration by cropping the 1–1.5-s time interval for long trials. This was done to allow for the analysis of at least three (3) presentations of the same identity. The subsampled matrices were correlated to the model matrices predicted by GNWT and IIT (see Fig. 1a, right panel) using Kendall's tau correlation. If the correlation was significant (see below) for at least one of the predicted matrices, we computed the difference between the transformed correlation (\((r+1)/2\)) to each theory, and compared this difference against a random distribution to obtain a P value. To generate a null distribution of cross-temporal RSA surrogate matrices, we repeated the procedure outlined above 1,024 times, randomly shuffling the labels. Where \({{\rm{obs}}}_{i,j}\) is the observed within-versus-between class difference at time points i and j, and \({\mu }_{{{\rm{surr}}}_{i,j}}\) and \({\sigma }_{{{\rm{surr}}}_{i,j}}\) are the mean and standard deviation of the surrogate representational similarity matrix at time points i and j, respectively. Cluster-based permutation tests112, z-score threshold of z = 1.5 for clustering, were used to evaluate significance. Next, a P value was obtained for each theory-predicted matrix, by locating its observed correlation within the null correlation distribution. P values were FDR corrected (q ≤ 0.05)75. For iEEG, the high-gamma power per electrode within the predefined anatomical ROI was averaged in 0.02-s non-overlapping windows. Electrodes were used as features for the RDM. The data were vectorized across all electrodes within a ROI (for example, samples × significant electrodes) to compute the RDMs. A total of 576 and 583 electrodes entered this analysis for the prefrontal and posterior ROI, respectively. The resultant RDM was subjected to a PCA, and the first two dimensions were plotted against each other to produce a 2D projection of dissimilarity scores across all pairs for each of the 100 subsampling repetitions. The PCA components were aligned across repetitions using Procrustes alignment and averaged together for visualization purposes113,114. For the category and orientation analysis, pseudotrials and temporal moving-average methods were used to optimize the RSA analysis and improve the signal-to-noise ratio. For identity, single trials were used. Vertices within the ROIs were used as features. For both iEEG and MEG, PPC46 was computed between each category-selective time series (face selective and object selective) and either the V1/V2 or the PFC time series. For iEEG, the PPC analysis included electrodes in V1/V2 visual areas, in PFC ROIs (see Supplementary Table 26), and face-selective and object-selective electrodes (see ‘Identification of task-responsive channels'), as long as they were ‘active' during the task. As both theories predict different types of activation (for example, ignition versus sustained activation), channels were categorized as active if they showed an increase in high-gamma power relative to baseline (−0.5 to −0.3 s, P < 0.05, signed-rank test) evaluated across all trials (task relevant + irrelevant, intermediate + long trials, combined across both categories), for the 0.3–0.5-s window (GNWT), or in all time windows: 0.3–0.5 s, 0.5–0.8 s and 1.3–1.5 s (IIT). Two GED spatial filters were built by contrasting either faces or objects against all other categories during the first 0.5 s after stimulus onset. Single-trial covariance matrices were computed separately for signal and reference for all vertices within the fusiform ROI identified from the FreeSurfer parcellation using the Desikan atlas116, and the Euclidean distance between them was z-scored. The reference covariance matrix was regularized to reduce overfitting and increase numerical stability. A GED spatial filter was extracted also for the PFC ROI, on parcels from the Destrieux atlas73, to identify the distributed pattern of sources that are responsive to visually presented stimuli. Specifically, a spatial filter was built by contrasting source-level frontal slow-frequency activity (30-Hz low-pass filter) after stimulus onset (0–0.5 s) against baseline (−0.5 to 0 s). V1/V2 areas were identified using the Wang Atlas74 and a singular values-decomposition approach. PPC was computed for each MEG time series–iEEG electrode pairing, for all face trials and object trials separately. To compute synchrony, time-frequency analysis of the broadband MEG and LFP signal was performed using Morlet wavelets (f = 2–30 Hz, in 1-Hz steps; 4 cycles; f = 30–180 Hz for iEEG or f = 30–100 Hz for MEG, in 2-Hz steps, f/4 cycles), and PPC was then computed by taking the difference in phase angle between MEG time series–iEEG electrode at each time t and frequency f for a specific trial and computing PPC across all trials in a category (for example, faces) as: For iEEG, PPC for each category-selective site was then averaged across all its pairings (for example, all PFC electrodes pairings or all V1/V2 pairings within that patient). The variability in electrode coverage across patients precluded a within-participants analysis. Therefore, to achieve sufficient statistical power, we pooled all derived PPC values from one electrode pairing (for example, face selective to the PFC) across all patients into one ROI-specific analysis. A similar approach was used on the MEG parcels. To quantify content-specific synchrony enhancement, the difference in PPC was computed between within-category and across-category trials (for example, for face-selective sites, the change in PPC was computed between faces versus objects trials) using a cluster-based permutation test106. This was done for both modalities. As an exploratory analysis, we also investigated dynamic functional connectivity using the Gaussian copula mutual information117 approach to evaluate the dependencies between time series. We used the same parameters as for the PPC analysis, with the following exceptions: for both MEG and iEEG, power was estimated through a multitaper-based method (using a frequency-dependent dynamic sliding window: 2–30 Hz, T = 4 cycles; 30–100 Hz, T4/f using a 0.25-s sliding window). For iEEG, the high-frequency range was extended from 30 to 180 Hz, T = 4/f cycles). DFC was performed per frequency band, 0.1-s sliding window and 0.02-s steps. For fMRI, connectivity was assessed through gPPI implemented in SPM119. The FFA and lateral occipital cortex were defined as seed regions per participant based on an anatomically constrained functional contrast. Candidate seed voxels within the above-mentioned anatomical ROIs were defined as those with z > 1 in the contrast of parameter estimates of all stimuli versus baseline. Three participants with less than 300 candidate seed voxels were excluded from the analysis. This was done to ensure that the seed voxels were visually driven. Next, using an unthresholded contrast of parameter estimates between ‘relevant and irrelevant faces' and ‘relevant and irrelevant objects', the 300 voxels most responsive to faces within the FFA anatomical ROIs were selected for the FFA seed, and the 300 voxels most responsive to objects within the LOC anatomical ROIs were selected for the LOC seed. gPPI analysis was performed per participant and seed region separately, including an interaction term between the seed time-series regressor (physiological term) and the task regressor (psychological term) at the participant-level GLM119, separately for task-relevant and irrelevant conditions, and also combining across tasks to increase statistical power. For combined conditions, the model design matrix for each participant included regressors for task-relevant and task-irrelevant faces, objects, letters and false fonts collapsed across conditions (four regressors) as well as a regressor for targets (irrespective of their category), yielding five regressors in total. As for separated conditions, the model design matrix included regressors for task-relevant and task-irrelevant faces, objects, letters and false fonts (eight regressors) as well as a regressor for targets (irrespective of their category), yielding nine regressors in total. For each seed, group-level analysis was performed using a cluster-based permutation test (preferred over the preregistered FDR correction), complemented by Bayesian analysis. See section 14 in Supplementary Information for a justification of this change to evaluate the statistical significance of face > object contrast parameter estimates across participants (P < 0.05). A series of conjunction analyses were performed on the fMRI data to identify (1) areas responsive to task goal, (2) areas responsive to task relevance, and (3) areas putatively involved in the neural correlates of consciousness. We note that the contrasts proposed below might overestimate the neural correlates of consciousness and that the fast-event-related design adopted here might be suboptimal to detect activity changes in the salience network120, that is, potentially underestimating some regions that might be involved in conscious processing. This contrast captures areas that show an increase of BOLD signal for targets but not for other stimuli. This contrast identifies areas displaying differential activity for all task-relevant stimuli, but are insensitive to non-task-relevant stimuli. Finally, the following conjunction was used to identify the putative NCC areas: [(TaskRelNonTar (stim id) > bsl) and (TaskIrrel (stim id) > bsl)] or [(TaskRelNonTar (stim id) < bsl) and (TaskIrrel (stim id) < bsl)], critically detecting areas that are responsive to any stimulus category irrespective of task, with consistent activation or deactivation. Thus, this analysis casts a wide net to identify areas that can potentially be the neural correlates of consciousness, whereas excluding areas that do not respond to task-relevant or irrelevant stimuli (meaning that areas that respond both to the task and to the content of perception are still included). To compute conjunctions, we first ran a GLM (see above) corrected for multiple comparisons (Gaussian random-field cluster-based inference). Equivalence to baseline was established using a JZS Bayes factor test, with a Cauchy prior (r scale value of 0.707, as implemented in Pingouin (0.5.1)121. For conjunctions including an ‘unequal to', a ‘logical and' operation was used between the directional z maps, after thresholded maps were binarized. For the putative NCC contrast, conjunctions were performed separately for activations and deactivations, using a ‘logical and' operator for the task-relevant and irrelevant z maps. The resulting maps were combined using a ‘logical or' operation to discard areas showing effects of opposite direction for task-relevant and task-irrelevant stimuli. This analysis was also done at the participant level, masked using the anatomical ROIs, to account for inter-participant variability. For each ROI, the proportion of participants with voxels included in the conjunction is reported. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data generated in this study are available under a CC BY 4.0 license. The M-EEG, fMRI and iEEG datasets are distributed through two methods: as downloadable data bundles and via an XNAT instance, which enables search functionality and single-participant downloads. All distribution formats include robust metadata, and detailed documentation of experimental procedures and dataset structure is available (https://cogitate-consortium.github.io/cogitate-data/). For further inquiries, please contact the corresponding author. Task and analysis codes have been shared under an MIT license. The task code129 has been shared on GitHub (https://github.com/Cogitate-consortium/cogitate-experiment-code). The analysis code130 has also been shared on GitHub (https://github.com/Cogitate-consortium/cogitate-msp1). Yaron, I., Melloni, L., Pitts, M. & Mudrik, L. The ConTraSt database for analysing and comparing empirical studies of consciousness theories. Integrated information theory (IIT) 4.0: formulating the properties of phenomenal existence in physical terms. Tononi, G., Boly, M., Massimini, M. & Koch, C. Integrated information theory: from consciousness to its physical substrate. Experimental and theoretical approaches to conscious processing. Dehaene, S., Lau, H. & Kouider, S. What is consciousness, and could machines have it? Melloni, L., Mudrik, L., Pitts, M. & Koch, C. Making the hard problem of consciousness easier. Melloni, L. et al. An adversarial collaboration protocol for testing contrasting predictions of global neuronal workspace and integrated information theory. Comment on Clark et al. J. Appl. Lamme, V. The crack of dawn: perceptual functions and neural mechanisms that mark the transition from unconscious processing to conscious vision. Malach, R. Local neuronal relational structures underlying the contents of human conscious experience. Brown, R. The HOROR theory of phenomenal consciousness. Brown, R., Lau, H. & LeDoux, J. E. Understanding the higher-order approach to consciousness. Crick, F. & Koch, C. Towards a neurobiological theory of consciousness. Chalmers, D. in Neural Correlates of Consciousness: Empirical and Conceptual Questions (ed. Mellers, B., Hertwig, R. & Kahneman, D. Do frequency representations eliminate conjunction effects? Clark, C., Costello, T., Mitchell, G. & Tetlock, P. E. Keep your enemies close: adversarial collaborations will improve behavioral sciences. Mashour, G. A., Roelfsema, P., Changeux, J.-P. & Dehaene, S. Conscious processing and the global neuronal workspace hypothesis. Koch, C., Massimini, M., Boly, M. & Tononi, G. Neural correlates of consciousness: progress and problems. Towards a principled account of spatial experience. & Roelfsema, P. R. The distinct modes of vision offered by feedforward and recurrent processing. Mudrik, L. et al. Cogitate - preregistration v4 - December 2022. Gerber, E. M., Golan, T., Knight, R. T. & Deouell, L. Y. Cortical representation of persistent visual stimuli. & Grill-Spector, K. Encoding model of temporal processing in human visual cortex. Podvalny, E. et al. Invariant temporal dynamics underlie perceptual stability in human visual cortex. Stokes, M. G. ‘Activity-silent' working memory in prefrontal cortex: a dynamic coding framework. Lepauvre, A., Hirschhorn, R., Bendtz, K., Mudrik, L. & Melloni, L. A standardized framework to test event-based experiments. & Kahneman, D. Belief in the law of small numbers. Kapoor, V. et al. Decoding internally generated transitions of conscious contents in the prefrontal cortex without subjective reports. Frässle, S., Sommer, J., Jansen, A., Naber, M. & Einhäuser, W. Binocular rivalry: frontal activity relates to introspection and action but not to perception. & Kahneman, D. Extensional versus intuitive reasoning: the conjunction fallacy in probability judgment. Nir, Y. et al. Coupling between neuronal firing rate, gamma LFP, and BOLD fMRI is related to interneuronal correlations. & Hsiao, S. S. Neural correlates of high-gamma oscillations (60–200 Hz) in macaque local field potentials and their potential implications in electrocorticography. Koch, C., Massimini, M., Boly, M. & Tononi, G. Posterior and anterior cortex — where is the difference that makes the difference? Haegens, S., Nácher, V., Luna, R., Romo, R. & Jensen, O. α-Oscillations in the monkey sensorimotor network influence discrimination performance by rhythmical inhibition of neuronal spiking. Ongoing neural oscillations influence behavior and sensory representations by suppressing neuronal excitability. Distinct ventral stream and prefrontal cortex representational dynamics during sustained conscious visual perception. Broday-Dvir, R., Norman, Y., Harel, M., Mehta, A. D. & Malach, R. Perceptual stability reflected in neuronal pattern similarities in human visual cortex. Jackendoff, R. Consciousness and the Computational Mind (MIT Press, 1987). Gaillard, R. et al. Converging intracranial markers of conscious access. Vinck, M., van Wingerden, M., Womelsdorf, T., Fries, P. & Pennartz, C. M. A. The pairwise phase consistency: a bias-free measure of rhythmic neuronal synchronization. A. et al. Driving fast-spiking cells induces gamma rhythm and controls sensory responses. Kahneman, D. Adversarial collaboration: an EDGE lecture. Lakatos, I. in Can Theories be Refuted? Cleeremans, A. et al. Learning to be conscious. Fleming, S. M. Awareness as inference in a higher-order state space. Melloni, L. & Singer, W. The explanatory gap in neuroscience. Aru, J., Bachmann, T., Singer, W. & Melloni, L. Distilling the neural correlates of consciousness. Tsuchiya, N., Wilke, M., Frässle, S. & Lamme, V. A. F. No-report paradigms: extracting the true neural correlates of consciousness. Popper, K. The Logic of Scientific Discovery (Routledge, 1935). Chis-Ciure, R., Melloni, L. & Northoff, G. A measure centrality index for systematic empirical comparison of consciousness theories. Clark, C. J. et al. Prosocial motives underlie scientific censorship by scientists: a perspective and research agenda. & Barack, D. L. Tasks and their role in visual neuroscience. & Manly, T. When attended and conscious perception deactivates fronto-parietal regions. Glover, G. H. Deconvolution of impulse response in event-related BOLD fMRI1. Pelli, D. G. The VideoToolbox software for visual psychophysics: transforming numbers into movies. Holdgraf, C. et al. iEEG-BIDS, extending the brain imaging data structure specification to human intracranial electrophysiology. Gramfort, A. et al. MNE software for processing MEG and EEG data. Li, G. et al. Optimal referencing for stereo-electroencephalographic (SEEG) recordings. Grossman, S. et al. Convergent evolution of face spaces across human face-selective neuronal groups and deep convolutional networks. & Kahana, M. J. Broadband shifts in local field potential power spectra are correlated with single-neuron spiking in humans. & Halgren, E. Automatic parcellation of human cortical gyri and sulci using standard anatomical nomenclature. & Kastner, S. Probabilistic maps of visual topography in human cortex. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Rouder, J. N., Speckman, P. L., Sun, D., Morey, R. D. & Iverson, G. Bayesian t tests for accepting and rejecting the null hypothesis. Kadipasaoglu, C. M., Conner, C. R., Whaley, M. L., Baboyan, V. G. & Tandon, N. Category-selectivity in human visual cortex follows cortical topology: a grouped icEEG study. Niso, G. et al. MEG-BIDS, the brain imaging data structure extended to magnetoencephalography. Appelhoff, S. et al. MNE-BIDS: organizing electrophysiological data into the BIDS format and facilitating their analysis. Ferrante, O. et al. FLUX: a pipeline for MEG analysis. Taulu, S., Kajola, M. & Simola, J. Suppression of interference and artifacts by the signal space separation method. & Oja, E. Independent component analysis: algorithms and applications. Hämäläinen, M. S. & Ilmoniemi, R. J. Interpreting magnetic fields of the brain: minimum norm estimates. & Kaufman, L. Magnetic source images determined by a lead-field analysis: the unique minimum-norm least-squares estimation. Zwiers, M. P., Moia, S. & Oostenveld, R. BIDScoin: a user-friendly application to convert source data to brain imaging data structure. & Rorden, C. The first step for neuroimaging data analysis: DICOM to NIfTI conversion. Esteban, O. et al. MRIQC: advancing the automatic prediction of image quality in MRI from unseen sites. Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. Gorgolewski, K. et al. Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in Python. McKinney, W. Data structures for statistical computing in Python. Smith, S. M. et al. Advances in functional and structural MR image analysis and implementation as FSL. Penny, W., Friston, K., Ashburner, J., Kiebel, S. & Nichols, T. Statistical Parametric Mapping: the Analysis of Funtional Brain Images 1st edn (Elsevier/Academic Press, 2007). Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Hautus, M. J. Corrections for extreme proportions and their biasing effects on estimated values of d′. Satterthwaite, T. D. et al. An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data. Approximate objective Bayes factors from P-values and sample size: the 3p√n rule. & Cohen, N. A novel blink detection method based on pupillometry noise. Yuval-Greenberg, S., Merriam, E. P. & Heeger, D. J. Spontaneous microsaccades reflect shifts in covert attention. Engbert, R. & Kliegl, R. Microsaccades uncover the orientation of covert attention. Comparative study of techniques for large-scale feature selection. Nichols, T. E. & Holmes, A. P. Nonparametric permutation tests for functional neuroimaging: a primer with examples. Maris, E. & Oostenveld, R. Nonparametric statistical testing of EEG- and MEG-data. Cichy, R. M. & Pantazis, D. Multivariate pattern analysis of MEG and EEG: a comparison of representational structure in time and space. A., Turner, B. O., Ashby, F. G. & Poldrack, R. A. Deconvolving BOLD activation in event-related designs for multivoxel pattern classification analyses. Nadeau, C. & Bengio, Y. in Advances in Neural Information Processing Systems 12 (eds Solla, S. A., Leen, T. K. & Müller, K.-R.) 307–313 (MIT Press, 2000). A practical solution to the pervasive problems of P values. & Turner, R. Statistical inference and multiple testing correction in classification-based multi-voxel pattern analysis (MVPA): random permutations and cluster size control. Schönemann, P. H. A generalized solution of the orthogonal procrustes problem. & Finos, L. Procrustes-based distances for exploring between-matrices similarity. A tutorial on generalized eigendecomposition for denoising, contrast enhancement, and dimension reduction in multichannel electrophysiology. Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. A. et al. A statistical framework for neuroimaging data analysis based on mutual information estimated via a Gaussian copula: Gaussian copula mutual information. & Brovelli, A. Frites: a Python package for functional connectivity analysis and group-level statistics of neurophysiological data. McLaren, D. G., Ries, M. L., Xu, G. & Johnson, S. C. A generalized form of context-dependent psychophysiological interactions (gPPI): a comparison to standard approaches. Liu, L. et al. Cogitate MEEG raw data. Khalaf, A. et al. Cogitate fMRI raw data. Seedat, A. et al. Cogitate iEEG raw data. Liu, L. et al. Cogitate MEEG BIDS data. Khalaf, A. et al. Cogitate fMRI BIDS data. Seedat, A. et al. Cogitate iEEG BIDS data. Cogitate Consortium et al. Cogitate-Consortium Cogitate-experiment-code: v1.0. We thank D. Potgieter for spearheading the ARC program; D. Kahneman for guidance to navigate adversarial collaborations; H. Berlin, W. Jaworski, H. Lau and C. Pennartz for insightful discussions during the 2-day meeting organized by the Templeton World Charity Foundation at the Allen Institute, Seattle in March 2018; H. Lau for helping conceptualize the proposed experiments; O. Devinsky, W. Doyle, P. Dugan and D. Friedman for supporting the recruitment and patient care at NYU; E. Yacoub, M. Kahana, P. Zeidman, K. Friston, J.-R. King, M. X. Cohen, M. A. Cohen, F. Al Roumi, S. Yuval-Greenberg, D. Draschkow and A. Brovelli for guidance on diverse data analyses; C. Schwiedrzik for insightful discussions and feedback throughout the different phases of this study (conceptualization, data analysis and initial draft); S. Brendecke and F. Bernoully for help with figures and stimulus materials; M. Smulders and S. Kusch for help with fMRI data acquisition; and the patients and their families for generously supporting this study. This research was supported by Templeton World Charity Foundation (TWCF0389 (https://www.templetonworldcharity.org/projects-resources/project-database/0389) and TWCF0486 (https://www.templetonworldcharity.org/projects-resources/project-database/0486)) and the Max Planck Society. Open access funding provided by Max Planck Society. These authors contributed equally: Oscar Ferrante, Urszula Gorska-Klimowska, Simon Henin, Rony Hirschhorn, Aya Khalaf, Alex Lepauvre, Ling Liu, David Richter, Yamil Vidal These authors jointly supervised this work: Michael Pitts, Liad Mudrik, Lucia Melloni Oscar Ferrante, Tara Ghafari & Dorottya Hetenyi Urszula Gorska-Klimowska, Csaba Kozma, Melanie Boly & Giulio Tononi Department of Neurology, New York University Grossman School of Medicine, New York, NY, USA Simon Henin, Jay Jeschke, Stephanie Montenegro, Alia Seedat, Sasha Devore & Lucia Melloni Neural Circuits, Consciousness and Cognition Research Group, Max Planck Institute for Empirical Aesthetics, Frankfurt am Main, Germany Alex Lepauvre, Tanya Brown, Praveen Sripad & Lucia Melloni Alex Lepauvre, David Richter, Yamil Vidal & Floris P. de Lange School of Psychological and Cognitive Sciences, Peking University, Beijing, China William James Center for Research, ISPA - Instituto Universitário, Lisbon, Portugal Marcelo Armendariz, Katarina Bendtz, David R. Mazumder & Gabriel Kreiman Center for Brains, Minds and Machines, Cambridge, MA, USA Marcelo Armendariz, Katarina Bendtz & Gabriel Kreiman Department of Imaging Neuroscience, UCL Queen Square Institute of Neurology, University College London, London, UK Georgia Institute of Technology, Atlanta, GA, USA Centre for Basic Research, Biomedical Research Foundation of the Academy of Athens (BRFAA), Athens, Greece IDG/McGovern Institute for Brain Research, Peking University, Beijing, China Cognitive Neuroimaging Unit, Commissariat à l'Energie Atomique (CEA), Institut National de la Santé et de la Recherche Médicale (INSERM), Université Paris-Saclay, NeuroSpin Center, Gif-sur-Yvette, France Tiny Blue Dot Foundation, Santa Monica, CA, USA Psychology Department, Reed College, Portland, OR, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Initial conceptualization of the experiment and development of predictions were done by D.J.C., F.F., H.B., S. Dehaene, C. Koch, G.T., L. Mudrik, M.P. Development of the experimental framework, design of the adversarial collaboration and overall consortium structure was conducted by L. Mudrik, M.P. Refinement of experimental design for preregistration was done by S.B., D.J.C., R.M.C., F.F., T.I.P., H.B., O.J., F.P.d.L., H.L., M.B., S. Dehaene, C. Koch, G.T., L. Mudrik, M.P. Project administration and leadership at each site were done by O.J. (Peking) for MEG site Principal Investigators; H.B. (Wisconsin) for iEEG site Principal Investigators. Data release plan and metadata organization were done by N.B., P.S., T.B. Management and coordination of High Performance Computing resources were performed by A.L., T.B., P.S. Data curation was performed by L. Melloni, P.S., N.B. Conceptualization of the experimental framework for standardization across sites was conducted by A.L., K.B., R.H., L. Mudrik and L. Melloni. The experimental framework was carried out by O.F., U.G.-K., A.K., L.L. D.R., K.B., S.M., A. Seedat, A. Sharafeldina and S.Y., along with site PIs. and R.H. iEEG data were collected by J.J., S.M. iEEG data quality were checked by U.G.-K., A.L., M.A., K.B., J.J., S.M. MEG data quality was checked by O.F., L.L., T.G. fMRI data quality was checked by A.K., D.R. iEEG decoding and synchrony analyses, unification across modalities and validation of methods were performed by S.H. Responsiveness, RSA, activation and LMMs for theory testing were done by A.L. Supervision of the iEEG team was provided by G.K., L. Melloni and S. Devore. MEG preprocessing and source modelling were done by O.F. Activation, connectivity and associated control analyses were performed by O.F. Decoding, RSA and associated control analyses were done by L.L. Supervision of the MEG team was provided by O.J. fMRI searchlight and ROI decoding, gPPI, and stimulus versus baseline decoding were performed by A.K. Putative NCC, generation of seeds for gPPI and ROIs for decoding analyses, and definition and generation of anatomical ROIs were done by Y.V. Supervision for the fMRI team was provided by H.B. Analysis coordination across teams was done by T.B. Expert methodological advice was provided by S.B., R.M.C., T.I.P. Visualizations were done by R.H. (behavioural and eye tracking); S.H. (iEEG RSA, activation, visual responsiveness, category selectivity, duration decoding and duration tracking), supervised by L. Melloni; O.F. (MEG decoding, RSA and leakage analysis), supervised by O.J., H.L. Overall figure integration was done by T.B., L. Mudrik, M.P. The original draft was written by L. Mudrik, M.P. All authors reviewed, edited, provided critical comments and approved the final manuscript before submission. A detailed authors contribution grid is available in Supplementary Fig. C. Koch is the Chief Scientist of the Tiny Blue Dot Foundation in Santa Monica, CA. S.Dehaene is a co-inventor on patent 2019 EP 2983586 (‘Methods to monitor consciousness'); and is an associate at NeuroMeters, a company that applies these methods in clinical practice. None of these affiliations impose restrictions on publication or present conflicts of interest related to this study. The other authors declare no competing interests. Nature thanks Asger Kirkeby-Hinrup, Benjamin Kozuch, Hakwan Lau and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, fMRI searchlight decoding accuracies (letters-falsefonts), collapsed across durations. Pattern classifiers trained on relevant stimuli and tested on irrelevant stimuli (left, purple) or vice versa (right, orange): Outlined colored regions on the inflated cortical surfaces (top: lateral views; bottom: medial views) indicate significant above-chance (50%) decoding. Here and below, significance was evaluated through a cluster-based permutation test (p < 0.05; two-sided). Sample sizes as reported in Fig. 2. b, iEEG ROIs decoding accuracies (letters-falsefonts) collapsed across durations. c, MEG cross-task decoding of category (letters-falsefonts) when classifiers were trained on relevant stimuli and tested on irrelevant stimuli (purple); or vice versa (orange), separately for the whole posterior (left) and prefrontal (right) ROIs. Underlying lines indicate significantly above-chance (50%) decoding. Error bars depict 95% CI across participants. d, iEEG cross-task temporal generalization of category decoding (letters-falsefonts) classifiers trained on task-relevant stimuli and tested on task-irrelevant stimuli. Rows: theory ROIs (top: posterior; bottom: prefrontal). Contoured red-shaded regions depict significant above-chance (50%) decoding. e, iEEG cross-task temporal generalization of category decoding (faces-objects), classifiers were trained on task-relevant stimuli and tested on task-irrelevant ones. Conventions as in d. f, iEEG cross-task temporal generalization of category decoding (faces-objects) from task-irrelevant to task-relevant stimuli, yet using pseudotrial aggregation to boost decoding accuracy. Conventions as in d. g, iEEG ROI decoding accuracies (faces-objects) using pseudotrials. Brain surfaces in panels a, b, g are from Freesurfer. a, iEEG within-task temporal generalization decoding of category (faces-objects) for pattern classifiers trained and tested on task-relevant stimuli. Contoured red-shaded regions depict significant above-chance (50%) decoding. Here and below, significance was evaluated through a cluster-based permutation test (p < 0.05; two-sided). 2. b, iEEG within-task temporal generalization decoding of category for task-irrelevant stimuli. Conventions as in a. c, MEG within-task average decoding of category (faces-objects), for the task-relevant (orange) and the task-irrelevant (purple) conditions, in posterior (left) and prefrontal (right) ROIs. Underlying lines depict significantly above-chance (50%) decoding assessed by cluster-based permutation test (p < 0.05). Error bars depict 95% CI estimated across participants. d, MEG within-task decoding of category (letters-falsefonts). Conventions as in c. e, fMRI searchlight decoding of category (faces-objects), collapsed across durations, for the task-relevant (left, orange) and task-irrelevant (right, purple) conditions. Outlined colors indicate regions on the inflated cortical surfaces showing significantly above-chance (50%) decoding (top: left/right lateral views; bottom: right/left medial views). f, iEEG ROIs decoding accuracies, collapsed across durations, within the task-relevant (left, orange) and the task-irrelevant (right, purple) stimuli. Brain surfaces in panels e,f are from Freesurfer. a, Left: iEEG ROIs decoding results of orientation (left/right/front view) over time as in Fig. 2, but using pseudotrials akin to the MEG analysis. Right: Regions with electrodes showing significant above-chance (33%) accuracies are indicated in outlined blue on the inflated surfaces (left: left lateral view; middle: posterior view; right: left medial view). Here and below, error bars depict 95% CI. Significance assessed using a cluster-based permutation test (p < 0.05, two-sided). 2. b, Two analyses were performed to evaluate potential leakage in MEG decoding, using independent data from the optimization phase (N = 32). Top: averaged stimulus-evoked response in face task-relevant trials, combined across durations, at different latencies, projected on the inflated surfaces. Activity in posterior areas (blue ellipse) showed the highest peak ~0.1-0.2 s, while prefrontal areas showed a later highest peak ~0.2-0.3 s. This challenges the leakage interpretation. Bottom: Analysis of face-object decoding in task-relevant trials across durations, separately within parcels in parietal and PFC. Left: Average decoding accuracy in an early time window (0.25-0.5 s) projected on two differently inflated surfaces to better depict gyri and sulci. Right: Time-resolved decoding of these parcels. Decoding is highest in posterior areas and lowest in anterior areas, with fairly similar time courses, suggesting a posterior-to-anterior gradient consistent with leakage. c, ROIs used in the decoding analysis including (blue) and excluding (red) PFC areas. Underlying lines indicate significantly worse decoding when including PFC. Histogram shows the differences in classification including and excluding frontal areas. fMRI accuracies including PFC show 1.2% increase compared to excluding PFC, observed in 56% of the participants. Notably, this slight increase was observed only in the combined features analysis and not the combined models' analysis (see Methods). Brain surfaces in panels a-c are from Freesurfer. a, Cross-temporal representational similarity matrices in Posterior ROI (Npatients=28, Nelectrodes = 583). The leftmost column shows similarity for letters vs. false fonts, separately for task-relevant (left) and task-irrelevant (right) trials. Principal Component Analysis (PCA) plots at 0.3 s illustrate the separability between letters and false fonts. The top rightward column display similarity for identity, while bottom rows show similarity for orientation. Contours indicate statistical significance based on cluster-based permutation tests (upper tail test, α = 0.05). PCA illustrates clear separability between letters and false fonts in the posterior cortex at 0.3 s, regardless of task relevance (top – task-relevant, bottom - task-irrelevant). This separability was largely sustained in the task-relevant condition but diminished between ~0.95 and 1.4 s. In the task-irrelevant condition, separability was significant only for a brief period at the beginning. While identity information was not sustained throughout the entire stimulus duration, elevated z-scores up to 1 s suggest a potential limitation in statistical power. No statistically significant orientation information was observed for any category. 3. b, Cross-temporal representational similarity matrices in Prefrontal ROI (Npatients=28, Nelectrodes = 576) for the same contrasts as and following the same conventions as in a. No contrast yielded statistically significant results in the PFC ROI. a, iEEG electrode coverage used to assess content-selective synchrony for IIT ROIs (top, Npatients = 4) & GNWT ROIs (bottom, Npatients = 21). Electrode coverage varied between ROIs as interareal connectivity was assessed between electrodes on a per-participant basis. b, iEEG Pairwise phase consistency (PPC) analysis of task-irrelevant trials reveals significant content-selective synchrony (e.g. faces > objects for face-selective electrodes; left; objects > faces for object-selective electrodes; right) in V1/V2 ROIs (top row), but not in PFC ROIs (bottom row). Color bars represent the average change in PPC (face and object trials) for each node (face-selective, object-selective). Positive values reflect stronger connectivity for faces, while negative values reflect stronger connectivity for objects. c, MEG (N = 65) cortical time-series were extracted per participant from cortical parcels in V1/V2 (blue), PFC (green) and in a fusiform (red) ROIs. Category-selective signals were obtained by creating a category-selective GED filter (i.e., contrasting face-object trials against any other stimulus category trials) on the activity extracted from the fusiform ROI. Here and below, significance was assessed using cluster-based permutation tests, p < 0.05, two-sided. d, MEG PPC analysis of task-irrelevant trials (N = 65) reveals significant category-selective synchrony below 25 Hz for the face-selective GED filter (i.e., faces > objects for face-selective electrodes) in both V1/V2 (top row) and PFC ROIs (bottom row) and for the object-selective synchrony (objects > faces for object-selective electrodes) in the PFC ROI only. Brain surfaces in panels a,c are from Freesurfer. a, iEEG PPC analysis of task-irrelevant trials did not reveal any significant category-selective synchrony cluster in posterior (top) or PFC (bottom) ROIs after removing the evoked response. Same conventions, sample size and statistical tests as in Extended Data Fig. b, MEG PPC analysis of task-irrelevant trials also did not reveal any synchrony cluster in any ROI after removing the evoked response. c, iEEG DFC analysis of task-irrelevant trials without removing the evoked response reveals significant content-selective connectivity between object-selective electrodes and V1/V2 electrodes (top-right), reflected as broadband (25–125 Hz) decrease in the change in DFC (e.g., faces < objects). Similar broadband changes in DFC (faces > objects) were observed for face-selective electrodes in PFC (bottom-left). d, MEG DFC analysis of task-irrelevant trials without removing the evoked response reveal significant content-selective synchrony between the face-selective GED filter node and both V1/V2 (top-left) and PFC (bottom-left). Smaller yet significant effects were detected for the object-selective GED filter (right). e, Generalized psychophysiological interactions (gPPI) task-related connectivity analysis of task-irrelevant (left) and task-relevant (right) trials revealed weak clusters of content-selective connectivity with FFA as the analysis seed (p < 0.01, uncorrected). Common significant regions showing task-related connectivity in task-irrelevant, task-relevant, and combined conditions include V1/V2, right intraparietal sulcus (IPS), and right inferior frontal gyrus (IFG). f, gPPI task-related connectivity analysis of task-irrelevant (left), task-relevant (middle), and combined conditions revealed weak clusters of content-selective connectivity with lateral occipital complex (LOC) as the analysis seed (p < 0.01, uncorrected). Overall, no common significant regions showed task-related connectivity. Brain surfaces in panel e, f are from Freesurfer. Left: Preregistered predictions of IIT (top) and GNWT (bottom) (see also ref. Key hypotheses (second column, Key hypotheses) are described alongside the three analyses used to test them (third column, Test): decoding (prediction #1; Fig. Potential outcomes and their interpretations are detailed in the fourth column (Possible outcome and interpretation), with outcomes aligning with predictions framed in green (pass) and contradictory outcomes framed in red (fail). Right: summary of the actual findings, integrating results across modalities and analyses. Key findings for each prediction are described (fifth column; ‘Result') with white denoting alignment with predictions, red indicating contradiction, white/red mixtures showing partial support or failure, and yellow indicating inconclusive results. Final conclusions synthesize these findings, using the same color coding. For IIT, the results mix a passed prediction (content-specific complex of neural units in posterior cortex, throughout the persistence of a percept, independent of the task) with a failure (maximum integrated information). These results are discussed in the main text, including their implications for other consciousness theories. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Cogitate Consortium., Ferrante, O., Gorska-Klimowska, U. et al. Adversarial testing of global neuronal workspace and integrated information theories of consciousness. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08941-z'>Single-cell transcriptomics reveal how root tissues adapt to soil stress</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 15:15:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Land plants thrive in soils showing vastly different properties and environmental stresses1. Root systems can adapt to contrasting soil conditions and stresses, yet how their responses are programmed at the individual cell scale remains unclear. Using single-cell RNA sequencing and spatial transcriptomic approaches, we showed major expression changes in outer root cell types when comparing the single-cell transcriptomes of rice roots grown in gel versus soil conditions. These tissue-specific transcriptional responses are related to nutrient homeostasis, cell wall integrity and defence in response to heterogeneous soil versus homogeneous gel growth conditions. We also demonstrate how the model soil stress, termed compaction, triggers expression changes in cell wall remodelling and barrier formation in outer and inner root tissues, regulated by abscisic acid released from phloem cells. Our study reveals how root tissues communicate and adapt to contrasting soil conditions at single-cell resolution. Crops such as rice thrive in arable soils that show natural heterogeneity. The heterogeneous nature of soils is characterized by uneven distributions of nutrients, water, microorganisms and organic content that pose a stark contrast to the uniformity of growth media, underscoring the fundamental importance to understand how plant roots navigate, adapt and thrive at molecular and cellular levels in natural soils. Roots have evolved diverse strategies to tackle the variability in soil conditions1,2. However, our understanding of how roots respond to complex soil environments at a cellular level of resolution remains limited. The application of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic approaches to plant organs grown in diverse environments has the potential to reveal gene-expression complexity throughout root developmental stages, and identify mechanisms governing cell type-specific responses to environmental stresses3,4. Soil stresses represent a major challenge in global agriculture5. For example, soil compaction stress reduces root penetration, thereby affecting nutrient and water uptake and subsequently crop yield6. Roots have developed adaptive growth responses for compacted soils; but the underlying genes, cell group-specific transcriptional responses and molecular mechanisms remain poorly understood. To discover the mechanisms governing root responses to soil compaction at a cellular resolution, we pioneered transcriptomic profiling of rice root tissues grown in soils with and without compaction using single-cell approaches. Protoplasts of rice primary roots, obtained from gel-based conditions, were initially adopted to generate a high-quality scRNA-seq reference dataset to show cell identities and differentiation trajectories7, and later compared to equivalent datasets generated from soil-grown roots. Using the 10X Genomics scRNA-seq platform, we profiled more than 47,000 root cells gathered from Xkitaake rice primary root tissues 2–3 days after germination across ten sets of independently grown seedlings. To enhance depth and assess data variability across laboratories, we integrated a previously published scRNA-seq dataset8 with our datasets. All datasets underwent processing with the COPILOT (cell preprocessing pipeline kallisto bustools) pipeline9, resulting in the integration of more than 79,000 high-quality cells to construct the final scRNA-seq atlas (Fig. To mitigate the effect of protoplasting on gene expression, we identified protoplasting-induced genes by means of bulk RNA sequencing (RNA-seq, Supplementary Table 2) and excluded them from data integration and differential expression analysis. This approach ensured the robustness of our scRNA-seq findings. a, Illustration of rice primary root anatomy with different cell types highlighted. c–h, Expression of identified cell type markers in scRNA-seq data, with colour scales indicating normalized, corrected UMI counts: trichoblast and/or atrichoblast (c), exodermis (d), sclerenchyma (e), cortex (f), endodermis (g) and phloem (h). i, Schematic of the rice primary root transverse section. j, Spatial transcriptomic visualization of major cell type markers in transverse root sections. Each dot represents a detected mRNA molecule, colour-coded by cell type. k–p, Spatial expression of cell type markers in 5-day-old rice roots using Molecular Cartography: trichoblast and/or atrichoblast (k), exodermis (l), sclerenchyma (m), cortex (n), endodermis (o) and phloem (p). q, Dot plot of cell type marker expression in gel samples. Dot size indicates the percentage of cells expressing each gene, and colour intensity represents average scaled expression. The full marker gene and their annotation list is in Supplementary Table 3. r, Visualization of major cell type marker expression in rice root longitudinal sections. s–u, Spatial analysis of trichoblast markers LOC_Os12g05380, OsGT3 (s); LOC_Os10g42750, OsCSLD1 (t); LOC_Os06g48050 (u), with detected mRNA molecules shown in red. Insets highlight expression initiation regions (yellow boxes, s,t 2X and u 3X). n = 3 biological replicates for gel-grown root longitudinal sections. Panels a,i adapted with permission from Xiaoying Zhu. To ensure representation from all major developmental stages, we assigned developmental annotations to major cell types by comparing each cell's transcriptome with bulk RNA-seq expression profiles of manually dissected root tissue segments corresponding to meristematic, elongation and maturation zones (Extended Data Fig. scRNA-seq studies often rely on pseudotime analysis to infer developmental stages, a computational method that orders cells on the basis of gene expression similarities but does not represent actual time. In our study, we integrated bulk RNA-seq data from rice root tissues at distinct developmental stages. Using stage-specific marker genes, we directly annotated developmental stages in our scRNA-seq dataset, aligning it more closely with experimental observations. This developmental-stage annotation is unique in our scRNA-seq dataset, compared to the previously published ones8,10,11. High Spearman correlation among transcriptional profiles of all samples confirmed minimal batch effects (Extended Data Fig. For cell type annotation, we initially used principal component analysis (PCA) and clustering techniques, followed by the calculation of z-scores for published markers (Methods). To address this limitation, we used Molecular Cartography, an optimized multiplexed fluorescence in situ hybridization technology12,13. This allowed us to explore in situ gene-expression profiles for the candidate cell type markers identified in our putatively annotated cell type clusters. Our spatial transcriptomic experiments validated the cell type-specific expression of more than 40 markers (Fig. We then refined our cell type annotation, relying on the expression patterns of validated markers in scRNA-seq clusters (Fig. The iterative feedback loop between scRNA-seq and spatial transcriptomics significantly increased the number of reliable markers for the major root cell types, enhancing cell annotation quality. Integrating scRNA-seq and spatial transcriptomic data also showed temporal gene-expression dynamics. Pseudotime analysis demonstrated a continuous differentiation trajectory of rice root epidermis (Extended Data Fig. 1c,d), and we examined temporal expression patterns of differentially expressed genes (DEGs) involved in root hair differentiation (Extended Data Fig. Among the DEGs, three genes (LOC_Os12g05380, OsGT3, encoding a putative xylosyltransferase (XXT); LOC_Os10g42750, OsCSLD1, encoding Cellulose Synthase Like D1; LOC_Os06g48050, unannotated) showed a sequential pattern of expression along root hair differentiation, detected in both scRNA-seq data and spatial transcriptomics data (Fig. Overall, our approach generated a high-resolution scRNA-seq atlas, with confidently annotated cell types and developmental stages. Xkitaake rice seedlings were cultivated in soils for 3 days, their roots were then harvested, after which scRNA-seq was conducted on protoplasts isolated from two biological replicates of 1-cm root tip segments. Root tip samples encompassed meristem, elongation and early maturation zones. To leverage our gel-based data for interpreting our soil-based data, gel-based and soil-based scRNA-seq datasets were integrated (Methods). Although the scRNA datasets obtained from different growth conditions could be distinguished with sample-wise correlation analyses, the gel-grown roots showed high one-to-one similarities with the soil-grown roots across almost all cell clusters (Extended Data Fig. Most validated cell type markers detected in our Molecular Cartography examination remained expressed in their target cell types under soil conditions (Fig. We thus relied on the expression patterns of these marker genes to annotate the major cell types in our soil-based scRNA-seq data (Extended Data Fig. We verified the relatively high correlation of individual cell types between the two growth conditions (Extended Data Fig. Our scRNA-seq analysis showed a notable decrease in the number of root hair cells detected under soil conditions. To verify this reduction in trichoblast cell numbers, we examined the expression of a root hair cell-specific marker line (proCSLD1::VENUS-N7). Imaging showed highly similar expression patterns between gel and soil conditions, indicating that the observed decrease is probably due to the loss of root hair cells during the protoplasting process (Extended Data Fig. In conclusion, we managed to annotate our soil-based scRNA-seq data with our knowledge gained from both gel-based scRNA-seq data and spatial transcriptomic application to soil-grown roots. c, UMAP projection with developmental-stage annotations, based on bulk RNA-seq data from rice roots grown in gel. d–l, Cell type marker expression in scRNA-seq and spatial data from roots in non-compacted soils: atrichoblast (d), trichoblast (e), exodermis (f), sclerenchyma (g), cortex (h), endodermis (i), vascular tissue (j), phloem (k) and xylem (l). Colour scale in feature plots shows normalized UMI counts. Spatial transcriptomics show marker expression in transverse sections, with dots representing detected mRNA molecules. Full marker gene list is in Supplementary Table 3. m, UMAP visualization of DEG numbers in gel versus soil conditions. The outer cell layers (exodermis, sclerenchyma and cortex) have more DEGs compared to the inner cell layers (endodermis and stele). n, Top enriched GO terms for upregulated genes in soil-grown roots include defence response, phosphorus metabolism, vesicle transport, hormone signalling and cell wall organization, mainly in outer cell layers. P values were calculated using a one-tailed hypergeometric test with g:Profiler2 g:SCS for multiple comparison correction. o,p, Nutrient uptake (o) and cell wall strengthening genes (p) are induced in soil-grown roots, particularly in epidermis, exodermis, sclerenchyma and cortex (red box), highlighting their role in adapting to heterogeneous soil environments. Grey boxes indicate genes not detected in the analysis. q, Schematics illustrating rice roots grown in homogeneous gel versus heterogeneous soils. r, Single-cell transcriptomics indicate that outer cell layers respond more to soil heterogeneity, enhancing nutrient uptake to support root development while mitigating local stress effects on growth. To delve into the mechanisms governing cell type-specific responses to soil growth conditions, we conducted differential expression analysis for confidently annotated root cell types and developmental stage-enriched groups (Methods). Notably, 31% of DEGs were altered in a single cell type or developmental stage, indicating that changes in growth conditions modulate distinct sets of genes in specific cell type contexts (Extended Data Fig. Most DEGs were found in the outer root cell types (epidermis, exodermis, sclerenchyma and cortex), whereas the inner stele layers (such as phloem and endodermis) showed relatively minor changes (Fig. This pattern indicates that even under non-stressed soil conditions, roots modify their gene expression compared to gel-grown roots, particularly in outer cell layers. Gene ontology (GO) analysis of these DEGs revealed the functional classes enriched in outer tissues of soil-grown roots notably include nutrient metabolism (particularly phosphate and nitrogen pathways), alongside vesicle-mediated transport, cell wall integrity, hormone-mediated signalling and defence responses, compared to axenic gel conditions (Fig. The increased expression of genes involved in nitrogen and phosphorus metabolism in outer cell layers further indicates that root cells dynamically adjust their metabolic processes to respond to fluctuating nutrient availability in soils. We also identified several micro nutrient (zinc and boron) uptake-related genes (OsZIP10 and OsBOR1) showing enhanced expression in the outer cell layers. Our scRNA-seq analysis indicates that roots use various adaptive strategies to improve nutrient uptake, including strengthening cell wall integrity, enhancing cell communication by means of hormone signalling and using vesicle-mediated transport mechanisms, in response to the heterogeneous distribution of nutrients in the soils. In our study, we used the model rice variety Xkitaake to establish our scRNA-seq resource given the wealth of functional resources available in this background including mutant collections15 and exploited in our recent study2. However, Xkitaake is a transgenic line containing the XA21 gene, which encodes a plasma membrane-localized protein that confers resistance to Xanthomonas oryzae pv. To assess the potential influence of XA21 on rice root gene expression, we conducted scRNA-seq on Kitaake genotype under both gel and soil conditions (Extended Data Fig. Cell type annotation revealed similar enrichment of DEGs in outer root cells as observed in Xkitaake (Extended Data Fig. We further validated the enriched GO terms through a comparative scRNA-seq analysis of the Kitaake genotype. The GO term enrichment patterns and associated gene expression changes (Supplementary Table 9), related to nutrient homeostasis, cell wall integrity, hormone-mediated signalling and vesicle-mediated transport were consistent between Xkitaake and Kitaake in the scRNA-seq analysis (Extended Data Fig. For defence responses, scRNA-seq analysis of Xkitaake compared to Kitaake showed relatively higher expression of defence-related genes in Xkitaake (Extended Data Fig. 4h,i), indicating that XA21 can enhance defence responses under changing growth conditions. However, when examining specific defence response genes in Kitaake under soil versus gel conditions, we also detected their induction in soil conditions (Extended Data Fig. Thus, although XA21 is not required for the enhanced defence response observed in natural soils as compared to the gel growth regime, it amplifies the defence response triggered by these growth condition changes. Hence, compared to when propagated in sterile homogeneous gel, roots grown in soils seem to adapt to their heterogeneous environment by upregulating defence, nutrient and cell wall-related gene expression across all the cell types. The outer cell layers are more responsive compared to the inner cell layers, reinforcing nutrient uptake (that is, ‘get nutrients in') and cell wall integrity, to facilitate root exploration for heterogeneous resources in soil (Fig. This cell layer-specific responsiveness also helps to protect developing roots from abiotic and biotic signals (that is, ‘keep stress out') that are unevenly distributed in natural soils (Fig. These important insights highlight the benefit of applying single-cell profiling approaches on samples grown in a natural soil environment. Root systems can adapt to contrasting soil stresses, yet how their responses are programmed at the individual cell scale remains unclear. Soil compaction reduces the ratio of air spaces versus soil particles, resulting in higher mechanical strength that impedes root growth and triggers adaptive responses6. To show how individual root cell types exposed to compaction stress modify their gene-expression profiles, scRNA-seq and spatial transcriptomic datasets were generated from roots grown at higher soil bulk density (1.6 g cm−3 compared to 1.2 g cm−3; Fig. Molecular Cartography revealed most validated cell type markers remained expressed in their target cell types under compacted soil growth conditions (Fig. This is consistent with the detected high correlation between two soil conditions across most cell layers (Extended Data Fig. b,c, Spatial expression maps of major cell type markers in transverse root sections from non-compacted (b) and compacted (c) soils. Dots represent detected mRNA molecules, colour-coded by cell type. n = 4 biological replicates for compacted soil-grown roots. d, The number of DEGs between non-compacted and compacted soil conditions for nine annotated rice primary root cell types. Exodermis and endodermis, marked by red asterisks, are the two cell types with the most DEGs, indicating that they are particularly influenced by soil compaction. e, Enriched GO terms for upregulated exodermis genes under compaction. Cell wall metabolism and ABA responses are highlighted (red arrows). f, Heatmap presenting the average of normalized gene expression for the upregulated DEGs relevant to cell wall remodelling in exodermis (top), and ABA biosynthesis in phloem-related vascular tissue (bottom). Colour bars indicate the scaled expression level in these cell types. g, Heatmap showing the spatial expression pattern of key ABA biosynthesis genes in compacted versus non-compacted soil conditions. The vascular tissues and phloem cell files are demarcated with a rectangular border highlighting the tissue-specific induction of ABA biosynthesis genes. h, Heatmap showing the spatial expression pattern of key ABA-responsive genes in compacted versus non-compacted soil conditions. The outer cell layers are marked with a rectangular border highlighting the outer tissue-specific induction of ABA-responsive genes. Next, we performed a comparative analysis to determine the most transcriptionally affected cell groups and understand the nature of their responses to soil compaction stress. We further checked the DEGs for each confidently annotated root cell type and developmental stage-enriched groups. Here 42% of DEGs were altered in a single cell type or developmental stage (Extended Data Fig. Notably, exodermis and endodermis emerged as the two cell layers particularly influenced by soil compaction, showing the highest number of DEGs (Fig. Analysing enriched GO terms for the most affected cell types, exodermis and endodermis, showed a significant association with cell wall component metabolism (Fig. The group of cell wall-related proteins with differential gene expressions included EXPANSINS (EXPA), a family of plant cell wall regulatory proteins that facilitate turgor-driven cell enlargement16. Notably, bulk RNA-seq of Xkitaake also showed similar induction of EXPA in compacted soil conditions (Supplementary Table 12 and Extended Data Fig. 7c and Supplementary Table 10), is consistent with the observed enlarged cell area for both cell types under compacted soil conditions as roots undergo radial expansion (Extended Data Fig. 7e–h and Supplementary Table 13), necessitating cell wall remodelling of outer root tissues17. A deeper analysis of the DEGs pinpointed several genes encoding xylanase inhibitors, indicative of secondary cell wall formation, given xylan's significant role as a secondary cell wall component18. In addition, xylanase inhibitors are important defence components, primarily found in the cell walls of monocots where they inhibit the hemicellulose-degrading activity of microbial xylanases. This indicates that root defence responses are also activated by soil compaction. Similar upregulation of cell wall-relevant genes was detected in the endodermis (Extended Data Figs. A group of water stress-responsive genes also showed enhanced expression in both exodermis and endodermis under compacted soil conditions (Extended Data Fig. The induction of water stress relevant genes indicates root tissues experience water stress in compacted soils, leading us to investigate the expression of genes relevant to abscisic acid (ABA), which is tightly linked to water stress21,22,23. Significantly, enriched GO terms for upregulated genes in exodermis also included the class ‘response to ABA' (Fig. We thus checked the spatial expression of ABA biosynthesis genes in our scRNA-seq dataset. We identified strong upregulation of OsAAO1 and OsNCED genes (which encode enzymes catalysing the last steps of ABA biosynthesis) in phloem-derived vascular tissue23 (Fig. We also noted induced expression of ABA-responsive genes in other outer cell layers (beside exodermis) in response to soil compaction (Fig. Moreover, we also found similar induction of ABA biosynthesis genes in our bulk RNA-seq in compacted soil conditions (Extended Data Fig. Our scRNA-seq analysis shows that ABA biosynthesis occurs predominantly in inner cell layers, whereas ABA responses are activated in outer cell layers. This aligns with published findings that ABA synthesized in the root stele moves radially outwards with water flux to activate responses in outer tissues24. Hence, our scRNA-seq dataset demonstrates how compaction stress drives coordinated, cell-specific responses to stress signals, such as ABA, progressing from the inner to the outer root cell layers. Soil compaction is known to exert water stress on roots as moisture release is reduced from the smaller soil pores25. The coordinated regulation of suberin and lignin accumulation in roots is essential to maintain the water balance for various plant species26,27. Our scRNA-seq analysis revealed upregulated expression of many lignin and suberin biosynthesis genes in outer (exodermis) and inner (endodermis) root cell layers that can form apoplastic water-impermeable barriers (Extended Data Fig. We found similar induction of several lignin and suberin biosynthesis genes in our bulk RNA-seq dataset in compacted soil conditions (Extended Data Fig. To validate our expression results, histochemical staining was performed for lignin (basic fuchsin) and suberin (fluorol yellow) in mature wild-type (WT) rice root tissues exposed to non-compacted and compacted soil conditions. Our imaging of these barrier components revealed higher lignification (Fig. 4b versus 4f) in root exodermal, endodermal and vascular cell types exposed to compacted soil conditions. To test whether compaction stress induced barrier formation is regulated by ABA, we characterized lignin and suberin levels in roots of the rice ABA biosynthesis mutant mhz5 grown in compacted soils28,29. MHZ5 expression is significantly induced by soil compaction in the phloem-related vascular tissue in our scRNA-seq dataset (Fig. In contrast to WT, mhz5 roots did not show induction of lignin and suberin levels in response to compacted soil conditions (Fig. Moreover, we also quantified lignin levels in WT and mhz5 root tips grown under both compacted and non-compacted soils, showing a significant increase of lignin in WT, whereas the mhz5 mutant showed no substantial difference under compacted conditions (Extended Data Fig. To further confirm the role of ABA in regulating barrier formation under compacted soil conditions, we analysed lignin and suberin patterns in two additional ABA biosynthetic mutants (aba1 and aba2), both of which showed minimal barrier induction in compacted soils (Extended Data Fig. 8e–l) Hence, our results show that ABA has a key role in triggering barrier formation during compaction stress conditions, similar to the radial oxygen loss barrier being induced in stagnant soil conditions30. a,b,e,f, Lignin staining (basic fuchsin, magenta, white arrowheads) of WT non-compacted (a) and compacted soil grown root (e) and suberin staining (fluorol yellow, yellow, yellow arrowheads) of WT radial root sections in non-compacted (b) and compacted soil (f) are shown. c,d,g,h, Similarly, lignin imaging of mhz5 roots in non-compacted (c) and compacted soils (g) and suberin imaging of mhz5 roots are shown in non-compacted (d) and compacted soil (h) conditions. Sections are roughly 2 cm behind the root tip. Staining experiments were repeated three times independently (n = 6 for non-compacted and n = 4 for compacted soils per experiment). i, Cumulative water loss in WT and mhz5 segments (3 cm long including the root tip) under non-compacted or compacted conditions. The models fitted are shown as a dashed line for both genotypes and growth conditions (two-phase decay). j, Radial water loss rates quantified at the time point when 50% of the water was lost from roots. Statistical comparison was done by a one-tailed t-test. k, Schematics illustrating rice root cell type-specific responses to soil compaction stress. Phloem relevant vascular tissue upregulates the expression of ABA biosynthesis genes. ABA targets outer root cell types, potentially following the outward water flow. ABA reaches outer cell layers, such as the exodermis, to induce water-impermeable barriers. ABA promotes suberin and lignin accumulation, forming water-impermeable barriers that enhance structural support, reduce radial water loss and protect root systems under compaction stress. Panel k adapted with permission from Xiaoying Zhu. One key link between secondary cell wall formation in barriers is the enhanced cell wall stiffness that helps to protect roots from soil mechanical stress. The higher expression and accumulation of lignin and suberin in the endodermis prompted us to analyse the cell wall stiffness of endodermal cells under both non-compacted and compacted soil conditions. Our phonon imaging revealed increased stiffness in the endodermal cell layer under compacted soil conditions (Extended Data Fig. 7i–n), providing direct evidence of rice roots enhancing cell wall rigidity to deal with mechanical stress. On the basis of the induction of key water stress-responsive genes and enhanced barrier formation under compaction, we sought to delineate the actual role of the barriers in dealing with water stress in compacted soils. To evaluate this, we performed radial water loss experiments using WT and mhz5 root tips grown in either non-compacted or compacted soil conditions. Three-centimetre-long root tips were excised from soil-grown roots and kept in a humidity-controlled environment to quantify the weight loss, as an indirect measurement of radial water loss. WT root segments grown in non-compacted soil conditions lost half of their water content in just 17 min, whereas root tips exposed to compaction stress took almost 25 min. Hence, cumulative water loss in compacted root tips was roughly 1.5 times slower than in non-compacted root tips (Fig. This reduction in radial water loss after exposure to compaction stress is not observed in mhz5 mutant root tips (Fig. Our results show that ABA has a key role in triggering adaptive responses to compaction stress, which include induction of lignin and suberin barriers in the exodermis and stele cell types, which collectively act to prevent root radial water loss. Our scRNA-seq approach provides spatial insights into the cascade of signalling events taking place in specific cell types when roots are exposed to soil compaction. In response to this soil stress phloem cells upregulate expression of biosynthesis genes for the abiotic stress signal ABA, which then targets outer root cell types such as the exodermis to form water-impermeable barriers to reduce root moisture loss (Fig. Our study shows how cellular-resolution transcriptomic approaches can provide unprecedented new insights into root–soil interactions and adaptive responses. Most root stress studies performed so far have been conducted in aseptic growth systems such as gel-based media. However, plant roots are normally exposed to a heterogeneous soil environment, encompassing a range of textures, microbiomes and levels of moisture and nutrients32. Single-cell transcriptomics showed key transcriptional differences among cell types when grown in a natural soil system versus an axenic gel system. Transcriptional differences were predominantly confined to outer root tissues, whereas inner root cell types showed limited response. Upregulated genes in soil-grown roots included NB-ARC, WRKY48 and those encoding cupin domain proteins and strip rust proteins, known to respond to bacterial, viral and fungal pathogens. Transcript levels of nucleotide-binding leucine-rich repeat genes (NLRs) are normally low in the absence of pathogens. The elevated spatial expression of NLRs indicates that outer root cell types are exposed to the soil microbes when cultured in real soil environments. Alternatively, plant roots may deliberately upregulate immune response component expression in outer root cell layers to prepare for the biotic heterogeneity in soil environments. This expression pattern illustrates how plant roots sense diverse elements within natural soils and change molecular responses enhancing readiness to biotic challenges, nutrient transport to drive growth and development to explore the soil environments effectively. Thus, sensing of the external environment concomitantly with cell signalling and cellular reprogramming collectively orchestrate the growth and adaptation of plant roots in soil environments. Root cell types growing in soils have to sense and respond to not only biotic but also abiotic stresses. Our study also explored how root cell types responded to the model abiotic soil stress, compaction. Radial expansion of outer root cell types (including exodermis, cortex and epidermis) represents a hallmark of plant adaptive growth response to soil compaction stress32 (Extended Data Fig. This adaptive growth response, primarily driven by radial cortical cell expansion, will necessitate the remodelling of cortical cell walls and, as a result of this expansion, all surrounding outer root cell layers would also undergo cell wall modifications. Consistent with this, our scRNA-seq dataset revealed enrichment of cell wall remodelling gene classes, including EXPA and GRPs (glycine rich protein genes), in outer cell types (Extended Data Fig. Indeed, our phonon imaging (Brillouin microscopy) provides direct evidence that rice roots reinforce cell wall rigidity at barriers to support and protect root systems and plants under compaction stress. Soil compaction not only imposes mechanical stress on roots, but also reduces water and nutrient absorption. The latter is due, in part, to compacted soil pores being more difficult for roots to extract water from28,33, creating water stress-like conditions. Consistent with this, our scRNA-seq dataset revealed upregulation of key ABA biosynthesis genes in root vascular cell types in response to compaction stress (Fig. Elevated ABA levels target outer root layers, potentially through the outward water flow24, triggering induced expression of ABA-responsive genes in these cell types (Fig. ABA-dependent root adaptive responses to compaction stress included elevated lignin and suberin accumulation in water barriers and stele cell types at the root maturation zone, as opposed to the younger regions (Extended Data Fig. We demonstrate that ABA-dependent formation of these barriers facilitates water retention in root tips during compaction stress. Our study provides direct evidence of cell wall remodelling through increased expression of suberin and lignin biosynthesis genes, specifically in the exodermis and endodermis, as revealed by scRNA-seq data. This enhanced accumulation of suberin and lignin was further validated using fluorescent dye staining and direct quantification of lignin in rice roots. Our scRNA-seq analysis revealed many aspects of cell wall remodelling closely linked to cell wall properties, morphology and growth. Beyond EXPA genes, we examined the expression of cellulose synthase (CESA) and xyloglucan biosynthesis genes (Extended Data Fig. Both groups showed enhanced expression, with CESAs slightly induced in sclerenchyma and xylem cells, whereas xyloglucan biosynthesis genes showed strong but less cell type-specific patterns. These findings indicate that distinct aspects of cell wall remodelling are regulated by different cell type-specific mechanisms. Besides ABA, ethylene and auxin are also reported to have roles in root responses to soil compaction. Comparative heatmaps of ethylene biosynthesis and signalling-related genes under compacted versus non-compacted soil conditions (Extended Data Fig. 9c,d,f) show the induction of several prominent ERF and EIL genes. However, no cell type-specific expression patterns were observed for these genes. Similarly, auxin signalling genes show increased expression under soil compaction (Extended Data Fig. 9a,b,e), but without cell type-specific induction. These findings indicate that ABA, rather than ethylene or auxin, drives cell type-specific gene expression changes in response to soil compaction. Is the ABA-mediated radial water loss prevention functionally connected to root growth in compacted soils? As our previous study has revealed that mhz5, as well as other ABA biosynthesis mutants, aba1 and aba2, all have relatively longer roots than WT in compacted soils28, we propose that increased water loss triggers enhanced root elongation. This may be a direct consequence of impaired cortical radial expansion and potentially reflects a root strategy to rapidly explore water resources. In summary, our single-cell and spatial transcriptomics data provide insights into how root cells sense and respond to their biotic soil environment and abiotic stresses such as compaction in a cell type-specific manner. The single-cell resolution of our approaches has been instrumental in pinpointing key genes and cell types, pathways and processes, stress signals and inter-cellular signalling mechanisms that enable roots to adapt to growth in soils. Leveraging these new soil-grown root datasets will underpin efforts to develop crops more resilient to complex edaphic stresses and contribute to future-proofing plants against challenging environmental conditions. To ensure that the presence of XA21 does not influence the gene-expression trends observed in our scRNA-seq analysis, we also included the non-transgenic Kitaake line for comparison. Rice seedlings were then inserted into Yoshida's media solidified with 0.15% gellan gum (Gelzan, Caisson)35, with the embryo positioned facing upwards. Rice seeds were kept at 30 °C in the dark for 2 or 3 days until they germinated. For gel-based growth conditions, germinated rice seedlings were then transferred to a Percival growth chamber set to 28 °C, and constant light (2,000 LUX) for 2–3 days before harvesting. To allow packing of soils to certain bulk densities, the soils were lightly sprayed with sterilized water and mixed thoroughly. Non-compacted soil condition was packed up to 1.2 g cm−3 (1.2 bulk density (BD)), Compacted soil was pressed to make 1.6 g cm−3 (1.6 BD). Soils were packed in three-dimensionally printed mesocosms at bulk density of 1.2 or 1.6 BD and then saturated with sterilized water14. In the soils (both compacted and non-compacted) used in our experiments, excess water was drained through gravitational pull to mimic the near-field capacity conditions. Germinated rice seedlings (maximum of four seedlings per mesocosm) with equivalent length radicles (roughly 0.5 cm) were placed below the soil surface (10 mm), and then grown in a Percival growth chamber set to 28 °C and constant light (2000 LUX) for 2–3 days before harvesting. Samples were frozen in liquid nitrogen and stored at −80 °C, and then processed by grinding with a blue homogenization pestle. RNA was isolated using the Zymo MagBead RNA Isolation kit according to the manufacturer's protocol (Zymo). RNA was used as input into the Lexogen QuantSeq 3′ FWD RNA-Seq library preparation procedure according to the manufacturer's protocol, using the unique molecular identifier (UMI) PCR add-on kit. Libraries were indexed and pooled on an Illumina NextSeq. Reads were aligned to Michigan State University Rice genome v.7 with the STAR aligner36, deduplicated using UMI-Tools37 and counted with HTSeq-Count. For rice seedling harvesting, gel-grown rice seedlings were directly pulled out from the growth media and root tips were cut in the enzyme solution within the optimal osmotic conditions. For soil-grown rice seedlings, the three-dimensionally printed mesocosms were opened and rinsed with gentle water flow. The seedlings were exposed and further rinsed with a gentle water flow to remove attached soil particles. Gentle brushing with a small paint brush was also carried out to remove the remaining soil particles. For gel-based scRNA-seq protoplasting sample, roughly 1-cm root tips were harvested from 15–40 roots, chopped with sharp razor for 1 min and then placed into a 35-mm petri dish containing a 70-μm cell strainer and 4.5 ml enzyme solution (4% [w/v] cellulase (ONOZUKA R-10, GoldBio), 1.5% Macerozyme R10 (GoldBio), 0.24% Pectolyase (Sigma P3026), 0.4 M mannitol, 20 mM MES (pH 5.7), 20 mM KCl, 10 mM CaCl2, 0.1% bovine serum albumin and 0.000194% (v/v) 2-mercaptoethanol). The digestion was incubated on an 85-rpm shaker at 28 °C for 2.5 h with extra pipette mixing every 30 min. The resulting cell solution was filtered twice through 40-μm cell strainers, transferred into a Falcon Round-Bottom Polystyrene Test Tubes and then centrifuged for 5 min at 500g in a swinging bucket rotor. The pellet was washed with 2 ml of washing solution (0.4 M mannitol, 20 mM MES (pH 5.7), 20 mM KCl, 5 mM CaCl2, 0.1% bovine serum albumin and 0.000194% (v/v) 2-mercaptoethanol), and centrifuged again at 500g for 3 min. The washing step was repeated for one more time and the pellet resuspended in the washing solution (normally 50–80 µl) without CaCl2 at a concentration of roughly 2,000 cells per µl. Cell concentration was counted using a C-chips disposable hemocytometer (Fuchs Rosenthal, 20 µl, VWR, catalogue no. For soil-based scRNA-seq protoplasting, the procedure mirrors that of gel-based RNA-seq protoplasting, with modifications to chopping time (reduced to 45 s) and digestion time (extended from 2.5 to 3 h). These adjustments aim to enhance protoplast yield without introducing excessive debris. Despite careful washing of soil from root tips, a significant number of epidermal cells were probably removed, potentially altering the proportions of trichoblast and atrichoblast cells under different growth conditions. We conducted root trichoblast cell-specific reporter image analysis in gel, non-compacted and compacted soil conditions and we did not see a difference in the number of cells expressing the proCSLD1:VENUS-N7 reporter38. Raw sequencing reads underwent demultiplexing from Illumina BCL files to generate FASTQ files for each sample using CellRanger mkfastq (v.3.1.0, 10X Genomics). Subsequently, reads were aligned to the Oryza sativa genome BSgenome object (BSgenome.Osativa.MSU.MSU7) along with the MSU7 gene annotation file. This alignment was carried out using the scKB script within the COPILOT preprocessing pipeline9, which integrates kallisto39 and bustools40,41. Quality filtering of cells was performed with the R package COPILOT9. COPILOT uses a non-arbitrary scheme to eliminate empty droplets and low-quality cells, using a 5% mitochondrial expression threshold as the criterion for searching the initial cut-off defining low-quality cells (parameter mt.threshold set to 5). A single iteration of COPILOT filtering (parameter filtering.ratio set to 1) was applied, effectively segregating high-quality cells from the background, as indicated by barcode rank plots. To address issues related to doublets and outliers, the resulting high-quality cells underwent further filtering, removing the top 1% of cells based on UMI counts. Putative doublets were identified and removed using DoubletFinder42 with the estimated doublet rate from the 10X Genomics Chromium Single Cell 3′ Reagent Kit user guide. Individual processing and examination of samples were performed, followed by data normalization using SCTransform43. As a standard step in scRNA-seq data processing, we identified protoplasting-induced genes using bulk RNA-seq (Supplementary Table 2). These genes were excluded from our analysis. Specifically, we conducted bulk RNA-seq comparisons between intact roots and digested roots to identify general protoplasting-induced genes. Furthermore, we compared roots digested for 2.5 versus 3 h to account for digestion time effects and further minimize their impact on gene-expression trends in the gel versus soil comparison (Extended Data Fig. All detected genes, excluding those associated with mitochondria, chloroplasts or affected by protoplasting (absolute log2 fold change greater than or equal to 2), were retained for analysis (Supplementary Tables 2, 5, 8 and 10). PCA was executed by calculating 50 principal components using the RunPCA function (with approx = FALSE). Subsequently, uniform manifold approximation and projection (UMAP) nonlinear dimensionality reduction was computed by means of the RunUMAP function using all 50 principal components with default parameters. These processing steps are detailed and documented in Jupyter notebooks (provided on GitHub at https://github.com/zhumy09/scRNA-seq-for-rice). 1, including two previously published samples (tz1, tz2; Supplementary Data 1). For the integrated object containing eight samples shown in Figs. These processing steps are detailed and documented in Jupyter notebooks (provided on GitHub at https://github.com/zhumy09/scRNA-seq-for-rice). The cell type annotation for both integrated objects was based on markers (Supplementary Table 3) that have been previously validated and show specific local expression on the atlas UMAP. Marker gene-expression z-scores were calculated depending on clustering. Clusters were defined using the Seurat FindClusters function by testing the modularity parameter, ranging from res = 2 (low) to res = 300 (high), until the reasonable cluster numbers were reached. Coarse and finely-resolved clusters were annotated by comparing average marker gene z-scores. Cells annotated with the same cell identity by both resolutions were considered confidently annotated, forming the consensus annotation. This combination effectively annotated rare cell types while capturing major cell types given that high resolution and low noise provided by low-resolution are balanced. New reference expression profiles for each cell type were built by averaging the expression values for cells in the consensus annotation. All cells were then re-annotated using the correlation-based approach, which calculates Pearson correlation coefficients between each cell and reference expression profiles for cell types, assigning each cell the cell type with the highest correlation coefficient. To eliminate the potential occurrence of specific cell groups being filtered out during our COPILOT-based scRNA-seq data preprocessing, possibility as a result of induced cell stresses, we also conducted an examination of the cell type identities for the low-quality cells and found no enrichment of any particular cell type (Supplementary Data 9). This confirmed that we have inclusively incorporated high-quality cells representative of all major cell types in an unbiased manner. For developmental-stage annotation, correlation annotation compared each cell from scRNA-seq to bulk data from morphologically defined sections (Supplementary Data 2) for both the 12-sample WT atlas and the 8-sample integrated object grown in gel versus soil. Jupyter notebooks illustrating the gene expression plotting process are available on GitHub at https://github.com/zhumy09/scRNA-seq-for-rice. Rice root epidermal cells were extracted from the integrated Seurat objects (12 gel-grown Xkitaake). The learn_graph and order_cell functions in Monocle3 package were used to generate pseudotime metadata. Owing to the complexity of defining epidermal principal points, we opted to calculate pseudotime values separately for atrichoblast and trichoblast cells. Subsequently, these values were merged back into the pseudotime metadata. Furthermore, we manually delineated ten developmental groups. Differential expression analysis for genes was conducted using the ‘graph_test' function within Monocle3. Jupyter notebooks illustrating the pseudotime analysis process are available on GitHub at https://github.com/zhumy09/scRNA-seq-for-rice. Pseudobulk methods, which aggregate cell-level counts for subpopulations of interest on a per-sample basis, have been identified as top performers for cross-condition comparisons in scRNA-seq48,49. Hence, we used a pseudobulk approach implemented in muscat (multi-sample multi-group scRNA-seq analysis tools)48. Pseudobulk expression profiles for individual cell types in each sample were aggregated for these subpopulations by summing the raw counts (RNA assay) using the ‘aggregateData' function. A gene was considered differentially expressed in a given subpopulation if the false discovery rate adjusted P value was less than or equal to 0.05, absolute fold change was greater than or equal to 1.5, and detection frequency was greater than or equal to 10% in any of the included conditions. The full tables containing gene-expression trends and GO term enrichment information for all detected genes and GO terms from the scRNA-seq data comparison across various growth conditions is available in Supplementary Data 10. Jupyter notebooks illustrating the pseudobulk differential expression analysis process are available on GitHub at https://github.com/zhumy09/scRNA-seq-for-rice. Root parts of rice seedlings were isolated and fixed in a paraformaldehyde (PFA)-Triton-X solution: 4% [w/v] PFA (Sigma, catalogue no. The fixation was conducted within a 20-ml glass scintillation vial (Fisher Sci, catalogue no. The vial, containing rice roots, was placed on ice under a vacuum chamber. Subsequently, the rice roots were rinsed with 1× PBS and dehydrated with an ethanol gradient (15, 30, 50, 70, 80, 90 and 100%), each concentration for 1 h on ice. The roots were then kept in 100% ethanol overnight. For clearing the roots, a mixture of ethanol and Histo-clear (VWR, catalogue no. The roots were included overnight at precisely 60 °C. The embedded roots were then mounted into plastic tissue embedding moulds (VWR, catalogue no. 15160-339) with properly adjusted orientation using flamed forceps. These root tissue sections were transferred to cover slips provided by Resolve Biosciences, and the cover slip was placed in a slide dryer at 42 °C overnight. To prevent detachment issues, the cover slip could be placed in a 60 °C incubator for 5–30 min before proceeding to the next step. This was followed by rehydration with an ethanol gradient (100, 90, 80, 70, 50, 30%). The tissue was then permeabilized with proteinase K (Invitrogen, catalogue no. The tissue was also refixed with a 4% [w/v] PFA solution and acetylated with an acetylation solution: 0.1 M triethanolamine (Sigma, 90279), 0.5% [v:v] acetic anhydride (Sigma, catalogue no. Finally, SlowFade antifade Mountant (Invitrogen, catalogue no. A slide box was used to store the cover slips with root tissue, tightly sealed with parafilm and shipped with dry ice to Resolve Biosciences for messenger RNA (mRNA) detection and imaging, with Molecular Cartography technique. In brief, preserved mRNA molecules were hybridized with specifically designed probes based on sequence complementarity. Each probe contained a long tail with many binding sites for various fluorescent dyes. These long tails facilitated several rounds of imaging of the same probe with different fluorescent colours, generating a unique barcode for each individual gene. The probe–mRNA complexes were sequentially coloured, imaged and decoloured for several imaging rounds. Fluorescent signal images captured on the root tissue sections were processed to identify individual mRNA molecules. Detected mRNAs corresponding to the same gene were assigned a unified identity and false-coloured for clear visualization and presentation. The Resolve Biosciences dataset comprises both stained root images and transcript detection profiles. Staining images using Calcofluor white to visualize cell boundaries were processed using the ImageJ app provided by Resolve Biosciences. Transcript information was stored in a .txt file, which could be loaded using the Molecular Cartography plugin. The resulting mRNA detection images were saved as screenshots. Subsequently, image brightness and contrast were adjusted using the auto-setting in ImageJ, for presentation. It is notable that the detected mRNA levels in roots grown in compacted soils were considerably lower compared to those grown in gel and non-compacted soil conditions. We suspect this may be due to reduced fixation efficiency. Roots grown in compacted soils undergo radial expansion, enhanced barrier formation and increased mucilage secretion (data not shown), all of which probably hinder formaldehyde penetration into inner cell layers. As a result, mRNA preservation efficiency is diminished, particularly for markers in the stele tissue. Despite these challenges, we successfully identified roughly 20 robust cell type-specific markers under compacted soil conditions, as detailed in Supplementary Data 8. Root tips (roughly 1 cm) from Xkitaake rice varieties were harvested from gel, non-compacted and compacted soil conditions, and flash-frozen in liquid nitrogen. For RNA isolation, root tips were ground to a fine powder in liquid nitrogen using a mortar and pestle, followed by the addition of 1 ml of RLT buffer to the powdered tissue. RNA was then isolated and purified using the RNeasy Mini Kit (Qiagen) according to the manufacturer's protocol. Raw reads were processed by removing adapter sequences and filtering out low-quality nucleotides (base quality lower than 5). HISAT2 was used to align reads to the Oryza sativa (Japonica) genome, and gene-expression levels were quantified using the fragments per kilobase of transcript sequence per million mapped reads) method. Differential gene expression (log2 fold change greater than or equal to 1.0) was analysed through read count normalization, model-dependent P value estimation (Padj ≤ 0.05), and false discovery rate (FDR) adjustment. A 4% agarose gel was prepared and poured into a square petri dish, allowing it to cool for 2 min. Rice roots were then embedded in the gel for 45 min. Transverse sections, each with a thickness of 500 µm at 0.7 cm from the root tips, were transferred to slides. Calcofluor white staining, at a concentration of 10 mg ml−1, was applied to the transverse sections on slides for 1 min. After aspirating the staining solution, a drop of sterilized water was added on top of the sections. The root transverse sections were imaged using Zeiss 880 Confocal microscopy (excitation wavelength 405 nm, emission wavelength 410–585 nm). For data collection with the confocal microscopy, we used Zen 2009 v.6.0.0.303. The images underwent the following processing steps: (1) Gaussian blur with x-sigma, y-sigma and z-sigma set to 1 µm. The resulting mesh files, representing the sample structure, were then manually segmented to identify individual cells. Rice roots (WT (cv Nipponbare), mhz5, aba1 and aba2) grown for 3 days under ± compaction conditions were gently removed from the 3D-printed soil columns, cleaned using deionized water and a thin brush, and embedded in 4% melted agarose. The agarose blocks containing the roots were then positioned in a vibratome (Leica), cut into 100-µm thick primary root cross-sections (1–1.5 cm or 2–2.5 cm behind the root tip), and stored in 20% ethanol. For lignin staining, the cross-sections were incubated for 10 min in a 0.2% solution of basic fuchsin dissolved in ClearSee54 and mixed 1:1 with aqueous calcofluor white to stain cell walls55. The stained cross-sections were quickly rinsed with ClearSee and then washed for 1.5 h in fresh ClearSee, replacing the solution halfway through. For suberin staining, the primary root cross-sections were stained for 10 min in a 0.01% fluorol yellow solution dissolved in pure ethanol, prepared from a 1% fluorol yellow solution dissolved in dimethylsulfoxide. The stained cross-sections were rinsed once with deionized water and incubated for 10 min in aqueous calcofluor white. Finally, the cross-sections were washed 2–3 times in 50% ethanol for 20 min. For confocal imaging, primary root tip cross-sections stained for lignin or suberin were mounted in a drop of ClearSee or 50% glycerol, respectively, and positioned on a Leica SP5 inverted confocal microscope. For both basic fuchsin or fluorol yellow with calcofluor white, a sequential scanning was configured with the corresponding settings mentioned above. The dried root tips (WT (cultivar Nipponbare) and mhz5 mutant root tips) were grinded into a fine powder using a microcentrifuge tube with two metal beads (3 mm) for 1 min 30 s at 20 Hz, and then solvent extracted with sequential extractions of water (1 ml, 30 min, 98 °C), ethanol (1 ml, 30 min, 76 °C), chloroform (1 ml, 30 min, 59 °C) and acetone (1 ml, 30 min, 54 °C). The extract-free samples were dried under vacuum (overnight, 50 °C) and considered as cell wall residue. Acetyl bromide lignin was determined as previously described56 with modifications. The absorption was measured at 280 nm with a microtitre plate reader (Microplate-reader SpectraMax 250, Sopachem), SoftMax Pro v.5 was used for collecting data and applying the extinction coefficient for grasses 17.75 g l−1 cm−1. Two technical replicates of each biological replicate were analysed. Rice seedlings (either WT or mhz5), grown for 3 days under ±compaction, were gently removed from the three-dimensionally printed soil columns. They were then delicately brushed with deionized water to remove soil particles, and the diameter of each seminal root was measured. The primary root of each seedling (4–6 seedlings were used for each replicate) was cut into a 3-cm segment, including the root tip. After gently blotting with paper towels, each segment was positioned inside a five-digit balance closed chamber (Automatic balance, Mettler Toledo) over a thin nylon mesh. A constant relative humidity was maintained by adding bags with silica gel, which maintained the relative humidity inside the chambers at 30–35%. The silica gel was replaced after every three replicates. The temperature and relative humidity were monitored using a digital logger. Phonon microscopy is an optical elastography technique that uses the phenomenon of Brillouin scattering to probe mechanical information in biological specimens with subcellular resolution. Phonon microscopy photoacoustically stimulates GHz frequency coherent acoustic phonons that, as they propagate through the specimen, periodically modulate the local refractive index that induces resonant optical scattering of a probe laser57. Through conservation of energy, the Brillouin scattered probe photons are frequency shifted by the phonon frequency (the so-called Brillouin frequency shift) and this can be detected either using a high-resolution spectrometer as with Brillouin microscopy58, or interferometrically in the time domain59. for normal optical incidence where n is the refractive index and λprobe is the optical probing wavelength. An absolute measurement of M requires knowledge about the mass density; however, refractive index and mass density of plant cells have been shown to vary substantially less than inter-specimen and inter-environmental variation in elasticity60. In this work, we use the relative difference in Brillouin frequency shift (\(\Delta {f}_{{\rm{B}}}\)) between the cell wall and the water:ethanol filled cytoplasm as a proxy for the relative difference in cell wall elasticity in compacted and non-compacted conditions. It is worth noting that the longitudinal modulus should not be directly compared with the Young's modulus, as the two describe elasticity at very different time and frequency scales (for example, Hz to kHz deformations compared with GHz); however, it has been shown that there is an empirical relationship between the two quantities61. The harvested and cleaned root tips (1.5 cm) were embedded in 4% molten agarose within a three-dimensionally printed root tip cassette. Agarose blocks containing the root tips were sectioned transversely into 50-µm slices. These root cross-sections were fixed in 20% ethanol for phonon imaging experiments. A cross-section was laid flat onto a photoacoustic transducer (200-nm thick partially transparent metal:dielectric cavity on a 170-µm sapphire cover slip), covered in roughly 50–100 µl of water:ethanol medium and then topped with a glass cover slip. Once placed into the phonon microscope, a region of interest was selected (for example, the endodermis) and a 2D raster scan was performed. A phonon time-of-flight signal was detected at each spatial pixel position, and the relative Brillouin frequency shift (ΔfB) and the acoustic attenuation (αB) were measured for each pixel using a fast Fourier transform and wavelet transform, respectively (Extended Data Fig. The spatial resolution of the technique will be determined by the optical diffraction limit (a function of optical wavelength and numerical aperture), and in this case was roughly 300 nm. To isolate the endodermal region of interest, the Brillouin and attenuation maps were manually segmented based on positioning, morphology and size. From these segmented datasets, ΔfB versus αB cluster maps were generated and then segmented using a two-component Gaussian mixture model. This grouped the data into two clusters that were labelled ‘background' and ‘cell wall'. Intervals of roughly 70% confidence were determined within these clusters and mean ΔfB and αB values were calculated. Using the above methodology, we report in Extended Data Fig. 7n that the relative Brillouin frequency shifts in compacted endodermal cell walls are statistically significantly greater than the equivalent cell walls grown in non-compacted conditions (P < 0.0001). Furthermore, the measurements extracted from the cytoplasm regions can be used as a control, and a Yuen's t-test indicates that the two groups are not statistically significantly different (P > 0.05). These data indicate that the compacted cell walls have greater elasticity than those grown in non-compacted soils. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. scRNA-seq data for Xkitaake and Kitaake roots grown under gel and soil conditions is available at National Center for Biotechnology Information (NCBI) BioProject PRJNA1055099 (GSE251706). 8 (PMID 33824350) is available at NCBI BioProject PRJNA706435 and PRJNA706099. Bulk RNA-seq data for developmental-stage annotation is available at NCBI BioProject PRJNA1082669 (GSE260671). Bulk RNA-seq data for protoplasting-induced genes is available at NCBI BioProject PRJNA1194134 (GSE283509). Bulk RNA-seq data for Xkitaake roots grown under compacted and non-compacted soil conditions are available at NCBI BioProject PRJNA1193632 (GSE283428). Source data are provided with this paper. Gene accession number information is available in Supplementary Table 14. Supplementary tables are provided with this paper. Supplementary Data 1–10 are available on the Nature Figshare platform at https://doi.org/10.6084/m9.figshare.25146260. The processed scRNA-seq for gel-grown rice roots is now publicly accessible through a user-friendly platform hosted on Shiny (https://rice-singlecell.shinyapps.io/orvex_app/). Karlova, R., Boer, D., Hayes, S. & Testerink, C. Root plasticity under abiotic stress. Zhu, M., Taylor, I. W. & Benfey, P. N. Single-cell genomics revolutionizes plant development studies across scales. Nolan, T. M. & Shahan, R. Resolving plant development in space and time with single-cell genomics. Stoessel, F., Sonderegger, T., Bayer, P. & Hellweg, S. Assessing the environmental impacts of soil compaction in life cycle assessment. Pandey, B. K. & Bennett, M. J. Uncovering root compaction response mechanisms: new insights and opportunities. Shahan, R. et al. A single-cell Arabidopsis root atlas reveals developmental trajectories in wild-type and cell identity mutants. Zhang, T. Q., Chen, Y., Liu, Y., Lin, W. H. & Wang, J. W. Single-cell transcriptome atlas and chromatin accessibility landscape reveal differentiation trajectories in the rice root. Hsu, C.-W., Shahan, R., Nolan, T. M., Benfey, P. N. & Ohler, U. Protocol for fast scRNA-seq raw data processing using scKB and non-arbitrary quality control with COPILOT. Liu, Q. et al. Transcriptional landscape of rice roots at the single-cell resolution. Guilliams, M. et al. Spatial proteogenomics reveals distinct and evolutionarily conserved hepatic macrophage niches. Guillotin, B. et al. A pan-grass transcriptome reveals patterns of cellular divergence in crops. & Sturrock, C. Non-invasive imaging of rice roots in non-compacted and compacted soil. Ethylene modulates cell wall mechanics for root responses to compaction. Zhang, B. et al. Control of secondary cell wall patterning involves xylan deacetylation by a GDSL esterase. Huang, G. et al. AUXIN RESPONSE FACTORS 6 and 17 control the flag leaf angle in rice by regulating secondary cell wall biosynthesis of lamina joints. The receptor kinase OsWAK11 monitors cell wall pectin changes to fine-tune brassinosteroid signaling and regulate cell elongation in rice. Zhu, J. K. Abiotic stress signaling and responses in plants. Bharath, P., Gahir, S. & Raghavendra, A. S. Abscisic acid-induced stomatal closure: an important component of plant defense against abiotic and biotic stress. Auler, P. A. et al. Metabolism of abscisic acid in two contrasting rice genotypes submitted to recurrent water deficit. Hydraulic flux-responsive hormone redistribution determines root branching. Bengough, A. G., McKenzie, B. M., Hallett, P. D. & Valentine, T. A. Root elongation, water stress, and mechanical impedance: a review of limiting stresses and beneficial root tip traits. Baxter, I. et al. Root suberin forms an extracellular barrier that affects water relations and mineral nutrition in Arabidopsis. Cantó-Pastor, A. et al. A suberized exodermis is required for tomato drought tolerance. Ethylene inhibits rice root elongation in compacted soil via ABA- and auxin-mediated mechanisms. Ethylene responses in rice roots and coleoptiles are differentially regulated by a carotenoid isomerase-mediated abscisic acid pathway. Shiono, K. et al. Abscisic acid is required for exodermal suberization to form a barrier to radial oxygen loss in the adventitious roots of rice (Oryza sativa). Shinozaki, K. & Yamaguchi-Shinozaki, K. Gene expression and Signal transduction in water-stress response. Ogorek, L. L. P., Gao, Y., Farrar, E. & Pandey, B. K. Soil compaction sensing mechanisms and root responses. Pandey, B. K. et al. Plant roots sense soil compaction through restricted ethylene diffusion. Jain, R. et al. Genome sequence of the model rice variety KitaakeX. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & Sudbery, I. UMI-tools: modeling sequencing errors in unique molecular identifiers to improve quantification accuracy. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. Melsted, P. et al. Modular, efficient and constant-memory single-cell RNA-seq preprocessing. Melsted, P., Ntranos, V. & Pachter, L. The barcode, UMI, set format and BUStools. McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Stuart, T. et al. Comprehensive integration of single-cell data. The single-cell transcriptional landscape of mammalian organogenesis. Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Crowell, H. L. et al. muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. Confronting false discoveries in single-cell differential expression. & Smyth, G. K. Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation. & Peterson, H. gprofiler2—an R package for gene list functional enrichment analysis and namespace conversion toolset g:Profiler. Villanueva, R. A. M. & Chen, Z. J. ggplot2: elegant graphics for data analysis. Using positional information to provide context for biological image analysis with MorphoGraphX 2.0. eLife 11, e72601 (2022). Sexauer, M., Shen, D., Schön, M., Andersen, T. G. & Markmann, K. Visualizing polymeric components that define distinct root barriers across plant lineages. Ursache, R., Andersen, T. G., Marhavý, P. & Geldner, N. A protocol for combining fluorescent proteins with histological stains for diverse cell wall components. De Meester, B. et al. Tailoring poplar lignin without yield penalty by combining a null and haploinsufficient CINNAMOYL-CoA REDUCTASE2 allele. Gusev, V. E. & Ruello, P. Advances in applications of time-domain Brillouin scattering for nanoscale imaging. Prevedel, R., Diz-Muñoz, A., Ruocco, G. & Antonacci, G. Brillouin microscopy: an emerging tool for mechanobiology. Pérez-Cota, F. et al. Picosecond ultrasonics for elasticity-based imaging and characterization of biological cells. Samalova, M. et al. Hormone-regulated expansins: expression, localization, and cell wall biomechanics in Arabidopsis root growth. Scarcelli, G. et al. Noncontact three-dimensional mapping of intracellular hydromechanical properties by Brillouin microscopy. He, A. Roeder, K. Birnbaum, R. Shahan and J. Scharwies for critical reading of the manuscript. We thank S. Bhattacharya and J. Kläver for their valuable support on our spatial transcriptomic experiments with Resolve Biosciences. We thank Xiaoying Zhu for drawing and editing the anatomical diagrams. We thank B. Cole for sharing the codes to generate 3D scRNA-seq UMAP video. We thank the staff of the Duke Phytotron for plant care, and the Duke Center for Genomic and Computational Biology for Illumina sequencing services. This study was supported by the Howard Hughes Medical Institute (P.N.B. Biotechnology and Biological Sciences Research Council (BBSRC) Discovery Fellowship grant no. D.M.O is indebted to the Research Foundation Flanders (FWO; grant no. BB/Z514482/1) and Horizon Europe ERC Starting grant (no. acknowledges funding from the Royal Academy of Engineering Research Fellowships scheme (grant no. F.P.-C. acknowledges funding from the Royal academy of Engineering (grant no. These authors contributed equally: Mingyuan Zhu, Che-Wei Hsu Department of Biology, Duke University, Durham, NC, USA Mingyuan Zhu, Che-Wei Hsu, Isaiah W. Taylor, Medhavinee Mijar, Trevor M. Nolan & Philip N. Benfey Howard Hughes Medical Institute, Duke University, Durham, NC, USA Mingyuan Zhu, Che-Wei Hsu, Isaiah W. Taylor, Medhavinee Mijar, Trevor M. Nolan & Philip N. Benfey Lucas L. Peralta Ogorek, Lokesh Verma, Poonam Mehra, Malcolm J. Bennett & Bipin K. Pandey VIB Center for Plant Systems Biology, Ghent, Belgium You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar analysed single-cell transcriptomics and bulk RNA-seq data. performed lignin, suberin imaging and relative water loss experiments. performed confocal images on rice root sections and cell area quantification. Correspondence to Malcolm J. Bennett, Philip N. Benfey or Bipin K. Pandey. was the cofounder and Chair of the Scientific Advisory Board of Hi Fidelity Genetics, Inc., a company that works on crop root growth. The other authors declare no competing interests. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, UMAP with annotations for rice root developmental stages. The cells labelled as “Maturation-1” and “Maturation-2” cannot be distinguished at this stage due to the current limitations in our knowledge. The dataset, tz1 and tz2 are from Zhang et al. The relatively low correlation observed between “sc_” and “tz” samples could be attributable to differences in cultivars and growth conditions. e, Heatmap showing gene expression pattern during differentiation of rice root epidermal cells. Three genes with different expression enrichment timing are highlighted. f-h, Expression curve of selected trichoblast markers along the pseudotime trajectory. i-k, UMAP showing the expression pattern of three selected epidermal cell specific genes. a, Schematics illustrating the experimental procedures of spatial transcriptomics. Rice roots were fixed with formaldehyde and sectioned to a thickness of 10 μm. Preserved mRNA molecules were hybridized with specifically designed probes based on sequence complementarity. Each probe contained a long tail with multiple binding sites for various fluorescent dyes. These long tails facilitated multiple rounds of imaging of the same probe with different fluorescent colors, generating a unique barcode for each individual gene. The probe-mRNA complexes were sequentially colored, imaged, and de-colored for multiple imaging rounds. Fluorescent signal images captured on the root tissue sections were processed to identify individual mRNA molecules. Detected mRNAs corresponding to the same gene were assigned a unified identity and false-colored for clear visualization and presentation. b-j, Spatial expression pattern of identified cell type specific markers in both scRNA-seq and spatial transcriptomics data. The root transverse section anatomy illustration is displayed in the bottom right corner of panel a. Magenta signal for vascular tissue marker expression is also shown to better indicate where the endodermis is. n = 9 biological replicates for gel-grown root transverse section spatial transcriptomic data. The full marker gene and their annotation list can be found in Supplementary Table 3. The draft of a was created using BioRender (https://biorender.com) and further edited with Photoshop. a, UMAP visualization of cell distribution in the integrative scRNA-seq object, which includes gel-based, non-compacted-soil-based and compacted-soil-based scRNA-seq data. For cell clusters in the maturation stage, there was no clear enrichment of any cell type-specific markers. This lack of distinction may be attributed to the convergent nature of mature root cells, a phenomenon also observed in our gel-based scRNA-seq atlas (Fig. At this stage, due to the absence of markers for mature root cells, we provisionally annotated the large group of cells as “mature root cells”. b, UMAP visualization of 55 cell clusters in the integrative scRNA-seq object, which includes gel-based, non-compacted-soil-based and compacted-soil-based scRNA-seq data. The z-scores (expression enrichment score) of major cell type markers were calculated for each cluster. We used the marker expression patterns (Supplementary Data 4, 6, 8) and the z-score maximum (Supplementary Table 4) to assign each cluster to different cell types. It is noteworthy that the number of captured epidermal cells (Atrichoblast and Trichoblast, cluster 39 and cluster 55) was significantly low under non-compacted soil conditions. To rule out the possibility that we accidentally filtered out epidermal cells as low-quality cells during scRNA-seq data processing with COPILOT, we examined the low-quality cell data. However, we did not observe any evident cell type enrichment in the low-quality cells, suggesting that epidermal cells were not erroneously filtered out as low-quality cells (Supplementary Data 9). c, UMAP visualization of cell distribution in the integrative scRNA-seq object. d, Cell proportion of 10 major cell types and 2 developmental stages in both gel conditions and non-compacted soil conditions. Despite gentle cleaning of soil particles from root tips, a significant number of epidermal cells were likely removed, potentially altering the proportions of trichoblast and atrichoblast cells under different growth conditions. Growth condition itself does not change the trichoblast cell proportion dramatically. Details can be checked in Extended Data Fig. There is a notable increase of exodermis and sclerenchyma cell number in non-compacted-soils samples compared to that in gel conditions. e, Cell proportion of 10 major cell types and 2 developmental stages in both non-compacted soil and compacted soil conditions. The limited number of trichoblast cells detected under soil condition could be due to the cleaning of soil particles from root tips. f, Correlation analysis among the transcriptomic profiles of cells from 10 major cell types and 2 developmental stages in both gel conditions and non-compacted soil conditions. Low correlation was detected for the trichoblast cells, possibly due to the limited number of annotated root hair cells. g, Correlation analysis among the transcriptomic profiles of cells from 10 major cell types and 2 developmental stages (meristem and matured root cells) in both non-compacted soil conditions and compacted soil conditions. Low correlation was detected for the trichoblast cells, possibly due to the limited number of annotated root hair cells. The datasets sc_192 to sc_195 are gel-based scRNA-seq samples. Although the correlation between gel-based and soil-based samples is high, they can still be distinguished from each other based on their differential expression pattern. i-k, Representative images (maximum projection) of pOsCSLD1::VENUS-N7 expressing rice primary roots in gel, non-compacted (NC, 1.2 g/cm3) and compacted (CMP, 1.6 g/cm3) soil conditions. 3 days old rice roots were harvested from gel, and ± compacted soils. Cleared root tips were imaged under SP8 confocal microscope. n = 3 biological replicates (roots), all showing similar trends. a, UMAP projection of scRNA-seq from roots grown in gel, and roots grown in non-compacted (NC) soils of a non-transgenic, Kitaake genotype. High correlation values (>0.94) between Xkitaake and Kitaake scRNA-seq profiles, support their overall gene expression similarities. c, The GO term of “Phosphorus metabolic process”, “Vesicle-mediated transport”, “Hormone signalling pathway”, and “Cell wall organization” are still the top enriched GO term for the up-regulated genes in Kitaake in contrasting growth conditions, sterilized gel vs natural soils. The absence of enrichment for the GO term “Defense response” in Kitaake suggests that the XA21 transgene enhances defense responses in Xkitaake under changing growth conditions. The similarity in enriched GO terms for upregulated genes at outer root cells (highlighted with the red box) when the growth condition was changed from homogeneous gel to heterogeneous soils suggests that enhanced nutrient uptake and strengthened cell wall integrity in outer cell layers are common strategies for roots to cope with soil stresses. The one-tailed hypergeometric test with g:Profiler2 g:SCS (Set Counts and Sizes) algorithm for multiple comparison correction was used for the p-value calculation. The similar induction of genes related to nutrient uptake and cell wall integrity in outer root cells (highlighted with the red box) suggests that Xkitaake and Kitaake respond similarly to the growth condition changes. This further validates that the major trends identified through scRNA-seq analysis on Xkitaake are independent of the XA21 transgene. Nutrient uptake: LOC_Os06g37010, Zinc transporter 10; LOC_OS11g12740, NRT1; LOC_Os10g30770, Inorganic phosphate transporter; LOC_Os12g37840, Boron transporter 1. f, Heatmap showing the induced expression of R (resistance) genes predominantly in outer cells in Xkitaake genotype in soil growth conditions compared to gel growth conditions. The R genes were induced when growth conditions shift to natural soils in Xkitaake, particularly in the outer cell layers, indicating the significant role of outer cell layers in the root's adaptation to soil environments. g, Heatmap showing the induced expression of R (resistance) genes in Kitaake genotype in soil growth conditions compared to gel growth conditions. The R genes were also induced when grown in natural soils in Kitaake, although the outer cell layer enrichment is not detected in Kitaake background. h, UMAP projection of scRNA-seq from Xkitaake and Kitaake roots grown in non-compacted soils. i, Heatmap showing expression pattern of R genes in Xkitaake and Kitaake genotypes grown in soil conditions. The R genes show higher expression in Xkitaake roots grown under soil conditions compared to Kitaake roots grown under the same conditions. This suggests that the induction of R genes in soil conditions (compared to gel) can be further enhanced by the XA21 transgene. However, XA21 is not essential for this induction, as it is also observed in the Kitaake background. j, Heatmap showing the induced expression of the other defense response related genes in Xkitaake in soil growth conditions compared to gel growth conditions. Other defense response related genes show increased expression when compared in gel vs soil conditions in Xkitaake background. However, these genes do not exhibit a stronger induction pattern specifically in the outer cell layers. k, Heatmap showing expression pattern of other defense response-related genes in Kitaake genotype in gel versus soil conditions. This analysis suggests that even in the absence of XA21, defense-related genes show increased expression in soil conditions compared to gel conditions. Annotation for the included defense genes: R gene family: LOC_Os11g44990, OsMG1; LOC_Os11g45090, OsPB3; LOC_Os11g44960, Yr2; LOC_Os11g12000, OsLRR; LOC_Os07g19320, Yr10; LOC_Os11g12040, RPM1; LOC_Os09g10054, RPS2; LOC_Os06g43670, Putative disease resistance protein RGA1. a, Cell type expression for the identified marker genes in non-compacted-soil samples. Dot colors indicate the average scaled expression of each gene in each cell type group with darker colors indicating higher expression levels. b, Cell type expression for the identified marker genes in compacted-soils samples. Dot colors indicate the average scaled expression of each gene in each cell type group with darker colors indicating higher expression levels. c-k, Expression of identified cell type markers in both scRNA-seq and spatial data under compacted soil conditions. The color scale for each scRNA-seq feature-plot represents normalized, corrected UMI counts for the indicated gene. Spatial data of major cell type markers is visualized in rice root transverse sections. n = 4 biological replicates for compacted-soil-grown root transverse section spatial transcriptomic data. l, The root transverse section anatomy illustration. Marker annotations: Atrichoblast: LOC_Os01g64840, NEP1_NEPGR Aspartic proteinase nepenthesin-1; Trichoblast: LOC_Os10g42750, OsCSLD1; Exodermis: LOC_Os03g37411, OsMATE12; Sclerenchyma: LOC_Os08g02300, OsSWN2; Cortex: LOC_Os03g04310, OsRAI1; Endodermis: LOC_Os01g15810, OsPRX5; Vascular tissue: LOC_Os01g19170, OsPGL13; Phloem: LOC_Os06g45410, MYB family transcription factor; Stele: LOC_Os10g03400, OsSNDP1. The full marker gene and their annotation list can be found in Supplementary Table 3. m, The total number of differentially expressed genes (DEGs) for 9 major cell types and 2 developmental stages (meristem and mature root cells). n, The number of cell types in which one specific gene exhibits differential expression between gel-based and soil-based scRNA-seq data. Most differentially expressed genes (DEGs) are detected in only one or two major cell types. a, Most of the DEGs for the comparative analysis of non-compacted soils-based and compacted-soils-based scRNA-seq data are detected in only one or two major cell types. c, Gene expression heatmap for the up-regulated DEGs relevant to cell wall component metabolism and water stress response in endodermis. d, Gene expression heatmap for the up-regulated DEGs relevant to water stress response in exodermis. The complete list of gene ID and annotations are included in Supplementary Table 14. e, Left panel: Heatmap showing differential expression (log2 fold change) of water stress responsive genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. scRNA-seq showed increased expression patterns for genes relevant to response to water stress, with stronger induction at outer cell layers (highlighted by the red box), suggesting the enhanced water stress response at outer cell layer under soil compaction. Right panel: Heatmap showing scaled expression of water stress responsive genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq analysis was carried out using three independent biological replicates for non-compacted (NC rep #1-3) and compacted (CMP rep #1-3) soil conditions. f, GO terms for the down-regulated genes in exodermis under compacted soils as compared to non-compacted soils. g, GO terms for the up-regulated genes in endodermis under compacted soils as compared to non-compacted soils. h, GO terms for the down-regulated genes in endodermis under compacted soils as compared to non-compacted soils. Cell wall remodelling and water stress relevant GO terms are highlighted with red arrows in panels f-h. The one-tailed hypergeometric test with g:Profiler2 g:SCS (Set Counts and Sizes) algorithm for multiple comparison correction was used for the p-value calculation in f-h. i, Left panel: Heatmap showing differential expression (log2 fold change) of suberin/lignin biosynthesis genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. Enhanced expression of suberin and lignin biosynthesis genes in exodermis (highlighted by the red box), suggest higher suberin and lignin accumulation in exodermis under soil compaction. Right panel: Heatmap showing scaled expression of suberin/lignin biosynthesis genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq analysis further confirmed the upregulation of suberin/lignin biosynthesis genes in compacted soil conditions. a, Left panel: Heatmap showing increased expression (log2 fold change) of xyloglucan biosynthesis genes in compacted soil conditions compared to non-compacted soil conditions in Xkitaake as detected by scRNA-seq. Xyloglucan biosynthesis genes are broadly upregulated across major cell types, with a slightly higher induction observed in the exodermis. Right panel: Heatmap showing scaled expression of xyloglucan biosynthesis genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq also reveals a general induction of xyloglucan biosynthesis genes in compacted soils. The upregulation of these genes suggests enhanced cell wall reinforcement in response to compacted soil conditions. b, Left panel: Heatmap showing differential expression (log2 fold change) of cellulose synthase (CESA) genes in compacted soil conditions compared to non-compacted soil conditions in Xkitaake as detected by scRNA-seq. Notably, CESA4, CESA7, and CESA8 exhibit increased gene expression in sclerenchyma, suggesting enhanced secondary cell wall formation in this tissue under soil compaction. Right panel: Heatmap showing scaled expression of CESA genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Interestingly, bulk RNA seq reveals a general down-regulation of CESA genes in compacted soils, although the decrease is subtle for most examined genes. The relatively stronger down-regulation of CESA1, CESA5, and CESA6, combined with the relatively weaker down-regulation of CESA4, CESA7, and CESA8, may suggest a transition toward secondary cell wall deposition. c, Left panel: Heatmap showing increased expression (log2 fold change) of expansin genes in compacted soil conditions compared to non-compacted soil conditions in Xkitaake as detected by scRNA-seq. Increased expression of expansin genes, particularly in the exodermis and cortex cell layers, suggests the enhanced cell expansion at exodermis and cortex under soil compaction. Right panel: Heatmap showing scaled expression of expansin genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq further supports the upregulation of expansin genes in compacted soil conditions. The up-regulation of expansin genes correlates with the radial expansion of rice roots in response to soil compaction. d, Left panel: Heatmap showing increased expression (log2 fold change) of xylanase inhibitor genes in compacted soil conditions compared to non-compacted soil conditions in Xkitaake as detected by scRNA-seq. Xylanase inhibitor genes are broadly upregulated across major cell types. As xylanase inhibitor is tightly relevant to defense response, it further suggests that soil compaction could induce defense response in rice root. Right panel: Heatmap showing scaled expression of xylanase inhibitor encoding genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq analysis further supports the upregulation of xylanase inhibitor genes in compacted soil conditions. The complete list of gene ID and annotations are included in Supplementary Table 14. e, Confocal imaging of root transverse sections from non-compacted and compacted soil conditions. n = 3 biological replicates (roots), all showing similar trends. f, The heatmap of cortical cell areas under both non-compacted and compacted soil conditions. n = 3 biological replicates (roots), all showing similar trends. Segmented cells are outlined in cyan and superimposed on the meshed surface where the cell wall signals are projected (greyscale). n = 3 biological replicates (roots), all showing similar trends. h, The histogram showing the cell area distribution of exodermal cell under both non-compacted and compacted soil conditions. The primary roots were harvested from compacted and non-compacted soils were radially sectioned and imaged using Brillouin microscopy. k,l, Similarly, maps of acoustic attenuation between the two cases demonstrate greater apparent longitudinal viscosity in the compacted soil conditions. m, Brightfield image of a rice root radial cross-section (red boxes are the relevant regions of interest in i-l). n, Violin plot showing the cell wall stiffness of rice primary root in compacted and non-compacted soil conditions. Yuen's t-tests (two-tailed) indicate that there is no statistically significant (NS, p value = 0.7500) relative shift in Brillouin frequency between the cytoplasm (control) regions in non-compacted (NC) and compacted (C) specimens. However, there is a clear frequency shift between endodermal cell-walls between the two cases (p value < 1.0000 × 10−10; *: p < 0.05) indicating greater elasticity for compacted cell-walls. For cell wall measurements, n = 1744 (non-compaction, endodermis), 1843 (compaction, endodermis), 11852 (non-compaction, cytoplasm), and 17592 (compaction, cytoplasm) units, were analyzed respectively. a, Feature plots of key ABA pathway genes showing higher expression in phloem companion cells and pericycle cells in compacted soil conditions (right panel). The below in left panel displays a UMAP with cell type annotations for integrated scRNA-seq data, incorporating data from both non-compacted and compacted soil conditions. Cells representing phloem-derived vascular tissue are highlighted with a red rectangle. Left panel schematic shows the key step of ABA biosynthesis pathway and genes involved in these steps. b, Heatmap showing scaled expression of ABA biosynthesis genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. The upregulation of genes involved in multiple ABA biosynthesis pathways suggests an increased ABA level in rice roots under soil compaction. c, Heatmap showing scaled expression of ABA responsive genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA sequencing further supported the upregulation of most genes relevant to response to ABA, detected in scRNA-seq dataset. d, Lignin measurements from WT and mhz5 root tips from non-compacted (NC, 1.2 BD) and compacted (CMP, 1.6 BD) soil conditions. 3 independent replicates for compacted soils (n = 3) and 5 independent replicates for non-compacted soils (n = 5) were used to measure the lignin amount in rice root tips grown in non-compacted and compacted soils. Each replicate contains 4 root tips for compacted soil and 6 root tips for non-compacted soil conditions to generate equal dry weight. e-l, ABA biosynthesis defects mitigate the accumulation of suberin and lignin at water barriers under soil compaction. Histochemical staining of two other ABA biosynthesis mutants, aba1 and aba2 rice mutant root cross sections grown in non-compacted or compacted soils (1.2 or 1.6 g/cm−3 bulk density, panels e-h or i-l, respectively) for 3 days after germination. Lignin staining with Basic Fuchsin is shown with magenta color (panels e, g, i, k, white arrowheads) and suberin staining with Fluorol Yellow is shown as yellow (panels f, h, j, l, yellow arrowheads). The scale bar (50/ 75 μm) is indicated on each panel. Histochemical staining experiments were repeated 3 times with an n of 4 (compacted roots) 6 (noncompacted roots) each time. m-t, Soil compaction enhances suberin and lignin depositions closer to root tips. Lignin staining is shown with magenta colour (panels m, o, q, s, white arrowheads) and suberin as yellow (panels n, p, r, t, yellow arrowheads). The scale bar is indicated in each panel. Histochemical staining experiments were repeated 3 times with 4 (compacted) and 6 (noncompacted) roots each time. a, Heatmap showing scaled expression of auxin signalling genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA-seq analysis revealed increased expression of genes encoding Auxin Response Factors (ARFs) and decreased expression of genes encoding auxin/indole-3-acetic acid (Aux/IAA) proteins, indicating enhanced auxin signalling in response to soil compaction. b, Heatmap showing differential expression (log2 fold change) of auxin signalling genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. scRNA-seq showed similar expression patterns for ARFs and Aux/IAAs, further supporting the activation of auxin signalling under soil compaction. c, Heatmap showing scaled expression of ethylene signalling genes in non-compacted and compacted soil conditions in Xkitaake, as revealed by bulk RNA-seq. Bulk RNA sequencing demonstrated increased expression of ethylene signalling components, suggesting enhanced ethylene responses under soil compaction. d, Heatmap showing differential expression (log2 fold change) of ethylene signalling genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. scRNA-seq confirmed the upregulation of ethylene signalling components, further corroborating the activation of ethylene responses in response to soil compaction. e, Heatmap showing differential expression (log2 fold change) of auxin pathway genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. Cell type-specific expression patterns of genes involved in auxin homeostasis, transport, receptor activity, and downstream signalling were analyzed. No distinct cell type-specific patterns were observed. f, Heatmap showing differential expression (log2 fold change) of ethylene pathway genes in compacted soil conditions compared to non-compacted soils in Xkitaake as revealed by scRNA-seq analysis. Cell type-specific expression patterns of genes involved in ethylene biosynthesis, perception, and downstream signalling were analyzed. While an overall increase in ethylene signalling-related gene expression was detected, no distinct cell type-specific patterns were observed. Bulk RNA-seq analysis was carried out using three independent biological replicates for non-compacted (NC-rep #1-3) and compacted (CMP-rep #1-3) soil conditions. The complete list of gene ID and annotations are included in Supplementary Table 14. b, Only a limited number of genes exhibit differential expression between root tissues protoplasted for 2.5 h and 3 h. To avoid introducing potential artifacts from protoplasting into our comparative analysis of scRNA-seq data from roots grown in gel and soil conditions, we excluded the 232 differentially expressed genes identified here from the scRNA-seq data analysis. The two-tailed Wald test with Benjamini–Hochberg FDR for multiple comparison correction was used for the p-value calculation. c, PCA plot showing clear separation of bulk RNA sequencing data for root samples grown under non-compacted soil and compacted soil conditions. d, Pearson correlation plot illustrating the distinct clustering of bulk RNA sequencing data for root samples grown under noncompacted (NC) and compacted (CMP) soil conditions. e, Volcano plot depicting the number of upregulated and downregulated genes under soil compaction as identified in the bulk RNA sequencing data. The two-tailed Wald test with Benjamini–Hochberg FDR for multiple comparison correction was used for the p-value calculation. f, Enriched GO terms for the upregulated genes in bulk RNA sequencing data in compacted soils compared to non-compacted soil conditions. Notably, both ABA and ethylene signalling pathways are induced. g, Enriched GO terms for the downregulated genes in bulk RNA sequencing data in compacted soils compared to non-compacted soil conditions. The two-tailed Fisher's exact test with Benjamini–Hochberg FDR for multiple comparison correction was used for the p-value calculation of GO term analysis. scRNA-seq sample high-quality cell numbers related to cell type and developmental-stage annotation related to Figs. Protoplast inducing gene list (include both protoplasted versus non-protoplasted and protoplasted for 3 h versus protoplasted for 2.5 h), together with rice mitochondrial and chloroplast gene lists. Cell type marker list identified through the integration of published data, single-cell RNA-seq data and spatial transcriptomics. Clusters annotation for Xkitaake gel, non-compacted soils and compacted soils scRNA-seq integrated object based on average z-scores of cell type markers. DEG list in each cell type in the comparison of Xkitaake scRNA-seq data between gel-grown roots and soil-grown roots. Enriched GO term for DEGs in each cell type in the comparison of Xkitaake scRNA-seq data between gel-grown roots and soil-grown roots. Clusters annotation for Xkitaake gel, non-compacted soils scRNA-seq and Kitaake gel, non-compacted soils scRNA-seq integrated object based on average z-scores of cell type markers. DEG list in each cell type in the comparison of Kitaake scRNA-seq data between gel-grown roots and soil-grown roots. Enriched GO term for DEGs in each cell type in the comparison of Kitaake scRNA-seq data between gel-grown roots and soil-grown roots. DEG list in each cell type in the comparison of Xkitaake scRNA-seq data between non-compacted-soil-grown roots and compacted-soil-grown roots. Enriched GO term for DEGs in each cell type in the comparison of Xkitaake scRNA-seq data between non-compacted-soil-grown roots and compacted-soil-grown roots. The summary file of our bulk RNA-seq results, including (1) the normalized read counts for Xkitaake roots protoplasted for 2.5 h (P2.5 hours), Xkitaake roots protoplasted for 3 h (P3 hours), Xkitaake roots grown in gel conditions (Gel), Xkitaake roots grown in non-compacted soil conditions (NC) and Xkitaake roots grown in compacted soil conditions (CMP); (2) DEGs for comparison between Xkitaake roots P3 hours and P2.5 hours, DEGs for comparison between Xkitaake CMP and NC; (3) enriched GO terms for Xkitaake CMP and NC upregulated genes and Xkitaake CMP and NC downregulated genes. Cell area of exodermis and cortex cells quantified for transverse sections of rice roots grown in non-compacted and compacted soils. Locus ID or putative gene annotation, and relevant biological functions for the genes included in figures in this paper. scRNA-seq sample COPILOT summary information and details related to annotation. A combined PDF summary file is also included. Raw reads processed file and averaged read counts for the RNA-seq data of manually dissected root tissue segments corresponding to meristematic, elongation and maturation zones. R codes for generating the reads files are also included. Combined feature plots representing the expression patterns of cell type markers in single-cell RNA-seq data. Each image represents the gene expressions of markers for one certain cell type. Expression patterns of cell type markers in spatial transcriptomics data for gel-grown roots. A PDF summary file that includes all the sample and gene information for visualization is included. The raw spatial transcriptomics data for gel-grown roots, as well as the list of candidate genes for probe designing are also included. Combined Feature plots representing the expression patterns of cell type markers in single-cell RNA-seq data for non-compacted soil grown roots. Each image represents the gene expressions of markers for one certain cell type. Expression patterns of cell type markers in spatial transcriptomics data for non-compacted soil grown roots. A PDF summary file that includes the sample and gene information for visualization is included. The raw spatial transcriptomics data for non-compacted soil grown roots is also included. Combined feature plots representing the expression patterns of cell type markers in scRNA-seq data for compacted soil grown roots. Each image represents the gene expressions of markers for one certain cell type. Expression patterns of cell type markers in spatial transcriptomics data for compacted soil grown roots. A PDF summary file that includes the sample and gene information for visualization is included. The raw spatial transcriptomics data for compacted soil grown roots is also included. Jupyter notebook for checking the cell type identity for low-quality cells after the COPILOT processing of soil-based scRNA-seq data is included. Full gene and GO term for differential expression analysis. CSV files including the full (all, not filtered by P value or fold change) gene and GO term for differential expression analysis of our scRNA-seq data. These are raw data to generate gene expression and GO term enrichment files for heatmap plotting. Animation showing 3D UMAP of the atlas with cell type annotations. Each dot represents an individual cell with a high-quality single-cell transcriptome. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Single-cell transcriptomics reveal how root tissues adapt to soil stress. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44220-025-00423-5'>The Global Flourishing Study: Study Profile and Initial Results on Flourishing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-30 09:03:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The Global Flourishing Study is a longitudinal panel study of over 200,000 participants in 22 geographically and culturally diverse countries, spanning all six populated continents, with nationally representative sampling and intended annual survey data collection for 5 years to assess numerous aspects of flourishing and its possible determinants. Relations between a composite flourishing index and numerous demographic characteristics are reported. Participants were also surveyed about their childhood experiences, which were analyzed to determine their associations with subsequent adult flourishing. Analyses are presented both across and within countries, and discussion is given as to how the demographic and childhood relationships vary by country and which patterns appear to be universal versus culturally specific. Brief comment is also given on the results of a whole series of papers in the Global Flourishing Study Special Collection, employing similar analyses, but with more-specific aspects of well-being. The Global Flourishing Study expands our knowledge of the distribution and determinants of well-being and provides foundational knowledge for the promotion of societal flourishing. This interest can be seen in various sectors, including psychology, economics, business, education, medicine, public health and public policy1,2,3,4,5,6,7,8,9,10. While this has led to an expansion in research, much remains unknown, especially as it pertains to flourishing on the global landscape. The research on well-being has been shaped largely by Western perspectives and has been carried out mostly in Western contexts. This work has thus been subject to some critique11, as indeed is the case with social–scientific research more generally12. Such limitations hinder our conceptions of, and knowledge concerning, well-being. The Global Flourishing Study (GFS) was conceived as a large, open-access, 5-year longitudinal panel research study of 22 countries (see Fig. 1), spanning all six populated continents, with nationally representative sampling in each country, to study the distribution and determinants of well-being, to advance our knowledge of flourishing in general and especially in non-Western contexts and to uncover what patterns are culturally specific and which seem more universal. Compared with other cross-national studies of well-being such as the World Values Survey or the Gallup World Poll (GWP)/World Happiness Report13, the GFS aims to supplement and expand upon these by both (1) providing longitudinal panel data on the same cohort of individuals over time and (2) providing a broader range of assessments on well-being or flourishing. Flourishing is an expansive concept1,2,5,14,15, and the working definition underpinning the GFS has been ‘the relative attainment of a state in which all aspects of a person's life are good, including the contexts in which that person lives. '5,16 Several aspects of this definition are important. First, flourishing is multidimensional—it concerns all aspects of a person's life. No assessment of flourishing will ever fully measure flourishing, only aspects of it. Second, flourishing may be conceived of as an ideal, but it also concerns the ‘relative attainment' of that ideal17. We are never perfectly flourishing in this life, and there is always room for improvement. Third, flourishing concerns both objective and subjective aspects of life, although subjective aspects are more amenable to survey research. Fourth, the understanding of what is ‘good' will vary across cultures and contexts, but there is arguably a great deal of common ground as well, and such common ground is a reasonable starting point for measurement5,18. Finally, flourishing includes the contexts in which a person lives; such contexts include one's communities and environment. While the terms ‘flourishing' and ‘well-being' are often used interchangeably, flourishing arguably has a connotation of also having the environment itself being conducive to growth and being a part of one's flourishing. While well-being might be defined as ‘the relative attainment of a state in which all aspects of a person's life are good, as they pertain to that individual,' flourishing also includes the well-being of the community and environment. However, since individual aspects of flourishing effectively constitute well-being, the two terms will, in many contexts, often be used interchangeably. There may also often be greater consensus across cultures around what is desirable for individual well-being than in understandings concerning what constitutes the right type of community or government, and so the composite flourishing index considered in the following focuses on those individual aspects5. We do, however, also offer further comment on analyses using more community-related assessments. Analyses concerning composite assessments of flourishing are presented as to how flourishing assessments vary across demographic groups and countries and how various types of childhood experiences relate to adult flourishing. Comment is offered on a range of other analyses and studies arising out of the first wave of the GFS data, and discussion given to the study's importance and promotion of flourishing in a global context. With the concept of flourishing being expansive, there is no unique or fully comprehensive way to partition ‘all aspects of a person's life.' Understandably, numerous well-being classifications have thus emerged1,2,5,14,19. 5, the survey development process20,21 incorporated numerous cross-cultural perspectives and questions extending well beyond those proposed in ref. 5—health, happiness, meaning, character, relationships and financial security. However, the GFS data and analyses do not presuppose the priority of these domains, and other researchers can certainly recategorize the items. A perhaps simpler division into domains derives from the World Health Organization's definition22 of ‘health' as a ‘state of complete physical, mental, and social well-being.' The categories used here further divide mental well-being into emotional, cognitive and volitional well-being, corresponding to traditional and empirical division of the mind into emotions, intellect and will23,24,25,26,27,28. Social well-being might itself have further been divided into relational and communal well-being; in more expansive understandings of flourishing, communal well-being might itself be sub-divided into community, political and economic well-being. Again, there is no unique way to divide the conceptual space. Much has been written on including a spiritual dimension within the WHO definition29, thereby yielding, ‘a state of complete physical, mental, social, and spiritual well-being,' which we have referred to elsewhere as the WHO+ definition16,17. It is, however, more difficult to attain consensus on understandings of spiritual well-being across the world religions, and we will make some remarks on this matter later in the Article. Finally, material and financial resources are sometimes taken as determinants of well-being and sometimes understood as constitutive elements of well-being30,31,32,33,34. These conceptualizations rightly vary depending on how well-being is defined. We do not take a definitive stance on this matter in the Article and present analyses alternatively including and excluding financial well-being. In what follows, we describe the GFS methodology, comment on results of the analyses for the composite flourishing index, offer brief comment on numerous papers concerned with more-specific aspects of well-being and conclude by offering discussion on the implications of the study and results for the promotion of flourishing. Table 2 presents nationally representative descriptive statistics of the observed sample. Slightly more than half were middle-aged (30–59 years old (53%)), women (51%), married (52%), employed (either for an employer or self-employed, 57%) and had between 9 and 15 years of education (57%). Most participants were born in the country in which they were surveyed (94%), and approximately one-third of participants attended religious services at least weekly (32%). Sample sizes in each country ranged from 1,473 (Turkey) to 38,312 (United States). Participant characteristics for each country are shown in Supplementary Tables 1a–22a. Table 3 presents nationally representative descriptive statistics of the observed sample for retrospective assessments of childhood experiences. Most reported ‘very good' or ‘somewhat good' relationships with mother (89%) and father (80%) and that their parents were married (75%); a large proportion reported their family ‘lived comfortably' (35%) or ‘got by' (41%) financially during their childhood. About 18% of participants reported experiencing abuse and 16% feeling like an outsider growing up; most assessed their health as ‘excellent' or ‘very good' (64%) in childhood. A large proportion of participants attended religious services at least once per week during childhood (41%). Over all the items used as childhood predictors, the percentage of respondents with any missing data was only 12.9%. With regard to demographic relations and associations with childhood experiences, we will first comment on the results of random effects meta-analysis across the 22 countries. We then turn to variations across countries. Table 4 presents ordered means of a composite flourishing index. The index includes two indicators each across six domains: happiness, health, meaning, character, relationships and financial security (see Table 5 below and ref. Table 4 presents weighted means so as to be nationally representative within country, and alternately includes means both without and with financial security (10 versus 12 indicators, respectively), with standard deviations, Gini coefficients and alpha reliability coefficients. When including financial security, Israel instead of Mexico has the second highest mean. Standard deviations for composite flourishing across countries range from 1.30 to 2.03. Gini coefficients for inequality tend to be highest in those countries reporting the lowest means. Cronbach's alpha for the index is relatively high in all countries. All subsequent tables report results with financial security. Analogous analyses without financial security are given in Supplementary Tables S26–S28. Table 6 presents random effects meta-analysis of composite flourishing means by demographic category. Aggregated across all countries, flourishing increases with age from 7.03 for those aged 18–49 years to 7.36 for those aged 80 years or older; it is relatively similar for men and women, but lower for those identifying as ‘other' genders. Flourishing tends to increase somewhat with education, but more dramatically with religious service attendance, ranging from a mean of 6.86 for those never attending to 7.67 for those attending more than weekly. It is slightly higher for those born in the country (7.16) than for those not (7.02). Results are generally similar when excluding financial security (Supplementary Table 26), but the relations with age are slightly more muted in these analyses. Table 7 presents results from random effects meta-analysis of multivariate regression of composite flourishing on all childhood predictors (Table 7) simultaneously. Good childhood relationships with mother (0.18; 95% CI: 0.11, 0.24) and father (0.11; 95% CI :0.07, 0.18) were positively associated with adult flourishing. There is some evidence that having parents who were never married (−0.11; 95% CI: −0.21, −0.01), or when one had died (−0.07; 95% CI: −0.16, 0.02), or possibly divorced (−0.07; 95% CI: −0.17, 0.02), compared with having parents who were married, was associated with lower adult flourishing. There is a notable gradient in flourishing with childhood subjective financial status, with finding it very difficult to get by associated with lower flourishing (−0.27; 95% CI: −0.35, −0.19) and living comfortably associated with higher adult flourishing (0.20; 95% CI: 0.14, 0.26) compared with those who got by. A similar gradient can be seen regarding self-rated health growing up: poor health is associated with lower (−0.34; 95% CI :−0.54, −0.14) adult flourishing, and excellent health with higher (0.46; 95% CI: 0.31, 0.62) flourishing compared with those in good health. Abuse (−0.34; 95% CI: −0.40, −0.29) and feeling like an outsider growing up (−0.26; 95% CI: −0.33, −0.18) were associated with lower adult flourishing. Attending religious services weekly was associated with higher adult flourishing (0.27; 95% CI :0.17, 0.36) compared with never attending. Flourishing tended to increase with age, but there was relatively little difference with regard to immigration status, or men versus women. Table 8 presents E-value (the minimum strength of the association an unmeasured confounder must have with both the outcome and the predictor, above and beyond all measured covariates, for an unmeasured confounder to explain away an association) sensitivity analyses35 of meta-analyzed childhood predictor associations to potential unmeasured confounding. Some of these associations between the childhood predictors and adult flourishing were moderately robust to potential unmeasured confounding. For example, to explain away the association between abuse in childhood with higher adult flourishing, an unmeasured confounder that was associated with both absence of abuse and higher flourishing by risk ratios of 1.74-fold each, above and beyond the measured covariates, could suffice, but weaker joint confounder associations could not; to shift the 95% confidence interval to include the null, an unmeasured confounder that was associated with both absence of abuse and higher adult flourishing by risk ratios of 1.64-fold each, above and beyond the measured covariates, could suffice, but weaker joint confounder associations could not. The overall patterns in the preceding, pooled over all countries, are illuminating but disguise important country-specific variation (Supplementary Information). We comment on this variation in the following, but caution is needed with regard to over-interpretation, especially in cases when magnitudes are not strong. Flourishing tends to increase with age in many countries, including Argentina, Australia, Brazil, Sweden and the United States, but not in all. In India, Egypt, Kenya and Japan, patterns are somewhat more U-shaped. In Poland and Tanzania, it is mostly decreasing with age. Other countries show more complex patterns with age. Men and women are fairly similar globally, but there are greater differences in certain countries. In Brazil, men report higher flourishing than women (0.39; 95% CI: 0.32, 0.47); in Japan, women report higher flourishing than men (−0.26; 95% CI: −0.32, −0.21). The patterns of married individuals reporting higher flourishing than those divorced (or separated) are nearly universal, but differences vary from 0.92 (0.63, 1.22) in Israel to 0.10 (−0.10, 0.30) in Argentina. In most countries, those married report notably higher flourishing than those who are single, but not in all: married report lower flourishing than single in India (−0.39; 95% CI: −0.50, −0.28) and Tanzania (−0.35; 95% CI: −0.51, −0.20). Those employed effectively universally report higher flourishing than unemployed, but again differences vary from 1.50 (95% CI: 1.18, 1.83) in the United States to just 0.09 (95% CI: −0.03, 0.22) in Kenya. There is, however, considerable variation across countries in comparisons of employed versus self-employed, retired or student, with those self-employed and retired reporting higher flourishing than those employed in many developing countries and students reporting notably higher flourishing than those employed in Poland, India, Japan, Tanzania, Israel, Egypt and Kenya. Those attending religious services more than once a week (or weekly) universally report higher flourishing than those never attending, but differences vary from 2.33 (2.12, 2.54) in Hong Kong to 0.15 (−0.01, 0.30) in India. In most countries, those with more education report higher flourishing, but the reverse is the case in Hong Kong and Australia. Associations with immigration status also vary by country, with those born in the country reporting notably higher flourishing in the Philippines, Poland, India, Israel, Mexico and Hong Kong, whereas the reverse is the case in Spain and Australia. Likewise, with the analyses concerning the childhood predictors, there are some patterns that are nearly universal and others that seem to vary by culture and context. Associations between good maternal relationship and adult flourishing were nearly universal but varied in magnitude from 0.63 (95% CI: 0.27, 0.99) in Indonesia to effectively null (−0.14; −0.43, 0.14) in Israel, and likewise for good paternal relationship. Compared with those with married parents growing up, those with divorced, separated, single or one or more deceased parents tended to have lower flourishing in adulthood, although Turkey was an anomaly with respect to divorce. Compared with a subjective financial status of getting by in childhood, living comfortably was essentially universally associated with higher adult flourishing with magnitudes ranging from 0.70 (0.53, 0.87) in Hong Kong to effectively null (−0.05; −0.17, 0.08) in Nigeria; and finding it very difficult was universally associated with lower adult flourishing with magnitudes ranging from −0.92 (−1.45, −0.40) in Turkey to null (0.00; −0.36, 0.35) in Sweden. Experiencing abuse and feeling like an outsider were likewise universally associated with lower adult flourishing. Excellent, as compared with good, childhood health was universally associated with higher adult flourishing, and while poor childhood health was nearly universally associated with lower adult flourishing, there were possibly a few exceptions: in Germany there was evidence of a positive association (0.40; 0.07, 0.74). Weekly religious service attendance growing up was associated with higher adult flourishing in almost all countries, with the largest effect sizes in Poland (1.02; 0.77,1.28) followed by Hong Kong and Turkey; however, Kenya, South Africa and Tanzania manifested slightly negative estimates (−0.22, −0.15, −0.05, respectively), although with confidence intervals including 0. Patterns across countries for immigration status, age/birth cohort and gender roughly followed that reported in the preceding for demographic statistics, although after multivariate adjustment, further gender differences emerged, with notable evidence for women having higher flourishing than men in Egypt, Japan and Hong Kong, and men having higher flourishing than women in Spain, United Kingdom, Brazil, Kenya and Argentina. The preceding analyses concerned composite flourishing. However, just as pooling across countries is of interest, but obscures country-specific relations, so also the use of a composite flourishing measure can obscure more nuanced relations with more-specific aspects of well-being. A series of papers, following the same methodology described in the preceding, has examined these various more-specific well-being domains, and many of these papers are part of the present GFS Special Collection. While these papers contain far more detailed analysis than can possibly be summarized here, some higher-level remarks may be of interest in trying to understand similarities and differences in relationships with more-specific aspects of well-being. We will provide a cursory overview of some of the results in each of the well-being domains in turn, focusing principally on any patterns that notably diverge from those reported concerning composite flourishing. Many, albeit not all, of the differences concern relations with age, gender and childhood adversity. Patterns for self-rated physical health36 followed that of composite flourishing in many respects but differed in that scores decreased with age, although with some variation (for example, with U-shaped patterns in Australia, Japan and Sweden), and men report better health than women overall, with the reverse notable only in Japan. Patterns with health limitations37 and physical pain38 were conversely similar, that is, increasing in age. Physical pain was one of the very few outcomes that was adversely associated with religious service attendance38,39. For pain, there was evidence in some countries (for example, South Africa and Israel) that those who financially just got by reported lower levels of adult pain than those who lived comfortably in childhood39. Patterns for self-rated mental health tended to follow those of flourishing, but were U-shaped with age overall, although increasing in Australia, United States and Sweden and decreasing with age in Israel and Tanzania40. Depression and anxiety, assessed with the Patient Health Questionnaire-4 (PHQ-4)41, both decreased with age, were less related to religious service attendance and were somewhat higher for women than men. With respect to emotional well-being, patterns for happiness and life satisfaction mostly follow those of flourishing, but are slightly more U-shaped (or ‘J-shaped') with age42, somewhat comparable to much previous research43. The increasing-with-age pattern for composite flourishing is thus shaped by other aspects of well-being documented in the following. There are also notable differences in the country-specific reporting and in some of the demographic relationships between life satisfaction, life evaluation and affective happiness, and further discussion of these matters and of differences between the GFS and the GWP is given elsewhere42. In contrast to life satisfaction/evaluation, the outcomes of balance, inner peace and optimism are all increasing with age44,45. Patterns with mastery are similar but somewhat smaller in magnitude46. In contrast to many outcomes, suffering is not strongly patterned with age overall, or with religious service attendance, but there is notable variation in this across countries; suffering varies more with marital status, employment, education and gender, with women reporting higher suffering than men in most countries47 even after multivariate adjustment48. Meaning and purpose manifest similar demographic relationships as with composite flourishing and are likewise increasing with age49. For meaning and purpose, the relationships with childhood adversity seem more complex. Death of a parent seems to matter more for meaning and purpose in Spain, Israel and Nigeria than in Australia, Philippines, Argentina or Japan (wherein estimates were positive, albeit with wide confidence intervals). In some countries (United States), difficult, or even very difficult (Argentina), childhood financial circumstances were associated with higher adult meaning, and in others (Poland), living comfortably was associated with lower adult meaning. By contrast, childhood abuse and feeling like an outsider were essentially universally associated with lower adult meaning50. Concerning character (‘volitional well-being'), assessments of promoting the good are mostly increasing with age (although not in all countries; in India and Tanzania this was decreasing with age); were slightly higher with women and notably higher with religious service attendance51,52. Similar patterns were manifest with more objective behaviors such as, within the last month, volunteering, charitable giving and helping a stranger, although charitable giving and helping were slightly higher among men, volunteering was more uniform in age (until decreasing at 80+), and helping a stranger decreased with increasing age53,54. The relationships of these behaviors with childhood adversity were also more complex. Childhood abuse and feeling like an outsider predicted higher adult volunteering, giving and helping; the relationship with childhood health with both volunteering and charitable giving was U-shaped with both excellent childhood health and poor childhood health predicting higher adult volunteering than good health55,56. By contrast, abuse, feeling like an outsider, poor childhood health and difficult childhood financial circumstances all tended to decrease capacity for delayed gratification, showing love/care for others, gratitude and hope57,58,59,60, although none of these variables mattered for forgiveness61. Forgiveness, gratitude and showing love/care were all increasing with age; hope was relatively flat with age; and adult delayed gratification was decreasing with age. With regard to social well-being, patterns for close social relationships mostly followed those of composite flourishing65,66 except women reported very slightly higher than men, and there was little relationship with education overall. Those with lower education reported higher relationship quality in Indonesia, Kenya, Sweden and the United States, but this pattern was reversed in Brazil, Israel, Japan and Poland. In general, those who were married reported the highest relationship quality, but widowed individuals did so in 6 of the 22 countries. Having an intimate friend was roughly constant with age (but highest among 80+); by contrast, having social support was U-shaped with age, higher for women and higher for students and those retired than for those employed; other patterns were relatively similar to that of composite flourishing67. Patterns showed somewhat more variation with communal dimensions of social well-being. For non-religious community engagement, weekly participation decreased with age, was slightly higher for men, higher for those single than married, higher with students than the employed, but itself increasing with religious service attendance68. Having a sense of belonging within the country followed many of the same patterns as composite flourishing but was slightly higher for women, slightly decreasing with education and notably higher for those born within the country69,70. Similar results pertain to the demographic relations with satisfaction with the city or area in which one lives (‘place satisfaction'), except for essentially no male–female difference and a weaker relation with being born within the country. Trust within the country and feeling one had a political voice were both slightly U-shaped with age and slightly higher for men; with political voice, but not trust, which was increasing with education71,72. Other patterns were relatively similar to that of composite flourishing. Concerning financial well-being, many of the patterns follow those of composite flourishing, especially with respect to the childhood predictors73. However, across the subjective and objective financial well-being indicators, there were additional important differences, especially with respect to age, gender and religious service attendance. With respect to spiritual well-being, as noted previously, how this is understood will vary by tradition and culture. Moreover, many of the religious or spiritual GFS assessments concern behaviors or beliefs that may be determinants of well-being, rather than constituting spiritual well-being per se. We consequently restrict comment to a single item involving feeling loved or cared for by God or a spiritual force74. Even this, of course, may not be applicable in non-theistic religious traditions. We thus focus on the sub-analysis that excludes respondents who indicate that the question is not relevant. The patterns for this indicator follow fairly closely those for composite flourishing except proportions are higher for women, slightly U-shaped with education and higher for those who have immigrated. The patterns across countries are complex, and there is no straightforward way to summarize results across all of the indicators and papers. There is considerable variability in the ordered means of countries across indicators (Supplementary Table 29). However, as will be discussed, such ordering of means needs to be interpreted cautiously since interpretation of items and response scales may vary by language and culture75,76. The interpretation of these thus often makes sense only on a relative basis within countries. In Table 9, we report, for each country, the four well-being indicators (out of 47 total) for which the country, relative to the other indicators, ranked highest, and the four indicators for which the country, relative to the other indicators, ranked lowest, to give some indication of the relative strengths and weaknesses of each country. With regard to the primary results from the flourishing index, several comments merit attention. On average, when pooled across the 22 countries, flourishing is essentially flat with age through ages 18–49 and then increases with age thereafter. This is in striking contrast to earlier work—focused mostly on life satisfaction/evaluation—which had suggested a more dramatically U-shaped pattern with age43. Even with life satisfaction, pooled over the 22 GFS countries, this is now more J-shaped than U-shaped. As noted in the preceding, the increasing-with-age pattern in 2023, when pooled across GFS countries, now also holds on average with many specific aspects of well-being, including balance, inner peace, mastery, optimism, meaning and purpose, promoting good, charitable giving, forgiveness, gratitude, showing love/care, relationship quality, feeling belonging, place satisfaction and feeling loved by God or a spiritual force. Earlier nationally representative 2022 data in the United States identified these patterns with all of the various flourishing domains, including life satisfaction77. There is also other corroborating evidence concerning changes in patterns between age and life evaluation and mental health78,79,80. While there are distinctions across measures, covariate controls and countries, the cumulative evidence here and elsewhere likely reflects some real changes in well-being's relation with age that have been taking place over time. While causes are likely diverse, mental health concerns with young adults are clearly on the rise. As noted, in some countries the patterns concerning flourishing and age are still somewhat more U-shaped (India, Egypt, Kenya, Japan) and in others (Poland, Tanzania) decreasing with age. Nevertheless, the overall global pattern is troubling. An open question concerns whether these relations constitute a newly patterned ‘age effect' or whether this is in fact a new ‘cohort effect.' This will be resolved only with more data over time. It is possible that because of social and economic conditions, it simply is more difficult to be young today, but that well-being for these young people will increase with age. It is, however, also possible, for example, that, within-person, over time, well-being will continue to follow a U-shaped pattern (but with the mean of that U lower) so that the younger cohorts will decline in well-being over time. Only future waves of data collection will be able to distinguish between these two possibilities, or their combination. It should also be noted that the answer to this question is not predetermined, and in fact depends a good deal on the extent to which policy is shaped to try to better support the well-being of young people. The well-being literature, focused mostly on life satisfaction/evaluation, has consistently replicated the results with marriage, employment and religious service attendance5,32,81, for example, and these patterns were relatively consistent across countries in the GFS. While the analyses here are purely descriptive, other longitudinal, experimental and quasi-experimental studies suggest that each of these factors also has a causal role; they are likely ‘pathways' to flourishing5. What was interesting in this context, however, is how substantially the magnitudes of differences in flourishing related to these demographic factors varied across countries. Future research could aim at trying to understand the explanations for these differences and whether different cross-cultural mechanisms or moderators (or possibly different patterns of confounding) may be at play. Similar considerations pertain to the differences across countries observed in the relations between flourishing and education, immigration and gender. We will, however, offer three additional, admittedly selected, summary points of interest. While many developed nations report comparatively higher levels of financial security and life evaluation, these same nations are not flourishing in other ways, often reporting lower meaning49, pro-sociality51 and relationship quality65. Japan reports strikingly low scores on many well-being indicators, and this appears to be not purely an artifact of a tendency to report in the middle of 0–10 response scales82 since the same patterns hold with binary indicators. Other, often middle-income, nations, such as Indonesia, Argentina, Mexico, Brazil and the Philippines report higher on these other more humanistic aspects of flourishing. While purely descriptive, the negative relationship between meaning and gross domestic product per capita is particularly striking49 and has been confirmed in previous work83. This may raise important questions about how to carry out economic development in ways that do not compromise meaning and purpose. The claim being made here is not a causal assertion about gross domestic product lowering meaning. Rather, the desired outcome of a society is presumably one with both high levels of economic development and high levels of meaning, and the question is then how to attain this. Similar considerations may pertain to relationships and character. Second, there can sometimes be complex relationships between adversity and subsequent flourishing. Often adversity is indeed detrimental, and work must be done to address conditions of adversity and suffering. But sometimes these adverse conditions can for some give rise to development and growth and certain forms of flourishing84,85. In some countries, more adverse childhood socioeconomic circumstances were associated with lower levels of adult pain, perhaps suggesting a developed resilience. In the United States and Argentina, difficult childhood financial circumstances were associated with higher adult meaning. These patterns pertain to some countries but not others. However, there is also variation across outcomes. Aggregating across all countries, childhood abuse and feeling like an outsider somewhat surprisingly predicted higher levels of adult volunteering, giving and helping but predicted lower levels of showing love/care for others. The effects on these seemingly closely related outcomes are in opposite directions. While more can and should be done to address suffering and adverse circumstances, some experience of suffering seems ubiquitous and an almost unavoidable part of the human condition86,87. In addition to trying to reduce suffering, we should also better seek to understand how we might approach and transform such suffering, when we cannot eliminate it, to enable, when possible, further flourishing85, and we should better understand the limits of such approaches. Third, in both the demographic analyses and in the childhood predictor analyses, religious service attendance was one of the factors most consistently associated with present or subsequent well-being, across countries and across outcomes. This is consistent with much previous literature focused principally on the West88,89,90,91,92, but now expanded to a broad range of countries. Service attendance was not beneficially associated with all well-being outcomes. It was associated with slightly higher levels of suffering and of physical pain, which may arise from either differing experiences of it or sensitivity to it. However, with the vast majority of other well-being outcomes, in both the demographic analyses and the childhood predictor analyses, there were beneficial dose–response associations with service attendance, often with a fairly steep gradient. This does not of course guarantee that such associations will be found in all countries, but it is nevertheless notable that they do pertain to many of the 22 GFS countries. While the demographic analyses are purely descriptive, the childhood predictor analyses are intended to provide some evidence for causality. The near ubiquity of these associations is thought-provoking, and this factor is arguably too often neglected in our understanding of well-being and its trends over time. While the present data are insufficient to provide definitive evaluation, it is quite possible that the divergence of material and more humanistic forms of well-being noted in the preceding is in large part due to the declining religiosity of more economically advanced nations. We may need a reconsideration of spiritual pathways to well-being. The strengths of the GFS are many and include nationally representative samples; broad cultural, geographical and non-Western coverage; large sample sizes; and indicators of numerous well-being and other outcomes along with their determinants. While there are other important multi-country international datasets that are already available (for example, GWP, World Values Survey, International Social Survey Program), the GFS has the advantage of a much wider breadth of well-being constructs and (starting with Wave 2) a longitudinal panel data structure. The GFS and the analyses presented in this Article, however, undoubtedly also have limitations. While the GFS has a broad range of constructs and indicators and broad geographical coverage, many of the constructs are assessed with a single indicator. Although resulting issues of power are partially offset by large sample sizes, this does not mitigate concerns about validity and conceptual breadth for any given construct. The single indicators were selected on the basis of previous work on longitudinal predictive power and strong correlations with other indicators within a scale21, but this does not entirely resolve the concerns. The assessments are also based on self-report survey responses and may be subject to self-report biases and mode effects, which we also comment on in the following. Moreover, such insights principally concern subjective rather than objective aspects of flourishing. It should also be noted that while the samples within each country were constructed to be approximately nationally representative, the countries themselves do not constitute a random sample of all countries. Rather, as noted, the countries were selected to (1) maximize coverage of the world's population, (2) ensure geographic, cultural and religious diversity and (3) prioritize feasibility and existing data collection infrastructure. The countries do come from all six populated continents and constitute nearly half of the world's population (with mainland China included in the second wave of the study, nearly two-thirds). However, no low-income countries are represented, although lower-middle-income, upper-middle-income, and high-income countries are. Results may of course differ in other countries, perhaps especially those with higher levels of conflict or adversity. Thus, for example, using 2023 GWP data, the mean life evaluation across the 22 GFS countries is 5.91 (95% CI: 5.49, 6.33) whereas the mean across all 142 GWP countries is 5.60 (95% CI: 5.41, 5.78); and further mean differences comparing GWP and GFS may arise due to seasonality and mode effects21. Conclusions here should thus be restricted to the 22 GFS countries. The variation of results across countries also makes clear that extrapolation to other countries may be unwarranted. Nevertheless, as noted, some of the patterns appear to be universal across the 22 countries, and for these one might speculate that they may hold more generally, although this is not guaranteed. With regard to limitations of the analyses presented in this Article, caution is needed in interpreting cross-national differences as these may be influenced by matters of translation75,76, different modes of assessment, differing interpretation of items and of response scales, and seasonal effects arising from data being collected in different countries at different times of the year. Table 5 should thus be interpreted as simply providing ordered means across what are somewhat differentially interpreted items, not as ‘rankings.' In addition, the demographic analyses are purely descriptive and should not be interpreted causally. The childhood predictors analyses constitute a synthetic longitudinal study and may provide some evidence for causality but should be interpreted cautiously. The childhood predictors assessments are retrospective and may be subject to recall bias. Nevertheless, for recall bias to completely explain away the observed associations would require that the effect of adult well-being on biasing retrospective assessments of the childhood predictors would essentially have to be at least as strong as the observed associations themselves93, and some of these were quite substantial. The childhood predictors analyses may also be subjected to unmeasured confounding, although these concerns were at least partially addressed through the E-value sensitivity analyses, and in some cases the associations were found to be quite robust. We did not regress current flourishing outcomes contemporaneously on all demographic factors as such analyses would have been purely cross-sectional and yet weaker in terms of capacity for providing causal evidence94. However, future waves of data will give further insight into potential causal relationships concerning well-being and its various potential determinants. The GFS, and these initial results, help provide foundational knowledge for the promotion of societal flourishing. Understanding the distribution of flourishing across the globe, and by demographic group, helps us to understand who needs help, and in what ways. It allows us to identify the groups toward whom interventions or policies might be targeted to improve well-being. Our analyses likewise give insight into how childhood experiences shape well-being, which may be of interest both descriptively and in future intervention and policy development aimed at preventing adverse childhood experiences that impede adult flourishing. One cannot, however, erase the past, and the analyses here also suggest that sometimes adversity can give rise to greater resilience and can be a pathway for growth. Understanding under what circumstances this can and does take place and how it might be facilitated, and its limits, will be an important direction for future research. Can we carry out economic development in ways that do not compromise meaning and purpose and relationships and character, given that many economically developed nations are not faring as well on these measures? With economic development and secularization, have we sometimes been neglecting, or even suppressing, powerful spiritual pathways to flourishing? The very word ‘flourishing' can arguably be used either as an abstract noun to indicate a state (as per the composite flourishing index) or to suggest a dynamic process of growth95. How can each nation grow and flourish? If society is to ultimately pursue flourishing, these questions of age, and of economic development, and of spiritual dynamics need to be taken into consideration. The GFS is an open-access data resource available through the Center for Open Science. In addition to our analyses, we hope many others will access the data and contribute other important insights, insights that might advance our knowledge of and capacity to promote human flourishing. We hope that the GFS itself, and the understandings that arise from it, will shift discussion and policy toward the promotion of flourishing. We will first describe the methodology for the questionnaire development; then the measures, data collection, and sampling; and finally the analytic methods used both for descriptive statistics and for assessing associations of flourishing with childhood experiences assessed retrospectively. Development of the questionnaire for the GFS took place over eight distinct phases: (1) selection of core well-being, religion and demographic questions; (2) solicitation of further well-being, social, political, psychological, economic, community and health questions from domain experts (for example, gratitude, social connection, etc.) worldwide; (3) feedback and questionnaire refinement from scholars around the world representing numerous academic disciplines on survey questions and structure; (4) input and questionnaire refinement from experts in multicultural survey research and survey translations; (5) feedback and questionnaire refinement from an open invitation to comment on the survey, posted publicly and sent to numerous listservs, with input from over 150 scholars; (6) further questionnaire refinement via input from Gallup's survey design specialists; (7) adaptation of items from an interviewer-administered to a self-administered survey instrument using best practices for web survey design to minimize item non response, illogical responses and incomplete responses; and (8) translation into other languages and subsequent confirmation by scholars in several participating countries that translations accurately captured the intended meaning of each question. This was then followed by piloting and cognitive interviews in all of the participating countries and subsequent introduction of retrospective childhood experience questions when it became clear that the intake and annual survey were to be separated in settings not administered by web. During the translation process, Gallup adhered to the TRAPD model (translation, review, adjudication, pretesting and documentation) for cross-cultural survey research96. 20,21 for more detailed information on the questionnaire development and the refinement of the survey, and survey items, at each stage. Wave 1 of the GFS data included the following countries and territories: Argentina, Australia, Brazil, Egypt, Germany, Hong Kong (Special Administrative Region of China), India, Indonesia, Israel, Japan, Kenya, Mexico, Nigeria, the Philippines, Poland, South Africa, Spain, Sweden, Tanzania, Turkey, United Kingdom and the United States (Fig. The countries were selected to (1) maximize coverage of the world's population, (2) ensure geographic, cultural and religious diversity and (3) prioritize feasibility and existing data collection infrastructure. Data collection was carried out by Gallup Inc. The study protocol was approved by the institutional review boards of Gallup and of Baylor University. Gallup secures in-country institutional review boards in countries where local authorities request one. Data for Wave 1 were collected principally during 2023, with some countries beginning data collection in late 2022 and exact dates varying by country97. Four additional waves of panel data on the participants will be collected annually from 2024 to 2027. Data from mainland China were not included in the first data release due to fieldwork delays. All Wave 1 and 2 data from mainland China are part of the April 2025 dataset release. The precise sampling design to ensure nationally representative samples varied by country, and further details are available in refs. Survey items included aspects of well-being such as happiness, health, meaning, character, relationships and financial stability5, along with other demographic, social, economic, political, religious, personality, childhood, community, health and well-being variables. Data were also collected concerning retrospective assessments of childhood experiences. The data are freely available to all through the Center for Open Science99, upon submission of a pre-registration. Data on gender were missing from 397 (0.2%). All participants provided informed consent to participate. Participants were generally compensated US$3–$6 depending on the country. Gender was assessed as male, female or other. Marital status was assessed as single/never married, married, separated, divorced, widowed or domestic partner. Employment was assessed as employed, self-employed, retired, student, homemaker, unemployed and searching, or other. Immigration status was dichotomously assessed with: ‘Were you born in this country, or not?' Religious tradition/affiliation was captured with the categories of Christianity, Islam, Hinduism, Buddhism, Judaism, Sikhism, Baha'i, Jainism, Shinto, Taoism, Confucianism, Primal/Animist/Folk religion, Spiritism, African-Derived, some other religion, and no religion/atheist/agnostic; precise response categories varied by country76. Racial/ethnic identity was assessed in some, but not all, countries, with response categories varying by country. Relationship with mother during childhood was assessed with the question: ‘Please think about your relationship with your mother when you were growing up. In general, would you say that relationship was very good, somewhat good, somewhat bad, or very bad?' Responses were dichotomized to very/somewhat good versus very/somewhat bad. An analogous variable was used for relationship with father. ‘Does not apply' was treated as a dichotomous control variable for respondents who did not have a mother or father due to death or absence. Parental marital status during childhood was assessed with responses of married, divorced, never married, and one or both had died. Financial status was measured with: ‘Which one of these phrases comes closest to your own feelings about your family's household income when you were growing up, such as when YOU were around 12 years old?' Responses were lived comfortably, got by, found it difficult and found it very difficult. Abuse was assessed with yes/no responses to ‘Were you ever physically or sexually abused when you were growing up?' Participants were separately asked: ‘When you were growing up, did you feel like an outsider in your family?' Childhood health was assessed by: ‘In general, how was your health when you were growing up? Immigration status was assessed with: ‘Were you born in this country, or not?' Religious service attendance during childhood was assessed with: ‘How often did YOU attend religious services or worship at a temple, mosque, shrine, church or other religious building when YOU were around 12 years old?' with responses of at least once per week, one to three times per month, less than once per month or never. For additional details on the assessments, see the GFS codebook (https://osf.io/cg76b) or ref. The primary outcome measure in this Article concerns a composite index for individual aspects of flourishing5 using two self-report questions in each of six domains: happiness and life satisfaction, physical and mental health, meaning and purpose, character and virtue, close social relationships, and financial and material stability (Table 2). This is an index—not a scale—and aggregates across a number of disparate aspects of well-being. There is a somewhat arbitrary nature to an index insofar as one could always include more, or fewer, or different items, but ideally a suitable well-being index would have considerable conceptual breadth across numerous domains100. This particular assessment has received considerable empirical validation in cross-cultural research101,102,103,104. Other papers in the special collection focus on specific aspects of well-being or its determinants, including numerous questions that are not included in the 12-item composite. Brief comment is made on the results of these and on when the patterns differ from that of composite flourishing to provide a more comprehensive description of the results. While the flourishing index assesses various individual aspects of flourishing, comment is also made in the following, and in the other papers in the special collection on more community-oriented aspects of flourishing. Also reported in the Supplementary Information are similar analyses to those in the tables but carried out using only the first five domains (excluding financial and material stability) as financial resources are often considered an important determinant of flourishing, rather than a constitutive part of flourishing. Descriptive statistics for the full sample, weighted to be nationally representative within each country, were estimated for each of the demographic variables. Nationally representative means for the flourishing index were estimated separately for each country and ordered from highest to lowest along with 95% confidence intervals, standard deviations, Gini coefficients (for assessing inequality in the population distribution of flourishing) and alpha reliability coefficients. Variation in means for the flourishing index across demographic categories were estimated, with all analyses initially conducted by country (Supplementary Tables S1–S22). Primary results in the tables consisted of a random effects meta-analyses105,106,107 of country-specific flourishing means in each specific demographic category along with 95% confidence intervals, standard errors, lower and upper limits of a 95% prediction interval across countries, heterogeneity (τ) and I2 for evidence concerning variation of flourishing means within a particular demographic variable across countries108. Meta-analysis was employed over hierarchical modeling so as not to presume measurement invariance107,109. Forest plots of estimates are available in the Supplementary Information (parts 2–4). All meta-analyses were conducted in R110 using the metafor package111. Within each country, a global test of variation of outcome across levels of each demographic variable was conducted, and a pooled P value112 across countries reported concerning evidence for variation within any country. Bonferroni corrected P-value thresholds are provided on the basis of the number of demographic variables113,114. Religious affiliation/tradition and race/ethnicity were used, when available, as control variables within country, and associations by country are presented in Supplementary Tables S1–S29 but are not meta-analyzed since the availability of these response categories varied by country. 107 for more extensive description on the analytic methodology for the demographic analyses for this Article and other papers in the special collection. Descriptive statistics for the observed sample, weighted to be nationally representative within country, were estimated for each childhood demographic category. A weighted linear regression model with complex survey adjusted standard errors was fit within each country of composite flourishing on all of the aforementioned childhood predictor variables simultaneously. In the primary analyses, random effects meta-analyses of the regression coefficients105,106,109 along with confidence intervals, estimated proportions of effects across countries with effect sizes larger than 0.1 or below −0.1, heterogeneity (τ) and I2 for evidence concerning effect size variation within a given predictor category across countries are given108. Forest plots of estimates are available in the Supplementary Information (part 4). Religious affiliation/tradition and race/ethnicity were used within country as control variables, when available, but these coefficients themselves were not included in the meta-analyses since categories/responses varied by country. Within each country, a global test of association of each childhood predictor variable group with outcome was conducted, and a pooled P value112 across countries was reported concerning evidence for association within any country. Bonferroni corrected P-value thresholds are provided on the basis of the number of childhood predictor variables113,114. For each childhood predictor, we calculated E-values to evaluate the sensitivity of results to unmeasured confounding. An E-value is the minimum strength of the association an unmeasured confounder must have with both the outcome and the predictor, above and beyond all measured covariates, for an unmeasured confounder to explain away an association35. 109 for a more extensive description on the analytic methodology for the childhood predictor analyses for this Article and other papers in the special collection. Missing data on all variables were imputed using multivariate imputation by chained equations, and five imputed datasets were used115,116,117,118. To account for variation in the assessment of certain variables across countries (for example, religious affiliation/tradition and race/ethnicity), the imputation process was conducted separately in each country. This within-country imputation approach ensured that the imputation models accurately reflected country-specific contexts and assessment methods. Sampling weights were included in the imputation models to account for specific-variable missingness that may have been related to probability of inclusion in the study. The GFS used different sampling schemes across countries based on availability of existing panels and recruitment needs97. All analyses accounted for the complex survey design components by including weights, primary sampling units and strata. Additional methodological detail, including accounting for the complex sampling design, is provided elsewhere97,98. All analyses were pre-registered with the Center for Open Science (https://osf.io/registries/gfs) before data access; all code to reproduce analyses are openly available in an online repository119. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data for Wave 1 of the Global Flourishing Study is available through the Center for Open Science upon submission of a pre-registration (https://doi.org/10.17605/OSF.IO/3JTZ8) and will be openly available without pre-registration beginning 202699. Please see https://www.cos.io/gfs-access-data for more information about data access. Code for the Global Flourishing Study in multiple software packages is openly available (https://doi.org/10.17605/osf.io/vbype). Seligman, M. E. Flourish: A Visionary New Understanding of Happiness and Well-Being (Simon and Schuster, 2011). & So, T. T. Flourishing across Europe: application of a new conceptual framework for defining well-being. Adler, M. D. & Fleurbaey, M. (eds) The Oxford Handbook of Well-Being and Public Policy (Oxford Univ. On the promotion of human flourishing. Kristjánsson, K. Flourishing as the Aim of Education: A Neo-Aristotelian View (Routledge, 2019). Psychological well-being as part of the public health debate? Insight into dimensions, interventions, and policy. Layard, R. & De Neve, J. E. Wellbeing (Cambridge Univ. Lomas, T. Making waves in the great ocean: a historical perspective on the emergence and evolution of wellbeing scholarship. Helliwell, J. F., Layard, R., Sachs, J. D. & Neve, J. E. D. World Happiness Report 2021 (Sustainable Development Solutions Network, 2021). Keyes, C. L. The mental health continuum: from languishing to flourishing in life. & Lomas, T. Terminology and the well-being literature. A flexible map of flourishing: the dynamics and drivers of flourishing, well-being, health, and happiness. VanderWeele, T. J. et al. Flourishing in critical dialogue. Su, R., Tay, L. & Diener, E. The development and validation of the Comprehensive Inventory of Thriving (CIT) and the Brief Inventory of Thriving (BIT). Global Flourishing Study: 2024 Questionnaire Development Report (OSF, 2024); https://osf.io/y3t6m Complete English Translation in Five Volumes (Ave Maria Press, 1948). McGrath, R. E. Integrating psychological and cultural perspectives on virtue: the hierarchical structure of character strengths. & Hall-Simmonds, A. Scarecrow, Tin Woodsman, and Cowardly Lion: the three-factor model of virtue. Hacker, P. M. S. The Intellectual Powers: A Study of Human Nature (Wiley, 2013). Hacker, P. M. S. The Passions: A Study of Human Nature (Wiley, 2017). Hacker, P. M. S. The Moral Powers: A Study of Human Nature (Wiley, 2021). Peng-Keller, S., Winiger, F. & Rauch, R. (eds) The Spirit of Global Health: The World Health Organization and the ‘Spiritual Dimension' of Health, 1946–2021 (Oxford Univ. Aristotle The Nicomachean Ethics: Translated with an Introduction (Oxford Univ. & Fitoussi, J. P. Report by the Commission on the Measurement of Economic Performance and Social Progress (Commission on the Measurement of Economic Performance and Social Progress, 2009). Diener, E. & Biswas-Diener, R. Happiness: Unlocking the Mysteries of Psychological Wealth (Wiley, 2011). (eds) Measuring Well-being: Interdisciplinary Perspectives from the Social Sciences and the Humanities (Oxford Univ. & VanderWeele, T. J. Aristotelian flourishing and contemporary philosophical theories of wellbeing. & Ding, P. Sensitivity analysis in observational research: introducing the E-value. Bradshaw, M. et al. Demographic variations in self-rated physical health across 22 countries: findings from the Global Flourishing Study. Paltzer, J. et al. Demographic variation in health problem-related limitations in life across 22 countries: a cross-national analysis. Macchia, L. et al. Demographic variation in physical pain across 22 countries: a cross-national analysis. Macchia, L. et al. Childhood predictors of physical pain: a cross-national analysis. An ultra-brief screening scale for anxiety and depression: the PHQ–4. Lomas, T. et al. Life evaluation, life satisfaction, and happiness: assessing inter-relations and 15 childhood and demographic factors across 22 countries in the Global Flourishing Study. Is well-being U-shaped over the life cycle? Lomas, T. et al. Demographic variation in balance in life across 22 countries: a cross-national analysis of the Global Flourishing Study. Lomas, T. et al. Demographic variation in inner peace across 22 countries: a cross-national analysis of the Global Flourishing Study. Cowden, R. G. et al. Sociodemographic variation in suffering: a cross-national analysis with 22 countries. Cowden, R. G. et al. Childhood predictors of suffering in adulthood: a cross-national analysis with 22 countries. Kim, E. S. et al. Mapping demographic variations in purpose and meaning across the world: a cross-national analysis of 22 countries in the Global Flourishing Study. Kim, E. S. et al. Identifying childhood roots of adult purpose and meaning across 22 countries (Global Flourishing Study). Chen, Y. et al. Variations in character involving an orientation to promote good across sociodemographic groups in 22 countries. Early life experiences and adult orientation to promote good in 22 countries. Nakamura, J. S. et al. Demographic variation in volunteering across 22 countries in the Global Flourishing Study. Nakamura, J. S. et al. Demographic variation in charitable giving and helping across 22 countries in the Global Flourishing Study. Nakamura, J. S. et al. Childhood predictors of volunteering across 22 countries in the Global Flourishing Study. Nakamura, J. S. et al. Childhood predictors of charitable giving and helping across 22 countries in the Global Flourishing Study. Delayed gratification across 22 countries: a cross-national analysis of demographic variation and childhood predictors. Lee, M. T. et al. Sociodemographic variation and childhood predictors of showing love and care for others across 22 countries: a cross-national analysis. Okuzono, S. S., Cowden, R. G., Yancey, G., Johnson, B. R. & VanderWeele, T. J. Childhood predictors of gratitude in adulthood: a cross-national analysis of 22 countries in the Global Flourishing Study. Cowden, R. G. et al. Childhood predictors of dispositional forgivingness in adulthood: a cross-national analysis with 22 countries. Okuzono, S. S., Cowden, R. G., Yancey, G., Johnson, B. R. & VanderWeele, T. J. Sociodemographic variation in gratitude using cross-national analysis with 22 countries. Cowden, R. G. et al. Sociodemographic variation in dispositional forgivingness: a cross-national analysis with 22 countries. Wilkinson, R. et al. From best friend to soul mate: sociodemographic variation in social relationship quality across 22 countries. Wilkinson, R. et al. Life course insights into social relationship quality: a cross-national analysis of 22 countries. Ritchie-Dunham, J. L. et al. Demographic variation in social support and intimate friend across 22 countries: a cross-national analysis. Shared horizons: community participation in secular and religious contexts across sociodemographic groups in 22 countries. Counted, V., Johnson, B. R., Allen, K., Miner, M. & VanderWeele, T. J. & Johnson, B. R. Demographic variation in social trust across 22 countries: a cross-national analysis. Ssozi, J. et al. Demographic variation in political voice across 22 countries: a cross-national analysis. Analysis of demographic variation and childhood predictors correlates of financial well-being across 22 countries. Cowden, R. G., Skinstad, D., Lomas, T., Johnson, B. R. & VanderWeele, T. J. Measuring wellbeing in the Global Flourishing Study: insights from a cross-national analysis of cognitive interviews from 22 countries. Johnson, K. A., Moon, J. W., VanderWeele, T. J., Schnitker, S. & Johnson, B. R. Assessing religion and spirituality in a cross-cultural sample: development of religion and spirituality items for the Global Flourishing Study. Chen, Y., Cowden, R. G., Fulks, J., Plake, J. F. & VanderWeele, T. J. National data on age gradients in well-being among US adults. Social Media and Mental Health: a collaborative review (New York Univ., 2024); tinyurl.com/SocialMediaMentalHealthReview Diener, E., Suh, E. M., Lucas, R. E. & Smith, H. L. Subjective well-being: three decades of progress. Tedeschi, R. G. & Calhoun, L. G. Posttraumatic growth: conceptual foundations and empirical evidence. Paul, J. II Salvifici Doloris (Ediciones Paulinas, 1984). Van Zeller, H. The Mystery of Suffering (Ave Maria Press, 2015). Religion as a Social Determinant of Public Health (Oxford Univ. Chen, Y., Kim, E. S. & VanderWeele, T. J. Religious-service attendance and subsequent health and well-being throughout adulthood: evidence from three prospective cohorts. Balboni, T. A. et al. Spirituality in serious illness and health. & Peteet, J. R. Handbook of Religion and Health (Oxford Univ. Associations of religious upbringing with subsequent health and well-being from adolescence to young adulthood: an outcome-wide analysis. Simple sensitivity analysis for differential measurement error. Can sophisticated study designs with regression analyses of observational data provide causal inferences? A Theology of Health: Wholeness and Human Flourishing (Univ. Survey Research Center Guidelines for Best Practice in Cross-Cultural Surveys (Institute for Social Research, Univ. Global Flourishing Study Methodology (OSF, 2024); https://osf.io/k2s7u Padgett, R. N. et al. Survey sampling design in Wave 1 of the Global Flourishing Study. The Global Flourishing Study (OSF, 2024); https://doi.org/10.17605/OSF.IO/3JTZ8 & Johnson, B. R. Multidimensional versus unidimensional approaches to well-being. Human flourishing in cross cultural settings. Evidence from the United States, China, Sri Lanka, Cambodia and Mexico. Höltge, J. et al. A systems perspective on human flourishing: exploring cross-country similarities and differences of a multisystemic flourishing network. Exploring the psychometric network of flourishing across 22 countries. Borenstein, M., Hedges, L. V., Higgins, J. P. T. & Rothstein, H. R. A basic introduction to fixed-effect and random-effects models for meta-analysis. Padgett, R. N. et al. Analytic methodology for the demographic variation analyses for Wave 1 of the Global Flourishing Study. Robust metrics and sensitivity analyses for meta-analyses of heterogeneous effects. Padgett, R. et al. Analytic methodology for childhood predictor analyses for Wave 1 of the Global Flourishing Study. Viechtbauer, W. Conducting meta-analyses in R with the metafor package. The harmonic mean P-value for combining dependent tests. Abdi, H. in Encyclopedia of Measurement and Statistics (ed. Some desirable properties of the Bonferroni correction: is the Bonferroni correction really so bad? Multiple imputation for missing data in epidemiological and clinical research: potential and pitfalls. van Buuren, S. Flexible Imputation of Missing Data 2nd edn (CRC Press, 2023). Global Flourishing Study statistical analyses code. We thank everyone who has contributed to the development of the GFS, as outlined in greater detail in ref. This includes especially the study participants in all countries, along with the translators, interviewers, partners, vendors and Gallup team in all of these countries, the many scholars from numerous global institutions who were involved in more substantial ways in question selection and validation of the survey instrument, and the very many people around the world who provided detailed feedback, suggestions, additions and revisions during the open global feedback phase of the questionnaire development. The Global Flourishing Study was supported by funding from the John Templeton Foundation (grant no. ), Well-Being for Planet Earth Foundation, Fetzer Institute (no. ), Well Being Trust, Paul L. Foster Family Foundation and the David and Carol Myers Foundation. The opinions expressed in this publication are those of the authors and do not necessarily reflect the views of these organizations. Tyler J. VanderWeele, Brendan Case, Ying Chen, Richard G. Cowden, Chris Felton, Hayami K. Koga, Matthew T. Lee, Tim Lomas, Katelyn N. G. Long, Sakurako S. Okuzono, Suzanne T. Ouyang, R. Noah Padgett, Renae Wilkinson & Jennifer Wortham Byron R. Johnson, Rebecca Bonhag, Matt Bradshaw, Thomas Breedlove, Chris Felton, Alex Fogleman, Craig Gundersen, Sung Joon Jang, Matthew T. Lee, Christos A. Makridis, John Ssozi, Robert D. Woodberry & George Yancey Regent University, Virginia Beach, VA, USA Arizona State University, Tempe, AZ, USA Westmont College, Santa Barbara, CA, USA George Fox University, Newberg, OR, USA Institute for Global Flourishing, New York, NY, USA Center for Open Science, Charlottesville, VA, USA University of Texas Health Science Center at San Antonio, San Antonio, TX, USA Wisconsin Lutheran College, Wauwatosa, WI, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived of and designed the study. contributed to the study conception and design. All authors commented on the initial draft and read and approved the final manuscript. reports consulting fees from Gloo Inc., along with shared revenue received by Harvard University in its license agreement with Gloo according to the University IP policy. The other authors declare no competing interests. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            