
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-28</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65876151/tower-of-london-dig/'>Archaeologists Excavated the Tower of London—and Unearthed Dozens of Hidden Skeletons</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Excavations revealed everything from a 14th-century Black Death group burial to three skeletons from the late 12th or early 13th centuries buried in coffins. As archaeologists dug deeper into the ground, they found roughly 20 more burials, including one group grave likely tied to the 1348 “Black Death” plague. “Undertaking these two excavations has provided us with a generational opportunity to enhance our understanding of the evolution off the Chapel of Saint Peter ad Vincula, and the buildings which stood before it,” Alfred Hawkins, curator of historic buildings at Historic Royal Palaces, the organization that oversees the Tower, said in a statement. The dig began as a trial excavation in 2019 to prepare the on-site Chapel Royal of St Peter ad Vincula for a new elevator. Subsequent excavations outside the chapel as deep as 10 feet below the surface revealed everything from a 14th-century Black Death group burial to three skeletons from the late 12th or early 13th centuries buried in coffins—an unusually expensive burial for that time. Jane Sidell, principal inspector of ancient monuments at Historic England, said the team is already gaining insight into the residents of the Tower in a way they never have before. “There is so much more to learn through further analysis about the people as well as the buildings of one of England's most evocative historic monuments.” It also served as a prison for high-status individuals (including King Henry VI), and housed the country's Royal Mint. But it has seemingly almost always had a chapel on its grounds. “The new excavations provide the opportunity to transform our understanding of the Tower's community,” Katie Faillace from Cardiff University's School of History, Archaeology, and Religion, said in a statement. “Our work uses a biomolecular technique known as isotope analysis, which tells us about health, diet, and mobility in the past, all from a tiny fragment of a tooth. This cutting-edge method was unparalleled potential for reconstructing the experiences of the people who lived and died at the Tower, allowing us to build a rich picture of individuals' lives.” Richard Madgwick, archaeological scientist at Cardiff University and part of the team, told National Geographic that one individual was likely a middle-aged female who died between 1480 and 1550. Clues indicate she likely lived as far away as Wales at one point, and had a diet featuring sugar—an expensive ingredient at the time. “I'm looking forward to starting analysis of some of the other amazing finds we have uncovered along the way,” Hawkins said. Along with the remains, the team found a rare burial shroud from the late 12th or early 13th century (fabric doesn't usually last through the ages), jewelry, shards of stained glass, and rare grave goods in the form of funerary incense pots dated to between 1150 and 1250 (with charcoal still inside them). “At the moment we've got these lovely two biographies,” Madgwick said. “It hints at the dynamic movement of people and the dynamic life trajectories of the people who were buried in the Tower, but it's going to be really exciting to see whether we've picked two anomalies, or whether we see the broader range of lifeways that we see of those buried here.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250828002406.htm'>Scientists recreate life's first step: Linking amino acids to RNA</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 14:47:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Chemists at UCL have shown how two of biology's most fundamental ingredients, RNA (ribonucleic acid) and amino acids, could have spontaneously joined together at the origin of life four billion years ago. But proteins cannot replicate or produce themselves -- they require instructions. These instructions are provided by RNA, a close chemical cousin of DNA (deoxyribonucleic acid). Senior author Professor Matthew Powner, based at UCL's Department of Chemistry, said: "Life relies on the ability to synthesize proteins -- they are life's key functional molecules. "Our study is a big step towards this goal, showing how RNA might have first come to control protein synthesis. "Life today uses an immensely complex molecular machine, the ribosome, to synthesize proteins. "We have achieved the first part of that complex process, using very simple chemistry in water at neutral pH to link amino acids to RNA. For the new study, the researchers took inspiration from biology, using a gentler method to convert life's amino acids into a reactive form. In order to form these thioesters, the amino acids react with a sulfur-bearing compound called pantetheine. Last year, the same team published a paper demonstrating pantetheine can be synthesized under early Earth-like conditions, suggesting it was likely to play a role in starting life. Lead author Dr Jyoti Singh, from UCL Chemistry, said: "Imagine the day that chemists might take simple, small molecules, consisting of carbon, nitrogen, hydrogen, oxygen, and sulfur atoms, and from these LEGO pieces form molecules capable of self-replication. This would be a monumental step towards solving the question of life's origin. "Our study brings us closer to that goal by demonstrating how two primordial chemical LEGO pieces (activated amino acids and RNA) could have built peptides**, short chains of amino acids that are essential to life. This discovery could potentially link metabolism, the genetic code and protein building." The reactions are too small to see with a visible-light microscope and were tracked using a range of techniques that are used to probe the structure of molecules, including several types of magnetic resonance imaging (which shows how the atoms are arranged) and mass spectrometry (which shows the size of molecules). *The Nobel laureate Christian de Duve proposed that life began with a "thioester world" -- a metabolism-first theory that envisages life was started by chemical reactions powered by the energy in thioesters. ** Peptides typically consist of two to 50 amino acids, while proteins are larger, often containing hundreds or even thousands of amino acids, and are folded into a 3D shape. These Simple Everyday Treatments Offer Drug-Free Relief for Knee Arthritis Common Painkillers Like Ibuprofen Could Be Fueling a Global Health Threat This Everyday Food Ingredient Might Be Messing With Your Brain Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/type-1-diabetes-patients-insulin-production-restored-with-new-cell/'>Type 1 Diabetes Patient's Insulin Production Restored with New Cell Transplant Therapy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists treated a person's type 1 diabetes with genetically modified insulin-producing cells that evaded immune system attacks. Insulin-producing cells can be genetically modified to hide from the immune system. People with type 1 diabetes must constantly rely on insulin injections or pumps, usually for the rest of their life after diagnosis. But now research suggests a new therapy could finally allow people with type 1 diabetes to make insulin on their own. A 42-year-old man who has lived most of his life with type 1 diabetes has become the first human to receive a transplant of genetically modified insulin-producing cells that can slip past the immune system's mistaken attacks. “This is the most exciting moment of my scientific career,” says cell biologist Per-Ola Carlsson of Uppsala University in Sweden, who helped develop the procedure. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Scientists injected nearly 80 million genetically tweaked cells into the participant's forearm muscle, and 12 weeks later the cells were still alive and producing insulin. About two million people in the U.S. live with type 1 diabetes, which typically requires an intensive regimen of insulin injections and blood sugar monitoring. If their blood sugar runs amok, people face severe risks, including heart attacks, nerve damage, vision problems, kidney disease, and more. For decades, scientists have struggled to develop therapies that can successfully replenish beta cells—the specialized insulin-producing cells that are found in the pancreas. Newly added functional beta cells are usually quickly destroyed because a type 1 diabetic immune system flags them as invaders. A few past attempts successfully transplanted donor islets—clusters of pancreatic cells that included beta cells—but these always ultimately triggered an aggressive immune response. And such a response requires recipients to take lifelong immunosuppressive drugs, which come with serious side effects, such as increased risks of infections and cancer. First, they broke down a deceased donor's pancreatic islets into single cells. To further evade immune system detection, the team made some cells express higher levels of a gene that discourages attacks by the body's natural killer cells and macrophages, two types of immune cells. Blood tests showed no measurable immune cell activation or antibody production in response to these cells. But within four to 12 weeks following the transplant, his levels rose slightly on their own after meals—showing that the new beta cells were releasing some insulin in response to glucose. Four adverse events occurred, but none were serious or related to the modified cells. Some people with type 1 diabetes may still have a small set of functional beta cells, but in more established cases, the immune system often whittles away all cells, Alonso says. For those established cases, she says, “cell-based therapy is where we need to go.” Alonso adds, however, that even though the new study is promising, it involved just one participant and is therefore preliminary. She also notes that the injected cells produced only 7 percent of the insulin needed for a person to be fully independent of additional medication. “It's a relatively short but nonetheless important finding,” says Kevan Herold, an endocrinologist at the Yale School of Medicine, who was also not involved in the study. In the meantime, he and his colleagues are developing a stem cell source that would be capable of producing large quantities of insulin-producing cells. While Herold thinks it's still too early to consider this approach for a cure, “these options are now here to change the disease in ways that have never been possible before,” he says. Humberto Basilio is a Mexican science and environmental journalist covering climate change, biodiversity, health, technology, archaeology, migration, and human rights. He is the current news intern at Scientific American. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/glow-in-the-dark-succulents-created-by-scientists-shine-in-multiple-colors/'>Glow-in-the-Dark Succulents Created by Scientists Shine in Multiple Colors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Houseplants become rechargeable night-lights after injection with tiny phosphor particles Researchers gave these succulents in the Echevaria genus a glow up by injecting them with luminescent particles. University students might soon have something other than black-light posters to brighten their dorm rooms. Researchers have created glow-in-the-dark plants by injecting succulents with materials similar to those that make the posters light up. The fleshy plants shine as brightly as a night light, and can be made to do so in a wide variety of colours — a first for glowing houseplants, according to the team. The idea of making glowing plants has captivated scientists since the late 1980s, when researchers made the first bioluminescent plant by inserting a gene from a firefly (Photinus pyralis) into a type of tobacco (Nicotiana tabacum). The biotechnology firm Light Bio in Sun Valley, Idaho, sells the petunia (Petunia hybrida), which glows a very faint green thanks to genes from a light-emitting mushroom. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Unlike the petunia, which emits light through chemical reactions in its cells, the succulent glows because of materials injected into its leaves. These materials — phosphor particles made of strontium and aluminium dosed with other metals — absorb energy from light at one wavelength, store some of that energy and then slowly re-emit it at a different wavelength for several hours. For instance, one material the scientists injected into their succulents absorbs ultraviolet and blue light, and re-emits it as green light. Whereas genetically engineered bioluminescent plants are, so far, limited in the range of colours they emit, afterglow phosphors span a wide variety of hues, including red and blue, and they can be combined to produce a white glow. The researchers purchased phosphors containing strontium aluminate and ground them down to particles of various sizes before injecting them into an assortment of plants. They found that particles around 7 micrometres in diameter glowed more brightly than did nanoparticles in plants, and were able to fill up the interior tissues of succulent leaves for a stronger, more uniform glow. To make every leaf glow, the researchers had to inject each one with phoshor particles, a process that takes about ten minutes. Co-author Shuting Liu, also at the South China Agricultural University, says that the team hopes to move away from injecting each leaf by using smaller particles, which can spread more readily throughout plants. Keith Wood, chief executive of Light Bio, says it's not yet clear how these particles affect the health of the plants over their lifespan. Additionally, the researchers acknowledge that more safety testing is needed; in particular, they need to know whether a leaf full of particles would be toxic if accidentally ingested. Wood is also sceptical that glowing plants will ever be a living lighting system, capable of replacing light bulbs. So far, researchers have fallen “spectacularly short of that objective”, he says. This article is reproduced with permission and was first published on August 27, 2025. Katherine Bourzac is a journalist based in San Francisco, who covers environment, climate, chemistry, health and computing for Nature, Science News, and other publications. First published in 1869, Nature is the world's leading multidisciplinary science journal. Nature publishes the finest peer-reviewed research that drives ground-breaking discovery, and is read by thought-leaders and decision-makers around the world. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03900-7'>An eyecare foundation model for clinical assistance: a randomized controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 09:38:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In the context of an increasing need for clinical assessments of foundation models, we developed EyeFM, a multimodal vision–language eyecare copilot, and conducted a multifaceted evaluation, including retrospective validations, multicountry efficacy validation as a clinical copilot and a double-masked randomized controlled trial (RCT). EyeFM was pretrained on 14.5 million ocular images from five imaging modalities paired with clinical texts from global, multiethnic datasets. Efficacy validation invited 44 ophthalmologists across North America, Europe, Asia and Africa in primary and specialty care settings, highlighting its utility as a clinical copilot. The RCT—a parallel, single-center, double-masked study—assessed EyeFM as a clinical copilot in retinal disease screening among a high-risk population in China. A total of 668 participants (mean age 57.5 years, 79.5% male) were randomized to 16 ophthalmologists, equally allocated into intervention (with EyeFM copilot) and control (standard care) groups. The primary endpoint indicated that ophthalmologists with EyeFM copilot achieved higher correct diagnostic rate (92.2% versus 75.4%, P < 0.001) and referral rate (92.2% versus 80.5%, P < 0.001). Secondary outcome indicated improved standardization score of clinical reports (median 33 versus 37, P < 0.001). Participant satisfaction with the screening was similar between groups, whereas the intervention group demonstrated higher compliance with self-management (70.1% versus 49.1%, P < 0.001) and referral suggestions (33.7% versus 20.2%, P < 0.001) at follow-up. Post-deployment evaluations indicated strong user acceptance. Our study provided evidence that implementing EyeFM copilot can improve the performance of ophthalmologists and the outcome of patients. This is a preview of subscription content, access via your institution Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout For the reproduction of our algorithm code, we have also deposited a minimum dataset (https://zenodo.org/records/15546254; ref. 41), which is publicly available for scientific research and non-commercial use. All requests for further data sharing will be reviewed by the data management committees from participating institutions and by the ethics committee of Shanghai Health and Medical Centre, China, to verify whether the request is subject to any intellectual property or confidentiality obligations and will be accessible with informed consents. Requests for access to deidentified individual-level data from this trial can be submitted via email to B.S. (shengbin@sjtu.edu.cn) with detailed proposals for approval and will be evaluated on a case-by-case basis and responded to within 60 days. Source data are provided with this paper. Zhou, Y. et al. A foundation model for generalizable disease detection from retinal images. Towards a general-purpose foundation model for computational pathology. Lu, M. Y. et al. A visual-language foundation model for computational pathology. High-performance medicine: the convergence of human and artificial intelligence. Collaboration between clinicians and vision–language models in radiology report generation. Norden, J, G. & Shah, N. R. What AI in health care can learn from the long road to autonomous vehicles. & Mishuris, R. G. Clinical trials informed framework for real world clinical implementation and deployment of artificial intelligence applications. & Brownstein, J. S. A call for artificial intelligence implementation science centers to evaluate clinical effectiveness. Gupta, A., Savarese, S., Ganguli, S. & Fei-Fei, L. Embodied intelligence via learning and evolution. Colunga-Lozano, L. E. et al. Clinical judgment shows similar and sometimes superior discrimination compared to prognostic clinical prediction models: a systematic review. Bommasani, R. et al. On the opportunities and risks of foundation models. Three epochs of artificial intelligence in health care. Lu, M. Y. et al. A multimodal generative AI copilot for human pathology. Yang, J. et al. Generalizability assessment of AI models across hospitals in a low-middle and high income country. Accurate prediction of disease-risk factors from volumetric medical scans by a deep vision model pre-trained with 2D scans. Matheny, M. E., Whicher, D. & Thadaney Israni, S. Artificial intelligence in health care: a report from the National Academy of Medicine. van de Sande, D. et al. To warrant clinical adoption AI models require a multi-faceted implementation evaluation. Hadziahmetovic, M., Nicholas, P., Jindal, S., Mettu, P. S. & Cousins, S. W. Evaluation of a remote diagnosis imaging model vs dilated eye examination in referable macular degeneration. Touvron, H. et al. Llama 2: open foundation and fine-tuned chat models. & Zamir, A. MultiMAE: Multi-modal Multi-task Masked Autoencoders. In Computer Vision – ECCV 2022 (eds Avidan, S. et al.) 348–367 (Springer-Verlag, 2022). of the 37th International Conference on Neural Information Processing Systems (eds Oh, A. et al.) 53728–53741 (Curran Associates, 2023). McMahan, B., Moore, E., Ramage, D., Hampson, S. & Arcas, B. A. Y. Communication-efficient learning of deep networks from decentralized data. of the 20th International Conference on Artificial Intelligence and Statistics (eds Singh, A. Chen, X. et al. FFA-GPT: an automated pipeline for fundus fluorescein angiography interpretation and question-answer. Singhal, K. et al. Toward expert-level medical question answering with large language models. Towards accurate differential diagnosis with large language models. Ting, D. S. W. et al. Development and validation of a deep learning system for diabetic retinopathy and related eye diseases using retinal images from multiethnic populations with diabetes. Wang, W. et al. Learning two-stream CNN for multi-modal age-related macular degeneration categorization. Prevalence and clinical characteristics of glaucoma in adult Chinese: a population-based study in Liwan District, Guangzhou. Bourne, R. et al. Trends in prevalence of blindness and distance and near vision impairment over 30 years: an analysis for the Global Burden of Disease Study. Eye disease and mortality, cognition, disease, and modifiable risk factors: an umbrella review of meta-analyses of observational studies. Xiong, K., Mao, H., Zhang, Q., Lei, C. & Liang, Y. Associations between vision impairment and multimorbidity among older Chinese adults: results from the China health and retirement longitudinal study. Singhal, K. et al. Large language models encode clinical knowledge. Lewis, J. R. & Sauro, J. in Human Centered Design (ed Kurosu, M.) 94–103 (Springer, 2009). Hillis, S. L. & Soh, B. P. Obuchowski-Rockette analysis for multi-reader multi-case (MRMC) readers-nested-in-test study design with unequal numbers of readers. This study was supported by the National Key R&D Program of China (2022YFC2502800), the National Natural Science Foundation of China (82388101) and the Beijing Natural Science Foundation (IS23096) to T.Y.W. ; the National Natural Science Foundation of China (62272298), the Noncommunicable Chronic Diseases-National Science and Technology Major Project (2023ZD0509202 & 2023ZD0509201), the National Key Research and Development Program of China (2022YFC2407000) to B.S. These authors contributed equally: Yilan Wu, Bo Qian, Tingyao Li, Yiming Qin, Zhouyu Guan, Tingli Chen, Yali Jia, Ping Zhang, Dian Zeng. These authors jointly supervised this work: Ya Xing Wang, Yih-Chung Tham, Ching-Yu Cheng, Tien Yin Wong, Bin Sheng. Yilan Wu, Yiming Qin, Dian Zeng, Yixiao Jin, Hongwei Ji, Ya Xing Wang & Tien Yin Wong Yilan Wu, Yiming Qin, Dian Zeng, Hongwei Ji & Tien Yin Wong Shanghai Belt and Road International Joint Laboratory for Intelligent Prevention and Treatment of Metabolic Disorders, Department of Computer Science and Engineering, School of Electronic, Information, and Electrical Engineering, Institute for Proactive Healthcare, Shanghai Jiao Tong University, Department of Endocrinology and Metabolism, Shanghai Sixth People's Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai Diabetes Institute, Shanghai Clinical Center for Diabetes, Shanghai Key Laboratory of Diabetes Mellitus, Shanghai, China Bo Qian, Tingyao Li, Yiming Qin, Zhouyu Guan, Huating Li, Ziyao Meng, Xiang Chen, Yuanqi Yao & Bin Sheng MOE Key Laboratory of AI, School of Electronic, Information, and Electrical Engineering, Shanghai Jiao Tong University, Shanghai, China Bo Qian, Tingyao Li, Yiming Qin, Ziyao Meng, Xiang Chen, Yuanqi Yao & Bin Sheng Department of Ophthalmology, Shanghai Health and Medical Centre, Wuxi, China Tingli Chen, Yi Huang, Xiyun Bian, Jing Wang, Xiaolong Yang, Haifang Zhang & Yihan Li Casey Eye Institute, Oregon Health and Science University, Portland, OR, USA Department of Biomedical Engineering, Oregon Health and Science University, Portland, OR, USA Department of Computer Science and Engineering, The Ohio State University, Columbus, OH, USA Department of Biomedical Informatics, The Ohio State University, Columbus, OH, USA Department of Ophthalmology & Visual Sciences, College of Medicine, The Ohio State University Wexner Medical Center, Columbus, OH, USA Sayoko Moroi, Joshua Evans, Alan Letson, Frederik Davidorf, Mona Adeli, Peter Chen & Thomas A. Mendel Rajiv Raman, Praveena Venkatakrishnan, Dhaivat Shah, Abhishek Kumar Tripathi, Dharshan Bhatt, Urvashi Kala & Abhishek Karra Benjamin Sommer Thinggaard, Benjamin Sommer Thinggaard, Frederik Pedersen, Bjarke Steenberg Smith, Andreas Abou Taha & Jakob Grauslund Tengku Ain Kamalden, José Alogo Obiang Ñehe & Celestino Edjang Nguema Mikue Universiti Malaya Medical Centre, Kuala Lumpur, Malaysia Tengku Ain Kamalden, Tengku Ain Kamalden, Tajunisah Iqbal, Penny Lott Pooi Wah, Marium Jamaluddin Ahmad, Nurul Najieha Amir & Irina Effendi-Tenang NIHR Biomedical Research Centre, Moorfields Eye Hospital NHS Foundation Trust, London, UK Yukun Zhou, Tengku Ain Kamalden, Tajunisah Iqbal, Penny Lott Pooi Wah, Marium Jamaluddin Ahmad, Nurul Najieha Amir & Irina Effendi-Tenang Yukun Zhou, Sobha Sivaprasad & Pearse A. Keane An Ran Ran, Dawei Yang, Simon K. H. Szeto, Julia Y. Y. Chan, Victor T. T. Chan, Sobha Sivaprasad & Pearse A. Keane Qingsheng Peng, Chenxi Zhang, Carol Y. Cheung, Yih-Chung Tham, Ching-Yu Cheng & Tien Yin Wong Centre for Innovation and Precision Eye Health and Department of Ophthalmology, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore Qingsheng Peng, Gavin Siew Wei Tan, Yih-Chung Tham & Ching-Yu Cheng State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat‐sen University, Guangdong Provincial Key Laboratory of Ophthalmology and Visual Science, Guangzhou, China Youxin Chen, Weihong Yu, Chenxi Zhang & Xinyu Zhao Youxin Chen, Weihong Yu, Rongping Dai, Chenxi Zhang & Xinyu Zhao Youxin Chen, Weihong Yu, Rongping Dai, Chenxi Zhang, Xinyu Zhao, Shiqun Lin & Yan Zhou Department of Ophthalmology, Union Hospital, Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China Department of Ophthalmology, The Second Affiliated Hospital of Naval Medical University (Shanghai Changzheng Hospital), Shanghai, China Institute for AI Industry Research, Tsinghua University, Beijing, China Shan Lin, Zejun Xu & Nian Guan The People's Hospital of Sixian County, Anhui, China Department of Ophthalmology, Pomeranian Hospitals, Wejherowo, Poland Department of Pediatric Ophthalmology, Kornel Gibinski University Clinical Center, Medical University of Silesia, Katowice, Poland Institute for Hospital Management, Tsinghua University, Beijing, China Guangdong Provincial Key Laboratory of Intelligent Information Processing, College of Electronics and Information Engineering, Shenzhen University, Shenzhen, China Department of Medicine and Therapeutics, The Chinese University of Hong Kong, Prince of Wales Hospital, Hong Kong Special Administrative Region, Hong Kong, China Baker Heart and Diabetes Insitute, Melbourne, Victoria, Australia Beijing Key Laboratory of Intelligent Diagnostic Technology and Devices for Major Blinding Eye Diseases, Tsinghua Medicine, Tsinghua University, Beijing, China Department of Ophthalmology, Peking University Third Hospital, Beijing, China Department of Ophthalmology, Vestfold Hospital, Tonsberg, Norway Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and You Wu helped with data collection. contributed to collaboration organization and provided critical revision of the manuscript for important intellectual content. All authors provided critical comments and reviewed the manuscript. Correspondence to Tien Yin Wong or Bin Sheng. is a patent holder of Optovue/Visionix, Inc., Optos plc and Genentech, Inc. She receives financial support from Genentech, Inc., and she receives financial compensation from Optovue/Visionix, Inc. and Genentech, Inc. P.A.K. is a co-founder of Cascader Ltd., has acted as a consultant for Retina Consultants of America, Roche, Boehringer Ingleheim and Bitfount and is an equity owner in Big Picture Medical. He has received speaker fees from Zeiss, Thea, Apellis and Roche. He has received travel support from Bayer and Roche. He has attended advisory boards for Topcon, Bayer, Boehringer Ingleheim and Roche. is a consultant for AbbVie Pte Ltd., Aldropika Therapeutics, Bayer, Boehringer Ingelheim, Zeiss, Genentech, Iveric Bio, Novartis, Opthea Limited, Plano, Quaerite Biopharm Research Ltd., Roche, Sanofi and Shanghai Henlius. He is an inventor, holds patents and is a co-founder of start-up companies EyRiS and Visre, which have interests in, and develop digital solutions for, eye diseases. All potential conflicts of interests for consultancy, advisory boards and positions in the start-up companies and financial renumeration, if any, are managed by institutional policies under SingHealth and Tsinghua University. The other authors declare no competing interests. Nature Medicine thanks Tae Keun Yoo and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Lorenzo Righetto, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. For patients attending ocular disease screening in the primary care centre where only low-cost examination technics are available, EyeFM can use its single modality and cross-modality ability to assist disease detection, followed with screening report writing assisted by its vision-question answering ability. Then, some patients will be referred to specialty care settings for further examination, where EyeFM can use its integrated modality disease detection or even zero-shot ability to facilitate further diagnosis. The image-report writing ability and vision-question answering can also assist to improve the efficiency of clinical report and patient letter drafting in specialty care settings. a) The diagram of pretraining process for EyeFM. EyeFM was first pretrained for its image encoder with five modalities of images, then conducted vision-language joint pretraining. The image module includes one encoder and five decoders. Each decoder comprises two Transformer blocks. The language module is based on LLaMA 2 architecture with 7 billion parameters. b) The diagram of human-in-the-loop process for EyeFM. EyeFM human-in-the-loop utilized DPO and federated learning for distributed knowledge evolution. First, we conducted retrospective validations, comparing EyeFM with prior benchmarks of medical foundation models. This step serves as the foundation for evaluating the model's performance and safety when progressing to clinical applications. Second, we conducted reader studies and real-world study prospectively to test the efficiency of EyeFM as a clinical copilot to assist ophthalmologists. At last, we validated EyeFM with a Randomised Controlled Trial (RCT). Colored chart, above, validation experiments and their corresponding relationship with the functions of EyeFM. Cells were coloured if the experiments have validated corresponding functions. Below, glossary table of the tasks, clinical scenario and experiments. The sample size for cataract detection on external eye photo was 198 and the P value was 0.102. The P value for haemorrhages was 0.083. The sample size was 759 for OCT segmentation. c) Cross-modality disease detection of ciDME that usually need to be diagnosed by OCT with CFP inputs only (left), the sample size was 405 and the P value was <0.001. Cross-modality disease detection of wet-AMD that usually need to be diagnosed by CFP with external eye photo inputs (right) the sample size was 332 and the P value was 0.583. d) Image-report generation, model performance was evaluated by automatic metrics labelled as the x-axis. e) Head-to-head comparison of answers generated by EyeFM and ophthalmologists, the measurement was summed score for quality, safety and empathy, ranged from 3–15 scores. P values were calculated with two-sided t-test between EyeFM and the better-performed reference model. CFP, colour fundus photo; OCT, optical coherence tomography; EEP, external eye photo; DR, diabetic retinopathy; DME, diabetic macular oedema; AMD, age-related macular degeneration; MMD, myopic macular degeneration; MA, microaneurysms; HE, hard exudates; HM, haemorrhages; SE, soft exudates. Participants included in the trial will first receive diagnosis and report by ophthalmologists by CFP, then receive additional OCT examinations for consultant-level reviewers to assess and revise the diagnosis and report. All diagnosis and reports before revision by consultant-level reviewers will be included in the analysis for correct diagnosis rate, correct referral rate and standardization score of reports. Only participants that are correctly diagnosed as ‘with fundus abnormality' will be included in the follow-up for patient compliance analysis. CFP, colour fundus photo; OCT, optical coherence tomography. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Wu, Y., Qian, B., Li, T. et al. An eyecare foundation model for clinical assistance: a randomized controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43018-025-01005-1'>Cancer-derived mitochondria fuel fibroblasts to become pro-tumorigenic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 09:14:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cancer-associated fibroblasts represent a functionally diverse and heterogeneous entity within the solid tumor microenvironment. Mitochondrial transfer from cancer cells to fibroblasts is now shown to act as a reprogramming stimulus, driving metabolic and functional differentiation of fibroblasts to support tumor growth. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 digital issues and online access to articles Prices may be subject to local taxes which are calculated during checkout Deyell, M., Garris, C. S. & Laughney, A. M. Br. Özdemir, B. C. et al. Cancer Cell 25, 719–734 (2014). Rhim, A. D. et al. Cancer Cell 25, 735–747 (2014). Elyada, E. et al. Cancer Discov. Cangkrama, M. et al. Nat. Biffi, G. et al. Cancer Discov. Caicedo, A. et al. Sci. Furnish, M. et al. Mol. Marabitti, V., Vulpis, E., Nazio, F. & Campello, S. Pharmacol. Foster, D. S. et al. Cancer Cell 40, 1392–1406.e7 (2022). Guo, J. L. et al. Tissue Eng. Department of Surgery, Stanford University School of Medicine, Stanford, CA, USA Daniel Delitto & Michael T. Longaker Hagey Laboratory for Pediatric Regenerative Medicine, Division of Plastic and Reconstructive Surgery, Stanford University School of Medicine, Stanford, CA, USA Daniel Delitto & Michael T. Longaker Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Michael T. Longaker. The authors declare no competing interests. Cancer-derived mitochondria fuel fibroblasts to become pro-tumorigenic. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02796-y'>Laser flash melting cryo-EM samples to overcome preferred orientation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 09:11:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Sample preparation remains a bottleneck for protein structure determination by cryo-electron microscopy. A frequently encountered issue is that proteins adsorb to the air–water interface of the sample in a limited number of orientations. This makes it challenging to obtain high-resolution reconstructions, or may even cause projects to fail altogether. We have previously observed that laser flash melting and revitrification of cryo-EM samples reduces preferred orientation for large, symmetric particles. Here we demonstrate that our method can in fact be used to scramble the orientation of proteins of a range of sizes and symmetries. The effect can be enhanced for some proteins by increasing the heating rate during flash melting or by depositing amorphous ice onto the sample prior to revitrification. This also allows us to shed light onto the underlying mechanism. Our experiments establish a set of tools for overcoming preferred orientation that can be easily integrated into existing workflows. Since the resolution revolution, cryo-electron microscopy (cryo-EM) has greatly expedited the process of protein structure determination1,2. Advances in instrumentation and single-particle analysis now routinely enable high-resolution reconstructions for a wide range of proteins and protein complexes3,4,5,6. However, sample preparation issues have remained a major bottleneck7. In particular, one common challenge involves preferred particle orientation8,9,10. When the sample solution is applied to the cryo-EM specimen support, the hydrophobic regions of the protein surface adsorb to the air–water interface, so that only a limited number of viewing directions are present after vitrification. Strong preferred orientation limits the resolution of a single-particle reconstruction along certain viewing directions and can even make it difficult to obtain a reconstruction at all11. Although several strategies can address preferred orientation issues, no universal solution has been found. Tilting the sample provides a wider range of particle views, but usually results in larger beam-induced motion, an increased ice thickness in the viewing direction and a defocus gradient across the viewing area12,13. Preferred orientation can be reduced by covering the air–water interface with detergents or small proteins to block adsorption of the particles of interest14,15. Moreover, different specimen supports, such as graphene, graphene oxide or other functionalized surfaces, can prevent some particles from diffusing to the interface16,17,18,19,20. However, adopting such strategies often requires time-consuming reoptimization of the sample preparation procedure. Adsorption at the air–water interface can also be minimized by limiting the time between sample application onto the specimen support and vitrification21,22,23,24,25,26. Although this interval can be shortened to a few milliseconds, particles generally diffuse to the interface within a few microseconds. Substantially faster preparation speeds would therefore be required to prevent the majority of particles from reaching the interface. Finally, machine-learning techniques have been developed to guess missing views27,28,29. However, such approaches cannot replace actual experimental observations. We have previously observed that flash melting and revitrification of cryo-EM samples with microsecond laser pulses reduces preferred orientation for some large, highly symmetric particles30,31,32. Flash melting seems to exert small forces on the proteins, altering their orientation and resulting in an uneven spatial distribution after revitrification, with particles clustering together in some areas30,32. High-resolution reconstructions suggest that the revitrification process preserves particle integrity and does not impose a fundamental limit on the obtainable spatial resolution30,33. However, the practicality of using flash melting to address preferred orientation and the underlying mechanism remain unclear. Here, we show that this effect extends to smaller proteins, as well as particles with lower symmetry. We also introduce two variants of the experiment that provide larger changes in preferred orientation for some proteins and that allow us to elucidate the underlying mechanism. Our experimental approach is illustrated in Figure 1a (refs. A cryo-EM sample is flash melted with 20-µs or 30-µs rectangular laser pulses, where the laser power is kept constant for the pulse duration (Fig. This causes some particles to detach from the air–water interface and change their orientation. Once the laser beam is switched off, the sample cools in just a few microseconds and revitrifies35,36, trapping the proteins in a non-equilibrium angular distribution (Fig. We performed in situ flash-melting experiments following previously described protocols, using a transmission electron microscope that we have modified for time-resolved experiments37,38. The sample is locally flash melted by aiming the laser beam (532 nm wavelength, 28-µm diameter spot size in the sample plane) at the center of a grid square of the holey gold specimen support (1.2 µm holes, 1.3 µm apart on 300-mesh gold). Typically, an area of 9–16 grid holes is revitrified. The samples are then transferred to a high-resolution electron microscope for imaging (Supplementary Figs. a, A cryo-EM sample is flash melted with a 30-µs laser pulse, which scrambles the orientation of the particles. When the laser is switched off, the sample revitrifies, trapping the particles in a non-equilibrium angular distribution. b, Schematic of the rectangular laser pulse shape. c–f Reconstructions, with the applied symmetry noted in parentheses, and angular distributions of conventional and revitrified samples (top and bottom, respectively) for the T20S proteasome (c), 50S ribosomal subunit (d), HIV-1 envelope ectodomain protein (e) and hemagglutinin (f). The SCF*, which is a measure of the degree of preferred orientation, as well as the resolution of the reconstructions are indicated. For hemagglutinin, the resolution appears to be overestimated, as judged by the quality of the map. Figure 1c demonstrates that flash melting reduces preferred orientation for a sample of the T20S proteasome. A reconstruction from a conventional sample is shown at the top, together with the angular distribution of the particles. It exhibits a series of maxima that arise from the T20S proteasome's strong preference to adsorb to the air–water interface on its side, resulting in sparse population of particles in the top views. Upon revitrification (bottom), the angular distribution broadens noticeably, with a larger fraction of the particles populating the less-favored views. At the same time, the sampling compensation factor (SCF*), a measure of the degree of preferred orientation, improves from 0.89 to 0.95. The SCF* takes values between 0 and 1, with 1 corresponding to a perfectly isotropic angular distribution5,39,40. An even greater reduction in preferred orientation can be achieved for the 50S ribosomal subunit. Unlike the highly symmetric T20S proteasome (point group D7), the asymmetric 50S ribosomal subunit (C1) exhibits strong preferred orientation (Fig. In a conventional sample, the angular distribution features a small number of pronounced maxima, with many views absent altogether (white) and a low SCF* of 0.42. Upon revitrification, most of the missing views become populated, and the SCF* improves to 0.74. At the same time, the streaky artefacts that are visible in the reconstruction from the conventional sample and that are the result of preferred orientation disappear (Supplementary Fig. Flash melting can also improve the angular distribution of much smaller proteins, such as the HIV-1 envelope ectodomain protein (210 kDa), for which the SCF* increases from 0.89 to 0.99 (Fig. Among these examples, a noticeable gain in resolution is observed only for the 50S ribosomal subunit, the only protein for which the preferred orientation is so pronounced that many views are not populated at all. This seems to be largely due to astigmatism in the micrographs recorded for the revitrified sample. Laser flash melting seems to be less efficient at reducing preferred orientation for proteins that are more strongly bound to the air–water interface. Hemagglutinin (170 kDa) exhibits a preference for top and bottom views (Fig. Although revitrification depopulates particles in the bottom views, the overall angular distribution remains similar, with the SCF* slightly decreasing from 0.60 to 0.56. This suggests either that flash melting detaches only a few proteins from the interface or that detached particles return to their preferred orientation rapidly after diffusing to the surface. Both explanations indicate that there is a strong interaction between hemagglutinin and the air–water interface. For the proteins studied here, diffusion to the interface is expected to occur in just a few microseconds, which is shorter than the duration of the experiment (30 µs). Experiments with shaped microsecond laser pulses provide clues about the mechanism through which the angular distribution of the particles is reshuffled. We have previously shown that cryo-EM samples partially crystallize during flash melting with rectangular laser pulses35,42. This led us to speculate whether the transient formation of crystallites, which preferentially form at the sample surface43, might exert small forces on the particles, leading to their detachment and reorientation30. We test this hypothesis by melting samples with shaped laser pulses that feature an intense 1-µs initial spike and ten times the laser power (Fig. This increases the heating rate during flash melting to about 2 × 108 K s−1, twice the critical heating rate of 1 × 108 K s−1, so that the sample does not crystallize44. If crystallization was the dominant mechanism for reshuffling the particle orientations, we would expect the angular distribution to barely change under these conditions. Contrary to our expectation, flash melting with such a shaped laser pulse decreases preferred orientation even more drastically for the 50S ribosomal subunit (Fig. The SCF* increases from 0.18 to 0.90 in revitrified areas of the sample, and the map resolution improves markedly, from 4.1 Å to 2.9 Å. Of note, the angular distribution of the sample is different from that in Figure 1d, despite both being prepared using the same procedure. This suggests that the interaction of the particles with the interface is very sensitive to preparation conditions and can be easily altered. a, A cryo-EM sample is flash melted with a shaped 30-µs laser pulse that features an intense leading edge. b, Schematic of the laser pulse shape, which features an intense spike at its onset (1-µs duration with the laser power increased ten-fold). c,d, Reconstructions and angular distributions of conventional and revitrified samples (top and bottom, respectively) for the 50S ribosomal subunit (c) and hemagglutinin (d). Clearly, transient crystallization of the sample cannot be the dominant mechanism for particle reorientation. Instead, the effect appears to intensify with increased heating rates, suggesting several possible mechanisms. Impulsive laser heating of a thin, suspended membrane, such as the holey gold film of the specimen support, induces drumming motions that typically persist on microsecond timescales45,46. With the larger initial heating power of the shaped laser pulses, such oscillations likely reach a larger amplitude and exert greater forces on the thin liquid film in which the particles are suspended. This could allow them to detach from the air–water interface more readily and reorient. Simulations indicate that, during the initial spike of the shaped laser pulse, the sample temperature briefly increases by about 25 K (Supplementary Fig. Because the evaporation rate increases exponentially with temperature48, this could cause more particles to detach. However, there are also changes in preferred orientation when samples are revitrified at atmospheric pressure, under which evaporation is substantially reduced33,49. It is therefore unlikely that evaporation is the dominant effect that causes particles to detach from the air–water interface. For hemagglutinin, even shaped laser pulses cannot reduce the amount of preferred orientation, although they induce some changes in the angular distribution (Fig. Our experiments suggest that the changes in the angular distribution induced by flash melting are the result of two opposing processes. First, particles must be successfully detached from the air–water interface to reorient. But given enough time, they can then diffuse back to the interface and return to their preferred orientation. A simple experiment allowed us to detach all particles from the interface before laser melting the sample, disentangling both processes (Fig. We deposited a 20-nm-thick layer of amorphous ice onto both sides of the sample by introducing water vapor into the volume surrounding the specimen in our modified transmission electron microscope50. After flash melting the sample with a rectangular laser pulse (Fig. 3b), the particles all began at a distance from the interface, having lost all memory of where they used to be located. The angular distribution obtained after revitrification then exclusively reflects the result of the particles' free diffusion. a, A 20-nm layer of amorphous ice is deposited in situ onto the cryo-EM sample before it is flash melted with a 30-µs laser pulse. b, Schematic of the rectangular laser pulse shape used. c–e, Reconstructions and angular distributions of conventional and revitrified samples (top and bottom, respectively) for the T20S proteasome (c), the 50S ribosomal subunit (d) and hemagglutinin (e). When we revitrify a T20S proteasome sample after depositing a layer of amorphous ice, we obtain the angular distribution in Fig. The estimated rotational timescale of the particles (about 5 µs) is shorter than the timescale of the experiment, so one might expect that this should yield an isotropic distribution. Instead, the distribution closely resembles that obtained without prior deposition of amorphous ice (Fig. 1c) and notably features a preference for side views. This clearly indicates that the particles have diffused back to the interface, which occurs on an estimated timescale of only about 4 µs, and have partially settled into their preferred orientations. The striking similarity of the angular distribution to that in Figure 1c suggests that both are the result of the free diffusion and partial readsorption of the particles. It therefore appears likely that, even without prior deposition of amorphous ice, the laser pulse efficiently detaches the T20S proteasome particles from the interface. Flash melting of a cryo-EM sample of the 50S ribosomal subunit, following the deposition of amorphous ice, similarly results in a broad angular distribution with a small degree of preferred orientation, indicating that some particles have readsorbed to the interface (Fig. The SCF* increased from 0.42 to 0.90. Curiously, however, the distribution substantially changed, featuring maxima at orientations that were previously only sparsely populated. This suggests that the interaction between the particles and the interface was altered in our experiments. As discussed above, the preferential orientation of the 50S ribosomal subunit varies, even in conventional samples prepared under otherwise identical conditions (Figs. 1d and 2c), suggesting that the properties of the interface are easily modified. Sample transfer and handling could have introduced traces of surface-active compounds, thereby modifying the chemical composition of the interface. At the laser powers used, less material is evaporated than has been deposited, so that the sample thickness slightly increases overall. As shown in Figure 3e, we obtain a similar result for hemagglutinin (Supplementary Fig. After amorphous ice deposition and revitrification, the particles show a substantial amount of preferred orientation, indicating that many have readsorbed to the interface. However, the prominent top views have been largely depopulated, whereas the bottom views now dominate the angular distribution. Our experiments yield a simple picture of the competing processes that occur during laser revitrification. Flash melting detaches some particles from the interface (Fig. 4a,b), a process that becomes more efficient as the heating rate is increased. As these particles rotate freely, their orientations are scrambled, resulting in improved angular distribution. In a competing process, particles diffuse back to the interface (Fig. 4c) before the sample is revitrified (Fig. 4d), allowing some proteins to efficiently readsorb in their preferred orientation. This process generally is more efficient for small proteins because of their shorter diffusion times. It could be possible to improve the angular distribution of such proteins by reducing the duration of the laser pulse to below the diffusion timescale (a few microseconds), so that the particles do not have enough time to reach the interface. In some experiments, proteins readsorbed in a different preferred orientation, such as when we deposited a layer of amorphous ice prior to revitrification. Apparently, our experiment has altered the interfacial properties, making adsorption in new orientations favorable. This implies that it should be possible to purposely modify the interface before flash melting, for example through vapor deposition of a hydrophilic compound onto the cryo-EM sample. Upon laser melting, all particles should then desorb and randomize their orientation. Such experiments are currently underway in our lab51. a, In a conventional cryo-EM sample, particles adsorb to the air–water interface in preferred orientations. b, Flash melting detaches the proteins from the interface, so that they are able to rotate freely. c, While the sample is liquid, many particles diffuse back to the interface, where some adsorb in their preferred orientation. d, The sample is revitrified and the ensemble is arrested in a non-equilibrium angular distribution. Our experiments also provide a practical toolbox for addressing preferred orientation, with some proteins showing dramatic improvements in their angular distribution. For example, revitrifying a cryo-EM sample of the 50S ribosomal subunit with shaped laser pulses improves the resolution by 1.2 Å under otherwise identical conditions (Fig. To put this in perspective, 18 times as many micrographs of a conventional sample would be needed to achieve the same improvement. Reshuffling the particle orientations through laser flash melting can therefore provide considerable cost and time savings. Our method can be easily combined with other established approaches for addressing preferred orientation. Importantly, as a simple physical approach, flash melting does not require any time-consuming changes to the sample-preparation procedure, but can be easily integrated into existing workflows. As we have previously shown, cryo-EM samples can also be revitrified in an optical microscope equipped for correlative light and electron microscopy experiments33. This approach is technically less involved and might therefore be more accessible to other labs. It could also be integrated into vitrification devices, some of which already use an optical microscope to assess sample quality. Laser-melting experiments could be directly integrated into high-resolution electron microscopes at cryo-EM facilities. If a sample is found to exhibit preferred orientation during data acquisition, it could then be revitrified on the fly with little additional time. Purified T20S proteasome (3.7 mg ml−1 in 50 mM Tris pH 8.0, 200 mM sodium chloride) and purified 50S ribosome52 (an optical density at 260 nm (OD260) of 40 per ml in 20 mM HEPES pH 7.5, 100 mM sodium chloride, 2 mM magnesium chloride) were gifts from A. Guskov, University of Gröningnen, and B. Beckert, DCI Lausanne respectively. The purified hemagglutinin sample (1 mg ml−1 in PBS pH 7.5) was purchased from MyBioSource (MBS434205). The cryo-EM samples of purified proteins were prepared by applying 3.5 µl of the sample solution to glow-discharged (0.25 mbar, 15 mA, 60 s) gold grids (UltrAuFoil, R1.2/1.3, 300 mesh, Quantifoil Micro Tools GmbH). Surplus sample was removed by blotting for 2–4 s using a blot force of 10 in a Vitrobot Mark IV (Thermo Fisher Scientific, held at 20 °C, 100% relative humidity). The HIV-1 envelope ectodomain protein (HIV-1 CH505TF Envelope SOSIP ectodomain)53 was generously provided by R. Henderson, Duke Center for HIV Structural Biology. The HIV-1 envelope ectodomain protein (5 mg ml−1 in 15 mM HEPES pH 7.4, 150 mM NaCl, 2.5% glycerol) was applied to glow discharged gold grids (Au-flat, 0.6 µm/1 µm hole diameter/pitch, 300 mesh, Electron Microscopy Sciences). The sample was blotted with the minimum blot time that can be set (0 s). The conventional sample was blotted using a blot force of 10, whereas the revitrified sample was blotted using blot forces of 5 and 10. All samples were plunge-frozen in liquid ethane and stored in liquid nitrogen until experiments were carried out. The revitrification experiments with rectangular laser pulses (Fig. 1) were performed in situ using a modified JEOL 2200FS transmission electron microscope equipped with a TemCam-XF416 (TVIPS) and an in-column Omega energy filter38. The samples were loaded into the microscope using a single-tilt side-entry cryo-transfer holder (Elsa, Gatan) held at a temperature of approximately 100 K. The sample was melted and revitrified in the center of multiple grid squares (typically 80 – 100) through irradiation with microsecond laser pulses30,32,35. A 30-µs laser pulse was used for the 50S and HIV-1 samples, and a 20-μs laser pulse for the T20S sample. The hemagglutinin sample was revitrified with two 30-µs pulses with a 250-ms delay between the pulses. The laser pulses were generated by chopping the continuous output of a 532 nm laser (Ventus532, Laser Quantum) with an acousto-optic modulator (AA Opto-Electronic). The laser beam was focused to a spot size of 28 μm FWHM (full width at half maximum) in the sample plane, as determined by a knife-edge scan, and aligned to the center of the grid squares. After revitrification, the grids were unloaded from the side entry holder and stored in liquid nitrogen until data collection. The revitrification experiments with shaped laser pulses (Fig. 2) were performed on a modified JEOL 2010F transmission electron microscope, equipped with a TemCam-XF416 (TVIPS)50,54,55. The plunge-frozen cryo-EM samples were inserted into the microscope using a single-tilt side-entry cryo-transfer holder (Elsa, Gatan) held at a temperature of approximately 100 K. Shaped laser pulses with an intense leading edge (1 µs initial spike at ten times the laser power) were generated by modulating the output of a continuous green laser (532 nm, Verdi Coherent) with an acousto-optic modulator (AA Opto-Electronic) that was controlled by an arbitrary function generator (Tektronix AFG1062)44. The laser beam was focused to a spot of 38 µm FWHM in the sample plane, as determined by imaging the beam with a CCD camera placed in a plane conjugate to the sample plane. The laser power required for laser melting was determined by using ordinary rectangular pulses. The initial spike was then added to perform the revitrification experiment. The experiments in Figure 3, in which the sample was revitrified after deposition of amorphous ice, were performed on a modified JEOL 2010F transmission electron microscope, equipped with a TemCam-XF416 (TVIPS)42,50,54,55. Amorphous ice was deposited by leaking water vapor into the volume surrounding the sample through a gas dosing valve50. The vapor enters the microscope through a stainless steel tube that terminates inside of the cold shield, in close proximity to the sample. To obtain a well-defined thickness, the deposition rate was first determined with the following procedure. A multilayer graphene sample on a gold specimen grid (UltrAu Foil, 600 mesh, R2/1) was cooled to about 100 K in an Elsa cryo-transfer holder (Gatan). While amorphous ice was being deposited on the graphene sheet, the diffraction patterns of the growing ice layer were recorded. During the deposition, the intensity of the molecular diffraction pattern of the amorphous ice initially increased linearly but then leveled off and plateaued once the ice film was so thick that multiple and inelastic scattering became important. From the characteristic curve shape of the diffraction intensity as a function of deposition time, the deposition rate was then determined (typically 0.3 nm s−1)42,50, and the deposition time was adjusted accordingly (typically 74 s to deposit about 10 nm on each side of the sample, 20 nm total). To perform the deposition and revitrification experiment, the cryo-EM sample was loaded into the microscope with a second Elsa cryo-transfer holder (Gatan, approximately at 100 K). To begin the deposition, the shutter of the specimen holder was opened for the predetermined amount of time, before the shutter was closed to stop any further deposition of amorphous ice. The gas dosing valve was then closed, and the cryo shield surrounding the sample was cooled to liquid nitrogen temperature. Once the pressure in the microscope column had dropped to below 5 × 10−5 Pa, the shutter of the cryo-transfer holder was opened, and the revitrification experiment was performed using rectangular laser pulses, as described above. The T20S datasets were recorded on a Titan Krios G3i, equipped with a BioQuantum energy filter and a K3 camera, at the Center for Microscopy and Image Analysis. The videos were recorded as tiff images in super-resolution mode and binned on the fly to a pixel size of 0.651 Å. The energy filter slit was set to 20 eV. The data were collected using a 100 µm objective aperture at a nominal magnification of ×130,000 during a 1.3-s exposure with a defocus range of –0.6 µm to –2.0 µm and a total dose of 67 e− Å−2. The datasets were processed in cryoSPARC v4.4 and v4.5 (ref. For the conventional sample, the revitrified sample and the sample revitrified after deposition of amorphous ice, 3,479, 6,105 and 3,641 micrographs were collected, respectively. The micrographs were subjected to patch motion-correction and patch contrast transfer function (CTF) estimation before manual curation on the basis of CTF resolution estimation, ice thickness and total full-frame motion. Particles were initially picked with a blob picker with a diameter of 100–200 Å from 1,519, 2,287 and 2,255 high-quality micrographs for the conventional sample, the revitrified sample and the sample revitrified after deposition of amorphous ice, respectively, and were then extracted with a box size of 512 px and down sampled to 128 px. After two rounds of two-dimensional (2D) classifications, the selected particles were used for ab initio reconstruction with two classes. The particles assigned to the better resolved class were refined against the obtained volume in a homogeneous refinement with C1 symmetry. The resulting volume was used to generate templates for template-based picking with a particle diameter of 150 Å. The particles were again extracted with a box size of 512 px, Fourier-cropped to a box size of 256 px, and subjected to two rounds of 2D classification. The selected particles were used for ab initio reconstruction with only one class, followed by homogeneous refinement with D7 symmetry. Subsequently, 50,000 particles were randomly selected, re-extracted at full box size and again directed into a homogeneous refinement with D7 symmetry. To ensure the comparability of the datasets obtained from the different experimental conditions, a low-pass-filtered volume (filtered to 30 Å) obtained from the conventional dataset was used as a reference input for homogeneous refinements for all the three datasets. Finally, an orientation diagnostics job in cryoSPARC was run to assess the angular distribution of the particles. The conventional, revitrified and deposited and revitrified 50S datasets were collected on a Titan Krios G4 equipped with a Falcon 4 camera at the Dubochet Center for Imaging facility in Lausanne, Switzerland. The videos were collected in eer format, with a 100 µm objective aperture inserted, at a nominal magnification of ×120,000, corresponding to a pixel size of 0.658 Å per pixel. Micrographs were collected with a defocus range of –0.4 µm to –1 µm using a total dose of 40 e− Å−2. The sample that had been revitrified with the shaped laser pulse and the corresponding conventional control were collected on a Titan Krios G3i, fitted with a BioQuantum energy filter and a K3 camera, at the Center for Microscopy and Image Analysis. The videos were recorded during a 1.2-s exposure at ×130,000 magnification, corresponding to a pixel size of 0.3255 Å in super-resolution mode. The data were binned on the fly to a pixel size of 0.651 Å per pixel. The energy filter slit was set to 20 eV and the objective aperture to 100 µm. The defocus used ranged from –0.4 µm to –1.6 µm, whereas the electron dose was 63 e− Å−2. For the 50S datasets, 11,121, 6,174, 9,532, 7,416 and 7,725 micrographs were collected for conventional, revitrified, deposited and revitrified, conventional (control for shaped pulses) and shaped pulse revitrified, respectively. The micrographs were patch motion-corrected (40 fractions, no up-sampling), and patch CTF-estimated before being subjected to a manual curation in which low-quality micrographs, judged by CTF estimation, relative ice thickness and total full-frame motion, were discarded. The selected 6,236, 5,870, 6,641, 3,485, and 5,227 high-quality micrographs were subjected to a blob picker with a particle diameter of 200–300 Å. The particles were extracted with a box size of 784 px and cropped to 392 px. After two rounds of 2D classification, selected particles were directed into an ab initio reconstruction with two to three classes. The particles from the best class were refined against the volumes from the best and worst class in one round of heterogeneous refinement. The selected particles from the heterogeneous refinement were directed into a homogeneous refinement with C1 symmetry. Subsequently, 50,000 particles were randomly selected, re-extracted at full box size and refined through homogeneous refinement using C1 symmetry and a low-pass filtered volume from the conventional dataset as a reference. Ultimately, an orientation diagnostics job in cryoSPARC was run to assess the angular distribution of the particles. Unlike the other reconstructions in this manuscript, the 50S ribosomal subunit maps were not sharpened to preserve the details in the flexible L1 stalk. The HIV-1 datasets were collected on a Titan Krios G4 using a Falcon 4i camera at the Dubochet Center for Imaging. The videos were recorded in eer format using a magnification of ×96,000, resulting in a pixel size of 0.83 Å per pixel. A total dose of 50 e− Å−2 was used and the defocus range was set to –1 µm to –2.4 µm. For the conventional HIV-1 dataset, 2,614 micrographs were collected. For the revitrified dataset, data were collected from two different grids; from the first grid, 5,447 micrographs were collected; from the second, 5,081 images were recorded. All datasets were subjected patch motion correction, patch CTF estimation and a manual curation based on CTF resolution estimation, relative ice thickness and the defocus range. Next, the particles were picked using a blob picker with a diameter of 100–150 Å. The particles were extracted with a box size of 480 px and down sampled to 240 px. The particles from the two revitrified datasets were combined before sorting in two rounds of two-dimensional classification. The selected particles were subjected to an ab initio reconstruction using two classes. Particles were further classified in three dimensions through two rounds of heterogeneous refinements, in which all selected particles from the first round of 2D classification were sorted against the good initial volume and five decoy classes, generated through the pre-mature termination of an ab initio reconstruction. The selected set of particles was then refined in a non-uniform refinement with C1 symmetry. Next, the particles were re-extracted at full box size, locally and globally CTF-refined before being directed into another non-uniform refinement, without dynamic masking and symmetry enforcement. Finally, 50,000 random particles were selected and again refined in a non-uniform refinement without dynamic masking using C1 symmetry and the filtered conventional volume as a reference, before being directed into an orientation diagnostics job. The four hemagglutinin datasets were recorded on a Titan Krios G4 equipped with a Falcon 4i and a SelectrisX energy filter at the Dubochet Center for Imaging. The videos were collected in eer format at a magnification of ×165,000, corresponding to a pixel size of 0.732 Å. A total dose of 40 e− Å−2 was used for the conventional, revitrified and leading-edge revitrified and deposited and revitrified dataset, respectively. A defocus range of –0.8 µm to –2.5 µm and an energy filter slit of 10 eV was applied for all four datasets. For the conventional, revitrified, deposited and revitrified and shaped pulse revitrified HA samples, 2,251, 6,476, 10,114 and 10,270 micrographs were collected. The micrographs were subjected to patch motion correction and patch CTF estimation. The micrographs were manually curated on the basis of their CTF resolution estimation, relative ice thickness and total full-frame motion, resulting in 1,651, 4,608, 7,982 and 8,590 high-quality micrographs. On 100 micrographs of each dataset, a denoiser was trained and subsequently applied to the curated micrographs. Particles were picked from denoised micrographs using a blob picker with a diameter of 80–140 Å. Next, the particles were extracted from the raw, non-denoised micrographs using a box size of 360 px, which was cropped to 180 px. The extracted particles were subjected to one to two rounds of 2D classification, followed by an ab initio reconstruction with two classes. The particles were then further cleaned up in one round of heterogeneous refinement using a decoy volume and the best-resolved ab initio volume as inputs. The sorted particles were then refined in a homogeneous refinement, before being re-extracted at full box size. Out of these, 50,000 random particles were selected and then subjected into a final homogeneous refinement with C3 symmetry, using the filtered conventional map as a reference, and an orientation diagnostics job. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Supplementary Information is available for this paper. Better, faster, cheaper: recent advances in cryo–electron microscopy. Nakane, T. et al. Single-particle cryo-EM at atomic resolution. Yip, K. M., Fischer, N., Paknia, E., Chari, A. & Stark, H. Atomic-resolution protein structure determination by cryo-EM. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Scheres, S. H. W. RELION: implementation of a bayesian approach to cryo-EM structure determination. Weissenberger, G., Henderikx, R. J. M. & Peters, P. J. Understanding the invisible hands of sample preparation for cryo-EM. & Glaeser, R. M. Retrospective on the early development of cryoelectron microscopy of macromolecules and a prospective on opportunities for the future. Routine single particle cryoEM sample and grid characterization by tomography. Glaeser, R. M. & Han, B.-G. Opinion: hazards faced by macromolecules when confined to thin aqueous films. Approaches to altering particle distributions in cryo-electron microscopy sample preparation. Addressing preferred specimen orientation in single-particle cryo-EM through tilting. Aiyer, S. et al. Overcoming resolution attenuation during tilted cryo-EM data collection. Abe, K. M., Li, G., He, Q., Grant, T. & Lim, C. J. Small LEA proteins mitigate air–water interface damage to fragile cryo-EM samples during plunge freezing. Chen, J., Noble, A. J., Kang, J. Y. Eliminating effects of particle adsorption to the air/water interface in single-particle cryo-electron microscopy: bacterial RNA polymerase and CHAPSO. Glaeser, R. M. et al. Factors that influence the formation and stability of thin, Cryo-EM specimens. High-yield monolayer graphene grids for near-atomic resolution cryoelectron microscopy. Protein denaturation at the air-water interface and how to prevent it. Cookis, T. et al. Streptavidin-affinity grid fabrication for cryo-electron microscopy sample preparation. Zheng, L. et al. Self-assembled superstructure alleviates air-water interface effect in cryo-EM. Need for speed: examining protein behavior during cryoEM grid preparation at different timescales. Dandey, V. P. et al. Time-resolved cryo-EM using Spotiton. Rima, L. et al. cryoWriter: a blotting free cryo-EM preparation system with a climate jet and cover-slip injector. Yang, Z. et al. Electrospray-assisted cryo-EM sample preparation to mitigate interfacial effects. Tyrrell, H. J. V. & Harris, K. R. Diffusion in Liquids – A Theoretical and Experimental Study. Structural dynamics: review of time-resolved cryo-EM. Zhang, H. et al. CryoPROS: correcting misalignment caused by preferred orientation using AI-generated auxiliary particles. & Zhou, Z. H. Resolving the preferred orientation problem in cryoEM reconstruction with self-supervised deep learning. Liu, Y.-T., Fan, H., Hu, J. J. & Zhou, Z. H. Overcoming the preferred-orientation problem in cryo-EM with self-supervised deep learning. Bongiovanni, G., Harder, O. F., Voss, J. M., Drabbels, M. & Lorenz, U. J. Near-atomic resolution reconstructions from in situ revitrified cryo samples. Lorenz, U. J. Microsecond time-resolved cryo-electron microscopy. Harder, O. F., Barrass, S. V., Drabbels, M. & Lorenz, U. J. Fast viral dynamics revealed by microsecond time-resolved cryo-EM. Bongiovanni, G., Harder, O. F., Drabbels, M. & Lorenz, U. J. Microsecond melting and revitrification of cryo samples with a correlative light-electron microscopy approach. Lorenz, U. J. et al. Methods to overcome preferred orientation in cryo-samples for single particle analysis. US patent 63/555160 (filed 19 February 2024). Voss, J. M., Harder, O. F., Olshin, P. K., Drabbels, M. & Lorenz, U. J. Microsecond melting and revitrification of cryo samples. Mowry, N. J., Kruger, C. R., Drabbels, M. & Lorenz, U. J. Direct measurement of the critical cooling rate for the vitrification of water. & Passmore, L. A. Ultrastable gold substrates for electron cryomicroscopy. Characterization of a time-resolved electron microscope with a Schottky field emission gun. Baldwin, P. R. & Lyumkis, D. Non-uniformity of projection distributions attenuates resolution in cryo-EM. Baldwin, P. R. & Lyumkis, D. Tools for visualizing and analyzing Fourier space sampling in cryo-EM. Lander, G. C. Single particle cryo-EM map and model validation: it's not crystal clear. Mowry, N. J., Krüger, C. R., Bongiovanni, G., Drabbels, M. & Lorenz, U. J. Backus, E. H. G., Grecea, M. L., Kleyn, A. W. & Bonn, M. Surface crystallization of amorphous solid water. Krüger, C. R., Mowry, N. J., Drabbels, M. & Lorenz, U. J. Shaped laser pulses for microsecond time-resolved cryo-EM: outrunning crystallization during flash melting. Kwon, O.-H., Barwick, B., Park, H. S., Baskin, J. S. & Zewail, A. H. Nanoscale mechanical drumming visualized by 4D electron microscopy. & Zewail, A. H. Biomechanics of DNA structures visualized by 4D electron microscopy. Astrath, N. G. C., Malacarne, L. C., Baesso, M. L., Lukasievicz, G. V. B. & Bialkowski, S. E. Unravelling the effects of radiation forces in water. List, R. J. Smithsonian Meteorological Tables 6th edn, Vol. Analysis of the evaporation coefficient and the condensation coefficient of water. Krüger, C. R., Mowry, N. J., Bongiovanni, G., Drabbels, M. & Lorenz, U. J. Electron diffraction of deeply supercooled water in no man's land. Curtis, W. A., Hruby, J., Krüger, C. R., Barrass, S. V., Drabbels, M. & Lorenz, U. J. Ultrathin liquid cells for microsecond time-resolved cryo-EM. Structural conservation of antibiotic interaction with ribosomes. LaBranche, C. C. et al. Neutralization-guided design of HIV-1 envelope trimers with high affinity for the unmutated common ancestor of CH235 lineage CD4bs broadly neutralizing antibodies. Olshin, P. K., Bongiovanni, G., Drabbels, M. & Lorenz, U. J. Atomic-resolution imaging of fast nanoscale dynamics with bright microsecond electron pulses. Bongiovanni, G., Olshin, P. K., Drabbels, M. & Lorenz, U. J. Intense microsecond electron pulses from a Schottky emitter. We thank B. Beckert (Dubochet Center for Imaging Lausanne), A. Guskov (University of Gröningen) and R. Henderson (Duke University) for providing the protein samples used in this study. A. Myasnikov, B. Beckert, S. Nazarov, I. Mohammed and E. Uchikawa, the members of the Dubochet Center for Imaging in Lausanne, and P. Szwedziak at the Center for Microscopy and Image Analysis in Zurich, are acknowledged for their assistance with cryo-EM data collection. We acknowledge current and former members of the lab for their help at various stages of the project. The work was supported by Swiss National Science Foundation Grant TMCG-2_213773 (U.J.L.) and by the Duke Center for HIV Structural Biology, NIH grant U54AI170752 (U.J.L. These authors contributed equally: Monique S. Straub, Oliver F. Harder, Nathan J. Mowry. Ecole Polytechnique Fédérale de Lausanne (EPFL), Laboratory of Molecular Nanodynamics, Lausanne, Switzerland Monique S. Straub, Oliver F. Harder, Nathan J. Mowry, Sarah V. Barrass, Jakub Hruby, Marcel Drabbels & Ulrich J. Lorenz Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar was responsible for conceptualizing this work. The methodology was performed by M.S.S., O.F.H., N.J.M., S.V.B. performed the data analysis. The writing of the original draft and data visualization were done by M.S.S. The reviewing and editing of the paper were performed by M.S.S. with the input of all coauthors. Correspondence to Ulrich J. Lorenz. The authors filed for three patents: patent application US-20250052992-A1 “Microsecond melting and revitrification of cryo samples with a correlative light electron microscopy setup”; publication date 13 February 2025; (OFH, MD, UJL). Patent application US 63/555,160 “Method to overcome preferred orientation in cryo-samples for single particle analysis” filed on 19 February 2024 (MSS, OFH, NJM, MD, UJL). Patent application US 63/767,702 “High-resolution liquid cells for microsecond time-resolved cryo-EM”; filed on 6 March 2025 (JH, MD, UJL). Nature Methods thanks Peter Peters and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Allison Doerr, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Straub, M.S., Harder, O.F., Mowry, N.J. et al. Laser flash melting cryo-EM samples to overcome preferred orientation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01662-x'>Dynamic TOM–TIM23 supercomplex directs mitochondrial protein translocation and sorting</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 09:11:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Structural & Molecular Biology (2025)Cite this article The mitochondrial translocase of the outer membrane (TOM) and translocase of the inner membrane 23 (TIM23) complexes are coupled to control protein import across the outer and inner membranes, respectively. However, the mechanisms of protein recognition and sorting in the TOM–TIM23 pathway remain unclear. Here we report cryo-electron microscopy structures of a translocating polypeptide substrate captured in the active TOM–TIM23 supercomplex from Saccharomyces cerevisiae. In the TOM complex, the polypeptide substrate adopts multiple conformations stabilized by hydrophilic residues from distinct regions of the Tom40 channel. In the TIM23 complex, the Tim17 and Mgr2 subunits create the translocation pathway, with a central restriction formed by four highly conserved hydrophobic residues. The substrate primarily interacts with hydrophobic residues along the Tim17–Mgr2 pathway. Substrate hydrophobicity modulates the association of Mgr2 with Tim17, enabling dynamic regulation of protein sorting toward either the matrix or membrane. These findings reveal a sophisticated translocation mechanism of the TOM–TIM23 supercomplex that ensures the efficient import of diverse mitochondrial proteins. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The cryo-EM maps of the substrate-engaged TOM complex and the substrate-engaged TIM23 complex were deposited to the EM Data Bank under accession numbers EMD-61256 and EMD-61257. The atomic structure coordinates of the substrate-engaged TOM complex the substrate-engaged TIM23 complex were deposited to the PDB under accession numbers 9J99 and 9J9B. MD simulation data are available on figshare (https://doi.org/10.6084/m9.figshare.27802086 (ref. Data and materials can be obtained from the corresponding authors upon request. Source data are provided with this paper. The code for the distribution landscape of TIM relative to TOM is available from GitHub (https://github.com/GuoQLabPKU/TIMTOM-distribution). The code for residue orientation analysis in MD is also available from GitHub (https://github.com/ComputBiophys/TIMOrientationAnalysis). Busch, J. D., Fielden, L. F., Pfanner, N. & Wiedemann, N. Mitochondrial protein transport: versatility of translocases and mechanisms. & Rehling, P. Protein transport along the presequence pathway. Bykov, Y. S., Rapaport, D., Herrmann, J. M. & Schuldiner, M. Cytosolic events in the biogenesis of mitochondrial proteins. Callegari, S., Cruz-Zaragoza, L. D. & Rehling, P. From TOM to the TIM23 complex—handing over of a precursor. Mokranjac, D. How to get to the other side of the mitochondrial inner membrane—the protein import motor. Araiso, Y., Imai, K. & Endo, T. Role of the TOM complex in protein import into mitochondria: structural views. Shiota, T. et al. Molecular architecture of the active mitochondrial protein gate. Structure of the mitochondrial import gate reveals distinct preprotein paths. Tucker, K. & Park, E. Cryo-EM structure of the mitochondrial protein-import channel TOM complex at near-atomic resolution. Guan, Z. Y. et al. Structural insights into assembly of human mitochondrial translocase TOM complex. Zarsky, V. & Dolezal, P. Evolution of the Tim17 protein family. Fielden, L. F. et al. Central role of Tim17 in mitochondrial presequence protein translocation. Sim, S. I., Chen, Y., Lynch, D. L., Gumbart, J. C. & Park, E. Structural basis of mitochondrial protein import by the TIM23 complex. Zhou, X. et al. Molecular pathway of mitochondrial preprotein import through the TOM–TIM23 supercomplex. Gebert, M. et al. Mgr2 promotes coupling of the mitochondrial presequence translocase to partner complexes. Ieva, R. et al. Mgr2 functions as lateral gatekeeper for preprotein sorting in the mitochondrial inner membrane. Rassow, J. et al. Mitochondrial protein import: biochemical and genetic evidence for interaction of matrix Hsp70 and the inner membrane protein Mim44. Schneider, H. C. et al. Mitochondrial Hsp70/Mim44 complex facilitates protein import. D'Silva, P., Liu, Q., Walter, W. & Craig, E. A. Regulated interactions of mtHsp70 with Tim44 at the translocon in the mitochondrial inner membrane. Ting, S. Y., Yan, N. L., Schilke, B. & Craig, E. A.Dual interaction of scaffold protein Tim44 of mitochondrial import motor with channel-forming translocase subunit Tim23. Kang, P. J. et al. Requirement for Hsp70 in the mitochondrial matrix for translocation and folding of precursor proteins. Liu, Q., D'Silva, P., Walter, W., Marszalek, J. Regulated cycling of mitochondrial Hsp70 at the protein import channel. Matouschek, A., Pfanner, N. & Voos, W. Protein unfolding by mitochondria. Global analysis of the mitochondrial N-proteome identifies a processing peptidase critical for protein stability. Mossmann, D., Meisinger, C. & Vogtle, F. N. Processing of mitochondrial presequences. Truscott, K. N. et al. A presequence- and voltage-sensitive channel of the mitochondrial preprotein translocase formed by Tim23. Alder, N. N., Jensen, R. E. & Johnson, A. E. Fluorescence mapping of mitochondrial TIM23 complex reveals a water-facing, substrate-interacting helix surface. Rapoport, T. A., Li, L. & Park, E. Structural and mechanistic insights into protein translocation. van den Berg, B. et al. X-ray structure of a protein-conducting channel. Ma, C. et al. Structure of the substrate-engaged SecA–SecY protein translocation machine. The mechanisms of integral membrane protein biogenesis. Dekker, P. J. et al. The Tim core complex defines the number of mitochondrial translocation contact sites and can hold arrested preproteins in the absence of matrix Hsp70–Tim44. Botte, M. et al. Cryo-EM structures of a LptDE transporter in complex with Pro-macrobodies offer insight into lipopolysaccharide translocation. Malhotra, K., Sathappa, M., Landin, J. S., Johnson, A. E. & Alder, N. N.Structural changes in the mitochondrial Tim23 channel are coupled to the proton-motive force. Hessa, T. et al. Molecular code for transmembrane-helix recognition by the Sec61 translocon. Herrmann, J. M., Neupert, W. & Stuart, R. A. Insertion into the mitochondrial inner membrane of a polytopic protein, the nuclear-encoded Oxa1p. Bohnert, M. et al. Cooperation of stop-transfer and conservative sorting mechanisms in mitochondrial protein transport. Park, K., Jung, S. J., Kim, H. & Kim, H. Mode of membrane insertion of individual transmembrane segments in Mdl1 and Mdl2, multi-spanning mitochondrial ABC transporters. Richter, F. et al. ROMO1 is a constituent of the human presequence translocase required for YME1L protease import. Nouwen, N., Berrelkamp, G. & Driessen, A. J. Charged amino acids in a preprotein inhibit SecA-dependent protein translocation. Gakh, O., Cavadini, P. & Isaya, G. Mitochondrial processing peptidases. Ford, H. C. et al. Towards a molecular mechanism underlying mitochondrial protein import through the TOM and TIM23 complexes. Ralser, M. et al. The Saccharomyces cerevisiae W303-K6001 cross-platform genome sequence: insights into ancestry and physiology of a laboratory mutt. Matsuda, T. & Cepko, C. L. Electroporation and RNA interference in the rodent retina in vivo and in vitro. Kirchhofer, A. et al. Modulation of protein properties in living cells using nanobodies. Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Kimanius, D., Dong, L., Sharov, G., Nakane, T. & Scheres, S. H. W. New tools for automated cryo-EM single-particle analysis in RELION-4.0. Zhang, K. Gctf: teal-time CTF determination and correction. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Wang, N. et al. Structural basis of human monocarboxylate transporter 1 inhibition by anti-cancer drug candidates. Jiang, W. et al. A transformation clustering algorithm and its application in polyribosomes structural profiling. Nucleic Acids Res. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Wittig, I., Braun, H. P. & Schagger, H. Blue native PAGE. Jo, S., Kim, T., Iyer, V. G. & Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Bussi, G., Donadio, D. & Parrinello, M. Canonical sampling through velocity rescaling. Parrinello, M. & Rahman, A. Polymorphic transitions in single crystals: a new molecular dynamics method. Darden, T., York, D. & Pedersen, L. Particle mesh Ewald: an N⋅log(N) method for Ewald sums in large systems. Hess, B., Bekker, H., Berendsen, H. J. C. & Fraaije, J. G. E. M. LINCS: a linear constraint solver for molecular simulations. Michaud-Agrawal, N., Denning, E. J., Woolf, T. B. & Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. Yang, Y. et al. Dynamic TOM-TIM23 supercomplex directs mitochondrial protein translocation and sorting. We thank X. Zhou for helping with sample preparation, N. Gao, X. Zhang and N. Li for helping with cryo-EM calculation and the National Center for Protein Sciences at Peking University for assistance with protein purification and mass spectrometry. The cryo-EM data were collected at the cryo-EM platform of Peking University. The computation was supported by the High-Performance Computing Platform of Peking University. This work was supported by the National Natural Science Foundation of China (32271269 to L.L. ), Qidong-SLS Innovation Fund (to L.L.) and National Key R&D Program of China (2024YFA0916800 to L.L. was supported in part by the Frontier Innovation Fund of Peking University Chengdu Academy for Advanced Interdisciplinary Biotechnologies. These authors contributed equally: Yuqi Yang, Shanshan Wang, Guopeng Wang, Yuke Lian. State Key Laboratory of Membrane Biology, School of Life Sciences, Peking University, Beijing, China Yuqi Yang, Shanshan Wang, Yuke Lian, Qiang Guo & Long Li School of Life Sciences, Peking University, Beijing, China Center for Quantitative Biology, Academy for Advanced Interdisciplinary Studies, Peking University, Beijing, China Lingfeng Xue & Chen Song Peking-Tsinghua Center for Life Sciences, Academy for Advanced Interdisciplinary Studies, Peking University, Beijing, China Wenhong Jiang, Qiang Guo, Chen Song & Long Li Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar prepared the protein samples for structural studies. performed the functional assays. collected the cryo-EM data and determined the structures. performed the MD simulations. analyzed the relative orientation between TOM and TIM23. supervised the project. All authors were involved in paper preparation. Correspondence to Chen Song or Long Li. The authors declare no competing interests. Nature Structural & Molecular Biology thanks Paul Schanda and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Katarzyna Ciazynska, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Size exclusion chromatography of the TOM-TIM23 supercomplex. The grey-shaded elution fractions are analyzed by BN-PAGE as shown on the right. b, Representative raw image of the supercomplex particles. c, Representative 2D classes of the supercomplex. Individual components of the supercomplex, including sfGFP, TOM, TIM23, and Fab, are visible in some classes. d, Flow chart of data processing for structure determination of the TOM complex. Details of data processing are provided in the Methods section. e, Focused 3D classification of the substrate in the Tom40 channel. f, Eulerian angle distribution of the particles used for TOM reconstruction, generated from RELION. g, Gold standard Fourier shell correlation (FSC) curve. The estimated resolution of the final map at 0.143 is generated from RELION. a, Flow chart of data processing for structure determination of the TIM23 complex. The starting particles are derived from the TOM calculation (Extended Data Fig. b, Representative 2D classes of the TIM23 complex. c, Gold standard Fourier shell correlation (FSC) curve. The estimated resolution of the final map at 0.143 is generated from CryoSPARC. d, Eulerian angle distribution of the particles used for TIM23 reconstruction, generated from CryoSPARC. e, 3D classification of the TOM-TIM23 supercomplex using the selected particles from focused TOM and TIM23 classifications and refinements. f, Distribution landscape of TIM23 relative to TOM. The dots indicate the spatial positions of TIM23 relative to TOM. The dots are color-coded by particle density, with red representing the areas of highest concentration. a, Density maps (meshes) and models of the selected peptide segments from the TOM complex. b, Density maps (meshes) and models of the selected peptide segments from the TIM23 complex. c, Density maps (meshes) and models of the selected peptide segments from and the substrate in the TIM23 complex are shown. Residues at the beginning and end of each segment are labeled. The residue side chains are shown as sticks. a, Structural comparison of Tom7 in the idle structure (Mandy's pink, PDB ID: 6UCU) and the active structure (forest green). The two copies of the subunit in a dimeric TOM complex are both shown in a side view and a top view. The phospholipid clamped by Tom7 and Tom40 in the active structure is shown as yellow balls and sticks with transparent density. b, Structural model of an anti-GFP Macrobody. MBP and the anti-GFP nanobody are colored khaki and plum, respectively. c, Purification of the supercomplex complexed with the Macrobody. The elution fractions (grey shaded) from size exclusion chromatography (left) are analyzed by BN-PAGE (top right) and SDS-PAGE (bottom right). BN-PAGE is stained with Coomassie blue. SDS-PAGE is subjected to immunoblotting with an anti-His tag antibody for detection of the Macrobody and Tom22. d, Cryo-EM analysis of the supercomplex with the Macrobody. The raw image (left), 2D classification (middle), and 3D reconstruction (right) are shown. The GFP density in the 2D classes is marked by green arrows. The resolution estimations are from the outputs of 3D classification using RELION. e, Focused 3D classification of the substrate in the Tom40 channel. The six classes are the calculation results from Extended Data Fig. The three representative classes are also shown in Fig. a, Superimposition of the idle (PDB ID: 8SCX) and active structure of Tim17-Tim23-Tim44. The subunits in the idle structure are all colored grey. Tim17, Tim23, and Tim44 in the active structure are colored the same as in Fig. 2. b, Superimposition of Tim17 in the idle (grey) and active (brownish pink) structures. c, Superimposition of Tim44-CTD in the idle (grey) and active (dark sea green) structures. d, Fitting of the idle Tim23 and Tim44 structures into the density of the active state. The density map is low pass filtered to 6 Å to emphasize the overall fitting the model. Views are shown from the Tim23 side (left) and top (right). Idle Tim23 and Tim44 are colored cyan and magenta, respectively. Tim44 in the active state is colored lime. TM3-4 of the Tim23 idle structure is outside the density map of the active state. The side chains of the Tim44 α2-3 helices in the active state are shown as sticks, with hydrophobic and hydrophilic residues labeled in black and lime, respectively. The shift of the Tim44 α2 and α3 helices is indicated by an arrow. e, Comparison of cryo-EM densities of the active and inactive TIM23 complexes (EMD-27825). Side views illustrate the thinned membrane at the Tim23 side of the active structure. The density map is low-pass filtered to 15 Å, with estimated membrane thicknesses labeled. f, Alignments of Tim17 sequences from representative eukaryotic species. The sequences are from Saccharomyces cerevisiae (S. cerevisiae, Uniprot P39515), Neurospora crassa (N. crassa, Uniprot P59670), Schizosaccharomyces pombe (S. pombe, Uniprot P87130), Kluyveromyces lactis (K. lactis, Uniport Q6CXS9), Candida glabrata (C. glabrata, Uniprot A0A0W0CKS7), Caenorhabditis elegans (C. elegans, Uniprot O44477), Drosophila melanogaster (D. melanogaster, Uniprot Q9VGA2), Mus musculus (M. musculus, Uniprot Q545U2), Homo sapiens (H. sapiens, Uniprot Q99595). g, Alignments of Mgr2 sequences from representative eukaryotic species. The sequences are from Saccharomyces cerevisiae (S. cerevisiae, Uniprot Q02889), Neurospora crassa (N. crassa, Uniprot Q7SDL7), Schizosaccharomyces pombe (S. pombe, Uniprot P79082), Caenorhabditis elegan (C. elegans, Uniprot Q93511), Drosophila melanogaster (D. melanogaster, Uniprot Q9VUM2), Mus musculus (M. musculus, Uniprot P60603), Homo sapiens (H. sapiens, Uniprot P60602). a, Surface presentations of Tim17-Mgr2 in a side view (left) and a cutaway side view (right). The lateral opening between Tim17 and Mgr2 in the lower half of the complex is outlined by black lines. The translocation pathway inside the complex is outlined by green lines. The restriction residues are colored green. b, Hydrophobicity distribution along the Tim17-Mgr2 translocation pathway. c, Charge distribution along the Tim17-Mgr2 translocation pathway. d, Surface presentations of the Sec translocon (PDB ID: 8RJC), shown in a side view (left) and a cutaway side view (right). The restriction (pore ring) residues are colored green. e, Hydrophobicity distribution in the Sec channel. f, Charge distribution in the Sec channel. g, Superimposition of the restriction residues in the Tim17-Mgr2 pathway and the Sec channel. Tim17, Mgr2, and Sec (PDB ID: 6ITC) are colored brownish pink, gold, and grey, respectively. The restriction residues of Tim17-Mgr2 and Sec are shown as balls and sticks, and colored forest green and grey, respectively. h, The substrate exit site of Tim17-Mgr2. The translocation direction toward the β sheet of Tim44 is indicated by a black arrow. i, same as h, except showing the hydrophobicity distribution of the exit site. The polypeptide substrate is represented as a curved line. a, Mapping of the substrate in the TIM23 complex. The substrate residues around the α1 and α2 regions of the B2 segments are mutated to Bpa. The supercomplexes are assembled in the yeast strain yLS420 transformed with the plasmids pLSP2-x that encode the photoactive substrates. The cells are subjected to UV irradiation, followed by Mgr2 pulldown. The substrate, Mgr2, and their crosslinking products are detected by immunoblotting against the StrepII tag (Mgr2) and Flag tag (substrate). Position 155 of the substrate outside the TIM23 complex is included as a negative control. b, Comparison of the substrate passing through the restrictions of Tim17-Mgr2 (left) and the Sec channel (right, PDB ID: 6ITC). Tim17 and Mgr2 are colored brownish pink and gold, respectively. The N-half and C-half of the Sec channel are colored sky blue and plum, respectively. The restriction residues are shown as balls and sticks with those from Tim17-Mgr2 in crystal blue and those from Sec in red. The substrate density is colored green. c, Spot assays to examine the growth of yeast strains with point mutations of the substrate-interacting residues of Tim17. d, Same as c except with additional Mgr2 knockout. e, Tim17 mutant selection on 5-FOA plates. The yeast strains, yLS270, yLS280, and yLS290, are transformed with Tim17 mutation plasmid pLSA4-X and selected on 5-FOA plates. The mutants that could not grow during selection are shown. f, Import of the metabolite carrier GGC1 via the TIM22 pathway into mitochondria with mutations of the restriction residues in Tim17. The imported and correctly folded GGC1 is detected as a ~ 100 kDa band in BN-PAGE 78. a, Side chain orientations of all substrate residues from α2 of the modeled substrate during simulated translocation through the Tim17 groove in the presence (left) or absence (right) of Mgr2. This panel extends the analysis from Fig. 4c to include all substrate residues, providing a comprehensive view of their orientations. b, Initial model used for the MD simulations. The side chains of the substrate are labeled and shown as balls and sticks. c, MD simulations of the substrate-engaged TIM23 complex, demonstrating that the complex remains impermeable to water diffusion, irrespective of Mgr2's presence. Water molecules are depicted as red and white spheres. The orange spheres are the phosphorous atoms of lipids. a, Scheme of the supercomplex assembly, showing two adjacent positions 135 and 139 of the substrate located in the TIM23 complex. Both positions are mutated to an amber codon for Bpa incorporation. b, Photo-crosslinking of the substrate with double Bpa sites. In wild type mitochondria, the double crosslinking product containing the substrate, Tim17, and Mgr2 could be detected. When Mgr2 is knocked out, no higher crosslinking bands above the “substrate x Tim17” band are observed. MD simulations trajectories illustrating the translocation of a polypeptide substrate through the Tim17–Mgr2 pathway. The substrate was steered into the matrix (downward) by a pulling force in MD simulations. The polypeptide is depicted in green with side chains highlighted as balls and sticks. Mgr2 is shown in transparent yellow. Key hydrophobic residues of Tim17 and Mgr2 that interact with the substrate are shown as color-coded balls and sticks: orange, F65 of Tim17; lime, W68 of Tim17; purple, L12 of Tim17; deep pink, F67 of Mgr2; silver, F70 of Mgr2. Notably, the applied force was not constant and could be accumulated in certain polypeptide-trapped locations during simulations. This could result in uneven translocation rates of the polypeptide substrate. Same as Supplementary Video 1, except that the substrate was pulled toward the IMS. Same as Supplementary Video 1, except that the simulation was conducted without Mgr2. Same as Supplementary Video 2, except that the simulation was conducted without Mgr2. Unprocessed western blots and gels. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Dynamic TOM–TIM23 supercomplex directs mitochondrial protein translocation and sorting. Nat Struct Mol Biol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Structural & Molecular Biology (Nat Struct Mol Biol) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250827010736.htm'>These “plastivore” caterpillars can devour a plastic bag in just 24 hours</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-28 04:41:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research reveals how “plastivore” caterpillars can metabolically degrade plastics in a matter of days, not decades, and store them internally as body fat – but at what cost? While this plastic degradation process has been demonstrated by waxworms at a small scale, this ongoing research project is helping us to better understand the biological mechanisms at work, the impact of an all-plastic diet on the health of these organisms, and their viability as a sustainable solution to plastic pollution. “Around 2,000 waxworms can break down an entire polyethylene bag in as little as 24 hours, although we believe that co-supplementation with feeding stimulants like sugars can reduce the number of worms considerably,” says Dr Bryan Cassone, a Professor of Insect Pest and Vector Biology in the Department of Biology at Brandon University, Canada. This research reveals that waxworms metabolically process the plastics down into lipids and store it as body fat. “This is similar to us eating steak – if we consume too much saturated and unsaturated fat, it becomes stored in adipose tissue as lipid reserves, rather than being used as energy,” says Dr Cassone. Dr Cassone and his team have identified two ways in which waxworms could contribute solutions to the ongoing plastic pollution crisis. As a bonus benefit, the mass production of waxworms would also generate a substantial surplus of insect biomass, which could represent an additional economic opportunity in aquaculture. “Our preliminary data suggests that they could become part of a very nutritious diet for commercial food fishes,” says Dr Cassone. These Simple Everyday Treatments Offer Drug-Free Relief for Knee Arthritis Common Painkillers Like Ibuprofen Could Be Fueling a Global Health Threat This Everyday Food Ingredient Might Be Messing With Your Brain Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            