
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-28</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/tampa-breaks-heat-record-as-heat-dome-bakes-eastern-u-s/'>Scorching Heat Dome Sends Tampa to 100 Degrees F for the First Time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 18:35:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Tampa Breaks 100 Degrees F for First Time on Record as Heat Wave Bakes Eastern U.S. The NWS HeatRisk is an experimental color-numeric-based index that provides a forecast risk of heat-related impacts to occur over a 24-hour period. “We're frequently over 90—for three, four months a year, almost every day it gets above 90,” says Tyler Fleming, a meteorologist at the National Weather Service's Tampa Bay office. Surrounded by water, Tampa—and Florida at large—is usually cursed with enough humidity to keep the overall air temperature, as a thermometer measures it, a bit lower. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. He says there wasn't any special factor that caused Tampa's heat record to occur on Sunday beyond the extremity of the current situation. Tampa is the highest-profile city to see a heat record fall. But the current bout of extreme heat has tied record temperatures in several other cities, including Jacksonville, Fla., and Charlotte, N.C. “When that high pressure is overhead, it pushes the air down,” Fleming says. “As the air sinks, it compresses and heats up.” If you live in an affected area, check out Scientific American's science-backed tips for staying healthy in extreme heat and for keeping your house cool. The current heat dome is expected to linger for several more days until the high-pressure system migrates westward, which, Fleming says, should return the region to what he calls “a more typical summer pattern.” On July 28, 16 million people are at extreme risk, which NWS describes as “rare and/or long-duration extreme heat with no overnight relief”; another 135.9 million are at major risk. The map estimates that by July 29, more than 12 million people will remain at extreme risk, and nearly 150 million will be at major risk. And as of July 30, nearly 115 million people are expected to be at major or extreme risk. The widespread high risk will not begin to abate until July 31. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02008-y'>Neural anticipation of virtual infection triggers an immune response</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 16:09:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Once contact with a pathogen has occurred, it might be too late for the immune system to react. Here, we asked whether anticipatory neural responses might sense potential infections and signal to the immune system, priming it for a response. We show that potential contact with approaching infectious avatars, entering the peripersonal space in virtual reality, are anticipated by multisensory–motor areas and activate the salience network, as measured with psychophysics, electroencephalography and functional magnetic resonance imaging. This proactive neural anticipation instigates changes in both the frequency and activation of innate lymphoid cells, mirroring responses seen in actual infections. Alterations in connectivity patterns between infection-sensing brain regions and the hypothalamus, along with modulation of neural mediators, connect these effects to the hypothalamic–pituitary–adrenal axis. Neural network modeling recapitulates this neuro–immune cross-talk. These findings suggest an integrated neuro–immune reaction in humans toward infection threats, not solely following physical contact but already after breaching the functional boundary of body–environment interaction represented by the peripersonal space. A vital function of an organism is to anticipate contact with threats to promptly activate a proper ‘fight-or-flight' response. Pathogens represent special forms of threats that must be detected and avoided. Through evolution, social species developed a series of behavioral responses, such as social distancing, aimed at preventing contacts and thus infections that have been termed the ‘behavioral immune system'3,4,5,6. In primates, a mechanism that might be functional to predict contact with potential harm has been described within a network of fronto–parietal neurons, which integrate tactile stimuli on the body with external sensory information close to the body, that is, the peripersonal space (PPS) system7,8. Once an external stimulus comes in contact with the body, another system reacts, that is, the immune system, composed of early- and late-acting arms (innate and adaptive immunity, respectively)9. A concerted effort of these two arms secures efficient pathogen clearance while preserving host tissue integrity10. Although reciprocal regulation of the neural and immune systems during actual disease is a research field in development11,12,13,14, there is no evidence of mutual interaction between the behavioral and biological immune systems that anticipates a concerted response to a potential infection before physical pathogen encounters. Here, we asked whether the human brain is equipped with a mechanism of early sensing and anticipation of contact with virtual infections that is able to trigger a reaction of early players of the immune system, that is, innate lymphocytes (for example, innate lymphoid cells (ILCs) and natural killer (NK) cells) similar to when responding to contact with a physical pathogen. We exposed healthy participants to potential infection threats using virtual reality (VR) in the form of human faces displaying clear signs of infection (infectious avatars) and entering participants' PPS. By using psychophysics, electroencephalography (EEG), functional magnetic resonance imaging (fMRI), mass spectrometry (MS) and flow cytometry, we measured behavioral, neural and immune responses to infectious avatars compared to responses evoked by control stimuli or actual contact with pathogens (that is, injection of a flu vaccine). Our results show that potential contact with virtual infection threats is predicted by fronto–parietal areas of the PPS system, activates the salience network and triggers a cascade of neuro–immune mediators, ultimately inducing changes in ILC frequency and activation. To trigger specific responses associated with virtual infection threats, we created a set of avatars showing clear signs of infection (infectious avatars) and two control conditions, namely neutral and fearful avatars (that is, an arousing but not pathogenic threatening stimulus; Fig. Results from explicit ratings, a seating distance scale15 and the Implicit Association Task16 demonstrated that infectious avatars were perceived as sick and contagious and evoked implicit avoidance responses compared to neutral and fearful avatars (Supplementary Table 1). Participants were first exposed to a neutral avatar (baseline) and then assigned to one of three cohorts encountering neutral, infectious or fearful avatars in a second session. Participants responded to tactile stimuli on their face, delivered either in empty space (unisensory) or combined with an avatar approaching (multisensory). Tactile stimuli (flash icon) were presented at five delays, with the avatar located at one of five distances (D5 = far, D1 = near). Reaction times (RTs) to multisensory stimuli were normalized by unisensory RTs (dotted line) and plotted by avatar distance. Shaded areas illustrate the PPS gradient, defined by significantly faster multisensory RTs (P < 0.05, corrected; paired-sample t-tests); N = 15 per cohort. Data are shown as mean ± s.e.m. (see Supplementary Table 3 for full statistics). d–h, EEG responses recorded from the central electrode (Cz) as an example (N = 16 per cohort). Black segments denote statistically significant time windows (cluster-based, nonparametric statistical tests; P < 0.01, corrected; solid lines = mean EEG response; dotted lines = 95% confidence interval). d, Evoked responses comparing unisensory (T) and multisensory (VT) stimulation as an index of multisensory processing. e, GFP analysis of PPS-related processing, that is, VT response differences for near versus far avatars. Insets show near–far current distribution differences at peak GFP effects. f, Significant GFP difference for near versus far stimulation between infectious and neutral avatars in the infection cohort. g, No GFP differences were found between two neutral avatars in the control cohort. h, Significant GFP differences between infectious and neutral avatars at the far location in the infection cohort, with the inset showing peak current distribution difference. We then tested whether infectious avatars induced a specific response from the PPS system by applying a validated multisensory paradigm measuring the spatial extent of PPS at the behavioral level6,17. Participants were asked to respond as fast as possible to a tactile stimulation on their face while an avatar face was shown as approaching in immersive VR (Fig. Reaction times (RTs) to touch18,19 were measured when the approaching avatar was at five possible distances (D5–D1; multisensory RT) and normalized for RT when no avatar was presented (unisensory tactile (T) RT). The PPS extent was then indexed as the distance of the visual stimulus inducing a multisensory facilitation effect, that is, faster RT to multisensory than unisensory stimuli (hereafter the PPS effect). To define a baseline for VR exposure, in the first session, all participants were exposed to neutral avatars, and in a second session, participants were divided into three cohorts and exposed to one of the three specific avatars, namely infection, neutral and fearful cohorts (Fig. 1a; the different cohorts were matched for participant sensitivity to disgust and anxiety; Supplementary Table 3). Thus, infectious avatars specifically elicited a PPS effect at farther distances, indicating that the PPS system anticipates potential contacts with a virtual pathogenic threat when it is still far from the body. We next searched for a neurophysiological marker of early detection of virtual infection compared to neutral stimuli entering the PPS. The visuo–tactile (VT) PPS paradigm was thus adapted to high-density EEG (128-channel EEG), in which participants were presented with either neutral or infectious avatars (infection cohort) or two different sets of neutral avatars (control cohort) and randomized in an event-related design. First, a cluster-based nonparametric statistical procedure was applied to determine a time window associated with VT multisensory processing (that is, in the presence of an avatar) compared to T processing only (Fig. Within this time window, we computed the Global Field Power (GFP20) as an index of evoked potential and compared GFP multisensory responses when a near or far avatar was presented as an index of PPS processing21,22. Results from both cohorts demonstrated a near–far difference arising from parietal and premotor areas, in line with previous neuroimaging studies23,24 (Fig. 1e), showing that the presence of an approaching avatar activates PPS-related brain areas. We then tested whether this PPS effect differed when an infectious or neutral avatar was presented. In the infection cohort, the contrast [near (neutral – infectious) – far (neutral – infectious)] showed a significant GFP difference between 129 and 150 ms (Fig. To understand the origin of this effect, we compared GFP responses within this time window in the presence of an infectious or neutral avatar in the near or far space. We found a significant difference for far stimuli between the infectious and neutral avatars, whereas no difference was observed for near stimuli. In the control cohort, no difference was found between the two sets of neutral avatars (Fig. Thus, a virtual infection threat evoked a different neural response than a neutral stimulation, already when presented at a far location, in line with and extending the behavioral results presented earlier (Fig. Inverse solution analyses illustrated that the source of this difference was localized to parietal areas, which are part of the PPS system (Fig. These electrophysiology findings demonstrate an anticipatory detection of virtual infections entering the PPS by multisensory–motor areas. To further localize these responses and clarify their directionality (see ‘Limitations of the study' section), we conducted an fMRI experiment (see below). We then tested whether virtual infections trigger an actual immune response. A new sample of participants was first exposed to neutral avatars, and then blood samples were taken immediately after the neutral VR stimulation to define an equivalent baseline. The participants were then divided into three cohorts (matched for sensitivity to disgust and anxiety; Supplementary Table 3), which were exposed to infectious, neutral or fearful avatars, and blood samples were taken after VR stimulation. To compare the immune response to virtual infection threats to that of a real pathogen, we tested a fourth cohort of participants who were not stimulated with VR but received an influenza vaccine (vaccine cohort), an ethically acceptable surrogate model for pathogen inoculation in humans (Fig. We assessed the early/antigen-independent phase of the lymphoid response by measuring the peripheral blood frequency and activation of innate lymphocytes, that is, ILCs and NK cells (Fig. To study a global index of the immune response, principal component analysis (PCA) was used on ILCs and NK cells (Supplementary Figs. Results revealed that virtual and real infections induced a similar stronger modulation of ILC frequency and activation indexes (evaluated as the differential expression of the activation markers CD25, CD27, CD69, NKp30, NKp44, NKp46, KLRG1, PD1 and HLA-DR) than neutral and fearful avatars (Fig. The main finding (that is, virtual avatars induce a stronger modulation of ILC frequency and activation indexes than neutral avatars) was further confirmed in an independent experiment (Fig. Participants were first exposed to neutral avatars (baseline). Blood samples were collected immediately after the baseline. Participants were then assigned to the neutral, infection or fearful cohort and exposed to the corresponding avatar condition (second session consisted of a 20-min cohort-specific VR stimulation, a 90-min break and a 10-min same cohort-specific VR stimulation). Blood samples were collected immediately at the end of the second VR session. In the vaccine cohort, blood samples were collected before and after flu vaccination (at the same time delay as for the VR cohorts, that is, with a 120-min time break between the first and second blood sampling). b, Scheme showing ILC identification from blood sampling. Peripheral blood mononuclear cells (PBMCs) were isolated, stained, and analyzed by flow cytometry. ILCs were identified as FCSlowSSClow lymphocytes that were negative for lineage markers and positive for CD127. c, ILC frequency was measured at the baseline and after the second session. The synthetic index of ILC frequency changes (first PCA component) in the four experimental cohorts is shown (analysis of variance (ANOVA): F3,56 = 4.71, P = 0.0053). d, Synthetic index of ILC activation changes (first PCA component) in the four cohorts (see Supplementary Fig. 8 for activation marker expression in each ILC subset; ANOVA: F3,56 = 5.45, P = 0.0023). e,f, Synthetic index of ILC frequency changes (e) and ILC activation changes (f) in two different independent cohorts (only neutral and infectious avatars were tested); N = 16 distinct participants per cohort (P values were derived from two-tailed t-tests). g, Correlation between synthetic ILC frequency and activation indexes, with different colors denoting the different cohorts (R = 0.85, P < 0.0001); shaded ellipses indicate the 66% confidence interval for the mean of each cohort. Black dots and lines represent the mean (dot) ± s.e.m. N = 15 distinct participants per cohort (c, d and g). ILC frequency and activation indexes showed a strong correlation and were both enhanced by virtual and real infections compared to the neutral cohort (Fig. No change was found in NK cell subset distribution and activation in response to real and virtual infections (Supplementary Figs. Because the ILC family comprises three main subsets (excluding NK cells), namely ILC1s, ILC2s and ILC precursors (ILCPs), we analyzed the frequency and activation variations of each subset individually to understand which one contributed to the difference observed in the global ILC analysis. 3a), suggesting that all three subsets are responsible for the global changes observed. The reduction in ILC1 frequency was associated with a higher activation index in the virtual and real infection cohorts than in the neutral cohort (Fig. These results suggest that heightened ILC1 activation, concomitant with decreased ILC1 frequency (Supplementary Fig. 14), is likely a consequence of their activation-induced migration from the blood to tissues, consistent with their involvement in early antiviral tissue responses29. Together, these data show that ILCs react to infections not only when they are detected in the body but also when they are processed as a potential threat approaching the body. Left, representative dot plots of ILC1s identified as CRTH2−cKit−, ILC2s as CRTH2+cKit+/− and ILCPs as CRTH2−cKit+ among total ILCs (that is, living lineage−CD127+ lymphocytes; see Fig. 2b) at baseline and after the second VR session. b, ILC activation by flow cytometry. Left, representative histograms of the percentage of CD25+ ILCs at baseline and after the second VR session as an illustrative example of ILC activation. Right, synthetic index of ILC1, ILC2 and ILCP activation changes (first PCA component) in the different cohorts (ANOVA ILC1s: F3,56 = 3.68, P = 0.017; ILC2s: F3,56 = 2.49, P = 0.07; ILCPs: F3,56 = 2.67, P = 0.056). In a and b, N = 15 distinct participants per cohort. To gather deeper insight into the localization and response pattern of the brain network detecting infectious avatars, we conducted an fMRI experiment by adapting the PPS paradigm to an fMRI-compatible VR setup. Within the same participants, we compared brain activation induced by tactile stimuli associated with either infectious or neutral avatars, presented in far space (that is, where infectious avatars evoked specific behavioral and electrophysiological effects) or in the near space. To disentangle specific brain processing responsible for the detection of infection threats from responses to a generic environmental threat, another group of participants was presented with either fearful or neutral avatars, with the same design. We first identified multisensory processing from a distributed bilateral fronto–parieto–occipital network (Fig. To test whether the activation of the PPS system, when participants faced a far infectious avatar, reflected anticipatory processing, we contrasted the blood oxygen level-dependent (BOLD) responses to tactile stimuli coupled with infectious versus neutral avatars in the far and near spaces. Activations (infectious > neutral) in the intraparietal sulcus and visual areas were common to both far and near conditions. In turn, activations in the right primary somatosensory cortex (S1), right anterior insula (aINS), bilateral premotor cortex, bilateral anterior cingulate cortex (ACC) and bilateral middle frontal gyrus (MFG) were specific for the far conditions (Fig. The direct contrast between responses to infectious versus neutral avatars in the far compared to the near space highlighted additional specific activations in the left medial prefrontal cortex (mPFC) and left aINS (Fig. These regions are part of the so-called salience network, that is, an ensemble of interconnected brain regions whose major role is detecting and filtering salient stimuli, including threats30, to recruit relevant functional networks implementing proper reactions. Some of the areas activated in our task (intraparietal sulcus, ACC and MFG) have been reported to be triggered during a cognitive task performed under an inflammatory state induced by a vaccine31. Here, we show that the PPS network and the salience network respond to virtual infections to implement fast responses. Importantly, this pattern of brain activations was specific to detection of virtual infection, as they also emerged when we directly compared activity induced by infection and fearful avatars (after subtracting the response to neutral avatars; Fig. We then studied how this brain mechanism of infection threat detection might project to the immune system to trigger an immune response, as the ILC reaction shown above. a, In the infection cohort, contrasts (red: VTFi + VTNi; blue: VTFn + VTNn) revealed VT activations specific to each face type. Activated regions included the bilateral superior/inferior parietal lobules (SPL and IPL, respectively), S1, supplementary motor area (SMA), middle cingulate cortex (MCC), temporo–parietal junction (TPJ), face-sensitive visual areas (occipital, fusiform and lingual gyri), cerebellum and insula. b, Similar activation patterns were observed for the fearful cohort using equivalent contrasts (N = 20). c, Infection-specific activations (infectious > neutral) varied by avatar proximity as shown for far space (red), near space (blue) and their direct contrast (green). d, Comparison between infectious and fearful avatars (versus neutral) for far (red) and near (blue) space and their direct contrast (green); N = 38. e, DCM revealed modulated hypothalamic connectivity during exposure to infectious avatars. Increased input: from mPFC and aINS. Decreased input: from MFG and ACC. fMRI activations were thresholded at cluster-level family-wise error correction, P < 0.05; i, infection; n, neutral; F, far; N, near; VISv, ventral visual areas; VISd, dorsal visual areas; OPJ, occipito–parietal junction; IPS, intraparietal sulcus; PMC, premotor cortex; MTG, middle temporal gyrus; HYP, hypothalamus. In a, c and e, N = 18 participants. Behavioral responses to threats are described in terms of ‘fight-or-flight', and their neural implementation has been characterized in rodents within a network of areas including the frontal cortex, amygdala, hypothalamus and periaqueductal gray32. The lateral hypothalamus is also a key area in the regulation of innate immune responses by the central nervous system, which could mainly occur by direct release of catecholamines via the sympathetic nervous system and by corticosteroids via the hypothalamic–pituitary–adrenal cortex (HPA) axis11,33. These responses might be further modulated by a number of signaling factors, including eicosanoids and neuroinflammatory mediators34,35. Thus, we first searched for involvement of the HPA axis in responding to virtual infection. We used dynamic causal modeling (DCM) analyses to reveal stimuli-specific changes in connectivity between the hypothalamus and key areas of the PPS system and the salience network being activated by infectious avatars compared to neutral or fearful avatars. Results showed different modulation of hypothalamic connectivity when infectious avatars were presented in the far space compared to neutral avatars. In particular, far infectious avatars were associated with connectivity upregulation between the hypothalamus and the aINS, MFG, mPFC in the left hemisphere and S1 in the right hemisphere (Fig. No modulation was found from the same analyses for conditions with avatars in the near space nor in the fearful avatar group. Thus, fMRI connectivity results point to a modulation of hypothalamus activity via the salience network in response to virtual infections anticipated by the PPS system that might be the upstream node of the neuro–immune pathway triggering the systemic innate immune response. Previous data showed that ILC functions are modulated by HPA-related hormones in models of endotoxin-induced systemic inflammation36. In line with these observations, ILC modulation by infectious avatars might be detected on a cascade of effects along the HPA axis. Thus, we performed MS quantification of a set of HPA-related hormones in the serum of individuals in the infection and neutral cohorts (Supplementary Table 9 and Supplementary Fig. Brain- and systemic-derived molecules, such as catecholamines and nonsteroidal metabolites of the arachidonic acid pathway, are largely involved in inflammatory responses, leukocyte chemotaxis, temperature and blood pressure regulation during infection34,35,41. Thus, we quantified eicosanoids as arachidonic acid metabolites (Supplementary Table 10) and neuroinflammatory factors (Supplementary Figs. 18 and 19) to study the cross-talk between neural signaling and immune responses triggered by infectious compared to neutral avatars. We first applied a data reduction approach to neural mediators by running three independent PCAs on HPA-related hormones, eicosanoids and neuroinflammatory factors (Fig. We extracted the first component of each PCA (explaining 39, 36 and 33% of the variance, respectively; Supplementary Figs. 20 and 21) as a synthetic index of each pathway, and we used it to predict the pattern of elicited immune response as captured from the ILC activation index. Univariate or multivariate linear regressions between neural signals and immune activation revealed no conclusive pattern, suggesting that the neuro–immune cross-talk could not be explained by simple relationships. Thus, we applied a machine learning-based approach, whereby the three synthetic indices of neuro–immune signaling were used to predict ILC activation in a single hidden layer neural network (Fig. We trained a single hidden layer neural network using the first PCA component of eicosanoids (neutral versus infectious; t(28) = 1.25, P = 0.23), HPA-related hormones (neutral versus infectious; t(28) = 2.04, P = 0.051) and neuroinflammatory factors (neutral versus infectious; t(28) = 2.06, P = 0.049) as input and the ILC activation index as an output on all participants from the neutral and infection cohorts. To compare network predictions with actual ILC activation, we used leave-one-out cross-validation; N = 15 distinct participants per cohort. P values of two-sided two-sample t-tests (neutral versus infectious) are reported. b,c, Comparison between predicted (x axis) and measured (y axis) ILC activation indexes in all participants (P = 0.000011; b) and differentiation between the infection and neutral cohorts in red (P = 0.000604) and gray (P = 0.215002; c), respectively. Linear regressions were performed (shaded area = 95% confidence interval); N = 15 distinct participants per cohort. d, Predicted ILC activation as a function of HPA-related hormones and eicosanoids. The network's input–output relation in two dimensions is shown from training a network on the whole population, while the neuroinflammation level was fixed at its mean value. e, Measured eicosanoids and HPA-related hormones in participants from the infection (red dots) and neutral (gray dots) cohorts (with neuroinflammation input fixed at its average); N = 15 distinct participants per cohort. Most participants in the infection cohort fall within the predicted activation hot spot (P = 0.008, two-sided χ2 test) indicated by the red line. The neural network was able to explain 71% of the variance in the empirical data by uncovering multivariate, nonlinear relationships between neural signaling and immune activation, with a stronger prediction power for the infection cohort, even if information about the cohort was not used for training (Fig. To interpret such complex, multidimensional relationships in the neuro–immune cross-talk, we mapped the predicted immune response as a function of the level of HPA-related hormones, eicosanoids and neuroinflammatory factors. We found that ILC activation increased almost linearly with HPA-related hormone levels and decreased almost linearly with neuroinflammatory factor levels. ILC activation was facilitated when eicosanoid levels were within a specific level, resulting in a nonlinear, Gaussian-shaped response profile. As a result, the highest predicted immune response was found in a ‘hot spot' of high HPA-related hormones, low neuroinflammatory factors and intermediate eicosanoid levels (Fig. As an empirical confirmation of such predicted effects, we found that actual data from participants from the infection cohort had a significantly higher probability than participants from the neutral cohort to fall into the hot spot predicted by the neural network (Fig. The same neural network trained on the neutral and fearful groups did not reveal significant relationships between neuro–immune signaling and ILC response (Supplementary Fig. The network also failed when any of its three inputs was removed, suggesting that all three families of signaling factors are essential to drive the immune response. Together, these analyses suggest that a virtual infection threat (and not a generic threat) induces a specific pattern of neuro–immune signaling, which is sufficient to drive ILC activation. Our integrated behavioral, neurophysiological, immunological and computational analyses provide a direct demonstration that potential infection threats (even when presented in VR) are processed by the PPS system and the salience network in an anticipatory way and preactivate the immune system by triggering ILC responses, likely via a nonlinear neuro–immune cross-talk involving the HPA axis. An important line of work describes the neural mechanisms allowing animals to detect threats and select appropriate behaviors1,7,8,42. Our results extend this framework to the response of the immune system to virtual infection threats. One key mechanism to defend oneself from infection is by early detection of potential threatening encounters to promptly decide whether to engage in ‘fight-or-flight'. Therefore, anticipation can result in a pathogen avoidance strategy, for example, by adopting social distancing behaviors as for the recent coronavirus disease 2019 (severe acute respiratory syndrome coronavirus 2) pandemic6,43,44,45, or in a pathogen defense approach, by activating immune cells to destroy the infection threat. Thus, in the absence of a consolidated avoidance policy, the findings from the infection cohort suggest that our immune system adopts fight anticipatory strategies, allowing our organism to react to immune threats not only once they are in the body but also when they overcome the primary functional boundary of self-environment interaction represented by the PPS. These results extend the spectrum of responses of the behavioral immune system from adapting social behavior4 to driving innate immune responses of effectors and counter-regulatory loops. Although surprising, our finding that immune responses can be triggered by simulated infections presented in VR is consistent with the principle of the smoke detector in biological systems46. Accordingly, the behavioral immune system evolved to minimize false-negative responses and is exquisitely sensitive to cues that superficially resemble the symptoms and signs of pathogen infections. In this study, VR proved to efficiently deliver such cues, demonstrating its potential as a valuable approach to further dissect the vital and complex link between the central nervous system and the immune system. First, given the lack of prior research on immune responses to virtual brain stimulation in humans, this study is exploratory. Second, the selection of immune markers, timing of virtual and real (vaccine) pathogen exposures, neuroinflammatory factors and use of machine learning were guided by current scientific knowledge and available tools. Third, the generalizability of the results requires further studies because ILC changes were only compared to one vaccine (FluarixTetra 2018–2019, used in the 2018 Swiss anti-influenza campaign). Additionally, as immune and PPS systems vary with age47,48,49, our focus on young adults limits the applicability to other age groups. Furthermore, we used a well-established paradigm with looming stimuli to probe PPS responses18,19, but whether static images elicit similar immune effects remain unknown and requires future research. Finally, the EEG results show different responses to infectious and neutral avatars from PPS-related areas, especially in the far space, that is, at the PPS boundary for infection stimuli. However, the direction of the GFP modulation is not fully consistent with the electrophysiology literature on PPS representation (normally showing higher responses for PPS-related stimuli). This might depend on the use of looming stimuli, compared to static stimuli used in previous studies21,22,23, of VT stimuli rather than audio–tactile stimuli and on the sampled spatial distances. Future studies should better clarify this point. In the present study, the fMRI results better allowed us to interpret the directionality of brain responses to infectious avatars. A theoretical limitation in the field is the potential confound between perceived infectiousness and disgust. Disgust is central to the behavioral immune system and helps avoid infectious threats50. Infectious-looking avatars may trigger more disgust than neutral ones, but this factor cannot be fully separated from contamination-avoidance mechanisms. We addressed this by (1) ensuring that our cohorts were matched for disgust sensitivity (Supplementary Table 3) and (2) showing that including disgust as a covariate in fMRI analyses did not affect the key results (Supplementary Fig. This research complies with all relevant ethical regulations. The experiments were conducted in accordance with the principles of the 1964 Declaration of Helsinki and were approved by the ethical committee ‘Commission cantonale d‘éthique de la recherche sur l‘être humain' of the Vaud and Geneva cantons, Switzerland (201701588). The different experimental cohorts were age and sex matched (total of 132 women; mean age = 26.8 years, range 18–49). Within each experiment, the different experimental cohorts (infectious, fearful and neutral) were tested for differences in sensitivity to disgust and anxiety, and no difference was found (Supplementary Table 1). All participants were recruited for a single experiment through the participant management software ‘Sona-Systems'. Participants were right-handed, reported no history of neurological, psychiatric or immune disorders, had no somatosensory impairments and had normal or corrected-to-normal vision. None reported specific diseases making the acquisition of blood samples unsafe (for example, hemophilia and anemia). The vaccine cohort was vaccinated with FluarixTetra 2018–2019. Participants were financially reimbursed for their time (20 Swiss Francs per h), and all provided written informed consent before participating. Visual stimuli were presented stereoscopically with a head-mounted display (Oculus Rift CV1, 110° field of view, resolution 2,160 × 1,200, refresh rate of 90 Hz). All visual stimuli consisted of female or male virtual faces with different facial features that were sex matched to the participants. Avatar faces could present a neutral expression, a fearful expression or signs of infectious disease and sickness. All avatars were created with ‘Poser 10 Pro' software (posersoftware.com) and were postprocessed using Blender (www.blender.org). Neutral and fearful avatars were selected from previous validated literature51. For infectious avatars, a validation procedure consisted of an initial database of 22 virtual faces (11 men and 11 women), each showing signs of sickness (Supplementary Methods 1 and 2). Tactile stimulation was delivered using a pair of cylindrical mechanical vibrators (Precision Microdrives, Pico Vibe Vibration Motor, diameter of 10 mm, stimulation frequency of 100 Hz, stimulation duration of 200 ms) taped to the cheeks of the participants and interfaced using a custom-made programmable microcontroller, which also sampled at 10 kHz the response times provided by a hand-held press button. For the fMRI experiment, a magnetic resonance-compatible pneumatic stimulator was used instead of non-magnetic resonance-compatible mechanical vibrators. To measure whether infectious avatars affected PPS representation, we adapted a well-validated multisensory interaction task previously used to measure the extent of the PPS behaviorally17,18,19. In this task, participants are required to press a button as fast as possible after receiving an automatized mild touch to their face, while concurrently observing a task-irrelevant visual stimulus looming toward their face. In different trials, touch is delivered when the visual stimulus is perceived at a different distance from the participant, and RT is measured to derive a proxy of the extent of PPS (Supplementary Method 3). Custom Python (v3.7) code was used to present the VR stimuli, based on the ExpyVR toolbox (v1.0; https://www.epfl.ch/labs/lnco/research/expyvr/) EEG data were acquired using a 128-channel system (ActiveTwo, Biosemi V.O.F.). Statistical analyses were performed in two steps. First, we identified time windows responding to multisensory (VT) stimuli (versus T). Then, within these time windows, we characterized distinct PPS responses for neutral and infectious avatars. In the first step, we compared multisensory (VT near and VT far) to unisensory (T) responses (Fig. To increase the power of this analysis, for each distance (near and far), we combined EEG data of condition 1 (neutral avatars) and condition 2 (neutral avatars in the control cohort and infectious avatars in the infection cohort) of both cohorts (control and infection). Significant differences between T and VT (near and far) were determined with a cluster-based, nonparametric statistical procedure as implemented in the Fieldtrip toolbox (v20171231 with MATLAB v2017a)52,53. This data-driven approach controls for the false-positive error rate in a situation of multiple comparisons (multiple time points and electrodes). Significant time windows (between −100 ms and 400 ms) in the contrast between VT and T were considered as multisensory responses and were selected for the second step of the analyses. In the second step, we conducted analyses based on the classic approach to study PPS22,23,54, in which PPS is defined as a multisensory modulation of tactile stimulation due to an external stimulus (here visual presentation of avatars), as a function of the distance of these stimuli from the body in space. We first estimated the PPS distance effect (VT near versus VT far) with EEG data from both cohorts together (Fig. All statistical analyses in the second step were performed on GFP. To localize neural activity in key contrasts, we performed a current density analysis in three-dimensional Tailarach/MNI space of scalp-recorded electrical activity using the sLORETA/eLORETA software package55. sLORETA estimates the distribution of electrical neural activity in three-dimensional space based on the measurements of a dense grid of 6,239 voxels at 5-mm spatial resolution, which are placed on the entire scalp surface covering the brain. This inverse solution algorithm assumes related orientations and strengths of neighboring neuronal sources (represented by adjacent voxels). Figure 1e,h shows the difference in estimated current distribution at the time point showing the strongest statistical difference in GFP within each significant time window. Venous blood was drawn from 60 healthy donors immediately after the baseline session (consisting of a 20-min VR stimulation with neutral avatars or following arrival of the participants assigned to the vaccine cohort) and after the second session (consisting of two VR stimulations accordingly to the assigned cohort). The second session consisted of a 20-min VR exposure, 90-min break and another 10-min VR exposure. Thirty individuals were randomly assigned to the neutral and infection cohorts (n = 15 per cohort), with an equal number of tested participants per condition per day. Two independent cohorts were further enrolled and assigned to the fearful or vaccine cohort. Participants assigned to the vaccine cohort waited for the same time delay as participants exposed to VR. Therefore, the time interval between the blood samplings was 120 min in all cohorts. To replicate the main findings, a new independent sample of participants (n = 16 per cohort, see Statistics for the sample size calculation) was recruited and randomly assigned to the neutral or infection cohort. For comparisons between the neutral and infection conditions, participant cohort assignment was not communicated to the immunologists analyzing blood samples to ensure blindness in the analysis. PBMCs were isolated by Lymphoprep centrifugation (600g, 20-min centrifugation without break, room temperature) and washed twice with PBS. Platelets were removed by centrifugation (200g, 10-min centrifugation with break, room temperature), and red blood cells were eliminated by incubating the cell pellets with 1 ml of red blood cell lysis buffer for 5 min at 37 °C. For serum collection, whole blood was centrifuged (1,800g, 10 min, room temperature). Following centrifugation, the serum was collected and immediately cryopreserved at −80 °C. Dead cells were excluded using the viability dye DAPI (Invitrogen, 1:10,000). Samples were acquired on an LSR SORP flow cytometer (BD using the BD FACSDiva software v8.0.2), and data were analyzed using FlowJo software v10.7.1. For the replication experiment, samples were acquired on an LSR Fortessa flow cytometer (BD using the BD FACSDiva software v8.0.2), and data were analyzed using FlowJo software v10.8.1 (TreeStar). This combination of markers allowed us to identify NKbright cells as living lineage−CD16−CD56bright lymphocytes, NKdim cells as living lineage−CD16+CD56dim lymphocytes and total ILCs as living lineage−CD16−CD56−CD127+ lymphocytes. ILCs were further divided into CRTH2−cKit− ILC1s, CRTH2+cKit+/− ILC2s and CRTH2−cKit+ ILCPs. The other markers were used to assess the activation status of NK cells and ILCs and were analyzed and plotted using GraphPad Prism version 10.4.1 (Supplementary Figs. All data were acquired on a Siemens Prisma 3T magnetic resonance scanner with a 32-channel receiver/transmitter head coil (see Supplementary Method 5 for fMRI adaptation of the PPS task). Functional volumes were acquired using a gradient echo planar imaging sequence over the whole brain (TR: 1,000 ms; TE: 32 ms; slice thickness: 2 mm; 66 axial slices; in-plane resolution: 2 × 2 mm2; multislice acceleration factor: 6). Four functional runs were acquired with the presentation of the experimental conditions, each with 380 volumes. Furthermore, T1-weighted structural images (mprage sequence, sagittal orientation, resolution: 1 × 1 × 1 mm3, TR: 2,000 ms, TE: 2.25 ms, flip angle: 8°) were recorded after the acquisition of functional images. All images were preprocessed using SPM12 software (with MATLAB v2021a) (Wellcome Department of Cognitive Neurology). Preprocessing steps included slice timing correction, realignment, minimal smoothing (full-width at half-maximum = 3 mm) and normalization to MNI space. A generalized linear model analysis, including the four experimental runs, was performed to estimate the BOLD responses (beta estimates) associated with the different experimental conditions. The model included seven regressors, one for each experimental condition, convoluted with the hemodynamic response, as well as the six rigid-body motion parameters and the frame-wise displacement as nuisance regressors56 (total of seven nuisance regressors). contrasts 1 and 2 to highlight the multisensory activations common to infectious/fearful and neutral conditions: contrast 5 to highlight the difference between contrasts 3 and 4: These single-participant contrasts were then used to compute the following group-level statistical tests: (1) one-sample t-test with contrast 1 in the infectious cohort (VTinfectious), (2) one-sample t-test with contrast 2 in the infectious cohort (VTneutral), (3) one-sample t-test with contrast 1 in the fearful cohort (VTfearful), (4) one-sample t-test with contrast 2 in the fearful cohort (VTneutral), (5) one-sample t-test with contrast 3 in the infectious cohort (dVTFi: VTFi > VTFn), (6) one-sample t-test with contrast 4 in the infectious cohort (dVTNi: VTNi > VTNn), (7) one-sample t-test with contrast 5 in the infectious cohort (dVTFi > dVTNi), (8) two-sample t-test (infectious versus fearful) with contrast 3 (dVTFi > dVTFf), (9) two-sample t-test (infectious versus fearful) with contrast 4 (dVTNi > dVTNf) and (10) two-sample t-test (infectious versus fearful) with contrast 5 (dVTFi > dVTNi) > (dVTFf > dVTNf). Finally, to control that our results cannot be explained by a simple effect of disgust during exposure to infectious avatars (for example, an increase in brain activity when disgust is experienced), we computed an additional one-sample t-test at the group level with contrast 3 by including as a covariate an assessment of sensitivity to disgusting stimuli using the germ aversion questionnaire57. Whole-brain results were corrected for multiple comparisons using family-wise error cluster-level correction at P < 0.05 (with a primary threshold of P < 0.005; Supplementary Fig. DCM (SPM12) analyses were performed to investigate the connectivity between the hypothalamus (central node of the HPA axis) and the cortical regions that were associated with infectious avatars presented in far space. The cortical regions activated by infectious avatars presented in the far space were selected from group comparisons 5 and 7. To extract the time series from each selected region, a generalized linear model with the four functional runs concatenated was computed. We directly modeled BOLD fluctuations associated with the difference between infectious and neutral conditions by defining a ‘task' condition (for example, VTF for both type of faces) and an ‘avatar' parametric modulation (1 for infectious trials and −1 for neutral trials). Next, we computed two DCM analyses (neural model options: bilinear neural model, one state per region, no stochastic effects, with input centering), one for each hemisphere (cortical regions selected from contrasts 5 and 7 in the right hemisphere + right hypothalamus and cortical regions selected from contrasts 5 and 7 in the left hemisphere + left hypothalamus). The ‘task' and ‘avatar' conditions for VTF were included in the two DCM analyses. The modulatory connectivity B was enabled for the ‘avatar' condition and was restricted to self-connections and connections between the hypothalamus and all other regions. The parametric empirical Bayes framework was used to identify (at the group level) the connections that were differently modulated across VTFinfectious and VTFneutral conditions. Connections were considered significantly modulated (VTFinfectious > VTFneutral or VTFinfectious < VTFneutral) when a posterior probability of 0.95 was reached. As control analyses, the same DCM analyses were computed with the corresponding conditions presented in the near space (VTNinfectious and VTNneutral), as well as in the fearful cohort. Serum samples were thawed and analyzed using different methods. For steroid quantification, serum samples (100 µl) were mixed with 550 µl of 5% H3PO4 and 75 µl of internal standard solution and extracted by solid-phase extraction on an OASIS MCX µElution 96-well plate (30 µm, 2 mg). Wells were washed and conditioned with 200 µl of methanol and 200 µl of water, respectively. Loaded samples were washed with 200 µl of 5% NH4OH and twice with 200 µl of water:methanol (4:1 (vol/vol)), and steroids were eluted with 2 × 100 µl of isopropanol. The solvent was then evaporated to dryness under N2 gas (TurboVap, Biotage), and final extracts were reconstituted with 100 µl of methanol. Extracted samples were analyzed by reversed-phase liquid chromatography (LC) coupled to tandem MS (MS/MS) in both positive and negative ionization mode using a TSQ Altis triple-quadrupole system interfaced with a Vanquish UHPLC system (Thermo Fisher Scientific). Chromatographic separation was performed in an Accucore aQ C18 column (2.6 μm, 100 mm × 2.1 mm inner diameter; Thermo Fisher Scientific). The mobile phase was composed of A (0.25 mM ammonium fluoride in water) and B (100% methanol) at a flow rate of 250 μl min−1, column temperature of 40 °C and sample injection volume of 2 µl. The column was then washed with solvent B for 1 min and equilibrated to initial conditions. The ion transfer tube temperature was set at 275 °C, the positive ion spray voltage was set at +3,500 V, and the negative ion spray voltage was set at −2,800 V. Scheduled multiple reaction monitoring with polarity switching was used as the acquisition mode with a minimum dwell time between 8 and 22 ms. Optimized collision energies for each metabolite were applied. Raw LC–MS/MS data were processed using TraceFinder 5.0 software (Thermo Fisher Scientific). For absolute quantification, calibration curves and stable isotope-labeled internal standards were used to determine the response factor. Linearity of the standard curves was evaluated for each metabolite using a 12-point range; in addition, peak area integration was manually curated and corrected when necessary. For eicosanoid quantification, serum samples (150 µl) were mixed with 150 µl of extraction buffer (citric acid/Na2HPO4, pH 5.6) and 10 µl of internal standard solution and extracted by solid-phase extraction using an OASIS HLB LP 96-well plate (60 µm, 60 mg). Wells were conditioned and equilibrated with 1 ml of methanol and 1 ml of water, respectively. Loaded samples were washed with water:methanol (90:10 (vol/vol)), and eicosanoids were eluted with 750 µl of methanol. The solvent was then evaporated to dryness under N2 gas (TurboVap, Biotage), and final extracts were reconstituted with 75 µl of methanol:water (6:1 (vol/vol)). Extracted samples were analyzed by reversed-phase LC–MS/MS40 in negative ionization mode using a 6495 triple-quadrupole system interfaced with a 1290 UHPLC system (Agilent Technologies). Chromatographic separation was performed in an Acquity BEH C18 column (1.7 μm, 150 mm × 2.1 mm inner diameter; Waters). The mobile phase was composed of A (water with 0.1% acetic acid) and B (acetonitrile:isopropanol 90:10 (vol/vol)) at a flow rate of 500 μl min−1, column temperature of 60 °C and sample injection volume of 2 µl. The column was then washed with solvent B for 0.9 min and equilibrated to initial conditions. ESI source conditions were set as follows: dry gas temperature of 290 °C, nebulizer at 25 psi and flow of 12 l min−1, sheath gas temperature of 400 °C and flow of 12 l min−1, nozzle voltage of 2,000 V and capillary voltage of 3,000 V. Dynamic multiple reaction monitoring was used as the acquisition mode with a total cycle time of 250 ms. Optimized collision energies for each metabolite were applied58. Raw LC–MS/MS data were processed using Agilent Quantitative analysis software (version B.07.00, MassHunter Agilent Technologies). For absolute quantification, calibration curves and stable isotope-labeled internal standards were used to determine the response factor. Linearity of the standard curves was evaluated for each metabolite using a 12-point range; in addition, peak area integration was manually curated and corrected when necessary. For other soluble mediators, the ‘human neuroinflammation panel 1 (740796)' LEGENDplex kit (Biolegend) was used, according to manufacturer's instructions. This platform allowed for the quantification of the concentration of 13 different soluble factors at the same time, namely VILIP-1, CCL2 (MCP-1), sTREM-2, BDNF, TGFβ1, VEGF, IL-6, sTREM-1, β-NGF, IL-18, TNF, sRAGE and CX3CL1 (Fractalkine). Steroids, eicosanoids and neuroinflammatory factors were analyzed and plotted using GraphPad Prism version 10.4.1 (Supplementary Figs. Univariate and multivariate linear regressions were first attempted to predict VR-induced changes in ILC activation from changes in eicosanoid, neuroinflammatory factor and hormone concentrations. These analyses resulted in nonsignificant effects (univariate: R2 = 0.001, 0.015 and 0.058 and P = 0.86, 0.53 and 0.20, for eicosanoids, neuroinflammatory factors and hormones, respectively; multivariate: R2 = 0.13 and P = 0.3), suggesting that the neural signals triggering an immune response are based on a more complex, nonlinear combination of inputs. For this reason, we developed a one-hidden-layer neural network with changes in eicosanoid, neuroinflammatory factor and hormones concentrations as input and changes in ILC activation as output (Supplementary Method 6). All data were checked for normality distribution (Shapiro–Wilk test). Because the majority of data were normally distributed, to ensure uniform statistical testing, we performed two-sided t-tests in all comparisons. Given the absence of previous studies assessing immunological responses to virtual threats, we first calculated the required sample size for the behavioral multisensory experiment. Based on previous experiments17,19, an averaged effect size f = 0.403 was calculated. Thus, we originally estimated, for immunomonitoring, a sample size of 15 participants per group, with a desired power of 0.95 (1 − β) on within-group comparisons via G*Power 3.1 software. For the replication experiment, based on the effect size of the data presented in the paper of the original cohorts (neural versus infection cohorts: Cohen's d = 1.114 for ILC frequency and d = 1.116 for ILC activation), the sample size with a P value of <0.05 and a power of 0.80 was determined to be 14 participants per cohort. To minimize the risk of dropouts and data loss, we enrolled and analyzed 16 participants per group. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data files can be found at https://osf.io/rg9fa/files/osfstorage. Custom code to replicate the results shown in Fig. 5 can be found at https://osf.io/rg9fa/files/osfstorage. Mobbs, D., Headley, D. B., Ding, W. & Dayan, P. Space, time, and fear: survival computations along defensive circuits. Fanselow, M. S. Negative valence systems: sustained threat and the predatory imminence continuum. Stroeymeyt, N. et al. Social network plasticity decreases disease transmission in a eusocial insect. Poirotte, C. et al. Mandrills use olfaction to socially avoid parasitized conspecifics. Serino, S. et al. Sharpening of peripersonal space during the COVID-19 pandemic. Graziano, M. S. & Cooke, D. F. Parieto–frontal interactions, personal space, and defensive behavior. Paul, W. E. Bridging innate and adaptive immunity. Akira, S., Uematsu, S. & Takeuchi, O. Pathogen recognition and innate immunity. Irwin, M. R. & Cole, S. W. Reciprocal regulation of the neural and innate immune systems. Koren, T. et al. Insular cortex neurons encode and retrieve specific immune responses. Brain motor and fear circuits regulate leukocytes during acute stress. Changing mindsets to enhance treatment effectiveness. & Ksionzky, S. Factors of interpersonal behavior and judgment in social groups. & Banaji, M. R. Understanding and using the implicit association test. Social perception of others shapes one's own multisensory peripersonal space. Dynamic sounds capture the boundaries of peripersonal space representation in humans. Body part-centered and full body-centered peripersonal space representations. Rapid recalibration of peri-personal space: psychophysical, electrophysiological, and neural network modeling evidence. Noel, J. P. et al. Peri-personal space encoding in patients with disorders of consciousness and cognitive-motor dissociation. Bernasconi, F. et al. Audio–tactile and peripersonal space processing around the trunk in human parietal and temporal cortex: an intracranial EEG study. Common and distinct brain regions processing multisensory bodily signals for peripersonal space and body ownership. Trabanelli, S. et al. Human innate lymphoid cells (ILCs): toward a uniform immune-phenotyping. Spits, H., Bernink, J. H. & Lanier, L. NK cells and type 1 innate lymphoid cells: partners in host defense. Vanoni, G. et al. Human primed ILCPs support endothelial activation through NF-κB signaling. Guia, S., Fenis, A., Vivier, E. & Narni-Mancinelli, E. Activating and inhibitory receptors expressed on innate lymphoid cells. Adams, N. M. & Sun, J. C. Spatial and temporal coordination of antiviral responses by group 1 ILCs. Harrison, N. A. et al. Neural origins of human sickness in interoceptive responses to inflammation. Roelofs, K. Freeze for action: neurobiological mechanisms in animal and human freezing. Quatrini, L., Vivier, E. & Ugolini, S. Neuroendocrine regulation of innate lymphoid cells. & Norris, P. C. Eicosanoid storm in infection and inflammation. Hirata, T. & Narumiya, S. Prostanoids as regulators of innate and adaptive immunity. Host resistance to endotoxic shock requires the neuroendocrine regulation of group 1 innate lymphoid cells. Spencer, R. L. & Deak, T. A users guide to HPA axis research. Korte, S. M. Corticosteroids in relation to fear, anxiety and psychopathology. Noda, Y. et al. Neurosteroids ameliorate conditioned fear stress: an association with sigma receptors. Quatrini, L. et al. Endogenous glucocorticoids control host resistance to viral infection through the tissue-specific regulation of PD-1 expression on NK cells. Gorica, E. & Calderone, V. Arachidonic acid derivatives and neuroinflammation. Tashjian, S. M., Zbozinek, T. D. & Mobbs, D. A decision architecture for safety computations. & Lo, N. C. Scientific and ethical basis for social-distancing interventions against COVID-19. Lisi, M. P., Scattolin, M., Fusaro, M. & Aglioti, S. M. A Bayesian approach to reveal the key role of mask wearing in modulating projected interpersonal distance during the first COVID-19 outbreak. Welsch, R., Wessels, M., Bernhard, C., Thones, S. & von Castell, C. Physical distancing and the perception of interpersonal distance in the COVID-19 crisis. Natural selection and the regulation of defensive responses. An evolutionary perspective on immunosenescence. Sorrentino, G. et al. How ageing shapes body and space representations: a comparison study between healthy young and older adults. Tybur, J. M., Cinar, C., Karinen, A. K. & Perone, P. Why do people vary in disgust? Ellena, G., Starita, F., Haggard, P. & Ladavas, E. The spatial logic of fear. Maris, E. & Oostenveld, R. Nonparametric statistical testing of EEG- and MEG-data. Oostenveld, R., Fries, P., Maris, E. & Schoffelen, J. M. FieldTrip: open source software for advanced analysis of MEG, EEG, and invasive electrophysiological data. & Wallace, M. T. Increased neural strength and reliability to audiovisual stimuli at the boundary of peripersonal space. Pascual-Marqui, R. D., Esslen, M., Kochi, K. & Lehmann, D. Functional imaging with low-resolution brain electromagnetic tomography (LORETA): a review. Z., Schlaggar, B. L. & Petersen, S. E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. The disgust scale: item analysis, factor structure, and suggestions for refinement. Kolmert, J. et al. Lipid mediator quantification in isolated human and guinea pig airways: an expanded approach for respiratory research. We acknowledge P. Romero and O. Blanke for their inspiring support to the project and their insightful comments on the paper. We also thank the flow cytometry facilities of the University of Lausanne and University of Geneva for excellent technical assistance. We are grateful for an interdisciplinary grant from the Faculty of Biology and Medicine University of Lausanne (C.J. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the paper. Open access funding provided by University of Lausanne. Present address: School of Health Sciences, HES-SO Valais-Wallis, Sion, Switzerland Present address: Translational Neural Engineering Lab, Neuro-X Institute, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland Present address: Department of Pharmacy, University of Naples Federico II, Naples, Italy Present address: Center for Mind/Brain Sciences–CIMeC, University of Trento, Rovereto, Italy Present address: Human Islet Isolation Facility, Nuffield Department of Surgical Sciences, University of Oxford, Oxford, UK Present address: Nuffield Department of Clinical Neurosciences, John Radcliffe Hospital, Oxford, UK These authors contributed equally: Sara Trabanelli, Michel Akselrod. These authors jointly supervised this work: Camilla Jandus, Andrea Serino Sara Trabanelli, Giulia Vanoni, Giuseppe Ercolano, Anthony Cornu & Camilla Jandus Sara Trabanelli, Georgia Papadopoulou, Giuseppe Ercolano, Anthony Cornu & Camilla Jandus Sara Trabanelli, Giulia Vanoni, Georgia Papadopoulou, Giuseppe Ercolano, Anthony Cornu & Camilla Jandus Sara Trabanelli, Georgia Papadopoulou, Anthony Cornu & Camilla Jandus Translational Research Centre in Onco-Hematology (CRTOH), Geneva, Switzerland Sara Trabanelli, Georgia Papadopoulou, Anthony Cornu & Camilla Jandus MySpace Lab and NeuroRehab Research Center, Service of University Neurorehabilitation (SUN), Lausanne University Hospital, Institution of Lavigny and University of Lausanne, Switzerland, Switzerland Michel Akselrod, Julia Fellrath, Tommaso Bertoni, Silvia Serino, Maren Born, Matteo Girondini, Giulia Ellena, Giulio Mastria, Petr Grivaz & Andrea Serino Metabolomics and Lipidomics Platform, Faculty of Biology and Medicine, University of Lausanne, Lausanne, Switzerland Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and G. Ercolano contributed to the funding acquisition. created new software used in the work and together with A.C. led the project administration. contributed equally as second authors. Correspondence to Camilla Jandus or Andrea Serino. The authors declare no competing interests. Nature Neuroscience thanks Michael Irwin, Asya Rolls and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Trabanelli, S., Akselrod, M., Fellrath, J. et al. Neural anticipation of virtual infection triggers an immune response. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250727235831.htm'>Scientists just cracked the cryptographic code behind quantum supremacy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 15:50:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Experts say quantum computing is the future of computers. Unlike conventional computers, quantum computers leverage the properties of quantum physics such as superposition and interference, theoretically outperforming current equipment to an exponential degree. However, this edge is not guaranteed for all calculations, raising fundamental questions regarding the conditions under which such an advantage exists. Motivated by this uncertainty, a team of researchers at Kyoto University has endeavored to understand the necessary and sufficient conditions for quantum advantage, using an approach combining techniques from quantum computing and cryptography, the science of coding information securely. "We were able to identify the necessary and sufficient conditions for quantum advantage by proving an equivalence between the existence of quantum advantage and the security of certain quantum cryptographic primitives," says corresponding author Yuki Shirakawa. The results imply that when quantum advantage does not exist, then the security of almost all cryptographic primitives -- previously believed to be secure -- is broken. Importantly, these primitives are not limited to quantum cryptography but also include widely-used conventional cryptographic primitives as well as post-quantum ones that are rapidly evolving. The established equivalence between quantum computing and cryptography also provides a stronger cryptographic foundation for future experimental demonstrations of quantum advantage, as well as for ongoing theoretical investigations in the field. The team expects that future research will extend characterization to other types of quantum advantage and lead to a more general theoretical framework. Defying Geology: Scientists Discover Giant Formations Beneath the North Sea Caught on Camera: Rare Footage Reveals How a Giant Earthquake Tore the Earth in Seconds Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02341-z'>China reins in the spiralling construction costs of nuclear power — what can other countries learn?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 15:19:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Shangwei Liu is a research fellow at the Belfer Center for Science and International Affairs, Harvard Kennedy School, Harvard University, Cambridge, Massachusetts, USA. Gang He is an assistant professor at the Marxe School of Public and International Affairs, Baruch College, CUNY Institute for Demographic Research and the Graduate Center, City University of New York, New York, USA. Minghao Qiu is an assistant professor at the School of Marine and Atmospheric Sciences and the Program in Public Health at Stony Brook University, Stony Brook, New York, USA. Daniel M. Kammen is a professor of energy in the Department of Civil and Systems Engineering, School of Advanced International Studies and Ralph O'Connor Sustainable Energy Institute, Johns Hopkins University, Baltimore, Maryland, USA. The Taipingling nuclear power plant in Huizhou, China, has been under construction since 2019. Credit: VCG via Getty Once again, the world is betting on nuclear power. The United States aims to quadruple its nuclear capacity by 2050, and more than 30 countries have pledged to triple global capacity by mid-century. China has more than 30 reactors under construction and in planning, and France has announced plans to build 14 reactors. Technology giants, including Amazon, Google and Microsoft, are also investing in nuclear to power their energy-hungry data centres and lower their carbon emissions. Historically, the industry has faced a ‘cost escalation curse'1–4. Building more nuclear reactors has led to higher, not lower, costs per watt (see ‘Costly construction'), hampering their economic viability. Here, we show that tailored regulations and policies can reverse the trend. Decades of nuclear-energy development in China demonstrate that construction costs can be brought down through a combination of stable regulations and efforts to strengthen domestic supply chains. Over the past two decades, China has been the main country to substantially and consistently expand its nuclear fleet, to 58 operating reactors in 2024. Since 2022, the government has been approving around ten new reactors each year, putting China on track to surpass the United States and become the world's largest holder of nuclear power capacity by 2030. State-affiliated research centres have outlined a goal of quintupling China's current nuclear capacity by 2050. We have compiled and analysed a data set on the construction costs of nuclear power plants in China, drawn from a wide range of publicly available sources (see Supplementary information for raw data and further analyses). Whereas construction costs increased substantially between the 1960s and 2000s, by around tenfold in the United States and by nearly twofold in France, they had halved in China by the early 2000s and have remained largely stable since. Various strategies were used to keep technological costs down, such as building larger plants for scale efficiency and leveraging accumulated experience. But these alone fail to fully explain the cost-reduction trend observed in China. The country has also managed to contain costs through strategic development of domestic supply chains, stable regulatory frameworks, state-backed incentive policies and effective construction management. The key driver of cost reduction has been China's deliberate effort to build domestic capacity in civil nuclear power, which unfolded in three stages. At the same time, it began researching and developing its own reactor design based on French technology. China ramped up capacity, building about 30 reactors using two domestically developed model types. After the Fukushima nuclear disaster in Japan in 2011, China accelerated the adoption of advanced safety features by collaborating with partners from the United States and France, while developing its own advanced reactor models. Substituting expensive imports with domestically produced components substantially reduces cost. However, for countries that import their nuclear technologies, domestic supply-chain building must be strategic. By contrast, around the 1980s, France deviated from the tried-and-tested US reactor design in favour of a national version, which hindered standardization and caused construction costs to soar3. Will AI's huge energy demands spur a nuclear renaissance? Will AI's huge energy demands spur a nuclear renaissance? China's regulatory structure and policy environment also play an important part. A consistent, state-backed industrial policy has provided stable electricity tariffs and low-interest financing for state-owned nuclear power companies (see go.nature.com/3uhsdfd). Its centralized nuclear governance structure helps to ensure regulatory stability and coordinates reactor standardization across the country. The Future of Nuclear Energy in a Carbon-Constrained World (MIT, 2018). Organisation for Economic Co-operation and Development & Nuclear Energy Agency. Unlocking Reductions in the Construction Costs of Nuclear: A Practical Guide for Stakeholders (OECD & NEA, 2020). Will AI's huge energy demands spur a nuclear renaissance? Nuclear power for AI: what it will take to reopen Three Mile Island safely I won three competitive grants in a row. How I'm advocating in Congress for the next generation of US scientists Why evaluating the impact of AI needs to start now Hard paths: how courage differs from bravery in science Meet the biotech-company founders driven by their child's rare disease IOP is China's premier research institution in condensed matter physics and related fields. Title: Associate or Senior Editor (Optoelectronic devices and engineering), Nature Location: Shanghai or Beijing - hybrid working model Closing dat... Title: Associate or Senior Editor, Nature Computational Science  Organization: Nature Portfolio  Location: New York, New Jersey, Shanghai or Beijin... Will AI's huge energy demands spur a nuclear renaissance? Nuclear power for AI: what it will take to reopen Three Mile Island safely An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/tetris-presents-math-problems-even-computers-cant-solve/'>Mathematicians and Computer Scientists Can't Stop Playing Tetris—But Not for the Reasons You Think</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The Impossible Problems Hidden in a Simple Game of Tetris Tetris pushes even supercomputers to their limits and amazes mathematicians Launched in 1984 by Russian programmer Alexey Pajitnov, Tetris quickly became a blockbuster and has had hundreds of millions of players over the years. I myself spent hours on my Game Boy trying to position falling bricks so that they would fill the playing field as completely as possible. Over the course of a game, those blocks fell faster and faster, and my thumbs could barely keep up with the controls. In principle, all games—even those as varied as Candy Crush Saga, Magic: The Gathering and Wordle—can be examined from a mathematical perspective. But Tetris has many special connections to mathematics. For instance, the game's goal strongly resembles geometry's parquet problems, in which you determine whether you can cover an area with an infinitely large set of tiles without any gaps. But Tetris is especially intriguing to mathematicians in terms of its complexity. More specifically, researchers have wondered about the computing power that it takes to determine how or if someone can truly “solve” Tetris, assuming conditions such as a finite number of bricks and the ability to know the order in which various shapes will appear. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. They have defined multiple complexity classes, or categories, including P and NP problems. Simply put, P problems are easy for conventional computers to solve, whereas NP problems are more difficult but, in the event you have a possible solution, easy to check. (NP problems can be thought of like a Sudoku puzzle: it may take hours to fill in the fields, but it only takes a few minutes to see whether the solution is correct.) To determine the complexity of a task, one must compare different problems with each other. If every algorithm that solves task A can also solve task B, for example, then A is more complex than B. Or as mathematicians put it: B is “reducible” to A. So how do we pick a good point of comparison? Tetris on a Nintendo Gameboy at the Computer Games Museum Berlin. Such a division is not possible for every given set. Finding out whether this exists or not proves to be extremely difficult: the three-partition problem is NP-complete. In 2003 computer scientists at the Massachusetts Institute of Technology demonstrated that the question of whether a Tetris board can be cleared in a given game situation can itself be mapped to the three-partition problem. In doing so, they proved that the questions “Can a set be divided into a three-partition?” and “Can the Tetris playing field be emptied?” are identical from a mathematical point of view. This insight means the puzzle of whether given bricks can be arranged appropriately falls into the category of NP-complete problems, making Tetris a highly complex game. And indeed, conventional computers will be overwhelmed very quickly: there is no algorithm that can solve the problem efficiently. Tetris has even more complex properties, as computer scientists Hendrik Jan Hoogeboom and Walter Kosters, both at Leiden University in the Netherlands, showed in a 2004 paper. If I gave you a predetermined number of ways for, say, 40 I-shaped tiles to fall onto an empty Tetris board, could you decide whether, among those eight ways, there is one for which the board ends up empty? Hoogeboom and Kosters proved that this question is, in fact, undecidable, even with an infinite amount of computing power. Of course, these questions likely have no effect on your success at Tetris. With the speed at which pieces fall, there is hardly any time to think about mathematical problems. Still, it's remarkable that after more than 40 years, Tetris appreciation continues to grow and evolve, even as the game remains essentially unchanged. For instance, a playing technique known as “rolling” (which allows very fast inputs to be made) has made it possible to advance further than ever before. This article originally appeared in Spektrum der Wissenschaft and was reproduced with permission. Manon Bischoff is a theoretical physicist and an editor at Spektrum der Wissenschaft, the German-language sister publication of Scientific American.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65500416/tinshemet-cave-burial-ground/'>Archaeologists May Have Discovered the World's Oldest Burial Ground</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. What could be the world's oldest human burial site—discovered in a cave in central Israel—shows that funerary ritual has been alive and well for at least 100,000 years. The Tinshemet Cave, which has been an active archaeological site for nearly a decade, has revealed at least five sets of Homo sapiens and Neanderthal-like hominin remains—all buried in the same ritualistic fashion and surrounded by more than 500 grave goods. In a study published this spring in Nature Human Behavior, the authors called the discovery “remarkable for its exceptional evidence of human morphological variability, with contemporaneous fossils of Homo sapiens and Neanderthal-like hominis.” The most fascinating thing about these burials is their age. The authors believe that this behavioral uniformity is due to intensified inter-population interactions and the mixing of different Homo groups. “This is an amazing revolutionary innovation for our species,” Yossi Zaidner, a director of the excavation and professor of archaeology at Hebrew University in Jerusalem, told the Associated Press. “It's actually the first time we are starting to use this behavior.” The researchers believe that, since some of the objects had to be brought in from hundreds of miles away and had no practical use, these objects are surefire indicators of ritual design. “Here we see a really complex set of behaviors,” Zaidner said, “not related to just food and surviving.” Even marking out burial sites is believed to have been a new concept 100,000 years ago. “It's a kind of claim you make to the neighbors, saying ‘this is my territory, this part of the land belongs to my father and my forefather' and so on and so on,” Israel Hershkovitz, a physical anthropologist at Tel Aviv University and co-director of the Tinshemet site, told AP. The region's mix of Neanderthals and Homo sapiens could have created a hybrid population, but research into the cave's skeletal remains have so far proven inconclusive on that front. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Rare ‘Book of the Dead' Found Buried in Egypt</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/you-dont-remember-being-a-baby-but-your-brain-was-making-memories/'>You Don't Remember Being a Baby, but Your Brain Was Making Memories</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You Don't Remember Being a Baby, but Your Brain Was Making Memories Brain scans capture memory formation in babies, raising new questions about why people forget their earliest years I don't remember much from that day, but a scar on my foot bears witness to what happened. This is true whether you ask a college student or a grandparent, suggesting that adults' lack of infant or toddler memories is not just the result of normal forgetting that occurs with the passage of time. The gap in our autobiographical memory from when we were a baby is known as “infantile amnesia.” This region, which is critical for memory, grows and changes throughout childhood, so it might not yet be available to infants. In this telling, babies are not so different from famous cases of amnesia such as those of Henry Molaison and Lonni Sue Johnson, both of whom suffered hippocampal damage in adulthood that made them unable to store memories. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Another possibility is that the infant brain can store memories, but we eventually lose access to them. Recent studies in mice show that not only is the hippocampus able to store memories early in life but that it may retain these memories into adulthood. For example, scientists were able to retrieve an otherwise forgotten memory by stimulating neurons in the hippocampus that had been active during an early experience. My lab has been on a decade-long quixotic adventure to study awake infants with functional magnetic resonance imaging (fMRI), a form of brain imaging that can measure activity from regions deep in the brain such as the hippocampus. Although this technology is used to study memory formation in adults and is safe for infants, it had not previously been used to study babies' memory. Infants move a lot, don't follow instructions and have a short attention span—all of which make it difficult to collect good data from them. As a result, most fMRI research in infants has been conducted while they sleep. But this wasn't an option for our investigations because memories are based on experiences in waking life. In a recent study, a team at my lab led by Tristan Yates, now a postdoctoral researcher at Columbia University, used this method to discover that the infant hippocampus can store memories beginning around one year of age. We showed infants photographs of faces, objects and scenes one at a time during fMRI. This result suggests that the infant hippocampus can create memories after only a brief experience. The effects were clearest after 12 months of age, in infants who had stronger overall memory and in the subregion of the hippocampus that is most important for remembering specific events (called episodic memory) in adults. But the work also raises more questions: How long do these hippocampal memories last? We tested for a few minutes, but infantile amnesia plays out over years. We tested individual photographs, but episodic memories involve complex events with multiple people, places and things interacting over space and time (for example, remember your last vacation). The deepest and most provocative questions relate to why most people's earliest memories are from age four to five (or later) if memories are being stored in their brain by age one. Are there any tricks or practices for accessing them? Would we even be able to make sense of them if so? Answering these questions will help resolve more than a century of scientific curiosity. Take a moment to reflect: What is your earliest memory? There might even be earlier memories locked away in your brain. Are you a scientist who specializes in neuroscience, cognitive science or psychology? Please send suggestions to Scientific American's Mind Matters editor Daisy Yuhas at dyuhas@sciam.com.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41566-025-01718-w'>Synthetic aperture waveguide holography for compact mixed-reality displays with large étendue</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 10:25:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mixed-reality (MR) display systems enable transformative user experiences across various domains, including communication, education, training and entertainment. To create an immersive and accessible experience, the display engine of the MR display must project perceptually realistic 3D images over a wide field of view observable from a large range of possible pupil positions, that is, it must support a large étendue. Current MR displays, however, fall short in delivering these capabilities in a compact device form factor. Here we present an ultra-thin MR display design that overcomes these challenges using a unique combination of waveguide holography and artificial intelligence (AI)-driven holography algorithms. One of the key innovations of our display system is a compact, custom-designed waveguide for holographic near-eye displays that supports a large effective étendue. This is co-designed with an AI-based algorithmic framework combining an implicit large-étendue waveguide model, an efficient wave propagation model for partially coherent mutual intensity and a computer-generated holography framework. Together, our unique co-design of a waveguide holography system and AI-driven holographic algorithms represents an important advancement in creating visually comfortable and perceptually realistic 3D MR experiences in a compact wearable device. Mixed reality (MR) aims to seamlessly connect people in hybrid physical–digital spaces, offering experiences beyond the limits of our physical world. These immersive platforms provide transformative capabilities to applications including training, communication, entertainment and education1,2, among others. To achieve a seamless and comfortable interface between a user and a virtual environment, the near-eye display must fit into a wearable form factor that ensures style and all-day usage while delivering a perceptually realistic and accessible experience comparable to the real world. Current near-eye displays, however, fail to meet these requirements3. To project the image produced by a microdisplay onto a user's retina, existing designs require optical bulk that is noticeably heavier and larger than conventional eyeglasses. Moreover, existing displays support only two-dimensional images, with limited capability to accurately reproducing the full light field of the real world, resulting in visual discomfort caused by the vergence–accommodation conflict4,5. Emerging waveguide-based holographic displays are among the most promising technologies to address the challenge of designing compact near-eye displays that produce perceptually realistic imagery. These displays are based on holographic principles6,7,8,9,10, which have been demonstrated to encode a static three-dimensional (3D) scene with a quality indistinguishable from reality in a thin film11 or to compress the functionality of an optical stack into a thin, lightweight holographic optical design12,13. Holographic displays also promise unique capabilities for near-eye displays, including per-pixel depth control, high brightness, low power and optical aberration correction capabilities, which have been explored using benchtop prototypes providing limited visual experiences14,15,16,17,18. Most recently, holographic near-eye displays based on thin optical waveguides have shown promise in enabling very compact form factors for near-eye displays19,20,21, although the image quality, the ability to produce 3D colour images or the étendue achieved by these proposals have been severely limited. A fundamental problem of all digital holographic displays is the limited space–bandwidth product, or étendue, offered by current spatial light modulators (SLMs)22,23. In practice, a small étendue fundamentally limits how large of a field of view and range of possible pupil positions, that is, eyebox, can be achieved simultaneously. While the field of view is crucial for providing a visually effective and immersive experience, the eyebox size is important to make this technology accessible to a diversity of users, covering a wide range of facial anatomies as well as making the visual experience robust to eye movement and device slippage on the user's head. A plethora of approaches for étendue expansion of holographic displays has been explored, including pupil replication as well as the use of static phase or amplitude masks20,24,25,26,27,28. By duplicating or randomly mixing the optical signal, however, these approaches do not increase the effective degrees of freedom of the (correlated) optical signals, which is represented by the rank of the mutual intensity (MI)29,30,31 (Supplementary Note 2). Hence, the image quality achieved by these approaches is typically poor, and perceptually important ocular parallax cues are not provided to a user32. One of the key challenges for achieving high image quality with holographic waveguide displays is to model the propagation of light through the system with high accuracy20. Non-idealities of the SLM, optical aberrations, coherence properties of the source, and many other aspects of a specific holographic display are difficult to model precisely, and minor deviations between simulated model and physical optical system severely degrade the achieved image quality. This challenge is drastically exacerbated for holographic displays using compact waveguides in large-étendue settings. First, the propagation of light has to be modelled with very high accuracy. Second, such a model needs to be efficient and scalable to our large-étendue settings. Recent advances in computational optics have demonstrated that artificial intelligence (AI) methods can be used to learn accurate propagation models of coherent waves through a holographic display, substantially improving the achieved image quality33,34. These learned wave propagation models typically use convolutional neural networks (CNNs), trained from experimentally captured phase–intensity image pairs, to model the opto-electronic characteristics of a specific display more accurately than purely simulated models35. However, as we demonstrate in this Article, conventional CNN-based AI models fail to accurately predict complex light propagation in large-étendue waveguides, partly because of the incorrect assumption of the light source being fully coherent. Other important problems include the efficiency of a model, such that it can be trained within a reasonable time from a limited set of captured phase–intensity pairs and run quickly at inference time, and scalability to large-étendue settings while ensuring accuracy and efficiency. Here, we reformulate the wave propagation learning problem as coherence retrieval based on the theory of partial coherence31,36. For this purpose, we derive a physics-based wave propagation model that parameterizes a low-rank approximation of the MI of the wave propagation operator inside a waveguide accounting for partial coherence, which models holographic displays more accurately than existing coherent models. Moreover, our approach parameterizes the wave propagation through waveguides with emerging continuous implicit neural representations37, enabling us to efficiently learn a model for partially coherent wavefront propagation at arbitrary spatial and frequency coordinates over a large étendue. Our implicit model achieves superior quality compared with existing methods; it requires an order of magnitude less training data and time than existing CNN model architectures, and its continuous nature generalizes better to unseen spatial frequencies, improving accuracy for unobserved parts of a wavefront. Along with our unique model, we design and implement a holographic display system, incorporating a holographic waveguide, holographic lens and micro-electromechanical system (MEMS) mirror. Our optical architecture provides a large effective étendue via steered illumination with an ultra-compact form factor and solves limitations of similar designs by removing unwanted diffraction noise and chromatic dispersion using a volume-holographic waveguide and holographic lens, respectively. Our architecture is inspired by synthetic aperture imaging38, where a large synthetic aperture is formed by digitally interfering multiple smaller, mutually coherent apertures. Our goal is to form a large display eyebox—the synthetic aperture built up from multiple scanned and mutually incoherent apertures, each limited in size by the instantaneous étendue of the system. This idea is inspired by classic synthetic aperture holography39, but we adapt this idea to modern waveguide holography systems driven by AI algorithms. After a wave propagation model is trained in a one-time preprocessing stage, a CGH algorithm converts the target content into one or multiple phase patterns that are displayed on the SLM. Large-étendue settings require the target content to contain perceptually important visual cues that change with the pupil position, including parallax and occlusion. Traditional 3D content representations used for CGH algorithms, such as point clouds, multilayer images, polygons or Gaussians40,41,42, however, are inadequate for this purpose. Light field representations, or holographic stereograms43,44, meanwhile, contain the desired characteristics. Motivated by this insight, we develop a light-field-based CGH framework that uniquely models our setting where a large synthetic aperture is composed of a set of smaller, mutually incoherent apertures that are, however, partially coherent within themselves. Our approach uniquely enables seamless, full-resolution holographic light field rendering for steered-illumination-type holographic displays. In summary, we present synthetic aperture waveguide holography as a system that combines a new and compact large-étendue waveguide architecture with AI-driven holographic algorithms. Through our prototypes, we demonstrate high 3D image quality within a 3D eyebox that is two orders of magnitude larger, marking a pivotal milestone towards practical holographic near-eye display systems. Our architecture is designed to produce high-quality full-colour 3D images with large étendue in a compact device form factor to support synthetic aperture holography based on steered waveguide illumination, as shown in Fig. Moreover, it allows a minimum footprint for beam steering using a MEMS mirror at the input side. Existing architectures, however, suffer from two problems: world-side light leakage from the waveguide and chromatic dispersion of the eyepiece lens. Here, we overcome conventional limitations with two state-of-the-art optical components to overcome conventional limitations: the angle-encoded holographic waveguide and the apochromatic holographic eyepiece lens. a, An illustration of the synthetic aperture waveguide holography principle. The illumination module consists of a collimated fibre-coupled laser, a MEMS mirror that steers the input light angle, and a holographic waveguide. Together, these components serve as a partially coherent backlight of the SLM. The SLM is synchronized with the MEMS mirror and creates a holographic light field, which is focused towards the user's eye using an eyepiece lens. Our design achieves an ultra-thin form factor as it consists only of flat optical elements and it does not require optical path length to form an image. The steered illumination mechanism produces a synthetic aperture, which supports a two-orders-of-magnitude larger étendue than that intrinsic to the SLM. Our angle-encoded holographic waveguide and apochromatic holographic lens design solve bidirectional diffraction noise and chromatic dispersion issues, respectively. b, An exploded view of the schematic image and captured photos of the holographic MR display prototype. The use of thin holographic optics achieves a total optical stack thickness of less than 3 mm (panel to lens). Top- and side-view photographs of the prototype are shown. Our holographic waveguide shares many characteristics with conventional surface relief grating (SRG)-type waveguides. Specifically, our design uses a similar pupil replication principle and layout, which consists of an in-coupler, an exit-pupil expanding grating and an out-coupler grating. By contrast, our couplers are constructed of uniquely designed volume Bragg gratings (VBGs) instead of SRGs. SRGs used in conventional waveguides47 have a degeneracy that supports multiple modes of diffraction. This causes a light leakage issue as the light is out-coupled bidirectionally, to both the world side and viewer's eye side. This d.c. noise significantly degrades the contrast of the displayed image19, and it is challenging to filter out this noise as it shares the optical path with the signal. On the contrary, VBGs exhibit a type of diffraction known as Bragg diffraction48,49, where only light that satisfies a specific incident angle and a narrow spectral bandwidth is diffracted to a single diffraction order with high efficiency. This single-direction diffraction greatly suppresses stray light and ghost images compared with conventional waveguides. In addition, we use the angle-encoded multiplexing method to cover the larger steering angle, where each grating supports only a portion of the target angular bandwidth with a specific narrow band wavelength with high efficiency. The multiplexing is performed by overlapping volume gratings with the same surface pitch but with different slant angles (see Supplementary Note 1 for additional details). This allows us to collectively cover the target angular bandwidth with high efficiency, while supporting three wavelengths of red (638 nm), green (520 nm) and blue (460 nm) without crosstalk. Our display architecture achieves an ultra-thin form factor of only 3 mm thickness from the SLM to the eyepiece lens, including a 0.6-mm waveguide and 2-mm holographic lens. The MEMS mirror in our display steers the illumination angles incident on our SLM, which creates a synthetic aperture (that is, eyebox) of size 9 × 8 mm2, supporting an eyebox volume that is two orders of magnitude larger than that intrinsic to the SLM. An in-depth discussion of the full system and additional components is found in the Methods and Supplementary Note 1. Our synthetic aperture waveguide holographic display is partially coherent because its synthetic aperture consists of a scanned set of mutually incoherent apertures. In addition, the instantaneous eyebox exhibits partial coherence due to factors such as millimetre-scale optical path length differences generated by the pupil replication process in the waveguide, mode instability of diode lasers and imperfect polarization management. Neither existing AI-based wave propagation models for coherent wavefronts21,33,34 nor recent waveguide propagation models20 are sufficient in adequately describing the complex behaviour of partially coherent light50,51. In this section, we first develop a partially coherent waveguide model to accurately characterize physical optical systems based on implicit neural representations52. Then, we adapt this model to our synthetic aperture waveguide holography setting. Our model can be trained automatically using camera feedback, it generalizes well to unseen spatial frequencies and it is used to produce the high-quality experimental results demonstrated in later sections. A partially coherent wavefront can be represented by its MI or its Fourier transform—the Wigner distribution function53,54. With our waveguide model, we aim to model the MI of the waveguide, \(J\left({{\bf{r}}}_{1},{{\bf{r}}}_{2}\right)\), where r1 and r2 are spatial coordinates on the SLM plane. This MI depends on the steered illumination angle, so it needs to be characterized for each of them separately. Representing such a high-resolution, four-dimensional function for many apertures within the synthetic aperture, however, is computationally intractable. Consider a discretized MI for 1,920 × 1,080 spatial SLM coordinates for each of 10 × 10 steering angles—the corresponding ensemble of MIs would require more than 100 terabytes of memory to be stored. Moreover, it is unclear how to smoothly interpolate these MIs between aperture positions as they are characterized at discrete positions. To address these challenges, we introduce a low-rank implicit neural representation for the ensemble of steering-angle-dependent MIs of the waveguide. Specifically, for each steering angle um, m = 1, …, M, the MI is approximated by K coherent spatial modes that are incoherently summed55 as \(J({{\bf{r}}}_{1},{{\bf{r}}}_{2},{\bf{u}})\approx \mathop{\sum }\nolimits_{k = 1}^{K}{f}_{{\rm{WG}}}^{\;k}({{\bf{r}}}_{1},{\bf{u}}){f}_{{\rm{WG}}}^{\;k}{({{\bf{r}}}_{2},{\bf{u}})}^{{\rm{H}}}\), where \({f}_{{\rm{WG}}}^{\;k}({{\bf{r}}}_{1},{\bf{u}})\) is a single coherent mode at spatial frequency u, k = 1, …, K indicates the mode index and H denotes the Hermitian transpose. Note that coherent waveguide models are a special case of this partially coherent model, namely, those of rank 1. This low-rank MI representation is inspired by coherence retrieval methods31. Rather than representing the ensemble of low-rank MIs explicitly, however, we introduce a novel waveguide representation based on emerging neural implicit representations37. This neural-network-parameterized representation is more memory efficient than a discretized representation and it is continuous, so it can be queried at any spatial or spatial frequency (that is, aperture) coordinate. Specifically, our implicit neural representation is a multilayer perceptron (MLP) network architecture, \({f}_{{\rm{WG}}}\left({\bf{r}},{\bf{u}};{\varPsi} \right):{{\mathbb{R}}}^{4}\to {{\mathbb{C}}}^{K}\), that represents the MI ensemble of the waveguide using network parameters Ψ. Once trained, the MLP can be queried with the input of spatial (r) and frequency (u) coordinates and it outputs the K modes of the corresponding MI. The primary benefits of this implicit neural representation over a conventional discrete representation are its memory efficiency (a few tens of megabytes versus terabytes) and, as we demonstrate in the following experiments, the fact that our representation generalizes better to aperture positions that were not part of the training data. Remarkably, even in a low-étendue setting, our implicit model achieves faster convergence and better accuracy than state-of-the-art CNN-based models based on explicit representations (Fig. 2d), so the implicit model can serve as a drop-in replacement for existing holographic displays. In a large-étendue setting, our implicit neural model is the only one achieving high-quality results. a, Using our prototype, we capture training and validation datasets consisting of sets of an SLM phase pattern as well as the corresponding aperture position and intensity image. The aperture positions are uniformly distributed across the synthetic aperture, enabling model training with large étendue. b, The captured dataset is used to train our implicit neural waveguide model. The parameters of our model are learned using backpropagation (dashed grey line) to predict the experimentally captured intensity images. c, A visualization of two waveguide modes of the trained model, including amplitude and phase, at two different aperture positions. Our model faithfully reconstructs the wavefront emerging out of the waveguide, exhibiting the patch-wise wavefront shapes expected from its pupil-replicating nature. d, Evaluation of wave propagation models with varying training dataset sizes for a single aperture (that is, low-étendue setting). Our model achieves a better quality using a dataset size that is one magnitude lower than state-of-the-art models34. e, Experimentally captured image quality for different wave propagation models in the low-étendue setting. Our implicit waveguide models the MI of input light arriving at the SLM plane. Each of the modes continues to propagate in free space, by distance z, to the image plane in the scene. We use an off-axis angular spectrum method (ASM)35,56 to implement this free-space wave propagation operator for each coherent mode, migrating the bandwidth over a large étendue. Finally, all propagated modes are incoherently summed over the full synthetic aperture to form the desired intensity, Iz, at distance z. where \({{\mathcal{P}}}_{z}(\cdot ;{\bf{u}})\) is the coherent wave propagation operator with propagation distance z through the aperture centred at u, and \(\left\langle \cdot \right\rangle\) is the mean operator denoting the incoherent sum of modes. As detailed in the Methods, to optimize experimental image quality, the wave propagation operator \({{\mathcal{P}}}_{z}\) also includes several learned components, including pupil aberrations and diffraction efficiency of the SLM. Using these equations, we can map phase patterns shown on the SLM ϕm with steering angle um to the image that a user would observe. To assess our partially coherent waveguide model with respect to state-of-the-art holographic wave propagation models, we capture a dataset from our display prototype consisting of triplets, each containing an SLM phase pattern, an aperture position and the corresponding intensity at the image plane (Fig. We then train models to predict the experimentally captured intensity images, given the known input phase patterns and aperture positions. To evaluate and compare waveguide models in a large-étendue setting, we train ours along with baseline models on a dataset captured at 72 aperture positions with our prototype. We evaluate the generalization capabilities of these models on a test set containing nine aperture positions that were not part of the training set. Quantitative results are presented in Table 1. Previously proposed explicit CNN models generalize poorly to the unseen aperture positions. Importantly, it takes more than 2 days of training for them to converge, which is somewhat impractical. Our implicit waveguide model is smaller in size and shows better generalization capabilities than explicit models, while also converging much faster. Moreover, when used in a partially coherent configuration, the accuracy of our model is drastically improved over all coherent models, as seen by the high quality achieved on both training and test sets when used with three or six modes. Figure 2a illustrates the data collection aspect of our approach: we show a sequence of phase patterns, ϕm, on the SLM and capture corresponding intensity images, Im, with a camera focused on the image plane. The implicit neural model is trained on these data, as shown in Fig. For this purpose, we simulate the forward image formation given the phase patterns ϕm and a random set of initialized model parameters. A loss function \({\mathcal{L}}\) measures the mean-squared error between simulated and captured intensity images. The backpropagation algorithm is applied to learn all model parameters, including the partially coherent waveguide model. This model comprises a set of coherent modes \({f}_{{\rm{WG}}}^{\;k}\), each being a complex-valued image on the SLM plane with amplitude and phase, that can be queried at the spatial frequencies corresponding to the MEMS steering angles um (Fig. In a low-étendue setting, that is, when operating with a fixed steering angle um, this implicit neural model achieves a higher quality with significantly less training data compared with state-of-the-art explicit CNN models34 (Fig. Therefore, our model serves as a drop-in replacement for existing wave propagation models with strictly better performance. Even though the gain in peak signal-to-noise ratio for explicit and implicit models on the validation set at convergence is only a few decibel (Fig. 2d), the generalization capabilities of our implicit model are far superior to those of the explicit model. 2e, where we show experimental results comparing a model-free free-space wave propagation operator35, the results achieved by the coherent explicit model34 and our partially coherent implicit model. The images used here are representative of a test set of images unseen during training and show that our model outperforms both baselines by a large margin. A large eyebox is crucial for guaranteeing high image quality when the eye moves and for making any display accessible to a diverse set of users with different binocular characteristics. Figure 3a illustrates the supported eyebox volume of our system (green volume), which is two orders of magnitude larger than that intrinsic to the SLM without steered illumination (orange volume). Moreover, our CGH framework fully utilizes the bandwidth corresponding to an eyebox size of 9 × 8 mm, unlike conventional holographic displays16,44,57, which use only a small fraction of the available bandwidth supported by the SLM (red volume). Figure 3b visualizes images of the phase aberrations of the waveguide system learned by our model at various positions within the supported eyebox. These aberrations vary drastically over the eyebox, making it challenging to be learned using approximately shift-invariant CNN-based models. We further validate our model's ability to produce high-quality imagery across an extended étendue and compare its performance with that of conventional holography in Fig. Figure 3c shows experimentally captured light fields of a two-dimensional resolution chart image located at optical infinity over the extended eyebox in the top row, while the insets in the bottom present the results at specific positions, indicated by corresponding colours. As expected, conventional holography provides a very small eyebox, which restricts the image to be seen only from the centre position. Our approach supports a significantly larger eyebox with high uniformity for transversely shifting eye positions. Conventional holography suffers from vignetting, and naive pupil-steered holography25 does not account for the axial shift of the eye, resulting in inconsistent image overlap. In addition, variations in the waveguide output and aberrations across different steering states significantly degrade image quality, even when the issue of inconsistent image overlap is addressed, as shown in the third row of Fig. Our method generates robust, high-quality imagery as the eye moves along the optical axis. Note that the pupil position denoted with a blue marker is not used during model training, yet our model achieves comparable image quality and consistent interpolation of parameters, demonstrating its generalization capabilities. a, Visualizations of the supported étendue (3D eyebox). Conventional holography, such as smooth phase techniques (red volume at the origin), supports only a portion of the eyebox that the SLM can produce (intrinsic étendue, orange volume). Our system supports a two-orders-of-magnitude larger eyebox than conventional holographic display systems (green volume). b, Visualization of our learned model across the expanded étendue. For better visualization and interpolation capabilities, refer to the Supplementary Video. c, Experimentally captured resolution chart images with expanded eyebox. Our system supports an eyebox size of 9 × 8 mm, which is significantly larger than the display-limited eyebox of conventional holographic displays. Using our implicit model, we can efficiently calibrate the system, achieving high image quality uniformly across the extended eyebox, as shown in the inset. d, Longitudinal eyebox expansion results. Markers show the corresponding pupil positions in the octahedron shown in a. At runtime, a phase-retrieval-like CGH algorithm computes one or multiple phase patterns that are displayed on the SLM to create a desired intensity image, volume or light field. In the large-étendue setting, conventional 3D content representations, such as point clouds, multilayer images or polygons40,41, are infeasible because the large synthetic aperture requires view-dependent effects to be modelled. For this reason, a light field L is a natural way to represent the target content in this setting. Existing light-field-based CGH algorithms aim to minimize the error between each ray of the target light field and the hologram converted into a light field representation34,43,44, typically using variants of the short-time Fourier transform. Supervising the optimized SLM phase on light rays, however, is not physically meaningful because the notion of a ray does not exist in physical optics at the scale of the wavelength of visible light. To address this issue, we propose a new light-field-based CGH framework. Unlike existing algorithms that rely on ray-based representations for light-field holograms or use random pupils in the Fourier domain58, we supervise our optimization reflecting the wave nature of light more accurately, by incorporating partial coherence and phase continuity at the target field. For this purpose, we formulate the physically correct partially coherent image formation of our hologram to simulate an image passing through the user's pupil, Iholo, and compare it with the corresponding image simulated incoherently from the light field, Ilf, as where 1μ is the indicator function for a set of subapertures μ (a subaperture is a small aperture state spanning the pupil positions closest to the corresponding light field view), and U is the set of all possible subaperture combinations within our synthetic aperture. We thus aim to optimize phase patterns for any possible pupil position, diameter and shape of the user simultaneously. For this purpose, we randomly sample a batch of subaperture configurations in each iteration of our optimization routine, with the loss function \({\mathcal{L}}\) measuring the mean-squared error for each subset of the apertures within the batch (see the Methods and Supplementary Note 5 for additional details). 4, we show experimentally captured 3D holograms. Figure 4a shows photographs captured at various focal distances—0 D (∞ m), 1.5 D (0.67 m) and 2.5 D (0.4 m)—for two different camera positions within the eyebox (that is, shifted laterally by 4.5 mm). Our results achieve the highest quality, exhibiting the best contrast and sharpness, and they demonstrate clear 3D refocusing capabilities as well as view-dependent parallax. The phase SLM patterns do not need to be recomputed for varying pupil positions, diameters or shapes, as all possible configurations are intrinsically accounted for by our unique CGH framework. All diffraction orders created by the SLM are jointly optimized to reduce artefacts59. a, Comparison of holographic rendering techniques using experimentally captured results with different focus states (far: 0 D, middle: 1.5 D, near: 2.5 D) and pupil positions. b, Experimental results with various pupil sizes demonstrate robust image quality in the focused object (left insets), with correctly represented depth of field (right insets) according to the pupil size. RGB channel images were captured separately at the same wavelength and merged to enhance the visual perception of the 3D effect and image quality (pseudo-colour). The co-design of our synthetic aperture waveguide holography hardware and AI-based holography algorithms enables a compact full-colour 3D MR system with large étendue, demonstrating a high-quality holographic light field across a large 3D eyebox within a compact display device form factor. Partial coherence is the key to achieving the high-quality results we demonstrate with a large étendue, because it enables the creation of light-field holograms with high-rank MI, thus offering more degrees of freedom than purely coherent image formations. Our prototype supports a 9 × 8 mm eyebox size. In Supplementary Note 8, we derive the relationship between the transverse eyebox size e, SLM size LSLM, eyepiece focal length f and the maximum steering angle θscan as Indeed, equation (6) shows that the eyebox size of our prototype approaches the theoretical upper bound we could achieve with any holographic display system, as our eyebox size is close to that of the SLM. This is particularly exciting because Fresnel holography configurations, such as ours, typically support only very small eyebox sizes. The proposed co-design of compact large-étendue waveguide architecture and AI-driven algorithms enables compact 3D MR display modes with high image quality across a large eyebox, offering a path towards true 3D holographic MR glasses. Our architecture is shown in Fig. The diffraction-limited resolution of this system is 1.2 arcmin of visual angle, which is close to the human visual acuity of 20/20 vision, that is, 1 arcmin. The layout of the waveguide is similar to typical pupil-replicating waveguides47, with three diffractive elements, an in-coupler, a pupil-expansion coupler and an out-coupler. In our compact prototype, each coupler is an angle-encoded multiplexed VBG hologram to cover the target steering angle of 20°, for three wavelengths of 460 nm, 520 nm and 638 nm. The grating is optically fabricated by recording interference pattern of two laser beams on photo-refractive polymer using custom-designed prisms61. The substrate thickness is 0.6 mm, and anti-reflective coating is applied for visible light. Yet, it is possible to achieve the full bandwidth of the MEMS mirror (400 Hz) with the use of high-frame-rate SLMs that are commercially available62. We do not filter higher-order copies, which contribute to the images by reducing contrast (for example, in Fig. We note that the effect of higher diffraction orders decreases as the pixel pitch size of the SLM decreases. While our SLM has an 8-μm pixel pitch, a pixel pitch below 4.8 μm can effectively separate higher diffraction orders for pupil sizes of 3 mm. The main purpose of the benchtop set-up is to test our waveguide and wave propagation models. For this prototype, we use a conventional eyepiece and SRG waveguide and expand the beam path using multiple 4-f systems to ablate the effect of undiffracted light by optically separating it from the waveguide, while keeping the specifications of wavelengths, SLM, MEMS mirror and geometric parameters similar to those of the compact prototype set-up, which uses the holographic waveguide and eyepiece. In a large-étendue setting, we need to simulate a coherent wave propagation operator for a wide range of spatial frequencies. This is computationally challenging for conventional wave propagation algorithms, because a very large number of spatial positions or, similarly, spatial frequencies must be processed simultaneously. We derive a computationally efficient implementation of this wave propagation in synthetic aperture waveguide holography displays based on bandwidth migration. The naive approach to modelling the propagation of a coherent input field fin with a steering angle um, for example, guided by steered illumination, is to apply a phase ramp function to the input as \({f}_{{\rm{in}}}{e}^{j2\uppi ({{\bf{u}}}^{m}\cdot {\bf{r}})}\). As mentioned above, however, adequately sampling this function would require a prohibitively large number of spatial frequencies to be considered. This off-axis wave propagation with bandwidth migration \({{\mathcal{P}}}_{z}\) can be represented as Note that this calculation can be performed at the same cost as the original wave propagation, without steering. The inverse Fourier transform along the shifted frequency coordinate does not affect the incoherent sum of each propagate mode, while avoiding aliasing in the transfer function \({{\mathcal{H}}}_{z}({\bf{u}}) {e}^{i{{\bf{x}}}^{m}\cdot {\bf{u}}}\) (see Supplementary Note 4 for details). In our implementation of equations (1) and (7), we additionally parameterize and learn non-idealities of the physical optical setup, including nonlinear voltage-to-phase mapping and imperfect diffraction efficiency of the SLM. Thus, in our partially coherent propagation model, each coherent mode is propagated to the target plane at z as where k and m are the index for coherent modes and the synthetic aperture, respectively, \({f}_{{\rm{WG}}}^{\;k}\left(\cdot \right),{c}_{{\rm{WG}}}^{k}\left(\cdot \right)\) are the kth mode of the waveguide output and the corresponding undiffracted component from the SLM, \({\phi }_{{\rm{SLM}}}(\cdot ;m)={\rm{LUT}}\left({\phi }^{m}\right)\) is the nonlinear phase response of the SLM represented as a small MLP33, and \({{\mathcal{P}}}_{z,{\mathcal{A}}}\) is propagation operator with learned aberrations \({\mathcal{A}}\): \({{\mathcal{P}}}_{z,{\mathcal{A}}}(\;f\;)={{\mathcal{F}}}^{-1}\left({\mathcal{F}}({{\mathcal{P}}}_{z}(\;f\;))\times {e}^{i{\mathcal{A}}}\right)\). These modes are added up incoherently and, optionally, an incoherent term to model additional stray light can be added as a separate mode. We note that modelling high diffraction orders and off-axis propagation are essential to capturing the accurate physics of this wave propagation, as presented in Table 1. The waveguide output can be understood as an ensemble of beam patches shaped by multiple total internal reflections and interactions with the gratings. These interactions are governed by the diffraction efficiency of the gratings, which varies spatially owing to non-uniformities, such as fabrication-induced defects, and angularly in response to changing input angles. In addition, propagation within the waveguide necessitates bandwidth migration, as described in equation (7), which reveals predictable wavefront shifts with varying input angles. This understanding motivates our use of a learned coordinate mapping (coordinate transform MLP) that flexibly learns these shifts with angle changes, enabling scalable modelling over a large étendue. Our model is general enough to model steered or non-steered illumination through a waveguide or for a free-space optical set-up. In all cases, we first input the spatial coordinates and steered angles into a coordinate transformation MLP with three layers, 32 features and rectified linear unit nonlinearity. This MLP outputs transformed spatial coordinates, which are then passed through a hyperbolic tangent layer to ensure a valid range. Finally, these transformed coordinates are used to query features using multi-resolution hash encoding37. The features from the hash table are subsequently fed into the main waveguide MLP with 3 layers, 64 features and rectified linear unit nonlinearity, as illustrated in Fig. 2a, and outputs K complex field values at given spatial coordinates. We use K = 4 for the benchtop set-up and K = 16 for the thin benchtop set-up. As discussed in the primary text, this model significantly reduces the number of parameters that need to be optimized and, thus, provides a much more compact representation that can be optimized faster and with less training data than comparable explicit CNN-based models. Moreover, this model is continuous and naturally interpolates the low-rank MI it represents to any spatial frequency on the synthetic aperture. We note that implicit neural representations offer a flexible design trade-off for modelling large-étendue optical systems, as memory scales with signal complexity rather than resolution. These representations have been shown to efficiently recover 3D refractive index maps and volumetric fluorescence images, outperforming classical methods in optical microscopy63,64. Unlike these approaches, which map 3D coordinates to a refractive index or image volume, our model maps 4D spatio-angular coordinates to the complex-valued wavefront or pupil aberrations, making it well suited for modelling large-étendue optical systems. A full-colour captured dataset specific to our holographic MR display prototype and a large-étendue captured dataset are available from the corresponding authors upon request. Computer codes supporting the findings of this study are available online at https://github.com/choisuyeon/sawh. Xiong, J., Hsiang, E.-L., He, Z., Zhan, T. & Wu, S.-T. Augmented reality and virtual reality displays: emerging technologies and future perspectives. Wang, G. et al. Development of metaverse for intelligent healthcare. Chang, C., Bang, K., Wetzstein, G., Lee, B. & Gao, L. Toward the next-generation vr/ar optics: a review of holographic near-eye displays from a human-centric perspective. Hoffman, D. M., Girshick, A. R., Akeley, K. & Banks, M. S. Vergence-accommodation conflicts hinder visual performance and cause visual fatigue. Gabor, D. A new microscopic principle. Grier, D. G. A revolution in optical manipulation. & Brooker, G. Non-scanning motionless fluorescence three-dimensional holographic microscopy. Huang, L. et al. Three-dimensional optical holography using a plasmonic metasurface. Shusteff, M. et al. One-step volumetric additive manufacturing of complex polymer structures. et al. Holographic three-dimensional telepresence using large-area photorefractive polymer. & Wang, J. Holographic optics for thin and lightweight virtual reality. Schwar, M., Pandya, T. & Weinberg, F. Point holograms as optical elements. Tay, S. et al. An updatable holographic three-dimensional display. Breaking crosstalk limits to dynamic holography using orthogonality of high-dimensional random vectors. & Kollin, J. S. Holographic near-eye displays for virtual and augmented reality. Yaraş, F., Kang, H. & Onural, L. State of the art in holographic displays: a survey. et al. 3D holographic head mounted display using holographic optical elements with astigmatism aberration compensation. Kim, J. et al. Holographic glasses for virtual reality. of ACM SIGGRAPH, 1–8 (Association for Computing Machinery, 2022). & Lanman, D. Waveguide holography for 3D augmented reality glasses. Gopakumar, M. et al. Full-colour 3D holographic augmented-reality displays with metasurface waveguides. & Ferreira, C. Space–bandwidth product of optical signals and systems. Wakunami, K. et al. Projection-type see-through holographic three-dimensional display. An, J. et al. Slim-panel holographic video display. Jang, C., Bang, K., Li, G. & Lee, B. Holographic near-eye display with expanded eye-box. & Park, Y. Ultrahigh-definition dynamic 3D holographic display by active control of volume speckle fields. Park, J., Lee, K. & Park, Y. Ultrathin wide-angle large-area digital 3D holographic display using a non-periodic photon sieve. & Chu, D. Holobricks: modular coarse integral holographic displays. Linear algebraic theory of partial coherence: discrete fields and measures of partial coherence. Analysis and Synthesis of Three-Dimensional Illumination Using Partial Coherence (Stanford Univ., 2011). & Wetzstein, G. Gaze-contingent ocular parallax rendering for virtual reality. Peng, Y., Choi, S., Padmanaban, N. & Wetzstein, G. Neural holography with camera-in-the-loop training. Choi, S. et al. Time-multiplexed neural holography: a flexible framework for holographic near-eye displays with fast heavily-quantized spatial light modulators. ACM SIGGRAPH, 1–8 (Association for Computing Machinery, 2022). Goodman, J. W. Introduction to Fourier Optics (Roberts and Company, 2005). Wolf, E. New theory of partial coherence in the space-frequency domain. Part II: steady-state fields and higher-order correlations. Müller, T., Evans, A., Schied, C. & Keller, A. Instant neural graphics primitives with a multiresolution hash encoding. Karaman, M., Li, P.-C. & O'Donnell, M. Synthetic aperture imaging for small scale systems. & Lucente, M. Synthetic aperture holography: a novel approach to three-dimensional displays. Zhao, Y., Cao, L., Zhang, H., Kong, D. & Jin, G. Accurate calculation of computer-generated holograms using angular-spectrum layer-oriented method. Recent progress in computer-generated holography for three-dimensional scenes. Choi, S., Chao, B., Yang, J., Gopakumar, M. & Wetzstein, G. Gaussian wave splatting for computer generated holography. In Processing and Display of Three-Dimensional Data (ed. & Wetzstein, G. Holographic near-eye displays based on overlap-add stereograms. Chae, M., Bang, K., Yoo, D. & Jeong, Y. Étendue expansion in holographic near eye displays through sparse eye-box generation using lens array eyepiece. Jang, C., Bang, K., Lee, B., Li, G. & Maimone, A. Lightguide based holographic display. Levola, T. Diffractive optics for virtual reality displays. Kogelnik, H. Coupled wave theory for thick hologram gratings. & Yeh, P. Optical Waves in Crystal Propagation and Control of Laser Radiation (Wiley, 1983). WISH: wavefront imaging sensor with high resolution. & Teich, M. C. in Fundamentals of Photonics Ch. Sitzmann, V., Martel, J., Bergman, A., Lindell, D. & Wetzstein, G. Implicit neural representations with periodic activation functions. Wightman, A. S., Balazs, N. & Kohn, W. in The Collected Works of Eugene Paul Wigner, Part I: Physical Chemistry, Part II: Solid State Physics, 110–120 (Springer, 2013). The Wigner distribution function applied to optical signals and systems. & Ojeda-Castañeda, J. Phase-Space Optics: Fundamentals and Applications (McGraw-Hill Education, 2010). Matsushima, K. Shifted angular spectrum method for off-axis numerical propagation. Shi, L., Li, B., Kim, C., Kellnhofer, P. & Matusik, W. Towards real-time photorealistic 3D holography with deep neural networks. Chakravarthula, P. et al. Pupil-aware holography. Gopakumar, M., Kim, J., Choi, S., Peng, Y. & Wetzstein, G. Unfiltered holography: optimizing high diffraction orders without optical filtering for compact holographic displays. Matsushima, K. & Shimobaba, T. Band-limited angular spectrum method for numerical simulation of free-space propagation in far and near fields. Jang, C. et al. Design and fabrication of freeform holographic optical elements. Bartlett, T. A., McDonald, W. C. & Hall, J. N. Adapting texas instruments DLP technology to demonstrate a phase spatial light modulator. In Emerging Digital Micromirror Device Based Systems and Applications XI (eds Douglass, M. R. et al.) Vol. Liu, R., Sun, Y., Zhu, J., Tian, L. & Kamilov, U. S. Recovery of continuous 3D refractive index maps from discrete intensity-only measurements using neural fields. Zhang, O. et al. Single-shot volumetric fluorescence imaging with neural fields. We acknowledge K. Jorabchi, A. Scott and F. Li for the fabrication and design of the holographic waveguide used in this work, A. Maimone for the holographic lens used in this work, M. Gopakumar for helpful discussions and C. Smith for the design and mechanical engineering support for prototyping. B. Jennings rendered the light frustum graphics in Fig. The research is supported by Meta. These authors contributed equally: Suyeon Choi, Changwon Jang. Reality Labs Research, Meta, Redmond, WA, USA Suyeon Choi, Changwon Jang & Douglas Lanman Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar supervised all aspects of the project. All authors took part in designing the experiments and writing the article and its Supplementary Information. Correspondence to Douglas Lanman or Gordon Wetzstein. Nature Photonics thanks YongKeun Park and Haoran Ren for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary video for visualizing our system, models and CGH comparisons. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Synthetic aperture waveguide holography for compact mixed-reality displays with large étendue. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03828-y'>Reevaluating the role of education on cognitive decline and brain aging in longitudinal cohorts across 33 Western countries</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 09:51:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Medicine (2025)Cite this article Why education is linked to higher cognitive function in aging is fiercely debated. Leading theories propose that education reduces brain decline in aging and enhances tolerance to brain pathology or that it does not affect cognitive decline but, rather, reflects higher early-life cognitive function. To test these theories, we analyzed 407,356 episodic memory scores from 170,795 participants older than 50 years, alongside 15,157 brain magnetic resonance imaging scans from 6,472 participants across 33 Western countries. More education was associated with better memory, larger intracranial volume and slightly larger volume of memory-sensitive brain regions. However, education did not protect against age-related decline or weakened effects of brain decline on cognition. The most parsimonious explanation for the results is that the associations reflect factors present early in life, including propensity of individuals with certain traits to pursue more education. Although education has numerous benefits, the notion that it provides protection against cognitive or brain decline is not supported. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Each dataset has different owners. Contact information to be used for data access is specified in Supplementary Table 3. Parts of the data used in preparation of this article were obtained from the Alzheimer's Disease Neuroimaging Initiative (ADNI) database (https://adni.loni.usc.edu/). As such, the investigators within the ADNI contributed to the design and implementation of ADNI and/or provided data but did not participate in the analysis or writing of this report. A complete listing of ADNI investigators can be found at https://adni.loni.usc.edu/wp-content/uploads/how_to_apply/ADNI_Acknowledgement_List.pdf. More information about the Vietnam Era Twin Study of Aging (VETSA), including a list of VETSA investigators, is available at https://psychiatry.ucsd.edu/research/programscenters/vetsa/index.html. GBD 2019 Dementia Forecasting Collaborators. Estimation of the global prevalence of dementia in 2019 and forecasted prevalence in 2050: an analysis for the Global Burden of Disease Study 2019. Lancet Public Health 7, e105–e125 (2022). Wolters, F. J. et al. Twenty-seven-year time trends in dementia incidence in Europe and the United States: the Alzheimer Cohorts Consortium. Chen, Y. et al. Dementia incidence trend in England and Wales, 2002–19, and projection for dementia burden to 2040: analysis of data from the English Longitudinal Study of Ageing. Lancet Public Health 8, e859–e867 (2023). Gerstorf, D. et al. Today's older adults are cognitively fitter than older adults were 20 years ago, but when and how they decline is no different than in the past. Livingston, G. et al. Dementia prevention, intervention, and care: 2024 report of the Lancet standing Commission. Suemoto, C. K. et al. Risk factors for dementia in Brazil: differences by region and race. Lock, S. L., Chura, L. R., Dilworth-Anderson, P. & Peterson, J. Equity across the life course matters for brain health. Ritchie, H., Samborska, V., Ahuja, N., Ortiz-Ospina, E. & Roser, M. Global Education. Our World in Data https://ourworldindata.org/global-education (2023). Opdebeeck, C., Martyr, A. & Clare, L. Cognitive reserve and cognitive function in healthy older people: a meta-analysis. B Aging Neuropsychol. Sepulcre, J. College education as a modulator of the aging brain. Cabeza, R. et al. Maintenance, reserve and compensation: the cognitive neuroscience of healthy ageing. Nyberg, L., Lovden, M., Riklund, K., Lindenberger, U. & Backman, L. Memory aging and brain maintenance. Arenaza-Urquijo, E. M. et al. Association between educational attainment and amyloid deposition across the spectrum from normal cognition to dementia: neuroimaging evidence for protection and compensation. Gazzina, S. et al. Education modulates brain maintenance in presymptomatic frontotemporal dementia. Del Ser, T., Hachinski, V., Merskey, H. & Munoz, D. G. An autopsy-verified study of the effect of education on degenerative dementia. Nyberg, L. et al. Educational attainment does not influence brain aging. Stern, Y., Barnes, C. A., Grady, C., Jones, R. N. & Raz, N. Brain reserve, cognitive reserve, compensation, and maintenance: operationalization, validity, and mechanisms of cognitive resilience. What is cognitive reserve? Theory and research application of the reserve concept. Stern, Y. et al. A framework for concepts of reserve and resilience in aging. Lovden, M. et al. No moderating influence of education on the association between changes in hippocampus volume and memory performance in aging. Aging Brain 4, 100082 (2023). Lovden, M., Fratiglioni, L., Glymour, M. M., Lindenberger, U. & Tucker-Drob, E. M. Education and cognitive functioning across the life span. Seblova, D., Berggren, R. & Lovden, M. Education and age-related decline in cognitive performance: systematic review and meta-analysis of longitudinal cohort studies. Schneeweis, N., Skirbekk, V. & Winter-Ebmer, R. Does education improve cognitive performance four decades after school completion? Glymour, M. M., Kawachi, I., Jencks, C. S. & Berkman, L. F. Does childhood schooling affect old age memory or mental status? Using state schooling laws as natural experiments. Gorman, E. Does schooling have lasting effects on cognitive function? Evidence from compulsory schooling laws. Brinch, C. N. & Galloway, T. A. Schooling in adolescence raises IQ scores. Lager, A., Seblova, D., Falkstedt, D. & Lovden, M. Cognitive and emotional outcomes after prolonged education: a quasi-experiment on 320 182 Swedish boys. Courtin, E. et al. Long-term effects of compulsory schooling on physical, mental and cognitive ageing: a natural experiment. & Tucker-Drob, E. M. How much does education improve intelligence? Walhovd, K. B., Lovden, M. & Fjell, A. M. Timing of lifespan influences on brain and cognition. Van Hootegem, A., Rogeberg, O., Bratsberg, B. & Lyngstad, T. H. Correlation between cognitive ability and educational attainment weakens over birth cohorts. Calandri, I. L. et al. Sex and socioeconomic disparities in dementia risk: a population attributable fractions analysis in Argentina. Walhovd, K. B. et al. Education and income show heterogeneous relationships to lifespan brain and cognitive differences across European and US cohorts. Paradela, R. S. et al. Population attributable fractions for risk factors for dementia in seven Latin American countries: an analysis using cross-sectional survey data. Gross, A. L. et al. Harmonisation of later-life cognitive function across national contexts: results from the Harmonized Cognitive Assessment Protocols. Lancet Healthy Longev. Tulving, E. Episodic memory: from mind to brain. Nyberg, L. in The Sage Handbook of Cognitive and Systems Neuroscience Vol. 1 (eds Boyle, G. J. et al.) 47–60 (Sage Publications, 2023). Nyberg, L. et al. Biological and environmental predictors of heterogeneity in neurocognitive ageing: evidence from Betula and other longitudinal studies. Borsch-Supan, A. et al. Data resource profile: the Survey of Health, Ageing and Retirement in Europe (SHARE). Walhovd, K. B. et al. Healthy minds 0–100 years: optimising the use of European brain imaging cohorts (‘Lifebrain'). Mehrbrodt, T., Gruber, S. & Wagner, M. Scales and Multi-item Indicators in the Survey of Health, Ageing and Retirement in Europe (SHARE-ERIC, 2019); https://share-eric.eu/fileadmin/user_upload/SHARE_Working_Paper/WP_Series_45_2019.pdf Zhang, Y. S. et al. Educational attainment and later-life cognitive function in high- and middle-income countries: evidence from the Harmonized Cognitive Assessment Protocol. B Psychol. Wood, S. N. & Scheipl, F. gamm4: generalized additive mixed models using ‘mgcv' and ‘lme4'. R package version 0.2-6 https://CRAN.R-project.org/package=gamm4 (2020). Cadar, D. et al. An international evaluation of cognitive reserve and memory changes in early old age in 10 European countries. Walhovd, K. B. et al. Brain aging differs with cognitive ability regardless of education. Heilbronner, R. L. et al. Official position of the American Academy of Clinical Neuropsychology on serial neuropsychological assessments: the utility and challenges of repeat test administrations in clinical and forensic contexts. Tucker-Drob, E. M., Johnson, K. E. & Jones, R. N. The cognitive reserve hypothesis: a longitudinal examination of age-associated declines in reasoning and processing speed. Strand, S., Deary, I. J. & Smith, P. Sex differences in cognitive abilities test scores: a UK national picture. Eurostat Regional Yearbook, 2024 Edition (Publications Office of the European Union, 2024); https://ec.europa.eu/eurostat/web/products-flagship-publications/w/ks-ha-24-001 Evans, M. D. R., Kelley, J., Sikora, J. Family scholarly culture and educational success: books and schooling in 27 nations. Deary, I. J., Pattie, A. & Starr, J. M. The stability of intelligence from age 11 to age 90 years: the Lothian birth cohort of 1921. Bratsberg, B., Fjell, A. M., Rogeberg, O. J., Skirbekk, V. F. & Walhovd, K. B. Differences in cognitive function at 18 y of age explain the association between low education and early dementia risk. Sharp, E. S. & Gatz, M. Relationship between education and dementia: an updated systematic review. Fürtjes, A. E. et al. Measurement characteristics and genome-wide correlates of lifetime brain atrophy estimated from a single MRI. (in the press). van Loenhoud, A. C., Groot, C., Vogel, J. W., van der Flier, W. M. & Ossenkoppele, R. Is intracranial volume a suitable proxy for brain reserve? Alzheimers Res. Engvig, A. et al. Effects of cognitive training on gray matter volumes in memory clinic patients with subjective memory impairment. Engvig, A. et al. Effects of memory training on cortical thickness in the elderly. Lovden, M. et al. Spatial navigation training protects the hippocampus against age-related changes during early and late adulthood. Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. Brathen, A. C. S. et al. Cognitive and hippocampal changes weeks and years after memory training. de Lange, A. G., Brathen, A. C. S., Rohani, D. A., Fjell, A. M. & Walhovd, K. B. The temporal dynamics of brain plasticity in aging. Judd, N. & Kievit, R. No effect of additional education on long-term brain structure—a preregistered natural experiment in thousands of individuals. Anderson, E. L. et al. Education, intelligence and Alzheimer's disease: evidence from a multivariable two-sample Mendelian randomization study. Okbay, A. et al. Genome-wide association study identifies 74 loci associated with educational attainment. Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Ackerman, P. L. Adult intelligence: the construct and the criterion problem. Li, W. et al. Timing in Early Childhood Education: How Cognitive and Achievement Program Impacts Vary by Starting Age, Program Duration, and Time since the End of the Program (Annenberg Institute for School Reform at Brown University, 2020); https://eric.ed.gov/?id=ED610271 Sameroff, A. J., Seifer, R., Baldwin, A. & Baldwin, C. Stability of intelligence from preschool to adolescence: the influence of social and family risk factors. Tucker-Drob, E. M. & Briley, D. A. Continuity of genetic and environmental influences on cognition across the life span: a meta-analysis of longitudinal twin and adoption studies. Edwin, T. H. et al. Trajectories of occupational cognitive demands and risk of mild cognitive impairment and dementia in later life: the HUNT4 70+ study. Kivimaki, M. et al. Cognitive stimulation in the workplace, plasma proteins, and risk of dementia: three analyses of population cohort studies. Lipnicki, D. M. et al. Age-related cognitive decline and associations with sex, education and apolipoprotein E genotype across ethnocultural groups and geographic regions: a collaborative cohort study. Fry, A. et al. Comparison of sociodemographic and health-related characteristics of UK Biobank participants with those of the general population. Ritchie, S. J., Bates, T. C. & Deary, I. J. Is education associated with improvements in general cognitive ability, or in specific skills? Fjell, A. M. et al. What is normal in normal aging? Effects of aging, amyloid and Alzheimer's disease on the cerebral cortex and the hippocampus. Gruber, S., Hunkler, C. & Stuck, S. Generating easySHARE: Guidelines, Structure, Content and Programming (SHARE-ERIC, 2014); https://share-eric.eu/fileadmin/user_upload/SHARE_Working_Paper/SHARE_WP_Series_17_2014.pdf Bergmann, M., Kneip, T., De Luca, G. & Scherpenzeel, A. Survey Participation in the Survey of Health, Ageing and Retirement in Europe (SHARE), Wave 1-7: Based on Release 7.0.0 (SHARE-ERIC, 2019); https://share-eric.eu/fileadmin/user_upload/SHARE_Working_Paper/WP_Series_41_2019_Bergmann_et_al.pdf & Gruber, S. easySHARE. Release version: 8.0.0. Walhovd, K. B. et al. Neurodevelopmental origins of lifespan changes in brain and cognition. Nilsson, L. G. et al. The Betula prospective cohort study: memory, health, and aging. Nilsson, L.-G. et al. Betula: a prospective cohort study on memory, health and aging. Rajaram, S. et al. The Walnuts and Healthy Aging Study (WAHA): protocol for a nutritional intervention trial with walnuts on brain aging. Vidal-Piñeiro, D. et al. Task-dependent activity and connectivity predict episodic memory network-based responses to brain stimulation in healthy aging. Bertram, L. et al. Cohort profile: the Berlin Aging Study II (BASE-II). Gerstorf, D. et al. Editorial. Shafto, M. A. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) study protocol: a cross-sectional, lifespan, multidisciplinary examination of healthy cognitive ageing. Idland, A. V. et al. Biomarker profiling beyond amyloid and tau: cerebrospinal fluid markers, hippocampal atrophy, and memory change in cognitively unimpaired older adults. Mueller, S. G. et al. The Alzheimer's Disease Neuroimaging Initiative. Neuroimaging Clin. Cattaneo, G. et al. The Barcelona Brain Health Initiative: a cohort study to define and promote determinants of brain health. Dagley, A. et al. Harvard Aging Brain Study: dataset and accessibility. Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Breitner, J. C. S., Poirier, J., Etienne, P. E. & Leoutsakos, J. M. Rationale and structure for a new Center for Studies on Prevention of Alzheimer's Disease (StoP-AD). Tremblay-Mercier, J. et al. Open science datasets from PREVENT-AD, a longitudinal cohort of pre-symptomatic Alzheimer's disease. LaMontagne, P. J. et al. OASIS-3: longitudinal neuroimaging, clinical, and cognitive dataset for normal aging and Alzheimer disease. Preprint at medRxiv https://doi.org/10.1101/2019.12.13.19014902 (2019). Kremen, W. S., Franz, C. E. & Lyons, M. J. Current status of the Vietnam Era Twin Study of Aging (VETSA). Reuter, M., Schmansky, N. J., Rosas, H. D. & Fischl, B. Within-subject template estimation for unbiased longitudinal image analysis. Fischl, B., Sereno, M. I. & Dale, A. M. Cortical surface-based analysis. II: Inflation, flattening, and a surface-based coordinate system. Fischl, B., Sereno, M. I., Tootell, R. B. & Dale, A. M. High-resolution intersubject averaging and a coordinate system for the cortical surface. Dale, A. M., Fischl, B. & Sereno, M. I. Cortical surface-based analysis. Segmentation and surface reconstruction. Destrieux, C., Fischl, B., Dale, A. & Halgren, E. Automatic parcellation of human cortical gyri and sulci using standard anatomical nomenclature. Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. Rutherford, S. et al. Charting brain growth and aging at high spatial precision. Rutherford, S. et al. The normative modeling framework for computational psychiatry. Van Rossum, G. & Drake, F. L. Python 3 Reference Manual (CreateSpace, 2009). Kia, S. M. et al. Closing the life-cycle of normative modeling using federated hierarchical Bayesian regression. Bates, D., Mächler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. Kuznetsova, A., Brockhoff, P. B. & Christensen, R. H. B. lmerTest package: tests in linear mixed effects models. Vidal-Piñeiro, D. et al. Reliability of structural brain change in cognitively healthy adult samples. R: a language and environment for statistical computing. R Foundation for Statistical Computing https://www.r-project.org/ (2024). Bürkner, P.-C. brms: an R package for Bayesian multilevel models using Stan. Carpenter, B. et al. Stan: a probabilistic programming language. Hofman, M. D. & Gelman, A. The No-U-Turn Sampler: adaptively setting path lengths in Hamiltonian Monte Carlo. Wickham, H. ggplot2: Elegant Graphics for Data Analysis 2nd edn (Springer, 2016). Mowinckel, A. M. & Vidal-Piñeiro, D. Visualization of brain statistics with R packages ggseg and ggseg3d. Robinson, D. fuzzyjoin: join tables together on inexact matching. The Lifebrain consortium is funded by EU Horizon 2020 grant agreement number 732592 (Lifebrain). The different substudies are supported by different sources. LCBC is supported by the European Research Council under grant agreement numbers 283634 and 725025 (to A.M.F.) and number 313440 (to K.B.W.) as well as the Norwegian Research Council (325878 and 262453 to A.M.F. ); the National Association for Public Health's Dementia Research Program, Norway (to A.M.F. ); and the University of Oslo through the UiO:Life Science convergence environment (to A.M.F.). Betula is supported by a scholar grant from the Knut and Alice Wallenberg Foundation to L.N. Barcelona is partially supported by a Spanish Ministry of Economy and Competitiveness grant to D.B.-F. (grant no. PID2022–137234OB-100 (AEI/FEDER, UE)) and to G.C. PID-2022–139298OA-C22 (MCIN /AEI /10.13039/501100011033 / FEDER, UE)); by the Walnuts and Healthy Aging Study (grant no. NCT01634841), funded by the California Walnut Commission, Sacramento, California; and by ICREA Academia 2019 and 2024 awards. BASE-II has been supported by the German Federal Ministry of Education and Research under grant numbers 16SV5537, 16SV5837, 16SV5538, 16SV5536K, 01UW0808, 01UW0706, 01GL1716A and 01GL1716B and by the European Research Council under grant agreement number 677804 (to S.K.). A.P.-L. is partly supported by grants from the National Institutes of Health (NIH) (R01AG076708), the Jack Satter Foundation and the BrightFocus Foundation. Part of the research was conducted using the UK Biobank resource under application number 32048. The funders had no role in study design, data collection and analysis, preparation of the manuscript or decision to publish. is funded by the South-Eastern Norway Regional Health Authorities (no. 2017095), by the Norwegian Health Association (no. 1513) and by Wellcome Leap's Dynamic Resilience Program (jointly funded by Temasek Trust) (no. Parts of the data used in preparation of this article were obtained from the Pre-Symptomatic Evaluation of Novel or Experimental Treatments for Alzheimer's Disease (PREVENT-AD) program. Data were provided, in part, by OASIS-3 (OASIS-3 principal investigators: T. Benzinger, D. Marcus and J. Morris; NIH P50AG00561, P30NS09857781, P01AG026276, P01AG003991, R01AG043434, UL1TR000448 and R01EB009352). Parts of the data collection and sharing for this project were provided by the Cambridge Centre for Ageing and Neuroscience (Cam-CAN). Cam-CAN funding was provided by the UK Biotechnology and Biological Sciences Research Council (grant no. BB/H008217/1), together with support from the UK Medical Research Council and the University of Cambridge. Parts of the data are from VETSA, which is funded by National Institute of Aging (NIA) R01 grants AG018384, AG018386, AG050595, AG022381 and AG076838. The content is the responsibility of the authors and does not necessarily represent the official views of the NIA, the NIH, the US Department of Veterans Affairs, the US Department of Defense, the National Personnel Records Center, the National Archives and Records Administration, the Internal Revenue Service, the National Opinion Research Center, the National Research Council, the National Academy of Sciences or the Institute for Survey Research. Temple University provided invaluable assistance in the conduct of the Vietnam Era Twin Registry. The Cooperative Studies Program of the US Department of Veterans Affairs provided financial support for development and maintenance of the Vietnam Era Twin Registry. We would also like to acknowledge the continued cooperation and participation of the members of the Vietnam Era Twin Registry and their families. Part of the data collection and sharing was funded by the Alzheimer's Disease Neuroimaging Initiative (ADNI) (NIH grant U01 AG024904) and DOD ADNI (Department of Defense award no. The ADNI is funded by the NIA and the National Institute of Biomedical Imaging and Bioengineering and through generous contributions from the following: AbbVie, the Alzheimer's Association; the Alzheimer's Drug Discovery Foundation; Araclon Biotech; BioClinica, Inc.; Biogen; Bristol Myers Squibb; CereSpir, Inc.; Cogstate; Eisai, Inc.; Elan Pharmaceuticals, Inc.; Eli Lilly and Company; EuroImmun; F. Hoffmann-La Roche, Ltd. and its affiliated company Genentech, Inc.; Fujirebio; GE Healthcare; IXICO, Ltd.; Janssen Alzheimer Immunotherapy Research & Development, LLC; Johnson & Johnson Pharmaceutical Research & Development, LLC; Lumosity; Lundbeck; Merck & Co., Inc.; MesoScale Diagnostics, LLC; NeuroRx Research; Neurotrack Technologies; Novartis Pharmaceuticals; Pfizer, Inc.; Piramal Imaging; Servier; Takeda Pharmaceutical Company; and Transition Therapeutics. The Canadian Institutes of Health Research is providing funds to support ADNI clinical sites in Canada. Private sector contributions are facilitated by the Foundation for the National Institutes of Health (https://fnih.org/). The grantee organization is the Northern California Institute for Research and Education, and the study is coordinated by the Alzheimer's Therapeutic Research Institute at the University of Southern California. ADNI data are disseminated by the Laboratory for Neuro Imaging at the University of Southern California. Parts of the data used in the preparation of this article were obtained from the Harvard Aging Brain Study (HABS - P01AG036694; https://habs.mgh.harvard.edu). The HABS was launched in 2010, funded by the NIA, and is led by principal investigators R. A. Sperling and K. A. Johnson at Massachusetts General Hospital/Harvard Medical School. The SHARE data collection has been funded by the European Commission, DG RTD through FP5 (QLK6-CT-2001-00360), FP6 (SHARE-I3: RII-CT-2006-062193, COMPARE: CIT5-CT-2005-028857, SHARELIFE: CIT4-CT-2006-028812), FP7 (SHARE-PREP: GA no. 211909, SHARE-LEAP: GA no. 227822, SHARE M4: GA no. 261982, DASISH: GA no. 283646) and Horizon 2020 (SHARE-DEV3: GA no. ; 676536, SHARE-COHESION: GA no. 870628, SERISS: GA no. 654221, SSHOC: GA no. 101015924) and by DG Employment, Social Affairs & Inclusion through VS 2015/0195, VS 2016/0135, VS 2018/0285, VS 2019/0332, VS 2020/0313, SHARE-EUCOV: GA no. 101052589 and EUCOVII: GA no. Additional funding from the German Federal Ministry of Education and Research (01UW1301, 01UW1801, 01UW2202), the Max Planck Society for the Advancement of Science, the NIA (U01_AG09740-13S2, P01_AG005842, P01_AG08291, P30_AG12815, R21_AG025169, Y1-AG-4553-01, IAG_BSR06-11, OGHA_04-064, BSR12-04, R01_AG052527-02, R01_AG056329-02, R01_AG063944, HHSN271201300071C, RAG052527A) and various national funding sources is gratefully acknowledged (see https://www.share-eric.eu/). Center for Lifespan Changes in Brain and Cognition, University of Oslo, Oslo, Norway Anders M. Fjell, Øystein Sørensen, Inge K. Amlien, Håkon Grydeland, Athanasia M. Mowinckel, Lars Nyberg, Markus H. Sneve, Marie Strømstad, Kristine B. Walhovd & Didac Vidal-Piñeiro Computational Radiology and Artificial Intelligence, Department of Radiology and Nuclear Medicine, Oslo University Hospital, Oslo, Norway Anders M. Fjell, Inge K. Amlien & Kristine B. Walhovd Ragnar Frisch Centre for Economic Research, Oslo, Norway Department of Medicine, Faculty of Medicine and Health Sciences, Institute of Neurosciences, University of Barcelona, Barcelona, Spain David Bartrés-Faz & Cristina Solé-Padullés Institut Guttmann, Institut Universitari de Neurorehabilitació adscrit a la UAB, Barcelona, Spain David Bartrés-Faz, Gabriele Cattaneo & Javier Solana Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain David Bartrés-Faz & Cristina Solé-Padullés Center for Lifespan Psychology, Max Planck Institute for Human Development, Berlin, Germany Andreas M. Brandmaier, Sandra Düzel & Ulman Lindenberger Department of Psychology, MSB Medical School Berlin, Berlin, Germany Max Planck UCL Centre for Computational Psychiatry and Ageing Research, Berlin, Germany Andreas M. Brandmaier, Simone Kühn & Ulman Lindenberger Fundació Institut d'Investigació en Ciències de la Salut Germans Trias i Pujol, Barcelona, Spain Gabriele Cattaneo MRC Cognition and Brain Sciences Unit, Department of Psychiatry, University of Cambridge, Cambridge, UK Center for Environmental Neuroscience, Max Planck Institute for Human Development, Berlin, Germany Simone Kühn Department of Psychiatry and Psychotherapy, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Simone Kühn Department of Sociology and Human Geography, University of Oslo, Oslo, Norway Torkild Hovde Lyngstad Umeå Center for Functional Brain Imaging, Umeå University, Umeå, Sweden Department of Medical and Translational Biology, Umeå University, Umeå, Sweden Department of Diagnostics and Intervention, Umeå University, Umeå, Sweden Hinda and Arthur Marcus Institute for Aging Research, Deanna and Sidney Wolk Center for Memory Health, Hebrew SeniorLife, Boston, MA, USA Department of Neurology, Harvard Medical School, Boston, MA, USA Oslo Delirium Research Group, Institute of Clinical Medicine, Campus Ahus, University of Oslo, Oslo, Norway Leiv Otto Watne Department of Geriatric Medicine, Akershus University Hospital, Lørenskog, Norway Leiv Otto Watne Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and D.V.-P. conceptualized the study. analyzed the data. wrote the paper. All authors critically revised the paper and approved the final version. Correspondence to Anders M. Fjell. A.P.-L. serves as a paid member of the scientific advisory boards for Neuroelectrics, Magstim Inc., TetraNeuron, Skin2Neuron, MedRhythms and AscenZion. He is co-founder of TI Solutions and co-founder and chief medical officer of Linus Health. A.P.-L. is also listed as an inventor on several issued and pending patents on the real-time integration of transcranial magnetic stimulation with electroencephalography and magnetic resonance imaging and applications of non-invasive brain stimulation in various neurological disorders as well as digital biomarkers of cognition and digital assessments for early diagnosis of dementia. The other authors declare no competing interests. Nature Medicine thanks Sarah Ackley, Jasmine Mah and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Tables 1–9 and Figs. MRI samples, Data availability, Education in the MRI cohorts, Memory testing in the MRI cohorts, MRI acquisition and preprocessing, SHARE sample distributions, Memory testing in SHARE and Sensitivity analyses of cognition. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Fjell, A.M., Rogeberg, O., Sørensen, Ø. et al. Reevaluating the role of education on cognitive decline and brain aging in longitudinal cohorts across 33 Western countries. Accepted: 06 June 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Medicine © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61797-9'>DynaTag for efficient mapping of transcription factors in low-input samples and at single-cell resolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-28 09:22:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Systematic discovery of transcription factor (TF) landscapes in low-input samples and at single cell level is a major challenge in the fields of molecular biology, genetics, and epigenetics. Here, we present cleavage under Dynamic targets and Tagmentation (DynaTag), enabling robust mapping of TF-DNA interactions using a physiological salt solution during sample preparation. DynaTag uncovers occupancy alterations for 15 TFs in stem cell and cancer tissue models. We highlight changes in TF-DNA binding for NANOG, MYC, and OCT4, during stem-cell differentiation, at both bulk and single-cell resolutions. Furthermore, using tumours of a small cell lung cancer model derived from a single female donor, DynaTag reveals increased chromatin occupancy of FOXA1, MYC, and the mutant p53 R248Q at enriched gene pathways (e.g. epithelial-mesenchymal transition), following chemotherapy treatment. Collectively, we believe that DynaTag represents a significant technological advancement, facilitating precise characterization of TF landscapes across diverse biological systems and complex models. Dynamic transcription factor (TF) interactions with DNA are crucial for gene regulation, cellular identity, development and diseases1,2. Aberrant TF activity is a prominent characteristic of various diseases, including cancer, where TFs often act as oncogenes or lose their native tumour-suppressive functions3,4. Mapping the genomic occupancy of TFs is therefore essential for elucidating gene regulatory networks, identifying molecular vulnerabilities and potential therapeutic targets. Chromatin immunoprecipitation followed by sequencing (ChIP-seq) has long been the gold standard for mapping protein–DNA interactions5. However, this method requires substantial input material for TFs, making it incompatible with single-cell applications. Cleavage Under Targets and Release Using Nuclease (CUT&RUN) has emerged as an alternative approach, allowing for in situ profiling of chromatin interactions—including those involving TFs—with lower input requirements6,7,8. Despite this technical advancement, CUT&RUN requires extensive ligation-based library preparation and, to date, has not been applied to TFs at single-cell resolution9. Cleavage Under Targets and Tagmentation (CUT&Tag) employs protein A fused to the transposase Tn5 (pA-Tn5) as a probe to directly insert sequencing adaptors into DNA at the target sites. CUT&Tag enables robust mapping of histone–DNA interactions in situ within nuclei, in small samples, with single-cell and spatial resolution10,11. However, employing CUT&Tag to map dynamic, low-affinity target–DNA interactions12,13, such as those involving TFs, remains challenging, primarily due to difficulties in retaining these interactions during sample preparation. More recently, alternative tagmentation-based techniques, such as ACT-seq, CoBATCH14, nano-CUT&Tag15 and ChIL-seq16 have emerged. Nevertheless, these technologies require the use of non-physiological, high-salt concentrations to suppress untargeted, false-positive tagmentation events. Importantly, TF–DNA interactions are sensitive to these stringent conditions, causing their dissociation from chromatin and incompatibility with these tagmentation-based technologies. To overcome these limitations, we present cleavage of Dynamic targets and Tagmentation (DynaTag), an adaptation of CUT&Tag. In contrast to CUT&Tag and related modifications10,14,15,16,17,18, DynaTag utilises a physiological intracellular salt solution19 throughout all nuclei handling steps to retain specific interactions while suppressing non-specific protein–DNA interactions. We extensively validate DynaTag for specificity and demonstrate its utility for studying TF occupancy and their dynamics at single-cell resolution and in bulk, with limited sample sizes, using models highly relevant to stem cell biology and cancer research. We rationalised that physiological intracellular salt conditions are required to preserve TF–DNA interactions in situ, as supported by our previous studies using an intracellular salt buffer to maintain the correct topology of dynamic nucleic acid structures inside living cells19. This cation buffer composition is based on electrophysiological salt concentration measurements in situ20 and thereby ensures the retainment of specific TF–DNA interactions during sample preparation (Fig. To first validate DynaTag alongside CUT&Tag, we profiled TFs and histone modifications involved in pluripotency and differentiation in mouse embryonic stem cells (ESCs). We included targets (CTCF, H3K4me3, and H3K27me3) that can be efficiently mapped by CUT&Tag due to their high DNA affinity and resistance to high-salt conditions during nuclei sample handling10. In addition, we included OCT4, SOX2, NANOG, and MYC, TFs associated with ESC identity, as well as YAP1. Only DynaTag but not CUT&Tag successfully generated sequencing libraries for all TFs (Supplementary Fig. Importantly, negative controls (using no primary but only secondary antibodies and pA-Tn5) from both techniques did not result in libraries. To better investigate which component of the DynaTag buffer was crucial for retaining specific TF–DNA interactions, we performed a systematic comparison of the DynaTag physiological salt buffer with other buffers used in CUT&Tag-adapted protocols. We utilised the nuclei washing procedure described in ACT-seq17, CoBATCH14, and G-quadruplex CUT&Tag18. Our results show that the wash buffers used in ACT-seq and CoBATCH caused a dramatic loss in library yield and led to untargeted Tn5 tagmentation in ACT-seq IgG control libraries (Supplementary Fig. Similarly, the use of reduced NaCl concentrations (121 or 150 mM) in the CUT&Tag wash buffer also led to untargeted tagmentation in the IgG controls (Supplementary Fig. Moreover, we showed that the 300 mM KCl buffer used in G-quadruplex CUT&Tag was responsible for failed library generation for all five tested TFs (Supplementary Fig. Consistent with this result, increasing the KCl concentration from 110 to 300 mM in the DynaTag buffer also abolished library generation (Supplementary Fig. To further exclude the occurrence of untargeted tagmentation, we performed DynaTag in the presence or absence of MgCl2 in the wash buffer, in both fixed and native nuclei. A Schematic illustration showing that only DynaTag captures TF interactions by using physiological salt concentrations (110 mM KCl, 10 mM NaCl, 1 mM MgCl2) during nuclei wash steps. In contrast, the use of 150 mM NaCl or 300 mM NaCl or KCl lead to untargeted tagmentation or allows only for the targeting of histone marks, respectively. B Correlation between replicates of DynaTag-mapped TFs in ESC per cell cycle phase. C FRiP (Fraction of Reads in Peaks) of TFs in ESC per cell cycle phase (G0/G1, S, G2/M) called from DynaTag data. Each dot represents an independent replicate N = 2. D IGV snapshots of TF read coverages profiled by DynaTag and ATAC-seq in ESCs. Occupancy is shown as counts per million for 2.5 kb up- and downstream of the peak centres. The signals are shown for 2.5 and 0.5 kb up- and downstream of the transcription start site (TSS). G Top 10 enriched motifs for each TF after motif enrichment analysis using HOMER2. Red bars show the expected motif for each TF. Omni-ATAC-seq uses Tn5 to map accessible chromatin by untargeted tagmentation21. We anticipated that replacing the Omni-ATAC-seq tagmentation buffer with the DynaTag buffer would eliminate library formation. Our results showed that only 150 mM NaCl generated libraries, suggesting untargeted tagmentation. In contrast, 300 mM NaCl or the DynaTag buffer did not produce any libraries (Supplementary Fig. 1K), implying a lack of untargeted tagmentation. Next, we assessed the quality of the DynaTag data, including reproducibility, peak quality and potential cell-cycle contributions. For this purpose, we stained the nuclei with DAPI post-tagmentation and sorted 10,000 nuclei in replicates, according to their cell-cycle state into G0/G1, S and G2/M phases. Genome-wide read distributions correlated well among the replicates (Fig. 1B) with excellent fraction of reads in peaks (FRiP) scores for all five TFs across the cell-cycle states (Fig. To identify peaks with unique or shared co-occupancy patterns, we systematically performed differential occupancy analysis (DOA) of the TFs (OCT4, SOX2, NANOG, MYC, YAP1) profiled in ESCs using DynaTag. Additionally, we examined whether the DynaTag peaks were located in accessible chromatin regions of ESCs, as determined by ATAC-seq. A genome browser snapshot illustrates two representative genes (Fig. In the left highlighted genomic region, which lies within the gene body of Kdm3b, all TFs except MYC showed clear occupancy signals. This specific snapshot demonstrated that TFs can independently occupy certain accessible genomic regions (Fig. Overall, DOA identified six different sets of peaks exhibiting differential occupancies among the five TFs (Fig. For instance, MYC, YAP1, and OCT4 predominantly occupied specific genomic regions (OSNYM, OSNYM, OSNYM), whereas other TFs showed significantly lower occupancy at these sites (FDR < 0.05, log2FC > 0.6); (Fig. In contrast, OCT4, SOX2, and NANOG co-occupied the same regions (OSNYM, OSNYM, OSNYM), either together with MYC or YAP1 or in the absence of MYC, YAP1, and OCT4 (Fig. These occupancy trends indicate specific regulatory programmes in which YAP1 and MYC act mutually exclusive together with OCT4, SOX2, and NANOG in ESCs, further underlining the specificity of DynaTag. Analysis of DynaTag peaks from all five TFs at transcription start sites (TSS) of ChIP-seq derived target genes from ChIP-Atlas22, revealed a remarkable coverage at the expected target genes for each TF (Supplementary Fig. To compare the quality of DynaTag data with existing technologies used to map TFs in murine ESCs, we analysed matched publicly available datasets (OCT4, NANOG, and SOX2) generated by ChIP-seq and CUT&RUN. DynaTag displayed superior enrichment (signal-to-background) and resolution (sharper signal) of binding of all TFs at the TSS of the ChIP-seq derived target genes (Fig. Systematic analyses of known TF binding motifs in the DynaTag, ChIP-seq and CUT&RUN peak sets revealed that the expected motifs were among the top 10 enriched motifs (highlighted in red). Enrichment of expected motifs in DynaTag peaks was notably higher compared to matched CUT&RUN peaks (Fig. 1F), indicating superior specificity of DynaTag over CUT&RUN. Taken together, these results suggest that, in contrast to CUT&Tag and its modified protocols, DynaTag robustly maps TF–DNA interactions with high reproducibility across different cell cycle states and with excellent signal-to-background ratios. Importantly, the physiological K+ concentration (110 mM KCl) in the DynaTag buffer is crucial to maintain the TF–DNA interactions during nuclei sample preparation, while preventing background signal from untargeted tagmentation. In comparison to CUT&RUN, DynaTag data showed superior resolution and specificity in uncovering the expected occupancy landscapes of all TF targets. To assess the ability of DynaTag to capture known TF alterations23,24, we differentiated ESCs into epiblast-like cells (EpiLCs) and compared the occupancy of OCT4, SOX2, NANOG, MYC, and YAP1 in both cell states. First, we determined the FRiP scores for each cell state, which were found to be equally good for all TFs and comparable between ESCs and EpiLCs (Supplementary Fig. DOA of matched TF DynaTag datasets for each cell-cycle state (G0/G1, S, and G2/M) revealed a consistent and substantial decrease in NANOG occupancy across all cell cycle phases. For example, we found on average 15,681 peaks (1807 stdev., log2FC > 0.5, FDR < 0.05) that were significantly higher in read coverage in ESCs, while only 8869 peaks (2210 stdev., log2FC < −0.5, FDR < 0.05) showed a significantly higher read coverage in EpiLCs (Fig. In contrast to NANOG, OCT4, and SOX2 showed a similar number of regions with increased and decreased occupancy changes, while MYC and YAP1 exhibited an overall increase in occupancy in EpiLCs compared to ESCs (Fig. 1D, E), we next investigated whether these behaviours could be quantitatively assessed between ESC and EpiLC states. 1E), the other TFs (O or S or N) showed a similar behaviour in co-occupancy (Fig. For example, for DynaTag peaks that showed a decrease in SOX2 occupancy in EpiLC vs ESC states (ESC > EpiLC), OCT4 and NANOG also displayed a decrease in occupancy in the same DNA regions, thus suggesting co-occupancy (Fig. In contrast, YAP1 and MYC exhibited distinct occupancy patterns without co-occupancy by the other TFs (Fig. A Differential occupancy analysis (DOA) highlights the number of differentially occupied regions (edgeR) for each TF using the DynaTag peaks. B Tornado plots of ESC and EpiLC TF bulk DynaTag occupancy show normalised (cpm) coverages (viridis) and ratios (log2, ESC vs EpiLC) of TF DynaTag at differentially bound (edgeR, >0.5 FC, FDR < 0.05) DynaTag peaks. Red indicates increased, blue decreased occupancy of transcription factors in ESC vs EpiLC (ratio in cpm, log2). E Western blot analysis of chromatin-bound proteins in ESCs and EpiLCs. Ponceau S serves as loading control. F Normalised enrichment scores (NES) derived from GSEA of RNA-seq data of ESC and EpiLC. G Prediction of TF footprints by TOBIAS based on ATAC-seq data in ESC and EpiLC. P values were computed using the TOBIAS BINDetect module51. To independently validate the DOA results, we assessed the protein levels of these TFs in the chromatin-bound fraction of ESCs and EpiLCs by western blot analysis (Fig. 2E) and performed gene set enrichment analysis derived from RNA-seq data (Fig. These two independent assessments confirmed decreased and increased occupancies of NANOG and MYC, respectively, while confirming similar trends of OCT4 and SOX2 in EpiLC versus ESC states. Our findings of increased YAP1 occupancy in EpiLCs compared to ESCs are in line with a previous report demonstrating that YAP1 overexpression in ESCs promotes EpiLC formation25. Next, we aimed to investigate whether ATAC-seq TF footprint analysis would be sufficient to predict the altered TF activities between ESC and EpiLC states revealed by DynaTag (Fig. However, these predictions failed to accurately capture changes in TF occupancy (Fig. Finally, we retrieved ChIP-seq-derived peaks22 for NANOG, OCT4, and SOX2 from ESC and EpiLC states, and repeated DOA with the DynaTag libraries using these ChIP peaks. These results revealed similar trends as observed for the DynaTag peaks, thus further validating previous DynaTag observations (Supplementary Fig. Collectively, these findings underscore the capability of DynaTag to robustly map TF dynamics in distinct cell states. Of note, TF footprint prediction is not robust enough to predict TF occupancy; instead, direct mapping using DynaTag of TF–DNA interactions is necessary to accurately delineate TF activities and their roles in gene regulation. CUT&Tag can robustly profile histone modifications at single-cell resolution11, providing tagmentation-based methods the unique ability to characterise complex samples and multicellular organisms. To robustly map TFs at the single-cell level, we established single-nuclei DynaTag (snDynaTag) and targeted NANOG, MYC, and OCT4 TFs in ESCs and EpiLCs. Genome browser examples show that the merged coverage (aggregated coverage) from single-nuclei profiles per TF and condition (ESC or EpiLC) aligns well with the respective bulk DynaTag profiles (Fig. The bottom row of the genome browser snapshots displays peaks where a clear change in occupancy occurred for the TFs, i.e., NANOG shows a loss of occupancy at particular peaks in the EpiLC state that are clearly visible in the ESC state. Importantly, FRiP scores and genome-wide correlation with the respective bulk data sets indicate that snDynaTag shows similar quality to and correlates well with the bulk DynaTag landscapes (Fig. To address whether snDynaTag has the potential to uncover differential occupancy of a particular TF in two closely related cell states, we derived fold changes of the genome-wide coverages of the aggregated TF snDynaTag for ESC vs EpiLCs. Coverage analyses of fold changes in the combined peaks derived from the two aggregated snDynaTag data sets revealed thousands of peaks for which the TFs displayed increased (log2FC > 0.5) or decreased (log2FC < −0.5) binding (Fig. In accordance with bulk NANOG and MYC DynaTag DOA trends (Fig. 2A–C), snDynaTag showed an overall loss in NANOG occupancy while an increase for MYC occupancy in the EpiLC state was observed (Fig. We removed nuclei with low counts (2nd percentile; Supplementary Fig. 4B) and applied a z-score normalisation approach (see 'Methods') to account for single-nucleus coverage at peaks that showed consistent changes across nuclei in the ESC versus EpiLC groups (Fig. We next investigated whether dimensionality reduction analysis of individual TF profiles from ESC and EpiLC states would reveal clusters predominantly populated by one cell type over the other. For all three TFs, we revealed distinct clusters originating from TF occupancy in ESC or EpiLCs (Fig. This suggests that TF snDynaTag can resolve cell-type-specific occupancy of TFs, thus demonstrating the capability to uncover alterations in TF occupancy with single-cell resolution. A Genome tracks showing the comparison of bulk DynaTag, aggregate snDynaTag profiles and the occupancy from single nuclei for NANOG, MYC and OCT4 in ESC and EpiLC states. B Genome-wide correlation between bulk and snDynaTag data for each TF in ESC and EpiLC. C Number of differential occupied regions obtained for ESC vs EpiLC (log2, >0.5 and <−0.5) for each TF in the DynaTag peaks. D Tornado plots of differential TF (NANOG, MYC and OCT4) snDynaTag occupied regions. Normalised aggregated coverages are shown for ESC and EpiLC, as well as fold changes (log2, ESC vs EpiLC). E Distribution of log1p, or log(x + 1), values for the total number of reads in peaks that passed z-score filtering, shown for cells after quantile filtering and linked to each TF in ESCs and EpiLCs profiled by snDynaTag. G UMAPs of snDynaTag for OCT4, NANOG and MYC showing distinct clusters for ESC and EpiLC states. To discover dynamic changes of TF–DNA interactions in a complex disease model in vivo, we investigated small cell lung cancer (SCLC). Importantly, nearly all tumours respond to platinum-based therapy but then acquire chemoresistance33. For this reason, we aimed to investigate changes in TF occupancies following chemotherapy treatments in a patient-derived xenotransplantation (PDX) model for SCLC. The PDX model showed initially a good response to in vivo treatment with cisplatin and etoposide, and we subsequently analysed recurring tumours and respective control samples (n = 2 control, n = 2 chemo-treated). We employed bulk DynaTag and single-nuclei RNA-seq (snRNA-seq) to study chemotherapy treatments in a PDX model for SCLC, which harboured mutations in TP53 and TP73, and predominant expression of ASCL1 (Fig. We mapped binding and quantified increased and decreased occupancies of 11 TFs (ASCL1, FOXA1, MYC, NEUROD1, NFIB, NRF1, POU2F3, p53, P73, SP2, and YAP1) in chemotherapy-treated relative to control tumours, for which transcripts were detectable (Fig. These results suggest increased FOXA1 and MYC activity in SCLC cells following chemotherapy, and a decreased number of cells driven by ASCL1 and POU2F3 activity in vivo. To determine if changes in TF occupancy are linked to functional alterations in SCLC, we integrated DynaTag data with gene pathway activity changes from matched snRNA-seq data of the same tumours (Fig. Functional integration revealed a significant increase in FOXA1 and MYC occupancies, as well as a significant decrease in ASCL1 and POU2F3 occupancies at the promoters of genes identified as altered in the GSEA (Fig. Transcriptional plasticity of key neuroendocrine TFs has recently been observed. Specifically, decreased ASCL1 along with increased MYC gene expression has been linked to a non-neuroendocrine state, thus suggesting selection of this phenotype in our PDX model29,34,35. Our results further showed significantly increased FOXA1 and MYC occupancies at genes involved in several metabolism pathways, such as fatty acid metabolism (Fig. Intriguingly, we also uncovered a significant increase in mutant p53 occupancy (p.R248Q variant) at promoters of genes related to epithelial-mesenchymal transition (EMT) (Fig. This suggests that the p53 variant acts as a potential driver of EMT, serving as a potential chemotherapy escape mechanism in SCLC. This finding aligns with previous reports linking the p53 R248Q variant to EMT, cell locomotion and increased aggressiveness36,37. We next asked whether the occupancy pattern of p53 R248Q identified by DynaTag in SCLC would resemble that observed by ChIP-seq for the same variant in non-small cell lung cancer (NSCLC)38. Strikingly, the DynaTag SCLC read coverage was notably enriched in peaks derived from the NSCLC p53 R248Q variant ChIP-seq data, while lacked any notable coverage in the wild type p53 peaks39 (Fig. Taken together, the DynaTag p53 landscapes acquired in SCLC suggest a gain-of-function for p53  R248Q in response to chemotherapy at promoters of signature genes related EMT, thus highlighting the potential of DynaTag for uncovering in vivo significant TF alterations. A Illustration of functional genomics set-up to identify altered gene-pathways that are differentially occupied by TFs in a SCLC PDX model treated with and without chemotherapy. C Profiles of TF read coverages across gene promoters of specific hallmarks that are differentially enriched in transcriptomes derived from snRNA-seq. D Matrix displaying the ratio (log2) of TF signal (Chemotherapy/Control) at promoters of genes from the affected pathways. Size of bubbles represents the p value (−log10 transformed) derived from a paired t test per TF signal per pathway. E Tornado plot matrix of differential p53 occupancy for wild type (WT) and R248Q p53 mutant. Rows indicate the ChIP-seq or DynaTag signals, cancer type (SCLC PDX or NSCLC) and sample type (normalised read coverage (cpm)), which were plotted in peaks (WT_NSCLC (3034), R248Q_NSCLC (3875)) derived from the different ChIP-seq and DynaTag data sets (columns). CUT&Tag and its modified versions typically employ high-salt concentrations (e.g. 150–300 mM NaCl or 300 mM KCl) to prevent untargeted tagmentation by protein A-Tn510,14,15,17,18. We reasoned that the intracellular environment, such as its ion composition and concentration, must be recapitulated during nuclei preparation steps to retain specific and prevent non-specific protein–DNA interactions. Our results show that continuously treating extracted nuclei with an intracellular-derived salt solution (DynaTag buffer) effectively suppresses untargeted tagmentation. Importantly, this post-extraction treatment also preserves specificity for both highly abundant, high-affinity binding targets (such as histone modifications) and low-abundance, low-affinity binding targets (such as TFs). We therefore conclude that conventionally used high salt concentrations reduce not only non-specific but also specific interactions, particularly for targets that are low in abundance and/or exhibit weak specific interactions with DNA. Of note, the physiological potassium concentration in the DynaTag buffer and the nuclei wash procedure were key to unlocking the robust mapping of TFs in small samples and at single-cell resolution. We emphasise that droplet-based single-cell CUT&Tag and nano-CUT&Tag rely on high-salt concentrations, which suggests that current protocols are likewise unsuitable as a robust method for mapping targets sensitive to stringent conditions. The authors themselves acknowledged that this remains a limitation requiring robust validation15. We have demonstrated here that DynaTag achieves superior data quality, occupancy resolution and signal-to-background ratio compared to ChIP-seq and CUT&RUN, as shown by comparisons of matched datasets generated from the same cellular system (Fig. Overall, we successfully mapped 15 different TF targets in both simple cellular models and complex tissue samples, which are highly relevant for developmental biology and cancer research. Additionally, from a single PDX tumour sample, we successfully conducted 22 distinct DynaTag reactions covering 11 different TF targets, highlighting the quality, versatility and efficiency of this method. Given that many researchers currently employ the original CUT&Tag procedure, we anticipate that transitioning to DynaTag will be both time- and cost-efficient. Quantification of transcript and protein levels supported the differential occupancy patterns observed with DynaTag. However, predicted TF activities based on ATAC-seq footprinting did not match these occupancy trends between ESCs and EpiLCs. This discrepancy reinforces the importance of validating TF activity predictions derived from ATAC-seq data. Spatially resolved ATAC-seq and CUT&Tag allow localisation of epigenetic activities at the single-cell level within their native microenvironment40,41,42. We anticipate that spatially resolved mapping of TF occupancy will become feasible by integrating DynaTag with existing spatial CUT&Tag methodologies40,43,44,45. In addition, future work on development of spatial technologies for the mapping of TF-DNA interactions may provide a useful tool to uncover biologically relevant TFs alterations in situ in complex tissues. In the discovery phase of our study, we examined a clinically relevant model of SCLC with high translational significance. SCLC remains among the deadliest cancers, with a 5-year survival rate below 5%. Although SCLC patients initially respond exceptionally well to chemotherapy, unknown resistance mechanisms lead to relapse in the majority of cases33. A multitude of TFs have been suggested to drive oncogenesis and neuroendocrine phenotypes in SCLC, which warrants the need to uncover TF occupancies throughout the deadly course of this disease. By applying DynaTag to identify previously unknown TF activities in SCLC following chemotherapy, we discovered significantly increased promoter occupancy by the gain-of-function R248Q variant of p53. This variant exhibited enhanced activity at promoters of genes relevant to EMT that also showed higher expression in chemotherapy-treated SCLC. Importantly, our findings provide insights into the role of the frequently mutated TP53 locus in SCLC, particularly since point mutations in p53 are associated with sensitivity to chemotherapy treatment26,27 and similar gain-of-function mutations have been documented in other cancer types46,47. Additionally, DynaTag revealed significantly increased occupancy of FOXA1 and MYC at promoters of genes related to multiple differentially expressed gene sets following chemotherapy, suggesting that these TFs may be critical for the survival of chemotherapy-resistant cells. Key changes in the experimental procedure allowed the development of DynaTag to resolve TF occupancy landscapes with so far unmatched sensitivity, selectivity and signal-to-background ratios in small samples and a complex disease model. Finally, we demonstrated that single-nuclei DynaTag of TFs is feasible, enabling future studies of TFs and other chromatin-associated factors in complex tissue samples. The patient sample S0273026 was collected under approved protocols of the institutional review board (IRB, ID: 19-1164) of the University of Cologne and provided written informed consent to share potentially identifying individual-level data, gender: female, age at the time of first diagnosis: 64 years. All animal experiments were approved and conducted according to the regulations of local animal welfare authorities of the State Agency for Nature, Environment and Consumer Protection of North Rhine-Westphalia State of Nordrhein-Westfalen (LANUV, AZ: 84-02.04.2018.A002). For epiblast-like cell differentiation, 200,000 ESCs were plated onto 15 μg/mL fibronectin-coated wells (Corning, 354,008) in 2i/L medium and allowed to attach overnight. A complete list of all chemical reagents is provided as Supplementary Data 1. For nuclei isolation, cells were incubated in NE1 buffer (20 mM HEPES-KOH pH 7.9 (Carl Roth, 6763.3), 10 mM KCl (Sigma-Aldrich, P9333), 0.5 mM Spermidine (Sigma-Aldrich, S0266), 0.1% TritonX-100 (Sigma-Aldrich, X100), 20% glycerol (Carl Roth, 3783.2), 1× Complete Protease Inhibitor (Roche, 11836170001)) for 3 min on ice. Nuclei were fixed with 0.1% formaldehyde (Thermo Scientific, 28906) in PBS for 1 min at room temperature while gently agitated. Formaldehyde was quenched with 74 mM glycine (Sigma-Aldrich, 50046), and nuclei were pelleted by centrifugation at 450 × g for 3 min at 4 °C. Total RNA from cultured cells were extracted using the NucleoSpin RNA Kit (Macherey-Nagel, 740955.50) according to the manufacturer's protocol. RNA-seq libraries were generated with the QuantSeq 3' mRNA-Seq Library Prep Kit FWD with Unique Dual Indices for Illumina (Lexogen, 115.384) and sequenced on an Illumina NovaSeq 6000 platform using NovaSeq 6000 SP Reagent Kit v1.5 (100 cycles) (Illumina, 20028401). All RNA-seq experiments were conducted in four independent technical replicates. Total RNA from SCLC PDX tumours was extracted analogously to in vitro cultured cells. For cDNA synthesis, 1000 ng of total RNA was reverse transcribed using the High-Capacity cDNA Reverse Transcription Kit (Applied Biosystems, 4368814). Quantitative PCR was performed using pre-designed primers from Integrated DNA Technologies (IDT) on a Bio-Rad CFX Opus Real-Time PCR platform. No template controls served as background estimation. Data are shown as log10-transformed 2−ΔCt values from two replicates. Following previously described approaches26,49, patient-derived tumour cells were isolated from the blood as circulating tumour cells and subcutaneously engrafted onto immunocompromised female (7–14 weeks) NOD scid gamma (NSG™) mice to establish a xenotransplant tumour model for patient S02730 (female, age at first diagnosis: 64 years). Following in vivo tumour growth and several passages in NSG female mice (7–14 weeks), tumour tissues were dissociated to single cell suspensions, and 1 × 106 cells were subcutaneously engrafted into both flanks of 7–14 weeks old NSG (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) female mice. At a tumour volume of 100–200 mm3, mice were randomised into control and chemotherapy cohorts; n = 2 mice for each condition. Tumours were harvested at a maximum volume of 1500 mm3, as permitted by the animal welfare authorities. Animals were hosted in a pathogen-free facility on a 12 h light/dark cycle at uniform temperature and humidity. For nuclei isolation from frozen tissue, samples were briefly thawed on ice and minced on ice with a scalpel. Nuclei were isolated by running the ‘Protein-M-Tube-1.0' programme. Three milliliters L2 (L1 with 0.4 % TritonX-100 (Sigma-Aldrich, X100) were added, and suspension was filtered through a 40 µm cell strainer (Sarstedt, 83.3945.40). Supernatant was carefully removed without agitating the layers, and nuclei were resuspended in the appropriate buffer for downstream application. For single-nuclei RNA-seq, all buffers were supplemented with 120 U/mL murine RNase inhibitor (NEB, M0314L). For single-nucleus RNA-seq, nuclei were isolated as described in the general methods. After isolation, nuclei were counted with trypan blue and fixed using the Evercode Nuclei Fixation v2 Kit (Parse Bioscience, ECF2003) and stored at −80 °C until library preparation. Library preparation of fixed nuclei was performed with the Evercode WT Kit (Parse Bioscience, ECW02130) according to the manufactures' protocol. Sequencing fastq files were demultiplexed into single-nucleus data using the Split Pipe pipeline off Parse Bioscience (v1.0.6). Single-nucleus RNA-seq data were analysed in R using the Seurat (v 5.0.1) package. First, all four samples were pre-processed separately to perform quality control, which included filtering data for low-quality cells by removing the upper and lower 2% of the nuclei-based transcript coverage. Subsequently, data was further normalised using the sctransform (v 0.4.1) package to prepare samples for integration. All four samples were then read into R in parallel and merged into one SeuratObject, the features were reduced to the union, and variable features were identified. Upon dimensionality reduction via PCA, neighbours and clusters were identified and plotted into UMAP. Cell types were assigned to clusters manually based on the expression of lineage-specific marker genes. To exclude non-malignant tumour microenvironment cells, the SeuratObject was subsetted to the small cell lung cancer clusters. For differential expression analysis, the count data were aggregated into pseudobulk counts per sample and a combined transcripts-by-sample count matrix was generated. The combined count matrix was filtered for transcripts of a rowSums <10. DESeq2 was run with standard parameters and the control samples as reference. Significantly differentially expressed genes were identified (log2FoldChange > 0.5 or <−0.5 and padj < 0.1). A slightly more relaxed adjusted p value of 0.1 was chosen as the single-nucleus RNA-seq was only performed in duplicates and therefore offers limited statistical power. The normalised enrichment score of significantly enriched pathways (NOM p val < 0.05) was plotted using ggplot2. Western blot analysis of subcellular fractions was performed as previously described in ref. Briefly, cells were first lysed in E1 buffer (50 mM HEPES-KOH pH 7.5 (Carl Roth, 6763.3), 140 mM NaCl (Sigma-Aldrich, S5886), 1 mM EDTA (Sigma-Aldrich, E6758), 10% glycerol (Carl Roth, 3783.2), 0.5% NP-40 alternative (Sigma-Aldrich, 492016), 0.25% TritonX-100 (Sigma-Aldrich, X100), 1 mM DTT (Fisher Scientific, R0861), 1× Complete Protease Inhibitor (Roche, 11836170001)) to isolate the cytoplasmatic fraction. Chromatin-bound proteins were isolated by incubation with ~500 U Benzonase (Sigma Aldrich, E1014) for 20 min at room temperature in E3 buffer (500 mM Tris pH 6.8 (Sigma-Aldrich, 93362), 500 mM NaCl (Sigma-Aldrich, S5886), 1× Complete Protease Inhibitor (Roche, 11836170001)). To increase the isolation efficiency of chromatin-bound proteins, chromatin was sonicated for five cycles (30 s ON, 30 s OFF) on a high setting of a Diagenode Bioruptor plus before Benzonase digest. Debris from all fractions was removed by centrifugation for 10 min at 16,000 × g and 4 °C. 10–20 µg of protein was separated in Mini-PROTEAN TGX Precast Protein Gels (Bio-Rad, 4561086) and transferred onto nitrocellulose membranes (Bio-Rad, 1704270). Total proteins were visualised by Ponceau S (Carl Roth, 5938.1) and served as a loading control. After blocking the membrane in 5% milk in TBS-T (20 mM Tris pH 7.0, 150 mM NaCl, 0.1% Tween 20), membranes were incubated 1:1000 diluted primary antibodies overnight at 4 °C. The next day, membranes were washed three times with TBS-T and incubated with 1:2500 diluted anti-rabbit-HRP secondary antibody (Invitrogen, 32460) and visualised with Clarity Max Western ECL Substrate (Bio-Rad, 1705062). All images were acquired on the Bio-Rad ChemiDoc MP Imaging System. ATAC-seq experiments were performed as previously described in ref. Briefly, 50,000 isolated nuclei were tagmented with 2.5 µL TDE1 enzyme (Illumina, 15027865) for 30 min at 37 °C. Upon completion of tagmentation, enzymatic reactions were immediately stopped by adding 3 volumes of ERC binging buffer from the MinElute Reaction Cleanup Kit (QIAGEN, 28204). Purified DNA was dual-indexed and amplified with Nextera XT Index Kit v2 (Illumina, FC-131-2001). All ATAC-seq experiments were performed in three to four independent replicates. TOBIAS51 was used to identify TF footprints in ATAC-seq data. We investigated TF footprints in the consensus peak regions of ESC and EpiLC in their respective BAM files. The analysis was performed as described for TOBIAS. TF footprints for all vertebrates were downloaded from (https://jaspar.genereg.net). Protein A-Tn5 was expressed and purified as previously described10. Briefly, T7 Express lysY/Iq competent E.coli (NEB, C3013I) were transformed with the pTXB1-pA-Tn5 (Addgene, 124601) and cultured in LB Broth (Sigma, L3022) with 100 µg/mL ampicillin (Thermo Scientific Chemicals, J60977.14) for 4 h at 37 °C prior to inoculation of additional 1970 mL LB broth supplemented with 100 µg/mL ampicillin for expression. The 2 L culture was incubated at 37 °C until OD600 reached 0.5–0.7 and was then cooled to 4 °C for 1 h. Expression was induced by adding 0.25 mM IPTG (Thermo Scientific, R0393) overnight at 18 °C. Bacteria were lysed in HEGX buffer (20 mM HEPES, pH 7.2 (KOH) (Carl Roth, 6763.3), 1 M NaCl (Sigma-Aldrich, S5886), 1 mM EDTA (Sigma-Aldrich, E6758), 10% glycerol (Carl Roth, 3783.2), 0.2% TritonX-100 (Sigma-Aldrich, X100)) supplemented with 1× Complete Protease Inhibitor (Roche, 11836170001) using a Sonicator (BIOBLOCK SCIENTIFIC, Vibra-Cell 75043) for 10 cycles (45 s on/15 s off) with 50% amplitude on ice. In the meantime, 2.5 mL chitin resin (NEB, S6651S) was packed into Econo-Pac Chromatography Column (BIO-RAD, 732-1010) and washed with 20 mL HEGX buffer. The washed chitin resin was incubated overnight at 4 °C on a roller with the cleared lysate to bind protein A-Tn5. Protein A-Tn5 was eluted by adding 6 mL HEGX buffer supplemented with 100 mM DTT (Fisher Scientific, R0861) for 48 h at 4 °C on a roller. Eluate was dialysed against 4 L of 2x Tn5 dialysis buffer (100 mM HEPES, pH 7.2 (KOH) (Carl Roth, 6763.3), 0.2 M NaCl (Sigma-Aldrich, S5886), 0.2 mM EDTA (Sigma-Aldrich, E6758), 1.7 mM DTT (Fisher Scientific, R0861), 0.2% TritonX-100 (Sigma-Aldrich, X100), 20% glycerol (Carl Roth, 3783.2)) using a Slide-A-Lyzer 10 MWCO Dialysis Cassette (Thermo Scientific, 66380) for 24 h. Protein was concentrated using an Amicon Ultra Centrifugal Filter, MWCO 30 (Merck, UFC9030) to reach ~11 µM protein A-Tn5. Concentrated protein was mixed with equal volume glycerol (Carl Roth, 3783.2) and stored at −20 °C. For transposase preparation, 16 µL of 100 µM annealed Tn5MEDS A and Tn5MEDS B oligonucleotides were mixed with 100 µL of Protein A-Tn5 glycerol stock and incubated for 50 min at 23 °C. The prepared transposase was stored at −20 °C. CUT&Tag experiments were performed as previously described with the following modifications10. Instead of immobilisation of nuclei on concanavalin A conjugated beads, nuclei were washed by centrifugation at 450 × g for 3 min at RT. 3 × 105 fixed nuclei were incubated with 1:100 diluted primary antibodies in 50 µL W150 buffer (20 mM HEPES pH 7.5 (KOH) (Carl Roth, 6763.3), 150 mM NaCl (Sigma-Aldrich, S5886), 2 mM EDTA (Sigma-Aldrich, E6758), 0.5 mM Spermidine ((Sigma-Aldrich, S0266), 0.01% (w/v) Digitonin (Carl Roth, 4005.1), 1% (w/v) BSA (Sigma-Aldrich, A7030)) on a nutating platform at 4 °C overnight. The next day, nuclei were washed once with 200 µL W150 buffer and incubated for 1 h at room temperature with 0.5 µg anti-rabbit IgG antibody (NOVUS BIOLOGICALS, NBP1 72763) in W150 buffer. Nuclei were washed three times with 200 µL W300 buffer, and Tn5 was subsequently activated by adding 10 mM MgCl2 (Invitrogen, AM9530G) for 1 h at 37 °C. Upon tagmentation, nuclei were counted using an automated cell counter and equal number of nuclei were transferred into new tubes and denaturated in 50 µL Denaturation buffer (10 mM Tris pH 8.0 (HCl) (Sigma-Aldrich, 93362), 50 mM NaCl (Sigma-Aldrich, S5886), 0.2% (w/v) SDS (Fisher Scientific, BP8200100), 0.5 ng/mL proteinase K (Ambion, AM2546)). Denaturation was stopped by purification of the tagmented DNA fragments using the MinElute Reaction Cleanup Kit (QIAGEN, 28204). Amplified sequencing libraries were purified with 1.3× SPRI beads (MAGBIO, AC-60050) ratio, followed by elution in 26 µL EB buffer. All CUT&Tag experiments were performed in two independent replicates. For DynaTag, 3 × 105 fixed nuclei were incubated with 1:100 diluted primary antibodies on a nutating platform at 4 °C overnight in AB buffer (25 mM HEPES pH 7.5 (KOH) (Carl Roth, 6763.3), 110 mM KCl (Sigma-Aldrich, P9333), 10 mM NaCl (Sigma-Aldrich, S5886), 1 mM MgCl2 (Invitrogen, AM9530G), 0.01% (w/v) Digitonin (Carl Roth, 4005.1), 1% (w/v) BSA (Sigma-Aldrich, A7030)), 0.5 mM spermidine (Sigma-Aldrich, S0266)) in 0.2 mL PCR tubes. The next day, nuclei were pelleted by centrifugation at 450 × g for 3 min at RT incubated for 1 h at room temperature with 0.5 µg anti-rabbit secondary antibody (NOVUS BIOLOGICALS, NBP1 72763) or 0.5 µg anti-mouse secondary antibody (abcam, ab46540). Nuclei were washed twice by gentle resuspension in 200 µL AB buffer. Nuclei were incubated with 1:200 diluted pA-Tn5 complex in AB buffer for 1 h at RT with gentle agitation. Nuclei were washed twice by gentle resuspension in 200 µL AB buffer. Tagmentation was initiated by incubating nuclei at 37 °C for 1 h in AB buffer supplemented with 10 mM MgCl2 (Invitrogen, AM9530G). Upon tagmentation, nuclei were pelleted by centrifugation at 450 × g for 3 min at RT and resuspended in 100 µL DynaTag buffer (AB buffer without BSA, spermidine and digitonin) and an aliquot was mixed with trypan blue and counted using an automated cell counter. Alternatively, nuclei can be separated by their cell cycle phase using FACS upon DAPI staining. 10,000 nuclei were transferred into new 0.2 mL PCR tubes and incubated for 3 h at 58 °C in 50 µL Denaturation buffer (10 mM Tris pH 8.0 (HCl) (Sigma-Aldrich, 93362), 50 mM NaCl (Sigma-Aldrich, S5886), 0.2% (w/v) SDS (Fisher Scientific, BP8200100), 0.5 ng/mL proteinase K (ambion, AM2546)). Denaturation was stopped by purification of the tagmented DNA fragments using the MinElute Reaction Cleanup Kit (QIAGEN, 28204). An appropriate number of cycles can be estimated via qPCR and will depend on epitope abundancy. Amplified sequencing libraries were double-sided size-selected using first a 0.5× SPRI beads (MAGBIO, AC-60050) ratio to remove large fragments followed by purification with 1.3× SPRI beads ratio, followed by elution in 26 µL EB buffer. All DynaTag experiments were performed in two independent biological replicates. BAM files were prepared using deeptools (v3.5.1) multiBamSummary (bins -bs 10). Correlation was plotted via deeptools plotCorrelation (--corrMethod pearson --skipZeros --whatToPlot scatterplot --log1p). Indexed BAM files were converted into cpm-normalised bigwig files using deeptools (v3.5.1) bamCoverage (-bs 10 --skipNAs --centerReads --normalizeUsing CPM -of bigwig). Bigwigs were loaded into IGV and visualised as group auto-scaled bars. Aligned BAM files were down-sampled to equal reads per TF using samtools view (-@ 8 -c -F 260) and sorted with samtools sort. Peaks were called from sorted BAM files via MACS2 (v2.2.7.1) (-f BAMPE -g mm --keep-all --nomodel --extsize 55 -B –SPMR) from mouse cells or (-f BAMPE -g hg --keep-all --nomodel --extsize 55 -B --SPMR) from human PDX samples and filtered for conserved peaks between replicates using bedtools multiIntersectBed. Conserved peaks per condition were concatenated, followed by sorting and merging to remove redundant peaks with bedtools sortBed | mergeBed. To count reads, bedtools coverage was used to generate bedgraph files, which were then converted into plain text files. For FRiP calculation, overlapping reads with peaks were identified using bedtools (v3.5.1) intersect (-wa -u), and reads were counted using samtools (v1.13) view (-c -F 0 × 2). Finally, overlapping peaks were divided by total reads. FRiP scores were plotted in R using ggplot2 (v3.5.0). Target genes (±1 kb) per TF were downloaded from the ChIP-Atlas repository22 and used to extract the chromosome coordinates to generate a BED file using the GenomicRanges (v1.54.1) R package. Random non-target genes were extracted via setdiff command. HOMER2 (v5.1, 7-16-2024) was used to quantify the top 10 most enriched known motifs in the matched ESC DynaTag, ChIP-seq and CUT&RUN data. For details, please refer to the code available at https://github.com/HaenselHertschEpiLab/DynaTag. For differential binding analysis, peak count matrices were read into R and analysed with edgeR (v 4.0. Dispersion was estimated with estimateGLMCommonDisp and estimateGLMTagwiseDisp, and data was fitted with glmFit followed by statistical estimation using glmLRT. DiffBind (Galaxy Version 2.10.0) coupled with DESeq2 and the significance threshold FDR < 0.1 was used in usegalaxy.eu52 to identify differentially occupied regions. Default median-of-ratios was used to normalise the count matrix by DEseq2. Around 4 × 105 mESCs were seeded for this experiment. Nuclei were centrifuged at 450 × g for 10 min and incubated with anti-Rabbit or anti-Mouse IgG antibody (1:50 dilution) on a rotator at RT for 1 h. Nuclei were then washed twice with 200 μL AB buffer and incubated with a 1:100 dilution of pA-Tn5 adaptor complex on a rotator at RT for 1 h. Cells were washed twice with 200 μL AB buffer, resuspended in 100 μL Tagmentation buffer (AB buffer with 10 mM MgCl2) and incubated at 37 °C for 1 h. After tagmentation, nuclei were pelleted at 600 × g for 3 min and resuspended in 150 μL DynaTag buffer (25 mM HEPES pH 7.5 (KOH), 110 mM KCl, 10 mM NaCl, 1 mM MgCl2). For single-cell DynaTag, the ICELL8 cx nanoliter dispensing system (TaKaRa) was used similarly as reported for CUT&Tag10. Nuclei were counted using trypan blue to 1.4 cells/20 nL in PBS and 1× Second Diluent (Takara Bio USA, Cat. Nuclei were stained with DAPI for visualisation, and 80 μL/sample was loaded to a source loading plate. Control wells with PBS (80 μL) were also included in the source plate. Nuclei were dispensed onto an ICELL8 350 v chip at 20 nL per well twice. After each dispense, chips were sealed, centrifuged at 3220 × g for 3 min and scanned using the automated microscopy image analysis software (CellSelect, Takara Bio USA). Additional single nuclei were manually selected using the manual triage function, and a filter file containing information of single-nuclei containing wells and control wells was generated. We obtained ~1500 single nuclei per chip. Then, 35 nL denaturation buffer (10 mM Tris pH 8.0, 50 mM NaCl, 0.2% SDS, 0.5 ng/mL proteinase K) was dispensed onto the single-nuclei containing wells, and chips were incubated in a thermocycler at 58 °C (80 °C lid) for 3 h followed by incubation overnight at room temperature. Next, chips were centrifuged at 3220 × g for 3 min and 35 nL quenching buffer (10 mM Tris pH 8.0, 0.1 mM phenylmethanesulfonyl fluoride (PMSF), 2.5% Tween 20) was dispensed. For indexing, 35 nL of 72 i5 and 72 i7 (set A and set B) unique indexes were dispensed on the single nuclei-containing wells. PCR products were collected by centrifugation at 3220 × g for 10 min at 4 °C using the single-cell collection kit (TaKaRa, 640212). Libraries were size-selected using SPRI beads. First, a ratio of 0.25× beads was used to remove larger fragments. Then, samples were transferred to a new tube, and a 1.3× ratio of beads was used. Beads were washed twice with 80% ethanol, let air-dry at RT for 5 min and incubated in elution buffer (10 mM Tris-Cl, pH 8.5) at RT for 10 min. All CUT&Tag experiments were performed in two to three independent replicates. For analysis, BCL files were demultiplexed into separate fastq files per cell using cellranger (v7.0.1). Fastq files were aligned to mm39 and mm10 using bowtie2 (v2.4.1) and aggregated into merged BAM files per TF using samtools (v1.13) merge for peak calling via MACS2 (v2.2.7.1). Using the aggregate called peaks, bedgraph files were generate per cell. Bedgraph files were converted into a large peak-by-cell count matrix and analysed in R using the Seurat (v 5.0.1) package. Using Seurat, the data was normalised like RNA-seq data and PCA followed by UMAP was run for dimensionality reduction. In order to correct for the variability of every peak signal across the single cells, we compute z-scores between the two cellular states ESC and EpiLC. Please note that the calculations below are, for sake of simplicity, only shown for one peak region. Let \({c}_{{ij}}^{{ESC}}\) be the \({n}^{{ESC}}\times {n}_{b}^{{ESC}}\) count matrix of the ESC cells and \({c}_{{ij}}^{{EpiLC}}\) the \({n}^{{EpiLC}}\times {n}_{b}^{{EpiLC}}\) count matrix of the EpiLC cells, we can then determine \({\bar{z}}_{{ESC}}\), \({\sigma }_{{ESC}}^{2}\) for ESC and \({\bar{z}}_{{EpiLC}}\), \({\sigma }_{{EpiLC}}^{2}\) for EpiLC by replacing the count matrix \({c}_{{ij}}\) with the respective cell-type-specific count matrix in the equations above. The z-score is then computed by: Peak area binning and z-score calculations Narrow peak files were filtered for peak scores (column 5) equal to or higher than 1000. For each peak, the centre is extended by ±2.5 kb (kilobases). The extended peak region is then divided into 100 bins of 50 bases. To obtain the corresponding read counts for each bin per cell, the aligned BAM file for each cell is used as input for BEDTools/2.31.0-GCC-12.3.0. For all cells of each TF (MYC, NANOG and OCT4) and developmental condition (ESC and EpiLC), a merged count matrix was created for the binned peaks. To calculate the z-score, cells with more than 250 reads were selected, and read counts in each cell were scaled to 10,000 using the scanpy.pp.normalize_total() function. For each peak (containing 100 count bins), the z-score, p value and adjusted p value were calculated. Peaks with an adjusted p value <0.001 were selected for downstream analysis and visualisation with UMAP. Python's Scanpy package (version 1.10.4) was used to analyse the single-cell data. For each cell, the number of peaks with non-zero counts (non-zero peaks) was calculated. Cells falling within the lower 15% and upper 5% quantiles of non-zero peaks were removed (quantile filtering). These plots show the distribution of log (x + 1) values for: the total number of peak reads per cell, the number of non-zero peaks per cell, before and after quantile filtering, and the non-zero peaks selected by z-score filtering. For data visualisation, the counts for all peaks were scaled to 10,000 using the scanpy.pp.normalize_total() function, and a log(x + 1) transformation was applied using scanpy.pp.log1p(). After z-score filtering, the dimensionality of the retained peaks was reduced to 10 principal components using scanpy.pp.pca(). A neighbourhood graph was then computed with 25 neighbours using scanpy.pp.neighbours(). Finally, two-dimensional UMAP embeddings were generated using the scanpy.tl.umap() function. After isolation, nuclei were counted with trypan blue and fixed using the Evercode Nuclei Fixation v2 Kit (Parse Bioscience, ECF2003) and stored at −80 °C until library preparation. Library preparation of fixed nuclei was performed with the Evercode WT Kit (Parse Bioscience, ECW02130) according to manufacturer's protocol. Sequencing fastq files were demultiplexed into single-nucleus data using the Split Pipe pipeline off Parse Bioscience (v1.0.6). Single-nucleus RNA-seq data were analysed in R using the Seurat (v 5.2.1) package. First, all four samples were pre-processed separately. Percentages of mitochondrial and ribosomal gene expression were computed for all cells. Cells with a ratio of mitochondrial versus endogenous genes expression higher than 0.1 were discarded. Additionally, cells expressing fewer than 1000 or more then 100,000 UMI counts, and fewer than 500 genes were also excluded. Raw expression data were normalised applying SCTransform function, regressing on percentage of mitochondrial gene expression. Principal component analysis (PCA) was performed using RunPCA function, and top 30 principal components were selected. All four samples were then integrated with batch correction by applying reciprocal PCA using the IntegrateLayers function. A k-nearest neighbours graph was computed and further refined into a shared nearest neighbour (SNN) graph using FindNeighbors function, using batch-corrected principal components as input. Cell clusters were defined at different resolutions applying Louvain algorithm to SNN graph using FindCluster function. To assess cluster stability, we calculated Jaccard indices for each cluster at each resolution by comparing the original clusters with those obtained from each subsample. A cluster was considered stable if its median Jaccard index across subsamplings was higher than 0.7553. We finally selected the resolution that produced the higher number and fraction of stable clusters. The top 30 cluster-specific genes were identified by first applying PrepSCTFindMarkers function, followed by FindMarkers function with the following parameters: pseudocount.use = 0.2, assay = ‘SCT', only.pos = TRUE, and min.pct = 0.2. Genes with an adjusted p value < 0.01 were selected and ranked by log fold change (logFC); the top 30 genes were retained. For cell visualisation in two dimensions uniform manifold approximation and projection (UMAP) was computed with RunUMAP function. To exclude non-malignant tumour microenvironment cells, the SeuratObject was subsetted to the small cell lang cancer clusters. For differential expression analysis, the count data were aggregated into pseudobulk counts per sample and a combined transcripts-by-sample count matrix was generated. The combined count matrix was filtered for transcripts of a rowSums <10. DESeq2 (v1.42.0) was run with standard parameters and the control samples as reference. Significantly differentially expressed genes were identified (log2FoldChange >0.5 or <−0.5 and padj <0.1). A slightly more relaxed adjusted p value of 0.1 was chosen as the single-nucleus RNA-seq was only performed in duplicates and therefore offers limited statistical power. GSEA was run using the local GSEA software (v4.3.3), the normalised enrichment score of significantly enriched pathways (NOM p value < 0.05) was plotted using ggplot2 (v.3.5.0). For estimation of changes in occupancy of TFs via DynaTag at genes associated with altered pathways via GSEA (v4.3.3), the chromosomal coordinates of the TSS (+250; −1000 bp) of each enriched gene per pathway were extracted. To estimate the statistical significance, a paired t test was performed per TF per pathway. The log2-transformed occupancy ratio was plotted as colour scale, and the −log10-transformed p value was plotted as size parameter in a factor-by-pathway bubble plot matrix. Of note, the p values were not FDR-corrected to account for multiple test problems as the gene sets did not contain excesses numbers of genes. Moreover, the genes with a DynaTag count of 0 in one of the treatment conditions were excluded to prevent division by 0. To visualise the DynaTag signal at the genomic coordinates used to calculate the occupancy ratio, deeptools (v3.5.1) computeMatrix (reference-point --beforeRegionStartLength 1000 --afterRegionStartLength 250 --referencePoint TSS --missingDataAsZero --skipZeros) followed by deeptools (v3.5.1) plotprofile (--perGroup --colours blue red --plotType=fill) was used. No statistical method was used to predetermine sample size. No data were excluded from the analyses. The Investigators were not blinded to allocation during experiments and outcome assessment. Measurements were taken from all samples. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The murine DynaTag, CUT&Tag and RNA-seq data are available at NCBI GEO under accession code GSE273872. Computation code is available under https://github.com/HaenselHertschEpiLab/DynaTag and within Zenodo54. Voss, T. C. & Hager, G. L. Dynamic regulation of transcriptional states by chromatin and transcription factors. Targeting transcription factors for cancer treatment. Sun, W. & Yang, J. Functional mechanisms for human tumor suppressors. Johnson, D. S., Mortazavi, A., Myers, R. M. & Wold, B. Genome-wide mapping of in vivo protein–DNA interactions. & Henikoff, S. An efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites. Zambanini, G., Nordin, A., Jonasson, M., Pagella, P. & Cantù, C. A new CUT&RUN low volume-urea (LoV-U) protocol optimized for transcriptional co-factors uncovers Wnt/β-catenin tissue-specific genomic targets. Womersley, H. J., Muliaditan, D., DasGupta, R. & Cheow, L. F. Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression. Kaya-Okur, H. S. et al. CUT&Tag for efficient epigenomic profiling of small samples and single cells. Bartosovic, M., Kabbe, M. & Castelo-Branco, G. Single-cell CUT&Tag profiles histone modifications and transcription factors in complex tissues. Chien, P. & Gierasch, L. M. Challenges and dreams: physics of weak interactions essential to life. Wang, Q. et al. CoBATCH for high-throughput single-cell epigenomic profiling. Bárcenas-Walls, J. R. et al. Nano-CUT&Tag for multimodal chromatin profiling at single-cell resolution. Harada, A. et al. A chromatin integration labelling method enables epigenomic profiling with lower input. Carter, B. et al. Mapping histone modifications in low cell number and single cells using antibody-guided chromatin tagmentation (ACT-seq). Hui, W. W. I., Simeone, A., Zyner, K. G., Tannahill, D. & Balasubramanian, S. Single-cell mapping of DNA G-quadruplex structures in human cancer cells. Evaluation of parameters critical for observing nucleic acids inside living Xenopus laevis oocytes by in-cell NMR spectroscopy. Corces, M. R. et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Zou, Z., Ohta, T. & Oki, S. ChIP-Atlas 3.0: a data-mining suite to explore chromosome architecture together with large-scale regulome data. Hayashi, K., Ohta, H., Kurimoto, K., Aramaki, S. & Saitou, M. Reconstitution of the mouse germ cell specification pathway in culture by pluripotent stem cells. LeBlanc, L. et al. Yap1 safeguards mouse embryonic stem cells from excessive apoptosis during differentiation. Evolutionary trajectories of small cell lung cancer under therapy. George, J. et al. Comprehensive genomic profiles of small cell lung cancer. Peifer, M. et al. Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. Dammert, M. A. et al. MYC paralog-dependent apoptotic priming orchestrates a spectrum of vulnerabilities in small cell lung cancer. Dooley, A. L. et al. Nuclear factor I/B is an oncogene in small cell lung cancer. Array comparative genomic hybridization-based characterization of genetic alterations in pulmonary neuroendocrine tumors. Rudin, C. M. et al. Molecular subtypes of small cell lung cancer: a synthesis of human and mouse model data. Herzog, B. H., Devarakonda, S. & Govindan, R. Overcoming chemotherapy resistance in SCLC. Ireland, A. S. et al. MYC drives temporal evolution of small cell lung cancer subtypes by reprogramming neuroendocrine fate. Stewart, C. A. et al. Single-cell analyses reveal increased intratumoral heterogeneity after the onset of therapy resistance in small-cell lung cancer. Nakazawa, S. et al. Dominant-negative p53 mutant R248Q increases the motile and invasive activities of oral squamous cell carcinoma cells. Dong, P. et al. Mutant p53 gain-of-function induces epithelial–mesenchymal transition through modulation of the miR-130b–ZEB1 axis. Ibusuki, R. et al. TP53 gain-of-function mutations promote osimertinib resistance via TNF-α–NF-κB signaling in EGFR-mutated lung cancer. Distinct mechanisms control genome recognition by p53 at its target genes linked to different cell fates. Deng, Y. et al. Spatial-CUT&Tag: spatially resolved chromatin modification profiling at the cellular level. Deng, Y. et al. Spatial profiling of chromatin accessibility in mouse and human tissues. Llorens-Bobadilla, E. et al. Solid-phase capture and profiling of open chromatin by spatial ATAC. Zhang, D. et al. Spatial epigenome–transcriptome co-profiling of mammalian tissues. Farzad, N. et al. Spatially resolved epigenome sequencing via Tn5 transposition and deterministic DNA barcoding in tissue. Guo, P. et al. Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues. Attardi, L. D. & Boutelle, A. M. Targeting p53 gain-of-function activity in cancer therapy: a cautionary tale. Chen, X. et al. Mutant p53 in cancer: from molecular mechanism to therapeutic modulation. Hayashi, K. & Saitou, M. Generation of eggs from mouse embryonic stem cells and induced pluripotent stem cells. Hodgkinson, C. L. et al. Tumorigenicity and genetic profiling of circulating tumor cells in small-cell lung cancer. Gillotin, S. Isolation of chromatin-bound proteins from subcellular fractions for biochemical analysis. Bentsen, M. et al. ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Evaluating single-cell cluster stability using the Jaccard similarity index. Hunold, P. et al. DynaTag for efficient mapping of transcription factors in low-input samples and at single-cell resolution. This work was founded by the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG)—SFB1399, project ID: 413326622— to R.H-H., R.K.T., M.P. Additional Funding was received through the Center of Molecular Medicine Cologne (R.H-H.), DFG SFB1399 (INST 216/1057-2), Fritz Thyssen Foundation (10.22.1.010MN), from the program “Netzwerke 2021”, an initiative of the Ministry of Culture and Science of the State of North-Rhine-Westphalia for the CANTAR project to R.H.H., R.K.T, J.G. and M.P., by the DFG (project ID: 497777992 to J.G. ), by the translational cancer program of the German Cancer Aid (Deutsche Krebshilfe, project TACTIC, grant number 70115201 to R.K.T. We are grateful to Dr. Simon Poepsel at the CMMC for sharing murine ESCs and Dr. Filippo Beleggia at the Department of Translational Genomics, University of Cologne, for sharing the antibody against NFIB. We thank Dr. Benjamin Czech Nicholson (Cancer Research UK Cambridge Institute) for critically reading the manuscript and providing valuable comments. Open Access funding enabled and organized by Projekt DEAL. Pascal Hunold, Giulia Pizzolato, Nadia Heramvand, Olivia van Ray, Martin Peifer & Robert Hänsel-Hertsch Pascal Hunold, Giulia Pizzolato, Nadia Heramvand, Laura Kaiser, Olivia van Ray, Roman Thomas, Julie George, Martin Peifer & Robert Hänsel-Hertsch Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar has received research support from Roche. The remaining authors declare no competing interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Hunold, P., Pizzolato, G., Heramvand, N. et al. DynaTag for efficient mapping of transcription factors in low-input samples and at single-cell resolution. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            