
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69442711/brains-sync-during-communication/'>Your Brain Is Naturally Telepathic, Research Suggests—Meaning Our Minds Are All Connected</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 17:13:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These moments often feel uncanny, like something more than a mere conversational coincidence. The “click” you experience with people isn't just your imagination, research suggests. The phenomenon is called neural coupling, which is when two or more people's brains “synchronize,” or simultaneously display the same activity patterns. Still, neural coupling isn't quite as powerful as the telepathy seen in movies—synchronizing with a friend doesn't mean you'll be able to transmit words directly into their mind. Uri Hasson, PhD, one of the first researchers to discover brain-to-brain coupling more than a decade ago, describes our brains as a sort of wireless transmitter. That may seem like science fiction at first, but Hasson says there's nothing mystical about the process. Hasson argues that his research shows communication is really “a single act performed by two brains.” He believes that all brains naturally couple with the outside world, reacting to whatever stimuli we're bombarded with. For example, if you show two monkeys a banana, their brains would likely react the same way, and the same goes for humans. Studies show that brain synchronization happens in many settings. For instance, researchers found neural coupling can occur during chess matches or collaborative music-making sessions—two activities that require focus and creativity. On the other hand, a 2014 study published in PLUS One found that synchronization can occur during a much more physical activity: kissing. But you don't necessarily need to pucker up or become a Grandmaster to optimize brain synchronization—some research suggests coupling could defy physical presence, too. For example, a 2022 study demonstrated that pairs working together on a video game showed increased synchronization, despite their only interaction being through on-screen movements. However, other unrelated experts who study neural coupling believe connectivity may be stronger face-to-face, as nonverbal actions like hand gestures aid in our communication, according to Hasson. Perhaps most surprisingly, synchronization isn't dependent on age or even intellect. One 2019 study published in the peer-reviewed journal Psychological Science found that infant and adult brains sync during play. The study analyzed the brain activity of adults as well as children from nine to 15 months in age During the experiment, an adult researcher interacted with each baby by playing with toys, singing nursery rhymes, and reading a book. According to the team, this may be subconsciously shaping our communication. “We were also surprised to find that the infant brain was often ‘leading' the adult brain by a few seconds, suggesting that babies do not just passively receive input but may guide adults toward the next thing they're going to focus on: which toy to pick up, which words to say,” Casey Lew-Williams, PhD, co-author study, says in a press release. The team found that students whose brainwaves were more in sync with their teacher and peers actually learned better, retaining more of the information they were taught. Dikker uses the analogy of walking next to someone extremely tall—you may have to take longer steps, or they may take shorter steps, but you're ultimately meeting at a pace somewhere in the middle. For example, if your brain can predict what the teacher is going to say next, you are ultimately learning better. She explains that conversations naturally have a rhythmic flow, so through this social coordination we can unlock “smooth interactions,” ultimately deepening our understanding of one another. Emma Frederickson graduated from Pace University where she studied communication and media. Prior to her time as an editor, she was a freelance science reporter. She enjoys covering everything from shipwrecks to pimple popping, but her favorite topics include climate change, conspiracy theories, and weird biology. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. Experts Found an Ancient Maya Map of the Universe</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69457568/how-wormholes-warp-your-reality/'>You're Living Alongside Invisible Wormholes, Scientists Say—and They May Be Warping Your Reality</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 14:49:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists in Greece have laid out a mathematical framework to help explain a so-far-unsolved gap in our understanding of how gravity scales up to the full size of our universe. Specifically, cosmologists can't yet account for how expansion of the universe increased in the “recent cosmological past”—a very relative term—because all attempts to quantify the change so far end up clashing with existing mathematical models. These spacetime Orbeez require only a bit of fine-tuning from theories that are already in the zeitgeist. An order of magnitude is not simply a 1:1 multiplier, so 120 orders of magnitude is likely far more than 120 times larger. (Thank you to several keen-eyed readers who wrote to explain that 120 orders of magnitude is many, many times larger than 120 times! With the real value, you are almost definitely getting struck by lightning.) If we want to balance the two possibilities, the scientists explain, we need to add at least one new complicating factor. The two main suggestions for this are, first, a changing cosmological constant explained by the idea of dark energy; or second, to deepen our theory of how gravity behaves on the cosmic scale, from a smooth background to a craggier, fjord-like complexity. This paper suggests combining both options: What if the solution is a believably craggier background gravity covered with instantons and wormholes, and their more rugged landscape naturally accounts for an amount of dark energy as well? When cosmologists try to model the universe, they break it down using practices like topology and manifold theory. They build in as many variables or ideas as they have at the time, leaving spaces where something is not yet known. And, just like in your high school calculus class, the goal is always to account for everything on both sides of any equal sign or other equivalence in your mathematical model. The physics of different scales simply don't match easily yet. Concrete observations by Albert Einstein and those before turned into beautiful theories that account for the visible world and parts of outer space; quantum mechanics twists those pieces until they no longer fit together into a completed puzzle. One way spacetime acts up is called quantum foam. If someone can be a writer's writer or a comedian's comedian, this is kind of the disruptive quantum mechanics of quantum mechanics. And it's in this foam where wormholes could exist in a way that brings cosmic ideas together without breaking any larger rules. As for where dark energy comes into this, it's still a part of our working model of the universe, wormholes or not. In this case, the wormholes would directly account for at least some of that dark energy: “[T]he wormhole density in a dynamical spacetime is not expected to be constant, therefore the obtained effective cosmological constant also acquires a dynamical nature, i.e., it corresponds to an effective dark energy sector.” And to account for the observed cosmological constant, all we need is “1016 microscopic wormholes,” or 10 quadrillion, “per cubic meter per second.” That, the scientists conclude, is well within the realm of possibility based on estimates. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. Experts Found an Ancient Maya Map of the Universe</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69457323/dna-last-neanderthal-sequenced/'>Scientists Sequenced the DNA of the ‘Last Neanderthal'—and It Alters Human History</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 14:15:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Discover new clues about how our ancient relatives disappeared from time. “There is nothing like looking, if you want to find something,” says Thorin Oakenshield in J.R.R. He and his team had been working the area since 1998 to find remnants of humanity's prehistoric forbearers, and after 17 years, they certainly found something: a piece of a jaw belonging to a Neanderthal. “I began to find {remnants of the Neanderthal's jaw} in 2015,” Slimak told the New Statesman in 2022, “but each year we find one tooth, or one fragment of bone.” Slimak determined that this particular Neanderthal lived 42,000 years ago, towards the end of that species' time on this planet. “Thorin the Neanderthal is also an end of lineage. An end of a way to be human.” To confirm his suspicions about Thorin's age and attempt to glean more information about not just when but how this particular specimen lived, Slimak and his colleagues had Thorin's genome analyzed. Nearly a decade before ever finding Thorin, Slimak had already theorized that any Neanderthals who had resided in the Rhône Valley would have been different from those in the surrounding areas. His assessment, at that point, was based on the stone tools found at various sites, noting that those in the Rhône Valley didn't reflect the newer tool-making style found at other locations. “It turns out that what I proposed 20 years ago was predictive,” Slimak told the publication Live Science. “Everything must be rewritten about the greatest extinction in humanity and our understanding of this incredible process that will lead Homo sapiens to remain the only survival of humanity,” Slimak said in assessing what this discovery means. “How can we imagine populations that lived for 50 millennia in isolation while they are only two weeks' walk from each other? Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. Experts Found an Ancient Maya Map of the Universe</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69383774/abandoned-city-under-city/'>Archaeologists Dug Under a City—and Found an Abandoned Second City Hidden Below the Surface</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Residents abandoned Nya Lödöse in 1624, moving to the more strategic location of Gothenburg, which has now grown to over one million residents. Gear-obsessed editors choose every product we review. We may earn commission if you buy from a link. Just under Gothenburg's industrial district of Gamlestaden, Archaeologists discovered remnants of the lost Late Middle Ages city of Nya Lödöse, a site that started forming in the late 1400s and was abandoned in 1624. Excavations, according to Arkeologerna, the State Historical Museums, revealed the southernmost part of Nya Lödöse, complete with an earthen rampart, a moat, a street with the foundation of homes dotted along its sides, and a fortified southern gate. Within a couple of decades, inhabitants again descended on Nya Lödöse, building up the fortifications of the city. Timber foundations still delineate the gate's site, and a nearby building found during excavations may have been a tollhouse for the gate. Nya Lödöse' short history ended by 1624, following the 1621 founding of Gothenburg nearby, a slightly more strategic location for trading and defense. Even though Sweden was in control of Gothenburg, according to the Museum of Gothenburg, the city was still a border town as the Danish king had control of counties nearby, so Gothenburg featured high earthen banks and a moat, attracting residents and becoming a new center for the Swedish west coast, presaging its modern status as one of Sweden's largest urban areas. Within three years, Nya Lödöse was fully abandoned and eventually faded into farmland, as the rampart and moat were gradually filled back in. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Tomb of Prehistoric King Dates Back 5,000 Years Advanced Seafaring Found in Ancient Southeast Asia Scientists Used 750-Year-Old DNA in a Cold Case Experts Found an Ancient Maya Map of the Universe</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a69394795/blue-goo-life/'>Scientists Found Deadly Blue Goo at the Bottom of the Ocean—Filled With Signs of Life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The goo, which is heavily alkaline serpentinite mud, contained lipid biomarkers, supporting the theory that early life began near ancient mud volcanoes We may earn commission if you buy from a link. But even these situations don't come close to qualifying as the most intense living conditions on Earth—the ones that incredible organisms known as extremophiles call home. The Aquifex genus of bacteria, for instance, thrive in the hot springs of Yellowstone, where temperatures can reach a balmy 205 degrees Fahrenheit. And the tiny multicellular parasite Henneguya salminicola eschews oxygen altogether. A new study, published in the journal Communications Earth & Environment, is adding more stunning evidence to the already impressive resume of extremophiles. Analyzing 5.4-foot-long core samples taken from one of the most intense environments on the planet—mud volcanoes near the Mariana Trench—scientists have found lipid biomarker evidence pointing to microbial life living within this serpentinite mud (which is created from a reaction that occurs when water interacts with igneous rock). At lower depths, this serpentinized rock, when oxidized, is a light green color. This blue mud is very hazardous to life, as it maintains an extremely alkaline pH of 12—similar to that of bleach—and is severely lacking in any nutrients. In fact, this goo is so inhospitable that finding biomarkers in the first place proved quite a challenge, as DNA can be too difficult to detect with such a low number of cells. “With the help of these biomarkers we were able to obtain insights into the survival strategies of methane- and sulfate-metabolizing microbes in this extreme environment.” These fats come from the cell membranes of bacteria and archaea, and their age can tell scientists if these microbial communities are currently flourishing in this environment or are fossilized evidence of past habitation. The study found that these particular organisms are indeed persisting below the seafloor, and are part of a larger community of sub-seafloor dwellers that, by some estimates, makes up an astonishing 15 percent of Earth's living biomass. But this new-blue-goo find is particularly appealing, as scientists have long identified ancient mud volcanoes as perfect habitats for the creation of early life on Earth. “What is fascinating about these findings is that life under these extreme conditions, such as high pH and low organic carbon concentrations is even possible,” Florence Schubotz, senior author of the study from University of Bremen, said in a press statement. “It is simply exciting to obtain insights into such a microbial habitat because we suspect that primordial life could have originated at precisely such sites.” Experts Found an Ancient Maya Map of the Universe</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/forbes-email-time-capsule-communicating-future/'>I Sent Hundreds of Thousands of E-mails to the Future. Here's the Story</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The Unlikely Story of an E-mail Time Machine This article is part of a package in collaboration with Forbes on time capsules, preserving information and communicating with the future. The world was different in 2005: flip phones were state of the art, Netflix only sent movies by mail, and kale was a garnish, not a main course. In those days, news websites were treated like awkward teenagers living in a basement apartment, kept at a distance from their respectable parents. In contrast, magazine sales teams seemed stuck in the Mad Men era, pitching tired concepts over martinis and treating online news like something you threw into a deal for free, like a branded pen. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Given this context, you can imagine my lack of enthusiasm when our sales department revealed that they had sold a big advertising sponsorship to an IT infrastructure company and asked me, the telecom reporter, if I could put together a special report on the concept of “Communicating.” Fortunately, my editor at the time, Michael Noer, was equally nonplussed at the idea of a dozen stories about routers and network switches. So we decided to approach the idea of “Communicating” from every single angle other than networking hardware. We commissioned Arthur C. Clarke to write a keynote essay about how technology was actually making it harder for people to communicate with each other. I wrote about the search for extraterrestrial intelligence and asked experts how they would craft their own messages to an alien species. And I interviewed two dozen luminaries I'd always wanted to meet about anything I could think of: Noam Chomsky on the spontaneous invention of language; Jane Goodall on how language makes it harder for primates to communicate; Stan Lee on why words are better with pictures. This was our thinking: Time capsules are boring. Your typical time capsule contains items that are either redundant (“Oh, great, a newspaper from 50 years ago, just like I could find in the library or on the Internet”) or meaningless (“I'm sure this toy meant something to the kid who put it in here, but now it's just trash”). We built a tool for our website that allowed users to write themselves a message and choose whether they wanted to receive it in one, three, five, 10 or 20 years. Then we threw it open to the public. Hundreds of thousands of messages queued up—which meant we had to figure out how to actually store and send all those e-mails, and that's harder than you might think. Today they might as well be cuneiform on clay tablets. How could we be sure we had a good way not only to preserve our e-mails but also to make sure they were sent on schedule? Our solution was, if you ask me, both clever and elegant. We'd build a program that could run on three computers that were owned by three different companies and lived in different places on the Internet. But if instance B and C didn't hear from that machine, instance B would send the e-mails, and so on. (As the current editor in chief of one of those zombie publications, I would like to note here that Scientific American just celebrated its 180th anniversary, and Forbes turned 108). So we decided to hedge our bets by storing the data at three very different institutions: first, at Forbes, the longtime lion of capitalism; second, at the Internet giant Yahoo, which had then become a $55-billion company after several years of fast growth; and finally, at Codefix Consulting, a one-man consultancy run by a friend of mine from college, Garrison Hoffman. We were hedging our bets: Legacy media, blazing-hot dot-com, plucky small business—we figured that between the three of those, we'd have all contingencies covered. Yahoo had layoffs within the first few months, and our clever little solution never sent a ping. Forbes was sold to an investment group that included the rock star Bono, and our plucky online newsroom got absorbed by the print team. I was a freelance journalist, working on a book, and the e-mail time capsule was the furthest thing from my mind. But the project survived because somebody cared: a few months before the time capsule was supposed to “reopen,” Garrison reached out to me. Codefix Consulting sent out all the e-mails again. Our best-laid plans had gone awry, but this new order seemed to work just as well. Garrison and I put reminders in our now totally digital calendars, scheduled to alert us when another decade passed. Just under two years later, Garrison Hoffman died, unexpectedly, at the age of 46. Searching my e-mail inbox in a mild panic, trying to figure out what to do, I found a message: This script should work with any sendmail compatible mail utility, and may also be configured as a backup server.” Garrison knew and followed the best practices of his trade. He'd documented his work, annotated his code and archived the files on a server where I could find them, just in case. And as you read this, nearly 18,000 people have received an e-mail from a different kind of old friend—their own past self. A one-man shop, and one man's dedication to his trade, proved the most reliable of all. Ironically, the original Forbes.com special report on Communicating is now a wreck—images gone, links broken. But the e-mails survived, and opening one is profound. Whether the people who sent them saw their 2005 dreams come true or not, they've had a rare chance to compare past hopes to present reality. My own e-mail to the future is actually kind of prosaic. “Somehow, some way, you got this thing to work properly,” the message continues. “Go track down Michael Noer, who owes you a magnum of expensive champagne for pulling this off.” That's why this time machine worked: not because of servers but because of humans, because of people who cared about each other and about something they had worked together to build. David M. Ewalt is editor in chief of Scientific American. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41565-025-02060-6'>Nanoengineered aqueous-hydrotrope hybrid liquid electrolyte solutions for efficient zinc batteries across a wide temperature range</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 11:33:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Nanotechnology (2025)Cite this article Aqueous zinc metal batteries are ideal candidates for grid storage applications. However, their practical application is hindered by a narrow operating temperature range and a limited electrolyte electrochemical stability window, both of which can be attributed to the water activity. Here, to minimize water activity in the electrolyte solution, we introduce a nanoengineered approach in which the water molecules are confined within a hydrophilic–hydrophobic water solvation sheath. The hydrogen-bond interaction with the hydrophilic groups in the inner solvation layer effectively suppresses water decomposition, and the hydrophobic solvents in the outer solvation layer establish a repulsive effect against water molecules. As a proof of concept, a hydrophobic and non-polar hydrofluoroether cosolvent is introduced into a Zn-ion aqueous electrolyte solution and tested together with various fluorinated hydrotrope molecules to favour the compatibility of the cosolvent with water. By such a water confinement strategy, an average Zn plating/stripping reversibility of 99.92% is achieved for over 4,000 cycles at 2.0 mA cm−2 and 2.0 mAh cm−2 in a Zn||Cu coin cell configuration. When tested in a Zn||VOPO4·2H2O lab-scale cell configuration, the selected aqueous-hydrotrope hybrid electrolyte solution enables long-lasting and highly reversible battery performance across temperatures from −80 °C to +60 °C. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data supporting the findings of this study are available in the article and its Supplementary Information. Source data are provided with this paper. Mathlouthi, M. Water content, water activity, water structure and the stability of foodstuffs. Food Control 12, 409–417 (2001). Google Scholar Sheng, D. et al. Hydrogen bond network regulation in electrolyte structure for Zn-based aqueous batteries. Google Scholar Wang, Y. et al. Enabling high-energy-density aqueous batteries with hydrogen bond-anchored electrolytes. Google Scholar Roy, K., Rana, A., Heil, J. N., Tackett, B. M. & Dick, J. E. For zinc metal batteries, how many electrons go to hydrogen evolution? An electrochemical mass spectrometry study. Google Scholar Emamian, S., Lu, T., Kruse, H. & Emamian, H. Exploring nature and predicting strength of hydrogen bonds: a correlation analysis between atoms-in-molecules descriptors, binding energies, and energy components of symmetry-adapted perturbation theory. Google Scholar Li, M. et al. Comprehensive understandings of hydrogen bond chemistry in aqueous batteries. Google Scholar Wang, F. et al. Highly reversible zinc metal anode for aqueous batteries. Google Scholar Lim, J. et al. Nanometric water channels in water-in-salt lithium ion battery electrolyte. Google Scholar Zhang, M. et al. Understanding the microscopic structure of a “water-in-salt” lithium ion battery electrolyte probed with ultrafast IR spectroscopy. Google Scholar Hao, J. et al. Boosting zinc electrode reversibility in aqueous electrolytes by using low-cost antisolvents. Google Scholar Wang, W. et al. Regulating interfacial reaction through electrolyte chemistry enables gradient interphase for low-temperature zinc metal batteries. Google Scholar Ming, F. et al. Co-solvent electrolyte engineering for stable anode-free zinc metal batteries. Google Scholar Robertson, A. E. et al. Mesoscale solubilization and critical phenomena in binary and quasi-binary solutions of hydrotropes. Fluid Phase Equilib. Google Scholar Kunz, W., Holmberg, K. & Zemb, T. Hydrotropes. Colloid Interface Sci. Google Scholar Dukhin, A. & Pavlenishvilli, D. “Water-in-salt” super concentrated electrolyte clusters are “micelles”. Colloids Surf. Google Scholar Miao, L. et al. Aqueous electrolytes with hydrophobic organic cosolvents for stabilizing zinc metal anodes. ACS Nano 16, 9667–9678 (2022). Google Scholar Dong, Y. et al. Non-concentrated aqueous electrolytes with organic solvent additives for stable zinc batteries. Google Scholar Bauduin, P., Renoncourt, A., Kopf, A., Touraud, D. & Kunz, W. Unified concept of solubilization in water by hydrotropes and cosolvents. Google Scholar Dong, D., Wang, T., Sun, Y., Fan, J. & Lu, Y.-C. Hydrotropic solubilization of zinc acetates for sustainable aqueous battery electrolytes. Google Scholar Zhao, Z. et al. A novel “water-in-ionic liquid” electrolyte for Zn metal batteries. ACS Energy Lett. Google Scholar Amann-Winkel, K. et al. X-ray and neutron scattering of water. Google Scholar Zheng, J. et al. Understanding thermodynamic and kinetic contributions in expanding the stability window of aqueous electrolytes. Google Scholar Zhang, C. et al. The electrolyte comprising more robust water and superhalides transforms Zn-metal anode reversibly and dendrite-free. Google Scholar Yang, W., Yang, Y., Yang, H. & Zhou, H. Regulating water activity for rechargeable zinc-ion batteries: progress and perspective. ACS Energy Lett. Google Scholar Miyazaki, K. et al. First-principles study on the peculiar water environment in a hydrate-melt electrolyte. Google Scholar Subramanian, D., Boughter, C. T., Klauda, J. B., Hammouda, B. & Anisimov, M. A. Mesoscale inhomogeneities in aqueous solutions of small amphiphilic molecules. Faraday Discuss. Article PubMed Google Scholar Huang, Z. et al. Anion chemistry in energy storage devices. Article PubMed Google Scholar Zhang, Y. et al. Suppressed dissolution of fluorine-rich SEI enables highly reversible zinc metal anodes for stable aqueous zinc-ion batteries. Google Scholar et al. Inhibiting VOPO4⋅xH2O decomposition and dissolution in rechargeable aqueous zinc batteries to promote voltage and capacity stabilities. Google Scholar Liu, S. et al. From room temperature to harsh temperature applications: fundamentals and perspectives on electrolytes in zinc metal batteries. Google Scholar Borodin, O. et al. Liquid structure with nano-heterogeneity promotes cationic transport in concentrated electrolytes. ACS Nano 11, 10462–10471 (2017). Google Scholar Cao, L. et al. Fluorinated interphase enables reversible aqueous zinc battery chemistries. Google Scholar Yang, C. et al. All-temperature zinc batteries with high-entropy aqueous electrolyte. Google Scholar Cong, J. et al. Kinetics compensation mechanism in cosolvent electrolyte strategy for aqueous zinc batteries. Google Scholar Wang, Y. et al. Sulfolane-containing aqueous electrolyte solutions for producing efficient ampere-hour-level zinc metal battery pouch cells. Google Scholar Liang, G. et al. Regulating inorganic and organic components to build amorphous-ZnFx enriched solid–electrolyte interphase for highly reversible Zn metal chemistry. Google Scholar Chang, N. et al. An aqueous hybrid electrolyte for low-temperature zinc-based energy storage devices. Google Scholar Wang, S. et al. Fast reaction kinetics and commendable low-temperature adaptability of zinc batteries enabled by aprotic water–acetamide symbiotic solvation sheath. Google Scholar Qiu, Y. et al. Möbius solvation structure for zinc-ion batteries. Article PubMed Google Scholar Cao, X. et al. Weak solvation effect induced optimal interfacial chemistry enables highly durable Zn anodes for aqueous Zn-ion batteries. Shi, J. et al. “Water-in-deep eutectic solvent” electrolytes for high-performance aqueous Zn-ion batteries. Xie, D. et al. ZnF2-riched inorganic/organic hybrid SEI: in situ-chemical construction and performance-improving mechanism for aqueous zinc-ion batteries. Sun, P. et al. Simultaneous regulation on solvation shell and electrode interface for dendrite-free Zn ion batteries achieved by a low-cost glucose additive. Jin, Y. et al. Stabilizing zinc anode reactions by polyethylene oxide polymer in mild aqueous electrolytes. Chen, Y. et al. Low current-density stable zinc-metal batteries via aqueous/organic hybrid electrolyte. Wang, D. et al. Solvation modulation enhances anion-derived solid electrolyte interphase for deep cycling of aqueous zinc metal batteries. Zhou, A. et al. Molecular recognition effect enabled by novel crown ether as macrocyclic host towards highly reversible Zn anode. Wan, F. et al. Reversible oxygen redox chemistry in aqueous zinc-ion batteries. Terban, M. W. & Billinge, S. J. Structural analysis of molecular materials using the pair distribution function. Egami, T. & Billinge, S. J. Underneath the Bragg Peaks: Structural Analysis of Complex Materials 16 (Elsevier, 2003). Gaussian 16 Rev. Lu, T. & Chen, F. Multiwfn: a multifunctional wavefunction analyzer. Article PubMed Boys, S. F. & Bernardi, F. The calculation of small molecular interactions by the differences of separate total energies. Some procedures with reduced errors. Pan, J., Zhang, Q., Xiao, X., Cheng, Y.-T. & Qi, Y. Design of nanostructured heterogeneous solid ionic coatings through a multiscale defect model. ACS Appl. Plimpton, S. Fast parallel algorithms for short-range molecular dynamics. Martínez, L., Andrade, R., Birgin, E. G. & Martínez, J. M. PACKMOL: a package for building initial configurations for molecular dynamics simulations. Article PubMed Mark, P. & Nilsson, L. Structure and dynamics of the TIP3P, SPC, and SPC/E water models at 298 K. J. Phys. Zhu, X., Lopes, P. E. & MacKerell, A. D. Jr Recent developments and applications of the CHARMM force fields. Wiley Interdiscip. Zielkiewicz, J. Structural properties of water: comparison of the SPC, SPCE, TIP4P, and TIP5P models of water. Article PubMed Obst, S. & Bradaczek, H. Molecular dynamics simulations of zinc ions in water using CHARMM. Patil, N. et al. An ultrahigh performance zinc-organic battery using poly (catechol) cathode in Zn (TFSI)2-based concentrated aqueous electrolytes. Canongia Lopes, J. N. & Pádua, A. A. CL&P: a generic and systematic force field for ionic liquids modeling. Schauperl, M. et al. Non-bonded force field model with advanced restrained electrostatic potential charges (RESP2). Humphrey, W., Dalke, A. & Schulten, K. VMD: visual molecular dynamics. Momma, K. & Izumi, F. VESTA 3 for three-dimensional visualization of crystal, volumetric and morphology data. Kresse, G. & Furthmüller, J. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Kresse, G. & Hafner, J. Ab initio molecular dynamics for liquid metals. Kresse, G. & Hafner, J. Ab initio molecular-dynamics simulation of the liquid-metal–amorphous-semiconductor transition in germanium. We thank the grant from the Research Grants Council of the Hong Kong Special Administrative Region, China (Project Number CityU C1002-21G; C.Z.). We also acknowledge the Shanghai Synchrotron Radiation Facility (SSRF) and European Synchrotron Radiation Facility (ESRF) for the provision of synchrotron radiation facilities. We thank the Momentum Transfer team for facilitating the measurements and J. Drnec for assistance and support in using beamline ID31. The measurement set-up was developed with funding from the European Union's Horizon 2020 Research and Innovation programme under the STREAMLINE project (grant agreement ID870313; W.H.K.). We acknowledge the support from Shanghai Pilot Program for Basic Research (W.L. ), Xiaomi Young Talents Program (W.L. ), and the National Natural Science Foundation of China (W2432001; W.H.K. These authors contributed equally: Xueying Zheng, Haotian Zhu, Zhongqiang Wang, Hua Yang. Department of Materials Science and Engineering, City University of Hong Kong, Hong Kong, China Xueying Zheng, Huilin Cui & Chunyi Zhi State Key Laboratory of Silicon and Advanced Semiconductor Materials, School of Materials Science and Engineering, Zhejiang University, Hangzhou, China Haotian Zhu, Ruhong Li, Haikuo Zhang, Jinze Wang & Xiulin Fan Institute of New Energy for Vehicles, School of Materials Science and Engineering, Tongji University, Shanghai, China Zhongqiang Wang, Wei Luo & Yiming Dai Spallation Neutron Source Science Center, Dongguan, China Hua Yang & Wang Hay Kan Institute of High Energy Physics, Chinese Academy of Sciences (CAS), Beijing, China Hua Yang & Wang Hay Kan Centre for Energy Storage, The University of Hong Kong, Hong Kong, China Chunyi Zhi Department of Mechanical Engineering, The University of Hong Kong, Hong Kong, China Chunyi Zhi State Key Laboratory of Material Processing and Die and Mould Technology, School of Materials Science and Engineering, Huazhong University of Science and Technology, Wuhan, China Yunhui Huang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived of the idea and designed the experiments. H. Zhu, R.L., H. Zhang and J.W. performed the theoretical simulations. and H.C. carried out the characterizations and electrochemical measurements. performed the synchrotron-based characterizations and analysis. drafted the paper with input from all co-authors. contributed to the scientific discussion and data interpretation. All authors have read and approved the final version of the paper. Correspondence to Wei Luo, Wang Hay Kan, Xiulin Fan, Chunyi Zhi or Yunhui Huang. The authors declare no competing interests. Nature Nanotechnology thanks Kyeounghak Kim, Dipan Kundu and Nian Liu for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. 1–77, Tables 1–6, Notes 1–47 and References. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Statistical source data for Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zheng, X., Zhu, H., Wang, Z. et al. Nanoengineered aqueous-hydrotrope hybrid liquid electrolyte solutions for efficient zinc batteries across a wide temperature range. Download citation Received: 14 December 2024 Accepted: 09 October 2025 Published: 17 November 2025 Version of record: 17 November 2025 DOI: https://doi.org/10.1038/s41565-025-02060-6 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Nanotechnology © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41590-025-02321-5'>Thrombospondin-1–CD47 signaling contributes to the development of T cell exhaustion in cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 11:00:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. T cell exhaustion is a major barrier to effective cancer immunotherapy. Although immune checkpoint blockade can reinvigorate exhausted T cells, not all patients achieve long-term responses, partly due to the refractory nature of terminally exhausted T cells. Beyond persistent antigen stimulation, the environmental drivers of exhaustion remain to be thoroughly characterized. Here we identify CD47 upregulation in tumor-infiltrating exhausted CD8+ T cells in both human and murine tumors. We reveal a novel role for the extracellular matrix protein thrombospondin-1 (TSP-1) in engaging CD47 on T cells to promote exhaustion. This interaction activates calcineurin–NFAT signaling, inducing upregulation of TOX and expression of inhibitory receptors, and impairing effector function during tumor progression. Importantly, disrupting the TSP-1–CD47 axis prevents T cell exhaustion and enhances tumor control. Our findings identify a novel pathway promoting T cell dysfunction and suggest that targeting the TSP-1–CD47 axis is a promising strategy to enhance T cell immunity and immunotherapy efficacy. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Prices may be subject to local taxes which are calculated during checkout All data generated and supporting the findings of this study are available within the Article. The scRNA-seq data are deposited in the Gene Expression Omnibus database (GSE303291 and GSE303017). Additional information and materials will be made available upon request. Source data are provided with this paper. & Kurachi, M. Molecular and cellular insights into T cell exhaustion. Thommen, D. S. & Schumacher, T. N. T cell dysfunction in cancer. Alfei, F. et al. TOX reinforces the phenotype and longevity of exhausted T cells in chronic viral infection. Khan, O. et al. TOX transcriptionally and epigenetically programs CD8+ T cell exhaustion. Scott, A. C. et al. TOX is a critical regulator of tumour-specific T cell differentiation. Seo, H. et al. TOX and TOX2 transcription factors cooperate with NR4A transcription factors to impose CD8+ T cell exhaustion. Yao, C. et al. Single-cell RNA-seq reveals TOX as a key regulator of CD8+ T cell persistence in chronic infection. McLane, L. M., Abdel-Hakeem, M. S. & Wherry, E. J. CD8 T cell exhaustion during chronic viral infection and cancer. Schietinger, A. et al. Tumor-specific T cell dysfunction is a dynamic antigen-driven differentiation program initiated early during tumorigenesis. Philip, M. et al. Chromatin states define tumour-specific T cell dysfunction and reprogramming. Lindberg, F. P. et al. Rh-related antigen CD47 is the signal-transducer integrin-associated protein. The CD47-signal regulatory protein alpha (SIRPa) interaction is a therapeutic target for human solid tumors. McCracken, M. N., Cha, A. C. & Weissman, I. L. Molecular pathways: activating T cells after cancer cell phagocytosis from blockade of CD47 “don't eat me” signals. Jaiswal, S. et al. CD47 is upregulated on circulating hematopoietic stem cells and leukemia cells to avoid phagocytosis. Brown, E. Integrin-associated protein (CD47): an unusual activator of G protein signaling. Reinhold, M. I., Lindberg, F. P., Kersh, G. J., Allen, P. M. & Brown, E. J. Costimulation of T cell activation by integrin-associated protein (CD47) is an adhesion-dependent, CD28-independent signaling pathway. Ticchioni, M. et al. Integrin-associated protein (CD47) is a comitogenic molecule on CD3-activated human T cells. Waclavicek, M. et al. T cell stimulation via CD47: agonistic and antagonistic effects of CD47 monoclonal antibody 1/1A4. Li, Z., He, L., Wilson, K. & Roberts, D. Thrombospondin-1 inhibits TCR-mediated T lymphocyte early activation. Li, Z. et al. Interactions of thrombospondins with α4β1 integrin and CD47 differentially modulate T cell behavior. Nath, P. R. et al. Loss of CD47 alters CD8+ T cell activation in vitro and immunodynamics in mice. Targeting the CD47/thrombospondin-1 signaling axis regulates immune cell bioenergetics in the tumor microenvironment to potentiate antitumor immune response. Paffenholz, S. V. et al. Senescence induction dictates response to chemo- and immunotherapy in preclinical models of ovarian cancer. Chen, Z. et al. TCF-1-centered transcriptional network drives an effector versus exhausted CD8 T cell-fate decision. Defining T cell states associated with response to checkpoint immunotherapy in melanoma. Caushi, J. X. et al. Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers. Beltra, J. C. et al. Developmental relationships of four exhausted CD8+ T cell subsets reveals underlying transcriptional and epigenetic landscape control mechanisms. Beckett, A. N. et al. CD47 expression is critical for CAR T-cell survival in vivo. Komori, S. et al. CD47 promotes peripheral T cell survival by preventing dendritic cell-mediated T cell necroptosis. Blazar, B. R. et al. CD47 (integrin-associated protein) engagement of dendritic cell and macrophage counterreceptors is required to prevent the clearance of donor lymphohematopoietic cells. Andrews, L. P. et al. LAG-3 and PD-1 synergize on CD8+ T cells to drive T cell exhaustion and hinder autocrine IFN-γ-dependent anti-tumor immunity. Bissinger, R. et al. Thrombospondin-1/CD47 signaling modulates transmembrane cation conductance, survival, and deformability of human red blood cells. Diercks, B. P. et al. ORAI1, STIM1/2, and RYR1 shape subsecond Ca2+ microdomains upon T cell activation. Lopez-Dee, Z., Pidcock, K. & Gutierrez, L. S. Thrombospondin-1: multiple paths to inflammation. Misenheimer, T. M. & Mosher, D. F. Calcium ion binding to thrombospondin 1. Lawler, J., Weinstein, R. & Hynes, R. O. Cell attachment to thrombospondin: the role of ARG-GLY-ASP, calcium, and integrin receptors. Budhu, S. et al. Targeting phosphatidylserine enhances the anti-tumor response to tumor-directed radiation therapy in a preclinical model of melanoma. Fischer, K. et al. Antigen recognition induces phosphatidylserine exposure on the cell surface of human CD8+ T cells. Pettersen, R. D., Hestdal, K., Olafsen, M. K., Lie, S. O. & Lindberg, F. P. CD47 signals T cell death. Identification of TAX2 peptide as a new unpredicted anti-cancer agent. Kaur, S. & Roberts, D. D. Emerging functions of thrombospondin-1 in immunity. Balint, S. et al. Supramolecular attack particles are autonomous killing entities released from cytotoxic T cells. Martinez-Torres, A. C. et al. CD47 agonist peptides induce programmed cell death in refractory chronic lymphocytic leukemia B cells via PLCγ1 activation: evidence from mice and humans. Tharmalingam, S. & Hampson, D. R. The calcium-sensing receptor and integrins in cellular differentiation and migration. Bahar, E., Kim, H. & Yoon, H. ER Stress-mediated signaling: action potential and Ca2+ as key players. Soto-Heredero, G., Desdin-Mico, G. & Mittelbrunn, M. Mitochondrial dysfunction defines T cell exhaustion. Scharping, N. E. et al. Mitochondrial stress induced by continuous stimulation under hypoxia rapidly drives T cell exhaustion. Rudloff, M. W. et al. Hallmarks of CD8+ T cell dysfunction are established within hours of tumor antigen encounter before cell division. Budhu, S. et al. CD8+ T cell concentration determines their efficiency in killing cognate antigen-expressing syngeneic mammalian cells in vitro and in mouse tissues. Hao, Y. et al. Seurat. Blondel, V. D., Guillaume, J.-L., Lambiotte, R. & Lefebvre, E. Fast unfolding of communities in large networks. & Fortunato, S. Community detection algorithms: a comparative analysis. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. We thank R. Ahmed for his valuable insights on this work. We also thank the Flow Cytometry Core Facilities at MSK Cancer Center and WCM for technical assistance. This research was funded in part through the NIH/NCI Cancer Center Support grant no. P30CA008748, the Swim Across America, Ludwig Institute for Cancer Research, Ludwig Center for Cancer Immunotherapy at MSK, Cancer Research Institute and the Parker Institute for Cancer Immunotherapy. was supported in part through a Postdoctoral Fellowship from the Parker Institute for Cancer Immunotherapy and an NIH grant (no. were supported in part through a grant from the Department of Defense (no. ; A.A. was supported in part through grants from the Breast Cancer Research Foundation (nos. was supported in part through a grant from the Department of Defense (no. was supported in part through a grant from the Nuovo Soldati Foundation and the Grand Hôpital de Charleroi. Present address: InduPro, Seattle, WA, USA Swim Across America and Ludwig Collaborative Laboratory, Department of Pharmacology, Weill Cornell Medicine, New York, NY, USA Chien-Huan Weng, Anais Assouvie, Lauren Dong, Sadna Budhu, Levi Mangarin, Yacine Marouf, Lucia Morgado-Palacin, Cailian Liu, Jonathan F. Khan, Isabell Schulze, Linda Hamadene, Daniel Hirschhorn, Jedd D. Wolchok & Taha Merghoub Sandra and Edward Meyer Cancer Center, Weill Cornell Medicine, New York, NY, USA Chien-Huan Weng, Anais Assouvie, Lauren Dong, Sadna Budhu, Levi Mangarin, Yacine Marouf, Lucia Morgado-Palacin, Cailian Liu, Jonathan F. Khan, Isabell Schulze, Linda Hamadene, Daniel Hirschhorn, Roberta Zappasodi, Niroshana Anandasabapathy, Jedd D. Wolchok & Taha Merghoub Parker Institute for Cancer Immunotherapy, San Francisco, CA, USA Chien-Huan Weng, Anais Assouvie, Lauren Dong, Jean-Christophe Beltra, Sadna Budhu, Levi Mangarin, Yacine Marouf, Lucia Morgado-Palacin, Cailian Liu, Jonathan F. Khan, Isabell Schulze, Linda Hamadene, Daniel Hirschhorn, Roberta Zappasodi, Niroshana Anandasabapathy, E. John Wherry, Jedd D. Wolchok & Taha Merghoub Department of Biomedicine, University of Basel, Basel, Switzerland Laboratory of Comparative Pathology, Memorial Sloan Kettering Cancer Center, The Rockefeller University, Weill Cornell Medicine, New York, NY, USA Memorial Sloan–Kettering Cancer Center, New York, NY, USA Dmitriy Zamarin, Fadi Samaan, Stephane Pourpe & David Schröder Icahn School of Medicine at Mount Sinai, New York, NY, USA Department of Medicine at Weill Cornell Medicine, New York, NY, USA Roberta Zappasodi & Jedd D. Wolchok Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School, New York, NY, USA Roberta Zappasodi & Jedd D. Wolchok Surface Oncology Inc., Cambridge, MA, USA Department of Dermatology, Weill Cornell Medicine, New York, NY, USA Department of Microbiology and Immunology, Weill Cornell Medicine, New York, NY, USA Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Institute for Immunology and Immune Health, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the study with T.M. designed, performed and analyzed experiments, and wrote the manuscript. A.A. performed western blots and conducted in vivo animal experiments together with C.-H.W. to assess the therapeutic efficacy of TAX2 peptide, in combinant with PD-1 blockade therapy in murine B16 melanoma and MC38 tumor models. performed western blots and conducted all the tumor killing experiments. contributed to scientific discussion and re-analysis of his data generated at the laboratory of E.J.W. designed and established the 2D tumor killing assay for the laboratory and designed the protocol to stain externalized phosphatidylserine (PS) in CD8+ TILs. performed bioinformatics data analysis and generated the plot results. L.M.-P. contributed to scientific editing and adjustment of the manuscript and associated materials during the revision. performed in vivo tumor implantation and drug treatment and contributed to tissue processing for flow cytometry analysis. performed and analyzed all histology work. contributed with scientific suggestions and offered human specimens derived from patients with ovarian cancer and MPB1 ovarian tumor cells. designed and performed the quantitative RT–PCR/mRNA expression profiling of in vitro experiments using Fluidigm. D.H. contributed to experimental design and scientific discussion. helped with the design and experimental setup of measuring Ca2+ influx in CD8+ T cells. helped with tumor measurements of MPB1 ovarian cancers using bioluminescence imaging. helped with human PBMCs from healthy donors and establishing the in vitro protocol for human PBMC stimulation. assisted with some of the experiments involving tumor implantation and measurements. optimized the staining of CD8+ TILs derived from patients with melanoma. offered scientific insight and her observation that tumor-infiltrating regulatory T cells express high levels of CD47. contributed to the early phase of study design and data interpretation. contributed to scientific discussions during the revision. conceptualized the study along with C.-H.W., and provided scientific supervision, secured funding and wrote the manuscript. Correspondence to Jedd D. Wolchok or Taha Merghoub. declare no competing interests related to this work. received royalties from Agenus for work on LAG-3 and TIM-3. D.H. is an inventor on patent applications related to work on OX40 and GITR. is an inventor on patent applications related to work on GITR, PD-1 and CTLA4. is consultant for Leap Therapeutics, Daiichi Sankyo and IFLI, serves as an SAB member for iTeos Therapeutics and receives grant support from Bristol Myers Squibb and Astrazeneca. was employed by Surface Oncology and is currently employed by InduPro. is a member of the Parker Institute for Cancer Immunotherapy. is an advisor for Absci, Arpelos Biosciences, Arsenal Biosciences, Coherus, Danger Bio, IpiNovyx, New Limit, Marengo, Pluto Immunotherapeutics, Related Sciences, Santa Ana Bio and Synthekine. is a founder of Arpelos Biosciences, Arsenal Biosciences and Danger Bio, and holds stock in Coherus. is a consultant for Shennon Biosciences, Panther Life Science, Verrica, Janssen, Immunitas, 23&me, Cellino, Kumquat and Genmab, and serves as an SAB member for Network Bio. J.D.W is a consultant for Ankyra Therapeutics, Apricity, Arsenal Biosciences, Ascentage Pharma, Bicara Therapeutics, Bristol Myers Squibb, Daiichi Sankyo, Imvaq, Takeda, Tizona, Trishula Therapeutics, Immunocore (Data Safety Board) and Scancell. received grant/research support from Bristol Myers Squibb. has equity in Apricity, Arsenal IO/CellCarta, Ascentage, Imvaq, Linneaus, Georgiamune, Maverick/Takeda, Tizona Therapeutics and Xenimmune. is an inventor on the following patents: Xenogeneic DNA vaccines, Newcastle disease viruses for cancer therapy, Myeloid-derived suppressor cell (MDSC) assay, Prediction of responsiveness to treatment with immunomodulatory therapeutics and method of monitoring abscopal effects during such treatment, Anti-PD-1 antibody, Anti-CTLA4 antibodies, Anti-GITR antibodies and methods of use thereof. acted in the capacity of consultant for Immunos Therapeutics, Daiichi Sankyo Co, TigaTx, Normunity and Pfizer. is a cofounder of and equity holder in IMVAQ Therapeutics. has received research support from Surface Oncology, Kyn Therapeutics, Infinity Pharmaceuticals, Peregrine Pharmaceuticals, Adaptive Biotechnologies, Leap Therapeutics, Aprea Therapeutics, Enterome SA and Realta Life Sciences, and currently receives research funding from Bristol Myers Squibb. is an inventor on patent applications related to work on oncolytic viral therapy, alpha virus–based vaccine, neoantigen modeling, immunomodulatory nanoparticles, bi-specific activators, FLT3L, CD40, IL-10, IL10R, GITR, OX40, PD-1, CTLA4 and chimeric receptors targeting melanoma differentiation antigens, B7-H3 and MUC-16, and listed as an inventor on a US Provisional Patent Application (Ser. 63/888,188) related to work on CD47 and TSP-1. Nature Immunology thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Laurie A Dempsey, in collaboration with the Nature Immunology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Representative plot of CD47 and TOX in CD8+ TILs isolated from single cell suspension of lesions of ovarian patients (n = 13). b, Quantifications of TOX, PD-1, and TIM-3 expression (mean fluorescent intensity, MFI) in CD47-Hi (red) and CD47-Lo (blue) tumor-infiltrating CD8 + T cells derived from lesions of 13 ovarian cancer patients (n = 13). Data shown are mean ± S.E.M. with exact P-values (two-sided paired t-test). c, Representative plot of CD47 and PD-1 in CD8+ TILs isolated from murine MPB1 ovarian tumors (n = 11). d, Quantifications of TOX, PD-1, and TIM-3 expression (mean fluorescent intensity, MFI) in CD47-Hi (red) and CD47-Lo (blue) tumor-infiltrating CD8 + T cells derived from 11 murine MPB1 tumors (n = 11). Data shown are mean ± S.E.M. with exact P-values (two-sided paired t-test). a. UMAP (uniform manifold approximation and projection) plots of single cell transcriptomic profiles of live-sorted CD8+ T cells derived from established B16 melanoma (day18 post tumor implantation). b, UMAP plots of single cell transcriptomic profiles of live-sorted CD8 + T cells (as in a) denoting sub-clusters defined based on the gene signatures of each sub-cluster. c, Heat map of scRNA-seq gene expression [row normalized log2 (counts per million) for DEGs; false discovery rate (FDR) < 0.10] among cells in these 5 sub-clusters shown in b. The 5 sub-clusters are: CD8+ Naïve T cells; Il7r-high memory precursor effector cells (MPECs); Tcf7-high precursor of exhausted T cells (pre-Tex); Klrg1-high effector T cells (Teff); Tox-high exhausted T cells (Tex). Leiden clustering resulted in 5 sub-clusters of CD8 + T cells. We labeled these sub-clusters based on their uniquely expressed genes, respectively: naïve CD8+ T cells, exhausted T cells, precursor exhausted CD8 + T cells and definitions that classify the memory and short-lived effector CD8+ T cells24,27,49. d, Bar plots of Cd47 and genes associated with T cell exhaustion among these 5 sub-clusters as defined in (c). Exact P-values are provided (two-sided unpaired t-test), as compared to cells in CD8+ Naïve T cell sub-cluster. UMAPs of the scRNA-seq dataset of tumor CD8+ T cells encompassing PRDM1hi MANA-specific CD8+ T cells derived from patients of non-small cell lung cancer (NSCLC) who received the first-in-human clinical trial of neoadjuvant anti-PD-1 (nivolumab; Clinical-Trials.gov identifier: NCT02259621). a, Representative plot of CD47, CD8a, TOX, PD-1, TIM-3, and LAG-3 levels in CD8+ T cells freshly isolated from spleens, tumor-draining lymph nodes (TdLNs) and tumors of established B16 melanoma from 5 mice (n = 5, biological replicates). b, Percentages (%) of CD47high, CD47 + TOX + , CD47 + PD-1 + , CD47 + TIM-3 + , or CD47 + LAG-3 + CD8 + T cells from (a). Data shown are representative of >2 independent experiments and are mean ± S.E.M. with exact P-values; Two-sided paired t-tests. c, Representative plots (left) of TNF and IFNγ of CD47-Hi (Red) and CD47-Lo (blue) CD8+ TILs isolated from 5 tumors. Frequencies (right) of TNF + IFNγ+ in CD47-Hi and CD47-Lo CD8+ TILs are presented on the right. Data shown are representative of > 2 independent experiments and are mean ± SEM with exact P-value; Two-sided paired t-tests. a, Representative plot of CD47 and congenic marker Thy1.2 levels in naïve-sorted CD8+ T cells from both Cd47+ /+ and Cd47+ /- Pmel-1 CD8+ T cell populations. Surface levels of CD47 (quantified by mean-fluorescent intensity, MFI) is shown below each plot. b, Representative plots of CD47 and PD-1 in transferred CD8+ T cells isolated from murine B16 melanoma (day 14 post transfer). c, Representative histogram of CD47 expression on tumor Pmel-1 Cd47+ /+ and Pmel-1 Cd47+ /- CD8+ TILs d, Quantification of surface CD47 levels (MFI) in transferred CD8+ T cells isolated from 5 tumors (n = 5). Data shown are mean ± S.E.M. with exact P-value (two-sided paired t-tests) and are representative of > 2 independent experiments. a, Accelerated depigmentation in TCR-transgenic Pmel-1 mice with Cd47+/-  genotype compared to age/sex-matched Pmel-1 Cd47+ /+ littermates (n = 3 per genotype). b, Representative histology of Pmel-1 Cd47+ /- mouse skin shows infiltration of CD8 + T cells (black arrows) in and around the inferior segment of anagen hair follicles containing melanocytes (white arrowheads), which express the model antigen-mouse homolog-pmel-17 of human gp100 recognized by Pmel-1 CD8+ T cells. Scale bars (100 μm) shown below serve as references. Data presented are representative slides of > 2 independent experiments. a-b, Relative protein levels of nuclear and cytoplasmic NFAT1 (top panel) and NFAT2 (lower panel) (a), JUNB and cJUN (b) were shown by normalizing to the band density of β-actin (loading control). c, Representative Western blot of nuclear and cytoplasmic protein extracts from in vitro activated CD8+ T cells that were conditioned for 4 h in the absence or presence of recombinant mouse TSP-1 (50 µg/ml, final concentration) were assessed for NFAT1 and NFAT2. β-actin serve as loading controls; Surface proteins CD47 and ORAI1 serve as additional controls to make sure that addition of TSP-1 does not change their expression. The relative density of each quantification has been normalized to the untreated condition (that is, without addition of TSP-1 and Synta66, which is an ORAI1 inhibitor) from either the nuclear or cytoplasmic portion of the blots. Data shown are representatives of > 3 independent experiments. a, Representative plots (left) showing TOX and PD-1 expression in OT-1 Cd47+ /+ and OT-1 Cd47-/- CD8+ T cells that were in vitro stimulated for 5 days with OVA (SIINFEKL) peptide. Recombinant mouse TSP-1 (10 µg/ml) was added to the culture as indicated. Quantifications (% and MFI) of CD8+ T cells that express PD-1 and either TOX or TIM-3 are shown in the right panel. Data are mean ± S.D. with exact P-values of 3 biological replicates of column-isolated naïve CD8+ T cells from spleens of 3 OT-1 Cd47+ /+ mice and 3 OT-1 Cd47-/- mice (n = 3), (two-sided unpaired t-tests). b, Representative plots of IFNγ and TNF in CD8+ T cells from (a) upon restimulation (left). Quantifications (% or MFI) of cells that produce TNF and IFNγ (right). Data are mean ± S.D. with exact P-values of 3 experimental replicates (n = 3), (two-sided unpaired t-tests). Data presented are representatives of > 2 independent experiments. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Weng, CH., Assouvie, A., Dong, L. et al. Thrombospondin-1–CD47 signaling contributes to the development of T cell exhaustion in cancer. Version of record: 17 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02100-3'>Distinct transcriptomic and epigenomic responses of mature oligodendrocytes during disease progression in a mouse model of multiple sclerosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 10:58:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Multiple sclerosis (MS) is a chronic autoimmune disease that targets mature oligodendrocytes (MOLs) and their myelin. MOLs are heterogeneous and can transition to immune-like states in MS. However, the dynamics of this process remain unclear. Here, we used single-cell multiome assay for transposase-accessible chromatin and RNA sequencing targeting oligodendroglia (OLG) from the experimental autoimmune encephalomyelitis (EAE) MS mouse model at multiple disease stages. We found that immune OLG states appear at early disease stages and persist to late stages, which can be consistent with epigenetic memory of previous neuroinflammation. Transcription factor activity suggested immunosuppression in OLG at early disease stages. Different MOLs exhibit differential responsiveness to EAE, with MOL2 exhibiting a stronger transcriptional immune response than MOL5/MOL6, and showed divergent responses at the epigenetic level during disease evolution. Our single-cell multiomic resource highlights dynamic and subtype-specific responses of OLG to EAE, which might be amenable to modulation in MS. Multiple sclerosis (MS) is an inflammatory autoimmune disease of the central nervous system (CNS)1. Oligodendrocytes (OLs) are the myelinating cells of the CNS, and their precursor cells (OL precursor cells (OPCs)) are present throughout the developing and adult CNS and are capable of differentiating into myelinating mature OLs (MOLs)2. MS has been generally viewed as primarily driven by T cells and B cells3. However, recent studies revealed the expression of immunomodulatory molecules in oligodendroglia (OLG) not only in MS but also in Alzheimer's disease and aging4,5,6,7,8. Single-cell/nucleus RNA sequencing (scRNA-seq) has been applied to reveal specific MOLs/OPC subpopulations in MS and experimental autoimmune encephalomyelitis (EAE)4,9,10. Assay for transposase-accessible chromatin using sequencing (ATAC-seq) is a high-throughput sequencing technique for assessing genome-wide chromatin accessibility11 and provides information on regions with open chromatin, which is required for gene expression12. By applying single-cell ATAC-seq (scATAC-seq) in combination with scRNA-seq independently, we previously found that a cohort of immune genes exhibit open chromatin in both control animals and in animals with EAE at peak disease, while their expression only increases in the context of EAE13. These studies were conducted with samples from single time points, at the peak of the disease in mouse EAE and from the late stage of disease in human postmortem MS tissue. Thus, the dynamics of OLG throughout the disease process have not been explored. Here, we investigate the epigenomic and transcriptional dynamics of OPCs and MOLs over the course of EAE. We applied simultaneous single-cell multiome ATAC-seq and RNA-seq to OLG sorted by fluorescence-activated cell sorting (FACS) from male and female mice with EAE at three distinct time points: early, peak and late stages. At the early stage of EAE, a subset of genes involved in antigen presentation showed increased expression and chromatin accessibility in OPCs and MOLs, indicating that the induction of an immune-like state in OLG occurs before the formation of fully developed lesions. Moreover, chromatin accessibility at these genes remained highly open at the late stage of EAE, indicating either partial maintenance or an epigenetic memory of this immune-like state. Furthermore, specific MOL subtypes acquire different patterns of change in genes related to the immune response at both expression and chromatin accessibility levels over the course of disease. We observed that white matter-enriched14 MOL2 showed higher immune signatures than MOL5/MOL6, which in turn exhibited induction of chromatin accessibility in genes with regenerative pathways, particularly in the late stage. Our study provides a resource available for browsing at the University of California, Santa Cruz, Cell Browser and Genome Browser15 (https://olg-dyn-eae-multiome.cells.ucsc.edu) and a deeper understanding of OLG dynamics in the inflammatory demyelination mouse model of MS, offering insights into these cell populations as potential targets for immune modulation and myelin regeneration. We induced EAE in Sox10:cre-RCE:loxP (enhanced green fluorescent protein (eGFP)) transgenic mice16,17 with injection of emulsion containing the MOG35–55 immunogenic peptide in complete Freund's adjuvant (CFA), followed by intraperitoneal injection of pertussis toxin. Spinal cord tissues from male and female mice induced with EAE were collected at three different stages: (1) early stage (days 8–9 after injection), (2) peak stage (days 14–15) and (3) late/chronic stage (days 37–40; Fig. Spinal cord samples from CFA-treated control (CFA-Ctrl) mice were also collected from the same stages, alongside spinal cord tissues from noninduced naive untreated control (Naive-Ctrl) mice. OLG were enriched based on eGFP by sorting by FACS (Extended Data Fig. 1a), after which we performed single-cell multiome RNA-seq and ATAC-seq (Fig. After sample-specific quality control filtering (Extended Data Fig. Louvain clustering was performed on the scRNA-seq (Extended Data Fig. 1c,e) and scATAC-seq datasets (Extended Data Fig. Neighbors' graphs from both modalities were overlapped to generate a new joined projection (Fig. 1g,h), annotated by cell type with marker gene expression and chromatin accessibility (Fig. As expected from the lineage tracing strategy, most of the cells were MOLs, OPCs and committed OL precursors. Nevertheless, other populations, in particular astrocytes and microglia, were also captured (Fig. a, Schematic of the methodology used in animal model establishment and multiome sequencing (image created using BioRender); blue dots, Naive-Ctrl; green dots, CFA-Ctrl; yellow dots, EAE early stage; red dots, EAE peak stage; brown dots, EAE late stage. b, Clinical scores of the mice used in the study (EAE: 38 mice in 19 multiome experiments, CFA-Ctrl n = 9; data are shown as mean ± s.d.). For EAE from peak and late stages, only mice that had reached a score of 3 were used in this study. c,d, Uniform manifold approximation and projection (UMAP) of cells profiled with simultaneous scRNA-seq (c) and scATAC-seq (d). Cell types are identified according to marker genes; Ast, astrocytes; Epen, ependymal cells; MG, microglia; COP, committed OL precursor. e, Joint UMAP from the weighted nearest neighbors graph of scRNA-seq and scATAC-seq modalities colored by cell type. f, Joint UMAP with cells colored by conditions on top of a UMAP with all cells (in gray). g, Normalized chromatin accessibility (left) and log2 expression (right) of representative marker genes of each cell type. We observed that a subset of MOLs transitioned to transcriptional/epigenomic states distinct from CFA-Ctrl at early stages of EAE, when lesions are just starting to develop (Fig. Moreover, a minority of MOLs retained peak-stage-like profiles at late stages, while most MOLs transitioned to CFA-Ctrl-like states, suggesting a return to homeostasis (Fig. Because disease-associated OLG states at peak EAE are characterized by chromatin accessibility and the expression of immune genes4,13, we investigated whether the observed transitions were driven by their immune status. Immune OLG (imOLG) were hardly observed in CFA-Ctrl mice, but were found in mice with EAE, with a higher percentage present at the peak stage than in the early and late stages (early stage 26.80%, peak stage 66.91% and late stage 32.41%; Fig. Chromatin accessibility at the promoter/gene body of the same immune genes identified a lower number of imOLG than by immune gene expression (early 14.69%, peak stage 37.71% and late stage 17.81%; Fig. This is most likely due to some OLG from CFA-Ctrl and Naive-Ctrl animals exhibiting already primed chromatin accessibility in immune gene loci but with low or no expression4,13, which made the cutoff value of immune status depicted from gene chromatin accessibility higher than from expression. Thus, our data indicate that the transition of both OPCs and MOLs to immune-like states at epigenomic and transcriptional levels occurs at early stages of EAE, when lesions are just starting to develop, and persists at late stages, despite resolving inflammation. a,b, Joint UMAP with OLG from CFA-Ctrl mice and mice with EAE. Projection of cells with immune status (red) and nonimmune status (green) identified by gene expression (a) and chromatin accessibility (b). c,d, Circos plots showing the number of cells with (red) or without (green) immune status identified by gene expression (c) and chromatin accessibility (d; downsampled by time point). e, Heat maps of the expression (top) and chromatin accessibility (CA; bottom) of MHC class I (top) and MHC class II (bottom) genes at different stages. f, Schematic of the 1×/2× IFNγ treatment experiment on mouse primary OPCs. Arrows represent the time points of sample collection (image created using BioRender). g,h, Heat maps showing normalized gene expression (top) and chromatin accessibility signal in 1-kb windows around the TSS (bottom; g) at defined enhancer regions (Methods) in the same 1-kb windows (bottom; h) in mouse primary OPCs with the first and second doses of IFNγ. Genes plotted were upregulated following IFNγ treatment compared to control treatment (P < 0.05 and false discovery rate (FDR) < 0.05) and showed no changes in chromatin accessibility signal after the first treatment with IFNγ in the absence of this cytokine for 96 h (P > 0.05 and FDR > 0.05). Genes that were significantly upregulated following the second dose of IFNγ compared to the first dose of IFNγ are marked with an asterisk (*; P < 0.05 and FDR < 0.05), while the remaining genes presented P < 0.05 in this comparison. i, RNAscope ISH from early-, peak- and late-stage EAE mice marked with probes for Sox10 (OLG) and H2-Ab1 (MHC class II). Statistical analyses were performed using a two-way ANOVA with a Tukey's multiple comparisons test and adjusted P values; NS, P > 0.05; ****P < 0.0001; data are shown as mean ± s.d. The expression and chromatin accessibility of some major histocompatibility complex (MHC) class I- and MHC class II-related genes were previously shown to be increased in EAE-specific OLG at peak stages of EAE4,13. Here, we found that, overall, the expression level of MHC class II genes was lower than that of MHC class I genes (Extended Data Fig. Notably, the expression of β-chain of MHC class I molecules (B2m), histocompatibility 2, K region locus 1 (H2-K1) and NLR family CARD domain-containing 5 (Nlrc5) was increased in EAE OLG at early stages (Fig. We identified MOL1, MOL2 and MOL5/MOL6 mature OLG subpopulations (Fig. We observed that the increase in expression of MHC class I and MHC class II genes at early stages was more expressive in OPCs than in MOL2 and MOL5/MOL6 (Extended Data Fig. The chromatin of some of the MHC class I genomic loci was accessible in OLG from CFA-Ctrl animals and exhibited a further increase in the early stages of EAE (Fig. By contrast, MHC class II genes had no or low expression in OLG from CFA-Ctrl mice and showed increased expression and chromatin accessibility in OPCs at early stages of EAE, but only at the peak stage for MOLs (Fig. A few MHC class I genes such as B2m, histocompatibility 2, D region locus 1 (H2-D1) and H2-K1 remained highly expressed at late EAE stages, while the expression of most other MHC class I and MHC class II genes was notably downregulated compared to at the peak stage. Nevertheless, chromatin accessibility at the promoter/gene body of both MHC class I and MHC class II genes remained high at the late stage (Fig. Thus, OLG exhibit persistent chromatin accessibility of specific immune genes at late stages of EAE. Because inflammation is decreased at these stages, this chromatin accessibility persistence in OLG might result from either epigenetic memory of peak immune-like states or from a mild inflammatory environment at late stages. a,b, Joint UMAP of OLG populations colored by cell subtypes (a) and time points (b). c, Bar plot of cell proportions from different time points in each OLG cell subtype. d, Circos plot showing the number of cells from different time points in OLG cell types (downsampled by time point). e, Circos plot showing the number of cells with (red) or without (green) immune status for each OLG cell type. f, Scaled expression level of immune response-related genes grouped by function in OLG cell subtypes. g, Scaled expression level of damage-associated response-related genes in OLG cell subtypes. h,i, Circos plots showing the number of cells with (black) or without (green) damage- and IFN-associated profiles, only IFN-associated profiles (blue) or only damage-associated profiles (gray) for OLG from different time points at gene expression (h) and chromatin accessibility (i) levels (downsampled by time point). To investigate whether a possible immune epigenetic memory in OPCs enhances immune characteristics after re-exposure to inflammatory stimuli, we treated primary mouse neonatal OPCs and the mouse OLG precursor cell line Oli-neu18 with IFNγ, either once or twice with a 96-h interval without IFNγ between treatments (Fig. RNA-seq and ATAC-seq were conducted 24 h after the first or second IFNγ treatment. We observed that there was a large number of genes for which chromatin accessibility around their transcription start sites (TSSs; Supplementary Table 1) and their potential enhancers (Supplementary Table 1) was maintained or kept at a relatively higher level, in the absence of IFNγ for 96 h, after treatment with this cytokine. Many of these genes, such as H2-Ab1, H2-K1 and H2-D1, retained chromatin accessibility in OPC populations but also in MOL populations in late-stage EAE (Fig. 2b), suggesting epigenetic memory in this MS model. Moreover, some of these genes also exhibited a further increase in chromatin accessibility after the second IFNγ treatment (Fig. Indeed, a specific subset of the genes presenting unaltered chromatin accessibility around the TSS and/or enhancers exhibited elevated expression in OPCs after exposure to the second dose of IFNγ compared to OPCs after one dose of IFNγ treatment (Fig. Moreover, in the Oli-neu cell line, quantitative PCR with reverse transcription (RT–qPCR) analysis revealed increased expression of specific genes within the MHC pathway, such as Nlcr5, H2-D1, H2-Ab1, H2-Aa, B2m and class II MHC transactivator (Ciita), when comparing the two IFNγ treatments to a single IFNγ treatment (Extended Data Fig. These results from cultured neonatal OPCs suggest that OPCs might retain, at the level of chromatin accessibility, epigenetic memory of a previous neuroinflammatory insult, which might make them more prone to transcriptionally reactivate an immunological profile. However, additional studies are needed to determine whether a similar epigenetic memory phenomenon occurs in adult OPCs and MOLs, given their distinct transcriptomic and epigenomic profiles. Our multiome data suggested that imOLG are present not only at the peak stage but also at the early and late stages of EAE. We thus used RNAscope in situ hybridization (ISH) to understand where these imOLG were located relative to EAE lesions. We defined lesions as white matter regions with a high number of cells due to inflammatory infiltrates (Extended Data Fig. Consistent with the multiome results, the MHC class II+ OLG (Sox10+H2-Ab1+) were observed in EAE spinal cord sections from the peak stage but also in early and late stages, albeit in different proportions (Fig. 2i,j and quantification in the Methods). There was no significant difference in the percentages of imOLG between lesion and nonlesion areas for all three stages (Fig. Consistent with previous observations4, we confirmed the presence of imOLG at the protein level by immunostaining against MHC class II (I-A/I-E), along with SOX10 (GFP; Extended Data Fig. Together, our data indicate that MHC class I and MHC class II genes maintain latent transcriptional and epigenetic states in OLG at late EAE stages, which may result in immune gene expression when facing recurrent inflammatory stimuli and contribute to the chronic persistent disease state. MOLs have recently been shown to be heterogeneous, with specific populations exhibiting regional preferences and different susceptibility to spinal cord injury4,7,13,14,19,20. Based on distinct gene expression profiles within OLG, we could further subdivide OPCs into three (OPC-α–OPC-γ), MOL2 into five (MOL2-α–MOL2-ε) and MOL5/MOL6 into ten cell states/subpopulations (MOL5/MOL6-α–MOL5/MOL6-κ; Fig. Interestingly, most of these states display prevalence toward specific EAE time points (Fig. 3a–c) and present specific expression of gene modules (Methods, Extended Data Fig. Overall, we observed a higher proportion of MOL2 at the peak and late stages of EAE than CFA-Ctrl and early-stage EAE (Fig. This increase could arise from conversion of MOL5/MOL6 into MOL2, being more resilient to the neuroinflammatory environment, and/or preferential differentiation of OPCs into MOL2 in the context of EAE. We then investigated whether any of these MOL populations were more prone to transition into immune-like states in EAE. In total, 38.35% of MOL2 were identified as imOLG, whereas only 18.75% of MOL5/MOL6 underwent this transition (Fig. 4d), indicating that MOL2 has higher immune responsiveness. Within the identified immune-related genes, some were more enriched in specific MOL populations, whereas others, involved, for instance, in cytokine and T cell immune responses, showed similar expression levels among all MOLs (Fig. Overall, our results suggest that MOL2 exhibit a stronger immune response to the neuroinflammatory environment than MOL5/MOL6 and may play a more important role in disease progress. In addition to the immune response, the disease-associated profile of OLG can also include a damage-associated response, which is related to apoptosis and survival mechanisms8,21. We found that the majority of OLG at the peak stage expressed high levels of both damage- and IFN-associated genes, whereas a subset of cells from the early and late stages expressed either damage- or IFN-associated profiles (Fig. By contrast, chromatin accessibility showed a stronger specificity in either damage- or IFN-associated genes (Fig. 3i), suggesting that the chromatin accessibility of these IFN- or damage-associated genes was already reduced at the peak stage, despite their transcription. A higher percentage of MOL2 displayed both damage- and IFN-associated profiles than MOL5/MOL6 at peak and late stages (Extended Data Fig. Additionally, we identified a subset of OPCs at the late stage that did not express either damage- or IFN-associated profiles (Fig. 3f,g), suggesting that these OPCs may have a nondisease phenotype and could potentially contribute to remyelination at the chronic stage of disease. Similar to immune-related genes, fewer damage- and IFN-associated OLG were identified by chromatin accessibility than gene expression (Fig. To explore whether other biological processes were dynamically modulated in OLG during the course of EAE, we performed differential gene expression analysis across the different stages in different OLG cell types (Methods). The differentially expressed genes were divided into Type 1 (high expression in CFA-Ctrl), Type 2 (high expression at the early stage), Type 3 (high expression at the peak stage) and Type IV (high expression at the late stage) main types and also subtypes (high expression in a main type associated with high expression in a secondary type; Fig. a,b, Heat maps of differentially expressed genes (DESeq2 Wald test and Benjamini–Hochberg multiple testing correction with a log2 (fold change) (log2 (FC)) of >1 and adjusted P value of <0.01; brown) between different time points in MOL5/MOL6 (a) and MOL2 (b), chromatin accessibility at promoters and gene bodies (purple) and chromatin accessibility at enhancer regions (green) of the same gene. The line plots represent the mean of the normalized and scaled gene expression (line in brown), chromatin accessibility at the promoter and gene bodies (line in purple) and chromatin accessibility at enhancer regions (line in green) of genes in different groups. Differentially expressed genes and associated GOs are shown in Supplementary Table 3 and 4. c, Normalized chromatin accessibility (left of each gene panel) and log2 expression (right of each gene panel) of genes related to the cholesterol biosynthetic process (Scd1, Idi1, Dhcr24 and Fdft1) in MOL5/MOL6 (left) and MOL2 (right). In OPCs, we observed an increase in IFN and cytokine signaling at early and peak stages (secondary subtype division Type 2–Type 3), and, as expected, genes related to antigen processing cross-presentation were increased in expression at the peak stage (Type 3), followed by a notable reduction at the late stage (Extended Data Fig. Furthermore, genes related to neuronal system and development, albeit not significant in Gene Ontology (GO) terms, such as doublecortin (Dcx) and glutamate receptor-interacting protein 1 (Grip1), were upregulated at the late stage (Type 4) in OPCs (Extended Data Fig. OPCs showed a drastic reduction of immune-associated cells at the gene expression level and, to some extent, at the chromatin level at the late stage of EAE compared to all other time points and MOL populations (Extended Data Fig. However, many of these genes retained chromatin accessibility, suggesting that they retain the epigenetic potential to initiate their transcription. Consistent with OPCs, the differentially expressed genes of MOL5/MOL6 were also divided into four distinct types and subtypes (Fig. Most of the genes with high expression at the peak stage (Type 3) in MOL5/MOL6 were immune related, such as antigen processing/cross-presentation (Psmb8, Psmb9 and Tap1), but their expression decreased drastically at the late stage (Fig. We observed an increase of interleukin signaling genes in MOL5/MOL6 (Type 2) at the early stage (Fig. We found an increase in cholesterol biosynthetic process-related genes at early stages (Type 2 subtype 1), such as stearoyl-CoA desaturase-1 (Scd1), isopentenyl-diphosphate-δ isomerase 1 (Idi1), 24-dehydrocholesterol reductase (Dhcr24), farnesyl-diphosphate farnesyltransferase 1 (Fdft1), squalene epoxidase (Sqle), farnesyl-diphosphate synthase (Fdps) and methylsterol monooxygenase 1 (Msmo1) in MOL5/MOL6. The formation of the myelin sheath necessitates highly coordinated levels of fatty acid and lipid synthesis process-related genes22. The increase in expression of cholesterol biosynthesis-related genes at the initial stage could suggest a form of preliminary remyelination adaptation to the very first cues given by EAE-driven demyelination (Extended Data Fig. At the late stage (Type 4), we observed the increased expression of genes in MOL5/MOL6 involved in extracellular matrix degradation and collagen chain trimerization, suggesting simultaneous matrix degradation and synthesis for tissue repair during chronic phases of the disease. We also observed increased expression of genes involved in the RHO GTPase cycle in MOL5/MOL6 at a secondary subtype division at late and early stages (Type 4 subtype 2), such as STEAP3 metalloreductase (Steap3) and copine 8 (Cpne8; Supplementary Tables 3 and 4) involved in actin cytoskeleton and microtubule processes, which play a role in myelination27, further suggesting that MOL5/MOL6 might activate gene regulatory programs associated with regeneration during these stages. We also identified four groups and subgroups of genes with differential expression between different time points for MOL2 (Fig. Similar to MOL5/MOL6, we found that cholesterol biosynthetic process-related genes also had a transitory increase in MOL2 at early-stage EAE compared to later stages (Fig. Many genes with high expression in MOL2 at the peak stage were immune-related genes (Type 3, such as Tap1, histocompatibility 2, T region locus 23 (H2-T23), integrin subunit-α 9 (Itga9) and Psmb8, which is consistent with the exacerbated immune response and higher number of imOLG at the peak stage (Fig. Interestingly, we also found that genes related to development and axon guidance, different from Type 1 genes, exhibited high expression in MOL2 at the late stage, such as plexin A4 (Plxna4), but also semaphorin 3B (Sema3b), EPH receptor (Ephb2 and Epha10) and ankyrin 1 (Ank1; Fig. In summary, our multiome data indicate that MOL2 initiate, as MOL5/MOL6, a cholesterol biosynthesis program in response to the arising neuroinflammatory environment but transition to an immune-like state during the course of EAE. We then explored expression differences between MOL2 and MOL5/MOL6 at each time point (Methods and Supplementary Table 5). At the peak stage of EAE, we identified 352 genes with increased expression in MOL2 compared to MOL5/MOL6 (Extended Data Fig. This finding supports that MOL2 exhibit a stronger immune profile than MOL5/MOL6. For MOL5/MOL6, we observed 80 genes with increased expression across different time points (Extended Data Fig. Among these, we identified several genes related to the neuronal system but also cell development, such as meis homeobox 2 (Meis2), neuroligin 3 (Nlgn3) and Sema6a, at early stages (Supplementary Tables 5 and 6). Differences between MOL2 and MOL5/MOL6 at the transcriptional level might correlate with changes at the epigenetic level. We thus compared the list of genes with differential chromatin accessibility at promoter/gene body regions and genes with differential expression across different stages and found that multiple genes showed notable changes in both expression and chromatin accessibility (Fig. This indicated that immune-related genes tend to have important changes in both gene expression and chromatin accessibility at promoter/gene body regions in all OLG populations. a, Venn diagram showing the number of genes with differential expression and/or differential chromatin accessibility among different disease stages (DESeq2 Wald test and Benjamini–Hochberg multiple testing correction with a log2 (FC) of >1 and adjusted P value of <0.01) in MOL2 (top) and MOL5/MOL6 (bottom). b, Top GO pathways by biological process terms (Fisher exact test and multiple testing correction using FDR with an adjusted P value of <0.05) for genes with both differential expression and differential chromatin accessibility in MOL2 (top) and MOL5/MOL6 (bottom) among different disease stages. c,d, Heat maps of differential chromatin accessibility at promoters and gene bodies (DESeq2 Wald test and Benjamini–Hochberg multiple testing correction with a log2 (FC) of >1 and adjusted P value of <0.01; purple) between different time points in MOL2 (c) and MOL5/MOL6 (d), gene expression (brown) and chromatin accessibility at enhancer regions (green) of the same gene. The line plots represent the average gene expression (line in brown), chromatin accessibility at promoters and gene bodies (line in purple) and chromatin accessibility at enhancer regions (line in green) of genes in different groups. Differential chromatin accessibility at promoters/gene bodies and associated GOs are shown in Supplementary Tables 3 and 4. e,f, Normalized chromatin accessibility of immune system process-related genes (Ido1, C1ra and Isg20) in MOL2 (e) and MOL5/MOL6 (f) at each time point. We then explored the dynamics of chromatin accessibility both at promoters/gene bodies and at associated distant enhancer regulatory regions during the disease course. Gene loci displaying differential chromatin accessibility at promoter/gene body regions between different time points were divided into high chromatin accessibility in CFA-Ctrl (Type 1), early stage (Type 2), peak stage (Type 3) and late stage (Type 4; Fig. In both OPCs and MOLs, most genes with differential chromatin and enhancer accessibility reached their highest levels of accessibility during the peak stage, with the association of immune-related functions (Supplementary Tables 3 and 4). Of note, a predominance in accessibility at genes associated with collagen biosynthesis and extracellular matrix organization was described in MOL5/MOL6 compared to MOL2 at late stages of EAE (Fig. When comparing MOL2 to MOL5/MOL6, we found that there was a group of antigen processing/presentation with high chromatin accessibility at the peak stage in both MOL2 and MOL5/6 (Extended Data Fig. Moreover, we found that more immune-related genes were upregulated in MOL2 than in MOL5/MOL6 at the peak stage. Some immune system process-related genes, such as indoleamine 2,3-dioxygenase 1 (Ido1), complement component 1, r subcomponent A (C1ra) and IFN-stimulated exonuclease gene 20 (Isg20) showed increased chromatin accessibility at the peak stage in MOL2 but not in MOL5/MOL6 (Fig. 5e,f and Supplementary Tables 3 and 4), further indicating that MOL2 triggers a more robust opening of the chromatin at immune genes after EAE than MOL5/MOL6. Recently, superenhancers, clusters with high levels of transcription factor binding and domains of regulatory chromatin (DORCs), have been suggested to have key roles in modulating gene expression12. Genes associated with DORCs were characterized by exceptionally large (five or more) numbers of significant peak–gene associations (Extended Data Fig. To explore the role of DORCs in the course of EAE, we defined differentially accessible domain-regulated genes in OLG subpopulations (Fig. We found three types of DORCs in MOL5/MOL6, among which only 2 domains were classified as Type 1 (high activity at the early stage), 49 domains were classified as Type 2 (high activity at the peak stage), and 24 domains were classified as Type 3 (high activity at the late stage; Fig. By contrast, we only found Type 2 (high activity at the peak stage) DORCs in MOL2. Type 2 DORCs in both MOL2 and MOL5/MOL6 were associated with genes involved in immune processes, with Gbp7, IFN regulatory factor 4 (Irf4) and Irgm1 included (Fig. For OPCs, 3 Type 1 DORCs and 49 Type 2 DORCs were identified, with many of them being immune related (Fig. a, Heat maps of normalized and scaled DORC scores of differentially expressed genes (DESeq2 Wald test and Benjamini–Hochberg multiple testing correction with a log2 (FC) of >1 and adjusted P value of <0.01) at different stages, 124 genes in MOL2 and 75 genes in MOL5/MOL6. DORCs were classified into three types: Type 1 with high activity in early-stage EAE, Type 2 with increased activity at the peak stage and Type 3 with increased activity at the late stage. Differential DORCs and associated GOs are shown in Supplementary Tables 3 and 4. b, Representative gene (Irgm1) in Type 2 DORCs in MOL2 (top) and MOL5/MOL6 (bottom). The genomic track represents its accessibility at different time points, and the links denote the significant correlation (P < 0.05) between peaks and Irgm1 (±500 kb from TSSs). The violin plots show Irgm1 expression in MOL2 (top) and MOL5/MOL6 (bottom). c, Normalized and scaled DORC score for Hoxb cluster genes (Type 3 DORCs in MOL5/MOL6) in OPC (left), MOL2 (middle) and MOL5/MOL6 (right). d, Feature plots showing DORC scores of Hoxb2, Hoxb3, Hoxb4, Hoxb5, Hoxb6 and Hoxb7. UMAP coordinates and population distributions are as in Fig. e, Normalized chromatin accessibility of Hoxa (top) and Hoxb (bottom) genes in MOL5/MOL6; the gray box highlights Hoxb genes in d. The presence of two additional DORC types (Types 1 and 3) in MOL5/MOL6 suggests that additional biological processes are regulated in this mature OL subtype compared to MOL2 (Fig. In particular, at this stage, we found increased DORCs regulating a group of homeobox (Hoxb) genes, such as Hoxb2, Hoxb3 and Hoxb4, in MOL5/MOL6 but not MOL2 (Fig. By contrast, all subtypes of OPCs present high DORC scores across all time points (Fig. The increased activity was only found in Hoxb but not in other HOX family gene regions (Fig. Thus, the increased chromatin accessibility of these developmental-related DORCs suggests that MOL5/MOL6 might have primed transcriptional programs compatible with nervous system repair and remyelination and promotion during the late stages of EAE. Although RNA velocity leverages splicing and RNA turnover to infer cellular transitional dynamics29, another key component for these dynamics are changes in the epigenomic landscape during cell transitions. To explore dynamics of OLG during disease, we applied MultiVelo30, a tool that integrates transcriptomics and epigenomics datasets to estimate cell-fate predictions. For a given gene, this tool can define the state of each cell into one of the following four phases: priming (brown), coupled-on (pink), decoupling (dark blue) and coupled-off (light blue; Methods, Fig. Due to the limited number of OPCs, some genes of interest did not present a complete transcriptional trajectory using MultiVelo (Methods and Extended Data Fig. We found that canonical OLG genes, such as Opalin, contactin 1 (Cntn1) and neuron navigator 1 (Nav1), showed coupled-on phases in MOL5/MOL6 in both CFA-Ctrl mice and in mice with EAE from early to late stages (Fig. However, many of these nervous system development-related genes exhibited either a priming or coupled-off phase in MOL2 (Fig. Thus, similar to our previous results, Multivelo analysis suggests a differential regenerative response of MOL2 and MOL5/MOL6 in the context of EAE. By contrast, immune-related genes, such as B2m, H2-D1 and Stat1, were observed to transition between the coupled-on and decoupling phases in most MOL2 cells, which indicated a highly open chromatin level and transcription (Fig. However, these immune genes showed a chromatin priming phase in MOL5/MOL6 in CFA-Ctrl animals, followed by a transient coupled-on phase at early-stage EAE. At the peak and late stages, B2m and H2-D1 transitioned into a coupled-off phase, indicating an important reduction in chromatin accessibility and gene expression of these immune-related genes at the chronic stage of the disease (Fig. Transition to an immune-like state has been previously shown to be incompatible with differentiation in OPCs5, and our data suggest that in MOL in EAE the enhanced immune-like state of MOL2 cells is distinct to a putative more remyelination-prone state of MOL5/MOL6 cells. Thus, these results further indicate a divergence in the response of different MOL populations to the evolving disease environment in EAE. a, Gene phase portraits predicted by MultiVelo for model 1 (M1, full line) and model 2 (M2, dotted line) genes. b,c, MultiVelo UMAPs of MOL2 (b) and MOL5/MOL6 (c) cells colored by time points. d,e, MultiVelo UMAPs of MOL2 and MOL5/MOL6 cell genes colored by gene state assigned by MultiVelo for representative central nervous system development- (d) and immune response-related (e) genes. To obtain insights into the molecular mechanisms mediating the divergent OLG responses in the different stages of EAE, we inferred the global gene regulatory network (GRN)31 for OPCs and MOL5/MOL6 and MOL2. We had previously found that the transcription factor BACH1 negatively regulated the induction of immune-related genes mediated by IFNγ13. Interestingly, we observed that its related transcription repressor BACH2 has more activity in CFA-Ctrl mice then in any EAE stage in both MOL2 and MOL5/MOL6 populations (Fig. At early stages of EAE, we found higher activity of NKX6.2 in both MOL2 and MOL5/MOL6, which might be consistent with the transitory regenerative responses observed at early stages. NKX6.2 has an important role in oligodendrogenesis and myelination32,33,34 and has been recently shown to drive OLG specification from induced pluripotent stem cells35,36. We also found that STAT3, which has immunosuppression potential37,38, showed higher positive activity in MOL5/MOL6 from the early stage, while at the peak stage in MOL2 and OPCs (Fig. a,b, Ranked transcription factor activities of the predicted transcription factors at different stages in MOL2 (a) and MOL5/MOL6 (b). The x axis shows transcription factor activity represented with a dot colored by stage and connected with a line. Transcription factor activity was calculated as the mean coefficient multiplied by the average expression in each time point. The y axis shows transcription factors ranked based on transcription factor activity at each time point (a zoom-in view is shown in Extended Data Fig. d, Relative expression of Stat3 and Cd274 in OPCs treated with control siRNA or Stat3 siRNA after IFNγ treatment, measured by RT–qPCR; **P < 0.01 and *P < 0.05. Student's two-tailed paired t-tests were used for comparisons between matched conditions (P = 0.0098 for Stat3 and P = 0.0409 for Cd274); n = 5 biologically independent samples per condition, each derived from primary OPC cultures isolated from a different mouse. e, Percentages of PD-L1+ OPCs treated with control siRNA or Stat3 siRNA after IFNγ treatment measured by flow cytometry; *P < 0.05. Student's two-tailed paired t-tests were used for comparisons between matched conditions (P = 0.0218 for PD-L1+ cells); n = 4 biologically independent samples per condition, each derived from primary OPC cultures isolated from a different mouse. We identified 26 transcription factors in MOL5/MOL6 and 23 transcription factors in MOL2, acting mainly with positive activities at the late stage. Among these transcription factors were TCF4, RUNX2, myocyte enhancer factor 2A (MEF2A) and POU homodomain transcription factors (Fig. 8a,b), with functions in nervous system development. In particular within MOL5/MOL6-predicted transcription factors, we found an increased number of candidates related to nervous system development at the late stage, including nuclear factor I A (NFIA) and TCF12, which are involved in the regulation of OL differentiation and maturation39,40,41 (Fig. Some transcription factors regulating cell differentiation, such as SOX4, were also found to mainly act at the late stage compared to other stages in MOL5/MOL6 (Fig. However, unlike in MOL5/MOL6, the differentiation-related factors TCF7L2 (ref. 42) and SOX4 exhibited the highest activity from the peak stage in OPCs (Extended Data Fig. 8c), suggesting that early CNS regeneration may already be initiated. These results further underscore differential regenerative potential of OLG during the late stages of the disease. STAT3 is a transcription factor that can be involved in immune suppression by, for instance, the upregulation of PD-L1 expression43,44. PD-L1 is an immune checkpoint protein whose immune-suppressing effects have been observed across a broad range of cell types45,46,47,48,49,50,51,52,53,54,55. We found that both the expression and chromatin accessibility of Cd274 (which encodes PD-L1) were increased at early and peak stages in OPCs and MOLs (Extended Data Fig. To investigate whether STAT3 regulates PD-L1 expression in OLG, we knocked down Stat3 expression using short interfering RNA (siRNA) or inhibited its activity with the inhibitor auranofin56,57 in primary cultured neonatal OPCs. We found that both Stat3 siRNA and auranofin effectively inhibited Stat3 expression following IFNγ treatment. Furthermore, flow cytometry analysis revealed a decrease in PD-L1 protein expression in OPCs treated with Stat3 siRNA (Fig. These findings suggest that certain immune suppression-related genes may be upregulated in OLGs during EAE in response to the inflammatory milieu, attempting to attenuate and regulate the immune response. The immune plasticity of OLG was first reported about 40 years ago58. In addition, MHC class I and MHC class II expression in OLG under inflammatory conditions was further observed in many subsequent studies59,60,61,62. Recently, with the emergence of single-cell transcriptomics, the concept of imOLG has been further expanded to neuroinflammation in the context of MS, Alzheimer's disease and aging in several studies4,5,8,62,63. We have recently shown that MHC and other immune-associated genes are highly expressed in EAE at the peak stage, with primed chromatin accessibility of some of these genes already in CFA-Ctrl animals4,13. In the current study, we applied multiomics scRNA-seq and scATAC-seq to comprehensively profile OLG cellular states throughout EAE, from onset to chronic stages. Our results show that several immune-related genes and their chromatin accessibility levels are already elevated in OLG, in particular in OPCs, at the early stage of EAE, when symptoms start emerging. Furthermore, these loci remain highly accessible even in the late disease stage. Immune-like OLG emerge early in EAE, before lesions are fully developed64. We also observe that the percentage of MHC class II+ OLG is similar inside and outside lesions, suggesting that the lesion environment is not essential for the transition of OLG to immune-like states. These findings are consistent with spatiotemporal transcriptomics analysis in EAE, indicating that disease-associated glia can be induced independent of lesions and that the disease-associated MOLs are in close proximity to disease-associated microglia and astrocytes64. Thus, it is plausible that the induction of imOLG might be mediated not only by infiltrating immune cells but also by other disease-associated glia or inflammatory environmental factors. The presence of imOLG at all stages of EAE could suggest differential functions at distinct time points. We observed that only a small percentage of Sox10+ cells express MHC class II genes. However, even minimal expression can have substantial functional implications. In MS and EAE, OLG are direct targets of the autoimmune response within the CNS. MHC class II expression in OLG might enable them to present antigens and activate CD4⁺ T cells, positioning OLG at the forefront of disease onset and progression. Nevertheless, our results showed immune-repressive pathway activation in MOLs at EAE early and peak stages, aligning with previous findings that OPCs and MOLs express PD-L1 (ref. Thus, MOLs could, in principle, activate or, alternatively, block immune responses in the context of EAE. At peak stages, given the presence of high numbers of professional antigen-presenting cells, such as microglia and macrophages, the role of immune MOLs might be more subtle and modulatory of the function of these other immune cells. We observe the persistence of MHC class I and MHC class II in OLG in the late stages of EAE, when imOLG might be involved in disease persistence, although a role in reducing inflammatory responses is also possible. IFNγ treatment of human HeLa cells has been shown to lead to long-term transcriptional memory65. Moreover, epigenetic memory of previous inflammatory events has also been observed in mouse epidermal stem cells66,67. Disease-associated astrocytes have also been suggested to present epigenetic memory following neuroinflammation68, and obesity has also been reported to lead to epigenetic memory in adipocytes, including genes involved in inflammatory signaling69. Here, we found that OLG can also retain memory of previous inflammatory insults at a chromatin accessibility level. In the most common course of MS, individuals with relapsing–remitting MS suffer from multiple remissions and relapses over the course of the disease, and the symptoms can worsen after each relapse1. Therefore, the observed epigenetic memory of immune-related genes in OPCs may contribute to faster and stronger immune gene expression following the next wave of stimulation during the relapse stage in individuals with MS. This immune epigenetic memory in OLG may contribute to the chronicity of the disease and the difficulty in treating demyelinating diseases. The heterogeneity of the OL lineage in development has been reported in our previous study19, and several subsequent studies have further confirmed the transcriptional and spatial preferences of distinct subpopulations of OL in development and disease4,7,14,20. In our analysis, we identified previously reported mature resting subtypes. The difference in the domain distribution between MOL2 and MOL5/MOL6, in combination with intrinsic factors, may thus contribute to the heterogeneity of MOL in response to the inflammatory environment and higher immune characteristics of MOL2. We observed notable disparities in CNS development functions between MOL2 and MOL5/MOL6. We saw that some genes related to CNS development always stay in a coupled-on stage in MOL5/MOL6 but not in MOL2, which suggests a more central role of MOL5/MOL6 in myelin maintenance and stability than MOL2 in disease, especially at the chronic stage. However, our results do not provide a definitive answer regarding whether MOL5/MOL6 at the late stages are newly generated or resilient cells that were already present before disease development. One hypothesis is that MOL5/MOL6 are newly generated and possess a stronger ability for CNS development. Alternatively, it is still possible that these MOL5/MOL6 survive in a severe inflammatory environment and contribute to remyelination once the inflammation in the CNS is diminished. To address this question, lineage tracing experiments need to be conducted in future studies. Our study provides a unique resource for understanding the intricate mechanisms underlying the gene regulation and chromatin accessibility of OLG in the context of MS using an animal model. In particular, our study sheds light on the distinct fates of MOL2 and MOL5/MOL6 during disease evolution, indicating that therapeutic strategies targeting MOLs need to be specific for the different populations. The road to fully understand the complex interplay of transcriptomics and epigenomics in MS requires further exploration, such as spatial resolution, which may provide a more comprehensive and accurate picture of the disease. Although our multiome analysis gives insights into the epigenetic potential and memory in OLG during the time course of the disease, at the onset of the disease, the inclusion of intermediate time points might allow further granularity and uncovering of additional cellular states in OLG. We also show that OPCs in vitro can acquire epigenetic memory in the form of chromatin accessibility at immune-related genes. Although it is possible that this epigenetic memory occurs in MOLs, it is challenging to assess this in vitro due to cell death of MOLs after prolonged culture in vitro. Future studies might elucidate whether this epigenetic memory also occurs in adult OPCs and MOLs in vivo and whether it plays a role after the second inflammatory challenge, as, for instance, in relapsing–remitting EAE models. The observed priming of OL differentiation and myelination gene programs in MOL5/MOL6, particularly at the early stages of EAE, suggest that there might be a window of opportunity for the promotion of MOL-associated remyelination in MS. Nevertheless, our results indicate only potential given the nature of epigenomic data, and it is thus unclear whether it is possible to harness this MOL capability to promote remyelination. A deeper analysis of the epigenetic landscape at a single-cell level by examining, for instance, activating and repressive histone modifications71 or DNA methylation might further elucidate mechanisms that could lead to the activation of these gene programs and promote remyelination in the context of MS. All animals were born, bred, housed and subjected to experimental treatment at Karolinska Institutet, Comparative Medicine Biomedicum animal facility for research on small rodents (KM-B). Sox10:cre-RCE:loxP (eGFP) transgenic mice were used in this study and were originally obtained by crossing mice with Cre recombinase under the control of the Sox10 promoter (The Jackson Laboratories, 025807; with a C57BL/6 genetic background) with reporter mice RCE:loxP-EGFP (with a CD1 background, 32037-JAX) to label the complete OL lineage. Breedings were performed with cre allele females and non-cre carrier males, with one male and up to two females. Breeding males carrying a hemizygous cre allele, along with the reporter allele, with non-cre females was avoided because it resulted in progeny expressing eGFP in all cells. None of the experimental animals in this study were subjected to previous procedures. The following housing conditions were used: dawn 06:00–07:00, daylight 07:00–18:00, dusk 18:00–19:00 and night 19:00–06:00. A maximum of five adult mice was housed per individually ventilated cage of type II (IVC seal safe GM500, Tecniplast). All animals were free from mouse viral pathogens, ectoparasites and endoparasites and mouse bacteria pathogens. Animal health surveillance was performed every third month according to the Federation of European Laboratory Animal Science Association recommendations74. Three basic and one complete Federation of European Laboratory Animal Science Association tests were performed per year (health monitoring was performed via PCR on exhaust air filter dust and by serology from sentinel animals). General housing parameters, such as relative humidity, temperature and ventilation, followed the European Convention for the Protection of Vertebrate Animals Used for Experimental and Other Scientific Purposes Treaty ETS 123, Strasbourg 18.03.1996/01.01.1991. Briefly, a relative air humidity of 55% ± 10% and temperature of 22 °C were maintain, and air quality was controlled with the use of stand-alone air handling units supplemented with HEPA-filtrated air. Monitoring of husbandry parameters was performed using ScanClime (Scanbur) units. Cages contained hardwood bedding (TAPVEI), nesting material, shredded paper, gnawing sticks and card box shelter (Scanbur). Water was provided by using a water bottle, which was changed weekly (water quality was assessed by 1SO 6222, SS EN-ISO 9308-2:2014 and SS EN-ISO 14189:2016 methods, Eurofins). Cages were changed once every week. All cage changes were performed in a laminar air flow cabinet (NinoSafe MCCU mobile cage changing unit) provided with a HEPA H14 EN1822 filter (0.3-μm particle size). Facility personnel wore dedicated scrubs, socks and shoes. Respiratory masks were used when working outside of the laminar air flow cabinet. Animals of both sexes were assigned to different experimental groups by randomization using the GraphPad randomization tool (GraphPad by Dotmatics). All experimental procedures on animals were performed following the European Directive 2010/63/EU, local Swedish directive L150/SJVFS/2019:9, Saknr L150 and Karolinska Institutet complementary guidelines for procurement and use of laboratory animals (Dnr. The procedures described were approved by the local committee for ethical experiments on laboratory animals in Sweden (Stockholms Norra Djurförsöksetiska nämnd), license numbers 1995-2019 and 7029-2020. For EAE induction, animals were injected subcutaneously with an emulsion of MOG35–55 in CFA (Hooke Laboratories, EK-2110; containing 1 mg of MOG35–55 per ml emulsion and 2–5 mg of killed mycobacterium tuberculosis H37Ra per ml emulsion, Hooke Laboratories), followed by the intraperitoneal injection of pertussis toxin (Hooke Laboratories, included in the induction kits) in 1× PBS (Gibco, 10010023) on days 0 and 1 (200–225 ng per animal, adjusted by lot according to the manufacturer's instructions). Scores of EAE were graded according to the following criteria: 0, asymptomatic; 1, limp tail or titubation; 2, limp tail and weakness of hindlimbs; 3, limp tail and complete paralysis of hindlimbs; 4, limp tail, complete paralysis of two hindlimbs with forelimb involvement; 5, moribund or dead; 0.5 for intermediate symptoms. Investigators were not blinded to group allocation and were aware of whether animals were assigned to the control or EAE group during both experimentation and outcome assessment. Accordingly, data collection and analysis were not blindly performed to the conditions of the experiments. CFA-Ctrl mice were injected subcutaneously with control emulsion containing CFA but without MOG35–55 (Hooke Laboratories, CK-2110), followed by the administration of pertussis toxin in PBS on days 0 and 1 (200–225 ng per animal, adjusted by lot according to the manufacturer's instructions). Spinal cords from mice with EAE were collected at the (1) early stage (days 8–9 after injection, score of 0–0.5, 14 mice in 7 multiome experiments), (2) peak stage (days 14–15, score of 3, 10 mice in 5 multiome experiments) or (3) late/chronic stage (days 37–40, score of 2–2.5, 14 mice in 6 multiome experiments). Spinal cord samples from CFA-Ctrl mice were also collected from the same stages (early stage, peak stage and late stage) with a score of 0 (at least four mice in two multiome experiments for each stage), alongside spinal cord tissues from noninduced Naive-Ctrl animals (3-month-old mice from the same strain, six mice in three multiome experiments). Scores for mice with EAE and CFA-Ctrl mice were plotted using GraphPad Prism version 9.0.0. At the early, peak and late stages, mice were perfused with cold PBS, and spinal cords were collected. Spinal cord tissues were then dissociated into a single-cell suspension according to the manufacturer's protocol for the Adult Brain Dissociation kit, mouse and rat (Miltenyi Biotec, 130-107-677; we did not perform the red blood cell removal step because the majority of red blood cells had been removed after PBS perfusion). For the debris removal step, most of the samples were processed with 38% Percoll except two Naive-Ctrl samples, two EAE peak-stage samples and two EAE late-stage samples, which were processed with debris removal solution (Miltenyi Biotec, 130-107-677) according to the manufacturer's protocol, to enrich more OPCs. Spinal cord single eGFP+ cells were enriched with a BD FACS Aria III Cell Sorter (BD Biosciences). The cells were then lysed and washed according to the demonstrated protocol for Nuclei Isolation for Single Cell Multiome ATAC + Gene Expression Sequencing (10x Genomics, CG000365), with the following modifications. Cells were centrifuged for 10 min at 300g and 4 °C, resuspended in lysis buffer (containing 10 mM Tris-HCl (pH 7.4), 10 mM NaCl, 3 mM MgCl2, 0.01% Tween-20, 0.01% IGEPAL (CA-630), 0.001% digitonin, 1% bovine serum albumin (BSA), 1 mM dithiothreitol (DTT) and 1 U μl−1 RNase inhibitor) and incubated on ice for 3 min. The nuclei were centrifuged for 5 min at 500g and 4 °C. Nuclei were washed once in wash buffer, followed by another wash with diluted 1× Nuclei buffer (10x Genomics, PN-1000283) containing 1% BSA, 1 mM DTT and 1 U μl−1 RNase inhibitor. Chromium Next GEM Single Cell Multiome ATAC + Gene Expression chemistry (10x Genomics, PN-1000283) was then applied to yield single-cell ATAC and RNA libraries. Libraries were sequenced on an Illumina Novaseq 6000 with a 50–8–24–49 read setup for ATAC (minimum of 25,000 read pairs per cell) and a 28–10–10–90 read setup for RNA (minimum of 20,000 read pairs per cell). Both male and female mice were used in our study. Most of the samples contained cells from one male mouse and one female mouse, except two early-stage EAE samples (one only contained male mice, another only contained female mice), two peak-stage EAE samples (one only contained male mice, another only contained female mice) and two late-stage EAE samples (one only contained male mice, another only contained female mice) for establishing a sex prediction model and validation. RNAscope ISH was performed on 14-μm spinal cord sections from the lumbar spinal cord of control mice and mice with EAE (n = 3 for each condition) with probes for mouse Sox10 (ACD, 435931) and H2-Ab1 (ACD, 414731-C2). The RNAscope ISH protocol for sections was performed following the manufacturer's instructions with minor modifications (RNAscope Multiplex Fluorescent Detection Reagents v2, 323110). Tissue sections were incubated in 1× target retrieval reagent (RNAscope Target Retrieval Reagents, 322000) for 5 min at 98 °C, followed two 2-min washes with DNase/RNase-free water. The samples were then incubated with protease IV (RNAscope Protease III and Protease IV Reagents, 322340) at room temperature for 20 min, followed by two 2-min washes with DNase/RNase-free water. The sections were hybridized with probes (C1:C2/C3 in a 1:50 dilution) for 2 h at 40 °C and washed twice with wash buffer (RNAscope Wash Buffer Reagents, 310091). Amplification steps were performed by incubating with v2Amp1 (30 min), v2Amp2 (30 min) and v2Amp3 (15 min) (RNAscope Multiplex Fluorescent Detection Reagents v2, 323110) at 40 °C, with two 2-min washes with wash buffer in between steps. The sections were incubated with v2-HRP-C1 for 15 min at 40 °C, and after two 2-min washes with wash buffer, tyramide signal amplification (TSA)-conjugated fluorophores (1:1,500 dilution in TSA buffer; RNAscope Multiplex TSA Buffer, 322810) were added and incubated for 30 min at 40 °C. The sections were washed twice with wash buffer and incubated with HRP blocker for 30 min at 40 °C. The v2-HRP, TSA-conjugated fluorophore and HRP blocker steps were then repeated for C2 and C3 channels, if applicable. At the end, the sections were incubated with DAPI (Sigma, D9542; 1:5,000) for 5 min and washed with PBS with 0.05% Tween-20 (VWR, 9005-64-5) for 2 min. For immunohistochemistry, after blocking with 5% normal donkey serum (Sigma) in 0.3% PBS/Triton X-100 for 1 h at room temperature, spinal cord sections were incubated overnight at 4 °C in the following primary antibodies: anti-MHC class II (Invitrogen, 14-5321-85, rat 1:50, clone M5/114.15.2) and anti-GFP (Abcam, ab13970, chicken 1:200) diluted in PBS/0.05% Tween-20/2% normal donkey serum. After washing the sections with PBS/0.05% Tween-20, secondary Alexa Fluor-conjugated antibodies (goat anti-rat (for MHC class II, 1:1,000; Invitrogen, A21434) and goat anti-chicken (for GFP, 1:1,000; Abcam, ab150169)) diluted in PBS/0.05% Tween-20/2% normal donkey serum were incubated for 2 h at room temperature. Slides were counterstained with DAPI, mounted with mounting medium and maintained at 4 °C until further microscopic analysis. Images were acquired using a Zeiss LSM800 confocal microscope (RNAscope) and Zeiss LSM980 microscope (immunohistochemistry) and processed in Fiji/ImageJ. For RNAscope, six ×20 randomly selected fields per mouse (three from lesions and three from nonlesions) were chosen for quantification. The percentage of MHC class II+ OLG was significantly higher at the peak stage (lesion: 5.33 ± 0.23%, nonlesion: 5.32 ± 0.23%) than at both early stage (lesion: 0.49 ± 0.849; nonlesion: 0.384 ± 0.333%) and late stage (lesion: 2.54 ± 0.23; nonlesion: 2.71 ± 0.32%) in both lesion (P = 0.0001) and nonlesion (P = 0.0001) areas (Fig. A two-way ANOVA with a Tukey's multiple comparisons test was performed using GraphPad Prism version 9.0.0 for comparing the percentages of Sox10+H2-Ab1+ cells between different time points and between lesion and nonlesion. Sox10:cre-RCE:loxP (eGFP) transgenic mice were used. The brains of postnatal days 3–6 pups were dissociated using a Neural Tissue Dissociation kit (Miltenyi Biotec, 130-092-628), according to the manufacturer's protocol. OPCs were obtained with MACS with CD140a microbeads following the manufacturer's protocol (CD140a Microbead kit, Miltenyi Biotec, 130-101-547). Cells were seeded in poly-L-lysine-coated (Sigma, P4707) dishes and grown on OPC proliferation medium comprising DMEM/F-12/GlutaMAX (Thermo Fisher, 10565018), N2 medium (Thermo Fisher, 17502048), penicillin–streptomycin (Thermo Fisher, 15140122), NeuroBrew (Miltenyi, 130-097-263), bFGF (40 ng ml−1; R&D, 233-FB) and PDGF-AA (20 ng ml−1; Peprotech, 315-17). Oli-neu cells (obtained from J. Trotter, Johannes Gutenberg University) were seeded in poly-L-lysine-coated (Sigma, P4707) dishes and grown on proliferation medium comprising DMEM (Gibco, 41966029) supplemented with 1× N2 supplement (Thermo Fisher, 17502048), 1× penicillin–streptomycin–glutamine (Gibco, 10378016), 340 ng ml−1 T3 (Sigma, T6397), 400 ng ml−1 T4 (Sigma, 89430), 10 ng ml−1 bFGF (R&D, 233-FB), 1 ng ml−1 PDGF-BB (R&D, 520-BB) and 0.5% fetal bovine serum (Gibco, 10500064). For the 2× IFNγ treatment, primary OPCs (n = 2) and Oli-neu cells (n = 3) were initially treated with IFNγ (100 ng ml−1; R&D, 485-MI-100) for 24 h. Afterward, the cells were cultured in proliferation medium without IFNγ for 96 h, and then a second dose of IFNγ was administered for an additional 24 h. For the 1× IFNγ treatment, cells were treated with IFNγ only at the time when the 2× IFNγ-treated cells received their second dose. RNA extraction was performed using an RNeasy kit (Qiagen, 74106), and RNA-seq libraries were prepared using Stranded Total RNA Prep, Ligation with Ribo-Zero Plus (Illumina, 20040525), according to the manufacturer's protocol. Libraries were sequenced on an Illumina NovaSeq X, with a 75–10–10–75 read setup. RNA extraction was performed using an RNeasy kit (Qiagen, 74106), followed by cDNA synthesis using a High-Capacity cDNA Reverse Transcription kit (Thermo Fisher, 4368814), according to the manufacturers' protocols. RT–qPCR was performed on a QuantStudio 5 using SYBR Green Master Mix (Thermo Fisher, 4385617; Supplementary Table 10). Relative gene expression was calculated using the change in cycling threshold (\({2}^{-\varDelta \varDelta \,{C}_{{\rm{t}}}}\)) method, normalizing to the housekeeping gene Gapdh. ATAC-seq was performed as previously described11, with minor adaptations. For each condition, 50,000 primary OPCs were collected, washed with PBS and lysed with 50 μl of lysis buffer (10 mM Tris-HCl (pH 7.4), 10 mM NaCl, 3 mM MgCl2, 0.1% IGEPAL (CA-630), 0.1% Tween-20 and 0.01% digitonin) on ice for 3 min. After lysis, 1 ml of wash buffer (10 mM Tris-HCl (pH 7.4), 10 mM NaCl, 3 mM MgCl2 and 0.1% Tween-20) was added, and the nuclei were pelleted by centrifugation at 500g for 10 min at 4 °C. After aspirating the supernatant, the nuclei were resuspended in 50 μl of transposition mix (25 μl of 2× TD buffer (Tagment DNA Buffer), 16.5 μl of PBS, 0.5 μl of 1% digitonin, 0.5 μl of 10% Tween-20, 2.5 μl of Tn5 Transposase (final 100 nM) and 5 μl of nuclease-free water) and incubated for 30 min at 37 °C in a thermomixer with mixing at 1,000 rpm. The DNA was purified both before and after PCR using a Zymo DNA Clean and Concentrator-5 kit (Zymo Research). Libraries were sequenced on an Illumina NovaSeq X with a 51–8–8–50 read setup. For transcription factor knockdown, siGENOME SMARTpool nontargeting control siRNA (D-001206-13-05, Dharmacon) or siRNA targeting Stat3 (M-040794-00-0005, Dharmacon) was used, which contains a pool of four siRNAs. One milligram of siRNA was diluted in OPTIMEM (31985062, Gibco), mixed with Lipofectamine 2000 (11668027, Invitrogen) and allowed to form complexes for 20 min at room temperature. Primary OPCs were then incubated with the siRNA complexes in OPTIMEM with N2 medium, NeuroBre, bFGF (40 ng ml−1) and PDGF-AA (20 ng ml&minu;1). After 72 h, IFNγ (100 ng ml−1) was added for 24 h. For the STAT3 inhibitor auranofin (Enzo Life Sciences), primary OPCs were cultured in OPC proliferation medium with 2 μM auranofin for 24 h, followed with IFNγ (100 ng ml−1) treatment for 24 h. Specific gene sequences were used for qPCR, and the primer sequences used are as shown in Supplementary Table 10. Relative gene expression was calculated using the \({2}^{-\varDelta \varDelta \,{C}_{{\rm{t}}}}\) method, normalizing to the housekeeping gene Gapdh. A two-tailed paired t-test was performed using GraphPad Prism version 9.0.0 for comparing the relative expression levels. For flow cytometry, OPCs were collected and washed with staining buffer (PBS/0.5% BSA). The cells were then stained using a LIVE/DEAD Fixable Near-IR Dead Cell Stain kit (for 633- or 635-nm excitation, Invitrogen) to determine cell viability, followed by staining of PD-L1-APC-conjugated antibody (1:100; BioLegend, 124312, clone 10F.9G2) for 30 min at 4 °C and washing once with staining buffer. Cellular fluorescence was measured with CantoII (BD Biosciences). Forward- and side-scatter parameters were used for the exclusion of doublets. Data were analyzed with FlowJo software 10.8.1 (TreeStar). In total, seven batches were collected from the sequencing facility. Fastq files from 28 samples were processed throughout the 10x Genomics standard pipeline. Gene expression and chromatin accessibility libraries were inputted into cellranger-arc2 (ref. 75) ‘count' v2.0.2 with default settings to align the biological readouts on the associated mm10 reference genome v2020-A-2.0.0. Sample aggregation of both transcriptomic and genomics metrics was done using the ‘aggr' of the same CellRanger executable file, without normalization. The aggregated count matrix and fragments file were loaded into R v4.3.2 (https://www.R-project.org). The former was lodged into a Seurat v5.0.3 (ref. 76) assay, whereas the latter was accommodated into a Signac v1.13.0 (ref. 77) chromatin assay associated with Ensembl78 base annotation v79_2.99.0 for mice where University of California, Santa Cruz, nomenclature was applied to provide gene name readability. If not specifically mentioned, the following pipeline descriptions belong to either the Seurat or Signac package. Cell-free mRNA contamination was removed from each droplet using Cellbender v0.3.0 (ref. Genes called in all samples by Cellbender were included into the feature list. Potential cells called in both CellRanger and Cellbender were included in the cell list. After determining that each cell could be uniquely identified, quality metrics for both gene expression and peak accessibility were calculated. Among others, the mitochondrial ratio, cell cycle score, nucleosome signal and TSS enrichment were generated to support the quality control cutoffs. Minimum and maximum outliers in counts, detected genes, TSS enrichment and percentage of mitochondrial information were removed from the analysis (Supplementary Methods). The aggregated fragments file was used in MACS2 v3.0.0 (ref. Peaks falling in nonstandard chromosomes or overlapping genomic blacklist regions from the mm10 genome were discarded. The fraction of reads in peaks was then calculated for each cell, and the mean value for each sample ranged from 0.64 to 0.79. In total, 976,949 entries were registered, with 20,611 promoter, 939,715 distal (10% inside the gene body) and 16,623 intergenic hits. To consider a peak associated with a given gene, the function LinkPeaks from Signac was used with a P value cutoff at 0.05. The P values were corrected using the Benjamini–Hochberg method, and only associations with a P value adjusted lower than 0.01 were selected for downstream analysis. Peak coaccessibility was performed using Cicero v1.3.9, and P values were calculated on z scores of coaccessibility scores, adjusted using the Benjamini–Hochberg method and selected if less than 0.01 (Supplementary Methods). The computation of fragment counts per cell in the gene body and promoter region (gene activity) was performed with the GeneActivity function from Signac, fetching all biotypes and extending 500 bp upstream of the TSS to catch the promoter region. All other parameters were set to default. This matrix was used later to infer chromatin accessibility for each gene. The detection of putative events where more than one cell in the same droplet occurs was performed using DoubletFinder v2.0.4 (ref. Using the multiplet rate provided by 10x Genomics (https://kb.10xgenomics.com/hc/en-us/articles/360001378811-What-is-the-maximum-number-of-cells-that-can-be-profiled), we were able to measure a 0.8%/8% theoretical number of doublets. Predicted doublets were removed from the downstream analysis (Supplementary Methods). In total, 69,152 cells were sequenced with a priori sex annotation. This knowledge was used to create a machine-learning sex classifier to investigate sex-dependent transcriptomic or epigenetic differences. Equal numbers of cells for each sex (30,940 cells) were extracted from the main object to a sex object to build the model. Mouse genes and positions from chromosomes X and Y were extracted from Ensembl v79_2.99.0 (ref. We then used the newly created model to process each cell of the main object. Correction of misassigned cells (8.32%) from a priori sex-annotated samples was performed (Supplementary Methods). Some studies have reported that immune responses are different in many aspects between males and females in both MS and its animal models82,83. Because most of our samples were mixed with one male mouse and one female mouse, we created a sex prediction model based on the expression of sex-related genes (Methods). This sex prediction model was validated with samples containing cells from only male or female mice with EAE, with an accuracy of 95.3% on the validation dataset. There was better prediction accuracy in the male sample (99.48%) than in the female sample (86.13%; Extended Data Fig. 9d,e) and compared the differences between male and female OLG based on the sex prediction results. For OLG subpopulations, apart from sex-related genes, such as X inactive specific transcript (Xist and Tsix), ubiquitously transcribed tetratricopeptide repeat containing Y-linked (Uty), eukaryotic translation initiation factor 2 subunit 3 structural gene Y-linked (Eif2s3y), dead-box helicase 3 Y-linked (Ddx3y) and lysine demethylase 5D (Kdm5d), no other genes were found to be differentially expressed between males and females (Extended Data Fig. Accordingly, we found no major differences between the OLG of males and females at the gene expression level in our data. We also did not observe a significant difference in the percentages of imOLG between male and female mice (Extended Data Fig. Thus, our data indicate that OLG responses to the neuroinflammatory environment in EAE are not characterized by sexual dimorphism. The obtained scaled matrix was then natural-log-transformed. The peak count matrix was processed through a term frequency inverse document frequency (TF-IDF) normalization, which computed a log (TF × IDF) with a scale factor of 10,000. The obtained scaled matrix was then natural-log transformed. To proceed to the reduction of the dataset dimensions, the 2,000 most variable genes were selected via variance stabilizing transformation, and 95% of the most common peaks were also selected. The expression of each gene was transformed by subtracting its average expression and scaled by dividing its standard deviation. Reduction of the dimensionality of the expression matrix on the 2,000 most variable genes was done by running a principal component (PC) analysis. For both PCs and LSI, the graphs were made with the ‘annoy' method with Euclidean distances and 50 trees. A ‘k' for the k-nearest neighbor algorithm was set to 20. For the construction of the graph based on PCs, an acceptable Jaccard index was set to 0. For the multimodal nearest neighbor graph, the same numbers of PCs and LSIs were used. A ‘k' for the k-nearest neighbor algorithm was set to 20, 200 approximate neighbors were computed, and the cutoff to discard the edge in the shared nearest neighbor graph was set to 0. The resolution of each clustering was assessed using the ‘clustree' R package iterating over different resolutions to pinpoint the most stable clustering resolution. Of note, an undetermined cluster specific at 97.86% for a unique sample at the early time point was standing out of all other clusters. In all visualizations of the cells, despite the high dimensionality of the data, the nearest neighbor graphs were reduced to only two dimensions using UMAP with the appropriate number of PCs and LSIs (‘Dimension reduction'), and other values were set as default. As sample libraries were not prepared on the same day and sequenced in the same sequencing run, and technical artifacts might arise and generate variability not related to biology in the dataset. However, accounting for such disequilibrium, without removing the subtle differences between time points, was challenging. Batch effect correction methods such as Harmony84 or Scanorama85 with soft parameters completely masked the differences between time points. Moreover, sample integration with ‘cca' and ‘rpca' was also overcorrecting time-point-specific cell populations. Although the popular batch effect methods were not suitable for our particular dataset, we relied on the strength that every sequencing run possessed at least one CFA-Ctrl sample (except on late time point and Naive-Ctrl samples). Thus, we investigated if the cells in each CFA-Ctrl cluster were well mixed across the different replicates. To analyze further at the cell mixing level, a local inverse Simpson's index (LISI) from Harmony v1.2.0, accounting for categorical variable diversity, was calculated for an equivalent number of cells in each CFA-Ctrl replicate, using the first two components of the joint projected UMAP. The same strategy was applied to the Naive-Ctrl samples, where 15 of the 17 major Naive-Ctrl clusters mainly contained cells ascending from the two replicates; one homogeneous cluster was coming from a microglia cluster and the other one was from a pericyte population. At the single-cell level, 76% of the cells maintained a LISI of at least 0.5. For all EAE time points, the same method was applied, yet the calculated LISI was lower than CFA-Ctrl or Naive-Ctrl (EAE score equal to 0), probably due to the heterogeneity of the EAE score at the sample collection within each time point (46% for early, 51% for peak and 49% for late). Although this method does not allow us to mitigate potential hidden batch effects, it provides a well-balanced cell distribution at a consistent EAE score. After removing cells with less than 200 detected genes, more than 8,000 detected genes and less than 10% of reads matching the mitochondrial genome using a dataset from the literature4, the transcriptomes of the remaining cells were normalized and scaled using SCTransform from Seurat. Cell-type annotation was assessed per cluster on the joined graph clustering resolution, and label transfer outputs, cell-type marker gene expression and chromatin accessibility were taken into consideration. More refined cell types for the MOL population were later considered. Exhaustive lists of immune-related genes published in Meijer et al.13, specific to ‘immune response' (GO:0002250) and ‘immune system process' (GO:0002376), were used to evaluate the immune capability of each cell. Cells from Naive-Ctrl animals were used as the background signal. After removing unexpressed genes in our dataset, subcategories with genes related to positive regulation of immune response, negative regulation of immune response, antigen processing and presentation, positive regulation of type I/II IFN-mediated signaling pathway, positive regulation of cytokine production, complement activation, CD4+ T cell-related immune response and CD8+ T cell-related immune response were extracted from the immune gene list, respectively, using the MGI immune database (https://www.informatics.jax.org/vocab/gene_ontology). For each cell type, immune scores were calculated for each immune subcategory and as a whole, using the AddModuleScore function from Seurat with default parameters. Under the null hypothesis that each cell follows the immune score distribution of Naive-Ctrl cells in a given population, each cell with a P value less than 0.05, rejecting the null hypothesis, was classified as ‘immune'. Cells accepting the alternative hypothesis (P value greater than 0.05), sharing a relationship between the immune score of a given cell and the immune score distribution of the Naive-Ctrl population, were classified as ‘nonimmune'. Due to the lack of cells annotated as neurons in Naive-Ctrl samples, this population was not annotated. The same approach was used on the gene activity matrix to compare transcriptional and epigenetic immune states. An investigation of cell status was also performed using damage-associated gene lists8 (95 genes) following the same strategy (Supplementary Table 9). To acknowledge the impact of CFA and pertussis toxin in CFA-Ctrl compared to Naive-Ctrl animals, we first performed a Pearson correlation on the gene expression matrix in each cell type between replicates of each time point. We found a high similarity of expression between OLs from CFA-Ctrl time points (Pearson correlation score between CFA-Ctrl replicates: 0.97–0.99; Extended Data Fig. We did not compare the difference between CFA-Ctrl replicates in neurons, ependymal cells and pericytes due to low cell numbers. Because the cells we used for sequencing were sorted based on GFP signal, these microglia are likely to be activated microglia that engulf OLs and are therefore more sensitive to ex vivo alteration. The finding that microglia are more sensitive than other CNS cell types has also been confirmed before in another study86. We did not compare the difference between CFA-Ctrl replicates in neurons, ependymal cells and pericytes due to low cell numbers. A gene expression matrix with CFA-Ctrl and Naive-Ctrl replicates was loaded into a SingleCellExperiment v1.24.0 (ref. 87) object where genes with less than ten counts were removed. Feature counts were then aggregated across cell types and broken down by sample. Each cell type was processed independently, and samples formed by less than 5 cells and time points formed by less than 30 cells were not considered for downstream investigation. The resulting matrix and associated metadata were loaded into an R object using DESeq2 v1.42.1 (ref. 88), with a design formula including sample names in addition to the debris removal method as a covariate. After size factors and dispersion estimations, results tables were extracted per contrast of Naive-Ctrl on CFA-Ctrl. Genes presenting an absolute log2 (FC) over 1 with an adjusted P value less than 0.01 and a baseMean (the average of the normalized count values, dividing by size factors, taken over all samples) greater than 1 were selected as differentially expressed. Seven genes passed the filters in MOLs (Adamts1, Gli2, Lypd6, Map7d2, Npr1, Rnf125 and Sema3c; Extended Data Fig. 9j), and one gene was differentially expressed in microglia between CFA-Ctrl and Naive-Ctrl samples (Csmd3). All cells belonging to the OL lineage were subsetted for fine-tuning annotation. One EAE early time point sample collected on day 8 after immunization with a score of 0 (without any symptom but with weight loss during the disease course) was removed from the analysis due to no EAE symptoms and similar gene expression as CFA-Ctrl. As no major gene expression differences were found between Naive-Ctrl and CFA-Ctrl in the OL lineage population, Naive-Ctrl samples were removed from the downstream analysis. Nevertheless, a few differences must be mentioned, specifically, the first 20 PCs and the first 9 LSIs were selected for graph construction. Additionally, cluster resolutions of 2.2, 0.8 and 3.6 were selected for gene expression, peak accessibility and joined modality, respectively. From the 47 potential OL lineage cell clusters on joined graph clustering resolution, we aggregated them using the average hierarchical clustering methods (Supplementary Methods). The OPC cluster was subdivided into three subclusters as we observed two time point-specific groups of OPCs and a third group expressing cycling genes. Each aggregated cluster was manually assigned to the main OL lineage cell types (OPC, COP, MOL1, MOL2 and MOL5/MOL6) using specific gene markers and label transfers. OL lineage cell type subclustering (α, β, γ, δ, ε, ζ, η, θ, ι and κ) was assessed and ordered along their average immune score for each main OL lineage cell type (Extended Data Fig. For instance, compared to other MOL2 populations, MOL2-ε, predominantly derived from the peak stage of EAE (Fig. 3c), exhibited an enrichment of immune-related genes, including IFN-induced protein with tetratricopeptide repeats 3 (Ifit3), Nlrc5 and IFNγ-induced GTPase (Igtp), among others (Supplementary Table 2). MOL2-α, which mainly came from CFA-Ctrl and EAE at the early stage, expressed higher lipid metabolic process genes, such as 3-hydroxy-3-methylglutaryl-CoA synthase 1 (Hmgcs1), cytochrome P450 family 27 subfamily A member 1 (Cyp27a1) and squalene epoxidase (Sqle; Supplementary Table 2). Oligodendrocytic myelin paranodal and inner loop protein (Opalin), which is associated with OL differentiation, was enriched in MOL5/MOL6-β. A major increase in the expression of actin cytoskeleton organization-related genes, such as actin binding LIM protein family member 2 (Ablim2), NCK-associated protein 5 (Nckap5) and prickle planar cell polarity protein 1 (Prickle1), was observed in MOL5/MOL6-ζ compared to other MOL5/MOL6 populations (Supplementary Table 2). The majority of MOL2-β and MOL2-γ were derived from late-stage EAE (Fig. 3b,c), with gene markers associated with metabolism and differentiation, like ectonucleotide pyrophosphatase/phosphodiesterase 6 (Enpp6) and S100 calcium binding protein B (S100b; Supplementary Table 2). In addition, most MOL5/MOL6-α, MOL5/MOL6-β and MOL5/MOL6-γ cells were mainly composed of cells from early-stage CFA-Ctrl and EAE samples, MOL5/MOL6-θ were mainly composed of cells from peak-stage EAE and cells from MOL5/MOL6-ζ and MOL5/MOL6-ι were drawn by late-stage EAE (Fig. Transcriptomic and epigenomic variations in specific cell types throughout the disease time course were assessed in a pseudobulk manner. This technique allows us to overcome the sparsity of the datasets and highlight dynamic features within three modalities, gene expression, gene/promoter accessibility and peak accessibility. These tri-omics modalities meet the prerequisites for analyses based on the negative binomial distribution (lack of intrasample variability, most of the features remain stable throughout the tested conditions, raw integer counts to specific genomic locations, sufficient number of replicates). Each modality was loaded into a SingleCellExperiment v1.24.0 (ref. 87) object where features with less than ten counts were removed. Feature counts were then aggregated across cell types and broken down by sample. Each cell type was processed independently, and samples formed by less than 5 cells and time points formed by less than 30 cells were not considered for the downstream investigation. The resulting matrix and associated metadata were loaded into an R object using DESeq2 v1.42.1 (ref. 88), with a design formula including sample names in addition to the debris removal method as a covariate. After size factors and dispersion estimations, a Wald test was performed followed by a multiple testing correction using the Benjamini–Hochberg method. The results tables were then extracted per contrast. We decided to investigate four contrasts (early/control, peak/early, late/early and peak/late), depicting a broad overview of the disease time course. Features presenting an absolute log2 (FC) greater than 1 with an adjusted P value of less than 0.01 and a baseMean (the average of the normalized count values, dividing by size factors, taken over all samples) of greater than 1 were selected to be displayed on the heat maps and GO analyses for the modalities represented by gene and promoter and were included into the transcription factor analysis for the modality represented by peaks. Differential gene expression analysis between males and females within each time point was performed similarly, merging gene counts by both sample and sex. Expression dynamics were assessed using velocyto v0.17.17 (ref. 29) to retrieve the number of spliced and unspliced reads in each sample. In addition to the CellRanger output directory, the GTF annotation file used by CellRanger was given, along with a GTF annotation file for mm10 repetitive elements from RepeatMasker (https://www.repeatmasker.org). Resulting loom files were loaded into scVelo89 in Python and directly imported via MultiVelo v0.1.3. After merging the samples, OL lineage cells were selected to undergo a log-transformed normalization on the 1,000 most variable features with at least ten read counts. The chromatin accessibility matrix from Macs2 output was loaded, and peaks were aggregated around each gene using peak annotation (‘Peak annotation') and feature linkage prediction (‘Peak annotation') inputted in the aggregate_peaks_10x function. OL lineage cells were selected to undergo a TF-IDF normalization. The 120,183 cells as well as the 798 genes with both expression and chromatin accessibility signals were picked to compute moments for velocity estimation for each cell across its 30 nearest neighbors calculated from Euclidean distances in the first 30 PC spaces of the expression matrix. Weighted nearest neighbor properties calculated previously on the OL lineage cells were used to smooth the epigenetic modality and were incorporated into RNA velocity to recover chromatin dynamics and carry out enhanced lineage predictions. This last step was done on MOL5/MOL6 and MOL2 populations separately. For each cell type, a new UMAP was generated; velocity, latent time and terminal states were processed; and CFA-Ctrl cells were set as root cells. MultiVelo classifies, if possible, each gene into two modules of biological dynamics. This tool is anchored in two models for the correlation of gene expression and chromatin accessibility changes within the latent time line: a first model (M1) where chromatin starts closing before the end of transcription and a second model (M2) where chromatin starts closing after the end of transcription. Moreover, the coupled kinetics of the transcriptomic and epigenomic profiles can be used as leverage to predict a current cell state for a given gene. The decoupling state is picked when there is a decorrelation between chromatin and unspliced transcripts dynamics. Due to the limited number of OPCs, some genes of interest did not present a complete transcriptional trajectory using MultiVelo. Therefore, we focused on key metrics to highlight their partial kinetics, including unspliced RNA, spliced RNA and chromatin accessibility levels of the same group of immune-related genes as in MOLs. We observed that spliced RNA levels of these immune-related genes were elevated in OPCs both at peak and in a subset of early-stage cells and were accessible at all time points. By contrast, unspliced RNA levels of these genes remained low across time points (Extended Data Fig. Thus, based on OPC unspliced and spliced dynamics, OPCs may have a faster RNA splicing process than MOLs to meet their immune functional demands at early and peak stages of the disease. Each peak-to-gene connection, occurring in at least ten cells and within a 50,000-bp distance from the gene TSS, with a positive Pearson correlation coefficient and a P value lower than 0.05 was considered a meaningful interaction. From the normalized matrix of meaningful peaks per cell, the normalized numbers of fragments in peaks were aggregated by gene using the peak-to-gene connections, for a total of 14,665 genes joined to at least one peak. The regulatory chromatin score for each cell of the generated genes was then multiplied by a scaling factor of 10,000. Notwithstanding the value of such enhancer scores, we were interested in deciphering more consequent DORCs. Therefore, we selected a pool of genes connected to at least five peaks to redo the peaks-to-genes connectivity calculation on a broader distance of 500,000 bp from the gene TSS. Similar to the regulatory chromatin of genes, the DORCs were normalized, aggregated per gene according to the new connection material and scaled to generate a DORC score. As calculating the sum of the peak counts conserves the negative binomial distribution of the epigenetic modality, both regulatory chromatin and DORC-regulated genes were reversed into unnormalized matrices (scaling factor and number of fragments in peaks for a given cell) to be loaded into a SingleCellExperiment object to ascertain, with confidence, gene variations between time points in a pseudobulk manner as described above, with the same parameters and thresholds. GRNs were inferred with Pando 1.0.0 (ref. Pando models the relationship between transcription factors and their binding sites in selected regulatory regions with the expression of target genes combining multiome RNA and ATAC information. The whole dataset was subset in MOL2 and MOL5/MOL6. For each MOL, we randomly subset a maximum of 2,000 cells per time point. We selected peak regions specific to each time point in MOL2 and MOL5/MOL6 (‘Multimodality features selection') as candidate regulatory regions to scan for transcription factor binding motifs with JASPAR (2022 release). The GRNs were inferred by fitting generalized linear models (GLMs) implemented in Pando for the expression of each gene. The regression model with peak–transcription factor pairs as independent variables and target genes as response variables was built for MOL2 and MOL5/MOL6 populations independently. The model estimates of each predicted transcription factor with a target gene can be interpreted as a measure of interaction between both. We calculated transcription factor activity by multiplying the mean estimates (coefficient) in the model by the average transcription factor expression at each time point. Transcription factor activities were ranked per the highest positive activity in each time point. Gene modules were selected with an R2 threshold of an adjusted P value lower than 0.05 and default parameters with the Pando find_modules function. A total of two replicates per condition were sequenced for both bulk RNA-seq and bulk ATAC-seq. Raw FastQ files were aligned and processed using the NextFlow core ATAC and RNA pipelines90, nextflow v24.04.2, to the genome assembly GRCm38. For bulk RNA-seq, gene-level integer raw counts tables from nf-core Salmon output were loaded into R and used as input for edgeR v4.2.0 (refs. The combined counts table was used to estimate the library size, perform CalcNormFactors with the trimmed mean of M values (TMM) method and perform the fit of genewise common dispersion (glmFit), followed by a likelihood ratio test (glmLRT). Benjamini and Hochberg's algorithm test was used to control the FDR. First, differential gene expression was tested for changes in the first dose of IFNγ treatment versus control. Second, the resulting genes with a P value of <0.05 and FDR of <0.05 were tested for differential gene expression changes after the second IFNγ treatment dose. The experiment design included batch (replicates) and treatment as covariates. Visualization of the expression levels of differentially expressed genes was performed after running VST normalization from DESeq2 v1.44.0 in the whole dataset, including all replicates from the five time points. Aligned reads for treatment and replicates were extracted from processed BAM files from the nf-core ATAC-seq pipeline. 93) coverage on the defined TSS windows and enhancer regions as a reference, an integer count table was built for enhancer regions and TSS windows, respectively, with defined regions as rows and samples as columns. Raw count tables on defined regions were loaded into R and used as input for edgeR v4.2.0, together with their associated metadata. Count tables were used to estimate the library size, perform CalcNormFactors with the TMM method and perform the fit of regionwise common dispersion (glmFit), followed by a likelihood ratio test (glmLRT). Benjamini and Hochberg's algorithm test was used to control the FDR. TSS windows and enhancer regions were selected based on a ⎸ log (FC) ⎸ of <0.5 and FDR of >0.05. To define the TSS windows, also referred to as promoter regions, all annotated TSSs from EnsEmbl Mouse genes version 102 were extracted. The resulting resized TSS regions were merged whenever there were overlaps between them using BedTools 2.17.0 mergeBed –s –nms. The resulting merged TSS regions were resized, adding upstream 500 bp and downstream 500 bp from the center. To avoid repeated regions, each window was annotated with the closest gene as a consensus. Windows including more than one TSS usually referred to alternative TSSs of the same gene. One gene can have several TSS windows assigned. To define enhancer regions, the regions defined as proximal and distal enhancers from ENCODE candidate cis-regulatory element (cCRE; GRCm38/mm10) assemblies and last updated 26 May 2021 were retrieved. Enhancer cCREs overlapping the defined TSS windows (‘IFNγ treatment, bulk RNA-seq and bulk ATAC-seq processing') were filtered out using BedTools v2.25.0 intersect –v. Resulting enhancer cCREs were merged in nonoverlapping regions with bedtools 2.17.0 mergeBed –s –nms and extended 500 bp upstream and downstream from the center. To avoid regions without evidence from our primary OPC bulk ATAC-seq analysis, any defined region not overlapping the consensus peaks from all conditions called by nf-ATACseq were discarded. To annotate enhancers to potentially regulated genes, each enhancer region was assigned to a candidate gene based on the previously calculated peaktogene interactions, CICERO, in our single-cell multiome data from OLG. One enhancer region can have several annotated cCREs and be assigned to different genes. Normalized counts were scaled from 0 to 1 by row (gene) for visualization purposes94,95,96 Samples with less than 5 cells and time points with less than 30 cells were removed before heat map generation. For gene expression and gene/promoter accessibility, normalized log transform aggregated gene expression and gene/promoter accessibility at each time point and cell type were calculated to generate heat maps scaled across time points from 0 (low) to 1 (high). For gene regulatory chromatin, the average value of the gene regulatory chromatin score at each time point and cell type was scaled across time points from 0 (low) to 1 (high). Missing values in any tested modalities were set to 0. For each modality, the average of the previously calculated scaled values per gene category and time point was used to produce the summarized dynamic line plots and their associated standard deviation of the distribution shown as a band of the same color. For each cell type, the top 50 features with the highest amplitude across time points were selected. Genes possessing an Entrez Gene ID underwent pathway enrichment via the ‘enrichPathway' function from the ReactomePA v1.46.0 R package, with a pvalueCutoff set at 0.05, a qvalueCutoff set to 0.05 and FDR (‘fdr') as a method of adjustment. A maximum of ten pathways (adjusted P value of <0.05) ranked by adjusted P value were displayed on dot plots. Network graphs were built on significant pathways (adjusted P value of <0.05) from the ‘emapplot' function, taking as input the results of the pairwise_termsim function, with the Jaccard similarity coefficient as the calculation method for a maximum of 50 pathways ranked by adjusted P value. Coverage of the DNA fragments within a given genomic region was determined using CoveragePlot from Signac. All fragments from 50 randomly selected cells per cluster and falling into a specific genomic region were carried into a 250-bp windowed matrix that was further down-binarized and plotted. The number of cells per cell subtype in each condition was aggregated, normalized across conditions to get a proportion of each cell subtype per condition and divided by the number of conditions to keep the sum of the proportion equal to 100%. The number of predicted male and female cells in each sex-specific sample was aggregated and normalized across samples to get a proportion of each sex per sample. Distribution of cell numbers in the top levels of the circos plots, matched to each corresponding bottom level. For an unbiased visualization of proportion, some circos plots were randomly downsampled by top levels, bottom levels or both (details are mentioned in the figure legends). The mouse genome was segmented into 100-bp windows, and fragments were assigned to their corresponding window, generating a binarized genome per cell matrix. For each group of cells, the fragment counts matrix was aggregated, multiplied by a scaling factor of 10,000 and divided by the number of fragments in the group. Data were processed inside notebooks within a singularity environment (‘Code availability') on a high-performance computer cluster running under Ubuntu 20.04 LTS. One EAE early time point sample with a score of 0 was removed from the analysis due to no EAE symptoms and similar gene expression as CFA-Ctrl, as well as a MOL cluster specific at 97.86% for a unique sample at the early time point. Differential gene expression was determined using the DESeq2 package using negative binomial GLMs, which is the expected distribution from single-cell experiments97,98. Animals of both sexes were randomly assigned to experimental groups using the GraphPad randomization tool. The number of cells in each sample was selected to fit the specifications outlined by 10x Genomics. The investigators were not blinded to allocation during experiments and outcome assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw fastq files, counts matrices and genomic tracks are available on Gene Expression Omnibus (GSE250589, GSE283085 and GSE283086). Data are also available for browsing at the University of California, Santa Cruz, Cell Browser and Genome Browser (https://olg-dyn-eae-multiome.cells.ucsc.edu) and at https://ki.se/en/mbb/research/research-division-of-molecular-neurobiology/goncalo-castelo-branco-group/oligointernode. Source data are provided with this paper. Jupyter notebooks to process the raw datasets and reproduce the figures are accessible at https://github.com/Castelo-Branco-lab/EAE_multiomics_2025. Bergles, D. E. & Richardson, W. D. Oligodendrocyte development and plasticity. & van Luijn, M. M. B and T cells driving multiple sclerosis: identity, mechanisms and potential triggers. Falcão, A.M. et al. Disease-specific oligodendrocyte lineage cells arise in multiple sclerosis. Kirby, L. et al. Oligodendrocyte precursor cells present antigen and are cytotoxic targets in inflammatory demyelination. Morabito, S. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease. Pandey, S. et al. Disease-associated oligodendrocyte responses across neurodegenerative diseases. Kaya, T. et al. CD8+ T cells induce interferon-responsive oligodendrocytes and microglia in white matter aging. Jakel, S. et al. Altered human oligodendrocyte heterogeneity in multiple sclerosis. Macnair, W. et al. snRNA-seq stratifies multiple sclerosis patients into distinct white matter glial responses. & Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Ma, S. et al. Chromatin potential identified by shared single-cell profiling of RNA and chromatin. Meijer, M. et al. Epigenomic priming of immune genes implicates oligodendroglia in multiple sclerosis susceptibility. Distinct oligodendrocyte populations have spatial preference and different responses to spinal cord injury. Speir, M. L. et al. UCSC Cell Browser: visualize your single-cell data. Matsuoka, T. et al. Neural crest origins of the neck and shoulder. Sousa, V. H., Miyoshi, G., Hjerling-Leffler, J., Karayannis, T. & Fishell, G. Characterization of NKX6-2-derived neocortical interneuron lineages. Jung, M. et al. Lines of murine oligodendroglial precursor cells immortalized by an activated neu tyrosine kinase show distinct degrees of interaction with axons in vitro and in vivo. Marques, S. et al. Oligodendrocyte heterogeneity in the mouse juvenile and adult central nervous system. Hilscher, M. M. et al. Spatial and temporal heterogeneity in the lineage progression of fine oligodendrocyte subtypes. Kenigsbuch, M. et al. A shared disease-associated oligodendrocyte signature among multiple CNS pathologies. Yeung, M. S. Y. et al. Dynamics of oligodendrocyte generation in multiple sclerosis. Bacmeister, C. M. et al. Motor learning promotes remyelination via new and surviving oligodendrocytes. Neely, S. A. et al. New oligodendrocytes exhibit more abundant and accurate myelin regeneration than those that survive demyelination. Mezydlo, A. et al. Remyelination by surviving oligodendrocytes is inefficient in the inflamed mammalian cortex. The function of RhoGTPases in axon ensheathment and myelination. Opposing roles for HOXA2 and HOXB2 in hindbrain oligodendrocyte patterning. La Manno, G. et al. RNA velocity of single cells. Li, C., Virgilio, M. C., Collins, K. L. & Welch, J. D. Multi-omic single-cell velocity models epigenome-transcriptome interactions and improves cell fate prediction. Fleck, J. S. et al. Inferring and perturbing cell fate regulomes in human brain organoids. Chelban, V. et al. Mutations in NKX6-2 cause progressive spastic ataxia and hypomyelination. Southwood, C. et al. CNS myelin paranodes require NKX6-2 homeoprotein transcriptional activity for normal structure. Dorboz, I. et al. Biallelic mutations in the homeodomain of NKX6-2 underlie a severe hypomyelinating leukodystrophy. Rapid and efficient generation of oligodendrocytes from human induced pluripotent stem cells using transcription factors. Chanoumidou, K. et al. One-step reprogramming of human fibroblasts into oligodendrocyte-like cells by SOX10, OLIG2, and NKX6.2. Mathew, R. et al. A negative feedback loop mediated by the BCL6–Cullin 3 complex limits TFH cell differentiation. Wong, Y. W. et al. Gene expression analysis of nuclear factor I-A deficient mice indicates delayed brain maturation. Talley, M. J., Nardini, D., Ehrman, L. A., Lu, Q. R. & Waclaw, R. R. Distinct requirements for TCF3 and TCF12 during oligodendrocyte development in the mouse telencephalon. Glasgow, S. M. et al. Mutual antagonism between SOX10 and NFIA regulates diversification of glial lineages and glioma subtypes. & Guo, F. Transcription factor 7-like 2 (TCF7L2) regulates CNS myelination separating from its role in upstream oligodendrocyte differentiation. Zerdes, I. et al. STAT3 activity promotes programmed-death ligand 1 expression and suppresses immune responses in breast cancer. Song, T. L. et al. Oncogenic activation of the STAT3 pathway drives PD-L1 expression in natural killer/T-cell lymphoma. Linnerbauer, M. et al. PD-L1 positive astrocytes attenuate inflammatory functions of PD-1 positive microglia in models of autoimmune neuroinflammation. Filippone, A. et al. PD1/PD-L1 immune checkpoint as a potential target for preventing brain tumor progression. Butte, M. J., Keir, M. E., Phamduy, T. B., Sharpe, A. H. & Freeman, G. J. Programmed death-1 ligand 1 interacts specifically with the B7-1 costimulatory molecule to inhibit T cell responses. Expression of programmed-death receptor ligands 1 and 2 may contribute to the poor stimulatory potential of murine immature dendritic cells. Smith, P. et al. Schistosoma mansoni worms induce anergy of T cells via selective up-regulation of programmed death ligand 1 on macrophages. Curiel, T. J. et al. Blockade of B7-H1 improves myeloid dendritic cell-mediated antitumor immunity. Hwang, H. J. et al. Therapy-induced senescent cancer cells contribute to cancer progression by promoting ribophorin 1-dependent PD-L1 upregulation. Pan, W. et al. NLRP3 inflammasome upregulates PD-L1 in ovarian cancer and contributes to an immunosuppressive microenvironment. et al. Tumour cell-released autophagosomes promote lung metastasis by upregulating PD-L1 expression in pulmonary vascular endothelial cells in breast cancer. Kim, N. H. et al. Auranofin blocks interleukin-6 signalling by inhibiting phosphorylation of JAK1 and STAT3. & Silberberg, D. H. Expression of H-2 antigen on oligodendrocytes is induced by soluble factors from concanavalin A activated T cells. Suzumura, A., Silberberg, D. H. & Lisak, R. P. The expression of MHC antigens on oligodendrocytes: induction of polymorphic H-2 expression by lymphokines. Turnley, A. M., Miller, J. F. & Bartlett, P. F. Regulation of MHC molecules on MBP positive oligodendrocytes in mice by IFN-γ and TNF-α. Major histocompatibility complex heavy chain accumulation in the endoplasmic reticulum of oligodendrocytes results in myelin abnormalities. Itoh, T., Horiuchi, M. & Itoh, A. Interferon-triggered transcriptional cascades in the oligodendroglial lineage: a comparison of induction of MHC class II antigen between oligodendroglial progenitor cells and mature oligodendrocytes. Sziraki, A. et al. A global view of aging and Alzheimer's pathogenesis-associated cell population dynamics and molecular signatures in human and mouse brains. Kukanja, P. et al. Cellular architecture of evolving neuroinflammatory lesions and multiple sclerosis pathology. Siwek, W., Tehrani, S. S. H., Mata, J. F. & Jansen, L. E. T. Activation of clustered IFNγ target genes drives cohesin-controlled transcriptional memory. Establishment, maintenance, and recall of inflammatory memory. Stem cells expand potency and alter tissue fitness by accumulating diverse epigenetic memories. Lee, H. G. et al. Disease-associated astrocyte epigenetic memory promotes CNS pathology. Hinte, L. C. et al. Adipose tissue retains an epigenetic memory of obesity after weight loss. Wu, J. et al. Glial reactions and degeneration of myelinated processes in spinal cord gray matter in chronic experimental autoimmune encephalomyelitis. Bartosovic, M. & Castelo-Branco, G. Multimodal chromatin profiling using nanobody-based single-cell CUT&Tag. & Brattelid, T. PREPARE: guidelines for planning animal research and testing. Kilkenny, C., Browne, W. J., Cuthill, I. C., Emerson, M. & Altman, D. G. Improving bioscience research reporting: the ARRIVE guidelines for reporting animal research. FELASA Working Group on Revision of Guidelines for Health Monitoring of Rodents and Rabbits et al. FELASA recommendations for the health monitoring of mouse, rat, hamster, guinea pig and rabbit colonies in breeding and experimental units. Zheng, G. X. et al. Massively parallel digital transcriptional profiling of single cells. Stuart, T. et al. Comprehensive integration of single-cell data. & Satija, R. Single-cell chromatin state analysis with Signac. Rainer, J., Gatto, L. & Weichenberger, C. X. ensembldb: an R package to create and use Ensembl-based annotation resources. Fleming, S. J. et al. Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender. Zhang, Y. et al. Model-based analysis of ChIP–seq (MACS). McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. Ryan, L. & Mills, K. H. G. Sex differences regulate immune responses in experimental autoimmune encephalomyelitis and multiple sclerosis. Sex differences in the inflammatory response of the mouse DRG and its connection to pain in experimental autoimmune encephalomyelitis. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Efficient integration of heterogeneous single-cell transcriptomes using Scanorama. Marsh, S. E. et al. Dissection of artifactual and confounding glial signatures by single-cell sequencing of mouse and human brain. Amezquita, R. A. et al. Orchestrating single-cell analysis with Bioconductor. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. & Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. The nf-core framework for community-curated bioinformatics pipelines. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Chen, Y., Chen, L., Lun, A. T. L., Baldoni, P. & Smyth, G. K. edgeR v4: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Xie, Z. et al. Gene set knowledge discovery with Enrichr. Kharchenko, P. V., Silberstein, L. & Scadden, D. T. Bayesian approach to single-cell differential expression analysis. Grün, D., Kester, L. & van Oudenaarden, A. Validation of noise models for single-cell transcriptomics. We thank E. Parvisto, P. Samadi Tari, C. Zullian and M. Averstad at the Comparative Medicine Biomedicum Facility (KI) for their advice and help with experimentation. We thank M. Haeussler and B. Wick for preparing the University of California, Santa Cruz, Cell Browser entry for our dataset. We acknowledge support from the National Genomics Infrastructure in Stockholm funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council. Part of the computation/data handling was enabled by resources provided by the National Academic Infrastructure for Supercomputing in Sweden and the Swedish National Infrastructure for Computing at the Uppsala Multidisciplinary Center for Advanced Computational Science, partially funded by the Swedish Research Council through grant agreement numbers 2022-06725 and 2018-05973. Part of the computing was also performed in the Linnarsson group Monod Linux cluster at MBB-KI, and we thank P. Lönnerberg for maintenance and support. was supported by the China Scholarship Council. L.A.R.R.-K. was funded by an ECTRIMS postdoctoral fellowship and National MS Society (NMSS, USA, TA-2305-41342). Work in the research group of G.C.-B. was supported by the Swedish Research Council (project grant 2019-01360 and Distinguished Professor grant 2023-00324), the European Union (Horizon Europe Research and Innovation Programme/ERC Advanced grant SingleMS, 101096064), the Swedish Brain Foundation (FO2023-0032), Knut and Alice Wallenberg Foundation (grants 2019-0107 and 2019-0089 and Wallenberg Scholar grant 2023-0280), the Göran Gustafsson Foundation for Research in Natural Sciences and Medicine, the Swedish Society for Medical Research (SSMF, grant JUB2019), Ming Wai Lau Centre for Reparative Medicine and Karolinska Institutet. Open access funding provided by Karolinska Institute. These authors contributed equally: Chao Zheng, Bastien Hervé. Laboratory of Molecular Neurobiology, Department of Medical Biochemistry and Biophysics, Karolinska Institutet, Stockholm, Sweden Chao Zheng, Bastien Hervé, Mandy Meijer, Leslie Ann Rubio Rodríguez-Kirby, Petra Kukanja, Mukund Kabbe, Tony Jimenez-Beristain, Eneritz Agirre & Gonçalo Castelo-Branco Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC Berlin), Berlin, Germany Neuroimmunology Unit, Department of Clinical Neuroscience, Center for Molecular Medicine, Karolinska Institute, Stockholm, Sweden André Ortlieb Guerreiro Cacais & Tomas Olsson Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the methodological framework with input from M.M., L.A.R.R.-K., A.O.G.C., T.O. optimized the tissue dissociation protocol, collected tissues and conducted cell sorting and single-cell multiome experiments. wrote the paper with input from all authors. is CEO, shareholder, cofounder and board member at Nexus Epigenomics. is a shareholder and board member at Nexus Epigenomics. The other authors declare no competing interests. Nature Neuroscience thanks Ozgun Gokce, Martin Kerschensteiner and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Representative fluorescence-activated cell sorting (FACS) gating strategy with a sample from EAE peak stage. b, Quality control metrics after removing low quality cells of the multiome ATAC + gene expression sequencing data. c, d, UMAP with Louvain clustering algorithm based on gene expression (c) and chromatin accessibility (d) of 10x Genomics chromium multiome ATAC + gene expression. e,f, UMAP of the cells colored by condition on top of UMAP with all cells (in gray), based on gene expression data (e) and chromatin accessibility data (f). g, UMAP with Louvain clustering algorithm based on joint projection of gene expression and chromatin accessibility modalities. h, UMAP based on scRNA-seq data (left), scATAC-seq data (middle), and joint UMAP created by combining scRNA-seq and scATAC-seq nearest neighbors graphs (right), with lines connecting cells by single-cell barcodes across modalities. a, Normalized chromatin accessibility (left) and log2 expression (right) of representative MHC-I and II associated genes in each time point. b, Heatmaps of the expression (left) and chromatin accessibility (right) of MHC-I and -II genes at different stages of EAE in OPC (left), MOL2 (upper right), and MOL5/6 (lower left). MOL1 not included due to the low number of cells. c, Normalized chromatin accessibility of representative MHC-I (B2m) and MHC-II (H2-Ab1) genes at different time points (top) and ATAC-seq reads alignment of 50 individual random cells per condition for the same genes (bottom). a,b, Feature plots showing gene expression (a) and gene activity score (b) of markers for OLG (Sox10), OPC (Pdgfra), COP (Bmp4), and MOL (Mbp, Fosb, Klk6, Ptgds) populations. c, UMAP of label transfer predictions from matched scRNA-seq data. d, Label transfer prediction scores from literature cell types to annotated clusters. 0: n = 4270, 1:3971, 2:3912, 3:3822, 4:3818, 5:3813, 6:3760, 7:3679, 8:3471, 9:3442, 10:3415, 11:3392, 12:3343, 13:3291, 14:3251, 15:3158, 16:3048, 17:2965, 18:2927, 19:2881, 20:2814, 21:2811, 22:2768, 23:2626, 24:2508, 25:2464, 26:2385, 27:2382, 28:2333, 29:2260, 30:2241, 31:2225, 32:2180, 33:2161, 34:1938, 35:1841, 36:1793, 37:1702, 38:1683, 39:1554, 40:1334, 41:1093, 42:1073, 43:1002, 44:677, 45:542, 46:164. e, Relative expression level (qRT-PCR) of MHC-I (upper)/II (lower) genes with 1x and 2x IFN-γ treatments in Oli-neu cells. Data are presented as mean values ± s.e.m. n = 3 biologically independent cultures per condition, with each replicate representing a separate seeding and treatment of Oli-neu cells. f, Representative lumbar spinal cord section from EAE peak stage with white lines outline DAPI-dense areas (lesions) (left). g, Representative lumbar spinal cord section from EAE early, peak, and late stages, immunostained against GFP (SOX10) and immune-related protein (MHC-II). n = 3 biologically independent experiments per condition. a, Joint UMAP of OLG populations colored by sub-cell types, on top of UMAP with all cells (in gray). b, Heatmap of the scaled expression of differentially expressed genes between OLG sub-cell types (percentage of cells in a cluster expressing a gene superior to 50%, log2FC > 0.5 and adjusted p-value < 0.05), colored by sub-cell types and time points. c, Scaled expression level of immune response related genes grouped by function in OLG sub-cell types. OPC-α:n = 2134, OPC-β:289, OPC-γ:1072, COP:1002, MOL56-α:4706,MOL56-β:37969, MOL56-γ:5595, MOL56-δ:4166, MOL56-ε:4733, MOL56-ζ:6299, MOL56-η:164, MOL56-θ:3392, MOL56-ι:1093, MOL56-κ:10704, MOL2-α:10854, MOL2-β:8988, MOL2-γ:1841, MOL2-δ:6227, MOL2-ε:8413, MOL1:542. d, Percentage of cells with (red) or without (green) immune status identified by gene expression (left) and chromatin accessibility (right) in each OLG sub-cell type. e,f, Circos plot showing the number of cells with (black) or without (green) damage- and IFN-associated profiles, only IFN-associated profile (blue), only damage-associated profiles (gray) for OPC (left), MOL2 (middle), and MOL5/6 (right) from different time points at gene expression (e) and chromatin accessibility (f) levels (downsampled by time point). a Heatmaps of differentially expressed genes (log2FC > 1 and p-value adjusted < 0.01) (in brown) between different time points in OPC, with chromatin accessibility at promoter and gene body (in purple) and at enhancer regions (in green) of the same gene. The line plots represent the averages of gene expression (line in brown), the chromatin accessibility at promoter and genes body (line in purple), and chromatin accessibility at enhancer regions (line in green) of genes in different groups. b,c, Circos plots showing the number of cells with (red) or without (green) immune status identified with gene expression (b) and chromatin accessibility (c) from different time points in OPC (left), MOL2 (middle) and MOL56 (right) (downsampled by time points). d, Violin plots showing the expression of myelination related gene Plp1 at different stages in MOL5/6. n=CFA-Ctrl:43293,Early:30937,Peak:15248,Late:30705. e, f Upset plots showing differentially expressed genes overlaps between MOL2 over MOL5/6 (e) or MOL5/6 over MOL2 (f), within each time point (Log2 Fold Change > 1 and q-value < 0.01)). g,h, Connections between enriched GO terms (p.adjust < 0.05) of genes with both differential expression and chromatin accessibility among different disease stages in MOL2 (g) and MOL5/6 (h). i, Heatmaps of differential chromatin accessibility at promoter and gene body (log2FC > 1 and p-value adjusted < 0.01) (in purple) between different time points in OPCs, of gene expression (in brown) and chromatin accessibility at enhancer regions (in green) of the same gene. Other features are such as in a).j,k, Chromatin accessibility normalized tracks of representative immune related genes (Irgm1, Igtp and Gbp7) with an important increase at peak stage in MOL2 (j) and MOL5/6 (k). The violin plot shows Gbp3 log2 expression in OLG. c, Heatmaps of normalized and scaled DORC score of differentially variable gene scores (log2FC > 1 and p-value adjusted < 0.01) at different stages, 52 genes in OPC. Differential DORCs and associated gene ontologies are shown in Supplementary Table 3, 4. d, Violin plots of DORCs score of Hoxb genes in OLG sub-cell types. OPC-α: n = 2134, OPC-β:289, OPC-γ:1072, COP:1002, MOL56-α:4706, MOL56-β:37969, MOL56-γ:5595, MOL56-δ:4166, MOL56-ε:4733, MOL56-ζ:6299, MOL56-η:164, MOL56-θ:3392, MOL56-ι:1093, MOL56-κ:10704, MOL2-α:10854, MOL2-β:8988, MOL2-γ:1841, MOL2-δ:6227, MOL2-ε:8413, MOL1:542. e, Normalized chromatin accessibility of Hoxa (upper) and Hoxb (lower) genes in MOL2. a,b, UMAPs of MOL2 (a) and MOL5/6 (b) colored by gene state assigned by MultiVelo, model 1 (left) (M1, chromatin is closing before transcriptional repression) and model 2 (right) (M2, chromatin is closing after transcriptional repression). c, MultiVelo UMAP of OPC colored by time points. d, Depiction of chromatin accessibility (top), moments of unspliced RNA (middle) and spliced RNA (bottom), calculated by MultiVelo for B2m, H2-D1, H2-K1, Stat1 and Stat2 in OPC. a,b,c, Activity score of selected transcription factor at different stages in MOL2 (a), MOL5/6 (b), and OPC (c). x axis: transcription factor activity score, y axis: transcription factors grouped by time point at which transcription factor activity is the highest. d, Activity score of selected transcription factor at different stages in MOL2 and MOL5/6. e, Relative expression level of Stat3 and Cd274 in OPCs treated with DMSO or Stat3 inhibitor Auranofin after IFN- γ treatment, measured by qPCR. a, Normalized tracks of chromatin accessibility (left) and expression (right) of Cd274 (PD-L1) in OPC, MOL2, and MOL5/6. b, Circos plot showing the prediction result (upper semicircle) and downsampled ground truth (bottom semicircle) of the sex. c, Bar plot showing the percentage of cells predicted to be male (blue) or female (red) in the samples used to build the sex prediction model. These samples came from different stages of EAE and only contained cells from one gender. d, Joint UMAP from the weighted nearest neighbors graph of scRNA-seq and scATAC-seq modalities colored by sex predition. e, Circos plot showing the proportion of predicted male and female cells in each cell type, downsampled by sex. f, Heatmap showing the expression of differentially expressed genes (log2FC > 1 and pvalue adjusted < 0.01) between male and female in OPC, COP, MOL2, MOL5/6. g, Circos plot showing the number of cells with (red) or without (green) immune status for predicted male and female cells (downsampled by sex). h, Heatmaps of the expression (left) and chromatin accessibility (right) of MHC-I (upper) and -II (lower) genes in predicted male and female cells at different stages in OLG. i, Correlation matrix showing the Pearson correlation scores of MG between different CFA-Ctrl replicates. j, Dot plot showing differentially expressed genes (log2FC > 1 and pvalue adjusted < 0.01) between Naïve-ctrl and CFA-Ctrl in MOL. Differential gene expression and chromatin accessibility, IFNγ doses, bulk RNA-seq and bulk ATAC-seq. Differential modality features between time points. GO of differential modalities between time points. GO of differential modalities between MOL5/MOL6 and MOL2. Differentially expressed genes and accessibility in OLG populations. GO of differentially expressed genes and accessibility in OLG populations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Distinct transcriptomic and epigenomic responses of mature oligodendrocytes during disease progression in a mouse model of multiple sclerosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02912-y'>A multi-omics molecular landscape of 30 tissues in aging female rhesus macaques</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 10:45:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article A systematic investigation of aging patterns across virtually all major tissues in nonhuman primates, our evolutionarily closest relatives, can provide valuable insights into tissue aging in humans, which is still elusive largely due to the difficulty in sampling. Here, we generated and analyzed multi-omics data, including transcriptome, proteome and metabolome, from 30 tissues of 17 female rhesus macaques (Macaca mulatta) aged 3–27 years. We found that certain molecular features, such as increased inflammation, are consistent across tissues and align with findings in mice and humans. We further revealed that tissue aging in macaques is asynchronous and can be classified into two distinct types, with one type exhibiting more pronounced aging degree, likely associated with decreased mRNA translation efficiency, and predominantly contributing to whole-body aging. This work provides a comprehensive molecular landscape of aging in nonhuman primate tissues and links translation efficiency to tissue-specific aging. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The omics data are freely and publicly available. Level-2 data, including transcriptome raw counts and log2 peak areas for the proteome and metabolome, are accessible on Figshare at https://doi.org/10.6084/m9.figshare.26963386 (ref. For the level-1 raw data, the raw FASTQ files of the transcriptome have been deposited in the Genome Sequence Archive at the National Genomics Data Center65,66 (GSA CRA026248), publicly accessible at https://ngdc.cncb.ac.cn/gsa. The raw LC–MS/MS data for the proteome have been deposited to the ProteomeXchange Consortium via the PRIDE67 partner repository with the dataset identifier PXD066108 and also deposited to OMIX66 (https://ngdc.cncb.ac.cn/omix, accession no. The raw LC–MS/MS data for the metabolome are deposited in OMIX (https://ngdc.cncb.ac.cn/omix, accession no. The code to reproduce the results in this study is available on Figshare at https://doi.org/10.6084/m9.figshare.26963386 (ref. 64) or GitHub at https://github.com/GonghuaLi/Macaca_30tissue_aging. Supplementary Data 10 provides a comprehensive PDF document generated from the coding of R Markdown (RMD) file, containing all code and results. López-Otín, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. The hallmarks of aging. Kaeberlein, M., Rabinovitch, P. S. & Martin, G. M. Healthy aging: the ultimate preventative medicine. Lopez-Otin, C., Blasco, M. A., Partridge, L., Serrano, M. & Kroemer, G. Hallmarks of aging: an expanding universe. Schaum, N. et al. Ageing hallmarks exhibit organ-specific temporal signatures. & Wyss-Coray, T. Asynchronous, contagious and digital aging. Zhang, W. et al. SIRT6 deficiency results in developmental retardation in cynomolgus monkeys. Colman, R. J. Non-human primates as a model for aging. Anderson, R. M. & Colman, R. J. Prospects and perspectives in primate aging research. Warren, W. C. et al. Sequence diversity analyses of an improved rhesus macaque genome enhance its biomedical utility. Wang, S. et al. Single-cell transcriptomic atlas of primate ovarian aging. Liang, D. et al. Cellular and molecular landscape of mammalian sinoatrial node revealed by single-cell RNA sequencing. Zhang, W. et al. A single-cell transcriptomic landscape of primate arterial aging. Li, M.-L. et al. 547 transcriptomes from 44 brain areas reveal features of the aging brain in nonhuman primates. Wang, S. et al. Deciphering primate retinal aging at single-cell resolution. Huang, D. et al. A single-nucleus transcriptomic atlas of primate testicular aging reveals exhaustion of the spermatogonial stem cell reservoir and loss of Sertoli cell homeostasis. Li, J. et al. A single-cell transcriptomic atlas of primate pancreatic islet aging. Jing, Y. et al. Single-nucleus profiling unveils a geroprotective role of the FOXO3 in primate skeletal muscle aging. Wolters, S. & Schumacher, B. Genome maintenance and transcription integrity in aging and disease. Zierer, J., Menni, C., Kastenmüller, G. & Spector, T. D. Integration of ‘omics' data in aging research: from biomarkers to systems biology. Consortium, G. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Melé, M. et al. The human transcriptome across tissues and individuals. Jiang, L. et al. A quantitative proteome map of the human body. Palmer, D., Fabris, F., Doherty, A., Freitas, A. & de Magalhães, J. P. Ageing transcriptome meta-analysis reveals similarities and differences between key mammalian tissues. Rozowsky, J. et al. The EN-TEx resource of multi-tissue personal epigenomes & variant-impact models. Keele, G. R. et al. Global and tissue-specific aging effects on murine proteomes. Shen, X. et al. Nonlinear dynamics of multi-omics profiles during human aging. Contrepois, K. et al. Molecular choreography of acute exercise. Kumar, L. & Futschik, M. E. Mfuzz: a software package for soft clustering of microarray data. Avelar, R. A. et al. A multidimensional systems biology analysis of cellular senescence in aging and disease. Battle, A. et al. Impact of regulatory variation from RNA to protein. Chick, J. M. et al. Defining the consequences of genetic variation on a proteome-wide scale. Kim, M.-S. et al. A draft map of the human proteome. Li, G. H. et al. System-level metabolic modeling facilitates unveiling metabolic signature in exceptional longevity. Nucleic Acids Res. Huang, K. et al. AgeAnnoMO: a knowledgebase of multi-omics annotation for animal aging. Nucleic Acids Res. A. et al. Inhibition of IL-11 signalling extends mammalian healthspan and lifespan. Colman, R. J., McKiernan, S. H., Aiken, J. M. & Weindruch, R. Muscle mass loss in rhesus monkeys: age of onset. Nilwik, R. et al. The decline in skeletal muscle mass with aging is mainly attributed to a reduction in type II muscle fiber size. Roy, R. R. et al. Fiber size and myosin phenotypes of selected rhesus hindlimb muscles after a 14-day spaceflight. Ohyama, Y., Redheuil, A., Kachenoura, N., Ambale Venkatesh, B. Imaging insights on the aorta in aging. Rezzani, R., Nardo, L., Favero, G., Peroni, M. & Rodella, L. F. Thymus and aging: morphological, radiological, and functional overview. Gingold, H. & Pilpel, Y. Determinants of translation efficiency and accuracy. Tahmasebi, S., Sonenberg, N., Hershey, J. W. & Mathews, M. B. Protein synthesis and translational control: a historical perspective. Cold Spring Harb. Wei, Y. N. et al. Transcript and protein expression decoupling reveals RNA binding proteins and miRNAs as potential modulators of human aging. Kelmer Sacramento, E. et al. Reduced proteasome activity in the aging brain results in ribosome stoichiometry loss and aggregation. Gerdes Gyuricza, I. et al. Genome-wide transcript and protein analysis highlights the role of protein homeostasis in the aging mouse heart. Hu, Z. et al. Ssd1 and Gcn2 suppress global translation efficiency in replicatively aged yeast while their activation extends lifespan. Syntichaki, P., Troulinaki, K. & Tavernarakis, N. Protein synthesis is a novel determinant of aging in Caenorhabditis elegans. Gerashchenko, M. V., Peterfi, Z., Yim, S. H. & Gladyshev, V. N. Translation elongation rate varies among organs and decreases with age. Nucleic Acids Res. Yu, L. et al. IgG is an aging factor that drives adipose tissue fibrosis and metabolic decline. de Magalhaes, J. P., Curado, J. & Church, G. M. Meta-analysis of age-related gene expression profiles identifies common signatures of aging. Liu, X. et al. Clusterin transduces Alzheimer-risk signals to amyloidogenesis. Perez-Jimenez, M. M. et al. Steroid hormones sulfatase inactivation extends lifespan and ameliorates age-related diseases. Benonisson, H. et al. CD3-bispecific antibody therapy turns solid tumors into inflammatory sites but does not install protective memory. Horai, Y., Kakimoto, T., Takemoto, K. & Tanaka, M. Quantitative analysis of histopathological findings using image processing software. Vijayashree, R., Aruthra, P. & Rao, K. R. A comparison of manual and automated methods of quantitation of oestrogen/progesterone receptor expression in breast carcinoma. Wang, S., Yang, D. M., Rong, R., Zhan, X. & Xiao, G. Pathology image analysis using segmentation deep learning algorithms. Jiao, X. et al. Design mechanism and property of the novel fluorescent probes for the identification of Microthrix parvicella in situ. Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. The RIN: an RNA integrity number for assigning integrity values to RNA measurements. Zhao, Y. et al. TPM, FPKM, or normalized counts? A comparative study of quantification measures for the analysis of RNA-seq data from the NCI patient-derived models repository. Zhao, S., Ye, Z. & Stanton, R. Misuse of RPKM or TPM normalization when comparing across samples and sequencing protocols. Zhou, Y. et al. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Li, G.-H. & Kong, Q.-P. A multi-omics molecular landscape of 30 tissues in aging female rhesus macaques (Macaca mulatta). Chen, T. et al. The Genome Sequence Archive family: toward explosive data growth and diverse data types. Database resources of the National Genomics Data Center, China National Center for Bioinformation in 2025. The PRIDE database at 20 years: 2025 update. This work was supported by National Key R&D Program of China (2023YFC3603400 to Q.-P.K. ), National Natural Science Foundation of China (82430049 to Q.-P.K. ), the CAS Project for Young Scientists in Basic Research (YSBR-076 to Q.-P.K. ), Major Special Projects of Yunnan province (2018ZF007), the project entitled ‘Transformation of sub totipotent stem cells based on the tree shrew model of multiple organ dysfunction syndrome' (SYDW[2020]19), West Light Foundation (to F.-H.X.) of the Chinese Academy of Sciences, Yunnan Province Kunming Medical University Joint Special Key Project (202301AY070001-034), Key Technologies and Translational Research on Clinical-Grade Umbilical Cord Mesenchymal Stem Cell Products for Reversing Aging, Yunnan Applied Basic Research Project (202401AW070011, 202101AS070058, 202201AS070080 and 202301AT070281), Reserve Talent Project of Young and Middle-aged Academic and Technical Leaders in Yunnan Province (202305AC160029), High-level Talent Promotion and Training Project of Kunming (Spring City Plan; 2020SCP001 to Q.-P.K. ), Yunnan Revitalization Talent Support Program Yunling Scholar Project (Q.-P.K. ), Yunnan Revitalization Talent Support Program Young Talent Project (G.-H.L. ), Ningbo Yongjiang Talent Introduction Programme (F.-H.X.) and Yunnan Revitalization Talent Support Program Top team (202505AT350003 and 202405AS350022). We thank Novogene for technical assistance in the transcriptomics, proteomics and metabolomics data acquisition, and C. Watts for help in proofreading the manuscript. These authors contributed equally: Gong-Hua Li, Xiang-Qing Zhu, Fu-Hui Xiao. State Key Laboratory of Genetic Evolution & Animal Models, National Resource Center for Non-Human Primates, Kunming Primate Research Center, Research Facility for Phenotypic & Genetic Analysis of Model Animals (Primate Facility), Key Laboratory of Healthy Aging Research of Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, China Gong-Hua Li, Fu-Hui Xiao, Longbao Lv, Fan-Qian Yin, Ming-Xia Ge & Qing-Peng Kong Stem Cells and Immune Cells Biomedical Techniques Integrated Engineering Laboratory of State and Regions, Cell Therapy Technology Transfer Medical Key Laboratory of Yunnan Province, Basic Medical Laboratory, 920th Hospital of the PLA Joint Logistics Support Force, Kunming, China Xiang-Qing Zhu, Xilong Zhao, Le Chang, Qiang Wang, Jing Zhao, Chuan Tian, Zian Li, Guangping Ruan, Rongqing Pang, Jing Gao, Lihua Ma & Xing-Hua Pan Department of Cardiology, The Affiliated Lihuili Hospital of Ningbo University, School of Medicine, Ningbo University, Ningbo, China CAS Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the study. performed the multi-omics analysis. conducted the experimental validation. performed the statistical analysis. performed H&E staining and scoring. performed immunofluorescence and scoring. drafted and revised the manuscript. All authors read and approved the final manuscript. Correspondence to Xing-Hua Pan or Qing-Peng Kong. The authors declare no competing interests. Nature Methods thanks João Pedro de Magalhães, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Madhura Mukhopadhyay, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Aging-associated mRNAs and proteins overlapping across 30 tissues were identified using a false discovery rate (FDR) < 0.05 and |βp|> 0.008. Here, βp represents the age effect size derived from a pooled linear regression model (molecular abundance ~ age + tissue), while βt represents the age effect size from tissue-specific linear models (molecular abundance ~ age per tissue). For each individual tissue, statistical significance was assessed using two-sided t-tests on the age coefficients derived from the tissue-specific linear models, and these P values were not adjusted for multiple comparisons. Exact P values are provided in Supplementary Data 1-2. The P values of <0.05, <0.01, and <0.001 in each tissue are denoted by *, **, and ***, respectively. A linear regression model was applied to each tissue to identify aging-associated mRNAs and proteins, selecting those with P < 0.05 and an absolute age effect size (|βt|) > 0.04. To ensure tissue specificity, candidate molecules were further required to be significant (P < 0.05) in no more than two tissues. a-b, the upregulated (a) and downregulated (b) tissue-specific aging-related mRNAs. c-d, the upregulated (c) and downregulated (d) tissue-specific aging-related proteins. Top one marker for each tissue is shown to the right of the heatmap. a-c, Volcano plots of aging-related mRNAs in mice (a), macaques (b), and humans (c). d-e, Venn plots comparing upregulated (d) and downregulated (e) mRNAs among mice, macaques, and humans. Overlapped upregulated aging-related mRNAs across the three species include B2M, ANXA1 and CSF1. Overlapped downregulated aging-related mRNAs include COL3A1, PIK3R1, and PANK1. f-g, Volcano plots of aging-related proteins in mice (f) and macaques (g). Note that the significance of protein data in mice is directly derived from Gregory's study (Cell Reports, 2023). h, Venn plot of significantly changed aging-related proteins. Among these, 28 overlapped upregulated proteins between mice and macaques include IGHM, PZP, IGKC, ITIH4, JCHAIN, KNG1, F2, AGT and CP, while three downregulated proteins are NRAS, FKBP4 and HSPH1. a, Tissue-specific aging trajectories for each cluster of proteins and metabolites based on normalized molecular abundances. b, Heatmap of protein enrichment for each cluster. Statistical significance for GO enrichment was evaluated using one-sided hypergeometric tests, and P values were adjusted for multiple comparisons using the Benjamini-Hochberg method to control the false discovery rate (FDR). GO terms with adjusted P < 1×10⁻⁸ are in blue. c, Heatmap of hierarchical clustering based on tissue-specific trajectories of proteins and metabolites. d, Scatter plot of viability and molecular alteration amplitude (MAA) in 30 tissues for each cluster. MAA was calculated as the change in average Z-score of molecular abundance in each cluster between the end point (elderly group) and starting point (juvenile group). Viability was determined as the average standard deviation of the molecular trajectories within each cluster. e, Heatmap of MAA for each cluster across tissues. f, K-means consensus clustering of protein and metabolite trajectories, setting k-values from 2 to 6. g, Tissue clustering by three different methods based on the aging trajectories of proteins and metabolites. a, Whole-body aging trajectories of mRNAs based on normalized gene expression. b, Tissue-specific aging trajectories of mRNAs in 30 tissues for each cluster. c, Heatmap of hierarchical clustering based on combined tissue-specific trajectories, incorporating both mRNA trajectories and protein–metabolite trajectories. d, Heatmap of MAA for each cluster across tissues from the combined tissue-specific trajectories. e, K-means consensus clustering of aging trajectories of the combined tissue-specific trajectories with k-values ranging from 2 to 6. f, Tissue clustering by three different methods based on the aging trajectories of mRNAs, proteins and metabolites. a, Immunofluorescence staining of p16 and p21 in seven representative Type I tissues, including spleen, thymus, aortic arch, ovary, thyroid gland, stomach, and kidney, in juvenile, young, and elderly groups. p16 and p21 are labeled in green and red, respectively. b-c, p16 (b) and p21 (c) area density in different groups in seven representative Type I tissues. Data are presented as mean ± SD with overlaid individual data points (dot plots). n = 5 sections per group based on availability, derived from multiple sections per tissue sample (see Methods). Statistical comparisons were performed using two-sided t-tests. Exact P values are provided in Supplementary Table 1. a, Immunofluorescence staining of p16 and p21 in five representative Type II tissues, including muscle, pancreas, liver, lung, and back skin in juvenile, young, and elderly groups. p16 and p21 are labeled in green and red, respectively. b-c, p16 (b) and p21 (c) area density in different groups in five representative Type II tissues. Data are presented as mean ± SD with overlaid individual data points (dot plots). n = 5 sections per group based on availability, derived from multiple sections per tissue sample (see Methods). Statistical comparisons were performed using two-sided t-tests. Exact P values are provided in Supplementary Table 1. Histological differences among juvenile, young, and elderly groups in seven representative Type I tissues (spleen, thymus, aortic arch, stomach, kidney, ovary, and thyroid gland) using hematoxylin and eosin (H&E) staining. The low-power microscope partial image (marked by the black box) is magnified on the right. Histograms on the far right depict the numbers of functional cells in each tissue for the three age groups. Data are presented as mean ± SD with overlaid individual data points (dot plots). n = 5 sections per group based on availability, derived from multiple sections per tissue sample (see Methods). Statistical comparisons were performed using two-sided t-tests. Exact P values are provided in Supplementary Table 1. Histological differences among juvenile, young, and elderly groups in five representative Type II tissues (liver, lung, muscle, back skin, and pancreas) using hematoxylin and eosin (H&E) staining. Data are presented as mean ± SD with overlaid individual data points (dot plots). n = 5 sections per group based on availability, derived from multiple sections per tissue sample (see Methods). Statistical comparisons were performed using two-sided t-tests. Exact P values are provided in Supplementary Table 1. a, Design and algorithm for estimating mRNA translation efficiency by integration of transcriptomes and proteomes. We hypothesize that, under a steady-state, protein concentration is stable, thus protein synthesis from mRNA is equal to protein degradation from existing proteins. In other words, the protein-to-mRNA ratio equals the mRNA translation efficiency-to-protein degradation (TED) ratio. Changes in TED (cTED) with aging were calculated using a linear regression model (TED ~ age), based on its age effect size (βt). b, Distribution of cTED in different tissues. Type I, II, and undefined tissues are in tomato, blue, and gold, respectively. Aging-associated mRNAs in each tissue. Aging-associated proteins in each tissue. Aging-associated metabolites in each tissue. Common aging-associated mRNAs across 30 tissues. Common aging-associated proteins across 30 tissues. Common aging-associated metabolites across 30 tissues. Tissue-specific aging-associated mRNAs for each tissue. Tissue-specific aging-associated proteins for each tissue. Tissue-specific aging-associated metabolites for each tissue. Source code and scripts used for data analysis and figure generation, along with the corresponding outputs to reproduce the results. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Li, GH., Zhu, XQ., Xiao, FH. et al. A multi-omics molecular landscape of 30 tissues in aging female rhesus macaques. Version of record: 17 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41565-025-02059-z'>-to-methanol electrocatalysis in acidic media via microenvironment-tuned cobalt phthalocyanine</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 10:17:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Nanotechnology (2025)Cite this article Electrosynthesis of value-added chemicals in strong acids can mitigate carbon loss and the operational cost of CO2 reduction reaction (CO2RR). However, molecular catalysis for CO2RR is typically conducted in neutral or alkaline environments. CO2RR in acidic media is challenged by the scarcity of catalyst candidates, competitive hydrogen evolution and slow product formation. Here we report a locally ionic yet simultaneously hydrophobic and aerophilic layered structure that modulates the microenvironment surrounding cobalt phthalocyanine (CoPc) molecular catalysts, enabling efficient, multielectron CO2RR in acidic media. Experiment and theoretical modelling reveal that the polarized electrostatic field arising from the cationic groups suppresses hydronium migration. Concurrently, the van der Waals forces between the reactant gas and alkyl groups improve local CO availability, combining to achieve a methanol partial current density of 132 mA cm−2 with 62% selectivity at a pH of ~1 and –1.37 VRHE for CoPc, exceeding previous reports on neutral or alkaline electrolytes. The improved CO coverage also enables the detection of *CHO and *CO intermediates from in situ spectroscopy. We validate our strategy on various molecules, which champion the efficient inhibition of hydrogen evolution and improved CO2RR partial current density in acidic media. CoPc-based layered structure with similar ionic, hydrophobic and aerophilic interfaces also yields comparable methanol productivity. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data are available from the corresponding authors upon reasonable request. Source data are available via Figshare at https://doi.org/10.6084/m9.figshare.29591042 and are provided with this paper. Kibria, M. G. et al. Electrochemical CO2 reduction into chemical feedstocks: from mechanistic electrocatalysis models to system design. Google Scholar Zhao, Q. et al. Selective etching quaternary MAX phase toward single atom copper immobilized mxene (Ti3C2Clx) for efficient CO2 electroreduction to methanol. ACS Nano 15, 4927–4936 (2021). Google Scholar Torbensen, K. et al. Molecular catalysts boost the rate of electrolytic CO2 reduction. ACS Energy Lett. Google Scholar Bonin, J., Maurin, A. & Robert, M. Molecular catalysis of the electrochemical and photochemical reduction of CO2 with Fe and Co metal-based complexes. Recent advances. Google Scholar Wu, Y., Jiang, Z., Lu, X., Liang, Y. & Wang, H. Domino electroreduction of CO2 to methanol on a molecular catalyst. Google Scholar Boutin, E. et al. Aqueous electrochemical reduction of carbon dioxide and carbon monoxide into methanol with cobalt phthalocyanine. Google Scholar Rooney, C. L. et al. Active sites of cobalt phthalocyanine in electrocatalytic CO2 reduction to methanol. Google Scholar Li, J. et al. Mechanism-guided realization of selective carbon monoxide electroreduction to methanol. Google Scholar Boutin, E., Salamé, A., Merakeb, L., Chatterjee, T. & Robert, M. On the existence and role of formaldehyde during aqueous electrochemical reduction of carbon monoxide to methanol by cobalt phthalocyanine. Google Scholar Ren, X. et al. In-situ spectroscopic probe of the intrinsic structure feature of single-atom center in electrochemical CO/CO2 reduction to methanol. Google Scholar Ding, J. et al. Atomic high-spin cobalt(II) center for highly selective electrochemical CO reduction to CH3OH. Su, J. et al. Strain enhances the activity of molecular electrocatalysts via carbon nanotube supports. Yao, L. et al. Unlocking the potential for methanol synthesis via electrochemical CO2 reduction using CoPc-based molecular catalysts. ACS Nano 18, 21623–21632 (2024). Cheon, S., Li, J. & Wang, H. In situ generated CO enables high-current CO2 reduction to methanol in a molecular catalyst layer. Zhu, Q. et al. The solvation environment of molecularly dispersed cobalt phthalocyanine determines methanol selectivity during electrocatalytic CO2 reduction. Yu, S. et al. CO2-to-methanol electroconversion on a molecular cobalt catalyst facilitated by acidic cations. Singh, A. et al. Molecular electrochemical catalysis of CO-to-formaldehyde conversion with a cobalt complex. Hutchison, P. et al. Proton-coupled electron transfer mechanisms for CO2 reduction to methanol catalyzed by surface-immobilized cobalt phthalocyanine. Erick Huang, J. et al. CO2 electrolysis to multicarbon products in strong acid. Ma, Z. et al. CO2 electroreduction to multicarbon products in strongly acidic electrolyte via synergistically modulating the local microenvironment. Gu, J. et al. Modulating electric field distribution by alkali cations for CO2 electroreduction in strongly acidic medium. Monteiro, M. C. O. et al. The role of cation acidity on the competition between hydrogen evolution and CO2 reduction on gold electrodes. Chen, X. et al. Electrochemical CO2-to-ethylene conversion on polyamine-incorporated Cu electrodes. Sun, M., Cheng, J. & Yamauchi, M. Gas diffusion enhanced electrode with ultrathin superhydrophobic macropore structure for acidic CO2 electroreduction. Xing, Z., Hu, X. & Feng, X. Tuning the microenvironment in gas-diffusion electrodes enables high-rate CO2 electrolysis to formate. ACS Energy Lett. Feng, S. et al. Stabilizing *CO2 intermediates at the acidic interface using molecularly dispersed cobalt phthalocyanine as catalysts for CO2 reduction. Fan, M. et al. Cationic-group-functionalized electrocatalysts enable stable acidic CO2 electrolysis. Li, G. et al. Backbone engineering of polymeric catalysts for high-performance CO2 reduction in bipolar membrane zero-gap electrolyzer. Zhang, Q. et al. A covalent molecular design enabling efficient CO2 reduction in strong acids. Song, Y. et al. Atomically thin, ionic-covalent organic nanosheets for stable, high performance carbon dioxide electroreduction. Song, Y. et al. Ultrathin, cationic covalent organic nanosheets for enhanced CO2 electroreduction to methanol. Yao, Y., Delmo, E. P. & Shao, M. The electrode/electrolyte interface study during the electrochemical CO2 reduction in acidic electrolytes. Bernasconi, F. et al. Operando observation of (bi)carbonate precipitation during electrochemical CO2 reduction in strongly acidic electrolytes. ACS Catal. Su, Y. et al. Exploring the impact of Nafion modifier on electrocatalytic CO2 reduction over Cu catalyst. Energy Chem. Wang, Y. H. et al. In situ Raman spectroscopy reveals the structure and dissociation of interfacial water. Li, C. Y. et al. In situ probing electrified interfacial water structures at atomically flat surfaces. Huang, B. et al. Cation-dependent interfacial structures and kinetics for outer-sphere electron-transfer reactions. Wang, Y. et al. Strong hydrogen-bonded interfacial water inhibiting hydrogen evolution kinetics to promote electrochemical CO2 reduction to C2+. ACS Catal. Ohlin, C. A., Dyson, P. J. & Laurenczy, G. Carbon monoxide solubility in ionic liquids: determination, prediction and relevance to hydroformylation. Yao, Y. et al. A surface strategy boosting the ethylene selectivity for CO2 reduction and in situ mechanistic insights. Zhu, S., Jiang, B., Cai, W., Bin & Shao, M. Direct observation on reaction intermediates and the role of bicarbonate anions in CO2 electrochemical reduction reaction on Cu surfaces. Wang, H., Zhu, J., Ren, X., Tong, Y. & Chen, P. Heterogeneous cobalt phthalocyanine/sulfur-modified hollow carbon sphere for boosting CO2 electroreduction and Zn-CO2 batteries. Lyu, F. et al. Pre-activation of CO2 at cobalt phthalocyanine-Mg(OH)2 interface for enhanced turnover rate. Have, I. C. T. et al. Uncovering the reaction mechanism behind CoO as active phase for CO2 hydrogenation. Wohar, M. M. & Jagodzinski, P. W. Infrared spectra of H2CO, H213CO, D2CO, and D213CO and anomalous values in vibrational force fields. Monteiro, M. C. O., Jacobse, L. & Koper, M. T. M. Understanding the voltammetry of bulk CO electrooxidation in neutral media through combined SECM measurements. Watkins, N. B. et al. Hydrodynamics change Tafel slopes in electrochemical CO2 reduction on copper. ACS Energy Lett. Latiff, N. M. et al. Carbon based copper(II) phthalocyanine catalysts for electrochemical CO2 reduction: effect of carbon support on electrocatalytic activity. Zhang, X. et al. Highly selective and active CO2 reduction electrocatalysts based on cobalt phthalocyanine/carbon nanotube hybrid structures. Thompson, A. P. et al. LAMMPS—a flexible simulation tool for particle-based materials modeling at the atomic, meso, and continuum scales. Cundary, T. R. & Gordon, M. S. UFF, a full periodic table force field for molecular mechanics and molecular dynamics simulations. Rappe, A. K. & Goddard, W. A. III Charge equilibration for molecular dynamics simulations. Download references acknowledges support from the National Natural Science Foundation of China (22522509), Guangdong Basic and Applied Basic Research Fund (2024A1515030164), Hong Kong Research Grant Council (11310624, 11309723), the Shenzhen Science and Technology Program (JCYJ20220818101204009) and the State Key Laboratory of Marine Environmental Health (SKLMP/SCRF/0060). acknowledges support from the National Key Research and Development Program of China (2023YFB381001), National Natural Science Fundation of China (52333007), Key-Area Research and Development Program of Guangdong Province (2024B0101040001), Shenzhen Key Laboratory of Functional Aggregate Materials (ZDSYS20211021111400001), the Science Technology Innovation Commission of Shenzhen Municipality (KQTD20210811090142053 and JCYJ20220818103007014) and the Innovation and Technology Commission (ITC-CNERC14SC01). X-ray absorption spectroscopy measurements were carried out at the Shanghai Synchrotron Radiation Facility of BL14W1 (https://cstr.cn/31124.02.SSRF.BL14W1). III and W.A.G. III acknowledge support from the Liquid Sunlight Alliance, which is supported by the US Department of Energy, Office of Science, Office of Basic Energy Sciences, Fuels from Sunlight Hub, under award number DE-SC0021266. acknowledges support from the China Experience Fund and the Stephen Slavens Faculty Scholar Endowment Fund from Oregon State University. These authors contributed equally: Yun Song, Charles B. Musgrave III. Department of Chemistry and State Key Laboratory of Marine Environmental Health, City University of Hong Kong, Hong Kong, China Yun Song, Jianjun Su, Libei Huang, Weihua Guo, Yong Liu, Geng Li, Yinger Xin, Qiang Zhang, Mingming He & Ruquan Ye City University of Hong Kong Shenzhen Research Institute, Shenzhen, China Yun Song, Jianjun Su, Libei Huang, Weihua Guo, Yong Liu, Geng Li, Yinger Xin, Qiang Zhang, Mingming He & Ruquan Ye Materials and Process Simulation Center, California Institute of Technology, Pasadena, CA, USA Charles B. Musgrave III & William A. Goddard III Guangdong Provincial Key Laboratory of Information Photonics Technology, School of Material and Energy, Guangdong University of Technology, Guangzhou, China Xing Feng Key Laboratory of Polymer Ecomaterials, Changchun Institute of Applied Chemistry, Chinese Academy of Sciences, Changchun, China Can Liao & Shunjie Liu Department of Chemistry and the Hong Kong Branch of Chinese National Engineering Research Center for Tissue Restoration and Reconstruction, The Hong Kong University of Science and Technology, Hong Kong, China Ryan Tsz Kin Kwok, Jacky W. Y. Lam & Ben Zhong Tang School of Chemical, Biological, and Environmental Engineering, Oregon State University, Corvallis, OR, USA Kai Shen Choong & Zhenxing Feng Guangdong Basic Research Center of Excellence for Aggregate Science, School of Science and Engineering, The Chinese University of Hong Kong (Shenzhen), Shenzhen, China Ben Zhong Tang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar proposed, designed and supervised the project. performed the catalyst preparation, most of the characterizations and the catalytic tests. III and W.A.G. III carried out the MD calculations and analysis. contributed to the data analysis. helped with the solid-state 13C NMR experiments. analysed the X-ray data. III and B.Z.T. supervised the research. All authors contributed to the discussion and manuscript preparation. Correspondence to Ben Zhong Tang, William A. Goddard III or Ruquan Ye. declare that two provisional patents titled ‘Methods for performing electrochemical reactions in strong acid for molecular catalyst' have been filed: one in the United States (application number 63/676,895) and one in China (application number 2025110457399). The other authors declare no competing interests. Nature Nanotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. 1–70, Notes 1–5 and Refs. Statistical source data. Spectroscopic source data. Statistical and spectroscopic source data. Statistical and spectroscopic source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Song, Y., Musgrave, C.B., Su, J. et al. Efficient CO2-to-methanol electrocatalysis in acidic media via microenvironment-tuned cobalt phthalocyanine. Download citation Received: 12 December 2024 Accepted: 08 October 2025 Published: 17 November 2025 Version of record: 17 November 2025 DOI: https://doi.org/10.1038/s41565-025-02059-z Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Nanotechnology ISSN 1748-3395 (online) ISSN 1748-3387 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251116105627.htm'>GLP-1 drugs like Ozempic deliver huge weight loss but new research reveals a hidden catch</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-17 03:41:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Three new Cochrane reviews report that GLP-1 drugs can lead to meaningful weight loss, although the strong involvement of pharmaceutical companies in many studies raises concerns. The analyses evaluated three GLP-1 receptor agonists used for weight management and found that each one produced clinically important weight loss when compared with placebo. Even so, there is still limited or uncertain evidence about their long-term safety, possible side effects, and how financial ties might influence study results. Glucagon-like peptide-1 (GLP-1) receptor agonists were first introduced in the mid-2000s to help people with type 2 diabetes. More recently, researchers have been testing GLP-1 receptor agonists in people with obesity. These medications work by imitating a natural hormone that slows digestion and helps individuals feel full for longer. The reviews found little to no difference between the drugs and placebo when looking at major cardiovascular events, mortality, or quality of life. "These drugs have the potential to bring about substantial weight loss, particularly in the first year," says Juan Franco, co-lead researcher from Heinrich Heine University Düsseldorf, Germany. "It's an exciting moment after decades of unsuccessful attempts to find effective treatments for people living with obesity." The authors also stressed that the use of these medications should be viewed in a broader health context, including issues such as access, affordability, and insurance coverage, so that existing health inequities are not made worse. Costs remain a major barrier, particularly for semaglutide and tirzepatide, while liraglutide has become more affordable since its patent expired. Most of the trials were conducted in middle- and high-income countries, with little or no representation from regions such as Africa, Central America, and Southeast Asia. "We need more data on the long-term effects and other outcomes related to cardiovascular health, particularly in lower-risk individuals," says Eva Madrid, co-lead researcher from the Universidad de Valparaíso, Chile. More independent studies from a public health perspective are needed." Scientists Create “Key” To Find Alien Life in Clouds Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            