
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-29</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02133-1'>Polymyxin B lethality requires energy-dependent outer membrane disruption</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 16:15:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Polymyxin antibiotics target lipopolysaccharides (LPSs) in both membranes of the bacterial cell envelope, leading to bacterial killing through a poorly defined mechanism. Here we demonstrate that metabolic activity is essential for the lethality of clinically relevant doses of polymyxin B (PmB) and leverage this insight to determine its mode of action. PmB killed exponential-phase Escherichia coli but did not eliminate stationary-phase cells unless a carbon source was available. Antibiotic lethality correlated with surface protrusions visible by atomic force microscopy and LPS loss from the outer membrane via processes that required LPS synthesis and transport but that were blocked by the MCR-1 polymyxin resistance determinant. While energy-dependent outer-membrane disruption was not directly lethal, it facilitated PmB access to the inner membrane, which the antibiotic permeabilized in an energy-independent manner, leading to cell death. This work reveals how metabolic inactivity confers tolerance of an important, membrane-targeting antibiotic. The bactericidal activity of many antibiotics is dependent upon cellular metabolic activity1,2,3,4,5. For the efficacy of membrane-targeting antibiotics or peptides, however, it has been proposed that metabolic activity is not required, because intact membranes are presumed essential in all physiological states6,7,8. In support of this, many membrane-targeting compounds are effective against metabolically active and inactive bacteria9,10,11,12,13. Polymyxin antibiotics are the only membrane-targeting drugs used clinically against Gram-negative bacteria, but often lack efficacy in vivo, despite potent bactericidal activity in vitro14,15,16,17,18. Two polymyxins are used therapeutically, polymyxin B (PmB) and polymyxin E (colistin), both of which consist of a cationic cyclic peptide component coupled to an extracyclic peptide chain which in turn is linked to an acyl tail14,15,18. Both polymyxin antibiotics target lipopolysaccharides (LPSs) via high-affinity, long-lasting interactions, leading to disruption of the outer membrane (OM)18,19,20. However, the nature of this OM disruption and the mechanism by which bacteria are killed by the antibiotic is largely unknown14,15,18,21,22,23,24. Several previous studies have shown that inner-membrane (IM) permeabilization, which is proposed to be required for killing, occurs after OM disruption14,15,21,25,26,27,28. Our previous work showed that LPS is the IM target of polymyxins, just as it is for the OM27,28. While it is unknown how polymyxins cross the OM to access the periplasmic space, the predominant model is ‘self-promoted' uptake, by which the cationic properties of the antibacterial disrupt cation bridges between LPS molecules, compromising barrier function and enabling antibiotic ingress through the OM14,29,30. Despite the membrane-damaging effect of polymyxins, they do not eradicate persister cells of Escherichia coli or Acinetobacter baumannii31,32. Stationary-phase E. coli has also been shown to have a high level of polymyxin tolerance, suggesting that the bactericidal activity of these membrane-targeting antimicrobials is affected by metabolic activity and/or growth state10. Accordingly, metabolic activity was found to be required for colistin lethality at clinically relevant doses13. However, at concentrations above those typically achieved clinically (>10–50 µg ml−1), polymyxin lethality was found to be unaffected by growth phase or metabolic activity13,32,33. In summary, there are substantial gaps in our understanding of how polymyxin antibiotics kill bacteria, and the requirement for metabolic activity remains unclear. Therefore, we aimed to determine whether and why metabolic activity was needed for polymyxin lethality and thereby gain a significantly improved understanding of the mechanism by which this class of antibiotics kill bacteria. To assess the role of growth and metabolic activity in PmB-mediated killing, we took equal numbers of E. coli in exponential phase (high metabolic activity) or stationary phase (low metabolic activity) and exposed them to a clinically relevant concentration of the antibiotic (4 µg ml−1) in minimal medium only (MM) or with glucose (MM + G) to stimulate metabolic activity (Fig. 1a and Extended Data Fig. Exponential-phase E. coli cells were rapidly killed by PmB regardless of whether glucose was present (Fig. 1a), and PmB efficiently killed stationary-phase bacteria in the presence of glucose, albeit after a lag phase of 15 min during which there was no decrease in viability (Fig. Strikingly, however, there was no detectable killing of stationary-phase E. coli during exposure to PmB in the absence of glucose, indicating that metabolic activity is required for the bactericidal activity of the antibiotic (Fig. This effect of glucose was dose dependent, and the non-metabolizable glucose analogue 2-deoxyglucose did not support PmB-mediated killing of stationary-phase E. coli (Extended Data Fig. Conversely, there does not appear to be a requirement for cell division for PmB killing, as MM without antibiotic (±glucose) did not support noticeable growth of E. coli from either exponential or stationary phase during the assay. a, Survival of exponential-phase (Exp) or stationary-phase (Stat) E. coli exposed to 4 µg ml−1 PmB in MM ± glucose, as determined by c.f.u. b, AFM phase images showing exponential- or stationary-phase E. coli cells exposed to 2.5 µg ml−1 PmB in MM ± glucose, shown as a function of time. Colour phase scale (scale insert in first row of image at t = 90 min), 6 degrees (row 1), 4 degrees (row 2), 5 degrees (rows 3 and 4). Except for stationary-phase cells without glucose, all bacteria were SYTOX positive by the end of the imaging (Supplementary Fig. It is worth noting that a lower PmB concentration was used for AFM relative to other experiments, due to the low density of cells used in these assays. The ‘phase' in degrees represents the shift in the phase of oscillation of the AFM cantilever. c, OM disruption of stationary-phase E. coli cells during the first 20 min of exposure to 4 µg ml−1 PmB, as determined by uptake of NPN fluorescent dye. d, OM disruption of E. coli PerimCherry as determined by egress of the fluorescent protein mCherry into the culture supernatants of stationary-phase bacteria exposed or not to 4 µg ml−1 PmB in MM ± glucose. e, Combined OM and IM disruption of stationary-phase E. coli exposed to 4 µg ml−1 PmB in MM ± glucose, as determined by uptake of the fluorescent nucleic acid dye SYTOX green. RFU, relative fluorescence units. For c and e, the blank value refers to the relevant fluorophore in medium without bacteria. For c and e, ‘No treat MM' refers to the absence of PmB in minimal medium; ‘No treat MM + G' refers to the absence of PmB in minimal medium containing 0.36% glucose. All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Significant differences were determined between stationary-phase MM + G with PmB and each of the other conditions by two-way repeated measures ANOVA. Using atomic force microscopy (AFM) to examine the effect of the antibiotic on the OM of living bacteria37,38, we tracked the effects of the antibiotics in real time: PmB exposure of exponential-phase E. coli resulted in rapid and profound changes to the cell surface, with the appearance of numerous protrusions, regardless of the presence of glucose (Fig. By contrast, stationary-phase E. coli only showed PmB-induced OM changes in the presence of glucose (Fig. 1b), correlating with the effects on bacterial viability (Fig. Scanning electron microscopy replicated the AFM observations, confirming these distinct, glucose-dependent phenotypes of stationary-phase cells exposed to PmB, while the antibiotic caused surface protrusions in exponential-phase E. coli regardless of the presence of glucose (Extended Data Fig. Next, using the small hydrophobic dye NPN (N-phenyl-1-naphthylamine)39, we showed that the binding of PmB to its lipid A target was unaffected by glucose or growth phase (Fig. This finding was supported by mass spectrometry27, which ruled out the presence of lipid A modifications in stationary-phase cells that might reduce PmB binding but intriguingly did show a large reduction in LPS abundance in cells exposed to PmB in MM + G (Extended Data Fig. We further investigated changes to OM integrity by measuring the leakage of heterologously expressed mCherry (28 kDa) from the periplasm40. In the presence of PmB and glucose, stationary-phase E. coli showed a significant increase of mCherry in the supernatant, compared with cells not exposed to the antibiotic from 90 min onwards. This suggests that minor OM permeabilization/disruption occurs rapidly, as measured by NPN, but that major disruption to the OM—as measured by mCherry release—occurs much more slowly, possibly via lysis (Fig. Finally, we used SYTOX green nucleic acid stain, to assess permeability of both the OM and IM during PmB exposure. As expected from the viability data (Fig. 1a), stationary-phase E. coli showed significantly increased permeability in the presence of glucose, after a lag of 15 min (Fig. 1e); permeability also occurred in the absence of glucose, but at significantly lower levels than for cells exposed to the antibiotic in the presence of glucose (Fig. Again, consistent with the viability data (Fig. 1a), PmB-induced IM permeability was unaffected by glucose in exponential-phase E. coli (Supplementary Fig. Taken together, these data show that PmB binds to E. coli regardless of metabolic state, leading to minor OM disruption sufficient to allow NPN ingress, but that bactericidal activity, OM protrusions and release of periplasmic protein require metabolic activity. It is worth noting that stationary-phase viability and OM/IM permeation were similarly dependent on glucose across a diverse panel of laboratory and clinical isolates, indicative of a broadly conserved phenotype (Extended Data Fig. High-resolution AFM imaging of the OM of stationary-phase E. coli showed a similar abundance and organization of OM protein networks as previously reported37,38 (Fig. 2a and Extended Data Fig. 5; note porous appearance of the OM in phase images). The porin networks were initially unaffected by PmB, but over time, the appearance of the PmB-exposed OM was dominated by protrusions at a scale >10 nm (Fig. On these living E. coli cells, we did not observe any PmB-induced LPS rearrangement into hexagonal lattices, such as previously reported based on AFM on solid-supported, collapsed OM vesicles22,23,24. a, High-magnification AFM height and phase images of stationary-phase E. coli exposed to 2.5 µg ml−1 PmB in MM + G, showing the porin network remains largely intact for at least 60 min of antibiotic challenge. Height scale (scale inset in first row of image at t = 90 min), 3 nm. b, Kdo analysis of LPS recovered from filtered supernatants of stationary-phase E. coli exposed, or not, to 4 µg ml−1 PmB in MM ± glucose for 15 min (n = 5). c, Levels of OmpF, GroEL and total LPS in E. coli exposed, or not, to 4 µg ml−1 PmB in MM ± glucose for 15 min. d, Quantification of LPS levels in E. coli exposed, or not, to 4 µg ml−1 PmB in MM ± glucose for 15 min. For b–d ‘No treat MM' refers to the untreated condition absence of PmB in minimal medium; ‘No treat MM + G' refers to the untreated condition absence of PmB in minimal medium containing 0.36% glucose. Unless stated otherwise, experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Statistical significance of differences was determined by one-way ANOVA; ***P < 0.001; ****P < 0.0001; NS, not significant. Further examination of PmB-induced OM changes revealed that LPS was released from cells exposed to PmB in the presence of glucose but not in the absence of the antibiotic or glucose (Fig. Analysis of bacterial cells found a corresponding loss of LPS from E. coli exposed to PmB in the presence but not absence of glucose (Fig. 2c,d and Supplementary Fig. 2), which was maximal by 15 min and fully consistent with the mass spectrometry results reported above (Extended Data Figs. This coincided with the appearance of surface protrusions, which subsequently increased in abundance, up to IM permeabilization as measured by SYTOX (Supplementary Fig. Moreover, LPS loss at 15 min was not due to cell lysis as, in contrast to LPS, there were no differences in levels of the cytoplasmic protein GroEL or OM porin OmpF across our experimental conditions (Fig. Combined, these data revealed that PmB at least initially leaves the OM protein networks intact but causes substantial and rapid LPS loss in the presence of glucose. PmB-induced LPS loss also occurred with Pseudomonas aeruginosa and with exponential-phase E. coli in Mueller–Hinton broth (MHB), indicative of a conserved process (Extended Data Fig. The observed dependence on metabolic activity for PmB lethality suggests an energy-dependent bacterial response to PmB challenge. As PmB-triggered LPS release required metabolic activity, we hypothesized that LPS synthesis and/or transport to the OM contributed to bacterial killing. To establish energy dependence, we sought to distinguish between carbon- and energy-related effects. Using equimolar concentrations of various carbon sources in MM, we measured stationary-phase E. coli survival under PmB challenge and, in parallel experiments without PmB, bacterial ATP production and overall LPS abundance (Fig. In these experiments, bacterial PmB susceptibility correlated with ATP production and with the amount of LPS associated with cells in the absence of PmB (Fig. 3a,b and Extended Data Fig. In the presence of PmB, the amount of LPS dropped to lower levels in the presence of carbon sources that yielded higher ATP production (Fig. 3a,b and Extended Data Fig. 7), confirming the importance of energy for PmB-induced LPS loss and bacterial killing. a, Four-way representation of the effect of various carbon sources, or none, on ATP production and LPS levels in stationary-phase E. coli in the absence of PmB, and bacterial survival and LPS levels after exposure to 4 µg ml−1 PmB. The more yellow the symbol, the more LPS is produced in the absence of PmB; the smaller the symbol, the more LPS that is lost during PmB exposure. b, SDS–PAGE analysis of LPS from stationary-phase E. coli exposed to various carbon sources in MM for 15 min in the absence or presence of 4 µg ml−1 PmB. c–f, Survival of stationary-phase E. coli in MM + G supplemented, or not, with PmB, with or without growth-inhibitory concentrations (1× MIC) LpxC inhibitors ACHN-975 (c) or CHIR-090 (d) or protein synthesis inhibitor tetracycline (e) or MsbA inhibitor G907 (f). g, Survival of E. coli wild type (WT) or LptD4213 strain exposed, or not, to 4 µg ml−1 PmB. h, LPS levels of non-treated E. coli or bacteria exposed to 4 µg ml−1 PmB ± LpxC inhibitors (CHIR-090, ARCN-975) or tetracycline for 15 min (n = 4). Panel includes an SDS–PAGE image of LPS band intensity. For h ‘No treat' refers to the untreated condition absence of PmB in minimal medium containing 0.36% glucose. It is worth noting that the experiments with G907 used E. coli LptD4213 to enable antibiotic ingress. Unless otherwise stated, all experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Statistical significance of differences was determined by one-way (h) or by two-way repeated measures ANOVA between the PmB-treated and PmB + antibiotic-treated groups (c–f) or between wild type with PmB and imp4213 with PmB (g). To establish a link between LPS production and PmB-induced LPS loss and killing, we performed PmB killing assays in the presence of various inhibitors41. Inhibition of UDP-3-O-acyl-N-acetylglucosamine deacetylase (LpxC), the rate-limiting step of lipid A synthesis, reduced the rate and degree of PmB-mediated killing of both E. coli and P. aeruginosa, whereas the protein synthesis inhibitor tetracycline had no effect (Fig. 3c–e and Extended Data Fig. There was also a slower progression of PmB-triggered surface protrusions in the presence of CHIR-090 (an LpxC inhibitor) relative to PmB alone (Extended Data Fig. We also found that an inhibitor of the LPS transporter MsbA (multicopy suppressor of the HtrB temperature-sensitive phenotype) reduced killing by PmB and slowed protrusion appearance, while a strain with reduced LPS transport protein D (LptD) function was also killed more slowly than the wild type (Fig. 3f,g and Extended Data Fig. For all inhibitors, PmB binding to the OM was not affected, as determined via NPN uptake, whereas reduction of LPS synthesis and transport significantly reduced PmB-induced IM permeation as measured by SYTOX ingress into the cytoplasm (Supplementary Figs. Further evidence for the importance of LPS transport came from experiments with novobiocin, which promotes LPS trafficking from the IM to the OM42 and was found to promote PmB-mediated LPS loss (Extended Data Fig. This finding also provides a molecular explanation for the potentiating effect of novobiocin on polymyxin susceptibility42,43. Combined, these experiments revealed that the magnitude of PmB-mediated LPS loss correlated with metabolic activity, which in turn correlated with bacterial killing. The role of metabolic activity in LPS loss was due, at least in part, to a requirement for LPS synthesis and transport. To differentiate between PmB action on the OM and on the IM, we examined whether PmB could kill metabolically active cells by LPS loss alone, without the need for permeabilization of the IM. For this purpose, we first exposed E. coli to PmB for 15 min in MM + G, long enough for LPS loss to reach its maximum but before IM disruption occurred and before colony-forming unit (c.f.u.) counts started to decrease relative to the inoculum (Fig. 1a,e and Extended Data Fig. We then washed the bacteria to remove unbound antibiotic, which also removed released LPS, followed by incubation in MM ± PmB and ±glucose. In the absence of PmB, bacteria survived at the same level over 2 h in the presence or absence of glucose (Fig. By contrast, bacteria that were re-exposed to PmB were efficiently killed, regardless of the presence of glucose (Fig. Therefore, PmB-mediated LPS loss alone is not sufficient to kill bacteria. Moreover, the requirement for metabolic activity for PmB-mediated killing appeared to only apply to LPS loss from the OM and is not required for subsequent IM disruption and killing. of stationary-phase E. coli first pre-treated with 4 µg ml−1 PmB in MM + G for 15 min before washing to remove unbound antibiotic, and next (from t = 0) exposed to 4 µg ml−1 PmB, or not, in MM ± glucose. b, LPS levels of untreated E. coli or bacteria exposed to 4 µg ml−1 PmB, 4 µg ml−1 PmBN or 10 mM EDTA for 15 min in MM + G. Note that experiments with EDTA used MM that was not supplemented with MgSO4 or CaCl2. Panel includes SDS–PAGE image of LPS band intensity. c, LPS levels of non-treated E. coli or bacteria exposed to 4 µg ml−1 PmB, 4 µg ml−1 PmBN or 10 mM EDTA for 15 min in MM. Panel includes SDS–PAGE image of LPS band intensity. d, Survival of stationary-phase E. coli exposed to a range of PmB concentrations for 2 h in MM + G, following a 15 min pre-treatment, or not, with 4 µg ml−1 PmB, 4 µg ml−1 PmBN or 10 mM EDTA in MM + G. e, Levels of bodipy-vancomycin labelling of E. coli exposed, or not, to 4 µg ml−1 PmB in MM ± glucose for 15 min. f, Survival of E. coli exposed, or not, to PmB with or without 10 mM EDTA in MM + G. g, Survival of E. coli exposed, or not, to PmB with or without 10 mM EDTA in MM. h, AFM phase images showing stationary-phase E. coli exposed to 2.5 µg ml−1 PmB and 10 mM EDTA in MM. Scale bar, 250 nm; phase scale (scale inset in image at t = 90 min), 5 deg. The same cell was imaged during the experiment. All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Statistical significance of differences was determined by one-way (in b, c and e) or two-way repeated measures ANOVA between No treat MM + G and PmB MM + G and between No treat MM and PmB MM (in a) between No treat and pre-treatment conditions (in d) between No treat and antibiotic conditions (in f and g). To further test the impact of LPS loss on bacterial killing, we exposed stationary-phase E. coli to the nonlethal PmB nonapeptide (PmBN)18, which also caused significant LPS loss within 15 min in the presence of glucose, and to EDTA (ethylenediamine tetraacetic acid), which caused LPS loss both in presence and absence of glucose (Fig. At the doses tested, neither EDTA nor PmBN had any effect on bacterial viability and did not trigger membrane protrusions (Extended Data Fig. Based on these findings, we hypothesized that polymyxin-triggered LPS release, while not bactericidal on its own, facilitates antibiotic entry into the periplasm, enabling PmB to access and permeate the IM and thereby kill the cell. To test this, we exposed stationary-phase E. coli to PmB, PmBN or EDTA for 15 min in MM + G to trigger LPS loss, as above, before removing these OM-disrupting agents (and released LPS) by washing, and finally measured bacterial survival in the presence of freshly added PmB. Each of the three pre-treatments enabled killing by PmB at concentrations below those required to kill untreated control cells, indicating that LPS loss sensitizes bacteria to PmB-mediated killing (Fig. It is worth noting that we saw a greater level of bacterial killing for experiments where bacteria were pretreated with a membrane-disrupting agent before washing and subsequent exposure to PmB (for example, Fig. 4a,d), compared with experiments without a pre-treatment step (for example, Fig. A possible explanation for this difference is that released LPS sequesters PmB, reducing its activity44. As this is washed away in the pre-treatment experiments, it cannot reduce the activity of the subsequent PmB dose. Furthermore, once the OM is compromised by the membrane-disrupting pre-treatment, subsequent PmB treatment can efficiently access the IM. Combined, this explains the higher levels of bacterial killing in the pre-treatment experiments relative to other assays. To demonstrate that PmB-triggered LPS loss causes the OM to become permeable to molecules of the size of PmB, we showed periplasmic ingress of a bodipy-conjugated analogue of the glycopeptide antibiotic vancomycin (Mw = 1723.35 g mol−1 versus 1301.56 g mol−1 for PmB) (Fig. However, minor permeabilization of the OM, as occurs with PmB in the absence of glucose, did not permit entry of vancomycin into the periplasm (Fig. As EDTA caused LPS loss from stationary-phase E. coli in the absence of glucose (Fig. 4c), we hypothesized that EDTA would enable PmB to access the IM and thereby kill E. coli in the absence of energy. First, in MM + G, there was a significant loss of viability of E. coli exposed to PmB alone and to PmB combined with EDTA, but not EDTA alone (Fig. However, levels of bacterial viability were the same for cells exposed to PmB alone and PmB–EDTA, indicating this combination was not synergistic at the concentrations used (Fig. We then repeated this experiment in MM without glucose. In keeping with our earlier findings, metabolically inactive cells were not killed by PmB alone (Fig. However, there was a significant loss of viability of metabolically inactive cells exposed to PmB and EDTA together (Fig. In addition, analysis of EDTA/PmB co-treatment of metabolically inactive cells by AFM revealed a roughening of the bacterial surface, with numerous protrusions that resembled the effects observed of PmB alone in the presence of glucose (Fig. Together, these experiments show that metabolic activity is required for PmB lethality because it supports LPS release, which provides the antibiotic with access to the IM, the disruption of which leads to bacterial killing. However, if LPS loss occurs via a PmB-independent mechanism, such as EDTA, then there is no need for metabolic activity, showing that IM disruption occurs via an energy-independent process. Mobile colistin resistance (mcr) genes, which encode phosphoethanolamine (pEtN) transferases, are globally disseminated in the Enterobacterales46,47,48. MCR-1, which is the most common variant, selectively modifies the 4′ phosphate of lipid A targeted by PmB, reducing susceptibility to polymyxin antibiotics47. Previous work showed that MCR-1 protected the IM of both whole cells and spheroplasts from colistin28,39. However, previous studies have shown that polymyxin antibiotics can disrupt the OM of polymyxin-resistant E. coli as evidenced by ingress of NPN or hydrophobic antibiotics such as rifampicin14,28,39. As such, the impact of MCR-1 on OM integrity during PmB exposure is unclear. Based on the work above, we hypothesized that pEtN modification would contribute to polymyxin resistance by inhibiting PmB-triggered LPS release, which would block access of the antibiotic to the IM. To test this, we used E. coli transformed with an inducible mcr-1 construct but allowed expression to occur at basal levels (that is, no inducer) so that we had minimal production of MCR-1 and thus only a 4-fold increase in PmB minimal inhibitory concentration (MIC) above the wild-type strain (Supplementary Table 1). Despite this modest change in susceptibility, MCR-1 fully protected E. coli from PmB at 4 µg ml−1 in MM + G, whereas there was a >4-log10 reduction in c.f.u. counts of the wild type (Fig. There was also a substantial reduction in the viability of E. coli that produced a catalytically inactive version of MCR-1 (MCR-1*) (Fig. As expected, MCR-1 reduced but did not prevent OM disruption as indicated by NPN uptake relative to the wild-type strain, whereas the IM was fully protected from PmB (Fig. a, Survival of stationary-phase MG1655, MG1655 mcr-1* or MG1655 mcr-1 exposed to 4 µg ml−1 PmB in MM + G, as determined by c.f.u. b, OM disruption of stationary-phase MG1655, MG1655 mcr-1* or MG1655 mcr-1 cells during the first 20 min of exposure to 4 µg ml−1 PmB in MM + G, as determined by uptake of NPN fluorescent dye. c, OM and IM disruption of stationary-phase MG1655, MG1655 mcr-1* or MG1655 mcr-1 exposed to 4 µg ml−1 PmB in MM + G, as determined by uptake of the fluorescent dye SYTOX green. d, Total LPS in stationary-phase MG1655 mcr-1* or MG1655 mcr-1 exposed, or not, to 4 µg ml−1 PmB in MM + G for 15 min. Graph shows quantification of LPS levels from three independent experiments. e, AFM phase images showing stationary-phase MG1655 mcr-1* and MG1655 mcr-1 exposed to 2.5 µg ml−1 PmB in MM + G. The same cell was imaged during the experiment. Scale bar, 250 nm; phase scale (scale inset in second row of image at t = 90 min), 4 deg. f, Mean surface roughness of MG1655 mcr-1* and MG1655 mcr-1 treated with 2.5 µg ml−1 PmB in MM + G. For b and c, the blank value refers to the relevant fluorophore in medium without bacteria. All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Statistical significance of differences was determined by one-way (in d) or two-way repeated measures ANOVA between MG1655 and mcr-1* or mcr-1 (in a, b and c). In support of our hypothesis, we next showed that MCR-1 prevented PmB-triggered LPS loss, as well as surface protrusions, both of which occurred in the wild type and the strain expressing the catalytically inactive MCR-1 variant (Fig. Therefore, MCR-1 mediated pEtN modification of lipid A prevents LPS loss from E. coli exposed to PmB, which contributes to resistance by blocking access of the antibiotic to the IM. However, it is not clear whether the lack of LPS release is due to reduced PmB binding or greater membrane integrity due to the pEtN modification or a combination of both factors. The mechanism(s) by which polymyxin antibiotics kill bacteria has been the focus of multiple studies, with various conclusions reached and models proposed14,15,18,21,22,50. In this work, we established that metabolic activity is required for bactericidal activity of a clinically relevant concentration of PmB and used this as a starting point to build a refined model for the mechanism by which polymyxin antibiotics kill bacteria (Fig. We found that the initial interaction of PmB with the OM was unaffected by the metabolic state of the bacteria, explaining how polymyxins can sensitize metabolically dormant persister bacteria to antibiotics32,51,52 (Fig. In metabolically inactive bacteria, there is no further disruption to the bacterial surface and no loss of viability. However, in metabolically active E. coli, PmB causes rapid, large-scale LPS loss, which required ATP-dependent LPS synthesis and transport (Fig. With the loss of OM integrity, PmB gains access to LPS in the IM, and surface protrusions appear, although it is not yet clear whether these processes are directly linked (Fig. Finally, the interaction of PmB with LPS in the IM results in IM disruption and bacterial killing via an energy-independent process, and OM disruption increases, sufficient to allow the egress of proteins from the periplasm (Fig. a, PmB binds to the OM, causing minor permeabilization, regardless of bacterial metabolic activity (insufficient damage to provide PmB with access to the periplasm). b, In metabolically active bacteria, PmB triggers LPS release via a process that requires ATP-dependent synthesis (Raetz pathway) and transport of LPS (further boosted by novobiocin (Nov)). c, The shedding of LPS compromises the integrity of the OM, resulting in surface protrusions, and enables ingress of the antibiotic into the periplasm where it targets LPS in the IM, leading to permeabilization. d, The loss of integrity of both the OM and IM results in bacterial killing. The full mechanism of PmB-triggered LPS loss remains to be elucidated. While EDTA and PmB both disrupt the cation bridges that stabilize LPS molecules in the OM, only EDTA causes LPS release in the absence of metabolic activity. This likely reflects the fact that EDTA chelates cations, whereas PmB displaces cations from LPS by binding lipid A phosphates. While cation displacement by PmB weakens inter-LPS interactions, it appears that this is only sufficient to cause minor permeabilization. We speculate that the insertion of PmB into the LPS monolayer causes slipping between molecules, and as more LPS is transported into the OM, this leads to loss of PmB-bound LPS from the OM. Such slipping could be consistent with the complex interplay between PmB and OM fluidity53,54. Several antibiotic classes require bacterial metabolic activity for lethality2,3,4,5. However, our finding that this requirement also applies to PmB goes against the common assumption that the efficacy of membrane-targeting antibacterials is independent of metabolic activity. It also provides an explanation for polymyxin tolerance observed in stationary-phase cells and persisters4,5,9,10,31,32,55,56. An important distinction, though, is that while many bactericidal antibiotics kill via metabolism-dependent production of reactive metabolites5,8,57, the key energy-dependent step required for polymyxin activity is disruption of the OM, which enables the antibiotic to permeabilize the IM and kill the bacterium. We have previously shown that sub-inhibitory concentrations of the LptD inhibitor murepavadin promoted polymyxin susceptibility of P. aeruginosa, via LPS accumulation in the IM, without reducing LPS transport to the OM27. Based on the work presented here, we would predict that growth inhibitory concentrations of inhibitors of the Lpt system would reduce killing by polymyxins. However, we could not test this because the available inhibitors (murepavadin and thanatin) are cationic peptides that cause OM disruption, including membrane blebbing, which would compromise the findings58,59,60. Instead, we capitalized on the observation that novobiocin promoted LPS transport by enhancing ATPase activity, which in turn increased bacterial susceptibility to polymyxin antibiotics, although it was not known why at the time42,43. Although our conclusions agree with some previous reports, they may appear inconsistent with others. For example, there is evidence that PmB is active against metabolically inactive bacteria. However, those studies used either exponential-phase cells that may retain a degree of metabolic activity or used assay conditions with low concentrations of nutrients or supra-physiological concentrations of polymyxins where specificity for LPS is lost due to the amphipathic nature of these antibiotics13,15,32,33,34,35,61. There have also been conflicting studies on the contribution of reactive oxygen species to bactericidal activity of PmB62,63,64,65. Our finding that energy is required for severe OM disruption does not rule out a role for reactive oxygen species, which could arise via LPS synthesis, for example. Finally, while we show surface protrusions, it is unclear whether these are released as vesicles, which has been reported to occur at sub-inhibitory PmB concentrations20,66,67,68,69,70,71. More work is also needed to determine whether PmB-induced vesicles are driven exclusively by antibiotic action on the membrane or part of bacterial stress response. For example, previous work indicated that a trigger of the Sigma E (RpoE) membrane stress response system resulted in increased vesicle production72, and there is evidence that the conjugative plasmid expression (Cpx) and reactive chlorine species (Rcs) systems, which respond to IM and OM stress, respectively, are activated by polymyxin exposure and contribute to bacterial survival73,74,75,76,77. As such, there remains much to understand about how bacteria respond to polymyxin antibiotics and how this influences survival. Despite the loss of LPS that occurred during PmB exposure, cells remained viable until IM disruption occurred, in agreement with our and others' previous work showing that IM damage is required for lethality25,27,52. Although the OM is an important load-bearing structure, much of the mechanical strength is provided by the porin network, which forms tight associations with LPS78,79,80,81 and which remained largely intact during PmB exposure (Fig. The mechanical integrity of the porin network may therefore explain how E. coli can tolerate major LPS loss without a reduction in viability38,78. Intrinsic and some acquired polymyxin resistance is associated with very high levels of lipid A modification with cationic pEtN and/or 4-amino-4-deoxy-l-arabinose (L-ara4N) moieties, which prevents PmB binding to the OM14,18,20,48. However, MCR-mediated resistance often involves modification of only some of the lipid A in the OM, which likely explains how PmB can bind and sufficiently permeabilize the OM for the ingress of hydrophobic antibiotics such as rifampicin28,39. We found that some OM permeabilization occurred for E. coli mcr-1 but also showed that the MCR-1-mediated lipid A modification prevented PmB-triggered LPS release, which in turn prevented access of the antibiotic to the IM. This, coupled with the protection of the IM via LPS modification27,28, provides distinct layers of protection against PmB. Minimal media, including M9 used in these studies, have been found to better predict antibiotic susceptibility of bacteria at infection sites than rich laboratory media, which likely reflects the reduced metabolic activity of organisms in vivo1,2,82,83. That said, as M9 does not fully replicate the host environment, further work will be needed to comprehensively understand the relationship between host environment, bacterial metabolism and polymyxin treatment outcomes. For example, one aspect of infection absent in these studies is host immunity. Previous work has shown that polymyxins synergize with complement and the antibacterial histone H2A84,85. PmB has been shown to reduce the pro-inflammatory activity of LPS, which likely modulates the host immune response86. In summary, this work advances our understanding of the mechanism of action of polymyxin antibiotics by showing that the nature and magnitude of PmB-mediated OM damage is dependent upon the metabolic activity of the cell, which in turns modulates access of the antibiotic to the IM, disruption of which is required for lethality but is not energy dependent (Fig. This work also provides an explanation for how polymyxin tolerance occurs in persister and stationary-phase cells, despite the membrane-disrupting activity of these antibiotics (Fig. The bacterial strains used in this study are listed in Supplementary Table 1 (refs. E. coli strains expressing mcr-1 or mcr-1* under the control of the arabinose-inducible araBAD (pBAD) promoter were based on those described previously and were transformed with plasmids synthesized by Thermo Fisher48. Unless stated otherwise, all strains were grown in MHB (Millipore) for 18 h at 37 °C with shaking (180 r.p.m.) Where indicated, exponential-phase bacteria were generated by inoculating fresh MHB with 1:1,000 dilution of stationary-phase cells followed by a second growth period of 3 h at 37 °C with shaking (180 r.p.m.) For all experimentation, bacteria were washed three times with 1× M9 minimal medium (Gibco) supplemented with 2 mM MgSO4 and 0.1 mM CaCl237 (referred to here as minimal medium, MM). Washed cells were then resuspended in MM ± 0.36% glucose (MM + G) to 108 c.f.u. ml−1. To enumerate bacterial c.f.u., 10-fold serial dilutions were made in PBS and plated onto Mueller–Hinton agar (MHB supplemented with 1.5% bacteriological agar). Inoculated agar plates were incubated statically for 18 h in air at 37 °C. The MIC of PmB, the LpxC inhibitors CHIR-090 and ACHN-975, tetracycline, the MsbA inhibitor G907, novobiocin and murepavadin against indicated bacterial strains was determined according to the well-established broth microdilution method95. In brief, a 96-well microtitre plate was used to prepare a range of antibiotic concentrations in 100 µl of MM + G or cation-adjusted MHB via a series of 2-fold serial dilutions. For antibiotic synergy determination, checkerboard analyses were performed by preparing 2-fold serial dilutions of two antibiotics, with each one across a different axis, generating an 8 × 8 matrix to assess the MICs of each antibiotic when used in combination96. Stationary-phase bacteria were diluted to 1 × 106 c.f.u. ml−1 in MM + G or cation-adjusted MHB and 100 µl used to seed each well of the microtitre plate to give a final inoculation density of 5 × 105 c.f.u. ml−1. Plates were then incubated statically at 37 °C for 18 h in air, at which point the MIC was defined as the lowest antibiotic concentration at which there was no visible growth of bacteria. In some cases, the extent of bacterial growth after incubation was also determined by obtaining measurements of optical density at 595 nm (OD595nm) using a Bio-Rad iMark microplate absorbance reader (Bio-Rad Laboratories). Following three washes in MM, stationary-phase or exponential-phase bacteria were added at a final concentration of 1 × 108 c.f.u. ml−1 to 3 ml of MM, MM + G or MM without cations and supplemented with 10 mM EDTA, containing PmB (4 µg ml−1) when required. This assay was also repeated in the presence of PmB and 1× MIC of a range of different antibiotics (CHIR-090, ACHN-975, tetracycline, G907), as well as in MM supplemented with a selection of equimolar concentrations of either glucose, rhamnose, arabinose, mannose, galactose, acetate, succinate or glycerol. Cultures were incubated at 37 °C with shaking (180 r.p.m.). At each time point (0, 15, 30, 60 and 120 min), aliquots were taken, serially diluted 10-fold in PBS and plated to determine bacterial viability by determination of c.f.u. Bacterial imaging by AFM followed protocols described previously37,38. Briefly, for sample preparation, round glass coverslips of 13 mm diameter were used. They were washed by sonication in 1% w/v SDS for 10 min at 37 kHz and 100% power, then rinsed with MilliQ water and ethanol and dried with a nitrogen gun. Then they were plasma cleaned for 2 min at 70% power. These two procedures were repeated once more. Then coverslips were coated with Vectabond: they were placed in a beaker with a 1:50 ratio of Vectabond solution to acetone (40 ml of acetone and 800 µl Vectabond solution) for 5 min, then rinsed with MilliQ water and dried with a nitrogen gun97. The coverslips were glued to microscope slides with water-soluble glue purchased from Bruker (Reprorubber thin pour, Flexbar). Subsequently, E. coli was cultured to stationary phase (18 h) in MHB, then diluted 1:100 into fresh MHB and incubated to mid-exponential phase. Bacteria were then prepared by centrifuging 3 times at 17,000 × g for 90 s and resuspended in MM or MM + G at an OD600 of 0.5. They were resuspended the fourth time in 100 µl 20 mM HEPES and incubated onto the Vectabond-coated glass coverslips for 5 min. They were then washed off with imaging media (MM or MM + G) to remove unadhered bacteria. SYTOX green nucleic acid stain (at a final concentration of 5 µM) was added and incubated at room temperature for 5 min. We first imaged untreated bacteria in MM or MM + G, then with a pipette this was removed as much as possible and replaced with either PmB, EDTA, PmBN, CHIR-090 or combinations, added in the same way. Scans were subsequently taken after the introduction of the drug. All experiments were performed in dynamic mode with a Nanowizard III AFM with UltraSpeed head (Bruker AXS), with an Andor Zyla 5.5 USB3 fluorescence camera on an Olympus IX 73 inverted optical microscope. FastScanD cantilevers were used, with a resonant frequency around 110 kHz and a spring constant of 0.25 N m−1. The drive frequency used ranged from 95 to 120 kHz, depending on the cantilever resonance measured by calibration. The setpoint was between 10 and 15 nm (approximately 50% to 70% relative to free amplitude). The whole-cell images were acquired at 2 to 2.5 Hz line frequencies, 2 × 2 µm2, and 512 × 512 or 512 × 256 pixels. Higher-magnification images were acquired over 500 × 500 nm2 and 512 × 512 pixels, recorded at 4 to 5 Hz line frequencies. 2a were acquired over 200 × 200 nm2 and 512 × 512 pixels recorded at 8 to 10 Hz line frequency. To process the images obtained from AFM the software Gwyddion 2.65 was used (https://gwyddion.net/)98. The following steps were taken to obtain a post-processed image: (1) level data by mean plane subtraction; (2) level by flattening base; (3) if there were strongly protruding features, apply a mask by selecting features above a 50% threshold, to prevent these from biasing the background correction and row alignment; (3) align rows by second polynomial line-by-line background subtraction (and exclude the masked region if applicable); (4) apply a 1–2 pixel Gaussian filter to the whole image to remove high-frequency noise (remove mask if applicable); and (5) define zero reference and adjust the colour scale to highlight key features. The E. coli cells were cultured in MHB to stationary phase before bacteria were collected using centrifugation, and the cell pellet was washed three times with M9 medium without glucose. The cells were then diluted to 108 c.f.u. ml−1 in M9 ± glucose with 4 µg ml−1 PmB, and samples were collected at various time points through centrifugation for further analysis. The samples obtained were treated with 2.5% glutaraldehyde in a 0.1 M sodium cacodylate buffer (pH = 7.3) and were allowed to fix overnight at 4 °C. After fixation, the cells were suspended in the 0.1 M sodium cacodylate buffer and subjected to dehydration using ethanol concentrations of 10%, 20%, 30%, 40%, 50%, 70%, 80% and 100%, with a 10 min incubation at each step. The drying procedure was performed via critical point drying, and a platinum coating of 10 nm thickness was applied after the samples were completely dried. The prepared samples were examined under a scanning electron microscope (Zeiss Crossbeam 550 FIB-SEM) at an accelerating voltage of 2 kV, and all images were analysed after conversion for differences in magnification. The ImageJ/Fiji software (version 1.53) was used in scanning electron microscope image processing and analysis. To detect OM damage by PmB, the well-established NPN uptake assay was performed39. The fluorescent probe NPN (Acros Organics), at a final concentration of 10 μM, was added to 100 µl of 2 × 108 c.f.u. ml−1 stationary-phase or exponential-phase bacteria in MM, MM + G or MM without cations and supplemented with 10 mM EDTA in a black microtitre plate with clear-bottomed wells (Greiner Bio-One). Relevant concentrations of PmB or PmB in combination with 1× MIC of various antibiotics were added to the wells of the microtitre plate to give a total volume of 200 µl. Fluorescence, at 37 °C with shaking, was measured immediately using a Tecan Infinite 200 pro plate reader using an excitation wavelength of 355 nm and an emission wavelength of 405 nm. Fluorescence measurements were obtained every min for 20 min, and the degree of OM permeabilization, referred to as the NPN uptake factor, was calculated using the following formula: The resulting NPN uptake factor values were normalized according to OD600nm. To further assess OM damage by PmB, PerimCherry E. coli was prepared by transforming the pPerimCh plasmid40 into E. coli MG1655. This plasmid encodes a constitutively expressed mCherry construct that accumulates in the periplasm. ml−1) were incubated with PmB in MM or MM + G. At each time point, 200 µl of the culture was centrifuged to separate bacteria and culture medium, and the supernatant was removed and placed in the wells of a black clear-bottomed 96-well plate (Greiner Bio). The fluorescence was measured with an excitation wavelength of 587 nm and an emission wavelength of 610 nm. To measure IM permeabilization by PmB, the well-established SYTOX green assay was used99. Following washes in MM, stationary-phase bacteria at an inoculum density of 1 × 108 c.f.u. ml−1 was added to 3 ml of MM or MM + G containing the relevant antibiotics. The fluorescent probe SYTOX green (Invitrogen) was added to these cultures at a final concentration of 1 µM. Aliquots (200 μl) were transferred to a black microtitre plate with clear-bottomed wells (Greiner Bio-One). Fluorescence and OD600nm were measured in a Tecan Infinite 200 Pro plate reader (excitation at 535 nm, emission at 617 nm) every 15 min for 2 h at 37 °C with shaking. Raw fluorescence readings were normalized according to OD600nm. Lipid A was prepared and analysed for modifications by mass spectroscopy as described previously27,28. Briefly, lipid A was detached from carbohydrates via mild acid hydrolysis (2% acetic acid, 100 °C 30 min), washed in H2O, loaded on the target and overlaid with 9H-Pyridol[3,4-B]indole (Norharmane, Sigma-Aldrich) used at 10 mg ml−1 in chloroform to methanol 9:1 and mixed before air drying and matrix-assisted laser desorption ionization time-of-flight analysis using a MALDI Biotyper Sirius system (Bruker Daltonics). The experimental conditions for LPS quantification were the same as those used for the antibiotic time-kill assay. ml−1 of stationary-phase bacteria was added to 3 ml cultures of MM or MM + G in the presence of either PmB, PmB with 1× MIC of various antibiotics, PmBN or EDTA. LPS quantification was also performed in the presence of PmB in MM supplemented with a selection of equimolar concentrations of either glucose, rhamnose, arabinose, mannose, galactose, acetate, succinate or glycerol. Following 15 min incubation at 37 °C with shaking (180 r.p.m. ), samples were pelleted at 16,000 × g and washed twice with PBS. Pellets were incubated with 2% SDS and DNAse1 and incubated for 10 min at room temperature. The pellet was resuspended in 100 µl of 1× sample buffer (4% β-mercaptoethanol, 10% glycerol, 0.1 M Tris, 2% SDS at pH 6.8). The volume of sample buffer was adjusted according to the fold change in OD600nm compared to time point zero to give equal biomass between samples. Next, samples were incubated for 10 min at 95 °C. They were then checked to contain equal biomass by separation on a 12% Tris–glycine gel, followed by Coomassie staining. The samples were then incubated with 100 µg ml−1 proteinase K overnight at 55 °C to remove protein and again separated on a 12% Tris–glycine gel. The gel was fixed overnight in a solution of 50% methanol and 5% acetic acid. LPS was then quantified using the Pro-Q Emerald 300 Lipopolysaccharide Gel Stain Kit (Invitrogen) according to the manufacturer's protocol. Densitometric analysis was performed using Fiji ImageJ with the resulting values used to interpolate the standard curve performed in Supplementary Fig. 8c, to calculate relative LPS abundance. Undigested total protein samples served as loading controls. GroEL and OmpF were detected by western blotting of total bacterial protein samples, obtained as described above for detection of LPS by sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS–PAGE), using the following primary antibodies: Monoclonal anti-GroEL antibodies were obtained from Abcam (clone, EPR28718-8; catalogue number AB318970, lot number 1096886-3) and used at 1:2,000 dilution; polyclonal Anti-OmpF antibodies were obtained from Invitrogen (catalogue number PAS-121442, lot number 2K4557550) and used at 1:5,000 dilution. Bound primary antibodies were detected using polyclonal Goat Anti-Rabbit HRP conjugated (catalogue number AB6721, lot number GR342221), used at a 1:2,000 dilution, and ECL western blotting detection reagent (Amersham, RPN, 2106). This protocol was provided by R. Hancock (https://cmdr.ubc.ca/bobh/method/kdo-assay/). Briefly, 50 µl of supernatant (generated from 15 min incubation of 1 × 108 c.f.u. ml−1 of stationary-phase bacteria in MM ± glucose, and ±4 µg ml−1) was boiled in 50 µl 0.5 N H2SO4 for 15 min, followed by the sequential addition of 200 µl arsenite reagent (0.3 µM NaAsO2 dissolved in 0.5 N HCl) and 800 µl 42 mM 2-thiobarbituric acid. The solution was boiled for a further 10 min before the addition of 1.5 ml butanol reagent (5.0 ml of concentrated HCl added to 95 ml n-butanol). The butanol layer was separated from the solution by centrifugation for 5 min at 2,500 × g. The Kdo (3-deoxy-D-manno-octulosonic acid) extinction coefficient (OD552nm − OD509nm) was calculated by measuring 200 µl for absorbance at OD552nm and OD509nm in a Tecan Infinite 200 Pro plate reader. The difference in OD was used to interpolate a standard curve generated by performing Kdo analysis on a 1/10 serial dilution of 5 mg ml−1 of purified rough LPS (Supplementary Fig. To quantify ATP levels of stationary-phase bacteria in different carbon sources conditions, an ATP standard curve was first generated using the BacTiter-Glo (Promega) kit. A 10-fold serial dilution series of ATP was performed in MM over a concentration range of 1,000 to 0.01 nM in a white microtitre plate (Greiner Bio-One). The BacTiter-Glo buffer was added to the substrate to make the BacTiter-Glo substrate. This substrate was combined with the dilution series, and luminescence was measured in a Tecan Infinite 200 Pro plate reader using a 100 ms integration time. Following this, stationary-phase bacteria at an inoculum density of 1 × 108 c.f.u. ml−1 was added to MM or MM containing a range of equimolar concentrations of different carbon sources. At each time point (0, 15 and 30 min), 100 µl of the culture was combined with 100 µl of the substrate in a white microtitre plate, and luminescence was measured. ATP concentrations under these different conditions were interpolated from the standard curve. Following three washes in MM, stationary-phase bacteria were added at a final concentration of 1 × 108 c.f.u. ml−1 to 3 ml of MM ± glucose and ±4 µg ml−1 PmB, in the presence of 10 µg ml−1 BODIPY FL Vancomycin (Invitrogen). Cultures were incubated at 37 °C for 15 min, followed by three washes in PBS to remove unbound BODIPY FL Vancomycin. Cultures were resuspended in 3 ml PBS and 200 µl added to a black clear-bottomed 96-well plate (Greiner Bio). Fluorescence and OD600nm were measured in a Tecan Infinite 200 Pro plate reader (excitation at 480 nm, emission at 520 nm). Raw fluorescence readings were normalized according to OD600nm. Experiments were performed on at least three independent occasions, and the resulting data are presented as the arithmetic mean of these biological repeats unless stated otherwise. Error bars, where shown, represent the standard deviation of the mean unless otherwise stated. For single comparisons, a two-tailed Student's t-test was used to analyse the data. For multiple comparisons at a single time point or concentration, data were analysed using one-way analysis of variance (ANOVA) or Kruskal–Wallis test. Where data were obtained at several different time points or concentrations, two-way ANOVA was used for statistical analyses. Appropriate post hoc tests (Dunnett's, Tukey's, Sidak's, Dunn's) were carried out to correct for multiple comparisons, with details provided in the figure legends. Asterisks on graphs indicate significant differences between data. Precise P values for all comparisons are provided in the source data files. All statistical analyses were performed using GraphPad Prism 7 software (GraphPad Software). Where images are shown for example gels, blots or AFM, these are representative examples of at least three replicates, unless otherwise stated. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data supporting the conclusions of this work are available via the UCL Research Data Repository at https://doi.org/10.5522/04/29282072.v1. Source data are provided with this paper. Fanous, J. et al. Limited impact of Salmonella stress and persisters on antibiotic clearance. Lopatkin, A. J. et al. Metabolic state more accurately predicts antibiotic lethality than growth rate. Conlon, B. P. et al. Persister formation in Staphylococcus aureus is associated with ATP depletion. Manuse, S. et al. Bacterial persisters are a stochastically formed subpopulation of low-energy cells. Ahmad, M., Aduru, S. V., Smith, R. P., Zhao, Z. The role of bacterial metabolism in antimicrobial resistance. Hurdle, J. G., O'Neill, A. J., Chopra, I. & Lee, R. E. Targeting bacterial membrane function: an underexploited mechanism for treating persistent infections. & Helaine, S. Persistent bacterial infections and persister cells. Stokes, J. M., Lopatkin, A. J., Lobritz, M. A. & Collins, J. J. Bacterial metabolism and antibiotic efficacy. Wang, B. et al. Antibacterial diamines targeting bacterial membranes. et al. Membrane-active macromolecules kill antibiotic-tolerant bacteria and potentiate antibiotics towards Gram-negative bacteria. Kim, W. et al. A new class of synthetic retinoid antibiotics effective against bacterial persisters. Gray, D. A. et al. Membrane depolarization kills dormant Bacillus subtilis cells by generating a lethal dose of ROS. Zheng, E. J. et al. Discovery of antibiotics that selectively kill metabolically dormant bacteria. & Edwards, A. M. Polymyxin and lipopeptide antibiotics: membrane-targeting drugs of last resort. Nang, S. C., Azad, M. A. K., Velkov, T., Zhou, Q. T. & Li, J. Rescuing the last-line polymyxins: achievements and challenges. Kaye, K. S. et al. Efficacy and safety of sulbactam-durlobactam versus colistin for the treatment of patients with serious infections caused by Acinetobacter baumannii-calcoaceticus complex: a multicentre, randomised, active-controlled, phase 3, non-inferiority clinical trial (ATTACK). Paul, M. et al. Colistin alone versus colistin plus meropenem for treatment of severe infections caused by carbapenem-resistant Gram-negative bacteria: an open-label, randomised controlled trial. Velkov, T., Thompson, P. E., Nation, R. L. & Li, J. Structure-activity relationships of polymyxin antibiotics. Solution structure of polymyxins B and E and effect of binding to lipopolysaccharide: an NMR and molecular modeling study. Potent activity of polymyxin B is associated with long-lived super-stoichiometric accumulation mediated by weak-affinity binding to lipid A. Nat. Trimble, M. J., Mlynárčik, P., Kolář, M. & Hancock, R. E. Polymyxin: alternative mechanisms of action and resistance. Antibiotic polymyxin arranges lipopolysaccharide into crystalline structures to solidify the bacterial membrane. Benn, G., Silhavy, T. J., Kleanthous, C. & Hoogenboom, B. W. Antibiotics and hexagonal order in the bacterial outer membrane. Reply to: Antibiotics and hexagonal order in the bacterial outer membrane. & Chopra, I. Polymyxin B and polymyxin B nonapeptide alter cytoplasmic membrane permeability in Escherichia coli. Leakage of periplasmic proteins from Escherichia coli mediated by polymyxin B nonapeptide. Sabnis, A. et al. Colistin kills bacteria by targeting lipopolysaccharide in the cytoplasmic membrane. Humphrey, M. et al. Colistin resistance in Escherichia coli confers protection of the cytoplasmic but not outer membrane from the polymyxin antibiotic. Antibiotic uptake into gram-negative bacteria. Hancock, R. E. & Chapple, D. S. Peptide antibiotics. Chung, E. S., Wi, Y. M. & Ko, K. S. Variation in formation of persister cells against colistin in Acinetobacter baumannii isolates and its relationship with treatment failure. Cui, P. et al. Disruption of membrane by colistin kills uropathogenic Escherichia coli persisters and enhances killing of other antibiotics. McCall, I. C., Shah, N., Govindan, A., Baquero, F. & Levin, B. R. Antibiotic killing of diversely generated populations of nonreplicating bacteria. Zavascki, A. P. et al. Pharmacokinetics of intravenous polymyxin B in critically ill patients. Hanafin, P. O. et al. A population pharmacokinetic model of polymyxin B based on prospective clinical data to inform dosing in hospitalized patients. Pogue, J. M. et al. Polymyxin susceptibility testing and interpretive breakpoints: recommendations from the United States Committee on Antimicrobial Susceptibility Testing (USCAST). Phase separation in the outer membrane of Escherichia coli. Benn, G. et al. OmpA controls order in the outer membrane and shares the mechanical load. MacNair, C. R. et al. Overcoming mcr-1 mediated colistin resistance with colistin in combination with other antibiotics. Heesterbeek, D. A. et al. Bacterial killing by complement requires membrane attack complex formation via surface-bound C5 convertases. & Edwards, A. M. Lipopolysaccharide as an antibiotic target. The antibiotic novobiocin binds and activates the ATPase that powers lipopolysaccharide transport. Mandler, M. D. et al. Novobiocin enhances polymyxin activity by stimulating lipopolysaccharide transport. Yokota, S. I. et al. Release of large amounts of lipopolysaccharides from Pseudomonas aeruginosa cells reduces their susceptibility to colistin. Zhang, B. et al. Synthesis of vancomycin fluorescent probes that retain antimicrobial activity, identify Gram-positive bacteria, and detect Gram-negative outer membrane damage. Ling, Z. et al. Epidemiology of mobile colistin resistance genes mcr-1 to mcr-9. Liu et al. Emergence of plasmid-mediated colistin resistance mechanism MCR-1 in animals and human beings in China: a microbiological and molecular biological study. Schumann, A. et al. Site-selective modifications by lipid A phosphoethanolamine transferases linked to colistin resistance and bacterial fitness. Balancing mcr-1 expression and bacterial survival is a delicate equilibrium between essential cellular defence mechanisms. Weerakoon, D., Petrov, K., Pedebos, C. & Khalid, S. Polymyxin B1 within the E. coli cell envelope: insights from molecular dynamics simulations. Chung, E. S. & Ko, K. S. Eradication of persister cells of Acinetobacter baumannii through combination of colistin and amikacin antibiotics. Role of the cell envelope in the antibacterial activities of polymyxin B and polymyxin B nonapeptide against Escherichia coli. Paracini, N., Clifton, L. A., Skoda, M. W. A. & Lakey, J. H. Liquid crystalline bacterial outer membranes are critical for antibiotic susceptibility. Ginez, L. D. et al. Changes in fluidity of the E. coli outer membrane in response to temperature, divalent cations and polymyxin-B show two different mechanisms of membrane fluidity adaptation. & Spector, M. P. Starvation- and stationary-phase-induced resistance to the antimicrobial peptide polymyxin B in Salmonella typhimurium is RpoS (sigma(S)) independent and occurs through both phoP-dependent and -independent pathways. & Ko, K. S. Antibiotic efficacy in Escherichia coli and Klebsiella pneumoniae under nutrient limitation and effectiveness of colistin-based antibiotic combinations to eradicate persister cells. Role of reactive oxygen species in antibiotic action and resistance. Ma, B. et al. The antimicrobial peptide thanatin disrupts the bacterial outer membrane and inactivates the NDM-1 metallo-β-lactamase. Wei, X. et al. Murepavadin induces envelope stress response and enhances the killing efficacies of β-lactam antibiotics by impairing the outer membrane integrity of Pseudomonas aeruginosa. Werneburg, M. et al. Inhibition of lipopolysaccharide transport to the outer membrane in Pseudomonas aeruginosa by peptidomimetic antibiotics. Goode, A., Yeh, V. & Bonev, B. B. Interactions of polymyxin B with lipopolysaccharide-containing membranes. Rapid killing of Acinetobacter baumannii by polymyxins is mediated by a hydroxyl radical death pathway. Gadar, K. et al. Disrupting iron homeostasis can potentiate colistin activity and overcome colistin resistance mechanisms in Gram-negative bacteria. Brochmann, R. P. et al. Bactericidal effect of colistin on planktonic Pseudomonas aeruginosa is independent of hydroxyl radical formation. Kolpen, M. et al. Increased bactericidal activity of colistin on Pseudomonas aeruginosa biofilms in anaerobic conditions. Koike, M., Iida, K. & Matsuo, T. Electron microscopic studies on mode of action of polymyxin. Lounatmaa, K., Mäkelä, P. H. & Sarvas, M. Effect of polymyxin on the ultrastructure of the outer membrane of wild-type and polymyxin-resistant strain of Salmonella. Contribution of bacterial outer membrane vesicles to innate bacterial defense. Park, J. et al. A novel decoy strategy for polymyxin resistance in Acinetobacter baumannii. Kulkarni, H. M., Nagaraj, R. & Jagannadham, M. V. Protective role of E. coli outer membrane vesicles against antibiotics. Sabnis, A., Ledger, E. V. K., Pader, V. & Edwards, A. M. Antibiotic interceptors: creating safe spaces for bacteria. Release of outer membrane vesicles by Gram-negative bacteria is a novel envelope stress response. Brand, C., Newton-Foot, M., Grobbelaar, M. & Whitelaw, A. Antibiotic-induced stress responses in Gram-negative bacteria and their role in antibiotic resistance. Steenhuis, M., Ten Hagen-Jongman, C. M., van Ulsen, P. & Luirink, J. Stress-based high-throughput screening assays to identify inhibitors of cell envelope biogenesis. Audrain, B. et al. Induction of the Cpx envelope stress pathway contributes to Escherichia coli tolerance to antimicrobial peptides. The Rcs phosphorelay system is specific to enteric pathogens/commensals and activates ydeI, a gene important for persistent Salmonella infection of mice. Wang, D., Korban, S. S. & Zhao, Y. The Rcs phosphorelay system is essential for pathogenicity in Erwinia amylovora. Rojas, E. R. et al. The outer membrane is an essential load-bearing element in Gram-negative bacteria. Arunmanee, W. et al. Gram-negative trimeric porins have specific LPS binding sites that are essential for porin biogenesis. Peterson, J. H., Yang, L., Gumbart, J. C. & Bernstein, H. D. Conserved lipid-facing basic residues promote the insertion of the porin OmpC into the E. coli outer membrane. Webby, M. N. et al. Lipids mediate supramolecular outer membrane protein assembly in bacteria. Davis, K. P. et al. Critical role of growth medium for detecting drug interactions in Gram-negative bacteria that model in vivo responses. Heithoff, D. M. et al. Re-evaluation of FDA-approved antibiotics with increased diagnostic accuracy for assessment of antimicrobial resistance. Duong, L. et al. Bactericidal activity of mammalian histones is caused by large membrane pore formation. & Finley, F. Lethal effect of complement and lysozyme on polymyxin-treated, serum-resistant Gram-negative bacilli. & Morrison, D. C. Evidence for lipopolysaccharide as the predominant proinflammatory mediator in supernatants of antibiotic-treated bacteria. Jensen, K. F. The Escherichia coli K-12 “wild types” W3110 and MG1655 have an rph frameshift mutation that leads to pyrimidine starvation due to low pyrE expression levels. Silhavy T. J., Berman M. L. & Enquist L. W. Experiments with Gene Fusions (Cold Spring Harbor Laboratory Press, 1984). Braun, M. & Silhavy, T. J. Imp/OstA is required for cell envelope biogenesis in Escherichia coli. Identification and characterization of a new gene of Escherichia coli K-12 involved in outer membrane permeability. Mobley, H. L. et al. Pyelonephritogenic Escherichia coli and killing of cultured human renal proximal tubular epithelial cells: role of hemolysin in some strains. Dortet, L., Bréchard, L., Poirel, L. & Nordmann, P. Rapid detection of carbapenemase-producing Enterobacteriaceae from blood cultures. Wan, Y. et al. Integrated analysis of patient networks and plasmid genomes to investigate a regional, multispecies outbreak of carbapenemase-producing Enterobacterales carrying both blaIMP and mcr-9 genes. Lee, D. G. et al. Genomic analysis reveals that Pseudomonas aeruginosa virulence is combinatorial. Wiegand, I., Hilpert, K. & Hancock, R. E. Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances. Odds, F. C. Synergy, antagonism, and what the chequerboard puts between them. Benn, G., Pyne, A. L. B., Ryadnov, M. G. & Hoogenboom, B. W. Imaging live bacteria at the nanoscale: comparison of immobilisation strategies. Nečas, D. & Klapetek, P. Gwyddion: an open-source software for SPM data analysis. A. et al. A guide for membrane potential measurements in Gram-negative bacteria using voltage-sensitive dyes. Larrouy-Maumus, G., Clements, A., Filloux, A., McCarthy, R. R. & Mostowy, S. Direct detection of lipid A on intact Gram-negative bacteria by MALDI-TOF mass spectrometry. S. Rutherford and K. Buchholz (Genentech), T. Clarke and S. Stoneham (Imperial College London), and G. Benn (University College London/University of Oxford) are thanked for helpful discussions. This work was funded by Biotechnology and Biological Sciences Research Council (award BB/Y003667/1 to A.M.E. and G.L.-M.; BB/X002446/1, BB/X001547/1 and BB/X000370/1 to B.B.B., B.W.H. and A.M.E., respectively) and by the Wellcome Trust (227923/Z/23/Z to B.W.H.). is supported by the Centre for Doctoral Training in the Advanced Characterisation of Materials, funded by the Engineering and Physical Sciences Research Council and Science Foundation Ireland (EP/S023259/1). We acknowledge Biotechnology and Biological Sciences Research Council (BB/R000042/1 to B.W.H.) and Engineering and Physical Sciences Research Council (EP/K031953/1, via the Interdisciplinary Research Centre in Early-Warning Sensing Systems for Infectious Diseases) for funding equipment. A.M.E. is supported in part by the National Institute for Health and Care Research Imperial Biomedical Research Centre. Centre for Bacterial Resistance Biology, Imperial College London, London, UK Carolina Borrelli, Edward J. A. Douglas, Sophia M. A. Riley, Aikaterini Ellas Lemonidi, Gerald Larrouy-Maumus & Andrew M. Edwards Department of Infectious Disease, Imperial College London, London, UK Carolina Borrelli, Edward J. A. Douglas, Sophia M. A. Riley, Aikaterini Ellas Lemonidi & Andrew M. Edwards Carolina Borrelli & Bart W. Hoogenboom Department of Physics & Astronomy, University College London, London, UK Carolina Borrelli & Bart W. Hoogenboom Department of Life Sciences, Imperial College London, London, UK School of Life Sciences, Queen's Medical Centre, University of Nottingham, Nottingham, UK Wen-Jung Lu & Boyan B. Bonev Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed the research; C.B., E.J.A.D, S.M.A.R., A.E.L., W.-J.L., G.L.-M. and A.M.E. performed the research; C.B., E.J.A.D, S.M.A.R., A.E.L., W.-J.L., B.B.B., A.M.E. and B.W.H. Correspondence to Andrew M. Edwards or Bart W. Hoogenboom. holds an executive position at AFM manufacturer Nanosurf. Nanosurf did not play any role in the design or execution of this study. The other authors declare no competing interests. Nature Microbiology thanks Jamie Hobbs, Andrew Mason and James Sáenz for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Visual representation of the set-up for all microbiological assays. E. coli MG1655 was grown in 3 ml of MHB for 16 h at 37oC 180 r.p.m. The culture was washed three times in MM and resuspended at an inoculum density of 1×108 in 3 ml of MM or MM + G. To test exponential phase E. coli, the overnight culture was diluted 1/1000 in fresh MHB and grown for a further 3 h. b, Survival of stationary phase E. coli exposed to 4 µg ml−1 PmB from t = 0 in the presence of a 1/10 dilution series of 0.36% glucose. A 1/100 dilution of 0.36% glucose supported PmB killing, however, a 1/1000 dilution substantially reduced the rate and degree of PmB killing. c, Survival of stationary phase E. coli exposed to 4 µg ml−1 PmB from t = 0 in MM + G or MM with an equimolar concentration of 2-deoxy-D-glucose, as determined by CFU counts. All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Significant differences were determined by two-way repeated measures ANOVA. Exponential (Exp) or – (Stat) E. coli cells were exposed to PmB (4 µg ml−1) in MM ± glucose and samples taken at the indicated time points. In keeping with the data from AFM studies, stationary phase cells exposed to PmB in the presence of glucose began to show surface protrusions from ~30 min, which increased over time. Conversely, stationary phase cells exposed to the polymyxin without glucose showed minimal to no surface protrusions. PmB triggered surface protrusions in exponential phase cells whether glucose was present or not. Images for stationary phase cells at 120 min are shown again in an enlarged image to highlight differences in surface appearance. Scale bar represents 400 nm (N = 1). a, From top to bottom, representative mass spectra showing unmodified lipid A from stationary phase E. coli at time 0 min in MM, stationary phase E. coli at time 15 min in MM, and stationary phase E. coli at time 15 min in MM + G, and stationary phase E. coli at time 15 min in MM + G exposed to 4 µg mL−1 PmB. The peak at ~1800 m/z corresponds to native hexa-acyl diphosphoryl lipid A containing four C14:0 3-OH, one C14:0 and one C12:0100. The peaks at ~1710 and ~1800 m/z correspond to the subtraction or addition of phosphate to the native form of lipid A. b, Overlaid mass spectra of stationary phase E. coli at time 15 min in MM + G (black) and stationary phase E. coli at time 15 min in MM + G exposed to 4 µg mL−1 PmB (red), indicating a substantial reduction in native lipid A. Survival of stationary phase Pseudomonas aeruginosa PA14 exposed to 4 µg ml−1 PmB in MM ± glucose, as determined by CFU counts. b, Survival of a panel of E. coli clinical isolates cultured to stationary phase, exposed to 4 µg ml−1 PmB in MM ± glucose for 2 h. c, Survival of C. freundii and E. asburiase clinical isolates cultured to stationary phase, exposed to 4 µg ml−1 PmB in MM ± glucose for 2 h. d, Survival of a K. pneumoniae clinical isolates cultured to stationary phase, exposed to 4 µg ml−1 PmB in MM ± glucose for 2 h. e, Survival of P. aeruginosa clinical isolates cultured to stationary phase, exposed to 4 µg ml−1 PmB in MM ± glucose for 2 h. f, Survival of A. baumannii clinical isolates cultured to stationary phase, exposed to 4 µg ml−1 PmB. g, OM disruption of stationary phase P. aeruginosa cells during the first 20 min of exposure to 4 µg ml−1 PmB, as determined by uptake of the NPN fluorescent dye. h, OM and IM disruption of stationary phase P. aeruginosa exposed to 4 µg ml−1 PmB in MM ± glucose, as determined by uptake of the fluorescent dye SYTOX green. For g, and h, the blank value refers to the relevant fluorophore in medium without bacteria. All experiments were replicated in n = 3 independent assays, error bars show the standard deviation of the mean. Significant differences were determined by two-way (a - h) repeated measures ANOVA. a, Combined brightfield and fluorescence (SYTOX) images of the AFM scan region for the experiments in Fig. 1b with exponential (Exp) and stationary (Stat) phase E. coli at 90 min post PmB treatment (the circled cells are those chosen for the image sequences shown here and were all SYTOX-positive by 90 mins, except for stationary phase E. coli in MM). b, Higher-magnification AFM height and phase scans of stationary phase E. coli. The surface of untreated cells is covered in a network of pores, here best visible in the phase images. The resolution of pores (trimeric OMPs) is ultimately compromised by the progressive roughening of the OM after PmB is added in the presence of glucose (MM + G). In MM + G, PmB caused disruption to the OM in the form of protrusions. Without glucose the appearance of the OM at the nanoscale did not change. Scalebars: a, 4 µm, b, 100 nm; height scales (scale inset in first row of image at t = 90 min): 20 nm; phase scale: 2 deg (row 2) and 1 deg (row 4). c, Quantification of protrusion height (left) and OM mean roughness (right). Protrusion height data are presented as the mean ± SD of median values of the height of features above 50% of the maximum height of each image. Mean roughness values were computed by Gwyddion and data are presented as the mean ± SD. Measurements were taken from 3 different 500 nm scans of 3 different E. coli MG1655 cells imaged in separate experiments. Levels of total LPS in E. coli exposed, or not, to 4 µg ml−1 PmB for 0-, 15-, and 30 min in MM + G. b, Bar graph of LPS levels according to densitometry analysis of a. Densitometric values were interpolated according to the standard curve in Supplementary Fig. S7C and subsequently normalized to the No treatment controls, to give ‘Relative abundance LPS'. c, Levels of total LPS in P. aeruginosa exposed, or not, to 4 µg ml−1 PmB in MM ± glucose. d, Bar graph of LPS levels according to densitometry analysis of a. Experiments were replicated in n = 4 a, b or n = 3 c, d independent assays. Error bars show the standard deviation of the mean. Significant differences were determined by one-way ANOVA P= **<0.01, ****<0.0001, ns=not significant. a, Survival of stationary phase E. coli exposed to 4 µg ml−1 PmB in MM ± equimolar concentrations of different sugars, as determined by CFU counts. b, Standard curve plotting Log10 ATP concentration (nM) against Log10 luminescence counts. Simple linear regression was performed using prism version 10.4.1, where the blue line represents the line of best fit, and the red dotted line the 95% confidence interval. c, ATP concentration according to standard curve interpolation, of stationary phase E. coli incubated in equimolar concentrations of different sugars across a 30-minute time course. d, Four-way correlation matrix showing correlation coefficients of PmB susceptibility, ATP concentration and densitometric analysis pre- and post-PmB treatment. All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. a, b, c Survival of stationary phase P. aeruginosa exposed, or not, to 4 µg ml−1 PmB in MM + G with or without 1X MIC of LpxC inhibitors CHIR-090 (a) or ACHN-975 (b) or the protein synthesis inhibitor tetracycline (c). (d) AFM phase images showing stationary phase E. coli MG1655 cells exposed to 2.5 µg ml−1 PmB in the presence or not of 0.125 µg ml−1 CHIR-090 in MM + G, shown as a function of time. (e) AFM phase images showing stationary phase E. coli imp4231 cells exposed to 2.5 µg ml−1 PmB in the presence or not of 0.5 µg ml−1 G907 in MM + G, shown as a function of time. Scalebar: d, e 250 nm. Phase scale (scale inset in first row of d and e, at t = 90 min): d, 2.5 deg, e 4 deg (top), 6 deg (bottom) All experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Significant differences were determined by two-way repeated measures ANOVA between PmB-treated and PmB + antibiotic-treated conditions. Previous work has shown that the antibiotic novobiocin promotes the rate of LPS transport from IM to OM via an interaction with LptB41. In turn, this leads to increased susceptibility to polymyxin B, although the mechanism was not established42. Based on the findings described in this manuscript, we hypothesised that increased LPS transport to the OM would promote PmB-mediated LPS loss, which correlates with bacterial killing by the antibiotic. a, b, Checkerboard broth microdilution assay showing the synergistic growth-inhibitory interaction between novobiocin and PmB (a), or PmBN (b) against E. coli, as determined OD595nm after 18 hr incubation and in line with previous findings42. Note: PmBN is an inactive polymyxin analogue that served as a useful control for increased entry of novobiocin caused by OM disruption. (c, d) Survival of E. coli exposed, or not, to 1 µg ml−1 PmB (c) or 1 µg ml−1 PmBN (d) with or without 32 µg ml−1 novobiocin. These concentrations were chosen because they showed maximal synergy in the PmBN checkerboard assay. For (c), there was no killing of E. coli by the polymyxin or novobiocin alone, but a significant reduction in the viability of E. coli in the presence of both antibiotics. By contrast, there was no reduction in bacterial viability in the presence of PmBN with novobiocin. Therefore, bacterial killing in these assays appears to be PmB-mediated, rather than increased ingress of novobiocin into cells caused by OM disruption. e, f, Combined OM and IM disruption of stationary phase E. coli exposed to 1 µg ml−1 PmB (e) or 1 µg ml−1 PmBN (f) in MHB with and without 32 µg ml−1 novobiocin, as determined by uptake of the fluorescent dye SYTOX green. Crucially, in the presence of PmBN, novobiocin did not cause membrane disruption, even at concentrations well above those required to inhibit growth in the checkerboard assay. However, novobiocin promoted PmB-mediated membrane disruption in a dose-responsive manner, confirming that novobiocin promotes the activity of PmB by increasing IM disruption, which is the key step for lethality27,42. Next, we wanted to examine the impact of novobiocin on PmB-mediated LPS loss. (g) Representative SDS-PAGE image of LPS band intensity of stationary phase E. coli exposed, or not, to 1 µg ml−1 PmB with and without 32 µg/ml novobiocin for 15 min in MHB. (h) Densitometric analysis of the LPS gel in (g), showing that PmB or novobiocin exposure alone at 1 µg ml−1 or 32 µg ml−1 respectively, had minimal impact on LPS abundance after 15 mins incubation (N = 4). However, there was a significant drop in LPS levels when PmB and novobiocin were used in combination at these concentrations. Therefore, we concluded that novobiocin enhances PmB activity by promoting LPS loss, most likely via its reported effect on increasing LPS transport from the IM to OM41,42. Unless otherwise stated, all experiments were replicated in n = 3 independent assays. Error bars show the standard deviation of the mean. Significant differences were determined by one- (h) or two-way repeated measures ANOVA (c, d, e, f). AFM low and high magnification scans showing stationary phase E. coli MG1655 exposed to 2.5 µg ml−1 PMBN (a) or 10 mM EDTA (b) in the presence of glucose for up to 90 minutes. Scalebars: large scans 250 nm, higher-magnification scans 100 nm; phase scale (scale inset in third row of a and b, at t = 90 min) (from top to bottom): a, 4 deg (row 1) and 0.7 deg (row 3), b, 5 deg (row 1) and 1 deg (row 3). Height scale (scale inset in third row of a and b, at t = 90 min): a, 3 nm, b, 10 nm. (representative images shown of N = 1). Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Unprocessed gels for Extended Data Fig. Statistical source data. Statistical source data. Statistical source data. Unprocessed gels for Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Polymyxin B lethality requires energy-dependent outer membrane disruption. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03977-0'>Full-spectrum extract from Cannabis sativa DKJ127 for chronic low back pain: a phase 3 randomized placebo-controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 15:37:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Chronic low back pain (CLBP) affects over half a billion people worldwide. Current pharmacologic treatments offer limited efficacy and carry substantial risks, warranting the development of safe and effective alternatives. This multicenter, randomized, placebo-controlled phase 3 trial evaluated the efficacy and safety of VER-01 in CLBP. The primary endpoint of phase A was a change in mean numeric rating scale (NRS) pain intensity, with a change in total neuropathic pain symptom inventory (NPSI) score as a key secondary endpoint in participants with a neuropathic pain component (PainDETECT > 18). The primary endpoint for phase D was time to treatment failure. The study met its primary endpoint in phase A, with a mean pain reduction of −1.9 NRS points in the VER-01 group (mean difference (MD) versus placebo = −0.6, 95% confidence interval (CI) = −0.9 to −0.3; P < 0.001). Pain further decreased to −2.9 NRS points in phase B, with effects sustained through phase C. The study also met its key secondary endpoint of phase A, with a mean NPSI decrease of −14.4 (standard error, 3.3) points from baseline in the VER-01 arm (MD versus placebo = −7.3, 95% CI = −13.2 to −1.3; P = 0.017). Although phase D did not meet its primary endpoint (hazard ratio = 0.75, 95% CI = 0.44–1.27; P = 0.288), pain increased significantly more with placebo upon withdrawal (MD = 0.5, 95% CI = 0.0–1.0; P = 0.034). In phase A, the incidence of adverse events—mostly mild to moderate and transient—was higher with VER-01 than with placebo (83.3% versus 67.3%; P < 0.001). VER-01 was well-tolerated, with no signs of dependence or withdrawal. VER-01 shows potential as a new, safe and effective treatment for CLBP. With over half a billion prevalent cases in 2020, low back pain is globally the leading cause for work loss, disability and reduced quality of life across all ages and in both sexes1,2. When pain persists for more than 3 months, it is defined as chronic low back pain (CLBP). CLBP is often associated with severe impairments in sleep quality and physical function, which further contribute to the overall disease burden4,5. The treatment of CLBP involves a multimodal treatment approach combining pharmacological and nonpharmacological interventions, the latter including physical activity, exercise and physiotherapy6,7,8,9,10. Pharmacological treatment options include the short-term use of nonsteroidal anti-inflammatory drugs (NSAIDs). However, NSAIDs are not suitable for long-term treatment due to severe side effects, including gastrointestinal ulcers, bleeding and an increased risk for cardiovascular events11,12. For patients requiring long-term analgesic treatment, opioids are frequently used, despite severe side effects and safety concerns. Approximately 20% of patients on long-term therapy experience opioid abuse, dependence, tolerance development and withdrawal symptoms13. The widespread use of opioids for CLBP combined with their high risk for dependence, misuse and fatal overdoses has substantially contributed to the global opioid epidemic, resulting in hundreds of thousands of deaths worldwide14,15. Accordingly, several clinical practice guidelines now advise against the use of opioids6,7, and there is widespread consensus among healthcare professionals, patient organizations and regulators on the urgent need to develop new nonaddictive analgesics for the short- and long-term treatment of CLBP with a superior safety profile. However, the quality of evidence supporting the clinical use of cannabis-based products remains low, primarily due to small sample sizes, short treatment durations, inconsistent dosing regimens, heterogeneous outcome measures and variability across cannabis-based preparations16,17,18,19. As naturally occurring botanical substances, cannabis plants exhibit substantial heterogeneity. Variability in the botanical raw material and manufacturing processes contributes to a substantial variety of product compositions, including levels of bioactive constituents20,21. Consequently, findings obtained with one cannabis extract cannot be extrapolated to others without appropriate comparative data. To ensure reproducible results, it is therefore essential to adequately characterize the investigational product and establish consistency across batches. The investigational product VER-01 was comprehensively characterized using chromatographic and spectrometric methods to quantify cannabinoids, terpenes, flavonoids, carotenes, phytosterols, vitamins and fats, and chromatographic fingerprinting confirmed batch-to-batch consistency across multiple lots of VER-01. By providing a large-scale, placebo-controlled phase 3 trial of adequate duration using a chemically well-defined, full-spectrum cannabis extract in CLBP, this study addresses a critical gap in the clinical research of cannabis-based pharmacotherapy in chronic pain. Between 7 July 2021 and 30 June 2023, 1,157 patients were assessed for eligibility and 820 participants were randomly assigned to VER-01 (n = 394) or placebo (n = 426; Fig. A total of 525 participants continued to phase B, 155 continued to phase C and 116 participants were randomly assigned to VER-01 (n = 52) or placebo (n = 64) in phase D (Figs. A total of 815 participants were included in the efficacy analysis of phase A, whereas 116 participants were included in that of phase D. The last visit of the study was on 26 March 2024. The efficacy, safety and tolerability analyses included all randomized participants who received at least one dose of the study medication during the respective study phase. a, The trial design of phase A. b, The trial design of phases B–D. Overall, the two groups in phase A were balanced with respect to baseline characteristics (Table 1). A total of 180 participants (22.0%) had a neuropathic pain component, and 193 (23.5%) were suffering from severe pain (numeric rating scale (NRS) ≥ 7) at baseline. A total of 461 participants (56.6%) were female, with a mean age of 52 years and a mean body mass index (BMI) of 29 kg m−2, showing no notable differences between the two groups. Baseline characteristics of participants with a neuropathic pain component were consistent across both treatment groups and matched those of all participants. Additionally, the demographics and baseline characteristics of participants in phases B, C and D were similar to those in phase A (Extended Data Tables 1–3). The most frequent concurrent diseases were hypertension (35.3%) and obesity (32.0%). In total, 99% of patients reported prior use of analgesics, with nonsteroidal anti-inflammatory and antirheumatic drugs being the most frequently reported medication (96.7%). The proportion of participants with at least two prior optimized analgesic therapies was balanced between arms, with 95.4% in the VER-01 arm and 94.8% in the placebo arm. Pain intensity at baseline was moderate to severe in both groups, with a mean NRS pain intensity of 6.1 (s.d. The study met its primary endpoint in phase A. Mean pain intensity significantly decreased from baseline by −1.9 (s.e. = 0.2) NRS points in the VER-01 arm compared to −1.4 (s.e. VER-01 demonstrated a greater pain reduction compared to placebo with a mean difference (MD) of −0.6 (95% confidence interval (CI) = −0.9 to −0.3; P < 0.001). The difference between VER-01 and placebo was significantly in favor of VER-01 in every single study week of the 12-week treatment phase, showing consistent improvement over time (Fig. Sensitivity analyses using both last observation carried forward (LOCF) and baseline observation carried forward (BOCF) imputation yielded similar results to the primary analysis and remained statistically significant (Supplementary Tables 10 and 11). Post hoc analysis revealed no statistical interaction between treatment and sex (Supplementary Table 4). a, Mean NRS pain intensity in phases A and B, as measured on an 11-point NRS for each study week of the placebo-controlled study phase A (VER-01, n = 390; placebo, n = 425) and the open-label phase B (VER-01, n = 524). b, Mean NPSI total scores for the subgroup of participants with a neuropathic pain component during phase A (VER-01, n = 88; placebo, n = 91). c, Change in mean NRS pain intensity after randomized withdrawal in phase D (VER-01, n = 51; placebo, n = 63). All bars display the range from mean ± s.e. The difference between VER-01 and placebo was even more pronounced in participants with a neuropathic pain component (MD = −1.5, 95% CI = −2.2 to −0.9; P < 0.001). Moreover, post hoc analyses revealed particularly pronounced effects of VER-01 in participants with severe pain (MD = −1.0, 95% CI = −1.8 to −0.1; P = 0.011). The study also met its key secondary endpoint in phase A, with significant improvements in neuropathic symptoms (as measured by the Neuropathic Pain Symptom Inventory (NPSI)) among participants with a PainDETECT score >18 at baseline. The mean NPSI total scores at baseline were comparable between groups, with 47.1 (s.d. The mean NPSI total score decreased by −14.4 (s.e. = 3.3) points from baseline in the VER-01 arm compared to −7.2 (s.e. = 2.8) in the placebo arm, with an MD of −7.3 (95% CI = −13.2 to −1.3; P = 0.017). The difference between VER-01 and placebo was significantly in favor of VER-01 for every single visit of the treatment phase, showing consistent improvement over time (Fig. The results of the primary and key secondary endpoints were further supported by all secondary efficacy endpoints (Table 2). The rate of participants with a ≥30% pain reduction was significantly higher for VER-01 compared to placebo (54.1% versus 39.5%), resulting in a number needed to treat to benefit (NNTB) of 6.8 (95% CI = 4.42–15.05; P < 0.001). Similarly, the rate of participants with a ≥50% pain reduction (32.2% versus 22.8%; P = 0.010) and a ≥2-point pain reduction (46.9% versus 35.6%; P = 0.001) was significantly higher in the VER-01 arm. Moreover, participants in the VER-01 arm took only about half the amount of rescue medication compared to participants in the placebo arm (mean (s.d.) Additionally, participants in the VER-01 group reported significant improvements in both sleep quality and physical function. Sleep quality improved by −2.2 NRS points in the VER-01 arm compared to −1.5 in the placebo arm, with an MD of −0.7 NRS points (95% CI, −1.0 to −0.3; P < 0.001). Change from baseline (CFB) in Roland Morris Disability Questionnaire (RMDQ) total score was also significantly in favor of VER-01, with −3.1 (4.0) versus −2.0 (4.1) points (MD = −1.1, 95% CI = −1.8 to −1.1; P < 0.001). Post hoc analysis revealed that 51.7% of participants in the VER-01 arm achieved an improvement of the RMDQ score by at least 30% from baseline compared to 42.2% in the placebo arm (P = 0.019). Improvements in pain intensity, sleep quality and physical function were associated with elevated quality of life and a positive global impression of change. Participants experienced further reductions in NRS pain intensity during the 6-month treatment period of phase B, with a decrease of 3 points compared to the phase A baseline (s.d. The percentage of participants experiencing a ≥30% reduction in pain further increased to 73.9%, while the percentage for those achieving a ≥50% pain reduction increased to 51.8%. In addition, participants reported further improvements in physical function, sleep quality and quality of life (Extended Data Table 4). Participants who completed phase B and participated in phase C were able to maintain their pain reduction over an additional 6-month period (Supplementary Fig. There were no signs of diminishing efficacy or dose escalation over time (Extended Data Fig. Time to treatment failure did not differ significantly between VER-01 and placebo in phase D (hazard ratio = 0.75, 95% CI = 0.44–1.27; P = 0.288), with median times of 22 days for VER-01 and 11 days for placebo. However, following withdrawal, participants receiving placebo experienced a significant pain increase from the phase D baseline compared to VER-01 (MD = 0.5, s.d. In phase A, treatment-emergent adverse events (TEAEs) were reported by 83.3% of participants in the VER-01 group, compared to 67.3% in the placebo group (P < 0.001). The rate of serious adverse events (SAEs) was comparable between VER-01 and placebo (6.2% versus 6.8%; P = 0.699; Table 3). The most common TEAEs (≥10% of participants in the VER-01 arm) were dizziness, headache, nasopharyngitis, fatigue, nausea, COVID-19, dry mouth and somnolence (Table 3). Most TEAEs for VER-01 were of mild-to-moderate intensity (94%). The incidence of drug-related TEAEs decreased over the course of the treatment period of phase A, converging toward a weekly incidence below 3% (Supplementary Table 8). Discontinuation due to AEs occurred in 17.3% of participants receiving VER-01, compared to 3.5% in the placebo (P < 0.001), most commonly due to dizziness, somnolence and nausea. Statistical comparisons of adverse event (AE) incidences were conducted post hoc. No deaths occurred during the study, and no clinically important treatment-related changes were observed for VER-01 in clinical laboratory parameters, vital signs or electrocardiograms (ECGs) when compared to placebo or throughout the study. Satisfaction with tolerability further increased during open-label, long-term treatment, reaching 83% at the end of phase B and 84% at the end of phase C. No AEs indicative of drug abuse, dependence or withdrawal, as classified under the respective standardized MedDRA queries, were reported (Supplementary Tables 5 and 9). Furthermore, there were no signs of drug abuse based on urine drug tests and the addiction behavior checklist (ABC), and no participant fulfilled the classification criteria for substance dependence according to the 10th revision of the International Classification of Diseases (ICD-10). There were no withdrawal symptoms as measured with the cannabis withdrawal scale (CWS) after abrupt treatment discontinuation in phase D. The daily total intensity score and daily total functional impairment score of the CWS were comparable between VER-01 and placebo) Supplementary Figs. A pooled analysis of TEAEs for all study phases is provided in Supplementary Tables 5–7. The results of the VER-CLBP-001 trial demonstrate that VER-01 provides meaningful pain reduction compared to the placebo, accompanied by distinct improvements in physical function and sleep quality, two key factors that contribute to participants perceiving the effects of VER-01 as clinically meaningful. Additionally, participants in the VER-01 arm required substantially lower rescue medication use less rescue medication. These results are of high clinical relevance, as substantiated by a significantly higher proportion of participants randomized to VER-01 achieving well-recognized thresholds of meaningful pain relief, including a ≥30% and ≥50% pain reduction, as well as a ≥2-point decrease on the 11-point NRS22,23. Notably, the NNTB for achieving a ≥30% pain response was 6.8, substantially below the widely accepted clinical relevance threshold of NNTB ≤10 in pain research and below the NNTB = 9 achieved with opioids in CLBP24,25,26. Moreover, significantly more participants in the VER-01 group reported a meaningful improvement in symptoms on the PGIC scale compared to the placebo group, an important indicator of the clinical relevance of treatment effects as recognized by the European Medicines Agency guideline on pain27. Importantly, prolonged treatment with VER-01 was associated with further reductions in pain intensity, as well as continued improvements in physical function, sleep quality and health-related quality of life. Notably, the treatment effect was even more pronounced in participants with a neuropathic pain component and those with severe pain at baseline. VER-01 was generally well-tolerated, particularly during long-term use. Most AEs associated with VER-01 were observed during the initial titration phase, were self-limiting and declined markedly within the first few days of treatment, suggesting a rapid onset of sustained tolerability. With 17%, the rate of AE-related discontinuations was higher than with placebo but below those reported for opioids in comparable study designs, with early discontinuations mainly seen in the titration phase28,29,30. Nature and timing of AE-related discontinuations suggest that physicians should advise patients that transient sensations of dizziness, somnolence and nausea may occur in the early titration phase to support treatment adherence. The overall rate of SAEs was comparable to placebo, and there were no signs of dose escalation, abuse, dependence or withdrawal. These results are particularly relevant given the substantial unmet medical need in CLBP, where many patients remain inadequately treated due to the inherent limitations of current pharmacological options, primarily NSAIDs and opioids. NSAIDs are unsuitable for long-term use due to severe gastrointestinal and cardiovascular risks11,12, while opioids are associated with a high risk of addiction, tolerance development, withdrawal symptoms and have a major role in the global opioid crisis13,14,15,31. Against this background, VER-01 presents a promising new nonaddictive, safe and effective treatment option, particularly for long-term use. Another important advantage of VER-01 over existing pharmacotherapies is its ability to significantly improve sleep quality and physical function. Chronic pain, particularly CLBP, is often associated with severe impairments in these domains, which not only diminish quality of life but also contribute to persistent pain and disease progression. Physical disability in CLBP is perpetuated by a vicious cycle of pain, stiffness and fear of movement, leading to progressive functional decline and reduced mobility32. Without effective intervention, this cycle reinforces long-term disability and further limits treatment success. Likewise, sleep disturbances are not merely a secondary symptom of pain but a key contributor to disease progression. As a result, patients with disrupted sleep often experience greater disability, more persistent pain and a higher prevalence of comorbid depression33,34. Given these interdependent factors, addressing not only pain but also improving physical function and sleep quality is essential for the effective management of CLBP. Recent meta-analyses have shown that existing pharmacotherapies, including NSAIDs and opioids, have only a small and unlikely clinically relevant effect in these domains, highlighting a major gap in current treatment options35,36. By demonstrating clinically meaningful improvements not only in pain but also in physical function and sleep quality, VER-01 represents a promising therapeutic advance that directly targets the multifaceted burden of CLBP. Notably, there remains a lack of rigorously designed, double-blind, placebo-controlled, multicenter clinical trials evaluating the efficacy and safety of standardized cannabis-based products in well-defined pain conditions, with well-defined dosing regimens and sufficiently long treatment durations. This double-blind, placebo-controlled, multicenter clinical trial addresses these gaps and contributes valuable evidence to the field, underscoring the importance of advancing the clinical understanding and therapeutic potential of full-spectrum cannabis extracts. Some limitations of this study merit consideration. First, VER-01 was not directly compared with opioids; however, a dedicated follow-up study, evaluating VER-01 against standard opioid therapy, has already been conducted and will be published separately37. Second, a formal blinding questionnaire was not administered and prior cannabis experience was not recorded, but multiple indicators support successful blinding—the magnitude of the placebo response closely mirrors that observed in other blinded, randomized, controlled CLBP trials28,29, and both treatment arms exhibited a similar, rapid NRS pain decline during the titration period. Furthermore, the nature of AEs was similar between groups, and no instances of unintentional unblinding were reported. Another limitation is that cognitive function was not formally assessed. Although there were no patient reports indicative of impairment, future studies involving cannabinoids might consider incorporating specific cognitive assessments. Finally, while the difference in time to treatment failure in phase D did not reach statistical significance, the observed trend strongly favors VER-01 with a hazard ratio of 0.75 (P = 0.288). The use of time to treatment failure as the primary endpoint to assess maintenance of efficacy represented a novel approach in CLBP, which led to uncertainty in estimating an appropriate sample size. However, the magnitude and direction of the effect, as well as the significantly larger pain increase among patients receiving placebo following withdrawal (MD = 0.5; P = 0.034), suggest that the lack of statistical significance in phase D reflects insufficient power rather than waning efficacy. In conclusion, this phase 3 study provides robust evidence supporting the efficacy and safety of VER-01 in the treatment of CLBP. These findings highlight the importance of further research with VER-01 in other chronic pain conditions and suggest that VER-01 could play an important role in modern pain management. The trial was divided into four phases (Fig. Phase A had a randomized, double-blind, placebo-controlled design with 1 week of run-in to assess baseline pain intensity, followed by 3 weeks of titration and 12 weeks of treatment. All participants entering phase B underwent retitration, as blinding was maintained for phase A throughout the entire study. Phase D had a double-blind, placebo-controlled, randomized withdrawal design with 4 weeks of treatment followed by 2 weeks of washout. The trial was conducted at 66 outpatient sites and university-based hospitals in Germany and Austria in accordance with the principles of the Declaration of Helsinki, the Good Clinical Practice guidelines of the International Council for Harmonization and applicable regulatory requirements. The trial was approved by ethics committees in each country (Supplementary Table 15), and written informed consent was obtained from all participants. No compensation was offered for participation in the study, except for reimbursement of meals and travel expenses. The trial was prospectively registered with ClinicalTrials.gov (NCT04940741) and EudraCT (2020-000107-36). Patients were primarily recruited via notices in doctors' offices, adverts in magazines and online advertising. Additionally, investigators participating in the study recruited patients from their existing databases. This study enrolled individuals who were at least 18 years of age and diagnosed with CLBP (low back pain for at least 3 months), with or without a neuropathic pain component. Individuals were only included if no treatable specific somatic cause for their CLBP was identified (for example, herniated vertebral disk) based on a detailed medical history assessment and a clinical examination. The presence of a neuropathic pain component at baseline was confirmed using the painDETECT questionnaire with a cut-off score >18 (ref. Eligible individuals must have had, on average, a pain score of at least 4 points on an 11-point NRS 1 month before enrollment and during the run-in period. Individuals were eligible if drug treatment was indicated and previous optimized treatments with nonopioid analgesics (for example, ibuprofen, diclofenac and metamizol) did not lead to sufficient pain relief or were unsuitable due to contraindications or intolerance. Treatment was considered optimized when further dose escalation was medically inadvisable due to potential adverse effects or when it was unlikely that a higher dose would offer additional therapeutic benefit. Participants were allowed to continue nondrug therapies for CLBP if these were maintained unchanged for at least 8 weeks before enrollment and if patients were willing to continue their therapy during phase A. Participants were eligible for inclusion into phase B if they had successfully completed phase A. Participants completing phase B were eligible for inclusion in phase D if they had a clinically meaningful reduction in NRS pain scores (≥30%) at the end of phase B compared to baseline of phase A. Participants not meeting inclusion criteria for phase D were eligible for enrollment in phase C. Participants were excluded if they had other painful comorbidities potentially interfering with their low back pain rating, a severe mental illness (for example, psychosis, schizophrenia and bipolar disorder), or a history of alcohol, drug or medication abuse. Moreover, participants who had used cannabis-based medicinal products within 30 days before screening were excluded. A full list of inclusion and exclusion criteria is provided in the Supplementary Note—Eligibility criteria. The investigational product VER-01 is a standardized full-spectrum extract derived from the patented Cannabis sativa L. strain DKJ127. The full-spectrum extract of the botanical raw material is manufactured in a Good Manufacturing Practice (GMP) certified facility and standardized to 5% tetrahydrocannabinol (THC). Each dose unit (119 µl) of the finished product VER-01 contains 50 µl of the full-spectrum extract, delivering 2.5 mg THC, 0.1 mg cannabigerol and 0.02 mg cannabidiol, with sesame oil as excipient. Moreover, VER-01 contains a complex, well-characterized mixture of terpenes, flavonoids, carotenes, phytosterols and other bioactive compounds. The placebo contained sesame oil, cannabis aroma and colorants to mimic the appearance and sensory characteristics of VER-01. The study products were provided in 30 ml amber glass bottles and taken orally with a dosing syringe, following the European Pain Federation recommendations, which advocate for oral preparations in the management of chronic pain40. The study included two randomizations, conducted at the beginning of phases A and D. Eligible participants were randomly assigned in a 1:1 ratio to VER-01 or placebo. Randomization was stratified according to the presence of a neuropathic pain component. Participants, investigators and study site personnel were masked to treatment assignment. VER-01 and the matching placebo were dispensed in identical amber glass bottles. At the screening visit, participants were provided with an eDiary and instructed on its use. Most assessments were recorded daily in the eDiary, while others were recorded on paper at the trial site and subsequently transferred to the electronic data capture system by site personnel. At visit A2, eligible participants were randomized. Participants were advised to take the study drug either consistently with or without food. The participants' individual optimal dose (both VER-01 and placebo) was determined during a 3-week titration period, following a written titration scheme. The morning and evening doses were adjusted independently. Intake of analgesics other than rescue medication was prohibited during phase A, the last 3 weeks of phase B and phase D. The use of rescue medication (ibuprofen) was restricted to a maximum of 3 days per week and a daily dose of 2,400 mg. In case ibuprofen was contraindicated, paracetamol was offered at a maximum daily dose of 4,000 mg. Participants were not allowed to initiate a nondrug therapy during phase A, nor during the last 9 weeks preceding inclusion into phase D. If an ongoing nondrug therapy was already in place, it had to be continued unchanged throughout participation in phases A and D. Participants used the eDiary to record their dosage, pain intensity (11-point NRS), sleep quality (11-point NRS) and use of rescue medication throughout all study phases. Additionally, participants used the eDiary to record potential withdrawal symptoms during phase D, using the CWS. Phase A included six visits (Fig. 2)—screening (A1), randomization (A2), end of titration (A3) and three visits at 4-week intervals during the treatment phase (A4–A6). Phase B comprised four visits—one at the end of titration (B7) and three visits at 2-month intervals during the treatment phase (B8—B10). Phase C consisted of three visits at 2-month intervals during the treatment phase (C11–C13) and a follow-up visit (C14). Phase D involved two visits—one at the end of treatment (D11) and a follow-up visit (D12). Safety assessments included AEs, laboratory parameters, vital signs, physical examinations, urine drug screens, pregnancy testing, 24-h ECG assessments (for a subset of 120 participants) and evaluation of substance dependence according to ICD-10 (F12.2) criteria. Investigators assessed the severity and causality of AEs, documenting onset, duration, seriousness and any actions or treatments taken. A full schedule of assessments is provided in the attached protocol. The primary endpoint in phase A was the CFB (week −1) to week 15 in the mean weekly pain intensity in the morning, measured on an 11-point NRS (0 = ‘no pain' and 10 = ‘worst pain imaginable'). Each item was scored on an 11-point NRS. No primary endpoint was defined for the open-label study phases B and C. The primary endpoint of phase D was the time to treatment failure, defined as an increase in the 7-day mean of NRS morning pain intensity by ≥20% and ≥1 point compared to the phase D baseline week 44. Across all study phases, the proportion of participants achieving ≥30% and ≥50% pain reduction was evaluated, reflecting outcomes widely considered clinically meaningful22,23,42. Additional diary-based secondary efficacy endpoints assessed in all study phases included change in mean pain intensity (11-point NRS), intake of rescue medication, sleep quality on an 11-point NRS (0 = ‘not impacted'; 10 = ‘completely impacted') and the proportion of participants with ≥30% and ≥50% sleep quality improvement. Visit-based secondary efficacy endpoints assessed in all study phases included CFB in NPSI total scores, PGIC on a seven-point Likert scale using the question ‘How is your low back pain in comparison to before participation in the study?' (0 = ‘very much better' to 6 = ‘very much worse') and quality of life evaluated with the short-form health survey 36v2 (SF-36)43,44. The SF-36 contains 36 questions about physical and mental well-being and was evaluated based on eight domain scores (physical functioning, role-physical, bodily pain, general health, vitality, social functioning, role-emotional and mental health), which were summarized to form two higher-ordered component summary measures (PCS and the mental health component summary). The following visit-based secondary efficacy endpoints were additionally assessed in study phase A: the degree of disability was evaluated with the RMDQ at baseline and end of treatment45. The RMDQ is a 24-item questionnaire that assesses the impact of low back pain on functional activities, with scores ranging from 0 to 24, where higher scores indicate greater disability. Higher sleep scores indicate a clinically favorable outcome. Safety was assessed based on the incidence of AEs, including their seriousness, severity and relationship to the study drug. Additionally, patients' satisfaction with tolerability was evaluated on a visit-by-visit basis using a five-point Likert scale. Substance dependence and abuse potential were measured visit-based with the ABC, a 20-item checklist used to monitor signs of addiction47. An ABC sum score of 3 or more indicates inappropriate drug use. Withdrawal symptoms were recorded daily in the eDiary with the CWS during treatment in phase D and during washout in phases C and D. The CWS consists of 19 items representing potential withdrawal symptoms associated with cannabis withdrawal. Each item is rated on a 0–10 scale, where 0 indicates ‘not at all' and 10 indicates ‘extremely'48. To demonstrate the superiority of VER-01 over placebo for the primary endpoint in phase A, a total of 732 participants had to be randomized in a 1:1 ratio. The sample size calculation was based on an assumed treatment difference of 0.6 NRS points, an s.d. For the key secondary endpoint of phase A, 180 participants with a neuropathic pain component had to be randomized in a 1:1 ratio to show superiority of VER-01 versus placebo. The enrollment was stopped after 808 participants were randomized. Eligible participants who were already in the run-in phase were still allowed to be randomized. Demographic data and other phase A baseline characteristics were summarized by treatment group and overall, separately for each study phase, based on the defined analysis sets. The primary endpoint of phase A was tested by an analysis of covariance model, with treatment as the main effect and with baseline characteristics (presence of a neuropathic pain component; NRS morning pain intensity, age, sex and country) as covariates. Depending on the type of intercurrent event, data were imputed assuming missing at random (MAR) or missing not at random (MNAR). If data were MAR, these were imputed using a multiple imputation (MI) model based on available data from similar participants in the same treatment group. If data were MNAR (for example, early discontinuation due to AEs or loss of efficacy), data were imputed under the conservative approach of ‘jump to reference'. This approach was considered conservative because it assumes that participants discontinuing VER-01 would experience outcomes similar to those in the placebo arm, regardless of any prior treatment benefit. Evidence from enriched-enrollment trials49,50,51 suggests that analgesic effects achieved before treatment discontinuation typically persist for several weeks, making a pain rebound above placebo unlikely. Additionally, participants discontinuing active treatment likely continue to benefit from nonspecific trial effects—such as positive expectations, enhanced clinical monitoring and behavioral adjustments—that are equally present in the control group. Finally, the nature of AEs prompting discontinuation was presumed to be mild to moderate and transient, making a lasting pain outcome worse than placebo unlikely. MI based on the MNAR assumption was performed using the suggested placebo MI algorithm in ref. In general, a two-step MI approach was followed. In the first step, an MCMC imputation model was calculated on 100 imputations to transform an arbitrary missing data pattern into a monotone missing pattern. In a second MI step, a monotone regression model was calculated on these 100 imputations. Parameter estimates were combined according to Rubin's rule. The key secondary endpoint was evaluated similarly to the primary endpoint of phase A using an analysis of covariance model, with treatment as the main effect and baseline characteristics (NPSI total score, age, sex and country) as covariates. Safety was assessed based on AE incidence with post hoc comparisons performed using a two-sided chi-squared test. To collect efficacy and safety data for at least 300 participants over 6 months, 500 participants were planned to be included in phase B. In addition, 150 participants were planned to be included in phase C to collect long-term efficacy and safety data for at least 100 participants over 12 months. For the sample size calculation of phase D, a treatment failure rate of 25% was assumed for VER-01 and 55% for placebo, building on ref. For the log-rank test comparing the two survival curves with regard to time to treatment failure, a sample size of 78 (39 per treatment group) was required to achieve 80% statistical power under a two-sided significance level of 5%. The primary endpoint of phase D was analyzed using a Cox proportional hazards model with treatment as the main effect and baseline characteristics (presence of a neuropathic pain component, NRS pain intensity at week 43, age and sex) as covariates. Events indicating treatment failure (such as overdosage of rescue medication or study discontinuation due to intolerability, lack of efficacy or noncompliance) were classified as treatment failure events. Data from participants with intercurrent events not indicating treatment failure were considered missing and imputed under the MAR assumption. Discontinuation of the study for reasons not indicating treatment failure led to censoring in time-to-event analyses. The analysis sets for phases A and D included all randomized participants who received at least one dose of study medication in the respective phase, with participants assigned to treatment groups as randomized (full analysis sets) or as treated (safety analysis sets). The safety analysis sets for phases B and C included all participants who received at least one dose of study medication in the respective phase. No independent data monitoring committee was used. All statistical analyses were conducted by an independent clinical research organization with SAS software, version 9.4 (SAS Institute). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Access to anonymized individual data and blank case report forms that underlie the results reported in this article can be requested by qualified researchers for academic purposes. Vertanical provides access within 3 months following review and approval of a research proposal, statistical analysis plan and execution of a data access agreement. Data are available to request after the indicated study has been approved in the USA and the European Union for a period of 5 years. Data will be shared through a secure online platform. ClinCase by Quadratek Data Solutions, version 2.7, was used to capture and store participants' clinical data. Analyses were performed with SAS software, version 9.4 (SAS Institute), in keeping with the statistical analysis plan. Global low back pain prevalence and years lived with disability from 1990 to 2017: estimates from the Global Burden of Disease Study 2017. Global, regional, and national burden of low back pain, 1990–2020, its attributable risk factors, and projections to 2050: a systematic analysis of the Global Burden of Disease Study 2021. Nijs, J. et al. Nociceptive, neuropathic, or nociplastic low back pain? The low back pain phenotyping (BACPAP) consortium's international and multidisciplinary consensus recommendations. Kelly, G. A., Blake, C., Power, C. K., O'Keeffe, D. & Fullen, B. M. The association between chronic low back pain and sleep: a systematic review. Tagliaferri, S. D. et al. Domains of chronic low back pain and assessing treatment effectiveness: a clinical perspective. Zhou, T., Salman, D. & McGregor, A. H. Recent clinical practice guidelines for the management of low back pain: a global comparison. Chronic low back pain: a narrative review of recent international guidelines for diagnosis and conservative treatment. Oliveira, C. B. et al. Clinical practice guidelines for the management of non-specific low back pain in primary care: an updated overview. Lim, T. H. et al. Nonpharmacological spine pain management in clinical practice guidelines: a systematic review using AGREE II and AGREE-REX tools. WHO guideline for non-surgical management of chronic primary low back pain in adults in primary and community care settings. Salvo, F. et al. Cardiovascular events associated with the long-term use of NSAIDs: a review of randomized controlled trials and observational studies. Wirth, T., Lafforgue, P. & Pham, T. NSAID: current limits to prescription. Vowles, K. E. et al. Rates of opioid misuse, abuse, and addiction in chronic pain: a systematic review and data synthesis. The Lancet Regional Health—Americas Opioid crisis: addiction, overprescription, and insufficient primary prevention. Volkow, N. D. & McLellan, A. T. Opioid abuse in chronic pain—misconceptions and mitigation strategies. The efficacy of cannabis in reducing back pain: a systematic review. Fisher, E. et al. Cannabinoids, cannabis, and cannabis-based medicine for pain management: a systematic review of randomised controlled trials. Strand, N. et al. Medical cannabis: a review from the American Society of Pain and Neuroscience. Lee, C. et al. Medical cannabis and its efficacy/effectiveness for the treatment of low-back pain: a systematic review. Liktor-Busa, E., Keresztes, A., LaVigne, J., Streicher, J. M. & Largent-Milnes, T. M. Analgesic potential of terpenes derived from Cannabis sativa. Botanical drug development: guidance for industry. Dworkin, R. H. et al. Interpreting the clinical importance of treatment outcomes in chronic pain clinical trials: IMMPACT recommendations. Interpretation of chronic pain clinical trial outcomes: IMMPACT recommended considerations. Petzke, F., Klose, P., Welsch, P., Sommer, C. & Häuser, W. Opioids for chronic low back pain: an updated systematic review and meta-analysis of efficacy, tolerability and safety in randomized placebo-controlled studies of at least 4 weeks of double-blind duration. & Häuser, W. Serotonin and noradrenaline reuptake inhibitors (SNRIs) for fibromyalgia. Häuser, W. et al. Cannabis-based medicines and medical cannabis for adults with cancer pain. Guideline on the clinical development of medicinal products intended for the treatment of pain (EMA/CHMP/970057/2011). Webster, L. R. et al. Oxytrex minimizes physical dependence while providing effective analgesia: a randomized controlled trial in low back pain. Buynak, R. et al. Efficacy and safety of tapentadol extended release for the management of chronic low back pain: results of a prospective, randomized, double-blind, placebo- and active-controlled phase III study. Ruoff, G. E., Rosenthal, N., Jordan, D., Karim, R. & Kamin, M. Tramadol/acetaminophen combination tablets for the treatment of chronic lower back pain: a multicenter, randomized, double-blind, placebo-controlled outpatient study. Crockett, S. D. et al. American gastroenterological association institute guideline on the medical management of opioid-induced constipation. Alfalogy, E., Mahfouz, S., Elmedany, S., Hariri, N. & Fallatah, S. Chronic low back pain: prevalence, impact on quality of life, and predictors of future disability. Whale, K. & Gooberman-Hill, R. The importance of sleep for people with chronic pain: current insights and evidence. Duo, L. et al. Sleep disorders in chronic pain and its neurochemical mechanisms: a narrative review. Effects of pharmacotherapy on sleep-related outcomes in adults with chronic low back pain: a systematic review and meta-analysis of randomised controlled trials. Nury, E. et al. Efficacy and safety of strong opioids for chronic noncancer pain and chronic low back pain: a systematic review and meta-analyses. Multicentre, randomized, open-label study to prove an additional benefit of the full-spectrum cannabis extract ver-01 over opioids in the treatment of patients with chronic non-specific low back pain. Draft guidance for industry on analgesic indications: developing drug and biological products; availability. Freynhagen, R., Ralf, B., Ulrich, G. & Tölle, T. R. painDETECT: a new screening questionnaire to identify neuropathic components in patients with back pain. Häuser, W. et al. European Pain Federation (EFIC) position paper on appropriate use of cannabis-based medicines and medical cannabis for chronic pain management. Bouhassira, D. et al. Development and validation of the neuropathic pain symptom inventory. Farrar, J. T., Young, J. P. J., LaMoreaux, L., Werth, J. L. & Poole, M. R. Clinical importance of changes in chronic pain intensity measured on an 11-point numerical pain rating scale. Ware, J. E. Jr. & Sherbourne, C. D. The MOS 36-item short-form health survey (SF-36). How to Score Version 2 of the SF-36® Health Survey (QualityMetric Incorporated, 2000). Part I: development of a reliable and sensitive measure of disability in low-back pain. Hays, R. D., Martin, S. A., Sesti, A. M. & Spritzer, K. L. Psychometric properties of the Medical Outcomes Study Sleep measure. The addiction behaviors checklist: validation of a new clinician-based measure of inappropriate opioid use in chronic pain. Allsop, D. J., Norberg, M. M., Copeland, J., Fu, S. & Budney, A. J. The cannabis withdrawal scale development: patterns and predictors of cannabis withdrawal and distress. Markman, J. et al. SUMMIT-07: a randomized trial of NKTR-181, a new molecular entity, full mu-opioid receptor agonist for chronic low-back pain. Wen, W., Sitar, S., Lynch, S. Y., He, E. & Ripa, S. R. A multicenter, randomized, double-blind, placebo-controlled trial to assess the efficacy and safety of single-entity, once-daily hydrocodone tablets in patients with uncontrolled moderate to severe chronic low back pain. Rauck, R. L. et al. Single-entity hydrocodone extended-release capsules in opioid-tolerant subjects with moderate-to-severe chronic low back pain: a randomized double-blind, placebo-controlled study. Ayele, B. T., Ilya, L., Geert, M. & Mallinckrodt, C. H. A multiple-imputation-based approach to sensitivity analyses and effectiveness assessments in longitudinal clinical trials. Langford, R. M. et al. A double-blind, randomized, placebo-controlled, parallel-group study of THC/CBD oromucosal spray in combination with the existing treatment regimen, in the relief of central neuropathic pain in patients with multiple sclerosis. This clinical trial was sponsored by Vertanical, which contributed to the study design and provided support with medical writing. Data collation and statistical analysis were conducted by an independent clinical research organization. No honoraria or payments were made for authorship. Open access funding provided by Medizinische Hochschule Hannover (MHH). Department of Anesthesiology and Intensive Care Medicine, Pain Clinic, Hannover Medical School, Hannover, Germany Metronomia Clinical Research GmbH, Munich, Germany Medical Center Pain Medicine and Mental Health, Saarbrücken, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar participated in data management and statistical analysis. directly accessed and verified the data. has received speaker honoraria from the Federal Association of Pharmaceutical Cannabinoid Companies (BPC), Cannabis Social Club Bolzano, Cannamedical, Demecan, Grunenthal, Hormosan, Meet Your Master, Medical Service of the Health Insurance Funds, Novartis, Stadapharm and Tilray. He has received consultancy fees from Tetra Pharm and Vertanical. He has received expert opinion fees from several local and social courts. His institution has received research support from Friedrich and Alida Gehrke Foundation, German Pension Insurance Braunschweig-Hannover, Lower Saxony Ministry for Science and Culture and Vayamed. received payments for advisory boards and talks from Sanofi, MSD, Tafalgie, Kyowa, Mundipharma, Grünenthal and Ethypharm. His institution received research support from the European Commission, Gemeinsamer Bundesausschuß (GBA), Medtronic, Pfizer, Mundipharma, Grünenthal and Vertanical. has received consulting fees/honoraria from VIDAL, Biotechnology Value Fund, GH Research, Global Life Science, Science Direct, UCB and University of Duisburg (Germany) and support for travel to meetings from Bioevents and the European Pain Federation. J.K. has received speaker honoraria from Aurora, Esteve, Experten Schulungsakademie, Grünenthal, Hexal, Lilly and Teva. His institution has received research support from Bundesministerium für Bildung und Forschung, Deutsche Forschungsgemeinschaft, Dres. is employed by Metronomia, which was contracted to perform data collation and statistical analysis. Nature Medicine thanks Ian Gilron, Marieke Niesters and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: J. Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Note (Eligibility criteria), Tables 1–15, Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Karst, M., Meissner, W., Sator, S. et al. Full-spectrum extract from Cannabis sativa DKJ127 for chronic low back pain: a phase 3 randomized placebo-controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03918-x'>Genomically matched therapy in advanced solid tumors: the randomized phase 2 ROME trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 14:31:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite recent advancements demonstrating the potential of tumor-agnostic biomarkers to guide effective therapies, randomized evidence supporting the clinical superiority of precision oncology approaches compared to standard therapies remains limited. The ROME trial was a multicenter, randomized, open-label phase 2 study comparing tailored treatment (TT) to standard of care (SoC) in patients with advanced solid tumors progressing after one or two lines of therapy. Comprehensive genomic profiling on tissue and blood was performed to identify actionable alterations. Overall response rate (ORR) was the primary endpoint, and progression-free survival (PFS), overall survival (OS), time to treatment failure (TTF), time to next treatment (TTNT) and safety were the secondary endpoints. TT achieved a significantly higher ORR (17.5% versus 10%; P = 0.0294) and improved median PFS (3.5 months versus 2.8 months; hazard ratio = 0.66 (0.53–0.82), P = 0.0002). TT also showed superior 12-month PFS rates (22.0% versus 8.3%). Median OS was similar, with a 52% crossover rate. Grade 3/4 adverse events were also similar (40% TT versus 52% SoC). These results highlight the potential of TT to improve outcomes for patients with diverse actionable genomic alterations. These results also provide relevant evidence supporting a tumor-agnostic precision oncology strategy and highlight the potential of TTs, guided by genomic profiling and MTB recommendations, to significantly improve outcomes for patients with diverse actionable genomic alterations. Precision oncology aims to enhance clinical outcomes by tailoring cancer treatments to individual patients, focusing on specific vulnerabilities identified through the analysis of cancer characteristics and the patient's clinical history1. However, implementing these strategies presents major challenges, such as detecting actionable biomarkers, interpreting their clinical significance within the context of an individual patient's history and systematically utilizing available tailored treatments (TTs). This complexity is further compounded by the variability in the frequency of actionable molecular alterations across different tumor types, making histology a less reliable factor for determining treatment options2. Additionally, standard therapeutic options vary considerably across different tumor types. As a result, the clinical significance of an actionable genetic alteration can differ substantially across various cancer types3. Among the different approaches for detailed molecular characterization of tumor samples, genomic tests are the most widely used in clinical trials to identify patients with actionable alterations. Recent advancements have uncovered that treatments targeting some tumor-agnostic biomarkers induce considerable clinical benefits regardless of histology. However, definitive evidence demonstrating the superiority of these approaches over standard therapies is lacking, with inconsistent outcomes reported across randomized agnostic precision oncology clinical trials. For instance, although the SHIVA trial did not show progression-free survival (PFS) benefits for off-label TT compared to standard treatments13, the MOSCATO-01 trial achieved its endpoint, underscoring the value of broad genomic testing and targeted drug treatments14. Large, prospective, non-randomized trials are being conducted to identify molecularly defined subpopulations of patients deriving benefit from TTs. The results of multiple cohorts of the Targeted Agent and Profiling Utilization Registry (TAPUR) study were published15,16,17,18. The European Society for Medical Oncology (ESMO) sought to address these challenges by developing tools such as ETAC-S, which contains minimum requirements for identifying TTs with tumor-agnostic potential based on mechanistic evidence paired with prospective clinical insights22. Another widely implemented framework is the ESMO Scale for Clinical Actionability of molecular Targets (ESCAT)23, which ranks genomic aberrations to guide therapeutic decisions. The SAFIR02-BREAST trial demonstrated the utility of ESCAT, indicating that only patients harboring genomic alterations ranked as tier I and tier II benefit from maintenance TT, whereas those harboring alterations with lower actionability levels do not24. Moreover, molecular tumor boards (MTBs) provide essential multidisciplinary expertise for implementing precision medicine25. However, the impact of MTBs on patient outcomes requires further validation, as current evidence is limited and varies based on factors such as drug availability and tumor types treated26,27,28,29. Although regulatory approvals for targeted therapies have increased recently, many MTB-recommended treatments continue to face narrow or delayed approvals, limiting patient access to TTs even when actionable alterations are detected. Consequently, referral to clinical trials often becomes a viable option for patients to access TTs. Challenges such as limited enrollment availability and strict inclusion criteria further hinder the efficiency of this approach. Therefore, collaborative policies are essential to overcome these issues and facilitate the implementation of precision oncology in clinical practice25. The nationwide ROME trial was designed to provide comprehensive genomic profiling and TT for patients with solid tumors across multiple medical oncology units in Italy. Additionally, this approach seeks to generate high-quality data to identify potentially effective treatments that deserve further clinical validation in specific biomarker-defined subsets of patients. From 13 October 2020 to 19 July 2023, 1,794 patients with Eastern Cooperative Oncology Group Performance Status (ECOG PS) of 0 or 1 and metastatic solid tumors, who had received up to two lines of therapy, were accrued across 40 oncological centers in Italy. These patients were screened for genomic alteration on tissue and peripheral blood with FoundationOne CDx and FoundationOne Liquid CDx centralized next-generation sequencing (NGS). A total of 897 patients with potentially actionable molecular alterations were identified and discussed during 127 weekly MTB meetings. Patients with mutations deemed actionable by at least one of the drugs available in the trial were subsequently randomized in a 1:1 ratio to received TT versus SoC. Actionable genomic alterations and MTB-selected treatment options for the intention-to-treat (ITT) population are reported in Extended Data Fig. Thus, 400 patients were ultimately randomized into the TT and SoC arms. The CONSORT diagram is presented in Fig. Patient characteristics and lines of prior therapy are detailed in Table 1 and Supplementary Table 1, respectively. The median age, baseline ECOG PS and number of previous lines of treatment were similar between arms. The TT arm had a higher proportion of women. The primary tumor origins were well balanced, with only glioblastomas and biliary tract tumors showing a slight overrepresentation in the SoC and TT arms, respectively, with no statistical differences observed. The TT arm also had a higher percentage of patients who received prior targeted therapy or hormonal therapy or a combination of chemotherapy and targeted therapy. In contrast, the SoC arm had more patients with prior immunotherapy. Among the 200 patients receiving TT, the ORR was 17.5% (95% confidence interval (CI): 12.5–23.5), which included 3% (six patients) achieving a complete response and 14.5% (29 patients) achieving a partial response. In contrast, the SoC arm achieved an ORR of 10% (95% CI: 6.2–15.0), with no complete responses and a 10% partial response rate (20 patients). The primary endpoint analysis conducted using the stratified analysis by Cochran–Mantel–Haenszel (CMH) test demonstrated a significant difference in ORR between groups (P = 0.0285; Extended Data Table 1). ORR varied across prespecified strata: in the breast cancer subgroup (Stratum A), ORR was 20.0% for TT versus 35.0% for SoC; in the non-colorectal gastrointestinal cohort (Stratum B), rates were 14.8% versus 9.1%; in non-small cell lung cancer (NSCLC) (Stratum C), 12.5% versus 6.7%; and in ‘Other malignancies' (Stratum D), 19.1% versus 6.4%. The unique discrepancy with general results was observed in the breast cancer subgroup, probably as a consequence of the small breast cancer cohort and its inherent heterogeneity. Furthermore, the Breslow–Day test for homogeneity of odds ratios across strata was not significant (P = 0.1231), failing to provide evidence of heterogeneity in the treatment effect across cancer types. PFS was significantly improved in the TT arm (median 3.5 months, 95% CI: 3.0–4.8) compared to the SoC arm (2.8 months, 95% CI: 2.5–3.2), with a hazard ratio (HR) of 0.66 (95% CI: 0.53–0.82, P = 0.0002; Fig. This benefit extended to key timepoints, showing substantial higher 9-month and 12-month PFS rates in the TT arm: 27.8% (95% CI: 21.4–34.2) at 9 months and 22.0% (95% CI: 16.0–28.0) at 12 months compared to 13.4% (95% CI: 8.4–18.5) and 8.3% (95% CI: 4.2–12.5), respectively, in the SoC arm. No significant difference in median overall survival (mOS) was observed between the TT and SoC arms (mOS: 9.1 months (TT) and 7.9 months (SoC); HR = 0.92, 95% CI: 0.72–1.19, P = 0.5322; Fig. A high crossover rate (59%) from SoC to TT occurred after disease progression. Full details are reported in Supplementary Tables 2 and 3. Approximately half of the enrolled patients in both arms had already received two prior lines of systemic therapy, which may have influenced both treatment responsiveness and overall clinical outcome. Analyses of the key secondary endpoints—time to treatment failure (TTF) and time to next treatment (TTNT)—showed superior outcomes in the TT arm compared to the SoC arm. At 12 months, TTF was 22.0% (95% CI: 16.0–28.0) in the TT arm and 8.6% (95% CI: 4.4–12.7) in the SoC arm (Fig. The TT arm also demonstrated a significantly higher proportion of patients remaining on treatment at 12 months (24.6%, 95% CI: 18.1–31.1) than the SoC arm (8.7%, 95% CI: 4.4–13.0) (Fig. Secondary survival endpoints in the ITT population. The overall incidence of grade ≥3 adverse events (AEs) was similar in the TT and SoC arms (40.0% (80/200) and 52.5% (105/200), respectively). Figure 3 presents a tornado plot illustrating these AEs, categorized by System Organ Class categories of the Common Terminology Criteria for Adverse Events (CTCAE) version 5. 3a) and all grade 5 AEs (Extended Data Fig. This figure shows the system or organ grade ≥3 AEs that occurred during the study for patients receiving SoC or TT. Further exploratory analyses in specific subgroups provided additional insights. To assess the efficacy of TT against SoC, we analyzed subgroups defined by clinically relevant molecular alterations: MSI-H, hTMB, BRAF mutations and ERBB2 (HER2) alterations (including amplifications and mutations). Within the ITT population, these alterations were detected with the following frequencies: hTMB/microsatellite stable (MSS) (≥10 mutations per megabase, 33.7%), MSI-H (4.5%), BRAF mutations (4.3%) and ERBB2 alterations (14.3%). The results of these exploratory subgroup analyses of the ORR are presented in Extended Data Table 3; the swimmer plots for duration of response are reported in Extended Data Figs. P values derived from exploratory analyses should be interpreted with caution, as these analyses are hypothesis generating rather than confirmatory. Due to the non-predefined nature of exploratory endpoints and the potential for an inflated α level, the statistical significance observed in these analyses serves primarily to generate hypotheses for future confirmatory studies rather than providing definitive evidence NE, not evaluable; NR, not reached. Of the 135 patients with hTMB and MSS status (65 in the SoC arm and 70 in the TT arm), all patients in the TT arm received immunotherapy (ipilimumab plus nivolumab or, in a few cases, nivolumab monotherapy). At 12 months, the PFS rate was 32.6% (95% CI: 21.3–43.9) in the TT arm versus 8.1% (95% CI: 0.9–15.3) in the SoC arm, representing a 25% absolute improvement in patients alive and progression free at 1 year. Furthermore, at 1 year, 57.1% (95% CI: 20.5–93.8) of patients in the TT arm remained on treatment compared to none in the SoC arm (HR = 0.15, 95% CI: 0.03–0.73; Fig. However, at 12 months, the OS rate was 71.4% in the TT arm compared to 33.3% and an mOS of 3.7 months in the SoC arm (HR = 0.38; Fig. Of the 14 randomized patients with BRAF alterations, five received SoC and nine received TT, consisting of a BRAF/MEK inhibitor combination. A notable difference in PFS was observed: PFS was not reached in the TT arm, whereas it was 2.0 months (95% CI: 1.6–not evaluable) in the SoC arm (HR = 0.07, 95% CI: 0.01–0.67; Fig. The TT arm also demonstrated superior OS compared to the SoC arm; mOS was not reached in the TT arm, whereas it was 8.1 months (95% CI: 1.9–not evaluable) in the SoC arm (HR = 0.26, 95% CI: 0.04–1.57; Fig. In the subgroup of 58 patients with HER2 alterations, various anti-HER2 agents were administered. One patient in this group had an ERBB3 mutation, was also HER2 positive and received an indication for anti-HER2 treatment. Furthermore, at 12 months, 14.4% (95% CI: 1.40–27.50) of patients in the TT arm remained progression free compared to no patients in the SoC arm. Patients with HER2-altered tumors receiving TT showed a trend toward higher OS compared to the SoC arm, with an mOS of 10.2 months (95% CI: 6.5–not evaluable) versus 5.6 months (95% CI: 2.8–13.7) (HR = 0.61, 95% CI: 0.32–1.16; Fig. The ROME trial provides evidence that TT, guided by comprehensive genomic profiling and expert MTB decision-making, significantly improves outcomes in pretreated patients with metastatic disease. Notably, the TT arm achieved complete responses in a patient population with advanced metastatic disease treated in later lines, whereas no complete responses were observed in the SoC arm, underlining the potential efficacy of TTs in achieving deeper and potentially more durable clinical responses. The improvement in ORR was consistent despite variations across predefined cancer type strata. Subgroup analyses revealed pronounced benefit in the ‘Other malignancies' stratum (19.1% versus 6.4%), positive trends in non-colorectal gastrointestinal (14.8% versus 9.1%) and NSCLC (12.5% versus 6.7%) subgroups and a lower ORR in the TT arm for breast cancer (20.0% versus 35.0%). The observed numerical imbalance across the four predefined cohorts resulted from a combination of protocol-driven and real-world factors. Although the initial design anticipated competitive enrollment with balanced strata, the trial's biomarker-driven framework inherently favored variability in recruitment. A key factor contributing to this disparity was the differential prevalence of actionable genomic alterations across tumor types. Stratum D, which included diverse malignancies, exhibited higher rates of potentially actionable targets compared to other strata. Molecular profiling success rates further influenced recruitment dynamics. Tumors with lower shed DNA or suboptimal sample quality, common in certain histologies, reduced liquid biopsy utility, whereas tissue availability varied by cancer type. Additionally, site-specific expertise and patient populations probably played an important role, with centers specializing in rare cancers contributing more to Stratum D, whereas those focused on common tumors faced stricter exclusion thresholds. Temporal factors, including profiling turnaround times and drug availability after protocol amendments, also contributed to uneven accrual. Despite these imbalances, the trial's statistical integrity remained robust, as previously discussed. The observed disparities reflect real-world heterogeneity in the distribution of actionable targets rather than methodological shortcomings. The ROME trial's agnostic design, prioritizing molecular over histologic stratification, inherently accommodates such variability, validating its approach in capturing the complexity of precision oncology (Fig. The recruitment pattern underscores a critical challenge in biomarker-driven trials: molecular prevalence and clinical exclusion criteria inevitably influence enrollment dynamics. The trial's outcomes, however, demonstrate that robust statistical methods can mitigate imbalances, ensuring meaningful conclusions even in heterogeneous populations. These insights support the feasibility of tumor-agnostic frameworks while emphasizing the need for adaptive designs in future precision oncology studies. A major methodological departure from prior precision oncology trials is the timing of molecular profiling. In contrast to most studies that also use archival tissue samples from primary tumors, the ROME trial specifically employed metastatic tissue collected upon disease progression after the last therapy (with few exceptions; see supplementary materials). This strategy provided a more precise evaluation of the tumor's mutational landscape at study entry. The addition of liquid biopsy analysis, comprehensive MTB assessment and early randomization (within the first two lines of therapy) further enhanced the accuracy of TT recommendations while mitigating the confounding effects of clonal heterogeneity frequently observed in later-stage disease. A key finding from this study is the significant long-term benefit of PFS associated with TT. At the 12-month mark, patients receiving TT demonstrated improved disease control, indicating that it can extend PFS beyond 1 year for certain patients. This benefit, observed even in pretreated metastatic cases, highlights the clinical importance of personalized treatment strategies based on individual molecular alterations. The observed significant improvements in ORR and PFS within the TT arm were not accompanied by a similar improvement in OS. The substantial crossover rate (58.8%) from the SoC arm to the TT arm after disease progression, although ethically appropriate, likely contributed to this lack of OS benefit. The high rate of non-crossover (41.2%) underscores the challenges of implementing precision therapies in late-line settings, where declining performance status and emergent complications often preclude treatment changes. This reality suggests that TTs may require earlier deployment to maximize clinical impact. This highlights the need for future studies to incorporate stratified crossover analyses to better understand the long-term effects of TTs on survival outcomes. Subgroup analyses focused on specific molecular alterations (hTMB, BRAF mutations and ERBB2 alterations) revealed considerable benefits. The hTMB subgroup demonstrated a particularly strong PFS benefit with immunotherapy (nivolumab with or without ipilimumab), showing efficacy across both MSI-H and MSS tumor types. This finding supports hTMB as a predictive biomarker for immunotherapy response, even within a biomarker-agnostic approach, particularly when using the TMB calculation algorithm employed by the Foundation One platform. Despite this promising conclusion, literature reports on TMB have been inconsistent, likely due to several factors. Key contributing elements include methodological variations, site-specific influences and a lack of standardization across the field. The complexity is further compounded by the use of different platforms and proprietary algorithms as well as variability in panel size, gene coverage, bioinformatics pipelines, sequencing depths and tumor purity. Additionally, cancer-type-specific variations play an important role in the assessment of TMB. Despite these challenges, the potential of hTMB as a predictive tool for immunotherapy response remains promising, highlighting the need for continued exploration and standardization efforts. Similarly, TTs showed efficacy against tumors harboring BRAF and HER2 alterations. The similar incidence of grade 3/4 AEs in the TT and SoC arms, albeit with different AE profiles (diarrhea and fatigue in TT versus neutropenia in SoC), underscores the importance of proactive, personalized management of toxicities to optimize the therapeutic benefit–risk ratio. Composed of a multidisciplinary team of specialists, the MTB enabled a mutation-agnostic and tumor-agnostic approach, tailoring treatment plans based on comprehensive molecular profiles rather than histology alone, avoiding the passive acceptance of the mutation/drug axiom. This approach, combined with the clinical data provided by referring clinicians, facilitated a patient-specific therapeutic strategy. In scenarios where multiple targeted treatment options were available for a single molecular pathway (such as PI3K/AKT or ERBB2 alterations), the MTB prioritized agents based on each patient's unique molecular profile and clinical characteristics. Factors driving these decisions included the specific type of alteration (for example, ERBB2 mutation versus amplification), published evidence on drug efficacy in specific contexts, prior therapeutic exposures and drug safety profiles. Notably, several agents (alpelisib, tepotinib, talazoparib, selpercatinib and pralsetinib) were introduced via protocol amendment in 2024. These updates, documented in Extended Data Table 4, preserved trial integrity through predefined statistical analysis plan (SAP) stratification while enhancing real-world relevance. The MTB's ability to address each patient's biological complexity and clinical heterogeneity optimized therapy selection, contributing substantially to the positive outcomes of this trial. In case of detection of multiple actionable alterations, the MTB evaluated the potential clinical benefit of each therapeutic option, considering evidence from published literature, drug mechanism of action, patient-specific clinical factors, prior therapeutic responses and toxicity risks. The presence of multiple actionable biomarkers considerably complicates therapeutic decision-making due to pathway interactions, conflicting clinical evidence and diverse drug toxicities. Managing this complexity often exceeds individual clinician capacity and simple biomarker–drug algorithms. In the era of precision oncology, the MTB model is proving essential for interpreting complex genomic data and guiding personalized treatment plans that extend beyond traditional histopathological and direct target–drug association models, as seen in the prescription of TTs based on demonstrated efficacy in controlled clinical studies. The ROME trial provides, to our knowledge, the strongest evidence supporting MTB implementation in clinical practice, thanks to its agnostic and randomized design. Previous evidence suggested a potential improvement in survival outcomes when MTBs were implemented, as published by Miller et al.31 in a prospective, non-randomized phase 2 study. MTB utility is also being assessed in other prospective clinical trials, such as the Danish phase 2, non-randomized ProTarget study, implementing a Drug Rediscovery Protocol (DRUP)-like design32. Another example is the TARGET study (NCT04723316), a large nationwide prospective trial based in the United Kingdom designed for this purpose, aiming to enroll 6,000 patients with advanced cancers33. Our findings underscore the critical need to further develop MTBs and implement advanced decision support tools to effectively integrate and interpret complex multiomic data, ultimately enhancing the precision of clinical recommendations. The management of dual biomarkers (that is, hTMB and HER2 alteration) posed unique challenges reflecting real-world precision oncology dilemmas, where pathway crosstalk (for example, MAPK activation dampening immunotherapy efficacy) and heterogeneous evidence bases complicate decisions. Our MTB addressed these scenarios by carefully evaluating multiple dimensions, including the biological relevance and actionability of each target (assessed using validated tools), variant allele frequency (VAF) of the detected alterations, quantitative thresholds for biomarkers such as TMB, strength of existing clinical evidence, potential interactions between targeted pathways, patient-specific clinical factors (including prior treatments, clinical status and comorbidities) and safety and toxicity profiles of candidate therapies, thus enabling structured and patient-centered decision-making. AI platforms can enhance MTB decisions in multitarget scenarios by structuring complex genomic/clinical data (for example, prioritizing therapies for HER2+/hTMB cases) and reducing cognitive overload through evidence-based recommendations. In Italy, the Alliance Against Cancer (ACC) consortium is advancing AI tools under initiatives such as CraNE/EUNetCCC to support MTBs in analyzing mutational profiles, aligning with European Union goals for precision oncology. These systems cross-reference real-world outcomes, biomarker–drug evidence and toxicity profiles, offering clinicians actionable hierarchies while maintaining human oversight. The work of the ACC exemplifies how AI augments, rather than replaces, expert deliberation, improving efficiency and consistency in biomarker-driven care. The complexities of precision oncology necessitate adaptive platform trials to expedite patient access to effective TTs and keep pace with rapidly advancing genomic knowledge34,35. Platform trials, often referred to as multiarm, multistage studies, are randomized controlled trials (RCTs) that evaluate multiple treatments within different arms of a single trial, all compared to a shared control arm. This design incorporates adaptive features, allowing for the removal of ineffective treatment arms and the inclusion of new arms tailored to specific drug–target interactions. Additionally, patients may be reassigned to different treatment arms based on their response to prior therapies or changes in biomarker profiles. By replacing the traditional sequential trial model, this approach accelerates the process of drug discovery and development36. Furthermore, it will be beneficial to complement RCTs with observational studies to assess the generalizability and real-world applicability of their results. Future precision oncology will strongly benefit from dynamic national and international clinical trial databases, comprehensively annotated with gene-specific and variant-specific targets, directly linked to advanced decision support platforms to facilitate patient stratification into molecularly driven trials. Although the ROME trial provides robust evidence regarding the efficacy and safety of molecularly guided therapies in a controlled environment, the complexities of real-world practice—including patient heterogeneity, varied access to genomic testing and differences in healthcare infrastructure—may influence the outcomes observed. Observational studies using robust real-world data sources, such as population-based cancer registries, electronic health records and multicenter collaborative networks, combined with advanced AI analysis, would be instrumental in addressing these aspects. Future research priorities include optimizing biomarker-driven patient selection and MTB processes, developing innovative trial designs, investigating specific strategies to overcome resistance mechanisms and exploring combination approaches with immunotherapy and other tailored agents to propel the field of precision oncology further and realize its potential. The upper boxes represent the two main eligibility criteria, molecular profiling, MTB evaluation, randomization, treatment with TT or SoC and possible crossover at disease progression. CGP, comprehensive genomic profiling; PD, progressive disease; R, randomization. 5575; February 2020) and by the ethics committee of each participating center. The investigational sites that approved the study protocol were as follows: AOU Policlinico Umberto I, Rome; European Institute of Oncology (IEO), IRCCS, Milan; Veneto Institute of Oncology IOV-IRCCS, Padua; Division of Medical Oncology, Pisa University Hospital; IRCCS Istituto Romagnolo per lo Studio dei Tumori (IRST) ‘Dino Amadori', Meldola; Istituto Nazionale Tumori IRCCS Fondazione ‘G. Pascale', Naples; IRCCS Istituto Nazionale Tumori Regina Elena (IRE), Rome; IRCCS Istituto di Candiolo, Candiolo; Medical Oncology Unit, ARNAS Garibaldi Catania; Azienda Ospedaliera Santa Maria di Terni, Terni; Pederzoli Hospital, Peschiera del Garda; IRCSS Ospedale Policlinico San Martino, Genoa; Ospedali Galliera, Genoa; Central Hospital of Belcolle, Viterbo; IRCCS Sacro Cuore Don Calabria Hospital, Negrar di Valpolicella; Santa Maria della Misericordia Hospital, Perugia; Nuovo Ospedale di Prato-Santo Stefano, Azienda USL Toscana Centro, Prato; CRO Aviano, National Cancer Institute, IRCCS, Aviano; Santa Maria delle Croci Hospital, AUSL Romagna, Ravenna; Oncology Unit ASST Papa Giovanni XXIII, Bergamo; Ordine Mauriziano Hospital, Turin; AOU Policlinico S. Andrea, Rome; Azienda Ospedaliera Universitaria Federico II, Napoli; Fondazione IRCCS Istituto Tumori di Milano; Fondazione Policlinico Universitario Campus Bio-Medico, Rome; A.O. Papardo-Messina; IRCCS Istituto Tumori ‘Giovanni Paolo II', Bari; Policlinico universitario ‘Mater Domini', Catanzaro; Centro Oncologico San Leopoldo Mandic, Isola Tiberina Gemelli isola; Rome; University Hospital, Ferrara; Misericordia Hospital, Grosseto; AUSL Piacenza Guglielmo da Saliceto Hospital, Piacenza; Comprehensive Cancer Centre, AUSL-IRCCS di Reggio Emilia, Reggio Emilia; Humanitas Gradenigo, Torino; Foundation IRCCS Casa Sollievo della Sofferenza, San Giovanni Rotondo; Ramazzini Hospital, Azienda Unità Sanitaria Locale Modena (AUSL), Carpi; AOUP ‘Paolo Giaccone', Palermo; and AOU delle Marche, Ancona. The trial is registered on ClinicalTrials.gov with identifier NCT04591431 and in the European Union Drug Regulating Authorities Clinical Trials Database (EudraCT) with number 2018-002190-21. Forty-one centers received ethics approval and participated in the study enrollment. All patients signed the specifically conceived informed consent form (ICF). This randomized, prospective, multicenter, proof-of-concept phase 2 clinical trial enrolled patients with inoperable advanced or metastatic solid tumors, regardless of histology. Eligible patients had failed at least one, but no more than two, prior lines of systemic therapy. An ECOG PS of 0 or 1 was required as well as being at least 18 years of age at the time of signing the ICF. All patients had to have measurable or evaluable disease defined per Response Evaluation Criteria in Solid Tumors version 1.1 (RECIST 1.1)37 or immune-related response criteria (irRC)38. If clinically indicated, magnetic resonance imaging (MRI) and positron emission tomography (PET) scans could have been performed. PET Response Criteria in Solid Tumors (PERCIST)39 were applied in cases where a baseline PET scan was available. Adequate renal, liver and bone marrow functions were required at baseline. Those with well-established actionable molecular targets, not detected in previous molecular profiling, for which approved and marketed targeted therapies were available—such as EGFR mutations or ALK translocations in lung cancer, BRAF mutations in melanoma, KIT mutations in gastrointestinal stromal tumors, ERBB2 amplification in breast cancer or tumor-agnostic biomarker-driven treatments already available in clinical practice—were also excluded. Furthermore, patients with only bone and/or brain metastases, uncontrolled brain disease (untreated and/or symptomatic) or those whose brain metastases had not been monitored for over 2 months were excluded. Patients with concurrent severe and/or uncontrolled medical conditions that could compromise their participation in the study (for example, uncontrolled diabetes, clinically relevant cardiac disease, uncontrolled hypertension, congestive heart failure, ventricular arrhythmias, active ischemic heart disease, myocardial infarction within the last year, chronic liver or renal disease, active gastrointestinal ulceration or severely impaired lung function) were also excluded (see supplementary materials for all inclusion/exclusion criteria). For genomic testing, a tissue sample obtained during the screening phase or within 6 months before enrollment was required. Samples collected within 3 months before the patient's ICF signature were acceptable, as were samples collected within 6 months, with prior acceptance by the MTB. Archived tissue samples were accepted for patients with glioblastomas and high-grade malignant gliomas. Patients with only one available biopsy, either liquid or solid, due to a failure of one method during the screening phase remained eligible for inclusion and discussion in the MTB. Conversely, if both tissue and liquid biopsy characterizations failed, patients were classified as screening failures. Molecular profiling was performed using FoundationOne CDX and FoundationOne Liquid CDX tests on tissue and liquid biopsies. DNA from formalin-fixed, paraffin-embedded tumor tissue samples was isolated using the DNA extraction method and analyzed with the FoundationOne CDx panel to assess alterations across 324 genes, including substitutions, insertions and deletions (indels), copy number alterations (CNAs) and gene rearrangements. Genomic signatures of MSI and TMB were also evaluated. For FoundationOne Liquid CDx, circulating cell-free DNA (cfDNA) was extracted from plasma collected from anticoagulated peripheral whole blood samples using FoundationOne Liquid CDx cfDNA blood collection tubes. Additionally, the test evaluated tumor fraction and genomic signatures such as blood-based tumor mutational burden (bTMB) and MSI-H status. Upon identification of actionable alterations in either tumor tissue or liquid biopsy, and after MTB discussion, selected patients were assigned to receive immunotherapy, targeted therapy or both based on MTB recommendations. A crossover between the two treatment arms was planned upon progressive disease. The study design is reported in Fig. An MTB convened weekly to review each eligible patient. The MTB consisted of medical oncologists, pathologists, geneticists, immunologists, bioinformaticians and other relevant specialists. The MTB then reviewed each case, considering comprehensive molecular profiling data alongside the patient's clinical features, such as performance status, comorbidities and concurrent treatments. The treating clinician presented each case selected by the steering committee, highlighting potential actionable therapeutic targets. Molecular alterations were evaluated using databases such as ClinVAR, OncoKB and COSMIC and the ESMO ESCAT scale. It systematically reviewed potential intrinsic resistance mechanisms, such as KRAS mutations in patients considered for PIK3CA or AKT inhibitors. For patients with multiple actionable alterations (for example, hTMB alongside HER2 amplification or BRAF mutation), the MTB prioritized therapies using a hierarchical framework prioritizing clinical urgency (for example, immunotherapy for MSI-H tumors regardless of co-alterations), followed by biomarker–drug evidence strength (for example, HER2-targeted agents over exploratory TMB-directed therapies when both were present) and, finally, considering toxicity overlap avoidance (for example, excluding PI3K inhibitors if preexisting hyperglycemia). Patients discussed at the MTB who were not randomized were excluded for various reasons, including the absence of target alterations, the presence of resistance mutations, known negative studies related to a specific alteration, rare mutations without literature data, low VAF, unavailability of drugs for specific alterations, clinical considerations, high levels of uncertainty regarding potentially targetable alterations and the presence of established targetable alterations with available standard treatments. MTB recommendations included randomizing patients to receive TT, referring them to a geneticist in the presence of somatic alterations with potential germline significance, modifying SoC only when necessary and suggesting enrollment in clinical trials or early drug access programs, when available in Italy. MTB activities are summarized in Extended Data Fig. Patients who met the inclusion and exclusion criteria and received a recommendation for a specific targeted agent or combination therapy from the MTB were randomized in a 1:1 ratio to receive either SoC or TT, according to their molecular profiles. In the SoC arm, patients were treated with standard chemotherapy, targeted therapy or immunotherapy based on primary tumor histology according to national guidelines and at the investigators' discretion. The MTB also proposed possible modifications based on molecular profiling and expert opinion in the control group. In the TT arm, patients received targeted therapy, immunotherapy or a combination that was selected according to their genomic profiles, as determined by the MTB. Combination therapies were proposed in cases where efficacy data for the combination were well established in different malignancies (for example, pertuzumab plus trastuzumab for HER2+ breast cancer and vemurafenib plus cobimetinib for BRAF-mutant melanoma) or when there was a biological rationale due to the presence of multiple actionable mutations. Such proposals were contingent upon the availability of published phase 1b/2 data supporting the combination in other tumor types. In cases involving immunotherapy, the combination of ipilimumab and nivolumab was usually preferred unless a patient's clinical condition or comorbidities suggested otherwise. A substantial amendment to the protocol was implemented in October 2022, introducing additional treatments to the existing options, including entrectinib, alpelisib, pralsetinib and selpercatinib. Investigational therapies included erlotinib, pertuzumab, vemurafenib, trastuzumab emtansine, alectinib, vismodegib, cobimetinib, atezolizumab, trastuzumab, ipatasertib, entrectinib, everolimus, palbociclib, lapatinib, ipilimumab, nivolumab, brigatinib, ponatinib, itacitinib, pemigatinib, alpelisib, tepotinib, pralsetinib, talazoparib and selpercatinib. In cases where hTMB coexisted with a targetable mutation, a combination of immunotherapy and targeted therapy was generally preferred, provided that safety data were available. All treatments were administered according to the investigator's brochure or the relevant regulatory clinical protocol. Each targeted agent was associated with a specific pathway, signature or gene alteration (Extended Data Table 4). Secondary endpoints included PFS, defined as the time from randomization to disease progression or death, whichever occurred first; TTF, defined as the time from randomization until patient withdrawal for any reason, including disease progression or death; TTNT, defined as the time from randomization to the start of the next line of therapy; and OS, defined as the time from randomization to death from any cause. Additional planned secondary endpoints not reported in this paper include concordance between molecular profile on tumor tissue and ctDNA, quality of life measurements, immune fitness in the two treatment arms and the association between the molecular evaluation and gene expression profiling. Tumor response was locally assessed according to RECIST version 1.1 and irRC. In patients who discontinued treatment for reasons other than disease progression or consent withdrawal, tumor assessments continued until documented progression. Survival was monitored until death, loss to follow-up, consent withdrawal or study termination. AEs were graded using CTCAE version 5.0, with the frequency and severity of AEs documented for each patient. Data were collected at the clinical sites and inserted into a web-based electronic case report form (eCRF) by investigators or designated personnel. The trial was overseen by an independent external data monitoring committee, which conducted periodic safety evaluations and reviewed efficacy during prespecified interim analyses. Per the study protocol, the data monitoring committee (DMC) conducted a prespecified interim analysis after 20% of the randomized patients (n = 76) had reached the final efficacy evaluation (day 308 or early termination), irrespective of treatment group assignment. The DMC reviewed these data and recommended proceeding with patient accrual as planned. As predefined, the stopping threshold for conditional power (CPt) was set at 5%. Comparisons were made between the observed TT group response rate at interim analysis and the projected end-of-study SoC group response rate (5%). At interim analysis, the TT group exhibited a 7.1% response rate. All authors had access to the data used for manuscript preparation and contributed to its drafting, critical review or editing. The healthcare business of Merck KGaA, Darmstadt, Germany (CrossRef Funder ID: 10.13039/100009945), reviewed this manuscript for medical accuracy only before journal submission. We hypothesized that TT would yield higher ORR compared to SoC, specifically a 20% ORR for TT versus 5% for SoC. According to the site of primary tumor, four cohorts were defined as breast cancer (Stratum A), non-colorectal gastrointestinal cancers (Stratum B), NSCLC (Stratum C) and ‘other malignancies' (Stratum D), with competitive enrollment across these strata. To detect a 15% difference between the two arms, assuming an α of 0.10 and a β of 0.20 (80% power) and employing a one-sided chi-square test, a total of 86 patients (43 in each arm of the four cohorts) were required. The four cohorts were evaluated as separate strata, resulting in a combined cohort size of 344 patients (86 patients per stratum). The overall ORR was assessed using a two-sided CMH test at a 5% significance level; an unstratified sensitivity analysis was also performed. Considering a dropout rate of 10%, 384 patients were required. Assuming actionable mutations in at least 30% of eligible patients, the planned screening sample size was 1,280. Due to the extended screening periods required for genomic testing and the inherent unpredictability of patient eligibility based on mutational test results, 12 additional patients undergoing screening at the time the target of 384 randomized participants was reached were permitted by the steering committee to enroll, resulting in a final total of 400 patients. All analyses were conducted using SAS software (v.9.4). The randomization list was generated with a dedicated SAS program using the PROC PLAN procedure by an independent statistician. A soft lock of the database was performed on 31 July 2024. SAP v.2.0 was finalized on 30 January 2025, after the database freezing and data review meeting, and governs all analyses presented here. The full SAP is available upon reasonable request. Descriptive statistics were used to summarize the data, with mean, s.d., quartiles for continuous variables and frequency distributions for categorical variables. Differences in clinical characteristics were evaluated with t-test, binomial test, chi-quadro and CMH. Kaplan–Meier survival analysis, accompanied by the log-rank test, was employed to compare OS and PFS across arms and subgroups. Homogeneous subgroups of patients were evaluated in a prespecified exploratory analysis. Exploratory biomarker analyses, including assessments of hTMB (≥10 mutations per megabase), BRAF V600E mutations and HER2 amplifications, were predefined in the SAP to evaluate treatment effect heterogeneity. These subgroups were selected based on emerging preclinical evidence during the trial's enrollment phase, with steering committee approval for post hoc stratification. All statistical analyses were conducted on the ITT population. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Individual deidentified participant data generated during the current study are available upon reasonable request from academic or qualified clinical researchers affiliated with recognized institutions, strictly for the purpose of conducting non-commercial, ethically approvable research aligned with the original scope of the trial. Applicants are required to submit a detailed research proposal, curriculum vitae and a declaration of non-conflict of interest. Requests must clearly describe the research objectives and methodology and must be reviewed and approved by the steering committee of the ROME trial during dedicated review sessions. Approval is granted based on scientific merit, data availability, intended data use and absence of overlapping research initiatives by the trial investigators. All approved requestors will be required to sign a data access agreement that restricts data use solely to the approved research project and prohibits any further distribution or use. Data will be shared via a secure data-sharing platform within 4–8 weeks of approval, contingent upon data volume and complexity. Data requests will be considered within 12 months of manuscript publication. The trial registration, study protocol and methodological details are publicly accessible through ClinicalTrials.gov (accession identifier NCT04591431; https://clinicaltrials.gov/ct2/show/NCT04591431). No other public repositories or datasets requiring accession codes were used in this study. Prasad, V., Fojo, T. & Brada, M. Precision oncology: origins, optimism, and potential. Schettini, F. et al. Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the ‘MOZART' prospective observational study. Marabelle, A. et al. Efficacy of pembrolizumab in patients with noncolorectal high microsatellite instability/mismatch repair-deficient cancer: results from the phase II KEYNOTE-158 study. André, T. et al. Antitumor activity and safety of dostarlimab monotherapy in patients with mismatch repair deficient solid tumors: a nonrandomized controlled trial. Marabelle, A. et al. Association of tumour mutational burden with outcomes in patients with advanced solid tumours treated with pembrolizumab: prospective biomarker analysis of the multicohort, open-label, phase 2 KEYNOTE-158 study. Hong, D. S. et al. Larotrectinib in patients with TRK fusion-positive solid tumours: a pooled analysis of three phase 1/2 clinical trials. Drilon, A. et al. Repotrectinib in ROS1 fusion–positive non–small-cell lung cancer. & Subbiah, V. Expanding the benefit: dabrafenib/trametinib as tissue-agnostic therapy for BRAF V600E-positive adult and pediatric solid tumors. Subbiah, V. et al. Tumour-agnostic efficacy and safety of selpercatinib in patients with RET fusion-positive solid tumours other than lung or thyroid tumours (LIBRETTO-001): a phase 1/2, open-label, basket trial. Meric-Bernstam, F. et al. Efficacy and safety of trastuzumab deruxtecan in patients with HER2-expressing solid tumors: primary results from the DESTINY-PanTumor02 phase II trial. Le Tourneau, C. et al. Molecularly targeted therapy based on tumour molecular profiling versus conventional therapy for advanced cancer (SHIVA): a multicentre, open-label, proof-of-concept, randomised, controlled phase 2 trial. Massard, C. et al. High-throughput genomics and clinical outcome in hard-to-treat advanced cancers: results of the MOSCATO 01 trial. Mangat, P. K., Garrett-Mayer, E., Perez, J. K. & Schilsky, R. L. The Targeted Agent and Profiling Utilization Registry Study: a pragmatic clinical trial. Schuetze, S. et al. Palbociclib in patients with soft tissue sarcoma with CDK4 amplifications: results from the Targeted Agent and Profiling Utilization Registry Study. Gupta, R. et al. Pertuzumab plus trastuzumab in patients with colorectal cancer with ERBB2 amplification or ERBB2/3 mutations: results from the TAPUR study. Alva, A. S. et al. Pembrolizumab in patients with metastatic breast cancer with high tumor mutational burden: results from the Targeted Agent and Profiling Utilization Registry (TAPUR) study. Meric-Bernstam, F. et al. Pertuzumab plus trastuzumab for HER2-amplified metastatic colorectal cancer (MyPathway): an updated report from a multicentre, open-label, phase 2a, multiple basket study. Kalinsky, K. et al. Effect of capivasertib in patients with an AKT1 E17K-mutated tumor: NCI-MATCH subprotocol EAY131-Y nonrandomized trial. The ESMO Tumour-Agnostic Classifier and Screener (ETAC-S): a tool for assessing tumour-agnostic potential of molecularly guided therapies and for steering drug development. Mateo, J. et al. A framework to rank genomic alterations as targets for cancer precision medicine: the ESMO Scale for Clinical Actionability of molecular Targets (ESCAT). Andre, F. et al. Genomics to select treatment for patients with metastatic breast cancer. Repetto, M. et al. Molecular tumour board at European Institute of Oncology: report of the first three year activity of an Italian precision oncology experience. Vingiani, A. et al. Molecular tumor board as a clinical tool for converting molecular data into real-world patient care. The molecular tumor board as a step in cancer patient management: a southern Italian experience. The Molecular Tumor Board of the Regina Elena National Cancer Institute: from accrual to treatment in real-world. Miller, R. W. et al. Molecular tumor board–assisted care in an advanced cancer population: results of a phase II clinical trial. Kringelbach, T. et al. ProTarget: a Danish Nationwide Clinical Trial on Targeted Cancer Treatment based on genomic profiling—a national, phase 2, prospective, multi-drug, non-randomized, open-label basket trial. Ortega-Franco, A. et al. TARGET National: a U.K.-wide liquid-based molecular profiling program to enhance recruitment to early-phase trials. Implementation of precision medicine in healthcare—a European perspective. The future of clinical trials—goals, ideas, and discussion. In memo – Magazine of European Medical Oncology 17, 77–86 (2024). Eisenhauer, E. A. et al. New response evaluation criteria in solid tumours: revised RECIST guideline (version 1.1). Guidelines for the evaluation of immune therapy activity in solid tumors: immune-related response criteria. From RECIST to PERCIST: evolving considerations for PET response criteria in solid tumors. The authors would like to thank P. Bruzzi, former Director of the Complex Structure at the National Institute for Cancer Research in Genoa and University Professor, for his valuable comments and advice, which helped enhance the clarity, readability and comprehensibility of the methodological and statistical aspects of this work. Erlotinib, pertuzumab, vemurafenib, trastuzumab emtansine, alectinib, vismodegib, cobimetinib, atezolizumab, trastuzumab, ipatasertib (GDC-0068), entrectinib and pralsetinib were provided by Roche; everolimus, lapatinib and alpelisib were provided by Novartis; palbociclib and talazoparib were provided by Pfizer; ipilimumab and nivolumab were provided by Bristol Myers Squibb; brigatinib was provided by Takeda; ponatinib and itacitinib (INCB039110) and pemigatinib (INCB054828) were provided by Incyte; selpercatinib was provided by Eli Lilly; and tepotinib was provided by the healthcare business of Merck KGaA, Darmstadt, Germany (CrossRef Funder ID: 10.13039/100009945). This study was supported through unrestricted grants provided by Roche, Takeda, Bristol Myers Squibb, Pfizer, Incyte and Eli Lilly. These contributions were regulated by contractual agreements with the study sponsor: the Fondazione per la Medicina Personalizzata (FMP). The funding entities had no role in study design, data collection, analysis, interpretation or manuscript preparation. European Institute of Oncology, IRCCS, Milan, Italy Simone Scagnoli, Giulia D'Amati, Bruna Cerbelli & Andrea Botticelli Oncology Unit, AOU Policlinico Umberto I, Rome, Italy Department of Translational Research and New Technologies in Medicine and Surgery, University of Pisa, and Division of Medical Oncology, Pisa University Hospital, Pisa, Italy IRCCS Istituto Romagnolo per lo Studio dei Tumori (IRST) ‘Dino Amadori', Meldola, Italy Ugo De Giorgi, Alessandra Piancastelli & Francesca Mannozzi Translational Oncology Research Unit, IRCCS Istituto Nazionale Tumori Regina Elena (IRE), Rome, Italy Department of Translational Research and New Technologies in Medicine and Surgery, University of Pisa, Pisa, Italy S. Camillo Hospital IRCCS, Lido di Venezia, Italy Medical Oncology Unit, ARNAS Garibaldi, Catania, Italy Department of Hepatopancreatobiliary (HPB) Surgery, Pederzoli Hospital, Peschiera del Garda, Verona, Italy IRCSS Ospedale Policlinico San Martino, UO Clinica Oncologia Medica, Genoa, Italy Department of Internal Medicine and Medical Specialties (DIMI), Università di Genova, Genoa, Italy Department of Medicine and Medical Oncology Unit, E.O. Medical Oncology and Breast Unit, Department of Oncology and Hematology, Central Hospital of Belcolle, Viterbo, Italy Department of Oncology, IRCCS Sacro Cuore Don Calabria Hospital, Negrar di Valpolicella, Verona, Italy Oncologia Medica 1, IRCSS Ospedale Policlinico San Martino, Genoa, Italy Medical Oncology Unit, Nuovo Ospedale di Prato-Santo Stefano, Azienda USL Toscana Centro, Prato, Italy Oncology Unit, Santa Maria delle Croci Hospital, AUSL Romagna, Ravenna, Italy Oncology Unit, ASST Papa Giovanni XXIII, Bergamo, Italy Department of Medicine and Surgery, Università degli Studi di Milano Bicocca, Milan, Italy Department of Oncology, Ordine Mauriziano Hospital, Turin, Italy Dipartimento Di Eccellenza In Medicina Molecolare E Biotecnologie Mediche, Università Federico II, Naples, Italy Division of Medical Oncology 2, IRCCS Regina Elena National Cancer Institute, Rome, Italy Department of Molecular Medicine, Sapienza University of Rome, laboratory affiliated to Istituto Pasteur Italia, Fondazione Cenci-Bolognetti, Rome, Italy Department of Clinical and Molecular Medicine, Sapienza University, Oncology Unit, Azienda Ospedaliera Universitaria Sant'Andrea, Rome, Italy Department of Advanced Diagnostics, Fondazione IRCCS Istituto Tumori di Milano, University of Milan, Milan, Italy Laboratory of Clinical Biochemistry, Sant'Andrea University Hospital, Rome, Italy Department of Neurosciences, Mental Health and Sensory Organs (NESMOS), Sapienza University of Rome, Rome, Italy Department of Medicine and Surgery, Università Campus Bio-Medico, Rome, Italy Medical Oncology, Fondazione Policlinico Universitario Campus Bio-Medico, Rome, Italy Medical Oncology 1, IRCCS Regina Elena National Cancer Institute, Rome, Italy Clinical Trial Center, Oncology Department A.O., Papardo-Messina, Italy Medical Oncology Unit, RCCS Istituto Tumori ‘Giovanni Paolo II', Bari, Italy Department of Experimental and Clinical Medicine, Magna Græcia University, Catanzaro, Italy Department of Medical Oncology, University Hospital, Ferrara, Italy Division of Medical Oncology 1, St. Chiara University Hospital, Pisa, Italy Oncology Unit, Foundation IRCCS Casa Sollievo della Sofferenza, San Giovanni Rotondo, Italy Oncology Unit, Ramazzini Hospital, Azienda Unità Sanitaria Locale Modena (AUSL), Carpi, Italy Medical Oncology 2, Istituto Oncologico Veneto IOV IRCCS, Padova, Italy Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Patient enrollment and investigation: G.C., M.B., S.L., S.S., L.F., V.G., U.D.G., P.A., M.A., C.C., P.C., E. Crimini, S.P., S.V., R. Bordonaro, S.B., G. Butturini, L.D.M., A.D.C., A.F., E.F., S.G., G.M., A.P., D.P., F.P., S.T., A.Z., D.M., F.C., F.M., G.T., A.B., A.S., V.A., D.Q., P.T., P.D., B.U., C.B., E.Z., C.P., A.T., L.B., E.M., C.M., A.R. P.M. had a consultant/advisory role for Bristol Myers Squibb, Roche Genentech, Merck Sharp & Dohme, Novartis, AMGEN, Merck Serono, Pierre Fabre and Incyte. had a role in advisory board for Roche, AstraZeneca, Daiichi Sankyo, Eli Lilly, Novartis, Pfizer, Gilead, Menarini, Exact Sciences, Bristol Myers Squibb and Merck. is employed in the Italian National Institute of Health (Istituto Superiore di Sanità) and is an unpaid member of the Technical and Scientific Committee of the Italian Medicine Agency (AIFA). was an invited speaker for Amgen, AstraZeneca, Bristol Myers Squibb, Incyte, GlaxoSmithKline, Eli Lilly, Merck Serono, Merck Sharp & Dohme, Pierre Fabre, Roche and Servier and had a role in advisory boards for Amgen, Astellas, AstraZeneca, Bayer, Bristol Myers Squibb, Daiichi Sankyo, GlaxoSmithKline, Incyte, Eli Lilly, Merck Serono, Merck Sharp & Dohme, Servier, Takeda, Rottapharm, BeiGene, Fosun Pharma and Nimbus Therapeutics. S.S. was an invited speaker for Pfizer, Eli Lilly, Novartis, Daiichi Sankyo, Gilead, Roche and AstraZeneca. L.F. reports speaking honoraria from Incyte, Bristol Myers Squibb snd Eli Lilly. He also received institutional research funding from Merck Sharp & Dohme, Bristol Myers Squibb, AstraZeneca, Incyte, BeiGene, Astellas, Daiichi Sankyo and Roche. He had a role in advisory boards for Merck Sharp & Dohme, AstraZeneca, Incyte, Taiho, Servier, Daiichi Sankyo, Eli Lilly and Astellas. has a role in leadership for AstraZeneca, Daiichi Sankyo, Eli Lilly, Exact Sciences, Gilead, Merck Sharp & Dohme, Novartis, Pfizer, Olema Oncology, Pierre Fabre and Menarini Stemline. She had a role as a speaker for AstraZeneca, Daiichi Sankyo, Eli Lilly, Exact Sciences, Gilead, GlaxoSmithKline, Novartis, Roche, Zentiva and Menarini Stemline and provided expert testimony for Eli Lilly. received consultation fees from Amgen, Astellas, AstraZeneca, Bayer, Bristol Myers Squibb, Esai, Ipsen, Johnson & Johnson Innovative Medicine (formerly Janssen), Merck KGaA, Merck Sharp & Dohme, Novartis and Pfizer and travel expenses for attending symposia from AstraZeneca, Ipsen and Pfizer. had a role as consultant/advisor for AstraZeneca, Eli Lilly, Daiichi Sankyo, Novartis, Pfizer, Seagen, Sophos and Gilead and received travel and accommodation fees from Daiichi Sankyo, Pfizer, AstraZeneca, Eli Lilly, Menarini, Novartis and Gilead. received congress travel support from Merck Sharp & Dohme, Pfizer, Bayer and Ipsen and had a role in advisory boards and steering committees for Pfizer, Bristol Myers Squibb, Merck Sharp & Dohme, Astellas, Roche, Johnson & Johnson, Ipsen, Bayer, Novartis, Merck, AstraZeneca and Gilead. received personal grant for advisory/consultant/speaker activities from Agendia, AstraZeneca, Daiichi Sankyo, Eli Lilly, Eisai, Exact Sciences, Gilead, GlaxoSmithKline, Ipsen, Roche, Seagen, Menarini, Stemline, Merck Sharp & Dohme, Novartis, Olema, Pierre Fabre and Pfizer. received financial fees from Roche, Menarini, Pfizer, Eli Lilly, Novartis, Daiichi Sankyo, AstraZeneca and Gentili. received honoraria for advisory boards, activities as a speaker, travel grants and research grants from Amgen, AstraZeneca, Daiichi Sankyo, Celgene, Eisai, Eli Lilly, Exact Sciences, Gilead, Ipsen, Italfarmaco, Menarini, Merck Sharp & Dohme, Novartis, Pierre Fabre, Pfizer, Roche, Seagen, Takeda and Viatris. He received research funding from AstraZeneca, Eisai and Roche. received honoraria for consultancy and advisory board from Roche, Novartis, Eli Lilly, AstraZeneca, Pfizer, Exact Sciences, Merck, Daiichi Sankyo, Gilead, Seagen, Menarini and Stemline. received fees for membership of an advisory board or lectures from Roche, AstraZeneca, Bristol Myers Squibb, Pfizer, Takeda, Eli Lilly, Bayer, Amgen, Sanofi, Pharmamar, Novocure, Mirati, Galecto, OSE, Illumina, Thermo Fisher Scientific, BeiGene and Merck Sharp & Dohme. had a role in advisory boards for Merck Sharp & Dohme and Daiichi Sankyo. has received personal fees (as a consultant and/or speaker bureau) from Boehringer Ingelheim, Roche, Merck Sharp & Dohme, Amgen, Thermo Fisher Scientific, Eli Lilly, Diaceutics, GlaxoSmithKline, Merck, AstraZeneca, Janssen, Diatech, Novartis and Hedera. received financial fees from Illumina, Roche, Eli Lilly, AstraZeneca, Exact Sciences and ADS Biotech. had a role in advisory boards for Molteni, Novartis, PharmaMar and Merck Sharp & Dohme. N.M. received advisory board consultation fees from Roche, Daiichi Sankyo, Pfizer and AstraZeneca. had a role in advisory boards and steering committees and has been an invited speaker for Novartis, Roche, Eli Lilly, Pfizer, AstraZeneca, Daiichi Sankyo, Gilead, Merck Sharp & Dohme, Bristol Myers Squibb and Gentili. received personal fees for advisory role, speaker engagements and travel accommodation expenses from Amgen, Astellas, AstraZeneca, Bayer, Bristol Myers Squibb, Celgene, Daiichi Sankyo, Eisai, Ipsen, Janssen, Incyte, Merck Serono, Merck Sharp & Dohme, Novartis, Roche, Sandoz, Sanofi and Servier. R. Berardi received consulting fees and/or donations to institution from AstraZeneca, Pfizer, Roche, Astellas, Eisai, Merck Sharp & Dohme, Menarini, Gilead, Pierre Fabre, Bristol Myers Squibb, Seagen, Incyte, Eli Lilly, Bayer and Daiichi Sankyo. received advisory/consulting fees from AbbVie, Bayer, Novartis, Orbus, Celgene, GlaxoSmithKline, Health4u, Braun Melsungen, Janssen, BioRegio-Stern, Servier, Novocure and TME Pharma and travel fees from Roche, Bayer and Servier. G. Masi received fees for advisory boards from AstraZeneca, Bayer, Ipsen, Merck Sharp & Dohme, Eisai, Roche and Sirtex; speaking honoraria from Amgen, AstraZeneca, Roche, Merck Sharp & Dohme, Eisai, Terumo and Sirtex; travel grants from AstraZeneca, Bayer, Ipsen, Merck Sharp & Dohme, Eisai and Roche; and research grants from Roche, Terumo and Sirtex. Nature Medicine thanks Alastair Greystoke, Nirupa Murugaesu and Xueliang Pan for their contribution to the peer review of this work. Primary Handling Editor: Ulrike Harjes, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Genomic alterations and tailored treatment proposed by MTB in ITT population. Molecular tumor board activity and treatment allocation in the intention-to-treat (ITT) population. a) Grade ≥ 3 adverse events that occurred in at least 5 patients. SoC: standard of care, TT: tailored treatment; CTCAE: Common Terminology Criteria for Adverse Events. TT: tailored treatment; SOC: standard of care; PD: Progressive disease; NE: Not evaluable; SD: Stable disease; PR: Partial response; CR: Complete response; ITT: intention-to-treat. TT: tailored treatment; SOC: standard of care; PD: Progressive disease; NE: Not evaluable; SD: Stable disease; PR: Partial response; CR: Complete response; hTMB: High tumor mutational burden; MSI-H: microsatellite instability high. The Molecular Tumor Board (MTB) ‘s organizational structure and core activities illustrate its workflow and multidisciplinary collaboration for precision oncology decision-making. Supplementary Table 1: Type of previous treatments received by patients in the ITT population; Supplementary Table 2: Crossover rate; Supplementary Table 3: Reasons for patients not undergoing crossover (SoC arm); Supplementary Table 4: Extension of tumor primary site listed as ‘Other' in Table 1; Protocol V4; Protocol Appendix 2; and CONSORT checklist. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Marchetti, P., Curigliano, G., Biffoni, M. et al. Genomically matched therapy in advanced solid tumors: the randomized phase 2 ROME trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a68114049/south-africa-rock-art-uncovered-mysterious-creature/'>Archaeologists Uncovered a Painting That May Prove the Existence of a Mysterious Creature</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 14:29:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thanks to some suggestive fossils and legends of old, talk of such a creature isn't a new concept. But the recent discovery of 200-year-old rock paintings found in South Africa now has scientists hypothesizing that this ancient creature may have been far more than just a legend. If we're lucky, it could provide further clues as to just how intertwined this horned serpent was with South Africa's indigenous San culture. “Altogether,” the Benoit wrote, “they suggest a case of indigenous paleontology.” “Of course, at this point it is speculative,” said Benoit, according to IFL Science, “but the tusked animal on the Horned Serpent panel was likely painted as a rain-animal, which means it was probably involved [in] rain-making ceremonies.” These ceremonies often evoked known extinct animals to help the people encourage the gods to send rain. The San were known to have a robust mix of animals belonging to their ‘spirit world,' but Benoit said that these animals were generally inspired by reality—even if extinct. There's plenty of leaps from legend to scientific grounding that may be too large for a fabled horned serpent to make. But after further studies, the 200-year-old paintings could spin a different tale. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Fire Patrol Finds Ancient Face Carving in Boulder Experts Find Missing Piece of Ramesses II Statue Scientists Say Daylight Savings Time is Hurting Us These Edge State Atoms Showcase a Quantum Mystery</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a68113623/vienna-roman-mass-grave-found/'>A Construction Crew Was Renovating a Soccer Field—and Found the Bodies of 150 Ancient Roman Soldiers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 14:19:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“Everything points to the catastrophic end of a military operation.” Yikes. When construction workers started churning up skeletal remains, a project to renovate a soccer field outside Vienna, Austria, morphed into an archaeological dig, and it wasn't long before experts realized this field was actually a major discovery in Roman warfare history. The limbs were intertwined with those of other individuals. This indicates a hasty covering of the dead with earth, i.e. not an orderly burial.” With every individual examined a male, most between the age of 20 and 30 years, and in good health combined with the causes of death due to injuries from blunt and sharp weapons, including spears, daggers, swords, and iron bolts, “the variety of injuries indicates a battle and not an execution site.” With cremation burial common in the European Roman Empire, finds of Roman skeletons from this period are considered rare, leading to this being one of the most significant Roman war discoveries in Central Europe. Instead, everything points to the catastrophic end of a military operation.” While most of the dead were robbed of their weapons and equipment, the team still discovered a Roman iron dagger with inlays of silver wire, several scales of armor that show distinct differences from known varieties, the metal cheek piece of a Roman helmet, two iron spearheads (one stuck in a hip bone, mind you), and hobnails from shoes that were made with leather and studded with nails, a type of footwear used by Roman soldiers. Hasenleitengasse may therefore mark the beginning of Vienna's urban history.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Fire Patrol Finds Ancient Face Carving in Boulder Experts Find Missing Piece of Ramesses II Statue Scientists Say Daylight Savings Time is Hurting Us These Edge State Atoms Showcase a Quantum Mystery</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03944-9'>Prevention of type 2 diabetes through prediabetes remission without weight loss</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 14:04:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Clinical practice guidelines recommend defined weight loss goals for the prevention of type 2 diabetes (T2D) in those individuals with increased risk, such as prediabetes. However, achieving prediabetes remission, that is, reaching normal glucose regulation according to American Diabetes Association criteria, is more efficient in preventing T2D than solely reaching weight loss goals. Here we present a post hoc analysis of the large, multicenter, randomized, controlled Prediabetes Lifestyle Intervention Study (PLIS), demonstrating that prediabetes remission is achievable without weight loss or even weight gain, and that it also protects against incident T2D. The underlying mechanisms include improved insulin sensitivity, β-cell function and increments in β-cell-GLP-1 sensitivity. Weight gain was similar in those achieving prediabetes remission (responders) compared with nonresponders; however, adipose tissue was differentially redistributed in responders and nonresponders when compared against each other—while nonresponders increased visceral adipose tissue mass, responders increased adipose tissue in subcutaneous depots. These data uncover essential pathways for prediabetes remission without weight loss and emphasize the need to include glycemic targets in current clinical practice guidelines to improve T2D prevention. Globally, more than 460 million people live with type 2 diabetes (T2D) and many go through comorbidities such as neuropathy, chronic kidney disease or cardiovascular (CV) disease, making T2D one of the top 10 leading causes of death worldwide. Indeed, the increase in global mortality burden until 2050 is projected to be mostly due to neoplasms, T2D and kidney diseases1. Moreover, the current data suggest that T2D incidence will continue to rise, with most cases attributable to diet quality2. Additionally, the highest total number of affected people live in middle-income and low-income countries, where current guideline-based treatments are not as easily available as in high-income countries. Therefore, prevention must still be regarded as a major pillar in achieving the WHO's global noncommunicable disease goals and in equitably reducing the major burden of T2D3. Prediabetes is the most prominent risk factor of T2D with a yearly progression rate of 5–10% and a lifetime progression risk of 74%4,5, and an independent risk factor for vascular diseases, cancer and neurodegenerative diseases4,6,7. According to the American Diabetes Association (ADA), prediabetes is diagnosed when glucose regulation is impaired and/or HbA1c elevated while criteria of T2D are not met8,9. The U.S. Diabetes Prevention Program (DPP) Outcome Study (DPPOS) has demonstrated that reversal of impaired glucose regulation to normal at least once during a lifestyle intervention (LI) was effective in reducing the risk of T2D and microvascular disease10,11. In the Prediabetes Lifestyle Intervention Study (PLIS)12, we extended these findings by establishing the concept of prediabetes remission13,14. Accordingly, weight-loss-induced remission of prediabetes is reached, once the normal glucose regulation (NGR; that is, fasting glucose < 5.6 mmol l−1 (100 mg dl−1), 2 h glucose < 7.8 mmol l−1 (140 mg dl−1) and HbA1c < 39 mmol mol−1 (5.7%)) is re-established. Weight-loss-induced prediabetes remission (>5% of initial body weight) was explained by improved insulin sensitivity and reduced visceral adipose tissue (VAT) volume. At long-term follow-up, participants reaching weight-loss-induced prediabetes remission had a 73% reduced risk of developing T2D compared to those who only met the weight loss goal (but not prediabetes remission) and also had reduced signs of kidney and small vessel damage13, which may be due to reduced glycemic exposure over time as has previously been indicated10. A comparable concept to prediabetes remission has successfully been established in people with T2D in the DIRECT trial, where substantial weight loss led to a return to nondiabetic glucose levels in up to 85% of participants, although the risk of redeveloping T2D was high15,16. In general, part of weight loss targeted LIs is physical exercise, which has been linked with reduced inflammation17,18 and has been shown to improve insulin sensitivity independently of body weight change19. In individuals with T2D, it has also been demonstrated that physical exercise can improve glycemia without a substantial weight loss effect20. However, neither DIRECT nor DPP reported on the preventive outcomes of patients who did not reduce body weight but achieved remission. Here, we provide evidence that non-weight-loss-induced remission of prediabetes protects from T2D development for up to 10 years after the LI started, and that it is characterized by increased subcutaneous adipose tissue (SCAT) compared to nonremission, where VAT increases. Moreover, we show that non-weight-loss-induced remission of prediabetes is mediated by an improvement in insulin sensitivity and insulin secretion. Our data provide new support for the importance of implementing glycemic targets into current treatment guidelines to improve T2D prevention14, and show that the sole assessment of weight trajectories without body fat distribution is not adequately informative for the pursuit of treatment success. We analyzed data from the ongoing multicenter LI study PLIS that recruited individuals with prediabetes in study centers throughout Germany within the framework of the German Center for Diabetes Research (DZD e.V. ), where participants received the intervention for 12 months and were followed up for up to 9 years with metabolic phenotyping including oral glucose tolerance tests (OGTTs) for glucose metabolism and whole-body magnetic resonance imaging (MRI) for assessment of body fat distribution before and after the intervention and during follow-up (see Methods for details). Of the 1,105 individuals originally included in PLIS, 234 (21.2%) did not lose or even gained weight during the year of the intervention. Of these, 51 returned to NGR, that is, were designated ‘responders' (R; 21.8%), while 183 (78.2%) were ‘nonresponders' (NR). Baseline characteristics for both groups are given in Extended Data Table 1. Overall, there were more women in both groups and R tended to have more women than NR (60.1% in NR versus 74.5% in R, P = 0.085). As has previously been reported for prediabetes remission11,13, R were younger (median = 54.4 ± 17.6 years) than NR (59.4 ± 15.5 years, P = 0.013), had lower fasting and 2 h glucose and slightly higher insulin sensitivity. Intervention intensity was not different between groups (Extended Data Table 1). At first, we investigated whether weight trajectories during the 1-year intervention period were different between R and NR. Maximal aerobic capacity did not differ between groups (18 ± 2.2 ml min kg−1 to 17.6 ± 2.5 ml min kg−1 in R versus 18.5 ± 1.6 ml min kg−1 to 18.4 ± 1.9 ml min kg−1 in NR, P group over time = 0.65; Extended Data Fig. Additionally, adherence to dietary advice based on the evaluation of food diaries was not different between groups, similarly neither habitual physical activity (HPA) score nor daily distance walked (in a subgroup) or lean body mass differed between groups during the intervention and during follow-up, respectively (Extended Data Fig. Taken together, these data show that prediabetes remission without weight loss compared to nonremission was independent of weight trajectories, overall body composition and physical fitness. Next, we explored the underlying determinants of improved glucose regulation in R versus NR. Glucose concentrations during the OGTT were lower in R and per definition decreased over time in R only (Fig. Also, consecutive insulin concentrations during the OGTT were higher in NR than in R before, during and after the intervention, but increased in R between 30 and 60 min during the second half of the intervention (Fig. a, Time course of glucose (R, n = 51; NR, n = 183) concentrations during the OGTT over the course of the LI. Means of raw values are depicted with error bars indicating 95% confidence intervals in all panels. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. The R (defined by NGR, that is, prediabetes remission, after 12 months of LI) group and the NR (that is, not reaching NGR) group are depicted in blue and orange, respectively. FDR-corrected group × time P values are 0.0008 for all comparisons. Dashed lines in a represent ADA criteria-based cutoff values for prediabetes for fasting glucose (5.6 mmol l−1) and 2 h OGTT glucose (7.8 mmol l−1). To better understand the increase of insulin concentrations in R, we calculated OGTT-derived indexes for insulin sensitivity, secretion and β-cell function. We found that throughout the LI, insulin sensitivity did not change in NR but increased in R. The oral glucose insulin sensitivity (OGIS) index increased from 336.45 ± 15.41 ml min−1 m−2 to 358.23 ± 18.88 ml min−1 m−2 in R, whereas it remained unchanged (299.42 ± 8.67 ml min−1 m−2 to 297.42 ± 9.09 ml min−1 m−2) in NR, P group over time: 0.0035 (Fig. Matsuda index of insulin sensitivity revealed the same development with a numeric increase from 8.94 ± 1.35 a.u. in NR, P group over time = 0.00041, Fig. An index of hepatic insulin resistance did not show differences between groups (Fig. Also, there was no indication of differences in muscle or adipose tissue insulin resistance (Fig. However, insulin secretion and β-cell function increased in R but not in NR. The C-peptide area under the curve (AUC)0–30 min/glucose AUC0–30 min, an index of insulin secretion, increased from 159.5 ± 17.13 pmol mmol−1 to 169.96 ± 19.3 pmol mmol−1 in R, but remained constant (157.89 ± 7.81 pmol mmol−1 to 161.81 ± 8.15 pmol mmol−1) in NR (P group over time = 0.043; Fig. The Adaptation Index, a marker of β-cell function, increased from 5.16 × 104 ± 3,833.80 a.u. in R and remained unchanged (4.61 × 104 ± 2,192.89 a.u. in NR (P group over time = 0.025; Fig. An overview of the changes in the hyperbolic relationship between insulin sensitivity and insulin secretion for the two groups is given in Fig. The hyperbola shows that while R were able to increase insulin sensitivity and secretion at the same time, NR did not improve insulin sensitivity or insulin secretion. Change of hepatic insulin clearance was not different between groups (3.91 ± 0.36 a.u to 3.68 ± 0.4 a.u in R versus 3.39 ± 0.17 a.u to 3.31 ± 0.16 a.u in NR, P group over time = 0.49; Fig. 2i) and, thus, cannot explain the increase in insulin concentrations observed in R. With the increase in insulin secretion and β-cell function alongside improved insulin sensitivity, remission mechanisms without weight loss differ from weight-loss-associated prediabetes remission, where the increase in insulin secretion has not been observed previously13. h, The hyperbolic relationship between insulin sensitivity (OGIS) and secretion (C-peptide/glucose AUC0–30 min—R, n = 51; NR, n = 183) all derived from OGTT at baseline and 12 months. The R (defined by NGR, that is, prediabetes remission, after 12 months of LI) group and the NR (that is, not reaching NGR) group are depicted in blue and orange, respectively. Means of raw values are depicted with error bars indicating 95% confidence intervals in a–g. In h and i, the base of the arrows depicts medians at baseline and the respective tip after 1 year of LI. P values directly above the x axes in a–g and i indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. Because insulin sensitivity and secretion are linked with ectopic adipose tissue deposition21,22,23, we then determined lipid accumulation in SCAT, VAT and ectopic lipid depots based on the MRI and 1H-spectroscopy data. Intrahepatic lipid content (IHL) was slightly higher in NR versus R (Extended Data Table 1) but remained constant in both groups during the LI (5.08 ± 1.71% to 5.9 ± 1.74% in R versus 7.65 ± 1.14% to 8.13 ± 1.18% in NR, P group over time = 0.74; Fig. 3a), suggesting no critical contribution of changes in IHL for nonweight-loss remission. However, while VAT did not increase in R despite weight gain, it increased along with weight gain in NR (4.31 ± 0.69 l to 4.24 ± 0.69 l in R versus 4.99 ± 0.32 l to 5.41 ± 0.36 l in NR, P group over time = 0.031; Fig. In contrast, SCAT increased more in R versus NR (14.41 ± 2.39 l to 15.7 ± 2.06 l in R versus 14.97 ± 1.06 l to 14.66 ± 0.98 l in NR, P group over time = 0.035) showing that with weight gain, R predominantly stored additional energy in SCAT but NR in VAT (Fig. Change in intramuscular fat was not different between groups (5.85 ± 1.24% to 6.15 ± 1.49% in R versus 6.4 ± 1.45% to 6.76 ± 1.49% in NR, P group over time = 0.83; Fig. These data highlight that lipid deposition during weight gain is most likely a crucial factor for improvements in glucose regulation during non-weight-loss prediabetes remission. a, Trajectories of hepatic lipid content as assessed with 1H-MRS (R, n = 51; NR, n = 183). e, Muscle fat content as assessed with MRI (R, n = 16; NR, n = 27). f, Group comparison of the adjusted PRS for VAT volume (R, n = 32; NR, n = 164). In a–e, means of raw values are depicted with error bars indicating 95% confidence intervals. In f, box plots are centered on medians, boxes extend to 25th and 75th percentiles and whiskers extend to 1.5× IQR (top and bottom). P values directly above the x axes in a–e indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. The R (defined by NGR, that is, prediabetes remission, after 12 months of LI) group and the NR (that is, not reaching NGR) group are depicted in blue and orange, respectively. To further investigate whether the genetic background is a key driver of ectopic lipid deposition during weight gain, we calculated a polygenic risk score (PRS) derived from a subset of single-nucleotide polymorphisms (SNPs; Methods) that have been described to be associated with VAT24. However, there were no differences in the PRS between R and NR with or without adjustments for covariates (Fig. Similar to previously published data on weight-loss-induced prediabetes remission, there was no association between the change in IHL and the proportion of remission (Fig. However, lower VAT increase or even reductions in VAT were linked with higher remission success (P = 0.024; Fig. According to the changes in SCAT and VAT, a higher SCAT/VAT ratio after the LI was associated with higher frequency of remission, again suggesting that the distribution of adipose tissue gained may determine prediabetes remission (P < 0.0001; Fig. Finally, there was a trend toward higher remission rates with weight gain (Fig. a, Stratification of R (defined by NGR, that is, prediabetes remission, after 12 months of LI, n = 51; blue) and NR (that is, not reaching NGR, n = 183) by postintervention hepatic lipid content as assessed with 1H-MRS. b, Change of hepatic lipid content during LI. c, Postintervention VAT volume as assessed with MRI. d, Change of VAT volume during LI. f, Change of body weight during LI. Top numbers inside the bars indicate stratum size (n). Large bottom numbers indicate BMI (a,c) or percent body weight change (b,d,e). Small bottom numbers indicate 95% CI, respectively. VAT is associated with increased low-grade inflammation and weight-loss-associated prediabetes remission13. Thus, we measured biomarkers of inflammation suggested to contribute to insulin resistance. However, there were no substantial differences in individual trajectories of inflammatory markers between R and NR (Supplementary Tables 1 and 2). Thus, low-grade inflammation, at least at the circulation level, may not explain improved insulin sensitivity during prediabetes remission. Because hormonal factors derived from adipose tissue are also linked with insulin sensitivity, we measured the adipokines leptin and adiponectin. Adiponectin levels, however, were not different at baseline (3096.48 ± 535.75 ng ml−1 in R, 2412.30 ± 207.49 ng ml−1 in NR, P = 0.094), but were substantially higher after the intervention in R versus NR (3371.57 ± 596.55 ng ml−1 in R, 2337.20 ± 217.24 ng ml−1 in NR, P = 0.0040, P group over time = 0.088; Extended Data Fig. 2), suggesting that increases in SCAT may chaperone improved insulin sensitivity by increased adiponectin secretion, at least in part. To further understand the improvement in insulin secretion and β-cell function in R, we assessed lipid content within triglyceride-containing VLDL particles, because a reduction in VLDL1 palmitate has been shown to be associated with improved β-cell secretory capacity during the remission of T2D25. Despite higher levels in total and specific lipid species-bound palmitate in NR as well as triglyceride levels, there was no difference in the trajectories during LI between R and NR (Extended Data Fig. 3a–l), suggesting that hepatic palmitate export may not have major impact on the insulin secretion phenotype during prediabetes remission. We then assessed concentrations of glucagon-like peptide 1 (GLP-1), gastric inhibitory polypeptide (GIP) and glucagon to investigate whether incretin responses during the OGTT or sensitivity (see below) may explain improved insulin secretion in R. As expected, in both groups GLP-1 concentrations increased between 0 and 30 min, and then decreased at 120 min (Fig. While there was no substantial difference in the trajectories over time between groups (before, P = 0.13; after, P = 0.14), the GLP-1AUC substantially increased in NR (who maintained that their glucose levels in the prediabetic range or advanced to overt T2D) but remained stable in R despite reduction of glucose levels to normal (P = 0.78; Fig. Incretin hormone levels during the OGTT at baseline and 12 months, n = 32 R versus 131 NR. Means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R (defined by NGR, that i,s prediabetes remission, after 12 months of LI) versus NR (that is, not reaching NGR) over time (that is, interaction term of group and time) derived from mixed effects models. Indicators above the time points indicate between-group difference at the respective time point derived from post hoc comparisons of the respective mixed effects models. GIP followed similar trajectories during the OGTT but remained elevated at 120 min in both groups (Fig. Neither group trajectories during the OGTT before or after nor GIPAUC differed between R and NR, rendering GIP unlikely to be involved in improved β-cell function during nonweight-loss prediabetes remission (Fig. However, glucagon was lower in R and remained unchanged during the LI, whereas in NR glucagon showed an additional increase after the intervention (Fig. Interestingly, reduction in glucagon concentrations between 0 and 30 min during the OGTT remained stable in NR, but increased substantially in R (Fig. Therefore, we then calculated the insulin secretion rate based on a previously published algorithm26,27,28 and plotted it against GLP-1 secretion as a measure of β-cell-GLP-1 sensitivity. Because of the recent data showing that prediabetes remission with weight loss reduces the risk for future T2D development13, we here investigated if remission of prediabetes despite the absence of weight loss or even with weight gain is also protective from future T2D development. Indeed, more NR than R developed T2D leading to a relative risk (RR) reduction of 71% in R over the period of up to 10 years (RR = 0.29, 95% CI = 0.09–0.91, P = 0.02; Fig. 6a,b) and thus prediabetes remission without weight loss was comparably protective for T2D development as prediabetes remission with weight loss (73%)13. a, Kaplan–Meier curve representing proportion of individuals without T2D over time. P value is derived from log-rank tests. b, Sankey plot of glycemic categories (that is, T2D, prediabetes and NGR) over time. P value is derived from two-sided Fisher's exact test. To exclude any potential effect that may be specific to our cohort and to enhance the generalizability of the results, we validated our findings in repository data from DPP. We also identified 494 individuals who did not lose or even gain body weight after 12 months of the intervention. Baseline characteristics are given in Supplementary Table 3. Similar to PLIS, R and NR increased BMI comparably, and NR increased VAT while R did not (Extended Data Fig. Additionally, R showed improved insulin sensitivity as well as insulin secretion (Extended Data Fig. 6a–c), even after adjusting for insulin sensitivity. Finally, individuals with prediabetes remission in DPP at 1 year without weight loss also had an RR reduction for future T2D of 73% (RR = 0.27, 95% CI: 0.11, 0.66, P < 0.0001) during a median follow-up time of 4.11 (IQR: 1.44, 13.48) years. These data reproduce the findings that prediabetes remission without weight loss is characterized by improved insulin sensitivity as well as improved insulin secretion and that excess lipid deposition into VAT may preclude persons from nonweight-loss prediabetes remission. Moreover, recent data indicate that the 1 h glucose OGTT value may better predict the development of T2D than the 2 h OGTT value29. We, therefore, performed the same analysis by using the proposed 1 h cutoff of 155 mg dl−1 to identify participants as having prediabetes or NGR, but this change in criteria for prediabetes did not change our findings regarding glucose and insulin metabolism during prediabetes remission (Extended Data Fig. Our data show that remission of prediabetes without weight loss is a genuine and reproducible response to an LI in a substantial proportion of participants (up to 22%) and that the response is highly effective in preventing T2D. Individuals going into remission without losing body weight are characterized by increased SCAT/VAT ratio, indicating that the site of fat deposition during an LI determines the probability of prediabetes remission. Elegant studies have shown that physical exercise reduces glycemia with no or only marginal effects on body weight20,30. 31,32 have shown that exercise without a weight loss effect can specifically reduce VAT. Additionally, diet composition has a role in VAT accumulation—diets high in polyunsaturated fatty acids have been shown to protect against VAT gain compared to those rich in saturated fats, even during overfeeding and weight gain33. Thus, physical exercise and dietary habits may have contributed to lower glycemia and to the observed SCAT and VAT distribution phenotype. Moreover, non-weight-loss-associated prediabetes remission was characterized by a combination of higher insulin sensitivity and improved insulin secretion, which contrasts this response from weight-loss-induced prediabetes remission in PLIS and DPP where remission was characterized by improved insulin sensitivity but not secretion and T2D remission in DIRECT, which was accompanied by improved β-cell function but not insulin sensitivity13,21,34. Notably, prediabetes remission without weight loss was protective from T2D development with an RR reduction of >70%, and thus to a comparable degree as weight-loss-induced prediabetes remission in the previous study13. Taken together, these data show that distinct physiological mechanisms underlie non-weight-loss-induced remission of prediabetes and that it is essential to differentiate between weight-loss and non-weight-loss strata, when investigating the mechanisms that can mediate prediabetes remission. Moreover, our data suggest that weight loss, as recommended in current guidelines, should be complemented by remission to NGR to prevent T2D in those people with prediabetes. This is in line with recent efforts delineating the clinical heterogeneity of obesity in projection for cardiometabolic diseases, specifically when cardiometabolic risk is discordant with the risk that is to be expected when obesity is classically defined solely by the BMI35. Furthermore, new statements and definitions of expert commissions for the diagnosis and management of obesity highlight the importance of body fat distribution and function beyond adiposity and underscore the clinical manifestation of organ dysfunction for the diagnosis of clinical obesity36,37. 38 showed that large amounts of SCAT in combination with little visceral adiposity are associated with high insulin sensitivity during a hyperinsulinemic–euglycemic clamp. Exercise can favorably influence SCAT composition, even without weight loss39, and SCAT of regularly exercising individuals is more expandable than that of sedentary people with obesity40. The importance of exercise for improving insulin sensitivity has previously been demonstrated41; however, lean body mass and VO2 max, a marker of maximum aerobic capacity, were not different between R and NR. Nonetheless, we cannot rule out that subtle differences in physical exercise (volume) may have impacted insulin sensitivity. Although we did not observe a difference in the PRS, the genetic basis of unfavorable body fat distribution and T2D risk has previously been reported42,43,44. Interestingly, thiazolidinediones (TZDs), a class of oral glucose-lowering medications that activate the transcription factor, peroxisome proliferator-activated receptor γ, led to a redistribution of fat mass from VAT to SCAT depots. TZDs act as insulin sensitizers at least in part by this mechanism45,46. They have also been shown to increase body weight and to improve insulin sensitivity and β-cell function, similar to our observation with nonweight-loss prediabetes remission45,47. This trend toward a favorable adipose tissue composition tracks closely with systemic adiponectin concentrations. Adiponectin is released by adipose tissue and is associated with increased systemic insulin sensitivity48. This finding is in line with recent findings of a gradual decrease of adiponectin levels from metabolically healthy lean to metabolically healthy obese to metabolically unhealthy obese persons49. Also, in the data presented here, adiponectin levels were higher in R after the intervention, which may contribute to their increased insulin sensitivity. In R, but not NR, insulin secretion and β-cell function increased during LI. Palmitate, partly a product of hepatic de novo lipogenesis, has a particularly detrimental role in reducing β-cell function, most likely by inducing β-cell dedifferentiation50. Hepatic palmitate secretion has been shown to be modifiable by LI and reduced VLDL1 palmitate export from the liver has been shown to be linked with T2D remission in the DIRECT trial as well as remission maintenance after weight-loss-dependent remission25. In our study, in the setting of non-weight-loss prediabetes remission, VLDL1 palmitate export did not change between R and NR, implying that hepatic VLDL1 palmitate export may not be responsible for the lack of increase in β-cell function in the NR. However, we did not assess VLDL-production rates, previously demonstrated to be important for T2D remission25. We then assessed the incretin system to explain the improvement in insulin secretion observed in R, but not in NR. Both GLP-1 and GIP enhance glucose-dependent insulin secretion51. GLP-1 secretion after glucose ingestion is only minimally reduced in impaired glucose tolerance (IGT) as well as in T2D52,53. However, its effectiveness in augmenting insulin secretion is reduced56. In our cohort, GLP-1 and GIP time courses during OGTT were not different between groups. GLP-1AUC remained stable in R despite reduced glucose levels and increased insulin levels (Fig. This may reflect improved GLP-1 sensitivity in R, potentially contributing to improved β-cell function by its improved insulinotropic effect that has been shown to be more pronounced during normoglycemic conditions57. Indeed, an index of β-cell-GLP-1 sensitivity showed increased sensitivity in R, but not in NR. Because GLP-1 has a glucagonostatic effect58, improved GLP-1 sensitivity may be supported by the finding that R showed improved glucagon suppression during the OGTT. However, we cannot exclude that this could also be a consequence of the improvements in glucose regulation restoring the glucosuppressive effect on glucagon that may also be influenced by diet59. Prediabetes remission without weight loss reduced the risk of subsequent T2D by 71% in our study. Previous studies of people with weight-loss-induced prediabetes remission showed similar risk reductions (73%)13,60, which is comparable to T2D risk reduction induced by pharmacological interventions61. In a post hoc analysis from the PROactive trial, weight gain with pioglitazone, a TZD, which leads to weight gain with a redistribution from VAT to SCAT and improvements in insulin sensitivity62,63, was associated with improved CV outcomes (excluding heart failure)64. Interestingly, recent data from our group show that prediabetes remission, in contrast to multimodal LIs targeting mainly weight loss, is also associated with reduced CV outcomes65,66. Together, these data indicate that there is a weight-independent component of the glycemic status, which is related to body fat distribution. Indeed, individuals reaching both prediabetes remission and the guideline-recommended weight loss target of 7% are 76% less likely to develop T2D8, compared to the group that only met the weight loss target60. These data highlight the importance of incorporating glycemic targets into practice guidelines in addition to weight loss targets. Remission of prediabetes is the most effective way to prevent future T2D cases, and our current data indicate that this is partially independent of weight loss. In fact, sustainable weight loss is rarely achievable with >90% of weight loss recidivism and the probability of obtaining normal body weight is very low with less than 1 per 1,000 for men and less than 1 per 600 for women in individuals with severe obesity67,68. Therefore, from a clinical perspective, primary treatment goals should focus on achieving metabolic health rather than weight loss alone. Bringing prediabetic hyperglycemia back to normoglycemia seems to be an important contribution and indicator for metabolic health69. This study is limited by the surrogate parameters derived from the OGTT for estimation of insulin sensitivity and secretion/β-cell function. Additionally, we cannot rule out confounders that were not assessed, such as genetic predisposition (beyond the investigated SNPs) and environmental factors. This study is additionally limited by the lack of a priori powering due to the post hoc nature of this analysis. To strengthen the validity of the data, we have therefore included data from the DPPOS cohort that replicate and support our current findings and interpretations. Additionally, the post hoc group stratification may lead to residual confounding effects, that is, the stratification into remission and nonremission leads to baseline differences, for example, in glucose levels, as has also been observed in studies investigating T2D remission mechanisms70 and in previous analyses from DPPOS10,11, which may affect the likelihood of remission at the individual level. Nonetheless, we show that for individuals who do achieve remission, the risk of future T2D development is substantially reduced—even in the absence of weight loss. To minimize confounding of our results by baseline misclassification, we included fasting as well as postchallenge glucose and HbA1c into our definition of remission, whereby test–retest reliability has been shown to be up to 98.6%71. While spontaneous remission from prediabetes to normoglycemia might be observable even without LI72, our findings are not likely to be influenced by an erroneous classification. We conclude that the protective effect of LI in individuals with prediabetes is not solely dependent on weight loss, as none of the participants included in the present analysis lost weight. Non-weight-loss prediabetes remission is characterized by a ‘TZD-like' body fat distribution. Current guideline recommendations involve multimodal LI with a focus on weight loss targets of 5–10%. In light of the current data, we recommend achieving metabolic health by incorporating glycemic targets to reach NGR (prediabetes remission) in addition to weight loss, thereby optimizing T2D risk reduction through a precision prevention approach. Participants of the risk-stratified, randomized, controlled and multicenter PLIS were recruited between 1 March 2012 and 31 August 2016 to assess potential superiority of an intensified LI compared to conventional DPP-based LI in improving glucose tolerance in people with prediabetes at high risk for T2D development. Sex was determined through self-report. People with prediabetes (impaired fasting glucose, IGT or both according to criteria from the ADA) between the ages of 18 and 75 years were included8. Participants with high IHL in addition to either high insulin resistance or low insulin secretion were stratified as high risk. After stratification, randomization of high-risk participants to receive either intensified LI or conventional LI and of low-risk participants to receive either conventional LI or a control intervention was performed. For this analysis, individuals from PLIS who did not experience weight loss over the 12-month period of those of the intensified LI, the conventional LI or the control intervention were included. Participants were classified as either R or NR. R were defined by return to normal fasting plasma glucose (that is, <5.6 mmol l−1), normal glucose tolerance (that is, 2 ­h post­load glucose <7.8 mmol l−1) and HbA1c less than 39 mmol mol−1 after 12 months of all three interventions. NR were defined by fasting plasma glucose, 2 h glucose or HbA1c higher than these thresholds after 12 months of all three interventions. Glucose regulation was assessed in the morning at 8 a.m. during an overnight fast through OGTT with blood sampling at 0, 30, 60, 90 and 120 min using a 75 g glucose load (Accu-Check Dextro O.G.T. Glucose concentrations were assessed using the glucose oxidase method in certified laboratories at each clinical study site. All insulin and C-peptide measurements were done at the Tübingen study site on an ADVIA Centaur XP system (Siemens Healthineers). Clinical chemistry parameters, including HbA1c, were assessed at the local routine diagnostic laboratories of the participating centers, and all of these were certified by the German accreditation council (DAkkS). Volunteers participated in this study for a mean of 5.2 ± 2.8 years (max = 9.9 years). Adherence to the dietary intervention was assessed by evaluation of food diaries through the study counselors during the counseling sessions, where food diaries of 4 consecutive days were evaluated. Markers of physical activity and fitness were assessed by the VO2 max during ergometry, quantification of lean body mass and the HPA. The VO2 max was assessed as the maximal oxygen uptake during incremental physical exertion on an ergometer, lean body mass was estimated through bioelectric impedance analysis and the HPA based on the self-report that addressed the following main components: physical activity during work, physical activity during leisure time and sports during leisure time74. In a small subgroup of individuals, step counts were extracted to document daily distance walked (R/NR, n = 6/17). Further details of the intervention are provided elsewhere12. Of these measures, lean body mass and the HPA were continuously assessed. Written informed consent was provided by each participant at inclusion. The study protocol is available online (https://www.dzdev.de/fileadmin/DZD/PDF/Papers/Forschung_KlinStud_Ethik_Studienprotokoll_PLIS_1.4.pdf) and approved by the ethics committee of the University Clinic of Tübingen (Tübingen, 55/2012; ClinicalTrials.gov registration: NCT01947595). Further details have been published previously12. Financial compensation for participants was granted exceptionally upon request, only if participants lived far away from the study center. Repository data from the U.S. DPP study were analyzed to verify results from PLIS in a different population75,76. Participants were recruited from 23 clinical study centers in the United States between 31 July 1996 and 18 May 1999, and randomized to either LI or metformin or placebo groups. For the present analysis, data from LI and placebo group participants were used, who did not lose body weight during the first 12 ­months and had complete data on measurements of BMI, fasting and 30­-min insulin, and fasting, 30 ­min and 120-­min glucose during an OGTT. Ethics approval for the study was provided by the institutional review board of each participating clinical center. Written informed consent was obtained from all DPP participants. The DPP study protocol is available online at https://repository.niddk.nih.gov/study/38. This is a post hoc analysis of a prespecified endpoint in PLIS. Main outcomes are indexes of insulin secretion and sensitivity and distribution of body fat as main determinants of response or nonresponse to an LI. The specifics of sampling and processing have been published elsewhere12. All measurements were taken from distinct participants. To evaluate insulin secretion, we used C­-peptide AUC divided by glucose AUC during the first 30 min of the OGTT82. The Adaptation Index was calculated as (AUCc­pep0–30 divided by AUCgluc0–30) × OGIS to provide an integrated measure for insulin secretion adapted to underlying insulin resistance83. AUCc­pep0–120 divided by AUCins0–120 was calculated as a marker of hepatic insulin clearance84. Insulin secretion rate was calculated by fitting C-peptide concentrations at each time point during the OGTT to an established model of C-peptide deconvolution26,27,28. An incremental AUC was obtained from GLP-1 measurements during the OGTT in a similar fashion. These values were plotted against each other to depict β-cell-GLP-1 sensitivity, as previously described86. Insulin secretion and sensitivity were assessed from OGTT-based indexes in DPP in a similar fashion. The MRI was performed to measure muscle fat, subcutaneous and visceral fat compartments in PLIS. The applied methods are described in detail elsewhere12,87. VAT volume was determined from T1-weighted fast spin echo images that were assessed with a slice thickness of 10 mm (ref. Segmentation of VAT was done between hip and thoracic diaphragm using an automatic fuzzy c-means algorithm and orthonormal snakes89. IHL was quantified in PLIS only. Localized proton magnetic resonance spectroscopy (1H­MRS) applying a single­voxel stimulated echo acquisition mode localization technique with short echo time in the posterior part of segment VII was applied for IHL assessment. The ratio of signal integrals of fat (methylene + methyl signal) and total signal (water + fat) was used to determine IHL content, which is expressed in percent. In one center, 1H-MRS was not available and IHL content was determined by a chemical-shift selective imaging technique that generates fat and water selective images as described elsewhere90. From these, manually determined regions of interest in liver segment 7 performed in fat and water selective images, respectively, were used for IHL-content assessment. To ensure comparability with MRS-guided assessments, IHL content was calculated by fat/(water + fat) × 100, including corrections for relaxation effects. Both techniques have been shown to yield comparable assessments of IHL content91. More details have been described previously92. In DPP, VAT measurements were obtained by lumbar spine CT scans at L2, L3, L4 and L5. IHL and SCAT were not quantified in DPP. Only in PLIS, fat-selective MRI sequences were used to measure intramuscular fat fraction of musculus erector spinae and musculus spinalis thoracis. A region of interest avoiding macroscopic fat septae was manually drawn at the Th10/Th11 intervertebral level in a single axial slice90,92. Close proximity SCAT was used for reference (that is, 100%) and fat fractions were calculated12,87. VLDL were isolated from plasma by ultracentrifugation applying an NaCl density gradient (ρ = 1.006 g ml−1) at 541,000g for 60 min at 4 °C on an Optima Max-XP ultracentrifuge (Beckman). Subsequently, 60 µl of the VLDL-containing fraction were used for lipid extraction and shotgun analysis as previously described93,94. Lipid extracts were resuspended in IPA/MeOH/CHCl3 (4:2:1; vol/vol/vol) with 7.5 mM ammonium acetate and then infused through the TriVersa NanoMate ion source (Advion Biosciences) into an Exploris 240 (Thermo Fisher Scientific) mass spectrometer. All spectra were imported by LipidXplorer (1.2.8.1) into a MasterScan database. Lipid identification was carried out as described95. Multianalyte profiling on the Luminex-100 system (Bio-Rad Laboratories), with a combination of Bio-Plex Pro Human Immunotherapy Panel 20-Plex, Bio-Plex Pro Huma Inflammation Panel, Bio-Plex Pro Human Cytokine Panel and the 2-Plex Panel for vascular cell adhesion molecule 1 and intercellular adhesion molecule 1, was used to assess concentrations of circulating inflammatory markers in serum. Bio-Plex Multiplex Immunoassays (Bio-Rad Laboratories) are based on the xMAP Technology licensed from Luminex; all measurements were conducted according to the manufacturer's protocols. A detailed description of the methods is given elsewhere96. Serum concentrations of leptin (11-LEPHU-E01; Alpco) and high molecular weight adiponectin (80-ADPHU-E01; ALPCO) were determined using ELISA, following the manufacturer's instructions. Plasma levels of total GIP, total GLP-1 and glucagon were measured as previously described using validated immunoassays97. Individuals from PLIS were genotyped onto two different versions of the Illumina Global Screening Array chip (280 on GSA-MD-24v1 and 549 on GSA-MD-24v2). A previously described set of 205 SNPs for predicted VAT, including only genome-wide substantial variants described in ref. 24 were selected and subsequently restricted to 186 SNPs. Quality control was performed on all participants using the PLINK 1.9 software. SNPs exclusion criteria included missingness threshold of >5%, minor allele frequency <1% and Hardy–Weinberg equilibrium with P < 0.05. The summary statistics were downloaded from the NHGRI–EBI GWAS on 8 October 2024 for study GCST008744 (refs. Linear mixed effects models with group (that is, R and NR), time point and the interaction group × time point as model terms adjusted for age, sex, BMI, risk stratification and intervention intensity as fixed effects and each individual as random effect were fitted to analyze longitudinal data. Additionally, the model evaluating insulin secretion included insulin sensitivity as fixed effect. All group × time P values were corrected for multiplicity using the Benjamini–Hochberg procedure. To avoid overreliance on significance testing and dichotomization, we report β coefficients for all models, including 95% confidence intervals in Extended Data Fig. Diagnostic plots were plotted for each model to ensure normal distribution of residuals by visual inspection. For group-wise, cross-sectional comparisons, two-sided Wilcoxon rank-sum tests were performed. Resulting P values were also corrected for multiplicity of comparisons (Benjamini–Hochberg). T2D incidence up to almost 9 years after intervention end was compared through risk ratios and Fisher's exact test using the epitools package, version 0.5-10.1. A Kaplan–Meier curve with log-rank test is also reported. In the PLIS dataset, missing values for liver, subcutaneous, and visceral fat were imputed using the mouse package (version 3.16.0). In the DPP dataset, no imputation was performed. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The datasets from this study are not publicly accessible due to national data protection regulations and ethical restrictions designed to protect participant privacy. They can be requested after publication by contacting the corresponding author. Each request will be reviewed by the PLIS data steering committee, and approved access will require a formal data use agreement. The PLIS data steering committee will respond to requests within 3 months. Data from the DPP referenced in this study can be obtained through the U.S. NIDDK Central Repository (https://repository.niddk.nih.gov/)99. These datasets include demographic details, anthropometric and laboratory assessments, imaging data and time-to-diabetes diagnosis. No custom code was generated for this study. Previously published packages and functions have been referenced. Burden of disease scenarios for 204 countries and territories, 2022–2050: a forecasting analysis for the Global Burden of Disease Study 2021. Incident type 2 diabetes attributable to suboptimal diet in 184 countries. Sandforth, L. et al. Prediabetes remission to reduce the global burden of type 2 diabetes. Tabák, A. G., Herder, C., Rathmann, W., Brunner, E. J. & Kivimäki, M. Prediabetes: a high-risk state for diabetes development. Lifetime risk of developing impaired glucose metabolism and eventual progression from prediabetes to type 2 diabetes: a prospective cohort study. Chow, H.-M. et al. Age-related hyperinsulinemia leads to insulin resistance in neurons and cell-cycle-induced senescence. Huang, Y. et al. Prediabetes and the risk of cancer: a meta-analysis. American Diabetes Association Professional Practice Committee. Prevention or delay of diabetes and associated comorbidities: standards of care in diabetes-2025. Perreault, L. et al. Regression from prediabetes to normal glucose regulation and prevalence of microvascular disease in the Diabetes Prevention Program Outcomes Study (DPPOS). Perreault, L. et al. Effect of regression from prediabetes to normal glucose regulation on long-term reduction in diabetes risk: results from the Diabetes Prevention Program Outcomes Study. Different effects of lifestyle intervention in high- and low-risk prediabetes: results of the randomized controlled Prediabetes Lifestyle Intervention Study (PLIS). Sandforth, A. et al. Mechanisms of weight loss-induced remission in people with prediabetes: a post-hoc analysis of the randomised, controlled, multicentre Prediabetes Lifestyle Intervention Study (PLIS). Birkenfeld, A. L. & Mohan, V. Prediabetes remission for type 2 diabetes mellitus prevention. Lean, M. E. et al. Primary care-led weight management for remission of type 2 diabetes (DiRECT): an open-label, cluster-randomised trial. Lean, M. E. et al. 5-year follow-up of the randomised Diabetes Remission Clinical Trial (DiRECT) of continued support for weight loss maintenance in the UK: an extension study. Petersen, A. M. W. & Pedersen, B. K. The anti-inflammatory effect of exercise. The anti-inflammatory effects of exercise: mechanisms and implications for the prevention and treatment of disease. Exercise increases muscle GLUT-4 levels and insulin action in subjects with impaired glucose tolerance. Boulé, N. G., Haddad, E., Kenny, G. P., Wells, G. A. Effects of exercise on glycemic control and body mass in type 2 diabetes mellitus: a meta-analysis of controlled clinical trials. Zhyzhneuskaya, S. V. et al. Time course of normalization of functional β-cell capacity in the diabetes remission clinical trial after weight loss in type 2 diabetes. Wagner, R. et al. Lower hepatic fat is associated with improved insulin secretion in a high-risk prediabetes subphenotype during lifestyle intervention. Preis, S. R. et al. Abdominal subcutaneous and visceral adipose tissue and insulin resistance in the Framingham Heart Study. Contribution of genetics to visceral adiposity and its relation to cardiovascular and metabolic disease. Al-Mrabeh, A. et al. Hepatic lipoprotein export and remission of human type 2 diabetes after weight loss. van Vliet, S. et al. Obesity is associated with increased basal and postprandial β-cell insulin secretion even in the absence of insulin resistance. & Polonsky, K. S. Estimation of insulin secretion rates from C-peptide levels: comparison of individual and standard kinetic parameters for C-peptide clearance. Influence of adiposity, insulin resistance, and intrahepatic triglyceride content on insulin kinetics. Bergman, M. et al. International diabetes federation position statement on the 1-hour post-load plasma glucose for the diagnosis of intermediate hyperglycaemia and type 2 diabetes. & Angadi, S. S. Obesity treatment: weight loss versus increasing fitness and physical activity for reducing health risks. Exercise without weight loss is an effective strategy for obesity reduction in obese individuals with and without type 2 diabetes. Ross, R. et al. Exercise-induced reduction in obesity and insulin resistance in women: a randomized controlled trial. Rosqvist, F. et al. Overfeeding polyunsaturated and saturated fat causes distinct effects on liver and visceral fat accumulation in humans. Petersen, K. F. et al. Reversal of nonalcoholic hepatic steatosis, hepatic insulin resistance, and hyperglycemia by moderate weight reduction in patients with type 2 diabetes. Coral, D. E. et al. Subclassification of obesity for precision prediction of cardiometabolic diseases. Definition and diagnostic criteria of clinical obesity. Busetto, L. et al. A new framework for the diagnosis, staging and management of obesity in adults. Amati, F. et al. Lower thigh subcutaneous and higher visceral abdominal adipose tissue content both contribute to insulin resistance. Exercise training remodels subcutaneous adipose tissue in adults with obesity even without weight loss. Long-term exercise training has positive effects on adipose tissue in overweight or obesity. A. et al. Effect of the volume and intensity of exercise training on insulin sensitivity. Smith, K. et al. Multi-ancestry polygenic mechanisms of type 2 diabetes. Hodgson, S. et al. Genetic basis of early onset and progression of type 2 diabetes in South Asians. Lotta, L. A. et al. Association of genetic variants related to gluteofemoral vs abdominal fat distribution with type 2 diabetes, coronary disease, and cardiovascular risk factors. The effects of rosiglitazone on insulin sensitivity, lipolysis, and hepatic and skeletal muscle triglyceride content in patients with type 2 diabetes. Carey, D. G. et al. Effect of rosiglitazone on insulin sensitivity and body composition in type 2 diabetic patients [corrected]. Gastaldelli, A. et al. Thiazolidinediones improve β-cell function in type 2 diabetic patients. Klöting, N. et al. Insulin-sensitive obesity. Petersen, M. C. et al. Cardiometabolic characteristics of people with metabolically healthy and unhealthy obesity. Expression of mesenchymal and α-cell phenotypic markers in Islet β-cells in recently diagnosed diabetes. & Meier, J. J. Gastric inhibitory polypeptide and glucagon-like peptide-1 in the pathogenesis of type 2 diabetes. Vilsbøll, T., Krarup, T., Deacon, C. F., Madsbad, S. & Holst, J. J. Reduced postprandial concentrations of intact biologically active glucagon-like peptide 1 in type 2 diabetic patients. Toft-Nielsen, M. B. et al. Determinants of the impaired secretion of glucagon-like peptide-1 in type 2 diabetic patients. Ritzel, R. et al. Glucagon-like peptide 1 increases secretory burst mass of pulsatile insulin secretion in patients with type 2 diabetes and impaired glucose tolerance. Jones, I. R., Owens, D. R., Luzio, S., Williams, S. & Hayes, T. M. The glucose dependent insulinotropic polypeptide response to oral glucose and mixed meals is increased in patients with type 2 (non-insulin-dependent) diabetes mellitus. Vilsbøll, T., Krarup, T., Madsbad, S. & Holst, J. J. Defective amplification of the late phase insulin response to glucose by GIP in obese type II diabetic patients. Bagger, J. I. et al. Glucagonostatic potency of GLP-1 in patients with type 2 diabetes, patients with type 1 diabetes, and healthy control subjects. Effects of glucagon-like peptide 1 on counterregulatory hormone responses, cognitive functions, and insulin secretion during hyperinsulinemic, stepped hypoglycemic clamp experiments in healthy volunteers. Schuppelius, B. et al. Alterations in glucagon levels and the glucagon-to-insulin ratio in response to high dietary fat or protein intake in healthy lean adult twins: a post hoc analysis. Jumpertz von Schwartzenberg, R., Vazquez Arreola, E., Sandforth, A., Hanson, R. L. & Birkenfeld, A. L. Role of weight loss-induced prediabetes remission in the prevention of type 2 diabetes: time to improve diabetes prevention. Kahn, S. E. et al. Effect of semaglutide on regression and progression of glycemia in people with overweight or obesity but without diabetes in the SELECT trial. B. et al. Complex distribution, not absolute amount of adiponectin, correlates with thiazolidinedione-mediated improvement in insulin sensitivity. Riera-Guardia, N. & Rothenbacher, D. The effect of thiazolidinediones on adiponectin serum level: a meta-analysis. Doehner, W. et al. Inverse relation of body weight and weight change with mortality and morbidity in patients with type 2 diabetes and cardiovascular co-morbidity: an analysis of the PROactive study population. Vazquez Arreola et al. Prediabetes Remission and Cardiovascular Morbidity and Mortality: A post-hoc analysis from the Diabetes Prevention Program Outcome Study and the DaQing Diabetes Prevention Outcome Study. Lancet Diabetes Endocrinology; provisionally accepted. Addressing weight loss recidivism: a clinical focus on metabolic rate and the psychological aspects of obesity. Fildes, A. et al. Probability of an obese person attaining normal body weight: cohort study using electronic health records. Sandforth, L. et al. Subphenotype-dependent benefits of bariatric surgery for individuals at risk for type 2 diabetes. Taylor, R. et al. Remission of human type 2 diabetes requires decrease in liver and pancreas fat content but is dependent upon capacity for β cell recovery. Chadha, C. et al. Reproducibility of a prediabetes classification in a contemporary population. Lazo-Porras, M. et al. Regression from prediabetes to normal glucose levels is more frequent than progression towards diabetes: the CRONICAS cohort study. Birkenfeld, A. L., Franks, P. W. & Mohan, V. Precision medicine in people at risk for diabetes and atherosclerotic cardiovascular disease: a fresh perspective on prevention. & Frijters, J. E. A short questionnaire for the measurement of habitual physical activity in epidemiological studies. Reduction in the incidence of type 2 diabetes with lifestyle intervention or metformin. Fowler, S. Diabetes Prevention Program (DPP) (version 9). Mari, A., Pacini, G., Murphy, E., Ludvik, B. A model-based method for assessing insulin sensitivity from the oral glucose tolerance test. Matsuda, M. & DeFronzo, R. A. Insulin sensitivity indices obtained from oral glucose tolerance testing: comparison with the euglycemic insulin clamp. Importance of changes in adipose tissue insulin resistance to histological response during thiazolidinedione treatment of patients with nonalcoholic steatohepatitis. Improved quantification of muscle insulin sensitivity using oral glucose tolerance test data: the MISI calculator. Muscle and liver insulin resistance indexes derived from the oral glucose tolerance test. & Pacini, G. Insulinogenic indices from insulin and C-peptide: comparison of β-cell function from OGTT and IVGTT. Ahrén, B., Pacini, G., Foley, J. E. & Schweizer, A. Improved meal-related β-cell function and insulin sensitivity by the dipeptidyl peptidase-IV inhibitor vildagliptin in metformin-treated patients with type 2 diabetes over 1 year. Piccinini, F. & Bergman, R. N. The measurement of insulin clearance. Allison, D. B., Paultre, F., Maggio, C., Mezzitis, N. & Pi-Sunyer, F. X. The use of areas under curves in diabetes research. de Jesús Garduno-Garcia, J. et al. Older subjects with β-cell dysfunction have an accentuated incretin release. Fat distribution patterns and future type 2 diabetes. Follow-up whole-body assessment of adipose tissue compartments during a lifestyle intervention in a large cohort at increased risk for type 2 diabetes. Würslin, C. et al. Topography mapping of whole body adipose tissue using a fully automated and standardized procedure. Liu, C.-Y., McKenzie, C. A., Yu, H., Brittain, J. H. & Reeder, S. B. Fat quantification with IDEAL gradient echo imaging: correction of bias from T(1) and noise. Comparative MR study of hepatic fat quantification using single-voxel proton spectroscopy, two-point Dixon and three-point IDEAL. Machann, J. et al. Hepatic lipid accumulation in healthy subjects: a comparative study using spectral fat-selective MRI and volume-localized 1H-MR spectroscopy. A rapid method of total lipid extraction and purification. Simultaneous metabolite, protein, lipid extraction (SIMPLEX): a combinatorial multimolecular omics approach for systems biology. Herzog, R. et al. LipidXplorer: a software for consensual cross-platform lipidomics. Liu, M. Y. et al. Multiplexed analysis of biomarkers related to obesity and the metabolic syndrome in human plasma, using the Luminex-100 system. Rasmussen, C. et al. Preanalytical impact on the accuracy of measurements of glucagon, GLP-1 and GIP in clinical trials. The NHGRI–EBI GWAS catalog: knowledgebase and deposition resource. The authors would like to thank all participants who participated and are still participating in the respective studies. Furthermore, the authors would like to acknowledge the fundamental work of the technical and nursing staff, without whom this work would not have been possible. PLIS and this post hoc analysis were supported by the German Center for Diabetes Research, which is funded by the German Federal Ministry for Education and Research and the German states where its partner institutions are located (01GI0925). was supported by the Helmholtz Young Investigators Groups funding programme (Project ID: VH-NG-1619) and the Cluster of Excellence EXC 2124 (project 03.007). The study was additionally funded by a grant from the German Research Foundation (DFG; GRK2816 to A.L.B.). Part of the work was supported by the European Foundation for the Study of Diabetes Future Leader Award (NNF21SA0072746 to N.J.W.A. ), Independent Research Fund Denmark (1052-00003B, 10.46540/4308-00056B and 10.46540/4285-00131B to N.J.W.A.) was supported by the Deutsche Forschungsgemeinschaft (German Research Foundation (DFG))—CRG/TRR 412. are supported by the DFG Clinician Scientist Program ‘MINT-CS' of the Medical Faculty of the University of Tübingen. Additionally, this work was supported in part by the Intramural Research Program of the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK). Mass spectrometry work was supported by funds of ‘Bundesministerium für Bildung, Wissenschaft und Forschung' (BMBFW) within the project DigiOmics4Austria (https://bit.ly/44A30s1, to R.A.). Open access funding provided by Helmholtz Zentrum München - Deutsches Forschungszentrum für Gesundheit und Umwelt (GmbH). These authors contributed equally: Andreas L. Birkenfeld, Reiner Jumpertz von Schwartzenberg. German Center for Diabetes Research (DZD), Neuherberg, Germany Arvid Sandforth, Felicia Gerst, Estela Lorza-Gil, Yurong Cheng, Leontine Sandforth, Sarah Katzenstein, Marlene Ganslmeier, Jochen Seissler, Hans Hauner, Nikolaos Perakakis, Robert Wagner, Jürgen Machann, Fritz Schick, Andreas Peter, Rainer Lehmann, Cora Weigert, Jennifer Maurer, Hubert Preissl, Martin Heni, Julia Szendrödi, Stefan Kopf, Michele Solimena, Peter Schwarz, Matthias Blüher, Hans-Ulrich Häring, Martin Hrabé de Angelis, Annette Schürmann, Stefan Kabisch, Knut Mai, Andreas F. H. Pfeiffer, Stefan Bornstein, Michael Stumvoll, Michael Roden, Norbert Stefan, Andreas Fritsche, Andreas L. Birkenfeld & Reiner Jumpertz von Schwartzenberg Internal Medicine IV, Department of Diabetology, Endocrinology and Nephrology, University of Tübingen, Tübingen, Germany Arvid Sandforth, Felicia Gerst, Estela Lorza-Gil, Yurong Cheng, Leontine Sandforth, Sarah Katzenstein, Marlene Ganslmeier, Hubert Preissl, Hans-Ulrich Häring, Norbert Stefan, Andreas Fritsche, Andreas L. Birkenfeld & Reiner Jumpertz von Schwartzenberg Institute for Diabetes Research and Metabolic Diseases of the Helmholtz Center Munich at the University of Tübingen, Tübingen, Germany Arvid Sandforth, Felicia Gerst, Estela Lorza-Gil, Yurong Cheng, Leontine Sandforth, Sarah Katzenstein, Marlene Ganslmeier, Jürgen Machann, Fritz Schick, Andreas Peter, Rainer Lehmann, Cora Weigert, Jennifer Maurer, Hubert Preissl, Hans-Ulrich Häring, Norbert Stefan, Andreas Fritsche, Andreas L. Birkenfeld & Reiner Jumpertz von Schwartzenberg Elsa Vazquez Arreola & Robert L. Hanson The NovoNordisk Foundation Center for Basic Metabolic Research and the Department of Biomedical Sciences, University of Copenhagen, Copenhagen, Denmark Diabetes Center, Department of Medicine IV, University Hospital Ludwig-Maximilians University Munich, Munich, Germany Nikolaos Perakakis, Peter Schwarz & Stefan Bornstein Paul-Langerhans-Institut Dresden (PLID) of the Helmholtz Center Munich at University Clinic Carl Gustav Carus, Dresden, Germany Department of Endocrinology and Diabetology, Medical Faculty and University Hospital, Heinrich Heine University Düsseldorf, Düsseldorf, Germany Institute for Clinical Diabetology, German Diabetes Center, Leibniz Center for Diabetes Research at Heinrich Heine University Düsseldorf, Düsseldorf, Germany Institute for Clinical Chemistry and Pathobiochemistry, Department for Diagnostic Laboratory Medicine, University Hospital of Tübingen, Tübingen, Germany Andreas Peter, Rainer Lehmann, Cora Weigert, Jennifer Maurer & Martin Heni Department of Pharmacy and Biochemistry, Institute of Pharmaceutical Sciences, and Interfaculty Centre for Pharmacogenomics and Pharma Research, Auf der Morgenstelle 8, University of Tübingen, Tübingen, Germany Department of Medicine I and Clinical Chemistry, University Hospital of Heidelberg, Heidelberg, Germany Chair for Endocrinology and Diabetology at the University of Luebeck, Medizinische Klinik 1, University Hospital Schleswig-Holstein, Lübeck, Germany Helmholtz Institute for Metabolic, Obesity and Vascular Research (HI-MAG) of the Helmholtz Zentrum München, University of Leipzig and University Hospital Leipzig, Leipzig, Germany Institute of Experimental Genetics, IEG Helmholtz Center Munich, Neuherberg, Germany Department of Human Nutrition, German Institute of Human Nutrition Potsdam-Rehbrücke, Nuthetal, Germany Stefan Kabisch, Knut Mai & Andreas F. H. Pfeiffer Diabetes & Obesity Theme, School of Cardiovascular and Metabolic Medicine & Sciences, Faculty of Life Sciences & Medicine, Kings College London, Denmark Hill Campus, The James Black Centre, London, UK Cluster of Excellence EXC 2124 ‘Controlling Microbes to Fight Infections' (CMFI), University of Tübingen, Tübingen, Germany M3 Research Center for Malignome, Metabolome and Microbiome, Faculty of Medicine, University of Tübingen, Tübingen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and A.S. conceptualized this post hoc analysis. accessed and analysed the data. All authors edited the manuscript, interpreted the data, contributed to data acquisition, approved the final version, and agreed to submit it for publication. receives grants from the German Diabetes Association, Wilhelm Doerenkamp Foundation and California Walnut Commission, and lecture fees from Sanofi, Boehringer Ingelheim, Berlin Chemie, Lilly Deutschland and Juzo Akademie. receives lecture fees from Eli Lilly, Novo Nordisk and Sanofi, and participates in a Data Safety Monitoring Board and Advisory Board for Eli Lilly. receives lecture fees from Novo Nordisk and Gesellschaft für Wissenschaftstransfer Dresden, advisory board fees from Bayer Vital and holds patent WO2021/092265A1 for diagnosis and treatment of nonalcoholic fatty liver disease and liver fibrosis. is consulting with Novo Nordisk. has received research support and speaker fees from Mercodia, Novo Nordisk, Merck/MSD and Boehringer Ingelheim. All other authors declare no competing interests. Nature Medicine thanks Neda Rajamand Ekberg, Nathan R. Weeldreyer, Peng Wei and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Liam Messin, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) as post-hoc corrected comparisons derived from the respective mixed effects models. Assessment of markers of diet adherence and physical activity during and after the lifestyle intervention (LI) in PLIS. b, Daily distance walked inferred by pedometer data after 6 months of LI in a subgroup of individuals (R: n = 6; NR: n = 17). c–g, The habitual physical activity score during LI and at follow-up. h–l, Lean body mass during LI and at follow-up. Chi-squared test was used for a, all other p values are derived from two-sided Wilcoxon rank sum tests. Means of raw values are depicted and error bars indicate s.d. N for each panel is depicted in the respective group bars. Means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) and those directly above the panels comparing the group comparisons at each timepoint, all derived from post-hoc corrected comparisons from the respective mixed effects models. VLDL palmitate subclasses, R: n=47; NR: n=169 for all panels, in PLIS. Means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) as post-hoc corrected comparisons derived from the respective mixed effects models. Arrow base indicates coordinate at baseline, arrow tip indicates coordinate at 12 months for the responder group (blue) and the non-responder group (orange), respectively. Means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R versus NR over time (that is interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) as post-hoc corrected comparisons derived from the respective mixed effects models. Indexes of insulin sensitivity (a) and insulin secretion (b) in DPP as described in the methods. c, The hyperbolic relationship between insulin sensitivity and secretion. Arrow base indicates coordinate at baseline, arrow tip indicates coordinate at 12 months for the R group (blue) and the NR group (orange), respectively. In panel C, means of raw values are depicted with error bars indicating 95% confidence intervals. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) as post-hoc corrected comparisons derived from the respective mixed effects models. Exact p-values: a: NR, p = 2.48 × 10−7. Indexes of insulin sensitivity (a, b), insulin secretion (c) and beta cell function (d) in PLIS with response defined as return of 60min glucose during the OGTT below 155 mg/dL. P values directly above the x axes indicate change in R versus NR over time (that is, interaction term of group and time) derived from mixed effects models. P values directly above the colored bars indicate the difference between baseline and 12 months for the responder group (blue) and the non-responder group (orange) as post-hoc corrected comparisons derived from the respective mixed effects models. e, The hyperbolic relationship between insulin sensitivity and secretion. Arrow base indicates coordinate at baseline, arrow tip indicates coordinate at 12 months for the R group and the NR group, respectively. Means of raw values are depicted with error bars indicating 95% confidence intervals for a-d. In e, the base of the arrows depict medians at baseline and the respective tip after 1 year of LI. β coefficients for group (baseline) and group: time (6 months, 12 months) with 95% confidence intervals from linear mixed effects models reported above. β-coefficients are depicted as point estimates and 95% confidence intervals. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Prevention of type 2 diabetes through prediabetes remission without weight loss. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02346-z'>Mechanically liberating polarization bubbles in van der Waals ferroelectrics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 13:45:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Materials (2025)Cite this article Ferroelectric topological textures have sparked intensive interest, due to their exciting applications in a new era of non-volatile and ultrahigh-density information storage. However, these textures remain largely dependent on the given heterostructures with engineered neighbouring layers to balance the competing energies. Here we report high-density polarization bubbles in van der Waals ferroelectric crystals CuInP2S6, without the need for a spatially confined heterostructure. From piezoresponse force microscopy, it is observed that the formation and distribution of bubble domains exist in the inherent coexistence of polar phases. Crucially, the phase ratio can be facilely tailored by external stimuli such as mechanical force, enabling the labyrinth domains to be manipulated into high-density isolated bubbles through a mechanism involving polar phase competition and flexoelectricity, as revealed through density functional theory and phase-field modelling. Our findings not only provide insights into the creation of topological structures in a controlled manner but also demonstrate potential memory applications based on bubble domains in van der Waals ferroelectrics. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data generated during this study are included in the Article and its Supplementary Information. Source data are provided with this paper. Further data are available from the corresponding authors upon request. Das, S. et al. Observation of room-temperature polar skyrmions. Wang, Y. J. et al. Polar meron lattice in strained oxide ferroelectrics. Govinden, V. et al. Spherical ferroelectric solitons. Yadav, A. K. et al. Spatially resolved steady-state negative capacitance. Tan, C. et al. Engineering polar vortex from topologically trivial domain architecture. Li, Z. W. et al. High-density array of ferroelectric nanodots with robust and reversibly switchable topological domain states. Kim, K. et al. Configurable topological textures in strain graded ferroelectric nanoplates. Zhang, Q. et al. Nanoscale bubble domains and topological transitions in ultrathin ferroelectric films. Prokhorenko, S. et al. Motion and teleportation of polar bubbles in low-dimensional ferroelectrics. Zhang, Q. et al. Deterministic switching of ferroelectric bubble nanodomains. Han, L. et al. High-density switchable skyrmion-like polar nanodomains integrated on silicon. Bakaul, S. R. et al. Freestanding ferroelectric bubble domains. Ming, W. et al. Flexoelectric engineering of van der Waals ferroelectric CuInP2S6. Lun, Y. et al. Ultralow tip-force driven sizable-area domain manipulation through transverse flexoelectricity. Yang, X. et al. Highly reproducible van der Waals integration of two-dimensional electronics on the wafer scale. Lu, D. et al. Monolithic three-dimensional tier-by-tier integration via van der Waals lamination. Liu, Y. et al. Van der Waals integration before and beyond two-dimensional materials. Zhang, D. et al. Ferroelectric order in van der Waals layered materials. Wang, C. et al. Towards two-dimensional van der Waals ferroelectrics. A. et al. Tunable quadruple-well ferroelectric van der Waals crystals. Dawei, Z. et al. Anisotropic ion migration and electronic conduction in van der Waals ferroelectric CuInP2S6. Jiang, X. et al. Manipulation of current rectification in van der Waals ferroionic CuInP2S6. Zhong, Z. et al. Robust threshold-switching behavior assisted by Cu migration in a ferroionic CuInP2S6 heterostructure. Zhu, H. et al. Highly tunable lateral homojunction formed in two-dimensional layered CuInP2S6 via in-plane ionic migration. You, L. et al. Origin of giant negative piezoelectricity in a layered van der Waals ferroelectric. Qi, Y. et al. Widespread negative longitudinal piezoelectric responses in ferroelectric crystals with layered structures. Deng, J. et al. Thickness-dependent in-plane polarization and structural phase transition in van der Waals ferroelectric CuInP2S6. Liu, F. et al. Room-temperature ferroelectricity in CuInP2S6 ultrathin flakes. Ding, B. et al. Observation of magnetic skyrmion bubbles in a van der Waals ferromagnet Fe3GeTe2. Zhang, Y. et al. Ferroelectric polarization-enhanced performance of flexible CuInP2S6 piezoelectric nanogenerator for biomechanical energy harvesting and voice recognition applications. Olaniyan, I. et al. Switchable topological polar states in epitaxial BaTiO3 nanoislands on silicon. Lukyanchuk, I. et al. Hopfions emerge in ferroelectrics. Lukyanchuk, I. et al. Topological foundations of ferroelectricity. Tikhonov, Y. et al. Controllable skyrmion chirality in ferroelectrics. Govinden, V. et al. Stability of ferroelectric bubble domains. Steffes, J. J. et al. Thickness scaling of ferroelectricity in BiFeO3 by tomographic atomic force microscopy. Pereira Gonçalves, M. A. et al. Theoretical guidelines to create and tune electric skyrmion bubbles. Lu, L. et al. Topological defects with distinct dipole configurations in PbTiO3/SrTiO3 multilayer films. Mizzi, C. A. et al. Does flexoelectricity drive triboelectricity? Park, S. M. et al. Selective control of multiple ferroelectric switching pathways using a trailing flexoelectric field. Lin, S. Q. et al. Electron transfer driven by tip-induced flexoelectricity in contact electrification. Zhang, Y. X. et al. Strain manipulation of ferroelectric skyrmion bubbles in a freestanding PbTiO3 film: a phase field simulation. Junquera, J. et al. Topological phases in polar oxide nanostructures. Velický, M. et al. Mechanism of gold-assisted exfoliation of centimeter-sized transition-metal dichalcogenide monolayers. Kresse, G. et al. Ab initio molecular dynamics for liquid metals. Kresse, G. et al. Efficiency of ab-initio total energy calculations for metals and semiconductors using a plane-wave basis set. Kresse, G. et al. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Gonze, X. Adiabatic density-functional perturbation theory. Wang, L. et al. Flexoelectronics of centrosymmetric semiconductors. Zhang, C. et al. Intrinsic ultrahigh negative Poisson's ratio in two-dimensional ferroelectric ABP2X6 materials. Liang, C. B. et al. Measurement of the flexoelectric coefficients in van der Waals materials with separation of piezoelectricity. Download references We thank S. Dong for the fruitful discussion. This work is supported by the National Key Research and Development Program of China grant numbers 2021YFA1400300 (J.H.) and 2019YFA0307900 (Xueyun Wang); National Natural Science Foundation of China grant numbers 12374080 (Xueyun Wang), 12202056 (X.J. and 92163101 (Xueyun Wang); the Ningbo Yongjiang Talent Introduction Programme grant number 2023A-390-G (X.J. ); and Ningbo Top Talent Project grant number 2020-DST-003 (W.Y. These authors contributed equally: Xingan Jiang, Tingjun Wang, Yixuan Zhang, Zunyi Deng. School of Aerospace Engineering & State Key Laboratory of Environment Characteristics and Effects for Near-space, Beijing Institute of Technology, Beijing, China Xingan Jiang, Tingjun Wang, Yixuan Zhang, Zunyi Deng, Xiangping Zhang, Jiaqian Kang, Xueyun Wang & Jiawang Hong Institute of Micro/Nano Materials and Devices, Ningbo University of Technology, Ningbo, China Xingan Jiang, Xiangdong Yang & Weiyou Yang State Key Laboratory for Mesoscopic Physics and Frontiers Science Center for Nano-optoelectronics, School of Physics, Peking University, Beijing, China Ruixue Zhu & Peng Gao Engineering Research Center for Semiconductor Integrated Technology, Institute of Semiconductors, Chinese Academy of Sciences, Beijing, China School of Physics and Optoelectronic Engineering, Beijing University of Technology, Beijing, China Xiaolei Wang School of Materials Science and Engineering & Advanced Research, Institute of Multidisciplinary Science, Beijing Institute of Technology, Beijing, China Hunan Key Laboratory of Two-Dimensional Materials, State Key Laboratory for Chemo/Biosensing and Chemometrics, College of Chemistry and Chemical Engineering, Hunan University, Changsha, China Department of Physics and Astronomy, Rutgers University, Piscataway, NJ, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Xueyun Wang and J.H. conceived the idea and directed the project. prepared the samples with the help of J.K. X.J. performed the PFM measurements and data analyses under the supervision of W.Y., Xueyun Wang and J.H. performed the phase-field modelling and data analysis under the supervision of H.H. conducted the DFT calculations under the supervision of J.H. performed the analyses of strain field and flexoelectricity under the scanning tip. X.J., W.Y., Xueyun Wang and J.H. co-wrote the manuscript. contributed to PFM data discussion. and S.-W.C. contributed to the polarization reconstruction analysis. and Xiaolei Wang contributed to the data discussion and interpretation. Correspondence to Xueyun Wang, Weiyou Yang or Jiawang Hong. The authors declare no competing interests. Nature Materials thanks Igor Lukyanchuk and Nagarajan Valanoor for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. There exists a polar phase with weak amplitude surrounding the polarization bubbles in the VPFM image. a, The diagram of PFM scans across ferroelectric domain with Ising type wall and Néel type wall. b, The amplitude response for the typical case. A clear domain wall can be clearly visible in the amplitude image for Ising domain. The Néel-type bubble domain (region 2) typically exhibit weaker amplitude compared to the surrounding regions (region 1 and 3), making domain wall less visible in the amplitude image. c, The amplitude response for our case in the CuInP2S6 ferroelectrics. The Néel-type bubble domains are surrounded by weak-amplitude region. The comparable amplitude intensity between the bubble domain (region 2) and surrounding regions (regions 1 and 3) makes the domain walls more easily observable. a, High-density elongated labyrinth domains. In some local region indicated by square boxes, the elongated labyrinth domains have evolved into isolated bubbles. b, High-density bubble domains. The elongated labyrinth domains are the precursor of bubbles as revealed by the weak phase signal (yellow) surrounding bubbles with labyrinth pattern. c, The zoomed-in image corresponding to the local area marked in panel a. d, The schematic evolution of elongated labyrinth domains disrupted into isolated bubbles. The phase and amplitude images for vertical PFM (VPFM) and lateral PFM (LPFM) measured at four sample rotation angles of 0°, 30°, 60° and 90°. The LPFM results of the bubble domains clearly reveal tilted domain walls. For instance, the bubble marked by white circle exhibits an S-shaped form in its LPFM domain walls, as labeled by the white dashed circles. The scale bar is 200 nm. a, The schematic of angle-resolved LPFM. The in-plane (IP) PFM was measured as a function of tip angle φ, with amplitude (A) and phase delay (θ) determined by fitting Acos(φ-θ-π/2). A and θ represent the polarization vector's length and angle relative to the defined direction, respectively. Numbers 1-4 denote sample rotation angles (0°, 30°, 60°, 90°). The piezoresponse vector field is constructed from IP components (Acosθ, Asinθ) as a function of position. b, Top view of reconstructed IP polarization field P, superimposed with color maps of IP piezoresponse vector map (θ). The distribution of polarization field demonstrates the center-divergence of IP polarization component. c, The trigonometric curve fittings for four representative points denoted as purple spots in panel b. d, The IP polarization field. e, Top view of 3D polarization field P, superimposed with color maps of the IP-phase (0°, 90° rotation) from the LPFM and out-of-plane (OP) phase (VPFM). Source data a, VPFM phase images. b, VPFM amplitude images. c, The schematic diagram for the distribution characteristics of bubble domains. Bubble domains tend to be densely distributed in the weak-amplitude region (HP phase) or near the boundaries of strong/weak amplitude region (LP/HP phase boundary). Notably, it can be observed that the weak-amplitude region with varied patterns significantly influences the bubble distribution. a-b, PFM phase and amplitude images before (a) and after (b) the force scanning of ~2.8 μN using an AFM tip. The tip force leads to the formation of HP phase with weak piezo-amplitude and bubbles also tend to densely form in the weak-amplitude HP region, or near the boundaries of strong/weak-amplitude region (LP/HP phase boundary). a, 3D PFM phase and amplitude image overlaid with topography for the region with the high-density elongated labyrinth domains. b, The corresponding 3D PFM phase and amplitude image after applying a force scanning of 2.8 μN. After the force scanning, weak-amplitude LP and strong-amplitude HP phases become spatially separated, with high-density bubbles formed in HP regions. The scanning area is 3 μm×3 μm. a, The simulated high-density polarization bubbles. A bubble is circled in red for analyzing the charge concentration (ρₑ), internal field (Ez), and polarization configuration. The simulation dimensions are 2040 nm×2040 nm×120 nm. b, The OP electric field of the top surface for the red-circled bubble in panel a. c, Bound charge density of selected bubble domain in panel a. d, Cross-sectional polarization view for the circled bubble along the A-B line in panel a. The blue (red) center depicts counter-clockwise (clockwise) vortex domain wall center. e, OP polarization component obtained along the blue line marked in panel d. Source data a-b, The phase and amplitude after applying different pulse tip voltages (+60 V, +90 V and -90 V) at the central point of the scanning region (indicated by the white arrow) using the conductive tip, respectively. The bubble domains with net upward polarization (net Pup) and downward polarization (Pdown) are circled in white and yellow, respectively. c, The schematic polarization configuration and amplitude response of bubble before and after applying pulse voltage, respectively. d, The distinct conductive differences between the bubble domains with net Pup and Pdown. e, Large areas of conductive bubble domains. 1–3, Notes 1–3 and Table 1. Crystallographic data for the LP phase. Crystallographic data for the HP phase. Bubbles size distribution data plotted in Fig. Reconstructed polarization data plotted in Fig. DFT data plotted in Fig. Domain percentage data plotted in Fig. DFT data plotted in Fig. 5b, flexoelectric field data plotted in Fig. 5c and phase-field simulation data plotted in Fig. Schematic of the LPFM data plotted in Extended Data Fig. 5a and polarization reconstruction data plotted in Extended Data Fig. Phase-field simulation data plotted in Extended Data Fig. 9b,c,e. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Jiang, X., Wang, T., Zhang, Y. et al. Mechanically liberating polarization bubbles in van der Waals ferroelectrics. Accepted: 08 August 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Materials © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63219-2'>Solving the many-electron Schrödinger equation with a transformer-based framework</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 13:37:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Accurately solving the Schrödinger equation for intricate systems remains a prominent challenge in physical sciences. Here we present QiankunNet, a neural network quantum state (NNQS) framework that combines Transformer architectures with efficient autoregressive sampling to solve the many-electron Schrödinger equation. At its core is a Transformer-based wave function ansatz that captures complex quantum correlations through attention mechanisms, effectively learning the structure of many-body states. The quantum state sampling employs layer-wise Monte Carlo tree search (MCTS) that naturally enforces electron number conservation while exploring orbital configurations. The framework incorporates physics-informed initialization using truncated configuration interaction solutions, providing principled starting points for variational optimization. Our systematic benchmarks demonstrate QiankunNet's versatility across different chemical systems. For molecular systems up to 30 spin orbitals, we achieved correlation energies reaching 99.9% of the full configuration interaction (FCI) benchmark, setting a new standard for neural network quantum states. Most notably, in treating the Fenton reaction mechanism, a fundamental process in biological oxidative stress, QiankunNet successfully handled a large CAS(46e,26o) active space, enabling accurate description of the complex electronic structure evolution during Fe(II) to Fe(III) oxidation. Recently, the Transformer architecture has revolutionized the field of natural language processing (NLP), giving rise to large language models (LLMs) with unprecedented capabilities1,2,3,4,5. Intrinsic versatility and adaptability of the Transformer architecture serve as the backbone of these LLMs, making it a powerful tool for addressing complex challenges, and its applications are extending far beyond language processing, permeating various domains and revolutionizing numerous fields. Particularly, deploying this architecture within the scientific field harbors the potential to propel us towards untapped frontiers of innovation and discovery. While the Transformer has demonstrated exceptional performance in tasks like image recognition6,7, protein representation and protein design8,9,10, and global weather forecasting11, and has shown promise in various quantum physics applications12,13,14, its potential for solving molecular Schrödinger equations through efficient autoregressive sampling and systematic application to complex chemical systems remains underexplored. Therefore, it is both fundamentally intriguing and practically important to explore the design of efficient Transformer-based architectures specifically tailored for solving the Schrödinger equation. Such architectures are expected to unlock new levels of understanding and advancements in quantum chemistry. In principle, the electronic structure and properties of all materials can be determined by solving the Schrödinger equation to obtain the wave function. However, in practice, it is a significant challenge to find a general approach to reduce the exponential complexity of the many-body wave function and extract its essential features. Various methods have been developed to solve the Schrödinger equation for realistic systems. Considering the fermionic nature of electrons, Slater determinants are used to represent the electron systems, ensuring that the wave function obeys exchange antisymmetric symmetries. To approximate the exact energy, several strategies have been devised, including perturbation theories15,16, the truncated configuration interaction which takes into account arbitrary linear combinations of excitations up to a certain order17, the coupled-cluster (CC) method which takes into account certain nonlinear combinations of excitations up to a certain order (e.g., CCSD, CCSD(T))18, the density matrix renormalization group (DMRG) algorithm19,20 which uses the one-dimensional matrix product state wave function ansatz, or the variational Monte Carlo (VMC) method21,22,23 which works for any wave function ansatz. However, these methods can fail in numerous cases, mostly due to the limited expressive power of the wave function ansatz. In 2017, Carleo and Troyer proposed their seminal work on the neural network quantum state (NNQS) algorithm, which introduced a groundbreaking approach for tackling many-spin systems within the exponentially large Hilbert space24. The main idea behind NNQS is to parameterize the quantum wave function with a neural network and optimize its parameters stochastically using the VMC algorithm. They also demonstrated that the neural network ansatz, when dealing with many-body quantum states, is more expressive than the tensor network states25,26,27,28,29, and its computational cost typically scales polynomially30. Moreover, as a general feature of Monte Carlo methods, large-scale parallelization can provide notable performance advantages. The NNQS framework has evolved along two distinct paths: first quantization14,30,31, which works directly in continuous space, and second quantization12,13,32,33,34,35,36,37,38,39,40,41,42, which operates in a discrete basis. While first quantization methods naturally incorporate the complete basis set limit, they face challenges in sampling efficiency, particularly for larger systems where Markov chain sampling can become ineffective. Second quantization methods, though better at enforcing symmetries and boundary conditions, encounter scalability limitations due to the rapid growth of computational costs with system size. Current implementations face two major challenges: the computational burden of energy evaluation, which scales as the fourth power of the number of spin orbitals, and the increasing complexity of neural network architectures required for larger systems32,33. Molecular systems present additional challenges due to their complex electronic correlation patterns. This complexity demands highly expressive models that can be efficiently scaled to capture intricate quantum effects while remaining computationally tractable. The development of such models, capable of balancing expressiveness with computational efficiency, remains a central challenge in the field. In this work, we present QiankunNet (Qiankun meaning “heaven and earth” in Chinese), a NNQS framework that combines the expressivity of Transformer architectures with efficient autoregressive sampling. At the heart of our approach lies a Transformer-based wave function ansatz that captures complex quantum correlations through its attention mechanism, effectively learning the structure of many-body states while maintaining parameter efficiency independent of system size, as illustrated in Fig. Building upon the autoregressive sampling approach introduced in the Neural Autoregressive Quantum States (NAQS) method by Barrett et al.33, we develop an efficient batched implementation that reformulates quantum state sampling as a tree-structured generation process. While NAQS demonstrated autoregressive sampling for quantum states, our implementation specifically leverages the Transformer architecture's parallel processing capabilities and introduces several key innovations: we adopt a Monte Carlo Tree Search (MCTS)-based autoregressive sampling approach that introduces a hybrid breadth-first/depth-first search (BFS/DFS) strategy. This provides more sophisticated control over the sampling process through a tunable parameter that allows adjustment of the balance between exploration breadth and depth. Specifically, our approach first uses BFS to accumulate a certain number of samples, then performs batch-wise DFS sampling. This strategy significantly reduces memory usage while enabling computation of larger and deeper quantum systems by managing the exponential growth of the sampling tree more efficiently. We also implement explicit multi-process parallelization for distributed sampling, going beyond the single-process GPU parallelization used in NAQS. This enables our method to partition unique sample generation across multiple processes, significantly improving scalability for large quantum systems. Moreover, our implementation incorporates key-value (KV) caching specifically designed for Transformer-based architectures. While NAQS uses MLP-based models that don't benefit from such caching, our Transformer-based approach achieves substantial speedups by avoiding redundant computations of attention keys and values during the autoregressive generation process. This approach eliminates the need for Markov Chain Monte Carlo methods, allowing direct generation of uncorrelated electron configurations. The framework is further enhanced by our physics-informed initialization scheme that incorporates truncated configuration interaction solutions, providing a principled starting point for variational optimization and significantly accelerates convergence. The framework's efficiency stems from our parallel implementation of local energy evaluation, utilizing a compressed Hamiltonian representation that significantly reduces memory requirements and computational cost. The sampling implementation employs an efficient pruning mechanism based on electron number conservation, substantially reducing the sampling space while maintaining physical validity. This combination of architectural innovations and computational optimizations enables QiankunNet to achieve both high accuracy and robust convergence. Our systematic benchmarks demonstrate unprecedented accuracy across diverse chemical systems: for molecular systems up to 30 spin orbitals, we achieve correlation energies reaching 99.9% of the FCI benchmark; most notably, in treating the Fenton reaction mechanism, QiankunNet successfully handles a large CAS(46e,26o) active space, enabling accurate description of complex transition metal electronic structure. These advances, from weak intermolecular forces to strongly correlated transition metal systems, demonstrate QiankunNet's broad applicability and pave the way for accurate quantum chemical calculations of previously intractable large-scale molecular systems. The remainder of this paper is organized as follows: we first present the theoretical framework of QiankunNet, followed by detailed benchmarks on various molecular systems, and conclude with discussions on future applications and potential improvements. b Training workflow of the variational Monte Carlo (VMC) optimization. Starting from physics-informed initialization, the model parameters are updated through gradient descent to minimize the energy expectation value. The process involves autoregressive sampling of electron configurations, local energy calculation, and parameter updates via backpropagation. To validate the effectiveness of QiankunNet, we compute the ground state energies of several molecules with an atomic basis set and compare them to results of other methods, such as Hartree-Fock (HF) and coupled cluster with up to double excitations (CCSD). Using a finite basis set, we can express the molecular Hamiltonian in the second quantized form: Through the Jordan-Wigner transformation, this electronic Hamiltonian can be mapped to a spin Hamiltonian: We also report results from NNQS methods using other neural-network ansatz (NAQS and MADE). As demonstrated in Table 1, QiankunNet achieves chemical accuracy in correlation energies, recovering 99.9% of the FCI ground-truth values across a benchmark set of 16 molecules. The method demonstrates encouraging performance when computing full potential energy surfaces, as evidenced by the C2 and N2 calculations with STO-3G basis set (Fig. Notably, QiankunNet captures the correct qualitative behavior in regions where standard CCSD and CCSD(T) methods show limitations, particularly at dissociation distances where multi-reference character becomes significant. While more advanced methods like DMRG would provide more rigorous benchmarks for strongly correlated systems, our comparison with CCSD and CCSD(T) illustrates QiankunNet's potential for handling challenging electronic structures. A comprehensive analysis of the model's performance with respect to its hyperparameters and pre-training strategies is provided in the Supplementary Information (Fig. Comparison of the energies obtained using QiankunNet and other traditional quantum chemistry approaches for a C2 and b N2, as a function of the nuclear separation with STO-3G basis set. QiankunNet outperforms all other approximation techniques. It agrees with full configuration interaction (FCI) results well even at structures where both coupled-cluster approaches (CCSD,CCSD(T)) failed due to the presence of strong correlations. FCI represents the exact solution within a given basis set. When comparing with other second-quantized NNQS approaches, the Transformer-based neural network adopted in QiankunNet can exhibit heightened accuracy. For example, the second quantized approaches, such as MADE method, cannot achieve the chemical accuracy for N2 system, while QiankunNet can achieve an accuracy two orders of magnitude higher. Another advanced second quantized method, NAQS, employs a multilayer perceptron (MLP) augmented with hard-coded pre- and post-processing steps to maintain the autoregressive property. However, the complexity of NAQS's neural network scales with the number of spin orbitals, leading to a dramatic increase in computational demands and memory usage as the system size grows. For instance, for a C2H4O molecule with 38 spin orbitals, the GPU VRAM requirement soars to an unsustainable 454 GB, far surpassing the 80GB memory capacity of an A100 GPU. Conversely, QiankunNet incorporates a decoder-only Transformer, making the number of network parameters independent of the number of spin orbitals (our model can potentially be simultaneously used for systems with different numbers of spin orbitals), as a result, QiankunNet demonstrates a more favorable time scaling compared to NAQS, for systems with fewer than 24 spin orbitals, NAQS calculation is faster. However, when assessing a system with 30 spin orbitals, like Li2O, QiankunNet boasts a computational speed 10 times faster than NAQS, and the calculations with QiankunNet can be extended to chemical systems up to 92 spin-orbitals, as illustrated in Fig. Beyond speed, our QiankunNet also exhibits higher accuracy. For instance, in the Li2O case, while NAQS captures 98.1% of the electron correlation energy, QiankunNet recovers an impressive 99.9% of the electron correlation energy. We also perform the calculation for the C2H4O molecule comprising 38 spin orbitals, and our method yields an energy of −151.1228 Ha, which is 2.32 mHa lower than the MADE approach (−151.12048 Ha)34. A detailed analysis of QiankunNet against other state-of-the-art Neural NNQS methods is presented in the Supplementary Information (Sec. We have also included results for the N2 molecule with bond length 2.118 Bohr in the cc-pVDZ basis set, which represents a significant computational challenge beyond the reach of conventional FCI methods. The N2/cc-pVDZ system (56 qubits, 14 electrons) contains 1.402 × 1012 configurations, making it intractable for FCI calculations. To our knowledge, no existing NNQS approaches have successfully tackled molecular systems of this scale in the full cc-pVDZ basis without approximations. Our method achieves a ground-state energy of −109.2788 Ha, which differs from the best DMRG result (−109.2821 Ha)43 by only 3.3 mHa. This represents the first demonstration of an NNQS method successfully handling a molecular system with over 1012 configurations, showcasing the method's capability beyond small, trivially-solvable systems. It is worth noting that our current calculation employs RHF orbitals as the single-particle basis. As demonstrated in quantum Monte Carlo studies44, using UHF orbitals typically leads to lower variational energies than using RHF orbitals. We expect that adopting UHF orbitals would further improve our energy and bring it even closer to the DMRG benchmark. However, even with RHF orbitals, our method achieves an accuracy within chemical precision (3.3 mHa), demonstrating the robustness of our approach across different orbital choices. The measurements here refer to the wall-clock time during which these two methods perform calculations until convergence is achieved. All runtime measurements were obtained with exclusive access to computational resources to ensure accurate benchmarking. Experiments were conducted on one NVIDIA A100 GPU. Building upon these promising results, we now turn our attention to more complex reactions. The Fenton reaction, a fundamental process involving Fe(II) oxidation by hydrogen peroxide, plays a crucial role in biological oxidative stress and has significant implications in cellular damage mechanisms. Since its discovery in 189445, this reaction has been the subject of intense mechanistic debate46,47,48. While the mechanistic details remain controversial, here we focus on examining the electronic structure of its key step using our NNQS approach. Specifically, we study the homolytic dissociation of H2O2 catalyzed by aqueous Fe(II) at low pH, which generates OH• radicals and Fe(III) species: This octahedral complex, formed by substituting one water molecule in [Fe(H2O)6]2+ with hydrogen peroxide, serves as a model system for studying the aqueous ferrous ion's reactivity. Our quantum simulation employs a CAS(46e,26o) active space with cc-pVTZ basis sets, where the active orbitals are selected using the automated AVAS (Automated Virtual Active Space) method49 (computational details are provided in Supplementary Information, Sec. This represents a significant expansion compared to previous studies 49 that employed a CAS(20e,13o) active space targeting only O2s+2p and Fe 3d electron shells. 4, our calculations reveal a higher transition state barrier (83 kcal/mol compared with 65 kcal/mol in previous work). The combination of a large basis set (cc-pVTZ) and extensive active space (46e,26o) enables us to accurately capture the electronic structure evolution during Fe-O bond cleavage. This success is particularly significant for transition metal chemistry, where the multi-reference character of electronic structures often challenges traditional methods. However, for a more complete description of this system, several additional factors need to be considered. While we use the relatively large cc-pVTZ basis set, even larger basis sets might be necessary for achieving higher accuracy in transition metal systems. Additionally, relativistic effects can significantly influence the electronic structure of transition metal complexes. Furthermore, given that this reaction occurs in solution, the inclusion of solvation effects is particularly important for obtaining quantitatively accurate energetics. These factors, while beyond the scope of our current study focusing on the electronic structure of the gas phase reaction, are crucial considerations for future investigations aiming at a comprehensive understanding of the Fenton reaction mechanism. The blue curve represents our work using an expanded (46e,26o) active space, while the green dashed curve shows previous results from ref. The reaction pathway exhibits three distinct stages: initial formation of [Fe(H2O)5(H2O2)]2+ through H2O2 coordination to the octahedral [Fe(H2O)6]2+ complex (left molecular structure); transition state featuring O-O bond elongation (middle); and product state comprising an OH• radical and [Fe(OH)(H2O)5]2+ complex (right). Our larger active space calculations reveal a higher energy barrier (83 kcal/mol, compared with 65 kcal/mol) and more pronounced energy differences along the reaction pathway, particularly in the transition and product regions. Electronic structure calculations were performed incorporating Fe 3d and O 2s+2p shells with cc-pVTZ basis sets. In this study, we introduce QiankunNet, which adapts the Transformer architecture to solve the many-electron Schrödinger equation. This marks a significant advancement in applying language model architectures to understand quantum electronic structures—the fundamental language of microscopic nature. Just as Transformer models have revolutionized NLP and protein structure prediction, we demonstrate their potential in capturing the intricate patterns of electron wave functions. We acknowledge the pioneering work of Psiformer14 in applying attention mechanisms to electron–electron interactions. Through first-quantization approaches and complete variational treatment of basis sets, Psiformer achieved impressive absolute energies. Our approach complements this development by incorporating active space methods, allowing us to focus on chemically relevant orbitals and directly apply physical and chemical insights. This proves particularly advantageous when dealing with complex electronic structures and weak interactions. QiankunNet's distinctive feature lies in its decoder-only Transformer architecture combined with MCTS autoregressive sampling. Unlike traditional Markov Chain Monte Carlo methods that require sample rejection and suffer from correlation issues, our autoregressive approach generates independent samples directly. The network learns during both sampling and inference phases, enhancing efficiency through parallel batch processing. This design makes QiankunNet particularly well-suited for GPU acceleration and large-scale applications. This study establishes a fundamental connection between language modeling architectures and quantum electronic structure calculations. By adapting Transformer models to solve the many-electron Schrödinger equation, we demonstrate their capability in capturing complex quantum mechanical phenomena. The success of neural network wavefunctions in ab initio calculations, exemplified by approaches like Psiformer and QiankunNet, suggests a promising direction for quantum chemistry. These methods combine high accuracy, scalability, and computational efficiency, offering new possibilities for electronic structure calculations of complex molecular systems. The integration of physical insights through active space methods and direct sampling techniques further enhances their practical utility. We anticipate that such neural network-based approaches will become increasingly important tools in quantum chemistry, complementing existing methods in tackling challenging molecular systems. These variables might represent various physical quantities such as spin state or particle occupation numbers. The challenge lies in describing the many-body wave function, which establishes a sophisticated mapping between the configuration space and a vast landscape of complex numbers. This mapping encodes both magnitude and phase information of the quantum state, essentially serving as a transformation engine that converts a given many-body configuration x into its corresponding quantum mechanical description Ψ(x). By leveraging artificial neural networks, we aim to construct an efficient approximation of the wave function's behavior. This neural network approach effectively creates a trainable computational framework that learns to replicate the wave function's response to different input configurations. Among the various neural architectures available in modern machine learning, we focus our investigation on the Transformer framework, particularly in its application to quantum many-body systems. The Transformer architecture implements a sophisticated attention-based structure: an embedding layer that maps the physical configurations into a high-dimensional representation space, followed by multiple self-attention layers that capture the intricate correlations between quantum variables. The architecture is particularly well-suited for quantum systems due to its ability to model long-range interactions and complex quantum correlations through its multi-head attention mechanism. The connection between NLP and quantum chemistry represents a substantive interdisciplinary bridge rather than a superficial analogy. By establishing a formal mathematical isomorphism between configuration strings in quantum systems and sentences in language models, we address the shared challenge of exponential complexity that has long hindered progress in both fields. Specifically, the Hilbert space dimension of 2N for N spin orbitals parallels the exponential growth of possible sentences in natural language, creating a compelling theoretical foundation for methodological transfer. In our implementation, Transformer architectures represent quantum wave functions with quantum bitstrings serving as inputs to predict the wave function. Particularly noteworthy is how the autoregressive property of Transformers enables efficient modeling of conditional probabilities in quantum states, mirroring their function in language prediction. Our QiankunNet consists of two complementary sub-networks: the amplitude sub-network and the phase sub-network. This dual-network architecture effectively captures both the magnitude and phase information necessary to fully describe a quantum state: where the amplitude sub-network is constructed using a Transformer decoder to represent the probability ∣Ψ(x)∣2, while the phase sub-network ϕ(x) employs a multi-layer perceptron (MLP) as shown in Fig. The Transformer architecture1,5 is built from layers, each containing a self-attention component composed of multiple independent attention heads. These heads operate in parallel, each with its own set of parameters, allowing the network to simultaneously learn different types of correlations. Within each head, the attention mechanism works by computing how relevant each input position is to every other position, creating a rich network of relationships within the quantum state configuration. The processing of quantum states through our Transformer network follows a carefully designed sequence of steps. This augmented input then undergoes an embedding process, where each binary value is transformed into a rich, high-dimensional representation that captures the quantum mechanical properties of the system. The network takes as input a token/qubit vector x = {x1, x2, . , xn} of length n, which is transformed into embeddings: Position information plays a crucial role in quantum states, so we incorporate learned positional encodings for each site in the configuration. The embedded and position-encoded information then flows through multiple identical Transformer layers. Each layer in our architecture processes the quantum state information through several stages. The masked multi-head attention mechanism allows the network to learn complex correlations while maintaining the autoregressive property—ensuring that each position only influences those that come after it. For each decoder layer l = 1, 2, …, L, the following operations are performed: The masked multi-head self-attention mechanism computes: where \(Q=P{E}^{(l-1)}{W}_{Q}^{(l)}\), \(K=P{E}^{(l-1)}{W}_{K}^{(l)}\), and \(V=P{E}^{(l-1)}{W}_{V}^{(l)}\) are the queries, keys, and values linearly transformed from the positionally encoded embeddings PE(l−1), M is the mask ensuring autoregressive property, and dk is the dimension of the keys. Residual connections and layer normalization are applied: where Sublayer(l)(PE(l−1)) represents the output of the masked multi-head self-attention sublayer for layer l. Position-wise feed-forward networks with GeLU activation function process the information: where \({W}_{1}^{(l)}\in {{\mathbb{R}}}^{{d}_{{{{\rm{model}}}}}\times {d}_{{{{\rm{ff}}}}}}\) and \({W}_{2}^{(l)}\in {{\mathbb{R}}}^{{d}_{{{{\rm{ff}}}}}\times {d}_{{{{\rm{model}}}}}}\) are weight matrices, \({b}_{1}^{(l)}\in {{\mathbb{R}}}^{{d}_{{{{\rm{ff}}}}}}\) and \({b}_{2}^{(l)}\in {{\mathbb{R}}}^{{d}_{{{{\rm{model}}}}}}\) are bias vectors, and GeLU is the Gaussian Error Linear Unit activation function. which serves as input to the next layer or as the final output if l = L. After processing through L layers, the Transformer decoder's final output undergoes linear transformation and softmax activation to produce the conditional probability distribution: The probability of a sequence x = (x1, x2, …, xn) can be factorized as a product of conditional probabilities: expressing the joint probability P(x) of the sequence as the product of conditional probabilities of each element xi given its preceding elements. The phase ϕ(x) is represented by a Multi-Layer Perceptron network that processes the input sequence x and outputs a real-valued phase, effectively encoding the complex quantum correlations present in the electronic wavefunction. Monte Carlo sampling is used to generate samples {x1, x2, …, xN} for quantum states. MCMC is a class of algorithms based on Markov chains for random sample generation, where the production of each state depends only on the previous state. A classic example of MCMC is the Metropolis-Hastings algorithm, which uses an acceptance-rejection mechanism to decide whether to move to a new state. It can be seen that a Markov chain that gradually approximates the target distribution \({p}_{\overrightarrow{\theta }}({{{\bf{x}}}})\) is constructed, so that the distribution of states on the chain eventually matches the target distribution. With a Metropolis sampling scheme, Choo et al.32 observed that the performance was highly dependent on the number of samples used, but this was constrained to a maximum batch size of Ns = 106. For VMC-based quantum chemistry, a more efficient scheme called autoregressive sampling can be introduced, which generates independent, uncorrelated samples directly from the target distribution. Rather than trying to model the distribution over every possible configuration of these discrete variables simultaneously, autoregressive sampling builds the distribution sequentially. It takes advantage of the fact that the probability of a specific configuration can be decomposed into the product of conditional probabilities. This approach allows for sampling one variable at a time, given the values of the previously sampled variables, thus reducing the complexity and computational burden significantly. Autoregressive sampling algorithm generates probability distributions p(x) of the special form The autoregressive sampling scheme allows for the generation of exact samples without the need for a pre-thermalization step or the discarding of intermediate samples (which is necessary in Monte Carlo (MC) sampling to avoid autocorrelations). This makes autoregressive neural networks generally more efficient than MC sampling. 5, the generation of a single quantum state configuration uses QiankunNet's sequential autoregressive sampling scheme. The architecture shows how each orbital's occupation probability is computed conditionally based on previously sampled orbitals through the Transformer decoder layers. We implement the sampling procedure by generating one bit at a time: we first compute ∣Ψ(x1)∣2 and sample the first bit x1, then using the sampled x1, we compute ∣Ψ(x2∣x1)∣2 and sample x2, followed by computing ∣Ψ(x3∣x1, x2)∣2 and sampling x3 using all previous bits, and so on until all N bits have been sampled. This sequential sampling strategy is essential for the VMC optimization of our neural quantum states, as it enables simultaneous sampling and network training: while generating configurations for energy estimation, the network parameters can be updated through gradient computation, effectively integrating the sampling and optimization processes into a unified workflow. This approach consists of decoder layers that process sequential conditional probabilities, where each step computes the conditional probability of a quantum variable vk given its previous variables. The dashed arrows indicate the autoregressive sampling process, where each variable is sampled conditionally on previously generated variables. Green boxes represent input tokens \({v}_{k}^{i}\) at different positions in the sequence. Blue boxes denote the Transformer decoder layers and phase subnetwork components. Orange boxes show the sampling operations at each step. Red dashed boxes highlight the newly sampled variables at each step, while green solid boxes indicate previously sampled variables that serve as context for subsequent sampling. Moreover, our training employs a dynamic sample adjustment mechanism that monitors the number of unique quantum states generated during sampling and automatically adjusts the total sample size accordingly. Through this adaptive approach, the sampling process naturally adjusts to the complexity of different molecular systems and the convergence state of the model, ensuring efficient training without manual tuning of sampling parameters for each specific case. The sampling process naturally forms a Monte Carlo quaternary tree structure, where each layer corresponds to a spatial orbital. While our model represents each spin–orbital as a binary token (0 or 1), the MCTS algorithm groups spin-orbitals by spatial orbital for efficiency. Since each spatial orbital contains two spin-orbitals (spin-up and spin-down), this yields four possible combined states at each tree node: “00”, “01”, “10”, or “11”. This grouping reduces the tree depth from 2N to N levels for a system with N spatial orbitals, effectively building up the complete electronic configuration through a hierarchical sampling process. Figure 6 illustrates our batched autoregressive sampling implementation. 6a presents the algorithm for generating a single configuration, showing how orbital occupations are sampled sequentially with electron number constraints. 6b depicts the tree structure used for batch generation, where multiple samples are generated simultaneously using a combination of breadth-first search (BFS) for the first few orbitals followed by depth-first search (DFS) for the remaining orbitals. This hybrid approach balances exploration diversity with memory efficiency. In the initial phase, the system explores the first few orbital positions using BFS, assigning probability weights to possible orbital configurations. This ensures reasonable coverage of the sampling space, avoiding premature convergence to local optima. Subsequently, the system shifts to a DFS strategy for processing the remaining orbital positions, a key design primarily aimed at optimizing computational resources, especially memory usage. Through this depth-first approach, the system only needs to retain state information of the current sampling path in memory, rather than simultaneously storing complete states of all possible paths, significantly reducing memory requirements, as shown in Algorithm 2 within Supplementary Information. This memory-efficient design enables the model to handle larger-scale quantum systems without being strictly limited by hardware resources. This hybrid sampling strategy is not only more efficient in computational resource utilization but also maintains sampling diversity while guiding the sample distribution to converge to the minimum energy quantum state configuration, accurately characterizing the ground state properties of quantum many-body systems. A key feature of our MCTS sampling implementation is the incorporation of physical constraints through efficient pruning strategies. Since the total number of electrons Ne is conserved in quantum chemical systems, we can significantly reduce the sampling space by eliminating invalid branches of the sampling tree. This pruning mechanism ensures that we only sample physically meaningful configurations while significantly reducing the computational cost. This tree-like structure of our sampling procedure bears resemblance to the MCTS used in AlphaGo50, but with a crucial distinction: while AlphaGo's MCTS relies on extensive backtracking to explore the game tree and evaluate different strategies, our method employs pure forward autoregressive sampling without any backtracking. This fundamental difference arises from the distinct nature of quantum state sampling, where our goal is to efficiently generate physically valid electronic configurations that follow the wave function probability distribution, rather than exploring multiple strategic paths as in game-playing scenarios. The absence of backtracking in our approach not only simplifies the sampling process but also significantly enhances its computational efficiency while maintaining the ability to capture the quantum state's probability distribution accurately. This sampling strategy enables the simultaneous generation of multiple configuration samples, dramatically improving the efficiency of quantum Monte Carlo calculations. By generating independent samples directly, we eliminate both the correlation issues inherent in MCMC methods and the computational overhead of backtracking processes, while also enabling more accurate estimation of physical observables through larger sample batches. 7, we showcase a comparison between the performance of MCMC sampling and the MCTS method for H2O system. From this comparison, it becomes evident that the computational time of the MCMC method increases as the number of samples grows, and the low acceptance probability associated with the Metropolis-Hastings algorithm renders it inefficient for larger sample sizes. In contrast, the computational time of the MCTS method exhibits a nearly constant behavior, since it is primarily dependent on the number of unique samples. A computational efficiency improvement of several orders of magnitude is achieved compared to the MCMC method. Figure 7b illustrates that, with the same number of samples (e.g., 105), the MCTS method achieves chemical accuracy within 1500 steps, whereas the MCMC method fails to attain chemical accuracy even after an extensive number of steps. a The autoregressive sampling (AS) algorithm generates one sample per run. Each node in (b) corresponds to a particular local sampling outcome, and the number on the edge pointing to each circle indicates the weight. Ns can be chosen to be any number (Ns = 1000 is used as an example). Each node has four possible states (unoccupied, spin-up, spin-down, or doubly occupied). The tree is naturally pruned based on electron number conservation: branches that would lead to configurations with incorrect total electron numbers are eliminated (shown in gray dashed lines). This pruning significantly reduces the sampling space and ensures that only physically valid configurations are generated. a The time comparison of H2O system for Markov Chain Monte Carlo (MCMC) and Monte-Carlo tree search (MCTS) autoregressive sampling methods. b The convergence comparison of H2O between MCMC and MCTS sampling methods. Ns  = 105 is used as an example, where Ns denotes the number of samples used in the variational Monte Carlo calculation for estimating the energy expectation value. The VMC algorithm then proceeds iteratively to optimize QiankunNet using energy as the loss function, as illustrated in Fig. 1b: at each step, we sample configurations from the current wave function ∣Ψ(x)∣2, compute the energy gradient using these samples, and update the network parameters using the AdamW optimizer, for more details, see Supplementary Information, Sec. This process continues until we reach convergence, yielding a neural network representation of the ground state that minimizes the energy expectation value. To further facilitate reproducibility, a minimal working example of the code, demonstrating its core functionality, is provided in Supplementary Data 1. The data and analysis scripts that support the findings of this study are available in figshare51. All other data generated during this study are included in this published article and its Supplementary Information files. Source data are provided with this paper. The code developed in this study has explicit restrictions on public sharing to protect intellectual property rights, potential commercial applications, and ongoing institutional research interests. However, to support academic collaboration and reproducibility, the source code can be made available to researchers affiliated with academic institutions for non-commercial research purposes only. Researchers interested in accessing the code can submit formal requests detailing their intended use and institutional affiliation to the corresponding author at shanghui.ustc@gmail.com. Access requests will be reviewed and responded to within two weeks. Upon approval, a Software License Agreement will be provided for review and signature. This agreement explicitly prohibits any commercial use, modification for commercial purposes, and redistribution of the code. To further facilitate reproducibility, a minimal working example of the code, demonstrating its core functionality, is provided in the Supplementary Data file. 31st International Conference on Neural Information Processing Systems, NIPS'17 6000–6010 (Curran Associates Inc., 2017). Radford, A., Narasimhan, K., Salimans, T., Sutskever, I. et al. Improving language understanding by generative pre-training. Language models are unsupervised multitask learners. 34th International Conference on Neural Information Processing Systems, NIPS'20 (Curran Associates Inc., 2020). Xiong, R. et al. On layer normalization in the transformer architecture. 37th International Conference on Machine Learning of Proceedings of Machine Learning Research, 119, 10525–10535 (PMLR, 2020). Dosovitskiy, A. et al. An image is worth 16x16 words: transformers for image recognition at scale. 9th International Conference on Learning Representations (ICLR, 2021); https://openreview.net/forum?id=YicbFdNTTy. Bao, H., Dong, L., Piao, S. & Wei, F. BEiT: BERT pre-training of image transformers. 10th International Conference on Learning Representations (ICLR, 2022); https://openreview.net/forum?id=p-BhZSz59o4. Highly accurate protein structure prediction with AlphaFold. Unsal, S. et al. Learning functional properties of proteins with language models. Vu, M. H. et al. Linguistically inspired roadmap for building biologically reliable protein language models. Accurate medium-range global weather forecasting with 3D neural networks. Viteritti, L. L., Rende, R. & Becca, F. Transformer variational wave functions for frustrated quantum spin systems. von Glehn, I., Spencer, J. S. & Pfau, D. A self-attention ansatz for ab-initio quantum chemistry. 11th International Conference on Learning Representations (ICLR, 2023); https://openreview.net/forum?id=xveTeHVlF7j. Helgaker, T., Jørgensen, P., Olsen, J., Perturbation Theory Ch. Møller, C. & Plesset, M. S. Note on an approximation treatment for many-electron systems. Shepard, R., The Multiconfiguration Self-Consistent Field Method, 63–200 (John Wiley & Sons Ltd, 1987). & Musiał, M. Coupled-cluster theory in quantum chemistry. White, S. R. Density matrix formulation for quantum renormalization groups. White, S. R. Density-matrix algorithms for quantum renormalization groups. & Rajagopal, G. Quantum Monte Carlo simulations of solids. & Lester, W. A. J. Quantum Monte Carlo and related approaches. Carleo, G. & Troyer, M. Solving the quantum many-body problem with artificial neural networks. & Das Sarma, S. Quantum entanglement in neural network states. & Carleo, G. Neural tensor contractions and the expressive power of deep neural quantum states. Efficient representation of quantum many-body states with deep neural networks. & Noé, F. Deep-neural-network solution of the electronic schrödinger equation. Pfau, D., Spencer, J. S., Matthews, A. G. D. G. & Foulkes, W. M. C. Ab initio solution of the many-electron schrödinger equation with deep neural networks. & Carleo, G. Fermionic neural-network states for ab-initio electronic structure. & Lvovsky, A. Autoregressive neural-network wavefunctions for ab initio quantum chemistry. & Clark, B. K. Neural network backflow for ab initio quantum chemistry. & Clark, B. K. Unifying view of fermionic neural network quantum states: from neural network backflow to hidden fermion determinant states. Wu, Y., Guo, C., Fan, Y., Zhou, P., Shang, H., Nnqs-transformer: an efficient and scalable neural network quantum states approach for ab initio quantum chemistry. International Conference for High Performance Computing, Networking, Storage and Analysis (SC'23) (Association for Computing Machinery, 2023). Fu, L., Wu, Y., Shang, H. & Yang, J. Transformer-based neural-network quantum state method for electronic band structures of real solids. Ma, H., Shang, H. & Yang, J. Quantum embedding method with transformer neural network quantum states for strongly correlated materials. Accurate calculation of interatomic forces with neural networks based on a generative transformer architecture. & Shang, H. Bridging the gap between transformer-based neural networks and tensor networks for quantum chemistry. & Shang, H. Fast and scalable neural network quantum states method for molecular potential energy surfaces. State-of-the-art density matrix renormalization group and coupled cluster theory studies of the nitrogen binding curve. Al-Saidi, W. A., Zhang, S. & Krakauer, H. Bond breaking with auxiliary-field quantum Monte Carlo. Kremer, M. L. Mechanism of the Fenton reaction. Dunford, H. B. Oxidations of iron (ii)/(iii) by hydrogen peroxide: from aquo to enzyme. Petit, A. S., Pennifold, R. C. & Harvey, J. N. Electronic structure and formation of simple ferryloxo complexes: mechanism of the Fenton reaction. Sayfutyarova, E. R., Sun, Q., Chan, G. K.-L. & Knizia, G. Automated construction of molecular active spaces from atomic valence orbitals. Silver, D. et al. Mastering the game of go with deep neural networks and tree search. This work is supported by National Natural Science Foundation of China (T2222026, 92270001, and 22288201), Innovation Program for Quantum Science and Technology (2021ZD0303306) and the Strategic Priority Research Program of the Chinese Academy of Sciences (XDB0450101). These authors contributed equally: Honghui Shang, Chu Guo. Honghui Shang, Chu Guo, Yangjun Wu, Zhenyu Li & Jinlong Yang Hefei National Laboratory, University of Science and Technology of China, Hefei, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the numerical simulations and analysis. Correspondence to Honghui Shang or Jinlong Yang. The authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Solving the many-electron Schrödinger equation with a transformer-based framework. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a68068223/quantum-supremacy-breakthrough/'>Scientists Say We've Finally Reached Quantum Supremacy. For Real This Time!</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Unfortunately, that doesn't mean we're suddenly living in the quantum era. We may earn commission if you buy from a link. Spend an afternoon reading about quantum computing, and it won't take long until you stumble across the term “quantum supremacy.” While this is quite a lofty-sounding boast, the idea is relatively simple: If a a quantum computer—with its useful qubits in superposition—can do calculations that would be impossible (or hugely impractical) for a classical supercomputer to do, it has achieved quantum supremacy. Google first tried to claim the supremacy crown back in 2019, and researchers in China argued the same a year later. But in both cases, experts weren't so sure if true supremacy had been achieved. In Google's case, its Sycamore quantum computer completed a task in three minutes and 20 seconds that the company said would take a classical computer 10,000 years to complete. As New Scientist notes, proving supremacy is notoriously difficult because you need to be sure that classical algorithms could never solve the task placed upon the quantum computer in question—and it seems that scientists at the University of Texas at Austin have finally pulled it off. In a study published on the preprint server arXiv, the U.S.-based research team led by scientists at UT Austin report that they've successfully demonstrated “unconditional separation” between quantum and classical computers. “A longstanding goal in quantum information science is to demonstrate quantum computations that cannot be feasibly reproduced on a classical computer,” the authors wrote. “We construct a task for which the most space-efficient classical algorithm provably requires between 62 and 382 bits of memory, and solve it using only 12 qubits[…]. Instead of creating some immensely complex calculation, the researchers focused on something a bit smaller. The quantum computer then optimized this procedure until Bob's output could be predicted before Alice revealed its state, reports to New Scientist state. While a quantum computer required only 12 qubits to complete this task, the authors noted that a classical computer would need somewhere between 62 and 382 bits to perform the same task—undeniable evidence of quantum information supremacy. “Unlike prior demonstrations of quantum advantage that rely on unproven complexity assumptions, our result is provable and permanent: no future development in classical algorithms can close this gap.” the authors wrote. Although this is good to know, scientists already had a sneaking suspicion that quantum computing would eventually outshine its classical counterpart due to the superposition properties of qubits. The real work moving forward will be building fault-tolerant quantum computers with more than 1,000 qubits—and, excitingly, things are progressing. In a paper published this week in the journal Nature, Harvard scientists successfully demonstrated a coherent 3,000 qubit computer with new methods of error correction.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a68040716/titanics-sunken-sister-ship/'>Divers Explored the Titanic's Sunken Sister Ship—and It's Incredible</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In doing so, they retrieved a trove of artifacts, despite the strong current and poor visibility. But the ship's lookout bell and port navigation lamp came to the surface in secure lift bags, along with ceramic tiles from the onboard Turkish bath, silver-plated first-class trays and a pair of items from the second-class cabins: a passenger's binoculars and a porcelain sink, according to a report from the Greek-language Kathimerini. Britannic, a slightly larger and newer sister ship of the Titanic, launched in 1914 as a White Star Line's Olympic-class luxury cruise liner, and was converted into a hospital ship for the British Royal Navy during World War I. Less than an hour later, the Britannic had fully sunk. Tragically, thirty of the 1,065 people on board died when two lifeboats were sucked into the ship's still-moving propellers. Under approval from Greece's Ministry of Culture, the 11-member multinational team used closed-circuit rebreathers to reach the Britannic, aiming for specific items during a weeklong effort this past spring. The divers had navigate tight quarters, in some cases with extremely fragile conditions. Each item they did recover was placed in a lift bag, and then a protective container, and immediately cleaned of grime from over 100 years underwater. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Fire Patrol Finds Ancient Face Carving in Boulder Experts Find Missing Piece of Ramesses II Statue Scientists Say Daylight Savings Time is Hurting Us These Edge State Atoms Showcase a Quantum Mystery</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a68063023/amazon-bigger-trees-climate-change/'>The Amazon's Trees Are Weirdly Getting Bigger Every Decade, Which Defies Logic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Known as the “lungs of the planet” due to its immense concentration of vegetation, the Amazon is central to humanity's fight against climate change, acting as a vital carbon sink for humanity's greenhouse gas emissions. While the Amazon is helpless against the ecological wrecking ball known as humanity, a new study shows that it still retains a remarkable ability to adapt. Published in the journal Nature Plants, this study—completed by an international team of scientists and led by University of Cambridge ecologist Adriane Esquivel-Muelbert—details how climate change (specifically, the rise in carbon dioxide levels) is having a surprisingly positive effect on average tree size in the Amazon. “Large trees are hugely beneficial for absorbing CO2 from the atmosphere and this study confirms that,” Esquivel-Muelbert said in a press statement. This shows the remarkable resilience of these forests, at least for now." While this fertilization benefit from increased carbon dioxide impacts trees both small and large, the most profound impacts were witnessed in the latter category. The authors note that this finding runs counter to the “large-trees-lose hypothesis,” which states that larger trees should be more susceptible to increased drought or windthrow (being uprooted by strong winds). A 2023 study from the University of New Brunswick in Canada found that boreal forests (northern high latitude forests) will grow 20 percent faster in 2050 due to warmer temperatures and the longer growing seasons they bring. But in both of these cases, the good news of chunkier or faster-growing trees may not necessarily outpace the climate change-induced disruptions these forests face—whether by humans, insect outbreaks, wildfires, or other extreme climate calamities. Reuters reports that 2024 was one of the worst years for tropical rainforest loss in history. “What happens to big trees—including how they deal with increasing climate threats and manage to disperse their seeds—is now mission-critical,” Oliver Phillips, a co-author of the study and ecologist from the University of Leeds, said in a press release. Deforestation is a huge threat-multiplier and will kill them if we let it.” Rainforests around the world will help us heal the planet as we transition away from fossil fuels to a green energy future. Fire Patrol Finds Ancient Face Carving in Boulder Experts Find Missing Piece of Ramesses II Statue Scientists Say Daylight Savings Time is Hurting Us These Edge State Atoms Showcase a Quantum Mystery</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250928095654.htm'>Earth was born dry until a cosmic collision made it a blue planet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 11:22:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At this time, however, there were hardly any elements necessary for life such as water or carbon compounds on the young planet. Only a later planetary collision probably brought water to Earth, paving the way for life. Earth is so far the only known planet on which life exists -- with liquid water and a stable atmosphere. However, the conditions were not conducive to life when it formed. Only celestial bodies that formed further away from the Sun in cooler regions were able to incorporate these components. When and how the Earth became a life-friendly planet is still not fully understood. Their results, recently published in the journal Science Advances, suggest that life on Earth was only made possible by a later event. Kruttasch is now an SNSF Postdoc Mobility Fellow at Imperial College London. Kruttasch explains: "A high-precision time measurement system based on the radioactive decay of manganese-53 was used to determine the precise age. This isotope was present in the early Solar System and decayed to chromium-53 with a half-life of around 3.8 million years." This method allowed ages to be determined with an accuracy of less than one million years for materials that are several billion years old. Considering that it only took up to 3 million years to determine the chemical properties of the Earth, this is surprisingly fast," says first author Kruttasch. Theia probably formed further out in the Solar System, where volatile substances such as water accumulated. "Thanks to our results, we know that the proto-Earth was initially a dry rocky planet. It can therefore be assumed that it was only the collision with Theia that brought volatile elements to Earth and ultimately made life possible there," says Kruttasch. Life-friendliness in the universe cannot be taken for granted This makes it clear that life-friendliness in the universe is anything but a matter of course," says Mezger. Models are needed that can fully explain not only the physical properties of the Earth and Moon, but also their chemical composition and isotope signatures," concludes Kruttasch. Note: Content may be edited for style and length. The Tropical Fruit That May Improve Blood Sugar and Cut Body Fat Scientists “Improve on Mother Nature” To Create Powerful Cancer-Killing Molecule Revolutionizing Alzheimer's Detection: Non-Invasive Test Could Detect Early Warning Signs Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250928095627.htm'>Miscarriages, down syndrome, and infertility all linked to this hidden DNA process</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 03:23:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When a woman becomes pregnant, the outcome of that pregnancy depends on many things -- including a crucial event that happened while she was still growing inside her own mother's womb. The DNA-containing chromosomes in those cells must be cut, spliced and sorted perfectly. "This can result in infertility, miscarriage or the birth of children with genetic diseases." He has pieced together the choreography of proteins that connect matching chromosome pairs -- ensuring that they are sorted correctly as egg and sperm cells develop and divide. Hunter's discoveries required methods to watch the molecular events of chromosome recombination unfold with unprecedented detail. This involved genetic engineering in budding yeast -- a model organism that has been used for decades to discover how fundamental cellular processes work. "The chromosome structures that we studied have changed very little across evolution," Hunter said. Early in the process of making sperm or eggs, those chromosome pairs line up, and the parental chromosomes break and rejoin to each other. First, they help ensure that each chromosome that is passed on to the offspring contains a unique mixture of genes from both parents. These connections guide the distribution of chromosomes when cells divide to produce eggs and sperm. Maintaining crossover connections is especially crucial in females, Hunter said. DNA strands of this structure are then cut to join the chromosomes forming a crossover. The immature egg cells can remain in suspended animation for decades after birth, until they are activated to undergo ovulation. "Maintaining the crossover connections over many years is a major challenge for immature egg cells," Hunter said. If chromosome pairs aren't connected by at least one crossover, they can lose contact with each other, like two people separated in a jostling crowd. This causes them to segregate incorrectly when the cell finally divides, producing egg cells with extra or missing chromosomes. This can cause infertility, miscarriage or genetic conditions such as Down syndrome, in which a child is born with an extra copy of chromosome 21, leading to cognitive impairment, heart defects, hearing loss and other problems. Researchers have identified dozens of proteins that bind and process these junctions. With this method, they made cells degrade one or more specific proteins within the junction-associated structures. In this way, they built up a picture in which a network of proteins function together to ensure that crossovers are formed. "This strategy allowed us to answer a question that previously wasn't possible," Hunter said. They identified key proteins such as cohesin that prevent an enzyme called the STR complex (or Bloom complex in humans) from inappropriately dismantling the junctions before they can form crossovers. This years-long research project in yeast is broadly relevant for human reproduction because the process has changed very little during evolution. In addition to Tang, the postdoc, seven undergraduates in the UC Davis College of Biological Sciences contributed to this work, including Jennifer Koo, Mohammad Pourhosseinzadeh, Emerald Nguyen, Natalie Liu, Christopher Ma, Hanyu Lu and Monica Lee. Hunter's research on crossover and homologous recombination uses advanced scientific facilities at the university's Proteomics Core Facility, MCB Light Microscopy Imaging Facility, Genome Center, Mouse Biology Program, and the Comprehensive Cancer Center. The Tropical Fruit That May Improve Blood Sugar and Cut Body Fat Scientists “Improve on Mother Nature” To Create Powerful Cancer-Killing Molecule Revolutionizing Alzheimer's Detection: Non-Invasive Test Could Detect Early Warning Signs Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250928095633.htm'>Heisenberg said it was impossible. Scientists just proved otherwise</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-29 03:23:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Physicists in Australia and Britain have reshaped quantum uncertainty to sidestep the restriction imposed by the famous Heisenberg uncertainty principle - a result that could underpin future ultra-precise sensor technology used in navigation, medicine and astronomy. The Heisenberg uncertainty principle, introduced in 1927, says that you can't know certain pairs of properties - such as a particle's position and momentum - with unlimited precision at the same time. In research published Sept. 24 in Science Advances, a team led by Dr Tingrei Tan from the University of Sydney Nano Institute and School of Physics has shown how to engineer a different trade-off to precisely measure position and momentum at the same time. If it's the hour hand, you can tell what hour it is and roughly what minute, but the minute reading will be very imprecise. This 'modular' measurement sacrifices some global information in exchange for much finer detail. "By applying this strategy in quantum systems, we can measure the changes in both position and momentum of a particle far more precisely," said first author Dr Christophe Valahu from the Quantum Control Laboratory team at the University of Sydney. "We give up global information but gain the ability to detect tiny changes with unprecedented sensitivity." Here, Dr Tan's team performed the first experimental demonstration by using a technological approach they had previously developed for error-corrected quantum computers, a result recently published in Nature Physics. "Ideas first designed for robust quantum computers can be repurposed so that sensors pick up weaker signals without being drowned out by quantum noise. With this, they showed that both position and momentum can be measured together with precision beyond the 'standard quantum limit' - the best achievable using only classical sensors. Our protocol works entirely within quantum mechanics," said Dr Ben Baragiola, co-author from RMIT. "The scheme is optimised for small signals, where fine details matter more than coarse ones. The ability to detect extremely small changes is important across science and technology. Ultra-precise quantum sensors could sharpen navigation in environments where GPS doesn't work (such as submarines, underground or spaceflight); enhance biological and medical imaging; monitor materials and gravitational systems; or probe fundamental physics. Rather than replacing existing approaches, it adds a complementary tool to the quantum-sensing toolbox. "Just as atomic clocks transformed navigation and telecommunications, quantum-enhanced sensors with extreme sensitivity could enable whole new industries," said Dr Valahu. Note: Content may be edited for style and length. The Tropical Fruit That May Improve Blood Sugar and Cut Body Fat Scientists “Improve on Mother Nature” To Create Powerful Cancer-Killing Molecule Revolutionizing Alzheimer's Detection: Non-Invasive Test Could Detect Early Warning Signs Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            