
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09373-5'>Genetic suppression features ABHD18 as a Barth syndrome therapeutic target</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 15:39:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cardiolipin (CL) is the signature phospholipid of the inner mitochondrial membrane, where it stabilizes electron transport chain protein complexes1. The final step in CL biosynthesis relates to its remodelling: the exchange of nascent acyl chains with longer, unsaturated chains1. However, the enzyme responsible for cleaving nascent CL (nCL) has remained elusive. Here, we describe ABHD18 as a candidate deacylase in the CL biosynthesis pathway. Accordingly, ABHD18 converts CL into monolysocardiolipin (MLCL) in vitro, and its inactivation in cells and mice results in a shift to nCL in serum and tissues. Notably, ABHD18 deactivation rescues the mitochondrial defects in cells and the morbidity and mortality in mice associated with Barth syndrome. This rare genetic disease is characterized by the build-up of MLCL resulting from inactivating mutations in TAFAZZIN (TAZ), which encodes the final enzyme in the CL-remodelling cascade1. We also identified a selective, covalent, small-molecule inhibitor of ABHD18 that rescues TAZ mutant phenotypes in fibroblasts from human patients and in fish embryos. This study highlights a striking example of genetic suppression of a monogenic disease revealing a canonical enzyme in the CL biosynthesis pathway. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout Raw data for genetic screens and RNA sequencing are available at NCBI GEO (Accession: GSE297454 and PRJNA1301228). The Bio-ID data are available through MassIVE (Accession: PXD064381 (MSV000098022)). Links to the publicly available datasets analysed in this manuscript are as follows: DepMap, https://depmap.org. Tissue expression data (GTEX), https://gtexportal.org; and tissue expression and subcellular localization data, https://www.proteinatlas.org. No new code was generated for this study. The function of tafazzin, a mitochondrial phospholipid-lysophospholipid acyltransferase. Schlame, M. & Ren, M. Barth syndrome, a human disorder of cardiolipin metabolism. Hornby, B. et al. Natural history comparison study to assess the efficacy of elamipretide in patients with Barth syndrome. Kim, A. Y., Vernon, H., Manuel, R., Almuqbil, M. & Hornby, B. Quality of life in Barth syndrome. Sabbah, H. N., Taylor, C. & Vernon, H. J. Temporal evolution of the heart failure phenotype in Barth syndrome and treatment with elamipretide. Taylor, C. et al. Clinical presentation and natural history of Barth syndrome: an overview. Wang, S. et al. Genetic modifiers modulate phenotypic expression of tafazzin deficiency in a mouse model of Barth syndrome. & Han, X. Cardiolipin remodeling in diabetic heart. Zhu, S. et al. Cardiolipin remodeling defects impair mitochondrial architecture and function in a murine model of Barth syndrome cardiomyopathy. Whited, K., Baile, M. G., Currier, P. & Claypool, S. M. Seven functional classes of Barth syndrome mutation. Diverse mitochondrial abnormalities in a new cellular model of TAFFAZZIN deficiency are remediated by cardiolipin-interacting small molecules. Global genetic networks and the genotype-to-phenotype relationship. Analysis of 589,306 genomes identifies individuals resilient to severe Mendelian childhood diseases. Pu, W. T. Experimental models of Barth syndrome. Systematic mapping of genetic interactions for de novo fatty acid synthesis identifies C12orf49 as a regulator of lipid metabolism. Blomen, V. A. et al. Gene essentiality and synthetic lethality in haploid human cells. Essential gene profiles for human pluripotent stem cells identify uncharacterized genes and substrate dependencies. Defining a cancer dependency map. & Cravatt, B. F. The pharmacological landscape and therapeutic potential of serine hydrolases. Morgenstern, M. et al. Quantitative high-confidence human mitochondrial proteome and its dynamics in cellular context. Price, T. R. et al. Lipidomic QTL in Diversity Outbred mice identifies a novel function for α/β hydrolase domain 2 (Abhd2) as an enzyme that metabolizes phosphatidylcholine and cardiolipin. & Cravatt, B. F. The metabolic serine hydrolases and their functions in mammalian physiology and disease. Brandner, K. et al. Taz1, an outer mitochondrial membrane protein, affects stability and assembly of inner membrane protein complexes: implications for Barth syndrome. Le, C. H. et al. Tafazzin deficiency impairs CoA-dependent oxidative metabolism in cardiac mitochondria. The mitochondrial transacylase, tafazzin, regulates for AML stemness by modulating intracellular levels of phospholipids. Identification of a cardiolipin-specific phospholipase encoded by the gene CLD1 (YGR110W) in yeast. Huang, Y. et al. Cardiac metabolic pathways affected in the mouse model of Barth syndrome. Kutschka, I. et al. Activation of the integrated stress response rewires cardiac metabolism in Barth syndrome. Liu, O., Chinni, B. K., Manlhiot, C. & Vernon, H. J. FGF21 and GDF15 are elevated in Barth syndrome and are correlated to important clinical measures. Tung, C. et al. Elamipretide: a review of its structure, mechanism of action, and therapeutic potential. Bononi, G., Tuccinardi, T., Rizzolio, F. & Granchi, C. α/β-Hydrolase domain (ABHD) inhibitors as new potential therapeutic options against lipid-related diseases. Ben Ali, Y. et al. Use of an inhibitor to identify members of the hormone-sensitive lipase family. Duncan, A. L. Monolysocardiolipin (MLCL) interactions with mitochondrial membrane proteins. Burkhalter, M. D. et al. Imbalanced mitochondrial function provokes heterotaxy via aberrant ciliogenesis. The Genotype-Tissue Expression (GTEx) project. Ghandi, M. et al. Next-generation characterization of the Cancer Cell Line Encyclopedia. Whole genome sequencing delineates regulatory, copy number, and cryptic splice variants in early onset cardiomyopathy. Stem cell models of TAFAZZIN deficiency reveal novel tissue-specific pathologies in Barth syndrome. Ye, C. et al. Deletion of the cardiolipin-specific phospholipase Cld1 rescues growth and life span defects in the tafazzin mutant: implications for Barth syndrome. Tyurina, Y. Y. et al. Lipidomics characterization of biosynthetic and remodeling pathways of cardiolipins in genetically and nutritionally manipulated yeast cells. Dudek, J. et al. Cardiolipin deficiency affects respiratory chain function and organization in an induced pluripotent stem cell model of Barth syndrome. McKenzie, M., Lazarou, M., Thorburn, D. R. & Ryan, M. T. Mitochondrial respiratory chain supercomplexes are destabilized in Barth syndrome patients. Structure of the intact 14-subunit human cytochrome c oxidase. & Robinson, N. C. Bound cardiolipin is essential for cytochrome c oxidase proton translocation. Sedlák, E. & Robinson, N. C. Destabilization of the quaternary structure of bovine heart cytochrome c oxidase upon removal of tightly bound cardiolipin. Benegiamo, G. et al. COX7A2L genetic variants determine cardiorespiratory fitness in mice and human. Pérez-Pérez, R. et al. COX7A2L is a mitochondrial complex III binding protein that stabilizes the III2+IV supercomplex without affecting respirasome formation. Mair, B., Aregger, M., Tong, A. H. Y., Chan, K. S. K. & Moffat, J. A method to map gene essentiality of human pluripotent stem cells by genome-scale CRISPR screens with inducible Cas9. Brockmann, M. et al. Genetic wiring maps of single-cell protein states reveal an off-switch for GPCR signalling. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Langmead, B., Trapnell, C., Pop, M. & Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Hart, T. et al. High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities. Herzog, K. et al. Lipidomic analysis of fibroblasts from Zellweger spectrum disorder patients identifies disease-specific phospholipid ratios. Chambers, M. C. et al. A cross-platform toolkit for mass spectrometry and proteomics. Proposed minimum reporting standards for chemical analysis. Pinault, M. et al. A 1D high performance thin layer chromatography method validated to quantify phospholipids including cardiolipin and monolysocardiolipin from biological samples. Rapid staining of lipids on thin-layer chromatograms with amido black 10B and other water-soluble stains. Analysis of mitochondrial respiratory chain supercomplexes using blue native polyacrylamide gel electrophoresis (BN-PAGE). Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Aken, B. L. et al. Ensembl 2017. & Dewey, C. N. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. Fonslow, B. R. et al. Digestion and depletion of abundant proteins improves proteomic coverage. Washburn, M. P., Wolters, D. & Yates, J. R. III Large-scale analysis of the yeast proteome by multidimensional protein identification technology. He, L., Diedrich, J., Chu, Y. Y. & Yates, J. R. 3rd Extracting accurate precursor information for tandem mass spectra by RawConverter. Xu, T. et al. ProLuCID: an improved SEQUEST-like algorithm with enhanced sensitivity and specificity. Tabb, D. L., McDonald, W. H. & Yates, J. R. III DTASelect and Contrast: tools for assembling and comparing protein identifications from shotgun proteomics. Gao, J. et al. CIMAGE2.0: an expanded tool for quantitative analysis of activity-based protein profiling (ABPP) data. Weerapana, E. et al. Quantitative reactivity profiling predicts functional cysteines in proteomes. Wang, S. et al. AAV gene therapy prevents and reverses heart failure in a murine knockout model of Barth syndrome. Ren, M. et al. Extramitochondrial cardiolipin suggests a novel function of mitochondria in spermatogenesis. Molenaars, M. et al. Metabolomics and lipidomics in Caenorhabditis elegans using a single-sample preparation. Kulik, W. et al. Bloodspot assay using HPLC–tandem mass spectrometry for detection of Barth syndrome. van der Sande, M. et al. Seq2science: an end-to-end workflow for functional genomics analysis. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Frölich, S., van der Sande, M., Schäfers, T. & van Heeringen, S. J. genomepy: genes and genomes at your fingertips. The Sequence Alignment/Map format and SAMtools. Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Sayols, S., Scherzinger, D. & Klein, H. dupRadar: a Bioconductor package for the assessment of PCR artifacts in RNA-Seq data. The human genome browser at UCSC. Ewels, P., Magnusson, M., Lundin, S. & Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Production of knockout mouse lines with Cas9. Highly accurate protein structure prediction with AlphaFold. Eberhardt, J., Santos-Martins, D., Tillack, A. F. & Forli, S. AutoDock Vina 1.2.0: New Docking Methods, Expanded Force Field, and Python Bindings. We thank J. C. Bozelli Jr; the late R. Epand for characterizing HAP1 TAZ clonal cell lines for altered mCL levels using NMR spectroscopy; the proteomics team at SPARC BioCentre at the Hospital for Sick Children for isolating biotinylated preys with KingFisher Flex, mass spectrometry data generation and analysis, and use of Agilent Seahorse equipment; Cayman Chemicals for lipidomics sample preparation and mass spectrometry data generation and analysis; the Imaging Facility and Nanoscale Biomedical Imaging Facility at the Hospital for Sick Children for confocal microscope use; A. Darbandi for TEM sample preparation and instrument use; J. Burgess, A. Zalazar, S. Prokop, M. Carpenter and T. Cunningham for animal care and zebrafish-facility maintenance at the Hospital for Sick Children; the SickKids Heart Centre Biobank Registry for access to patient samples; the Temerty Faculty of the Medicine Flow Cytometry Facility at the University of Toronto for instrument use; and the core staff in the BSU, Histology and Molecular Technology facilities at the CRUK Scotland Institute. was the recipient of Mito2i graduate fellowships. This work was partly supported by the Azrieli PCHP Catalyst Program to I.C.S., the German Research Foundation (DFG; TRR259_397484323) to M.B., a BTHS IDEA grant to J.M., Cancer Research UK to the CRUK Scotland Institute (A31287), the Canadian Institutes for Health Research (PJT-GMX-463531 to J.M. ), the Ontario Research Fund (RE11 to C.B., J.M. is the GlaxoSmithKline chair in Genetics and Genome Biology at the Hospital for Sick Children. These authors contributed equally: Sanna N. Masud, Anchal Srivastava These authors jointly supervised this work: Sebastian M. B. Nijman, Vincent A. Blomen, Jason Moffat Program in Genetics and Genome Biology, The Hospital for Sick Children, Toronto, Ontario, Canada Sanna N. Masud, Patricia Mero, Jiarun Wei, David Thomson Taylor, Adrian Granda Farias, Nicholas Mikolajewicz, Olga Sizova, Andrea Habsid, Kevin R. Brown, Katherine Chan, Robert Lesurf, Lauryl M. J. Nutter, Seema Mital & Jason Moffat Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada Sanna N. Masud, Victoria Saba Echezarreta, Jiarun Wei, Adrian Granda Farias, Brandon M. Murareanu, Michelle Lohbihler, Stephanie Protze, Brenda J. Andrews, Charles Boone, Ian C. Scott & Jason Moffat Scenic Biotech, Amsterdam, The Netherlands Anchal Srivastava, Lennard van Buren, Simon van Heeringen, Linda Clijsters, Jeroen van Ameijde, Bastiaan Evers, Thijn R. Brummelkamp, Sebastian M. B. Nijman & Vincent A. Blomen Program in Developmental and Stem Cell Biology, The Hospital for Sick Children, Toronto, Ontario, Canada Victoria Saba Echezarreta, Laura McDonald & Ian C. Scott Transgenic Technology Laboratory, CRUK Scotland Institute, Glasgow, UK Eve Anderson, Freya Nye & Douglas Strathdee Institute for Biomedical Engineering, University of Toronto, Toronto, Ontario, Canada David Thomson Taylor & Jason Moffat Angela Shaw & Maximilian Billmann McEwen Stem Cell Institute, University Health Network, Toronto, Ontario, Canada Brandon M. Murareanu, Michelle Lohbihler & Stephanie Protze Department of Genetic Medicine, Johns Hopkins School of Medicine, Baltimore, MD, USA Olivia Sniezek Carney & Hilary Vernon Lucy Nedyalkova, Michael Costanzo, Brenda J. Andrews & Charles Boone Program in Molecular Medicine, The Hospital for Sick Children, Toronto, Ontario, Canada Craig Simpson & Leanne Wybenga-Groot Lundbeck La Jolla Research Center, San Diego, CA, USA Nhi Nho, Radu M. Suciu & Micah J. Niphakis Department of Pediatrics, The Hospital for Sick Children, University of Toronto, Toronto, Ontario, Canada Tanya Papaz & Seema Mital Division of Cardiology, Labatt Family Heart Centre, Toronto, Ontario, Canada Tanya Papaz & Seema Mital The Centre for Phenogenomics, The Hospital for Sick Children, Toronto, Ontario, Canada Department of Computer Science and Engineering, University of Minnesota, Minneapolis, MN, USA Ted Rogers Centre for Heart Research, Toronto, Ontario, Canada Oncode Institute, Division of Biochemistry, The Netherlands Cancer Institute, Amsterdam, The Netherlands RIKEN BioResource Research Center, Kyoto, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Analysis and interpretation: S.N.M., A. Srivastava, V.S.E., P.M., L.V.B., N.M., A. Shaw, S.v.H., B.M.M., S.P., L.M.J.N., K.R.B., R.L., M.B., M.J.N. Drafting of the manuscript: S.N.M., P.M., A. Srivastava, S.M.B.N., V.A.B. and J.M., with input from all authors. Correspondence to Sebastian M. B. Nijman, Vincent A. Blomen or Jason Moffat. serves on the Advisory Committee of Bristol Myers Squibb, Tenaya Therapeutics and Rocket Pharmaceuticals. The other authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Precision-recall curves for five biological replicate CRISPR screens in HAP1 TAZ KO cells using the reference core essential gene set (CEG2) defined in Hart et al. 2017. (b) Scatterplots representing TAZ qGIs derived from all possible pairwise combinations of five replicate screens. Pearson's correlation based on comparison of qGI scores calculated on all grey points (|qGI | >0.5, FDR < 0.5) is reported in all cases, with R and p-value (TAZ_051 vs. TAZ_186 P = 3.6667×10−91, TAZ_203 vs. TAZ_208 P = 9.31965×10−210, TAZ_203 vs. TAZ_264 P = 1.68248×10−163, TAZ_208 vs. TAZ_264 P = 3.79×10−83) determined using the stat_cor() method in the ggpubr R package. Significant points in both screens highlighted (red). (c) Fitness effect (LFC) distributions for reference core essential (CEG2) and non-essential gene sets across five TAZ KO query screens, the ABHD18-c1 screen and the TAZ;ABHD18-c2 screen respectively. (d) Enrichment for Gene Ontology (GO), molecular function, GO bioprocesses and Reactome terms among genes that exhibited a significant negative GI with TAZ (significant in at least 2 TAZ replicates, |qGI | >0.5, FDR < 0.5) (top), and positive GI with TAZ (bottom). Number of genes overlapping a particular term and term size are indicated. P-values determined by Fisher's one-tailed test using gProfileR. (e) Mutagenesis-based viability screens in HAP1 WT, and two independent TAZ cell lines. X-axis: log10-transformed mean number of insertions (± strand). Significant genes (P adj<1e-3, as compared to four WT controls) are dark grey. Selected genes are marked in red. (f) Overview of unique insertions in the ABHD18 gene in both TAZ KO (s1 and s2 clones) gene-trap screens. Gene transcript (top), sense integrations (middle), and antisense insertions (bottom). (g) Competitive growth assays. Co-cultures of TAZ;ABHD18 double KO cells with either TAZ KO cells or WT HAP1 cells followed over time by flow cytometry and normalised to day 0. (h) Relative proliferation of WT, TAZ KO, ABHD18 KO and ABHD18;TAZ double-KO RPE1 pools, presented as means of n = 2 biological replicates). Data presented as means ± s.d. with significance calculated using one-way ANOVA followed by Dunnett's post-hoc test. (j) Representative flow cytometric analysis of OCT4 & SOX2 expression in WT, TAZ, ABHD18 and ABHD18;TAZ H1 hPSC pools. (k) Bar plot showing relative fitness effects when overexpressing either TAZ (TAZ OE) or ABHD18 (ABHD18 OE) across double mutant backgrounds. Data presented are means ± s.d. (n = 3 biological replicates) with significance determined by one-way ANOVA followed by Tukey's multiple comparisons test. (l) Volcano plot showing qGI scores and associated significance (-log10(P value)) for TAZ;ABHD18-c2 CRISPR screen. (m) Scatterplot of the overlap between TAZ;ABHD18-c2 and ABHD18-c1 qGIs showing GI overlap. Yellow: Shared positive GIs, blue: shared negative GIs (qGI < −0.5, >0.5; FDR < 0.5) in both screens. (n) Heat map displaying genes (y axis) with significant interactions with TAZ across five replicate screens and at least one additional screened query (x-axis) (|qGI | >0.5, FDR < 0.5). Positive and negative qGI scores are indicated by yellow and blue, respectively. (o) Bar plot illustrating relative cell proliferation of partial LOF mutant TAZ;ABHD18-c8 (partial LOF of ABHD18) compared with WT HAP1. Data presented as means ± s.d. (n = 3 biological replicates) with significance determined by one-way ANOVA followed by Dunnett's post-hoc test. (p) Scatterplot displaying profile similarity of ABHD18 across genome-wide DepMap CRISPR/Cas9 screens. Similarity was quantified by taking all pairwise gene-gene Pearson correlation coefficients of CERES score profiles across 563 screens (19Q2 DepMap data release). Distribution of 17,633 CERES profile similarity is plotted as a quantile-quantile plot (right). Pathway analysis of ABHD18 profile similarity for all 17,634 genes represented in DepMap were mean summarized by pathway as defined in the HumanCyc standard (left). Pathway similarity and dissimilarity with ABHD18 were tested using a two-sided Wilcoxon rank-sum test with multiple-hypothesis correction using the Benjamini and Hochberg procedure. (a) Confocal images of ABHD18-FLAG HEK293 Flp-In T-Rex cells stained for FLAG (green) and HSP60 (red). Intensity measurements across the indicated white line are indicated in the bottom right panel. At least 30 cells from 2 biological replicate experiments were analyzed and a representative image shown. (b) GO biological process (BP), molecular function (MF), and cellular compartment (CC) enrichments for BioID preys captured by both N- and C-term miniTurbo-tagged ABHD18 in either glucose or galactose growth conditions. Enrichment terms for significantly enriched gene sets (p < 0.05, maximum term size 105) are indicated and bars depict -log10 (FDR) determined by p-values calculated using gProfileR. (c) Scatterplots of interaction specificity of N- and C-term tagged ABHD18 detected preys captured with BioID in either glucose or galactose growth conditions. Average spectral counts of preys captured in proximity to either N- or C-term miniTurbo BirA-tagged ABHD18 are plotted against their specificity across hundreds of baits from the Human Cell Map BioID dataset. Avg Spec was calculated using SAINTexpress v3.6.1. Data are representative of three biologically independent experiments per condition. (d) Representative TEM images of HAP1 parental, TAZ KO, ABHD18 KO, TAZ;ABHD18 double-KO cells, green: mitochondria. Representative of n = 4 biological replicate cell pellets per genotype. (a) BN-PAGE immunoblots (total OxPHOS, CII, CIII, CIV, CV) and in-gel activity assays (CI, CIV) from isolated mitochondria of HAP1 mutant cell lines. Blots probed for individual complexes and supercomplexes. Total OXPHOS (n = 5 biological replicates) obtained using antibody cocktail targeting CI, CII, CIII, CIV, and CV. CIII and CIV blots (n = 3 biological replicates), and CIV activity are representative (n = 2 biological replicates). (b) Bar plots of Seahorse Cell Mito Stress Test for respiratory parameters: basal respiration, maximal respiration, mitochondrial ATP generation, and glycolytic ATP generation across mutant HAP1 cell lines. Data presented are means ± s.d. of n = 4 technical replicates. Scatterplots of OCR and ECAR measurements taken during seahorse assays (right). (c) Dot plots of lipidomics analysis of CL and MLCL abundance in HAP1 WT and mutant cell lines (left and middle panel respectively). Relative abundance shown as bar plot above dot plot for each genotype. Bar plot quantifying MLCL/CL ratio changes across HAP1 cell lines using total CL and MLCL species abundance (right). (d) Dot plots showing lipidomics analysis of CL and MLCL abundance in an additional set of HAP1 mutant cell lines (n = 3 clonal mutant cell lines, n = 4 WT cell lines). (e) Thin Layer Chromatography (TLC) showing wild-type ABHD18[WT], but not catalytic dead ABHD18[S199A] recombinant protein deacylates different cardiolipin species to yield MLCL and DLCL, and deacylates MLCL to yield DLCL. Negative control does not contain any ABHD18 protein present, only buffer. (a) Representative CL spectrum of control wildtype mice (n = 3 animals, left), and Abhd18 knockout mice (n = 3 animals, right) obtained by blood spot assay. Blue: summation of carbon atoms in the fatty acid side chains. Red: CL peaks for each carbon group along with number of double bonds. (b) qRT-PCR analysis of Taz and Abhd18 expression levels in mice heart tissue. Expression was normalized to Gapdh. Data are presented as means ± s.e.m. (c) Schematic showing sterility test cross between Taz−/Y:Abhd18−/− double KO male mice and Taz+/−Abhd18−/− female mice. Six measurements were taken for each image using ImageJ: 1. Anterior Wall Thicknesss (AWT) – diastole (d), 2. Left Ventricle End Diastole Diameter (LVEDD), 3. Posterior Wall Thickness (PWT) – diastole (d), 4. Left Ventricle End Systole Diameter (LVESD) and 6. (a) Dot plots showing lipidomic analysis of CL and MLCL abundance in HAP1 and TAZ-c1 KO cells treated with either DMSO or 1μM of ABD646 for 5 days (left and middle panel respectively). Bar plot quantifying MLCL/CL ratio changes across HAP1 cell lines using total CL and MLCL species abundance (right). (b) Bar plots illustrating the proliferative effect of 10 μm ABD646 on HAP1 and TAZ KO cells. All values normalized to HAP1 DMSO (n = 3 biological replicates), with significance determined by one-way ANOVA followed by Dunnett's post-hoc test. (c) TEM images of HAP1 TAZ KO cells treated with DMSO or ABD646 for five days. Scale bars, 1 μm (top) and 200 nm (bottom). (d) Schematic illustrating mode of action of three substrate oxidation pathway inhibitors for glucose/pyruvate (UK5099), long chain fatty acids (LCFA, Etomoxir), and glutamine oxidation (BPTES). (e) Bar plots of seahorse substrate oxidation tests using Etomoxir, BPTES, and UK5099 on HAP1 WT and mutant cells, with ABD646 or DMSO treatment. Data are maximal respiration values (OCR pmol/min, n = 6 technical replicates). (f) Representative TEM images showing partial restoration of mitochondrial morphologies observed in BTHS fibroblasts treated with 1μM of ABD646 for 5 days. Scale bars: 1 μm, and 200 nm. n = 2 biological replicate cell pellets per treatment. (g) Flow cytometry analysis of mitochondrial mass (Mitotracker Green FM) and membrane potential (Mitotracker CMXROS) in fibroblasts from patients with BTHS treated with vehicle or 5μM ABD646 for five days. (h) BN-PAGE immunoblots and in-gel activity assays for mitochondrial complexes in fibroblasts from patients with BTHS treated with vehicle or 5μM ABD646 for five days. (i) Dot plots showing lipidomics analysis of CL and MLCL abundance in BTHS fibroblasts treated with either DMSO or 1μM of ABD646 for 5 days (left and middle panel respectively). Bar plot quantifying MLCL/CL ratio changes across HAP1 cell lines using total CL and MLCL species abundance (right). (j) Representative images at day 5 of zebrafish embryos injected with increasing concentrations of TAZ MOs (2.5ng-10.0 ng) (left; scale bars, 200 μm). Bar plot quantifying observed phenotypes across doses of TAZ MOs (right). (k) 24 h acute treatment of ABD646 has chronic benefit for TAZ MO zebrafish. Representative images of zebrafish embryos following either a 24-h treatment with ABD646 on WT (n = 45), and TAZ MO treated (n = 40) embryos. Embryos were washed once a day for 5 consecutive days with imaging and quantification on day 5 (left; scale bars, 200 μm). Right, quantification of embryos demonstrating “rescue” or near WT phenotypes per wash day. (l) Box plot of ABHD18 gene expression across HAP1 and TAZ mutant clones in nutrient rich (IMDM) or nutrient limiting (DMEM) media conditions. Data are median read counts ± interquartile range (IQR), n = 3 biological replicates per media condition. Significance determined by one-way ANOVA followed by Dunnett's post hoc test. (m) Bar plot of spearman correlations between TAZ and ABHD18 gene expression across tissue samples. Data from the Genotype-Tissue Expression (GTEx) Project. This file contains the methods for the chemical synthesis of ABD646, Supplementary Figs. 1 and 2, Supplementary Methods Tables 1–4 and full descriptions for Supplementary Videos 1–4 and Supplementary Tables 1–4 (supplied separately). Genetic screens and related data analyses. RNA-seq data from mouse hearts. Video of a wild-type GFP-labelled fish heart showing cardiac contractions following treatment with drug vehicle (see Methods for details). Video of a wild-type GFP-labelled fish heart treated for 5 days with ABD646 showing cardiac contractions (see Methods for details). Video of a GFP-labelled fish heart showing cardiac contractions following treatment with Taz morpholinos and subsequently with drug vehicle (see Methods for details). Video of a GFP-labelled fish heart showing cardiac contractions following treatment with Taz morpholinos and subsequently with ABD646 for 5 days (see Methods for details). Supplementary Video Summary Slide. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Masud, S.N., Srivastava, A., Mero, P. et al. Genetic suppression features ABHD18 as a Barth syndrome therapeutic target. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02807-0'>‘Almost unimaginable': these ants are different species but share a mother</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 15:27:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Queen Iberian harvester ants (Messor ibericus) can give birth to ants of their own species (left) and, using a cloning trick, offspring of a different one (Messor structor, right). These queen Iberian harvester ants (Messor ibericus) are sexual parasites that rely on the sperm of males of the ant species Messor structor. “It's an absolutely fantastic, bizarre story of a system that allows things to happen that seem almost unimaginable,” says Jacobus Boomsma, an evolutionary biologist at the University of Copenhagen. Iberian harvester ants co-exist with M. structor in some parts of Europe, which has historically given M. ibericus queens an abundant supply of M. structor males to mate with. Pupating ants make milk — and scientists only just noticed Pupating ants make milk — and scientists only just noticed Further analyses solved the mystery: Iberian harvester queens clone M. structor ants to maintain a supply of their sperm. They then mate with those M. structor ants to produce hybrid workers that take care of the colony, including by building the nest and foraging for food. In effect, M. ibericus has domesticated M. structor and its genome, Romiguier says. Read the related News & Views: ‘Ant queens produce sons of two distinct species'. Pupating ants make milk — and scientists only just noticed Bright city lights make birds around the world sing longer Opportunity for skilled hands and sharp minds—at any career stage As a junior research group leader, you will pursue a larger research project with your own budget, hosted by the Institutes of Chemistry and Physics. Pupating ants make milk — and scientists only just noticed An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09430-z'>Analog optical computer for AI inference and combinatorial optimization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 15:23:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Artificial intelligence (AI) and combinatorial optimization drive applications across science and industry, but their increasing energy demands challenge the sustainability of digital computing. Most unconventional computing systems1,2,3,4,5,6,7 target either AI or optimization workloads and rely on frequent, energy-intensive digital conversions, limiting efficiency. These systems also face application-hardware mismatches, whether handling memory-bottlenecked neural models, mapping real-world optimization problems or contending with inherent analog noise. Here we introduce an analog optical computer (AOC) that combines analog electronics and three-dimensional optics to accelerate AI inference and combinatorial optimization in a single platform. This dual-domain capability is enabled by a rapid fixed-point search, which avoids digital conversions and enhances noise robustness. With this fixed-point abstraction, the AOC implements emerging compute-bound neural models with recursive reasoning potential and realizes an advanced gradient-descent approach for expressive optimization. We demonstrate the benefits of co-designing the hardware and abstraction, echoing the co-evolution of digital accelerators and deep learning models, through four case studies: image classification, nonlinear regression, medical image reconstruction and financial transaction settlement. Built with scalable, consumer-grade technologies, the AOC paves a promising path for faster and sustainable computing. Its native support for iterative, compute-intensive models offers a scalable analog platform for fostering future innovation in AI and optimization. Computing today is digital, but analog has a future. Exponential advances in digital hardware have both driven and benefited from the rise of artificial intelligence (AI), but its escalating energy and latency demands push digital specialization to its limits8. Existing hardware demonstrations focus either on AI inference1,2,3,10,11,12,13, which accounts for 90% of energy in commercial deployments14, or combinatorial optimization7,15, but none efficiently accelerate both on the same analog hardware. Here we introduce the analog optical computer (AOC), a non-traditional computing platform designed for both AI inference and combinatorial optimization. By combining optical and analog electronic components within a feedback loop, the AOC rapidly performs a fixed-point search without digital conversions. In each loop iteration of approximately 20 ns, optics handle matrix–vector multiplications, whereas analog electronics perform nonlinear operations, subtraction and annealing (Fig. Over multiple iterations, the fixed-point nature of the AOC enhances noise robustness, which is essential for analog hardware. The microLED array (1) is the light source and represents neural network activations or optimization variables. The spatial light modulator (2) stores neural network weights or optimization problem coefficients, and multiplies them with the incoming light. The nonlinearity, subtraction, annealing and other computations are applied in analog electronics (4). d, ML inference as a fixed-point search. The AOC hardware is used to accelerate ML inference, using an ML equilibrium model that entails finding their fixed points. e, Quadratic optimization as a fixed-point search. A schematic of the convergence to minima over time for an optimization problem with continuous (horizontal bars) and binary (vertical arrows) variables, experiencing gradients (red) from other variables until convergence to the fixed point (green). f, Applications realized on the AOC hardware. The AOC hardware performs inference for MNIST and Fashion-MNIST classification tasks and nonlinear regression, and also solves industrial optimization problems, including medical image reconstruction and transaction settlement between financial institutions. The AOC's fully analog architecture and fixed-point abstraction address two key challenges in unconventional computing. First, hybrid architectures typically accelerate linear operations but rely on digital nonlinearities, resulting in energy-intensive conversions2,13, which are eliminated in the AOC. Second, they often face an application-hardware gap: memory-bound AI models are hard to accelerate14, and prevalent binary optimization formulations limit practical applicability16. With its unifying fixed-point abstraction, the AOC closes this gap (Fig. For inference, the AOC accelerates emerging iterative models, including fixed-point models such as deep-equilibrium networks17, which are compute-bound and costly on digital chips but naturally suited for the AOC. These models enable iterative reasoning with dynamic inference time computation18,19. For optimization, the AOC supports quadratic unconstrained mixed optimization (QUMO), a flexible formulation with binary and continuous variables that captures real-world problems20. The current small-scale AOC accelerates equilibrium models with up to 4,096 weights at 9-bit precision, performing image classification and nonlinear regression tasks (Fig. To enable these inference tasks, we design a differentiable digital twin (AOC-DT) that achieves over 99% correspondence with the physical hardware. For optimization, the AOC solves QUMO problems with up to 64 variables, tackling real-world applications such as medical image reconstruction and financial transaction settlement. The AOC-DT is used to demonstrate scalable solutions for industrial problems, such as reconstruction of a brain scan with over 200,000 problem variables. Compared with well-known heuristics21 and commercial solvers22, we set state-of-the-art results on several instances from a standard quadratic optimization benchmark23. Designed with consumer-grade optical and electronic components, the AOC leverages mature manufacturing processes, with future scalability relying on tighter coupling of integrated analog electronics with integrated three-dimensional (3D) optics. By eliminating digital–analog conversions and merging compute and memory to bypass the von Neumann bottleneck, the AOC can achieve substantial efficiency gains albeit specialized. With projected performance around 500 tera-operations per second (TOPS) per watt at 8-bit precision—over 100-times more efficient than leading graphics processing units (GPUs)24—the AOC represents a promising step towards sustainable computing. The AOC hardware unifies machine learning (ML) inference and optimization paradigms through an iterative fixed-point search. In ML, the inference of equilibrium17 and energy-based25 models entails finding their fixed points, whereas in optimization, objective minima represent fixed points of gradient-descent-based methods. At each iteration t, the continuous real-valued state vector \({{\bf{s}}}_{t}\in {{\mathbb{R}}}^{N}\) is updated to st+1, with each iteration corresponding to a signal round-trip time in hardware. Although equation (1) is discrete, the AOC operates in a continuous clock-free manner. The annealing schedule \(\alpha (t):[0,T]\to {\mathbb{R}}\) controls the state magnitude reduction per iteration, similar to residual connections in neural networks26, where T is the number of timesteps in the annealing schedule. The factor β determines the matrix–vector product scale, where the matrix W encodes neural network weights or an optimization problem, and \(f:{{\mathbb{R}}}^{N}\to {{\mathbb{R}}}^{N}\) is an element-wise nonlinear function. The coefficient γ ∈ (0, 1) introduces momentum which, in continuous time, corresponds to the second-order differential equation dynamics (Supplementary Information section G.10), generalizing the AOC abstraction beyond first-order models such as Hopfield networks. The bias vector \({\bf{b}}\in {{\mathbb{R}}}^{N}\) represents additional problem-specific information. The introduced fixed-point abstraction is ideally suited for analog feedback devices, such as the AOC, as it is compute-bound, requires no intermediate memory and is noise-tolerant: the attracting fixed point pulls the trajectory closer at every iteration, counteracting analog noise. The AOC hardware combines 3D optical and analog electronic technologies to accelerate all the compute operations in the fixed-point abstraction described by equation (1): matrix–vector multiplication, nonlinearity, annealing, addition and subtraction. In each fixed-point iteration, the analog signal alternates between optical and electrical domains, giving the system state st a dual opto-electronic nature. Matrix–vector multiplication occurs in the optical domain, where the state vector st is encoded in the light intensity of arrays of micro light-emitting diodes (microLEDs), whereas the weight matrix W is represented by spatial light modulator (SLM) pixels (Fig. Light from each microLED fans out across an SLM row for element-wise multiplication, and the resulting light signals are summed column-wise by a photodetector array27. In contrast to planar optical architectures, the AOC leverages 3D optics with its efficient fan-in and fan-out of light in the third dimension through the use of spherical and cylindrical optics, thus enabling inherently parallel and scalable multiplication operations of larger matrices28. The remaining operations in equation (1) are implemented via analog electronics: a hyperbolic tangent (tanh) function for the element-wise nonlinearity, summing and difference amplifiers for addition and subtraction of analog signals, and variable gain amplifiers for the annealing schedule α(t) and the β factor. The circuit layout, with highlighted voltage readout position, is detailed in Extended Data Fig. The AOC hardware executes the iterative update rule from several to thousands of iterations until convergence to a fixed point, when the signal amplitudes are read out digitally. This all-analog operation minimizes the overhead of analog-to-digital conversions. The current hardware includes 16 microLEDs and 16 photodetectors, supporting a 16-variable state vector st, along with two SLMs to handle positive and negative entries of the matrix W (Fig. This configuration is sufficient to support inference for ML models and optimization tasks with up to 256 weights and can extend to 4,096 weights via problem decomposition, maintaining fully analog fixed-point iterations. The AOC fixed-point abstraction targets a balance between generality and efficient hardware implementation. To illustrate its versatility, the remainder of the paper presents four case studies highlighting how equilibrium ML models can be applied to classification and regression tasks, and how the QUMO paradigm can represent real-world applications in finance and healthcare, while utilizing the same AOC hardware. The AOC supports neural equilibrium models, which have been widely applied across various domains from language17 to vision29. Equilibrium models typically follow a fixed-point iterative update rule, st+1 = Network(st), with examples including classic Hopfield networks30 and their modern variants31, as well as deep-equilibrium models17. These models operate as self-recurrent neural networks with constant input, driving the hidden state to a fixed point that represents the network output (Fig. Their dynamic depth enables recursive reasoning, leads to improved scaling laws32 and enhances out-of-distribution generalization compared with feedforward models18,33. Recent self-recurrent language models with billions of parameters show impressive reasoning capabilities, surpassing fixed-depth models in representation power34,35. We demonstrate that the AOC supports models with such recurrent nature and achieves greater out-of-distribution generalization (Extended Data Fig. 2a, the complete neural network architecture includes an input projection (IP) layer, the equilibrium model and an output projection (OP) layer. a, Top: the neural network architecture used for training and inference. During training, input data are projected into the latent space via an IP layer, processed by the AOC hardware's digital twin (AOC-DT) and passed through the OP layer. b, During inference, MNIST and Fashion-MNIST images are passed through the IP layer and are fed into the AOC hardware. Intermediate states may be projected out to monitor progress, schematically showing evolution from a higher to a lower entropy distribution. Once converged, a softmax nonlinearity in the OP layer selects the highest-probability class, whereas in regression, the OP layer outputs a continuous value y, with the MSE evaluated against ground truth (GNDTH) to assess performance. GNDTH is the true curve we want to regress against. c, The nonlinear regression results are demonstrated for the AOC hardware over a Gaussian curve (left) and sinusoidal curve (right) with MSE losses of 3.75 × 10−3 and 1.21 × 10−2, respectively. Higher accuracies are achieved for larger models (4,096 weights), which are realized with a time-multiplexing technique. For equilibrium models, the fixed-point iterative update follows from equation (1) by setting α(t) → α, γ(t) → 0, b → b + xproj, and using the element-wise tanh nonlinearity: The trained weight matrix W is quantized to 9-bit integers (Methods), separated into positive and negative components, and loaded into the corresponding SLMs. The initial state is set to s0 = b + xproj. During inference, the original data x go through the IP layer as xproj = WIPx + bIP, where WIP and bIP are the trained IP weights and biases. Finally, the inference result is obtained by applying the OP layer to the AOC solution as y = WOPs*  + bOP, where WOP and bOP are the trained OP weights and bias. In the current hardware, the equilibrium model is implemented for a single-layer network with a 256-weight matrix, without symmetry constraints. A recurrent multilayer neural network can be constructed using a lower subdiagonal block-wise matrix (Fig. We note that neural networks conventionally apply the activation function and weight matrix in a different order than in equation (2). However, this leads to only a minor difference in practice owing to the self-recursive process36. The left panel shows a multilayer deep neural network (DNN) with recurrence, where orange, green and blue denote different layers. The right panel further represents this architecture through a single block iterated over time, capturing the recurrent multilayered model structure. All three panels depict equivalent representations of the recurrent multilayer network. The hardware realizes the equilibrium model as in equation (2) for two ML inference tasks: image classification and nonlinear regression (Fig. In both cases, models are trained digitally through the AOC-DT and are deployed on the hardware without further calibration, which requires high hardware precision and high fidelity of the AOC-DT (Methods). We show that the AOC hardware can run nonlinear regression models. Regression tasks require continuous-valued outputs, which are challenging owing to the inherently noisy nature of analog computations. This is in contrast to classification tasks discussed below where the output labels are discrete and only the class of the largest probability is selected. From a model perspective, nonlinear regression tasks are well suited to the 256-weight AOC as they require small input (\({W}_{{\rm{IP}}}\in {{\mathbb{R}}}^{16\times 1}\)) and output (\({W}_{{\rm{OP}}}\in {{\mathbb{R}}}^{1\times 16}\)) projection matrices. We select two nonlinear functions for regression: Gaussian and sinusoidal curves. In agreement with the AOC-DT results (see Supplementary Table 2), the hardware reproduces accurately both functions, as shown in Fig. The sinusoidal curve presents a greater challenge for accurate fitting than the Gaussian curve owing to its multiple minima and maxima, requiring higher AOC-DT fidelity. However, at no point does the AOC-DT curve fall outside the AOC standard deviation. For the Modified National Institute of Standards and Technology (MNIST) and Fashion-MNIST datasets, the inputs \({\bf{x}}\in {{\mathbb{R}}}^{28\times 28}\) are rescaled to [−1, 1] range and flattened into vectors. The AOC results demonstrate the viability of digital training with subsequent weight transfer for opto-electronic analog inference. The contribution of the equilibrium model running on the AOC further becomes apparent when comparing the AOC results with a linear classifier, which consists of digitally trained IP layer, a middle layer and an OP layer. We also train a simple feedforward model comprising an IP layer, a middle layer with tanh nonlinearity and an OP layer. Both linear classifier and feedforward models have the same number of parameters as the AOC hardware. Although the AOC achieves slightly higher accuracy (Fig. Looking ahead, this potential may materialize in a form of test-time compute in sequence modelling tasks34,35 or inference of generative diffusion models. In practice, model sizes tend to exceed what a given hardware can support, including traditional GPUs. To address this, we demonstrate time-multiplexing on the AOC by training a 4,096-weight ensemble equilibrium model, composed of 16 independent 256-weight equilibrium models. Classification accuracies for these time-multiplexed models are shown in Fig. Across all architectures considered for classification tasks, the IP layer accounts for the majority of parameters, whereas nonlinearities within the AOC primarily drive performance differences compared with a linear classifier. Additional classification results, including ablation studies of the optical and electronic contributions with untrained IP layers, are provided in Supplementary Information section C.1. Across all classification and regression tasks, the AOC-DT requires around nine iterations per input to reach convergence. On the AOC hardware, these fixed points can be achieved in 180 ns and, ideally, their sampling would occur immediately afterwards. In practice, to ensure the state stability and mitigate noise, we sample over a fixed 6.4-μs window at 6.25-MHz frequency (Extended Data Fig. As the sampling rate is about eight-times slower than the hardware round-trip time, individual iterations cannot be resolved on the AOC. Compared with classification tasks, the regression tasks show greater sensitivity to noise, requiring up to 11 repeated runs for averaging to obtain smooth curves (Supplementary Fig. We note that owing to the iterative process, the training and inference times of the AOC-DT running in silico are approximately nine-times slower than an equivalent feedforward model. Equilibrium models, beyond being compute heavy, are also suitable for analog acceleration owing to the attractor nature of their iterative inference process. This provides enhanced robustness to analog noise compared with deep feedforward networks, which is a critical property for the AOC performance at scale (Extended Data Fig. The QUMO formulation represents a wide class of combinatorial optimization problems aimed at minimizing the objective function \(F({\bf{x}})=-\frac{1}{2}{{\bf{x}}}^{{\rm{T}}}W{\bf{x}}-{{\bf{b}}}^{{\rm{T}}}{\bf{x}}\), where the vector x includes binary and continuous variables, and the information about the optimization problem is encoded in the weight matrix W and the constant vector b. Without loss of generality, one may consider the values {0, 1} for binary and the interval [0, 1] for continuous variables. These variables are represented through the element-wise nonlinearity over the system state vector st in equation (1). Besides being nonlinear, most optimization problems are constrained. A problem with linear inequality constraints highlights the greater expressiveness of the QUMO over the standard QUBO formulation, commonly used across many non-traditional platforms. For example, only one additional continuous variable, typically referred to as slack variable, is required for mapping one inequality constraint to the QUMO problem with a penalty method. In contrast, the QUBO formulation suffers from a large mapping overhead: 10 to 100 binary variables are needed to represent a single constraint with either binary or unary encoding (Supplementary Information section G.8). We next demonstrate solving QUMO problems on the AOC hardware for two applications: medical image reconstruction and transaction settlement. We implement compressed sensing on the AOC hardware, a technique enabling accurate signal reconstruction from fewer measurements than traditionally required39,40. Compressed sensing accelerates image acquisition, reducing scan times and enhancing patient comfort. For magnetic resonance imaging (MRI), a sparse image representation is typically achieved using techniques such as wavelet regularization that penalize ‘unnatural' reconstructions. The standard regularization choice is the ℓ1-norm, which promotes sparsity and enables optimization via convex solvers. Minimizing the ℓ0-norm may yield better reconstruction in theory41,42, although the optimization problem is deemed impractical in this case and, hence, remains largely unexplored in applied image reconstruction tasks. With the AOC hardware, we can address this original hard problem by formulating the compressed-sensing approach as the QUMO optimization problem: Here the first term ensures data fidelity between measurements \({\bf{y}}\in {{\mathbb{R}}}^{M}\) and the image \({\bf{x}}\in {{\mathbb{R}}}^{N}\) in the wavelet domain, and the matrix \({\bf{A}}\in {{\mathbb{R}}}^{M\times N}\) represents the MRI acquisition process consisting of Fourier and inverse wavelet transforms with an undersampling mask (Methods). To reduce the MRI scan time, the number of measurements M needs to be smaller than the number of pixels N; hence this data-fidelity term has infinitely many solutions on its own. The image pixels are normalized to x ∈ [0, 1]N in the wavelet domain and σ ∈ {0, 1}N is a binary vector that controls the sparsity of x. Lastly, 1 and (⋅)T denote the vector of ones and the transpose operation in equation (3), respectively. The generalization of this reconstruction problem to the complex-valued variables is presented in Supplementary Information section G.3. 10), formulated as a 64-variable 9-bit QUMO problem with equal split between binary and continuous variables. As an example of a realistic MRI process, we omit 37.5% of the measurements. By minimizing the data-fidelity term only, we obtain a poor reconstruction (Fig. The greyscale bar denotes brightness values in the image, with 0 corresponding to black and 1 to white. Bottom: reconstructions by the AOC-DT for a real brain image from the FastMRI dataset at typical acceleration rates. Top: a schematic of the transaction-settlement process among multiple financial parties, with settled (green) and unsettled (red) transactions. Bottom: the number of settled transactions achieved by the AOC hardware for a 41-variable QUMO instance as a function of block coordinate descent steps. c, The AOC hardware solves synthetic 3-bit to 8-bit precision QUMO and QUBO instances with 16 variables, requiring fewer than 1,000 samples to reach at least ≥95% and 100% objective proximities for the QUMO and QUBO instances, respectively. e, The relative speed-up of the AOC-DT compared with the Gurobi solver is shown for a QUMO-reformulated subset of QPLIB benchmark instances (QBL). The state-of-the-art solutions are found for instances 3,860 and 3,584. f, The hardest QBL instances require more than 60 s for the Gurobi solver to find the best-known solutions of 10 QBL instances in their original formulation. Solutions to each subproblem, corresponding to one step of BCD, are used to create the subsequent ones, all of which are solved using the fixed-point abstraction realized on the AOC hardware. The convergence to the optimal solution takes around 30 –40 BCD steps with 1,000–1,500 AOC iterations per BCD step. The final image reconstruction closely matches the original line, as shown in Fig. We note that all QUMO instances are solved in an entirely analog manner without any digital post-processing. To validate the QUMO formulation for compressed sensing at scale, we use the AOC-DT to reconstruct a brain scan image of 320 × 320 pixels from the FastMRI dataset44, which results in the QUMO problem with more than 200,000 variables. For the typical undersampling rates of 4 and 8, we achieve reconstructions with mean squared error (MSE) below 0.07 (Fig. For optimization in the financial domain, we use the AOC hardware to solve a transaction-settlement problem. Each securities transaction is an exchange of securities for a payment, known as a delivery-versus-payment transaction. Clearing houses process batches of such transactions; for example, the subsidiaries of the Depository Trust and Clearing Corporation (DTCC) processed securities transactions valued at US$3 quadrillion in 202345. Within each batch, the transaction-settlement objective is to maximize the total number or total value of settled transactions, which is NP-hard46. This is a difficult optimization problem given the volume of transactions, legal constraints and additional requirements (for example, collateral and credit facilities). A prevalent approach for solving the transaction-settlement problem is to formulate it as a linear optimization problem with binary variables and linear inequality constraints46. This formulation can be mapped to a QUMO problem, where the inequality constraints are efficiently incorporated into the objective function by introducing continuous slack variables (Supplementary Information section G.8). We design a transaction-settlement instance generator that produces industrially relevant transaction-settlement scenarios for the given numbers of transactions, financial parties and assets. We also implement a pre-processing technique to eliminate trivial constraints. As an example, we generate a scenario with 46 transactions between 37 parties (Fig. 4b), resulting in 30 constraints, which is reduced to an effective 41-variable QUMO instance. 4b, the AOC hardware finds the globally optimal solution in seven BCD steps for this transaction-settlement scenario. Similar to reconstruction of the Shepp–Logan image, the QUMO instances across all transaction-settlement scenarios are solved in an entirely analog manner. In addition, we evaluate several smaller-size scenarios derived from real settlement data47. After pre-processing, these reduce to QUMO instances of 8 variables, on which the AOC hardware achieves a 100% success rate (see Supplementary Table 8). In contrast, quantum hardware performance on the same problems yields success rates of 40–60% (ref. For the AOC hardware, a comprehensive evaluation is conducted on a diverse set of challenging synthetic QUMO and QUBO problems. The optimization problems include instances with 16 binary and continuous variables, with dense and sparse weight matrices up to 8-bit precision. For 100 instances, the AOC hardware achieves over 95% and 100% proximity to the optimal objectives of QUMO and QUBO instances, respectively, under 1,000 samples (Fig. Using the AOC-DT, algorithmic performance is validated on the hardest quadratic binary problems with linear inequality constraints from the quadratic programming library (QPLIB) benchmark23, formulated as QUMO instances. The AOC approach is compared with the commercial Gurobi solver22, which requires over a minute to reach the best-known solutions for these problems. Figure 4e,f shows that the AOC-DT is up to three orders of magnitude faster in all instances, except for two, one of which it is unable to solve. Moreover, the AOC solver discovers the new best solutions for two heavily constrained instances (3,584 and 3,860), each with over 500 binary and 10,000 continuous variables in the QUMO formulation, in about 40 s. For instance, for 3,584, Gurobi matches the AOC solution in about 54,000 s, whereas proving its global optimality takes 4.5 days. The details of the AOC-DT parameters are provided in Supplementary Information section G.4 with additional benchmarks in Supplementary Information section G.5. Addressing practical applications with the AOC necessitates hardware scalability from hundreds of millions to billions of weights. For example, typical MRI scans with resolutions around 100,000 pixels require systems capable of processing around 20,000 variables when using decomposition techniques such as BCD, which is equivalent to handling around 400 million weights. Similarly, deep learning models with a few billion weights are standard for real-world applications48, which, with mixture of experts models or techniques such as ensembling, could be reduced to many parallel models that are an order of magnitude smaller. The AOC hardware has the potential to scale to these requirements through a modular architecture that decomposes the core optical matrix–vector multiplication operation into multiplication of smaller subvectors and submatrices. At scale, the AOC hardware will consist of multiple modules, each performing a part of the full-weight matrix multiplication. Each module will include a microLED array, a photodetector array and an SLM. With current SLMs supporting 4 million pixels, matrices with up to 4 million weights could be realized. When combined with integrated driving electronics, this results in a miniaturized module with dimensions of around 4 cm. Utilizing the third dimension for matrix–vector multiplication thus enables scalable in-memory computing. In contrast, emerging planar optical computers1,2 are constrained by the reticle-size limit of the chip area that is used for both routing and computing, which limits the matrix size28. Furthermore, as microLEDs are incoherent light sources, optical paths need to be matched only within the system bandwidth (gigahertz) rather than the source wavelength (hundreds of terahertz), which is a fundamental manufacturability advantage over coherent systems. Achieving the required miniaturization, however, is both a challenge and an opportunity to drive advancements in 3D optical technologies with broader applications. The miniaturized optical modules are coupled with integrated analog electronic models in a 3D mesh (Extended Data Fig. We envision that the AOC can support models with 0.1 billion to 2 billion weights, requiring 50 to 1,000 optical modules. The module count can be halved if a single optical module supports both positive and negative weights13. All AOC components, including microLEDs, photodetectors, SLMs and analog electronics, have an existing and growing manufacturing ecosystem with wafer-scale production. At the same time, complementing optics with analog electronics offers numerous opportunities to expand the compute primitives, including nonlinearities, the hardware can support, thereby enhancing its expressiveness. For a 100-million-weight matrix with 25 AOC modules, the power consumption is estimated to be 800 W, resulting in a computing speed of 400 peta-OPS and an efficiency of 500 TOPS W−1 (2 fJ per operation) at 8-bit weight precision (Supplementary Information section A.1). In contrast, the latest GPUs achieve a system efficiency of up to 4.5 TOPS W−1 at the same precision for dense matrices24. In conclusion, the AOC architecture shows promise for scaling to practical ML and optimization tasks, offering a potential 100-fold improvement in power efficiency. The current AOC hardware uses a rapid fixed-point search to power inference tasks, such as regression and classification, using equilibrium models with promising reasoning capabilities, and to successfully solve QUMO problems including medical image reconstruction and transaction settlement. Cross-validation with the digital twin, coupled with evaluation on large problems, offers confidence in the hardware's performance as it scales. Looking ahead, the AOC's co-design approach—aligning the hardware with the ML and optimization algorithms—could spur a flywheel of future innovations in hardware and algorithms, pivotal for a sustainable future of computing. The key components of our experimental set-up are shown in Fig. The basic components are the optical sources (input vector), a system of fan-out optics to project the light onto the modulator matrix and a system of fan-in optics to project the light onto a photodetector array (output vector). The corresponding schematic is shown in Extended Data Fig. The incoherent light sources are an array of 16 independently addressable microLEDs. Each microLED is driven with a bias current and an offset voltage. Mathematical positive values are represented by microLED drive currents greater than the bias value. Negative values are represented by drive currents less than the bias value. The diameter of each emitter is 50 μm and the pitch is 75 μm. The sources are fabricated in gallium nitride wafers on a sapphire substrate and the die is wire-bonded onto a printed circuit board (Fig. The emission spectrum is centred at 520 nm with a full-width of half-maximum of 35 nm and the operational −3-dB bandwidth is 200 MHz at 20 mA, see Supplementary Fig. After the sources, there is a polarizing beamsplitter (PBS). Each path performs two functions: first, they allow us to use both polarizations of the unpolarized light output; second, they allow us to perform non-negative and non-positive multiplications with only intensity modulation. The modulator matrix is a reflective parallel-aligned nematic liquid-crystal SLM. The microLEDs are arranged in a one-dimensional line (let this be the y axis) and are imaged onto the SLM using a 4F system composed of a high-numerical-aperture (Thorlabs TL10X-2P, numerical aperture 0.5, ×10 magnification, 22-mm field number) collection objective and a lower-numerical-aperture lens group composed of 2 achromatic doublets with combined focal length 77 mm. There is a cylindrical lens, Thorlabs LJ1558L1, in infinity space of this 4F system. Each matrix element occupies a patch of 12 (height) × 10 (width) pixels of the modulator array. An 8-bit look-up table is used to linearize the SLM response as a function of grey level. From here, the light is directed towards the intended photodetector array through a second PBS. The light from each column of the SLM is collected by an array of 16 silicon photodetectors to perform the required summation operation. The active area of each element is 3.6 × 0.075 mm2. The photodetectors are on a pitch of 0.125 mm. The photocurrents from each photodetector element are amplified by a linear trans-impedance amplifier (Analog Devices MAX4066). Each trans-impedance amplifier provides 25-kΩ gain and is characterized by an input referred noise of \(3\,{\rm{pA}}\,\sqrt{\text{Hz}}\) and has differential outputs. The corresponding 2 sets (1 per photodetector board) of 16 differential pairs of signals are fed to the main boards where the per-channel nonlinear operation and other analog electronic processing is carried out. Each of the 16 signals sees the following circuitry: (1) a variable gain amplifier (VGA; Texas Instruments VCA824) to allow the input signal range to be set and equalized across channels; (2) a difference amplifier to perform the operation of subtracting the negative input signal from the positive one and achieve signed voltages (signed multiplications); (3) a VGA that adds and subtracts signals from the described path, referred to as gradient term, to the annealing and momentum terms, as per equation (1), while providing a common gain control to all these paths; (4) an electronic switch (ADG659) to open and close the loop to set and reset the solving state; (5) a buffer amplifier to distribute the signal to the gradient, annealing and momentum paths; (6) a bipolar differential pair to implement the tanh nonlinearity; (7) a VGA to adjust the signal level between the nonlinearity and the required voltage and current onto the microLED alternating-current input circuit. Both the annealing and momentum paths have VGAs with a common external control so that we can implement time-varying annealing and momentum schedules. Each channel also has an offset to the common control signal added to allow minor adjustment or correction of channel-to-channel variations. The other VGAs are set with digital-to-analog converters controlled over an inter-integrated circuit (I2C) bus. This allows slower control at per-experiment timescales. The per-channel nonlinear function is an approximation to a tanh. This is shown in Supplementary Fig. For ML workloads, the input domain of the tanh function is unrestricted by hardware; there are no gain variations across channels. The trained weights and equilibrium model input ensure that signals evolve accurately. For optimization workloads, binary and continuous variables require different handling in hardware. Here we set the gain after the trans-impedance amplifier and before the tanh nonlinearity to be lower for continuous variables than for binary variables. This adjustment ensures that the input domain of the nonlinearity results in a more linear output for continuous variables than for binary variables. We characterized and calibrated the key opto-electronic and electronic components to equalize the gain of each AOC path. For example, we calibrate the optical paths by applying a set of 93 reference matrices and for each we digitally compute the result of the vector–matrix product. We then adjust the gain per channel slightly so that, averaged over the set of 93 computed vectors, the AOC result is as close as possible to the digital result. The mean MSE across all dot products is 5.5 × 10−3, and the matrix–vector multiplication MSE as a function of matrix (instance) is shown in Supplementary Fig. For these experiments, we configure the system in open-loop mode without feedback and turn off the annealing and momentum paths. In commercial deployments, training consumes less than 10% of the energy and, hence, is not targeted by the AOC. The equilibrium models are trained through our digital twin, which is based on equation (2). In the digital domain during training, the convergence criterion is set to ∣∣st+1 − st∣∣ < ε, with ε = 10−3. The AOC-DT models up to seven non-idealities measured on the AOC device; each non-ideality can be switched on and off (Supplementary Fig. The weight matrix is normalized to fulfil ∥W∥∞ = 1 throughout training to simulate the passive SLM. This separation of scale is necessary as several nonlinear non-idealities occur between the matrix multiplication and the gain in equation (2), as discussed in Supplementary Information section D. The weight matrix is initialized with the default Pytorch initialization for a 16 × 16 matrix, the bias term is initialized to 0 and β is initialized at 1. We trained all models with a batch size of B = 8, at a learning rate of η = 3 × 10−4 for MNIST and Fashion-MNIST and η = 7 × 10−4 for regression tasks. In all cases, models are trained end-to-end, with the equilibrium-section trained through our AOC-DT using the implicit gradient method17, which avoids storing activations for the fixed-point iterations. This decouples memory cost from iteration depth as intermediate activations do not need to be stored. In all experiments, the α gain in equation (2) is set to 0.5 to strike a balance between overall signal amplitude and speed of convergence. Low α values cause the signal to be too weak, resulting in a low signal-to-noise ratio (Supplementary Information section D). Once training has completed, the weight matrix W is quantized to signed 9-bit integers using Whenever we report AOC-DT results, we report results obtained with the quantized matrix. Exporting trained models to the AOC requires several further steps. First, the model inputs x and the bias term b need to be condensed into a single vector bAOC = b + x followed by clamp to ensure the values fit into the dynamic range of the AOC device (Supplementary Information section D). Second, as the optical matrix multiplication is implemented using SLMs, elements of the weight matrix are bounded by one such that all quantization-related factors disappear. However, the original maximum element of the matrix max(W) needs to be re-injected, which we achieve via the β gain in equation (2), approximately restoring the original matrix W. The quantized matrix is split into positive and negative parts, \({W}_{{\rm{Q}}}={W}_{{\rm{Q}}}^{+}-{W}_{{\rm{Q}}}^{-}\), and each part is displayed on its respective SLM. Each classification instance (that is, MNIST or Fashion-MNIST test image) is run once on the AOC, and the fixed point is sampled at the point marked in Extended Data Fig. 3 after a short 2.5-μs cooldown window after the switch is closed, as shown in Extended Data Fig. The sampling window extends over 40 samples at 6.25 MHz, corresponding to 6.4 μs. For classification, the input is projected from 784 to 16 dimensions, the output is projected from 16 to 10 classes. For regression tasks, the IP and OP layers transform a scalar to 16 dimensions and back, respectively. 2c were obtained by averaging over 11 repeats for each input. Importantly, the solve-to-solve variability appears to be centred close to the curve produced by the AOC-DT, enabling us to average this variability out (Supplementary Fig. We can expand the model sizes supported by the hardware by using an ensemble of small models that fit on it. These smaller 256-weight models are independent at inference time but are trained jointly by receiving slices 16-sized slices of a larger input vector and stacking their outputs before the OP. MNIST or Fashion-MNIST images are scaled to the range [−1, 1] and, projected to 4,096 dimensions and split into 16 slices of 16 dimensions. Each of the 16 equilibrium models then runs its respective slice of input vectors to a fixed-point. The first curve (I) is a Gaussian rescaled such that the Gaussian curve approximately stretches from −1 to 1, \({f}_{{\rm{I}}}(x)=2{{\rm{e}}}^{-{x}^{2}/2{\sigma }^{2}}-1\) for σ = 0.25 and x ∈ [−1, 1]. For training sets, we choose 10,000 equidistant points xi in the range [−1, 1] whereas for test regression datasets, we choose 200 points randomly xi ≈ U([−1, 1]). We trained the MNIST and Fashion-MNIST models on 48,000 images from their respective training set, validated on a set of 12,000 images and tested them on the full test set comprising 10,000 images. For experimental results, the error bars in Fig. 2d were estimated using a Bayesian approach for the decision variable ct ∈ {0, 1, …, 9} for each sample t along the sampling window per image. We assume an uninformative prior p(ct) = beta(1, 1), which we then update with the observed number of correct decisions nsuccess and failures nfailure over the sampling window. To address optimization problems involving positive and negative weights on the AOC hardware, QUMO instances without linear terms can have up to eight variables, which applies to both transaction-settlement scenarios and reconstruction of one-dimensional line of the Shepp–Logan phantom image. The weight matrices are unsigned in synthetic QUMO and QUBO hardware benchmarks; hence the AOC hardware can accommodate up to 16-variable instances in the absence of linear terms. Such instance size difference arises because, when both positive and negative weights are present, non-idealities in the dual-SLM configuration reduce the accuracy of matrix–vector multiplication. To mitigate this, a single SLM is used to process both positive and negative weights, effectively halving the number of variables per instance. For the transaction-settlement scenario and the Shepp–Logan phantom image slice, their 41-variable and 64-variable QUMO instances are decomposed into smaller 7-variable QUMO instances. For each of these subinstances, the 7 variables are connected with the rest of the variables via a linear vector b, which is incorporated into the quadratic matrix W via an additional binary variable. This decomposition is repeated for each subinstance and the linear vector b is updated at the end of each BCD iteration to create the next QUMO instance. Such an approach yields 8-variable QUMO instances and a single SLM is used to represent their positive and negative matrix elements, with analog electronics handling their subtraction, which effectively utilizes the full 16-variable capacity available in hardware. For the one-dimensional Shepp–Logan phantom image, 12 out of 32 measurements are omitted, corresponding to a 37.5% data loss or a 1.6 undersampling (acceleration) rate. Although typical MRI acceleration ranges from 2 to 8, this rate is used here owing to the image's non-smoothness at a 32-pixel resolution. In the AOC, binary variables are encoded using a hyperbolic tangent function, whereas continuous variables utilize the near-linear region of the function, connecting optimization variables to state variables via x = f(s). In simulations at scale with the AOC-DT, linear and sign functions are used for continuous and binary variables, respectively. To ensure that some variables take indeed continuous values in the global optimal solution, we plant random continuous values and generate synthetic 16-variable QUMO instances. As the number of continuous variables increases for a given problem size, the problem instances become computationally easier to solve. Consequently, we consider instances with up to eight continuous variables. We generate up to 8-bit dense and sparse instances. The sparse instances belong to the QUBO model on three-regular graphs that are NP-hard51, although NP-hardness does not imply that every random instance is difficult to solve. To make these instances more challenging to solve, we verify that their global objective minimizer is distinct from the signs of the principal eigenvector of the weight matrix52. The QPLIB is a library of quadratic programming instances23 collected over almost a year-long open call from various communities, with the selected instances being challenging for state-of-the-art solvers. As described in the main part of the paper, we consider only the hardest instances within the QPLIB:QBL class of problems, which contains instances with quadratic objective and linear inequality constraints. The QPLIB:QCBO class of problems, which contains instances with quadratic objective and linear equality constraints, and the QPLIB:QBN class of problems, which contains QUBO instances, are considered in Supplementary Information section G.5. Typically, multiple hyperparameters need to be calibrated for heuristic methods to achieve their best performance in solving optimization problems. From the hardware perspective, such an annealing schedule provides an explicit stopping criteria, which is an advantage for an all-analog hardware implementation as it avoids the complexity of multiple intermediate readouts that stochastic heuristic approaches suffer from53. In principle, the three main parameters {α0, β, γ} of the AOC fixed-point update rule need to be adjusted for each optimization instance. We further perform a linear stability analysis of the algorithm to evaluate reasonable exploration regions for these two parameters and find that by scaling the β parameter as β = β0/λlargest, where λlargest is the largest eigenvalue of the weight matrix W, we get scaled parameters β0 and α0 being in a similar optimal unit range across a wide range of problems. We design a two-phase approach for the AOC-DT to operate similar to a black-box solver that can quickly adjust the critical parameters within the given time limit. During the ‘exploration' phase, we evaluate the relative algorithm performance across a vast range of parameters (α0, β0). A subset of ‘good' parameters is then passed for more extensive investigation in the ‘deep search' phase (Supplementary Information section G.1). We note that for two QPLIB:QUMO instances, namely, 5,935 and 5,962, we developed a pre-processing technique that greedily picks variables with the highest impact on the objective functions and considers their possible values, which is accounted in the reported time speed-up of the AOC-DT. For a fair comparison, we ensure that all methods use similar computing resources. Although the implementation of GPU- or central-processing-unit-based solvers can require highly varying engineering efforts, we try to estimate the cost of running solvers on the hardware, on which they are designed to run, and vary the time limit across solvers accordingly to ensure similar cost per solver run. In what follows, the Julia-based AOC-DT runs on a GV100 GPU for 5–300 s per instance across all benchmarks. In the case of Gurobi, our licence allows us to use only up to eight cores, and its time to achieve the best solution for the first time is used (not the time to prove its optimality). More details about the AOC hardware and the AOC-DT performance on different optimization instances are provided in Supplementary Information section G.5. All data supporting the findings of this study are available in source data provided with the paper and via Zenodo at https://doi.org/10.5281/zenodo.15088326 (ref. Source data are provided with this paper. Code for the AOC digital twin supporting optimization and machine learning models will be released upon publication at https://microsoft.github.io/AOCoptimizer.jl and https://github.com/microsoft/aoc (under MIT license). Shastri, B. J. et al. Photonics for artificial intelligence and neuromorphic computing. Deep learning with coherent nanophotonic circuits. Chen, Y. et al. All-analog photoelectronic chip for high-speed vision tasks. McMahon, P. L. et al. A fully programmable 100-spin coherent ising machine with all-to-all connections. Cai, F. et al. Power-efficient combinatorial optimization using intrinsic noise in memristor Hopfield neural networks. Hao, L., William, M., Hanzhao, Y., Sachin, S. & Kim, C. H. An Ising solver chip based on coupled ring oscillators with a 48-node all-to-all connected array architecture. Johnson, M. W. et al. Quantum annealing with manufactured spins. Trends in deep learning hardware: Bill Dally (NVIDIA) YouTube https://www.youtube.com/watch?v=kLiwvnr4L80&t=770s (2023). Psaltis, D. & Farhat, N. Optical information processing based on an associative memory model of neural nets with thresholding and feedback. Parallel convolutional processing using an integrated photonic tensor core. Inference in artificial intelligence with deep optics and photonics. Hamerly, R., Bernstein, L., Sludds, A., Soljacic, M. & Englund, D. Large-scale optical neural networks based on photoelectric multiplication. Wang, T. et al. An optical neural network using less than 1 photon per multiplication. Ivanov, A., Dryden, N., Ben-Nun, T., Li, S. & Hoefler, T. Data movement is all you need: a case study on optimizing transformers. Mohseni, N., McMahon, P. L. & Byrnes, T. Ising machines as hardware solvers of combinatorial optimization problems. Jacobs, B. Quantum-Inspired Classical Computing (QuICC) HR001121S0041 Call for Proposals Report (DARPA, Microsystems Technology Office, 2021). Graves, A. Adaptive computation time for recurrent neural networks. Nye, M. et al. Show your work: scratchpads for intermediate computation with language models. Kalinin, K. P. et al. Analog iterative machine (AIM): using light to solve quadratic optimization problems with mixed variables. Microsoft Quantum Inspired Optimisation (QIO) provider (2022); https://learn.microsoft.com/en-us/azure/quantum/provider-microsoft-qio. Furini, F. et al. QPLIB: a library of quadratic programming instances. LeCun, Y. et al. A tutorial on energy-based learning. 1 (eds Bakir, G., Hofman, T., Scholkopt, B., Smola, A. Deep residual learning for image recognition. In Los Alamos Conference on Optics 1979 Vol. McMahon, P. L. The physics of optical computing. Hopfield, J. J. Neural networks and physical systems with emergent collective computational abilities. Dense associative memory for pattern recognition. In International Conference on Learning Representations (2019). Learning iterative reasoning through energy minimization. In International Conference on Machine Learning 5570–5582 (PMLR, 2022). Scaling up test-time compute with latent reasoning: a recurrent depth approach. Implicit language models are RNNs: balancing parallelization and expressivity. In International Conference on Machine Learning (PMLR, 2025). A comparative study of two modeling approaches in neural networks. Lucas, A. Ising formulations of many NP problems. Barahona, F., Jünger, M. & Reinelt, G. Experiments in quadratic 0-1 programming. & Tao, T. Near-optimal signal recovery from random projections: universal encoding strategies? Kabashima, Y., Wadayama, T. & Tanaka, T. A typical reconstruction limit for compressed sensing based on Lp-norm minimization. Sparse approximation based on a random overcomplete basis. Tseng, P. Convergence of a block coordinate descent method for nondifferentiable minimization. Zbontar, J. et al. FastMRI: an open dataset and benchmarks for accelerated MRI. DTCC 2023 Annual Report (Depository Trust & Clearing Corp., 2023); https://www.dtcc.com/-/media/Files/Downloads/Annual%20Report/2023/DTCC-2023-AR-Print.pdf. MSc thesis, KTH Royal Institute of Technology (2020). Braine, L. et al. Quantum algorithms for mixed binary optimization applied to transaction settlement. Javaheripi, M. et al. Phi-2: the surprising power of small language models. Pezeshki, B., Tselikov, A., Kalman, R. & Danesh, C. Wide and parallel LED-based optical links using multi-core fiber for chip-to-chip communications. In Optical Fiber Communication Conference (OFC) 2021 https://opg.optica.org/abstract.cfm?URI=OFC-2021-F3A.1 (2021). Kingma, D. P. Adam: a method for stochastic optimization. Garey, M. R., Johnson, D. S. & Stockmeyer, L. Some simplified NP-complete problems. 6th Annual ACM Symposium on Theory of Computing 47–63 (1974). Kalinin, K. P. & Berloff, N. G. Computational complexity continuum within Ising formulation of NP problems. Reifenstein, S., Kako, S., Khoyratee, F., Leleu, T. & Yamamoto, Y. Coherent Ising machines with optical error correction circuits. Chi, J. et al. Datasets for “The analog optical computer for machine learning inference and combinatorial optimization”. We acknowledge G. Mourgias-Alexandris and I. Haller for contributions to the first-generation AOC hardware; S. Jordan, B. Lackey, A. Barzegar, F. Hamze and M. Troyer for discussions about optimization problems and for providing us with the QUBO benchmarks (Wishart, Tile3D, RCDP); J. Cummins for contributions to medical image reconstruction; A. Grace for contributions to the AOC hardware; J. Westcott, N. Farrell and T. Burridge for the AOC mechanical parts; S. Y. Siew for help with microLEDs; M. Schapira and colleagues at Microsoft Research, Cambridge, UK, and S. Bramhavar from ARIA for discussions. acknowledges the support from HORIZON EIC-2022-PATHFINDERCHALLENGES-01 HEISINGBERG Project 101114978 and the support from Weizmann-UK Make Connection Grant 142568. Kirill P. Kalinin, Jannes Gladrow, Jiaqi Chu, James H. Clegg, Daniel Cletheroe, Douglas J. Kelly, Babak Rahmani, Grace Brennan, Burcu Canakci, Fabian Falck, Heiner Kremer, Greg O'Shea, Lucinda Pickup, Ant Rowstron, Christos Gkantsidis, Francesca Parmigiani & Hitesh Ballani Michael Hansen, Jim Kleewein, Saravan Rajmohan & Victor Ruhle Chief Technology Office, Barclays, London, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the ML experiments and analysed their data. performed the optimization experiments and analysed their data. designed and ran the digital twin experiments and analysed the data for ML models. designed and ran the digital twin experiments and analysed the data for optimization problems. provided technical input to ML experiments. and J.K. provided technical input to the hardware applications. wrote the paper with input from all authors. Correspondence to Kirill P. Kalinin, Francesca Parmigiani or Hitesh Ballani. Nature thanks Yang Wang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Photo of AOC, used to simultaneous run ML models and solve optimization instances. b, AOC photo inside a rack, with all required equipment included. Polarizing beamsplitter 1 (PBS 1) splits the light by linear polarization state and sends light to one of two modulators. The reflected light is modulated by polarization (multiplication) and the action of PBS 1 makes this an amplitude modulation. Each SLM is imaged onto its corresponding detector using LG1 and LG2 through PBS 1 and PBS 2. The state vector s(t) in equation (1) and equation (2) is measured at the pointed indicated as output. Artistic representation of AOC at 100 million weights scale, showing the 3D mesh structure of the required 50 modules, with each module with size of 4 × 106 weights (2000 variables) and distinct optical modules for positive and negative weights. We selected four time points of the trace where we projected the measured state into classification space using the output projection and applied a softmax to obtain probabilities. Here, the initial entropy of the class-probability distribution is high but drastically falls off over iterations. c, Example traces of a sinusoidal regression task for x = − 0.55. The point of interest is marked by the vertical dashed line and appears to follow the same trajectory from close to −1 to its target value. d, Multi-seed study of the Gaussian regression task for 100 independently trained and tested equilibrium models to test how reliable the AOC-DT models the AOC device. a, Out-of-distribution generalization of the equilibrium model. b, Robustness of the equilibrium model to noise for feedforward models with varying number of layers and parameter-matched single-layer DEQs. This file contains sections A–G, including Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Kalinin, K.P., Gladrow, J., Chu, J. et al. Analog optical computer for AI inference and combinatorial optimization. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09478-x'>Spatial joint profiling of DNA methylome and transcriptome in tissues</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 15:23:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The spatial resolution of omics analyses is fundamental to understanding tissue biology1,2,3. The capacity to spatially profile DNA methylation, which is a canonical epigenetic mark extensively implicated in transcriptional regulation4,5, is lacking. Here we introduce a method for whole-genome spatial co-profiling of DNA methylation and the transcriptome of the same tissue section at near single-cell resolution. Applying this technology to mouse embryogenesis and the postnatal mouse brain resulted in rich DNA–RNA bimodal tissue maps. These maps revealed the spatial context of known methylation biology and its interplay with gene expression. The concordance and distinction in spatial patterns of the two modalities highlighted a synergistic molecular definition of cell identity in spatial programming of mammalian development and brain function. By integrating spatial maps of mouse embryos at two different developmental stages, we reconstructed the dynamics that underlie mammalian embryogenesis for both the epigenome and transcriptome, revealing details of sequence-, cell-type- and region-specific methylation-mediated transcriptional regulation. This method extends the scope of spatial omics to include DNA cytosine methylation, enabling a more comprehensive understanding of tissue biology across development and disease. DNA methylation is a key epigenetic mechanism that modulates gene expression and lineage specification by regulating chromatin structure and accessibility to transcriptional machinery4,5. This epigenetic modification varies dynamically across cell types, developmental stages and environmental conditions6. Abnormal cytosine-methylation patterns are associated with various diseases, including cancer7, autoimmune diseases8 and inflammation9. Furthermore, altered DNA-methylation levels have been observed in ageing tissues10, contributing to biomarkers and predictive clock models for chronological and biological age11. Despite advances in single-cell methylome analysis5,12, the lack of spatial information regarding intact tissues limits our understanding of gene regulation during tissue development and disease progression. Emerging spatial multi-omics technologies now enable profiling of molecular features in native tissue microenvironments, significantly enhancing our understanding of biological complexity in situ. However, current spatial methods are limited to histone modifications, chromatin accessibility, transcriptomes and selected protein panels1,2,3. Direct spatial mapping of DNA methylation has not been available, leaving this crucial epigenetic layer underexplored. Here we introduce a technology for spatial joint profiling of the DNA methylome and transcriptome (spatial-DMT) on the same tissue section at near single-cell resolution. The spatial maps uncovered intricate spatiotemporal regulatory mechanisms of gene expression in a native tissue context. Spatial-DMT investigates interactive molecular hierarchies in development, physiology and pathogenesis in a spatially resolved manner. Spatial-DMT combines microfluidic in situ barcoding1, cytosine deamination conversion12 and high-throughput next-generation sequencing to achieve spatial methylome profiling directly in tissue. The experimental workflow of spatial-DMT is illustrated in Fig. 1a,b, with a step-by-step protocol provided in Extended Data Fig. 1 and detailed reagent information provided in Supplementary Tables 1–4. A tissue section is fixed, permeabilized, treated with HCl treatment, and subjected to Tn5 transposition and reverse transcription (RT). Spatial barcodes are sequentially ligated, after which cDNA and gDNA are separated using streptavidin beads. cDNA is processed using template switching, after which a cDNA library is prepared; gDNA is processed using enzymatic methyl-seq (EM-seq) conversion and splint ligation, after which a library is constructed. b, Chemistry workflow of DNA methylation and RNA library preparation. The solid line indicates the maximum number of CpGs in the mouse genome, large dashed line indicates 10% of the total number of CpGs and small dashed line indicates 1% of the total number of CpGs. d, Box plots showing CG/CH retention rates in E11 and E13 embryos and P21 brain tissues. Number of pixels are as stated in part c for these datasets. e, Number of genes (left) and UMIs (right) per pixel. For data from previous studies1,2, the numbers of pixels are as follows (left to right): ref. For the other datasets, number of pixels are as stated in part c. The box plots show the median (centre line), first and third quartiles (box limits) and 1.5× interquartile range (whiskers). f, Spatial clusters (top) and UMAP analysis (bottom) of DNA methylation (DNAm, left), RNA transcription (RNA, middle) and integrated data (spatial DNAm + RNA (WNN), right) in E11 mouse embryo (pixel size, 50 μm; ROI (red dashed box), 5 × 5 mm2; technical replicates, n = 2). Integrated analysis reveals more-refined spatial clusters and distinct anatomical regions, including brain, spinal cord, heart and craniofacial regions. D, DNA; R, RNA; W, WNN. g, Spatial mapping and joint clustering of DNA methylation and RNA data in E11 facial and forebrain regions (pixel size, 10 μm; ROI (red dashed box), 1 × 1 mm2). h, Spatial mapping of GABAergic cortical interneurons and telencephalon cells on the basis of deconvolution of transcriptomic pixels using a scRNA-seq reference25. Cell-type distributions align with the anatomic references from the Allen developing mouse atlases55. Specifically, hydrochloric acid (HCl) was applied to fixed frozen tissue sections to disrupt the nucleosome structures and remove histones to improve Tn5 transposome accessibility for DNA-methylation profiling13. Next, Tn5 transposition was performed and adapters containing a universal ligation linker were inserted into genomic DNA (gDNA). To further reduce the size of the large gDNA fragment and improve yield, we adopted a multi-tagmentation strategy as previously described13. Specifically, we implemented two rounds of tagmentation to balance DNA yield with experimental time and minimize the risk of RNA degradation. mRNAs were then captured by the biotinylated reverse transcription primer (poly-biotinylated deoxythymidine (dT) primer with unique molecular identifiers (UMIs) and a universal linker sequence; Supplementary Table 1), followed by reverse transcription to synthesize the complementary DNA (cDNA). Both genomic fragments and cDNA in the tissue were then ligated to spatial barcodes sequentially. In brief, two sets of spatial barcodes (barcodes A1–A50 and B1–B50; Supplementary Tables 2 and 3) flowed perpendicularly to each other in the microfluidic channels. They were covalently conjugated to the universal linker through the templated ligation. This results in a two-dimensional grid of spatially barcoded tissue pixels in the region of interest (ROI), each defined by the unique combination of barcodes Ai and Bj (i = 1–50, j = 1–50; barcoded pixels, n = 2,500). Barcoded gDNA fragments and cDNA were then released after reverse crosslinking. Subsequently, cDNA was subjected to a template switching reaction and the cDNA library was constructed (Fig. 1b, right, and Extended Data Fig. gDNA was treated with enzymatic methyl-sequencing (EM-seq) conversion, splint ligation and DNA library construction (Fig. To minimize DNA damage during methylome profiling, we used enzymatic methyl-seq, an enzyme-based alternative to bisulfite conversion (Fig. In this process, modified cytosines (the sum of 5-methylcytosine and 5-hydroxymethylcytosine) were both oxidized by the ten–eleven translocation methylcytosine dioxygenase 2 (TET2) protein and protected from deamination by the APOBEC protein, whereas unmodified cytosines were deaminated to uracil. The gDNA fragments were then amplified using uracil-literate VeraSeq Ultra DNA polymerase. Because C residues on the PCR handle of barcode B were changed to T residues after deamination and PCR, we modified the P7 primer (N70X-HT) by replacing C residues with T residues to complement the A residues on the PCR handle and amplify the fragments. Barcodes have been designed such that no crosstalk under C-to-T deamination is possible. To demonstrate the ability of spatial-DMT to co-profile DNA methylation and transcription in complex tissues, we profiled mouse embryos on embryonic days 11 (E11) and 13 (E13) and mouse brain on postnatal day 21 (P21). Mouse embryos were profiled at two pixel resolutions: 50 μm for the head and upper body and 10 μm for zoomed-in mapping of the facial and forebrain region. Postnatal mouse brains were profiled at 20 μm pixel resolution. We developed a computational pipeline to preprocess and analyse the data (Supplementary Fig. DNA methylation and RNA expression had high concordance (Pearson's r = 0.9836 for DNA methylation and r = 0.9752 for RNA expression; Methods) between the replicate E11 embryo maps with matched body parts (Extended Data Fig. Uniform manifold approximation and projection (UMAP) co-embedding of the two replicate spatial maps revealed co-localization of pixels from similar body parts but from different datasets (Extended Data Fig. Notably, marker genes, including Frem1 (face), Ank3 (brain) and Trim55 (heart), showed spatial expression patterns consistent with their known tissue-specific expression profiles (Extended Data Fig. Low-signal pixels were filtered on the basis of the knee-plot cut-off threshold (Extended Data Fig. After stringent quality control, 32.2–65.7% of reads (887,671,712–1,882,630,968) were retained, yielding 355,069–753,052 reads per pixel across 1,699–2,493 pixels in the E11, E13 and P21 samples (Supplementary Table 5). On average, 136,639–281,447 CpGs were covered per pixel across E11, E13 and P21 samples (Supplementary Table 5), comparable to previous single-cell DNA-methylation studies of mouse embryos and brain samples5,10,12,14 (Fig. Duplication rates ranged from approximately 20 to 53% across samples (Extended Data Fig. The retention rate, defined as the percentage of unconverted cytosines owing to methylation or incomplete conversion12, of mitochondrial DNA was minimal (below 1%; Extended Data Fig. The CpG retention rates were consistently between 70% and 80% across all samples, whereas the methylation level of non-CpG sites (mCH; H denotes A, C or T) remained low (mCA < 1% in embryos; mCA ≈ 3–4% in the postnatal brain; Fig. The mCA level was higher in the P21 mouse brain, consistent with the known increase in non-CpG methylation in postnatal neuronal tissues5,15. Analysis of the methylation-free linker sequences showed that more than 99% of cytosines were successfully converted, further confirming conversion efficiency (Extended Data Fig. Furthermore, no evidence of RNA contamination, such as poly(A), poly(T) or template switching oligonucleotide sequences, was detected in the DNA-methylation libraries (Extended Data Fig. We further assessed the genomic distribution of CpG coverage and found it to be uniformly distributed across genomic regions (Extended Data Fig. Furthermore, methylation levels across various chromatin states were consistent with known biology and comparable to those reported in published databases5,10,12,14,16 (Extended Data Fig. For instance, DNA-methylation levels were low at transcription start sites but increased upstream and downstream of these regions (Extended Data Fig. Collectively, our approach yielded accurate and unbiased profiling of DNA methylation across the genome. Simultaneously, we generated high-quality RNA data from the same tissue sections, enabling direct comparisons between transcriptional activity and epigenetic states. Specifically, we identified expression of 23,822–28,695 genes in our spatial maps (Supplementary Table 5). At the pixel level, the average number of detected genes per pixel ranges from 1,890 (E11 embryo, 10 μm; 3,596 UMIs) to 4,626 (E11 embryo, 50 μm; 16,709 UMIs), comparable to previous spatial transcriptomic studies of the mouse embryo and brain1,2 (Fig. Lower-resolution pixels (for example, 50 μm) captured more UMIs and expressed genes, probably owing to the inclusion of more cells in each pixel. Mammalian embryogenesis is an intricately programmed process with complex gene-expression dynamics regulated by epigenetic mechanisms, including DNA methylation, at spatiotemporal scales17. We applied spatial-DMT to produce spatial tissue maps of the DNA methylation and gene expression of the E11 mouse embryo. The DNA methylome and transcriptome define cell identity independently, with pixels clustered by methylation read-outs sampled from variably methylated regions (VMRs)18 and expression levels from variably transcribed genes (Fig. The two modalities can be integrated to achieve improved discrimination of intercellular and spatial diversity using a weighted nearest neighbour (WNN) method19 (Fig. Superimposing the WNN clusters over histological images suggests clear correspondence with anatomical structures, for example, craniofacial region (W0), hindbrain–spinal cord (W2) and embryonic heart (W6), consistent with the tissue histology (Fig. The correlations between single-modality (RNA and DNA methylation) clusters and their integrated WNN clustering results were evaluated (Fig. Each modality captured distinct yet complementary aspects of cellular identity and their integration through WNN analysis yielded clusters with enhanced resolution. RNA-expression profiles had a broader dynamic range, which may result in distinct clustering granularity across modalities. To quantify the relative contributions of each modality to the integrated clusters, we computed modality weights for individual spatial pixels (Extended Data Fig. This analysis revealed that some clusters were defined predominantly by gene expression (for example, W6, cardiac tissue), whereas others by DNA methylation (for example, W11, craniofacial region). Collectively, these results highlight the advantage of spatial multi-omics integration in resolving cellular heterogeneity missed by single-modality analyses. We examined VMR methylations specific to the brain (W2), craniofacial (W0) and heart (W6) regions (Fig. 1f) and neighbouring gene expression (Fig. 2a,b, cluster W2; Extended Data Fig. 6a,b, cluster W0; and Extended Data Fig. Spatial cluster-specific gene expression is frequently associated with low DNA methylation at the neighbouring VMRs, as exemplified by signature genes Runx2, Mapt, and Trim55, which mark the craniofacial regions (jaw and upper nasal), the brain and spinal cord, and the heart, respectively (Fig. 2c and see Extended Data Figs. 5a, 6c and 7c for additional signature gene examples). Testing the epigenetically regulated genes from each cluster for Gene Ontology (GO) enrichment identified the developmental process related to the anatomical structures (Extended Data Figs. Although the canonical negative correlations between DNA methylation and RNA expression have been observed in many genes, we also identified genes, for example, Ank3, Atp11c, Cyfip2, Lmln and Khdrbs2, for which expression is positively correlated with the methylation levels of the associated VMRs (Fig. For instance, Ank3, primarily located at the axonal initial segment and nodes of Ranvier in neurons of the central and peripheral nervous systems20, had high levels of expression and DNA methylation in the brain region (Fig. The positive correlations between DNA methylation and RNA expression have been documented at enhancers21, gene bodies22 and Polycomb targets23, suggesting a complex mechanism of DNA methylation in transcriptional regulation, contingent on its interaction with transcriptional factor (TF) binding and histone modifications. a, Heat map of DNA methylation levels for the top ten differentially methylated genomic loci in brain and spinal cord (cluster W2) of E11 embryo. Each row represents a specific genomic locus and each column represents a different cluster. b, Heat map of expression levels for nearby genes corresponding to the genomic loci in a. Each row represents a specific gene and each column represents a different cluster. c, UMAP visualization and spatial mapping of DNA methylation (methylation percentage) and RNA-expression levels (log-normalized expression) for selected marker genes across different clusters. d, Correlation analysis between DNA methylation and RNA expression of nearby genes. Negative correlations (blue) indicate a repressive effect of methylation, whereas positive correlations (orange) indicate activation. Correlation coefficient and P value were calculated using the two-sided Pearson correlation test and P values were adjusted for multiple comparisons using the Benjamini–Hochberg method. e, Scatterplots of enriched TF motifs in the respective clusters. To directly assess the influence of cell-type-specific methylation on TF binding, we performed motif enrichment analysis on differentially hypomethylated VMRs and examined the gene expression of each TF gene. Our findings uncovered a relationship between the expression of TFs and the DNA methylation at their binding sites. For example, TFs associated with heart development, Hand2, Tbx20 and Meis1, were expressed in cluster W6, the corresponding hypomethylated VMRs of which were enriched in the binding motifs of these TFs (Fig. Similar sets of tissue-specific TFs are identified for other embryo structures, for example, Ebf1 and Pbx1 in the brain and spinal cord and Sox9, Ebf1 and Zeb2 in the craniofacial region (Fig. Notably, Ebf1 was expressed and its binding motif was also enriched across all three tissue regions (Fig. EBF1 was reported as an interaction partner for TET2 and was suggested as a sequence-specific mechanism for regulating DNA methylation in cancer24. To precisely resolve the fine structure of transcriptional regulation in the craniofacial and forebrain regions of the E11 embryo, we used a 10 μm pixel size microfluidic chip to produce a spatial map with near single-cell resolution (Fig. 1g,h and Extended Data Fig. Integration of our spatial dataset with a single-cell reference25 revealed a strong concordance between the two datasets (Extended Data Fig. Cell-type deconvolution using the single-cell RNA-sequencing (scRNA-seq) reference25 identified distinct clusters that correspond precisely to known anatomical structures of the developing mouse brain. Notably, two spatially defined clusters, W7 and W11, captured key telencephalic compartments. W11 was enriched for telencephalon progenitors in the ventricular zone of the pallium, a neurogenic niche characterized by active cell division and proliferation, whereas W7 corresponded to γ-aminobutyric-acid-releasing (GABAergic) cortical interneurons localized in the mantle zone, where newborn neurons migrate, accumulate and differentiate to establish cortical architecture26 (Fig. GO enrichment analysis further supported these regional identities, highlighting biological processes associated with neurogenesis and progenitor proliferation in W11, and neuron projection and migration in W7 (Extended Data Fig. Moreover, the cell-type lineage tree constructed from the scRNA-seq reference confirmed the developmental trajectory, positioning telencephalon progenitors as direct precursors to GABAergic cortical interneurons25. Beyond the forebrain, our spatial analysis also resolved refined sensory structures in the developing olfactory system (cluster W5 and W10; Fig. Sensory neurons were notably enriched in W5 (Extended Data Fig. 8f,g), spatially localized adjacent to the forebrain. This spatial pattern aligns closely with the established developmental trajectory of the olfactory system, in which olfactory sensory neurons progressively form connections with the forebrain as embryogenesis progresses27. Together, these findings provide a high-resolution view of neural and sensory system formation, underscoring the power of spatial-DMT in resolving intricate anatomical structures and capturing the spatiotemporal dynamics. Mammalian embryogenesis is a precisely timed process with dynamic DNA methylation and gene expression underlying cell differentiation and tissue development16. By using pseudotime and spatial-DMT analyses on embryos of two different gestational ages (E11 and E13), we can investigate the dynamics of DNA methylation, gene expression and their interactions at both spatial and temporal scales. We first analysed the developing brain, focusing on the differentiation trajectory from oligodendrocyte progenitors to premature oligodendrocytes. This pseudotemporal process is associated with coupled DNA methylation and gene expression in different patterns. For example, loss of DNA methylation can be both associated with gene activation (for example, Nrg3, an oligodendrocyte marker28) and silencing (for example, Pdgfra, an oligodendrocyte precursor marker29; Fig. a, Spatial mapping of oligodendrocyte progenitors (left) and premature oligodendrocytes (middle) identified by label transfer from mouse embryo scRNA-seq data17 to spatial-DMT, with pseudotemporal reconstruction of the oligodendrogenesis plotted in space (right). b, Pseudotime heat maps of VMRs that become demethylated (left) and expression changes of nearby genes from oligodendrocyte progenitors to premature oligodendrocytes (right). Each row represents a specific genomic locus (left) and nearby genes (right), with columns representing tissue pixels ordered by pseudotime. The colour scale indicates the z scores of DNA methylation and gene expression. c,d, UMAP visualization (c) and spatial distribution (d) of integrated E11 and E13 WNN analysis. Spatial tissue pixels from different developmental stages conform well and match with the tissue types. e,f, Comparative analysis of RNA-expression levels (log-normalized expression) with two-sided Wilcoxon rank-sum test, unadjusted P = 8.47 × 10−36, P = 7.62 × 10−54, P = 4.00 × 10−37 from top to bottom (e) and DNA methylation (methylation percentage) with two-sided Wilcoxon rank-sum test, unadjusted P = 2.07 × 10–82, P = 5.45 × 10−82, P = 8.55 × 10−80 from top to bottom (f) for upregulated genes in E13 brain and spinal cord. g, GO enrichment analysis from one-sided hypergeometric test of biological processes related to demethylated and upregulated genes in the brain and spinal cord from E11 to E13 stages. h, Spatial mapping of the expression levels (log-normalized expression) of DNA-methylation-related enzymes in the brain and spinal cord regions across E11 and E13 stages. To further illustrate the temporal molecular dynamics of embryonic development, we performed spatial mapping of the E13 mouse embryo (Extended Data Fig. First, we validated our RNA dataset by integrating it with a published spatial ATAC–RNA (ATAC, assay for transposase-accessible chromatin) reference2, which revealed a strong concordance (Supplementary Fig. Co-clustering the two datasets identified 11 distinct cluster populations, each corresponding to the histological location in both samples (Supplementary Fig. By integrating spatial data from E11 and E13 embryos (Fig. 3c,d), we identified genes upregulated in E13 (Fig. 9b,d, left) associated with notable loss of DNA methylation (Fig. These genes are implicated in the corresponding tissue functions. For example, Usp9x, Ank3 and Shank2 are critically involved in neuronal development, synaptic organization, morphogenesis and transmission30,31,32 (Fig. Ctnna1, Pecam1 and Lamb1 are pivotal for maintaining cardiac tissue integrity33, vascular development34 and cardiac tissue structuring35, respectively (Extended Data Fig. Functional analysis of upregulated genes further corroborated their association with biological processes, for example, synapse organization, neuron projection organization and dendritic spine morphogenesis in the brain (Fig. 3g), and sphingolipid metabolic processes in the heart (Extended Data Fig. Our tissue map data more precisely timed these gene and pathway activations to a specific stage of embryo development, tissue location and shed light on their epigenetic regulatory mechanisms. Notably, besides regulators of tissue development, some DNA methylation writers36, readers37 and eraser enzymes38, for example, Dnmt1, Dnmt3a, Mecp2 and Tet1, showed higher expression in the E13 embryo, suggesting elevated biochemical activity that drove the global DNA methylation dynamics (Fig. mCH methylation, particularly mCA, is uniquely abundant in the brain5,15. To evaluate the spatial heterogeneity of non-CpG cytosine methylation, we applied spatial-DMT to the cortical and hippocampal regions of a P21 mouse brain (Fig. Initial analysis of global mCA and mCG levels revealed relatively low methylation in the dentate gyrus (DG), cornu ammonis (CA)1/2 and CA3 regions, compared with the cortex (Extended Data Fig. a, Bright-field image of the P21 mouse brain section (pixel size, 20 μm; ROI area, 2 × 2 mm2) showing the analysed regions, including the DG, CA1, CA2, CA3 and cortex (n = 1). b, Spatial distribution of clusters identified in DNA-methylation data (left), RNA-transcription data (middle) and integrated DNA and RNA data using WNN analysis (right). c–e, Scatter plot displaying the relationship between CpG and CpA methylation changes (log-transformed fold change) and gene-expression changes (log-transformed fold change; gene exp. (log fold)) for marker genes in DG (c), CA1/2 (d) and CA3 (e). Each dot represents a specific gene, coloured by the log-fold change in gene expression, with red indicating upregulation and blue indicating downregulation. These plots demonstrate how differential methylation at CpG and CpA sites correlates with changes in gene expression, illustrating the complex regulatory landscape across different brain regions. f–h, Spatial mapping of CpG (left) and CpA (middle) methylation and RNA-expression levels (right) for selected genes in different brain regions. The spatial distribution of methylome and transcriptome clusters matched anatomical structures shown by the histology image, reflecting the arealization of this brain region (Fig. Expression of known cell-specific marker genes is spatially distributed in regions in which these cell types were enriched. For instance, Prox1, a TF crucial for neurogenesis and the maintenance of granule cell identity39, was prominently expressed in the DG (Fig. Satb1, which has a role in cortical neuron differentiation and layer formation40, was strongly enriched in the cerebral cortex (Extended Data Fig. Bcl11b, a TF essential for neuronal progenitor cell differentiation41, demonstrated elevated expression in the hippocampus (Extended Data Fig. To elucidate the regulatory roles of mCG and mCA on gene transcription, we compared the two modifications for the signature genes of each cluster against all other clusters combined. By correlating the modification levels with gene expression, we identified genes potentially regulated by mCG, mCA or both (Fig. 4c,f and Extended Data Fig. 10c), broadly marking the DG and CA1/2 regions, respectively. By contrast, Ntrk3, a receptor tyrosine kinase crucial for nervous system function42, was highly expressed in the hippocampal CA1/2 and DG regions, correlating primarily with mCG levels but not with mCA (Fig. Similarly, Satb1 expression in the cortex was strongly correlated with mCG but not with mCA levels (Extended Data Fig. The silencing of Cux1, a TF involved in neuronal development and function43, had a negative correlation with CA and CG hypermethylation in the CA3 region. By contrast, in the CA1/2 region, Cux1 expression showed a negative correlation only with CA hypermethylation and seemed independent of mCG levels (Fig. Across both sequence contexts, negative correlations between DNA methylation and gene expression are more prevalent than positive ones, highlighting the predominantly repressive nature of these epigenetic modifications (Extended Data Fig. Collectively, mCG and mCA regulate transcription in a gene-specific manner, jointly defining the cell identity across cell types and brain regions. For example, Syt1 and Rbfox3 are broadly expressed across neurons in all cortical layers (Supplementary Fig. By contrast, Cux2, Cux1 and Satb2 were highly expressed in upper-layer neurons, whereas Bcl11b was enriched in the deeper cortical layers (Supplementary Fig. To further resolve cell identities, we integrated our spatial-DMT dataset with a reference scRNA-seq dataset44 to associate spatial clusters with previously defined cell types. For instance, W0, W3 and W5 were identified as oligodendrocytes, DG granule neurons and telencephalon-projecting excitatory neurons, respectively (Supplementary Fig. Aggregated gene-expression and DNA-methylation levels across pixels in each cluster were highly correlated with the corresponding scRNA-seq transcriptomes44 (Supplementary Fig. 4d) and single-cell DNA methylome profiles5 (Supplementary Fig. 4e), as illustrated by W3 (DG) and W5 (CA1) (Fig. Cell-type deconvolution using the same scRNA-seq reference44 revealed spatial organization of diverse cell types, consistent with known brain anatomy (Supplementary Fig. For example, cortical excitatory neurons showed expected laminar distribution: TEGLU7, TEGLU8, TEGLU4 and TEGLU3 were enriched in layers 2/3, 4, 5 and 6, respectively. In the hippocampus, TEGLU24 and TEGLU23 mapped to CA1/2 and CA3 regions, whereas in granule neurons, DGGRC2 localized to the DG. Furthermore, DEGLU1 was enriched in the thalamic region. All cell types were mapped to expected brain anatomical regions in agreement with the scRNA-seq reference44. These observations underscore the robustness of spatial-DMT in resolving complex tissue architectures and simultaneously profiling the epigenome and transcriptome with their spatial context, which remains challenging for conventional anatomical and single-cell approaches. The spatially resolved methylation landscape enables the identification of subtle, region-specific epigenetic variations that may otherwise be undetected. To illustrate this, we performed regional differential methylation analysis, comparing the ventricular and mantle zone of the hindbrain–spinal cord (Fig. This analysis identified differential methylation at promoters and enhancers specific to neural progenitor cells, as shown by the enrichment of corresponding methylated histone H3 K4 (H3K4me1). These differential methylated loci co-localized with enhancers functionally implicated in brain and neural tube development (mEnhA9), which are associated with TF binding sites crucial for neuronal differentiation, including for the TFs FOXO4 (ref. a, Differential region analysis between adjacent WNN cell clusters (left). Chromatin features enriched at hypomethylated genome loci in the blue region (middle) and those enriched in the red region (right) are shown. A summary of the ChromHMMfullStack (full-stack chromatin hidden Markov model) annotation is provided in Supplementary Table 6. HM, histone modification; TFBSrm, transcription factor binding sites from roadmap dataset. b, Differential region analysis in the same WNN (W7) cell cluster but across spatially distant locations (left). Chromatin features enriched at hypomethylated loci in the blue (middle) and red (right) regions are shown. False discovery rate (FDR) from one-sided Fisher's exact test. f, Enriched TF motifs from Homer one-sided hypergeometric test q value (on the basis of Benjamini–Hochberg procedure) in differentially hypomethylated VMRs between D0 and D4 cluster pixels derived from cluster R3 in the E11 embryo (two-sided Wilcoxon rank-sum test, adjusted P value on the basis of Bonferroni correction). g, Differentially expressed genes corresponding to the D0 and D4 pixels originated from the R3 cluster in the E11 embryo (Wilcoxon rank-sum test (two-sided), adjusted P value on the basis of Bonferroni correction). Spatial methylation profiles may reveal subtle differences between distantly located pixels in the same WNN cell cluster. We examined two W7 pixel groups located in the forebrain and spinal cord, respectively (Fig. In the forebrain, regions with loss of methylation were enriched for binding sites of FOXI1, TFs crucial for auditory development48. By contrast, spinal-cord-specific hypomethylation correlated with occupancy by TLX1, a TF essential for spinal cord maturation and neuronal differentiation49. These findings demonstrate that spatial-DMT can resolve subtle epigenetic heterogeneity across distinct tissue microenvironments. Besides capturing regional TF-mediated cell state differences, spatial-DMT enables the analysis of diverse cell traits through other methylation-based features. For example, partially methylated domains (PMDs), which lose methylation over successive mitotic divisions, serve as indicators of mitotic activity50. By spatially mapping PMD methylation in E11 and E13 embryos (Fig. 5c,d), we identified distinct regional patterns. For instance, regions such as the forebrain and hindbrain–spinal cord had higher PMD methylation, whereas embryonic heart tissue demonstrated lower PMD methylation levels, reflecting active cardiogenesis at these developmental stages51 (Fig. These gradients potentially reflect the spatial organization of progenitor cells and their differentiation trajectories. 5e), cortical layers displayed higher PMD methylation, consistent with reduced proliferative capacity typical of differentiated neurons52. By contrast, the DG has comparatively lower PMD methylation, consistent with the presence of neural stem and progenitor cells in the subgranular zone that continue to undergo mitotic division and neurogenesis53. Finally, the spatially resolved variations in the methylome and transcriptome can complement each other in distinguishing differentiated cell states (Fig. Despite originating from the same RNA-defined clusters (R3), DNA methylation-defined cluster 0 (D0) and 4 (D4) cells had distinct subpopulations when stratified by their VMRs. Motif enrichment analysis of low-methylation sites in D0 and D4 identified regulatory elements associated with facial and cardiac morphogenesis (Fig. 5f), whereas corresponding gene-expression changes were limited (Fig. This may reflect epigenetically primed subpopulations that share similar transcriptional states. Together, these findings highlight the power of spatial-DMT in delineating cell identity beyond the resolution of transcriptomic analysis alone. Understanding how epigenetic regulation is spatially organized in tissues remains a major challenge. Although single-cell DNA methylation and transcriptomic technologies have revealed diverse regulatory architectures in both health and disease4,5,16, they require tissue dissociation, disrupting the native context. Here we developed spatial-DMT, a pioneering spatial multi-omics technology that enables simultaneous profiling of DNA methylation and RNA expression in the same intact tissue sections, preserving cellular and anatomical architectures. Spatial-DMT generated high-quality data, with strong reproducibility across technical replicates and concordance with established single-cell and spatial transcriptomic references. When applied to mouse embryos at E11 and E13, it revealed methylome dynamics during embryogenesis. In the postnatal brain, it resolved the neuronal mCH landscape, consistent with elevated mCA levels in mature neurons5. With 10 μm resolution, spatial-DMT approaches single-cell granularity, enabling precise delineation of fine tissue structures and region-specific epigenetic features that are typically obscured by tissue dissociation. Our data indicate that gene expression may be differentially influenced by mCG and mCA for different cell types in a spatially restricted manner, potentially owing to the distinct genomic distribution of these methylation marks. Although the observed associations are correlative, they offer valuable insights into the complex interplay between regulatory elements on gene expression. This integrative data analysis also demonstrated that each modality captures distinct yet complementary aspects of cellular states, enhancing cell state differentiation beyond what is achievable with single-modality approaches. This may reflect regulatory redundancy, in which distinct epigenetic mechanisms converge to produce similar transcriptional outcomes. Conversely, the opposite scenario is equally plausible—transcriptional states may diverge despite similar epigenetic landscapes, as epigenetic regulation represents only one layer influencing gene expression. As a result, spatial‐DMT offers a cost-effective strategy for high-resolution, unbiased genome-wide methylome analysis. Future development of spatial-DMT may incorporate additional molecular modalities, including chromatin conformation (HiC), accessibility (ATAC-seq), histone modifications (CUT&Tag), metabolome (mass spectrometry imaging) and surface proteins (CITE-seq), to enhance the resolution of gene regulatory mechanisms in situ. The current enzymatic approach does not distinguish between 5-methylcytosine and 5-hydroxymethylcytosine, this limitation may be overcome by using emerging methods that enable simultaneous measurement of the full spectrum of cytosine base modifications54. Coupling our method with long-read sequencing technologies, such as Nanopore or PacBio, could simplify detection and improve coverage of epigenetic modifications. Adaptation of the protocol to formalin-fixed paraffin-embedded (FFPE) specimens would broaden its clinical use, whereas optimizing tagmentation and data-processing pipelines will further improve sensitivity and interpretability. The development of advanced spatial computational deconvolution strategies is essential to resolve signals from lower-resolution pixels, minimize potential biases and enhance biological interpretability. Broader application across tissues, developmental stages and species will further establish the generalizability of spatial-DMT for spatial multi-omics profiling in both basic and translational research. In summary, spatial-DMT advances spatial omics analyses by enabling the simultaneous mapping of DNA methylation and transcription in the native tissue context. This approach provides a powerful framework for exploring epigenetic regulation, facilitates the discovery of methylation biomarkers, deepens our understanding of disease mechanisms and benefits both fundamental and translational research. Mouse C57 embryo sagittal frozen sections (MF-104-11-C57 and MF-104-13-C57) were purchased from Zyagen. Freshly collected E11 or E13 mouse embryos were snap frozen in optimal cutting temperature (O.C.T.) compounds and sectioned at 7–10 μm thickness. Tissue sections were collected on poly-l-lysine-coated glass slides (Electron Microscopy Sciences, 63478-AS). Juvenile mouse brain tissue (P21) was obtained from the C57BL/6 mice housed in the University of Pennsylvania Animal Care Facilities under pathogen-free conditions. All procedures used were approved by the Institutional Animal Care and Use Committee. Mice were euthanized at P21 using CO2 inhalation, followed by transcranial perfusion with cold Dulbecco's PBS (DPBS). After isolation, brains were embedded in Tissue-Tek O.C.T. Coronal cryosections of 8–10 μm were mounted on the back of Superfrost Plus microscope slides (Fisher Scientific, 12-550-15). Unloaded Tn5 transposome (C01070010) was purchased from Diagenode and the transposome was assembled following the manufacturer's guidelines. The oligonucleotides used for transposome assembly were: Tn5ME-B, 5′-/5Phos/CATCGGCGTACGACTAGATGTGTATAAGAGACAG-3′; Tn5MErev, 5′-/5Phos/CTGTCTCTTATACACATCT-3′. DNA oligonucleotides used for PCR and library construction are shown in Supplementary Table 1. All DNA barcode sequences are provided in Supplementary Tables 2 (barcode A) and 3 (barcode B) and all other chemicals and reagents are listed in Supplementary Table 4. Chrome photomasks were purchased from Front Range Photomasks, with a channel width of either 20 or 50 μm. The moulds for polydimethylsiloxane (PDMS) microfluidic devices were fabricated using standard photolithography. The manufacturer's guidelines were followed to spin-coat SU-8-negative photoresist (Microchem, SU-2025 and SU-2010) onto a silicon wafer (WaferPro, C04004). The heights of the features were about 20 and 50 μm for 20- and 50-μm-wide devices, respectively. PDMS microfluidic devices were fabricated using the SU-8 moulds. After degassing for 30 min, the mixture was cured at 66–70 °C for 2–16 h. Solidified PDMS was extracted from the moulds for further use. Frozen tissue slides were quickly thawed for 1 min in a 37 °C incubator. The tissue was fixed with 1% formaldehyde in PBS containing 0.05 U ml−1 RNase inhibitor (Enzymatics) for 10 min and quenched with 1.25 M glycine for another 5 min at room temperature. After fixation, tissue was washed twice with 1 ml of DPBS–RNase inhibitor and cleaned with deionized H2O. The tissue was subsequently permeabilized with 100 μl of 0.5% Triton X-100 plus 0.05 U ml−1 RNase inhibitor for 30 min at room temperature, then washed twice with 200 μl DPBS–RNase inhibitor for 5 min each. After permeabilization, the tissue was treated with 100 μl of 0.1 N HCl for 5 min at room temperature to disrupt histones from the chromatin, then washed twice with 200 μl of wash buffer (10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM MgCl2, 1% BSA and 0.1% Tween 20) plus 0.05 U ml−1 RNase inhibitor for 5 min at room temperature. After 60 min incubation, the first round of transposition mixture was removed and a second round of 50 μl of fresh transposition mixture was added and incubated for another 60 min at 37 °C. To stop the transposition, 200 μl of 40 mM EDTA with 0.05 U ml−1 RNase inhibitor was added with incubation for 5 min at room temperature. After that, 200 μl 1× NEB3.1 buffer plus 1% RNase inhibitor was used to wash the tissue for 5 min at room temperature. The tissue was then washed again with 200 μl of DPBS–RNase inhibitor for 5 min at room temperature before proceeding with the in situ reverse transcription reaction. The tissue was incubated for 30 min at room temperature, then at 45 °C for 90 min in a humidified container. After the reverse transcription reaction, tissue was washed with 1× NEB3.1 buffer plus 1% RNase inhibitor for 5 min at room temperature. For in situ ligation with the first barcode (barcode A), the first PDMS chip was covered at the tissue ROI. For alignment purposes, a 10× objective (KEYENCE BZ-X800 fluorescence microscope, BZ-X800 Viewer Software) was used to take the bright-field image. The PDMS device and tissue slide were clamped tightly with a custom acrylic clamp. Barcode A was first annealed with ligation linker 1 by mixing 10 μl of 100 μM ligation linker, 10 μl of 100 μM individual barcode A and 20 μl of 2× annealing buffer (20 mM Tris-HCl pH 7.5–8.0, 100 mM NaCl2 and 2 mM EDTA). Then the PDMS chip and clamp were removed after washing the tissue with 1× NEB 3.1 buffer for 5 min. The slide was then washed with deionized water and dried using compressed air. For in situ ligation with the second barcode (barcode B), the second PDMS chip was covered at the ROI and a bright-field image was taken with the 10× objective. An acrylic clamp was applied to clamp the PDMS and tissue slide together. Annealing of barcode B (B1–B50, 25 μM) and preparation of the ligation mixture are the same as barcode A. The whole device was incubated at 37 °C for 30 min in a humidified container. The PDMS chip and clamp were then removed, and the slide was washed with deionized water and dried using compressed air. A bright-field image was then taken for further alignment. For tissue lysis, the ROI was digested with 100 μl of the reverse crosslinking mixture (0.4 mg ml−1 proteinase K, 1 mM EDTA, 50 mM Tris-HCl pH 8.0, 200 mM NaCl and 1% SDS) at 58–60 °C for 2 h in a humidified container. The lysate was then collected in a 0.2-ml PCR tube and incubated on a 60 °C shaker overnight. For gDNA and cDNA separation, the lysate was purified with Zymo DNA Clean and Concentrator kit and eluted with 100 μl nuclease-free water. After washing, beads were resuspended in 100 μl of 2× B&W buffer (50 μl 1 M Tris-HCl pH 8.0, 2,000 μl 5 M NaCl, 10 μl 0.5 M EDTA and 2,940 μl nuclease-free water) containing 2 μl of SUPERase In RNase inhibitor, then mixed with the gDNA–cDNA lysate and allowed to bind for 1 h with agitation at room temperature. Supernatant (200 μl) was collected from the above separation process for further methylated gDNA detection and library construction. Next, 1 ml of DNA binding buffer was added to the 200 μl supernatant and purified with the Zymo DNA Clean and Concentrator kit again, then eluted in 84 μl (3 × 28 μl) nuclease-free water. The NEBNext enzymatic methyl-seq conversion module (EM-seq) was then used to detect methylated DNA in the sample by converting unmethylated cytosines to uracil; the manufacturer's guidelines were followed. In brief, 5 μl of diluted 1:1,300 of 500 mM Fe (II) solution was added to the mixture and incubated for 1 h at 37 °C in a thermocycler. The sample was then incubated for another 30 min at 37 °C. TET2 converted DNA was then purified with 90 μl of solid-phase reversible immobilization (SPRI) beads and eluted with 16 μl nuclease-free water. The thermocycler was preheated to 85 °C, 4 μl formamide was added to the converted DNA and incubated for 10 min at 85 °C in the preheated thermocycler. After the reaction, the heated sample was immediately placed on ice to maintain the open chromatin structure, then reagents from the kit were added (68 μl nuclease-free water, 10 μl APOBEC reaction buffer, 1 μl BSA and 1 μl APOBEC) to deaminate unmethylated cytosines to uracil for 3 h at 37 °C in a thermocycler. Deaminated DNA was then cleaned up using 100 μl (1:1 ratio) of SPRI beads and eluted in 20 μl nuclease-free water. The gDNA tube was heat-shocked for 3 min at 95 °C and immediately put on ice for 2 min. Then, 10 μl of 0.75 μM pre-annealed Splint Ligate P5 (SLP5) adapter was added. This adapter was diluted from a 12 μM stock, which contained 6 μl of 100 μΜ SLP5RC oligo, 8.4 μl of 100 μΜ SLS5ME-A-H10 oligo, 5 μl of 10× T4 RNA Ligase Buffer and 30.6 μl nuclease-free water in a PCR tube that was incubated at 95 °C for 1 min, then gradually cooled by −0.1 °C s−1 to 10 °C on a thermocycler. Next, 80 μl of ligation master mixture was added to the gDNA tube at room temperature. The splint ligation mixture was then splinted into five 0.2-ml PCR tubes, 20 μl per tube. The tubes were shaken at 1,000 rpm for 10 s and spun down, then incubated for 45 min at 37 °C, followed by 20 min at 65 °C to inactivate the ligase. For splint ligation indexing PCR, 80 μl of the PCR reaction mixture was mixed in each splint ligated tube. The mixture was then aliquoted into a clean PCR tube with 50 μl volume and run on a thermocycler with the setting below, 98 °C for 1 min, then cycling at 98 °C for 10 s, 57 °C for 20 s and 72 °C for 30 s, for 13–19 cycles, followed by 72 °C for 10 s. The reaction was removed once the quantitative PCR (qPCR) signal began to plateau. The amplified PCR products were pooled and purified with a 0.8× volume ratio of SPRI beads (bead-to-sample ratio) and the completed DNA library was eluted in 15 μl nuclease-free water. In brief, 400 μl of 1× B&W buffer with 0.05% Tween 20 was used to wash the beads twice. Then, the beads were washed once with 400 μl of 10 mM Tris-HCl pH 8.0 containing 0.1% Tween 20 for 5 min at room temperature. Streptavidin beads with bound cDNA molecules were placed on a magnetic rack and washed once with 250 μl nuclease-free water before being resuspended in a template switching oligonucleotide solution (44 μl 5× Maxima reverse transcription buffer, 44 μl of 20% Ficoll PM-400 solution, 22 μl of dNTPs, 5.5 μl of 100 mM template switch oligo, 11 μl Maxima H Minus reverse transcriptase, 5.5 μl of RNase inhibitor (Enzymatics) and 88 μl nuclease-free water). Resuspended beads were then incubated for 30 min with agitation at room temperature and for 90 min at 42 °C, with gentle agitation. After the reaction, beads were washed with 400 μl of 10 mM Tris pH 8.0 containing 0.1% Tween 20 and washed without resuspension in 250 μl nuclease-free water. Water was removed on the magnetic rack and the beads were resuspended in the PCR solution (100 μl of 2× Kappa Master mix, 8.8 μl of 10 μM primers 1 and 2 and 92.4 μl nuclease-free water). Next, the beads were mixed well and 50 μl of the PCR mixture was split into four 0.2-ml PCR tubes. After five cycles of PCR reaction, four PCR tubes were placed on a magnetic rack and 50 μl of the clear PCR solution was transferred to four optical-grade qPCR tubes, adding 2.5 μl of 20× Evagreen dye to each tube. The amplified PCR product was purified with a 0.8× volume ratio of SPRI beads and eluted in 20 μl nuclease-free water. A Nextera XT DNA Library Prep Kit was used for cDNA library preparation. In brief, 2 μl (2 ng) of purified cDNA (1 ng μl−1), 10 μl Tagment DNA buffer, 5 μl Amplicon Tagment mix and 3 μl nuclease-free water were mixed and incubated at 55 °C for 5 min. Then, 5 μl NT buffer was added to stop the reaction with incubation at room temperature for 5 min. Master mix, 1 μl of 10 μM P5 primer (N501) and 1 μl of 10 μM indexed P7 primer (N70X) and 8 μl nuclease-free water) was added. The PCR product was then purified with a 0.7× ratio of SPRI beads and eluted in 15 μl nuclease-free water to obtain the cDNA library. An Agilent Bioanalyzer D5000 ScreenTape was used to determine the size distribution and concentration of the library before sequencing. Next-generation sequencing was conducted on an Illumina NovaSeq 6000 sequencer and NovaSeq X Plus system (150 bp paired-end mode). For RNA-sequencing data, Read 2 was processed to extract barcode A, barcode B and the UMIs. Using the STARsolo pipeline56 (v.2.7.10b), these processed data were mapped to the mouse genome reference (mm10). This step generated a gene matrix that captures both gene-expression and spatial-positioning information, encoded through the combination of barcodes A and B. The gene matrix was then imported into R for downstream spatial transcriptomic analysis using Seurat package (v.5.1.0)57. For DNA-methylation data, adaptor sequences were trimmed before demultiplexing the FASTQ files using the combination of barcodes A and B. Methylation levels at individual CG and CH sites were stored as continuous values between 0 and 1, representing the fraction of methylated reads after quality filtering. These processed CG–CH files were then analysed independently using the MethSCAn pipeline18 to identify VMRs59, defined as fused genome intervals with methylation-level variance in the top 2%. We used default parameter settings when running MethSCAn, and the MethSCAn filter min-sites parameter was determined from the read coverage knee plot (Extended Data Fig. The methylation levels and residuals of VMRs were then imported into R for downstream DNA-methylation analysis. We mapped the exact location of pixels on the bright-field tissue image using a custom Python script (https://github.com/zhou-lab/Spatial-DMT-2024/tree/main/Data_preprocess/Image), before removing additional empty barcodes on the basis of read-count thresholds determined by the knee plot (Extended Data Fig. Clustering and data visualization were conducted using R in RStudio. For RNA data, we used the SCTtransform function in the Seurat package (v.5.1.0), built using a regularized negative binomial model, for normalization and variance stabilization. Dimensionality reduction was performed using RunPCA function with the SCTtransformed assay. Finally, a UMAP embedding was computed using the same principal components with RunUMAP function. Owing to the inherent sparsity of DNA-methylation data, it is impractical to analyse methylation status solely at the individual CpG level. Binary information at sparse loci cannot be used directly to construct a feature matrix suitable for downstream analysis. In our study, we adopted the VMR framework, which divides the genome into variable-sized tiles and calculates the average methylation level across CpGs in each tile for each pixel18. This approach results in a continuous-valued matrix, in which rows correspond to pixels and columns represent genomic tiles, with values ranging from 0 to 1. VMR methylation levels and residuals were then imputed using the iterative principal component analysis approach as suggested in the MethSCAn instructions. Initially, missing residual values were replaced with zero and missing methylation levels were replaced with the average values for that VMR interval. The principal component analysis approach was iteratively applied until updated values stabilized to a threshold. The imputed residual matrix for VMRs was then imported into the existing Seurat object as another modality. The first ten principal components from the residual matrix were used for clustering and UMAP embedding. To visualize clusters in their spatial locations, the SpatialDimPlot function was used after clustering on the basis of gene expression or VMR residuals. UMAP embedding was visualized with the DimPlot function. The FindMarkers function was applied to select genes and VMRs that were differentially expressed or methylated for each cluster. For spatial mapping of individual VMR methylation levels or gene expression, we applied the smoothScoresNN function from the FigR package60. The SpatialFeaturePlot function was then used to visualize VMR methylation levels and gene expression across all pixels. To illustrate the relationships between clustering results from different modalities, we generated the confusion matrix and alluvial diagram using the pheatmap and ggalluvial R package61. To integrate spatial DNA methylation and RNA data, WNN analysis in Seurat was applied using the FindMultiModalNeighbors function19. On the basis of the WNN graph, clustering, UMAP embedding and spatial mapping of identified clusters were performed for integrated visualization. For the integration of spatial transcriptomics data of E11 and E13 mouse embryos, the top 3,000 integration features were selected, followed by the use of PrepSCTIntegration and IntegrateData functions to generate an integrated dataset. Similarly, to integrate with public single-cell transcriptomic data25,44, we first identified anchors using the FindIntegrationAnchors function in Seurat, followed by data integration using the IntegrateData function. To integrate DNA-methylation data, common VMRs between both developmental stages were obtained and the integrated CCA method from the IntegrateLayers function was used to join the methylation data from the two developmental stages. A Wilcoxon signed-rank test was performed to compare the methylation levels and gene-expression differences between the two time points. To perform TF motif enrichment, we first used the MethSCAn diff function on distinct groups of cells to identify differentially methylated VMRs on the basis of the clustering assignment. The HOMER62 findMotifsGenome function was then applied to analyse the enrichment of known TF motifs using its default database. We followed the same parameter settings used in MethSCAn, with motif lengths of 5, 6, 7, 8, 9, 10, 11 and 12. Enrichment analysis of individual CpGs in the differential regions (Fig. 5a,b) was performed using knowYourCG (https://github.com/zhou-lab/knowYourCG), which provides a comprehensive annotation database for each CpG, including chromatin states, TF binding sites, motif occurrences, PMD annotations and more. To avoid inflated odds ratios for high-coverage data, genomic uniformity was quantified using fold enrichment, defined as the ratio of observed overlaps to expected overlaps. The expected number of overlaps was calculated as: (number of CpGs sequenced × number of CpGs in the chromatin state feature)/total number of CpGs in the genome. Correlation analysis was performed for different clusters. We first used the findOverlaps function in GenomicRanges package (v.4.4)63 to map VMRs to overlapped genes. Then, the Pearson correlation test was applied to obtain the correlation between mapped genes and corresponding VMRs. The Benjamini–Hochberg procedure was used to adjust all P values. GO enrichment analysis was conducted using the enrichGO function from clusterProfiler package64 (v.4.2). For GO enrichment in the comparative analysis of E11 and E13 mouse embryos, the FindMarkers function in Seurat package was used to find differential genes and VMRs in the same cluster from integrated data across two developmental stages. Differentially upregulated genes (false discovery rate ≤ 0.05) with demethylated VMRs were used for the GO analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw and processed data reported in this paper are deposited in the Gene Expression Omnibus (GEO) with accession code GSE270498. Resulting FASTQ files were aligned to the mouse reference genome (mm10). Published data for data quality comparison and integrative data analysis include a single-cell atlas of mouse embryos (https://oncoscape.v3.sttrcancer.org/atlas.gs.washington.edu.mouse.rna/downloads and https://omg.gs.washington.edu/), a mouse brain atlas (http://mousebrain.org/adolescent/downloads.html) and the Allen Mouse Brain Atlas (https://developingmouse.brain-map.org/). The data analysis pipeline and code to reproduce analyses are available at GitHub (https://github.com/zhou-lab/Spatial-DMT-2024/)65 and Zenodo (https://doi.org/10.5281/zenodo.15843594)66. Liu, Y. et al. High-spatial-resolution multi-omics sequencing via deterministic barcoding in tissue. Zhang, D. et al. Spatial epigenome–transcriptome co-profiling of mammalian tissues. Guo, P. et al. Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues. Tian, W. et al. Single-cell DNA methylation and 3D genome architecture in the human brain. Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Luo, C., Hajkova, P. & Ecker, J. R. Dynamic DNA methylation: in the right place at the right time. Koelsche, C. et al. Sarcoma classification by DNA methylation profiling. Yang, R. et al. Hydrogen sulfide promotes Tet1- and Tet2-mediated Foxp3 demethylation to drive regulatory T cell differentiation and maintain immune homeostasis. Brown, T. A. et al. Alterations in DNA methylation corresponding with lung inflammation and as a biomarker for disease development after MWCNT exposure. Hernando-Herraez, I. et al. Ageing affects DNA methylation drift and transcriptional cell-to-cell variability in mouse muscle stem cells. Field, A. E. et al. DNA methylation clocks in aging: categories, causes, and consequences. Nichols, R. V. et al. High-throughput robust single-cell DNA methylation profiling with sciMETv2. Zhao, T. et al. Spatial genomics enables multi-modal study of clonal heterogeneity in tissues. Shareef, S. J. et al. Extended-representation bisulfite sequencing of gene regulatory elements in multiplexed samples and single cells. Guo, J. U. et al. Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain. He, Y. et al. Spatiotemporal DNA methylome dynamics of the developing mouse fetus. The single-cell transcriptional landscape of mammalian organogenesis. Kremer, L. P. M. et al. Analyzing single-cell bisulfite sequencing data with MethSCAn. Integrated analysis of multimodal single-cell data. Kapfhamer, D. et al. Chromosomal localization of the ankyrinG gene (ANK3/Ank3) to human 10q21 and mouse 10. Yin, Y. et al. Impact of cytosine methylation on DNA binding specificities of human transcription factors. Yang, X. et al. Gene body methylation can alter gene expression and is a therapeutic target in cancer. Li, Y. et al. Genome-wide analyses reveal a role of Polycomb in promoting hypomethylation of DNA methylation valleys. Guilhamon, P. et al. Meta-analysis of IDH-mutant cancers identifies EBF1 as an interaction partner for TET2. Qiu, C. et al. A single-cell time-lapse of mouse prenatal development from gastrula to birth. The glial nature of embryonic and adult neural stem cells. & Greer, C. A. in The Neurobiology of Olfaction Vol. Transient overexposure of neuregulin 3 during early postnatal development impacts selective behaviors in adulthood. Rivers, L. E. et al. PDGFRA/NG2 glia generate myelinating oligodendrocytes and piriform projection neurons in adult mice. Bridges, C. R. et al. USP9X deubiquitylating enzyme maintains RAPTOR protein levels, mTORC1 signalling and proliferation in neural progenitors. Zhou, D. et al. AnkyrinG is required for clustering of voltage-gated Na channels at axon initial segments and for normal action potential firing. Ha, S. et al. Cerebellar Shank2 regulates excitatory synapse density, motor coordination, and specific repetitive and anxiety-like behaviors. Sheikh, F. et al. α-E-catenin inactivation disrupts the cardiomyocyte adherens junction, resulting in cardiomyopathy and susceptibility to wall rupture. Derrick, C. J. et al. Lamb1a regulates atrial growth by limiting second heart field addition during zebrafish heart development. Feng, J. et al. Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. Goll, M. G. & Bestor, T. H. Eukaryotic cytosine methyltransferases. Ito, S. et al. Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification. Lavado, A., Lagutin, O. V., Chow, L. M. L., Baker, S. J. & Oliver, G. Prox1 is required for granule cell maturation and intermediate progenitor maintenance during brain neurogenesis. Close, J. et al. Satb1 is an activity-modulated transcription factor required for the terminal differentiation and connectivity of medial ganglionic eminence-derived cortical interneurons. Simon, R. et al. A dual function of Bcl11b/Ctip2 in hippocampal neurogenesis. Targeted deletion of all isoforms of the trkC gene suggests the use of alternate receptors by its ligand neurotrophin-3 in neuronal development and implicates trkC in normal cardiogenesis. Expression of Cux-1 and Cux-2 in the subventricular zone and upper layers II–IV of the cerebral cortex. Zeisel, A. et al. Molecular architecture of the mouse nervous system. Zhou, W. et al. DNA methylation loss in late-replicating domains is linked to mitotic cell division. Savolainen, S. M., Foley, J. F. & Elmore, S. A. Histology atlas of the developing mouse heart with emphasis on E11.5 to E18.5. Frade, J. M. & Ovejero-Benito, M. C. Neuronal cell cycle: the neuron itself and its circumstances. & Chen, Z. Perivascular instruction of cell genesis and fate in the adult brain. Current perspective on the regulation of FOXO4 and its role in disease progression. Essential transcription factors for induced neuron differentiation. Mao, L. et al. HOXC9 links cell-cycle exit and neuronal differentiation and is a prognostic marker in neuroblastoma. The forkhead transcription factor Foxi1 is a master regulator of vacuolar H-ATPase proton pump subunits in the inner ear, kidney and epididymis. Xu, Y. et al. Tlx1 and Tlx3 coordinate specification of dorsal horn pain-modulatory peptidergic neurons. Fabyanic, E. B. et al. Joint single-cell profiling resolves 5mC and 5hmC and reveals their distinct gene regulatory effects. Allen Institute for Brain Science. Kaminow, B., Yunusov, D. & Dobin, A. STARsolo: accurate, fast and versatile mapping/quantification of single-cell and single-nucleus RNA-seq data. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Zhou, W. et al. BISCUIT: an efficient, standards-compliant tool suite for simultaneous genetic and epigenetic inference in bulk and single-cell studies. Kremer, L. P. M. et al. DNA methylation controls stemness of astrocytes in health and ischaemia. Kartha, V. K. et al. Functional inference of gene regulation using single-cell multi-omics. Brunson, J. C. ggalluvial: layered grammar for alluvial plots. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Lawrence, M. et al. Software for computing and annotating genomic ranges. clusterProfiler: an R package for comparing biological themes among gene clusters. Fu, H. & Deng, Y. Spatial joint profiling of DNA methylome and transcriptome (spatial-DMT). Lee, C. N., Fu, H., Zhou, W. & Deng, Y. Spatial-DMT. Cao, Y. et al. Single-cell bisulfite-free 5mC and 5hmC sequencing with high sensitivity and scalability. We acknowledge support from the Packard Fellowship for Science and Engineering (to Y.D. and the National Institute of Health and National Institute of General Medical Sciences (R35-GM146978) (to W.Z.). was supported in part by the Institute for RNA Innovation of the Perelman School of Medicine at the University of Pennsylvania. These authors contributed equally: Chin Nien Lee, Hongxiang Fu Department of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, PA, USA Chin Nien Lee, Hongxiang Fu, Angelysia Cardilla, Wanding Zhou & Yanxiang Deng Center for Computational and Genomic Medicine, The Children's Hospital of Philadelphia, Philadelphia, PA, USA Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar were responsible for the methodology. conducted all spatial joint profiling of DNA methylation and transcription experiments in E11 (50 μm), E11 (10 μm), E11 replicate (50 μm), E13 embryos and P21 mouse brain. A.C. assisted with experiments, including reagent preparation for E11 (50 μm), E13 embryos and P21 mouse brain. The original draft was prepared by C.N.L., H.F., W.Z. All authors reviewed, edited and approved the paper. Correspondence to Chin Nien Lee, Wanding Zhou or Yanxiang Deng. are inventors on a provisional patent application related to this work (application number: 63/809,054). is a scientific adviser at AtlasXomics. The other authors declare no competing interests. Nature thanks Chengqi Yi and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The diagram illustrates the chemistry workflow for simultaneous DNA methylation and RNA library preparation in Spatial-DMT. Initially, tissue sections are fixed and permeabilized to preserve tissue architecture and molecular integrity while allowing reagent penetration. Following permeabilization, sections undergo HCl treatment to disrupt the protein structure and remove nucleosome histones to improve Tn5 transposome accessibility. Next, Tn5 transposition integrates adapters into genomic DNA, and in situ reverse transcription (RT) converts mRNA into complementary DNA (cDNA) directly within the tissue. Streptavidin bead separation then facilitates the differential enrichment and processing of nucleic acids: cDNA molecules undergo a template switching reaction, followed by Nextera XT library preparation, while genomic DNA (gDNA) is subjected to NEBNext EM-seq conversion for methylation detection, splint ligation, and subsequent library construction. a, Correlation analysis between replicates based on DNA methylation (left) and RNA transcription (right) data from E11 embryo samples with two-sided Pearson correlation test. b, c, Integrative analysis (b) and spatial distribution (c) of clusters from two independent Spatial-DMT experiments using E11 embryo samples. d-f, Spatial mapping of DNA methylation and RNA expression levels for selected marker genes in facial (d), brain (e), and heart (f) regions from two independent Spatial-DMT experiments using E11 embryo samples. a, Barcode rank plots showing the distribution of reads per spatial barcode for E11, E13 embryos, and P21 brain samples. Valid barcodes are shown in blue, and filtered barcodes are shown in orange. c, Box plots showing the mitochondrial retention rate for E11, E13 embryos, and P21 brain samples. d, Spatial distribution heatmaps of global DNA methylation levels in E11, E13 embryos (mCG), and P21 mouse brain (mCG and mCA). e, The conversion rate of unmethylated cytosine in the linker sequence in E11, E13 embryos, and P21 mouse brain. f, Percentage of reads with poly A (≥ 30 adenine), poly T (≥ 30 thymine), and TSO sequence (AAGCAGTGGTATCAACGCAGAGTACATGGG) in the DNA libraries of E11, E13 embryos, and P21 mouse brain. b, DNA methylation levels under different chromatin states, comparing E11, E13 embryos, and P21 brain samples with reference datasets5,10,12,14,16. c, Line plots showing the mean DNA methylation levels (%) across different genomic regions for E11, E13 embryos, and P21 brain samples. Error bands represent mean values ± 1 standard deviation. d, Histogram and pie chart showing the size distribution and genomic annotation of variably methylated regions (VMRs) identified in the E11 embryo. The pie chart indicates the percentage of VMRs in different genomic features, such as promoters, exons, and intergenic regions. e, Confusion matrix illustrating the correspondence between clustering results derived from RNA transcription and DNA methylation data in the E11 embryo. f, Alluvial diagram showing the relationships among clusters identified by DNA methylation, RNA, and WNN integration in the E11 embryo. g, Modality weights indicating the relative contributions of gene expression and DNA methylation for each WNN cluster in the E11 embryo. a, Heatmaps illustrating the spatial distribution of DNA methylation levels (top rows) and RNA expression levels (bottom rows) for marker genomic loci and corresponding genes in the brain and spinal cord regions (W2) of E11 mouse embryo. b, GO enrichment analysis from one-sided hypergeometric test for differentially expressed genes in clusters W2. The dot plots show the top GO terms for biological processes enriched in each cluster. c, In situ hybridization (ISH) images from the Allen Developing Mouse Brain Atlas55 show the spatial distribution of markers of oligodendrocyte progenitors and premature oligodendrocytes in the E11 mouse embryo. These ISH images highlight the regions where these progenitor and premature cells are localized within the tissue. a, Heatmap of DNA methylation levels for the top 10 differentially methylated genomic loci in the craniofacial region (cluster W0) of E11 mouse embryo. Each row represents a specific genomic locus, and each column represents a different cluster. b, Heatmap of expression levels for nearby genes corresponding to the genomic loci in a. Each row represents a specific gene, and each column represents a different cluster. c, Spatial mapping of DNA methylation and RNA expression levels for selected marker genes in the craniofacial region (W0) of E11 mouse embryo. d, GO enrichment analysis from one-sided hypergeometric test for differentially expressed genes in clusters W0. The dot plots show the top GO terms for biological processes enriched in each cluster. a, Heatmap of DNA methylation levels for the top 10 differentially methylated genomic loci in the heart region (cluster W6) of E11 mouse embryo. Each row represents a specific genomic locus, and each column represents a different cluster. b, Heatmap of RNA expression levels for nearby genes corresponding to the genomic loci in a. Each row represents a specific gene, and each column represents a different cluster. c, Spatial mapping of DNA methylation and RNA expression levels for selected marker genes in the heart region (W6) of E11 mouse embryo. d, GO enrichment analysis from one-sided hypergeometric test for differentially expressed genes in clusters W6. The dot plots show the top GO terms for biological processes enriched in each cluster. a, Spatial distribution of all clusters for DNA methylation (left) and RNA transcription (right). Cells are colored according to cell annotations from scRNA-seq data25 (b) and unsupervised clustering results from Spatial-DMT (c). d, Spatial mapping of DNA methylation, RNA expression levels, and in situ hybridization (ISH) images from the Allen Developing Mouse Brain Atlas55 show the spatial distribution of selected markers in clusters W7 and W11. e, GO enrichment analysis from one-sided hypergeometric test for differentially expressed genes in clusters W7 and W11. The dot plots show the top GO terms for biological processes enriched in each cluster. f, Spatial distribution of olfactory epithelial cells and olfactory sensory neurons revealed by cell type decomposition of spatial transcriptomic pixels using a scRNA-seq reference25. g, Spatial mapping of DNA methylation and gene expression level of Fstl5 (olfactory sensory neurons marker). a, Spatial distribution and UMAP visualization of clusters identified from spatial DNA methylation data (left), spatial RNA transcription data (middle), and integrated DNA and RNA data using WNN analysis (right) for the E13 embryo. Each color represents a different cluster, illustrating the spatial distribution of methylation patterns, gene expression profiles, and the integrated data across different regions such as the forebrain, midbrain, hindbrain or spinal cord, heart, and face, nose and jaw region. b, c, Comparative analysis of RNA expression (b) and DNA methylation levels (c) for upregulated genes in E13 heart region, indicating significant changes in methylation and gene expression patterns during embryonic development with two-sided Wilcoxon test. d, Heatmaps comparing global RNA expression (left) and DNA methylation levels (right) in brain and spinal cord regions between E11 and E13 embryos. e, GO enrichment analysis from one-sided hypergeometric test for biological processes related to demethylated and upregulated genes in the heart region of E13 embryo. a, UMAP visualization of clusters identified from spatial DNA methylation data (left), spatial RNA transcription data (middle), and integrated DNA and RNA data using WNN analysis (right) for the P21 brain. b, Spatial mapping of CpG (left) and CpA (middle) methylation levels, and RNA expression levels (right) for Satb1 in the P21 brain. c, Spatial mapping of CpG (left) and CpA (middle) methylation levels, and RNA expression levels (right) for Bcl11b in the P21 brain. d, Density plot showing the correlation of gene expression with CpA and CpG methylation in the P21 mouse brain. The plot illustrates the distribution of correlation coefficients, with red indicating CpA methylation and green indicating CpG methylation. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Lee, C.N., Fu, H., Cardilla, A. et al. Spatial joint profiling of DNA methylome and transcriptome in tissues. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09437-6'>Ancient DNA connects large-scale migration with the spread of Slavs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 15:15:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The second half of the first millennium ce in Central and Eastern Europe was accompanied by fundamental cultural and political transformations. This period of change is commonly associated with the appearance of the Slavs, which is supported by textual evidence1,2 and coincides with the emergence of similar archaeological horizons3,4,5,6. However, so far there has been no consensus on whether this archaeological horizon spread by migration, Slavicisation or a combination of both. Genetic data remain sparse, especially owing to the widespread practice of cremation in the early phase of the Slavic settlement. Here we present genome-wide data from 555 ancient individuals, including 359 samples from Slavic contexts from as early as the seventh century ce. Our data demonstrate large-scale population movement from Eastern Europe during the sixth to eighth centuries, replacing more than 80% of the local gene pool in Eastern Germany, Poland and Croatia. Yet, we also show substantial regional heterogeneity as well as a lack of sex-biased admixture, indicating varying degrees of cultural assimilation of the autochthonous populations. Comparing archaeological and genetic evidence, we find that the change in ancestry in Eastern Germany coincided with a change in social organization, characterized by an intensification of inter- and intra-site genetic relatedness and patrilocality. On the European scale, it appears plausible that the changes in material culture and language between the sixth and eighth centuries were connected to these large-scale population movements. This study combines a temporal transect of the Elbe-Saale region in Eastern Germany with the wide-angle view of large-scale demographic and cultural transformations that emerged similarly in other Eastern and Central European regions. Traditionally, on the basis of historical writings, this transformation is attributed to the emigration of ‘Germanic' peoples from East-Central Europe and the arrival of a new population that contemporaries described as ‘Slavs'. At least since the first century bce, the lands between the Rhine and Vistula River were settled by numerous peoples and tribes for whom Roman observers used the umbrella term ‘Germani'7. These Germanic peoples were in contact with the Roman Empire west of the Rhine and south of the Danube, and since the late second century ce increasingly raided Roman provinces8,9. In the MP, many of them left and settled on Roman territory7,8,9,10, among them Vandals, Goths, Franks and Longobards11,12. During the seventh century, Slavs are first mentioned east of the Saale, but they soon expanded westward17, forming a contact zone between Slavic- and Germanic-speaking groups. In the seventh century, there is evidence for the presence of Slavs in much of East-Central and Southeastern Europe. Where Slavs lived, Roman, Germanic and other pre-Slavic infrastructures were usually replaced by rather simple ways of life, archaeologically characterized by small settlements of pit houses, cremation burials, handmade, undecorated pottery and modest, low-metal material culture18, known as the Prague-Korchak group3,4. More complex social systems and regional rulership developed later in the contact zones with Byzantium and the Christian west. The similarity of early Slavic cultures was often attributed to a swift spread of Slavs from Northeast of the Carpathians, although debates continue, not only about their geographical origin (Supplementary Note 1,1). In Poland19, the non-native (allochthonist) view assumes Slavic origin from Ukraine–Belarus18, whereas the native (autochthonist) concept asserts that their ancestors inhabited Polish territory since the Bronze Age. Some scholars doubt Slavic expansion by migrations and assume that there was ‘Slavicisation' of existing populations5,18,20,21,22,23,24,25 (Supplementary Note 1,2). Previous modern26 and ancient DNA studies have supported gene flow into the Northern Balkans27 and the Russian Volga-Oka region28, but also argued for population continuity in Poland29, so that the scale and sequence of these movements and their association with ‘Slavic' material culture has remained unclear. Eventually, this cultural transformation led to the replacement of Germanic and other languages in East-Central and Southeastern Europe and the introduction of Slavic languages, which today represent the largest linguistic group in Europe30. Yet, this presumed joint spread of language and material culture is difficult to trace, given that the first longer texts in Slavic were written in the late ninth century5,31. Together with previously published data from Roman and early medieval Europe27,28,29, the newly analysed ancient DNA from the Elbe-Saale region and complementary data transects from the Northwestern Balkans, Poland, Latvia and Ukraine identify large-scale population movement and a major demographic shift. This can be linked to historical information about the spread of Slavic groups in the sixth to eighth centuries and provides a plausible vector for the spread of Slavic languages across much of Eastern Europe21,23,32. We selected skeletal remains from 591 ancient individuals from 26 different sites from Central and Eastern Europe (Supplementary Tables 1 and 2), creating, in combination with previously published data, a dense sampling transect for three regions: (1) Elbe-Saale Region in Eastern Germany as the main study area; (2) the Northwestern Balkans; and (3) Poland–Northwestern Ukraine (Extended Data Fig. Complementary to these three transects, we generated new data and collected published data from the Baltics and Northwestern Russia to form a reference transect in the east. After hybridization DNA capture and quality filtering (Methods), genome-wide data for 555 unique individuals with a median coverage of 538k single nucleotide polymorphisms (SNPs) (on 1240k data) were available for analysis, including 359 ancient individuals from the SP, as well as 205 individuals predating the cultural transformations connected to the emergence of the Slavs (Fig. We analyse the ancient genome-wide data (Supplementary Table 11) together with an extended dataset of more than 11,500 present-day Europeans (Supplementary Table 3), covering all major Slavic-speaking groups, including data from more than 600 individuals belonging to the Sorbian minority in Eastern Germany33. a, We assign newly reported (n = 550) and published genomic data (n = 723) to three study transects: the Northwestern Balkans (n = 301) (orange to yellow), Elbe-Saale region (Eastern Germany) (n = 483) (light and dark red), and Poland–Northwestern Ukraine (n = 489) (light and dark blue). We further analyse newly reported and published data from the Baltics as well as the Russian Volga-Oka region as reference, in total covering 1,840 ancient individuals. b, Chronological sequence of 1,243 newly reported (coloured) and previously published (grey) samples analysed in this study. We selected samples from all three study regions that predate and postdate the transition to the SP. To visualize genome-wide ancestry diversity before and after the spread of Slavic groups, we performed principal component analysis (PCA) on 10,528 present-day Europeans and projected our newly reported and other relevant ancient genome-wide data onto their genetic variation (Fig. When comparing the SP samples to earlier and present-day data from our three study regions, we observe that the genetic composition within the transects changed markedly between about 600 and 800 ce. In general, the Roman and MP samples that predate the arrival of Slavic groups show high genetic heterogeneity in PCA space, with most samples from Germany and Poland29,34 clustering with present-day continental Northern German, Dutch and Scandinavian populations (Extended Data Fig. a, The reference PCA computed using 10,528 present-day Europeans. b, Hierarchical clustering (ward.D2) of Mahalanobis distances between selected ancient groups based on the first ten principal components. BA, Bronze Age; EIA, Early Iron Age; EMA, Early Middle Ages; IA, Iron Age; MA, Middle Ages; MGS, Mödling; MP, migration period; SP, Slavic period. c, Ancient genome-wide data (n = 835) from the Northwestern Balkans, Eastern Germany and Poland–Northwestern Ukraine projected onto the modern reference PCA. Large symbols denote the mean PC1 and PC2 coordinates. In Eastern Germany and the Northwestern Balkans, most of the genetic diversity within the Roman and MP clusters follows a north–south cline along PC1. More unexpectedly, we detect a high number of MP individuals with non-local, Southern European ancestry in the Elbe-Saale region of Eastern Germany, although this area was never part of the Roman Empire. Using qpAdm36,37, we measure on average between approximately 15% and 25% of Southern European ancestry in all 4 MP sites of the region (Extended Data Fig. Both PCA-based MOBEST38 analysis and F4 statistics indicate that this non-local ancestry was most probably derived from contemporaneous source populations in Italy and/or the Northern Balkan Peninsula (or other areas of the Roman Empire where people of this ancestry were located) (Extended Data Fig. Previous studies already identified mixed communities of northern and southern ancestry in Hungary and Northern Italy that were interpreted as amalgamation between Northern European newcomers and the local romanized population39,40. In contrast to these earlier results, we do not find evidence that the two different ancestries were correlated to differences in material culture (Supplementary Table 4). Applying a generalized linear model, we demonstrate that neither the presence of grave goods overall, nor certain types of artefacts (such as weapons or brooches) are significantly correlated with either PCA position or ADMIXTURE profiles (Supplementary Fig. Instead, we find the only significant (P < 0.05) correlation between ancestry and material culture among the burial constructions, where we show that individuals buried in pits feature on average higher Northern European ancestry (Supplementary Fig. The spatial organization of the burials was also not determined by similarity in ancestry. Instead, we observe that individuals were buried close to their biological relatives, within small kin groups composed of individuals with Northern European, Southern European or mixed ancestry, reflecting a high degree of admixture between individuals with different ancestry backgrounds during the MP. Consequently, our data from Eastern Germany demonstrate that the cosmopolitan character of the Roman Empire not only affected the incorporated territories but also facilitated exchange and mobility along its borders and beyond into barbarian lands (Barbaricum), resulting in an unprecedented genetic diversity in Central Europe during34 and, in the case of Eastern Germany, even after its existence. Although the causes and circumstances of their movement to the Elbe-Saale region remain open for speculation, these newcomers apparently adapted the fashions and traditions of the local populations, resulting in a rather homogenous material culture within a group of individuals with diverse genetic backgrounds. However, this diversity had collapsed in the subsequent SP (Supplementary Note 6). In contrast to the preceding MP, the genetic profile of Eastern Germany during the SP has shifted considerably and clusters nearly exclusively with present-day Slavic-speaking populations (for example, Poles and Belarussians), indicative of a fundamental replacement of genetic ancestry (Fig. A similar pattern is seen in the Northwestern Balkans, Poland–Northwestern Ukraine as well as the Volga-Oka region in Russia28, illustrating that this influx of new genetic material was not limited to certain regions but affected wide areas of Central and Eastern Europe, consistent with the rather simple, very similar archaeological horizons observed during the SP (Supplementary Figs. To formally test whether these patterns observed from PCA are consistent with gene-flow events from the east into our study regions, we used F-statistics to quantify genetic affinities of SP individuals to preceding MP and succeeding present-day groups (Fig. The divergence between pre-Slavic and Slavic-associated groups is verified both in the distribution of genetic distances (FST) (Supplementary Fig. Both on the population and the individual scale, SP individuals from all three study regions uniformly show less genetic affinity to the preceding local populations than to ancient and present-day groups from Eastern Europe and Baltics (Supplementary Figs. a, Trajectories of changes in affinity to Eastern Europe in ancient and present-day individuals from the Northwestern Balkans (n = 301), Eastern Germany (n = 483) and Poland–Northwestern Ukraine (n = 489), as measured using F4 statistics of the form F4(Han Chinese, test; Italy, Poland) (for the Northwestern Balkans) and F4(Han Chinese in Beijing, China (CHB), test; Denmark, Poland) (for Eastern Germany and Poland–Northwestern Ukraine). b, Supervised ADMIXTURE modelling for the study regions. Ancient and present-day samples (n = 1,344) decomposed into 12 ancestral ancestry components. For different time periods, individual results were averaged and plotted according to their mean date. Influx of specific ancestries mentioned in the text are indicated using arrows. Relevant Y chromosome haplogroups associated with these autosomal ancestries are highlighted. c, Average sum of IBD segments (sIBD (in cM)) shared between nine ancient MP and SP groups. For all pairs of populations, the sum of IBD segments longer than 12 cM shared between members was calculated and normalized to the total number of pairs. The sums are depicted as symmetrical matrix. Hierarchical cluster analysis applying Ward's minimum variance method to the columns is added as a dendrogram. Our results reveal that SP individuals display Baltic or Northeastern European-related ancestry that was previously absent in the three study regions. To quantitatively estimate this influx, we decomposed ancestral sources using a supervised clustering approach implemented in the software ADMIXTURE41. Specifically, we assembled modern populations into 12 metapopulations that serve as proxies for the source ancestries in Central Europe (Methods and Supplementary Note 4.1). Applying our ancestry decomposition to the ancient genome-wide data, we find that (despite differences in the local trajectories) Northeastern European ancestry (BAL, represented by present-day individuals from Belarus, Lithuania and Latvia) was either completely absent or only a minor ancestry component throughout most of prehistory in our study transects (Fig. 3), accounting for 6 ± 2%, 5 ± 1% and 7 ± 2% of the total MP ancestry in the Northwestern Balkans, Eastern Germany and Poland–Northwestern Ukraine, respectively. However, consistent with PCA (Supplementary Fig. 3a–c), BAL ancestry increased after 600 ce and became the largest ancestry component in all three study regions, reaching 47 ± 2%, 65 ± 1% and 63 ± 2%, respectively, during the SP. Outside our three study transects, we furthermore identify a major surge of BAL ancestry (from 0% in the MP to approximately 27%) in the Avar-associated population of Mödling, Austria42, confirming an early arrival in the Pannonian Basin in the seventh century ce as reported by written sources, followed by substantial admixture with local groups (Extended Data Fig. Only in Northwestern Russia do we detect a different trajectory: in the Volga-Oka area, the Slavic transition coincides with a significant decrease of BAL ancestry (from 65 ± 2% to 55 ± 7%), suggesting that the SP newcomers originated from a region further to the west of the Volga-Oka area where they incorporated additional ancestry not local to Eastern Europe. To showcase this shared descent, we applied ancIBD43 to identify segments that are identical by descent (IBD) that are shared between the MP and SP populations. We highlight that SP groups in Croatia, Eastern Germany and Poland–Ukraine share comparably large amounts of IBD with each other, despite the vast geographic distance between the three study regions, but share nearly no segments with the preceding populations (Fig. This IBD-sharing signal, including a large fraction of segments longer than 16 cM, clearly indicates that ancient individuals from Slavic-associated contexts descend from a common source population that migrated westwards and southwards at most a few generations earlier across Central Europe (Extended Data Figs. Such evidence for large-scale population movement also explains the previously detected pattern of high levels of sharing of IBD between present-day pairs of individuals across Eastern Europe44 (Supplementary Fig. 56) and rejects the idea that this signal was caused predominantly by consistently low population densities45. To obtain a finer-scale characterization of genetic ancestries across space and time, we applied a hierarchical cluster detection approach to a network of around 2,500 individuals constructed from these pairwise IBD-sharing similarities (Supplementary Table 38 and Supplementary Note 4.3.2). We identify a large IBD-sharing community that contains most of our new and published SP individuals as well as multiple other contemporary samples from Central and Southeastern Europe. Within this larger cluster, we identify two distinct sub-communities: one primarily includes SP individuals from north of the Carpathian Mountains, whereas the other comprises individuals buried further south. This separation may reflect two geographically diverging waves of expansion or different patterns of incorporation of the local populations (Extended Data Fig. Yet, at least sporadic gene flow from Eastern Europe into Pannonia and the Balkans must have already occurred during the Iron Age and Roman Period, as we identify a substantial number of individuals within the SP cluster buried in Austria, Hungary, Serbia and Montenegro during the time period from 500 bce to 300 ce, predating the large-scale population movements of the sixth and seventh centuries (Supplementary Figs. Using newly generated early medieval data from the Polish site Gródek, Hrubieszów County, near the Ukrainian border, which represents some of the oldest Slavic inhumation burials from Poland (dating between the seventh and ninth centuries ce), as a proximal source (both in time and space) for the incoming BAL-enriched ancestry, we calculate using qpAdm that approximately 82 ± 1%, 83 ± 6%, 93 ± 3% and 65 ± 4% of the local gene pool in the Northwestern Balkans, Eastern Germany, Poland–Northwestern Ukraine and the Volga-Oka valley, respectively, were replaced during the SP by migrants from Eastern Europe (referred here to as ‘SP ancestry'; Methods) (Fig. These results contradict a model of substantial population continuation from the Iron Age or MP to the Middle Ages in present-day Western and Central Poland, where previous research claimed an autochthonous origin of the SP gene pool29,46,47 (Extended Data Fig. Yet more samples are needed to assess the overall degree of genetic replacement over the larger area. Applying qpAdm to model present-day groups using ancient source populations, we show that Eastern European ancestry is the dominant genetic component in all Slavic-speaking populations today and is also found in neighbouring non-Slavic-speaking groups in Central Europe and regions bordering to the south (Extended Data Fig. We measure the highest proportions of Eastern European ancestry in present-day Ukraine, Belarus and Poland, from where it gradually decreases to the east and south (Extended Data Fig. Notably, we observe a profound duality to the west, in Eastern Germany, with the present-day German-speaking population from Saxony exhibiting around 40% SP ancestry and the Slavic-speaking Sorbs of Upper Lusatia (Saxony) exhibiting 88% SP ancestry (comparable to modern Poles) (Extended Data Fig. This agrees with previous studies on the genetic isolation of the Sorbs33,48 and is consistent with them representing the descendants of these Slavic groups that were minimally (or at least less) integrated into the reproductive networks of the expanding German-speaking settlement east of Elbe and Saale from the twelfth century onwards49,50,51. Conversely, we suggest that the German eastward expansion and earlier Frankish conquest is probably associated with the reduction in SP ancestry observed in the German-speaking population. a, Contours indicate the averaged MOBEST maximum probability at search time 1,950 years before present for 20 individuals from Niederwünsch (denoting the mean prediction of the geographic regions where the ancestors of these individuals originated). This is supplemented by five lines of evidence: (A) ancient and present-day groups from the Baltics show the highest genetic similarity to SP individuals; (B) Bronze Age and Iron Age individuals from Estonia, Ingria and Karelia are less related to SP individuals than groups from Lithuania and Latvia; (C) populations in Western Russia feature too high proportions of Steppe and/or Siberian ancestry; (D) SP individuals are enriched in EEF and depleted in WHG ancestry compared with Bronze Age and Iron Age populations from the Baltics; (E) Putative migration directions inferred using pairwise mean sIBD sharing values between SP sites77 (n > 2). b, Comparison of linguistic split times (left) and genetic admixture dates in SP groups (right). Divergence date distributions for the Balto-Slavic and Slavic subgroups were extracted from a sample of 37,004 trees78. Shown are non-local ancestry proportions on the autosomes, X chromosome and the Y chromosome (Y-chromosome haplogroups R1a, N and I2 for SP populations; E, G, J and T for MP populations). Points denote qpAdm (autosomes and X chromosome) or maximum likelihood (Y chromosome) estimates. Estimates were obtained as described in Supplementary Notes 7.2 using ancient source groups. The corresponding data can be found in Supplementary Table 47. ♂ indicates an excess of non-local males in the admixture process; ♀ indicates a non-local female bias. Both F4 and FST statistics identify the highest genetic similarity between SP individuals and present-day populations from the Baltics, Poland and Belarus (Supplementary Figs. These are also the regions where BAL and SP ancestry (here approximated by medieval samples from Gródek) are maximized today and where the highest proportions of R1a haplotypes (specifically R1a-M458 and R1a-M558) are found among the male population. In patterns of haplotype sharing between the ancient and modern individuals, this similarity was mirrored in a distinctive IBD signal (Extended Data Fig. 23–25): SP individuals from all three study regions share more and longer IBD fragments with Eastern Europeans than with any other Eurasian group, establishing direct genetic relatedness between present-day Balto-Slavic speakers and SP individuals in Central and Southeastern Europe (Extended Data Fig. This pattern of excess affinity to Northern and Northeastern Europeans is not only evident in the comparison with present-day data but also in the archaeogenetic record: Comparing the SP individuals to other ancient samples, we show that they, independent of their geographic origin, share the highest drift and largest sum of IBD with Bronze and Iron Age groups from Lithuania, Latvia and Estonia52,53, and are (as shown by F-statistics) more closely related to these individuals than to any other population in post-Neolithic Europe (for IBD see Supplementary Fig. Yet, in contrast to the Bronze Age Baltic samples, we note that SP individuals from all study regions exhibit substantially less Western hunter-gatherer (WHG) and more early European farmer (EEF) ancestry (Supplementary Figs. This suggests that the SP groups in Central Europe were already admixed, most probably between a WHG and Steppe ancestry-enriched Baltic Bronze Age-related source from the sub-Neolithic forest zone and at least one EEF-enriched source from the south. Using qpAdm, we identify various groups in Southeastern and East-Central Europe that constitute working proxies for such an EEF-enriched donor, yet we are not able to precisely identify the most likely representative (Supplementary Fig. Across all fitting two-way models (P > 0.01) (and most non-fitting), the admixture proportions are highly similar, with the Eastern German and Polish-Northwestern Ukrainian SP samples receiving around 71% Baltic (95% confidence interval: 66.5%–76%) and around 29% (95% confidence interval: 24%–33.5%) EEF-enriched ancestry (Supplementary Fig. However, we highlight that the demographic trajectories that led to the formation of the SP gene pool were potentially more complex than a simple two-way admixture event. Although we calculate similar estimates of Baltic Bronze Age-derived ancestry applying a non-negative least squares approach based on PCA- and ADMIXTURE results54, mirroring previous results from genome-wide genealogies55, all models profit from the inclusion of an additional Western European source (Supplementary Figs. Thus, which vector population(s) ultimately transmitted EEF-enriched ancestry to the Northeast cannot be resolved fully for now (Supplementary Note 4.4.2-3). Assuming a two-way admixture process, using DATES (distribution of ancestry tracts of evolutionary signals)56 we obtained an average date of approximately 1000 bce for this admixture event that formed the SP gene pool (972 bce ± 250 for Niederwünsch and 906 bce ± 362 for Poland_EMA, respectively) (Fig. Of note, these DATES estimates overlap with the more recent part of the distribution of divergence estimates between Baltic and Slavic languages (Fig. Both phylogenetic analysis of cognate-coded basic vocabulary data (Fig. To identify the most plausible geographic location for this initial formation of the SP gene pool, we applied MOBEST to perform spatiotemporal interpolation of the genetic affinities of SP individuals from the study regions to approximately 5,660 previously published ancient samples from Western Eurasia, obtaining similarity probabilities across Europe that can be interpreted as proxies for geographical origin at a specific time. We set the prediction time to 1,950 years before the present, providing us the most likely origin of an individual at this time point (and thus before the demographic transition in Central Europe). Averaging the probability surfaces, we infer a region spanning the south of Belarus and north of Ukraine as the best spatial proxy for the origin of the SP individuals in our three study transects (Fig. Such a range would agree well with the area where many linguists propose the earliest development of Slavic languages and archaeologists locate the origin of Slavic-associated material culture5 (Supplementary Fig. 44); however, more ancient DNA (aDNA) data are needed to conclusively assess the genetic landscape of this region. From there, Northeastern European ancestry is likely to have spread east, west and south, admixing with or even replacing the local gene pools (Fig. Although we cannot precisely measure the onset of this expansion or its duration, we highlight that DATES estimates for admixture between local and immigrant ancestries in SP individuals are generally recent and similar across the study transects, consistent with admixture processes starting in the sixth and early seventh century and agreeing with historically recorded arrival dates of Slavic groups in these regions (Extended Data Fig. The detection of substantial genetic introgression from the northeast into regions in which Slavic came to be spoken26,27,44,65,66 indicates that the diffusion of Slavic language and Eastern European-derived ancestry were related, although the degree of their overlap cannot be ascertained. This provides a plausible explanation for the high genetic relatedness across present-day Slavic-speaking groups26,44, which was previously linked to the spread of the Slavic languages44,65. However, we highlight that such a simplified model does not capture the more complex regional dynamics that emerge from historical and archaeological evidence, and are still evident in language boundaries that do not correspond to genetic differences across the Balkans and Central Europe66. To investigate possible sex biases in these expansion and admixture processes, we compared estimates of SP-related ancestry on the X chromosomes and the autosomes to identify proportion differences indicative of male-biased admixture (Fig. Notably, we find no evidence for sex bias in any of the SP populations in Germany, Croatia, Poland or Russia (|z | <2; Fig. However, we observe that the previously undetected gene flow of Southern European-related ancestry into the MP population of Eastern Germany was significantly female-biased in most studied sites (Fig. The Slavic groups that we studied also showed fundamentally different social organization compared with the preceding MP population (Supplementary Note 8). Most notably, we highlight more intense inter-site and intra-site genetic relatedness in the Elbe-Saale region (Extended Data Fig. 2c), reflected by patrilineally organized pedigrees that comprise large numbers of individuals (Extended Data Figs. The cemeteries of the preceding MP in Eastern Germany were characterized by small units of biological relatedness, mostly consisting of fewer than four first- and second-degree relatives. At the site level, we identified for each individual an average of 1.16 ± 0.18 close relatives (here defined as all relationships up to third degree) (in Brücken specifically: 0.64 ± 0.14; Supplementary Fig. This pattern is also mirrored in IBD sharing within sites (which also captures distant genetic relatedness greater than third degree), with the proportion of pairs of individuals that share any IBD larger than 12 cM ranging between 1 ± 0.4%, 6.8 ± 1.8% and 5.2 ± 1.8% in Brücken, Deersheim and Obermöllern, respectively. By contrast, during the SP, we show that the number of close relatives at the sites increased nearly sixfold to 6.41 ± 0.4%. We even observe one case of seven offspring from the same couple (Extended Data Figs. Notably, four of the seven siblings had reached reproductive age, with three of them having offspring. As most of them were male, we can assume that several grown-up daughters might had gone elsewhere to marry. Moreover, the majority of offspring being male points towards additional unsampled female siblings (to statistically account for an equivalent number of females born). Notably, for all unions (in which at least one parent was identified on site), we find 52 sons (62% of the offspring; 95% confidence interval: 51–72%) but only 32 daughters (38% of the offspring; 95% confidence interval: 28–49%) (exact binomial test; P = 0.03753). More distant genetic relationships also increased. For Niederwünsch and Steuden (approximately 8 km apart), 18.3 ± 0.6% and 15.3 ± 2.1% of the pairs of individuals share IBD fragments indicative of recent common descent (more than 12 cM apart). Although occupation periods in MP sites were shorter than in Niederwünsch (preventing the emergence of large-pedigree structures), the rather short-lived site of Steuden demonstrates that the differences in intra-community relatedness between MP and SP cemeteries were not caused exclusively by differences in the duration of site occupation. Although these extensive kinship networks evidenced a high degree of relatedness among all individuals within sites, we do not find a single case of close consanguinity (defined here as offspring of first cousin unions or closer) (Supplementary Fig. We also identify at least 11 cases of individuals reproducing with multiple partners, pointing to polygamy or serial monogamy. Despite a 2.7:1 ratio of half-siblings sharing the same father (95% confidence interval: 0.43–0.91) versus those sharing the same mother (95% confidence interval: 0.09–0.57), we do not find a single instance of levirate marriages as practiced in late Avar-period communities in the Carpathian Basin67. Although close relatives were buried significantly closer together than non-related individuals in both the MP and the SP, only during the SP did cemeteries feature a significant correlation between genetic and spatial distances, suggesting that the cemeteries were planned and structured around these larger kin groups (Mantel statistic based on Spearman's rank correlation; P = 0.0001 for both Niederwünsch and Steuden) (Extended Data Fig. This signal is most prominent in the site of Steuden, where at least 27% of all variance in spatial distances between graves is explained by genetic relatedness. Although the sex ratio of adults across the SP sites is balanced (Exact binomial test; P = 1 for Steuden, P = 0.08794 for Niederwünsch), females have on average significantly fewer close relatives than males (Fisher's exact test; P = 0.008) and feature overall an increased pairwise mismatch rate compared with males (Wilcoxon rank sum test; test statistic (W) = 13,976,553, P = 0.04218; Supplementary Fig. 67) and a lower sum of IBD segments shared within sites (Welch two-sample t-test; t = −3.707, d.f. However, females share more IBD segments between different sites than males, thereby demonstrating higher inter-site relatedness among females in contrast to higher intra-site relatedness among males (Welch two-sample t-test; t = 2.9513, d.f. This demonstrates that exogenous origin was more common for females than for males, suggesting a patrilocal inheritance organization, and agrees well with the nearly exclusively patrilineally organized pedigrees (for example, 88% patrilineal lineages (95% confidence interval: 68–97%) versus 12% matrilineal lineages (95% confidence interval: 4–32%)) and the significant underrepresentation of female offspring at the sites (Extended Data Figs. Across all SP sites in Eastern Germany, we find only one instance of mitochondrial haplogroups being transmitted further than one daughter generation. This contrasts with the preceding MP population, for which we detect no difference in the number of close relatives between males and females (Fisher's exact test; P = 0.74) and no difference in pairwise mismatch rate in general (Wilcoxon rank sum test; W = 3,215,752, P = 0.06), potentially suggesting a less strictly patrilineal social system before the arrival of Slavic groups (Supplementary Fig. However, owing to the overall smaller number of identified relatives, these tests might be less statistically conclusive and underestimate signals of MP female exogamy and patrilineal practices. Patterns of patrilocal organization into kin groups are broadly similar across regions and might have contributed to the previously described turnover and homogenization of the paternal gene pool (Extended Data Fig. In particular, we find an identical pattern of correlation of spatial and genetic distances in SP Velim, Croatia (Mantel statistic based on Spearman's rank correlation; P = 0.0001; Supplementary Figs. 64 and 65d) as well as evidence for patrilocality and female exogamy (for example, more close biological relatives among males than among females (Fisher's exact test; P = 0.027) and higher pairwise mismatch rates among females compared to males (Wilcoxon rank sum; W = 156,550, P = 0.02377; Supplementary Fig. By contrast, at Velim the mean number of close relatives in the site is significantly lower than in Niederwünsch or Steuden (1.01 ± 0.17). Within a shared pattern of patrilocality, SP fine-scale organization differed substantially across Central Europe owing to the complex, regionally contingent nature of this expansion. Rather than simple replacement, partial integration of the local population was probably dependent on the fortunes of specific groups or families. However, the substantial number of individuals from sites across Eastern Germany, Croatia and Poland–Northwestern Ukraine who share comparably large amounts of IBD with each other confirms that these Slavic-associated groups were closely linked as the result of a shared biological origin and recent geographical expansion. Here we present several results in a wide chronological and topographical range, based on fine-grained genetic transect studies of a number of complete cemeteries. Across Central Europe, we show that during the Roman Iron Age and subsequent MP genetic diversity increased not only within the empire, but also outside it. Southeastern Europe experienced an influx from the Eastern Mediterranean, whereas large parts of East-Central Europe were inhabited by people mainly related to modern inhabitants of Northern and Northwestern Europe. We also show a fifth and sixth century admixture of Mediterranean ancestry in Eastern Germany, prevalently but not exclusively through females. They and their offspring became well-integrated in the regional societies, and their burials show no signs of inferior status or cultural differences, a finding very different from those at early medieval cemeteries in Hungary, Italy39 and England68 with respect to incoming Northern versus local, Southern and Western European ancestry. Most importantly, our results are relevant for the debate on whether the spread of Slavs was due to large migratory movements and the expansion of a homogeneous population, or to the gradual acculturation of regional populations. We were able to connect the fundamental transformations of culture in large parts of Eastern Europe to a substantial movement of people most plausibly from regions between the Eastern Baltics and the Northwestern Pontic region. This genetically inferred area agrees well with archaeological hypotheses for the origins of Slavic material culture east of the Vistula River, for instance, in the Kyivan culture (second or third to fifth centuries ce)4,5,20,21. However, without aDNA from this region and period, more precision remains impossible. SP ancestry spread to many directions from the sixth century onwards. Yet we can only detect it when the immigrants stopped cremating their dead at different stages between the eighth and tenth centuries. Only few early inhumation burials (for example, Velim and Gródek) provide insights in line with more recent sites. Consequently, it remains unclear whether most of the SP ancestry arrived in a single pulse or over a prolonged time span. As described in previous publications27, our results indicate that sporadic individual percolation from Eastern Europe into the Balkans preceded (and potentially succeeded) any large-scale movements, pointing to long-term mobility accompanying the sixth and seventh century demographic transformation. To what extent the previous population disappeared, continued to live alongside the newcomers or mixed with them depended on the region69,70,71. Among our study regions, the shift was most marked close to the areas of origin, as in Poland and Eastern Germany. By contrast, we found more admixture with local populations in the Northwestern Balkans, the Carpathian Basin and the Volga-Oka area. Remarkably, the Slavic population buried in Eastern Germany in the late tenth to twelfth century, nearly four centuries after the first Slavic groups settled in the wider region, had little genetic exchange with the neighbouring Thuringian population, although archaeological finds demonstrate regular interactions between these groups. To some extent, this discrepancy might be caused by the selection of studied sites within a more heterogeneous genetic landscape. However, our results agree with the drastic break between the preceding Germanic material culture and that of the Early SP, probably after an intermediate phase of scarce settlement in most of the Northwestern Slavic area after the mid-sixth century69,70,71,72,73,74,75, and disagree with the model of a Slavicisation of the existing population. This change was driven by substantial movements of people and can be linked with the dispersion of Slavic languages and the spread of a new material culture. However, our data do not support total population replacement across Eastern Europe, but point to a complex, regionally contingent demic diffusion with partial integration. SP ancestry in the Balkan Peninsula and Western Russia did not form the majority in all areas. Consequently, besides large-scale movements, several other mechanisms, including genetic and cultural assimilation of the autochthonous populations by the expanding SP gene pool, are needed to explain the ancient and present-day genetic patterns observed on the margins of the diffusion area. Where the newcomers mixed with local populations, they did so without significant gender bias. Unlike in the MP, the SP communities in the Elbe-Saale region relied more on biological relatedness in their organization, and their large pedigrees provide valuable information about their reproductive practices and social structure. Similar to earlier Avar communities in Pannonia67,76, the multi-generational SP communities we studied in Eastern Germany practiced a consistent reproductive strategy based on patrilineal and local descent, female exogamy, strict avoidance of consanguinity, and—in several cases—multiple reproductive partners. Not all of these patterns observed in Eastern Germany were shared with the SP societies in the Northern Balkans, which, in many aspects of their organization, appear more similar to preceding MP groups in Croatia, Eastern Germany, Hungary or Italy39,40. In some of these cases, previously established social practices may have survived the substantial demographic and linguistic changes, adding another level of genetic complexity to the new set of questions concerning social organization and interaction within and between Slavic-related societies across Europe. In modern ethnic and national terminology, ‘Slavs' denotes all speakers of Slavic languages and/or citizens of the Slavic nation states. This concept of an ethnic collective spanning several nations is much more marked than among the Germanic or Romance speakers in the rest of Europe (Supplementary Note 1). The extent to which Slavic identity mattered diverged; it was important for nineteenth and twentieth century Slavic nationalisms, Pan-Slavism and Russian imperialism, but regional or national allegiances often carried more weight. The prejudices of their western neighbours who tended to regard Slavs as culturally inferior reinforced sentiments of Slavic commonality. It is therefore important to be precise in the scholarly use of the term. In written sources since the sixth and seventh century in Byzantium and the west, groups of Slavs or Wends increasingly appear in a wide range of lands beyond and along the Danube and the Elbe rivers. We can make use of different sources to understand how large parts of Europe became Slavic: outside perceptions about Slavs in texts; archaeological traces of shared cultural practices among early Slavs (particularly the Prague-Korchak culture); linguistic reconstructions of a common Slavic language prior to the particular Slavic idioms; and shifts in ancestry of the medieval gene pool, which point to migrations. We should not take these disciplinary results as proxies for each other as attributes of a coherent people called Slavs; yet, they provide different perspectives on the Slavicisation of Europe during the Early Middle Ages. Combining them allows us to overcome simplistic theories of an expansion of the Slavs and instead understand the common dynamic and the different ways in which Slavic peoples began to form in many parts of Europe. We therefore use Slavs for populations named in this way in contemporary texts, without implying that they self-identified as such. These Slavic groups can be localized, but hardly circumscribed. We do not use genetic or archaeological features in regions where Slavs spread to distinguish between Slavs and non-Slavs, or between speakers and non-speakers of Slavic languages, although we assume that these phenomena overlapped to a considerable degree. For better comparison of genetic differences between groups pre- and post-dating the proposed expansion of groups associated with Slavic material culture, we refer to samples dating between 300 and 600 ce as MP individuals or groups, and samples dating between 600 and 1200 ce as SP individuals or groups. Although this naming scheme is simplified compared to the well-established historical and archaeological chronologies of our three study transects, it facilitates the quantification of the genetic turnover between groups pre- and post-dating 600 ce across Central Europe. We highlight that our SP samples were selected to represent sites that are considered Slavic in an archaeological sense. Besides this cultural attribution, sites were selected for their completeness and state of archaeological description or publication. However, we do not know if the buried individuals considered themselves as belonging to a Slavic ethnic or cultural community and whether or how they understood their common descent from groups of earlier periods in Eastern Europe. Similarly, whether they spoke a Slavic language can only be hypothetically assumed. We thus prefer a temporal nomenclature that is agnostic as to genetic, cultural or linguistic designations. When necessary, we refer collectively to SP samples with Eastern Europe-specific ancestry as having SP ancestry. This term describes an Eastern European-derived ancestry component due to a recent common origin shared among most SP individuals in all three study transects. In our analyses this ancestry is approximated using data (n = 8) from the Polish site Gródek, Hrubieszów County, dating between 600 and 900 ce and representing the spatially and temporally closest surrogate in our dataset to the (so far) unsampled source population in Eastern Europe. This naming system is intended to differentiate medieval SP ancestry from the present-day genetic landscape of Eastern Europe, which is (at least partially) derived from this ancient source. We use the term with intention since the observed movements caused significant and lasting demographic change in the genetic landscape of Europe due to the large-scale, permanent translocation of people from one region to another. Within tables and figures, we refer to groups of published individuals by the names given in the Allen Ancient DNA resource v.54.180. Sample sizes, context information and publication names can be found in Supplementary Tables 7–11. In the main text and Supplementary Notes, we used the following abbreviations for archaeological time periods: N, Neolithic; C, Chalcolithic; EBA, Early Bronze Age; MBA, Middle Bronze Age; LBA, Late Bronze Age; IA, Iron Age; RA, Roman Age; EMA, Early Middle Ages; MA, Middle Ages; H, historical. Provenance information for samples from all archaeological sites is given in Supplementary Information Note 2, together with short descriptions of each site, the institution owning the samples (or custodians of the samples), the responsible co-author who obtained permission to analyse and the year of the permission granted. We obtained all permissions for the work with archaeological and anthropological material in this study. Sampling of bone and teeth samples took place in the clean room facilities of the ArcheoGen laboratory at the Department of Archeology and Museology, Masaryk University in Brno, Czech Republic as well as in the clean room facilities of the Max Planck Institute for Evolutionary Anthropology. For teeth, we either cut along the cementum–enamel junction and collected powder by drilling into the pulp chamber or accessed the pulp chamber by drilling the tooth transversally. Other skeletal material such as long bones were used when no teeth or petrous bones were available. In these cases, areas with best preserved compact bone tissue were drilled. We collected between 25 and 50 mg of bone or tooth powder per sample for DNA extractions. The pulverized bone samples were sent to clean room facilities dedicated to ancient DNA work at the Max Planck Institute for Evolutionary Anthropology for further processing and aDNA acquisition. New radiocarbon dates for this study were measured on the bone and tooth fragments sampled for DNA. These dates were obtained at the Curt-Engelhorn-Center Archaeometry gGmbH, Mannheim, using MICADAS-AMS. Collagen was extracted from the previously sampled bones, purified by ultrafiltration and freeze-dried. Ancient DNA was extracted following a modified protocol of Dabney et al.84, as described at https://www.protocols.io/view/ancient-dna-extraction-from-skeletal-material-baksicwe, where we replaced the extended-MinElute-column assembly for manual extractions with columns from the Roche High Pure Viral Nucleic Acid Large Volume Kit85, and for automated extraction with a protocol that replaced spin columns with silica beads in the purification step86. We generated double-indexed87 single-stranded88 libraries using 25 µl of DNA extract and following established protocols89. We enriched libraries using in-solution capture probes synthesized by Agilent Technologies for 1,237,207 SNPs along the nuclear genome91. Libraries were sequenced in-house on an Illumina HiSeq 4000 platform to an average depth of 20 million reads and after demultiplexing processed through EAGER92. After demultiplexing based on a unique pair of indexes, raw sequence data were processed using EAGER92. This included clipping sequencing adaptors from reads with AdapterRemoval (v.2.3.1)93 and mapping of reads with BWA (Burrows–Wheeler aligner) v.0.7.1294 against the Human Reference Genome Hs37d5, with seed length (-l) disabled, maximum number of differences (-n) of 0.01 and a quality filter (-q) of 30. We removed duplicate reads with the same orientation and start and end positions using DeDup v.0.12.292. Terminal base deamination damage calculation was done using mapDamage v.2.0.695, specifying a length (-l) of 100 bp. For the ten libraries that underwent UDG half treatment, we used BamUtil v.1.0.14 (https://genome.sph.umich.edu/wiki/BamUtil:_trimBam) to clip two bases at the start and end of all reads for each sample to remove residual deaminations, thus removing genotyping errors that could arise due to ancient DNA damage. For that, we used a custom script (https://github.com/TCLamnidis/Sex.DetERRmine) for the calculation of each relative coverage as well as their associated error bars97. Females are expected to have an X rate of 1 and a Y rate of 0, while males are expected to have a rate of 0.5 for both X and Y chromosomes. We used the ANGSD (Analysis of Next Generation Sequencing Data) package98 (v.0.923) to test for heterozygosity of polymorphic sites on the X chromosome in male individuals, applying a contamination threshold of 5% at the results of method 1. For male and female samples, we estimated contamination levels on the mtDNA using Schmutzi99 (v.1.5.4) by comparing the consensus mitogenome of the ancient sample to a panel of 197 worldwide mitogenomes as a potential contamination source, applying a contamination threshold of 5%. We used the program pileupCaller (v.1.4.0.2) (https://github.com/stschiff/sequenceTools.git) to genotype the trimmed BAM files of 10 UDG half libraries. A pileup file was generated using samtools mpileup with parameters -q 30 -Q 30 -B containing only sites overlapping with our capture panel. From this file, for each individual and each SNP on the 1240k panel36,37,100, one read covering the SNP was drawn at random, and a pseudo-haploid call was made—that is, the ancient individual was assumed homozygous for the allele on the randomly drawn read for the SNP in question. We used the parameter -SingleStrandMode, which causes pileupCaller to ignore reads aligning to the forward strand at C/T polymorphisms and at G/A polymorphisms to ignore reads aligning to the reverse strand, which should remove post-mortem damage in ancient DNA libraries prepared with the non-UDG single-stranded protocol. To maximize our resolution, we filled missing data in the single-stranded libraries with additional genotypes present in the trimmed, double-stranded but not in the single-stranded libraries. To process the mitochondrial DNA data, we extracted reads from 1240k data using samtools (v.1.3.1)101 and mapped these to the revised Cambridge reference sequence (rCRS). We subsequently called consensus sequences using Geneious R9.8.1102 and used HaploGrep 2 (v.2.4.0)103 (https://haplogrep.uibk.ac.at/; with PhyloTree version 17-FU1) to determine mitochondrial haplotypes. We then manually assigned Y-chromosome haplogroups using pileups of Y-SNPs included in the 1240k panel that overlap with SNPs included on the ISOGG SNP index v.15.73 (Y-DNA Haplogroup Tree 2019-2020; 2020.07.11). We used the ancient-DNA-specific genotype caller MLE function of ATLAS104 (https://bitbucket. ATLAS can also calculate the base-quality recalibration (the recal function) that we performed in batches among libraries sequenced in the same sequencing run, accounting for specific sequencing errors. ATLAS recalibration also corrects the base qualities accounting for the empirical ancient DNA damage pattern observed from the data and reduces the effect of reference bias introduced by genome mapping by relying on a list of 10 million highly conserved genomic positions across 88 mammal species downloaded from ensembl (https://grch37.ensembl.org/). We called genotype likelihoods on the whole 1000 Genomes SNPs panel of around 20 million SNPs and used these calls as input data for GLIMPSE105 (v.2.0.0) (https://github.com/odelaneau/GLIMPSE), applying the default parameters and using the 1000 Genomes reference panel. The function GLIMPSE_phase was used to perform simultaneous imputation and phasing on genomic chunks of 2,000,000 base pairs with a buffer of 200,000 base pairs. Samples with more than 600k SNPs exhibiting a genotype posterior of ≥0.99 after imputation were included in downstream IBD analysis. We then estimated segments that were IBD either in the context of: (1) other ancient individuals; or (2) present-day populations. To investigate IBD sharing between pairs of ancient individuals we used the program ancIBD43. We applied the HapBLOCK function of ancIBD to perform the pairwise estimation with default parameters and only shared blocks of more than 8 cM containing more than 220 SNPs per centimorgan were considered. To further filter for possible false-positive hits in our IBD network analysis, we considered only shared IBD segments longer than 12 cM, and if a pair of individuals had segments of less than 16 cM, we included them only if they had more than one such segment. To compare IBD segments shared with present-day individuals as well as estimates of effective population size, we used BEAGLE106,107 (v.5.2) to phase the newly imputed genotypes. Following Morez et al.108, the window and overlap lengths were set as wider than any chromosome (window length 380 cM and overlap length 190 cM) to maximize the information used for phasing the genomes. The 1000 Genomes phase 3 dataset (http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a) and GRCh37 genomic maps (http://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/) provided by BEAGLE were used for phasing. We phased ancient and present-day data together since the BEAGLE phasing algorithm (hidden Markov model-based haplotype clustering) improves widely as the sample size increases. The identification of IBD segments was done using RefinedIBD109, which can also detect IBD fragments shorter than 8 cM. The window size was set to 3 cM. The minimal size for a segment to be considered shared by IBD is 1 cM, the same threshold used in Margaryan et al.110 and Morez et al.108. Finally, we removed gaps between IBD segments that have at most one discordant homozygote and that are less than 0.6 cM in length and aggregated the sum and number of IBD segments between each pair of ancient and present-day individuals. To infer biological relatedness between individuals, we applied two independent approaches: We calculated the pairwise mismatch rate111 in all pairs of individuals from our pseudo-haploid dataset to double-check for potential duplicate individuals and to determine first-, second- and third-degree relatives. For this purpose, we also used BREADR112 which utilizes Bayesian posterior probabilities for the classification of the genetic relationships. We estimated the parts of the genome that are IBD in all pairs of individuals. Both methods use genotype likelihoods to estimate the three k coefficients (k0, k1 and k2), which define the probability that two individuals have zero, one or two alleles that are IBD at a random site in the genome. We calculated estimates of genetic diversity and effective population size using three different approaches. We calculated the length of runs of homozygosity (RoH) using the software HapROH115 (v.0.6) in our pseudo-haploid data. A SNP cutoff of 300k SNPs was used, as well as the default 1000 Genomes reference panel. The number of RoH of size 4–8 cM per individual reflects the rate at which distant relatives have children, providing information about the sizes of mate pools (Ne) averaged over the hundreds of years prior to when individuals lived; offspring of close kin unions are reflected by sums of RoH >50 cM across the genome in runs of homozygosity of >12 cM. The ancestry variability is measured in the ratio \({F}_{{\rm{ST}}}/{F}_{{\rm{ST}}}^{\text{max}}\) (where \({F}_{{\rm{ST}}}^{\text{max}}\) is the maximal value of the fixation index FST). We generated 1,000 bootstrap replicate matrices for computing \({F}_{{\rm{ST}}}/{F}_{{\rm{ST}}}^{\text{max}}\) to compare bootstrap means and identify significantly different variabilities. We used the method IBDNe117 to estimate ancestry-specific historical effective population size from around 4 generations to around 50 generations ago using identity-by-descent segments inferred from our imputed diploid data. We removed IBD segments shorter than 2 cM as well as IBD segments from avuncular and closer relationships using the parameter filtersamples: true. Confidence intervals for the estimated effective population size were calculated using 500 bootstrap replicates. We merged our ancient DNA data with previously published datasets of ancient individuals reported by the Reich Lab in the Allen Ancient DNA Resource v.54.1 (https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data) (1240k SNP panel)80. For comparisons with present-day groups, we compiled and curated a high-resolution, quality-filtered reference dataset containing genotypes for 426,135 SNPs (the intersection of several different Affymetrix and Illumina chip types) from 12,176 contemporary individuals sampled from 49 (mostly European and West Asian) populations from previously published datasets as described in Gretzinger et al.68. Sample sizes are given in Supplementary Table 3. The whole 1240k panel (1240 K; 1.2 M SNPs). Used exclusively for comparison between ancient individuals as well as ancIBD and qpAdm analysis. Used for HO West Eurasian PCA. Used for most analyses including ancient and present-day populations, e.g. PCA, F4 statistics and ADMIXTURE. Used for the calculation of SP ancestry proportions in present-day populations as well as FST and RefinedIBD analysis. We computed principal components on three different sets of European and West Asian populations and projected ancient individuals using lsqproject: YES and shrinkmode: YES. Poplist: Abkhasian, Adygei, Albanian, Armenian, Balkar, Basque, BedouinA, BedouinB, Belarusian, Bulgarian, Canary_Islander, Chechen, Chuvash, Croatian, Cypriot, Czech, Druze, English, Estonian, Finnish, French, Georgian, Greek, Hungarian, Icelandic, Iranian, Italian_North, Italian_South, Jew_Ashkenazi, Jew_Georgian, Jew_Iranian, Jew_Iraqi, Jew_Libyan, Jew_Moroccan, Jew_Tunisian, Jew_Turkish, Jew_Yemenite, Jordanian, Kumyk, Lebanese, Lezgin, Lithuanian, Maltese, Mordovian, North_Ossetian, Norwegian, Orcadian, Palestinian, Polish, Russian, Sardinian, Saudi, Scottish, Sicilian, Spanish, Spanish_North, Syrian, Turkish, Ukrainian. Poplist: Norway, Sweden, Denmark, Netherlands, Belgium, Germany, England, Wales, Scotland, NIreland, Ireland, Orkney, France, Spain, Portugal, Finland, Lithuania, Latvia, Estonia, Mordovia, Russia, Belarus, Poland, Ukraine, Slovakia, Hungary, Italy, Slovenia, Croatia, Bosnia, Serbia, Romania, Bulgaria, Montenegro, North Macedonia, Albania, Greece. Poplist: Norway, Sweden, Denmark, Netherlands, Belgium, NGermany, England, Wales, Scotland, NIreland, Ireland, Orkney, France, Finland, Lithuania, Latvia, Estonia, Mordovia, Russia, Belarus, Poland. F3 and F4 statistics were computed with ADMIXTOOLS v.3.0119 (https://github.com/DReichLab). Significant deviation from zero can be interpreted as rejection of the tree population typology ((Outgroup, X);(Pop1, Pop2)). Under the assumption that no gene flow occurred between Pop1 and Pop2 and the Outgroup, a positive F-statistic suggests affinity between X and Pop2, whilst a negative value indicates affinity between X and Pop1. Standard errors were calculated with the default block jackknife 5 cM in size. As outgroups for F3 and F4 statistics, we either used haploid genotypes from YRI or CHB. We calculated FST using smartpca software v.16000 from the EIGENSOFT package (v.6.0.1)118 with the fstonly, inbreed, and fsthiprecision options set to YES. We estimated ancestry proportions using qpWave36,120 (v.410) and qpAdm36 (v.810) from ADMIXTOOLS v.3.0119 with the allsnps: YES and inbreeding: YES options. Standard errors were calculated with the default block jackknife 5 cM in size. We used three basic sets of outgroups: This set was adapted from Patterson et al.79 and used to infer ancestry components from WHGs, EEFs and Yamnaya/Poltavka pastoralists (OldSteppe). YRI.SG, Poland, Finland, Sweden, Denmark, Ireland, Wales, Italy, Spain, Belgium and the Netherlands68. This set was adapted from Gretzinger et al.68 and used to infer ancestry components from SP individuals in present-day Central and Eastern Europeans. OldAfrica, WHGB, Russia_Ust_Ishim.DG, CHG, EHG, Iran_GanjDareh_N, Israel_Natufian_published, Jordan_PPNB, Laos_Hoabinhian, OldSteppe, Turkey_N, Russia_MA1_HG, Morocco_Iberomaurusia. This set was adapted and modified from Antonio et al.34 to investigate the formation of the SP gene pool. To analyse potential sex bias in the admixture process, we used qpAdm to estimate SP admixture proportions on the autosomes (default option) and on the X chromosome (option “chrom: 23”) using the abovementioned outgroups. Thus, a negative z-score means that there is more SP admixture on the X chromosome than on the autosomes, indicating that the SP admixture was female-biased. We performed model-based clustering analysis using two different approaches in ADMIXTURE: We applied ADMIXTURE41 in unsupervised mode using modern and ancient individuals at K = 2 to 12. Variants with minor allele frequency of 0.01 were removed and PLINK was used for linkage disequilibrium (LD) pruning with a window size of 200, a step size of 5 and an R2 threshold of 0.5. We applied ADMIXTURE41 in supervised mode using modern reference populations at K = 12. This analysis was run on haploid data with the parameter “haploid” set to all (=“*”). To obtain point estimates for populations, we averaged individual point estimates and calculated the s.e.m. The Q matrix of this ADMIXTURE analysis was also used as input for FSTruct as described by the authors116. We constructed maximum likelihood trees using TreeMix (v.1.12)122. For each tree, we performed a round of global rearrangements after adding all populations (-global) and calculated 100 bootstrap replicates to assess the uncertainty of the fitted model (-bootstrap). Sample size correction was disabled. Admixture dates were calculated using DATES (Distribution of Ancestry Tracts of Evolutionary Signals) (v.4010)56, which calculates the decay of ancestry covariance coefficients between every pair of available overlapping SNPs between the test individuals and the source populations over increasing-genetic-distance window. We used standard settings with default bin size of 0.001 Morgans applied in our estimates (flag “binsize: 0.001”). We used a standard of 29 years per generation to convert the generation times in years since admixture. We use Bayesian phylogenetic inference to estimate the ages of language divergence as described78. Analyses were performed on the IE-CoR database that stores data on cognate relationships (shared word origin) between 161 Indo-European languages, in a reference set of 170 basic meanings (https://iecor.clld.org). Divergence date distributions for the Balto-Slavic and Slavic subgroups were extracted from the sample of 37,004 trees resulting from the main analysis of Heggarty et al.78. A critical evaluation of this approach is discussed in Supplementary Note 9. However, we highlight that these computational estimates for the splits of the Baltic and Slavic languages compare well with estimates produced by methods of traditional Indo-European linguistics. These estimates are based on the reconstructed rapidity of diversification relative to that observed in the other branches of Indo-European (some of which is recorded in writing thousands of years before Baltic and Slavic), the nature of contacts and convergence between Balto-Slavic and other languages (IE and non-IE), and the character of the reconstructible Proto-Balto–Slavic vocabulary—but also attempted links to the archaeological picture57. The dating of the split of Proto-Balto–Slavic is generally in agreement with the results of traditional historical linguistics as described above; and earlier glottochronological approaches have yielded similar dating—for example, an estimation at the fifteenth and fourteenth century bce123. All software used in this work is publicly available. Data visualization and descriptive statistical tests were performed in R (v.4.1.1). The following R packages were used: Rsamtools (v.2.12.0), vegan (v.2.6-2), factoextra (v.1.0.7), ggplot2 (v.3.3.6), ggExtra (v.0.10.0), ggforce (v.0.3.3), rnaturalearth (v.0.1.0), sf (v.1.0.-8), raster (v.3.5-21), rgdal (v.1.5-32), spatstat (v.2.3-4), maptools (v.1.1-4), gstat (v.2.0-9), sp (v.1.5-0), labdsv (v.2.0-1), rcarbon (v.1.5.1), magrittr (v.2.0.3), dplyr (v.1.0.9), reshape 2 (v.1.4.4), and tidyverse (v.1.3.2). Y-chromosome and mtDNA haplogroups were determined using the ISOGG SNP index (v.15.73) and PhyloTree (v.17-FU1) reference databases, respectively. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Unmapped, raw sequencing data (fastq files) from the newly reported ancient individuals will be available on publication from the European Nucleotide Archive under accession number PRJEB81250. A poseidon package of the genotype data analysed in this paper is available on the Poseidon Community Archive (https://www.poseidon-adna.org/#/archive_explorer). Previously published genotype data for ancient and present-day individuals was reported by the Reich laboratory in the Allen Ancient DNA Resource v.54.1 (https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data). The Genome Reference Consortium Human Build 37 (GRCh37/hg19) is available via the National Center for Biotechnology Information under accession number PRJNA31257. The revised Cambridge reference sequence is available via the National Center for Biotechnology Information under NCBI Reference Sequence NC_012920.1. Published genotype data for the present-day British sample are available from the Wellcome Trust Case Control Consortium (WTCCC) via the European Genotype Archive (https://www.ebi.ac.uk/ega/) under accession number EGAD00010000634. Published genotype data for the present-day Irish sample are available from the WTCCC via the European Genotype Archive under accession number EGAD00010000124. Published genotype data for the present-day Lithuanian sample are available from https:// figshare.com/articles/Patterns_of_genetic_ structure_and_adaptive_positive_selection_in_ the_Lithuanian_population_from_ high-density_SNP_data/7964159. Pohl, W. The Avars: A Steppe Empire in Central Europe, 567–822 (Cornell Univ. Dulinicz, M. & Moździoch, S. The Early Slavic Settlement in Central Europe in the Light of New Dating Evidence, Vol. Parczewski, M. Die Anfänge Der Frühslawischen Kultur in Polen, Vol. 17 (Österreichische Gesellschaft für Ur- und Frühgeschichte, Wien, 1993). Kazanski, M. in Encyclopedia of Slavic Languages and Linguistics Online (ed. Szmoniewski, B. Ethnogenesis of Slavs viewed from Polish perspective. Pohl, W. Die Germanen (De Gruyter, 2000). Halsall, G. Barbarian Migrations and the Roman West, 376–568 (Cambridge Univ. Meier, M. Geschichte Der Völkerwanderung: Europa, Asien Und Afrika, Vol. Wolfram, H. History of the Goths (Univ. Volk Und Herrschaft Im Frühen Mittelalter. Castritius, H., Geuenich, D. & Werner, M. Die Frühzeit Der Thüringer: Archäologie, Sprache, Geschichte, Vol. Frühmittelalter Zwischen Harz Und Elbe, Vol. Bemmann, J. in Die Frühzeit der Thüringer (eds Castritius, H., Geuenich, D. & Werner, M.) 63–82 (De Gruyter, 2009). Brachmann, H. Slawische Stämme an Elbe Und Saale. Zu Ihrer Geschichte Und Kultur Im 6. Parczewski, M. Origins of early Slav culture in Poland. Kaczanowski, P. & Parczewski, M. (eds) Archeologia O Początkach Słowian. Curta, F. in Migration Histories of the Medieval Afroeurasian Transition Zone 101–138 (Brill, 2020). Curta, F. Slavs in the Making History, Linguistics, and Archaeology in Eastern Europe (ca. Kushniarevich, A. et al. Genetic heritage of the Balto-Slavic speaking populations: a synthesis of autosomal, mitochondrial and Y-chromosomal data. Olalde, I. et al. A genetic history of the Balkans from Roman frontier to Slavic migrations. Peltola, S. et al. Genetic admixture and language shift in the medieval Volga-Oka interfluve. Stolarek, I. et al. Genetic history of East-Central Europe in the first millennium ce. Barford, P. M. The Early Slavs: Culture and Society in Early Medieval Eastern Europe (Cornell Univ. Lübke, C. in Neue Wege der Frühmittelalterforschung. Bilanz und Perspektive (eds Pohl, W., Diesenberger, M. & Zeller, B.) Brather, S. Archäologie Der Westlichen Slawen. Gross, A. et al. Population-genetic comparison of the Sorbian isolate population in Germany with the German KORA population using genome-wide SNP arrays. Stable population structure in Europe since the Iron Age, despite high mobility. Ancient Rome: a genetic crossroads of Europe and the Mediterranean. Massive migration from the steppe was a source for Indo-European languages in Europe. Lazaridis, I. et al. Genomic insights into the origin of farming in the ancient Near East. Schmid, C. & Schiffels, S. Estimating human mobility in Holocene Western Eurasia with large-scale ancient genomic data. Amorim, C. E. G. et al. Understanding 6th-century barbarian social organization and migration through paleogenomics. Vyas, D. N. et al. Fine-scale sampling uncovers the complexity of migrations in 5th–6th century Pannonia. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Ancient DNA reveals reproductive barrier despite shared Avar-period culture. Accurate detection of identity-by-descent segments in human ancient DNA. Ralph, P. & Coop, G. The geography of recent genetic ancestry across Europe. Al-Asadi, H., Petkova, D., Stephens, M. & Novembre, J. Estimating recent migration and population-size surfaces. Stolarek, I. et al. Goth migration induced changes in the matrilineal genetic structure of the central-east European population. Kokowski, A. Gothic migrations: in search of the truth. Veeramah, K. R. et al. Genetic variation in the Sorbs of eastern Germany in the context of broader European genetic diversity. Higounet, C. Die Deutsche Ostsiedlung Im Mittelalter (Siedler, 1986). Bünz, E. Ostsiedlung Und Landesausbau in Sachsen. Die Kührener Urkunde von 1154 Und Ihr Historisches Umfeld, Vol. Lübke, C. in The Making of Medieval History (eds Loud, G. A. The arrival of Siberian ancestry connecting the Eastern Baltic to Uralic speakers further east. de Gennaro, L. et al. PANE: fast and reliable ancestral reconstruction on ancient genotype data with non-negative least square and principal component analysis. Speidel, L. et al. High-resolution genomic history of early medieval Europe. Chintalapati, M., Patterson, N. & Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. Pronk, T. in The Indo-European Language Family. Villanueva Svensson, M. The Rise of Acuteness in Balto-Slavic (Brill, 2023). Derksen, R. in Encyclopedia of Slavic Languages and Linguistics Online (ed. Matasović, R. Toward a relative chronology of the earliest Baltic and Slavic sound changes. Young, S. in The Indo-European Languages (ed. in Handbook of Comparative and Historical Indo-European Linguistics (eds Klein, J., Joseph, B. Olander, T. Proto-Slavic Inflectional Morphology (Brill, 2015). Hellenthal, G. et al. A genetic atlas of human admixture history. Barbieri, C. et al. A global analysis of matches and mismatches between human genetic and linguistic histories. Gnecchi-Ruscone, G. A. et al. Network of large pedigrees reveals social practices of Avar communities. Macháček, J. et al. Runes from Lány (Czech Republic)—The oldest inscription among Slavs. A new standard for multidisciplinary analysis of runic bones. Bichlmeier, H. in New Perspectives on the Early Slavs and the Rise of Slavic: Contact and Migration (eds Klír, T., Boček, V. & Jansens, N.) 43–76 (Univ. Donat, P. & Fischer, R. E. Die Anfänge slawischer Siedlung westlich der Oder. Methodische Überlegungen zu Problemen aktueller archäologischer und onomastischer Forschungen. Leube, A. Germanische Völkerwanderungen und ihr archäologischer Fundniederschlag II. East Central and Eastern Europe in the Middle Ages (Brill, 2020). Biermann, F. in Die Frühen Slawen—Von der Expansion zu Gentes und Nationes, Vol. Dulinicz, M. Frühe Slawen Im Gebiet Zwischen Unterer Weichsel Und Elbe. Ancient genomes reveal origin and rapid trans-Eurasian migration of 7th century Avar elites. Language trees with sampled ancestors support a hybrid model for the origin of Indo-European languages. Large-scale migration into Britain during the Middle to Late Bronze Age. The Allen Ancient DNA Resource (AADR) a curated compendium of ancient human genomes. Orfanou, E., Himmel, M., Aron, F. & Haak, W. Minimally-invasive sampling of pars petrosa (os temporale) for ancient DNA extraction V.2. The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kbp). Bronk Ramsey, C. Radiocarbon calibration and analysis of stratigraphy: the OxCal program. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. Korlević, P. et al. Reducing microbial and human contamination in DNA extractions from ancient bones and teeth. & Meyer, M. Extraction of highly degraded DNA from ancient bones, teeth and sediments for high-throughput sequencing. Kircher, M., Sawyer, S. & Meyer, M. Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform. Gansauge, M.-T. & Meyer, M. Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA. Meyer, M. & Kircher, M. Illumina sequencing library preparation for highly multiplexed target capture and sequencing. Rohland, N., Harney, E., Mallick, S., Nordenfelt, S. & Reich, D. Partial uracil-DNA-glycosylase treatment for screening of ancient DNA. Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. EAGER: efficient ancient genome reconstruction. Schubert, M., Lindgreen, S. & Orlando, L. AdapterRemoval v2: rapid adapter trimming, identification, and read merging. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Jónsson, H., Ginolhac, A., Schubert, M., Johnson, P. L. F. & Orlando, L. mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters. Ancient Fennoscandian genomes reveal origin and spread of Siberian ancestry in Europe. & Nielsen, R. ANGSD: analysis of next generation sequencing data. Renaud, G., Slon, V., Duggan, A. T. & Kelso, J. Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA. Ancient human genomes suggest three ancestral populations for present-day Europeans. Kearse, M. et al. Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Link, V. et al. ATLAS: analysis tools for low-depth and ancient samples. Efficient phasing and imputation of low-coverage sequencing data using large reference panels. Browning, S. R. & Browning, B. L. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. & Browning, S. R. Fast two-stage phasing of large-scale sequence data. Morez, A. et al. Imputed genomes and haplotype-based analyses of the Picts of early medieval Scotland reveal fine-scale relatedness between Iron Age, early medieval and the modern people of the UK. Browning, B. L. & Browning, S. R. Improving the accuracy and efficiency of identity-by-descent detection in population data. Margaryan, A. et al. Population genomics of the Viking world. Kennett, D. J. et al. Archaeogenomic evidence reveals prehistoric matrilineal dynasty. B. et al. BREADR: an R package for the Bayesian estimation of genetic relatedness from low-coverage genotype data. Popli, D., Peyrégne, S. & Peter, B. M. KIN: a method to infer relatedness from low-coverage ancient DNA. Lipatov, M., Sanjeev, K., Patro, R. & Veeramah, K. R. Maximum likelihood estimation of biological relatedness from low coverage sequencing data. & Steinrücken, M. Parental relatedness through time revealed by runs of homozygosity in ancient DNA. Morrison, M. L., Alcala, N. & Rosenberg, N. A. FSTruct: an FST-based tool for measuring ancestry variation in inference of population structure. Browning, S. R. & Browning, B. L. Accurate non-parametric estimation of recent effective population size from segments of identity by descent. Patterson, N., Price, A. L. & Reich, D. Population structure and eigenanalysis. Reich, D. et al. Reconstructing Native American population history. The genomic history of southeastern Europe. Pickrell, J. K. & Pritchard, J. K. Inference of population splits and mixtures from genome-wide allele frequency data. Novotná, P. & Blažek, V. Glottochronology and its application on the Balto-Slavic languages. Data were produced by the Ancient DNA Core Unit of the Max Planck Institute for Evolutionary Anthropology which is funded by the Max Planck Society and we are thankful to the laboratory personnel for processing our samples. This project has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (HistoGenes, grant agreement 856453 ERC-2019-SyG) and the Max Planck Society. acknowledge financial support for this research by the Finnish Cultural Foundation, Jane and Aatos Erkko Foundation, and the Kone Foundation. are supported by the Czech Science Foundation (grant: GA ČR EXPRO GX21-17092X “The Formation of Multi-ethnic Complex Societies in Early Medieval Moravia. Collective Action Theory and Interdisciplinary Approach (FORMOR)”). are supported by the Czech Ministry of Education, Youth and Sports (RES-HUM project; CZ.02.01.01/00/22_008/0004593). The authors thank A. Tönjes and P. Kovacs for providing access to genomic data from the Sorbian minority in Germany; the Sorbian community in Bautzen for helpful discussions; and K. Prüfer for help with data procession. Open access funding provided by Max Planck Society. These authors contributed equally: Felix Biermann, Hellen Mager These authors jointly supervised this work: Zuzana Hofmanová, Johannes Krause Department of Archaeogenetics, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany Joscha Gretzinger, Hellen Mager, Luca Traverso, Guido A. Gnecchi Ruscone, Sanni Peltola, Elina Salmela, Gunnar U. Neumann, Rita Radzeviciute, Zuzana Hofmanová & Johannes Krause State Office for Heritage Management and Archaeology Saxony-Anhalt and State Museum of Prehistory, Halle, Germany Felix Biermann, Jörg Orschiedt, Arnold Muhl, Ralf Schwarz & Harald Meller Department of Linguistic and Cultural Evolution, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany Department of Archaeology and Museology, Masaryk University, Brno, Czech Republic Denisa Zlámalová, Guido A. Gnecchi Ruscone, Pavlína Ingrová, Päivi Onkamo, Jiří Macháček & Zuzana Hofmanová Laboratory of Anthropology, Institute of Zoology and Biomedical Research, Jagiellonian University, Krakow, Poland Doctoral School of Humanities and Art, Maria Curie-Skłodowska University, Lublin, Poland Institute of Archaeology, Maria Curie-Skłodowska University, Lublin, Poland Stanisław Staszic Museum, Hrubieszów, Poland Leibniz Institute for the History and Culture of Eastern Europe, Leipzig, Germany Institute of Archeology, Jagiellonian University, Krakow, Poland State Archaeology Department of Schleswig-Holstein, Schleswig, Germany Max Planck-Harvard Research Center for the Archaeoscience of the Ancient Mediterranean, Initiative for the Science of the Human Past at Harvard, Department of History, Harvard University, Cambridge, MA, USA Institute of Archaeological Sciences, Eötvös Loránd University, Budapest, Hungary Institute for Advanced Study, Princeton, NJ, USA Croatian Academy of Sciences and Arts, Zagreb, Croatia Institute for Medieval Research, Austrian Academy of Sciences, Vienna, Austria Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar AH, AM, B. Bartecki, B. Borowska, B.N.-W., E.M., E.S., F.B., F.D., G.Z., H. Meller, I.W., J. Květina, J.M., J.O., J.W., M.K., M. Majer, M. McCormick, M.S., M. Wojenka, M. Wołoszyn, P.O., R.J., R.L., R.S. provided archaeological samples and context information. wrote the paper with input from all co-authors. Correspondence to Joscha Gretzinger, Walter Pohl, Zuzana Hofmanová or Johannes Krause. The authors declare no competing interests. Nature thanks Steffen Patzold, Ron Pinhasi, Pontus Skoglund and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The timeline lists major historical events associated with the Slavs in Central Europe. The map features schematized historical attestations for the appearance of Slavs (Sklavenoi – Slavi – Winedi). a, Temporal distribution of samples analysed in this study, with site occupancy ranges indicated by bars. Non-transparent symbols indicate radiocarbon-dated samples; transparent symbols are scattered uniformly along site occupancy ranges. b, Temporal distribution of published samples analysed in this study. c, Spatial distribution of Eastern German sites analysed in this study. a, West Eurasian PCA (present-day individuals are depicted as grey background points) with projection of BRC (n = 56), DRH (n = 49), OBM (n = 30), and RTW (n = 15) and other relevant Iron Age, Roman Period, and Late Antique populations (coloured symbols). b, Schematic overview of the geographic origin of ancient individuals with South European ancestry from Eastern Germany. The contours indicate the averaged maximum probability at search time 200 CE (multiplied by 100) for 10 individuals from BRC, DRH, OBM and RTW (denoting the mean prediction of the geographic regions where the ancestors of these individuals originated) as inferred using MOBEST. Results from two-way qpAdm analyses are shown as pie charts for each site. c, Schematic distribution of different artifact types among BRC, DRH, and OBM burials. The respective ancestry composition of each individual in respect to northern (WBI, CNE, NOR, BAL & FIN) and southern ancestry (CWE, WAS, NEA & AFR) (from supervised ADMIXTURE) is indicated as bar chart. a, Trajectories of Northeastern European (BAL), East Mediterranean (WAS & NEA), and Northwest European (CNE & NOR) ancestry in ancient individuals from the Northwestern Balkans (n = 301), Eastern Germany (n = 483), and Poland-Northwestern Ukraine (n = 489), as measured using a supervised clustering approach implemented in ADMIXTURE. Error bands represent two standard errors. b, Averaged ancestry estimates from supervised ADMIXTURE analyses are shown as pie charts for sites in Central Europe before (n = 268) and after 600 CE (n = 975). Northwest European ancestry (CNE & NOR) is shown in blue, Northeastern European ancestry (BAL) in violet, and other ancestry in grey. a, Left: map highlighting the “Slavic Period” IBD-sharing community (1.1) (n = 602) identified applying a hierarchical cluster detection approach to a network constructed from pairwise IBD-sharing similarities between 2,657 ancient, post-Neolithic Eurasian individuals. A “Northern SP” sub-cluster (1.1.1) (n = 329) is highlighted in blue, a “Southern SP” sub-cluster (1.1.2) (n = 264) in orange. A boxplot comparison of Latitude coordinates in both clusters is indicated, showing significant differentiation among a North-South gradient (Welch Two Sample t-test; t = 16.331, df = 589.69, p < 2.2e-16). Right, ancient individuals from the Northern SP sub-cluster and Southern SP sub-cluster projected onto the modern European genetic variation. b, Hierarchical cluster analysis applying Ward's minimum variance method to the normalized, average sIBD sharing (> 1 cM) between IBD clusters (n = 24) identified using community detection. The dendrogram and statistical support for the bifurcations from multiscale bootstrap resampling are shown. The ancestry compositions of selected relevant clusters (inferred using supervised ADMIXTURE) are indicated as pie charts. c, Average sIBD (> 1 cM) between the Slavic Period cluster and 23 major IBD clusters identified using community detection. Error bars indicate three standard errors. a, Hierarchical cluster analysis applying Ward's minimum variance method to the normalized, average IBD sharing (> 1 cM) between relevant ancient (n = 14) and present-day (n = 45) populations from Eastern and Northwestern Europe. The dendrogram and statistical support for the bifurcations from multiscale bootstrap resampling are shown. b, Normalized, average IBD sharing (> 1 cM) between SP individuals (n = 182) from Niederwünsch, Eastern Germany, and present-day European, Asian, and North African populations (n = 215). a, Results from qpAdm analyses. Two- and three-way models of ancient source groups were fitted on relevant present-day groups from Southeast, Central, and East Europe (n = 48). The proportion of GRK-related (SP) ancestry is shown. For groups with multiple significant qpAdm models (p > 0.01), the GRK proportions from all feasible models are shown as box plots. b, Pie charts depicting the median ancestry proportions from all significant (p > 0.01), feasible qpAdm models for present-day European groups. GRK-related (SP) ancestry is shown in black, summed non-SP ancestry is shown in white. a, Divergence date distributions for the Balto-Slavic family from other Indo-European languages, extracted from 37,004 trees generated in Heggarty et al.78. Posterior probabilities of sister-group relationship, median split times and respective 95% Highest Posterior Distributions (HPD) are indicated above. Divergence dates are given independently for each of six sister group relationships found in the posterior tree sample, in decreasing order of probability. b, Schematic of the underlying Maximum Clade Credibility (MCC) tree from all tree distributions. The posterior probabilities of Balto-Slavic forming a clade with Germanic-Italic-Celtic (0.63) and Indo-Iranic (0.11) are indicated. c, Divergence date distributions for the Baltic and Slavic languages (left) and the South and East/West Slavic languages (right), extracted from 37,004 trees generated in Heggarty et al.78. Median split times and respective 95% HPD are indicated above. a, Reconstructed pedigrees of Niederwünsch, Eastern Germany, coloured according to family lineages. Numbers correspond to the Genetic IDs listed in Supplementary Table 1. b, Distribution of mtDNA (n = 146) and Y chromosome haplogroups (n = 75). c, Trajectories of BAL ancestry over 5 generations across three major pedigrees of families A (n = 41), B (n = 19) and C (n = 33). a, The network on the left shows all pairs of ancient individuals (n = 674) sharing IBD segments longer than 12 cM. We note that early medieval individuals from England, Northern and Eastern Germany form a loose cluster, while Slavic Period individuals from Croatia, Eastern Germany, and Poland-Northwestern Ukraine plot tightly together. Empty symbols denote reconstructed individuals that are not present in the genetic data. b, The pie charts show the proportion of the total intra-site IBD shared between pairs of males and females for the SP sites Niederwünsch (n = 182), Steuden (n = 36), and Obermöllern (n = 22) in Eastern Germany. Line thickness is proportional to the fraction of shared IBD. Shown below are trends in mtDNA and Y chromosome haplotype diversity (h) calculated for seven MP & SP sites in Eastern Germany and MP & SP Gródek in Eastern Poland. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Gretzinger, J., Biermann, F., Mager, H. et al. Ancient DNA connects large-scale migration with the spread of Slavs. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250903075202.htm'>Metformin's mysterious metal effect could explain its big health benefits</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 13:45:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The widely used diabetes drug metformin changes blood metal levels in humans. Apart from lowering blood sugar levels, it is also known to have a broad range of beneficial side effects such as against tumors, inflammations and atherosclerosis. However, although it has been used for more than 60 years now, its mechanism of action is still not clear, hampering the development of even better drugs against these conditions. Kobe University endocrinologist Wataru Ogawa says: "It is known that diabetes patients experience changes in the blood levels of metals such as copper, iron and zinc. So, we wanted to know whether metformin actually affects blood metal levels in humans, which had not been clarified." To do so, Ogawa and his team enlisted about 200 diabetes patients at Kobe University Hospital, half of which took metformin and half of which did not, in a study to analyze their blood serum levels for those metals and various metal deficiency indicators. Ogawa says: "It is significant that we could show this in humans. Recently, Japan has approved the use of imeglimin, a new diabetes drug that is a derivative of metformin but that should not be able to bind metals the same way as its parent. "Imeglimin is thought to have a different method of action, and we are already conducting studies to compare the effects the two drugs have," says Ogawa. It is not just about understanding the current drugs, however. It was conducted in collaboration with a researcher from the Kagayaki Diabetes and Endocrinology Clinic Sannomiya. Astronomers Discover Mysterious New World at Edge of the Solar System Weight Loss Breakthrough: Scientists Develop Edible “Fat Sponges” From Green Tea and Seaweed Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/military/a65969058/archaeologists-find-bodies-under-nazi-wolfs-lair-hitler/'>Archaeologists Found 5 Mysterious Bodies Under the Floors of Hitler's Secret Lair</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 13:27:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Amateur archaeologists were recently searching for stray artifacts in an abandoned building in Poland when they made a sinister discovery beneath the floorboards: five decaying bodies with their hands and feet missing. The site was an infamous villa that belonged to Nazi Party leader Hermann Göring within Adolf Hitler's secret military headquarters, the Wolfsschanze (Wolf's Lair), which was deserted after a 1944 assassination attempt on Hitler and later partially destroyed by retreating German forces near the end of World War II. “Everyone wondered what might have happened there ... We tried to think of something, but nothing reasonable comes to mind,” Latebra member Dominik Markiewicz told Reuters. Were they some occult rituals of Third Reich fanatics? How did it take so long for the bodies to be discovered? While the Latebra Foundation has been searching the Wolf's Lair site for years, mostly finding “uniform buttons, tools, and machine parts,” per Reuters, the archaeologists only recently focused on Göring's villa due to the sheer amount of ground they had to cover. Hitler ordered its destruction shortly after his last visit in November 1944, with the actual demolition occurring in late January 1945, after the Soviets began their Vistula–Oder offensive. Under Soviet control, the site fell into disrepair and was closed off to any exploration or excavation. Besides, archaeologists wouldn't have necessarily looked to find remains at the Wolf's Lair, as it wasn't a death camp. Designed as Hitler's Eastern Front headquarters, this highly secretive complex served as the strategic center for the Nazi leadership to orchestrate their military campaign against the Soviet Union. Who were the five people buried where Nazi leaders once plotted? A new report in National Geographic pieces together some clues from the initial investigation: (Construction concluded on the compound in June of 1941.) They point to the presence of “dart-shaped stones called belemnites” near the bodies, which were reportedly sometimes placed at pagan burial sites as a “good luck charm.” But belemnite also naturally occurs in that area, so it's tough to definitively interpret the stones' proximity to the graves as evidence of ritualistic practices. Since the corpses date back to a period between 1918 and 1939, the Latebra Foundation plans to proceed with radiocarbon dating. This analysis aims to pinpoint the bodies' time of death more precisely, potentially narrowing it down to a span of just a few years. But, as NatGeo notes, even if these three adults and two children were buried before the Nazis ever set foot within the Wolf's Lair, that also carries a disturbing implication: Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. His stories for the Enthusiast Group have involved coordinating with organizations like the National Parks Service and the Secret Service, and travelling to notable historical sites and archaeological digs, from excavations of America' earliest colonies to the former homes of Edgar Allan Poe. Underwater UFO is a Threat, Says Ex-Navy Officer Stanford Professor: Aliens Are Already Among Us New Documents Reveal Why the USS Thresher Sunk Ex-Navy Official: UFOs Are Hiding in Our Oceans</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65953141/new-lifeform-57-genetic-instructions/'>Scientists Built a New Lifeform With Just 57 Genetic Instructions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists sought to work out genetic errors by creating their own artificial genome, which replaced E. coli's original genome and used less genetic material. The genetic code for just about all forms of life on Earth is fairly universal. Why mess with something that hasn't changed in billions of years? It turns out some codons have extra copies of directions for encoding amino acids and signals for switching the formation of proteins on and off. Chin wanted to see if life could still function with less. But now, he has gone even further with the synthetic organism Syn57—E. Narrowing down the codons that encode protein synthesis can make some organisms resistant to viruses—synthetic genetic codes transferred to organisms can actually block viruses, which genetically alter an organism by injecting cells with RNA that carries codes for producing viral proteins. “Emerging methods for the total synthesis of genomes provide opportunities to explore genome sequences that cannot be accessed by editing,” Chin and his research team said in a study recently published in Science. “Synthetic genomes may be radically different from those accessed by natural evolution, and genome synthesis provides a route to generating sequence and function that is [nonexistant] with respect to extant life.” Previously, other teams of researchers have removed stop codons, which signal protein synthesis to end, from the E. coli genome. This was done through mutagenesis, or creating gene mutations by tweaking DNA. However, mutagenesis has its downsides—it often ends up with excess mutations, and can have difficulty adjusting the numbers of sense codons (which encode amino acids) versus stop codons. Some re-coded groups of codons needed to be tolerated over an entire region of the genome, while in other cases, a codon only needed to be replaced with a viable substitute. Several strains of E. coli were cultured before the final version emerged. As bacterial strains within the lab-created genome cloned themselves, they were analyzed for growth. Taking a closer look at two strains revealed that recoded regions of the genome actually limited growth. In upcoming research, he plans to continue testing the limits of a genome. This synthetic life had its genome replaced instead of being created from scratch. Maybe it could tell us something about how life—and the genes that determine its traits—originated on Earth. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Experts Found First Branch On Animal Tree of Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-local-universe-may-be-misleading-cosmologists-about-dark-energy-and/'>How a Nearby Cosmic Void Could Be Distorting Our Understanding of the Universe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-03 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A Cosmic Void May Be Skewing Our Understanding of the Universe Our understanding of cosmology hinges on how well we know our own local universe, which remains poorly mapped and poorly understood “You can journey to the ends of the earth in search of success,” 19th-century Baptist preacher Russell Conwell is said to have proclaimed, “but if you're lucky, you will discover happiness in your own backyard.” Modern cosmology has stepped far beyond our cosmic backyard. Our surveys stride across the universe, swallowing millions of galaxies at a time. We have mapped and measured the most subtle accelerations of cosmic expansion. But our understanding of all of that hinges on how well we know our own local neighborhood, which remains poorly mapped and poorly understood. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In April 2024 the Dark Energy Spectroscopic Instrument (DESI) collaboration made headlines with a stunning announcement. Data from the first year of a galaxy survey that captured precise measurements of more than 13 million galaxies revealed slight but significant evidence that dark energy may be weakening with time. This new result adds to the growing list of problems faced by the leading model of cosmology, called LCDM for “lambda cold dark matter,” which hypothesizes that around 95 percent of all the stuff in the cosmos is either dark energy or dark matter. But to see out into the wider universe and make these grand measurements, we must first look through the nearby cosmos, and that may bias our observations. So what does our nearby universe look like? They need to map out every region of sky, going as far into the universe as possible, and capture every single galaxy, no matter how small and dim. Most astronomical surveys, however, typically achieve only two out of those three goals. It's a not a particularly deep depression: this local span of the universe looks to be just 10 to 20 percent less dense than the cosmic average. Cosmologists assume, and comprehensive surveys have shown, that the universe is homogenous at large scales, meaning basically the same in all directions. Yet it's not exactly clear where homogeneity kicks in or how far out our local universe can be different without running afoul of LCDM. But we're not exactly sure if the KBC void actually exists. It does appear, given our limited surveys of the nearby universe, that there are indeed fewer galaxies in our billion-light-year patch than there are outside of it. But again, comprehensive astronomical surveys are notoriously tricky. The researchers' tool wasn't a new telescope or instrument, though. So the researchers used Bayesian statistics, a kind of statistical approach that cleanly incorporates prior knowledge and assumptions. They ran simulation after simulation, varying every possible parameter, to build up a suite of mock galaxy surveys that were statistically compatible with the actual data. But there was no sign of the KBC void. But the lesson of these studies is clear. “They are in your own backyard, if you but dig for them.” If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            