
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-28</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/did-astronomers-photograph-ufos-orbiting-earth-in-the-1950s/'>Some Scientists See UFOs in Old Telescope Data. Others See a Teachable Moment</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>UFOs Are Just One Explanation for Mysterious Patterns in Old Telescope Data New peer-reviewed research reporting strange lights in the pre-space-age sky is sparking curiosity and controversy This month prominent peer-reviewed journals have published two papers that link apparent flashes of light seen by a telescope 70 years ago to potential artificial objects in space. “I think there are many in the UFO community who really want to know what's going on,” says Adam Frank, an astrophysicist at the University of Rochester, who has written frequently about the prospects of alien life. “I think it is worthwhile for us to have these open, transparent investigations. This is a great way to show people how science works.” Beatriz Villarroel, an astronomer and theoretical physicist at the Nordic Institute for Theoretical Physics in Sweden, is the driving force behind both papers. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. About 2,000 photographic plates were used as part of the survey. Each was a glass sheet coated in an emulsion, or a layer of chemicals, that reacted to incoming light, mostly from stars across the sky. They used image processing software to look for transients—short-lived celestial events, such as stars flaring in brightness or fading from view, that are often associated with extreme astrophysics. In principle, besides fluctuating stars transients can be associated with other things, too, such as extreme space weather events impinging on the upper atmosphere, sunlight glinting off reflective objects near Earth, as well as flaws in the telescope or the imaging process. Among the events that the researchers identified, they noted several examples where multiple transients appeared aligned in a straight line across a single photographic plate, a configuration that Villarroel argues is unlikely to occur by any known natural phenomenon. (Other experts, such as Princeton University astrophysicist Robert Lupton, say that finding several such patterns in thousands of star-spangled plates could easily be mere coincidence.) They wrote that transients appeared to be 45 percent more likely on dates within 24 hours of a nuclear test. In their paper, Bruehl and Villarroel linked these transients to global reports of UFO sightings and found a small association, with transients being identified on the same day as a UFO, or UAP (unidentified anomalous phenomenon), sighting. They claimed an 8.5 percent increase in the number of transients identified on days with UAP reports and suggested this might mean the two were linked, referencing “a well-known strand of UAP lore suggesting that nuclear weapons may attract UAP” to observe the events. A candidate transient is highlighted (blue circles) in this figure from an associated paper by Beatriz Villarroel and colleagues. Derived from digital scans of photographic plates from the Palomar Sky Survey and color-inverted to emphasize detail, these images show what the authors say could be flashes of light from artificial objects in outer space, years before humans sent spacecraft into Earth orbit. “Nuclear tests obviously have an impact on the atmosphere,” he says. For example, the first telecommunications satellite, NASA's Telstar 1, was knocked out by an electromagnetic pulse from the American high-altitude nuclear test Starfish Prime in November 1962. “It looks like we are dealing with something that looks pretty artificial in a time when there shouldn't be anything there,” Villarroel says. Each photographic plate took a 50-minute exposure of the sky, leading Villarroel to suggest that these putative objects were stationary in space, possibly in geostationary orbit some 22,000 miles (35,000 kilometers) above Earth. Bright objects in lower orbits—or other less sensational possibilities such as meteors—would have left a streak instead of a starlike dot of light, she says. Often, he says, evidence has been little more than “fuzzy blob photos” or hearsay. “What's interesting about these two papers is: they played by the rules by which science can evaluate evidence,” he adds, referencing Carl Sagan's famous adage that “extraordinary claims require extraordinary evidence.” The papers were not accepted by arXiv.org, an online preprint server where nonpaywalled scientific papers are often uploaded, with the website noting to Villarroel that the Scientific Reports paper in particular “does not contain sufficient or substantive scholarly research.” There are also a host of simpler explanations for Villaroel's transients—which have drawn previous scrutiny from skeptical astronomers—that don't require the extraordinary claim of UFOs. Sean Kirkpatrick, who was head of the Pentagon's UFO-investigating All-Domain Anomaly Resolution Office (AARO) from July 2022 to December 2023, says he thinks the link to nuclear testing and Earth's shadow is key. “The first thing that comes to mind is solar flare radiation or ionized particle radiation from nuclear testing,” split-second bursts of light in the upper atmosphere that would manifest as a point source rather than a streak. Another possible explanation is high-altitude balloons, which were used to do nuclear monitoring. “You're going to have a lot of those around nuclear testing, and so if people see them, a lot of people are going to report them,” he says. Kirkpatrick says Villarroel and her team could use today's geostationary satellites to see if they could reproduce the transients seen by Palomar, building a replica photographic plate and determining if it produces similar transients from known orbiting objects. “What they have not done is prove this technique works on today's geostationary objects,” he says. If it does, that could open the avenue to other non-UAP explanations to explore, such as pieces of sunlight-catching ice or rock lingering in trapped orbits far above Earth. Nigel Hambly, an astronomer at the University of Edinburgh, who has previously analyzed Villarroel's work and who has extensive experience with photographic plates from Palomar and other observatories, says one way to check would be to study the original plates themselves rather than using digital copies. “I've been caught out many times by apparently real things turning up in my data,” he says, especially when working with plates that weren't stored in pristine conditions. “When you actually physically examine the plates under a microscope, you begin to get a feeling for what's real and what's spurious,” he says. “It's wonderful to take a set of phenomena we don't understand and study them,” he says. He adds that Villarroel and her team could test their same methodology on other photographic plate archives to see if the transients show up elsewhere. It would be great to figure them out.” He says that another possibility could be meteors that flew straight down into the telescope's view rather than across it; that would make the objects appear as dots of light instead of streaks. Villarroel says that she welcomes alternative ideas for what these transients might be. “Even if this turns out to be some new physical phenomenon, that's super exciting,” she says. “That would mean we have discovered something new that nobody knew existed.” If that does turn out to be the case, there are plenty of other searches for extraterrestrials—such as NASA's upcoming Habitable Worlds Observatory—that can whet the appetite of those longing for signs of life elsewhere in the cosmos. “It's one of the most important scientific questions that we have,” Frank says. Jonathan O'Callaghan is an award-winning freelance journalist covering astronomy, astrophysics, commercial spaceflight and space exploration. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69144634/ardi-skeleton-human-evolution-climbing-to-walking/'>She Was Half Ape, Half Human—and She May Hold the Secret to What Makes Us Who We Are</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In this 4.4-million-year-old skeleton, scientists may have found the missing step between climbing and walking. We may earn commission if you buy from a link. As humans, it's important to remember that some of our oldest primate ancestors were apelike creatures. Besides our much larger brains and visibly obvious differences, such as facial features, one thing we evolved to be much more adept at than chimps and gorillas (which we diverged from millions of years ago) is walking on two feet. But just when did human predecessors first descend from the trees and walk upright? A 4.4-million-year-old female skeleton named Ardi may answer that question. Predating even the iconic Lucy by a million years, she belongs to the species Ardipithecus ramidus, and is revealing more about our origins and how we became bipedal. Biological anthropologist Thomas Prang of Washington University in St. Louis and his research team discovered that Ardi was able to walk upright but still retained a grasping foot and other apelike features. “Our observations of the human and ape fossil record are inconsistent with recently proposed models of human origins, which envision the last common ancestor of humans and chimpanzees as a generalized arboreal ape,” he said in a study recently published in Communications Biology. “Instead, our results strongly imply that humans evolved from an African ape-like ancestor.” This is part of a larger clade that gorillas also belong to. What Prang found was that Ardi is a mashup of primitive and modern features. She has a big toe designed for grasping branches, but other features of her feet, cranial base and pelvis suggest that her species was adapting to bipedalism. Both put the animal's center of mass in a position closer to support, so apes are at much less risk of suffering lethal backward falls. Gorillas, chimps, bonobos and gibbons have short forefeet for their body masses, while orangutans and larger monkeys have the longest forefeet. The human forefoot is longer than those of chimps and gorillas and shorter than those monkeys, but there is still some overlap with bonobos. African apes and monkeys with longer feet should have a wider talar trochlea, just as humans also have a wide talar trochlea for their forefoot length and body mass. There are aspects of Ardi's talus that appear in extant chimps, gorillas and monkeys. While Ardi displays some of these qualities, her feet had evolved further. “Collectively, these results are inconsistent with human and chimpanzee evolution from a generalized arboreal ancestor that lacked adaptations for terrestrial quadrupedalism, vertical climbing, and suspension.” This might just be how we finally stood up. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. We May Have Been Wrong About the Origin of Life Archaeologists Found the Ruins of a Lost Lake City</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/did-dark-matter-help-supersize-the-universe/'>A ‘Warm' Model of the Early Universe Could Solve Two Cosmic Mysteries at Once</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Physicists Propose a Simpler, Hotter Origin for the Cosmos Theorists have found that a “warm” version of cosmic inflation is consistent with known physics, linking it to the hunt for dark matter Such an exponential expansion may seem minor, but it is equivalent to a grape in the palm of your hand swelling to become tens of thousands of times larger than the observable universe. Known as cosmic inflation, this strange, fleeting period is usually considered to have been an expansion of near nothingness because, at the time, most of the universe's elementary particles had yet to blink into existence. In other words, the standard view of cosmic inflation suggests the universe didn't really begin as a hot, dense fireball but rather as a cold void that only later reheated into a plasmatic soup of particles by some poorly understood process. But a new theoretical study published in the journal Physical Review Letters suggests that inflation may have been warm from the start. “What we have shown with this paper is that actually being warm during inflation is extremely generic and extremely simple,” says its lead author Kim Berghaus, a postdoctoral scholar in theoretical physics at the California Institute of Technology. “It takes us to this footing of ‘This may have actually occurred in nature, and we can go look for it,'” Berghaus adds. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Most physicists think it happened in the first 0.00000000000000000000000000000001 second (10–32 second) of cosmic time, by which point the universe had expanded by a factor of as much as 1050. The most straightforward way to account for this preternatural smoothness is a period of inflation that provides time for the baby universe to reach a uniform temperature. These acted as seeds for cosmic structures; as their gravity glommed together more matter, denser regions would eventually grow to become star-filled galaxies and galaxy clusters. As of yet, there is no direct observational evidence for inflation, so sorting out its details is a task for theorists. Only then did the “newborn universe as fireball” scenario proceed. But precisely how this reheating step would have occurred wasn't well-understood, says Vahid Kamali, a visiting professor at McGill University and an associate professor at Bu-Ali Sina University in Iran, who studies early-universe cosmology and wasn't involved in the new research. Physicists wondered if this step was needed or if there was a way for the whole inflationary process to stay warm. Cold inflation was in some ways too simple, Berera says. “When systems interact, we expect there to be friction and particle production,” he says. “And inflation, in the standard picture, didn't have that.” Critics argued that warm inflation would have effectively burned itself out, prematurely churning out interacting particles that would have sapped its potential energy. Berera and his colleagues published a paper in 2016 that found such a model using interactions and fields similar to those known in the Standard Model. Their calculations show that a feeble interaction between the inflaton field and elementary particles called gluons would be sufficient to warm up inflation. Gluons carry the strong nuclear force, which glues together fundamental particles called quarks to make protons and neutrons. “What they have done is to make this connection that you can have warm inflation with Standard Model interactions,” says Rudnei Ramos, a theoretical physicist at Rio de Janeiro State University in Brazil, who co-authored the 2016 paper but was not involved in the new study. In cold inflation, the initial fluctuations are all inconceivably small and quantum, Berera says, and must later transition to larger, so-called classical interactions during the reheating. But in warm inflation, “it's not a big issue,” he says, “because they're already classical.” It would be a very light, chargeless particle called an axion, Berghaus says. Physicists have been on the quest for axions for decades because some possible variants of these particles might constitute most or all of the universe's dark matter. There are hints that axions might exist, including a faint background glow in space that was detected by the New Horizons spacecraft in 2022. Projects such as the Axion Dark Matter Experiment, a collaboration of the University of Washington, Lawrence Livermore National Laboratory and other institutions, are currently hunting these particles by using intense magnetic fields to convert them to detectable microwave photons. While the prospect of testability makes this new model very exciting, Kamali says, there is still much more to be done to reconcile it with other theories in cosmology. Even so, the allure of thoroughly explaining cosmic inflation within the Standard Model's well-known tenets is likely to prove irresistible for eager theorists and experimentalists alike. “In our work, there is an opportunity for a discovery that can probe the connection between particle physics and the big bang,” Berghaus says. “Because our proposed theory connects intimately with the Standard Model, it is testable.” Stephanie Pappas is a freelance science journalist based in Denver, Colo. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03473-y'>Plants have a secret language that scientists are only now starting to decipher</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 11:58:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Amber Dance is a freelance science journalist in the Los Angeles area. Plant cell walls, such as in these root cells, not only act as physical containers but also contribute to important signalling processes. The squishy contents typically found within would become objects of ongoing study. They were considered passive containers for the exciting biology inside. “For a long time, the cell wall was really thought to be dead,” says Alice Cheung, a plant molecular biologist and biochemist at the University of Massachusetts Amherst. Even then, its complex mix of sugar molecules linked into long, branching polysaccharides kept away all but the most intrepid biochemists. Century-old genetics mystery of Mendel's peas finally solved Century-old genetics mystery of Mendel's peas finally solved But now, with the help of modern molecular methods to analyse the wall's make-up and assembly, researchers are starting to uncover more. They're finding that the cell wall is an active, even chatty participant in cellular growth, reproduction and responses to infection. By eavesdropping on those signals, and tweaking or adding to them, scientists are exploring innovative ways to improve agriculture with cell-wall science: protecting crops from disease and engineering new plants or sturdy hybrids. “The plant cell wall is one of the most sophisticated systems in terms of communication,” says Li-Jia Qu, a plant biologist at Peking University in Beijing. His long-term goal is to use what's been learnt about those messages to interbreed distantly related plants, creating exciting crops that could expand agriculture into new lands. The wall is the plant's interface with its environment, including salt and other stressors or disease agents such as moulds, so it must notice damage and adapt accordingly. The latter are highly complex molecules, branched in diverse ways and decorated with various extras such as methyl groups. It also manufactures defensive molecules such as antimicrobial peptides and reactive oxygen species. Such signals are already being co-opted by farmers. By spraying molecules derived from the cell walls of algae or fungi over their fields, they can prime the plants for pathogens that might arrive later. The method could help growers to avoid harsh fungicides, he says. Molina has co-founded two companies that make use of this technique, making extracts from fungi or plants as crop protectives. Credit: Raymond Wightman/Sainsbury Laboratory, University of Cambridge and Alexis Peaucelle/INRAE The current inoculants are fairly crude mixtures derived from plants or pathogens, says Cyril Zipfel, a plant immunologist at the University of Zurich, Switzerland. He's working to understand the signalling that underlies the immune pathway, which should enable scientists to create more specific, or even synthetic, treatments. Reapplication for slow-growing crops could get costly, but Molina thinks that farmers might be able to focus the use of inoculants to times when risk of infection is high, such as after it rains, to prevent mildew. Another challenge is that whenever plants devote resources to boosting their defences, this takes materials and energy away from growth, so farmers must apply the treatments judiciously. But for plant cells to grow, the cell walls must expand first, says Sebastian Wolf, a plant molecular biologist at the University of Tübingen in Germany. Pectin is a complicated molecule constructed from at least a dozen sugars, connected by more than 20 types of linkage, says Wolf. Depending on those modifications, it can be rigid, supporting a sturdy plant, or softer when the plant needs to grow. That's why pectin is often used to make marmalade: the initially soft pectin molecules cross-link and soak up water, taking on a stiffer, gelatinous consistency. How CRISPR could yield the next blockbuster crop How CRISPR could yield the next blockbuster crop In the growing plant, there's one key modification that makes pectin soft or rigid: methyl groups attached to its component sugars. When the wall needs more materials for growth or reinforcement, the cell inside manufactures a methyl-decorated form that is thought to be fairly soluble, so it can be secreted into the surrounding wall. Calcium ions in the wall bind to two sugars at a time and cross-link the pectin into a stiffer material that can absorb water. They reminded him of mutants with defects in cellular signalling involving cellulose, leading him to wonder whether pectin, too, had a role not just in cell-wall structure, but also in cellular conversations. Continuing this research at the French National Research Institute for Agriculture, Food and Environment in Versailles, Wolf discovered a cell-wall signal that contributes to plant growth controls1,2. The input that Wolf found starts when cell-surface receptors notice an overabundance of methyl-decorated pectin. Cell-wall signalling can even help growing cells to adopt fancy forms, including the puzzle-piece shapes seen in what are known as pavement cells — interlocking surface cells that give strength and structure to plant leaves (see ‘Cell walls fitting together'). When Anderson and his colleagues studied the signals sent by the cell wall as pavement cells developed in A. thaliana, they found evidence for another conversation initiated by methyl-free pectin and a receptor, called FERONIA, that senses this form of pectin3. This pectin sticks to a receptor complex on the cell surface that includes FERONIA. Wolf is continuing to investigate the role of the cell wall in shaping cells, and reported in a preprint earlier this year that plant stem cells must control the methyl status of wall pectin to maintain their stemness and form new plant parts4. He thinks that there is potential to use these signalling pathways to influence the way in which plants are shaped. For example, when plants make less cellulose, they grow into “stubby” forms. But, he cautions, researchers need to learn more about the underlying growth pathways first. FERONIA is found all over plants, and interacts with a range of partners to influence not just leaf-cell shape, but also a variety of other systems, from root growth to environmental-stress responses. Given that FERONIA binds to pectin to maintain the wall's integrity, without it, the wall becomes weak and porous. They can make seeds, he adds, “but very few”. Century-old genetics mystery of Mendel's peas finally solved How CRISPR could yield the next blockbuster crop Apple revival: how science is bringing historic varieties back to life Trees of the Amazon are becoming even mightier Transcriptional interferences ensure one olfactory receptor per ant neuron Mapping Plasmodium transitions and interactions in the Anopheles female John Gurdon obituary: Biologist who made cloning possible The quest to make babies with lab-grown eggs and sperm IOP is China's premier research institution in condensed matter physics and related fields. SOCH is inviting applications or nominations for Senior Lecturer/Lecturer positions. We welcome exceptional candidates who are enthusiastic about c... The Hong Kong Univeristy of Science and Technology (Guangzhou) SLS invites applications for multiple tenure-track/tenured faculty positions at all academic ranks. Century-old genetics mystery of Mendel's peas finally solved How CRISPR could yield the next blockbuster crop Apple revival: how science is bringing historic varieties back to life An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41598-025-24486-7'>A cul-de-sac effect makes Emilia-Romagna more prone to floods in a changing climate</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 10:41:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The disastrous flood of May 2023 in Emilia-Romagna, Italy, displaced thousands of residents and had severe impacts on the economy, with extensive damage to infrastructure—roads, buildings, bridges—and losses in agriculture and livestock. The flood was caused by two consecutive precipitation events, during which no hourly rainfall extremes were recorded, but for which accumulated rainfall over several days produced nonetheless extreme flooding, with a return period of over 500 years. The persistent, long-lasting precipitation was fueled by an uninterrupted vertically integrated water flux from the Adriatic Sea over the Po Valley, driven by a cyclonic circulation over Italy that remained stationary for several days. A “cul-de-sac” effect, due to mountains that blocked moisture fluxes from the Adriatic Sea, amplified rainfall and was a root cause of the disaster. In this study, we analyze the dynamics of this case study in the context of the large-scale atmospheric circulation, focusing on the role of the stationary cyclonic structure over Italy, a feature that also characterized a similar event over the same area in 2024. Furthermore, by examining the frequency of stationary cyclones in the Mediterranean region over recent decades, we are able to suggest that the persistent, dangerous configuration observed during the 2023 and 2024 events should be of concern to other Mediterranean areas that share similar conditions. A preliminary analysis also suggests that this class of events may become more frequent in a changing climate with important implications for the early warning systems. Several studies have argued that climate change is expected to increase the frequency of heavy rainfall events, which could in turn lead to more frequent and intense flooding. In fact, heavy precipitation events are generally expected to increase more than mean precipitation under a warmer climate, which is consistent with the higher capacity of warmer air to hold moisture1,2,3. This is also reflected in a larger portion of the Earth projected to experience increases in heavy precipitation under global warming, compared to mean precipitation2. However, flood risk is influenced by multiple factors, including soil moisture, land use, and runoff sensitivity to sea level, making it difficult to link changes in extreme precipitation to flood occurrence4. The identification of moisture sources has been the focus of numerous studies5,6,7,8,9, contributing to a deeper understanding of the functioning of the hydrological cycle in different regions of the planet and providing insights into the role of synoptic-scale systems, such as atmospheric rivers and low-level jets. In this work we focus on a class of events in which the transport of moisture plays a crucial role in shaping precipitation patterns and, therefore, extreme flooding events. We do so by examining the case of the 2023 Emilia-Romagna flood in Italy. In that instance there was no hourly extreme rainfall, but prolonged and continuous rainfall over several days. It caused one of Italy's most devastating floods in recent history, leading to over 36,000 displacements, 15 fatalities, and estimated damages of EUR 8.8 billion to infrastructure, water bodies, and ecosystems10. In recent years, concern about extreme precipitation events and their role in extreme flooding has grown across Europe, and particularly in the Mediterranean region. As oceans and landmasses warm, the relationship between evaporation, moisture transport, and precipitation is expected to evolve11. Enhanced moisture convergence, combined with the greater moisture-holding capacity of a warmer atmosphere, can support heavy rainfall3. That, in turn can be further enhanced by horizontal moisture fluxes, which tend to scale with the Clausius–Clapeyron relationship1,12. In our case study we identify a set of conditions, involving both moisture advection by stationary cyclones and local orography, that appear to increase the severity of flooding. We believe that a better understanding of the drivers behind these events can lead to improved prediction and preparedness13. The flood of May 2023 was characterized by significant rainfall. Some locations experienced return periods exceeding 500 years14. Yet, the flood was not strictly by a short-lived heavy rainfall event. It therefore cannot be directly attributed solely to the larger moisture-holding capacity of an increasingly warmer atmosphere. Interestingly, and despite the low probabilities involved in such an event, a very similar flooding extreme happened again in the same area just a year later, in September 2024. This second flooding event was slightly less intense, but it exhibited comparable driving circulation patterns. We use historical data to investigate the primary meteorological mechanisms underlying such events and to assess potential links to climate change. Notably, another event—less intense but still exceeding 30 mm/day—occurred on 10 May. 1 shows a time series of precipitation over a 60 × 60 km domain centered on Forlì—one of the most affected towns in Emilia-Romagna. The blue line is based on ARPAE—ARchivio Climatologico per l'Italia centro-Settentrionale, a high-resolution observational dataset (see Methods section). Despite the well-known limitations of ERA5 in representing local precipitation patterns in certain terrain conditions, the present case study was accurately captured by ERA5 in terms of both timing and magnitude when compared to the higher-resolution ARPAE dataset. Vertically integrated water vapor content is shown in magenta, while the vertically integrated horizontal water vapor convergence is in black. The water vapor content (magenta solid line panel a, Fig. 1) exceeds the regional daily median (central magenta dashed line panel a, Fig. 1), yet precipitation is primarily associated with the vertically integrated horizontal water vapor convergence (black line, panel a, Fig. 1, panels b and c), preceded the onset of precipitation by several hours. Time series of quantities averaged in a box of 60 × 60 km centered over Forlì town. Panel (a) shows daily time series during the 2023 year for ERA5 vertically integrated moisture convergence (black), ARPAE precipitation (blue, 5 km resolution), ERA5 precipitation (red, 25 km resolution), ERA5 precipitable water (magenta). Dashed lines represent minimum, mean and maximum climatological values. Panels (b) and (c) show hourly time series of vertically integrated moisture convergence (black) and ERA5 precipitation (red) during the first and the second phases of the flood event of May 2023. Both precipitation events in May were associated with a cyclonic circulation, centered between central Italy and Corsica, which remained stationary for an extended period due to blocking subtropical anticyclones. To illustrate the temporal evolution of key meteorological fields we focus on the event of 1–3 May, but that of 15–17 May was similar14. The cyclone drew moist air from the Adriatic Sea and began impacting eastern Emilia-Romagna on 1 May, as evidenced by the vertically integrated water vapor convergence (Fig. As seen in the timeseries analysis, the spatial patterns of water vapor convergence (left panels) over Emilia-Romagna also precede those of precipitation (right panels). The magnitude of this convergence was driven by sustained southeastward horizontal water vapor transport, visualized by black arrows in Fig. 2 (left panels), advecting moisture into the region. Time evolution (from May 01, 12:00 to May 03, 00:00 of the vertically integrated moisture convergence flux (color patterns in left panels) together with vertically integrated water transport direction (black arrows in left panels) and precipitation (right panels) also from ERA5. The yellow square indicates Forlì location. In fact the Po Valley and its adjacent areas represent a semi-closed geographical basin that is open only to south-easterly winds, specifically those generated by stationary cyclones centered in the Gulf of Genoa or, more generally, on the lee (southern) side of the Alps. This particular configuration is also the reason why the term cul-de-sac was introduced. Indeed, no hourly precipitation extremes were recorded during the 36-hour event, while persistent moist air advection was maintained by the quasi-stationary cyclone over central Italy. Rainfall was moderate to intense—ranging between 3 and 20 mm/h—but not extreme, and Convective Available Potential Energy (CAPE) was relatively low (not shown), indicating only weak instability and limited convective potential. However, the presence of an orographic barrier promoted moist air uplift and intensified rainfall via adiabatic cooling. This was sufficient to support sustained precipitation over the region14. In summary, we can conclude that the extraordinary accumulation of rainfall associated to the Emilia-Romagna flood can be attributed to the combination of two main conditions: cyclone persistence which leads to sustained vertically integrated moisture flux and orographic features that enhance moisture convergence. Based on the synoptic analysis and moisture transport mechanisms described above, it is clear that the events that occurred in May 2023 and May 2024 (not shown) share these key features: the stationary position of a cyclone near central Italy and an associated, prolonged vertically integrated moisture flux toward regions with enhancing orographic features conducive to flow convergence. These conditions, found in Emilia-Romagna, have led to extreme rainfall totals, even in the absence of deep convection or hourly precipitation extremes. The next logical question is to ask whether such conditions are becoming more frequent due to a changing climate and where else in the Mediterranean might we find them. To explore this, we examined long-term changes in Mediterranean cyclone density persistence by analyzing cyclone track data provided by Flaounas et al.15, a data set in which the importance of cyclone tracking methods for representing cyclone genesis and trajectories is evaluated. The results by Flaounas et al.15, confirm that cyclones predominantly occur over maritime areas, particularly near the Gulf of Genoa, the Italian Peninsula, and the Adriatic and Ionian Seas. Genoa cyclones, after having persisted near their genesis area, tend to slowly move eastward and southeastward toward Greece and the Balkans, where they typically undergo gradual filling. We introduce here an objective metric—the Cyclone Density Persistence (CDP) index—to assess which areas of the Mediterranean are most prone to stationary cyclones: This index was computed over 3°×3° grid cells and averaged over the 1979–2020 period (see Methods). Figure 3a reveals that the Italian Peninsula, along with the Tyrrhenian and Ionian Seas, exhibits the highest levels of cyclone activity (highlighted with blue rectangles). Lower activity is observed in the southern Mediterranean. Furthermore, an analysis of the temporal evolution of the CDP index (Fig. 3b) reveals higher values of the index in the 2000–2020 period relative to 1979–1999, especially over the Tyrrhenian, Ionian, and Balearic Seas (blue rectangles). Cyclone Density Persistence index (CDP, as defined in the Main text and in Data and Methods) resulting from extra-tropical storm tracks over the Mediterranean basin. Panel (a) (top) shows the 1979–2020 annual climatology of CDP index values on a 3ox3o grid boxes over the Mediterranean basin. Panel (b) (bottom) shows the difference between CDP mean values for the 2000–2020 period and CDP values for the 1979–1999 period. Positive value: increase, negative values: decrease. White stars indicate statistically significant differences (see methods). This spatial pattern suggests that there are some regions partially enclosed by orographic features that are likely to become more prone to experiencing prolonged water vapor advection associated with stationary or slow-moving cyclones. Such terrain is likely to confine moisture advection associated with the cyclone's circulation, enhancing convergence that can persist over multiple days. Emilia-Romagna emerges as a hotspot for this mechanism, alongside other coastal areas of Albania as shown in Fig. 4, where the composite of vertically integrated moisture convergence flux—associated to persistent (more than two days) cyclones over central Italy—is shown in terms of differences between 2000 and 2020 and 1979–1999 (Fig. 4, panel a) together with the orographic features (Fig. These regions are particularly vulnerable to moisture convergence associated with slow-moving Mediterranean cyclones over central Italy and slow-moving cyclones over this domain are increasing in the last decades (blue patterns in Fig. Similar conditions can be expected for Occitanie and Provence (France), Valencia and Catalonia (Spain) when considering cyclones hovering over the western Mediterranean basin (Fig. The analysis of CDP and of its change over time suggests that the observed warming of the Mediterranean may be increasing the likelihood of prolonged precipitation events and the risk of flooding wherever orography and moisture convergence interact to produce flood-risk conditions. Focus on areas prone to stationary cyclones over central Italy. Black box indicates the domain used to select stationary cyclones (black stars during 1979–1999, red stars during 2000–2020) lasting more than two days, to create time composites of associated vertically integrated moisture convergence flux difference between the two periods (panel a). Orange arrows point towards areas more prone to water convergence due to the vertically integrated water fluxes, induced by stationary cyclones, and confined by the cul-de-sac orographic features. White stars indicate statistically significant differences (see methods). This is a preliminary study, and more work is needed to characterize other critical conditions, such as soil conditions or the state of hydraulic infrastructure, which can contribute to the formation of extreme flooding events. However, even at this early stage, the analysis highlights the potential for cyclone density persistence metrics such as the CDP index to serve as predictive tools in the development of Early Warning Systems for flooding, particularly for areas enclosed by orographic features that are likely to be associated with smaller catchments that can easily be overwhelmed by persistent precipitation. Indicators such as these could improve our ability to detect vulnerability and the effectiveness of forecasts across time scales—from short—and medium-range to seasonal outlooks—to help communities manage the risks of extreme events. Further work is required to assess whether the observed increase in cyclone density persistence over the past four decades persists under various future warming scenarios, and how such changes might affect flood risk across the Mediterranean. Rainfall data for the year 2023 are sourced from two datasets at daily resolution: ARPAE (ARchivio Climatologico per l'Italia centro-Settentrionale): A high-resolution observational dataset with a horizontal resolution of 5 km, available from 1961 to the present16. ERA5 Reanalysis: Provided by the ECMWF with a horizontal resolution of 25 km17, through the Copernicus Climate Change Service (C3S18. To conduct a detailed analysis of the May 2023 flood events, we utilized hourly ERA5 reanalysis data for 2023. The following variables were extracted: Vertically integrated horizontal water convergence. Vertically integrated water vapor content. Vertically integrated zonal and meridional water vapor transport. Daily ERA5 data for the period 1994–2024 were also used to compute climatological averages of vertically integrated water content and horizontal water convergence (represented by dashed lines in Fig. To analyze historical changes in cyclone frequency and stationarity across the Mediterranean basin, we employed cyclone track data from Flaounas et al.15. From these tracks, we computed: Annual Cyclone Density (ACD): The total number of unique cyclone occurrences per year within each 3° × 3° grid cell (each cyclone is counted only once per cell). Cyclone Persistence (CP): The cumulative number of days cyclones remained within a given grid cell across the year. Cyclone Density Persistence (CDP): Defined as the product of ACD and CP, this index quantifies both the frequency and duration of cyclone activity. CDP is expressed in units of [days]. 3 and 4 is verified at the 85% level with a bootstrap method. ETOPO orography: NOAA National Centers for Environmental Information. 2022: ETOPO 2022 15 Arc-Second Global Relief Model. NOAA National Centers for Environmental Information. Robust responses of the hydrological cycle to global warming. Scoccimarro, E. & Gualdi, S. Heavy daily precipitation events in the CMIP6 Worst-Case scenario: projected Twenty-First-Century changes. Trenberth, K. E., Dai, A., Rasmussen, R. M. & Parsons, D. B. Tabari, H. Climate change impact on flood and extreme precipitation increases with water availability. Hoyos, I. et al. Moisture origin and transport processes in Colombia, Northern South America. Gimeno, L., Drumond, A., Nieto, R., Trigo, R. M. & Stohl, A. On the origin of continental precipitation. Gimeno, L. et al. Major mechanisms of atmospheric moisture transport and their role in extreme precipitation events. Recent progress on the sources of continental precipitation as revealed by moisture transport analysis. Natural Hazards and Earth System Sciences. Vázquez, M., Nieto, R., Liberato, M. L. R. & Gimeno, L. Atmospheric moisture sources associated with extreme precipitation during the peak precipitation month, Weather and Climate Extremes, 30, 100289, ISSN 2212–0947. Allan, R. P. et al. Advances in understanding large-scale responses of the water cycle to climate change [published online ahead of print]. Cloux, S., Garaboa-Paz, D., Insua-Costa, D., Miguez-Macho, G. & Pérez-Muñuzuri, V. Extreme precipitation events in the mediterranean area: contrasting two different models for moisture source identification. Flaounas, E. et al. A composite approach to produce reference datasets for extratropical cyclone tracks: application to Mediterranean cyclones. & Naveau, P. A comparison of moderate and extreme ERA-5 daily precipitation with two observational data sets. NOAA National Centers for Environmental Information. ETOPO 2022 15 Arc-Second global relief model. This work has been supported by the EU project BlueAdapt under grant agreement 101057764 and Artemis under grant agreement 101225852. CMCC Foundation - Euro-Mediterranean Center on Climate Change, Bologna, Italy Enrico Scoccimarro, Andrea Borrelli, Lorenzo Sangelantoni, Leone Cavicchia, Stefano Tibaldi & Giulio Boccaletti Institute of Bioeconomy – National Research Council, Rome, Italy Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and wrote the paper; A.B. The authors declare no competing interests. Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Scoccimarro, E., Borrelli, A., Sangelantoni, L. et al. A cul-de-sac effect makes Emilia-Romagna more prone to floods in a changing climate. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02853-z'>Selective RNA sequestration in biomolecular condensates directs cell fate transitions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 10:31:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Controlling stem cell differentiation is a longstanding goal in biomedical research. Here we explore how cell fate is influenced by RNA condensates, specifically P-bodies, which modulate gene expression posttranscriptionally. We profiled the transcriptomes of biomolecular condensates in diverse developmental contexts spanning multiple vertebrate species. Our analyses revealed conserved, cell type-specific sequestration of untranslated RNAs encoding cell fate regulators. P-body RNA contents do not reflect active gene expression in each cell type but are enriched for translationally repressed transcripts characteristic of the preceding developmental stage. Mechanistically, P-body contents are controlled by microRNAs and can be profoundly reshaped by perturbing AGO2 or polyadenylation site usage. Applying these insights to stem cell differentiation, we show that manipulating P-body assembly or microRNA activity can direct naive mouse and human pluripotent stem cells toward totipotency or primed human embryonic cells toward the germ cell lineage. Our findings link cell fate decisions to RNA condensates across vertebrates and provide a means of controlling cell identity. Post-transcriptional mechanisms are critical for establishing and maintaining cell identity1,2. For instance, RNA splicing, decay and alternative polyadenylation are pivotal regulatory mechanisms during development, reprogramming and tissue homeostasis3,4,5,6,7,8,9,10. There is a growing appreciation that RNA regulation is associated with transcript compartmentalization in biomolecular condensates11,12,13,14,15,16. These condensates include P-bodies, evolutionarily conserved cytoplasmic structures containing RNA and RNA-binding proteins (RBPs)11,17,18,19,20,21,22. RNAs can be directed into P-bodies by proteins involved in translation repression, microRNA (miRNA)-mediated pathways and messenger RNA (mRNA) decay processes17,20,23,24,25,26. While P-bodies were initially characterized as hotspots for mRNA decay20,21, recent observations suggest that they also regulate translation by sequestering mRNA away from translational machinery27,28,29,30,31,32,33. In some contexts, P-body transcripts reenter the ribosome pool upon P-body dissolution17,20,23,24,25,26,34, suggesting that P-bodies provide a nuanced, dynamic mechanism to fine-tune gene expression. Emerging evidence suggests a role for P-bodies in regulating cell fate by sequestering mRNA encoding chromatin remodelers and transcription factors28,35,36. P-body dysfunction is implicated in Parkinson's disease37 and cancer38,39,40,41, suggesting that disrupted RNA sequestration may destabilize cell identity and contribute to pathologies. Still, a precise role for RNA sequestration in P-bodies during differentiation and development remains unclear. To address this knowledge gap, we purified intact P-bodies and quantified constituent coding and noncoding RNA across multiple cell types and species. Our findings revealed RNA sequestration in P-bodies as a conserved regulatory mechanism often leading to the storage of specific transcripts that reflect preceding developmental stages. We demonstrate that the ribosome occupancy and corresponding protein levels of mRNAs enriched in P-bodies, including fate-instructive transcripts, increased following acute dissolution of P-bodies. Mechanistically, miRNAs regulate mRNA sequestration in a context-dependent manner. Finally, we show that perturbing RNA sequestration activates a totipotency transcriptional program in naive human pluripotent cells and facilitates the conversion of primed human embryonic stem (ES) cells into primordial germ cell-like cells (PGCLCs). Our findings define a fundamental, conserved role for P-bodies in cell fate specification and suggest strategies for exploiting RNA condensates to direct cell identity. To profile P-body contents, we adapted a fluorescence-activated sorting method29,42 using a green fluorescent protein (GFP)-LSM14A expression construct in HEK293T cells (Fig. LSM14A is an established protein component of P-bodies29, and GFP-LSM14A puncta colocalized with the P-body marker EDC4 (ref. After cell lysis, intact GFP-LSM14A particles were detectable by microscopy and were readily isolated using fluorescence-activated particle sorting (FAPS) (Fig. By contrast, particles were undetectable in cells expressing a cytoplasmic GFP control (Fig. Both short hairpin RNA (shRNA)-mediated suppression of DDX6 and CRISPR knockout (KO) resulted in the complete loss of GFP+ particles (Fig. To confirm the presence of RNA in GFP-LSM14A particles, we stained lysates from GFP-LSM14A-expressing cells with SYTOX Blue, an RNA-binding fluorescent dye. These findings confirm the robust labeling and isolation of P-bodies using GFP-LSM14A. a, Schematic for the purification and transcriptomic profiling of P-bodies from HEK293T cells based on GFP-LSM14A expression. b, Representative IF imaging of GFP-LSM14A puncta (green), colocalizing with EDC4 puncta (red) in HEK293T cells. Nuclei were counterstained with 4,6-diamidino-2-phenylindole (DAPI; blue) (scale bar, 10 μm). c, Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in HEK293T cells. d, Representative imaging of GFP-LSM14A puncta (green) in control and DDX6-KO HEK293T cells. Nuclei were counterstained with DAPI (blue) (scale bar, 10 μm) (left). Unpaired two-sided Student's t-test, median ± s.d., ****P < 0.0001. e, Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in control and DDX6-KO HEK293T cells. f, MA plot of RNA-seq data depicting P-body-enriched genes in red and cytoplasm-enriched genes in blue in HEK293T cells (n = 2 biological independent samples per group, P < 0.05). FC, fold change; FPKM, fragments per kilobase of transcript per million mapped reads. g, GO pathway analysis of P-body-enriched mRNA in HEK293T cells, using expressed genes as background. Enrichment was tested using two-sided Fisher's exact test with multiple-testing correction (Benjamini–Hochbergfalse discovery rate (FDR)). h, GO pathway analysis of cytoplasm-enriched mRNA in HEK293T cells, using expressed genes as background. Enrichment was tested using two-sided Fisher's exact test with multiple-testing correction (Benjamini–Hochberg FDR). i, Representative FISH imaging of POLK RNA molecules (red) combined with imaging of GFP-LSM14A puncta (green). Nuclei were counterstained with DAPI (blue) (scale bar, 10 μm). j, Quantification of POLK mRNA molecules in P-bodies based on FISH (n = 50 cells; right) and P-body sequencing (right); median ± s.d. k, Read coverage distribution over the gene body of the longest annotated isoforms for genes enriched in P-bodies or the cytoplasm (cyto) in HEK293T cells. l, PolyA tail length as determined in ref. 50 compared to P-body enrichment based on Smart-seq and snapTotal-seq (Pearson correlation test). m, Translation efficiency (TE; log2 (Ribo-seq counts/RNA-seq counts)) negatively correlates with mRNA enrichment in P-bodies in HEK293T cells (Pearson correlation test (two sided), P = 3.94 × 10−98). We characterized RNA from purified P-bodies and corresponding cytosolic fractions using Smart-seq44 (hereafter P-body-seq). We detected 3,994 mRNAs enriched in P-bodies of HEK293T cells relative to the cytosol (Fig. Gene ontology (GO) analysis revealed that P-body-enriched mRNAs encoded regulators of RNA processing, transcription, chromatin organization and cell cycle (Fig. 1g), while cytosolic mRNAs were involved in housekeeping functions such as metabolic processes and structural components (Fig. To ensure that our analyses were not biased by polyA-directed library preparations, we performed snapTotal-seq45, an alternative low-input library preparation that uses random primers rather than oligo(dT). Moreover, we compared our dataset to P-body-enriched transcripts previously reported in HEK293T cells29 and found that 3,038 mRNAs (76%) were shared between the datasets (Extended Data Fig. To confirm P-body-seq results using an orthogonal approach, we employed single-molecule fluorescence in situ hybridization (FISH) (smFISH) in conjunction with immunofluorescence (IF)46 and verified the localization of POLK and TET2 within P-bodies (Fig. In line with previous reports26,38, approximately 70% of total POLK and TET2 mRNA was localized to P-bodies determined using either smFISH or P-body-seq, demonstrating substantial transcript sequestration in P-bodies. P-bodies have been proposed as sites for RNA decay20,21,47,48; however, P-body enrichment showed poor correlation with transcript half-life measurements in HEK293T cells49, consistent with previous reports29 (Extended Data Fig. Read distribution analysis also showed no evidence of increased truncated transcripts in P-bodies relative to the cytoplasm (Fig. In addition, we observed poor correlation between mean polyA tail length50 and P-body enrichment (r = −0.028 in Smart-seq data and r = −0.075 in snapTotal-seq data) (Fig. 1l), indicating that P-body-localized RNAs are intact and are not preferentially deadenylated relative to cytoplasmic transcripts. We next asked whether properties intrinsic to RNAs correlated with P-body enrichment. Transcript length had no appreciable relationship with P-body localization, but transcripts with high AU content were enriched among P-body-targeted mRNAs (Extended Data Fig. Notably, AU-rich sequences are associated with inefficient translation28,51,52. Accordingly, analysis of published ribosome profiling data obtained in HEK293T cells50 revealed reduced translation efficiency for P-body-associated transcripts (Fig. These data suggest that transcripts sequestered in P-bodies are translationally repressed compared to cytoplasmic-enriched mRNAs. P-body-associated transcripts can reenter translation following genetic or environmental perturbation19,25,29,51,53,54, suggesting that their sequestration may influence developmental fate decisions. Yet, defining P-body function in complex biological processes has been challenging, in part because sequencing-based approaches to define P-body contents have been limited to nonvertebrates or transformed cell lines25,29,55. We thus applied P-body-seq to profile transcripts in various human cell types across developmental stages. We stably integrated GFP-LSM14A at the AAVS1 locus in human ES cells and cultured them under naive and primed conditions, which model the pre- and post-implantation epiblast, respectively56. We also induced differentiation of the same human pluripotent stem cell line into progenitors of all three germ layers (mesoderm, endoderm and neural progenitors) and ultimately into neurons (Fig. IF confirmed robust differentiation into the expected lineages (80% efficiency or greater; Extended Data Fig. 2a), which was supported by the expression of lineage-specific genes (Extended Data Fig. We detected EDC4+ P-bodies in each cell type, albeit in varying amounts (Extended Data Fig. 2c,d), suggesting differences in RNA sequestration between cell types. a, Schematic highlighting developmental stages profiled by P-body-seq. b, Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in the indicated samples. c, Principal-component (PC) analysis of RNA-seq data for the indicated samples. d, Heatmap showing expression levels of differentially enriched mRNAs between purified P-body fractions of the indicated samples, with manual clustering based on P-body-enriched genes (log2 (FC) > 0, P value < 0.05) that are specific to each cell type or enriched in all cell types (cluster I). Differential enrichment was assessed using DESeq2 (two-sided Wald test, Benjamini–Hochberg Padj values, P < 0.05; n = 2 biologically independent samples per group). e, GO pathway analysis using expressed genes as a background for P-body-enriched mRNA in the indicated samples. Enrichment was tested using two-sided Fisher's exact test with multiple-testing correction (Benjamini–Hochberg FDR). f, Gene tracks showing 8C-related genes from RNA-seq data of P-bodies and cytoplasm fractions of naive ES cells. g, log2 (FC) of current cell fate markers in the current and downstream cell fate, where log2 (FC) > 0 indicates P-body enrichment and log2 (FC) < 0 indicates P-body depletion. Endo, endoderm; meso, mesoderm; NPC, neural progenitor cell. h, Venn diagrams showing the overlap of P-body-enriched genes in the downstream cell type (log2 (FC) > 0, P < 0.05) with genes that are more highly expressed in the current cell type than in the downstream cell type (current/downstream log2 (FC) > 0, P < 0.05). Differential enrichment was assessed using DESeq2 (two-sided Wald test, Benjamini–Hochberg Padj values, P < 0.05). n = 2 biologically independent samples per group. i, Odds ratio of the overlap between the comparisons in h. Center points represent the odds ratio, and error bars represent 95% confidence intervals; n of gene sets is reflected in h. Dashed line indicates an odds ratio of 1 (odds ratios >1 indicate positive association between groups). j, log2 (FC) of current fate lineage-specific genes (current/downstream log2 (FC) > 2.5, P < 0.05) in the current and downstream cell state; box plots show the median (center line), interquartile range (IQR; box, 25th–75th percentiles) and whiskers extending to 1.5× the IQR). n = 2 biologically independent samples per group (n of gene sets is reflected in h). k, GSEA performed on P-body-versus-cytoplasm differential expression in neurons for the neural progenitor-related gene expression signature and the neuron-related gene expression signature. Enrichment significance was calculated with the permutation test (two sided), with multiple-testing correction using the Benjamini–Hochberg method (FDR < 0.05). l, GSEA performed on P-body-versus-cytoplasm differential expression in human naive ES cells for human blastomere and naive ES cell-related gene sets from refs. 58,103. Enrichment significance was calculated with the permutation test (two sided), with multiple-testing correction using the Benjamini–Hochberg method (FDR < 0.05). To profile P-body contents, we sorted GFP-LSM14A+ particles from each cell type. Particle size and fluorescence varied between cell types (Fig. 2b), likely reflecting differential expression of GFP-LSM14A from the AAVS1 locus, but, in all cases, we isolated sufficient RNA for P-body-seq. Supporting our observations in HEK293T cells, analysis of half-life data in human pluripotent stem cells57 suggested that RNA in P-bodies is not preferentially degraded compared to cytoplasmic RNA (Extended Data Fig. We observed diverse RNA biotypes in P-bodies, including protein-coding mRNA and lincRNA (Extended Data Fig. We note, however, that RNA sequestration in P-bodies is not merely a reflection of expression level, as the expression of P-body-enriched transcripts was distributed similarly relative to that of cytoplasmic and unenriched transcripts (Extended Data Fig. We likewise identified repetitive elements in P-bodies, although we did not detect marked differences in their abundance between P-bodies and the cytoplasm (Extended Data Fig. Based on principal-component analysis, we observed unique mRNA profiles for P-bodies from each cell type, suggesting that distinct cell identities have unique transcripts stored in P-bodies (Fig. Principal-component analysis further suggested a stepwise progression from naive ES cells toward mesoderm and/or endoderm lineages on one path and the ectoderm lineage on another (Fig. Notably, we found that P-body contents did not necessarily cluster most closely with the cytoplasm from the same cell type (Fig. For example, P-body samples from neurons cluster most closely with cytoplasmic samples from neural progenitors and P-body contents of mesoderm progenitors cluster with the cytoplasm of primed ES cells (Fig. These data suggest that P-bodies sequester transcripts from stem and progenitor cells to suppress their protein-level expression during differentiation. We grouped transcripts based on their enrichment across cell types (Fig. A fraction of transcripts was enriched in the P-bodies of all cell types, and GO categories linked to DNA damage and cell cycle progression were common across cell types (Fig. By contrast, transcripts enriched in the P-bodies of single cell types encoded fate-instructive factors, such as LEUTX and DUXA in naive ES cells (Fig. Both LEUTX and DUXA are associated with totipotency and the eight-cell (8C) embryo58,59, the developmental stage preceding naive pluripotency. To explore whether P-bodies sequester transcripts important for stem and progenitor cell maintenance during differentiation, we identified P-body-enriched transcripts related to cell fate and examined their localization in differentiated cell types and corresponding developmental precursors (Fig. P-bodies of differentiated cell types contained transcripts associated with the preceding developmental state, while these transcripts were largely cytoplasmic or unenriched in precursor cells (Fig. smFISH analysis of OCT4 (POU5F1) in human primed ES cells and endoderm progenitors confirmed these findings, with OCT4 transcripts transitioning from primarily cytoplasmic localization in primed ES cells to P-bodies during endoderm differentiation (Extended Data Fig. We also observed a significant overlap between transcripts that were more expressed in precursor cells and P-body enriched in differentiated cells (Fig. Finally, we conducted gene set enrichment analysis (GSEA) using differential gene expression data from P-body and cytoplasmic fractions for each cell type (Fig. In neurons, we observed robust enrichment of a transcriptional signature characteristic of neural progenitor cells in the P-body fraction (Fig. Consistent with these findings, P-body dissolution impaired neuronal specification from neural progenitors28,35,36, providing functional evidence that P-bodies sequester genes related to stem and progenitor cell self-renewal to facilitate differentiation. Conversely, a neuronal transcriptional signature was enriched in the cytoplasm of neurons, where these transcripts are expected to be available to translational machinery (Fig. Next, we assessed the functional requirement of P-bodies in endoderm differentiation using a CRISPR interference (CRISPRi)-based approach to deplete DEAD-box helicase 6 (DDX6)28 and dissolve P-bodies19 (Extended Data Fig. DDX6 depletion significantly impaired differentiation toward AFP+ hepatocytes (Extended Data Fig. 3e) and suppressed the induction of other hepatic differentiation markers such as ALB, G6PC and CYP3A7 compared to control cells (Extended Data Fig. These results suggest that P-bodies are essential for differentiation to hepatocytes. To test whether P-body-associated transcripts eventually decline after differentiation is completed, we extended neuronal differentiation to 20 d (Extended Data Fig. Analysis of P-body contents in mature neurons showed substantial overlap (~50%) with transcripts sequestered from neurons at day 7 of differentiation (Extended Data Fig. 3h); however, prolonged culture led to progressive depletion of neural progenitor-associated transcripts from P-bodies (Extended Data Fig. Similar patterns were observed in mature endoderm progenitors, which showed decreased enrichment for primed ES cell expression profiles compared to newly differentiated endoderm progenitors (Extended Data Fig. We next examined P-body-associated transcripts across pluripotent states. In primed ES cells, P-bodies showed enrichment of naive ES cell-specific gene signatures (Extended Data Fig. 3c), consistent with observations that P-body dissolution promotes primed-to-naive state conversion28. Similarly, P-bodies in naive ES cells were enriched for gene expression signatures characteristic of the preceding developmental stage, including zygotic genome activation (ZGA), the 8C embryo and 8C-like cells (8CLC)58,59 (Fig. While naive human ES cells typically express low levels of ZGA- and 8CLC-related transcripts58, our data suggest that these transcripts are sequestered into P-bodies to prevent their translation and inhibit inappropriate reversion to an earlier developmental state. Conversely, naive pluripotency transcripts were enriched in the cytoplasm relative to P-bodies (Fig. 2l), likely reflecting active expression of genes supporting the naive cell fate. These findings suggest that RNA condensates in ES cells selectively sequester totipotency-associated factors. Altogether, these data indicate that P-body contents do not simply reflect the gene expression profiles of a given cell type but are instead enriched for transcripts characteristic of the preceding developmental stage. Next, we investigated whether P-body contents and functions are conserved across vertebrate species. We first considered mouse naive and primed ES cells60. Similar to their human counterparts, naive mouse ES cells resemble the in vivo pre-implantation mouse epiblast, while the primed state represents post-implantation epiblast cells61. Conversion from naive to primed mouse ES cells was efficient (Fig. 4a), with both cell types harboring similar numbers of P-bodies (Fig. a, Schematic of the conversion from naive to primed state mouse ES cells (left). Representative bright-field images of naive and primed mouse ES cells (scale bar, 50 μm) (right). b, Heatmap showing expression levels of naive-specific and primed-specific transcripts (n = 2 biological independent samples per group). c, Representative IF imaging of EDC4 puncta (green) in naive and primed mouse ES cells. Cell membranes were labeled with phalloidin (PHALL; red), and nuclei were counterstained with DAPI (blue) (scale bar, 10 μm) (left). ; not significant (NS), P > 0.05, P = 0.0843. d, Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in naive and primed mouse ES cells. e, Heatmap showing expression levels of differentially enriched mRNAs between purified P-body fractions of naive and primed mouse ES cells, with GO pathway analysis using expressed genes as a background for P-body-enriched transcripts for the indicated clusters. Clusters were generated manually to represent genes that are P-body enriched in both naive and primed mouse ES cells or specifically enriched in one cell type. Gene number in each transcript cluster is indicated in the figure (n = 2 biological independent samples per group, P < 0.05; reg., regulation). f, GSEA analysis of blastomere-related genes63 in the purified P-body fraction versus the cytoplasmic fraction from naive mouse ES cells (normalized enrichment score (NES) = 1.28, P = 0.03). Enrichment significance was calculated using a permutation test (two sided), with multiple-testing correction using the Benjamini–Hochberg method. g, Schematic of the strategy for P-body dissolution in mouse naive ES cells carrying blastomere-specific reporters (MERVL-tdTomato and Zscan4-GFP). h,i, Flow cytometric analysis (h) and quantification (i) of MERVL-tdTomato+ and Zscan4-GFP+ cells upon Lsm14a, Ddx6 and Eif4enif1 KO in mouse naive ES cells. P-body-seq analyses of mouse naive and primed ES cells (Fig. 3d) revealed that a significant fraction of P-body-enriched transcripts exhibited cell type-specific localization (54%) (Fig. 4b), reinforcing our observations in human pluripotent stem cells. GO analysis revealed that transcripts enriched in P-bodies of both naive and primed ES cells were associated with cell cycle progression and RNA processing, similar to human datasets (Fig. Transcripts exclusively enriched in P-bodies of naive and primed ES cells were associated with chromatin remodeling and growth factor response, respectively (Fig. Direct cross-species comparison revealed that ~50% of P-body-enriched transcripts were shared between mice and humans in both naive and primed cell states (Extended Data Fig. To test whether this mechanism is conserved in phylogenetically distant vertebrates, we transduced chicken ES cells with a GFP-LSM14A lentivirus. Chicken ES cells are derived at the blastoderm stage and contribute to chimeric embryos62. After P-body-seq, we found that mRNAs enriched in P-bodies of chicken ES cells exhibited extensive overlap with the P-body transcriptomes of mice and humans (Extended Data Fig. Comparative analysis revealed similar transcript features among all three species, particularly in the preferential sequestration of AU-rich mRNA in P-bodies, regardless of transcript length (Extended Data Fig. Thus, RNA sequestration in P-bodies is conserved across vertebrates. Further analysis using the odds ratio statistic identified a significant positive association of chicken ES cells with both mouse and human ES cells, with the strongest association observed between mouse and human samples (Extended Data Fig. GO analysis of P-body-associated transcripts shared between chicken ES cells and mouse and human naive and primed ES cells revealed enrichment in categories related to RNA processing, transcription, chromatin organization and cell cycle (Extended Data Fig. These data indicate that transcripts sequestered in chicken ES cell P-bodies encode important regulatory factors, consistent with our findings in human and mouse ES cells. In human naive pluripotent stem cells, we found that P-bodies sequester transcripts related to the 8C stage embryo (Fig. In mice, the equivalent developmental stage occurs in the two-cell embryo, although rare cells in naive mouse ES cell cultures, known as 2C cells, transiently express genes characteristic of the two-cell state63. To determine whether transcripts enriched in the P-bodies of mouse naive ES cells comprise a signature characteristic of totipotent cells, we performed GSEA. Consistent with our findings in human naive ES cells, we observed a significant enrichment of 2C-associated transcripts among P-body-enriched mRNAs, regardless of library preparation method (Fig. smFISH analysis confirmed that the naive pluripotency markers Dppa2 and Zfp42 were predominantly cytoplasmic in naive ES cells but primarily P-body enriched in primed ES cells (Extended Data Fig. This observation further indicates that P-body-based regulation of developmental processes is conserved between species. To test the functional consequence of sequestration of 2C transcripts, we employed CRISPR–Cas9 to KO Ddx6 and dissolve P-bodies in mouse ES cells harboring the 2C-specific reporters MERVL-tdTomato and Zscan4c-GFP64,65 (Fig. Ddx6 depletion significantly increased signal from both reporters (Fig. 3h,i) without altering cell proliferation (Extended Data Fig. 5f), suggesting that disrupting P-bodies facilitates the otherwise rare conversion of naive ES cells to the 2C state. RNA sequencing (RNA-seq) of MERVL and Zscan4 double-positive cells confirmed increased expression of genes (Obox3, Zscan4b, Zscan4d, Zscan4e and Zfp352) and transposable elements (MERVL, MT2B1, MLT2E) characteristic of the 2C state following Ddx6 suppression (Fig. 3j,k), suggesting that P-body dissolution enhances cell fate conversion. KO of other essential P-body factors, encoded by Eif4enif1 and Lsm14a32, similarly led to P-body loss and activation of the MERVL and Zscan4 reporters (Fig. 3h,i), supporting our hypothesis that P-bodies sequester fate-instructive 2C transcripts that are transiently suppressed and enter translation upon P-body disruption. To explore this possibility further, we assessed the integrity of P-body-associated transcripts in ES cells, including those encoding 2C regulators. Analysis of both snapTotal-seq and Smart-seq data revealed over 70% overlap in P-body-associated transcripts (Extended Data Fig. Further comparison between P-body enrichment and mRNA half-lives in ES cells based on 4-thiouridine labeling66 demonstrated minimal correlation between mRNA stability and P-body enrichment (r = −0.069) (Extended Data Fig. We found no appreciable evidence of truncated mRNAs or preferential deadenylation67 of P-body-associated RNAs compared to cytosolic fractions (r = −0.092) (Extended Data Fig. These findings suggest that, in ES cells, similar to our observations in HEK293T cells, P-body-associated transcripts are not preferentially degraded compared to cytosolic transcripts. We observed a strong, inverse correlation between P-body enrichment and translation efficiency, suggesting that RNAs localized to P-bodies in naive and primed ES cells are characterized by poor translation efficiency (Fig. These results indicate that P-body-associated RNAs in ES cells are intact and translationally suppressed. a, Translation efficiency (log2 (Ribo-seq counts/RNA-seq counts)) negatively correlates with mRNA enrichment in P-bodies in naive and primed mouse ES cells. Ribosome profiling data are from ref. c, Representative intracellular flow cytometry plots for DDX6 in Ddx6-FKBP12F36V GFP-LSM14A-expressing mouse naive ES cells, either untreated (dimethylsulfoxide; DMSO) or treated with dTAG-13 at the indicated time points. d, Representative IF imaging of EDC4 puncta (red) in Ddx6-FKBP12F36V GFP-LSM14A-expressing mouse naive ES cells, either untreated (DMSO) or treated with dTAG-13 for 6 h. Nuclei were counterstained with DAPI (blue) (scale bar, 10 μm). e, P-body number in Ddx6-FKBP12F36V GFP-LSM14A-expressing mouse naive ES cells, either untreated (DMSO) or treated with dTAG-13 at the indicated time points. DMSO, n = 70 cells; 3 h, dTAG-13, n = 70 cells; 6 h, dTAG-13, n = 70 cells; 9 h, dTAG-13, n = 70 cells; unpaired two-sided Student's t-test, median ± s.d., ****P < 0.0001. f, Cumulative distribution function plot showing ribosome occupancy (log2 (FC)) of P-body-enriched and P-body-depleted mRNAs for untreated (DMSO) versus dTAG-13-treated (6 h) Ddx6-FKBP12F36V GFP-LSM14A-expressing mouse naive ES cells; Wilcox test P = 2.97 × 10−129. g, Box plots showing the change in ribosome occupancy (log2 (ribosome-bound/total RNA) in the degron, log2 (ribosome-bound/total RNA) in the control) of P-body-enriched genes versus all other genes. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Statistical significance was assessed using unpaired two-sided t-test (mean ± s.d., ****P < 0.0001; Padj = 1.6 × 10−38 (Holm's method); all other genes, n = 9,133; P-body, n = 2,843). Enrichment significance was calculated with the permutation test (two-sided), with multiple-testing correction using the Benjamini–Hochberg method (FDR < 0.05). i, Box plots showing the change in ribosome occupancy of P-body-enriched 2C-related genes63 compared to non-P-body-enriched 2C genes. Boxes indicate the IQR (25th–75th percentiles), center show lines the median, and whiskers extend to 1.5× IQR. Unpaired two-sided t-test, mean ± s.d., **P < 0.01; Padj = 0.044 (Holm's method); all other genes, n = 254; P-body, n = 104. j, Box plots showing the change in protein levels (log2 (degron/control)) of all P-body-enriched genes compared to P-body-depleted genes (cytoplasm) after 1 d and 3 d (D1 and D3) of dTAG-13 treatment. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Unpaired two-sided t-test, mean ± s.d., ****P < 0.0001; Padj = 8.5 × 10−7, Padj = 6.6 × 10−6 (Holm's method); cytoplasm, n = 1,090; P-body, n = 1,661. k, Box plots showing the change in protein levels log2 (degron/control) of P-body-enriched 2C-related genes compared to P-body-depleted genes (cytoplasm) after 1 d and 3 d of dTAG-13 treatment. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. ; not significant, P > 0.05; *P < 0.05; Padj = 0.086, Padj = 0.047 (Holm's method); cytoplasm, n = 28; P-body, n = 42. l, Heatmap showing protein levels of 2C-related genes after 1 d and 3 d of dTAG-13 treatment compared to control samples. To directly assess whether P-body-enriched mRNAs engage with translational machinery in stem cells after P-body dissolution, we generated a targeted DDX6 degron system in mouse naive ES cells (Ddx6-FKBP12F36V) (Fig. 4b) and performed polysome profiling after acute DDX6 loss. Addition of degradation tag-13 (dTAG-13) facilitated DDX6 protein degradation and eliminated detectable P-bodies within 6 h (Fig. Consistent with our model, we observed a significant increase in the ribosome occupancy of P-body-associated mRNAs following DDX6 suppression compared to total cytosolic RNAs (P < 2.2−16; Fig. GO analysis confirmed that transcripts exhibiting enhanced ribosome occupancy were associated with stem cell maintenance (Extended Data Fig. Given the observation that disrupting P-bodies in mouse naive ES cells facilitated conversion to the 2C state, we next asked whether 2C-related transcripts demonstrated higher ribosome occupancy after DDX6 degradation. Indeed, 2C-associated mRNAs enriched in P-bodies exhibited increased ribosome occupancy after P-body dissolution (Fig. These data provide further evidence that mRNAs sequestered in P-bodies engage translational machinery upon their release from P-bodies in ES cells, which in turn, alters cell fate. To measure protein-level changes, we performed large-scale, quantitative proteomics in Ddx6 degron ES cells after dTAG-13 treatment for 1 and 3 d. P-body dissolution caused significant upregulation of proteins encoded by P-body-associated mRNAs (Fig. Elevated levels of these proteins facilitated the transition of ES cells to a 2C-like state, as shown by the increased abundance of totipotency factors such as ZSCAN4C, ZSCAN4D, ZSCAN4F, USP17LE and EIF1 (ref. These findings demonstrate that disrupting RNA sequestration in P-bodies promotes the translation of proteins important for cell fate transitions. In line with previous studies, a fraction of P-body-enriched RNAs underwent degradation after acute loss of DDX6 (Extended Data Fig. These downregulated mRNAs were linked to differentiation processes (Extended Data Fig. 6f), perhaps reflecting a separate mechanism for protecting transcripts needed for differentiation until appropriate signals or conditions are achieved during development70. Our data suggest cell type-specific sequestration of mRNA into P-bodies, yet the underlying mechanism remained unclear. We first considered RNA modifications, focusing on N6-methyladenosine (m6A), the most abundant modification in eukaryotic mRNA71. We used a human ES cell model that permits doxycycline-inducible ablation of the m6A methyltransferase METTL3 (TET-OFF METTL3)72 (Extended Data Fig. Eight days of doxycycline treatment suppressed METTL3 expression (Extended Data Fig. 7b) and reduced global m6A levels (Extended Data Fig. 7c), while pluripotency genes remained unaffected (Extended Data Fig. IF for EDC4+ puncta revealed no change in P-body numbers upon METTL3 deletion (Extended Data Fig. We profiled P-body contents and found no significant differences in METTL3-KO cells compared to the control (85% overlap; Extended Data Fig. Moreover, only a small fraction of P-body-enriched RNA was m6A methylated, and enrichment remained largely unchanged after METTL3 KO (Extended Data Fig. These results indicate that m6A methylation does not primarily direct RNA sequestration into P-bodies in human ES cells, consistent with findings on stress granules in mouse ES cells73. We next explored the potential involvement of RBPs in RNA sequestration within P-bodies by mapping the P-body enrichment of transcripts targeted by 53 RBPs using published crosslinking immunoprecipitation followed by sequencing (CLIP–seq) data74 (Fig. Targets of RBPs that localize to RNA granules, including PUM2 and HNRNPU19,75, were enriched in P-bodies; however, these proteins are widely expressed and unlikely to confer cell type specificity. De novo motif analysis of P-body-enriched transcripts predicted recognition by multiple RBPs that were conserved across cell types, with no evident cell type-specific patterns (Extended Data Fig. GO analysis of the identified RBP targets revealed enrichment for pathways consistent with known P-body functions, including 3′ untranslated region (3′-UTR) processing, ribonucleoprotein granule assembly and mRNA stabilization (Extended Data Fig. These data suggest that RBPs are unlikely to drive cell type-specific RNA sequestration. a, mRNA targets of 53 RBPs were analyzed for P-body enrichment in HEK293T cells using CLIP–seq data summarized in ref. 29. Circle size reflects the number of targets per RBP. Color indicates Benjamini–Hochberg Padj from a two-sided Wilcoxon test comparing target log2 (FC) values to those of all genes not bound by that RBP. AGO (Argonaute) protein family members are highlighted in red. b, Uniform manifold approximation and projection (UMAP) analysis of small RNA-seq data for the indicated samples based on differentially expressed genes between purified P-body and cytoplasmic fractions. c, Heatmap showing expression levels of selected miRNAs in P-body and cytoplasmic fractions in mouse naive ES cells (n = 4 biological independent samples per group). d, Quantification of P-body enrichment for miRNA targets (blue) and their corresponding translation efficiency (green) from mouse naive ES cells. Ribosome profiling data are from ref. 68. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Unpaired two-sided Wilcoxon test compares the targets of each miRNA to all genes that are not targets of P-body-enriched miRNA (PB) with Benjamini–Hochberg adjustment (*P < 0.05, **P < 0.01, ***P < 0.001; n and exact P values are noted in the source data). e,f, Box plots showing P-body enrichment of let-7a (let-7a targets, n = 395; non-P-body targets, n = 5,275) (e) and miR-300 and miR-381 (miR-300 and miR-381 targets, n = 353; non-P-body targets, n = 5,275) (f) targets in naive and primed mouse ES cells and MEFs. Unpaired Wilcoxon tests compare target genes to nontargets within each cell type (Benjamini–Hochberg Padj values and Wilcoxon r effect size reported in the figure). Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Paired, two-sided Wilcoxon tests compare enrichment between cell types (Benjamini–Hochberg Padj values and Wilcoxon r effect size are shown in the figure). g, Box plot comparing P-body enrichment of miRNA targets between WT and Ago2-KO mouse naive ES cells. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. h, Schematic of induced Pluripotent Stem (iPS) cell reprogramming, including perturbation of the APA regulator NUDT21. i, PolyA site usage index (PDUI) of P-body- and cytoplasm-enriched transcripts. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Two-sided Wilcoxon test, Benjamini–Hochberg Padj values and Wilcoxon r effect size are reported in the figure (MEF control, n = 1,967; MEF reprogrammed control (reprog. j, Change in polyadenylation site usage after Nudt21 KO from Poly(A) Site sequencing (PAS-seq) data for genes that are lost from Nudt21-KO P-bodies compared to genes in the P-bodies of Nudt21-KO and control KO cells. Boxes indicate the IQR (25th–75th percentiles), center lines show the median, and whiskers extend to 1.5× IQR. Two-sided Wilcoxon test, Benjamini–Hochberg Padj values and Wilcoxon r effect size are reported in the figure. Further inspection of the CLIP–seq data revealed enrichment for targets of the Argonaute protein family (AGO1, AGO2, AGO3 and AGO4) within P-bodies (Fig. AGO proteins are essential components of the RNA-induced silencing complex and cooperate with miRNAs to suppress mRNA translation76,77. Previous studies have shown localization of AGO proteins in P-bodies and suggested a connection between certain miRNAs and mRNA recruitment to ribonucleoprotein granules in nonmammalian cells and transformed cell lines23,78,79,80. Moreover, miRNAs are expressed in a cell type-specific manner and play a prominent role in cell fate transitions81,82. To test the role of miRNAs in directing RNA sequestration of developmental transcripts into P-bodies, we profiled small RNA populations in both P-body and cytoplasmic fractions across mouse and human pluripotent cells and neural progenitors using small noncoding RNA-seq83 (Extended Data Fig. Uniform manifold approximation and projection plot analysis revealed clustering of naive and primed mouse ES cell samples according to their cellular origin (Fig. Subsets of miRNAs were selectively enriched in P-bodies of pluripotent cells as well as progenitor cells (Fig. 8b–d), including miR-300 and miR-let-7, which are known regulators of stem cell potency84,85. Targets of miRNAs enriched in P-bodies were also sequestered within these RNA condensates (Fig. Further analysis86 revealed that P-body-targeted transcripts were enriched for specific miRNA binding sites that were absent in cytoplasmic transcripts (Extended Data Figs. We plotted P-body enrichment for targets of miR-let-7, miR-300 and miR-381 across cell types. In both naive and primed ES cells, we observed increased enrichment of their targets in P-bodies (Fig. By contrast, we found no enrichment for target transcripts in mouse embryonic fibroblast (MEF) P-bodies (Fig. Together, our data suggest that miRNA-targeted genes are sequestered into P-bodies in a cell type-specific manner. Previous studies demonstrated the importance of AGO2 in the sequestration of let-7 targets into P-bodies in transformed cells78. To investigate the involvement of AGO2 in global RNA sequestration, we generated Ago2-KO mouse ES cells87 (Extended Data Fig. 9d), although P-bodies appeared modestly smaller (Extended Data Fig. We then sorted P-bodies from Ago2-KO ES cells and compared their transcriptome to WT counterparts. We found that Ago2 KO significantly reduced miRNA-targeted mRNA levels in P-bodies (Fig. Next, we investigated whether modulation of polyA site usage (that is, alternative polyadenylation (APA)) influenced RNA sequestration into P-bodies. APA generates RNA isoforms that commonly differ in the length of their 3′ UTR88. Because miRNA target sequences are frequently found in the 3′ UTR, we reasoned that forcing proximal polyadenylation (that is, shorter 3′ UTRs) would relieve miRNA-based targeting of transcripts into P-bodies. To test this, we suppressed expression of Nudt21, encoding a component of the CFIm complex that facilitates distal polyadenylation5,89, in MEFs during reprogramming to pluripotency (Fig. We chose this context because a matched dataset of APA changes is available5, permitting direct comparison between polyA site usage and P-body enrichment. Nudt21 knockdown (KD) led to increased reprogramming efficiency (Extended Data Fig. 9e) and expression of pluripotency-related genes5 (Extended Data Fig. Similar to Ago2 KO, P-body numbers were largely unaffected by Nudt21 suppression (Extended Data Fig. P-body-seq from uninduced MEFs as well as reprogramming intermediates with and without Nudt21 silencing revealed that, in MEFs and day 3 control reprogramming cells, 3′ UTRs of P-body-targeted RNAs were longer than cytoplasmic counterparts, suggesting that regulatory sequences within 3′ UTRs mediate P-body enrichment (Fig. After Nudt21 suppression, we observed a global reduction in 3′ UTR length and the 3′ UTRs of transcripts enriched in P-bodies had similar lengths relative to those of cytoplasmic transcripts (Fig. We next used published data to compare changes in polyadenylation site usage to changes in transcript localization between P-bodies and the cytoplasm after Nudt21 KD5. Notably, transcripts enriched in P-bodies of control cells but absent from P-bodies after Nudt21 KD (that is, genes that relocated away from P-bodies) had shorter 3′ UTRs after Nudt21 suppression (Fig. Analysis of previously published polyA site usage data5 revealed that these transcripts contained significantly fewer miRNA target sites after Nudt21 KD (Extended Data Fig. These results suggest that the loss of miRNA binding sites prevents mRNA localization to P-bodies. We next examined whether manipulating specific miRNAs could impact RNA localization and cell identity by depleting miR-300 in mouse ES cells using an antisense inhibitor (Fig. Following 48 h of anti-miR-300 treatment, we observed a significant reduction of miR-300 targets in P-bodies (Fig. a, Schematic of the strategy for miR-300 inhibition in mouse naive ES cells. b, Quantitative PCR with reverse transcription (RT–qPCR) analysis of expression of miR-300 targets in P-bodies of mouse naive ES cells after miR-300 inhibition and for the control; n = 3 biological independent samples per group, unpaired two-sided Student's t-test, mean ± s.d. d, Representative IF imaging of EDC4 puncta (green) and Nanog-MS2 (red) in Nanog-let-7WT (top) and Nanog-let-7Mut cells (bottom). Nuclei were counterstained with DAPI (blue) (scale bar, 10 μm). e, RT–qPCR analysis of Nanog expression in Nanog-let-7WT and Nanog-let-7Mut cells compared to control cells (Nanog KO). n = 3 biologically independent samples per group, mean ± s.d. f, Representative western blot showing NANOG protein levels in Nanog-let-7WT and Nanog-let-7Mut cells compared to control cells (Nanog KO). g–i, Representative pictures (g) and quantification of alkaline phosphatase staining of cell colony number (h) and size (i) of Nanog-let-7WT and Nanog-let-7Mut cells cultured with fetal bovine serum and leukemia inhibitory factor (LIF). Unpaired two-sided Student's t-test, n = 3 biologically independent samples per group, mean ± s.d. ; *P < 0.05, ***P < 0.001; P = 0.0002, P = 0.0135. j, Heatmap showing expression levels of differentially enriched 8C-related mRNAs between purified P-body and cytoplasmic fractions in human naive ES cells (n = 2 biological independent samples per group). k, Schematic of the strategy for P-body dissolution in naive human ES cells carrying a blastomere-specific reporter (TPRX1-GFP). l, Quantification of TPRX1-GFP+ cells upon DDX6 KD. Unpaired two-sided Student's t-test, n = 4 biologically independent samples per group, mean ± s.d., ****P < 0.0001. m, Representative IF imaging of H3Y1 (green)-positive cells in naive human ES cells upon DDX6 KO compared to control cells. Nuclei were counterstained with DAPI (blue) (scale bar, 50 μm) (left). Unpaired two-sided Student's t-test, single-guide (sg)Control (n = 5 fields), sgDDX6 (n = 8 fields), mean ± s.d., ****P < 0.0001 (right). n,o, RNA-seq data from primed, naive and TPRX1-positive human ES cells after DDX6 KD, showing expression levels of pluripotency- and totipotency-related transcripts (n) and transposable elements (o). p, GSEA analysis for the human PGCLC-related gene expression signature96 in P-body-versus-cytoplasmic differential expression in human primed ES cells. Enrichment significance was calculated using the permutation test (two sided), with multiple-testing correction using the Benjamini–Hochberg method. q, Schematic of human iPS cell (hiPSC)-to-PGCLC differentiation. r, Left: flow cytometric analysis of TFAP2C-GFP and BLIMP1-Tomato expression after 4 d of PGCLC differentiation in three-dimensional aggregates. Right: quantification of TFAP2C-GFP+ and BLIMP1-Tomato+ cells by flow cytometry with three experiments. Error bars indicate mean ± s.d., n = 3 biologically independent samples per group; statistical significance was determined using a two-sided unpaired Student's t-test; ***P < 0.001, P = 0.0003. s, RNA-seq analysis of hiPSCs (BTAG cell line96), intermediate mesenchymal-like cells (iMeLCs) and TFAP2C-and-BLIMP1-negative and -positive cells after DDX6 suppression, showing expression levels of pluripotency, mesenchymal and PGC-related genes. To test whether releasing miR-300 targets from P-bodies was sufficient to induce a change in cell fate (Fig. 6a), we examined the expression of 2C-related genes. We observed upregulation of 2C-related transcripts, including Zscan4 and MERVL, upon miR-300 suppression, suggesting that miR-300 restricts reversion of pluripotent cells to an earlier developmental fate (Fig. Notably, these transcripts are not direct targets of miR-300, suggesting that their upregulation results from a change in cell fate, rather than regulation from the miRNA itself. These findings indicate that RNA sequestration in P-bodies can be manipulated to change cell identity. We next tested whether mRNAs could be directed into P-bodies through the addition of miRNA target sequences. We generated reporter constructs based on the pluripotency master regulator encoded by Nanog, which included a 3′ UTR with either six WT or six mutant let-7 sequences (let-7WT and let-7Mut)90 and 24 bacteriophage MS2 stem loops (Extended Data Fig. These stem loops recruit a tagged version of the MS2 coat protein (JF546 HaloTag-NLS-MCP) to permit direct, subcellular visualization of Nanog mRNA. After transfection into Nanog-KO ES cells, Nanog-let-7WT-MS2 transcripts colocalized with the P-body marker EDC4 (Fig. 9j), suggesting that let-7 target sequences were sufficient to direct Nanog mRNA into P-bodies. By contrast, Nanog-let-7Mut-MS2 transcripts failed to accumulate in P-bodies (Fig. Both constructs produced similar steady-state RNA levels (Fig. 6e), excluding the possibility that the observed localization patterns arose from differences in RNA abundance. NANOG protein levels, however, were lower in cells expressing Nanog-let-7WT-MS2, supporting a nondegradative, posttranscriptional regulatory mechanism (Fig. Finally, we investigated whether miRNA-dependent recruitment of RNAs into P-bodies could be leveraged to alter cell identity. Under naive conditions, Nanog-KO cells self-renew; however, when switched to serum–LIF ES culture medium, NANOG helps to sustain self-renewal. We therefore transitioned Nanog-KO cells expressing either Nanog-let-7WT-MS2 or Nanog-let-7Mut-MS2 from naive to standard culture conditions. Cells expressing Nanog-MS2-let-7WT had substantially lower self-renewal capacity than Nanog-let-7Mut-MS2-expressing cells, as evidenced by increased colony number and size in clonal assays (Fig. These data highlight the crucial role of miRNA-mediated RNA sequestration in regulating cell potency. We next sought to manipulate P-bodies to facilitate otherwise inefficient cell fate conversions from human pluripotent stem cells to rare, clinically relevant cell types. We first focused on human totipotent-like cells, which have been identified in vitro at a frequency of 0.1% in naive pluripotent cell culture58. P-bodies in naive human pluripotent stem cells sequester transcripts associated with the totipotent 8C state (Figs. To test whether P-body dissolution could overcome this restriction, we generated human naive ES cells carrying a totipotency-specific reporter, TPRX1-GFP59 (Fig. DDX6 suppression increased the proportion of TPRX1-GFP+ cells by 100-fold (~0.1% to ~10%) over previous reports58 (Fig. As further validation, we used CRISPRi to deplete DDX6 (ref. 28), which activated H3Y1, a DUX4 target gene expressed in the 8C embryo92(Fig. Proliferation rates were unchanged between DDX6-KD and control samples (Extended Data Fig. 9k), suggesting that disrupting P-bodies increased the rate of cell fate conversion rather than cell expansion. Sorted TPRX1+ cells displayed increased expression of genes (for example, DUXA, ZSCAN5B, ZSCAN4) and transposable elements (for example, LTR12, MLT2A1) (Fig. 6n,o) associated with totipotency and the 8C state, while downregulating primed pluripotency markers such as POU5F1 and MER51B93, consistent with enhanced cell fate conversion. These findings demonstrate that P-bodies sequester totipotency-associated transcripts to suppress their expression and suggest that manipulating P-bodies could enable the induction and expansion of rare human cell types in vitro. We next explored whether modulating P-bodies could enhance the conversion from human ES cells to PGCLCs, which are difficult to derive in vitro, share transcriptional features with totipotent stem cells94,95 and are highly relevant for reproductive medicine. Our data indicated that P-bodies in primed human ES cells sequester transcripts associated with a primordial germ cell signature96 (Fig. 6p), including FXR1, ING2, EIF4G3 and MEIOC (Extended Data Fig. We introduced shRNAs targeting DDX6 into primed human pluripotent cells carrying primordial germ cell (PGC)-specific reporters, TFAP2C-GFP and PRDM1 (BLIMP1)-Tomato96 (Fig. We induced germ cell fate and achieved a maximum conversion efficiency of less than 8% in standard conditions; however, DDX6 suppression consistently increased the formation of TFAP2C-and-BLIMP1 double-positive PGCLCs to greater than 24% (Fig. Furthermore, DDX6-depleted TFAP2C+BLIMP1+ PGCLCs exhibited elevated expression of PGC markers, including TFAP2C, PRDM1 and DPPA3 (Fig. 6s), without significant changes in cell proliferation (Extended Data Fig. These findings demonstrate that P-body dissolution facilitates the efficient programming of primed human ES cells toward the germ cell lineage. Our work further establishes a framework for using condensate biology to expand clinically relevant stem cell populations, highlighting P-body modulation as a strategy for cell fate engineering. We identified RNA sequestration through P-bodies as a fundamental regulatory mechanism in development and differentiation (Extended Data Fig. Using FAPS, we compared P-body contents between cell types at distinct developmental stages. Our analyses revealed cell type-specific RNA sequestration patterns that reflected transcriptional profiles characteristic of preceding developmental stages. These data suggest that transcripts important for stem and progenitor cells are directed to P-bodies to inhibit their translation upon differentiation. Functionally, disrupting P-bodies in naive mouse ES cells led to the upregulation of established 2C reporters and drove a transcriptional signature characteristic of the 2C state. We observed increased ribosome occupancy and protein levels for P-body-enriched transcripts, including 2C genes, after acute disruption of P-bodies in mouse naive ES cells, suggesting that sequestration directly inhibited translation. Together, these data provide functional evidence that P-body-based regulation is sufficient to alter cell fate. Our findings suggest the potential of manipulating P-body-mediated regulation to facilitate conversion of pluripotent stem cells into clinically relevant cell types. While human totipotent stem cells were previously identified at a low frequency (~0.1%) in naive pluripotent cell populations58 or generated using chemical reprogramming at low efficiency (~1%)59, disrupting P-body-mediated regulation strongly enhanced the frequency of TPRX1-GFP+ totipotent-like cells to nearly 10% under naive culture conditions. Totipotent cells hold promise for producing high-quality blastoids and for generating differentiated cells with unprecedented efficiency59,97,98. We also manipulated RNA condensates to promote specification of human pluripotent stem cells into PGCLCs, which could assist research on germ cell specification and the development of therapeutics for infertility95,96,99. Moreover, our P-body-seq identified fate-instructive factors within P-bodies in adult progenitor cells of the three germ layers, suggesting that manipulating P-bodies in these cells could similarly redirect cell fate or expand cell populations that are currently challenging to propagate. These insights establish a foundation for leveraging condensate biology to engineer cell fate. P-body-mediated RNA sequestration is likely to function in biological contexts beyond development, including stem cell maintenance and homeostasis. For example, OCT4 must be precisely regulated to support stem cell self-renewal100; too little OCT4 leads to precocious differentiation, while only 1.5-fold overexpression directs cells toward primitive endoderm and mesoderm101. P-body sequestration of key transcription factors in progenitor cells may fine-tune their expression to preserve stem cell self-renewal. Indeed, we observe signal for some transcription factors (for example, those encoded by FGF20 and SOX1; Fig. 2g) in both the cytoplasm and P-bodies of stable stem cell populations. Our findings suggest that differentiating cells may employ RNA sequestration as a reversible mechanism that enables developmental plasticity during fate transitions. Analogous to the ‘gatekeeping' role of NANOG in balancing pluripotency and differentiation102, P-body localization may permit stem and progenitor cells to sample the environment and direct cell fate accordingly. Supporting this model, sequestered transcripts ultimately decline in terminally differentiated cells, suggesting that P-body localization is involved in processes requiring rapid changes in gene expression programs, such as development and stress responses. The observation that P-bodies store distinct transcripts in diverse developmental contexts raises the question of how particular mRNAs are selected for sequestration. We show that a subset of miRNAs, which are often cell type specific, is enriched in P-bodies, and disrupting miRNA–target interaction through multiple orthogonal approaches prevented P-body enrichment of corresponding miRNA targets. That said, RNA sequestration patterns likely reflect a complex interplay of regulatory factors that intersect with miRNA activity and the presence of a given miRNA in multiple cell types does not imply that its targets will exhibit identical localization outcomes between different cell types. Additional factors, including miRNA target abundance, 3′ UTR length, the presence or absence of RBPs and combinatorial targeting of transcripts by multiple miRNAs likely contribute to cell type-specific P-body enrichment. These cells were maintained at 37 °C in a culture medium suitable for naive mouse ES cells, which consisted of a 1:1 ratio of DMEM/F12 medium (Sigma-Aldrich) and Neurobasal Medium (Life Technologies). The culture medium was supplemented with 1× MEM Non-essential Amino Acid Solution (Sigma-Aldrich), 1 mM sodium pyruvate (Sigma-Aldrich), 2 mM l-glutamine (Sigma-Aldrich), 100 U ml−1 penicillin (Sigma-Aldrich), 100 mg ml−1 streptomycin (Sigma-Aldrich), 50 μM β-mercaptoethanol (Sigma-Aldrich), N2 and B27 supplements (referred to as N2B27 medium), two small-molecule inhibitors, PD0325901 (1 μM, Axon Medchem) and CHIR99021 (3 μM, Axon Medchem), and mLIF (10 ng ml−1, R&D Systems). For the induction of mouse primed ES cells, a seeding density of 1.0 × 105 mouse naive ES cells was used. These cells were plated into the wells of a 12-well plate that had been precoated with Matrigel (Corning). The culture medium for this induction consisted of N2B27 medium supplemented with 20 ng ml−1 activin A (PeproTech), 12 ng ml−1 bFGF (PeproTech) and 1% KnockOut Serum Replacement (KSR, Life Technologies). Conventional human primed ES cells (UCLA-4 and WIBR3 ‘TET-OFF METTL3', both female cell lines) as well as CRISPRi GEN1C hiPSCs28, were cultivated on Matrigel-coated dishes in mTeSR1 medium (Stemcell Technologies) at 37 °C. These cells were propagated by using a 0.02% EDTA solution (Sigma-Aldrich) for passaging purposes. Cell passaging occurred every 4–5 d to ensure proper maintenance. In the context of the doxycycline experiment, the TET-OFF METTL3 cell line was cultured in mTeSR1 medium supplemented with 2 μg ml−1 doxycycline (Sigma-Aldrich) for 1 week. To achieve reversion to a naive state, female UCLA-4 human ES cells that had been passaged 6 d beforehand underwent a procedure involving washing with 1× PBS (Life Technologies) and then treatment with TrypLE Express Enzyme (1×, Life Technologies) for 3 min. This enzymatic treatment enabled cell dissociation into a single-cell suspension, which was subsequently plated at a density of 30,000 cells per 9.5 cm2 on irradiated CF-1 MEFs (a mixture of pooled male and female cells). This process occurred in human ES cell medium supplemented with 10 μM Y-27632 (Axon Medchem). After a 2-d incubation period, the medium was changed to 5i/LAF and then replaced daily. The 5i/LAF medium composition consisted of a 50:50 mixture of DMEM/F12 (Sigma-Aldrich) and Neurobasal Medium (Life Technologies), supplemented with 1× N2 supplement (Life Technologies), 1× B27 supplement (Life Technologies), 10 ng ml−1 bFGF (PeproTech), 1% non-essential amino acids (Sigma-Aldrich), l-glutamine (2 mM, Sigma-Aldrich), penicillin–streptomycin (Sigma-Aldrich), 0.1 mM β-mercaptoethanol (Sigma-Aldrich), 50 μg ml−1 BSA (Sigma-Aldrich), 0.5 μM IM-12 (Axon Medchem), 0.5 μM SB590885 (Axon Medchem), 1 μM WH-4-023 (Axon Medchem), 10 μM Y-27632 (Axon Medchem), 20 ng ml−1 activin A (PeproTech), 20 ng ml−1 rhLIF (PeproTech), 0.5% KSR (Life Technologies) and 1 μM PD0325901 (Axon Medchem). The cells were then replated after passage through a 40-μm cell strainer in 5i/LAF medium on irradiated CF-1 MEFs. The established naive human ES cell lines were cultured on irradiated CF-1 MEFs at a density of 2.5 × 106 cells per 9.5 cm2 in 5i/LAF medium and were passaged every 6–7 d. The cells were provided with fresh medium daily. These naive human ES cells were maintained under low-oxygen conditions (5% O2) at 37 °C. Human mesoderm progenitors were generated from human primed ES cells using the STEMdiff Mesoderm Differentiation Kit (Stemcell Technologies) according to the manufacturer's instructions. For the derivation of human endoderm progenitors from human primed ES cells, the STEMdiff Definitive Endoderm Differentiation Kit (Stemcell Technologies) was used following the manufacturer's instructions. To induce neural progenitors, human primed ES cells were cultivated in neural induction medium. This medium composition included a 1:1 ratio of Advanced DMEM/F12 medium and Neurobasal Medium, supplemented with 1× N2 supplement (Life Technologies), 1× B27 supplement (Life Technologies), l-glutamine (2 mM, Sigma-Aldrich), 4 μM CHIR99021 (Axon Medchem), 3 μM SB432542 (Stemcell Technologies), 5 μg ml−1 BSA, 10 ng ml−1 human LIF (PeproTech) and 0.1 μM Compound E (γ-Secretase Inhibitor XXI, PeproTech). After 9 d, the cells were passaged onto Matrigel-coated dishes using Accutase (Sigma-Aldrich). They were then maintained in neural induction medium with the omission of Compound E. For the purpose of neuronal differentiation, the neural progenitors were cultured on Matrigel-coated dishes, using DMEM/F12 medium (Sigma-Aldrich), 1× N2 supplement (Life Technologies), 1× B27 supplement (Life Technologies), 300 ng ml−1 cAMP (Sigma-Aldrich) and 0.2 μM vitamin C (Sigma-Aldrich). hPSCs were sequentially differentiated toward anterior-most primitive streak, definitive endoderm and then mature endoderm progenitors, as previously described105. Briefly, the following medium compositions were used on each day of differentiation: day 1, CDM2 base medium supplemented with 100 ng ml−1 activin A (PeproTech), 3 μM CHIR99021 (Axon Medchem), 20 ng ml−1 FGF2 (PeproTech) and 50 nM PI-103 (Tocris); day 2, CDM2 base medium supplemented with 100 ng ml−1 activin A, 250 nM LDN-193189 and 50 nM PI-103; day 3, CDM3 base medium supplemented with 20 ng ml−1 FGF2, 30 ng ml−1 BMP4 (R&D Systems), 75 nM TTNPB (Selleck) and 1 μM A-83-01 (Reprocell); days 4–6, CDM3 base medium supplemented with 10 ng ml−1 activin A, 30 ng ml−1 BMP4 and 1 μM forskolin (Sigma-Aldrich). The composition of CDM2 basal medium is 50% IMDM medium, 50% F12, l-glutamine (2 mM, Sigma-Aldrich), penicillin–streptomycin (Sigma-Aldrich), 1 mg ml−1 polyvinyl alcohol (Sigma), 1% (vol/vol) chemically defined lipid concentrate (Thermo Fisher), 450 μM 1-thioglycerol (Sigma-Aldrich), 0.7 μg ml−1 recombinant human insulin (Sigma-Aldrich) and 15 μg ml−1 human transferrin (Sigma). The composition of CDM3 basal medium is 45% IMDM, 45% F12, l-glutamine (2 mM, Sigma-Aldrich), penicillin–streptomycin (Sigma-Aldrich), 10% KSR (Thermo Fisher), 1 mg ml−1 polyvinyl alcohol (Sigma-Aldrich) and 1% (vol/vol) chemically defined lipid concentrate (Thermo Fisher). SFD serum-free medium consists of 75% IMDM (Life Technologies), 25% Ham's F12 (CellGro), 0.5× N-2 Supplement (Gibco), 0.5× B27 without retinoic acid (Gibco), 0.1% BSA (Sigma-Aldrich), 50 μg ml−1 ascorbic acid phosphate magnesium (Wako) and 4.5 × 10−4 M monothioglycerol. BTAG human iPSCs were maintained on iMatrix-511 (Reprocell) in StemFit medium (Reprocell) supplemented with 100 ng ml−1 bFGF (PeproTech). PGCLC induction was achieved as previously described96. iMeLCs were induced by plating 2.0 × 105 hiPSCs per well of a 12-well plate coated with human plasma fibronectin (Millipore, FC010) in GK15 medium (GMEM (Life Technologies) supplemented with 15% KSR, 0.1 mM NEAA, 2 mM l-glutamine, 1 mM sodium pyruvate and 0.1 mM 2-mercaptoethanol) containing 50 ng ml−1 activin A (PeproTech), 3 μM CHIR99021 and 10 μM ROCK inhibitor (Y-27632, Axon Medchem). hPGCLCs were induced by plating 4 × 103 iMeLCs per well in a cell-repellent V-bottom 96-well plate (Greiner Bio-One, 651970) in GK15 medium supplemented with 1,000 U ml−1 hLIF (Millipore, LIF1005), 200 ng ml−1 BMP4 (HumanZyme), 100 ng ml−1 SCF (R&D Systems, 455-MC), 50 ng ml−1 EGF (R&D Systems, 236-EG) and 10 μM ROCK inhibitor. For HEK293T cells, mouse ES cells and MEFs, pEGFP-LSM14A from the Weil laboratory29 was assembled into pLV-EF1a-Ires-Blast (Addgene, 85133). For human ES cells, the EGFP-LSM14A fragment from pEGFP-LSM14A was amplified using primers F (CCTACCCTCGTAAAGCCGGGCTACCGGTCGCCACCATG) and R (AACTAGAAGGCACAGTTAATGGATCCTTAGGGTCCAAAAGC) and assembled into pAAVS1-tet-iCas9-BFP2 (Addgene, 125519), replacing the iCas9-BFP2 sequence with pEGFP-LSM14A. To introduce a sequence encoding FKBP12F36V-HA-2A-mCherry in place of the endogenous Ddx6 stop codon, a first donor plasmid was created by integrating two 200-bp homology arms specific to the Ddx6 gene into the pNQL004-SOX2-FKBPV-HA2-P2A-mCherry targeting construct (Addgene, 175552). In conjunction with this donor plasmid, a Ddx6-targeting sgRNA (AGGTACATACGTGCTTGTTA) was cloned into the pSpCas9 (BB)-2A-Hygro plasmid (Addgene, 127763). Both donor plasmids were transfected into GFP-LSM14A mouse ES cells using the Lipofectamine 3000 transfection method. Cells exhibiting stable mCherry expression were subsequently isolated using fluorescence-activated cell sorting to establish single-cell clones. Homozygous insertion was confirmed by genotyping PCR. For this purpose, a forward primer (GCTGGGGACAGAGATCAAACC) and a reverse primer (GTAGGGCTATGCGGCCCTAC) within the DDX6 homology arms were used, along with a reverse primer specific to the mCherry sequence (ATCTGGGCAACCCCTTCTTCC). Loss of endogenous DDX6 upon dTAG-13 treatment was confirmed through western blot analysis. The Ago2-KO cell line was established with a paired CRISPR–Cas9 approach implemented on WT mouse ES cells, as previously described in ref. GFP-positive cells were isolated by single-cell sorting in 96-well plates. To verify the deletion, PCR using the Ago2KOΔEx1_Fw (GAAGGCGAAAAAGCCTCCCC) and Ago2KOΔEx1_Rev (GAGCTAGCTTCCCGTCGC) primers was conducted. To target the mouse Ddx6 gene, shERWOOD UltramiR Lentiviral shRNAs (82, ACAGCTGAACCAGTTGAAGAA; 83, CTGGGCTACTCTTGCTTTTAA) were procured from Transomic Technologies, as previously described28. To accomplish Nudt21 gene silencing, oligonucleotide pairs encoding shRNA sequences specific to Nudt21 (GGACAACTTTCTTCAAATT) were annealed and cloned into the pSICOR-mCherry-puro vector (Addgene, 31845). The efficacy of KD was validated by RT–qPCR. In the context of miRNA KD experiments, miR-300 and control miRNA inhibitors (miRNA Power Inhibitors, Qiagen) were directly introduced into the cell culture medium. These inhibitors were added at a final concentration of 50 nM, according to the manufacturer's recommendations. For knockin TPRX1-EGFP generation, sgRNA (GCTCCCGAGCTAGTTTGGCG) targeting the insertion site of the donor construct containing homologous arms flanked by an EGFP–puromycin cassette, kindly gifted by the Esteban laboratory (Guangzhou Institutes of Biomedicine and Health,Chinese Academy of Sciences)59, was cloned into the pDonor plasmid. Donor plasmids were electroporated into UCLA-4 human ES cells using the Neon Transfection System. Homozygous insertion was confirmed by genotyping PCR. Plasmid pNZ176, containing Halo-NLS-MCP, was a kind gift from T. Stasevich (Department of Biochemistry and Molecular Biology, Colorado State University). The Halo-NLS-MCP fragment was amplified by PCR from this vector in parallel with PCR amplification of the Ires BFP fragment from the TRE KRAB Cas9 Ires BFP plasmid (Addgene, 85449) using Phusion polymerase (New England Biolabs) according to the manufacturer's recommendations. The resulting fragments were used for NEBuilder HiFi DNA Assembly (New England Biolabs). BFP-positive cells were isolated using fluorescence-activated cell sorting. The MS2 reporter construct for Nanog was generated using a multistep cloning approach. First, Nanog was amplified by PCR from the pMXs-Nanog (Addgene, 13354) plasmid using Phusion polymerase (New England Biolabs) according to the manufacturer's recommendations. Primers to amplify Nanog were CGCTGTGATCGTCACTTGGCGCCGCCATGAGTGTGGGTC and CGCTGTGATCGTCACTTGGCCCACCATGAGTGTGGGTC. The PCR product was treated with DpnI (New England Biolabs) and purified using a PCR cleanup kit (Qiagen). The resulting Nanog fragment was used for Gibson assembly (New England Biolabs) with pUB_smFLAG_ActB_MS2 (Addgene, 81083) digested with NotI and NheI (New England Biolabs). The resulting pNANOG-MS2 plasmid was used for Gibson assembly (New England Biolabs) with the let-7WT and let-7Mut sequences amplified from the RL-6xB and RL-6xBMUT vectors (a kind gift fromM.R. Fabian (McGill Centre for Translational Research in Cancer, McGill University))90 using the following primers: Wt_F, TGAAATATGATCTAGAGGACAGCCTATTGAACTACCTCACTCG; Mut_F, TGAAATATGATCTAGAGCACAGCCTATTGAACTACCCCTCACT; R, TTGTAGGTTAGCGGCCGCACCGAATGCG. The resulting PiggyBac vectors carrying Nanog-let-7WT-MS2 or Nanog-MS2-let-7Mut-MS2 cassettes were used to transfect Nanog-KO ES cells expressing HALO-NLS-MCP IRES BFP. P-body purification was performed as previously described. Briefly, GFP-LSM14A cells were subjected to lysis for 20 min on ice using lysis buffer (comprising 50 mM Tris at pH 7.4, 1 mM EDTA, 150 mM NaCl and 0.2% Triton X-100), supplemented with 65 U ml−1 of the ribonuclease inhibitor RNaseOut (Promega) and an EDTA-free protease inhibitor cocktail (Roche Diagnostics). After this step, the lysates underwent centrifugation at 200g for 5 min at 4 °C to remove nuclei from the mixture. Residual DNA was removed by incubation in the presence of 10 mM MgSO4, 1 mM CaCl2 and 4 U ml−1 RQ1 DNase (Promega) for 30 min at room temperature. After centrifugation at 10,000g for 7 min at 4 °C, pellets were resuspended in 40 μl lysis buffer containing 80 U RNaseOut (Promega) to generate the cytoplasmic fraction. P-bodies were then isolated from this cytoplasmic fraction using FACSAria-based sorting. After sorting, the P-body samples were pelleted by centrifugation at 10,000g for 7 min at 4 °C. These pellets, alongside aliquots of the corresponding presorting cytoplasmic fraction, were stored at −80 °C. For P-body-seq, RNA was isolated from P-body and cytoplasmic fractions using the miRNeasy Micro Kit (Qiagen). Complementary DNA (cDNA) libraries were constructed using the SMART-Seq v4 Ultra kit and the Nextera XT kit for Illumina (Takara Bio). When indicated in the text, libraries were prepared using the snapTotal-seq protocol as previously described45. Libraries were sequenced with paired-end 150-bp reads. Smart-seq data were analyzed by mapping reads using hisat2.1.0 (ref 107) to the mouse mm10 genome assembly, the human hg38 assembly or the chicken gal6 assembly, all retrieved from UCSC. Gene counts were annotated using featureCounts version 1.6.2 using the corresponding UCSC refGene annotation. snapTotal-seq data were processed using the nf-core RNA-seq pipeline (version 3.9) within a Singularity container. Reads were aligned to the mouse mm10 genome assembly and the human hg38 genome assembly, both of which were retrieved from UCSC with corresponding UCSC refGene annotations. Raw counts were obtained from salmon (version 1.10.2), and the resulting dds objects from nf-core were exported to R for downstream analysis. To determine P-body-enriched and -depleted genes, lowly expressed genes (<10 reads over all samples) were filtered out. Next, DESeq2 version 1.40.2 was used to calculate differential expression between the P-body and cytoplasmic fractions, setting an FDR of P < 0.05 as a threshold for significance. DESeq2 analysis was performed comparing progenitor cell types with the differentiated cell type in pairwise comparisons (that is, naive versus primed; primed versus endoderm progenitors, mesoderm progenitors and mature endoderm progenitors; and neural progenitors versus neurons). Cell type-specific gene sets were defined as genes with log2 (FC) > 0 and Padj < 0.05. For GSEA, these gene sets were further filtered with a log2 (FC) threshold of >2.5 and Padj < 0.05. GSEA were performed using the fgsea version 1.26.0 R package with nperm = 50,000. Gene ranks were determined with the stat parameter in the results tables generated using DESeq2. GO analysis was performed using enrichGO from the clusterProfiler package, with the universe parameter set to only genes detected in sequencing data. Orthologs were determined using Ensembl BioMart genes version 108. To calculate GC content and transcript length distributions for P-body- and cytoplasm-enriched genes, we used the biomaRt package to access cDNA sequences from Ensembl (hsapiens_gene_ensembl). Sequences were retrieved using the ‘cdna' attribute, with GC content calculated as the percentage of G and C nucleotides and transcript length as sequence length in nucleotides. For uniformity, only the longest isoform per gene was retained. Transposable element abundance was quantified by mapping reads to RepeatMasker109 (mm39 and hg38) annotations using a custom pipeline to identify repeat family expression patterns across samples. miRNA targets were obtained from the miRDB database110, in which targets with a threshold score greater than the 75th percentile were deemed a target. To infer targets of RBPs, we used previously compiled CLIP data29 from the CLIPdb 1.0 database, focusing on targets of RBPs that are bound at the 3′ UTR of the target genes. For miRNA and RBP plots, the log2 (FC) P-body-to-cytoplasm ratio of the targets was plotted. To calculate GC content and transcript length distributions for P-body- and cytoplasm-enriched genes, we used the biomaRt package to access cDNA sequences from Ensembl (hsapiens_gene_ensembl). Sequences were retrieved using the cdna attribute, with GC content calculated as the percentage of G and C nucleotides and transcript length as sequence length in nucleotides. For uniformity, only the longest isoform per gene was retained. To determine the relationship between P-body enrichment and expression levels, RPKM of genes in naive and primed human ES cells were obtained from GSE111020 (ref. To assess the correlation between P-body enrichment and mRNA half-life, publicly available SLAM-seq data from HEK293T cells49,111, mouse ES cells66 and human iPSCs were used57. Additionally, polyA tail length, as an indicator of mRNA stability, was evaluated by comparing PAIso–seq data from mouse ES cells67. For each dataset, a Pearson correlation was calculated. Gene coverage plots were generated using PyCoverage (geneBody_coverage.py from the RSeQC package). BED12 files were created for P-body-enriched genes, cytoplasm-enriched genes and all transcripts for each cell type (gene sets defined previously in RNA-seq methods). Gene annotation files in BED format were downloaded from the UCSC Genome Browser for mm10 and hg38 genome assemblies, selecting Genes and Gene Predictions group, NCBI RefSeq track and the UCSC RefSeq (refGene) table. These files were converted to BED12 format with custom R code, with which block sizes were calculated as the difference between exon start and end positions and block starts were calculated as the difference between exon start and transcript start. Only the longest isoform per gene was retained. Filtered genome BED12 files for the specified gene lists were then used in the geneBody_coverage.py command, with BAM files for each fraction (two replicates each) analyzed per cell type and species. Resulting outputs were plotted in R, displaying read counts across gene bodies. RNA-seq data from polysome-containing samples and total RNA samples were analyzed using the nf-core/rnaseq pipeline (version 3.9) with default settings. Reads were aligned to the mouse mm10 genome assembly, retrieved from UCSC using the corresponding UCSC refGene annotation. Raw counts were obtained from salmon version 1.10.2. To determine differentially expressed genes in DDX6-KD samples, lowly expressed genes (<10 reads over all samples) were filtered out. Next, DESeq2 was used to call differential expression between control and dTAG-13-treated total RNA samples. An FDR of P < 0.05 was used as the threshold for significance of differentially expressed genes. For polysome fractionation analysis, genes were filtered to protein-coding genes using the mm10 refGene annotation. Ribosome occupancy was calculated as log2 (ribosome-bound/total RNA) using DESeq2. GSEA was performed as described above using mouse gene sets from 2C63, naive104 and primed69 cells. Ribo-seq data were obtained from ref. 68. To download Ribo-seq raw data, GSE133794 was downloaded from the Gene Expression Omnibus and processed using the nf-core/rnaseq pipeline (version 3.9). Reads were aligned to the mouse mm10 genome assembly, retrieved from UCSC using the corresponding UCSC refGene annotation. Raw reads were obtained from salmon version 1.10.2. Reads were normalized by RPKM, and lowly expressed genes (<10 reads over all samples) were filtered out. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The accession number for RNA-seq, small RNA-seq, PANDORA-seq and polysome profiling data reported in this paper is GSE245943. Source data are provided with this paper. A., Van Der Werf, I., Jamieson, C. & Signer, R. A. J. Post-transcriptional regulation of homeostatic, stressed, and malignant stem cells. Beyond the genome: RNA control of stem cells. Cheng, Y. et al. m6A RNA methylation maintains hematopoietic stem cell identity and symmetric commitment. Lee, H. et al. Stage-specific requirement for Mettl3-dependent m6A mRNA methylation during haematopoietic stem cell differentiation. Brumbaugh, J. et al. Nudt21 controls cell fate by connecting alternative polyadenylation to chromatin signaling. Vivori, C. et al. Dynamics of alternative splicing during somatic cell reprogramming reveals functions for RNA-binding proteins CPSF3, hnRNP UL1, and TIA1. Han, H. et al. MBNL proteins repress ES-cell-specific alternative splicing and reprogramming. Batista, P. J. et al. m6A RNA modification controls cell fate transition in mammalian embryonic stem cells. New insights into the epitranscriptomic control of pluripotent stem cell fate. m6A mRNA methylation facilitates resolution of naive pluripotency toward differentiation. Roden, C. & Gladfelter, A. S. RNA contributions to the form and function of biomolecular condensates. Wadsworth, G. M. et al. RNA-driven phase transitions in biomolecular condensates. An RNA-centric view of transcription and genome organization. Pei, G., Lyons, H., Li, P. & Sabari, B. R. Transcription regulation by biomolecular condensates. & Di Stefano, B. Biomolecular condensates in immune cell fate. & Parker, R. P-bodies and stress granules: possible roles in the control of translation and mRNA degradation. & Izaurralde, E. P bodies: at the crossroads of post-transcriptional pathways. & Slavoff, S. A. P-bodies: composition, properties, and functions. Riggs, C. L., Kedersha, N., Ivanov, P. & Anderson, P. Mammalian stress granules and P bodies at a glance. Standart, N. & Weil, D. P-bodies: cytosolic droplets for coordinated mRNA storage. & Blelloch, R. Decoupling the impact of microRNAs on translational repression versus RNA degradation in embryonic stem cells. Assembly, function, and connections with disease. Wang, C. et al. Context-dependent deposition and regulation of mRNAs in P-bodies. Brengues, M., Teixeira, D. & Parker, R. Movement of eukaryotic mRNAs between polysomes and cytoplasmic processing bodies. The RNA helicase DDX6 controls cellular plasticity by modulating P-body homeostasis. Hubstenberger, A. et al. P-body purification reveals the condensation of repressed mRNA regulons. Teixeira, D., Sheth, U., Valencia-Sanchez, M. A., Brengues, M. & Parker, R. Processing bodies require RNA for assembly and contain nontranslating mRNAs. Aizer, A. et al. Quantifying mRNA targeting to P-bodies in living human cells reveals their dual role in mRNA decay and storage. Ayache, J. et al. P-body assembly requires DDX6 repression complexes rather than decay or Ataxin2/2L complexes. The dynamics of mRNA turnover revealed by single-molecule imaging in single cells. Structural and biochemical insights to the role of the CCR4–NOT complex and DDX6 ATPase in microRNA repression. Coordinated repression of pro-differentiation genes via P-bodies and transcription maintains Drosophila intestinal stem cell identity. The Parkinson's disease protein α-synuclein is a modulator of processing bodies and mRNA stability. Kodali, S. et al. RNA sequestration in P-bodies sustains myeloid leukaemia. & Franchini, D. M. Cancer cell adaptability: turning ribonucleoprotein granules into targets. Processing body (P-body) and its mediators in cancer. Anderson, P., Kedersha, N. & Ivanov, P. Stress granules, P-bodies and cancer. Safieddine, A. et al. Cell-cycle-dependent mRNA localization in P-bodies. Hernandez, G. et al. Decapping protein EDC4 regulates DNA repair and phenocopies BRCA1. Picelli, S. et al. Full-length RNA-seq from single cells using Smart-seq2. & Zong, C. Single-cell total-RNA profiling unveils regulatory hubs of transcription factors. Lyubimova, A. et al. Single-molecule mRNA detection and counting in mammalian tissue. Brothers, W. R., Ali, F., Kajjo, S. & Fabian, M. R. The EDC4–XRN1 interaction controls P-body dynamics to link mRNA decapping with decay. Blake, L. A., Watkins, L., Liu, Y., Inoue, T. & Wu, B. Subtelny, A. O., Eichhorn, S. W., Chen, G. R., Sive, H. & Bartel, D. P. Poly(A)-tail profiling reveals an embryonic switch in translational control. Courel, M. et al. GC content shapes mRNA storage and decay in human cells. & Filipowicz, W. Relief of microRNA-mediated translational repression in human cells subjected to stress. Balagopal, V. & Parker, R. Polysomes, P bodies and stress granules: states and fates of eukaryotic mRNAs. Matheny, T., Rao, B. S. & Parker, R. Transcriptome-wide comparison of stress granules and P-bodies reveals that translation plays a major role in RNA partitioning. Reduced MEK inhibition preserves genomic stability in naive human embryonic stem cells. Global analysis reveals multiple pathways for unique regulation of mRNA decay in induced pluripotent stem cells. Taubenschmid-Stowers, J. et al. 8C-like cells capture the human zygotic genome activation program in vitro. Mazid, M. A. et al. Rolling back human pluripotent stem cells to an eight-cell embryo-like stage. & Pera, M. F. The pluripotent state in mouse and human. Intarapat, S. & Stern, C. D. Chick stem cells: current progress and future prospects. Macfarlan, T. S. et al. Embryonic stem cell potency fluctuates with endogenous retrovirus activity. Yang, F. et al. DUX–miR-344–ZMYM2-mediated activation of MERVL LTRs induces a totipotent 2C-like state. Li, D., Yang, J., Huang, X., Zhou, H. & Wang, J. eIF4A2 targets developmental potency and histone H3.3 transcripts for translational control of stem cell pluripotency. Herzog, V. A. et al. Thiol-linked alkylation of RNA to assess expression dynamics. Quan, Y. et al. Cnot8 eliminates naive regulation networks and is essential for naive-to-formative pluripotency transition. The translational landscape of ground state pluripotency. Factor, D. C. et al. Epigenomic comparison reveals activation of ‘seed' enhancers during transition from naive to primed pluripotency. Cheng, S. et al. Mammalian oocytes store mRNAs in a mitochondria-associated membraneless compartment. & He, C. RNA N6-methyladenosine methylation in post-transcriptional gene expression regulation. Bayerl, J. et al. Principles of signaling pathway modulation for enhancing human naive pluripotency induction. Khong, A., Matheny, T., Huynh, T. N., Babl, V. & Parker, R. Limited effects of m6A modification on mRNA partitioning into stress granules. Yang, Y. C. et al. CLIPdb: a CLIP–seq database for protein–RNA interactions. Vessey, J. P. et al. Dendritic localization of the translational repressor Pumilio 2 and its contribution to dendritic stress granules. Lin, S. & Gregory, R. I. MicroRNA biogenesis pathways in cancer. & Parker, R. MicroRNA-dependent localization of targeted mRNAs to mammalian P-bodies. Sen, G. L. & Blau, H. M. Argonaute 2/RISC resides in sites of mammalian mRNA decay known as cytoplasmic bodies. Leung, A. K., Calabrese, J. M. & Sharp, P. A. Quantitative analysis of Argonaute protein reveals microRNA-dependent localization to stress granules. & Blelloch, R. H. Regulation of microRNA function in somatic stem cell proliferation and differentiation. Galagali, H. & Kim, J. K. The multifaceted roles of microRNAs in differentiation. Shi, J. et al. PANDORA-seq expands the repertoire of regulatory small RNAs by overcoming RNA modifications. Du, P. et al. An intermediate pluripotent state controlled by MicroRNAs is required for the naive-to-primed stem cell transition. Moradi, S., Asgari, S. & Baharvand, H. Concise review: harmonies played by microRNAs in cell fate reprogramming. Licursi, V., Conte, F., Fiscon, G. & Paci, P. MIENTURNET: an interactive web tool for microRNA–target enrichment and network-based analysis. Ngondo, R. P. et al. Argonaute 2 is required for extra-embryonic endoderm differentiation of mouse embryonic stem cells. Zhu, Y. et al. Molecular mechanisms for CFIm-mediated regulation of mRNA alternative polyadenylation. Fabian, M. R. et al. Mammalian miRNA RISC recruits CAF1 and PABP to affect PABP-dependent deadenylation. Enhanced single RNA imaging reveals dynamic gene expression in live animals. A. et al. DUX4 expression in cancer induces a metastable early embryonic totipotent program. Levin-Ferreyra, F. et al. Transposable element activity captures human pluripotent cell states. Seydoux, G. & Braun, R. E. Pathway to totipotency: lessons from germ cells. Saitou, M. Mammalian germ cell development: from mechanism to in vitro reconstitution. Robust in vitro induction of human germ cell fate from pluripotent stem cells. Du, P. & Wu, J. Hallmarks of totipotent and pluripotent stem cell states. Hancock, G. V., Wamaitha, S. E., Peretz, L. & Clark, A. T. Mammalian primordial germ cell specification. Core transcriptional regulatory circuitry in human embryonic stem cells. Chambers, I. et al. Nanog safeguards pluripotency and mediates germline development. Modeling sex differences in humans using isogenic induced pluripotent stem cells. Martinez-Val, A. et al. Dissection of two routes to naive pluripotency using different kinase inhibitors. Improving the safety of human pluripotent stem cell therapies using genome-edited orthogonal safeguards. Piazzolla, D. et al. Lineage-restricted function of the pluripotency factor NANOG in stratified epithelia. Zhang, Y., Park, C., Bennett, C., Thornton, M. & Kim, D. Rapid and accurate alignment of nucleotide conversion sequencing reads with HISAT-3N. Tarailo-Graovac, M. & Chen, N. Using RepeatMasker to identify repetitive elements in genomic sequences. & Wang, X. miRDB: an online database for prediction of functional microRNA targets. A., Duffy, E. E., Kiefer, L., Sullivan, M. C. & Simon, M. D. TimeLapse-seq: adding a temporal dimension to RNA sequencing through nucleoside recoding. We thank all members of the Brumbaugh and Di Stefano groups for stimulating scientific discussions. We are grateful to M. Esteban for the TPRX1-GFP construct, J. Hanna for TET-OFF METTL3 human ES cells, T.J. Stasevich for helpful discussions and MS2 constructs, K. Loh and L.T. is a Cancer Prevention and Research Institute of Texas (CPRIT) Scholar in Cancer Research. is supported by the CPRIT Recruitment of First-Time, Tenure-Track Faculty Award (RR200079), the American Society of Hematology Scholar Award, the B+ Foundation, Worldwide Cancer Research and the National Institutes of Health (NIH; R35GM147126, R01CA291649 and R21AI193649). J.B. is grateful for support from the NIH (R35GM142884) and the Boettcher Foundation's Webb-Waring Biomedical Research Awards program. was supported by the NIH award 1F30HD114315. This project was supported by the Cytometry and Cell Sorting Core at Baylor College of Medicine with funding from the CPRIT Core Facility Support Award (CPRIT-RP180672) and the NIH (CA125123 and RR024574). These authors contributed equally: Patrizia Pessina, Mika Nevo. Stem Cells and Regenerative Medicine Center, Baylor College of Medicine, Houston, TX, USA Patrizia Pessina, Srikanth Kodali, Yingzhi Cui, Emily J. Park, Florencia Levin-Ferreyra, Alejandra Rivera Tostado & Bruno Di Stefano Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX, USA Patrizia Pessina, Srikanth Kodali, Yingzhi Cui, Emily J. Park, Florencia Levin-Ferreyra, Alejandra Rivera Tostado & Bruno Di Stefano Department of Molecular and Cellular Biology, Baylor College of Medicine, Houston, TX, USA Patrizia Pessina, Srikanth Kodali, Yingzhi Cui, Emily J. Park, Florencia Levin-Ferreyra, Alejandra Rivera Tostado & Bruno Di Stefano Dan L. Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX, USA Patrizia Pessina, Srikanth Kodali, Yingzhi Cui, Emily J. Park, Florencia Levin-Ferreyra, Alejandra Rivera Tostado & Bruno Di Stefano Department of Molecular, Cellular and Developmental Biology, University of Colorado Boulder, Boulder, CO, USA Mika Nevo, Eduard Casas & Justin Brumbaugh University of Colorado Cancer Center, Anschutz Medical Campus, Aurora, CO, USA Mika Nevo, Eduard Casas & Justin Brumbaugh Charles C. Gates Center for Regenerative Medicine, University of Colorado Anschutz Medical Campus, Aurora, CO, USA Mika Nevo, Eduard Casas & Justin Brumbaugh J. David Gladstone Institutes, San Francisco, CA, USA Alicia L. Richards, Erica Stevenson, Nevan J. Krogan & Danielle L. Swaney Alicia L. Richards, Erica Stevenson, Nevan J. Krogan & Danielle L. Swaney Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA, USA Alicia L. Richards, Erica Stevenson, Nevan J. Krogan & Danielle L. Swaney BioFrontiers Institute, University of Colorado, Boulder, CO, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar created the 8C human ES reporter line; A.R.T. derived and expanded chicken ES cells; F.L.-F. performed TET-OFF METTL3 cell culture experiments. Correspondence to Justin Brumbaugh or Bruno Di Stefano. 's laboratory has received research support from Vir Biotechnology, F. Hoffmann-La Roche and Rezo Therapeutics. also has financially compensated consulting agreements with the Icahn School of Medicine at Mount Sinai, New York, Interline Therapeutics, Rezo Therapeutics, GEn1E Lifesciences and Twist Bioscience (all within the last 36 months). The other authors declare no competing interests. Nature Biotechnology thanks Jacob Hanna and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Representative image of pre-sorted cell lysate containing GFP-LSM14A+ P-bodies (scale: 10μm). (b) Representative imaging of GFP-LSM14A puncta (green) in control and DDX6 KD HEK293T cells. Nuclei were counterstained with DAPI (blue) (scale: 10μm) (left panel). Unpaired two-sided Student's t-test, median ± s.d., ****: p<0.0001. (c) Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in control and DDX6 KD HEK293T cells. (d) Representative flow cytometry plots showing gating for SYTOX BLUE+ events in GFP+ and GFP− gates of pre-sorted cytoplasmic fraction from HEK293T cells. (e) Venn diagram showing the overlap between P-body-associated mRNAs in HEK293T from SMART-Seq and SnapTotal-Seq. (f) Venn diagram showing the overlap between P-body-associated mRNAs in HEK293T of this study (blue) and a published dataset from29 (grey). (g) Representative FISH imaging of TET2 RNA molecules (red) combined with imaging of GFP-LSM14A puncta (green). Nuclei were counterstained with DAPI (blue) (scale: 5 μm). Quantification of TET2 mRNA molecules in P-bodies based on FISH (n=30 cells; left) and P-body sequencing (right). (h) Correlation between mRNA half-life as determined in ref. 49 and P-body enrichment in HEK293T cells, Pearson correlation test (two-sided), p=3.08e-106. (i) Length (left panel) and GC content (right panel) density plots of mRNAs in purified P-body and cytoplasmic fractions of HEK293T cells. (a) Representative IF images of lineage-specific markers for the indicated samples. Nuclei were counterstained with DAPI (blue) (scale: 25μm) (upper panel). Quantification of lineage-specific positive cells in the indicated samples (bottom panel). (b) Heatmap showing expression levels of lineage-specific genes for the indicated samples based on RNA-seq analysis (n=2 biologically independent samples per group). (c) Representative IF imaging of EDC4 puncta (green) in the indicated samples. Nuclei were counterstained with DAPI (blue) (scale: 10μm). (d) P-body number in the indicated samples. (e) Correlation between mRNA half-life as determined in Ref. 57 and P-body enrichment in primed human ES cells, Pearson correlation test (two-sided). (f) Distribution of biotypes within the P-bodies and cytoplasm of each cell type. (g) Density plots of mean RPKM for genes in each category: P-body enriched (log2FC>0, p<0.05), cytoplasm enriched (log2FC<0, p<0.05), or unenriched (everything else). Two-sided Wilcoxon test with Benjamini–Hochberg adjustment shown. (h) Ratio of repetitive elements in purified P-body vs. cytoplasmic fractions of the indicated samples. (i) Differential expression analysis of transposable element expression in purified P-body vs. cytoplasmic fractions of the indicated samples. Differential enrichment was assessed using DESeq2 (two-sided Wald test, Benjamini–Hochberg adjusted p-values). (j) Heatmap showing expression levels of differentially enriched mRNAs between purified P-body fractions of the indicated samples. Manual clustering represents P-body enriched genes that are shared between two or more cell types. Gene number in each cluster is indicated in the figure (n=2 biologically independent samples per group, p<0.05). (a, b) Representative FISH imaging of OCT4 RNA molecules (red) combined with imaging of GFP-LSM14A puncta (green) in primed ES cells (a) and endoderm progenitors (b) with corresponding quantification (n=30 cells; right). Nuclei were counterstained with DAPI (blue) (scale: 5μm). (c) GSEA of P-body vs cytoplasm differential expression in human primed ES cells, endoderm progenitors, and mesoderm progenitors, with gene sets related to the differentiated and preceding developmental stage. (d) A schematic of the strategy for P-body dissolution in human endoderm progenitors during their differentiation to hepatocytes. (e) Representative IF imaging of AFP (green) positive cells in hPSCs upon DDX6 KO compared to control cells. Nuclei were counterstained with DAPI (blue) (scale: 100μm) (Left panel). (f) qRT-PCR analysis for the indicated genes in hepatocytes. (g) A schematic of neuron maturation. (h) Venn diagram showing the overlap between P-body-associated mRNAs in neurons cultured for 7 and 20 days. (i) GSEA of P-body vs cytoplasm differential expression in day 7 and day 20 neurons, with a neural progenitor related gene sets. Enrichment significance was calculated by permutation test (two-sided), with multiple testing correction using the Benjamini–Hochberg method. (j) A schematic of endoderm progenitor maturation. (k) GSEA of P-body vs cytoplasm differential expression in endoderm progenitors and mature endoderm progenitors with a gene set for primed human ES cell-related genes. Enrichment significance was calculated by permutation test (two-sided), with multiple testing correction using the Benjamini–Hochberg method. (l) GSEA of P-body vs cytoplasm differential expression in naïve human ES cells with a gene set for 8-cell embryo58. Enrichment significance was calculated by permutation test (two-sided), with multiple testing correction using the Benjamini–Hochberg method. (a) qRT-PCR analysis of naïve-specific and primed-specific gene expression in naïve and primed mouse ES cells (n=3 biologically independent samples per group). (b) Heatmap showing expression levels of differentially enriched mRNAs between cytoplasmic fractions of naïve and primed mouse ES cells, with GO pathway analysis of upregulated genes in each cluster. Clusters were generated manually to reflect genes that are cytoplasm enriched in both or one of the cell types. Gene number in each cluster is indicated in the figure (n=2 biologically independent samples per group, p<0.05). (c, d) Venn diagrams showing the overlap between P-body enriched mRNAs in human and mouse naïve ES cells (c) and human and mouse primed ES cells (d). (e) Representative bright field image (left panel) and GFP expression (right panel) in GFP-LSM14A chicken ES cells (scale: 50μm). (f) Heatmap showing P-body enrichment in human and mouse primed and naïve cells for genes enriched in P-bodies from chicken ES cells. (g) Length (left panel) and GC content (right panel) density plots of mRNAs enriched in purified P-body and cytoplasmic fractions of chicken ES cells. (h) Forest Plot of odds ratios for P-body enriched genes in each cell type; Center points represent the odds ratio, and error bars represent 95% confidence intervals (CI), n naive hESC=2633, n primed hESC=2980, n naive mESC=2379, n primed mESC=2439, n chicken ESC=1515. (i, j) GO pathway analysis of P-body enriched mRNAs, showing common pathways between chicken ES cells and mouse and human naïve (i) and primed (j) ES cells. Enrichment was tested by two-sided Fisher's exact test with multiple testing correction (Benjamini–Hochberg method). (a) GSEA for blastomere-related transcripts63 in P-body vs cytoplasm differential enrichment data from mouse ES cells using SnapTotal-Seq. Enrichment significance was calculated by permutation test (two-sided), with multiple testing correction using the Benjamini–Hochberg method. (b, c) Representative FISH imaging of Dppa2 RNA molecules (red) combined with imaging of GFP-LSM14A puncta (green) in naïve (b) and primed (c) ES cells with corresponding quantification (n=30 cells; right). Nuclei were counterstained with DAPI (blue) (scale: 5μm). (d, e) Representative FISH imaging of Rex1 (Zfp42) RNA molecules (red) combined with imaging of GFP-LSM14A puncta (green) in naïve (d) and primed (e) ES cells with corresponding quantification (n=30 cells; right). Nuclei were counterstained with DAPI (blue) (scale: 5μm). (f) Cell cycle analysis of MERVL-negative and -positive cells after Ddx6 KD. n=3 biologically independent samples per group, mean ± s.d. (g) Venn diagram showing the overlap between P-body-associated mRNAs in ES cells from SMART-Seq and SnapTotal-Seq. (h) mRNA half-life as determined in66 compared to P-body enrichment in primed mouse ES cells, 2C genes63 are highlighted in pink, Pearson correlation test (two-sided). (i) Read coverage distribution over the gene body of the longest annotated isoforms of genes enriched in P-bodies in naïve mouse ES cells using SMART-Seq and SnapTotal-Seq. (j) Poly-A tail length as determined in67 compared to P-body enrichment based on SMART-Seq and SnapTotal-Seq, Pearson correlation test (two-sided), p=2.35e-69. (a) Representative western blot showing DDX6 protein levels in Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells, either untreated (DMSO) or treated with dTAG-13 for 6 h. n=3 independent experiments. (b) Representative flow cytometry plots showing gating for GFP-LSM14A+ P-bodies in Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells, either untreated (DMSO) or treated with dTAG-13 for 6 h. (c) Representative IF imaging of EDC4 puncta (red) in Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells, treated with dTAG-13 for 3 and 9 h. Nuclei were counterstained with DAPI (blue) (scale: 10mm). (d) GO terms for P-body enriched genes with increased ribosome occupancy of Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells following dTAG13 treatment for 6 h). Enrichment was tested by two-sided Fisher's exact test with multiple testing correction (Benjamini–Hochberg method). (e) Volcano plot of RNA-seq data depicting differential expression of total RNA fraction in Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells following dTAG13 treatment for 6 h, with P-body enriched genes in red and cytoplasm enriched genes in blue (n=3 biologically independent samples per group, p < 0.05). (f) GO terms of P-body enriched genes that showed downregulated gene expression in Ddx6-FKBP12F36V GFP-LSM14A mouse naïve ES cells following dTAG13 treatment for 6 h. Enrichment was tested by two-sided Fisher's exact test with multiple testing correction (Benjamini–Hochberg method). (b) Representative IF imaging of METTL3 (green) in control (no dox) and METTL3 KO induced (dox) ES cells. Nuclei were counterstained with DAPI (blue) (scale: 50μm) (left panel). qRT-PCR analysis of METTL3 expression in control (no dox) and METTL3 KO induced (dox) ES cells. Unpaired two-sided Student's t-test, n=3 biologically independent samples per group, mean ± s.d., ****: p<0.0001 (right panel). (c) m6A RNA dot blot in in control (no dox) and METTL3 KO induced (dox) ES cells. Methylene Blue staining was used as loading control. (d) Representative IF imaging of NANOG (green) in control (no dox) and METTL3 KO induced (dox) ES cells. Nuclei were counterstained with DAPI (blue) (scale: 50μm) (left panel). qRT-PCR analysis of NANOG expression in control (no dox) and METTL3 KO induced (dox) ES cells. Unpaired two-sided Student's t-test, n=3 biologically independent samples per group, mean ± s.d., n.s. : p>0.05; p=0.5542 (e) Representative IF imaging of EDC4 puncta (green) in control (no dox) and METTL3 KO induced (dox) ES cells. Nuclei were counterstained with DAPI (blue) (scale: 10μm) (left panel). P-body number in control (no dox, n=56) and METTL3 KO induced (dox, n=56) ES cells (right panel). (f) Venn diagram showing the overlap between P-body-associated mRNAs in WT (no dox) and METTL3 KO induced (dox) ES cells, and m6A-methylated mRNAs in human ES cells. (g) Motifs enriched in the top 100 P-body enriched genes for each cell type, determined using STREME. (h) Upset plot of the RNA Binding Proteins that were predicted to bind the motifs identified in (g) for each human cell type, determined by Tomtom with the Ray 2013 database. (i) GO terms for proteins that recognize the enriched motifs in all human cell types. Enrichment was tested by two-sided Fisher's exact test with multiple testing correction (Benjamini–Hochberg method). (a) Changes in traditional (upper panel) and PANDORA (lower panel) small RNA distribution between purified P-body and cytoplasmic fractions in the indicated samples. (b) Heatmap showing expression levels of selected miRNAs in P-body and cytoplasmic fractions in mouse naïve ES cells. (c) Heatmap showing expression levels of selected miRNAs in P-body and cytoplasmic fractions in primed human ES cells. (d) Heatmap showing expression levels of selected miRNAs in P-body and cytoplasmic fractions in human neural progenitors. Boxes indicate the interquartile range (25th–75th percentile), center lines the median, and whiskers extend to 1.5X IQR. Unpaired two-sided Wilcoxon tests comparing the targets to all genes that are not targets of P-body enriched miRNA, with Benjamini–Hochberg adjustment, *: p<0.05, **: p<0.01, ***p<0.001, n and exact p-values are reported in Source Data ED Fig 8. (f, g) Box plot showing P-body enrichment of targets of miRNAs enriched in P-bodies of primed ES cells (f) and human neural progenitors (g). Boxes indicate the interquartile range (25th–75th percentile), center lines the median, and whiskers extend to 1.5X IQR. Unpaired two-sided Wilcoxon tests compare the targets to all genes that are not targets of P-body enriched miRNA, with Benjamini–Hochberg adjustment, *: p<0.05, **: p<0.01, ***: p<0.001, n and exact p-values are reported in Source Data ED Fig 8 (h, i) MIENTURNET predictions for miRNA targeting specifically P-body-enriched mRNA in naïve (h) and primed (i) mouse ES cells. P-values were calculated using the hypergeometric test implemented in MIENTURNET. (a, b) MIENTURNET predictions for miRNA targeting specifically P-body-enriched mRNA in primed human ES (a) and neural progenitors (b). P-values were calculated using the hypergeometric test implemented in MIENTURNET. (c) Representative western blot showing AGO2 protein levels in WT and AGO2 KO mouse naïve cells. (d) Representative IF images of EDC4 puncta (green) in naïve and primed mouse ES cells. Cell membranes were labeled with Phalloidin (red), and nuclei were counterstained with DAPI (blue) (scale: 10μm) (left panel). Unpaired two-sided Student's t-test, median ± s.d., *: p<0.05, n.s. (e) Representative images and quantification of Alkaline Phosphatase-positive, transgene-independent iPSC colonies in control and Nudt21 knockdown cells at day 15 of reprogramming. Unpaired two-sided Student's t-test, n=3 biologically independent samples per group, mean ± s.d., **: p<0.01; p=0.0022. (f) qRT-PCR analysis of the expression of ES-specific genes in control and Nudt21 KD reprogramming samples. Unpaired two-sided Student's t test, n=3 biologically independent samples per group, mean ± s.d., ***: p<0.001, ****: p<0.0001; p=0.0001. (g) Representative IF imaging of EDC4 puncta (green) in control and Nudt21 KD reprogramming samples. Nuclei were counterstained with DAPI (blue) (scale: 10μm) and P-body number in control (n=90 cells) and Nudt21 KD induced (dox, n=90 cells) reprogramming samples (right panel). (h) The number of miRNA binding sites per gene isoform for genes that are lost from Nudt21 knockdown P-bodies but retained in Control knockdown P-bodies, in the most highly expressed isoform in each condition. Boxes indicate the interquartile range (25th–75th percentile), center lines the median, and whiskers extend to 1.5X IQR. (i) A schematic of the strategy for the generation of Let7wt and Let7mut reporter cell lines. Unpaired two-sided Student's t test, n=3 biologically independent samples per group, median ± s.d., *: p<0.05, n.s. (l) Heatmap showing expression levels of selected PGCLC-related mRNAs in P-body and cytoplasmic fractions in human primed ES cells (n=2 biologically independent samples per group). Unpaired two-sided Student's t test, n=3 biologically independent samples per group, median ± s.d., *: p<0.05, n.s. (n) A model showing miRNA-mediated RNA sequestration in cell fate. P-body-enriched genes in mouse cell types. P-body-enriched genes in human cell types. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Selective RNA sequestration in biomolecular condensates directs cell fate transitions. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02874-8'>A single-chain derivative of an integrin-activating antibody potentiates organoid growth in Matrigel and collagen hydrogels</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 10:09:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Current methods of culturing human epithelial organoids from adult stem cells may not be compatible with clinical applications as they rely on xenogeneic, chemically undefined or non-standardized components such as the basement membrane extract Matrigel. Matrigel provides a source of extracellular matrix molecules, including laminins and collagen IV, which interact with β1 integrins expressed on organoid cells. Here we describe a single-chain (sc) version of antibody TS2/16 that allosterically activates integrin β1 function in organoids. The addition of monomeric scTS2/16 to organoid medium results in up to a fivefold increase in the yield of all gastrointestinal organoids grown in Matrigel. Collagen I is well defined, available in clinical-grade formulations and, when combined with scTS2/16, may support the clinical application of epithelial organoids derived from gastrointestinal tissues and other epithelial sources. Organoids are self-organizing, miniature versions of epithelial tissues that can be expanded over long time periods. Since the 2009 report of a 3D technique to culture adult stem cell-derived intestinal organoids1, protocols have been published for the isolation of organoids representing epithelial tissues derived from all three germ layers1. However, efforts to apply organoids in areas such as drug screening and clinical transplantation have been hindered by limitations of the culture conditions, which typically include xenogeneic, chemically undefined or non-standardized components. One essential element of these culture conditions is a growth factor cocktail mimicking the stem cell niche. This typically has to be optimized for each individual tissue, yet always contains three key ingredients: potent activators of the Wnt pathway (for example, Wnt3A or its surrogate, plus R-spondins), a tyrosine kinase receptor activator (for example, EGF) and inhibitors of TGFβ and/or BMP (for example, noggin). A second critical component for proper organoid formation is a cell culture substrate consisting of extracellular matrix (ECM) proteins, which in vivo are presented by the basement membrane2 (Fig. This thin layer of a specialized ECM forms the supportive structure for epithelia and includes various laminins (Lns), collagen type IV and, to a lesser degree, arginine–glycine–aspartic acid (RGD) motif-containing molecules such as fibronectin. Lns are heterotrimeric proteins assembled from one of five α chains, one of three β chains and one of three γ chains. In higher organisms, these isoforms show varying degrees of cell and tissue specificity. Collagen IV is a heterotrimeric molecule composed of three α-type chains, which form polygonal networks through several types of intermolecular interactions. Both collagen type IV and Lns are produced in vivo by epithelial cells and form independent networks that are connected by nidogen and the heparan sulfate proteoglycan perlecan. Its major component, embryonic α1β1γ1 Ln (Ln-111), has essential roles in the polarization and differentiation of epithelial cells4. The ECM protein collagen type IV is another major component. Engelbreth–Holm–Swarm tumor extracts, collected from mice, are marketed under the names Matrigel and basement membrane extract (BME). Both are enriched for ECM and, especially Matrigel, have been used extensively for culturing epithelial cells1, including 3D structures from mammary epithelium4. Matrigel has also enabled the establishment of adult stem cell-based organoid technology5. The use of Matrigel is hampered by its cost, batch-to-batch variability, contamination by mouse proteins and ethical considerations on the use of experimental animals. Clinical-grade Matrigel has not been available. a, In vivo, integrin receptors on the basal membrane of epithelial cells contact a mix of ligand proteins of the basal lamina (basal lam.). In vitro, these are replaced by Matrigel or BME proteins. b, Organoids were grown in BME in the absence (control, Cntrl) or presence of a strong inhibitory integrin β1 Ab (AIIB2). Subsequent staining for basal marker integrin β1, in both conditions, was performed on organoids with permeabilized cells. Yellow marked boxes indicate examples of line scans to quantify the intensity profile of the markers shown in c. Scale bars, 50 µm. c, Intensity plot of fluorescent apical and basal markers in cross-sections of the membrane. Permeabilized control- and mAb AIIB2-treated intestinal organoids as shown in b were stained for basal marker integrin β1, apical marker ZO-1 and nuclear marker DAPI as a reference. Apical and basal positions (black vertical lines) were identified based on F-actin staining with phalloidin (not shown), and the mean profile (sharp colored lines) of multiple organoids (n = 8 per condition) was calculated (shadowed areas show the s.d.). d, Quantification of Ki67-positive nuclei after 2 days of culture of control and mAb AIIB2-treated intestinal organoids (N = 1 independent experiment), using immunofluorescence staining (one-tailed t-test P value is 8.8 × 10−3 for n = 4 biological replicates). f, For quantification of viable cells (N = 1 independent experiment), an ATP-sensitive luminescence assay (CellTiter-GLO) was performed at the indicated days. Data were normalized based on the mean values of day 0. The contact that epithelial cells make in vivo with ECM proteins (or in vitro with Matrigel) is mediated by αβ1 integrins6. Interactions with Lns proceed via α3β1 and α6β1 (ref. A third class of ECM proteins, which includes fibronectin, interacts with α5β1 integrin through the short RGD sequence. Various studies have described synthetic hydrogels (for example, polyisocyanide (PIC)8 or polyethyleneglycol (PEG))9, functionalized with integrin-binding motifs taken from ECM proteins. The individual chains of heterotrimeric laminins act cooperatively in binding to integrins with high affinity, and as a consequence, no simple binding motifs have been defined. Therefore, recombinant full-length Ln-111 was coupled to a PIC gel10. The in vitro synthesis of trimeric laminins in mammalian cells is, however, complex and expensive. By contrast, several synthetic PEG gels have been described in which the GFOGER peptide, alone or in conjunction with the RGD adhesion motif, maintains gastrointestinal organoids, albeit less efficiently than the Matrigel benchmark11. Similar results were obtained with ductal pancreas organoids12. To improve the growth of human intestinal organoids in hydrogels, we were inspired by allosteric antibodies activating the integrin β1 chain operating on leukocytes, first described approximately three decades ago13. Like epithelial cells, leukocytes express several β1 integrins, which mediate interactions with ECM components as well as other cells. Leukocyte adhesion is a highly regulated and essential process in the immune response and inflammation. The human integrin β1-specific TS2/16 antibody14 strongly enhances the adhesion by α2β1, α5β1 and α6β1 integrins with their respective ECM ligands, as well as of α4β1 to endothelial VCAM protein. A similar TS2/16-induced increase in cell-substrate adhesiveness has been observed for non-hematopoietic cells: for example, neuronal cells in defined stages of chicken embryo development were induced to bind to Ln in vivo in the presence of the β1-activating antibody TASC15. The non-constitutive nature of integrin interactions implies that ligand binding requires structural changes in the integrins themselves. This was first shown for the αVβ3 and αIIbβ3 integrins16. Electron microscopy (EM) studies subsequently revealed that the β1 integrins adopt similar conformational changes17. Integrins are known to exist in three formational states16: two low-affinity states, bent–closed and extended–closed, and one high-affinity state characterized by an extended conformation with an open ligand binding domain (headpiece). A variety of β1 subunit-specific antibodies have been used to induce or stabilize these conformational states and measure their affinity (Supplementary Fig. This well-described ability of TS2/16 to enhance leukocyte adhesion inspired us to explore its properties in epithelial organoid cultures. We designed a single-chain variable fragment (scFv) of the original TS2/16 antibody that similarly activates β1 integrins with the additional advantage of being expressed in bacteria at high yields. The potency of this designed integrin activator in organoid culture is shown across various human epithelial organoids in 3D and 2D in Matrigel and collagen I hydrogels. Adult stem cell-derived intestinal organoids, showing a basal-out phenotype in the presence of external Matrigel, invert their polarity to basal-in when cultured in the absence of Matrigel19. We tested the role of integrin β1, dominantly expressed in various organoid models (Extended Data Fig. The rat anti-integrin β1 antibody AIIB2 has previously been shown to allosterically block the recognition of β1 integrins for all three major integrin ligand classes (Ln, collagen and RGD-class proteins)18. This antibody enabled us to comprehensively test the role of the β1 integrin component in basal–apical polarization of diverse organoid types. The ‘basal-out' phenotype of these organoids, cultured in the presence of BME, inverted within 12 h to a basal-in phenotype following exposure to AIIB2 (Fig. This condition resulted in an immediate abrogation of growth, measured as a strong decrease in the percentage of cells expressing the nuclear proliferation marker Ki67 (Fig. 1e) and in a quantitative cellular ATP assay (Fig. These observations implied that integrin-mediated signals normally maintain growth and are blocked by AIIB2. Further confirmation for cell cycle exit in response to AIIB2 was seen during a time-course experiment on 2D BME-coated transwells loaded with a human ileal organoid line, stably expressing the fluorescent ubiquitination based cell cycle indicator (FUCCI) reporter(Extended Data Fig. A quantitative PCR analysis, performed on 3D BME-grown duodenal organoids, indicated that AIIB2-arrested cells differentiate into enterocytes and goblet cells (Extended Data Fig. We constructed a humanized IgG version and single-chain (sc) derivative of the mouse TS2/16 antibody, the latter for ease of production in bacteria (Fig. 14) and used human codon-optimized synthetic DNA (Extended Data Fig. 4) to clone these into huIgG1 H- and қL-chain vectors. The humanized IgG was subsequently purified using standard chromatography purification (Fig. 2a) and connected via a 3×(GGGGS) linker. Both protein versions (Extended Data Fig. 5) were cloned into a pET expression vector using Escherichia coli codon-optimized DNA. To obtain pure sc versions, a routine protocol for protein extraction, solubilization and refolding was implemented20. On average, we obtained 1 mg pure scTS2/16 per 50 ml of bacterial suspension, sufficient for approximately 7 l of cell culture medium at a final concentration of 5 nM. sodium dodecyl sulfate polyacrylamide gel (SDS-PAGE) analysis of the purified scFv version of TS2/16 (VH–VL) is shown in Fig. We also developed a protocol to produce scTS2/16 in the human cell line HEK293 (Extended Data Fig. Functionally, both huIgG1 and scFv versions (VH–VL) induced adhesion at low nM concentrations of K562 (erythroleukemia) cells to fibronectin, a standard assay for integrin-mediated adhesion (Fig. We decided to focus on the scFv (VH–VL) version of TS2/16 (scTS2/16) for subsequent studies. a, Integrin recognition-providing VH and VL chains of the mouse hybridoma TS2/16 were cloned into a human IgG1 backbone or combined in a single-chain antibody version. b, SDS-PAGE analysis of purified TS2/16 IgG (±180 kDa) and scFv (±26 kDa) (shown is the VH–VL version) under reducing and non-reducing conditions (N = 2). Proteins were visualized by Coomassie Blue staining. The positions of separate H and L chains of IgG under reducing conditions are indicated. The molecular weights of markers in kilodaltons (kDa) are indicated on the left margin. c, Bright-field images of the K562 leukocyte cell-based adhesion reporter assay. K562 cells were allowed to adhere to plastic-coated fibronectin for 1 h at 37 °C in the absence or presence of 5 nM of IgG- or scFv-type TS2/16 antibody. Cells, remaining after three buffer washes under the conditions indicated, are shown (triplicate values). d, A luminescence cell viability assay (CellTiter-Glo) was used to quantify the adhesion of K562 cells (N = 3 for number of experiments) to the fibronectin-coated plate under conditions shown in c. RLU values are in millions; means of triplicate values (n = 3) and s.d. are indicated (one-tailed t-test P = ±6.6 × 10−5 for IgG TS2/16 versus no Ab, and 7.7 × 10−7 for ctr versus scTS2/16). 22) binds to Ln-111, the embryonic Ln present in BME. 8), but these are absent in BME. The specific role of α6β1 integrin in Ln-111 binding was confirmed by exogenous expression of integrin α6β1 in K562 cells. Ln-111 recognition acquired following integrin-α6 transfection was evident, but certainly not maximal. It could be strongly enhanced by adding the activating TS2/16 antibody23. We tested whether this phenomenon would translate to potentiated adhesion of colon organoid cells to coated BME. The adhesion in the absence of scTS2/16 (11 × 105 relative light units (RLU)) reached its maximum (22 × 105 RLU) at 2 nM (Fig. The AIIB2 blocking antibody inhibited the spontaneous adhesion of organoid cells, as a validation of this adhesion assay. As BME contains both Ln-111 and collagen IV, we tested the contributions of these components independently in organoid adhesion. Again, we detected maximum enhancement effects of scTS2/16 at 2 nM for the individual components. The inhibitory AIIB2 antibody completely inhibited the adhesion of organoids to both coats (Supplementary Fig. To assess whether the integrin α6 subunit was also crucial in mediating growth of colon organoids in BME, we added the inhibitory integrin α6-specific GoH3 antibody24 while regularly splitting colon organoid cultures. 3b indicated a strong GoH3-induced reduction of growth. We then tested whether activation of the β1 integrins could enhance the growth of colonic organoids. Addition of scTS2/16 to standard 3D colon organoid culture in BME at 5 nM led to a 3-fold increase in colon organoid growth (0.9 × 106 to 2.8 × 106 RLU). This was observed after 10 days of culture by assessment of total genomic DNA, ATP-dependent luminescent light units (Fig. 3c) and bright-field images of the organoid cultures (Fig. We supplemented 1 nM MnCl2-containing, complete organoid medium with 1 mM MnCl2. We then directly compared the Mn2+ effects on adhesion and proliferation to 5 nM scTS2/16. While both induced an increase in adhesion, only scTS2/16 potentiated growth (Supplementary Fig. a, Quantification of colon organoid-derived single cells adhering to coated BME expressed as RLU × 105 from a cellular ATP-driven luminescence assay (CellTiter-Glo). The graph represents trend lines based on mean ± s.d. b, Routine cultivation of identically processed fragmented intestinal organoids in the absence or presence of integrin subunit α6 blocking mAb GoH3. Quantification of ATP-driven luminescence assay (RLU × 105) after 7 days of culturing is presented as mean ± s.d. (n = 4 biological replicates) (one-tailed t-test P = 3.4 × 10−5 for cntr versus GoH3). c, Routine cultivation of identically processed fragmented intestinal organoids in the absence or presence of scTS2/16. (n = 4 biological replicates) of total genomic DNA (y-axis) and relative luminescence (×106) in the cell viability assay at indicated days are depicted. d, Bright-field culture images of intestinal organoids from the experiment shown in c, as present on days 1 and 9. e, Culture images of a cloning assay, starting with a very dilute single-cell suspension. f, Quantification of the number of organoids (right y-axis) scored after 7 days from organoids as presented in e; the left y-axis shows cell viability expressed as RLU × 103 from a cellular ATP-driven luminescence assay (CellTiter-GLO). P values one-tailed t-tests comparing the luminescence of the control versus that of GoH3, scTS2/16 or both are 8.7 × 10−2, 2.0 × 10−4 and 1.3 × 10−1, respectively. For number of cells, these values are in succession 1.6 × 10−5, 5.7 × 10−7 and 2.9 × 10−1. The positive effects of integrin β1-stimulatory antibodies on the outgrowth of organoids from single cells were much less strong for the two other β1-chain conformation-stabilizing antibodies, 9EG7 (ref. The TS2/16-stabilized high-affinity conformation of β1 integrins on leukocytes enhances intercellular (VCAM-1)29 and cell-to-ECM adhesion17,30. Our data show that an scFv version of the antibody also improves the establishment and expansion of epithelial organoids by the enhancement of α6β1-mediated interaction with Ln-111/collagen IV as presented in the BME. The pertinent organoids were grown in BME supplemented with control medium or with 5 nM of either the inhibitory mAb AIIB2 or the stimulating scTS2/16 antibody. AIIB2-induced inhibition and scTS2/16 enhancement of growth (compared with the medium without antibody) were observed in all cases (fold increases in examples shown in Fig. 4 were 5-fold (stomach), 2.2-fold (duodenum), 1.6-fold (liver), 8.1-fold (pancreas), 1.5-fold (jejunum) and 1.4-fold (ileum)) (Fig. To exclude major morphological changes resulting from the use of scFv, high-magnification confocal images of 3D BME-scTS2/16-grown jejunal organoids are presented in Supplementary Fig. To assess whether scTS2/16-stimulated growth caused shifts in the transcriptome of the cells, bulk sequence experiments were performed on control and Ab-stimulated 3D BME cultures of P26N colon, Duo-72 duodenum and J2 jejunum organoids (Extended Data Fig. The statistical analysis revealed no significant changes. We concluded that growth of gastrointestinal tract organoids depends on β1-class integrins and is strongly enhanced by the addition of scTS2/16 to their medium. For quantification of the number of viable cells, an ATP-sensitive luminescence assay (CellTiter-GLO) was performed on the same day the images were taken. One-tailed t-test P values for the control versus AIIB2 and scTS2/16 are 1.5 × 10−4 and 3.1 × 10−3 for the stomach (number of experiments, N = 1; n = 3 biological replicates). Based on the scFv-generated improvement of gastrointestinal organoids in BME, we tested the hypothesis that organoid growth in an alternative hydrogel would also benefit from the addition of scTS2/16. In a first attempt to enhance growth by scTS2/16 in a collagen type I hydrogel, we focused on colon organoids. A commercially available preformulated isotonic form of collagen I with neutral pH forms a 3D hydrogel by simply warming to 37 °C (PureCol EZ Gel). In total, 4 α-integrin subunits (α1, α2, α10 and α11) harbor a collagen-binding I-domain insert and share the β1 subunit. Of these, we observed the expression of α2β1 in BME-grown colon organoids, both at the RNA level (Extended Data Fig. In accordance with these findings, the scTS2/16 amplified adhesion of colon organoid cells toward PureCol EZ Gel (uninduced 7.5 × 105 to 22.5 × 105 when scTS2/16-induced) (Fig. By contrast, the inhibitory AIIB2 antibody blocked the uninduced and induced components of this adhesion (Fig. P26N colon organoids showed a ±3-fold increase of their growth in collagen (Supplementary Fig. Also in these colon organoids, no major morphological changes were observed in confocal images (Supplementary Fig. a, BME-grown intestinal organoids stained for integrin subunit α2 and nuclear marker DAPI (N = 1). b, Quantification of intestinal organoid-derived single cells adhering to coated collagen I, using a cellular ATP-driven luminescence assay (CellTiter-Glo). The graph represents trend lines based on mean ± s.d. (RLU × 105) (n = 3 biological replicates) for each condition. c, Sanger sequence validation of ITGA2 gene KO using CRISPR–Cas9 base editing (base C>T editing). The targeted base is highlighted in orange, and the PAM sequence is identified in blue. Single colonies were grown in the presence of scTS2/16 and picked for sequencing. d, Parental and ITGA2-KO organoids were grown in parallel in BME and PureCol EZ Gel, in the presence of scTS2/16 (N = 3 for number of experiments). In parallel wells (n = 1 biological replicate), tissue yield was measured by counting cells (numbers shown in the images). e,f, Primary biopsies (N = 3 experiments) of colon (e) and cholangiocyte (N = 2 experiments) (f) tissue were processed and embedded in collagen I in the presence or absence of scTS2/16. For the colon, images and quantification were done after 10 days of culture (n = 4 biological replicates). For primary cholangiocytes, collagen-derived organoids (Supplementary Fig. Subsequently, scTS2/16 sensitivity was measured (N = 1 for number of experiments, with 6 biological replicates) by counting the number of cells (P value one-tailed t-test for scFv effect 5.8 × 10−5). We then created null alleles of the gene for the integrin α2 subunit (ITGA2) by CRISPR to confirm the relevance of the collagen-recognizing (α2β1) integrin for colon organoid growth in collagen hydrogels (Fig. As predicted, ITGA2 gene mutation blocked organoid growth in collagen I, even in the presence of scTS2/16. While BME enabled isolation of the ITGA2−/− cells, growth was slightly reduced compared with the parental organoids, indicating that BME growth of organoids is at least in part controlled by collagen IV (Fig. We then tested fresh colon biopsies for organoid outgrowth in collagen I hydrogel, both in the presence and absence of scTS2/16. Colon tissue from donors (N = 3) was enzymatically processed and epithelial cell suspensions were embedded in collagen I. Both BME-established organoid lines, as well as those from primary biopsies, showed difficulty expanding in collagen hydrogel after repeated splitting. A ‘niche-inspired' medium was recently designed for intestinal organoids38. Each of these contributed to improved growth. In addition, CHIR and forskolin further enhanced the culture. At passage 1, organoids growing in the absence of scTS2/16 lost their apical–basal polarity as well as the proliferation marker Ki67 (Supplementary Fig. We then tested the same culture conditions growing an ileum organoid line harboring fluorescent reporters for all three secretory cell types, enabling monitoring of ongoing differentiation. These N39 triple-reporter organoids could be passaged at least six times in collagen (Supplementary Fig. 10d), but appeared fully dependent on it for the addition of scTS2/16. Cells from passages P0 and P2 showed differentiation toward all three secretory cell types when subjected to differentiation medium. 10a presents fluorescent confocal images of the reporters while Supplementary Fig. 10b provides a quantitative analysis of these reporters based on flow cytometry. When ITGA2-expressing human liver bile duct organoids (Extended Data Fig. 1) were stimulated by scTS2/16, increased adhesion toward collagen I ensued. Guided by these results, we tested the outgrowth of cholangiocyte organoids in collagen I hydrogels, directly from primary liver biopsies. There were consistently more organoids in the scTS2/16 medium compared with the control medium from 3 to 13 (ctr versus scFv) and from 13 to 22 (ctr versus scFv), respectively (n = 2) (Supplementary Fig. These proved to be epCAMhigh, as expected for this cell type. A substantial twofold increase in number of organoids (ctr 15 to 35) was seen for ctr and scTS2/16, respectively. Based on the principles established in the 3D cultures, we tested whether collagen-based culturing of organoids in 2D would similarly be enhanced by scTS2/16. Such cultures have been described previously, in which intestinal cells were cultured on a monolayer of myofibroblasts39. Other protocols allow short-term human organoid growth40 or apply a more complex collagen sandwich technique in combination with the use of small molecules (CHIR and valproic acid) to maintain growth41. Here we tested scTS2/16-supported growth of P26N colon, Duo-72 duodenal and J2-jejunal organoids in transwells. These were coated with the same collagen used in 3D. Growth was compared with that in BME-coated inserts under otherwise identical conditions. Results are shown in Supplementary Fig. scTS2/16-supplemented P26N colon cultures produced 2.8-fold to 4.5-fold more tissue, on BME and collagen, respectively. For the Duo-72 duodenum cultures, improvements in response to scTS2/16 were measured both on BME (amplification ±2.4-fold) and collagen (amplification ±2.4-fold). For the jejunum line (Supplementary Fig. 14), these amplification factors were ±7.5 (BME) and ±4.75 (PureCol EZ Gel). Bulk RNA sequencing confirmed the stability of these in vitro intestinal cultures (Extended Data Fig. Given these positive short-term 2D results, we then tested whether these effects would be maintained in long-term culture. We first cultured Duo-72 duodenum organoids in collagen-coated transwell inserts in the presence and absence of scTS2/16 (Fig. We passaged both types of cultures four times, restarting with a fixed number of single cells at each round. By extrapolating these values over four passages, we found that approximately 15 times more cells were produced in the presence of scTS2/16. 6b), we cultured J2 jejunum organoids in collagen-coated inserts for six passages in the presence of scTS2/16. We then tested the competency of these cells to differentiate to the major specialized intestinal cell types. a, Jejunal organoid cultures were started with 5 × 104 cells (n = 2) on PureCol EZ Gel-coated inserts of transwells. Intestinal expansion medium, ±scTS2/16, was present in the upper and lower compartments. Passaging of control and scFv cultures took place after approaching full confluency in the latter. In both control and scTS2/16 conditions, cells were counted and gain factors calculated (table at the bottom). Freshly coated inserts were reloaded with 5 × 104 cells. Cumulative growth as depicted in the graph was calculated by multiplying the measured gain factors in successive passages. b, Duodenal organoids were scTS2/16 grown in collagen I-coated transwells. Both scTS2/16-supplemented differentiation media were applied only to the bottom compartment (liquid–air culture). Here we exploit the functional importance of integrin β1 in epithelial organoid cultures derived from the human small intestine, colon, stomach, pancreas and liver. We describe an sc integrin-activating antibody, scTS2/16, that is easily produced and strongly enhances organoid outgrowth. We observe potent growth-blocking effects of the inhibitory integrin β1 subunit-specific antibody AIIB2, while the activating integrin β1 subunit antibody TS2/16 enhances organoid growth. Moreover, scTS2/16 is highly potent in supporting the establishment and growth of organoids in the standard BME hydrogel, simply by addition to the culture medium as a soluble monomer. We observe a crucial role for the integrin α6 component in BME-dependent growth. We currently believe that organoid growth in BME results from the parallel interaction of α2β1 with collagen IV and α6β1 with Ln-111, both natural ECM components present in BME. We also find that scTS2/16 supports the growth of organoids in hydrogels of purified collagen I. CRISPR-mediated ITGA2-gene knockout showed the α2β1 integrin to be responsible for this effect. Our 2D protocol reveals that scTS2/16 enables potent, long-term culturing of intestinal tissue on collagen coats. The tissue generated under these conditions remains competent to differentiate into secretory Paneth cells and cells participating in the absorption of nutrients (enterocytes) and the production of mucus (goblet cells) and hormones (enteroendocrine cells). The TS2/16 antibody reinforces adhesion and growth in organoids most probably by providing a counterforce to contractility of the cytoskeleton. 43), identical to the effect of Rho-associated coiled-coil containing protein kinase (ROCK) inhibitors44. Involvement of ROCK inhibition in growth was shown in primary keratinocytes, which are conditionally transformed following prolonged exposure to small-molecule inhibitors of the ROCK kinase45. Functional evidence for the initiating role of integrin–ligand signaling in these effects was nicely shown by deleting ITGB1 in preimplantation epiblasts46. Several earlier studies describe well-defined hydrogels for gastrointestinal organoids. Generally, these studies have used organoids established in Matrigel as starting material. The first two involved synthetic PEG gel11,12, functionalized with ligand mimetic peptides PHSRN-RGD (Fn) and GFOGER (Col), combined with a mix of ECM-sequestering peptides. The collagen mimetic GFOGER peptide clearly dominated in driving proliferation over the PHSRN-RGD peptides. This is in line with our collagen results and with fibrinogen hydrogel findings by others47. The second study describes growth of normal mouse ductal pancreas organoids, and mouse and human adenocarcinoma organoids12. In these three organoid types, growth was again largely dependent on the GFOGER peptides. Notably, growth was reduced in the presence of antagonists of laminin–α3β1 or α6β1 interactions in adenocarcinoma-type organoids. It is unclear whether this Ln-driven growth is a characteristic of the adenocarcinoma status, is specific for mouse species or is a result of the PEG-bound ECM-binding peptides retaining cell-secreted laminins. It thus remains unclear whether normal human pancreas organoids would also grow in this gel. An obvious possibility is to supplement such systems with scTS2/16, thereby improving integrin interaction with both ligand mimetics and cell-secreted ECM proteins. A PIC hydrogel, decorated with high concentrations of Matrigel isolated laminin–entactin, or recombinantly produced laminin-111, successfully maintained human liver organoid cultures10. One previous study has described that human colon organoids can be grown in collagen I directly from primary tissue39. We predict that this approach will benefit from the addition of scTS2/16. The Yersinia protein invasin is known to activate α3β1, α4β1, α5β1 and α6β1 integrins, but not the collagen receptors α1β1 and α2β1. We have shown recently that an invasin coat replaces the use of Matrigel in 2D organoid sheet cultures48. This approach is complementary to the current study: invasin acts fully independently of a hydrogel yet does not work in solution, unlike scTS2/16. The observation that the activation of β1 integrins by scTS2/16 in leukocytes extends to epithelial organoid cells opens a range of applications. The availability of this reagent, which allows organoids to grow in collagen-only hydrogels, may avoid the need for BME or Matrigel. Moreover, its production in E. coli or in the HEK293 line allows the generation of clinical-grade batches. Clinical-grade collagen preparations are already widely available for tissue repair37 and cosmetic applications. Mouse and human colon organoids have been shown to be transplantable in mice49,50,51. To date, two ASC-based early-phase organoid clinical trials have been registered: the transplantation of autologous colon organoids as treatment of ulcerative colitis (clinical trial UMIN000030117, Japanese Registry) and the transplantation of salivary gland organoids for treatment of radiation-induced xerostomia (dry mouth syndrome) (clinical trial NCT04593589). Moreover, an scTS2/16-induced high-affinity state of the transplanted organoid cells may instruct these to engraft more efficiently. P26N human colon organoids were grown for 2 days in BME ± the rat anti-integrin β1 antibody AIIB2 (Developmental Studies Hybridoma Bank (DSHB) catalog number AIIB2, Research Resource Identifier (RRID): AB_528306) at a concentration of 1 μg ml−1. Organoids were collected using cell recovery solution (Thermo Fisher) and fixed in 4% formaldehyde (pH 7.4) at 4 °C for 1 h. Hereafter, organoids were washed with PBS 0.1% Tween (PBST) (2% BSA). Three-dimensional imaging of organoids was performed essentially as described previously52. In short, organoids were incubated overnight at 4 °C with primary antibodies binding ZO-I (Thermo Fisher Scientific catalog number 61-7300, RRID: AB_2533938), ITGB1 gene product (R and D Systems catalog number MAB1778, RRID: AB_357909) and Ki67 (Millipore catalog number AB9260, RRID: AB_2142366) or ITGA2-encoded gene (Abcam catalog number ab30487, RRID: AB_775702). Following extensive washing with PBST, binding of primary reagents was visualized by overnight incubation in PBST–2% BSA containing donkey anti-rabbit Alexa 568 (Thermo Fisher Scientific catalog number A10042, RRID: AB_2534017) and goat anti-mouse Alexa488 (Thermo Fisher Scientific catalog number A-11070, RRID: AB_2534114). Phalloidin Atto 747N (Sigma-Aldrich) was used for direct visualization of F-actin. DNA was stained with DAPI D1306 (Thermo Fisher). Organoids were optically cleared in a glycerol–fructose clearing solution before imaging. Organoids were imaged on a Leica SP8 confocal microscope. Analysis was done with ImageJ software. We used a simplified workflow to sequence the variable regions of the mouse TS2/16 hybridoma (American Type Culture Collection (ATCC) HB-243) as described53. A human codon-optimized DNA sequence encoding the VH and VL domains of antibody TS2/16 was synthesized by GeneArt (Thermo Fisher) and cloned into pFUSEss-CHIg-hG1 and pFUSE2ss-CLIg-hK, respectively (Invivogen). Expression vectors were co-transfected into Expi293F human suspension cells using ExpiFectamine (Thermo Fisher). IgG protein was purified by affinity chromatography using CaptivAHF protein-A affinity resin (Repligen), followed by size-exclusion chromatography using a Superdex 200 Increase 10/300 GL column (Cytiva). E. coli codon-optimized DNA coding sequences of the TS2/16 scFvs (Supplementary Fig. 3) were synthesized by GeneArt (Thermo Fisher) and cloned into a pET21 vector harboring a C-terminal 6x-HIS tag (Novagen) using standard molecular biology techniques. Protein expression induction, using 1 mM IPTG (Sigma), was done in BL21(DE3) cells starting with an optical density (OD)600 of 0.6 in standard lysogeny broth (LB) medium. Production was at 28 °C for approximately 18 h. Bacterial pellets obtained were resuspended in lysis buffer (50 mM Tris pH 7.5, 200 mM NaCl, 10% glycerol, 0.4% Triton ×100) and then sonicated (5 cycles; 10 s on, 20 s off). The insoluble fraction, obtained by centrifugation at 20,000 × g for 15 min, was guanidine–hydrochloride solubilized and refolded as described previously20. The partially purified and refolded scFv was further purified using Ni-NTA (Qiagen), followed by size-exclusion chromatography using a Superdex S200 increase 10/300 GL size-exclusion column (Cytiva) in HBS (10 mM HEPES, pH 7.2, 150 mM NaCl). An analysis of the quality of protein purification was performed running samples, under reducing and non-reducing conditions, on a 4–15% TGX precast SDS-PAGE (Biorad) gel. Proteins were visualized using InstantBlue Coomassie Protein stain (Abcam). The coding sequence of the TS2/16 scFv (VH–VL orientation) containing the artificial human kappa L-chain signal sequence, and C-terminal tags (8×His tag), HRV 3C site (LEVLFQ/GP), Fc-tag (huIgG1) and a stop codon, was cloned into the pcDNA3.1 mammalian expression vector (Extended Data Fig. Expression was performed by transient transfection of Expi293 cells using the ExpiFectamine transfection kit (Gibco) according to the manufacturer's instruction. After 4 days, the TS2/16 scFv-Fc was captured from the conditioned media with CaptivA−HF protein-A affinity resin (Repligen, CA-PRI-0100) and eluted with IgG elution buffer (Thermo Scientific) containing 300 mM NaCl and neutralized with 100 mM Tris pH 8.0 to reach eventually pH 7.0. The sample buffer was exchanged to 1 × HBS (20 mM HEPES pH 7.3, 300 mM NaCl) using Amicon centrifugal filters (MWCO 30 kDa) and treated with PreScission protease (Cytiva) overnight at 4 °C. Next, the Fc-tag was captured with CaptivA−HF protein-A affinity resin and the PreSission protease with Glutathione Sepharose 4C (Cytiva). Finally, the TS2/16 scFv was purified on size-exclusion chromatography with Superdex 75 Increase 10/300 GL column (Cytivia) on the ÄKTA pure chromatography system (Cytivia) in 1 × HBS running buffer (20 mM HEPES pH 7.3, 300 mM NaCl). The TS2/16 scFv was then concentrated using Amicon centrifugal filters (Millipore, MWCO 10 kDa) and sterilized by filtration over a 0.22-μm centrifugal filter (Merck). Protein concentration was determined via NanoDrop A280 reading using the respective protein molecular weight and extinction coefficient. Size-exclusion chromatography and SDS-PAGE analysis for final quality control are shown in (Extended Data Fig. Deletion of the ~30 amino acid specific-determining loop in the I-like domain is essential for stable secretion of this ectodomain. The DNA coding sequence was synthesized by GeneArt (Thermo Fisher) and cloned into vector pcDNA3.1 (Addgene). Expression vector was transfected into Expi293F human suspension cells using ExpiFectamine (Thermo Fisher). Purification of the C-terminally Flag epitope tagged was performed by absorption of secreted protein to M2-agarose beads (Sigma), and elution by excess 3 × Flag peptides (Sigma). Eluted proteins were subsequently subjected to size-exclusion chromatography using a Superdex 200 Increase 10/300 GL column (Cytiva). All washes and antibody incubations were in standard PBS. The anti-integrin β4 antibody (Abcam catalog number ab110167, RRID: AB_10866385), the anti-β1 antibody (R and D Systems catalog number MAB1778, RRID: AB_357909) and the anti-HUTS4 (Millipore catalog number MAB2079Z, RRID: AB_11214024), M2 anti-Flag (Sigma-Aldrich catalog number F3165, RRID: AB_259529), anti-Flag HRP (Abcam catalog number ab49763, RRID: AB_869428) and anti-6 × HIS HRP (R and D Systems catalog number MAB050H, RRID: AB_357354) antibodies were used. We used 4–15% SDS-PAGE gel (Biorad) under reducing conditions, and PVDF membrane (Immobilon) for blotting. Enhanced chemiluminescence (ECL) (GE HealthCare) was used in the study. Erythroleukemia cell line K562 cells (Thermo Fisher Scientific) were cultured in DMEM (Gibco) with 100 U ml−1 penicillin–streptomycin and 5% FBS (Sigma-Aldrich). Bovine fibronectin (Thermo Fisher) was diluted to 20 μg ml−1 in PBS and incubated overnight at 4 °C in a flat-bottom 96-well plate (Greiner). Subsequently, wells were blocked with 2% BSA in PBS for 30 min at room temperature. K562 cells were resuspended in DMEM/F12 advanced medium (Gibco). Following incubation of cells for 90 min at 37 °C, unbound cells were removed by three washes with DMEM-F12 advanced medium. The number of adherent cells was analyzed by bright-field microscopic images and a luciferase-based ATP assay (CellTiter-Glo Promega). Colon (P26N) and duodenum (Duo-72) organoids were enzymatically processed to single cells and incubated with integrin component-specific antibodies, each directly conjugated to a different fluorophore. Stained cells were forward scatter (FSC) and side scatter (SSC) gated to remove debris and to focus the integrin analysis on live cells. Analysis of antibody staining was performed on 104 cells per analysis and compared with the same unstained cells. Stainings were performed by using combinations of the directly fluorochrome-conjugated antibodies listed below: Catalog number ab 303014, Biolegend, RRID: AB_493580, APC/cyanine 7 anti-human CD29 antibody Ts2/16 integrin β1 Catalog number ab 313619, Biolegend, RRID: AB_2128022, Pacific blue anti-human/mouse CD49f antibody GoH3 integrin α6 Catalog number ab 328009, Biolegend, RRID: AB_893368, PE anti-human CD49e antibody SAM-1 integrin α5 Catalog number ab 343808, Biolegend, RRID: AB_10641282, APC anti-human CD49c antibody ASC-1 integrin α3 Catalog number ab 328311, Biolegend, RRID: AB_2566271, PE/cyanine 7 anti-human CD49a antibody integrin α1 Catalog number ab 30486, Abcam, RRID: AB_775704,FITC anti-integrin α2 antibody (AK7) 42) organoids cultured in BME were released from the matrix by incubation with dispase (10 U ml−1) at 37 °C (ref. The organoids were then washed with cold DMEM/F12 medium and centrifuged (5 min, 4 °C, 300 × g) to remove residual dispase. The pelleted organoids were treated with TrypLE (no rho kinase inhibitor) at 37 °C to generate cell clusters. This enzymatic reaction was stopped by adding cold DMEM/F12, followed by centrifugation to pellet the cell clusters. The resulting cell clusters were resuspended in 2 mg ml−1 PureCol EZ Gel or BME and plated as small collagen droplets (10–20 µl). After polymerization, the collagen droplets were overlaid with complete expansion medium supplemented with either 1 mM Mn2+ or 5 nM scTS2/16. After day 1, the ileum organoids were maintained in expansion medium supplemented with either 1 mM Mn2+ or 5 nM scTS2/16, with the medium refreshed every 2–3 days. On day 7, organoid cultures were imaged using bright-field microscopy followed by quantification of viable organoids using CellTiter-Glo. The FUCCI sequence comprising mCherry-Cdt1-T2A-Geminin-hmAzami-Green was PCR amplified from a construct provided as a gift by G. Kops (Hubrecht Institute). With the use of a Gibson Assembly (NEBuilder HiFi DNA Assembly), it was cloned into a p2Tol-based vector harboring puromycin resistance (Addgene category catalog number 714859). Reporter organoids were generated by co-electroporation of 5 µg of the FUCCI expression construct together with 5 µg mT2TP transposase mediating the tol2-dependent random integration into the cell's genome. Stable transfectants were isolated based on puromycin resistance and expression of both FUCCI fluorescent signals. RNA was extracted using a QIAwave RNA Minikit (catalog number 74534) according to the manufacturer's protocol. Real-time PCR was performed on the Biorad CFX96 Real Time PCR System using iQ SYBR Green Supermix 1708880. For single-guide RNA (sgRNA) plasmid preparation, the following sequence was ordered containing a universal priming part and the sgRNA (underlined): TTGTGTAATACTGATTCCCCGGTGTTTCGTCCTTTCCACAAG. The additional bold cytosine residue is included to obtain sufficient transcription from the U6 promoter. Side-directed mutagenesis in SpCas9-vector 47511 was done using Q5 high-fidelity polymerase with the mentioned primer and the following universal 5′/phos/- GTTTTAGAGCTAGAAATAGCAAGTTAAAATAAGGC primer. Following PCR cleanup, T4-ligase and Dpn1 were used to relegate and generate the plasmid; this was done according to the manufacturer's protocol (NEB). DNA was transformed into DH5α bacterial cells, and plasmids were verified using Sanger sequencing (Macrogen Europe BV) with sequence primer GGGCAGGAAGAGGGCCTAT. Organoid electroporation was performed as described previously54 with some modifications. Wild-type organoids were maintained in expansion medium. Two days before electroporation, 1.25% (vol/vol) DMSO was added to the organoid medium. Electroporation was done with 106 cells per electroporation with BTXpress solution containing 2.5 μg ITGA2 gene directed sgRNA plasmid, 5 μg plasmid DNA encoding Piggybac transposase, 5 μg plasmid DNA encoding Piggybac hygromycin cassette55 and 7.5 μg C>T base editor_eGFP plasmid (Addgene plasmid number 112100). Hygromycin 100 μg μl−1 (Invivogen) selection was started 5 days post-transfection. Then, 14 days after selection, surviving hygromycin selection clones were individually picked and RT-PCR obtained DNA, sequenced with the following primers: ITGA2_fwd GCACCTGCCACCTTCTCCATCA; ITGA2_rev GAAAAGAGGAAAGAAGCAGATT. During clonal outgrowth and selection, scTS2/16 was always present in the media. Proteins were run on a 4–15% SDS-PAGE gel (Biorad) under reducing and non-reducing conditions. Marker proteins were obtained from Thermo Scientific (26616, Pager ruler). SDS-PAGE gel was transferred to a PVDF membrane (Thermo Scientific reference 88518). Transfer efficiency was controlled with Ponceau staining. For blocking, 5% non-fat milk protein in PBS was used. For direct HIS-tagged protein staining, a mouse anti-HIS tag HRP conjugated antibody (R and D Systems catalog number MAB050H, RRID: AB_357354) was used. Indirect staining for the HIS-tag was performed using anti-HIS (Abcam catalog number S2917, RRID: AB_10641949) in combination with anti-mouse-HRP (R and D Systems catalog number MAB050H, RRID: AB_357354). FLAG-tagged proteins were directly visualized using HRP-conjugated M2 anti-FLAG (Abcam, RRID: AB_869428) or indirectly by combining M2 anti-flag (Sigma-Aldrich catalog number F3165, RRID: AB_259529) and anti-mouse-HRP (Abcam catalog number ab6808, RRID: AB_955441). Chemiluminescent substrate ECL (GE-Healthcare) was used. RNA was extracted using a QIAwave RNA Minikit (catalog number 74534) according to the manufacturer's protocol. RNA integrity was measured using the Agilent RNA 6000 Nano kit with the Agilent 2100 Bioanalyzer, and RNA concentrations were determined using the Qubit RNA HS Assay Kit (Thermo Fisher Scientific). RNA libraries were prepared with the TruSeq Stranded messenger RNA polyA kit and single-end (1 × 50 base pairs) sequenced on an Illumina NextSeq 2000. Reads were mapped to the human GRCh37 genome assembly. Differential gene expression analysis was performed using the DESeq2 package in RStudio (v.2022.02.2). Data visualization was performed using the packages ggplot2 and pheatmap in RStudio or manually plotted using GraphPad Prism (v.8.2.0). The coefficient of determination (R2) was determined using either the ggplot2 package in RStudio or GraphPad Prism. Organoids grown in BME (Cultrex) were passaged by releasing them first from the matrix by incubation for 20 min on ice in cell recovery solution (Thermo Fisher) supplemented with 10 μM of the Rho-kinase inhibitor Y-27632 (Abmole)5. Subsequent DMEM/F12-washed organoids were either mechanically sheared or TrypLE-digested to single cells. Following additional washing with DMEM/F12, organoids were resuspended either in fresh BME or, alternatively, in PureCol EZ Gel (Advanced Biomatrix). For complete gelation of the BME or PureCol EZ Gel gel, plates were incubated at 37 °C for 30 min or 60 min, respectively. Passaging of organoids growing in collagen was achieved by incubation at 37 °C for 30 min maximally, with collagenase from Clostridium histolyticum (Sigma-Aldrich C0130) diluted in DMEM/F12 to a concentration of 1 mg ml−1. For complete removal of the proteolytic enzyme, organoids were washed twice with an excessive volume of DMEM/F12 medium before re-entry into PureCol EZ Gel. The β1-specific inhibitory AIIB2 antibody was added to organoid cultures at 30 nM. The ITGA6-inhibitory antibody G0H3 was added to organoid cultures as 1:1,000 diluted ascites. The Fab versions of SNAKA51, 9EG7 and 12G10 antibodies were provided by J. Li, Harvard Medical School, Boston Children's Hospital. Culture images were taken with an EVOS cell imaging microscope. Transwell inserts (Thincerts, 12-well, pore size 0.4 μm, Greiner Bio-One 665641) were coated overnight with 2% dilutions of BME (ref. 5) or PureCol EZ Gel in DMEM-F12 medium at 4 °C. Inserts were loaded with approximately ±50,000 cells, prepared as a TrypLE-dissociated single-cell suspension. Following digestion, cells were 40-μm-filtered and washed twice in DMEM-F12 medium in the continuous presence of rho-kinase inhibitor Y-27632 (Abmole). Cells were resuspended in 250 µl relevant culture medium supplemented with rho-kinase inhibitor Y-27632 (overnight) and optionally long term with 5 nM scTS2/16. Subsequently, 1 ml of identical medium was added to the lower compartment. Passaging of the cultures was started by removing media from both compartments, briefly washing with DMEM-F12 and then filling both compartments with TrypLE. Patterning medium42 is intestinal medium lacking the p38 inhibitor nicotinamide and PGE2, but has 3 μM CHIR 99021 (Stemgent 04-0004-base) and 5 nM scTS2/16. Maturation medium42 is patterning medium lacking CHIR 99021 but having recombinant human IL22 (200-22 Peprotech) at a concentration of 2 ng ml−1. Wnt-Surrogate was present at a 30× lower final concentration (±16 pM). For immunofluoresence staining, to allow differentiation of organoids, the intestinal expansion medium in the bottom compartment was replaced for 14 days by ‘patterning' medium and successively by ‘maturation' medium42. The upper compartment was exposed to air during this period. Hereafter, the medium was removed, tissue 4% PFA-fixed for 20 min, washed twice with excess PBS and permeabilized by incubating in PBS–0.1% Triton X-100–2% BSA for 1 h. Finally, tissue was washed with PBS 2% BSA. For visualization of enterocytes, we stained for the marker ApoA1 by using rabbit antibody PA5-88109 (Thermo Fisher) (RRID: AB_2804657). Goblet cells were visualized by mouse anti-muc2 antibody 11197 (Abcam) (RRID: AB_297837); for enteroendocrine cells, rabbit antibody 23342-1-AP (Proteintech) detecting chromogranin A (RRID: AB_2879259); and for lysozyme secreted by Paneth cells, rabbit antibody GTX72913 (Gene Tex) (RRID: AB_374812). Alexa-conjugated secondary antibodies (1:500) used were donkey anti-mouse Alexa488 (1:500) Thermo Fisher A21202 (RRID: AB_141607) and donkey anti-rabbit Alexa 568 Thermo Fisher A10042 (RRID: AB_2534017). Actin was stained using Phalloidin-Atto 647 from Sigma. DNA was stained with DAPI D1306 (Thermo Fisher). All reagents were diluted in PBS–2% BSA. 42) organoids cultured in BME were first released from the matrix protein using dispase (10 U ml−1)38. The organoids were incubated with dispase at 37 °C for 30 min, after which they were washed twice with cold DMEM/F12 and collected by centrifugation (5 min, 4 °C, 300 × g). The washed organoids were then treated with TrypLE (without rho kinase inhibitor) at 37 °C for 5 min to dissociate the organoids into clusters of cells. To stop enzymatic activity, the cell suspension was washed again with cold DMEM/F12 and centrifuged (5 min, 4 °C, 300 g). The resulting cell pellet was resuspended in 1 ml of one of the following conditions: DMEM/F12, complete expansion medium (EM), EM + Rho kinase inhibitor Y-27632 or EM + Rho kinase inhibitor Y-27632 + scTS2/16. The cell suspension was herein incubated for 30 min (room temperature), allowing conditioning before embedding in hydrogel. After this incubation, cells were centrifuged (5 min, 4 °C, 300 × g) and cell pellets resuspended in 2 mg ml−1 collagen EZcol hydrogel. The suspension was plated in 10–20-µl domes which were then incubated at 37 °C for 1 h to allow polymerization. The next day, Y-27632 was removed from the medium, and cultures were maintained in EM with 5 nM scTS2/16 to analyze the effect of diffusion of scFv in collagen hydrogels. After 5 days of culture, the number of outgrown organoids was quantified using CellTiter-Glo viability assay (Promega). Ileum (N39) organoids were cultured in expansion medium consisting of DMEM/F12 supplemented with 100 U ml−1 P/S, 10 mM HEPES, 1× Glutamax, 2% B-27 with VitA, 3% (v/v) recombinant R-spondin-3, 2% (v/v) recombinant noggin, 1 mM N-acetylcysteine, 0.5 nM Wnt surrogate, 0.1 µg ml−1 hIGF-1 (Peprotech), 0.05 µg ml−1 hFGF2 (Peprotech), 10 nM gastrin, 0.5 µM A83-01, 0.05 µg ml−1 hEGF, 3 µM CHIR 99021 and 10 µM forskolin (Tocris)42. For collagen cultures, ileum organoids grown in BME were first released from the matrix by incubation with dispase (10 U ml−1 for 30 min at 37 °C). Hereafter, organoids were washed twice with cold DMEM/F12 and collected by centrifugation at 300 × g for 5 min at 4 °C. Organoids were then treated with TrypLE (no rho kinase inhibitor) at 37 °C for 5 min and further disrupted mechanically to dissociate organoids into small clumps of cells (avoiding complete single-cell dissociation). Organoids were washed again with cold DMEM/F12, centrifuged under the same conditions and resuspended in 2 mg ml−1 PureCol EZ Gel. The matrix–cell suspension was seeded in 10–20-µl droplets onto a culture plate, which was subsequently incubated at 37 °C for 1 h to allow polymerization. Hereafter, the droplets were covered with the expansion medium supplemented with 10 μM Y-27632 rho-kinase inhibitor (initial 12 h) and 5 nM scTS2/16. The following day, this medium was refreshed, with the expansion medium lacking rho kinase inhibitor but continuing with 5 nM scTS2/16. This medium was refreshed every 2–3 days. For passaging of organoids, the collagen domes were incubated with 1 mg ml−1 collagenase from Clostridium histolyticum (Sigma-Aldrich) for 30 min at 37 °C, and digestion monitored using bright-field microscopy (small clumps of cells). Organoids were collected in a 15-ml tube and washed twice (centrifugation for 5 min at 300 × g) with an excess volume of cold DMEM/F12. The clumps of cells were washed twice and resuspended in fresh PureCol EZ Gel (2 mg ml−1). Then, 7 days following passaging, the expansion medium was replaced with differentiation medium in all 3 hydrogel conditions: (1) collagen, (2) collagen + scTS2/16 and (3) BME. The differentiation (patterning) medium was prepared as described previously42. After 5 days, mature epithelial cells were observed in the collagen- and BME-cultured organoids. At this point, organoids in all hydrogel conditions were fixed to maintain consistency in imaging and downstream analyses. For quantitative assessment of differentiated secretory cells, we used flow cytometry (BD Fortessa). A total of 5,000 DAPI-negative single cells per hydrogel condition were analyzed to compare differentiation efficiencies to the expansion culture. Organoids were fixed in 4% formaldehyde (pH 7.4) and subsequently analyzed using SP8 confocal microscopy. The human small-intestinal duodenum line Duo-72 was described56. Jejunum J2 organoids were provided by M. Estes, Baylor College of Medicine. 5e, N = 4) were derived from the Utrecht Portal for Organoid Technology. Patients (University Medical Center Utrecht (UMCU)) provided informed consent for the generation of such organoids based on protocol UPORT Cancer Biobank 21-042 and was in accordance with the Declaration of Helsinki and Dutch law. This study complies with all relevant ethical regulations regarding research involving human participants. Primary liver human ductal organoids (Fig. 5f, N = 3) were derived from UMC biopsy codes 057932, 061530 and 061500. The study was approved by the UMCU ethical committee (TCBio 21-042) and was in accordance with the Declaration of Helsinki and Dutch law. This study complies with all relevant ethical regulations regarding research involving human participants. The ductal pancreas line, code HUB-08-A2-007, was obtained from the biobank of UMCU (Toetsingscommisie Biobanken TCBio (Biobank research Ethic Committee)) approved by protocol 12-093 HUB-Cancer and informed donor consent. Established human liver and pancreas ductal lines were described earlier59. All patients participating in this study signed informed consent forms approved by the responsible authority. In all cases, patients can withdraw their consent at any time, leading to the prompt disposal of their tissue and any derived material. Future distribution of organoids to an academic party will have to be authorized by the Medical Ethics Committee of the UMCU (TCBio). All effects measured between two groups, in response to agonistic (TS2/16) or antagonistic (AIIB2 or G0H3) antibodies, such as CellTiterGLO units—number of cells—or fold-change differences in gene expression levels in qPCR, were tested for significance using one-tailed t-tests. P values and the number and type of replicates (designated n) are provided in the relevant legends of the main figures, extended data figures and supplementary figures. Significance of results from bulk sequence comparisons (Extended Data Figs. 9 and 10) were analyzed using a Wald test for negative binomial distribution in the expression between scFv and control samples. Bonferroni correction was applied to adjust for multiple testing. The number of independent identical experiments is designated N in all legends. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Bulk RNA sequencing data are available at the European Genome–Phenome Archive under code EGAD50000001606. Other data supporting the findings of this study are available at the European Genome–Phenome Archive under code EGAS50000001113. 9 and 10 are available via GitHub at https://github.com/Hubrecht-Clevers/Integrin_B1_TS216. Source data are provided with this paper. Clevers, H. Modeling development and disease with organoids. Bissell, M. J., Hall, H. G. & Parry, G. How does the extracellular matrix direct gene expression?. Influence of a reconstituted basement membrane and its components on casein gene expression and secretion in mouse mammary epithelial cells. Single Lgr5 stem cells build crypt–villus structures in vitro without a mesenchymal niche. Targeting integrin pathways: mechanisms and advances in therapy. Zhang, Y. et al. Polyisocyanide hydrogels as a tunable platform for mammary gland organoid formation. Cruz-Acuna, R. et al. PEG-4MAL hydrogels for human organoid generation, culture, and in vivo delivery. Ye, S. et al. A chemically defined hydrogel for human liver organoid culture. Fully synthetic matrices for in vitro culture of primary human intestinal enteroids and endometrial organoids. Below, C. R. et al. A microenvironment-inspired synthetic three-dimensional model for pancreatic ductal adenocarcinoma organoids. Byron, A. et al. Anti-integrin monoclonal antibodies. Hemler, M. E. et al. Glycoproteins of 210,000 and 130,000 m.w. on activated T cells: cell distribution and antigenic relation to components on resting cells and T cell lines. Neugebauer, K. M. & Reichardt, L. F. Cell-surface regulation of β1-integrin activity on developing retinal neurons. Relating conformation to function in integrin α5β1. The α1/β1 and α6/β1 integrin heterodimers mediate cell attachment to distinct sites on laminin. Controlling epithelial polarity: a human enteroid model for host–pathogen interactions. Arimori, T., et al. Fv-clasp: an artificially designed small antibody fragment with improved production compatibility, stability, and crystallizability. Nishiuchi, R. et al. Ligand-binding specificities of laminin-binding integrins: a comprehensive survey of laminin–integrin interactions using recombinant α3β1, α6β1, α7β1 and α6β4 integrins. Sonnenberg, A. et al. Integrin recognition of different cell-binding fragments of laminin (P1, E3, E8) and evidence that α6β1 but not α6β4 functions as a major receptor for fragment E8. Expression and function of the cytoplasmic variants of the integrin alpha 6 subunit in transfected K562 cells. Activation-dependent adhesion and interaction with isoforms of laminin. A complex of platelet glycoproteins Ic and IIa identified by a rat monoclonal antibody. Regulation of integrin α5β1 conformational states and intrinsic affinities by metal ions and the ADMIDAS. Sato, T. et al. Paneth cells constitute the niche for Lgr5 stem cells in intestinal crypts. & Hemler, M. E. Monoclonal antibody 9EG7 defines a novel beta 1 integrin epitope induced by soluble ligand and manganese, but inhibited by calcium. Identification of a novel anti-integrin monoclonal antibody that recognises a ligand-induced binding site epitope on the beta 1 subunit. Maqueda, A. et al. Activation pathways of α4β1 integrin leading to distinct T-cell cytoskeleton reorganization, Rac1 regulation and Pyk2 phosphorylation. In vitro expansion of human gastric epithelial stem cells and their responses to bacterial infection. Boj, S. F. et al. Organoid models of human and mouse ductal pancreatic cancer. Long-term expansion of functional mouse and human hepatocytes as 3D organoids. Gelse, K., Poschl, E. & Aigner, T. Collagens—structure, function, and biosynthesis. The collagen suprafamily: from biosynthesis to advanced biomaterial development. & Dong, Y. Collagen-based biomaterials for tissue engineering. Fujii, M. et al. Human intestinal organoids maintain self-renewal capacity and cellular diversity in niche-inspired culture condition. Jee, J. H. et al. Development of collagen-based 3D matrix for gastrointestinal tract-derived organoid culture. Wang, Y. et al. Self-renewing monolayer of primary colonic or rectal epithelial cells. Towards a defined ECM and small molecule based monolayer culture system for the expansion of mouse and human intestinal stem cells. He, G. W. et al. Optimized human intestinal organoid model reveals interleukin-22-dependency of Paneth cell formation. Lee, R. T., Berditchevski, F., Cheng, G. C. & Hemler, M. E. Integrin-mediated collagen matrix reorganization by cultured human vascular smooth muscle cells. An in vitro study of scarring formation mediated by human Tenon fibroblasts: effect of Y-27632, a rho kinase inhibitor. Chapman, S., McDermott, D. H., Shen, K., Jang, M. K. & McBride, A. The effect of rho kinase inhibition on long-term keratinocyte proliferation is rapid and conditional. Mole, M. A. et al. Integrin β1 coordinates survival and morphogenesis of the embryonic lineage upon implantation and pluripotency transition. Broguiere, N. et al. Growth of epithelial organoids in a defined hydrogel. Wijnakker, J. et al. Integrin-activating Yersinia protein invasin sustains long-term expansion of primary epithelial cells as 2D organoid sheets. Yui, S. et al. Functional engraftment of colon epithelium expanded in vitro from a single adult Lgr5+ stem cell. Sugimoto, S., Kobayashi, E., Kanai, T. & Sato, T. In vivo intestinal research using organoid transplantation. Dekkers, J. F. et al. High-resolution 3D imaging of fixed and cleared organoids. Meyer, L. et al. A simplified workflow for monoclonal antibody sequencing. Fujii, M., Matano, M., Nanki, K. & Sato, T. Efficient genetic engineering of human intestinal organoids using electroporation. Andersson-Rolf, A. et al. One-step generation of conditional and reversible gene knockouts. Identification of novel human Wnt target genes using adult endodermal tissue-derived organoids. Saxena, K. et al. Human intestinal enteroids: a new model to study human rotavirus infection, host restriction, and pathophysiology. Establishment and culture of human intestinal organoids derived from adult stem cells. We thank R. van Ineveld for help with immunofluorescence tissue staining, T. Grenier for processing primary colon tissue and assistance with qPCR, and L. Huang and S. Lim for flow cytometry. We thank A. Brousali, J. Salij and O. Kranenburg of the Utrecht Platform for Organoid Technology (U-PORT; UMC Utrecht) for patient inclusion and tissue acquisition. This study was supported by the NWO Gravitation Project Material Driven Regeneration (024.003.013 ZWK MDR) funded by the Ministry of Education, Culture and Science of the government of the Netherlands (W.d.L., J.W., D.K., H.C.) and by the European Union's Horizon 2020 research and innovation program INTENS (668294)(H.C.) Present address: Synaffix/Lonza Company, Oss, The Netherlands Present address: Roche Pharmaceutical Research and Early Development, Basel, Switzerland These authors contributed equally: Wim B. M. de Lau, Joost J. Oncode Institute, Hubrecht Institute, Royal Netherlands Academy of Arts and Science (KNAW) and UMC, Utrecht, The Netherlands A. P. M. Wijnakker, Daniel Krueger, Daisong Wang, Matthijs S. Abendroth, Robin Schreurs, Johan H. van Es & Hans Clevers Oncode Institute, Princess Maxima Center of Pediatric Oncology, Utrecht, The Netherlands Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed most of the culture experiments. did most of the protein purifications with help from F.R. performed part of the confocal staining. performed production and purification of the single-chain antibodies in mammalian cells. performed part of the cell culture during the first revision, qPCR analysis and media preparation. designed the PureCol EZ Gel media and did functional testing during the second revision. D.W. gave advice on the differentiation protocols. provided scientific feedback throughout the project. and H.C. are inventors on a patent application (P095058WO) directly related to this study. H.C. is inventor of several patents related to organoid technology; his full disclosure is given at https://www.uu.nl/staff/JCClevers. The other authors declare no competing interests. Nature Biotechnology thanks Gil Garnier, Victor Hernandez and Kim Jensen for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. We consulted a series of tissue- and organoid-derived single cell RNA-seq databases to obtain a concise overview, specifically, of integrin β1 and selected integrin α-component expressions. Script that we used to compose this table is to be found at; https://github.com/Hubrecht-Clevers/Integrin_B1_TS216. A, N39 Ileum organoids, stably expressing the live cell cycle progression-reporter FUCCI, were cultured on BME-coated transwell inserts. Yellow signal arises as a mixed red/green signal, occurring at the transition from G1 to S-phase. The green signal, representing cycling cells, starts declining from 6 hrs on. Duo-72 Duodenal organoids were cultured in 3D hydrogel in regular expansion medium or supplemented with 10 nM AIIB2 ab for the indicated times. RNA, isolated for each condition, was converted to cDNA and subjected to qPCR using primer sets for enterocyte markers ANPEP, CYP2C9, and CYP2C19, or goblet cell marker TFF3. ΔCt (delta Ct) values were calculated for indicated target genes and reference gene GAPDH and beta-actin within the same sample. Their average and STDEV are indicated and based on 4 technical replicates (PCR reactions performed on mRNA of the same organoid sample). DNA sequences were cloned into the vector pFUSEss-CHIg-hG1 and pFUSE2ss-CLIg-hK. Yellow and green marked amino acids sequences represent the VHregion and the VL region of TS2/16, respectively. The V-domains are interconnected by (G4S)3 linker marked in grey. Note C-terminal 6xHIS tag (underlined) is included in the pET21 bacterial expression vector. A; The coding sequence contains the artificial human kappa light chain signal sequence and a C-terminal 8xHis tag, followed by the HRV 3 C protease cleavage site (LEVLFQ/GP), a Fc-tag (constant region of human IgG1) and a stop codon. B, the scTS2/16-Fc was captured from conditioned medium after which the Fc was enzymatically cleaved. Upon capturing this Fc-moiety, the remaining scTS2/16 component underwent final SEC-purification. C, the end result (N = 3) was evaluated on SDS-PAGE under reducing and non-reducing conditions. MW of relevant marker proteins indicated in kDa. A; Protein sequence integrin β1 ectodomain with designation numbering: Yellow characters for Plexin-Semaphorin-Integrin (PSI) domain. Green characters designate the A domain. The FLAG-tag is underlined, and the stop codon is indicated with an asterisk. Upper panel; protein-G agarose beads were incubated with huIgG1 version of antibody TS2/16 and several control integrin β1-specific antibodies, as indicated. An integrin β4 subunit -specific antibody was included as a negative control. After washing, beads were incubated with FLAG-tagged recombinant integrin β1 ectodomain (110 kDa). Beads were again washed and finally incubated with reducing SDS sample buffer. SDS-PAGE gel was blotted and capture of the β1-ectodomain tested using HRP-linked anti-Flag M2 mAb (Sigma). Results were visualized using Enhanced chemiluminescence (ECL). Lower panels; Beads loaded with M2 anti-Flag mAb were incubated with Flag-tagged β1 ectodomain (in PBS + 2% BSA) (lane 2 + 3) or just PBS-2% BSA (lane1). After washing, beads were incubated with HIS-tagged scTS2/16 antibody (lane 1 + 3) or PBS–2% BSA (lane2). Beads were washed and incubated with reducing SDS sample buffer. SDS_PAGE gel was blotted and stained directly with an HRP-linked anti HIS-ab (lower right panel), or indirectly using mouse M2 anti-Flag antibody followed by anti-mouse HRP-conjugate (loer right panel). Results were in each case visualized using Enhanced chemiluminescence (ECL). The left margins indicate protein sizes in kDa. Plates were then blocked by incubating for 30 min. Colon organoids were enzymatically processed to single cells and resuspended in DMEM-F12 medium. For each coating condition the inhibitory AIIB2 ab (30 nM) was also included, to test the relative involvement of integrin ß1. Upon incubation for 1,5 hrs (370C), wells were extensively washed with DMEMF12 medium and remaining cells quantified by CellTiter-GLO, an ATP-driven luminescence assay (Relative Light Units – RLU.106). Data presented as mean + /- STDEV (n = 3). Organoids derived from the duodenum, jejunum, ileum, and colon were cultured in expansion medium with or without scTS2/16. PolyA RNA was converted to cDNA and analysed using bulk sequencing. A; volcano plot showing log2fold changes and p-value (using a Wald test for negative binomial distribution) in expression between scFv and control samples. Using Bonferroni correction to adjust p-values for multiple testing showed no significant difference (padj < 0.01) between scFv and control samples. B; a heatmap showing the 30 most up and down regulated genes based on log2fold change expression of the same samples. Rows were clustered using hierarchical clustering based on Euclidian distance. C; A sample distance matrix showing Euclidian distance, comparing tissues under control- and scTS2/16 conditions, visualizes their close genetic identity D; Principal Component Analysis (PCA) plot indicating the virtually identical transcriptomic make-up of control and scTS2/16 pairs within the four tissues relative to inter-tissue variance. A sample distance matrix showing Euclidian distance, comparing tissues under control- and scTS2/16 conditions, visualizes their close genetic identity. Columns and rows are clustered using hierarchical clustering. E; heat map showing DESEQ2 normalised counts of cell type-specific markers for colon grown in control- and scFv-supplemented media. Rows were clustered using hierarchical clustering based on Euclidian distance. F; similar analysis for small-intestine as shown in E. Organoids from duodenum and colon were cultured on BME or PureCol EZgel, in expansion medium, +/- scTS2/16. PolyA RNA, purified from these lines, was converted to cDNA and subjected to bulk-sequencing. A; volcano plot showing log2fold changes in expression and pvalue differences (using a Wald test for negative binomial distribution) between the averages of scFv and control samples. Using Bonferroni correction to adjust p-values for multiple testing showed no significant difference (padj < 0.01) between scFv and control samples. B; a heatmap showing the 30 most up- and down regulated genes based on log2fold change expression of the same samples. Rows were clustered using hierarchical clustering based on Euclidian distance. C; A sample distance matrix showing Euclidian distance, comparing tissues under control- and scTS2/16 conditions, visualizes their close genetic identity. D; Principal Component Analysis (PCA) plot indicating the virtually identical transcriptomes of control and scTS2/16 pairs within the three tissues relative to inter-tissue variance. Columns and rows are clustered using hierarchical clustering. E; heat maps showing DESEQ2 normalised counts of cell type-specific markers for colon grown in control-, and scTS2/16 supplemented media. Rows were clustered using hierarchical clustering based on Euclidian distance. F; similar analysis for pancreas as shown in E. G; similar analysis for small intestine as shown in E. Statistical source data for Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. A single-chain derivative of an integrin-activating antibody potentiates organoid growth in Matrigel and collagen hydrogels. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64566-w'>Altered milk tryptophan and tryptophan metabolites in women living with HIV</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 10:05:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Children born to women living with HIV (WLWH) suffer increased morbidity and, in low-income settings, have two to three times the mortality of infants born to women without HIV. The basis for this increase remains elusive. In low-income settings, breastfeeding is recommended because health benefits outweigh the risk of transmission, especially when maternal antiretroviral therapy is provided. We profile the milk metabolome of 326 women with and without HIV sampled longitudinally for 18 months postpartum using global metabolomics. We identify perturbations in several metabolites, including tryptophan, dimethylarginine, and a recently discovered antiviral ribonucleotide, that are robustly associated with maternal HIV infection. Quantitative tryptophan and kynurenine levels in both milk and plasma reveal that these perturbations reflect systemic depletion of tryptophan and alterations in tryptophan catabolism in WLWH. Finally, we validate these signatures of maternal HIV infection in an independent cohort of healthier WLWH. Taken together, our findings demonstrate that milk tryptophan content and availability decrease among WLWH, which may indicate perturbations in milk tryptophan catabolism. The link between this perturbation and the increased morbidity and mortality of children born to WLWH merits further investigation. Children born to women living with HIV (WLWH) experience significantly increased morbidity and mortality, even when they themselves are not infected (HIV-exposed, or CHEU). While CHEU in high-income countries are more likely to be hospitalized1,2,3, those in low-income countries have mortality rates two to three times higher than infants born to women without HIV (WWoH)4,5,6,7. Most of this morbidity and mortality is due to infections, particularly pneumonia, diarrhea, and meningitis7,8. Additionally, CHEU faces challenges related to growth and cognitive development7. Although maternal antiretroviral therapy has improved CHEU health outcomes, they still experience significant risks, the basis of which remains unclear. HIV infection, even with optimal viral suppression by antiretroviral therapy, is associated with chronic inflammation characterized by persistent macrophage activation, elevated levels of interferon-inducible cytokines, and intestinal barrier dysfunction9. This systemic inflammation also affects pregnant WLWH and is reflected in their CHEU infants10,11,12,13,14. Similar to their mothers, CHEU infants display perturbed biomarkers of immune monocyte activation, including elevated interferon-inducible cytokines, which impair immune function and germinal center development throughout infancy15,16,17,18. Such changes are not unique to HIV as maternal immune activation due to maternal influenza and other infections is associated with the development of neurodevelopmental disorders19,20,21,22,23,24 as well as the development of tissue-specific immunity and inflammation25,26,27 in their offspring. HIV-associated immune dysregulation is largely driven by metabolic alterations, particularly in the indoleamine 2,3-dioxygenase (IDO) pathway and accelerated tryptophan catabolism, both of which impair immune function28,29. Tryptophan, an essential amino acid obtained solely from the diet, is primarily catabolized via the kynurenine pathway, with its metabolites playing key roles in immune regulation, neuronal function, and intestinal homeostasis30,31,32. A key marker of IDO activation is the kynurenine-to-tryptophan (KT) ratio33, which is elevated in HIV infection and associated with disease progression, dementia, and mortality34. Although the KT ratio decreases with viral suppression, it does not normalize completely35,36. High KT ratios are linked to microbial dysbiosis, TH17 cell loss, and an imbalance between TH17 and Treg cells in the gastrointestinal mucosa29,37. Additionally, elevated KT ratios have been associated with stunting in children38,39, suggesting that disrupted tryptophan metabolism may have broader health impacts. Breast milk is essential for optimal growth and development, particularly for infants born to WLWH in low- and middle-income countries. The dietary source of tryptophan for nursing infants is human milk. There are very few reports describing levels of tryptophan in milk, it appears that tryptophan, free and bound, are higher in colostrum and then decline with time40,41,42,43,44. Utilizing longitudinal samples from a cohort of WLWH and WWoH prior to the routine use of antiretroviral therapy, we characterize the metabolomic profile of breast milk over the first 18 months postpartum. We identify a metabolomic signature of milk in WLWH that demonstrates decreased levels of tryptophan throughout lactation. This depletion of tryptophan is associated with elevations of the type I interferon response28. Using targeted assays for tryptophan and kynurenine in both milk and plasma, we demonstrate that quantitative levels of tryptophan and kynurenine are systemically altered in WLWH. Furthermore, in later experiments we identify an initially unknown compound, which is highly elevated in the milk of WLWH, as 3'-deoxy-3'4'-didehydro-cytidine (ddhC), the free base of an interferon-inducible innate antiviral ribonucleotide45. Lastly, we confirm this milk metabolomic signature of HIV in a second, healthier cohort of WLWH on antiretroviral therapy from Haiti (see study overview, Fig. a Large, global metabolomic study of milk of WLWH versus WWoH. Open circles denote the number of samples at each study visit included in the final analysis. The number of participants in each group is indicated in parentheses on the left, and the number of samples is indicated on the right. b Differentially abundant metabolites in the milk WLWH vs WWoH, shown in rank order. c Targeted quantification of tryptophan and kynurenine levels in maternal milk and plasma. d Identification of the unknown compound X-12127 as 3'-deoxy-3',4'-didehydro-cytidine (ddhC). e Validation of milk metabolomic findings in an independent cohort of WLWH from the Haiti study. Points show the timing of sample collections, and boxplots indicate the median and interquartile ranges. To investigate the milk metabolome of WLWH, global metabolomics was performed on 1599 whole human milk samples collected as part of a randomized clinical trial46. The Zambia Exclusive Breastfeeding Study (ZEBS) was conducted in Lusaka, Zambia, between 2001 and 2008 and investigated the feeding strategy for WLWH recommended by the World Health Organization at that time. This trial was conducted prior to the routine use of antiretroviral therapy, with most women receiving single-dose nevirapine for prevention of vertical transmission per standard of care. Women/infant pairs were followed for 24 months postpartum. Longitudinal breast milk samples spanning the first 24 months of lactation were selected from 38 WWoH and a stratified sample of 288 WLWH enrolled in the trial. The clinical characteristics of the study participants are shown in Table 1. Thirty-six (12.5%) of WLWH died during the course of the study compared to no WWoH (p < 0.05). WLWH had lower CD4 T cell counts (315 vs. 840 cells/mm3) and higher CD8 T cell counts (759 vs 563 cells/mm3) than WWoH (p < 0.001 and p = 0.001, respectively). Children of WWoH weighed more at birth (3.3 vs 2.9 kg; p < 0.01) and 3 of these children died during the study. All but 2 of the 1599 whole breast milk samples were successfully characterized and included 765 named and 74 unnamed metabolites for a total of 839 metabolites. Of the 765 named metabolites, 20 metabolites classified as “drug-antibiotic” or “drug-antiviral” were removed from further analyses, leaving a total of 819 metabolites included in the analyses. Samples were excluded from this analysis if there were insufficient samples at that timepoint for meaningful interpretation (n = 72) or if the milk was collected post-weaning (n = 99), leaving 1,426 samples spanning 18 months of lactation in the final analysis (Fig. Overall milk metabolomic profiles were strongly influenced by study visit (4.0% variance, p < 0.001), and by HIV infection (0.19% variance, p < 0.001), maternal CD4 count (0.14% variance, p < 0.001), maternal BMI (0.12% variance, p < 0.001), and infant sex (0.07% variance, p = 0.009) (Supplementary Fig. Linear mixed effects regression identified 103 metabolites with significantly altered levels in WLWH versus WWoH when all study visits were considered in aggregate (Supplementary Data 1). Post hoc comparisons stratified by study visit identified a core set of altered metabolites (Fig. 3; Supplementary Data 2), including tryptophan, dimethylarginine, cytosine, and an uncharacterized compound, X-12127. Levels of tryptophan, dimethylarginine, cytosine, and X-12127 were significantly and consistently altered in the milk of WLWH across the entire study time course (p < 0.001; Fig. a Volcano plot showing differences in metabolite abundances at the 1-month timepoint. Blue and red points metabolites are significantly decreased and increased in WLWH versus WWoH, respectively. b Normalized abundances of selected metabolites across the study course. Solid lines indicate mean abundances, and shaded areas denote 95% confidence intervals. Red asterisks along the top denote study visits at which the selected compound was differentially abundant in WLWH versus WWoH. Black asterisks along the bottom denote study visits at which the compound was selected as a predictive feature in the random forests modeling. All metabolites that were significantly differentially abundant at the 1-month timepoint are shown. As expected, tryptophan, cytosine and X-12127 were strongly correlated with baseline maternal CD4 count, baseline maternal plasma viral load, and contemporaneous breast milk viral load (Supplementary Fig. N-acetylalanine (p = 0.006) and N-acetylserine (p = 0.007) were elevated in WLWH with greater separation at the earlier timepoints, 1 and 4 months. In contrast, the linoleic acid metabolites 9,10-DiHOME and 12,13-DiHOME were significantly elevated in WLWH at 1 month (p < 0.05), but not over the entire study time course. X-07765, an unknown metabolite significantly elevated at the one-month timepoint in the linear regression model (p = 0.003), did not appear to be as robust a marker as the other metabolites and was not significant over the entire time course (p = NS; Fig. Other features, including many lipid and amino acid metabolites, were found to be altered later in lactation (Supplementary Fig. 3 and Supplementary Data 4) and may reflect the far-reaching impact of HIV infection on milk composition. In an orthogonal analysis, random forests models for maternal HIV status also identified many of these same features as significant predictors of maternal HIV (Fig. Finally, sub-analyses of WLWH whose children remained uninfected during the course of the study (38 WWoH versus 154 WLWH with CHEU) and those whose children survived through the entire study course (35 WWoH versus 143 WLWH) confirmed the metabolic signature of maternal HIV infection to be independent of child outcome (Supplementary Fig. Taken together, these results point to a robust and sustained effect of HIV infection on the composition of metabolically relevant compounds in human milk. The striking separation in normalized abundances of tryptophan and X-12127 across the study course (Fig. 2B) led to further investigations of these two metabolites. Of note, although tryptophan abundance declined slightly between the 1-week and 1-month milk samples in WWoH, there was a notable dip in tryptophan abundance in the WLWH, followed by some recovery and continued significant separation of tryptophan throughout the study course. The normalized abundance of X-12127 rose sharply between 1 week and 1 month and remained elevated, a pattern that was mirrored by cytosine. It was also noted that dimethylarginine, N-acetylalanine, and N-acetylserine all had similarly shaped normalized abundance curves. The significant depletion of tryptophan across the study duration was mirrored by an increase in levels of X-12100 (tentatively identified as a hydroxylated tryptophan compound) as well as a consistent, albeit smaller, increase in levels of kynurenine (Supplementary Fig. Although none of the comparisons in the analysis stratified by visit demonstrated significant differences after adjustment for multiple comparisons (Supplementary Data 2), kynurenine was increased in the mean effects model (p = 0.007, Supplementary Data 1). Moreover, the kynurenine to tryptophan (KT) ratio, a marker of IDO activity, was significantly elevated at all study visits (Fig. a Boxplot of kynurenine/tryptophan log-ratio in WLWH versus WWoH stratified by study visit, as calculated from the global metabolomics panel (n = 1426; linear regression adjusted p < 0.001 all timepoints). b Scatterplots of global versus quantitative tryptophan and kynurenine abundances and KT log-ratio at the 4-month timepoint (n = 98). Solid blue line indicates linear regression line and shaded gray area represents 95% confidence intervals. Spearman correlation coefficients and p-values are shown in parentheses in each subtitle. c Tryptophan, kynurenine, and KT log-ratio values in WLWH versus WWoH at the 4-month study visit, as calculated from the quantitative panel. Box plots show the center line as the median, whiskers show the maxima and minima, and box limits show interquartile ranges. To confirm the findings of the elevated KT ratio from the global metabolomics panel, a quantitative KT panel was performed on 98 milk samples from 21 WWoH and 77 WLWH at the 4-month timepoint (Supplementary Table 3). The 4-month timepoint was selected because paired milk and plasma samples were available from almost all participants at this timepoint due to the trial design. Quantitative and global KT values were highly correlated (Fig. 3B, rho = 0.9132 and 0.8792 for tryptophan and kynurenine, respectively; both p < 2.2e−16), indicating that the global panel could be used for accurate estimation of the quantitative KT-ratio. Milk tryptophan levels were significantly lower, 0.367 ug/mL versus 0.572 ug/mL (p < 0.001), and kynurenine levels were significantly higher, 0.177 ug/mL versus 0.125 ug/mL (p = 0.028) in WLWH versus WWoH (Fig. Correspondingly, the KT log-ratio was significantly higher in WLWH as well (p < 0.001, Fig. The lower levels of tryptophan in the milk of WLWH could reflect lower plasma levels of tryptophan in WLWH, decreased transfer of tryptophan to the milk of WLWH, or altered tryptophan metabolism in the milk of WLWH. Given that tryptophan transport across the gut can be perturbed by infection47, we sought to determine whether transport across the breast epithelium was similarly affected. To distinguish between these possibilities, quantitative plasma and milk tryptophan and kynurenine levels were compared from 118 women (31 WWoH and 87 WLWH, Supplementary Fig. As expected, milk and plasma levels were highly correlated (rho = 0.50, p < 0.001 for kynurenine; rho = 0.66, p < 0.001 for tryptophan). Plasma tryptophan levels were significantly lower, 5.03 ug/mL versus 8.13 ug/mL (p < 0.001), and kynurenine levels were significantly higher, 0.468 ug/mL versus 0.369 ug/mL (p < 0.001) in WLWH versus WWoH (Supplementary Fig. To further investigate the transfer of tryptophan or kynurenine to milk, we compared the quantitative levels in 92 women (21 WWoH and 71 WLWH, Supplementary Fig. 8 and Supplementary Table 3) who had both plasma and breast milk samples collected at the 4-month timepoint (Supplementary Fig. Milk levels of tryptophan were 6.8% and 8.3% of the plasma levels for WWoH and WLWH, respectively; this proportion did not differ significantly by maternal HIV infection (p = 0.14, Supplementary Fig. A similar trend was observed for kynurenine with breast milk levels at 32.4% and 36.6% of the corresponding plasma levels for WWoH and WLWH, respectively (p = 0.42, Supplementary Fig. Altogether, these results suggest that the decreased levels of tryptophan and increased levels of kynurenine observed in the milk of WLWH reflect plasma levels as opposed to selective transfer to the milk or altered local metabolism. Levels of an unknown compound, initially labeled X-12127, were significantly elevated in WLWH compared to WWoH at all study visits except immediately postpartum (Fig. Given the robustness of this marker, we sought to better characterize this compound by mass spectrometry. The compound, with an m/z value of 226.0824 in positive ionization mode, was analyzed using tandem MS/MS, revealing a cytosine fragment and two water loss events, suggesting the presence of two OH groups and a composition similar to 2'-deoxycytidine (dC) minus two hydrogen atoms (Fig. Structural similarity exploration and predicted fragmentation indicated 3'-deoxy-3',4'-didehydro-cytidine (ddhC) as a likely candidate. This was confirmed through standard acquisition and extensive comparison of fragmentation patterns at different collision energies, leading to the definitive identification of the compound as ddhC (Fig. a Comparison of retention time (RT) for X-12127 and ddhC standard. MS/MS spectra of X-12127 and ddhC standard obtained on an Agilent 6546 LC/Q-TOF MS system in the positive targeted MS/MS scan mode by varying the collision energy (CE) at 5 ev (b), 20 ev (c), and 40 ev (d). e Chemical structure of 3'-deoxy-3',4'-didehydro-cytidine (ddhC). To assess the generalizability of these findings, global metabolomics profiling was performed on the milk from a previously described cohort of 22 WLWH and 25 WWoH from Haiti (Table 1)48. Notably, these women were on antiretroviral therapy with a mean CD4 count of 541 cells/mm3, and milk was sampled at a single visit during the first 6 months of lactation (median 45 days; Fig. Regression estimates of the associations between HIV status and metabolites were obtained while including infant age in days as a covariate given the non-uniformity of sampling. Nevertheless, estimates from the Haiti cohort were strongly correlated to those from the main cohort at the 1 month and 4 month visits (r = 0.126 and 0.386, p < 0.001 for all metabolites, respectively; r = 0.812 and 0.789, p < 0.001 when only the metabolites observed to be significantly associated with HIV status in the ZEBS cohort were included; Supplementary Fig. Three metabolites were significantly increased in the milk of WLWH, cytosine (p < 0.001), methionine sulfone (p = 0.090) and a different unknown metabolite, X-12193 (p = 0.054) (Supplementary Fig. Tryptophan trended lower and kynurenine trended higher in the milk of WLWH in the Haiti cohort as observed in the main cohort, although neither reached statistical significance. Notably, the KT log-ratio was significantly higher in the milk of WLWH (unadjusted p = 0.05). Approximately 1.3 million children are born to WLWH annually, and despite maternal antiretroviral therapy, these children continue to experience differences in immune function, growth, and cognition. In this study, we find that the milk of WLWH is characterized by decreased levels of tryptophan and perturbations of tryptophan catabolism via the kynurenine pathway. These interferon-inducible metabolic alterations may serve as a common denominator to explain some of the increased morbidity and mortality experienced by CHEU. Normal immune function depends on the regulation of key components of innate and adaptive immunity in lymphoid tissues. Interplay between antigen-presenting cells, such as macrophages and dendritic cells, is critical to host defense by maintaining mucosal barriers and attenuating inflammation. Immune dysregulation mediated by cellular immunity is associated with alterations in indoleamine 2,3-dioxygenase and tryptophan catabolism in individuals with HIV29. Chronic viral infections are also known to alter tryptophan catabolism31. Recent studies of post-acute sequelae of SARS-CoV-2 infection (PASC) demonstrate that individuals with long COVID have decreased circulating serotonin levels secondary to viral RNA-induced type I interferons, reducing tryptophan uptake through the gut epithelium47. Additionally, a tryptophan-deficient diet is sufficient to lead to circulating tryptophan and serotonin deficiency in a mouse model47. Tryptophan starvation appears to be part of the innate immune response to bacterial and parasitic infections49. Tryptophan catabolism, much of which is carried out by intestinal microbes, serves an essential role in physiological homeostasis and immune regulation and can rapidly be adapted in the stress response32. HIV infection may stimulate IDO-1 activity through virally associated increases in type I and II interferons and microbial products. Multiple tryptophan metabolites are endogenous ligands of the aryl hydrocarbon receptor (AHR), which serves as an environmental sensor that integrates immune responses50. Thus, deficiency or perturbation of tryptophan catabolites can lead to alterations of the gut microbiota and immunity by alterations in the IDO1-AHR axis51. Importantly, T helper cell subsets, including TH17 and regulatory T cells, express AHR, so alterations in these pathways may alter immune homeostasis and immune development in children of women living with HIV50. In the context of growth, tryptophan serves as a limiting amino acid for protein metabolism. Additionally, alterations in tryptophan catabolism have been associated with stunting38,39. Cognitive effects of tryptophan depletion are thought to be due in part to decreased availability of serotonin, as serotonin is critical for neuronal differentiation, migration, and synapse formation during embryonic development51. Altered tryptophan catabolism may also explain some of the neurocognitive deficits in children of WLWH by leading to increases in neurotoxic metabolites, such as quinolinic acid, and decreases in neuroprotective metabolites, such as kynurenic acid30. Kynurenine and quinolinic acid are elevated in the cerebrospinal fluid of people with HIV and correlate with the severity of HIV-associated neurocognitive disorder31,52. Type I interferons also lead to the production of 3'-deoxy-3',4'-didehydro-cytidine triphosphate (ddhCTP), an innate antiviral compound produced by the interferon-inducible protein viperin (virus inhibitory protein, endoplasmic reticulum-associated, interferon-inducible)45. ddhC, which is highly elevated in the milk of WLWH, is the free base of this naturally occurring antiviral ribonucleotide and is also being investigated as a biomarker for acute viral infection53,54. ddhCTP has direct antiviral properties and acts as a chain terminator for the RNA-dependent RNA polymerases from multiple members of the Flavivirus genus45, Zika Virus, and likely SARS-CoV-254,55,56. Viperin is a multifunctional protein and product of an ancient gene that is part of the radical S-adenosyl-L-methionine (SAM) superfamily of enzymes56. The metabolic pathways, products, and serum cytokine associations of viperin are being elucidated54. Viperin is co-transcribed with cytidylate monophosphate kinase 2 (CMPK2) during interferon stimulation. It has been proposed that CMPK2 primarily functions to produce enough substrate for the viperin-mediated production of ddhCTP, which catalyzes the conversion of cytidine triphosphate to ddhCTP. Cytosine was also persistently elevated in the milk of WLWH. Viral infection has been associated with alterations in host cytosine metabolism58 and the production of ddhCTP59. While ddhCTP is a viral chain terminator, host RNA and DNA polymerases are not affected by ddhCTP45. We found cytosine elevations in both our primary cohort and our validation cohort. We hypothesize that the elevated levels of cytosine in the milk of WLWH reflect the breakdown of ddhCTP. The elevation of dimethylarginine alone and with the N-acetylated amino acids (N-acetylalanine and N-acetylserine) in our cohort is consistent with oxidative stress and increased protein catabolism. Our study does not discriminate between symmetric versus asymmetric dimethylarginine. There is emerging data on symmetric arginine methyltransferases potentially playing an important role in immune cell development and homeostasis60. Alterations in T cell development may underpin the immunologic abnormalities seen in children of WLWH. These findings raise the question of whether replacement of tryptophan in the diet of children of WLWH, if low tryptophan is found in investigations of CHEU, will lead to resolution of the immune, cognitive, and growth perturbations experienced by this population. It is not known if the low milk tryptophan levels are sufficient, how well the tryptophan is absorbed, or the proportion of bound tryptophan in the milk. Systemic inflammation has been described in CHEU infants, which may alter tryptophan catabolism, leading to increases in neurotoxic metabolites. These possibilities deserve careful study. Breastfeeding remains the recommended form of infant feeding because health benefits outweigh the risk of HIV transmission. With effective maternal antiretroviral therapy, HIV transmission risk has decreased to the point where shared decision-making about breastfeeding entered US guidelines in 2023. The decreased levels of milk tryptophan in the setting of chronic HIV infection are likely due to downregulation of tryptophan transporters in the maternal gut, resulting in lower tryptophan plasma levels47. Whether the pro-inflammatory milieu of milk similarly affects infant tryptophan absorption is unknown. Prudence is also demanded as inflammatory imprinting from the milk of WLWH might drive tryptophan catabolism down the IDO pathway and lead to subsequent buildup of neurotoxic tryptophan metabolites in the developing infant. That CHEU infants display perturbed biomarkers of monocyte activation, including elevations in interferon-inducible cytokines, is supportive of this possibility15,16,17,18. Careful study of tryptophan replacement in the setting of chronic viral inflammation in animal models will be necessary to ensure that replacement will result in beneficial effects. If replacement alone is insufficient, then replacement with agents to modulate the kynurenine pathway, such as IDO-inhibitors, which are in development, may be necessary31. The primary strength of this investigation is the longitudinal profiling of almost 1600 milk samples from a deeply characterized cohort from a randomized clinical trial. The dense and consistent sampling over the first 18 months postpartum allows us to identify both temporally specific and persistent effects of maternal HIV infection. Much of the present study focuses on the first 4 months postpartum, which represents a critical window for infant development as well as the primary period of exclusive breastfeeding, and we also observed robust longitudinal perturbations in the milk metabolome. That this large study utilizes samples taken in the pre-antiretroviral therapy era is both a blessing and limitation. It is often difficult to deconvolute the profound metabolic signatures of different antiretroviral agents and HIV infection itself, which makes analysis of pure disease signatures more complex in the setting of antiretroviral therapy. However, the strength and robustness of the signatures for HIV infection observed in the present study are likely to some extent due to the severity of disease progression in the absence of effective antiretroviral therapy; maternal CD4 counts, viral loads, and tragically, mortality are all abundant evidence to this point. Growing evidence suggests that antiretroviral therapy reduces but does not fully ameliorate the chronic inflammation associated with HIV infection. Neither, unfortunately, does maternal antiretroviral therapy fully normalize the adverse health outcomes of CHEU. Consistent with these points, the metabolomic signature in the milk of WLWH is partially observed in the validation cohort of women on antiretroviral therapy with higher CD4 counts. Finally, the present study only includes samples from WLWH and WWoH with no measurements in the infants. Consequently, the effect on the infants has to be inferred as it was not directly measured. Replication of these findings in a large cohort of WLWH and their infants in contemporary settings with universal maternal antiretroviral therapy is necessary to confirm the hypothesized mechanisms on the health outcomes of CHEU. We identified a robust and persistent metabolomic signature in the milk of WLWH comprising the kynurenine/tryptophan axis as well as dimethylarginine, cytosine, and ddhC, the free base of a recently discovered antiviral molecule, ddhCTP, that spans the first 18 months of lactation. This signature is consistent with a model in which chronic viral inflammation associated with maternal HIV infection leads to induction of the type I interferon response. Virally induced elevation of type I interferons decreases transport of tryptophan across the gastrointestinal wall, thereby leading to systemic depletion of tryptophan and subsequent reduction of tryptophan levels in the milk of WLWH. Since human milk is the only source of nutrition for exclusively breastfed infants, maternal tryptophan deficiency and chronic immune activation during gestation and lactation may drive subsequent serotonin deficiency and altered tryptophan catabolism in children of women living with HIV. If alterations in tryptophan and tryptophan catabolism are directly confirmed in infants, this may provide opportunities for targeted interventions to ameliorate the substantial global burden of immunologic, neurocognitive, and growth abnormalities observed in CHEU infants. The Zambia Exclusive Breastfeeding Study (ZEBS)46 conducted in Lusaka, Zambia between 2001 and 2008 (clinicaltrials.gov ID NCT00310726) investigated a feeding strategy for WLWH recommended by the World Health Organization at that time. The strategy, intended to decrease infant infection with HIV during an era prior to the widespread availability of antiretroviral medications, was to rapidly wean infants from breastfeeding after 4 months of lactation. In ZEBS, WLWH were randomized to either breastfeed until usual breastfeeding cessation or to rapidly wean at 4 months. Almost all women received only a single dose of nevirapine for the prevention of vertical transmission, as was the standard of care at the time. Mother/infant pairs were followed for 24 months postpartum. From this parent trial, a sub-study of WLWH and WWoH was selected based on sample availability as described below. The clinical characteristics of the women included in this sub-study are given in Table 1. Whole breast milk and plasma samples were stored at −80C as previously published61. Breast milk sample selection for the global metabolomics panel. Longitudinal breast milk samples spanning the first 24 months of lactation from women living without (N = 38) and with HIV (N = 288) were selected on the basis of sample availability. Samples were selected from all 38 women living without HIV (WWoH) included in the trial, 35 of their infants survived (279 samples), and 3 infants died (9 samples). A stratified sample of WLWH was selected for inclusion by known infant outcomes for this metabolomics study. Specifically, the samples selected for study inclusion were as follows: A random sample of 76 (out of 622) WLWH with infants who did not experience HIV infection and who survived (N = 76; 396 samples) All WLWH whose infants did not experience HIV infection but died in the first 24 months of life (N = 78; 193 samples) All WLWH whose infants tested positive for HIV between 1 week and 1 month of life (N = 53; 221 samples) or after 1 month of life (N = 81; 401 samples). To account for the parent trial study intervention, only milk samples from women who reported they were actively still breastfeeding were included in the final analysis. Breast milk (n = 101) and plasma samples (n = 125) were run for quantitative KT analysis. A random subset of 21 women without HIV and 80 women living with HIV, 20 each from the groups above, with paired maternal plasma and milk samples at the 4-month timepoint, were selected for the comparison. The 4-month timepoint was selected because this was the first plasma collection during lactation. Milk samples from WWoH (n = 25) and WLWH (n = 22) from a previously described cross-sectional study were used as an independent validation cohort48. In contrast to the main study cohort, these women were primarily on antiretroviral therapy and had relatively minor disease progression, most with CD4 counts above 350. Global metabolomics was performed on 1599 whole human milk samples collected longitudinally using ultra high-performance liquid chromatography/tandem mass spectrometry by Metabolon Inc. according to published methods62,63,64. Samples were prepared using the automated MicroLab STAR® system from Hamilton Company. Several recovery standards were added prior to the first step in the extraction process for QC purposes. To remove protein, dissociate small molecules bound to protein or trapped in the precipitated protein matrix, and to recover chemically diverse metabolites, proteins were precipitated with methanol under vigorous shaking for 2 min (Glen Mills GenoGrinder 2000) followed by centrifugation. The resulting extract was divided into multiple fractions: two for analysis by two separate reverse phase (RP)/UPLC-MS/MS methods with positive ion mode electrospray ionization (ESI), one for analysis by RP/UPLC-MS/MS with negative ion mode ESI, one for analysis by HILIC/UPLC-MS/MS with negative ion mode ESI, while the remaining fractions were reserved for backup. Samples were placed briefly on a TurboVap® (Zymark) to remove the organic solvent. The sample extracts were stored overnight under nitrogen before preparation for analysis. QA/QC: Several types of controls were analyzed in concert with the experimental samples: a pooled client matrix (CMTRX) sample generated by taking a small volume of each experimental sample served as a technical replicate throughout the data set; extracted water samples served as process blanks; and a cocktail of QC standards that were carefully chosen not to interfere with the measurement of endogenous compounds were spiked into every analyzed sample, allowed instrument performance monitoring and aided chromatographic alignment. Four technical replicates (n = 4) of either client matrix (CMTRX) or QC samples are run for every 36 study samples in the global metabolomics panel. For the ZEBS Cohort 1599, human milk samples (n = 1) were analyzed with an additional 180 QC samples. All methods utilized a Waters ACQUITY ultra-performance liquid chromatography (UPLC) and a Thermo Scientific Q-Exactive high resolution/accurate mass spectrometer interfaced with a heated electrospray ionization (HESI-II) source and Orbitrap mass analyzer operated at 35,000 mass resolution63. The dried sample extracts were then reconstituted in solvents compatible to each of the four methods. Each reconstitution solvent contained a series of standards at fixed concentrations to ensure injection and chromatographic consistency. One aliquot was analyzed using acidic positive ion conditions, chromatographically optimized for more hydrophilic compounds (PosEarly). In this method, the extract was gradient eluted from a C18 column (Waters UPLC BEH C18-2.1 × 100 mm, 1.7 µm) using water and methanol, containing 0.05% perfluoropentanoic acid (PFPA) and 0.1% formic acid (FA). Another aliquot was also analyzed using acidic positive ion conditions; however, it was chromatographically optimized for more hydrophobic compounds (PosLate). In this method, the extract was gradient eluted from the same aforementioned C18 column using methanol, acetonitrile, water, 0.05% PFPA, and 0.01% FA, and was operated at an overall higher organic content. Another aliquot was analyzed using basic negative ion optimized conditions using a separate dedicated C18 column (Neg). The basic extracts were gradient eluted from the column using methanol and water, however, with 6.5 mM Ammonium Bicarbonate at pH 8. The fourth aliquot was analyzed via negative ionization following elution from a HILIC column (Waters UPLC BEH Amide 2.1 × 150 mm, 1.7 µm) using a gradient consisting of water and acetonitrile with 10 mM Ammonium Formate, pH 10.8 (HILIC). The MS analysis alternated between MS and data-dependent MSn scans using dynamic exclusion. The scan range varied slightly between methods but covered 70–1000 m/z. Raw data were uploaded to a publicly available database; see data availability below. Data Analysis: Compounds were identified by comparison to library entries of purified standards or recurrent unknown entities based on authenticated standards that contain the retention time/index (RI), mass to charge ratio (m/z), and fragmentation data. Biochemical identifications are based on three criteria: retention index within a narrow RI window of the proposed identification, accurate mass match to the library +/− 10 ppm, and the MS/MS forward and reverse scores between the experimental data and authentic standards. Metabolite Quantification and Data Normalization: Peaks were quantified using area-under-the-curve. A data normalization step was performed to correct variation resulting from instrument inter-day tuning differences. Essentially, each compound was corrected in run-day blocks by registering the medians to equal one (1.00) and normalizing each data point proportionately. Frozen human milk samples from the Haiti Cohort were processed by Metabolon, Inc. using the CMTRX made from the ZEBS Cohort in order for the data to be merged. Instrument variability was determined by calculating the median relative standard deviation (RSD) for the internal standards that were added to each sample prior to injection into the mass spectrometers. Overall process variability was determined by calculating the median RSD for all endogenous metabolites present in 100% of the Client Matrix samples, which are technical replicates of pooled client samples. For the Haiti Cohort 47 human milk samples (n = 1) were analyzed with an additional 8 QC samples. The independent identification of ddhC was conducted at the Northwest Metabolomics Research Center, University of Washington. Breast milk samples were processed using a protein precipitation method. Briefly, 250 µL methanol was added to 50 µL breast milk, vortexed, stored at −20 °C for 20 min, centrifuged at 18,000 x g for 15 min at 4 °C, and 150 µL of the supernatant was collected. Samples were dried in a Vacufuge at 30 °C and reconstituted in 500 µL of LC-matching solvent before LC-MS analysis. MS analysis was performed using an Agilent 6546 LC/Q-TOF MS using HILIC chromatography. Accurate mass analysis identified the molecular formula of X-12127 as C9H11N3O4, based on the [M + H]+ ion. Tandem MS/MS with a collision energy ramp of 5–40 V revealed its fragmentation patterns. While no database matches were found, the presence of a cytosine fragment suggested the metabolite might be ddhC. This identification was confirmed by matching the experimental MS/MS spectrum and retention time of [M + H]+ with a ddhC standard, and the identification was finalized using a parallel analysis of a ddhC reference standard with the study samples. Plasma and whole breast milk samples were analyzed for Tryptophan and Kynurenine by LC-MS/MS (Metabolon Method TAM220: “LC-MS/MS Method for The Determination of 15 Metabolites in Human Plasma (GFR-Panel)”. The method was adapted to analyze whole breast milk samples. Briefly, the plasma and whole milk samples were extracted with an organic solvent containing isotopically labeled internal standards (Tryptophan-d5, CIL Catalog# DLM-1092-0 and Kynurenine-d6, CIL Catalog# DLM-7842-0), injected on the Agilent 1290/SCIEX QTrap 5500 LC MS/MS system with a BEH Amide UHPLC column operating in positive mode using electrospray ionization, and the peak area of each analyte product ions measured against the peak area of the product ions of the corresponding internal standards. Quantitation was performed using a weighted linear least squares regression analysis generated from fortified calibration standards with ranges of 0.0250-10.0ug/mL for L-Kynurenine and 0.200-80.0ug/mL for L-Tryptophan that were generated immediately prior to the run (Tryptophan, Sigma Catalog# 93659 and Kynurenine, Sigma Catalog# 61250). For the quantitative/targeted KT panels, 6 QC samples are run per batch. These are spiked samples with varying concentrations of analyte stocks for Tryptophan and Kynurenine. There were 101 milk samples and 125 plasma samples analyzed, with an additional 24 QC samples for this study. Principal coordinates analysis and permutational multivariate analysis of variance (PERMANOVA) were performed using Euclidean distances calculated on normalized metabolite abundances. Linear mixed effects models of the form metabolite ~ HIVExposure*Visit + (1 | PID), where HIVExposure is an indicator variable for WLWH versus WWoH, Visit is a categorical variable for the study visit, and PID is the unique participant identifier, were used to identify differentially abundant metabolites as well as differences in KT log-ratio. Estimated marginal means were used to conduct the post hoc comparisons, either averaged or stratified by study visit. Results are reported as t-ratios, which are the estimates of the regression coefficients divided by their standard error. For the quantitative KT panel data, comparisons were performed using a Kruskal-Wallis (for more than two categories) or Welch's t (for two categories) test. Spearman's rank correlation was used to compare global and quantitative KT data and to compare metabolite abundances with maternal CD4 counts and viral loads. Random forest classification models were constructed separately for each study visit with HIV exposure as the outcome. A two-step approach was utilized to select both the optimal number of features as well as the specific features used for each model. In the first step, one hundred forests each comprising 10,000 trees were built to obtain feature importance values calculated as mean decrease in accuracy. In the second step, tenfold cross-validation with a sequentially reduced number of features was then used to identify the optimal number of features to be used. Finally, a sparse model was constructed for each study visit containing the optimal number of metabolites calculated in step 2 (up to a maximum of twenty features to aid interpretability), with features selected by the highest importance as calculated in step 1. Model accuracy, sensitivity, specificity, and Matthew's correlation coefficient were calculated from the out-of-bag error estimate for the final sparse model. Analysis of the Haiti validation cohort was performed using linear regression models of the form metabolite ~ HIVExposure + InfantAge, as there were no repeated measures. Infant age in days was included as a covariate in this regression model, given the range of ages at which milk samples were collected. Regression estimates from this validation cohort were compared to those from the main cohort at the 1-month and 4-month timepoints, as this best matched the distribution of infant ages between the two cohorts. All statistical analyses were conducted in the R statistical environment (version 4.1.3). A full list of all R packages used in the analyses is given in Table 2. All p-values were adjusted for multiple comparisons using the Benjamini-Hochberg false discovery rate method, and a cutoff of p < 0.05 was accepted as significant. All women provided written informed consent. The original protocol was approved by institutional review boards at Boston University, Children's Hospital of Los Angeles, Columbia University, and the University of Zambia Research Ethics Committee. The current study was approved by Children's Hospital of Los Angeles (IRB# CCI-03-110-CR002), University of California, Los Angeles (IRB# 16-000798), and Columbia University (AA6449 and AJ8851). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The MS data generated in this study have been deposited in the MetaboLights database under accession codes MTBLS2307 and MTBLS2573. The analysis code is available at GitHub65 [https://github.com/fanli-gcb/BMM] and Zenodo [https://doi.org/10.5281/zenodo.16658313]. Labuda, S. M. et al. Rates of Hospitalization and Infection-Related Hospitalization Among Human Immunodeficiency Virus (HIV)-Exposed Uninfected Children Compared to HIV-Unexposed Uninfected Children in the United States, 2007-2016. Epalza, C. et al. High incidence of invasive group B streptococcal infections in HIV-exposed uninfected infants. Paul, M. E. et al. Morbidity and mortality during the first two years of life among uninfected children born to human immunodeficiency virus type 1-infected women: the women and infants transmission study. Kuhn, L. et al. Elevations in mortality associated with weaning persist into the second year of life among uninfected children born to HIV-infected mothers. Marinda, E. et al. Child mortality according to maternal and infant HIV status in Zimbabwe. Arikawa, S., Rollins, N., Newell, M. L. & Becquet, R. Mortality risk and associated factors in HIV-exposed, uninfected children. & Evans, C. Children who are HIV-exposed and uninfected: evidence for action. McNally, L. M. et al. Effect of age, polymicrobial disease, and maternal HIV status on treatment response and cause of severe pneumonia in South African children: a prospective descriptive study. & Hemelaar, J. Maternal HIV infection is associated with distinct systemic cytokine profiles throughout pregnancy in South African women. Dirajlal-Fargo, S. et al. HIV-exposed-uninfected infants have increased inflammation and monocyte activation. Schnittman, S. R. et al. Changes in Immune Activation During Pregnancy and the Postpartum Period in Treated HIV Infection. Components of metabolic syndrome associated with lower neurocognitive performance in youth with perinatally acquired HIV and youth who are HIV-exposed uninfected. Vyas, P. et al. Impact of HIV status on systemic inflammation during pregnancy. Impaired functionality of antigen presenting cells in HIV- exposed uninfected infants in the first six months of life. Jalbert, E. et al. HIV-exposed uninfected infants have increased regulatory T cells that correlate with decreased T cell function. A. et al. Altered innate immune development in HIV-exposed uninfected infants. Yin, L. et al. Maternal immunity shapes biomarkers of germinal center development in HIV-exposed uninfected infants. Egorova, M., Egorov, V. & Zabrodskaya, Y. Maternal Influenza and Offspring Neurodevelopment. Elgueta, D., Murgas, P., Riquelme, E., Yang, G. & Cancino, G. I. Consequences of Viral infection and cytokine production during pregnancy on brain development in offspring. Estes, M. L. & McAllister, A. K. Maternal immune activation: Implications for neuropsychiatric disorders. Han, V. X. et al. Maternal acute and chronic inflammation in pregnancy is associated with common neurodevelopmental disorders: a systematic review. Knuesel, I. et al. Maternal immune activation and abnormal brain development across CNS disorders. & Bonett, D. Adult schizophrenia following prenatal exposure to an influenza epidemic. & Lim, A. I. Maternal-driven immune education in offspring. Hofsink, N., Groenink, L. & Plosch, T. The fetal programming effect of maternal immune activation (MIA) on the offspring's immune system. Lim A. I. et al. Prenatal maternal infection promotes tissue-specific immunity and inflammation in offspring. The kynurenine pathway of tryptophan catabolism, CD4+ T-cell recovery, and mortality among HIV-infected Ugandans initiating antiretroviral therapy. Favre, D. et al. Tryptophan catabolism by indoleamine 2,3-dioxygenase 1 alters the balance of TH17 to regulatory T cells in HIV disease. Cao, B. et al. Dysregulation of kynurenine pathway and potential dynamic changes of kynurenine in schizophrenia: A systematic review and meta-analysis. Mehraj, V. & Routy, J. P. Tryptophan catabolism in chronic viral infections: handling uninvited guests. Immune regulation through tryptophan metabolism. Pedraz-Petrozzi, B. et al. LC-MS/MS-based quantification of tryptophan, kynurenine, and kynurenic acid in human placental, fetal membranes, and umbilical cord samples. Huengsberg, M. et al. Serum kynurenine-to-tryptophan ratio increases with progressive disease in HIV-infected patients. Gut epithelial barrier dysfunction and innate immune activation predict mortality in treated HIV infection. The effect of HAART-induced HIV suppression on circulating markers of inflammation and immune activation. Vujkovic-Cvijin, I. et al. Dysbiosis of the gut microbiota is associated with HIV disease progression and tryptophan catabolism. Plasma Kynurenine to Tryptophan ratio is negatively associated with linear growth of children living in a slum of bangladesh: results from a community-based intervention study. Plasma Tryptophan and the Kynurenine-Tryptophan ratio are associated with the acquisition of statural growth deficits and oral vaccine underperformance in populations with environmental enteropathy. Fazzolari-Nesci, A., Domianello, D., Sotera, V. & Raiha, N. C. Tryptophan fortification of adapted formula increases plasma tryptophan concentrations to levels not different from those found in breast-fed infants. Kamimura, S., Eguchi, K. & Sekiba, K. Tryptophan and its metabolite concentrations in human plasma and breast milk during the perinatal period. O'Rourke, L. et al. Tryptophan metabolic profile in term and preterm breast milk: implications for health. van Sadelhoff J. H. J. et al. Short communication: differences in levels of free amino acids and total protein in human foremilk and Hindmilk. Zanardo, V., Bacolla, G., Biasiolo, M. & Allegri, G. Free and bound tryptophan in human milk during early lactation. Gizzi, A. S. et al. A naturally occurring antiviral ribonucleotide encoded by the human genome. Effects of early, abrupt weaning on HIV-free survival of children in Zambia. Wong, A. C. et al. Serotonin reduction in post-acute sequelae of viral infection. Bender, J. M. et al. Maternal HIV infection influences the microbiome of HIV-uninfected infants. The tryptophan catabolite or kynurenine pathway in COVID-19 and critical COVID-19: a systematic review and meta-analysis. The aryl hydrocarbon receptor: an environmental sensor integrating immune responses in health and disease. Nunzi E. et al. Host-microbe serotonin metabolism. The kynurenine pathway and quinolinic acid: pivotal roles in HIV associated neurocognitive disorders. Mehta, R. et al. Antiviral metabolite 3'-deoxy-3',4'-didehydro-cytidine is detectable in serum and identifies acute viral infections including COVID-19. Sala, S. et al. Integrative molecular structure elucidation and construction of an extended metabolic pathway associated with an ancient innate immune response in COVID-19 patients. & Beard, M. R. The role of viperin in the innate antiviral response. Seo, J. Y., Yaneva, R. & Cresswell, P. Viperin: a multifunctional, interferon-inducible protein that regulates virus replication. Nasr, N. et al. HIV-1 infection of human macrophages directly induces viperin, which inhibits viral production. The specific metabolome profiling of patients infected by SARS-COV-2 supports the key role of tryptophan-nicotinamide pathway and cytosine metabolism. & Marliere, P. Cytosine drives evolution of SARS-CoV-2. Metz, P. J. et al. Symmetric Arginine Dimethylation is selectively required for mRNA splicing and the initiation of Type I and Type III Interferon signaling. Tobin, N. H. et al. Maternal HIV infection and the milk microbiome. Evans, A. M., DeHaven, C. D., Barrett, T., Mitchell, M. & Milgram, E. Integrated, nontargeted ultrahigh performance liquid chromatography/electrospray ionization tandem mass spectrometry platform for the identification and relative quantification of the small-molecule complement of biological systems. Precision of a clinical metabolomics profiling platform for use in the identification of inborn errors of metabolism. High Resolution Mass Spectrometry improves data quantity and quality as compared to unit mass resolution mass spectrometry in high-throughput profiling metabolomics. We would like to acknowledge and thank the study participants and the ZEBS and Haiti Study Teams. This work received funding from the National Institutes of Health, National Institute of Child Health and Human Development NICHD R01 HD 39611 (LK, DT), NICHD R01 HD 40777 (GMA), NICHD 1-R01 HD 057617 (LK, GMA), NIH R21 HD 49287 (JS), Doris Duke Charitable Foundation CU52209601 (LK), PEPFAR GHS-A-00-00020-00 (LK) and NIH P30DK035816 (WZ, DR). Overall support for the International Maternal Pediatric Adolescent AIDS Clinical Trials Network (IMPAACT) was provided by the National Institute of Allergy and Infectious Diseases (NIAID) with co-funding from the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD) and the National Institute of Mental Health (NIMH), all components of the National Institutes of Health (NIH). Nicole H. Tobin, Fan Li, Kathie G. Ferbas & Grace M. Aldrovandi UW Northwest Metabolomics Center, Seattle, WA, USA Division of Pediatric Allergy and Immunology, Department of Pediatrics, Duke University School of Medicine, Durham, NC, USA Gertrude H. Sergievsky Center, Vagelos College of Physicians and Surgeons; and Department of Epidemiology, Mailman School of Public Health, Columbia University Irving Medical Center, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar interpreted the data, and N.H.T. FL performed the statistical analyses. All authors reviewed and approved the final manuscript. The authors declare no competing interests. All authors approved this manuscript for publication. Nature Communications thanks Gerard Gonzales and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tobin, N.H., Li, F., Zhu, W. et al. Altered milk tryptophan and tryptophan metabolites in women living with HIV. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64911-z'>Multi-lens ultrasound arrays enable large scale three-dimensional micro-vascularization characterization over whole organs</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-28 10:05:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mapping microcirculation at the whole-organ scale in 3D is crucial for understanding vascular pathologies and improving diagnostics. Although 3D ultrasound localization microscopy (ULM) enables microscopic resolution by localizing intravenously injected microbubbles in small animal models, visualizing entire organs in large animals or humans remains challenging due to limited field of view, low sensitivity, and probe technological complexity. Here, we demonstrate how a multi-lens array method overcomes these limitations. Combined with 3D ULM, it maps and quantifies large vascular volumes (up to 120 × 100 × 82 mm³) at high spatial resolution (125–200 µm) with a volumetric acquisition rate of 312 Hz, using low-cost technology. This approach enables deeper insights into hemodynamics from large vessels to pre-capillary arterioles, by providing vast and rich datasets of whole-organ vascularization. It could also facilitate diagnosis of microcirculation disorders and monitoring of small-vessel disease treatments by addressing key limitations of current imaging modalities. Microcirculation plays a vital role in sustaining tissue health by regulating blood flow and facilitating the exchange of gases, nutrients, and waste products. In key organs such as the heart, kidneys, and liver, proper microcirculation is essential for their optimal function. Any structural or functional alteration in these networks can lead to severe clinical outcomes, including heart failure, renal failure, and various chronic diseases. By evaluating the integrity of the circulatory system from the largest arteries down to small arterioles, clinicians could identify and treat dysfunctions early, potentially preventing irreversible damage1. Nevertheless, the amount of information to deal with in order to reach such a goal is massive. Building a 100 × 100 × 100 mm3 raw data volume with a 100 µm voxel size at a classical 300 Hz frame rate for hemodynamic estimations leads to a typical acquisition movie of 300 billion of voxels per second. CT angiography (CTA) is the clinical gold standard to non-invasively visualize blood vessels at the whole-organ scale by injecting ionizing contrast agents, providing detailed anatomical insights into vascular structures in clinic. The recent advancement of photon-counting computed tomography (PCCT) has further improved CTA by offering enhanced spatial resolution and better tissue contrast2. However, CTA, even with PCCT, has limited capacity to assess flow dynamics due to its static nature and temporal constraints. Magnetic Resonance Imaging (MRI), particularly with 4D flow imaging, enables detailed whole-organ or whole-body visualization of flow dynamics by capturing blood flow patterns over time in three dimensions3. This capability makes it highly effective for hemodynamic assessments and functional imaging. However, MRI faces limitations, including a spatial resolution limited to a few tenths of a millimetre, high costs, and limited accessibility, which can hinder its widespread clinical use. On the other hand, ultrasound is being routinely used to assess blood flow and its dynamics in the clinic, but it usually suffers from low resolution and sensitivity. Nevertheless, ultrasound imaging is an active research field, and major technical advances achieved over the past decades open perspectives for developing high-performance imaging modes that could rapidly be translated to clinics, given the relatively low cost and portability of ultrasound technologies4,5,6. In the past decade, ultrasound localization microscopy (ULM) was developed to map microcirculation at the microscopic scale in preclinical organ models7,8. This technique has progressed to the point of demonstrating 2D proof-of-concept in patients to image organs in super-resolution, such as the brain9, heart10, and kidney11. However, 2D imaging has inherent limitations, including its inability to provide a complete view of the organ under study. Improper positioning of the imaging plane can lead to increased risks of diagnostic misinterpretations, highlighting the importance of developing three-dimensional approaches to overcome these constraints12. Recently, three-dimensional (3D) ULM has been applied to map the whole organ microcirculation in small animals13,14,15,16,17,18, providing valuable insights into organ function and enabling a more comprehensive understanding of their physiological processes. However, physiology, anatomy, and size of small animal organs are hardly comparable to human organs and translation of 3D ULM to large animal models has been limited by current technology. State-of-the-art matrix array probes are associated with high complexity, expensive, and cumbersomeness hardware, primarily due to the necessity to connect each individual element of the probe to a receive channel. Thousands of elements and receive channels are thus required to cover the entire volume of a human organ, which limits their current applications. To alleviate these issues, we have recently introduced the concept of multi-lens arrays both in simulations19 and in in vitro experiments with a simplified prototype20. It consists in using a large aperture probe filled with large elements combined to individual acoustic diverging lens. Large elements provide increased sensitivity while covering a wide probe surface. Diverging lenses are used to restore a low directivity, compensating for the reduced angular aperture typically associated with large elements. This design improves the focusing quality and enhances image resolution. Comprising of a few hundred elements, the probe is compatible with ultrasonic scanners commonly available in clinical settings. When combined to 3D ULM methods, these arrays can assess the blood flows across several spatial scales, down to the micron scale across the whole organ. In this manuscript, we introduce a first large multi-lens array probe and a full post-processing framework to enable the imaging and quantification of microcirculation of living organs in unprecedented large volumes. Applied to map organs in a large animal swine model, we demonstrate ULM imaging over massive volumes (up to 120 × 100 × 82 mm3), including ex-vivo heart, in-vivo kidney and liver with an effective temporal resolution depending on the accumulation time required for ULM reconstruction. It provides vast and rich mdatasets of whole organ vascularization and hemodynamics that could prove invaluable for advancing AI-driven medical and comprehensive medicine. In this article, we presented the results of a multi-lens matrix array probe with a large aperture of 104 × 82 mm² and a reduced number of elements (252 elements), as illustrated in Fig. A schematic diagram of a single large element coupled to a compound diverging lens is shown in Fig. 1b, highlighting the principle of this composite material. It consists of a plano-convex lens combined to plano-concave lens which according to Snell's law enables to create a strong diverging lens (part Principles and definitions in the Supplementary Methods section). a Schematic representation of the multi-lens array probe with a large aperture (~104 × 82 mm²) composed of 252 elements, compared to a conventional fully populated probe with the same number of elements (aperture~14 × 11 mm²) depicted within the black box. c A schematic diagram of the plano-convex diverging lens is shown in black, with red lines indicating the beam path within the lens. The beam path through the blue plano-concave lens is also illustrated. (a–c created using Autodesk Inventor Professional 2025.0.1). d Schematic and 3D maps of maximum transmitted acoustic pressure field, illustrating directivity performances over a volume of 60 × 60 × 60 mm³: d.i the compound lens material coupled to a single large element of an (3λ × 3λ) compared to d.ii a single large element without lens (3λ × 3λ), and d.iii a single small element (λ/2 × λ/2). d.iv Graphs show the angular directivity [degrees] as a function of intensity threshold level, measured at a depth (z) of 35 mm from the maximum pressure field shown in (d). for three configurations: single lens with large element (solid green), large element without lens (blue dashed), and small element (red curve). e Spatial distribution of the intensity field received by the multi-lens probe (e.i) over a large volume (~105 × 100 × 100 mm³) compared to conventional probes, fully populated (e.ii) and sparse matrix arrays (e.iii). (e.iv) Graph showing the intensity threshold level as a function of the covering volume for the three probes: multi-lens array (solid blue curve), fully populated array (red curve), and sparse array (orange dashed curve). 1c further illustrates the refraction principles of the two diverging lenses in the compound design, showing how both lenses contribute to beam divergence. Figure 1d compares the simulated acoustic field emitted by a large element with a compound lens to a large element with no lens and a small element used in a conventional dense matrix. i) and the small element (Fig. iii) show low directivity and a wider field of view. Large element with and without the compound lens show high energy transmission (Fig. The large element with the compound lens is the only approach that combines low directivity and high transmitted energy. The angular directivity at −6 dB, measured at a depth of 35 mm in the pressure field, is presented in the Fig. The angular directivity for the large element with compound lens, the large element and the small element were found to be 33.17°, 10.27°, and 51.13°, respectively. The sensitivity response of the multi-lens array probe was compared to a dense matrix array and a sparse array with the same number of elements, as shown in Fig. Sensitivity was defined as normalized intensities received from a transmitter placed in the medium of interest. 1e.i illustrate the sensitivity response of the multi-lens array probe, revealing high sensitivity across a large simulated volume (105 × 100 × 100 mm³). In contrast, the dense matrix array demonstrates a confined sensitivity below the aperture (Fig. ii), while the sparse array shows low sensitivity, especially in the far field (Fig. These qualitative observations were quantified and presented in Fig. iv, which shows that the multi-lens array achieves significantly higher sensitivity for an intensity above −18 dB within the simulated volume and covering up to 1050 cm³. In comparison, the dense matrix and sparse array showed lower sensitivity, covering 443 cm³ and 580 cm³ of the total volume, respectively. The simulated points spread functions (PSF) of the multi-lens array probe was performed and quantified, revealing a grating lobe level of approximately –25.50 dB, –24.57 dB, and –25.36 dB, and lateral resolution was found to be 1.15 mm, 1.17 mm, and 1.26 mm at –6 dB for imaging depths of 30 mm, 45 mm, and 60 mm, respectively. These results were obtained using synthetic aperture imaging with 252 transmit sources. Additionally, we evaluated image quality under ultrafast imaging conditions using a reduced number of transmit events (16 sources), to reflect the practical configuration used during acquisitions. The tip of a needle was imaged in vitro to study the PSF of the multi-lens array probe, as illustrated in Fig. Grating lobes with an intensity of -22.40 dB and a lateral resolution of 1.06 mm at −6 dB were quantified for an ultrafast imaging sequence using 16 sources in transmit (Fig. The measured -6 dB width of the PSF was consistent with the theoretical resolution calculated based on the ultrasonic wavelength, focal distance, and array aperture. a Schematic representation of the experimental flow phantom setup. Microbubbles (MB) were injected into an 870 µm inner diameter tube and 3D Ultrasound Localization Microscopy (ULM) was performed using the multi-lens array. Papadacci, C. (2025) https://BioRender.com/e2rxni8 and Autodesk Inventor Professional 2025.0.1 b Point spread function in 3D (PSF) of the multi-lens. c Quantification of the grating lobes and the lateral resolution at -6dB (horizontal dashed line); the red graph represents the maximum intensity projection profile in lateral dimension of the PSF using 16 sources in transmit to perform ultrafast ultrasound imaging. d 3D MB density of the tube (imaging volume ~100 × 93 × 30 mm3). e 3D absolute flow map of the MB velocity distribution through the tube. f Cross section of the tube velocities in the blue dashed box in (e) highlights a Poiseuille profile. Significant differences between the neighbouring voxels ( ~ 75 µm) within the velocity profile were demonstrated using an unpaired two-sided Student's t-test. P values from left to right are P1 = 0.0004, P2 = 3.23 × 10-11, P3 = 0.0022, P4 = 4.22 × 10-05, P5 = 0.0059, P6 = 8.03 × 10-05. n samples corresponds to the number of MB passing through each voxel/vox (84, 102, 124, 156, 159, 314, 205, 105 and 51 from left to right). g Flow rate measurements as function of the inlet flow rate set by the syringe pump, showing a linear relationship as shown by blue curve fit (y = 1.06x – 7.83; y: measured flow rate and x: inlet flow rate, coefficient of determination (R2) = 0.98); Red dots represent the mean values of the data points (n = 10 samples for each measurement) and the horizontal red lines indicate the error bars as ± SD: 73 ± 6, 101 ± 3, 143 ± 5 and 209 ± 5 (from left to right); Measurements are technical replicates. 3D ULM was performed on a large flow phantom containing a tube with injected microbubbles (MBs). The results presented in Fig. 2d, showed the reconstructed MB density volume of the tube, revealing a large imaging volume of 100 × 93 × 30 mm³. Grating lobes were filtered out by the tracking algorithm. Absolute flow velocity was measured in the tube (Fig. 2e), revealing a Poiseuille flow profile (Fig. A statistical Student's t-test showed significant differences between neighbouring voxels along the velocity profile (P values from left to right are P1 = 0.0004, P2 = 3.23 × 10−11, P3 = 0.0022, P4 = 4.22 × 10−05, P5 = 0.0059, P6 = 8.03 × 10−05), indicating a resolution of 75 µm. The inner tube diameter was measured using skeletonization: 916 ± 38 µm which was in a good agreement with the manufacturer tube diameter (870 µm). The maximum absolute velocities at the centre of the tube were measured for the four flow rates: 62 ± 5 mm s−1, 86 ± 4 mm s−1, 122 ± 5 mm s−1, and 177 ± 4 mm s−1. The measured flow rates were in a good agreement with the imposed flow rates (R2 = 0.98), Fig. For more complex spatial and flow patterns, the twisted-tube configuration was imaged. The 3D microbubble (MB) density map (Supplementary Fig. reveals a detailed reconstruction of the tubes over a large volume, clearly illustrating the separation between the intertwined tube segments (Supplementary Fig. Flow direction mapping demonstrated both upward and downward flow within the structure (Supplementary Fig. Quantification of flow velocity revealed a Poiseuille profile, with high spatial resolution (Supplementary Fig. 3a) in a large imaging volume of 120 × 100 × 82 mm3 as illustrated by the 3D MB density volume in Fig. a Experimental setup diagram: 3D Ultrasound localization Microscopy (ULM) imaging using the multi-lens array probe driven by an ultrasound system of 256 channels to image the heart with a retrograde and microbubble perfusion. Papadacci, C. (2025) https://BioRender.com/hyvll4l and Autodesk Inventor Professional 2025.0.1. b 3D Microbubbles (MB) density map of the coronary network of the porcine heart illustrating the large imaging volume (~120 × 100 × 82 mm3). c Views of the MB density map (yz and xy views) of the coronary porcine heart. d Maximum Intensity Projection (MIP) of Power Doppler data acquired from the isolated porcine heart using the multi-lens probe. e 3D flows velocity map showing the MB velocity through the coronary networks. f Magnified view of the red box in (e) volume (53 × 41 × 38 mm3). g Analysis of the Poiseuille flow profile in a large vessel 1 in (f). Significant differences between the neighbouring voxels (~150 µm) within the velocity profile were demonstrated using an unpaired two-sided Student's t-test. P values from left to right are P1 = 0.0116, P2 = 0.0088, P3 = 7.25 × 10−08. n samples correspond to the number of MB passing through each voxel (15, 37, 65, 86, 105 and 95 from left to right). h Magnified view of the region in the white box in (b) (30 × 43 × 45 mm volume). i Quantification of the vessel's radii by applying skeletonization algorithm on the density in (f). j 3D flow map showing arterial (red) and venous (blue) flow; arrows indicate upward and downward directions. k Flow rate as a function of radius (blue stars) provides an exponent of 2.3 which is in good agreement with Murray's law. l number of MB distribution as a function of MBs velocity. m Fourier shell correlation (FSC; solid blue curve) to estimate the resolution of 3D ULM coronary mapping (~125 µm). Coronary network can be visualized in different directions as illustrated by the YZ and XY projections of the 3D density (Fig. 3d shows the maximum intensity projection (XZ view) of the Power Doppler image of the ex vivo porcine heart. The high temporal resolution achieved by the 3D ultrafast ultrasound imaging allowed for the estimation of absolute flow velocities via tracked MBs, as shown in Fig. 3e, with velocity ranges from 10 mm s−1 in small vessels to over 300 mm s−1 in the largest vessels. A Poiseuille flow profile was observed by analysing the velocity profile in a large vessel (Vessel 1 in the magnified view in Fig. A statistical Student's t-test revealed significant differences of velocity values along the profile between neighbouring voxels (voxel size = 150 µm) containing tracked MB velocities as shown in Fig. A magnified view of 3D density in Fig. 3i to estimate vessel radii, revealing a range from approximately 75 µm for the smallest vessels to 600 µm for the largest vessels. Figure 3j shows the flow direction in the coronary network, where red (downward) can be associated with the arterial flow system and blue (upward) the venous flow system. Fitting to a power law, the flow rate radius relationship provides an exponent of 2.3 (R² = 0.82), as shown in Fig. 3k, which is consistent with Murray's law21. The number of tracked MBs within the coronary network is shown as function of their estimated velocity in Fig. 3l, with a maximum velocity of 300 mm s−1. The Fourier Shell Correlation (FSC) analysis was performed to estimate spatial resolution for the 3D ULM coronary mapping, achieving a resolution of approximately 125 µm at the intersection of the FSC and ½ bit curves, as shown in Fig. The multi-lens probe was positioned on the right lateral side of the swine, in front of the kidney, and held using an articulated arm. Ultrasound acquisitions were synchronized with the Electrocardiogram (ECG) R-peak wave, during periods of minimal respiratory motion, as illustrated in Fig. 4b shows a B-mode image of the porcine kidney obtained with a portable ultrasound device. 3D ULM of the kidney was performed over a large volume of 60 × 80 × 40 mm³, and the results is presented as 3D MB density maps in Fig. 4c and a YZ projection view in Fig. The 3D absolute flow velocity shown in Fig. 4e, was evaluated, reaching a maximum velocity of approximately 280 mm s−1 in the large vessels. 3D directional flow velocities reveal the arterial flow system in red and the venous flow system in blue (Fig. ii for XZ and YZ projections view, respectively). A maximum intensity projection (YZ view) of the Power Doppler image of the in vivo porcine kidney is shown in Fig. A skeletonization algorithm was applied on the volume showing in white box in Fig. 4c, to estimate vessel radii, revealing a range of 70 µm to 400 µm as illustrated in Fig. A spatial resolution of 147 µm was obtained by applying the FSC, as shown in Fig. Fitting to a power law, the flow-rate radius relationship provides an exponent of 2.13 (R2 = 0.70), which is in good agreement with Murray's law, as shown in Fig. The number of tracked Mbs within the kidney vasculature is shown as function of their estimated velocity in Fig. 4.l shows a Poiseuille flow profile by analysing the velocity profile in porcine kidney within the vessel (i) illustrated in the Fig. A statistical Student's t-test revealed significant differences of velocity values along the profile between neighbouring voxels (voxel size = 128 µm) containing tracked MB velocities. Dynamic ULM was performed (Fig. 4m), which highlights higher velocities during systolic phase compared to the diastolic phase. a Experimental setup diagram: 3D ultrasound localization microscopy (ULM) of in vivo kidney using multi-lens. Papadacci, C. (2025) https://BioRender.com/i6ehye3 and Autodesk Inventor 2025.0.1. The acquisition windows (green) triggered between two respiratory phases (solid black curve) and synchronized with the R-peak of the electrocardiogram (ECG) signal (blue curve). b B-mode image of the kidney captured by Philips device. c 3D microbubbles (MB) density map of the kidney showing the size of imaging volume (60 × 80 × 40 \({{{{\rm{mm}}}}}^{3}\)). d YZ projection of kidney MB density. e 3D flows map of MB velocity distribution. f 3D flow velocity maps (XZ and YZ views) showing arterial (red) and venous (blue) flow; arrows indicate upward (red), downward (blue) directions. g Maximum Intensity Projection of Power Doppler kidney using the multi-lens. h Magnified view of the region in the white box in (c) (28 × 16 × 38 mm3); and his related Skeletonization to quantify vessel's radii. i Fourier shell correlation (FSC; solid blue curve) to estimate resolution within the 3D ULM of kidney (~147 µm). j Flow rate as a function of radius (blue stars) provides an exponent of 2.13 which is in a good agreement with Murray's law. l, Analysis of Poiseuille flow profile of vessel (i) in (e). Significant differences between the neighbouring voxels (~128 µm) within the velocity profile were demonstrated using an unpaired two-sided Student's t-test. P values from left to right are 0.0469 and 0.0153. Red dots indicate regions of peak velocity. n samples are the number of MB passing through voxels (13, 10, 15,10 from left to right). m Maps showing the MB velocity in systole and diastole phases. n Velocity variation over time (in percentage according to ECG) for vessels (V1 in solid blue and V2 in green) and resistivity index RI1 and RI2 for V1 and V2, respectively. Figure 5a illustrates the implemented 3D rigid inter-block motion correction pipeline. For each acquisition block at a low respiratory phase, a Power Doppler volume reconstruction was generated. An intensity-based registration between these volumetric Power Doppler blocks was then performed to compute a transformation, which was subsequently applied to the corresponding tracked microbubble points for each block. a Illustration of acquisition blocks obtained during low respiratory phases and synchronized with the cardiac cycle. The power Doppler volumes are colour-coded according to each corresponding acquisition. \({{{{\rm{pD}}}}}_{{{{{\rm{i}}}}}_{({{{\rm{x}}}},{{{\rm{y}}}},{{{\rm{z}}}})}}\) denotes the power Doppler block used as the reference for 3D intensity-based rigid registration. \({T}_{{{{{\rm{i}}}}}_{({{{\rm{x}}}},{{{\rm{y}}}},{{{\rm{z}}}})}}\) represents the transformation parameters resulting from intensity-based rigid registration between the reference power Doppler volume and the volume to be aligned. \({P}_{{{{{\rm{i}}}}}_{({{{\rm{x}}}},{{{\rm{y}}}},{{{\rm{z}}}})}}\) corresponds to the microbubble (MB) tracked positions within each block to be spatially corrected. \({P\,}_{{{{{\rm{corrected}}}}}_{{{{\rm{i}}}}({{{\rm{x}}}},{{{\rm{y}}}},{{{\rm{z}}}})}}\) represents the MB positions within each block after inter-block motion correction. b Motion correction results: b.1, density map of the kidney cortex (blue box; ~ 28 × 45 × 33 mm³) before motion correction; b.2, density map after correction. c, Magnified views corresponding to the white boxes (~23 × 13 × 18 mm³) in (b), highlighting detailed changes before and after correction. d Cross-sectional profiles of two representative vessels: d.i, distance profile across vessel i from (c), showing the normalized amplitude as a function of distance before (red curve) and after (solid blue curve) correction; d.ii, corresponding profile for vessel ii from (b). 5b presents a qualitative comparison of the cortical density map of the kidney before (Fig. 5c represents a magnified view of the region highlighted by the white box in Fig. Additionally, a quantitative comparison of vessel density measurements (vessels i and ii) shows an increase in density after motion correction, indicating improved vessel visualization and alignment. The multi-lens probe was positioned above the swine, placed in front of the liver, and held using an articulated arm. Similarly to the kidney experiment, ultrasound acquisitions were synchronized with the ECG R-peak trigger and during periods of minimal respiratory motion as illustrated in Fig. Real-time B-mode imaging using a portable ultrasound device allowed 2D visualization of the liver (Fig. The multi-lens array probe enabled wide-field imaging, enabling high-depth visualization of the liver's vasculature and capturing a substantial imaging volume of 65 × 100 × 82 mm³, as demonstrated by the 3D density maps in Fig. 6e illustrates the 3D absolute flow velocity within the liver, reaching a maximum velocity value of 300 mm s−1. The direction of MBs flow through the liver's vasculature is shown in Fig. 6f, with upward flow in red and downward flow in blue. Dynamic velocity measurement was performed for arterial (pulsatile), vein (pulsatile/retrograde) and portal (no pulsatility) vein vessels, representing in V1, V2 and V3 respectively in the Fig. 6g, which highlighted higher velocities during systolic phase compared to the diastolic phase in case of an arterial vessel in Fig. iii suggests that the flow velocity is uniform in the portal vein. 6h presents a Poiseuille flow profile by analysing the velocity distribution within the vessel (i) in porcine liver flow velocity illustrated in the Fig. A statistical Student's t-test revealed significant differences of velocity values along the profile between neighbouring voxels (voxel size = 159 µm) containing tracked MB velocities. Fitting the flow-rate-to-radius relationship to a power law yielded an exponent of 2.36 (R² = 0.61), which is consistent with Murray's law (Fig. Spatial resolution of 200 µm was measured using Fourier Shell Correlation (FSC), as demonstrated in Fig. a Experimental setup diagram: Ultrasound Localization Microscopy (ULM) imaging was performed in vivo on liver using a multi-lens probe positioned above the liver and driven by a 256-channel ultrasound system. Papadacci, C. (2025) https://BioRender.com/bp11wr0 and Autodesk Inventor Professional 2025.0.1. The acquisition windows (green) were triggered between two respiratory phases (solid black curve) and synchronized with the R-wave of the electrocardiogram (ECG) signal (blue curve). b B-mode images of the porcine liver captured by portable device. d Maximum Intensity Projection of Power Doppler of liver using the multi-lens probe. e flow velocity map of tracked microbubbles (MBs). f 3D directional flow velocity map of MBs. arrows indicate upward (red), downward (blue) directions. g Dynamic velocity variation over time (in percentage) for vessels (V1, V2 and V3) according to ECG signal in graph with dashed line: g.i Dynamic velocity over time in an arterial vessel 1 (red curve). g.ii Dynamic velocity over time in a vein vessel 2 (blue curve). g.iii Dynamic velocity over time in a portal vein (green curve). h Analysis of the Poiseuille flow profile in vessel (i) in (e). Significant differences between the neighbouring voxels (~159 µm) within the velocity profile were demonstrated using an unpaired two-sided Student's t-test. P values from left to right are 0.0082, 0.0139. n samples correspond to the number of microbubbles (MBs) passing through voxels (33, 37, 34 and 41 from left to right). Red dots indicate the regions of peak velocity. i, Flow rate (Q) as a function of radius (r) (blue stars; data points) provides an exponent of 2.36 which is in a good agreement with Murray's law. j Fourier shell correlation (FSC; solid blue curve) to estimate resolution of 3D ULM of liver (~200 µm). In this study, an ultrasonic multi-lens array approach was developed to map and quantify micro-scale vascular network and flow dynamics. This method overcomes the limitations of conventional ultrasound imaging by enabling high-resolution volumetric imaging of large organs. This innovative ultrasonic imaging method was initially conceived and validated through numerical simulations, followed by in vitro experiments on large phantoms. It was then applied to an explanted swine heart and in vivo swine kidney and liver, demonstrating whole-organ volumetric imaging capabilities. It enabled a complete visualization of the vascular network with a spatial resolution of a few tenths of micrometre (down to 125 µm) with a very high temporal resolution (312 Hz frame rate), providing anatomical details with high precision. Fine vascular quantification was achieved by measuring the vessel radius at every voxel, ensuring structural mapping across the entire vascular tree. Beyond structural imaging, the method offered advanced flow dynamic assessment across all vascular scales, capturing both macro and microcirculatory hemodynamic. This detailed analysis allowed for the differentiation of arteries and veins based on axial flow direction and enabled the measurement of absolute flow velocity along with its Poiseuille profile, which is a key indicator of laminar flow behaviour. These precise flow measurements pave the way for advanced physiological evaluations, including velocity distribution mapping, flow rate calculation, pulsatility index, and the assessment of Murray's law, a principle that links vessel diameter to optimal flow efficiency. This level of anatomical and functional measurements details beyond those of the conventional imaging techniques, allowing assessment of vascular structures and flow. The combination of whole-organ volumetric imaging with high-resolution vascular quantification effectively addresses key limitations of existing modalities, such as ultrasound Doppler imaging, CT angiography, and 4D flow MRI. The high spatial resolution achieved with this technique is due to the implementation of 3D Ultrasound Localization Microscopy (ULM) and 3D dynamic ULM22,23, going beyond the diffraction limit that traditionally constrains standard ultrasound resolution. The method demonstrated a spatial resolution approximately ten times finer than conventional ultrasound, enabling the visualization of small vascular structures. Additionally, the limited field of view commonly associated with classical matrix arrays was resolved through the use of a low-frequency, and large-aperture matrix array. This innovative approach allowed the reconstruction of large volumetric datasets, both laterally and in depth, enabling the visualization of entire organs without compromising image resolution or frame rate. The enhanced resolution allows precise quantification at a small scale, significantly improving anatomical visualization compared to standard ultrasound and CT angiography. Beyond structural imaging, the method also offers flow dynamic assessment, overcoming the temporal limitations of CT angiography. Assessing flow dynamics across all vascular scales also surpasses 4D flow MR,I which is often limited to the flow assessment of large vessels. Furthermore, the non-invasive nature and real-time capabilities make it a cost-effective and accessible alternative, reducing the financial and logistical barriers often associated with MRI. For the explanted swine heart, a retrograde perfusion was set to perfuse the coronary network10,14,24, which was imaged and quantified over the entire heart. The swine model is considered an ideal experimental model to study the coronary artery network because cardiac, vascular anatomies and dimensions are comparable to those of humans. It is often used to study coronary artery diseases because swine models have negligible collateral circulation, so each coronary artery supplies a specific cardiac region, which is ideal to induce myocardial infarction. The method would be particularly suited for preclinical cardiovascular research, particularly in the study of physiological processes, drug development and ex vivo investigations of ischaemia-reperfusion injury25. Note that Modulating temperature to increase motion could provide an interesting experimental model for evaluating the accuracy and robustness of motion correction algorithms. Moreover, being able to map the complex network of a swine heart, envision the capability of mapping the coronary network in humans. Direct applications include the characterization of human grafts for assessing graft quality, addressing the critical challenge of determining the donor heart's viability for transplantation26. The swine's kidney network was mapped over a large volume (60 × 80 × 40 mm3). The large arterial network visualized through Doppler imaging in Supplementary Fig. 5 demonstrated qualitative similarity to X-ray scan, with comparable arteries clearly identified in both imaging modalities. The entire network, including smaller vessels, were evaluated. Translation to human applications is anticipated in the near future, as no fundamental barriers exist to prevent its realization. Imaging kidney microcirculation in patients is particularly important in chronic kidney disease. In most cases, the peripheral kidney microvascular is damaged, yet there are currently no non-invasive and quantitative diagnostic tools to effectively assess this condition. Swine models of chronic kidney disease could also be a natural continuation of this work27, where quantitative description of flow distribution could be used to detect microvascular impairment. Liver imaging in ULM is more challenging due to respiratory, cardiac, stomach motions28 and most of the liver is placed behind the ribs, next to the lungs, where strong reflection can occur. Still, we were able to map a large volume of liver (65 × 100 × 82 mm3) and to retrieve and quantify 3D maps. Translation to humans is envisioned with the current technology and could benefit to patients with chronic liver disease. Early detection of chronic liver diseases is particularly important, but current non-invasive diagnostic methods are insensitive to early lesions. 3D ULM has the potential to detect early liver fibrosis as well as early development of hepatic tumour. Accurately capturing the dynamics and structure of microcirculation across entire organs enables the creation of detailed datasets that cover a broad range of physiological and pathological conditions. It would enable the development of sophisticated algorithms in artificial intelligence and machine learning capable of detecting patterns and anomalies in medical images with remarkable precision, often surpassing human capabilities. Moreover, the concept of digital twins, a virtual replica of a patient's physical characteristics, marks a significant advancement in personalized medicine. These digital models allow for simulations and predictions of disease progression and treatment outcomes. Leveraging extensive datasets, healthcare providers can customize treatments based on individual patient profiles, leading to more effective and efficient care. Thus, by mapping and analysing microcirculation across whole organs, researchers can generate vast and rich datasets that are invaluable for advancing AI-driven medical analysis, digital twins, and comprehensive medicine. The method relies on the development of a custom matrix array and post-processing algorithms. The matrix array aperture is large compared to conventional ultrasonic matrix arrays, allowing its footprint to cover entire swine organs like the heart and the kidney. The piezoelectric elements are large, three times larger than the wavelength, allowing only 252 piezoelectric elements to cover the entire aperture. As a matter of comparison, to cover the same aperture with half-wavelength-size piezoelectric elements used in conventional arrays, a total of 15042 elements would have been needed, which is simply not achievable with current technologies. Large elements also present the advantage of delivering more energy in the medium of interest and being more sensitive in receiving compared to half-wavelength elements. This is particularly important when comparing to sparse array techniques, often limited by their low signal-to-noise ratio29. Large elements must be combined into a diverging lens, which has the ability to lower the directivity of large elements, which is required to maximize the antenna gain of the array. A diverging lens can be of different geometry: plano-convex, plano-concave or compound, depending on the speed of sound of the lens material. For an equal number of elements, the multi-lens array was shown to be more sensitive than the small-aperture probe and sparse array. Sensitivity of small aperture probe suffered on the side of the volume mainly due to the limited angular aperture associated to small aperture. Sensitivity of the sparse array was particularly low in the far field due to the small surface area of the small elements, which limits the amount of energy transmitted and received. Row-column addressable (RCA) arrays have emerged as a promising alternative to fully populated matrix probes for 3D ultrasound imaging30,31,32, including applications such as ultrafast imaging33,34 and Ultrasound Localization Microscopy (ULM)35. By independently addressing rows and columns, RCA arrays drastically reduce the number of active channels, simplifying hardware architecture. However, the design of RCA arrays imposes certain limitations. Specifically, achieving a large aperture suitable for imaging entire organs is challenging due to the inherently large elevational dimension, which can lead to signal decorrelation and degraded image quality. To address this, recent developments have introduced RCA arrays equipped with acoustic lenses36,37 or mechanically curved configurations24 to enable diverging wave transmission and increase the field of view. While the row-column approach provides higher frequencies and improved spatial resolution with a reduced number of active channels, it is better suited for superficial or medium-depth imaging. In contrast, the proposed multi-lens array was specifically designed to achieve a larger 3D field of view and greater penetration depth, which are essential for imaging deep and voluminous whole abdominal organs such as the liver, kidney, and heart. The two systems are thus complementary, each optimized for distinct anatomical targets and imaging requirements. While fully populated matrix arrays have demonstrated volumetric ULM in localized brain regions, their field of view and application remain organ-specific38,39. Our custom multi-lens array approach offers scalable, whole-organ imaging across diverse deep tissues, though adaptation for transcranial human use will require dedicated redesign. The 1 MHz central frequency of the multi-lens array could be an important parameter to enable ultrasonic waves to travel through the human skull, by reducing high attenuation and aberration. Thus, it could have the potential to image the whole brain vasculature in large animal models or humans through the skull, offering promising capabilities for the early diagnosis of neurovascular pathologies40,41. The multi-lens array probe was characterized in vitro. The 3D Velocity map was quantified and a laminar flow was characterized by the Poiseuille profile. The profile was used to calculate a spatial resolution using dynamic information9 and a resolution of 75 µm was found, corresponding to a resolution of λ/20, which is in agreement with the literature for ULM methods42,43. Cost of the system (less than €200k) was considered low in comparison to the investment usually required for fully populated matrix arrays (~€1 M). Like any other approach, the multi-lens array approach has some limitations. In this study, 252 elements were chosen to fit the research ultrasound device composed of 256 channels. A higher number of elements could have been chosen to achieve an even larger field of view. For the same configurations, an array of 1008 elements driven with multiplexed approaches44,45 could achieve a field of view of 200 mm by 160 mm. The current spatial resolution, estimated at 75 × 75 × 75 µm3 could be not enough to visualize the smallest capillaries (less than 10 µm). It could be improved by developing multi-lens arrays operating at higher frequency (2 to 3 MHz), while maintaining a good penetration in the far field. Thanks to 3D ULM, grating lobes do not appear to be a limitation and did not appear on the density and velocity maps due to the two thresholds applied during the localization process. The first is based on signal amplitudes which allows to suppress high random noise and grating lobes that have lower amplitudes than the main lobe of microbubbles. This processing step could be optimized by modelling a more realistic PSF, which takes into account grating lobes instead of a Gaussian PSF, which only fits the main lobe. Machine learning approaches could also be an interesting solution to maximize the number of detected microbubbles46. Quasi real-time contrast-enhanced Doppler imaging (0.3 vol s−1 at a grid reconstruction resolution of λ ≈ 1.5 mm) with the multi-lens probe was employed as a guidance tool to facilitate precise probe positioning and to confirm effective acoustic coupling throughout the acquisition process. To further improve and accelerate experimental procedures, the development of a faster algorithm for real-time contrast-enhanced Doppler imaging could be investigated. In addition, for real-time B-mode imaging, B-mode-based intra-block motion correction, colour Doppler or elastography modes, grating lobes could remain an issue. Alternatively, introducing a randomized distribution of element positions within the aperture could break the array symmetry and suppress grating lobes. In addition to hardware-level improvements, several post-processing strategies could be employed to mitigate image artifacts and enhance resolution, including deconvolution techniques47,48, multiply and sum beamforming techniques49,50, spatiotemporal matrix beamforming51, or short-lag spatial coherence of backscattered echoes52 could be used. Finally, advanced machine learning based approaches may offer further enhancement of image quality by learning to suppress noise and artifacts adaptively53. Regarding the microbubble injection protocol employed in this study, it was not optimized for clinical practice. Future work will focus on improving and adapting this protocol for clinical applications. The size of the array may cause patient discomfort when pressure is applied to ensure acoustic coupling. To alleviate this issue, an acoustically transparent pad could be placed in front of the probe to distribute pressure more evenly and enhance patient comfort54. Thanks to its low central frequency, the multi-lens probe is not too impacted by attenuation and imaging depth could be increased in the presence of the pad. Furthermore, probe design could be adapted and optimized to accommodate specific organ features, such as their size and vascular complexity. Reduced sensitivity can be observed at the edges of the density volumes (coronary network, Fig. To alleviate this issue, different approaches could be considered, such as increasing intersources pitch or transmitting successively with sources located on the sides of the probe. The in vivo 3D ULM images could be enhanced by applying the SVD beamformer to correct phase aberration through different tissues55. Furthermore, the detection of slowly moving MBs through the small vessels could be improved using adaptive spatiotemporal SVD56, or amplitude-modulated SVD for ultrafast imaging57. Motion artifacts were minimized through an optimized experimental setup and acquisition strategy designed to reduce physiological motion in the data. An inter-block motion correction algorithm was applied, resulting in improved MB density and alignment of vascular structures. While B-mode–based motion correction is a common alternative in ULM literature10,58, its effectiveness requires a high B-mode image quality, which could not be achieved by the current multi-lens probe. To assess intra-block motion, each block was subdivided into three shorter segments to compute intra-block power Doppler volumes. This analysis, detailed in the Supplemental Materials, demonstrated the presence of residual motion within blocks and the benefit of increased temporal segmentation. Further improvement could be achieved by implementing frame-by-frame intra-block motion correction, enabling more precise compensation for physiological motion such as pulsatility. SVD on short temporal sliding windows and advancing methods such as Iterative Closest Point (ICP)59, could also be used to improve the localization of the slowest bubbles. While individual glomeruli were not explicitly segmented in this study, their size (~150 µm) falls within the resolution range of our imaging system. Previous studies have demonstrated glomerular detection using Ultrasound Localization Microscopy11,60, but required extensive postprocessing methods. In contrast, our approach prioritizes whole-organ 3D vascular imaging with flow dynamics, enabling global assessment of renal perfusion and microvascular integrity. Future work may incorporate dedicated segmentation strategies to extract glomerular features from these volumetric datasets. The liver imaging results presented in this study are preliminary and currently show limited vascular detail compared to other organs. This can be attributed to the inherent complexity of hepatic microvasculature, as well as suboptimal acquisition parameters not specifically optimized for liver imaging. While major vessels were captured in 3D with flow information, further improvements, such as optimized microbubble dilution, and longer acquisition durations, will be necessary to fully demonstrate the super-resolution potential of the system in the liver. Future work will focus on addressing these factors to enhance vascular detail and clinical relevance. While CT angiography was used to validate kidney imaging, it was not performed for the liver due to the complexity of contrast timing and breath-hold synchronization in porcine models. As liver angiography is not standard clinical or preclinical practice, the liver results presented here serve as a feasibility demonstration, with future studies planned to include anatomical validation. In this study, we introduced a large multi-lens array probe combined with a robust post-processing pipeline to assess vascular networks and dynamics at the micro-scale across large volumes. The method was rigorously validated through simulations as well as in vitro, ex vivo, and in vivo experiments on the kidney and liver. This approach holds significant potential to become a pivotal tool for preclinical research and clinical applications, particularly when evaluating complex microcirculation networks in 3D over large fields of view. Its potential is further underscored by the simplicity of the underlying technology, making it highly accessible for widespread adoption in research laboratories and hospitals. The experiments were approved by the animal ethics committee and were carried out according to project authorization [#28654-2020121509545743]. The pigs are housed in group enclosures with solid floors, providing a surface area exceeding the standard requirements (minimum 0.7 m² per animal). The enclosures are equipped with alternating day and night lighting, maintained at a temperature of 18 to 22 °C, with humidity levels ranging from 45% to 65%, and an air exchange rate of 10 volumes per hour. The animals are fed daily with appropriate pellets at a rate of 20 g kg−1 body weight, have continuous access to fresh drinking water, and are provided with sawdust bedding and toys to promote their well-being. The same multi-lens array probe was used in simulation, for in vitro validation and during ex vivo and in vivo experiments. The probe was composed of three acoustic principal layers. The first layer consists of 252 large transducer matrix piezocomposite elements (14 × 18). The size of an individual element was set to 4.5 × 4.5 mm² representing 3 times the acoustic wavelength (3λ² ≈ 4.5 mm², wavelength; λ ≈ 1.5 mm). The pitch between elements was set to 5.48 mm (corresponding 3.65λ) in both directions, this choice represents a compromise design to avoid large lens overlap while limiting grating lobe artefacts and preserving pressure field uniformity. The transducer layer was aligned with a second layer composed of diverging acoustic lenses (plano-convex lenses), also arranged in 14 × 18 configuration. The two layers were assembled with structural adhesive and an acoustic matching layer. A third flat lens layer (silicon plano-concave lens) was moulded above the diverging lens layer, completing the assembly of a compound diverging lens. Other layers as backing and mechanical stiffener were added behind the piezocomposite. Technical schematic details of the custom multi-lens probe are shown in Supplementary Fig. The central frequency was set to 1 MHz with a 67% bandwidth to minimize attenuation. The multi-lens array probe performance was assessed through simulation using k-Wave toolbox61. The transmit pressure field of a two-cycle pulse from an individual element of the array was simulated in a volume of 60 × 60 × 60 mm3 within a grid spacing of λ/10 (λ = 1.54 mm) and a time step of 30 ns. It was compared to the transmit pressure field of a large element of the same size without a lens and a small element (λ/2 × λ/2) used in conventional array probes. The speed of sound and density of the simulated medium were set to 1540 m s−1 and 1000 kg m−3, respectively. The maximum acoustic pressure during propagation was recorded in the volume of interest and presented in decibels (dB) for the three elements, as shown in Fig. The amplitude profile of the pressure field was used to quantify directivity as the angle at −6 dB at a 35 mm depth along a circular arc. Comparison results for the three elements are plotted in Fig. A synthetic phantom was designed consisting in three scatterers at depths of 30, 45, 60 mm, distributed at the centre of a volume of 70 × 104 × 100 mm3. The multi-lens probe and a synthetic aperture imaging sequence62 were simulated to obtain Radio Frequency (RF) signals of the scatterers. RF signals were beamformed (detailed in the section Beamforming processing) and 3D B-mode volumes were displayed to visualize numerical points spread functions as a function of depth (Supplementary Fig. The simulation was executed in CUDA language, provided by the k-Wave toolbox, and processed on GPU units of NVIDIA®, model RTX A6000 (GPU Memory = 48 giga bytes). The points spread functions were quantified by applying maximum intensity projection in the lateral direction to assess the grating lobes and the lateral resolution. Sensitivity simulations in receive were conducted using the Field II toolbox63,64. The multi-lens array probe was compared to two arrays with the same frequency (1 MHz) and number of elements (252) but with small elements size (λ/2 × λ/2). Sensitivity of a dense matrix probe and a large sparse array probe (with the same aperture size as the multi-lens probe) was studied. Sensitivity can be defined as the array capability to detect weak acoustic signals. In this study, it was defined as the intensity of received signals from a small transducer by the array probes across various spatial locations. The sensitivity responses of the three probes were evaluated through the following steps: a small transmitter, with a size of λ/2, was modelled within a 105 × 100 × 100 mm³ volume to emit an acoustic wave toward the matrix array probes at different locations on a grid spaced of λ in all spatial directions. The arrays received the ultrasonic waves and simulated RF signals were stored in memory for each point location of the transmitter. where \({{{{\rm{RF}}}}}_{i}({{{\rm{x}}}},{{{\rm{y}}}},{{{\rm{z}}}},{{{\rm{t}}}})\) is the Radio-frequency signal as a function of time t, recorded by the element i for the transmitter located at the (x,y,z) coordinates. \({t}_{0}\) and T are the start and end recording time, respectively. Intensity spatial distributions were normalized with the maximum absolute intensity value obtained by the multi-lens array probe and presented in decibels scale (dB), to observe regions of highest and lowest intensity within the field of view. The sensitivity response of the three probes was compared and assessed and by presenting the intensity threshold level as a function of the number of voxels within the volume of 105 × 100 × 100 mm3 as shown in Fig. Several experimental studies were performed using the multi-lens array, including point spread function characterization, resolution and flow rate validations on a large flow phantom, ex vivo assessment of an isolated porcine heart, and in vivo proof of concept on a porcine kidney and liver. For the in vitro studies, a flow phantom consisting of a polyethylene tube immersed in a water tank was imaged (inlet diameter ~870 µm, depth = 60 mm). A solution of 100 µL of microbubbles (Sonovue® Braco, Italy) mixed with 100 mL of water was continuously injected into the tube using a syringe pump (KDS Legato 130, KD Scientific, Holliston, USA) at a rate of 75 mL h−1 during acquisition. Additionally, imaging was performed on a twisted-tube configuration to assess more complex flow patterns. This setup used a polyethylene tube with an inner diameter of 870 µm. The tube was inclined, extending from 42 mm to 80 mm of imaging depth. For the ex vivo experiment on the isolated perfused heart, a porcine heart from a 30 ± 1 kg pig was anesthetized, explanted, cannulated at the aorta. Retrograde perfusion was established through the aorta according to the Langendorff method10,14,24, with an oxygenated Krebs-Henseleit solution and a perfusion rate of approximately 0.25 L min−1. The heart was immersed in a box made of a TPX polymer, chosen for its low acoustic attenuation and impedance mismatch. Heart motion was reduced by setting the Krebs solution temperature to 3 C°. A 100% concentration of MBs solution was added to the perfusion flow at 0.25 mL min−1 and injected into the aorta using a syringe pump set to a rate of 15 mL h−1, corresponding to a 1:1000 dilution of the initial MB solution. The multi-lens array was positioned in the apical view orientation (Fig. For the in vivo experiments, the studies were performed on three swines (12 weeks old and weighing 30 to 36 kg). The swines were anesthetized with sedation with Zoletil 0.1 mL kg−1 IM, then Comfortan 1 mL IV, then isoflurane 2% anaesthesia with air-oxygen mixture, after tracheal intubation and the use of a Minerve Alpha 100® respirator. X-ray dye (iodixanol, Visipaque™ 320, GE Healthcare) was injected through the catheter and GE® (model OEC 9900 Elite) X-ray generator was used to image the kidney artery network. The images of renal angiography of porcine 1 were stored in memory and illustrated in the Supplementary Fig. Real-time B-mode imaging was performed by a trained physician with a linear phased array connected to portable ultrasound scanner (Philips® Lumify, probe S4-1U) to acquire B-mode images for initial anatomical localization and alignment. The images were stored in memory. The multi-lens array probe was positioned in front of the kidney and fixed using an articulated arm. Subsequently, quasi-real-time contrast-enhanced Doppler imaging with the multi-lens probe was employed as a guidance tool to facilitate precise probe positioning and to confirm effective acoustic coupling throughout the acquisition process. An initial bolus of 1 mL of microbubble solution was injected through the intravenous system into the porcine ear. This was followed by additional intermittent bolus injections of 0.5 mL every minute for the rest of the acquisition. The ultrasound acquisitions were triggered between two respiratory phases to minimize motions artefacts. Similarly, real-time B-mode imaging was performed by a trained physician to image the liver network after MBs injection. The multi-lens array probe was positioned in front of the liver, fixed using an articulated arm, and an initial bolus of 1 mL of microbubble solution was injected through the intravenous system into the porcine ear. This was followed by additional intermittent bolus injections of 0.5 mL every minute for the rest of the acquisition. The ultrasound acquisitions were triggered between two respiratory phases to minimize motion artifacts. Physiological parameters during the experiment (Electrocardiogram, respiratory signals) were monitored and acquired with a PowerLab device (ADInstruments®, Dunedin, New Zealand), equipped with spirometers and differential biological potential amplifiers, and managed by LabChart software. To ensure efficient acoustic coupling and avoid air bubbles between the probe surface and the tissue, specifically, the skin in in vivo experiments and the heart container in ex vivo settings, a custom-degassed gel was employed for both ex vivo and in vivo experiments. The gel consisted of a 1:1 mixture of water and commercial ultrasound gel, which was degassed via centrifugation for 4 min. Following gel application to the probe surface, the probe was carefully angled and gradually applied to the tissue to eliminate air bubbles and ensure optimal acoustic coupling. The ultrasound acquisition sequences were performed using the multi-lens array probe described in section (Multi-lens array) driven by a programmable ultrasound device with 256 transmit-receive channels (Vantage, Verasonics®, USA). Transmit schemes were programmed to perform 3D ultrafast ultrasound imaging in order to preserve a sufficient temporal resolution. The selected configuration consisted of 16 sequential transmissions, each using a single element of the probe to emit a diverging wave. Backscattered echoes were recorded with all elements after each transmission and RF data were stored in memory, digitalized at 100% bandwidth (two samples per wavelength). Diverging wave emissions consisted in short pulses of two cycles centred at a frequency of 1 MHz. Given the imaging depth, the pulse repetition frequency was set to 5000 Hz and associated volume rate was 312.5 Hz. The acquisitions were divided into blocks of 830 ms, corresponding to 256 volumes for the in vitro and ex vivo experiments, and into blocks of 1.2 s for the in vivo kidney and liver experiments. The number of blocks in one acquisition varied depending on the application: 1000 blocks were acquired for the tube phantom, 400 blocks for the ex vivo heart, 290 blocks for the kidney and 133 blocks for the liver porcine. Each block was repeated every 300 ms, allowing the RF data block to be stored on the memory disk. The acquisition durations were as follows: 13 min for the flow phantom imaging shown in Fig. 2, 5 minutes for the isolated porcine heart in Fig. The total amount of data per acquisition block varied from 1.1 gigabytes to 2.8 gigabytes. 3D rigid motion correction was performed between blocks using a Power Doppler-based registration workflow14. Specifically, both the 3D Power Doppler volumes and the corresponding microbubble (MB) tracked positions for each block were used as inputs. Intensity information in 3D power Doppler images was used to compute the transformation that maximized the Advanced Normalized Correlation between a selected reference Power Doppler volume (chosen as the one with the highest number of tracked MBs) and each of the other 3D Power Doppler volumes to be aligned. The registration algorithm was implemented using Elastix 5.0 (https://elastix.lumc.nl/) and performed in full 3D space. A rigid transformation comprising both rotation and translation components was estimated and subsequently applied to the microbubble (MB) tracks to spatially align all datasets across blocks. The transformation matrix was computed using Elastix's rigid parameter configuration, which employs a multi-resolution registration framework based on the MultiResolutionRegistration method and uses a B-spline interpolator (BSplineInterpolator) in combination with the Euler transform model. For intra-block motion correction presented in the Supplementary Fig. 4, we implemented a 3D registration strategy within each acquisition block. Each block was arbitrary divided into three temporal sub-blocks, for which 3D Power Doppler volumes were computed and rigidly registered using the same framework as inter-block correction. The corrected sub-blocks were then concatenated to form a motion-compensated volume for the full block. This corrected Power Doppler volume, along with the corrected microbubble tracks, was then used as input for the standard inter-block motion correction. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data availability statement: All 3D volumes (tubes, heart, kidney and liver, provided in both density and speed) are available as NIfTI files at https://doi.org/10.5281/zenodo.16965337 (ref. 65) enabling independent analysis and quantification. All codes used to generate the results and figures in this study are available upon request. The human microcirculation – regulation of flow and beyond. Bie et al. Photon-counting CT: Review of initial clinical results. Bissell, M. M. et al. 4D Flow cardiovascular magnetic resonance consensus statement: 2023 update. Tanter, M. & Fink, M. Ultrafast imaging in biomedical ultrasound. Sigrist, R. M. S., Liau, J., Kaffas, A. E., Chammas, M. C. & Willmann, J. K. Ultrasound Elastography: Review of techniques and clinical applications. Huang, C. et al. Super-resolution ultrasound localization microscopy based on a high frame-rate clinical ultrasound scanner: an in-human feasibility study. Errico, C. et al. Ultrafast ultrasound localization microscopy for deep super-resolution vascular imaging. Christensen-Jeffries, K., Browning, R. J., Tang, M.-X., Dunsby, C. & Eckersley, R. J. In vivo acoustic super-resolution and super-resolved velocity mapping using microbubbles. Demené, C. et al. Transcranial ultrafast ultrasound localization microscopy of brain vasculature in patients. Yan, J. et al. Transthoracic ultrasound localization microscopy of myocardial vasculature in patients. Bodard, S. et al. Ultrasound localization microscopy of the human kidney allograft on a clinical ultrasound scanner. Fenster, A., Parraga, G. & Bax, J. Three-dimensional ultrasound scanning. Chavignon, A., Hingot, V., Orset, C., Vivien, D. & Couture, O. 3D transcranial ultrasound localization microscopy for discrimination between ischemic and hemorrhagic stroke in early phase. Demeulenaere, O. et al. Coronary flow assessment using 3-dimensional ultrafast ultrasound localization microscopy. In vivo whole brain microvascular imaging in mice using transcranial 3D Ultrasound Localization Microscopy. Heiles, B. et al. Volumetric Ultrasound Localization Microscopy of the Whole Rat Brain Microvasculature. Whole organ volumetric sensing Ultrasound Localization Microscopy for characterization of kidney structure. & Couture, O. Volumetric Ultrasound Localization Microscopy. Favre, H., Pernot, M., Tanter, M. & Papadacci, C. Boosting transducer matrix sensitivity for 3D large field ultrasound localization microscopy using a multi-lens diffracting layer: a simulation study. Favre, H., Pernot, M., Tanter, M. & Papadacci, C. Transcranial 3D ultrasound localization microscopy using a large element matrix array with a multi-lens diffracting layer: an in vitro study. Bourquin, C., Porée, J., Lesage, F. & Provost, J. In vivo pulsatility measurement of cerebral microcirculation in rodents using dynamic ultrasound localization microscopy. Bourquin, C. et al. Quantitative pulsatility measurements using 3D dynamic ultrasound localization microscopy. Caudoux, M. et al. Curved Toroidal Row Column Addressed Transducer for 3D Ultrafast Ultrasound Imaging. Demeulenaere, O. et al. Assessment of coronary microcirculation alterations in a porcine model of no-reflow using ultrasound localization microscopy: a proof of concept study. Pedreira, O. et al. Quantitative stiffness assessment of cardiac grafts using ultrasound in a porcine model: A tissue biomarker for heart transplantation. Engel, J. E. et al. Recovery of renal function following kidney-specific VEGF therapy in experimental renovascular disease. Huang, C. et al. Super-resolution ultrasound localization microscopy based on a high frame-rate clinical ultrasound scanner: an in-human feasibility study. Experimental 3-D Ultrasound Imaging with 2-D Sparse Arrays using Focused and Diverging Waves. 3-D ultrasound imaging performance of a row-column addressed 2-D array transducer: A measurement study. In 2013 IEEE International Ultrasonics Symposium (IUS) 1460–1463 https://doi.org/10.1109/ULTSYM.2013.0370 (2013). Rasmussen, M. F., Christiansen, T. L., Thomsen, E. V. & Jensen, J. 3-D imaging using row-column-addressed arrays with integrated apodization - part i: apodization design and line element beamforming. Christiansen, T. L. et al. 3-D imaging using row–column-addressed arrays with integrated apodization— part ii: transducer fabrication and experimental results. Flesch, M. et al. 4D in vivo ultrafast ultrasound imaging using a row-column addressed matrix and coherently-compounded orthogonal plane waves. Sauvage, J. et al. 4D Functional imaging of the rat brain using a large aperture row-column array. Tan, Q. et al. Transcutaneous Imaging of Rabbit Kidney Using 3-D Acoustic Wave Sparsely Activated Localization Microscopy With a Row-Column-Addressed Array. Audoin, M., Salari, A., Tomov, B. G., Arendt Jensen, J. & Thomsen, E. V. Novel Diverging Acoustic Lens Geometries for Row-Column Array Transducers. in 2023 IEEE International Ultrasonics Symposium (IUS) 1–4. Salari, A. et al. Beamformer for a lensed row–column array in 3-D Ultrasound Imaging. Coudert, A. et al. 3-D Transcranial Ultrasound Localization Microscopy Reveals Major Arteries in the Sheep Brain. Xing, P. et al. 3D ultrasound localization microscopy of the nonhuman primate brain. SAHS, A. L. Extracerebral neurovascular disease: a short review. Sargurupremraj, M. et al. Genetic complexities of cerebral small vessel disease, blood pressure, and dementia. Christensen-Jeffries, K. et al. Super-resolution Ultrasound Imaging. McCall, J. R., Santibanez, F., Belgharbi, H., Pinton, G. F. & Dayton, P. A. Non-invasive transcranial volumetric ultrasound localization microscopy of the rat brain with continuous, high volume-rate acquisition. Wang, N. et al. A Multiplexed 32 × 32 2D Matrix Array transducer for flexible sub-aperture volumetric ultrasound imaging. Highly integrated multiplexing and buffering electronics for large aperture ultrasonic arrays. Shin, Y. et al. Context-aware deep learning enables high-efficacy localization of high concentration microbubbles for super-resolution ultrasound localization microscopy. Yang, T. C. On conventional beamforming and deconvolution. Improving the efficiency of deconvolution algorithms for sound source localization. Bertolo, A., Sauvage, J., Tanter, M., Pernot, M. & Deffieux, T. XDoppler: Cross-Correlation of Orthogonal Apertures for 3D Blood Flow Imaging. Ultrafast 3-D Ultrasound imaging using row–column array-specific frame-multiply-and-sum beamforming. Berthon, B. et al. Spatiotemporal matrix image formation for programmable ultrasound scanners. Lediju, M. A., Trahey, G. E., Byram, B. C. & Dahl, J. J. Short-lag spatial coherence of backscattered echoes: imaging characteristics. Stevens, T. S. W. et al. Dehazing Ultrasound using diffusion models. Bendjador, H., Deffieux, T. & Tanter, M. The SVD beamformer: physical principles and application to ultrafast adaptive ultrasound. Adaptive Spatiotemporal SVD Clutter Filtering for Ultrafast Doppler Imaging Using Similarity of Spatial Singular Vectors | IEEE J. Mag. Zhang, G. et al. Amplitude-modulated singular value decomposition for ultrafast ultrasound imaging of gas vesicles. Dencks, S. et al. Super-resolution ultrasound: from data acquisition and motion correction to localization, tracking, and evaluation. Zhang, Z. Iterative point matching for registration of free-form curves and surfaces. Denis, L. et al. Sensing ultrasound localization microscopy for the visualization of glomeruli in living rats and humans. Treeby, B. E. & Cox, B. T. k-Wave: MATLAB toolbox for the simulation and reconstruction of photoacoustic wave fields. A., Nikolov, S. I., Gammelmark, K. L. & Pedersen, M. H. Synthetic aperture ultrasound imaging. Field: A program for simulating ultrasound systems: 10th nordic-baltic conference on biomedical imaging. Calculation of pressure fields from arbitrarily shaped, apodized, and excited ultrasound transducers. HAIDOUR, N. et al. 3D ULM data of Multi-lens ultrasound arrays enable large scale three-dimensional micro-vascularization characterization over whole organs. This work was supported by Inserm research accelerator (Inserm ART) in Biomedical Ultrasound and was funded by the European Union, European Research Council (ERC) grant agreement number 101042470 to C.P. The GPU RTX A6000 used in this work was awarded by the NVIDIA Academic Hardware Grant Programme to C.P. We gratefully acknowledge Vermon® for manufacturing the custom multi-lens ultrasound probe. We thank Alexandre Dizeux (Physics for Medicine) for his great assistance with data visualization and for creating the cover image. Institute Physics for Medicine Paris – INSERM, ESPCI Paris-PSL, CNRS, Paris, France Nabil Haidour, Hugues Favre, Philippe Mateo, Juliette Reydet, Mathieu Pernot, Mickael Tanter & Clement Papadacci INSERM U955-IMRB, University Paris-Est Creteil, National Veterinary School of Alfort, Paris, France Alain Bizé, Lucien Sambin, Jianping Dai, Paul-Matthieu Chiaroni & Bijan Ghaleh Interventional Cardiology Department, Henri Mondor Hospital, Creteil, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar N.H., H.F. and J.R. designed the simulation pipeline. P.M. designed the Langendorff ex vivo setup. designed the in vivo setup. All authors edited and approved the final version of the manuscript. The authors declare no competing interests. Nature Communications thanks Georges Chabouh, Matthieu Toulemonde and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Haidour, N., Favre, H., Mateo, P. et al. Multi-lens ultrasound arrays enable large scale three-dimensional micro-vascularization characterization over whole organs. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            